<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Marburg_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Marburg/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="header_wrapper"><DIV class="menu_wrapper" role="navigation" aria-label="Main Menu"><UL class="menu"><LI><A href="#">Team ▾</A></LI><LI><A href="/Team:Marburg/Team">Team</A></LI><LI><A href="/Team:Marburg/Collaborations">Collaborations</A></LI><LI><A href="/Team:Marburg/Attributions">Attributions</A></LI></UL><LI><A href="#">Project ▾</A></LI><LI><A href="/Team:Marburg/Description">Description</A></LI><LI><A href="/Team:Marburg/Design">Design</A></LI><LI><A href="/Team:Marburg/Experiments">Experiments</A></LI><LI><A href="/Team:Marburg/Notebook">Notebook</A></LI><LI><A href="/Team:Marburg/InterLab">InterLab</A></LI><LI><A href="/Team:Marburg/Model">Model</A></LI><LI><A href="/Team:Marburg/Results">Results</A></LI><LI><A href="/Team:Marburg/Demonstrate">Demonstrate</A></LI><LI><A href="/Team:Marburg/Measurement">Measurement</A></LI><LI><A href="/Team:Marburg/Labautomation">Labautomation</A></LI><LI><A href="#">Parts ▾</A></LI><LI><A href="/Team:Marburg/Parts">Parts Overview</A></LI><LI><A href="/Team:Marburg/Basic_Part">Basic Parts</A></LI><LI><A href="/Team:Marburg/Composite_Part">Composite Part</A></LI><LI><A href="/Team:Marburg/Part_Collection">Part Collection</A></LI><LI><A href="/Team:Marburg/Improve">Improve</A></LI><LI><A href="/Team:Marburg/Safety">Safety</A></LI><LI><A href="#">Human Practices ▾</A></LI><LI><A href="/Team:Marburg/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Marburg/Public_Engagement">Public Engagement</A></LI><LI><A href="#">Awards ▾</A></LI><LI><A href="/Team:Marburg/Human_Practices">Integrated Human Practices</A></LI><LI><A href="/Team:Marburg/Public_Engagement">Education and Public Engagement</A></LI><LI><A href="/Team:Marburg/Model">Model</A></LI><LI><A href="/Team:Marburg/Measurement">Measurement</A></LI><LI><A href="/Team:Marburg/Basic_Part">Best New Basic Part</A></LI><LI><A href="/Team:Marburg/Composite_Part">Best New Composite Part</A></LI><LI><A href="/Team:Marburg/Part_Collection">Best Part Collection</A></LI><LI><A href="/Team:Marburg/Software">Software</A></LI><LI><A href="https://igem.org/2018_Judging_Form?id=2560">Judging Form</A></LI></DIV></DIV><MAIN><DIV class="titleWrapper"><DIV class="title">Results</DIV></DIV><ARTICLE><P><I>However beautiful the strategy, you should occasionally look at the results</I><B>   -- Winston Churchill</B></P>
After having conceived our projects, we went to the lab and tried to fulfill our expectations. Successes and failure were used to improve our projects strategy by iterating theoretical and practical working steps.
On this page, we want to convince you that we managed to realize some of our visions and are able to show achievements in all of our sub-projects.





<DIV class="collapsible"><DIV class="btn_expand"><DIV class="btn_e_title">Foundational experiments with  V. natriegens</DIV><DIV class="btn_e_desc"><B>Rational design</B> is only possible when all factors are known. Here we show how we investigated basic <B>facts</B> about <I>V. natriegens </I> to help drive future research.
		</DIV></DIV><DIV class="content"><DIV class="collapsible"><DIV class="btn_expand">Cloning in <I> V. natriegens </I></DIV><DIV class="content"><H3>Gibson cloning in <I>Vibrio natriegens</I></H3><P>The next step in our project is to test different cloning methods.
A new and fast chassis needs a fast and clever cloning strategy. We wanted to show that the most commonly used cloning methods, Golden Gate cloning <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2677662/pdf/pone.0005553.pdf"><ABBR title="Golden Gate Shuffling: A One-Pot DNA Shuffling Method Based on Type IIs Restriction Enzymes">(Carola Engler <I>et al.</I>
2009)
  </ABBR></A></P><P>



 and Gibson assembly  

 <A href="https://www.nature.com/articles/nmeth.1318.pdf"><ABBR title="Enzymatic assembly of DNA molecules up to several hundred kilobases">(Daniel G Gibson <I>et al.</I>
2009),
  </ABBR></A><FIGURE style="width: 45%; float: right;"><FIGCAPTION><B>Picture 1</B> On the left we have a negative control and then the three plate top are the transformations with the plasmids prepared from <I>V. natriegens</I>. The plates below are the transformations with the plasmids prepared from <I>E. coli.</I> From left to right is ne number of the assemblies pYTK 1/RFP 1 to pYTK 3/RFP 3) 
		</FIGCAPTION></FIGURE>
 work, using the toolbox we designed for <I>V. natriegens</I>.  

Demonstrating the functionality of Gibson cloning in <I>V. natriegens</I>, we successfully exchanged a <DFN data-info="Green Fluorescent Protein"> GFP</DFN> dropout from the pYTK plasmid from the Duber Toolbox 

 <A href="https://pubs.acs.org/doi/pdfplus/10.1021/sb500366v"><ABBR title="A Highly Characterized Yeast Toolkit for Modular, Multipart Assembly">(Michael E. Lee <I>et al.</I>
2015),
  </ABBR></A> 




 with an <DFN data-info="Red Fluorescent Protein"> RFP</DFN> dropout. 


 <A href="https://2018.igem.org/Team:Marburg/Attributions"><ABBR title="Link to the atributions page">Alex Lepak 
  </ABBR></A> 

advised us to try if it made a difference if we take prepared plasmids from<I> E. coli</I> or from <I>Vibrio natriegens</I>. 
So in total we prepared six Gibson assemblies, combining RFP 1 to 3 (<I>V. natriegens</I>) with pYTK 1 to 3 ( <I>V. natriegens</I>) and RFP 1 to 3(<I>E. coli</I>) with pYTK 1 to 3 (<I>E. coli</I>).
After repeating one of the transformations we had colonies on every plate.





</P><P>

Consequently, we send sequenced samples of colonies showing red fluorescence and found all of them to harbor the correctly assembled plasmid without mutations.

Cloning efficiency  for Gibson assembly is usually higher than 90%, where very few cells still showed  green fluorescence. ( Picture 4)

</P><FIGURE style="width: 30%; float: left;"><FIGCAPTION><B>Picture 2</B> The 3 Plates top are Gibson with 5 fragments, 4 fragments and 3 fragments
    The two plates below are controls for the electroporation one with PYTK and one   
    without Plasmid. 
		</FIGCAPTION></FIGURE><P>



With this, we also prove that the efficiency of electroporation is high enough for Gibson cloning in <I>V. natriegens</I>.
We found no significant difference when taking the plasmids from an <I>E. coli </I> preparation.

After showing Gibson cloning with two fragments we wanted to check if we could do this method with up to five fragments. 

To test this we amplified pieces of the pYTK  and <DFN data-info="Red Fluorescent Protein"> RFP</DFN> plasmids according to figure X  and Y. 












Using the same protocol as mentioned before, and also successfully demonstrated correct assembly, which we affirmed by sequencing. 













While we had the Echo in our Lab, a machine for “pipetting” very low volumes, down to 2.5 nL, together we also tried doing Gibson assembly with it. The total volume of the mix was exactly one µL.
After assembling the samples in the 96 well plate and after transformation we got colonies for every cloning.
Again, we counted the cells to calculate the efficiency and again reached over 90% of correct transformants. 




<FIGURE style="width: 39%; float: right;"><FIGCAPTION><B>Picture 4</B> Picture of gibson assembly and aqua cloning in <I>V. natriegens</I> from the Echo
		</FIGCAPTION></FIGURE></P><H3>Golden Gate cloning and cloning in one day</H3>



 

Our Toolbox, the <A href="https://2018.igem.org/Team:Marburg/Part_Collection"><ABBR title="Link to the part collection page"> Marburg Collection
  </ABBR></A> is build for Golden Gate cloning and intended to be used in <I>V. natriegens</I>.

Implementing this method, we want to enable combinatorial building of plasmids in which you can decide on the position for every part.
To demonstrate that this method works, we devised an experiment, of building a <A href="https://static.igem.org/mediawiki/2018/9/9b/T--Marburg--Plasmid_card_GG_step_tag_.png"><ABBR title="Level1 Strep Tag"> Lvl2 strep
  </ABBR></A>
 tag.




To create the short tag we ordered two oligonucleotides for an annealing. For the backbone, we again took the pYTK plasmid to have a visual output to identify the incorrect colonies.


Triplicates of the annealing and the backbone were combined to give 9 combinations. We performed 18 variants of the experiment in parallel, in <I>V. natriegens </I>and<I> E. coli</I>, to have a direct comparison. 

After the Golden Gate assembly with BsmB1 we transformed 5µL of each mix by electroporation.
All clonings worked fine. Regrettably, the high efficiency resulted in a bacterial lawn, so  we couldn’t count the colonies in total. The ratio between fluorescing and non-fluorescing colonies looked quite good. To give a better impression we made pictures under UV light to visualize the colonies producing GFP better. 





<FIGURE style="width: 45%; float: right;"><FIGCAPTION><B>Picture 1</B>  
 ( A3/V1-3) and (A3/E1-3)       ( A2/V1-3) and (A2/E1-3)       ( A1/V1-3) and (A1/E1-3)
Plates from annealing A3 under the UV light table. The wrong colonies can be seen as stronger glowing spots ( still containing PYTK ( GFP)) instead of the strep tag
		</FIGCAPTION></FIGURE>






We sequenced in six different plasmids the Strep-tag and his flanking regions.  
The sequencings where all correct.





The next step for Golden Gate cloning after seeing that it worked, was to attempt if we could do the cloning from transforming the Golden Gate assembly to the miniprep in one day (12h) and build a level two plasmid in three days.

For every day the workflow is nearly the same. Transformation, recovery, plating, picking of colonies and harvesting the plasmids from the cells. Overnight you do the Golden Gate assembly for the next day.




So we made transformations of eleven level zero plasmids to assemble for the next day a LVL1 GFP 
<A href="https://static.igem.org/mediawiki/2018/9/91/T--Marburg--GG_Plasmidcard_LVL_1_GFP_.png"><ABBR title="Link to the part collection page"> LVL 1 RFP
  </ABBR></A>

)

 and a 

<A href="https://static.igem.org/mediawiki/2018/9/99/T--Marburg--GG_Plasmidcard_LVL_1_RFP_.png"><ABBR title="Link to the part collection page"> LVL 1 RFP
  </ABBR></A>
 plasmid.


From these level one plasmids we wanted to build a level two plasmid with the 
<A href="https://static.igem.org/mediawiki/2018/c/c2/T--Marburg--GG_Plasmidcard_LVL_Plasmid_RFP_and_RFP_.png"><ABBR title="Link to the part collection page"> RFP and the GFP
  </ABBR></A> cassette .


The only varying factor things in the timeline for each day are the times to pick colonies.
On the first day we needed 11.5h to get from the first transformed plasmids to the first finished minipreps.
Proving that cloning in under 12h is definitely possible. In Weinstock <I>et al.</I> 2016, it is recommended that you can see the first colonies after four hours and that incubation takes 3h. 
Also, we could see the first colonies after 4h. Unfortunately, not on all plates.  Still, except for one plasmid we could inoculate all after 6h. We then also incubated for 3h. 
Waiting for the last Plasmid to grow properly took two additional hours, so 13.5 in total.
Golden Gate assembly takes about 8.5h  

<FIGURE style="width: 45%; float: right;"><FIGCAPTION><B>Picture 2</B>  
 Pictures of the 6 plates from Annealing 3 (A3) with the Backbone prepared from E.coli (E1-E3) and from <I>Vibrio natriegens</I> (V1-V3). The glowing colonies are containing the undigested pYTK instead of the Step-tag
		</FIGCAPTION></FIGURE>


After transforming the LVL 1 and LVL 2 plasmids we got the impression that more complex plasmids take longer to grow.
So the whole workflow took us 53h to get the LVL 2 plasmid, which we confirmed later on by sequencing. 

One major problem is that the visual output of GFP and RFP is that it sometimes takes a lot of time. Our hypothesis is, that because many fluorescent proteins need oxygen to function, and <I>Vibrio natriegens</I> uses a lot of oxygen for his growth, local oxygen levels are depleted so much that they can’t become functional. Therefore, it takes time until we see the visual output on agar plates.
A possible solution could be using plates with vents.

For picking the LVL1 cells we waited nine hours and incubated 4.5h to be sure to select the correct colonies.
The LVL 2 cells took ten hours to grow properly. 
We think that Plasmid for complex proteins or over-expressing something, the metabolic burden could increase the time of waiting. 

<P>
Cloning in <I>Vibrio natriegens</I> on the other hand is still much faster than cloning in <I> E. coli</I>, where you also have to struggle with the same problems.
Weather you have a normal work day of eight hours or if you work like an iGEMer, you can save in each case a whole day by using Vibrio natriegens. That is immense. You can save third of the time by not waiting that long for cultures to grow.
Additionally, there is still a lot of potential to build faster <I>Vibrio natriegens</I> strains.
This could revolutionize cloning.
All this combined with lab automatisation, clever machines doing the cloning steps, as we did with with the echo or the opentron ,performing heat shock transformations in 96 well plates and pick automatically after automatically plating the colonies. You could start your cloning on Friday, and as we are used to say in Marburg: “ Monday its finished”.
This could be what accelerating synbio could look like.
</P><H3>Aquacloning</H3>

Aquacloning was first described by Beyer <I>et al.</I> (2015) and is a method for in vivo DNA assembly without the requirement of any additional enzymes. Similar to yeast recombination, linear PCR fragments are transformed into bacterial cells where the fragments are combined by the endogenous DNA repair machinery. The required fragments are PCR amplified from plasmid or genomic templates with primers that add homologous sequences (25 – 30 bp) to the neighboring fragments in a similar manner like Gibson Assembly. Aquacloning can be used for many different applications like site directed mutagenesis or assembling multiple fragments.
Initially, we tried to reproduce the experiment carried out by Beyer <I>et al.</I> (2015), to turn a sfGFP into a blue fluorescent protein by changing a single amino acid (Y66H). We could observe blue colonies and sequenced the plasmids to confirm the desired mutation. An alignment of the chromatogram is shown in figure 1. <FIGURE style="width: 50%; float: right"><FIGCAPTION><B>Figure 1: Alignement of Aquacloning. </B> The site directed mutagenesis performed with Aquaclonging (Y66H) was confirmed by sequencign for both tested clones. A tyrosin codon was changed to a histidine codon.
        </FIGCAPTION></FIGURE> 
After having shown the general applicability in <I>V. natriegens</I>, we tried to use Aquacloning to reproduce the results we gained with Gibson Assembly.  We were able to observe correctly assembled plasmids with three but neither with four nor with five fragments. 
The experiment was designed such that correctly assembled plasmids result in RFP expressing V. natriegens colonies. The sfGFP CDS of pYTK, the plasmid used in the previous experiment was replaced by a RFP CDS. The resulting agar plate is shown in figure 2. <FIGURE style="width: 50%; float: right"><FIGCAPTION><B> Figure 2: Gibson Cloning. </B> A plate full of colonies with a correctly assembed plasmid</FIGCAPTION></FIGURE>We do not observe a single white or green colony. The phenotypic appearance of the colonies can be seen as a strong evidence for correctly assembled plasmids. Therefore we were able to demonstrate high efficiency of Aquacloning with three fragments and therefore we present an in vivo alternative for Gibson Assembly.  
Due to the limited amount of time, we were not able to repeat the experiments with four and five fragments, but we are certain that optimizing transformation protocols, the amount and ratio of DNA fragments or the length of homologous flanks will result in successfully assembled plasmids even for this very challenging application. 



</DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Growth and Selection of <I> V. natriegens </I></DIV><DIV class="content"><H3><P>Growth experiments  </P></H3><FIGURE style="width: 45%; float: right;"><FIGCAPTION><B>Figure 1:</B> The Graph shows four growth curves three times V. natriegens (blue, red and orange) and once E. coli (NEB Turbo)( green)) 
		</FIGCAPTION></FIGURE><P>After reading papers about V. natriegens deciding to order the strain ATCC 14048, we were eager to make our first growth curves.</P><P>Because <I>V. natriegens</I> is a marine bacterium organism and from literature (et al), we knew that we had to supplement our standard LB media with an additional 1.5% NaCl. </P>

Being inexperienced with the new organism, we only managed to get a doubling time slightly better than for <I>E. coli.</I>  




Not disheartened by this, we did some more research and found several points we wanted to try.


<P>We tested the following media, LB 2.5 NaCl, LB v2 salts and Brainheart infusion v2 salts.  </P><FIGURE style="width: 45%; float: left;"><FIGCAPTION><B>Figure 2:</B> The Graph shows four growth curves three times V. natriegens (blue, red and orange) and once E. coli (NEB Turbo)( green) 
		</FIGCAPTION></FIGURE><P>v2 salts are: 204 mM NaCl, 4.2 mM KCl, 23.14mM MgCl<SUB>2</SUB> Brainheart infusion can not be autoclaved together with the v2 salts. From this we learned that growth was most vigorous in BHIv2, closely followed by LBv2. So we decided to make LBv2 our standard media going forward. It is cheaper as well as just being standard LB media, supplemented after autoclaving with the three common salts NaCl, KCl and MgCl<SUB>2</SUB>.</P><P>Additionally, some aspects of handling could be improved. For instance reducing the time outside of the incubator for measuring, thereby avoiding cooling of the media, and using baffled flasks to improve aeration. Applying these, we managed to get much better results, even beating the reported doubling time of 9.8 min </P><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC279347/"><ABBR title="PSEUDOMONAS NATRIEGENS, A MARINE BACTERIUM WITH A GENERATION TIME OF LESS THAN 10 MINUTES">(Eagon <I> et al.</I> 1962).</ABBR></A> 




	 The graph shows a growth curve of V. natriegens in BHIv2; V.natriegens in LBv2 and the fastest E.coli stain (NEB Turbo) in LB media.

<P> We achieved an incredible growth rate of 7 min. Also, using baffled flasks, our overnight cultures routinely reached OD's from 14 to over 16. By using 

<A href="https://2018.igem.org/Team:Marburg/Human_Practices"><ABBR title="link to integrated human practice">enpresso</ABBR></A> 

 capsules, which slowly release glucose into the media, we could even best that and reach staggering OD's of around 26.</P><H3>pH- Tolerance of <I>Vibrio natiregens</I></H3><P>A defining trait of microorganisms is their pH tolerance. Since we want to produce 3-hydroxypropionate in <I>V. natriegens</I> it is important to know how much acidity our strain tolerates. 
Therefore, we analyzed its growth under different pH conditions. In order to do this, we prepared LBv2 medium and tested on pHs ranging from 2.5 to 12. 
Then we inoculated them from a preculture of <I>V. natriegens</I>. After incubating for 15 minutes or for 6 hours we plated the cultures on LBv2 plates with a physiological pH of 7. The exact protocol can be seen in the experiments section.</P><P>
The experiment showed that<I> V. natriegens</I> tolerates a pH ranging from 6 to 9.5 and has its optimum at pH 7. It was observed that the growth capacity drops drastically when the pH drops under 6 or over 9.5. Nevertheless, in the span from 6 to 9.5 it shows sufficient growth for lab-work.  From these results, we learned, that a growth medium optimized for 3HPA production would have to have a pH of 9.5. This would create a larger buffer towards lower pH values, enabling a higher 3HPA yield without killing the organism. 
</P><P>
Furthermore, these results confirm that <I>V. natriegens</I> has no problems with pH values in physiological ranges. Since most growth media and buffers work with pH values inside this range, there shouldn’t be any problems related to the pH. Nevertheless, we have information on growth capacity, but we don’t know how protein expression or other cellular processes are influenced. More experiments should be made to further characterize the pH tolerance of <I>V. natriegens,</I> but our results delivered some basic first predictions.</P><H3>Salt experiments</H3><FIGURE style="width: 22%; float: right;"><FIGCAPTION><B>Picture 3</B>   After 6h V. natriegens is visually growing up to a NaCl concentration of 7%
 
		</FIGCAPTION></FIGURE><P>To find the perfect salt concentration for V. natriegens, we conducted new experiments using different salt concentrations ranging from standard LB media with no additional salt and then added NaCl in  1% steps, up to a concentration of 12 %  in total.
<FIGURE style="width: 22%; float: left;"><FIGCAPTION><B>Picture 4</B>   After 24h V. natriegens is visually growing to a NaCl concentration of 8%
 
		</FIGCAPTION></FIGURE>
We measured the OD600 undiluted too see to which concentration the cells grew in a certain period of time. 
In a salt concentration of 9%, V. natriegens reached an OD600 of 0.3 after 48h of incubation after slowly adapting from six hours of incubation to 7% NaCL and after 24 hours to 8% NaCl. ( Pictures 1 to 3 )</P><FIGURE style="width: 22%; float: right;"><FIGCAPTION><B>Picture 5</B>   After 48h V. natriegens is visually growing to a NaCl concentration of 9%
 
		</FIGCAPTION></FIGURE>






The pictures 11 to 13 show a gradient of the salt concentration from one to twelve % NaCl in total.



<P>Afterwards we incubated the cultures that showed no growth on media with the ideal  concentration of 2.5% and let them recover to  check if they were hibernating, but still alive after being exposed to the high salt concentration.
Until 10% of total salt in the media, V. natriegens did not recover from growing in higher salt concentrations.</P><H4>Antibiotic tolerance of <I>Vibrio natriegens</I></H4><P><FIGURE style="width: 45%; float: right;"><FIGCAPTION><B>Figure 6:</B> A) This picture shows spotted colonies of <I>V. natriegens </I> on  gradients of the most common antibiotics. Except for cycloserine and penicillin there as alay a certain point when the wild type stopped growing. So selection is possible. 
		</FIGCAPTION></FIGURE></P><P>
	The required concentrations for selection in <I>V. natriegens </I>is still unknown for most standard antibiotics. Since these were a prerequisite for our project, we strove to find them early on.
</P><P>
	An easy method to get a good idea of the effective concentrations is to make agar plates with a gradient , then set spots of  your culture along that gradient. <B>(Figure 1)</B></P><P>
	We found that the penicillins ampicillin and penicillin, the arguably most used antibiotics, seemed to be completely ineffective at inhibiting <I>V. natriegens </I>. Fortunately,  <I>V. natriegens </I> susceptible for the penicillin derivative carbenicillin. Since the carbenicillin resistance is mediated through the same enzyme as ampicillin resistance, the ß-lactamase, all constructs carrying this resistance can still be selected for.
</P><P>
	  
	 The concentrations in the following table are working fine in our lab for selecting transformants with the resistance cassettes from our Toolbox.
</P><TABLE width="605"><TBODY><TR><TD><P>
					WT ingibition
					  
					 [µg/mL]
				</P></TD><TD><P>
					WT counter-selection
				</P><P>
					[µg/mL]
				</P></TD><TD><P>
					inhibition with resistance [µg/mL]
				</P></TD></TR><TR><TD><P>
					Chloramphenicol
				</P></TD><TD><P>
					2
				</P></TD><TD><P>
					2
				</P></TD><TD><P>&lt;12
				</P></TD></TR><TR><TD><P>
					Carbenicillin
				</P></TD><TD><P>
					50
				</P></TD><TD><P>
					200
				</P></TD><TD><P>
					600
				</P></TD></TR><TR><TD><P>
					Kanamycin
				</P></TD><TD><P>
					65
				</P></TD><TD><P>
					200
				</P></TD><TD><P>
					400
				</P></TD></TR><TR><TD><P>
					Tetracycline*
				</P></TD><TD><P>
					7
				</P></TD><TD><P>
					11
				</P></TD><TD><P>&lt;12
				</P></TD></TR><TR><TD><P>
					Streptomycin
				</P></TD><TD><P>
					80
				</P></TD><TD><P>
					/
				</P></TD><TD><P>
					/
				</P></TD></TR><TR><TD><P>
					Gentamycin
				</P></TD><TD><P>
					30
				</P></TD><TD><P>
					/
				</P></TD><TD><P>
					/
				</P></TD></TR><TR><TD><P>
					Glycloserine
				</P></TD><TD><P>&lt;200
				</P></TD><TD><P>
					/
				</P></TD><TD><P>
					/
				</P></TD></TR><TR><TD><P>
					Penicillin
				</P></TD><TD><P>&lt;100
				</P></TD><TD><P>
					/
				</P></TD><TD><P>
					/
				</P></TD></TR></TBODY></TABLE><P>
	  
	 (Figure 7 Scheme of the most common antibiotics in synthetic biology and their relevant concentration in correlation with selection <I>V. natriegens </I> with and without a resistance cassette )
</P><P>
	  
	  
	 The first four antibiotics, Chloramphenicol, Carbenicillin, Kanamycin and Tetracycline are the antibiotics we commonly used in our project. For Golden Gate cloning we routinely switch between Chloramphenicol and Kanamycin.
	  
	 Because, it is highly sensitive to light,  and we are using inducible promoters that are activated by a derivative of tetracycline and also reacting to the antibiotic, tetracycline only is used as our emergency antibiotic.
</P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">
	Enabling transformation in 	<I>		Vibrio natriegens	</I></DIV><DIV class="content">




	 The first transformations we tried were based on E. Coli standard protocols and did not work. In the paper of 
<A href="https://www.nature.com/articles/nmeth.3970.pdf"><ABBR title="&lt;i&gt;Vibrio natriegens&lt;/i&gt; as a fast-growing host for molecular biology."> Weinstock <I> et al.</I> 2016 </ABBR></A>

 you can find a protocol for transformation with chemically competent cells and for electroporation.

	 In the protocol for chemically competent cells it is stated that there must be a knockout of the DNAses for it to work. We tried it without the knockout with positive results, but for a re-transformation at least 550 ng of the desired DNA is needed. 



<FIGURE style="width: 45%; float: right;"><FIGCAPTION><B>Figure 1:</B> On the left site the two plates top are positive controls, if the cells survived the procedure of making them competent and freezing at -80°C

	 On the left side below are two negative controls without Plasmid for showing that the antibiotic plates have a selecting concentration.
	
	 Then from left to right the concentration of DNA used of the transformations increases  each time as triplicates in a row. The concentrations used are  140ng , 280ng  and 560ng
		</FIGCAPTION></FIGURE><P>
	
	 We used the 

<A href="https://2018.igem.org/File:T--Marburg--PYTK_Plasmid_card.png "><ABBR title="link to the pYTK plasmid">pYTK   </ABBR></A>



	     plasmid from the Dueber Toolbox 

<A href="https://pubs.acs.org/doi/pdfplus/10.1021/sb500366v "><ABBR title="A Highly Characterized Yeast Toolkit for Modular, Multipart
Assembly">pYTK   </ABBR></A>



	      which has a GFP dropout for visual differentiation, as well as a Chloramphenicol resistance.
</P><P>
	We think the issue is that during the making and freezing of the competent cells, a lot of cells die, which releases the DNase from the wild type into the media. The DNases digest the DNA before it can get into the cells.
</P><P>
	The reason why the transformation works without the knockout is that the DNases can't digest all 550 ng DNA and some is left over and can be taken up.

	 You only get single colonies and this efficiency is too bad for cloning. The amount of DNA needed is too great for realistic applications and you would still need
	<I>
		E. Coli
	</I>
	 for re-transformation.
	
	 Since we think the DNase is the root of our problem, after we knock it out we want to give chemically competent cells and heat shock transformation another try.
	
	 Next we tested the electroporation protocol, because it is reported to have a much higher efficiency.


<A href="https://www.sciencedirect.com/science/article/pii/0167478188901583 "><ABBR title="DNA transfection of Escherichia coli by electroporation">Akira Taket 1988 </ABBR></A>

	 

	  We made some minor adjustments to the protocol because we did not have the same equipment that was used in the paper and we wanted to use 3000xg instead of the rpm value for the centrifuge to make the protocol applicable to other labs.
</P><P><FIGURE style="width: 55%; float: left;"><FIGCAPTION><B>Figure 2:</B></FIGCAPTION></FIGURE></P><P> The picture shows four plates of <I> Vibrio natriegens </I>, transformed with different concentrations of pYTK. Top left: 50ng;  top right: 5ng; down below left: 0.5ng; down below right: 0.1ng. For better visualising the GFP the plates are lightened by UV light.
</P></DIV></DIV></DIV></DIV></ARTICLE></MAIN><P>
	We used an electroporator from Eppendorf whose only variable parameter was the voltage.
	
	 This protocol worked fine with 100 ng of DNA.
	
	 But the efficiency still left something to be desired. Especially when we think about cloning, so we optimized the protocol and our handling again.
		 We made sure that all buffers and centrifugation tubes were cooled to 4 °C when making competent cells before they reached an OD<SUB>600nm </SUB>of around 0.5 and that the samples were kept on ice to inhibit further growth.
</P><P>
	We increased the centrifugation strength because after diluting the cells to a fix OD of 16, we produced 30 aliquots which isn't enough when you do triplicates.
	
	 We also froze the cells in liquid nitrogen instead of in dry ice.
	
	 For  the electroporation we tested  with different gap sizes, ranging from 1 to 2 mm. 1mm proved to work better for Vibrio natriegens.
	
	 We preheat the recovery media to 50 °C because mixing the 4 °C cells in the chilled cuvette lowers the temperature of the media. Preheating the media to 37 °C would result in a temperature approximating 30 °C after pipetting and mixing with the cold cells.

	 Afterwards we put the electroporated aliquots in a 37 °C water bath to help them recover while we are treating the other samples. More cells survive, therefore the efficiency increases.
</P><P>
	
	 In the paper of<A href="https://www.nature.com/articles/nmeth.3970.pdf"><ABBR title="&lt;i&gt;Vibrio natriegens&lt;/i&gt; as a fast-growing host for molecular biology."> Weinstock <I> et al.</I> 2016, </ABBR></A>
 

	  a voltage between 700 and 900V is recommended. One of our advisers said that a higher voltage increases efficiency so we tested voltages of 700V up to 1600V in steps of 100V.
</P><P>
	The difference between 1600V and 900V was not noteworthy. It might be that with an increasing voltage (efficiency)also more cells  die during the procedure, resulting in a consistent final efficiency.
	
	 Electroporating 24 samples takes time and during the first and the last electroporation the first cells have about 15 min more time to recover, directly i a water bath.
	
	 After applying these new changes the efficiency increased we are now able to successfully transform with concentrations as low as DNA of 0.1ng. Which is nearly what is achieved when working with <I> E. Coli</I> with heat shock. <I>E. Coli </I> still performs better in this, but we are working with the <I> Vibrio natriegens </I> wild type and it is already good enough for cloning. 
</P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">TEM- Microscopy</DIV><DIV class="content"><P>Working with <I>V. natriegens </I> all the time, we also wanted to see it with our own eyes. To do that, we used several methods of transmission electron microscopy ( TEM ) and scanning electron microscopy (SEM).Our microtomes we made in two ways, by fixation in epoxy resin and in freezing to a grid. Thereby we can show you cuts through the cell (Picture 1-4 ) 
The method we also showed was platinum coal steaming the cells (Picture 5) and negative staining for visualising attachments like flagella.( Picture 6 and 7 )</P><P>
On one picture we also have a three dimensional impression of the cells from scanning electron microscopy. (Picture 8 and 9 )

From the size of the pixels and the magnification  you can calculate a scale bar, or directly measure the cells.
What we can conclude from this is, Vibrio natriegens is about one to two µm long and has a diameter of around 1µm. It has a single polar flagellum and it looks like it is dividing along its equatorial axis. 
We could also make out something in the cell what could be storage compartments. ( Picture 10 )</P><FIGURE style="width: 45%; float: right;"><FIGCAPTION><B>Picture 1</B> Picture of dividing cell. preparation by cryo conservation in Epoxy resin 
		</FIGCAPTION></FIGURE><FIGURE style="width: 45%; float: right;"><FIGCAPTION><B>Picture 2</B>  Picture of a single cell prepared by cryo conversation
		</FIGCAPTION></FIGURE><FIGURE style="width: 45%; float: right;"><FIGCAPTION><B>Picture 3</B>  Picture of dividing cells after fixation in epoxy resin
		</FIGCAPTION></FIGURE><FIGURE style="width: 45%; float: right;"><FIGCAPTION><B>Picture 4</B>  Cut through a single cell preparation in epoxy resin 
		</FIGCAPTION></FIGURE><FIGURE style="width: 40%; float: left;"><FIGCAPTION><B>Picture 5</B>  Platinum coal stained cells  
		</FIGCAPTION></FIGURE><FIGURE style="width: 40%; float: right;"><FIGCAPTION><B>Picture 6</B>  Picture of a single cell visualised by negative staining  
		</FIGCAPTION></FIGURE><FIGURE style="width: 43%; float: left;"><FIGCAPTION><B>Picture 7</B>   Picture of a single cell with flagella  by negative staining  
		</FIGCAPTION></FIGURE><FIGURE style="width: 43%; float: right;"><FIGCAPTION><B>Picture 8</B>   Scanning electron microscopy:  crowd of cells  
		</FIGCAPTION></FIGURE><FIGURE style="width: 45%; float: left;"><FIGCAPTION><B>Picture 9</B>   Cut through a single cell showing something that could be storage compartments 
		</FIGCAPTION></FIGURE>.

		
</DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Flow cytometry </DIV><DIV class="content"><P> 

At many different occasions, at meetups or conferences, we showed the growth curve of <I>V. natriegens </I>compared to <I>E. coli</I>. Other scientists were impressed by the extremely fast growth rate but even more by the high OD<SUB>600</SUB> that we could show. We were asked many times if the high OD<SUB>600</SUB> is really due to a high cell density or if it is rather caused by other components like cell debris or substances secreted to the medium which contribute to the measured absorbance. 



To find out more about the growth dynamics, we decided to acquire a growth curve of <I>V. natriegens</I> in the most direct way, by counting cells in a flow cytometer. We inoculated three baffled flasks from stationary cultures and took samples in 15 minute intervals while the bacteria were incubated at 37 °C with shaking at 220 rpm. The OD<SUB>600</SUB> of these samples was measured in a normal photometer and the cultures were then immediately analyzed by flow cytometry. The flow cytometer directs the samples through a thin capillary so that the cells pass a laser bean one by one and thus, can be counted and analyzed independently. A constant flow rate and time for data acquisition was set, which results in measuring a defined sample volume. Together with the counted events, the cells per volume can be calculated.<FIGURE style="width: 60%; float: right"><FIGCAPTION><B> Figure 1: Comparison of OD<SUB>600</SUB> and events/µL</B> OD<SUB>600</SUB> is shown in red and events/µL measured with flow cytometry is shown in blue </FIGCAPTION></FIGURE> 
A comparison of the OD<SUB>600</SUB> to events/µL values are shown in figure 1. 

When we planned this experiment, we were most curious about the composition of the culture in the stationary phase to answer the question if the high <SUB>OD</SUB> that <I>V. natriegens</I> can reach is the result of a high cell density or if it can be traced back to other substances. Interestingly, both values, OD<SUB>600</SUB> and events/µL start to stagnate at a similar time point (165 min). We interpret this result as a confirmation that the high <SUB>OD</SUB> is indeed caused by bacterial cells.

By carefully comparing the shape of both growth curves, we realized that, in fact, the most striking data in this plot can be found at the beginning of the experiment. While exponentially increasing values can be seen right from the start for the curve created from the OD<SUB>600</SUB> data, a short lag phase is apparent when events/µL are plotted (figure 1). We tried to find an explanation for this observation and realized that the absorbance of a culture does not necessarily correlate with the concentration of cells but rather with the biomass inside the flask. 

<FIGURE style="width: 40%; float: left"><FIGCAPTION><B> Figure 2: Scatterpolt acquired by flow cytometry</B> A single sample after 45 minutes is shown as an example. Each dot represents one event </FIGCAPTION></FIGURE> 

Fortunately, additional data can be obtained from the forward and side scatter of the laser beam in a flow cytometer which provide information about the size and inner complexity of the analyzed cells, respectively. Figure 2 exemplary shows one sample at t = 45 min. The side scatter (SSC-A) is plotted on the Y-Axis versus the side scatter (FSC-A) on the X-Axis. Each dot in this scatter plot represents one detected event and a heatmap can be used to visualize many events with the same properties (red = high event cont; blue = low event count). The population in the top right corner comprises roughly 98 % of all events and represents fully viable cells, since this is the population that increases in number throughout the growth experiment (peaking at &gt;99% of all measured events at mid-log phase). The population in the bottom left corner most likely consists of sick or dormant cells and cell debris, since the number of events in this population remains low throughout the experiment.<FIGURE style="width: 60%; float: right"><FIGCAPTION><B> Figure 3: Comparison of forward scatter versus events/µL</B> The measured events/µL are shown in blue and on the left Y-axis. The forward scatter is displayed in green and on the right Y-axis </FIGCAPTION></FIGURE> 

We plotted the mean forward scatter values of all cells together with the events/µL. The mean side forward value dramatically increases during the first data points with a peak after 45 minutes. This is also the same time point which we identified as the beginning of the exponential growth phase. During the subsequent course of the experiment the forward scatter values decrease again, reaching a minimum when the culture goes into stationary phase.

Taking all three datasets into account, we suggest that the cells start to grow in size upon provision of fresh medium but initially without undergoing cell division. This results in an increase volume of individual cells and thus, an increase of the measured OD<SUB>600</SUB> but without significant changes to the cell concentration in the culture. After 45 minutes, when the forward scatter peaks, we assume that a majority of cells reach maximum cell volume initiate rapid division, quickly entering the exponential phase. During the ensuing time points, exponential growth can be observed and the decrease of the forward scatter is a hint for a reduction in mean cell size.   <FIGURE style="width: 30%; float: right"><FIGCAPTION><B> Figure 4: Histograms of forward scatter during the course of the experiments</B> The histograms are plotted from top to bottom during the time course of the experiment </FIGCAPTION></FIGURE>
To additionally visualize the composition of the measured cells in regard to the forward scatter, we created figure 4 showing histograms of the forward scatter at different timepoints. It is apparent that the population is heterogeneous at the beginning of the experiment and at the end when the cultures again reaches stationary phase. During the period of exponential growth, the sample is more homogenous. The already discussed trend in the forward scatter curve can also be observed with these histograms which show a shift to the right when the forward scatter peaks and a shift to the left for the following time points.</P>
 

We want to thank Dr. Max Mundt who carried out the experiments with us and who helped with analyzing the data.
        
</DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Genome sequencing</DIV><DIV class="content">
					
					The goal of synthetic biology is to modify biological organisms in an engineering manner using them as nano-factories for a cheap and environmentally friendly production of valuable resources. To reach this goal a detailed genotyping of the corresponding host is a crucial step. Concerning this we sequenced the genome of our host <I>Vibrio natriegens</I> for our synthetic biological purposes.
					One of our goals was to identity and characterize chromosomal integration sites. This made the genotyping a critical step for the design of the correct homologous flanks. But the genome map we built became as well a valuable tool for our whole team for the planing and design of genetic constructs.
					
					We got the organism <I>Vibrio natriegens</I> strain ATCC 14048 from the <A href="https://www.dsmz.de/catalogues/details/culture/DSM-759.html?tx_dsmzresources_pi5%5BreturnPid%5D=304"><ABBR title="Shop V. natriegens on the German collection of microorganisms and cell cultures">german collection of microorganisms and cell cultures</ABBR></A> and sequenced it by Illumina sequencing. Illumina sequencing is a next generation sequencing (NGS) method acquiring data by ‘sequencing by synthesis’ (SBS) chemistry. Since 2014 the short read, massively parallel sequencing technique lead to an explosion of sequencing data output and a drop in cost per base pair, launching the next generation of sequencing methods (<A href="https://www.illumina.com/documents/products/illumina_sequencing_introduction.pdf"><ABBR title="Illumina sequencing introduction">Illumina</ABBR></A>). Using Illumina sequencing, we were able to sequence the whole genome in a manner of hours at an affordable cost.
					The workflow of Illumina is based on three steps: Library preparation, short read sequencing and genome assembly. For library preparation several kits can be used. Firstly the template DNA needs to be isolated and purified. Then, the DNA fragmentation and tagging are done simultaneously by a method developed by Illumina called tegmentation. Here bead-linked transposomes carrying Illumina sequencing adapters perform DNA fragmentation, normalization and adapter ligation in one step <A href="https://www.illumina.com/products/by-type/sequencing-kits/library-prep-kits/nextera-dna-flex.html?scid=2017249VU1"><ABBR title="Nextera DNA Flex Library Prep Kit">LINK</ABBR></A>. After that the fragments are amplified and the fragments of suitable size purified. Then, the library can finally be loaded on a flow cell for sequencing. The flow cell is covered by embedded adapter sequences were the fragments can bind to. By bridge amplification the single fragments are amplified into clonal clusters. Finally the clusters are synthesized with fluorescent labeled nucleotides. A designed software records the light’s wavelength and intensity and saves them into a text file containing the different reads <A href="https://www.illumina.com/documents/products/illumina_sequencing_introduction.pdf"><ABBR title="Illumina sequencing introduction">LINK</ABBR></A>. Finally the reads are assembled into bigger scaffolds by bioinformatic software tools like in this case SPAdes 3.12.0 (<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3342519/"><ABBR title="Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, et al. 2012. SPAdes: A New Genome Assembly Algorithm and Its Applications to Single-Cell Sequencing. Journal of Computational Biology 19: 455–477.">Bankevich <I>et al.</I>, 2012</ABBR></A>).
					
					We prepared the library using the Nextera DNA Flex Library Prep Kit <A href="https://www.illumina.com/products/by-type/sequencing-kits/library-prep-kits/nextera-dna-flex.html"><ABBR title="Nextera DNA Flex Library Prep Kit">LINK</ABBR></A> by Illumina and sequenced the genome with the MiSeq system<A href="https://www.illumina.com/systems/sequencing-platforms/miseq.html"><ABBR title="MiSeq system">LINK</ABBR></A> with a coverage of 500-fold. The protocols for library preparation and sequencing were given by Illumina <A href="https://support.illumina.com/custom-protocol-selector.html"><ABBR title="Illumina protocol selector">LINK</ABBR></A>. Subsequently we analyzed the data with SPAdes 3.12.0 (<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3342519/"><ABBR title="Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, et al. 2012. SPAdes: A New Genome Assembly Algorithm and Its Applications to Single-Cell Sequencing. Journal of Computational Biology 19: 455–477.">Bankevich <I>et al.</I>, 2012</ABBR></A>), Prokka 1.12 (<A href="https://academic.oup.com/bioinformatics/article/30/14/2068/2390517"><ABBR title="Seemann T. 2014. Prokka: rapid prokaryotic genome annotation. Bioinformatics 30: 2068–2069.">Seemann, 2014</ABBR></A>) and Geneious 11.1.5.
					The Illumina sequencing yielded multiple DNA contigs – short snippets of the genome – that were assembled to bigger scaffolds using SPAdes 3.12.0 (<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3342519/"><ABBR title="Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, et al. 2012. SPAdes: A New Genome Assembly Algorithm and Its Applications to Single-Cell Sequencing. Journal of Computational Biology 19: 455–477.">Bankevich <I>et al.</I>, 2012</ABBR></A>). SPAdes is an open source software developed for genome assembly of single-cell data as well as multi-cell data. Version 3.12.0 is available since May, 2018. Subsequently we looked for annotations with Prokka 1.12 (<A href="https://academic.oup.com/bioinformatics/article/30/14/2068/2390517"><ABBR title="Seemann T. 2014. Prokka: rapid prokaryotic genome annotation. Bioinformatics 30: 2068–2069.">Seemann, 2014</ABBR></A>). Prokka is an open source software for prokaryotic genome annotations written in Perl. Finally we checked the genome for secondary metabolites using antiSMASH 3.0 (<A href="https://academic.oup.com/nar/article/43/W1/W237/2467910"><ABBR title="Weber T, Blin K, Duddela S, Krug D, Kim HU, Bruccoleri R, Lee SY, Fischbach MA, Müller R, Wohlleben W, et al. 2015. antiSMASH 3.0—a comprehensive resource for the genome mining of biosynthetic gene clusters. Nucleic Acids Research 43: W237–W243.">Weber <I>et al.</I>, 2015</ABBR></A>). Here we found three secondary metabolite clusters for nonribosomal peptides (NRPs), bacteriocin and ectoine (see link below for the .gb file).
					NRPs are biologically active compounds built by nonribosomal-peptide-synthetases in a ribosome independent manner. These compounds are interesting for clinical applications due to their use as last resort antibiotics, antitumor or antifungal drugs or as immunosuppressants (<A href="https://doi.org/10.1021/ar7000414"><ABBR title="Walsh CT. 2008. The Chemical Versatility of Natural-Product Assembly Lines. Accounts of Chemical Research 41: 4–10.">Walsh, 2008</ABBR></A>; <A href="http://www.sciencedirect.com/science/article/pii/S0959440X10000126"><ABBR title="Strieker M, Tanović A, Marahiel MA. 2010. Nonribosomal peptide synthetases: structures and dynamics. Current Opinion in Structural Biology 20: 234–240.">Strieker <I>et al.</I>, 2010</ABBR></A>). Bacteriocins are heat resistant antimicrobial peptides produced by bacteria to kill or inhibit the growth of other strains without harming the bacteria themselves. Due to their large diversity in structure and function they are interesting for the industry for food preservation or as new potential drugs in the future (<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4033612/"><ABBR title="Yang S-C, Lin C-H, Sung CT, Fang J-Y. 2014. Antibacterial activities of bacteriocins: application in foods and pharmaceuticals. Frontiers in Microbiology 5.">Yang <I>et al.</I>, 2014</ABBR></A>). The ectoine cluster may make this organism interesting for further research and for industry applications as well due to the use of ectoine in medicaments against cold and dermatitis or their use in eye drops and nasal sprays (<A href="https://www.hindawi.com/journals/ja/2014/176597/"><ABBR title="Werkhäuser N, Bilstein A, Sonnemann U. 2014. Treatment of Allergic Rhinitis with Ectoine Containing Nasal Spray and Eye Drops in Comparison with Azelastine Containing Nasal Spray and Eye Drops or with Cromoglycic Acid Containing Nasal Spray. Journal of Allergy.">Werkhäuser <I>et al.</I>, 2014</ABBR></A>).
					
					We checked for published genomes on the NCBI database and found two annotations for <I>Vibrio natriegens</I> ATCC 14048. Using <A href="https://www.geneious.com/geneious/"><ABBR title="Geneious NGS analysis tool">Geneious</ABBR></A> we aligned our sequences against the database and got an 80.5% similarity against the genome with the accession number <A href="https://www.ncbi.nlm.nih.gov/nuccore/1140770138"><ABBR title="V. natriegens ATCC 14048 Chr. 1">NZ_CP009977.1</ABBR></A> and a 99.97% identity against the genome with the accession number <A href="https://www.ncbi.nlm.nih.gov/nuccore/CP016345"><ABBR title="V. natriegens ATCC 14048 Chr. 1">NZ_CP016345.1</ABBR></A>. Our sequences can be downloaded from the links below.
					
					Both sequences downloaded from the NCBI database have been sequenced with PacBio. The sequence <A href="https://www.ncbi.nlm.nih.gov/nuccore/1140770138"><ABBR title="V. natriegens ATCC 14048 Chr. 1">NZ_CP009977.1</ABBR></A> has a coverage of 96.48-fold and was assembled with RS HGAP v. 3 while the sequence <A href="https://www.ncbi.nlm.nih.gov/nuccore/CP016345"><ABBR title="V. natriegens ATCC 14048 Chr. 1">NZ_CP016345.1</ABBR></A> has a coverage of 178.8-fold and was assembled with HGAP v. 2.
					<H3>Acknowledgements</H3>
					We would like to thank Dr. Javier Serrania for the possibility and support sequencing the genome of our host Vibrio natriegens. This work became a useful and very valuable tool for our whole team and our whole project and could not have been done without his support.
					</DIV></DIV><DIV class="collapsible"><DIV class="btn_expand"> Mutation rate in <I>Vibrio natrigens</I></DIV><DIV class="content"><P>
	When we first presented our project, many people asked about the mutation rate of <EM>Vibrio natriegens,</EM> worrying its high growth speed would be accompanied by a higher mutation rate. Some were concerned that random mutations could undo their work in the lab, or lead to a pathogenic <EM>Vibrio natriegens</EM> mutant, others were hoping this could speed up their mutation experiments.</P><P>
	The mutation rate is the frequency with which new mutations appear in an organism. A mutation in a gene can be a silent mutation, meaning it has no effect, or lead to gene function loss, or alter the genes function. These mutations can be the result of DNA-damage, caused for example by radiation. They can also happen spontaneously through mistakes the DNA-polymerase makes during DNA-replication. The spontaneous mutation rate of most organisms is known, such as <EM>Escherichia coli’s</EM> and <EM>Vibrio cholerae’s</EM>. It was determined by selecting for mutants which had changed phenotype,
		<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1209226/"><ABBR title="(S. E. Luria and M. Delbrück,: Mutations of Bacteria from Virus Sensitivity to Virus Resistance,  Genetics, (1943); 28(6), pp 491–511">(S. E. Luria and M. Delbrück 1943)</ABBR></A> 
		 or in recent years, as it became cheaper, by analyzing the genome trough genome sequencing. This method is more precise than estimating the mutation rate based on a phenotypical change
		 <A href="http://www.pnas.org/content/112/44/E5990"><ABBR title="(Patricia L. Foster, Heewook Lee, Ellen Popodi, Jesse P. Townes, and Haixu Tang,: Determinants of spontaneous mutation in the bacterium Escherichia coli as revealed by whole-genome sequencing, PNAS, (2015); 112(44)">(Patricia L. Foster 2015)</ABBR></A><A href="https://www.biorxiv.org/content/early/2016/08/01/066829"><ABBR title="(Marcus M Dillon, Way Sung, Robert Sebra, Michael Lynch and Vaughn Cooper,: Genome-wide biases in the rate and molecular spectrum of spontaneous mutations in Vibrio cholerae and Vibrio fischeri,  Molecular Biology and Evolution, (2017); 34(1), pp 93–109">(Marcus M Dillon 2017)</ABBR></A>.
		The mutation rate plays a major role in multiple arears of biology, influencing for example the rate with which pathogenic microorganisms gain antibiotic resistances or adapt to a new environment
		<A href="https://www.nature.com/articles/ng.2656"><ABBR title="(Christopher B Ford, Rupal R Shah, Midori Kato Maeda, Sebastien Gagneux, Megan B Murray, Ted Cohen, James C Johnston, Jennifer Gardy, Marc Lipsitch and Sarah M Fortune,: &lt;i&gt;Mycobacterium tuberculosis&lt;/i&gt; mutation rate estimates from different lineages predict substantial differences in the emergence of drug-resistant tuberculosis, Nature Genetics, (2013); 45, pp 784-790"> (Christopher B Ford 2013)</ABBR></A>. </P><P>
	We conducted mutation experiments to estimate the mutation rate (probability of mutation per cell per division or generation) of <EM>Vibrio natriegens</EM> by using the number of mutation events and the final number of cells in a culture 
	<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2041832/"><ABBR title="Patricia L. Foster,: Methods for Determining Spontaneous Mutation Rates, Methods Enzymol., (2006); 409, pp 195-213">(Patricia L. Foster 2006)</ABBR></A>. To determine the number of mutation events, mutants have to first be identified. This can be done with rifampicin, an antibiotic which inhibits mRNA transcription by obstructing its elongation path through binding to the β-chain. Some microorganisms can gain a rifampicin resistance through a specific point mutation or deletion in the β-chain of the RNA-Polymerase
	<A href="https://www.karger.com/Article/FullText/484246"><ABBR title="Wu E.Y. and Hilliker A.K. Identification of Rifampicin Resistance Mutations in Escherichia coli, Including an Unusual Deletion Mutation, J Mol Microbiol Biotechnol, (2017); 27, pp 356-362">(Wu and Hilliker, 2017)</ABBR></A>. By plating <EM>Vibrio natriegens</EM> out on rifampicin plates, thes <EM>Vibrio natriegens</EM> mutants can be selected.</P><P>
	By testing sister colonies of <EM>Vibrio natriegens</EM> in this manner and counting the resistant colonies we estimated the mutation rate with the Lea-Coulson median estimator 
	<A href="https://link.springer.com/article/10.1007/BF02986080"><ABBR title="(Lea and Coulson,: The distribution of the numbers of mutants in bacterial populations, Journal of Genetics, (1949); 49:264">(Lea and Coulson, 1949)</ABBR></A>. 
	For this we inoculated 14 sister cultures of <EM>Vibrio natriegens</EM> with the same OD<SUB>600 (</SUB>About 0.0001) out of an exponential pre-culture. When the sister cultures almost reached the stationary phase, we platted them on rifampicin- and on plats without an antibiotic. After evaluating these plates, we could estimate the Median number of mutants in a culture, by counting all rifampicin resistant cultures and applying the Lea-Coulson median estimator. The final number of cells in the culture was estimated by counting the colony forming units on the plates without antibiotics.</P><P>
	The calculated number of mutations per culture with the Lea-Coulson median estimator was 5,7925 and the estimated final number of cells in the culture were 3165000000 cells. The number of mutations per culture is divided by the final number of cell 
	<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2041832/"><ABBR title="Patricia L. Foster,: Methods for Determining Spontaneous Mutation Rates, Methods Enzymol., (2006); 409, pp 195-213">(Patricia L. Foster 2006)</ABBR></A>. 
	The mutation rate we estimated was 1,83017E-09, and there for a bit lower than the mutation rate of <EM>Escherichia coli</EM> which was estimated by Luria and Delbrück in a comparable experiment for of about with was calculated to be 3,2E-09
	 <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1209226/"><ABBR title="(S. E. Luria and M. Delbrück,: Mutations of Bacteria from Virus Sensitivity to Virus Resistance,  Genetics, (1943); 28(6), pp 491–511">(S. E. Luria and M. Delbrück 1943)</ABBR></A>. 
	 Since our number of mutations per culture was estimated to be between 4 and 15 we determined the number of mutations per culture with the Lea-Coulson median estimator, unlike how the mutation rate of <EM>Escherichia coli</EM> was determined. For <EM>Escherichia coli</EM> the p<SUP>0</SUP> method was used, because there were  between 0,3 and 2,3 mutations per culture
	  <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2041832/"><ABBR title="Patricia L. Foster,: Methods for Determining Spontaneous Mutation Rates, Methods Enzymol., (2006); 409, pp 195-213">(Patricia L. Foster 2006)</ABBR></A>. 
	 Because <EM>Escherichia coli</EM> grows to a lesser OD<SUB>600</SUB>, cultures do not undergo as many cell divisions, thus fewer cells have the opportunity to mutate and less mutation events take place. However, our experiment suggests, that the number of mutants per cells are higher in <EM>Escherichia coli</EM>, meaning that the possibility to choose a mutated colony is higher when working with <EM>Escherichia coli</EM>. </P><P>
	We are now able to answer the question of our fellow researchers regarding <EM>Vibrio natriegens’</EM> mutation rate. After these experiments we know that concerns about unwanted spontaneous mutations when switching from <EM>Escherichia coli </EM>to <EM>Vibrio natriegens</EM> are unwarranted. It can be assumed that <EM>Vibrio natriegens</EM> can speed up mutation experiments anyway, since there are more mutations occurring in the culture, simply because there is a higher density, increasing genetic variability within it. Our data is not conclusive but enables an estimation of the mutation rate of <EM>Vibrio natriegens.</EM> A more precisely calculated mutation rate can be obtained through whole genome sequencing of multiple <EM>Vibrio natriegens</EM> cultures as has been done for <EM>Escherichia coli</EM><A href="http://www.pnas.org/content/early/2012/09/17/1210309109.short"><ABBR title="Heewook Lee, Ellen Popodi, Haixu Tang, and Patricia L. Foster,: Rate and molecular spectrum of spontaneous mutations in the bacterium Escherichia coli as determined by whole-genome sequencing, National Academy of Sciences, (2012)">(Heewook Lee <I>et al. </I> 2012)</ABBR></A>.</P><P>
	When we first presented our project, many people asked about the mutation rate of <EM>Vibrio natriegens,</EM> worrying its high growth speed would be accompanied by a higher mutation rate. Some were concerned that random mutations could undo their work in the lab, or lead to a pathogenic <EM>Vibrio natriegens</EM> mutant, others were hoping this could speed up their mutation experiments.</P><P>
	The mutation rate is the frequency with which new mutations appear in an organism. A mutation in a gene can be a silent mutation, meaning it has no effect, or lead to gene function loss, or alter the genes function. These mutations can be the result of DNA-damage, caused for example by radiation. They can also happen spontaneously through mistakes the DNA-polymerase makes during DNA-replication. The spontaneous mutation rate of most organisms is known, such as <EM>Escherichia coli’s</EM> and <EM>Vibrio cholerae’s</EM>. It was determined by selecting for mutants which had changed phenotype,
		<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1209226/"><ABBR title="(S. E. Luria and M. Delbrück,: Mutations of Bacteria from Virus Sensitivity to Virus Resistance,  Genetics, (1943); 28(6), pp 491–511">(S. E. Luria and M. Delbrück 1943)</ABBR></A> 
		 or in recent years, as it became cheaper, by analyzing the genome trough genome sequencing. This method is more precise than estimating the mutation rate based on a phenotypical change
		 <A href="http://www.pnas.org/content/112/44/E5990"><ABBR title="(Patricia L. Foster, Heewook Lee, Ellen Popodi, Jesse P. Townes, and Haixu Tang,: Determinants of spontaneous mutation in the bacterium Escherichia coli as revealed by whole-genome sequencing, PNAS, (2015); 112(44)">(Patricia L. Foster 2015)</ABBR></A><A href="https://www.biorxiv.org/content/early/2016/08/01/066829"><ABBR title="(Marcus M Dillon, Way Sung, Robert Sebra, Michael Lynch and Vaughn Cooper,: Genome-wide biases in the rate and molecular spectrum of spontaneous mutations in Vibrio cholerae and Vibrio fischeri,  Molecular Biology and Evolution, (2017); 34(1), pp 93–109">(Marcus M Dillon 2017)</ABBR></A>.
		The mutation rate plays a major role in multiple arears of biology, influencing for example the rate with which pathogenic microorganisms gain antibiotic resistances or adapt to a new environment
		<A href="https://www.nature.com/articles/ng.2656"><ABBR title="(Christopher B Ford, Rupal R Shah, Midori Kato Maeda, Sebastien Gagneux, Megan B Murray, Ted Cohen, James C Johnston, Jennifer Gardy, Marc Lipsitch and Sarah M Fortune,: &lt;i&gt;Mycobacterium tuberculosis&lt;/i&gt; mutation rate estimates from different lineages predict substantial differences in the emergence of drug-resistant tuberculosis, Nature Genetics, (2013); 45, pp 784-790"> (Christopher B Ford 2013)</ABBR></A>. </P><P>
	We conducted mutation experiments to estimate the mutation rate (probability of mutation per cell per division or generation) of <EM>Vibrio natriegens</EM> by using the number of mutation events and the final number of cells in a culture 
	<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2041832/"><ABBR title="Patricia L. Foster,: Methods for Determining Spontaneous Mutation Rates, Methods Enzymol., (2006); 409, pp 195-213">(Patricia L. Foster 2006)</ABBR></A>. To determine the number of mutation events, mutants have to first be identified. This can be done with rifampicin, an antibiotic which inhibits mRNA transcription by obstructing its elongation path through binding to the β-chain. Some microorganisms can gain a rifampicin resistance through a specific point mutation or deletion in the β-chain of the RNA-Polymerase
	<A href="https://www.karger.com/Article/FullText/484246"><ABBR title="Wu E.Y. and Hilliker A.K. Identification of Rifampicin Resistance Mutations in Escherichia coli, Including an Unusual Deletion Mutation, J Mol Microbiol Biotechnol, (2017); 27, pp 356-362">(Wu and Hilliker, 2017)</ABBR></A>. By plating <EM>Vibrio natriegens</EM> out on rifampicin plates, thes <EM>Vibrio natriegens</EM> mutants can be selected.</P><P>
	By testing sister colonies of <EM>Vibrio natriegens</EM> in this manner and counting the resistant colonies we estimated the mutation rate with the Lea-Coulson median estimator 
	<A href="https://link.springer.com/article/10.1007/BF02986080"><ABBR title="(Lea and Coulson,: The distribution of the numbers of mutants in bacterial populations, Journal of Genetics, (1949); 49:264">(Lea and Coulson, 1949)</ABBR></A>. 
	For this we inoculated 14 sister cultures of <EM>Vibrio natriegens</EM> with the same OD<SUB>600 (</SUB>About 0.0001) out of an exponential pre-culture. When the sister cultures almost reached the stationary phase, we platted them on rifampicin- and on plats without an antibiotic. After evaluating these plates, we could estimate the Median number of mutants in a culture, by counting all rifampicin resistant cultures and applying the Lea-Coulson median estimator. The final number of cells in the culture was estimated by counting the colony forming units on the plates without antibiotics.</P><P>
	The calculated number of mutations per culture with the Lea-Coulson median estimator was 5,7925 and the estimated final number of cells in the culture were 3165000000 cells. The number of mutations per culture is divided by the final number of cell 
	<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2041832/"><ABBR title="Patricia L. Foster,: Methods for Determining Spontaneous Mutation Rates, Methods Enzymol., (2006); 409, pp 195-213">(Patricia L. Foster 2006)</ABBR></A>. 
	The mutation rate we estimated was 1,83017E-09, and there for a bit lower than the mutation rate of <EM>Escherichia coli</EM> which was estimated by Luria and Delbrück in a comparable experiment for of about with was calculated to be 3,2E-09
	 <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1209226/"><ABBR title="(S. E. Luria and M. Delbrück,: Mutations of Bacteria from Virus Sensitivity to Virus Resistance,  Genetics, (1943); 28(6), pp 491–511">(S. E. Luria and M. Delbrück 1943)</ABBR></A>. 
	 Since our number of mutations per culture was estimated to be between 4 and 15 we determined the number of mutations per culture with the Lea-Coulson median estimator, unlike how the mutation rate of <EM>Escherichia coli</EM> was determined. For <EM>Escherichia coli</EM> the p<SUP>0</SUP> method was used, because there were  between 0,3 and 2,3 mutations per culture
	  <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2041832/"><ABBR title="Patricia L. Foster,: Methods for Determining Spontaneous Mutation Rates, Methods Enzymol., (2006); 409, pp 195-213">(Patricia L. Foster 2006)</ABBR></A>. 
	 Because <EM>Escherichia coli</EM> grows to a lesser OD<SUB>600</SUB>, cultures do not undergo as many cell divisions, thus fewer cells have the opportunity to mutate and less mutation events take place. However, our experiment suggests, that the number of mutants per cells are higher in <EM>Escherichia coli</EM>, meaning that the possibility to choose a mutated colony is higher when working with <EM>Escherichia coli</EM>. </P><P>
	We are now able to answer the question of our fellow researchers regarding <EM>Vibrio natriegens’</EM> mutation rate. After these experiments we know that concerns about unwanted spontaneous mutations when switching from <EM>Escherichia coli </EM>to <EM>Vibrio natriegens</EM> are unwarranted. It can be assumed that <EM>Vibrio natriegens</EM> can speed up mutation experiments anyway, since there are more mutations occurring in the culture, simply because there is a higher density, increasing genetic variability within it. Our data is not conclusive but enables an estimation of the mutation rate of <EM>Vibrio natriegens.</EM> A more precisely calculated mutation rate can be obtained through whole genome sequencing of multiple <EM>Vibrio natriegens</EM> cultures as has been done for <EM>Escherichia coli</EM><A href="http://www.pnas.org/content/early/2012/09/17/1210309109.short"><ABBR title="Heewook Lee, Ellen Popodi, Haixu Tang, and Patricia L. Foster,: Rate and molecular spectrum of spontaneous mutations in the bacterium Escherichia coli as determined by whole-genome sequencing, National Academy of Sciences, (2012)">(Heewook Lee <I>et al. </I> 2012)</ABBR></A>.</P></DIV></DIV></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand"><DIV class="btn_e_title">Results Part Collection</DIV><DIV class="btn_e_desc">
			We created a simple and reliable workflow for the <B>characterization</B> of parts from the <B>Marburg Collection</B> in <I>V. natriegens</I><A href="https://2018.igem.org/Team:Marburg/Measurement">(Link to Measurement)</A>.
Experimental data for constitutive and inducible promoters, RBS strength, terminator read-through, ori dependent plasmid copy number and the behavior of our newly designed connectors were obtained.
		</DIV></DIV><DIV class="content"><P> 
    

After creation of the Marburg Collection, we wanted to characterize the parts in<I> V. natriegens</I>. When we started with our project, we had no clue about the behavior of the genetic parts that were integrated into our toolbox. Previous research mainly focused on microbiological description rather than characterization of synthetic constructs as we already discussed in our <I>V. natriegens</I> review <A href="https://2018.igem.org/Team:Marburg/Description">(Link to Description)</A>.
We decided to characterize the parts in our Marburg Collection and hence we did pioneering work to provide the scientific community the data that enable rational utilization of <I>V. natriegens</I> for various applications in synthetic biology.
Before acquiring the final data, we established a fast and convenient platereader workflow that is tailored to the fast growth rate of <I>V. natriegens</I>. We demonstrate its superior performance and discuss considerations in terms of plasmidal context and data analysis on our measurement page <A href="https://2018.igem.org/Team:Marburg/Measurement">(Link to Measurement)</A>. 


</P><DIV class="collapsible"><DIV class="btn_expand">Promoter Characterization</DIV><DIV class="content">
After having established an experimental and data analysis workflow and after determining the optimal plasmidal context for reporter experiments, we started to apply our knowledge to characterize the parts in our Marburg Collection. <FIGURE style="width: 70%; float: center"><FIGCAPTION><B> Figure 1: Relative promoter strength of Anderson promoters </B> Data were normalized over the strongest construct J23100. Error bars represent the standard deviation of the measurements of three independent experiments </FIGCAPTION></FIGURE>

We started by measuring the promoter strength of the Anderson Promoter library in <I>V. natriegens</I>. Firstly, we assembled 19 test plasmids with golden-gate-assembly and measured their expression strength, following our selfmade workflows. The results are shown in figure 1. We observed an even distribution of the tested promoters throughout the dynamic range. The strongest promoter (J23100) yielded 40 fold stronger signal than the promoter dummy and was used as a reference to calculate relative promoter strengths. The test constructs were built with dummy connectors which did not possess insulator elements. We assume that this resulted in additional expression caused by transcription throughout the rest of the plasmid, e.g. ori and antibiotic resistance. This is thought to add the same extent of signal to all measured promoters thus reducing the overall dynamic range. To further evaluate this assumption, we could repeat this experiment with one of our insulators instead of the dummy connector.<H3> Characterization of pTet</H3>
In addition to constitutive promoters, the Marburg Collection contains two inducible promoters, pTet and pTrc. For all experiments with inducible promoters, we added the respective inducer concentration to the preculture as well as to the main culture to ensure constant expression.The first experiments were performed with the pTet promoter that can be induced by the tetracycline derivative anhydrotetracycline (ATc). ATc is much less cytotoxic but still capable of binding and altering the structure of the repressor TetR, leading to release of the promoter and enabling transcription. To measure the dose response behavior of the pTet, we made a dilution series of ATc. Following the recommendation of our advisors (Stefano Vecchione), we started with the concentration commonly used in <I>E. coli</I>, started with the concentration (100 ng/mL). The starting concentration was diluted twofold in 20 subsequent steps. Our results are shown in figure 2. The absence of bars for the four highest concentrations is due to the fact that the cultures did not reach an OD<SUB>600</SUB> of 0.2 in the six hours of the measurement.  Remarkably, we observed reasonable growth of those same cultures in the preculture already induced with the identical amount of ATc. Knowing that luminescence is produced at the end of an enzymatic cascade, starting with intermediates of the phospholipid metabolism <A href="://www.ncbi.nlm.nih.gov/pmc/articles/PMC372803/"><ABBR title="E. A. Meighen, Synthetic Molecular biology of bacterial bioluminescence. (1991) 123–142">( Meighen 1991)</ABBR></A>, we reckon that very strong induction could decrease the fitness of cells and that after dilution in room temperature medium, strained cells are not able to recover from the stationary phase.  
However, we only observed this phenomenon in experiments with pTet, although we obtained higher signals for the strongest constitutive promoters as well as for the highly induced pTrc. We checked for toxicity of ATc but could not see a measurable effect. Another possibility is that TetR interacts with components inside the cell and that high ATc increases these interactions.  
Blast searches of TetR against the genome of <I>V. natriegens </I>identified one protein that shares some homology with the N-terminal part of TetR which could result in cross talk between the host and the inducible promoter.<FIGURE style="width:70%; float: center"><FIGCAPTION><B> Figure 2: Dose response of pTet with ATc.</B> J23100 was used as positive control and for normalization. Error bars represent the standard deviation of the measurements of three independent experiments </FIGCAPTION></FIGURE>

All measured data were normalized to the strongest constitutive promoter J23100. Saturation occurred at a dilution of 2^6 (~ 1.6 ng/mL) and an exponential reduction of luminescence signal can be observed for higher dilutions. In the absence of ATc, the signal is twelve fold lower compared to saturation. 
pTet allows relatively tight control of gene expression and is therefore well suited for driving the expression of potentially toxic proteins. On the other hand, we were not able to induce strong expression that can compete with strong constitutive promoters or the fully induced pTrc.<H3> Characterization of pTrc</H3>
pTrc is the second tested inducible promoter. It contains <I>lac</I> operator sites and is therefore regulated by the repressor LacI which is constitutively expressed from a downstream gene.  pTrc can be induced Isoopropyl-β-D-thiogalactopyranosid (IPTG), a chemical derivative of lactose <A href=" https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3922697/ "><ABBR title="Daniel Camsund, Thorsten Heidorn, Peter Lindblad, Design and analysis of LacI-repressed promoters and DNA-looping in a cyanobacterium (2014) 8:4">( Camsund <I>et al.</I>2014)</ABBR></A>. Similar to our experiments with pTet, we made a dilution series starting with the commonly used IPTG concentration for <I>E. coli</I> 0.5 mM. We observed a five fold induction and a saturation that occurred at a dilution of 2^5 (~15 µM). The strongest expression is similar to the expression gained from the strongest constitutive promoter J23100 while the expression in the absence of inducer equals medium strong promoters.  
As a consequence, we do not recommend using pTrc in constructs where a tight control of gene expression is desired. Instead, pTrc is well suited when strong expression is required.<FIGURE style="width: 70%; float: center"><FIGCAPTION><B> Figure 3: Dose response of pTrc with IPTG.</B> J23100 was used as positive control and for normalization. Error bars represent the standard deviation of the measurements of three independent experiments </FIGCAPTION></FIGURE>
Taking the results of both inducible promoters into account, we made two observation. In both cases, the dynamic range is smaller compared to <I>E. coli </I>and the inducer concentration that facilitates saturation is 32 and 64 fold lower for pTrc and pTet, respectively, than the concentration that is typically used for <I>E. coli</I>. A possible explanation could be found in the fast growth of <I>V. natriegens</I> which might result in a lower concentration of the repressor proteins in the cells, finally leading to a less restricted control of the negatively regulated promoters. However, we do not have experimental support for our idea.</DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Characterization of Connectors</DIV><DIV class="content">

One novel key feature of our toolbox are the connectors. They were designed in order to function as insulators to prevent crosstalk between neighboring transcription units <A href="https://2018.igem.org/Team:Marburg/Design">(Link to Design)</A>. Therefore a perfectly insulating connector would prevent the readthrough from backbone sequences that most probably caused the notably high expression that was measured in the promoter experiment for the dummy promoter. In addition to blocking transcriptional readthrough, a good connector must not possess any cryptic promoter activity. 
We focused on characterizing the 5’ Connector because we expect the stronger influence on signal strengths. For characterizing our connector parts, we created 20 test plasmids with the <I>lux</I> operon as the reporter. 
In our toolbox we provide five short connectors, which solely possess the fusion sites for LVL2 cloning, and five long connectors which additionally harbor self-designed insulators <A href="https://2018.igem.org/Team:Marburg/Design">(Link to Design)</A>.
Each of these ten connectors were cloned with the constitutive promoter J23100, to check for effects on an active promoter, and with the Promoter Dummy to quantify the extent of transcriptional activity that reaches the Promoter Dummy. <FIGURE><DIV class="imageContainer1x2"><DIV>A</DIV><DIV>B</DIV></DIV><FIGCAPTION><B>Figure 4:  Results of Connector measurments</B> A) Connector constructs built with J23100 as promoter part  B) Connector constructs built with the Dummy Promoter as promoter part

    </FIGCAPTION>

The acquired data are shown in figure 4. The data were normalized over the test construct J23100, that was used in the promoter experiment and constructed with the connector dummies. For the five constructs with the active promoter and the long connectors we observed extremely varying signals (figure 4, A). We measured a range from 0.2 to 2 fold change compared to the reference construct. It has been shown that the sequence directly upstream of small synthetic promoters can greatly impact the transcription efficiency <A href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0176013"><ABBR title=" Swati B. Carr, Jacob Beal, Douglas M. Densmore , Reducing DNA context dependence in bacterial promoters (2017)">( Carr <I>et al.</I>2017)</ABBR></A>. In case of the long connectors, the sequence upstream of the promoter forms the terminator and could affect the efficiency of RNA-polymerase binding to the -35 and -10 regions.  
For the constructs built with small connectors, we also observed varying signals but to a lesser extent compared to the long connectors (figure 4, B).
For all ten connectors that are provided in our toolbox, we show a tenfold range in the measured luminescence/OD<SUB>600</SUB> signal. As a conclusion, we recommend to carefully consider the combination of promoter and 5’ Connector for rationally designing constructs. 

Taking a look at the constructs that were built with the Promoter Dummy, we also see a huge difference in the expression signals. For the long connectors we expected a negligibly low reporter expression which we observed for two out of five long 5’ Connectors resulting in a 14 fold signal reduction compared to the “Promoter Dummy” reference. We interpret this result as the confirmation that those two connectors function as insluators! 
 The remarkably strong signal observed for the remaining three connectors could be due to inefficient terminators or cryptic promoters in the pretended “neutral sequence”. 

For the remaining five constructs possessing the five short 5’ connectors we observed a range from 0.3 to 5.5 fold compared to the “Promoter Dummy” reference. We are not able to give an experimental explanation for this observation but we could imagine that the LVL2 fusion sites, the only four bases that differ in these constructs, could constitute a weak promoter together with surrounding sequences.
Summarizing the connector characterization, we found that sequences upstream of short synthetic promoters greatly affect reporter expression, which is in accordance with literature <A href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0176013"><ABBR title=" Swati B. Carr, Jacob Beal, Douglas M. Densmore, Reducing DNA context dependence in bacterial promoters (2017)">( Carr <I>et al.</I>2017)</ABBR></A>. Moreover, we demonstrated that two of our five self-designed connectors efficiently reduce the signal resulting from other sources than the actual promoter.
We additionally conclude that algorithms that predict the “neutrality” of sequences alone are not sufficient to create well functioning insulators. 

</FIGURE></DIV><DIV class="collapsible"><DIV class="btn_expand">Characterization of Terminators</DIV><DIV class="content"><P><FIGURE style="width: 40%; float: right"><FIGCAPTION><B> Figure 5: Terminator test construct</B> LVL2 plasmids were created for these experiments consisting of a RFP transcription unit with the strong constitutive promoter J23100, followed by the <I>lux</I> operon with the promoter dummy. The terminator located at the 3' end of the RFP transcription unit is the part which is characterized in this experiment. 
</FIGCAPTION></FIGURE>
The Marburg Collection contains five terminators plus one terminator dummy that can be used as a placeholder. To obtain experimental data for this category of parts, we built a set of terminator test constructs to measure the extent of transcriptional readthrough and therefore the strength of a terminator. The terminator test constructs are built as LVL2 plasmid with our toolbox. The strongest constitutive promoter J23100 drives the expression of RFP which is the first transcription unit. The Lux operon is placed downstream with the promoter dummy instead of an active promoter. Both transcription units are separated by the terminator, which is the focus of this characterization, downstream of the RFP CDS 

With this setup, transcriptional activity of the RFP reporter is blocked by the terminator. Therefore the measured luminescence signal can be seen as an indicator for the efficiency of the terminator. 
As discussed previously, RFP is not suitable for precise quantitative characterizations <A href="https://2018.igem.org/Team:Marburg/Measurement">(Link to Measurement)</A>. Therefore we did not calculate ratios of the reporter upstream and downstream of the tested reporter as was described in previous experiments  <A href="https://www.nature.com/articles/nmeth.2515"><ABBR title="Ying-Ja Chen, Peng Liu, Alec A K Nielsen, Jennifer A N Brophy, Kevin Clancy, Todd Peterson, Christopher A Voigt, Characterization of 582 natural and synthetic terminators and quantification of their design constraint (2013) 659–664">( Chen <I>et al.</I>2013)</ABBR></A>. However, we used the existence of an RFP signal as control for the correctly assembled test constructs and for the activity of the promoter driving RFP.<FIGCAPTION><B> Figure 6: Characterization of terminator read through</B> Displayed is the relative luminescence signal to the control construct J23100 obtained for constructs with the respective terminator. Error bars represent the standard deviation of the measurements of three independent experiments.  
</FIGCAPTION>

The data shown in figure 6 were acquired and analyzed following our novel workflow described on the measurement page <A href="https://2018.igem.org/Team:Marburg/Measurement">(Link to Measurement)</A>. Like in all previous experiments, the obtained raw data for each sample were normalized over the construct J23100 from the promoter characterization.The strongest signal was observed for B1002 and B0010 with a relative signal 0.65 and 0.50 respectively, suggesting these two terminators as rather inefficient. In contrast, we could show a signal reduction four and eight fold for B0015 and B1006, respectively. 
By comparing our data to the characterization provided in the iGEM registry for <I>E. coli</I>, we can show that the general trend is similar for both organisms. Exemplary, the terminator with the highest described efficiency in <I>E. coli</I> (B1006) also was found to reduce the luminescence signal most in our experiments. 
In general, we found stronger signals for the reporter downstream of the terminator than what was described for <I>E. coli</I>. However, it has to be noted, that we used the highly sensitive reporter Lux instead of a fluorescent protein. Therefore we assume that we were able to detect a higher degree of transcriptional readthrough, which would not be distinguishable from the background when using a fluorescence reporter.However, we cannot exclude species specific differences that cause a generally higher degree of transcriptional readthrough over terminators for<I> V. natriegens</I> compared to <I>E. coli</I>

In addition to the terminators shown in figure 6 we also measured a test construct possessing B1003. The, by far, lowest signal was found for B1003 with a 15000 fold reduction of the signal. To ensure that this extremely weak signal is authentic and not caused by cross talk from neighbouring wells, we tested this sample in an otherwise empty 96 well plate and confirmed the general existence of a signal. However, we can not exclude the presence of single mutations within the <I>lux</I> operon dramatically diminishing the overall generation of luminescence. This result was not displayed in figure 6 to omit extreme stretching of the Y-Axis thus loosing visual information for the other tested terminators.
</P></DIV><DIV class="collapsible"><DIV class="btn_expand">Measuring RBS strength</DIV><DIV class="content"><P> 

For quantifying the RBS strengths, unfortunately, we could not use our favorite reporter the lux operon because in case of this operon, each CDS possesses its own RBS. Therefore, replacing the RBS upstream of the first CDS (LuxA) alone does not suffice to achieve a difference in reporter expression dependent on the RBS strength. Consequently we used sfGFP, the reporter that showed the second best performance in our initial reporter experiment <A href="https://2018.igem.org/Team:Marburg/Measurement">(Link to Measurement)</A>.<FIGURE style="width:50%; float: right"><FIGCAPTION><B> Figure 7: RBS strength measured with sfGFP</B> Test constructs were built with the same parts except for the RBS part. The sample &quot;Empty&quot; represents <I>V. natriegens</I> with a plasmid without a sfGFP reporter. The error bars indicate the standard deviation of four technical replicates. </FIGCAPTION></FIGURE>

We built test constructs that are identical in all used parts except for the used RBS. The resulting data are shown in figure 7. The sample labeled with “empty” represents <I>V. natriegens</I> with a plasmid without a reporter. Apparently, the difference between the tested constructs expressing sfGFP in different amounts does not differ much from the non expressing control. However, it is possible to obtain some information about the order of the tested RBS. The strongest signal was observed for B0034 followed by B0030. These RBS are also described as strong in the iGEM registry. In case of B0031 and B0032, the measured signal is lower than for the non-sfGFP expressing strain. Therefore no conclusion can be provided for these two parts. 

We expected that the order of the strength of the tested RBS should be similar to <I>E. coli</I>. Prior to the experiment, we created an alignment of the 16S rRNA of both organisms and found that the Anti-Shine-Dalgarno sequence, the bases that are responsible for binding the Shine-Dalgarno sequence on the mRNA, do not differ between both organisms (figure 8).
<FIGURE style="width:98%; float: center"><FIGCAPTION><B> Figure 8: Alignment of 16S rRNA of <I>E.coli</I> MG1655 and<I> V. natriegens</I> ATCC14048 </B> The Anti-Shine-Dalgarno sequences is indicated with an additional annotation. The 3' sequence of the 16S rRNA does not differ between both organism </FIGCAPTION></FIGURE>
On the measurement page <A href="https://2018.igem.org/Team:Marburg/Measurement">(Link to Measurement)</A>. we suggested the <I>lux</I> operon as the reporter of choice for all experiments that focus on measuring transcriptional activity . We see the RBS characterization as the confirmation that using sfGFP as reporter does not yield reliable data for very weak expression. 
In future experiments, enzymatic reporters such as LacZ or β-glucuronidase (GUS) could be tested for their suitability in experiments for the quantification of translational efficiency or post translational effects (e.g. degradation tags)

</P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Analyzing degradation tags</DIV><DIV class="content"><P>


Our toolbox contains the three degradation tags M0050, M0051 and I11012. Similar to our RBS experiments, we could not use the <I>lux</I> operon as a reporter because the degradation tag is only added to the C-terminal end of the last enzyme encoded in the operon. Therefore, we again used sfGFP as reporter and test constructs were designed with one of the tags fused to the CDS of sfGFP. A non-tagged sfGFP construct serves as the reference.




<FIGURE style="width: 40%; float: right"><FIGCAPTION><B> Figure 9: Fluorescence signal for degradation tag test constructs</B> The respective degradation tags were appended to the C-terminus of sfGFP. The error bars indicate the standard deviation of four technical replicates. 
</FIGCAPTION></FIGURE>
As expected, appending a degradation tag to a protein decreases its concentration. The strongest decrease and therefore the highest degradation was shown for M0050 with a signal that is not distinguishable from the non-sfGFP expressing control. The second strongest signal reduction was shown for I11012, followed by M0051, which showed the least efficient degradation (figure 9). Our results are in qualitative accordance with the description of these tags for <I>E. coli</I>, which are provided in the registry. 
The tested degradation tags belong to the family of SsrA degradation tags. Naturally, they help to degrade incompletely translated proteins by labeling them for Clp proteases (Farrell <I>et al.</I> 2015). We checked for the presence of ClpP, one of the proteases involved in degrading SsrA tagged proteins in <I>E. coli</I>  and found a highly homologous protein in <I>V. natriegens</I>. Therefore we assume that the mechanism of Clp mediated degradation is conserved between both organisms, which is in accordance with our results.

</P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Influence of Codon optimization</DIV><DIV class="content"><P>

Each organism has a preferred codon usage that affects the efficiency of translation through the abundance of tRNAs (Rocha, 2004). The tRNA composition differs between organism which can result in a loss of  protein expression. The goal of our project is to replace <I>E. coli</I> in as many applications as possible. We know that scientists all over the world have been extensively working with <I>E. coli</I> for decades and have collected huge collections of plasmids for this organism. We already showed in many experiments that parts taken from<I> E. coli</I> are in general functional in <I>V. natriegens</I>. <FIGURE style="width: 40%; float: right"><FIGCAPTION><B> Figure 10: Comparing codon optimized sfGFP with non-optimized sfGFP.</B> The error bars indicate the standard deviation of four technical replicates. 
</FIGCAPTION></FIGURE>


However, we wanted to test if optimizing the codons of a CDS can enhance the expression levels. Therefore, we ordered the DNA sequence of a sfGFP CDS with optimized codons for <I>V. natriegens</I>. In figure 10 we compare the signal of two constructs with and without an optimized sequence. Our experimental data suggest that codon optimization results in a considerable increase of the fluorescence signal. In conclusion, while an acceptable level of expression can be observed for parts without optimizing the codon usage, newly synthesized sequences optimized for <I>V. natriegens</I> can increase the expression levels. This information could prove important for industrial applications where even small changes in product yield can decide if an application is economically advantageous.  We have to note that this conclusion is based on a single experiment with a single tested CDS. To further confirm the impact of codon optimization on expression levels, this experiment should be repeated with various sequences.


</P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Characterization of origins of replication</DIV><DIV class="content">
					
					Origins of replication (Oris) are genetic elements where DNA replication is initiated. In plasmids the <ABBR title="origin of replication">Ori</ABBR> sequence is responsible for its maintenance and for the copy number inside the cell (<A href="https://www.cell.com/cell/abstract/0092-8674(83)90502-0"><ABBR title="Selzer G, Som T, Itoh T, Tomizawa J. 1983. The origin of replication of plasmid p15A and comparative studies on the nucleotide sequences around the origin of related plasmids. Cell 32: 119–129.">Selzer <I>et al.</I>, 1983</ABBR></A>; <A href="http://www.asmscience.org/content/journal/microbiolspec/10.1128/microbiolspec.PLAS-0001-2013"><ABBR title="Brantl S. 2014. Plasmid Replication Control by Antisense RNAs. Microbiology Spectrum 2.">Brantl, 2014</ABBR></A>).
					
					The origins of replication colE1, pMB1 and p15A belong to the same family. They do not code for any enzyme but are replicated by the hosts RNA polymerase (<A href="http://www.sciencedirect.com/science/article/pii/0168952591903706"><ABBR title="Cesareni G, Helmer-Citterich M, Castagnoli L. 1991. Control of ColE1 plasmid replication by antisense RNA. Trends in Genetics 7: 230–235.">Cesareni <I>et al.</I>, 1991</ABBR></A>; <A href="http://www.asmscience.org/content/journal/microbiolspec/10.1128/microbiolspec.PLAS-0001-2013"><ABBR title="Brantl S. 2014. Plasmid Replication Control by Antisense RNAs. Microbiology Spectrum 2.">Brantl, 2014</ABBR></A>). The polymerase transcribes a region 508 bp upstream the <ABBR title="origin of replication">Ori</ABBR> sequence (<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC348984/"><ABBR title="Tomizawa J, Itoh T. 1981. Plasmid ColE1 incompatibility determined by interaction of RNA I with primer transcript. Proceedings of the National Academy of Sciences of the United States of America 78: 6096–6100.">Tomizawa &amp; Itoh, 1981</ABBR></A>; <A href="https://www.cell.com/cell/abstract/0092-8674(83)90502-0"><ABBR title="Selzer G, Som T, Itoh T, Tomizawa J. 1983. The origin of replication of plasmid p15A and comparative studies on the nucleotide sequences around the origin of related plasmids. Cell 32: 119–129.">Selzer <I>et al.</I>, 1983</ABBR></A>) synthesizing a pre-primer RNA called RNA II. During transcription the RNA II underlies conformation changes building secondary structures(<A href="http://www.asmscience.org/content/journal/microbiolspec/10.1128/microbiolspec.PLAS-0001-2013"><ABBR title="Brantl S. 2014. Plasmid Replication Control by Antisense RNAs. Microbiology Spectrum 2.">Brantl, 2014</ABBR></A>). This structures contain typical loops (<A href="http://www.sciencedirect.com/science/article/pii/0168952591903706"><ABBR title="Cesareni G, Helmer-Citterich M, Castagnoli L. 1991. Control of ColE1 plasmid replication by antisense RNA. Trends in Genetics 7: 230–235.">Cesareni <I>et al.</I>, 1991</ABBR></A>) that binds to the plasmids’ <ABBR title="origin of replication">Ori</ABBR> sequence building an RNA-DNA hybrid (<A href="http://www.sciencedirect.com/science/article/pii/0168952591903706"><ABBR title="Cesareni G, Helmer-Citterich M, Castagnoli L. 1991. Control of ColE1 plasmid replication by antisense RNA. Trends in Genetics 7: 230–235.">Cesareni <I>et al.</I>, 1991</ABBR></A>; <A href="http://www.asmscience.org/content/journal/microbiolspec/10.1128/microbiolspec.PLAS-0001-2013"><ABBR title="Brantl S. 2014. Plasmid Replication Control by Antisense RNAs. Microbiology Spectrum 2.">Brantl, 2014</ABBR></A>). The RNA II is than cleaved by the hosts RNase H to become a mature primer (<A href="http://www.sciencedirect.com/science/article/pii/0168952591903706"><ABBR title="Cesareni G, Helmer-Citterich M, Castagnoli L. 1991. Control of ColE1 plasmid replication by antisense RNA. Trends in Genetics 7: 230–235.">Cesareni <I>et al.</I>, 1991</ABBR></A>; <A href="http://www.asmscience.org/content/journal/microbiolspec/10.1128/microbiolspec.PLAS-0001-2013"><ABBR title="Brantl S. 2014. Plasmid Replication Control by Antisense RNAs. Microbiology Spectrum 2.">Brantl, 2014</ABBR></A>).
					<H4>Results</H4>
					For our collection we characterized three <ABBR title="origins of replication">Oris</ABBR> commonly used in molecular biology: colE1, pMB1 and p15A. We measured two different plasmids, one with and another without a LUX cassette. Both plasmids consist of a <A href="http://parts.igem.org/Part:BBa_K2560057"><ABBR title="Marburg Collection kanamycin resistance">kanamycin resistance</ABBR></A> cassette and one of the three Oris described (<A href="http://parts.igem.org/Part:BBa_K2560036"><ABBR title="Marburg collection ori colE1">colE1</ABBR></A>, <A href="http://parts.igem.org/Part:BBa_K2560037"><ABBR title="Marburg collection ori pMB1">pMB1</ABBR></A>, <A href="http://parts.igem.org/Part:BBa_K2560046"><ABBR title="Marburg collection ori p15A">p15A</ABBR></A>). The <A href="http://parts.igem.org/Part:BBa_K2560051"><ABBR title="Marburg collection LUX">LUX</ABBR></A> expression plasmid contained a constitutively expressed LUX cassette of ~6kb (figure 1). The other one contained a connector sequence to build an ‘empty’ plasmid. By comparing this constructs you may consider that the copy number is not only influenced by the LUX expression but also by the plasmids sizes. This Oris belong to the same family differing in mutations in the RNA I region (<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC348984/"><ABBR title="Tomizawa J, Itoh T. 1981. Plasmid ColE1 incompatibility determined by interaction of RNA I with primer transcript. Proceedings of the National Academy of Sciences of the United States of America 78: 6096–6100.">Tomizawa &amp; Itoh, 1981</ABBR></A>; <A href="https://www.cell.com/cell/abstract/0092-8674(83)90502-0"><ABBR title="Selzer G, Som T, Itoh T, Tomizawa J. 1983. The origin of replication of plasmid p15A and comparative studies on the nucleotide sequences around the origin of related plasmids. Cell 32: 119–129.">Selzer <I>et al.</I>, 1983</ABBR></A>).
					<FIGURE><FIGCAPTION><B>Figure 1: Plasmid map of the plasmids built for the quantification.</B>
							Two different plasmids were assembled and quantified via qPCR. The first plasmid was an 'empty' plasmid consisting of a resistance cassette and the corresponding origin of replication. A short connector sequence was designed to fuse the resistance cassette and the origin. For the second plasmid the connector sequence was replaced by the LUX operon. This way we compare a very small plasmid without expression with a huge one with high expression.
						</FIGCAPTION></FIGURE>
					We measured the plasmids’ copy number by qPCR using the absolute quantification method.
					A qPCR is set up the same way like a normal PCR but with addition of a DNA binding fluorophore in this case <A href="https://secure.eurogentec.com/product/research-takyon-for-sybr-assay-no-rox.html"><ABBR title="Eurogentec Takyon qPCR master mix">SYBR Green</ABBR></A>. SYBR Green binds double stranded DNA emitting a high signal while unbound SYBR Green shows only low fluorescence (<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC484200/"><ABBR title="Zipper H, Brunner H, Bernhagen J, Vitzthum F. 2004. Investigations on DNA intercalation and surface binding by SYBR Green I, its structure determination and methodological implications. Nucleic Acids Research 32: e103.">Zipper <I>et al.</I>, 2004</ABBR></A>). In every PCR cycle the number of double stranded DNA is duplicated emitting an increasing fluorescence signal. This signal is detected after every cycle by the qPCR machine and the value is saved. After the run finished, normally after ~40 cycles, a signal threshold is determined and the corresponding cycle when the threshold was reached is saved for further analysis.For the qPCR run first total DNA from our host containing the plasmids of interest was isolated in the exponential phase (OD<SUB>600</SUB> ~ 0.5), purified using the <A href="https://www.analytik-jena.de/en/life-science/products/prod/cat/isolation-of-microbial-dna/prod/innuprep-bacteria-dna-kit.html"><ABBR title="innuPREP Bacteria DNA Kit">innuPREP Bacteria DNA Kit from Analytik Jena</ABBR></A> and all samples normalized to ~5ng/ul with the <A href="https://www.thermofisher.com/de/de/home/references/newsletters-and-journals/bioprobes-journal-of-cell-biology-applications/bioprobes-issues-2011/bioprobes-64-april-2011/the-qubit-2-0-fluorometer-april-2011.html"><ABBR title="Qubit 2.0 DNA analyzer">Qubit 2.0 fluorometer from ThermoFisher scientific</ABBR></A>. Subsequently a dilution series was made in 1.5ml tubes diluting the DNA 7 times 1:2. This way the dilution series contained 8 steps reaching from 2<SUP>0</SUP> to 2<SUP>-7</SUP>. Two different primer pairs were used for the analysis: one matching the housekeeping gene <I>dxs</I> present once on the genome and the other matching the kanamycin resistance cassette on the plasmid. We choose the <I>dxs</I> sequence to be able to compare our results to the values reported by literature for <I>E. coli</I>. The DNA samples used for the amplification of the kanamycin cassette were the same used for the dilutions 2<SUP>-4</SUP> and 2<SUP>-5</SUP>. The threshold cycles (Ct) acquired in triplicates from the <I>dxs</I> sequence were used for a standard curve. By comparing the Ct values from the resistance cassette with the corresponding standard curve the number of copies could be determined as multiples from the <I>dxs</I> sequence. It should be considered that the <I>dxs</I> sequence is coded on the first chromosome of <I>V. natriegens</I> at ~ one o’clock. Due to that probably the sequence is present more than once because of multifork replication of the genome.
					
					To build the standard curve the Ct values were plotted on the y-axis and the corresponding dilution steps on the x-axis. The x-axis was set logarithmic and the standard curve was calculated with Excel. The curve’s formula was then used to calculate the corresponding x-value from the resistance cassette’s Ct values. Because the x-values describe a theoretical dilution the Ct values were multiplied with this value and with their corresponding dilution to obtain the final amount of multiplies compared to the genome. For every <ABBR title="origin of replication">Ori</ABBR> an own standard curve was calculated.
					
					In our experiments we showed that the plasmids’ copy number controlled by three different Oris differ a lot when comparing <I>V. natriegens</I> with <I>E. coli</I>.One possible explanation might be different expression levels of RNA I and RNA II respecting the rate of RNA I – RNA II bounds (<A href="http://www.sciencedirect.com/science/article/pii/0168952591903706"><ABBR title="Cesareni G, Helmer-Citterich M, Castagnoli L. 1991. Control of ColE1 plasmid replication by antisense RNA. Trends in Genetics 7: 230–235.">Cesareni <I>et al.</I>, 1991</ABBR></A>) due to the divergent metabolism in <I>V. natriegens</I> and <I>E. coli</I>. Another plausible explanation might be the different methylation patterns in both organisms probably affecting the formation of the RNA II secondary structures and subsequently its binding affinity to the DNA (<A href="https://www.sciencedirect.com/science/article/pii/0092867487901735?via%3Dihub"><ABBR title="Russell DW, Zinder ND. 1987. Hemimethylation prevents DNA replication in E. coli. Cell 50: 1071–1079.">Russell &amp; Zinder, 1987</ABBR></A>; <A href="http://www.sciencedirect.com/science/article/pii/0168952591903706"><ABBR title="Cesareni G, Helmer-Citterich M, Castagnoli L. 1991. Control of ColE1 plasmid replication by antisense RNA. Trends in Genetics 7: 230–235.">Cesareni <I>et al.</I>, 1991</ABBR></A>).
					
					It was shown that mutations especially in the loop I structure might be responsible for Ori compatibility and copy number control (<A href="https://www.cell.com/cell/abstract/0092-8674(83)90502-0"><ABBR title="Selzer G, Som T, Itoh T, Tomizawa J. 1983. The origin of replication of plasmid p15A and comparative studies on the nucleotide sequences around the origin of related plasmids. Cell 32: 119–129.">Selzer <I>et al.</I>, 1983</ABBR></A>; <A href="http://www.sciencedirect.com/science/article/pii/0168952591903706"><ABBR title="Cesareni G, Helmer-Citterich M, Castagnoli L. 1991. Control of ColE1 plasmid replication by antisense RNA. Trends in Genetics 7: 230–235.">Cesareni <I>et al.</I>, 1991</ABBR></A>). The copy number is mainly determined by two factors: the binding efficiency of the RNA II to the DNA – specially controlled by the stabilization of stem-loop IV – (<A href="http://www.sciencedirect.com/science/article/pii/0168952591903706"><ABBR title="Cesareni G, Helmer-Citterich M, Castagnoli L. 1991. Control of ColE1 plasmid replication by antisense RNA. Trends in Genetics 7: 230–235.">Cesareni <I>et al.</I>, 1991</ABBR></A>) and the interference of the complementary RNA I to the RNA II pre-primer (<A href="http://www.asmscience.org/content/journal/microbiolspec/10.1128/microbiolspec.PLAS-0001-2013"><ABBR title="Brantl S. 2014. Plasmid Replication Control by Antisense RNAs. Microbiology Spectrum 2.">Brantl, 2014</ABBR></A>). The RNA I is transcribed constitutively from the complementary strand from RNA II pre-primer (<A href="http://www.asmscience.org/content/journal/microbiolspec/10.1128/microbiolspec.PLAS-0001-2013"><ABBR title="Brantl S. 2014. Plasmid Replication Control by Antisense RNAs. Microbiology Spectrum 2.">Brantl, 2014</ABBR></A>). Binding of RNA I to RNA II prevents the correct folding of the pre-primer (<A href="http://www.asmscience.org/content/journal/microbiolspec/10.1128/microbiolspec.PLAS-0001-2013"><ABBR title="Brantl S. 2014. Plasmid Replication Control by Antisense RNAs. Microbiology Spectrum 2.">Brantl, 2014</ABBR></A>). This way the RNA-DNA hybrid can not be formed and subsequently the primer maturation can not take place (<A href="http://www.asmscience.org/content/journal/microbiolspec/10.1128/microbiolspec.PLAS-0001-2013"><ABBR title="Brantl S. 2014. Plasmid Replication Control by Antisense RNAs. Microbiology Spectrum 2.">Brantl, 2014</ABBR></A>).
					
					<FIGURE><FIGCAPTION><B>Figure 2: Quantification of plasmid copy number in dependency of different Oris.</B>
							The columns show the average of the calculated multiplies for the different plasmids. The blue columns show the numbers for the plasmids containing the ~6kb LUX cassette. The orange columns show the numbers for the ‘empty’ plasmids without reporter. For every column six measurements have been calculated. Looking at the ‘empty’ plasmids it is clearly shown that colE1 and p15A remain high copy plasmids like in E. coli with a copy number of ~200 copies per cell. For pMB1 the copy number is scaled down becoming a low copy number Ori in V. natriegens. Looking at the LUX plasmids it is clearly shown that the colE1 Ori remains at a high copy number while pMB1 and p15A drop down to a significantly lower level.
						</FIGCAPTION></FIGURE><H3>Acknowledgements</H3>
					We would like to thank Marcel Wagner for his support during our project with the quantification of plasmids copy number, the set up of this experiments and the handling with the qTower. Thanks to his support we were able to perform and evaluate this experiments to fulfill this study.
				</DIV></DIV></DIV></DIV></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand"><DIV class="btn_e_title">Strain Engineering</DIV><DIV class="btn_e_desc">
			Here we show how to create genetic modifications using the natural competence regulator TfoX of <I>V. natriegens</I> and the Flp/<I>frt</I> system. Additionally we detected and characterized 15 possible sites for
             genomic integrations. With these genetic tools, we were able to create the strains <B>VibriClone</B> for fast cloning, <B>VibriInteract</B> for accelerated protein interaction studies and started establishing <B>VibriExpress</B> for expressing high amounts of proteins.
		</DIV></DIV><DIV class="content"><DIV class="collapsible"><DIV class="btn_expand">Methods</DIV><DIV class="content"><H3>Natural Competence</H3>
In our project we wanted to build many different strains. For this we have to make many genomic changes of the wildtype of <I>Vibrio natriegens</I>. Natural transformation is a good method for multiplex genome editing in a short time (Figure 1).

<FIGURE style="display: block; float: right; width: 60%;"><FIGCAPTION><B>Figure 1:</B>Workflow of a natural transformation.
</FIGCAPTION></FIGURE>
To get in touch with  multiplex genome editing by natural transformation, we constructed a linear DNA fragment for transformation. We have chosen an antibiotic resistance for selection, which has to be different to the resistance marker on the plasmid in the strain (Figure 2).
<FIGURE style="display: block; float: left; width:40%;"><FIGCAPTION><B>Figure 2:</B>Schematic figure on how to build a cassette for integration. Amplify the upstream and downstream regions of the gene of interest and the antibiotic cassette with frt sites. These three Fragments will be fused. 
</FIGCAPTION></FIGURE>
With a linear DNA fragment containing the chloramphenicol cassette with upstream and downstream regions of the <I>dns</I> gene, we try to delete this gene by natural transformation in a <I>V. natriegens</I> strain without genetic modifications (Figure 3A). To check if the deletion and the natural transformation was successfull, we did colony-PCRs with primers binding inside the inserted fragments (1. upstream-CmR -downstream: ~1940bp; 2. CmR -downstream: ~1440bp; 3. upstream-CmR: ~1440bp). We could prove that the deletion cassette was inserted into the genome and this shows that the natural transformation was successful (Figure 3b). 
<FIGURE style="display: block; float: right; width: 50%;"><FIGCAPTION><B>Figure 3:</B>1 %- Agarose gel of the test digest. 2-logDNA ladder is used as marker on agarose gels.
</FIGCAPTION></FIGURE>
Using the natural competence regulator Tfox, we were able to transform pYTK.
To our best knowledge we were able to show the first natural transformation with a plasmid.
We could demonstrate natural transformation using our lab automation process by downscaling all reaction volumes, which has not been shown before. We were surprised since it has been reported that high amounts of DNA are needed for the uptake of the DNA <A href="https://www.sciencedirect.com/science/article/pii/S0717345818300083" target="blank"><ABBR title="Zeaiter, Z., Mapelli F., Crotti, E., Borin, S. Methods for the genetic manipulation of marine bacteria. Electronic Journal of Biotechnology. (2018) 33: 17-28">(Zeaiter<I>et al.</I>2017)</ABBR></A>.
To make the <I>tfox</I> part accessible to the scientific society, we wanted this part to be integrated into the Marburg Collection. By utilizing our MoClo based cloning system for our <I>tfox</I> expression, the expression level can specifically tailored for the intended application like transforming linear DNA fragment, ssDNA oligos or plasmids.
<FIGURE style="display: block; float: right; width: 25%;"><FIGCAPTION><B>Figure 4: Plates showing the successful transformation of plasmids using natural transformation. Left plate is showing transformation of cells with pYTK. Right plate is showing negative control.
</B></FIGCAPTION></FIGURE><H4>Lvl0</H4>
For implementing and domesticating the <I>tfox</I> gene into our Marburg collection, we had to built a Level 0 plasmid.
At first we amplified the <I>tfoX</I> gene from the pMMB-<I>tfox</I> plasmid <A href="https://www.ncbi.nlm.nih.gov/pubmed/28571309" target="blank"><ABBR title="Dalia TN, Hayes CA, Stolyar S, Marx CJ, McKinlay JB, Dalia AB. Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens. ACS Synth Biol. (2017) 15;6(9):1650-1655.">(Dalia<I>et al.</I>2017)</ABBR></A> and build a Lvl 0 plasmid to integrate this part in the Marburg Toolbox (if you want to see how to build a level 0 plasmid take a look at the Marburg Collection). After we were able to construct and build the plasmid, we first did a test digest with EcoRI and PstI to see if it was correct (Figure 4). Then we sent the plasmid for sequencing, we also received confirmation of the accuracy of our plasmid.<FIGURE style="display: block; float: left; width: 40%;"><FIGCAPTION><B>Figure 5:</B>1 %- Agarose gel of the test digest. 2-logDNA ladder is used as marker on agarose gels.</FIGCAPTION></FIGURE><H4>Lvl1</H4>
In order to construct two plasmids with full transcription units, with two different promoters we assembled <I>tfoX</I> together with the constitutive  promoters J23100 or J23110 into a transcriptional unit by golden gate assembly.<FIGURE style="display: block; float: right; width: 60%;"><FIGCAPTION><B>Figure 6:</B>1 %- Agarose gel of the test digest. 2-logDNA ladder is used as marker on agarose gels.</FIGCAPTION></FIGURE>
After transformation of this reactions into <I>E. coli</I> we were able to get colonies that seemed to be positive for the Lvl 1 plasmids encoding <I>tfoX</I> because they showed the right size of fragment after plasmid preparation and test digestion, whenever we sent them for sequencing, some bp of the <I>tfox</I> gene were always missing (Figure 6).We suggest that these problems could be caused because <I>tfox</I> is maybe somehow toxic to <I>E. coli</I> cells. 

<FIGURE style="display: block; float: left; width: 30%;"><FIGCAPTION><B>Figure 7:</B>Alignment of sequencing results of our first constructed Lvl1 tfoX plasmids showing missing basepairs at places at which black fragments are missing.
</FIGCAPTION></FIGURE><H4><I>tet</I> promoter</H4>
In addition to constitutive promoters, the Marburg Collection contains two inducible promoters, pTet and pTrc. To build our level 1 plasmid we chose the pTet promoter. The pTet promoter can be induced by the tetracycline derivative anhydrotetracycline (ATc). ATc is much less cytotoxic than IPTG but still capable of binding and altering the structure of the repressor TetR, leading to release of the promoter and enabling transcription.
<FIGURE style="display: block; float: right; width: 50%;"><FIGCAPTION><B>Figure 8:</B>1 %- Agarose gel of the test digest. 2-logDNA ladder is used as marker on agarose gels.</FIGCAPTION></FIGURE>
Using this suitable promoter for <I>V. natriegens</I> we successfully assembled a plasmid containing a full transcriptional unit with a pTet promoter and did not had any problems during the cloning process. By test digestion and sequencing we could prove, that now the sequence of <I>tfox </I>was correct (Figure 7 and 8).<H3>Establishing Flp/<I>frt</I> in <I>V. natriegens</I></H3><P>One of our goals was to create fast genomic modifications, establishing the Flp/<I>frt</I> system in <I>V. natriegens</I>. Therefore, we assembled a plasmid encoding the FLP recombinase, amplified from pBR-<I>flp</I> into the backbone K2560002, creating a Lvl 0 plasmid by Golden Gate assembly. Next, we used this created part K2560271 to assemble a Level 1 plasmid, encoding a full transcriptional unit under the control of a constitutive promoter. This plasmid was transformed into a <I>V. natriegens</I> strain with a successfully integrated chloramphenicol resistance cassette by electroporation. The resulting clones were tested by colony PCR and positive clones were cultivated in LBV2-Kanamycin. The three-phase streaking pattern on LBV2-Kanamycin was performed to get single colonies. These colonies were streaked on replica plates with LBV2-Kanamycin and LBV2-Kanamycin-Chloramphenicol to screen for chloramphenicol sensitive clones. Of 100 clones, only eleven clones were not chloramphenicol sensitive. Eight chloramphenicol sensitive clones were tested by four different colony PCRs: one to amplify the region of the gene of interest, a second PCR to amplify the upstream and chloramphenicol fragment, a third PCR would amplify the downstream-chloramphenicol fragment and the last PCR would amplify only the chloramphenicol cassette. The first PCR should result in a fragment size of 2400 bp in case of a successful excision mediated by the FLP recombinase. In the wildtype genotype, this PCR would result in a fragment size of 3900 bp. The second, third and fourth PCR would not show a fragment in case of the chloramphenicol excision.</P><P>In all eight tested clones, the excision of the chloramphenicol resistance cassette was proved.</P><CENTER><FIGURE style="display: center; width: 50%;"><FIGCAPTION><B>Figure 1:</B>Overview of Flp mediated excision of the chloramphenicol resistance cassette.
Arrows are marking the primers used for the PCR in Figure 2. 
</FIGCAPTION></FIGURE><FIGURE style="display: center; width: 50%;"><FIGCAPTION><B>Figure 2:</B>1 %- Agarose gels of the PCR products using primers of Figure 1, 2-logDNA ladder is used as marker on agarose gels. Expected fragment sizes were calculated using Geneious.</FIGCAPTION></FIGURE></CENTER>
Thus, we established a new method for creating genomic modifications in the world’s fastest growing organism <I>V. natriegens</I>. This is again one example, that commonly used methods are transferable to this amazing organism, without much effort.
Here we just show that Flp mediated excision is working. In future the functionality of Flp mediated insertion, translocation and inversion need to be demonstrated.



</DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Integration</DIV><DIV class="content"><H2>Results</H2><P>We wanted to investigate, whether the insertion of a fragment at our integration sites influences the growth rate of V. natriegens. Here we demonstrate that we successfully integrated a full transcriptional unit of a chloramphenicol resistance in all of the 15 vital integration sites by natural transformation.</P><P>
To verify a correct integration, we performed a colony PCR with sequence specific primers for the five most promising integration sites 12, 15, 19, 20 and 22 on both chromosomes. All fragments had the expected size of 1200bp.
</P><P>
	We further sequenced the PCR products of sequence 12, 15, 19 and 20 by singer sequencing. All of them showed a correct alignment and a integration at the right locus.
</P><P>
	To investigate, whether an integration at our integration sites results in a disadvantage in growth, we measured the growth of the five sequences and compared them to the WT.
</P><P>
	The data show that the mutants with the inserted sequences 12 and 20 grow much slower than the WT. In the case of sequence 12 a possible reason is that the ABC-transporter binding protein, which is upstream of the insertion site, got affected by the insertion. For the case of sequence 20, a disturbance in the function of the RNA pyrophosphohydrolase, responsible for the degradation of transcripts, could lead to an accumulation of undesired transcripts. This could result in a deficit in growth. Because of the loss in growth, these two insertion sites are not ideal for integration.
</P><P>
	Sequence 15 almost grows as fast as the WT. Here also a disturbation of the transposon resolvase could be the reason for the lower growth rate. This integration site is already suitable, but we found better ones.
</P><P>
	For sequence 19 and 22 we showed that an insertion at this locus has no effect in growth. Both strains grow equally, so that they are perfect suitablefor integration. 
</P><P>
	Thus, we characterized two optimal integration sites in the genome of V. natriegens. Interestingly both of the them are localized at the second chromosome of V. natriegens.  The second chromosome could be more suitable for integration because it is known, that most essential genes are encoded on the first chromosome of V. natriegens (Weinstock et al. 2016). 
</P><P>
	In future, more optimal integration sites could be found on this chromosome. 
</P><P>
	These optimal integration sites can further be used to generate genomic modifications for strain engineering e.g. the integration of a T7 cassette or katG for cold resistance. Thus, making V. natriegens our new lab pet.
</P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Strains</DIV><DIV class="content">
Based on our self-developed methods for fast genomic modifications, we could establish three strains for accelerated everyday lab use. We established our first cloning strain <B>VibriClone 1.0</B> and lay the foundation for <B>VibrioClone 2.0</B>. We got first results for <B>VibriExpress</B> on which we can do further work on in the future. With <B>VibriInteract</B> we successfully performed a Vibrio Two-Hybrid assay enabling fast protein-interaction studies.

<DIV class="collapsible"><DIV class="btn_expand">VibriClone</DIV><DIV class="content"><DIV><H3>VibriClone</H3>
	To build our first VibriClone we investigated which deletion is the most important one for a cloning strain. It has been shown for V. cholera that the transformation efficiency greatly increases when the two extracellular nucleases are knocked out. Thus, the DNA to be transformed into the organism is no longer degraded

	<A href="”https://www.ncbi.nlm.nih.gov/pubmed/18757542”"><ABBR title="”Blokesch," m.="">(Blokesch<I>et al.</I>2008)</ABBR></A>

    . In order to increase our transformation efficiency, we decided to delete the <I>dns</I> as our first target as it has been reported, that a knockout improves the plasmid amount and quality. Furthermore, for transformation by chemical competent cells or natural transformation the deletion is one of the crucial parameters.<P> The first step of creating a <I>Δdns</I> deletion strain was to build a selected product with the upstream and downstream regions of the <I>dns</I> (~500bp) and the antibiotic cassette chloramphenicol (~940bp). <I>frt</I> sequences, needed for deletion of <I>CmR</I> are used as overlapping sequences. The same amount of every fragment was used for the PCR. We were able to successfully create the fragment (Figure 1 A). 
    <I>V. natriegens</I> is able to take linear DNA Fragments up and integrate it by homologous integration into the genome. With the linear DNA fragment, it was possible to delete the <I>dns</I>V. natriegens strain without genetic modifications. To check if the deletion was successfully we made colony-PCRs with primers binding inside the inserted fragments (1. upstream-CmR -downstream: ~1940bp; 2. CmR -downstream: ~1440bp; 3. upstream-CmR: ~1440bp). We could prove that the deletion cassette was inserted into the genome (Figure 1 B). 
    </P><FIGURE style="display:block; margin:0 auto 0 auto; width: 50%;"><FIGCAPTION><B>Figure 1:</B>1 %- Agarose gel of the test digest. 2-logDNA ladder is used as marker on agarose gels.</FIGCAPTION></FIGURE><P>The next step was to cure the antibiotics cassette. In order to leave the possibility open to work with plasmids with a chloramphenicol resistance, it is very important to delete the inserted antibiotic cassette. Because of the <I>frt</I>-sites around the <I>CmR</I> it is possible to to cut it out. Therefore we use the pBR-Flp plasmid and transform it into the cells. Introducing this plasmid into <I>V. natriegens</I> and expressing Flp, the recombinase will recognize the integrated <I>frt</I> sites. By Flp, mediated recombination the resistance cassette will be cut out of the genome, creating the deletion of the desired gene. 
    A colony PCR, negative for <I>CmR</I> will prove the loss of resistence and the strain is almost ready for the usage. Therefore we used primers binding inside and outside of the cassette (1. upstream-CmR -downstream: ~1940bp; 2. CmR: ~940bp; 3. CmR -downstream: ~1440bp; 4. upstream-CmR: ~1440bp). We could prove that the CmR was cut out of the genome (Figure 2).</P><FIGURE style="display:block; margin:0 auto 0 auto; width: 50%;"><FIGCAPTION><B>Figure 2:</B>1 %- Agarose gel of the test digest. 2-logDNA ladder is used as marker on agarose gels.
</FIGCAPTION></FIGURE>

    The next step was getting rid of the plasmid containing the <I>tfox</I> gene. Therefore we grow a culture with 1mM IPTG and inoculate them a few times. After restricting the culture, picking single colonies and plate them on LBv2 with and without carbenicillin - the resistance on the plasmid. On Figure 3 you can see that 23 clones grow on LBv2 but not on LB V2 + carbenicillin. From this we can conclude that these cells have lost the pMMB-<I>tfox</I> plasmid. The next step is now to do colony PCRs to have a second backup, whether the plasmid is certainly no longer in the strain.<FIGURE style="display:block; margin:0 auto 0 auto; width: 50%;"><FIGCAPTION><B>Figure 3:</B>Plates LB V2 with and without carbenicillin to screen for clones.</FIGCAPTION></FIGURE><P>We were able to design and build the first VibriClone strain - VibriClone 1.0. With this achievement we can give the scientific society a possibility to work faster, get a higher efficient in transformation and to get faster results. However, as the name VibriClone 1.0 suggests it gets even better. Therefore we started with an upgrade called VibriClone 2.0.</P><H3>VibriClone 2.0</H3>
    The upgrade of the cloning strain should even have more practical and user-friendly features as the first one has already.
    We would like to knock out both exonucleases (Dns and ExeM) so that it would be possible to transform DNA without degrading. In addition, we would like to insert a point mutation in recA, that the homologous recombination between plasmids and into the genome. Moreover, we also want to make it possible to store in the fridge. Therefore we insert the katG gene encodes an enzyme with an catalase and peroxidase activity into the genome. Furthermore, we also wanted to make the blue-and-white selection possible for our VibriClone, we can achieve that by integrating the <I>lac</I> operon into the genome.<H4><I>Exonuclease</I></H4>
     As selected product we use the<I> dns </I>cassette. To use the cotransformation we build a cassette with 3000bp homologous regions upstream and downstream  of the <I>dns</I> and the chloramphenicol cassette in the middle. It was possible for us to amplify the individual fragments and in each case to fuse the upstream or the downstream fragment to the chloramphenicol cassette. Unfortunately we were not able to build the whole fragment yet.<FIGURE style="float:left; width: 30%;"><FIGCAPTION><B>Figure 4:</B>1 %- Agarose gel of the test digest. 2-logDNA ladder is used as marker on agarose gels.
</FIGCAPTION></FIGURE><FIGURE style="float:right; width: 40%;"><FIGCAPTION><B>Figure 5:</B>Sequencing of recA. With a zoom in the region of the point mutation.</FIGCAPTION></FIGURE><H4>RecA</H4>
     For our VibriClone 2.0, we would like to have a modification in the gene <I>recA</I>. It should be a point mutation from G to A at position 720. Thereby the glycin at the place 158 chance to aspartic acid. We use this model to modify our strain VibriClone. To insert a point mutation in <I>recA</I>, we built two fragments with the mutation in the overhang.
     It was possible for us to use a fusion PCR to construct an unselected product that contains the point mutation. 
     To check if the point mutation really took place, we sequenced the PCR product. The result clearly showed the modification we wanted to achieve.<FIGURE style="float:right; width: 30%;"><FIGCAPTION><B>Figure 6:</B>1 %- Agarose gel of the test digest. 2-logDNA ladder is used as marker on agarose gels.
</FIGCAPTION></FIGURE><H4>KatG</H4>
     Another feature, which would improve the applicability of <I>V. natriegens</I> as a cloning chassis, would be to engineer cold resistance, due to fact that we all are used to store our plates of <I>E. coli</I> in the fridge. For integrating the <I>katG</I> into the genome, we combined introducing <I>katG</I> with the deletion of the second nuclease by utilizing 3000 bp homologous overhangs of the upstream and downstream flanking genomic region of <I>exeM</I>. We were able to build this unselected fragment successfully (<I>exeM</I> upsteam - <I>katG</I> - <I>exeM</I> downstream: 8327 bp) (Figure 6). With this unselected product it is possible to perform a co-transformation and to integrate the point mutation into the genome.<FIGURE style="float:left; width: 30%;"><FIGCAPTION><B>Figure 7:</B>1 %- Agarose gel of the test digest. 2-logDNA ladder is used as marker on agarose gels.
</FIGCAPTION></FIGURE><H4>The <I>lac</I> operon</H4>
     In order to use the 𝛼-complementation we build a cassette consist of the upstream and downstream regions of <I>lacZ</I> and the lactose operon from DH5𝛼. Thus, it is possible to introduce the <I>lactose</I> operon by cotransformation as an unselected product with the other described products. 
     The <I>lactose</I> operon of <I>E. coli</I> has a size of 6326bp. Since that is very large we decided to use smaller homologous flanks. It was shown that the natural co-transformation work well with 1000bp flanks, so we use this size. Otherwise the linear DNA Fragment has a size of ~12000bp. In this case it was a good decision because to get the smaller fragment with 8342bp was hard enough. It only worked once to fuse the three fragments. But then the result could not be reproduced.<H4>Cotransformation</H4>
     In 2017 the group of Ankur B. Dalia showed that it was possible to integrate linear PCR products with an selected product containing an antibiotic cassette that replaces a gene and unselected products to modify the genome in the way you like in <I>V. natriegens</I> by using natural transformation 

     <A href="https://www.ncbi.nlm.nih.gov/pubmed/28571309"><ABBR title="Dalia TN, Hayes CA, Stolyar S, Marx CJ, McKinlay JB, Dalia AB. Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens. ACS Synth Biol. (2017) 15;6(9):1650-1655..">(Dalia<I>et al.</I>2017)</ABBR></A>

     . Our plan was to use this method to perform the genetic changes that we have considered and where we have built the fragments for. Since we were not able to build the dns deletion fragment with 3000bp, we built a smaller cassette (1000bp) in parallel to start creating the VibriClone 2.0 by using natural transformation. So far the natural transformation with the small selected product and the unselected products of the genomic modifications don´t work. This is not surprising because it was shown that the efficiency is much lower with 1000bp flanks 

     <A href="https://www.ncbi.nlm.nih.gov/pubmed/28571309"><ABBR title="Dalia TN, Hayes CA, Stolyar S, Marx CJ, McKinlay JB, Dalia AB. Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens. ACS Synth Biol. (2017) 15;6(9):1650-1655..">(Dalia<I>et al.</I>2017)</ABBR></A>

     . We wanted to try it nevertheless. However it is very important to get the selected product with big homologous regions. Thus it would be possible with a high efficiency to carry out the modifications and obtain our improved strain - VibriClone 2.0.</DIV></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">VibriExpress</DIV><DIV class="content">
Our goal was to create a T7 strain <B>VibriExpress</B>, by deleting the <I>lon</I> of <I>V. natriegens</I> for minimizing protein degradation, integrating the T7 system for expression of high protein amounts and deleting the <I>dns</I> encoding for a exonuclease to facilitate transformation efficiency of the plasmid encoding the gene of interest. This goal should be achieved designing and creating linear DNA fragments and co-transforming those fragments, using the natural competence of <I>V. natriegens</I><A href="https://www.ncbi.nlm.nih.gov/pubmed/28571309"><ABBR title="Dalia, T. N., C. A. Hayes, S. Stolyar, C. J. Marx, J. B. McKinlay and A. B. Dalia (2017). Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens. ACS Synth Biol 6(9): 1650-1655. ">(Hayes <I>et al.</I> 2017)</ABBR></A>

.
We followed two different strategies to create linear DNA fragments for the integration of the T7 RNA Polymerase into the <I>V. natriegens</I> genome and for the deletion of <I>lon</I> and <I>dns</I>. One of them was to integrate the T7 cassette to the endogenous locus of dns, integrating the T7 cassette and deleting <I>dns</I> at the same time. Therefore, the fusion of four different fragments to one cassette encoding the T7 RNA Polymerase, the chloramphenicol resistance and upstream and downstream homologous regions of <I>dns</I> was necessary. We successfully amplified all four needed single fragments and were able to fuse two fragments to each other but unfortunately, the fusion PCR, performed in a variety of combinations did not work. The reason for this could be, the palindromic <I>frt</I> sites attached to the chloramphenicol resistance cassette for Flp mediated excision afterwards.
Fortunately, we followed a second strategy in which the <I>dns</I> deletion cassette acts as a selected product consisting a chloramphenicol resistance cassette. The T7 RNA Polymerase gene was fused to up- and downstream homologous regions of the integration site 19 for genomic integration.

<FIGURE style="float: right; width: 30%;"><FIGCAPTION><B>Figure 1:</B>1 %- Agarose gels of the PCR products <I>dns</I> deletion cassette with an xprected fragment size of 2900 bp, <I>lon</I> deletion cassette with an expected fragment size of 6000 bp and the T7 integration cassette.with an expected fragment size of 6700 bp. 2-logDNA ladder is used as marker on agarose gels. Fragment sizes were calculated using DNAstar Geneious. </FIGCAPTION></FIGURE>
We successfully created three linear fragments for creation of the <B>VibriExpress</B> strain: the linear fragment for the <I>dns</I> deletion, the <I>lon</I> deletion cassette and the T7 integration cassette.<P>Those three fragments were transformed into <I>V. natriegens</I> by natural transformation. The transformation in general worked and we got clones in which the <I>dns</I> deletion cassette integrated. This was proved by colony PCR. In theory, the T7 integration cassette and the lon deletion cassette should also integrate to the <I>V. natriegens</I> genome at the same time. However, co-transformation of these linear fragments did not work because of more than 200-screened clones no one was showing the desired genetic modifications of <I>lon</I> deletion or T7 integration.</P><FIGURE style="display:block;float: center; width: 50%;"><FIGCAPTION><B>Figure 2:</B>1 %- Agarose gels of the PCR products of the colony PCR for T7 integration. In case of an integration a fragment size of 6700 bp was expected. In case of wiltipe genotype a size of 2000 bp was expected. 2-logDNA ladder is used as marker on agarose gels. Fragment sizes were calculated using DNAstar SeqBuilder. </FIGCAPTION></FIGURE><P>This could be due to the big fragment sizes of the <I>lon</I> and T7 cassettes (from 6000 – 8000 bp). Another reason why the co-transformation did not work could be that it is recommended to use a selected product with long up- and downstream homologous regions. Up to a size of 3000 bp the co-transformation efficiency increases with the size of the homologous region 

<A href="https://www.ncbi.nlm.nih.gov/pubmed/28571309"><ABBR title="Dalia, T. N., C. A. Hayes, S. Stolyar, C. J. Marx, J. B. McKinlay and A. B. Dalia (2017). Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens. ACS Synth Biol 6(9): 1650-1655.">(Hayes <I>et al.</I> 2017)</ABBR></A>

. We used fragments with 1000 bp up- and downstream- homology because we were never able to fuse the 3000 bp up and downstream fragments to the chloramphenicol resistance cassette but still tried to perform co-transformation although there was a small chance to get a clone with the integration of the desired modifications.
For future work, fragments that are difficult to fuse to each other could be created in a plasmid based manner and introduced into the <I>V. natriegens</I> cells by conjugation 

<A href="https://www.ncbi.nlm.nih.gov/pubmed/27571549"><ABBR title="Weinstock, M. T., E. D. Hesek, C. M. Wilson and D. G. Gibson (2016). Vibrio natriegens as a fast-growing host for molecular biology. Nat Methods13(10): 849-851." s="">(Hesek <I>et al.</I> 2016)</ABBR></A>
.</P><P>Another promising solution for the integration of the T7 system is the utilization of the Flp/<I>frt</I> system for Flp mediated insertion. We could already demonstrate, that genomic excisions by FLP recombinase are possible in <I>V. natriegens</I>. The next step would be to amplify the T7 cassette with FRT site overhangs and to use this fragment for integration at the locus where <I>dns</I> was deleted by FLP mediated excision of the chloramphenicol resistance cassette, leaving a <I>frt</I> site behind. Unfortunately, this could not be tested due to the lack of time.</P>

Nevertheless, we were able to show, that <I>V. natriegens</I> is suitable for the production of proteins because we could successfully overexpress and purify proteins using <I>tcr</I> and T5 promoter based systems. See results of the Metabolic Engineering sub-project.


</DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">VibriInteract</DIV><DIV class="content"><DIV><H3><I>V. natriegens</I> is a suited chassis to test protein interactions based on B2H</H3>
The first step, after the theoretical work for establishing VibriInteract, was to check if <I>V. natriegens</I> ATCC 14048 is suitable for use in the conventional B2H system. For this, we plated wildtype cells on different agar plates. On LBv2 agar plates supplemented with 0.5 mM IPTG and 40 µg/ml Xgal, we could observe growth but no blue colour of colonies. This led us to thinking that the <I>lac</I> operon is probably not functional or that the β-galactosidase encoded by <I>lacZ</I>, which is needed for blue white screening  is not functional. Later on, we confirmed that lacZ of <I>V. natriegens</I> lacks amino acids in the midsection. Hereby, we learned that we can’t use blue-white screening for our interaction studies. On M63 minimal medium supplemented with maltose we could observe growth after less than a day.  Thus, we learned that our wildtype strain is able to utilize maltose. We got the same result when plating V. natriegens in MacConkey agar supplemented with maltose. On those plates we could observe a red colour of colonies which means that acidification takes place which led us to thinking that cells utilized maltose with acids as byproduct. This means that <I>V. natriegens</I> is a suited chassis to test protein interactions based on B2H. 
<FIGURE style="display:block; margin:0 auto 0 auto; width: 70%;"><FIGCAPTION><B>Figure 1:</B>Wildtype <I>V. natriegens</I> cells plated on (Left) LBv2 agar plates supplemented with Xgal and IPTG, (Middle)MacConkey agar plates supplemented with 1 % maltose and (Right) M63 minimal medium supplemented with 0,2 % maltose. Cells were incubated for one day and imagined by scanning.
</FIGCAPTION></FIGURE></DIV><DIV style="display_block;"><FIGURE style="display:block; float:left; width: 30%;" align="center"><FIGCAPTION><B>Figure 2:</B>1 %- Agarose gels of test PCRs for integration clones amplifying 1) The complete ΔcyaA deletion casette, 2) the chloramphenicol resistance cassette, 3) upstream cyaA + chloramphenicol resistance cassette, 4) chloramphenicol resistance cassette + downstream cyaA  . 2-log DNA ladder is used as marker on agarose gels.
</FIGCAPTION></FIGURE><H3>The adenylate cyclase of <I>V. natriegens</I> was deleted successfully</H3>
Next, we established a <I>ΔcyaA</I> deletion strain. Therefore, we created insertion cassettes as described. <A href="https://2018.igem.org/Team:Marburg/Design target=" blank="">Learn more about the design of our deletion cassettes following this link!</A><I>V. natriegens</I> could successfully be transformed with the deletion cassette using natural transformation. <A href="https://2018.igem.org/Team:Marburg/Description target=" blank="">Learn more about natural transformation following this link!</A> We did test PCRs to verify our integration using primers inside and outside of the cassette. Primers outside of the cassette should show shorter fragments in our integration clone compared to wildtype (cyA::CmR: ~2300bp; wildtype: 3900bp). Primers inside of the fragment should only show fragments if the cassette was integrated into the genome (CmR: ~940bp; cyA-upstream-CmR: ~1600bp; CmR-cyA-downstream: ~1600bp). When checked on an agarose gel, we could verify that our deletion cassette was inserted into the genome of <I>V. natriegens</I> (Figure 2).<FIGURE style="display:block; float:right; width: 40%;" align="center"><FIGCAPTION><B>Figure 3:</B>1 %- Agarose gels of test PCRs for loop out of the chloramphenicol resistance cassette amplifying 1) The complete ΔcyaA deletion casette, 2) the chloramphenicol resistance cassette, 3) upstream cyaA + chloramphenicol resistance cassette, 4) chloramphenicol resistance cassette + downstream cyaA  . 2-log DNA ladder is used as marker on agarose gels.
</FIGCAPTION></FIGURE><P>Then, we had to loop out the resistance cassette. The correct clones were transformed with a level 1 plasmid designed by ourselves carrying the gene encoding the Flp recombinase which recognizes the frt site. Transformation clones were tested via test PCR to examine if they carry the <I>flp</I> recombinase (not shown here). Correct clones were cultivated and streaked onto plates. Of those plates, single colonies were restreaked onto replica plates supplemented with and without chloramphenicol. 11 out of 100 clones in total showed sensitivity for chloramphenicol. Clones only growing on the plate without chloramphenicol were tested via test PCR. For this purpose, we used the same primers used for verification of insertion clones. Wildtype and our insertion mutant should show the same fragment sizes mentioned above. If the second homologous recombination was successful and the clone no longer carries a chloramphenicol resistance, only primers outside of the deletion cassette should result in a fragment (<I>ΔcyA</I>: ~1440 bp) (Figure 3). We could successfully get rid of the chloramphenicol resistance in every clone tested. With this, we were a step closer to establishing <B>VibriInteract</B>. </P><FIGURE style="display:block; float:left; width: 25%;" align="center"><FIGCAPTION><B>Figure 4:</B>1 %- Agarose gels of test PCR amplifying <I>tfoX</I> . 2-log DNA ladder is used as marker on agarose gels.
</FIGCAPTION></FIGURE>
The last step, to achieve a functional strain was to cure <I>ΔcyA</I> of the plasmid pMMB-<I>tfoX</I> carrying <I>tfoX</I> which carries a carbenicillin resistance. After cultivation in LBv2 supplemented with 1 mM IPTG, cells were streaked onto agar without any antibiotics. Single colonies were restreaked on replica plates supplemented with and without Carbenicillin. One clone of 50 clones in total showed sensitivity to carbenicillin. Clones that only grew on plates without antibiotic, were tested via test PCR amplifying <I>tfoX</I> (tfoX: ~650bp). When checked on an agarose gel, we could verify that <I>ΔcyA</I> no longer carries pMMB-<I>tfoX</I>. Finally, we had established <B>VibriInteract</B> (Figure 4).
</DIV><DIV style="display: block;"><H3>VibriInteract can be used for complementation in protein interaction studies based on B2H</H3>
After establishing <B>VibriInteract</B>, we confirmed that it is no longer able to utilize maltose as a carbon source. We plated VibriInteract (ΔcyA) together with wildtype as a control onto MacConkey agar plates supplemented with maltose. We expected VibriInteract to show  white colonies, since it should no longer be able to utilize maltose. Unfortunately, we observed red colonies on MacConkey plates (Figure 5, top). This suggests that some sort of acidification occured during the fermentation of a substrate. <I>V. natriegens</I> was shown to be able to form acids by mixed acid fermentation and to especially form formate and acetate under aerobic conditions. Under anaerobic conditions, cultures form mainly formate. 

<A href="https://www.ncbi.nlm.nih.gov/pubmed/28887417" target="blank"><ABBR title="Hoffart E1, Grenz S1, Lange J1, Nitschel R1, Müller F1, Schwentner A1, Feith A1, Lenfers-Lücker M1, Takors R1, Blombach B2. (2017) High substrate uptake rates empower Vibrio natriegens as production host for industrial biotechnology. Appl Environ Microbiol. 2017 01614-17">(Hoffart <I>et. al</I>, 2017)</ABBR></A>

. There are three substrates present in the medium that could be responsible for the outcome of this experiment. The first probable substrate is lactose which is an ingredient of the ready made MacConkey agar. Although we hypothesized,based on bioinformatic analysis and its missing ability to cleave Xgal, that the <I>lac</I> operon is not functional in <I>V. natriegens</I>, it could still be working. Maybe the  β-galactosidase cannot cleave Xgal but could still degrade lactose. This could be tested by preparing MacConkey agar without lactose to check if colonies still turn red. The second feasible substrate is peptone which consists of peptides and aminoacids. The last one is maltose, which could still be somehow degraded by <I>V. natriegens</I>. This is not likely to have occured because fortunately, we also plated <B>VibriInteract</B> together with wildtype as a control onto M63 minimal medium supplemented with maltose. On those plates, we could observe growth for wildtype but nearly no growth for <B>VibriInteract</B> (Figure 5, bottom). Our deletion was successful and the strain can no longer utilize maltose successfully. There was still some growth of the strain on minimal medium because absence of adenylate cyclase and consequently absence of cAMP does not lead to completely abolished expression of the regulated genes but to a 50-fold reduction in gene expression. Thus, we conclude that we can use M63 minimal medium supplemented with maltose to test for complementation of adenylate cyclase to perform protein interaction studies based on B2H. By this, conditions for the first test of a Vibrio Two Hybrid (V2H) were all set.<FIGURE style="display:block; margin:0 auto 0 auto; width: 50%;"><FIGCAPTION><B>Figure 5:</B>Wildtype and the ΔcyA plated onto (left)MacConkey agar plates supplemented with 1 % maltose and (Right) M63 minimal medium supplemented with 0,2 % maltose. Cells were incubated for one day and imagined by scanning.
</FIGCAPTION></FIGURE></DIV><DIV style="display: block;"><FIGURE style="float:left; width: 40%;" align="center"><FIGCAPTION><B>Figure 6:</B>Spot Assay of interaction of negative and positive control plasmids on M63 minimal medium supplemented with 0,2 % maltose.
</FIGCAPTION></FIGURE><H3>Plasmids used for B2H assay are applicable in V2H</H3>
We then certified that pUT18 and pKT25 plasmids used in B2H assays also work in <I>V. natriegens</I>. For this purpose, we transformed <B>VibriInteract</B> with plasmids of the commercially sold Euromedex B2H kit. As negative control, cotransformation of cells with the bait plasmids pKT25 and pUT18C were performed. As positive control, cotransformation of cells with pKT25-Zip and pUT18C-Zip plasmids were performed. Transformation clones were restreaked and later on inoculated in LBv2 supplemented with corresponding antibiotics. Those cultures were spotted onto M63 minimal medium supplemented with maltose. Eight clones of each transformation were tested. Six out of eight clones containing negative control plasmids showed little growth while two clones did not show any growth (Figure 6, left). Seven out of eight clones containing positive control plasmids showed normal growth while one clone showed very little growth (Figure 6, right). In general, we see a clear difference between the positive and the negative control. The positive control has an increased growth compared to the negative control. Consequently, we can use plasmids of the B2H assay in our V2H assay. Amazingly, spotted cultures could be analyzed after only 6 hours of incubation at 37 °C. For B2H assays in the <I>E. coli</I> strain BTH101 it is recommended to incubate plates for four  to even eight days! 
</DIV><H3>VibriInteract grows faster than BTH101</H3>
For further characterization we did a growth experiment to compare the doubling time of <B>VibriInteract</B> to the doubling time of our wildtype strain and to the doubling time of BTH101, the E. coli strain commonly used for B2H assays. Therefore, we diluted overnight night cultures of the three strains to an ΔOD<SUB>600nm</SUB> of 0.05 in LBv2 medium for V. natriegens strains or in LB medium for BTH101. The baffled flasks were incubated shaking at 37 °C.  In general, we could observe that VibriInteract has a slighty slower growth rate with a doubling time of approximately 18 min compared to the doubling time of approximately 16 min of our wildtype strain in this assay. But when comparing the doubling time of approximately 18 min to the doubling time of the <I>E. coli</I> strain BTH101 of approximately 25 min, we could verify that <B>VibriInteract</B> still has a faster growth rate. VibriInteract did not reach an OD similar to wildtype in this experiment. This could be due to the low level  expression of catabolic enzymes of mainly fermenting processes which are regulated by cAMP. The final OD would’ve been important for <B>VibriExpress</B> but for <B>VibriInteract</B> the density of a culture only plays a minor role since it is not a producer strain.
To conclude, VibriInteract has a higher growth rate than BTH101 which is the commonly used strain for Bacterial Two-Hybrid assays.
 <FIGURE style="display:block; margin:0 auto 0 auto; width: 50%;"><FIGCAPTION><B>Figure 7:</B>Growth curve of VibriInteract in comparison to V. natriegens wildtype and the E. coli strain BTH101. On the x axis time is shown on the y-axis theOD600nm is shown.
</FIGCAPTION></FIGURE><H3>VibriInteract shows modified cell morphology</H3>
We could observe a slower growth rate of VibriInteract in comparison to wildtype. Thus,  we decided to analyze cells concerning their cell morphology since defects in growth can often be due to defects general cell growth, inefficient cell division etc which can be examined using microscopy. For this purpose microscopic pictures of VibriInteract cells and wildtype cells were taken. Interestingly, what we could observe are what looks like chains of cells next to normal looking single cells. Those chains weren’t made of single cells but rather cells that could not complete cell division completely. In our analysis we could also observe such cells for wildtype but in a much lower number. In earlier experiments, we discovered that cells start to expand upon provision of fresh medium but without undergoing cell division using flow cytometry (See Part Collection results). In this experiment these chains of cells would have been defined as one event instead of several. In contrast to the mentioned experiment, cells of VibriInteract imagined here were taken from exponentially growing cultures instead of freshly inoculated cultures. cAMP is not only a key regulator of many catabolic processes but is also involved other major regulatory processes. One these processes is cell division regulation. It was suggested before that cAMP plays a role in regulating a mediator protein between PBP2 and the separation apparatus in <I>E. coli</I><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC210606/" target="blank"><ABBR title="D'Ari, R., A. Jaffe, P. Bouloc &amp; A. Robin, (1988) Cyclic AMP and cell division in Escherichia coli. Journal of bacteriology 170: 65-70.">(D'Ari <I>et al.</I>, 1988)</ABBR></A>

. V. natriegens cells could also be affected in a similar manner. This would explain the phenotype of VibriInteract. Although cells show this abnormal phenotype, plasmids of the B2H assay were functional in VibriInteract and cultures exhibited a sufficient growth rate.
<FIGURE style="display:block; margin:0 auto 0 auto; width: 50%;"><FIGCAPTION><B>Figure 8:</B>Microscopic analysis of VibriInteract compared to wildtype. Pictures show representative cells of each strain in exponential growth phase. Cells were imagined on 1 % agarose buffered with TPM and 0.2 % LBv2. Scale bar is 5 µm.
</FIGCAPTION></FIGURE></DIV></DIV></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">CRISPR/Cas9</DIV><DIV class="content"><P>In our project, we provided a first indication of a successful mutation via CRISPR/Cas9.</P><P>


As outlined in the description of this experiment, we assembled a LVL2 plasmid possessing the Cas9 CDS as well as the gRNA with the dns spacer. In comparison with the cotransformation of the Cas9-Plasmid and the gRNA-plasmid, the transformation of one plasmid has a higher efficiency, especially when transformed in combination with the donor nucleooligotides. </P><P>


For the knock-out of the dns in <I>V. natriegens</I>, we cotransformed the LVL2 plasmid once with the suitable oligotides of the lagging strand, once with those of the leading strand and as a control once in the absence of the oligos. Figure 1 shows the plates of these transformations.
The control lacking the donor DNA showed no bacterial growth. This indicates that Cas9 is able to cleave the DNA in <I>V. natriegens</I> when coexpressed with gRNA, targeting the chromosome. This leads to the double strand break (DSB) of the DNA and cell death <A href=" https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3748948/pdf/nihms436755.pdf "><ABBR title=" CRISPR-assisted editing of bacterial genomes.">(Jiang <I>et al.</I>,2013)</ABBR></A>. 

The product of the transformation in which the donor nucleooligonucleotides were provided presented a strong bacterial growth on the plates. As was shown, the LVL2 plasmid is lethal to <I>V. natriegens</I>. The addition of donor nucleooligotides facilitates the repair of the DSB through homologous recombination, thereby incorporating the desired mutation into the strand <A href=" http://www.jbc.org/content/early/2018/03/29/jbc.TM118.000372.full.pdf"><ABBR title="Homologous recombination and the repair of DNA double-strand breaks.">(Wright <I>et al.</I>,2013).</ABBR></A><FIGURE style="display:block; margin:0 auto 0 auto; width:600px;"><FIGCAPTION><B> Figure 1: Crispr/Cas in <I>V. natriegens</I>. </B> The picture shows colonies containing LVL2-Cas9-gRNA plasmid and A: no donor templates (no bacterial growth), B: donor templates of the leading strand and C: donor templates of the lagging strand (both bacterial lawn).</FIGCAPTION></FIGURE></P><P>


This first experiment suggests that our CRIPSPR/Cas9 is able to generate mutations.

</P></DIV></DIV></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand"><DIV class="btn_e_title">Results Metabolic Engineering</DIV><DIV class="btn_e_desc">We achieved to characterize different <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN>s and <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> in <I>V. natriegens</I> and showed <B>functionality of our pathway</B> via coupled enzyme assays and mass spectrometry. Additionaly we tested a <B>model-predicted pathway</B> variant and built a <B>pathway library</B> with 390 further variants.</DIV></DIV><DIV class="content"><P>
	Our major goal was to accelerate the workflow of metabolic engineering. For that we tested enzymes from different organisms and could find the most promising candidates for our pathway. Additionaly we detected 3-hydroxypropionate via mass spectrometry by combining cell extracts of <I>V. natriegens</I> expressing <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> or <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN>. Beside that we constructed a rationally designed pathway and tested it in vitro. We were able to verify its functionality in <I>V. natriegens</I> by applying an enzyme activity assay. Furthermore, we successfully created a pathway library containing 390 pathway variants and cloned a biosensor for high throughput screening of <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN>.
	</P><DIV class="collapsible"><DIV class="btn_expand">Enzyme activities of <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN> and <DFN data-info="Malonyl-CoA reductase">Mcr</DFN></DIV><DIV class="content"><H3>Enzyme activities of Acc from different organisms</H3><P><FIGURE style="width: 35%; float: right;"><FIGCAPTION><B>
				Figure 1: Enzyme activity of cell extracts from <I>V. natriegens</I> with different <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN>s
			</B>The bar chart shows, that <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN> + BirA from <I>S. elongatus</I> has the highest specific activity with 164 mU/mg, while <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN> + BirA from <I>E. coli</I> has an activity of 14 mU/mg and wild type cells have no measurable activity.
		</FIGCAPTION></FIGURE>
	Our overall goal was to accelerate metabolic engineering by developing a workflow for high throughput pathway assembly and pathway screening. The first step for that is to find the best enzyme candidates. We tested the native acetyl-CoA carboxylase (Acc) from <I>V. natriegens</I> as well as the <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN> from <I>E. coli</I> and <I>S. elongatus</I> together with their activating biotin ligase BirA. In the Choice of Enzymes section of Metabolic Engineering we explained why we chose these enzyme candidates (<A href="https://2018.igem.org/Team:Marburg/Design"><ABBR title="link to ori characterization of the part collection"> link to ori characterization of the part collection</ABBR></A>). We cultivated <I>V. natriegens</I> expressing these enzyme variants, harvested them and used the French press to get cell extracts. The cell extracts were used for an enzyme activity assay to see if the <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN>s are functional and which of them is the most active one (Figure 1). 
	For the native <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN> we got nearly no measurable activity no matter which amounts of cell extract or acetyl-CoA we added. This result correlates with our expectation that the native <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN> is less active than the other enzymes, because it is located on the chromosome and has therefore just one copy.  Moreover, the  <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN> is strictly regulated and only active when fatty acid biosynthesis is required. Since we harvested the cells in the stationary phase, we assume that the enzyme wasn’t in its active state. The <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN> from <I>E. coli</I> showed a higher activity but it was beaten by the <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN> from <I>S. elongatus</I>, that has a specific activity of 164 mU/mg. We think this high activity is due to the phylogenetic distance, that makes different regulation mechanisms more likely. Therefore, the native regulation mechanisms don’t influence activity of <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN> from <I>S. elongatus</I>. Furthermore, the need of high amounts of fatty acids for thylakoid formation makes an active <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN> crucial.
</P><FIGURE style="width: 50%; float: left;"><FIGCAPTION><B>
				Figure 2: NADPH-dependent absorption at 365nm
			</B>The graph shows, that the absorption of NADPH at 365nm is constant. The initial mixture contained MOPS Buffer, MgCl2, ATP, NADPH and KHCO3. After addition of cell extract the absorption stays constant, but after addition of acetyl-CoA as start substrate, the absorption drops. The dashed line indicate the slopes that were used for calculating the specific activity. The two peaks are artifacts, induced by pipetting of reagents into the cuvette.
		</FIGCAPTION></FIGURE><P>
	The specific activity was calculated via determination of absorption slopes (Fig 2). To ensure that the decrease of absorption is definitely due to malonyl-CoA addition and not addition of cell extract, we first added cell extract and saw no change of absorption. However, after malonyl-CoA addition the absorption instantly dropped. Therefore, we suppose that the measured NADPH consumption correlates with <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN> activity. Unfortunately, we haven’t measured <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN> activity from <I>C. glutamicum</I> yet because we built the corresponding plasmid library just recently and had no time for biochemical characterization. However, we measured a rationally designed construct bearing <DFN data-info="Acetyl-CoA Carboxylase subunit BC"><I>accBC</I></DFN>, <DFN data-info="Acetyl-CoA Carboxylase subunit D"><I>accD</I></DFN> and <DFN data-info="Biotin Ligase"><I>birA</I></DFN> from <I>C. glutamicum</I> and <DFN data-info="Malonyl-CoA reductase"><I>mcr</I></DFN> from <I>C. aurantiacus</I> and were able to see activity (Fig. 6).
</P><H3><DFN data-info="Malonyl-CoA reductase">Mcr</DFN> is soluble and functional in <I>V. natriegens</I></H3><P>
	The second enzyme in our pathway is the malonyl-CoA reductase (Mcr). The enzyme comes from <I>C. aurantiacus</I>, a thermophilic organism, therefore we wanted to test, if the <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> is correctly expressed and functional in <I>V. natriegens</I>. For that we overexpressed it, lysed the cells via French press and made an SDS-PAGE as well as an enzyme activity assay.
</P><P>
	On the SDS-PAGE we compared cell extract from wild type cells with extract from <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> producing cells. Figure 3 shows, that the band for <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> with a mass of 131 kDa is only visible in <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> producing cells. This result confirms, that the protein is soluble and that <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> production works.
</P><FIGURE style="width: 50%; float: right;"><FIGCAPTION><B>
				Figure 3: SDS-PAGE of cell extract from <I>V. natriegens</I> with and without <DFN data-info="Malonyl-CoA reductase">Mcr</DFN></B>The gel shows the overall protein composition in cell extract of <I>V. natriegens</I> with and without <DFN data-info="Malonyl-CoA reductase">Mcr</DFN>. The expected fragment for <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> at 131 kDa is clearly visible only in the extract of <DFN data-info="Malonyl-CoA reductase"><I>mcr</I></DFN> expressing cells, indicating that the protein is soluble.
			</FIGCAPTION></FIGURE><P>
	In the enzyme activity assay, we also used cell extract from wild type cells and <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> producing cells. Figure 4 shows a significant activity with extract of <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> producing cells. The specific activity is 48 mU/mg. Therefore, we know now, that acetyl-CoA carboxylase, as well as malonyl-CoA reductase are both functional in our strain. The model predicted a higher activity of <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> compared with <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN>, but our measurements suggest the opposite. However, it isn’t possible to compare these data directly, because the calculations of specific activities depend on the protein amount and we used extract instead of purified proteins. If in the <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN> extract the percentage of <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN> is higher than the <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> percentage in the <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> extract, we got incomparable results. 

	</P><DIV style="display: inline-block;"><FIGURE class="mobile_w_full" style="width: 50%; float: right;"><FIGCAPTION style="float: right;width: 50%;"><B>
				Figure 4: Enzyme activity of <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> from <I>C. aurantiacus</I></B>The bar chart shows, that cell extract of <I>V. natriegens</I> with <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> has a specific activity of 48 mU/mg while wild type cell extract only has a specific activity of under 2 mU/mg, thus showing the functionality of <DFN data-info="Malonyl-CoA reductase">Mcr</DFN>.
			</FIGCAPTION></FIGURE><DIV class="mobile_w_full" style="width: 50%;float: left;">
	Therefore, our measurements are just approximations and the real activities are even higher. The next step to further validate <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> production by <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> is to use our biosensors to screen for it. We unfortunately weren’t able to use the biosensors for <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> screening in our in vitro assays because the construction of the sensor finished just recently and before using the sensor, it must be calibrated and a protocol for in vitro sensor screening has to be prepared. And to use the sensor in vivo, the <DFN data-info="Malonyl-CoA reductase"><I>mcr</I></DFN> gene first has to be cloned into another vector to be compatible with the sensor plasmid.</DIV></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Mass spectrometry of 3-hydroxypropionic acid</DIV><DIV class="content"><P>
	So far, we could show, that <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN> and <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> are functional in <I>V. natriegens</I> and have sufficient activities. Next, we wanted to see if these enzymes can be combined to produce <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN>. For that we combined cell extract from <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> producing cells with extract from <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN> producing cells and incubated the mixture with 400µM NADPH, 3.33mM ATP, 2mM Acetyl-CoA, 6.67mM MgCl2 and 50mM KHCO3 to let the reaction happen in vitro. To detect 3-hydroxypropionate we used triple quadrupole mass spectrometry (Fig. 5). To simultaneously test the hypothesis of our (<A href="https://2018.igem.org/Team:Marburg/Model"><ABBR title="metabolic model">metabolic model</ABBR></A>), that <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN> is the rate limiting step in the route, we used two different ratios of <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN>/<DFN data-info="Malonyl-CoA reductase">Mcr</DFN>. For each measurement we used the same amount of <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> extract (6µl in a total volume of 50µl) and varied the amount of <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN> extract (1.5µl/0.36µl). As a positive control we also used purified<DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN> (130µg/ml). We incubated the mixture for 30 minutes at 37°C, centrifuged it and used the supernatant for derivatization with 3-nitrophenyl-hydrazine
	<A href="https://doi.org/10.1002/elps.201200601"><ABBR title="Han, J., Gagnon, S., Eckle, T., Borchers, C.H., 2013. Metabolomic analysis of key central carbon metabolism carboxylic acids as their 3-nitrophenylhydrazones by UPLC/ESI-MS. Electrophoresis 34, 2891–2900.">(Han <I>et al.</I> 2013)</ABBR></A>. To be able to quantify <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> later, we used samples with and without supplementation of a defined amount of <DFN data-info="3-hydroxypropionic acid">3HPA</DFN>. For <DFN data-info="Liquid Chromatography-Mass Spectrometry">LC-MS</DFN> we used in the first quadrupole a m/z filter of 224, then split the compound in the second quadrupole with a collision energy of 30V and for the third quadrupole we used a m/z filter of 194. All measurements were made with negative polarity. One important aspect is that we have to distinguish between <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> and the structural isomer lactate, which is also produced by <I>V. natriegens</I><A href="https://doi.org/10.1128/AEM.01614-17"><ABBR title="Hoffart, E., Grenz, S., Lange, J., Nitschel, R., Müller, F., Schwentner, A., Feith, A., Lenfers-Lücker, M., Takors, R., Blombach, B., 2017. High substrate uptake rates empower Vibrio natriegens as production host for industrial biotechnology. Appl. Environ. Microbiol. 83.">(Hoffart <I>et al.</I> 2017)</ABBR></A>. For the differentiation we added cell extract of <I>V. natriegens</I> wild type with 200µM <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> or 200µM lactate respectively. We came to the result that a good differentiation is possible by measuring the retention time, which was in our hands 2.947 minutes for <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> and 3.089 minutes for lactate.
</P><FIGURE><DIV class="imageContainer2x2"><DIV>a</DIV><DIV>b</DIV><DIV>c</DIV><DIV>d</DIV></DIV><FIGCAPTION><B>
        		Figure 5: <DFN data-info="High Performancy Liquid Chromatography-Mass Spectrometry">HPLC-MS</DFN> to differentiate between <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN> and lactate and to quantify 3HPA in cell extract
        	</B>All measurements were performed in a triple quadrupole mass spectrometer using the mass filters 224→194, a collision energy of 30V and negative polarity. We differentiated between <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> and lactate (<B>a</B>) and measured different amounts of cell extract, 1.5µl per 50µl reaction (<B>b</B>), 0.36µl (<B>c</B>) or purified propionyl-CoA carboxylase with D107I mutation, making the enzyme specific for acetyl-CoA carboxylation as a positive control (<B>d</B>).
        </FIGCAPTION></FIGURE><P>
	Lactate has a longer retention time than <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> enabling a clear distinction. In the measurements of our cell extract samples, we could clearly identify 3HPA and see a peak at a retention time of 2.947 minutes. By comparing the ion counts of our extracts with the ion counts of our extracts with supplemented 3HPA we can calculate the amount of produced 3HPA, that can be seen in Table 1. We got 3HPA concentrations ranging from 92.48 to 453.34 µM. The fact that the first sample with 4-fold higher <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN> extract amounts also has 4-fold higher 3HPA amounts shows the accuracy of the measurement but it also led us to the assumption, that the rate-limiting step in the reaction was the <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN> abundance and that the 2mM Acetyl-CoA, that were injected at the beginning, weren’t consumed at all. If it would be consumed, the number of enzymes wouldn’t have played a role. Therefore, we recommend to repeat the assay and increase the incubation time at 37°C from 30 minutes to 2 hours or to photometrically determine the time point, when all the substrate is consumed. Nevertheless, we achieved to detect and quantify 3HPA in vitro with cell extracts of <I>V. natriegens</I>. Additionally, we developed a measuring protocol to differentiate between 3HPA and lactate. The next step will now be, to cotransform the plasmids for both enzymes into <I>V. natriegens</I> and determine in vivo production of 3HP.
</P><FIGURE><FIGCAPTION><B>Table 1</B><DFN data-info="3-hydroxypropionic acid">3HPA</DFN> concentrations measured by <DFN data-info="Liquid Chromatography-Mass Spectrometry">LC-MS</DFN></FIGCAPTION><TABLE style="text-align: left;"><TBODY><TR><TH>Sample</TH><TH><DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN> concentration [µM]</TH></TR><TR><TD>1.5µl Acc extract</TD><TD>453.34</TD></TR><TR><TD>0.36µl Acc extract</TD><TD>92.48</TD></TR><TR><TD>Purified enzyme</TD><TD>269.67</TD></TR></TBODY></TABLE></FIGURE></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Rational plasmid design and pathway library</DIV><DIV class="content"><H3>Construction and testing of our model-predicted pathway plasmid</H3><P>
	
	One of our approaches was the construction of a rationally designed plasmid, bearing <DFN data-info="Acetyl-CoA Carboxylase subunit BC"><I>accBC</I></DFN>, <DFN data-info="Acetyl-CoA Carboxylase subunit D"><I>accD</I></DFN>, <I>birA</I> and <DFN data-info="Malonyl-CoA reductase"><I>mcr</I></DFN> with optimized expression levels, enabling an optimal flux. For that we implemented a model for a two-scale metabolic flux analysis, which can be seen in the Model section (<A href="https://2018.igem.org/Team:Marburg/Model"><ABBR title="link to model section">link to model section</ABBR></A>). For that we first ordered the coding regions for these genes, codon optimized for <I>V. natriegens</I> and cloned them via Golden Gate into the level 0 backbone of the Marburg Collection. Then we used the toolbox to build level 1 plasmids, each of them bearing the transcriptional units, and finally combined them to a functional level 2 vector harbouring the whole pathway. Which parts we used can be seen in the Rational Design section of Metabolic Engineering (<A href="https://2018.igem.org/Team:Marburg/Model"><ABBR title="link to model section">link to design section</ABBR></A>). To investigate pathway performance, we transformed the plasmid into <I>V. natriegens</I>, lysed the cells and used them for an activity assay, starting with Acetyl-CoA (Figure 6). The data show that cells  that contain the pathway plasmid consume NADPH a lot faster, indicating that the pathway is working and 3-hydroxypropionate is produced. The specific activity is 0.025 mU/mg, what is low in comparison to the activities of our single enzyme measurements. We think a major reason for that is the quality of the cell extract. After preparing it we stored it at -20° for about two weeks because we had many other experiments simultaneously  we primarily had to focus on. When cell extract is not fresh, its activity can drop rapidly. Nevertheless, the assay showed that there is activity, so our pathway works in <I>V. natriegens</I>. The next step to get better results would be to repeat the assay with fresh extract. It would also be useful to let the reaction happen and then use <DFN data-info="Liquid Chromatography-Mass Spectrometry">LC-MS</DFN> or our malonyl-CoA sensor to screen for malonyl-CoA.  If there is detectable malonyl-CoA, then the pathway flow is not optimal, because malonyl-CoA should not accumulate.
</P><H3>Construction of the pathway library</H3><FIGURE style="width: 30%; float: right;"><FIGCAPTION><B>
				Figure 6: Enzyme activity for cell extract of <I>V. natriegens</I> with and without the rationally designed pathway
			</B>The bar plot shows the specific activities of cell extract from <I>V. natriegens</I> with and without our rationally designed pathway plasmid. The result is that wild type cells have an activity of 0.003 mU/mg, while extract of cells performing the pathway has a specific activity of 0.025, showing that the pathway works and produces 3-hydroxypropionate.
		</FIGCAPTION></FIGURE><P>
	So far, we showed, that the <DFN data-info="Acetyl-CoA Carboxylase">Acc</DFN>, as well as the <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> are both soluble and functional in <I>V. natriegens</I> and we were able to prove <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN> production in cell extracts. Furthermore, we built a rationally designed pathway and implemented it into <I>V. natriegens</I>. However, our aim was not only to establish one pathway but rather a whole library of pathway variations. As mentioned in the Pathway Library section of Metabolic Engineering (<A href="https://2018.igem.org/Team:Marburg/Design"><ABBR title="link to design section">link to design section</ABBR></A>), we planned to construct 1446 pathway variants. Since the cloning of so many constructs is nearly unrealistic, we used the liquid transfer robot Echo 525 Liquid Handler from Labcyte to make the cloning. Although the robot works very fast and is able to mix all golden gate reactions in less than one hour, we wanted to start with a shorter number of reactions, to test if our strategy works. Nobody has ever tested golden gate cloning coupled with natural transformation in that robot. Therefore, we focused on one 96 well plate for each plasmid category. We actually planned and performed the cloning of all 24 constructs of category I, 96 constructs of category II, 96 constructs of category III, all 81 constructs of category IV, all 27 constructs of category V, all 81 constructs of category VI and all 81 constructs of category VII, yielding 486 constructs in total. For the cloning we prepared a 384 well source plate containing each part of the Marburg Collection. We then prepared a picking list for the robot and it started to transfer all 486 golden gate mixtures into 96 well plates. A more detailed description of the Echo Liquid Handler is specified in the Labautomation-section  [Link zu labautomation]. The overview about all plasmids is shown in the Pathway library section of Metabolic Engineering (<A href="https://2018.igem.org/Team:Marburg/Design"><ABBR title="link to design section">link to design section</ABBR></A>). After running the golden gate reaction in a thermocycler, we used natural transformation to transform the constructs into <I>V. natriegens</I>. For that we added Tfox-bearing cells to each well, performed the protocol for natural transformation, that is shown in the Methods section of Strain Engineering (<A href="https://2018.igem.org/Team:Marburg/Design"><ABBR title="link to design section"> link to design section</ABBR></A>) and plated them out. To our pleasure 390 plates showed colonies after transformation. Theoretically, the next step would be to make minipreps and check plasmid correctness, but according to our metabolic engineering workflow, we would prefer to directly use the biosensor to test how efficient the pathway variants are. At the end the lack of time hindered us from testing the pathway library, but our previous cloning approaches with the Marburg Collection had efficiencies of at least 80%, therefore we assume that at least 312 constructs out of these 390 are correct, which is still a high number and sufficient to continue with biosensor-guided product screening and directed evolution to optimize selected pathway variants in the future.
</P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Biosensors and Directed Evolution</DIV><DIV class="content"><H3>Biosensors</H3><P>
	In the first major experiment we tested induction of
	
		lux
	
	expression over a <DFN data-info="3-hydroxypropionic acid"> 3-HPA </DFN> gradient. When we first saw the plate reader data, we were very optimistic that we had successfully cloned a functional <DFN data-info="3-hydroxypropionic acid"> 3-HPA </DFN> sensor into
	
		V. natriegens.
	
	 Just from the luminescence values, a clear trend to more emission at medium concentrations was apparent.
</P><P>
	To explain this, we argued that a higher concentration of <DFN data-info="3-hydroxypropionic acid"> 3-HPA </DFN> caused so much stress, the cells died, and that lower concentrations were insufficient for induction. The difference between the uninduced and the most highly expressing sample was about six-fold. 

<FIGURE style="width: 45%; float: right;"><FIGCAPTION><B>Figure 7:</B> A) The concentration of <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> plotted versus the pH 
			B) The concentration of 3HPA plotted versus the luminescence
		</FIGCAPTION></FIGURE>



	 This was lower than the values reported by 


<A href="http://www.nature.com/articles/s41598-017-01850-w"><ABBR title="Characterisation of a 3-Hydroxypropionic Acid-Inducible System from Pseudomonas Putida for Orthogonal Gene Expression Control in Escherichia Coli and Cupriavidus Necator.">Hanko <I> et al. </I> 2018.</ABBR></A></P><P>
	What we also observed was a severe loss of vitality, materializing in form of significantly reduced growth which completely ceased at the two highest concentrations, 10 mM and 6.6 mM respectively. This would represent a significantly lower resilience to <DFN data-info="3-hydroxypropionic acid"> 3-HPA </DFN> than previously reported for
	
		E. coli
	<A href="http://www.ncbi.nlm.nih.gov/pubmed/26709549"><ABBR title="Development of a 3-Hydroxypropionate Resistant Escherichia Coli Strain">(Liu
	
		et al.
	
	 2016).</ABBR></A>


	 There, the threshold at which first detrimental effects could be seen was 200 mM. If we want to make industrial production of <DFN data-info="3-hydroxypropionic acid"> 3-HPA </DFN> in
	
		V. natriegens
	
	 a reality, a way to increase its resistance is indispensable. Fortunately, strategies to implement such increased resilience have already been shown to work in
	
		E. coli
	<A href="http://www.ncbi.nlm.nih.gov/pubmed/26709549"><ABBR title="Development of a 3-Hydroxypropionate Resistant Escherichia Coli Strain">(Liu
	
		et al.
	
	 2016).</ABBR></A>
 So, given the time, similar solutions for
	
		V. natriegens
	
	 are not hard to imagine.
</P><P>
	Several different factors could be responsible for the toxicity. At first, we thought that <DFN data-info="3-hydroxypropionic acid"> 3-HPA's </DFN> structural similarity to cellular substrates, like for instance lactate, could cause inhibition of enzymes acting on these substrates. The mechanism of toxicity for <DFN data-info="3-hydroxypropionic acid"> 3-HPA </DFN> is most likely its acidity. <DFN data-info="3-hydroxypropionic acid"> 3-HPA </DFN> is a weak acid with a pKs of 4.50, and in our basic research (<A href="https://static.igem.org/mediawiki/2018/3/3a/T--Marburg--pH_Tolerance_Assay.pdf"><ABBR title="link to protocol for pH tolerance assay">link to protocol for pH tolerance assay</ABBR></A>)  we showed that a low pH does indeed slow the growth of
	
		V. natriegens,
	
	or can even be lethal. This could be compensated in part by choosing an alkaline starting pH of the culture media. A step that is favored, as V. natriegens is less susceptible to a high pH up to 9.5.
</P><P>
	However, this does not explain the higher luminescence for medium concentrations, where growth does not appear to be impacted.
</P><P>
	When we took a closer look at our control, the
	
		lux
	
	 operon driven by J23107 (
	<A href="http://parts.igem.org/Part:BBa_J23107">
		http://parts.igem.org/Part:BBa_J23107
	</A>
	), we found the same pattern of dead cells at high concentrations, high <I>lux</I> expression in the middle and background expression at lower concentrations. J23107 is a constitutive promoter of medium strength, chosen to be close to the expression levels we saw from our uninduced sensor. Regrettably, this shows that our biosensor is not functional, but presents us with an interesting enigma to solve.
</P><P>
	It is plausible that <DFN data-info="3-hydroxypropionic acid"> 3-HPA </DFN>, which is very similar to some substrates for enzymes of the
	
		lux
	
	 operon, could in some way interact with said enzymes. Another explanation voiced by a team member is based on the fact that through division, cells dilute their cell content. This includes metabolites, <DFN data-info="desoxy nucleic acid"> DNA</DFN> and proteins. If the cells divide more slowly, they have more time to accumulate these. To accumulate the proteins responsible for
	
		lux
	
	 luminescence, which would result in a higher per OD luminescence. Our observations do not quite fit this, since we not only see a higher luminescence per OD, but a much higher luminescence overall. Also conceivable is, that resources that would normally go into cell growth are diverted into
	
		lux
	
	 expression.
</P><FIGURE style="width: 45%; float: left;"><FIGCAPTION><B>Figure 8:</B> Bar-chart with the luminescence on the y-, and the <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> concentration on the x-axis. 
		</FIGCAPTION></FIGURE><P>
	There are no sources to substantiate that claim, but we found another promising lead in a source on the behavior of
	
		lux
	
	 expression depending on the pH 

<A href="http://www.ncbi.nlm.nih.gov/pubmed/12676702"><ABBR title="Effect of Temperature, PH, and Initial Cell Number on LuxCDABE and Nah Gene Expression during Naphthalene and Salicylate Catabolism in the Bioreporter Organism Pseudomonas Putida">(Dorn
	
		et al.
	
	 2003).</ABBR></A>

	  There, it was shown that pH is a major influence in
	
		lux
	
	 expression. They observed the peak luminescence at a slightly acidic pH of around 6. This is remarkably close to the values we measured for the concentrations of 1.9 mM to 2.9 mM, where we also observed the strongest luminescence. Therefore, when luminescence is influenced that strongly by pH, that explains why our constitutively expressed <I>lux</I> reporter showed the same pattern. Amazingly, this effect was already apparent only five minutes after induction.
</P><P>
	For the malonyl-CoA sensor, we had difficulties to clone the level 2 plasmids required for them to work. Once we solved this issue, testing may proceed.
</P><P>
	There are several other sensors for <DFN data-info="3-hydroxypropionic acid"> 3-HPA </DFN> as well as for malonyl-CoA as discussed in our 

<A href="https://2018.igem.org/Team:Marburg/Design"><ABBR title="This link leads to the design page">design section.</ABBR></A></P><P>
	One promising way to directly screen for productivity of our <DFN data-info="3-hydroxypropionic acid"> 3-HPA </DFN> pathway would be to co-express the <I>lux</I> operon under a strong constitutive promoter. Since the substrates for the luminescence reaction are part of the fatty acid synthesis pathway, and malonyl-CoA is at the root of this pathway, a higher conversion rate of malonyl-CoA to <DFN data-info="3-hydroxypropionic acid"> 3-HPA </DFN> should diminish luminescence output. This simple competitive assay could give a rudimentary indication of the <DFN data-info="malonyl-CoA reductase"> Mcr </DFN> activity.
</P><H3>Directed evolution</H3><P>
	After building a pathway library, transferring it into <I>V. natriegens</I> and establishing biosensors, the next step in our metabolic engineering workflow is to implement directed evolution for pathway optimization. So far, we could show that our enzymes are functional and <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN> is produced, but there are still many possibilities to optimize this metabolic pathway. Unfortunately, lack of time forced us to skip the introduction of evolution into our approach. Nevertheless, we planned the whole procedure so that everyone can start directly with the implementation. As already mentioned in the <A href="https://2018.igem.org/Team:Marburg/Design"><ABBR title="Design section of metabolic engineering">Design section of metabolic engineering</ABBR></A>, we followed two strategies. The first one is to re-feed <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> into the central metabolism and forcing the strain to perform this metabolic route. For that we decided to introduce genes for a propionyl-CoA synthase (Pcs), a propionyl-CoA carboxylase (Pcc), a methylmalonyl-CoA epimerase and a methylmalonyl-CoA mutase. Pcs converts 3HPA into propionyl-CoA, which then gets carboxylated by Pcc to form (S)-2-methylmalonyl-CoA. Methylmalonyl-CoA epimerase and methylmalonyl-CoA mutase finally catalyze the isomerization of (S)-2-methylmalonyl-CoA via (R)-2-methylmalonyl-CoA to succinyl-CoA. By deleting housekeeping genes for succinyl-CoA production, the only possibility for the cell to produce this compound is by using our pathway. Coupled with the fast growth of <I>V. natriegens</I>, cells will fast adapt to this route and naturally optimize it. To start with this strategy, the next step would be to design oligos for genomic deletion of succinyl-CoA production. Then MuGent can be used to delete these genes, as shown in the Methods section of Strain Engineering (<A href="https://2018.igem.org/Team:Marburg/Design"><ABBR title="link to design section"> link to design section</ABBR></A>). Afterwards, the genes for our bypass can be introduced and directed evolution is started. For our second strategy, where we use <DFN data-info="Fluorescence Activated Cell Sorting">FACS</DFN> and <DFN data-info="Multiplex Automated Genome Engineering">MAGE</DFN> to introduce genome modifications, we already looked for possible modifications, we could introduce via oligos. A table of that modifications is shown in the Further Improved section of Metabolic Engineering (<A href="https://2018.igem.org/Team:Marburg/Design"><ABBR title="link to design section"> link to design section</ABBR></A>). Here the next steps would be to compare the list of modified genes from other organisms with <I>V. natriegens</I> and design oligos for these modifications. After calibrating the biosensor one can start with <DFN data-info="Multiplex Automated Genome Engineering">MAGE</DFN>-directed optimization coupled with <DFN data-info="Fluorescence Activated Cell Sorting">FACS</DFN>-directed selection of promising cells.
</P></DIV></DIV></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand"><DIV class="btn_e_title">Sources </DIV></DIV><DIV class="content"><H3>References</H3>

Dorn, J.G., Frye, R.J., Maier, R.M., 2003. Effect of temperature, pH, and initial cell number on <I>luxCDABE</I> and <I>nah</I> gene expression during naphthalene and salicylate catabolism in the bioreporter organism <I>Pseudomonas putida</I> RB1353. Appl. Environ. Microbiol. 69, 2209-16

Han, J., Gagnon, S., Eckle, T., Borchers, C.H., 2013. Metabolomic analysis of key central carbon metabolism carboxylic acids as their 3-nitrophenylhydrazones by UPLC/ESI-MS. Electrophoresis 34, 2891–2900. 
https://doi.org/10.1002/elps.201200601

Hanko, E.K.R., Minton, N.P., Malys, N., 2017. Characterisation of a 3-hydroxypropionic acid-inducible system from <I>Pseudomonas putida</I> for orthogonal gene expression control in <I>Escherichia coli</I> and <I>Cupriavidus necator</I>
Hoffart, E., Grenz, S., Lange, J., Nitschel, R., Müller, F., Schwentner, A., Feith, A., Lenfers-Lücker, M., Takors, R., Blombach, B., 2017. High substrate uptake rates empower Vibrio natriegens as production host for industrial biotechnology. Appl. Environ. Microbiol. 83. https://doi.org/10.1128/AEM.01614-17

Liu, M., Han, X., Xian, M., Ding, Y., Liu, H., Zhao, G., 2016. Development of a 3-hydroxypropionate resistant <I>Escherichia coli</I> strain.

Blokesch, M., Schoolnik, G. K., The Extracellular Nuclease Dns and Its Role in Natural Transformation of Vibrio cholerae, JOURNAL OF BACTERIOLOGY, (2008), 7232–7240.

D'Ari, R., A. Jaffe, P. Bouloc &amp; A. Robin, (1988) Cyclic AMP and cell division in Escherichia coli. Journal of bacteriology 170: 65-70. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC210606/


Dalia TN, Hayes CA, Stolyar S, Marx CJ, McKinlay JB, Dalia AB. Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens. ACS Synth Biol. (2017) 15;6(9):1650-1655.
Hoffart E, Grenz S, Lange J, Nitschel R, Müller F, Schwentner A, Feith A, Lenfers-Lücker M, Takors R, Blombach B2. (2017) High substrate uptake rates empower Vibrio natriegens as production host for industrial biotechnology. Appl Environ Microbiol. (2017) 01614-17. https://www.ncbi.nlm.nih.gov/pubmed/28887417
Weinstock, M. T., E. D. Hesek, C. M. Wilson and D. G. Gibson (2016). &quot;Vibrio natriegens as a fast-growing host for molecular biology.&quot; Nat Methods 13(10): 849-851.

Zeaiter, Z., Mapelli F., Crotti, E., Borin, S. Methods for the genetic manipulation of marine bacteria. Electronic Journal of Biotechnology. (2018) 33: 17-28.<A href="https://static.igem.org/mediawiki/2018/1/1f/T--Marburg--VnatriegensSecondMetabolites.gb"><ABBR title="Secondary metabolites predicted with antiSMASH 3.0">Secondary metabolites</ABBR></A><A href="https://static.igem.org/mediawiki/2018/7/7b/T--Marburg--Vibrio_natriegens_Chr1.gb"><ABBR title="V. natriegens ATCC 14048 Chr. 1 Vibrigens assembly">Vibrio natriegens Chromosome 1</ABBR></A><A href="https://static.igem.org/mediawiki/2018/4/41/T--Marburg--Vibrio_natriegens_Chr2.gb"><ABBR title="V. natriegens ATCC 14048 Chr. 2 Vibrigens assembly">Vibrio natriegens Chromosome 2</ABBR></A><B>Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, et al. 2012.</B> SPAdes: A New Genome Assembly Algorithm and Its Applications to Single-Cell Sequencing. <I>Journal of Computational Biology</I><B>19</B>: 455–477.
					https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3342519/ (October 18, 2018)
					<B>Seemann T. 2014.</B> Prokka: rapid prokaryotic genome annotation. <I>Bioinformatics</I><B>30</B>: 2068–2069.
					https://academic.oup.com/bioinformatics/article/30/14/2068/2390517 (October 18, 2018)
					<B>Strieker M, Tanović A, Marahiel MA. 2010.</B> Nonribosomal peptide synthetases: structures and dynamics. <I>Current Opinion in Structural Biology</I><B>20</B>: 234–240.
					http://www.sciencedirect.com/science/article/pii/S0959440X10000126 (October 18, 2018)
					<B>Walsh CT. 2008.</B> The Chemical Versatility of Natural-Product Assembly Lines. <I>Accounts of Chemical Research</I><B>41</B>: 4–10.
					https://doi.org/10.1021/ar7000414 (October 18, 2018)
					<B>Weber T, Blin K, Duddela S, Krug D, Kim HU, Bruccoleri R, Lee SY, Fischbach MA, Müller R, Wohlleben W, <I>et al.</I> 2015.</B> antiSMASH 3.0—a comprehensive resource for the genome mining of biosynthetic gene clusters. <I>Nucleic Acids Research</I><B>43</B>: W237–W243.
					https://academic.oup.com/nar/article/43/W1/W237/2467910 (October 18, 2018)
					<B>Werkhäuser N, Bilstein A, Sonnemann U. 2014.</B> Treatment of Allergic Rhinitis with Ectoine Containing Nasal Spray and Eye Drops in Comparison with Azelastine Containing Nasal Spray and Eye Drops or with Cromoglycic Acid Containing Nasal Spray. <I>Journal of Allergy.</I>
					https://www.hindawi.com/journals/ja/2014/176597/ (October 18, 2018)
					<B>Yang S-C, Lin C-H, Sung CT, Fang J-Y. 2014.</B> Antibacterial activities of bacteriocins: application in foods and pharmaceuticals. <I>Frontiers in Microbiology</I><B>5</B>.
					https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4033612/ (October 18, 2018)
				</DIV></DIV><FOOTER><DIV style="position: absolute;left: 0;bottom: 0;color: #b1b1b1;margin: 0.5%;font-size: 0.8em;">B. Marchal</DIV></FOOTER></DIV></DIV></BODY></HTML>