<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Marburg_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Marburg/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="header_wrapper"><DIV class="menu_wrapper" role="navigation" aria-label="Main Menu"><UL class="menu"><LI><A href="#">Team ▾</A></LI><LI><A href="/Team:Marburg/Team">Team</A></LI><LI><A href="/Team:Marburg/Collaborations">Collaborations</A></LI><LI><A href="/Team:Marburg/Attributions">Attributions</A></LI></UL><LI><A href="#">Project ▾</A></LI><LI><A href="/Team:Marburg/Description">Description</A></LI><LI><A href="/Team:Marburg/Design">Design</A></LI><LI><A href="/Team:Marburg/Experiments">Experiments</A></LI><LI><A href="/Team:Marburg/Notebook">Notebook</A></LI><LI><A href="/Team:Marburg/InterLab">InterLab</A></LI><LI><A href="/Team:Marburg/Model">Model</A></LI><LI><A href="/Team:Marburg/Results">Results</A></LI><LI><A href="/Team:Marburg/Demonstrate">Demonstrate</A></LI><LI><A href="/Team:Marburg/Measurement">Measurement</A></LI><LI><A href="/Team:Marburg/Labautomation">Labautomation</A></LI><LI><A href="#">Parts ▾</A></LI><LI><A href="/Team:Marburg/Parts">Parts Overview</A></LI><LI><A href="/Team:Marburg/Basic_Part">Basic Parts</A></LI><LI><A href="/Team:Marburg/Composite_Part">Composite Part</A></LI><LI><A href="/Team:Marburg/Part_Collection">Part Collection</A></LI><LI><A href="/Team:Marburg/Improve">Improve</A></LI><LI><A href="/Team:Marburg/Safety">Safety</A></LI><LI><A href="#">Human Practices ▾</A></LI><LI><A href="/Team:Marburg/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Marburg/Public_Engagement">Public Engagement</A></LI><LI><A href="#">Awards ▾</A></LI><LI><A href="/Team:Marburg/Human_Practices">Integrated Human Practices</A></LI><LI><A href="/Team:Marburg/Public_Engagement">Education and Public Engagement</A></LI><LI><A href="/Team:Marburg/Model">Model</A></LI><LI><A href="/Team:Marburg/Measurement">Measurement</A></LI><LI><A href="/Team:Marburg/Basic_Part">Best New Basic Part</A></LI><LI><A href="/Team:Marburg/Composite_Part">Best New Composite Part</A></LI><LI><A href="/Team:Marburg/Part_Collection">Best Part Collection</A></LI><LI><A href="/Team:Marburg/Software">Software</A></LI><LI><A href="https://igem.org/2018_Judging_Form?id=2560">Judging Form</A></LI></DIV></DIV><MAIN><DIV class="titleWrapper"><DIV class="title">Demonstrate</DIV></DIV><ARTICLE><P><I>The important achievement of Apollo was demonstrating
that humanity is not forever chained to this planet
and our visions go rather further than that and our opportunities are unlimited.</I><B>-- Neil Armstrong</B></P><H3>Vibrio Basics</H3><P>
	Realizing <I>V. natriegens</I> as a widely used <B>host organism</B> for synthetic biology requires well-funded knowledge about it! Realizing this, we prioritized <B>fundamental research</B> early on. We showed the unparalleled speed of <I>V. natriegens</I> replication, defined a range of optimal growth conditions, including pH and salt tolerance, and  the ease of its <B>genetic accessibility</B>.

	<A href="https://2018.igem.org/Team:Marburg/Experiments"><ABBR title="Link to the experiments page">  protocols </ABBR></A></P><P>
	We managed to enable transformation <A href="https://2018.igem.org/Team:Marburg/Experiments"><ABBR title="Link to the experiment page"> demonstrated  </ABBR></A>

	for with high electroporation efficiency and heat-shock transformation to drive synthetic biology research.


</P><P>
	In combination with our  

	<A href="https://2018.igem.org/Team:Marburg/Part_Collection"><ABBR title="Link to the part collection page"><B>Marburg-Collection</B>,</ABBR></A>

	we accomplished cloning of simple plasmids, from transformation to miniprep, <B>under 12 hours</B>, and assembly and preparation of level 2 <B>golden gate</B> constructs in <B>under three days</B>!
</P><P>
	Additionally, our team successfully implemented five fragment <A href="https://2018.igem.org/Team:Marburg/Results"><ABBR title="Link to the results page">Gibson cloning </ABBR></A> as well as <A href="https://2018.igem.org/Team:Marburg/Results"><ABBR title="Link to the results page">Aqua cloning </ABBR></A>
	             and achieved high <B>reliability</B> at <B>high performance</B>.

</P><P>
	We sequenced both chromosomes with <B>Illumina sequencing</B>, mapped them to existing genome maps and ran <B>automated annotation</B> tools to identify genetic features.
</P><P>
	Working concentrations for most common antibiotics were elucidated and used throughout the project.
</P><P>
	Applying several electron microscopic methods, we could, apart from generating
<A href="https://2018.igem.org/File:T--Marburg--Josef_1.png"><ABBR title="Link to a EM picture of V. natriegens"><B>nice pictures,</B></ABBR></A>
	  highlight shape, form and volume of <I>V. natriegens. </I>Fortunately, we could observe several cell divisions in mid process.
</P><H3>Strain Engineering</H3>


We successfully demonstrated a genome engineering workflow in <I>V. natriegens</I>, using linear fragments and transforming them by natural competence induced by the regulator protein <B>TfoX</B>. Furthermore, we established the <B>Flp/<I>frt</I></B> system in <I>V. natriegens</I> for excisions of selection markers that were integrated into the genome.<P>
For <B>VibriClone 1.0</B>, we could successfully delete the nuclease <I>dns</I> and create linear <DFN data-info="desoxy nucleaic acid"> DNA </DFN> cassettes for further genomic modifications, to improve this strain and allow a highly efficient cloning. Moreover we designed <B>VibriClone 2.0</B> with additional features, such as deletion of both nucleases, a <I>recA1</I> mutation, cold resistance and the ability for blue white selection, which makes our strain more suitable as the next generation cloning chassis.</P><P>
In case of <B>VibriXpress</B> we detected a protease with a high similarity to the Lon protease of <I>E. coli</I> that needs to be deleted for high protein yield and could design and create cassettes for the integration of the T7 polymerase, the deletion of <I>lon</I> and the deletion of <I>dns</I>. We were not able to cotransform these fragments into <I>V. natriegens</I>, but could show, that the integration of the <I>dns</I> deletion cassette was successful.</P><P>
	
We created the strain <B>VibriInteract</B> carrying the deletion of <I>cyaA</I>. The excision of the selection marker, using the Flp/<I>frt</I> system and curing the plasmid for natural transformation was successful. By this, we provided a strain for fast protein interaction studies, could demonstrate its functionality by <DFN data-info="Vibrio 2 Hybrid"> V2H </DFN>assays and characteized its growth and cell morphology.</P><P>
We successfully characterized 15 neutral chromosomal integration sites in <I>Vibrio natriegens</I>, eight of them on chromosome 1 and seven on chromosome 2. Furthermore, we verified the correct integration of 5 of them on both chromosomes, by sequencing and characterized their impact on the growth rate. Insertion of the two sequences (12 and 20) results in a deficit in growth and sequence 15 only affects the growth in a small impact. For the sequences 19 and 22 our results show that an integration has almost no effect on the growth. Therefore, they are very suitable for integration, and open up a broad range of application, such strain engineering or integrating enzymes in metabolic engineering.</P><H3>Marburg Collection</H3><P>We designed and constructed the <A href="https://2018.igem.org/Team:Marburg/Part_Collection"><ABBR title="Link to the parts page">Marburg Collection</ABBR></A> the most flexible golden-gate based toolbox for prokaryotes. It contains <B>123 LVL0 parts</B> including:  
constitutive and inducible promoters, <DFN data-info="ribosomal binding site"> RBS,</DFN> reporter and tools for genome engineering, terminators, oris, resistance cassettes and a set of self-designed connectors.  </P>

All parts were submitted to the registry to help future <DFN data-info="international Genetically Engineered Maschine"> iGEM</DFN> teams in <B>achieving</B> ambitious projects and, for increased convenience, we additionally enable download of plasmid maps of all parts in our wiki  

To characterize our parts, we established a <B>fast</B> and <B>convenient platereader workflow</B> tailored to species-specific properties of<I> V. natriegens</I> and evaluated the <B>optimal plasmidal context</B>.   

Little characterization has been done for genetic parts in <I>V. natriegens</I>, so we applied our own workflow to obtain the very <B>first experimental data</B> for promoter strength, dose dependency of inducible promoters, <DFN data-info="ribosomal binding site"> RBS,</DFN> strength, terminator read-through and the insulating behavior of our <B>novel connector parts</B>.   

We <B>characterized</B> our ori parts in<I> V. natriegens</I> by showing their impact on <B>reporter</B> expression and furthermore,  <DFN data-info="quantitative Polymerase Chain Reaction"> qPCR,</DFN> experiments revealed differences in plasmid copy numbers depending on reporter expression.   
<P>
To additionally ease Golden-Gate cloning we developed the software tool   

<A href="https://2018.igem.org/Team:Marburg/Software"><ABBR title="Link to the software award page"><B>Click ‘n’ Clone</B></ABBR></A>
 which provides a <DFN data-info="Graphical User Interface">GUI</DFN> in which a user can simply select the desired parts for building a plasmid. A detailed <B>pipetting protocol</B> for manual operation or a <B>picking list</B> that is compatible with lab automation is given as result.

</P><H3>Metabolic Engineering</H3><P>By <B>characterizing</B> all involved enzymes, we laid the <B>foundation</B> to work with the  <DFN data-info="3-hydroxypropionic acid"><B>3-HPA</B></DFN> pathwayin <I> Vibrio natriegens</I>. 
We tested several <DFN data-info="Acetyl-CoA carboxylase"> Accs</DFN> from a range of organisms and <A href="https://2018.igem.org/Team:Marburg/Results"><ABBR title="Link to the results page">demonstrated,</ABBR></A> their activity in <I> V. natriegens.</I><DFN data-info="Acetyl-CoA carboxylase"> Acc</DFN> from <I>Synechococcus elongatus </I>proved to be the most promising contender for <B>maximum productivity</B>.</P><P>Also, the <DFN data-info="Malonyl-CoA reductase"> Mcr</DFN> was <A href="https://2018.igem.org/Team:Marburg/Results"><ABBR title="Link to the results page">shown</ABBR></A> to be <B>soluble and functional</B>. 
Combining <DFN data-info="Malonyl-CoA reductase"> Mcr</DFN> and <DFN data-info="Acetyl-CoA carboxylase"> Acc</DFN> in vitro, we could quantitatively <B>detect our product</B><DFN data-info="3-hydroxypropionic acid"> 3-HPA </DFN>, and additionally, <B>reliably</B> differentiate it from its structural isomer lactate.
</P><P>
Drawing from the strength of our <A href="https://2018.igem.org/Team:Marburg/Model"><ABBR title="Link to the model page"><B>model</B>,</ABBR></A> we <B>rationally designed</B> our pathway for optimal, directed metabolite flow.
</P><P>
Moreover, we used the <A href="https://2018.igem.org/Team:Marburg/Part_Collection"><ABBR title="Link to the parts page">Marburg Collection</ABBR></A> to create a library containing <B>over 390 pathway variants</B>. These harbor different combinations of promoter, <DFN data-info="ribosomal binding site"> RBS,</DFN> and coding sequences for <DFN data-info="Acetyl-CoA carboxylase"> Acc</DFN>, <DFN data-info="Malonyl-CoA reductase"> Mcr</DFN> and BirA. 
</P><P>
To enable <B>rapid</B> in vivo and in vitro <B>product screening</B>, we <B>successfully cloned</B> a  <DFN data-info="3-hydroxypropionic acid"> 3-HPA </DFN><A href="https://2018.igem.org/Team:Marburg/Results"><ABBR title="Link to the results page"><B>biosensor</B></ABBR></A>.
</P><P>
Benefiting from the <B>flexibility</B> of the<A href="https://2018.igem.org/Team:Marburg/Part_Collection"><ABBR title="Link to the parts page">Marburg Collection</ABBR></A> in combination with sensor based <B>real time product screening</B>, we created the possibility for directed evolution by designing a workflow utilizing <DFN data-info="Fluorescence Activated Cell Sorting"> FACS</DFN> and <DFN data-info="Multiplex Automated Genome Engineering  "> MAGE</DFN>. 
</P><P>
Furthermore, we designed a bypass to re-feed our product into the central metabolism, thereby introducing <B>evolutionary pressure</B> to adopt and strengthen this route 
</P><P>
To conclude, we considered each aspect of metabolic engineering and successfully <B>accelerated</B> the workflow of <B>pathway optimization</B>.
</P></ARTICLE></MAIN><FOOTER><DIV style="position: absolute;left: 0;bottom: 0;color: #b1b1b1;margin: 0.5%;font-size: 0.8em;">B. Marchal</DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>