<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Marburg_Collaborations skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Marburg/Collaborations</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="header_wrapper"><DIV class="menu_wrapper" role="navigation" aria-label="Main Menu"><UL class="menu"><LI><A href="#">Team ▾</A></LI><LI><A href="/Team:Marburg/Team">Team</A></LI><LI><A href="/Team:Marburg/Collaborations">Collaborations</A></LI><LI><A href="/Team:Marburg/Attributions">Attributions</A></LI></UL><LI><A href="#">Project ▾</A></LI><LI><A href="/Team:Marburg/Description">Description</A></LI><LI><A href="/Team:Marburg/Design">Design</A></LI><LI><A href="/Team:Marburg/Experiments">Experiments</A></LI><LI><A href="/Team:Marburg/Notebook">Notebook</A></LI><LI><A href="/Team:Marburg/InterLab">InterLab</A></LI><LI><A href="/Team:Marburg/Model">Model</A></LI><LI><A href="/Team:Marburg/Results">Results</A></LI><LI><A href="/Team:Marburg/Demonstrate">Demonstrate</A></LI><LI><A href="/Team:Marburg/Measurement">Measurement</A></LI><LI><A href="/Team:Marburg/Labautomation">Labautomation</A></LI><LI><A href="#">Parts ▾</A></LI><LI><A href="/Team:Marburg/Parts">Parts Overview</A></LI><LI><A href="/Team:Marburg/Basic_Part">Basic Parts</A></LI><LI><A href="/Team:Marburg/Composite_Part">Composite Part</A></LI><LI><A href="/Team:Marburg/Part_Collection">Part Collection</A></LI><LI><A href="/Team:Marburg/Improve">Improve</A></LI><LI><A href="/Team:Marburg/Safety">Safety</A></LI><LI><A href="#">Human Practices ▾</A></LI><LI><A href="/Team:Marburg/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Marburg/Public_Engagement">Public Engagement</A></LI><LI><A href="#">Awards ▾</A></LI><LI><A href="/Team:Marburg/Human_Practices">Integrated Human Practices</A></LI><LI><A href="/Team:Marburg/Public_Engagement">Education and Public Engagement</A></LI><LI><A href="/Team:Marburg/Model">Model</A></LI><LI><A href="/Team:Marburg/Measurement">Measurement</A></LI><LI><A href="/Team:Marburg/Basic_Part">Best New Basic Part</A></LI><LI><A href="/Team:Marburg/Composite_Part">Best New Composite Part</A></LI><LI><A href="/Team:Marburg/Part_Collection">Best Part Collection</A></LI><LI><A href="/Team:Marburg/Software">Software</A></LI><LI><A href="https://igem.org/2018_Judging_Form?id=2560">Judging Form</A></LI></DIV></DIV><MAIN><DIV class="titleWrapper"><DIV class="title">Collaborations</DIV></DIV><ARTICLE><H2>Interlab Study in <I>V. natriegens</I></H2>
Our goal is to bring <I>Vibrio natriegens</I> and all its advantages to the synbio community as a whole. To do that, we need to learn more about how it behaves in other labs.
In order to find out more about this was to have other iGEM teams conduct experiments with <I>V. natriegens</I>. With this in mind, we decided to expand on the InterLab study, because the participants were already  familiar with it. The plan was to include as many teams as possible and provide them with the same protocol, <I>V. natriegens</I>, and have them conduct the same simple experiment, the InterLab Cell Measurement. 
The original InterLab was designed to solve one of biology’s biggest problems, inherent unpredictability of living organisms, like bacteria. This unpredictability makes it difficult to compare measurements  and compare devices from one lab to another. Therefore, a common basis of comparison is paramount for scientists to be able to build on each others work. This is especially true in synthetic biology because our models that require precise measurements. This engineering approach makes standardized units even more important. With this in mind, iGEM created a GFP measurement procedure and compared the data from labs all over the world

<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4777433/?report=classic"><ABBR title="(Jacob Beal, Traci Haddock-Angelli, Markus Gershater, Kim de Mora, Meagan Lizarazo, Jim Hollenhorst and Randy Rettberg: Reproducibility of Fluorescent Expression from Engineered Biological Constructs in E. coli, PLoS One, 2016; 11(3)">(Beal <I>et al.</I>, 2016) </ABBR></A>

 . All teams tested the same constructs, the only variables being the lab. Since we want <I>Vibrio natriegens</I> to become the new go-to chassis, testing it like E.coli can give us valuable insight.
Nearly everything was done the same as in the original iGEM InterLab, E.coli was just replaced by Vibrio natriegens. The teams transformed the plasmids into <I>Vibrio natriegens</I> and conducted GFP measurements. This gave us invaluable insight into how people unfamiliar with Vibrio natriegens adapt to using it.  
To complement the results of your experiments we asked all participants to fill out a quick survey. 
We wanted to hear about their difficulties with the experiments and Vibrio natriegens; with this information we could improve our protocols based on their feedback in order to optimize them for possible future worldwide adoption.

<DIV><FIGURE style="float:right; width:35% "><FIGCAPTION>
          Figure 1: A map of Europe with white dots, each representing a team who collaborated with us.</FIGCAPTION></FIGURE>

	11 teams from Germany, Switzerland, Austria, Greece, Spain, Norway and the Netherlands took part in the Vibrigens InterLab Collaboration. The following list is containing links to the wikis of all teams participating:
   <UL><LI><A href="https://2018.igem.org/Team:TU_Darmstadt/Collaborations" target="blank">Click here to visit TU_Darmstadt's collaborations page</A></LI><LI><A href="https://2018.igem.org/Team:JMU_Wuerzburg" target="blank">Click here to visit JMU_Wuerzburg's collaborations page</A></LI><LI><A href="https://2018.igem.org/Team:ETH_Zurich/Collaborations" target="blank">Click here to visit ETH Zurich's collaborations page</A></LI><LI><A href="https://2018.igem.org/Team:Thessaloniki/Collaborations" target="blank">Click here to visit Thessaloniki's collaborations page</A></LI><LI><A href="https://2018.igem.org/Team:Valencia_UPV/Collaborations#phillip" target="blank">Click here to visit Valencia_UPV's collaborations page</A></LI><LI><A href="https://2018.igem.org/Team:BOKU-Vienna/Collaborations" target="blank">Click here to visit BOKU-Vienna's collaborations page</A></LI><LI><A href="https://2018.igem.org/Team:NAWI_Graz/Collaborations" target="blank">Click here to visit NAWI_Graz's collaborations page</A></LI><LI><A href="https://2018.igem.org/Team:NTNU_Trondheim/Collaborations" target="blank">Click here to visit NTNU_Trondheim's collaborations page</A></LI><LI><A href="https://2018.igem.org/Team:TU-Eindhoven" target="blank">Click here to visit TU-Eindhoven's collaborations page</A></LI><LI><A href="https://2018.igem.org/Team:Hamburg/Collaborations#main-content" target="blank">Click here to visit iGEM Hamburg's collaborations page</A></LI><LI><A href="https://2018.igem.org/Team:Stuttgart/Collaborations" target="blank">Click here to visit Stuttgart's collaborations page</A></LI></UL></DIV><P>
    Stuttgart, Hamburg, TU_Darmstadt, JMU_Wuerzburg, ETH Zurich, and BOKU-Vienna performed the complete Vibrigens InterLab experiment. Thessaloniki and NTNU_Trondheim each performed a growth curve experiment. The measurements from iGEM Hamburg cannot be used for evaluation since the negative control showed a higher fluorescence than the positive control at times. This is likely due to a contamination or a pipetting error.
    Unfortunately, the other three teams on the list couldn’t produce results due to various reasons. The Valencia_UPV Team got their own <I>Vibrio natriegens</I> from CECT, listed as the same strain we use. 
    They encountered problems with transformation. We think this might have to do with the strain used, since most other teams were able to insert their plasmids using our strain. Regrettably we don’t trust this website to have the right strain, because in the description of the organism it lists <I>Vibrio natriegens</I> as an Archaea. TU-Eindhoven couldn’t finish the experiments due to issues not related to the Vibrigens InterLab. NAWI_Graz had no valued measurements and were missing two of the devices, because they broke during transportation and the plates were too moist. 
    For the growth curve experiment the teams simply measured the OD<SUB>600</SUB> every 10 minutes for 5 hours, in the Vibrigens InterLab the iGEM InterLab cell measurement experiment was repeated.</P><DIV style="display: inline-block"><FIGURE style="float:right; width:45% "><FIGCAPTION>
          Figure 2: The Vibrigens InterLab Kit with a Vibrio 101 flyer.</FIGCAPTION></FIGURE><B>Content of our <I>V. natriegens</I> InterLab study kit:</B><UL><LI>8 InterLab plasmids</LI><LI>Electroporation media</LI><LI>Recovery media</LI><LI>V2 salts</LI><LI><I>Vibrio natriegens </I>on a plate, ATTC14048</LI><LI>Flyer <I>Vibrio natriegens</I> 101</LI></UL>
We created a <I>Vibrio natriegens</I> flyer containing the most important facts and most useful protocols for the organism. Our protocol for the Vibrigens InterLab is in our lab book on our Wiki.
For the Vibrigens growth curve we only sent V2 salts, <I>Vibrio natriegens</I> on a plate strain ATTC14048 and the <I>Vibrio natriegens</I> 101 Flyer.
</DIV><DIV class="collapsible"><DIV class="btn_expand">Results</DIV><DIV class="content"><FIGURE style="float:right; width:35% "><FIGCAPTION>
          Figure 3: Pie chart showing the percentage of teams that would want to work with Vibrio natriegens again.</FIGCAPTION></FIGURE>


In our Google form we asked our participants to tell us all about their difficulties with the experiments and Vibrio natriegens and to answer some general questions. When asked if they could imagine working with <I>Vibrio natriegens</I> in the future, 7 out of 8 teams answered yes (figure 3).
We could identify four main critique points based on their feedback:<OL><LI><B>The shipping</B>
We received a lot of feedback on our shipping. Some of the plates broke during transport because we didn’t pack them as well as we should have. For the future we recommend using bubble wrap around the plates or enough crumpled paper to really cushion the plates. Relying only on parafilm and paper towels wasn’t enough to send plates in an envelope instead of a box. Boxes provide more protection against mechanical impact and are therefore better suited for this.</LI><LI><B>Transformation</B>
Some teams had problems transforming the plasmids into Vibrio natriegens and needed two attempts until they were able to insert the plasmid into <I>V. natriegens</I>. Two of the teams couldn’t perform the transformation, so we sent them the strains. In our lab we observed that the more experience you have, the higher the success rate of transformations. We believe that with guidance and a few more tries the teams would have been able to complete this step, but it would have taken precious time away from their projects.</LI><LI><B>Fast growth</B>
Some teams were so used to good ol’ <I>E. coli</I> they weren’t prepared for <I>Vibrio natriegens</I>. The fast growth of <I>V. natriegens</I> surprised them and threw off their lab routine. In some cases they ended up overgrown plates. We think that we can solve this problem in the future by simply spreading the word on <I>Vibrio natriegens</I>. If people are confident that the organism will deliver on its promises and grow fast, they can plan their workflow accordingly.</LI><LI><B>Not storable in the fridge</B>
In the beginning of our project we realized that <I>Vibrio natriegens</I> had problems surviving in the fridge. If left outside in room temperature the plates became overgrown. We are working on a strain that is able to survive cold temperatures to solve this problem. We want to insert a KatG catalase into our <B>VibriClone</B> strain, but we haven’t completed this experiment as of now.</LI></OL><DIV style="display: block;"><H3>InterLab promotors in <I>Vibrio natriegens</I></H3>
First, we conducted the Vibrigens InterLab (Fig. 4) ourselves and established what promoters were the strongest for us. We plotted a graph to show how strong the promoters were in relation to each other by comparing the data gathered after 3h of growth time.
The strongest fluorescence was emitted by device 5. Device 4 and 2 were very similar, but had a much weaker fluorescence than 5. Devices 1, 3 and 6 had incredibly low GFP values.
Also, we plotted a graph to show how strong the promoters were in relation to each other by combining the data gathered after 3h of growth time from all participating teams.
</DIV><DIV><FIGURE style="float:left; width:49% "><FIGCAPTION>
           Figure 4: Graph showing the Marburg team's Vibrigens InterLab result.</FIGCAPTION></FIGURE><FIGURE style="float:right; width:49% "><FIGCAPTION>
          Figure 5: Graph showing the participating teams Vibrigens InterLab results. </FIGCAPTION></FIGURE></DIV><P> </P><P> </P><P> </P><P> </P><P> </P><P> </P><P> </P><P> </P>


    We calculated the mean emission of each device for each team and the strongest fluorescence was emitted by device 5. 
Device 1 and 4 were very similar and came in at about half the fluorescence signal as 5. Devices 2 and 6 had a quite low emission and device 3 almost no signal.
<DIV style="display: block;"><FIGURE style="float: left;width: 50%;"><FIGCAPTION>
          Figure 6: Graph showing our and the participating teams Vibrigens Growth Curve results.</FIGCAPTION></FIGURE></DIV><H3>Growth Curves of <I>V. natriegens</I></H3>
We compared our curve to the growth curves from other teams.
The data points from our measurement aren’t connected because we didn’t measure
inbetween. After their first growth curve didn’t show the results we expected NTNU was kind
enough to repeat the experiment, that is why they have two curves. In two hours NTNU
reached an OD<SUB>600</SUB> of about 3,5 both times, Thessaloniki reached an OD<SUB>600</SUB> 5 and we an OD<SUB>600</SUB> eached 12.
<P> </P><P> </P><P> </P><P> </P><P> </P><P> </P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Discussion and Outlook</DIV><DIV class="content"><H3>Science</H3><H4>Vibrigens InterLab</H4>
It can be observed that Vibrio natriegens produces a much weaker fluorescence signal at the same OD when compared to E.coli using the same plasmids. In some cases, the fluorescence with a plasmid meant to express GFP is weaker than the blank. This happens because the cells block light more than they emit fluorescence at high densities. According to our measurements the strongest promoter was Test Device 5, followed by 4, 2, 1, 3 and 6. The other teams also found device 5 to be the strongest, but followed by 1, 4, 2, 6 and 3. Device 5, Part BBa_J364008 with the promoter J23104 is unequivocally the strongest. The experiments showed similar results, except that Device 1 was much weaker in our measurements than for the other teams. Device 6 and three had such weak GFP expression that small variations can change their place in the order of strength.
We had some difficulty comparing resulting because the units of fluorescence are not standardized. We could only compare results and not analyze them on their own. Some teams had a much higher values then the other. This doesn’t mean that they measured a higher fluorescence, just that the values were given in another frame. The only quality control we could conduct on the data given to us were common sense checks, for example, the negative control had to show a lower fluorescence than the positive control. These checks excluded one team from our experiment.
<P> </P><H4>Growth curve</H4>
In the five hours in which the experiment was performed by the other team, we reached an OD600 of about 12, and they reached one of 3,5 to 5. There are also some small factors that can combine to truly impact growth. Vibrio natriegens needs a lot of oxygen, so it’s important to take big baffled flasks and keep them shaking at all times. It is also very sensitive to cold, so taking the flasks out of the incubator to take samples can have an impact on the growth speed. These small adjustments come with experience, and because the other teams are much more used to working with E.coli they treated the organism similarly, which worked well, but not as well as it did in our lab (after we gained experience). We think these small tricks created the difference between our growth curves. With experience it would be possible for them to achieve the same growth rate we started with growth curves that only reached an OD600 of about 1, and now we are able to produce ones with an OD600 of 14. 
<P> </P><H4>GFP/LUX</H4>
From all the data that we gathered from this experiment we formed an opinion on whether GFP measurements are adequate to realize the InterLab studies full potential. We think that by replacing GFP by Lux and measuring luminescence instead of fluorescence we could make a more universal experiment, easier to reproduce not only from lab to lab, but also from organism to organism. One of the Lux operon advantages is the almost complete absence of background signal without reporter expression. This is the reason why the Lux operon is suitable to analyze extremely low levels of expression caused by weak promoters or terminator readthrough. We also noticed we have a lower detection threshold when using Lux in our lab, which is why we did all measurements using Lux.
The variability in the interLab results, represented by the large error bars and the order of strength changing, is in part a result of the low accuracy of GFP measurements in general. With fluorescence there is an excitation beam that produces light scattering and can falsify your emission measurement, resulting less accurate data. Lux has a lower detection threshold because there is no light being emitted to excite the Chromophore, instead a chemical reaction causes luciferin to emit light. There is no background! One big drawback of Lux is that it would be much more difficult to calibrate. For GFP, you can simply use a known concentration of fluorescein.
<P> </P><H3>General</H3><H4> Organization</H4>
First of all, we were surprised by how complicated it was organizing and running an InterLab study. Writing universally understandable protocols for students on different levels of knowledge proved difficult. We only had to manage 11 teams, but this small number should not be underestimated. Each country has its own postage rules, which we had to learn about before we could send anything. 11 packages, each with about 40 aliquots needed to be prepared, labeled and wrapped individually took a lot of manpower and patience to pack. We can’t imagine what a logistical nightmare working with 340 teams would be.
Preparing the InterLab earlier would have been better. Tasks you never thought about will have to be completed and something will inevitably go wrong, so it is very important to leave margin for error while planing. Also sending the kits out when there was more time until the wiki freeze might have given more teams, not just NTNU (thank you!), time to repeat experiments. 
Having undergraduate students test out your protocols, not team members who know the organism, would have been beneficial to find where small tips that we see as general knowledge at this point should be inserted into the protocol.
We thought the language barrier might present a problem, but were pleasantly surprised by how easy understanding each other was. It was still difficult to communicate with the other teams at times. Email is somewhat formal and because we are young and inexperienced it still takes effort to compose emails (especially in a foreign language), which leads to much slower communication. It would have been better to have instant messaging where we could reach the teams fast and casually instead of email. To help with the experiments a group chat for all teams to exchange tips would have made getting information out faster, easier and possibly yielded better results.
Another problem we encountered was that teams didn’t fill in our excel sheets instead making their own with the data we asked for. Maybe if we had explained how we would interpret the data the participants would have known to fill in the correct excel sheets.
<P> </P><H3>Shipping</H3>
From this experience we learned that there is much room for improvement. Not only should we have packed the plates in a more secure manner, sending duplicates would have made the experiment less susceptible to human error. Unfortunately our lack of experience combined with our lack of resources made it so that we didn’t have boxes big enough to accommodate extra bubble wrap and duplicates. We didn’t realize we needed those until it was too late, at which point we had already bought the shipping containers. 
<P> </P><H2>Outlook</H2>
If we were to conduct an InterLab type experiment again, we would take all that we learned and mentioned above, and implement it. 
If the team next year works with Vibrio natriegens aswell, they could examine if teams that have worked with V. natriegens before had better results because of their experience with the organism. Probably by then we would have the VibriClone strain that can live in colder temperatures, eliminating the fridge problem.

<P>
Our vision is to establish our Vibrigens Interlab study over the next years by improving everything we learned this year, and subsequent years. Vibrio natriegens made an appearance in many labs in europe, but next year we want to expand to other continent, to share with more people how this organism can truly change the day to day workflow in labs everywhere.</P></DIV></DIV><H2> German Meetup </H2><P>
		As this years Marburg iGEM Team 2018 had the honor to continue the tradition of our predecessors to host the german iGEM Meetup. The meeting was established to exchange experiences earned while participating in the competition as well as getting feedback about our projects from people with a fresh perspective. 
		<FIGURE style="width: 40%; float: right; text-align: center;"><FIGCAPTION>People discussing at posters</FIGCAPTION></FIGURE>
		Marburg is not only at the heart of Germany, but also a very charming city with a rich academic tradition while balancing at the cutting edge of Science. 
		On Friday the 22th, the teams started arriving in Marburg, where our city's campground had been booked for the meetup. Quickly, the tents were set up and everyone settled in. After giving out some welcome gifts, a tour of the old town was undertaken. Small groups of iGEMees were led by a member of Marburg’s team to discover some hidden gems in the narrow alleyways between the medieval buildings. 
		<FIGURE style="width: 40%; float: left; text-align: center;"><FIGCAPTION>The Marburg iGEM Team proudly presenting the meetups schedule</FIGCAPTION></FIGURE>
		The next day, Saturday, started with a breakfast buffet at the great auditorium, followed by speakers from the German association of synthetic biology (GASB), crystals first, a startup, and Rüdiger Trojok, a well-known member of Germanys growing biohacker scene.
		To sharpen our presentation skills as well as getting fresh impulses and new directions beyond the current state of the projects, the participating teams brought posters with an outline of their work and preliminary results. All teams decided on interesting projects covering a wide range of topics, and presented those convincingly, resulting in a very stimulating atmosphere for all iGEMees.
		We want to thank all the people who came to our little town and made this weekend the great experience it was.<B> We want to thank the teams from 
			<A href="https://2018.igem.org/Team:Bielefeld-CeBiTec">Bielefeld</A>,
			<A href="https://2018.igem.org/Team:Duesseldorf">Düsseldorf</A>,
			<A href="https://2018.igem.org/Team:Hamburg">Hamburg</A>,
			<A href="https://2018.igem.org/Team:FAU_Erlangen">Erlangen</A>,
			<A href="https://2018.igem.org/Team:TU_Darmstadt">Darmstadt</A>,
			<A href="https://2018.igem.org/Team:JMU_Wuerzburg">Würzburg</A>,
			<A href="https://2018.igem.org/Team:Aachen">Aachen</A>,
			<A href="https://2018.igem.org/Team:HSHL">Hamm Lippstadt</A>,
			<A href="https://2018.igem.org/Team:Munich">München</A>,
			<A href="https://2018.igem.org/Team:Tuebingen">Tübingen</A>,
			<A href="https://2018.igem.org/Team:Stuttgart">Stuttgart</A>,
			<A href="https://2018.igem.org/Team:Goettingen">Göttingen</A> and our visitor from
			<A href="https://2018.igem.org/Team:NCHU_Taichung
">Taiwan</A> for attending!
		We are looking forward to seeing you again in Boston!</B><FIGURE class="mobile_w_full" style="width: 80%; float: top; text-align: center; margin: 0 auto 0 auto"><FIGCAPTION>Group photo by the riverside</FIGCAPTION></FIGURE></P><H2>Accessible Wiki - The Collaboration</H2>
We didn’t want to stop at designing our own wiki barrier-free to make the results of the iGEM competition accessible to everyone. Because, let’s face it, there are many  more amazing and interesting projects out there. This is why we decided to encourage other teams to design their wiki accessible, too. By participating in  this collaboration, all parties benefit. Firstly, people with vision impairment can access the web pages easily. Secondly, the teams create a broader audience for their results. Lastly and most importantly, this collaboration is spreading awareness to the issue of an inclusive society that is long overdue.<P>
For this to happen, we put together a guide containing the basic criteria needed for an accessible website. Click here to get to our accessible wiki guide. After spreading our campaign over social media we contacted over 80 teams in personal messages. Interestingly, the majority of the other teams had never heard of accessible web design. Some did and had already implemented some criteria into their wiki.</P><DIV><H3>Participating Teams:</H3><B>iGEM BioIQSBarcelona</B> who designed their wiki partly accessible<A href="https://2018.igem.org/Team:BioIQS-Barcelona" target="blank">Click here to learn more about their project!</A><P><B>iGEM Barcelona</B> who designed their wiki accessible for colour-blind people.<A href="https://2018.igem.org/Team:UPF_CRG_Barcelona" target="blank">Click here to learn more about their project!</A></P><P><B>iGEM Chalmers Gothenburg</B> who designed all their graphics accessible using alternative text.<A href="https://2018.igem.org/Team:Chalmers-Gothenburg" target="blank">Click here to learn more about their project!</A></P><P><B>iGEM KCL</B> who met some of the criteria stated in our guide.<A href="https://2018.igem.org/Team:KCL_UK" target="blank">Click here to learn more about their project!</A></P><P><B>iGEM Purdue</B> who met many of the criteria found in our guide.<A href="https://2018.igem.org/Team:Purdue" target="blank">Click here to learn more about their project!</A></P><P><B>iGEM Sorbonne U Paris</B> who met a lot of criteria stated in our guide.<A href="https://2018.igem.org/Team:Sorbonne_U_Paris" target="blank">Click here to learn more about their project!</A></P><P><B>iGEM Westminster UK</B> who designed their wiki partly accessible.<A href="https://2018.igem.org/Team:Westminster_UK" target="blank">Click here to learn more about their project!</A></P></DIV><DIV style="float:left;text-algin:justify;">
A very special THANK YOU goes out to the iGEM team of the Sorbonne University of Paris! After the initial contact they were highly motivated to contribute. We were thrilled to have found a team that was equally excited about this project as we were and still are. Several long and substantial mails later, both of our teams could improve our wikis. The team of Sorbonne gave us tips on how to design our wiki as responsive as possible and introduced us to e.g. tags so that your browser can recognize the language you’re writing in and by this the browser can translate it to other languages automatically. They designed their wiki barrier-free and additionally gave us very helpful tips to improve our guide to an accessible wiki. Using their tips, like e.g. adding examples, our guide is now easier to understand so that even beginner wiki managers can use our guide to design an accessible wiki.</DIV><DIV style="float:right; text-align: justify;">
When contacting the iGEM team of the Kings College in London, our call for an accessible iGEM competition was met with a (to us) new idea. KCL has a public campaign in which they ask to design websites accessible for people with special educational needs (sen). We were  happy to participate in their campaign. It’s just as easy to design your wiki suitable for sen people as it is to design a wiki accessible for people with visual impairment!</DIV><DIV style="display:block; float:left;text-algin:justify;">To conclude, through this collaboration with all these amazing teams, we could improve the Human Practices part of our project by spreading awareness on the inclusion of people with disabilities into the scientific community. For the future, we hope for the iGEM community to adopt this design and by this to be a shining example for the rest of the scientific and general public. One possibility to do this could be to embrace accessible web design as a medal criterium or a requirement in general.</DIV><H2>iGEM Duesseldorf - Postcards</H2>
iGEM Duesseldorf wanted to create a co-culture toolbox.  With their project “Trinity” they want to make microbial-communities usable for diverse purposes. Since 2016, iGEM Düsseldorf is doing a postcard campaign with the purpose of educating the public about topics in synthetic biology. We were happy to join and designed a postcard which fit with our project. <B>#vibriGAINZ</B>. When the postcards from teams all over the world arrived we were delighted to see the work everyone put into them. Additionally, by this we got interested in all those teams and had a further look into their project<P><A href="" target="blank">Click here to learn more about their project Trinity!</A></P><DIV><H2>iGEM US_AFRL_CarrollHS - Mike the Microbe</H2><B>iGEM US_AFRL_CarrollHS</B> was trying to destroy biofilms using engineered <I>E. coli</I>. As part of their Human Practices project, they created Mike the Microbe. With this little fellow they want to connect iGEM teams from all over the world. We loved to participate and had Mike the Microbe join us in the lab for a day.<P><A href="https://2018.igem.org/Team:US_AFRL_CarrollHS" target="blank">Click here to learn more about their project!</A></P></DIV><DIV><H3>iGEM Göttingen - ITC measurements</H3>
iGEM Göttingen is trying to detect and inactivate glyphosate using <I>B. subtilis</I> and <I>E. coli</I>. This year, together with the Research group of our first PI Gert Bange, we helped the iGEM Team Göttingen by trying to detect the protein interaction between the substrate glyphosate and the EPSP synthases AroE and AroA from <I>B. subtilis</I> and <I>E. coli</I>. Pietro Giammarinaro instructed us for the ITC measurement. Unfortunately, we could not detect any interaction.<P><A href="https://2018.igem.org/Team:Goettingen" target="blank">Click here to learn more about their project!</A></P></DIV><DIV><H2>iGEM IIT Madras - ChassiDex</H2><B>iGEM IIT Madras</B> is developing new tools for working with <I>Acinetobacter baylyi</I>. With this they want to establish this organism as a possible new chassis organism in synthetic biology. Last year’s team established a database for host organisms other than  <I>E. coli</I> which can be used in synthetic biology. The team continues their site called ChassiDex this year as well, so we had the chance to contribute our knowledge about <I>Vibrio natriegens</I>. Hopefully we can help the scientific community by sharing the knowledge we gained and protocols that worked for us in the past months.<P><A href="https://chassidex.org/" target="blank">Click here to learn more about ChassiDex!</A><A href="https://2018.igem.org/Team:IIT-Madras" target="blank">Click here to learn more about their project ADaPtat1on!</A></P></DIV><DIV><H2>iGEM Thessaloniki - InterLab</H2><P><FIGURE style="width: 50%; float: right"><FIGCAPTION>The Divices 1-4 from Thessaloniki and our negativ control. </FIGCAPTION></FIGURE>
We collaborated with this year’s Thessaloniki iGEM team by taking part in their InterLab. They are working on a synthetic biology circuit which ensures a stable protein expression. By making the gene expression independent from the potentially fluctuating copy number of plasmids, better predictions can be made about the stoichiometry when metabolic pathways are integrated into the chromosome. In this experiment we tested their four test devices. </P><P>We hope our results represent the desired outcome of the InterLab. We cultivated the cells in an incubator that was in frequent use by other team members, so the cultures weren’t shaken for multiple short intervals. If our data varies from what was expected, one explanation could be, that the Escherichia coli cells didn’t incubate correctly and therefore didn’t grow as well.</P><P><A href="https://2018.igem.org/Team:Thessaloniki" target="blank">Click here to get more information about their project GALENE!</A></P></DIV><DIV><H2>iGEM Westminster UK - Plastic Lab</H2><B>iGEM Westminster</B> wants to establish a way to degrade polystyrene (which is one of the least recycled plastics) by combining chemistry and biology. In our labs we sometimes tend to not think about the amount of waste we produce. As scientists we should try to use plastic responsibly by avoiding wastefulness,  while keeping the workflow at an optimum level. We contributed to their project by filming a video showing plastic use in our labs.<P><A href="https://2018.igem.org/Team:Westminster_UK" target="blank">Click here to learn more about their project Biotroopers!</A></P></DIV><DIV><H2>Surveys</H2><P>Surveys are one of the most reliable methods to get feedback from a broad audience. So,of course, a lot of iGEM teams from all over the world are conducting them to evaluate their project. We were happy to participate to support their amazing projects!</P><H3>iGEM Lund</H3>
iGEM Lund wanted to increase recombinant protein yield via co-expression of <I>Vitreoscilla hemoglobin</I>. They conducted a survey about workload and structures within the team. We hope we could help to improve their project.<A href="https://2018.igem.org/Team:Lund" target="blank">Click here to get more information about their project!</A><H3>iGEM Macquarie</H3>
The iGEM Team of the Macquarie University wanted to produce Chlorophyll-Induced Vesicles in <I>E. coli</I> to add a new artificial cell compartment for production of substances toxic for the cell.  We happily participated in their survey.<A href="https://2018.igem.org/Team:Macquarie_Australia" target="blank">Click here to get more information about their project ChiVes!</A><H3>iGEM Montpellier</H3>
iGEM Montpellier wanted to develop a new non hormonal contraception method using engineered bacteria. To get an impression of people’s opinions about contraception and their project, they conducted a survey.  We were happy to take part as we think that a new method of contraception which is not relying on hormones is long overdue!<A href="https://2018.igem.org/Team:Montpellier" target="blank">Click here to get more information about their project Vagineering!</A><H3>iGEM Imperial College</H3>
The iGEM Team of the Imperial College in London wanted to develop of the first aerobic electrogenetic control system in <I>E. coli</I>. As their Human Practices project, they designed an app for team communication. We filled out a survey about common issues we face as an iGEM team to help them develop an app suited for other iGEM teams. We can’t wait to tell the next iGEM team about it!<A href="https://2018.igem.org/Team:Imperial_College" target="blank">Click here to get more information about their project PixCell!</A><H3>iGEM Sorbonne</H3>
iGEM Sorbonne Université wanted to produce sugar in the microalgae <I>Chlamydomonas reinhardtii</I> for eco friendly sugar production compared to the production on arable land. They conducted a survey to improve their project and we were happy to take part! <A href="https://2018.igem.org/Team:Sorbonne_U_Paris" target="blank">Click here to get more information about their project Suga[R]evolution!</A></DIV></ARTICLE></MAIN><FOOTER><DIV style="position: absolute;left: 0;bottom: 0;color: #b1b1b1;margin: 0.5%;font-size: 0.8em;">B. Marchal</DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>