"
Page
Discussion
View source
History
teams
Log in
 
Team:Bologna/T-REx Story
From 2009.igem.org
HOME
TEAM
PROJECT
SOFTWARE
MODELING
WET LAB
PARTS
HUMAN PRACTICE
JUDGING CRITERIA
Story of a T-REX
(Trans-Repressor of Expression)
The main goal of our project is the realization of a device for post-transcriptional regulation of gene expression in Escherichia coli, in order to switch protein production of a gene of interest.
The Parts
To achieve this aim we designed, using BASER tool, two complementary 50bp non-coding sequences:
CIS-repressing, to be assembled upstream of the target protein coding sequence; it contains a ribosomal binding site (RBS):
5' Prefix - AACACAAACTATCACTTTAACAACACATTACATATACATTAAAATATTACAAAGAGGAGAAA - 3' Suffix
TRANS-repressor, to be placed under the control of a different promoter; it is complementary to the CIS-repressing, and contains a RBS cover in two versions of different length (4 and 7 nucleotides):
5' Prefix - CCTCTTTGTAATATTTTAATGTATATGTAATGTGTTGTTAAAGTGATAGTTTGTGTT - 3' Suffix
5' Prefix - CCTCTTTGTAATATTTTAATGTATATGTAATGTGTTGTTAAAGTGATAGTTTGTGTT - 3' Suffix
The BioBricks
Both CIS and TRANS sequences were synthesized (by Sigma-Aldrich) as single strand oligos; they were delivered in our lab on 07/31/2009. We then started our work in the wet-lab, to produce the CIS-repressing and the TRANS-repressor BioBricks.
As first, we made an E-P digestion of the annealed sequences, following standard protocols. We purified the digested parts from agarose gel and we attempted ligation in BBa_PSB1A2  (high copy number vector), but we did not obtain transformants using competent DH5α cells.
We tried the same ligation many other times using different procedures, in order to verify every single passage. For each attempt we tried several cloning vectors, featuring either low or high copy number (BBa_PSB3K3; BBa_PSB4A3; BBa_PSB1A2; BBa_PSB1AK3; BBa_PSB1K3) and several restriction enzyme combinations (E-P, X-P, E-S). We also tried using vectors with BBa_P1010 (“Death” gene), in order to avoid purification in agarose gel.
Every cloning attempt yielded at most few colonies. Moreover, digestion of plasmids isolated from those colonies revealed electrophoretic bands of unexpected MW, when checked in agarose gel or by colony PCR.
Comments
Here below is reported a brief synthesis of all our attempts done while trying to clone these parts. We checked several possible occurring problems and we tried to solve them one by one.
Were ss DNAs correctly annealed?
We purified our annealed parts from agarose gel, in order to eliminate all the smears (Fig. 1), before proceeding with digestion and ligation.
Figure 1. Annealed parts before digestion
Since we could not digest/ligate any of them, we tried to amplify our annealed parts with PCR, in order to increase their purity. We used standard prefix and suffix as primers:
Prefix: 5' GAATTCGCGGCCGCTTCTAGAG 3'
Suffix: 3' TACTAGTAGCGGCCGCTGCAG 5'
We tried to purify PCR products with either electrophoresis in agarose gel (Fig. 2) or with a PCR-extraction column. Both these attempts brought no colonies after digestion/ligation.
Figure 2. PCR products on agarose gel
Were our digestions effective?
As the efficiency of restriction enzymes is considerably reduced when there aren’t enough bases near their restriction sites, we used longer primers in our PCR protocols, adding 7 bases to the end of each part. The added bases were the same presents near restriction sites in all standard vectors. The new primers were then:
Prefix: 5' ATTTCTGGAATTCGCGGCCGCTTCTAGAG 3'
Suffix: 3' TACTAGTAGCGGCCGCTGCAGGCTTCCT 5'
Following enzyme manufacturer’s advice, we also doubled digestion time. We made several cloning attempts with these longer parts, obtaining once again no success.
Was there something wrong in our sequences prefix or suffix?
We sequenced our annealed strands and we found some strange results about P and E restriction sites, as if there were two different kinds of terminal sequences.
X and S sites were correct, so we tried an X-S digestion. Since X and S sticky ends are complementary, our part could ligate either in the right or in the wrong way: we were then expecting two possible kinds of plasmid. However, the inserts ligated in the wrong way would have prevented any further digestion both for X and S, allowing us to recognize them.
For the first time the ligation returned a lot of colonies. We checked some of them with two different colony PCRs. The results seemed positive as the bands were of the right length in both cases.
Figure 3. Colony PCR with standard Prefix and Suffix as primers: inserts were all at about 100bp as expected
Figure 4. Colony PCR with standard VR and VF2 primers: inserts were all at about 300bp as expected (100bp is the length of our parts and 200bp are nucleotides due to primers’ binding on plasmid)
We picked some of this colonies, but the control digestion of the plasmids isolated from them revealed once again wrong length of the bands checked in agarose gel.
Since the sequencing of these parts showed once again a negative result, we still can't explain which was the cloning problem.
One potential answer is that we do not get enough DNA to complete ligation after purification from gel, because our parts are less than 100bp long, and 100bp is a critical length for our gel-extraction kit. Anyhow, even if using the BBa_p1010 procedure, at least one purification step is required for the PCR products and this has probably been the critical passage in all our cloning attempt. Indeed, most of the times the amount of extracted DNA was below the limit of detection, or presented very low concentration.
As we are running out of time, we have decided to put aside this cloning attempt and concentrate our energies on characterization of other devices used in our project, within the established deadline of 10/21/2009.
Up
Retrieved from " http://2009.igem.org/Team:Bologna/T-REx_Story "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
