<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Bioware_Project_Results_ITRDiary"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg_Bioware/Project/Results/ITRDiary">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg_Bioware/Project/Results/ITRDiary&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_Bioware/Project/Results/ITRDiary&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_Bioware/Project/Results/ITRDiary&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg_Bioware/Project/Results/ITRDiary" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg Bioware/Project/Results/ITRDiary</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="menu"><UL id="menu"><LI><DL><DT class="menu_button_left"><A href="https://2010.igem.org/Team:Freiburg_Bioware">Home</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Team_2010">Team</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Team_2010">Team 2010</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Collaboration">Collaboration</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Cuckoo_Clock">Cuckoo Clock Competition</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Photo_Gallery">Photo Gallery</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Statistics">Statistics</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Contribution">Contribution</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#highlights">Project</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results">Summary</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#highlights">Highlights</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Project_Description">Introduction</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results">Results</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#modularization">   Modularization</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#targeting">   Targeting</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#arming">   Arming</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#superconstruct">   Tumor Killing</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling">   Modeling</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Methods">Methods</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Virus_Construction_Kit">Virus Construction Kit Manual</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks">BioBricks</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks">Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/capsid_coding">Capsid Coding Parts</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/vector_plasmid">Vector Plasmid Parts</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Notebook">Notebook</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal">Labjournal</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Materials">Cloning Material</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Methods">Methods</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Meetings">Meetings</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Statistics">Statistics</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Sponsors">Sponsors</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Safety">Safety</A></DT></DL></LI><LI><DL><DT class="menu_button_right"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling">Modeling</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling">Modeling Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Virus_Production">Virus Production</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Virus_Infection">Virus Infection</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Structure_Modeling">Structure Modeling</A></DD></DL></LI></UL></DIV><DIV class="pageContent"><P><SPAN style="font-family:Segoe Script; font-size:25px; font-weight:bold;">Dear Diary,</SPAN></P><P>once upon a time there were two inverted terminal repeats (ITRs) – the left and the right ITR. They flanked a CMV promoter, followed by a β-globin intron, a multiple cloning site and a hGH polyA site in order to be packaged into Adeno-associated virus serotype 2 particles. 

One day some biology students, participating at the iGEM competition, wanted to produce BioBricks of these two DNA sequences.
<SPAN style="font-family:Segoe Script; font-size:15px; font-weight:bold;">26.05.2010</SPAN></P><P>For this purpose they designed and ordered primers in order to delete the PstI restriction sites which flank the ITRs via site-directed mutagenesis. <SPAN style="font-family:Segoe Script; font-size:15px; font-weight:bold;">02.06.2010</SPAN></P><P>Primers for removal of NotI restriction sites, which also flank the ITRs, were designed and ordered. <SPAN style="font-family:Segoe Script; font-size:15px; font-weight:bold;">10.06.2010</SPAN></P><P>The students conducted the first attempt of removing the PstI restriction sites: The <B>1<SUP>st</SUP></B> PCR for site-directed mutagenesis was performed…
… but didn’t yield any results.<SPAN style="font-family:Segoe Script; font-size:15px; font-weight:bold;">18.06.2010</SPAN></P><P>The team encountered a problem: The designed primers bound to both ITRs, thus amplification of just one ITR was not possible. So they hit upon the idea of digesting the plasmid with AlwNI in order to produce two different-sized fragments. Prior to PCR these two sequences could be separated by electrophoresis. </P><P><SPAN style="font-family:Segoe Script; font-size:15px; font-weight:bold;">25.06.2010</SPAN></P><P>Plasmid was digested with AlwNI.</P><P><SPAN style="font-family:Segoe Script; font-size:15px; font-weight:bold;">28.06.2010</SPAN></P><P>The <B>2<SUP>nd</SUP></B> site-directed mutagenesis PCR was performed…
… but again didn’t work - probably due to GC-rich regions which result in the very strong secondary structure of the <B>inverted terminal repeats</B> (see left picture :) ).
The students didn’t burry their head in the sand: New primers with higher annealing temperatures were designed and ordered.<SPAN style="font-family:Segoe Script; font-size:15px; font-weight:bold;">30.06.2010</SPAN></P>Simultaneously the eager FreiGEMs figured out an alternative plan for the ITR BioBrick production, which required no PCR steps (!): They wanted to take advantage of the NotI and PstI restriction sites flanking the ITRs. For this purpose special oligos needed to be designed: After hybridization they should deliver NotI or PstI overhangs in that way that these restriction sites become removed after ligation. <P>By providing RFC10 restriction sites the ITR_oligo construct could be cloned into the iGEM standard plasmid resulting in ITR BioBrick. The FreiGEMs named this alternative plan the “ITR fancy method”. </P><P><SPAN style="font-family:Segoe Script; font-size:15px; font-weight:bold;">05.07.2010</SPAN></P><P>The “ITR fancy method” was specified:
Oligos were designed which needed to be hybridized and provided necessary RFC10 prefix or suffix restriction sites. They possesed compatible overhangs to the PstI and NotI restriction sites of the ITRs but shouldn’t generate new restriction sites after ligation.<SPAN style="font-family:Segoe Script; font-size:20px; font-weight:bold;">ITR fancy method:</SPAN>
•	Digestion of pAAV_MCS with AlwNI: results in one large fragment (2982 bp) containing the right ITR and one small fragment (1674 bp) containing the left ITR. <SPAN style="font-family:Segoe Script; font-size:15px; font-weight:bold;">Left ITR:</SPAN>
•	Digestion of left ITR fragment with PstI and NotI
•	Digestion of pSB1C3 with EcoRI and NotI
•	Ligation of left ITR with prefix and pSB1C3</P><P>•	Digestion of pSB1C3_prefix_leftITR with XbaI and NotI
•	Digestion of pSB1C3 with PstI and XbaI
•	Ligation of left ITR with suffix and pSB1C3</P><P><SPAN style="font-family:Segoe Script; font-size:15px; font-weight:bold;">Right ITR:</SPAN>
•	Digestion of right ITR fragment with NotI and PstI 
•	Digestion of pSB1C3 with XbaI and PstI
•	Ligation of right ITR with prefix and pSB1C3</P><P>•	Digestion of pSB1C3_rightITR with EcoRI and NotI 
•	Digestion of pSB1C3 with EcoRI and NotI 
•	Ligation of right ITR with special suffix and pSB1C3 </P><P>•	Digestion of pSB1C3_prefix_rightITR_specialsuffix with EcoRI and SpeI
•	Digestion of pSB1C3 with EcoRI and SpeI
•	Ligation of pSB1C3 with right ITR</P><P>
On the same day the <B>3<SUP>rd</SUP></B> PCR attempt with new primers was conducted. Two reactions per ITR were performed – one without and one with 10% DMSO… but again no PCR product was detectable.</P><P><SPAN style="font-family:Segoe Script; font-size:15px; font-weight:bold;">12.07.2010</SPAN></P><P>First steps of the “ITR fancy method” were performed.  Result: ITRs with RFC10 prefix.</P><P><SPAN style="font-family:Segoe Script; font-size:15px; font-weight:bold;">14.07.2010</SPAN></P><P>The ambitious students wanted to play save and also ordered the left ITR according to the RFC10 standard at a gene synthesis company. Nevertheless they continued with PCR and cloning, because they knew by experience that DNA synthesis could possibly take long. 
</P><P><SPAN style="font-family:Segoe Script; font-size:15px; font-weight:bold;">15.07.2010</SPAN></P><P>Unfortunately it turned out that the first try of the “ITR fancy method” didn’t work. The students were not surprised: Digestion of plasmid, gel extraction, followed by second digestion and gel extraction of just 150 bp small ITR fragments leaves nearly no DNA for ligation. Nevertheless cloning was repeated.
</P><P><SPAN style="font-family:Segoe Script; font-size:15px; font-weight:bold;">17.07.2010</SPAN></P><P>Test digestion revealed: 
RFC10 prefix was successfully fused to the left ITR. <SPAN style="font-family:Segoe Script; font-size:15px; font-weight:bold;">19.07.2010</SPAN></P><P>Test digested of the right ITR also revealed positive results: Right ITR was provided with RFC10 prefix.</P><P>Some team impressions:<SPAN style="font-family:Segoe Script; font-size:15px; font-weight:bold;">“Yess, that’s great! Finally something worked with our ITRs!” (Bea)</SPAN></P><P><SPAN style="font-family:Segoe Script; font-size:15px; font-weight:bold;">“The most astonishing test digestion gel of our entire working time!” (Adrian)</SPAN></P><P><SPAN style="font-family:Segoe Script; font-size:15px; font-weight:bold;">21.07.2010</SPAN></P><P>Sequencing not only revealed that the prefixes were successfully fused to the ITRs, but also that besides site-directed mutagenesis also sequencing of the ITRs following the standard procedure does not work.
</P><P>The students concluded that the polymerase isn’t able to read through and that their ITRs must fulfil a great job in forming very strong secondary structures! 
It turned out that a “G” was missing in the prefix of the left ITR. Therefore new oligos were designed and ordered.
</P><P><SPAN style="font-family:Segoe Script; font-size:15px; font-weight:bold;">26.07.2010</SPAN></P><P>The left ITR fancy method was started again with the newly arrived oligos. 
In addition to that cloning of the right ITR was continued: Fusion of special suffix to right ITR.<SPAN style="font-family:Segoe Script; font-size:15px; font-weight:bold;">29.07.2010</SPAN></P><P>pGA14_newPrefix_leftITR and pGA14_prefix_rightITR_specialSuffix was prepared and test digested. The ITR bands didn’t exactly match to the expected fragment size. In addition a second band could be detected at about 500 bp. The students weren’t able to explain this and sent three samples for sequencing.
</P><P><SPAN style="font-family:Segoe Script; font-size:15px; font-weight:bold;">31.07.2010</SPAN>
Left ITR:</P><P>The sequencing company didn’t manage to read through the left ITR samples until now. The FreiGEMs interpreted this as good news: Problems with sequencing indicated very strong secondary structures – caused by (for example) ITRs! :)
Therefore “ITR fancy method” was continued: Suffix was fused to left ITR.

Rigth ITR:</P><P>Sequencing results confirmed test digestion: The right ITR was provided with RFC10 prefix. However due to short chromatogram range it was not possible to validate that the “special” suffix was fused to the ITR. 
The students didn’t bounce back and continued with the last step of their ITR fancy method: </P><P>The right ITR needed to be subcloned into a plasmid providing the iGEM restriction sites, resulting in RFC10 suffix downstream of the ITR.</P><P><SPAN style="font-family:Segoe Script; font-size:15px; font-weight:bold;">02.08.2010</SPAN></P><P>Last mini-prep and test digestion was conducted: The expected fragment sizes (156 resp. 157) could be detected in the referring range between 100 and 200 bp. 

Now the students wanted to verify these results and sent the samples for sequencing – this time following special procedures.
</P><P><SPAN style="font-family:Segoe Script; font-size:15px; font-weight:bold;">04.08.2010</SPAN></P><P>Eight sequencing files (which needed to be analyzed base by base &quot;per hand&quot;) later, alignments showed that the ITRs are OK and were successfully converted into the RFC10 BioBrick standard.  
After subcloning the ITRs into the iGEM pSB1C3 plasmid, the students were happy and proud of their success. 
</P><P><SPAN style="font-family:Segoe Script; font-size:20px; font-weight:bold;">So they all lived happily ever after and had much fun with their new ITR BioBricks.</SPAN></P><P>
By the way: The students obtained satisfaction, because the ordered rebellious ITRs didn’t arrive until now – even DNA synthesis seems to be difficult….
I guess the lesson is clear: Never perform PCR with ITRs, nor try to order them from gene synthesis companies. And: Better do not try to sequence them! ;) <SPAN style="font-family:Segoe Script; font-size:20px; font-weight:bold;">Good luck!</SPAN></P><H3><SPAN class="mw-headline" id="Source">Source</SPAN></H3><UL><LI>Berns, K., 1990. Parvovirus replication. Microbiology and Molecular Biology Reviews, 54(3), p.316. Available at: <A href="http://mmbr.asm.org/cgi/content/abstract/54/3/316" class="external free" rel="nofollow">http://mmbr.asm.org/cgi/content/abstract/54/3/316</A>.
</LI></UL><UL><LI>Srivastava, a, Lusby, E.W. &amp; Berns, K I, 1983. Nucleotide sequence and organization of the adeno-associated virus 2 genome. Journal of virology, 45(2), pp.555-64. Available at: <A href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=256449&amp;tool=pmcentrez&amp;rendertype=abstract" class="external free" rel="nofollow">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=256449&amp;tool=pmcentrez&amp;rendertype=abstract</A>.
</LI></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Freiburg_Bioware/Project/Results/ITRDiary">http://2010.igem.org/Team:Freiburg_Bioware/Project/Results/ITRDiary</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg_Bioware/Project/Results/ITRDiary" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg_Bioware/Project/Results/ITRDiary" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg_Bioware/Project/Results/ITRDiary&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg_Bioware/Project/Results/ITRDiary&amp;oldid=190207" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>