"
Page
Discussion
View source
History
teams
Log in
 
Team:Freiburg Bioware/Project/Targeting Fusion
From 2010.igem.org
Home
Team
Team 2010
Collaboration
Cuckoo Clock Competition
Photo Gallery
Statistics
Contribution
Project
Summary
Highlights
Introduction
Results
   Modularization
   Targeting
   Arming
   Tumor Killing
   Modeling
Methods
Virus Construction Kit Manual
BioBricks
Overview
Capsid Coding Parts
Vector Plasmid Parts
Notebook
Labjournal
Cloning Material
Methods
Meetings
Statistics
Sponsors
Safety
Modeling
Modeling Overview
Virus Production
Virus Infection
Structure Modeling
Tumor Targeting by N-terminal Fusion to Virus Capsid
Design
VP2 Fusion
VP1 Insertion
Cloning Verified by Colony PCR
Fluorescence Analysis of Viral Stocks
Concentrating VP1UP_NLS_mVenus_VP2/3 Viruses
Nuclear Localization by Fluorescence Microscopy
Transduction Efficacy by Flow Cytometry
Infectious Titer by qPCR
Killing cells: Time-Lapse
 
 
Tumor Targeting by N-terminal Fusion to Virus Capsid
Design
VP2 Fusion
The Freiburg2010 team fused motifs which are desired to be surface exposed to the N-terminus of VP2/3 open reading frame, avoiding steric hindrance by connecting them with linker. We ensured in frame coupling by designing or using the referring parts in the Freiburg RFC25 standard, thus creating fusion constructs via NgoMIV and AgeI. For retargeting AAV2 virus particles towards EGFR over expressing cancer cells we investigated the ZEGFR:1907 Affibody and the E01 DARPin – in combination with VP2/3_587-KO (HSPG knock down) – as surface exposed motifs. We also created so called “all-in-one” virus particles by fusing VP2/3_His-Tag or VP2/3_BAP via Middle Linker to the Affibody. Purification or imaging approaches could be conducted by His-Tag or CFP motifs, respectively. All constructs were cloned downstream of the CMV promoter (Fig.1).
Table 1: List of VP2 fusion parts
  
Figure 1: Cloning scheme of VP2 fusion
First of all we fused the Affibody ZEGFR:1907 to the Short-, Middle-, Long- or SEG Linker and cloned the resulting constructs downstream of the CMV promoter. Afterwards either unmodified VP2/3 or HSPG affinity knock down VP2/3 were fused to the different linkers. Further on CFP and His-Tag were coupled to VP2/3 by the Middle Linker (Fig. 2).
  Figure 2: Agarose Gel Electrophoresis. Fusion of VP2/3 to CMV_Affibody_Linker or CMV_CFP_MiddleLinker constructs.
We conducted three fragment ligations with the DARPin E01, MiddleLinker_VP2/3 or MiddleLinker_VP2/3(587KO) and pSB1C3_CMV plasmid (Fig. 3).
 
Figure 3: Agarose Gel Electrophoresis. A) Three fragment ligation of DARPin fused to N-terminus of VP2/3. B) Test digestion verified successful cloning.
For 100 % replacement of wildtype VP2 the fusion plasmid was co-transfected to the Rep/Cap plasmid which contained a start codon mutation of VP2. Wildtype VP2 knock-out was achieved by site-directed mutagenesis (Fig. 4).
Figure 4: Knock out of VP2 in the Cap coding region of Rep/Cap plasmid: Knock-out of start codon via site-directed mutagenesis (note the C in the sequence).
 
VP1 Insertion
We inserted three diffent motifs into the VP1 open reading frame. They were fused, together with the unique uspstream region of VP1 (VP1up) and a nuclear localization sequence (NLS), according to the RFC25 standard to VP2/3 (Fig. 2). Again, all of these parts were driven by CMV promoter.
Table 2: List of VP1 insertion parts
Figure 5: Cloning scheme of VP1 insertion
First VP1up-NLS was fused to the surface exposed motif – Affibody, His-Tag or mVenus – and the resulting constructs were cloned downstream of the CMV promoter. This was followed by either fusing unmodified VP2/3 or HSPG affinity knock down VP2/3 (Fig. 6) to the different kinds of motifs.
Figure 6: Agarose Gel Electrophoresis. Fusion of VP2/3(587KO) to pCMV_VP1up_NLS_ZEGFR:1907, pCMV_VP1up_NLS_6xHis or pCMV_VP1up_NLS_mVenus
We designed site-directed mutagenesis primer for the VP1 start codon modification in the Rep/Cap plasmid in order to ensure 100 % replacement with modified VP1 proteins (Fig. 7).
 
Figure 7: Knock out of VP1 in Cap coding region of Rep/Cap plasmid: Knock-out of start codon via site-directed mutagenesis.
 
Cloning Verified by Colony PCR
Big amounts of samples necessitated fast and exact analysis of cloning. For this purpose colony PCR was conducted. We verified successful VP2/3 fusion by using primer that exclusively bound to a specific location in the VP2/3 sequence (Fig. 8). In the case of successful ligation a 880 bp fragment was amplified via Taq polymerase.
Figure 8: Primer Cap 3500 for and VP 4200 rev. Colony PCR using these primer results in a 880 bp fragment
 
Figure 9 demonstrates a colony PCR example: VP2 fusion constructs containing VP2/3(587) sequence and the so called “All in One” constructs were cloned into pSB1C3_CMV. Successful cloning could be verified (positive control: pAAV_RC; negative control: pSB1C3_lITR).
Figure 9: Colony PCR of VP2 N-terminal fusion (Affibody_Linker, CFP_MiddleLinker, His-Tag_MiddleLinker: samples 1 - 6) and “All-in-One” constructs (samples 7 – 10).
 
Fluorescence Analysis of Viral Stocks
We wanted to develop a protocol for harvesting as much viruses as possible. For this purpose we took advantage of viruses containing mVenus fused to VP1. In order to find out whether most viruses were located in the cell pellet, cell suspensions were centrifuged at 10.000 g or 20.000 g. Afterwards mVenus fluorescence was measured at 526 nm. The resulting values were compared to cell culture supernatant, 20.000 g centrifuged supernatant and resuspended cell pellet (Fig. 10).
Figure 10: Fluorescence signal of mVenus at 526 nm, inserted into the virus capsid. Transduced cells were resuspended, pelleted or supernatant was taken in order to determine the location of most viruses. Control: DMEM.
 
Concentrating VP1UP_NLS_mVenus_VP2/3 Viruses
For further analysis, viral stocks originating from cell culture supernatant needed to be concentrated. Therefore ultrafiltration with protein concentrators (Sartorius VivaSpin 20; 20.000Da MWCO) was conducted. Figure 11 shows fluorescence microscopy image of an aliquot from ultrafiltrated cell culture supernatant / cell lysate containing VP1up_NLS_mVenus_VP2/3 viruses.
Figure 11: Fluorescence Microscopy. A) Brightfield picture of an aliquot of concentrated VP1up_NLS_mVenus_VP2/3 containing virus stock. B) At 505 nm excitation mVenus fluorescence of the virus particles could be detected. C) Merged image
 
Nuclear Localization by Fluorescence Microscopy
AAV-293 cells were transfected with a 50:50 ratio of the Rep/Cap(VP1KO) to the CMV_VP1_NLS_mVenus_VP2/3 plasmid. We packaged mCherry, driven by the CMV promoter, into the virus capsids and followed protein expression via fluorescence microscopy. 30 hours post transfection mCherry fluorescence was detectable in the whole cytosol of the successfully transfected cells, demonstrating that DNA located between the AAV2 ITRs is already transcribed in the producer cell line. In contrast to that mVenus fluorescence signal could be observed only in the nuclei (Fig. 12). This indicated that the nuclear localization sequence targets the single VP1_NLS_mVenus_VP2/3 proteins efficiently to the cell nucleus, where assembly and packaging of the virus particles takes place. 
 
Figure 12: Fluorescence Microscopy. A) Brightfiled picture. B) Excitation at 555 nm showed mCherry signal in the cytosol. C) Excitation at 505 nm revealed mVenus fluorescence in the nuclei, indicating functionality of the nuclear localization sequence inserted – together with mVenus – into VP1. D) Merged image.
Transduction Efficacy by Flow Cytometry
For determination of transduction efficacy flow cytometry analysis was conducted. The Affibody ZEGFR:1907 was fused with SEG-, middle- and long linker to the VP2/3 open reading frame (ORF) in order to investigate differences in infection efficacy due to different linker lengths. 250.000 AAV-293 cells were transfected with 1 µg total DNA. Different ratios of VP2 fusion constructs in respect to the Rep/Cap plasmid were co-transfected. 72 hours post transfection viruses were harvested and two different cell lines, HT1080 and A431, were transduced with 1 mL virus stock. By encapsulating mVenus coding sequence, the amount of transduced cells could be determined via flow cytometry.
Figure 13: Flow cytometry. Test of transduction efficiency with HT1080 and A431 cells by detecting mVenus expression from ZEGFR:1907_Middlelinker_VP2/3 virus particles (Transfection ratio: 50:50 in respect to Rep/Cap plasmid). A) Gating non transduced cells (control); subcellular debris and cellular aggregates can be distinguished from single cells by size, estimated via forward scatter (FS Lin) and granularity, estimated via side scatter (SS Lin). B) : Non transduced cells plotted against mVenus fluorescence (Analytical gate was set such that 1% or fewer of negative control cells fell within the positive region (R6). C) Gating transduced cells. D) Transduced cells plotted against mVenus fluorescence, R10 comprised transduced, mVenus expressing cells. E) Overlay of non-transduced (red) and transduced (green) cells plotted against mVenus fluorescence.
Figure 14: Flow cytometry. Investigating transduction efficiency with HT1080 and A431 cells by detecting mVenus expression from ZEGFR:1907_SEG_VP2/3 virus particles (Transfection ratio: 50:50 in respect to Rep/Cap plasmid). A) Gating non transduced cells (control); subcellular debris and cellular aggregates can be distinguished from single cells by size, estimated via forward scatter (FS Lin) and granularity, estimated via side scatter (SS Lin). B) : Non transduced cells plotted against mVenus fluorescence (Analytical gate was set such that 1% or fewer of negative control cells fell within the positive region (R6). C) Gating transduced cells. D) Transduced cells plotted against mVenus fluorescence, R10 comprised transduced, mVenus expressing cells. E) Overlay of non-transduced (red) and transduced (green) cells plotted against mVenus fluorescence.
Figure 15: Flow cytometry. Investigating transduction efficiency with HT1080 and A431 cells by detecting mVenus expression from ZEGFR:1907_LongLinker_VP2/3 virus particles (Transfection ratio: 50:50 in respect to Rep/Cap plasmid). A) Gating non transduced cells (control); subcellular debris and cellular aggregates can be distinguished from single cells by size, estimated via forward scatter (FS Lin) and granularity, estimated via side scatter (SS Lin). B) : Non transduced cells plotted against mVenus fluorescence (Analytical gate was set such that 1% or fewer of negative control cells fell within the positive region (R6). C) Gating transduced cells. D) Transduced cells plotted against mVenus fluorescence, R10 comprised transduced, mVenus expressing cells. E) Overlay of non transduced (red) and transduced (green) cells plotted against mVenus fluorescence.
Additionally viruses containing a His-Tag or CFP, fused via Middle Linker to the VP2/3 ORF, were analyzed. Figure 16 overviews transduction efficacy of all constructs, transfected in a 10:90, 25:75 and 50:50 ratio in respect to Rep/Cap plasmid.  
 
Figure 16: Flow cytometry analysis. Transduced and therefore mVenus positive HT1080 and A431 cells, infected with virus particles consisting of different ratios of VP2 fusion constructs in respect to Rep/Cap plasmid.
 
Transduction of HT1080 cells revealed that all viral particles regardless of which motifs inserted into the capsids remained infectious with efficacies up to 85 %. In general the amount of mVenus positive cells decreased only slightly when harboring more modified VP2 subunits. This indicates that larger peptides could be inserted into the AAV2 capsids without affecting virus assembly and packaging. A431 cells, which overexpress EGF receptor, were generally transduced with reduced efficacy.
Further on the Affibody ZEGFR:1907 or mVenus were inserted into the VP1 ORF together with a nuclear localization signal. 250.000 AAV-293 cells were transfected with 1 µg total DNA and different ratios of VP1 insertion constructs in respect to the Rep/Cap(VP1KO) plasmid were co-transfected. 72 hours post transfection viruses were harvested and HT1080 and A431 cells were transduced. 48 hours later the number of transduced, or to be precise mVenus positive, cells was determined via flow cytometry (Fig. 17 a, b).
Figure 17 a: Flow cytometry analysis. Transduced and therefore mVenus positive HT1080 cells, infected with virus particles containing different ratios of VP1 insertion constructs in respect to Rep/Cap(VP1KO) plasmid.
 
Figure 17 b: Flow cytometry analysis. Transduced and therefore mVenus positive A431 cells, infected with virus particles containing different ratios of VP1 insertion constructs in respect to Rep/Cap(VP1KO) plasmid.
 
Again results revealed that all virus particles remained infectious, regardless of whether inserting the Affibody or mVenus. This indicated that VP1 tolerated larger peptides inserted downstream of its unique N-terminal region and that this modification still allowed virus assembly and packaging.
Infectious Titer by qPCR
We transfected 250.000 AAV-293 cells with 1 µg of total DNA composed of equal amounts of Rep/Cap, pHelper and vector plasmid. VP2 fusion or VP1 insertion plasmids were co-transfected with two different ratios in respect to Rep/Cap(VP1KO) or Rep/Cap(VP2KO):  25 % VP2 fusion or VP1 insertion proteins allow better assembly and packaging of the virus particles, compared to 50 % VP2 fusion or VP1 insertion proteins which increase the chance of  integration into the capsids.
We created viruses containing the ZEGFR:1907 Affibody fused to VP2 or inserted into VP1 and the DARPin E01 fused to VP2. These types of AAV2 particles were produced in two versions: With or without HSPG binding affinity knock down (587KO).
Viruses were harvested three days post transfection. The genomic titer was determined via qPCR by amplification of a specific sequence located in the CMV promoter of the vector plasmid (Table 3).
Table 3: Quantitative Real-Time PCR. Determination of genomic titer. Data were corrected for negative control value.
Co-transfected Construct
Ratio
Genomic Titer /1ml
Corrected For Negative Control
Affibody_MiddleLinker_VP2/3
25:75
2,20E+08
Affibody_MiddleLinker_VP2/3
50:50
2,39E+08
Affibody_MiddleLinker_VP2/3(587KO)
25:75
1,17E+09
Affibody_MiddleLinker_VP2/3(587KO)
50:50
5,44E+08
CFP_MiddleLinker_VP2/3(587KO)
25:75
3,63E+09
CFP_MiddleLinker_VP2/3(587KO)
50:50
1,67E+09
6xHis_MiddleLinker_VP2/3(587KO)
25:75
3,37E+09
6xHis_MiddleLinker_VP2/3(587KO)
50:50
1,38E+09
VP1up_NLS_Affibody_VP2/3
25:75
3,52E+09
VP1up_NLS_Affibody_VP2/3
50:50
1,50E+09
VP1up_NLS_Affibody_VP2/3(587KO)
25:75
6,98E+08
VP1up_NLS_Affibody_VP2/3(587KO)
50:50
5,30E+08
VP1up_NLS_6xHis_VP2/3(587KO)
25:75
5,25E+09
VP1up_NLS_6xHis_VP2/3(587KO)
50:50
1,65E+09
DARPin_MiddleLinker_VP2/3
25:75
4,36E+08
DARPin_MiddleLinker_VP2/3
50:50
3,93E+08
DARPin_MiddleLinker_VP2/3(587KO)
25:75
1,00E+09
DARPin_MiddleLinker_VP2/3(587KO)
50:50
3,99E+08
Control: Rep/Cap
100%
1,55E+08
Control: Rep/Cap(587KO)
100%
5,39E+08
 
We investigated transduction of different cell lines. For this purpose 100.000 HT1080, HeLa or A431 cells were seeded and transduced with 50 µL virus stock and harvested 24 hours later.  Infectious titers were determined via qPCR and normalized to the genomic titers (Fig. 17-19).
Figure 18 shows infection efficacy of DARPin exposing viruses. Transduction of HT1080 cells was almost not affected as long as binding to HSPG was not knocked down. HeLa cells were also infected less efficient compared to the controls. However, A431 cells which overexpress EGFR were not infected by the controls. Transduction is rescued by integration of the DARPin into the virus capsid. By additionally knocking down the HSPG binding affinity these cells are transduced 10 times better, reaching wild type capsid HT1080 infection efficacy. These results indicated that specific re-targeting of AAV2 virus particles towards EGFR over expressing tumor cells was achieved by N-terminal fusion of targeting motifs to VP2.
Figure 18: DARPin E01 VP2 Fusion. Infectious titers were determined with or without HSPG knock down for HT1080, HeLa and A431 cells. Control:Rep/Cap plasmid with and without HSPG knock down.
 
We additionally obtained similar results for Affibody exposing viruses: A431 cell transduction could also be rescued by capsid integration of the Affibody. Again infection efficacy was increased by knocking down the natural tropism of the AAV2 viruses towards HSPG (Fig. 19). These data emphasized the functionality of the VP2 fusion constructs for specifically targeting tumor cells for therapeutic or imaging applications. 
Figure 19: Affibody ZEGFR:1907 VP2 Fusion. Infectious titers were determined with or without HSPG knock down for HT1080, HeLa and A431 cells. Control:Rep/Cap plasmid with and without HSPG knock down.
 
Figure 20 repesents the infectivity of viruses containing VP1 inserted Affibody molecules in respect to three different cell types (HT1080, HeLa, A431). The data clearly shows that virus capsids assembled and remained infectious. These AAV2 particles also featured specific binding properties to A431 cells, validating the functionality of  the VP1 insertion strategy. The two outlier (HeLa VP1up_NLS_Affibody_VP2/3 25:75 & HT1080 VP1up_NLS_Affibody_VP2/3(587KO) 50:50) should be left unconsidered.
Figure 20: Affibody ZEGFR:1907 VP1 Insertion. Infectious titers were determined for 25:75 and 50:50 transfection ratios with or without HSPG knock down for HT1080, HeLa and A431 cells. Control:Rep/Cap plasmid with and without HSPG knock down.
 
Killing cells: Time-Lapse
HT1080 and A431 cells were transduced with so called “All-in-One” viruses containing the Affibody ZEGFR:1907 fused to VP2/3(587KO_His-Tag) and packaged with the guanylate kinase fused to the thymidine kinase coding sequence (GMK-TK). A time series of pictures was started directly after adding 20 µM Ganciclovir (Fig.21).
Figure 21 a: Time-lapse. HT1080 (control) cells transduced with GMK-TK packaged viruses and treated with 20 µM Ganciclovir A) 0 hours, B) 7 hours, C) 15 hours and D) 23 hours post transduction.
 
Figure 21 b: Time-lapse. A431 cells transduced with GMK-TK packaged viruses and treated with 20 µM Ganciclovir A) 0 hours, B) 7 hours, C) 15 hours and D) 23 hours post transduction.
 
Virus and Ganciclovir treatment only slightly affect the morphology of HT1080 cells. After 23 hours of incubation in 20µM Ganciclovir the cells had almost the same appearance as at time point zero (Fig. 21 a). In comparison to that A431 epidermoid carcinoma cells were efficiently killed after transduction and Ganciclovir add-on: After 23 hours nearly all cells were lysed (Fig. 21 b). These results clearly demonstrate that we were able to specifically target EGFR over-expressing tumor cells via capsid integrated Affibody and that these transduced cells were efficiently killed by expressing the GMK-TK which converted prodrug Ganciclovir into its cell-toxic monophosphate.
Retrieved from " http://2010.igem.org/Team:Freiburg_Bioware/Project/Targeting_Fusion "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
