<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Bioware_NoteBook_Labjournal_October"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg_Bioware/NoteBook/Labjournal/October">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg_Bioware/NoteBook/Labjournal/October&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/October&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/October&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg_Bioware/NoteBook/Labjournal/October" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg Bioware/NoteBook/Labjournal/October</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="menu"><UL id="menu"><LI><DL><DT class="menu_button_left"><A href="https://2010.igem.org/Team:Freiburg_Bioware">Home</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Team_2010">Team</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Team_2010">Team 2010</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Photo_Gallery">Photo Gallery</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Cuckoo_Clock">Cuckoo Clock Competition</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Collaboration">Collaboration</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Example_Page">Example Page</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project">Project</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project">Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#highlights">Highlights</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#highlights">Results</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#modularisation">   Modularisation</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#targeting">   Targeting</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#arming">   Arming</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#superconstruct">   All-in-One</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Methods">Methods</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Virus_Construction_Kit">Virus Construction Kit Manual</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks">BioBricks</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks">Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/capsid_coding">Capsid Coding Parts</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/vector_plasmid">Vector Plasmid Parts</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/miscellaneous">Miscellaneous Parts</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A class="current" href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook">Notebook</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal">Labjournal</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Materials">Materials</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Methods">Methods</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Experiments">Experiments</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Meetings">Meetings</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Statistics">Statistics</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Sponsors">Sponsors</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Safety">Safety</A></DT></DL></LI><LI><DL><DT class="menu_button_right"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling">Modeling</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling">Modeling Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Virus_Production">Virus Production</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Virus_Infection">Virus Infection</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Structure_Modeling">Structure Modeling</A></DD></DL></LI></UL></DIV><DIV class="pageContent"><P><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook" class="external text" rel="nofollow">=&gt; Back to Notebook overview</A></P><DIV class="box_right"><LEFT><U1>NoteBook Navigator</U1></LEFT><UL><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/March">March (labday 1)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/April">April (labday 2 - 5)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/May">May (labday 6 - 17)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/June">June (labday 18 - 45)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/July">July (labday 46 - 75)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/August">August part 1 (labday 76 - 92)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/August2">August part 2 (labday 93 - 106)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/September">September part 1 (labday 107 - 123)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/September2">September part 2 (labday 124 - 135)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/October">October part 1 (labday 136 - 149 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/October2">October part 2 (labday 150 - 166 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/November">November  (labday 167 - 170 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/Cellculture">Cellculture</A></LI></UL></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#136._labday_01.10.2010"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">136. labday 01.10.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Test_Digestions_of_yesterdays_minipreps"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Test Digestions of yesterdays minipreps</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#mini_preps_of_CD_clones"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">mini preps of CD clones</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#test_digestion_of_CD_clones"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">test digestion of CD clones</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Biobrick_assembly_pSB1C3_lITR_hTERT_beta-globin_CD"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Biobrick assembly pSB1C3_lITR_hTERT_beta-globin_CD</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Sequencing_results_of_pCerulean_Zegfr:1907_MiddleLinker_VP2.2F3_HSPG-KO"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Sequencing results of pCerulean_Zegfr:1907_MiddleLinker_VP2/3_HSPG-KO</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Picking_clones_of_pGA14_MiddleLinker_VP2.2F3_insCap_and_pGA14_MiddleLinker_VP2.2F3_HSPG-KO"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Picking clones of pGA14_MiddleLinker_VP2/3_insCap and pGA14_MiddleLinker_VP2/3_HSPG-KO</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sequencing_results_of_pSB1C3_001_VP2.2F3_587-KO_BAP_and_pSB1C3_001_VP2.2F3_587-KO_6xHis"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Sequencing results of pSB1C3_001_VP2/3_587-KO_BAP and pSB1C3_001_VP2/3_587-KO_6xHis</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Impressions_of_transfection_of_AAV293_with_pCerulean_VP1up_NLS_mVenus_VP2.2F3"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">Impressions of transfection of AAV293 with pCerulean_VP1up_NLS_mVenus_VP2/3</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Cloning_of_hGH_rITR_into_pSB1C3_lITR_phTERT_beta-globin_CFP_and_lITR_phTERT_beta-globin_mGMK_TK30_into_pSB1C3_hGH_rITR"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">Cloning of hGH_rITR into pSB1C3_lITR_phTERT_beta-globin_CFP and lITR_phTERT_beta-globin_mGMK_TK30 into pSB1C3_hGH_rITR</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#New_BL21_cells_ready_to_use.21"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">New BL21 cells ready to use!</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-12"><A href="#137._labday_02.10.2010"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">137. labday 02.10.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Repetition_of_the_approach_of_subcloning_the_ViralBricks_HSPG-ko_BAP_and_His_into_VP2.2F3_with_different_strategies"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Repetition of the approach of subcloning the ViralBricks HSPG-ko BAP and His into VP2/3 with different strategies</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#BioBrick_Assembly_of_CD_and_mCherry"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">BioBrick Assembly of CD and mCherry</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Mini_Prep_and_test_digestion_of_pGa14_MiddleLinker_VP2.2F3_HSPG-KO"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Mini Prep and test digestion of pGa14_MiddleLinker_VP2/3_HSPG-KO</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Sequencing_analysis_of_HSPG-ko_in_several_IRCK_constructs"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Sequencing analysis of HSPG-ko in several IRCK constructs</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Mini_Prep_of_pGa14_MiddleLinker_VP2.2F3_inscap_and_pSB1C3_Darpin"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Mini Prep of pGa14_MiddleLinker_VP2/3_inscap and pSB1C3_Darpin</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Test_digestion_of_pGA14_MiddleLinker_VP2.2F3_inscap"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">Test digestion of pGA14_MiddleLinker_VP2/3_inscap</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Mass_Midi-Prep_III"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">Mass Midi-Prep III</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Analysis_of_viral_harvest_via_spectrometer"><SPAN class="tocnumber">2.8</SPAN><SPAN class="toctext">Analysis of viral harvest via spectrometer</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#200.C2.B5l-Transduction_for_testing_the_pTERT-promoter"><SPAN class="tocnumber">2.9</SPAN><SPAN class="toctext">200µl-Transduction for testing the pTERT-promoter</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#Transfection"><SPAN class="tocnumber">2.10</SPAN><SPAN class="toctext">Transfection</SPAN></A></LI><LI class="toclevel-1 tocsection-23"><A href="#138._labday_03.10.2010"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">138. labday 03.10.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Proceeding_with_the_approach_of_subcloning_the_ViralBricks_HSPG-ko_BAP_and_His_into_VP2.2F3_with_different_strategies"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Proceeding with the approach of subcloning the ViralBricks HSPG-ko BAP and His into VP2/3 with different strategies</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Midi-Prep_of_pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR_clone1"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Midi-Prep of pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR clone1</SPAN></A></LI><LI class="toclevel-1 tocsection-26"><A href="#139._labday_04.10.2010"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">139. labday 04.10.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#Three_fragment_ligation:_pSB1C3_001_CMV_VP1up_NLS_Targeting_HSPG-KO_VP2.2F3"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Three fragment ligation: pSB1C3_001_CMV_VP1up_NLS_Targeting_HSPG-KO_VP2/3</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#.C2.B4Seeding_AAV293_for_Transfection"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">´Seeding AAV293 for Transfection</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#FACS_Analysis_of_pTERT_mVenus-loaded-viral_particles_and_pCerulean_VP1up_His_VP2.2F3"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">FACS Analysis of pTERT_mVenus-loaded-viral particles and pCerulean_VP1up_His_VP2/3</SPAN></A></LI><LI class="toclevel-3 tocsection-30"><A href="#Assembly_of_pSB1C3_CMV_DARPin_VP2.2F3_insCap_and_pSB1C3_CMV_DARPin_VP2.2F3_HSPG-KO"><SPAN class="tocnumber">4.3.1</SPAN><SPAN class="toctext">Assembly of pSB1C3_CMV_DARPin_VP2/3_insCap and pSB1C3_CMV_DARPin_VP2/3_HSPG-KO</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#Miniprep_of_serveral_constructs"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">Miniprep of serveral constructs</SPAN></A></LI><LI class="toclevel-1 tocsection-32"><A href="#140._labday_05.10.2010"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">140. labday 05.10.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#Site-directed_mutagenesis_in_pSB1C3_mGMK_TK30_construct"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Site-directed mutagenesis in pSB1C3_mGMK_TK30 construct</SPAN></A></LI><LI class="toclevel-2 tocsection-34"><A href="#Cellculture:_Bad_news"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Cellculture: Bad news</SPAN></A></LI><LI class="toclevel-2 tocsection-35"><A href="#Repetition_of_pCerulean_VP1up_NLS_DARPin"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Repetition of pCerulean_VP1up_NLS_DARPin</SPAN></A></LI><LI class="toclevel-2 tocsection-36"><A href="#Harvest_of_viral_stocks"><SPAN class="tocnumber">5.4</SPAN><SPAN class="toctext">Harvest of viral stocks</SPAN></A></LI><LI class="toclevel-2 tocsection-37"><A href="#Repetition_of_Cloning_lITR_phTERT_beta-globin_mGMK_TK30_into_pSB1C3_hGH_rITR"><SPAN class="tocnumber">5.5</SPAN><SPAN class="toctext">Repetition of Cloning lITR_phTERT_beta-globin_mGMK_TK30 into pSB1C3_hGH_rITR</SPAN></A></LI><LI class="toclevel-2 tocsection-38"><A href="#Seeding_HT1080_and_AAV431_for_transduction"><SPAN class="tocnumber">5.6</SPAN><SPAN class="toctext">Seeding HT1080 and AAV431 for transduction</SPAN></A></LI><LI class="toclevel-2 tocsection-39"><A href="#Cloning_of_pSB1C3_lITR_phTERT_beta-globin_CD_and_pSB1C3_lITR_CMV_beta-globin_CD"><SPAN class="tocnumber">5.7</SPAN><SPAN class="toctext">Cloning of pSB1C3_lITR_phTERT_beta-globin_CD and pSB1C3_lITR_CMV_beta-globin_CD</SPAN></A></LI><LI class="toclevel-2 tocsection-40"><A href="#Repetition_of_the_test-digestion_of_p714.2Cp715.2Cp719.2Cp720.2Cp721.2Cp724_and_p726"><SPAN class="tocnumber">5.8</SPAN><SPAN class="toctext">Repetition of the test-digestion of p714,p715,p719,p720,p721,p724 and p726</SPAN></A></LI><LI class="toclevel-2 tocsection-41"><A href="#Repetition:_Cloning_of_VP2.2F3_BAP_HSPG-KO_and_VP2.2F3_His_HSPG-KO_into_pCerulean_Zegfr:1907_Middlelinker_and_VP2.2F3_His_HSPG-KO_into_pCerulean_VP1up_NLS_mVenus"><SPAN class="tocnumber">5.9</SPAN><SPAN class="toctext">Repetition: Cloning of VP2/3_BAP_HSPG-KO and VP2/3_His_HSPG-KO into pCerulean_Zegfr:1907_Middlelinker and VP2/3_His_HSPG-KO into pCerulean_VP1up_NLS_mVenus</SPAN></A></LI><LI class="toclevel-2 tocsection-42"><A href="#PCR_of_VP1-3"><SPAN class="tocnumber">5.10</SPAN><SPAN class="toctext">PCR of VP1-3</SPAN></A></LI><LI class="toclevel-1 tocsection-43"><A href="#141._labday_06.10.2010"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">141. labday 06.10.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-44"><A href="#Cellculture"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Cellculture</SPAN></A></LI><LI class="toclevel-2 tocsection-45"><A href="#Transduction_of_HT1080_and_A431_cells"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Transduction of HT1080 and A431 cells</SPAN></A></LI><LI class="toclevel-2 tocsection-46"><A href="#Sequence_analysis_of_pSB1C3_CD"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">Sequence analysis of pSB1C3_CD</SPAN></A></LI><LI class="toclevel-2 tocsection-47"><A href="#Site-directed_mutagenesis_of_pSB1C3_CD"><SPAN class="tocnumber">6.4</SPAN><SPAN class="toctext">Site-directed mutagenesis of pSB1C3_CD</SPAN></A></LI><LI class="toclevel-2 tocsection-48"><A href="#Sequencing_analysis"><SPAN class="tocnumber">6.5</SPAN><SPAN class="toctext">Sequencing analysis</SPAN></A></LI><LI class="toclevel-2 tocsection-49"><A href="#Test_Digestion_of_pSB1C3_CMV_DARPin_MiddleLinker_VP2.2F3_insCap_and_pSB1C3_CMV_DARPin_MiddleLinker_VP2.2F3_HSPG-KO"><SPAN class="tocnumber">6.6</SPAN><SPAN class="toctext">Test Digestion of pSB1C3_CMV_DARPin_MiddleLinker_VP2/3_insCap and pSB1C3_CMV_DARPin_MiddleLinker_VP2/3_HSPG-KO</SPAN></A></LI><LI class="toclevel-2 tocsection-50"><A href="#Repetition:_Cloning_of_CFP_into_pSB1C3_lITR_phTERT_beta-globin_and_hgH_rITR_into_pSB1C3_lITR_CMV_beta-globin_mCherry_and_into_pSB1C3_lITR_phTERT_beta-globin_mCherry"><SPAN class="tocnumber">6.7</SPAN><SPAN class="toctext">Repetition: Cloning of CFP into pSB1C3_lITR_phTERT_beta-globin and hgH_rITR into pSB1C3_lITR_CMV_beta-globin_mCherry and into pSB1C3_lITR_phTERT_beta-globin_mCherry</SPAN></A></LI><LI class="toclevel-2 tocsection-51"><A href="#Biotinylation_of_the_monoclonal_IgG_antibody_A20"><SPAN class="tocnumber">6.8</SPAN><SPAN class="toctext">Biotinylation of the monoclonal IgG antibody A20</SPAN></A></LI><LI class="toclevel-2 tocsection-52"><A href="#Repetition_of_VP1-3_PCR"><SPAN class="tocnumber">6.9</SPAN><SPAN class="toctext">Repetition of VP1-3 PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-53"><A href="#Cloning_of_Rep_into_Rep52_and_Rep78"><SPAN class="tocnumber">6.10</SPAN><SPAN class="toctext">Cloning of Rep into Rep52 and Rep78</SPAN></A></LI><LI class="toclevel-1 tocsection-54"><A href="#142._labday_07.10.2010"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">142. labday 07.10.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-55"><A href="#DARPin_VP2_Fusion:_Sequencing_results"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">DARPin VP2 Fusion: Sequencing results</SPAN></A></LI><LI class="toclevel-2 tocsection-56"><A href="#Repetition_of_cloning_of_ordered_Rep_into_Rep78"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">Repetition of cloning of ordered Rep into Rep78</SPAN></A></LI><LI class="toclevel-2 tocsection-57"><A href="#Transformation_of_VP1-3_.26_pSB1C3_001_and_p190_.26Rep52"><SPAN class="tocnumber">7.3</SPAN><SPAN class="toctext">Transformation of VP1-3 &amp; pSB1C3_001 and p190 &amp;Rep52</SPAN></A></LI><LI class="toclevel-2 tocsection-58"><A href="#Test_digestion_of_different_constructs"><SPAN class="tocnumber">7.4</SPAN><SPAN class="toctext">Test digestion of different constructs</SPAN></A></LI><LI class="toclevel-1 tocsection-59"><A href="#143._labday_08.10.2010"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">143. labday 08.10.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-60"><A href="#Mini-Prep_and_test_digestion_of_several_constructs"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">Mini-Prep and test digestion of several constructs</SPAN></A></LI><LI class="toclevel-2 tocsection-61"><A href="#Test-PCR_of_pSB1C3_lITR_phTERT_beta-globin_mGMK_TK30_hgH_rITR"><SPAN class="tocnumber">8.2</SPAN><SPAN class="toctext">Test-PCR of pSB1C3_lITR_phTERT_beta-globin_mGMK_TK30_hgH_rITR</SPAN></A></LI><LI class="toclevel-2 tocsection-62"><A href="#Trafo_evaluation_of_VP1-3_and_Rep52_constructs"><SPAN class="tocnumber">8.3</SPAN><SPAN class="toctext">Trafo evaluation of VP1-3 and Rep52 constructs</SPAN></A></LI><LI class="toclevel-2 tocsection-63"><A href="#Repetiton:_Site-directed_mutagenesis_in_pSB1C3_mGMK_TK30_construct"><SPAN class="tocnumber">8.4</SPAN><SPAN class="toctext">Repetiton: Site-directed mutagenesis in pSB1C3_mGMK_TK30 construct</SPAN></A></LI><LI class="toclevel-1 tocsection-64"><A href="#144._labday_09.10.2010"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">144. labday 09.10.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-65"><A href="#Mini-Prep_of_pSB1C3_CD_SDM-PstI"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">Mini-Prep of pSB1C3_CD_SDM-PstI</SPAN></A></LI><LI class="toclevel-2 tocsection-66"><A href="#Preparation_of_competent_cells_and_Test-Trafo"><SPAN class="tocnumber">9.2</SPAN><SPAN class="toctext">Preparation of competent cells and Test-Trafo</SPAN></A></LI><LI class="toclevel-2 tocsection-67"><A href="#Test_PCR_of_qPCR_primer_for_phTERT"><SPAN class="tocnumber">9.3</SPAN><SPAN class="toctext">Test PCR of qPCR primer for phTERT</SPAN></A></LI><LI class="toclevel-2 tocsection-68"><A href="#Cloning_of_p40_in_front_of_VP123"><SPAN class="tocnumber">9.4</SPAN><SPAN class="toctext">Cloning of p40 in front of VP123</SPAN></A></LI><LI class="toclevel-2 tocsection-69"><A href="#Site-directed_mutagenesis_of_pSB1C3_leftITR_phtert_betaglobin_mgmktk30_hGH_rITR"><SPAN class="tocnumber">9.5</SPAN><SPAN class="toctext">Site-directed mutagenesis of pSB1C3_leftITR_phtert_betaglobin_mgmktk30_hGH_rITR</SPAN></A></LI><LI class="toclevel-2 tocsection-70"><A href="#Cloning_HSPG-KO_empty_into_pSB1C3-001_pCMV_.5BAAV2.5DVP123_453_RGD_and_pSB1C3-001_pCMV_.5BAAV2.5DVP123_453_Z34C"><SPAN class="tocnumber">9.6</SPAN><SPAN class="toctext">Cloning HSPG-KO_empty into pSB1C3-001_pCMV_[AAV2]VP123_453_RGD and pSB1C3-001_pCMV_[AAV2]VP123_453_Z34C</SPAN></A></LI><LI class="toclevel-2 tocsection-71"><A href="#Cellculture_2"><SPAN class="tocnumber">9.7</SPAN><SPAN class="toctext">Cellculture</SPAN></A></LI><LI class="toclevel-1 tocsection-72"><A href="#145._labday_10.10.2010"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">145. labday 10.10.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-73"><A href="#Repetition_of_Test-PCR_of_qPCR_primer_for_phTERT"><SPAN class="tocnumber">10.1</SPAN><SPAN class="toctext">Repetition of Test-PCR of qPCR primer for phTERT</SPAN></A></LI><LI class="toclevel-2 tocsection-74"><A href="#New_XL1b_and_BL21_cells_ready_to_use.21"><SPAN class="tocnumber">10.2</SPAN><SPAN class="toctext">New XL1b and BL21 cells ready to use!</SPAN></A></LI><LI class="toclevel-2 tocsection-75"><A href="#Cloning_Super_Constructs_and_VP2_N-terminal_Fusion_HSPG-KO_constructs_into_pSB1C3_001_CMV_or_pSB1C3_CMV"><SPAN class="tocnumber">10.3</SPAN><SPAN class="toctext">Cloning Super Constructs and VP2 N-terminal Fusion_ HSPG-KO constructs into pSB1C3_001_CMV or pSB1C3_CMV</SPAN></A></LI><LI class="toclevel-2 tocsection-76"><A href="#Midi-Prep"><SPAN class="tocnumber">10.4</SPAN><SPAN class="toctext">Midi-Prep</SPAN></A></LI><LI class="toclevel-2 tocsection-77"><A href="#Checking_trafo_plates_of_Viralbrick_approaches.2C_SDM_of_pSB1C3_leftITR_phtert_betaglobin_mgmktk30_hGH_rITR_and_P40_VP123"><SPAN class="tocnumber">10.5</SPAN><SPAN class="toctext">Checking trafo plates of Viralbrick approaches, SDM of pSB1C3_leftITR_phtert_betaglobin_mgmktk30_hGH_rITR and P40_VP123</SPAN></A></LI><LI class="toclevel-2 tocsection-78"><A href="#Preparations_for_ELISA"><SPAN class="tocnumber">10.6</SPAN><SPAN class="toctext">Preparations for ELISA</SPAN></A></LI><LI class="toclevel-2 tocsection-79"><A href="#Mini-Prep_of_pSB1C3_gMK_TK30_SDM-PstI_new"><SPAN class="tocnumber">10.7</SPAN><SPAN class="toctext">Mini-Prep of pSB1C3_gMK_TK30_SDM-PstI_new</SPAN></A></LI><LI class="toclevel-2 tocsection-80"><A href="#Cloning_of_lITR_phTERT_beta-globin_mGMK_TK30_into_pSB1C3_hgH_rITR"><SPAN class="tocnumber">10.8</SPAN><SPAN class="toctext">Cloning of lITR_phTERT_beta-globin_mGMK_TK30 into pSB1C3_hgH_rITR</SPAN></A></LI><LI class="toclevel-2 tocsection-81"><A href="#Cloning_of_pSB1C3_lITR_phTERT_beta-globin_CFP_into_pSB1C3_hgH_rITR"><SPAN class="tocnumber">10.9</SPAN><SPAN class="toctext">Cloning of pSB1C3_lITR_phTERT_beta-globin_CFP into pSB1C3_hgH_rITR</SPAN></A></LI><LI class="toclevel-2 tocsection-82"><A href="#New_ligation_approach_of_Rep52"><SPAN class="tocnumber">10.10</SPAN><SPAN class="toctext">New ligation approach of Rep52</SPAN></A></LI><LI class="toclevel-1 tocsection-83"><A href="#146._labday_11.10.2010"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">146. labday 11.10.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-84"><A href="#Colony_PCR_of_Cloning_VP2_Fusion_and_Super_constructs_into_pSB1C3"><SPAN class="tocnumber">11.1</SPAN><SPAN class="toctext">Colony PCR of Cloning VP2 Fusion and Super constructs into pSB1C3</SPAN></A></LI><LI class="toclevel-2 tocsection-85"><A href="#Preparation_of_SDS-PAGE_gel_.2810.25.29"><SPAN class="tocnumber">11.2</SPAN><SPAN class="toctext">Preparation of SDS-PAGE gel (10%)</SPAN></A></LI><LI class="toclevel-2 tocsection-86"><A href="#Colony_PCR_of_p40_VP123_capins"><SPAN class="tocnumber">11.3</SPAN><SPAN class="toctext">Colony PCR of p40_VP123_capins</SPAN></A></LI><LI class="toclevel-2 tocsection-87"><A href="#Cloning_of_lITR_CMV_betaglobin_and_lITR_phTERT_betaglobin_into_pSB1C3_CD"><SPAN class="tocnumber">11.4</SPAN><SPAN class="toctext">Cloning of lITR_CMV_betaglobin and lITR_phTERT_betaglobin into pSB1C3_CD</SPAN></A></LI><LI class="toclevel-2 tocsection-88"><A href="#Seeding_HT1080_and_A431_for_testing_different_concentrations_of_ganciclovir_by_MTT-Assay"><SPAN class="tocnumber">11.5</SPAN><SPAN class="toctext">Seeding HT1080 and A431 for testing different concentrations of ganciclovir by MTT-Assay</SPAN></A></LI><LI class="toclevel-2 tocsection-89"><A href="#FACS-Analysis"><SPAN class="tocnumber">11.6</SPAN><SPAN class="toctext">FACS-Analysis</SPAN></A></LI><LI class="toclevel-2 tocsection-90"><A href="#Preparation_of_the_ELISA"><SPAN class="tocnumber">11.7</SPAN><SPAN class="toctext">Preparation of the ELISA</SPAN></A></LI><LI class="toclevel-2 tocsection-91"><A href="#Trafo_evaluation_of_Rep52"><SPAN class="tocnumber">11.8</SPAN><SPAN class="toctext">Trafo evaluation of Rep52</SPAN></A></LI><LI class="toclevel-1 tocsection-92"><A href="#147._labday_12.10.2010"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">147. labday 12.10.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-93"><A href="#Sequencing_analysis:_pSB1C3_mGMK_TK30_and_pSB1C3_CD"><SPAN class="tocnumber">12.1</SPAN><SPAN class="toctext">Sequencing analysis: pSB1C3_mGMK_TK30 and pSB1C3_CD</SPAN></A></LI><LI class="toclevel-2 tocsection-94"><A href="#Test_digestion_of_pSB1C3_CD_and_pSB1C3_CD_SDM-PstI_new"><SPAN class="tocnumber">12.2</SPAN><SPAN class="toctext">Test digestion of pSB1C3_CD and pSB1C3_CD_SDM-PstI_new</SPAN></A></LI><LI class="toclevel-2 tocsection-95"><A href="#qPCR_of_virus_stock:_P816.2F_TKGMk-Standard.3B_R.2FC:_P468_from_5.10._.28VLP_harvested_by_Adrian.29"><SPAN class="tocnumber">12.3</SPAN><SPAN class="toctext">qPCR of virus stock: P816/ TKGMk-Standard; R/C: P468 from 5.10. (VLP harvested by Adrian)</SPAN></A></LI><LI class="toclevel-2 tocsection-96"><A href="#Cloning_sr39_into_mGMK_TK30"><SPAN class="tocnumber">12.4</SPAN><SPAN class="toctext">Cloning sr39 into mGMK_TK30</SPAN></A></LI><LI class="toclevel-2 tocsection-97"><A href="#Concentration_of_virus_stock_for_Western_Blot_analysis"><SPAN class="tocnumber">12.5</SPAN><SPAN class="toctext">Concentration of virus stock for Western Blot analysis</SPAN></A></LI><LI class="toclevel-2 tocsection-98"><A href="#MTT_Assay_.28First_Try.29_Transduction_of_HT1080_and_A431"><SPAN class="tocnumber">12.6</SPAN><SPAN class="toctext">MTT Assay (First Try) Transduction of HT1080 and A431</SPAN></A></LI><LI class="toclevel-2 tocsection-99"><A href="#Mini-Prep_and_test_digestion_of_several_constructs_2"><SPAN class="tocnumber">12.7</SPAN><SPAN class="toctext">Mini-Prep and test digestion of several constructs</SPAN></A></LI><LI class="toclevel-2 tocsection-100"><A href="#Repetiton:_Site-directed_mutagenesis_in_pSB1C3_CD_construct"><SPAN class="tocnumber">12.8</SPAN><SPAN class="toctext">Repetiton: Site-directed mutagenesis in pSB1C3_CD construct</SPAN></A></LI><LI class="toclevel-1 tocsection-101"><A href="#148._labday_13.10.2010"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">148. labday 13.10.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-102"><A href="#qPCRs_of_virus_stocks"><SPAN class="tocnumber">13.1</SPAN><SPAN class="toctext">qPCRs of virus stocks</SPAN></A></LI><LI class="toclevel-2 tocsection-103"><A href="#SDS_Page_of_unmodified_AAV2_capsids"><SPAN class="tocnumber">13.2</SPAN><SPAN class="toctext">SDS Page of unmodified AAV2 capsids</SPAN></A></LI><LI class="toclevel-2 tocsection-104"><A href="#Midi-Prep_2"><SPAN class="tocnumber">13.3</SPAN><SPAN class="toctext">Midi-Prep</SPAN></A></LI><LI class="toclevel-2 tocsection-105"><A href="#Repetiton:_Test_digestion_of_pSB1C3_lITR_phTERT_betaglobin_mGMK_TK30_SDM-PstI_hgH_rITR"><SPAN class="tocnumber">13.4</SPAN><SPAN class="toctext">Repetiton: Test digestion of pSB1C3_lITR_phTERT_betaglobin_mGMK_TK30_SDM-PstI_hgH_rITR</SPAN></A></LI><LI class="toclevel-2 tocsection-106"><A href="#Sequencing_analysis:_pSB1C3_leftITR_pCMV_betaglobin_mGMK_TK30_hgH_rITR_.28P825.29"><SPAN class="tocnumber">13.5</SPAN><SPAN class="toctext">Sequencing analysis: pSB1C3_leftITR_pCMV_betaglobin_mGMK_TK30_hgH_rITR (P825)</SPAN></A></LI><LI class="toclevel-2 tocsection-107"><A href="#Repetition_of_cloning_of_Rep_into_Rep78"><SPAN class="tocnumber">13.6</SPAN><SPAN class="toctext">Repetition of cloning of Rep into Rep78</SPAN></A></LI><LI class="toclevel-2 tocsection-108"><A href="#Midi-Prep_3"><SPAN class="tocnumber">13.7</SPAN><SPAN class="toctext">Midi-Prep</SPAN></A></LI><LI class="toclevel-1 tocsection-109"><A href="#149._labday_14.10.2010"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">149. labday 14.10.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-110"><A href="#Mini-Prep_and_test_digestion_of_several_constructs_3"><SPAN class="tocnumber">14.1</SPAN><SPAN class="toctext">Mini-Prep and test digestion of several constructs</SPAN></A></LI><LI class="toclevel-2 tocsection-111"><A href="#Continuation_of_Western_Blot_of_unmodified_AAV2_capsids"><SPAN class="tocnumber">14.2</SPAN><SPAN class="toctext">Continuation of Western Blot of unmodified AAV2 capsids</SPAN></A></LI><LI class="toclevel-2 tocsection-112"><A href="#Cloning_hGH_rITR_into_.28form_P728.29_into_pSB1C3_lITR_phTERT_betaglobin_mGMK_TK30_SDM-PstI_.28P822.29"><SPAN class="tocnumber">14.3</SPAN><SPAN class="toctext">Cloning hGH_rITR into (form P728) into pSB1C3_lITR_phTERT_betaglobin_mGMK_TK30_SDM-PstI (P822)</SPAN></A></LI><LI class="toclevel-2 tocsection-113"><A href="#Cloning_of_the_ViralBricks_453-empty_and_587_empty"><SPAN class="tocnumber">14.4</SPAN><SPAN class="toctext">Cloning of the ViralBricks 453-empty and 587 empty</SPAN></A></LI><LI class="toclevel-2 tocsection-114"><A href="#SDS_PAGE_of_viral_capsids"><SPAN class="tocnumber">14.5</SPAN><SPAN class="toctext">SDS PAGE of viral capsids</SPAN></A></LI><LI class="toclevel-2 tocsection-115"><A href="#Sequence_analysis_of_super_constructs"><SPAN class="tocnumber">14.6</SPAN><SPAN class="toctext">Sequence analysis of super constructs</SPAN></A></LI><LI class="toclevel-2 tocsection-116"><A href="#Transfection_with_pCMV.2C_p40.2C_pAAV_RC"><SPAN class="tocnumber">14.7</SPAN><SPAN class="toctext">Transfection with pCMV, p40, pAAV_RC</SPAN></A></LI><LI class="toclevel-2 tocsection-117"><A href="#Trafo_evaluation_of_Rep78"><SPAN class="tocnumber">14.8</SPAN><SPAN class="toctext">Trafo evaluation of Rep78</SPAN></A></LI><LI class="toclevel-2 tocsection-118"><A href="#Picking_clones"><SPAN class="tocnumber">14.9</SPAN><SPAN class="toctext">Picking clones</SPAN></A></LI><LI class="toclevel-2 tocsection-119"><A href="#Cloning_of_CD_into_pSB1C3_hGH_rITR"><SPAN class="tocnumber">14.10</SPAN><SPAN class="toctext">Cloning of CD into pSB1C3_hGH_rITR</SPAN></A></LI></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="136._labday_01.10.2010"><P style="font-size:17px; background-color:#00dd77;">136. labday 01.10.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Test_Digestions_of_yesterdays_minipreps"><P style="font-size:15px; background-color:#66bbff;"><B>Test Digestions of yesterdays minipreps</B></P></SPAN></H4><P><B>Investigator: Achim, Anna</B></P><UL><LI>001_CMV: Cut with EcoRI, PstI &amp; PvuII
</LI><LI>P654 (Lane 1)
</LI><LI>P655 (Lane 2)
</LI><LI>P686 (Lane 3)
</LI><LI>Control P320 (Lane 4)
</LI></UL><UL><LI>CMV_ZEGFR:1907_VP2/3_inscap: Cut with EcoRI &amp; SspI
</LI><LI>P678 (Lane 5)
</LI><LI>P679 (Lane 6)
</LI><LI>Control P514 (Lane 7)
</LI><LI><B>Results:</B>The expected bands should run at 1700 and 3000 bp. P679 was sent for sequencing.
</LI></UL><UL><LI>CMV_ZEGFR:1907_VP2/3_inscap_HSPG-KO: Cut with EcoRI &amp; SspI
</LI><LI>P680 (Lane 8)
</LI><LI>P681 (Lane 9)
</LI><LI>P684 (Lane 10)
</LI><LI>P685 (Lane 11)
</LI><LI>Control P613 (Lane 12)
</LI><LI><B>Results:</B>The expected bands should run at 1700 and 3000 bp. P684 was sent for sequencing.
</LI></UL><UL><LI>pCerulean_Vp1up_NLS_Darpin: Cut with BamHI &amp; XbaI
</LI><LI>P676 (Lane 13)
</LI><LI>P677 (Lane 14)
</LI><LI>P683 (Lane 15)
</LI><LI> Control P365 (Lane 20)
</LI><LI><B>Results:</B>The expected bands should run at 500 and 4500 bp. However, just one band at ~3000 bp is visible on the gel. 
</LI></UL><P>
Samples Stefan: Cut with Xba &amp; Spe
</P><UL><LI>P670 (Lane 16)
</LI><LI>P671 (Lane 17)
</LI><LI>P672 (Lane 18)
</LI><LI>P673 (Lane 19)
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">I guess that the pSB1C3_001_CMV did NOT work since the control with BLA looks nearly the same. We should repeat the experiment by cloning the CMV promoter into another vector. I would recommend it because we have a lot of constructs which we should fuse to the CMV promoter and in order to avoid the three-fragment ligation.</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="mini_preps_of_CD_clones"><P style="font-size:15px; background-color:#66bbff;"><B>mini preps of CD clones</B></P></SPAN></H4><P><B>Investigator: Kira</B></P><P>c(p689)= 115 ng/ul 
c(690)= 151, 20 ng/ul 
c(691)= 122,27 ng/ul 
c(p692) = 126,42 ng/ul </P><H4><SPAN class="mw-headline" id="test_digestion_of_CD_clones"><P style="font-size:15px; background-color:#66bbff;"><B>test digestion of CD clones</B></P></SPAN></H4><P><B>Investigator: Kira</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>sample</B> Volume/µL
</TD></TR><TR><TD align="left"> DNA   </TD><TD align="left"> 4,0 µl
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 2 µl
</TD></TR><TR><TD align="left"> Buffer no. 4  </TD><TD align="left"> 2,0 µl
</TD></TR><TR><TD align="left"> Enzyme 1 XbaI </TD><TD align="left"> 1,0 µl
</TD></TR><TR><TD align="left"> Enzyme 2 AgeI  </TD><TD align="left"> 1,5 µl
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 9,5 µl
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>20</B></TD></TR></TBODY></TABLE><P>incubation @ 37 C for approx. 2 h
</P><P>1% agarose gel </P><H4><SPAN class="mw-headline" id="Biobrick_assembly_pSB1C3_lITR_hTERT_beta-globin_CD"><P style="font-size:15px; background-color:#66bbff;"><B>Biobrick assembly pSB1C3_lITR_hTERT_beta-globin_CD</B></P></SPAN></H4><P><B>Investigator: Kira</B></P><P>c(pSB1C3_lITR_hTERT_beta-globin)= 333 ng/ul 
c(pSB1C3_CD)= 151 ng/ul </P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>vector</B> Volume/µL </TD><TD align="left"><B>insert</B> Volume/µL
</TD></TR><TR><TD align="left"> DNA   </TD><TD align="left"> 4,5 µl</TD><TD align="left"> 6
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 3 µl</TD><TD align="left"> 3
</TD></TR><TR><TD align="left"> Buffer no. 4  </TD><TD align="left"> 3,0 µl</TD><TD align="left"> 3
</TD></TR><TR><TD align="left"> Enzyme 1 XbaI </TD><TD align="left"> 0 µl</TD><TD align="left">1,5
</TD></TR><TR><TD align="left"> Enzyme 2 SpeI  </TD><TD align="left"> 1,5 µl</TD><TD align="left"> 0
</TD></TR><TR><TD align="left"> Enzyme 3 PstI-HF </TD><TD align="left"> 1,0 </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 17</TD><TD align="left"> 15,5
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>25</B></TD></TR></TBODY></TABLE><P>incubation @ 37 C for approx. 2 h
</P><P>1% agarose gel </P><P><B>Ligation</B></P><P>DNA-mix: 8 ul (vector 4,6ul)+(insert 3,4 ul) 
T4 ligase: 1 ul 
T4 buffer: 1 ul 
Incubation @ RT for 30 min </P><P>Transformation was performed according to the standard protocol w BL21 cells. </P><H4><SPAN class="mw-headline" id="Sequencing_results_of_pCerulean_Zegfr:1907_MiddleLinker_VP2.2F3_HSPG-KO"><P style="font-size:15px; background-color:#66bbff;"><B>Sequencing results of pCerulean_Zegfr:1907_MiddleLinker_VP2/3_HSPG-KO</B></P></SPAN></H4><P><B>Investigator: Hanna</B><B>Comment:</B> All N-terminal fusion approaches with VP2/3_HSPG-KO revealed positive results except of pCerulean_Zegfr:1907_MiddleLinker_VP2/3_HSPG-KO. Another clone was picked, preped, test digested and sent for sequencing. </P><P><B>Conclusion:</B> Sequencing results revealed positive results. </P><H4><SPAN class="mw-headline" id="Picking_clones_of_pGA14_MiddleLinker_VP2.2F3_insCap_and_pGA14_MiddleLinker_VP2.2F3_HSPG-KO"><P style="font-size:15px; background-color:#66bbff;"><B>Picking clones of pGA14_MiddleLinker_VP2/3_insCap and pGA14_MiddleLinker_VP2/3_HSPG-KO</B></P></SPAN></H4><P><B>Investigator: Hanna</B>
Unfortunately there grew a bacteria lawn over night - it was hardly not possible to pick clones. Nevertheless I tried and picked 2 clones of pGA14_MiddleLinker_VP2/3_insCap and pGA14_MiddleLinker_VP2/3_HSPG-KO. 
<B>To do: </B> Mini-Prep and test digestion.
</P><H4><SPAN class="mw-headline" id="Sequencing_results_of_pSB1C3_001_VP2.2F3_587-KO_BAP_and_pSB1C3_001_VP2.2F3_587-KO_6xHis"><P style="font-size:15px; background-color:#66bbff;"><B>Sequencing results of pSB1C3_001_VP2/3_587-KO_BAP and pSB1C3_001_VP2/3_587-KO_6xHis</B></P></SPAN></H4><P><B>Investigator: Hanna</B><B>Comment:</B> For the creation of our super constructs, the His-Tag and the BAP motif need to be cloned into VP2/3 for N-terminal fusion to VP2. Sequencing results showed that the 6xHis and BAP motif was<B> not</B> cloned into VP2/3_insCap.
</P><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Sample gallery</CAPTION></TABLE><P><B>To do:</B> Clone 587-KO_6xHis and 587-KO_BAP into pSB1C3_001_VP2/3_insCap 1. via digestion of inserts and 2. via hybridization of referring oligos - digestion of vector with 800-900 ng.
</P><H4><SPAN class="mw-headline" id="Impressions_of_transfection_of_AAV293_with_pCerulean_VP1up_NLS_mVenus_VP2.2F3"><P style="font-size:15px; background-color:#66bbff;"><B>Impressions of transfection of AAV293 with pCerulean_VP1up_NLS_mVenus_VP2/3</B></P></SPAN></H4><P><B>Investigator: Adrian</B>
Yesterday AAV293 cells were transfected with pCerulean_VP1up_NLS_mVenus_VP2/3 and GMK-TK30 as gene of interest. Today, nuclear localization of the produced mVenus_VP2/3 fusion protein was visible and can be nicely seen in the following pictures: </P><P><B>Conclusion:</B> The VP2/3 fusion particle was transcribed and translated AND was transported back into the nucleus in order to be packaged by the ITR-flanked gene of interest. </P><H4><SPAN class="mw-headline" id="Cloning_of_hGH_rITR_into_pSB1C3_lITR_phTERT_beta-globin_CFP_and_lITR_phTERT_beta-globin_mGMK_TK30_into_pSB1C3_hGH_rITR"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of hGH_rITR into pSB1C3_lITR_phTERT_beta-globin_CFP and lITR_phTERT_beta-globin_mGMK_TK30 into pSB1C3_hGH_rITR</B></P></SPAN></H4><P><B>Investigator: Stefan </B></P><P>Vector name:
</P><UL><LI>pSB1C3_lITR_phTERT_beta-globin_CFP (P682)</LI><LI>pSB1C3_hGH_rITR (P186)</LI></UL><P>Insert name:
</P><UL><LI>pSB1C3_hGH_rITR (P186)</LI><LI>pSB1C3_lITR_phTERT_beta-globin_mGMK_TK30 (P672)</LI><LI>pSB1C3_lITR_phTERT_beta-globin_mGMK_TK30 (P673)</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume for P186 /µl</B></TD><TD align="right"><B>volume of P186 X+E /µl</B></TD><TD align="right"><B>volume of P672 + P673 /µl</B></TD><TD align="right"><B>volume of P682 /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">12 </TD><TD align="right">12</TD><TD align="right">5 </TD><TD align="right">14
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2,5 </TD><TD align="right">2</TD><TD align="right"> 2</TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">3 </TD><TD align="right">2 </TD><TD align="right"> 2</TD><TD align="right"> 2
</TD></TR><TR><TD>Enzyme XbaI</TD><TD align="right">1</TD><TD align="right">1 </TD><TD align="right">-</TD><TD align="right">-
</TD></TR><TR><TD>Enzyme PstI</TD><TD align="right">1</TD><TD align="right">- </TD><TD align="right">-</TD><TD align="right">1
</TD></TR><TR><TD>Enzyme EcoI</TD><TD align="right">-</TD><TD align="right">1 </TD><TD align="right">1</TD><TD align="right">-
</TD></TR><TR><TD>Enzyme SpeI</TD><TD align="right">-</TD><TD align="right">- </TD><TD align="right">1</TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">6</TD><TD align="right">6</TD><TD align="right">9</TD><TD align="right">-
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 25</TD><TD align="right"> 25</TD><TD align="right">20</TD><TD align="right">20
</TD></TR></TBODY></TABLE><P><B>Gel:</B>
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 115 Volt</P><P style="color:#66bbff;"><I>Comment</I>: Since the first gel revealed problems with samples 672, 673 and 682 another digestion was prepared (same approach as shown above), this time using a 0,8% gel and another loading dye (without SDS).</P><P>0,4 g Agarose, 50 ml TAE (0,8%), 3 µl GELRED , at 115 Volt</P><P style="color:#66bbff;"><I>Comment</I>: Since sample 672 showed no bands this sample was discarded and cloning continued only using 673.</P><P><B>Gel extraction</B>: 
Was performed according to protocol.
</P><P><B>T4 Ligation</B>: 
Ligation was performed over night.
</P><TABLE border="1"><TBODY><TR><TD>ligation name </TD><TD align="right">186 (X+E) + 673</TD><TD align="right">186 + 682
</TD></TR><TR><TD>volume of vector </TD><TD align="right">3,51 </TD><TD align="right"> 4,49
</TD></TR><TR><TD>volume of insert</TD><TD align="right">5,38 </TD><TD align="right">2,64
</TD></TR></TBODY></TABLE><P><B>Transformation</B>: 
Will be done tomorrow using BL21 cells.
</P><H4><SPAN class="mw-headline" id="New_BL21_cells_ready_to_use.21"><P style="font-size:15px; background-color:#66bbff;"><B>New BL21 cells ready to use!</B></P></SPAN></H4><P><B>Investigator: Stefan </B></P><H3><SPAN class="mw-headline" id="137._labday_02.10.2010"><P style="font-size:17px; background-color:#00dd77;">137. labday 02.10.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Repetition_of_the_approach_of_subcloning_the_ViralBricks_HSPG-ko_BAP_and_His_into_VP2.2F3_with_different_strategies"><P style="font-size:15px; background-color:#66bbFF;"><B>Repetition of the approach of subcloning the ViralBricks HSPG-ko BAP and His into VP2/3 with different strategies</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="color:#66bbff;"><I>Comment</I>: Unfortunately, the last experiment revealed that subcloning of the two ViralBricks did not work out which was shown by the sequencing results. Therefore we need to repeat the approach of subcloning the ViralBricks into the pSB1C3_VP2/3 which than can be used for fusing it to the n-terminal targeting approaches. Two different strategies will be carried out which means that we are digesting the pSB1C3_Viralbricks AND hybridize the oligos (BAP and His) in order to obtain some positive results.</P><P>The vector (P415) and the two ViralBricks BAP-ko and His-ko were digested with BamHi and PvuII. The vector was dephosphorylated following the standard protocol. 
The vector was loaded on a 1% agarose gel before dephosphorylation was conducted. The results can be seen above in the gel picture: </P><P><B>Result: </B></P><P>The ligation was performed over night at 18°C with T4-ligase.
</P><H4><SPAN class="mw-headline" id="BioBrick_Assembly_of_CD_and_mCherry"><P style="font-size:15px; background-color:#66bbFF;"><B>BioBrick Assembly of CD and mCherry</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="color:#66bbff;"><I>Comment</I>: We decided to choose some additional GOI´s for the vectorplasmid so the &quot;new&quot; GOI´s need to be fused to the leftITR_Promoter_betaglobin constructs. Since only one clone grew on the plate prepared by Kira where she fused the CD (cytosine deaminase) to the above mentioned construct I am going to repeat it aswell. </P><P><B>Digestion of the constructs:</B></P><UL><LI>P689 = pSB1C3_CD</LI><LI>P626 = iGEM_mCherry</LI><LI>P434 = pSB1C3_leftITR_CMV_betaglobin</LI><LI>P434 = pSB1C3_leftITR_phTERT_betaglobin</LI><LI>P525 = pSB1C3_VP2/capins</LI></UL><P><B>Loading plan: </B></P><PRE>P689 (pSB1C3_CD)     P626 (iGEM mCherry)     P434 (pSB1C3_lITR_CMV_ß-globin)     P437 (pSB1C3_lITR_phTERT_ß-globin)
</PRE><P><B>Results:</B></P><P>
After gel extraction has been performed, the ligation was carried out.
</P><P><B>Ligation:</B></P><P>
The ligation mix was transformed into XL1-Blue cells and plated on agar plates containing chloramphenicol.
</P><P><B>Next steps:</B>
Picking clones and perform Mini-Prep. After the Mini-Preps have been performed this construct needs to be sequenced and tested in cell culture.
</P><P><B>comment:</B> just one clone? uhhhh, when did you take out the plate? i put it at around 1 am..maybe they need a while to grow, no? (Kira)  Yep, I found another clone ;-) But the two look pretty good. I took them out today  in the afternoon - I guess that is enough time! But if you want I can put them back into the 37° C room??! (Bea)
</P><H4><SPAN class="mw-headline" id="Mini_Prep_and_test_digestion_of_pGa14_MiddleLinker_VP2.2F3_HSPG-KO"><P style="font-size:15px; background-color:#66bbff;"><B>Mini Prep and test digestion of pGa14_MiddleLinker_VP2/3_HSPG-KO</B></P></SPAN></H4><P><B>Investigator: Hanna</B><B>Comment:</B> In order to fuse the DARPin to the N-terminus of VP2/3_insCap and VP2/3_HSPG-KO, these motifs need to be fused to the Middle Linker (performed on 30.9.2010). In order to be able to immediately continue with cloning, BL21 cells were transformed and clones were picked in the noon. Now, the DNA needs to be preped and test digested in order to perform a 3-fragment-ligation with the DARPin and the pSB1C3_001_CMV plasmid. Unfortunately the cells of the VP2/3_insCap construct grew not densely enough. Therefore two new clones were picked from the plate and were inoculated in 10 mL LB medium. The other construct (pGa14_MiddleLinker_VP2/3_HSPG-KO) was preped and test digested.
</P><P><B>TEST DIGESTION:</B> 
For both clones: </P><UL><LI> DNA: 4 µL
</LI><LI> Buffer 4: 1 µL
</LI><LI> BSA: 1 µL
</LI><LI> EcoRI: 0.5 µL
</LI><LI> PstI: 0.5 µL
</LI><LI> H2O: 3 µL
</LI></UL><P>
Incubation: 1 hour. 
Agarose gel: 0.8 %, 115 Volt, 1 hour. <B>Comment:</B> The choice of these restriction enzymes was not clever, because if cloning didn't work, I expect one band at 2379 bp (the insert should hardly be seen) - if it worked one band at 2379 bp and a second at 2005 bp. Fortunately the bands could be separated ans revealed positive results for both clones : <B>Next steps:</B></P><UL><LI> Mini-Prep and test digestion of pGa14_MiddleLinker_VP2/3_ins Cap.
</LI><LI> 3 fragment ligation of MiddleLinker_VP2/3_ins Cap or MiddleLinker_VP2/3_HSPG-KO + DARPin + pSB1C3_001_CMV backbone.
</LI></UL><H4><SPAN class="mw-headline" id="Sequencing_analysis_of_HSPG-ko_in_several_IRCK_constructs"><P style="font-size:15px; background-color:#66bbff;"><B>Sequencing analysis of HSPG-ko in several IRCK constructs</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P><B>pSB1C3_001_RC_IRCK_HSPG-ko_P5tataless_RFC10 clone 1 (P687):</B></P><P><B>pSB1C3_001_RC_IRCK_HSPG-ko_P5tataless_RFC10 clone 2(P688):</B></P><P><B>pSB1C3_001_RC_IRCK_VP1-ko_HSPG-ko_P5tataless cl1 (P644):</B></P><P><B>pSB1C3_001_RC_IRCK_VP2-ko_HSPG-ko_P5tataless cl1 (P646):</B></P><P style="color:#66bbff;"><I>Comment</I>: Sequencing results show that all plasmids contain the HSGP-ko. Working will be continued using P644, P646 and P687.</P><H4><SPAN class="mw-headline" id="Mini_Prep_of_pGa14_MiddleLinker_VP2.2F3_inscap_and_pSB1C3_Darpin"><P style="font-size:15px; background-color:#66bbff;"><B>Mini Prep of pGa14_MiddleLinker_VP2/3_inscap and pSB1C3_Darpin</B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><UL><LI>pSB1C3_Darpin clone1 c=158.5 ng/µl
</LI><LI>pSB1C3_Darpin clone2 c=205.6 ng/µl
</LI><LI>pGA14_MiddleLinker_VP2/3_inscap clone 1 c=147.5 ng/µl
</LI><LI>pGA14_MiddleLinker_VP2/3_inscap clone 2 c=121.0 ng/µl
</LI></UL><P><B>Test-digestion:</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>pSB1C3_Darpin clone1/2</B></TD></TR><TR><TD align="left"> DNA   </TD><TD align="left"> 1,5 µl
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 1 µl
</TD></TR><TR><TD align="left"> Buffer no. 4  </TD><TD align="left"> 1 µl
</TD></TR><TR><TD align="left"> Enzyme 1 XbaI </TD><TD align="left"> 0,4 µl
</TD></TR><TR><TD align="left"> Enzyme 2 AgeI  </TD><TD align="left"> 0,4 µl
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 5,7 µl
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>10</B></TD></TR></TBODY></TABLE><P>
Hanna wrote that a 3rd enzyme is need for the test digestion of pGA14_MiddleLinker_VP2/3_inscap but I couln't find any sequence of pGA14_MiddleLinker_VP2/3_inscap or pGA14</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">You can either find these constructs in my workingfolder &quot;DARPin&quot; or the pGA14_MiddleLinker in the &quot;Linker-Folder&quot; and the VP2/3_insCap in many workingfolders!!!! You could have also digest the construct with EcoRI and SpeI... then you would expect a 2 and a 2.3 kb fragment... menas: you have to run the gel at least 1 hour - but it works (see the previous test digestion of the MiddleLinker_VP2/3_insCap construct!!!!).</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Test_digestion_of_pGA14_MiddleLinker_VP2.2F3_inscap"><P style="font-size:15px; background-color:#66bbff;"><B>Test digestion of pGA14_MiddleLinker_VP2/3_inscap </B></P></SPAN></H4><P><B>Investigator: Achim</B></P><UL><LI>pGA14_MiddleLinker_VP2/3_inscap clone 1 c=147.5 ng/µl (P698)
</LI><LI>pGA14_MiddleLinker_VP2/3_inscap clone 2 c=121.0 ng/µl (P699)
</LI><LI>Control: pGA14_MiddleLinker (P301)
</LI></UL><UL><LI>Digestion with EcoRI &amp; SalI; should yield two bands of 3400 &amp; 1000 bp. The negative control should only be cut once.
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:342px;"><DIV class="thumbcaption">Both samples show correctly sized bands. The negative control ran further than expected, it should be located at 4500 bp.</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Mass_Midi-Prep_III"><P style="font-size:15px; background-color:#66bbff;">Mass Midi-Prep III </P></SPAN></H4><P><B>Investigator: Chris W. </B></P><P style="font-size:13px; color:#003399;"> Midi-Prep of:</P><P>pSB1C3_001_RC_IRCK_VP1-ko_HSPG-ko_P5tataless cl1 =P700 =B521
pSB1C3_001_RC_IRCK_VP2-ko_HSPG-ko_P5tataless cl1 =P701 =B523
pSB1C3_001_RC_IRCK_HSPG-ko_P5tataless_RFC10 clone 1 =P702 =B561
pCerulean_ZEGFR:1907_MiddleLinker_VP2/3_HSPG-KO clone 2 =P703 =B543
pCerulean_CFP_MiddleLinker_VP2/3_HSPG-KO =P704 =B507
pCerulean_ZEGFR:1907_SEG_VP2/3_HSPG-KO =P705 =B509
pCerulean_ZEGFR:1907_ShortLinker_VP2/3_HSPG-KO =P706 =B510
pCerulean_ZEGFR:1907_LongLinker_VP2/3_HSPG-KO =P707 =B511
pCerulean_6xHis_MiddleLinker_VP2/3_HSPG-KO =P708 =B512
pCerulean_VP1up_NLS_ZEGFR:1907_VP2/3_HSPG-KO =P709 =B513
pCerulean_VP1up_NLS_6xHis_VP2/3_HSPG-KO =P710 =B514
pCerulean_VP1up_NLS_mVenus_VP2/3_HSPG-KO =P711 =B515 
pSB1C3_001_RC_IRCK_P5tataless clone 1 =P712 =B516</P><P>
The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
</P><TABLE border="1"><TBODY><TR><TD> plasmid-no. </TD><TD align="right">P700</TD><TD align="right">P701</TD><TD align="right">P702</TD><TD align="right">P703</TD><TD align="right">P704</TD><TD align="right">P705</TD><TD align="right">P706</TD><TD align="right">P707</TD><TD align="right">P708</TD><TD align="right">P709</TD><TD align="right">P710</TD><TD align="right">P711</TD><TD align="right">P712
</TD></TR><TR><TD> concentration (ng/µl)</TD><TD align="right">695 </TD><TD align="right">370 </TD><TD align="right">601</TD><TD align="right">1548</TD><TD align="right">3996</TD><TD align="right">3630</TD><TD align="right">2460</TD><TD align="right">2610</TD><TD align="right">1860</TD><TD align="right">2913</TD><TD align="right">2964</TD><TD align="right">3030</TD><TD align="right">1200
</TD></TR></TBODY></TABLE></DIV><H4><SPAN class="mw-headline" id="Analysis_of_viral_harvest_via_spectrometer"><P style="font-size:15px; background-color:#66bbff;">Analysis of viral harvest via spectrometer</P></SPAN></H4><P><B>Investigator: Adrian, Patrick</B><B>The motivation</B>: We want to figure out if it is possible to purify our viral stocks via centrifugation with 10.000 g and/or 20.000 g. In theory/ according to the literature the viral particles should be in the cells and attached to the HSPGs at the cellular surfaces.
<B>The plan</B>: Two different viral stocks were prepared:

Stock 1 with VP2-mVenus-fusion-Capsid loaded with TKGMK (4 wells from a 6 well plate)</P><UL><LI>R/C 1: 50% pCerulean_VP1up_NLS_mVenus_Vp2/3 P501</LI><LI>R/C 2: 50% Rep/Cap2: P449 pAAV_RC_4fachmut_VP1-ko:</LI><LI>GOI: TKGMK: P82.b</LI><LI>pHelper</LI></UL><P>
Stock 2 &quot;standard&quot; virus (4 wells from a 6 well plate)</P><UL><LI>R/C 1 P486</LI><LI>GOI: TKGMK: P82.b</LI><LI>pHelper</LI></UL><P>These two stocks got harvested according to standard protocol (scratching cells, transfer them in to 15 ml falcons, resuling 12ml solution total). After centrifugation (10 min 200g), the supernatant got transferred into two new 15 ml falcons and the pellets were resuspended with 10 ml DMEM Medium. The Stocks got freezed thawed two times so finally there were:
</P><OL><LI>DMEM as negative control</LI><LI>resuspended Pellet with mVenus-fusioned viral capsids</LI><LI>resuspended Pellet with &quot;standard virus&quot;</LI><LI>supernatant with mVenus-fusioned viral capsids</LI><LI>supernatant with &quot;standard virus&quot;</LI></OL><P>
The fluorescence of each stock was measured via spectrometer. After that, stocks 2 and 4 were centrifugated 10min with 10.000 g followed by an other fluorescence measuring. Finally the stocks got spin down with 20.000 g for 10 min
<B>The results</B>:
</P><P><B>The conclusions</B>: 
The highest amount of fluorescence was measured in the supernatant, the individual centrifugation steps had <B>no decrease in fluorescence</B> as consequence!
</P><P>
Keep in mind, without further investigation it is not valid to say that we can purify our stocks via 10-20.000 g centrifugation steps, because we dont know if the viral capsids are still intact! The FACS analysis are not sufficient enough to make a decision, because our last samples had YFP as GOI (=&gt; so efficient cell sorting was not possible). We need to do qPCRs to make a valid evidence.
</P><H4><SPAN class="mw-headline" id="200.C2.B5l-Transduction_for_testing_the_pTERT-promoter"><P style="font-size:15px; background-color:#66bbff;">200µl-Transduction for testing the pTERT-promoter</P></SPAN></H4><P><B>Investigator: Adrian, Patrick </B>
We transduced A431, HT1080 and AAV293 cells with 4 different viral stocks:</P><OL><LI>control vector from 15.9, 750.000 cells, 3.3 µg approach</LI><LI>P25 R/C= 326 pTERT_mVenus</LI><LI>P26 pAAV_RC_RepCapIns_SDMKpnI clone 1 P431 pTERT_mVenus</LI><LI>His-linker-fusion</LI></OL><P><FONT size="4" color="#FF0000">CRUCIAL: There was not enough viral stock: so the whole Transduction was performed with 200µl instead of 1000µl</FONT>
There are 45 samples to FACS on monday 4.10. Tripple values for each stock of each cell lines.
</P><H4><SPAN class="mw-headline" id="Transfection"><P style="font-size:15px; background-color:#66bbff;">Transfection</P></SPAN></H4><P><B>Investigators: Adrian, Patrick</B>
Transfection followed standard protocol (3.3 µg of each plasmid, 1 ml 0.3M CaCl<SUB>2</SUB> and 1 ml of 2xHBS buffer PH 11.12) but with 20 min incubation time (Ca-DNA-2xHBS)

following stocks:
</P><UL><LI>mVenus standard stock 10 x cm R/C: P468<SUP>2</SUP> plates</LI><LI>TKGMK stock 10 x cm<SUP>2</SUP> plates R/C: P468</LI><LI>viral particles with GOI (mVenus) missing HGH one 6well plate</LI><LI>viral particles with GOI (mVenus) missing beta globin one 6well plate</LI><LI>viral particles with R/C missing HSPG binding motif (P687) one 6well plate</LI><LI>viral particles with new standard R/C (P668) one 6well plate</LI></UL><H3><SPAN class="mw-headline" id="138._labday_03.10.2010"><P style="font-size:17px; background-color:#00dd77;">138. labday 03.10.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Proceeding_with_the_approach_of_subcloning_the_ViralBricks_HSPG-ko_BAP_and_His_into_VP2.2F3_with_different_strategies"><P style="font-size:15px; background-color:#66bbFF;"><B>Proceeding with the approach of subcloning the ViralBricks HSPG-ko BAP and His into VP2/3 with different strategies</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="color:#66bbff;"><I>Comment</I>: Ligation was carried out over night and transformation will be performed today.</P><P>Since the digested vector was not sufficient for all four ligation I decided to perform the ligation only with the hybridized oligos 587-ko His and the digested Viralbricks 587-HSPG-ko His and BAP. If the digested BAP insert reverals no positive results, the hybrizied oligos for BAP can be used for another approach. 
The transformation was carried out in 
</P><UL><LI>BL-21 cells</LI><LI>and plated on agar plastes containing chloramphenicol</LI></UL><H4><SPAN class="mw-headline" id="Midi-Prep_of_pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR_clone1"><P style="font-size:15px; background-color:#66bbff;"> Midi-Prep of pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR clone1 </P></SPAN></H4><P><B>Investigator: Chris W. </B></P><P style="font-size:13px; color:#003399;"> Midi-Preps of pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR clone1 =P713 =B200 </P><P>
The Midi-Prep were performed according to the standard protocol yielding the following concentrations:
</P><TABLE border="1"><TBODY><TR><TD> plasmid-no. </TD><TD align="right">P713
</TD></TR><TR><TD> concentration (ng/µl)</TD><TD align="right">1248,3
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="139._labday_04.10.2010"><P style="font-size:17px; background-color:#00dd77;">139. labday 04.10.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Three_fragment_ligation:_pSB1C3_001_CMV_VP1up_NLS_Targeting_HSPG-KO_VP2.2F3"><P style="font-size:15px; background-color:#66bbFF;"><B>Three fragment ligation: pSB1C3_001_CMV_VP1up_NLS_Targeting_HSPG-KO_VP2/3</B></P></SPAN></H4><P><B>Investigator: Achim</B></P><P>Four ligations in total: 
</P><UL><LI>Cloning CMV &amp; VP1up_NLS_mVenus_HSPG-KO_VP2/3 into pSB1C3_001
</LI><LI>Cloning CMV &amp; VP1up_NLS_His_HSPG-KO_VP2/3 into pSB1C3_001
</LI><LI>Cloning CMV &amp; VP1up_NLS_affibody_HSPG-KO_VP2/3 into pSB1C3_001
</LI><LI>Cloning CMV into pSB1C3_001
</LI></UL><P>Used plasmids:
</P><UL><LI>pSB1C3_CMV (P145)
</LI><LI>c=225.5 ng/µl -&gt; 8 µl
</LI><LI>Cut with EcoRI &amp; SpeI
</LI></UL><LI>pCerulean_VP1up_NLS_mVenus_HSPG-KO_VP2/3 (P711)
</LI><LI>c=3030 ng/µl -&gt; 0.7 µl
</LI><LI>Cut with XbaI &amp; PstI
</LI><LI>pCerulean_VP1up_NLS_His_HSPG-KO_VP2/3 (P710)
</LI><LI>c=2965 ng/µl -&gt; 0.7 µl
</LI><LI>Cut with XbaI &amp; PstI
</LI><LI>pCerulean_VP1up_NLS_affibody_HSPG-KO_VP2/3 (P709)
</LI><LI>c=2913 ng/µl -&gt; 0.7 µl
</LI><LI>Cut with XbaI &amp; PstI
</LI><LI>pSB1C3_001_BLA (P320)
</LI><LI>c=408.5 ng/µl -&gt; 5 µl
</LI><LI>Cut with EcoRI &amp; PstI for three fragment ligations
</LI><LI>Cut with EcoRI &amp; SpeI for ligation with CMV
</LI><H4><SPAN class="mw-headline" id=".C2.B4Seeding_AAV293_for_Transfection"><P style="font-size:15px; background-color:#66bbFF;"><B>´Seeding AAV293 for Transfection</B></P></SPAN></H4><P><B>Investigator: Patrick</B></P><P>9 6-well plates were prepared for the transfection tomorrow. 480000 cells were seeded into each well.
</P><H4><SPAN class="mw-headline" id="FACS_Analysis_of_pTERT_mVenus-loaded-viral_particles_and_pCerulean_VP1up_His_VP2.2F3"><P style="font-size:15px; background-color:#66bbFF;"><B>FACS Analysis of pTERT_mVenus-loaded-viral particles and pCerulean_VP1up_His_VP2/3</B></P></SPAN></H4><P><B>Investigators: Kerstin, Anissa</B>
45 samples of the three cell lines: AAV293, A431 and HT1080 got transduced (see 2.10) with following viral stocks:1.</P><UL><LI>pHelper</LI><LI>R/C : 326</LI><LI>pTERT-mVenus</LI></UL><P>2.</P><UL><LI>pHelper </LI><LI>R/C : P431</LI><LI>pTERT-mVenus</LI></UL><P>3.</P><UL><LI>pHelper </LI><LI>R/C P158a</LI><LI>CMV_mVenus</LI></UL><P>4.</P><UL><LI>pHelper </LI><LI>R/C: pCerulean_VP1up_His_VP2/3</LI><LI>pAAV_RC_4fachmut_VP1-ko</LI><LI>CMV_mVenus </LI></UL><H5><SPAN class="mw-headline" id="Assembly_of_pSB1C3_CMV_DARPin_VP2.2F3_insCap_and_pSB1C3_CMV_DARPin_VP2.2F3_HSPG-KO"><P style="font-size:15px; background-color:#66bbFF;"><B>Assembly of pSB1C3_CMV_DARPin_VP2/3_insCap and pSB1C3_CMV_DARPin_VP2/3_HSPG-KO</B></P></SPAN></H5><P><B>Investigators: Hanna</B><B>Comment:</B> We also want to test the DARPin_E01 for the N-terminal fusion approaches. Today a 3 fragment ligation will be performed in order to complete the VP2 fusion constructs with the DARPin.
Because 2 test digestions showed that the fusion of the DARPin to VP1up_NLS didn't work as ecpected (insert is too large) this has to be repeated before also the VP1 insertion with the DARPin can be completed. </P><P style="font-size:15px; font-weight: bold; color: blue;">Digestion</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of pSB1C3_CMV /µl </TD><TD align="right">volume of pG14_ML_VP2/3_insCap /µl </TD><TD align="right">volume of pG14_ML_VP2/3_HSPG-KO /µl </TD><TD align="right">volume of pSB1C3_001_DARPin /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 9 </TD><TD align="right"> 6.8 </TD><TD align="right"> 10 </TD><TD align="right"> 5
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right"> 2 </TD><TD align="right"> 3 </TD><TD align="right"> 3 </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right"> 2 </TD><TD align="right"> 3 </TD><TD align="right"> 3 </TD><TD align="right"> 2
</TD></TR><TR><TD>SpeI</TD><TD align="right"> 1 </TD><TD align="right"> - </TD><TD align="right"> - </TD><TD align="right"> -
</TD></TR><TR><TD>PstI</TD><TD align="right"> 1 </TD><TD align="right"> 1 </TD><TD align="right"> 1 </TD><TD align="right"> -
</TD></TR><TR><TD>NgoMIV</TD><TD align="right"> - </TD><TD align="right"> 1 </TD><TD align="right"> 1 </TD><TD align="right"> -
</TD></TR><TR><TD>XmnI</TD><TD align="right"> - </TD><TD align="right"> 1 </TD><TD align="right"> 1 </TD><TD align="right"> -
</TD></TR><TR><TD>XbaI</TD><TD align="right"> - </TD><TD align="right"> - </TD><TD align="right"> - </TD><TD align="right"> 4
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right"> 5 </TD><TD align="right"> 14.2 </TD><TD align="right"> 11 </TD><TD align="right"> 9
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 20 </TD><TD align="right"> 30 </TD><TD align="right"> 30 </TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><UL><LI>Incubation: 2.5 h
</LI></UL><P style="font-size:15px; font-weight: bold; color: blue;">Agarose-Gel:</P><P>
0.5 g Agarose, 50 mL TAE (1 %), 3 µL GELRED, at 115 Volt, running time: 45 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (6x)/µl
</TH><TH>Expected size 1
</TH></TR><TR><TD>CMV
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD>~ 2700 bp
</TD></TR><TR><TD>MiddleLinker_VP2/3_insCap
</TD><TD>30 µl
</TD><TD>4 µl
</TD><TD>~ 2000 bp
</TD></TR><TR><TD>MiddleLinker_VP2/3_HSPG-KO
</TD><TD>30 µl
</TD><TD>4 µl
</TD><TD>~ 2000 bp
</TD></TR><TR><TD>DARPin
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD>~ 475 bp
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color: blue;">Gelex</P><UL><LI> CMV: 48.99 ng/µL
</LI><LI> MiddleLinker_VP2/3_insCap: 12.95 ng/µL
</LI><LI> MiddleLinker_VP2/3_HSPG-KO ng/µL
</LI><LI> DARPin: 9.59 ng/µL
</LI></UL><P style="font-size:15px; font-weight: bold; color: blue;">Ligation</P><TABLE border="1"><TBODY><TR><TD><B>Construct</B></TD><TD><B>CMV (µl)</B></TD><TD><B>MiddleLinker_VP2/3_insCap (µl)</B></TD><TD><B>DARPin (µl)</B></TD></TR><TR><TD> pSB1C3_CMV_DARPin_MiddleLinker_VP2/3_insCap
</TD><TD> 1
</TD><TD> 3.5
</TD><TD> 3.5
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD><B>Construct</B></TD><TD><B>CMV (µl)</B></TD><TD><B>MiddleLinker_VP2/3_HSPG-KO (µl)</B></TD><TD><B>DARPin (µl)</B></TD></TR><TR><TD> pSB1C3_CMV_DARPin_MiddleLinker_VP2/3_HSPG-KO
</TD><TD> 1
</TD><TD> 3.5
</TD><TD> 3.5
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color: blue;">Trafo</P><P>
Was performed following the standard protocol. 
Cells: XL1b
</P><H4><SPAN class="mw-headline" id="Miniprep_of_serveral_constructs"><P style="font-size:15px; background-color:#66bbFF;"><B>Miniprep of serveral constructs</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P><B>Glycerol stocks were prepared:</B>
B573 pSB1C3_lITR_CMV_betaglobin_mCherry clone1
B574 pSB1C3_lITR_CMV_betaglobin_mCherry clone2	
B575 pSB1C3_lITR_CMV_betaglobin_CD clone1	
B576 pSB1C3_lITR_CMV_betaglobin_CD clone2
B577 pSB1C3_lITR_phTERT_betaglobin_mCherry clone1
B578 pSB1C3_lITR_phTERT_betaglobin_mCherry clone2
B579 pSB1C3_lITR_phTERT_betaglobin_CFP_hgH_rITR clone 1
B580 pSB1C3_lITR_phTERT_betaglobin_CFP_hgH_rITR clone 2
B581 pSB1C3_lITR_phTERT_betaglobin_CFP_hgH_rITR clone 1	
B582 pSB1C3_lITR_phTERT_betaglobin_CFP_hgH_rITR clone 2	
B583 pSB1C3_lITR_phTERT_betaglobin_CD clone 1
B584 pSB1C3_lITR_phTERT_betaglobin_CD clone 2
B585 pSB1C3_lITR_phTERT_betaglobin_CD clone 3
B586 pSB1C3_001_VP2/3_capins_587_KO_His_clone1	
B587 pSB1C3_001_VP2/3_capins_587_KO_His_clone2
B588 pSB1C3_001_VP2/3_capins_587_KO_His_clone3
B589 pSB1C3_001_VP2/3_capins_587_KO_His_oligo_clone1	
B590 pSB1C3_001_VP2/3_capins_587_KO_His_oligo_clone2	
B591 pSB1C3_001_VP2/3_capins_587_KO_His_oligo_clone3
B592 pSB1C3_001_VP2/3_capins_587_KO_Bap_clone1	
B593 pSB1C3_001_VP2/3_capins_587_KO_Bap_clone2	
B594 pSB1C3_001_VP2/3_capins_587_KO_Bap_clone3</P><P><B>Mini-Prep was performed according to the standard protocol</B>
P714 pSB1C3_lITR_CMV_betaglobin_mCherry clone1 c = 89,3 ng/µl
P715 pSB1C3_lITR_CMV_betaglobin_mCherry clone2 c = 312,4 ng/µl
P716 pSB1C3_lITR_CMV_betaglobin_CD clone1 c = 254,4 ng/µl
P717 pSB1C3_lITR_CMV_betaglobin_CD clone2 c = 314,8 ng/µl
P718 pSB1C3_lITR_phTERT_betaglobin_mCherry clone1 c = 409,8 ng/µl
P719 pSB1C3_lITR_phTERT_betaglobin_mCherry clone2 c = 322,1 ng/µl
P720 pSB1C3_lITR_phTERT_betaglobin_CFP_hgH_rITR clone 1 c = 265,1 ng/µl
P721 pSB1C3_lITR_phTERT_betaglobin_CFP_hgH_rITR clone 2 c = 238,8 ng/µl
P722 pSB1C3_lITR_phTERT_betaglobin_mGMK_TK30_hgH_rITR clone 1 c = 145,9 ng/µl
P723 pSB1C3_lITR_phTERT_betaglobin_mGMK_TK30_hgH_rITR clone 2 c = 100,5 ng/µl
P724 pSB1C3_lITR_phTERT_betaglobin_CD clone 1 c = 176,9 ng/µl
P725 pSB1C3_lITR_phTERT_betaglobin_CD clone 2 c = 165,0 ng/µl
P726 pSB1C3_lITR_phTERT_betaglobin_CD clone 3 c = 149,0 ng/µl
P727 pSB1C3_CMV c = 225,5 ng/µl
P728 pSB1C3_hGH_rITR c = 129,6 ng/µl
P729 pSB1C3_leftITR_CMV_beta-globin c = 243,4 ng/µl
P730 pSB1C3_leftITR_hTERT_beta-globin c = 81,8 ng/µl
P731 pSB1C3_001_VP2/3_insCap c = 466,1 ng/µl
P732 pSB1C3_001_VP2/3_capins_587_KO_His_clone1 c = 107,6 ng/µl
P733 pSB1C3_001_VP2/3_capins_587_KO_His_clone2 c = 110,3 ng/µl
P734 pSB1C3_001_VP2/3_capins_587_KO_His_clone3 c = 121,7 ng/µl
P735 pSB1C3_001_VP2/3_capins_587_KO_His_oligo_clone1 c = 101,0 ng/µl
P736 pSB1C3_001_VP2/3_capins_587_KO_His_oligo_clone2 c = 101,6 ng/µl
P737 pSB1C3_001_VP2/3_capins_587_KO_His_oligo_clone3 c = 87,0 ng/µl
P738 pSB1C3_001_VP2/3_capins_587_KO_Bap_clone1 c = 106,5 ng/µl
P739 pSB1C3_001_VP2/3_capins_587_KO_Bap_clone2 c = 110,2 ng/µl
P740 pSB1C3_001_VP2/3_capins_587_KO_Bap_clone3 c = 71,46 ng/µl
P741 pGGTBT7-ZEGFR:1907_Middlelinker_EGFP_His c = 115,4 ng/µl</P><H3><SPAN class="mw-headline" id="140._labday_05.10.2010"><P style="font-size:17px; background-color:#00dd77;">140. labday 05.10.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Site-directed_mutagenesis_in_pSB1C3_mGMK_TK30_construct"><P style="font-size:15px; background-color:#66bbFF;"><B>Site-directed mutagenesis in pSB1C3_mGMK_TK30 construct</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="color:#66bbff;"><I>Comment</I>: Since we are still waiting for the ordered TK30, idea was to mutate the PstI site in the TK30 region. After mutating the PstI it is ready for submitting. Addtionally the ordered and received sr3ß can be subcloned. </P><P><B>Protocol using the QuickChange Ligthning Kit from Stratagene: </B><A href="https://2010.igem.org/Image:Freiburg10_SDM_PstI_TK30_05_10.tif" class="external free" rel="nofollow">https://2010.igem.org/Image:Freiburg10_SDM_PstI_TK30_05_10.tif</A>
The follwoing PCR was used:
</P><UL><LI>95 °C 2'</LI><LI>95 °C 20&quot;</LI><LI>60 °C 1'</LI><LI>68 °C 2'</LI><LI>Goto 2 repeat 17</LI><LI>68 °C 5'</LI></UL><P>After the PCR program was conducted, I added 1 µL of DpnI provided with the Kit to each sample and incubated it for 10 minutes. 
For transformation XL-10 Gold cells with 2µL beta-mercaptoethanol have been used and plated on agar plates containing chloramphenicol.
</P><H4><SPAN class="mw-headline" id="Cellculture:_Bad_news"><P style="font-size:15px; background-color:#66bbFF;"><B>Cellculture: Bad news</B></P></SPAN></H4><P>Something happened to our virus-production cell line AAV293.<FONT color="#FF0000">Nearly all AAV293 cells are dead</FONT>. </P><P>The whole Loop-Insertion Transfection from 3.10 is lost, our actual culture flasks aswell. We dont know exactly the cause of this cellular mass mortality. Maybe someting is wrong with the new FCS-stocks.
</P><H4><SPAN class="mw-headline" id="Repetition_of_pCerulean_VP1up_NLS_DARPin"><P style="font-size:15px; background-color:#66bbFF;"><B>Repetition of pCerulean_VP1up_NLS_DARPin</B></P></SPAN></H4><P><B>Investigators: Achim</B></P><UL><LI> The Test Digestion (1.10.) of the last attempts clones didn't look convincing. We sent clone 1 for sequencing yesterday, but we didn't recieve any results yet. That's why I decided to repeat the experiment today, in case results are negative.
</LI></UL><UL><LI>Details: See first run from 28.9.
</LI></UL><P>Prep. Gel:
</P><H4><SPAN class="mw-headline" id="Harvest_of_viral_stocks"><P style="font-size:15px; background-color:#66bbFF;"><B>Harvest of viral stocks</B></P></SPAN></H4><P><B>Investigators:</B></P><P>
Number of viral stocks: 4
</P><UL><LI>mVenus standard stock 10 x cm R/C: P468<SUP>2</SUP> plates =&gt; Harvest in 2 50 ml Falcons, four times freeze thaw centrifugation step with 2500g for 10 min</LI><LI>TKGMK stock 10 x cm<SUP>2</SUP> plates R/C: P468 =&gt; Harvest in 2 50 ml Falcons, four times freeze thaw centrifugation step with 2500g for 10 min</LI><LI>viral particles with GOI (mVenus) missing HGH one 6well plate =&gt; Harvest in 1 50 ml Falcon, four times freeze thaw centrifugation step with 2500g for 10 min</LI><LI>viral particles with GOI (mVenus) missing beta globin one 6well plate =&gt; Harvest in 1 50 ml Falcon, four times freeze thaw centrifugation step with 2500g for 10 min</LI><LI>viral particles with R/C missing HSPG binding motif one 6well plate =&gt; Harvest in 1 50 ml Falcon, four times freeze thaw centrifugation step with 2500g for 10 min</LI><LI>viral particles with new standard R/C (intact HSPG binding motif) one 6well plate =&gt; Harvest in 1 50 ml Falcon, four times freeze thaw centrifugation step with 2500g for 10 min</LI></UL><P>
Motivation: 

The two new standard vectors are from now on called <B>Control Vectors</B>, to get valide data. The TKGMK Stock will be used for the MTT-ASSAY.
</P><H4><SPAN class="mw-headline" id="Repetition_of_Cloning_lITR_phTERT_beta-globin_mGMK_TK30_into_pSB1C3_hGH_rITR"><P style="font-size:15px; background-color:#66bbff;"><B>Repetition of Cloning lITR_phTERT_beta-globin_mGMK_TK30 into pSB1C3_hGH_rITR</B></P></SPAN></H4><P><B>Investigator: Anna </B></P><P style="font-size:13px; color:red;">Comment: Second approach of Cloning lITR_phTERT_beta-globin_mGMK_TK30 into pSB1C3_hGH_rITR, first one didn't work (see labday 01.10.).</P><P>Vector name:
</P><UL><LI>pSB1C3_hGH_rITR (P186)</LI><P>c= 129,6 ng/µl
</P></UL><P>Insert name:
</P><UL><LI>pSB1C3_lITR_phTERT_beta-globin_mGMK_TK30 (P673)</LI><P>c= 266,4 ng/µl
</P></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume for Pp728 /µl</B></TD><TD align="right"><B>volume of P673 /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">11,6 </TD><TD align="right">7,5
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right">2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right">2
</TD></TR><TR><TD>Enzyme XbaI</TD><TD align="right">1</TD><TD align="right">-
</TD></TR><TR><TD>Enzyme EcoRI</TD><TD align="right">1</TD><TD align="right">1
</TD></TR><TR><TD>Enzyme SpeI</TD><TD align="right">-</TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">2,4</TD><TD align="right">6,5
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 20</TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><P>
0,4 g Agarose, 50 ml TAE (0,8%), 3 µl GELRED , at 115 Volt</P><P><B>Gel extraction</B>: 
Was performed according to the standard protocol.
</P><TABLE border="1"><TBODY><TR><TD align="right">rITR_hgH</TD><TD align="right">lITR_phTERT_betaglobin_mGMK_TK30
</TD></TR><TR><TD>Expected size of fragment </TD><TD align="right"> 2690 bp</TD><TD align="right">3045 bp
</TD></TR><TR><TD>DNA-concentration [ng/µl] </TD><TD align="right">37,4 </TD><TD align="right">19,81
</TD></TR></TBODY></TABLE><P><B>T4 Ligation</B>: </P><TABLE border="1"><TBODY><TR><TD>volume of vector </TD><TD align="right">1,1 µl
</TD></TR><TR><TD>volume of insert</TD><TD align="right">6,9 µl
</TD></TR></TBODY></TABLE><P><B>Transformation</B>: 
Was done following the standard protocol using BL21 cells.
</P><H4><SPAN class="mw-headline" id="Seeding_HT1080_and_AAV431_for_transduction"><P style="font-size:15px; background-color:#66bbFF;"><B>Seeding HT1080 and AAV431 for transduction</B></P></SPAN></H4><UL><LI>3*6well plates HT1080</LI><LI>3*6well plates A431</LI></UL><P>for transduction with following viral stocks:
</P><UL><LI>mVenus Control Vector to quantify the other stocks </LI><LI>mVenus without beta-globin</LI><LI>mVenus without HGH</LI><LI>mVenus in viral vectors without HSPG-Binding-site(P667)</LI><LI>mVenus in viral vectors with HSPG-Binding-site, the new state-of-the-art-R/C (P668)</LI></UL><H4><SPAN class="mw-headline" id="Cloning_of_pSB1C3_lITR_phTERT_beta-globin_CD_and_pSB1C3_lITR_CMV_beta-globin_CD"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of pSB1C3_lITR_phTERT_beta-globin_CD and pSB1C3_lITR_CMV_beta-globin_CD</B></P></SPAN></H4><P><B>Investigator: Anissa</B></P><P>Vector name:
</P><UL><LI>pSB1C3_lITR_phTERT_beta-globin (p730)</LI><LI>pSB1C3_lITR_CMV_beta-globin (P729)</LI><P>Insert name:
</P><LI>VB:pSB1C3_CD(P690)</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume of P730  /µl</B></TD><TD align="right"><B>volume of P729  /µl</B></TD><TD align="right"><B>volume of P690 /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">12</TD><TD align="right">4,1</TD><TD align="right">9,9
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right">1,5</TD><TD align="right">2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right">1,5 </TD><TD align="right">2
</TD></TR><TR><TD>Enzyme AgeI</TD><TD align="right">1</TD><TD align="right">1 </TD><TD align="right">-
</TD></TR><TR><TD>Enzyme NgoMIV</TD><TD align="right">-</TD><TD align="right">- </TD><TD align="right">1
</TD></TR><TR><TD>Enzyme PstI</TD><TD align="right">1</TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">1</TD><TD align="right">5,9</TD><TD align="right">4,1
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 20</TD><TD align="right"> 15</TD><TD align="right">20
</TD></TR></TBODY></TABLE><P><B>Gel:</B>
for vectors and insert :
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 110 Volt</P><P><B>Gel extraction</B>: 
Was performed according to protocol.
</P><P><B>T4 Ligation</B>: </P><UL><LI>Volume vector P729:  1,39 µl</LI><LI>Volume insert:  6,61 µl</LI></UL><UL><LI>Volume vector P730:  3,55 µl</LI><LI>Volume insert:  4,45 µl</LI></UL><P><B>Transformation</B>: 
Was performed according to standard protocol using BL21 cells.
</P><H4><SPAN class="mw-headline" id="Repetition_of_the_test-digestion_of_p714.2Cp715.2Cp719.2Cp720.2Cp721.2Cp724_and_p726"><P style="font-size:15px; background-color:#66bbff;"><B>Repetition of the test-digestion of p714,p715,p719,p720,p721,p724 and p726</B></P></SPAN></H4><P><B>Investigator: Anissa</B></P><UL><LI>pSB1C3_lITR_CMV_beta-globin_mCherry: Cut with EcoRI, PstI &amp; NdeI
</LI><LI>P714 (Lane 1)
</LI><LI>P715 (Lane 2)
</LI><LI><B>Results:</B>Looks good.
</LI></UL><UL><LI>pSB1C3_lITR_phTERT_beta-globin_mCherry: Cut with EcoRI, PstI &amp; SacII
</LI><LI>P719 (Lane 3)
</LI><LI><B>Results:</B>Looks good.
</LI></UL><UL><LI>pSB1C3_lITR_phTERT_beta-globin_CFP_hgh_rITR: Cut with EcoRI, AgeI &amp; PstII
</LI><LI>P720 (Lane 4)
</LI><LI>P721 (Lane 5)
</LI><LI><B>Results:</B>The expected bands should run at 671 bp,2033 bp and 1855 bp. Sequencing has to be repeated
</LI></UL><UL><LI>pSB1C3_lITR_phTERT_beta-globin_CD: Cut with PstI &amp; EcoRI
</LI><LI>P724 (Lane 6)
</LI><LI>P726 (Lane 7)
</LI><LI><B>Results:</B>The expected bands should run at 2035 and 2460 bp. The experiment has been repeated already today.
</LI></UL><H4><SPAN class="mw-headline" id="Repetition:_Cloning_of_VP2.2F3_BAP_HSPG-KO_and_VP2.2F3_His_HSPG-KO_into_pCerulean_Zegfr:1907_Middlelinker_and_VP2.2F3_His_HSPG-KO_into_pCerulean_VP1up_NLS_mVenus"><P style="font-size:15px; background-color:#66bbff;"><B>Repetition: Cloning of VP2/3_BAP_HSPG-KO and VP2/3_His_HSPG-KO into pCerulean_Zegfr:1907_Middlelinker and VP2/3_His_HSPG-KO into pCerulean_VP1up_NLS_mVenus</B></P></SPAN></H4><P><B>Investigator: Stefan </B></P><P style="font-size:13px; color:red;">Comment: Cloning of 587-ko_His and 587-ko_BAP into pSB1C3_001_VP2/3_capins did not work out last time. Since cloning of VP2/3_capins started before sequencing results arrived, this experiment has to be repeated.</P><P>Vector name:
</P><UL><LI>pCerulean_Zegfr:1907_Middlelinker (P408)</LI><LI>pCerulean_VP1up_NLS_mVenus (P426)</LI></UL><P>Insert name:
</P><UL><LI>pSB1C3_001_VP2/3_capins_587_KO_His_clone3 (P734)</LI><LI>pSB1C3_001_VP2/3_capins_587_KO_Bap_clone1 (P738)</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume for inserts (P734 + P738)  /µl</B></TD><TD align="right"><B>volume of P408  /µl</B></TD><TD align="right"><B>volume of P426 /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">14 </TD><TD align="right">3</TD><TD align="right">3
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">3 </TD><TD align="right">2</TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">3 </TD><TD align="right">2 </TD><TD align="right"> 2
</TD></TR><TR><TD>Enzyme NgoMIV</TD><TD align="right">1</TD><TD align="right">- </TD><TD align="right">-
</TD></TR><TR><TD>Enzyme PstI</TD><TD align="right">1</TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>Enzyme MscI</TD><TD align="right">1</TD><TD align="right">- </TD><TD align="right">-
</TD></TR><TR><TD>Enzyme AgeI</TD><TD align="right">-</TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">7</TD><TD align="right">11</TD><TD align="right">11
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 30</TD><TD align="right"> 20</TD><TD align="right">20
</TD></TR></TBODY></TABLE><P><B>Gel:</B>
0,4 g Agarose, 50 ml TAE (0,8%), 3 µl GELRED , at 115 Volt</P><P><B>Gel extraction</B>: 
Was performed according to protocol.
</P><P><B>T4 Ligation</B>: </P><TABLE border="1"><TBODY><TR><TD>ligation name </TD><TD align="right">408 + 734</TD><TD align="right">408 + 738</TD><TD align="right">426 + 734</TD><TD align="right">426 + 738
</TD></TR><TR><TD>volume of vector </TD><TD align="right">3,81 </TD><TD align="right"> 3,81</TD><TD align="right"> 3,56</TD><TD align="right"> 3,56
</TD></TR><TR><TD>volume of insert</TD><TD align="right">4,19 </TD><TD align="right">4,19</TD><TD align="right">4,44</TD><TD align="right">4,44
</TD></TR></TBODY></TABLE><P><B>Transformation</B>: 
Was performed according to standard protocol using BL21 cells.
</P><H4><SPAN class="mw-headline" id="PCR_of_VP1-3"><P style="font-size:15px; background-color:#66bbFF;"><B>PCR of VP1-3</B></P></SPAN></H4><P><B>Investigator: Kira</B></P><P><B>Comment:</B> this approach was already repeated at least 2 times, but still even if its not one of the very important constructs, it should be finished by end of the month.
</P><P>PCR*PCR program:
</P><P>c(pKEX_VP1)=488 ng/ul 
c(pKEX_VP2)=484 ng/ul 
c(pKEX_VP2)=485 ng/ul </P><P>VP-1 praefix primer: 089
VP-2 praefix primer: 090
VP-3 praefix primer: 091
VP1-3 suffix reg_25: 0120
</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume in µl
</TD></TR><TR><TD> 5x Phusion HF buffer  </TD><TD align="right"> 10
</TD></TR><TR><TD> 10 mM dNTP mix </TD><TD align="right">1
</TD></TR><TR><TD>primer_for (1:10 dilution)</TD><TD align="right">  2,5
</TD></TR><TR><TD>primer_rev (1:10 dilution)</TD><TD align="right"> 2,5
</TD></TR><TR><TD>DNA template (1:100)</TD><TD align="right">  0,5
</TD></TR><TR><TD>DMSO</TD><TD align="right">  0
</TD></TR><TR><TD>Phusion polymerase</TD><TD align="right"> 0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right"> 33
</TD></TR><TR><TD><B>Total volume (e.g. 50 µl)</B></TD><TD align="right"> 50
</TD></TR></TBODY></TABLE><P>
The PCR programm for all 3 samples was the same due to the weakest primer, which is VP-1 praefix primer </P><TABLE border="1"><TBODY><TR><TD>Cycles</TD><TD>Temperature</TD><TD>Time
</TD></TR><TR><TD>98°C</TD><TD>30 sec
</TD></TR><TR><TD>10x</TD><TD>98°C</TD><TD>10 sec
</TD></TR><TR><TD>60°C</TD><TD>25 sec
</TD></TR><TR><TD>72°C</TD><TD>1 min 6 sec
</TD></TR><TR><TD>20x</TD><TD>98°C</TD><TD>15 sec
</TD></TR><TR><TD>72°C</TD><TD>1 min 31 sec
</TD></TR><TR><TD>1x</TD><TD>72°C</TD><TD>5 min
</TD></TR><TR><TD>Hold 4°C
</TD></TR></TBODY></TABLE><P>
1% agarose gel </P><P>The gel shows only the primer bands on the bottom of the gel --&gt; PCR failed again. Its going to be repeated tomorrow. </P><H3><SPAN class="mw-headline" id="141._labday_06.10.2010"><P style="font-size:17px; background-color:#00dd77;">141. labday 06.10.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Cellculture"><P style="font-size:15px; background-color:#66bbFF;"><B>Cellculture</B></P></SPAN></H4><P><B>Investigator Patrick</B>
Not only the 293 cells look bad. The A431 cells and the HT1080 cells die, too. There is no contamination detectable. Maybe there is something wrong with the DMEM because the now 100 % serum-free AAV293 cells dont die.
</P><H4><SPAN class="mw-headline" id="Transduction_of_HT1080_and_A431_cells"><P style="font-size:15px; background-color:#66bbFF;"><B>Transduction of HT1080 and A431 cells</B></P></SPAN></H4><P><FONT color="#FF0000">The cells are not adherent so no transduction was performed</FONT>. </P><H4><SPAN class="mw-headline" id="Sequence_analysis_of_pSB1C3_CD"><P style="font-size:15px; background-color:#66bbFF;"><B>Sequence analysis of pSB1C3_CD</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="color:#66bbff;"><I>Comment</I>: After test digestion revealed positive results, we sent one clone for sequencing (P690) with VF-2 and VR-2 primers. The tube numbers are: SB_1 and SB_2. </P><P><B>Results</B>:
</P><UL><LI>Prefix looks well. Alle restriction sites can be found in the prefix RFC25.</LI><LI>Suffixlooks well. Alle restriction sites can be found in the suffix RFC25.</LI><LI>BUT: There seems to be a PstI recognition site within the sequence. Therefore the site-directed mutagenesis needs to be repeated.</LI></UL><H4><SPAN class="mw-headline" id="Site-directed_mutagenesis_of_pSB1C3_CD"><P style="font-size:15px; background-color:#66bbFF;"><B>Site-directed mutagenesis of pSB1C3_CD</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="color:#66bbff;"><I>Comment</I>: Since the sequence analysis revealed that the PstI restriction iste within the CD is NOT removed I prepared another Quickchange in order to mutate the PstI site. </P><P><B>Protocol: </B></P><P>
The following PCR was used:
</P><UL><LI>95 °C 2'</LI><LI>95 °C 20&quot;</LI><LI>60 °C 10&quot;</LI><LI>68 °C 1'45&quot;</LI><LI>Goto 2 repeat 17</LI><LI>68 °C 5'</LI></UL><P>After the PCR program was conducted, I added 1 µL of DpnI provided with the Kit to each sample and incubated it for 10 minutes at 37°C. 
For transformation XL-10 Gold cells with 2µL beta-mercaptoethanol have been used and plated on agar plates containing chloramphenicol.
</P><H4><SPAN class="mw-headline" id="Sequencing_analysis"><P style="font-size:15px; background-color:#66bbFF;"><B>Sequencing analysis</B></P></SPAN></H4><P><B>Investigator: Stefan</B><FONT color="#FF0000"><B>Comment:</B>
Sequencing reults of pSB1C3_CMV_Z<SUB>EGFR:1907</SUB>_VP2/3_CapIns (P679), pSB1C3_CMV_Z<SUB>EGFR:1907</SUB>_VP2/3_CapIns_HSPG-ko (P684), pSB1C3_001_VP2/3_587_HSPG-ko_His(P734) and pSB1C3_001_VP2/3_587_HSPG-ko_BAP(P738). In P679 and P684, CMV_Z<SUB>EGFR:1907</SUB> was cloned in front of VP2/3, in P734 and P738, 587_HSPG-ko_His/BAP was cloned into the 587 loop.</FONT></P><H4><SPAN class="mw-headline" id="Test_Digestion_of_pSB1C3_CMV_DARPin_MiddleLinker_VP2.2F3_insCap_and_pSB1C3_CMV_DARPin_MiddleLinker_VP2.2F3_HSPG-KO"><P style="font-size:15px; background-color:#66bbFF;"><B>Test Digestion of pSB1C3_CMV_DARPin_MiddleLinker_VP2/3_insCap and pSB1C3_CMV_DARPin_MiddleLinker_VP2/3_HSPG-KO </B></P></SPAN></H4><P><B>Investigator: Hanna </B><B>Comment:</B> Besides the Affibody we also want to test and characterize the DARPin E01 as a targeting motif for EGFR over expressing tumor cells. The day before yesterday a 3 fragment ligation was performed with the CMV promoter, DARPin and pSB1C3_VP2/3_insCap/pSB1C3_VP2/3_HSPG-KO. 
2 clones were picked from each approach and preped today. <B>TEST DIGESTION:</B></P><UL><LI> DNA: 2 µL
</LI><LI> Buffer 3: 1 µL
</LI><LI> BSA (10x): 1 µL
</LI><LI> XbaI: 0.5 µL
</LI><LI> BmgBI (cuts in the DARPin sequence): 0.5 µL
</LI><LI> H2O: 5 µL
</LI></UL><P>
Incubation: 2 hours. </P><P><B>Conclusion:</B> Test digestion looks well: Expected fragments = 756 bp and 4409 bp. 
Two samples were sent for sequencing to GATC using the CMV-F primer. </P><H4><SPAN class="mw-headline" id="Repetition:_Cloning_of_CFP_into_pSB1C3_lITR_phTERT_beta-globin_and_hgH_rITR_into_pSB1C3_lITR_CMV_beta-globin_mCherry_and_into_pSB1C3_lITR_phTERT_beta-globin_mCherry"><P style="font-size:15px; background-color:#66bbff;"><B>Repetition: Cloning of CFP into pSB1C3_lITR_phTERT_beta-globin and hgH_rITR into pSB1C3_lITR_CMV_beta-globin_mCherry and into pSB1C3_lITR_phTERT_beta-globin_mCherry</B></P></SPAN></H4><P><B>Investigator: Stefan </B></P><P style="font-size:13px; color:red;">Comment: </P><P>Vector name:
</P><UL><LI>pSB1C3_lITR_phTERT_beta-globin (P437)</LI><LI>pSB1C3_lITR_CMV_beta-globin_mCherry (P715)</LI><LI>pSB1C3_lITR_phTERT_beta-globin_mCherry (P719)</LI></UL><P>Insert name:
</P><UL><LI>pSB1C3_CFP (P666)</LI><LI>pSB1C3_hgH_rITR (P728)</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume for inserts (P666 + P728)  /µl</B></TD><TD align="right"><B>volume of vectors P437 + P715 + P719  /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">10 </TD><TD align="right">4
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right">2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right">2
</TD></TR><TR><TD>Enzyme PstI</TD><TD align="right">1</TD><TD align="right">1
</TD></TR><TR><TD>Enzyme XbaI</TD><TD align="right">1</TD><TD align="right">-
</TD></TR><TR><TD>Enzyme SpeI</TD><TD align="right">-</TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">4</TD><TD align="right">10
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 20</TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><P><B>Gel:</B>
0,4 g Agarose, 50 ml TAE (0,8%), 3 µl GELRED , at 115 Volt</P><P><B>Gel extraction</B>: 
Was performed according to protocol.
</P><P><B>T4 Ligation</B>: </P><TABLE border="1"><TBODY><TR><TD>ligation name </TD><TD align="right">437 + 666</TD><TD align="right">715 + 728</TD><TD align="right">719 + 728
</TD></TR><TR><TD>volume of vector </TD><TD align="right">5,08 </TD><TD align="right"> 5,13</TD><TD align="right"> 5,34
</TD></TR><TR><TD>volume of insert</TD><TD align="right">2,92 </TD><TD align="right">2,87</TD><TD align="right">2,66
</TD></TR></TBODY></TABLE><P><B>Transformation</B>: 
Was performed according to standard protocol using BL21 cells.
</P><H4><SPAN class="mw-headline" id="Biotinylation_of_the_monoclonal_IgG_antibody_A20"><P style="font-size:15px; background-color:#66bbff;"><B>Biotinylation of the monoclonal IgG antibody A20</B></P></SPAN></H4><P>The biotinylation of the A20 antibody that recognizes the assembled virus was performed with a biotinylation Kit purchased from Dojindo. 
</P><UL><LI>The first biotinylation reaction was performed according to the protocol with 150µg of the purified antibody but resulted in not detectable protein concentrations.
</LI><LI>The biotinylation reaction was repeated for a second time with 200µg and resulted in satisfying protein concentrations.
</LI><LI>Successful biotinylation has to be tested in an other experiment.
</LI></UL><H4><SPAN class="mw-headline" id="Repetition_of_VP1-3_PCR"><P style="font-size:15px; background-color:#66bbFF;"><B>Repetition of VP1-3 PCR</B></P></SPAN></H4><P><B>Investigator: Kira</B>
PCR program:
</P><P>c(pKEX_VP1)=488 ng/ul 
c(pKEX_VP2)=484 ng/ul 
c(pKEX_VP2)=485 ng/ul </P><P>VP-1 praefix primer: 089
VP-2 praefix primer: 090
VP-3 praefix primer: 091
VP1-3 suffix reg_25: 0120
</P><P>Regarding the primer sequences, DMSO will be added to the PCR samples. 
</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume in µl
</TD></TR><TR><TD> 5x Phusion HF buffer  </TD><TD align="right"> 10
</TD></TR><TR><TD> 10 mM dNTP mix </TD><TD align="right">1
</TD></TR><TR><TD>primer_for (1:10 dilution)</TD><TD align="right">  2,5
</TD></TR><TR><TD>primer_rev (1:10 dilution)</TD><TD align="right"> 2,5
</TD></TR><TR><TD>DNA template (1:100)</TD><TD align="right">  0,5
</TD></TR><TR><TD>DMSO</TD><TD align="right">  2
</TD></TR><TR><TD>Phusion polymerase</TD><TD align="right"> 0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right"> 31
</TD></TR><TR><TD><B>Total volume (e.g. 50 µl)</B></TD><TD align="right"> 50
</TD></TR></TBODY></TABLE><P>
The PCR programm for all 3 samples was the same due to the weakest primer, which is VP-1 praefix primer </P><TABLE border="1"><TBODY><TR><TD>Cycles</TD><TD>Temperature</TD><TD>Time
</TD></TR><TR><TD>98°C</TD><TD>30 sec
</TD></TR><TR><TD>10x</TD><TD>98°C</TD><TD>15 sec
</TD></TR><TR><TD>59°C</TD><TD>25 sec
</TD></TR><TR><TD>72°C</TD><TD>1 min 11 sec
</TD></TR><TR><TD>20x</TD><TD>98°C</TD><TD>15 sec
</TD></TR><TR><TD>72°C</TD><TD>1 min 36 sec
</TD></TR><TR><TD>1x</TD><TD>72°C</TD><TD>5 min
</TD></TR><TR><TD>Hold 4°C
</TD></TR></TBODY></TABLE><P>
1% agarose gel <A href="/File:Freiburg10_VP1-3_PCR.jpg" title="File:Freiburg10 VP1-3 PCR.jpg">File:Freiburg10 VP1-3 PCR.jpg</A></P><P>Digestion of plasmid backbone:
</P><P>pSB1C3_001 is used as backbone
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>vector</B> Volume/µL
</TD></TR><TR><TD align="left"> DNA  </TD><TD align="left"> 3,0 µl
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 2 µl
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 2,0 µl
</TD></TR><TR><TD align="left"> Enzyme 1 AgeI </TD><TD align="left"> 1,5 µl
</TD></TR><TR><TD align="left"> Enzyme 2 SpeI  </TD><TD align="left"> 1,0 µl
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 10,5 µl
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>20</B></TD></TR></TBODY></TABLE><P>incubation @ 37 C for approx. 2 h
</P><P>1% agarose gel <A href="/File:Freiburg10_digestion-pSB1C3_001.jpg" title="File:Freiburg10 digestion-pSB1C3 001.jpg">File:Freiburg10 digestion-pSB1C3 001.jpg</A></P><P>Digestion of PCR product:
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>PCR product</B> Volume/µL
</TD></TR><TR><TD align="left"> DNA   </TD><TD align="left"> 30,0 µl
</TD></TR><TR><TD align="left"> BSA (100x) </TD><TD align="left"> 4,5 µl
</TD></TR><TR><TD align="left"> Buffer no. 4  </TD><TD align="left"> 4,5 µl
</TD></TR><TR><TD align="left"> Enzyme 1 NgoMIV </TD><TD align="left"> 1 µl
</TD></TR><TR><TD align="left"> Enzyme 2 SpeI  </TD><TD align="left">1,0 µl
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 4 µl
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>45</B></TD></TR></TBODY></TABLE><P>
incubation @ 37 C for approx. 2 h </P><P>ligation with T4 ligase was performed over-night </P><H4><SPAN class="mw-headline" id="Cloning_of_Rep_into_Rep52_and_Rep78"><P style="font-size:15px; background-color:#66bbFF;"><B>Cloning of Rep into Rep52 and Rep78</B></P></SPAN></H4><P><B>Investigator: Kira</B><B>Comment:</B> In order to perfom PCR, we have to clone the ordered Rep gene into Rep52 and Rep78 backbones. </P><P>Regarding the activity conditions of the enzymes, digestion was performed in 2 steps.  </P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>insert</B></TD><TD align="left"><B>Rep78</B></TD><TD align="left"><B>Rep52</B></TD></TR><TR><TD align="left"> DNA   </TD><TD align="left"> 3 µl</TD><TD align="left"> 2 µl</TD><TD align="left"> 7 µl
</TD></TR><TR><TD align="left"> BSA (100x) </TD><TD align="left"> 2 µl</TD><TD align="left"> 2 µl</TD><TD align="left"> 2 µl
</TD></TR><TR><TD align="left"> Buffer no. 3  </TD><TD align="left"> 2,0 µl</TD><TD align="left"> 2,0 µl</TD><TD align="left"> 2,0 µl
</TD></TR><TR><TD align="left"> Enzyme SwaI  </TD><TD align="left">1,0 µl </TD><TD align="left">1,0 µl </TD><TD align="left">1,0 µl
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 12 µl</TD><TD align="left"> 13 µl</TD><TD align="left"> 8 µl
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>20</B></TD></TR></TBODY></TABLE><P>
incubation @ 25 C for approx. 2 h 
Dephosphorylation was performed because of the blunt cleavage. PCR phosphorylation followed in order to get rid off the buffer because Buffer 3 is not appropriate for further digestion.
after PCR phosphorylation: 
c(pMA_RC-insert)= 21,75 ng/ul
c(rep52)= 75,82 ng/ul
c(rep78)= 2,47 ng/ul
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>insert</B></TD><TD align="left"><B>Rep78</B></TD><TD align="left"><B>Rep52</B></TD></TR><TR><TD align="left"> DNA   </TD><TD align="left"> 30 µl</TD><TD align="left"> 30 µl</TD><TD align="left"> 30 µl
</TD></TR><TR><TD align="left"> BSA (100x) </TD><TD align="left"> 0 µl</TD><TD align="left"> 0 µl</TD><TD align="left"> 0 µl
</TD></TR><TR><TD align="left"> Buffer no. 2  </TD><TD align="left"> 4,5 µl</TD><TD align="left"> 4,5 µl</TD><TD align="left"> 4,5 µl
</TD></TR><TR><TD align="left"> Enzyme HindIII  </TD><TD align="left">1,0 µl </TD><TD align="left">1,0 µl </TD><TD align="left">1,0 µl
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 9,5 µl</TD><TD align="left"> 9,5 µl</TD><TD align="left">9,5 µl
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>45</B></TD></TR></TBODY></TABLE><P>incubation for 2h at 37C</P><P><A href="/File:Rep52%26Rep78.jpg" title="File:Rep52&amp;Rep78.jpg">File:Rep52&amp;Rep78.jpg</A></P><P>the gel shows that Rep78 backbone got lost.. Regarding the concentration, which was measured after PCR purification it might be possible --&gt; digestion of Rep78 will be repeated tomorrow, while Rep52 is ready for ligation. </P><H3><SPAN class="mw-headline" id="142._labday_07.10.2010"><P style="font-size:17px; background-color:#00dd77;">142. labday 07.10.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="DARPin_VP2_Fusion:_Sequencing_results"><P style="font-size:15px; background-color:#66bbFF;"><B>DARPin VP2 Fusion: Sequencing results</B></P></SPAN></H4><P><B>Investigator: Hanna</B><B>COMMENT:</B> The DARPin was fused with the Middle Linker to the VP2/3 BioBrick and cloned into the pSB1C3 backbone. Yesterday's test digestion looked well, so one clone of the VP2/3_insCap and one clone of the VP2/3_HSPG-KO construct was sent for sequencing.</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:422px;"><DIV class="thumbcaption">pSB1C3_CMV_DARPin_MiddleLinker_VP2/3_HSPG-KO</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:422px;"><DIV class="thumbcaption">pSB1C3_CMV_DARPin_MiddleLinker_VP2/3_insCap</DIV></DIV></DIV><P><B>CONCLUSION:</B> Sequencing looked well: 3 fragment ligation was successful - because the CMV-F primer was used for sequencing, the CMV promoter was successfully inserted. As shown in the alignments also the DARPin was inserted. This is the first time that the DARPin was sequenced - thus the original sequence must also be OK! 
These two constructs are finished and are ready to be tested in cell culture via qPCR, ELISA,... and can be stored in our BioBrick Box for submitting. </P><H4><SPAN class="mw-headline" id="Repetition_of_cloning_of_ordered_Rep_into_Rep78"><P style="font-size:15px; background-color:#66bbFF;"><B>Repetition of cloning of ordered Rep into Rep78</B></P></SPAN></H4><P><B>Investigator: Kira</B><B>Comment:</B> Because the gel from yesterday did not show any bands in the Rep78 line, I desided to repeat this approach but to use more DNA, in order to be able to continue with the second digestion (see labday 06.10.2010) 
</P><P>The same procedure but with 7 ul Rep78 (c=556 ng/ul). After the first digestion, dephosporylation and PCR purification were performed and DNA was measured. </P><P>As a control, the ordered Rep was used again. </P><P>c(p190)= 28 ng/ul 
c(Rep78)= 6,90 ng/ul </P><P>the second digestion as performed despite the low concentration. </P><P><A href="/File:Freiburg10_2010-10-07.jpg" title="File:Freiburg10 2010-10-07.jpg">File:Freiburg10 2010-10-07.jpg</A></P><P>The gel shows once again no detectable DNA of Rep78. </P><H4><SPAN class="mw-headline" id="Transformation_of_VP1-3_.26_pSB1C3_001_and_p190_.26Rep52"><P style="font-size:15px; background-color:#66bbFF;"><B>Transformation of VP1-3 &amp; pSB1C3_001 and p190 &amp;Rep52 </B></P></SPAN></H4><P><B>Investigator: Kira</B></P><P>The transformation  was performed according to the standard protocol with BL21. Compared to VP1-3 transformed cells, Rep52 cells revealed no visible pellet after centrifugation, thus 300 ul of the cells suspension was plated.</P><H4><SPAN class="mw-headline" id="Test_digestion_of_different_constructs"><P style="font-size:15px; background-color:#66bbFF;"><B>Test digestion of different constructs</B></P></SPAN></H4><P><B>Investigator: Anna</B></P><P><B>Gel1_Part1:</B> The 453 constructs were cut with BamHI and SspI, the 587 constructs with SalI and PvuII.
The inserts were cloned into pSB1C3_RepCap_p5tatless.</P><P><B>All of the following constructs were cut with EcoRI and PstI:</B></P><P><B>Gel1_Part2:</B></P><P><B>Gel2:</B></P><H3><SPAN class="mw-headline" id="143._labday_08.10.2010"><P style="font-size:17px; background-color:#00dd77;">143. labday 08.10.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Mini-Prep_and_test_digestion_of_several_constructs"><P style="font-size:15px; background-color:#66bbff;"><B>Mini-Prep and test digestion of several constructs</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P style="color:#66bbff;"><I>Comment:</I> Mini-Preps were performed according to protocol:</P><P>
Glycerol stocks were prepared:</P><UL><LI>B641 pSB1C3_lITR_phTERT_beta-globin_CFP clone 1</LI><LI>B642 pSB1C3_lITR_phTERT_beta-globin_CFP clone 2</LI><LI>B643 pSB1C3_lITR_CMV_beta-globin_mCherry_hgH_rITR clone 1</LI><LI>B644 pSB1C3_lITR_CMV_beta-globin_mCherry_hgH_rITR clone 2</LI><LI>B645 SB1C3_lITR_phTERT_beta-globin_mCherry_hgH_rITR clone 1</LI><LI>B646 pSB1C3_lITR_phTERT_beta-globin_mCherry_hgH_rITR clone 2</LI></UL><P>
Mini-Prep was performed according to standard protocol:
</P><UL><LI>P791 pSB1C3_lITR_phTERT_beta-globin_CFP clone 1 c = 244.3 ng/µl</LI><LI>P792 pSB1C3_lITR_phTERT_beta-globin_CFP clone 2 c = 214.3  ng/µl</LI><LI>P793 pSB1C3_lITR_CMV_beta-globin_mCherry_hgH_rITR clone 1 c = 209.2  ng/µl</LI><LI>P794 pSB1C3_lITR_CMV_beta-globin_mCherry_hgH_rITR clone 2 c = 262.4  ng/µl</LI><LI>P795 pSB1C3_lITR_phTERT_beta-globin_mCherry_hgH_rITR clone 1 c = 234.7  ng/µl</LI><LI>P796 pSB1C3_lITR_phTERT_beta-globin_mCherry_hgH_rITR clone 2 c = 207.2  ng/µl</LI></UL><P><B>Test digestion:</B></P><P style="color:#66bbff;"><I>Comment:</I>Additionally, P771 and P781 from yesterday's Mini-Prep were digested again.</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>P791 + P792 / µl</B></TD><TD align="left"><B>P793 - P796 / µl</B></TD><TD align="left"><B>P771 / µl</B></TD><TD align="left"><B>P781 / µl</B></TD></TR><TR><TD align="left"> DNA   </TD><TD align="left"> 1,5</TD><TD align="left"> 1</TD><TD align="left"> 3 </TD><TD align="left"> 7,7
</TD></TR><TR><TD align="left"> Buffer 4 </TD><TD align="left"> 1</TD><TD align="left"> 1</TD><TD align="left"> 1</TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> BSA (10x)  </TD><TD align="left"> 1</TD><TD align="left"> 1</TD><TD align="left"> 1</TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> PstI </TD><TD align="left"> 0,3 </TD><TD align="left"> 0,3 </TD><TD align="left"> 0,3</TD><TD align="left"> 0,3
</TD></TR><TR><TD align="left"> EcoRI </TD><TD align="left">0,3 </TD><TD align="left">0,3 </TD><TD align="left">-</TD><TD align="left">-
</TD></TR><TR><TD align="left"> BpmI</TD><TD align="left">0,3 </TD><TD align="left">- </TD><TD align="left">-</TD><TD align="left">-
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 5,6</TD><TD align="left"> 6,4</TD><TD align="left">4,2</TD><TD align="left">-
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>10</B></TD><TD align="left"><B>10</B></TD><TD align="left"><B>10</B></TD><TD align="left"><B>10</B></TD></TR></TBODY></TABLE><P><B>Results:</B></P><P style="color:#66bbff;"><I>Comment:</I>Samples P791 to P796 perform like expected, they can be used for further cloning or testing in cell culture. P771 show strange bands, a test PCR will be performed to verify successful cloning. P781 does not show any DNA at all like last time. A new SDm will be performed for pSB1C3_mGMK_TK30 (P675).</P><H4><SPAN class="mw-headline" id="Test-PCR_of_pSB1C3_lITR_phTERT_beta-globin_mGMK_TK30_hgH_rITR"><P style="font-size:15px; background-color:#66bbff;"><B>Test-PCR of pSB1C3_lITR_phTERT_beta-globin_mGMK_TK30_hgH_rITR</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P style="color:#66bbff;"><I>Comment:</I> Test digestion showed strange results (an additional band at 500bp which does not fit any expectations), therfore an additional test-PCR was performed to verify successful cloning of the complete constructs.</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>positive control / µl</B></TD><TD align="left"><B>negative control / µl</B></TD><TD align="left"><B>sample P771 / µl</B></TD></TR><TR><TD align="left"> 5x Phusion HF buffer   </TD><TD align="left"> 10</TD><TD align="left"> 10</TD><TD align="left"> 10
</TD></TR><TR><TD align="left"> 10mM dNTP mix </TD><TD align="left"> 1</TD><TD align="left"> 1</TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Primer for (1:10 dilution)  </TD><TD align="left"> O107 2,5</TD><TD align="left"> O29 2,5</TD><TD align="left"> O29 2,5
</TD></TR><TR><TD align="left"> Primer rev (1:10 dilution) </TD><TD align="left"> O108 2,5 </TD><TD align="left"> O191 2,5 </TD><TD align="left"> O191 2,5
</TD></TR><TR><TD align="left"> DNA template </TD><TD align="left">1 (1:100) </TD><TD align="left">1 (1:100) </TD><TD align="left">1 (1:100)
</TD></TR><TR><TD align="left"> DMSO </TD><TD align="left">1 </TD><TD align="left">1 </TD><TD align="left">1
</TD></TR><TR><TD align="left"> Phusion Polymerase </TD><TD align="left">0,5 </TD><TD align="left">0,5 </TD><TD align="left">0,5
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 32,5</TD><TD align="left"> 32,5</TD><TD align="left">32,5
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>50</B></TD><TD align="left"><B>50</B></TD><TD align="left"><B>50</B></TD></TR></TBODY></TABLE><P><B>Gel:</B>
0,5 g agarose, 50 ml TAE (1%), 115V, 50 minutes
</P><P style="color:#66bbff;"><I>Comment:</I> Test PCR showed successful cloning of lITR_phTERT_beta-globin_mGMK_TK30 into pSB1C3_hgH_rITR. SDM to remove PstI in TK30 can be performed.</P><H4><SPAN class="mw-headline" id="Trafo_evaluation_of_VP1-3_and_Rep52_constructs"><P style="font-size:15px; background-color:#66bbFF;"><B>Trafo evaluation of VP1-3 and Rep52 constructs</B></P></SPAN></H4><P><B>Investigator: Kira</B>
VP1-3 contain lots of colonies, while Rep52 plate is empty </P><H4><SPAN class="mw-headline" id="Repetiton:_Site-directed_mutagenesis_in_pSB1C3_mGMK_TK30_construct"><P style="font-size:15px; background-color:#66bbFF;"><B>Repetiton: Site-directed mutagenesis in pSB1C3_mGMK_TK30 construct</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P style="color:#66bbff;"><I>Comment</I>: The first try to remove the PstI restriction site delivered no plasmid DNA, so another approach needs to be started.</P><P><B>Protocol using the QuickChange Lightning Kit from Stratagene: </B><B>Ingredients: </B></P><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD align="right">Volume / µl
</TD></TR><TR><TD> 10x reaction buffer </TD><TD align="right">2,5
</TD></TR><TR><TD> dNTP</TD><TD align="right">0,5
</TD></TR><TR><TD> forward primer: O191 </TD><TD align="right">0,69
</TD></TR><TR><TD> reverse primer: O192 </TD><TD align="right">0,71
</TD></TR><TR><TD> DNA Template</TD><TD align="right">5
</TD></TR><TR><TD> QuikSolution Reagent</TD><TD align="right">0,75
</TD></TR><TR><TD>QuikChangeLightning Enzyme</TD><TD align="right">0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">14,35
</TD></TR><TR><TD>Total volume</TD><TD align="right">25
</TD></TR></TBODY></TABLE><P>
The following PCR program was used:
</P><UL><LI>95 °C 2 minutes</LI><LI>95 °C 20 seconds</LI><LI>60 °C 1 minute</LI><LI>68 °C 2 minutes</LI><LI>Goto step 2 repeat 17 times</LI><LI>68 °C 5 minutes</LI></UL><P><B>Digestion using DpnI:</B>
To digest methylated and hemi-methylated DNA, 1 µl of DpnI was added and the mixture was incubated for 10 minutes at 37 °C. <B>Transformation:</B>
Since XL-10 Gold cells delivered a bacteria lawn on the plates, this time XL1b cells were used.
</P><H3><SPAN class="mw-headline" id="144._labday_09.10.2010"><P style="font-size:17px; background-color:#00dd77;">144. labday 09.10.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Mini-Prep_of_pSB1C3_CD_SDM-PstI"><P style="font-size:15px; background-color:#66bbFF;"><B>Mini-Prep of pSB1C3_CD_SDM-PstI</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P style="color:#66bbff;"><I>Comment</I>: Mini-Prep was performed according to standard protocol.</P><P>Glycerol stocks were prepared:</P><UL><LI>B647 = pSB1C3_CD_SDM-PstI clone 1</LI><LI>B648 = pSB1C3_CD_SDM-PstI clone 2</LI></UL><P>Mini-Prep was performed according to standard protocol:</P><UL><LI>P797 = pSB1C3_CD_SDM-PstI clone 1 c = 8,2 ng/µl</LI><LI>P798 = pSB1C3_CD_SDM-PstI clone 2 c = 7,1 ng/µl</LI></UL><P style="color:#66bbff;"><I>Comment</I>: Since Mini-Prep from XL-10 cells yielded that little plasmid, a re-trafo will be performed using BL21 cells.</P><P><B>Transformation:</B>
A 1:10 dilution of both plasmids was prepared and 1 µl was added to BL21 cells.
Transformation was performed according to standard protocol. Cells were plated on agar plates containing chlorampenicol.
</P><H4><SPAN class="mw-headline" id="Preparation_of_competent_cells_and_Test-Trafo"><P style="font-size:15px; background-color:#66bbFF;"><B>Preparation of competent cells and Test-Trafo</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P>Preparation of BL21 (red line) and XL1-blue (green X) cells was performed according to standard protocol.<B>Do not use cells until further notice!</B></P><H4><SPAN class="mw-headline" id="Test_PCR_of_qPCR_primer_for_phTERT"><P style="font-size:15px; background-color:#66bbFF;"><B>Test PCR of qPCR primer for phTERT</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="color:#66bbff;"><I>Comment</I>: The GOI which is flanked by the ITRs and encapsidated in the capsid is the mGMK_T30 driven by the pHTERT promoter which is only active in several tumor cells. Since we have never tested the primers if they work I wanted to do that. This time I tried to use the TAQ polymerase which we received from Peqlab. Since I could not find a standard protocol for the PCR provided by peqlab I tried to compare several manuals and combine them. </P><P><B>Protocol: </B>
I did not tried the approach with ENhancer Solution. 
</P><P>
I checked the protocol today (10.10) and recognized that I added accidentially to much buffer to the PCR-Mix. Maybe this could be another explanation for the negative results of the PCR.
</P><P>The following PCR was used:
</P><UL><LI>94 °C 3'</LI><LI>94 °C 30&quot;</LI><LI>50 °C 30&quot;</LI><LI>72 °C 10&quot;</LI><LI>Goto 2 repeat 17</LI><LI>68 °C 7'</LI></UL><P><B>Result</B>:

Unfortunately the expected PCR product with a size of 182 bp could not be detected as it can be seen in the agarose gel. I can not say if the protocol needs to be modified or if the primer pair did not anneal adequatly. Therefore, I am going to repeat the PCR tomorrow by using the Phsuion polymerase for checking the results.
</P><H4><SPAN class="mw-headline" id="Cloning_of_p40_in_front_of_VP123"><P style="font-size:15px; background-color:#66bbFF;"><B>Cloning of p40 in front of VP123</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="color:#66bbff;"><I>Comment</I>: We want to test the promoter strenth of several promoters (CMV, P40, p5, p5tataless) and compare them in order to provide the possibility of choosing the proper promoter for each experiment and to titrate the plasmids during transfection.  Therefore, the p40 promoter needs to be fused in front of the VP123 construct in order to compare the CMV and the p40 promter </P><P><B>Protocol: </B></P><P>
The following <B>results</B> of the agarose gel can be seen. The fragments were cut out and a gel extraction has been performed:
</P><P>
For transformation BL21 cells have been used and plated on agar plates containing chloramphenicol.
</P><H4><SPAN class="mw-headline" id="Site-directed_mutagenesis_of_pSB1C3_leftITR_phtert_betaglobin_mgmktk30_hGH_rITR"><P style="font-size:15px; background-color:#66bbFF;"><B>Site-directed mutagenesis of pSB1C3_leftITR_phtert_betaglobin_mgmktk30_hGH_rITR</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="color:#66bbff;"><I>Comment</I>: Since we still have the PstI restriction site within the sequence we need to delete the restriction enzyme site.</P><P><B>Protocol: </B></P><P>
The following PCR was used:
</P><UL><LI>95 °C 2'</LI><LI>95 °C 20&quot;</LI><LI>60 °C 10&quot;</LI><LI>68 °C 2'50&quot;</LI><LI>Goto 2 repeat 17</LI><LI>68 °C 5'</LI></UL><P>After the PCR program was conducted, I added 1 µL of DpnI provided with the Kit to each sample and incubated it for 10 minutes at 37°C. 
For transformation XL-1 cells have been used because the bacterias of the last site-directed mutageneis approaches using XL-10 cells showes extensive grwoth (bacterial lawn). I plated on the transformaned cells on agar plates containing chloramphenicol.
</P><H4><SPAN class="mw-headline" id="Cloning_HSPG-KO_empty_into_pSB1C3-001_pCMV_.5BAAV2.5DVP123_453_RGD_and_pSB1C3-001_pCMV_.5BAAV2.5DVP123_453_Z34C"><P style="font-size:15px; background-color:#66bbFF;"><B>Cloning HSPG-KO_empty into pSB1C3-001_pCMV_[AAV2]VP123_453_RGD and pSB1C3-001_pCMV_[AAV2]VP123_453_Z34C</B></P></SPAN></H4><P><B>Investigator: Achim</B></P><UL><LI>In order to obtain transduction data for our targeting approaches, we still need constructs with RGD/Z34C in the 453 loop as well as HSPG_KO_empty in the 587 loop. 
</LI></UL><UL><LI>I digested pSB1C3_HSPG-KO_empty (P611) as well as pSB1C3-001_pCMV_[AAV2]VP123_453_RGD (P634) and pSB1C3-001_pCMV_[AAV2]VP123_453_Z34C (P641)with BamHI and PvuII and ligated the HSPG-KO_empty motive into both vectors.
</LI></UL><UL><LI>Digestion:20µl,Buffer 4, BSA, 1µl enzyme, 37°
</LI></UL><UL><LI>VB:pSB1C3_001_587_KO_empty (P611) 
</LI><LI>c=160 ng/µl -&gt;14 µl 
</LI></UL><LI>pSB1C3-001_pCMV_[AAV2]VP123_453_RGD (P634)
</LI><LI>c=239 ng/µl (1:10 dilution of midiprep)-&gt;5 µl
</LI><LI>pSB1C3-001_pCMV_[AAV2]VP123_453_Z34C (P641)
</LI><LI>c=229 ng/µl (1:10 dilution of midiprep) -&gt; 5µl
</LI><UL><LI>Dephosphorylation: 1h, 37°C, antarctic phosphatase
</LI></UL><UL><LI>Prep gel:
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:323px;"><DIV class="thumbcaption">the gel picture for the insert has not been saved. But it did show a faint band at ~50 bp.</DIV></DIV></DIV><UL><LI>Ligation: T4
</LI></UL><H4><SPAN class="mw-headline" id="Cellculture_2"><P style="font-size:15px; background-color:#66bbFF;"><B>Cellculture</B></P></SPAN></H4><P><B>Investigator Patrick </B></P>
Transduction of HT1080 and A431. Yesterday 3 6-well plates of HT1080 and A431 were seeded with 200.000 cells into each well. The transductions were performed with the following AAV2s <DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Nothing to complain about, but to compliment on this nice wiki entry! well done pat! ;-)</DIV></DIV></DIV><UL><LI> mVenus without beta-globin
</LI><LI> mVenus without HGH
</LI><LI> mVenus HSPG KO
</LI><LI> mVenus standard
</LI><LI> mVenus new standard
</LI></UL><P>
Expected results:
</P><UL><LI> mVenus without beta-globin: beta globin is said to improve the expression therefore we expect a worse mVenus expression compared to the standard AAVs.
</LI><LI> mVenus without HGH: HGH is responsible for the mRNA polyadenylation of the virus so we expect expect a worse mVenus expression compared to the standard AAVs, too.
</LI><LI> mVenus HSPG KO: the primary AAV2 receptor (the HSPG receptor) was knocked out therefore we expect a worse tranduction rate (about 0,3 to 0,4 of the standard transduction rate, assuming that there are HSPGs on the HT1080 and A431 cell surface). The worse the better.
</LI><LI> mVenus standard: this will be the reference for the other four transduction experiments (no point mutations, GOI: mVenus)
</LI><LI> mVenus new standard: AAV2 with 4 point mutations, inserted rep &amp; cap and &quot;remutated&quot; KpnI restriction site hopefully yiels results similar to the mVenus standard.
</LI></UL><H3><SPAN class="mw-headline" id="145._labday_10.10.2010"><P style="font-size:17px; background-color:#00dd77;">145. labday 10.10.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Repetition_of_Test-PCR_of_qPCR_primer_for_phTERT"><P style="font-size:15px; background-color:#66bbFF;"><B>Repetition of Test-PCR of qPCR primer for phTERT</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="color:#66bbff;"><I>Comment</I>: The GOI which is flanked by the ITRs and encapsidated in the capsid is the mGMK_T30 driven by the pHTERT promoter which is only active in several tumor cells. Since we have never tested the primers if they work I wanted to do that. For several reason I decided to use the Phusion polymerase to perform the PCR. First, we are familiar with the handling of this polymerase and have a protocol which we already used several times and second, since I could not find any standard protocols for the TAQ-polymerase provided by PeqLab, as mentioned yesterday, I will rely on the Phusion polymerase.</P><P><B>Protocol: </B></P><P><B>Results</B>: After PCR was completed, I loaded a 1.5% agarose gel. The results can be seen in the picture: There is a intensive band aat around 150-180bp which correspond to the expected fragment size of 182 bp. We can conclude that the the primers work and that they can be used in the qPCR approaches. One note along the way: I´ve missed the fact that the buffer in which the gel was loaded, was TBE and not TAE. Still the fragment looks well, even though the gel did not appeared to ran normal.</P><H4><SPAN class="mw-headline" id="New_XL1b_and_BL21_cells_ready_to_use.21"><P style="font-size:15px; background-color:#66bbFF;"><B>New XL1b and BL21 cells ready to use!</B></P></SPAN></H4><P><B>Investigator: Stefan </B></P><H4><SPAN class="mw-headline" id="Cloning_Super_Constructs_and_VP2_N-terminal_Fusion_HSPG-KO_constructs_into_pSB1C3_001_CMV_or_pSB1C3_CMV"><P style="font-size:15px; background-color:#66bbFF;"><B>Cloning Super Constructs and VP2 N-terminal Fusion_ HSPG-KO constructs into pSB1C3_001_CMV or pSB1C3_CMV</B></P></SPAN></H4><P><B>Investigator: Hanna </B></P><P style="color:#66bbff;"><I>Comment</I>: In order to send our final VP2 N-terminal Fusion constructs with HSPG konock down and our super constructs to the registry, they need to be cloned into the pSB1C3 backbone. 
We would prefer sending them in the pSB1C3_001 plasmid, because it does its ViralBricks restriction sites were deleted. Therefore one can cut out the 587_KO ViralBrick and replace it by any ViralBrick of interest. 
Because test digestion of pSB1C3_001_CMV didn't show any clear results, I will digest both clones and in addition to that pSB1C3_CMV. </P><P>
Incubation: 2 h, 37°C </P><H4><SPAN class="mw-headline" id="Midi-Prep"><P style="font-size:15px; background-color:#66bbff;">Midi-Prep</P></SPAN></H4><P><B>Investigator: Chris W. </B></P><P style="font-size:13px; color:#003399;"> Midi-Prep of:</P><P>pCerulean_VP1up_NLS_Zegfr:1907_VP2/3-Capins clone 2 =P799 =B401
pSB1C3_CMV_DARPin_MiddleLinker_VP2/3_insCap clone 2 =P800 =B595
pSB1C3_CMV_DARPin_MiddleLinker_VP2/3_HSPG-KO clone 1 =P801 =B596
pSB1C3_left-ITR_phTERT_beta-globin_mGMK_TK30_hGH_right-ITR clone 1 =P802 =B622</P><P>
The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
</P><TABLE border="1"><TBODY><TR><TD> plasmid-no. </TD><TD align="right">P799</TD><TD align="right">P800</TD><TD align="right">P801</TD><TD align="right">P802
</TD></TR><TR><TD> concentration (ng/µl)</TD><TD align="right">662,8 </TD><TD align="right">911,3 </TD><TD align="right">495,6</TD><TD align="right">673,9
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Checking_trafo_plates_of_Viralbrick_approaches.2C_SDM_of_pSB1C3_leftITR_phtert_betaglobin_mgmktk30_hGH_rITR_and_P40_VP123"><P style="font-size:15px; background-color:#66bbFF;"><B>Checking trafo plates of Viralbrick approaches, SDM of pSB1C3_leftITR_phtert_betaglobin_mgmktk30_hGH_rITR and P40_VP123</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="color:#66bbff;"><I>Comment</I>: Unfortunately, no cells grew on the agar plates containing cm. Therefore, we repeat the Trafo ones again and plate them on new agar plates hoping that colonies will grow over night. </P><P>I used the ligations prepared yesterday and transformed new bacterial cells (XL-1B for SDM and BL21 for the other three ligations):
</P><UL><LI>pSB1C3_p40_VP123</LI><LI>P771_SDM (PstI): pSB1C3_leftITR_phtert_betaglobin_mgmktk30_hGH_rITR</LI><LI>pSB1C3-001_pCMV_[AAV2]VP123_453_RGD_587-ko</LI><LI>pSB1C3-001_pCMV_[AAV2]VP123_453_Z34C_587-ko</LI></UL><H4><SPAN class="mw-headline" id="Preparations_for_ELISA"><P style="font-size:15px; background-color:#66bbff;">Preparations for ELISA</P></SPAN></H4><P><B>Investigator: Volker M. </B></P><P>Two 96-well plates that were coated with 200µg A20 antibody per well for 48h at 4°C were washed three times with 200µl PBST Buffer and incubated over night at 4°C with 300µl PBST + 0.2 % BSA.
</P><P>As a preparation 4 liters of PBS buffer were prepared.
</P><H4><SPAN class="mw-headline" id="Mini-Prep_of_pSB1C3_gMK_TK30_SDM-PstI_new"><P style="font-size:15px; background-color:#66bbFF;"><B>Mini-Prep of pSB1C3_gMK_TK30_SDM-PstI_new</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P>Glycerol stocks were prepared:</P><UL><LI>B649 = pSB1C3_gMK_TK30_SDM-PstI_new clone 1</LI><LI>B650 = pSB1C3_gMK_TK30_SDM-PstI_new clone 2</LI></UL><P>Mini-Prep was performed according to standard protocol:</P><UL><LI>P803 = pSB1C3_gMK_TK30_SDM-PstI_new clone 1 c = 282,5 ng/µl</LI><LI>P804 = pSB1C3_gMK_TK30_SDM-PstI_new clone 2 c = 295,6 ng/µl</LI></UL><P>Test digestion:</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>P803 + P804 / µl</B></TD></TR><TR><TD align="left"> DNA   </TD><TD align="left"> 2,5
</TD></TR><TR><TD align="left"> Buffer 4 </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> BSA (10x)  </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> PstI </TD><TD align="left"> 0,3
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 5,2
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>10</B></TD></TR></TBODY></TABLE><P>Gel:
0,5g agarose, 50 ml TAE (1%), 3 µl GELRED, 115 Volt, running time ~50 minutes</P><P style="color:#66bbff;"><I>Comment</I>: Vector was cut using PstI in order to verify successful deletion of the PstI restriction site in TK30. As it can be seen above, both vectors are linearized, no fragment was cut out. In case of failed SDM, one additional band at roughly 450 bp was expected. Since this band does not show up, successful deletion is confirmed.</P><H4><SPAN class="mw-headline" id="Cloning_of_lITR_phTERT_beta-globin_mGMK_TK30_into_pSB1C3_hgH_rITR"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of lITR_phTERT_beta-globin_mGMK_TK30 into pSB1C3_hgH_rITR</B></P></SPAN></H4><P><B>Investigator: Stefan </B></P><P style="font-size:13px; color:red;">Comment: In order to test our GOI with another promotor, lITR_phTERT_beta-globin_mGMK_TK30 needs to be cloned upstream of hgH_rITR. </P><P>Vector name:
pSB1C3_hgH_rITR (P728)</P><P>Insert name:
pSB1C3_lITR_phTERT_beta-globin_mGMK_TK30 (P670)</P><P><B>Digestion:</B></P><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume P670  /µl</B></TD><TD align="right"><B>volume P728  /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">10 </TD><TD align="right">7,7
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right">2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right">2
</TD></TR><TR><TD>Enzyme EcoI</TD><TD align="right">1</TD><TD align="right">1
</TD></TR><TR><TD>Enzyme XbaI</TD><TD align="right">-</TD><TD align="right">1
</TD></TR><TR><TD>Enzyme SpeI</TD><TD align="right">1</TD><TD align="right">-
</TD></TR><TR><TD>H2O</TD><TD align="right">4</TD><TD align="right">6,3
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 20</TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><P><B>Gel:</B>
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 115 Volt</P><P><B>Gel extraction</B>: 
Was performed according to protocol.
</P><P><B>T4 Ligation</B>: </P><TABLE border="1"><TBODY><TR><TD>ligation name </TD><TD align="right">728 + 670
</TD></TR><TR><TD>volume of vector </TD><TD align="right">2,35
</TD></TR><TR><TD>volume of insert</TD><TD align="right">5,65
</TD></TR><TR><TD>T4 ligase buffer (10x)</TD><TD align="right">1
</TD></TR><TR><TD>T4 ligase </TD><TD align="right">1
</TD></TR></TBODY></TABLE><P><B>Transformation</B>: 
Was performed according to standard protocol using BL21 cells.
</P><H4><SPAN class="mw-headline" id="Cloning_of_pSB1C3_lITR_phTERT_beta-globin_CFP_into_pSB1C3_hgH_rITR"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of pSB1C3_lITR_phTERT_beta-globin_CFP into pSB1C3_hgH_rITR </B></P></SPAN></H4><P><B>Investigator: Anna </B></P><P style="font-size:13px; color:red;">Comment: </P><P>Insert name:
</P><UL><LI>pSB1C3_lITR_phTERT_beta-globin_CFP (P791)</LI></UL><P>Vector name:
</P><UL><LI>pSB1C3_hgH_rITR (P728)</LI></UL><P>New vector name:
</P><UL><LI>pSB1C3_lITR_phTERT_beta-globin_CFP_hgH_rITR </LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume for inserts (P666 + P728)  /µl</B></TD><TD align="right"><B>volume of vectors P437 + P715 + P719  /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">7,7 </TD><TD align="right">6,3
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right">2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right">2
</TD></TR><TR><TD>Enzyme EcoRI</TD><TD align="right">1</TD><TD align="right">0,7
</TD></TR><TR><TD>Enzyme XbaI</TD><TD align="right">1</TD><TD align="right">-
</TD></TR><TR><TD>Enzyme SpeI</TD><TD align="right">-</TD><TD align="right">0,7
</TD></TR><TR><TD>Enzyme BmpI</TD><TD align="right">-</TD><TD align="right">0,7
</TD></TR><TR><TD>H2O</TD><TD align="right">6,3</TD><TD align="right">7,6
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 20</TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><P><B>Gel:</B>
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 115 Volt</P><LI>Gel for Vector: See Cloning above.&lt;/li&gt;<B>Gel extraction</B>: 
Was performed according to protocol.

<TABLE border="1"><TBODY><TR><TD>Sample </TD><TD align="right">Vector</TD><TD align="right">Insert
</TD></TR><TR><TD>DNA concentration [ng/µl]</TD><TD align="right">19,6 </TD><TD align="right"> 14,6
</TD></TR><TR><TD>Fragment size [bp]</TD><TD align="right"> 2600 </TD><TD align="right">1800
</TD></TR></TBODY></TABLE><P><B>T4 Ligation</B>: </P><P>Volume(pSB1C3_hGH_rITR)= 6 µl
Volume (lITR_phTERT_beta-globin_CFP_hgH_rITR) = 2 µl
</P><P><B>Transformation</B>: 
Was performed according to standard protocol using BL21 cells.
</P><H4><SPAN class="mw-headline" id="New_ligation_approach_of_Rep52"><P style="font-size:15px; background-color:#66bbFF;"><B>New ligation approach of Rep52</B></P></SPAN></H4><P><B>Investigator: Kira</B>
Since ligation with Quickligase (because of one blunt end)  revealed no colonies on the plate, I decided to perform ligation with T4 ligase. 
Transformation according to the standard protocol with BL21 cells </P><H3><SPAN class="mw-headline" id="146._labday_11.10.2010"><P style="font-size:17px; background-color:#00dd77;">146. labday 11.10.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Colony_PCR_of_Cloning_VP2_Fusion_and_Super_constructs_into_pSB1C3"><P style="font-size:15px; background-color:#66bbff;"><B>Colony PCR of Cloning VP2 Fusion and Super constructs into pSB1C3</B></P></SPAN></H4><P><B>Investigator: Achim, Hanna</B><B>Comment:</B> Because yesterday's cloning didn't deliver a good separartion of the expected gel bands. Nevertheless ligation and trafo was performed. In order to immediately find out, whether we received successful results a colony PCR will be performed. 
Two clones were picked from each plat. In addition to that a positive (pAAV_RC) and a negative control (pSB1C3_lITR) was prepared. 
Used primer: 4200 rev and Cap3500 for. Expected fragment size: 885 bp. 
PCR was performed following the standard protocol. </P><P>
All samples match the positive control!
<B>To do:</B> Mini-Prep and sequencing.
</P><H4><SPAN class="mw-headline" id="Preparation_of_SDS-PAGE_gel_.2810.25.29"><P style="font-size:15px; background-color:#66bbff;"><B>Preparation of SDS-PAGE gel (10%)</B></P></SPAN></H4><P><B>Investigator: Hanna</B><B>Comment:</B> In order to perform a Western Blot of different virus capsids (with and without capsid-motifs), 2 10% SDS polyacrylamid gels were prepared. <B>Resolving gel: 15 mL</B></P></LI><LI> H<SUB>2</SUB>O: 5.9 mL
</LI><LI> Acryl-bisacrylamide mix (30%): 5 mL
</LI><LI> Tris (1.5 M, pH 8.8): 3.8 mL
</LI><LI> SDS (10%): 0.15 mL
</LI><LI> Ammonium persulfate (10%): 0.15 mL
</LI><LI> TEMED: 0.006 mL
</LI><P><B>Stacking gel (5%): 5 mL </B></P><UL><LI> H<SUB>2</SUB>O: 3.4 mL
</LI><LI> Acryl-bisacrylamide mix (30%): 0.83 mL
</LI><LI> Tris (1.5 M, pH 6.8): 0.63 mL
</LI><LI> SDS (10%): 0.05 mL
</LI><LI> Ammonium persulfate (10%): 0.05 mL
</LI><LI> TEMED: 0.005 mL
</LI></UL><H4><SPAN class="mw-headline" id="Colony_PCR_of_p40_VP123_capins"><P style="font-size:15px; background-color:#66bbff;"><B>Colony PCR of p40_VP123_capins</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="color:#66bbff;"><I>Comment</I>: Since the first attempt did not work,and no cells grew on the plate and the same ligation was transformed again into BL-21 and a lot of clones grew on the plate, I decided to perform a colony PCR in order to check several colonies and to inoculate at the same day for a Midi-Prep. </P><P><B>Protocol:</B></P><UL><LI>Primer used: O162</LI><LI>Primer used: O38</LI></UL><P>
The PCR products were loaded on a 1% agarose gel. The results can be seen above in the gel picture: <B>Result: We can see two things: The cloning of p40 to VP123 worked quiet well AND the Robust PCR Kit which was used for the first time worked as well.</B></P><H4><SPAN class="mw-headline" id="Cloning_of_lITR_CMV_betaglobin_and_lITR_phTERT_betaglobin_into_pSB1C3_CD"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of lITR_CMV_betaglobin and lITR_phTERT_betaglobin into pSB1C3_CD</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P style="font-size:13px; color:red;">Comment: To produce another GOI for testing in cell culture, the cytosine deaminase needs to be assembled with lITR_promotor_betaglobin. In the next step hgH_rITR needs to be added.</P><P>Vector name:
pSB1C3_CD clone 1 
pSB1C3_CD clone 2 </P><P>Insert name:
pSB1C3_lITR_CMV_beta-globin (P729)
pSB1C3_lITR_phTERT_beta-globin (P730)</P><P><B>Digestion:</B></P><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume CD clone 1 + 2  /µl</B></TD><TD align="right"><B>volume P729  /µl </B></TD><TD><B>volume P730  /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">6 </TD><TD align="right">14</TD><TD align="right">6
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right">2</TD><TD align="right">2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right">2</TD><TD align="right">2
</TD></TR><TR><TD>Enzyme EcoI</TD><TD align="right">1</TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>Enzyme XbaI</TD><TD align="right">1</TD><TD align="right">- </TD><TD align="right">-
</TD></TR><TR><TD>Enzyme SpeI</TD><TD align="right">-</TD><TD align="right">1</TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">8</TD><TD align="right">- </TD><TD align="right">8
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 20</TD><TD align="right"> 20</TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><P><B>Gel:</B>
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 115 Volt</P><P style="font-size:13px; color:red;"> CD clone 1 yielded to bands around 2500 bp to 3000 bp. Since the vector was cut only using EcoRI and SpeI, it was expected to be linearized, not to be cut into two fragments this size. Therefore, this sample was discarded and cloning was continued using CD clone 2. </P><P><B>Gel extraction</B>: 
Was performed according to protocol.
</P><P><B>T4 Ligation</B>: </P><TABLE border="1"><TBODY><TR><TD>ligation name </TD><TD align="right">729 + CD cl2</TD><TD align="right">730 + CD cl2
</TD></TR><TR><TD>volume of vector </TD><TD align="right">3,67</TD><TD align="right">2,82
</TD></TR><TR><TD>volume of insert</TD><TD align="right">4,33</TD><TD align="right">5,18
</TD></TR><TR><TD>T4 ligase buffer (10x)</TD><TD align="right">1</TD><TD align="right">1
</TD></TR><TR><TD>T4 ligase </TD><TD align="right">1</TD><TD align="right">1
</TD></TR></TBODY></TABLE><P><B>Transformation</B>: 
Was performed according to standard protocol using BL21 cells.
</P><H4><SPAN class="mw-headline" id="Seeding_HT1080_and_A431_for_testing_different_concentrations_of_ganciclovir_by_MTT-Assay"><P style="font-size:15px; background-color:#66bbff;"><B>Seeding HT1080 and A431 for testing different concentrations of ganciclovir by MTT-Assay</B></P></SPAN></H4><P><B>Investigator: Kerstin, Anissa</B></P><UL><LI>Seeding 4x 96-well plates: 2x HT1080 and 2x A431
</LI></UL><H4><SPAN class="mw-headline" id="FACS-Analysis"><P style="font-size:15px; background-color:#66bbff;"><B>FACS-Analysis</B></P></SPAN></H4><P><B>Investigator: Kerstin</B></P><P>...
</P><H4><SPAN class="mw-headline" id="Preparation_of_the_ELISA"><P style="font-size:15px; background-color:#66bbff;"><B>Preparation of the ELISA</B></P></SPAN></H4><P><B>Investigator: Volker</B></P><P>
The AAV particle standard that contains 3.6x10^9 viral particles was dissolved in 500µl as described in the protocol of the Progen AAV Titration ELISA and a absorption spectrum was measured.
These purified and concentrated viral particles could be used for biophysical measurements, there for the possibility to detect the viral particles by absorption was interesting for us.
</P><P>The Spectrum measured in the NanoDrop is the following:
</P><H4><SPAN class="mw-headline" id="Trafo_evaluation_of_Rep52"><P style="font-size:15px; background-color:#66bbff;"><B>Trafo evaluation of Rep52</B></P></SPAN></H4><P><B>Investigator:Kira</B>
T4 ligation seems to worked out. The plate contained many colonies.
</P><H3><SPAN class="mw-headline" id="147._labday_12.10.2010"><P style="font-size:17px; background-color:#00dd77;">147. labday 12.10.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Sequencing_analysis:_pSB1C3_mGMK_TK30_and_pSB1C3_CD"><P style="font-size:15px; background-color:#66bbFF;"><B>Sequencing analysis: pSB1C3_mGMK_TK30 and pSB1C3_CD</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><H4><SPAN class="mw-headline" id="Test_digestion_of_pSB1C3_CD_and_pSB1C3_CD_SDM-PstI_new"><P style="font-size:15px; background-color:#66bbFF;"><B>Test digestion of pSB1C3_CD and pSB1C3_CD_SDM-PstI_new </B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P style="color:#66bbff;"><I>Comment</I>: Sequencing results revealed that there is CFP rather than CD in P819. In order to verify this and exclude the possibility that the wrong plasmid was sent for sequencing test digestions were performed. Additionally the original vector at which the SDM to remove the PstI was performed was digested as well to verify its insert.</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>digest 1 / µl</B></TD><TD align="left"><B>digest 2 / µl</B></TD></TR><TR><TD align="left"> DNA   </TD><TD align="left"> 1</TD><TD align="left"> 7
</TD></TR><TR><TD align="left"> Buffer 4 </TD><TD align="left"> 1</TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> BSA (10x)  </TD><TD align="left"> 1</TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Enzyme I </TD><TD align="left"> SpeI 0,3 </TD><TD align="left"> PstI 0,3
</TD></TR><TR><TD align="left"> Enzyme II </TD><TD align="left"> XbaI 0,3 </TD><TD align="left"> XbaI 0,3
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 6,4</TD><TD align="left"> 0,4
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>10</B></TD><TD align="left"><B>10</B></TD></TR></TBODY></TABLE><P style="color:#66bbff;"><I>Comment</I>: Using these digestion approaches, it was expected for CFP to deliver always bands at ~750 bp. For the CD it was expected to show bands at 90 bp, 1200 bp and 2000 bp if the CD still contains the PstI restictiton site, 1300 bp and 2000 bp if it was removed.</P><P>Gel:
0,5g agarose, 50 ml TAE (1%), 3 µl GELRED, 115 Volt, running time ~50 minutes</P><P style="color:#66bbff;"><I>Comment</I>: As it can be seen, P819 does contain CFP, therefore a mistakenly sequenced plasmid can be excluded. Also it can be seen that P690 contains the CD. Therefore an additonal SDM of P690 to remove the PstI restriction site is necessary.</P><H4><SPAN class="mw-headline" id="qPCR_of_virus_stock:_P816.2F_TKGMk-Standard.3B_R.2FC:_P468_from_5.10._.28VLP_harvested_by_Adrian.29"><P style="font-size:15px; background-color:#66bbff;"><B>qPCR of virus stock: P816/ TKGMk-Standard; R/C: P468 from 5.10. (VLP harvested by Adrian)</B></P></SPAN></H4><P><B>Investigator: Achim</B></P><DIV class="WordSection1"><P class="MsoNormal"><U>Protocol for quantitative real-time PCR of virus
particles</U></P><P class="MsoNormal">Date: 12.10.</P><P class="MsoNormal">Achim</P><P class="MsoNormal">Virus Stock: A.F.; P816; TKGMK-Standard; R/C: P468; 5.10;
consists of supernatant from pelleted cell fragments.</P><P class="MsoNormal">Following the Protocol used by (Rohr et al., 2002).</P><P class="MsoNormal">I digested 5 µl virus dilution with 7,5 µl DNAse I (7,5 units)
and 25 µl 50 mM MgCl2 (end concentration 25 mM) in a final volume of 50 µl at
37°C for 30 min. </P><P class="MsoListParagraph" style="text-indent:-18.0pt"><SPAN style="font-family:
Wingdings">è</SPAN>DNAse
should be heat inactivated at 65°C for 10 min. I forgot that step. The enzyme
should however be inactivated in the initial denaturation step.</P><P class="MsoNormal">I prepared the following PCR reactions:</P><TABLE class="MsoTableGrid" border="1" cellspacing="0" cellpadding="0" style="border-collapse:collapse;border:none"><TBODY><TR><TD width="71" valign="top" style="width:53.3pt;border:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">Sample</P></TD><TD width="51" valign="top" style="width:37.9pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="51" valign="top" style="width:37.9pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">2</P></TD><TD width="51" valign="top" style="width:37.9pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">3</P></TD><TD width="51" valign="top" style="width:37.9pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">4</P></TD><TD width="50" valign="top" style="width:37.85pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5</P></TD><TD width="50" valign="top" style="width:37.85pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">6</P></TD><TD width="51" valign="top" style="width:38.3pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">7</P></TD><TD width="51" valign="top" style="width:38.25pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">8</P></TD><TD width="63" valign="top" style="width:47.25pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"> </P></TD><TD width="51" valign="top" style="width:38.35pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">9</P></TD><TD width="51" valign="top" style="width:38.35pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">10</P></TD></TR><TR><TD width="71" valign="top" style="width:53.3pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"> </P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1.1</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1.2</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">2.1</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">2.2</P></TD><TD width="50" valign="top" style="width:37.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">S.1</P></TD><TD width="50" valign="top" style="width:37.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">S.2</P></TD><TD width="51" valign="top" style="width:38.3pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">NP1</P></TD><TD width="51" valign="top" style="width:38.25pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">NP2</P></TD><TD width="63" valign="top" style="width:47.25pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">MM for samples 1 -8</P></TD><TD width="51" valign="top" style="width:38.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">NH1</P></TD><TD width="51" valign="top" style="width:38.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">NH2</P></TD></TR><TR><TD width="71" valign="top" style="width:53.3pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">PCRmix</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="50" valign="top" style="width:37.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="50" valign="top" style="width:37.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="51" valign="top" style="width:38.3pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="51" valign="top" style="width:38.25pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="63" valign="top" style="width:47.25pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">112.5</P></TD><TD width="51" valign="top" style="width:38.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="51" valign="top" style="width:38.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD></TR><TR><TD width="71" valign="top" style="width:53.3pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">Primer for</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="50" valign="top" style="width:37.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="50" valign="top" style="width:37.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="51" valign="top" style="width:38.3pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="51" valign="top" style="width:38.25pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="63" valign="top" style="width:47.25pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">9</P></TD><TD width="51" valign="top" style="width:38.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">-</P></TD><TD width="51" valign="top" style="width:38.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">-</P></TD></TR><TR><TD width="71" valign="top" style="width:53.3pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">Primer rev</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="50" valign="top" style="width:37.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="50" valign="top" style="width:37.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="51" valign="top" style="width:38.3pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="51" valign="top" style="width:38.25pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="63" valign="top" style="width:47.25pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">9</P></TD><TD width="51" valign="top" style="width:38.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">-</P></TD><TD width="51" valign="top" style="width:38.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">-</P></TD></TR><TR><TD width="71" valign="top" style="width:53.3pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">Template </P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">2</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">2</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">2</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">2</P></TD><TD width="50" valign="top" style="width:37.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5</P></TD><TD width="50" valign="top" style="width:37.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5</P></TD><TD width="51" valign="top" style="width:38.3pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">-</P></TD><TD width="51" valign="top" style="width:38.25pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">-</P></TD><TD width="63" valign="top" style="width:47.25pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">-</P></TD><TD width="51" valign="top" style="width:38.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">-</P></TD><TD width="51" valign="top" style="width:38.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">-</P></TD></TR><TR><TD width="71" valign="top" style="width:53.3pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">H2O</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">8.5</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">8.5</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">8.5</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">8.5</P></TD><TD width="50" valign="top" style="width:37.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5.5</P></TD><TD width="50" valign="top" style="width:37.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5.5</P></TD><TD width="51" valign="top" style="width:38.3pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">10.5</P></TD><TD width="51" valign="top" style="width:38.25pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">10.5</P></TD><TD width="63" valign="top" style="width:47.25pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">-</P></TD><TD width="51" valign="top" style="width:38.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="51" valign="top" style="width:38.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD></TR><TR><TD width="71" valign="top" style="width:53.3pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">Total</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="51" valign="top" style="width:37.9pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="50" valign="top" style="width:37.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="50" valign="top" style="width:37.85pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="51" valign="top" style="width:38.3pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="51" valign="top" style="width:38.25pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="63" valign="top" style="width:47.25pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"> </P></TD><TD width="51" valign="top" style="width:38.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="51" valign="top" style="width:38.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD></TR></TBODY></TABLE><P class="MsoNormal"> </P><TABLE class="MsoTableGrid" border="1" cellspacing="0" cellpadding="0" style="border-collapse:collapse;border:none"><TBODY><TR><TD width="214" valign="top" style="width:160.35pt;border:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">Step</P></TD><TD width="214" valign="top" style="width:160.35pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">Time</P></TD><TD width="214" valign="top" style="width:160.4pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">Temp</P></TD></TR><TR><TD width="214" valign="top" style="width:160.35pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">Initial denaturation</P></TD><TD width="214" valign="top" style="width:160.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">7’</P></TD><TD width="214" valign="top" style="width:160.4pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">95°C</P></TD></TR><TR><TD width="214" valign="top" style="width:160.35pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"> </P></TD><TD width="214" valign="top" style="width:160.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"> </P></TD><TD width="214" valign="top" style="width:160.4pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"> </P></TD></TR><TR><TD width="214" valign="top" style="width:160.35pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">Denaturation</P></TD><TD width="214" valign="top" style="width:160.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">10’’</P></TD><TD width="214" valign="top" style="width:160.4pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">95°C</P></TD></TR><TR><TD width="214" valign="top" style="width:160.35pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">Annealing/Extension</P></TD><TD width="214" valign="top" style="width:160.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">30’’</P></TD><TD width="214" valign="top" style="width:160.4pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">60°C</P></TD></TR><TR><TD width="214" valign="top" style="width:160.35pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">Number of Cycles</P></TD><TD width="214" valign="top" style="width:160.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">45</P></TD><TD width="214" valign="top" style="width:160.4pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"> </P></TD></TR></TBODY></TABLE><P class="MsoNormal"> </P><P class="MsoNormal"><U>Results:</U></P><P class="MsoListParagraphCxSpFirst" style="text-indent:-18.0pt"><SPAN style="font-family:Wingdings">è</SPAN>Both
negative controls containing primers were contaminated with DNA.</P><P class="MsoListParagraphCxSpMiddle" style="text-indent:-18.0pt"><SPAN style="font-family:Wingdings">è</SPAN>Negative
controls containing only H2O were negative.</P><P class="MsoListParagraphCxSpMiddle" style="text-indent:-18.0pt"><SPAN style="font-family:Wingdings">è</SPAN>Values
for plasmids/reaction vary; from 1 – 100 for the same virus stock</P><P class="MsoListParagraphCxSpMiddle" style="text-indent:-18.0pt"><SPAN style="font-family:Wingdings;color:red">è</SPAN>Dilution of the virus solution: 1:10; 2µl PCR sample: <B><U><SPAN style="color:red">3.2 * 10^5 virus particles/ml</SPAN></U></B></P><P class="MsoListParagraphCxSpLast" style="text-indent:-18.0pt"><SPAN style="font-family:Wingdings">è</SPAN>Because
of the contaminated negative controls and the high deviation, I will repeat the
experiment tomorrow to verify the results.</P><P class="MsoNormal"><SPAN lang="DE">References:</SPAN></P><P><SPAN lang="DE" style="font-size:11.0pt;font-family:&quot;Calibri&quot;,&quot;sans-serif&quot;">Rohr,
U., Wulf, M., Stahn, S., Steidl, U., Haas, R., Kronenwett, R., et al. </SPAN><SPAN style="font-size:11.0pt;font-family:&quot;Calibri&quot;,&quot;sans-serif&quot;">(2002). Fast and
reliable titration of recombinant adeno-associated virus type-2 using
quantitative real-time PCR. <I>Journal of virological methods</I>, <I>106</I>(1),
81-8. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/12367732.</SPAN></P><P class="MsoNormal"> </P></DIV><H4><SPAN class="mw-headline" id="Cloning_sr39_into_mGMK_TK30"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning sr39 into mGMK_TK30</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="color:#66bbff;"><I>Comment</I>: We do not only want to submit the tk30, a mutant version of the thymidine kinase, but as well another mutant protein which is named sr39. SR39 seems to have a better activity than the tk30 without being fused to the guanylate kinase (mgmk), but the sr39 also works quite well as a fusion protein. We ordered one part of the sr39 and we are now subcloning it into the mgmnk_tk30 construct in order to obtain mgmk_sr39. </P><P><B>Protocol:</B></P><P>
The digested fragments were loaded on a 1.2% agarose gel. The results can be seen above in the gel picture: I cut out the small visible band in the right lane, and the upper intensive band of the left lane. <B>Result: </B></P><P>
After gel extraction has been performed, I ligated the two fragments and transformed BL-21 cells, plated them on cm agar plates and incubated them over night at 37°C.
</P><H4><SPAN class="mw-headline" id="Concentration_of_virus_stock_for_Western_Blot_analysis"><P style="font-size:15px; background-color:#66bbff;"><B>Concentration of virus stock for Western Blot analysis</B></P></SPAN></H4><P><B>Investigator: Hanna </B>
For the first Western Blot try, we investigate whether is is enough to concentrate the virus stock (from cell supernatant) and then to simply load it onto a SDS-PAGE gel and perfom immuno-staining. 
For this purpose 8 mL virus stock (pHelper, pAAV_RC, GOI=YFP) were concentrated via amicon filtration. </P><UL><LI> First, amicon filter was washed with 4 mL PBS.
</LI><LI> Then 4 mL virus stock was loaded onto the filter and centrifuged at 3000 rpm for 15 minutes.
</LI><LI> Flow-through was discarded and additional virus stock was loaded, centrifuged at 3000 rpm for 25 minutes.
</LI><LI> Flow-through was discarded, remaining solution was re-suspended and additional virus stock was added. Amicon filter was centrifuged at 3000 rpm for 50 minutes.
</LI><LI> Flow-through was discarded. Residual protein solution: 500 µL. Filter was washed with 3 mL PBS, centrifuging 35 minutes two times.  
</LI><LI> 2 x 40 µL protein solution (16x) was taken and mixed with Laemmli buffer and stored over night @ 4°C.
</LI></UL><P><B>To do:</B> SDS-PAGE and Western Blot tomorrow. 
Perfomance of another try by harvesting viruses of the cell pellet with RIPA buffer &amp; freeze/ thaw. 
Western Blot of unmodified, and modified capsids (N-terminal fusion and VP1 insertion with CFP ref. mVenus) --&gt; size shift should be detectable.</P><H4><SPAN class="mw-headline" id="MTT_Assay_.28First_Try.29_Transduction_of_HT1080_and_A431"><P style="font-size:15px; background-color:#66bbff;"><B>MTT Assay (First Try) Transduction of HT1080 and A431</B></P></SPAN></H4><P><B>Investigator: Kerstin, Anissa </B></P><P><B> Results: </B></P><H4><SPAN class="mw-headline" id="Mini-Prep_and_test_digestion_of_several_constructs_2"><P style="font-size:15px; background-color:#66bbFF;"><B>Mini-Prep and test digestion of several constructs</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P>Glycerol stocks were prepared:</P><UL><LI>B651 = pSB1C3_P40_VP123</LI><LI>B652 = pSB1C3_lITR_phTERT_ß-globin_mGMK_TK30_PstI_SDM_hgH_rITR</LI><LI>B653 = pSB1C3_lITR_phTERT_ß-globin_mGMK_TK30_PstI_SDM_hgH_rITR</LI><LI>B654 = pSB1C3_lITR_CMV_ß-globin_mGMK_TK30_hgH_rITR</LI><LI>B655 = pSB1C3_lITR_CMV_ß-globin_mGMK_TK30_hgH_rITR</LI><LI>B656 = pSB1C3_CMV_VP123_453_Z34C_HSPG-KO</LI><LI>B657 = pSB1C3_CMV_VP123_453_Z34C_HSPG-KO</LI><LI>B658 = pSB1C3_CMV_VP123_453_RGD_HSPG-KO</LI><LI>B659 = pSB1C3_CMV_VP123_453_RGD_HSPG-KO</LI><LI>B660 = pSB1C3_lITR_phTERT_ß-globin_CFP_hgH-rITR</LI><LI>B661 = pSB1C3_lITR_phTERT_ß-globin_CFP_hgH-rITR</LI></UL><P>Mini-Prep was performed according to standard protocol:</P><UL><LI>P821 = pSB1C3_P40_VP123 c = 327,4 ng/µl</LI><LI>P822 = pSB1C3_lITR_phTERT_ß-globin_mGMK_TK30_pSTI_SDM_hgH_rITR c = 342,9 ng/µl</LI><LI>P823 = pSB1C3_lITR_phTERT_ß-globin_mGMK_TK30_pSTI_SDM_hgH_rITR c = 324,0 ng/µl</LI><LI>P824 = pSB1C3_lITR_CMV_ß-globin_mGMK_TK30_hgH_rITR c = 81,6 ng/µl</LI><LI>P825 = pSB1C3_lITR_CMV_ß-globin_mGMK_TK30_hgH_rITR c = 148,4 ng/µl</LI><LI>P826 = pSB1C3_CMV_VP123_453_Z34C_HSPG-KO c = 105,8 ng/µl</LI><LI>P827 = pSB1C3_CMV_VP123_453_Z34C_HSPG-KO c = 121,1 ng/µl</LI><LI>P828 = pSB1C3_CMV_VP123_453_RGD_HSPG-KO c = 188,4 ng/µl</LI><LI>P829 = pSB1C3_CMV_VP123_453_RGD_HSPG-KO c = 140,4 ng/µl</LI><LI>P830 = pSB1C3_lITR_phTERT_ß-globin_CFP_hgH-rITR c = 197,2 ng/µl</LI><LI>P831 = pSB1C3_lITR_phTERT_ß-globin_CFP_hgH-rITR c = 207,3 ng/µl</LI></UL><P>Test digestion:</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>P822 + P823 / µl</B></TD><TD align="left"><B>P824 + P825 / µl</B></TD><TD align="left"><B>P830 + P831 / µl</B></TD></TR><TR><TD align="left"> DNA   </TD><TD align="left"> 1,5</TD><TD align="left"> 1,5</TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> Buffer 4 </TD><TD align="left"> 1</TD><TD align="left"> 1</TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> BSA (10x)  </TD><TD align="left"> 1</TD><TD align="left"> 1</TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> PstI </TD><TD align="left"> 0,3 </TD><TD align="left"> - </TD><TD align="left"> -
</TD></TR><TR><TD align="left"> XbaI </TD><TD align="left"> 0,3 </TD><TD align="left"> 0,3 </TD><TD align="left"> 0,3
</TD></TR><TR><TD align="left"> SpeI </TD><TD align="left"> - </TD><TD align="left"> 0,3 </TD><TD align="left"> 0,3
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 5,9</TD><TD align="left"> 5,9</TD><TD align="left"> 5,4
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>10</B></TD><TD align="left"><B>10</B></TD><TD align="left"><B>10</B></TD></TR></TBODY></TABLE><P>Gel:
0,5g agarose, 50 ml TAE (1%), 3 µl GELRED, 115 Volt, running time ~50 minutes</P><P style="color:#66bbff;"><I>Comment</I>: P822 and P823 smear a lot, therefore evaluation is difficult. Another test digestion would be reasonable. P824 contains hgH_rITR, therefore it will be discarded. P825 looks well and was sent for sequencing.
P830 and P831 look well and will be sent for sequencing tomorrow.</P><H4><SPAN class="mw-headline" id="Repetiton:_Site-directed_mutagenesis_in_pSB1C3_CD_construct"><P style="font-size:15px; background-color:#66bbFF;"><B>Repetiton: Site-directed mutagenesis in pSB1C3_CD construct</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P style="color:#66bbff;"><I>Comment</I>: Sequencing revealed that there is no CD in our pSB1C3_CD but rather CFP. So, anonther approach of the SDM needs to be performed.</P><P><B>Protocol using the QuickChange Lightning Kit from Stratagene: </B><B>Ingredients: </B></P><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD align="right">Volume / µl
</TD></TR><TR><TD> 10x reaction buffer </TD><TD align="right">2,5
</TD></TR><TR><TD> dNTP</TD><TD align="right">0,5
</TD></TR><TR><TD> forward primer: O191 </TD><TD align="right">0,56
</TD></TR><TR><TD> reverse primer: O192 </TD><TD align="right">0,57
</TD></TR><TR><TD> DNA Template</TD><TD align="right">0,5 (1:10 dilution)
</TD></TR><TR><TD> QuikSolution Reagent</TD><TD align="right">0,75
</TD></TR><TR><TD>QuikChangeLightning Enzyme</TD><TD align="right">0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">14,35
</TD></TR><TR><TD>Total volume</TD><TD align="right">25
</TD></TR></TBODY></TABLE><P>
The following PCR program was used:
</P><UL><LI>95 °C 120 s</LI><LI>95 °C 20 s</LI><LI>60 °C 60 s</LI><LI>68 °C 105 s</LI><LI>Goto step 2 repeat 17 times</LI><LI>68 °C 300 s</LI></UL><P><B>Digestion using DpnI:</B>
To digest methylated and hemi-methylated DNA, 1 µl of DpnI was added and the mixture was incubated for 10 minutes at 37 °C. <B>Transformation:</B>
Trafo was performed according to protocol, but using XL1b cells instead of XL-10 Gold cells.
</P><H3><SPAN class="mw-headline" id="148._labday_13.10.2010"><P style="font-size:17px; background-color:#00dd77;">148. labday 13.10.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="qPCRs_of_virus_stocks"><P style="font-size:15px; background-color:#66bbFF;"><B>qPCRs of virus stocks</B></P></SPAN></H4><P><B>Investigator: Achim</B></P><DIV class="WordSection1"><TABLE class="MsoTableGrid" border="1" cellspacing="0" cellpadding="0" style="border-collapse:collapse;border:none"><TBODY><TR><TD width="450" valign="top" style="width:337.6pt;border:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">Sample:</P></TD><TD width="181" valign="top" style="width:135.6pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">Plasmid, Glycerol stock:</P></TD><TD width="181" valign="top" style="width:135.6pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">Sample No.</P></TD></TR><TR><TD width="450" valign="top" style="width:337.6pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"><SPAN style="color:black">pSB1C3_001_RC_IRCK_HSPG-ko_P5tataless_RFC10
  clone 1</SPAN></P></TD><TD width="181" valign="top" style="width:135.6pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">B561</P></TD><TD width="181" valign="top" style="width:135.6pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD></TR><TR><TD width="450" valign="top" style="width:337.6pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"><SPAN style="font-size:10.0pt;color:black">pSB1C3_001_RC_IRCK_P5tataless
  clone1</SPAN></P></TD><TD width="181" valign="top" style="width:135.6pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">P668</P></TD><TD width="181" valign="top" style="width:135.6pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">2</P></TD></TR><TR><TD width="450" valign="top" style="width:337.6pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"><SPAN style="font-size:10.0pt;color:black">pSB1C3_lITR_CMV_beta-globin_mVenus_rITR
  clone1</SPAN></P></TD><TD width="181" valign="top" style="width:135.6pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">P378</P></TD><TD width="181" valign="top" style="width:135.6pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">3</P></TD></TR><TR><TD width="450" valign="top" style="width:337.6pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"><SPAN style="font-size:10.0pt;color:black">pSB1C3_lITR_CMV_mVenus_hGH_rITR
  clone1</SPAN></P></TD><TD width="181" valign="top" style="width:135.6pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">P377</P></TD><TD width="181" valign="top" style="width:135.6pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">4</P></TD></TR><TR><TD width="450" valign="top" style="width:337.6pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"><B><SPAN style="font-size:10.0pt;color:black">pAAV_RC_RepCapIns_SDMKpnI</SPAN></B></P></TD><TD width="181" valign="top" style="width:135.6pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">P468</P></TD><TD width="181" valign="top" style="width:135.6pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5</P></TD></TR></TBODY></TABLE><P class="MsoNormal">                                                                                                </P><P class="MsoNormal">Digestion: 37°C, 30 min</P><TABLE class="MsoTableGrid" border="1" cellspacing="0" cellpadding="0" style="border-collapse:collapse;border:none"><TBODY><TR><TD width="134" valign="top" style="width:100.75pt;border:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"> </P></TD><TD width="128" valign="top" style="width:96.2pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="128" valign="top" style="width:96.15pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">2</P></TD><TD width="128" valign="top" style="width:96.15pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">3</P></TD><TD width="128" valign="top" style="width:96.15pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">4</P></TD><TD width="127" valign="top" style="width:95.4pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5</P></TD><TD width="126" valign="top" style="width:94.45pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">MM x6</P></TD></TR><TR><TD width="134" valign="top" style="width:100.75pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">Virus solution</P></TD><TD width="128" valign="top" style="width:96.2pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5</P></TD><TD width="128" valign="top" style="width:96.15pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5</P></TD><TD width="128" valign="top" style="width:96.15pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5</P></TD><TD width="128" valign="top" style="width:96.15pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5</P></TD><TD width="127" valign="top" style="width:95.4pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5</P></TD><TD width="126" valign="top" style="width:94.45pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"> </P></TD></TR><TR><TD width="134" valign="top" style="width:100.75pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">DNAseI (7,5 U)</P></TD><TD width="128" valign="top" style="width:96.2pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">7,5</P></TD><TD width="128" valign="top" style="width:96.15pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">7,5</P></TD><TD width="128" valign="top" style="width:96.15pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">7,5</P></TD><TD width="128" valign="top" style="width:96.15pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">7,5</P></TD><TD width="127" valign="top" style="width:95.4pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">7,5</P></TD><TD width="126" valign="top" style="width:94.45pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">45</P></TD></TR><TR><TD width="134" valign="top" style="width:100.75pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">MgCl2 50 mM</P></TD><TD width="128" valign="top" style="width:96.2pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5</P></TD><TD width="128" valign="top" style="width:96.15pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5</P></TD><TD width="128" valign="top" style="width:96.15pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5</P></TD><TD width="128" valign="top" style="width:96.15pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5</P></TD><TD width="127" valign="top" style="width:95.4pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5</P></TD><TD width="126" valign="top" style="width:94.45pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">30</P></TD></TR><TR><TD width="134" valign="top" style="width:100.75pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">H2O</P></TD><TD width="128" valign="top" style="width:96.2pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">32.5</P></TD><TD width="128" valign="top" style="width:96.15pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">32.5</P></TD><TD width="128" valign="top" style="width:96.15pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">32.5</P></TD><TD width="128" valign="top" style="width:96.15pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">32.5</P></TD><TD width="127" valign="top" style="width:95.4pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">32.5</P></TD><TD width="126" valign="top" style="width:94.45pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">195</P></TD></TR><TR><TD width="134" valign="top" style="width:100.75pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">Total</P></TD><TD width="128" valign="top" style="width:96.2pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">50</P></TD><TD width="128" valign="top" style="width:96.15pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">50</P></TD><TD width="128" valign="top" style="width:96.15pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">50</P></TD><TD width="128" valign="top" style="width:96.15pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">50</P></TD><TD width="127" valign="top" style="width:95.4pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">50</P></TD><TD width="126" valign="top" style="width:94.45pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">300</P></TD></TR></TBODY></TABLE><P class="MsoNormal"> </P><P class="MsoNormal">PCR reaction:</P><TABLE class="MsoTableGrid" border="1" cellspacing="0" cellpadding="0" width="861" style="width:645.85pt;border-collapse:collapse;border:none"><TBODY><TR style="height:13.75pt"><TD width="71" valign="top" style="width:53.2pt;border:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">Sample</P></TD><TD width="40" valign="top" style="width:30.35pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">2</P></TD><TD width="40" valign="top" style="width:30.35pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">3</P></TD><TD width="40" valign="top" style="width:30.35pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">4</P></TD><TD width="40" valign="top" style="width:30.35pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5</P></TD><TD width="40" valign="top" style="width:30.35pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">6</P></TD><TD width="40" valign="top" style="width:30.35pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">7</P></TD><TD width="40" valign="top" style="width:30.35pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">8</P></TD><TD width="40" valign="top" style="width:30.35pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">9</P></TD><TD width="40" valign="top" style="width:30.35pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">10</P></TD><TD width="40" valign="top" style="width:30.35pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">11</P></TD><TD width="42" valign="top" style="width:31.65pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12</P></TD><TD width="40" valign="top" style="width:30.35pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">13</P></TD><TD width="40" valign="top" style="width:30.35pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">14</P></TD><TD width="115" valign="top" style="width:85.95pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"> </P></TD><TD width="47" valign="top" style="width:35.4pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">15</P></TD><TD width="60" valign="top" style="width:45.1pt;border:solid windowtext 1.0pt;
  border-left:none;padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">16</P></TD></TR><TR style="height:41.2pt"><TD width="71" valign="top" style="width:53.2pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt;height:41.2pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"> </P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:41.2pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1.1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:41.2pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1.2</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:41.2pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">2.1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:41.2pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">2.2</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:41.2pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">3.1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:41.2pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">3.2</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:41.2pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">4.1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:41.2pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">4.2</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:41.2pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5.1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:41.2pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5.2</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:41.2pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">S.1</P></TD><TD width="42" valign="top" style="width:31.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:41.2pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">S.2</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:41.2pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">NP1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:41.2pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">NP2</P></TD><TD width="115" valign="top" style="width:85.95pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:41.2pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">MM for samples 1 -14 (x15)</P></TD><TD width="47" valign="top" style="width:35.4pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:41.2pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">NH1</P></TD><TD width="60" valign="top" style="width:45.1pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:41.2pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">NH2</P></TD></TR><TR style="height:12.85pt"><TD width="71" valign="top" style="width:53.2pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">PCRmix</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="42" valign="top" style="width:31.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="115" valign="top" style="width:85.95pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">187,5</P></TD><TD width="47" valign="top" style="width:35.4pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="60" valign="top" style="width:45.1pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD></TR><TR style="height:27.45pt"><TD width="71" valign="top" style="width:53.2pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">Primer for 1:10</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="42" valign="top" style="width:31.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="115" valign="top" style="width:85.95pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">15</P></TD><TD width="47" valign="top" style="width:35.4pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">-</P></TD><TD width="60" valign="top" style="width:45.1pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">-</P></TD></TR><TR style="height:27.45pt"><TD width="71" valign="top" style="width:53.2pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">Primer rev 1:10</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="42" valign="top" style="width:31.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">1</P></TD><TD width="115" valign="top" style="width:85.95pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">15</P></TD><TD width="47" valign="top" style="width:35.4pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">-</P></TD><TD width="60" valign="top" style="width:45.1pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:27.45pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">-</P></TD></TR><TR style="height:12.85pt"><TD width="71" valign="top" style="width:53.2pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">Template </P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">2</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">2</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">2</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">2</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">2</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">2</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">2</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">2</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">2</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">2</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5</P></TD><TD width="42" valign="top" style="width:31.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">-</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">-</P></TD><TD width="115" valign="top" style="width:85.95pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">-</P></TD><TD width="47" valign="top" style="width:35.4pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">-</P></TD><TD width="60" valign="top" style="width:45.1pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:12.85pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">-</P></TD></TR><TR style="height:13.75pt"><TD width="71" valign="top" style="width:53.2pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">H2O</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">8.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">8.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">8.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">8.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">8.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">8.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">8.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">8.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">8.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">8.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5.5</P></TD><TD width="42" valign="top" style="width:31.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">5.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">10.5</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">10.5</P></TD><TD width="115" valign="top" style="width:85.95pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">-</P></TD><TD width="47" valign="top" style="width:35.4pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD><TD width="60" valign="top" style="width:45.1pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">12.5</P></TD></TR><TR style="height:13.75pt"><TD width="71" valign="top" style="width:53.2pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">Total</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="42" valign="top" style="width:31.65pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="40" valign="top" style="width:30.35pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="115" valign="top" style="width:85.95pt;border-top:none;border-left:
  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal"> </P></TD><TD width="47" valign="top" style="width:35.4pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD><TD width="60" valign="top" style="width:45.1pt;border-top:none;border-left:none;
  border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
  padding:0cm 5.4pt 0cm 5.4pt;height:13.75pt"><P class="MsoNormal" style="margin-bottom:0cm;margin-bottom:.0001pt;line-height:
  normal">25</P></TD></TR></TBODY></TABLE><P class="MsoNormal"> </P></DIV><P>Results:
</P><H4><SPAN class="mw-headline" id="SDS_Page_of_unmodified_AAV2_capsids"><P style="font-size:15px; background-color:#66bbFF;"><B>SDS Page of unmodified AAV2 capsids</B></P></SPAN></H4><P><B>Investigator: Hanna</B></P><P><B>Comment:</B> Yesterday a 8x concentration of AAV2 supernatant-stock was performed. 10 µL Laemmli-buffer was added to 40 µL AAV stock (8x) and stored over night at 4°C. 
Today a SDS-PAGE of these samples will be performed. 
Samples were incubated for 10 minutes at 95°C for denaturation. <B>Loading plan:</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>5 µL PageRuler Prestained Protein Ladder</B></TD><TD align="left"><B>10 µL Sample</B></TD><TD align="left"><B>15 µL Sample</B></TD><TD align="left"><B>25 µL Sample</B></TD></TR></TBODY></TABLE><P>
After running front reached the end of the gel, PAGE was stopped. <B>Western Blot:</B>
Three layers of transfer buffer soaked whatman paper were placed onto the blotter. A polyvinylidene difluoride membrane was activated by methanol (5 seconds) and then incubated for 5 minutes in transfer buffer. Membrane was then placed onto the whatman paper, gel was carefully layed on top. Air bubbles were removed, ensuring elictricity flow. After coverage by 3 further layers of whatman, blotting was performed at 200 mA for 1 hour. <B>Blocking:</B>
In order to minimize nonspecific antibody interactions, membrane was incubated in blocking buffer (TBS (pH 7.5), 5% milk powder) for one hour, shaking. <B>Primary antibody incubation:</B>
A 1:10 dilution of the B1 antibody in TBS + 1% milk was prepared. Membrane was incubated in 50 mL falcon over night at 4°C, rotating.
</P><H4><SPAN class="mw-headline" id="Midi-Prep_2"><P style="font-size:15px; background-color:#66bbff;">Midi-Prep</P></SPAN></H4><P><B>Investigator: Chris W. </B></P><P style="font-size:13px; color:#003399;"> Midi-Prep of:</P><P>pSB1C3_CMV_VP123_453_Z34C =P832 =B656
pSB1C3_CMV_VP123_453_RGD_HSPG-KO =P833 =B658
pSB1C3_001_RC_IRCK_HSPG-ko_P5tataless_RFC10 clone 1 =P834 =B561
pSB1C3_001_RC_IRCK_P5tataless clone 1 =P835 =B516
pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR clone1 =P836 =B200
pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR clone1 =P837 =B200</P><P>
The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
</P><TABLE border="1"><TBODY><TR><TD> plasmid-no. </TD><TD align="right">P832</TD><TD align="right">P833</TD><TD align="right">P834</TD><TD align="right">P835</TD><TD align="right">P836</TD><TD align="right">P837
</TD></TR><TR><TD> concentration (ng/µl)</TD><TD align="right">3615,8 </TD><TD align="right">1504,7 </TD><TD align="right">x</TD><TD align="right">x</TD><TD align="right">x</TD><TD align="right">x
</TD></TR></TBODY></TABLE><P style="font-size:13px; color:#003399;"> Comment: P834-837 given to sven</P><H4><SPAN class="mw-headline" id="Repetiton:_Test_digestion_of_pSB1C3_lITR_phTERT_betaglobin_mGMK_TK30_SDM-PstI_hgH_rITR"><P style="font-size:15px; background-color:#66bbFF;"><B>Repetiton: Test digestion of pSB1C3_lITR_phTERT_betaglobin_mGMK_TK30_SDM-PstI_hgH_rITR</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P style="color:#66bbff;"><I>Comment</I>: Repetiton of yesterday's test digestion which yielded no clear band at 2000 bp as it was expected. </P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>P822/P823 / µl</B></TD></TR><TR><TD align="left"> DNA   </TD><TD align="left"> 1,5
</TD></TR><TR><TD align="left"> Buffer 4 </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> BSA (10x)  </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> SpeI </TD><TD align="left"> 0,3
</TD></TR><TR><TD align="left"> XbaI </TD><TD align="left"> 0,3
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 5,9
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>10</B></TD></TR></TBODY></TABLE><P>
Gel:
0,45g agarose, 50 ml TAE (0,9%), 3 µl GELRED, 90 Volt, running time ~110 minutes</P><P style="color:#66bbff;"><I>Comment</I>: This time both samples look well. P822 will be sent for sequencing.</P><H4><SPAN class="mw-headline" id="Sequencing_analysis:_pSB1C3_leftITR_pCMV_betaglobin_mGMK_TK30_hgH_rITR_.28P825.29"><P style="font-size:15px; background-color:#66bbFF;"><B>Sequencing analysis: pSB1C3_leftITR_pCMV_betaglobin_mGMK_TK30_hgH_rITR (P825)</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P>For sequencing the following primers were used_
</P><UL><LI>qPCR primer CMV for (O20)</LI><LI>GOI primer for (O30)</LI><LI>seq. primer TK GMK for (O23)</LI><LI>TK30 SDM Pst for (O191)</LI></UL><P>Sequencing comfirmed that the pysical sequence fits our theoretical. Next step will be to remove the remaining PstI restriction site within TK30.
</P><H4><SPAN class="mw-headline" id="Repetition_of_cloning_of_Rep_into_Rep78"><P style="font-size:15px; background-color:#66bbFF;"><B>Repetition of cloning of Rep into Rep78</B></P></SPAN></H4><P><B>Investigator: Kira</B><B>Comment:</B> The last 2 approaches to digest Rep78 and to clone failed because Rep78 was almost gone after PCR purification and was not detectable on the gel, even if I used the double amount of DNA. Thus, new over night culture was grown in order to perform the experiment again </P><P>Regarding the activity conditions of the enzymes, digestion was performed in 2 steps.  </P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>Rep78</B></TD></TR><TR><TD align="left"> DNA   </TD><TD align="left"> 3 µl
</TD></TR><TR><TD align="left"> BSA (100x) </TD><TD align="left"> 2 µl
</TD></TR><TR><TD align="left"> Buffer no. 3  </TD><TD align="left"> 2,0 µl
</TD></TR><TR><TD align="left"> Enzyme SwaI  </TD><TD align="left">1,0 µl
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 12 µl
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>20</B></TD></TR></TBODY></TABLE><P>
incubation @ 25 C for approx. 2 h 
Dephosphorylation was performed because of the blunt cleavage. PCR phosphorylation followed in order to get rid off the buffer because Buffer 3 is not appropriate for further digestion.
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>Rep78</B></TD></TR><TR><TD align="left"> DNA   </TD><TD align="left"> 30 µl
</TD></TR><TR><TD align="left"> BSA (100x) </TD><TD align="left"> 0 µl
</TD></TR><TR><TD align="left"> Buffer no. 2  </TD><TD align="left"> 4,5 µl
</TD></TR><TR><TD align="left"> Enzyme HindIII  </TD><TD align="left">1,0 µl
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 9,5 µl
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>45</B></TD></TR></TBODY></TABLE><P>incubation for 2h at 37C</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><A href="/File:Rep78_2010_10_13.jpg" title="File:Rep78 2010 10 13.jpg">File:Rep78 2010 10 13.jpg</A><DIV class="thumbcaption">600px</DIV></DIV></DIV></DIV><P>Ligation with T4 ligase </P><P>v(insert)=4,4 ul
v(vector)= 3,6 ul
T4 buffer= 1 ul
T4 ligase= 1 ul</P><P>incubation at RT for 45 min</P><P>Transformation according to the standard protocol with BL21 </P><H4><SPAN class="mw-headline" id="Midi-Prep_3"><P style="font-size:15px; background-color:#66bbff;">Midi-Prep</P></SPAN></H4><P><B>Investigator: Chris W. </B></P><P style="font-size:13px; color:#003399;"> Midi-Prep of:</P><P>pSB1C3_CMV_VP123_capins =P862 =B474
pSB1C3_P40_VP123 =P863 =B651</P><P>
The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
</P><TABLE border="1"><TBODY><TR><TD> plasmid-no. </TD><TD align="right">P862</TD><TD align="right">P863
</TD></TR><TR><TD> concentration (ng/µl)</TD><TD align="right">3310,5 </TD><TD align="right">2209,2
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="149._labday_14.10.2010"><P style="font-size:17px; background-color:#00dd77;">149. labday 14.10.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Mini-Prep_and_test_digestion_of_several_constructs_3"><P style="font-size:15px; background-color:#66bbFF;"><B>Mini-Prep and test digestion of several constructs</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P>Glycerol stocks were prepared:</P><UL><LI>B658 = pSB1C3_CD_SDM-PstI_III clone 1</LI><LI>B659 = pSB1C3_CD_SDM-PstI_III clone 2</LI><LI>B660 = pSB1C3_mGMK_sr39 clone 1</LI><LI>B661 = pSB1C3_mGMK_sr39 clone 2</LI></UL><P>Mini-Prep was performed according to standard protocol:</P><UL><LI>P858 = pSB1C3_CD_SDM-PstI_III clone 1 c = 241,7 ng/µl</LI><LI>P859 = pSB1C3_CD_SDM-PstI_III clone 2 c = 212,1 ng/µl</LI><LI>P860 = pSB1C3_mGMK_sr39 clone 1 c = 231,8 ng/µl</LI><LI>P861 = pSB1C3_mGMK_sr39 clone 2 c = 265,2 ng/µl</LI></UL><P>Test digestion:</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>P858 + P859 / µl</B></TD><TD align="left"><B>P860 + P861 / µl</B></TD></TR><TR><TD align="left"> DNA   </TD><TD align="left"> 7</TD><TD align="left"> 1,5
</TD></TR><TR><TD align="left"> Buffer 4 </TD><TD align="left"> 1</TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> BSA (10x)  </TD><TD align="left"> 1</TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Enzyme I </TD><TD align="left"> PstI 0,3 </TD><TD align="left"> BstXI 0,3
</TD></TR><TR><TD align="left"> Enzyme II </TD><TD align="left"> EcoRI 0,3 </TD><TD align="left"> XbaI 0,3
</TD></TR><TR><TD align="left"> Enzyme III </TD><TD align="left"> - </TD><TD align="left"> SpeI 0,3
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 0,4</TD><TD align="left"> 5
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>10</B></TD><TD align="left"><B>10</B></TD></TR></TBODY></TABLE><P>Gel:
0,5g agarose, 50 ml TAE (1%), 3 µl GELRED, 115 Volt, running time ~50 minutes</P><P style="color:#66bbff;"><I>Comment</I>: .</P><H4><SPAN class="mw-headline" id="Continuation_of_Western_Blot_of_unmodified_AAV2_capsids"><P style="font-size:15px; background-color:#66bbFF;"><B>Continuation of Western Blot of unmodified AAV2 capsids</B></P></SPAN></H4><P><B>Investigator: Hanna</B></P><P><B>Comment:</B> Yesterday, SDS PAGE, Western Blot and primary B1 mouse antibody incubation was performed. 
Today, secondary antibody incubation and ECL analysis will be done. </P><UL><LI> Membrane was washed 3 x 7 minutes in TBST (0.2% Tween).
</LI><LI> A 1:5000 dilution of secondary goat anti-mouse HRP (sc2005) coupled antibody in TBS was prepared.
</LI><LI> Membrane was incubated in secondary antiobody for 1 hour.
</LI><LI> Membrane was washed 3 x 10 minutes.
</LI><LI> Pierce ECL Western Blotting Substrate was prepared: 350 µL Luminol Enhancer Solution + 350 µL Peroxidase Solution
</LI><LI> Analysis of the membrane revealed that the samples didn't run well. Much unspecific antibody binding was detectable. 
</LI></UL><P>
Because of these dissatisfying results SDS PAGE will be repeated. Instead of performing Western Blot, gel wioll be stained by Coomassie in order to optimize SDS-PAGE conditions. </P><H4><SPAN class="mw-headline" id="Cloning_hGH_rITR_into_.28form_P728.29_into_pSB1C3_lITR_phTERT_betaglobin_mGMK_TK30_SDM-PstI_.28P822.29"><P style="font-size:15px; background-color:#66bbFF;"><B>Cloning hGH_rITR into (form P728) into pSB1C3_lITR_phTERT_betaglobin_mGMK_TK30_SDM-PstI (P822) </B></P></SPAN></H4><P>Investigator Patrick
</P><P>Digestions, 1 h 40 minutes, 37 °C:
</P><UL><LI>P728: 1 µl XbaI, 1 µl PstI, 2 µl BSA, 2 µl Buffer 4, 6 µl DNA, 8 µl H2O
</LI><LI>P822: 1 µl SpeI, 1 µl PstI, 2 µl BSA, 2 µl Buffer 4, 6 µl DNA, 8 µl H2O
</LI></UL><P>Expected results for the 1% agarose gel:
</P><UL><LI>P728: about 2100 and 650 bp
</LI><LI>P822: about about 4900 and 10 bp (wont be visible on the gel)
</LI></UL><P>
As usual the PSB1C3 bands indicate a higher bp amount than they actually got.

The gelextraction ...
</P><UL><LI>P728: 9 ng/µl
</LI><LI>P822: 67,5 ng/µl
</LI></UL><P>
... and following ligation (2 µl Insert, 6 µl vector, 1 µl T4 DNA Ligase, 1 µl T4 DNA Ligase Buffer, 40 minutes, RT) were performed according to the standard protocol. After the transformation (with BL21) the cells were plated and put into the 37°C room.
</P><H4><SPAN class="mw-headline" id="Cloning_of_the_ViralBricks_453-empty_and_587_empty"><P style="font-size:15px; background-color:#66bbFF;"><B>Cloning of the ViralBricks 453-empty and 587 empty </B></P></SPAN></H4><P>These ViralBricks were not ordered as hybridized oligos, but can be cloned from the Cap-Gene-Synthesis into the ViralBrick MCS.
</P><TABLE border="1"><TBODY><TR><TD> components
</TD><TD>volume of p320 (ViralBrick-BLA)  /µl
</TD><TD>volume of p320 (ViralBrick-BLA) /µl
</TD><TD>volume of p668 (pSB1C3_RepCap_IRCK) /µl
</TD><TD>volume of p668 (pSB1C3_RepCap_IRCK) /µl
</TD></TR><TR><TD> DNA
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>7µl
</TD><TD>7µl
</TD></TR><TR><TD> BSA (100x)
</TD><TD>2.5µl
</TD><TD>2.5µl
</TD><TD>2.5µl
</TD><TD>2.5µl
</TD></TR><TR><TD> Buffer 4 (10x)
</TD><TD>2.5µl
</TD><TD>2.5µl
</TD><TD>2.5µl
</TD><TD>2.5µl
</TD></TR><TR><TD>SspI
</TD><TD>1µl
</TD><TD>0
</TD><TD>1µl
</TD><TD>0
</TD></TR><TR><TD>SalI
</TD><TD>1µl
</TD><TD>0
</TD><TD>1µl
</TD><TD>0
</TD></TR><TR><TD>BamHI
</TD><TD>0
</TD><TD>1µl
</TD><TD>0
</TD><TD>1µl
</TD></TR><TR><TD>PvuII
</TD><TD>0
</TD><TD>1µl
</TD><TD>0
</TD><TD>1µl
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>16µl
</TD><TD>16µl
</TD><TD>11µl
</TD><TD>11µl
</TD></TR><TR><TD><B>Total volume</B></TD><TD>25µl
</TD><TD>25µl
</TD><TD>25µl
</TD><TD>25µl
</TD></TR></TBODY></TABLE><LI> Incubation: 4h<TABLE border="1"><TBODY><TR><TH>Sample Marker
</TH><TH>Sample ViralBrick_BLA 453 cut
</TH><TH>Sample ViralBrick_BLA 587 cut
</TH><TH>Sample pSB1C3_001_RepCap_IRCK 453 cut
</TH><TH>Sample pSB1C3_001_RepCap_IRCK 587 cut
</TH></TR><TR><TH>Lane
</TH><TD>7µl
</TD><TD>30µl
</TD><TD>30µl
</TD><TD>30µl
</TD><TD>30µl
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="SDS_PAGE_of_viral_capsids"><P style="font-size:15px; background-color:#66bbFF;"><B>SDS PAGE of viral capsids</B></P></SPAN></H4><P><B>Investigator: Hanna</B><B>Comment:</B> In order to optimize SDS PAGE with the viral capsid proteins, I didn't perform Western Blot, but Coomassie staining. 
</P><P><B>1.: Preparation of 10% polyacrylamide gel: </B>
Following standard protocol.</P><P><B>2.: Preparation of virus samples: </B>
Two different virus stocks with unmodified capsid were taken. 2 samples were prepared of each: One sample was untreated, the other one was centrifuged in order to sediment cell fragments. 
10 µL Laemmli buffer (5x) was added to 40 µL virus sample and incubated at 95°C for 5 minutes. </P><P><B>3.: SDS PAGE </B>
Loading plan: </P><TABLE border="1"><TBODY><TR><TD align="left"><B>10 µL PageRuler Prestained Protein Ladder</B></TD><TD align="left"><B>10 µL Sample 1 (not centrifuged)</B></TD><TD align="left"><B>20 µL Sample 1 (not centrifuged)</B></TD><TD align="left"><B>10 µL Sample 1 (centrifuged)</B></TD><TD align="left"><B>20 µL Sample 1 (centrifuged)</B></TD><TD align="left"><B>10 µL Sample 2 (not centrifuged)</B></TD><TD align="left"><B>20 µL Sample 2 (not centrifuged)</B></TD><TD align="left"><B>10 µL Sample 2 (centrifuged)</B></TD><TD align="left"><B>20 µL Sample 2 (centrifuged)</B></TD></TR></TBODY></TABLE><P>
Gel was run 1.5 hours at 10 mA.
</P><P><B>4.: Coomassie staining </B>
Gel was put into coomassie bath, heated in microwave and incubated for 1 hour, shaking. 
In order to remove background coomassie staining, gel was incubated in acetic acid (20%) for 2 hours, shaking. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">SDS PAGE of AAV2 stock - unmodified capsids</DIV></DIV></DIV></DIV><P>
Results revealed that samples ran well (vertically :) ) - 20 µL seems to be too much. Interestingly centrifuging the virus stocks in order to remove cell debris didn't lead to better results. <B>To do:</B> Perform Western Blot again - with 10 µL Virus stock, 0.5% instead of just 0.2% TBS-milk and longer washing steps. </P><H4><SPAN class="mw-headline" id="Sequence_analysis_of_super_constructs"><P style="font-size:15px; background-color:#66bbFF;"><B>Sequence analysis of super constructs</B></P></SPAN></H4><P><B>Investigator: Hanna</B><B>Comment:</B> We assembled targeting super constructs which specifically bind the EGF receptor with VP2 N-terminal fused Z<SUB>EGFR:1907</SUB> Affibody and can either be purified via a loop inserted His-Tag or visualized via Streptavidin coupled fluorescent proteins. 
Further on these loop motifs were vloned to the &quot;mVenus VP1 insertion&quot;. </P><P>
Sequencing results showed that insertion of these constructs into pSB1C3_CMV worked! <B>Next steps:</B> Put sample sinto BioBrick Box. 
Midi-Prep + Transfection. </P><H4><SPAN class="mw-headline" id="Transfection_with_pCMV.2C_p40.2C_pAAV_RC"><P style="font-size:15px; background-color:#66bbFF;"><B>Transfection with pCMV, p40, pAAV_RC</B></P></SPAN></H4><P><B>Investigator: Kira</B><B>Comment</B>: Transfection was performed in order to evaluate the promoter activities. 
</P><P>Yesterday, the cells were seeded in the 6 well plate by Sven. Transformation was performed according to the manufacturer protocol (Qiagen). 
</P><H4><SPAN class="mw-headline" id="Trafo_evaluation_of_Rep78"><P style="font-size:15px; background-color:#66bbFF;"><B>Trafo evaluation of Rep78</B></P></SPAN></H4><P><B>Investigator: Kira</B>
Transformation was successful. 2 clones will be picked for mini prep tomorrow.
</P><H4><SPAN class="mw-headline" id="Picking_clones"><P style="font-size:15px; background-color:#66bbFF;"><B>Picking clones</B></P></SPAN></H4><P><B>Investigator: Kira</B></P><P>Rep78 and Rep52 were picked (2 clones from each plate), in order to continue with mini prep and PCR amplification.
</P><H4><SPAN class="mw-headline" id="Cloning_of_CD_into_pSB1C3_hGH_rITR"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of CD into pSB1C3_hGH_rITR</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P style="font-size:13px; color:red;">.</P><P>Vector name:
pSB1C3_hGH_rITR (P728)</P><P>Insert name:
pSB1C3_CD_SDM-PstI_III clone 1 (P858)
pSB1C3_CD_SDM-PstI_III clone 2 (P859)</P><P><B>Digestion:</B></P><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume P858 + P859  /µl</B></TD><TD align="right"><B>volume P728  /µl </B></TD></TR><TR><TD> DNA  </TD><TD align="right">6 </TD><TD align="right">8
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right">2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right">2
</TD></TR><TR><TD>Enzyme EcoI</TD><TD align="right">1</TD><TD align="right">1
</TD></TR><TR><TD>Enzyme XbaI</TD><TD align="right">-</TD><TD align="right">1
</TD></TR><TR><TD>Enzyme SpeI</TD><TD align="right">1</TD><TD align="right">-
</TD></TR><TR><TD>H2O</TD><TD align="right">8</TD><TD align="right">6
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 20</TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><P><B>Gel:</B>
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 115 Volt</P><P><B>Gel extraction</B>: 
Was performed according to protocol.
</P><P><B>T4 Ligation</B>: </P><TABLE border="1"><TBODY><TR><TD>ligation name </TD><TD align="right">728 + 858</TD><TD align="right">728 + 859
</TD></TR><TR><TD>volume of vector </TD><TD align="right">3,14</TD><TD align="right">3,14
</TD></TR><TR><TD>volume of insert</TD><TD align="right">4,86</TD><TD align="right">4,86
</TD></TR><TR><TD>T4 ligase buffer (10x)</TD><TD align="right">1</TD><TD align="right">1
</TD></TR><TR><TD>T4 ligase </TD><TD align="right">1</TD><TD align="right">1
</TD></TR></TBODY></TABLE><P><B>Transformation</B>: 
Was performed according to standard protocol using BL21 cells.
</P></LI></DIV><CENTER><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/October2" class="external text" rel="nofollow"><B>=&gt; Go to Labjournal October part 2 (labday 146 - 155 )</B></A></CENTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/October">http://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/October</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg_Bioware/NoteBook/Labjournal/October" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg_Bioware/NoteBook/Labjournal/October" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/October&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/October&amp;oldid=197498" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>