"
Page
Discussion
View source
History
teams
Log in
 
Team:Freiburg Bioware/NoteBook/Labjournal/August2
From 2010.igem.org
Home
Team
Team 2010
Photo Gallery
Cuckoo Clock Competition
Collaboration
Example Page
Project
Overview
Highlights
Results
   Modularisation
   Targeting
   Arming
   All-in-One
Methods
Virus Construction Kit Manual
BioBricks
Overview
Capsid Coding Parts
Vector Plasmid Parts
Miscellaneous Parts
Notebook
Labjournal
Materials
Methods
Experiments
Meetings
Statistics
Sponsors
Safety
Modeling
Modeling Overview
Virus Production
Virus Infection
Structure Modeling
=> Back to Notebook overview
NoteBook Navigator
March (labday 1)
April (labday 2 - 5)
May (labday 6 - 17)
June (labday 18 - 45)
July (labday 46 - 75)
August part 1 (labday 76 - 92)
August part 2 (labday 93 - 106)
September part 1 (labday 107 - 123)
September part 2 (labday 124 - 135)
October part 1 (labday 136 - 145 )
October part 2 (labday 146 - 155 )
October part 3 (labday 156 - 166 )
November  (labday 167 - 170 )
Cellculture
Contents
1 93. labday 18.08.2010
1.1 Continuation: pSB1C3_SDM_SspI_Bla14FM clone 5 SDM (P222)
1.2 Mini Preps pEGFP_C1,pSB1C3_Rep40,pSB1C3_Rep78,pMA_T_Affibody,pSB1C3_SspIdel_BLA clone 2+3,pSB1C3_AAP, pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR,pSB1C3_lITR_CMV_mVenus_hGH_rITR,pSK_CD_SDMPstI
1.3 Continuation of Repetition: Biobrick production of Rep 68 (17.08.2010)
1.4 Transfection of AAV293 cells
1.5 Seeding Ht1080
1.6 Cloning of pMA_T_affibody and pSB1C3_CFP
2 94. labday 19.08.2010
2.1 Continuation: pSB1C3_SDM_SspI_Bla14FM clone 5 SDM (P222)
2.2 Cloning of pSB1C3_CFP and middlelinker
2.3 Test digestion of pSB1C3_Rep78
2.4 Sent for sequencing
2.5 Test Digestion of P242, P243, P245, P246, P247
2.6 Test digestion of pAAV_RC-ins-rep
2.7 Midiprep of dsAAV_CMV_EGFP and pAAV_iGEM_mVenus_YF
2.8 Harvest viral particles and Transduction of HT1080 cells
2.9 New ladder mix and testing
2.10 SDM of CD NgoMIV
2.11 Picking clones of pSB1C3_affibody and pSB1C3_Rep68
3 95. labday 20.08.2010
3.1 Sequence analysis of sequencing reads prepared yesterday (19.08.2010)
3.2 Continuation: pSB1C3_SDM_SspI_Bla14FM clone 5 SDM (P222)
3.3 Subcloning cap from pMA_RepCap into pAAV_RC which contains four mutations and the synthesized Rep part
3.4 Mini-Preps of pSB1C3_Rep68_ex and pSB1C3_ZEGFR:1907
3.5 Hybridisation of the oligos for NLS and cloning NLS into pSB1C3
3.6 Seeding HT1080 cells
3.7 PCR and ligation for biobrick-production of pCerulean
3.8 Biobrick production of VP1_up
3.9 BioBrick assembly for pSB1C3_Affibody_Linker and pSB1C3_YFP_His-Tag
4 96. labday 21.08.2010
4.1 Transfection of AAV293, Harvest viral particles, Transduction of HT1080 and Fluorescent microscopy
4.2 Midi-Prep of pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR clone1
4.3 Re-Trafo of pSB1C3_SDM_SspI_Bla14FM
5 97. labday 22.08.2010
5.1 Test-digestion of pCerulean (P273,4,5), pSB1C3_VP1up (P279, 280), pSB1C3_NLS (P281,282), pAAV_RC_final (P283)
5.2 Mini-Prep of pCerulean
6 98. labday 23.08.2010
6.1 Primer design: Sequencing primer for BioBrick Assembly of vectorplasmid and phTERT qPCR primer
6.2 Mini-Prep of pSB1C3_CFP_middlelinker and pSB1C3_ZEGFR:1907 / testdigestion of pSB1C3_CFP_middlelinker and pSB1C3_lITR_pHTERT
6.3 Continuation of BioBrick assembly for pSB1C3_Affibody_Linker
6.4 Cloning of pSB1C3_ß-globin_YFP and pSB1C3_hGH_rITR
6.5 Repetition: Quickchange site-directed mutagenesis of pSB1C3_SDM_SspI_Bla14FM
6.6 FACS analysis, Seeding HT1080 and Hek293 cells
6.7 Miniprep of pSBC13_Bla
7 99. labday 24.08.2010
7.1 Sequence analysis of pSB1C3_CFP_middlelinker
7.2 Harvest viral particles, Transduction of HT1080
7.3 Mini-prep of SDM NgoMIV CD
7.4 Sent for sequencing: pSB1C3_6xHis
7.5 Subcloning Cap into pAAV_RC_ins-rep
7.6 Test digestion of pCerulean (p273)
7.7 Cloning of VP1up into PCerulean
7.8 Sequence analysis of sequencing reads prepared yesterday (23.08.2010)
7.9 Test digestion of SDM NgoMIV CD
7.10 Picking clones of pSB1C3_Affibody_linker and pSB1C3_ß-Globin_YFP_hGH_rITR
7.11 Test digestion of pSB1C3_SDM_SspI_Bla14FM (P222), pSB1C3_BLA (P286, P288)
7.12 Test digestion of SDM NgoMIV CD
7.13 Picking clones of pSB1C3_Affibody_linker and pSB1C3_ß-Globin_YFP_hGH_rITR
7.14 Test digestion of pSB1C3_SDM_SspI_Bla14FM (P222), pSB1C3_BLA (P286, P288)
8 100. labday 25.08.2010
8.1 Cloning of pSB1C3_Rep40, pSB1C3_Rep68 and pMA_RC_insert
8.2 Picking clones of pAAV_RC-ins_rep_cap and pCeruleanVP1up
8.3 Cloning of pSB1C3_6xHis with middlelinker and pSB1C3_6xHis with pGA14_mVenus_YFP
8.4 Sequence analysis of pSB1C3_6xHis
8.5 Inoculation of pGA14_middlelinker and pSB1C3_6xHis clone1
8.6 Cloning of pSB1C3_SspI_BLA and pSB1C3_CFP_PvuII
8.7 Mini-Preps and test digestion of pSB1C3_Affibody_Middle-Linker, pSB1C3_Affibody_Short-Linker, pSB1C3_Affibody_SEG-Linker, pSB1C3_Affibody_Long-Linker and pSB1C3_betaglobin_mVenus_hGH_rITR
8.8 Sequenzing evaluation of SDM NgoMIV
9 101. labday 26.08.2010
9.1 Mini-Preps of pGA14_middle linker and pSB1C3_6xHis clone 1
9.2 Mini-Preps and test digestion of pAAV_RC-ins_rep_cap, pCeruleanVP1up and pAAV_RC_ins-rep-cap.
10 102. labday 27.08.2010
10.1 Sequence analysis of pSB1C3_Bla_final (P309)and pCerulean_VP1up (P303)
10.2 Hybridisation and cloning of loop insertions
10.3 Cloning of pCerulean_VP1up_NLS
10.4 DARPin E_01
10.5 Update VP1 insertion
10.6 Sequence analysis of pSB1C3_BLA_final (P309)
10.7 Mini-Preps and test digestion of pSB1C3_6xHis_middlelinker and pSB1C3_6xHis_mVenus
10.8 Repetition of test digestion of pAAV_RC_ins-rep-cap, pAAV_RC_ins-rep, pSB1C3_Rep40_RC_Insert and pSB1C3_Rep68_RC_Insert.
10.9 Midi-Prep of pSB1C3_VCK_Bla
11 103. labday 28.08.2010
11.1 Mini-Prep of pSB1C3_NLS
11.2 Cell culture
11.2.1 Harvest viral Stock and Transductiot of HT and A431 cells
11.2.2 Seeding AAV293 for Transfection at 30.8
11.3 Picking clones from the ViralBricks
12 104. labday 29.08.2010
12.1 P282/B234 mini-prep
12.2 Digestion of pSB1C3_NLS (P324)
12.3 Harvesting cultures from the ViralBricks
13 105. labday 30.08.2010
13.1 BioBrick assembly of pSB1C3_leftITR_pTERT & pSB1C3_ß-globin_YFP_hGH_rITR
13.2 Continuation: Digestion of pSB1C3_NLS (P324)
13.3 CD biobrick production
13.4 Repetition of test digestion of pSB1C3_Rep40_RC_Insert and pSB1C3_Rep68_RC_Insert.
13.5 Midi-Prep of pAAV_RC_insertparts clone1
13.6 FACS analysis, Seeding HT1080 and AAV293
13.7 Mini-Preps and test digestion of Loop insertion BioBricks
13.8 Repetition of subcloning Cap-insert into pAAV_RC
14 106. labday 31.08.2010
14.1 Cloning ZEGFR:1907 and 6xHis_middlelinker into pCerulean
14.2 PCR of SV 40 terminator
14.3 Cloning of NLS into pCerulean_VP1up
14.4 Cloning CFP_middlelinker (from pSB1C3_CFP_middlelinker, P276) into pCerulean (P273)
14.5 Sequencing results of pSB1C3_6xHis_middlelinker and pSB1C3_6xHis_mVenus
14.6 Contiuation: Repetition of subcloning Cap-insert into pAAV_RC
14.7 2nd Repetition of test digestion of Loop insertion BioBricks
14.8 Midi-Prep of pAAV_RC & pHelper
14.9 Transformation of Biobrick CD
93. labday 18.08.2010
Continuation: pSB1C3_SDM_SspI_Bla14FM clone 5 SDM (P222)
Investigator: Patrick
There is a lawn on the plate although chloramphenicol was added. Some clones were picked and a new agar plate inoculated to hopefully receive some 'pickable' clones.
Additionally a new SDM was performed: PCR program:
95°C 2 min (1x)
95°C 20s, 60°C 10 s, 68°C 30 s (30s/kb) (18x)
68°C 5 minutes
Ingredients
Volume
10x reaction buffer
2,5
DNA template ( about 10 ng)
1 µl diluted P222
forward primer:
0,58 µl O108 (pSB1C3 PvuII rev)
reverse primer:
0,59 µl O109 (pSB1C3 PvuII for)
DMSO (2%)
0,5
dNTP Mix from the kit
0,5 µl
QuickSolution Reagent
0,75 µl
Quickchange Lightning Enzyme (1.25U)
0,5 µl
H2O
18,08 µl
Total volume
25 µl
All further steps including the transformation were performed according to the standard protocol. This time 2 agar plates were inoculated. One of them with the resuspended pellet and the other one with a 1:100 dilution of the resuspended pellet.
Mini Preps pEGFP_C1,pSB1C3_Rep40,pSB1C3_Rep78,pMA_T_Affibody,pSB1C3_SspIdel_BLA clone 2+3,pSB1C3_AAP, pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR,pSB1C3_lITR_CMV_mVenus_hGH_rITR,pSK_CD_SDMPstI
Investigator: Jessica
Preps and test digestions were made:
pSK_CD_SDMPstI
Given Plasmid-Number: P248; DNA concentration: 302,7 ng/µL ;
Given Plasmid-Number: P249; DNA concentration: 267,0 ng/µL ;
P248, P249 and P223 as control digested with PstI  --> no satisfying result,control was not cut but samples two times... sent for sequencing with primer: GATC_std_T7minus1
pSB1C3_Rep40
Given Plasmid-Number: P231 (clone3); DNA concentration: 234,00 ng/µL
Given Plasmid-Number: P232 (clone2); DNA concentration: 234,96 ng/µL
Given Plasmid-Number: P233 (clone1); DNA concentration: 237,12 ng/µL
pSB1C3_Rep78
Given Plasmid-Number: P234 (clone2); DNA concentration: 277,56 ng/µL
Given Plasmid-Number: P235 (clone1); DNA concentration: 294,19 ng/µL
Given Plasmid-Number: P236 (clone3); DNA concentration: 378,87 ng/µL
pSB1C3_AAP
Given Plasmid-Number: P239 (clone1); DNA concentration: 219,82 ng/µL
Given Plasmid-Number: P240 (clone2); DNA concentration: 220,45 ng/µL
Given Plasmid-Number: P241 (clone3); DNA concentration: 209,57 ng/µL
has to be edited
@Jessica: What are these plasmid numbers?? (Bea)
Given Plasmid-Number: P230; DNA concentration: 367,36 ng/µL ;
Given Plasmid-Number: P242; DNA concentration: 202,09 ng/µL ;
Given Plasmid-Number: P243; DNA concentration: 369,34 ng/µL ;
Given Plasmid-Number: P245; DNA concentration: 207,12 ng/µL ;
Given Plasmid-Number: P246; DNA concentration: 198,35 ng/µL ;
Given Plasmid-Number: P247; DNA concentration: 212,22 ng/µL ;
Test digestion
buffer used: 4 ; Restriction-enzymes used: Enzyme XbaI ; Enzyme SpeI
Plasmid
Components
P231-233 Volume/µL
P234 Volume/µL
P235 Volume/µL
236 Volume/µL
P239-242 Volume/µL
DNA
4
3.5
3
2.5
4
BSA (10x)
2
2
2
2
2
Buffer 4 (10x)
2
2
2
2
2
SpeI
0.5
0.5
0.5
0.5
0.5
XbaI
0.5
0.5
0.5
0.5
0.5
H2O
11
11,5
12
12.5
11
Total volume
20
20
20
20
20
Comment: Test digestion of P231-233 and P239-242 contained equal volumes of ingredients. Therefore the charts were simplyfied.
Comment: P231 and P239 will be sent for sequenzing tomorrow. We picked for additional clones of Rep78 for Mini-Prep and test digestion tomorrow.
Continuation of Repetition: Biobrick production of Rep 68 (17.08.2010)
Investigator: Christian L. , Stefan
Aim of the experiment: We want to produce biobricks from Rep 68.
Vector: name: pSB1C3_CFP P51.1
Insert: name: Rep68_SDM_PstI clone 1 P224
buffer used: 4
Restriction-enzymes used:
XbaI
SpeI
DNA concentration (P51.1): 132,1 ng/µl
DNA concentration (P224): 350.29 ng/µl
Digestion of plasmid backbone and PCR product:
components
volume of pSB1C3_CFP /µl
volume of Rep68_SDM_PstI clone 1 /µl
DNA
10
29
BSA (10x)
2
4
Buffer 4 (10x)
2
4
SpeI
1
1
XbaI
1
1
H2O
4
1
Total volume
20
40
Incubation at 37°C, 2 h
Dephoshorylation of Vector pSB1C3_CFP after digestion:
add 2 µl of 10x Antarctic Phopshatase Reaction Buffer to DNA cut with XbaI and SpeI
add 1 µl of Antarctic Phosphatase and mix
incubate for 60 minutes at 37°C
Gelextration of pSB1C3
Gelextration of pSB1C3 was performed according to standard protocol
0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 115 Volt, running time:45
PCR Purification Purification of Rep68 was performed according to the standard protocol
Concentration measured: 16,4 ng/µl
Ligation For the ligation 1µl T4 buffer (10x) and 1µl T4 ligase were used.
vector /µl
insert /µl
pSB1C3 + Rep68
3,9
4,1
Incubation at room temperature for 45 minutes.
Transformation:
The transformation was done following the standard protocol using XL1b cells.
Transfection of AAV293 cells
Investigator: Adrian, Kerstin
2x106 cells were seeded (instead of 3x106) --> hopefully better virus-production
used plasmids:
P50   c= 429 ng/µl (RC)
P64   c= 500 ng/µl (pHelper)
P229  c= 162 ng/µl (GOI=YFP)
P228  c= 540 ng/µl (GOI=eGFP)
P158a c= 1800ng/µl (RC mutant)
P158a c= 1770ng/µl (RC mutant)
1. plate: Lipo-Transfection
2. plate: Lipo-Transfection
3. plate: 3,3 µg (RC), 3,3 µg (pHelper), 3,3 µg (YFP)
4. plate: 10 µg (RC), 10 µg (pHelper), 3,3 µg (YFP)
5. plate: 10 µg (RC mutant P158a), 10 µg (pHelper), 3,3 µg (YFP)
6. plate: 10 µg (RC mutant P158b), 10 µg (pHelper), 3,3 µg (YFP)
7. plate: 3,3 µg (RC mutant P158a), 3,3 µg (pHelper), 3,3 µg (YFP)
8. plate: 3,3 µg (RC mutant P158b), 3,3 µg (pHelper), 3,3 µg (YFP)
9. plate: 10 µg (RC), 10 µg (pHelper), 3,3 µg (eGFP)
10. plate: 10 µg (RC), 10 µg (pHelper), 10 µg (eGFP)
11. plate: 10 µg (RC), 10 µg (pHelper), 20 µg (eGFP)
12. plate: 10 µg (RC), 10 µg (pHelper), 40 µg (eGFP)
Seeding Ht1080
Investigators: Adrian, Kerstin
Cloning of pMA_T_affibody and pSB1C3_CFP
Comment: pMA_T_affibody (p237) from Retrafo and pSB1C3_CFP (p51.5) were digested with EcoRI and PstI.
Investigator: Anna
Vector:
pSB1C3_CFP (p51.1): c= 132,1 ng/µl
Insert:
pMA_T_affibody (p237): c= 233,96 ng/µl
components
Vector p51.1
Insert p237
DNA
9
8
BSA (10x)
2
2
Buffer 4 (10x)
2
2
Enzyme EcoRI
1
1
Enzyme PstI
1
1
H2O
5
6
Total volume
20
20
Gelextraction of pSB1C3_RFC25_CFP:
0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 120 Volt, running time:55
4µl loading dye (6x) for the sample, Marker: GeneRuler ladder mix (Fermentas)
c(pSB1C3)= 5,4 ng/µl
c(affibody)= 5,6 ng/µl
Ligation of PCR products and vector:
For the Ligation 1µl T4 buffer (2x) and 1µl T4 ligase were used. Incubation time: 30 min
vector /µl
insert /µl
pSB1C3 + affibody
5,91
2,09
Transformation:
The transformation was done following the standard protocol using XL1 blue cells.
Result: Clones of pSB1C3_affibody will be picked tomorrow.
94. labday 19.08.2010
Continuation: pSB1C3_SDM_SspI_Bla14FM clone 5 SDM (P222)
Investigator: Patrick
Again there was a bacterial lawn on the agar plate. Therefore some clones from the overgrown agar plated were picked and diluted. The 1:10^4, 1:10^5, 1:10^6 dilution each were put onto an agar plate (37°C, over night). Additionally a transformation with XL1B cells from the remaining SDM DNA sample was performed and diluted, too. The 1:10^4 and the 1:10^6 dilution were plated. Hopefully there will be some 'pickable' clones tomorrow.
Cloning of pSB1C3_CFP and middlelinker
Investigator: Jessica
buffer used: 4; Restriction-enzymes used: Enzyme 1: AgeI ; Enzyme 2: PstI ; Enzyme3: NgoMIV
Plasmids:
pSB1C3_CFP P51.2
pGA14_middle linker P65
Components
Mastermix
P51.2/µL
P65/µL
DNA
-
9,9
10,8
BSA (10x)
-
2
2
Buffer 4 (10x)
-
2
2
Enzyme 1 AgeI
-
1
-
Enzyme 2  PstI
-
1
1
Enzyme 3  NgoMIV
-
-
1
H2O
-
4,1
3,2
Total volume
-
20
20
Incubation: 90 minutes
Agarosegel
0.5 g Agarose, 50 ml TAE (1 %), 3 µL GELRED, at 115 Volt, running time: 45 minutes
concentrations measured via NanoDrop:
SB1C3_CFP:  5,0 ng/µl
middle linker:1,7 ng/µl
T4 Ligation
Volume insert: 1,24 µl
Volume vector: 6,76 µl
Trafo was prepared with XL1blue
Test digestion of pSB1C3_Rep78
Investigator: Chris L., Stefan
Preps and test digestions were made:
pSB1C3_Rep78
Given Plasmid-Number: P252 (clone4); DNA concentration: 352,47 ng/µL
Given Plasmid-Number: P253 (clone5); DNA concentration: 408,23 ng/µL
Given Plasmid-Number: P254 (clone6); DNA concentration: 349,18 ng/µL
Given Plasmid-Number: P255 (clone7); DNA concentration: 320,25 ng/µL
Test digestion
buffer used: 4 ; Restriction-enzymes used: Enzyme XbaI ; Enzyme SpeI
Plasmid
Components
P252 Volume/µL
P253 Volume/µL
P254 Volume/µL
255 Volume/µL
DNA
2.5
2
2.5
3
BSA (10x)
2
2
2
2
Buffer 4 (10x)
2
2
2
2
SpeI
0.5
0.5
0.5
0.5
XbaI
0.5
0.5
0.5
0.5
H2O
12.5
13
12.5
12
Total volume
20
20
20
20
Incubation time: 1 h; Incubation temperature: 37°
0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , running time:5 minutes at 90 Volt and 40 minutes at 115 Volt
4µl loading dye (6x) for the sample, Marker: GeneRuler ladder mix (Fermentas)
Sent for sequencing
Investigator: Chris L., Stefan
pSB1C3_Rep52 clone3 (P202) sent for sequencing with O162 (VF2). pSB1C3_Rep40 clone3 (P231), pSB1C3_AAP clone1 (P239), pSB1C3_Rep78 clone4 (P252) and pSB1C3_Rep78 clone7 (255) sent for sequencing with O162 (VF2) and O51 (VR2) pAAV_RC_ins-rep clone1 (P250) and pAAV_RC_ins-rep clone2 (P251) sent for sequencing with O19 (VP1 primer for pKEX reverse)
Test Digestion of P242, P243, P245, P246, P247
Investigator: Achim
P242: pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR clone1
P243: pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR clone2
P245: pSB1C3_lITR_CMV_mVenus_hGH_rITR clone1
P246: pSB1C3_lITR_CMV_mVenus_hGH_rITR clone2
P247: pSB1C3_lITR_CMV_mVenus_hGH_rITR clone3
(Left picture: Test digestion from 18.8., right picture: Repetition of test digestion from 19.8.)
Fragments sizes:
P242/243: 1400/3400
P245/P246/P247: 1400/2900
Bands are diffuse and size estimation is difficult because of strange marker behaviour, but at least the second digestion shows two distinct bands which look like they have the correct sizes. P242 and P245 were sent for sequencing.
Test digestion of pAAV_RC-ins-rep
Investigator: Chris L., Stefan
Preps and test digestions were made:
pAAV_RC-ins-rep
Given Plasmid-Number: P250 (clone1); DNA concentration: 467,5 /µL
Given Plasmid-Number: P251 (clone2); DNA concentration: 108,4 ng/µL
Test digestion
buffer used: 3 ; Restriction-enzymes used: Enzyme SwaI ; Enzyme BstEII
Plasmid
Components
P250 Volume/µL
P251 Volume/µL
DNA
1,7
7.3
BSA (10x)
1
1
Buffer 3 (10x)
1
1
SpeI
0.5
0.5
XbaI
0.5
0.5
H2O
5.3
-
Total volume
10
10.3
Incubation time with SwaI: 60 minutes; Incubation temperature: 25° Incubation time with BstEII: 75 minutes; Incubation temperature: 60°
0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , running time: 45 minutes at 115 Volt
4µl loading dye (6x) for the sample, Marker: GeneRuler ladder mix (Fermentas)
Midiprep of dsAAV_CMV_EGFP and pAAV_iGEM_mVenus_YF
Investigators: Chris W.
Comment: Midi-Preps of dsAAV_CMV_EGFP (p31, B24) and pAAV_iGEM_mVenus_YFP (p39, B34)
The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
plasmid-no.
p31
p39
concentration (ng/µl)
2420,46 & 2586,64
1148,07 & 978,99
Harvest viral particles and Transduction of HT1080 cells
Investigators: Adrian, Kerstin
Transfectionplan (16.08.2010)
Plate
amount of cells
1
100 000
2
200 000
3
400 000
4
500 000
5
800 000
6
1 000 000
7
1 200 000
8
1 500 000
9
1 750 000
10
1 750 000 not sure
The cells were transfected with 40µg YFP, 3,3µg Rep/Cap and 3,3µg pHelper.
Transduction of HT1080
Comments: We want to find out, if the efficiency of virus production can be influenced by the amount of transfected cells.
Plate 1 YFP: 200.000 cells per well
1
2
3
A
control, no cells
165µl virus (1)
165µl virus (2)
B
control, no virus
165µl virus (1)
165µl virus (2)
Plate 2 YFP: 200.000 cells per well
1
2
3
A
control, no cells
165µl virus (3)
165µl virus (4)
B
control, no virus
165µl virus (3)
165µl virus (4)
Plate 3 YFP: 200.000 cells per well
1
2
3
A
control, no cells
165µl virus (5)
165µl virus (6)
B
control, no virus
165µl virus (5)
165µl virus (6)
Plate 4 YFP: 200.000 cells per well
1
2
3
A
control, no cells
165µl virus (7)
165µl virus (8)
B
control, no virus
165µl virus (7)
165µl virus (8)
Plate 5 YFP: 200.000 cells per well
1
2
3
A
control, no cells
165µl virus (9)
165µl virus (10)
B
control, no virus
165µl virus (9)
165µl virus (10)
New ladder mix and testing
Investigator: Jessica
new Generuler ladder mix from Fermentas arrived
one tube (containing 100µl solution) was prepared with 100µl loading dye and 400 µl miliporewater
two test-gels were prepared:
1% TAE agarose gel with samples of
0,5 µl
1,0 µl
2,0 µl
3,0 µl
4,0 µl
5,0 µl
6,0 µl
7,0 µl
8,0 µl
9,0 µl
10,0 µl
sample P250
1% TAE agarose gel containing 20% H2O with samples of ( 20% H2O was an idea I have read in a journal but it doesn't seem like a good idea
0,5 µl
1,0 µl
2,0 µl
3,0 µl
4,0 µl
5,0 µl
6,0 µl
7,0 µl
8,0 µl
9,0 µl
10,0 µl
sample P251
Comments: an example of these laddervolumes is patches at the bench (right up)
SDM of CD NgoMIV
Investigator:Kira
Comment: Lightning kit was used for quickchange reaction
component
Volume/µl
DNA: (1:100)
3,3
10x reaction buffer:
2,5
primer forward NgoMIV (1:10)
0,55
primer reverse NgoMIV (1:10)
0,55
dNTP
0,5
DMSO
0
QuickSolution Reagent
0,75
H2O
16,35
Lightning Polymerase
0,5
PCR
segment
cycles
temperature
time
1
1
95
2min
2
18
95
20sec
60
10 sec
68
2 min15 sec
3
68
5 min
sample was digested with 1 ul DpnI for 10 min at 37°C
Transformation was performed according to the standard protocol.
Picking clones of pSB1C3_affibody and pSB1C3_Rep68
Investigator: Anna
Plasmid
Resistance
Bacterial strain
pSB1C3_affibody°
Chloramphenicol
XL1 blue
pSB1C3_Rep68°°
Chloramphenicol
XL1 blue
°: 2 clones °°: 3 clones each
Clones were inoculated in 10 mL DYT medium containing 10 µL of antibiotics and were put in 37°C room on shaker.
95. labday 20.08.2010
Sequence analysis of sequencing reads prepared yesterday (19.08.2010)
Investigator: Bea
GENERAL COMMENT: pSB1C3 {Rep 52 and Rep 78{} were sent for sequencing BUT they cannot be used for subcloning the ordered rep protein because in the produced BioBricks the restriction enzyme needed for subcloning (SwaI) is NOT present in the BioBricks pSB1C3_Rep52 and rep78.
Comments: Sequence analysis of pAAV_RC containing the subcloned "rep" (ordered)
Primer used: VP1 primer fro pKex
Plasmid sequenced: P250
Sequence sample: CL-9
Stored in folder:
xy
Results:
Next steps: xy
Comments: Sequence analysis of pAAV_RC containing the subcloned "rep" (ordered)
Primer used: VP1 primer fro pKex
Plasmid sequenced: P251
Sequence sample: CL-10
Stored in folder:
xy
Results:
Next steps: xy
Comments: Sequence analysis of pSB1C3_Rep40
Primer used: VR-2
Plasmid sequenced: P231
Sequence sample: CL-2
Stored in folder: DKFZ Rep40ex
xy
Results:
Sequence read is good (prefix is ok) but there was found a mutation in Rep52 protein: G2886A (nucleotides) --> V337I.
Next steps: xy
Comments: Sequence analysis of pSB1C3_AAP
Primer used: VR-2
Plasmid sequenced: P239
Sequence sample: CL-3
Stored in folder: DKFZ AAP
xy
Results:
xy
Next steps: xy
top of page
Continuation: pSB1C3_SDM_SspI_Bla14FM clone 5 SDM (P222)
Investigator: Patrick
The XL1B transformation didnt work or was too much diluted because there were no clones on the plates. The dilution steps of the transformed XL10-Gold cells worked quite well. Now there are 'pickable' clones.
top of page
Subcloning cap from pMA_RepCap into pAAV_RC which contains four mutations and the synthesized Rep part
Investigator: Patrick
pAAV_RC_ins-rep clone 1 (P250) and pMA_RepCap Vector_SDM_InsPvuII clone 1 (P211) were digested with PspLI and BspMI according to the standard protocol (Subcloning_cap_into_pAAV_RC).
Expected size of the bands:
Insert: 920 bp (P211)
Vector: 6414 bp (P250)
The gelextraction yielded the following DNA concentrations:
P 211 : 2,45 ng/µl
P 250 : 11,15 ng/µl
The marked bands were cut out and the DNA was extracted. The ligation was performed accodring to the standard prtocol with a total volume of 10 µl: 2,67 µl vector backbone(P250), 5,33 µl Insert following a transformation according to the standard protocol with XL1B cells.
Mini-Preps of pSB1C3_Rep68_ex and pSB1C3_ZEGFR:1907
Investigator: Chris L.,Stefan
Glycerol stocks were prepared:
B218 = pSB1C3_Rep68_ex clone1
B219 = pSB1C3_Rep68_ex clone2
B220 = pSB1C3_Rep68_ex clone3
B221 = pSB1C3_ZEGFR:1907 clone 1
B222 = pSB1C3_ZEGFR:1907 clone 2
Mini-Prep following the standard protocol
B218 = pSB1C3_Rep68_ex clone1 c=286.72 ng/µl
B219 = pSB1C3_Rep68_ex clone2 c=277.06 ng/µl
B220 = pSB1C3_Rep68_ex clone3 c=298.18 ng/µl
B221 = pSB1C3_ZEGFR:1907 clone 1 c=174.20 ng/µl
B222 = pSB1C3_ZEGFR:1907 clone 2 c=162.48 ng/µl
Test digestion
buffer used: 4 ; Restriction-enzymes used: XbaI ; SpeI ; EcoRI ; PstI
Components
P264 Volume/µL
P265 Volume/µL
P266 Volume/µL
P267 Volume/µL
P268 Volume/µL
DNA (800 ng)
2.8
2.9
2.7
4.6
4.9
BSA (10x)
1
1
1
1
1
Buffer 4 (10x)
1
1
1
1
1
SpeI
0.5
0.5
0.5
-
-
XbaI
0.5
0.5
0.5
-
-
PstI
-
-
-
0.5
0.5
EcoRI
-
-
-
0.5
0.5
H2O
4.2
4.1
4.3
2.4
2.1
Total volume
10
10
10
10
10
Incubation time: 1 h; Incubation temperature: 37°
0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , running time:5 minutes at 90 Volt and 40 minutes at 115 Volt
4µl loading dye (6x) for the sample, Marker: GeneRuler ladder mix (Fermentas)
Comment:pSB1C3_Rep68_ex clone 3 (P266) sent for sequencing with O162 (VF2) and O51 (VR2). pSB1C3_ZEGFR:1907 clone 1 (P267) sent for sequencing with O51 (VR2).
Hybridisation of the oligos for NLS and cloning NLS into pSB1C3
Investigator: Anissa
Hybridisation: program: Origami 1
Hybridisation was peformed according the standard protocol
Digestion of pSB1C3_CFP
components
Vector /µl
DNA
6,62
BSA (10X)
1,5
Buffer 4
1,5
Enzyme EcoRI
1
Enzyme PstI
1
H2O
3,38
Endvolume
15
Gel:After digestion, the sample was loaded on a 1% agarose-gel.
Digest pSB1C3_CFP
Gelextraction: After the gel run, the vector backbone pSB1C3 was cut and purified according to the standard protocol
Ligation: the original concentration of the hybridisation was 209,7 ng/µl, for the ligation a 1:10 dilution was made
components
Volume /µl
Concentration/ ng/µl
Insert (NLS)
6,79
2,097
Vector (pSB1C3_cut)
1,21
8,3
Transformation: Trafo into XL1 blue was performed according the standard protocol
Seeding HT1080 cells
Investigator: Kerstin, Adrian
Seeding HT1080 cells for transduction with viral stocks from 18.08.
We harvest four T75 flask (two of them were used three times (3 passages)) and seed them into eight 6er dishes (200.000 cells per well)
PCR and ligation for biobrick-production of pCerulean
Investigator: Christian L. , Stefan
DNA template: pEGFP_C1 (P230) C=367,4 ng/µl
PCR: (was performed following the standard protocol)
Ingredients
Volume / µl
5X Phusion HF buffer
10
10 mM dNTP mix
1
forward primer: O163 (pEGFP-C1_rev)
2.5
reverse primer: O164 (pEGFP-C1_for)
2.5
DNA Template(1:1000)
3 µl
DMSO (2%)
1
Phusion Polymerase
0,5
H2O
29.5
Total volume
50
PCR program:
PCR Program
temperature/ °C
time/ sec.
1
98
60
2
98
15
8x
59
25
3
72
60
4
98
15
17x
66
25
5
72
60
1x
72
5
Hold
4
-
Comments: Errors in table, please correct!!! (TB)
Gelextraction of PCR product
Gelextraction was performed according to the standard protocol of the PCR product.
Comment: 0,4 g Agarose,50 ml TAE (0,8%), 3 µl GELRED , at 115 Volt, running time: 45 minutes
Digestion of PCR product and vector:
components
PCR product /µl
vector /µl
DNA
10
29
BSA (10x)
2
2
Buffer 4 (10x)
2
2
Enzyme EcoRI
1
1
Enzyme PstI
1
1
H2O
4
5
Total volume (e.g. 15,20,25,30 µl)
20
40
Purification of the PCR product with PCR purification was made according to the standard protocol.
Gelextraction of CFP
Gelextraction was performed according to the standard protocol.
>
Comment: 0.5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 115 Volt, running time: 45 minutes
Ligation
The  Ligation was performed as following:
Vector Volume: 4,1 µl
Insert Volume: 3,9 µl
1µl T4-Ligase buffer (10x)
8µl (Vector + Insert) mix
1µl T4-Ligase
Incubation for 45 minutes at room temperature
Transformation:
performed according to the standard protocol (XL1B). The cells were plated on a agar plate with Kanamycin.
Biobrick production of VP1_up
Investigator: Achim
DNA template: pKex_VP1 (P27) C=99,3 ng/µl
PCR: (was performed following the standard protocol)
Ingredients
Volume / µl
5X Phusion HF buffer
10
10 mM dNTP mix
1
forward primer: O89 (Präfix_VP1_ex_RFC25_for)
2.5
reverse primer: O167 (pEGFP-C1_for)
2.5
DNA Template(1:100)
1 µl
DMSO (2%)
-
Phusion Polymerase
0,5
H2O
Total volume
50
PCR program:
PCR Program
temperature/ °C
time/ sec.
1
98
60
2
98
15
8x
60
25
3
72
15
4
98
15
17x
66
25
1x
72
15
Hold
4
-
Digestion of PCR product and vector:
components
PCR product /µl
vector /µl
DNA
20
5,2
BSA (10x)
3
2
Buffer 4 (10x)
3
2
Enzyme NgOMIV
1
1
Enzyme SpeI
1
1
H2O
2
8,8
Total volume (e.g. 15,20,25,30 µl)
30
20
Ligation with T4-Ligase and transformation with XL1B were carried out. Clones have to be picked tomorrow.
BioBrick assembly for pSB1C3_Affibody_Linker and pSB1C3_YFP_His-Tag
Investigators: Volker
The aim of this experiment is to assemble the affibodies with the the five linkers and the YFP with the His-Tag. All cloning is done according to RFC 25 and will be used to test the affibody if it really binds the EGF-receptor, for the N-terminal fusions and to supply it to other groups in the registry.
This experiment went wrong on Saturday 21.08. and was repeated on Sunday 22.08. and gave the following results. Results from the first experiment are not shown but can be senn in the written Labjournal.
Digestion
The following digestions were performed:
components
Affibody
Affibody
Affibody
Short Linker
Middle Linker
Long Linker
SEG Linker
GSAT Linker
YFP
His-Tag
DNA
8.6 µl
8.6 µl
8.6 µl
7.7 µl
8.2 µl
8.0 µl
8.1 µl
8.4 µl
5.4 µl
14 µl
BSA (100x)
2µl
2µl
2µl
0µl
0µl
0µl
0µl
0µl
2µl
0µl
Buffer (10x)
2µl
2µl
2µl
2µl
2µl
2µl
2µl
2µl
2µl
2µl
Enzyme 1 (1µl each)
PstI
PstI
PstI
PstI
PstI
PstI
PstI
PstI
EcoRI
EcoRI
Enzyme 2 (1µl each)
AgeI
AgeI
AgeI
NgoMIV
NgoMIV
NgoMIV
NgoMIV
NgoMIV
AgeI
NgoMIV
H2O
5.4 µl
5.4 µl
5.4 µl
8.3 µl
7.8 µl
8.0 µl
7.9 µl
7.6 µl
8.6 µl
2.0 µl
Total volume
20µl
20µl
20µl
20µl
20µl
20µl
20µl
20µl
20µl
20µl
Incubation: 2 h
Preparative Agarose-Gel
1 g Agarose, 100 ml TAE (1%), 6 GELRED (3-6µl), at 120Volt, running time: 130 minutes
Lane
Sample
Sample/µl]
Loading dye (5x/6x)/µl
Expected size 1 (Geneious)
Expected size 2 (Geneious)
1
Affibody
20 µl
4 µl
bp
bp
2
Affibody
20 µl
4 µl
bp
bp
3
Affibody
20 µl
4 µl
bp
bp
10
His-Tag
20 µl
4 µl
bp
bp
2 g Agarose, 100 ml TAE (2%), 6 GELRED (3-6µl), at 120 Volt, running time: 130 minutes
Lane
Sample
Sample/µl]
Loading dye (5x/6x)/µl
4
Short Linker
20 µl
4 µl
5
Medium Linker
20 µl
4 µl
6
Long Linker
20 µl
4 µl
7
SEG Linker
20 µl
4 µl
8
GSAT Linker
20 µl
4 µl
9
YFP
20 µl
4 µl
Sequencing results of pSB1C3_hTERT
Gel extraction
Sample
Weight
Volume
Concentration
Affibody
0.4g
1200µl
29.78
Affibody
0.44g
1320µl
24.86
Short-Linker
0.27 g
810µl
0.22
Middle-Linker
0.38 g
1140µl
3.67
Long-Linker
0.35 g
1050µl
1.31
SEG-Linker
0.34 g
1020µl
2.13
GSAT-Linker
0.22 g
660µl
3.58
Ligation
Sample
Insert
V(Insert)
Vector
V(Vector)
pSB1C3_Affibody_Short-Linker
Affibody
1.06µl
Short-Linker
6.94µl
pSB1C3_Affibody_Middle-Linker
Affibody
5.25µl
Middle-Linker
2.75µl
pSB1C3_Affibody_Long-Linker
Affibody
2.93µl
Long-Linker
5.07µl
pSB1C3_Affibody_SEG-Linker
Affibody
2.46µl
SEG-Linker
5.54µl
pSB1C3_Affibody_GSAT-Linker
Affibody
3.41µl
GSAT-Linker
4.59µl
96. labday 21.08.2010
Transfection of AAV293, Harvest viral particles, Transduction of HT1080 and Fluorescent microscopy
Investigators: Adrian, Kerstin
The Motivation
Measure the Transduction efficiency from 19.8
The Motivation
Transfection of AAV293 cells from 19.8 (ten 10cm2 dishes with 2.000.000 cells each) with original standard protocoll!!! We want to try different 2xHBS Buffers (pH) and their influence on transfection efficiency!
The Plan
Requiered goods: 0,3 M CaCl2 , 2x HBS-Buffer , autoclaved millipore water, falcons, eppis
Working steps:
Remove the three plasmids to be co-transfected (the recombinant  pAAV expression plasmid or control plasmid, pAAV-RC, and pHelper)  from storage at –20°C.
Pipet 10 µg of each of the three plasmid DNA solutions into an eppi.
Add 1ml of 0.3 M CaCl2 and mix gently
Pipet  1ml of 2 × HBS  into a 15-ml falcon.
pipet the DNA/CaCl2 solution dropwise into the falcon (use the Pipetus).
Apply the DNA/CaCl2/HBS suspension to the plate of cells in a dropwise fashion, swirling gently to distribute the DNA suspension evenly in the medium.
Return the tissue culture plate to the 37°C room for 6 hours.
At the end of the incubation period, remove the medium from the plate and wash cells with warm PBS, replace it with 10 ml of fresh DMEM growth medium.
Return the plate to the 37°C incubator for an additional 72 hours.
We try 5 different 2xHBS buffers:
pH: 7.06
pH: 7.08
pH: 7.10
pH: 7.12
pH: 7.14
1. Plate: 10µg of Rep/Cap, pHelper and GOI=mVenus; pH(2xHBS)= 7.06
2. Plate: 10µg of Rep/Cap, pHelper and GOI=mVenus; pH(2xHBS)= 7.08
3. Plate: 10µg of Rep/Cap, pHelper and GOI=mVenus; pH(2xHBS)= 7.10
4. Plate: 10µg of Rep/Cap, pHelper and GOI=mVenus; pH(2xHBS)= 7.12
5. Plate: 10µg of Rep/Cap, pHelper and GOI=mVenus; pH(2xHBS)= 7.14
6. Plate: 10µg of Rep/Cap, pHelper and GOI=TKGMK;  pH(2xHBS)= 7.08
7. Plate: 10µg of Rep/Cap, pHelper and GOI=TKGMK;  pH(2xHBS)= 7.10
8. Plate: 10µg of Rep/Cap, pHelper and GOI=mVenus; pH(2xHBS)= 7.14
The transfected cells were seeded at 19.8 (2.000.000 cells per 10 cm2)
Used Plasmids: Standart protocol: 10µg of each plasmid
GOI: mVENUS => P262, conc:1148ng/µl; used amount: 8,71µl
GOI: TKGMK  => P82b, conc:1500ng/µl; used amount: 6,67µl
pHELPER => P64, conc:503ng/µl; used amount: 20µl
RepCap => 10.8: 1348ng/µl; used amount: 7,41µl
Harvest viral paricles from 18.08.2010 and transduction of HT1080 cells following the standard protocol
Analysis via Fluorescent microscopy (Transduction from 19.08.2010)
Results:
Midi-Prep of pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR clone1
Investigators: Chris W.
Comment: Midi-Preps of P 242, B200
The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
plasmid-no.
P269
p270
concentration (ng/µl)
1035,10
1376,95
Re-Trafo of pSB1C3_SDM_SspI_Bla14FM
Investigator: Stefan
Comment: Mini-Prep of pSB1C3_SDM_SspI_Bla14FM showed extremely low plasmid concentrations. It was proposed that this could correspond to using not the standard lab cells but rather XL-10 Gold cells for trafo. Therefore a re-trafo was performed using BL21 cells. 2 µl of each clone was added to BL21 cells and trafo was performed. Clones can be picked tonight (Volker)!
97. labday 22.08.2010
Test-digestion of pCerulean (P273,4,5), pSB1C3_VP1up (P279, 280), pSB1C3_NLS (P281,282), pAAV_RC_final (P283)
Investigator: Patrick
Digestions:
0,5 µl EcoRI, 0,5 µl Enzyme PstI, 1 µl BSA (10x), 1 µl Buffer 4 (10x), 1 µl pCerulean (P73), 6 µl H2O; total volume = 10 µl
0,5 µl EcoRI, 0,5 µl Enzyme PstI, 1 µl BSA (10x), 1 µl Buffer 4 (10x), 1 µl pCerulean (P74), 6 µl H2O; total volume = 10 µl
0,5 µl EcoRI, 0,5 µl Enzyme PstI, 1 µl BSA (10x), 1 µl Buffer 4 (10x), 1 µl pCerulean (P75), 6 µl H2O; total volume = 10 µl
1 µl NotI, 1 µl BSA (10x), 1 µl Buffer 4 (10x), 1 µl pSB1C3_VP1up (P279), 6 µl H2O; total volume = 10 µl
1 µl NotI, 1 µl BSA (10x), 1 µl Buffer 4 (10x), 1 µl pSB1C3_VP1up (P280), 6 µl H2O; total volume = 10 µl
0,5 µl EcoRI, 0,5 µl Enzyme PstI, 1 µl BSA (10x), 1 µl Bufer 4 (10x), 1 µl pSB1C3_NLS (P281) , 6 µl H2O; total volume = 10 µl
0,5 µl EcoRI, 0,5 µl Enzyme PstI, 1 µl BSA (10x), 1 µl Bufer 4 (10x), 1 µl pSB1C3_NLS (P282) , 6 µl H2O; total volume = 10 µl
0,5 µl Acc65I, 0,5 µl BspMI, 1 µl BSA (10x), 1 µl Buffer 3 (10x) , 1 µl pAAV_RC_final ,  6 µl H2O; total volume = 10 µl
Incubation time: 37°C, 1h
Expected size of the fragments:
pCerulean clone 1 : 771 & 3915 bp
pCerulean clone 2 : 771 & 3915 bp
pCerulean clone 3 : 771 & 3915 bp
pSB1C3_VP1up clone 1 : 468 & 2030 bp
pSB1C3_VP1up clone 2 : 468 & 2030 bp
pSB1C3_NLS clone 1 : 78 & 2031 bp
pSB1C3_NLS clone 2 : 78 & 2031 bp
pAAV_RC_final : 615 & 6720 bp
pCerulean clone 1 (P273), pSB1C3_VP1up clone 2 (P280), pSB1C3_NLS clone 2 (P282) and pAAV_RC_final (P283) were sent for sequencing although the results don't look very promising.
Labelling:
pCerulean clone 1 (P273) : iGEM2-CMV-F
pSB1C3_VP1up clone 2 (P280) : iGEM1-VF2
pSB1C3_NLS clone 2 (P282) : iGEM276-O51_VR2 and iGEM3-O51_VR2 (was ordered twice due to an ordering mistake)
pAAV_RC_final (P283) : iGEM4-SK
pAAV_RC_final (P283) : iGEM4-VP1 pKEX rev
pAAV_RC_final (P283) : iGEM4-Cap2800for
pAAV_RC_final (P283) : iGEM4-Cap2800rev
pAAV_RC_final (P283) : iGEM4-Cap3500for
Mini-Prep of pCerulean
Investigator: Stefan
Glycerol stocks were prepared:
B225 = pCerulean clone1
B226 = pCerulean clone2
B227 = pCerulean clone3
Mini-Prep following the standard protocol
P273 = pCerulean clone1 c= 419,52 ng/µl
P274 = pCerulean clone2 c= 383,37 ng/µl
P275 = pCerulean clone3 c= 382,72 ng/µl
Test digestion will be performed tomorrow.
98. labday 23.08.2010
Primer design: Sequencing primer for BioBrick Assembly of vectorplasmid and phTERT qPCR primer
Investigator: Bea
Comment: Two goals will be accomplished with this primers.
The first primer pair (left picture) serve as sequencing primers for the pSB1C3 vector plasmid. Problem was that the designed VR-2/VF-2 primers cannot be used for sequencing because the ITRs flank several assembled parts, mostly CMV, beta globin, GOI and hGH. These primers were designed to bind in the flanked region, directly after/before left and right ITR repsectivel and can therfore be used to sequence most of the flanked sequence inbetween the ITRs.
The second primer pair (right picture) will be used for performing the qPCR with th econtruct containing the phTERT promoter. The qPCR primers used for the other qRT-PCR cannot be used because these two primers anneal in the CMV region. The new designed primers bind in the phTERT region and can therefore bee used for determinining the infectious titers.
Primer design. Ordered at Sigma Aldrich
Mini-Prep of pSB1C3_CFP_middlelinker and pSB1C3_ZEGFR:1907 / testdigestion of pSB1C3_CFP_middlelinker and pSB1C3_lITR_pHTERT
Investigator: Jessica
Mini-Prep following the standard protocoll
P276 PSB1C3_CFP_middlelinker clone1 c=207,7 ng/µl
P277 PSB1C3_CFP_middlelinker clone2 c=203,0 ng/µl
P278 PSB1C3_CFP_middlelinker clone3 c=227,9 ng/µl
P279 pSB1C3_VP1up clone1 c=183,0 ng/µl
P280 pSB1C3_VP1up clone2 c=178,1 ng/µl
P281 pSB1C3_NLS clone1 c=155,7 ng/µl
P282 pSB1C3_NLS clone2 c=142,6 ng/µl
P283 pAAV_RC_final c=147,7 ng/µl
P284 pSB1C3_ZEGFR:1907 clone 1 c=146,5 ng/µl
P285 pSB1C3_ZEGFR:1907 clone 1 c=161,9 ng/µl
pSB1C3_Zegfr:1907 wurde bereits von Stefan geprept... was hast du hier gemacht Jessy? Retrafo vom Glycerolstock???
Test digestion:
Perfomed with 10 µL of total volume
 
Mastermix/µL
P256/µL
P257/µL
P276/µL
P277/µL
P278/µL
DNA
-
5
5,2
4,3
4,4
4,0
BSA (10x)
6
1
1
1
1
1
Buffer   4 (10x)
6
Enzyme   XbaI
3
Enzyme  PstI
3
H2O
-
2
1,8
2,7
2,6
3,0
Total volume
18
10
10
10
10
10
incubation time: 50'
incubation temparatur: 37°C
5µl new ladder mix
Sequencing:
Plasmid used: P276: pSB1C3_CFP_middlelinker
Primer used: VR-2
Tube name: JG 276
Continuation of BioBrick assembly for pSB1C3_Affibody_Linker
Investigator: Anna
Comment: Transformation was done of pSB1C3_Affibody with Short-Linker, Middle-Linker, Long-Linker, SEG-Linker and GSAT-Linker. Unfortunately, the digestion of pSB1C3_YFP_His-Tag didn't work (see 20.08).
Transformation was done following the standard protocol using XL1 blue cells.
To do: Repetition of BioBrick assembly for pSB1C3_YFP_His-Tag.
Cloning of pSB1C3_ß-globin_YFP and pSB1C3_hGH_rITR
Investigator: Anna
Comment: pSB1C3_ß-globin_mVenus (p178) was digested with SpeI and PstI and pSB1C3_hGH_rITR (p186) was digested with XbaI and PstI .
Vector:
pSB1C3_ß-globin_YFP (p178): c= 237,2 ng/µl
Insert:
pSB1C3_hGH_rITR (p186): c= 105,2 ng/µl
components
Vector (p178)
Insert (p186)
DNA
4,2
14
BSA (10x)
1,5
2,5
Buffer 4 (10x)
1,5
2,5
Enzyme EcoRI
1
1,5
Enzyme PstI
1
1,5
H2O
5,8
3
Total volume
15
25
Gelextraction of pSB1C3_RFC25_CFP:
0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 120 Volt, running time:55
Marker: GeneRuler ladder mix (Fermentas)
c(pSB1C3_ß-globin_YFP)= 16.9 ng/µl; size: 3293 bp
c(hGH_rITR)= 7,5 ng/µl; size: 657 bp
Ligation of PCR products and vector:
For the Ligation 1µl T4 buffer (2x) and 1µl T4 ligase were used. Incubation time: 30 min
vector /µl
insert /µl
pSB1C3_ß-globin_mVenus + hGH_rITR
3,41
4,59
Transformation:
The transformation was done following the standard protocol using XL1 blue cells.
To do: Picking clones of pSB1C3_ß-globin_mVenus_hGH_rITR.
Repetition: Quickchange site-directed mutagenesis of pSB1C3_SDM_SspI_Bla14FM
Investigator: Stefan
Comment: To delete a PvuII restriction site in the CAT marker, a SDM will be performed.
PCR reaction:
Ingredients
v / µl
10x Pfu Ultra II buffer
2,5
dNTP
0,5
forward primer: O109 (pSB1C3 PvuII for)
0,59
reverse primer: O108 (pSB1C3 PvuII rev)
0,58
DNA Template 10ng (1:100 dilution)
4
DMSO (1%)
0,5
PfuUltra II fusion
0,5
H2O
15,83
Total volume
25
PCR program:
PCR Program
temperature/ °C
time / s
1
95
120
2
95
30
3
55
60
4 (Step 2-4: 20x)
68
130
Hold
4
SDM was performed according to protocol.
Transformation was performed according to protocol using XL1b cells.
FACS analysis, Seeding HT1080 and Hek293 cells
Investigator: Kerstin, Adrian
Seeding HEK293 cells (100.000, 200.000, 300.000, 400.000, 500.000, 600.000 and 700.000 cells per dish)for transfection
FACS analysis of transduction from 21.08.2010:
plate/well
transfection plan
living cells
YFP expression
eGFP expression
1.1
control, no cells
1.2
non treated cells
86,8%
1.3
Lipo-Transfection I
83%
63,1%
1.4
Lipo-Transfection I
84,3%
67,5%
1.5
Lipo-Transfection I
86,4%
68,4%
1.6
Lipo-Transfection I
81,7%
66,5%
2.2
non treated cells
83,9%
2.3
Lipo-Transfection II
82,6%
65%
2.4
Lipo-Transfection II
84,7%
68,7%
2.5
Lipo-Transfection II
81,5%
62,6%
2.6
Lipo-Transfection II
83,1%
69,9%
3.2
non treated cells
83,3%
3.3
3,3µg of RC, pHelper&YFP
81,5%
50,7%
3.4
3,3µg of RC, pHelper&YFP
75,6%
40%
3.5
3,3µg of RC, pHelper&YFP
84,7%
63,7%
3.6
3,3µg of RC, pHelper&YFP
82,3%
61,5%
4.3
10µg RC mut(P158a), pHelper, 3,3µg YFP
85,8%
66,5%
4.4
10µg RC mut(P158a), pHelper, 3,3µg YFP
84,5%
68,2%
4.5
10µg RC mut(P158b), pHelper, 3,3µg YFP
76,9%
52,4%
4.6
10µg RC mut(P158b), pHelper, 3,3µg YFP
56,6%
31,8%
5.2
non treated cells
88,2%
5.3
3,3µg RC mut(P158a), pHelper&YFP
83,9%
57,5%
5.4
3,3µg RC mut(P158a), pHelper&YFP
89,1%
48,0%
5.5
3,3µg RC mut(P158b), pHelper&YFP
88,8%
29,4%
5.6
3,3µg RC mut(P158b), pHelper&YFP
85,1%
40,6%
6.3
10µg RC, pHelper&eGFP
82,5%
75,2%
6.4
10µg RC, pHelper, 3,3µgeGFP
87,6%
80,1%
6.5
10µg RC, pHelper, 10µgeGFP
81,9%
41,8%
6.6
10µg RC, pHelper, 10µgeGFP
83,4%
27,95%
7.2
non treated cells
88,7%
7.3
10µg RC, pHelper, 20µgeGFP
87,1%
45,4%
7.4
10µg RC, pHelper, 20µgeGFP
86,6%
46,9%
7.5
10µg RC, pHelper, 40µgeGFP
89,8%
0,5% ??? (negative control?)
7.6
10µg RC, pHelper, 40µgeGFP
84,8%
54,2%
8.2
non treated cells
90,9%
8.3
10µg RC, pHelper, 3,3µg YFP
83,5%
41,2%
8.4
10µg RC, pHelper, 3,3µg YFP
84,9%
42,6%
8.5
negative control ?
86,7%
26,8%
8.6
negative control ?
84,3%
33,7%
Seeding HT1080 cells for transduction with viral stocks from 23.08.2010:
3x 6er wells(Transductionplan 23.08.2010):
Miniprep of pSBC13_Bla
Investigator: Chris L.
Plasmid Mini-Prep
Gycerol stock was prepared and labeled for pSB1C3_Bla clone 1.1 (B236), 1.2 (B237), 2.1 (B238) and 2.2 (B239).B167.
Miniprep was performed according to protocol.
Samples were labeled:
pSB1C3_Bla clone 1.1 = P286
pSB1C3_Bla clone 1.2 = P287
pSB1C3_Bla clone 2.1 = P288
pSB1C3_Bla clone 2.2 = P289
Concentrations:
pSB1C3_Bla clone 1.1 (P286): 95.24 ng/µl
pSB1C3_Bla clone 1.2 (P287): 84.92 ng/µl
pSB1C3_Bla clone 2.1 (P288): 43.66 ng/µl
pSB1C3_Bla clone 2.2 (P289): 43.95 ng/µl
Test digestion: pSB1C3_Bla 1.1, 1.2, 2.1 and 2.2 were digested with NotI. pSB1C3_Bla 1.1, 1.2, 2.1 and 2.2 were digested with SspI and SalI. pSB1C3_Bla 1.1, 1.2, 2.1 and 2.2 were digested with BamHI and PvuII.
components
volume of P286/µl
volume of P286/µl
volume of P286/µl
volume of P287/µl
volume of P287/µl
volume of P287/µl
volume of P288µl
volume of P288/µl
volume of P288µl
volume of P289/µl
volume of P289/µl
volume of P289/µl
DNA
2
2
2
2
2
2
4
4
4
4
4
4
BSA (10x)
1
1
1
1
1
1
1
1
1
1
1
1
Buffer 4 (10x)
1
1
1
1
1
1
1
1
1
1
1
1
Enzyme NotI
0,5
0
0
0,5
0
0
0,5
0
0
0,5
0
0
Enzyme SspI
0
0,5
0
0
0,5
0
0
0,5
0
0
0,5
0
Enzyme SalI
0
0,5
0
0
0,5
0
0
0,5
0
0
0,5
0
Enzyme BamHI
0
0
0,5
0
0
0,5
0
0
0,5
0
0
0,5
Enzyme PvuII
0
0
0,5
0
0
0,5
0
0
0,5
0
0
0,5
H2O
5.5
5
5
5,5
5
5
3.5
3
3
3.5
3
3
Total volume /µl
10
10
10
10
10
10
10
10
10
10
10
10
Incubation time: 1 h, Incubation temperature: 37°
2 Gels were made:
0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , running time: 15 minutes at 90 Volt and 30 minutes at 120 Volt
2µl loading dye (6x) for the sample, Marker: 5 µl GeneRuler ladder mix (Fermentas)
Sample
Sample/µl]
Loading dye (6x)/µl
Expected size 1 (Geneious)
Expected size 2 (Geneious)
P286 NotI /µl
10 µl
2 µl
905 bp
2052 bp
P286 SspI/SalI /µl
10 µl
2 µl
854 bp
2099 bp
P286 BamHI/PvuII/µl
10 µl
2 µl
837 bp
2116 bp
P287 NotI /µl
10 µl
2 µl
905 bp
2052 bp
P287 SspI/SalI /µl
10 µl
2 µl
854 bp
2099 bp
P287 BamHI/PvuII/µl
10 µl
2 µl
837 bp
2116 bp
P288 NotI /µl
10 µl
2 µl
905 bp
2052 bp
P288 SspI/SalI /µl
10 µl
2 µl
854 bp
2099 bp
P288 BamHI/PvuII/µl
10 µl
2 µl
837 bp
2116 bp
P289 NotI /µl
10 µl
2 µl
905 bp
2052 bp
P289 SspI/SalI /µl
10 µl
2 µl
854 bp
2099 bp
P289 BamHI/PvuII/µl
10 µl
2 µl
837 bp
2116 bp
Marker: GeneRuler ladder mix
Marker
Sample P286 NotI /µl
Sample P286 SspI/SalI /µl
Sample P286 BamHI/PvuII/µl
Sample P287 NotI /µl
Sample P287 SspI/SalI /µl
Sample P287 BamHI/PvuII/µl
Marker
Lane
1
3
4
5
7
8
9
11
Marker: GeneRuler ladder mix
Marker
Sample P288 NotI /µl
Sample P288 SspI/SalI /µl
Sample P288 BamHI/PvuII/µl
Sample P289 NotI /µl
Sample P289 SspI/SalI /µl
Sample P289 BamHI/PvuII/µl
Marker
Lane
1
3
4
5
7
8
9
11
Comments: Strange results, seems like the test digestion didn't work out. Repitition of test digestion tomorrow.
99. labday 24.08.2010
Sequence analysis of pSB1C3_CFP_middlelinker
Investigator: Jessica
pSB1C3_CFP_middlelinker is ready
Harvest viral particles, Transduction of HT1080
Investigator: Kerstin
Harvest viral particles following the standard protocol
Transduction of three 6-well plates:
Plate 1 YFP: 150.000 cells per well
1
2
3
A
control, no cells
500µl virus (1)
500µl virus (2)
B
control, no virus
500µl virus (1)
500µl virus (2)
Plate 2 YFP: 150.000 cells per well
1
2
3
A
control, no cells
500µl virus (3)
500µl virus (4)
B
control, no virus
500µl virus (3)
500µl virus (4)
Plate 3 YFP: 200.000 cells per well
1
2
3
A
control, no cells
500µl virus (5)
500µl virus (9)
B
control, no virus
500µl virus (9)
500µl virus (5)
(1)= 10µg RC, 10µg pHelper, 10µg GOI (YFP); pH(2xHBS)=7,06
(2)= 10µg RC, 10µg pHelper, 10µg GOI (YFP); pH(2xHBS)=7,08
(3)= 10µg RC, 10µg pHelper, 10µg GOI (YFP); pH(2xHBS)=7,10
(4)= 10µg RC, 10µg pHelper, 10µg GOI (YFP); pH(2xHBS)=7,12
(5)= 10µg RC, 10µg pHelper, 10µg GOI (YFP); pH(2xHBS)=7,14
(9)= 10µg RC, 10µg pHelper, 3,3µg GOI (eGFP)
Mini-prep of SDM NgoMIV CD
Investigator: Kira
Motivation: Harvest DNA for further analysis, e.g. test digestion
Mini-prep was performed on 3 colonies.
c(clone1) = 358, 05 ng/ul c(clone2) = 352, 10 ng/ul c(clone 3) = 376, 53 ng/ul
Sent for sequencing: pSB1C3_6xHis
Investigator: Jessica
Plasmid: P84 c= 107,8 ng/µl
Primer: VR2
tube number: JG84
Subcloning Cap into pAAV_RC_ins-rep
Investigator: Stefan
Comment: Sequencing of pAAV_RC_final showed that Cap was not inserted succsessfully. Therefore it has to be repeated.
Digestion of vector and insert:
Insert: pMA_RepCap Vector_SDM_InsPvuII clone 1 (P211)
Vector: pAAV_RC_ins-rep clone1 (P250)
A two-step digestion was performed:
1st step
P211 / µl
P250 / µl
DNA
5,8
5,2
buffer 3
2
2
Enzyme BsiWI
1
1
H2O
11,2
11,8
total volume
20
20
2nd step
P211 / µl
P250 / µl
Mix obtained from step 1
20
20
Enzyme BspMI
1
1
H2O
1,25
1,25
total volume
22,25
22,25
Samples were loaded on a 0,8% agarose gel an run at 115V for about 60 minutes.
pAAV RC final
Gelextraction:
Gelex was performed according to protocol exept for elution was done with 60 µl instead of 20 µl.
c(P211)= 1,14 ng/µl
c(P250)= 5,60 ng/µl
T4 Ligation:
T4 ligation was performed according to protocol. used DNA amounts:
v(P211)= 2,56 µl
v(P250)= 5,44 µl
Transformation:
Trafo was performed according to protocol using XL1b cells.
Test digestion of pCerulean (p273)
Investigator: Anissa
Comments:Sequencing of pCerulean showed strange results, because the qualtity of sequenzing was not good. For working further with pCerulean, we started with a test digestion, to be sure, everything works.
Because the sequencing showed no AgeI, we cut one time with AgeI and EcoRI, the other time with NotI.
Components
p273/µL
p273/µL
DNA
2
2
BSA (10x)
1,5
1,5
Buffer 4 (10x)
1,5
1,5
Enzyme 1
0,5 NotI
0,5 EcoRI
Enzyme 2
0,5 NotI
0,5 AgeI
H2O
9
9
Total volume
15
15
Comments: Results of digestion seem to be all right
Cloning of VP1up into PCerulean
Investigator: Anissa
VP1up was cut out of pSB1C3 and into pCerulean
Digestion:
Components
Vector/µL
Insert/µL
DNA
3,7
8,4
BSA (10x)
1,5
2
Buffer 4 (10x)
1,5
2
Enzyme 1 EcoRI
1
1
Enzyme 2 PstI
1
1
H2O
6,3
5,6
Total volume
15
20
Gel:
A 1% agarose-gel was made, after 30 minutes samples were cut
Gelextraction was performed according the standardprotocol
Ligation:
Components
used volume for T4 ligation/µL
concentration /ng/µl
Vector
5,15
27,7
Insert
2,85
15,4
In addition 1µl T4-ligase and 1µl T4-ligase-buffer were added
Transformation: was performes into Xl1 blue according the standard-protocol
Sequence analysis of sequencing reads prepared yesterday (23.08.2010)
Investigator: Bea
GENERAL COMMENT: pSB1C3 {NLS and VP1up} were sent for sequencing. Aswell as the pAAV_RC_final which contains all four mutations in the Rep-Cap gene and the integrated and synthesiezd rep and cap gene and the plasmid pCerulean whith the CMV promoter and sv40 polyadenylation site.
Comments: Sequence analysis of pAAV_RC_final containing the subcloned "rep" (ordered) and "Cap (ordered) plus the four mutations to delete the restriction sites PstI, BamHi and SalI.
Primer used:
VP1 primer for pKex
GATC_std_SK
Primer Cap 2800 rev
Primer Cap 2800 for
Primer Cap 3500 for
Plasmid sequenced: P283
Sequence sample: iGEM4
Stored in Geneious-folder: RepCap insert ordered
pAAV RC final
Results:
Sequencing of the mutation ok. Rep integration worked. Cap integration needs to be repeated.
Next steps: Repetition of the integration of the synthesized cap gene performed by Stefan (see topic).
Comments: Sequence analysis of pSB1C3_NLS
Primer used: VR-2
Plasmid sequenced: P282
Sequence sample: iGEM3_O51_VR-2
Stored in folder: pSB1C3_NLS
pSB1C3_NLS
Results:
Insertion of the nuclear localisation signal (NLS) into the pSB1C3_NLS as one step in the N-terminal insertion of targeting molecules. Sequencing results look good. prefix and suffix ok!
Next steps: Fuse NLS to targeting molecule.
Comments: Sequence analysis of pSB1C3_VP1up
Primer used: VF-2
Plasmid sequenced: P280
Sequence sample: iGEM1_VF-2
Stored in folder: N-Terminal targeting --> pSB1C3_VP1up
pSB1C3_VP1up
Results:
Prefix ok, but annotation is wrong. VP1up should start with: GC!!  Suffix ok! The additional EcoRI before the prefix standard can be due to the bad sequence read quality.
Next steps: Clone VP1up into pCerulean in order to obtain the plasmid with a CMV promoter and the sv40 polyadenylation site.
Comments: Sequence analysis of pCerulean
Primer used: GATC_CMV-F
Plasmid sequenced: P273
Sequence sample: iGEM2_CMV-F
Stored in folder: N-Terminal targeting --> pCerulean
pCerulean
Results:
It seem that the suffix is not correct which means that AgeI, PstI and NotI is missing. This can be due to the bad sequence quality. Prefix is ok!
Next steps: For verification a test digestion with the missing restriction sites will be perforemd and another round of sequencing will be conducted.
top of page
Test digestion of SDM NgoMIV CD
Investigator: Kira
Motivation: In order to check if site directed mutagenesis was successfull, test digestion was performed with 3 clones, which were picked yesterday as well as with 'original' DNA, which still contains all the restriction sites
Test digestion:
Components
clone 1/µL
clone 2/µL
clone 3/µL
original/µL
DNA (500ng)
1,4
1,4
1,4
1,4
BSA (10x)
1
1
1
1
Buffer 4 (10x)
1
1
1
1
Enzyme 1 NgoMIV
0,5
0,5
0,5
0,5
H2O
6,1
6,1
6,1
6,1
Total volume
10
10
10
10
Bitte schneid mich aus !
According to the gel, NgoMIV SDM was successful in clones 1 and 2. Clone 3 well reveales 2 bands of unknown origin. Samples 1 and 3 were sent for sequencing.
Picking clones of pSB1C3_Affibody_linker and pSB1C3_ß-Globin_YFP_hGH_rITR
Investigator: Anna
Comments: A 1:1000 dilution of pSB1C3_Affibody_GSAT-Linker was prepared.
To do: Mini-Prep of pSB1C3_Affibody_(Short/Middle/Long and SEG-Linker) and pSB1C3_ß-Globin_YFP_hGH_rITR. Picking clones of pSB1C3_Affibody_GSAT-Linker.
Test digestion of pSB1C3_SDM_SspI_Bla14FM (P222), pSB1C3_BLA (P286, P288)
Investigator: Achim
CommentsNew test digestion of two SDM-attempts, this time we also digested the original vector containing the PvuII restriction site to see differences.
The original vector is being cut as expected: the uncut vector can be seen in different conformations, the linearized vector shows one distinct band and the digestions cutting out BLA show a band at ~900 bp. Our SDM attempts show inconsistent results, not only does the Pvu site still seem to be in the backbone, there are no more proper insert bands visible either. We therefore conclude that either the DpnI digestion or the dilution of the transformation must have gone wrong (differing plasmids...). Tomorow we'll try a new cloning approach using an old vector with deleted Pvu restriction site and MscI & XbaI digestion. In case this also fails we also yet have to prep and test digest the repetition of the mutagenesis which was carried out by Stefan.
pSB1C3_VP1up
Results:
Prefix ok, but annotation is wrong. VP1up should start with: GC!!  Suffix ok! The additional EcoRI before the prefix standard can be due to the bad sequence read quality.
Next steps: Clone VP1up into pCerulean in order to obtain the plasmid with a CMV promoter and the sv40 polyadenylation site.
Comments: Sequence analysis of pCerulean
Primer used: GATC_CMV-F
Plasmid sequenced: P273
Sequence sample: iGEM2_CMV-F
Stored in folder: N-Terminal targeting --> pCerulean
pCerulean
Results:
It seem that the suffix is not correct which means that AgeI, PstI and NotI is missing. This can be due to the bad sequence quality. Prefix is ok!
Next steps: For verification a test digestion with the missing restriction sites will be perforemd and another round of sequencing will be conducted.
top of page
Test digestion of SDM NgoMIV CD
Investigator: Kira
Motivation: In order to check if site directed mutagenesis was successfull, test digestion was performed with 3 clones, which were picked yesterday as well as with 'original' DNA, which still contains all the restriction sites
Test digestion:
Components
clone 1/µL
clone 2/µL
clone 3/µL
original/µL
DNA (500ng)
1,4
1,4
1,4
1,4
BSA (10x)
1
1
1
1
Buffer 4 (10x)
1
1
1
1
Enzyme 1 NgoMIV
0,5
0,5
0,5
0,5
H2O
6,1
6,1
6,1
6,1
Total volume
10
10
10
10
Bitte schneid mich aus !
According to the gel, NgoMIV SDM was successful in clones 1 and 2. Clone 3 well reveales 2 bands of unknown origin. Samples 1 and 3 were sent for sequencing.
Picking clones of pSB1C3_Affibody_linker and pSB1C3_ß-Globin_YFP_hGH_rITR
Investigator: Anna
Comments: A 1:1000 dilution of pSB1C3_Affibody_GSAT-Linker was prepared.
To do: Mini-Prep of pSB1C3_Affibody_(Short/Middle/Long and SEG-Linker) and pSB1C3_ß-Globin_YFP_hGH_rITR. Picking clones of pSB1C3_Affibody_GSAT-Linker.
Test digestion of pSB1C3_SDM_SspI_Bla14FM (P222), pSB1C3_BLA (P286, P288)
Investigator: Achim
CommentsNew test digestion of two SDM-attempts, this time we also digested the original vector containing the PvuII restriction site to see differences.
The original vector is being cut as expected: the uncut vector can be seen in different conformations, the linearized vector shows one distinct band and the digestions cutting out BLA show a band at ~900 bp. Our SDM attempts show inconsistent results, not only does the Pvu site still seem to be in the backbone, there are no more proper insert bands visible either. We therefore conclude that either the DpnI digestion or the dilution of the transformation must have gone wrong (differing plasmids...). Tomorow we'll try a new cloning approach using an old vector with deleted Pvu restriction site and MscI & XbaI digestion. In case this also fails we also yet have to prep and test digest the repetition of the mutagenesis which was carried out by Stefan.
100. labday 25.08.2010
Cloning of pSB1C3_Rep40, pSB1C3_Rep68 and pMA_RC_insert
Investigator: Chris L.
buffer used: 2; Restriction-enzymes used: Enzyme 1: HindIII ; Enzyme 2: BstEII
Plasmids:
pSB1C3_Rep40 P231
pSB1C3_Rep68 P266
Insert:
pMA_RC_insert P190
Components
P190/µL
P231/µL
P266/µL
DNA
5.9
6.4
5
BSA (10x)
2
2
2
Buffer 2 (10x)
2
2
2
Enzyme 1 HindIII
1
1
1
Enzyme 2 BstEII
1
1
1
H2O
8.1
7.6
9
Total volume
20
20
20
Incubation: 90 minutes; 37°C with HindIII
Incubation: 90 minutes; 60°C with BstEII
Agarosegel
0.5 g Agarose, 50 ml TAE (1 %), 3 µL GELRED, 5 min at 90 Volt, 40 min at 115 Volt
Concentrations measured via NanoDrop:
pSB1C3_Rep40: 81.97 ng/µl
pSB1C3_Rep68: 48,68 ng/µl
RC_Insert: 3,60 ng/µl
T4 Ligation of pSB1C3_Rep40 with RC_Insert
Volume insert: 6,54 µl
Volume vector: 1,46 µl
T4 Ligation of pSB1C3_Rep68 with RC_Insert
Volume insert: 5,46 µl
Volume vector: 2,54 µl
Trafo was prepared with XL1blue and Cm
Picking clones of pAAV_RC-ins_rep_cap and pCeruleanVP1up
Investigator: Chris L.
To do: Mini-Prep of pAAV_RC-ins_rep_cap and pCeruleanVP1up.
Cloning of pSB1C3_6xHis with middlelinker and pSB1C3_6xHis with pGA14_mVenus_YFP
Investigator: Jessica
buffer used: 4; Restriction-enzymes used: Enzyme 1: AgeI ; Enzyme 2: PstI ; Enzyme3: NgoMIV
Plasmids:
pSB1C3_6xHis P84
pGA14_middle linker P65
GA14_mVenus_YFP P60
Components
Mastermix
P84/µL
P65/µL
P60/µL
DNA
-
13,91
13,52
8,93
BSA (10x)
-
2
2
2
Buffer 4 (10x)
-
2
2
2
Enzyme 1 AgeI
-
1
-
-
Enzyme 2  PstI
-
1
1
1
Enzyme 3  NgoMIV
-
-
1
1
H2O
-
0,09
0,48
5,07
Total volume
-
20
20
20
Incubation:110 minutes; 37°C
Agarosegel
0.5 g Agarose, 50 ml TAE (1 %), 3 µL GELRED, at 120 Volt
concentrations measured via NanoDrop:
SB1C3_6xHis:  1,7 ng/µl
middle linker:  3,8 ng/µl
mVenus_YFP:  3,2 ng/µl
T4 Ligation pSB1C3_6xHis_middlelinker
Volume insert: 0,25 µl
Volume vector: 7,75 µl
T4 Ligation pSB1C3_6xHis_mVenus_YFP
Volume insert: 2,64 µl
Volume vector: 5,36 µl
Trafo was prepared with XL1blue and Cm
Sequence analysis of pSB1C3_6xHis
Investigator: Jessica
P84 was sequenced for continuation of working with it
pSB1C3_6xHis P84 is ready to use, but is empty. a new tube will be made tomorrow
Inoculation of pGA14_middlelinker and pSB1C3_6xHis clone1
Investigator: Jessica
both tubes (P65 and P84)are empty --> inoculation with glycerolstocks
B48 pGA14_middlelinker was inoculated with 10ml DYT and 10µl Amp
B64 pSB1C3_6xHis clone1 was inoculated with 10ml DYT and 10µl Cm
Cloning of pSB1C3_SspI_BLA and pSB1C3_CFP_PvuII
Investigator: Achim
Comment:I ligated a sequence from pSB1C3_CFP_PvuII missing the PvuII restriction site into pSB1C3_SspI_BLA to obtain a pSB1C3 vector without PvuII in the cat marker.
Vector:
pSB1C3_SspI_BLA (p222): c= 308,32 ng/µl
Insert:
pSB1C3_CFP_PvuII (p129): c= 264,86 ng/µl
components
Vector
Insert
DNA
3,8
4,9
BSA (10x)
2
2
Buffer 4 (10x)
2
2
Enzyme MscI
1
1
Enzyme XbaI
1
1
H2O
10,2
9,1
Total volume
20
20
Gelextraction:
0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 110 Volt, running time:45
Marker: GeneRuler ladder mix (Fermentas)
c(Insert)= 15,66 ng/µl; size: 730 bp
c(Vector)= 14,42 ng/µl; size: 2200 bp
Ligation of PCR products and vector:
For the Ligation 1µl T4 buffer (2x) and 1µl T4 ligase were used. Incubation time: 60 min due to blunt end ligation
vector /µl
insert /µl
pSB1C3_BLA
0,86
7,14
Transformation:
The transformation was done following the standard protocol using XL1 blue cells.
Comment: Two clones were picked, but because we already obtained our plasmid via SDM in the meantime, no prep was performed. Glycerol stocks of the two clones were created just in case, B264 and B265.
Mini-Preps and test digestion of pSB1C3_Affibody_Middle-Linker, pSB1C3_Affibody_Short-Linker, pSB1C3_Affibody_SEG-Linker, pSB1C3_Affibody_Long-Linker and pSB1C3_betaglobin_mVenus_hGH_rITR
Investigator: Stefan
Glycerol stocks were prepared:
B240 = pSB1C3_Affibody_Middle-Linker clone1
B241 = pSB1C3_Affibody_Middle-Linker clone2
B242 = pSB1C3_Affibody_Short-Linker clone 1
B243 = pSB1C3_Affibody_Short-Linker clone 2
B246 = pSB1C3_Affibody_SEG-Linker clone1
B247 = pSB1C3_Affibody_SEG-Linker clone2
B248 = pSB1C3_Affibody_Long-Linker clone 1
B249 = pSB1C3_Affibody_Long-Linker clone 2
B244 = pSB1C3_betaglobin_mVenus_hGH_rITR clone 1
B245 = pSB1C3_betaglobin_mVenus_hGH_rITR clone 2
Mini-Prep following the standard protocol
P290 = pSB1C3_Affibody_Middle-Linker clone1 c= 227,36 ng/µl
P291 = pSB1C3_Affibody_Middle-Linker clone2 c= 219,05 ng/µl
P292 = pSB1C3_Affibody_Short-Linker clone 1 c= 196,63 ng/µl
P293 = pSB1C3_Affibody_Short-Linker clone 2 c= 224,76 ng/µl
P296 = pSB1C3_Affibody_SEG-Linker clone1 c= 218,68 ng/µl
P297 = pSB1C3_Affibody_SEG-Linker clone2 c= 229,66 ng/µl
P298 = pSB1C3_Affibody_Long-Linker clone 1 c= 229,73 ng/µl
P299 = pSB1C3_Affibody_Long-Linker clone 2 c= 202,05 ng/µl
P294 = pSB1C3_betaglobin_mVenus_hGH_rITR clone 1 c= 425,14 ng/µl
P295 = pSB1C3_betaglobin_mVenus_hGH_rITR clone 2 c= 328,28 ng/µl
Test digestion
Restriction-enzymes used: for P290-P293 and P296-P299: NotI ; for P294-P295: NgoMIV and PstI
comment: The same amount of ingredients were used for P290-P293 and P296-P299, therefore these approaches will be merged into the chart. The same goes for P294-P295.
Components
P290-P293 and P296-P299
P294-P295
DNA
1
1
BSA (10x)
1
1
Buffer 4 (10x)
1
1
NotI
0,5
-
NgoMIV
-
0,5
PstI
-
0,5
H2O
6,5
6
Total volume
10
10
Incubation time: 70 minutes; Incubation temperature: 37°
0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , running time:5 minutes at 90 Volt and 50 minutes at 115 Volt
2µl loading dye (6x) for the sample, Marker: GeneRuler ladder mix (Fermentas)
Comment: Sizes of fragments look like expected. Clone 1 of each plasmid will be sent for sequencing tomorrow.
Sequenzing evaluation of SDM NgoMIV
Investigator: Kira
2 samples have been sent for sequencing yesterday. According to the data, both samples do not contain any NgoMIV restriction site anymore.
101. labday 26.08.2010
Mini-Preps of pGA14_middle linker and pSB1C3_6xHis clone 1
Investigator: Jessica
Glycerol stocks were not prepared because I have inoculated from glycerol stocks
Mini-Prep following the standard protocol
P301 = pGA14_middle linker (= P65)
c=305,7
P300 = pSB1C3_6xHis clone 1 (= P84)
c=177,5
Mini-Preps and test digestion of pAAV_RC-ins_rep_cap, pCeruleanVP1up and pAAV_RC_ins-rep-cap.
Investigator: Chris L.
Glycerol stocks were prepared:
B250 = pCerulean_VP1up clone1
B251 = pCerulean_VP1up clone2
B252 = pCerulean_VP1up clone3
B253 = pCerulean_VP1up clone4
B254 = pSB1C3_VCK_Bla clone1
B255 = pSB1C3_VCK_Bla clone2
B256 = pSB1C3_VCK_Bla clone3
B257 = pSB1C3_VCK_Bla clone4
B262 = pAAV_RC_ins-rep-cap clone1
B263 = pAAV_RC_ins-rep-cap clone2
Mini-Prep following the standard protocol
P302 = pCerulean_VP1up clone1
c=410,70 ng/µl
P303 = pCerulean_VP1up clone2
c=387,78 ng/µl
P304 = pCerulean_VP1up clone3
c=321,04 ng/µl
P305 = pCerulean_VP1up clone4
c=350,34 ng/µl
P306 = pSB1C3_VCK_Bla clone1
c=82,53 ng/µl
P307 = pSB1C3_VCK_Bla clone2
c=92,70 ng/µl
P308 = pSB1C3_VCK_Bla clone3
c=82,22 ng/µl
P309 = pSB1C3_VCK_Bla clone4
c=105,34 ng/µl
P314 = pAAV_RC_ins-rep-cap clone1
c=517,04 ng/µl
P315 = pAAV_RC_ins-rep-cap clone2
c=444,33 ng/µl
Test digestion:
components
volume of P302/µl
volume of P303/µl
volume of P304/µl
volume of P305/µl
volume of P306/µl
volume of P306/µl
volume of P306µl
volume of P307/µl
volume of P307µl
volume of P307/µl
volume of P308/µl
volume of P308/µl
volume of P308/µl
volume of P309/µl
volume of P309/µl
volume of P309/µl
volume of P314/µl
volume of P315/µl
DNA
1
1
1
1
2
2
2
2
2
2
2
2
2
2
2
2
1
1
BSA (10x)
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
Buffer 4 (10x)
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
-
-
Buffer 2 (10x)
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
1
1
Enzyme NotI
-
-
-
-
0,5
-
-
0,5
-
-
0,5
-
-
0,5
-
-
-
-
Enzyme SspI
-
-
-
-
-
0,5
-
-
0,5
-
-
0,5
-
-
0,5
-
-
-
Enzyme SalI
-
-
-
-
-
-
0,5
-
-
0,5
-
-
0,5
-
-
0,5
-
-
Enzyme BamHI
-
-
-
-
-
-
0,5
-
-
-
-
-
-
-
-
-
-
-
Enzyme PvuII
-
-
-
-
-
-
0,5
-
-
-
-
-
-
-
-
-
-
-
Enzyme Acc65I
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
0,75
0,75
Enzyme XcmI
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
0,5
0,5
Enzyme PstI
0,5
0,5
0,5
0,5
-
-
-
-
-
-
-
-
-
-
-
-
-
-
Enzyme EcoRI
0,5
0,5
0,5
0,5
-
-
-
-
-
-
-
-
-
-
-
-
-
-
H2O
6
6
6
6
5,5
5
5
5,5
5
5
5,5
5
5
5,5
5
5
5,75
5,75
Total volume /µl
10
10
10
10
10
10
10
10
10
10
10
10
10
10
10
10
10
10
Incubation time: 1 h, Incubation temperature: 37°
Preparation of gel:
1 g Agarose, 100 ml TAE (1%), 6 µl GELRED , at 115 Volt, running time: 50 minutes
Results:
pSB1C3_VCK_Bla: Test digestion looks like expected. Clone 4 was sent for sequencing.
pCerulean_VP1up: Gel looks well as well. clone 2 was sent for sequencing.
pAAV_RC_ins-rep-cap: Test  digestion looks strange. Just one big band with more than 10000 bp and one very small with 150 bp. Maybe the restriction enzymes didn´t cut. (@Christian: please insert picture)
102. labday 27.08.2010
Sequence analysis of pSB1C3_Bla_final (P309)and pCerulean_VP1up (P303)
Investigator: Bea
GENERAL COMMENT: pSB1C3_Bba_final (P309, SB_9) and pCerulean_Vp1up (P303; SB_10) were sent for sequencing. In the case of SB1C3_Bba_final, which means that the whole vector was assembled from the buttom,the purpose was to test the designed primers. In the other case we wanted to verify the insertion of Vp1up inot pCerulean.
Comments: Sequence analysis of pSB1C3_leftITR_CMV_betaglobin_mVenus_hGH_rightITR.
Primer used:
pSB1C3_Bba_seq_for
pSB1C3_Bba_seq_rev
Plasmid sequenced: P??
Sequence sample: ??
Stored in Geneious-folder: BioBricks --> final parts
pSB1C3_leftITR_CMV_betaglobin_mVenus_hGH_rightITR
Results:
Results look good. The forward primer worked, still waiting for the results of the reverse primer. The scar inbetween the CMV promoter and the beat globin intron corresponds to the expected result.
Next steps: Wwait until sequencing results of the reverse primer can be confirmed aswell. If that will not be the case, order new sequensing primer.
top of page
Comments: Sequence analysis of pCerulean_Vp1up.
Primer used: CMV-F
Plasmid sequenced: P303
Sequence sample: ??
Stored in Geneious-folder: N-terminal Targeting --> pCerulean_Vp1up
pCerulean_Vp1up
Results:
Results are good. Insertion of the Vp1up region can be confirmed.
Next steps: Fuse pCerulean_Vp1up to the NLS.
top of page
Hybridisation and cloning of loop insertions
Investigator: Achim, Volker
We hybridized the oligos for the different loop insertions and cloned them into the pSB1C3_BLA backbone.
Hybridisation
4 Inserts contained overlapping ends and had to be filled up using Klenov fragments. We therefore added Klenow buffer and dNTPs to those samples.
Components
453 BAP
587 BAP
587 KO BAP
453 RGD
587 RGD
587 KO RGD
453 HIS
587 HIS
587 KO HIS
587 KO EMPTY
453 Z34C
587 Z34C
587 KO Z34C
587 KO Z34C SPACER
Oligo 1
O124: 10
O126: 10
O128: 10
O130: 10
O132: 10
O134: 10
O135: 10
O137: 10
O139: 10
O141: 10
O143: 10
O145: 10
O147: 10
O149: 10
Oligo 2
O125: 10
O127: 10
O129: 10
O131: 10
O133: 10
O151: 10
O136: 10
O138: 10
O140: 10
O142: 10
O144: 10
O146: 10
O148: 10
O150: 10
TrisHCl pH8
4
4
4
4
4
4
4
4
4
4
-
-
-
-
5mM MgCl2
8
8
8
8
8
8
8
8
8
8
-
-
-
-
Klenow Buffer
-
-
-
-
-
-
-
-
-
-
4
4
4
4
dNTP Mix
-
-
-
-
-
-
-
-
-
-
1
1
1
1
H2O
8
8
8
8
8
8
8
8
8
8
15
15
15
15
Total volume
40
40
40
40
40
40
40
40
40
40
40
40
40
40
Hybridisation was carried out according to standard protocoll
Klenov fill-in reaction:
Added 1 µl of NEB Klenow fragment to samples 11,12,13,14
incubated for 1h  @ 37°C
Digestion of pSB1C3_BLA vector and Samples 11-14
We digested the standard vector with the 453 (Ssp/Sal) and the 587 (Bam/Pvu) standard. The samples that were filled in were also digested to create sticky ends.
Components
V453
V587
11
12
13
14
DNA
2,5
2,5
3,5
3,5
3,5
3,5
BSA (10x)
-
-
-
-
-
-
Buffer 4 (10x)
2
2
2
2
2
2
Enzyme 1
Ssp: 1
Bam: 1
Ssp: 1
Bam: 1
Bam: 1
Bam: 1
Enzyme 2
Sal:1
Pvu: 1
Sal: 1
Pvu: 1
Pvu: 1
Pvu: 1
H2O
13,5
13,5
12,5
12,5
12,5
12,5
Total volume
20
20
20
20
20
20
Gel Extraction
Ligation
Cloning of pCerulean_VP1up_NLS
Investigator: Anissa
Comment:Cloning did not work, because no NLS-band could be seen in the gel... Will be repeated on monday in two new approaches. One time VP1up will be cloned into pSB1C3_NLS and recloned as a fusion-product into pCerulean. Another time the oligos of NLS will be hybridisized, cut with rescriction enzymes and purificated with the Qiaex II kit.Then the NLS will be ligated into the pCerulean_VP1up
Digestion:
components
Vector
Insert
DNA
2,6
32
BSA (10x)
1,5
-
Buffer 4 (10x)
1,5
4
Enzyme
1 PstI
2 PstI
Enzyme
1 AgeI
2 NgoMIV
H2O
7,4
-
Total volume
15
40
Gel: that's only the picture of the 2% agarose gel for seperating the NLS. But no band could be seen.
DARPin E_01
Investigator: Bea
General overview of DARPins:
Natural protein ankyrin repeat molecules are motifs which can be found in proteins. These motifs are mediating protein-protein interactions. This suggests that ankyrin repeat (AR) proteins can be used for designing binding molecules. In Kohl et al. (2003) and Binz et al. (2003) the authors designed a structural framework with fixed consensus regions and randomized positions of interacting residues.
The repetitive nature of the ankyrin proteins allows modifications in their variable and modular binding surface. Therefore consensus sequences of natural ankyrin proteins were used to design novel and stable scaffolds for binding proteins.  Designed Ankyrin Proteins (DARPins) are well expressed, monomeric in solution, thermodynamically stable and have the ability to fold fast. In the publication of Steiner et al. (2008) screening libraries were created by useing the signal recognition particle (SRP) translocation pathway for phage display. The proteins containing the appropriate translocation signal sequence are efficiently displayed on filamentous phage particles. Screening for DARPins binding to specific target proteins like EGF-R (Erb1) were performed by phage ELISAs. The extracellular domain I-III of the receptor ErbB1 was fused to the Fc part of human IgG1 and used for selection. The previously described combinatorial N3C library was used as a template for SRP selection. After phage ELISA ans sequencing have been performed only one clone (E_01) could be selected with high-affinity binding characteristics. For broaden the diversity of ErbB1 DARPins three more clones were selected by epitope masking (E_67, E_68 and E_69). Binding and epitope localization experiments showed that all selected clones recognize a competing epitope as the dominant binder E_01, except of E_69 which recognizes an epitope not competing with E_01. The low diversity obtained for DARPins against ErbB1 suggests that the binding interfaces are well suited for dominant selection. Additionally the selected clones showed no cross-reactivity with other ErbB-family receptors (Figure 2C).
The dominant DARPpin E_01 has very high affinities to the target protein ErbB1 (Table 3) and can be used as a potential targeting molecule four our approach in fusing the DARPin to the N-terminal VP proteins.
Overview of DARPinE_01 used in our approach:
The designed ankyrin repeat protein used as a potential targeting molecule consists of three internal capping repeats and the C-and N-terminal capping repeats. Each internal repeat module comprises of one beta-turn and two hydrophobic alpha-helices. The potential interaction residues are located in the beta-turn the first alpha-helices of the AR-proteins. The complete nucleotide and corresponding amino acid sequence can be found in the appendix.
top of page
Update VP1 insertion
Investigator: Anissa, Bea
Done:
pSB1C3_VP1up was succesfully created by Achim. For details see labday from 2010-22-08.
pCerulean_VP1up was suceesfully created by Anissa. pCerulean was sequenced and the VP1up which contains the upstream 137 aa´s of the VP1 protein could be successfully cloned into the plasmid backbone pCerulean (obtained by performeing a PCR).
pSB1C3_NLSHybridization of the nuclear localization sequence (NLS) was successfully incorporated into the pSB1C3_CFP.
Today, Anissa tried to fuse the NLS to the VP1up sequence. Cutting the pSB1C3_NLS with NgomIV and PstI leads to a 45 nt fragment which normally can be resoluted in a 2% agarose gel. But no fragment could be detected in the gel (for further details see: Cloning of pCerulean_VP1up_NLS).
On monday another approach will be performed: The NLS oligos will be hybridized and digested with NgomIV and SpeI. With the Qiaex II Kit (protocol for desalting and concentrating DNA solutions),the hybrifized oligos will be purified. Parallel, another digestion approach will be conducted.
Overview
Next steps:
We obtain pSB1C3_VP23_HSPG_ko by conducting a PCR with the pAAV_RC_final (which contains the integrated "cap".
Fuse the obtained construct pSB1C3_Targeting Molcule to the pSB1C3_VP23_HSPG_ko.
Affibody ZEGFR:1907
DARPin E_01
6xHis Tag
CFP
Fuse pSB1C3_VP23_HSPG_ko_Targeting molecule to the construct pCerulean_VP1up_NLS.
Finally, for providing the possibilty to express obtained construct in trans: perform a site-directed mutagenesis with the pAAV_RC_final to remove the startcodon of VP1.
top of page
Sequence analysis of pSB1C3_BLA_final (P309)
Investigator: Bea
GENERAL COMMENT: pSB1C3_BLA_final (P309) was sent for sequencing. the beta lactamse was inserted and two site-directed mutagenesis have been performed in order to delete the loop insertion restriction enzymes in the CAT marker.
Comments: Sequence analysis of pSB1C3_BLA_final containing the two deleted restriction sites  (SspI and PvuII)in the CAT marker.
Primer used:
pQE-RP
VF-2
Plasmid sequenced: P309
Sequence sample: SB_9
Stored in Geneious-folder: pSB1C3_BLA
pSB1C3_BLA_final
Results:
The two performed site-directed mutagenesis performed at the backbone of the pSB1C3 containing the beta-lactamase can be confrimed as it can be seen in the two pictures above. Additionally the BLA sequence was sequenced aswell as the insertion of the BLA which can be confirmed with no mutations.
Next steps: This new plasmid can be used for further experiments. Clone the PCR prdouct of the pAAV_RC into the pSB1C3 and subclone the loop insertion motifs into the vector.
top of page
Mini-Preps and test digestion of pSB1C3_6xHis_middlelinker and pSB1C3_6xHis_mVenus
Investigator: Stefan
Glycerol stocks were prepared:
B258 = pSB1C3_6xHis_middlelinker clone1
B259 = pSB1C3_6xHis_middlelinker clone2
B260 = pSB1C3_6xHis_mVenus clone 1
B261 = pSB1C3_6xHis_mVenus clone 2
Mini-Prep following the standard protocol
P310 = pSB1C3_6xHis_middlelinker clone1
c= 185,31 ng/µl
P311 = pSB1C3_6xHis_middlelinker clone2
c= 178,87 ng/µl
P312 = pSB1C3_6xHis_mVenus clone 1
c= 247,62 ng/µl
P313 = pSB1C3_6xHis_mVenus clone 2
c= 268,98 ng/µl
Test digestion
Restriction-enzymes used: EcoRI and PstI
Components
P310+311
P312+313
DNA
2
2
BSA (10x)
1
1
Buffer 4 (10x)
1
1
EcoRI
0,5
0,5
PstI
0,5
0,5
H2O
5
5
Total volume
10
10
Incubation time: 90 minutes; Incubation temperature: 37°
1 g Agarose, 100 ml TAE (1%), 6 µl GELRED , running time: 55 minutes at 120 Volt
Comment: Test digestion looks good. Clones 1 of each plasmid (P310 and p312) were sent in for sequencing.
Repetition of test digestion of pAAV_RC_ins-rep-cap, pAAV_RC_ins-rep, pSB1C3_Rep40_RC_Insert and pSB1C3_Rep68_RC_Insert.
Investigator: Chris L.
P250 = pAAV_RC_ins-rep clone1
c=467,50 ng/µl
P314 = pAAV_RC_ins-rep-cap clone1
c=517,04 ng/µl
P315 = pAAV_RC_ins-rep-cap clone2
c=444,33 ng/µl
P316 = pSB1C3_Rep40_ins clone 1
c=302,22 ng/µl
P317 = pSB1C3_Rep40_ins clone 2
c=274,37 ng/µl
P318 = pSB1C3_Rep68_ins clone 1
c=491,69 ng/µl
P319 = pSB1C3_Rep68_ins clone 2
c=498,71 ng/µl
Test digestion:
components
volume of P250/µl
volume of P314/µl
volume of P315/µl
volume of P316/µl
volume of P317/µl
volume of P318/µl
volume of P319/µl
DNA
1
1
1
1
1
1
1
BSA (10x)
1
1
1
1
1
1
1
Buffer 2 (10x)
1
1
1
1
1
1
1
Enzyme Acc65I
0,75
0,75
0,75
-
-
-
-
Enzyme XcmI
0,5
0,5
0,5
-
-
-
-
Enzyme HindIII
-
-
-
0,5
0,5
0,5
0,5
Enzyme BstEII
-
-
-
0,5
0,5
0,5
0,5
H2O
5,75
5,75
5,75
6
6
6
6
Total volume /µl
10
10
10
10
10
10
10
Incubation time: 1 h, Incubation temperature: 37° for P250, P314 and P315
Incubation time: 45 min, Incubation temperature: 37° with HindIII for P316, P317, P318 and P319 ; 45 min, Incubation temperature: 60° with BstEII for P316, P317, P318 and P319 >br /> Preparation of gel:
1 g Agarose, 100 ml TAE (1%), 6 µl GELRED , at 120 Volt, running time: 45 minutes
Marker: GeneRuler ladder mix
Marker
Sample P250 10 /µl
Sample P314 10 /µl
Sample P315 10 /µl
Sample P316 10 /µl
Sample P317 10 /µl
Sample P318 10 /µl
Sample P319 10 /µl
Lane
1
3
4
5
7
8
9
10
Comment: The P314 and P315 looks like the control vector. The RepCap Vector_SDM_InsPvuII insertion didn´t work. The P316, P317, P318 and P318 was cut with the "wrong" enzymes. The same fragment size as the original vector appeared. Next time cut with EcoRI.
Midi-Prep of pSB1C3_VCK_Bla
Investigators: Chris W.
Comment: Midi-Preps of P 320, B257
The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
plasmid-no.
P320
concentration (ng/µl)
408
103. labday 28.08.2010
Mini-Prep of pSB1C3_NLS
Investigator: Stefan
comments: No glycerol stock and test digestion was performed because bacteria culture inocculated from glyerol stock (B233).
Mini-Prep following the standard protocol
P321 = pSB1C3_NLS (1)
c= 180,45 ng/µl
P322 = pSB1C3_NLS (2)
c= 187,26 ng/µl
COMMENT (Patrick): These mini-preps will be thrown away because B233 was used for inoculation due to a wrong entry in our plasmid excel sheet.  B233 this is NOT the sequenced clone ! We sequenced B234/P282. Therefore 2x10 ml DYT + Cm were inoculated with B234.
Cell culture
Harvest viral Stock and Transductiot of HT and A431 cells
P169: 100.000 cells
P169: 200.000 cells
P169: 300.000 cells
P169: 400.000 cells
P169: 500.000 cells
P169: 600.000 cells
P169: 700.000 cells
P262: 100.000 cells
P262: 200.000 cells
P262: 300.000 cells
P262: 400.000 cells
P262: 500.000 cells
P262: 600.000 cells
P262: 700.000 cells
200.000 cells per well
1
2
3
A
control, no cells
1
2
B
control, no virus
1
2
1
2
3
A
control, no cells
3
4
B
control, no virus
3
4
1
2
3
A
control, no cells
5
6
B
control, no virus
5
6
1
2
3
A
control, no cells
7
8
B
control, no virus
7
9
1
2
3
A
control, no cells
11
13
B
10
12
14
Seeding AAV293 for Transfection at 30.8
8 plates with 200.000 cells each were seeded
Picking clones from the ViralBricks
Investigator: Volker
Clones from all 14 transformations of the ViralBricks were picked. Because the probability that the ligation worked and that the synthesized oligos do not conatin mutations is quite bad I picked 8 to 10 clones from each ligation, resulting in approximately 150 clones at all. The clones were used to inoculate 5ml of a mixture of 60%LB-media nd 40%DYP-media because our stocks were not sufficient for this experiment.
104. labday 29.08.2010
P282/B234 mini-prep
Investigator: Patrick ... was performed according to the standard protocol.
Labelling:
P323, pSB1C3_NLS clone 2.1, 129 ng/µl
P324, pSB1C3_NLS clone 2.2, 141 ng/µl
Digestion of pSB1C3_NLS (P324)
Investigator: Patrick
The NLS was cut out with NgoMIV and EcoRI, therefore the expected size of the hoped-for fragment is 55 bp. About 30 ng DNA can be detected in a 1% gel. According to my calculation there will be maximal 80 ng insert.
55bp/2019bp = 0,027
0,027x140 ng/µl = 3,81 ng/µl
80 ng/3,81 ng/µl = 21 µl
Digestion: 21 µl pSB1C3_NLS (P324), 2,5 µl EcoRI, 2,5 µl NgoMIV, 3 µl Buffer 4, 1 µl H2O.
Digestion Time: 2 h 20 minutes.
The 2% agarose gel was loaded with 36 µl digesteion product including 6 µl 30% glycerol solution. I did not use loading dye because i to prevent that the dye overlay the mutual 55 bp insert.
The gelextraction was performed according to the the standard protocl. The resolution of the picture ist not sufficient to be able to see the band clearly as the gel was put onto the UV-light table a very thin band with the expected size could be seen.
DNA concentration measurement: 3,28 & 7,7 ng/µl (two measurements). Especially the second value is doubtable because according to my calculation above the insert conccentration cant be higher than 3,81 ng/µl. Labeled: NLS, put into 4°C room.
New digestion over night following a gelrun tomorrow: 23 µl pSB1C3_NLS (P324), 2 µl EcoRI, 2 µl NgoMIV, 3 µl Buffer 4.
Harvesting cultures from the ViralBricks
Investigator: Volker
All approximately 150 cultures were pelletted (twice 2 ml) and the supernatant thrown away. The pellets were freezed and the rest of the culture stored in the 4°C room. These pellets will be preped, test digested and sequenced subsequently until right clones are identified.
105. labday 30.08.2010
BioBrick assembly of pSB1C3_leftITR_pTERT & pSB1C3_ß-globin_YFP_hGH_rITR
Investigator: Achim
comment: final assembly of a vector containing all aav elements & the tert promoter
Digestion:
components:
pSB1C3_ß-globin_YFP_hGH_rITR(Vector): P294
pSB1C3_leftITR_pTERT (Insert): P256
DNA
2,4
8,4
BSA (10x)
2
2
Buffer 4 (10x)
2
2
Enzyme1
1 (EcoRI)
1 (EcoRI)
Enzyme2
1 (XbaI)
1 (SpeI)
H2O
11,6
5,6
Total volume
20
20
Preparation of gel:
0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 110 Volt, running time: 45 minutes
Expected sizes of constructs:
pSB1C3_ß-globin_YFP_hGH_rITR:  ~4000 bp
pSB1C3_leftITR_pTERT: 2896 bp, ~640 bp
The corresponding bands were cut out and Gel-Extraction was performed according to protocol.
concentrations measured via NanoDrop:
pSB1C3_ß-globin_YFP_hGH_rITR:  34.73 ng/µl
pSB1C3_leftITR_pTERT: 9.48 ng/µl
Ligation:
T4 ligation was performed according to protocol.
Volumes used:
vector: 2,9 µl
insert: 5,1 µl
Transformation:
bacterial strain used: XL1bB
antibiotic used for plate: chlorampenicol
Transformation was performed according to protocol.
Plate was prepared and put in 37°C room.
Continuation: Digestion of pSB1C3_NLS (P324)
Investigator: Patrick
The gelextraction was performed with QIAEX II Gel Extraction Kit which should also allow to extract fragments smaller than 70 bp (at least 40 bp). According to the QIAGEN Gel Extraction Kit the lenght of the DNA fragment should be about 70 bp. Unfortunately the DNA fragment was blurred so a quite large piece had to be cut out.
The extraction yielded 4,26 and 3,77 ng/µl (two measurements) but this sample also quite polluted. ... put into freezer
CD biobrick production
Investigator: Kira
NA samples were diluted 1:100
Ingredients
CD sample
5X Phusion HF buffer
10 µl
10 mM dNTP mix
1µl
forward primer: O158
2,5µl
reverse primer: O159
2,5 µl
DNA Template
0,5 µl
DMSO
0 µl
Phusion Polymerase
0,5 µl
H2O
33µl
Total volume
50 µl
PCR program:
Cycles
Temperature
Time
98°C
1
8x
98°C
15"
60°C
25"
72°C
60"
17x
98°C
15"
65°C
25"
72°C
60"
1x
72°C
5'
Hold 4°C
Digestion of plasmid backbone:
c (pSB1C3) = 151, 1 ng/ µl
Components
vector Volume/µL
DNA 1 µg
6,0 µl
BSA (100x)
0,2 µl
Buffer no. 4 (10x)
2,0 µl
Enzyme 1 XbaI
0,5 µl
Enzyme 2 AgeI HF
0,5 µl
H2O
10,8 µl
Total volume
20
incubation @ 37 C for approx. 2 h
Digestion of PCR product:
Components
PCR product Volume/µL
DNA
30,0 µl
BSA (100x)
0,4 µl
Buffer no. 4 (10x)
4,0 µl
Enzyme 1 XbaI
1,5 µl
Enzyme 2 AgeI HF
1,0 µl
H2O
3,1 µl
Total volume
40
incubation @ 37 C for approx. 2 h
1% agarose gel
Ligation
T4 ligase was used
1 ul T4 Buffer
1 ul T4 Ligase
9 ul (6,9 ul vector+ 1,1 ul insert) DNA-mix
incubation over-night @ 18C
Repetition of test digestion of pSB1C3_Rep40_RC_Insert and pSB1C3_Rep68_RC_Insert.
Investigator: Chris L.
P316 = pSB1C3_Rep40_ins clone 1
c=302,22 ng/µl
P317 = pSB1C3_Rep40_ins clone 2
c=274,37 ng/µl
P318 = pSB1C3_Rep68_ins clone 1
c=491,69 ng/µl
P319 = pSB1C3_Rep68_ins clone 2
c=498,71 ng/µl
Test digestion:
components
volume of P316/µl
volume of P317/µl
volume of P318/µl
volume of P319/µl
DNA
1
1
1
1
BSA (10x)
1
1
1
1
Buffer 4 (10x)
1
1
1
1
Enzyme XcmI
0,5
0,5
0,5
0,5
H2O
6,5
6,5
6,5
6,5
Total volume /µl
10
10
10
10
Incubation time: 1 h, Incubation temperature: 37° >br /> Preparation of gel:
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 120 Volt, running time: 45 minutes
Comment: Test digestion looks good. Clones 2 of each plasmid (P317 and P319) were sent in for sequencing with VR2 Primer.
Midi-Prep of pAAV_RC_insertparts clone1
Investigators: Chris W.
Comment: Midi-Preps of B208
The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
plasmid-no.
P325
p326
concentration (ng/µl)
2481,04
2050,66
FACS analysis, Seeding HT1080 and AAV293
Investigators: Kerstin, Adrian
FACS analysis of Transduction from 28.08.2010
Seeding HT1080 cells for Transduction (31.08.2010): 5x 6-well plates, 270.000 cells per well
Seeding AAV293 cells for Transfection (01.09.2010): 10 plates, 300.000 cells per well
Mini-Preps and test digestion of Loop insertion BioBricks
Investigator: Anna
Comment: Two clones of each ViralBrick were prepared and test digested.
Mini-Preps:
p331
p335
p339
Construct
pSB1C3_587_KO_BAP_clone3.1
pSB1C3_587_RGD_clone5.1
pSB1C3_453_His_clone7.1
DNA-Concentration / µl
65,08
94,29
73,49
p342
p345
Construct
pSB1C3_587_His_clone8.3
pSB1C3_587_KO_empty_clone10.1
DNA-Concentration / µl
69,05
66,73
Test digestion:
1% Agarose gel: 1 g Agarose, 100 ml TAE, 6 µl Gelred
Marker: Gene ruler ladder DNA mix, 5 µl
Used enzymes: SalI, BamHI (in general, SDS has to be added because of the strong binding of enzymes to DNA!)
Comment: Test digestion had no significant results. In addition, one sample was forgotten, it was not clear which one (description makes no sense). Test digestion has to be repeated.
Repetition of subcloning Cap-insert into pAAV_RC
Investigator: Stefan
Comments: There are still problems with BspMI. It was assumed that longer digestion could improve the results, therefore digestion with BspMI will be performed overnight.
P.S: Welcome back Hanna!
Digestion of vector and insert:
Insert: pMA_RepCap Vector_SDM_InsPvuII clone 1 (P211)
Vector: pAAV_RC_ins-rep clone1 (P250)
A two-step digestion was performed:
1st step
P211 / µl
P250 / µl
DNA
8,5
2
buffer 3
4,6
5
Enzyme BsiWI
1
1
H2O
32,4
38
total volume
46
46
2nd step
P211 / µl
P250 / µl
Mix obtained from step 1
46
46
Enzyme BspMI
4
4
total volume
50
50
Digestion will be performed overnight and continued tomorrow.
106. labday 31.08.2010
Cloning ZEGFR:1907 and 6xHis_middlelinker into pCerulean
Investigator: Hanna
Comment: In order to fuse the His-Tag to VP2, the His-Tag-Middlelinker construct will be cloned into the pCerulean plasmid (contains CMV promoter and SV40 terminator) today. In the next step, VP2 will be cloned behind the His-Tag-Middlelinker.
In addition to that the Affibody (ZEGFR:1907) will be also cloned into pCerulean (without linker). Also here, the next step will be fusing VP2 behind the this targeting molecule.
Practical Cloning:
plasmid:
Vector: name: pCerulean; number: P273
Insert: name: pSB1C3_6xHis_middlelinker (P310) + pSB1C3_Zegfr:1907 (because P267 was empty, P285 was used! See lab day 23.08.!)
new vector name: pCerulean_6xHis_middlelinker + pCerulean_Zegfr:1907
buffer used: 4; Restriction-enzymes used: XbaI and PstI
Comments: Because the P267 tube was empty (!), P285 was used which was prepared by Jessy - I don't know whether this is a trafo of the glycerol stock? Sequenced?
Digestion
components
volume of pCerulean/µl
volume of pSB1C3_6xHis_middlelinker/µl
volume of pSB1C3_ZEGFR:1907
DNA
3.6
16.2
11.5
BSA (10x)
3
2.5
2
Buffer 4 (10x)
3
2.5
2
Enzyme 1 XbaI
1.5
1
1
Enzyme 2 PstI
1.5
1
1
H2O
17.4
1.8
2.5
Total volume
30
25
20
Incubation: 1.5 h
Agarose-Gel:
0.8 g Agarose, 55 ml TAE (1.45 %), 3 µL GELRED, at 115 Volt, running time: 35 minutes
Sample
Sample/µl]
Loading dye (6x)/µl
Expected size 1 (Geneious)
P285
20 µl
4 µl
210 bp
P273
30 µl
6 µl
~ 3900 bp
P310
25 µl
5 µl
86 bp
Marker: GeneRuler ladder mix
Marker /µL
Sample P285 /µl
Sample P273 /µl
Sample P310 /µl
Lane
5.5
24
25
25
Gel extraction
Gel measurement:
Sample
Weight
Volume
Concentration
P285
240 mg
20 µL
5.4 ng/µL
P273
130 mg
20 µL
14.5 ng/µL
P310
220 mg
20 µL
2.4 ng/µL
Comment: The DNA concentrations were very bad, which could be due to the fact, that the 6xHis_middlelinker construct's size is just 86 bp and the affibody's size is just 210 bp...
Ligation
1. pCerulean_6xHis_middlelinker
6xHis_middlelinker
pCerulean
Volume/µl
2.57
6.43
2. pCerulean_Zegfr:1907
Zegfr:1907
pCerulean
Volume/µl
2.72
6.28
Trafo
Trafo was performed following the standard protocol. Cells: XL1b (60 µL), DNA amount per sample: 3 µL; cells were plated onto Kanamycin-plates.
PCR of SV 40 terminator
Investigator: Bea
Comment: In order to obtain the SV40 poly adenylation site and clone it into the iGEM standard plasmid pSB1C3 a PCR needs to be performed with the pEGFP-C1 which contains the SV40 terminator.
Plasmid used for PCR: pEGFP-C1 (P230)
1:1000 dilution of P230 has been prepared
Plasmid used as vector: pSB1C3_CFP1 (P51.2)
PCR (was performed following the standard protocol)
Ingredients
Volume / µl
5X Phusion HF buffer
10
10 mM dNTP mix
1
forward primer: O160
2,5
reverse primer: O161
2,5
DNA Template
3
Phusion Polymerase
0,5
H2O
30,5
Total volume
50
PCR program:
PCR Program
temperature/ °C
Time
1
98
1min
2
98
15s
8x
59
25s
3
72
7s
4
98
15s
17x
67
25s
5
72
7s
6x
72
5min
Hold
4
After the PCR was performed, the sample was loaded on a 1.5% agarose gel and run for 30 minutes. As it can be seen in the picture below the expected size of the PCR product (260bp) can be confirmed. The lower band corresponds to the added primers and does not seem to be an additional band.
For cloning the obtained fragment into the pSB1C3 standard vector, the construct need to be digested with the same restriction enzymes as the PCR product will be cut with. In this case the primers were designed with XbaI overhang for the prefix, and with SpeI-NotI-PstI overhang for the suffix. Therefore the vector will be cut with XbaI and PstI. The sample were incubated at 37 C for approx. 1.5 h
Plasmid used: pSB1C3_CFP (P51.2) c=150ng/µL
Add BSA
XbaI and PstI were used
Digestion of vector:
Components
vpSB1C3_CFP/µL
DNA
8
BSA (100x)
2
Buffer no. 4 (10x)
2
Enzyme 1 XbaI
1
Enzyme 2 AgeI HF
1
H2O
6
Total volume
20
Incubation of the sample at 37°C
Incubation for 120 minutes
After the PCR has been cut out of the gel and purified with the Qiagen Gel Extraction Kit obtaining a concentration of 45 ng/µl measured with the Nanodrop, the complete PCR product volume of 29 µL was digested with XbaI and PstI for 2 hours.
Digestion of the PCR product SV40
Components
vPCR product/µL
DNA
29,0 µl
BSA (10x)
4,0 µl
Buffer no. 4 (10x)
4,0 µl
Enzyme 1 XbaI
1,0 µl
Enzyme 2 AgeI HF
1,0 µl
H2O
1,0 µl
Total volume
40
Incubation of the sample at 37°C
Incubation for 120 minutes
The digested PCR product SV40 was purifed with the PC Purification Kit and concentration was measured.
cSV40= 31,5 ng/µl
cpSB1C3= 7,74 ng/µL
For T4 ligation of the two fragments, the needed volumes of insert (SV40) and vector (pSB1C3) were calculated with the labtools program. The total volume DNA (vector/insert) mix will be 8 µL.
vSV40= 0,67 µL
vpSB1C3= 7,33 µL
vT4 buffer= 1 µL
vt4 ligase= 1 µL
The ligation reaction was carried out for 40 minutes at room temperature. The ligated plasmid was transformed into XL1-Blue cells and plated on agar plates containing the appropriate amount of chloramphenicol.
Cloning of NLS into pCerulean_VP1up
Investigator: Anissa
Comment: NLS was already 2 times cut of the pSB1C3_NLS and purified by Patrick (NLS 1: of the 29.08.10, NLS 2: of the 30.08.10) . Now, pCerulean_VP1up will be digested and NLS will be ligated into it.
Digestion of pCerulean_VP1up (p303)
components
Vector
DNA
2,6
BSA (10x)
1,5
Buffer 4 (10x)
1,5
Enzyme PstI
1
Enzyme AgeI
1
H2O
7,4
Total volume
15
Gel:
A 1% gel was made and the sample was running 30 minutes.
Afterwards a gelextraction according the qiagen-protocol was performed.
Ligation:
components
Volume /µl
concentration /ng/µl
NLS 2 insert of 30.08.10
1,51
3
NLS 1 insert of 29.08.10
1,51
3
pCerulean_VP1up
6,49
29
Transfomation: was performed into XL1 blue on a kanamycin plate according the standard protocol.
Cloning CFP_middlelinker (from pSB1C3_CFP_middlelinker, P276) into pCerulean (P273)
Investigator: Patrick
P276 digestion (207 ng/µl): 14 µl DNA sample, 2 µl Buffer 4, 2 µl BSA, 1 µl XbaI, 1 µl PstI-HF.
P273 digestion (419 ng/µl): 4 µl DNA sample, 1 µl Buffer 4, 1 µl BSA, 1 µl XbaI, 1 µl PstI-HF.
Digestion time: 1 h 45 min.
Expected size of the fragments:
P276: 786 & 2058 bp
P273: 756 & 3938 bp
My dear friend, where is your initial comment? Nice gel :) Yours sincerely, Hanna :)
see: http://www.molbiotech.uni-freiburg.de/iGEM/wiki2010/index.php/August_2010#Strategy_for_VP2_N-terminal_fusion_and_VP1_insertion_of_targeting_moelcules
Haha, good idea! :)
Gelextraction: Obviously there was something wrong with the gel or the sample. The marked bands were cut out and the gelextraction was performed acording to he standard protocol, yielding the following concentrations:
P276 up: 12,6 ng/µl
P276 down: 9,9 ng/µl
P273: 37,7 ng/µl
All of them were polluted.
There were two bands at a level where no bands should be. P 276 looks also strange because accidentally 6,5 µl 1kb GeneRuler were pipetted into the sample. The expected bands are all blurred. Even the expected 756 bp fragment of P273 is not at the level it should be. To ensure i dont throw away the insert i cut out both bands from P276. Now tomorrow there will be a ligation and a transformation with two samples instead of one
Sequencing results of pSB1C3_6xHis_middlelinker and pSB1C3_6xHis_mVenus
Investigator: Hanna
1st Comment: Sequencing of pSB1C3_6xHis_middlelinker looked good:
2nd Comment: Sequencing of mVenus looked good. The His-Tag delivered 2 additional bp and the suffix 1 transition (see picture). This could be due to bad sequencing qualities at this region... but needs to be verified!
Contiuation: Repetition of subcloning Cap-insert into pAAV_RC
Investigator: Stefan
Comments: Overnight digestion does not look good, therefore a new approach will be performed today using different concentrations of spermidine (0,5mM, 2,5mM and 10mM) because in Oller et al (Biochemistry, 1991) it was suggested that this improves digestion with BspMI.
A two-step digestion was performed:
1st step
P211 / µl
P250 / µl
DNA
8,5
2
buffer 3
4,6
5
Enzyme BsiWI
1
1
H2O
32,4
38
total volume
46
46
Incubation: 70 minutes; 55°C
2nd step
P211 / µl
P250 / µl
Mix obtained from step 1
46
46
Enzyme BspMI
4
4
total volume
50
50
Incubation: 2,5h ; 37°C
Agarosegel
0.4 g Agarose, 50 ml TAE (0,8 %), 3 µL GELRED, at 115 Volt
Gel extraction
Gel-Ex was performed according to protocol.
NanoDrop:
pMA_RepCap-insert: c= 7,42 ng/µl
pAAV_RC_ins-rep: c=12,9 ng/µl
T4 Ligation
Volume insert: 3,43 µl
Volume vector: 4,57 µl
Ligation incubated for 50 minutes at room temperature.
Transformation Trafo was preformed according to standard protocol using XL1blue cells on plates containing Ampicilin.
2nd Repetition of test digestion of Loop insertion BioBricks
Investigator: Achim, Anna
Comment: First test digestion didn't work (see 31.08.10), this time a new approach was done with NotI.
Test digestion:
components
Volume for each sample /µl
DNA
10
BSA (10x)
1,5
Buffer 4 (10x)
1,5
Enzyme NotI
0,5
H2O
1,5
Total volume /µl
15
Sample-No.
1+2
3+4
5+6
7+8
9+10
11+12
13+14
15+16
17+18
19+20
21-23
24+25
26+27
28+29
Components
453 BAP
587 BAP
587 KO BAP
453 RGD
587 RGD
587 KO RGD
453 HIS
587 HIS
587 KO HIS
587 KO EMPTY
453 Z34C
587 Z34C
587 KO Z34C
587 KO Z34C SPACER
Oligo 1
O124: 10
O126: 10
O128: 10
O130: 10
O132: 10
O134: 10
O135: 10
O137: 10
O139: 10
O141: 10
O143: 10
O145: 10
O147: 10
O149: 10
Oligo 2
O125: 10
O127: 10
O129: 10
O131: 10
O133: 10
O151: 10
O136: 10
O138: 10
O140: 10
O142: 10
O144: 10
O146: 10
O148: 10
O150: 10
30: pSb1C3_Bla (cut version) 31: pSb1C3_Bla (uncut)
Incubation time: 1 h, Incubation temperature: 37°
Preparation of gel:
1 g Agarose, 100 ml TAE (1%), 6 µl GELRED , at 115 Volt, running time: 45 minutes
Expected fragment sizes /bp:
pSB1C3: 2051 bp Bla: 905
BAP: ~150
RGD: ~ 130
Z34C: ~ 200
His: ~ 125
The following clones were sent for sequencing: 2,3,5,7,9,11,13,16,17,19 and 25.
Comment: Clones (see above) were sent for sequencing with VR2 Primer. In addition, 8 mL DYT medium was inoculated with each of the clones for preparation of glycerol stocks. For sequencing results go to labday 01.09.10.
Midi-Prep of pAAV_RC & pHelper
Investigators: Chris W.
Comment: Midi-Preps of pHelper=P356 and pAAV_RC=P357
The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
plasmid-no.
P356
P357
concentration (ng/µl)
1068,0
1274.80
Transformation of Biobrick CD
Investigator: Kira
Transformation was performed according to the standard protocol.
=> Go to Labjournal September part 1 (labday 107 - 123)
Retrieved from " http://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/August2 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
