<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Bioware_Project_Project_Description"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg_Bioware/Project/Project_Description">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg_Bioware/Project/Project_Description&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_Bioware/Project/Project_Description&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_Bioware/Project/Project_Description&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg_Bioware/Project/Project_Description" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg Bioware/Project/Project Description</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="menu"><UL id="menu"><LI><DL><DT class="menu_button_left"><A href="https://2010.igem.org/Team:Freiburg_Bioware">Home</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Team_2010">Team</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Team_2010">Team 2010</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Collaboration">Collaboration</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Cuckoo_Clock">Cuckoo Clock Competition</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Photo_Gallery">Photo Gallery</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Statistics">Statistics</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Contribution">Contribution</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#highlights">Project</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results">Summary</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#highlights">Highlights</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Project_Description">Introduction</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results">Results</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#modularization">   Modularization</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#targeting">   Targeting</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#arming">   Arming</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#superconstruct">   Tumor Killing</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling">   Modeling</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Methods">Methods</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Virus_Construction_Kit">Virus Construction Kit Manual</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks">BioBricks</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks">Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/capsid_coding">Capsid Coding Parts</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/vector_plasmid">Vector Plasmid Parts</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Notebook">Notebook</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal">Labjournal</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Materials">Cloning Material</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Methods">Methods</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Meetings">Meetings</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Statistics">Statistics</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Sponsors">Sponsors</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Safety">Safety</A></DT></DL></LI><LI><DL><DT class="menu_button_right"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling">Modeling</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling">Modeling Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Virus_Production">Virus Production</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Virus_Infection">Virus Infection</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Structure_Modeling">Structure Modeling</A></DD></DL></LI></UL></DIV><DIV class="pageContent"><H1>Introduction</H1><H2>Contents</H2><P class="MsoToc1"><A href="#_Toc275994657"><SPAN lang="DE">Overview</SPAN><SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">.. </SPAN><SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">1</SPAN></A></P><P class="MsoToc2"><A href="#_Toc275994658"><SPAN lang="DE">The
Experimental System</SPAN><SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">.. </SPAN><SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">1</SPAN></A></P><P class="MsoToc2"><A href="#_Toc275994659"><SPAN lang="DE">Layers of
specificity</SPAN><SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">. </SPAN><SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">2</SPAN></A></P><P class="MsoToc1"><A href="#_Toc275994660">Introduction to
Adeno-Associated
Virus Serotype 2<SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">. </SPAN><SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">4</SPAN></A></P><P class="MsoToc2"><A href="#_Toc275994661">Biology of the AAV-2<SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">. </SPAN><SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">4</SPAN></A></P><P class="MsoToc3"><A href="#_Toc275994662">Genomic organization<SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">. </SPAN><SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">4</SPAN></A></P><P class="MsoToc3"><A href="#_Toc275994663">Replication<SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">. </SPAN><SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">5</SPAN></A></P><P class="MsoToc3"><A href="#_Toc275994664">Integration<SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">. </SPAN><SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">11</SPAN></A></P><P class="MsoToc3"><A href="#_Toc275994665">Rescue<SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">12</SPAN></A></P><P class="MsoToc3"><A href="#_Toc275994666">Rep proteins<SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">. </SPAN><SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">12</SPAN></A></P><P class="MsoToc3"><A href="#_Toc275994667">VP proteins<SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">. </SPAN><SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">16</SPAN></A></P><P class="MsoToc3"><A href="#_Toc275994668">Trafficking<SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">23</SPAN></A></P><P class="MsoToc2"><A href="#_Toc275994669">Helper Genes<SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">. </SPAN><SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">27</SPAN></A></P><P class="MsoToc2"><A href="#_Toc275994670">Recombinant Viruses and
Mosaic
Viruses<SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">. </SPAN><SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">29</SPAN></A></P><P class="MsoToc2"><A href="#_Toc275994671"><SPAN lang="DE">Gene Therapy</SPAN><SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">. </SPAN><SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">30</SPAN></A></P><P class="MsoToc2"><A href="#_Toc275994672">Immune Response<SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">. </SPAN><SPAN style="color: windowtext; display: none; text-decoration: none;" lang="DE">31</SPAN></A></P><P class="MsoNormal"><SPAN lang="DE"> </SPAN></P><DIV style="border-style: none none solid; border-color: -moz-use-text-color -moz-use-text-color windowtext; border-width: medium medium 1pt; padding: 0cm 0cm 1pt;"><H1 style="margin-left: 0cm; text-indent: 0cm;"><SPAN lang="DE"> </SPAN></H1><H1 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc275994657"><SPAN lang="DE">Overview</SPAN></A></H1></DIV><H2 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc275994041"><SPAN lang="DE">The Experimental System</SPAN></A></H2><P class="MsoNormal">Therapy using viral vectors is an promising
approac. In an
first step the plasmids of the AAV-2 Helper-free System were
genetically
modifyed by converting it into BioBricks and inserting of targeting
molecules
into the constructs. These plasmids were then used to transfect the
producer
cell line AAV-293. After an incubation of three days the viral vectors
were
harvested and used to transduce different target cells. The succesful
transduction can then for example be measured by detecting the
fluorescence of
fluorescent proteins in the target cells</P><P class="MsoNormal">The majority of the modifications that were
introduced into
the viral vector aimed to allow differential targeting of tumor cell
over
healthy off-target cells.</P><H2 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc275994042"><SPAN lang="DE">Layers of specificity</SPAN></A></H2><P class="MsoNormal">Employment of viral vectors for means of therapy
is idea in
the context of personalized medicie that gets more and more interest.
In such
applications the reduction of side effects and the safety of the
patient in
general is of the highest priority.</P><P class="MsoNormal">In order to satisfy this requirement we designed
our Therapy
Vector with several layers of Specificity:</P><P class="MsoNormal">The targeting of the viral vector towards the
desired target
cell (e.g. tumor cells) is the basic idea behind the emplyment of viral
vectors
for therapeutical means. There for the natural tropismn has to be
knocked down
and a desired tropism has to be introduced that allows differential
targeting
of pathological but not of off-target cells. To fulfill this mission
our Virus
Construction Kit offers you different solutions.</P><P class="MsoNormal">Off-target cells that were transduced by mistake
can be
preserved from an undesired therapy effect when the therapeutic gene is
controley by a tissue specific promoter. For this mean a promoter has
to be
used that is as specific for the pathological tissue as possible. We
included
the human telomerase promoter (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K404106">phTERT</A>)
which is often activated in tumor cells and is there for able to allow
differential experssion of a therapeutic geneproduct in pathological
cells.</P><P class="MsoNormal">For reasons of safety Therapeutic vector do not
directly
trigger appoptosis in the successfully targeted cells. To include one
further
layer of specificity and safety we decided to arm our therapy vector
with
different prodrug convertases. Neither the single application of the
harmless
prodrug nor the single expression of the convertase has a noteworthy
effect of
the transduced cell. Only in cells that express the prodrug convertase
and have
a sufficient cytoplasmatic concentration of the belonging prodrug
apoptosis is
triggered. This dependency of the therapy on a prodrug can be employed
to
protect tissues or other persons that could come in contact with the
therapeutical vector. This aspect was specially inportant for the
development
of a viral vector that is able to infect humans in the context of a
undergraduate project for the iGEM competition. Therefor this approach
gained
our preference over other possibly equivalent arming possibilities
described in
the tumor therapy with viral vectors.</P><P class="MsoNormal"> </P><DIV class="WordSection2"><DIV style="border-style: none none solid; border-color: -moz-use-text-color -moz-use-text-color windowtext; border-width: medium medium 1pt; padding: 0cm 0cm 1pt;"><H1 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc275994043">Introduction to </A>Adeno-Associated Virus
Serotype 2</H1></DIV><H2 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc274911362">Biology of the AAV-2</A></H2><H3 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc275994045">Genomic organization</A></H3><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse; margin-left: 4.8pt; margin-right: 4.8pt;" align="left" border="1" cellpadding="0" cellspacing="0"><TBODY><TR style="height: 191.45pt;"><TD style="border: 1pt solid windowtext; padding: 0cm 5.4pt; width: 426.05pt; height: 191.45pt;" valign="top" width="568"><P class="MsoCaption" style="margin-left: 17.85pt; text-indent: 0cm;"><A name="_Ref275946742">Figure
</A>1: Genomic organization of the wt-AAV-2. The inverted
terminal repeats (ITRs) flank the two open reading frames (ORFs). The
four-nonstructural proteins encoded from the <I>rep</I> gene are
driven by the p5 and p19 promoters, whereas the structural Cap proteins
are regulated by the p40 promoter. Additionally, the Assembly
Activating Protein (AAP) was found recently within the <I>cap</I> gene.</P></TD></TR></TBODY></TABLE><P class="MsoNormal">The Adeno-associated virus serotype-2 (AAV-2)
genome is a linear,
single-stranded (ss) 4675 bp DNA virus. Due to its small size, gene
genomic
organization is condensed and gene regulation is complex. The viral
nucleotide
sequence consist of two open reading frames (ORFs) coding for Rep- and
Cap
proteins and are flanked on either side by identical inverted terminal
repeat
(ITR) structures which are palindromic and form hairpin structures
(Srivastava et al. 1983). </P><P class="MsoNormal"> </P><P class="MsoNormal">The ITRs serve as primers for the host cells’ DNA
polymerase,
which converts the single-stranded virus genome into double-stranded
DNA (ds
DNA) as a part of the viruses’ replicative cycle. They also play
important
roles in viral genome integration into and rescue from the hosts
genome, the
formation of concatamers in the host cell nucleus and encapsidation of
the
viral genome into preforemd capsids (Berns 1990). Due to these
essential functions, the ITR structures cannot be deleted from
a viral vector and need to be delivered in <I>cis.</I></P><H4 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc275994046">Organization
of the Inverted Terminal </A>Repeat Structure</H4><P class="MsoNormal"><SPAN style="color: black;">The inverted terminal
repeat
structures can be subdivided into several palindromic motives: A and A’
form a
stem loop which encases B and B’ as well as C and C’. Those motives
form both
arms of the T-shaped structure. The functional motives on the ITR are
two
regions that bind Rep 68/78, called Rep-binding elements (RBE on the
stem and
RBE’ on the B arm) and the terminal resolution site (trs) in which the
rep
proteins introduce single-stranded nicks. The 3’ OH end of the A motive
acts as
a primer for DNA replication </SPAN><SPAN style="color: black;">(Im
&amp; Muzyczka 1990)</SPAN><SPAN style="color: black;">(Lusby et al. 1980)</SPAN><SPAN style="color: black;">.</SPAN></P><DIV align="center"><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse; margin-left: -2.25pt; margin-right: -2.25pt;" align="left" border="0" cellpadding="0" cellspacing="0"><TBODY><TR style="height: 293.05pt;"><TD style="padding: 0cm 5.4pt; width: 390.3pt; height: 293.05pt;" valign="top" width="520"><P class="MsoCaption" style="margin-left: 17.85pt; text-indent: 0cm;">Figure 2: Organization
of the ITRs, which are the only <I>cis</I>-required element in viral
genome integration and replication. </P></TD></TR></TBODY></TABLE></DIV><P class="MsoNormal"> </P><H3 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc275994047">Replication</A></H3><P class="MsoNormal">The 3’ OH end of the viral DNA folds onto itself
as part of
the inverted terminal repeat (ITR) structure and thus serves as a
primer for
elongation by the host cell’s DNA polymerase. The polymerases strand
displacement activity unfolds the opposite ITR structure and elongation
continues until the 5’ template end is reached <SPAN style="color: black;">(Lusby et
al. 1980)</SPAN>.</P><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse; margin-left: -2.25pt; margin-right: -2.25pt;" align="left" border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="border: 1pt solid windowtext; padding: 0cm 5.4pt; width: 460.5pt;" valign="top" width="614"><P class="MsoCaption" style="margin-left: 17.85pt; text-indent: 0cm;">Figure 3: Figure 3:
Schematic overview of AAV-2 replication.</P></TD></TR></TBODY></TABLE><P class="MsoNormal"> </P><P class="MsoNormal">The remaining hairpin structure that served as the
origin of
replication then acts as a target for the Rep 68/78 protein: It binds
to the
Rep-binding site (RBS) and unwinds the double-stranded DNA in a way
that the
terminal resolution site (trs) is being displayed in a single-stranded
form on
a stem loop. This enables the endonuclease catalytic domain of the Rep
protein
to introduce a nick of the parental strand at this site, which in turn
serves
as a new primer for DNA polymerase. The polymerase resolves the hairpin
structure
through strand displacement and copies the remaining end of the
parental strand<SPAN style="color: black;">(Im &amp; Muzyczka 1990)</SPAN> .</P><P class="MsoNormal">Sometimes, nicking does not occur after
polymerases have
partially copied the virus DNA. In this case, the newly synthesized 3’
end acts
as a primer and the host cell’s DNA polymerase copies the whole
sequence once
again, displacing the ITR strands in the middle of the sequence. This
leads to
a dsDNA containing the whole virus genome twice, called a duplex dimer
(DD).
Those dimers can be resolved back to duplex monomers (DM) by the Rep
proteins</P><P class="MsoNormal">After replication, the dsDNA separates again
forming new
ssDNA in (+) and (-) polarity with hairpin structures at its ends. The
Rep
40/52 proteins are involved in this process. Newly synthesized copies
are
either encapsidated into virus capsids or replicated again (Gonçalves,
2005a). Double-stranded genomes are formed as well through annealing of
(+) - and
(-) single strands. Both mechanisms occur during infection and
contribute to
transgene expression (Schultz &amp; Chamberlain 2008). </P><P class="MsoNormal">If the double stranded virus DNA exists in an
episomal form
inside the nucleus, it tends to form linear as well as circular
concatamers,
which are formed by ligation of duplex monomers(Schultz &amp;
Chamberlain 2008). </P><H4 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc275994048">Viral
promoters</A></H4><P class="MsoNormal">Three viral promoters are coordinating gene
expression in
the wildtype AAV-2. Each promoter regulates different open reading
frames
(ORFs) of regulatory proteins (p5 and p19 promoter) and structural
proteins
(p40 promoter). A general overview is provided in<B>Error! Reference
source not
found.</B>. p5 and p19 promoters are repressed in absence of helper
proteins
provided by Ad or HSV whereas transactivation of p5 and p19 occurs in
presence
of helper viruses. Furthermore, the larger Rep proteins activate the
p40
promoter. Since overexpression of Rep78 leads to cell cycle arrest,
high levels
of Rep78/68 lead to repression of the p5 promoter. </P><DIV align="center"><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse; margin-left: -2.25pt; margin-right: -2.25pt;" align="left" border="0" cellpadding="0" cellspacing="0"><TBODY><TR style="height: 373.8pt;"><TD style="padding: 0cm 5.4pt; width: 284.5pt; height: 373.8pt;" valign="top" width="379"><P class="MsoCaption" style="margin-left: 17.85pt; text-indent: 0cm;"><A name="_Ref275633331">Figure </A>4: Regulation of the viral promoters
located within the t AAV-2 genome. In the absence of helper viruses,
gene expression is suppressed, whereas activation of p5 and p19 occurs
in the presence of helper proteins by interacting of Rep proteins with
cellular and helper proteins.</P></TD></TR></TBODY></TABLE></DIV><P class="MsoNormal" style="text-align: center;" align="center"> </P><H5 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Ref275645430">p5 promoter</A></H5><P class="MsoNormal">The p5 promoter, located downstream of the <I>rep</I>
and <I>cap</I>
ORF (Figure 5: The p5 promoter of the wtAAV-2 is located upstream of
the rep
and cap ORF and contains several elements, which interact with Rep and
endogenous proteins.)of the wtAAV-2, regulates gene expression of the
two larger
non-structural proteins Rep 78 and Rep 68 that are essential in genome
replication and viral genome integration into several hotspots of the
human
chromosome. </P><P class="MsoNormal">Several binding elements for cellular and viral
proteins
involved in regulation can be found in the p5 promoter (Figure 5)
therefore
playing an important role in gene transcription, integration and
replication,
dependent on the presence or absence of helper viruses such as
adenovirus (Ad)
or herpes simplex virus (HSV) (Murphy et al. 2007). Besides regulation
of gene expression, the p5 integration efficient
element (p5IEE) containing the rep binding element (RBE) and a terminal
resolution site (trs) is responsible for mediating site specific
integration
into the human genome (Philpott et al. 2002). </P><DIV align="center"><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse; margin-left: -2.25pt; margin-right: -2.25pt;" align="left" border="0" cellpadding="0" cellspacing="0"><TBODY><TR style="height: 203.8pt;"><TD style="padding: 0cm 5.4pt; width: 433.85pt; height: 203.8pt;" valign="top" width="578"><P class="MsoCaption" style="margin-left: 17.85pt; text-indent: 0cm;"><A name="_Ref275633992">Figure </A>5: The p5 promoter of the wtAAV-2 is
located upstream of the rep and cap ORF and contains several elements,
which interact with Rep and endogenous proteins.</P></TD></TR></TBODY></TABLE></DIV><P class="MsoNormal"><SPAN lang="EN-GB"> </SPAN></P><P class="MsoNormal">Containing two consensus sequences for binding
immediate
early E1A gene product from adenoviruses (Chang et al. 1989), p5
promoter is
transactivated in the presence of helper viruses whereas suppression
occurs in
absence of adenoviral proteins by low levels of Rep proteins (Beaton et
al. 1989). Regulating of Rep78/68 by its negative feedback loop is
critical
since overexpression leads to cell cycle arrest in the S-phase (Berthet
et al. 2005) and suppression of cellular promoters (Jing et al. 2001).</P><H5 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc275994050">p5
TATA-less promoter</A></H5><P class="MsoNormal">In contrast to the natural location of the p5
promoter, the
iGEM team Freiburg 2010 provides the RepCap plasmid with a relocated p5
promoter
downstream of the <I>RepCap</I> genes (Figure 6). Additionally the p5
promoter
lacks the TATA box element (AVIGEN 1997)<B>. </B>Those modifications
result in an attenuated expression of the larger
Rep proteins therefore leading to normal transcription of the Rep
proteins
driven by p19 promoter and enhanced expression of the Cap proteins,
which are
under the control of the p40 promoter. Additionally, removing the p5
promoter
downstream of the <I>RepCap</I> genes and deletion of the TATA box
eliminates
contamination with wtAAVs. Hence, alteration of the p5 promoter is
useful for
enhanced production of recombinant viral particles attenuating
repression of
Rep78/68 and improving gene transcription of the capsid proteins and
Rep
proteins involved in genome packaging. </P><DIV align="center"><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse; margin-left: -2.25pt; margin-right: -2.25pt;" align="left" border="0" cellpadding="0" cellspacing="0"><TBODY><TR style="height: 214.9pt;"><TD style="padding: 0cm 5.4pt; width: 236.7pt; height: 214.9pt;" valign="top" width="316"><P class="MsoCaption" style="margin-left: 17.85pt; text-indent: 0cm;"><A name="_Ref275634090">Figure
</A>6: p5 TATA-less promoter is located downstream of the rep and
cap ORF.</P></TD></TR></TBODY></TABLE></DIV><P class="MsoNormal"> </P><H5 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Ref275628172">p19</A> promoter</H5><P class="MsoNormal">p19 promoter drives gene expression of the smaller
Rep proteins
Rep52 and Rep40. In absence of a helper virus infection the promoter is
inactive by repression of all four Rep proteins, but is transactivated
by
interaction of both the Sp1 site and Rep protein Rep78/68 bound to the
Rep
binding element (RBE) (Lackner &amp; Muzyczka 2002). By forming a DNA
loop (Pereira &amp; Muzyczka 1997) and bringing the two promoters in
proximal distance ()
additional cellular factors bound to p5 promoter interact with the p19
promoter
leading to transcriptional activation of Rep52/40 (Lackner &amp;
Muzyczka 2002). </P><DIV align="center"><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse; margin-left: -2.25pt; margin-right: -2.25pt;" align="left" border="0" cellpadding="0" cellspacing="0"><TBODY><TR style="height: 215.65pt;"><TD style="padding: 0cm 5.4pt; width: 235.95pt; height: 215.65pt;" valign="top" width="315"><P class="MsoCaption" style="margin-left: 17.85pt; text-indent: 0cm;">Figure 7: Forming of
the DNA loop brings the p5 rep binding element in proximal distance to
the Sp1 site foud in the p19 promoter.</P></TD></TR></TBODY></TABLE></DIV><P class="MsoNormal"> </P><H5 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc275994052">p40
promoter</A></H5><P class="MsoNormal"><SPAN lang="EN-GB">The P40 promoter is derived
from the
adeno-associated virus serotype 2 (AAV2) genome, where it is located at
40 map
units. It regulates the transcription of the capsid proteins VP1, VP2
and VP3</SPAN><SPAN lang="EN-GB">(Labow, Hermonat, &amp; Berns, 1986</SPAN><SPAN lang="EN-GB">; </SPAN><SPAN lang="EN-GB">Cassinotti, Weitzand, &amp;
Tratschin, 1988)</SPAN><SPAN lang="EN-GB">.
</SPAN></P><P class="MsoNormal"><SPAN lang="EN-GB">Several sequence regions have
been
identified to be important for maximal promoter activity: Two Sp1
sites, which
are located 250 (Sp1-50) and 270 (GGT-70) base pairs upstream of the
transcriptional start point and to which Sp1 or Sp1-like proteins bind </SPAN><SPAN lang="EN-GB">(Pereira &amp; Muzyczka 1997)</SPAN><SPAN lang="EN-GB">. </SPAN></P><P class="MsoNormal"><SPAN lang="EN-GB">Referring to the virus genome,
p40 can also
be induced through transactivation by the Rep proteins. The Sp1-50,
together
with the CArG-140 site of the P19 promoter, are the main elements
involved in
this process. The Rep proteins, which bind to the Rep binding element
in the
terminal repeat or the P5 promoter, can induce P19 or P40 by
interaction with
their bound Sp1 proteins thereby forming a DNA-loop </SPAN><SPAN lang="EN-GB">(Pereira &amp; Muzyczka 1997)</SPAN><SPAN lang="EN-GB">.
In addition
to that, the TATA box, located at 230, is also required for P40
activity.
Furthermore the ATF-80 and the AP1-40 elements are also important for
maximal
promoter induction </SPAN><SPAN lang="EN-GB">(Pereira &amp; Muzyczka
1997)</SPAN><SPAN lang="EN-GB">.</SPAN></P><H3 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc274911366">Integration</A></H3><P class="MsoNormal"><A name="_Toc274911367">The AAV-2 is the only
known
mammalian virus that integrates into a specific location in the human
genome in
the presence of Rep78/68: Chromosome 19, 19q13.3-qter. The site of
integration
was termed AAVS1. </A></P><P class="MsoNormal">The mechanism of Rep-mediated integration into
AAVS1 is not
yet completely understood and seems to be imprecise and variable.
Deletions or
insertions often occur in the integration process(Schultz &amp;
Chamberlain
2008). Linden et al. (1996) proposed a mechanism that is consistent
with the
observed patterns in which AAV exists in an integrated form:</P><P class="MsoNormal">AAVS1 bears a Rep-binding site (RBS) which is
similar to the
RBE in the virus genomes ITR. The Rep proteins are able to
simultaneously bind
to AAVS1 and the viral RBE, thereby bringing both strands into close
proximity
towards each other. After binding to the AAVS1 site, Rep acts as an
endonuclease, the same way it does when binding to the AAV ITR,
introducing a
single strand-nick between two thymidine residues close to the binding
site. This
produces a free 3’-OH end which acts as a primer for the host cells’
DNA
polymerase. After replicating the displaced strand, the polymerase
switches
templates and replicates the AAV DNA, thereby linking AAVS1 and AAV
together.
Prior to integration, the AAV genome often exists in circular and/or
concatameric form, resulting in multiple consecutive AAV copies in the
host
genome. Another explanation for this phenomenon could be a circularized
AAV
monomer that is being replicated several times in a rolling-circle
manner
before being integrated into the host genome. </P><P class="MsoNormal">Another template switch back to the AAVS1 sequence
creates a
second link between virus and host. This integration mechanism leaves
single-stranded
gaps that need to be repaired by cellular enzymes before integration is
complete. Since successful integration of AAV depends on these cellular
repair
mechanisms, integration happens more frequently in dividing cells, in
which
repair functions are more active.</P><P class="MsoNormal">Recently, a sequence in the p5 promoter region
that enhances
site-specific integration through interaction with Rep78/68 has been
identified,
this motif was labeled p5 integration enhancer element (p5IEE).
Apparently, p5IEE
is sufficient to create the AAVS1 - Rep68/78 - Viral DNA -complex
necessary for
specific integration, even if the ITRs containing RBEs are not present.</P><H3 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc275994054">Rescue</A></H3><P class="MsoNormal">If the latently infected host cell is
superinfected with
adenovirus, the integrated virus genome can be rescued from the human
chromosome and proceed its lytic lifecycle. Adenovirus gene products
act as
activators on AAV gene expression, leading to an excision of the viral
sequences. Like in the integration process, the Rep 78/68 proteins
catalyze the
excision by introduction of single strand nicks at the terminal
resolution
sites within the terminal repeat structures flanking the AAV genome.
DNA
polymerase, displacing the single-stranded AAV sequence, then elongates
the
resulting free 3’ OH ends. The incomplete single-stranded AAV sequence
missing
one terminal repeat primes upon itself at the homologous D motives,
allowing
DNA polymerase to copy it. This results in full-length, single-stranded
AAV molecules,
which are being able to re-enter the replicative cycle (Srivastava
2008), (Samulski 1993).</P><H3 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc275994055">Rep proteins</A></H3><H4 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc275994056">Overview</A></H4><P class="MsoNormal">The Adeno-associated virus (AAV) consists of two
open
reading frames (ORF), <I>rep</I> and <I>cap</I> ORF. Thefour
non-structural <I>rep</I> genes are driven by two promoters located at
map
units 5 (p5 promoter) and 19 (p19 promoter). Rep proteins are involved
in genome encapsidation, regulation of gene expression and replication
of the viral genome. </P><DIV align="center"><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse; margin-left: -2.25pt; margin-right: -2.25pt;" align="left" border="0" cellpadding="0" cellspacing="0"><TBODY><TR style="height: 232.5pt;"><TD style="padding: 0cm 5.4pt; width: 399.65pt; height: 232.5pt;" valign="top" width="533"><P class="MsoCaption" style="margin-left: 17.85pt; text-indent: 0cm;"> </P><P class="MsoCaption" style="margin-left: 17.85pt; text-indent: 0cm;">Figure 8: Genomic
organization of the AAV-2 genome. The <I>rep</I> gene codes for four
non-structural proteins – Rep40, Rep52, Rep68 and Rep78 – which are
involved in gene regulation, genome encapsidation and viral DNA
integration.</P></TD></TR></TBODY></TABLE></DIV><P class="MsoNormal"> </P><P class="MsoNormal">The two larger proteins Rep78/68 play an essential
role in
viral genome integration and regulation of AAV gene expression, whereas
the
smaller Rep proteins are involved in viral genome encapsidation. Rep
proteins
act both as repressors and activators of AAV transcription in respect
to the
absence and presence of helper viruses such as adenoviruses (Ad) or
herpes
simplex viruses (HSV) by interacting with several cellular proteins
(Nash et al. 2009).</P><P class="MsoNormal">Furthermore, in the absence of Rep proteins, as it
is the
case in recombinant AAVs, integration of the viral genome into the
human genome
is rare and random. There are several hotspots for integration of wtAAV
genomes
such as the human chromosome 19q13.42, known as the AAVSI site, but as
well some
other accessible chromatin regions for preferred integration have been
found (5p13.3
and 3p24.3). Integration into the human genome is mediated by the two
regulatory proteins Rep68 and Rep78 driven by the AAV p5 promoter. The
proteins
bind to the Rep binding site (RBS) which is located within the inverted
terminal
repeats (ITRs). The minimal consensus Rep binding site (RBS) <SPAN style="font-family: &quot;Courier New&quot;;">GAGT GAGC</SPAN> is found within
the ITRs and
in the p5 integration-efficient element (p5IEE) of the p5 promoter
(Hüser et al., 2010). Rep78/68 proteins possess DNA-binding
(reference), helicase
(reference) and site-specific endonuclease activity located within the
first
200 amino acids (Davis et al. 2000). Since the N-terminal region is
unique to
the larger Rep proteins, the two smaller Rep proteins possess other
biological
functions. Rep52/40 gene expression is driven by the p19 promoter which
is
located within <I>rep</I> ORF and the proteins are involved in
encapsidating
the viral genome into the preformed capsids. Gene expression of these
proteins
is suppressed in absence of adenovirus infection by binding of Rep78/68
to the
p5 promoter. Gene expression of p19 and p40 is transacvtivated by the
Rep proteins
Rep78/68 during coinfection.</P><H5 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc274911369">Rep 78</A></H5><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse;" border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="border: 1pt solid windowtext; padding: 0cm 5.4pt; width: 464.3pt;" valign="top" width="619"><P class="MsoNormal" style="margin-left: 17.85pt; text-indent: 0cm;">Rep78 in a nutshell:</P><P class="MsoNormal" style="margin-left: 17.85pt; text-indent: 0cm;">• 78 kDa</P><P class="MsoNormal" style="margin-left: 17.85pt; text-indent: 0cm;">• Endonuclease
activity </P><P class="MsoNormal" style="margin-left: 17.85pt; text-indent: 0cm;">• ATPase and helicase
activity </P><P class="MsoNormal" style="margin-left: 17.85pt; text-indent: 0cm;">• Regulate viral gene
expression</P><P class="MsoNormal" style="margin-left: 17.85pt; text-indent: 0cm;">• Involved in genome
integration into human chromosome</P></TD></TR></TBODY></TABLE><P class="MsoNormal"> </P><P class="MsoNormal">Regulated by the p5 promoter, Rep78 is the largest
non-structural protein found in the wtAAV. Besides regulation of gene
expression and viral genome replication, Rep78 has been found to play a
functional role in AAV site-specific integration into the human genome
(Hüser et al. 2010). In absence of Ad helper viruses, overexpression of
Rep78 leads to
cell cycle arrest by interacting with cell-cycle regulating
phosphatases
causing DNA damage by its intrinsic endonuclease activity (Berthet et
al. 2005) and induces apoptosis. Due to its ability to bind to the Rep
binding
site (RBS) in the p5 integration-efficient element (p5IEE) of the p5
promoter, Rep78
mediates gene expression and retain a constant level of Rep proteins by
suppressing transcriptional activity of the p5 promoter in absence of
Ad
viruses (Yue et al. 2010). Interaction of Rep78 with cellular factors
such as
transcription factors (Lackner &amp; Muzyczka 2002) provides the basis
for gene
regulation by Rep78 in associated with endogenous molecules. </P><H5 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc274911370">Rep 68</A></H5><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse;" border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="border: 1pt solid windowtext; padding: 0cm 5.4pt; width: 464.3pt;" valign="top" width="619"><P class="MsoNormal" style="margin-left: 17.85pt; text-indent: 0cm;">Rep68 in a nutshell</P><P class="MsoNormal" style="margin-left: 17.85pt; text-indent: 0cm;">• 68 kDa</P><P class="MsoNormal" style="margin-left: 17.85pt; text-indent: 0cm;">• Endonuclease
activity </P><P class="MsoNormal" style="margin-left: 17.85pt; text-indent: 0cm;">• ATPase and helicase
activity </P><P class="MsoNormal" style="margin-left: 17.85pt; text-indent: 0cm;">• Regulate gene
expression</P><P class="MsoNormal" style="margin-left: 17.85pt; text-indent: 0cm;">• Involved in genome
integration into human chromosome</P></TD></TR></TBODY></TABLE><P class="MsoNormal"> </P><P class="MsoNormal">Rep68 is a regulatory protein driven by the p5
promoter with
an apparent molecular weight of 68 kDa lacking 92 amino acids from the
carboxy
terminus due to splicing of mRNA coding for the two larger Rep proteins.</P><P class="MsoNormal">The non-structural protein Rep68 belongs to the
superfamily
3 (SF3) helicase found in other small DNA and RNA viruses such as
simian virus
40 (SV40) and bovine papillomavirus (Mansilla-Soto et al. 2009).
Formation of oligomeric complexes of Rep proteins provides
the basis for the functional versatility of the two larger regulatory
proteins.
The AAA<SUP>+</SUP> motor domain is known to function as an initiator
for
oligomerization of the Rep proteins. The cooperative effect of both
domains
appears to be further regulated by ATP binding as well as different DNA
substrates such as dsDNA and ssDNA. Assembly of different nucleoprotein
structures suggest that viral replication and genome integration is
regulated
and controlled by distinct Rep complexes which means that in presence
of dsDNA
Rep68 assembles to smaller complexes than in presence of ssDNA
resulting in
octamers. </P><H5 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc274911372">Rep52</A></H5><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse;" border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="border: 1pt solid windowtext; padding: 0cm 5.4pt; width: 464.3pt;" valign="top" width="619"><P class="MsoNormal" style="margin-left: 17.85pt; text-indent: 0cm;">Rep52 in a nutshell</P><P class="MsoNormal" style="margin-left: 17.85pt; text-indent: 0cm;">• 52 kDa</P><P class="MsoNormal" style="margin-left: 17.85pt; text-indent: 0cm;">• ATPase and helicase
activity </P><P class="MsoNormal" style="margin-left: 17.85pt; text-indent: 0cm;">• Involved in genome
encapsidation</P></TD></TR></TBODY></TABLE><P class="MsoNormal">Rep 52 is under the control of the p19 promoter
and shares
the same N-terminus with Rep78. It was shown that Rep52 possesses
helicase and
ATPase activity with 3´-5´polarity (Smith &amp; Kotin 1998). Despite
the helicase activity, Rep52 and Rep78 share a putative
zinc-finger domain, which suggest interactions with diverse cellular
factors (Nash et al. 2009) such as transcription factors (Lackner &amp;
Muzyczka 2002) and TATA-binding proteins (Hermonat et al. 1998).</P><H5 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc274911371">Rep40</A></H5><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse;" border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="border: 1pt solid windowtext; padding: 0cm 5.4pt; width: 464.3pt;" valign="top" width="619"><P class="MsoNormal" style="margin-left: 17.85pt; text-indent: 0cm;">Rep40 in a nutshell</P><P class="MsoNormal" style="margin-left: 17.85pt; text-indent: 0cm;">• 40 kDa</P><P class="MsoNormal" style="margin-left: 17.85pt; text-indent: 0cm;">• ATPase and helicase
activity </P><P class="MsoNormal" style="margin-left: 17.85pt; text-indent: 0cm;">• Involved in genome
encapsidation</P></TD></TR></TBODY></TABLE><P class="MsoNormal">The smallest Rep protein (Rep40) possesses
helicase and
ATPase activity as well, but does not have strict requirements for DNA
duplexes
containing a 3´single-stranded end. Rep40 helicase activity requires
bivalent
ions such as Mg<SUP>2+</SUP> or Mn<SUP>2+</SUP> and is most active
using ATP as
substrate. Lacking the zinc finger domain, present in Rep52, Rep40
requires
dimerization for functional helicase activity (Collaco et al. 2003).
Rep40/52 proteins are required for translocation of the
single-stranded, viral genomes into the preformed capsids proceeding
with the
3´end of the DNA (King et al. 2001). </P><DIV align="center"><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse; margin-left: -2.25pt; margin-right: -2.25pt;" align="left" border="1" cellpadding="0" cellspacing="0"><TBODY><TR style="height: 192.55pt;"><TD style="border: 1pt solid windowtext; padding: 0cm 5.4pt; width: 214pt; height: 192.55pt;" valign="top" width="285"><P class="MsoCaption" style="margin-left: 17.85pt; text-indent: 0cm;">Figure 9: Crystal
structure of the SF-3 helicase (PDB: 1SH9).</P></TD></TR></TBODY></TABLE></DIV><P class="MsoNormal" style="page-break-after: avoid;"> </P><H3 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc274911373">VP proteins</A></H3><P class="MsoNormal">The AAV capsid consists of 60 capsid protein
subunits
composed of the three cap proteins VP1, VP2, and VP3, which are encoded
in an
overlapping reading frame. Arranged in a stoichiometric ratio of
1:1:10, they
form an icosahedral symmetry. The mRNA encoding for the cap proteins is
transcribed from p40 and alternative spliced to minor and major
products.
Alternative splicing and translation initiation of VP2 at a
nonconventional ACG
initiation codon promote the expression of the VP proteins. VP1, VP2
and VP3 share
a common C terminus and stop codon, but begin with a different start
codon. The
N termini of VP1 and VP2 play important roles in infection and contain
motifs
that are highly homologous to a phospholipase A2 (PLA2) domain and
nuclear
localization signals (NLSs). These elements are conserved in almost all
parvoviruses. (Johnson et al. 2010a)</P><P class="MsoNormal"> </P><P class="MsoNormal" style="margin-bottom: 0.0001pt;"> </P><TABLE class="MsoTableGrid" style="border: medium none ; width: 466.1pt; border-collapse: collapse; margin-left: -2.25pt; margin-right: -2.25pt;" align="left" border="1" cellpadding="0" cellspacing="0" width="621"><TBODY><TR style="height: 134.65pt;"><TD style="border: 1pt solid windowtext; padding: 0cm 5.4pt; width: 440.85pt; height: 134.65pt;" valign="top" width="588"><P class="MsoCaption" style="margin-left: 17.85pt; text-indent: 0cm;">Figure 10 Genomic
organization of the AAV 2 genome. The <I>cap</I> gene codes for the
three capsid proteins VP1, Vp2 and VP3, which are responsible for the
assembly of the capsid </P></TD></TR></TBODY></TABLE><H4 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc274911374">VP1</A></H4><P class="MsoNormal">Whereas VP1 is translated from the minor spliced
mRNA, VP2
and VP3 are translated from the major spliced mRNA. The minor spliced
product
is approximately 10-fold less abundant than the major spliced mRNA.
Thus, there
is much less VP1 than VP2 and VP3 resulting in a capsid stoichiometric
ratio of
1:1:10. The N terminus of VP1 has an extension of 65 amino acids
including an
additional extension of 138 N-terminal amino acids forming the unique
portion
of VP1. It contains a motif of about 70 amino acids that is highly
homologous
to a phospholipase A2 (PLA2) domain. Furthermore, there are nuclear
localization sequences (BR)(+) which are supposed to be necessary for
endosomal
escape and nuclearentry. <SPAN lang="DE">(Bleker et al. 2006)</SPAN><SPAN lang="DE">, </SPAN><SPAN lang="DE">(Johnson et al. 2010b)</SPAN><SPAN lang="DE">, </SPAN><SPAN lang="DE">(DiPrimio et al. 2008)</SPAN><SPAN lang="DE">.</SPAN></P><P class="MsoNormal"><SPAN lang="DE"> </SPAN></P><P class="MsoNormal">Phospholipases are enzymes that hydrolyze
phospholipids into
fatty acids and other lipophilic substances and can be found in
mammalian
tissues but also in insect and snake venom. They are subdivided into
four major
classes, termed A, B, C and D distinguished by the type of reaction
they
catalyse whereas the position of hydrolysis on the glycerol backbone
defines
the class of phospholipase.</P><P class="MsoNormal"> </P><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse; margin-left: -2.25pt; margin-right: -2.25pt;" align="left" border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="border: 1pt solid windowtext; padding: 0cm 5.4pt; width: 464.3pt;" valign="top" width="619"><P class="MsoCaption" style="margin-left: 17.85pt; text-indent: 0cm;">Figure 11: The
schematic depiction of some AAV2 domains. VP1 contains a phospholipase
A2 (PLA2) domain and four basic regions (BR1–4) located at the
N-terminus of VP1 – VP3. The HSPG binding domain is generated by the
basic residues at positions R484, R487, K532, R585, and R588 which are
located near the C-terminus of theVP proteins. The NGR motif 511–513
forms an integrin <SPAN style="font-family: Symbol;">a</SPAN>5<SPAN style="font-family: Symbol;">b</SPAN>1 binding domain. Adapted from
(Michelfelder &amp; Trepel 2009)</P></TD></TR></TBODY></TABLE><P class="MsoNormal"> </P><P class="MsoNormal">It specifically recognizes and hydrolyzes the sn-2
acyl bond
of phospholipids releasing arachidonic acid and lysophospholipids.</P><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse; margin-left: -2.25pt; margin-right: -2.25pt;" align="left" border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="border: 1pt solid windowtext; padding: 0cm 5.4pt; width: 464.3pt;" valign="top" width="619"><P class="MsoCaption" style="margin-left: 17.85pt; text-indent: 0cm;">Figure 12: adapted
from http://www.biochemtech.uni-halle.de/im/1182353660_397_00_800.jpg</P></TD></TR></TBODY></TABLE><P class="MsoNormal"> </P><P class="MsoNormal">The reaction mechanism will be depicted by the
following
image:</P><P class="MsoNormal"> </P><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse; margin-left: -2.25pt; margin-right: -2.25pt;" align="left" border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="border: 1pt solid windowtext; padding: 0cm 5.4pt; width: 464.3pt;" valign="top" width="619"><P class="MsoCaption" style="margin-left: 17.85pt; text-indent: 0cm;">Figure 13 adapted from
“Interfacial Enzymology: The Secreted Phospholipase A2-Paradigm”</P></TD></TR></TBODY></TABLE><P class="MsoNormal"> </P><P class="MsoNormal">There are two possible mechanism but the one on
the right
side has a lower transient state. The mechanism of sPLA2 on the right
side is
initiated by a His-48/Asp-99/calcium complex within the active site.
The sn-2
carbonyl oxygen becomes polarized by the calcium ion while also
influencing
catalytic water molecule, w5..Via the bridging second water molecule w6
His-48
improves the nucleophilicity of the catalytic water. According to
propositions
two water molecules are needed to bypass the distance between the
catalytic
histidine and the ester.. Asp-99 is thought to enhance the basicity of
His-48
through hydrogen bonding. Substituting the His-48 with an asparagine
maintains
wild-typ activity because the functional group on asparagines can
function to lower
the pKa of the bridging water molecule, too (Berg et al. 2001). </P><P class="MsoNormal"><SPAN style="line-height: 150%;">The phospholipase
A2 is suggested
to mediate membrane disruption of the vesicular compartment, which
would allow
escape of the virion into the cytosol, although this has not been
demonstrated
for AAV so far.The propensity of cellular PLA2 to cleave phospholipids
is usually
regulated by intracellular Ca<SUP>2+</SUP> levels as well as
phosphorylation of
residues near the catalytic domain of the PLA2. Not surprisingly, the
N-terminus of VP1 contains a GXG binding site and several
phosphorylation
sites.</SPAN></P><P class="MsoNormal"><SPAN style="line-height: 150%;"> </SPAN></P><P class="MsoNormal"><SPAN style="line-height: 150%;">According to the
structural
modeling of VP1 the N-terminus can translocate through the 5-fold axis
of
symmetry in the capsid and expose the first 185 residues of VP1
(comment:
experimental data also “</SPAN><SPAN style="line-height: 150%;">strongly
suggest
that N-termini of VP1 </SPAN>harboring the PLA2 domain can be exposed
on the
capsid surface through the pores at the fivefold symmetry axes” (Girod
et al. 2002)(Bleker et al. 2005a). It only takes about 19 amino acids
to reach
through a phospholipid bilayer so this length would be sufficient for
the
presentation of the NLS and di-lysine sequences to the cytosol assuming
the
PLA2 domain<SPAN style="line-height: 150%;"> had penetrated through an
endosomal
membrane or Golgi. </SPAN><SPAN style="font-size: 10pt; line-height: 150%;">(Michelfelder &amp; Trepel
2009)</SPAN></P><P class="MsoNormal"> </P><P class="MsoNormal">Analysing individual steps in the life cycle of
several VP1up
mutants and wtAAV-2 lead to the following conclusions: (i) mutations in
VP1up
did not affect DNA packaging or replication but resulted in a strong
reduction of
infectivity; (ii) this decrease in virus infectivity correlated with a
loss in
pvPLA2 activity; (iii) binding to the cell surface and entry into cells
was not
affected in VP1up mutants; (iv) however, these mutants showed obviously
reduced
and delayed Rep expression. (Girod et al. 2002) Summarizing these
results the pvPLA2 activity is required for a step
in the life cycle of the virus following perinuclear accumulation of
virions
but (Girod et al. 2002) before the onset of early gene expression.</P><P class="MsoNormal"> </P><P class="MsoNormal">Maybe future work will uncover whether the PLA2
domain in
AAV performs optimally in a specific vesicular compartment, prefers a
specific
phospholipid substrate, operates at multiple cellular membranes such as
the
endosome and the nuclear envelope, or if its activity is regulated by
cellular
components. </P><P class="MsoNormal"> </P><H4 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc274911375">VP2</A></H4><P class="MsoNormal">The translation of VP2 from the major spliced mRNA
is less
efficiently compared to the translation of VP3 because it initiates at
a Thr
codon (ACG). VP2 and VP1 have an extension at the N terminus that
remains
internal when exposing the capsid to experimental conditions like low
pH or
heat. The N terminus of VP1 has an extension of 65 amino acids and
similar to
VP1 it has two functional elements: a phospholipase A2 (PLA2) domain
and nuclear
localization signals (BR)(+). The exact role of VP2 remains unknown,
although
the protein is thought to be nonessential for viral assembly and
infectivity (Johnson et al. 2010b), (DiPrimio et al. 2008).</P><H4 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc274911376">VP</A>3</H4><P class="MsoNormal">Contained in VP1 and VP2, VP3 is the primary
capsid protein
that determines the surface topology of the AAV capsid. The capsid in
turn
dictates antigenicity and tropism. In comparison to the initiation of
VP1 and
VP2 the initiation of VP3 is because of a Met codon highly efficient
(DiPrimio et al. 2008). </P><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse; margin-left: -2.25pt; margin-right: -2.25pt;" align="left" border="1" cellpadding="0" cellspacing="0"><TBODY><TR style="height: 222.55pt;"><TD style="border: 1pt solid windowtext; padding: 0cm 5.4pt; width: 441.8pt; height: 222.55pt;" valign="top" width="589"><P class="MsoCaption" style="margin-left: 17.85pt; text-indent: 0cm;">Figure 14: Genomic
organization of the AAV-2 genome. The three capsid proteins VP1, Vp2
and VP3 have a similar C terminus. The N termini of VP1 and VP2 contain
a phospholipase A2 (PLA2) domain and nuclear localization signals
(BR)(+).</P></TD></TR></TBODY></TABLE><P class="MsoNormal"> </P><H4 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc275994065">Natrual
Tropism and HSPG motif</A></H4><P class="MsoNormal" style="">The primary receptor of AAV-2 is
the heparan sulfate proteoglycan (HSPG) receptor (Perabo et al. 2006).
Its binding motif consists of five amino-acids located on the
capsid surface (Trepel, Vectors, et al. 2009).</P><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse;" border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="border: 1pt solid windowtext; padding: 0cm 5.4pt; width: 464.3pt;" valign="top" width="619"><P class="MsoNormal" style="margin-left: 17.85pt; text-indent: 0cm;">Figure 15: adapted
from (Trepel, Vectors, et al. 2009). Schematic depiction of the 5 basic
amino acids forming the HSPG motif: R484/R487, K532, R585/587. Other
domains or binding motifs are not shown in this picture.</P></TD></TR></TBODY></TABLE><P class="MsoNormal" style=""> </P><P class="MsoNormal" style="">HSPG belongs to the
glycosaminoglycanes as well as heparin and consists of heparan sulfate <SPAN lang="EN">glycosaminoglycan attached to a</SPAN> core protein and can
be found on
every human cell surface. </P><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse; margin-left: -2.25pt; margin-right: -2.25pt;" align="left" border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="border: 1pt solid windowtext; padding: 0cm 5.4pt; width: 464.3pt;" valign="top" width="619"><P class="MsoCaption" style="margin-left: 17.85pt; text-indent: 0cm;">Figure 16: adapted
from (Sinnis et al. 2007) amidosulfated disaccharid sequence in heparin.</P></TD></TR></TBODY></TABLE><P class="MsoNormal" style=""> </P><P class="MsoNormal" style="">Its acid residues bear negative
charges and are therefore prone to electrostatic interactions with e.g.
the
positively charged HSPG binding motif of AAV-2. Other interactions with
polar
residues are possible, too.</P><P class="MsoNormal" style="">Regarding AAV-2, two point
mutations in AAV-2 (R585A and R588A) are sufficient to eliminate
heparin binding
(Opie et al. 2003). The biobricks with this knockout are annotated with
„HSPG-ko“.</P><P class="MsoNormal"> </P><H4 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc275994066"><SPAN lang="DE">Assembly-activating protein</SPAN></A></H4><P class="MsoNormal">A gene encoding for the assembly-activating
protein (AAP)
was recently (in 2010) discovered in the Adeno-associated virus (AAV)
serotype
2 genome. Its gene product is conserved among all AAV serotypes,
illustrating
its essential role in virus life cycle. Its functions comprise
transport of the
viral structural proteins to the nucleolus and involvement in following
capsid
assembly.</P><P class="MsoNormal">The AAP gene, located in the Cap coding region, is
translated from an alternative open reading frame (ORF) with
unconventional
start codon. If modifications need to be introduced in the AAV capsid –
for
example for targeting approaches – the AAP has to be taken in account
in order
to prevent virus assembly impairment.</P><P class="MsoNormal"> </P><P class="MsoNormal"> </P><H3 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc275994067">Trafficking</A></H3><P class="MsoNormal">For creating efficient AAV2 vectors, precise
knowledge of
the events following virus transduction is necessary. The subsequent
scheme and
summary is intended to be an introduction into the complex process of
virus
transduction. </P><DIV align="center"><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse; margin-left: -2.25pt; margin-right: -2.25pt;" align="left" border="1" cellpadding="0" cellspacing="0"><TBODY><TR style="height: 354.75pt;"><TD style="border: 1pt solid windowtext; padding: 0cm 5.4pt; width: 316.8pt; height: 354.75pt;" valign="top" width="422"><P class="MsoCaption" style="margin-left: 17.85pt; text-indent: 0cm;">Figure 17: Adapted
from (Hildegard Büning et al. 2008)</P></TD></TR></TBODY></TABLE></DIV><P class="MsoNormal"> </P><P class="MsoNormal">After several contacts with cellular structures
like heperan
sulphate proteoglycan (HSPG) the viral capsid proteins get rearranged.
Clathrin-mediated endocytosis and cellular trafficking into
thecell’scenter
follows. After acidification and following endosomal escape, the viral
genome
is transferred into the nucleus and replicated (lytic phase) or
integrated into
the host genome (latent phase). </P><P class="MsoNormal"><SPAN style="color: black;">Before entering the
cell, the
viral particle has in average 4.4 contacts with the cellular surface.</SPAN><SPAN style="color: black;">(Seisenberger et al. 2001)</SPAN><SPAN style="color: black;">
The main receptor of AAV2 is heperan sulfate proteoglycan (HSPG). After
contact
with HSPG the capsid structure gets rearranged </SPAN><SPAN style="color: black;">(Levy et al. 2009)</SPAN><SPAN style="color: black;">. This
is probably essential for interaction with other cofactors, which leads
to
endocytosis. The factors respectively co-receptors of the cellular
surface are
known to enhance the initial binding affinity of HSPG: Fibroblast
growth factor
receptor 1 (FGFR-1), hepatocyte growth factor receptor (HGFR) and
laminin
receptor. It is known that AAVs affect both: </SPAN><SPAN style="color: black;" lang="DE">α</SPAN><SPAN style="color: black;">V</SPAN><SPAN style="color: black;" lang="DE">β</SPAN><SUB><SPAN style="color: black;">5</SPAN></SUB><SPAN style="color: black;"> and
</SPAN><SPAN style="color: black;" lang="DE">α</SPAN><SPAN style="color: black;">V</SPAN><SPAN style="color: black;" lang="DE">β</SPAN><SUB><SPAN style="color: black;">1</SPAN></SUB><SPAN style="color: black;">integrin.
The </SPAN><SPAN style="color: black;" lang="DE">α</SPAN><SPAN style="color: black;">V</SPAN><SPAN style="color: black;" lang="DE">β</SPAN><SUB><SPAN style="color: black;">1</SPAN></SUB><SPAN style="color: black;"> -binding
site is an asparagine-glycine-arginine motif
</SPAN><SPAN style="color: black;">(Asokan et al. 2006)</SPAN><SPAN style="color: black;">.
These integrins interact with intracellular molecules like Rho, Rac and
Cdc42
GTPases. Figure 2 depicts the following cascade.</SPAN></P><DIV align="center"><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse; margin-left: -2.25pt; margin-right: -2.25pt;" align="left" border="1" cellpadding="0" cellspacing="0"><TBODY><TR style="height: 232.6pt;"><TD style="border: 1pt solid windowtext; padding: 0cm 5.4pt; width: 320.35pt; height: 232.6pt;" valign="top" width="427"><P class="MsoCaption" style="margin-left: 17.85pt; text-indent: 0cm;">Figure 18: Adapted
from (Sanlioglu et al. 2000a)</P></TD></TR></TBODY></TABLE></DIV><P class="MsoNormal"> </P><P class="MsoNormal">The initial contact with HSPG, FGFR-1, HGFR and/or
laminin
is followed by an interaction with αVβ5 and/or αVβ1 which
propably leads to an intracellular activation of enzymes involved in
the
rearrangement of cytoskeletal proteins like actin, via PI3K-pathway
(Kapeller &amp; Cantley 1994), (Li et al. 1998). In general, the
receptor-mediated
endocytosis (RME) is a complex process proteins and co-factors form
clathrin
coated pits as shown in Figure 16.</P><DIV align="center"><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse; margin-left: -2.25pt; margin-right: -2.25pt;" align="left" border="1" cellpadding="0" cellspacing="0"><TBODY><TR style="height: 249pt;"><TD style="border: 1pt solid windowtext; padding: 0cm 5.4pt; width: 351.6pt; height: 249pt;" valign="top" width="469"><P class="MsoCaption" style="margin-left: 17.85pt; text-indent: 0cm;"><A name="_Ref275689978">Figure
</A>19:</P></TD></TR></TBODY></TABLE></DIV><P class="MsoNormal"> </P><P class="MsoNormal">The adaptor proteins (APs) AP1, AP2, AP3 and AP4
are
complexes built of four subunits (Collins et al. 2002), (Asokan et al.
2006). Except for AP2, which requires
GTP-bound-Arf1, the APs are linked via phosphatidylinositol
(4,5)-bisphosphate
(PIP2) to the cell membrane (Robinson 2004). APs recognize short
cytoplasmatic
motifs like YXX-phi (phi: bulky hydrophobic AA) of transmembrane
receptors. In general,
the recognition sites (mu-subunits) in the AP-complexes have to be
phosphorylated
by kinases (Ohno et al. 1995).</P><P class="MsoNormal">The actual scaffold of the endosome is build by
the
triskelion formed clathrins. The rigide backbone of clathrins is formed
by
three heavy chains (Ybe et al. 1999) and three light chains are
regulating
assembly competence (BRODSKY et al. 1991). After building the clathrin
scaffold, dynamine is responsible for pinching-off the clathrin-coated
pits (CCPs)
from the cell’s membrane (Summerford &amp; Samulski 1998) (Sanlioglu et
al. 2000b). </P><H4 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc275994068">Endosomal
transport and escape</A></H4><P class="MsoNormal">Still there are possible additional entering
pathways, for
example knocking down microtubuli and microfilament arrangement does
not
prevent transduction completely (Kelley 2008). Currently it is thought
that endosomal escape happens in the cytoplasma. After
pinching off, the endosomes move via motor proteins along microtubuli
and
microfilaments towards the nuclear area. While trafficking through the
cell the
early endosoms getting acidulated (Sonntag et al. 2006).</P><P class="MsoNormal"><SPAN style="color: black;">Additional entering
pathways were
postulated for the virus, for example, it has been shown that
proteasomal degradation
via ubiquitination hampers transduction efficiency </SPAN><SPAN style="color: black;">(Douar et al. 2001)</SPAN><SPAN style="color: black;">.</SPAN></P><P class="MsoNormal"><SPAN style="color: black;">The first viral
particles in the
nuclear area can be detected after 15 minutes </SPAN><SPAN style="color: black;">(Seisenberger et al. 2001)</SPAN><SPAN style="color: black;">
and an accumulation of virions takes place after 30 minutes post
transfection.</SPAN><SPAN style="color: black;">After arrival, the viral
genomes are transported into the nucleus. It is not entirely clear in
which way
the transport is accomplished. The viral particles seem to use
different
pathways to enter the nucleus, either via the nuclear pore complexes
with their
maximal pore size of 23 nm. In this case, the viral capsid (25 nm
diameter) has
to be remodeled. Controversial results were published in the past,
detecting
intact viral particles (lu et al., 2000), but according to Lux et al.
no intact
capsids were detectable when lower amounts of viral particles were
transduced </SPAN><SPAN style="color: black;">(Lux et al. 2005)</SPAN><SPAN style="color: black;">.</SPAN></P><P class="MsoNormal"><SPAN style="color: black;">Obviously further
investigation of
intracellular trafficking is essential for optimizing the AAV2 for
medical
applications.</SPAN></P><H2 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc275994069">Helper Genes</A></H2><P class="MsoNormal">The AAV Helper-Free System by Stratagene
(Waldbronn, Germany)
is a modularized system for the production of infectious recombinant
AAV-2
virion not depending on a coinfection with any helper virus. It is put
into
practice by the three plasmids pHelper, pAAV-RC, recombinant pAAV
vector
containing the gene of interest (GOI) and the recombinant cell line
AAV-293. </P><P class="MsoNormal">The AAV-2 is a replication-deficient parvovirus,
which
originally needs a co-infection of adenovirus or herpes virus for
replication. To
realize a functional replication of AAV-2 without a co-infection, the
AAV
Helper-Free System allocates the pHelper plasmid and the AAV-293 host
cells.
The pHelper plasmid encodes for nearly all of the required adenovirus
gene
products for replication (VA, E2a, E4). The AAV-293 host cells express
stably
the remaining important replication genes (E1A, E1B). </P><P class="MsoNormal">Due to the fact that AAV-2 needs all relevant
replication
genes for productive infection and that the important replication-genes
are
dispersed, the AAV Helper-Free System describes a saver alternative to
retroviral
or adenoviral gene delivery (Stratagene n.d.)</P><P class="MsoNormal">The AAV-293 host cells contain the E1A and E1B
genes. The
E1A gene is the first gene to be expressed during an adenovirus
infection. The
E1A gene produces two different mRNAs resulting in two different
proteins. The
expressed E1A proteins transactivate and induce transcription of other
early
genes (like E2 and E4). In this case, E1A proteins do not bind directly
onto
control regions, but interact with other host proteins, which are
binding to
those regions (Chang et al. 1989) (Modrow et al. 2003).</P><DIV align="center"><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse;" border="1" cellpadding="0" cellspacing="0"><TBODY><TR style="height: 261.85pt;"><TD style="border: 1pt solid windowtext; padding: 0cm 5.4pt; width: 176.25pt; height: 261.85pt;" valign="top" width="235"><P class="MsoNormal" style="margin-left: 17.85pt; text-indent: 0cm;">Figure 20: Schematic
overview of the Helper Free System provided by Stratagene.</P></TD></TR></TBODY></TABLE></DIV><P class="MsoNormal"> </P><P class="MsoNormal">In the AAV-2 infection cycle, E1A proteins
stimulate the
expression of the p19 promoter and the p5 promoter, which are required
to begin
with the <I>rep</I>-gene transcription of the AAV(Chang et al. 1989)
(Tratschin et al. 1984).</P><P class="MsoNormal">The E1B region encodes two polypeptides with
overlapping
reading frames, the major 21-Mr product and the 55-Mr moiety. It has
been shown
that only the 55-Mr polypeptide is required for effective helper
function. It
enables along with the E4orf6, a stable accumulation of AAV-specific
cytoplasmic
RNA, capsid proteins and DNA replication(Samulski &amp; Shenk 1988). In
this context it has to be mentioned, that Stratagene
deleted the E4orf6 out of its kit, because of its oncolytic activity.
But it
has been explained, that deletion of E4orf6 has no effect on virion
production
efficiency(Clark et al. 1999) . </P><P class="MsoNormal">The pHelper plasmid exists of the E2a gene, the E4
gene and
the VA gene as well as a pUC ori and an f1 ori. The E2A gene encodes a
72 kD
protein which is produced early in infection (Modrow et al. 2003). One
helper function of E2A is to increase the processivity of
replication. In the presence of E2A protein, short replication
products, which
are equivalent to break offs of the elongation strand of the template,
are
obviously reduced suggestion that E2A supports full-length replication
of short
substrates. In immune-depletions, co-localizations between the E2A, the
AAV Rep
protein and the AAV DNA have been shown (Ward et al. 1998) .</P><DIV align="center"><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse; margin-left: -2.25pt; margin-right: -2.25pt;" align="left" border="0" cellpadding="0" cellspacing="0"><TBODY><TR style="height: 238.95pt;"><TD style="padding: 0cm 5.4pt; width: 346.8pt; height: 238.95pt;" valign="top" width="462"><P class="MsoNormal" style="margin-left: 17.85pt; text-indent: 0cm; page-break-after: avoid;"> </P></TD></TR></TBODY></TABLE></DIV><P class="MsoNormal">It has been reported that E2A has affections on
the AAV
promoter regulation of spliced p5 and p19 as well as unspliced p40. E2A
could
also be responsible for the production of the AAV capsid proteins
(Carter et al. 1992). To which extend it really takes elementary
responsiblitiy for any of
the listed functions is not found out yet.</P><P class="MsoNormal">The VAI and VAII genes encode for two RNA-species,
with a
high GC-percentage and distinct secondary structure. The VAI RNA, which
is
implicated to have a helper function in AAV, usually plays a
fundamental role
in adenovirus’ protein expression. There it blocks the phosphorylation
of the
initiation factor eIF-2, whereby the amino acid chain at the ribosome
breaks
off. (Modrow et al. 2003) </P><P class="MsoNormal">The expression of the AAV proteins may also be
under the VAI
adenovirus control. VAI may increase the AAV capsid production, but it
also may
play a role in RNA metabolism.(West et al. 1987) </P><P class="MsoNormal">The E4 gene exists of seven open reading frames.
In this
content, the proteins occurring from the gene are named E4-ORF1 up to
E4-ORF7.
All proteins are under the control of one promoter and arise from
alternative
splicing. The E4ORF6 is implicated to have a helper function in AAV. It
promotes the formation of a dsDNA from the genomic ssDNA of the native
virus.</P><H2 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc274911377">Recombinant Viruses
and Mosaic
Viruses</A></H2><H2 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc274911378"><SPAN lang="DE">Gene
Therapy</SPAN></A></H2><P class="MsoNormal">Treating inherited and acquired diseases such as
cancer is
still one of the most challenging fields in today’s biomedical
research. Ever
since Sidney Farber published a study in 1949 about several folic acid
antagonists,
which prevent tumor progression (FARBER 1949), cancer was treated with
chemotherapy, surgery and radiation (Halperin 2006). Nevertheless, due
to side effects caused by systemic applications and
the lack of specificity, new treatments must be found for improved
therapeutic
efficacy and enhanced selectivity of the anticancer agents. One
promising approach
of treating cancer is suicide gene therapy or gene-directed enzyme
prodrug
therapy (GDEPT) including two steps of treatment: Targeted introduction
of a
gene encoding for enzymes into tumor cells, followed by the
administration of a
non-toxic prodrug which is converted into an anti-cancer metabolite. </P><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse; margin-left: 4.8pt; margin-right: 4.8pt;" align="left" border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="border: 1pt solid windowtext; padding: 0cm 5.4pt; width: 464.3pt;" valign="top" width="619"><P class="MsoCaption" style="margin-left: 17.85pt; text-indent: 0cm;">Figure
21: Schematic overview of gene-directed enzyme
prodrug therapy (GDEPT). The suicide gene is introduced into the cancer
cells. Administration of the prodrug leads to cell death in the cells
expressing the enzyme, which converts the prodrug into the toxic
product.</P></TD></TR></TBODY></TABLE><P class="MsoNormal">Gene delivery using viral vectors to specifically
target
cells gained increasing attention in the last years being efficient in
combination with suicide gene therapy. Several prodrug/enzyme systems
have been
reported (Greco &amp; Dachs 2001). Ganciclovir (GCV)/herpes simplex
virus
thymidine kinase (HSV-TK), 5-fluorocytosine/cytosine deaminase (CD) and
cyclophosphamide/cytochrome P450 systems have been widely used and
their activity
has been demonstrated in several preclinical studies (Greco &amp; Dachs
2001).</P><P class="MsoNormal"> </P><P class="MsoNormal">Using transgenic HSV – thymidine kinase or
cytosine
deaminase from <I>E. coli</I> for prodrug activation in tumor therapy
several
advantages can be found. Besides efficient killing of targeted tumor
cells, neighboring,
non-transduced cells are killed as well, providing an important effect
in
treating cancer. The bystander phenomenon was first reported by Moolten
(1986) showing that HSV-TK negative cells surrounded by HSV-TK positive
cells did not
survive prodrug treatment.</P><TABLE class="MsoTableGrid" style="border: medium none ; border-collapse: collapse; margin-left: -2.25pt; margin-right: -2.25pt;" align="left" border="1" cellpadding="0" cellspacing="0"><TBODY><TR><TD style="border: 1pt solid windowtext; padding: 0cm 5.4pt; width: 460.5pt;" valign="top" width="614"><P class="MsoCaption" style="margin-left: 17.85pt; text-indent: 0cm;">Figure 22: Efficient
tumor killing is desired in cancer treatment. Locally administered
prodrugs are converted to toxic metabolites by delivered enzmye in the
infected cells. By passive diffusion, gap junction intercellular
communication or immune-related response, non-transduced tumor cells
are killed as well. </P></TD></TR></TBODY></TABLE><P class="MsoNormal"> </P><P class="MsoNormal">Transfer of toxic molecules between transduced and
non-transduced
cells can be achieved either through gap junctions (Yang et al. 1998)
(Trepel, Stoneham, et al. 2009), via apoptotic bodies (Freeman et al.
1993) or by diffusion of soluble toxic substances (Huber et al. 1993). </P><H2 style="margin-left: 0cm; text-indent: 0cm;"><A name="_Toc275994072">Immune Response</A></H2><P class="MsoNormal">Virus infections cause common human disease,
including the
familiar cold, influenza, mumps and measles. They are also associated
with
severe diseases, for example with Ebola or Marburg fever, with
Hepatitis and
AIDS. The immune system protects us from these infections by creating a
barrier
that prevents viruses from entering the body or by detecting and
eliminating
them in the corpus. </P><P class="MsoNormal">Every virus lives and reproduces in its own
specific host.
Reproduction can only take place in cell cytoplasm, whose components
are needed
in order to compensate for the lack of virus metabolism and
biosynthesis-appliance. Viruses, which are located outside the cells,
can be
detected by antibodies, triggering an immune response by the members of
the
innate immune system, such as macrophages, neutrophils and natural
killer-cells.</P><P class="MsoNormal">Inside a cell the virus can only be detected by
cytotoxic T
cells: While it uses the cellular machinery for reproduction, some of
the viral
proteins are degraded by proteasomes and become presented on the cell
surface
by MHC-I-peptides. These exposed virus components are recognized by
cytotoxic
CD8+ T cells, which induce death of infected cells. The degraded viral
proteins
can also be accessible on MHC-II-peptides, which are detected by CD4+ T
helper
cells, which trigger and enforce the immune response for example by
production
of specific antibodies. </P><P class="MsoNormal">The Adeno-associated virus (AAV) is not associated
with any
human disease. Nevertheless, usage of recombinant AAV (rAAV) as
therapeutic
vector system harbors risks of immune responses. </P><P class="MsoNormal">AAV establishes a latent infection and often
integrates at a
specific site on q arm of chromosome 19, which is termed AAVSI site
(Hernandez
et al., 1999). This leads to several obstacles for usage of AAV vectors
for
therapeutic applications like gene silencing, insertions in gene
sequences and
immunotoxocity, a dangerous immune response to the vector or the
transgene product
(Mingozzi &amp; High 2007). Humans are the only natural hosts for AAV-2
besides
rhesus macaques. Due to wild-type AAV infections humans keep a
population of
antigen-specific memory CD8+ T cells (Mingozzi &amp; High 2007). IgG
antibodies
are predominantly involved in the secondary immune response. 91% of
Irish blood
donors show a high repertoire of specific IgG1 and IgG2 subclasses and
low
doses of IgG3 (Madsen et al. 2009). </P><P class="MsoNormal">In vivo studies with AAVlacZ show that AAV vectors
induce
the secretion of chemokines and cytokines like gamma interferon (IFN-γ)
(Zaiss et al. 2002). Studies in vitro show responses of IFN-γ,
interleukin
10 (IL-10) and interleukin 13 (IL-13) after stimulation peripheral
blood
mononuclear cells (PBMC) from donors with AAV-2. This demonstrates a
reaction
of long-live CD4+ T helper-cells that are reactivated (Madsen et al.
2009). These
results reveal that most Europeans are already infected with
wildtype-AAV-2.
Researchers suggest that more than 30% of mankind is already infected.
In vitro
studies from the United States support this hypothesis. One group found
anti-AAV-antibodies in the blood sera at 80% of randomly chosen
volunteers
(Moskalenko et al. 2000). Other investigators show that 0,14% of the
examined
CD8+ T cells purified from PBMC are capsid specific for AAV-2
(Mingozzi, Maus,
et al. 2007). These preexisting memory-CD8+ T cells could be
responsible for
the difference in vector-infusion outcome between humans (the natural
host) and
other species. </P><P class="MsoNormal">AAV-2 use distinct cellular receptors, e.g.
heparin sulfate
proteoglycan (HSPG), αVβ5 integrin and human fibroblast growth factor
receptor 1 (FGFR1) to become internalized (Favaro et al. 2009). These
findings
led researchers to the conclusion that the presence of an intact
heparinbinding
motif and the capsid t-cell responses are correlated. One group ablated
the
heparin-binding site in AAV-2 and observed no CD8+ T cell response. But
it did
not seem to influence T helper responses as measured by IgG isotypes
and
antigen-stimulated secretion of cytokines (Vandenberghe et al. 2006).</P><P class="MsoNormal">Approaches using peptides derived from the
sequence of the
VP1 viral capsid protein revealed a total of 59 t-cell epitopes. This
demonstrates the difficulty to avoid the immune system by modifying the
AAV
capsid (Madsen et al. 2009). Other approaches in mice reveal that
different
serotypes of AAV show the ability to cross-react with existing memory-T
cells
(Sabatino et al. 2005). Also in dogs different AAVs use some common
peptides on
their surface to activate the immune system (Wang et al. 2010). This
shows the
high conservation of the epitopes among multiple serotypes of AAV. </P><P class="MsoNormal">While proposing several possible solutions to
avoid the
immune system, the polymorphic nature of the human MHC and the high
conservation of peptides on the surface of different serotypes of AAV
may
complicate these approaches (Mingozzi, Hasbrouck, et al. 2007). In
general it
can be said that the immune response to AAV is not severe as caused by
other
virus-types. This is due to the fact that AAVs fail to trigger
inflammatory
reactions dendritic cells need to differentiate into professional
antigen-presenting cells (Mingozzi, Maus, et al. 2007). These
antigen-presenting cells are needed for the activation of CD4+ T
helper-cells
which are needed for the completely feedback to the immune system.
Nevertheless
dendritic cells can be activated through the ability of AAV-2 to bind
the HSPG
binding motif with resultant AAV2 antigen inclusion, processing and
MHC-I
presentation (Wang et al. 2010). CD4+ T helper-cells can also be
activated by other
antigen-presenting cells therefore it is conceivable to block CD4+
cells during
treatment with AAV. The activation of CD8+-t-cells through CD4+ T cells
is
depleted and the immune response is even more reduced than within the
normal
infection process. </P><P class="MsoNormal">Some researchers have found AAV vector DNA in the
semen of
dogs and fear the risk of germline transmission (Jiang et al. 2006)
although
these findings are controversially discussed. In a rabbit model it was
demonstrated that semen was just positive for vector sequences
following
intravascular injection but not following intramuscular injection.
Infectious
vector particles were just detected up to four days after treatment and
were
undetectable thereafter. So the investigators suggest that AAV-2
presents a low
risk of germline transmission for humans and there is no contemplation
for male
infertility so far (Favaro et al. 2009). </P><P class="MsoNormal">AAV vectors have been used in several phases of
clinical
trials for Leber’s congenital amaurosis (LCA), hemophilia B, Cystic
fibrosis,
Arthritis, Muscular dystrophy, Parkinson’s disease, Canavan's disease,
Alzheimer's disease, Batten's disease and Hereditary emphysema.</P><P class="MsoNormal"> </P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Freiburg_Bioware/Project/Project_Description">http://2010.igem.org/Team:Freiburg_Bioware/Project/Project_Description</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg_Bioware/Project/Project_Description" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg_Bioware/Project/Project_Description" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg_Bioware/Project/Project_Description&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg_Bioware/Project/Project_Description&amp;oldid=203705" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>