"
Page
Discussion
View source
History
teams
Log in
 
Team:Freiburg Bioware/Project/insertion of motifs into surface-exposed loops
From 2010.igem.org
Home
Team
Team 2010
Collaboration
Cuckoo Clock Competition
Photo Gallery
Statistics
Contribution
Project
Summary
Highlights
Introduction
Results
   Modularization
   Targeting
   Arming
   Tumor Killing
   Modeling
Methods
Virus Construction Kit Manual
BioBricks
Overview
Capsid Coding Parts
Vector Plasmid Parts
Notebook
Labjournal
Cloning Material
Methods
Meetings
Statistics
Sponsors
Safety
Modeling
Modeling Overview
Virus Production
Virus Infection
Structure Modeling
Loop Insertion
Modification of the Viral Capsid of the AAV2 using Viral Bricks
For therapeutical applications in human gene transfer, the broad tropism for heparan sulfate proteoglycan (HSPG) has to be knocked-out and a novel tropism has to be inserted. This retargeting can be achieved either by insertion of functional motifs into the two major surface exposed loops or by fusion of these motifs to the N-terminus of the viral coat proteins. The graphic on the right shows parts of the three-dimensional structure of a viral coat protein. The parts of the loop regions that are coded in the ViralBricks are shown in purple for the 453 loop and in blue for the 587 loop.
Three-dimensional representation of the AAV2 showing the amino acids of the 453 and 587 loops that are coded by the corresponding Viral Bricks.
Cloning of Viral Bricks into capsid coding parts
In order to make loop insertions more convenient the following restriction sites were inserted into all capsid coding parts and already existing restriction sites were removed from the constructs. The choice of these restriction sites was reasoned by enzyme performance, buffer compatibilities and the number of existing restriction sites that had to be removed at other positions. All restriction endonucleases were purchased from NEB.
His-Affinity tag
Schematic figure of His-affinity Tag insertion into the viral capsid
The insertion of a His-Affinity Tag into the exposed major surface loops of the viral vectors allows their specific affinity purification employing e.g. Ni-NTA affinity chromatography. This purification method was tested using the cell culture lysate of transfected AAV-293 cells that were either grown in DMEM supplemented with 10% FCS or in the serum-free GIBCO® FreeStyle™ 293 Expression Medium (Invitrogen). The usage of serum-free media is a technological modification meant to facilitate the production of pure viral vectors. Purified viral vectors are important for several applications such as animal models and biophysical characterizations. On the other hand, the elution of the His-tagged viral vectors allows also enrichment of transgene viral vectors. In order to answer the question if and to what degree viral vectors are transferred into the media, the cells were centrifugated, then divided into the pellet fraction and the supernatant. Physical cell lysis was performed for both fractions of the two produced batches (serum-free and FCS media) by performing four cycles of freeze and thaw.
 
Experimental setup for the purification of His-tag presenting viral particles produced by AAV-293 cells in DMEM or the protein- and animal-free Freestyle medium. Purified viral particles can be detected by ELISA using an anti-poly-Histidine antibody or qPCR for encapsidated vector plasmids, respectively
 
Material and Methods:
Transfection of the AAV-293 producer cells was performed in five 10 cm petri dishes with 3.6x10^6 cells, resulting in a confluency of about 70-80% according to the standard protocol either with cells grown in GIBCO® FreeStyle™ 293 Expression Medium (Invitrogen, protein- and animal-origin free) or in DMEM supplemented with 10% FCS (PAA). For transfection, the composite parts pCMV_VP123(587-His) and RepVP123(587-KO)_p5-TATA-less were used in an 1:1 ratio together with pHelper and [AAV2]-left-ITR_pCMV_betaglobin_mVenus_hGH_[AAV2]-right-ITR. Cells were spun down at 200 x g for five minutes and the samples were divided into the pellet and the supernatant fractions. Physical cell lysis was performed by four cycles of freeze and thaw for all four samples. The cell lysate / supernatant fractions were incubated with 800 µl of His-Affinity Gel (kindly provided by Zymo Resear    ch, USA) at 4 °C for 18 hours with 200 rpm constant agitation. The beads were then collected in 5 ml gravity-flow columns and washed five times with one column volume of PBS each. The His-affinity gel was subsequently washed with PBS, 25 mM Imidazole to remove unspecifically bound proteins. Elution was performed in an second step with PBS, 500 mM Imidazole to elute the His-tagged viral vectors. The genomic titer of the purified viral vectors was detected via q-PCR. In an ELISA, viral vectors were captured employing the monoclonal antibody A20 (kindly provided by PD Dr. J. Kleinschmidt, DKFZ, Heidelberg) that exclusively recognizes assembled AAV capsids. His-Tags present in assembled viral capsids were subsequently detected with an HRP-tagged secondary anti-His-Tag antibody (1:2000 diluted, A7058, Sigma). HRP presence was detected using the peroxidase substrate ABTS. Generation of blue-green color (absorption at 405 nm) was measured in a Tecan Sunrise plate reader. Sample data were blanked with the average of the non-template controls (NTC).
Results and Discussion
Presence of the His-affinity tag in the viral capsid was detected and the ELISA enabled quantification of the purification procedure efficiency. The absorbance measured for the elution fractions of the 1/10 diluted samples sums up to 2.3 for the DMEM- and 0.5 for the Free Style 293-grown cells, assigning the DMEM-grown cells a five times higher production efficiency. Comparison between the cell pellet and the supernatant fractions revealed that 70 - 80% of the viral particles can be found inside the producer cells.
According to these results, producer cells should be grown in complex media for in vitro and cell culture experiments. Use of serum-free produced viral vectors is recommended for mouse or other animal experiments and possible therapeutical applications where even the presence of traces amounts of fetal calf serum should be avoided. Combination with different purification approaches such as gel filtration chromatography using i.e. Superdex 200 columns (GE Healthcare) enables the production of highly purified viral vectors for several different applications.
A: Schematic overview of the sandwich ELISA for the detection of His-tagged viral particles
B: ELISA from viral particles produced by AAV-293 cells in DMEM or Free Style medium, divided into cell pellet and cell culture supernatant samples. The particles were purified using Ni-NTA affinity chromatography with Imidazole in PBS as washing and elution agent
C: Absorption measurements from plate shown in B. Undiluted Äkta fractions converted ABTS peroxidase substrate at 405 nm
D: As C, whereas Äkta fractions were 10-fold diluted
Biotinylation Acceptor Peptide (BAP)
Schematic overview of BAP insertion into AAV-2 viral capsids, followed by biotinylation and binding of streptavidin-coupled molecules
The BAP (Biotinylation Acceptor Peptide) included in the Virus Construction Kit is a 15 amino acid peptide identified by Schatz et al. (1993) in a library screening approach and published under the number #85. This peptide with the sequence GLNDIFEAQKIEWHE contains a central lysine residue that can be specifically biotinylated by the prokaryotic holoenzyme biotin synthetase, encoded by the BirA gene of E. coli. Specific biotinylation of this peptide sequence can be performed in vivo by cotransfecting a plasmid with the BirA gene as described for the AAV by Arnold et al. (2006) or by an in vitro coupling approach using the purified Escherichia coli enzyme biotin ligase (BirA).
Biotin molecules specifically coupled to the viral loops can either be used to attach Streptavidin-coupled molecules to the viral capsid or to chemically couple molecules with a reactive group to biotin. Consequently, the inserted motif can enable the visualization of single virus particles by coupling fluorophores to the virus capsid empowering further uses of the Virus Construction Kit in fundamental virological research. In addition, targeting molecules such as Streptavidin-coupled affinity molecules (i.e. Antibodies, Nanobodies or Affibodies) can be coupled for manifold targeting approaches.
Materials and Methods: 
Transfection of the AAV-293 producer cells was performed in five 10 cm petri dishes with 3.4x10^6 cells (grown in DMEM supplemented with 10% FCS) resulting in a confluence of 70-80% according to the standard protocol. Producer cells were harvested and together with the culture supernatant subjected to four cycles of freeze and thaw cell lysis. The cell lysate was centrifuged at 4000 rpm for 15 minutes and concentrated in a VivaSpin VS2002 column with 10 kDa molecular weight cut-off to yield two milliliters. Genomic DNA attached to the virus particles was degraded using 250 units of Benzonase (EC 3.1.30.2, Sigma-Aldrich) at 37 °C for one hour. The concentrated cell lysate was washed three times with 4 ml 20 mM bis-Tris buffer pH 6.0, 100 mM NaCl. From this concentrated sample, 500 µl were loaded on the ÄKTA purifier (GE Healthcare) equipped with a Superdex 200 gel filtration column (GE Healthcare). This purification is also described by Smith et al. (2003). Fractions around the void volume giving a UV absorbance peak were pooled and applied to a Amicon Ultra column (Millipore, size limit 100 kDa, 2 ml loading volume) and concentrated to 500 µl. This purified virus sample was washed four times with 10 mM Tris-buffer pH 8.0 buffer which is recommended for the BirA biotin ligase that was kindly provided by Avidity LCC (Colorado, USA).
The BAP-containing viral vector sample (500 µl) was mixed according to the manufacturer protocol with each 72 µl Biomix A, Biomix B and Biotin. To reach maximal biotinylation, a volume of 5 µl containing 25000 units of the biotin ligase BirA was added to the reaction mixture and incubated for 6 h at 30 °C. In order to remove unbound biotin, the biotinylated viral vectors were washed five times with 10 mM Tris buffer pH 8.0.
Biotinylation was verified with an ELISA as depicted in figure A. For the detection of successfully biotinylated viral vectors, MaxiSorp 96-well plates (Nunc) were coated with 200 ng monoclonal A20 antibody per well for eight hours at 4°C, and blocked over night with PBST + 0.5 % BSA. This plate was incubated for 1 h at room temperature with 100 µl of serial diluted viral vectors. After washing the plates three times, the recommended amount of Streptavidin-HRP conjugate (Sigma-Aldrich) was added to each well for 1 h at room temperature. Again, the plates were washed three times followed by detection of absorbance caused by converted ABTS substrate at 405 nm. Additionally, the genomic AAV-2 titer was determined by qPCR.
 
A: Schematic overview of the detection of BAP-presenting viral particles immobilized by A20 capture antibodies. Biotinylated capsids can be detected by streptavidin-coupled HRP
B: ELISA AAV-2 particles carrying a BAP insertion in loop 587 which can be specifically biotinylated in vitro
Results and Discussion:
As can be seen in figure B, biotinylation of assembled AAV particles was achieved with the dilution series ranging from 2- to 128-fold. Correlating this assay with the qPCR experiment for the detection of encapsidated vector plasmid (genomic titer using CMV-promoter primers) yields a value of 4.70E+07 DNase-resistant particles (DRP). Consequently, the presence of approximately 3.6x10^5 biotinylated DRPs can be detected using the described ELISA. Compared to the virus particle detection ELISA emploing A20 as capture- and detection antibody, the detection limit of the biotinylated viral capsids is about 10fold more sensitive. This may be explained by the higher affinity of strepavidin towards biotin relative to the A20 antibody affinity or multiple biotinylation of a single virus particle.
Miniaturized antibody binding domain (Z34C)
Schematic overview of Z34C insertion into AAV-2 viral capsids enabling antibody arming for therapy
The idea of this targeting approach is to utilize a minimized fragment of the Staphylococcal Protein A  that was first described in Staphylococcus aureus. These gram-positive bacteria have evolved the 508 amino acid long protein A that has a high affinity for the Fc-domain of antibodies to protect itself from the immune system. Binding to the constant region of the antibodies is accomplished by the  Z-Domain  of Protein A that is 58-59 amino acids long, has alone a high affinity (Kd= 14,9 nM) for the antibodies and a three-helix bundle structure. In  [Braisted & Wells; 1996]  the authors reduced the secundary structure to an two-helix bundle. This size reduction has lead to an drastic reduction of the affinity for IgG (>10^5 fold) which could be recovered by 13 amino acid exchanges resulting in a 38 amino acid long peptide with an satisfying affinity for IgG (Kd = 185 nM) termed  Z38 . This binding domain was subsequently improved in  [Starovasnik et al.; 1997]  by the insertion of a disulfide bridge connecting the ends of the helices leading to the binding domain  Z34C  which shows an increased affinity for IgG (Kd = 20 nM).
 
Development of the Z34C motif, a miniaturized antibody binding motif
This engineered antibody binding domain of 34 amino acids was then inserted into capsids of different viral vectors amongst others also the AAV. In [Ried et al.; 2002]  the  Z34C  domain was inserted at position 587 into the capsid of the AAV resulting in viral vector that can be targeted to different target cells without genetic engineering. This targeting approach was then improved in  [Gigout et al.; 2005]  by the creation of mosaic vectors that contain only ~25% of recombinant VP-Proteins what resulted in 4 to 5 orders of magnitude more infectiosity compared to all-mutant viruses
Material and Methods:
Transfection of the AAV-293 producer cells was performed for three different loop insertions of Z34C in each three 10 cm petri dishes with 3.4x10^6 cells resulting in a confluence of about 70-80%. For the transfection, either the composite parts pCMV_VP123(453-Z34C), pCMV_VP123(587KO- Z34C) or pCMV_VP123(587Ko- Z34C-Spacer) were cotransfected with pHelper, [AAV2]-left-ITR_pCMV_betaglobin_mVenus_hGH_[AAV2]-right-ITR and RepVP123(587KO). The producer cells were harvested with the culture supernatant and subjected to four cycles of freeze and thaw cell lysis. The cell lysate was centrifuged at 4000 rpm for 15 minutes and concentrated in a VivaSpin VS2002 (Sartorius Stedem) with a 10 kDa molecular weight cut-off to two milliliters and washed three to five times with 20 mM Bis-Tris buffer, pH 6. The cell lysates were incubated with 250 units of Benzonase (Sigma-Aldrich) to remove contaminant genomic DNA and loaded on an ÄKTA purifier (GE Healthcare) equipped with a Superdex 200 gel filtration column (GE Healthcare). Fractions (500 µl) around the void volume containing viral particles were collected and used in a sandwich ELISA. 96 well plates were coated with 200 ng Cetuximab (Imclone/Merck/Bristol-Myers Squibb). Detection was performed using the monoclonal antibody A20 (kindly provided by PD Dr. J. Kleinschmidt, DKFZ Heidelberg) that was biotinylated using a Biotinylation kit (Dojindo, Japan) and Streptavidin-HRP (Sigma-Aldrich). The HRP presence was detected by the conversion of the substrate ABTS at 405 nm. The average of the non-template controls (NTC) was subtracted from the sample data.
Results and Discussion:
A: Sandwich-ELISA scheme for the detection of Z34C-presenting viral particles. Immobilization is achieved by binding a IgG molecule. Intact viral capsids can be specifically bound by the biotinylated A20 antibody which can be detected by Strepavidin-HRP
B: ELISA using the principle described in A for loop insertion samples obtained by gel filtration chromatography of 453- and 587-Z34C particles. Undiluted and 10-fold diluted samples were employed
C: ELISA signals obtained by absorbance measurements from B at 405 nm, undiluted samples
D: ELISA signals obtained by absorbance measurements from B at 405 nm, diluted samples
Three samples of different viral particles with insertions of the antibody-binding motif Z34C were successfully purified by the gel filtration chromatography. The fractions around the void volume were subsequently used in two different sandwich ELISAs. The first one aims at the detection of the particle’s ability to bind IgG-antibodies. The therapeutical antibody Cetuximab was employed to test the affinity of the virus particles. As displayed in figure 1A, only Z34C-presenting particles should be immobilized and only assembled viral capsid will be detected due to the affinity of the A20 antibody (as described above). Figure 1B shows that absorbance signal were obtained in case of the 587KO-Z34C and 587KO-Z34C-spacer insertions. Loop 453-Z34C did not yield any significant absorbance signals. In addition, the assay was performed with undiluted samples and those which were 10-fold diluted in PBST + 0.5 % BSA. Signal strengths in the undiluted ELISA samples indicated that the assay was saturated, therefore the diluted sample data were used for evaluation. The ABTS conversion indicates that highest amount of Z34C-presenting viral particles is present in the Äkta fractions 7-9 of the two different 587 insertions. No viral particles with a binding affinity for the IgG antibody were present in the 453 samples.
A: Sandwich-ELISA scheme for the detection of AAV-2 particles. Immobilization is achieved by binding to the A20 capture molecule which also acts as the biotinylated detection antibody after washing. Intact viral capsids can be specifically detected by Strepavidin-HRP
B: ELISA using the principle described in A for loop insertion samples obtained by gel filtration chromatography of 453- and 587-Z34C particles. Undiluted and 10-fold diluted samples were employed
C: qPCR data for the CMV-promoter-based DRP-titer determination. Samples as in B
D: ELISA signals obtained by absorbance measurements from B at 405 nm, diluted samples
A second approach was conducted to reveal the reason for the absence of Z34C-presenting particles in case of the 453 insertion. The sandwich ELISA, as shown in figure 2A, uses the A20 antibody to capture and detect all assembled viral capsids independent of the presence of a Z34C motif giving the so-called physical AAV titer (see figure 2D). To correlate these measurements, we additionally conducted qPCR measurements to determine the amount of encapsidated vector plasmids, the so-called DRP (DNAse-resistant particle) titer (see figure 2C). Comparison reveals that both assays yielded a peak signal around the Äkta fraction 8. The qPCR data indicated that all samples contain approximately 2 x 10^9 copies of the vector plasmid per milliliter. Assuming equal packaging efficiencies for the vector plasmids, all three loop insertion approaches contain comparable amounts of viral capsids. The sandwich-ELISA yielded a peak around fraction eight for all three loop-insertion approaches. Since the signal strength for the 587KO-Z34C samples is significantly higher, it can be assumed that additionally to the affinity of the A20 antibody for assembled viral capsids, the Z34C-containing viral particles add a high affinity for the Fc-part of the detection antibody. Hence the qPCR-titration only depends on the number of encapsidated AAV-vector plasmids and not on the viral capsid and its degree of modification, it is the more quantitative method of particle titration which proved equal for all three loop insertion approaches.
Summarizing, the insertion of a 34/39 amino acid functional motif into the two most promising integration sites of the AAV-2 capsid (see figure 3) revealed no integration of modified viral capsid proteins (VP1-3) into assembled viral capsids for the 453 integration site. As also seen by the data obtained for the linker-containing insertion, the 587 region tolerates insertions of at least 39 amino acids. For the 453 integration site, either the inserted Z34C motif loses its ability to bind IgG molecules or the influence of the inserted motif causes strong structural changes that disturb the ability of the modified VP proteins to be integrated into the virus capsid.
 
Loop insertion sites visualized in the 3D structure of a single viral coat protein (VP3)
 
Retrieved from " http://2010.igem.org/Team:Freiburg_Bioware/Project/insertion_of_motifs_into_surface-exposed_loops "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
