<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Bioware_NoteBook_Labjournal_July"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg_Bioware/NoteBook/Labjournal/July">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg_Bioware/NoteBook/Labjournal/July&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/July&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/July&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg_Bioware/NoteBook/Labjournal/July" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg Bioware/NoteBook/Labjournal/July</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="menu"><UL id="menu"><LI><DL><DT class="menu_button_left"><A href="https://2010.igem.org/Team:Freiburg_Bioware">Home</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Team_2010">Team</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Team_2010">Team 2010</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Photo_Gallery">Photo Gallery</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Cuckoo_Clock">Cuckoo Clock Competition</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Collaboration">Collaboration</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Example_Page">Example Page</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project">Project</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project">Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#highlights">Highlights</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#highlights">Results</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#modularisation">   Modularisation</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#targeting">   Targeting</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#arming">   Arming</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#superconstruct">   All-in-One</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Methods">Methods</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Virus_Construction_Kit">Virus Construction Kit Manual</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks">BioBricks</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks">Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/capsid_coding">Capsid Coding Parts</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/vector_plasmid">Vector Plasmid Parts</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/miscellaneous">Miscellaneous Parts</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A class="current" href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook">Notebook</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal">Labjournal</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Materials">Materials</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Methods">Methods</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Experiments">Experiments</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Meetings">Meetings</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Statistics">Statistics</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Sponsors">Sponsors</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Safety">Safety</A></DT></DL></LI><LI><DL><DT class="menu_button_right"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling">Modeling</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling">Modeling Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Virus_Production">Virus Production</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Virus_Infection">Virus Infection</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Structure_Modeling">Structure Modeling</A></DD></DL></LI></UL></DIV><DIV class="pageContent"><P><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook" class="external text" rel="nofollow">=&gt; Back to Notebook overview</A></P><DIV class="box_right"><LEFT><U1>NoteBook Navigator</U1></LEFT><UL><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/March">March (labday 1)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/April">April (labday 2 - 5)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/May">May (labday 6 - 17)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/June">June (labday 18 - 45)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/July">July (labday 46 - 75)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/August">August part 1 (labday 76 - 92)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/August2">August part 2 (labday 93 - 106)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/September">September part 1 (labday 107 - 123)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/September2">September part 2 (labday 124 - 135)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/October">October part 1 (labday 136 - 149 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/October2">October part 2 (labday 150 - 166 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/November">November  (labday 167 - 170 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/Cellculture">Cellculture</A></LI></UL></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Juli"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Juli</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#46.Labortag_01.07.2010"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">46.Labortag 01.07.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#qPCR"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">qPCR</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Thawing_HT1080_cells"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Thawing HT1080 cells</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Investigation_of_the_Kleinschmidt_sequencing"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">Investigation of the Kleinschmidt sequencing</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-6"><A href="#47.Labortag_02.07.2010"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">47.Labortag 02.07.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Fluoroscence_Pictures_of_Transduction_3"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Fluoroscence Pictures of Transduction 3</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Seeding_and_Splitting_HEK"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Seeding and Splitting HEK</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#New_ologos.2Fprimers_arrived"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">New ologos/primers arrived</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Annotation_of_the_453_insertion_site"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">Annotation of the 453 insertion site</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Searching_new_restriction_sites_in_the_585.2F588_WT_loop"><SPAN class="tocnumber">1.2.5</SPAN><SPAN class="toctext">Searching new restriction sites in the 585/588 WT loop</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Searching_new_restriction_sites_in_the_585.2F588_KO_loop"><SPAN class="tocnumber">1.2.6</SPAN><SPAN class="toctext">Searching new restriction sites in the 585/588 KO loop</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Searching_new_restriction_sites_in_the_453_loop"><SPAN class="tocnumber">1.2.7</SPAN><SPAN class="toctext">Searching new restriction sites in the 453 loop</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#48.Labortag_03.07.2010"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">48.Labortag 03.07.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Planing_the_gene_synthesis_to_replace_the_three_restriction_sites_in_the_rep_ORF"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Planing the gene synthesis to replace the three restriction sites in the rep ORF</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Modification_of_the_Cap_genes"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Modification of the Cap genes</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Cellculture"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">Cellculture</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#49.Labortag_04.07.2010:"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">49.Labortag 04.07.2010:</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Transfection_nr.3"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">Transfection nr.3</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#50.Labortag_05.07.2010:"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">50.Labortag 05.07.2010:</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#Sequencing_of_pAAV_iGEM-mVenus-YFP"><SPAN class="tocnumber">1.5.1</SPAN><SPAN class="toctext">Sequencing of pAAV_iGEM-mVenus-YFP</SPAN></A></LI><LI class="toclevel-3 tocsection-22"><A href="#Dirty_and_fancy_method_for_ITR_BioBrick_production"><SPAN class="tocnumber">1.5.2</SPAN><SPAN class="toctext">Dirty and fancy method for ITR BioBrick production</SPAN></A></LI><LI class="toclevel-3 tocsection-23"><A href="#Beta-globin_BioBrick_production"><SPAN class="tocnumber">1.5.3</SPAN><SPAN class="toctext">Beta-globin BioBrick production</SPAN></A></LI><LI class="toclevel-3 tocsection-24"><A href="#hGH_BioBrick_production"><SPAN class="tocnumber">1.5.4</SPAN><SPAN class="toctext">hGH BioBrick production</SPAN></A></LI><LI class="toclevel-3 tocsection-25"><A href="#Splitting_HT_and_HEK_cells"><SPAN class="tocnumber">1.5.5</SPAN><SPAN class="toctext">Splitting HT and HEK cells</SPAN></A></LI><LI class="toclevel-3 tocsection-26"><A href="#Preparation_of_HT1080_3x6er_dishes_for_Transduction"><SPAN class="tocnumber">1.5.6</SPAN><SPAN class="toctext">Preparation of HT1080 3x6er dishes for Transduction</SPAN></A></LI><LI class="toclevel-3 tocsection-27"><A href="#Preparation_of_6x10cm_petri_dishes_HEK293"><SPAN class="tocnumber">1.5.7</SPAN><SPAN class="toctext">Preparation of 6x10cm  petri dishes HEK293</SPAN></A></LI><LI class="toclevel-3 tocsection-28"><A href="#Repetition_of_ITR_restriction_site_modification_via_PCR"><SPAN class="tocnumber">1.5.8</SPAN><SPAN class="toctext">Repetition of ITR restriction site modification via PCR</SPAN></A></LI><LI class="toclevel-3 tocsection-29"><A href="#Search_for_restriction_sites_in_the_sequence_LQA.2FRGQA.2FRQA"><SPAN class="tocnumber">1.5.9</SPAN><SPAN class="toctext">Search for restriction sites in the sequence LQA/RGQA/RQA</SPAN></A></LI><LI class="toclevel-3 tocsection-30"><A href="#Results_sequencing_alignment_of_GMK_TK30"><SPAN class="tocnumber">1.5.10</SPAN><SPAN class="toctext">Results sequencing alignment of GMK_TK30</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#51.Labortag_6.07.2010:"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">51.Labortag 6.07.2010:</SPAN></A></LI><LI class="toclevel-3 tocsection-32"><A href="#Harvesting_AAVs_from_4_10cm_dishes.2C_Transduction_of_3x6er_dishes"><SPAN class="tocnumber">1.6.1</SPAN><SPAN class="toctext">Harvesting AAVs from 4 10cm dishes, Transduction of 3x6er dishes</SPAN></A></LI><LI class="toclevel-3 tocsection-33"><A href="#Search_for_restriction_sites_before_and_after_the_insertion_sites_453_and_585.2F588"><SPAN class="tocnumber">1.6.2</SPAN><SPAN class="toctext">Search for restriction sites before and after the insertion sites 453 and 585/588</SPAN></A></LI><LI class="toclevel-3 tocsection-34"><A href="#Transduction_of_3x6er_dishes"><SPAN class="tocnumber">1.6.3</SPAN><SPAN class="toctext">Transduction of 3x6er dishes</SPAN></A></LI><LI class="toclevel-3 tocsection-35"><A href="#Search_for_restriction_sites_around_the_453_and_585.2F588_integration_sites"><SPAN class="tocnumber">1.6.4</SPAN><SPAN class="toctext">Search for restriction sites around the 453 and 585/588 integration sites</SPAN></A></LI><LI class="toclevel-3 tocsection-36"><A href="#Search_for_restriction_sites_in_the_sequence_LQA.2FRGQA.2FRQA_2"><SPAN class="tocnumber">1.6.5</SPAN><SPAN class="toctext">Search for restriction sites in the sequence LQA/RGQA/RQA</SPAN></A></LI><LI class="toclevel-2 tocsection-37"><A href="#52.Labortag_07.07.2010:"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">52.Labortag 07.07.2010:</SPAN></A></LI><LI class="toclevel-3 tocsection-38"><A href="#Sequencing_of_GIO"><SPAN class="tocnumber">1.7.1</SPAN><SPAN class="toctext">Sequencing of GIO</SPAN></A></LI><LI class="toclevel-3 tocsection-39"><A href="#Interview_with_Bild_der_Wissenschaft"><SPAN class="tocnumber">1.7.2</SPAN><SPAN class="toctext">Interview with Bild der Wissenschaft</SPAN></A></LI><LI class="toclevel-3 tocsection-40"><A href="#Transfection"><SPAN class="tocnumber">1.7.3</SPAN><SPAN class="toctext">Transfection</SPAN></A></LI><LI class="toclevel-3 tocsection-41"><A href="#New_pcr_run_with_new_polymerase_and_different_MgSO4_concentrations"><SPAN class="tocnumber">1.7.4</SPAN><SPAN class="toctext">New pcr run with new polymerase and different MgSO4 concentrations</SPAN></A></LI><LI class="toclevel-3 tocsection-42"><A href="#Checking_the_results_of_the_following_sequencing_reactions_for_the_expression_constructs_from_the_DKFZ"><SPAN class="tocnumber">1.7.5</SPAN><SPAN class="toctext">Checking the results of the following sequencing reactions for the expression constructs from the DKFZ</SPAN></A></LI><LI class="toclevel-2 tocsection-43"><A href="#53.Labortag_08.07.2010:"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">53.Labortag 08.07.2010:</SPAN></A></LI><LI class="toclevel-3 tocsection-44"><A href="#Final_planing_of_the_Cap_modifications"><SPAN class="tocnumber">1.8.1</SPAN><SPAN class="toctext">Final planing of the Cap modifications</SPAN></A></LI><LI class="toclevel-2 tocsection-45"><A href="#54.Labortag_09.07.2010:"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">54.Labortag 09.07.2010:</SPAN></A></LI><LI class="toclevel-3 tocsection-46"><A href="#PCR_of_hgH"><SPAN class="tocnumber">1.9.1</SPAN><SPAN class="toctext">PCR of hgH</SPAN></A></LI><LI class="toclevel-3 tocsection-47"><A href="#Order_oligos_for_GMK_TK30"><SPAN class="tocnumber">1.9.2</SPAN><SPAN class="toctext">Order oligos for GMK_TK30</SPAN></A></LI><LI class="toclevel-3 tocsection-48"><A href="#Quickchange_for_the_PstI_restriction_sites_in_pAAV-RC"><SPAN class="tocnumber">1.9.3</SPAN><SPAN class="toctext">Quickchange for the PstI restriction sites in pAAV-RC</SPAN></A></LI><LI class="toclevel-2 tocsection-49"><A href="#55.Labortag_11.07.2010"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">55.Labortag 11.07.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-50"><A href="#Splitting_of_HT1080_cells"><SPAN class="tocnumber">1.10.1</SPAN><SPAN class="toctext">Splitting of HT1080 cells</SPAN></A></LI><LI class="toclevel-2 tocsection-51"><A href="#56.Labortag_12.07.2010:"><SPAN class="tocnumber">1.11</SPAN><SPAN class="toctext">56.Labortag 12.07.2010:</SPAN></A></LI><LI class="toclevel-3 tocsection-52"><A href="#Cloning_CFP_into_pSB1C3"><SPAN class="tocnumber">1.11.1</SPAN><SPAN class="toctext">Cloning CFP into pSB1C3</SPAN></A></LI><LI class="toclevel-3 tocsection-53"><A href="#Seeding_HT1080_.2B_Passage"><SPAN class="tocnumber">1.11.2</SPAN><SPAN class="toctext">Seeding HT1080 + Passage</SPAN></A></LI><LI class="toclevel-3 tocsection-54"><A href="#PCR_of_beta-globin:"><SPAN class="tocnumber">1.11.3</SPAN><SPAN class="toctext">PCR of beta-globin:</SPAN></A></LI><LI class="toclevel-3 tocsection-55"><A href="#Midi-Prep_preparation_pAAV_RC"><SPAN class="tocnumber">1.11.4</SPAN><SPAN class="toctext">Midi-Prep preparation pAAV_RC</SPAN></A></LI><LI class="toclevel-3 tocsection-56"><A href="#First_steps_in_producing_ITR_BioBricks_via_Fancy_Method"><SPAN class="tocnumber">1.11.5</SPAN><SPAN class="toctext">First steps in producing ITR BioBricks via Fancy Method</SPAN></A></LI><LI class="toclevel-3 tocsection-57"><A href="#Ordering_genes_for_the_replacement_of_Rep_and_Cap_parts"><SPAN class="tocnumber">1.11.6</SPAN><SPAN class="toctext">Ordering genes for the replacement of Rep and Cap parts</SPAN></A></LI><LI class="toclevel-3 tocsection-58"><A href="#Sequencing_of_GOI"><SPAN class="tocnumber">1.11.7</SPAN><SPAN class="toctext">Sequencing of GOI</SPAN></A></LI><LI class="toclevel-2 tocsection-59"><A href="#57.Labortag_13.07.2010:"><SPAN class="tocnumber">1.12</SPAN><SPAN class="toctext">57.Labortag 13.07.2010:</SPAN></A></LI><LI class="toclevel-3 tocsection-60"><A href="#Endotoxin-free_Midi_Prep_of_pAAV_RC"><SPAN class="tocnumber">1.12.1</SPAN><SPAN class="toctext">Endotoxin-free Midi Prep of pAAV_RC</SPAN></A></LI><LI class="toclevel-3 tocsection-61"><A href="#HEK-293_cells"><SPAN class="tocnumber">1.12.2</SPAN><SPAN class="toctext">HEK-293 cells</SPAN></A></LI><LI class="toclevel-3 tocsection-62"><A href="#Oligos_pMGMK_TK30_arrived"><SPAN class="tocnumber">1.12.3</SPAN><SPAN class="toctext">Oligos pMGMK_TK30 arrived</SPAN></A></LI><LI class="toclevel-3 tocsection-63"><A href="#Transduction"><SPAN class="tocnumber">1.12.4</SPAN><SPAN class="toctext">Transduction</SPAN></A></LI><LI class="toclevel-3 tocsection-64"><A href="#Results_of_Trafo_of_cloning_CFP_into_pSB1C3"><SPAN class="tocnumber">1.12.5</SPAN><SPAN class="toctext">Results of Trafo of cloning CFP into pSB1C3</SPAN></A></LI><LI class="toclevel-3 tocsection-65"><A href="#Repetition_of_cloning_of_CFP_into_pSB1C3"><SPAN class="tocnumber">1.12.6</SPAN><SPAN class="toctext">Repetition of cloning of CFP into pSB1C3</SPAN></A></LI><LI class="toclevel-3 tocsection-66"><A href="#TRANSFORMATION_of_Quickchange_PstI"><SPAN class="tocnumber">1.12.7</SPAN><SPAN class="toctext">TRANSFORMATION of Quickchange PstI</SPAN></A></LI><LI class="toclevel-2 tocsection-67"><A href="#58.Labortag_14.07.2010"><SPAN class="tocnumber">1.13</SPAN><SPAN class="toctext">58.Labortag 14.07.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-68"><A href="#Trafo_evaluation_of_Quickchange_PstI"><SPAN class="tocnumber">1.13.1</SPAN><SPAN class="toctext">Trafo evaluation of Quickchange PstI</SPAN></A></LI><LI class="toclevel-3 tocsection-69"><A href="#Quickchange_for_the_PstI_restriction_sites_in_pAAV-RC_.28modified.29"><SPAN class="tocnumber">1.13.2</SPAN><SPAN class="toctext">Quickchange for the PstI restriction sites in pAAV-RC (modified)</SPAN></A></LI><LI class="toclevel-3 tocsection-70"><A href="#Results_of_trafo_cloning_pSB1C3_CFP"><SPAN class="tocnumber">1.13.3</SPAN><SPAN class="toctext">Results of trafo cloning pSB1C3_CFP</SPAN></A></LI><LI class="toclevel-3 tocsection-71"><A href="#Seed_AAV-293"><SPAN class="tocnumber">1.13.4</SPAN><SPAN class="toctext">Seed AAV-293</SPAN></A></LI><LI class="toclevel-3 tocsection-72"><A href="#DAAD_Timetable"><SPAN class="tocnumber">1.13.5</SPAN><SPAN class="toctext">DAAD Timetable</SPAN></A></LI><LI class="toclevel-3 tocsection-73"><A href="#Summary"><SPAN class="tocnumber">1.13.6</SPAN><SPAN class="toctext">Summary</SPAN></A></LI><LI class="toclevel-3 tocsection-74"><A href="#Ordering_ITR.2BTK.2FGMK"><SPAN class="tocnumber">1.13.7</SPAN><SPAN class="toctext">Ordering ITR+TK/GMK</SPAN></A></LI><LI class="toclevel-3 tocsection-75"><A href="#Endotoxoin_free_Midi_prep_mAAV_RC"><SPAN class="tocnumber">1.13.8</SPAN><SPAN class="toctext">Endotoxoin free Midi prep mAAV_RC</SPAN></A></LI><LI class="toclevel-3 tocsection-76"><A href="#MGMK_TK30:PCR_with_iGEM_restriction-sites_and_cloning_into_pAAV_RFC25"><SPAN class="tocnumber">1.13.9</SPAN><SPAN class="toctext">MGMK_TK30:PCR with iGEM restriction-sites and cloning into pAAV_RFC25</SPAN></A></LI><LI class="toclevel-3 tocsection-77"><A href="#Ordering_of_Oligos"><SPAN class="tocnumber">1.13.10</SPAN><SPAN class="toctext">Ordering of Oligos</SPAN></A></LI><LI class="toclevel-2 tocsection-78"><A href="#59.Labortag_15.07.2010"><SPAN class="tocnumber">1.14</SPAN><SPAN class="toctext">59.Labortag 15.07.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-79"><A href="#Picking_clones_for_mini-prep"><SPAN class="tocnumber">1.14.1</SPAN><SPAN class="toctext">Picking clones for mini-prep</SPAN></A></LI><LI class="toclevel-3 tocsection-80"><A href="#Splitting_of_HT1080_cells_2"><SPAN class="tocnumber">1.14.2</SPAN><SPAN class="toctext">Splitting of HT1080 cells</SPAN></A></LI><LI class="toclevel-3 tocsection-81"><A href="#PstI_Quickchange_Trafo_from_14.07"><SPAN class="tocnumber">1.14.3</SPAN><SPAN class="toctext">PstI Quickchange Trafo from 14.07</SPAN></A></LI><LI class="toclevel-3 tocsection-82"><A href="#MINI_PREP_.28PstI_Quickchange_Trafo_from_13.07.29"><SPAN class="tocnumber">1.14.4</SPAN><SPAN class="toctext">MINI PREP (PstI Quickchange Trafo from 13.07)</SPAN></A></LI><LI class="toclevel-3 tocsection-83"><A href="#TEST_DIGESTION_.28PstI_Quickchange_Trafo_from_13.07.29"><SPAN class="tocnumber">1.14.5</SPAN><SPAN class="toctext">TEST DIGESTION (PstI Quickchange Trafo from 13.07)</SPAN></A></LI><LI class="toclevel-3 tocsection-84"><A href="#FACS"><SPAN class="tocnumber">1.14.6</SPAN><SPAN class="toctext">FACS</SPAN></A></LI><LI class="toclevel-3 tocsection-85"><A href="#Plasmid_Mini-Prep_of_pSB1C3_CFP"><SPAN class="tocnumber">1.14.7</SPAN><SPAN class="toctext">Plasmid Mini-Prep of pSB1C3_CFP</SPAN></A></LI><LI class="toclevel-3 tocsection-86"><A href="#2nd_try:_First_steps_in_producing_ITR_BioBricks_via_Fancy_Method"><SPAN class="tocnumber">1.14.8</SPAN><SPAN class="toctext">2nd try: First steps in producing ITR BioBricks via Fancy Method</SPAN></A></LI><LI class="toclevel-2 tocsection-87"><A href="#60.Labortag_16.07.2010"><SPAN class="tocnumber">1.15</SPAN><SPAN class="toctext">60.Labortag 16.07.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-88"><A href="#Continuation_of_Fancy_ITR-method"><SPAN class="tocnumber">1.15.1</SPAN><SPAN class="toctext">Continuation of Fancy ITR-method</SPAN></A></LI><LI class="toclevel-3 tocsection-89"><A href="#Results_sequencing_of_pSB1C3_CFP"><SPAN class="tocnumber">1.15.2</SPAN><SPAN class="toctext">Results sequencing of pSB1C3_CFP</SPAN></A></LI><LI class="toclevel-3 tocsection-90"><A href="#BioBrick_production_with_hGH_and_beta-globin_by_cloning_it_into_pSB1C3"><SPAN class="tocnumber">1.15.3</SPAN><SPAN class="toctext">BioBrick production with hGH and beta-globin by cloning it into pSB1C3</SPAN></A></LI><LI class="toclevel-3 tocsection-91"><A href="#Transfection_2"><SPAN class="tocnumber">1.15.4</SPAN><SPAN class="toctext">Transfection</SPAN></A></LI><LI class="toclevel-3 tocsection-92"><A href="#Mini_Prep_of_pAAV_RFC25_mgmk_TK30"><SPAN class="tocnumber">1.15.5</SPAN><SPAN class="toctext">Mini Prep of pAAV_RFC25_mgmk_TK30</SPAN></A></LI><LI class="toclevel-3 tocsection-93"><A href="#Preparation_of_50x_TAE_and_0.2C5M_EDTA"><SPAN class="tocnumber">1.15.6</SPAN><SPAN class="toctext">Preparation of 50x TAE and 0,5M EDTA</SPAN></A></LI><LI class="toclevel-3 tocsection-94"><A href="#Test_digestion_of_the_QuickChange_for_PstI_.28310.29_and_PstI_.284073.29"><SPAN class="tocnumber">1.15.7</SPAN><SPAN class="toctext">Test digestion of the QuickChange for PstI (310) and PstI (4073)</SPAN></A></LI><LI class="toclevel-3 tocsection-95"><A href="#Primers_to_convert_the_expression_constructs_into_the_iGEM_standard_were_ordered"><SPAN class="tocnumber">1.15.8</SPAN><SPAN class="toctext">Primers to convert the expression constructs into the iGEM standard were ordered</SPAN></A></LI><LI class="toclevel-2 tocsection-96"><A href="#61.Labortag_17.07.2010:"><SPAN class="tocnumber">1.16</SPAN><SPAN class="toctext">61.Labortag 17.07.2010:</SPAN></A></LI><LI class="toclevel-3 tocsection-97"><A href="#Picking_clones_of_pGA14_Pr.C3.A4fix-leftITR_Trafo"><SPAN class="tocnumber">1.16.1</SPAN><SPAN class="toctext">Picking clones of pGA14_Präfix-leftITR Trafo</SPAN></A></LI><LI class="toclevel-3 tocsection-98"><A href="#Mini-Preps_of_pGA14_Pr.C3.A4fix-lITR_cultures"><SPAN class="tocnumber">1.16.2</SPAN><SPAN class="toctext">Mini-Preps of pGA14_Präfix-lITR cultures</SPAN></A></LI><LI class="toclevel-3 tocsection-99"><A href="#Test_digestion_of_pGA14_Pr.C3.A4fix-lITR_.28Ligation.29"><SPAN class="tocnumber">1.16.3</SPAN><SPAN class="toctext">Test digestion of pGA14_Präfix-lITR (Ligation)</SPAN></A></LI><LI class="toclevel-3 tocsection-100"><A href="#Trafo_of_pGA14_Pr.C3.A4fix_rITR"><SPAN class="tocnumber">1.16.4</SPAN><SPAN class="toctext">Trafo of pGA14_Präfix_rITR</SPAN></A></LI><LI class="toclevel-2 tocsection-101"><A href="#62.Labortag_18.07.2010"><SPAN class="tocnumber">1.17</SPAN><SPAN class="toctext">62.Labortag 18.07.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-102"><A href="#Cellculture_2"><SPAN class="tocnumber">1.17.1</SPAN><SPAN class="toctext">Cellculture</SPAN></A></LI><LI class="toclevel-3 tocsection-103"><A href="#Test_digestion_of_pAAV_RFC_mgmk_tk30:_P54.2C_P55.2C_P56.2C_P57"><SPAN class="tocnumber">1.17.2</SPAN><SPAN class="toctext">Test digestion of pAAV_RFC_mgmk_tk30: P54, P55, P56, P57</SPAN></A></LI><LI class="toclevel-2 tocsection-104"><A href="#63._Labortag_19.07.2010"><SPAN class="tocnumber">1.18</SPAN><SPAN class="toctext">63. Labortag 19.07.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-105"><A href="#rITR"><SPAN class="tocnumber">1.18.1</SPAN><SPAN class="toctext">rITR</SPAN></A></LI><LI class="toclevel-3 tocsection-106"><A href="#Mini-Preps_of_pGA14_Pr.C3.A4fix-lITR_cultures_2"><SPAN class="tocnumber">1.18.2</SPAN><SPAN class="toctext">Mini-Preps of pGA14_Präfix-lITR cultures</SPAN></A></LI><LI class="toclevel-3 tocsection-107"><A href="#Test_digestion_of_pGA14_Pr.C3.A4fix-rITR_.28Ligation.29"><SPAN class="tocnumber">1.18.3</SPAN><SPAN class="toctext">Test digestion of pGA14_Präfix-rITR (Ligation)</SPAN></A></LI><LI class="toclevel-3 tocsection-108"><A href="#Sequencing_of_ITRs"><SPAN class="tocnumber">1.18.4</SPAN><SPAN class="toctext">Sequencing of ITRs</SPAN></A></LI><LI class="toclevel-3 tocsection-109"><A href="#Quantitative_real_time_PCR"><SPAN class="tocnumber">1.18.5</SPAN><SPAN class="toctext">Quantitative real time PCR</SPAN></A></LI><LI class="toclevel-3 tocsection-110"><A href="#Cloning_strategy_for_gmk_tk_added.21.21"><SPAN class="tocnumber">1.18.6</SPAN><SPAN class="toctext">Cloning strategy for gmk_tk added!!</SPAN></A></LI><LI class="toclevel-3 tocsection-111"><A href="#pAAV_RC_site_directed_mutagenesis:_Digestion_of_the_two_mutagenesis_reaction_of_PstI_site:_310_and_4073"><SPAN class="tocnumber">1.18.7</SPAN><SPAN class="toctext">pAAV_RC site directed mutagenesis: Digestion of the two mutagenesis reaction of PstI site: 310 and 4073</SPAN></A></LI><LI class="toclevel-3 tocsection-112"><A href="#pHelper_plasmid_for_Midi-Prep_prepared"><SPAN class="tocnumber">1.18.8</SPAN><SPAN class="toctext">pHelper plasmid for Midi-Prep prepared</SPAN></A></LI><LI class="toclevel-3 tocsection-113"><A href="#Trafo_of_IGEM_2009_linkers"><SPAN class="tocnumber">1.18.9</SPAN><SPAN class="toctext">Trafo of IGEM 2009 linkers</SPAN></A></LI><LI class="toclevel-2 tocsection-114"><A href="#64._Labortag_20.07.2010"><SPAN class="tocnumber">1.19</SPAN><SPAN class="toctext">64. Labortag 20.07.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-115"><A href="#Quickchange_site_directed_mutagenesis:"><SPAN class="tocnumber">1.19.1</SPAN><SPAN class="toctext">Quickchange site directed mutagenesis:</SPAN></A></LI><LI class="toclevel-3 tocsection-116"><A href="#Endotoxin-free_Midi_Prep_of_pHelper"><SPAN class="tocnumber">1.19.2</SPAN><SPAN class="toctext">Endotoxin-free Midi Prep of pHelper</SPAN></A></LI><LI class="toclevel-3 tocsection-117"><A href="#Quantitative_real_time_PCR_2"><SPAN class="tocnumber">1.19.3</SPAN><SPAN class="toctext">Quantitative real time PCR</SPAN></A></LI><LI class="toclevel-3 tocsection-118"><A href="#Cloning_of_p51.28pSB1C3_RFC25.29_for_His-tag_insert"><SPAN class="tocnumber">1.19.4</SPAN><SPAN class="toctext">Cloning of p51(pSB1C3_RFC25) for His-tag insert</SPAN></A></LI><LI class="toclevel-2 tocsection-119"><A href="#65._Labortag_21.07.2010"><SPAN class="tocnumber">1.20</SPAN><SPAN class="toctext">65. Labortag 21.07.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-120"><A href="#Sequencing_of_ITRs_2"><SPAN class="tocnumber">1.20.1</SPAN><SPAN class="toctext">Sequencing of ITRs</SPAN></A></LI><LI class="toclevel-3 tocsection-121"><A href="#Quantitative_real_time_PCR_3"><SPAN class="tocnumber">1.20.2</SPAN><SPAN class="toctext">Quantitative real time PCR</SPAN></A></LI><LI class="toclevel-3 tocsection-122"><A href="#Mini-Preps_of_linker"><SPAN class="tocnumber">1.20.3</SPAN><SPAN class="toctext">Mini-Preps of linker</SPAN></A></LI><LI class="toclevel-3 tocsection-123"><A href="#Results_of_Trafo_of_cloning_pAAV_RC_SDM_of_PstI_310_and_4073"><SPAN class="tocnumber">1.20.4</SPAN><SPAN class="toctext">Results of Trafo of cloning pAAV_RC SDM of PstI 310 and 4073</SPAN></A></LI><LI class="toclevel-3 tocsection-124"><A href="#PCR_with_pUB_V5_mGMK_TK30"><SPAN class="tocnumber">1.20.5</SPAN><SPAN class="toctext">PCR with pUB_V5_mGMK_TK30</SPAN></A></LI><LI class="toclevel-3 tocsection-125"><A href="#Gelextraction.2C_Ligation_and_Transformation_for_cloning_of_pSB1C3_His_RFC25_.28continuation_of_cloning_from_64th_lab_day.29"><SPAN class="tocnumber">1.20.6</SPAN><SPAN class="toctext">Gelextraction, Ligation and Transformation for cloning of pSB1C3_His_RFC25 (continuation of cloning from 64th lab day)</SPAN></A></LI><LI class="toclevel-3 tocsection-126"><A href="#Picking_clones_pf_VP1-3_pEX_plasmids_from_DKFZ"><SPAN class="tocnumber">1.20.7</SPAN><SPAN class="toctext">Picking clones pf VP1-3 pEX plasmids from DKFZ</SPAN></A></LI><LI class="toclevel-2 tocsection-127"><A href="#66._Labortag_22.7.2010"><SPAN class="tocnumber">1.21</SPAN><SPAN class="toctext">66. Labortag 22.7.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-128"><A href="#cell_culture"><SPAN class="tocnumber">1.21.1</SPAN><SPAN class="toctext">cell culture</SPAN></A></LI><LI class="toclevel-3 tocsection-129"><A href="#Mini_Prep_and_test_digestion_pKEX_VP1-3_w.2Fo_PstI.284073.29"><SPAN class="tocnumber">1.21.2</SPAN><SPAN class="toctext">Mini Prep and test digestion  pKEX_VP1-3 w/o PstI(4073)</SPAN></A></LI><LI class="toclevel-3 tocsection-130"><A href="#Cloning_together_of_pAAV_RC_PstI_310_and_4073_with_two_enzymes"><SPAN class="tocnumber">1.21.3</SPAN><SPAN class="toctext">Cloning together of pAAV_RC PstI 310 and 4073 with two enzymes</SPAN></A></LI><LI class="toclevel-3 tocsection-131"><A href="#Sequencing_of_linker"><SPAN class="tocnumber">1.21.4</SPAN><SPAN class="toctext">Sequencing of linker</SPAN></A></LI><LI class="toclevel-3 tocsection-132"><A href="#Biobricks_of_hgH_and_.C3.9F-Globin"><SPAN class="tocnumber">1.21.5</SPAN><SPAN class="toctext">Biobricks of hgH and ß-Globin</SPAN></A></LI><LI class="toclevel-3 tocsection-133"><A href="#Picking_clones_of_pSB1C3_His_RFC25_and_pAAV_RFC25_GMK_TK30_for_Mini-Prep"><SPAN class="tocnumber">1.21.6</SPAN><SPAN class="toctext">Picking clones of pSB1C3_His_RFC25 and pAAV_RFC25_GMK_TK30 for Mini-Prep</SPAN></A></LI><LI class="toclevel-2 tocsection-134"><A href="#67._Labortag_23.7.2010"><SPAN class="tocnumber">1.22</SPAN><SPAN class="toctext">67. Labortag 23.7.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-135"><A href="#Picking_clones_of_pAAV_RC_1.1"><SPAN class="tocnumber">1.22.1</SPAN><SPAN class="toctext">Picking clones of pAAV_RC_1.1</SPAN></A></LI><LI class="toclevel-3 tocsection-136"><A href="#Results_of_Trafo_Biobricks_of_hgH_and_.C3.9F-Globin"><SPAN class="tocnumber">1.22.2</SPAN><SPAN class="toctext">Results of Trafo Biobricks of hgH and ß-Globin</SPAN></A></LI><LI class="toclevel-3 tocsection-137"><A href="#Mini-Prep_and_Test_digestion_of_pAAV_RFC25_mGMK_TK30"><SPAN class="tocnumber">1.22.3</SPAN><SPAN class="toctext">Mini-Prep and Test digestion of pAAV_RFC25_mGMK_TK30</SPAN></A></LI><LI class="toclevel-3 tocsection-138"><A href="#Mini-Prep_of_pSB1C3_His6"><SPAN class="tocnumber">1.22.4</SPAN><SPAN class="toctext">Mini-Prep of pSB1C3_His6</SPAN></A></LI><LI class="toclevel-2 tocsection-139"><A href="#68._Labortag_24.7.2010"><SPAN class="tocnumber">1.23</SPAN><SPAN class="toctext">68. Labortag 24.7.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-140"><A href="#Cell_culture:_Seeding_of_HEK293_for_Transfektion_with_P81.2C_P82_and_P83"><SPAN class="tocnumber">1.23.1</SPAN><SPAN class="toctext">Cell culture: Seeding of HEK293 for Transfektion with P81, P82 and P83</SPAN></A></LI><LI class="toclevel-3 tocsection-141"><A href="#Cell_culture:_Transduction_of_HT1080_with_AAVs"><SPAN class="tocnumber">1.23.2</SPAN><SPAN class="toctext">Cell culture: Transduction of HT1080 with AAVs</SPAN></A></LI><LI class="toclevel-3 tocsection-142"><A href="#Discussion_round_nr._4"><SPAN class="tocnumber">1.23.3</SPAN><SPAN class="toctext">Discussion round nr. 4</SPAN></A></LI><LI class="toclevel-3 tocsection-143"><A href="#Stocktaking_of_superior_goods"><SPAN class="tocnumber">1.23.4</SPAN><SPAN class="toctext">Stocktaking of superior goods</SPAN></A></LI><LI class="toclevel-2 tocsection-144"><A href="#69._Labortag_25.7.2010"><SPAN class="tocnumber">1.24</SPAN><SPAN class="toctext">69. Labortag 25.7.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-145"><A href="#Clone_picking_from_pSB1C3_hgH_and_pSB1C3_betaglobin"><SPAN class="tocnumber">1.24.1</SPAN><SPAN class="toctext">Clone picking from pSB1C3_hgH and pSB1C3_betaglobin</SPAN></A></LI><LI class="toclevel-2 tocsection-146"><A href="#70._Labortag_26.7.2010"><SPAN class="tocnumber">1.25</SPAN><SPAN class="toctext">70. Labortag 26.7.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-147"><A href="#Mini_Prep_and_test_digestion_pSB1C3_hgh_and_pSB1C3_beta-globin"><SPAN class="tocnumber">1.25.1</SPAN><SPAN class="toctext">Mini Prep and test digestion  pSB1C3_hgh and pSB1C3_beta-globin</SPAN></A></LI><LI class="toclevel-3 tocsection-148"><A href="#Test_digestion_P87:_pAAV_RC_PstI-301-4073_clone_1_and_P88:_pAAV_RC_PstI-301-4073_clone_2"><SPAN class="tocnumber">1.25.2</SPAN><SPAN class="toctext">Test digestion  P87: pAAV_RC_PstI-301-4073 clone 1 and P88: pAAV_RC_PstI-301-4073 clone 2</SPAN></A></LI><LI class="toclevel-3 tocsection-149"><A href="#Oligos_for_hTERT_promoter"><SPAN class="tocnumber">1.25.3</SPAN><SPAN class="toctext">Oligos for hTERT promoter</SPAN></A></LI><LI class="toclevel-3 tocsection-150"><A href="#Retrafo_with_pTVvector_containing_the_CMV_promoter"><SPAN class="tocnumber">1.25.4</SPAN><SPAN class="toctext">Retrafo with pTVvector containing the CMV promoter</SPAN></A></LI><LI class="toclevel-3 tocsection-151"><A href="#Continuation:_ITRs"><SPAN class="tocnumber">1.25.5</SPAN><SPAN class="toctext">Continuation: ITRs</SPAN></A></LI><LI class="toclevel-3 tocsection-152"><A href="#Biobrick_of_GMK"><SPAN class="tocnumber">1.25.6</SPAN><SPAN class="toctext">Biobrick of GMK</SPAN></A></LI><LI class="toclevel-2 tocsection-153"><A href="#71._Labortag_27.7.2010"><SPAN class="tocnumber">1.26</SPAN><SPAN class="toctext">71. Labortag 27.7.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-154"><A href="#Cloning_of_linkers_in_pSB1C3_RFC25_CFP1.1"><SPAN class="tocnumber">1.26.1</SPAN><SPAN class="toctext">Cloning of linkers in pSB1C3_RFC25_CFP1.1</SPAN></A></LI><LI class="toclevel-3 tocsection-155"><A href="#Result_sequencing_of_pAAV_RFC25_mGMK_TK30"><SPAN class="tocnumber">1.26.2</SPAN><SPAN class="toctext">Result sequencing of pAAV_RFC25_mGMK_TK30</SPAN></A></LI><LI class="toclevel-3 tocsection-156"><A href="#Quickchange_.40_Pst_in_RepX_68_and_RepX_78"><SPAN class="tocnumber">1.26.3</SPAN><SPAN class="toctext">Quickchange @ Pst in RepX 68 and RepX 78</SPAN></A></LI><LI class="toclevel-3 tocsection-157"><A href="#Site-directed_mutagenesis_of_pAAV_RC_PstI_310.2B4073_.28pAAV_RC_1.1.29_with_BamHI_and_SalI"><SPAN class="tocnumber">1.26.4</SPAN><SPAN class="toctext">Site-directed mutagenesis of pAAV_RC_PstI_310+4073 (pAAV_RC_1.1) with BamHI and SalI</SPAN></A></LI><LI class="toclevel-3 tocsection-158"><A href="#Cell_culture_2"><SPAN class="tocnumber">1.26.5</SPAN><SPAN class="toctext">Cell culture</SPAN></A></LI><LI class="toclevel-3 tocsection-159"><A href="#ITRs:_Trafo_of_overnight-ligation_approaches"><SPAN class="tocnumber">1.26.6</SPAN><SPAN class="toctext">ITRs: Trafo of overnight-ligation approaches</SPAN></A></LI><LI class="toclevel-3 tocsection-160"><A href="#PCR:_beta-Globin"><SPAN class="tocnumber">1.26.7</SPAN><SPAN class="toctext">PCR: beta-Globin</SPAN></A></LI><LI class="toclevel-3 tocsection-161"><A href="#Repetition_of_PCR_for_hGH_BioBrick_Production"><SPAN class="tocnumber">1.26.8</SPAN><SPAN class="toctext">Repetition of PCR for hGH BioBrick Production</SPAN></A></LI><LI class="toclevel-3 tocsection-162"><A href="#Picking_clones_from_pAAV_RFC25_mGMK"><SPAN class="tocnumber">1.26.9</SPAN><SPAN class="toctext">Picking clones from pAAV_RFC25_mGMK</SPAN></A></LI><LI class="toclevel-3 tocsection-163"><A href="#Picking_clones_from_Retrafo_of_pTV_CMV_clone_7_and_clone_8"><SPAN class="tocnumber">1.26.10</SPAN><SPAN class="toctext">Picking clones from Retrafo of pTV_CMV clone 7 and clone 8</SPAN></A></LI><LI class="toclevel-2 tocsection-164"><A href="#72._Labortag_28.7.2010"><SPAN class="tocnumber">1.27</SPAN><SPAN class="toctext">72. Labortag 28.7.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-165"><A href="#Results_Trafo_pAAV_RC_1.1_BamHI_and_pAAV_RC_1.1_SalI"><SPAN class="tocnumber">1.27.1</SPAN><SPAN class="toctext">Results Trafo pAAV_RC_1.1_BamHI and pAAV_RC_1.1_SalI</SPAN></A></LI><LI class="toclevel-3 tocsection-166"><A href="#Proceed_with_BioBrick_Production_of_hGH_and_beta-globin"><SPAN class="tocnumber">1.27.2</SPAN><SPAN class="toctext">Proceed with BioBrick Production of hGH and beta-globin</SPAN></A></LI><LI class="toclevel-3 tocsection-167"><A href="#Results_of_ITR_Trafo"><SPAN class="tocnumber">1.27.3</SPAN><SPAN class="toctext">Results of ITR Trafo</SPAN></A></LI><LI class="toclevel-3 tocsection-168"><A href="#Trafo_evaluation_Rep_68.2678_Pst"><SPAN class="tocnumber">1.27.4</SPAN><SPAN class="toctext">Trafo evaluation Rep 68&amp;78 Pst</SPAN></A></LI><LI class="toclevel-3 tocsection-169"><A href="#Cellculture_3"><SPAN class="tocnumber">1.27.5</SPAN><SPAN class="toctext">Cellculture</SPAN></A></LI><LI class="toclevel-3 tocsection-170"><A href="#Mini-Prep_and_Test_digestion_of_pAAV_RFC25_mgmk_and_pTV_CMV_7_and_pTV_CMV_8"><SPAN class="tocnumber">1.27.6</SPAN><SPAN class="toctext">Mini-Prep and Test digestion of pAAV_RFC25_mgmk and pTV_CMV_7 and pTV_CMV_8</SPAN></A></LI><LI class="toclevel-3 tocsection-171"><A href="#Site-directed_mutagenesis_of_SspI_and_PvuII_in_pSB1C3_CFP"><SPAN class="tocnumber">1.27.7</SPAN><SPAN class="toctext">Site-directed mutagenesis of SspI and PvuII in pSB1C3_CFP</SPAN></A></LI><LI class="toclevel-3 tocsection-172"><A href="#Affibody"><SPAN class="tocnumber">1.27.8</SPAN><SPAN class="toctext">Affibody</SPAN></A></LI><LI class="toclevel-3 tocsection-173"><A href="#Picking_clones_from_Trafo_pSB1C3_RFC25_longlinker.2C_pSB1C3_RFC25_GSAT.2C_pSB1C3_RFC25_SEG"><SPAN class="tocnumber">1.27.9</SPAN><SPAN class="toctext">Picking clones from Trafo pSB1C3_RFC25_longlinker, pSB1C3_RFC25_GSAT, pSB1C3_RFC25_SEG</SPAN></A></LI><LI class="toclevel-3 tocsection-174"><A href="#Preparation_for_competent_XL1b"><SPAN class="tocnumber">1.27.10</SPAN><SPAN class="toctext">Preparation for competent XL1b</SPAN></A></LI><LI class="toclevel-3 tocsection-175"><A href="#ReTrafo_of_pGL3_hTERT"><SPAN class="tocnumber">1.27.11</SPAN><SPAN class="toctext">ReTrafo of pGL3_hTERT</SPAN></A></LI><LI class="toclevel-3 tocsection-176"><A href="#Trafo_of_PAAV_RC_1.1_SDM_of_BamHI_and_SalI"><SPAN class="tocnumber">1.27.12</SPAN><SPAN class="toctext">Trafo of PAAV_RC_1.1 SDM of BamHI and SalI</SPAN></A></LI><LI class="toclevel-3 tocsection-177"><A href="#Picking_clones_from_the_2._Trafo_of_left_and_right_ITR_.28Hanna.29"><SPAN class="tocnumber">1.27.13</SPAN><SPAN class="toctext">Picking clones from the 2. Trafo of left and right ITR (Hanna)</SPAN></A></LI><LI class="toclevel-2 tocsection-178"><A href="#73._Labortag_29.7.2010"><SPAN class="tocnumber">1.28</SPAN><SPAN class="toctext">73. Labortag 29.7.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-179"><A href="#Mini-Prep_and_Test_digestion_of_Rep_68_and_Rep_78"><SPAN class="tocnumber">1.28.1</SPAN><SPAN class="toctext">Mini-Prep and Test digestion of Rep 68 and Rep 78</SPAN></A></LI><LI class="toclevel-3 tocsection-180"><A href="#Results_sequencing_of_pTV_CM_clone_7_and_8_.28received_from_iGEM_team_2008.29"><SPAN class="tocnumber">1.28.2</SPAN><SPAN class="toctext">Results sequencing of pTV_CM clone 7 and 8 (received from iGEM team 2008)</SPAN></A></LI><LI class="toclevel-3 tocsection-181"><A href="#Sequencing_results_of_pAAV_RFC25_mGMK"><SPAN class="tocnumber">1.28.3</SPAN><SPAN class="toctext">Sequencing results of pAAV_RFC25_mGMK</SPAN></A></LI><LI class="toclevel-3 tocsection-182"><A href="#Stock_solution_of_Ampicilin"><SPAN class="tocnumber">1.28.4</SPAN><SPAN class="toctext">Stock solution of Ampicilin</SPAN></A></LI><LI class="toclevel-3 tocsection-183"><A href="#Mini-Prep_and_Test_digestion_of_ITR_constructs"><SPAN class="tocnumber">1.28.5</SPAN><SPAN class="toctext">Mini-Prep and Test digestion of ITR constructs</SPAN></A></LI><LI class="toclevel-3 tocsection-184"><A href="#Picking_clones_of_pGL3_hTERT"><SPAN class="tocnumber">1.28.6</SPAN><SPAN class="toctext">Picking clones of pGL3_hTERT</SPAN></A></LI><LI class="toclevel-3 tocsection-185"><A href="#Picking_clones_of_pAAV_RC_1.1_SDM_of_BamHI_and_SalI"><SPAN class="tocnumber">1.28.7</SPAN><SPAN class="toctext">Picking clones of pAAV_RC_1.1 SDM of BamHI and SalI</SPAN></A></LI><LI class="toclevel-3 tocsection-186"><A href="#Picking_clones_of_pSB1C3_hGH_and_pSB1C3_betaglobin"><SPAN class="tocnumber">1.28.8</SPAN><SPAN class="toctext">Picking clones of pSB1C3_hGH and pSB1C3_betaglobin</SPAN></A></LI><LI class="toclevel-3 tocsection-187"><A href="#PCR_with_pTV_CMV_with_designed_oligos_for_amplifying_CMV_in_RFC10_standard"><SPAN class="tocnumber">1.28.9</SPAN><SPAN class="toctext">PCR with pTV_CMV with designed oligos for amplifying CMV in RFC10 standard</SPAN></A></LI><LI class="toclevel-3 tocsection-188"><A href="#Mini-Prep_and_Test_digestion_of_pSB1C3_RFC25_longlinker.2C_pSB1C3_RFC25_SEG_and_pSB1C3_RFC25_GSAT"><SPAN class="tocnumber">1.28.10</SPAN><SPAN class="toctext">Mini-Prep and Test digestion of pSB1C3_RFC25_longlinker, pSB1C3_RFC25_SEG and pSB1C3_RFC25_GSAT</SPAN></A></LI><LI class="toclevel-3 tocsection-189"><A href="#Picking_clones_of_pSB1C3_CFP_w.2Fo_SspI_and_PvuII"><SPAN class="tocnumber">1.28.11</SPAN><SPAN class="toctext">Picking clones of pSB1C3_CFP w/o SspI and PvuII</SPAN></A></LI><LI class="toclevel-3 tocsection-190"><A href="#Inoculation_of_pSB1C3_RFC25_CFP"><SPAN class="tocnumber">1.28.12</SPAN><SPAN class="toctext">Inoculation of pSB1C3_RFC25_CFP</SPAN></A></LI><LI class="toclevel-2 tocsection-191"><A href="#74._Labortag_30.7.2010"><SPAN class="tocnumber">1.29</SPAN><SPAN class="toctext">74. Labortag 30.7.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-192"><A href="#Continuation_of_PCR_with_pTV_CMV_and_digestion_of_pSB1C3"><SPAN class="tocnumber">1.29.1</SPAN><SPAN class="toctext">Continuation of PCR with pTV_CMV and digestion of pSB1C3</SPAN></A></LI><LI class="toclevel-3 tocsection-193"><A href="#Repetition_of_test_digestion_of_Rep_68_and_Rep_78"><SPAN class="tocnumber">1.29.2</SPAN><SPAN class="toctext">Repetition of test digestion of Rep 68 and Rep 78</SPAN></A></LI><LI class="toclevel-3 tocsection-194"><A href="#Continuation:_left_ITR"><SPAN class="tocnumber">1.29.3</SPAN><SPAN class="toctext">Continuation: left ITR</SPAN></A></LI><LI class="toclevel-3 tocsection-195"><A href="#Continuation:_right_ITR"><SPAN class="tocnumber">1.29.4</SPAN><SPAN class="toctext">Continuation: right ITR</SPAN></A></LI><LI class="toclevel-3 tocsection-196"><A href="#ITR_gel_pictures"><SPAN class="tocnumber">1.29.5</SPAN><SPAN class="toctext">ITR gel pictures</SPAN></A></LI><LI class="toclevel-3 tocsection-197"><A href="#ZEGFR:1907"><SPAN class="tocnumber">1.29.6</SPAN><SPAN class="toctext">ZEGFR:1907</SPAN></A></LI><LI class="toclevel-3 tocsection-198"><A href="#Mini-Prep_and_Test_digestion_of_pSB1C3_SDM_SspI.2C_pSB1C3_SDM_PvuII.2C_pSB1C3_beta-globin.2C_pSB1C3_hgH.2C_pAAV_RC_1.1_BamHI_long_PCR.2C_pAAV_RC_1.1_BamHI_short_PCR.2CpGL3_hTERT_and_pAAV_RC_1.1_SalI_.28short_PCR.29.2C_pAAV_RC_1.1_BamHI_long_PCR.2C_pAAV_RC_1.1_BamHI_short_PCR"><SPAN class="tocnumber">1.29.7</SPAN><SPAN class="toctext">Mini-Prep and Test digestion of pSB1C3_SDM_SspI, pSB1C3_SDM_PvuII, pSB1C3_beta-globin, pSB1C3_hgH, pAAV_RC_1.1_BamHI long PCR, pAAV_RC_1.1_BamHI short PCR,pGL3_hTERT and pAAV_RC_1.1_SalI (short PCR), pAAV_RC_1.1_BamHI long PCR, pAAV_RC_1.1_BamHI short PCR</SPAN></A></LI><LI class="toclevel-3 tocsection-199"><A href="#Midi-prep_of_pSB1C3_CFP"><SPAN class="tocnumber">1.29.8</SPAN><SPAN class="toctext">Midi-prep of pSB1C3_CFP</SPAN></A></LI><LI class="toclevel-2 tocsection-200"><A href="#75._Labortag_31.7.2010"><SPAN class="tocnumber">1.30</SPAN><SPAN class="toctext">75. Labortag 31.7.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-201"><A href="#Trafo_of_ITR-over-night_ligations"><SPAN class="tocnumber">1.30.1</SPAN><SPAN class="toctext">Trafo of ITR-over-night ligations</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Juli">Juli</SPAN></H2><H3><SPAN class="mw-headline" id="46.Labortag_01.07.2010"><P style="font-size:17px; background-color:#00dd77;">46.Labortag 01.07.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="qPCR"><P style="font-size:15px; background-color:#66bbFF;"><B>qPCR</B></P></SPAN></H4><P><B>Investigators: Hanna, Chris W. (guided by Sven)</B></P><P>Protocol will be inserted :)
</P><H4><SPAN class="mw-headline" id="Thawing_HT1080_cells"><P style="font-size:15px; background-color:#66bbFF;"><B>Thawing HT1080 cells</B></P></SPAN></H4><P><B>Investigators: Adrian, Bea, Patrick</B></P><UL><LI>HT1080 cells were thawed following standard protocols</LI><LI>HT1080 passage 3</LI><LI>Incubation at 37°C</LI></UL><H4><SPAN class="mw-headline" id="Investigation_of_the_Kleinschmidt_sequencing"><P style="font-size:15px; background-color:#66bbFF;"><B>Investigation of the Kleinschmidt sequencing</B></P></SPAN></H4><P><B>Investigator: Volker</B></P><P>The sequence data from VM_2_30.06.-CMV-F and VM_3_30.06.-CMV-F arrived and showed as expected a homologous region with the ORFs of the corresponding VPs.</P><P>The process of VPex1, VPex2 and VPex3 sequencing is nearly finished. For further progress the oligos that prime at different locations in the Rap/Cap ORFs are required.
The actual progress of the sequencing is shown in the image on the right side:
</P><UL><LI>For the sequencing of VPex1 and VPex2 one further read will be required to achieve a continuous contig. 
</LI><LI>The sequencing of VPex3 is completed but contains two point mutations.
</LI></UL><P>The investigation of the point mutations revealed that the: 
</P><UL><LI>first mutation (TAC=&gt;TAT) is silent and codes for Tyrosine. The original codon TAC has a codon usage of 58% in Homo sapiens and 25% in E. coli where as the new codon TAT has a codon usage of 42% in H. sapiens and 75% in E. coli. There for this mutation could be an optimisation of the sequence for an approach to express the VP in E.coli.
</LI><LI>second mutation (CCG=&gt;CCA) is silent and codes for Proline. The original codon CCG has a codon usage of 11% in H. sapiens and 77% in E.coli weher as the new codon CCA has a codon usage of 27% in H. sapiens and 15%. This mutation would lead to an optimiced expression in human cells.
</LI></UL><P>Therefor the combination of these two &quot;optimized&quot; codon does not make any sense in this combination.
</P><H3><SPAN class="mw-headline" id="47.Labortag_02.07.2010"><P style="font-size:17px; background-color:#00dd77;">47.Labortag 02.07.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Fluoroscence_Pictures_of_Transduction_3"><P style="font-size:15px; background-color:#66bbFF;"><B>Fluoroscence Pictures of Transduction 3</B></P></SPAN></H4><P><B>Investigators: Chris W, Patrick</B></P><P>total celldeath! 
Looks like we should not transduce 3x10^5 cells with 1000 µl (or more) of our viral solution because almost all cells died in all approaches and the medium indicator was yellow.
So we can not say if resuspendig the cells with the viral soultion is a useful alteration of the standart protocol.
</P><P style="font-size:12px; font-weight: bold; color: red;">@Chris and Patrick: The cells which has not been resuspended died aswell??  yes (Patrick) ... so u think the volume (1000µL) of the viral stock solution  is too high and cell death is not caused by the &quot;rough&quot; reuspension?  yes (Patrick)  ... Could u maybe upload some pictures even though it did not work, but to documentate it properly!! thx Bea ... No, already thrown away and we could not take good pictures because of very thick bunches of dead cells, results confirmed by Sven (Patrick)  </P><P><A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=Freiburg10_Massacre.jpg" class="new" title="Freiburg10 Massacre.jpg">Media:Freiburg10 Massacre.jpg</A></P><H4><SPAN class="mw-headline" id="Seeding_and_Splitting_HEK"><P style="font-size:15px; background-color:#66bbFF;"><B>Seeding and Splitting HEK</B></P></SPAN></H4><P><B>Investigators: Johannes, Kira, Patrick, Chris W</B></P><UL><LI> 4 Petri dishes (each 10ml DMEM, 1,5*10^6 cells) and one 75 cell culture bottle (20ml DMEM, 1,5*10^6 cells). The cell density was not sufficient so we could not seed 3x10^6 in the 10 cm dishes as planed initially.
</LI><LI> New DMEM Medium was made (10% FCS,). We received a Sodium-Pyruvat stock solution (100x).
</LI></UL><H4><SPAN class="mw-headline" id="New_ologos.2Fprimers_arrived"><P style="font-size:15px; background-color:#66bbFF;"><B>New ologos/primers arrived</B></P></SPAN></H4><P><B>Investigator: Patrick</B></P><P>ST00147058: long pITR right for, labeled with number 31 
ST00147059: long pITR right rev, labeled with number 32 
ST00147606: long pITR left for, labeled with number 33 
ST00147061: long pITR lef rev, labeled with number 34 </P><P>An additional 1:10 working dilution was made with each oligo.
</P><P style="font-size:12px; font-weight: bold; color: red;"> @ Achim: plz write it down into our oligo excel sheet. </P><H4><SPAN class="mw-headline" id="Annotation_of_the_453_insertion_site"><P style="font-size:15px; background-color:#66bbFF;"><B>Annotation of the 453 insertion site</B></P></SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">The integration site between G453 and T454</DIV></DIV></DIV><P><B>Investigators: Patrick &amp; Volker</B></P><P>The 453 insertion site is acoording to [Boucas et al., 2009] a promising alternative to the 585/588 integration site.
The Peptides were integrated between the 453 Glycine and the 454 Threonine. This risidues were annotated in the gene maps (Genious) and visualized in the 3D structure (PyMOL).
</P><H4><SPAN class="mw-headline" id="Searching_new_restriction_sites_in_the_585.2F588_WT_loop"><P style="font-size:15px; background-color:#66bbFF;"><B>Searching new restriction sites in the 585/588 WT loop</B></P></SPAN></H4><P><B>Investigator: Volker</B></P><P>The sequence of the major site for peptide integration into the capsid proteins was extracted from the viral sequence and pasted into the program <A href="https://www.dna20.com/" class="external text" rel="nofollow">DNA 2.0</A>. In an first atempt the 4 amino acids that form the loop (5' - RGNR - 3') were used with the two flanking amino acids from the wild type sequence on each side. 
The goal of this theoretical investigation is to use alternative codons to insert restriction sites into the wild type loop to make later cloning easier.
This was repeated in an second atempt with five flanking amino acids but did not reveal additional useful restriction sites.
</P><P>Informations for the restriction enzymes were looked up in the <A href="http://www.neb.com/nebecomm/products/category1.asp?#2" class="external text" rel="nofollow">NEB Product catalogue</A>.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption">Search for restriction sites in the 585/588 WT loop  sequence</DIV></DIV></DIV></DIV><P>The three restriction sites are:
</P><P><B>BbvI restriction site:</B>
Recognized sequence: 
5'- GCAGC(N)8^ -3'
3'- CGTCG(N)12^ -3'
Sites found in pAAV_RC at position:</P><UL><LI> 17 times in the ORFs of Rep &amp; Cap =&gt; can't be used
</LI></UL><P><B>BstEII restriction site:</B>
Recognized sequence: 
5'- G^GTNACC -3'
3'- CCANTG^G -5'</P><UL><LI> There is only one BstEII restriction site in the vector that we want to use to insert the fragment (x-x)
</LI><LI> The cutting site could be removed =&gt; theoretically useful restriction site.
</LI><LI> Location fits nearly perfect because peptids ae usually integrated before the R585.
</LI></UL><P><B>PstI restriction site</B>
Recognized sequence:
5'- CTGC^AG -3'
3'- G^ACGTC -5'</P><UL><LI>PStI is part of the iGEM standard and can't be used for this purpose.
</LI></UL><H4><SPAN class="mw-headline" id="Searching_new_restriction_sites_in_the_585.2F588_KO_loop"><P style="font-size:15px; background-color:#66bbFF;"><B>Searching new restriction sites in the 585/588 KO loop</B></P></SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Search for restriction sites in the 585/588 KO loop  sequence</DIV></DIV></DIV><P><B>Investigator: Volker</B></P><P><B>PstI restriction site</B>
Recognized sequence:
5'- CTGC^AG -3'
3'- G^ACGTC -5'</P><UL><LI>PStI is part of the iGEM standard and can't be used for this purpose.
</LI></UL><P>Summary: There is no codon combination availible to encode the promising BstEII restriction site in this R585A_R588A_KO sequence.</P><H4><SPAN class="mw-headline" id="Searching_new_restriction_sites_in_the_453_loop"><P style="font-size:15px; background-color:#66bbFF;"><B>Searching new restriction sites in the 453 loop</B></P></SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Search for restriction sites in the 453 loop  sequence</DIV></DIV></DIV><P><B>Investigator: Volker</B></P><P><B>KpnI restriction site</B>
Recognized sequence:
5'- GGTAC^C -3'
3'- C^CATGG -5'
Sites found in pAAV_RC at position:</P><UL><LI>1721 and 3973
</LI><LI>first restriction site could be replace without problem becauce it lies in the region that will be synthesized to replace the three iGEM restriction sites in the Rep ORF.
</LI><LI>Restriction site is potentially useful
</LI></UL><P><B>AvaI restriction site</B>
Recognized sequence:
5'- C^YCGRG -3'
3'- GRGCY^C -5'
Sites found in pAAV_RC at position:</P><UL><LI> six times in the ORFs of Rep and Cap =&gt; can't be used
</LI></UL><P><B>DraIII restriction site</B>
Recognized sequence:
5'- TTT^AAA -3'
3'- AAA^TTT -5'
Sites found in pAAV_RC at position:</P><UL><LI>1181 (Rep ORF)
</LI><LI>2893 (Cap ORF)
</LI><LI>4463 (short after the Cap ORF)
</LI><LI>4930 (f1 ORI)
</LI></UL><P><B>AvaII restriction site</B>
Recognized sequence:
5'- G^GWCC -3'
3'- CCWG^G -5'
Sites found in pAAV_RC at position:</P><UL><LI> eight times in the ORFs of Rep and Cap =&gt; can't be used
</LI></UL><P>Summary: 
The KpnI restriction site seems to be a good possibility to make the 453 insertion site ready for easy integration of peptides.
Especially the position where the cut is performed is ideal for our purpose because peptides are usually integrated after the Glycine 453 as described for example in [Boucas et al.; 2009].
</P><H3><SPAN class="mw-headline" id="48.Labortag_03.07.2010"><P style="font-size:17px; background-color:#00dd77;">48.Labortag 03.07.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Planing_the_gene_synthesis_to_replace_the_three_restriction_sites_in_the_rep_ORF"><P style="font-size:15px; background-color:#66bbFF;"><B>Planing the gene synthesis to replace the three restriction sites in the rep ORF</B></P></SPAN></H4><P><B>Investigator: Volker</B></P><P>The construct pAAV_RC that we have bougth from Stratagene will be a central part of our Virus construction kit and therefor it has to be cleared from restriction sites that are used in the iGEM-standard. 
There are five iGEM restriction sites in the pAAV-RC:</P><UL><LI>PstI at position (310)
</LI><LI>EcoRI at position (1578)
</LI><LI>PstI at position (1773)
</LI><LI>EcoRI at position (1796)
</LI><LI>PstI at position (4073)
</LI></UL><P>The relative location to each other and other planed modifications in this sequence make it reasonable to replace the PstI (310) and PStI (4073) with site directed mutagenesis and to order a synthesis of the region with the three other restriction sites.</P><P>The primers for the site directed mutagenesis were designed:</P><P><B>Designed Primer: SDM PstI (310)</B></P><UL><LI>PstI at position (310)</LI><LI>Recognized sequence: 5'- CTGC^AG -3'</LI><LI>The codon 5'- CTG - 3' will be replaced by 5'- CTC -3'</LI><LI>Both codons encode Leucine. The condon usage in <I>H. sapiens</I> decreases from 36% to 20%  </LI></UL><P>5'- CTGACCGTGGCCGAGAAGCT<B>C</B>CAGCGCGACTTTCTGACGGA - 3'</P><P><B>Designed Primer: SDM PstI (4073)</B></P><UL><LI>Recognized sequence: 5'- CTGC^AG -3'</LI><LI>The codon 5'- CTG - 3' will be replaced by 5'- CTC -3'</LI><LI>Both codons encode Leucine. The condon usage in <I>H. sapiens</I> decreases from 36% to 20%  </LI></UL><P>5'- CGTGGAGATCGAGTGGGAGCT<B>C</B>CAGAAGGAAAACAGCAAACGCTGG -3'</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption">Restriction sites of the RFC25 standard located in the Rep ORF shown in dark blue and single cuting restriction endunucleases shown in light blue</DIV></DIV></DIV></DIV><P>The region with the three restriction sites that are located near to each other is show in the picture above. The restriction sites indicated in light blue are single cutters and can therefor be use to clone the synthesized fragment into the original sequence of pAAV-RC.
For this purpose the restriction sites BstBI (1435) and SwaI (2004) will be used because they are availible in the lab stock and have 100% fidelity in one of the NEB buffers.
The required construct with four flanking base pairs on each side will be 583 bp long.</P><UL><LI>EcoRI at position (1578)</LI><LI>Recognized sequence: 5'- G^AATTC -3'</LI><LI>The codon 5'- GAA -3' will be replaced by 5'- GAG -3'</LI><LI>Both codons encode Glutamic acid. The codon usage in <I>H.sapiens</I> increases from 41% to 59%. </LI></UL><UL><LI>PstI at position (1773)</LI><LI>Recognized sequence: 5'- CTGC^AG -3'</LI><LI>The codon 5'- TGC - 3' will be replaced by 5'- TGT -3'</LI><LI>Both codons encode Cysteine. The condon usage in <I>H. sapiens</I> decreases from 58% to 42%  </LI></UL><UL><LI>EcoRI at position (1796)
</LI><LI>Recognized sequence: 5'- G^AATTC -3'</LI><LI>The codon 5'- AAT -3' will be replaced by 5'- AAC -3'</LI><LI>Both codons encode for Asparagine. The codon usage in <I>H. sapiens</I> increases from 44% to 56%.</LI></UL><P>This are all the restriction sites in the ORFs for the Rep and Cap proteins. In the backbone of the pAAV_RC there are more restriction sites in the iGEM standard. These restriction sites should not cause any problems when the insert in cloned into a standard iGEM vector.
An other interesting question is what has been changed in the regulatory region p5. This will be important to determine the sequence that is required for the functional expression of the pAAV-RC.
</P><H4><SPAN class="mw-headline" id="Modification_of_the_Cap_genes"><P style="font-size:15px; background-color:#66bbFF;"><B>Modification of the Cap genes</B></P></SPAN></H4><P><B>Investigator: Volker</B></P><P>The HSPG-binding-motif can be knocked out by the two mutations R585A and R588A.
</P><P>The sequence 5'-CTC CAG AGA GGC AAC AGA CAA -3' codes in the pAAV-RC construct for the amino acids LQRGNRQ.
In order to achieve the recommanded mutation the sequence has to be changed to: 5'-CTC CAG GCT GGC AAC GCT CAA -3' which codes for LQAGNAQ.</P><P>The problem is that in this R585A R588A KO version there is no possible codon usage to encode a BstEII restriction site that would make it possible to integrate peptides in an easy fashion.
An R585A R588L KO version would contain the BstEII restriction site and could be a possible alternative to the R585A R588A KO version.
</P><P>Questions to adress on monday:
</P><UL><LI>Is the proposed R585A_R588L_KO mutation possible?
</LI><LI>Are iGEM standard vectors availible that do not contain KpnI and BstEII?
</LI></UL><H4><SPAN class="mw-headline" id="Cellculture"><P style="font-size:15px; background-color:#66bbFF;"><B>Cellculture</B></P></SPAN></H4><P><B>Investigators: Patrick, Chris W.</B></P><P>HT1080 cells were split.
Deviations from the standart protocol: The cells were centrifuged with 2000 G instead of 200 G. Then the cells were put into the incubator for 4 hours and then examined again. It seems that most of them survived this mistake.
</P><P>The 293 cells were checked for cell density. They will be &gt;70% confluent tomorrow and therefore transfected.
</P><H3><SPAN class="mw-headline" id="49.Labortag_04.07.2010:"><P style="font-size:17px; background-color:#00dd77;">49.Labortag 04.07.2010:</P></SPAN></H3><H4><SPAN class="mw-headline" id="Transfection_nr.3"><P style="font-size:15px; background-color:#66bbff;">Transfection nr.3</P></SPAN></H4><P><B>Investigators: Chris W, Patrick</B>
Transfection with P41 was performed according standart protocol. 
Two transfektions where carried out with 10µg (3,33 µg each plasmid) DNA and the other two with 20µg DNA (6,66 µg each plasmid).
</P><H3><SPAN class="mw-headline" id="50.Labortag_05.07.2010:"><P style="font-size:17px; background-color:#00dd77;">50.Labortag 05.07.2010:</P></SPAN></H3><H4><SPAN class="mw-headline" id="Sequencing_of_pAAV_iGEM-mVenus-YFP"><P style="font-size:15px; background-color:#66bbff;">Sequencing of pAAV_iGEM-mVenus-YFP</P></SPAN></H4><P><B>Investigator: Hanna </B>
Alignment of sequenced hGH region with pAAV_iGEM_mVenus-YFP delivered following picture:</P><P>There seemed to be a nucleotide insertion. But by having a closer look, we recognized that Geneious mis-interpreted the signal at this point. Further on we could validate that YFP was also successfully cloned into this clone. Unfortunately the sequencing of the gene of interest didn't work - neither with the forward, nor with the reverse primer. Their sequences were checked on the order form: Everything OK! <B>To do: Repeat sequencing of GOI!</B></P><H4><SPAN class="mw-headline" id="Dirty_and_fancy_method_for_ITR_BioBrick_production"><P style="font-size:15px; background-color:#66bbff;">Dirty and fancy method for ITR BioBrick production</P></SPAN></H4><P><B>Investigator: Hanna</B>
A new strategy for ITR BioBrick production was figured out. This method does NOT require any PCR steps! :) 
We want to take advantage of the fact, that the ITRs are flanked by NotI and PstI. 
&quot;Short version&quot; of the strategy: </P><UL><LI> Digestion of pAAV_MCS with AlwNI. Result: one large fragment (2982 bp) with right ITR; one small fragment (1674 bp) with left ITR.
</LI><LI> Separate fragments via gel run.
</LI><LI> Digest them with NotI and PstI.
</LI><LI> Präfix and suffix oligos were designed. They need to be hybridized and ligated to the left and right ITR with several intermediate steps (subcloning into pSB1C3 etc.). The oligos posess compatible overhangs to the PstI and NotI restriction sites of the ITR but do not generate new restriction sites after ligation! 
</LI><LI> For detailed instruction, check up here: <A href="/File:Freiburg10_ITRBioBrick_dirty_fancy.pdf" title="File:Freiburg10 ITRBioBrick dirty fancy.pdf">File:Freiburg10 ITRBioBrick dirty fancy.pdf</A></LI></UL><H4><SPAN class="mw-headline" id="Beta-globin_BioBrick_production"><P style="font-size:15px; background-color:#66bbff;">Beta-globin BioBrick production</P></SPAN></H4><P><B>Investigator: Hanna</B>
The beta-globin sequence (pAAV_MCS) contains no iGEM-restriction sites. Therefore we decided NOT to order the sequence, but to perform a PCR with RFC10 overhang primers: </P><H4><SPAN class="mw-headline" id="hGH_BioBrick_production"><P style="font-size:15px; background-color:#66bbff;">hGH BioBrick production</P></SPAN></H4><P><B>Investigator: Hanna </B>
The hGH sequence (pAAV_MCS) contains no iGEM-restriction sites. Therefore we decided NOT to order the sequence, but to perform a PCR with RFC10 overhang primers: </P><H4><SPAN class="mw-headline" id="Splitting_HT_and_HEK_cells"><P style="font-size:15px; background-color:#66bbff;">Splitting HT and HEK cells</P></SPAN></H4><UL><LI><B>Investigators: Adrian, Be</B>a
</LI><LI>HEK Cells: over 100% confluent!!!
</LI><LI>HT-cells: over 90%   confluent!!!
</LI></UL><H4><SPAN class="mw-headline" id="Preparation_of_HT1080_3x6er_dishes_for_Transduction"><P style="font-size:15px; background-color:#66bbff;">Preparation of HT1080 3x6er dishes for Transduction</P></SPAN></H4><UL><LI><B>Investigators: Adrian, Bea</B></LI><LI> 3x 6well dishes got prepared with 3ml DMEM and 300.000 cells (0,0375ml of our Cell-falcon)
</LI><LI> Transduction on 6.7.2010
</LI></UL><H4><SPAN class="mw-headline" id="Preparation_of_6x10cm_petri_dishes_HEK293"><P style="font-size:15px; background-color:#66bbff;">Preparation of 6x10cm  petri dishes HEK293</P></SPAN></H4><UL><LI><B>Investigator: Adrian</B></LI><LI> 6x 10cm petri dishes with 10ml with 3.000.000 cells (0,344ml of cell-falcon)
</LI><LI> Transfection on 7.7.2010
</LI></UL><H4><SPAN class="mw-headline" id="Repetition_of_ITR_restriction_site_modification_via_PCR"><P style="font-size:15px; background-color:#66bbff;">Repetition of ITR restriction site modification via PCR</P></SPAN></H4><P><B>Investigator: Achim, Chris</B></P><LI> experiment date: 5.7.2010
</LI><LI>Because the previous attempt to replace the PSTI and NOTI restriction sites in the ITRs with the IGEM standard were unsucessful, new, longer primers were designed an ordered. 
</LI><LI>a new pcr reaction with higher annealing temperatures was prepared:
</LI><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">v(pAAV_fragment left ITR, 0% DMSO) /µl </TD><TD align="right">v(pAAV_fragment left ITR, 10% DMSO) /µl</TD><TD align="right">v(pAAV_fragment right ITR, 0% DMSO) /µl </TD><TD align="right">v(pAAV_fragment right ITR, 10% DMSO) /µl</TD><TD align="right">positive control from Gerrit: XBL PMA DBSA (106 bp)
</TD></TR><TR><TD> 10xThermoPol buffer</TD><TD align="right"> 5</TD><TD align="right"> 5</TD><TD align="right"> 5</TD><TD align="right"> 5</TD><TD align="right"> 5
</TD></TR><TR><TD> dNTP mix </TD><TD align="right">1 </TD><TD align="right">1 </TD><TD align="right">1 </TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD> Primer for</TD><TD align="right"> 1,5</TD><TD align="right"> 1,5</TD><TD align="right"> 1,5</TD><TD align="right"> 1,5</TD><TD align="right"> 1,5
</TD></TR><TR><TD> Primer rev</TD><TD align="right"> 1,5</TD><TD align="right"> 1,5</TD><TD align="right"> 1,5</TD><TD align="right"> 1,5</TD><TD align="right"> 1,5
</TD></TR><TR><TD>DNA Template</TD><TD align="right"> 0,513 </TD><TD align="right"> 0,513</TD><TD align="right"> 0,415</TD><TD align="right"> 0,415</TD><TD align="right"> 0,345
</TD></TR><TR><TD>MgSO4</TD><TD align="right"> 1</TD><TD align="right"> 1</TD><TD align="right"> 1</TD><TD align="right"> 1</TD><TD align="right"> 1
</TD></TR><TR><TD>DMSO</TD><TD align="right"> 0</TD><TD align="right"> 5</TD><TD align="right"> 0</TD><TD align="right"> 5</TD><TD align="right"> 0
</TD></TR><TR><TD>Vent Polymerase</TD><TD align="right"> 0,5</TD><TD align="right"> 0,5</TD><TD align="right"> 0,5</TD><TD align="right"> 0,5</TD><TD align="right"> 0,5
</TD></TR><TR><TD>H2O</TD><TD align="right"> 38,99</TD><TD align="right"> 33,99</TD><TD align="right"> 39,09</TD><TD align="right"> 34,09</TD><TD align="right"> 39,16
</TD></TR><TR><TD>Volume total</TD><TD align="right"> 50</TD><TD align="right"> 50</TD><TD align="right"> 50</TD><TD align="right"> 50</TD><TD align="right"> 50
</TD></TR></TBODY></TABLE><P>
PCR program:
</P><TABLE border="1"><TBODY><TR><TD>Cycles</TD><TD>Temperature</TD><TD>Time
</TD></TR><TR><TD>95°C</TD><TD>3&quot;
</TD></TR><TR><TD>95°C</TD><TD>30&quot;
</TD></TR><TR><TD>10x</TD><TD>62+-2°C</TD><TD>22&quot;
</TD></TR><TR><TD>72°C</TD><TD>12&quot;
</TD></TR><TR><TD>95°C</TD><TD>30&quot;
</TD></TR><TR><TD>15x</TD><TD>66+/-2°C</TD><TD>22&quot;
</TD></TR><TR><TD>72°C</TD><TD>12&quot;
</TD></TR><TR><TD>1x</TD><TD>72°C</TD><TD>5'
</TD></TR><TR><TD>Hold 4°C
</TD></TR></TBODY></TABLE><LI> The PCR was run 
</LI><LI> 2% agarose gel was prepared. Running the gel at 110 V for 45 minutes. 
</LI><LI> No bands were observed on the gel -&gt; pcr didn't work. 
Possible reasons: *Old Vent polymerase -&gt; New polymerase ordered
</LI><LI>Mg concentration could be optimized
</LI><LI>Template okay? Template will be loaded on gel as well next time
</LI><H4><SPAN class="mw-headline" id="Search_for_restriction_sites_in_the_sequence_LQA.2FRGQA.2FRQA"><P style="font-size:15px; background-color:#66bbff;">Search for restriction sites in the sequence LQA/RGQA/RQA</P></SPAN></H4><P><B>Investigator: Volker</B></P><P>The idea to use a R588L mutation instead of the R588A that is published in the literature was discussed with the supervisers. It was proposed to search for other amino acid exchanges that alter the loop less drastically. For this reason the mutation N587Q was examined for possible restriction sites.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Search for restriction sites in the 585/588 KO loop  sequence</DIV></DIV></DIV><P style="clear:both;"><LI> :) </LI></P><H4><SPAN class="mw-headline" id="Results_sequencing_alignment_of_GMK_TK30"><P style="font-size:15px; background-color:#66bbff;">Results sequencing alignment of GMK_TK30</P></SPAN></H4><P><B>Investigator: Bea</B><B>Result</B>: The obtained sequencing results indicate that the DNA we received from Amor contains the GMK_TK30 fusion enzyme, but tk30 contains some restriction sites. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Sequence alignment of GMK_TK30</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Sequence alignment of GMK_TK30</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Sequence alignment of GMK_TK30</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Sequence alignment of GMK_TK30</DIV></DIV></DIV></DIV><UL><LI>To do (06.07.2010): Order primers for BioBrick Fusionenzyme and order genes of GMK and TK (wtTK, TK30, sr39????)</LI><LI> remove iGEM restriction sites</LI></UL><H3><SPAN class="mw-headline" id="51.Labortag_6.07.2010:"><P style="font-size:17px; background-color:#00dd77;">51.Labortag 6.07.2010: </P></SPAN></H3><H4><SPAN class="mw-headline" id="Harvesting_AAVs_from_4_10cm_dishes.2C_Transduction_of_3x6er_dishes"><P style="font-size:15px; background-color:#66bbff;">Harvesting AAVs from 4 10cm dishes, Transduction of 3x6er dishes</P></SPAN></H4><P><B>Investigators: Adrian, Anna, Bea, Kerstin</B></P><P>We want to investigate which of the following approaches yields a better transduction efficiency.
</P><UL><LI>Half of the transfected cells were exposed 3 cycles of thawing and freezing and then centrifuged (15 ml falcon, 2100G). The supernatant was transfered into a new 15 ml falcon and used for transduction. 
</LI><LI>The other half was centrifuged at 300 G for 5min . The supernatant was transfered into another 15 ml falcon and the pellet was resuspended with 5 ml of DMEM. The content of these two falcons was also exposed to 3 cycles of thawing and freezing and then used for transduction.
</LI></UL><UL><LI> We have got 2 10 cm cellculture dishes with 10µg and 2 dishes with 20µg DNA used for transfection. 
</LI></UL><P>Approach with standart protocol (one dish with 10 and the other with 20µg Plasmids)(2100 G and 3 cycles of freezing and thawing)
</P><P>
Deviations from the standart protocol:
</P><UL><LI> The cells were centrifuged at 2100 G instead of 10.000 G
</LI><LI> The viruses were harvested after 44 hours
</LI><LI> 3 cycles of freezing and thawing
</LI></UL><H4><SPAN class="mw-headline" id="Search_for_restriction_sites_before_and_after_the_insertion_sites_453_and_585.2F588"><P style="font-size:15px; background-color:#66bbff;">Search for restriction sites before and after the insertion sites 453 and 585/588</P></SPAN></H4><P><B>Investigator: Volker</B></P><H4><SPAN class="mw-headline" id="Transduction_of_3x6er_dishes"><P style="font-size:15px; background-color:#66bbff;">Transduction of 3x6er dishes </P></SPAN></H4><P><B>Investigators: Adrian Bea</B></P><UL><LI> used plasmids: 10µg, 20µg YFP
</LI><LI> we have 10µg and 20µg from standard protocol =&gt; SV
</LI><LI> we have 10µg and 20µg from standard protocol pellet =&gt; Pellet
</LI><LI> we have 10µg and 20µg from standard protocol suspension =&gt; Super
</LI><LI> amount of 500µl 
</LI></UL><P>1. Plate I(A is up)
</P><TABLE border="1"><TBODY><TR><TD> 10µg dropped Super </TD><TD align="right">10µg dropped Super gently resuspending</TD><TD align="right">10µg dropped Super gently resuspending
</TD></TR><TR><TD> 10µg Super gently resuspending Super</TD><TD align="right">20µg dropped Super gently resuspending 1:10 dilution</TD><TD align="right"> control
</TD></TR></TBODY></TABLE><P>
2. Plate I(A is up)
</P><TABLE border="1"><TBODY><TR><TD> 10µg dropped Super </TD><TD align="right">10µg dropped Super gently resuspending</TD><TD align="right">10µg dropped Super gently resuspending
</TD></TR><TR><TD> 10µg Super gently resuspending Super</TD><TD align="right">20µg dropped Super gently resuspending 1:10 dilution</TD><TD align="right"> control
</TD></TR></TBODY></TABLE><P>
3. Plate I(A is up)
</P><TABLE border="1"><TBODY><TR><TD> 10µg dropped Pellet </TD><TD align="right">10µg dropped Pellet gently resuspending</TD><TD align="right">10µg dropped Pellet gently resuspending
</TD></TR><TR><TD> 10µg Super gently resuspending Pellet</TD><TD align="right">20µg dropped Pellet gently resuspending 1:10 dilution</TD><TD align="right"> control
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Search_for_restriction_sites_around_the_453_and_585.2F588_integration_sites"><P style="font-size:15px; background-color:#66bbff;">Search for restriction sites around the 453 and 585/588 integration sites</P></SPAN></H4><P><B>Investigator: Volker</B></P><P>For this purpose a list of all commercially availible restriction enzymes was created that do not match in the ORFs of pAAV-RC.
</P><H4><SPAN class="mw-headline" id="Search_for_restriction_sites_in_the_sequence_LQA.2FRGQA.2FRQA_2"><P style="font-size:15px; background-color:#66bbff;">Search for restriction sites in the sequence LQA/RGQA/RQA</P></SPAN></H4><P><B>Investigator: Volker</B></P><P>The idea to use a R588L mutation instead of the R588A that is published in the literature was discussed with the supervisers. It was proposed to search for other amino acid exchanges that alter the loop less drastically. For this reason the mutation N587Q was examined for possible restriction sites.</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption">bla Bla bla</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption">bla Bla bla</DIV></DIV></DIV><P style="clear:both;"><LI>.</LI></P><H3><SPAN class="mw-headline" id="52.Labortag_07.07.2010:"><P style="font-size:17px; background-color:#00dd77;">52.Labortag 07.07.2010:</P></SPAN></H3><H4><SPAN class="mw-headline" id="Sequencing_of_GIO"><P style="font-size:15px; background-color:#66bbff;">Sequencing of GIO</P></SPAN></H4><P><B>Investigator: Hanna</B>
Comments: Neither sequencing with forward nor with reverse primer (O29 + O30) worked. Therefore P40 (pAAV_iGEM_mVenus-YFP) was sent for sequencing once again. </P><UL><LI>Name: Hanna P40 </LI><LI>V (P40) = 11 µL + 19 µL H2O </LI><LI>Name of primer forward: Hanna 1 (O30)</LI><LI>Name of primer reverse: Hanna 2 (O29)</LI><LI>V(primer) = 3 µL + 27 µL H2O </LI></UL><H4><SPAN class="mw-headline" id="Interview_with_Bild_der_Wissenschaft"><P style="font-size:15px; background-color:#66bbff;">Interview with Bild der Wissenschaft</P></SPAN></H4><UL><LI><B>Adrian</B> did a good job ;-)
</LI><LI>The article will be send to us next week
</LI><LI>For additional question, Jessica will be send an email
</LI></UL><H4><SPAN class="mw-headline" id="Transfection"><P style="font-size:15px; background-color:#66bbff;">Transfection</P></SPAN></H4><P><B>Investigators: Adrian, Bea</B></P><UL><LI>Transfection of AAV-293 cells with pAAV_iGEM-mVenus_YFP
</LI></UL><H4><SPAN class="mw-headline" id="New_pcr_run_with_new_polymerase_and_different_MgSO4_concentrations"><P style="font-size:15px; background-color:#66bbff;">New pcr run with new polymerase and different MgSO4 concentrations</P></SPAN></H4><UL><LI>While this time the positive control was actually positive, no pcr product could be seen. The reasons are unclear. (Secondary structures, primer binding...)
</LI></UL><H4><SPAN class="mw-headline" id="Checking_the_results_of_the_following_sequencing_reactions_for_the_expression_constructs_from_the_DKFZ"><P style="font-size:15px; background-color:#66bbff;">Checking the results of the following sequencing reactions for the expression constructs from the DKFZ</P></SPAN></H4><P><B>Investigator: Volker</B></P><P>
pKEX-VP1ex: sequence confirmed as described in  [Name et al.,] with two pointmutations (ACG=&gt;GCG) and (ATG=&gt;CTG) to avoid the translation of VP2 and VP3.</P><P>pKEX-VP2ex: sequence confirmed as described in  [Name et al.,] with one mutation (ATG=&gt;CTG) to avoid the translation of VP3 and a second mutation (AAGACG=&gt;CATATG) to switch to the main start codon ATG and to introduce a restriction site (which one??) before the start codon.</P><P>pKEX-VP3ex: sequence confirmed as described in  [Name et al.,] with two silent pointmutations and a second mutation to introduce a restriction site before the start codon.</P><P>pKEX-Rep40ex: Sequence as expected until now, further sequencing reaction required with the primer: GATC_std_pTeSp-1</P><P>pKEX-Rep52ex: Point mutation directly at the beginning of the exon sequence, perhaps functionality in this context? At the end of the sequence some umcertainities. For the beginning further sequencing reaction required with the primer: GATC_std_pTeSp-1</P><P>
pKEX-Rep68ex: One point mutation in the second condon that changes the coded amino acids from P to A, a second mutation (ATG=&gt;GGG) at the beginning of the Rep40/Rep52 ORFs to avoid the translation of these sequences and a silent mutation. One further sequencing reaction is required with the primer: GATC_std_pcDNA1.1-RP.</P><P>pKEX-Rep78ex: One point mutation in the second condon that changes the coded amino acids from P to A, a second mutation (ATG=&gt;GGG) at the beginning of the Rep40/Rep52 ORFs to avoid the translation of these sequences, a silent mutation and the same point mutation at the beginning of the exon as seen before in pKEX-Rep52ex.
</P><H3><SPAN class="mw-headline" id="53.Labortag_08.07.2010:"><P style="font-size:17px; background-color:#00dd77;">53.Labortag 08.07.2010:</P></SPAN></H3><H4><SPAN class="mw-headline" id="Final_planing_of_the_Cap_modifications"><P style="font-size:15px; background-color:#66bbff;"> Final planing of the Cap modifications</P></SPAN></H4><P><B>Investigator: Volker</B></P><P><B>BbvCI</B>		
50|100|10|100	37°<A href="/File:Freiburg10_BbvC-I-cutsite_1_v1_000007.gif" title="File:Freiburg10 BbvC-I-cutsite 1 v1 000007.gif">File:Freiburg10 BbvC-I-cutsite 1 v1 000007.gif</A>
Recognition sites in Rep/Cap ORFs:   
Recognition sites in pAAV_RC:    
Recognition site in pSB1C3:    </P><P><B>KpnI</B>		
100|75|0|50	37°C + BSA<A href="/File:Freiburg10_Kpn-I-cutsite_1_v1_000021.gif.gif" title="File:Freiburg10 Kpn-I-cutsite 1 v1 000021.gif.gif">File:Freiburg10 Kpn-I-cutsite 1 v1 000021.gif.gif</A>
Recognition sites in Rep/Cap ORFs:   
Recognition sites in pAAV_RC:    
Recognition site in pSB1C3:    </P><P><B>SalI</B>		
10|100|100|100	37°C<A href="/File:Freiburg10_Sal-I-cutsite_1_v1_000016.gif.gif" title="File:Freiburg10 Sal-I-cutsite 1 v1 000016.gif.gif">File:Freiburg10 Sal-I-cutsite 1 v1 000016.gif.gif</A>
Recognition sites in Rep/Cap ORFs:   
Recognition sites in pAAV_RC:    
Recognition site in pSB1C3:    </P><P><B>AarI</B>			
|||<A href="/File:Freiburg10_.gif" title="File:Freiburg10 .gif">File:Freiburg10 .gif</A>
Recognition sites in Rep/Cap ORFs:   
Recognition sites in pAAV_RC:    
Recognition site in pSB1C3:    </P><P><B>ScaI</B>			
100|100|10|100	37°C<A href="/File:Freiburg10_Sca-I-cutsite_1_v1_00001.gif.gif" title="File:Freiburg10 Sca-I-cutsite 1 v1 00001.gif.gif">File:Freiburg10 Sca-I-cutsite 1 v1 00001.gif.gif</A>
Recognition sites in Rep/Cap ORFs:  
Recognition sites in pAAV_RC:    
Recognition site in pSB1C3: 2   </P><P><B>BamHI</B>			
75|100|100|100	37°C + BSA<A href="/File:Freiburg10_BamH-I-cutsite_1_v1_000025.gif.gif" title="File:Freiburg10 BamH-I-cutsite 1 v1 000025.gif.gif">File:Freiburg10 BamH-I-cutsite 1 v1 000025.gif.gif</A>
Recognition sites in Rep/Cap ORFs:   
Recognition sites in pAAV_RC:    
Recognition site in pSB1C3:    </P><P><B>PvuII</B>			
100|100|100|100	37°C<A href="/File:Freiburg10_Pvu-II-cutsite_1_v1_000013.gif.gif" title="File:Freiburg10 Pvu-II-cutsite 1 v1 000013.gif.gif">File:Freiburg10 Pvu-II-cutsite 1 v1 000013.gif.gif</A>
Recognition sites in Rep/Cap ORFs:   
Recognition sites in pAAV_RC:    
Recognition site in pSB1C3:  1  </P><P><B>EagI</B>		
10|25|100|10	37°C<A href="/File:Freiburg10_Eag-I-cutsite_1_v1_000021.gif.gif" title="File:Freiburg10 Eag-I-cutsite 1 v1 000021.gif.gif">File:Freiburg10 Eag-I-cutsite 1 v1 000021.gif.gif</A>
Recognition sites in Rep/Cap ORFs:   
Recognition sites in pAAV_RC:    
Recognition site in pSB1C3:  2  </P><H3><SPAN class="mw-headline" id="54.Labortag_09.07.2010:"><P style="font-size:17px; background-color:#00dd77;">54.Labortag 09.07.2010:</P></SPAN></H3><P><B>Investigators: Anissa, Kerstin, Anna</B></P><UL><LI>New DMEM-Medium was prepared (10% FCS). Investigators: Bea, Patrick
</LI><LI>HT and 293 cells were split into 4 flasks, 2 flasks each. Investigator: Kira
</LI><LI>AAV stocks were prepared, put into the -80°C freezer and labeled with the date, name, AAV, the amount of DNA used for transfection and drop/resuspend to make clear whether the DNA was just pippeted dropwise to the 293 cells or gently mixed with the cells. The numbers on the 15 ml falcons refer to DNA amount and DNA administration, too. Investigator: Patrick
</LI></UL><H4><SPAN class="mw-headline" id="PCR_of_hgH"><P style="font-size:15px; background-color:#66bbff;">PCR of hgH</P></SPAN></H4><UL><LI>PCR program: Unnamed
</LI></UL><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl
</TD></TR><TR><TD> 5x Phusion HF buffer  </TD><TD align="right"> 10
</TD></TR><TR><TD> 10 mM dNTP mix </TD><TD align="right"> 1
</TD></TR><TR><TD> hgH_primer_for</TD><TD align="right"> 2,5
</TD></TR><TR><TD> hgH_primer_rev</TD><TD align="right"> 2,5
</TD></TR><TR><TD>DNA template</TD><TD align="right"> 0,38
</TD></TR><TR><TD>DMSO</TD><TD align="right"> -
</TD></TR><TR><TD>Phusion polymerase</TD><TD align="right"> 0,5
</TD></TR><TR><TD>H2O</TD><TD align="right"> 33,1
</TD></TR><TR><TD><B>Total volume (e.g. 50 µl)</B></TD><TD align="right"> 50
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color: blue;">Agarose-Gel:</P><P>
0,625 g Agarose, 50 ml TAE (1,25%), 3 µl GELRED (3-6µl), at 110 Volt, running time: 60 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (5x/6x)/µl
</TH><TH>Expected size (Geneious)
</TH></TR><TR><TD>p47: hgH
</TD><TD>50 µl
</TD><TD>10 µl
</TD><TD>488 bp
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker (8 µl)
</TH><TH>Sample 50 /µl
</TH></TR><TR><TH>Lane
</TH><TD>1
</TD><TD>4
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color: blue;">Gelextraction</P><P>
Gel measurement:
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Sample</B></TD><TD align="left"><B>weight</B></TD></TR><TR><TD align="left"> p47: hgH </TD><TD align="left"> 0,13 g
</TD></TR></TBODY></TABLE><UL><LI>DNA concentration (hgH): 19,83 ng/µl
</LI></UL><P><B>Comments:</B> PCR was succesful:-)
to do: Ligation of PCR product into standard plasmid which we´ve received wihthin the DNA distrivution kit of iGEM-HQ .
</P><H4><SPAN class="mw-headline" id="Order_oligos_for_GMK_TK30"><P style="font-size:15px; background-color:#66bbff;">Order oligos for GMK_TK30</P></SPAN></H4><P><B>Investigator: Bea</B>
Oligos have been ordered at sigmaaldrich in order to insert prefix and suffix to fusionenzyme mGMK_TK30 in RFC25. 
</P><P style="font-weight: bold; color: red;">to do: Order gene tk30 and part of gene sr39</P><H4><SPAN class="mw-headline" id="Quickchange_for_the_PstI_restriction_sites_in_pAAV-RC"><P style="font-size:15px; background-color:#66bbff;">Quickchange for the PstI restriction sites in pAAV-RC</P></SPAN></H4><P><B>Investigators: Kira and Volker</B>
Quick change, Gel and test digestion with PstI. (more precise informations will be added)
@Kira and Volker: did the Quickchange actually work? Could you please add the information onto the wiki!! Thx!
</P><P><B>PCR program</B>: PstI 
</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume in µl
</TD></TR><TR><TD> 10x Phusion HF buffer  </TD><TD align="right"> 2.5 µl
</TD></TR><TR><TD> 10 mM dNTP mix </TD><TD align="right"> 0.5 µl
</TD></TR><TR><TD> __310 ____primer_for (1:10 dilution)</TD><TD align="right">  0,39 µl
</TD></TR><TR><TD> __310____primer_rev (1:10 dilution)</TD><TD align="right">  0,39 µl
</TD></TR><TR><TD>DNA template (1:40 dilution)</TD><TD align="right">  0,5 µl
</TD></TR><TR><TD>DMSO</TD><TD align="right">  0 µl
</TD></TR><TR><TD>Phusion polymerase</TD><TD align="right">  0.5 µl
</TD></TR><TR><TD>H2O</TD><TD align="right"> 20,22 µl
</TD></TR><TR><TD><B>Total volume (e.g. 50 µl)</B></TD><TD align="right"> 25 µl
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume in µl
</TD></TR><TR><TD> 10x Phusion HF buffer  </TD><TD align="right"> 2.5 µl
</TD></TR><TR><TD> 10 mM dNTP mix </TD><TD align="right"> 0.5 µl
</TD></TR><TR><TD> __4073____primer_for (1:10 dilution)</TD><TD align="right">  0.33 µl
</TD></TR><TR><TD> ___4073___primer_rev (1:10 dilution)</TD><TD align="right">  0.33 µl
</TD></TR><TR><TD>DNA template (1:40 dilution)</TD><TD align="right">  0.5 µl
</TD></TR><TR><TD>DMSO</TD><TD align="right">  0 µl
</TD></TR><TR><TD>Phusion polymerase</TD><TD align="right">  0.5 µl
</TD></TR><TR><TD>H2O</TD><TD align="right"> 20.32 µl
</TD></TR><TR><TD><B>Total volume (e.g. 50 µl)</B></TD><TD align="right"> 25 µl
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD> cycles   </TD><TD align="right">temperature </TD><TD align="right">time
</TD></TR><TR><TD> 1 </TD><TD align="right">95 C </TD><TD align="right">2 min
</TD></TR><TR><TD> 20 </TD><TD align="right">95 C </TD><TD align="right">30 sec
</TD></TR><TR><TD align="right">66 C </TD><TD align="right">1 min
</TD></TR><TR><TD align="right">68 C </TD><TD align="right">5 min
</TD></TR></TBODY></TABLE><UL><LI> The PCR-products were digested with DpnI (1µl instead of 0.5 µl)
</LI><LI> aragose gel revealed for both samples very distinct bands
</LI></UL><P><B>TEST DIGESTION</B></P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume  /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 10
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 3 (10x)</TD><TD align="right"> 2
</TD></TR><TR><TD>Enzyme (no.Lab: 48)</TD><TD align="right"> 1
</TD></TR><TR><TD>H2O</TD><TD align="right"> 5
</TD></TR><TR><TD><B>Total volume </B></TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><UL><LI> agarose gel was performed with digested samples and revealed 3 distinct bands, while only 1-2 bands have been expected. Because of the density of the bands it might be that the digestion with DpnI did not work properly, thus additional 1 µl of DpnI was added to each sample. Incubation @ 37 C lasted 1 h.
</LI></UL><H3><SPAN class="mw-headline" id="55.Labortag_11.07.2010"><P style="font-size:17px; background-color:#00dd77;">55.Labortag 11.07.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Splitting_of_HT1080_cells"><P style="font-size:15px; background-color:#66bbff;">Splitting of HT1080 cells</P></SPAN></H4><P><B>Investigator: Patrick</B>
HT 1080 and HEK 293 cells were split.  
@ Kira: fetch new trypsin if it is not sufficient for the next splitting !
</P><H3><SPAN class="mw-headline" id="56.Labortag_12.07.2010:"><P style="font-size:17px; background-color:#00dd77;">56.Labortag 12.07.2010:</P></SPAN></H3><H4><SPAN class="mw-headline" id="Cloning_CFP_into_pSB1C3"><P style="font-size:15px; background-color:#66bbff;">Cloning CFP into pSB1C3</P></SPAN></H4><P><B>Investigator: Anna, Kerstin, Anissa</B></P><P style="font-size:13px; font-weight: bold; color: blue;"><U>Practical Cloning:</U></P><UL><LI>experiment date: 12.07.2010 ; time: 11.00
</LI><LI>plasmid: pSB1c3_CFP (p48) 
</LI><LI>Vector: name: pSB1c3 number:- production date: - origin: iGEM
</LI><LI>Insert: name: pMA_CFP number:- production date: - origin: Sven
</LI></UL><LI>new vector name: pSB1c3_CFP (p48)
</LI><LI>buffer used: 3 ; Restriction-enzymes used: Enzyme 1 (no. Lab:28) EcoRI ; Enzyme 2 (no.Lab:48) PstI ;Enzyme 3 (no.Lab:77) DpNI
</LI><LI>DNA concentration (vector): 25 µg/µl ; DNA concentration (insert): 436,8 µg/µl
</LI><P><B>Comments:</B>Third enzyme was only used to digest template from vector pcr
</P><P style="font-size:15px; font-weight: bold; color: blue;">Digestion</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl </TD><TD align="right">volume of insert /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right">4 </TD><TD align="right">5
</TD></TR><TR><TD> BSA (100x) </TD><TD align="right">0,2 </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer ____ (10x)</TD><TD align="right">1,5 </TD><TD align="right">2
</TD></TR><TR><TD>Enzyme 1 (no.Lab:23)</TD><TD align="right">0,5 </TD><TD align="right">1
</TD></TR><TR><TD>Enzyme 2 (no.Lab:48)</TD><TD align="right">0,5 </TD><TD align="right">1
</TD></TR><TR><TD>Enzyme 3 (no.Lab:77)</TD><TD align="right">0,5 </TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">7,8 </TD><TD align="right">9
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 15</TD><TD align="right">20
</TD></TR></TBODY></TABLE><UL><LI>Incubation: 1h
</LI></UL><P style="font-size:15px; font-weight: bold; color: blue;">Agarose-Gel:</P><P>
0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED (3-6µl), at 110 Volt, running time:45 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (5x/6x)/µl
</TH><TH>Expected size 1 (Geneious)
</TH><TH>Expected size 2 (Geneious)
</TH><TH>Expected size 3 (Geneious)
</TH><TH>Expected size 4 (Geneious)
</TH></TR><TR><TD> 2
</TD><TD>20 µl
</TD><TD>3,3 µl
</TD><TD>732 bp (Insert)
</TD><TD>2074 bp (Vector)
</TD><TD>____ bp
</TD><TD>____ bp
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>Sample 23,3 µl
</TH></TR><TR><TH>Lane
</TH><TD>1
</TD><TD>3
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color: blue;">Gelextraction</P><P>
Gel measurement:
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Sample</B></TD><TD align="left"><B>weight</B></TD></TR><TR><TD align="left"> 2 (Insert) </TD><TD align="left"> 0,14g
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color: blue;">Ligation</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Insert</B></TD><TD align="left"><B>Vector</B></TD></TR><TR><TD align="left"><B>Volume/µl</B></TD><TD align="left"> 6,77 µl</TD><TD align="left"> 2,23 µl
</TD></TR></TBODY></TABLE><P><B>TRAFO-EVALUATION</B></P><UL><LI> Unfortunately the agar plate with p48, pSB1c3_CFP [XL1B, Cm, A.B.] does not contain any colonies. Thus, the plate was incubated @ 37 C for another 5 hours. Still no detectable colonies. This was double checked by Sven.
</LI></UL><H4><SPAN class="mw-headline" id="Seeding_HT1080_.2B_Passage"><P style="font-size:15px; background-color:#66bbff;">Seeding HT1080 + Passage</P></SPAN></H4><UL><LI><B>Investigators: Adrian, Patrick</B></LI><LI> 3* 6well dishes
</LI></UL><TABLE border="1"><TBODY><TR><TD>3*10^5 cells </TD><TD align="right">3*10^5 cells</TD><TD align="right">3*10^5 cells
</TD></TR><TR><TD>3*10^5 cells </TD><TD align="right">3*10^5 cells</TD><TD align="right">3*10^5 cells
</TD></TR></TBODY></TABLE><UL><LI> The amount of cells was doubled in first place and diluted in following steps (for further information ask patrick and adrian)
</LI></UL><H4><SPAN class="mw-headline" id="PCR_of_beta-globin:"><P style="font-size:15px; background-color:#66bbff;">PCR of beta-globin:</P></SPAN></H4><UL><LI><B>Investigator: Achim</B></LI><LI>PCR program: BETA...(?)
</LI></UL><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl
</TD></TR><TR><TD> 5x Phusion HF buffer  </TD><TD align="right"> 10
</TD></TR><TR><TD> 10 mM dNTP mix </TD><TD align="right"> 1
</TD></TR><TR><TD> beta-globin_primer_for</TD><TD align="right"> 2,5
</TD></TR><TR><TD> beta-globin_primer_rev</TD><TD align="right"> 2,5
</TD></TR><TR><TD>DNA template</TD><TD align="right"> 8 ng
</TD></TR><TR><TD>DMSO</TD><TD align="right"> -
</TD></TR><TR><TD>Phusion polymerase</TD><TD align="right"> 0,5
</TD></TR><TR><TD>H2O</TD><TD align="right"> 33,2
</TD></TR><TR><TD><B>Total volume (e.g. 50 µl)</B></TD><TD align="right"> 50
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color: blue;">Agarose-Gel:</P><P>
0,625 g Agarose, 50 ml TAE (1,25%), 3 µl GELRED (3-6µl), at 115 Volt, running time: 30 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (5x/6x)/µl
</TH><TH>Expected size (Geneious)
</TH></TR><TR><TD>p49: beta-globin
</TD><TD>50 µl
</TD><TD>10 µl
</TD><TD>509 bp
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker (8 µl)
</TH><TH>Sample 50 /µl
</TH></TR><TR><TH>Lane
</TH><TD>1
</TD><TD>4
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color: blue;">Gelextraction</P><P>
Gel measurement:
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Sample</B></TD><TD align="left"><B>weight</B></TD></TR><TR><TD align="left"> p49: hgH </TD><TD align="left"> 0,21 g
</TD></TR></TBODY></TABLE><UL><LI>DNA concentration (hgH): 34,87 ng/µl
</LI></UL><P><B>Comments:</B> PCR was succesful:-)
to do: Ligation of PCR product into standard plasmid which we´ve received wihthin the DNA distrivution kit of iGEM-HQ .
</P><H4><SPAN class="mw-headline" id="Midi-Prep_preparation_pAAV_RC"><P style="font-size:15px; background-color:#66bbff;">Midi-Prep preparation pAAV_RC</P></SPAN></H4><P><B>Investigator: Bea</B></P><UL><LI>200mL LB-Medium and 200µl Amp
</LI><LI>inoculated with glycerol stock containing pAAV_RC
</LI><LI>20:00 p.m.: put intp 37°C room on shaker
</LI></UL><H4><SPAN class="mw-headline" id="First_steps_in_producing_ITR_BioBricks_via_Fancy_Method"><P style="font-size:15px; background-color:#66bbff;">First steps in producing ITR BioBricks via Fancy Method</P></SPAN></H4><P><B>Investigators: Hanna, Chris W.</B></P><P style="font-size:15px; font-weight: bold; color: fuchsia;">Digestion of pGA14_Cerulean-CFP</P><P><B>1. Digestion with EcoRI and NotI:</B></P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 3.5
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 3 (10x) </TD><TD align="right"> 2
</TD></TR><TR><TD>Enzyme EcoRI (no.Lab:23)</TD><TD align="right"> 1
</TD></TR><TR><TD>Enzyme NotI (no.Lab:46)</TD><TD align="right"> 1
</TD></TR><TR><TD>H2O</TD><TD align="right"> 10.5
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><P>
Incubation: 1.5 hours, 37°C
</P><P>Gel run: 20 µL sample + 4 µL loading dye (6x); 8µL marker 
Expected fragment size: ~ 2900 bp
</P><P>Gel extraction:

Sample weight: 0.15 g 
Gel extraction was performed following the standard protocol. 
DNA concentration: 26.8 ng/µL
<B>1. Digestion with XbaI and PstI:</B></P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 3.5
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 3 (10x) </TD><TD align="right"> 2
</TD></TR><TR><TD>Enzyme XbaI (no.Lab:63)</TD><TD align="right"> 1.25
</TD></TR><TR><TD>Enzyme PstI (no.Lab:48)</TD><TD align="right"> 0.75
</TD></TR><TR><TD>H2O</TD><TD align="right"> 10.5
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><P>
Incubation: 1.5 hours, 37°C
</P><P>Gel run: 20 µL sample + 4 µL loading dye (6x); 8µL marker 
Expected fragment size: ~ 2900 bp
</P><P>Gel extraction:

Sample weight: 0.17 g 
Gel extraction was performed following the standard protocol. 
DNA concentration: 22.9 ng/µL
</P><P style="font-size:15px; font-weight: bold; color: fuchsia;">Digestion of pAAV_MCS (P9) with AlwNI</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 11.6
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right"> -
</TD></TR><TR><TD> Buffer 4 (10x) </TD><TD align="right"> 2.5
</TD></TR><TR><TD>Enzyme AlwNI (no.Lab:120)</TD><TD align="right"> 2.5
</TD></TR><TR><TD>H2O</TD><TD align="right"> 8.4
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 25
</TD></TR></TBODY></TABLE><P>
Incubation: 1 hours + 20 minutes, 37°C
</P><P>Gel run: 20 µL sample + 4 µL loading dye (6x); 8µL marker 
Expected fragment sizes: 2982 bp = right ITR and 1674 bp = left ITR
</P><P>Gel extraction:
</P><UL><LI> Sample: right ITR = 0.15 g </LI><LI> Sample: left ITR = 0.12 g </LI></UL><P>Gel extraction was performed following the standard protocol. 
DNA concentration: 
</P><UL><LI> right ITR: 44.5 ng/µL
</LI><LI> left ITR: 32.8 ng/µL
</LI></UL><P style="font-size:15px; font-weight: bold; color: fuchsia;">Digestion of ITR fragments with NotI and PstI</P><P>
Digestion of <B>both</B> fragments was performed as follows:
</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 19 µL
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right"> 1.5
</TD></TR><TR><TD> Buffer 3 (10x) </TD><TD align="right"> 2.5
</TD></TR><TR><TD>Enzyme NotI (no.Lab:46)</TD><TD align="right"> 1
</TD></TR><TR><TD>Enzyme PstI (no.Lab:48)</TD><TD align="right"> 1
</TD></TR><TR><TD>H2O</TD><TD align="right"> -
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 25
</TD></TR></TBODY></TABLE><P>
Incubation: 1 hours + 15 minutes, 37°C
</P><P>Gel run:
</P><UL><LI> due to the small fragment sizes, we prepared a 2% agarose gel: 1.3 g agarose + 65 ml TAE buffer
</LI><LI> 25 µL of each sample + 5 µL loading dye (6x); 8µL marker </LI><LI> Expected fragment sizes: ~ 150 bp (both)
</LI><LI> Big problem: Small fragments and little amounts -&gt; 150 bp fragments were nearly not visible. In addition to that, the right ITR sample delivered 2 fragments in the referring range. Therefore two gel extractions (&quot;right ITR oben&quot; and &quot;right ITR unten&quot;) were performed.
</LI></UL><P>Gel extraction:
</P><UL><LI> Sample: right ITR oben = 0.11 g </LI><LI> Sample: right ITR unten = 0.12 g </LI><LI> Sample: left ITR = 0.2 g </LI></UL><P>Gel extraction was performed with special columns provided from Sven. In order to elute the samples 12 µL EB were used, columns were placed for 2 minutes into 50°C water bath and then centrifuged at 13200 rpm for 1 minute. 
DNA concentration: 
</P><UL><LI> right ITR oben: 23.9 ng/µL
</LI><LI> right ITR unten: 13.8 ng/µL
</LI><LI> left ITR: 4.46 ng/µL
</LI></UL><P style="font-size:15px; font-weight: bold; color: fuchsia;">Hybridization of ITR-Präfix-Oligos</P><P><B>left ITR:</B></P><UL><LI> 10 µL Oligo 1 (O43 = Präfix_for)
</LI><LI> 10 µL Oligo 2 (O44 = Präfix_rev)
</LI><LI> 4 µL 100 mM TrisHCl pH8
</LI><LI> 8 µL 5 mM MgCl2
</LI><LI> 8 µL H2O
</LI></UL><P>Total: 40 mL
</P><P><B>right ITR:</B></P><UL><LI> 10 µL Oligo 1 (O47 = Präfix_for)
</LI><LI> 10 µL Oligo 2 (O48 = Präfix_rev)
</LI><LI> 4 µL 100 mM TrisHCl pH8
</LI><LI> 8 µL 5 mM MgCl2
</LI><LI> 8 µL H2O
</LI></UL><P>Total: 40 mL
</P><P>Because some of the oligos produce strong secondary structures, we used the ORIGAMI1 program:
1) initial denaturation: 99°C, 7 minutes
2) 99°C, 1 minute
3) 73 x repetition of 2) -&gt; -1°C, R = 0.3 °C/sec
4) hold 4°C</P><P style="font-size:15px; font-weight: bold; color: fuchsia;">Ligation of pGA14 + ITR + Präfix</P><P><B>1. right ITR oben:</B></P><UL><LI> Präfix: 1 µL
</LI><LI> ITR: 7 µL
</LI><LI> vector (pGA14): 9 µL
</LI><LI> buffer: 2 µL
</LI><LI> T4 ligase: 1 µL
</LI></UL><P>Total volume: 20 µL
</P><P><B>2. right ITR unten:</B></P><UL><LI> Präfix: 1 µL
</LI><LI> ITR: 7 µL
</LI><LI> vector (pGA14): 9 µL
</LI><LI> buffer: 2 µL
</LI><LI> T4 ligase: 1 µL
</LI></UL><P>Total volume: 20 µL
</P><P><B>3. left ITR:</B></P><UL><LI> Präfix: 1 µL
</LI><LI> ITR: 7 µL
</LI><LI> vector (pGA14): 9 µL
</LI><LI> buffer: 2 µL
</LI><LI> T4 ligase: 1 µL
</LI></UL><P>Total volume: 20 µL
</P><P>Samples will be incubating over night at 16°C.
</P><H4><SPAN class="mw-headline" id="Ordering_genes_for_the_replacement_of_Rep_and_Cap_parts"><P style="font-size:15px; background-color:#66bbff;">Ordering genes for the replacement of Rep and Cap parts</P></SPAN></H4><P><B>Investigator: Volker</B> guided by Sven &amp; Tobi
</P><P>The replacement for the Rep ORF from base 1510 to base 2008 and for the Cap ORF from base 3064 to base 3985 was ordered from Mr. Gene. 
</P><P><A href="/File:Freiburg10_Rep_Cap_Replacement.pdf" title="File:Freiburg10 Rep Cap Replacement.pdf">File:Freiburg10 Rep Cap Replacement.pdf</A></P><H4><SPAN class="mw-headline" id="Sequencing_of_GOI"><P style="font-size:15px; background-color:#66bbff;">Sequencing of GOI</P></SPAN></H4><P><B>Investigator: Hanna</B>
This time the sequencing delivered good results: </P><P style="font-size:15px; font-weight: bold; color: fuchsia;">Conclusion: The designed primers can always be used in order to sequence any gene of interest!!!</P><H3><SPAN class="mw-headline" id="57.Labortag_13.07.2010:"><P style="font-size:17px; background-color:#00dd77;">57.Labortag 13.07.2010:</P></SPAN></H3><H4><SPAN class="mw-headline" id="Endotoxin-free_Midi_Prep_of_pAAV_RC"><P style="font-size:15px; background-color:#66bbff;">Endotoxin-free Midi Prep of pAAV_RC</P></SPAN></H4><P><B>Investigators: Chris, Stefan, Patrick</B>
Deviations from the Standard Protocol: 
As the lysate was applied to the column filter, about 1/5 of the lysate was spilled. 
Result of the Midi-prep: about 18 ng/µl =&gt; waste 
A new Midi-Pred will be prepared for tomorrow: 200 ml LB, 0,2 ml Amp, E.coli XL-1-blue (with pAAV_RC plasmid).
</P><H4><SPAN class="mw-headline" id="HEK-293_cells"><P style="font-size:15px; background-color:#66bbff;">HEK-293 cells</P></SPAN></H4><P>The cells were split according to the standard protocol. Investigator: Patrick
</P><H4><SPAN class="mw-headline" id="Oligos_pMGMK_TK30_arrived"><P style="font-size:15px; background-color:#66bbff;">Oligos pMGMK_TK30 arrived</P></SPAN></H4><P>New Oligos arrived for inserting RFC25 to thymidinkinase_gmk construct!(number O60 to O65). They were resuspended in water and a 1:10 dilution was done. They were put in Primer Box 2. 
</P><P style="font-size:12px; font-weight: bold; color: green;"><U>to do: perform PCR tomorrow and clone PCR product into pAAV_RFC25</U></P><H4><SPAN class="mw-headline" id="Transduction"><P style="font-size:15px; background-color:#66bbff;">Transduction</P></SPAN></H4><P><B>Investigators: Adrian, Patrick</B></P><UL><LI> 3 Plates got transduced
</LI></UL><P>1. Plate I(A is up) completely 10 µg
</P><TABLE border="1"><TBODY><TR><TD> 300µl dropped </TD><TD align="right">300µl resuspended</TD><TD align="right">450µl resuspended
</TD></TR><TR><TD> 600µl dropped</TD><TD align="right">600µl resuspended</TD><TD align="right"> control
</TD></TR></TBODY></TABLE><P>
2. Plate II(A is up)completely 18 µg
</P><TABLE border="1"><TBODY><TR><TD> 300µl dropped </TD><TD align="right">300µl resuspended</TD><TD align="right">450µl resuspended
</TD></TR><TR><TD> 600µl dropped</TD><TD align="right">600µl resuspended</TD><TD align="right"> control
</TD></TR></TBODY></TABLE><P>
3. Plate III(A is up) completely 24 µg
</P><TABLE border="1"><TBODY><TR><TD> 300µl dropped </TD><TD align="right">300µl resuspended</TD><TD align="right">450µl resuspended
</TD></TR><TR><TD> 600µl dropped</TD><TD align="right">600µl resuspended</TD><TD align="right"> control
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Results_of_Trafo_of_cloning_CFP_into_pSB1C3"><P style="font-size:15px; background-color:#66bbff;">Results of Trafo of cloning CFP into pSB1C3</P></SPAN></H4><P><B>Investigator: Bea</B>
Results: no clones on trafo plate even after longer incubation at 37 °C<B>Solution: perform digestion of linearized plasmid pSB1C3 and pMA_CFP again!!!</B></P><H4><SPAN class="mw-headline" id="Repetition_of_cloning_of_CFP_into_pSB1C3"><P style="font-size:15px; background-color:#66bbff;">Repetition of cloning of CFP into pSB1C3</P></SPAN></H4><P><B>Investigator: Bea</B></P><UL><LI>experiment date: 13.07.2010 ; time: 16:00 
</LI><LI>Vector: name: pSB1c3 number: do not have any number production date: - origin: iGEM HQ sent per DNA distribution kit
</LI><LI>Insert: name: pMA_CFP number:- production date: - origin: iGEM 2009
</LI></UL><LI>new vector name: pSB1c3_CFP (P48)
</LI><LI>buffer used: 3 ; Restriction-enzymes used: Enzyme 1 (no. Lab: 23) EcoRI ; Enzyme 2 (no.Lab: 48) PstI ;Enzyme 3 (no.Lab: 77) DpnI
</LI><LI>DNA concentration (vector: pSB1C3): 25 ng/µl ; DNA concentration (insert pMA_CFP): 436,8 µg/µl
</LI><P><B>Comments:</B>The standard plasmid vector (pSB1C3) received from the iGEM HQ is linearized (PCR product) and need to be cut with EcoRI, PstI and optionally with DpnI to digest the DNA template.  
</P><P style="font-size:15px; font-weight: bold; color: blue;">Digestion of pSB1C3 and pMA_CFP </P><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume of pSB1C3 /µl</B></TD><TD align="right"><B>volume of pMA_CFP /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">4 </TD><TD align="right">4,6
</TD></TR><TR><TD> BSA (100x) </TD><TD align="right">1,5 </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 3 (10x)</TD><TD align="right">1,5 </TD><TD align="right">2
</TD></TR><TR><TD>Enzyme EcoRI (no.Lab:23)</TD><TD align="right">0,5 </TD><TD align="right">1
</TD></TR><TR><TD>Enzyme PstI (no.Lab:48)</TD><TD align="right">0,5 </TD><TD align="right">1
</TD></TR><TR><TD>Enzyme DpnI (no.Lab:77)</TD><TD align="right">0,5 </TD><TD align="right">-
</TD></TR><TR><TD>H2O</TD><TD align="right">6,5 </TD><TD align="right">9
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 15</TD><TD align="right">20
</TD></TR><TR><TD><B>Incubation</B></TD><TD>30 min at 37°C in PCR cycler</TD><TD>45min at 37°C in PCR cycler
</TD></TR><TR><TD><B>commment</B></TD><TD>after digestion: 80°C 20 min heat kill</TD><TD> load on agarose gel
</TD></TR></TBODY></TABLE><UL><LI>Load pMA_CFP digestion reaction onto 1% agarose gel
</LI></UL><P style="font-size:15px; font-weight: bold; color: blue;">1% Agarose-Gel</P><P>
0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED (3-6µl), at 110 Volt, running time:45 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (/6x)/µl
</TH><TH>Expected size 1 (Geneious)
</TH><TH>Expected size 2 (Geneious)
</TH></TR><TR><TD>pMA_CFP
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD>732 bp (Insert)
</TD><TD>2974 bp (Vector)
</TD></TR></TBODY></TABLE><P><B>Loading plan for agarose gel</B>: 
Marker used: GeneRuler ladder mix (Fermentas)</P><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>Sample 24 µl
</TH></TR><TR><TH>Lane
</TH><TD>1
</TD><TD>3
</TD></TR></TBODY></TABLE><P><B>Results</B>: Expected bands too high, but lower band (= CFP) was cut out and gel extraction have been performed folowing standard protocol.
</P><P style="font-size:12px; font-weight: bold; color: blue;">Gel extraction </P>has been performed and Nanodrop measurement reveals following concentraion of the insert CFP:
<UL><LI>c(CFP)= 6,75ng/µL
</LI></UL><UL><LI>Ligation ratio of vector/insert was calculated with LabTools
</LI><LI>v(pSB1C3)= 4,4 µL
</LI><LI>v(CFP)= 4,6 µL
</LI><LI>vQuickligase Buffer (2x)= 10µL
</LI><LI>vQuickLigase = 1µL
</LI></UL><P style="font-size:12px; font-weight: bold; color: blue;">Transformation </P><UL><LI>used bacterial strain: XL-1B
</LI><LI>2µL of DNA (pSB1C3_CFP)
</LI><LI>Trafo plate in 37°C room for incubating at 23:00
</LI></UL><H4><SPAN class="mw-headline" id="TRANSFORMATION_of_Quickchange_PstI"><P style="font-size:15px; background-color:#66bbff;">TRANSFORMATION of Quickchange PstI</P></SPAN></H4><P><B>Investigator: Kira</B></P><UL><LI>Transfection was performed according to the standard protocol with XL1B cells.
</LI></UL><H3><SPAN class="mw-headline" id="58.Labortag_14.07.2010"><P style="font-size:17px; background-color:#00dd77;">58.Labortag 14.07.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Trafo_evaluation_of_Quickchange_PstI"><P style="font-size:15px; background-color:#66bbff;">Trafo evaluation of Quickchange PstI</P></SPAN></H4><P><B>Investigator: Kira</B>
Both plates contained lots of colonies, while 8-10 colonies are considered for inoculation to perform a mini prep and an additional test digestion. 
However, the experiment has to be repeated.
</P><H4><SPAN class="mw-headline" id="Quickchange_for_the_PstI_restriction_sites_in_pAAV-RC_.28modified.29">Quickchange for the PstI restriction sites in pAAV-RC (modified)</SPAN></H4><P><B>Investigators: Kira</B></P><P><B>PCR program</B>: PstI 
</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume in µl
</TD></TR><TR><TD> 10x Phusion HF buffer  </TD><TD align="right"> 2.5 µl
</TD></TR><TR><TD> 10 mM dNTP mix </TD><TD align="right"> 0.5 µl
</TD></TR><TR><TD> __310 ____primer_for (1:10 dilution)</TD><TD align="right">  0,39 µl
</TD></TR><TR><TD> __310____primer_rev (1:10 dilution)</TD><TD align="right">  0,39 µl
</TD></TR><TR><TD>DNA template (1:40 dilution)</TD><TD align="right">  0,5 µl
</TD></TR><TR><TD>DMSO</TD><TD align="right">  0.5 µl
</TD></TR><TR><TD>Phusion polymerase</TD><TD align="right">  0.5 µl
</TD></TR><TR><TD>H2O</TD><TD align="right"> 19.72 µl
</TD></TR><TR><TD><B>Total volume (e.g. 50 µl)</B></TD><TD align="right"> 25 µl
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume in µl
</TD></TR><TR><TD> 10x Phusion HF buffer  </TD><TD align="right"> 2.5 µl
</TD></TR><TR><TD> 10 mM dNTP mix </TD><TD align="right"> 0.5 µl
</TD></TR><TR><TD> __4073____primer_for (1:10 dilution)</TD><TD align="right">  0.33 µl
</TD></TR><TR><TD> ___4073___primer_rev (1:10 dilution)</TD><TD align="right">  0.35 µl
</TD></TR><TR><TD>DNA template (1:40 dilution)</TD><TD align="right">  0.5 µl
</TD></TR><TR><TD>DMSO</TD><TD align="right">  0.5 µl
</TD></TR><TR><TD>Phusion polymerase</TD><TD align="right">  0.5 µl
</TD></TR><TR><TD>H2O</TD><TD align="right"> 19.82 µl
</TD></TR><TR><TD><B>Total volume (e.g. 50 µl)</B></TD><TD align="right"> 25 µl
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD> cycles   </TD><TD align="right">temperature </TD><TD align="right">time
</TD></TR><TR><TD> 1 </TD><TD align="right">95 C </TD><TD align="right">2 min
</TD></TR><TR><TD> 20 </TD><TD align="right">95 C </TD><TD align="right">30 sec
</TD></TR><TR><TD align="right">68 C </TD><TD align="right">7 min
</TD></TR></TBODY></TABLE><UL><LI>PCR products were digested with 0.5 µl and incubated @ 37 C for 1 h
</LI></UL><P><B>TRANSFORMATION</B></P><UL><LI>The transformation of XL1B was performed according to the standard protocol but with DYT instead of LB.
</LI></UL><H4><SPAN class="mw-headline" id="Results_of_trafo_cloning_pSB1C3_CFP"><P style="font-size:15px; background-color:#66bbff;">Results of trafo cloning pSB1C3_CFP</P></SPAN></H4><P><B>Investigator: Bea</B>
Comments: experiment have been repeated because no clones were grown after the last trafo!</P><UL><LI>There have been clones grown on the trafo plate. Four clones were picked and inoculated on LB_Medium containing Cm (Chloramphenicol).
</LI><LI>18:00: 37°C room on shaker. Incubating over night!
</LI></UL><H4><SPAN class="mw-headline" id="Seed_AAV-293"><P style="font-size:15px; background-color:#66bbff;">Seed AAV-293</P></SPAN></H4><P><B>Investigators: Adrian ,Bea</B></P><UL><LI> 2x75cm2 culture flasks
</LI><LI> after counting via neubauer chamber (2,2*20*10000*(x/4))= should be around 3,xxx so it would be possible to prepare 4x10cm petri dishes with 3.000.000 cells per plate and passage 1.500.000 cells to a 75cm.
</LI></UL><H4><SPAN class="mw-headline" id="DAAD_Timetable"><P style="font-size:15px; background-color:#66bbff;">DAAD Timetable</P></SPAN></H4><P><B>Investigators: Adrian, Kerstin</B>
Prepared timetable for DAAD
</P><H4><SPAN class="mw-headline" id="Summary"><P style="font-size:15px; background-color:#66bbff;">Summary</P></SPAN></H4><P><B>Investigators: all</B></P><P style="font-size:12px; color: blue;">have not been done yet! to do for tomorrow! deadline of submitting project descrpition is 16th of july!!  (Bea)</P><H4><SPAN class="mw-headline" id="Ordering_ITR.2BTK.2FGMK"><P style="font-size:15px; background-color:#66bbff;">Ordering ITR+TK/GMK</P></SPAN></H4><P><B>Investigators: Bea, Sven</B></P><UL><UL><LI>It has been ordered the TK30 with NgoMIV add-on prefix and ageI and SpeI add-on suffix combined with the  wt_left_ITR in the RFC10 standard. 
</LI></UL></UL><H4><SPAN class="mw-headline" id="Endotoxoin_free_Midi_prep_mAAV_RC"><P style="font-size:15px; background-color:#66bbff;">Endotoxoin free Midi prep mAAV_RC</P></SPAN></H4><P><B>Investigator: Chris W.</B>Midi prep done with the Qiagen-Kit
OD of the Bacteria: 3.07
concentration:  429,8 and 378,5 ng/µl
Labeled as P50 and stored in the -20° Freezer
</P><H4><SPAN class="mw-headline" id="MGMK_TK30:PCR_with_iGEM_restriction-sites_and_cloning_into_pAAV_RFC25"><P style="font-size:15px; background-color:#66bbff;">MGMK_TK30:PCR with iGEM restriction-sites and cloning into pAAV_RFC25</P></SPAN></H4><P><B>Investigators: Achim,Bea,Anissa</B></P><P style="font-size:20px; font-weight: bold; color: blue;"><U>PCR:</U></P><UL><LI>PCR program:mGMK_TK30
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume in µl</B></TD></TR><TR><TD> 5x Phusion HF buffer  </TD><TD align="right">10
</TD></TR><TR><TD> 10 mM dNTP mix </TD><TD align="right">1
</TD></TR><TR><TD>pMGMK_TK30_RFC25_prefix_for (060) (1:10 dilution)</TD><TD align="right">2,5 (0,5 µM)
</TD></TR><TR><TD>pMGMK_TK30_RFC25_suffix_rev (063) (1:10 dilution)</TD><TD align="right">2,5 (0,5 µM)
</TD></TR><TR><TD>DNA template</TD><TD align="right"> ~8 ng
</TD></TR><TR><TD>DMSO</TD><TD align="right"> -
</TD></TR><TR><TD>Phusion polymerase</TD><TD align="right">0,5
</TD></TR><TR><TD>H2O</TD><TD align="right">fill up to 50 µl
</TD></TR><TR><TD><B>Total volume (e.g. 50 µl)</B></TD><TD align="right">50
</TD></TR></TBODY></TABLE><P><B>Comments:</B>Oligos with add-on tails (prefix and suffix), but TK30 still contains three iGEM restriction sites (TK30 without restriction sites was ordered today)
</P><P style="font-size:20px; font-weight: bold; color: blue;"><U>Practical Cloning:</U></P><UL><LI>Vector: name:pAAV_RFC25 number:p42 
</LI><LI>Insert: name: MGMK_TK30 (after PCR with iGEM standard) number:-
</LI></UL><LI>new vector name: pAAV_RFC25_MGMK_TK30
</LI><LI>buffer used:4 ; Restriction-enzymes used: Enzyme 1 (no. Lab:149) AgeI ; Enzyme 2 (no.Lab:63) XbaI
</LI><LI>DNA concentration (vector):321,5 ng/µl ; DNA concentration (insert): 
</LI><P><B>Comments:</B>_____
</P><P style="font-size:15px; font-weight: bold; color: blue;">Digestion</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl </TD><TD align="right">volume of insert (=PCR product) /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right">3,1 (1 µg) </TD><TD align="right"> 30 (0,691 µg)
</TD></TR><TR><TD> BSA (100x) </TD><TD align="right">2 </TD><TD align="right"> 5 µl
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right"> 5µl
</TD></TR><TR><TD>AgeI (no.Lab:149)</TD><TD align="right">1 </TD><TD align="right"> 2 µl
</TD></TR><TR><TD>XbaI(no.Lab:63)</TD><TD align="right">0,75 </TD><TD align="right"> 1 µl
</TD></TR><TR><TD>H2O</TD><TD align="right">11,15 </TD><TD align="right"> 7 µl
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right">20 </TD><TD align="right"> 50 µl
</TD></TR></TBODY></TABLE><UL><LI>Incubation: Vector: 1h; PCR product: 2h
</LI></UL><P style="font-size:15px; font-weight: bold; color: blue;">Agarose-Gel:</P><P>
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED (3-6µl), at 110 (PCR); 115 (vector) Volt, running time:____ minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (5x/6x)/µl
</TH><TH>Expected size (Geneious)
</TH></TR><TR><TD>PCR product
</TD><TD>50 µl
</TD><TD>10 µl
</TD><TD>~1692 bp
</TD></TR><TR><TD>vector
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD>4646 bp
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>Sample 60 /µl
</TH></TR><TR><TH>Lane
</TH><TD>8 µl
</TD><TD>PCR
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>Sample 24 /µl
</TH></TR><TR><TH>Lane
</TH><TD>8 µl
</TD><TD>vector
</TD></TR></TBODY></TABLE><P><B>Comments:</B> two gels were made, one for the vector, one for the pcr product </P><UL><LI>Gel extraction was carried out according to standard protocol
</LI><LI>Nanodrop: 
</LI><LI>pAAV_rfc 25 Digestion: 15,07 ng/µl
</LI><LI>mgmk tk30 pcr product: 23,04 ng/µl
</LI></UL><UL><LI>Puricifation using Quiagen QIAquick PCR Purification Kit Protocol
</LI><LI>c(mgmk_tk30) = 8,1ng/µL
</LI></UL><LI>Ligation of mgmk tk30 and pAAV_rfc 25
</LI><LI>v(vector) = 2,97 µL
</LI><LI>v(insert) = 6,03 µL
</LI><UL><LI>Trafo
</LI><LI>put in 37°C room at 22:00
</LI></UL><H4><SPAN class="mw-headline" id="Ordering_of_Oligos"><P style="font-size:15px; background-color:#66bbff;">Ordering of Oligos</P></SPAN></H4><P><B>Investigator:Volker</B></P><P>The following four Oligos were ordered:</P><UL><LI>SDM BamHI (729) for
</LI><LI>SDM BamHI (729) rev
</LI><LI>SDM SalI (1110) for
</LI><LI>SDM SalI (1110) rev
</LI></UL><H3><SPAN class="mw-headline" id="59.Labortag_15.07.2010"><P style="font-size:17px; background-color:#00dd77;">59.Labortag 15.07.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Picking_clones_for_mini-prep"><P style="font-size:15px; background-color:#66bbff;">Picking clones for mini-prep</P></SPAN></H4><P><B>Investigators: Adrian, Chris, Stefan</B></P><P>4 clones (1-4) of pAAV_RFC25_mgmk_tk30 where picked according to protocol for mini-prep (tomorrow). 
Antibiotic: Amp 
Time: 18:30
<B>Investigators: Stefan, Patrick</B><B>New millipore H2O, LB + Agar and DMEM medium were prepared</B></P><H4><SPAN class="mw-headline" id="Splitting_of_HT1080_cells_2"><P style="font-size:15px; background-color:#66bbff;">Splitting of HT1080 cells</P></SPAN></H4><P><B>Investigators: Chris, Patrick </B>
HT 1080 cells were split according to the standard protocol
</P><H4><SPAN class="mw-headline" id="PstI_Quickchange_Trafo_from_14.07"><P style="font-size:15px; background-color:#66bbff;">PstI Quickchange Trafo from 14.07</P></SPAN></H4><P><B>Investigator: Kira</B>
Trafo evaluation: Unfortunately no colonies are present.
</P><H4><SPAN class="mw-headline" id="MINI_PREP_.28PstI_Quickchange_Trafo_from_13.07.29"><P style="font-size:15px; background-color:#66bbff;">MINI PREP (PstI Quickchange Trafo from 13.07)</P></SPAN></H4><P><B>Investigator: Kira</B></P><P>@Kira: ich habe die Clone benannt: SDM310 Nr. 1 ist jetzt P58 und sdm4073 nr 1 ist jetzt P59. Bitte eintragen.
</P><H4><SPAN class="mw-headline" id="TEST_DIGESTION_.28PstI_Quickchange_Trafo_from_13.07.29"><P style="font-size:15px; background-color:#66bbff;">TEST DIGESTION (PstI Quickchange Trafo from 13.07)</P></SPAN></H4><P><B>Investigator: Kira</B></P><H4><SPAN class="mw-headline" id="FACS"><P style="font-size:15px; background-color:#66bbff;">FACS</P></SPAN></H4><P><B>Investigators: Kerstin, Anna</B></P><P>FACS analysis of different treated HT1080 cells was done (18 samples)
</P><P>The first percentage refers to the living and fluorescent cells and the second percentage to the total amout of living cells.
</P><P>The data are listed this way:
</P><P>1. Plate I(A is up)
</P><TABLE border="1"><TBODY><TR><TD>A1</TD><TD align="right">A2</TD><TD align="right">A3
</TD></TR><TR><TD>B1</TD><TD align="right">B2</TD><TD align="right">B3
</TD></TR></TBODY></TABLE><P>1=  Plate 1 A1 : 0% YFP-positive Cells (there were only a few cells in this sample)
2=  Plate 1 A2 : 27,7% YFP-positve Cells  from 73,3% 
3=  Plate 1 A3 : 17,3%       &quot;            from 63,1% 
4=  Plate 2 A1 : 32,9%       &quot;            from 75,3% 
5=  Plate 2 A2 : 27,2%       &quot;            from 76,5% 
6=  Plate 2 A3 : 30,7%       &quot;            from 73,4% 
7=  Plate 3 A1 : 26,8%       &quot;            from 76,0%  
8=  Plate 3 A2 : 30,5%       &quot;            from 74,2% 
9=  Plate 3 A3 : 30,7%       &quot;            from 75,2% 
10= Plate 1 B1 : 6,25%       &quot;            from 36,1% 
11= Plate 1 B2 : 6,42%       &quot;            from 68,4% 
12= Plate 1 B3 : 0,05%       &quot;            from 89,5% 
13= Plate 2 B1 : 20,5%       &quot;            from 72,5%  
14= Plate 2 B2 : 15,7%       &quot;            from 73,6% 
15= Plate 2 B3 : 0,02%       &quot;            from 85,4% 
16= Plate 3 B1 : 19,5%       &quot;            from 73,3%  
17= Plate 3 B2 : 20,8%       &quot;            from 75,7% 
18= Plate 3 B3 : 0,07%       &quot;            from 84,8% </P><P>The calculated value written into the table describes the fluorescent amount of cells from the living cells. 
Example:  1 A2 : 27,7% YFP-positve cells of 73,3% living cells : 27,7/73,3 = 0,378
</P><P>
1. Plate I(A is up) completely 10 µg
</P><TABLE border="1"><TBODY><TR><TD> 300µl dropped 0,00</TD><TD align="right">300µl resuspended 0,378</TD><TD align="right">450µl resuspended 0,274
</TD></TR><TR><TD> 600µl dropped 0,173</TD><TD align="right">600µl resuspended 0,094</TD><TD align="right"> control 5,6x10^-4
</TD></TR></TBODY></TABLE><P>
2. Plate II(A is up)completely 18 µg
</P><TABLE border="1"><TBODY><TR><TD> 300µl dropped 0,437</TD><TD align="right">300µl resuspended 0,356</TD><TD align="right">450µl resuspended 0,418
</TD></TR><TR><TD> 600µl dropped 0,283</TD><TD align="right">600µl resuspended 2,1x10^-3</TD><TD align="right"> control 2,3x10^-4
</TD></TR></TBODY></TABLE><P>
3. Plate III(A is up) completely 24 µg
</P><TABLE border="1"><TBODY><TR><TD> 300µl dropped 0,353</TD><TD align="right">300µl resuspended 0,411</TD><TD align="right">450µl resuspended 0,408
</TD></TR><TR><TD> 600µl dropped 0,266</TD><TD align="right">600µl resuspended 0,275</TD><TD align="right"> control 8,3x10^-4
</TD></TR></TBODY></TABLE><P><B>Conclusions </B></P><UL><LI> Higher amounts of Plasmids (24, 30µg) seems to have no obvious effect on YFP expression
</LI><LI> Theres no difference in YFP expression comparing resuspended samples to not resuspended samples
</LI><LI> The amount of dead HT cells is about 25%, it is not clear if it is either AAV induced or mechanic stress (Trypsin, washing procedure)
</LI></UL><P>Next steps:
</P><UL><LI> does the amount of GOI (ng plasmids) has an effect on YFP expression  
</LI><LI> compare transduced to non transduced cells (vitality), or in other words is the mechanic stress responsible for 25% dead cell ratio?
</LI><LI> comparison of transduction efficiency of different virus stock production routines
</LI></UL><H4><SPAN class="mw-headline" id="Plasmid_Mini-Prep_of_pSB1C3_CFP"><P style="font-size:15px; background-color:#66bbff;">Plasmid Mini-Prep of pSB1C3_CFP</P></SPAN></H4><P><B>Investigators: Kerstin, Anissa</B></P><UL><LI>experiment date: 15.07.2010 ; time:10.00
</LI><LI>new vector name:pSB1C3_CFP
</LI></UL><P><U>Glycerol Stocks</U></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD><TD align="left"><B>Clone 3</B></TD><TD align="left"><B>Clone 4</B></TD></TR><TR><TD align="left"><B>Bacteria strain</B></TD><TD align="left"> XL1 blue </TD><TD align="left"> XL1 blue </TD><TD align="left"> XL1 blue </TD><TD align="left"> XL1 blue
</TD></TR><TR><TD align="left"><B>Plasmidname</B></TD><TD align="left"> pSB1C3_CFP </TD><TD align="left"> pSB1C3_CFP </TD><TD align="left">pSB1C3_CFP </TD><TD align="left"> pSB1C3_CFP
</TD></TR><TR><TD align="left"><B>Date</B></TD><TD align="left"> 15.7.2010 </TD><TD align="left"> 15.7.2010 </TD><TD align="left"> 15.7.2010 </TD><TD align="left"> 15.7.2010
</TD></TR><TR><TD align="left"><B>given glycerol-stock no.</B></TD><TD align="left"> B38 </TD><TD align="left"> B39 </TD><TD align="left"> B40 </TD><TD align="left"> B41
</TD></TR><TR><TD align="left"><B>given plasmid no.</B></TD><TD align="left"> p48 </TD><TD align="left"> p51 </TD><TD align="left"> p52 </TD><TD align="left"> p53
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color: blue;">Test digestion</P><UL><LI>buffer used: 3 ; Restriction-enzymes used: Enzyme 1 (no. Lab:48)PstI ; Enzyme 2 (no.Lab:23) EcoRI
</LI><LI>Plasmid
</LI><LI>Given Plasmid-Number: p48; DNA concentration: 195,1 ng/µl ;
</LI><LI>Given Plasmid-Number: p51; DNA concentration: 230,1 ng/µl;
</LI><LI>Given Plasmid-Number: p52; DNA concentration: 134,1 ng/µl;
</LI><LI>Given Plasmid-Number: p53; DNA concentration: 244,1 ng/µl;
</LI></UL><P><B>Comments:</B>p48 and p51 were sequenced
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>Volume/µL</B></TD><TD align="left"><B>Mastermix</B></TD></TR><TR><TD align="left"> DNA </TD><TD align="left"> Variable (800-1000 ng) </TD><TD align="left"> -
</TD></TR><TR><TD align="left"> BSA (100x) </TD><TD align="left"> 1,5 </TD><TD align="left"> 7,5
</TD></TR><TR><TD align="left"> Buffer no.3    (10x) </TD><TD align="left"> 1,5 </TD><TD align="left"> 7,5
</TD></TR><TR><TD align="left"> Enzyme 1 (no. Lab:48    ) </TD><TD align="left"> 0,5 </TD><TD align="left"> 2,5
</TD></TR><TR><TD align="left"> Enzyme 2 (no. Lab:23    ) </TD><TD align="left"> 0,5 </TD><TD align="left"> 2,5
</TD></TR><TR><TD align="left"> H2O </TD><TD align="left"> variable </TD><TD align="left"> -
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"> 15 </TD><TD align="left"> -
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD> Sample   </TD><TD align="right"> Volume sample/ µl </TD><TD align="right"> Volume H2O / µl
</TD></TR><TR><TD> p48 </TD><TD align="right"> 5 </TD><TD align="right"> 6
</TD></TR><TR><TD> p51  </TD><TD align="right"> 4 </TD><TD align="right"> 7
</TD></TR><TR><TD> p52  </TD><TD align="right"> 7 </TD><TD align="right"> 4
</TD></TR><TR><TD> p53  </TD><TD align="right"> 4 </TD><TD align="right"> 7
</TD></TR></TBODY></TABLE><UL><LI>Incubation: 1h
</LI></UL><P style="font-size:15px; font-weight: bold; color: blue;">Agarose-Gel:</P><P>
0,5 g Agarose, 50 TAE (1%), 3GELRED (3-6µl), at 110Volt, running time:30 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (6x)/µl
</TH><TH>Expected size  (Geneious)
</TH></TR><TR><TD>p48
</TD><TD>15 µl
</TD><TD>3 µl
</TD><TD>700 bp
</TD></TR><TR><TD>p51
</TD><TD>15 µl
</TD><TD>3 µl
</TD><TD>700 bp
</TD></TR><TR><TD>p52
</TD><TD>15 µl
</TD><TD>3 µl
</TD><TD>700 bp
</TD></TR><TR><TD>p53
</TD><TD>15 µl
</TD><TD>3 µl
</TD><TD>700 bp
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>Sample p48 /µl
</TH><TH>Sample p51 /µl
</TH><TH>Sample p52 /µl
</TH><TH>Sample p53 /µl
</TH></TR><TR><TH>Lane
</TH><TD>8 µl
</TD><TD>18µl
</TD><TD>18µl
</TD><TD>18µl
</TD><TD>18µl
</TD></TR></TBODY></TABLE><P><B>Comments:</B> Gel was leaky, therefore two samples &quot;flew through&quot;. Plasmid p48 and p51 were sent for sequencing.
</P><H4><SPAN class="mw-headline" id="2nd_try:_First_steps_in_producing_ITR_BioBricks_via_Fancy_Method"><P style="font-size:15px; background-color:#66bbff;">2nd try: First steps in producing ITR BioBricks via Fancy Method</P></SPAN></H4><P><B>Investigator: Hanna</B>
Unfortunately it turned out, that the digestion step of the 1st try didn't work: After the transformation very much colonies grew on the plates. Sequencing delivered, that survival of the bacteria was due to re-ligated vector... Therefore the whole experiment will be performed again today.
</P><P style="font-size:15px; font-weight: bold; color: fuchsia;">Digestion of pGA14_Cerulean-CFP</P><P><B>1. Digestion with EcoRI and NotI:</B></P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 3.4
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x) </TD><TD align="right"> 2
</TD></TR><TR><TD>Enzyme EcoRI-HF </TD><TD align="right"> 1
</TD></TR><TR><TD>Enzyme NotI-HF </TD><TD align="right"> 1
</TD></TR><TR><TD>H2O</TD><TD align="right"> 10.6
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><P>
Incubation: 1.5 hours, 37°C in thermocycler
</P><P>Gel run: 20 µL sample + 4 µL loading dye (6x); 8µL marker 
Expected fragment size: ~ 2900 bp
</P><P>Gel extraction:

Sample weight: 0.31 g 
Gel extraction was performed following the standard protocol. Abberation: After applying EB-buffer, the column was incubated for two minutes at 50°C.   
DNA concentration: 43 ng/µL 
<B>1. Digestion with XbaI and PstI:</B></P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 3.4
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x) </TD><TD align="right"> 2
</TD></TR><TR><TD>Enzyme XbaI </TD><TD align="right"> 1
</TD></TR><TR><TD>Enzyme PstI-HF </TD><TD align="right"> 1
</TD></TR><TR><TD>H2O</TD><TD align="right"> 10.6
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><P>
Incubation: 1.5 hours, 37°C in thermocycler
</P><P>Gel run: 20 µL sample + 4 µL loading dye (6x); 8µL marker 
Expected fragment size: ~ 2900 bp
</P><P>Gel extraction:

Sample weight: 0.31 g 
Gel extraction was performed following the standard protocol. Abberation: After applying EB-buffer, the column was incubated for two minutes at 50°C. 
DNA concentration: 38.6 ng/µL
</P><P style="font-size:15px; font-weight: bold; color: fuchsia;">Digestion of pAAV_MCS with AlwNI</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 17.86 = 5 µg
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right"> -
</TD></TR><TR><TD> Buffer 4 (10x) </TD><TD align="right"> 3
</TD></TR><TR><TD>Enzyme AlwNI (no.Lab:120)</TD><TD align="right"> 3
</TD></TR><TR><TD>H2O</TD><TD align="right"> 6.14
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 30
</TD></TR></TBODY></TABLE><P>
Incubation: 1.5 hours , 37°C in thermocycler
</P><P>Gel run: 30 µL sample + 6 µL loading dye (6x); 8µL marker 
Expected fragment sizes: 2982 bp = right ITR and 1674 bp = left ITR
</P><P>Gel extraction:
</P><UL><LI> Sample: right ITR = 0.19 g </LI><LI> Sample: left ITR = 0.17 g </LI></UL><P>Gel extraction was performed following the standard protocol. Abberation: After applying EB-buffer, the column was incubated for two minutes at 50°C. Further on 30 µL EB-buffer was used.
DNA concentration: 
</P><UL><LI> right ITR: <B>59.8 ng/µL</B></LI><LI> left ITR: <B>46.2 ng/µL</B><B> :)</B></LI></UL><P style="font-size:15px; font-weight: bold; color: fuchsia;">Digestion of ITR fragments with NotI and PstI</P><P>
Digestion of right ITR fragment:
</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 30 µL
</TD></TR><TR><TD> BSA (100x) </TD><TD align="right"> 0.5
</TD></TR><TR><TD> Buffer 4 (10x) </TD><TD align="right"> 3.74
</TD></TR><TR><TD>Enzyme NotI-HF </TD><TD align="right"> 1.5
</TD></TR><TR><TD>Enzyme PstI-HF </TD><TD align="right"> 1.5
</TD></TR><TR><TD>H2O</TD><TD align="right"> -
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 37.4
</TD></TR></TBODY></TABLE><P>
Incubation: 50 minutes, 37°C in thermocycler
</P><P>Digestion of left ITR fragment:
</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 30 µL
</TD></TR><TR><TD> BSA (100x) </TD><TD align="right"> 0.5
</TD></TR><TR><TD> Buffer 4 (10x) </TD><TD align="right"> 3.72
</TD></TR><TR><TD>Enzyme NotI-HF </TD><TD align="right"> 1.5
</TD></TR><TR><TD>Enzyme PstI-HF </TD><TD align="right"> 1.5
</TD></TR><TR><TD>H2O</TD><TD align="right"> -
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 37.2
</TD></TR></TBODY></TABLE><P>
Incubation: 50 minutes, 37°C in thermocycler

Gel run:
</P><UL><LI> due to the small fragment sizes, a 2% agarose gel was prepared.
</LI><LI> 5 µL loading dye was added to the whole volume of each sample; 8µL marker </LI><LI> 115 V, 40 minutes </LI><LI> Expected fragment sizes: ~ 150 bp (both)
</LI><LI> Problem: Small fragments, little amounts, blurred 150 bp bands -&gt; fragments were cut out broad-minded. 
</LI></UL><P>Gel extraction:
</P><UL><LI> Sample: right ITR oben = 0.4 g </LI><LI> Sample: left ITR = 0.36 g </LI></UL><P>Gel extraction was performed following standard protocol. After applying EB buffer to the columns, they were placed for 2 minutes into 50°C water bath and then centrifuged at 13200 rpm for 1 minute. 
DNA concentration: 
</P><UL><LI> right ITR: 4 ng/µL 
</LI><LI> left ITR: 4 ng/µL
</LI></UL><P style="font-size:15px; font-weight: bold; color: fuchsia;">Hybridization of ITR-Präfix-Oligos</P><P><B>left ITR:</B></P><UL><LI> 10 µL Oligo 1 (O43 = Präfix_for)
</LI><LI> 10 µL Oligo 2 (O44 = Präfix_rev)
</LI><LI> 4 µL 100 mM TrisHCl pH8
</LI><LI> 8 µL 5 mM MgCl2
</LI><LI> 8 µL H2O
</LI></UL><P>Total: 40 mL
</P><P><B>right ITR:</B></P><UL><LI> 10 µL Oligo 1 (O47 = Präfix_for)
</LI><LI> 10 µL Oligo 2 (O48 = Präfix_rev)
</LI><LI> 4 µL 100 mM TrisHCl pH8
</LI><LI> 8 µL 5 mM MgCl2
</LI><LI> 8 µL H2O
</LI></UL><P>Total: 40 mL
</P><P>Because some of the oligos produce strong secondary structures, we used the ORIGAMI1 program:
1) initial denaturation: 99°C, 7 minutes
2) 99°C, 1 minute
3) 73 x repetition of 2) -&gt; -1°C, R = 0.3 °C/sec
4) hold 4°C</P><P style="font-size:15px; font-weight: bold; color: fuchsia;">Ligation of pGA14 + ITR + Präfix</P><P><B>1. right ITR</B></P><UL><LI> Präfix: 2 µL
</LI><LI> ITR: 10 µL
</LI><LI> vector (pGA14): 5 µL
</LI><LI> buffer: 2 µL
</LI><LI> T4 ligase: 1 µL
</LI></UL><P>Total volume: 20 µL
</P><P><B>2. left ITR</B></P><UL><LI> Präfix: 2 µL
</LI><LI> ITR: 10 µL
</LI><LI> vector (pGA14): 5 µL
</LI><LI> buffer: 2 µL
</LI><LI> T4 ligase: 1 µL
</LI></UL><P>Total volume: 20 µL
</P><P><B>3. control 1:</B></P><UL><LI> vector 1 (pGA14, digested with EcoRI + NotI): 5 µL
</LI><LI> buffer: 2 µL
</LI><LI> T4 ligase: 1 µL
</LI><LI> H2O: 12 µL
</LI></UL><P>Total volume: 20 µL
</P><P><B>4. control 2:</B></P><UL><LI> vector 2 (pGA14, digested with XbaI + PstI): 5 µL
</LI><LI> buffer: 2 µL
</LI><LI> T4 ligase: 1 µL
</LI><LI> H2O: 12 µL
</LI></UL><P>Total volume: 20 µL
</P><P>Samples will be incubating over night at 16°C.
<B>To do:</B> Trafo (Agar plates were already prepared.)
</P><H3><SPAN class="mw-headline" id="60.Labortag_16.07.2010"><P style="font-size:17px; background-color:#00dd77;">60.Labortag 16.07.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Continuation_of_Fancy_ITR-method"><P style="font-size:15px; background-color:#66bbff;">Continuation of Fancy ITR-method</P></SPAN></H4><P><B>Investigator: Hanna</B></P><P style="font-size:15px; font-weight: bold; color: fuchsia;"><U>Trafo</U></P><P>Trafo was performed following standard protocol. Aberration: Instead of 2 µL, 3 µL DNA was added to BL21 bacteria (50 µL).
</P><P style="font-size:15px; font-weight: bold; color: fuchsia;"><U>right ITR</U></P><P>It turned out, that the strategy by digesting pGA14 with XbaI and PstI cannot work. pGA14 needs to be digested with <B>EcoRI</B> and PstI: </P><P style="font-size:15px; font-weight: bold; color: fuchsia;">Digestion of pGA14_Cerulean-CFP with EcoRI and PstI</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 3.4
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x) </TD><TD align="right"> 2
</TD></TR><TR><TD>Enzyme EcoRI-HF </TD><TD align="right"> 1
</TD></TR><TR><TD>Enzyme PstI-HF </TD><TD align="right"> 1
</TD></TR><TR><TD>H2O</TD><TD align="right"> 10.6
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><P>
Incubation: 1 hour, 37°C in thermocycler
</P><P>Gel run: 1% gel, 20 µL sample + 4 µL loading dye (6x); 8µL marker 
Expected fragment size: ~ 2900 bp
</P><P>Gel extraction:

Sample weight: 170 mg 
Gel extraction was performed following the standard protocol. Abberation: After applying EB-buffer, the column was incubated for two minutes at 50°C.   
DNA concentration: 26.8 ng/µL 
</P><P style="font-size:15px; font-weight: bold; color: fuchsia;">Digestion of pAAV_MCS with NotI and PstI</P><P>In order to create ITRs with NotI and PstI restriction sites, the AlwNI-digestion step was skipped. Therefore 50% of the constructs will also contain the left ITR... this means: picking more colonies and sequencing! </P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 10.71
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x) </TD><TD align="right"> 2
</TD></TR><TR><TD>Enzyme EcoRI-HF </TD><TD align="right"> 1
</TD></TR><TR><TD>Enzyme PstI-HF </TD><TD align="right"> 1
</TD></TR><TR><TD>H2O</TD><TD align="right"> 3.29
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><P>
Incubation: 1 hour, 37°C in thermocycler
</P><P>Gel run: 2% gel, 20 µL sample + 4 µL loading dye (6x); 8µL marker 
Expected fragment size: ~ 150 bp
<B>Gel run delivered no satisfying results. Therefore the gel was discarded and the strategy changed: The residual right ITR solution of yesterday's elution will be taken for ligation.</B></P><P style="font-size:15px; font-weight: bold; color: fuchsia;">Ligation of right ITR + Präfix + pGA14</P><UL><LI> ITR: 7 µL
</LI><LI> pGA14: 7 µL
</LI><LI> Präfix: 1.5 µL
</LI><LI> Buffer: 2 µL
</LI><LI> T4 ligase: 1 µL
</LI><LI> H2O: 1.5 µL
</LI></UL><P>Total: 20 µL
</P><UL><LI> pGA14: 7 µL
</LI><LI> Buffer: 2 µL
</LI><LI> T4 ligase: 1 µL
</LI><LI> H2O: 10 µL
</LI></UL><P>Total: 20 µL
</P><P>The ligation-solution will be incubating over night.
<B>To do:</B></P><UL><LI> Mini-Prep of pGA14_präfix_leftITR  
</LI><LI> Trafo of pGA14_präfix_rightITR
</LI><LI> Midi-Prep of pGA14
</LI></UL><H4><SPAN class="mw-headline" id="Results_sequencing_of_pSB1C3_CFP"><P style="font-size:15px; background-color:#66bbff;">Results sequencing of pSB1C3_CFP</P></SPAN></H4><P><B>Investigator: Bea</B>
Plasmid: pSB1C3_CFP 
Primer used: VR2 
Plasmid names: P48_KK and P51_AB<B>Comments</B>: Sequencing results of cloning experiment performed at 14.-15.07.2010. </P><P style="font-size:13px; font-weight: bold; color: green;"><B>Results</B>: Sequencing reads are good. Cloning of CFP into pSB1C3 can be confirmed. There are no insertions, transitions or deletions in the sequence or the iGEM restriction sites. Using this plasmid in order to create our iGEM BioBricks in RFC10 standard can be confirmed. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Sequencing results of <B>pSB1C3_CFP (plasmid number P48)</B>. There are no mutations in the CFP sequence and in the RFC10 iGEM restiction sites (blue boxes).</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Sequencing results of <B>pSB1C3_CFP (plasmid number P51</B>). There are no mutations in the CFP sequence and in the RFC10 iGEM restiction sites (blue boxes).</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="BioBrick_production_with_hGH_and_beta-globin_by_cloning_it_into_pSB1C3"><P style="font-size:15px; background-color:#66bbff;"> BioBrick production with hGH and beta-globin by cloning it into pSB1C3</P></SPAN></H4><P><B>Investigator: Bea</B>
Plasmid:  pSB1C3_CFP  
Plasmid names: pSB1C3 (P51 or P48) and PCR prdouct of hGH and beta-globin<B>Comments</B>: While cloning theoretically the hGH PCR product into the vector pSB1C3, it was recognized that the primers that were used for PCR for
</P><UL><LI>hGH RFC10 production and
</LI><LI>beta-globin intron RFC10 production
</LI></UL><P>do not contain the required &quot;G&quot; at the end of the prefix  and the &quot;T&quot; in the suffix add-on in the RFC10 standard. They were already used by performing a PCR with the add-on prefix and suffix primer, but we did not clone it into the pSB1C3 because of the wrong add-on tails. Therefore new primers were designed and ordered. 
</P><P style="font-size:15px; font-weight: bold; color: purple;">Ordered oligos:</P><P>hGH
</P><UL><LI>suffix (reverse)
</LI><LI>prefix (forward)
</LI></UL><P>beta-globin
</P><UL><LI>suffix (reverse)
</LI><LI>prefix (forward)
</LI></UL><P>CMV
</P><UL><LI>suffix (reverse)
</LI><LI>prefix (forward)
</LI></UL><P>TK_GMK
</P><UL><LI>tk_gmk_prefix for
</LI><LI>tk_gmk_suffix_rev
</LI><LI>gmk_suffix_rev
</LI></UL><P style="font-size:14px; font-weight: bold; color: Turquoise;">To do: Upload cloning strategy of tk_gmk and BioBrick production</P><P>
--<A href="/wiki/index.php?title=User:Beate&amp;action=edit&amp;redlink=1" class="new" title="User:Beate (page does not exist)">Bea</A> 15:21, 16 July 2010 (UTC) <B>Comment: reverse beta-globin primer is correct!? (Hanna)</B></P><H4><SPAN class="mw-headline" id="Transfection_2"><P style="font-size:15px; background-color:#66bbff;">Transfection</P></SPAN></H4><P><B>Investigator: Adrian</B>
30µg Plasmid were used (pHelper + pAAV_RC+ pAAV_RFC25_mgmk_TK30)
VIRUS nix:pHelper + pAAV_RC
VIRUS 1  :pHelper + pAAV_RC+ pAAV_RFC25_mgmk_TK30 clone 1 : P54
VIRUS 2  :pHelper + pAAV_RC+ pAAV_RFC25_mgmk_TK30 clone 2 : P55
VIRUS 3  :pHelper + pAAV_RC+ pAAV_RFC25_mgmk_TK30 clone 3 : P56</P><H4><SPAN class="mw-headline" id="Mini_Prep_of_pAAV_RFC25_mgmk_TK30"><P style="font-size:15px; background-color:#66bbff;">Mini Prep of pAAV_RFC25_mgmk_TK30</P></SPAN></H4><P><B>Investigator: Adrian</B>
4ml were preparated
</P><UL><LI>pAAV_RFC25_mgmk_TK30 clone1 conc: 291,43 ng/µl : P54
</LI><LI>pAAV_RFC25_mgmk_TK30 clone2 conc: 243,99 ng/µl : P55
</LI><LI>pAAV_RFC25_mgmk_TK30 clone3 conc: 247,00 ng/µl : P56
</LI><LI>pAAV_RFC25_mgmk_TK30 clone4 conc: 246,01 ng/µl : P57
</LI></UL><P>Test digestion of all four plasmids to check if the mgmg_TK30 insert is present. Investigator: Patrick, Chris

Each digestion sample:
</P><UL><LI>0,2 µl BSA (I picked the wrong tube. It should have been 2 µl BSA)
</LI><LI>2 µl Buffer 4
</LI><LI>0,5 µl XbaI
</LI><LI>0,5 µl SpeI
</LI><LI>13,8 µl H2O
</LI><LI>3 µl DNA
</LI></UL><P>
Digestion: 65 minutes, 37°C. Results: No mgmk_TK30 insert detectable. I will try it again on saturday or sunday.
</P><H4><SPAN class="mw-headline" id="Preparation_of_50x_TAE_and_0.2C5M_EDTA"><P style="font-size:15px; background-color:#66bbff;">Preparation of 50x TAE and 0,5M EDTA</P></SPAN></H4><P><B>Investigators: Chris W., Patrick</B></P><P>1 L TAE (50x)	
</P><P>242 g tris base
57.1 ml glacial acetic acid
100 ml 0.5 M EDTA
ad 1 L Milipor
</P><P>1 L EDTA (0.5 M)
</P><P>146,13 g  Ethylendiamintetraessigsäure
ad 800 ml Milipor
~20 g NaOH
pH mit NaOH auf 8 einstellen
ad 1 L Milipor
</P><H4><SPAN class="mw-headline" id="Test_digestion_of_the_QuickChange_for_PstI_.28310.29_and_PstI_.284073.29"><P style="font-size:15px; background-color:#66bbff;">Test digestion of the QuickChange for PstI (310) and PstI (4073)</P></SPAN></H4><P><B>Investigator: Kira &amp; Volker</B></P><P>The digestion showed two bands with apropirate lengt for each of the 6 clones as they were expected.
The constructs were sent to GATC for sequencing. </P><UL><LI>pAAV_RC_SDM_PstI(310) with the primer GATC_std_QE-FP
</LI><LI>pAAV_RC_SDM_PstI (4073) with the primer GATC_std_M13_FP
</LI></UL><H4><SPAN class="mw-headline" id="Primers_to_convert_the_expression_constructs_into_the_iGEM_standard_were_ordered"><P style="font-size:15px; background-color:#66bbff;">Primers to convert the expression constructs into the iGEM standard were ordered</P></SPAN></H4><P><B>Investigator: Volker</B></P><UL><LI>Praefix VP1ex
</LI><LI>Praefix VP2ex
</LI><LI>Praefix VP3ex
</LI><LI>Suffix VP123ex
</LI><LI>Praefix Rep68_78ex
</LI><LI>Praefix Rep 40_52ex
</LI><LI>Praefix AAPex
</LI><LI>Suffix Rep 40_68ex
</LI><LI>Suffix Rep 52_78ex
</LI><LI>Suffix AAPex
</LI></UL><H3><SPAN class="mw-headline" id="61.Labortag_17.07.2010:"><P style="font-size:17px; background-color:#00dd77;">61.Labortag 17.07.2010:</P></SPAN></H3><H4><SPAN class="mw-headline" id="Picking_clones_of_pGA14_Pr.C3.A4fix-leftITR_Trafo"><P style="font-size:15px; background-color:#66bbff;">Picking clones of pGA14_Präfix-leftITR Trafo</P></SPAN></H4><P><B>Investigator: Hanna</B>
Bacteria also grew on control plates. Control means: Ligation of digested vector (pGA14_Cerulean) without insert (Präfix and left ITR). Therefore it seemed that somehow pGA14 religated (Can't be if vector digested completely). Therefore 10 colonies were picked, in order to increase the chance to get a successfully ligated plasmid.
</P><H4><SPAN class="mw-headline" id="Mini-Preps_of_pGA14_Pr.C3.A4fix-lITR_cultures"><P style="font-size:15px; background-color:#66bbff;">Mini-Preps of pGA14_Präfix-lITR cultures</P></SPAN></H4><P><B>Investigator: Hanna</B></P><P>No glycerol stocks were prepared, because I'm going to perform the test digestion today. 
10 Mini-Preps were performed following the standard protocol.
</P><H4><SPAN class="mw-headline" id="Test_digestion_of_pGA14_Pr.C3.A4fix-lITR_.28Ligation.29"><P style="font-size:15px; background-color:#66bbff;">Test digestion of pGA14_Präfix-lITR (Ligation)</P></SPAN></H4><P><B>Investigator: Hanna</B></P><P style="font-size:14px; font-weight: bold; color: blue;">Test digestion</P><UL><LI>buffer used: 4; Restriction-enzymes used: Enzyme 1: EcoRI-HF ; Enzyme 2: PstI-HF
</LI><LI>Plasmid: pGA14_Präfix_lITR 1 - 10 (all: ~ 400 ng/µL)
</LI></UL><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>Volume/µL</B></TD><TD align="left"><B>Mastermix</B></TD></TR><TR><TD align="left"> DNA </TD><TD align="left"> 2.5 </TD><TD align="left"> -
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> - </TD><TD align="left"> 12
</TD></TR><TR><TD align="left"> Buffer 4 (10x) </TD><TD align="left"> - </TD><TD align="left"> 12
</TD></TR><TR><TD align="left"> Enzyme 1 EcoRI-HF </TD><TD align="left"> - </TD><TD align="left"> 6
</TD></TR><TR><TD align="left"> Enzyme 2 PstI-HF </TD><TD align="left"> - </TD><TD align="left"> 6
</TD></TR><TR><TD align="left"> H2O </TD><TD align="left"> - </TD><TD align="left"> 54
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"> 10 </TD><TD align="left"> 90
</TD></TR></TBODY></TABLE><UL><LI>Incubation: 55 minutes
</LI></UL><P style="font-size:15px; font-weight: bold; color: blue;">Agarose-Gel:</P><P>
0.75 g Agarose, 50 ml TAE (1.5 %), 3 µL GELRED, at 115 Volt, running time: 30 minutes

Expected fragment size: Hopefully 160 bp = left ITR :) If pGA14_Cerulean religated the expected fragment size would be ~ 800 bp.
</P><UL><LI>Marker: GeneRuler ladder mix (6x)
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>Sample 1
</TH><TH>Sample 2
</TH><TH>Sample 3
</TH><TH>Sample 4
</TH><TH>Sample 5
</TH><TH>Sample 6
</TH><TH>Sample 7
</TH><TH>Sample 8
</TH><TH>Sample 9
</TH><TH>Sample 10
</TH></TR><TR><TH>Lane
</TH><TD>8 µL
</TD><TD>10 µL
</TD><TD>10 µL
</TD><TD>10 µL
</TD><TD>10 µL
</TD><TD>10 µL
</TD><TD>10 µL
</TD><TD>10 µL
</TD><TD>10 µL
</TD><TD>10 µL
</TD><TD>10 µL
</TD></TR></TBODY></TABLE><P><B>Comments:</B> Numbers refer to &quot;colony 1, 2, 3,...&quot;
</P><P style="font-size:14px; font-weight: bold; color: fuchsia;">Wonderful news: All 10 test digestions delivered positive results: 150 bp fragments and no 800 bp fragment was detectable. Means: left ITR + Präfix was inserted successfully into pGA14!!! :)</P><P style="font-size:12px; color: blue;">yees, thats great! Finally something worked with our ITRs. well done Hanna!! (Bea)</P><P style="font-size:12px; color: black;">the most astonishingly test digestion gel of our entire working time! :)(Adrian)</P><P style="font-size:12px; font-weight: bold; color: fuchsia;">Next step: Fusion of suffix to left ITR</P><P>
Glycerol stocks of all 10 colonies were prepared: B46.1 - B46.10; Plasmid names: P62.1 - P62.10
</P><H4><SPAN class="mw-headline" id="Trafo_of_pGA14_Pr.C3.A4fix_rITR"><P style="font-size:18px; font-weight: bold; color: fuchsia;"><U>Trafo of pGA14_Präfix_rITR</U></P></SPAN></H4><P><B>Investigator: Hanna</B>
Transformation of BL21 bacteria with pGA14_Präfix_rITR which ligated over night was performed following standard protocol. Abberation: 3µL DNA was added to 50 µL cells.
<B>To do:</B> Mini-Prep + Test digestion of pGA14_Präfix_rITR; Sequencing of pGA14_Präfix_lITR and pGA14_Präfix_rITR.
</P><H3><SPAN class="mw-headline" id="62.Labortag_18.07.2010"><P style="font-size:15px; background-color:#66bbff;">62.Labortag 18.07.2010</P></SPAN></H3><P><B>Investigator: Adrian</B><B>Calculation of ganciclovir cymeven concentration</B></P><UL><LI>Ganciclovir: m = 255,23 g/mol
</LI><LI>dissolved in PBS (?), NaCL (0,9%) ?
</LI><LI>Concentrations : 0,5 µM = 0,0000005M
</LI><LI> xg = (0,0000005 mol/l *255,23 g/mol)/(1l PBS resp. 1l 0,9% NaCl)
</LI><LI> x = 0,000128g
</LI><LI> x = 0,128 mg
</LI><LI> 0,128 mg solved in 1l of PBS = 1l of 0,5µM solution
</LI><LI> steril filtrated under steril bench
</LI></UL><P>
preparation of Aliquots:
</P><H4><SPAN class="mw-headline" id="Cellculture_2"><P style="font-size:15px; background-color:#66bbff;">Cellculture</P></SPAN></H4><P>The HEK-293 cells were split according to the standard protocol. The HT 1080 cells were split and sowed (3x10^5 cells each well).
The amount of cells/ml was determined with the Neubauer-Meteringchamber: (8 cells counted/4)x2,2x20x10^4 = 888000 cells/ml. Two 6-well dishes could be prepared.
</P><TABLE border="1"><TBODY><TR><TD> only DMEM </TD><TD align="right">3x10^5 cells+DMEM</TD><TD align="right">3x10^5 cells+DMEM
</TD></TR><TR><TD> 3x10^5 cells+DMEM</TD><TD align="right">3x10^5 cells+DMEM</TD><TD align="right"> 3x10^5 cells+DMEM
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Test_digestion_of_pAAV_RFC_mgmk_tk30:_P54.2C_P55.2C_P56.2C_P57"><P style="font-size:15px; background-color:#66bbff;">Test digestion of pAAV_RFC_mgmk_tk30: P54, P55, P56, P57</P></SPAN></H4><P><B>Investigators: Chris, Patrick</B></P><P>To check whether the mutual mgmk_tk30 insert is present in the plasmid we digested it with AgeI and EcoRI. If the insert is present there will an DNA fragmet with a size of about 1700 bp.
</P><TABLE border="1"><TBODY><TR><TD> P54 </TD><TD align="right"> P55 </TD><TD align="right"> P56 </TD><TD align="right"> P57
</TD></TR><TR><TD> 291,4 ng/µl </TD><TD align="right">244 ng/µl </TD><TD align="right">247 ng/µl </TD><TD align="right"> 246 ng/µl
</TD></TR></TBODY></TABLE><P>
Mastermix:
</P><UL><LI>10 µl Buffer 4
</LI><LI>4,4 µl AgeI
</LI><LI>2,5 µl EcoRI
</LI><LI>no BSA (accroding to NEB)
</LI><LI>83,1 µl H2O
</LI><LI>Total volume: 100 µl
</LI></UL><P>17 µl of the mastermix were pipetted to 3 µl plasmid sample. Digestion time: 75 minutes.
</P><P>Gelrun (1% Agarose), 55 minutes, 115 mV: Results: See picture. Interpretation of the results: ? The &quot;upper&quot; DNA seems to be undigested vector underneath the probably digested vector with an estimated size of about 5600 bp. The mutual insert (mgmk_tk30) should be about 1700 bp. There are probably several DNA coil structures visible but not our mutual insert. To do : ligate again.
</P><P>Aferwards i noticed that i miscalculated the Mastermix.
</P><H3><SPAN class="mw-headline" id="63._Labortag_19.07.2010"><P style="font-size:17px; background-color:#00dd77;">63. Labortag 19.07.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="rITR"><P style="font-size:15px; background-color:#66bbff;">rITR</P></SPAN></H4><P style="font-size:14px; font-weight: bold; color: fuchsia;">Picking clones of pGA14_Präfix-rITR Trafo</P><P>
Few bacteria also grew on control plates. Control means: Ligation of digested vector (pGA14_Cerulean) without insert (Präfix and right ITR).
6 colonies were picked, in order to increase the chance to get a successfully ligated plasmid.
</P><H4><SPAN class="mw-headline" id="Mini-Preps_of_pGA14_Pr.C3.A4fix-lITR_cultures_2"><P style="font-size:15px; background-color:#66bbff;">Mini-Preps of pGA14_Präfix-lITR cultures</P></SPAN></H4><P><B>Investigators: Hanna, Jessy</B></P><P>Glycerol stocks were prepared: B47.1, B47.2, B47.3, B47.4, B47.6 
6 Mini-Preps were performed following the standard protocol.
</P><H4><SPAN class="mw-headline" id="Test_digestion_of_pGA14_Pr.C3.A4fix-rITR_.28Ligation.29"><P style="font-size:15px; background-color:#66bbff;">Test digestion of pGA14_Präfix-rITR (Ligation)</P></SPAN></H4><P><B>Investigator: Hanna, Jessy</B></P><P style="font-size:15px; font-weight: bold; color: blue;">Test digestion</P><UL><LI>buffer used: 4; Restriction-enzymes used: Enzyme 1: EcoRI-HF ; Enzyme 2: PstI-HF
</LI><LI>Plasmid: pGA14_Präfix_rITR 1 - 6 (all: ~ 500 ng/µL) -&gt; 1 µg
</LI></UL><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>Volume/µL</B></TD><TD align="left"><B>Mastermix</B></TD></TR><TR><TD align="left"> DNA </TD><TD align="left"> 2 </TD><TD align="left"> -
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> - </TD><TD align="left"> 6.4
</TD></TR><TR><TD align="left"> Buffer 4 (10x) </TD><TD align="left"> - </TD><TD align="left"> 6.4
</TD></TR><TR><TD align="left"> Enzyme 1 EcoRI-HF </TD><TD align="left"> - </TD><TD align="left"> 4
</TD></TR><TR><TD align="left"> Enzyme 2 PstI-HF </TD><TD align="left"> - </TD><TD align="left"> 4
</TD></TR><TR><TD align="left"> H2O </TD><TD align="left"> - </TD><TD align="left"> 43.2
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"> 10 </TD><TD align="left"> 64
</TD></TR></TBODY></TABLE><UL><LI>Incubation: 45 minutes
</LI></UL><P style="font-size:15px; font-weight: bold; color: blue;">Agarose-Gel:</P><P>
0.75 g Agarose, 50 ml TAE (1.5 %), 3 µL GELRED, at 90 - 100 Volt, running time: 45 minutes

Expected fragment size: Hopefully 160 bp = right ITR :) If pGA14_Cerulean religated the expected fragment size would be ~ 800 bp.
</P><UL><LI>Marker: GeneRuler ladder mix (6x)
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>Sample 1
</TH><TH>Sample 2
</TH><TH>Sample 3
</TH><TH>Sample 4
</TH><TH>Sample 5
</TH><TH>Sample 6
</TH><TH>Marker
</TH></TR><TR><TH>Lane
</TH><TD>8 µL
</TD><TD>10 µL
</TD><TD>10 µL
</TD><TD>10 µL
</TD><TD>10 µL
</TD><TD>10 µL
</TD><TD>10 µL
</TD><TD>10 µL
</TD></TR></TBODY></TABLE><P><B>Comments:</B> Numbers refer to &quot;colony 1, 2, 3,...&quot;
</P><P style="font-size:14px; font-weight: bold; color: fuchsia;">Wonderful news: All 6 test digestions delivered positive results: ~ 160 bp fragments and no 800 bp fragment were detectable. Means: right ITR + Präfix was inserted successfully into pGA14!!! :)</P><P style="font-size:12px; font-weight: bold; color: fuchsia;">Next step: Ordering new oligos: Präfix of lITR, suffix of right and left ITRs, continuation of fancy cloning protocol</P><P>New Oligos (left ITR Präfix and Suffix + right ITR Suffix) were ordered. 
</P><H4><SPAN class="mw-headline" id="Sequencing_of_ITRs"><P style="font-size:15px; background-color:#66bbff;">Sequencing of ITRs</P></SPAN></H4><P><B>Investigator: Hanna</B>
pGA14_Präfix_lITR and pGA14_Präfix_rITR were sent for sequencing to GATC:
1) pGA14_Präfix_lITR: c(P62.1) = 422.38 ng/µL -&gt; V = 5 µL + 25 µL H2O 
2) pGA14_Präfix_rITR: c(P63.1) = 501 ng/µL -&gt; V = 4.2 µL + 25.8 µL H2O 
Primer: GATC_std_pQE-FP
</P><H4><SPAN class="mw-headline" id="Quantitative_real_time_PCR"><P style="font-size:15px; background-color:#66bbff;">Quantitative real time PCR</P></SPAN></H4><P><B>Investigator: Hanna</B>
Template: DNA of HT1080 cells transducted with 1) 10 µL and 2) with 30 µL virus stock solution.<B>Pipetting scheme:</B>
Master Mix</P><UL><LI> 2x QuantiFast SYBR Green PCR Mix: 400 µL
</LI><LI> Primer for and rev: 32 µL of 1:10 dilution = 10 µM
</LI><LI> Water: 208 µL
</LI><LI> Total: 640 µL
</LI></UL><P>
Template: 5 µL 
To each template 20 µL master mix were added.
PCR program:
</P><UL><LI> Hold: 95°C, 5 minutes
</LI><LI> Cycling: 95°C for 10 seconds, 60°C for 30 seconds
</LI><LI> Melt: Ramp from 60°C to 95°C, rising by 1°C each step, wait for 45 seconds on first step, then wait for 5 seconds for each step afterwards. Aquire to Melt A on FAM.
</LI></UL><H4><SPAN class="mw-headline" id="Cloning_strategy_for_gmk_tk_added.21.21"><P style="font-size:15px; background-color:#66bbff;">Cloning strategy for gmk_tk added!!</P></SPAN></H4><P><B>Investigator: Bea</B><A href="/wiki/images/a/aa/Freiburg10_BioBrick_GMK_TK30_07_07_2010_with_new_oligos.pdf" class="internal" title="Freiburg10 BioBrick GMK TK30 07 07 2010 with new oligos.pdf">Media:Freiburg10 BioBrick GMK TK30 07 07 2010 with new oligos.pdf</A></P><H4><SPAN class="mw-headline" id="pAAV_RC_site_directed_mutagenesis:_Digestion_of_the_two_mutagenesis_reaction_of_PstI_site:_310_and_4073"><P style="font-size:15px; background-color:#66bbff;">pAAV_RC site directed mutagenesis: Digestion of the two mutagenesis reaction of PstI site: 310 and 4073</P></SPAN></H4><P><B>Investigator: Volker</B></P><P><B>Digestion of pAAV_RC </B></P><LI> experiment date: 19.07.2010
</LI><LI> plasmid: name: pAAV_RC number: P58 / P59 
</LI><LI> buffer used: 4 ; Restriction-enzymes used: Enzyme  (no. Lab:66) XmnI 
</LI><LI> DNA concentration (vector):  ng/µL
<TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">P58 /µl </TD><TD align="right">P59 /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 4,34</TD><TD align="right">3,76
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right"> 2</TD><TD align="right"> 2
</TD></TR><TR><TD>Enzyme XmnI (no.Lab: 66)</TD><TD align="right">1 </TD><TD align="right"> 1
</TD></TR><TR><TD>H2O</TD><TD align="right"> 10,66</TD><TD align="right"> 11,24
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"><B>20</B></TD><TD align="right"><B>20</B></TD></TR></TBODY></TABLE><P>
Incubate at 37°C 
</P></LI><LI>Gelextraction was prepared according to the protocol</LI><LI>weight of gel: 310: 150 mg; 4073: 150mg </LI><LI>Nanodrop: 310: 5,5ng/µl; 4073: 7,0ng/µl</LI><LI>Ligation was prepared according to the protocol </LI><LI>Ratio: 310:4073 --&gt; 2,52 µl:6,48µl</LI><P>Ligation was frozen in -20°C</P><H4><SPAN class="mw-headline" id="pHelper_plasmid_for_Midi-Prep_prepared"><P style="font-size:15px; background-color:#66bbff;">pHelper plasmid for Midi-Prep prepared</P></SPAN></H4><P><B>Investigator:  Stefan</B>
200ml DYT media + 200µl Amp were inoculated with pHelper-plasmid glycerol stock and put on shaker in 37°C room.
For Midi-Prep:  Use Endotoxin-free Midi Prep not our Midi Prep!
</P><H4><SPAN class="mw-headline" id="Trafo_of_IGEM_2009_linkers"><P style="font-size:15px; background-color:#66bbff;">Trafo of IGEM 2009 linkers</P></SPAN></H4><UL><LI>We borrowed several linkers/tags from Gerrit and did a Trafo in XL1B cells:
</LI><LI>small linker 
</LI><LI>middle linker
</LI><LI>long linker
</LI><LI>Strep-Tag
</LI><LI>Split
</LI><LI>GSAT
</LI><LI>SEG
</LI></UL><UL><LI>Amount of DNA used: 0,5 µl
</LI><LI>Plates incubated at 37°C overnight
</LI></UL><H3><SPAN class="mw-headline" id="64._Labortag_20.07.2010"><P style="font-size:17px; background-color:#00dd77;">64. Labortag 20.07.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Quickchange_site_directed_mutagenesis:"><P style="font-size:15px; background-color:#66bbff;">Quickchange site directed mutagenesis:</P></SPAN></H4><P><B>Investigators: Achim,Anissa</B> 1) VP1ex P27.2 2) VP2ex P28.2  3) VP3ex P29.2 
PCR reaction:
</P><UL><LI> 2.5 µL 10x Pfu Ultra II buffer
</LI><LI> 1) 1,44µl 2) 1,83µl 3) 1,76µl  template (1:100 dilution) 
</LI><LI> 0.34 µL primer SDM PstI (4073)forward (of 1:10 dilution)
</LI><LI> 0.35 µL primer SDM PstI (4073) reverse (of 1:10 dilution)
</LI><LI> 0,5 µL DMSO (primers form very strong secondary structures)
</LI><LI> 0.5 µL dNTP
</LI><LI> 1) 18,9 2) 18,5 3) 18,6 µL dH2O
</LI><LI> 0.5 µL PfuUltra II fusion (1.25 U)
</LI></UL><P>-&gt; end volume: 25 µL
<B>PCR program:</B> Quickge

Trafo was carried out, using 2µl DNA, strain:XL1 blue
</P><P style="font-size:12px; font-weight: bold; color: blue;">Transformation with ligation from july 19th </P><UL><LI>used bacterial strain: BL21
</LI><LI>2µL of DNA (pAAV_RC w/o PstI 310/4073)
</LI><LI>Trafo plate in 37°C room for incubating at 12.30
</LI><LI> no clones to pick at 22.00
</LI></UL><P><B>Picking Clones from Linker Trafo</B></P><UL><LI> clones were picked at 22.00
</LI></UL><H4><SPAN class="mw-headline" id="Endotoxin-free_Midi_Prep_of_pHelper"><P style="font-size:15px; background-color:#66bbff;">Endotoxin-free Midi Prep of pHelper</P></SPAN></H4><P><B>Investigators: Chris, Stefan</B>
Midi-Prep was done according to protocol.
Result of the Midi-prep: 503,2 ng/µl
Plasmid (P64) was placed in Stratagene-Box.
</P><H4><SPAN class="mw-headline" id="Quantitative_real_time_PCR_2"><P style="font-size:15px; background-color:#66bbff;">Quantitative real time PCR</P></SPAN></H4><P><B>Investigator: Hanna</B>
Because there was one positive non-template control (H2O) yesterday, we performed the qPCR one more time.
For this purpose we also calculated the plasmid pipetting for the standard curve again:
</P><UL><LI> c(P9) = 272 ng/µL --&gt; 3 x 1:10 = 1:1000 dilution.
</LI><LI> 2.68 µL of the 1:1000 dilution was added to 52.32 µL H2O = 2600000 copies/µL.
</LI><LI> By performing 1:10 dilutions further 6 &quot;standard samples&quot; were created: 260000 copies/µL; 26000 copies/µL; 2600 copies/µL; 260 copies/µL, 26 copies/µL, 2.6 copies/µL
</LI></UL><P>
Template: DNA of HT1080 cells transducted with 1) 10 µL and 2) with 30 µL virus stock solution.<B>Pipetting scheme:</B>
Master Mix</P><UL><LI> 2x QuantiFast SYBR Green PCR Mix: 400 µL
</LI><LI> Primer for and rev: 32 µL of 1:10 dilution = 10 µM
</LI><LI> Water: 208 µL
</LI><LI> Total: 640 µL
</LI></UL><P>
Template: 5 µL 
To each template 20 µL master mix were added.
Comment: To non-template control 2 17 µL + 3µL H2O was added (not enough master mix was left).

PCR program:
</P><UL><LI> Hold: 95°C, 5 minutes
</LI><LI> Cycling: 95°C for 10 seconds, 60°C for 30 seconds
</LI><LI> Melt: Ramp from 60°C to 95°C, rising by 1°C each step, wait for 45 seconds on first step, then wait for 5 seconds for each step afterwards. Aquire to Melt A on FAM.
</LI></UL><H4><SPAN class="mw-headline" id="Cloning_of_p51.28pSB1C3_RFC25.29_for_His-tag_insert"><P style="font-size:15px; background-color:#66bbff;">Cloning of p51(pSB1C3_RFC25) for His-tag insert</P></SPAN></H4><P><B>Investigators: Anna, Stefan</B></P><LI>buffer used:4 ; Restriction-enzymes used: Enzyme SpeI (no. Lab:71), NgOMIV (no. Lab:113)
</LI><LI>DNA concentration (vector): 230 ng/µl
<TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right"> V (p51)/ µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 5
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2
</TD></TR><TR><TD>Enzyme: SpeI (no.Lab:71)</TD><TD align="right">1
</TD></TR><TR><TD>Enzyme: NgOMIV (no.Lab: 113)</TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">9
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"><B>20</B></TD></TR></TBODY></TABLE><P>1% Agarose-gel, 3µl Gelred</P><P>Loading dye(5x): 4µl</P><P>Loading plan: marker (10µl), free, sample (24µl)</P><P>expected size of fragment: 2080 bp
</P><H3><SPAN class="mw-headline" id="65._Labortag_21.07.2010"><P style="font-size:17px; background-color:#00dd77;">65. Labortag 21.07.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Sequencing_of_ITRs_2"><P style="font-size:15px; background-color:#66bbff;">Sequencing of ITRs</P></SPAN></H4><P><B>Investigator: Hanna</B>
P62.1 (=left ITR + RFC23-Präfix in pGA14) and P63.1 (= right ITR-Präfix in pGA14) were sequenced:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Sequencing results of <B>pGA14_lITR-RFC23Präfix</B>.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Sequencing results of <B>pGA14_rITR-RFC10Präfix</B>).</DIV></DIV></DIV></DIV><P>
Sequencing delivered, that the präfixes were fused successfully to the ITRs. The mismatch (orange box) was due to the alignment with the RFC10 präfix... (oligos were ordered!), what means, that the sequencing is OK.
Interestingly it seems, that not only we had problems with PCR of the ITR -&gt; Very strong secondary structures. Even sequencing of the ITRs seems impossible under normal conditions: Exactly at the point where the ITR sequence starts the sequencing signal delivered no clear results. </P><P style="clear:both;">Conclusion: Fancy method worked until now.</P><H4><SPAN class="mw-headline" id="Quantitative_real_time_PCR_3"><P style="font-size:15px; background-color:#66bbff;">Quantitative real time PCR</P></SPAN></H4><P><B>Investigator: Hanna</B>
Because the non-template control and the not-transduced cell-DNA controls delivered positive PCR results for the CMV-promoter, a control qPCR was prepared: 
1. not-transduced cell-DNA controls (K1 and K2) were diluted: 1:100
2. Oligos (O20, O21) were 1:10 diluted
3. Master Mix was prepared:</P></LI><LI> 75 µL SYBR Green
</LI><LI> 39 µL H2O
</LI><LI> 6 µL Oligos
</LI><P>4. Pipetting scheme:
</P><UL><LI> 1. K1
</LI><LI> 2. K2
</LI><LI> 3. NTC1
</LI><LI> 4. NTC2
</LI><LI> 5. Sdt7
</LI></UL><H4><SPAN class="mw-headline" id="Mini-Preps_of_linker"><P style="font-size:15px; background-color:#66bbff;">Mini-Preps of linker</P></SPAN></H4><UL><LI>experiment date: 21.07.2010 ; time: 10:30Uhr
</LI><LI>name of investigator: Anissa, Kerstin
</LI></UL><P><U>Glycerol Stocks</U></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD><TD align="left"><B>Clone 3</B></TD><TD align="left"><B>Clone 4</B></TD><TD align="left"><B>Clone 5</B></TD><TD align="left"><B>Clone 6</B></TD><TD align="left"><B>Clone 7</B></TD></TR><TR><TD align="left"><B>Bacteria strain</B></TD><TD align="left"> XL1 B </TD><TD align="left"> XL1 B </TD><TD align="left"> XL1 B </TD><TD align="left"> XL1 B </TD><TD align="left"> XL1 B</TD><TD align="left"> XL1 B</TD><TD align="left"> XL1 B
</TD></TR><TR><TD align="left"><B>Plasmidname</B></TD><TD align="left"> pGA14_middle linker </TD><TD align="left"> pGA14_GSAT linker </TD><TD align="left"> pGA14_long linker </TD><TD align="left"> pGA14_split </TD><TD align="left"> pGA14_strep </TD><TD align="left"> pGA14_short linker </TD><TD align="left"> pGA14_SEG
</TD></TR><TR><TD align="left"><B>given glycerol-stock no.</B></TD><TD align="left"> B48 </TD><TD align="left"> B49 </TD><TD align="left"> B50 </TD><TD align="left"> B51 </TD><TD align="left"> B52</TD><TD align="left"> B53</TD><TD align="left"> B54
</TD></TR><TR><TD align="left"><B>given plasmid no.</B></TD><TD align="left"> p65 </TD><TD align="left"> p66 </TD><TD align="left"> p67 </TD><TD align="left"> p68 </TD><TD align="left"> p69 </TD><TD align="left"> p70 </TD><TD align="left"> p71
</TD></TR></TBODY></TABLE><P><B>Comments:</B> Different Linker-fragments are very short (5-178bp) --&gt; no Test-digestion; directly sequencing
</P><H4><SPAN class="mw-headline" id="Results_of_Trafo_of_cloning_pAAV_RC_SDM_of_PstI_310_and_4073"><P style="font-size:15px; background-color:#66bbff;"><B>Results of Trafo of cloning pAAV_RC SDM of PstI 310 and 4073</B></P></SPAN></H4><P><B>Investigator: Bea</B><I>Comments:</I> Cloning together of the two site-directed mutagenesis performed with the pAAV_RC
</P><P style="font-size:12px; font-weight: bold; color: blue;"><B>Results:</B> no clones grown on LB-agar plate, because pAAV_RC was digested with only one blunt-end restriction enzyme. Luckily, th enzyme vut twice, and one time in the Amp gene. Therefore wrong ligation of the two fragments lacked the Amp resistance. </P><P style="font-size:14px; font-weight: bold; color: red;">To do: new cloning together of pAAV_RC PstI 310 and 4073 with TWO other enzymes.</P><H4><SPAN class="mw-headline" id="PCR_with_pUB_V5_mGMK_TK30"><P style="font-size:15px; background-color:#66bbff;"><B>PCR with pUB_V5_mGMK_TK30</B></P></SPAN></H4><P><B>Investigator: Adrian, Bea</B><I>Comments:</I> Perform PCR with two primers
</P><UL><LI>pmGMK_TK30_prefix_RFC25_for
</LI><LI>pmGMK_TK30_suffix_RFC25_rev
</LI></UL><P>and pUB_V5_hisB_mGMK_tk30 (P15) and clone it into pAAV_RFC25.</P><P style="font-size:17px; font-weight: bold; color: blue;">PCR:</P><UL><LI>PCR program: GMKTK
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume in µl</B></TD></TR><TR><TD> 5x Phusion HF buffer  </TD><TD align="right">10
</TD></TR><TR><TD> 10 mM dNTP mix </TD><TD align="right">1
</TD></TR><TR><TD>pMGMK_TK30_RFC25_prefix_for (060) (1:10 dilution)</TD><TD align="right">2,5 (0,5 µM)
</TD></TR><TR><TD>pMGMK_TK30_RFC25_suffix_rev (063) (1:10 dilution)</TD><TD align="right">2,5 (0,5 µM)
</TD></TR><TR><TD>DNA template: pUB_V5_hisB_mgmk_tk30</TD><TD align="right"> ~2 µL of 1:1000 dilution
</TD></TR><TR><TD>DMSO</TD><TD align="right"> -
</TD></TR><TR><TD>Phusion polymerase</TD><TD align="right">0,5
</TD></TR><TR><TD>H2O</TD><TD align="right">31,5µL
</TD></TR><TR><TD><B>Total volume (e.g. 50 µl)</B></TD><TD align="right">50
</TD></TR></TBODY></TABLE><P><B>Comments:</B>New Oligos (pmGMK_TK30_prefix_RFC25_for and pmGMK_TK30_suffix_RFC25_rev) with add-on tails (prefix and suffix), but TK30 still contains three iGEM restriction sites but will be ready for use in cell culture <B>Result of PCR product</B> (theoretically)</P><P style="font-size:17px; font-weight: bold; color: blue;">Digestion of pAAV_RFC25</P><UL><LI>Vector: name:pAAV_RFC25 number:p42 
</LI><LI>new vector name: pAAV_RFC25_MGMK_TK30
</LI><LI>buffer used:4 ; Restriction-enzymes used: Enzyme 1 (no. Lab:149) AgeI ; Enzyme 2 (no.Lab:63) XbaI
</LI><LI>DNA concentration (vector):321,5 ng/µl
</LI></UL><P><B>Comments:</B> Digestion of pAAV_RFC25 wihle performing PCR with P42
</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl </TD><TD align="right">volume of insert (=PCR product) /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right">4,73 </TD><TD align="right"> -
</TD></TR><TR><TD> BSA (100x) </TD><TD align="right">2 </TD><TD align="right"> -
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right"> -
</TD></TR><TR><TD>AgeI (no.Lab:149)</TD><TD align="right">1 </TD><TD align="right"> -
</TD></TR><TR><TD>XbaI(no.Lab:63)</TD><TD align="right">1 </TD><TD align="right"> -
</TD></TR><TR><TD>H2O</TD><TD align="right">9,27 </TD><TD align="right"> -
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right">20 </TD><TD align="right"> -
</TD></TR></TBODY></TABLE><UL><LI>Incubation: Vector: 1.5 h
</LI></UL><P style="font-size:15px; font-weight: bold; color: blue;">Agarose-Gel:</P><UL><LI>0,5 g Agarose
</LI><LI>50 ml TAE (1%)
</LI><LI>3 µl GELRED (3-6µl)
</LI><LI>110 Volt, running time: 50 minutes
</LI></UL><P><B>Results of digestion with XbaI and AgeI of pAAV_RFC25 and PCR with pUB_V5_gmk_tk30 </B></P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Digestion of pAAV_RFC25 witth XbaI and AgeI and PCR of pUB_V5_gmk_tk30 with prefix and suffix primer in RFC25 standard loaded on 1% agarose gel.</DIV></DIV></DIV><P><B>Result</B>: Gel was loaded with 50µL of pCR product and 20µL of pAAV_RFC25 digestion. Expected bands can be seen, but pAAV_RFC25 too high and there are some additional bands. We cut out only the intensive band at around 1700 bp.</P><P style="font-size:17px; color: blue;"><B>Comment:</B>After a PCR there can be two ways to proceed with the PCR product:  </P><OL><LI> load 7-8µL of PCR sample on an analytical gel and watch agarose gel for additional bands. If there are some more bands than expected load sample on preparative gel.If there aren´t any additional bands. proceed with digestion of PCR product in order to ligate it into vector.
</LI><LI>load total PCR sample on preparative gel. Ask Sven or Tobi in which volume you should elute after cutting out the gel (Gel extraction), but you should elute sample at least in 30µL.
</LI></OL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><A href="/File:Freiburg10_General_plan_Overview_PCR.png" title="File:Freiburg10 General plan Overview PCR.png">File:Freiburg10 General plan Overview PCR.png</A></DIV></DIV></DIV><UL><LI>After gel extraction was performed, concentration of the two constructs was measured and revealed:
</LI><LI>c(pAAV_RFC25)= 27,33ng/µL
</LI><LI>c(gmk_tk30)= 24,53ng/µL
</LI></UL><LI>After the gel extraction, the PCR product gmk_tk30 was digested again with XbaI and AgeI 
</LI><LI>and purified with the PCR purification Kit provided by Qiagen
</LI><P><B>Digestion of the PCR product gmk_tk30</B></P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl </TD><TD align="right">volume of insert (=PCR product) /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right">- </TD><TD align="right"> 30
</TD></TR><TR><TD> BSA (100x) </TD><TD align="right">-</TD><TD align="right"> 0,5
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">- </TD><TD align="right"> 4
</TD></TR><TR><TD>AgeI (no.Lab:149)</TD><TD align="right">- </TD><TD align="right"> 1
</TD></TR><TR><TD>XbaI(no.Lab:63)</TD><TD align="right">- </TD><TD align="right"> 1
</TD></TR><TR><TD>H2O</TD><TD align="right">- </TD><TD align="right"> 3,5
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right">- </TD><TD align="right"> 40
</TD></TR></TBODY></TABLE><UL><LI>Incubation:  1:45 h
</LI><LI>Ligation and Trafo were performed following standard protocols.
</LI></UL><H4><SPAN class="mw-headline" id="Gelextraction.2C_Ligation_and_Transformation_for_cloning_of_pSB1C3_His_RFC25_.28continuation_of_cloning_from_64th_lab_day.29"><P style="font-size:15px; background-color:#66bbff;"><B>Gelextraction, Ligation and Transformation for cloning of pSB1C3_His_RFC25 (continuation of cloning from 64th lab day)</B></P></SPAN></H4><P><B>Investigator: Bea, Kerstin, Stefan</B></P><P><B>Extraction</B></P><UL><LI>Gel extraction performed following standard protocol provided by Qiagen</LI></UL><P><B>Ligation</B></P><UL><LI>Measure DNA-concentration with Nanodrop </LI><LI>c(pSB1C3) = 18,2,8 ng/µL</LI><LI>c(His-tag) = 11,2 ng/µL</LI><LI>Calculation of volume needed for ligation: 
</LI><LI>c(pSB1C3) = 8,26 µL</LI><LI>c(His-tag) = 0,74 µL</LI></UL><P><B>Transformation</B></P><UL><LI>Transformation has been followed the standard protocol <A href="/wiki/images/4/4e/Freiburg10_Cloning_Protocol.pdf" class="internal" title="Freiburg10 Cloning Protocol.pdf">Media:Freiburg10_Cloning Protocol.pdf</A>
Deviation of standard protocol added: medium added to cells before incubation on thermomixer changed to DYT. Standard protocol changed. </LI></UL><H4><SPAN class="mw-headline" id="Picking_clones_pf_VP1-3_pEX_plasmids_from_DKFZ"><P style="font-size:15px; background-color:#66bbff;"><B>Picking clones pf VP1-3 pEX plasmids from DKFZ</B></P></SPAN></H4><P><B>Investigator: Bea</B>
Two clones of each plasmid were picked and inoculated in 10mL DYT medium containing 10 µL ampicillin
</P><UL><LI>VPex-1 clone 1
</LI><LI>VPex-1 clone 2
</LI><LI>VPex-2 clone 1
</LI><LI>VPex-2 clone 2
</LI><LI>VPex-3 clone 1
</LI><LI>VPex-3 clone 2
</LI></UL><P>and they were put in the 37°c room at 19:15
</P><H3><SPAN class="mw-headline" id="66._Labortag_22.7.2010"><P style="font-size:17px; background-color:#00dd77;">66. Labortag 22.7.2010 </P></SPAN></H3><H4><SPAN class="mw-headline" id="cell_culture"><P style="font-size:15px; background-color:#66bbff;">cell culture</P></SPAN></H4><P><B>Investigator: Adrian</B>
HT 1080 cells were split according to the standart protocol.
</P><H4><SPAN class="mw-headline" id="Mini_Prep_and_test_digestion_pKEX_VP1-3_w.2Fo_PstI.284073.29"><P style="font-size:15px; background-color:#66bbff;"><B>Mini Prep and test digestion  pKEX_VP1-3 w/o PstI(4073)</B></P></SPAN></H4><P><B>Investigators: Kerstin, Jessica</B><B>PstI should be deleted in VP1, VP2, VP3</B></P><UL><LI> Glycerol stock were prepared: 
</LI><LI> B55-B60
</LI></UL><LI>Plasmids
</LI><LI> pKEX_VP1 clone1:488,2 P72
</LI><LI> pKEX_VP2 clone1:484,8 P73
</LI><LI> pKEX_VP3 clone1:485,8 P74
</LI><LI> pKEX_VP1 clone2:499,2 P75
</LI><LI> pKEX_VP2 clone2:511,7 P76
</LI><LI> pKEX_VP3 clone2:470,4 P77</LI><P>Test digestion of three plasmids to check if the PstI 4073 is deleted (compared to the origin P27-P29)
</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">P72 /µl </TD><TD align="right">P27 /µl </TD><TD align="right">P73/µl</TD><TD align="right">P28/µl</TD><TD align="right">P74/µl</TD><TD align="right">P29
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 1,6</TD><TD align="right">1,7</TD><TD align="right">1,6</TD><TD align="right">8,1</TD><TD align="right">7,5</TD><TD align="right">7,4
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">1 </TD><TD align="right"> 1</TD><TD align="right"> 1</TD><TD align="right"> 1</TD><TD align="right"> 1</TD><TD align="right"> 1
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right"> 1</TD><TD align="right"> 1</TD><TD align="right"> 1</TD><TD align="right"> 1</TD><TD align="right"> 1</TD><TD align="right"> 0,5
</TD></TR><TR><TD>H2O</TD><TD align="right"> 10,9</TD><TD align="right"> 10,8</TD><TD align="right">10,9</TD><TD align="right">4,4</TD><TD align="right">5,0</TD><TD align="right">5,1
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"><B>15</B></TD><TD align="right"><B>15</B></TD><TD align="right"><B>15</B></TD><TD align="right"><B>15</B></TD><TD align="right"><B>15</B></TD><TD align="right"><B>15</B></TD></TR></TBODY></TABLE><P>
Digestion: 45 minutes, 37°C. Results:
</P><P>Deletion was succesful, there are still 2 PstI restriction sites in P72-74
</P><H4><SPAN class="mw-headline" id="Cloning_together_of_pAAV_RC_PstI_310_and_4073_with_two_enzymes"><P style="font-size:15px; background-color:#66bbff;">Cloning together of pAAV_RC PstI 310 and 4073 with two enzymes</P></SPAN></H4><P><B>Investigators: Chris, Patrick</B></P><P>Clone P58 (pAAV_RC_SDM310.1) and P59 (pAAV_RC_SDM4073) will be digested with HindII and SpeI. The smaller fragment (about 1700 bp) of P58 and the larger fragment of P59 (about 5900 bp) will be ligated to receive a plasmid carrying both mutations.
</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl (P59) </TD><TD align="right">volume of insert /µl (P58)
</TD></TR><TR><TD> DNA  </TD><TD align="right">2 </TD><TD align="right"> 5,5
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2</TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 2 (10x)</TD><TD align="right">2 </TD><TD align="right"> 2
</TD></TR><TR><TD>HindIII </TD><TD align="right">2 </TD><TD align="right"> 2
</TD></TR><TR><TD>SpeI </TD><TD align="right">2 </TD><TD align="right"> 2
</TD></TR><TR><TD>H2O</TD><TD align="right">12 </TD><TD align="right"> 9,5
</TD></TR><TR><TD><B>Total volume </B></TD><TD align="right">20 </TD><TD align="right"> 20 µl
</TD></TR></TBODY></TABLE><P>Incubation: at 37°C, 70 minutes.
</P><P>
The Gelextraction was performed according to the standard protocol. 
Weight of P58 gel extract: 50 µg 
Weight of P59 gel extract: 98 µg 
The received concentrations (measured by nanodrop):
P58: 3 ng/µl , P59: 4,7 ng/µl

Quickligation was not performed according to the standard protocol:
</P><UL><LI>2 µl P58 Insert
</LI><LI>2 µl P59 Vector
</LI><LI>5 µl Quickligase buffer (2x)
</LI><LI>1 µl Quickligase
</LI><LI>Total volume: 10 µl
</LI></UL><P>Incubation: 15 minutes at roomtemperature.

Transformation was performed according to the standard protocol with E. coli BL21. The agar plates were put into the 37°C room at 23:00.

New Ampicillin was prepared and put into the -20°C freezer.
</P><H4><SPAN class="mw-headline" id="Sequencing_of_linker"><P style="font-size:15px; background-color:#66bbff;">Sequencing of linker</P></SPAN></H4><P><B>Investigator: Anissa</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Linker</B></TD><TD align="left"><B>Sequencing</B></TD></TR><TR><TD align="left"> p65: middle linker </TD><TD align="left">was sucessful
</TD></TR><TR><TD align="left"> p66: GSAT linker </TD><TD align="left"> sequence didn’t arrive yet
</TD></TR><TR><TD align="left"> p67: long linker </TD><TD align="left"> was sucessful
</TD></TR><TR><TD align="left"> p68: split linker </TD><TD align="left"> false, <B>no linker</B> but YFP.Gerrit’s searching for the right sample
</TD></TR><TR><TD align="left"> p69: strep linker </TD><TD align="left"> the suffix of strep has a <B>double AgeI</B>! That’s why we have to cut off next time with AgeI (and digest long enough)
</TD></TR><TR><TD align="left"> p70: short linker </TD><TD align="left"> was sucessful
</TD></TR><TR><TD align="left"> p71: SEG linker </TD><TD align="left"> was sucessful
</TD></TR></TBODY></TABLE><P>
middle linker:
</P><P>
GSAT linker:
</P><P>
long linker:
</P><P>
Split linker:
</P><P>
Strep tag:
</P><P>Short linker:
</P><P>
SEG linker:
</P><H4><SPAN class="mw-headline" id="Biobricks_of_hgH_and_.C3.9F-Globin"><P style="font-size:15px; background-color:#66bbff;"><B>Biobricks of hgH and ß-Globin</B></P></SPAN></H4><P><B>Investigators: Stefan, Anna</B></P><P style="font-size:12px; font-weight: bold; color: blue;">Could you upload some agarose gels eg the gels after the digestion and the PCR? Thanks a lot! (Bea)</P><P><I>Comments:</I> Perform PCR with two primers
</P><P style="font-size:17px; font-weight: bold; color: blue;">PCR of ß-Globin and hgH:</P><UL><LI>DNA template: pAAV_MCS; concentration: 360 ng/µl
</LI></UL><UL><LI>PCR program: BGLOB
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume in µl</B></TD></TR><TR><TD> 5x Phusion HF buffer  </TD><TD align="right">10
</TD></TR><TR><TD> 10 mM dNTP mix </TD><TD align="right">1
</TD></TR><TR><TD>ß_globin_suffix (085) (1:10 dilution)</TD><TD align="right">2,5 (0,5 µM)
</TD></TR><TR><TD>ß_globin_prefix (086) (1:10 dilution)</TD><TD align="right">2,5 (0,5 µM)
</TD></TR><TR><TD>DNA template: pAAV_MCS (1:1000)</TD><TD align="right"> 5,6 µL of 1:1000 dilution
</TD></TR><TR><TD>DMSO</TD><TD align="right"> -
</TD></TR><TR><TD>Phusion polymerase</TD><TD align="right">0,5
</TD></TR><TR><TD>H2O</TD><TD align="right">27,9 µL
</TD></TR><TR><TD><B>Total volume (e.g. 50 µl)</B></TD><TD align="right">50
</TD></TR></TBODY></TABLE><UL><LI>PCR program: HGH
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume in µl</B></TD></TR><TR><TD> 5x Phusion HF buffer  </TD><TD align="right">10
</TD></TR><TR><TD> 10 mM dNTP mix </TD><TD align="right">1
</TD></TR><TR><TD> (083) (1:10 dilution)</TD><TD align="right">2,5 (0,5 µM)
</TD></TR><TR><TD> (084) (1:10 dilution)</TD><TD align="right">2,5 (0,5 µM)
</TD></TR><TR><TD>DNA template: pAAV_MCS (1:1000)</TD><TD align="right"> 5,6 µL of 1:1000 dilution
</TD></TR><TR><TD>DMSO</TD><TD align="right"> -
</TD></TR><TR><TD>Phusion polymerase</TD><TD align="right">0,5
</TD></TR><TR><TD>H2O</TD><TD align="right">27,9 µL
</TD></TR><TR><TD><B>Total volume (e.g. 50 µl)</B></TD><TD align="right">50
</TD></TR></TBODY></TABLE><P style="font-size:17px; font-weight: bold; color: blue;">Digestion of pSB1C3_RCF25_CFP:</P><UL><LI>DNA-Concentration: 230,9 ng/µl
</LI></UL><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right">5
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">3
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">3
</TD></TR><TR><TD>PstI </TD><TD align="right">1
</TD></TR><TR><TD>XbaI </TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">17
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right">30
</TD></TR></TBODY></TABLE><P style="font-size:14px; font-weight: bold; color: blue;">Gel running of digested vector and PCR products</P><UL><LI> 1% Agarose gel
</LI><LI> 3 µl Gelred, 8 µl DNA-Ladder-Mix
</LI><LI> 125 Volt, running time: 40 minutes
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>Sample/µl  </B></TD><TD align="right"> Loading dye (5x)/µl </TD><TD align="right"><B> Expected size/bp</B></TD></TR><TR><TD> Vector: 30  </TD><TD align="right">6 </TD><TD align="right">2080
</TD></TR><TR><TD> ß-Globin: 50 </TD><TD align="right">10 </TD><TD align="right">524
</TD></TR><TR><TD> hgH: 50 </TD><TD align="right">10 </TD><TD align="right">479
</TD></TR></TBODY></TABLE><P style="font-size:17px; font-weight: bold; color: blue;">Gelextraction</P><UL><LI>Gel extraction performed following standard protocol provided by Qiagen
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>Sample/  </B></TD><TD align="right"> Weight/g </TD><TD align="right"> QG buffer/µl</TD></TR><TR><TD> Vector  </TD><TD align="right">0,13</TD><TD align="right">390
</TD></TR><TR><TD> ß-Globin  </TD><TD align="right">0,28 </TD><TD align="right">840
</TD></TR><TR><TD> hgH </TD><TD align="right">0,24 </TD><TD align="right">720
</TD></TR></TBODY></TABLE><P style="font-size:17px; font-weight: bold; color: blue;">Digestion of PCR products</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right">30
</TD></TR><TR><TD> BSA (100x) </TD><TD align="right">0,5
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">4
</TD></TR><TR><TD>PstI </TD><TD align="right">1
</TD></TR><TR><TD>XbaI </TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">3,5
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right">40
</TD></TR></TBODY></TABLE><P style="font-size:17px; font-weight: bold; color: blue;">Purification of PCR products, Ligation, Transformation</P><P><B>procedure has been followed a different protocol (see Sven); has to be added in the standard protocols !</B></P><UL><LI>Measure DNA-concentration with Nanodrop:
</LI></UL><P>c(pSB1C3) = 12,42 ng/µL 
c(ß-Globin) = 51,93 ng/µL 
c(hgH)= 36,87 ng/µL 
</P><UL><LI>Calculation of volume needed for ligation:
</LI></UL><P>c(pSB1C3) = 7,3 µL 
c(ß-Globin) = 1,7 µL
</P><P>
c(pSB1C3) = 7,62 µL 
c(hgH) = 1,38 </P><P><B>Note: after heatshock cells were put on the shaker without adding 750 µl DYT, it was recognized and added later after 25 min of shaking !</B></P><H4><SPAN class="mw-headline" id="Picking_clones_of_pSB1C3_His_RFC25_and_pAAV_RFC25_GMK_TK30_for_Mini-Prep"><P style="font-size:15px; background-color:#66bbff;">Picking clones of pSB1C3_His_RFC25 and pAAV_RFC25_GMK_TK30 for Mini-Prep</P></SPAN></H4><P><B>Investigators: Stefan</B></P><P>Three clones of each construct were picked and inoculated in 10mL DYT medium containing 10 µL ampicillin (pAAV_RFC25_GMK_TK30) or 10 µL chloramphenicol (pSB1C3_His_RFC25).
</P><P>
They were put in the 37°C room at 20:33.
</P><H3><SPAN class="mw-headline" id="67._Labortag_23.7.2010"><P style="font-size:17px; background-color:#00dd77;">67. Labortag 23.7.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Picking_clones_of_pAAV_RC_1.1"><P style="font-size:15px; background-color:#66bbff;">Picking clones of pAAV_RC_1.1</P></SPAN></H4><P><B>Investigator: ChrisW, Patrick</B>
The inoculated agar plates from 'Cloning together of pAAV_RC PstI 310 and 4073 with two enzymes' were checked for colonies. Three clones were picked and in each case inoculated 20 ml DYT + Amp at 10:20.
</P><P>Mini prep of two pAAV_RC_SDM_PstI_310_4073. Results: It was checked visually whether enough E. coli BL21 cells had grown. Therefore only two midi preps were carried out, according to the standard protocol (Final elution with 50 µl). Unfortunately both of them yielded a quite low plasmid concentration, probably due to the short growth time (7 h). 
pAAV_RC mini pstI 301,4073: 44,7 ng/µl 
pAAV_RC mini 2 pstI 301,4073: 83,1 ng/µl </P><P>30 µl of both plasmids were sent for sequencing to GATC, named PS1 and PS2. Primers used: GATC_std_PolyC-D and GATC_std_SK. </P><P>Glycerol stock: 
B 67: pAAV_RC_SDM_PstI_310_4073 (1) 
B 68: pAAV_RC_SDM_PstI_310_4073 (2) </P><H4><SPAN class="mw-headline" id="Results_of_Trafo_Biobricks_of_hgH_and_.C3.9F-Globin"><P style="font-size:15px; background-color:#66bbff;">Results of Trafo Biobricks of hgH and ß-Globin</P></SPAN></H4><P><B>Investigator: Anissa, Bea</B></P><P style="font-size:12px; color:#68bbff;"><I><B>Comments:</B></I> No clones of pSB1C3_hGH and pSB1C3_betaGlobin grown on trafo plate. Possible reason can be the forgotten DYT medium added to the cells after the heat-shock. Cells were incubated at 37°C for 40 minutes without medium and were added afterewards.</P><P style="font-size:14px;">TO DO: Repeat ligation of 
<UL><LI>hgh PCR product and
</LI><LI>beta-globin PCR product
</LI></UL>
with pSB1C3 cut with XbaI and PstI.</P><UL><LI>Measure DNA-concentration with Nanodrop:
</LI></UL><P>c(pSB1C3) = 38,9 ng/µL 
c(ß-Globin) = 51,93 ng/µL 
c(hgH)= 36,87 ng/µL 
</P><UL><LI>Calculation of volume needed for ligation:
</LI></UL><P>c(pSB1C3) = 3,8 µL 
c(ß-Globin) = 5,2 µL
</P><P>
c(pSB1C3) = 3,8 µL 
c(hgH) = 5,2 <B>Attention: There was not enough cut vector. In both Ligations were only 2,2 µl of vector added. So incubation lasted 15 minutes and for transformation 4 µl were used.  </B></P><H4><SPAN class="mw-headline" id="Mini-Prep_and_Test_digestion_of_pAAV_RFC25_mGMK_TK30"><P style="font-size:15px; background-color:#66bbff;">Mini-Prep and Test digestion of pAAV_RFC25_mGMK_TK30 </P></SPAN></H4><P><B>Investigator: Adrian, Bea</B></P><P style="font-size:13px; color:#68bbff;"><I><B>Comments:</B></I> Mini-Prep, test digestion and sequencing is carried out in order to verify insertion of  the PCR product mGMK_TK30 with RFC25 add-on tails into pAAV_RFC25(overhangs). Aim is to use the construct in order to conduct experiments with the gmk_tk30 fusionenzyme. </P><P><B>Mini-Prep of pAAV_RFC25_mGMK_TK30 clone 1 to 3: </B>
After performing Mini-Preps  of clone 1 - 3 (elution volume with EB buffer was 50 µL) , concentration was measured and revealed following concentrations: 
</P><UL><LI>P81 = pAAV_RFC25_mGMK_TK30 clone 1: <B>c = 486 ng/µL</B></LI><LI>P82 = pAAV_RFC25_mGMK_TK30 clone 2: <B>c = 426 ng/µ</B>L
</LI><LI>P83 = pAAV_RFC25_mGMK_TK30 clone 3. <B>c = 457 ng/µL</B></LI></UL><P><B>Test digestion of pAAV_RFC25_mGMK_TK30 clone 1 to 3:</B></P><UL><LI>P81 = pAAV_RFC25_mGMK_TK30 clone 1
</LI><LI>P82 = pAAV_RFC25_mGMK_TK30 clone 2
</LI><LI>P83 = pAAV_RFC25_mGMK_TK30 clone 3
</LI></UL><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of <B>P81</B>/µl </TD><TD>volume of <B>P82</B>/µL</TD><TD>volume of <B>P83/</B>µL
</TD></TR><TR><TD> DNA  </TD><TD align="right">2 </TD><TD>2</TD><TD>2
</TD></TR><TR><TD> BSA (100x) </TD><TD align="right">1,5 </TD><TD>1,5</TD><TD>1,5
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">1,5 </TD><TD>1,5</TD><TD>1,5
</TD></TR><TR><TD>PstI </TD><TD align="right">0,5</TD><TD>0,5</TD><TD>0,5
</TD></TR><TR><TD>XbaI </TD><TD align="right">0,5</TD><TD>0,5</TD><TD>0,5
</TD></TR><TR><TD>H2O</TD><TD align="right">9</TD><TD>9</TD><TD>9
</TD></TR><TR><TD><B>Total volume /µl</B></TD><TD align="right">15</TD><TD>15</TD><TD>15
</TD></TR></TBODY></TABLE><P><B>Expected sizes:</B></P><UL><LI>mGMK_TK30:  1700bp
</LI><LI>pAAV_RFC25: 4600bp
</LI></UL><P style="font-size:13px; color:#11bb88;"><B>Results of samples loaded 1% agarose gel and ran 50 minutes at 110 V:</B></P><P style="font-size:13px; color:#11bb56;"><B>Sequencing of pAAV_RFC25_mGMK_TK30</B></P><P>70ng/µL were sent to GATC from each clone. They were labeled:
</P><UL><LI>AF_P81 (clone 1)
</LI><LI>AF_P82 (clone 2)
</LI><LI>AF_P83 (clone 3)
</LI></UL><H4><SPAN class="mw-headline" id="Mini-Prep_of_pSB1C3_His6"><P style="font-size:15px; background-color:#66bbff;">Mini-Prep of pSB1C3_His6</P></SPAN></H4><P><B>Investigator: Adrian</B></P><P>pSB1C3_6xHis clone 1 = P84 : 107,8 ng/µl (eluted with 50µl)
pSB1C3_6xHis clone 2 = P85 : 122,8 ng/µl (eluted with 50µl)
pSB1C3_6xHis clone 3 = P86 : 98,8 ng/µl  (eluted with 50µl)</P><H3><SPAN class="mw-headline" id="68._Labortag_24.7.2010"><P style="font-size:17px; background-color:#00dd77;">68. Labortag 24.7.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Cell_culture:_Seeding_of_HEK293_for_Transfektion_with_P81.2C_P82_and_P83"><P style="font-size:15px; background-color:#66bbff;"><B>Cell culture: Seeding of HEK293 for Transfektion with P81, P82 and P83</B></P></SPAN></H4><P><B>Investigator: Adrian</B></P><UL><LI>unfortunately the T75 flask with HEK-cells was overpopulated =&gt; to much cells =&gt;yellow medium and differentiation of the HEK-cells =&gt; thrown away
</LI><LI> I had to thaw cells, (P4) so there'll be no transfection at Monday :(
</LI></UL><H4><SPAN class="mw-headline" id="Cell_culture:_Transduction_of_HT1080_with_AAVs"><P style="font-size:15px; background-color:#66bbff;">Cell culture: Transduction of HT1080 with AAVs</P></SPAN></H4><P><B>Investigator: Adrian</B></P><P>
I. Plate I(A is up)
</P><TABLE border="1"><TBODY><TR><TD> control: no virus added</TD><TD align="right">virus without GOI added</TD><TD align="right"> virus with 10µg amount of GOI added
</TD></TR><TR><TD> control: no virus added</TD><TD align="right">virus without GOI added</TD><TD align="right"> virus with 10µg amount of GOI added
</TD></TR></TBODY></TABLE><P>II. Plate II(A is up)
</P><TABLE border="1"><TBODY><TR><TD> control: no virus added</TD><TD align="right">virus without GOI added</TD><TD align="right"> virus with 10µg amount of GOI added
</TD></TR><TR><TD> control: no virus added</TD><TD align="right">virus without GOI added</TD><TD align="right"> virus with 10µg amount of GOI added
</TD></TR></TBODY></TABLE><P>
III. Plate II(A is up)
</P><TABLE border="1"><TBODY><TR><TD> control: no virus added</TD><TD align="right">virus without GOI added</TD><TD align="right"> virus with 10µg amount of GOI added
</TD></TR><TR><TD> control: no virus added</TD><TD align="right">virus without GOI added</TD><TD align="right"> virus with 10µg amount of GOI added
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Discussion_round_nr._4"><P style="font-size:15px; background-color:#66bbff;">Discussion round nr. 4</P></SPAN></H4><P><B>Investigators: Adrian and his evil twins</B></P><UL><LI> adrian created a poster
</LI></UL><H4><SPAN class="mw-headline" id="Stocktaking_of_superior_goods"><P style="font-size:15px; background-color:#66bbff;">Stocktaking of superior goods</P></SPAN></H4><P><B>Investigator: Adrian</B>
superior goods:
</P><UL><LI> DYT/LB medium
</LI><LI> tips
</LI><LI> LB-agar
</LI></UL><H3><SPAN class="mw-headline" id="69._Labortag_25.7.2010"><P style="font-size:17px; background-color:#00dd77;">69. Labortag 25.7.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Clone_picking_from_pSB1C3_hgH_and_pSB1C3_betaglobin"><P style="font-size:15px; background-color:#66bbff;">Clone picking from pSB1C3_hgH and pSB1C3_betaglobin</P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="font-size:13px; color:#11bb88;"><B>Comments: Clones were very small and not many clones could be detected. Possible reasons: </B>
1) The ratio for a proper ligation need to be 3:1 insert: vector. There was not enough plasmid of the digested vector pSB1C3. 
2) The digested fragments of the PCR products hGH and beta-globin and the cutted vector were stored at -20°C. This could lead to the bad results of the transformation, because the overhangs were degraded. </P><P>
Two clones were picked of each trafo plate:
</P><UL><LI>pSB1C3_hGH
</LI><LI>pSB1C3_betaglobin
</LI></UL><P>They were inoculated in 10mL DYT medium containing 10µL chloramphenicol and put into 37°C room for incubating. The reason for picking only two clones was that not many more clones could be detected easily.
</P><H3><SPAN class="mw-headline" id="70._Labortag_26.7.2010"><P style="font-size:17px; background-color:#00dd77;">70. Labortag 26.7.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Mini_Prep_and_test_digestion_pSB1C3_hgh_and_pSB1C3_beta-globin"><P style="font-size:15px; background-color:#66bbff;">Mini Prep and test digestion  pSB1C3_hgh and pSB1C3_beta-globin</P></SPAN></H4><P><B>Investigators: Anissa, Kerstin</B></P><UL><LI> Glycerol stock were prepared: 
</LI><LI> B69-B71
</LI></UL><LI>Plasmids
</LI><LI> pSB1C3_hgh clone1: 139 ng/µl   p89
</LI><LI> pSB1C3_hgh clone2: 209,4 ng/µl   p90
</LI><LI> pSB1C3_beta-globin clone1: 217,5 ng/µl   p91 </LI><P> 
Incubation time: 1,5h Incubation temperature:37°C
</P><P>Test digestion with Xba and PstI to check if hgh and beta-globin were cloned succesfully into the pSB1C3, respectively.
</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">p89 /µl </TD><TD align="right">p90 /µl </TD><TD align="right">p91/µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 4,3</TD><TD align="right">4,3</TD><TD align="right">4,1
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">1,5 </TD><TD align="right"> 1,5</TD><TD align="right"> 1,5
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right"> 1,5</TD><TD align="right"> 1,5</TD><TD align="right"> 1,5
</TD></TR><TR><TD>H2O</TD><TD align="right"> 6,7</TD><TD align="right"> 6,7</TD><TD>  6,9
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"><B>15</B></TD><TD align="right"><B>15</B></TD><TD align="right"><B>15</B></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Test_digestion_P87:_pAAV_RC_PstI-301-4073_clone_1_and_P88:_pAAV_RC_PstI-301-4073_clone_2"><P style="font-size:15px; background-color:#66bbff;">Test digestion  P87: pAAV_RC_PstI-301-4073 clone 1 and P88: pAAV_RC_PstI-301-4073 clone 2</P></SPAN></H4><P><B>Investigator: Chris W.</B></P><UL><LI>Plasmids
</LI><LI> pAAV_RC_PstI-301-4073 clone1: 44,2 ng/µl   p87
</LI><LI> pAAV_RC_PstI-301-4073 clone2: 83,2 ng/µl   p88
</LI><LI> pAAV_RC: 429,8 ng/µl   p50 </LI></UL><P> Incubation time: 1,5h Incubation temperature:37°C
</P><P>Test digestion with PstI 0,5µl
</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">p87 /µl </TD><TD align="right">p88 /µl </TD><TD align="right">p50/µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 13,6</TD><TD align="right">7,2</TD><TD align="right">0,72
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">1,5 </TD><TD align="right"> 1,5</TD><TD align="right"> 1,5
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right"> 1,5</TD><TD align="right"> 1,5</TD><TD align="right"> 1,5
</TD></TR><TR><TD>H2O</TD><TD align="right"> 2,9</TD><TD align="right"> 9,3</TD><TD>  15,78
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"><B>20</B></TD><TD align="right"><B>20</B></TD><TD align="right"><B>20</B></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Oligos_for_hTERT_promoter"><P style="font-size:15px; background-color:#66bbff;">Oligos for hTERT promoter</P></SPAN></H4><P><B>Investigator: Bea</B></P><P>Oligos were ordered to modify hTERt promoter in order to obtain hTERT in the iGEM RFC10 standard.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:922px;"><DIV class="thumbcaption">Ordered oligos at Sigma-Aldrich for amplifying hTERT promoter.</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Retrafo_with_pTVvector_containing_the_CMV_promoter"><P style="font-size:15px; background-color:#66bbff;">Retrafo with pTVvector containing the CMV promoter</P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="font-size:13px; color:#11bb88;"><B>Comments</B>: In order to obtain the CMV promoter in the correct RFC10 standard, oligos werde designed and ordered. For using the plasmid obtained from the iGEM team 2008, a retrafo has to be performed.  </P><P style="font-size:14px;"> Re-transformation: </P><UL><LI>Construct used: expressionvector + CMV (BBa_TV_CMV)
</LI><LI>clone 7
</LI><LI>clone 8
</LI></UL><LI>concentration measureed with NanoDrop:
</LI><LI>clone 7 (Bba_TV_VMV_7): c = 334, 65 ng/µL
</LI><LI>clone 8 (Bba_TV_VMV_8): c = 183,77 ng/µL
</LI><LI>Plasmid need to be diluted in order to obatain the final 100pg. </LI><LI>Dilution: 1:1000
</LI><LI>Volume needed of 1:1000 dilution of each clone:
</LI><LI>clone 7 (Bba_TV_VMV_7): v = 0,29 µL
</LI><LI>clone 8 (Bba_TV_VMV_8): v = 0,54 µL
</LI><LI>100 µL aliquot of <B>XL1-B</B> were thawed on ice and divided into 50µL each. The proper amount of each clone was added.
</LI><H4><SPAN class="mw-headline" id="Continuation:_ITRs"><P style="font-size:17px; background-color:#66bbff;">Continuation: ITRs</P></SPAN></H4><P><B>Investigator: Hanna</B>
In order to convert also the left ITR into RFC10 standard, the FANCY method was repeated with the newly arrived oligos (präfix and suffix).
</P><P style="font-size:20px; font-weight: bold; color: blue;"><U>Practical Cloning: Digestion of pAAV_MCS with AlwNI</U></P><UL><LI>plasmid: P9 (270 ng/µL) = pAAV_MCS
</LI><LI>buffer used: 4 ; Restriction-enzymes used: Enzyme 1 (no. Lab:120) AlwNI
</LI></UL><P><B>Comments:</B> No.
</P><P style="font-size:15px; font-weight: bold; color: blue;">Digestion</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right"> Volume [µL]
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 15
</TD></TR><TR><TD> BSA (100x) </TD><TD align="right"> 0.2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right"> 2
</TD></TR><TR><TD>Enzyme AlwNI (no.Lab:120)</TD><TD align="right"> 2
</TD></TR><TR><TD>H2O</TD><TD align="right"> 0.8
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><UL><LI>Incubation: 1 h 20 minutes 
</LI></UL><P style="font-size:15px; font-weight: bold; color: blue;">Agarose-Gel:</P><P> 
0.5 g Agarose, 50 mL TAE (1 %), 3 µL GELRED, at 115 Volt, running time: 50 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (6x)/µl
</TH><TH>Expected size 1 (Geneious)
</TH><TH>Expected size 2 (Geneious)
</TH></TR><TR><TD>P9
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD>1674 bp = left ITR
</TD><TD>2982 bp = right ITR
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker [µL]
</TH><TH>Sample P9 [µL]
</TH></TR><TR><TH>Lane
</TH><TD>8
</TD><TD>24
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color: blue;">Gelextraction</P><P>
Gel measurement:
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Sample</B></TD><TD align="left"><B>weight</B></TD><TD align="left"><B>concentration</B></TD></TR><TR><TD align="left"> left ITR fragment </TD><TD align="left"> 0.07 g </TD><TD align="left"> 35.65 ng/µL
</TD></TR><TR><TD align="left"> right ITR fragment </TD><TD align="left"> 0.15 g </TD><TD align="left"> 66.05 ng/µL
</TD></TR></TBODY></TABLE><P style="font-size:20px; font-weight: bold; color: blue;"><U>Practical Cloning: Digestion of left ITR fragment with NotI + PstI</U></P><UL><LI>buffer used: 4 ; Restriction-enzymes used: Enzyme 1 NotI-HF; Enzyme 2 PstI-HF
</LI></UL><P><B>Comments:</B> Due to the digestion of an gelex- and very small fragment (ITR ~ 150 bp), the whole gelex-volume was used for digestion. 
</P><P style="font-size:15px; font-weight: bold; color: blue;">Digestion</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right"> Volume [µL]
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 19 µL = 0.68 µg
</TD></TR><TR><TD> BSA (100x) </TD><TD align="right"> 0.25
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right"> 2.5
</TD></TR><TR><TD>Enzyme NotI-HF </TD><TD align="right"> 1
</TD></TR><TR><TD>Enzyme PstI-HF </TD><TD align="right"> 1
</TD></TR><TR><TD>H2O</TD><TD align="right"> 1.25
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 25
</TD></TR></TBODY></TABLE><UL><LI>Incubation: 1 h 15 minutes 
</LI></UL><P style="font-size:15px; font-weight: bold; color: blue;">Agarose-Gel:</P><P> 
1.3 g Agarose, 65 mL TAE (2 %), 3 µL GELRED, at 115 Volt, running time: 45 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (5x)/µl
</TH><TH>Expected size 1 (Geneious)
</TH><TH>Expected size 2 (Geneious)
</TH></TR><TR><TD>left ITR fragment
</TD><TD>25 µl
</TD><TD>4 µl
</TD><TD>145 bp = left ITR
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker [µL]
</TH><TH>Sample left ITR [µL]
</TH></TR><TR><TH>Lane
</TH><TD>8
</TD><TD>29
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color: blue;">Gelextraction</P><P>
Gel measurement:
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Sample</B></TD><TD align="left"><B>weight</B></TD><TD align="left"><B>concentration</B></TD></TR><TR><TD align="left"> left ITR </TD><TD align="left"> 0.09 g </TD><TD align="left"> 3.59 ng/µL
</TD></TR></TBODY></TABLE><P style="font-size:20px; font-weight: bold; color: blue;"><U>Practical Cloning: Digestion of pGA14_Präfix-rITR with EcoRI + PstI</U></P><UL><LI>buffer used: 4 ; Restriction-enzymes used: Enzyme 1 EcoRI-HF; Enzyme 2 PstI-HF
</LI></UL><P><B>Comments:</B> No. 
</P><P style="font-size:15px; font-weight: bold; color: blue;">Digestion</P><P>
Plasmid: P63.1; c = 501.14 ng/µL
</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right"> Volume [µL]
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 4
</TD></TR><TR><TD> BSA (100x) </TD><TD align="right"> 0.2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right"> 2
</TD></TR><TR><TD>Enzyme NotI-HF </TD><TD align="right"> 1
</TD></TR><TR><TD>Enzyme PstI-HF </TD><TD align="right"> 1
</TD></TR><TR><TD>H2O</TD><TD align="right"> 11.8
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><UL><LI>Incubation: 1 h 15 minutes 
</LI></UL><P style="font-size:15px; font-weight: bold; color: blue;">Agarose-Gel:</P><P> 
1.3 g Agarose, 65 mL TAE (2 %), 3 µL GELRED, at 115 Volt, running time: 45 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (5x)/µl
</TH><TH>Expected size 1 (Geneious)
</TH><TH>Expected size 2 (Geneious)
</TH></TR><TR><TD>Präfix - right ITR
</TD><TD>20 µl
</TD><TD>3.5 µl
</TD><TD>165 bp = left ITR
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker [µL]
</TH><TH>Sample Präfix-rITR [µL]
</TH></TR><TR><TH>Lane
</TH><TD>8
</TD><TD>23
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color: blue;">Gelextraction</P><P>
Gel measurement:
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Sample</B></TD><TD align="left"><B>weight</B></TD><TD align="left"><B>concentration</B></TD></TR><TR><TD align="left"> Präfix-rITR </TD><TD align="left"> 0.08 g </TD><TD align="left"> 7.55 ng/µL
</TD></TR></TBODY></TABLE><P>
Gel picture of left ITR fragment and Präfix-rITR fragment: </P><P><B>Comment:</B> The left ITR fragment was, as expected, nearly not detectable. Nevertheless it was cut out of the gel, because the same probleme occured during the first fancy-performance, which turned out to be successful! 
</P><P style="font-size:20px; font-weight: bold; color: blue;"><U>Practical Cloning: Digestion of pGA14 with EcoRI + NotI </U></P><UL><LI>plasmid: P61 (200 ng/µL) = pGA14_Cerulean
</LI><LI>buffer used: 4 ; Restriction-enzymes used: Enzyme 1 = EcoRI-HF; Enzyme 2 = NotI-HF
</LI></UL><P><B>Comments:</B> This digested vector used used for both ligation approaches.
</P><P style="font-size:15px; font-weight: bold; color: blue;">Digestion</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right"> Volume [µL]
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 10
</TD></TR><TR><TD> BSA (100x) </TD><TD align="right"> 0.3
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right"> 3
</TD></TR><TR><TD>Enzyme EcoRI-HF </TD><TD align="right"> 1.5
</TD></TR><TR><TD>Enzyme NotI-HF </TD><TD align="right"> 1.5
</TD></TR><TR><TD>H2O</TD><TD align="right"> 13.7
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 30
</TD></TR></TBODY></TABLE><UL><LI>Incubation: 1 h 15 minutes 
</LI></UL><P style="font-size:15px; font-weight: bold; color: blue;">Agarose-Gel:</P><P> 
0.5 g Agarose, 50 mL TAE (1 %), 3 µL GELRED, at 115 Volt, running time: 60 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (5x)/µl
</TH><TH>Expected size 1 (Geneious)
</TH></TR><TR><TD>pGA14
</TD><TD>30 µl
</TD><TD>4 µl
</TD><TD>2887 bp
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker [µL]
</TH><TH>Sample pGA14 [µL]
</TH></TR><TR><TH>Lane
</TH><TD>8
</TD><TD>34
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color: blue;">Gelextraction</P><P>
Gel measurement:
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Sample</B></TD><TD align="left"><B>weight</B></TD><TD align="left"><B>concentration</B></TD></TR><TR><TD align="left"> pGA14 </TD><TD align="left"> 0.08 g </TD><TD align="left"> 13.63 ng/µL
</TD></TR></TBODY></TABLE><P style="font-size:20px; font-weight: bold; color: blue;"><U>Hybridization of ITR-Oligos</U></P><P><B>left ITR:</B></P><UL><LI> 10 µL Oligo 1 (O100 = lITR_<B>Präfix</B>_RFC10_for bzw. O104 = rITR_<B>Suffix</B>_RFC10_for)
</LI><LI> 10 µL Oligo 2 (O101 = lITR_<B>Präfix</B>_RFC10_rev bzw. O105 = rITR_<B>Suffix</B>_RFC10_rev)
</LI><LI> 4 µL 100 mM TrisHCl pH8
</LI><LI> 8 µL 5 mM MgCl2
</LI><LI> 8 µL H2O
</LI></UL><P>Total: 40 mL
</P><P>Because some of the oligos produce strong secondary structures, we used the ORIGAMI1 program:
1) initial denaturation: 99°C, 7 minutes
2) 99°C, 1 minute
3) 73 x repetition of 2) -&gt; -1°C, R = 0.3 °C/sec
4) hold 4°C</P><P style="font-size:20px; font-weight: bold; color: blue;"><U>Ligation</U></P><P>
1. Ligation of left ITR + Präfix + pGA14:
</P><UL><LI> Präfix-Oligos: 1 µL
</LI><LI> left ITR: 8 µL
</LI><LI> pGA14: 8 µL
</LI><LI> Buffer: 2 µL
</LI><LI> T4 Ligase: 1 µL
</LI></UL><P>Total: 20 µL </P><P>2. Ligation of Präfix-rITR  + Suffix+ pGA14:
</P><UL><LI> Präfix-Oligos: 1 µL
</LI><LI> Präfix-rITR: 8 µL
</LI><LI> pGA14: 8 µL
</LI><LI> Buffer: 2 µL
</LI><LI> T4 Ligase: 1 µL
</LI></UL><P>Total: 20 µL 
The ligation approaches were incubated at 16°C over night (thermocycler).
</P><H4><SPAN class="mw-headline" id="Biobrick_of_GMK"><P style="font-size:17px; background-color:#66bbff;">Biobrick of GMK</P></SPAN></H4><P><B>Investigators: Stefan</B></P><P><B>aim:</B> Biobrick construction of pAAV_RFC25_gmk for further addition of TK30.<B>result:</B> Transformation worked out, clones can be picked. 
expected construct:</P><P>Digestion with XbaI and AgeI and ligate into vector which was digested with XbaI and AgeI as well.</P><P>
PCR of gmk:
</P><UL><LI>DNA template: puB6_V5_His6_clone1 + TK/GMK; concentration: 493,7 ng/µl
</LI></UL><UL><LI>PCR program: GMK
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume in µl</B></TD></TR><TR><TD> 5x Phusion HF buffer  </TD><TD align="right">10
</TD></TR><TR><TD> 10 mM dNTP mix </TD><TD align="right">1
</TD></TR><TR><TD>pmGMK_TK30_prefix_RFC25_for (082) (1:10 dilution)</TD><TD align="right">2,5 (0,5 µM)
</TD></TR><TR><TD>pmGMK_suffix_RFC25_rev (080) (1:10 dilution)</TD><TD align="right">2,5 (0,5 µM)
</TD></TR><TR><TD>DNA template: puB6_V5_His6_clone1 + TK/GMK (1:100)</TD><TD align="right"> 0,5 µL of 1:100 dilution
</TD></TR><TR><TD>DMSO</TD><TD align="right"> -
</TD></TR><TR><TD>Phusion polymerase</TD><TD align="right">0,5
</TD></TR><TR><TD>H2O</TD><TD align="right">33 µL
</TD></TR><TR><TD><B>Total volume (e.g. 50 µl)</B></TD><TD align="right">50
</TD></TR></TBODY></TABLE><P style="font-size:17px; font-weight: bold; color: blue;">Digestion of pAAV_RCF25:</P><UL><LI>DNA-Concentration: 321,5 ng/µl
</LI></UL><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right">4
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">3
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">3
</TD></TR><TR><TD>AgeI </TD><TD align="right">1
</TD></TR><TR><TD>XbaI </TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">18
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right">30
</TD></TR></TBODY></TABLE><P style="font-size:17px; font-weight: bold; color: blue;">Gel running of digested vector and PCR products</P><UL><LI> 1% Agarose gel
</LI><LI> 3 µl Gelred, 8 µl DNA-Ladder-Mix
</LI><LI> 125 Volt, running time: 60 minutes
</LI></UL><P>
Sample of Hanna was loaded as well, see extra journal entry.</P><TABLE border="1"><TBODY><TR><TD><B>Sample/µl  </B></TD><TD align="right"> Loading dye (5x)/µl </TD><TD align="right"><B> Expected size/bp</B></TD></TR><TR><TD> Vector/ 30  </TD><TD align="right">6 </TD><TD align="right">4636
</TD></TR><TR><TD> gmk/    50 </TD><TD align="right">10 </TD><TD align="right">603
</TD></TR></TBODY></TABLE><P style="font-size:17px; font-weight: bold; color: blue;">Gelextraction</P><UL><LI>Gel extraction performed following standard protocol provided by Qiagen
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>Sample/  </B></TD><TD align="right"> Weight/g </TD><TD align="right"> QG buffer/µl</TD></TR><TR><TD> pAAV_RFC25  </TD><TD align="right">0,15 </TD><TD align="right">450
</TD></TR><TR><TD> gmk  </TD><TD align="right">0,21 </TD><TD align="right">630
</TD></TR></TBODY></TABLE><P style="font-size:17px; font-weight: bold; color: blue;">Digestion of PCR products</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right">30
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">4
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">4
</TD></TR><TR><TD>AgeI </TD><TD align="right">1
</TD></TR><TR><TD>XbaI </TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">0
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right">40
</TD></TR></TBODY></TABLE><P style="font-size:17px; font-weight: bold; color: blue;">Purification of PCR products, Ligation, Transformation</P><P><B>Purification has been followed by a different protocol (see Sven); has to be added in the standard protocols !</B></P><UL><LI>Measure DNA-concentration with Nanodrop:
</LI></UL><P>c(pAAV_RFC25) = 35,46 ng/µL 
c(gmk) = 72,84 ng/µL </P><UL><LI>Calculation of volume needed for ligation:
</LI></UL><P>c(pAAV_RFC25) = 7,56 µL 
c(gmk) = 1,44 µL</P><H3><SPAN class="mw-headline" id="71._Labortag_27.7.2010"><P style="font-size:17px; background-color:#00dd77;">71. Labortag 27.7.2010 </P></SPAN></H3><H4><SPAN class="mw-headline" id="Cloning_of_linkers_in_pSB1C3_RFC25_CFP1.1"><P style="font-size:15px; background-color:#66bbff;">Cloning of linkers in pSB1C3_RFC25_CFP1.1 </P></SPAN></H4><P><B>Investigator: Jessica</B><B>Linkers longlinker, SEG and GSAT should be cloned into the RFC25-standard</B></P><UL><LI> P66: GSAT_linker_pMA 242,7 ng/µl
</LI><LI> P67: LonglinkerA50_pMA 230,4 ng/µl
</LI><LI> P71: SEG_pMA 211,8 ng/µl
</LI><LI> P51: pSB1C3_RFC25_CFP1.1 230,1 ng/µl
</LI></UL><P>all linkers and pSB1C3_RFC25_CFP1.1 are digested with AgeI and XbaI for 2 hours
</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl (P66) </TD><TD align="right">volume of insert /µl (P67)</TD><TD align="right">volume of insert /µl (P71)</TD><TD align="right">volume of insert /µl (P51)
</TD></TR><TR><TD> DNA  </TD><TD align="right">8,2 </TD><TD align="right"> 8,7</TD><TD align="right"> 9,4</TD><TD align="right"> 8,7
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2</TD><TD align="right"> 2</TD><TD align="right"> 2</TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right"> 2</TD><TD align="right"> 2</TD><TD align="right"> 2
</TD></TR><TR><TD>AgeI </TD><TD align="right">1 </TD><TD align="right"> 1</TD><TD align="right"> 1</TD><TD align="right"> 1
</TD></TR><TR><TD>XbaI </TD><TD align="right">1 </TD><TD align="right"> 1</TD><TD align="right"> 1</TD><TD align="right"> 1
</TD></TR><TR><TD>H2O</TD><TD align="right">5,8 </TD><TD align="right"> 5,3</TD><TD align="right"> 4,6</TD><TD align="right"> 5,3
</TD></TR><TR><TD><B>Total volume </B></TD><TD align="right">20 </TD><TD align="right"> 20 µl</TD><TD align="right"> 20 µl</TD><TD align="right"> 20 µl
</TD></TR></TBODY></TABLE><UL><LI> 1% Agarose gel
</LI><LI> 3 µl Gelred,  µl DNA-Ladder-Mix for P51, Low range for P66,67,71
</LI><LI> 130 Volt, running time: 45 minutes for P51, 90 Volt, running time 30 minutes for P66,67,71
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>Sample/µl </B></TD><TD><B> Expected size/bp</B></TD></TR><TR><TD> Vector P51  </TD><TD align="right">2078
</TD></TR><TR><TD> Insert P66 </TD><TD align="right">126
</TD></TR><TR><TD> Insert P67 </TD><TD align="right">54
</TD></TR><TR><TD> Insert P71 </TD><TD align="right">126
</TD></TR></TBODY></TABLE><UL><LI> weight of P66 gel extract: 0,04g
</LI><LI> weight of P67 gel extract: 0,09g
</LI><LI> weight of P71 gel extract: 0,1g
</LI><LI> weight of P51 gel extract: 0,15g
</LI></UL><P>finally called:
</P><UL><LI> pSB1C3_RFC25_longlinker <B>P96</B></LI><LI> pSB1C3_RFC25_GSAT <B>P97</B></LI><LI> pSB1C3_RFC25_SEG <B>P98</B></LI></UL><P><B>Nanodrop</B></P><UL><LI> Longlinker (P67): 2,1ng/µl
</LI><LI> SEG (P71): 3,7ng/µl
</LI><LI> GSAT (P66): 1,4ng/µl
</LI><LI> pSB1C3_RFC25: 29,6ng/µl</LI></UL><P><B>Ligation</B></P><UL><LI>P51:P66 - 2,2µl:6,8µl
</LI><LI>P51:P67 - 2,53µl:6,47µl
</LI><LI>P51:P71 - 5,54µl:3,46µl</LI></UL><P><B>Transformation</B></P><UL><LI> 4µl DNA was added to 50µl XL1b
</LI></UL><H4><SPAN class="mw-headline" id="Result_sequencing_of_pAAV_RFC25_mGMK_TK30"><P style="font-size:15px; background-color:#66bbff;">Result sequencing of pAAV_RFC25_mGMK_TK30</P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="font-size:13px; color:#66bbff;"><B>Comments:</B> Results of the PCR and cloning experiment from lab day: 23.07.2010 (Mini-Prep). The sucessful insertion of mGMK_TK 30 into the vector pAAV_RFC25 can be confirmed and is ready for cell culture.</P><P>Results sequencing of <I>plasmid <B>P81</B></I> (pAAV_RFC25_mGMK_TK30 clone 1)</P><P>Results sequencing of <I>plasmid <B>P82</B></I> (pAAV_RFC25_mGMK_TK30 clone 2)</P><P>Results sequencing of <I>plasmid <B>P83</B></I> (pAAV_RFC25_mGMK_TK30 clone 3)</P><H4><SPAN class="mw-headline" id="Quickchange_.40_Pst_in_RepX_68_and_RepX_78"><P style="font-size:15px; background-color:#66bbff;">Quickchange @ Pst in RepX 68 and RepX 78</P></SPAN></H4><P><B>Investigator: Kira</B></P><P><B>PCR program</B>: PstI 
</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume in µl
</TD></TR><TR><TD> 10x Phusion HF buffer  </TD><TD align="right"> 2.5 µl
</TD></TR><TR><TD> 10 mM dNTP mix </TD><TD align="right"> 0.5 µl
</TD></TR><TR><TD> __310 ____primer_for (1:10 dilution)</TD><TD align="right">  0,39 µl
</TD></TR><TR><TD> __310____primer_rev (1:10 dilution)</TD><TD align="right">  0,39 µl
</TD></TR><TR><TD>DNA template (1:20 dilution)</TD><TD align="right">  0,5 µl
</TD></TR><TR><TD>DMSO</TD><TD align="right">  0 µl
</TD></TR><TR><TD>Phusion polymerase</TD><TD align="right">  0.5 µl
</TD></TR><TR><TD>H2O</TD><TD align="right"> 20,22 µl
</TD></TR><TR><TD><B>Total volume (e.g. 50 µl)</B></TD><TD align="right"> 25 µl
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD> cycles   </TD><TD align="right">temperature </TD><TD align="right">time
</TD></TR><TR><TD> 1 </TD><TD align="right">95 C </TD><TD align="right">2 min
</TD></TR><TR><TD> 20 </TD><TD align="right">95 C </TD><TD align="right">30 sec
</TD></TR><TR><TD align="right">66 C </TD><TD align="right">1 min
</TD></TR><TR><TD align="right">68 C </TD><TD align="right">5 min
</TD></TR></TBODY></TABLE><UL><LI> The PCR-products were digested with 0.5 µl DpnI
</LI><LI> Transformation was performed according to the standard protocol in XL1B cells
</LI></UL><H4><SPAN class="mw-headline" id="Site-directed_mutagenesis_of_pAAV_RC_PstI_310.2B4073_.28pAAV_RC_1.1.29_with_BamHI_and_SalI"><P style="font-size:15px; background-color:#66bbff;"><B>Site-directed mutagenesis of pAAV_RC_PstI_310+4073 (pAAV_RC_1.1) with BamHI and SalI</B></P></SPAN></H4><P><B>Investigator: Adrian, Bea</B></P><P style="font-size:13px; color:#66bbff;"><B>Comments:</B> Site-directed mutagenesis was perfomed with pAAV_RC_PstI_310+4073 (pAAV_RC_1.1) in order to delete BamHI and SalI in the plasmid. These restriciton sites need to be removed because they are the single-cutter in the loops which will be used for targeting. </P><P><B>Protocol: Different PCR programs were used for performing the Quickchange site-directed mutagenesis protocol.</B></P><OL><LI> Longer elongation/extension time 
</LI><LI> Shorter elongation/extension time
</LI></OL><P style="font-size:13px; color:#25bbAA;">1. <B>PCR reaction with PCR program with longer elongation time</B>: </P><P>DNA template used: pAAV_RC_PstI_310+4073 (pAAV_RC_1.1) = P87
A dilution of 1:20 was prepared and it was calculated the amount of DNA template needed for each site-directed mutagenesis reaction.
</P><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume in µl</B></TD><TD align="right"><B>volume in µl</B>|
</TD></TR><TR><TD> 10x Phu Ultra HF II buffer  </TD><TD align="right"> 2.5 µl</TD><TD align="right"> 2.5 µl
</TD></TR><TR><TD> 10 mM dNTP mix </TD><TD align="right"> 0.5 µl</TD><TD align="right"> 0.5 µl
</TD></TR><TR><TD>  primer_for (1:10 dilution)</TD><TD align="right"><B>O68= SDM SalI: 0,67 µl</B></TD><TD align="right"><B> O66= SDM BamHI: 0,68 µl</B></TD></TR><TR><TD>  primer_rev (1:10 dilution)</TD><TD align="right"><B>O69= SDM SalI: 0,67 µl</B></TD><TD align="right"><B>O67= SDM BamHI: 0,72 µl</B></TD></TR><TR><TD><B>P87</B> (DNA template) (1:20 dilution)</TD><TD align="right">  0,83 µl</TD><TD align="right">  0,83 µl
</TD></TR><TR><TD>DMSO</TD><TD align="right">  -</TD><TD align="right">  -
</TD></TR><TR><TD>Phusion polymerase</TD><TD align="right">  0.5 µl</TD><TD align="right">  0.5 µl
</TD></TR><TR><TD>H2O</TD><TD align="right"> 19,33 µl</TD><TD align="right"> 18,77 µl
</TD></TR><TR><TD><B>Total volume (e.g. 50 µl)</B></TD><TD align="right"> 25 µl</TD><TD align="right"> 25 µl
</TD></TR></TBODY></TABLE><P style="font-size:13px; color:#25bbAA;"><B>Used PCR programs for both SDMs</B>: </P><P><B>Comment:</B> Program has a long extension time because Quikchange Kit (Stratagene) recommends 1 minute per kb. Our vector has a length of 7300 bp therefore 7 minutes and 30 seconds were chosen for proper elongation of the whole vector.
</P><TABLE border="1"><TBODY><TR><TD><B>Rounds</B></TD><TD align="right"><B>Temperature</B></TD><TD align="right"><B>Time</B></TD></TR><TR><TD>1 </TD><TD align="right">95°C </TD><TD align="right">2'
</TD></TR><TR><TD>20 </TD><TD align="right">95°C </TD><TD align="right">30
</TD></TR><TR><TD align="right">60°C </TD><TD align="right">1'
</TD></TR><TR><TD align="right">68°C </TD><TD align="right">7 min 30 sec
</TD></TR><TR><TD>   Hold </TD><TD align="right">4°C </TD></TR></TBODY></TABLE><P style="font-size:13px; color:#25bbAA;"><B>2. PCR reaction with PCR program with shorter elongation time</B>: </P><P>DNA template used: pAAV_RC_PstI_310+4073 (pAAV_RC_1.1) = P87
A dilution of 1:20 was prepared and it was calculated the amount of DNA template needed for each site-directed mutagenesis reaction.
</P><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume in µl</B></TD><TD align="right"><B>volume in µl</B>|
</TD></TR><TR><TD> 10x Phu Ultra HF II buffer  </TD><TD align="right"> 2.5 µl</TD><TD align="right"> 2.5 µl
</TD></TR><TR><TD> 10 mM dNTP mix </TD><TD align="right"> 0.5 µl</TD><TD align="right"> 0.5 µl
</TD></TR><TR><TD>  primer_for (1:10 dilution)</TD><TD align="right"><B>O68= SDM SalI: 0,67 µl</B></TD><TD align="right"><B> O66= SDM BamHI: 0,68 µl</B></TD></TR><TR><TD>  primer_rev (1:10 dilution)</TD><TD align="right"><B>O69= SDM SalI: 0,67 µl</B></TD><TD align="right"><B>O67= SDM BamHI: 0,72 µl</B></TD></TR><TR><TD><B>P87</B> (DNA template) (1:20 dilution)</TD><TD align="right">  0,83 µl</TD><TD align="right">  0,83 µl
</TD></TR><TR><TD>DMSO</TD><TD align="right">  -</TD><TD align="right">  -
</TD></TR><TR><TD>Phusion polymerase</TD><TD align="right">  0.5 µl</TD><TD align="right">  0.5 µl
</TD></TR><TR><TD>H2O</TD><TD align="right"> 19,33 µl</TD><TD align="right"> 18,77 µl
</TD></TR><TR><TD><B>Total volume (e.g. 50 µl)</B></TD><TD align="right"> 25 µl</TD><TD align="right"> 25 µl
</TD></TR></TBODY></TABLE><P style="font-size:13px; color:#25bbAA;"><B>Used PCR programs for both SDMs</B>: </P><P><B>Comment:</B> Quikchange Protocol (Stratagene) recommends 30 seconds per kb chosing a elongation temperature of 68°C. Our vector has a length of 7300 bp therefore 4 minutes were chosen for proper elongation of the whole vector. In order to see wheter this PCR program works aswell the same PCR ingredients were chosen as it can be seen in the table above.
</P><TABLE border="1"><TBODY><TR><TD><B>Rounds</B></TD><TD align="right"><B>Temperature</B></TD><TD align="right"><B>Time</B></TD></TR><TR><TD>1 </TD><TD align="right">95°C </TD><TD align="right">2'
</TD></TR><TR><TD>20 </TD><TD align="right">95°C </TD><TD align="right">30
</TD></TR><TR><TD align="right">60°C </TD><TD align="right">1'
</TD></TR><TR><TD align="right">68°C </TD><TD align="right">4'
</TD></TR><TR><TD>   Hold </TD><TD align="right">4°C </TD></TR></TBODY></TABLE><P><B>Summary</B>: Four site-directed mutagenesis have been performed with the plasmid pAAV_RC_PstI_310+4073 (pAAV_RC_1.1) = P87. 
</P><UL><LI>Site-directed mutagenesis in order to delete restriction site BamHI
</LI><LI>with shorter elongation time (modified protocol)
</LI><LI>with longer elongation time
</LI></UL><LI>Site-directed mutagenesis in order to delete restriction site SalI
</LI><LI>with shorter elongation time (modified protocol)
</LI><LI>with longer elongation time
</LI><LI>The samples  were digested with 0,5µL DpnI after site-directed mutagenesis were performed in order to remove bacterial DNA template.
</LI><LI>Trafos have been performed with all four samples. Trafo plates are containing ampicillin.
</LI><H4><SPAN class="mw-headline" id="Cell_culture_2"><P style="font-size:15px; background-color:#66bbff;"><B>Cell culture</B></P></SPAN></H4><P><B>Investigator: Adrian</B></P><UL><LI> 3 x 6er dishes for Transduction were seeded and passaged(P14)
</LI></UL><H4><SPAN class="mw-headline" id="ITRs:_Trafo_of_overnight-ligation_approaches"><P style="font-size:15px; background-color:#66bbff;"><B>ITRs: Trafo of overnight-ligation approaches</B></P></SPAN></H4><P><B>Investigator: Hanna</B></P><P style="font-size:13px; color:#Fuchsia;"><B>Comments:</B> Cloning approach was continued: RFC10-Präfix_rightITR was ligated to RFC10-Suffix and pGA14 and leftITR was ligated to RFC10-Präfix and pGA14 over night. Trafo of these samples was performed in order to pick clones tomorrow.   </P><P>
Trafo was performed following the standard protocol. Abberance: 4 µL DNA was added to BL21 cells. </P><H4><SPAN class="mw-headline" id="PCR:_beta-Globin"><P style="font-size:15px; background-color:#66bbff;"><B>PCR: beta-Globin</B></P></SPAN></H4><P><B>Investigator: Hanna</B>
Because sequencing delivered, that cloning of the beta-globin intron into pSB1C3 didn't work (Cerulean was still in the vector), the PCR was performed one more time: </P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume [µl]
</TD></TR><TR><TD> 5x Phusion HF buffer  </TD><TD align="right"> 10
</TD></TR><TR><TD> 10 mM dNTP mix </TD><TD align="right"> 1
</TD></TR><TR><TD> beta-globin_primer_for</TD><TD align="right"> 2,5
</TD></TR><TR><TD> beta-globin_primer_rev</TD><TD align="right"> 2,5
</TD></TR><TR><TD>DNA template</TD><TD align="right"> 1 ng = 2.8 µL of 1:1000 dilution of pAAV_MCS, c=360 ng/µL
</TD></TR><TR><TD>DMSO</TD><TD align="right"> -
</TD></TR><TR><TD>Phusion polymerase</TD><TD align="right"> 0,5
</TD></TR><TR><TD>H2O</TD><TD align="right"> 30.7
</TD></TR><TR><TD><B>Total volume (e.g. 50 µl)</B></TD><TD align="right"> 50
</TD></TR></TBODY></TABLE><P><B>PCR program:</B></P><UL><LI> 98°C, 1'
</LI></UL><UL><LI> 98°C, 30
</LI><LI> 63°C, 25
</LI><LI> 72°C, 15
</LI></UL><P>-&gt; 8x </P><UL><LI> 98°C, 30
</LI><LI> 72°C, 20
</LI></UL><P>-&gt; 20x </P><UL><LI> 1 x 72°C, 5'
</LI><LI> Hold: 4°C
</LI></UL><H4><SPAN class="mw-headline" id="Repetition_of_PCR_for_hGH_BioBrick_Production"><P style="font-size:15px; background-color:#66bbff;"><B>Repetition of PCR for hGH BioBrick Production</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="font-size:13px; color:#25bbAA;"><B>Comments:</B> Sequencing of pSB1C3_hGH did not reveal any insertion of hGH into pSB1C3. Instead of hGH CFP was found in the vector indicating that undigested or partially digested vector religated. Therefore a new pCR for generating the BioBrick was performed over night.  </P><P>PCR reaction:
</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume [µl]
</TD></TR><TR><TD> 5x Phusion HF buffer  </TD><TD align="right"> 10
</TD></TR><TR><TD> 10 mM dNTP mix </TD><TD align="right"> 1
</TD></TR><TR><TD> hGH_primer_for</TD><TD align="right"> 2,5
</TD></TR><TR><TD> hGH-globin_primer_rev</TD><TD align="right"> 2,5
</TD></TR><TR><TD>DNA template</TD><TD align="right"> 1 ng = 2.8 µL of 1:1000 dilution of pAAV_MCS, c=360 ng/µL
</TD></TR><TR><TD>DMSO</TD><TD align="right"> -
</TD></TR><TR><TD>Phusion polymerase</TD><TD align="right"> 1
</TD></TR><TR><TD>H2O</TD><TD align="right"> 30.7
</TD></TR><TR><TD><B>Total volume (e.g. 50 µl)</B></TD><TD align="right"> 50
</TD></TR></TBODY></TABLE><P style="font-size:13px; color:#25bbAA;"><B>Used PCR programs</B>: </P><P><B>Comment:</B> PCR will be performed over night.</P><TABLE border="1"><TBODY><TR><TD><B>Rounds</B></TD><TD align="right"><B>Temperature</B></TD><TD align="right"><B>Time</B></TD></TR><TR><TD>1 </TD><TD align="right">98°C </TD><TD align="right">1'
</TD></TR><TR><TD>8 </TD><TD align="right">98°C </TD><TD align="right">30
</TD></TR><TR><TD align="right">58°C </TD><TD align="right">25
</TD></TR><TR><TD align="right">72°C </TD><TD align="right">12
</TD></TR><TR><TD>17 </TD><TD align="right">98°C </TD><TD align="right">30
</TD></TR><TR><TD align="right">69°C </TD><TD align="right">25
</TD></TR><TR><TD align="right">72°C </TD><TD align="right">12
</TD></TR><TR><TD>   Hold </TD><TD align="right">4°C </TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Picking_clones_from_pAAV_RFC25_mGMK"><P style="font-size:15px; background-color:#66bbff;"><B>Picking clones from pAAV_RFC25_mGMK </B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="font-size:13px; color:#25bbAA;"><B>Comment:</B>  Trafo plate looked good. On control plate only few clones grown in contrary to the XL1-B{pAAV_RFC25-mGMK} plate. Two clones were picked.</P><UL><LI>10 mL containing 10µL  ampiciliin were inoculated.
</LI><LI>Put into 37°C room on shaker.
</LI></UL><H4><SPAN class="mw-headline" id="Picking_clones_from_Retrafo_of_pTV_CMV_clone_7_and_clone_8"><P style="font-size:15px; background-color:#66bbff;"><B>Picking clones from Retrafo of pTV_CMV clone 7 and clone 8 </B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="font-size:13px; color:#25bbAA;"><B>Comment:</B>  Trafo plates looked good. Enough clones grown. Two clones of each plate were picked</P><UL><LI>10 mL containing 10µL  ampiciliin were inoculated with each clone respectively.
</LI><LI>Put into 37°C room on shaker.
</LI></UL><H3><SPAN class="mw-headline" id="72._Labortag_28.7.2010"><P style="font-size:17px; background-color:#00dd77;">72. Labortag 28.7.2010 </P></SPAN></H3><H4><SPAN class="mw-headline" id="Results_Trafo_pAAV_RC_1.1_BamHI_and_pAAV_RC_1.1_SalI"><P style="font-size:15px; background-color:#66bbff;"><B>Results Trafo pAAV_RC_1.1_BamHI and pAAV_RC_1.1_SalI</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="font-size:13px; color:#25bbAA;"><B>Results:</B> Too many bacteria grown resulting in a bacterial lawn. Possible reasons could be some problems with the new prepared ampicillin because trafo from Hanna containing the ITRs had the same results. Trouble shooting done by Hanna. </P><H4><SPAN class="mw-headline" id="Proceed_with_BioBrick_Production_of_hGH_and_beta-globin"><P style="font-size:15px; background-color:#66bbff;"><B>Proceed with BioBrick Production of hGH and beta-globin</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="font-size:13px; color:#25bbAA;"><B>Comments:</B> PCR was started yesterday and conducted over night.</P><P><B>Proceeding:</B></P><UL><LI>Prepare 1% agarose gel and load samples on gel. 
</LI><LI>Run time: 45 minutes
</LI></UL><P style="font-size:15px; color:#25bbAA;"><B>Result of gel:</B></P><P>Marker did not resolute very good. Expected sizes of PCR products:
</P><UL><LI>hGH = 510 bp
</LI><LI>beta globin intron: 524 bp
</LI></UL><P>It can be seen (or estimated) that two intensive bands are visible at around 500 bp. Therefore these two bands were cut out and a gel extraction was performed. The lower visible band at around 100 bp of hGH is an additional band generated at the pCR reaction. The two intensive blurred bands belong to the loading dye added to the samples.
</P><P>
After gel extraction was performed, the PCR prdoucts hGH and beta-globin were digested:
</P><P style="font-size:15px; color:#25bbAA;">DIGESTION of PCR products</P><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>hGH/µl</B></TD><TD align="right"><B>beta-globin /µL</B></TD></TR><TR><TD> pSB1C3_CFP  </TD><TD align="right">29 </TD><TD align="right">29
</TD></TR><TR><TD> BSA (100x) </TD><TD align="right"> 0,4</TD><TD align="right"> 0,4
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">4</TD><TD align="right">4
</TD></TR><TR><TD>XbaI </TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>PstI </TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">4,6</TD><TD align="right">4,6
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"><B>40</B></TD><TD align="right"><B>40</B></TD></TR></TBODY></TABLE><UL><LI>Incubate for 2h at 37°C
</LI><LI>Perform PCR purification following standard protocols
</LI><LI>Elute in 20µL EB-buffer, put on 50°C for 3 minutes before centrifugating
</LI><LI>Measure DNA with NanoDrop:
</LI><LI>c(hGH) = 58,14 ng/µL
</LI><LI>c(ß-globin) = 83,10ng/µL
</LI></UL><P>
In the meantime prepare the <B>digestion of vector pSB1C3_CFP</B> (P51)
</P><P style="font-size:15px; color:#25bbAA;">DIGESTION of pSB1C3_CFP</P><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume /µl</B></TD></TR><TR><TD> pSB1C3_CFP  </TD><TD align="right">5
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2
</TD></TR><TR><TD>XbaI </TD><TD align="right">1
</TD></TR><TR><TD>PstI </TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">9
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"><B>20</B></TD></TR></TBODY></TABLE><UL><LI>Incubate for 2h at 37°C
</LI><LI>Load sample on 1% preparative agarose gel 
</LI><LI>Run for 45 minutes at 110 V
</LI></UL><P style="font-size:15px; color:#25bbAA;"><B>Result of gel:</B></P><P>The vector was digested with XbaI and PstI. Expected sizes have been:
</P><UL><LI>pSB1C3 = 2072bp
</LI><LI>CFP: 732bp
</LI></UL><P>It can be seen that the vector pSB1C3_RFC25_CFP was digested and two visible bands can be seen on the gel. The lower band corresponds to the CFP fragment (~700 - 800 bp), the higher band corresponds to the plasmid backbone at around 2000-3000 bp. The higher band was cut out and extracted by gel extraction. 
</P><UL><LI>Perform gel extraction and measure DNA with NanoDrop
</LI><LI>c(psb1C3) = 29,52 ng/µL
</LI></UL><P style="font-size:15px; color:#25bbAA;"><B>LIGATION of pSB1C3 with hGH and beta-globin</B></P><P>Calculated ratio of pSB1C3 (vector) and hgh(insert) or betaglobin (insert)
</P><P>
The ligated samples
</P><UL><LI>pSB1C3_hGH
</LI><LI>pSB1C3_betaglobin
</LI></UL><P>were transformed with XL1B and incubated on agar-plates containing chloramphenicol over night at 37°C.
</P><H4><SPAN class="mw-headline" id="Results_of_ITR_Trafo"><P style="font-size:15px; background-color:#66bbff;">Results of ITR Trafo</P></SPAN></H4><P><B>Investigator: Hanna</B></P><P>Because (as mentioned above) there was a bacteria lawn on the right-ITR plate and many colonies on the left-ITR plate three different approaches were performed: </P><P><B>1. Trafo</B> of rightITR-RFC25_pGA14 and leftITR-Präfix_pGA14 was conducted one more time:</P><P>Abberances: </P><UL><LI> 3µL DNA-sample was added to 50 µL cells (BL21).
</LI><LI> Cells were incubated on ice for just 10 minutes.
</LI><LI> Heat shock: 42 seconds at 42°C.
</LI><LI> Incubation for just 35 minutes at 37°C, rotating (800 rpm).
</LI><LI> Agar plates were prepared with Lab-Ampicillin.
</LI></UL><P>Incubation at 37°C -&gt; Picking clones in the evening. <B>2. Control:</B> Bacteria were taken from the right-ITR-lawn an resuspended in 100 µL DYT. They were plated on &quot;LabAmp-plates&quot; and stored at 37°C. 
-&gt; If something is wrong with our Ampicillin the non-transformed bacteria have progeny-advantages and will crowd out the transformed cells. This means: If there will be no colonies detectable on the new plates, something is wrong with our Ampicillin!  
Analysis: Tonight.<B> 3. Picking clones of the left ITR plate:</B> Two large and two small clones were picked from the left ITR plate and each was resuspened in 50 µL DYT. An &quot;LabAmp-plate&quot; was quartered and the four resuspened colonies were individualized each on one quarte of the plate. 
Analysis: Tonight.</P><H4><SPAN class="mw-headline" id="Trafo_evaluation_Rep_68.2678_Pst"><P style="font-size:15px; background-color:#66bbff;"><B>Trafo evaluation Rep 68&amp;78 Pst</B></P></SPAN></H4><P><B>Investigator: Kira</B></P><P>Both plates Rep 68 &amp; Rep 78 contain lots of colonies. Approx. 4 colonies from each plate will be picked and inoculated for Mini prep tomorrow.
</P><H4><SPAN class="mw-headline" id="Cellculture_3"><P style="font-size:15px; background-color:#66bbff;"><B>Cellculture</B></P></SPAN></H4><P><B>Investigator: Jessica</B>
HEK were split according to the standard protocol in 5 flasks</P><H4><SPAN class="mw-headline" id="Mini-Prep_and_Test_digestion_of_pAAV_RFC25_mgmk_and_pTV_CMV_7_and_pTV_CMV_8"><P style="font-size:15px; background-color:#66bbff;">Mini-Prep and Test digestion of pAAV_RFC25_mgmk and pTV_CMV_7 and pTV_CMV_8</P></SPAN></H4><P><B>Investigator: Stefan</B></P><P style="font-size:13px; color:#68bbff;"><I><B>Comments:</B></I>
mgmk was brought into iGEM standard vector. To verify the construct it will be sent for sequencing.
pTV_CMV_7 and pTV_CMV_8 are clones we recieved from the iGEM 2008 team. Because there are mistakes in the prefix and suffix a PCR will have to be done. They will be sent for sequencing as well. </P><P>Mini-Prep of pAAV_RFC25_mgmk clone 1 and 2:
After performing Mini-Preps concentration was measured with Nanodrop:
</P><UL><LI>pAAV_RFC25_mgmk clone 1: c = 326,0 ng/µl
</LI><LI>pAAV_RFC25_mgmk clone 2: c = 411,3 ng/µl
</LI></UL><P>Mini-Prep of pTV_CMV_7 clone 1 and 2:
After performing Mini-Preps concentration was measured with Nanodrop:
</P><UL><LI>pTV_CMV_7 clone 1: c = 378,2 ng/µl
</LI><LI>pTV_CMV_7 clone 2: c = 412,9 ng/µl
</LI></UL><P>Mini-Prep of pTV_CMV_8 clone 1 and 2:
After performing Mini-Preps concentration was measured with Nanodrop:
</P><UL><LI>pTV_CMV_8 clone 1: c = 337,6 ng/µl
</LI><LI>pTV_CMV_8 clone 2: c = 366,6 ng/µl
</LI></UL><P><B>Test digestion:</B>
pAAV_RFC25_mgmk was digested with XbaI and AgeI.
pTV_CMV_7 and pTV_CMV_8 were digested with XbaI and PstI.
</P><UL><LI>P99 = pAAV_RFC25_mgmk clone 1
</LI><LI>P100 = pAAV_RFC25_mgmk clone 2
</LI><LI>P101 = pTV_CMV_7 clone 1
</LI><LI>P102 = pTV_CMV_7 clone 2
</LI><LI>P103 = pTV_CMV_8 clone 1
</LI><LI>P104 = pTV_CMV_8 clone 2
</LI></UL><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of <B>P99</B>/µl </TD><TD>volume of <B>P100</B>/µL</TD><TD>volume of <B>P101/</B>µL </TD><TD>volume of <B>P102</B>/µl </TD><TD>volume of <B>P103</B>/µL</TD><TD>volume of <B>P104/</B>µL
</TD></TR><TR><TD> DNA  </TD><TD>2,5 </TD><TD>2 </TD><TD>2,5 </TD><TD>2 </TD><TD>2,5 </TD><TD>2,5
</TD></TR><TR><TD> BSA (10x) </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5
</TD></TR><TR><TD>XbaI </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5
</TD></TR><TR><TD>AgeI </TD><TD>0,5 </TD><TD>0,5 </TD><TD>/ </TD><TD>0/ </TD><TD>/ </TD><TD>/
</TD></TR><TR><TD>PstI </TD><TD>/ </TD><TD>/ </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5
</TD></TR><TR><TD>H2O</TD><TD>8,5 </TD><TD>9 </TD><TD>9 </TD><TD>8,5 </TD><TD>8,5 </TD><TD>8,5
</TD></TR><TR><TD><B>Total volume /µl</B></TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15
</TD></TR></TBODY></TABLE><P><B>Expected sizes:</B></P><UL><LI>mgmk:  603bp
</LI><LI>pAAV_RFC25: 4636bp</LI></UL><UL><LI>CMV:  679bp
</LI><LI>pTV: 3408bp
</LI></UL><P style="font-size:13px; color:#11bb88;"><B>Results of samples loaded 1% agarose gel and ran ~50 minutes at 115 V:</B></P><P style="font-size:13px; color:#11bb56;"><B>Sequencing</B></P><P>70ng/µL were sent to GATC from each clone. They were labeled:
</P><UL><LI>SB 1 (pAAV_RFC25_mgmk clone 2 (P100))
</LI><LI>SB 2 (pTV_CMV_7 clone 1 (P101))
</LI><LI>SB 3 (pTV_CMV_8 clone 1 (P103))</LI></UL><P>For SB 1 there was sent the GOI_forward primer (O30) as well.
</P><H4><SPAN class="mw-headline" id="Site-directed_mutagenesis_of_SspI_and_PvuII_in_pSB1C3_CFP"><P style="font-size:15px; background-color:#66bbff;"><B>Site-directed mutagenesis of SspI and PvuII in pSB1C3_CFP </B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><P style="font-size:13px; color:#66bbff;"><B>Comments:</B> Site-directed mutagenesis was perfomed with pSB1C3_CFP in order to delete SspI and PvuII in the plasmid. These restriciton sites need to be removed because they are also in the loop we want to modify.  </P><P style="font-size:13px; color:#25bbAA;"><B>PCR reaction with PCR program with SspI and PvuII</B>: </P><P>DNA template used: ppSB1C3_CFP = P51</P><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume in µl</B></TD><TD align="right"><B>volume in µl</B>|
</TD></TR><TR><TD> 10x PfuUltra II buffer  </TD><TD align="right"> 2.5 µl</TD><TD align="right"> 2.5 µl
</TD></TR><TR><TD> 10 mM dNTP mix </TD><TD align="right"> 0.5 µl</TD><TD align="right"> 0.5 µl
</TD></TR><TR><TD>  primer_for (1:10 dilution)</TD><TD align="right"><B>O107= SDM SspI: 0,55 µl</B></TD><TD align="right"><B> O109= SDM PvuII: 0,58 µl</B></TD></TR><TR><TD>  primer_rev (1:10 dilution)</TD><TD align="right"><B>O106= SDM SspI: 0,55 µl</B></TD><TD align="right"><B>O108= SDM PvuII: 0,59 µl</B></TD></TR><TR><TD><B>P51</B> (DNA template) )</TD><TD align="right">  0,43 µl</TD><TD align="right">  0,43 µl
</TD></TR><TR><TD>DMSO</TD><TD align="right">  -</TD><TD align="right">  0,5µl
</TD></TR><TR><TD>PfuUltra II Fusion</TD><TD align="right">  0.5 µl</TD><TD align="right">  0.5 µl
</TD></TR><TR><TD>H2O</TD><TD align="right"> 16,1 µl</TD><TD align="right"> 15,53 µl
</TD></TR><TR><TD><B>Total volume (e.g. 50 µl)</B></TD><TD align="right"> 25 µl</TD><TD align="right"> 25 µl
</TD></TR></TBODY></TABLE><P style="font-size:13px; color:#25bbAA;"><B>Used PCR programs for  SDMs</B>: </P><P><B>1) PvuII</B></P><TABLE border="1"><TBODY><TR><TD><B>Rounds</B></TD><TD align="right"><B>Temperature</B></TD><TD align="right"><B>Time</B></TD></TR><TR><TD>1 </TD><TD align="right">95°C </TD><TD align="right">2'
</TD></TR><TR><TD>20 </TD><TD align="right">95°C </TD><TD align="right">30
</TD></TR><TR><TD align="right">78°C </TD><TD align="right">1'
</TD></TR><TR><TD align="right">68°C </TD><TD align="right">2 min 50 sec
</TD></TR><TR><TD>   Hold </TD><TD align="right">4°C </TD></TR></TBODY></TABLE><P><B>2) SspI</B></P><TABLE border="1"><TBODY><TR><TD><B>Rounds</B></TD><TD align="right"><B>Temperature</B></TD><TD align="right"><B>Time</B></TD></TR><TR><TD>1 </TD><TD align="right">95°C </TD><TD align="right">2'
</TD></TR><TR><TD>20 </TD><TD align="right">95°C </TD><TD align="right">30
</TD></TR><TR><TD align="right">79°C </TD><TD align="right">1'
</TD></TR><TR><TD align="right">68°C </TD><TD align="right">2 min 40 sec
</TD></TR><TR><TD>   Hold </TD><TD align="right">4°C </TD></TR></TBODY></TABLE><UL><LI> Transformation was prepared according to the standard protocol
</LI></UL><H4><SPAN class="mw-headline" id="Affibody"><P style="font-size:15px; background-color:#66bbff;"><B>Affibody</B></P></SPAN></H4><P><B>Investigator: Hanna</B>
In order to target the EGF-receptor (EGFR/ErbB-1/Her1) an affibody molecule can be fused to the N-terminus of e.g. VP2, which is expressed in trans. The ZEGFR:1907 was chosen for this approach. Because just the amino acid sequence is published by Friedman et al., 2008, the sequence was translated with Mr. Gene's help :) and optimized for mammalian expression. <B>Strategy:</B></P><H4><SPAN class="mw-headline" id="Picking_clones_from_Trafo_pSB1C3_RFC25_longlinker.2C_pSB1C3_RFC25_GSAT.2C_pSB1C3_RFC25_SEG"><P style="font-size:15px; background-color:#66bbff;"><B>Picking clones from Trafo pSB1C3_RFC25_longlinker, pSB1C3_RFC25_GSAT, pSB1C3_RFC25_SEG </B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><P style="font-size:13px; color:#25bbAA;"><B>Comment:</B>  Trafo plates looked good. Enough clones grown. Two clones of each plate were picked</P><UL><LI>10 mL containing 10µL  chloramphenicol were inoculated with each clone respectively.
</LI><LI>Put into 37°C room on shaker.
</LI></UL><H4><SPAN class="mw-headline" id="Preparation_for_competent_XL1b"><P style="font-size:15px; background-color:#66bbff;"><B>Preparation for competent XL1b </B></P></SPAN></H4><P><B>Investigator: Jessica</B>
15 ml DYT was prepared with tetracyclin and XL1b from glycerolstock and incubates over night. goes on on thursday.
</P><H4><SPAN class="mw-headline" id="ReTrafo_of_pGL3_hTERT"><P style="font-size:15px; background-color:#66bbff;"><B>ReTrafo of pGL3_hTERT</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="font-size:13px; color:#25bbAA;"><B>Comment:</B> pGL3_hTERT received from Kira. Contains ampiciliin restitance. Retrafo was performed in order to obtain enough DNA to conduct PCR for add-on tails.</P><P style="font-size:14px;"> Re-transformation: </P><UL><LI>Construct used: pGL3_hTert
</LI><LI>concentration c = 276ng/µL
</LI><LI>Plasmid need to be diluted in order to obatain the final 100pg. </LI><LI>Dilution: 1:1000
</LI><LI>Volume needed of 1:1000 dilution of pGL3_hTert:
</LI><LI>v = 0,37µL
</LI></UL><LI>100 µL aliquot of <B>XL1-B</B> were thawed on ice and the plasmid DNA was added.
</LI><H4><SPAN class="mw-headline" id="Trafo_of_PAAV_RC_1.1_SDM_of_BamHI_and_SalI"><P style="font-size:15px; background-color:#66bbff;"><B>Trafo of PAAV_RC_1.1 SDM of BamHI and SalI</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="font-size:13px; color:#25bbAA;"><B>Comment:</B> Trafo did not work (see above) therefore th etRafo have been performed again</P><H4><SPAN class="mw-headline" id="Picking_clones_from_the_2._Trafo_of_left_and_right_ITR_.28Hanna.29"><P style="font-size:15px; background-color:#66bbff;"><B>Picking clones from the 2. Trafo of left and right ITR (Hanna)</B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><P style="font-size:13px; color:#25bbAA;"><B>Comment:</B>  Trafo plates looked good. Enough clones grown. Four clones of each plate were picked</P><UL><LI>10 mL containing 10µL  Amp were inoculated with each clone respectively.
</LI><LI>Put into 37°C room on shaker.
</LI></UL><P style="font-size:13px; color:#25bbAA;"><B>Comment @Hanna:</B> also on control plate are clones grown... not on the quartered plate. back in the 37°C room</P><H3><SPAN class="mw-headline" id="73._Labortag_29.7.2010"><P style="font-size:17px; background-color:#00dd77;">73. Labortag 29.7.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Mini-Prep_and_Test_digestion_of_Rep_68_and_Rep_78"><P style="font-size:15px; background-color:#66bbff;"><B>Mini-Prep and Test digestion of Rep 68 and Rep 78 </B></P></SPAN></H4><P><B>Investigator: Kira</B></P><P>3 clones from each plate were picked and inoculated. The mini prep DNA extraction revealed followed DNA concentrations:
</P><P>Rep 68
clone 1: 470 ng/μl
clone 2: 556,15 ng/μl
clone 3: 686,40 ng/μl
</P><P>Rep 78
clone 1: 566,78 ng/μl
clone 2: 519,68 ng/μl
clone 3: 521,88 ng/μl
</P><P>All 6 samples were test digested with PstI to check if the site directed mutagenesis was successful. 
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>Volume/µL</B></TD><TD align="left"><B>Mastermix</B></TD></TR><TR><TD align="left"> DNA </TD><TD align="left"> Variable (800 ng) </TD><TD align="left"> -
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 1 </TD><TD align="left"> 6
</TD></TR><TR><TD align="left"> Buffer no. 4   (10x) </TD><TD align="left"> 1 </TD><TD align="left">6
</TD></TR><TR><TD align="left"> Enzyme 1 ( PstI    ) </TD><TD align="left"> 0,5 </TD><TD align="left"> 3
</TD></TR><TR><TD align="left"> H2O </TD><TD align="left"> variable </TD><TD align="left"> -
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>10</B></TD><TD align="left"> -
</TD></TR></TBODY></TABLE><UL><LI>Incubation: 1h @ 37 C
</LI></UL><P>1% agarose gel 
</P><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker /µl
</TH><TH>Sample Rep 68 clone 1 /µl
</TH><TH>Sample Rep 68 clone 2 /µl
</TH><TH>Sample Rep 68 clone 3 /µl
</TH><TH>Sample Rep 78 clone 1 /µl
</TH><TH>Sample Rep 78 clone 2 /µl
</TH><TH>Sample Rep 78 clone 3 /µl
</TH></TR><TR><TH>Lane
</TH><TD>7
</TD><TD>12
</TD><TD>12
</TD><TD>12
</TD><TD>12
</TD><TD>12
</TD><TD>12
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Results_sequencing_of_pTV_CM_clone_7_and_8_.28received_from_iGEM_team_2008.29"><P style="font-size:15px; background-color:#66bbff;"><B>Results sequencing of pTV_CM clone 7 and 8 (received from iGEM team 2008)</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="font-size:13px; color:#25bbAA;"><B>Comment:</B>  Sequencing results obtained from the Mini-Prep performed yesterday from the constructs pTV_CMV_7 and pTV_CMV_8 received of the iGEM Freiburg team 2008. A Retrafo has been performed and obtained plasmids of this retrafo have been send for sequencing. </P><P><B>Results:</B> Seqeuncing reads look good. Plasmids contain expected CMV promoter which is already in the iGEM RFC10 standard but with one missing &quot;G&quot; in the prefix. 
</P><P style="font-size:16px;">Results prefix RFC10:</P><P style="font-size:16px;">Results pTV_CMV_7:</P><P style="font-size:16px;">Results pTV_CMV_8:</P><H4><SPAN class="mw-headline" id="Sequencing_results_of_pAAV_RFC25_mGMK"><P style="font-size:15px; background-color:#66bbff;"><B>Sequencing results of pAAV_RFC25_mGMK</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="font-size:13px; color:#25bbAA;"><B>Comment:</B>  Sequencing results of pAAV_RFC25_mGMK. A PCR has been perfomed in order to obtain the desired mGMK construct in the RFC25 standard. After Mini-Prep has been perfomed clone 1 has been sent to GATC for sequencing.</P><P><B>Conclusion:</B> The PCR amplification with designed primers to insert prefix and suffix in RFC25 standard can be verified. No nucleotide transitions can be detected. The prefix and suffix are in the designed standards. Additionally it can be confirmed that the mGMK was inserted successfully into the pAAV_RFC25 plasmid and do NOT contain any iGEM restriction sites When receiving the ordered tk30 from Mr.Gene it can be fused to the mGMK and will the be ready for using and sending to the IGEM registry (maybe this needed to be cloned into pSB1C3).
</P><H4><SPAN class="mw-headline" id="Stock_solution_of_Ampicilin"><P style="font-size:15px; background-color:#66bbff;"><B>Stock solution of Ampicilin</B></P></SPAN></H4><P><B>Investigators: Johannes, Stefan</B></P><P>0,3g Ampicilin were dissolved in 3ml Millipore H20. PCR tubes were filled with 60 µl each and stored at -20 °C.
</P><H4><SPAN class="mw-headline" id="Mini-Prep_and_Test_digestion_of_ITR_constructs"><P style="font-size:15px; background-color:#66bbff;">Mini-Prep and Test digestion of ITR constructs</P></SPAN></H4><P><B>Investigator: Hanna</B>
The RFC10-Prefix was fused to the <B>left ITR</B> and cloned into the pGA14_Cerulean plasmid. 4 clones of the second trafo (the first trafo didn't work due to wrong preparation of the Ampicillin stock solution, see 28.7.2010) were picked yesterday. Mini-Prep was performed today. 
Plasmid-no.: (pGA14_10Prefix_lITR) 
P111: 321.91 ng/µL 
P112: 489.96 ng/µL
P113: 341.49 ng/µL
P114: 402.3 ng/µL
Glycerol stock were prepared: B85.1 - B85.1.
</P><P>The &quot;Intermediate-Step-Suffix&quot; was fused to the <B>RFC10-Prefix-rITR</B> construct and cloned into pGA14. 4 clones of the second trafo were picked yesterday. Mini-Prep was performed today. 
Plasmid-no.: (pGA14_10Prefix_PreSuffix_rITR) 
P115: 401.07 ng/µL
P116: 291.7 ng/µL
P117: 312.98 ng/µL
P118: 326.78 ng/µL
Glycerol stock were prepared: B84.1 - B84.1.
</P><P style="font-size:15px; font-weight: bold; color: blue;">Test digestion</P><UL><LI>buffer used: 4 ; Restriction-enzymes used: Enzyme NotI-HF
</LI><LI>Plasmid: P111 - P118
</LI></UL><P>
Pipetting scheme for for each digestion:
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>Volume/µL</B></TD></TR><TR><TD align="left"> DNA </TD><TD align="left"> 4 µL
</TD></TR><TR><TD align="left"> BSA (100x) </TD><TD align="left"> 0.1
</TD></TR><TR><TD align="left"> Buffer no.4 (10x) </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Enzyme NotI-HF </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> H2O </TD><TD align="left"> 3.9
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>10</B></TD></TR></TBODY></TABLE><UL><LI>Incubation: 55 minutes
</LI></UL><P style="font-size:15px; font-weight: bold; color: blue;">Agarose-Gel:</P><P>
0.8 g Agarose, 55 mL TAE (1.45 %), 3 µL GELRED, at 115 Volt, running time: 35 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (5x)/µl
</TH><TH>Expected size 1 (Geneious)
</TH><TH>Expected size 2 (Geneious)
</TH></TR><TR><TD>pGA14_Prefix_lITR clone 1
</TD><TD>10 µl
</TD><TD>2 µl
</TD><TD>~ 150 bp
</TD><TD>~ 2900 bp
</TD></TR><TR><TD>pGA14_Prefix_lITR clone 2
</TD><TD>10 µl
</TD><TD>2 µl
</TD><TD>~ 150 bp
</TD><TD>~ 2900 bp
</TD></TR><TR><TD>pGA14_Prefix_lITR clone 3
</TD><TD>10 µl
</TD><TD>2 µl
</TD><TD>~ 150 bp
</TD><TD>~ 2900 bp
</TD></TR><TR><TD>pGA14_Prefix_lITR clone 4
</TD><TD>10 µl
</TD><TD>2 µl
</TD><TD>~ 150 bp
</TD><TD>~ 2900 bp
</TD></TR><TR><TD>pGA14_Prefix_PreSuffix_rITR clone 1
</TD><TD>10 µl
</TD><TD>2 µl
</TD><TD>145 bp
</TD><TD>~ 2900 bp
</TD></TR><TR><TD>pGA14_Prefix_PreSuffix_rITR clone 2
</TD><TD>10 µl
</TD><TD>2 µl
</TD><TD>145 bp
</TD><TD>~ 2900 bp
</TD></TR><TR><TD>pGA14_Prefix_PreSuffix_rITR clone 3
</TD><TD>10 µl
</TD><TD>2 µl
</TD><TD>145 bp
</TD><TD>~ 2900 bp
</TD></TR><TR><TD>pGA14_Prefix_PreSuffix_rITR clone 4
</TD><TD>10 µl
</TD><TD>2 µl
</TD><TD>145 bp
</TD><TD>~ 2900 bp
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker /µl
</TH><TH>Sample pGA14_Prefix_lITR clone 1 /µl
</TH><TH>Sample pGA14_Prefix_lITR clone 2 /µl
</TH><TH>Sample pGA14_Prefix_lITR clone 3 /µl
</TH><TH>Sample pGA14_Prefix_lITR clone 4 /µl
</TH><TH>Sample pGA14_Prefix_PreSuffix_rITR clone 1 /µl
</TH><TH>Sample pGA14_Prefix_PreSuffix_rITR clone 2 /µl
</TH><TH>Sample pGA14_Prefix_PreSuffix_rITR clone 3 /µl
</TH><TH>Sample pGA14_Prefix_PreSuffix_rITR clone 4 /µl
</TH></TR><TR><TH>Lane
</TH><TD>6
</TD><TD>12
</TD><TD>12
</TD><TD>12
</TD><TD>12
</TD><TD>12
</TD><TD>12
</TD><TD>12
</TD><TD>12
</TD></TR></TBODY></TABLE><P><B>Comments:</B>Because samples can be sent to GATC just until 7 p.m., gel run was checked already after 15 minutes:
</P><P>
No bands can be detected at the expected sizes. The fragment size seems to be inbetween the expectation of either 150 and 750 bp. Further on there is also an additional band visible. Because of that it's not possible to say, whether cloning worked. Therefore the three clones marked in the picture were sent to GATC for sequencing: </P><UL><LI> pGA14_Prefix_lITR clone 1: 6.5 µL plasmid + 23.5 µL H2O = &quot;Hanna 1&quot;
</LI><LI> pGA14_Prefix_lITR clone 4: 5.2 µL  plasmid + 24.8 µL H2O = &quot;Hanna 2&quot;
</LI><LI> pGA14_Prefix_PreSuffix_rITR clone 1: 5.2 µL  plasmid + 24.8 µL H2O = &quot;Hanna 3&quot;
</LI></UL><H4><SPAN class="mw-headline" id="Picking_clones_of_pGL3_hTERT"><P style="font-size:15px; background-color:#66bbff;">Picking clones of pGL3_hTERT</P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="font-size:13px; color:#25bbAA;"><B>Comments:</B> Clones will be picked from trafo plate in order to obatin single colonies in order to perform Mini-prep tomorrow.</P><UL><LI>Bacterial strain used: XL1-B
</LI><LI>Two clones were picked of trafo plate
</LI><LI>Inoculating of 10 mL DYT medium containing 10µL ampicillin
</LI></UL><H4><SPAN class="mw-headline" id="Picking_clones_of_pAAV_RC_1.1_SDM_of_BamHI_and_SalI"><P style="font-size:15px; background-color:#66bbff;">Picking clones of pAAV_RC_1.1 SDM of BamHI and SalI</P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="font-size:13px; color:#25bbAA;"><B>Comments:</B> All trafo plates looked good, enough clones grew, aside from trafo plate <B>XL1-B {pAAV_RC_1.1_SalI}</B>: long PCR, where no clones grew on plate. Clones will be picked from each trafo plate in order to obatin single colonies in order to perform Mini-Prep tomorrow. </P><UL><LI>Bacterial strain used: XL1-B
</LI><LI>Two clones were picked of each trafo plate
</LI><LI><B>XL1-B {pAAV_RC_1.1_BamHI}</B>: long PCR
</LI><LI><B>XL1-B {pAAV_RC_1.1_BamHI}</B>: short PCR
</LI><LI><B>XL1-B {pAAV_RC_1.1_SalI}</B>: short PCR
</LI></UL><LI>Inoculating of 10 mL DYT medium containing 10µL ampicillin
</LI><H4><SPAN class="mw-headline" id="Picking_clones_of_pSB1C3_hGH_and_pSB1C3_betaglobin"><P style="font-size:15px; background-color:#66bbff;">Picking clones of pSB1C3_hGH and pSB1C3_betaglobin</P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="font-size:13px; color:#25bbAA;"><B>Comments:</B> Clones will be picked from trafo plate in order to obatin single colonies in order to perform Mini-Prep tomorrow.</P><UL><LI>Bacterial strain used: XL1-B
</LI><LI>Two clones were picked of each trafo plate
</LI><LI><B>XL1-B {pSB1C3_hGH}</B></LI><LI><B>XL1-B {pSB1C3_betaglobin}</B></LI></UL><LI>Inoculating of 10 mL DYT medium containing 10µL chloramphenicol
</LI><H4><SPAN class="mw-headline" id="PCR_with_pTV_CMV_with_designed_oligos_for_amplifying_CMV_in_RFC10_standard"><P style="font-size:15px; background-color:#66bbff;">PCR with pTV_CMV with designed oligos for amplifying CMV in RFC10 standard</P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="font-size:13px; color:#25bbAA;"><B>Comments:</B> Retrafo of th epTV_CMv constructs has been performed and seqeuncing revealed that the plasmid contains the CMV promoter. Therefore, PCR in order to add prefix and suffix in RFC10 standard can be performed. </P><UL><LI>Plasmid used: pTV_CMV_7 clone 2 (P102)
</LI><LI>Plasmid has to be diluted to provide the possibilty of pipetting the volume.
</LI><LI>1:1000 dilution of 412ng/µL --&gt; c(new) = 0,412ng/µL
</LI></UL><LI>Primer used:
</LI><LI>pCMV_prefix_RFC10_for (1:10)
</LI><LI>pCMV_suffix_RFC10_rev (1:10)
</LI><P><B>PCR reaction:</B></P><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume /µl</B></TD></TR><TR><TD> 5x Phusion HF buffer  </TD><TD align="right"> 10
</TD></TR><TR><TD> 10 mM dNTP mix </TD><TD align="right"> 1
</TD></TR><TR><TD> pCMV_prefix_for_RFC10 (1:10)</TD><TD align="right"> 2,5
</TD></TR><TR><TD> pCMV_suffix_for_RFC10 (1:10)</TD><TD align="right"> 2,5
</TD></TR><TR><TD> pTV_CMV_7 clone 1 (P102)</TD><TD align="right"> 1.8 ng = 4,5 µL of 1:1000 dilution of P102, c=412 ng/µL
</TD></TR><TR><TD>DMSO</TD><TD align="right"> -
</TD></TR><TR><TD>Phusion polymerase</TD><TD align="right"> 0,5
</TD></TR><TR><TD>H2O</TD><TD align="right"> 29
</TD></TR><TR><TD><B>Total volume (e.g. 50 µl)</B></TD><TD align="right"> 50
</TD></TR></TBODY></TABLE><P><B>Used PCR programs</B>: </P><P style="font-size:13px; color:#25bbAA;"><B>Comment:</B> PCR will be performed over night. </P><TABLE border="1"><TBODY><TR><TD><B>Rounds</B></TD><TD align="right"><B>Temperature</B></TD><TD align="right"><B>Time</B></TD></TR><TR><TD>1 </TD><TD align="right">98°C </TD><TD align="right">1'
</TD></TR><TR><TD>30 </TD><TD align="right">98°C </TD><TD align="right">15
</TD></TR><TR><TD align="right">70°C </TD><TD align="right">25
</TD></TR><TR><TD align="right">72°C </TD><TD align="right">12
</TD></TR><TR><TD>   Hold </TD><TD align="right">4°C </TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Mini-Prep_and_Test_digestion_of_pSB1C3_RFC25_longlinker.2C_pSB1C3_RFC25_SEG_and_pSB1C3_RFC25_GSAT"><P style="font-size:15px; background-color:#66bbff;">Mini-Prep and Test digestion of pSB1C3_RFC25_longlinker, pSB1C3_RFC25_SEG and pSB1C3_RFC25_GSAT</P></SPAN></H4><P><B>Investigator: Jessica</B><B>Nanodrop</B></P><UL><LI> pSB1C3_RFC25_longlinker clone1: 187,8 ng/µl <B>P105</B></LI><LI> pSB1C3_RFC25_longlinker clone2: 197,4 ng/µl <B>P106</B></LI><LI> pSB1C3_RFC25_SEG clone1: 185,6 ng/µl <B>P107</B></LI><LI> pSB1C3_RFC25_SEG clone2: 182,6 ng/µl <B>P108</B></LI><LI> pSB1C3_RFC25_GSAT clone1: 179,5 ng/µl <B>P109</B></LI><LI> pSB1C3_RFC25_GSAT clone2: 202,5 ng/µl <B>P110</B></LI></UL><P><B>Test digestion</B> 1µg DNA
</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">P105 /µl </TD><TD align="right">P107 /µl </TD><TD align="right">P109 /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 5,3</TD><TD align="right">5,3</TD><TD align="right">5,6
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">1 </TD><TD align="right"> 1</TD><TD align="right"> 1
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right"> 1</TD><TD align="right"> 1</TD><TD align="right"> 1
</TD></TR><TR><TD>H2O</TD><TD align="right"> 2,2</TD><TD align="right"> 2,2</TD><TD align="right">1,9
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"><B>10</B></TD><TD align="right"><B>10</B></TD><TD align="right"><B>10</B></TD></TR></TBODY></TABLE><P>
agarose gel: 2%, digestion: 45 minutes, 37°C 
</P><P><B>P105, P107 and P109 are sent for sequenzing with Reverse Primer (VR2)</B></P><P style="font-size:13px; color:#68bbff;"><I><B>Comments:</B></I>this is no satisfying result. we can't imagine what this bands stand for. maybe the CFP is still in the pSB1C3 and the digestion of the pSB1C3_RFC25_CFP didn't work. P105, P107 and P109 are sent for sequenzing to look for the CFP respectively the linkers  </P><H4><SPAN class="mw-headline" id="Picking_clones_of_pSB1C3_CFP_w.2Fo_SspI_and_PvuII"><P style="font-size:15px; background-color:#66bbff;">Picking clones of pSB1C3_CFP w/o SspI and PvuII</P></SPAN></H4><P><B>Investigator: Jessica</B></P><UL><LI>Bacterial strain used: XL1-B
</LI><LI>Four clones were picked of trafo plate
</LI><LI>Inoculating of 10 mL DYT medium containing 10µL chloramphenicol</LI></UL><P><B>Hanna will do the Mini Prep tomorrow</B></P><H4><SPAN class="mw-headline" id="Inoculation_of_pSB1C3_RFC25_CFP"><P style="font-size:15px; background-color:#66bbff;">Inoculation of pSB1C3_RFC25_CFP</P></SPAN></H4><P><B>Investigator: Jessica </B></P><UL><LI> 200ml DYT was prepared with 200µl chloramphenicol and the glycerol stock no. B39
</LI><LI> stored in the 37°C room over night for Midi Prep tomorrow
</LI></UL><H3><SPAN class="mw-headline" id="74._Labortag_30.7.2010"><P style="font-size:17px; background-color:#00dd77;">74. Labortag 30.7.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Continuation_of_PCR_with_pTV_CMV_and_digestion_of_pSB1C3"><P style="font-size:15px; background-color:#66bbff;">Continuation of PCR with pTV_CMV and digestion of pSB1C3</P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="font-size:13px; color:#68bbff;"><B>Comments:</B> Retrafo of the pTV_CMv constructs has been performed and sequencing revealed that the plasmid contains the CMV promoter. Therefore, PCR was be performed overnight. After PCR the PCR product was loaded on a 1% preparative agarose gel. While PCR was separated by the agarose gel, digestion of pSB1C3_CFP was performed.</P><P><B>Proceeding:</B></P><UL><LI>Prepare 1% agarose gel and load samples on gel. </LI><LI>Run time: 45 minutes</LI></UL><P style="font-size:15px; color:#68bbff;"><B>Result of gel:</B></P><P>Expected sizes of PCR products:
</P><UL><LI>CMV = 681 bp</LI></UL><P>
After gel extraction was performed, the PCR prdouct CMV was digested:
</P><P style="font-size:15px; color:#68bbff;">DIGESTION of PCR products</P><TABLE border="1"><TBODY><TR><TH><B>Components</B></TH><TH><B>Volume/µL</B></TH></TR><TR><TD>PCR product: CMV</TD><TD>29</TD></TR><TR><TD>BSA (100x)</TD><TD>0,4</TD></TR><TR><TD>Buffer 4 (10x)</TD><TD>4</TD></TR><TR><TD>Enzyme XbaI</TD><TD>1</TD></TR><TR><TD>Enzyme PstI</TD><TD>1</TD></TR><TR><TD>H20</TD><TD>4,6</TD></TR><TR><TD><B>Total volume</B></TD><TD><B>40</B></TD></TR></TBODY></TABLE><UL><LI>Incubate for 2h at 37°C</LI><LI>Perform PCR purification following standard protocols</LI><LI>Elute in 20µL EB-buffer, put on 50°C for 3 minutes before centrifugating</LI><LI>Measure DNA with NanoDrop:</LI><UL><LI>c(CMV) = 87,4ng/µL</LI></UL></UL><P>
In the meantime prepare the <B>digestion of vector pSB1C3_CFP</B> (P51)
</P><P style="font-size:15px; color:#68bbff;">DIGESTION of pSB1C3_CFP</P><TABLE border="1"><TBODY><TR><TH><B>Components</B></TH><TH><B>Volume/µL</B></TH></TR><TR><TD>pSB1C3_CFP</TD><TD>6</TD></TR><TR><TD>BSA (10x)</TD><TD>2</TD></TR><TR><TD>Buffer 4 (10x)</TD><TD>2</TD></TR><TR><TD>Enzyme XbaI</TD><TD>1</TD></TR><TR><TD>Enzyme PstI</TD><TD>1</TD></TR><TR><TD>H20</TD><TD>8</TD></TR><TR><TD><B>Total volume</B></TD><TD><B>20</B></TD></TR></TBODY></TABLE><UL><LI>Incubate for 2h at 37°C</LI><LI>Load sample on 1% preparative agarose gel </LI><LI>Run for 45 minutes at 110 V</LI></UL><P style="font-size:15px; color:#68bbff"><B>Result of gel:</B></P><P>The vector was digested with XbaI and PstI. Expected sizes have been:</P><UL><LI>pSB1C3 = 2072bp</LI><LI>CFP: 732bp</LI></UL><UL><LI>Proceed with ligation of pSB1C3 and CMV</LI><UL><LI>v (vector) = 7,37µL </LI><LI>v (insert)= 1,63µL </LI></UL><LI> Trafo has been performed with XL1B cells. </LI><UL><LI>100µL if XL1B cells have been transformed with 2µL of pSB1C3_CMV</LI><LI>Transformed cells have been plated on agar plates containing chloramphenicol</LI></UL></UL><P><B>Next steps: Picking clones of trafo plate and performing Mini-Prep. When all BioBricks are ready conduct assembly strategy and test constructs.</B></P><H4><SPAN class="mw-headline" id="Repetition_of_test_digestion_of_Rep_68_and_Rep_78"><P style="font-size:15px; background-color:#66bbff;"><B>Repetition of test digestion of Rep 68 and Rep 78 </B></P></SPAN></H4><P><B>Investigator: Kira</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>Volume/µL</B></TD><TD align="left"><B>Mastermix</B></TD></TR><TR><TD align="left"> DNA </TD><TD align="left"> Variable (800 ng) </TD><TD align="left"> -
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 1 </TD><TD align="left"> 6
</TD></TR><TR><TD align="left"> Buffer no. 4   (10x) </TD><TD align="left"> 1 </TD><TD align="left">6
</TD></TR><TR><TD align="left"> Enzyme 1 ( PstI    ) </TD><TD align="left"> 0,5 </TD><TD align="left"> 3
</TD></TR><TR><TD align="left"> H2O </TD><TD align="left"> variable </TD><TD align="left"> -
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>10</B></TD><TD align="left"> -
</TD></TR></TBODY></TABLE><UL><LI>Incubation: 1h @ 37 C
</LI></UL><P>1% agarose gel 
</P><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><P>1/3 of the digested solution was diluted in 2/3 water because the agarose gel from yesterday showed very distinct bands
</P><TABLE border="1"><TBODY><TR><TH>Marker /µl
</TH><TH>Sample Rep 68 wt
</TH><TH>Sample Rep 68 clone 1 /µl
</TH><TH>Sample Rep 68 clone 2 /µl
</TH><TH>Sample Rep 68 clone 3 /µl
</TH><TH>Sample Rep 78 clone 1 /µl
</TH><TH>Sample Rep 78 clone 2 /µl
</TH><TH>Sample Rep 78 clone 3 /µl
</TH><TH>Sample Rep 78 wt
</TH></TR><TR><TH>Lane
</TH><TD>7
</TD><TD>12
</TD><TD>12
</TD><TD>12
</TD><TD>12
</TD><TD>12
</TD><TD>12
</TD><TD>12
</TD><TD>12
</TD></TR></TBODY></TABLE><P>TAE buffer was changed before start, the samples have been diluted to 1/3 but the bands are still very distinct. However, it is possible to analyse the gel. Rep 68 contains no PstI restriction sites anymore, while Rep 78 shows one band, which might be due to the additional PstI restiction site. 
</P><P>clone1_Rep68 and clone1_ Rep78 were sent for sequencing
</P><H4><SPAN class="mw-headline" id="Continuation:_left_ITR"><P style="font-size:15px; background-color:#66bbff;"><B>Continuation: left ITR</B></P></SPAN></H4><P><B>Investigator: Hanna</B>
GATC didn't manage to do the sequencing of the left ITR-Präfix construct until now. This can be interpreted as positive news, because this increases the fact that the ITR is in the plasmid (very strong secondary structures!). Therefore cloning will  be continued today:
</P><P style="font-size:20px; font-weight: bold; color: fuchsia;"><U>Practical Cloning:</U></P><P style="font-size:15px; font-weight: bold; color: fuchsia;">Digestion of pGA14_Cerulean (P61) and of pGA14_10Präfix-lITR (P111)</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of pGA14_Cerulean (P61) /µl </TD><TD align="right">volume of pGA14_10Präfix-lITR (P111) /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 7.5 </TD><TD align="right"> 9.3
</TD></TR><TR><TD> BSA (100x) </TD><TD align="right"> 0.2 </TD><TD align="right"> 0.2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right"> 2 </TD><TD align="right"> 2
</TD></TR><TR><TD>Enzyme 1 </TD><TD align="right"> PstI-HF: 1 </TD><TD align="right"> NotI-HF: 1
</TD></TR><TR><TD>Enzyme 2 </TD><TD align="right"> XbaI: 1 </TD><TD align="right"> XbaI: 1
</TD></TR><TR><TD>H2O</TD><TD align="right"> 8.3 </TD><TD align="right"> 6.5
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 20 </TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><UL><LI>Incubation: ~2 h
</LI></UL><P style="font-size:15px; font-weight: bold; color: fuchsia;">Agarose-Gel:</P><P><B>P61:</B>
0.5 g Agarose, <B>50 mL TBE!!!</B> (1 %), 3 µL GELRED, at 115 Volt, running time: 45 minutes<B>P111:</B>
0.8 g Agarose, <B>50 mL TBE!!!</B> (1.45 %), 3 µL GELRED, at 115 Volt, running time: 35 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (5x)/µl
</TH><TH>Expected size 1 (Geneious)
</TH><TH>Expected size 2 (Geneious)
</TH></TR><TR><TD>P61
</TD><TD>20 µl
</TD><TD>5 µl
</TD><TD>2896 bp
</TD><TD>~ 750 bp
</TD></TR><TR><TD>P111
</TD><TD>20 µl
</TD><TD>5 µl
</TD><TD>151 bp
</TD><TD>~ 2890 bp
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><P style="font-size:15px; font-weight: bold; color: fuchsia;">Gelextraction</P><P>
Gel measurement:
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Sample</B></TD><TD align="left"><B>Concentration</B></TD></TR><TR><TD align="left"> P61 </TD><TD align="left"> 12.44 ng/µL
</TD></TR><TR><TD align="left"> P111 </TD><TD align="left"> 11.21 ng/µL
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color: fuchsia;">Oligo-Hybridization</P><P><B>RFC10-Suffix of left ITR:</B></P><UL><LI> 10 µL Oligo 1 (O102)
</LI><LI> 10 µL Oligo 2 (O103)
</LI><LI> 4 µL 100 mM TrisHCl pH8
</LI><LI> 8 µL 5 mM MgCl2
</LI><LI> 8 µL H2O
</LI></UL><P>Total: 40 mL
<B>Thermocycler program: ORIGAMI1</B>
1. 99°C, 7' 
2. 99°C, 1' 
73 x, -1°C; R = 0.3 °/s 
3. Hold 4°C
</P><P style="font-size:15px; font-weight: bold; color: fuchsia;">Ligation</P><UL><LI> left ITR fragment: 6.2 µL
</LI><LI> Suffix: 1 µL
</LI><LI> pGA14: 11.5 µL
</LI><LI> Buffer: 2.2 µL
</LI><LI> T4 Ligase: 1 µL
</LI></UL><P><B>Total:</B> 21.9 µL

Sample will be incubating over night in the thermocycler at 16°C.
</P><H4><SPAN class="mw-headline" id="Continuation:_right_ITR"><P style="font-size:15px; background-color:#66bbff;">Continuation: right ITR</P></SPAN></H4><P><B>Investigator: Hanna</B>
Sequencing results delivered that the right ITR was succesfully ligated to the pGA14 plasmid. Unfortunately just about 100 bp were successfully sequenced. Within the ITR region sequencing seemed one more time not to be possible due to the very strong secondary structures which hinder the polymerase to read through. Therefore we can just validate that the ITR with the prefix is inside pGA14, but it's not possible to give a statement about the ITR quality or whether the suffix was successfully fused to the sequence. Theoretically the digested plasmid is not able to religate without ITR and Suffix sequence. But noone knows what our <I>E.colis</I> are able to achieve :)
</P><P>Nevertheless we decided to continue with the experiment: </P><P style="font-size:20px; font-weight: bold; color: fuchsia;"><U>Practical Cloning:</U></P><P style="font-size:15px; font-weight: bold; color: fuchsia;">Digestion of pGA14_Cerulean (P61) and of pGa14_Präfix-PreSuffix-rITR (P115)</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of pGA14_Cerulean (P61) /µl </TD><TD align="right">volume of pGa14_Präfix-PreSuffix-rITR (P115) /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 7.5 </TD><TD align="right"> 7.5
</TD></TR><TR><TD> BSA (100x) </TD><TD align="right"> 0.2 </TD><TD align="right"> 0.2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right"> 2 </TD><TD align="right"> 2
</TD></TR><TR><TD>Enzyme 1 </TD><TD align="right"> EcoRI-HF: 1 </TD><TD align="right"> EcoRI-HF: 1
</TD></TR><TR><TD>Enzyme 2 </TD><TD align="right"> SpeI: 1 </TD><TD align="right"> SpeI: 1
</TD></TR><TR><TD>H2O</TD><TD align="right"> 8.3 </TD><TD align="right"> 8.3
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 20 </TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><UL><LI>Incubation: ~2 h
</LI></UL><P style="font-size:15px; font-weight: bold; color: fuchsia;">Agarose-Gel:</P><P><B>P61:</B>
0.5 g Agarose, <B>50 mL TBE!!!</B> (1 %), 3 µL GELRED, at 115 Volt, running time: 45 minutes<B>P115:</B>
0.8 g Agarose, <B>50 mL TBE!!!</B> (1.45 %), 3 µL GELRED, at 115 Volt, running time: 35 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (5x)/µl
</TH><TH>Expected size 1 (Geneious)
</TH><TH>Expected size 2 (Geneious)
</TH></TR><TR><TD>P61
</TD><TD>20 µl
</TD><TD>5 µl
</TD><TD>2895 bp
</TD><TD>~ 750 bp
</TD></TR><TR><TD>P115
</TD><TD>20 µl
</TD><TD>5 µl
</TD><TD>171 bp
</TD><TD>~ 2890 bp
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><P style="font-size:15px; font-weight: bold; color: fuchsia;">Gelextraction</P><P>
Gel measurement:
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Sample</B></TD><TD align="left"><B>Concentration</B></TD></TR><TR><TD align="left"> P61 </TD><TD align="left"> 16.65 ng/µL
</TD></TR><TR><TD align="left"> P115 </TD><TD align="left"> 7.95 ng/µL
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color: fuchsia;">Ligation</P><UL><LI> right ITR fragment: 2.2 µL
</LI><LI> pGA14: 5.8 µL
</LI><LI> Buffer: 1 µL
</LI><LI> T4 Ligase: 1 µL
</LI></UL><P><B>Total:</B> 10 µL

Sample will be incubating over night in the thermocycler at 16°C.
</P><H4><SPAN class="mw-headline" id="ITR_gel_pictures"><P style="font-size:15px; background-color:#66bbff;">ITR gel pictures</P></SPAN></H4><P><B>Investigator: Hanna</B></P><P>Digestion worked. The marked bands were cut out. </P><P>Digestion of the right ITR fragment worked well. Unfortunately the left ITR fragment is too large. The ecpectation was ~ 151 bp. It seemed to consist of ~ 300 bp. --&gt; Two ITRs in tandem??? Hopefully GATC will find a way in order to sequence the ITRs!
</P><H4><SPAN class="mw-headline" id="ZEGFR:1907"><P style="font-size:15px; background-color:#66bbff;"><B>ZEGFR:1907</B></P></SPAN></H4><P><B>Investigator: Hanna (guided by Sven)</B>
The Affibody-BioBrick (RFC25) was ordered at Geneart. The sequence does not contain any iGEM restriction sites. We'll receive the DNA in ~ 8 work days. <B>Strategy:</B> See Lab day 28.07.2010. <B>More information:</B> See <A href="/wiki/index.php?title=Informationssammlung_zum_AAV&amp;action=edit&amp;redlink=1" class="new" title="Informationssammlung zum AAV (page does not exist)">Informationssammlung zum AAV</A> and <A href="/wiki/index.php?title=Theoretical_cloning&amp;action=edit&amp;redlink=1" class="new" title="Theoretical cloning (page does not exist)">Theoretical cloning</A> !!
</P><H4><SPAN class="mw-headline" id="Mini-Prep_and_Test_digestion_of_pSB1C3_SDM_SspI.2C_pSB1C3_SDM_PvuII.2C_pSB1C3_beta-globin.2C_pSB1C3_hgH.2C_pAAV_RC_1.1_BamHI_long_PCR.2C_pAAV_RC_1.1_BamHI_short_PCR.2CpGL3_hTERT_and_pAAV_RC_1.1_SalI_.28short_PCR.29.2C_pAAV_RC_1.1_BamHI_long_PCR.2C_pAAV_RC_1.1_BamHI_short_PCR"><P style="font-size:15px; background-color:#66bbff;">Mini-Prep and Test digestion of pSB1C3_SDM_SspI, pSB1C3_SDM_PvuII, pSB1C3_beta-globin, pSB1C3_hgH, pAAV_RC_1.1_BamHI long PCR, pAAV_RC_1.1_BamHI short PCR,pGL3_hTERT and pAAV_RC_1.1_SalI (short PCR), pAAV_RC_1.1_BamHI long PCR, pAAV_RC_1.1_BamHI short PCR</P></SPAN></H4><P><B>Investigator: Anissa, Stefan</B></P><P style="font-size:13px; color:#68bbff;"><B>Comments: There are some (SspI, PvuII, BamHI, SalI) restriction sites in our standard vector which are also present in the viral loops. Therefore, the sites in the vector have to be removed.
The beta-globin and hGH constructs are BioBricks for sending to iGEM headquarters. Sequencing is needed. hTERT is  tumor-specific promotor we obtained from Kira. Sequencing is needed.</B></P><P>Mini-Prep of pSB1C3_SDM_SspI clone 1 to 4:
After performing Mini-Preps concentration was measured with Nanodrop:
</P><UL><LI>pSB1C3_SDM_SspI clone 1 (=P125): c = 321,9 ng/µl
</LI><LI>pSB1C3_SDM_SspI clone 2 (=P126): c = 290,3 ng/µl
</LI><LI>pSB1C3_SDM_SspI clone 3 (=P127): c = 309,0 ng/µl
</LI><LI>pSB1C3_SDM_SspI clone 4 (=P128): c = 298,1 ng/µl</LI></UL><P>
Mini-Prep of pSB1C3_SDM_PvuII clone 1 to 4:
After performing Mini-Preps concentration was measured with Nanodrop:
</P><UL><LI>pSB1C3_SDM_PvuII clone 1 (=P129): c = 308,4 ng/µl
</LI><LI>pSB1C3_SDM_PvuII clone 2 (=P130): c = 316,9 ng/µl
</LI><LI>pSB1C3_SDM_PvuII clone 3 (=P131): c = 445,6 ng/µl
</LI><LI>pSB1C3_SDM_PvuII clone 4 (=P132): c = 445,4 ng/µl</LI></UL><P>
Mini-Prep of pSB1C3_beta-globin clone 1 and 2:
After performing Mini-Preps concentration was measured with Nanodrop:
</P><UL><LI>pSB1C3_beta-globin clone 1 (=P133): c = 221,9 ng/µl
</LI><LI>pSB1C3_beta-globin clone 2 (=P134): c = 206,3 ng/µl</LI></UL><P>
Mini-Prep of pSB1C3_hgH clone 1 and 2:
After performing Mini-Preps concentration was measured with Nanodrop:
</P><UL><LI>pSB1C3_hgH clone 1 (=P135): c = 245,5 ng/µl
</LI><LI>pSB1C3_hgH clone 2 (=P136): c = 226,2 ng/µl</LI></UL><P>
Mini-Prep of pAAV_RC_1.1_BamHI (long PCR) clone 1 and 2:
After performing Mini-Preps concentration was measured with Nanodrop:
</P><UL><LI>pAAV_RC_1.1_BamHI (long PCR) clone 1 (=P137): c = 537,8 ng/µl
</LI><LI>pAAV_RC_1.1_BamHI (long PCR) clone 2 (=P138): c = 546,9 ng/µl</LI></UL><P>
Mini-Prep of pAAV_RC_1.1_BamHI (short PCR) clone 1 and 2:
After performing Mini-Preps concentration was measured with Nanodrop:
</P><UL><LI>pAAV_RC_1.1_BamHI (short PCR) clone 1 (=P139): c = 547,0 ng/µl
</LI><LI>pAAV_RC_1.1_BamHI (short PCR) clone 2 (=P140): c = 494,0 ng/µl</LI></UL><P>
Mini-Prep of pGL3_hTERT clone 1 and 2:
After performing Mini-Preps concentration was measured with Nanodrop:
</P><UL><LI>pGL3_hTERT clone 1 (=P141): c = 286,1 ng/µl
</LI><LI>pGL3_hTERT clone 2 (=P142): c = 319,9 ng/µl</LI></UL><P>
Mini-Prep of pAAV_RC_1.1_SalI (short PCR) clone 1 and 2:
After performing Mini-Preps concentration was measured with Nanodrop:
</P><UL><LI>pAAV_RC_1.1_SalI (short PCR) clone 1 (=P143): c = 480,0 ng/µl
</LI><LI>pAAV_RC_1.1_SalI (short PCR) clone 2 (=P144): c = 510,7 ng/µl</LI></UL><P><B>Test digestion:</B>
pSB1C3_SDM_SspI was digested with SspI.
pSB1C3_SDM_PvuII was digested with PvuII.
pSB1C3_beta-globin was digested with XbaI and PstI.
pSB1C3_hgH was digested with XbaI and PstI.
pAAV_RC_1.1_BamHI (long PCR) was digested with BamHI.
pAAV_RC_1.1_BamHI (short PCR) was digested with BamHI.
pGL3_hTERT was digested with BglII and MluI.
pAAV_RC_1.1_SalI (short PCR) was digested with SalI.</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of <B>P125</B>/µl </TD><TD>volume of <B>P126</B>/µl</TD><TD>volume of <B>P127/</B>µl </TD><TD>volume of <B>P128</B>/µl </TD><TD>volume of <B>neg. control P51.1</B>/µl</TD><TD>volume of <B>P129</B>/µl</TD><TD>volume of <B>P130/</B>µl </TD><TD>volume of <B>P131</B>/µl</TD><TD>volume of <B>P132/</B>µl </TD><TD>volume of <B>neg. control P51.1</B>/µl</TD><TD>volume of <B>P133</B>/µl</TD><TD>volume of <B>P134</B>/µl</TD><TD>volume of <B>P135</B>/µl</TD><TD>volume of <B>P136/</B>µl </TD><TD>volume of <B>P137</B>/µl </TD><TD>volume of <B>P138</B>/µl</TD><TD>volume of <B>neg. control P53.3</B>/µl</TD><TD>volume of <B>P139/</B>µl </TD><TD>volume of <B>P140</B>/µl</TD><TD>volume of <B>neg. control P53.2</B>/µl</TD><TD>volume of <B>P141/</B>µl </TD><TD>volume of <B>P142</B>/µl</TD><TD>volume of <B>P143/</B>µl</TD><TD>volume of <B>P144</B>/µl</TD><TD>volume of <B>neg. control P53.1</B>/µl
</TD></TR><TR><TD> DNA  </TD><TD>3 </TD><TD>3 </TD><TD>3 </TD><TD>3 </TD><TD>7 </TD><TD>3 </TD><TD>3 </TD><TD>2 </TD><TD>2 </TD><TD>7 </TD><TD>4 </TD><TD>4 </TD><TD>4 </TD><TD>4 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>2 </TD><TD>1,5 </TD><TD>2 </TD><TD>2 </TD><TD>3 </TD><TD>3 </TD><TD>2 </TD><TD>1,8 </TD><TD>2
</TD></TR><TR><TD> BSA (10x) </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5 </TD><TD>1,5
</TD></TR><TR><TD>Enzyme 1 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5
</TD></TR><TR><TD>Enzyme 2 </TD><TD>/ </TD><TD>/ </TD><TD>/ </TD><TD>/ </TD><TD>/ </TD><TD>/ </TD><TD>/ </TD><TD>/ </TD><TD>/ </TD><TD>/ </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>/ </TD><TD>/ </TD><TD>/ </TD><TD>/ </TD><TD>/ </TD><TD>/ </TD><TD>0,5 </TD><TD>0,5 </TD><TD>/ </TD><TD>/ </TD><TD>/
</TD></TR><TR><TD>H2O </TD><TD>8,5 </TD><TD>8,5 </TD><TD>9,5 </TD><TD>9,5 </TD><TD>4,5 </TD><TD>8,5 </TD><TD>8,5 </TD><TD>8,5 </TD><TD>8,5 </TD><TD>4,5 </TD><TD>7 </TD><TD>7 </TD><TD>7 </TD><TD>7 </TD><TD>10 </TD><TD>10 </TD><TD>9,5 </TD><TD>10 </TD><TD>9,5 </TD><TD>9,5 </TD><TD>8 </TD><TD>8 </TD><TD>9,5 </TD><TD>9,7 </TD><TD>9,5
</TD></TR><TR><TD><B>Total volume /µl</B></TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15 </TD><TD>15
</TD></TR></TBODY></TABLE><P><B>Expected sizes:</B>
pSB1C3_SDM_SspI
</P><UL><LI>no cut</LI></UL><P>control:
</P><UL><LI>2072bp</LI></UL><P>pSB1C3_SDM_PvuII
</P><UL><LI>no cut</LI></UL><P>control:
</P><UL><LI>2072bp</LI></UL><P>pSB1C3_beta-globin
</P><UL><LI>756bp</LI><LI>2058bp</LI></UL><P>pSB1C3_hgH
</P><UL><LI>502bp</LI><LI>2058bp</LI></UL><P>pAAV_RC_1.1_BamHI (long and short PCR):
</P><UL><LI>no cut</LI></UL><P>control:
</P><UL><LI>811bp</LI><LI>6524bp</LI></UL><P>pGL3_hTERT
</P><UL><LI>467bp</LI><LI>4801bp</LI></UL><P>pAAV_RC_1.1_SalI (short PCR):
</P><UL><LI>no cut</LI></UL><P>control:
</P><UL><LI>7331bp</LI></UL><P style="font-size:13px; color:#11bb88;"><B>Results of samples loaded 1% agarose gel and ran ~50 minutes at 115 V:</B></P><P style="font-size:13px; color:#11bb56;"><B>Sequencing</B></P><P>Sequencing will be done next week.
</P><H4><SPAN class="mw-headline" id="Midi-prep_of_pSB1C3_CFP"><P style="font-size:15px; background-color:#66bbff;"><B>Midi-prep of pSB1C3_CFP</B></P></SPAN></H4><P><B>Investigator: Anissa</B></P><P style="font-size:13px; color:#68bbff;"><B>Comments:</B> The mini-prep of pSB1C3_CFP was depleted, so a double midi-prep was made for serving the pool. They were named p51.1 and p51.2. Both are from the glycerol-stock B39, which means they are sequenzed and ready to use.</P><UL><LI> concentration p51.1: 132,06 ng/µL
</LI><LI> concentration p51.2: 151,47 
</LI></UL><H3><SPAN class="mw-headline" id="75._Labortag_31.7.2010"><P style="font-size:17px; background-color:#00dd77;">75. Labortag 31.7.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Trafo_of_ITR-over-night_ligations"><P style="font-size:15px; background-color:#66bbff;"><B>Trafo of ITR-over-night ligations</B></P></SPAN></H4><P><B>Investigator: Hanna</B></P><P>The last step of the ITR fancy method was done ytesterday. Means: If cloning worked, both ITRs should be in the RFC10 standard. Ligation was done over night and the referring trafo will be done today. 
Trafo was performed following the standard protocol. 
Abberations:
3 µL sample was added to BL21 cells. Incubation time on ice was just 15 minutes. Incubation at 37°C for 45 minutes.
</P><CENTER><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/August" class="external text" rel="nofollow"><B>=&gt; Go to Labjournal August part 1 (labday 76 - 92)</B></A></CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/July">http://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/July</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg_Bioware/NoteBook/Labjournal/July" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg_Bioware/NoteBook/Labjournal/July" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/July&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/July&amp;oldid=197548" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>