<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Bioware_NoteBook_Labjournal_August"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg_Bioware/NoteBook/Labjournal/August">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg_Bioware/NoteBook/Labjournal/August&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/August&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/August&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg_Bioware/NoteBook/Labjournal/August" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg Bioware/NoteBook/Labjournal/August</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="menu"><UL id="menu"><LI><DL><DT class="menu_button_left"><A href="https://2010.igem.org/Team:Freiburg_Bioware">Home</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Team_2010">Team</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Team_2010">Team 2010</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Photo_Gallery">Photo Gallery</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Cuckoo_Clock">Cuckoo Clock Competition</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Collaboration">Collaboration</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Example_Page">Example Page</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project">Project</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project">Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#highlights">Highlights</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#highlights">Results</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#modularisation">   Modularisation</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#targeting">   Targeting</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#arming">   Arming</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#superconstruct">   All-in-One</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Methods">Methods</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Virus_Construction_Kit">Virus Construction Kit Manual</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks">BioBricks</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks">Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/capsid_coding">Capsid Coding Parts</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/vector_plasmid">Vector Plasmid Parts</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/miscellaneous">Miscellaneous Parts</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A class="current" href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook">Notebook</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal">Labjournal</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Materials">Materials</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Methods">Methods</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Experiments">Experiments</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Meetings">Meetings</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Statistics">Statistics</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Sponsors">Sponsors</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Safety">Safety</A></DT></DL></LI><LI><DL><DT class="menu_button_right"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling">Modeling</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling">Modeling Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Virus_Production">Virus Production</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Virus_Infection">Virus Infection</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Structure_Modeling">Structure Modeling</A></DD></DL></LI></UL></DIV><DIV class="pageContent"><P><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook" class="external text" rel="nofollow">=&gt; Back to Notebook overview</A></P><DIV class="box_right"><LEFT><U1>NoteBook Navigator</U1></LEFT><UL><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/March">March (labday 1)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/April">April (labday 2 - 5)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/May">May (labday 6 - 17)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/June">June (labday 18 - 45)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/July">July (labday 46 - 75)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/August">August part 1 (labday 76 - 92)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/August2">August part 2 (labday 93 - 106)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/September">September part 1 (labday 107 - 123)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/September2">September part 2 (labday 124 - 135)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/October">October part 1 (labday 136 - 149 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/October2">October part 2 (labday 150 - 166 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/November">November  (labday 167 - 170 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/Cellculture">Cellculture</A></LI></UL></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#76.Labortag_01.08.2010"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">76.Labortag 01.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Picking_clones_of_pSB1C3_CMV"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Picking clones of pSB1C3_CMV</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-3"><A href="#77.Labortag_02.08.2010"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">77.Labortag 02.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Sequenzing_of_pSB1C3_RFC25_longlinker.2C_pSB1C3_RFC25_SEG.2C_pSB1C3_RFC25_GSAT"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Sequenzing of pSB1C3_RFC25_longlinker, pSB1C3_RFC25_SEG, pSB1C3_RFC25_GSAT</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Sequencing_of_pGA14_Prefix-leftITR"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Sequencing of pGA14_Prefix-leftITR</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Repetition_of_test_digestion_pSB1C3_CFP_SDM_SspI_and_PvuII"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Repetition of test digestion pSB1C3_CFP SDM SspI and PvuII</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Mini-Prep_and_Test_digestion_of_pSB1C3_CMV"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Mini-Prep and Test digestion of pSB1C3_CMV</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Sent_for_sequencing"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Sent for sequencing</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Preparation_of_competent_E.coli"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">Preparation of competent E.coli</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Last_Mini-Prep_and_test_digestion_of_ITRs"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">Last Mini-Prep and test digestion of ITRs</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#BioBrick_production:_mGMK_in_pSB1C3"><SPAN class="tocnumber">2.8</SPAN><SPAN class="toctext">BioBrick production: mGMK in pSB1C3</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Sequencing_results_of_SDM_Rep_68_.26_78"><SPAN class="tocnumber">2.9</SPAN><SPAN class="toctext">Sequencing results of SDM Rep 68 &amp; 78</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#78.Labortag_03.08.2010"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">78.Labortag 03.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Results_of_sequencing_pSB1C3_CFP_SDM_PvuII"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Results of sequencing pSB1C3_CFP_SDM_PvuII</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Repetition_of_the_quickchange_site-directed_mutagenesis_of_pAAV_RC_1.1_SalI"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Repetition of the quickchange site-directed mutagenesis of pAAV_RC_1.1_SalI</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#PCR_for_Biobrickproduction_of_Rep_40.2C_52.2C_68.2C_78_and_APP"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">PCR for Biobrickproduction of Rep 40, 52, 68, 78 and APP</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Continuation_of_preparation_of_competent_E.coli"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Continuation of preparation of competent E.coli</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Sequencing_of_pSB1C3"><SPAN class="tocnumber">3.5</SPAN><SPAN class="toctext">Sequencing of pSB1C3</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Design_of_Primer_for_p5_Promoter_WT_and_TATA-less_BioBrick_production"><SPAN class="tocnumber">3.6</SPAN><SPAN class="toctext">Design of Primer for p5 Promoter WT and TATA-less BioBrick production</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#79.Labortag_04.08.2010"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">79.Labortag 04.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Cloning_of_right_ITR_and_left_ITR_into_pSB1C3_RFC25_CFP"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Cloning of right ITR and left ITR into pSB1C3_RFC25_CFP</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#Cloning_of_SDM_SspI_and_SDM_PvuII"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Cloning of SDM SspI and SDM PvuII</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#2nd_Repetition_of_Quickchange_site-directed_mutagenesis_of_pAAV_RC_1.1_SalI"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">2nd Repetition of Quickchange site-directed mutagenesis of pAAV_RC_1.1_SalI</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Mini-Prep_of_pSB1C3_mGMK"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">Mini-Prep of pSB1C3_mGMK</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#cell_culture"><SPAN class="tocnumber">4.5</SPAN><SPAN class="toctext">cell culture</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#Repetition_of_cloning_of_SDM_SspI_and_SDM_PvuII"><SPAN class="tocnumber">4.6</SPAN><SPAN class="toctext">Repetition of cloning of SDM SspI and SDM PvuII</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#Test_transformation_of_XL1_competent_cells"><SPAN class="tocnumber">4.7</SPAN><SPAN class="toctext">Test transformation of XL1 competent cells</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#Sequencing_results_of_ITRs"><SPAN class="tocnumber">4.8</SPAN><SPAN class="toctext">Sequencing results of ITRs</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#Cloning_of_Rep68.2C_Rep78_and_AAP_into_pSB1C3"><SPAN class="tocnumber">4.9</SPAN><SPAN class="toctext">Cloning of Rep68, Rep78 and AAP into pSB1C3</SPAN></A></LI><LI class="toclevel-1 tocsection-30"><A href="#80.Labortag_05.08.2010"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">80.Labortag 05.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#New_LB_Agar_was_prepared."><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">New LB Agar was prepared.</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#Sequenc_analysis_of_BioBricks:_CMV.2C_hGH_and_beta_globin"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Sequenc analysis of BioBricks: CMV, hGH and beta globin</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#Cellculture"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Cellculture</SPAN></A></LI><LI class="toclevel-2 tocsection-34"><A href="#Picking_clones_of_pSB1C3_AAP_2.2C_pSB1C3_AAP.2C_pSB1C3_Rep78_2.2C_pSB1C3_Rep78.2C_pSB1C3_Rep68_2_and_pSB1C3_Rep68"><SPAN class="tocnumber">5.4</SPAN><SPAN class="toctext">Picking clones of pSB1C3_AAP_2, pSB1C3_AAP, pSB1C3_Rep78_2, pSB1C3_Rep78, pSB1C3_Rep68_2 and pSB1C3_Rep68</SPAN></A></LI><LI class="toclevel-2 tocsection-35"><A href="#PCR_and_ligation_for_biobrick-production_of_pSB1C3_hTERT"><SPAN class="tocnumber">5.5</SPAN><SPAN class="toctext">PCR and ligation for biobrick-production of pSB1C3_hTERT</SPAN></A></LI><LI class="toclevel-2 tocsection-36"><A href="#Transformation_evaluation_of_XL1B_cells_and_repetition_of_transformation"><SPAN class="tocnumber">5.6</SPAN><SPAN class="toctext">Transformation evaluation of XL1B cells and repetition of transformation</SPAN></A></LI><LI class="toclevel-2 tocsection-37"><A href="#Cloning_of_SDM_SspI_and_SDM_PvuII_2"><SPAN class="tocnumber">5.7</SPAN><SPAN class="toctext">Cloning of SDM SspI and SDM PvuII</SPAN></A></LI><LI class="toclevel-2 tocsection-38"><A href="#Miniprep_of_p158_and_p159"><SPAN class="tocnumber">5.8</SPAN><SPAN class="toctext">Miniprep of p158 and p159</SPAN></A></LI><LI class="toclevel-1 tocsection-39"><A href="#81.Labortag_06.08.2010"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">81.Labortag 06.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-40"><A href="#Cloning_Rep40_.26_Rep52_into_pSB1C3"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Cloning Rep40 &amp; Rep52 into pSB1C3</SPAN></A></LI><LI class="toclevel-2 tocsection-41"><A href="#BioBrick_Assembly_of_pSB1C3_beta_globin_mVenus"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">BioBrick Assembly of pSB1C3_beta globin_mVenus</SPAN></A></LI><LI class="toclevel-2 tocsection-42"><A href="#Test_digestion_of_SalI_SDM"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">Test digestion of SalI SDM</SPAN></A></LI><LI class="toclevel-2 tocsection-43"><A href="#ITR_test_digestion_of_pAAV_MCS"><SPAN class="tocnumber">6.4</SPAN><SPAN class="toctext">ITR test digestion of pAAV_MCS</SPAN></A></LI><LI class="toclevel-2 tocsection-44"><A href="#Miniprep_of_pSBC13_Rep78.2C_pSB1C3_Rep68.2C_pSB1C3_AAP.2C_pSB1C3_rightITR_and_psB1C3_leftITR"><SPAN class="tocnumber">6.5</SPAN><SPAN class="toctext">Miniprep of pSBC13_Rep78, pSB1C3_Rep68, pSB1C3_AAP, pSB1C3_rightITR and psB1C3_leftITR</SPAN></A></LI><LI class="toclevel-1 tocsection-45"><A href="#82.Labortag_07.08.2010"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">82.Labortag 07.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-46"><A href="#Cellculture_2"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Cellculture</SPAN></A></LI><LI class="toclevel-2 tocsection-47"><A href="#Continuation:_Cloning_Rep40_.26_Rep52_into_pSB1C3"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">Continuation: Cloning Rep40 &amp; Rep52 into pSB1C3</SPAN></A></LI><LI class="toclevel-2 tocsection-48"><A href="#Mini-Prep_and_test_digestion_of_pSB1C3_SDM_SspI.2FPvuII"><SPAN class="tocnumber">7.3</SPAN><SPAN class="toctext">Mini-Prep and test digestion of pSB1C3_SDM_SspI/PvuII</SPAN></A></LI><LI class="toclevel-2 tocsection-49"><A href="#Evaluation_of_test_transformation_with_XL1_blue_cells_and_pUCI"><SPAN class="tocnumber">7.4</SPAN><SPAN class="toctext">Evaluation of test transformation with XL1 blue cells and pUCI</SPAN></A></LI><LI class="toclevel-2 tocsection-50"><A href="#Biobrick_production_of_p5_promoter_from_pTAV2_and_pAAV_RC"><SPAN class="tocnumber">7.5</SPAN><SPAN class="toctext">Biobrick production of p5 promoter from pTAV2 and pAAV_RC</SPAN></A></LI><LI class="toclevel-2 tocsection-51"><A href="#Sequencing_results_of_SalI_SDM"><SPAN class="tocnumber">7.6</SPAN><SPAN class="toctext">Sequencing results of SalI SDM</SPAN></A></LI><LI class="toclevel-1 tocsection-52"><A href="#83._Labortag_08.08.2010"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">83. Labortag 08.08.2010</SPAN></A></LI><LI class="toclevel-1 tocsection-53"><A href="#84._Labortag_09.08.2010"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">84. Labortag 09.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-54"><A href="#Midi-Prep_Inoculation"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">Midi-Prep Inoculation</SPAN></A></LI><LI class="toclevel-2 tocsection-55"><A href="#Continuation:_Cloning_Rep40_.26_Rep52_into_pSB1C3_2"><SPAN class="tocnumber">9.2</SPAN><SPAN class="toctext">Continuation: Cloning Rep40 &amp; Rep52 into pSB1C3</SPAN></A></LI><LI class="toclevel-2 tocsection-56"><A href="#Repetition:_BioBrick_production_of_Rep40_.26_Rep52"><SPAN class="tocnumber">9.3</SPAN><SPAN class="toctext">Repetition: BioBrick production of Rep40 &amp; Rep52</SPAN></A></LI><LI class="toclevel-2 tocsection-57"><A href="#Sequence_analysis_of_pSB1C3_mGMK"><SPAN class="tocnumber">9.4</SPAN><SPAN class="toctext">Sequence analysis of pSB1C3_mGMK</SPAN></A></LI><LI class="toclevel-2 tocsection-58"><A href="#Mini-Prep_of_pSB1C3_hTERT.2C_pSB1C3_beta-globin_mVenus.2C_pSB1C3_pTAV2.28P5.29.2C_pSB1C3_pAAV_RC.28P5TATAles.29"><SPAN class="tocnumber">9.5</SPAN><SPAN class="toctext">Mini-Prep of pSB1C3_hTERT, pSB1C3_beta-globin_mVenus, pSB1C3_pTAV2(P5), pSB1C3_pAAV_RC(P5TATAles)</SPAN></A></LI><LI class="toclevel-2 tocsection-59"><A href="#Cell_culture_2"><SPAN class="tocnumber">9.6</SPAN><SPAN class="toctext">Cell culture</SPAN></A></LI><LI class="toclevel-3 tocsection-60"><A href="#Transfection_on_HEK293"><SPAN class="tocnumber">9.6.1</SPAN><SPAN class="toctext">Transfection on HEK293</SPAN></A></LI><LI class="toclevel-3 tocsection-61"><A href="#Transduction_of_HT1080"><SPAN class="tocnumber">9.6.2</SPAN><SPAN class="toctext">Transduction of HT1080</SPAN></A></LI><LI class="toclevel-3 tocsection-62"><A href="#Passaging_and_seeding_of_A431"><SPAN class="tocnumber">9.6.3</SPAN><SPAN class="toctext">Passaging and seeding of A431</SPAN></A></LI><LI class="toclevel-2 tocsection-63"><A href="#BioBrick_Assembly_of_pSB1C3_leftITR_CMV_and_pSB1C3_hGH_rightITR"><SPAN class="tocnumber">9.7</SPAN><SPAN class="toctext">BioBrick Assembly of pSB1C3_leftITR_CMV and pSB1C3_hGH_rightITR</SPAN></A></LI><LI class="toclevel-2 tocsection-64"><A href="#Repetition:_Cloning_of_SDM_SspI_and_SDM_PvuII"><SPAN class="tocnumber">9.8</SPAN><SPAN class="toctext">Repetition: Cloning of SDM SspI and SDM PvuII</SPAN></A></LI><LI class="toclevel-2 tocsection-65"><A href="#Design_and_ordering_of_primer_for_P40_Promoter_BioBrick_production"><SPAN class="tocnumber">9.9</SPAN><SPAN class="toctext">Design and ordering of primer for P40 Promoter BioBrick production</SPAN></A></LI><LI class="toclevel-1 tocsection-66"><A href="#85._Labortag_10.08.2010"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">85. Labortag 10.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-67"><A href="#Retrafo_with_RepCap_gene_.28Mr.Gene.29"><SPAN class="tocnumber">10.1</SPAN><SPAN class="toctext">Retrafo with RepCap gene (Mr.Gene)</SPAN></A></LI><LI class="toclevel-2 tocsection-68"><A href="#Continuation:_Cloning_Rep40_.26_Rep52_into_pSB1C3_3"><SPAN class="tocnumber">10.2</SPAN><SPAN class="toctext">Continuation: Cloning Rep40 &amp; Rep52 into pSB1C3</SPAN></A></LI><LI class="toclevel-2 tocsection-69"><A href="#Midi-Prep_of_pAAV_RC.2C_pAAV_RC_1.2_SDM_SalI_.28P158.29.2C_pAAV_RFC25_mGMK_TK30_clone_1_.28P81.29_and_2_.28P82.29"><SPAN class="tocnumber">10.3</SPAN><SPAN class="toctext">Midi-Prep of pAAV_RC, pAAV_RC_1.2 SDM SalI (P158), pAAV_RFC25_mGMK_TK30 clone 1 (P81) and 2 (P82)</SPAN></A></LI><LI class="toclevel-2 tocsection-70"><A href="#Results_of_sequencing_pSB1C3_pTAV2.28P5.29_and_Psb1C3_pAAV_RC.28P5TATAless.29"><SPAN class="tocnumber">10.4</SPAN><SPAN class="toctext">Results of sequencing pSB1C3_pTAV2(P5) and Psb1C3_pAAV_RC(P5TATAless)</SPAN></A></LI><LI class="toclevel-2 tocsection-71"><A href="#Repetition:_BioBrick_production_of_Rep40_and_Rep_52"><SPAN class="tocnumber">10.5</SPAN><SPAN class="toctext">Repetition: BioBrick production of Rep40 and Rep 52</SPAN></A></LI><LI class="toclevel-2 tocsection-72"><A href="#Preparation_of_competent_XL1blue_and_testtrafo"><SPAN class="tocnumber">10.6</SPAN><SPAN class="toctext">Preparation of competent XL1blue and testtrafo</SPAN></A></LI><LI class="toclevel-2 tocsection-73"><A href="#Repetition:_Mini-Prep_and_test_digestion_of_pSB1C3_SDM_SspI.2FPvuII"><SPAN class="tocnumber">10.7</SPAN><SPAN class="toctext">Repetition: Mini-Prep and test digestion of pSB1C3_SDM_SspI/PvuII</SPAN></A></LI><LI class="toclevel-2 tocsection-74"><A href="#Picking_clones_of_pSB1C3_SDM_SspI.2FPvuII"><SPAN class="tocnumber">10.8</SPAN><SPAN class="toctext">Picking clones of pSB1C3_SDM_SspI/PvuII</SPAN></A></LI><LI class="toclevel-1 tocsection-75"><A href="#86._labday_11.08.2010"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">86. labday 11.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-76"><A href="#Transduction_of_HT1080_with_no_AAV.2C_AAV_without_GOI_and_AAV_with_10.C2.B5g"><SPAN class="tocnumber">11.1</SPAN><SPAN class="toctext">Transduction of HT1080 with no AAV, AAV without GOI and AAV with 10µg</SPAN></A></LI><LI class="toclevel-2 tocsection-77"><A href="#Biobrick_Assembly:Picking_Clones.2C_Miniprep.2C_Testdigestion_of_CMV.26lITR.2F_hGH_.26_rITR"><SPAN class="tocnumber">11.2</SPAN><SPAN class="toctext">Biobrick Assembly:Picking Clones, Miniprep, Testdigestion of CMV&amp;lITR/ hGH &amp; rITR</SPAN></A></LI><LI class="toclevel-2 tocsection-78"><A href="#Biobrick_Assembly:_bSB1C3_lITR_CMV_.26_pSB1C3_betaglobin_mVenus"><SPAN class="tocnumber">11.3</SPAN><SPAN class="toctext">Biobrick Assembly: bSB1C3_lITR_CMV &amp; pSB1C3_betaglobin_mVenus</SPAN></A></LI><LI class="toclevel-2 tocsection-79"><A href="#Continuation_of_Cloning_of_Rep40.2F52"><SPAN class="tocnumber">11.4</SPAN><SPAN class="toctext">Continuation of Cloning of Rep40/52</SPAN></A></LI><LI class="toclevel-2 tocsection-80"><A href="#PCR_of_bla_14FM_and_ligation_into_pSB1C3_SDM_SspI_and_pSB1C3_SDM_PvuII_for_virobrick-production_T"><SPAN class="tocnumber">11.5</SPAN><SPAN class="toctext">PCR of bla_14FM  and ligation into pSB1C3_SDM_SspI and pSB1C3_SDM_PvuII for virobrick-production T</SPAN></A></LI><LI class="toclevel-2 tocsection-81"><A href="#Test_digestion_pSB1C3_CFP_SDM_SspI_and_PvuII"><SPAN class="tocnumber">11.6</SPAN><SPAN class="toctext">Test digestion pSB1C3_CFP_SDM SspI and PvuII</SPAN></A></LI><LI class="toclevel-2 tocsection-82"><A href="#pSB1C3_SDM_Ssp1_.28P125.29_sent_for_sequenzing"><SPAN class="tocnumber">11.7</SPAN><SPAN class="toctext">pSB1C3_SDM_Ssp1 (P125) sent for sequenzing</SPAN></A></LI><LI class="toclevel-2 tocsection-83"><A href="#Picking_clones_of_pSB1C3_SDM_SspI.2FPvuII_.28P185.29"><SPAN class="tocnumber">11.8</SPAN><SPAN class="toctext">Picking clones of pSB1C3_SDM_SspI/PvuII (P185)</SPAN></A></LI><LI class="toclevel-1 tocsection-84"><A href="#87._labday_12.08.2010"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">87. labday 12.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-85"><A href="#Virus_production.2C_seeding_A431_cell"><SPAN class="tocnumber">12.1</SPAN><SPAN class="toctext">Virus production, seeding A431 cell</SPAN></A></LI><LI class="toclevel-2 tocsection-86"><A href="#XL1blue_are_ready_to_use"><SPAN class="tocnumber">12.2</SPAN><SPAN class="toctext">XL1blue are ready to use</SPAN></A></LI><LI class="toclevel-2 tocsection-87"><A href="#Mini-Prep_of_pSB1C3_SDM_SspI.2FPvuII_and_pMA_RC_insert"><SPAN class="tocnumber">12.3</SPAN><SPAN class="toctext">Mini-Prep of pSB1C3_SDM_SspI/PvuII and pMA_RC_insert</SPAN></A></LI><LI class="toclevel-2 tocsection-88"><A href="#Biobrick_Assembly:_pSB1C3_lITR_CMV_.26_pSB1C3_betaglobin_mVenus"><SPAN class="tocnumber">12.4</SPAN><SPAN class="toctext">Biobrick Assembly: pSB1C3_lITR_CMV &amp; pSB1C3_betaglobin_mVenus</SPAN></A></LI><LI class="toclevel-2 tocsection-89"><A href="#Repetition_of_cloning_of_bla14FM_viralbrick_MCS_into_pSB1C3_SDM_SspI"><SPAN class="tocnumber">12.5</SPAN><SPAN class="toctext">Repetition of cloning of bla14FM_viralbrick_MCS into pSB1C3_SDM_SspI</SPAN></A></LI><LI class="toclevel-2 tocsection-90"><A href="#Quickchange_of_the_EaglII_restriction_site_in_the_PMA_RepCap_vector"><SPAN class="tocnumber">12.6</SPAN><SPAN class="toctext">Quickchange of the EaglII restriction site in the PMA_RepCap vector</SPAN></A></LI><LI class="toclevel-2 tocsection-91"><A href="#Sequencing_results_of_Rep68.2C_Rep78_and_AAP_BioBricks"><SPAN class="tocnumber">12.7</SPAN><SPAN class="toctext">Sequencing results of Rep68, Rep78 and AAP BioBricks</SPAN></A></LI><LI class="toclevel-2 tocsection-92"><A href="#Comment_13.08.2010_.28Hanna.29:_We_wondered_how_the_whole_Rep_stuff_could_be_sequenced_reverse_AND_forward_without_any_forward_sequencing_primer..._By_checking_the_.22sequencing_book.22_we_found_out_that_always_the_wrong_primers_were_used:_the_GOI_primers_can_be_used_in_order_to_sequence_the_Gene_of_interest_.3D_GOI_.28.21.29_in_pAAV_MCS_and_NOT_the_pSB1C3.21.21.21.21.21.21.21.21.21"><SPAN class="tocnumber">12.8</SPAN><SPAN class="toctext">Comment 13.08.2010 (Hanna): We wondered how the whole Rep stuff could be sequenced reverse AND forward without any forward sequencing primer... By checking the &quot;sequencing book&quot; we found out that always the wrong primers were used: the GOI primers can be used in order to sequence the Gene of interest = GOI (!) in pAAV_MCS and NOT the pSB1C3!!!!!!!!!</SPAN></A></LI><LI class="toclevel-2 tocsection-93"><A href="#Order_oligos_for_CD-biobrick"><SPAN class="tocnumber">12.9</SPAN><SPAN class="toctext">Order oligos for CD-biobrick</SPAN></A></LI><LI class="toclevel-2 tocsection-94"><A href="#BioBrick_assembly_of_pSB1C3_leftITR_CMV_mVenus"><SPAN class="tocnumber">12.10</SPAN><SPAN class="toctext">BioBrick assembly of pSB1C3_leftITR_CMV_mVenus</SPAN></A></LI><LI class="toclevel-2 tocsection-95"><A href="#Opened_a_BioBrick_box"><SPAN class="tocnumber">12.11</SPAN><SPAN class="toctext">Opened a BioBrick box</SPAN></A></LI><LI class="toclevel-1 tocsection-96"><A href="#88._labday_13.08.2010"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">88. labday 13.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-97"><A href="#Seeding_HT1080_for_transduction_at_saturday.2C_Seeding_HEK293_for_transfection"><SPAN class="tocnumber">13.1</SPAN><SPAN class="toctext">Seeding HT1080 for transduction at saturday, Seeding HEK293 for transfection</SPAN></A></LI><LI class="toclevel-2 tocsection-98"><A href="#FACS_of_transduction_from_11.8"><SPAN class="tocnumber">13.2</SPAN><SPAN class="toctext">FACS of transduction from 11.8</SPAN></A></LI><LI class="toclevel-2 tocsection-99"><A href="#Continuation:pSB1C3_SDM_SspI_bla14FM"><SPAN class="tocnumber">13.3</SPAN><SPAN class="toctext">Continuation:pSB1C3_SDM_SspI_bla14FM</SPAN></A></LI><LI class="toclevel-2 tocsection-100"><A href="#Mini-preps_of_pSB1C3_hgh.2CpSB1C3_beta_globin.2CpSB1C3_left_ITR.2CpSB1C3_right_ITR.2CpSB1C3_mGMK.2CpSB1C3_CMV"><SPAN class="tocnumber">13.4</SPAN><SPAN class="toctext">Mini-preps of pSB1C3_hgh,pSB1C3_beta globin,pSB1C3_left ITR,pSB1C3_right ITR,pSB1C3_mGMK,pSB1C3_CMV</SPAN></A></LI><LI class="toclevel-2 tocsection-101"><A href="#Design_and_ordering_of_oligos_for_VP_fusion_targeting_approaches"><SPAN class="tocnumber">13.5</SPAN><SPAN class="toctext">Design and ordering of oligos for VP fusion targeting approaches</SPAN></A></LI><LI class="toclevel-2 tocsection-102"><A href="#Mini-Prep_and_Test_digestion_of_Rep40_and_Rep52_of_the_cloning_experiment_.28.22Cloning_of_Rep40.2F52.22_11.08.2010_Investigator:_Anna.29"><SPAN class="tocnumber">13.6</SPAN><SPAN class="toctext">Mini-Prep and Test digestion of Rep40 and Rep52 of the cloning experiment (&quot;Cloning of Rep40/52&quot; 11.08.2010 Investigator: Anna)</SPAN></A></LI><LI class="toclevel-2 tocsection-103"><A href="#Repetition_of_Rep_40_and_Rep_52_ligation_and_transformation_with_the_vector_pSB1C3_CFP"><SPAN class="tocnumber">13.7</SPAN><SPAN class="toctext">Repetition of Rep 40 and Rep 52 ligation and transformation with the vector pSB1C3_CFP</SPAN></A></LI><LI class="toclevel-2 tocsection-104"><A href="#Strategy_for_VP2_N-terminal_fusion_and_VP1_insertion_of_targeting_moelcules"><SPAN class="tocnumber">13.8</SPAN><SPAN class="toctext">Strategy for VP2 N-terminal fusion and VP1 insertion of targeting moelcules</SPAN></A></LI><LI class="toclevel-2 tocsection-105"><A href="#.3Cp_style.3D.22font-size:15px.3B_background-color:.2366bbff.3B.22.3EpSB1C3_Rep52_.28P202.29_sent_for_sequencing"><SPAN class="tocnumber">13.9</SPAN><SPAN class="toctext">&lt;p style=&quot;font-size:15px; background-color:#66bbff;&quot;&gt;pSB1C3_Rep52 (P202) sent for sequencing</SPAN></A></LI><LI class="toclevel-2 tocsection-106"><A href="#Repetition:_Biobrick_production_of_Rep_68.2C_78_and_AAP"><SPAN class="tocnumber">13.10</SPAN><SPAN class="toctext">Repetition: Biobrick production of Rep 68, 78 and AAP</SPAN></A></LI><LI class="toclevel-1 tocsection-107"><A href="#89._labday_14.08.2010"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">89. labday 14.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-108"><A href="#Seeding_HEK293_for_transfection_at_monday_16.8"><SPAN class="tocnumber">14.1</SPAN><SPAN class="toctext">Seeding HEK293 for transfection at monday 16.8</SPAN></A></LI><LI class="toclevel-2 tocsection-109"><A href="#Transduction_with_mVenus_loaded_viral_particles"><SPAN class="tocnumber">14.2</SPAN><SPAN class="toctext">Transduction with mVenus loaded viral particles</SPAN></A></LI><LI class="toclevel-2 tocsection-110"><A href="#Sequence_analysis_of_pSB1C3_SDM_SspI_BLA"><SPAN class="tocnumber">14.3</SPAN><SPAN class="toctext">Sequence analysis of pSB1C3_SDM_SspI_BLA</SPAN></A></LI><LI class="toclevel-2 tocsection-111"><A href="#Mini_Preps_of_PMA_RepCap_vector_and_pSB1C3_left_ITR_CMV_betaglobin_mVenus"><SPAN class="tocnumber">14.4</SPAN><SPAN class="toctext">Mini Preps of PMA_RepCap vector and pSB1C3_left ITR_CMV_betaglobin_mVenus</SPAN></A></LI><LI class="toclevel-2 tocsection-112"><A href="#Mini-Prep_of_pSB1C3_leftITR_CMV_mVenus"><SPAN class="tocnumber">14.5</SPAN><SPAN class="toctext">Mini-Prep of pSB1C3_leftITR_CMV_mVenus</SPAN></A></LI><LI class="toclevel-2 tocsection-113"><A href="#Test_digestion_of_pSB1C3_pHTERT"><SPAN class="tocnumber">14.6</SPAN><SPAN class="toctext">Test digestion of pSB1C3_pHTERT</SPAN></A></LI><LI class="toclevel-2 tocsection-114"><A href="#Picking_clones_..."><SPAN class="tocnumber">14.7</SPAN><SPAN class="toctext">Picking clones ...</SPAN></A></LI><LI class="toclevel-1 tocsection-115"><A href="#90._labday_15.08.2010"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">90. labday 15.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-116"><A href="#Sequence_analysis_of_pSB1C3_Rep52"><SPAN class="tocnumber">15.1</SPAN><SPAN class="toctext">Sequence analysis of pSB1C3_Rep52</SPAN></A></LI><LI class="toclevel-2 tocsection-117"><A href="#Mini-Prep_of_pSB1C3_Rep68.2C_pSB1C3_Rep78.2C_pSB1C3_RepAAP.2C_pSB1C3_Rep40.2C_pSB1C3_Rep52"><SPAN class="tocnumber">15.2</SPAN><SPAN class="toctext">Mini-Prep of pSB1C3_Rep68, pSB1C3_Rep78, pSB1C3_RepAAP, pSB1C3_Rep40, pSB1C3_Rep52</SPAN></A></LI><LI class="toclevel-2 tocsection-118"><A href="#Inoculation_of_CD_colony"><SPAN class="tocnumber">15.3</SPAN><SPAN class="toctext">Inoculation of CD colony</SPAN></A></LI><LI class="toclevel-1 tocsection-119"><A href="#91._labday_16.08.2010"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">91. labday 16.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-120"><A href="#Minipreps_of_P119.2C_P122.2C_P204.2C_P205"><SPAN class="tocnumber">16.1</SPAN><SPAN class="toctext">Minipreps of P119, P122, P204, P205</SPAN></A></LI><LI class="toclevel-2 tocsection-121"><A href="#Cloning_of_pSB1C3_SspIdel_BLA_clone_2_.26_3_to_get_rid_of_mutations"><SPAN class="tocnumber">16.2</SPAN><SPAN class="toctext">Cloning of pSB1C3_SspIdel_BLA clone 2 &amp; 3 to get rid of mutations</SPAN></A></LI><LI class="toclevel-2 tocsection-122"><A href="#New_working_solutions_were_prepared"><SPAN class="tocnumber">16.3</SPAN><SPAN class="toctext">New working solutions were prepared</SPAN></A></LI><LI class="toclevel-2 tocsection-123"><A href="#Further_pSB1C3_SDM_SspI_Bla14FM_sequencing"><SPAN class="tocnumber">16.4</SPAN><SPAN class="toctext">Further pSB1C3_SDM_SspI_Bla14FM sequencing</SPAN></A></LI><LI class="toclevel-2 tocsection-124"><A href="#Mini_prep_of_pKS-CD"><SPAN class="tocnumber">16.5</SPAN><SPAN class="toctext">Mini prep of pKS-CD</SPAN></A></LI><LI class="toclevel-2 tocsection-125"><A href="#Fluorescence_microscopy_of_TKGMK_loaded_particles_treated_cells"><SPAN class="tocnumber">16.6</SPAN><SPAN class="toctext">Fluorescence microscopy of TKGMK loaded particles treated cells</SPAN></A></LI><LI class="toclevel-2 tocsection-126"><A href="#SDM_of_PstI_in_pKS-CD"><SPAN class="tocnumber">16.7</SPAN><SPAN class="toctext">SDM of PstI in pKS-CD</SPAN></A></LI><LI class="toclevel-2 tocsection-127"><A href="#Biobrick_production_of_pSB1C3_lITR_CMV_beta-globin_mVENUS_hgh_rITR_and_pSB1C3_lITR_CMV_mVENUS_hgh_rITR"><SPAN class="tocnumber">16.8</SPAN><SPAN class="toctext">Biobrick production of pSB1C3_lITR_CMV_beta-globin_mVENUS_hgh_rITR and pSB1C3_lITR_CMV_mVENUS_hgh_rITR</SPAN></A></LI><LI class="toclevel-2 tocsection-128"><A href="#Preparation_for_Midi-Preps_of_B24_and_B34"><SPAN class="tocnumber">16.9</SPAN><SPAN class="toctext">Preparation for Midi-Preps of B24 and B34</SPAN></A></LI><LI class="toclevel-2 tocsection-129"><A href="#Cloning_of_pAAV_RC_.28P50.29_with_pMA_RC_insert_.28P190.29"><SPAN class="tocnumber">16.10</SPAN><SPAN class="toctext">Cloning of pAAV_RC (P50) with pMA_RC_insert (P190)</SPAN></A></LI><LI class="toclevel-2 tocsection-130"><A href="#Repetition:_Biobrick_production_of_Rep40.2C_Rep68.2C_Rep78_and_AAP"><SPAN class="tocnumber">16.11</SPAN><SPAN class="toctext">Repetition: Biobrick production of Rep40, Rep68, Rep78 and AAP</SPAN></A></LI><LI class="toclevel-2 tocsection-131"><A href="#Design_and_Ordering_of_Oligos"><SPAN class="tocnumber">16.12</SPAN><SPAN class="toctext">Design and Ordering of Oligos</SPAN></A></LI><LI class="toclevel-1 tocsection-132"><A href="#92._labday_17.08.2010"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">92. labday 17.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-133"><A href="#Overview_of_the_state-of_the_art_plasmid_preparation"><SPAN class="tocnumber">17.1</SPAN><SPAN class="toctext">Overview of the state-of the art plasmid preparation</SPAN></A></LI><LI class="toclevel-2 tocsection-134"><A href="#Further_pSB1C3_SDM_SspI_Bla14FM_sequencing_2"><SPAN class="tocnumber">17.2</SPAN><SPAN class="toctext">Further pSB1C3_SDM_SspI_Bla14FM sequencing</SPAN></A></LI><LI class="toclevel-2 tocsection-135"><A href="#pSB1C3_SDM_SspI_Bla14FM_clone_5_SDM_.28P222.29"><SPAN class="tocnumber">17.3</SPAN><SPAN class="toctext">pSB1C3_SDM_SspI_Bla14FM clone 5 SDM (P222)</SPAN></A></LI><LI class="toclevel-2 tocsection-136"><A href="#Sent_for_sequencing_2"><SPAN class="tocnumber">17.4</SPAN><SPAN class="toctext">Sent for sequencing</SPAN></A></LI><LI class="toclevel-2 tocsection-137"><A href="#Cloning_of_pMA_RC-Insertparts_in_pAAV_RC_1.2_SDM_SalI"><SPAN class="tocnumber">17.5</SPAN><SPAN class="toctext">Cloning of pMA_RC-Insertparts in pAAV_RC 1.2 SDM SalI</SPAN></A></LI><LI class="toclevel-2 tocsection-138"><A href="#Repetition:_Biobrick_production_of_Rep_68"><SPAN class="tocnumber">17.6</SPAN><SPAN class="toctext">Repetition: Biobrick production of Rep 68</SPAN></A></LI><LI class="toclevel-2 tocsection-139"><A href="#Transduction_of_HT1080_cells"><SPAN class="tocnumber">17.7</SPAN><SPAN class="toctext">Transduction of HT1080 cells</SPAN></A></LI><LI class="toclevel-2 tocsection-140"><A href="#Midi-Prep_of_pAAV_igEM_mVenus_YFP_and_dsAAV_CMV_EGFP"><SPAN class="tocnumber">17.8</SPAN><SPAN class="toctext">Midi-Prep of pAAV_igEM_mVenus_YFP and dsAAV_CMV_EGFP</SPAN></A></LI><LI class="toclevel-2 tocsection-141"><A href="#BioBrick_assembly_for_pSB1C3_lITR_CMV_beta-globin_YFP_rITR"><SPAN class="tocnumber">17.9</SPAN><SPAN class="toctext">BioBrick assembly for pSB1C3_lITR_CMV_beta-globin_YFP_rITR</SPAN></A></LI><LI class="toclevel-2 tocsection-142"><A href="#BioBrick_assembly_for_pSB1C3_lITR_pTERT"><SPAN class="tocnumber">17.10</SPAN><SPAN class="toctext">BioBrick assembly for pSB1C3_lITR_pTERT</SPAN></A></LI><LI class="toclevel-2 tocsection-143"><A href="#Picking_clones_of_pSB1C3_lITR_CMV_.28betaglobin.29_mVenus_hGH_rITR.2C_EGFP_C1_and_pMA_T_affibody"><SPAN class="tocnumber">17.11</SPAN><SPAN class="toctext">Picking clones of pSB1C3_lITR_CMV_(betaglobin)_mVenus_hGH_rITR, EGFP_C1 and pMA_T_affibody</SPAN></A></LI><LI class="toclevel-2 tocsection-144"><A href="#Trafo_evaluation_of_SDM_PstI_in_CD_and_inoculation"><SPAN class="tocnumber">17.12</SPAN><SPAN class="toctext">Trafo evaluation of SDM PstI in CD and inoculation</SPAN></A></LI></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="76.Labortag_01.08.2010"><P style="font-size:17px; background-color:#00dd77;">76.Labortag 01.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Picking_clones_of_pSB1C3_CMV"><P style="font-size:15px; background-color:#66bbFF;"><B>Picking clones of pSB1C3_CMV</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="font-size:13px; color:#66bbff;"><B>Comments:</B> Trafo was performed friday, and trafo plate were incubated over night. Clones need to be picked in order to perform Mini-Prep</P><UL><LI>Bacterial strain used: XL1-B</LI><LI>Two clones were picked of trafo plate</LI><LI>Inoculating of 10 mL DYT medium containing 10µL chloramphenicol</LI><LI> Put in 37°C room on rotary shaker</LI></UL><H3><SPAN class="mw-headline" id="77.Labortag_02.08.2010"><P style="font-size:17px; background-color:#00dd77;">77.Labortag 02.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Sequenzing_of_pSB1C3_RFC25_longlinker.2C_pSB1C3_RFC25_SEG.2C_pSB1C3_RFC25_GSAT"><P style="font-size:15px; background-color:#66bbFF;"><B>Sequenzing of pSB1C3_RFC25_longlinker, pSB1C3_RFC25_SEG, pSB1C3_RFC25_GSAT </B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><P style="font-size:13px; color:#66bbff;"><B>Comments:</B> Linkers (we got from Gerrit) we cloned in the RFC25-standard (pSB1C3_RFC25). Result looks good, linkers are in the vector pSb1C3_RFC25 <UL><LI>pSB1C3_RFC25_longlinker <B>P105 und P106</B></LI><LI>pSB1C3_RFC25_SEG <B>P107 und P108</B></LI><LI>pSB1C3_RFC25_GSAT <B>P109 und P110</B></LI></UL></P><H4><SPAN class="mw-headline" id="Sequencing_of_pGA14_Prefix-leftITR"><P style="font-size:15px; background-color:#66bbFF;"><B>Sequencing of pGA14_Prefix-leftITR</B></P></SPAN></H4><P><B>Investigator: Hanna</B>
Eventually GATC partially managed to sequence pGA14_Prefix-left ITR clone 1 and 2. 
The sequencing results were better than before but in the ITR region still not evaluable due to the strong secondary structures. 
It seemed that, despite of successful insertion of the RFC10-Prefix, something went wrong because there were 2 bases missing in the NotI restriction site. But by having a closer look, it became obvious that the Geneious misinterpreted the chromatogram and overlaid the peaks of the missing bases. </P><P>
For the first time sequencing of parts of the 3' end worked: The remaining NotI restriction site, which was already eliminated this week and also the &quot;backbone-PstI&quot; can be seen: </P><P>
Sequencing of very GC-rich regions :) :) :) </P><P>
Sequencing of clone 4 delivered that the ITR and the RFC10-Prefix were not inserted in the vector. The RFC25 standard of the multiple cloning site is still in the plasmid: </P><P>
Therefore clone 4 was dicarded from the plasmid- and glycerol stock. </P><H4><SPAN class="mw-headline" id="Repetition_of_test_digestion_pSB1C3_CFP_SDM_SspI_and_PvuII"><P style="font-size:15px; background-color:#66bbFF;"><B>Repetition of test digestion pSB1C3_CFP SDM SspI and PvuII </B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><UL><LI>buffer used: 2; Restriction-enzymes used: Enzyme 1: SspI ; Enzyme 2: PvuII
</LI></UL><UL><LI>Plasmids (all: ~ 700 ng/µL):</LI><UL><LI>pSB1C3_CFP <B>P51.1</B></LI><LI>pSB1C3_SDM_SspI <B>P125</B></LI><LI>pSB1C3_SDM_PvuII <B>P129</B></LI><LI>pSB1C3_SDM_PvuII <B>P131</B></LI></UL></UL><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>Mastermix</B></TD><TD align="left"><B>P125/µL</B></TD><TD align="left"><B>P51.1.1/µL</B></TD><TD align="left"><B>P129/µL</B></TD><TD align="left"><B>P131/µL</B></TD><TD align="left"><B>P51.1.2/µL</B></TD></TR><TR><TD align="left"> DNA  </TD><TD align="left"> -</TD><TD align="left"> 2,4</TD><TD align="left"> 2,3</TD><TD align="left"> 1,6</TD><TD align="left"> 5,3</TD><TD align="left">5,3
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 6</TD><TD align="left"> 1</TD><TD align="left"> 1</TD><TD align="left"> 1</TD><TD align="left"> 1</TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Buffer 2 (10x)  </TD><TD align="left"> 6</TD><TD align="left"> 1</TD><TD align="left"> 1</TD><TD align="left"> 1</TD><TD align="left"> 12</TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Enzyme 1 SspI (68)  </TD><TD align="left">-</TD><TD align="left"> 1</TD><TD align="left"> 1</TD><TD align="left"> -</TD><TD align="left"> -</TD><TD align="left"> -
</TD></TR><TR><TD align="left"> Enzyme 2 PvuII (50)  </TD><TD align="left"> -</TD><TD align="left"> -</TD><TD align="left"> -</TD><TD align="left"> 1</TD><TD align="left"> 1</TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> -</TD><TD align="left"> 4,6</TD><TD align="left"> 4,7</TD><TD align="left"> 5,4</TD><TD align="left"> 1,7</TD><TD align="left"> 1,7
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"> 12</TD><TD align="left"> 10</TD><TD align="left"> 10</TD><TD align="left"> 10</TD><TD align="left"> 10</TD><TD align="left"> 10
</TD></TR></TBODY></TABLE><UL><LI>Incubation: 65 minutes</LI></UL><P><B>Agarosegel</B>
0.45 g Agarose, 50 ml TEB (0,5 %), 3 µL GELRED, at 115 Volt, running time: 45 minutes
</P><UL><LI>Marker: GeneRuler ladder mix (6x)</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>P125
</TH><TH>P51.1 SspI
</TH><TH>-</TH><TH>P129
</TH><TH>P131
</TH><TH>P51.1 PvuII
</TH></TR><TR><TH>Lane
</TH><TD>7 µL
</TD><TD>10 µL
</TD><TD>10 µL
</TD><TD> -
</TD><TD>10 µL
</TD><TD>10 µL
</TD><TD>10 µL
</TD></TR></TBODY></TABLE><UL><LI><B>P125</B> isn't cut, therefore you see the bands for supercoiled, nicked and relaxed</LI><LI><B>P51.1</B> is cut once, therefore you can see a faster running band</LI><LI><B>P129</B> and <B>P131</B> should be cut once because of the PvuII in the CFP but there is another band we can't identify</LI><LI><B>P51.1</B> should be cut two times but there is also another unidentified band</LI><UL><LI>perhaps there is another PvuII in the vector we can't find (???)</LI></UL></UL><P style="font-size:13px; color:#66bbff;"><B>Comments:</B> SDM of SspI is ready! SDM of PvuII looks confusing because of the lower band. P 129 will be sequenced but this doesn't solve this problem. there has to be one more rs we don't know, perhaps. we will check it...anywise</P><H4><SPAN class="mw-headline" id="Mini-Prep_and_Test_digestion_of_pSB1C3_CMV"><P style="font-size:15px; background-color:#66bbff;"><B>Mini-Prep and Test digestion of pSB1C3_CMV  </B></P></SPAN></H4><P><B>Investigator: Kerstin</B><B>Nanodrop</B></P><UL><LI> pSB1C3_CMV clone1: 157,6 ng/µl <B>P145</B></LI><LI> pSB1C3_CMV clone2: 155,6 ng/µl <B>P146</B></LI></UL><P><B>Test digestion</B> 900ng DNA
</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">P145 /µl </TD><TD align="right">P146 /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 5,7</TD><TD align="right">5,8
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">1,5 </TD><TD align="right"> 1,5
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right"> 0,5 </TD><TD align="right"> 0,5
</TD></TR><TR><TD> Enzyme 1: XbaI </TD><TD align="right"> 0,5 </TD><TD align="right"> 1,5
</TD></TR><TR><TD> Enzyme 2: PstI </TD><TD align="right"> 0,5 </TD><TD align="right"> 0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right"> 5,3</TD><TD align="right"> 5,2
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"><B>15</B></TD><TD align="right"><B>15</B></TD></TR></TBODY></TABLE><P>
agarose gel: 1,5%, digestion: 90 minutes, 37°C 
</P><P><B>P145 is sent for sequencing with Reverse Primer (VR2)</B></P><P style="font-size:13px; color:#68bbff;"><I><B>Comments:</B></I>Results look good. CMV has the expected size (~ 662bp) </P><PRE><B>&lt;/p&gt;</B></PRE><H4><SPAN class="mw-headline" id="Sent_for_sequencing"><P style="font-size:15px; background-color:#66bbFF;"><B>Sent for sequencing</B></P></SPAN></H4><P><B>Investigator: Jessica</B><B>Hanna</B></P><UL><LI><B>P145</B> left ITR</LI><UL><LI>Primer: GATC_std_pQE-FP and GATC_std_m13-FP</LI></UL><LI><B>P150</B> right ITR</LI><UL><LI>Primer: GATC_std_pQE-FP and GATC_std_m13-FP</LI></UL></UL><P><B>Anissa</B></P><UL><LI><B>P136</B> pSB1C3_hgh</LI><UL><LI>Primer: O51_Reverse Primer (VR2)</LI></UL><LI><B>P139</B> pAAV_BamHI</LI><UL><LI>Primer: O36_Rep_1250_rev</LI></UL><LI><B>P143</B> pAAV_RC_SalI</LI><UL><LI>Primer: O53_Rep_1250_for</LI></UL><LI><B>P141</B> pGL3_QTERT</LI><UL><LI>Primer: GATC_std_pTal-FP</LI></UL><LI><B>P133</B> pSB1C3_betaGlobin</LI><UL><LI>Primer: O51_ReversePrimer (VR2)</LI></UL><LI><B>P145</B> pSB1C3_CMV</LI><UL><LI>Primer: O51_ReversePrimer (VR2)</LI></UL></UL><P><B>Jessica</B></P><UL><LI><B>P129</B> pSB1C3_SDM_PvuII</LI><UL><LI>Primer: GATC_std_pTeSp-2</LI></UL></UL><H4><SPAN class="mw-headline" id="Preparation_of_competent_E.coli"><P style="font-size:15px; background-color:#66bbFF;"><B>Preparation of competent E.coli</B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><UL><LI>15ml DYT was prepared with 15µl tetracycline and inoculate with XL1B</LI><LI>10ml DYT w/o antibiotics was prepared and inoculate with BL21 (just for glycerol stock, no competent cells)</LI></UL><P>both were incubate over night in 37°C room
</P><H4><SPAN class="mw-headline" id="Last_Mini-Prep_and_test_digestion_of_ITRs"><P style="font-size:15px; background-color:#66bbFF;"><B>Last Mini-Prep and test digestion of ITRs</B></P></SPAN></H4><P><B>Investigator: Hanna</B></P><P style="font-size:18px; font-weight: bold; color:#FF00FF;"><U>Plasmid Mini-Prep</U></P><UL><LI>new vector name: pGA14_RFC10_leftITR and pGA14_RFC10_rightITR
</LI></UL><P><U><B>Glycerol Stocks</B></U><B>pGA14_RFC10_leftITR</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD><TD align="left"><B>Clone 3</B></TD></TR><TR><TD align="left"><B>Bacteria strain</B></TD><TD align="left"> XL1blue </TD><TD align="left"> XL1blue  </TD><TD align="left"> XL1blue
</TD></TR><TR><TD align="left"><B>Plasmidname</B></TD><TD align="left"> pGA14_RFC10_leftITR </TD><TD align="left"> pGA14_RFC10_leftITR </TD><TD align="left"> pGA14_RFC10_leftITR
</TD></TR><TR><TD align="left"><B>Date</B></TD><TD align="left"> 2.8.10 </TD><TD align="left"> 2.8.10 </TD><TD align="left"> 2.8.10
</TD></TR><TR><TD align="left"><B>given glycerol-stock no.</B></TD><TD align="left"> B115 </TD><TD align="left"> B116 </TD><TD align="left"> B117
</TD></TR><TR><TD align="left"><B>given plasmid no.</B></TD><TD align="left"> P147 </TD><TD align="left"> P148 </TD><TD align="left"> P149
</TD></TR></TBODY></TABLE><P><B>pGA14_RFC10_rightITR</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD><TD align="left"><B>Clone 3</B></TD></TR><TR><TD align="left"><B>Bacteria strain</B></TD><TD align="left"> XL1blue </TD><TD align="left"> XL1blue  </TD><TD align="left"> XL1blue
</TD></TR><TR><TD align="left"><B>Plasmidname</B></TD><TD align="left"> pGA14_RFC10_rightITR </TD><TD align="left"> pGA14_RFC10_rightITR </TD><TD align="left"> pGA14_RFC10_rightITR
</TD></TR><TR><TD align="left"><B>Date</B></TD><TD align="left"> 2.8.10 </TD><TD align="left"> 2.8.10 </TD><TD align="left"> 2.8.10
</TD></TR><TR><TD align="left"><B>given glycerol-stock no.</B></TD><TD align="left"> B112 </TD><TD align="left"> B113 </TD><TD align="left"> B114
</TD></TR><TR><TD align="left"><B>given plasmid no.</B></TD><TD align="left"> P150 </TD><TD align="left"> P151 </TD><TD align="left"> P152
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color:#FF00FF;">Test digestion</P><UL><LI>buffer used: 4 ; Restriction-enzymes used: Enzyme 1 XbaI ; Enzyme 2 SpeI
</LI><LI>Plasmid: pGA14_RFC10_leftITR:
</LI><LI>Given Plasmid-Number: P147; DNA concentration: 240.22 ng/µL ;
</LI><LI>Given Plasmid-Number: P148; DNA concentration: 268.97 ng/µL ;
</LI><LI>Given Plasmid-Number: P149; DNA concentration: 234.16 ng/µL ;
</LI></UL><UL><LI>Plasmid: pGA14_RFC10_rightITR:
</LI><LI>Given Plasmid-Number: P150; DNA concentration: 221.71 ng/µL ;
</LI><LI>Given Plasmid-Number: P151; DNA concentration: 191.13 ng/µL ;
</LI><LI>Given Plasmid-Number: P152; DNA concentration: 215.14 ng/µL ;
</LI></UL><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>P147</B> Volume/µL </TD><TD align="left"><B>P150</B> Volume/µL
</TD></TR><TR><TD align="left"> DNA </TD><TD align="left"> 4.2 </TD><TD align="left"> 4.5
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 1 </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 1 </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Enzyme 1 XbaI </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5
</TD></TR><TR><TD align="left"> Enzyme 2 SpeI </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 2.8 </TD><TD align="left"> 2.5
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>P147</B></TD><TD align="left"><B>P150</B></TD></TR></TBODY></TABLE><UL><LI>Incubation: 1 h
</LI></UL><P style="font-size:15px; font-weight: bold; color:#FF00FF;">Agarose-Gel:</P><P>
0.83 g Agarose, 53 mL <B>TBE</B> (1.57%), 3 µL GELRED, at 115 Volt, running time: 45 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (5x)/µl
</TH><TH>Expected size 1 (Geneious)
</TH><TH>Expected size 2 (Geneious)
</TH></TR><TR><TD>P147
</TD><TD>10 µl
</TD><TD>2 µl
</TD><TD>157 bp
</TD><TD>2902 bp
</TD></TR><TR><TD>P150
</TD><TD>10 µl
</TD><TD>2 µl
</TD><TD>156 bp
</TD><TD>2903 bp
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker /µL
</TH><TH>Sample P147 /µL
</TH><TH>Sample P150 /µL
</TH></TR><TR><TH>Lane
</TH><TD>6.5
</TD><TD>12
</TD><TD>12
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color:#FF00FF;">Test digestion delivered positive results for both ITRs: The expected fragment sizes (156 resp. 157 bp) could be detected in the referring range between the 100 and 200 bp marker nucleotides:</P><P><B>Comment:</B> In order to verify the results, P147 and P150 were sent for sequencing. Because sequencing never worked before, they will be sequenced under special conditions this time.</P><H4><SPAN class="mw-headline" id="BioBrick_production:_mGMK_in_pSB1C3"><P style="font-size:15px; background-color:#66bbff;"><B>BioBrick production: mGMK in pSB1C3</B></P></SPAN></H4><P><B>Investigator: Stefan</B>
Backbone taken from: </P><UL><LI>pSB1C3_CFP (P51.2): 151,1 ng/µl</LI></UL><P>Insert taken from: </P><UL><LI>pAAV_RFC25_mGMK (P100): 411,3 ng/µl</LI></UL><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">pSB1C3_CFP(P51.2) /µl</TD><TD align="right">pAAV_RFC25_mGMK (P100) /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right">8 </TD><TD align="right"> 5
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2</TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right"> 2
</TD></TR><TR><TD>XbaI </TD><TD align="right">1 </TD><TD align="right"> 1
</TD></TR><TR><TD>AgeI </TD><TD align="right">1 </TD><TD align="right"> 1
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">6 </TD><TD align="right"> 9
</TD></TR><TR><TD><B>Total volume </B></TD><TD align="right">20 </TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><UL><LI> 1% Agarose gel
</LI><LI> 3 µl Gelred
</LI><LI>7 µl DNA-Ladder-Mix
</LI><LI> 115 Volt, running time: 50 minutes
</LI></UL><P>Afer 50 minutes a picture (see below) was taken. Because there was no clear pattern (two bands) for the pSB1C3 backbone, the band of the insert mGMK was cut out and the gel ran another 20 minutes. After that, there were still two bands. Both were cut out and it was continued with two possible backbones. They were labeled vector (lower) and vector (upper).</P><TABLE border="1"><TBODY><TR><TD><B>Sample/µl </B></TD><TD><B> Expected size/bp</B></TD></TR><TR><TD> Vector pSB1C3  </TD><TD align="right">2082
</TD></TR><TR><TD> Insert mGMK </TD><TD align="right">603
</TD></TR></TBODY></TABLE><UL><LI> weight of insert mGMK gel extract: 0,17 g
</LI><LI> weight of vector (lower) pSB1C3 gel extract: 0,08 g
</LI><LI> weight of vector (upper) pSB1C3 gel extract: 0,09 g
</LI></UL><P><B>Nanodrop</B></P><UL><LI> insert mGMK : 4,13 ng/µl
</LI><LI> vector (lower) pSB1C: 5,47 ng/µl
</LI><LI> vector (upper) pSB1C: 0,60 ng/µl =&gt; for calculation of ligation approach calculated with 1 ng/µl
</LI></UL><P><B>Ligation</B></P><UL><LI>vector (lower) : insert - 4,16 µl : 4,84 µl
</LI><LI>vector (upper) : insert - 7,43 µl : 1,57 µl</LI></UL><P><B>Transformation</B>
Two approaches were prepared, one for each vector and put in 37 °C room overnight.
</P><H4><SPAN class="mw-headline" id="Sequencing_results_of_SDM_Rep_68_.26_78"><P style="font-size:15px; background-color:#66bbFF;"><B>Sequencing results of SDM Rep 68 &amp; 78</B></P></SPAN></H4><P><B>Investigator: Hanna and Volker</B>
The sequencing results delivered that the PstI restriction sites of Rep 68 and Rep 78 were successfully deleted via side-directed mutageneis:
</P><CENTER><TABLE class="gallery" cellspacing="0" cellpadding="0"><P><CAPTION>Sequencing results of SDM Rep 68 &amp; 78</CAPTION></P><TBODY><TR><TD><DIV class="gallerybox" style="width: 535px;"><DIV class="gallerytext"><P>Rep 68
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 535px;"><DIV class="gallerytext"><P>Rep 78
</P></DIV></DIV></TD></TR></TBODY></TABLE></CENTER><H3><SPAN class="mw-headline" id="78.Labortag_03.08.2010"><P style="font-size:17px; background-color:#00dd77;">78.Labortag 03.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Results_of_sequencing_pSB1C3_CFP_SDM_PvuII"><P style="font-size:15px; background-color:#66bbff;"><B>Results of sequencing pSB1C3_CFP_SDM_PvuII</B></P></SPAN></H4><P><B>Investigator: Jessica</B><B>SDM of PvuII:</B></P><P><B>PvuII is succesfully deleted in the vector pSB1C3_RFC25_CFP</B></P><H4><SPAN class="mw-headline" id="Repetition_of_the_quickchange_site-directed_mutagenesis_of_pAAV_RC_1.1_SalI"><P style="font-size:15px; background-color:#66bbff;"><B>Repetition of the quickchange site-directed mutagenesis of pAAV_RC_1.1_SalI</B></P></SPAN></H4><P><B>Investigator: Kerstin, Anissa</B></P><P style="font-size:13px; color:#68bbff;"><I><B>Comments:</B></I> Sequenzing revealed no mutagenesis of SalI in pAAV_RC_1.1_SalI, so SDM now will be repeated.</P><P>PCR-reaction:
</P><TABLE border="1"><TBODY><TR><TD><B>Volume / µl</B></TD><TD align="right"><B>ingredients</B></TD><TD align="right"><B>recommended /µl</B></TD></TR><TR><TD> 2,5  </TD><TD align="right">10X Pfu Ultra II buffer </TD><TD align="right"> 2,5
</TD></TR><TR><TD> 4,18 </TD><TD align="right">template (~10 ng)</TD><TD align="right"> 4,18 of 1:100 dilution
</TD></TR><TR><TD> 0,58</TD><TD align="right">forward primer: O68</TD><TD align="right"> 62,5 ng
</TD></TR><TR><TD>0,58 </TD><TD align="right">reverse primer: O69 </TD><TD align="right"> 62,5 ng
</TD></TR><TR><TD>0,5</TD><TD align="right">dNTP</TD><TD align="right"> 250 µM each dNTP
</TD></TR><TR><TD>16,16</TD><TD align="right">H<SUB>2</SUB>O </TD></TR><TR><TD>0,5</TD><TD align="right">PfuUltra II fusion (1.25) </TD></TR></TBODY></TABLE><P>
PCR program:
</P><TABLE border="1"><TBODY><TR><TD><B>Rounds</B></TD><TD align="right"><B>temperature/ °C</B></TD><TD align="right"><B>Time</B></TD></TR><TR><TD>1 </TD><TD align="right">95 </TD><TD align="right"> 2 minutes
</TD></TR><TR><TD> 20 </TD><TD align="right">95</TD><TD align="right">30 seconds
</TD></TR><TR><TD>20 </TD><TD align="right">80,4 </TD><TD align="right"> 1 minute
</TD></TR><TR><TD>20</TD><TD align="right">68</TD><TD align="right"> 7,5 minutes
</TD></TR></TBODY></TABLE><P>
Plasmid was transformed into BL21 cells. Clones have to be picked tomorrow.
</P><P style="font-size:13px; color:#ff0000;"><I><B>Comments:</B></I>NO CLONES, trafo didn't work, because annealing temperature was totally to high! (80,4 °C instead of 55°C) annealing temperature should always be 55°C (in case of troubleshooting temperature can be increased up to MAX. 68°C !)</P><H4><SPAN class="mw-headline" id="PCR_for_Biobrickproduction_of_Rep_40.2C_52.2C_68.2C_78_and_APP"><P style="font-size:15px; background-color:#66bbff;"><B>PCR for Biobrickproduction of Rep 40, 52, 68, 78 and APP</B></P></SPAN></H4><P><B>Investigators: Volker, Anna</B></P><P> 	
Aim of the experiment:
We wanted to produce biobricks from  the expression plasmids Rep40ex, Rep52 ex, the Rep 68 and 78 in which the PstI at position 310 was silenced and the AAP. This was performed by a PCR with primes that annealed to the ends of the later biobrick but also contained restriction sites that can be cut afterwards and cloned into pSB1C3.
</P><UL><LI>Plasmids used as template:
</LI></UL><P>Rep_68_ex (p119):   c = 470,6 ng/µl 
Rep_78_(p122):      c = 201,08 ng/µl 
Rep_40_(p22.2):     c = 673,1 ng/µl 
Rep_52_(p23.2):     c = 532,22 ng/µl 
pAAV-RC containing AAP ORF (p50):    c = 378,5 ng/µl </P><UL><LI>Primer used:
</LI></UL><P>For Rep_40_ex: Praefix_40_52_ex &amp; Suffix_40_68_ex
For Rep_52_ex: Praefix_40_52_ex &amp; Suffix_52_78_ex
For Rep_68_ex: Praefix_68_78_ex &amp; Suffix_40_68_ex
For Rep_78_ex: Praefix_68_78_ex &amp; Suffix_52_78_ex
For AAP_ex: Praefix_AAP_ex &amp; Suffix_AAP_ex
</P><UL><LI><B>PCR:</B></LI></UL><P>(was performed following the standard protocol)
</P><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD align="right"><B>Volume / µl</B></TD><TD align="right"><B>Rep68</B></TD><TD align="right"><B>Rep78</B></TD><TD align="right"><B>Rep40</B></TD><TD align="right"><B>Rep52</B></TD><TD align="right"><B>AAP</B></TD></TR><TR><TD> 5X Phusion HF buffer </TD><TD align="right">10 </TD></TR><TR><TD> 10 mM dNTP mix</TD><TD align="right">1</TD></TR><TR><TD> forward primer: </TD><TD align="right">2,5</TD></TR><TR><TD> reverse primer: </TD><TD align="right">2,5</TD></TR><TR><TD> DNA Template</TD><TD align="right">***</TD><TD align="right">4,2 µl </TD><TD align="right">1 µl </TD><TD align="right">2,9 µl</TD><TD align="right">3,8 µl</TD><TD align="right">5,3 µl
</TD></TR><TR><TD> Phusion Polymerase</TD><TD align="right">0,5</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">*** </TD><TD align="right"> 28,3 µl</TD><TD align="right"> 31,5 µl</TD><TD align="right"> 29,6 µl</TD><TD align="right"> 28,7 µl</TD><TD align="right">27,2 µl
</TD></TR><TR><TD>Total volume</TD><TD align="right">50
</TD></TR></TBODY></TABLE><P>PCR program:
</P><TABLE border="1"><TBODY><TR><TD><B>PCR Program</B></TD><TD align="right"><B>temperature/ °C</B></TD><TD align="right"><B>Time</B></TD><TD align="right"><B>Rep40</B></TD><TD align="right"><B>Rep50</B></TD><TD align="right"><B>Rep68</B></TD><TD align="right"><B>Rep78</B></TD><TD align="right"><B>AAP</B></TD></TR><TR><TD>1</TD><TD align="right">98 </TD><TD align="right">1min</TD></TR><TR><TD>2</TD><TD align="right">98 </TD><TD align="right">15s</TD></TR><TR><TD>8x</TD><TD align="right">*** </TD><TD align="right">25s</TD><TD align="right">63°C</TD><TD align="right">62°C</TD><TD align="right">63°C</TD><TD align="right">62°C</TD><TD align="right">64°C
</TD></TR><TR><TD>3 </TD><TD align="right">72</TD><TD align="right">***</TD><TD align="right">15s  </TD><TD align="right">18s</TD><TD align="right">24s </TD><TD align="right">27s </TD><TD align="right"> 10s
</TD></TR><TR><TD>4 </TD><TD align="right">98</TD><TD align="right"> 15s</TD></TR><TR><TD>17x</TD><TD align="right">***</TD><TD align="right">25s</TD><TD align="right">68°C</TD><TD align="right">66°C</TD><TD align="right">68°C</TD><TD align="right">64°C</TD><TD align="right">68°C
</TD></TR><TR><TD>5</TD><TD align="right">72</TD><TD align="right">***</TD><TD align="right">15s</TD><TD align="right">18s</TD><TD align="right">24s</TD><TD align="right">27s</TD><TD align="right">10s
</TD></TR><TR><TD>6x</TD><TD align="right">72</TD><TD align="right">5min</TD></TR><TR><TD>Hold</TD><TD align="right">4</TD></TR></TBODY></TABLE><P>Five µl of PCR-product were used for an analytical gel to see if the PCR worked. </P><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>BioBrick production for Rep and AAP</CAPTION><TBODY><TR><TD><DIV class="gallerybox" style="width: 535px;"><DIV class="gallerytext"><P>Alalytical gel
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 535px;"><DIV class="gallerytext"><P>Praeparative gel
</P></DIV></DIV></TD></TR></TBODY></TABLE><P style="color:red;">Comment: The result is that the PCR worked for the longer Rep proteins (68&amp;78) with which a Quick-change was performed to remove the PstI(310) but not with the constructs for the smaller Rep variants (40&amp;52) that were recieved from PD Kleinschmidt. As the longer Rep variants the AAP PCR Reaction also resulted in a PCR-product of the expected size. Unfortunately there were a secondary band of ~90bp for Rep_68_ex. For this reason we performed a praeparative gel and decided to cut the bands of all PCR products.</P><P>The bands marked in the gel picture were cut and a gel extraction was performed.
</P><P style="font-size:15px; font-weight: bold; color: blue;">Gel extraction</P><P>
Gel measurement:
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Sample</B></TD><TD align="left"><B>Weight</B></TD><TD align="left"><B>Volume</B></TD><TD align="left"><B>Concentration</B></TD></TR><TR><TD align="left"> Rep_68_ex
</TD><TD align="left"> 0,1 g
</TD><TD align="left"> 20 µl
</TD><TD align="left"> 17,6 ng/µl
</TD></TR><TR><TD align="left"> Rep_78_ex
</TD><TD align="left"> 0,15g
</TD><TD align="left"> 20 µl
</TD><TD align="left"> 80,6 ng/µl
</TD></TR><TR><TD align="left"> AAP_ex
</TD><TD align="left"> 0,19 g
</TD><TD align="left"> 20µl
</TD><TD align="left"> 77,78 ng/µl
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Continuation_of_preparation_of_competent_E.coli"><P style="font-size:15px; background-color:#66bbFF;"><B>Continuation of preparation of competent E.coli</B></P></SPAN></H4><P><B>Investigator: Jessica</B>
preparation of competent XL1blue was finished according to the standard protocol
</P><UL><LI>aliquots of 60µl (to use for 1 trafo) are stored in -80°C freezer</LI></UL><H4><SPAN class="mw-headline" id="Sequencing_of_pSB1C3"><P style="font-size:15px; background-color:#66bbFF;"><B>Sequencing of pSB1C3</B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><P style="font-size:13px; color:#68bbff;"><I><B>Comments:</B></I> We sent for sequencing  <B>pSB1C3_RFC25_longlinker (P105)</B> to check the strange result of the test digestion of pSB1C3_SDM_PvuII(01.08.10. 
pSB1C3_SDM_PvuII is too long for sequencing wherefore we sent pSB1C3_RFC25_longlinker because the longlinker just have 54kb (bp, nicht kb (Volker) (necessary is just the backbone we don't have.</P><UL><LI>Primer:</LI><UL><LI>RESgen-241698</LI><LI>Reverse primer (VR2)</LI><LI>o_F1-fw (from Gerrit)</LI></UL></UL><H4><SPAN class="mw-headline" id="Design_of_Primer_for_p5_Promoter_WT_and_TATA-less_BioBrick_production"><P style="font-size:15px; background-color:#66bbFF;"><B>Design of Primer for p5 Promoter WT and TATA-less BioBrick production</B></P></SPAN></H4><P><B>Investigator: Hanna</B></P><H3><SPAN class="mw-headline" id="79.Labortag_04.08.2010"><P style="font-size:17px; background-color:#00dd77;">79.Labortag 04.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Cloning_of_right_ITR_and_left_ITR_into_pSB1C3_RFC25_CFP"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of right ITR and left ITR into pSB1C3_RFC25_CFP</B></P></SPAN></H4><P><B>Investigator: Patrick</B>
Intention: Get biobricks ready. 
pGA14_lITR_RFC10 (P147), pGA14_rightITR (P150) and pSB1C3_RFC25_CFP (P51.1) were digested with EcoRI and PstI (Buffer 4) according to the standard protocol. Digestion Time: 80 minutes.

GelRun: 1% Agarose Gel. Expected results (from left to right):
</P><UL><LI>pSB1C3_RFC25_CFP: about 2100 bp and 800 bp.
</LI><LI>pGA14_leftITR_RFC10: the size of the insert should be 135 bp.
</LI><LI>pGA14_rightITR_RFC10: the size of the insert should be 138bp.
</LI></UL><P>
The mutual vector (now without CFP) and inserts (left ITR and right ITR) were cut out. The gelextraction was performed according to the standard protocol. DNA concentration of the extracts:
</P><UL><LI> pSB1C3_RFC25_CFP: 2,9 ng/µl
</LI><LI> left ITR: 1,4 ng/µl
</LI><LI> right ITR: 2,0 ng/µl
</LI></UL><P>
The Quick Ligation was not performed according to the standard protocol:
</P><UL><LI> left ITR + pSB1C3: 5µl Buffer, 1 µl Quick-Ligase, 2,5 µl left ITR, 1,5 µl pSB1C3_RFC25.
</LI><LI> right ITR + pSB1C3: 5µl Buffer, 1 µl Quick-Ligase, 2,5 µl right ITR, 1,5 µl pSB1C3_RFC25.
</LI></UL><P>
Transformation: performed according to the standard protocol (BL21). The cells were plated on a agar plate with chloramphenicol. The clones will be picked tomorrow.
</P><H4><SPAN class="mw-headline" id="Cloning_of_SDM_SspI_and_SDM_PvuII"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of SDM SspI and SDM PvuII</B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><UL><LI>Vector: name: pSB1C3_SDM_SspI <B>P125</B></LI><LI>Insert: name: pSB1C3_SDM_PvuII <B>P129</B></LI><LI>new vector name: pSB1c3_SDM_SspI/PvuII <B>P157</B></LI><LI>buffer used: 4 ; Restriction-enzymes used: Enzyme 1 (no. Lab: 141) BstI ; Enzyme 2 (no.Lab: 144) BpmI 
</LI><LI>DNA concentration (vector): 321,9 ng/µl ; DNA concentration (insert): 308,4 µg/µl
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume of pSB1C3_SDM_SspI /µl</B></TD><TD align="right"><B>volume of pSB1C3_SDm_PvuII /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">4,66 </TD><TD align="right">4,87
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right">2
</TD></TR><TR><TD>Enzyme BstI (no.Lab:141)</TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>Enzyme BpmI (no.Lab:144)</TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">9,34 </TD><TD align="right">9,13
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 20</TD><TD align="right">20
</TD></TR></TBODY></TABLE><P>
0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 130 Volt, running time:45
</P><P><B>Loading plan for agarose gel</B>: 
Marker used: GeneRuler ladder mix (Fermentas)</P><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>Sample 125, 20µl
</TH><TH>Sample 129, 20µl
</TH></TR><TR><TH>Lane
</TH><TD>1
</TD><TD>3
</TD><TD>5
</TD></TR></TBODY></TABLE><P><B>Results</B>: digesttemperature of BstI is 55°C, plasmid was just digested at 37°C... :( (</P><H4><SPAN class="mw-headline" id="2nd_Repetition_of_Quickchange_site-directed_mutagenesis_of_pAAV_RC_1.1_SalI"><P style="font-size:15px; background-color:#66bbff;"><B>2nd Repetition of Quickchange site-directed mutagenesis of pAAV_RC_1.1_SalI</B></P></SPAN></H4><P><B>Investigator: Kerstin, Anissa</B></P><P style="font-size:13px; color:#68bbff;"><I><B>Comments:</B></I> Last trafo didn't work. Annealing temperature was to high (80,4°C instead of 55°C) </P><P>Two approaches were made (short and long PCR:
</P><P>PCR-reaction:
</P><TABLE border="1"><TBODY><TR><TD><B>Volume / µl</B></TD><TD align="right"><B>ingredients</B></TD><TD align="right"><B>recommended /µl</B></TD></TR><TR><TD> 2,5  </TD><TD align="right">10X Pfu Ultra II buffer </TD><TD align="right"> 2,5
</TD></TR><TR><TD> 1,83 </TD><TD align="right">template (~10 ng): p139 (c: 546,97 ng/µl)</TD><TD align="right"> 1,83µl of 1:100 dilution
</TD></TR><TR><TD> 0,58</TD><TD align="right">forward primer: O68</TD><TD align="right"> 62,5 ng
</TD></TR><TR><TD>0,58 </TD><TD align="right">reverse primer: O69 </TD><TD align="right"> 62,5 ng
</TD></TR><TR><TD>0,5</TD><TD align="right">dNTP</TD><TD align="right"> 250 µM each dNTP
</TD></TR><TR><TD>18,51</TD><TD align="right">H<SUB>2</SUB>O </TD></TR><TR><TD>0,5</TD><TD align="right">PfuUltra II fusion (1.25) </TD></TR></TBODY></TABLE><P>
PCR program (long):
</P><TABLE border="1"><TBODY><TR><TD><B>Cycles</B></TD><TD align="right"><B>temperature/ °C</B></TD><TD align="right"><B>Time</B></TD></TR><TR><TD>1 </TD><TD align="right">95 </TD><TD align="right"> 2 minutes
</TD></TR><TR><TD> 20 </TD><TD align="right">95</TD><TD align="right">30 seconds
</TD></TR><TR><TD>20 </TD><TD align="right">55 </TD><TD align="right"> 1 minute
</TD></TR><TR><TD>20</TD><TD align="right">68</TD><TD align="right"> 7,5 minutes
</TD></TR></TBODY></TABLE><P>
PCR program (short):
</P><TABLE border="1"><TBODY><TR><TD><B>Cycles</B></TD><TD align="right"><B>temperature/ °C</B></TD><TD align="right"><B>Time</B></TD></TR><TR><TD>1 </TD><TD align="right">95 </TD><TD align="right"> 2 minutes
</TD></TR><TR><TD> 20 </TD><TD align="right">95</TD><TD align="right">30 seconds
</TD></TR><TR><TD>20 </TD><TD align="right">55 </TD><TD align="right"> 1 minute
</TD></TR><TR><TD>20</TD><TD align="right">68</TD><TD align="right"> 4 minutes
</TD></TR></TBODY></TABLE><P>Plasmids were transformed into BL21 cells. Clones have to be picked tomorrow.
</P><H4><SPAN class="mw-headline" id="Mini-Prep_of_pSB1C3_mGMK"><P style="font-size:15px; background-color:#66bbff;"><B>Mini-Prep of pSB1C3_mGMK</B></P></SPAN></H4><P><B>Investigator: Chris W., Bea</B></P><P>
Vector name: pSB1C3_mGMK upper band 1.1 and pSB1C3_mGMK upper band 1.2 and pSB1C3_mGMK lower band 2.1 and pSB1C3_mGMK lower band 2.2
</P><P>Mini-Prep following the standart Protokoll
</P><LI>P153 = pSB1C3_mGMK upper band 1.1 = 248,3 ng/µl
</LI><LI>P154 = pSB1C3_mGMK upper band 1.2 = 251,6 ng/µl
</LI><LI>P155 = pSB1C3_mGMK lower band 1.1 = 263 ng/µl 
</LI><LI>P156 = pSB1C3_mGMK lower band 1.2 = 235,2 ng/µl
<B>Test digestion:</B></LI><LI>Perfomed with 15 µL of total volume with all four clones</LI><TABLE border="1" cellpadding="0" cellspacing="0" width="621" style="border-collapse: collapse;table-layout:fixed;width:466pt"><TBODY><TR style="mso-height-source:userset;height:21.75pt"><TD height="29" class="xl6724191" width="143" style="height:21.75pt;width:107pt"> </TD><TD class="xl6824191" width="131" style="border-left:none;width:98pt">Mastermix/µL</TD><TD class="xl6824191" width="87" style="border-left:none;width:65pt">P153/µL</TD><TD class="xl6824191" width="84" style="border-left:none;width:63pt">P154/µL</TD><TD class="xl6824191" width="90" style="border-left:none;width:68pt">P155/µL</TD><TD class="xl6824191" width="86" style="border-left:none;width:65pt">P156/µL</TD></TR><TR style="mso-height-source:userset;height:21.75pt"><TD height="29" class="xl6624191" width="143" style="height:21.75pt;border-top: none;width:107pt"><SPAN lang="EN-US" style="mso-ansi-language:EN-US">DNA</SPAN></TD><TD class="xl6524191" style="border-top:none;border-left:none">-</TD><TD class="xl6524191" style="border-top:none;border-left:none">3,5</TD><TD class="xl6524191" style="border-top:none;border-left:none">3,5</TD><TD class="xl6524191" style="border-top:none;border-left:none">3,5</TD><TD class="xl6524191" style="border-top:none;border-left:none">3,5</TD></TR><TR style="mso-height-source:userset;height:21.75pt;mso-yfti-irow: 1"><TD height="29" class="xl6624191" width="143" style="height:21.75pt;border-top: none;width:107pt"><SPAN lang="EN-US" style="mso-ansi-language:EN-US">BSA (10x)</SPAN></TD><TD class="xl6524191" style="border-top:none;border-left:none">7,5</TD><TD rowspan="4" class="xl6524191" style="border-top:none">4</TD><TD rowspan="4" class="xl6524191" style="border-top:none">4</TD><TD rowspan="4" class="xl6524191" style="border-top:none">4</TD><TD rowspan="4" class="xl6524191" style="border-top:none">4</TD></TR><TR style="mso-height-source:userset;height:21.75pt;mso-yfti-irow: 2"><TD height="29" class="xl6624191" width="143" style="height:21.75pt;border-top: none;width:107pt"><SPAN lang="EN-US" style="mso-ansi-language:EN-US">Buffer
  4(10x)</SPAN></TD><TD class="xl6524191" style="border-top:none;border-left:none">7,5</TD></TR><TR style="mso-height-source:userset;height:21.75pt;mso-yfti-irow: 3"><TD height="29" class="xl6624191" width="143" style="height:21.75pt;border-top: none;width:107pt"><SPAN lang="EN-US" style="mso-ansi-language:EN-US">Enzyme
  XbaI</SPAN></TD><TD class="xl6524191" style="border-top:none;border-left:none">2,5</TD></TR><TR style="mso-height-source:userset;height:21.75pt;mso-yfti-irow: 4"><TD height="29" class="xl6624191" width="143" style="height:21.75pt;border-top: none;width:107pt"><SPAN lang="EN-US" style="mso-ansi-language:EN-US">Enzyme
  AgeI</SPAN></TD><TD class="xl6524191" style="border-top:none;border-left:none">2,5</TD></TR><TR style="mso-height-source:userset;height:21.75pt;mso-yfti-irow: 5;mso-yfti-lastrow:yes"><TD height="29" class="xl6624191" width="143" style="height:21.75pt;border-top: none;width:107pt"><SPAN lang="EN-US" style="mso-ansi-language:EN-US">H2O</SPAN></TD><TD class="xl6524191" style="border-top:none;border-left:none">-</TD><TD class="xl6524191" style="border-top:none;border-left:none">7,5</TD><TD class="xl6524191" style="border-top:none;border-left:none">7,5</TD><TD class="xl6524191" style="border-top:none;border-left:none">7,5</TD><TD class="xl6524191" style="border-top:none;border-left:none">7,5</TD></TR><TR style="mso-height-source:userset;height:21.75pt"><TD height="29" class="xl6624191" width="143" style="height:21.75pt;border-top: none;width:107pt">Total volume</TD><TD class="xl6524191" style="border-top:none;border-left:none">15</TD><TD class="xl6524191" style="border-top:none;border-left:none">15</TD><TD class="xl6524191" style="border-top:none;border-left:none">15</TD><TD class="xl6524191" style="border-top:none;border-left:none">15</TD><TD class="xl6524191" style="border-top:none;border-left:none">15</TD></TR></TBODY></TABLE><UL><LI>All four samples were loaded on a 1% agarose gel</LI><LI>P153 = pSB1C3_mGMK upper band 1.1 </LI><LI>P154 = pSB1C3_mGMK upper band 1.2 </LI><LI>P155 = pSB1C3_mGMK lower band 1.1</LI><LI>P156 = pSB1C3_mGMK lower band 1.2</LI></UL><P><B> Sequencing</B>: 
</P><UL><LI> Plasmid used: P156: pSB1C3_mGMK lower band 1.2</LI><LI> Primer used: VR-2</LI><LI> Tube name: Bea_1</LI></UL><H4><SPAN class="mw-headline" id="cell_culture"><P style="font-size:15px; background-color:#66bbff;"><B> cell culture </B></P></SPAN></H4><P><B>Investigator: Adrian</B><B> Plan for the next virus production procedures </B>
The Motivation: investigation of the influence of different GOI amounts.

The Plan: keep rep/cap and pHelper amount stable (3,3 µg each =&gt; 6,6 µg), differ the GOI (YFP) amount in each stock. 

Viral Stocks:
</P><UL><LI> 3,3 µg YFP + 3,3 µg pHelper + 3,3  rep/cap
</LI><LI> 10 µg YFP + 3,3 µg pHelper + 3,3  rep/cap
</LI><LI> 20 µg YFP + 3,3 µg pHelper + 3,3  rep/cap
</LI></UL><UL><LI>3,3 µg pHelper + 3,3  rep/cap (<B>without GOI !!!</B>)
</LI></UL><UL><LI>one GMK_TK clone (with the confirmed sequence) + 3,3 µg pHelper + 3,3  rep/cap
</LI></UL><P><B>Transduction plan</B></P><LI>five 6-well-plates wille be transduced
<TABLE border="1"><TBODY><TR><TD>A
</TD><TH>150µl AAV stock 1
</TH><TH>300µl AAV stock 1
</TH><TH>control no Virus
</TH></TR><TR><TH>B
</TH><TD>150µl AAV stock 2
</TD><TD>300µl AAV stock 2
</TD><TD>150µl AAV without GOI
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Repetition_of_cloning_of_SDM_SspI_and_SDM_PvuII"><P style="font-size:15px; background-color:#66bbff;"><B>Repetition of cloning of SDM SspI and SDM PvuII</B></P></SPAN></H4><P><B>Investigator: Jessica</B></P></LI><LI>Vector: name: pSB1C3_SDM_SspI <B>P125</B></LI><LI>Insert: name: pSB1C3_SDM_PvuII <B>P129</B></LI><LI>new vector name: pSB1C3_RFC25_SDM_SspI/PvuII <B>P157</B></LI><LI>buffer used: 4 ; Restriction-enzymes used: Enzyme 1 (no. Lab: 141) BtsI ; Enzyme 2 (no.Lab: 144) BpmI 
</LI><LI>DNA concentration (vector): 321,9 ng/µl ; DNA concentration (insert): 308,4 µg/µl
</LI><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume of pSB1C3_SDM_SspI /µl</B></TD><TD align="right"><B>volume of pSB1C3_SDm_PvuII /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">4,66 </TD><TD align="right">4,87
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right">2
</TD></TR><TR><TD>Enzyme ClaI (no.Lab:152)</TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>Enzyme BpmI (no.Lab:144)</TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">9,34 </TD><TD align="right">9,13
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 20</TD><TD align="right">20
</TD></TR></TBODY></TABLE><P>incubation time: 1,5h

0,5 g Agarose,50 ml TBE (1%), 3 µl GELRED (gel was shared with Anna) , at  Volt, running time:
</P><P><B>Loading plan for agarose gel</B>: 
Marker used: GeneRuler ladder mix (Fermentas)</P><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>Sample 125, 20µl
</TH><TH>Sample 129, 20µl
</TH></TR><TR><TH>Lane
</TH><TD>1
</TD><TD>3
</TD><TD>5
</TD></TR></TBODY></TABLE><P>This appproach also didn't work, same result. the guess is that BpmI doesn't work anymore. will be repeated by Chris W. on 05.08.10. the result will show that the guess is right and BpmI from labstock is changed out with a new one.
</P><H4><SPAN class="mw-headline" id="Test_transformation_of_XL1_competent_cells"><P style="font-size:15px; background-color:#66bbff;"><B>Test transformation of XL1 competent cells</B></P></SPAN></H4><P><B>Investigator: Kira</B></P><P>Test transformation was performed in order to test the efficiency of produced chemical competent XL1 blue cells.
</P><P>2 plates were prepared: one with 50 pg and the second with 100 pg pUC 18 plasmid (50 pg/㎕)
50 pg plate: 1 ㎕ pUC + 50 ㎕ XL1B cells
100 pg plate: 2 ㎕ pUC + 50 ㎕ XL1B cells
</P><P>Transformation was performed according to the standard protocol and the plates were incubated at 37 C.
</P><H4><SPAN class="mw-headline" id="Sequencing_results_of_ITRs"><P style="font-size:15px; background-color:#66bbff;"><B>Sequencing results of ITRs</B></P></SPAN></H4><P><B>Investigator: Hanna</B>
8 sequencing files were analyzed &quot;per hand&quot; base by base. Alignments (will be inserted soon) delivered that the right ITR is 100% OK and was successfully converted into the RFC10 BioBrick standard. </P><P>The sequencing and alignments of the left ITR delivered that a 15 bp fragment in the middle of the sequence is lacking. Therefore further test digestions have to be performed. Nevertheless also this ITR was successfully converted into the RFC10 BioBrick standard. Secondary structure analysis showed that the stemloop-structure will be nevertheless forming. We will try to test whether the referring fragment is also missing in the pAAV_MCS. If it's also lacking there we will continue with this ITR and test whether it functions.
</P><H4><SPAN class="mw-headline" id="Cloning_of_Rep68.2C_Rep78_and_AAP_into_pSB1C3"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of Rep68, Rep78 and AAP into pSB1C3</B></P></SPAN></H4><P><B>Investigator: Anna</B></P><P style="font-size:13px; color:#68bbff;"><I><B>Comments</B></I>: The PCR of Rep 40/52 didn't work (see 03.08), whereas the PCR of Rep 68/78 and AAP was succesful. Ligation was done with two different samples of pSB1C3 (see agarose gel). Samples from digestion and from ligation are stored in the 4°C freezer.</P><UL><LI>Digestion of PCR products and vector:
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>PCR product /µl</B></TD><TD align="right"><B>vector /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">19 </TD><TD align="right">10
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">3 </TD><TD align="right"> 3
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">3 </TD><TD align="right">3
</TD></TR><TR><TD>Enzyme <B>XbaI </B></TD><TD align="right">1 </TD><TD align="right">1</TD></TR><TR><TD>Enzyme <B>SpeI </B></TD><TD align="right">1 </TD><TD align="right">1</TD></TR><TR><TD>H2O</TD><TD align="right">3 </TD><TD align="right">12
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 30</TD><TD align="right">30
</TD></TR></TBODY></TABLE><P style="font-size:13px; color:#68bbff;"><I><B>Comments</B></I>: Digestion was done with XbaI and SpeI, it has to be checked if the inserts are cloned into the vector in the right orientation. </P><UL><LI>Purification of Rep68, 78 and AAP:
</LI></UL><P>For the purification 95 µl of buffer PBI was used.
</P><P>c(Rep68)= 17,6 ng/µl 
c(Rep78)= 80,60 ng/µl 
c(AAP)= 77,78 ng/µl
</P><UL><LI>Gelextraction of pSB1C3_RFC25_CFP:
</LI></UL><P>0,5 g Agarose,50 ml TBE (1%), 3 µl GELRED , at 115 Volt, running time:55 
5µl loading dye (6x) for the sample, Marker: GeneRuler ladder mix (Fermentas)
</P><P>
c(pSB1C3)= 2,87 ng/µl 
c(pSB1C3_2)= 9,46 ng/µl
</P><UL><LI>Quickligation of PCR products and vector:
</LI></UL><P>For the Ligation 10µl buffer (2x) and 1µl Quickligase were used.
</P><TABLE border="1"><TBODY><TR><TD align="right"><B>Sample-no.</B></TD><TD align="right"><B>vector /µl</B></TD><TD align="right"><B>insert /µl</B></TD></TR><TR><TD> pSB1C3 + Rep68 </TD><TD align="right">1.1 </TD><TD align="right">6,51 </TD><TD align="right">2,4
</TD></TR><TR><TD> pSB1C3_2 + Rep68 </TD><TD align="right">1.2 </TD><TD align="right">3,9 </TD><TD align="right"> 5,02
</TD></TR><TR><TD> pSB1C32 + Rep78 </TD><TD align="right">2.1 </TD><TD align="right">8,2 </TD><TD align="right">0,8
</TD></TR><TR><TD> pSB1C3_2 + Rep78 </TD><TD align="right">2.2 </TD><TD align="right">6,82 </TD><TD align="right">2,18
</TD></TR><TR><TD> pSB1C3 + AAP </TD><TD align="right">3.1 </TD><TD align="right">8,71 </TD><TD align="right">0,92
</TD></TR><TR><TD> pSB1C3_2 + AAP  </TD><TD align="right">3.2 </TD><TD align="right">8,1 </TD><TD align="right">0,9
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 9</TD></TR></TBODY></TABLE><UL><LI>Transformation:
</LI></UL><P>The transformation was done following the standard protocol using B21 cells.
</P><H3><SPAN class="mw-headline" id="80.Labortag_05.08.2010"><P style="font-size:17px; background-color:#00dd77;">80.Labortag 05.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="New_LB_Agar_was_prepared."><P style="font-size:15px; background-color:#66bbff;"><B>New LB Agar was prepared.</B></P></SPAN></H4><P><B>Investigator: Patrick</B></P><H4><SPAN class="mw-headline" id="Sequenc_analysis_of_BioBricks:_CMV.2C_hGH_and_beta_globin"><P style="font-size:15px; background-color:#66bbff;"><B>Sequenc analysis of BioBricks: CMV, hGH and beta globin</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="font-size:13px; color:#68bbff;"><B>Comments</B>: Sequence analysis of three BioBricks which were cloned into the iGEM standard plasmid pSB1C3. Cloning of this three plasmids were performed at: </P><P style="font-size:13px; color:#b2222;"><B>Sequencing results of pSB1C3_CMV: </B></P><UL><LI>Plasmid sent for sequencing: </LI><LI>Primer used: VR-2 </LI><LI><B>Results:</B> Sequence read looks good. The incorporation of the PCR product can be confirmed. It can be seen that the insert is in the RFC10 standard and prefix and suffix have the right sequence. Therefore, this BioBrick can be send to the registry and used for BioBrick assembly.But: there is a mutation in the termintor (sequence picture do not show this mutation)</LI></UL><P>&lt;img src=&quot;&quot; /&gt;</P><P style="font-size:13px; color:#b2222;"><B>Sequencing results of pSB1C3_betaglobin: </B></P><UL><LI>Plasmid sent for sequencing: </LI><LI>Primer used: VR-2 </LI><LI><B>Results:</B> Sequence read looks good. The incorporation of the PCR product can be confirmed. It can be seen that the insert is in the RFC10 standard and prefix and suffix have the right sequence. Therefore, this BioBrick can be send to the registry and used for BioBrick assembly.</LI></UL><P style="font-size:13px; color:#b2222;"><B>Sequencing results of pSB1C3_hGH: </B></P><UL><LI>Plasmid sent for sequencing: </LI><LI>Primer used: VR-2 </LI><LI><B>Results:</B> Sequence read looks good. The incorporation of the PCR product can be confirmed. It can be seen that the insert is in the RFC10 standard and prefix and suffix have the right sequence. Therefore, this BioBrick can be send to the registry and used for BioBrick assembly.</LI></UL><H4><SPAN class="mw-headline" id="Cellculture"><P style="font-size:15px; background-color:#66bbff;"><B> Cellculture </B></P></SPAN></H4><P><B>Investigator: Adrian</B>
The Motivation: Transfection
The Action: HEK cells were split into four T75 flasks
The Plan: The cells should be ready at saturday for seeding and monday for transfection
We recived new HT1080 cells, they'll be split tomorrow (thx to Sven)

We recived new tumor cells which overexpress EGFR (thx to barbara)
</P><H4><SPAN class="mw-headline" id="Picking_clones_of_pSB1C3_AAP_2.2C_pSB1C3_AAP.2C_pSB1C3_Rep78_2.2C_pSB1C3_Rep78.2C_pSB1C3_Rep68_2_and_pSB1C3_Rep68"><P style="font-size:15px; background-color:#66bbff;"><B>Picking clones of pSB1C3_AAP_2, pSB1C3_AAP, pSB1C3_Rep78_2, pSB1C3_Rep78, pSB1C3_Rep68_2 and pSB1C3_Rep68</B></P></SPAN></H4><P><B>Investigator: Anissa</B>
Of each construct two clones were picked. plates are still stored in the cold-room, tomorrow mini-prep will be done.
</P><H4><SPAN class="mw-headline" id="PCR_and_ligation_for_biobrick-production_of_pSB1C3_hTERT"><P style="font-size:15px; background-color:#66bbff;"><B>PCR and ligation for biobrick-production of pSB1C3_hTERT</B></P></SPAN></H4><P><B>Investigator: Anissa,Kerstin</B></P><P style="font-size:13px; color:#68bbff;"><I><B>Comments</B></I>: PCR was performed one time without DMSO and one time with DMSO. Only the approach with DMSO showed bands in the analytic gel after PCR. That's why only this approach will be noted. </P><UL><LI><B>PCR:</B></LI></UL><P>(was performed following the standard protocol)
</P><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD align="right"><B>Volume / µl</B></TD></TR><TR><TD> 5X Phusion HF buffer </TD><TD align="right">10
</TD></TR><TR><TD> 10 mM dNTP mix</TD><TD align="right">1
</TD></TR><TR><TD> forward primer: O111 (phTERT_prefix_for_RFC10) </TD><TD align="right">2,5
</TD></TR><TR><TD> reverse primer: O112 (phTERT_suffix_rev_RFC10) </TD><TD align="right">2,5
</TD></TR><TR><TD> DNA Template</TD><TD align="right">0,35 µl
</TD></TR><TR><TD> DMSO (2%)</TD><TD align="right"> 1
</TD></TR><TR><TD> Phusion Polymerase</TD><TD align="right">0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">32,15
</TD></TR><TR><TD>Total volume</TD><TD align="right">50
</TD></TR></TBODY></TABLE><P>PCR program:
</P><TABLE border="1"><TBODY><TR><TD><B>PCR Program</B></TD><TD align="right"><B>temperature/ °C</B></TD></TR><TR><TD>1</TD><TD align="right">98
</TD></TR><TR><TD>2</TD><TD align="right">98
</TD></TR><TR><TD>8x</TD><TD align="right">58
</TD></TR><TR><TD>3 </TD><TD align="right">72
</TD></TR><TR><TD>4 </TD><TD align="right">98
</TD></TR><TR><TD>17x</TD><TD align="right">70
</TD></TR><TR><TD>5</TD><TD align="right">72
</TD></TR><TR><TD>6x</TD><TD align="right">72
</TD></TR><TR><TD>Hold</TD><TD align="right">4
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>BioBrick production of pSB1C3_hTERT</CAPTION><TBODY><TR><TD><DIV class="gallerybox" style="width: 535px;"><DIV class="gallerytext"><P>Preparative gel
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 535px;"><DIV class="gallerytext"><P>analytic gel
</P></DIV></DIV></TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color: blue;">Gel extraction</P><P>
Gel measurement:
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Sample</B></TD><TD align="left"><B>Weight</B></TD><TD align="left"><B>Volume</B></TD><TD align="left"><B>Concentration</B></TD></TR><TR><TD align="left"> hTERT
</TD><TD align="left"> 0,15g
</TD><TD align="left"> 20 µl
</TD><TD align="left"> 37,49 ng/µl
</TD></TR><TR><TD align="left"> pSB1C3_cut
</TD><TD align="left"> 0,08g
</TD><TD align="left"> 20 µl
</TD><TD align="left"> 4,56 ng/µl
</TD></TR></TBODY></TABLE><UL><LI>Digestion of PCR product and vector:
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>PCR product /µl</B></TD><TD align="right"><B>vector /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">18</TD><TD align="right">7,58
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">0,3 (100X used)</TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2,5 </TD><TD align="right">2
</TD></TR><TR><TD>Enzyme <B>XbaI </B></TD><TD align="right">1 </TD><TD align="right">1</TD></TR><TR><TD>Enzyme <B>PstI </B></TD><TD align="right">1 </TD><TD align="right">1</TD></TR><TR><TD>H2O</TD><TD align="right">2,2 </TD><TD align="right">6,42
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 25</TD><TD align="right">20
</TD></TR></TBODY></TABLE><P>
Ligation with T4-Ligase and transformation with BL 21 was made. Clones have to be picked tomorrow.
</P><H4><SPAN class="mw-headline" id="Transformation_evaluation_of_XL1B_cells_and_repetition_of_transformation"><P style="font-size:15px; background-color:#66bbff;"><B>Transformation evaluation of XL1B cells and repetition of transformation</B></P></SPAN></H4><P><B>Investigator: Kira </B>
Against our expectations both agar plates contain very few colonies. In order to figure out if the lack of colonies is due to the XL1 blue cells or loss of pUC activity, test transformation will be repeated this evening with recently prepared XL1 blue cells as well as with Lab-XL1B cells.
50 pg pUC-plate contains 8 colonies
100 pg pUC-plate contains 18 colonies 
Transformation will be performed again with 100 pg pUC (= 2 ㎕ pUC).
</P><H4><SPAN class="mw-headline" id="Cloning_of_SDM_SspI_and_SDM_PvuII_2"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of SDM SspI and SDM PvuII</B></P></SPAN></H4><P><B>Investigator: Chris W.</B></P><UL><LI>Vector: name: pSB1C3_SDM_SspI <B>P125</B></LI><LI>Insert: name: pSB1C3_SDM_PvuII <B>P129</B></LI><LI>new vector name: pSB1c3_SDM_SspI/PvuII <B>P157</B></LI><LI>buffer used: 4 ; Restriction-enzymes used: Enzyme 1 (no. Lab: 144) BpmI ; Enzyme 2 (no.Lab: 152) ClaI 
</LI><LI>DNA concentration (vector): 321,9 ng/µl ; DNA concentration (insert): 308,4 µg/µl
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume of pSB1C3_SDM_SspI /µl</B></TD><TD align="right"><B>volume of pSB1C3_SDm_PvuII /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">4,66 </TD><TD align="right">4,87
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right">2
</TD></TR><TR><TD>Enzyme BpmI (no.Lab:144)</TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>Enzyme ClaI (no.Lab:152)</TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">9,34 </TD><TD align="right">9,13
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 20</TD><TD align="right">20
</TD></TR></TBODY></TABLE><P style="color:red;">Comment: I guess enzyme 144 was BpmI and enzyme 152 was ClaI ? (Jessica)</P><P style="color:red;">Comment: u r right, changed (Chris)</P><P>
0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 130 Volt, running time:45
</P><P><B>Loading plan for agarose gel</B>: 
Marker used: GeneRuler ladder mix (Fermentas)</P><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>Sample 125, 20µl
</TH><TH>Sample 129, 20µl
</TH></TR><TR><TH>Lane
</TH><TD>1
</TD><TD>5
</TD><TD>7
</TD></TR></TBODY></TABLE><P>
The mutual vector (now without SspI) and insert (now without PvuII) were cut out. The gelextraction was performed according to the standard protocol. DNA concentration of the extracts:
</P><UL><LI> P125 Sspl: 12,8 ng/µl
</LI><LI> P129 PvuII: 11,6 ng/µl
</LI></UL><P>
The  Ligation was performed as following:
</P><UL><LI> Vector Volume: 2,81 µl
</LI><LI> Insert Volume: 5,19 µl
</LI></UL><UL><LI> 1µl T4-Ligase buffer (10x)
</LI><LI> 8µl (Vector + Insert) mix
</LI><LI> 1µl T4-Ligase
</LI></UL><P> Incubate for 30min
</P><P><B>Transformation:</B> performed according to the standard protocol (BL21). The cells were plated on a agar plate with chloramphenicol
</P><H4><SPAN class="mw-headline" id="Miniprep_of_p158_and_p159"><P style="font-size:15px; background-color:#66bbff;"><B>Miniprep of p158 and p159</B></P></SPAN></H4><P><B>Investigator: Bea &amp; Volker</B></P><P style="font-size:20px; font-weight: bold; color: blue;"><U>Plasmid Mini-Prep</U></P><P>For the tripple mutant of pAAV-RC a Quick-change reaction was carried out to remove the last restriction site (SalI) that is required for the Viral Brick standard. For this construct glycerol stocks and minipreps were carried out for two clones.</P><P><U>Glycerol Stocks</U></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD></TR><TR><TD align="left"><B>Bacteria strain</B></TD><TD align="left"> BL-21 </TD><TD align="left"> BL-21
</TD></TR><TR><TD align="left"><B>Plasmidname</B></TD><TD align="left"> pAAV_RC_1.2 SDM SalI
</TD><TD>align=&quot;left&quot;| pAAV_RC_1.2 SDM SalI
</TD></TR><TR><TD align="left"><B>Date</B></TD><TD align="left"> 05.08.2010 </TD><TD align="left"> 05.08.2010
</TD></TR><TR><TD align="left"><B>given glycerol-stock no.</B></TD><TD align="left"> B126 </TD><TD align="left"> B127
</TD></TR><TR><TD align="left"><B>given plasmid no.</B></TD><TD align="left"> p158 </TD><TD align="left"> p159
</TD></TR></TBODY></TABLE><P>
TO do: Test digestion and sequencing!!!
</P><H3><SPAN class="mw-headline" id="81.Labortag_06.08.2010"><P style="font-size:17px; background-color:#00dd77;">81.Labortag 06.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Cloning_Rep40_.26_Rep52_into_pSB1C3"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning Rep40 &amp; Rep52 into pSB1C3</B></P></SPAN></H4><P><B>Investigator: Patrick </B>
Intention: get biobrick ready 
PCR of pKEX-2XL.Rep 40 (P22) and pKEX-2XL.Rep 52 (P23) following gelrun (1%) and gelextraction. 
Used Primers:
</P><UL><LI> Praefix Rep40_52ex (O94)
</LI><LI> Suffix Rep 40_68ex (O96)
</LI><LI> Suffix Rep 52_78ex (O97)
</LI></UL><P>
PCR programm of Rep40:                          
</P><UL><LI>98°C   1 min
</LI></UL><UL><LI>98°C   15 sec
</LI><LI>63°C   25 sec
</LI><LI>72°C   15 sec   Repeat this cycle 8 times.
</LI></UL><UL><LI>98°C   15 sec
</LI><LI>68°C   25 sec
</LI><LI>72°C   15 sec   Repeat this cycle 17 times.
</LI></UL><UL><LI>72°C   5 min
</LI><LI>Hold 4°C
</LI></UL><P>
PCR programm of Rep52:                          
</P><UL><LI>98°C   1 min
</LI></UL><UL><LI>98°C   15 sec
</LI><LI>62°C   25 sec
</LI><LI>72°C   20 sec   Repeat this cycle 8 times.
</LI></UL><UL><LI>98°C   15 sec
</LI><LI>66°C   25 sec
</LI><LI>72°C   20 sec   Repeat this cycle 17 times.
</LI></UL><UL><LI>72°C   5 min
</LI><LI>Hold 4°C
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>ingredients</B></TD><TD align="right"><B>Rep 40</B></TD><TD align="right"><B>Rep40 + DMSO</B></TD><TD align="right"><B>Rep52</B></TD><TD align="right"><B> Rep52 + DMSO</B></TD></TR><TR><TD> 5x Phusion HF buffer</TD><TD align="right">10 µl </TD><TD align="right">10 µl</TD><TD align="right">10 µl</TD><TD align="right">10 µl
</TD></TR><TR><TD>10 mM dNTP mix</TD><TD align="right">1 µl </TD><TD align="right">1 µl</TD><TD align="right">1 µl</TD><TD align="right">1 µl
</TD></TR><TR><TD>for Primer O94 (1:10 dilution, 05 µM)</TD><TD align="right">2,5 µl  </TD><TD align="right">2,5 µl</TD><TD align="right">2,5 µl</TD><TD align="right">2,5 µl
</TD></TR><TR><TD>rev Primer (1:10 dilution, 0,5 µM) </TD><TD align="right">2,5 µl O96</TD><TD align="right">2,5 µl O96</TD><TD align="right">2,5 µl O97</TD><TD align="right">2,5 µl O97
</TD></TR><TR><TD>DNA template </TD><TD align="right"> 1 µl, 112,7 ng/µl</TD><TD align="right">1 µl, 112,7 ng/µl</TD><TD align="right">0,9 µl, 142,3 ng/µl</TD><TD align="right">0,9 µl, 142,3 ng/µl
</TD></TR><TR><TD>DMSO</TD><TD align="right">0 µl</TD><TD align="right">0,5 µl</TD><TD align="right">0 µl</TD><TD align="right">0,5 µl
</TD></TR><TR><TD>Phusion Polymerase</TD><TD align="right">0,5 µl</TD><TD align="right">0,5 µl</TD><TD align="right">0,5 µl</TD><TD align="right">0,5 µl
</TD></TR><TR><TD>H2O</TD><TD align="right">32,5 µl</TD><TD align="right">32 µl</TD><TD align="right">32,6 µl</TD><TD align="right">32,1 µl
</TD></TR><TR><TD>Total volume</TD><TD align="right">50 µl</TD><TD align="right">50 µl</TD><TD align="right">50 µl</TD><TD align="right">50 µl
</TD></TR></TBODY></TABLE><P>Ecpected size of Rep40: 940 bp, expected size of Rep52:1198 bp. There are two samples of Rep40 and Rep52 and the PCR of one of each was run with 1% DMSO because the Rep40 Praefix has a strong secondary structure and was used as a primer for Rep 40 and Rep 52</P><P>
Digestion of pSB1C3 (P51.1) with EcoRI-HF and SpeI following gelrun (0.8%) and gelextraction. 
Expected size of the fragments: about 2000 bp and 800bp </P><P>All marked fragments were extracted. The Gelextraction was performed according to the standard protocol following a digestion of the PCR products with EcoRI HF and SpeI and a purification of the PCR product. The Ligation was not performed according to the standard protocol: 1 µl T4 DNA Ligase, 1 µl (10x) Buffer, 3 µl vector pSB1C3 (60 ng/µl) and 5 µl insert (all concentrations about 100 ng/µl). The Transformation (with BL21) was performed according to the standard protocol. Tomorrow there will hopefully be some clones on the agar plates.
</P><H4><SPAN class="mw-headline" id="BioBrick_Assembly_of_pSB1C3_beta_globin_mVenus"><P style="font-size:15px; background-color:#66bbff;"><B>BioBrick Assembly of pSB1C3_beta globin_mVenus</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: BioBricks are ready to use. The sequenced pSB1C3_beta globin (P133) and pGA14_mVenus (P60) will be digested with different enzymes and will be assembled in order to produce the first step in assembling the whole vector together. </P><UL><LI>First the plasmids P133 and P60 were digested:</LI></UL><TABLE border="2" cellpadding="1" cellspacing="1" width="528" style="border-collapse: collapse;table-layout:fixed;width:396pt"><TBODY><TR style="height:15.0pt"><TD height="20" class="xl65" width="176" style="height:15.0pt;width:132pt"> </TD><TD class="xl66" width="176" style="border-left:none;width:132pt">pSB1C3_betaglobin/µL</TD><TD class="xl66" width="176" style="border-left:none;width:132pt">pGA14_mVenus/µL</TD></TR><TR style="height:15.0pt"><TD height="20" class="xl67" width="176" style="height:15.0pt;border-top:none; width:132pt">DNA</TD><TD class="xl68" style="border-top:none;border-left:none">7</TD><TD class="xl68" style="border-top:none;border-left:none">9</TD></TR><TR style="height:15.0pt"><TD height="20" class="xl67" width="176" style="height:15.0pt;border-top:none; width:132pt">BSA (100x)</TD><TD class="xl68" style="border-top:none;border-left:none">0,2</TD><TD class="xl68" style="border-top:none;border-left:none">0,2</TD></TR><TR style="height:30.0pt"><TD height="40" class="xl67" width="176" style="height:30.0pt;border-top:none; width:132pt">Buffer 4(10x)</TD><TD class="xl68" style="border-top:none;border-left:none">2</TD><TD class="xl68" style="border-top:none;border-left:none">2</TD></TR><TR style="height:30.0pt"><TD height="40" class="xl67" width="176" style="height:30.0pt;border-top:none; width:132pt">Enzyme XbaI</TD><TD class="xl68" style="border-top:none;border-left:none">1</TD><TD class="xl68" style="border-top:none;border-left:none">1</TD></TR><TR style="height:30.0pt"><TD height="40" class="xl67" width="176" style="height:30.0pt;border-top:none; width:132pt">Enzyme AgeI</TD><TD class="xl68" style="border-top:none;border-left:none">1</TD><TD class="xl68" style="border-top:none;border-left:none">1</TD></TR><TR style="height:15.0pt"><TD height="20" class="xl67" width="176" style="height:15.0pt;border-top:none; width:132pt">H2O</TD><TD class="xl68" style="border-top:none;border-left:none">8,8</TD><TD class="xl68" style="border-top:none;border-left:none">6,8</TD></TR><TR style="height:30.0pt"><TD height="40" class="xl67" width="176" style="height:30.0pt;border-top:none; width:132pt">Total volume</TD><TD class="xl69" style="border-top:none;border-left:none">20</TD><TD class="xl69" style="border-top:none;border-left:none">20</TD></TR></TBODY></TABLE><UL><LI> Incubation of plasmids at 37°C for 2 hours </LI><LI> Load samples on preparative 1% agarose gel and run at 110 V, 45 minutes </LI></UL><P><B>Results:</B> As it can be seen in the picture below, digestion of the vector pSB1C3_betaglobin (left lane) revealed a band at around 2500 - 3000 bp. The expected size after digetsing with SpeI and PstI was: 2555 bp. The band ran a little bit higher than expected, anyhow the band was cut out of the gel. The right lane belongs to the pGA14_mVenus which was digested with XbaI and PstI which resulted in two fragments. mVENUS was digested. Expected sizes were: mVenus = 756bp and pGA14 = 2900bp. The expected sizes correspond to the bands seen in the picture below. The smaller fragment belonging to mVenus was cut out of the gel as well. 
</P><P>
After gel extraction has been performed, the concentrations of the two fragments were measured and we obtained following concentrations:
</P><UL><LI>c(pSB1C3_betaglobin) = 15,62 ng/µL </LI><LI>c(mVenus) = 6,83  ng/µL </LI></UL><P>
For ligation the T4-Ligase were used. 
</P><UL><LI>v(pSB1C3_betaglobin) = 2,64 µL </LI><LI>v(mVenus) =5,36 µL </LI><LI>v(10xT4 Ligase buffer) = 1 µL </LI><LI>v(T4-Ligase) = 1 µL </LI><LI><B>Total volume = 10 µL </B></LI></UL><P>
Additionally a control ligation was performed. 
</P><UL><LI>v(pSB1C3_betaglobin) = 2,64 µL </LI><LI>v(H<SUB>2</SUB>0) =5,36 µL </LI><LI>v(10xT4 Ligase buffer) = 1 µL </LI><LI>v(T4-Ligase) = 1 µL </LI><LI><B>Total volume = 10 µL </B></LI><LI>Ligation duration was 45 minutes on room temperature.</LI></UL><P>
After ligation a transformation was performed.
</P><UL><LI> Used bacterial strain: XL1-B</LI><LI> LB-agar plates containing chloramphenicol were plated and put in the 37°C room over night</LI></UL><P style="font-size:13px; color:#003399;"><B>To do</B>: Mini-Prep and test digestion in order to verify assembly of beta globin and YFP. After verification of the fusion of the two fragments this construct is ready for the next BioBrick assembly step. </P><H4><SPAN class="mw-headline" id="Test_digestion_of_SalI_SDM"><P style="font-size:15px; background-color:#66bbff;"><B>Test digestion of SalI SDM</B></P></SPAN></H4><P><B>Investigator: Hanna</B></P><P style="font-size:15px; font-weight: bold; color:#ff00ff;">Test digestion</P><UL><LI>buffer used: 3 ; Restriction-enzymes used: Enzyme 1 (no. Lab: 53): SalI ; Enzyme 2: XbaI </LI><LI>Plasmid
</LI><LI>Given Plasmid-Number: P158; DNA concentration: 398.8 ng/µL ;</LI><LI>Given Plasmid-Number: P159; DNA concentration: 437.4 ng/µL ;</LI></UL><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>P158 Volume/µL</B></TD><TD align="left"><B>P159 Volume/µL</B></TD></TR><TR><TD align="left"> DNA </TD><TD align="left"> 2.5 </TD><TD align="left"> 2.3
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 1 </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Buffer 3 (10x) </TD><TD align="left"> 1 </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> SalI (no. Lab: 53) </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5
</TD></TR><TR><TD align="left"> XbaI </TD><TD align="left"> 0.75 </TD><TD align="left"> 0.75
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 4.25 </TD><TD align="left"> 4.45
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>10</B></TD><TD align="left"><B>10</B></TD></TR></TBODY></TABLE><P>
Incubation: 1 h
</P><P style="font-size:15px; font-weight: bold; color:#ff00ff;">Agarose-Gel:</P><P>
0.5  g Agarose, 50 mL TBE (1 %),3 µL GELRED, at 115 Volt, running time: 45 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (5x)/µl
</TH><TH>Expected size 1 (if SDM didn't work)
</TH><TH>Expected size 2 (if SDM didn't work)
</TH><TH>Expected size 3 (if SDM didn't work)
</TH></TR><TR><TD>P158
</TD><TD>10 µl
</TD><TD>2 µl
</TD><TD>6129 bp
</TD><TD>1143 bp
</TD><TD>61 bp
</TD></TR><TR><TD>P159
</TD><TD>10 µl
</TD><TD>2 µl
</TD><TD>6129 bp
</TD><TD>1143 bp
</TD><TD>61 bp
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker /µL
</TH><TH>Sample P158 /µl
</TH><TH>Sample P159 /µl
</TH><TH>Marker /µL
</TH></TR><TR><TH>Lane
</TH><TD>6.5
</TD><TD>10
</TD><TD>10
</TD><TD>7
</TD></TR></TBODY></TABLE><P><B>Comments:</B> Test digestion looked good: No 1143 bp fragment was detectable in neither test digestion. 
P158 was sent for sequencing to GATC. </P><H4><SPAN class="mw-headline" id="ITR_test_digestion_of_pAAV_MCS"><P style="font-size:15px; background-color:#66bbff;"><B>ITR test digestion of pAAV_MCS</B></P></SPAN></H4><P><B>Investigator: Hanna</B></P><P style="color:#ff00ff;"><B>Comment:</B> Because sequencing of the right ITR delivered that perhaps a 15 bp fragment is missing in the sequence, we wanted to check, whether this fragment is also not present in the original pAAV_MCS plasmid or whether the loss of these 15 bp happened during cloning. </P><P style="font-size:15px; font-weight: bold; color:#ff00ff;">Test digestion</P><UL><LI>buffer used: 4 ; Restriction-enzymes used: Enzyme 1: NotI; Enzyme 2: PstI</LI><LI>Plasmid
</LI><LI>Plasmid-Number: from Sven; DNA concentration: 259 ng/µL ;</LI></UL><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>Volume/µL</B></TD></TR><TR><TD align="left"> DNA </TD><TD align="left"> 3.8
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 1.5
</TD></TR><TR><TD align="left"> Buffer 4 (10x) </TD><TD align="left"> 1.5
</TD></TR><TR><TD align="left"> NotI-HF </TD><TD align="left"> 0.75
</TD></TR><TR><TD align="left"> PstI-HF </TD><TD align="left"> 0.75
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 3.2
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>15</B></TD></TR></TBODY></TABLE><P>
Incubation: 1 h
</P><P style="font-size:15px; font-weight: bold; color:#ff00ff;">Agarose-Gel:</P><P>
0.5  g Agarose, 50 mL TBE (1 %),3 µL GELRED, at 115 Volt, running time: 45 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (5x)/µl
</TH><TH>Expected size <B>ITRs</B></TH><TH>Expected size 2
</TH><TH>Expected size 3
</TH><TH>Expected size 4
</TH></TR><TR><TD>pAAV_MCS
</TD><TD>15 µl
</TD><TD>2 µl
</TD><TD>145 bp
</TD><TD>1216 bp
</TD><TD>555 bp
</TD><TD>2608 bp
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker /µL
</TH><TH>Sample /µl
</TH><TH>Marker /µL
</TH></TR><TR><TH>Lane
</TH><TD>6.5
</TD><TD>12
</TD><TD>6.5
</TD></TR></TBODY></TABLE><P style="font-size:15px; color:#ff00ff;"><B>Comments:</B> The test digestion showed that there're two bands between the 100 and 200 bp marker nucleotides! By comparing the gel picture with the last mini-prep digestion of the fancy method we found out that the space between the two bands are similar. Because of that we assumed that the 15 pb are already missing in the original Stratagene plasmid!!!  </P><P style="font-size:15px; color:#800080;"><B>Conclusion:</B> Because we already tested the Stratagene Kit via YFP expression, we can conclude that the usage of the &quot;wrong&quot; rigth ITR works. Therefore we decided to continue working with this &quot;mutated&quot; = <B> ENGINEERED </B> :) right ITR as BioBrick.</P><H4><SPAN class="mw-headline" id="Miniprep_of_pSBC13_Rep78.2C_pSB1C3_Rep68.2C_pSB1C3_AAP.2C_pSB1C3_rightITR_and_psB1C3_leftITR"><P style="font-size:15px; background-color:#66bbff;"><B>Miniprep of pSBC13_Rep78, pSB1C3_Rep68, pSB1C3_AAP, pSB1C3_rightITR and psB1C3_leftITR</B></P></SPAN></H4><P><B>Investigator: Kerstin</B></P><P style="font-size:13px; color: blue;">Plasmid Mini-Prep</P><P><U><B>Glycerol Stocks</B></U><B>pSB1C3_Rep78</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD></TR><TR><TD align="left"><B>Bacteria strain</B></TD><TD align="left"> BL-21 </TD><TD align="left"> BL-21
</TD></TR><TR><TD align="left"><B>given glycerol-stock no.</B></TD><TD align="left"> B128 </TD><TD align="left"> B129
</TD></TR><TR><TD align="left"><B>given plasmid no.</B></TD><TD align="left"> P160 </TD><TD align="left"> p161
</TD></TR><TR><TD align="left"><B>DNA-concentration ng/µl</B></TD><TD align="left"> 139,1 </TD><TD align="left"> 71,0
</TD></TR></TBODY></TABLE><P><B>pSB1C3_2_Rep78</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD></TR><TR><TD align="left"><B>Bacteria strain</B></TD><TD align="left"> BL-21 </TD><TD align="left"> BL-21
</TD></TR><TR><TD align="left"><B>given glycerol-stock no.</B></TD><TD align="left"> B130 </TD><TD align="left"> B131
</TD></TR><TR><TD align="left"><B>given plasmid no.</B></TD><TD align="left"> P162 </TD><TD align="left"> p163
</TD></TR><TR><TD align="left"><B>DNA-concentration ng/µl</B></TD><TD align="left"> 112,1 </TD><TD align="left"> 79,7
</TD></TR></TBODY></TABLE><P><B>pSB1C3_Rep68</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD></TR><TR><TD align="left"><B>Bacteria strain</B></TD><TD align="left"> BL-21 </TD><TD align="left"> BL-21
</TD></TR><TR><TD align="left"><B>given glycerol-stock no.</B></TD><TD align="left"> B132 </TD><TD align="left"> B133
</TD></TR><TR><TD align="left"><B>given plasmid no.</B></TD><TD align="left"> P164 </TD><TD align="left"> p165
</TD></TR><TR><TD align="left"><B>DNA-concentration ng/µl</B></TD><TD align="left"> 80,2 </TD><TD align="left"> 72,8
</TD></TR></TBODY></TABLE><P><B>pSB1C3_2_Rep68</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD></TR><TR><TD align="left"><B>Bacteria strain</B></TD><TD align="left"> BL-21 </TD><TD align="left"> BL-21
</TD></TR><TR><TD align="left"><B>given glycerol-stock no.</B></TD><TD align="left"> B134 </TD><TD align="left"> B135
</TD></TR><TR><TD align="left"><B>given plasmid no.</B></TD><TD align="left"> P166 </TD><TD align="left"> p167
</TD></TR><TR><TD align="left"><B>DNA-concentration ng/µl</B></TD><TD align="left"> 85,6 </TD><TD align="left"> 69,1
</TD></TR></TBODY></TABLE><P><B>pSB1C3_AAP</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD></TR><TR><TD align="left"><B>Bacteria strain</B></TD><TD align="left"> BL-21 </TD><TD align="left"> BL-21
</TD></TR><TR><TD align="left"><B>given glycerol-stock no.</B></TD><TD align="left"> B136 </TD><TD align="left"> B137
</TD></TR><TR><TD align="left"><B>given plasmid no.</B></TD><TD align="left"> P168 </TD><TD align="left"> p169
</TD></TR><TR><TD align="left"><B>DNA-concentration ng/µl</B></TD><TD align="left"> 63,6 </TD><TD align="left"> 56,3
</TD></TR></TBODY></TABLE><P><B>pSB1C3_2_AAP</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD></TR><TR><TD align="left"><B>Bacteria strain</B></TD><TD align="left"> BL-21 </TD><TD align="left"> BL-21
</TD></TR><TR><TD align="left"><B>given glycerol-stock no.</B></TD><TD align="left"> B138 </TD><TD align="left"> B139
</TD></TR><TR><TD align="left"><B>given plasmid no.</B></TD><TD align="left"> P170 </TD><TD align="left"> p171
</TD></TR><TR><TD align="left"><B>DNA-concentration ng/µl</B></TD><TD align="left"> 135,1 </TD><TD align="left"> 114,2
</TD></TR></TBODY></TABLE><P><B>pSB1C3_rightITR</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD></TR><TR><TD align="left"><B>Bacteria strain</B></TD><TD align="left"> BL-21 </TD><TD align="left"> BL-21
</TD></TR><TR><TD align="left"><B>given glycerol-stock no.</B></TD><TD align="left"> B140 </TD><TD align="left"> B141
</TD></TR><TR><TD align="left"><B>given plasmid no.</B></TD><TD align="left"> P172 </TD><TD align="left"> p173
</TD></TR><TR><TD align="left"><B>DNA-concentration ng/µl</B></TD><TD align="left"> 73,7 </TD><TD align="left"> 66,4
</TD></TR></TBODY></TABLE><P><B>pSB1C3_leftITR</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD></TR><TR><TD align="left"><B>Bacteria strain</B></TD><TD align="left"> BL-21 </TD><TD align="left"> BL-21
</TD></TR><TR><TD align="left"><B>given glycerol-stock no.</B></TD><TD align="left"> B142 </TD><TD align="left"> B143
</TD></TR><TR><TD align="left"><B>given plasmid no.</B></TD><TD align="left"> P174 </TD><TD align="left"> p175
</TD></TR><TR><TD align="left"><B>DNA-concentration ng/µl</B></TD><TD align="left"> 74,9 </TD><TD align="left"> 80,3
</TD></TR></TBODY></TABLE><P><B>Test digestion:</B>
pSB1C3_Rep78, pSB1C3_Rep68 and pSB1C3_AAP were digested with XbaI and SpeI
pSB1C3_rightITR and pSB1C3_leftITR were digested with EcoRI and PstI.</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of <B>P160</B>/µl </TD><TD>volume of <B>P161</B>/µl</TD><TD>volume of <B>P162/</B>µl </TD><TD>volume of <B>P163</B>/µl </TD><TD>volume of <B>P164</B>/µl</TD><TD>volume of <B>P165</B>/µl</TD><TD>volume of <B>P166</B>µl </TD><TD>volume of <B>P167</B>/µl</TD><TD>volume of <B>P168</B>µl </TD><TD>volume of <B>P169</B>/µl</TD><TD>volume of <B>P170</B>/µl</TD><TD>volume of <B>P171</B>/µl</TD><TD>volume of <B>P172</B>/µl</TD><TD>volume of <B>P173</B>µl </TD><TD>volume of <B>P174</B>/µl </TD><TD>volume of <B>P175</B>/µl
</TD></TR><TR><TD> DNA  </TD><TD>5,8 </TD><TD>11,3 </TD><TD>7,1 </TD><TD>10 </TD><TD>9,9 </TD><TD>10,9 </TD><TD>9,3 </TD><TD>11,6 </TD><TD>12,5 </TD><TD>14,2 </TD><TD>5,9 </TD><TD>7,0 </TD><TD>10,9 </TD><TD>12,0 </TD><TD>10,7 </TD><TD>9,7
</TD></TR><TR><TD> BSA (10x) </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2 </TD><TD>2
</TD></TR><TR><TD>Enzyme 1 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5
</TD></TR><TR><TD>Enzyme 2 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5
</TD></TR><TR><TD>H2O </TD><TD>9,2 </TD><TD>3,7 </TD><TD>7,9 </TD><TD>5,0 </TD><TD>5,1 </TD><TD>4,1 </TD><TD>5,7 </TD><TD>3,4 </TD><TD>2,5 </TD><TD>0,8 </TD><TD>9,1 </TD><TD>8 </TD><TD>4,1 </TD><TD>3 </TD><TD>4,3 </TD><TD>5,3
</TD></TR><TR><TD><B>Total volume /µl</B></TD><TD>20 </TD><TD>20 </TD><TD>20 </TD><TD>20 </TD><TD>20 </TD><TD>20 </TD><TD>20 </TD><TD>20 </TD><TD>20 </TD><TD>20 </TD><TD>20 </TD><TD>20 </TD><TD>20 </TD><TD>20 </TD><TD>20 </TD><TD>20
</TD></TR></TBODY></TABLE><P><B>Expected sizes:</B></P><UL><LI>Rep78: 1,8 kb
</LI><LI>Rep68: 1,6kb</LI><LI>AAP: 650bp </LI></UL><UL><LI>left ITR: 150bp
</LI><LI>right ITR: 150bp
</LI></UL><P style="font-size:13px; color:#11bb88;"><B>Results of samples loaded 1,5% agarose gel and ran ~50 minutes at 115 V:</B></P><P style="font-size:13px; color:#11bb88;"><B>Results of samples loaded 0,8% agarose gel and ran ~50 minutes at 115 V:</B></P><P style="font-size:13px; color:#11bb56;"><B>Sequencing</B></P><P>P160, P165 and P170 were send for sequencing.
</P><H3><SPAN class="mw-headline" id="82.Labortag_07.08.2010"><P style="font-size:17px; background-color:#00dd77;">82.Labortag 07.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Cellculture_2"><P style="font-size:15px; background-color:#66bbff;"><B>Cellculture</B></P></SPAN></H4><P><B>Investigator: Patrick</B>
HEK 293 cells were split and seeded according to the standard protocol: 2x T75 flask and 10x10cm cellculture dishes.
</P><H4><SPAN class="mw-headline" id="Continuation:_Cloning_Rep40_.26_Rep52_into_pSB1C3"><P style="font-size:15px; background-color:#66bbff;"><B>Continuation: Cloning Rep40 &amp; Rep52 into pSB1C3</B></P></SPAN></H4><P><B>Investigator: Patrick </B>
Results: no clones could be picked because there were no clones. Maybe the PCR purification or one of the following steps did not work (the PCR purifications were quite bad) so i purified the PCR products again yielding very low DNA concentrations (0-13,28 ng/µl) and still very obvious pollution of the sample. Nonetheless a ligation with the insert Rep40+DMSO, Rep52, Rep52+DMSO and the vector pSB1C3 was carried out following a transformation and according to the standard protocols. Hopefully there will be some clones tomorrow.
</P><H4><SPAN class="mw-headline" id="Mini-Prep_and_test_digestion_of_pSB1C3_SDM_SspI.2FPvuII"><P style="font-size:15px; background-color:#66bbff;"><B>Mini-Prep and test digestion of pSB1C3_SDM_SspI/PvuII</B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><P style="font-size:18px; font-weight: bold; color:#66bbff;"><U>Plasmid Mini-Prep</U></P><UL><LI>new vector name: pSB1C3_SDM_SspI/PvuII
</LI></UL><P><U><B>Glycerol Stocks</B></U><B>pSB1C3_SDM_SspI/PvuII</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD><TD align="left"><B>Clone 3</B></TD></TR><TR><TD align="left"><B>Bacteria strain</B></TD><TD align="left"> BL21 </TD><TD align="left"> BL21  </TD><TD align="left"> BL21
</TD></TR><TR><TD align="left"><B>Plasmidname</B></TD><TD align="left"> pSb1C3_SDM_SspI/PvuII </TD><TD align="left"> pSb1C3_SDM_SspI/PvuII </TD><TD align="left"> pSb1C3_SDM_SspI/PvuII
</TD></TR><TR><TD align="left"><B>Date</B></TD><TD align="left"> 7.8.10 </TD><TD align="left"> 7.8.10 </TD><TD align="left"> 7.8.10
</TD></TR><TR><TD align="left"><B>given glycerol-stock no.</B></TD><TD align="left"> B144 </TD><TD align="left"> B145 </TD><TD align="left"> B146
</TD></TR><TR><TD align="left"><B>given plasmid no.</B></TD><TD align="left"> P157.1 </TD><TD align="left"> P157.2 </TD><TD align="left"> P157.3
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color:#66bbff;">Test digestion</P><UL><LI>buffer used: 2 ; Restriction-enzymes used: Enzyme 1 SspI ; Enzyme 2 PvuII
</LI><LI>Plasmid: pSB1C3_SDM_SspI/PvuII:
</LI><LI>Given Plasmid-Number: P157.1; DNA concentration: 246,3 ng/µL ;
</LI><LI>Given Plasmid-Number: P157.2; DNA concentration: 231,7 ng/µL ;
</LI><LI>Given Plasmid-Number: P157.3; DNA concentration: 219,6 ng/µL ;
</LI></UL><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>P157.1</B> Volume/µL </TD><TD align="left"><B>P157.2</B> Volume/µL </TD><TD align="left"><B>P157.3</B> Volume/µL
</TD></TR><TR><TD align="left"> DNA </TD><TD align="left"> 2,8 </TD><TD align="left"> 3,0 </TD><TD align="left"> 3,19
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> - </TD><TD align="left"> - </TD><TD align="left"> -
</TD></TR><TR><TD align="left"> Buffer no. 2 (10x) </TD><TD align="left"> 1 </TD><TD align="left"> 1 </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Enzyme 1 SspI </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5
</TD></TR><TR><TD align="left"> Enzyme 2 PvuII </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 5,2 </TD><TD align="left"> 5 </TD><TD align="left"> 4,81
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>10</B></TD><TD align="left"><B>10</B></TD><TD align="left"><B>10</B></TD></TR></TBODY></TABLE><UL><LI>Incubation: 45 min
</LI></UL><P style="font-size:15px; font-weight: bold; color:#66bbff;">Agarose-Gel:</P><P>
0.45 g Agarose, 50 mL <B>TAE</B>, 3 µL GELRED, at 115 Volt, running time: 45 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (6x)/µl
</TH></TR><TR><TD>P157.1
</TD><TD>10 µl
</TD><TD>1,6 µl
</TD></TR><TR><TD>P157.2
</TD><TD>10 µl
</TD><TD>1,6 µl
</TD></TR><TR><TD>P157.3
</TD><TD>10 µl
</TD><TD>1,6 µl
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker /µL
</TH><TH>Sample P157.1 /µL
</TH><TH>Sample P157.2 /µL
</TH><TH>Sample P157.3 /µL
</TH></TR><TR><TH>Lane
</TH><TD>7
</TD><TD>10
</TD><TD>10
</TD><TD>10
</TD></TR></TBODY></TABLE><P>
I expect one cut from the PvuII RS in the CFP --&gt; linearized </P><P><B>Comment:</B></P><P style="font-size:13px; color:#ff0000;"><I><B>Comments:</B></I>Oh f... it makes no sense... if the cloning didn't work, there can just be 3 bands (one time SspI and twice PvuII). The results of the test digestion from aug 1th showed that there can be one RS PvuII more. but then you have a maximum of 4 bands. <B>P157.2</B> show <B>5</B> bands. i can't interpret this result at the moment. interpretation will follow</P><H4><SPAN class="mw-headline" id="Evaluation_of_test_transformation_with_XL1_blue_cells_and_pUCI"><P style="font-size:15px; background-color:#66bbff;"><B>Evaluation of test transformation with XL1 blue cells and pUCI</B></P></SPAN></H4><P><B>Investigator: Kira</B></P><P>Transformation was performed with lab XL1B and iGEM XL1B cells. iGEM plate contained just 25 colonies, while lab plate contained around 50 colonies.
Conclusion: the production of competent cells has to be repeated.
</P><H4><SPAN class="mw-headline" id="Biobrick_production_of_p5_promoter_from_pTAV2_and_pAAV_RC"><P style="font-size:15px; background-color:#66bbff;"><B>Biobrick production of p5 promoter from pTAV2 and pAAV_RC</B></P></SPAN></H4><P><B>Investigator: Kira</B></P><UL><LI><B>PCR:</B></LI></UL><P>DNA samples were diluted 1:100
</P><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD align="right"><B>p 32</B></TD><TD align="right">p 50
</TD></TR><TR><TD> 5X Phusion HF buffer </TD><TD align="right">10 µl</TD><TD align="right">10 µl
</TD></TR><TR><TD> 10 mM dNTP mix</TD><TD align="right">1µl </TD><TD align="right">1 µl
</TD></TR><TR><TD> forward primer: O114 </TD><TD align="right">2,5µl</TD><TD align="right">2,5 µl
</TD></TR><TR><TD> reverse primer: O115 </TD><TD align="right">2,5 µl</TD><TD align="right">2,5 µl
</TD></TR><TR><TD> DNA Template</TD><TD align="right">0,3 µl</TD><TD align="right"> 0,4 µl
</TD></TR><TR><TD> DMSO </TD><TD align="right">0 µl</TD><TD align="right">0 µl
</TD></TR><TR><TD> Phusion Polymerase</TD><TD align="right">0,5 µl</TD><TD align="right"> 0,5 µl
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">33,2 µl</TD><TD align="right">28,6 µl
</TD></TR><TR><TD>Total volume</TD><TD align="right">50 µl</TD><TD align="right"> 50 µl
</TD></TR></TBODY></TABLE><P>PCR program:
</P><TABLE border="1"><TBODY><TR><TD>Cycles</TD><TD>Temperature</TD><TD>Time
</TD></TR><TR><TD>98°C</TD><TD>1
</TD></TR><TR><TD>8x</TD><TD>98°C</TD><TD>30&quot;
</TD></TR><TR><TD>60°C</TD><TD>25&quot;
</TD></TR><TR><TD>72°C</TD><TD>6&quot;
</TD></TR><TR><TD>17x</TD><TD>98°C</TD><TD>30&quot;
</TD></TR><TR><TD>70°C</TD><TD>25&quot;
</TD></TR><TR><TD>72°C</TD><TD>6&quot;
</TD></TR><TR><TD>1x</TD><TD>72°C</TD><TD>5'
</TD></TR><TR><TD>Hold 4°C
</TD></TR></TBODY></TABLE><P>Digestion of plasmid backbone:
</P><P>c (pSB1C3) = 151, 1 ng/ µl
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>vector</B> Volume/µL
</TD></TR><TR><TD align="left"> DNA 1 µg  </TD><TD align="left"> 6,0 µl
</TD></TR><TR><TD align="left"> BSA (100x) </TD><TD align="left"> 0,2 µl
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 2,0 µl
</TD></TR><TR><TD align="left"> Enzyme 1 XbaI </TD><TD align="left"> 1 µl
</TD></TR><TR><TD align="left"> Enzyme 2 PstI HF </TD><TD align="left"> 1 µl
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 9,8 µl
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>20</B></TD></TR></TBODY></TABLE><P>incubation @ 37 C for approx. 2 h
</P><P>1% agarose gel with PCR products:
</P><P>Concentrations after gel extraction:
</P><P>c(p32) = 67,95 ng/µl 
c(p50) = 32,45 ng/µl 
c(vector) = 7,54 ng/µl </P><P>Digestion of PCR products:
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>PCR product</B> Volume/µL
</TD></TR><TR><TD align="left"> DNA 1 µg  </TD><TD align="left"> 23,0 µl
</TD></TR><TR><TD align="left"> BSA (100x) </TD><TD align="left"> 0,3 µl
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 3,0 µl
</TD></TR><TR><TD align="left"> Enzyme 1 XbaI </TD><TD align="left"> 1,5 µl
</TD></TR><TR><TD align="left"> Enzyme 2 PstI HF </TD><TD align="left"> 1,5 µl
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 0,7 µl
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>30</B></TD></TR></TBODY></TABLE><P>Concentrations after PCR purification:
</P><P>c(p32) = 21, 63 ng/µl 
c(p50) = 19,48 ng/µl </P><P>Ligation:
</P><P>Quickligase was used for ligation.
</P><P>p32 DNA-Mix: 8 µl vector + 1 µl insert 
p50 DNA-Mix: 7,9 µl vector + 1,1 µl insert 
Transformation:
</P><P>Transformation was performed according to the standard protocol with DYT and BL21 cells. The cells were spread on the agar plates containing chloramphenicol.
</P><H4><SPAN class="mw-headline" id="Sequencing_results_of_SalI_SDM"><P style="font-size:15px; background-color:#66bbff;"><B>Sequencing results of SalI SDM</B></P></SPAN></H4><P><B>Investigator: Hanna</B></P><P style="color:#ff00ff;"><B>Comment:</B> The test digestion already showed that the SDM of the SalI restriction site in pAAV_RC worked. This could be validated by sequencing the plasmid with the Rep-1250_for primer: </P><P>P58 can be used for further RepCap-experiments.
</P><H3><SPAN class="mw-headline" id="83._Labortag_08.08.2010"><P style="font-size:17px; background-color:#00dd77;">83. Labortag 08.08.2010</P></SPAN></H3><P>HT 1080 cells were split and seeded: 3x10^6 cells per into each well of a 6-well plate.
</P><P>A431 cells were washed and received new medium.
</P><P>There were no clones on the agar plates so they were put back into the 37°C room. Maybe there will be some clones tomorrow.
</P><H3><SPAN class="mw-headline" id="84._Labortag_09.08.2010"><P style="font-size:17px; background-color:#00dd77;">84. Labortag 09.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Midi-Prep_Inoculation"><P style="font-size:15px; background-color:#66bbff;"><B>Midi-Prep Inoculation</B></P></SPAN></H4><P><B>Investigators: Chris W., Patrick</B></P><UL><LI>pAAV_RC
</LI><LI>pAAV_RC_1.2 SDM SalI (P158)
</LI><LI>pAAV_RC_mGMK_TK30 clone 1 (P81)
</LI><LI>pAAV_RC_mGMK_TK30 clone 2 (P82)
</LI></UL><H4><SPAN class="mw-headline" id="Continuation:_Cloning_Rep40_.26_Rep52_into_pSB1C3_2"><P style="font-size:15px; background-color:#66bbff;"><B>Continuation: Cloning Rep40 &amp; Rep52 into pSB1C3</B></P></SPAN></H4><P><B>Investigator: Patrick </B>
Two Rep52+DMSO clones grew. They were picked and tomorrow there will be a midi prep and a test digestion.
Simultaneously a new approach for cloning Rep40 and Rep52 into pSB1C3 was performed. (see next header)
</P><H4><SPAN class="mw-headline" id="Repetition:_BioBrick_production_of_Rep40_.26_Rep52"><P style="font-size:15px; background-color:#66bbff;"><B>Repetition: BioBrick production of Rep40 &amp; Rep52</B></P></SPAN></H4><P><B>Investigator: Anna</B></P><P style="font-size:13px; color:#68bbff;"><I><B>Comments</B></I>: Another approach was done because there were no clones for Rep 40 and only two for Rep 52 (Cloning see 06.08.2010). PCR was performed again with DMSO. Samples from gel extraction are stored in 4°C freezer. </P><P><B>PCR </B></P><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD align="right"><B>Rep 40</B></TD><TD align="right"><B>Rep52</B></TD></TR><TR><TD> 5X Phusion HF buffer </TD><TD align="right">10 µl</TD><TD align="right">10 µl
</TD></TR><TR><TD> 10 mM dNTP mix</TD><TD align="right">1µl </TD><TD align="right">1 µl
</TD></TR><TR><TD> forward primer: O094 </TD><TD align="right">2,5µl</TD><TD align="right">2,5 µl
</TD></TR><TR><TD> reverse primer: O0096/ O097 </TD><TD align="right">2,5 µl</TD><TD align="right">2,5 µl
</TD></TR><TR><TD> DNA Template (1ng)</TD><TD align="right">1 µl</TD><TD align="right"> 1 µl
</TD></TR><TR><TD> DMSO </TD><TD align="right">1 µl</TD><TD align="right">1 µl
</TD></TR><TR><TD> Phusion Polymerase</TD><TD align="right">0,5 µl</TD><TD align="right"> 0,5 µl
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">31,5 µl</TD><TD align="right">31,5 µl
</TD></TR><TR><TD>Total volume</TD><TD align="right">50 µl</TD><TD align="right"> 50 µl
</TD></TR></TBODY></TABLE><P><B>PCR programm</B></P><TABLE border="1"><TBODY><TR><TD><B>PCR Program</B></TD><TD align="right"><B>temperature/ °C</B></TD><TD align="right"><B>Time</B></TD><TD align="right"><B>PCR products</B></TD></TR><TR><TD>1</TD><TD align="right">98 </TD><TD align="right">1min</TD></TR><TR><TD>2</TD><TD align="right">98 </TD><TD align="right">15s</TD></TR><TR><TD>8x</TD><TD align="right">*** </TD><TD align="right">15s</TD><TD align="right">63°C
</TD></TR><TR><TD>3 </TD><TD align="right">72</TD><TD align="right">***</TD><TD align="right">15s
</TD></TR><TR><TD>4 </TD><TD align="right">98</TD><TD align="right"> 15s</TD></TR><TR><TD>19x</TD><TD align="right">***</TD><TD align="right">15s</TD><TD align="right">68°C
</TD></TR><TR><TD>5</TD><TD align="right">72</TD><TD align="right">***</TD><TD align="right">15s
</TD></TR><TR><TD>6x</TD><TD align="right">72</TD><TD align="right">5min</TD></TR><TR><TD>Hold</TD><TD align="right">4</TD></TR></TBODY></TABLE><P><B>Agarose gel</B></P><P>0.5  g Agarose, 50 mL TAE (1 %),3 µL GELRED, at 120 Volt, running time: 45 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample / µl]
</TH><TH>Loading dye (6x) / µl
</TH><TH>Expected size
</TH></TR><TR><TD>p22 (Rep40)
</TD><TD>44 µl
</TD><TD>7 µl
</TD><TD>940 bp
</TD></TR><TR><TD>p23 (Rep52)
</TD><TD>44 µl
</TD><TD>7 µl
</TD><TD>~1200 bp
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker /µL
</TH><TH>Sample Rep40 /µl
</TH><TH>Sample Rep52 /µl
</TH></TR><TR><TH>Lane
</TH><TD>7
</TD><TD>51
</TD><TD>51
</TD></TR></TBODY></TABLE><P><B>Gel extraction</B>
PCR products were eluted with 20 µl BE puffer following the standard protocol.
c(Rep40)= 28,03 ng/µl
c(Rep52)= 121,16 ng/µl</P><P style="font-size:13px; color:#68bbff;"><I><B>Comments</B></I>: Digestion was done with the wrong enzymes (EcoRI HF and SpeI), it will be repeated on 11.08. with XbaI and SpeI</P><H4><SPAN class="mw-headline" id="Sequence_analysis_of_pSB1C3_mGMK"><P style="font-size:15px; background-color:#66bbff;"><B>Sequence analysis of pSB1C3_mGMK</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: mGMK was cloned into the pSB1C3 backbone in order to send it to the parts registry!. The sequence analysis revealed that the mGMK was <B>successfully </B> cloned into pSB1C3 plasmid. This polasmid is in the RFC25 standard.</P><UL><B>Results sequencing: Sequence read ok! </B>
Primer used: VR-2

Comment: A mutation has been found in the pSB1C3 backone which was found generally in the backbone which we received from the iGEM HQ.
</UL><H4><SPAN class="mw-headline" id="Mini-Prep_of_pSB1C3_hTERT.2C_pSB1C3_beta-globin_mVenus.2C_pSB1C3_pTAV2.28P5.29.2C_pSB1C3_pAAV_RC.28P5TATAles.29"><P style="font-size:15px; background-color:#66bbff;"><B>Mini-Prep of pSB1C3_hTERT, pSB1C3_beta-globin_mVenus, pSB1C3_pTAV2(P5), pSB1C3_pAAV_RC(P5TATAles)</B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><P><B>Bold text</B>
Mini-Prep following the standart Protokoll
</P><LI>P177 = pSB1C3_hTERT clone1 = 163,6 ng/µl
</LI><LI>P178 = pSB1C3_hTERT clone2 = 312,0 ng/µl
</LI><LI>P179 = pSB1C3_betaglobin_mVenus clone1 = 237,2 ng/µl
</LI><LI>P180 = pSB1C3_betaglobin_mVenus clone2 = 228,2 ng/µl 
</LI><LI>P181 = pSB1C3_pTAV2(P5) clone1 = 108,8ng/µl
</LI><LI>P181 = pSB1C3_pTAV2(P5) clone2 = 140,9 ng/µl
</LI><LI>P181 = pSB1C3_pAAV_RC(P5TATAless) clone1 = 165,0 ng/µl
</LI><LI>P181 = pSB1C3_pAAV_RC(P5TATAless) clone2 = 176,6 ng/µl
</LI><LI>P181 = pSB1C3_pAAV_RC(P5TATAless) clone3 = 176,9 ng/µl
<B>Test digestion:</B></LI><LI>Perfomed with 10 µL of total volume with all 9 clones</LI><TABLE border="1" cellpadding="0" cellspacing="0" width="621" style="border-collapse: collapse;table-layout:fixed;width:466pt"><TBODY><TR style="mso-height-source:userset;height:21.75pt"><TD height="29" class="xl6724191" width="143" style="height:21.75pt;width:107pt"> </TD><TD class="xl6824191" width="131" style="border-left:none;width:98pt">Mastermix/µL</TD><TD class="xl6824191" width="87" style="border-left:none;width:65pt">P176/µL</TD><TD class="xl6824191" width="84" style="border-left:none;width:63pt">P177/µL</TD><TD class="xl6824191" width="90" style="border-left:none;width:68pt">P178/µL</TD><TD class="xl6824191" width="86" style="border-left:none;width:65pt">P179/µL</TD><TD class="xl6824191" width="86" style="border-left:none;width:65pt">P180/µL</TD><TD class="xl6824191" width="86" style="border-left:none;width:65pt">P181/µL</TD><TD class="xl6824191" width="86" style="border-left:none;width:65pt">P182/µL</TD><TD class="xl6824191" width="86" style="border-left:none;width:65pt">P183/µL</TD><TD class="xl6824191" width="86" style="border-left:none;width:65pt">P184/µL</TD></TR><TR style="mso-height-source:userset;height:21.75pt"><TD height="29" class="xl6624191" width="143" style="height:21.75pt;border-top: none;width:107pt"><SPAN lang="EN-US" style="mso-ansi-language:EN-US">DNA</SPAN></TD><TD class="xl6524191" style="border-top:none;border-left:none">-</TD><TD class="xl6524191" style="border-top:none;border-left:none">4,3</TD><TD class="xl6524191" style="border-top:none;border-left:none">2,3</TD><TD class="xl6524191" style="border-top:none;border-left:none">3,0</TD><TD class="xl6524191" style="border-top:none;border-left:none">2,4</TD><TD class="xl6524191" style="border-top:none;border-left:none">6,4</TD><TD class="xl6524191" style="border-top:none;border-left:none">5,0</TD><TD class="xl6524191" style="border-top:none;border-left:none">4,2</TD><TD class="xl6524191" style="border-top:none;border-left:none">4,0</TD><TD class="xl6524191" style="border-top:none;border-left:none">4,0</TD></TR><TR style="mso-height-source:userset;height:21.75pt;mso-yfti-irow: 1"><TD height="29" class="xl6624191" width="143" style="height:21.75pt;border-top: none;width:107pt"><SPAN lang="EN-US" style="mso-ansi-language:EN-US">BSA (10x)</SPAN></TD><TD class="xl6524191" style="border-top:none;border-left:none">10</TD><TD rowspan="4" class="xl6524191" style="border-top:none">3</TD><TD rowspan="4" class="xl6524191" style="border-top:none">3</TD><TD rowspan="4" class="xl6524191" style="border-top:none">3</TD><TD rowspan="4" class="xl6524191" style="border-top:none">3</TD><TD rowspan="4" class="xl6524191" style="border-top:none">3</TD><TD rowspan="4" class="xl6524191" style="border-top:none">3</TD><TD rowspan="4" class="xl6524191" style="border-top:none">3</TD><TD rowspan="4" class="xl6524191" style="border-top:none">3</TD><TD rowspan="4" class="xl6524191" style="border-top:none">3</TD></TR><TR style="mso-height-source:userset;height:21.75pt;mso-yfti-irow: 2"><TD height="29" class="xl6624191" width="143" style="height:21.75pt;border-top: none;width:107pt"><SPAN lang="EN-US" style="mso-ansi-language:EN-US">Buffer
  4(10x)</SPAN></TD><TD class="xl6524191" style="border-top:none;border-left:none">10</TD></TR><TR style="mso-height-source:userset;height:21.75pt;mso-yfti-irow: 3"><TD height="29" class="xl6624191" width="143" style="height:21.75pt;border-top: none;width:107pt"><SPAN lang="EN-US" style="mso-ansi-language:EN-US">Enzyme
  XbaI</SPAN></TD><TD class="xl6524191" style="border-top:none;border-left:none">5</TD></TR><TR style="mso-height-source:userset;height:21.75pt;mso-yfti-irow: 4"><TD height="29" class="xl6624191" width="143" style="height:21.75pt;border-top: none;width:107pt"><SPAN lang="EN-US" style="mso-ansi-language:EN-US">Enzyme
  AgeI</SPAN></TD><TD class="xl6524191" style="border-top:none;border-left:none">5</TD></TR><TR style="mso-height-source:userset;height:21.75pt;mso-yfti-irow: 5;mso-yfti-lastrow:yes"><TD height="29" class="xl6624191" width="143" style="height:21.75pt;border-top: none;width:107pt"><SPAN lang="EN-US" style="mso-ansi-language:EN-US">H2O</SPAN></TD><TD class="xl6524191" style="border-top:none;border-left:none">-</TD><TD class="xl6524191" style="border-top:none;border-left:none">2,7</TD><TD class="xl6524191" style="border-top:none;border-left:none">4,7</TD><TD class="xl6524191" style="border-top:none;border-left:none">4,0</TD><TD class="xl6524191" style="border-top:none;border-left:none">4,6</TD><TD class="xl6524191" style="border-top:none;border-left:none">0,6</TD><TD class="xl6524191" style="border-top:none;border-left:none">2,0</TD><TD class="xl6524191" style="border-top:none;border-left:none">2,8</TD><TD class="xl6524191" style="border-top:none;border-left:none">3,0</TD><TD class="xl6524191" style="border-top:none;border-left:none">3,0</TD></TR><TR style="mso-height-source:userset;height:21.75pt"><TD height="29" class="xl6624191" width="143" style="height:21.75pt;border-top: none;width:107pt">Total volume</TD><TD class="xl6524191" style="border-top:none;border-left:none">30</TD><TD class="xl6524191" style="border-top:none;border-left:none">10</TD><TD class="xl6524191" style="border-top:none;border-left:none">10</TD><TD class="xl6524191" style="border-top:none;border-left:none">10</TD><TD class="xl6524191" style="border-top:none;border-left:none">10</TD><TD class="xl6524191" style="border-top:none;border-left:none">10</TD><TD class="xl6524191" style="border-top:none;border-left:none">10</TD><TD class="xl6524191" style="border-top:none;border-left:none">10</TD><TD class="xl6524191" style="border-top:none;border-left:none">10</TD><TD class="xl6524191" style="border-top:none;border-left:none">10</TD></TR></TBODY></TABLE><UL><LI>All 9 samples were loaded on a 1% agarose gel</LI><LI>P176 = pSB1C3_hTERT clone1 </LI><LI>P177 = pSB1C3_hTERT clone2 </LI><LI>P178 = pSB1C3_betaglobin clone1</LI><LI>P179 = pSB1C3_betaglobin clone2</LI><LI>P180 = pSB1C3_pTAV2(P5) clone1</LI><LI>P181 = pSB1C3_pTAV2(P5) clone2</LI><LI>P182 = pSB1C3_pAAV_RC(P5TATAless) clone1</LI><LI>P183 = pSB1C3_pAAV_RC(P5TATAless) clone2</LI><LI>P184 = pSB1C3_pAAV_RC(P5TATAless) clone3</LI></UL><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (6x)/µl
</TH><TH>Expected size 1 (Geneious)
</TH></TR><TR><TD>P176
</TD><TD>10 µl
</TD><TD>1,6 µl
</TD><TD>474 bp
</TD></TR><TR><TD>P177
</TD><TD>10 µl
</TD><TD>1,6 µl
</TD><TD>474 bp
</TD></TR><TR><TD>P178
</TD><TD>10 µl
</TD><TD>1,6 µl
</TD><TD>509 bp
</TD></TR><TR><TD>P179
</TD><TD>10 µl
</TD><TD>1,6 µl
</TD><TD>509 bp
</TD></TR><TR><TD>P1780
</TD><TD>10 µl
</TD><TD>1,6 µl
</TD><TD>164 bp
</TD></TR><TR><TD>P181
</TD><TD>10 µl
</TD><TD>1,6 µl
</TD><TD>164 bp
</TD></TR><TR><TD>P182
</TD><TD>10 µl
</TD><TD>1,6 µl
</TD><TD>164 bp
</TD></TR><TR><TD>P183
</TD><TD>10 µl
</TD><TD>1,6 µl
</TD><TD>164 bp
</TD></TR><TR><TD>P184
</TD><TD>10 µl
</TD><TD>1,6 µl
</TD><TD>164 bp
</TD></TR></TBODY></TABLE><P style="font-size:13px; color:#66bbff;"><B>Comments:</B> hTERT and betaglobin_mVenus is inserted in pSB1C3, pTAV(P5) and pAAV_RC (P5TATAless) will be sequenced</P><P>Primer: Reverse Primer VR2
</P><H4><SPAN class="mw-headline" id="Cell_culture_2"><P style="font-size:15px; background-color:#66bbff;"><B>Cell culture</B></P></SPAN></H4><P><B>Investigators: Adrian, Kerstin, Patrick</B></P><H5><SPAN class="mw-headline" id="Transfection_on_HEK293">Transfection on HEK293</SPAN></H5><P>
The motivation: first we want to create new stock of AAV without GOI, the next viral stock is with correct TK/GMK, the last stocks are with different YFP-amounts to check if the amount of GOI is a critical step for transduction effency (transgene expression).
</P><TABLE border="1"><TBODY><TR><TD> Plate  </TD><TD align="right">pAAV_RC/µg</TD><TD align="right">pHelper/µg</TD><TD align="right">GOI/µg
</TD></TR><TR><TD> 1</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> no GOI
</TD></TR><TR><TD> 2</TD><TD align="right">6,6 </TD><TD align="right"> 3,3</TD><TD align="right"> no GOI
</TD></TR><TR><TD> 3</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 3,3 TK_GMK clone1
</TD></TR><TR><TD> 4</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 3,3 TK_GMK clone1
</TD></TR><TR><TD> 5</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 3,3 TK_GMK clone2
</TD></TR><TR><TD> 6</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 3,3 YFP
</TD></TR><TR><TD> 7</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 10 YFP
</TD></TR><TR><TD> 8</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 20 YFP
</TD></TR><TR><TD> 9</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 40 YFP
</TD></TR><TR><TD> 10</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 60 YFP
</TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="Transduction_of_HT1080">Transduction of HT1080</SPAN></H5><P>Motivation
The Motivation: checking survivalrate of Cells, with either no Virus, virus without GOI and virus with YFP (10µg stock)
The Plan: 
Plate I
</P><TABLE border="1"><TBODY><TR><TD> control, no cells </TD><TD align="right">no GOI 150µl</TD><TD align="right">10µg 500µl|
</TD></TR><TR><TD> control, no virus</TD><TD align="right">no GOI 300µl</TD><TD align="right"> 10µg 750µl|
</TD></TR></TBODY></TABLE><P>
Plate II
</P><TABLE border="1"><TBODY><TR><TD> control, no cells </TD><TD align="right">no GOI 150µl</TD><TD align="right">10µg 500µl|
</TD></TR><TR><TD> control, no virus</TD><TD align="right">no GOI 300µl</TD><TD align="right"> 10µg 750µl|
</TD></TR></TBODY></TABLE><P>Plate III
</P><TABLE border="1"><TBODY><TR><TD> control, no cells </TD><TD align="right">no GOI 150µl</TD><TD align="right">10µg 500µl|
</TD></TR><TR><TD> control, no virus</TD><TD align="right">no virus</TD><TD align="right"> 10µg 750µl|
</TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="Passaging_and_seeding_of_A431">Passaging and seeding of A431</SPAN></H5><P>one 6er dish was prepared for making pictures and one T75 Flask.
</P><H4><SPAN class="mw-headline" id="BioBrick_Assembly_of_pSB1C3_leftITR_CMV_and_pSB1C3_hGH_rightITR"><P style="font-size:15px; background-color:#66bbff;"><B>BioBrick Assembly of pSB1C3_leftITR_CMV and pSB1C3_hGH_rightITR</B></P></SPAN></H4><P><B>Investigator: Bea, Achim, Jo</B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: In order to proceed with the BioBrick assembly the pasmids pSB1C3_left ITR and pSB1C3_CMV were used and pSB1C3_rigthITR and pSB1C3_hGH were used in order to obatin plasmids with both sequences.</P><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>BioBrick assembly of pSB1C3_hGH_rightITR and pSB1C3_leftITR_CMV</CAPTION><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>Cloning strategy of pSB1C3_leftITR CMV
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>pSB1C3_hGH_rightITR
</P></DIV></DIV></TD></TR></TBODY></TABLE><P><B>Digestion:</B></P><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>pSB1C3_left ITR</B></TD><TD align="right"><B>pSB1C3_CMV</B></TD><TD align="right"><B>pSB1C3_right ITR</B></TD><TD align="right"><B> pSB1C3_hGH</B></TD></TR><TR><TD> DNA  </TD><TD align="right">13,4 </TD><TD align="right">9,5 </TD><TD align="right">13,6 </TD><TD align="right">6,6
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right"> 2 </TD><TD align="right">2 </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right">2 </TD><TD align="right">2 </TD><TD align="right"> 2
</TD></TR><TR><TD>Enzyme1 </TD><TD align="right">1 (SpeI) </TD><TD align="right">1 (Xba1)</TD><TD align="right">1 (EcoRI) </TD><TD align="right">1 (EcoRI)
</TD></TR><TR><TD>Enzyme2 </TD><TD align="right">1 (PstI)</TD><TD align="right">1 (PstI) </TD><TD align="right">1 (XbaI)</TD><TD align="right">1 (SpeI)
</TD></TR><TR><TD>H2O</TD><TD align="right">0,6 </TD><TD align="right">4,5 </TD><TD align="right">0,4 </TD><TD align="right">7,4
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 20</TD><TD align="right">20 </TD><TD align="right"> 20</TD><TD align="right">20
</TD></TR></TBODY></TABLE><P>Preparation of gel:
0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 115 Volt, running time: 45 minutes
</P><P>Expected sizes of constructs:
</P><UL><LI>pSB1C3_CMV:  650 bp
</LI><LI>pSB1C3_lITR: 2200 bp
</LI><LI>pSB1C3_hGH:  500 bp
</LI><LI>pSB1C3_rITR: 2200 bp
</LI></UL><P>The correspnding bands were cut out and Gel-Extraction was performed according to protocol.
</P><P>concentrations measured via NanoDrop:
</P><UL><LI>pSB1C3_CMV:  13,51 ng/µl
</LI><LI>pSB1C3_lITR: 18,49  ng/µl
</LI><LI>pSB1C3_hGH:  7,22  ng/µl
</LI><LI>pSB1C3_rITR: 20,24  ng/µl
</LI></UL><P><B>Ligation:</B>
For ligation, T4 ligase was used.
</P><UL><LI> 1µl T4-Ligase buffer (10x)
</LI><LI> 8µl (Vector + Insert) mix
</LI><LI> 1µl T4-Ligase
</LI></UL><P>Ligation was carried out as following:
</P><UL><LI> Ligation 1:
</LI><LI>pSB1C3_CMV:  4,4 ng/µl
</LI><LI>pSB1C3_lITR: 3,6  ng/µl
</LI></UL><LI>Ligation 2:
</LI><LI>pSB1C3_hGH:  5,25 µl
</LI><LI>pSB1C3_rITR: 2,75 µl
</LI><P> Incubation for 40 minutes.
</P><P><B>Transformation:</B>
Transformation carried out according to standard protocol (BL21). Cells were plated on an agar plate with chloramphenicol.
</P><H4><SPAN class="mw-headline" id="Repetition:_Cloning_of_SDM_SspI_and_SDM_PvuII"><P style="font-size:15px; background-color:#66bbff;"><B>Repetition: Cloning of SDM SspI and SDM PvuII</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P style="color:red;">Comments: Last week's cloning of SDM_SspI and SDM_PvuII didn't work out, so this is the repetition. In this approach the same restriction enzymes and amounts of DNA were used.</P><UL><LI>Vector: name: pSB1C3_SDM_SspI <B>P125</B></LI><LI>Insert: name: pSB1C3_SDM_PvuII <B>P129</B></LI></UL><UL><LI>buffer used: 4
</LI><LI>Restriction-enzymes used:
</LI><LI>BpmI (no. Lab: 144)
</LI><LI>ClaI (no.Lab: 152)
</LI></UL><LI>DNA concentration (P125): 321,9 ng/µl
</LI><LI>DNA concentration (P129): 308,4 ng/µl
</LI><P><B>Digestion:</B></P><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume of pSB1C3_SDM_SspI /µl</B></TD><TD align="right"><B>volume of pSB1C3_SDm_PvuII /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">4,66 </TD><TD align="right">4,87
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right">2
</TD></TR><TR><TD>BpmI (no.Lab:144)</TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>ClaI (no.Lab:152)</TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">9,34 </TD><TD align="right">9,13
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 20</TD><TD align="right">20
</TD></TR></TBODY></TABLE><P><B>Gel extraction:</B>
Preparation of gel:
0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 115 Volt, running time: 45 minutes
</P><P>Expected sizes of constructs:
</P><UL><LI>pSB1C3_SDM_SspI: 1819 bp
</LI><LI>pSB1C3_SDM_PvuII: 981 bp
</LI></UL><P>The correspnding bands were cut out and Gel-Extraction was performed according to protocol.
</P><P>concentrations measured via NanoDrop:
</P><UL><LI> Sspl: 19,85 ng/µl
</LI><LI> PvuII: 10,57 ng/µl
</LI></UL><P><B>Ligation:</B>
For ligation, T4 ligase was used.
</P><UL><LI> 1µl T4-Ligase buffer (10x)
</LI><LI> 8µl (Vector + Insert) mix
</LI><LI> 1µl T4-Ligase
</LI></UL><P>The Ligation was performed as following:
</P><UL><LI> SDM_SspI Volume: 4,03 µl
</LI><LI> SDM_PvuII Volume: 3,97 µl
</LI></UL><UL><LI> 1µl T4-Ligase buffer (10x)
</LI><LI> 8µl (Vector + Insert) mix
</LI><LI> 1µl T4-Ligase
</LI></UL><P> Incubating for 45 minutes.
</P><P><B>Transformation:</B>
Transformation performed according to the standard protocol (BL21). The cells were plated on a agar plate with chloramphenicol.
</P><H4><SPAN class="mw-headline" id="Design_and_ordering_of_primer_for_P40_Promoter_BioBrick_production"><P style="font-size:15px; background-color:#66bbff;"><B>Design and ordering of primer for P40 Promoter BioBrick production</B></P></SPAN></H4><P><B>Investigator: Hanna</B>
In order to generate a P40 promoter BioBrick = &quot;Cap-Promoter&quot;, oligos were designed and ordered at Sigma Aldrich:</P><H3><SPAN class="mw-headline" id="85._Labortag_10.08.2010"><P style="font-size:17px; background-color:#00dd77;">85. Labortag 10.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Retrafo_with_RepCap_gene_.28Mr.Gene.29"><P style="font-size:15px; background-color:#66bbff;"><B>Retrafo with RepCap gene (Mr.Gene)</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="font-size:13px; color:#66bbff;"><B>Comments</B>: We received the rep_cap insert ordered from Mr. Gene today. In order to used and clone the insert into the desired sequences a retrafo has to be performed.   </P><P><B> Re-transformation: </B></P><UL><LI>Construct used: pMA_REP_CAP</LI><LI> We reveived 5 µg (lyophilzied). These were resuspended in 100 µL H<SUB>2</SUB>O and vortexted.</LI><LI>c = 50 ng/µL</LI><LI>c = Plasmid need to be diluted in order to obatain the final 500pg</LI><LI>Dilution: 1:100</LI><LI>100 µL aliquot of <B>XL1-B</B> were thawed on ice. The proper amount of 1µL was added to the cells.</LI><LI>Transformed cells were plated on LB plates containing ampicillin</LI></UL><H4><SPAN class="mw-headline" id="Continuation:_Cloning_Rep40_.26_Rep52_into_pSB1C3_3"><P style="font-size:15px; background-color:#66bbff;"><B>Continuation: Cloning Rep40 &amp; Rep52 into pSB1C3</B></P></SPAN></H4><P>A midi prep of the two Rep52+DMSO clones was performed yielding concentrations of 261,35 ng/µl (clone 1) and 240,47 ng/µl (clone 2). Unfortunately we noticed today that we should not use EcoRI to clone Rep52 into pSB1C3 because EcoRI cuts two times within the sequence of Rep52.
</P><H4><SPAN class="mw-headline" id="Midi-Prep_of_pAAV_RC.2C_pAAV_RC_1.2_SDM_SalI_.28P158.29.2C_pAAV_RFC25_mGMK_TK30_clone_1_.28P81.29_and_2_.28P82.29"><P style="font-size:15px; background-color:#66bbff;"><B>Midi-Prep of pAAV_RC, pAAV_RC_1.2 SDM SalI (P158), pAAV_RFC25_mGMK_TK30 clone 1 (P81) and 2 (P82)</B></P></SPAN></H4><P><B>Investigators: Patrick, Chris W.</B></P><P>The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
</P><TABLE border="1"><TBODY><TR><TD> pAAV_RC </TD><TD align="right">P81a</TD><TD align="right">P81b</TD><TD align="right">P82a</TD><TD align="right">P82b</TD><TD align="right">P158a</TD><TD align="right">P158b
</TD></TR><TR><TD> 1348,49 ng/µl</TD><TD align="right">1783,86 ng/µl</TD><TD align="right"> 2096,25 ng/µl</TD><TD align="right">1915 ng/µl</TD><TD align="right">1469,35 ng/µl</TD><TD align="right">1834,25 ng/µl</TD><TD align="right">1771,27 ng/µl
</TD></TR></TBODY></TABLE><P>
The annotations a and b were added because two midi-preps of P81, P82 and P158 were carried out.
</P><H4><SPAN class="mw-headline" id="Results_of_sequencing_pSB1C3_pTAV2.28P5.29_and_Psb1C3_pAAV_RC.28P5TATAless.29"><P style="font-size:15px; background-color:#66bbff;"><B>Results of sequencing pSB1C3_pTAV2(P5) and Psb1C3_pAAV_RC(P5TATAless)</B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><P>@Jessi: Which clones are ok, which not?? text does not correspond to pictures! (Bea)
</P><P>
sequencing of <B>P180</B> was successful, P5 with TATAbox is in pSB1C3
</P><P>
sequencing of <B>P183</B> was successful but there is a G missing --&gt; plamid is thrown away
</P><P>
sequencing of <B>P184</B> was successful, P5 is in without TATAbox. this plasmid can be used. <B>P181</B> and <B>P180</B> are thrown away because of the result of the test digestion.
</P><H4><SPAN class="mw-headline" id="Repetition:_BioBrick_production_of_Rep40_and_Rep_52"><P style="font-size:15px; background-color:#66bbff;"><B>Repetition: BioBrick production of Rep40 and Rep 52</B></P></SPAN></H4><P><B>Investigator: Anna</B></P><P style="font-size:13px; color:#68bbff;"><I><B>Comments</B></I>: PCR was repeated once again (that was bad!), because digestion was done with the wrong enzymes (EcoRI HF and SpeI). There are some differences in the PCR programm in comparison to the last (see 09.10.)</P><P><B>PCR </B></P><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD align="right"><B>Rep 40</B></TD><TD align="right"><B>Rep52</B></TD></TR><TR><TD> 5X Phusion HF buffer </TD><TD align="right">10 µl</TD><TD align="right">10 µl
</TD></TR><TR><TD> 10 mM dNTP mix</TD><TD align="right">1µl </TD><TD align="right">1 µl
</TD></TR><TR><TD> forward primer: O094 </TD><TD align="right">2,5µl</TD><TD align="right">2,5 µl
</TD></TR><TR><TD> reverse primer: O0096/ O097 </TD><TD align="right">2,5 µl</TD><TD align="right">2,5 µl
</TD></TR><TR><TD> DNA Template 1:100 (1ng)</TD><TD align="right">1 µl</TD><TD align="right"> 1 µl
</TD></TR><TR><TD> DMSO </TD><TD align="right">1 µl</TD><TD align="right">1 µl
</TD></TR><TR><TD> Phusion Polymerase</TD><TD align="right">0,5 µl</TD><TD align="right"> 0,5 µl
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">31,5 µl</TD><TD align="right">31,5 µl
</TD></TR><TR><TD>Total volume</TD><TD align="right">50 µl</TD><TD align="right"> 50 µl
</TD></TR></TBODY></TABLE><P><B>PCR programm</B></P><TABLE border="1"><TBODY><TR><TD><B>PCR Program</B></TD><TD align="right"><B>temperature/ °C</B></TD><TD align="right"><B>Time</B></TD><TD align="right"><B>PCR products</B></TD></TR><TR><TD>1</TD><TD align="right">98 </TD><TD align="right">1min</TD></TR><TR><TD>2</TD><TD align="right">98 </TD><TD align="right">15s</TD></TR><TR><TD>8x</TD><TD align="right">*** </TD><TD align="right">15s</TD><TD align="right"><B>62,5°C</B></TD></TR><TR><TD>3 </TD><TD align="right">72</TD><TD align="right">***</TD><TD align="right"><B>25s</B></TD></TR><TR><TD>4 </TD><TD align="right">98</TD><TD align="right"> 15s</TD></TR><TR><TD>19x</TD><TD align="right">***</TD><TD align="right">15s</TD><TD align="right"><B>67°C</B></TD></TR><TR><TD>5</TD><TD align="right">72</TD><TD align="right">***</TD><TD align="right"><B>25s</B></TD></TR><TR><TD>6x</TD><TD align="right">72</TD><TD align="right">5min</TD></TR><TR><TD>Hold</TD><TD align="right">4</TD></TR></TBODY></TABLE><P><B>Agarose gel</B></P><P>0.5  g Agarose, 50 mL TAE (1 %),3 µL GELRED, at 120 Volt, running time: 45 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample / µl]
</TH><TH>Loading dye (6x) / µl
</TH><TH>Expected size
</TH></TR><TR><TD>p22 (Rep40)
</TD><TD>50 µl
</TD><TD>8 µl
</TD><TD>940 bp
</TD></TR><TR><TD>p23 (Rep52)
</TD><TD>50 µl
</TD><TD>8 µl
</TD><TD>~1200 bp
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker /µL
</TH><TH>Sample Rep40 /µl
</TH><TH>Sample Rep52 /µl
</TH></TR><TR><TH>Lane
</TH><TD>7
</TD><TD>58
</TD><TD>58
</TD></TR></TBODY></TABLE><P style="font-size:13px; color:#68bbff;"><I><B>Comments</B></I>: Digestion of the samples will be done tomorrow with XbaI and SpeI.</P><P><B>Gelextraction</B></P><P>Elution was done with 20 µl Pe buffer following the standard protocol.</P><P>c(Rep40)= 23,1 ng/µl
c(Rep52)= 78,0 ng/µl
</P><H4><SPAN class="mw-headline" id="Preparation_of_competent_XL1blue_and_testtrafo"><P style="font-size:15px; background-color:#66bbff;"><B>Preparation of competent XL1blue and testtrafo</B></P></SPAN></H4><P><B>Investigator: Jessica</B>
preparation of competent E.coli XL1blue was made according to the standardprotocol (91 eppis with <B>60µl</B>) they are in the green boxes in -80°C
testtrafo was made
</P><UL><LI>one plate with 50pg/µl</LI><LI>one plate with 100pg/µl</LI></UL><P>...trafo is failed because amp was missing on the plates (resistence in pUC). trafo will be repeated on aug 11th by Tobi...
</P><H4><SPAN class="mw-headline" id="Repetition:_Mini-Prep_and_test_digestion_of_pSB1C3_SDM_SspI.2FPvuII"><P style="font-size:15px; background-color:#66bbff;"><B>Repetition: Mini-Prep and test digestion of pSB1C3_SDM_SspI/PvuII</B></P></SPAN></H4><P><B>Investigator: Adrian, Stefan</B></P><P style="font-size:18px; font-weight: bold; color:#66bbff;"><U>Plasmid Mini-Prep</U></P><UL><LI>vector name: pSB1C3_SDM_SspI/PvuII
</LI></UL><P><U>Glycerol Stocks</U>
pSB1C3_SDM_SspI/PvuII
Only clone 1 grew, therefore there is just one stock = B159
</P><P style="font-size:15px; font-weight: bold; color:#66bbff;">Test digestion</P><UL><LI>buffer used: 2
</LI><LI>Restriction-enzymes used:
</LI><LI>Enzyme 1 SspI
</LI><LI>Enzyme 2 PvuII
</LI></UL><LI>Plasmid: pSB1C3_SDM_SspI/PvuII:
</LI><LI>Plasmid-Number: P185; DNA concentration: 47,9 ng/µL </LI><P style="font-size:13px; color:#ff0000;">comment: DNA was eluted into previously used tube, therefore contamination is possible.</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>P185</B> Volume/µL </TD><TD align="left"><B>P51.1</B> Volume/µL
</TD></TR><TR><TD align="left"> DNA </TD><TD align="left"> 18,1 </TD><TD align="left"> 6,8
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 2,5 </TD><TD align="left"> 2,5
</TD></TR><TR><TD align="left"> Buffer no. 2 (10x) </TD><TD align="left"> 2,5 </TD><TD align="left"> 2,5
</TD></TR><TR><TD align="left"> Enzyme 1 SspI </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5
</TD></TR><TR><TD align="left"> Enzyme 2 PvuII </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 0,9 </TD><TD align="left"> 12,2
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>10</B></TD><TD align="left"><B>10</B> |
</TD></TR></TBODY></TABLE><UL><LI>Incubation: 45 min
</LI></UL><P style="font-size:15px; font-weight: bold; color:#66bbff;">Agarose-Gel:</P><P>
0.4 g Agarose, 50 mL <B>TAE</B>, 3 µL GELRED, at 115 Volt, running time: 45 minutes
</P><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker /µL
</TH><TH>Sample P185 /µL
</TH><TH>Sample P51.1 /µL
</TH></TR><TR><TH>Lane
</TH><TD>7
</TD><TD>17
</TD><TD>17
</TD></TR></TBODY></TABLE><P>One cut is expected (PvuII in CFP).</P><P><B>Comment: There are 6 bands now. This test digestion again looks very different from that of P157.2 done by Jessica (lab day 82). Results should be discussed tomorrow.</B></P><H4><SPAN class="mw-headline" id="Picking_clones_of_pSB1C3_SDM_SspI.2FPvuII"><P style="font-size:15px; background-color:#66bbFF;"><B>Picking clones of pSB1C3_SDM_SspI/PvuII</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P style="font-size:13px; color:#66bbff;"><B>Comments:</B> Test digestion didn't work out, three additional clones were picked from the plate we get from yesterday's Trafo to retry tomorrow.</P><UL><LI>Bacterial strain used: BL21</LI><LI>Inoculating of 10 mL DYT medium containing 10µL chloramphenicol</LI><LI> Put in 37°C room on shaker at 20.45hours.</LI></UL><H3><SPAN class="mw-headline" id="86._labday_11.08.2010"><P style="font-size:17px; background-color:#00dd77;">86. labday 11.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Transduction_of_HT1080_with_no_AAV.2C_AAV_without_GOI_and_AAV_with_10.C2.B5g"><P style="font-size:15px; background-color:#66bbff;"><B>Transduction of HT1080 with no AAV, AAV without GOI and AAV with 10µg</B></P></SPAN></H4><P><B>The motivation</B>: In the past, the percentage of living cells in FACS was very low (~ 70%) so we want to check if the AAV-induced stress is responsible for cellular death, or the detaching procedure (trypsin and mechanical stress).<B>The plan</B>
Plate I
</P><TABLE border="1"><TBODY><TR><TD> control, no cells</TD><TD align="right">no GOI 150µl</TD><TD align="right">10µg 500µl
</TD></TR><TR><TD> control, no virus</TD><TD align="right">no GOI 300µl</TD><TD align="right"> 10µg 750µl
</TD></TR></TBODY></TABLE><P>
Plate II
</P><TABLE border="1"><TBODY><TR><TD> control, no cells </TD><TD align="right">no GOI 150µl</TD><TD align="right">10µg 500µl
</TD></TR><TR><TD> control, no virus</TD><TD align="right">no GOI 300µl</TD><TD align="right"> 10µg 750µl
</TD></TR></TBODY></TABLE><P>Plate III
</P><TABLE border="1"><TBODY><TR><TD> control, no cells </TD><TD align="right">no GOI 150µl</TD><TD align="right">10µg 500µl
</TD></TR><TR><TD> control, no virus</TD><TD align="right">no virus</TD><TD align="right"> 10µg 750µl
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Biobrick_Assembly:Picking_Clones.2C_Miniprep.2C_Testdigestion_of_CMV.26lITR.2F_hGH_.26_rITR"><P style="font-size:15px; background-color:#66bbff;"><B>Biobrick Assembly:Picking Clones, Miniprep, Testdigestion of CMV&amp;lITR/ hGH &amp; rITR</B></P></SPAN></H4><P><B>Investigator: Chris L., Achim</B>
Test Digestion showed that the cloning was succesful.

NanoDrop concentrations:
</P><UL><LI>hGH_rITR1:105,15 ng/µl
</LI><LI>hGH_rITR2:96.97 ng/µl
</LI><LI>CMV_lITR1:231.61 ng/µl
</LI><LI>CMV_lITR2:129.19 ng/µl
</LI></UL><H4><SPAN class="mw-headline" id="Biobrick_Assembly:_bSB1C3_lITR_CMV_.26_pSB1C3_betaglobin_mVenus"><P style="font-size:15px; background-color:#66bbff;"><B>Biobrick Assembly: bSB1C3_lITR_CMV &amp; pSB1C3_betaglobin_mVenus</B></P></SPAN></H4><P><B>Investigator: Chris L., Achim</B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: In order to proceed with the BioBrick assembly the plasmids pSB1C3_leftITR_CMV was cut with SpeI and PstI, pSB1C3_betaglobin_mVenus was cut with XbaI and PstI. The insert fragment containing betaglobin_mVenus was ligated to the vector fragment containing the leftITR_CMV promotor.</P><P><B>Digestion:</B></P><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>pSB1C3_leftITR_CMV</B></TD><TD align="right"><B>pSB1C3_betaglobin_mVenus</B></TD></TR><TR><TD> DNA  </TD><TD align="right">5,2 </TD><TD align="right">8,4
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right">2
</TD></TR><TR><TD>Enzyme1 </TD><TD align="right">1 (SpeI) </TD><TD align="right">1 (Xba1)
</TD></TR><TR><TD>Enzyme2 </TD><TD align="right">1 (PstI)</TD><TD align="right">1 (PstI)
</TD></TR><TR><TD>H2O</TD><TD align="right">8,8 </TD><TD align="right">5,6
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 20</TD><TD align="right">20
</TD></TR></TBODY></TABLE><P>Preparation of gel:
0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 115 Volt, running time: 45 minutes
</P><P>Expected sizes of constructs:
</P><UL><LI>pSB1C3_leftITR_CMV:  2800 bp
</LI><LI>pSB1C3_betaglobin_mVenus: 1200 bp, 2000 bp
</LI></UL><P>The corresponding bands were cut out and Gel-Extraction was performed according to protocol.
</P><P>concentrations measured via NanoDrop:
</P><UL><LI>pSB1C3_leftITR_CMV: 55,5 ng/µl
</LI><LI>pSB1C3_betaglobin_mVenus:: 10,4 ng/µl
</LI></UL><H4><SPAN class="mw-headline" id="Continuation_of_Cloning_of_Rep40.2F52"><P style="font-size:15px; background-color:#66bbff;"><B>Continuation of Cloning of Rep40/52</B></P></SPAN></H4><P><B>Investigator: Anna</B></P><UL><LI>Digestion of PCR products and vector:
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>PCR product /µl</B></TD><TD align="right"><B>vector /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">19 </TD><TD align="right">10
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">3 </TD><TD align="right">3
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">3 </TD><TD align="right">3
</TD></TR><TR><TD>Enzyme <B>XbaI </B></TD><TD align="right">1 </TD><TD align="right">1</TD></TR><TR><TD>Enzyme <B>SpeI </B></TD><TD align="right">1 </TD><TD align="right">1</TD></TR><TR><TD>H2O</TD><TD align="right">3 </TD><TD align="right">12
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 30</TD><TD align="right">30
</TD></TR></TBODY></TABLE><P style="font-size:13px; color:#68bbff;"><I><B>Comments</B></I>: This time digestion was done with XbaI and SpeI (means it has to be checked if the inserts are cloned into the vector in the right orientation). </P><UL><LI>Purification of Rep40 and Rep52:
</LI></UL><P>For the purification 95 µl of buffer PBI was used.
</P><P>c(Rep40)= 12,5 ng/µl 
c(Rep52)= 51,8  ng/µl </P><UL><LI>Gelextraction of pSB1C3_RFC25_CFP:
</LI></UL><P>0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 120 Volt, running time:55 
5µl loading dye (6x) for the sample, Marker: GeneRuler ladder mix (Fermentas)
</P><P>
c(pSB1C3)= 10,59 ng/µl 
c(pSB1C3_2)= 4,25 ng/µl
</P><UL><LI>Ligation of PCR products and vector:
</LI></UL><P>For the Ligation 1µl T4 buffer (2x) and 1µl T4 ligase were used.
</P><TABLE border="1"><TBODY><TR><TD align="right"><B>Sample-no.</B></TD><TD align="right"><B>vector /µl</B></TD><TD align="right"><B>insert /µl</B></TD></TR><TR><TD> pSB1C3 + Rep40 </TD><TD align="right">1 </TD><TD align="right">3,65 </TD><TD align="right">4,35
</TD></TR><TR><TD> pSB1C3 + Rep52 </TD><TD align="right">2 </TD><TD align="right">5,87 </TD><TD align="right">2,13
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 8</TD></TR></TBODY></TABLE><UL><LI>Transformation:
</LI></UL><P>The transformation was done following the standard protocol using B21 cells.
</P><H4><SPAN class="mw-headline" id="PCR_of_bla_14FM_and_ligation_into_pSB1C3_SDM_SspI_and_pSB1C3_SDM_PvuII_for_virobrick-production_T"><P style="font-size:15px; background-color:#66bbff;"><B>PCR of bla_14FM  and ligation into pSB1C3_SDM_SspI and pSB1C3_SDM_PvuII for virobrick-production T</B></P></SPAN></H4><P><B>Investigator: Anissa,Kerstin</B></P><P style="font-size:13px; color:#68bbff;"><I><B>Comments</B></I>: Because it still is unclear if SDM of pSB1C3_SspI_PvuII has worked, we decided to use the one time mutagenisised vector pSB1C3_SDM_SspI, to clone bla into it and to make the second mutation of PvuII after bla is inserted.  . </P><UL><LI><B>PCR:</B></LI></UL><P>(was performed following the standard protocol)
</P><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD align="right"><B>v(pTB106_bla14FM) - DMSO / µl</B></TD><TD align="right"><B>v(pTB106_bla14FM) + DMSO / µl</B></TD></TR><TR><TD> 5X Phusion HF buffer </TD><TD align="right">10 </TD><TD align="right">10
</TD></TR><TR><TD> 10 mM dNTP mix</TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD> forward primer: O116(P extreme f)</TD><TD align="right">2,5 </TD><TD align="right">2,5
</TD></TR><TR><TD> reverse primer: O117 (P extreme r) </TD><TD align="right">2,5 </TD><TD align="right">2,5
</TD></TR><TR><TD> DNA Template 1ng </TD><TD align="right">5,3 µl </TD><TD align="right">5,3 µl
</TD></TR><TR><TD> DMSO (1%)</TD><TD align="right"> 0,5 </TD><TD align="right"> 0,5
</TD></TR><TR><TD> Phusion Polymerase</TD><TD align="right">0,5 </TD><TD align="right">0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">28,2 </TD><TD align="right">27,7
</TD></TR><TR><TD>Total volume</TD><TD align="right">50 </TD><TD align="right">50
</TD></TR></TBODY></TABLE><UL><LI><B> PCR program:</B></LI></UL><TABLE border="1"><TBODY><TR><TD><B>PCR Program</B></TD><TD align="right"><B>temperature/ °C</B> |<B>time</B></TD></TR><TR><TD>1</TD><TD align="right">98  </TD><TD align="right"> 1'
</TD></TR><TR><TD>2</TD><TD align="right">98 </TD><TD align="right"> 15
</TD></TR><TR><TD>8x</TD><TD align="right">61  </TD><TD align="right"> 25
</TD></TR><TR><TD>3 </TD><TD align="right">72 </TD><TD align="right"> 15
</TD></TR><TR><TD>4 </TD><TD align="right">98 </TD><TD align="right"> 15
</TD></TR><TR><TD>5</TD><TD align="right">72 </TD><TD align="right"> 15
</TD></TR><TR><TD>1x</TD><TD align="right">72 </TD><TD align="right"> 5'
</TD></TR><TR><TD>Hold</TD><TD align="right">4
</TD></TR></TBODY></TABLE><UL><LI><B>Digestion PCR product:</B></LI></UL><UL><LI> DNA-Concentration: 4,2 ng/µl
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume / µl</B></TD></TR><TR><TD> DNA </TD><TD align="right"> 28,5
</TD></TR><TR><TD> BSA 100x </TD><TD align="right"> 0,4
</TD></TR><TR><TD>Buffer: 4 10x </TD><TD align="right"> 4
</TD></TR><TR><TD> Enzyme 1 (XbaI) </TD><TD align="right"> 1
</TD></TR><TR><TD> Enzyme 2 (PstI) </TD><TD align="right"> 1
</TD></TR><TR><TD> H<SUB>2</SUB>O </TD><TD align="right"> 5,1
</TD></TR><TR><TD> Total volume </TD><TD align="right"> 40
</TD></TR></TBODY></TABLE><UL><LI><B> Purification of PCR product: (QlAquick Gel Extraction Kit) </B></LI></UL><P>
preparation of purification was made according to the standardprotocol
</P><P style="font-size:13px; color:#FF0033;"><I><B>Comments</B></I>: The wrong enzymes were used. Experiment has to be done tomorrow.</P><H4><SPAN class="mw-headline" id="Test_digestion_pSB1C3_CFP_SDM_SspI_and_PvuII"><P style="font-size:15px; background-color:#66bbFF;"><B>Test digestion pSB1C3_CFP_SDM SspI and PvuII </B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><UL><LI>buffer used: 2; Restriction-enzymes used: Enzyme 1: ClaI ; Enzyme 2: BpmI ; Enzyme 3: SspI ; Enzyme 4: PvuII
</LI></UL><UL><LI>Plasmids:</LI><UL><LI>pSB1C3_SDM_SspI/PvuII <B>P185</B> (first tree lanes, cut with different enzymes)</LI><LI>pSB1C3_RFC25_longlinker <B>P105</B></LI><LI>pSB1C3_CFP <B>P51.1</B></LI></UL></UL><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>P185/µL</B></TD><TD align="left"><B>P185/µL</B></TD><TD align="left"><B>P185/µL</B></TD><TD align="left"><B>P105/µL</B></TD><TD align="left"><B>P51.1/µL</B></TD></TR><TR><TD align="left"> DNA  </TD><TD align="left"> 18</TD><TD align="left"> 18</TD><TD align="left"> 18</TD><TD align="left"> 4,8</TD><TD align="left">6,8
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 2,5</TD><TD align="left"> 2,5</TD><TD align="left"> 2,5</TD><TD align="left"> 2,5</TD><TD align="left"> 2,5
</TD></TR><TR><TD align="left"> Buffer 2 (10x)  </TD><TD align="left"> -</TD><TD align="left"> -</TD><TD align="left"> -</TD><TD align="left"> 2,5</TD><TD align="left"> 2,5
</TD></TR><TR><TD align="left"> Buffer 4 (10x)  </TD><TD align="left"> 2,5</TD><TD align="left"> 2,5</TD><TD align="left"> 2,5</TD><TD align="left"> -</TD><TD align="left"> -
</TD></TR><TR><TD align="left"> Enzyme 1 ClaI (152)  </TD><TD align="left"> 0,5</TD><TD align="left"> 0,5</TD><TD align="left"> 0,5</TD><TD align="left"> -</TD><TD align="left"> -
</TD></TR><TR><TD align="left"> Enzyme 2 BpmI (144)  </TD><TD align="left"> 0,5</TD><TD align="left"> 0,5</TD><TD align="left"> 0,5</TD><TD align="left"> -</TD><TD align="left"> -
</TD></TR><TR><TD align="left"> Enzyme 3 SspI (68)  </TD><TD align="left"> -</TD><TD align="left"> 0,5</TD><TD align="left"> -</TD><TD align="left"> 0,5</TD><TD align="left"> 0,5
</TD></TR><TR><TD align="left"> Enzyme 4 PvuII (50) </TD><TD align="left"> -</TD><TD align="left"> -</TD><TD align="left"> 1</TD><TD align="left"> 0,5</TD><TD align="left"> 0,5
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 1</TD><TD align="left"> -</TD><TD align="left"> 0,5</TD><TD align="left"> 14,2</TD><TD align="left"> 12,2
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"> 25</TD><TD align="left"> 25</TD><TD align="left"> 25</TD><TD align="left"> 25</TD><TD align="left"> 25
</TD></TR></TBODY></TABLE><UL><LI>Incubation: 75 minutes</LI></UL><P><B>Agarosegel</B>
0.5 g Agarose, 50 ml TAE, 3 µL GELRED, at 115 Volt, running time: 60 minutes
</P><P>
comments: Test digestion of pSB1C3_SDM_SspI/PvuII with ClaI, BpmI and SspI looks the same as with ClaI and BpmI. Therefore we can assume that there is no additional restriction site within the vector. But still the bands above 3000 bp cannotz be explained. A new test digestion will be performed tomorrow to check the size of the vector.
The approach cut with ClaI, BpmI and PvuII looks strange. Propably, PvuII does not work correctly. New PvuII-HF was ordered and delivered so this can be verified tomorrow.
pSB1C3_RFC25_longlinker digested with SspI and PvuII seems to be not completely digested This could correspond to non-working PvuII.
pSB1C3_CFP looks like expected.
</P><H4><SPAN class="mw-headline" id="pSB1C3_SDM_Ssp1_.28P125.29_sent_for_sequenzing"><P style="font-size:15px; background-color:#66bbff;"><B>pSB1C3_SDM_Ssp1 (P125) sent for sequenzing</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P>comment: To verify the correct deletion of the SspI restriction site it was sent for sequenzing.
</P><P>Primer used: GATC_std_pTeSp-2
</P><H4><SPAN class="mw-headline" id="Picking_clones_of_pSB1C3_SDM_SspI.2FPvuII_.28P185.29"><P style="font-size:15px; background-color:#66bbff;"><B>Picking clones of pSB1C3_SDM_SspI/PvuII (P185)</B></P></SPAN></H4><P><B>Investigator: Adrian</B></P><P>comment: pSB1C3_SDM_SspI/PvuII (P185) is empty. For plasmid production DYT + Cm was inoculated from the glycerol stock (B159).</P><H3><SPAN class="mw-headline" id="87._labday_12.08.2010"><P style="font-size:17px; background-color:#00dd77;">87. labday 12.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Virus_production.2C_seeding_A431_cell"><P style="font-size:15px; background-color:#66bbff;">Virus production, seeding A431 cell</P></SPAN></H4><P>Investigator: Adrian 
In first place we want to create new stock of AAV without GOI, the next viral stock is with correct TK/GMK, the last stocks are with different YFP-amounts to check if the amount of GOI is a critical step for transduction effency (transgene expression).

Ten viral stocks with 3 ml each were prepared.
</P><TABLE border="1"><TBODY><TR><TD>Plate  </TD><TD align="right">pAAV_RC/µg</TD><TD align="right">pHelper/µg</TD><TD align="right">GOI/µg
</TD></TR><TR><TD> 1</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> no GOI
</TD></TR><TR><TD> 2</TD><TD align="right">6,6 </TD><TD align="right"> 3,3</TD><TD align="right"> no GOI
</TD></TR><TR><TD> 3</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 3,3 TK_GMK clone1
</TD></TR><TR><TD> 4</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 3,3 TK_GMK clone1
</TD></TR><TR><TD> 5</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 3,3 TK_GMK clone2
</TD></TR><TR><TD> 6</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 3,3 YFP
</TD></TR><TR><TD> 7</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 10 YFP
</TD></TR><TR><TD> 8</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 20 YFP
</TD></TR><TR><TD> 9</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 40 YFP
</TD></TR><TR><TD> 10</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 60 YFP
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="XL1blue_are_ready_to_use"><P style="font-size:15px; background-color:#66bbff;"><B>XL1blue are ready to use</B></P></SPAN></H4><P><B>Investigator: Jessica</B>there are 60µl aliquots to use for one trafo
</P><H4><SPAN class="mw-headline" id="Mini-Prep_of_pSB1C3_SDM_SspI.2FPvuII_and_pMA_RC_insert"><P style="font-size:15px; background-color:#66bbff;"><B>Mini-Prep of pSB1C3_SDM_SspI/PvuII and pMA_RC_insert</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P>Gycerol stock was prepared for pMA_RC_insert and labeled B167. No glyerol stock for pSB1C3_SDM_SspI/PvuII because cells from earlier stock werer used for Mini-Prep.
</P><P>Miniprep was performed according to protocol.</P><P>Samples were labeled:</P><UL><LI>pSB1C3_SDM_SspI/PvuII = P185.1
</LI><LI>pMA_RC_insert = P190
</LI></UL><P>Concentrations:
</P><UL><LI>pSB1C3_SDM_SspI/PvuII (P185.1): 257,2 ng/µl
</LI><LI>pMA_RC_insert (P190): 339,5 ng/µl
</LI></UL><P>
No test digestion was performed.
</P><H4><SPAN class="mw-headline" id="Biobrick_Assembly:_pSB1C3_lITR_CMV_.26_pSB1C3_betaglobin_mVenus"><P style="font-size:15px; background-color:#66bbff;"><B>Biobrick Assembly: pSB1C3_lITR_CMV &amp; pSB1C3_betaglobin_mVenus</B></P></SPAN></H4><P><B>Investigator: Chris L., Achim</B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: In order to proceed with the BioBrick assembly the plasmids pSB1C3_leftITR_CMV was cut with SpeI and PstI, pSB1C3_betaglobin_mVenus was cut with XbaI and PstI. The insert fragment containing betaglobin_mVenus was ligated to the vector fragment containing the leftITR_CMV promotor.</P><P><B>Ligation:</B>
For ligation, T4 ligase was used.
</P><UL><LI> 1µl T4-Ligase buffer (10x)
</LI><LI> 8µl (Vector + Insert) mix
</LI><LI> 1µl T4-Ligase
</LI></UL><P>Ligation was carried out as following:
</P><UL><LI>pSB1C3_leftITR_CMV: 6,98 µl 
</LI><LI>pSB1C3_betaglobin_mVenus: 1,02 µl
</LI></UL><P> Incubation for 40 minutes.
</P><UL><LI>Transformation:
</LI></UL><P>The transformation was done following the standard protocol using XL1B cells.
</P><H4><SPAN class="mw-headline" id="Repetition_of_cloning_of_bla14FM_viralbrick_MCS_into_pSB1C3_SDM_SspI"><P style="font-size:15px; background-color:#66bbff;"><B>Repetition of cloning of bla14FM_viralbrick_MCS into pSB1C3_SDM_SspI</B></P></SPAN></H4><P><B>Investigator: Anissa</B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>:Yesterday the pcr-product of bla14FM was cut with the wrong enzymes, so no overhangs could result. Today it will be repeated with the rihgt ones (SpeI and NgomIV) .</P><UL><LI><B>Digestion of the PCR product bla14FM for 2 h :</B></LI></UL><UL><LI> DNA-Concentration: 10,2 ng/µl
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume / µl</B></TD></TR><TR><TD> DNA </TD><TD align="right"> 18
</TD></TR><TR><TD> BSA 100x </TD><TD align="right"> 3
</TD></TR><TR><TD>Buffer: 4 10x </TD><TD align="right"> 3
</TD></TR><TR><TD> Enzyme 1 (NgomIV) </TD><TD align="right"> 1
</TD></TR><TR><TD> Enzyme 2 (SpeI) </TD><TD align="right"> 1
</TD></TR><TR><TD> H<SUB>2</SUB>O </TD><TD align="right"> 4
</TD></TR><TR><TD> Total volume </TD><TD align="right"> 30
</TD></TR></TBODY></TABLE><UL><LI><B> Purification of PCR product: (QlAquick Gel Extraction Kit) </B></LI></UL><P>
preparation of purification was made according to the standardprotocol
</P><UL><LI><B>Digestion  of vector backbone pSB1C3_SDM_SspI for 2 h:</B></LI></UL><UL><LI>1 µg vector has been used
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume / µl</B></TD></TR><TR><TD> DNA </TD><TD align="right"> 3,12
</TD></TR><TR><TD> BSA 100x </TD><TD align="right"> 1,5
</TD></TR><TR><TD>Buffer: 4 10x </TD><TD align="right"> 1,5
</TD></TR><TR><TD> Enzyme 1 (NgomIV) </TD><TD align="right"> 1
</TD></TR><TR><TD> Enzyme 2 (SpeI) </TD><TD align="right"> 1
</TD></TR><TR><TD> H<SUB>2</SUB>O </TD><TD align="right"> 6,88
</TD></TR><TR><TD> Total volume </TD><TD align="right"> 15
</TD></TR></TBODY></TABLE><UL><LI>the cut vector was loaded on a preperative 1% agarose gel and the gel was running for 45 minutes
</LI></UL><UL><LI>gel was cut and gelextraction was performed according the qiagen standard protocol.
</LI></UL><UL><LI><B>Ligation of pSB1C3_SDM_SspI and bla14FM:</B></LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume / µl</B></TD></TR><TR><TD>t4 buffer (10X)  </TD><TD align="right"> 1
</TD></TR><TR><TD> t4 Ligase </TD><TD align="right"> 1
</TD></TR><TR><TD>vector DNA </TD><TD align="right"> 2,72
</TD></TR><TR><TD> Insert </TD><TD align="right"> 5,28
</TD></TR></TBODY></TABLE><UL><LI><B>Transformation: pSB1C3_SDM_SspI_bla14FM was transformed into BL21 following the stanard protocol</B></LI></UL><H4><SPAN class="mw-headline" id="Quickchange_of_the_EaglII_restriction_site_in_the_PMA_RepCap_vector"><P style="font-size:15px; background-color:#66bbff;"><B>Quickchange of the EaglII restriction site in the PMA_RepCap vector</B></P></SPAN></H4><P><B>Investigator: Chris L., Achim</B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: We performed a QuikChange mutagenesis on the synthesized PMA_RepCap vector (P190). The EaglII restriction enzyme also cuts NotI (IGEM standard) therefore EaglII has to be changed into a different recognition site. Primers were designed to change it into PvuII. The new QuikChange Lighting Site-Directed Mutagenesis Kit from Stratagene was used, therefore the mutagenesis was carried out as described in the QuikChange Manual. The PCR product was transformed into XL10-Gold cells, we used the IGEM standard protocol, except for the addition of ß-ME to the XL10-Gold cells.  : </P><P>PCR-reaction:
</P><TABLE border="1"><TBODY><TR><TD><B>Volume / µl</B></TD><TD align="right"><B>ingredients</B></TD><TD align="right"><B>recommended /µl</B></TD></TR><TR><TD> 2,5  </TD><TD align="right">10X reaction buffer </TD><TD align="right"> 2,5
</TD></TR><TR><TD> 2,9 </TD><TD align="right">P190 template (~10 ng)</TD><TD align="right"> 2,9 of 1:100 dilution
</TD></TR><TR><TD> 0,51</TD><TD align="right">forward primer: O122</TD><TD align="right"> 62,5 ng
</TD></TR><TR><TD>0,51 </TD><TD align="right">reverse primer: O123 </TD><TD align="right"> 62,5 ng
</TD></TR><TR><TD>0,5</TD><TD align="right">dNTP</TD><TD align="right"> 250 µM each dNTP
</TD></TR><TR><TD>17,6</TD><TD align="right">H<SUB>2</SUB>O </TD></TR><TR><TD>0,5</TD><TD align="right">QuikChange Lightning ennzyme(1.25) </TD></TR></TBODY></TABLE><P>
PCR program:
</P><TABLE border="1"><TBODY><TR><TD><B>Rounds</B></TD><TD align="right"><B>temperature/ °C</B></TD><TD align="right"><B>Time</B></TD></TR><TR><TD>1 </TD><TD align="right">95 </TD><TD align="right"> 2 minutes
</TD></TR><TR><TD>18 </TD><TD align="right">95</TD><TD align="right">20 seconds
</TD></TR><TR><TD>18 </TD><TD align="right">60 </TD><TD align="right"> 10 seconds
</TD></TR><TR><TD>18</TD><TD align="right">68</TD><TD align="right"> 2 minutes (4 kb * 30 sec)
</TD></TR><TR><TD>1</TD><TD align="right">68</TD><TD align="right"> 5 minutes
</TD></TR></TBODY></TABLE><P>
The PCR product was digestet with Dpn I for 10 minutes at 37°C
</P><P>Plasmid was transformed into XL10-Gold cells.
</P><H4><SPAN class="mw-headline" id="Sequencing_results_of_Rep68.2C_Rep78_and_AAP_BioBricks"><P style="font-size:15px; background-color:#66bbff;"><B>Sequencing results of Rep68, Rep78 and AAP BioBricks</B></P></SPAN></H4><P><B>Investigator: Hanna</B>
Comment: pSB1C3_Rep78 (P160), pSB1C3_Rep68 (P165) and pSB1C3_AAP (P170) were sent for sequencing twice. No sequencing made sense. Paradoxically the Rep78 sequencing result fit to the p5 promoter BioBrick... 
By having a closer look to the BioBrick PCR we found out that the fragment sizes didn't fit to the expected sequences. In addition to that the test digestion also showed some discrepancies. 
Because we weren't able to find out in which steps of the BioBrick production things went wrong, we decided to discard all referring glycerol stocks and plasmids and to perform the Rep68, Rep78 and AAP BioBrick PCR again!!!
</P><H4><SPAN class="mw-headline" id="Comment_13.08.2010_.28Hanna.29:_We_wondered_how_the_whole_Rep_stuff_could_be_sequenced_reverse_AND_forward_without_any_forward_sequencing_primer..._By_checking_the_.22sequencing_book.22_we_found_out_that_always_the_wrong_primers_were_used:_the_GOI_primers_can_be_used_in_order_to_sequence_the_Gene_of_interest_.3D_GOI_.28.21.29_in_pAAV_MCS_and_NOT_the_pSB1C3.21.21.21.21.21.21.21.21.21"><P style="font-size:15px; background-color:#9acd32;"><B>Comment 13.08.2010 (Hanna):</B> We wondered how the whole Rep stuff could be sequenced reverse AND forward without any forward sequencing primer... By checking the &quot;sequencing book&quot; we found out that always the wrong primers were used: the GOI primers can be used in order to sequence the <B>G</B>ene <B>o</B>f <B>i</B>nterest = GOI (!) in pAAV_MCS and NOT the pSB1C3!!!!!!!!!</P></SPAN></H4><H4><SPAN class="mw-headline" id="Order_oligos_for_CD-biobrick"><P style="font-size:15px; background-color:#66bbFF;"><B>Order oligos for CD-biobrick</B></P></SPAN></H4><P><B>Investigator: Jessica and Kira</B>ordered on aug 12th</P><H4><SPAN class="mw-headline" id="BioBrick_assembly_of_pSB1C3_leftITR_CMV_mVenus"><P style="font-size:15px; background-color:#66bbff;"><B>BioBrick assembly of pSB1C3_leftITR_CMV_mVenus</B></P></SPAN></H4><P><B>Investigator: Stefan </B></P><P>comment: In order to assemble our complete constructs mVenus has to be included into the pSB1C3_leftITR_CMV vector. <B>Digestion:</B></P><TABLE border="1"><TBODY><TR><TD> components:   </TD><TD align="right">pSB1C3_leftITR_CMV P188(Vector) </TD><TD align="right">pGA14_mVenus P60 (Insert)
</TD></TR><TR><TD> DNA  </TD><TD align="right">6 </TD><TD align="right">8
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right">2
</TD></TR><TR><TD>Enzyme1 </TD><TD align="right">1 (SpeI) </TD><TD align="right">1 (Xba1)
</TD></TR><TR><TD>Enzyme2 </TD><TD align="right">1 (PstI)</TD><TD align="right">1 (PstI)
</TD></TR><TR><TD>H2O</TD><TD align="right">8 </TD><TD align="right">6
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 20</TD><TD align="right">20
</TD></TR></TBODY></TABLE><P>
Preparation of gel:
0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 110 Volt, running time: 45 minutes
</P><P>Expected sizes of constructs:
</P><UL><LI>pSB1C3_leftITR_CMV:  2869 bp
</LI><LI>pGA14_mVenus: 2896 bp, 756 bp
</LI></UL><P>The corresponding bands were cut out and Gel-Extraction was performed according to protocol.
</P><P>concentrations measured via NanoDrop:
</P><UL><LI>pSB1C3_leftITR_CMV: 39,2 ng/µl
</LI><LI>pGA14_mVenus: 8,2 ng/µl
</LI></UL><P><B>Ligation:</B></P><P>T4 ligation was performed according to protocol. </P><P>Volumes used:
</P><UL><LI>vector: 1,67 ng/µl
</LI><LI>insert: 6,33 ng/µl
</LI></UL><P><B>Transformation:</B></P><P>bacterial strain used: XL1bB</P><P>antibiotic used for plate: chlorampenicol</P><P>Transformation was performed according to protocol.
Plate was prepared and put in 37°C room.
</P><H4><SPAN class="mw-headline" id="Opened_a_BioBrick_box"><P style="font-size:15px; background-color:#66bbff;"><B>Opened a BioBrick box</B></P></SPAN></H4><P><B>Investigator: Bea</B>
In order to &quot;save&quot; the plasmids/BioBricks which has been prepared and confirmed for submitting it to the parts registry a new box have been prepared. The box should contain 30 µL of the desired construct cloned into the pSB1C3 vector and the BBa number given from the headquarter. </P><P>The first BioBricks which are in this box are:
</P><UL><LI>pSB1C3_hGH (P136): Bba_K404002</LI><LI>pSB1C3_betaglobin (P133: Bba_K404001 </LI><LI>pSB1C3_CMV (P145): Bba_K404003</LI><LI>pSB1C3_leftITR (P175): Bba_K404006</LI><LI>pSB1C3_rightITR (P172): Bba_K404005</LI><LI>pSB1C3_mGMK (P156): Bba_K404004</LI></UL><P>In our nomenclature plasmid excel list the constructs which we aliquoted should be highlighted in blue and commented with the given Bba number. Additionally it should be taken in consideration that enough DNA should be in the &quot;working&quot; plasmid box. Therefore, if necessary, inoculate DYT medium for Mini-Preps.
In this case ALL the glycerol stocks were used  for incoluating 10 mL DYT medium containing 10 µL chloramphenicol respectively.
</P><UL><P style="color:#66bbff;">To do: Mini-Preps of the over night cultures.</P></UL><H3><SPAN class="mw-headline" id="88._labday_13.08.2010"><P style="font-size:17px; background-color:#00dd77;">88. labday 13.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Seeding_HT1080_for_transduction_at_saturday.2C_Seeding_HEK293_for_transfection"><P style="font-size:15px; background-color:#66bbff;"><B> Seeding HT1080 for transduction at saturday, Seeding HEK293 for transfection</B></P></SPAN></H4><P>Investigators: Kerstin, Adrian
We seeded cells according to standard protocols.
</P><H4><SPAN class="mw-headline" id="FACS_of_transduction_from_11.8"><P style="font-size:15px; background-color:#66bbff;"><B>FACS of transduction from 11.8</B></P></SPAN></H4><P>The YFP expression is still quite low. This was kind of anticipated, the success in this experiment is the fact, <FONT color="#FF0000">that the amount of living cells is at a very high level! (~90%)</FONT>. <B>The conclusion</B>: Detach the cells fast! Use PBS which is not at 37°C (the PBS was 25min in the water bath), transfer the cells fast on ice after detaching!<B>The next steps are</B>:
</P><UL><LI>Is it possible, that freeze/thaw circles reduce the transduction efficiency?</LI><LI>Why is the YFP-expression that low? (~28-30%) </LI></UL><H4><SPAN class="mw-headline" id="Continuation:pSB1C3_SDM_SspI_bla14FM"><P style="font-size:15px; background-color:#66bbff;"><B>Continuation:pSB1C3_SDM_SspI_bla14FM</B></P></SPAN></H4><P><B>Investigator: Patrick </B>
Intention: receive enough plasmid for a sequencing and maybe futher working steps. 
Three clones were picked in the morning to inoculate DYT and perform a mini-prep in the evening and to send them for sequencing. The three clones were picked and prepped according to the standard protocol yielding the following DNA concentrations:
</P><UL><LI>pSB1C3_SDM_SspI_bla14FM clone 1: 65,15 ng/µl , Sequencing labeled: PS3.1
</LI><LI>pSB1C3_SDM_SspI_bla14FM clone 2: 22,3 ng/µl , Sequencing labeled: PS3.2
</LI><LI>pSB1C3_SDM_SspI_bla14FM clone 3: 62,3 ng/µl , Sequencing labeled: PS3.3
</LI></UL><P>
Used primer: seq_pSB1C3_VR2_rev (O51) 
Results: see <A href="http://www.molbiotech.uni-freiburg.de/iGEM/wiki2010/index.php/Laborjournal#Sequence_analysis_of_pSB1C3_SDM_SspI_BLA" class="external free" rel="nofollow">http://www.molbiotech.uni-freiburg.de/iGEM/wiki2010/index.php/Laborjournal#Sequence_analysis_of_pSB1C3_SDM_SspI_BLA</A></P><H4><SPAN class="mw-headline" id="Mini-preps_of_pSB1C3_hgh.2CpSB1C3_beta_globin.2CpSB1C3_left_ITR.2CpSB1C3_right_ITR.2CpSB1C3_mGMK.2CpSB1C3_CMV"><P style="font-size:15px; background-color:#66bbff;"><B>Mini-preps of pSB1C3_hgh,pSB1C3_beta globin,pSB1C3_left ITR,pSB1C3_right ITR,pSB1C3_mGMK,pSB1C3_CMV</B></P></SPAN></H4><P style="font-size:13px; color:#003399;"><B>Comments</B>:new mini-preps of the glycerol-stocks ofpSB1C3_hgh(=B103),pSB1C3_beta globin(=B100),pSB1C3_left ITR (=B143),pSB1C3_right ITR (=B140),pSB1C3_mGMK (=B125),pSB1C3_CMV   : </P><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>concentration in ng/µl</B></TD></TR><TR><TD> P191 (pSB1C3_right ITR)</TD><TD align="right"> 100,74
</TD></TR><TR><TD>P192 (pSB1C3_beta-globin) </TD><TD align="right">240,08
</TD></TR><TR><TD>P193 (pSB1C3_CMV) </TD><TD align="right"> 333,89
</TD></TR><TR><TD>P194 (pSB1C3_hgh) </TD><TD align="right">328,42
</TD></TR><TR><TD> P195 (pSB1C3_mGMK) </TD><TD align="right">258,63
</TD></TR><TR><TD> P196 (pSB1C3_left ITR) </TD><TD align="right">173,98
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Design_and_ordering_of_oligos_for_VP_fusion_targeting_approaches"><P style="font-size:15px; background-color:#66bbff;"><B>Design and ordering of oligos for VP fusion targeting approaches</B></P></SPAN></H4><P><B>Investigator: Hanna</B></P><P><B>Comment:</B> In order to target the EGFR receptor of our A431 cells different approaches have been figured out: Besides loop insertions, N-terminal fusion to the VP2 protein will be performed. In addition to that a VP1 insertion approach (after Grieger et al., 2007) was planned. 
Fusion and insertion molecules will be e.g. His-tag, Affibody, Darpin, GFP. These approaches can be combinated with the loop insertions.  
For this purpose the following oligos were ordered today: </P><H4><SPAN class="mw-headline" id="Mini-Prep_and_Test_digestion_of_Rep40_and_Rep52_of_the_cloning_experiment_.28.22Cloning_of_Rep40.2F52.22_11.08.2010_Investigator:_Anna.29"><P style="font-size:15px; background-color:#66bbff;"><B>Mini-Prep and Test digestion of Rep40 and Rep52 of the cloning experiment (&quot;Cloning of Rep40/52&quot; 11.08.2010 Investigator: Anna) </B></P></SPAN></H4><P><B>Investigator: Chris L., Achim</B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: There are small and big colonies on the plate. Maybe problems with the agar plate. Picked colonies of pSB1C3_Rep40 sample 1 and pSB1C3_Rep52 sample 1 are small colonies. Picked colonies of pSB1C3_Rep40 sample 2 + 3 and pSB1C3_Rep52 sample 2 + 3 are standard size colonies.</P><P><B>Nanodrop</B></P><UL><LI> Rep 40 Sample 1: 156,26 ng/µl <B>P197</B></LI><LI> Rep 40 Sample 2: 200,05 ng/µl <B>P198</B></LI><LI> Rep 40 Sample 3: 144,06 ng/µl <B>P199</B></LI><LI> Rep 52 Sample 1: 173,36 ng/µl <B>P200</B></LI><LI> Rep 52 Sample 2: 160,75 ng/µl <B>P201</B></LI><LI> Rep 52 Sample 3: 295,77 ng/µl <B>P202</B></LI></UL><P><B>Test digestion</B> 800ng DNA
</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">P145 /µl </TD><TD align="right">P146 /µl </TD><TD align="right">P145 /µl </TD><TD align="right">P146 /µl</TD><TD align="right">P145 /µl </TD><TD align="right">P146 /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 5,1</TD><TD align="right">4</TD><TD align="right">5,6</TD><TD align="right">4,6</TD><TD align="right">5</TD><TD align="right">2,7
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right"> 2</TD><TD align="right">2</TD><TD align="right">2</TD><TD align="right">2</TD><TD align="right">2</TD><TD align="right">2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right"> 2</TD><TD align="right">2</TD><TD align="right">2</TD><TD align="right">2</TD><TD align="right">2</TD><TD align="right">2
</TD></TR><TR><TD> Enzyme 1: XbaI </TD><TD align="right"> 0,5</TD><TD align="right">0,5</TD><TD align="right">0,5</TD><TD align="right">0,5</TD><TD align="right">0,5</TD><TD align="right">0,5
</TD></TR><TR><TD> Enzyme 2: SpeI </TD><TD align="right"> 0,5</TD><TD align="right">0,5</TD><TD align="right">0,5</TD><TD align="right">0,5</TD><TD align="right">0,5</TD><TD align="right">0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right"> 9,9</TD><TD align="right">11</TD><TD align="right">9,4</TD><TD align="right">10,4</TD><TD align="right">10</TD><TD align="right">12,3
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"><B>20</B></TD><TD align="right"><B>20</B></TD><TD align="right"><B>20</B></TD><TD align="right"><B>20</B></TD><TD align="right"><B>20</B></TD><TD align="right"><B>20</B></TD></TR></TBODY></TABLE><P>
Incubation time: 45 minutes, Incubation temperature: 37°

0,5 g Agarose, 50ml x TAE, 3µl GELRED, at 115 Volt, running time: 45 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (6x)/µl
</TH><TH>Expected size 1 (Geneious)
</TH><TH>Expected size 2 (Geneious)
</TH></TR><TR><TD>Rep 40 1
</TD><TD>20 µl
</TD><TD> 4 µl
</TD><TD>____ bp
</TD><TD> 940 bp
</TD></TR><TR><TD>Rep 40 2
</TD><TD>20 µl
</TD><TD> 4 µl
</TD><TD>____ bp
</TD><TD> 940 bp
</TD></TR><TR><TD>Rep 40 3
</TD><TD>20 µl
</TD><TD> 4 µl
</TD><TD>____ bp
</TD><TD> 940 bp
</TD></TR><TR><TD>Rep 52 1
</TD><TD>20 µl
</TD><TD> 4 µl
</TD><TD>____ bp
</TD><TD>1198 bp
</TD></TR><TR><TD>Rep 52 2
</TD><TD>20 µl
</TD><TD> 4 µl
</TD><TD>____ bp
</TD><TD>1198 bp
</TD></TR><TR><TD>Rep 52 3
</TD><TD>20 µl
</TD><TD> 4 µl
</TD><TD>____ bp
</TD><TD>1198 bp
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>Sample Rep40 1 /20µl
</TH><TH>Sample Rep40 2 /20µl
</TH><TH>Sample Rep40 3 /20µl
</TH><TH>Sample Rep52 1 /20µl
</TH><TH>Sample Rep52 2 /20µl
</TH><TH>Sample Rep52 3 /20µl
</TH></TR><TR><TH>Lane
</TH><TD> 1
</TD><TD> 2
</TD><TD> 3
</TD><TD> 4
</TD><TD> 5
</TD><TD> 6
</TD><TD> 7
</TD></TR></TBODY></TABLE><P>agarose gel: 1%, digestion: 45 minutes, 37°C 
</P><P style="font-size:13px; color:#68bbff;"><I><B>Comments:</B></I>Results look bad. Rep52 clone 3 looks good, send for sequencing. </P><PRE><B>&lt;/p&gt;</B></PRE><H4><SPAN class="mw-headline" id="Repetition_of_Rep_40_and_Rep_52_ligation_and_transformation_with_the_vector_pSB1C3_CFP"><P style="font-size:15px; background-color:#66bbff;"><B>Repetition of Rep 40 and Rep 52 ligation and transformation with the vector pSB1C3_CFP</B></P></SPAN></H4><P><B>Investigator: Chris L.</B></P><UL><LI>Ligation of PCR products and vector:
</LI></UL><P>For the Ligation 1µl T4 buffer (10x) and 1µl T4 ligase were used.
</P><TABLE border="1"><TBODY><TR><TD align="right"><B>Sample-no.</B></TD><TD align="right"><B>vector /µl</B></TD><TD align="right"><B>insert /µl</B></TD></TR><TR><TD> pSB1C3 + Rep40 </TD><TD align="right">1 </TD><TD align="right">4,92 </TD><TD align="right">3,08
</TD></TR><TR><TD> pSB1C3 + Rep52 </TD><TD align="right">2 </TD><TD align="right">6,47 </TD><TD align="right">1,53
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 10 </TD><TD align="right"> 10
</TD></TR></TBODY></TABLE><UL><LI>Transformation:
</LI></UL><P>The transformation was done following the standard protocol using XL1B cells.
</P><H4><SPAN class="mw-headline" id="Strategy_for_VP2_N-terminal_fusion_and_VP1_insertion_of_targeting_moelcules"><P style="font-size:15px; background-color:#66bbff;"><B>Strategy for VP2 N-terminal fusion and VP1 insertion of targeting moelcules</B></P></SPAN></H4><P><B>Investigator: Hanna</B></P><P><B>Comment:</B> In order to target the EGFR receptor of tumor cells via Affibody or Darpin (antibody fragment is also possible!!!), or in order to expose e.g. His-Tags on the virus surface, two approaches have been figured out. </P><DIV class="center"><DIV class="floatnone"><A href="/File:Freiburg10_N-terminalFusion_Strategy3.JPG" title="File:Freiburg10 N-terminalFusion Strategy3.JPG">File:Freiburg10 N-terminalFusion Strategy3.JPG</A></DIV></DIV><H4><SPAN class="mw-headline" id=".3Cp_style.3D.22font-size:15px.3B_background-color:.2366bbff.3B.22.3EpSB1C3_Rep52_.28P202.29_sent_for_sequencing">&lt;p style=&quot;font-size:15px; background-color:#66bbff;&quot;&gt;pSB1C3_Rep52 (P202) sent for sequencing</SPAN></H4><P><B>Investigator: Chris L.</B></P><P>Primer used: Reverse Primer VR2
</P><H4><SPAN class="mw-headline" id="Repetition:_Biobrick_production_of_Rep_68.2C_78_and_AAP"><P style="font-size:15px; background-color:#66bbff;"><B>Repetition: Biobrick production of Rep 68, 78 and AAP</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P> 	
Aim of the experiment:
We want to produce biobricks from Rep 68, Rep 78 and the AAP.
</P><UL><LI>Plasmids used as template:
</LI></UL><P>Rep_68_ex (p119):   c = 470,6 ng/µl 
Rep_78_(p122):      c = 566,8 ng/µl 
pAAV-RC containing AAP ORF (p50):    c = 378,5 ng/µl </P><UL><LI>Primer used:
</LI></UL><P>For Rep_68_ex: Praefix_68_78_ex &amp; Suffix_40_68_ex
For Rep_78_ex: Praefix_68_78_ex &amp; Suffix_52_78_ex
For AAP_ex: Praefix_AAP_ex &amp; Suffix_AAP_ex
</P><UL><LI><B>PCR:</B></LI></UL><P>(was performed following the standard protocol)
</P><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD align="right"><B>Volume / µl</B></TD><TD align="right"><B>Rep68</B></TD><TD align="right"><B>Rep78</B></TD><TD align="right"><B>AAP</B></TD></TR><TR><TD> 5X Phusion HF buffer </TD><TD align="right">10 </TD></TR><TR><TD> 10 mM dNTP mix</TD><TD align="right">1</TD></TR><TR><TD> forward primer: </TD><TD align="right">2,5</TD></TR><TR><TD> reverse primer: </TD><TD align="right">2,5</TD></TR><TR><TD> DNA Template</TD><TD align="right">***</TD><TD align="right">2,5 µl </TD><TD align="right">2 µl</TD><TD align="right">3 µl
</TD></TR><TR><TD> DMSO (2%)</TD><TD align="right"> - </TD><TD align="right"> - </TD><TD align="right"> 1 µl
</TD></TR><TR><TD> Phusion Polymerase</TD><TD align="right">0,5</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">*** </TD><TD align="right"> 31 µl</TD><TD align="right"> 31,5 µl</TD><TD align="right">29,5 µl
</TD></TR><TR><TD>Total volume</TD><TD align="right">50
</TD></TR></TBODY></TABLE><P>PCR program:
</P><TABLE border="1"><TBODY><TR><TD><B>PCR Program</B></TD><TD align="right"><B>temperature/ °C</B></TD><TD align="right"><B>Time</B></TD><TD align="right"><B>Rep68</B></TD><TD align="right"><B>Rep78</B></TD><TD align="right"><B>AAP</B></TD></TR><TR><TD>1</TD><TD align="right">98 </TD><TD align="right">1min</TD></TR><TR><TD>2</TD><TD align="right">98 </TD><TD align="right">15s</TD></TR><TR><TD>3</TD><TD align="right">*** </TD><TD align="right">25s</TD><TD align="right">63°C</TD><TD align="right">62°C</TD><TD align="right">64°C
</TD></TR><TR><TD>4 (step 2-4 8x) </TD><TD align="right">72</TD><TD align="right">***</TD><TD align="right">24s </TD><TD align="right">27s </TD><TD align="right"> 10s
</TD></TR><TR><TD>5 </TD><TD align="right">98</TD><TD align="right"> 15s</TD></TR><TR><TD>6</TD><TD align="right">***</TD><TD align="right">25s</TD><TD align="right">68°C</TD><TD align="right">64°C</TD><TD align="right">68°C
</TD></TR><TR><TD>7</TD><TD align="right">72</TD><TD align="right">***</TD><TD align="right">24s</TD><TD align="right">27s</TD><TD align="right">10s
</TD></TR><TR><TD>8 (step 6-8 17x)</TD><TD align="right">72</TD><TD align="right">5min</TD></TR><TR><TD>Hold</TD><TD align="right">4</TD></TR></TBODY></TABLE><P>Used agarose gel:
0,5 g Agarose,50 ml TBE (1%), 3 µl GELRED , at 115 Volt, running time:45 minutes
</P><P style="font-size:15px; font-weight: bold; color: blue;">Gel extraction</P><P>
Gel extraction was perfomed according to protocol.
</P><UL><LI>Digestion of PCR products and vector:
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>PCR products /µl</B></TD><TD align="right"><B>vector /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">29 </TD><TD align="right">11
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">4 </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">4 </TD><TD align="right">2
</TD></TR><TR><TD>Enzyme <B>XbaI </B></TD><TD align="right">1 </TD><TD align="right">1</TD></TR><TR><TD>Enzyme <B>SpeI </B></TD><TD align="right">1 </TD><TD align="right">1</TD></TR><TR><TD>H2O</TD><TD align="right">1 </TD><TD align="right">3
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 40</TD><TD align="right">20
</TD></TR></TBODY></TABLE><P style="font-size:13px; color:#68bbff;"><I><B>Comments</B></I>: Digestion was done with XbaI and SpeI, it has to be checked if the inserts are cloned into the vector in the right orientation. </P><UL><LI>Purification of Rep68, 78 and AAP:
</LI></UL><P>For the purification 200 µl of buffer PBI was used.
</P><P>c(Rep68)= 14,3 ng/µl 
c(Rep78)= 12,0 ng/µl 
c(AAP)= 25,1 ng/µl
</P><UL><LI>Gelextraction of pSB1C3_RFC25_CFP:
</LI></UL><P>0,5 g Agarose,50 ml TBE (1%), 3 µl GELRED , at 115 Volt, running time:50 minutes 
4µl loading dye (6x) for the sample, Marker: GeneRuler ladder mix (Fermentas)
</P><P>
c(pSB1C3)= 6,9 ng/µl </P><UL><LI>T4 ligation of PCR products and vector:
</LI></UL><P>For the Ligation 1µl buffer (10x) and 1µl T4 ligase were used.
</P><TABLE border="1"><TBODY><TR><TD align="right"><B>vector /µl</B></TD><TD align="right"><B>insert /µl</B></TD></TR><TR><TD> pSB1C3 + Rep68 </TD><TD align="right">3,78 </TD><TD align="right">4,24
</TD></TR><TR><TD> pSB1C32 + Rep78 </TD><TD align="right">3,13 </TD><TD align="right">4,87
</TD></TR><TR><TD> pSB1C3 + AAP </TD><TD align="right">6,42 </TD><TD align="right">1,58
</TD></TR></TBODY></TABLE><UL><LI>Transformation:
</LI></UL><P>The transformation was done following the standard protocol using XL1b cells.
</P><H3><SPAN class="mw-headline" id="89._labday_14.08.2010"><P style="font-size:17px; background-color:#00dd77;">89. labday 14.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Seeding_HEK293_for_transfection_at_monday_16.8"><P style="font-size:15px; background-color:#66bbff;"><B>Seeding HEK293 for transfection at monday 16.8</B></P></SPAN></H4><P>Investigator: Kerstin, Adrian
We want to knoe which confluence of HEK293 cells is optimal for AAV production.<B>The plan</B>
We seed different amounts of cells in 10 cm<SUP>2</SUP>, and check the highest titer.</P><TABLE border="1"><TBODY><TR><TD> Plate  </TD><TD align="right">amount of cells
</TD></TR><TR><TD> 1</TD><TD align="right">100 000
</TD></TR><TR><TD> 2</TD><TD align="right">200 000
</TD></TR><TR><TD> 3</TD><TD align="right">400 000
</TD></TR><TR><TD> 4</TD><TD align="right">500 000
</TD></TR><TR><TD> 5</TD><TD align="right">800 000
</TD></TR><TR><TD> 6</TD><TD align="right">1 000 000
</TD></TR><TR><TD> 7</TD><TD align="right">1 200 000
</TD></TR><TR><TD> 8</TD><TD align="right">1 500 000
</TD></TR><TR><TD> 9</TD><TD align="right">1 750 000
</TD></TR><TR><TD> 10</TD><TD align="right">1 750 000
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Transduction_with_mVenus_loaded_viral_particles"><P style="font-size:15px; background-color:#66bbff;"><B>Transduction with mVenus loaded viral particles</B></P></SPAN></H4><P>Investigator: Kerstin, Adrian</P><P>the transduction was performed with following viral stocks:
</P><TABLE border="1"><TBODY><TR><TD> Plate  </TD><TD align="right">pAAV_RC/µg</TD><TD align="right">pHelper/µg</TD><TD align="right">GOI/µg
</TD></TR><TR><TD> 1</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> no GOI
</TD></TR><TR><TD> 2</TD><TD align="right">6,6 </TD><TD align="right"> 3,3</TD><TD align="right"> no GOI
</TD></TR><TR><TD> 3</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 3,3 TK_GMK clone1
</TD></TR><TR><TD> 4</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 3,3 TK_GMK clone1
</TD></TR><TR><TD> 5</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 3,3 TK_GMK clone2
</TD></TR><TR><TD> 6</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 3,3 YFP
</TD></TR><TR><TD> 7</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 10 YFP
</TD></TR><TR><TD> 8</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 20 YFP
</TD></TR><TR><TD> 9</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 40 YFP
</TD></TR><TR><TD> 10</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 60 YFP
</TD></TR></TBODY></TABLE><P>Platte 1:</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>300µl virus 1</TD><TD>300µl virus 2</TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>600µl virus 1</TD><TD>600µl virus 2</TD></TR></TBODY></TABLE><P>
Platte 2:</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>300µl virus 3</TD><TD>300µl virus 4</TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>600µl virus 3</TD><TD>600µl virus 4</TD></TR></TBODY></TABLE><P>
Platte 3:</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>300µl virus 5</TD><TD>300µl virus 6</TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>600µl virus 5</TD><TD>600µl virus 6</TD></TR></TBODY></TABLE><P>
Platte 4:</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>300µl virus 7</TD><TD>300µl virus 8</TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>600µl virus 7</TD><TD>600µl virus 8</TD></TR></TBODY></TABLE><P>
Platte 5:</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>300µl virus 9</TD><TD>300µl virus 10</TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>600µl virus 9</TD><TD>600µl virus 10</TD></TR></TBODY></TABLE><P>
Platte 6:</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>300µl virus 3</TD><TD>300µl virus 4</TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>600µl virus 3</TD><TD>600µl virus 4</TD></TR></TBODY></TABLE><P>
Platte 7:</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>300µl virus 3</TD><TD>300µl virus 4</TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>500µl virus 3</TD><TD>500µl virus 4</TD></TR></TBODY></TABLE><P>
Platte 8:</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>300µl virus 5</TD><TD>300µl virus 5</TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>600µl virus 5</TD><TD>600µl virus 5</TD></TR></TBODY></TABLE><P>
The test for functionality for TK GMK was done with the Prodrug Cymeven ganciclovir 500mg from Roche. The final concentration was 0,05 mM in the wells. After two days pictures were taken.
</P><H4><SPAN class="mw-headline" id="Sequence_analysis_of_pSB1C3_SDM_SspI_BLA"><P style="font-size:15px; background-color:#66bbff;"><B>Sequence analysis of pSB1C3_SDM_SspI_BLA</B></P></SPAN></H4><P><B>Investigator: Patrick, Bea</B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: Sequence analysis of three clones revealed that all sequences contain some mutations in the regions where the primer annealed. This was due to the extreme primer lentgh. Primers that long should be ordered as HPLC purified in order to avoid mutations in the primer sequence.</P><OL><LI>Clone 1: Mutation in the SalI site of the loop insertion site in the Viral Brick MCS lead to the removal of the recognition site. CANNOT be used for the loop insertions!!!!</LI><LI>Clone 2:  Mutation in the region of the primer upstream of the loop insertion sites SalI and PvuII.</LI><LI>Clone 3: Mutation in the region of the primer downstream of the loop insertion sites SalI and PvuII.</LI></OL><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Sequencing results of pSB1C3_SDM of SspI_BLA clone 2 and pSB1C3_SDM of SspI_BLA clone 3</CAPTION><TBODY><TR><TD><DIV class="gallerybox" style="width: 485px;"><DIV class="gallerytext"><P>Clone 2
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 485px;"><DIV class="gallerytext"><P>Clone 3
</P></DIV></DIV></TD></TR></TBODY></TABLE><P style="font-size:13px; color:#33cc66;"><B>Next steps: </B> Two additional clones will be picked and sent for sequencing (clones 4 and 5). In order to obtain a construct without any mutations, clone 2 and clone 3 couldbe cloned together. Each clone will be digested with SalI and BamHI. Clone 2 will be the &quot;insert&quot;, clone 3 is going to be the &quot;vector&quot;. Therefore the fragments will (hopefully) not contain anymore mutations in the relevant sequence. </P><H4><SPAN class="mw-headline" id="Mini_Preps_of_PMA_RepCap_vector_and_pSB1C3_left_ITR_CMV_betaglobin_mVenus"><P style="font-size:15px; background-color:#66bbff;"><B>Mini Preps of PMA_RepCap vector and pSB1C3_left ITR_CMV_betaglobin_mVenus</B></P></SPAN></H4><P><B>Investigator: Chris L., Bea</B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>:new mini-preps of the glycerol-stocks of pSB1C3_leftITR_CMV_betaglobin_mVenus clone 1 (=B179), pSB1C3_leftITR_CMV_betaglobin_mVenus clone 2 (=B180), pSB1C3_leftITR_CMV_betaglobin_mVenus clone 3 (=B181), pMA_RepCap Vector_SDM_InsPvuII clone 1, (=B182), pMA_RepCap Vector_SDM_InsPvuII clone 2 (=B183) and pMA_RepCap Vector_SDM_InsPvuII clone 3 (=B184)</P><P><B>Comment: The plamsid numbers in the picture (P179-P184) correspond to the glycerol stocks B179-B184. The corresponding plasmid numbers are: P208-P213. </B><B>Nanodrop</B></P><UL><LI> pSB1C3_leftITR_CMV_betaglobin_mVenus clone 1: 256,64 ng/µl <B>B179</B></LI><LI> pSB1C3_leftITR_CMV_betaglobin_mVenus clone 2: 274,86 ng/µl <B>B180</B></LI><LI> pSB1C3_leftITR_CMV_betaglobin_mVenus clone 3: 217,20 ng/µl <B>B181</B></LI><LI> pMA_RepCap Vector_SDM_InsPvuII clone 1: 253,71 ng/µl <B>B182</B></LI><LI> pMA_RepCap Vector_SDM_InsPvuII clone 2: 195,64 ng/µl <B>B183</B></LI><LI> pMA_RepCap Vector_SDM_InsPvuII clone 3: 144,61 ng/µl <B>B184</B></LI></UL><P><B>Test digestion</B> 800ng DNA
</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">B179 /µl </TD><TD align="right">B180 /µl </TD><TD align="right">B181 /µl </TD><TD align="right">B182 /µl</TD><TD align="right">B183 /µl </TD><TD align="right">B184 /µl</TD><TD align="right">pMA_RepCap Vector /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 3,1</TD><TD align="right">2,9</TD><TD align="right">3,7</TD><TD align="right">3,1</TD><TD align="right">4</TD><TD align="right">5,5</TD><TD align="right"> 2,4
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right"> 2</TD><TD align="right">2</TD><TD align="right">2</TD><TD align="right">2</TD><TD align="right">2</TD><TD align="right">2</TD><TD align="right">2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right"> 2</TD><TD align="right">2</TD><TD align="right">2</TD><TD align="right">2</TD><TD align="right">2</TD><TD align="right">2</TD><TD align="right">2
</TD></TR><TR><TD> Enzyme 1: NgoMIV </TD><TD align="right"> 0,5</TD><TD align="right">0,5</TD><TD align="right">0,5</TD><TD align="right">0</TD><TD align="right">0</TD><TD align="right">0 </TD><TD align="right">0
</TD></TR><TR><TD> Enzyme 2: EcoRI </TD><TD align="right"> 0,5</TD><TD align="right">0,5</TD><TD align="right">0,5</TD><TD align="right">0</TD><TD align="right">0</TD><TD align="right">0</TD><TD align="right">0
</TD></TR><TR><TD> Enzyme 2: PvuII </TD><TD align="right"> 0</TD><TD align="right">0</TD><TD align="right">0 </TD><TD align="right">0,5</TD><TD align="right">0,5</TD><TD align="right">0,5</TD><TD align="right">0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right"> 11,9</TD><TD align="right">12,1</TD><TD align="right">11,3</TD><TD align="right">12,4</TD><TD align="right">11,5</TD><TD align="right">10</TD><TD align="right"> 13,1
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"><B>20</B></TD><TD align="right"><B>20</B></TD><TD align="right"><B>20</B></TD><TD align="right"><B>20</B></TD><TD align="right"><B>20</B></TD><TD align="right"><B>20</B></TD><TD align="right"><B>20</B></TD></TR></TBODY></TABLE><P>
Incubation time: 45 minutes, Incubation temperature: 37°

0,5 g Agarose, 50ml x TAE, 3µl GELRED, at 115 Volt, running time: 45 minutes
</P><P><B>Results of test digestion</B></P><P><B>Results</B>: Two (three) different approaches can be seen on the agarose gel.
</P><OL><LI style="font-size:13px; color:#66bbff;">P178-P181 (blue): The plasmid pSB1C3_leftITR_CMV_betaglobin_mVenus was digested with NgoMIV and EcoRI. Expected bands were: <B> 2873bp and 1343 </B>. As it can be seen in the picture, the two fragments ran too high (~300bp). As it is almost normal for the pSB1C3 plasmid, we should figure out why this is always the problem for this vector.</LI><LI style="font-size:13px; color:#cc3333;">P182-P183 (red): The plasmid pMA_RepCap_SDM_insPvuII was digested with PvuII as well as the control vetor pMA_RepCap where no site-directed mutagenesis was performed in order to see the difference. As it can be seen in the picture above, the expected band sizes of <B>263bp,1384bp and 2133bp</B> correspond to the bands detected in the gel. The control vector instead reveals only two bands detectable. Therefore site-directed mutagenesis was successfully performed.</LI><LI style="font-size:13px; color:#33cc33;">P177 (green): pSB1C3_hTERT. For further details see extra part below (Test digestion of pSB1C3_hTERT.</LI></OL><H4><SPAN class="mw-headline" id="Mini-Prep_of_pSB1C3_leftITR_CMV_mVenus"><P style="font-size:15px; background-color:#66bbff;"><B>Mini-Prep of pSB1C3_leftITR_CMV_mVenus</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P>Glycerol stocks were prepared:
</P><LI>B177 = pSB1C3_leftITR_CMV_mVenus clone 1
</LI><LI>B178 = pSB1C3_leftITR_CMV_mVenus clone 2


Mini-Prep following the standard protokol
</LI><LI>P206 = pSB1C3_leftITR_CMV_mVenus clone 1 = 277,9 ng/µl
</LI><LI>P207 = pSB1C3_leftITR_CMV_mVenus clone 2 = 260,3 ng/µl

<B>Test digestion:</B>
or test digestion, pSB1C3_leftITR_CMV (P188) was digested as well. <TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD align="right"><B>clone 1/µl</B></TD><TD align="right"><B>clone 2/µl</B></TD><TD align="right"><B>P188/µl</B></TD></TR><TR><TD> DNA: </TD><TD align="right">3,5 </TD><TD align="right">3,5 </TD><TD align="right">4
</TD></TR><TR><TD> BSA (10x): </TD><TD align="right">2  </TD><TD align="right">2 </TD><TD align="right">2
</TD></TR><TR><TD> Buffer 4: </TD><TD align="right">2 </TD><TD align="right">2 </TD><TD align="right">2
</TD></TR><TR><TD> Enzyme 1 EcoRI: </TD><TD align="right">0,5 </TD><TD align="right">0,5 </TD><TD align="right">0,5
</TD></TR><TR><TD> Enzyme 2 AgeI</TD><TD align="right">0,5 </TD><TD align="right">0,5</TD><TD align="right">0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right"> 6,5</TD><TD align="right"> 6,5</TD><TD align="right">6
</TD></TR></TBODY></TABLE><UL>
All samples were loaded on a 1% agarose gel:
</UL></LI><LI>GeneRuler Ladder Mix (7µl)</LI><LI>P206 = pSB1C3_leftITR_CMV_mVenus clone 1 </LI><LI>P207 = pSB1C3_leftITR_CMV_mVenus clone 2 </LI><LI>P188 = pSB1C3_leftITR_CMV</LI><P>
comment: Through digestion, fragments would have similar sizes. Therefore, a new digestion will be performed tomorrow.
</P><H4><SPAN class="mw-headline" id="Test_digestion_of_pSB1C3_pHTERT"><P style="font-size:15px; background-color:#66bbff;"><B>Test digestion of pSB1C3_pHTERT</B></P></SPAN></H4><P><B>Investigator: Bea </B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: pSB1C3_hTERT was sent for sequencing and revealed that the PCR and the inserting of the PCR product : promoter for hTERT was sucessfully performed. But the sequencing results showed a new AgeI restriciton site in the plasmid backbone. We could not see if the new recognition site was due to the bad sequence read or that there is really a new AgeI site. Therefore, a test digestion will be perfomed with PstI and AgeI in order to figure our wheter AgeI is present in the backbone or not.</P><P style="font-size:13px;"><B>Sequence analysis of pSB1C3_hTERT: </B></P><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Sequencing results of pSB1C3_hTERT</CAPTION></TABLE><P><B>First the sequenced plasmid P177 was digested:</B></P><TABLE border="1" cellpadding="1" cellspacing="1" width="284" style="border-collapse: collapse;table-layout:fixed;width:214pt"><TBODY><TR style="mso-height-source:userset;height:22.5pt"><TD height="30" class="xl6312733" width="142" style="height:22.5pt;width:107pt"> </TD><TD class="xl6412733" width="142" style="border-left:none;width:107pt">P177/µL</TD></TR><TR style="mso-height-source:userset;height:22.5pt"><TD height="30" class="xl6512733" width="142" style="height:22.5pt;border-top:none; width:107pt">DNA</TD><TD class="xl6612733" style="border-top:none;border-left:none">3</TD></TR><TR style="mso-height-source:userset;height:22.5pt"><TD height="30" class="xl6512733" width="142" style="height:22.5pt;border-top:none; width:107pt">BSA (10x)</TD><TD class="xl6612733" style="border-top:none;border-left:none">1,5</TD></TR><TR style="mso-height-source:userset;height:22.5pt"><TD height="30" class="xl6512733" width="142" style="height:22.5pt;border-top:none; width:107pt">Buffer 4(10x)</TD><TD class="xl6612733" style="border-top:none;border-left:none">1,5</TD></TR><TR style="mso-height-source:userset;height:22.5pt"><TD height="30" class="xl6512733" width="142" style="height:22.5pt;border-top:none; width:107pt">Enzyme PstI</TD><TD class="xl6612733" style="border-top:none;border-left:none">0,5</TD></TR><TR style="mso-height-source:userset;height:22.5pt"><TD height="30" class="xl6512733" width="142" style="height:22.5pt;border-top:none; width:107pt">Enzyme AgeI</TD><TD class="xl6612733" style="border-top:none;border-left:none">0,5</TD></TR><TR style="mso-height-source:userset;height:22.5pt"><TD height="30" class="xl6512733" width="142" style="height:22.5pt;border-top:none; width:107pt">H2O</TD><TD>8</TD></TR><TR><TD height="30" width="142" style="height:22.5pt; width:107pt"><B>Total volume</B></TD><TD><B>15</B></TD></TR></TBODY></TABLE><UL><LI>Incubation time: 45minutes</LI><LI>Incubation tempereature: 37°C</LI></UL><P>After the incubation,the samples were loaded on the 1% agarose gel, and run at 110 V for 45 minutes.
<B>Result of the test digestion:</B></P><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Test digestion of pSB1C3_hTERT</CAPTION></TABLE><P style="font-size:13px; color:#33cc33;"><B>Results:</B> As it can be seen in the sequencing results of pSB1C3_hTERT there could have been an additional AgeI site in the vector. If AgeI would have been in the vector, two fragments have been expected by digetsing the plasmid with AgeI and PstI. As it can be seen in the agarose gel, lane 11 (labeled with the green P177) only reveals one lane instead of two. Therefore it can be assumed that NO AgeI site is in the pSB1C3 backbone.</P><H4><SPAN class="mw-headline" id="Picking_clones_..."><P style="font-size:15px; background-color:#66bbff;"><B>Picking clones ... </B></P></SPAN></H4><P><B>Investigator: Patrick</B>
15 Clones were picked saturday night so that a mini-prep could be performed on sunday.
</P><H3><SPAN class="mw-headline" id="90._labday_15.08.2010"><P style="font-size:17px; background-color:#00dd77;">90. labday 15.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Sequence_analysis_of_pSB1C3_Rep52"><P style="font-size:15px; background-color:#66bbff;"><B>Sequence analysis of pSB1C3_Rep52</B></P></SPAN></H4><P><B>Investigator: Bea </B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: Clone 3 was sent for sequencing because this clone revealed the right band sizes of the test digestion performed friday. For more details see: <A href="http://www.molbiotech.uni-freiburg.de/iGEM/wiki2010/index.php/Laborjournal#Mini-Prep_and_Test_digestion_of_Rep40_and_Rep52_of_the_cloning_experiment_.28.22Cloning_of_Rep40.2F52.22_11.08.2010_Investigator:_Anna.29" class="external text" rel="nofollow">Test digestion</A></P><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Sequencing results of pSB1C3_Rep52 clone 3</CAPTION></TABLE><B>Results:</B><P style="color:#00bbff;"> Sequence assembly showed that the suffix of the Rep protein 52 is ok and that PCR worked well. </P><P style="color:#990000;">There can be seen that a silent mutation from AGG to AGA was inserted, but it will have no effect on the protein because codon usage in <I>H. sapiens </I>is nearly the same for both codons.</P><P>The prefix could not be analysed because of bad sequence read quality. For further verification, another sequencing with the forward primer has to be performed.
<B>Next steps: Send plasmid for another sequencing round with forward primer in order to analyse the prefix. </B><A href="http://www.molbiotech.uni-freiburg.de/iGEM/wiki2010/index.php/Laborjournal" class="external text" rel="nofollow">top of page</A></P><H4><SPAN class="mw-headline" id="Mini-Prep_of_pSB1C3_Rep68.2C_pSB1C3_Rep78.2C_pSB1C3_RepAAP.2C_pSB1C3_Rep40.2C_pSB1C3_Rep52"><P style="font-size:15px; background-color:#66bbff;"><B>Mini-Prep of pSB1C3_Rep68, pSB1C3_Rep78, pSB1C3_RepAAP, pSB1C3_Rep40, pSB1C3_Rep52</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><UL><B>Glycerol stocks were prepared:</B><LI>B185 = pSB1C3_Rep68 clone 1
</LI><LI>B186 = pSB1C3_Rep68 clone 2
</LI><LI>B187 = pSB1C3_Rep68 clone 3</LI><LI>B188 = pSB1C3_Rep78 clone 1
</LI><LI>B189 = pSB1C3_Rep78 clone 2
</LI><LI>B190 = pSB1C3_Rep78 clone 3</LI><LI>B191 = pSB1C3_AAP clone 1
</LI><LI>B192 = pSB1C3_AAP clone 2
</LI><LI>B193 = pSB1C3_AAP clone 3</LI><LI>B194 = pSB1C3_Rep40 clone 1
</LI><LI>B195 = pSB1C3_Rep40 clone 2</LI><LI>B196 = pSB1C3_Rep52 clone 1
</LI><LI>B197 = pSB1C3_Rep52 clone 2</LI><LI>B198 = pSB1C3_SDM_SspI_Bla14FM clone 4
</LI><LI>B199 = pSB1C3_SDM_SspI_Bla14FM clone 5<B>Mini-Prep following the standard protokol</B></LI><LI>P208 = pSB1C3_Rep68 clone 1 c=230,0
</LI><LI>P209 = pSB1C3_Rep68 clone 2 c=160,5
</LI><LI>P210 = pSB1C3_Rep68 clone 3 c=128,6 </LI><LI>P211 = pSB1C3_Rep78 clone 1 c=150,8
</LI><LI>P212 = pSB1C3_Rep78 clone 2 c=108,6
</LI><LI>P213 = pSB1C3_Rep78 clone 3 c=131,3</LI><LI>P214 = pSB1C3_AAP clone 1 c=142,1
</LI><LI>P215 = pSB1C3_AAP clone 2 c=148,3
</LI><LI>P216 = pSB1C3_AAP clone 3 c=163,0</LI><LI>P217 = pSB1C3_Rep40 clone 1 c=164,5
</LI><LI>P218 = pSB1C3_Rep40 clone 2 c=&lt;145,1br /&gt;</LI><LI>P219 = pSB1C3_Rep52 clone 1 c=156,0
</LI><LI>P220 = pSB1C3_Rep52 clone 2 c=164,0</LI><LI>P221 = pSB1C3_SDM_SspI_Bla14FM clone 4 c=220,4
</LI><LI>P222 = pSB1C3_SDM_SspI_Bla14FM clone 5 c=252,2</LI></UL><P><B>Test digestion:</B>
No test digestion for pSB1C3_SDM_SspI_Bla14FM clone 4 and 5.
For test digestion, pSB1C3_leftITR_CMV_mVenus (P206-207) and pSB1C3_leftITR_CMV (P188) was digested as well. 
Samples P208-220 were digested with XbaI and SpeI.
Samples P207, P207 and P188 were digested with XbaI and NgoMIV.</P><P>
Gel used:
100 ml TAE (1x), 1 g agarose, 6 µl GELRED
115 V, 50 minutes
</P><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD><B>P208/µl</B></TD><TD><B>P209/µl</B></TD><TD><B>P210/µl</B></TD><TD><B>P211/µl</B></TD><TD><B>P212/µl</B></TD><TD><B>P213/µl</B></TD><TD><B>P214/µl</B></TD><TD><B>P215/µl</B></TD><TD><B>P216/µl</B></TD><TD><B>P217/µl</B></TD><TD><B>P218/µl</B></TD><TD><B>P219/µl</B></TD><TD><B>P220/µl</B></TD><TD><B>P206/µl</B></TD><TD><B>P207/µl</B></TD><TD><B>P188/µl</B></TD></TR><TR><TD> DNA: </TD><TD>4</TD><TD>6 </TD><TD>7  </TD><TD>6 </TD><TD>9 </TD><TD>7 </TD><TD>6 </TD><TD>6 </TD><TD>6</TD><TD>6 </TD><TD>6  </TD><TD>6 </TD><TD>6 </TD><TD>3,5 </TD><TD>3,5 </TD><TD>4
</TD></TR><TR><TD> BSA (10x): </TD><TD>2  </TD><TD>2 </TD><TD>2</TD><TD>2  </TD><TD>2 </TD><TD>2</TD><TD>2  </TD><TD>2 </TD><TD>2</TD><TD>2  </TD><TD>2 </TD><TD>2</TD><TD>2  </TD><TD>2 </TD><TD>2</TD><TD>2
</TD></TR><TR><TD> Buffer 4: </TD><TD>2  </TD><TD>2 </TD><TD>2</TD><TD>2  </TD><TD>2 </TD><TD>2</TD><TD>2  </TD><TD>2 </TD><TD>2</TD><TD>2  </TD><TD>2 </TD><TD>2</TD><TD>2  </TD><TD>2 </TD><TD>2</TD><TD>2
</TD></TR><TR><TD> Enzyme 1 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5</TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5</TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5</TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5</TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5</TD><TD>0,5
</TD></TR><TR><TD> Enzyme 2 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5</TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5</TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5</TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5</TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5</TD><TD>0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD> 11</TD><TD> 9</TD><TD>8</TD><TD> 9</TD><TD> 6</TD><TD>8</TD><TD> 9</TD><TD> 9</TD><TD>9</TD><TD> 9</TD><TD> 9</TD><TD>9</TD><TD> 9</TD><TD> 11,5</TD><TD>11,5</TD><TD> 11
</TD></TR></TBODY></TABLE><P>
comment: Rep- and AAP approaches have to be repeated. We did not realize that the vector can religate! Plasmids and glycerol stocks were thrown away.
</P><H4><SPAN class="mw-headline" id="Inoculation_of_CD_colony"><P style="font-size:15px; background-color:#66bbff;"><B>Inoculation of CD colony</B></P></SPAN></H4><P><B>Investigator: Kira</B></P><P>One colony was picked from agar plate and dispersed into 10 ml DYT+ Amp and incubated over night @ 37 C.
</P><H3><SPAN class="mw-headline" id="91._labday_16.08.2010"><P style="font-size:17px; background-color:#00dd77;">91. labday 16.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Minipreps_of_P119.2C_P122.2C_P204.2C_P205"><P style="font-size:15px; background-color:#66bbff;"><B>Minipreps of P119, P122, P204, P205</B></P></SPAN></H4><P><B>Investigator: Achim</B></P><H4><SPAN class="mw-headline" id="Cloning_of_pSB1C3_SspIdel_BLA_clone_2_.26_3_to_get_rid_of_mutations"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of pSB1C3_SspIdel_BLA clone 2 &amp; 3 to get rid of mutations</B></P></SPAN></H4><P><B>Investigator: Achim</B></P><UL><LI>Sequencing showed that clone 2 had a mutation in the vector, clone 3 had one in the BLA sequence. Therefore, both were cut with BamHI and SalI, then the BLA fragment of clone 2 and the vector fragment of clone 3 were ligated. The final construct shouldn't contain any more mutations.
</LI></UL><P style="font-size:15px; color:#ff00ff;">Nice picture, but where's the protocol? Results? (Hanna) ;) </P><H4><SPAN class="mw-headline" id="New_working_solutions_were_prepared"><P style="font-size:15px; background-color:#66bbff;"><B>New working solutions were prepared</B></P></SPAN></H4><P><B>Investigator: Patrick</B></P><UL><LI> DMEM
</LI><LI> Amp
</LI><LI> H2O millipore
</LI><LI> DYT
</LI></UL><H4><SPAN class="mw-headline" id="Further_pSB1C3_SDM_SspI_Bla14FM_sequencing"><P style="font-size:15px; background-color:#66bbff;"><B>Further pSB1C3_SDM_SspI_Bla14FM sequencing</B></P></SPAN></H4><P><B>Investigator: Patrick</B>
Two additional Clones were picked (clones 4 &amp; 5) and sent for sequencing, labeled: iGEM4.1 and iGEM4.2. 
Used Primer: seq_pSB1C3_VR2_rev (O51)
</P><H4><SPAN class="mw-headline" id="Mini_prep_of_pKS-CD"><P style="font-size:15px; background-color:#66bbff;"><B>Mini prep of pKS-CD</B></P></SPAN></H4><P><B>Investigator: Kira</B></P><P>c(pKS-CD) = 291, 39 ng/µl
</P><P>Yielded DNA was used for site directed mutagenesis
</P><H4><SPAN class="mw-headline" id="Fluorescence_microscopy_of_TKGMK_loaded_particles_treated_cells"><P style="font-size:15px; background-color:#66bbff;">Fluorescence microscopy of TKGMK loaded particles treated cells</P></SPAN></H4><P>The fluorescence was quite low (~ 5%)in nearly every well! The 40µg YFP wells were the only exception. </P><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Death to HT1080</CAPTION><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>HT1080 only
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>HT1080 with gancliclovir only
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>HT1080 with TK_GMK without ganciclovir
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>HT1080 with TK_GMK (300µl AAV stock) with ganciclovir
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>HT1080 with TK_GMK (600µl AAV stock) with ganciclovir
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>Remember: This was just a qualitative check of our constructs! The scale bar is missing but its not that dramatic. As you can see, our construct works! It is possible to kill tumor cells with our vector.
</P><H4><SPAN class="mw-headline" id="SDM_of_PstI_in_pKS-CD"><P style="font-size:15px; background-color:#66bbff;"><B>SDM of PstI in pKS-CD</B></P></SPAN></H4><P><B>Investigator: Jessica, Kira</B></P><P>Motivation: the desired gen contains 2 iGEM restrictions sites, thus in oder to use it for further cloning, these site have to be deleted. The first SDM was performed on PstI restriction site. 
</P><UL><LI><B>P223</B> pKS_CD</LI><LI><B>O154</B> CD_PstI FP</LI><LI><B>O155</B> CD_PstI RP</LI></UL><TABLE border="1"><TBODY><TR><TD><B>component</B></TD><TD><B>Volume/µl</B></TD></TR><TR><TD> DNA: (1:20) </TD><TD>0,5
</TD></TR><TR><TD> 10x reaction buffer: </TD><TD>2,5
</TD></TR><TR><TD> primer forward PstI (1:10) </TD><TD>0,56
</TD></TR><TR><TD> primer reverse PstI (1:10) </TD><TD>0,56
</TD></TR><TR><TD> dNTP </TD><TD>0,5
</TD></TR><TR><TD>DMSO </TD><TD>0,5
</TD></TR><TR><TD>H2O </TD><TD>14,38
</TD></TR><TR><TD>PfuTurbo DNA Polymerase </TD><TD>0,5
</TD></TR></TBODY></TABLE><P>PCR
</P><TABLE border="1"><TBODY><TR><TD><B>segment</B></TD><TD><B>cycles</B></TD><TD><B>temperature</B></TD><TD><B>time</B></TD></TR><TR><TD> 1 </TD><TD>1 </TD><TD>95 </TD><TD>2min
</TD></TR><TR><TD> 2 </TD><TD>20 </TD><TD>95 </TD><TD>30sec
</TD></TR><TR><TD>55 </TD><TD> 1min
</TD></TR><TR><TD>68 </TD><TD> 4 min30 sec
</TD></TR><TR><TD> 3 </TD><TD> 4</TD><TD> hold
</TD></TR></TBODY></TABLE><P>
sample was digested with DpnI for 1h at 37°C
</P><P>Transformation was performed according to the standard protocol.
</P><H4><SPAN class="mw-headline" id="Biobrick_production_of_pSB1C3_lITR_CMV_beta-globin_mVENUS_hgh_rITR_and_pSB1C3_lITR_CMV_mVENUS_hgh_rITR"><P style="font-size:15px; background-color:#66bbff;"><B>Biobrick production of pSB1C3_lITR_CMV_beta-globin_mVENUS_hgh_rITR and pSB1C3_lITR_CMV_mVENUS_hgh_rITR</B></P></SPAN></H4><P><B>Investigator: Anna, Anissa</B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: hgh_rITR was cloned one time into pSB1C3_lITR_CMV_beta-globin_mVENUS and one time into pSB1C3_lITR_CMV_mVENUS to receive a control for the improtance of beta-globin  </P><UL><LI><B>Digestion:</B></LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume of pBS1C3_hgh_rITR INSERT = p186 /µl</B></TD><TD align="right"><B>volume of pBS1C3_lITR_CMV_beta-globin_mVENUS VECTOR = p208 /µl</B></TD><TD align="right"><B>volume of pBS1C3_lITR_CMV_mVENUS VECTOR = p207 /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">19</TD><TD align="right">6 </TD><TD align="right">3,84
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">3</TD><TD align="right"> 3</TD><TD align="right">1,5
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">3 </TD><TD align="right"> 3 </TD><TD align="right">1,5
</TD></TR><TR><TD>SpeI° /XbaI*</TD><TD align="right">1*</TD><TD align="right">1° </TD><TD align="right">1°
</TD></TR><TR><TD>PstI</TD><TD align="right">1 </TD><TD align="right">1</TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">3 </TD><TD align="right">6 </TD><TD align="right">6,16
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right">30</TD><TD align="right">20 </TD><TD align="right">15
</TD></TR></TBODY></TABLE><P>
The samples were digested for 2 h at 37°C
</P><UL><LI> Gel: samples were loaded on a 1% agarose-gel. running time: 45 minutes
</LI></UL><UL><LI>gelextraction was performed according with the standard-protocol
</LI></UL><UL><LI>Ligation:
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>concentration/ ng/µl</B></TD></TR><TR><TD> p186  </TD><TD align="right">8,97
</TD></TR><TR><TD>p207 </TD><TD align="right">25,57
</TD></TR><TR><TD> p208</TD><TD align="right">17,73
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>amount for ligation /µl</B></TD></TR><TR><TD> p186 (for p207)</TD><TD align="right">4,2
</TD></TR><TR><TD>p207 </TD><TD align="right">3,8
</TD></TR><TR><TD>p186 (for p208)</TD><TD align="right">4,7
</TD></TR><TR><TD> p208</TD><TD align="right">3,3
</TD></TR><TR><TD> T4 ligation buffer</TD><TD align="right">1
</TD></TR><TR><TD> T4 Ligase</TD><TD align="right">1
</TD></TR></TBODY></TABLE><UL><LI>Transformation: was performed according with the standard-protocol
</LI></UL><H4><SPAN class="mw-headline" id="Preparation_for_Midi-Preps_of_B24_and_B34"><P style="font-size:15px; background-color:#66bbff;"><B>Preparation for Midi-Preps of B24 and B34</B></P></SPAN></H4><P><B>Investigator: Anna</B></P><UL><LI>40 ml DYT was prepared with 40 µl Ampicillin and inoculated with B24 and B34, both were incubated over night in 37°C room.</LI></UL><P style="font-size:13px; color:#ff00ff;">What is B24 and B34? Can you please be more precise - I also want to understand what's happening also when I am not able to check up in the excel sheets ;) (Hanna)</P><H4><SPAN class="mw-headline" id="Cloning_of_pAAV_RC_.28P50.29_with_pMA_RC_insert_.28P190.29"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of pAAV_RC (P50) with pMA_RC_insert (P190)</B></P></SPAN></H4><P><B>Investigator: Chris L.</B></P><UL><LI>Vector: name: pAAV_RC <B>P50</B></LI><LI>Insert: name: pMA_RC_insert <B>P190</B></LI></UL><UL><LI>buffer used: 3
</LI><LI>Restriction-enzymes used:
</LI><LI>BstEII (no. Lab: 17)
</LI><LI>SwaI (no.Lab: 135)
</LI></UL><LI>DNA concentration (P50): 378,5 ng/µl
</LI><LI>DNA concentration (P190): 339,5 ng/µl
</LI><P><B>Digestion:</B></P><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume of pMA_RC_insert /µl</B></TD><TD align="right"><B>volume of pAAV_RC /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">4,4 </TD><TD align="right">2,6
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 3 (10x)</TD><TD align="right">2 </TD><TD align="right">2
</TD></TR><TR><TD>SwaI (no.Lab:135)</TD><TD align="right">0,5 </TD><TD align="right">0,5
</TD></TR><TR><TD>BstEII (no.Lab:17)</TD><TD align="right">0,5 </TD><TD align="right">0,5
</TD></TR><TR><TD>H2O</TD><TD align="right">10,6 </TD><TD align="right">12,4
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 20</TD><TD align="right">20
</TD></TR></TBODY></TABLE><P style="color:red;">Comments: Incubation at 25° after addition of SwaI for 1,5 hours. Then incubation at 60° after addition of BstEII for 1,5 hours.</P><P><B>Gel extraction:</B>
Preparation of gel:
0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 115 Volt, running time: 45 minutes
</P><P>Expected sizes of constructs:
</P><UL><LI>pAAV_RC       6839 bp and 493 bp  
</LI><LI>pMA_RC_Insert 3292 bp and 493 bp
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>Sample P190, 20µl
</TH><TH>Sample P 50, 20µl
</TH></TR><TR><TH>Lane
</TH><TD>3
</TD><TD>5
</TD><TD>7
</TD></TR></TBODY></TABLE><B>Results:</B><P style="color:#00bbff;"> Bands are really weak, tomorrow I`ll take more DNA in sample. 
<B>Next steps: Tomorrow repetition of experiment </B></P><H4><SPAN class="mw-headline" id="Repetition:_Biobrick_production_of_Rep40.2C_Rep68.2C_Rep78_and_AAP"><P style="font-size:15px; background-color:#66bbff;"><B>Repetition: Biobrick production of Rep40, Rep68, Rep78 and AAP</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P> 	
Aim of the experiment:
We want to produce biobricks from Rep40, Rep68, Rep78 and AAP.
</P><UL><LI>Plasmids used as template:
</LI></UL><P>Rep_68_ex (p119):   c = 470,6 ng/µl 
Rep_78_(p122):      c = 566,8 ng/µl 
pAAV-RC containing AAP ORF (p50):    c = 378,5 ng/µl </P><UL><LI>Primer used:
</LI></UL><P>For Rep_68_ex: Praefix_68_78_ex &amp; Suffix_40_68_ex
For Rep_78_ex: Praefix_68_78_ex &amp; Suffix_52_78_ex
For AAP_ex: Praefix_AAP_ex &amp; Suffix_AAP_ex
</P><UL><LI><B>PCR:</B></LI></UL><P>(was performed following the standard protocol)
</P><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD align="right"><B>Volume / µl</B></TD><TD align="right"><B>Rep68</B></TD><TD align="right"><B>Rep78</B></TD><TD align="right"><B>AAP</B></TD><TD align="right"><B>Rep40</B></TD></TR><TR><TD> 5X Phusion HF buffer </TD><TD align="right">10 </TD></TR><TR><TD> 10 mM dNTP mix</TD><TD align="right">1</TD></TR><TR><TD> forward primer: </TD><TD align="right">2,5</TD></TR><TR><TD> reverse primer: </TD><TD align="right">2,5</TD></TR><TR><TD> DNA Template</TD><TD align="right">***</TD><TD align="right">2,5 µl </TD><TD align="right">2 µl</TD><TD align="right">3 µl</TD><TD align="right">1 µl
</TD></TR><TR><TD> DMSO (2%)</TD><TD align="right"> - </TD><TD align="right"> - </TD><TD align="right"> 1 µl </TD><TD align="right">-
</TD></TR><TR><TD> Phusion Polymerase</TD><TD align="right">0,5</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">*** </TD><TD align="right"> 31 µl</TD><TD align="right"> 31,5 µl</TD><TD align="right">29,5 µl </TD><TD align="right">31,5 µl
</TD></TR><TR><TD>Total volume</TD><TD align="right">50
</TD></TR></TBODY></TABLE><P>PCR program:
</P><TABLE border="1"><TBODY><TR><TD><B>PCR Program</B></TD><TD align="right"><B>temperature/ °C</B></TD><TD align="right"><B>Time</B></TD><TD align="right"><B>Rep68</B></TD><TD align="right"><B>Rep78</B></TD><TD align="right"><B>AAP</B></TD><TD align="right"><B>Rep40</B></TD></TR><TR><TD>1</TD><TD align="right">98 </TD><TD align="right">1min</TD></TR><TR><TD>2</TD><TD align="right">98 </TD><TD align="right">15s</TD></TR><TR><TD>3</TD><TD align="right">*** </TD><TD align="right">25s</TD><TD align="right">63°C</TD><TD align="right">62°C</TD><TD align="right">64°C</TD><TD align="right">63°C
</TD></TR><TR><TD>4 (step 2-4 8x) </TD><TD align="right">72</TD><TD align="right">***</TD><TD align="right">24s </TD><TD align="right">27s </TD><TD align="right"> 10s </TD><TD align="right">15s
</TD></TR><TR><TD>5 </TD><TD align="right">98</TD><TD align="right"> 15s</TD></TR><TR><TD>6</TD><TD align="right">***</TD><TD align="right">25s</TD><TD align="right">68°C</TD><TD align="right">64°C</TD><TD align="right">68°C </TD><TD align="right">68°C
</TD></TR><TR><TD>7 (step 5-7 17x)</TD><TD align="right">72</TD><TD align="right">***</TD><TD align="right">24s</TD><TD align="right">27s</TD><TD align="right">10s </TD><TD align="right">15s
</TD></TR><TR><TD>8</TD><TD align="right">72</TD><TD align="right">5min</TD></TR><TR><TD>Hold</TD><TD align="right">4°C</TD></TR></TBODY></TABLE><P>Used agarose gel:
0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 115 Volt, running time:45 minutes
</P><P style="font-size:13px; color:#68bbff;"><I><B>Comments</B></I>: I forgot to dilute Rep68. Therefore, a new PCR will have to be performed tonight. </P><P style="font-size:15px; font-weight: bold; color: blue;">Gel extraction</P><P>
Gel extraction was perfomed according to protocol.
</P><UL><LI>Digestion of PCR products and vector:
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>PCR products /µl</B></TD><TD align="right"><B>vector /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">29 </TD><TD align="right">11
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">4 </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">4 </TD><TD align="right">2
</TD></TR><TR><TD>Enzyme <B>XbaI </B></TD><TD align="right">1 </TD><TD align="right">1</TD></TR><TR><TD>Enzyme <B>SpeI </B></TD><TD align="right">1 </TD><TD align="right">1</TD></TR><TR><TD>H2O</TD><TD align="right">1 </TD><TD align="right">3
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 40</TD><TD align="right">20
</TD></TR></TBODY></TABLE><P style="font-size:13px; color:#68bbff;"><I><B>Comments</B></I>: Digestion was done with XbaI and SpeI, it has to be checked if the inserts are cloned into the vector in the right orientation. </P><UL><LI>Purification of Rep40, 78 and AAP:
</LI></UL><P>For the purification 200 µl of buffer PBI was used.
</P><P>c(Rep40)= 6,5 ng/µl 
c(Rep78)= 14,3 ng/µl 
c(AAP)= 39,6 ng/µl
</P><UL><LI>Gelextraction of pSB1C3_RFC25_CFP:
</LI></UL><P>0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 115 Volt, running time:50 minutes 
4µl loading dye (6x) for the sample, Marker: GeneRuler ladder mix (Fermentas)
</P><P>
c(pSB1C3)= 19,5 ng/µl </P><UL><LI>T4 ligation of PCR products and vector:
</LI></UL><P>For the Ligation 1µl buffer (10x) and 1µl T4 ligase were used.
</P><TABLE border="1"><TBODY><TR><TD align="right"><B>vector /µl</B></TD><TD align="right"><B>insert /µl</B></TD></TR><TR><TD> pSB1C3 + Rep40 </TD><TD align="right">1,57 </TD><TD align="right">6,43
</TD></TR><TR><TD> pSB1C32 + Rep78 </TD><TD align="right">1,71 </TD><TD align="right">6,29
</TD></TR><TR><TD> pSB1C3 + AAP </TD><TD align="right">5,56 </TD><TD align="right">2,44
</TD></TR></TBODY></TABLE><UL><LI>Transformation:
</LI></UL><P>The transformation was done following the standard protocol using XL1b cells.
</P><H4><SPAN class="mw-headline" id="Design_and_Ordering_of_Oligos"><P style="font-size:15px; background-color:#66bbff;"><B>Design and Ordering of Oligos</B></P></SPAN></H4><P><B>Investigator: Hanna</B></P><P style="font-size:15px;color:#ff00ff;"><B>SV40 terminator</B></P><P><B>Comment:</B> The CMV promoter is often used in combination with a SV40 terminator. Because we want to use this promoter in the context of the VP2 fusion and VP1 insertion approaches, oligos were designed in order to produce a SV40 terminator BioBrick (from the pEGFP-C1 plasmid).
</P><P style="font-size:15px;color:#ff00ff;"><B>pEGFP_backbone</B></P><P><B>Comment:</B> In order to perform the VP2 N-terminal fusion and VP1 insertion approaches, we decided to use pEGFP-C1 as expression plasmid. This plasmid contains an EGFP under the control of the CMV promoter and the SV40 terminator. In order to exchange the EGFP with our targeting constructs a PCR of the CMV_backbone_SV40Terminator should be performed. The referring primers were designed with EcoRI and PstI restriction sites. One should keep in mind that this expression plasmid mustn't digested with NgoMIV and it contains Kanamycin-resistance!
</P><H3><SPAN class="mw-headline" id="92._labday_17.08.2010"><P style="font-size:17px; background-color:#00dd77;">92. labday 17.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Overview_of_the_state-of_the_art_plasmid_preparation"><P style="font-size:15px; background-color:#66bbff;"><B>Overview of the state-of the art plasmid preparation</B></P></SPAN></H4><P><B>Investigator: Bea </B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: In order to obtain a overview (more or less plausible ;-)) I sorted out how the final plasmid need to be constructed. In the picture you can see the most important plasmids which we will need for the retargeting approaches starting tomorrow.</P><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Overview scheme</CAPTION></TABLE><P><A href="http://www.molbiotech.uni-freiburg.de/iGEM/wiki2010/index.php/Laborjournal" class="external text" rel="nofollow">top of page</A></P><H4><SPAN class="mw-headline" id="Further_pSB1C3_SDM_SspI_Bla14FM_sequencing_2"><P style="font-size:15px; background-color:#66bbff;"><B>Further pSB1C3_SDM_SspI_Bla14FM sequencing </B></P></SPAN></H4><P><B>Investigators: Bea, Patrick</B></P><P>Sequencing results: there is no BLA in clone 4. Clone 5 showed a flawless BLA so it will be used for the next site directed mutagenesis.
</P><H4><SPAN class="mw-headline" id="pSB1C3_SDM_SspI_Bla14FM_clone_5_SDM_.28P222.29"><P style="font-size:15px; background-color:#66bbff;"><B>pSB1C3_SDM_SspI_Bla14FM clone 5 SDM (P222)</B></P></SPAN></H4><P>Investigator: Patrick 
Intention: remove PvuII restriction site </P><P>PCR program:
</P><UL><LI>95°C 2 min (1x)
</LI><LI>95°C 20s, 60°C 10 s, 68°C 30 s (30s/kb) (18x)
</LI><LI>68°C 5 minutes
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD><B>Volume</B></TD></TR><TR><TD> 10x reaction buffer </TD><TD>2,5
</TD></TR><TR><TD> DNA template ( about 10 ng)</TD><TD> 1 µl diluted P222
</TD></TR><TR><TD> forward primer: </TD><TD> 0,58 µl O108 (pSB1C3 PvuII rev)
</TD></TR><TR><TD> reverse primer: </TD><TD> 0,59 µl O109 (pSB1C3 PvuII for)
</TD></TR><TR><TD> DMSO (2%)</TD><TD> 0,5
</TD></TR><TR><TD> dNTP Mix from the kit</TD><TD> 0,5 µl
</TD></TR><TR><TD> QuickSolution Reagent</TD><TD>0,75 µl
</TD></TR><TR><TD> Quickchange Lightning Enzyme (1.25U)</TD><TD>0,5 µl
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD> 18,08 µl
</TD></TR><TR><TD>Total volume</TD><TD>25 µl
</TD></TR></TBODY></TABLE><P>Further procedure including the transformation was performed according to the standard protocol.
</P><H4><SPAN class="mw-headline" id="Sent_for_sequencing_2"><P style="font-size:15px; background-color:#66bbff;">Sent for sequencing</P></SPAN></H4><UL><LI> pMA_RC (P222) sent for sequencing with O41 (Cap 4000 for) to check if the EaglII restiction site was successfully changed to PvuII.
</LI><LI> pAAV_RC (P158) sent for sequencing with GATC_std_SK, Rep_1250 for (O35), GATC_std_UP-2 to check if all deleted restriction sites are present (PstI, BamHI, SalI).
</LI></UL><P>Investigators: Bea, Patrick
</P><H4><SPAN class="mw-headline" id="Cloning_of_pMA_RC-Insertparts_in_pAAV_RC_1.2_SDM_SalI"><P style="font-size:15px; background-color:#66bbff;">Cloning of pMA_RC-Insertparts in pAAV_RC 1.2 SDM SalI</P></SPAN></H4><P><B>Investigator: Jessica</B><B>P211</B> will be cloned in the <B>P158</B> vector that contains 4 mutation (BamHI, SalI, PstI 320 + 4073)<B>Digestion:</B></P><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>pMA_RC_Insertparts</B></TD><TD align="right"><B>pAAV_RC 1.2 SDM SalI</B></TD></TR><TR><TD> DNA  </TD><TD align="right">5 </TD><TD align="right">4
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">1 </TD><TD align="right"> 1
</TD></TR><TR><TD> Buffer 3 (10x)</TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>Enzyme1 BstEII </TD><TD align="right">0,5  </TD><TD align="right">0,5
</TD></TR><TR><TD>Enzyme2 SwaI</TD><TD align="right">0,5 </TD><TD align="right">0,5
</TD></TR><TR><TD>H2O</TD><TD align="right">2 </TD><TD align="right">3
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right">10</TD><TD align="right">10
</TD></TR></TBODY></TABLE>
digestion was splitted in two parts: <UL><LI>1. digestion with SwaI at 25°C for 1,5h</LI><LI>2. digestion with BstEII at 60°C for 1,5h</LI></UL><P>Preparation of gel:
0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at  Volt, running time:  minutes
</P><P>Expected sizes of constructs:
</P><UL><LI>pMA_RC-Insertparts:  <B>493bp</B>, 3287bp
</LI><LI>pAAV_RC 1.2 SDM SalI: <B>6839bp</B>, 478bp
</LI></UL><P>The corresponding bands were cut out and Gel-Extraction was performed according to protocol.
</P><P><B>concentrations</B> measured via NanoDrop:
</P><UL><LI>pMA_RC-Insertparts:  5,9 ng/µl
</LI><LI>pAAV_RC 1.2 SDM SalI:8,6 ng/µl
</LI></UL><P><B>Ligation</B>
Volume insert: 1,95 µl
Volume vector: 6,08 µl<B>Trafo</B> was prepared with BL21
</P><H4><SPAN class="mw-headline" id="Repetition:_Biobrick_production_of_Rep_68"><P style="font-size:15px; background-color:#66bbff;"><B>Repetition: Biobrick production of Rep 68</B></P></SPAN></H4><P><B>Investigator: Christian L. , Stefan</B></P><P> 	
Aim of the experiment:
We want to produce biobricks from Rep 68.
</P><UL><LI>Plasmid used as template:
</LI></UL><P>Rep_68_ex (p224):   c = 350,29 ng/µl </P><UL><LI>Primer used:
</LI></UL><P>For Rep_68_ex: Praefix_68_78_ex &amp; Suffix_40_68_ex</P><UL><LI><B>PCR:</B></LI></UL><P>(was performed following the standard protocol)
</P><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD><B>Volume Rep68 / µl</B></TD></TR><TR><TD> 5x Phusion HF buffer </TD><TD>10
</TD></TR><TR><TD> 10 mM dNTP mix</TD><TD>1
</TD></TR><TR><TD> forward primer: </TD><TD>2,5
</TD></TR><TR><TD> reverse primer: </TD><TD>2,5
</TD></TR><TR><TD> DNA Template</TD><TD>3
</TD></TR><TR><TD> DMSO (2%)</TD><TD> -
</TD></TR><TR><TD> Phusion Polymerase</TD><TD>0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD> 30,5
</TD></TR><TR><TD>Total volume</TD><TD>50
</TD></TR></TBODY></TABLE><P>PCR program:
</P><TABLE border="1"><TBODY><TR><TD><B>PCR Program</B></TD><TD align="right"><B>temperature/ °C</B></TD><TD align="right"><B>Time</B></TD></TR><TR><TD>1</TD><TD align="right">98 </TD><TD align="right">1min
</TD></TR><TR><TD>2</TD><TD align="right">98 </TD><TD align="right">15s
</TD></TR><TR><TD>3</TD><TD align="right">63 </TD><TD align="right">25s
</TD></TR><TR><TD>4 (step 2-4 8x) </TD><TD align="right">72</TD><TD align="right">24s
</TD></TR><TR><TD>5 </TD><TD align="right">98</TD><TD align="right"> 15s
</TD></TR><TR><TD>6</TD><TD align="right">68</TD><TD align="right">25s
</TD></TR><TR><TD>7 (step 5-7 17x)</TD><TD align="right">72</TD><TD align="right">24s
</TD></TR><TR><TD>8</TD><TD align="right">72</TD><TD align="right">5min
</TD></TR><TR><TD>Hold</TD><TD align="right">4°C
</TD></TR></TBODY></TABLE><P>Used agarose gel:
0,5 g Agarose,50 ml TBE (1%), 3 µl GELRED , at 115 Volt, running time:45 minutes
</P><P>
Gel extraction was perfomed according to protocol.
comment: Elution was put in 4°C room. Work will be continued tomorrow.
</P><H4><SPAN class="mw-headline" id="Transduction_of_HT1080_cells"><P style="font-size:15px; background-color:#66bbff;"><B>Transduction of HT1080 cells</B></P></SPAN></H4><P><B>Transduction plan</B></P><P><B> Investigator: Adrian, Kerstin</B></P><LI>1. plate 50.000 cells, YFP
<TABLE border="1"><TBODY><TR><TH>A
</TH><TD>control no cells
</TD><TD>300 µl virus (40 µg YFP)
</TD><TD>300 µl virus (20 µg YFP)
</TD></TR><TR><TH>B
</TH><TD>control no virus
</TD><TD>500µl P38 (pAAV2_mVenus)
</TD><TD>1000µl P38 (pAAV2_mVenus)
</TD></TR></TBODY></TABLE></LI><LI>2. plate 50.000 cells, YFP
<TABLE border="1"><TBODY><TR><TH>A
</TH><TD>control no cells
</TD><TD>300 µl virus (40 µg YFP)
</TD><TD>300 µl virus (20 µg YFP)
</TD></TR><TR><TH>B
</TH><TD>control no virus
</TD><TD>500µl P38 (pAAV2_mVenus)
</TD><TD>1000µl P38 (pAAV2_mVenus)
</TD></TR></TBODY></TABLE></LI><LI>3. plate 100.000 cells, YFP  
<TABLE border="1"><TBODY><TR><TH>A
</TH><TD>control no cells
</TD><TD>300 µl virus (40 µg YFP)
</TD><TD>300 µl virus (20 µg YFP)
</TD></TR><TR><TH>B
</TH><TD>control no virus
</TD><TD>500µl P38 (pAAV2_mVenus)
</TD><TD>1000µl P38 (pAAV2_mVenus)
</TD></TR></TBODY></TABLE></LI><LI>4. plate 100.000 cells, YFP
<TABLE border="1"><TBODY><TR><TH>A
</TH><TD>control no cells
</TD><TD>300 µl virus (40 µg YFP)
</TD><TD>300 µl virus (20 µg YFP)
</TD></TR><TR><TH>B
</TH><TD>control no virus
</TD><TD>500µl P38 (pAAV2_mVenus)
</TD><TD>1000µl P38 (pAAV2_mVenus)
</TD></TR></TBODY></TABLE></LI><LI>5. plate 100.000 cells TKGMK
<TABLE border="1"><TBODY><TR><TH>A
</TH><TD>control no cells
</TD><TD>150 µl virus (TKGMK clone 1)
</TD><TD>180 µl virus (TKGMK clone 1)
</TD></TR><TR><TH>B
</TH><TD>control no virus
</TD><TD>200 µl (TKGMK clone2)
</TD><TD>200µl  (no GOI)
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Midi-Prep_of_pAAV_igEM_mVenus_YFP_and_dsAAV_CMV_EGFP"><P style="font-size:15px; background-color:#66bbff;"><B>Midi-Prep of pAAV_igEM_mVenus_YFP and dsAAV_CMV_EGFP</B></P></SPAN></H4><P><B>Investigators: Chris, Anna </B></P><P style="font-size:13px; color:#003399;"><B>Comment</B>: Midi-Preps of dsAAV_CMV_EGFP (p31, B24) and pAAV_iGEM_mVenus_YFP (p39, B34)</P><P>The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
</P><TABLE border="1"><TBODY><TR><TD> plasmid-no. </TD><TD align="right">p31</TD><TD align="right">p39
</TD></TR><TR><TD> concentration (ng/µl)</TD><TD align="right">539,05 </TD><TD align="right">162,37
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="BioBrick_assembly_for_pSB1C3_lITR_CMV_beta-globin_YFP_rITR"><P style="font-size:15px; background-color:#66bbff;"><B>BioBrick assembly for pSB1C3_lITR_CMV_beta-globin_YFP_rITR</B></P></SPAN></H4><P><B>Investigator: Achim</B></P></LI><LI>This construct without the hGH polyadenylation sequence will be used to determine wether/how much hGH affects translation of the virus gene.
</LI><P>Vector: </P><UL><LI>pSB1C3_lITR_CMV_ß_YFP (P209): c=274,86 ng/µl</LI></UL><P>Insert: </P><UL><LI>pSB1C3_rITR (P172/P191): c=73,3 ng/µl</LI></UL><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">Vector</TD><TD align="right">Insert
</TD></TR><TR><TD> DNA  </TD><TD align="right">3,6 </TD><TD align="right"> 20,5
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">3</TD><TD align="right"> 3
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">3 </TD><TD align="right"> 3
</TD></TR><TR><TD>Enzyme 1 </TD><TD align="right">1(SpeI) </TD><TD align="right"> 1(XbaI)
</TD></TR><TR><TD>Enzyme 2 </TD><TD align="right">1(PstI)</TD><TD align="right"> 1 (PstI)
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">18,4 </TD><TD align="right"> 1,5
</TD></TR><TR><TD><B>Total volume </B></TD><TD align="right">30 </TD><TD align="right"> 30
</TD></TR></TBODY></TABLE><UL><LI> 1% Agarose gel
</LI><LI> 3 µl Gelred
</LI><LI>7 µl DNA-Ladder-Mix
</LI><LI> 115 Volt, running time: 30 minutes
</LI></UL><P>The Insert containing the right ITR was cut out after 30 minutes, the Vector was cut out after 45 minutes</P><TABLE border="1"><TBODY><TR><TD><B>Sample/µl </B></TD><TD><B> Expected size/bp</B></TD></TR><TR><TD> Vector </TD><TD align="right">4100
</TD></TR><TR><TD> Insert </TD><TD align="right">150
</TD></TR></TBODY></TABLE><UL><LI> weight of insert gel extract: 170 mg
</LI><LI> weight of vector gel extract: 200 mg
</LI></UL><P><B>Nanodrop</B></P><UL><LI> insert : 11,4 ng/µl
</LI><LI> vector : 26,7 ng/µl
</LI></UL><P><B>Ligation</B></P><UL><LI>vector: 6,36 µl
</LI><LI>insert: 1,64 µl</LI></UL><P><B>Transformation</B></P><H4><SPAN class="mw-headline" id="BioBrick_assembly_for_pSB1C3_lITR_pTERT"><P style="font-size:15px; background-color:#66bbff;"><B>BioBrick assembly for pSB1C3_lITR_pTERT</B></P></SPAN></H4><P><B>Investigator: Anissa</B></P><UL><LI>Construct with the pTERT tumor specific promoter
</LI></UL><P>Vector: </P><UL><LI>pSB1C3_lITR (P196): 174 ng/µl</LI></UL><P>Insert: </P><UL><LI>pSB1C3_phTERT (P177): c=312 ng/µl</LI></UL><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">Vector</TD><TD align="right">Insert
</TD></TR><TR><TD> DNA  </TD><TD align="right">5,75 </TD><TD align="right"> 6,41
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">1,5</TD><TD align="right"> 1,5
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">1,5 </TD><TD align="right"> 1,5
</TD></TR><TR><TD>Enzyme 1 </TD><TD align="right">1(SpeI) </TD><TD align="right"> 1(XbaI)
</TD></TR><TR><TD>Enzyme 2 </TD><TD align="right">1(PstI)</TD><TD align="right"> 1 (PstI)
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">4,25 </TD><TD align="right"> 3,59
</TD></TR><TR><TD><B>Total volume </B></TD><TD align="right">15 </TD><TD align="right"> 15
</TD></TR></TBODY></TABLE><UL><LI> 1% Agarose gel
</LI><LI> 3 µl Gelred
</LI><LI>7 µl DNA-Ladder-Mix
</LI><LI> 115 Volt, running time: 45 minutes
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>Sample/µl </B></TD><TD><B> Expected size/bp</B></TD></TR><TR><TD> Vector </TD><TD align="right">2200
</TD></TR><TR><TD> Insert </TD><TD align="right">500
</TD></TR></TBODY></TABLE><UL><LI> weight of insert gel extract: 350 mg
</LI><LI> weight of vector gel extract: 250 mg
</LI></UL><P><B>Nanodrop</B></P><UL><LI> insert : 17,7 ng/µl
</LI><LI> vector : 27,6 ng/µl
</LI></UL><P><B>Ligation</B></P><UL><LI>vector: 5,57 µl
</LI><LI>insert: 2,43 µl</LI></UL><P><B>Transformation</B></P><H4><SPAN class="mw-headline" id="Picking_clones_of_pSB1C3_lITR_CMV_.28betaglobin.29_mVenus_hGH_rITR.2C_EGFP_C1_and_pMA_T_affibody"><P style="font-size:15px; background-color:#66bbFF;"><B>Picking clones of pSB1C3_lITR_CMV_(betaglobin)_mVenus_hGH_rITR, EGFP_C1 and pMA_T_affibody</B></P></SPAN></H4><P><B>Investigator: Anna</B></P><TABLE border="1"><TBODY><TR><TD> Plasmid   </TD><TD align="right">Resistance </TD><TD align="right">Bacterial strain
</TD></TR><TR><TD> pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR°  </TD><TD align="right">Chloramphenicol</TD><TD align="right">XL1 blue
</TD></TR><TR><TD> pSB1C3_lITR_CMV_mVEnus_hGH_rITR°  </TD><TD align="right">Chloramphenicol</TD><TD align="right">XL1 blue
</TD></TR><TR><TD> EGFP_C1</TD><TD align="right">Kanamycin </TD></TR><TR><TD>pMA_T_affibody </TD><TD align="right">Ampicillin</TD></TR></TBODY></TABLE><P>°: 3 clones each</P><P>Clones were inoculated in 10 mL DYT medium containing 10 µL of antibiotics and were put in 37°C room on shaker.</P><H4><SPAN class="mw-headline" id="Trafo_evaluation_of_SDM_PstI_in_CD_and_inoculation"><P style="font-size:15px; background-color:#66bbff;"><B>Trafo evaluation of SDM PstI in CD and inoculation</B></P></SPAN></H4><P><B>Investigator: Kira</B></P><P>The plate contained lots of colonies and 2 of them were picked and inoculated into DYT+Amp
</P></DIV><CENTER><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/August2" class="external text" rel="nofollow"><B>=&gt; Go to Labjournal August part 2 (labday 93 - 106)</B></A></CENTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/August">http://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/August</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg_Bioware/NoteBook/Labjournal/August" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg_Bioware/NoteBook/Labjournal/August" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/August&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/August&amp;oldid=197545" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>