<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Bioware_NoteBook_Labjournal_June"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg_Bioware/NoteBook/Labjournal/June">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg_Bioware/NoteBook/Labjournal/June&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/June&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/June&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg_Bioware/NoteBook/Labjournal/June" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg Bioware/NoteBook/Labjournal/June</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="menu"><UL id="menu"><LI><DL><DT class="menu_button_left"><A href="https://2010.igem.org/Team:Freiburg_Bioware">Home</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Team_2010">Team</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Team_2010">Team 2010</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Photo_Gallery">Photo Gallery</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Cuckoo_Clock">Cuckoo Clock Competition</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Collaboration">Collaboration</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Example_Page">Example Page</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project">Project</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project">Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#highlights">Highlights</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#highlights">Results</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#modularisation">   Modularisation</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#targeting">   Targeting</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#arming">   Arming</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#superconstruct">   All-in-One</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Methods">Methods</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Virus_Construction_Kit">Virus Construction Kit Manual</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks">BioBricks</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks">Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/capsid_coding">Capsid Coding Parts</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/vector_plasmid">Vector Plasmid Parts</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/miscellaneous">Miscellaneous Parts</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A class="current" href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook">Notebook</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal">Labjournal</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Materials">Materials</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Methods">Methods</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Experiments">Experiments</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Meetings">Meetings</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Statistics">Statistics</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Sponsors">Sponsors</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Safety">Safety</A></DT></DL></LI><LI><DL><DT class="menu_button_right"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling">Modeling</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling">Modeling Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Virus_Production">Virus Production</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Virus_Infection">Virus Infection</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Structure_Modeling">Structure Modeling</A></DD></DL></LI></UL></DIV><DIV class="pageContent"><P><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook" class="external text" rel="nofollow">=&gt; Back to Notebook overview</A></P><DIV class="box_right"><LEFT><U1>NoteBook Navigator</U1></LEFT><UL><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/March">March (labday 1)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/April">April (labday 2 - 5)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/May">May (labday 6 - 17)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/June">June (labday 18 - 45)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/July">July (labday 46 - 75)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/August">August part 1 (labday 76 - 92)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/August2">August part 2 (labday 93 - 106)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/September">September part 1 (labday 107 - 123)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/September2">September part 2 (labday 124 - 135)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/October">October part 1 (labday 136 - 149 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/October2">October part 2 (labday 150 - 166 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/November">November  (labday 167 - 170 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/Cellculture">Cellculture</A></LI></UL></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#18._Labortag_01.06.2010:"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">18. Labortag 01.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Modifying_MCS_of_pAAV_MCS_vector"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Modifying MCS of pAAV_MCS vector</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-3"><A href="#19._Labortag_02.06.2010:"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">19. Labortag 02.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Picking_clones_of_pAAV_RFC25MCS"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Picking clones of pAAV_RFC25MCS</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Design_and_ordering_of_Oligos_.28NotI.29"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Design and ordering of Oligos (NotI)</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Sponsoring_work:"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Sponsoring work:</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#20._Labortag_03.06.2010:"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">20. Labortag 03.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#pAAV_RFC25_MCS_-.3E_problem"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">pAAV_RFC25_MCS -&gt; problem</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Picking_clones_of_Thymindinkinase_of_Amor"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Picking clones of Thymindinkinase of Amor</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#21._Labortag_04.06.2010:"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">21. Labortag 04.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#DKFZ_plasmid_Retrafos.2C_TK.2FGMK_Mini-Prep"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">DKFZ plasmid Retrafos, TK/GMK Mini-Prep</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#22._Labortag_05.06.2010:"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">22. Labortag 05.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Insertion_of_iGEM_expression_parts_into_pAAV_MCS"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Insertion of iGEM expression parts into pAAV_MCS</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#23._Labortag_07.06.2010:"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">23. Labortag 07.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Mini-Preps_.28Kleinschmidt-plasmids_and_pAAV_iGEM-MCS.29"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Mini-Preps (Kleinschmidt-plasmids and pAAV_iGEM-MCS)</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Kleinschmidt-plasmids"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Kleinschmidt-plasmids</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Site-directed_mutagenesis_of_pAAV_iGEM-MCS_.28PstI.29"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">Site-directed mutagenesis of pAAV_iGEM-MCS (PstI)</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#24._Labortag_08.06.2010:"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">24. Labortag 08.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Cloning_of_mVenus_YFP_into_pAAV_iGEM-MCS.2C_continuation_of_site-directed_mutagenesis"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Cloning of mVenus_YFP into pAAV_iGEM-MCS, continuation of site-directed mutagenesis</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Continuation_of_site-directed_mutagenesis"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">Continuation of site-directed mutagenesis</SPAN></A></LI><LI class="toclevel-1 tocsection-21"><A href="#25._Labortag_09.06.2010:"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">25. Labortag 09.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#pAAV_iGEM-MCS_mVenus-YFP.2C_Site-directed_mutagenesis"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">pAAV_iGEM-MCS_mVenus-YFP, Site-directed mutagenesis</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Picking_of_Clones_from_pAAV_iGEM-MCS_mVenus-YFP_Vektor_oben"><SPAN class="tocnumber">8.2</SPAN><SPAN class="toctext">Picking of Clones from pAAV_iGEM-MCS_mVenus-YFP_Vektor_oben</SPAN></A></LI><LI class="toclevel-1 tocsection-24"><A href="#26._Labortag_10.06.2010:"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">26. Labortag 10.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Site-directed_mutagenesis_of_PstI_in_ITRs"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">Site-directed mutagenesis of PstI in ITRs</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#Mini-Prep_of_pAAV_iGEM_Venus_YFP_and_test_digestion"><SPAN class="tocnumber">9.2</SPAN><SPAN class="toctext">Mini-Prep of pAAV_iGEM_Venus_YFP and test digestion</SPAN></A></LI><LI class="toclevel-1 tocsection-27"><A href="#27._Labortag_11.06.2010:"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">27. Labortag 11.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#Site_directed_mutagenisis_of_10.06.2010"><SPAN class="tocnumber">10.1</SPAN><SPAN class="toctext">Site directed mutagenisis of 10.06.2010</SPAN></A></LI><LI class="toclevel-1 tocsection-29"><A href="#28._Labortag_12.06.2010:"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">28. Labortag 12.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#Transformation_plates_stored_in_4.C2.B0C_room"><SPAN class="tocnumber">11.1</SPAN><SPAN class="toctext">Transformation plates stored in 4°C room</SPAN></A></LI><LI class="toclevel-1 tocsection-31"><A href="#29._Labortag_13.06.2010:"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">29. Labortag 13.06.2010:</SPAN></A></LI><LI class="toclevel-1 tocsection-32"><A href="#30._Labortag_14.06.2010:"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">30. Labortag 14.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#Site-directed_mutagenesis:"><SPAN class="tocnumber">13.1</SPAN><SPAN class="toctext">Site-directed mutagenesis:</SPAN></A></LI><LI class="toclevel-2 tocsection-34"><A href="#TK-GMK-Plasmid"><SPAN class="tocnumber">13.2</SPAN><SPAN class="toctext">TK-GMK-Plasmid</SPAN></A></LI><LI class="toclevel-2 tocsection-35"><A href="#Cell_culture"><SPAN class="tocnumber">13.3</SPAN><SPAN class="toctext">Cell culture</SPAN></A></LI><LI class="toclevel-1 tocsection-36"><A href="#31._Labortag_15.06.2010:"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">31. Labortag 15.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-37"><A href="#Solutions_for_transfection_were_prepared"><SPAN class="tocnumber">14.1</SPAN><SPAN class="toctext">Solutions for transfection were prepared</SPAN></A></LI><LI class="toclevel-1 tocsection-38"><A href="#32._Labortag_16.06.2010:"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">32. Labortag 16.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-39"><A href="#Sponsoring:"><SPAN class="tocnumber">15.1</SPAN><SPAN class="toctext">Sponsoring:</SPAN></A></LI><LI class="toclevel-2 tocsection-40"><A href="#Sequencing_of_TK-GMK:"><SPAN class="tocnumber">15.2</SPAN><SPAN class="toctext">Sequencing of TK-GMK:</SPAN></A></LI><LI class="toclevel-2 tocsection-41"><A href="#Mega_primer_PCR:"><SPAN class="tocnumber">15.3</SPAN><SPAN class="toctext">Mega primer PCR:</SPAN></A></LI><LI class="toclevel-1 tocsection-42"><A href="#33._Labortag_17.06.2010:"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">33. Labortag 17.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-43"><A href="#Primer_designing:_VP1_primer_for_pKEX_forward.2Freverse"><SPAN class="tocnumber">16.1</SPAN><SPAN class="toctext">Primer designing: VP1 primer for pKEX forward/reverse</SPAN></A></LI><LI class="toclevel-2 tocsection-44"><A href="#Primer_designing:_CMV_forward.2Freverse_qPCR"><SPAN class="tocnumber">16.2</SPAN><SPAN class="toctext">Primer designing: CMV_forward/reverse_qPCR</SPAN></A></LI><LI class="toclevel-2 tocsection-45"><A href="#Ordering_of_required_reagents:"><SPAN class="tocnumber">16.3</SPAN><SPAN class="toctext">Ordering of required reagents:</SPAN></A></LI><LI class="toclevel-1 tocsection-46"><A href="#34._Labortag_18.06.2010"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">34. Labortag 18.06.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-47"><A href="#Transfection"><SPAN class="tocnumber">17.1</SPAN><SPAN class="toctext">Transfection</SPAN></A></LI><LI class="toclevel-2 tocsection-48"><A href="#Theoretical:_Digestion_of_pAAV_MCS_for_PCR_to_design_ITR_BioBrick"><SPAN class="tocnumber">17.2</SPAN><SPAN class="toctext">Theoretical: Digestion of pAAV_MCS for PCR to design ITR BioBrick</SPAN></A></LI><LI class="toclevel-1 tocsection-49"><A href="#35._Labortag_19.06.2010:_Literature_seminar"><SPAN class="tocnumber">18</SPAN><SPAN class="toctext">35. Labortag 19.06.2010: Literature seminar</SPAN></A></LI><LI class="toclevel-1 tocsection-50"><A href="#36.Labortag_20.06.2010:"><SPAN class="tocnumber">19</SPAN><SPAN class="toctext">36.Labortag 20.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-51"><A href="#Picking_clones_of_sequenced_pAAV_igEM_mVenus_YFP"><SPAN class="tocnumber">19.1</SPAN><SPAN class="toctext">Picking clones of sequenced pAAV_igEM_mVenus_YFP</SPAN></A></LI><LI class="toclevel-1 tocsection-52"><A href="#37.Labortag_21.06.2010:"><SPAN class="tocnumber">20</SPAN><SPAN class="toctext">37.Labortag 21.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-53"><A href="#Second_Transfection.2C_Transduction"><SPAN class="tocnumber">20.1</SPAN><SPAN class="toctext">Second Transfection, Transduction</SPAN></A></LI><LI class="toclevel-2 tocsection-54"><A href="#Feching_dry_ice_for_the_experiments"><SPAN class="tocnumber">20.2</SPAN><SPAN class="toctext">Feching dry ice for the experiments</SPAN></A></LI><LI class="toclevel-2 tocsection-55"><A href="#Annotation_of_the_BRs_and_the_PLA2_domains"><SPAN class="tocnumber">20.3</SPAN><SPAN class="toctext">Annotation of the BRs and the PLA2 domains</SPAN></A></LI><LI class="toclevel-2 tocsection-56"><A href="#Endotoxin-free_Midi-Prep_of_pAAV_iGEM_mVenus-YFP"><SPAN class="tocnumber">20.4</SPAN><SPAN class="toctext">Endotoxin-free Midi-Prep of pAAV_iGEM_mVenus-YFP</SPAN></A></LI><LI class="toclevel-2 tocsection-57"><A href="#Second_Transfection"><SPAN class="tocnumber">20.5</SPAN><SPAN class="toctext">Second Transfection</SPAN></A></LI><LI class="toclevel-2 tocsection-58"><A href="#Preparing_Viral_Stocks"><SPAN class="tocnumber">20.6</SPAN><SPAN class="toctext">Preparing Viral Stocks</SPAN></A></LI><LI class="toclevel-2 tocsection-59"><A href="#HT1080_Transduction"><SPAN class="tocnumber">20.7</SPAN><SPAN class="toctext">HT1080 Transduction</SPAN></A></LI><LI class="toclevel-1 tocsection-60"><A href="#38.Labortag_22.06.2010:"><SPAN class="tocnumber">21</SPAN><SPAN class="toctext">38.Labortag 22.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-61"><A href="#Midiprep_of_pHelper_plasmid.3B_cloning_of_the_rfc25_MCS"><SPAN class="tocnumber">21.1</SPAN><SPAN class="toctext">Midiprep of pHelper plasmid; cloning of the rfc25 MCS</SPAN></A></LI><LI class="toclevel-1 tocsection-62"><A href="#39.Labortag_23.06.2010:"><SPAN class="tocnumber">22</SPAN><SPAN class="toctext">39.Labortag 23.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-63"><A href="#Preparation_of_Viral_Stock.2C_Seeding_cells"><SPAN class="tocnumber">22.1</SPAN><SPAN class="toctext">Preparation of Viral Stock, Seeding cells</SPAN></A></LI><LI class="toclevel-2 tocsection-64"><A href="#Design_of_Primer"><SPAN class="tocnumber">22.2</SPAN><SPAN class="toctext">Design of Primer</SPAN></A></LI><LI class="toclevel-2 tocsection-65"><A href="#Sequencing_of_pKEX-VP1"><SPAN class="tocnumber">22.3</SPAN><SPAN class="toctext">Sequencing of pKEX-VP1</SPAN></A></LI><LI class="toclevel-2 tocsection-66"><A href="#Thaw_HT1080_and_AAV-293_cells"><SPAN class="tocnumber">22.4</SPAN><SPAN class="toctext">Thaw HT1080 and AAV-293 cells</SPAN></A></LI><LI class="toclevel-2 tocsection-67"><A href="#Picking_clones_of_pAAV_RFC25"><SPAN class="tocnumber">22.5</SPAN><SPAN class="toctext">Picking clones of pAAV_RFC25</SPAN></A></LI><LI class="toclevel-2 tocsection-68"><A href="#Transduction_results"><SPAN class="tocnumber">22.6</SPAN><SPAN class="toctext">Transduction results</SPAN></A></LI><LI class="toclevel-2 tocsection-69"><A href="#Preparation_of_viral_stocks"><SPAN class="tocnumber">22.7</SPAN><SPAN class="toctext">Preparation of viral stocks</SPAN></A></LI><LI class="toclevel-1 tocsection-70"><A href="#40.Labortag_24.06.2010:"><SPAN class="tocnumber">23</SPAN><SPAN class="toctext">40.Labortag 24.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-71"><A href="#Labmeeting.2C_Presentation.2FUpdate_by_Patrick_and_Adrian"><SPAN class="tocnumber">23.1</SPAN><SPAN class="toctext">Labmeeting, Presentation/Update by Patrick and Adrian</SPAN></A></LI><LI class="toclevel-2 tocsection-72"><A href="#Midi-Prep_of_pAAV_iGEM_mVenus-YFP"><SPAN class="tocnumber">23.2</SPAN><SPAN class="toctext">Midi-Prep of pAAV_iGEM_mVenus-YFP</SPAN></A></LI><LI class="toclevel-2 tocsection-73"><A href="#Mini_Prep_of_P42_and_P43_pAAV-RFC-25"><SPAN class="tocnumber">23.3</SPAN><SPAN class="toctext">Mini Prep of P42 and P43 pAAV-RFC-25</SPAN></A></LI><LI class="toclevel-2 tocsection-74"><A href="#ITR_modification_via_PCR"><SPAN class="tocnumber">23.4</SPAN><SPAN class="toctext">ITR modification via PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-75"><A href="#Work_on_pKEX_backbones"><SPAN class="tocnumber">23.5</SPAN><SPAN class="toctext">Work on pKEX backbones</SPAN></A></LI><LI class="toclevel-1 tocsection-76"><A href="#41.Labortag_25.06.2010:"><SPAN class="tocnumber">24</SPAN><SPAN class="toctext">41.Labortag 25.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-77"><A href="#PCR_of_ITRs.2C_Splitting_HT1080_and_AAV-293_cells.2C_Seeding_of_HT1080_cells.2C_Creating_Virus_Stock_Economics.2C_Medium_check"><SPAN class="tocnumber">24.1</SPAN><SPAN class="toctext">PCR of ITRs, Splitting HT1080 and AAV-293 cells, Seeding of HT1080 cells, Creating Virus Stock Economics, Medium check</SPAN></A></LI><LI class="toclevel-2 tocsection-78"><A href="#Splitting_of_HT1080_and_AAV-293_cells.2C_Seeding_of_HT1080_cells"><SPAN class="tocnumber">24.2</SPAN><SPAN class="toctext">Splitting of HT1080 and AAV-293 cells, Seeding of HT1080 cells</SPAN></A></LI><LI class="toclevel-2 tocsection-79"><A href="#Creating_Virus_Stock_Economics"><SPAN class="tocnumber">24.3</SPAN><SPAN class="toctext">Creating Virus Stock Economics</SPAN></A></LI><LI class="toclevel-2 tocsection-80"><A href="#Medium_check"><SPAN class="tocnumber">24.4</SPAN><SPAN class="toctext">Medium check</SPAN></A></LI><LI class="toclevel-2 tocsection-81"><A href="#Sequencing_results_of_pAAV_RFC25"><SPAN class="tocnumber">24.5</SPAN><SPAN class="toctext">Sequencing results of pAAV_RFC25</SPAN></A></LI><LI class="toclevel-2 tocsection-82"><A href="#Annotation_and_studies_of_the_AAV_structure"><SPAN class="tocnumber">24.6</SPAN><SPAN class="toctext">Annotation and studies of the AAV structure</SPAN></A></LI><LI class="toclevel-1 tocsection-83"><A href="#42.Labortag_26.06.2010"><SPAN class="tocnumber">25</SPAN><SPAN class="toctext">42.Labortag 26.06.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-84"><A href="#Transduction_of_HT1080_cells"><SPAN class="tocnumber">25.1</SPAN><SPAN class="toctext">Transduction of HT1080 cells</SPAN></A></LI><LI class="toclevel-2 tocsection-85"><A href="#Paper_reading_session"><SPAN class="tocnumber">25.2</SPAN><SPAN class="toctext">Paper reading session</SPAN></A></LI><LI class="toclevel-1 tocsection-86"><A href="#43.Labortag_28.06.2010:"><SPAN class="tocnumber">26</SPAN><SPAN class="toctext">43.Labortag 28.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-87"><A href="#Checking_HT1080_for_YFP-Expression.2C_Creating_plan:_urgent_things_to_do"><SPAN class="tocnumber">26.1</SPAN><SPAN class="toctext">Checking HT1080 for YFP-Expression, Creating plan: urgent things to do</SPAN></A></LI><LI class="toclevel-2 tocsection-88"><A href="#Sequencing_results_of_P42"><SPAN class="tocnumber">26.2</SPAN><SPAN class="toctext">Sequencing results of P42</SPAN></A></LI><LI class="toclevel-2 tocsection-89"><A href="#Theoretical_construction_of_our_biobricks"><SPAN class="tocnumber">26.3</SPAN><SPAN class="toctext">Theoretical construction of our biobricks</SPAN></A></LI><LI class="toclevel-2 tocsection-90"><A href="#Urgent_Things_to_do:"><SPAN class="tocnumber">26.4</SPAN><SPAN class="toctext">Urgent Things to do:</SPAN></A></LI><LI class="toclevel-2 tocsection-91"><A href="#Continuation_of_ITR_PCR"><SPAN class="tocnumber">26.5</SPAN><SPAN class="toctext">Continuation of ITR PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-92"><A href="#ITRs_.28theoretical.29"><SPAN class="tocnumber">26.6</SPAN><SPAN class="toctext">ITRs (theoretical)</SPAN></A></LI><LI class="toclevel-1 tocsection-93"><A href="#44.Labortag_29.06.2010:"><SPAN class="tocnumber">27</SPAN><SPAN class="toctext">44.Labortag 29.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-94"><A href="#Discussionround"><SPAN class="tocnumber">27.1</SPAN><SPAN class="toctext">Discussionround</SPAN></A></LI><LI class="toclevel-2 tocsection-95"><A href="#Seeding_HT-cells_for_transfection_and_titering"><SPAN class="tocnumber">27.2</SPAN><SPAN class="toctext">Seeding HT-cells for transfection and titering</SPAN></A></LI><LI class="toclevel-2 tocsection-96"><A href="#Seeding_HT1080-cells_for_transduction_and_qPCR"><SPAN class="tocnumber">27.3</SPAN><SPAN class="toctext">Seeding HT1080-cells for transduction and qPCR</SPAN></A></LI><LI class="toclevel-2 tocsection-97"><A href="#Transduction_of_HT1080_cells_for_qPCR"><SPAN class="tocnumber">27.4</SPAN><SPAN class="toctext">Transduction of HT1080 cells for qPCR</SPAN></A></LI><LI class="toclevel-1 tocsection-98"><A href="#45.Labortag_30.06.2010:"><SPAN class="tocnumber">28</SPAN><SPAN class="toctext">45.Labortag 30.06.2010:</SPAN></A></LI><LI class="toclevel-2 tocsection-99"><A href="#Titering_Procedure_via_qPCR.2C_Transduction_of_6well_plates_no.4"><SPAN class="tocnumber">28.1</SPAN><SPAN class="toctext">Titering Procedure via qPCR, Transduction of 6well plates no.4</SPAN></A></LI><LI class="toclevel-2 tocsection-100"><A href="#beta-globin_intron"><SPAN class="tocnumber">28.2</SPAN><SPAN class="toctext">beta-globin intron</SPAN></A></LI><LI class="toclevel-2 tocsection-101"><A href="#hGH_sequence"><SPAN class="tocnumber">28.3</SPAN><SPAN class="toctext">hGH sequence</SPAN></A></LI><LI class="toclevel-2 tocsection-102"><A href="#ITRs:_new_strategy"><SPAN class="tocnumber">28.4</SPAN><SPAN class="toctext">ITRs: new strategy</SPAN></A></LI><LI class="toclevel-2 tocsection-103"><A href="#Sequencing_of_pAAV_iGEM-mVenus-YFP"><SPAN class="tocnumber">28.5</SPAN><SPAN class="toctext">Sequencing of pAAV_iGEM-mVenus-YFP</SPAN></A></LI><LI class="toclevel-2 tocsection-104"><A href="#Investigation_of_the_Kleinschmidt_sequencing"><SPAN class="tocnumber">28.6</SPAN><SPAN class="toctext">Investigation of the Kleinschmidt sequencing</SPAN></A></LI></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="18._Labortag_01.06.2010:"><P style="font-size:17px; background-color:#00dd77;">18. Labortag 01.06.2010:</P></SPAN></H4><H5><SPAN class="mw-headline" id="Modifying_MCS_of_pAAV_MCS_vector"><P style="font-size:15px; background-color:#66bbff;">Modifying MCS of pAAV_MCS vector</P></SPAN></H5><P><B>Investigators: Anissa, Adrian, Bea, Chris W., Hanna, Patrick, Volker, Sven </B></P><P><B>Oligos received from Sigma-Aldrich</B>
(right ITR of pAAV_MCS, left ITR of pAAV_MCS and MCS RFC25 for pAAV)
</P><UL><LI>Hybrization of received oligos: MCS RFC25 for pAAV (forward) and MCS RFC25 for pAAV (reverse)
</LI><LI>Centrifuge tubes prior to open tubes (13.000 rpm, 30 sec)
</LI><LI>MCS RFC25 for pAAV (forward): Add 92µL Millipore H<SUB>2</SUB>O (Volume on obtained sheet)
</LI><LI>MCS RFC25 for pAAV (reverse): Add 394 µL Millipore H<SUB>2</SUB>O (Volume on obtained sheet)
</LI></UL><LI>Vortex the resuspended DNA
</LI><LI>Make aliquots of both Oligos (1:10): 10 µL Oligo + 90 µLH<SUB>2</SUB>O (final volume usually 100 µl)
</LI><LI>Mix together(into PCR-tube):
</LI><TABLE border="1"><TBODY><TR><TD align="right"><B>Volume/µL</B></TD><TD align="right"><B>solution</B></TD></TR><TR><TD align="right"> 10 (1:10)</TD><TD align="right">Oligo 1: MCS RFC25 for pAAV (forward)
</TD></TR><TR><TD align="right">10</TD><TD align="right">Oligo 2: MCS RFC25 for pAAV (reverse)
</TD></TR><TR><TD align="right">4</TD><TD align="right"> 100mM TrisHCl pH8
</TD></TR><TR><TD align="right">8</TD><TD align="right">5mM MgCl2
</TD></TR><TR><TD align="right">8</TD><TD align="right"> H20
</TD></TR></TBODY></TABLE><UL><LI> Program: ORIGAMI 1 modified for long oligos:
</LI><LI> 1    99°C    7’
</LI><LI> 2    99°C    1’
</LI><LI> -1°C  R=0.3 °/s
</LI><LI> Goto 2 rep 74
</LI><LI> Hold 4°C
</LI></UL><UL><LI>While hybridization of oligos is performed, digestion of pAAV_MCS vector can be conducted </LI></UL><P>following standard protocol for cloning. 
</P><UL><LI>Title: Ligation MCS_Oligo with pAAV_MCS
</LI><LI>Plasmid: pAAV_MCS
</LI><LI>Buffer used: 3
</LI><LI>BSA: Yes
</LI><LI>Measure DNA-concentration with Nanodrop
</LI><LI>DNA-Concentration:260 ng/uL
</LI><LI>Restriction-enzyms used: <A href="http://www.neb.com/nebecomm/DoubleDigestCalculator.asp" class="external free" rel="nofollow">http://www.neb.com/nebecomm/DoubleDigestCalculator.asp</A></LI></UL><P>Enzyme1 (Nr. Lab: 152): ClaI
Enzyme2 (Nr. Lab: 15): BglII     
</P><UL><LI>Digestion components :
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>pAAV_MCS</B></TD></TR><TR><TD> DNA  </TD><TD align="right"> 4
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">3
</TD></TR><TR><TD> Buffer 3 (10x)</TD><TD align="right">3
</TD></TR><TR><TD>Enzyme: ClaI (no.Lab:152)</TD><TD align="right">2
</TD></TR><TR><TD>Enzyme: BglII (no.Lab:15)</TD><TD align="right">1
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">17
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"><B>30</B></TD></TR></TBODY></TABLE><UL><LI>Incubate for 1,5 h at 37°C
</LI></UL><UL><LI>1% Agarose gel 
</LI><LI>1% agarose gel was prepared, gel ran for 45 minutes( first: 90V, after 15 minutes: 115 V)
</LI></UL><UL><LI>Amount of loading dye added
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>sample/µL </B></TD><TD align="right"><B>loading dye/µL</B></TD></TR><TR><TD> marker: 8 </TD><TD align="right">contains loading dye
</TD></TR><TR><TD>pAAV_MCS: 24 </TD><TD align="right">6
</TD></TR></TBODY></TABLE><UL><LI>Expected size of fragments
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>sample</B></TD><TD align="right"><B>expected size</B></TD></TR><TR><TD align="right"> pAAV_MCS: cut with ClaI and BglII</TD><TD align="right">4580 bp
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="19._Labortag_02.06.2010:"><P style="font-size:17px; background-color:#00dd77;">19. Labortag 02.06.2010:</P></SPAN></H4><H5><SPAN class="mw-headline" id="Picking_clones_of_pAAV_RFC25MCS"><P style="font-size:17px; background-color:#66bbff;">Picking clones of pAAV_RFC25MCS</P></SPAN></H5><P><B>Investigators: Adrian, Bea, Chris W., Hanna, Anissa</B></P><P><B>Practical work:</B>
Control plate contained no clones. :)
4 colonies were picked and grown @ 37°C over night.
</P><H5><SPAN class="mw-headline" id="Design_and_ordering_of_Oligos_.28NotI.29"><P style="font-size:17px; background-color:#66bbff;">Design and ordering of Oligos (NotI)</P></SPAN></H5><P><B>Investigators: Adrian, Bea, Chris W., Hanna, Anissa</B></P><P>Oligos for site directed mutagenesis of the NotI restriction sites in pAAV_MCS (ITRs) were designed: <A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=Freiburg10_NotI_ITR_Oligos.pdf" class="new" title="Freiburg10 NotI ITR Oligos.pdf">Media:Freiburg10 NotI ITR Oligos.pdf</A></P><H5><SPAN class="mw-headline" id="Sponsoring_work:"><P style="font-size:15px; background-color:#66bbff;">Sponsoring work:</P></SPAN></H5><P>Sponsoring letter was adapted for Quiagen.
</P><H4><SPAN class="mw-headline" id="20._Labortag_03.06.2010:"><P style="font-size:17px; background-color:#00dd77;">20. Labortag 03.06.2010: </P></SPAN></H4><H5><SPAN class="mw-headline" id="pAAV_RFC25_MCS_-.3E_problem"><P style="font-size:15px; background-color:#66bbff;"> pAAV_RFC25_MCS -&gt; problem</P></SPAN></H5><P><B>Investigators: Anissa, Bea, Melanie, Christian L.</B></P><P><B>Comment</B>: Continue with Mini-Prep and test digestion of pAAV_RFC25_MCS 
Mini-Prep and test digestion have been performed: </P><P><SPAN style="color:red; font-weight:bold;">Problem</SPAN>: Designed oligos (MCS_RFC25) for altering the MCS of the pAAV_MCS vector cannot be used.  Two startcodons are in the MCS. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Oligo MCS_RFC25: contains 2 Startcodons. the &quot;wrong&quot; Codon is the first ATG which results in the peptide chain with 30 aa. The second ATG is the right ATG which is right before the Gene of Interest.</DIV></DIV></DIV></DIV><P>
The two startcodons are not in the same open reading frame (ORF). Therefore two proteins will be produced. The Gene of Interest and the short peptide (30 aa).
The idea of the oligos was to ligate the oligos into the pAAV_MCS vector. The oligos contained two overhangs which correspond to the sequences of the two restriction sites ClaI and BglII: 
The pAAV_MCS vector was digested with ClaI and BglII and then we ligated the oligo and the vector. Problem was that we did not notice that the overhang of ClaI and the sequence of EcoRI of the MCS_RFC25 resulted in another ATG startcodon. 
<B>Possible Solutions</B>: 
</P><UL><OL><LI> first: modify MCS with ordered oligos of Sven (shorter MCS which cannot be used for pEX)and clone mVenus
</LI><LI> Perform site-directed-mutagenesis (QuikChange from Stratagene)
</LI><LI> Order new MCS-oligos and consider that <B>no</B> new ATG is produced. For example: add another base between ClaI overhang and EcoRI sequence. -----AT<B>X</B>G---- <B>This solution is the more possible one we are going to perform.</B></LI></OL></UL><P><SPAN style="color:blue; font-weight:bold;">Practical work</SPAN></P><UL><UL><LI>Preparing four glycerol stocks (2:1)
</LI><LI>numbers: B4 - B7 (for details see nomenclature)
</LI><LI>stored in -80°C, Box 1
</LI></UL></UL><UL><LI><B>MiniPrep</B></LI><LI>Nanodrop concentrations
</LI></UL><TABLE border="1"><TBODY><TR><TD align="right"><B>Sample</B></TD><TD align="right"><B>Concentration/ng*µl-1</B></TD></TR><TR><TD align="right"> P11 </TD><TD align="right">340,5
</TD></TR><TR><TD align="right"> P12 </TD><TD align="right">364,0
</TD></TR><TR><TD align="right"> P13 </TD><TD align="right">358,5
</TD></TR><TR><TD align="right"> P14 </TD><TD align="right">284,4
</TD></TR></TBODY></TABLE><UL><LI>Test digestion
</LI></UL><TABLE border="1"><TBODY><TR><TD> Components   </TD><TD align="right">Volume µl </TD><TD align="right"> Mastermix µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 800 </TD><TD align="right"> --
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right"> 1,5 </TD><TD align="right"> 7,5
</TD></TR><TR><TD> Buffer No.2 (10x)</TD><TD align="right"> 1,5 </TD><TD align="right"> 7,5
</TD></TR><TR><TD>Enzyme 1 (no.Lab:45) Nde I </TD><TD align="right"> 0,5 </TD><TD align="right"> 2,5
</TD></TR><TR><TD>Enzyme 2 (no.Lab:71) Spe I </TD><TD align="right"> 0,5 </TD><TD align="right"> 2,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right"> variable </TD><TD align="right"> --
</TD></TR><TR><TD><B>Total volume </B></TD><TD align="right"> 15 </TD><TD align="right"> 20</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD> Sample   </TD><TD align="right"> Volume/ µl </TD><TD align="right"> H<SUB>2</SUB>O / µl
</TD></TR><TR><TD> P11  </TD><TD align="right"> 2,3 </TD><TD align="right"> 8,7
</TD></TR><TR><TD> P12  </TD><TD align="right"> 2,2 </TD><TD align="right"> 8,8
</TD></TR><TR><TD> P13  </TD><TD align="right"> 2,2 </TD><TD align="right"> 8,8
</TD></TR><TR><TD> P14  </TD><TD align="right"> 2,8 </TD><TD align="right"> 8,2
</TD></TR></TBODY></TABLE><UL><LI>Incubation: 1,5 h
</LI></UL><LI>Agarose-Gel

</LI><LI>Materials
</LI>
0,5 g Agarose, 50 ml TAE, 3µl GELRED, at 100 Volt, running time: 45 minutes


<TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (5x/6x)/µl
</TH><TH>Expected size 1 (Geneious)
</TH><TH>Expected size 2 (Geneious)
</TH></TR><TR><TD>P11
</TD><TD>15 µl
</TD><TD>3 µl
</TD><TD>3677 bp
</TD><TD> 974 bp
</TD></TR><TR><TD>P12
</TD><TD>15 µl
</TD><TD>3 µl
</TD><TD>3677 bp
</TD><TD> 974 bp
</TD></TR><TR><TD>P13
</TD><TD>15 µl
</TD><TD>4 µl
</TD><TD>3677 bp
</TD><TD> 974 bp
</TD></TR><TR><TD>P14
</TD><TD>15 µl
</TD><TD>4 µl
</TD><TD>3677 bp
</TD><TD> 974 bp
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>Sample P11 /18 µl
</TH><TH>Sample P12 /18 µl
</TH><TH>Sample P13 /19 µl
</TH><TH>Sample P14 /19 µl
</TH></TR><TR><TH>Lane
</TH><TD> 1
</TD><TD> 3
</TD><TD> 5
</TD><TD> 7
</TD><TD> 9
</TD></TR></TBODY></TABLE><P><B>Results of agarose-gel:</B></P><UL><UL><LI>Expected fragments of 3677 bp and 974 bp can be verified. The insertion of the RFC25_MCS has been inserted. </LI></UL>
For further verification the insert has to be sequenced (to do for 04.06.2010). </UL><H5><SPAN class="mw-headline" id="Picking_clones_of_Thymindinkinase_of_Amor"><P style="font-size:15px; background-color:66bbff;">Picking clones of Thymindinkinase of Amor</P></SPAN></H5><UL><LI>5 clones of the XL-1 Blue colonies containing the plasmid <B>pUB6/HV5/His6 with the thymidinkinase-guanylate fusion enzyme</B> have been picked from LBamp-agarplates
</LI><LI>1 clone of the XL-1 Blue colonies containing the plasmid <B>pUB6/HV5/His6 without the thymidinkinase (control) has been picked from LBamp-agarplates</B></LI><LI>all clones have been inoculated in 10 mL LB containing 10 µL Amp. Incubation: 37°C over-night.
</LI><LI><B>to do: Mini-Prep of pUB6/HV5/His6 with the thymidinkinase and pUB6/HV5/His6 without the thymidinkinase (control)</B></LI></UL><P><SPAN style="color:blue; font-weight:bold;">Idea</SPAN></P><UL><UL><LI>Insertion of Kozak consensus sequence before MCS to enhance gene expression (cloning consideration in Stratagene manual)
</LI><LI> New RFC standard with Kozak sequence for eucaryotes??
</LI></UL></UL><H4><SPAN class="mw-headline" id="21._Labortag_04.06.2010:"><P style="font-size:17px; background-color:#00dd77;">21. Labortag 04.06.2010: </P></SPAN></H4><H5><SPAN class="mw-headline" id="DKFZ_plasmid_Retrafos.2C_TK.2FGMK_Mini-Prep"><P style="font-size:15px; background-color:#66bbff;">DKFZ plasmid Retrafos, TK/GMK Mini-Prep</P></SPAN></H5><P><B>Investigators: Adrian, Bea, Chris W., Hanna</B></P><P style="font-size:15px; font-weight: bold; color: green;"><U>DKFZ</U></P><P><B>Comments</B>: Plasmids of PD Kleinschmidt of the DKFZ arrived. The DNA was dried on a whatman paper.
Plasmids received: </P><UL><LI><B>pXX6</B> all Adenovirus helper genes without AAV genes
</LI><LI><B>pKEX-2XL.Rep40</B> expression plasmid for Rep protein 40 
</LI><LI><B>pKEX-2XL.Rep 52</B> expression plasmid for Rep protein 52
</LI><LI><B>pKEX-2XL.Rep 68</B> expression plasmid for Rep protein 68
</LI><LI><B>pKEX-2XL.Rep 78</B> expression plasmid for Rep protein 78
</LI><LI><B>pCMV-VP(HS)</B> expression plasmid for VP proteins in combination (assemly competent)
</LI><LI><B>pKEX-VP1</B> expression plasmid for single VP proteins (Cap: VP1, VP2 und VP3). Be careful: Single constructs are not assembly competent; Ruffing et al 1992; Steinbach et al. 1997;
</LI><LI><B>pKEX-VP2</B>expression plasmid for single VP proteins (Cap: VP1, VP2 und VP3). Be careful: Single constructs are not assembly competent; Ruffing et al 1992; Steinbach et al. 1997;
</LI><LI><B>pKEX-VP3</B>expression plasmid for single VP proteins (Cap: VP1, VP2 und VP3). Be careful: Single constructs are not assembly competent; Ruffing et al 1992; Steinbach et al. 1997;
</LI><LI><B>pTRUF_CMV_eGFP</B>single stranded vector for eGFP expression
</LI><LI><B>dsAAV_CMV_eGFP</B>&quot;self complementary&quot; expression vector from X. Xiao
</LI><LI><B>pTAV2 (whole AAV genome in &quot;blue script&quot; vector)</B></LI><LI><B>pDG</B> complete helper plasmid, nedded for vector production ; Grimm et al. 1998
</LI></UL><UL><LI>In order to obtain the DNA following steps have been performed:
</LI><LI>cut out the spot where the DNA is spotted with a clean scalpel (note: scalpel should not have any contamination).
</LI><LI>put a 0,5 mL Eppi in a 1,5 mL Eppi. Put little holes in the smaller eppi.
</LI><LI>transfer Whatman paper into Eppi
</LI><LI>add 50 µL TE-EF (Redissolving buffer) to whatman paper and wait 15 minutes
</LI><LI>centrifuge eppis at 2000 rpm, 10 minutes
</LI></UL><LI>Transformation with obtained plasmids was performed.
</LI><P style="font-size:15px; font-weight: bold; color: green;"><U>Fusion Enzyme: Thymidinkinase/Guanylate Kinase (TK/GMK)</U></P><P style="font-size:12px; font-weight: bold; color: blue;"><U>Plasmid Mini-Prep</U></P><UL><LI>experiment date: 04.06.2010 ; time: 3,5h
</LI><LI>name of investigator: Adrian, Bea, ChrisW.,Hanna
</LI><LI>new vector name: pUB_V5_His6 + TK/GMK (fusionenzyme)
</LI></UL><P><U>Glycerol Stocks</U></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD><TD align="left"><B>Clone 3</B></TD><TD align="left"><B>Clone 4</B></TD><TD align="left"><B>Clone 5</B></TD><TD align="left"><B>Control</B></TD></TR><TR><TD align="left"><B>Bacteria strain</B></TD><TD align="left"> XL-1 Blue </TD><TD align="left"> XL-1 Blue </TD><TD align="left"> XL-1 Blue </TD><TD align="left"> XL-1 Blue </TD><TD align="left"> XL-1 Blue </TD><TD align="left"> XL-1 Blue
</TD></TR><TR><TD align="left"><B>Plasmidname</B></TD><TD align="left"> pUB_V5_His6 + TK/GMK </TD><TD align="left"> pUB_V5_His6 + TK/GMK </TD><TD align="left"> pUB_V5_His6 + TK/GMK </TD><TD align="left"> pUB_V5_His6 + TK/GMK </TD><TD align="left"> pUB_V5_His6 + TK/GMK </TD><TD align="left"> pUB_V5_His6 + TK/GMK
</TD></TR><TR><TD align="left"><B>Date</B></TD><TD align="left"> 04.06.2010 </TD><TD align="left"> 04.06.2010 </TD><TD align="left"> 04.06.2010 </TD><TD align="left"> 04.06.2010  </TD><TD align="left"> 04.06.2010 </TD><TD align="left"> 04.06.2010
</TD></TR><TR><TD align="left"><B>given number</B></TD><TD align="left"> B8 </TD><TD align="left"> B9 </TD><TD align="left"> B10 </TD><TD align="left"> B11 </TD><TD align="left"> B12 </TD><TD align="left"> B13
</TD></TR></TBODY></TABLE><P><U>Given Plasmid-Number</U></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD><TD align="left"><B>Clone 3</B></TD><TD align="left"><B>Clone 4</B></TD><TD align="left"><B>Clone 5</B></TD><TD align="left"><B>Control</B></TD></TR><TR><TD align="left"><B>given number</B></TD><TD align="left"> P15 </TD><TD align="left"> P16 </TD><TD align="left"> P17 </TD><TD align="left"> P18 </TD><TD align="left"> P19 </TD><TD align="left"> P20
</TD></TR></TBODY></TABLE><P style="font-size:12px; font-weight: bold; color: blue;"><U>Nanodrop concentration</U></P><UL><LI>Plasmid
</LI><LI>Given Plasmid-Number: P15; DNA concentration: 493,7 ng/µL ;
</LI><LI>Given Plasmid-Number: P16; DNA concentration: 464,4 ng/µL;
</LI><LI>Given Plasmid-Number: P17; DNA concentration: 445,6 ng/µL;
</LI><LI>Given Plasmid-Number: P18; DNA concentration: 562,9 ng/µL;
</LI><LI>Given Plasmid-Number: P19; DNA concentration: 528,1 ng/µL;
</LI><LI>Given Plasmid-Number: P20; DNA concentration: 499,9 ng/µL;
</LI></UL><P><B>Comments:</B>A Plasmid-Mini Prep with the received Fusionenzyme Thymidinkinase/Guanylate Kinase (TK/GMK)from Amor has been performed.
The DNA will be sent to GATC for sequencing.
</P><UL><LI>pUB6_V5_His6 - clone 1 (P15) (3 µL added to 27µL H20) has been sent to GATC.
</LI><LI>Expected results: Saturday
</LI></UL><H4><SPAN class="mw-headline" id="22._Labortag_05.06.2010:"><P style="font-size:17px; background-color:#00dd77;">22. Labortag 05.06.2010: </P></SPAN></H4><H5><SPAN class="mw-headline" id="Insertion_of_iGEM_expression_parts_into_pAAV_MCS"><P style="font-size:15px; background-color:#66bbff;">Insertion of iGEM expression parts into pAAV_MCS</P></SPAN></H5><P><B>Investigators: Adrian, Bea, Melanie, Hanna</B></P><P style="font-size:15px; font-weight: bold; color: blue;">Hybridization:</P><UL><LI>Hybridization of received oligos: iGEM expression parts (RFC25 without EcoRI, NotI)
</LI><LI>Prior to opening the tubes, they were centrifugated at 13.000 rpm for 30 sec.
</LI><LI>expression part MCS_for (charge-no: ST00114065): 108 µL Millipore H<SUB>2</SUB>O (Volume on obtained sheet)were added.
</LI><LI>expression part MCS_for (charge-no: ST00114066): 165 µL Millipore H<SUB>2</SUB>O (Volume on obtained sheet)were added.
</LI></UL><LI>Resuspended DNA was vortexted.
</LI><LI>Aliquots of both Oligos (1:10) were prepared: 10 µL Oligo + 90 µL H<SUB>2</SUB>O (final volume usually 100 µl).
</LI><LI>Mix together(into PCR-tube):
</LI><TABLE border="1"><TBODY><TR><TD align="right"><B>Volume/µL</B></TD><TD align="right"><B>solution</B></TD></TR><TR><TD align="right"> 10 (1:10)</TD><TD align="right">Oligo 1: expression part MCS_for (charge-no: ST00114065)
</TD></TR><TR><TD align="right">10 (1:10)</TD><TD align="right">Oligo 2: expression part MCS_for (charge-no: ST00114066)
</TD></TR><TR><TD align="right">4</TD><TD align="right"> 100 mM TrisHCl pH8
</TD></TR><TR><TD align="right">8</TD><TD align="right">5 mM MgCl2
</TD></TR><TR><TD align="right">8</TD><TD align="right"> H20
</TD></TR></TBODY></TABLE><UL><LI> Program: ORIGAMI 1 modified for long oligos:
</LI><LI> 1    99°C    7’
</LI><LI> 2    99°C    1’
</LI><LI> -1°C  R=0.3 °/s
</LI><LI> Goto 2 rep 74
</LI><LI> Hold 4°C
</LI></UL><UL><LI>While hybridization of oligos was performed, digestion of pAAV_MCS vector was conducted </LI></UL><P>following standard protocol for cloning. 
</P><P style="font-size:15px; font-weight: bold; color: blue;">Digestion:</P><UL><LI>Title: Ligation iGEM expression parts (=&quot;iGEM-MCS&quot;) with pAAV_MCS
</LI><LI>Plasmid: pAAV_MCS
</LI><LI>Buffer used: 3
</LI><LI>BSA: Yes
</LI><LI>DNA-Concentration: 260 ng/uL
</LI><LI>Restriction-enzyms used:
</LI></UL><P>Enzyme1 (Nr. Lab: 152): ClaI
Enzyme2 (Nr. Lab: 15): BglII     
</P><UL><LI>Digestion components :
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>pAAV_MCS</B></TD></TR><TR><TD> DNA  </TD><TD align="right"> 5.8 µL
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">3 µL
</TD></TR><TR><TD> Buffer 3 (10x)</TD><TD align="right">2 µL
</TD></TR><TR><TD>Enzyme: ClaI (no.Lab:152)</TD><TD align="right">2 µL
</TD></TR><TR><TD>Enzyme: BglII (no.Lab:15)</TD><TD align="right">1 µL
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">16.2 µL
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"><B>30 µL</B></TD></TR></TBODY></TABLE><UL><LI>Mixture was incubated for 1,5 h at 37°C.
</LI></UL><P style="font-size:15px; font-weight: bold; color: blue;">Agarose-Gel:</P><UL><LI>1% agarose gel was prepared, gel ran for 45 minutes(110 V)
</LI></UL><UL><LI>Amount of loading dye added
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>sample/µL </B></TD><TD align="right"><B>loading dye/µL</B></TD></TR><TR><TD> marker: 8 </TD><TD align="right">contains loading dye
</TD></TR><TR><TD>pAAV_MCS: 30 </TD><TD align="right">6 (6x loading dye)
</TD></TR></TBODY></TABLE><UL><LI>Expected size of fragments
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>sample</B></TD><TD align="right"><B>expected size</B></TD></TR><TR><TD align="right"> pAAV_MCS: cut with ClaI and BglII</TD><TD align="right">4580 bp
</TD></TR></TBODY></TABLE><P>&lt;/ul&gt;</P><P style="font-size:15px; font-weight: bold; color: blue;">Gelextraction:</P><P>
Gel measurement: 
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Sample</B></TD><TD align="left"><B>weight</B></TD></TR><TR><TD align="left">pAAV_MCS </TD><TD align="left"> 60 mg
</TD></TR></TBODY></TABLE><UL><LI>Gelextraction was performed following standard protocol. 
</LI><LI>DNA-concentrations were meassured: pAAV_MCS 18.2 ng/µL, Oligos: 136.7 ng/µL -&gt; 1:10 dilution was prepared: 13.67 ng/µL
</LI></UL><P style="font-size:15px; font-weight: bold; color: blue;">Ligation:</P><TABLE border="1"><TBODY><TR><TD align="left"><B>iGEM-MCS</B></TD><TD align="left"><B>pAAV_MCS</B></TD></TR><TR><TD align="left"><B>Volume/µl</B></TD><TD align="left"> 0.4 </TD><TD align="left"> 8.6
</TD></TR></TBODY></TABLE><P>
Trafo was performed (using XL1B cells) following standard protocol.
</P><H4><SPAN class="mw-headline" id="23._Labortag_07.06.2010:"><P style="font-size:17px; background-color:#00dd77;">23. Labortag 07.06.2010:</P></SPAN></H4><H5><SPAN class="mw-headline" id="Mini-Preps_.28Kleinschmidt-plasmids_and_pAAV_iGEM-MCS.29"><P style="font-size:15px; background-color:#66bbff;">Mini-Preps (Kleinschmidt-plasmids and pAAV_iGEM-MCS)</P></SPAN></H5><P><B>Investigators: Investigators: Achim, Kira,Jessy, Chris W., Hanna, Adrian, Bea</B></P><P style="font-size:15px; font-weight: bold; color: fuchsia;"><U>pAAV_iGEM-MCS</U></P><P style="font-size:15px; font-weight: bold; color: blue;"><U>Plasmid Mini-Prep</U></P><UL><LI>experiment date:07.06.2010; time: whole day
</LI><LI>name of investigator: Achim, Kira,Jessy, Chris W., Hanna, Adrian, Bea
</LI></UL><P><U>Glycerol Stocks</U></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD><TD align="left"><B>Clone 3</B></TD><TD align="left"><B>Clone 4</B></TD></TR><TR><TD align="left"><B>Bacteria strain</B></TD><TD align="left"> XL1B </TD><TD align="left"> XL1B </TD><TD align="left"> XL1B </TD><TD align="left"> XL1B
</TD></TR><TR><TD align="left"><B>Plasmidname</B></TD><TD align="left"> pAAV_iGEM-MCS </TD><TD align="left"> pAAV_iGEM-MCS</TD><TD align="left"> pAAV_iGEM-MCS</TD><TD align="left"> pAAV_iGEM-MCS
</TD></TR><TR><TD align="left"><B>Date</B></TD><TD align="left"> 07.06.2010 </TD><TD align="left"> 07.06.2010</TD><TD align="left"> 07.06.2010</TD><TD align="left"> 07.06.2010
</TD></TR><TR><TD align="left"><B>given number</B></TD><TD align="left"> - </TD><TD align="left"> B27 </TD><TD align="left"> B28</TD><TD align="left"> B29
</TD></TR></TBODY></TABLE><P><U>Given Plasmid-Number</U></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD><TD align="left"><B>Clone 3</B></TD><TD align="left"><B>Clone 4</B></TD></TR><TR><TD align="left"><B>given number</B></TD><TD align="left"> - </TD><TD align="left"> P34.2 </TD><TD align="left"> P34.3</TD><TD align="left"> P34.4
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color: blue;">Test digestion</P><UL><LI>buffer used: 4 ; Restriction-enzymes used: Enzyme 1 (no. Lab:___) AgeI ; Enzyme 2 (no.Lab:___) NdeI
</LI><LI>Plasmid
</LI><LI>Given Plasmid-Number: P34.2; DNA concentration: 433.78 ng/µL ;
</LI><LI>Given Plasmid-Number: P34.3; DNA concentration: 408.48 ng/µL ;
</LI><LI>Given Plasmid-Number: P34.4; DNA concentration: 409.80 ng/µL ;
</LI></UL><P><B>Comments:</B>Clone no.1 was discarded. 
<B>Test digestion:</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>Volume/µL</B></TD><TD align="left"><B>Mastermix</B></TD></TR><TR><TD align="left"> DNA (clone 2)</TD><TD align="left"> 1000 ng </TD><TD align="left"> -
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> no </TD><TD align="left"> -
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 1.5 µL</TD><TD align="left"> -
</TD></TR><TR><TD align="left"> Enzyme 1 (no. Lab:    ) AgeI </TD><TD align="left"> 0.75 µL </TD><TD align="left"> -
</TD></TR><TR><TD align="left"> Enzyme 2 (no. Lab:    ) NdeI </TD><TD align="left"> 0.5 µL</TD><TD align="left"> -
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> variable </TD><TD align="left"> -
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>15 µL</B></TD><TD align="left"> -
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD> Sample   </TD><TD align="right"> Volume sample/ µl </TD><TD align="right"> Volume H<SUB>2</SUB>O / µl
</TD></TR><TR><TD> P34.2  </TD><TD align="right">2.3 </TD><TD align="right"> 9.95
</TD></TR><TR><TD> P34.3  </TD><TD align="right"> 2.4 </TD><TD align="right"> 9.85
</TD></TR><TR><TD> P34.4  </TD><TD align="right"> 2.4</TD><TD align="right"> 9.85
</TD></TR></TBODY></TABLE><UL><LI>Incubation: 45 min, 37°C
</LI></UL><P style="font-size:15px; font-weight: bold; color: blue;">Agarose-Gel:</P><P>
0.5 g Agarose, 50 ml TAE (1%), 3 µL GELRED (3-6µl), at 110 Volt, running time: 45 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (5x/6x)/µl
</TH><TH>Expected size 1 (Geneious)
</TH><TH>Expected size 2 (Geneious)
</TH></TR><TR><TD>P34.2
</TD><TD>15 µl
</TD><TD>3 µl
</TD><TD>3686 bp
</TD><TD>951 bp
</TD></TR><TR><TD>P34.3
</TD><TD>15 µl
</TD><TD>3 µl
</TD><TD>3686 bp
</TD><TD>951 bp
</TD></TR><TR><TD>P34.4
</TD><TD>15 µl
</TD><TD>3 µl
</TD><TD>3686 bp
</TD><TD>951 bp
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>Sample
</TH><TH>Sample
</TH><TH>Sample
</TH></TR><TR><TH>Lane
</TH><TD>34.2 / 18 µl
</TD><TD> 34.3 / 18 µl
</TD><TD>34.4 / 18 µl
</TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="Kleinschmidt-plasmids"><P style="font-size:15px; background-color:#66bbff;">Kleinschmidt-plasmids</P></SPAN></H5><P><B>Investigators: Kira, Achim, Jessy, Bea, Adrian, Hanna</B></P><P style="font-size:15px; font-weight: bold; color: blue;"><U>Plasmid Mini-Prep</U></P><UL><LI>experiment date: 07.06.2010 ; time: 10 – 20 h
</LI><LI>Kleinschmidt-plasmids
</LI></UL><P><U>Glycerol Stocks</U></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD><TD align="left"><B>Clone 3</B></TD><TD align="left"><B>Clone 4</B></TD><TD align="left"><B>Clone 5</B></TD><TD align="left"><B>Clone 6</B></TD><TD align="left"><B>Clone 7</B></TD><TD align="left"><B>Clone 8</B></TD><TD align="left"><B>Clone 9</B></TD></TR><TR><TD align="left"><B>Bacteria strain</B></TD><TD align="left"> XL1B </TD><TD align="left"> XL1B </TD><TD align="left"> XL1B </TD><TD align="left"> XL1B </TD><TD align="left"> XL1B </TD><TD align="left"> XL1B </TD><TD align="left"> XL1B </TD><TD align="left"> XL1B </TD><TD align="left"> XL1B
</TD></TR><TR><TD align="left"><B>Plasmidname</B></TD><TD align="left"> pXX6 </TD><TD align="left"> pKEX-2XL.Rep 40 </TD><TD align="left"> pKEX-2XL.Rep 52 </TD><TD align="left"> pKEX-2XL.Rep 68 </TD><TD align="left"> pKEX-2XL.Rep 78 </TD><TD align="left"> pCMV-VP(HS) </TD><TD align="left"> pKEX-VP1 </TD><TD align="left"> pKEX-VP2 </TD><TD align="left"> pKEX-VP3
</TD></TR><TR><TD align="left"><B>Date</B></TD><TD align="left"> 07.06.2010 </TD><TD align="left"> 07.06.2010 </TD><TD align="left"> 07.06.2010 </TD><TD align="left"> 07.06.2010 </TD><TD align="left"> 07.06.2010  </TD><TD align="left"> 07.06.2010  </TD><TD align="left"> 07.06.2010  </TD><TD align="left"> 07.06.2010  </TD><TD align="left"> 07.06.2010
</TD></TR><TR><TD align="left"><B>given number</B></TD><TD align="left"> B14 </TD><TD align="left"> B15 </TD><TD align="left"> B16 </TD><TD align="left"> B17 </TD><TD align="left"> B18 </TD><TD align="left"> B19 </TD><TD align="left"> B20 </TD><TD align="left"> B21 </TD><TD align="left"> B22
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 10</B></TD><TD align="left"><B>Clone 11</B></TD><TD align="left"><B>Clone 12</B></TD><TD align="left"><B>Clone 13</B></TD></TR><TR><TD align="left"><B>Bacteria strain</B></TD><TD align="left"> XL1B </TD><TD align="left"> XL1B </TD><TD align="left"> XL1B </TD><TD align="left"> XL1B
</TD></TR><TR><TD align="left"><B>Plasmidname</B></TD><TD align="left"> pTRUF_CMV_eGFP </TD><TD align="left"> dsAAV_CMV_eGFP </TD><TD align="left"> pTAV2</TD><TD align="left"> pDG
</TD></TR><TR><TD align="left"><B>Date</B></TD><TD align="left"> 07.06.2010  </TD><TD align="left"> 07.06.2010  </TD><TD align="left"> 07.06.2010  </TD><TD align="left"> 07.06.2010
</TD></TR><TR><TD align="left"><B>given number</B></TD><TD align="left"> B23 </TD><TD align="left"> B24 </TD><TD align="left"> B25 </TD><TD align="left"> B26
</TD></TR></TBODY></TABLE><P><U>Given Plasmid-Number</U></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD><TD align="left"><B>Clone 3</B></TD><TD align="left"><B>Clone 4</B></TD><TD align="left"><B>Clone 5</B></TD><TD align="left"><B>Clone 6</B></TD><TD align="left"><B>Clone 7</B></TD><TD align="left"><B>Clone 8</B></TD><TD align="left"><B>Clone 9</B></TD><TD align="left"><B>Clone 10</B></TD><TD align="left"><B>Clone 11</B></TD><TD align="left"><B>Clone 12</B></TD><TD align="left"><B>Clone 13</B></TD></TR><TR><TD align="left"><B>given number</B></TD><TD align="left"> P21 </TD><TD align="left"> P22 </TD><TD align="left"> P23 </TD><TD align="left"> P24 </TD><TD align="left"> P25 </TD><TD align="left"> P26 </TD><TD align="left"> P27 </TD><TD align="left"> P28 </TD><TD align="left"> P29 </TD><TD align="left">  P30 </TD><TD align="left"> P31 </TD><TD align="left"> P32 </TD><TD align="left"> P33
</TD></TR><TR><TD align="left"><B>measured concentration</B></TD><TD align="left">351,01 </TD><TD align="left"> 673,1 </TD><TD align="left"> 532,22 </TD><TD align="left"> 579,05 </TD><TD align="left"> 725,31 </TD><TD align="left"> 659,68</TD><TD align="left"> 692,8 </TD><TD align="left"> 545,46 </TD><TD align="left"> 568,34 </TD><TD align="left">  420,62 </TD><TD align="left"> 446,95 </TD><TD align="left"> 496,8 </TD><TD align="left"> 472,58
</TD></TR></TBODY></TABLE><P><B>Comments:</B> 
Many things went wrong today!
</P><UL><LI> Glycerol stocks must be vortexted!
</LI><LI> Check mini-prep buffers - especially buffer PE (needs to contain ethanol!)
</LI><LI> Always check volumes - try to estimate if volume makes sense (check pipettes!)
</LI><LI> Don't discard bacteria cultures until glycerol stocks and mini-preps are successfully done!!! 
</LI></UL><P style="font-size:15px; font-weight: bold; color: red;"><B>Today's conclusion: Better ask 2 times than do something wrong without asking!!!</B></P><H5><SPAN class="mw-headline" id="Site-directed_mutagenesis_of_pAAV_iGEM-MCS_.28PstI.29"><P style="font-size:15px; background-color:#66bbff;">Site-directed mutagenesis of pAAV_iGEM-MCS (PstI)</P></SPAN></H5><P><B>Quickchange site directed mutagenesis:</B>
PCR reaction:
</P><UL><LI> 2.5 µL 10x Pfu Ultra II buffer
</LI><LI> 0.5 µL template (therefore the a 1:20 dilution of the pAAV_iGEM-MCS (433 ng/µL) was prepared) = 10.825 ng
</LI><LI> 0.56 µL primer 1 (of 1:10 dilution)
</LI><LI> 0.56 µL primer 2 (of 1:10 dilution)
</LI><LI> 1 µL DMSO (primers form very strong secondary structures)
</LI><LI> 0.5 µL dNTP
</LI><LI> 18.88 µL dH<SUB>2</SUB>O
</LI><LI> 0.5 µL PfuUltra II fusion (1.25 U)
</LI></UL><P>-&gt; end volume: 25 µL
<B>PCR program:</B>
1 x : 2' 95°C (HotStart polymerase)
20 x : 30 s 95°C -&gt; 1' 55°C -&gt; 5' 68°C
1 x : 4°C (over night)

Experiment will be continued tomorrow.
</P><H4><SPAN class="mw-headline" id="24._Labortag_08.06.2010:"><P style="font-size:17px; background-color:#00dd77;">24. Labortag 08.06.2010:</P></SPAN></H4><H5><SPAN class="mw-headline" id="Cloning_of_mVenus_YFP_into_pAAV_iGEM-MCS.2C_continuation_of_site-directed_mutagenesis"><P style="font-size:15px; background-color:#66bbff;"> Cloning of mVenus_YFP into pAAV_iGEM-MCS, continuation of site-directed mutagenesis</P></SPAN></H5><P><B>Investigators: Kira, Jessy, Hanna, Achim</B></P><P style="font-size:15px; font-weight: bold; color: fuchsia;"><U>Analysis of iGEM-MCS sequence (RFC25 without EcoRI and NotI):</U></P><P>The alignment with the theoretical pAAV_iGEM-MS delivered a &quot;C-deletion&quot; within the NgoMIV restriction site. Due to that the pAAV_iGEM-MCS lacks this site. We controlled the bill of delivery and noted that the deletion must be GATC's fault.
All in all this doesn't matter, because parts which are in iGEM standard can be inserted and therefore they will deliver the lacking NgoMIV restriction site. Further on this could be an advantage because after digestion the fragment which is cut out can be better detected in the gel.
</P><P style="font-size:15px; font-weight: bold; color: fuchsia;"><U>Insertion of mVenus_YFP into pAAV_iGEM-MCS</U></P><P style="font-size:15px; font-weight: bold; color: blue;">Digestion:</P><LI>plasmid: insert: pGA14mVenusGeneart; number: - origin:Sven
</LI><LI>plasmid: vector: pAAV_iGEM-MCS; number: P32.2  production date:05.06.2010 origin: ____ 
</LI><LI>new vector name: pAAV_iGEM-MCS_mVenus </LI><LI>buffer used:4 ; Restriction-enzymes used: Enzyme AgeI (no. Lab:149); Enzyme XbaI (no. Lab: 63)
</LI><LI>DNA concentration (vector):433,78ng/µl ; DNA concentration (insert): 530 ng/µl<TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right"> V (pAAV_iGEM-MCS)/ µl </TD><TD align="right"> I(pGA14mVenus_Geneart) / µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 2,3</TD><TD align="right">3,8
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2</TD><TD align="right">2
</TD></TR><TR><TD> Buffer 3 (10x)</TD><TD align="right">2</TD><TD align="right">2
</TD></TR><TR><TD>Enzyme: AgeI (no.Lab:149)</TD><TD align="right">1,25</TD><TD align="right">1,25
</TD></TR><TR><TD>Enzyme: XbaI (no.Lab:63)</TD><TD align="right">0,75</TD><TD align="right">0,75
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">11,7</TD><TD align="right">10,2
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"><B>20</B></TD><TD align="right"><B>20</B></TD></TR></TBODY></TABLE></LI><LI> Incubation: 1 1/2 h at 37°C

0.5 g Agarose, 50 ml TAE (1%), 3 µL GELRED (3-6µl), at 115 Volt, running time: 
<TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (6x)/µl
</TH><TH>Expected size 1 (Geneious)
</TH><TH>Expected size 2 (Geneious)
</TH></TR><TR><TD>P34.2
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD>4617 bp
</TD><TD>22 bp
</TD></TR><TR><TD>pGA14mVenus
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD>2870 bp
</TD><TD>774 bp
</TD></TR></TBODY></TABLE></LI><LI>Gel loaded with vector and insert samples, 24 µl each &amp; 8 µl marker
</LI><LI>after 20 minutes, the insert band was cut out. Two overlapping bands were visible in the vector well, after 1 1/2 hours those bands were separated and both were cut out. <TABLE border="1"><TBODY><TR><TD> Sample</TD><TD align="right"> weight </TD><TD align="right"> concentration
</TD></TR><TR><TD> Vektor_Oben  </TD><TD align="right">0,32 g  </TD><TD align="right">29,8 ng/µl
</TD></TR><TR><TD> Vektor_Unten </TD><TD align="right">0,14 g  </TD><TD align="right">12,8 ng/µl
</TD></TR><TR><TD> Insert</TD><TD align="right">0,3 g  </TD><TD align="right">12,9 ng/µl
</TD></TR></TBODY></TABLE></LI><LI>the exact volume of insert and vector was calculated with LabTools:
</LI><LI>Ligation I with Vektor_Oben: 
</LI><LI>Vector: 4,16 µl
</LI><LI>Insert: 4,84 µl
</LI><LI>Ligation II with Vektor_Unten:
</LI><LI>Vector: 6 µl
</LI><LI>Insert: 3 µl
</LI><H5><SPAN class="mw-headline" id="Continuation_of_site-directed_mutagenesis"><P style="font-size:15px; background-color:#66bbff;">Continuation of site-directed mutagenesis</P></SPAN></H5><P style="font-size:15px; font-weight: bold; color: blue;">Digestion with DpnI:</P><LI>0.5 µl DpnI added
</LI><LI>incubated for 1 hour at 37°C

<P style="font-size:15px; font-weight: bold; color: blue;">Trafo:</P></LI><LI>Cells used for transformation: XL1B
</LI><LI>Centrifugation for 3 min at 8000 rpm instead of 6000 rpm (accidentaly)
</LI><LI>Plated on agar plate: pAAV_IGEM-MCS Mutagenese PST1 8.6.2010 AM XL1B
</LI><LI>incubated over night
<H4><SPAN class="mw-headline" id="25._Labortag_09.06.2010:"><P style="font-size:17px; background-color:#00dd77;">25. Labortag 09.06.2010:</P></SPAN></H4><H5><SPAN class="mw-headline" id="pAAV_iGEM-MCS_mVenus-YFP.2C_Site-directed_mutagenesis"><P style="font-size:15px; background-color:#66bbff;">pAAV_iGEM-MCS_mVenus-YFP, Site-directed mutagenesis</P></SPAN></H5><P><B>Investigators: Jessy, Achim, Sven, Toby, Hanna</B></P><P style="font-size:15px; font-weight: bold; color: red;"><B>No pAAV_iGEM-MCS_w/oPstI transformed bacteria (site-directed mutagensis) grew on the ampicillin agar plates over night! </B></P> Experiment is conducted again, guided by Toby (Thanks a lot!!!).<P style="font-size:15px; font-weight: bold; color: red;"><B>Further on the cloning of mVenus-YFP into pAAV_iGEM-MCS seemed to fail: </B></P> Only on the agar plate cotaining the bacteria transformed with the &quot;vector_oben&quot; ligation, which actually should be the &quot;wrong&quot; ligation (gelextraction of a band which contained fragments that were too large, see picture in lab journal), revealed colonies. Therefore this experiments is also conducted one more time, guided by Sven (thanks also a lot!!!)<P style="font-size:15px; font-weight: bold; color: fuchsia;"><U>Insertion of mVenus_YFP into pAAV_iGEM-MCS</U></P></LI><LI>experiment date: 9.6.2010
</LI><UL><LI>plasmid: 
</LI><LI>Vector: name: pAAV_iGEM-MCS number: 34.2 production date: 5.6.2010 
</LI><LI>Insert: name: pOG14_mVenus number: - production date: ____ origin: Sven :)
</LI></UL><LI>new vector name: pAAV_iGEM-MCS_mVenus-YFP
</LI><LI>buffer used: NEB4 ; Restriction-enzymes used: Enzyme 1 (no. Lab:___) AgeI ; Enzyme 2 (no.Lab:___) XbaI
</LI><LI>DNA concentration (vector): 433 ng/µL ; DNA concentration (insert): 530 ng/µL
</LI><P style="font-size:15px; font-weight: bold; color: blue;">Digestion</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl </TD><TD align="right">volume of insert /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right">2.83  </TD><TD align="right"> 2.77
</TD></TR><TR><TD> BSA (100x) </TD><TD align="right"> 0.5 </TD><TD align="right"> 0.5
</TD></TR><TR><TD> Buffer NEB4 (10x)</TD><TD align="right"> 3.0 </TD><TD align="right"> 3.0
</TD></TR><TR><TD>Enzyme 1 (AgeI)</TD><TD align="right"> 1.75 </TD><TD align="right"> 1.75
</TD></TR><TR><TD>Enzyme 2 (XbaI)</TD><TD align="right"> 1.25 </TD><TD align="right"> 1.25
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">20.73 </TD><TD align="right"> 20.67
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 30 </TD><TD align="right"> 30
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color: blue;">Agarose-Gel:</P><P>
0.5 g Agarose, 50 ml TAE (1 %), 3 µL GELRED (3-6µl), at 115 Volt
</P><P style="font-size:15px; font-weight: bold; color: blue;">Gelextraction</P><UL><LI> insert (mVenus-YFP): 26 ng/µL
</LI><LI> vector (pAAV_iGEM-MCS): 30 ng/µL 
</LI></UL><P style="font-size:15px; font-weight: bold; color: blue;">Ligation</P><UL><LI> vector: 5.85 µL
</LI><LI> insert: 3.15 µL
</LI><LI> ligase: 1 µL
</LI><LI> buffer: 10 µL
</LI></UL><P style="font-size:15px; font-weight: bold; color: fuchsia;"><U>Test digestion of pAAV_iGEM-MCS</U></P><P>p style=&quot;font-size:15px; font-weight: bold; color: blue;&quot;&gt;Test digestion&lt;/p&gt;</P><UL><LI>buffer used: 1 ; Restriction-enzymes used: Enzyme 1 (no. Lab:___) NgoMIV
</LI><LI>Plasmid
</LI><LI>Given Plasmid-Number: P34.2 ; DNA concentration: 433.78 ng/µL;
</LI><LI>Given Plasmid-Number: P34.3 ; DNA concentration: 408.48 ng/µL;
</LI><LI>Given Plasmid-Number: P34.4 ; DNA concentration: 409.8 ng/µL;
</LI></UL><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>P34.2 [µL] </B></TD><TD align="left"><B>P34.3 [µL] </B></TD><TD align="left"><B>P34.4 [µL] </B></TD></TR><TR><TD align="left"> DNA </TD><TD align="left"> 2.3 </TD><TD align="left"> 2.5 </TD><TD align="left"> 2.4
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 1 </TD><TD align="left"> 1 </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Buffer no. 1 (10x) </TD><TD align="left"> 1 </TD><TD align="left"> 1 </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Enzyme 1 (no. Lab: 113) ngoMIV </TD><TD align="left"> 1 </TD><TD align="left"> 1 </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 4.7 </TD><TD align="left"> 4.5 </TD><TD align="left"> 4.6
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>10</B></TD><TD align="left"><B>10</B></TD><TD align="left"><B>10</B></TD></TR></TBODY></TABLE><UL><LI>Incubation: 1 h, 37°C
</LI></UL><P style="font-size:15px; font-weight: bold; color: blue;">Agarose-Gel:</P><P>
0.5 g Agarose, 50 ml TAE (1 %), 3 µL GELRED, at 115 Volt, running time: 45 minutes + 20 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (6x)/µl
</TH><TH>Expected size 1 (Geneious)
</TH><TH>Expected size 2 (Geneious)
</TH></TR><TR><TD>P34.2
</TD><TD>10 µl
</TD><TD>2 µl
</TD><TD>3728 bp
</TD><TD>903 bp
</TD></TR><TR><TD>P34.3
</TD><TD>10 µl
</TD><TD>2 µl
</TD><TD>3782 bp
</TD><TD>903 bp
</TD></TR><TR><TD>P34.4
</TD><TD>10 µl
</TD><TD>2 µl
</TD><TD>3782 bp
</TD><TD>903 bp
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>Sample P34.2 /µl
</TH><TH>Sample P34.3 /µl
</TH><TH>Sample P34.4 /µl
</TH></TR><TR><TH>Lane
</TH><TD> 8 µL
</TD><TD> 10 µL
</TD><TD> 10 µL
</TD><TD> 10 µL
</TD></TR></TBODY></TABLE><P><B>Comments:</B> Digestion of P34.2 delivered just one fragment size, revealing that - as expected after sequencing (C-deletion: no NgoMIV site in iGEM-MCS) - there was just one NgoMIV restriction site in the vector. In contrast to that the digestion of P34.3 and P34.4 delivered two bands - whereas the band of the smaller fragments looked like a smier. Because of this obscure and non-satisfying results (besides our assumption that there's something wrong with the marker :) )we decided to sequence the P34.3 vector:
</P><UL><LI> c(P34.3): 408.48 ng/µL
</LI><LI> Volume (plasmid): 5.14 µL
</LI><LI> Volume (vector): 24.86 µL
</LI><LI> Name of eppi: HW_34
</LI><LI> Primer: GATC_std_pTeSp-1
</LI></UL><H5><SPAN class="mw-headline" id="Picking_of_Clones_from_pAAV_iGEM-MCS_mVenus-YFP_Vektor_oben"><P style="font-size:15px; background-color:#66bbff;">Picking of Clones from pAAV_iGEM-MCS_mVenus-YFP_Vektor_oben</P></SPAN></H5><P><B>Investigator: Hanna, Achim</B></P><UL><LI>experiment date: 9.6.2010
</LI><LI>plasmid: 
</LI><LI>Vector: name: pAAV_iGEM-MCS_Vektor_oben  
</LI><LI>Insert: name: pOG14_mVenus 
</LI></UL><LI>new vector name: pAAV_iGEM-MCS_mVenus-YFP_Vektor_oben
</LI><P>
The Clones were picked and incubated in 10 mL LB-Medium containing 10µL ampicillin over night at 37°C on a rotary shaker.
</P><H3><SPAN class="mw-headline" id="26._Labortag_10.06.2010:"><P style="font-size:17px; background-color:#00dd77;">26. Labortag 10.06.2010:</P></SPAN></H3><H4><SPAN class="mw-headline" id="Site-directed_mutagenesis_of_PstI_in_ITRs"><P style="font-size:15px; background-color:#66bbff;">Site-directed mutagenesis of PstI in ITRs</P></SPAN></H4><P style="font-size:12px; font-weight: bold; color: blue;">1. Site directed mutagenisis</P><UL><LI> Aim: deletion of PstI from both ITR's
</LI><LI> 2 PCR tubes got prepared, one for the left ITR and one for the right. (The SDM was performed according to the standart protocol)
</LI><LI><A href="/wiki/images/1/1e/Freiburg10_Site_directed_Mutagenesis_pAAV_MCS_deletion_PstI.pdf" class="internal" title="Freiburg10 Site directed Mutagenesis pAAV MCS deletion PstI.pdf">Media:Freiburg10_Site_directed_Mutagenesis_pAAV_MCS_deletion_PstI.pdf‎</A></LI></UL><H4><SPAN class="mw-headline" id="Mini-Prep_of_pAAV_iGEM_Venus_YFP_and_test_digestion"><P style="font-size:15px; background-color:#66bbff;">Mini-Prep of pAAV_iGEM_Venus_YFP and test digestion</P></SPAN></H4><P><B>Investigators: Jessy, Achim</B></P><P style="font-size:12px; font-weight: bold; color: blue;">2.1 Mini-Prep of pAAV_iGEM_mVenus_YFP</P><P>Title: <B>pAAV_iGEM_mVenus_</B><U>Glycerol Stocks</U></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD></TR><TR><TD align="left"><B>Bacteria strain</B></TD><TD align="left"> XL1 blue </TD><TD align="left"> XL1 blue
</TD></TR><TR><TD align="left"><B>Plasmidname</B></TD><TD align="left"> pAAV_iGEM_mVenus</TD><TD align="left"> pAAV_iGEM_mVenus
</TD></TR><TR><TD align="left"><B>Date</B></TD><TD align="left"> 10.06.2010 </TD><TD align="left"> 10.06.2010
</TD></TR><TR><TD align="left"><B>given number</B></TD><TD align="left"> B30 </TD><TD align="left"> B31
</TD></TR></TBODY></TABLE><P><U>Given Plasmid-Number</U></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD></TR><TR><TD align="left"><B>given number</B></TD><TD align="left"> P35 </TD><TD align="left"> P36
</TD></TR></TBODY></TABLE><P>Title: <B>pAAV_iGEM_mVenus_YFP</B> (guided by Sven) 
investigator: Bea
<U>Glycerol Stocks</U></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD><TD align="left"><B>Clone 3</B></TD><TD align="left"><B>Clone 4</B></TD></TR><TR><TD align="left"><B>Bacteria strain</B></TD><TD align="left"> BL-21 </TD><TD align="left"> BL-21 </TD><TD align="left"> BL-21 </TD><TD align="left"> BL-21
</TD></TR><TR><TD align="left"><B>Plasmidname</B></TD><TD align="left"> pAAV_iGEM_mVenus_YFP </TD><TD align="left"> pAAV_iGEM_mVenus_YFP </TD><TD align="left"> pAAV_iGEM_mVenus_YFP </TD><TD align="left"> pAAV_iGEM_mVenus_YFP
</TD></TR><TR><TD align="left"><B>Date</B></TD><TD align="left"> 10.06.2010 </TD><TD align="left"> 10.06.2010 </TD><TD align="left"> 10.06.2010 </TD><TD align="left"> 10.06.2010
</TD></TR><TR><TD align="left"><B>given number</B></TD><TD align="left"> B32 </TD><TD align="left"> B33 </TD><TD align="left"> B34 </TD><TD align="left"> B35
</TD></TR></TBODY></TABLE><P><U>Given Plasmid-Number</U></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Clone 1</B></TD><TD align="left"><B>Clone 2</B></TD><TD align="left"><B>Clone 3</B></TD><TD align="left"><B>Clone 4</B></TD></TR><TR><TD align="left"><B>given number</B></TD><TD align="left"> P37 </TD><TD align="left"> P38 </TD><TD align="left"> P39 </TD><TD align="left"> P40
</TD></TR></TBODY></TABLE><P style="font-size:12px; font-weight: bold; color: blue;">2.2 Test digestion</P><UL><LI>buffer used: 4 ; Restriction-enzymes used: Enzyme XbaI (no. Lab:___) ; Enzyme AgeI (no.Lab:___) ____
</LI><LI>Plasmid
</LI><LI>Given Plasmid-Number: P37; DNA concentration: 114,8 ng/µL ;
</LI><LI>Given Plasmid-Number: P38 ; DNA concentration: 179,8 ng/µL;
</LI><LI>Given Plasmid-Number: P39 ; DNA concentration: 180,8ng/µL ;
</LI><LI>Given Plasmid-Number: P40; DNA concentration: 189,2 ng/µL;
</LI></UL><P><B>Comments:</B> Clones were picked from trafo-plate in the morning and mini-prep was performed in the late afternoon. Due to this and the fact that bacterial strain was BL-21, DNA-concentrations are quite low.

Investigators: Bea, Chris W. </P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>Volume/µL</B></TD><TD align="left"><B>Mastermix</B></TD><TD align="left"><B>Sample: P37</B></TD><TD align="left"><B>Sample: P38</B></TD><TD align="left"><B>Sample: P39</B></TD><TD align="left"><B>Sample: P40</B></TD></TR><TR><TD align="left"> DNA </TD><TD align="left"> 800 ng </TD><TD align="left"> - </TD><TD align="left">7,0 µL</TD><TD align="left">4,5 µL</TD><TD align="left">4,4 µL</TD><TD align="left"> 4,2 µL
</TD></TR><TR><TD align="left"> BSA (10x) yes </TD><TD align="left"> 1,5 µL </TD><TD align="left"> 7,5 µL </TD><TD align="left">MM 4,5 µL</TD><TD align="left">MM 4,5 µL</TD><TD align="left">MM 4,5 µL</TD><TD align="left"> MM 4,5 µL
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 1.5 µL</TD><TD align="left"> 7,5 µL </TD><TD align="left">&quot;</TD><TD align="left">&quot;</TD><TD align="left">&quot;</TD><TD align="left"> &quot;
</TD></TR><TR><TD align="left"> Enzyme 1 (no. Lab:    ) AgeI </TD><TD align="left"> 0.75 µL </TD><TD align="left"> 3,75 µL </TD><TD align="left">&quot;</TD><TD align="left">&quot;</TD><TD align="left">&quot;</TD><TD align="left"> &quot;
</TD></TR><TR><TD align="left"> Enzyme 2 (no. Lab:    ) XbaI </TD><TD align="left"> 0.5 µL</TD><TD align="left"> 2,5 µL </TD><TD align="left">&quot;</TD><TD align="left">&quot;</TD><TD align="left">&quot;</TD><TD align="left"> &quot;
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> variable </TD><TD align="left"> - </TD><TD align="left">3,75 µL</TD><TD align="left">0,25 µL</TD><TD align="left">6,35 µL</TD><TD align="left"> 6,55 µL
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>15 µL</B></TD><TD align="left"> - </TD><TD align="left">15 µL</TD><TD align="left">15 µL</TD><TD align="left">15 µL</TD><TD align="left"> 15 µL
</TD></TR></TBODY></TABLE><UL><LI>Incubation: 45 min, 37°C
</LI></UL><H3><SPAN class="mw-headline" id="27._Labortag_11.06.2010:"><P style="font-size:17px; background-color:#00dd77;">27. Labortag 11.06.2010:</P></SPAN></H3><H4><SPAN class="mw-headline" id="Site_directed_mutagenisis_of_10.06.2010"><P style="font-size:15px; background-color:#66bbff;">Site directed mutagenisis of 10.06.2010</P></SPAN></H4><P><B>Transformation</B>
Transformation was performed according to the standard protocol <A href="/wiki/images/4/4e/Freiburg10_Cloning_Protocol.pdf" class="internal" title="Freiburg10 Cloning Protocol.pdf">Media:Freiburg10_Cloning Protocol.pdf</A>
Cells were plated on agar plates (2x 1:100; 2x pellet)containing ampicillin and stored over night at 37°C.
</P><P style="font-size:12px; font-weight: bold; color: blue;">Sequencing</P><P>p39 was send to GATC for sequenzing
</P><UL><LI> p39: 180,83 ng*µl^-1
</LI><LI> volume plasmid: 8,3 µl
</LI><LI> volume water: 21,7 µl
</LI><LI> primer: GATC_std_pTeSp-1
</LI></UL><H3><SPAN class="mw-headline" id="28._Labortag_12.06.2010:"><P style="font-size:17px; background-color:#00dd77;">28. Labortag 12.06.2010:</P></SPAN></H3><H4><SPAN class="mw-headline" id="Transformation_plates_stored_in_4.C2.B0C_room"><P style="font-size:15px; background-color:#66bbff;">Transformation plates stored in 4°C room</P></SPAN></H4><P><B>Investigator: Kira</B>
The plates were stored in the iGEM shelf @ 4 °C. Even if the transformation was not very successfull, some colonies can be picked and inoculated either tomorrow or on Monday.
<B>Note: Site-directed mutagenesis does not generate lots of intact plasmids, so few colonies are normal!! (Tobias)</B></P><H3><SPAN class="mw-headline" id="29._Labortag_13.06.2010:"><P style="font-size:17px; background-color:#00dd77;">29. Labortag 13.06.2010:</P></SPAN></H3><P><B>Investigators: Jessy, Patrick, Hanna</B>
We weren't able to detect any colonies on the plates - just a few air bubbles :) To be entirely sure, we put them back into the 37°C room over night.
</P><H3><SPAN class="mw-headline" id="30._Labortag_14.06.2010:"><P style="font-size:17px; background-color:#00dd77;">30. Labortag 14.06.2010:</P></SPAN></H3><P style="font-size:13px; font-weight: bold; color: fuchsia;"><U>pAAV_iGEM-mVenus-YFP:</U></P><P>The sequencing data of pAAV_iGEM_mVenus-YFP was analyzed. The alignment with the &quot;theoretical&quot; pAAV_iGEM_mVenus-YFP showed that we successfully inserted mVenus-YFP into pAAV_iGEM-MCS.
</P><H4><SPAN class="mw-headline" id="Site-directed_mutagenesis:"><P style="font-size:15px; background-color:#66bbff;"><U>Site-directed mutagenesis:</U></P></SPAN></H4><P>Also today no colonies were detectable on any plates!
</P><H4><SPAN class="mw-headline" id="TK-GMK-Plasmid"><P style="font-size:15px; background-color:#66bbff;"><U>TK-GMK-Plasmid</U></P></SPAN></H4><P><B>Investigators: Adrian, Hanna </B>
puB6_V5_His6_clone1 + TK/GMK (P15) will be sequenced again: </P><UL><LI> name: AF 1
</LI><LI> primer: GATC_std_BGH-reverse
</LI><LI> volume(plasmid) = 4.25 µL
</LI><LI> volume (H<SUB>2</SUB>O) = 25.75 µL
</LI></UL><P>Results (15.06.): Unfortunately just ~ 300 bp were sequenced. </P><P style="font-size:13px; font-weight: bold; color: purple;">To do: Therefore more primers have to be designed in order to sequence the whole ~2000 bp TK-GMK fusion construct! </P><H4><SPAN class="mw-headline" id="Cell_culture"><P style="font-size:15px; background-color:#66bbff;">Cell culture</P></SPAN></H4><P>AAV 293 and HT1080 Cells have been splitted and plated out on 10 cm dishes for transfektion.
The Cells were accounted by using the Neubauer-Meteringchamber.
After harvesting by following the standard protocol the pallet have been resuspendiated in 15ml of DTT medium.
2,5µl of this cell suspension have been mixed with 47.5µl of trypan blue.
</P><P>We counted 12,5x 10^6 cells/ml for the T293 AAV cell line and 10x10^6 cells /ml for the HT1080 cell line.
</P><P>The AAV 293 cells have been plated out on four dishes with 250µl of the cell suspension, on one plated with 1ml and on another dish
with 1.5ml.
note that that the date is wrong on the plates and the flasks! Cells have been already plated out on the 14th. of June.
</P><P>We also seated two flasks one for each cell line. Therefor we used 1ml of the HT1080 cell suspension and two ml of the the AAV293 cells.
</P><H3><SPAN class="mw-headline" id="31._Labortag_15.06.2010:"><P style="font-size:17px; background-color:#00dd77;">31. Labortag 15.06.2010:</P></SPAN></H3><H4><SPAN class="mw-headline" id="Solutions_for_transfection_were_prepared"><P style="font-size:15px; background-color:#66bbff;">Solutions for transfection were prepared</P></SPAN></H4><P><B>Investigators: Adrian, Achim, Chris W., Hanna</B></P><UL><LI> 1 x TE-Buffer: 1.2114 g TRIS, 0.2 mL EDTA, 81 mL milipore-H<SUB>2</SUB>O were mixed. pH was adjusted with HCl (1M and 5M) to 7.50. Volume was adjusted to 100 mL with milipore-H<SUB>2</SUB>O. Solution was autoclaved (sterilization by filtration is also possible). </LI><LI> 1 M CaCl2: 147.02 g CaCl2 x 2H<SUB>2</SUB>O (M = 147.02 g/mol) was solved in 1 Liter H<SUB>2</SUB>O and sterilized by filtration.</LI><LI> 2 x HBS: 0.027 g Na2HPO4, 1.636 g NaCl, 1.19155 g HEPES were dissolved in ~ 30 mL milipore-H<SUB>2</SUB>O. pH was adjusted to 7.10. Volume was adjusted to 100 mL and sterilized by filtration.</LI></UL><H3><SPAN class="mw-headline" id="32._Labortag_16.06.2010:"><P style="font-size:17px; background-color:#00dd77;">32. Labortag 16.06.2010:</P></SPAN></H3><H4><SPAN class="mw-headline" id="Sponsoring:"><P style="font-size:15px; background-color:#66bbff;">Sponsoring:</P></SPAN></H4><P><B>Investigators: Anna, Kerstin, Anissa</B>
Sponsoring letters were examined and modified one more time. </P><H4><SPAN class="mw-headline" id="Sequencing_of_TK-GMK:"><P style="font-size:15px; background-color:#66bbff;">Sequencing of TK-GMK:</P></SPAN></H4><P><B>Investigator: Hanna</B>
Two primers were designed and ordered from Sigma-Aldrich. Probably we will receive them on Friday.</P><H4><SPAN class="mw-headline" id="Mega_primer_PCR:"><P style="font-size:15px; background-color:#66bbff;">Mega primer PCR:</P></SPAN></H4><P><B>Investigators: Bea, Hanna </B></P><H3><SPAN class="mw-headline" id="33._Labortag_17.06.2010:"><P style="font-size:17px; background-color:#00dd77;">33. Labortag 17.06.2010:</P></SPAN></H3><H4><SPAN class="mw-headline" id="Primer_designing:_VP1_primer_for_pKEX_forward.2Freverse"><P style="font-size:15px; background-color:#66bbff;">Primer designing: VP1 primer for pKEX forward/reverse </P></SPAN></H4><P><B>Investigator: Volker</B>
The construnts that we receieved from PD Dr. Kleinschmidt (DKFZ) are mainly in the pKEX backbone. The sequences of the backbone and the inserts are not availible, for this reason we decided to sequence the constructs. The first step is to sequence from one construct into the backbone. In the second step we will designe primers that bind in the backbone and point into the direction of the insert.
These primers will then be used to sequence the inserts of all the constructs.</P><P><B>Forward primer pKEX in VP1 from bp 4124 to 414</B></P><UL><LI>VP1 primer for pKEX forward: 5' - CAACAAGTCTGTTAATGTGGAC - 3' 22bp; 		TM: 50°C; 	CG% 41%
</LI></UL><P><B>Reverse primer pKEX in VP1 from bp 2081 to 2100</B></P><UL><LI>VP1 primer for pKEX reverse: 5' - GTGGGCCAGGTTTGAGCTTC - 3' 20bp;		TM: 54°C	CG%: 60%
</LI></UL><P>The amplicon that should  be produced by the primers is 199 bp long.
</P><H4><SPAN class="mw-headline" id="Primer_designing:_CMV_forward.2Freverse_qPCR"><P style="font-size:15px; background-color:#66bbff;">Primer designing: CMV_forward/reverse_qPCR</P></SPAN></H4><P><B>Investigator: Volker</B>
Primers for the titering of the CMV region from the literature [Rohr et al., 2002] and [Rohr et al., 2005] were compared and analysed.
The sequences from 2002 showed a differce in one nucleotide compared to the other primer and to our sequence. For this reason the second set [Rohr et al., 2005] was ordered.</P><UL><LI>CMV_forward_qPCR: 5' - GGGACTTTCCTACTTGGCA - 3'
</LI><LI>CMV_reverse_qPCR: 5' - GGCGGAGTTGTTACGACA - 3'
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:688px;"><DIV class="thumbcaption">Obtaining fragments by digestion with AlwNI. Fragments contain left and right ITR respectively</DIV></DIV></DIV></DIV><P style="clear:both;">These primers will be used in a quantitative PCR assay with Sybr-green to measure the genomic titer and the infectious titer of the viral particles we will produce in the future.</P><H4><SPAN class="mw-headline" id="Ordering_of_required_reagents:"><P style="font-size:15px; background-color:#66bbff;">Ordering of required reagents: </P></SPAN></H4><P><B>Investigator: Volker</B></P><UL><LI>Nuclease S1 was ordered from Promega (10000Units for 36,00€)
</LI><LI>Proteinase K was ordered from Sigma  (5mg for 31,30€; BioUltra, ≥30 units/mg protein, lyophilized powder)
</LI></UL><P>These reagents are required for the procedure to determine the genomic and the infectious titer.</P><H3><SPAN class="mw-headline" id="34._Labortag_18.06.2010"><P style="font-size:17px; background-color:#00dd77;">34. Labortag 18.06.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Transfection"><P style="font-size:15px; background-color:#66bbff;">Transfection</P></SPAN></H4><P>Transfection via calcium phosphat with the prepared solutions (31. Labortag 15.06.2010)
</P><P><B>Investigators: Chris W., Hanna, Patrick, Adrian, Volker</B>
Transfections with the following plasmids:
</P><P>1) 
</P><UL><LI> pAAV_iGEM_mVenus_YFP (glycerol stock B34, P39) P39 has the correct sequence (confirmed)
</LI><LI> pHelper
</LI><LI> pAAV_RC
</LI></UL><P>2) 
</P><UL><LI> pAAV_iGEM_mVenus_YFP (glycerol stock B33, P38) P38 (we dont know if P39 has the correct sequence!)was used because the amount of P39 <I><A href="/wiki/index.php?title=Was_not_sufficient_enough&amp;action=edit&amp;redlink=1" class="new" title="Was not sufficient enough (page does not exist)">was not sufficient enough</A></I> for four Transfections!
</LI><LI> pHelper
</LI><LI> pAAV_RC
</LI></UL><P>Plasmid concentrations: 
pAAV_RC: 1 µg/µl 
pHelper: 280 ng/µl 
pAAV_iGEM_mVenus_YFP, P39: 180,83 ng/µl 
pAAV_iGEM_mVenus_YFP, P38: 179,75 ng/µl </P><P>Cellculture clone 1 and 2 were transfected with P39. Cellculture clone 3 and 4 were transfected with P38. </P><P>Deviations from the standard protocol (currently incomplete):
We could only pipet 3,3 µg (instead of 10 µg) from each of the 3 plasmids into the 15 ml falcons due to insufficient amount of plasmid.
</P><P>Adrian tried to examine the precipation of the CaCl2+ DNA Clusters. We have to optimize the pH of our 2xHBS at the moment it is 11.12
</P><H4><SPAN class="mw-headline" id="Theoretical:_Digestion_of_pAAV_MCS_for_PCR_to_design_ITR_BioBrick"><P style="font-size:15px; background-color:#66bbff;">Theoretical: Digestion of pAAV_MCS for PCR to design ITR BioBrick</P></SPAN></H4><P><B>Investigator: Bea</B><B>Idea:</B></P><UL><LI>Digest vector in order to obtain two fragments which contain the left and the right ITR respectively. </LI><LI>Separation (Agarose-gel) and gel extraction of fragments</LI><LI>Perform PCR with iGEM-Primers (forward primer contains EcoRI and Xbai; reverse primer contains SpeI and PstI)</LI><LI>to do:</LI><UL><LI> design primers
</LI><LI> digestion of pAAV_MCS etc.
</LI></UL></UL><P><B>Theoretical:</B> Digestion of pAAV_MCS with AlwNI produces two fragments which can be separated by agaorse gel.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Obtaining fragments by digestion with AlwNI. Fragments contain left and right ITR respectively</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="35._Labortag_19.06.2010:_Literature_seminar"><P style="font-size:17px; background-color:#00dd77;">35. Labortag 19.06.2010: Literature seminar</P></SPAN></H3><P><B>Investigators: Bea, Hanna, Melanie, Denis, Volker, Christian W., Adrian and Achim</B></P><P>Topics: 
</P><UL><LI>The ITRs
</LI><LI>The Cap
</LI></UL><H3><SPAN class="mw-headline" id="36.Labortag_20.06.2010:"><P style="font-size:17px; background-color:#00dd77;">36.Labortag 20.06.2010: </P></SPAN></H3><H4><SPAN class="mw-headline" id="Picking_clones_of_sequenced_pAAV_igEM_mVenus_YFP"><P style="font-size:15px; background-color:#66bbff;">Picking clones of sequenced pAAV_igEM_mVenus_YFP</P></SPAN></H4><P><B>Investigators: Bea and Chris W</B></P><UL><LI>Incoluate 250 mL  DYT medium (add 250µL ampiciliin) with B34 (glycerol stock: BL-21 E.coli cells with P39 plasmid: pAAV-iGEM_mVenus_YFP </LI><LI>Incubate on rotary shaker over-night (o/n) at 37°C </LI><LI>to do: perform Midi-Prep tomorrow (21.06.2010)</LI></UL><H3><SPAN class="mw-headline" id="37.Labortag_21.06.2010:"><P style="font-size:17px; background-color:#00dd77;">37.Labortag 21.06.2010: </P></SPAN></H3><H4><SPAN class="mw-headline" id="Second_Transfection.2C_Transduction"><P style="font-size:15px; background-color:#66bbff;">Second Transfection, Transduction</P></SPAN></H4><P><B>Investigators: Patrick, Johannes</B>
2x HBS was steril filtrated into two 50 ml flacons. The falcons were put into the cellculture fridge.
</P><P>New medium is prepared:
</P><P>DMEM-Medium
</P><UL><LI>500ml DMEM with Glutamax
</LI><LI>10% FCS (50 ml)
</LI><LI>5 ml 100x Pen/Strep
</LI><LI>5 ml 100x NaPyruvat 
</LI></UL><P>H-DMEM-Medium
</P><UL><LI>400 ml DMEM with Glutamax
</LI><LI>90 ml FCS
</LI><LI>5 ml 100x Pen/Strep
</LI><LI>5 ml 100x NaPyruvat
</LI></UL><H4><SPAN class="mw-headline" id="Feching_dry_ice_for_the_experiments"><P style="font-size:15px; background-color:#66bbff;">Feching dry ice for the experiments </P></SPAN></H4><P><B>Investigators:Volker</B></P><P>Dry ice can be fetched from the department for macromolecular chemistry. The chemical depot is located in the second basement. The dry ice can be retrieved from monday to friday from 11:00 to 11:30 and from 14:15 to 14:30. The requestforms for this institute can be found in the iGEM folder and should be filled out, signed by an advisor and taken to the chemical depot. A styrofoam container is required for the transport.</P><H4><SPAN class="mw-headline" id="Annotation_of_the_BRs_and_the_PLA2_domains"><P style="font-size:15px; background-color:#66bbff;">Annotation of the BRs and the PLA2 domains </P></SPAN></H4><P><B>Investigator:Volker</B>
The Bacis Regions which are important for the nuclear localisation were annotated in the protein sequences of VP 1-3 according to:
<A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=Freiburg10_Separate_basic_region_motifs_within_the_adeno-associated_virus_capsid_proteins_are_essential_for_infectivity_and_assembly.pdf" class="new" title="Freiburg10 Separate basic region motifs within the adeno-associated virus capsid proteins are essential for infectivity and assembly.pdf">[Grieger et al, 2006]</A></P><H4><SPAN class="mw-headline" id="Endotoxin-free_Midi-Prep_of_pAAV_iGEM_mVenus-YFP"><P style="font-size:15px; background-color:#66bbff;">Endotoxin-free Midi-Prep of pAAV_iGEM_mVenus-YFP </P></SPAN></H4><P><B>Investigators: Chris W., Hanna (guided by Sven)</B>
Midi-Prep was performed following standard protocol: <A href="/File:Freiburg10_Endotoxinfreie_Midi.pdf" title="File:Freiburg10 Endotoxinfreie Midi.pdf">File:Freiburg10 Endotoxinfreie Midi.pdf</A>
DNA concenration was measured: 507 ng/µL 
Plasmid was further used for a... </P><H4><SPAN class="mw-headline" id="Second_Transfection"><P style="font-size:15px; background-color:#66bbff;">Second Transfection</P></SPAN></H4><P><B>Investigators: Patrick, Johannes</B></P><P>Six transfections were performed according to the standart protocol: <A href="/wiki/images/8/8d/Freiburg10_Transfection_protocoll.pdf" class="internal" title="Freiburg10 Transfection protocoll.pdf">Media:Freiburg10_Transfection_protocoll.pdf</A>
Three 10 cm cellcluture dishes with 293 cells were transfected with 10 µg DNA (3,33 µg each plasmid) and the other three 10 cm cellculture dishes with 30 µg DNA (10 µg DNA each plasmid).
</P><P>Plasmids:
</P><UL><LI> pAAV_iGEM-MCS_mVenus (P41), 507 ng/µl
</LI><LI> pAAV_RC , 1000 ng/µl
</LI><LI> pHelper , 280 ng/µl
</LI></UL><H4><SPAN class="mw-headline" id="Preparing_Viral_Stocks"><P style="font-size:15px; background-color:#66bbff;">Preparing Viral Stocks</P></SPAN></H4><P><B>Investigators: Patrick, Johannes</B></P><P>According to the standard protocol (see AAV Helper-free System manual). No dilution was performed.
</P><H4><SPAN class="mw-headline" id="HT1080_Transduction"><P style="font-size:15px; background-color:#66bbff;">HT1080 Transduction</P></SPAN></H4><P><B>Investigators: Patrick, Johannes</B>
Four 10 cm cellculture dishes with HT1080 cells have been provided for the transduction.</P><UL><LI>1 ml of the AAV-2-mVenus (derived from P 39) was pipeted to into dish 1. Note: the virus solution (buffer) showed a yellow coloration.
</LI><LI>1 ml of the AAV-2-mVenus (derived from P 39) was pipeted to into dish 2. Note: the virus solution showed (buffer) an orange coloration.
</LI><LI>1 ml of the AAV-2-mVenus (derived from P 38) was pipeted to into dish 3. Note: the virus solution showed (buffer) an orange coloration.
</LI><LI>1 ml of the AAV-2-mVenus (derived from P 38) was pipeted to into dish 4. Note: the virus solution showed (buffer) an orange coloration.
</LI></UL><P>Apart from the dilution, the transduction was performed according to the standart protocol.
</P><H3><SPAN class="mw-headline" id="38.Labortag_22.06.2010:"><P style="font-size:17px; background-color:#00dd77;">38.Labortag 22.06.2010:</P></SPAN></H3><H4><SPAN class="mw-headline" id="Midiprep_of_pHelper_plasmid.3B_cloning_of_the_rfc25_MCS"><P style="font-size:15px; background-color:#66bbff;"> Midiprep of pHelper plasmid; cloning of the rfc25 MCS</P></SPAN></H4><P><B>Investigators: Bea, Chris W, Adrian, Achim</B></P><UL><LI>Hybridisation of o<B>ligos RFC25 EcoR1+BglII for &amp; RFC25 EcoR1+BglII rev</B></LI><LI>Programm ORIGAMI1 was modified for normal oligos:
</LI><LI>1  95°C  7'
</LI><LI>2  95°C  1'
</LI><LI>-1°C R=0,3°s
</LI><LI>Rep 70
</LI></UL><UL><LI>conc of Oligos: 138,3µg/µl
</LI></UL><P><B>Digestion</B></P><LI> plasmid: name: pAAV_MCS number: P9 production date: 01.06.2010 origin: Adrian, Hanna, Bea
</LI><LI> new vector name: pAAV_RFC25
</LI><LI> buffer used: 3 ; Restriction-enzymes used: Enzyme 1 (no. Lab:23) EcoRI ; Enzyme 2 (no.Lab:15) BglII
</LI><LI> DNA concentration (vector): 261,6 ng/µL


<TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of vector /µl </TD><TD align="right">volume of insert /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 5,8</TD><TD align="right"> none
</TD></TR><TR><TD> BSA (100x) </TD><TD align="right"> none </TD><TD align="right"> none
</TD></TR><TR><TD> Buffer 3 (10x)</TD><TD align="right">3 </TD><TD align="right"> none
</TD></TR><TR><TD>Enzyme EcoRI (no.Lab:23)</TD><TD align="right">1,5 </TD><TD align="right"> none
</TD></TR><TR><TD>Enzyme BglII (no.Lab:15)</TD><TD align="right">1,5</TD><TD align="right"> none
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right"> 18,2</TD><TD align="right"> none
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 30</TD><TD align="right"> none
</TD></TR></TBODY></TABLE></LI><LI> Incubation: 1,5 h


</LI><LI>A 1% agarose gel was prepared for plasmid separation
</LI><UL><LI> Production of antibiotika-stock-solutions (10 x 1ml AMP)
</LI></UL><UL><LI> Midi Prep of pAAV-Helper: concentration. 732,6 µg/µl
</LI></UL>



Ligation of Oligos: RFC25 and pAAV-MCS.

<UL><LI> dilution of our Oligos RFC from 138,3 µg/µl 1:10 13,83 (71bp long)=&gt; 0,64µl
</LI><LI> Vektor pAAV-MCS 23.3 µg/µl                                       =&gt; 8,36µl
</LI><LI> two plates are placed in the 37°C room
</LI></UL><UL><LI>Trafo has been performed following standard protocols
</LI></UL><H3><SPAN class="mw-headline" id="39.Labortag_23.06.2010:"><P style="font-size:17px; background-color:#00dd77;">39.Labortag 23.06.2010:</P></SPAN></H3><H4><SPAN class="mw-headline" id="Preparation_of_Viral_Stock.2C_Seeding_cells"><P style="font-size:15px; background-color:#66bbff;"> Preparation of Viral Stock, Seeding cells</P></SPAN></H4><H4><SPAN class="mw-headline" id="Design_of_Primer"><P style="font-size:15px; background-color:#66bbff;">Design of Primer</P></SPAN></H4><P><B>Investigators: Achim, Hanna</B></P><P>In order to sequence the hGH (polyadenylation) sequence and in general the GOI (gene of interest :) ) primers were designed: <A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=Primer_GOI%2BpolyA_pAAV_iGEM-MCS_mVenus_YFP.pdf" class="new" title="Primer GOI+polyA pAAV iGEM-MCS mVenus YFP.pdf">Media:Primer GOI+polyA pAAV iGEM-MCS mVenus YFP.pdf</A>
These primer will be used in order to sequence P38 (pAAV_iGEM_mVenus-YFP) which delivered fluorescencing HT1080 cells - in comparison to the already sequenced P39 clone. Further on they can be used in general for sequencing any GOI. 
These oligos were ordered at sigma aldrich as E@sy Oligos.

P38 was send for sequencing with GATC_std_pTESp-1 primer: 12 µL P39 (180 ng/µL) + 18 µL H<SUB>2</SUB>O
</P><H4><SPAN class="mw-headline" id="Sequencing_of_pKEX-VP1"><P style="font-size:15px; background-color:#66bbff;">Sequencing of pKEX-VP1</P></SPAN></H4><P><B>Investigators: Achim, Hanna, Adrian, Chris W.</B>
Primer: 
</P><UL><LI> VP1 primer for pKEX forward: V = 3 µL + 27 µL H<SUB>2</SUB>O
</LI><LI> VP1 primer for pKEX reverse: V = 3 µL + 27 µL H<SUB>2</SUB>O
</LI></UL><P>Vector:
</P><UL><LI> pKEX-VP1 (P27.2): c = 693 ng/µL, V = 3 µL + 27 µL H<SUB>2</SUB>O
</LI></UL><H4><SPAN class="mw-headline" id="Thaw_HT1080_and_AAV-293_cells"><P style="font-size:15px; background-color:#66bbff;">Thaw HT1080 and AAV-293 cells </P></SPAN></H4><P><B> Investigators: Achim, Hanna (guided by Sven)</B>
Thawing of HT1080 and AAV-293 cells was performed following this protocol: <A href="/wiki/images/2/2f/Freiburg10_Thawing_cells.pdf" class="internal" title="Freiburg10 Thawing cells.pdf">Media:Freiburg10 Thawing cells.pdf</A></P><H4><SPAN class="mw-headline" id="Picking_clones_of_pAAV_RFC25"><P style="font-size:15px; background-color:#66bbff;">Picking clones of pAAV_RFC25</P></SPAN></H4><P><B>Investigators: Achim, Chris W., Hanna</B>
Clones were picked, 250 mL DYT were inoculated.
Bacteria cultures are stored at 37°C over night.
</P><H4><SPAN class="mw-headline" id="Transduction_results"><P style="font-size:15px; background-color:#66bbff;"> Transduction results </P></SPAN></H4><P><B>Investigators: Adrian (guided by Sven)</B></P><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Results of transduction, ~ 1% of the HT1080 showed expression of mVenus.</CAPTION><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>Bright light image 1
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>Fluoresce microscopy image 1
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>Bright light image 1
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>Fluoresce microscopy image 1
</P></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Preparation_of_viral_stocks"><P style="font-size:15px; background-color:#66bbff;">Preparation of viral stocks</P></SPAN></H4><P><B>Investigator: Johannes, Bea</B></P><P>
Steril filtration of virus particles was performed (guided by Sven). Viral stocks are stored at -80°C.
</P><H3><SPAN class="mw-headline" id="40.Labortag_24.06.2010:"><P style="font-size:17px; background-color:#00dd77;">40.Labortag 24.06.2010:</P></SPAN></H3><H4><SPAN class="mw-headline" id="Labmeeting.2C_Presentation.2FUpdate_by_Patrick_and_Adrian"><P style="font-size:15px; background-color:#66bbff;"> Labmeeting, Presentation/Update by Patrick and Adrian</P></SPAN></H4><P>Delivery of:
</P><P>Proteinkinase K (will be used for the degradation of the viral capsid prior to qPCR)

Benzonase (will be used fur viral stock preparation)

Primer for for ITR-conversion to iGEM-standars
</P><UL><LI>Presentations of Adrian and Patrick: <A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=Adrian.ppt" class="new" title="Adrian.ppt">Media:Adrian.ppt</A><A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=Patrick.ppt" class="new" title="Patrick.ppt">Media:Patrick.ppt‎</A></LI></UL><H4><SPAN class="mw-headline" id="Midi-Prep_of_pAAV_iGEM_mVenus-YFP"><P style="font-size:15px; background-color:#66bbff;">Midi-Prep of pAAV_iGEM_mVenus-YFP</P></SPAN></H4><P><B>Investigators: Hanna, Chris W</B> (Adrian inserted RNAse and Ethanol into Buffers)
</P><UL><LI> new kit from Qiagen was used
</LI><LI> 35 ml of the over-night culture was filled into a 50 mL falcon and centrifuged at <B>5000 g</B> at <B>4°C</B> for <B>5 minutes</B>.
</LI><LI> supernatant was discarded
</LI><LI> Midi-prep was performed according to manual of Qiagen
</LI><LI> final concentrations: 380,4 µg/µl
</LI><LI> Name of eppi: <B>P44</B></LI></UL><H4><SPAN class="mw-headline" id="Mini_Prep_of_P42_and_P43_pAAV-RFC-25"><P style="font-size:15px; background-color:#66bbff;">Mini Prep of P42 and P43 pAAV-RFC-25</P></SPAN></H4><P><B>Investigator: Bea</B></P><UL><LI> Mini-Prep was performed following standard protocols
</LI><LI> conc: P42: 321,5 µg/µl
</LI><LI> conc: P43: 310,1 µg/µl
</LI></UL><H4><SPAN class="mw-headline" id="ITR_modification_via_PCR"><P style="font-size:15px; background-color:#66bbff;">ITR modification via PCR</P></SPAN></H4><UL><LI>Primers for ITR modification were resuspended and labeled: left forward: o24, left reverse: o25, right forward: o26, right reverse: o27
</LI></UL><H4><SPAN class="mw-headline" id="Work_on_pKEX_backbones"><P style="font-size:15px; background-color:#66bbff;">Work on pKEX backbones</P></SPAN></H4><P><B>Investigator: Volker</B></P><P>The sequencing data was evaluated and BLAST searches were carried out on NCBI to answer the question weather the sequence belongs to the backbone pKEX. This question could be answered positively. 
There are standard Primers from GATC that prime in the sequence. The three expression plasmids from PD Kleinschmidt will therefor be sequenced tomorrow.
Additionnaly nine primers for sequences in the AAV Rep and Cap genes were designed.
</P><H3><SPAN class="mw-headline" id="41.Labortag_25.06.2010:"><P style="font-size:17px; background-color:#00dd77;">41.Labortag 25.06.2010:</P></SPAN></H3><H4><SPAN class="mw-headline" id="PCR_of_ITRs.2C_Splitting_HT1080_and_AAV-293_cells.2C_Seeding_of_HT1080_cells.2C_Creating_Virus_Stock_Economics.2C_Medium_check"><P style="font-size:15px; background-color:#66bbff;">PCR of ITRs, Splitting HT1080 and AAV-293 cells, Seeding of HT1080 cells, Creating Virus Stock Economics, Medium check</P></SPAN></H4><P style="font-size:15px; font-weight: bold; color: fuchsia;">Preparation for PCR of ITRs</P><P><B>Investigator: Bea</B></P><P><B> Digestion of pAAV_MCS (P9)</B></P><UL><LI> experiment date: 25.06.2010
</LI><LI> plasmid: name: pAAV_MCS number: P9 production date: 08.05.2010 origin: SH
</LI><LI> buffer used: 4 ; Restriction-enzymes used: Enzyme 1 (no. Lab:113) NgoMIV ; Enzyme 2 (no.Lab:120) AlwNI
</LI><LI> DNA concentration (vector): 267,9 ng/µL
</LI></UL><P>
In order to receive the best PCR-results, two digestion reactions/approaches have been performed.  
The first digestion was cut only with AlwNI, the second digestion was cut with AlwNI and NgomIV.</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">v(pAAV_MCS 1) /µl </TD><TD align="right">v(pAAV_MCS 2) /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 4</TD><TD align="right"> 4
</TD></TR><TR><TD> BSA (100x) </TD><TD align="right">none </TD><TD align="right"> none
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right"> 2</TD><TD align="right"> 2
</TD></TR><TR><TD>Enzyme AlwNI (no.Lab: 120)</TD><TD align="right">1 </TD><TD align="right"> 1
</TD></TR><TR><TD>Enzyme NgoMIV (no.Lab:113)</TD><TD align="right"> none </TD><TD align="right"> 1
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right"> 13,0</TD><TD align="right"> 12,0
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"><B>20</B></TD><TD align="right"><B>20</B></TD></TR></TBODY></TABLE><LI> Incubation:1,5h
</LI><LI> 1% agarose gel has been prepared. Running the gel at 110 V for 45 minutes. 
</LI><LI> Loading plan: Marker (8 µL), pAAV_MCS 1 (24 µL), paav_MCS 2 (24 µL)
<H4><SPAN class="mw-headline" id="Splitting_of_HT1080_and_AAV-293_cells.2C_Seeding_of_HT1080_cells"><P style="font-size:15px; background-color:#66bbff;">Splitting of HT1080 and AAV-293 cells, Seeding of HT1080 cells</P></SPAN></H4><P><B>Investigator: Hanna </B></P><P><B>HT1080:</B></P></LI><LI> Cells showed 80 % confluence
</LI><LI> Cells were washed with PBS, detached with Trypsine
</LI><LI> 10 mL DMEM was added, cell suspension was centrifuged for <B>5 minutes</B> at 200 g.
</LI><LI> Supernatant was discarded, pellet was resuspended with 8 mL DMEM.
</LI><LI><B>Cells were counted: 47.5  µL Trypan blue (stains dead cells) and 2.5 µL cell suspension were mixed and applied onto a Neubauer counting chamber</B> -&gt; 2.2 x 20 x 10^-4 cells per mL
</LI><LI> Therefore 27 µL of the cellsuspension was added to each 3 mL DMEM in the 6-well plates. <B>-&gt; 2 6-well plates were prepared for transduction!</B></LI><LI> Further on cells were splitted: 110 µL cell suspension was pipetted into 2 75T flasks each containing 20 mL DMEM and stored in the 37°C incubator.
</LI><P><B>AAV-293 cells:</B></P><UL><LI> Cells showed ~ 50% confluence 
</LI><LI> Cells were splitted following the standard protocol.
</LI></UL><H4><SPAN class="mw-headline" id="Creating_Virus_Stock_Economics"><P style="font-size:15px; background-color:#66bbff;">Creating Virus Stock Economics</P></SPAN></H4><P><B>Investigator: Adrian</B></P><H4><SPAN class="mw-headline" id="Medium_check"><P style="font-size:15px; background-color:#66bbff;">Medium check</P></SPAN></H4><P><B>Investigator: Adrian</B></P><UL><LI>three little culture bottles got prepared (H-DMEM, DMEM and PBS) and set into the incubator. In the following days they'll get checked for bacerial growth.</LI></UL><H4><SPAN class="mw-headline" id="Sequencing_results_of_pAAV_RFC25"><P style="font-size:15px; background-color:#66bbff;">Sequencing results of pAAV_RFC25</P></SPAN></H4><P><B>Investigators: Adrian, Bea, Hanna</B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">SpeI restriction site deletion in sequenced data (on bottom).</DIV></DIV></DIV></DIV><P style="clear:both;"><B>sequenced sample</B>: pAAV_RFC25 clone 2 (P43)</P><P><B>used primer</B>: GATC_std_pTeSp-1.ab1 
The alignment with the theoretical construct showed, that there's a exchange of one base in the RFC25-multiple cloning site. This substitution leads to the deletion of the SpeI restriction site. 
Therefore we checked the order sheet (everything OK!).
We sent the second clone (P42) for sequencing (Hanna).
</P><UL><LI> V(Plasmid) = 6.5 µL
</LI><LI> V(H<SUB>2</SUB>O) = 23.5 µL
</LI><LI> Primer: pTESP-1
</LI></UL><H4><SPAN class="mw-headline" id="Annotation_and_studies_of_the_AAV_structure"><P style="font-size:15px; background-color:#66bbff;">Annotation and studies of the AAV structure</P></SPAN></H4><P><B>Investigator: Volker</B></P><P>The NGR- and the HPSG motif were annotated in the gene sequences according to <A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=Freiburg10_Michelfelder_Trepel_2009_AAV_and_Their_Redirection_to_Cell-Type_Specific_Receptors.pdf" class="new" title="Freiburg10 Michelfelder Trepel 2009 AAV and Their Redirection to Cell-Type Specific Receptors.pdf">[Michelfelder &amp; Trepe; 2009]</A>
The pdb sequence were visualized with the programm <A href="http://www.pymol.org|" class="external text" rel="nofollow">PyMOL</A> and the important motifs were highlighted.
</P><P><B>Impressions from PyMOL</B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:341px;"><DIV class="thumbcaption">The viral structure of AAV-2.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Sturcture of a single AAV-2 VP2 monomer</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">B-Factor indicating mobility in the crystal</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">HPSG-binding-motif</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:552px;"><DIV class="thumbcaption">Residues of the HPSG-binding-motif</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">The NGR-motif that is responsible for the integrin a5β1 binding</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="42.Labortag_26.06.2010"><P style="font-size:17px; background-color:#00dd77;">42.Labortag 26.06.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Transduction_of_HT1080_cells"><P style="font-size:15px; background-color:#66bbff;">Transduction of HT1080 cells</P></SPAN></H4><P><B>Investigator: Adrian</B></P><UL><LI> Transduction of 2x6 wells was successfully done (14.25)
</LI><LI> one six well is transduced with 10µg the other with 30µg DNA amount (see HEK293 infection protokoll for further information)</LI></UL><P>(A is up)
</P><TABLE border="1"><TBODY><TR><TD> stock solution  </TD><TD align="right">1:10 Dilution </TD><TD align="right">1:50 Dilution
</TD></TR><TR><TD> 1:100 Dilution  </TD><TD align="right"> 1:500 Dilution</TD><TD align="right"> control (500µl DMEM)
</TD></TR></TBODY></TABLE><UL><LI>the delution steps (1:10, 1:50, 1:100, 1:500) are placed upon the virus stocks in the -80°C freezer in 15 ml falcons.
</LI></UL><P style="font-size:15px; font-weight: bold; color: fuchsia;"><U>Did primers for sequencing GOI arrive?</U></P><H4><SPAN class="mw-headline" id="Paper_reading_session"><P style="font-size:15px; background-color:#66bbff;">Paper reading session</P></SPAN></H4><P><B>Investigators: Igor, Anna, Patrick, Stefan and Volker</B>
Several papers were read and infomation on cap-modifications were extracted.
</P><UL><LI>Trasitions of seven Tyrosines (252, 272, 444, 500, 700, 704 and 730) to Phenylalanines as described in <A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=Freibur10_Next_generation_of_adeno-associated_virus_2_vectors-_point_mutations_in_tyrosines_lead_to_high-efficiency_transduction_at_lower_doses.pdf" class="new" title="Freibur10 Next generation of adeno-associated virus 2 vectors- point mutations in tyrosines lead to high-efficiency transduction at lower doses.pdf">[Zhong et al. 2008]</A></LI><LI>Transitions of two amino acids (R459D and N551D) leading to reduced sero prevalence as described in <A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=Designer_Gene_Delivery_Vectors-_Molecular_Engineering_and_Evolution_of_Adeno-Associated_Viral_Vectors_for_Enhanced_Gene_Transfer.pdf" class="new" title="Designer Gene Delivery Vectors- Molecular Engineering and Evolution of Adeno-Associated Viral Vectors for Enhanced Gene Transfer.pdf">[Inchan Kwonand, David V. Schaffer]‎</A></LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption">Proposed transitions Y252F, Y272F, Y444F, Y500F, Y700F, Y704F and Y730F in red &amp; R459D and N551D in green</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="43.Labortag_28.06.2010:"><P style="font-size:17px; background-color:#00dd77;">43.Labortag 28.06.2010: </P></SPAN></H3><H4><SPAN class="mw-headline" id="Checking_HT1080_for_YFP-Expression.2C_Creating_plan:_urgent_things_to_do"><P style="font-size:15px; background-color:#66bbff;">Checking HT1080 for YFP-Expression, Creating plan: urgent things to do</P></SPAN></H4><P><B>Investigator: Sven, Patrick, Adrian</B></P><P style="font-size:15px; font-weight: bold; color: fuchsia;"><U>Checking HT1080 for YFP-Expression </U></P><UL><LI>Hannas 2*6 wells (24.6) transduced by (Adrian 25.6) 
</LI></UL><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Results of transduction nr.2</CAPTION><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>10µg_unverd_2_(c1).JPG 48h post transduction
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>10µg_unverd_1_(c1).JPG 48h post transduction
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>30µg_unverd_2_(c1).JPG 48h post transduction
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>30µg_unverd_(c1).JPG 48h post transduction
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>30µg_unverd_4_(c1).JPG 48h post transduction
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>30µg unverd 3 (c1).JPG 48h post transduction
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>30µg_unverd_6_(c1).JPG 48h post transduction
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>30µg_unverd_5_(c1).JPG 48h post transduction
</P></DIV></DIV></TD></TR></TBODY></TABLE><UL><LI> The transduced cells will be checked again (29.6.).
</LI></UL><H4><SPAN class="mw-headline" id="Sequencing_results_of_P42"><P style="font-size:15px; background-color:#66bbff;">Sequencing results of P42</P></SPAN></H4><P><B>Investigator: Hanna</B></P><P>Sequencing of pAAV_RFC25 (P42): 
</P><P>Sequencing delivered, that there weren't any base exchanges or deletions in this clone. Therefore P43 was dismissed (Plasmid and glycerol stock). </P><H4><SPAN class="mw-headline" id="Theoretical_construction_of_our_biobricks"><P style="font-size:15px; background-color:#66bbff;">Theoretical construction of our biobricks</P></SPAN></H4><P><B>Investigators: Hanna, Adrian, Chris W, Patrick, Volker</B>
Theoretical cloning
</P><H4><SPAN class="mw-headline" id="Urgent_Things_to_do:"><P style="font-size:15px; background-color:#66bbff;">Urgent Things to do:</P></SPAN></H4><UL><LI>Modification CAP:
</LI><LI>HSPG -&gt; 2 AS
</LI><LI>Targeting (585,588...) -&gt; 2 AS -&gt; 1 or 2 restriction sites?! Investigator: Volker
</LI><LI> Seroprevalence -&gt; 2 AS
</LI><LI> thyrosine for ubiquitination Y252F, Y272F, Y444F, Y500F, Y700F, Y704F and Y730F 
</LI><LI>Primer: start codons
</LI><LI>EGF-Rezeptor: A431-Zellinie
</LI><LI>Affibody: order the Gensequence
</LI><LI>bispecific AK -&gt; order Diabodys (are there any restriction sites?) Investigator: kristian???
</LI></UL><UL><LI>Ordering the ITRs <B>Investigator: Hanna</B></LI><LI>Stratagene: right and left
</LI><LI>wild-type 
</LI></UL><UL><LI>Modification: REP <B>Investigator: Volker</B></LI><LI> 3 restriction sites -&gt; site directed mutagenisis vs. synthetic? -&gt;order (check which is the cheaper approach)
</LI></UL><UL><LI>HGH <B>Investigator: Hanna</B></LI><LI> restriction sites? -&gt; are there any available biobricks?
</LI><LI>biobrick production
</LI></UL><UL><LI>Beta-globin: what is the function? -&gt; cut out?
</LI><LI>where is the start and the end?
</LI><LI>restriction sites?
</LI><LI>biobrick production
</LI></UL><UL><LI>GMK-TK
</LI><LI>restriction sites: primer
</LI><LI>biobrick production
</LI><LI>ordering the SR39
</LI></UL><UL><LI>tumor specific promotors
</LI><LI>more Information!!!
</LI><LI> telomerase specific promoters!!! 
</LI></UL><H4><SPAN class="mw-headline" id="Continuation_of_ITR_PCR"><P style="font-size:15px; background-color:#66bbff;">Continuation of ITR PCR</P></SPAN></H4><P><B>Investigators: Hanna, Chris W., Adrian</B></P><UL><LI> Analytic gel was run: 1.7 µL loading dye (6x) was added to 8.3 µL of each sample (left ITR: 5 µL DMSO + 10 µL DMSO; right ITR: 5 µL DMSO + 10 µL DMSO). 
</LI><LI> 110 V, 35 minutes
</LI><LI> No ITr bands were detectable - just 2 weak &quot;primer-bands&quot;
</LI></UL><P>Conclusion: PCR didn't work. Because of that new primers were designed and ordered (Achim), which are longer and posses therefore higher annealing temperatures.
</P><H4><SPAN class="mw-headline" id="ITRs_.28theoretical.29"><P style="font-size:15px; background-color:#66bbff;">ITRs (theoretical)</P></SPAN></H4><P><B>Investigators:Adrian, Achim, Hanna</B></P><UL><LI> trs sequences (GTTGG) are present in both ITRs (Stratagene)!
</LI><LI> assumption: transcription despite of ITR-secondary structure - no single-strand nick at trs-sequence in transduced cells ?
</LI></UL><H3><SPAN class="mw-headline" id="44.Labortag_29.06.2010:"><P style="font-size:17px; background-color:#00dd77;">44.Labortag 29.06.2010:</P></SPAN></H3><H4><SPAN class="mw-headline" id="Discussionround"><P style="font-size:15px; background-color:#66bbff;"> Discussionround </P></SPAN></H4><UL><LI>Affibodys 
</LI><LI><B>Investigator: Anna</B></LI><LI>Affibodys are 6-7kD Proteins derived from Protein A/Z
</LI><LI>they are binding to domain 3 from IGFR
</LI><LI>13 AS are replaceable for changing the tropism
</LI></UL><UL><LI>Tyrosine Mutants
</LI><LI><B>Investigator: Adrian</B></LI><LI>replacements are done in VP3! =&gt; Volker?!
</LI><LI>no multi-mutants are available (instead of single mutations many Y-&gt;F)
</LI><LI>best candidates are: Y730F and Y444F.
</LI></UL><UL><LI>Second strand DNA-synthesis in transduced cells
</LI><LI><B>Investigator: Achim</B></LI><LI> Second strand synthesis is the main limiting for transduction/transgen expression.
</LI></UL><H4><SPAN class="mw-headline" id="Seeding_HT-cells_for_transfection_and_titering"><P style="font-size:15px; background-color:#66bbff;">Seeding HT-cells for transfection and titering</P></SPAN></H4><P><B>Investigators: Sven, Adrian, Bea</B></P><UL><LI>The pellet was resolved in 4ml we had a 550.000 cells/ml
</LI><LI> 2x6 well dishes got prepared:
</LI></UL><P>1. Plate I(A is up)
</P><TABLE border="1"><TBODY><TR><TD> 3x10^5 cells </TD><TD align="right">3x10^5 cells </TD><TD align="right">3x10^5 cells
</TD></TR><TR><TD> 3x10^5 cells  </TD><TD align="right"> 3x10^5 cells</TD><TD align="right"> 3x10^5 cells
</TD></TR></TBODY></TABLE><P>2. Plate II (A is up)
</P><TABLE border="1"><TBODY><TR><TD>  about 3x10^5 cells  </TD><TD align="right">only medium</TD><TD align="right">only medium
</TD></TR><TR><TD> about 3x10^5 cells  </TD><TD align="right">only medium</TD><TD align="right">only medium
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Seeding_HT1080-cells_for_transduction_and_qPCR"><P style="font-size:15px; background-color:#66bbff;">Seeding HT1080-cells for transduction and qPCR</P></SPAN></H4><P><B>Investigator: Hanna</B></P><UL><LI> Cell density was determined with trypan blue and counting chamber: 2 x 20 x 10^4 cells/mL
</LI><LI> wanted: 5 x 10^4 cells --&gt; 125 µL cell suspension per well
</LI><LI> Cells were seeded into a 24-well-plate (row C and D!): 1.5 mL DMEM + 125 µL cell suspension per well
</LI><LI> cells were incubated for 1 hour at 37°C
</LI></UL><P>
Further on, cells were splitted: ~ 3 mL cell suspension was added to ~ 17 mL DMEM
</P><H4><SPAN class="mw-headline" id="Transduction_of_HT1080_cells_for_qPCR"><P style="font-size:15px; background-color:#66bbff;">Transduction of HT1080 cells for qPCR</P></SPAN></H4><P><B>Investigator: Chris W. </B></P><H3><SPAN class="mw-headline" id="45.Labortag_30.06.2010:"><P style="font-size:17px; background-color:#00dd77;">45.Labortag 30.06.2010:</P></SPAN></H3><H4><SPAN class="mw-headline" id="Titering_Procedure_via_qPCR.2C_Transduction_of_6well_plates_no.4"><P style="font-size:15px; background-color:#66bbff;"> Titering Procedure via qPCR, Transduction of 6well plates no.4</P></SPAN></H4><P style="font-size:15px; font-weight: bold; color: fuchsia;"><U>Harvesting cells for qPCR</U></P><P><B>Investigators: Chris W, Hanna</B>
Transduced HT1080 cells were harvested following Sven's standard protocol.
</P><P style="font-size:15px; font-weight: bold; color: blue;">Transduction of 2 x 6well plates nr.4</P><P><B>Investigators: Bea, Adrian</B></P><UL><LI>Master plan: Different Transduction Mechanisms
</LI><LI>rise the amount of transduced particels, instead of 500µl, 1000µl of the AAV-stock getting pipetted into each well.
</LI><LI>the second plate gets stimulated via UV-light to boost the expression of DNA-Repair mechanisms. This should enhance the second strand synthesis of our ssDNA-GOI =&gt; a rise of YFP Expression!
</LI></UL><P>1. Plate I(A is up)
</P><TABLE border="1"><TBODY><TR><TD> 3x10^5 cells  </TD><TD align="right">3x10^5 cells </TD><TD align="right">3x10^5 cells
</TD></TR><TR><TD> 3x10^5 cells  </TD><TD align="right"> 3x10^5 cells</TD><TD align="right"> 3x10^5 cells
</TD></TR></TBODY></TABLE><P>Transduction:
</P><TABLE border="1"><TBODY><TR><TD> 1000 µl of Viral Stock  DROPPED </TD><TD align="right">1000 µl of Viral Stock gently resuspended</TD><TD align="right">1500 µl of Viral Stock hard resuspending
</TD></TR><TR><TD> 1000 µl of Viral Stock  DROPPED </TD><TD align="right"> 1000 µl of Viral Stock gently resuspended</TD><TD align="right"> no Transduction
</TD></TR></TBODY></TABLE><P>2. Plate II (A is up)
</P><TABLE border="1"><TBODY><TR><TD>  about 3x10^5 cells  </TD><TD align="right">only medium</TD><TD align="right">only medium
</TD></TR><TR><TD> about 3x10^5 cells  </TD><TD align="right">only medium</TD><TD align="right">only medium
</TD></TR></TBODY></TABLE><P>
Transduction:
</P><TABLE border="1"><TBODY><TR><TD>  1500 µl of Viral Stock gently resuspended  </TD><TD align="right">no Transduction</TD><TD align="right">no Transduction
</TD></TR><TR><TD>  1500 µl of Viral Stock gently resuspended  </TD><TD align="right">no Transduction</TD><TD align="right">no Transduction
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="beta-globin_intron"><P style="font-size:15px; background-color:#66bbff;">beta-globin intron</P></SPAN></H4><P><B>Investigators: Adrian, Bea</B>
We found some interesting literature which points out that introns have an influence on the eucaryotic gene expression on DNA and RNA level. Next steps are going to be the designing of the beta-globin intron and maybe further analysis of the other mentioned introns. Maybe two or three more introns can be ordered.
<A href="/wiki/images/1/10/Freiburg10_LeHir_et_al_How_introns_influence_and_enhance_eukaryotic_gene_expression_2003.pdf" class="internal" title="Freiburg10 LeHir et al How introns influence and enhance eukaryotic gene expression 2003.pdf">Media:Freiburg10_LeHir_et_al_How_introns_influence_and_enhance_eukaryotic_gene_expression_2003.pdf</A><A href="/wiki/images/a/a1/Freiburg10_Noe_et_al_An_intron_is_required_dro_dihydrofolate_reductase_protein_stability_2008.pdf" class="internal" title="Freiburg10 Noe et al An intron is required dro dihydrofolate reductase protein stability 2008.pdf">Media:Freiburg10_Noe_et_al_An_intron_is_required_dro_dihydrofolate_reductase_protein_stability_2008.pdf</A><A href="/wiki/images/6/6d/Freiburg10_Mashadrizeh_et_al_A_systematic_study_of_the_function_of_the_human_beta-globin_introns_on_the_expression_of_the_human_coagulation_factor_in_cultured_Chinese_hamster_ovary_cells_2009.pdf" class="internal" title="Freiburg10 Mashadrizeh et al A systematic study of the function of the human beta-globin introns on the expression of the human coagulation factor in cultured Chinese hamster ovary cells 2009.pdf">Media:Freiburg10_Mashadrizeh_et_al_A_systematic_study_of_the_function_of_the_human_beta-globin_introns_on_the_expression_of_the_human_coagulation_factor_in_cultured_Chinese_hamster_ovary_cells_2009.pdf‎</A></P><UL><LI><A href="/wiki/images/2/29/Chicken_beta-globin_insulator_overcomes_variegation_of_transgenes_in_Xenopus_embryos.pdf" class="internal" title="Chicken beta-globin insulator overcomes variegation of transgenes in Xenopus embryos.pdf">Media:Chicken_beta-globin_insulator_overcomes_variegation_of_transgenes_in_Xenopus_embryos.pdf‎</A></LI></UL><H4><SPAN class="mw-headline" id="hGH_sequence"><P style="font-size:15px; background-color:#66bbff;">hGH sequence</P></SPAN></H4><P><B>Investigator: Hanna</B></P><UL><LI> orientation in pAAV_MCS is reverse annotated, but in the same orientation as in human growth hormone I gene!
</LI><LI> Blast and alignment with other expression vectors and the human growth hormone I revealed that Stratagene annotated 1 bp too much
</LI><LI> &quot;A common feature of mRNA in higher eukaryotes (but not in yeast) is the presence of the highly conserved sequence AAUAAA in the region from 11-30 nucleotides upstream of the site of poly(A) addition. (...) The signal is needed for both cleavage and polyadenylation.&quot; (Genes VIII, Lewin, P. 720)
</LI><LI> good news: There're no iGEM restriction sites in the sequence (RFC25)
</LI><LI> Conclusion: Instead of ordering the whole sequenc in iGEM standard it would be probably easier (and cheaper) to perform a PCR.
</LI><LI><B>Done</B>: Oligos were designed for PCR!
</LI></UL><H4><SPAN class="mw-headline" id="ITRs:_new_strategy"><P style="font-size:15px; background-color:#66bbff;">ITRs: new strategy</P></SPAN></H4><P><B>Investigator: Hanna</B>
Plan:
</P><UL><LI> Digest pAAV_MCS with AlwNI. Result: one large fragment containing the left ITR + a small fragment containing right ITR.
</LI><LI> Digest both fragments with NotI and PstI.
</LI><LI> Hybridize oligos containing RFC10 restriction sites (designed by Hanna), ligate them via intermediate steps (ligating into a vector) with ITRs . Comment: One oligo posseses a blunt end - therefore &quot;intermediate step-vector&quot; needs to be digested with an enzyme producing a blunt site.
</LI><LI><B> Done:</B> Oligos were designed. Need to be checked!
</LI></UL><H4><SPAN class="mw-headline" id="Sequencing_of_pAAV_iGEM-mVenus-YFP"><P style="font-size:15px; background-color:#66bbff;">Sequencing of pAAV_iGEM-mVenus-YFP</P></SPAN></H4><P><B>Investigators: Hanna, Chris W</B>. </P><UL><LI> 3 µL of O28, 29 and 30 each were mixed with 27 µL water -&gt; primer fpr sequencing of GOI and of hGH sequence
</LI><LI> 11.1 µL plasmid + 18.9 µL H<SUB>2</SUB>O
</LI></UL><H4><SPAN class="mw-headline" id="Investigation_of_the_Kleinschmidt_sequencing"><P style="font-size:15px; background-color:#66bbff;">Investigation of the Kleinschmidt sequencing</P></SPAN></H4><P><B>Investigator: Volker</B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">The CMV promoter sequence was found in front of the expression constructs of VP2 and VP3</DIV></DIV></DIV></DIV><P>The sequencing of den unknown backbone pKEX revealed that the construct VP1ex from the DKFZ seemed not to contain a regulatory region for the expresion of the VP1 ORF, but in the backbonesequence there were some standard primers from GATC that could be used to sequence the insert of all expression constructs.
This sequencing was done at the 26.06.2010. The read into the construct VP1 showed a long homologous region with the ORF of VP1 where as the sequencing of VP2ex and VP3ex did not show a homologous region with the ORFs of VP2 and VP3. 
The test weather primers of GATC bind to the unknown region gave a positive hit in both constructs with the primer GATC_std_CMV_f. This primer is specific for the sequence of the CMV promoter. After this indice the sequences VM_2 26.06. -pBR1.ab1 and VM_3 26.06. -pBR1.ab1 were aligned with the sequence of the CMV  promoter provieded in the stratagene kit in the plasmid pAAV-MCS. The majority of the sequence fitted well with CMV promoter as indicated in the image in purple. At the 5' end of the CMV promoter there were some missmatches in the alignment indicated in yellow.
The fact that VP2ex and VP3ex contained a CMV promoter element which was not included in VP1ex was unexpected. The sequencing with the standard primer GATC_std_CMV_f which binds at the 3' end of the CMV promoter and will probably sequence the ORF of VP2ex and VP3ex was ordered.</P><CENTER><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/July" class="external text" rel="nofollow"><B>=&gt; Go to Labjournal July (labday 46 - 75)</B></A></CENTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/June">http://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/June</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg_Bioware/NoteBook/Labjournal/June" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg_Bioware/NoteBook/Labjournal/June" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/June&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/June&amp;oldid=197564" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>