"
Page
Discussion
View source
History
teams
Log in
 
Team:Freiburg Bioware/Project/Results/Modularization Vector Plasmid
From 2010.igem.org
(Redirected from Team:Freiburg Bioware/Project/Results/Modularisation Vector Plasmid )
Home
Team
Team 2010
Collaboration
Cuckoo Clock Competition
Photo Gallery
Statistics
Contribution
Project
Summary
Highlights
Introduction
Results
   Modularization
   Targeting
   Arming
   Tumor Killing
   Modeling
Methods
Virus Construction Kit Manual
BioBricks
Overview
Capsid Coding Parts
Vector Plasmid Parts
Notebook
Labjournal
Cloning Material
Methods
Meetings
Statistics
Sponsors
Safety
Modeling
Modeling Overview
Virus Production
Virus Infection
Structure Modeling
Modularization
Contents
Modularization of the vectorplasmid
Introduction to Modularization of Vectorplasmid. 1
Recombinant and Modular Vectorplasmid Carrying GOI 2
Cloning and Combination Strategies for the Vectorplasmid. 3
Testing functionality of Assembled Vectorplasmid. 7
Fluorescence Microscopy of Target Cells Demonstrates GOI Expression. 7
Analysis of Target Cells by Flow Cytometry demonstrates GOI Expression. 8
Influence of hGH terminator BioBrick on GOI Expression. 8
Influence of Beta-globin intron Biobrick on GOI Expression. 11
Functionality of the Full Assembled Vectorplasmid Demonstrated by GOI Expression. 14
Conclusion. 16
References. 17
 
Modularization of the RepVP123 plasmids
Overview of RepVP123 plasmid. 2
Modularization: Overview.. 2
Modularization: Removing iGEM restriction sites and establishing loop insertion capability. 3
Modifications in Rep. 3
Modifications in VP123. 4
Modularization: Adapting pSB1C3 to loop insertions – pSB1C3_001. 7
Turning-off natural tropism: HSPG-knock-out. 9
 
Introduction to Modularization of vector plasmid
Producing recombinant virus particles for therapeutical means is, besides specifically target cells, purification and quantification assays of AAV-2, one intention of the Virus Construction Kit provided by the iGEM team Freiburg_Bioware 2010. For obtaining a modular toolkit, the complex components of AAV-2 were extracted and redesigned to match the iGEM standard. Functional activity was tested in cell culture.
Differing from the wildtype AAV-2 genome, the Helper Free System provided by Stratagene comprises three plasmids and a specialized production cell line. AAV-293 cells derived from the HEK cell line express the stably integrated E1A and E1B helper proteins for efficient virus production. The plasmid containing the inverted terminal repeats (ITRs) is encapsidated into the preformed capsids after production of single-stranded DNA therefore also known as vectorplasmid (pGOI). Promoter, beta-globin intron and the hGH terminator signal are flanked by the ITRs and serve in the host cell for regulation of transgene expression. In addition to that, the plasmid coding for the Rep and Cap proteins (pRC) can be provided in trans leading to a layer of specificity due to the fact that the two genes are not packaged into the capsid since lacking of the ITRs impairs encapsidation. Another advantage of the Helper Free System can be attributed to cotransfection of another helper plasmid (pHelper), which provides the necessary proteins normally obtained by superinfection with helper viruses such as adenovirus or herpes simplex virus. These helper genes are required for full viral assembly by regulating gene expression of Rep and Cap proteins.
Recombinant and Modular Vectorplasmid Carrying GOI
The iGEM team Freiburg_Bioware 2010 provides a modular Virus Construction Kit for therapeutical applications, quantification assays and purification approaches depending on capsid modifications and the gene of interest flanked by the inverted terminal repeats (ITRs. In order to produce BioBrick-compatible standardized biological parts, we reengineered the plasmids and added new components for gene therapy approaches and analysis of biological activity of assembled BioBrick parts. Each element required for intact and functional plasmids comprising the ITRs, a promoter, a putative enhancer element and the hGH terminator was PCR amplified and fused together de novo. As shown in Figure 1, the vectorplasmid was assembled with the produced BioBricks consisting of the left and right ITR ( BBa_K404100 and BBa_K40410 ), a promoter (pCMV : BBa_K404102 or phTERT: BBa_K404106 )) , the beta-globin intron ( BBa_K404107 ), the gene of interests (fluorescent proteins mVenus: BBa_I757008 and mCherry: BBa_J06504 , suicide genes mGMK_TK30: ( BBa_K404112 , mGMK_SR39: BBa_K404315 and CD: BBa_K404112 ) and the hGH terminator ( BBa_K404108 ).
Figure 1: Overview of the theoretical sequence of each BioBrick provided within the Virus Construction Kit for an intact and fully functional rAAV genome. The plasmid in the lowest panel was used for tumor killing in combination with plasmids coding for modified capsid proteins. More detailed infomartion about these constructs can be found under ‘Arming: Killing the tumor’ and ‘N-terminal fusion for Targeting’.
 
Cloning and Combination Strategies for the Vectorplasmid
Organization of the recombinant viral DNA was modified ensuring several layers of specificity to our systems including a tumor-specific promoter and suicide genes encoding prodrug convertases. In order to modularize the rAAV sequence, each plasmid element (Figure 1) was PCR-amplified and cloned into the iGEM standard plasmid pSB1C3. Furthermore, the iGEM team Freiburg_Bioware 2010 performed three site-directed mutagenesis in the gene of interest TK30 ( BBa_K404109 ) and cytosine deaminase ( BBa_K404112 ) for deletion of PstI and NgoMIV iGEM site (for further information see the results page of ‘Arming – Killing the tumor’). Since the inverted terminal repeats (ITRs) are GC-rich regions forming T-shaped hairpins during replication, PCR amplification was not possible. Hence a cloning strategy was developed by the iGEM team Freiburg in order to provide BioBrick-compatible ITRs (see ‘Method Development of Cloning Strategy for ITRs’).
In Figure 2 the schematic overview of the modularization process can be seen which has been followed to conduct the assembly steps required for functional vectorplasmids.
Figure 3: Assembly procedure for fusion of BioBricks and composite parts to a fully assembled and functional plasmid coding for your gene of interest. This plasmid can be cotransfected with two helper plasmids providing protein for assembly and encapsidating of the rAAV genome (your gene of interest) into the capsids.
 
The iGEM team Freiburg_Bioware provides two examples demonstrating the assembly procedure for constructing vectorplasmids. The first representative example is the fusion of the BioBrick part beta-globin to the composite parts containing the 5´ elements of the plasmids, which are left ITR and CMV or phTERT promoter, respectively.
As shown in Figure 3 the theoretical cloning performed for assembling the BioBricks beta-globin intron and leftITR_CMV together can be observed.
Figure 4: Theoretical cloning of the composite part leftITR_CMV to the beta-globin intron BioBrick leading to the plasmid leftITR_CMV_beta-globin intron.
 
The plasmids were digested with both XbaI and PstI (beta-globin intron: BBa_K404107 ) or SpeI and PstI (leftITR_CMV) and loaded on an agarose gel. As demonstrated in the preparative gel in Figure 4, the expected bands could be detected under UV light and the extracted DNA could be successfully ligated. Each assembly step for producing BioBrick intermediates was conducted following the same strategy.
 
Figure 5: Assembly intermediate in fusion of the vectorplasmids containing different promoters. Fusion of the BioBrick part beta-globin ( BBa_K404107 ) intron to the composite parts leftITR_pCMV and leftITR_phTERT, respectively, was performed following the BioBrick assembly strategy by digesting the insert with PstI and XbaI and the vectors with SpeI and PstI. The left lane shows the expected fragment at around 560 bp which corresponds to the beta-globin intron fragment, in contrast to the two lanes in the center and on the right which correspond to linearized plasmids after digesting with above mentioned iGEM restriction sites. M, GeneRuler DNA ladder mix; Insert, pSB1C3_beta-globin intron; Vector pCMV, pSB1C3_leftITR_pCMV; Vector phTERT, pSB1C3_leftITR_phTERT.
 
Separated fragments were extracted using the Gel Extraction Kit provided by Qiagen (Hilden, Germany) and ligated with T4-ligase. After ligation has been carried out, E. coli XL-1B cells were transformed and incubated over night at 37°C. Picking clones from the transformation plate was performed the following day and DYT medium was inoculated incubating overnight. Plasmid DNA was isolated and test digestion revealed that cloning was successful obtaining the composite part leftITR_CMV_beta-globin intron ( BBa_K404117 ).
Plasmid production incorporating all required elements for transgene expression and genome encapsidation into empty viral capsids was performed by fusing the downstream elements consisting of the hGH terminator and right ITR to the intermediate part providing the gene of interest and the promoter fused to the left ITR. Figure 5 demonstrates the assembly performed with pSB1C3_leftITR_phTERT_beta-globin intron_mVenus and pSB1C3_hGH_rightITR ( BBa_K404116 ). The fragment obtained after digestion on the left lane fits to the hGH-terminator_rightITR length. The isolated fragments were ligated and successful assembly was confirmed by test digestion obtaining the vectorplasmid pSB1C3_leftITR_phTERT_beta-globin intron_mVenus_hGH_rightITR ( BBa_K404124 ).
Figure 6: Modularization of the assembled vectorplasmid containing the phTERT promoter and mVenus as gene of interest. Fusion of the composite pSB1C3_leftITR_phTERT_beta-globin intron_mVenus part to the composite parts pSB1C3_hGH_rightITR was performed following the BioBrick assembly strategy by digesting the insert with XbaI and PstI and the vector with SpeI and PstI. The left lane corresponds to linearized plasmid after digesting with above mentioned iGEM restriction sites whereas the right lane reveals an intensive band at around 650 bp confirming the expected size of 657 bp of hGH_rITR. M, GeneRuler DNA ladder mix; Vector, pSB1C3_leftITR_phTERT_beta-globin intron_mVenus; Insert, pSB1C3_ pSB1C3_hGH_rightITR.
 
Since cloning does not confirm biological activity, we analyzed the plasmids and their functional components, hGH terminator and beta-globin intron, in cell culture. Assembled plasmids have been cotransfected, using AAV-293 cells, which provide the stable integrated E1A and E1B genes, with helper plasmids required for capsid assembly and genome encapsidation (pRC and pHelper) in a molar ratio of 1:1:1 (pGOI:pRC:pHelper). Virus particles containing the single stranded DNA were harvested 72 hours post transfection and HT1080 cells transduced with constant volumes of viral vectors. 48 hours post infection; transduced cells expressing the gene of interest were analyzed by flow cytometry. Facilitating and demonstrating the analysis of functionality of the assembled plasmid, mVenus was used in first place since fluorescent proteins enable facile visualization using fluorescent microscopy and flow cytometry analysis.
Testing functionality of Assembled Vectorplasmid
Fluorescence Microscopy of Target Cells Demonstrates GOI Expression
Qualitative analysis of mVenus expression by fluorescence microscopy was conducted using Axio Observer Z1 showing that transduced HT1080 cells and non-transduced cells could be easily distinguished. In Figure 6 cells were excited with 505nm and fluorescence emission at 536nm was detected. Therefore, successful infection of tumor cells by recombinant viral particles carrying the assembled vectorplasmid coding for mVenus could be demonstrated.
A
B
C
D
Figure 7: Fluorescence microscopy (Exciatation: 505nm, Emission: 536nm) was performed for detection of transduced cell expression mVenus. A:Cells detected in bright field picture B: Detection of mVenus expression can be observed.
 
Analysis of Target Cells by Flow Cytometry demonstrates GOI Expression
Characterizing the function of the hGH terminator, the beta-globin intron and the complete plasmid, several approaches were conducted followed by analysis via flow cytometry.
Influence of hGH terminator BioBrick on GOI Expression
The iGEM team Freiburg provides the hGH plolyadenylation sequence within the ‘Virus Construction Kit’ due to the fact that almost every eukaryotic mRNA is processed at their 3´ and 5´end except for histone mRNAs (Millevoi et al. 2006). Pre-mRNAs contain two canonical conserved sequences. First, the polyadenylation signal “AATAAA” which is recognized by the multiprotein complex and second the GT-rich region (downstream sequence element, DSE) which is located 30 nucleotides downstream of the cleavage site. The assembled 3´end-processing machinery cleaves the mRNA transcript immediately after a CA-nucleotide therefore defining the cleavage site (Danckwardt et al. 2008). Recombinant vectorplasmids were engineered containing the inverted terminal repeats (ITRs), a strong eukaryotic promoter (CMV promoter: BBa_K404102 ) and mVenus as gene of interest with and without the hGH terminator signal. Transduction of HT1080 cells with constant volume of viral particles containing the vectorplasmids and measuring mVenus expression 24 hours post infection by flow cytometry demonstrated that transgene expression of the constructs lacking the hGH termination signal is significantly reduced as shown in Figure 7 and Figure 8 confirming the expected results that hGH is essential for mRNA processing. The iGEM team Freiburg_Bioware 2010 therefore suggests using the provided hGH termination signal within the Virus Construction Kit for optimal gene expression.
Vectorplasmid lacking hGH termination signal
Vectorplasmid containing hGH terminator signal
Figure 8: Flow cytometry analysis of vectorplasmids with and without hGH terminator. A: Gating non transduced cells (control); subcellular debris and clumps can be distinguished from single cells by size, estimated forward scatter (FS Lin) and granularity, estimated side scatter (SS Lin) B: Non transduced cells applied against mVenus (Analytical gate was set such that 1% or fewer of negative control cells fell within the positive region (R5). C: Gating transduced cells (R2 ≙R14) (used plasmids for transfection: GOI: pSB1C3_lITR_CMV_beta-globin intron_mVenus_rITR ( BBa_K404127 ), pHelper, pRC). D: Transduced cells plotted against mVenus, R10 comprises transduced cells by detecting mVenus expression. E: Overlay of non-transduced (red) and transduced (green) cells applied against mVenus.F: Gating non-transduced cells (control) G: Non-transduced cells applied against mVenus. H: Gating transduced cells (R2 ≙R14) (used plasmids for transfection: GOI: reassembled pSB1C3_lITR_CMV_beta-globin_mVenus_hGH_rITR ( BBa_K404119 ), pHelper, pRC). I: Transduced cells applied against mVenus, R10 comprised transduced cells, by detecting mVenus expression. J: Overlay of non-transduced (red) and transduced (green) cells applied against mVenus.
 
 
Figure 9: Flow cytometry analysis of vectorplasmids with and without hGH terminator. YFP expression of viral genomes was determined by flow cytomery 24 hour post infection. Results demonstrate that mVenus expression of vectorplasmids lacking the hGH terminator is reduced significantly proving that the polyadenylation signal is essential for viral gene expression using recombinant viral vectors engineered by using components of the Virus Construction Kit.
 
Influence of Beta-globin intron Biobrick on GOI Expression
Providing an element assumed to be an enhancer of transgene expression (Nott et al. 2003), the iGEM team Freiburg tested a beta-globin intron derived from the human beta globin gene which can be fused upstream of the desired gene of interest. The beta-globin intron BioBrick consists of a partial chimeric CMV promoter followed by the intron II of the beta-globin gene. The 3´end of the intron is fused to the first 25 bases of human beta globin gene exon 3. The beta globin intron BioBrick is assumed to enhance eukaryotic gene expression (Nott et al. 2003). Analysis was conducted as described for the hGH terminator experiment (see above). As shown in Figure 9 and Figure 10 the vectorplasmid missing the beta-globin intron showed a negligible difference in mVenus expression compared to viral genomes containing the beta-globin intron. Considering these results and taking into account that a constant volume of viral particles has been used for transduction, the difference between the construct containing and lacking the beta-globin intron is minimal. Since packaging efficiency of the AAV-2 decreases with increasing sizes of the insert (Dong et al. 1996), the iGEM team Freiburg_Bioware suggests using the beta-globin intron in dependence on the size of your transgene.
Vectorplasmid lacking beta-globin intron
Vectorplasmid containing beta-globin intron
Figure 10: Flow cytometry analysis of vectorplasmids with and without beta-globin intron. A: Gating non transduced cells (control); subcellular debris and clumps can be distinguished from single cells by size, estimated forward scatter (FS Lin) and granularity, estimated side scatter (SS Lin) B: Non transduced cells applied against mVenus (Analytical gate was set such that 1% or fewer of negative control cells fell within the positive region (R5). C: Gating transduced cells (R2 ≙R14) (used plasmids for transfection: GOI: pSB1C3_lITR_CMV_mVenus_hGH_rITR ( BBa_K404128 ), pHelper, pRC). D: Transduced cells plotted against mVenus, R10 comprised transduced cells, by detecting mVenus expression E: Overlay of non-transduced (red) and transduced (green) cells applied against mVenus F: Gating non-transduced cells (control). G: Non-transduced cells applied against mVenus (R5).H: Gating transduced cells (R2 ≙R14) (used plasmids for transfection: GOI: reassembled pSB1C3_lITR_CMV_beta-globin_mVenus_hGH_rITR ( BBa_K404119 ), pHelper, pRC). I: Transduced cells applied against mVenus, R10 comprised transduced cells, by detecting mVenus expression. J: Overlay of non-transduced (red) and transduced (green) cells applied against mVenus.
 
Figure 11: Flow cytometry analysis of vectorplasmids with and without beta-globin intron. 48 hours post transfection, viral particles were harvested by freeze-thaw lysis and centrifugation followed by HT1080 transduction. YFP expression of vectorplasmids was determined by flow cytometry 24 hours post infection. The vectorplasmid missing the beta-globin intron showed a negligible difference in mVenus expression compared to viral plasmid containing the beta-globin intron.
 
Functionality of the Full Assembled Vectorplasmid Demonstrated by GOI Expression
Producing recombinant virus particles for therapeutical applications is, besides specific cell targeting, purification and quantification assays of AAV-2, one intention of the Virus Construction Kit provided by the iGEM team Freiburg_Bioware 2010. For obtaining a modular toolkit, the complex biological system of the Adeno-associated virus serotype 2 was examined by an exhaustive literature search. Subsequently, the essential components for AAV-2 particle production were extracted and redesigned to match the iGEM standard.
The provided tripartite system is independent of a superinfection  of Adeno- or herpes simplex viruses since the genes encoding the required helper-proteins are co-transfected. Inside the eukaryotic host cell, the DNA sequence containing the inverted terminal repeats (ITRs) is extracted and later encapsidated into the preformed capsids after production of single-stranded DNA. Consequently, this plasmid is known as the vector plasmid (pGOI). Promoter, beta-globin intron and the hGH terminator signal are flanked by the ITRs (ITRs, BBa_K404100 and BBa_K404101 ) and regulate transgene expression. The vector plasmid containing the desired gene of interest is cotransfected with the RepCap plasmid ( BBa_K404001 , BBa_K404102 or BBa_K404103 ) and the pHelper plasmid. To obtain the fully assembled vector plasmid, several assembly steps have to be performed.  
After assembly of plasmids containing all required elements, vector plasmid functionality was confirmed in cell culture. AAV-293 cells stably expressing the E1A and E1B proteins were transfected with three plasmids (pHelper, pRC, pGOI). Virus particles were harvested 72 hours post transfection and the tumor cell line HT1080 was transduced with the recombinant viral vectors encapsidating the gene of interest mVenus ( BBa_I757008 ).
In the beginning, the iGEM team Freiburg_Bioware 2010 used a commercial vector plasmid, pAAV-MCS (Stratagene), to determine whether virus particle production by AAV-293 cells could be achieved. iGEM RFC 25 restriction enzyme sites were introduced and the fluorescent protein mVenus was subcloned into this plasmid. Subsequently, this plasmid was used as a reference and compared to the assembled vector plasmid pSB1C3_lITR_CMV_beta-globin_mVenus_hGH_rITR (abbreviated pSB1C3_mVenus, BBa_K404119 ). Fluorescence expression data obtained by flow cytometry analysis are shown in Figure 11 and Figure 12. Comparing mVenus expression of the reference plasmid and the pSB1C3_mVenus plasmid revealed that biological functionality of the reassembled plasmid was preserved.
pSB1C3_mVenus ( BBa_K404119 )
pAAV_mVenus (reference)
Figure 12: Flow cytometry analysis of fluorescent protein expression in transduced HT1080 cells. For viral particle production, AAV-293 cells were transfected with the reassembled vector plasmid ( BBa_K404119 ) or the reference plasmid, respectively. A: Gating non transduced cells (control); subcellular debris and cellular aggreates can be distinguished from single cells by size, estimated forward scatter (FS Lin) and granularity, estimated side scatter (SS Lin) B: Non-transduced cells plotted against cells expressing mVenus (Analytical gate was set such that 1% or fewer of negative control cells fell within the positive region (R5) C: Gating transduced cells (R2 ≙ R14) (plasmids used for transfection: pGOI: pSB1C3_lITR_CMV_beta-globin_mVenus_hGH_rITR (pSB1C3_mVenus: BBa_K404119 ), pHelper, pRC. D: Transduced cells plotted against cells expressing mVenus. R10 comprises transduced cells detected by mVenus fluorescence. E: Overlay of non-transduced (red) and transduced (green). F: Gating non-transduced cells (control). G: Non-transduced cells plotted against cells expressing mVenus (R5). H: Gating transduced cells (R14 ≙ R2) (plasmids used for transfection: pGOI: pAAV_mVenus, pHelper). I: Transduced cells plotted against cells expressing mVenus. R10 comprises transduced cells detected by mVenus fluorescence. J: Overlay of non-transduced (red) and transduced (green) cells plotted against mVenus expression. 
 
 
 
Figure 13: Quantification of mVenus fluorescence by flow cytometry analysis. The data source is identical to Figure 13. Compared to the reference vector plasmid, mVenus expression in HT1080 cells is enhanced in case of the assembled plasmid (pSB1C3_mVenus) 
Conclusion
Idea of the modular ‘Virus Construction Kit’ is to provide all required elements for producing recombinant, functional virus particles delivering encapsidated genes of interest to specific cells. First step was to modify and modularize the vectorplasmid comprising basically the cis-elements for replication (ITRs), a strong eukaryotic or tissue specific promoter (pCMV or phTERT), the gene of interest (fluorescent proteins or suicide genes) and the hGH termination signal. Each element was successfully cloned and reassembled resulting in functional vectorplasmids determined by flow cytometry and fluorescence microscopy analyses. Experiments have been performed with mVenus since measurement of fluorescent proteins can be easily performed and visualized. Considering the results, the iGEM team Freiburg_Bioware 2010 then tested the construct containing the suicide genes thymidine kinase and cytosine deaminase. Further details demonstrating efficient tumor killing, using prodrug-activating systems, see results page Arming .
References
Danckwardt, S., Hentze, M.W. & Kulozik, A.E., 2008. 3' end mRNA processing: molecular mechanisms and implications for health and disease. The EMBO journal, 27(3), 482-98. Available at: http://www.ncbi.nlm.nih.gov/pubmed/18256699.
Dong, J.Y., Fan, P.D. & Frizzell, R.a., 1996. Quantitative analysis of the packaging capacity of recombinant adeno-associated virus. Human gene therapy, 7(17), 2101-12. Available at: http://www.ncbi.nlm.nih.gov/pubmed/8934224.
Millevoi, S. et al., 2006. An interaction between U2AF 65 and CF I(m) links the splicing and 3' end processing machineries. The EMBO journal, 25(20), 4854-64. Available at: http://www.ncbi.nlm.nih.gov/pubmed/17024186.
Nott, A., Meislin, S.H. & Moore, M.J., 2003. A quantitative analysis of intron effects on mammalian gene expression. RNA (New York, N.Y.), 9(5), 607-17. Available at: http://www.ncbi.nlm.nih.gov/pubmed/12702819.
 
Move to Top
 
Overview of RepVP123 plasmid
Modularization: Overview
In our terminology the term “RepVP123” encompasses the whole AAV2 genome excluding the ITRs. The rep locus comprises four proteins related to genome replication while the cap locus codes for the proteins VP1, VP2, VP3 and the assembly-associated protein (AAP), which are required for viral capsid assembly. Source of the RepVP123 BioBrick supplied within iGEM team Freiburg_Bioware 2010 Virus Construction Kit is the wild-type AAV2 RepVP123, as provided e. g. in the pAAV vector from Stratagene. In order to introduce iGEM standard and additionally enabling the possibility to modify the viral capsid via integration of certain motives within the viral loops 453 and 587 a total of twelve mutations within RepVP123 (see Figure 1) and additionally two mutations within the pSB1C3 backbone were performed either by Site-Directed Mutagenesis (SDM) or by ordering and cloning of specifically designed gene sequences matching the required demands. Modifying the pSB1C3 led to iGEM team Freiburg_Bioware’s variant of this backbone, pSB1C3_001.
Figure 1 Mutations implemented into RepVP123 in order to establish both iGEM standard and loop insertion capability. Green arrows indicate integrated restriction sites, red red arrows indicate deleted restriction sites. KpnI was deleted first and reinstated afterwards. (see text).
 
Plasmid name:
Functionality (determinded in cell culture via transduction and flow cytometry ):
4x mutations (PstI (310), BamHI (859), SalI (1239), PstI (4073))
inserted rep fragment
inserted cap fragment
reinstated KpnI
pAAV
pSB1C3_001
HSPG-ko
pAAV_RC (wild-type)
yes
 
 
 
 
x
 
 
pAAV_RC_4x mutant
yes
x
 
 
 
x
 
 
pAAV_RC_inserts
no
x
x
x
 
x
 
 
pAAV_RC_Cap
yes
x
 
x
 
x
 
 
pAAV_RC_RepVP123
yes
x
x
x
x
x
 
 
pSB1C3_RepVP123_ p5TATAless
yes
x
x
x
x
 
x
 
pSB1C3_RepVP123_ HSPG-ko_p5TATAless
yes
x
x
x
x
 
x
x
Figure 2 Table contains complete overview about all plasmids containing RepVP123 which were used by iGEM team Freiburg_Bioware 2010.
Modularization: Removing iGEM restriction sites and establishing loop insertion capability
Modifications in Rep
Figure 3 Restriction sites within the wild-type rep gene sequence, which were removed via cloning of synthetized rep gene fragment into the plasmid. The red box indicates the region spanned by the synthetic sequence.
Making the RepVP123 wild-type compatible with the iGEM standards required the removal of five restriction sites (see Figure 1). This was achieved using site-directed mutagenesis for PstI (position 310) and PstI (4073). The remaining three iGEM restriction sites EcoRI (1578), PstI (1773) and EcoRI (1796) were replaced by a synthetic gene fragment, since the rep ORF contained these restriction sites in close proximity to each other plus an additional KpnI restriction site which was also not desired (see Figure 2). This gene fragment was cloned into the rep gene using HindIII and SwaI, which are single-cutting restriction enzymes adjacent to the target area. Additionally, BamHI (859) and SalI (1239) were removed, because these enzymes were required for genetically inserting the loop modifications in VP123.
Modifications in VP123
In order to implement the restriction sites necessary for targeting via loop insertions, the gene coding for the VP proteins was modified as well. The introduction of these restriction required up to four base mutations in a row, hence it was decided to synthesize this gene fragment and replace the wild-type sequence in RepVP123 as well.
Figure 4 Restriction sites within cap sequence showing introduced loop insertion restriction sites into cap to enable cloning of targeting or purification motifs into both 453 and 587 loops. Again, the red box indicates gene sequence which was synthetized.
 
Figure 5 Results for transduction efficiency measured by flow cytometry. Fluorescence is measured in surviving cells. The tested RepVP123 containing four point mutation to delete iGEM and loop insertion restriction sites does not show any difference within mVenus expression compared to the wild-type and therefore can be verified working.
Alongside to creating an iGEM compatible plasmid, infectivity of the modified construct was tested in cell culture via flow cytometry. Then experiments confirmed that single cloning steps did not interfere with natural viral infectivity at first (see above). But cloning of the synthesized rep gene fragment into the plasmid dramatically reduced transduction efficiency as detected by flow cytometry. Scrutinizing each mutation and its potential impact, suggested that abolished transduction was related to the mutation, which removed the KpnI (1721) site. This site is located within a splice site, which is crucial for the Rep proteins, and thus even silent mutations may interfere with virus production (see extra topic “Rep proteins”).
 
 
Figure 8 AAV-293 cells were transfected with three plasmids pHelper, pSB1C3_[AAV2]-left-ITR_pCMV_betaglobin_mVenus _hGH_[AAV2]-right-ITR and pAAV_RC_inserts (see above) or pAAV_RC (see below) providing essential genes and proteins for producing viral particles. After 48 hour post transfection, viral particles were harvested by freeze-thaw lysis and centrifugation followed by HT1080 transduction. mVenus expression of viral genomes was determined by flow cytomery 24 hours post infection. Fluorescence is measured in surviving cells. Results show that insertion of both rep and cap syntheses disrupts viral infectivity.
Therefore, additional constructs were designed containing only the synthetic cap sequence or both, the synthesized sequences plus a re-mutation of KpnI (1721), in order to re-establish the wild-type splice site within the rep ORF. Results from cell culture obtained via FACS revealed that in fact the poor results were related to the KpnI restriction site deletion. Both constructs showed a transduction efficiency corresponding to the unmodified wild-type RepVP123’s transduction efficiency.
Figure 9 Fluorescence of cells transduced with mVenus carrying rAAV measured by flow cytometry. AAV-293 cells were transfected with three plasmids pHelper, pSB1C3_[AAV2]-left-ITR_pCMV_betaglobin_mVenus_hGH_[AAV2]-right-ITR and RepVP123 constructs providing essential genes and proteins for producing viral particles. 48 hours post transfection, viral particles were harvested by freeze-thaw lysis and centrifugation followed by HT1080 transduction. mVenus expression of viral genomes was determined by flow cytometry 24 hours post infection. Results show that cap integration does not influence infectivity. Fluorescence is measured in surviving cells. Recreation of KpnI within rep splice site recovers transduction efficiency.
 
Modularization: Adapting pSB1C3 to loop insertions – pSB1C3_001
To fulfill iGEM requirements all plasmids need to be submitted in pSB1C3, therefore primers were ordered for amplifying RepVP123 containing all modifications done so far by PCR and cloning the into pSB1C3. Still, pSB1C3 contains two restriction sites for SspI and PvuII restriction enzymes in its CAT marker. Since these are necessary for cloning ViralBricks in this vector, the iGEM Team Freiburg_Bioware 2010 decided in agreement with iGEM Headquarters to implement a new standard for the pSB1C3 backbone which was named pSB1C3_001. Both restriction sites interfering with ViralBrick insertions were mutated to make SspI and PvuII single-cutters (see method development).
Figure 10 Comparison of pSB1C3 (upper row) and pSB1C3_001 (lower row). Deletions of SspI and PvuII are marked by red boxes.
RepVP123 containing both rep and cap synthetic gene fragments including the re-mutation of KpnI and the downstream p5TATA-less promotor was cloned into the newly constructed pSB1C3_001. Testing this newly assembled plasmid in cell culture revealed unexpected data. Not only did the newly assembled plasmid work (see Figure 10), but in comparison to pAAV containing the same RepVP123 construct, pSB1C3_001 showed an about 3 times higher transduction efficiency. Although exact reasons are still unknown, these results are probably related to the reduced length of pSB1C3_001 compared to the original pAAV plasmid of approximately 1000 base pairs.
 
Figure 11 AAV-293 cells were transfected with three plasmids pHelper, pSB1C3_001_[AAV2]-Rep-VP123_p5-TATAless or pAAV_RC_IRCK and pSB1C3_[AAV2]-left-ITR_pCMV_beta-globin_mVenus_hGH_[AAV2]-right-ITR providing essential genes and proteins for producing viral particles. After 48 hours post transfection, viral particles were harvested by freeze-thaw lysis and centrifugation followed by HT1080 transduction. mVenus expression of viral genomes was determined by flow cytomery after 24 hours post infection. Fluorescence is measured in surviving cells. Results showed functionality of RepVP123 within pSB1C3_001 vector and additionally increased transduction efficiency.
Turning-off natural tropism: HSPG-knock-out
Shutting-down the natural viral tropism is essential for targeting specifically tumor cells and not infecting healthy cells. Therefore, the iGEM team Freiburg_Bioware 2010 decided to knock-out the viral natural tropism delivered by the heperan sulfate proteoglycan-(HSPG) binding site within the viruses 587 loop. The knock-out was cloned by designing primers containing the required base exchanges and performing a SDM. Like performed before, this RepVP123 variant was tested in cell culture as well and evaluated by flow cytometry. Results show that mutation of HSPG-binding motif has severe impact on transduction efficiency thus enabling a viral particle carrying this knock-out and additional targeting motifs, e.g. within the loops or presented via N-terminal fusion to bind target cells’ receptors and therefore infecting target cells at a much higher rate compared to unspecific infection of other cell types within an organism (see Figure 12).
To quantify differences in infectivity, the infectious titer of viral particles built-up of RepVP123 with and without HSPG binding motif was determined by qPCR (see Figure 14) for different cell lines. Results show that the implemented HSPG-knock-out verifies results obtained from flow cytometry, infectious titers severely compared to RepVP123 with intact HSPG binding motif.
Figure 12 Alignment of 587 loop within viral VP123: The upper sequence shows a strand containing the HSPG binding motif (AGA, in red boxes), the lower sequence contains the HSPG-ko introduced by the iGEM team Freiburg_Bioware 2010 (GCT and GCC, blue boxes).
 
Figure 13 Transduction efficiency of HT1080 cells measured by flow cytometry. Fluorescence is measured in surviving cells. Knock-out of HSPG binding motif greatly reduces transduction efficiency compared to RepVP123 containing the motiv.
 
Figure 14 Infectious titers of RepVP123 with and without natural HSPG binding motif tested in different cell lines via qPCR. Shutting-down the HSPG binding motif reduces infectious titer in both HT1080 and HeLa cell lines. For A431 cells, no infectious titer could be detected via qPCR, which is probably related to poor transduction efficiency of A431 cells.
Move to Top
Retrieved from " http://2010.igem.org/Team:Freiburg_Bioware/Project/Results/Modularization_Vector_Plasmid "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
