"
Page
Discussion
View source
History
teams
Log in
 
Team:Freiburg Bioware/NoteBook/Labjournal/September2
From 2010.igem.org
Home
Team
Team 2010
Photo Gallery
Cuckoo Clock Competition
Collaboration
Example Page
Project
Overview
Highlights
Results
   Modularisation
   Targeting
   Arming
   All-in-One
Methods
Virus Construction Kit Manual
BioBricks
Overview
Capsid Coding Parts
Vector Plasmid Parts
Miscellaneous Parts
Notebook
Labjournal
Materials
Methods
Experiments
Meetings
Statistics
Sponsors
Safety
Modeling
Modeling Overview
Virus Production
Virus Infection
Structure Modeling
=> Back to Notebook overview
NoteBook Navigator
March (labday 1)
April (labday 2 - 5)
May (labday 6 - 17)
June (labday 18 - 45)
July (labday 46 - 75)
August part 1 (labday 76 - 92)
August part 2 (labday 93 - 106)
September part 1 (labday 107 - 123)
September part 2 (labday 124 - 135)
October part 1 (labday 136 - 149 )
October part 2 (labday 150 - 166 )
November  (labday 167 - 170 )
Cellculture
Contents
1 September
1.1 124. labday 19.09.2010
1.1.1 Continuation with Colony-PCR of pSB1C3_CD and ViralBrick motif Z34C
1.1.2 Testtransduction of the final modified capsid coding constructs
1.1.3 Miniprep of serveral constructs
1.2 125. labday 20.09.2010
1.2.1 Picking the clones and preparing the over-night culture
1.2.2 Minipreps and Sequencing of ViralBrick clones
1.2.3 Cloning of pCerulean_Affibody_VP2/3 and pSB1C3_Affibody_middlelinker_GFP_his
1.2.4 Cloning of P5_TATAless downstream of pSB1C3_001_RC_insrepcap_KpnI_back and P5 upstream of pSB1C3_001_RC_insrepcap_KpnI_back
1.2.5 Test digestion of VP2-N-terminal fusion constructs
1.2.6 BioBrick production: PCR of pAAV_RC_InsRepCap_KpnIback_VP1-ko and VP2-ko
1.2.7 2x repetition of CD biobrick
1.3 126. labday 21.09.2010
1.3.1 Trafo evalutation of CD biobrick plate
1.3.2 Continuation of cloning of pCerulean_Affibody_VP2/3 and pSB1C3_Affibody_middlelinker_GFP_his
1.3.3 Sequencing result of pSB1C3_EGFP_His upper and (!) lower band
1.3.4 Midi-Prep of pHelper & pSB1C3_leftITR_CMV_beta-globin_mVenus_hGH_rightITR
1.3.5 New aliquots (60µl) of XL1blue can be used !  (Jessica)
1.4 127. labday 22.09.2010
1.4.1 Sequencing results of ViralBrick ligations using dephosphorylated vector
1.4.2 Cloning pSB1C3_Viral Brick 587KO-empty (B274/P542) into pSB1C3_VP2/3_Capins (B438/P514)
1.4.3 Cellculture: Serum-free cells
1.4.4 PCR of P40
1.4.5 Mini-Prep of pSB1C3_001_RC_InsRepCap_KpnIback_VP1-ko
1.4.6 Cloning of CMV promotor into pSB1C3_VP123_inscap
1.4.7 Sequencing of Zegfr:1907-Linker Library, His-Tag and imaging appoaches
1.4.8 Mass Midi-Prep
1.4.9 RNA isolation and cDNA synthesis
1.4.10 cDNA synthesis
1.5 128. labday 23.09.2010
1.5.1 PCR of VB constructs to remove mutations
1.5.2 Colony PCR of new clones from the last ligations
1.5.3 Contuniation: Cloning pSB1C3_Viral Brick 587KO-empty (B274/P542) into pSB1C3_VP2/3_Capins (B438/P514)
1.5.4 Mini-Prep and test digestion of pCerulean_Affibody_VP2/3, pSB1C3_Affibody_middlelinker_EGFP_His, pSB1C3_001_RC_InsRepCap_KpnIback_VP2-ko
1.5.5 PCR for 587 Z34C ViralBricks
1.5.6 PCR of Vp1_ex, Vp2_ex and Vp3_ex and cloning into pSB1C3_001
1.5.7 RNA Purity Gel
1.5.8 hTERT PCR Reaction
1.5.9 Cloning of pBAD_Affibody_middlelinker_EGFP_His
1.5.10 Picking clones of pSB1C3_CMV_VP123_inscap
1.5.11 Cloning of ***** into pSB1C3_CFP
1.5.12 Repetition: Cloning of P5_TATAless downstream of pSB1C3_001_RC_insrepcap_KpnI_back and P5 upstream of pSB1C3_001_RC_insrepcap_KpnI_back
1.6 129. labday 24.09.2010
1.6.1 Sequencing results of VB 13.4
1.6.2 Contuniation: Cloning pSB1C3_Viral Brick 587KO-empty (B274/P542) into pSB1C3_001_VP2/3_Capins (B438/P514)
1.6.3 Sequencing evaluation of VP2/3_Gsat-linker and VP2/3_cap_Gsat-linker
1.6.4 New CD primer ordered
1.6.5 Sequencing of pCerulean_Zegfr:1907_VP2/3
1.6.6 Repetition: PCR for biobrick production: Cloning RepCap into pSB1C3
1.6.7 Picking clones of pSB1C3_CFP_email
1.6.8 MiniPreps and Test digestion of pSB1C3_CMV_VP123_capins
1.6.9 MiniPreps of 587 Z34C ViralBricks and pSB1C3_p40
1.6.10 Re-Trafo of DARPin
1.7 130. labday 25.09.2010
1.7.1 Sequencing of VB: 13.1, 13.4 (approach w. dephosphorylated vector, second clone picking round
1.7.2 Miniprep of serveral constructs
1.7.3 Sequencing results of pSb1C3_Affibody_middlelinker_EGFP_His
1.7.4 Sequencing result of pSB1C3_CMV_VP123_capins_clone1
1.7.5 Cloning of 453 and 587 ViralBricks into pSB1C3_CMV_VP123_capins
1.7.6 Sequencing results: pSB1C3_001_VP2/3_HSPG-KO
1.7.7 Cloning of pBAD_Affibody_middlelinker_EGFP_His and pSB1C3_Affibody_middlelinker_EGFP_His
1.8 131. labday 26.09.2010
1.8.1 Continuation of cloning of pBAD_Affibody_middlelinker_EGFP_His and pSB1C3_Affibody_middlelinker_EGFP_His
1.8.2 Picking clones of different constructs
1.8.3 Continuation of cloning of 453 and 587 inserts into pSB1C3_CMV_VP123_capins
1.8.4 Colony PCR of remaining 587 constructs
1.8.5 Sequencing results: pCerulean_Zegfr:1907_VP2/3
1.8.6 N-terminal VP2-Fusion & VP1 Insertion: Fusion of VP2/3_HSPG-KO
1.9 132. labday 27.09.2010
1.9.1 Sent for sequencing
1.9.2 Picking clones of VP1 insertion and VP2 fusion HSPG-KO approaches
1.9.3 MiniPreps of different constructs
1.9.4 Miniprep of serveral constructs
1.9.5 Re-Trafo of pSB1C3_pTAV (P5)
1.9.6 Repetition: Cloning of P5_TATAless downstream of pSB1C3_001_RC_insrepcap_KpnI_back and P5 upstream of pSB1C3_001_RC_insrepcap_KpnI_back
1.9.7 Picking clones of VP1 insertion and VP2 fusion HSPG-KO approaches
1.9.8 Picking clones of VP1 insertion and VP2 fusion HSPG-KO approaches
1.10 133. labday 28.09.2010
1.10.1 Cloning of the ViralBricks BAP-HSPG-ko and His-HSPG-ko into pSB1C3_VP2/3
1.10.2 Test digestion: ViralBricks in pSB1C3_CMV_VP123_capins
1.10.3 Sequencing result of pSB1C3_email and pBAD_Affibody_middlelinker_EGFP_His
1.10.4 Preparation of PBS Buffer and PBS Buffer with Imidazole
1.10.5 ViralBrick Sequencing results: pSB1C3_587_KO_Z34C and pSB1C3_587_KO_Z34C_Spacer
1.10.6 Cloning of Darpin into pCerulean_Vp1up_NLS and pSB1C3
1.10.7 Cloning of pSB1C3_001_CMV and pSB1C3_001_Vp2/3_HSPG-Ko_CMV_ZEGFR:1907
1.10.8 CD assembly with new suffix primer
1.10.9 test digestion of all CD constructs
1.10.10 Cloning of pSB1C3_Affibody_middlelinker_EGFP_His
1.10.11 Sequencing analysis of pSB1C3_001_RC_IRCK, pSB1C3_001_RC_IRCK_VP1-ko_P5tataless and pSB1C3_001_RC_IRCK_VP2-ko_P5tataless
1.10.12 Cloning of HSPG_KO_empty into pSB1C3_001_RC_IRCK_VP1-ko_P5tataless and pSB1C3_001_RC_IRCK_VP2-ko_P5tataless
1.10.13 Cloning of 587_KO_Z34C and 587_KO_Z34C_Spacer into pSB1C3_001_CMV_VP123_capins
1.10.14 Mini-Prep and Test-digestion of VP1 insertion and VP2 fusion HSPG-KO approaches
1.11 134. labday 29.09.2010
1.11.1 CD assembly with new suffix primer
1.11.2 Miniprep of serveral constructs
1.11.3 mGMK_TK30 approaches
1.11.4 Design of mutagenesis primers for deletion of PstI in mGMK_TK30
1.11.5 Cloning of Darpin into pCerulean_Vp1up_NLS and pSB1C3 Part Two
1.11.6 Cloning of pGGTBT7_Affibody_middlelinker_EGFP_His
1.11.7 Cloning of CMV promotor and Affibody into pSB1C3_001_VP2/3_capins and pSB1C3_001_VP2/3_HSPG_KO
1.11.8 Cloning of CFP into pSB1C3_lITR_phTERT_beta-globin
1.11.9 Sequencing results: VP1 insertion and VP2 fusion: Fusion of VP2/3_HSPG-KO
1.11.10 Cloning of HSPG_KO_empty into pSB1C3_001_RC_IRCK_P5tataless
1.11.11 Mass Midi-Prep II
1.12 135. labday 30.09.2010
1.12.1 picking CD clones
1.12.2 ViralBricks BAP-ko and His-ko in VP23
1.12.3 Cloning of Darpin into pCerulean_Vp1up_NLS and pSB1C3 Part Three
1.12.4 Test digestion of pCerulean_ZEGFR:1907_MiddleLinker_VP2/3_HSPG-KO clone 2
1.12.5 Cloning VP2/3_Capins and VP2/3_HSPG-KO downstream of Middle Linker
1.12.6 Miniprep of serveral constructs
1.12.7 Test digestion of pSB1C3_ZEGFR:1907 _middlelinker_EGFP_His and mCherry iGEM 2008
1.12.8 Test digestion of pSB1C3_001_CMV_affibody_HSPG_KO and  pSB1C3_001_CMV_VP123_587_KO_Z34C
1.12.9 Midiprep of pSB1C3_001_CMV_VP1/2/3_capins_587bla and pSB1C3_001_RC_IRCK_P5tataless clone1
1.12.10 Cloning of VP2/3_BAP_HSPG-KO and VP2/3_His_HSPG-KO into pCerulean_Zegfr:1907_Middlelinker and VP2/3_His_HSPG-KO into pCerulean_VP1up_NLS_mVenus
September
124. labday 19.09.2010
Continuation with Colony-PCR of pSB1C3_CD and ViralBrick motif Z34C
Investigator: Bea
Comment: Since the colony PCR of the last try did not work exactly the way we expected it, another colony PCR approach with new primers was conducted.
Loading plan:
upper lanes
M   CD1  CD2  CD3  CD4  CD5  CD6  CD7  CD8  CD9  CD10   +ctrl(CFP) +ctrl(BAP) -ctrl(pAAV_MCS)   11Q1  12Q1  13Q1 13Q2  M
lower lanes
M   11T41  11T42  11T43  11T44    12T41  12T42  12T43  12T44    13T41  13T42  13T43  13T44    14T41  14T42 14T43  14T44  M
Expected sizes of the PCR products are:
Cytosine deaminase (CD) = 1300bp
Z34C loops insertion motif = 220bp
Positive control 1 = CFP = 750bp
Positive control 2 = BAP_587 = 170bp
Results:
The Colony PCR of the Cytosine Deaminase (CD) did not work out. The detectable bands at around 700 bp correspond to CFP. Therefore, BioBrick production of the CD must be repeated. The following three bands represent the controls. Three different controls were used. The first corresponds to pSB1C3_CFP, the second to BAP_587 and third control is the negative control. Unfortunately now bands can be detected in the postive controls. A faint band can be seen in the negative control which is the only lane in which no band should be detectable.
The following bands after the three controls correspond to the approaches of the Z34C Viralbrick production. Some bands in the lower and the upper lanes show positive results. The corresponding inoculated overnight culture were centrifuged and prepared in order to perform a Mini-Prep tomorrow.
Comment: hmm, seems that the CD-assambly didn't work at all.. Should i try it again or are we gonna skip it for good due to other priorities?(kira)
No, we cannot skip it. Can we change something else?? Any ideas what we can alter in the protocol? Because problem is that we are still waiting for the tk... so we should aswell focus on the other prodrug activating enzmye!! (Bea)
Testtransduction of the final modified capsid coding constructs
Investigator: Adrian
Aim of the experiment:
During the project 22 silent nucleotide exchange mutations were introduced into the capsid coding construct to make it compatible with the RFC standards and to have single cutting restriction enzymes flanking the 453 and the 587 loop sequence. Two point mutations had to be dismissed, because either a first test transduction showed that the construct was not working anymore or the insertion of the synthesized gene posed serious problems, because the restriction enzyme did not work.
Finally, we have a construct that is shown to produce infectious particles comparable to the current AAV systems and carries 20 point mutations. Now we can announce that the Adeno-associated Virus is compatible to the RFC standard and the idea to replace the loop sequences via ViralBricks works!
Impression from the moment:
The construct with the insertion of the Rep and the Cap synthesis worked! Reintroduction of the KpnI restriction site recovered the functionality of the viral genome!
Miniprep of serveral constructs
Investigator: Stefan
Glycerol stocks were prepared:
B430 = pCerulean_ZEGFR:1907_Longlinker_VP2/3
B431 = pCerulean_CFP_Middlelinker_VP2/3_insCap
B432 = pCerulean_ZEGFR:1907_Shortlinker_VP2/3_insCap
Mini-Prep was performed according to the standard protocol
P527 = pCerulean_ZEGFR:1907_Longlinker_VP2/3 c = 230,61 ng/µl
P528 = pCerulean_CFP_Middlelinker_VP2/3_insCap c = 290,81 ng/µl
P529 = pCerulean_ZEGFR:1907_Shortlinker_VP2/3_insCap c = 269,39 ng/µl
Several other constructs were preped, but need to be confirmed before being added to plasmid/glycerol stock.
125. labday 20.09.2010
Picking the clones and preparing the over-night culture
Investigator: Kira
VP2/3_Gsat-linker as well as VP2/3_cap_Gsat-linker plates contain colonies, thus 3 clones from each plate were inoculated into 5 ml DYT+ 5ul Chloramphenicol and incubated @ 37 C
Minipreps and Sequencing of ViralBrick clones
Investigator: Achim
The following clones were prepped:
Quickligation:
11.1, c = 85,41 ng/µl
13.1, c = 96,93 ng/µl
T4-Ligation:
11.4, c = 76,42 ng/µl
12.1, c = 80,36 ng/µl
12.4, c = 85,43 ng/µl
13.1, c = 90,06 ng/µl
13.2, c = 80,87 ng/µl
13.4, c = 76,62 ng/µl
14.1, c = 76,50 ng/µl
14.2, c = 98,52 ng/µl
Clones 11.4, 12.1, 13.2, 14.2 (T4) were sent for sequencing.
Cloning of pCerulean_Affibody_VP2/3 and pSB1C3_Affibody_middlelinker_GFP_his
Investigator: Jessica
Comment:Affibody_VP2/3 will be cloned in pCerulean and Affibody_middlelinker will be cloned together with GFP_His in pSB1C3
P407= pCerulean_CFP_middlelinker c=482,76 ng/µl
P516= pSB1C3_ZEGFR:1907_VP2/3 clone 1 c= 191,6 ng/µl
P290= pSB1C3_Affibody_Middlelinker clone 1 c= 227,4 ng/µl
P518= pSB1C3_GFP_RFC25_upper band clone 1 c= 138,3 ng/µl
P520= pSB1C3_GFP_RFC25_lower band clone 1 c= 141,6 ng/µl
Digestion:
components
P407
P516
P290
P518
P520
DNA
3,5
10
7
10
10
BSA (10x)
2
2
2
2
2
Buffer 4 (10x)
2
2
2
2
2
EcoRI
0,8
0,8
-
-
-
AgeI
0,8
0,8
0,8
-
-
NgoMIV
-
-
-
0,8
0,8
SpeI
-
-
0,8
0,8
0,8
H2O
10,9
4,4
7,4
4,4
4,4
Total volume
20
20
20
20
20
1,0 g Agarose,100 ml TAE (1%), 6 µl GELRED , at 120 Volt
P516, P518, P520 will be repeated tomorrow (no DNA after gel-ex)
Gelextraction:
The gelextraction was performed according to the standard protocol. DNA concentration of the extracts:
P407  c=  6,5 ng/µl
P516  c=  -   ng/µl
P290  c=  7,3 ng/µl
P518  c=  -   ng/µl
P520  c=  -   ng/µl
stands in 4°C room for comtinuation tomorrow
Cloning of P5_TATAless downstream of pSB1C3_001_RC_insrepcap_KpnI_back and P5 upstream of pSB1C3_001_RC_insrepcap_KpnI_back
Investigator: Anissa
Digestion of the constructs:
P320 = pSB1C3_001_RC_insrepcap_KpnI_back c=408 ng/µL
P320 = pSB1C3_001_RC_insrepcap_KpnI_back c=408 ng/µL
P180 = pSB1C3_pTAV2 (P5)clone 1 c=108,8 ng/µL
P184 = pSB1C3_pAAV_RC (P5TATAless)clone 3 c=176,9 ng/µL
Components
vP320 upstream /µL
vP180 upstream/µL
vP320 downsteam /µL
vP184 downsteam/µL
DNA
2,5
13,8
2,5
8,5
BSA (10x)
1,5
2
1,5
1,5
Buffer no. 4 (10x)
1,5
2
1,5
1,5
Enzyme 1
XbaI 1
SpeI 1
SpeI 1
XbaI 1
Enzyme 2
EcoRI 1
EcoRI 1
PstI 1
PstI 1
H2O
7,5
0,2
7,5
1,5
Total volume
15
20
15
15
Loading plan:
M    P320(upstream)    P320(downstream)    P180    P184
Results:
After gel extraction has been performed, the ligation was carried out.
Ligation:
v=P320_up =7,37µL
v=P180_up =0,63µL
v=P320_down =7µL
v=P184_down =1µL
The ligation mix was transformed into XL1-Blue cells and plated on agar plates containing chloramphenicol.
Next steps:
Picking clones and perform Mini-Prep.
Test digestion of VP2-N-terminal fusion constructs
Investigator: Hanna
Comment: On Satureday a colony PCR was performed of:
1. pCerulean_Zegfr:1907_ShortLinker_VP2/3 (P530)
2. pCerulean_Zegfr:1907_SEG_VP2/3 (P531)
3. pCerulean_CFP_MiddleLinker_VP2/3 (P532)
4. pCerulean_6xHis_MiddleLinker_VP2/3 (P533)
5. pCerulean_6xHis_MiddleLinker_VP2/3_insCap (P534)
6. pCerulean_Zegfr:1907_MiddleLinker_VP2/3_insCap (P535)
Because all constructs (4 clones of each approach), including the negative control (pAAV_RFC25), showed positive results, one test digestion of each construct was performed.
Digestion
components
Components Master Mix /µl
BSA (10x)
7
Buffer 4 (10x)
7
EcoRI
3.5
SpeI
3.5
H2O
35
--> 2 µL DNA + 8 µL Master Mix
Incubation: 1.5 h
Agarose-Gel:
0.4 g Agarose, 50 mL TAE (0.8 %), 3 µL GELRED, at 100 Volt, running time: 50 minutes
Sample
Sample/µl]
Loading dye (6x)/µl
Expected size 1
Expected size 2
P 530
10 µl
2 µl
2710 bp
3937 bp
P 531
10 µl
2 µl
2710 bp
3937 bp
P 532
10 µl
2 µl
2710 bp
3937 bp
P 533
10 µl
2 µl
2710 bp
3937 bp
P 534
10 µl
2 µl
2710 bp
3937 bp
P 535
10 µl
2 µl
2710 bp
3937 bp
Marker: GeneRuler ladder mix
Marker /µL
Sample P530 /µl
Sample P531 /µl
Sample P532 /µl
Sample P533 /µl
Sample P534 /µl
Sample P535 /µl
Lane
4
12
12
12
12
12
12
Test Digestion N-terminal VP2 Fusion
Comment: Test digestion delivered positive results: Expected fragment size after VP2/3 fusion to Zegfr:1907_Linker: 2180 bp - compared to failed VP2/3 insertion: ~ 220 bp.
Expected fragment size after VP2/3 fusion to CFP_MiddleLinker: 2710 bp - compared to failed VP2/3 insertion: ~ 760 bp.
pCerulean_ZEGFR:1907_Longlinker_VP2/3
pCerulean_CFP_Middlelinker_VP2/3_insCap
pCerulean_ZEGFR:1907_Shortlinker_VP2/3_insCap
pCerulean_ZEGFR:1907_Shortlinker_VP2/3
pCerulean_ZEGFR:1907_SEG_VP2/3
pCerulean_CFP_MiddleLinker_VP2/3
pCerulean_6xHis_MiddleLinker_VP2/3
pCerulean_6xHis_MiddleLinker_VP2/3_insCap
pCerulean_ZEGFR:1907_MiddleLinker_VP2/3_insCap
will be sent for sequencing to GATC.
BioBrick production: PCR of pAAV_RC_InsRepCap_KpnIback_VP1-ko and VP2-ko
Investigator: Stefan
DNA samples were diluted 1:1000
P455 = pAAV_RC_1.2 SDM SalI
P463 = pAAV_RC_CapIns_prepSDM clone 3
PCR progam:
Ingredients
Volume P455 / µl
Volume P463 / µl
5X Phusion HF buffer
10
10
10 mM dNTP mix
1
1
forward primer: O93
2,5
2,5
reverse primer: O115
2,5
2,5
DNA Template
3
3
DMSO
2
2
Phusion Polymerase
0,5
0,5
H2O
28,5
28,5
Total volume
50
50
PCR program:
Cycles
Temperature / °C
Time / s
1
98
60
2 ( step 2-4: 8x)
98
15
3
59
25
4
72
75
5 (step 5-6: 17x)
98
15
6
72
85
7
72
300
Hold
4
4
Gel:
0,5 g agarose (1%), 50 ml TAE, 3µl GELRED running time: 45 minutes at 110V
Gel-Extraction:
Gel-extraction was performed according to standard protocol. The complete amount eluted was used for PCR digestion.
Digestion of PCR product:
Components
PCR products Volume/µL
DNA
30
BSA (10x)
4
Buffer no. 4 (10x)
4
EcoRI
1
SpeI
1
H2O
-
Total volume
40
Digestion was performed at 37 °C for 2 hours.
PCR-Purification:
Gel-extraction was performed according to standard protocol.
c (P455) = 13,5 ng/µl
c (P463) =  ng/µl
Digestion of plasmid backbone:
Plasmid used: pSB1C3_VCK_Bla (P320)
c (pSB1C3_VCK_Bla) = 408,0 ng/ µl
Components
vector Volume/µL
DNA
3
BSA (10x)
2
Buffer no. 4 (10x)
2
EcoRI
1
SpeI
1
H2O
11
Total volume
20
Digestion was performed at 37 °C for 2 hours.
Gel:
0,5 g agarose (1%), 50 ml TAE, 3µl GELRED running time: 50 minutes at 120V
Gel-Extraction:
Gel-extraction was performed according to standard protocol.
c(P320) = 0,8 ng/µl
Ligation:
1 µl T4 DNA ligase
1 µl 10x Buffer
8 µl DNA mix (vector + insert)
Amounts of DNA used:
c(P320) = 7,73 µl
c(P455) = 0,27 µl
c(P320) = 7,19 µl
c(P463) = 0,81 µl
Incubation time: 45 minutes at room temperature.
Transformation:
Transformation was performed according to standard protocol using XL1b cells and Chloramphenicol as antibiotic.
2x repetition of CD biobrick
Investigator: Kira
Ingredients
CD sample
5X Phusion HF buffer
10 µl
10 mM dNTP mix
1µl
forward primer: O158
2,5µl
reverse primer: O159
2,5 µl
DNA Template (1:100 dil)
0,5 µl
DMSO
0 µl
Phusion Polymerase
0,5 µl
H2O
33µl
Total volume
50 µl
PCR program:
Cycles
Temperature
Time
98°C
1
10x
98°C
15"
58°C
25"
72°C
40"
17x
98°C
15"
65°C
25"
72°C
40"
1x
72°C
5'
Hold 4°C
Digestion of plasmid backbone:
c (pSB1C3) = 151, 1 ng/ µl
Components
vector Volume/µL
DNA 1 µg
6,0 µl
BSA (100x)
0,2 µl
Buffer no. 4 (10x)
2,0 µl
Enzyme 1 XbaI
0,5 µl
Enzyme 2 AgeI HF
0,5 µl
H2O
10,8 µl
Total volume
20
incubation @ 37 C for approx. 6 h
1% agarose gel
According to the gel results, PCR was repeated twice.. But both tries revealed the same unexpected results. Insert from the last cloning procedure was found and used for the ligation and further transformation.
Ligation
T4 ligase was used
1 ul T4 Buffer
1 ul T4 Ligase
8 ul (0,9 ul vector+ 7,1 ul insert) DNA-mix
incubation @ RT for 45 min
Transformation was performed according to the standard protocol and the cells were spread on the agar plate containing Chloramphenicol.
126. labday 21.09.2010
Trafo evalutation of CD biobrick plate
Investigator: Kira
the agar plate was checked at approx. 4 pm and there were no colonies visible. Taking in account that the plate was incubated from 10.30 pm till 8 am @ 37C due to the electricity failure, the plate was put back @ 37C at 4.30 pm
Comment: the plate was picked this morning by a lab member and put in the coldroom
Continuation of cloning of pCerulean_Affibody_VP2/3 and pSB1C3_Affibody_middlelinker_GFP_his
Investigator: Jessica
Comment:Affibody_VP2/3 will be cloned in pCerulean and Affibody_middlelinker will be cloned together with GFP_His in pSB1C3
P407= pCerulean_CFP_middlelinker c=482,76 ng/µl
P516= pSB1C3_ZEGFR:1907_VP2/3 clone 1 c= 191,6 ng/µl
P290= pSB1C3_Affibody_Middlelinker clone 1 c= 227,4 ng/µl
P518= pSB1C3_GFP_RFC25_upper band clone 1 c= 138,3 ng/µl
P520= pSB1C3_GFP_RFC25_lower band clone 1 c= 141,6 ng/µl
Digestion:
components
P516
P518
P520
DNA
10
10
10
BSA (10x)
2
2
2
Buffer 4 (10x)
2
2
2
EcoRI
0,8
-
-
AgeI
0,8
-
-
MscI
0,8
-
-
NgoMIV
-
0,8
0,8
SpeI
-
0,8
0,8
H2O
3,6
4,4
4,4
Total volume
20
20
20
0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 120 Volt
Gelextraction:
The gelextraction was performed according to the standard protocol. DNA concentration of the extracts:
P516=  c=  ng/µl
P518=  c=  ng/µl
P520=  c=  ng/µl
T4 Ligation:
The  Ligation was performed as following:
Vector Volume:  µl
Insert Volume:  µl
1µl T4-Ligase buffer (10x)
8µl (Vector + Insert) mix
1µl T4-Ligase
Incubating for 40 minutes.
Transformation:
Trafo was performed according to the standard protocol (XL1blue). The cells were plated on a agar plate with Kana/Cm
Sequencing result of pSB1C3_EGFP_His upper and (!) lower band
Investigator: Jessica
1. pSB1C3_EGFP_His suffix:
2. pSB1C3_EGFP_His prefix:
results look in both plasmids and are used for cloning of pSB1C3_Affibody_middlelinker_GFP_His
Midi-Prep of pHelper & pSB1C3_leftITR_CMV_beta-globin_mVenus_hGH_rightITR
Investigators: Chris W.
Midi-Preps of pHelper=P540 and pSB1C3_leftITR_CMV_beta-globin_mVenus_hGH_rightITR=P541
The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
plasmid-no.
P540
P541
concentration (ng/µl)
480,45
1264,20
New aliquots (60µl) of XL1blue can be used !  (Jessica)
127. labday 22.09.2010
Sequencing results of ViralBrick ligations using dephosphorylated vector
Investigator: Achim
Sequencing confirmed a correct sequence for clone 11.4 (453 Z34C). Clone 12.1 (587 Z34C)has several missing bases at the 3' end of the insert, clone 13.2 has an a -> c mutation and a c insertion in the insert (although the a->c might be a sequencing error.). Clone 14.2 contained a c deletion.
We have several new approaches:
-Correction of the mutations via PCR: We'll use the hybridisation oligo for the side of the insert containing the mutations, the rfc 25 primer on the other side. The amplified insert will be ligated into dephosphorized vector.
-As a backup, SDM-primers will be ordered and should be available next week.
-Also, a new colony pcr with several clones from the last ligation plates will be carried out.
-clone 13.4 (already prepped) will be sent for sequencing
Cloning pSB1C3_Viral Brick 587KO-empty (B274/P542) into pSB1C3_VP2/3_Capins (B438/P514)
Investigator Patrick
After an over night digestion with BamHI and PvuII at 37°C ....
My dear friend - what do you mean with "..."?? overnight digest at which temperature?? Did you look it up whether the enzymes show star activity at longer digestions or not?? ;-)
The Gelextraction with P514 was performed according to the standard protocol yielding 16,9 ng/ml.
The P542 insert should be 48 bp long and therefore was extracted with a QIAGEN kit able to extract even very small DNA fragments (40 bp) (QiaEX II Gel Extraction Kit (150)).
Expected size of the fragments:
P542: 2112 & 48 bp
P514: 3956 & 48 bp
The 48 bp band of P514 can't be seen clearly on the picture due to the bad resolution.
Ligation with T4 DNA ligase: 1 µl buffer (10x), 1 µl T4 DNA ligase, 7 µl Vector, 1 µl Insert, 70 minutes.
The transformation with XL1B was performed according to the standard protocol and plated: pSB1C3_VP2/3_Capins_587KO-empty
Cellculture: Serum-free cells
Investigator Patrick
We want our virus production to be serum-free to simplify the following purification.
15 ml cell-medium, 25% serum-free: 3,75 ml serum-free DMEM + 11,25 ml DMEM
15 ml cell-medium, 50% serum-free: 7,5 ml serum-free DMEM + 7,5 DMEM
PCR of P40
Investigator: Jessica
DNA sample P432 was diluted 1:1000
Ingredients
Volume / µl
5X Phusion HF buffer
10
10 mM dNTP mix
1
forward primer: O152
2,5
reverse primer: O188
2,5
DNA Template
3,0
DMSO
-
Phusion Polymerase
0,5
H2O
31
Total volume
50,5
PCR program:
Cycles
Temperature / °C
Time / s
98
60
8x
98
15
61
25
72
8
17x
98
15
69
25
72
8
1x
72
300
Hold
4
Digestion of plasmid backbone:
Plasmid used: pSB1C3_CFP (P51.2)
c (pSB1C3_CFP) = 151, 1 ng/ µl
Components
vector Volume/µL
DNA
10
BSA (10x)
2
Buffer no. 4 (10x)
2
XbaI
1
PstI
1
H2O
4
Total volume
20
incubation @ 37 C for approx. 2 h
Comment:
Digestion of PCR product:
Components
PCR products Volume/µL
DNA
40
BSA (10x)
6
Buffer no. 4 (10x)
6
PstI
2
XbaI
2
H2O
4
Total volume
60
Digestion was performed at 37 °C for 2 hours.
PCR-Purification:
c (P432) = 31,8ng/µl
Gel-Extraction:
Gel-extraction was performed according to standard protocol.
c(P51.2) = 3,7ng/µl
Ligation:
1 µl T4 DNA ligase
1 µl 10x Buffer
8 µl DNA mix (vector + insert)
Amounts of DNA used:
(P51.2) = 7,74 µl
(P432) =  0,26µl
Incubation time: 45 minutes at room temperature.
Transformation:
Transformation was performed according to standard protocol using XL1b cells and Chloramphenicol as antibiotic.
Mini-Prep of pSB1C3_001_RC_InsRepCap_KpnIback_VP1-ko
Investigator: Stefan
Comment:
Glycerol stocks were prepared:
B463 = pSB1C3_001_RC_InsRepCap_KpnIback_VP1-ko_P5tataless clone 1
B464 = pSB1C3_001_RC_InsRepCap_KpnIback_VP1-ko_P5tataless clone 2
Mini-Prep was performed according to standard protocol:
P548 = pSB1C3_001_RC_InsRepCap_KpnIback_VP1-ko_P5tataless clone 1 c = 435,82 ng/µl
P549 = pSB1C3_001_RC_InsRepCap_KpnIback_VP1-ko_P5tataless clone 2 c = 409,21 ng/µl
Cloning of CMV promotor into pSB1C3_VP123_inscap
Investigator: Anna
Vector name: pSB1C3_VP123_inscap P489
Insert name: pSB1C3_CMV P193
new vector name: pSB1C3_CMV_VP123_capins P580
buffer used: 4
DNA concentration (vector): 350,0 ng/µl ; DNA concentration (insert): 333,5 µg/µl
components
volume of pSB1C3_VP123_capins /µl
volume of pSB1C3_CMV /µl
DNA
3
4,5
BSA (10x)
1,5
1,5
Buffer 4 (10x)
1,5
1,5
Enzyme I
EcoI HF 1
EcoI HF 1
Enzyme II
XbaI 1
SpeI 1
H2O
7
5,5
Total volume (e.g. 15,20,25,30 µl)
15
15
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 120 Volt, running time:45
Loading plan for agarose gel:
Loading dye with SDS (6X), 3 µl
Marker used: GeneRuler ladder mix (Fermentas), 4 µl
Marker
Sample 489, 18µl
Sample 193, 18µl
Lane
1
3
5
Fragment size
4404 bp
681 bp
Gel extraction:
pSB1C3_VP123_capins_cut
CMV
DNA-concentration [ng/µl]
14,09
32,61
T4 Ligation:
Volume vector: 3,86 µl
Volume insert: 4,14 µl
Transformation:
Was performed following the standard protocol using XL1B cells.
Sequencing of Zegfr:1907-Linker Library, His-Tag and imaging appoaches
Investigator: Hanna
Comment: After several problems were managed (cloning difficulties, colony PCR troubles, interchanges of samples,...), sequences of the final constructs needed to be verified in order to test the constructs in cell culture.
Sample gallery
pCerulean_CFP_MiddleLinker_ VP2/3_InsCap (P528) forward
pCerulean_CFP_MiddleLinker_ VP2/3_InsCap (P528) reverse
pCerulean_ZEGFR:1907_ShortLinker_ VP2/3_InsCap (P527) forward
pCerulean_ZEGFR:1907_ShortLinker_ VP2/3_InsCap (P527) reverse
pCerulean_6xHis_MiddleLinker_ VP2/3 (P533) forward
pCerulean_6xHis_MiddleLinker_ VP2/3 (P533) reverse
pCerulean_6xHis_MiddleLinker_ VP2/3_insCap (P534) forward
pCerulean_6xHis_MiddleLinker_ VP2/3_insCap (P534) reverse
pCerulean_ZEGFR:1907_MiddleLinker_ VP2/3_insCap (P535) forward
pCerulean_ZEGFR:1907_MiddleLinker_ VP2/3_insCap (P535) reverse
pCerulean_ZEGFR:1907_SEG_ VP2/3_insCap (P502) reverse
pCerulean_ZEGFR:1907_LongLinker_ VP2/3_insCap (P503) reverse
pCerulean_ZEGFR:1907_MiddleLinker_ VP2/3 (P504) reverse
Comment: Sequences looked well. The 6 samples with VP2/3_insCap will be tested today in cell culture.
Mass Midi-Prep
Investigator: Chris W.
Midi-Prep of:
pSB1C3_001_RC_InsRepCap_KpnIback_P5tataless clone 1 =P544
pSB1C3_001_RC_InsRepCap_KpnIback_P5tataless clone 2 =P545
pSB1C3_001_P5_RC_InsRepCap_KpnIback clone 1 =P546
pSB1C3_001_P5_RC_InsRepCap_KpnIback clone 2 =P547
pHelper =P550
pHelper =P551
pSB1C3_litr_pTert_ßglobin_mvenus_hgh_ritr clone 1 =P552
pAAV_RC_RepCapIns_SDMKpnl_VP1ko clone 1 =P553
pAAV_RC_RepCapIns_SDMKpnl_VP2ko clone 1 =P554
pAAV_RC_RepCapIns_SDMKpnl_VP2ko clone 1 =P555
pCerulean_ZEGFR:1907_SEG_VP2/3_CapIns clone 3 =P556
pCerulean_ZEGFP:1907_LongLinker_VP2/3_CapIns clone 2 =P557
pCerulean_CFP_Middlelinker_VP2/3_insCap =P558
pCerulean_ZEGFR:1907_Shortlinker_VP2/3_insCap =P559
pCerulean_6xHis_MiddleLinker_VP2/3_insCap =P560
pCerulean_ZEGFR:1907_MiddleLinker_VP2/3_insCap =P561
pCerlulean_ZEGFR:1907_Shortlinker_VP2/3_insCap clone 4.8 =P562
The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
plasmid-no.
P544
P545
P546
P547
P550
P551
P552
P553
P554
P555
P556
P557
P558
P559
P560
P561
P562
concentration (ng/µl)
1851,91
1822,3
1783,48
2625,57
1524,63
3333,48
2641,57
2165,22
1818,23
2318,58
3255,81
2818,15
2761,32
3289,23
3134,26
3725,27
1965,28
RNA isolation and cDNA synthesis
Investigator Kira
Motivation: in order to figure out if our cell lines do express telomerase, cell pellets were used for RNA isolation.
RNeasy Kit was used for RNA isolation. The cells were harvested and pellets washed 2x with PBS.  add RLT buffer
homogenize the lysate with syringe ans blunt needle
add 1 volume of 70% EtOH
centrifuge for 15s @ 10.000 rpm
add 700 ul Buffer RW1 and centrifuge again
add 500 ul Buffer RPE and centrifuge
add 500 ul Buffer RPE and centifuge for 2 min
place the column in 1.5 ml tube and add RNase free water (50 ul)
concentrations:
c(293)= 41, 68 ng/ul
c(a431) = 135, 48 ng/ul
Gel apparatus (chamber, tray and comb) for RNA purity gel has to be prepared a day ahead with : water, SDS and NaOH
cDNA synthesis
Investigator: Kira
The yielded DNA was used for cDNA synthesis.
QuantiTect Reverse Transcription (Qiagen) reaction was performed according to the manufacturer protocol.  DNA elimination reaction mix:
components
a431/µl
293/µl
7xgDNA Wipeout Buffer
2
2
Template RNA
1,6
2
RNase-free H2O
10,4
10
Total volume
14
14
Reverse-transcription reaction mix:
components
1xreaction
MM
Quantiscript Reverse Transcriptase
1
3
5x Quantiscript RT buffer
4
12
RT primer Mix
1
3
DNA elimination reaction mix
14
-
Total volume
20
-
incubation for 15 min @ 42C
incubation for 3 min @ 95C
128. labday 23.09.2010
PCR of VB constructs to remove mutations
Investigator Achim
Colony PCR of new clones from the last ligations
Investigator Achim
Sequencing of the previously picked clones revealed that the inserts were successfully cloned into pSB1C3. Sadly, apart from 453 Z34C every other insert contained various mutations (deletions, insertions, base exchanges in different positions). The reasons are unclear, probably difficulties with the fill in reaction or incorrectly synthesized oligos. We picked more clones from the last ligations' plates and carried out a colony pcr on them:
Contuniation: Cloning pSB1C3_Viral Brick 587KO-empty (B274/P542) into pSB1C3_VP2/3_Capins (B438/P514)
Investigator Patrick
3 Clones were picked in the morning and in each case inoculated 10 ml DYT. Clone 3 didnt grew fast enough. There were not enough bacteria in the eppi even after 8 ml were centrifuged. The mini-prep of clone 1 and 2 was performed according to the standrd protocol yielding the following concentrations:
Clone 1: 36,64 ng/µl
Clone 2: 7,8 ng/µl
Therefore only clone 1 was sent for sequencing  (labelling: pat1, primer:VR2) and only this clone received a plasmid number: P569
No test-digestion was performed.
Because all medium was used to receive a sufficient amount of bacteria from clone 1 there was no cell suspension left to prepare a glycerol stock. Therefore again each clone was used to inoculated 10 ml DYT, following a mini-prep tomorrow.
Sequencing results:
Obviously there is something wrong with the backbone or the sequencing.
Mini-Prep and test digestion of pCerulean_Affibody_VP2/3, pSB1C3_Affibody_middlelinker_EGFP_His, pSB1C3_001_RC_InsRepCap_KpnIback_VP2-ko
Investigator: Jessica
Comment:
Glycerol stocks were prepared:
B465 = pCerulean_ZEGFR:1907_VP23 clone1
B466 = pCerulean_ZEGFR:1907_VP23 clone2
B467 = pSB1C3_ZEGFR:1907_middlelinker_EGFP_His clone1
B468 = pSB1C3_ZEGFR:1907_middlelinker_EGFP_His clone2
B469 = pSB1C3_001_RC_InsRepCap_KpnIbackVP2-KO clone1
B470 = pSB1C3_001_RC_InsRepCap_KpnIbackVP2-KO clone2
Mini-Prep was performed according to standard protocol:
P563 = pCerulean_ZEGFR:1907_VP23 clone1 = 227,4 ng/µl
P564 = pCerulean_ZEGFR:1907_VP23 clone2 = 337,2 ng/µl
P565 = pSB1C3_ZEGFR:1907_middlelinker_EGFP_His clone1 = 252,7 ng/µl
P566 = pSB1C3_ZEGFR:1907_middlelinker_EGFP_His clone2 = 209,5 ng/µl
P567 = pSB1C3_001_RC_InsRepCap_KpnIback_VP1-KO clone1 = 304,7 ng/µl
P568 = pSB1C3_001_RC_InsRepCap_KpnIback_VP1-KO clone2 = 383,1 ng/µl
Test digestion of
P563 = pCerulean_ZEGFR:1907_VP23 clone1
P564 = pCerulean_ZEGFR:1907_VP23 clone2
P565 = pSB1C3_ZEGFR:1907_middlelinker_EGFP_His clone1
P566 = pSB1C3_ZEGFR:1907_middlelinker_EGFP_His clone2
Components
P563-566 Volume/µL
DNA
2
BSA (10x)
1
Buffer no. 4 (10x)
1
PstI
0,5
XbaI
0,5
H2O
5
Total volume
10
Results:
P566 = P565 ; P567 = P566 - P566 is not OK!!!
P563 and P566 was sent for sequencing with primer VF2 and VF2/VR2
PCR for 587 Z34C ViralBricks
Investigators: Achim, Anna
Plasmids used as template:
12_T4_1:  c = 80,36 ng/µl
13_T4_2:  c = 80,87 ng/µl
14_T4_2:  c = 98,52 ng/µl
Primer used:
Template
Primer for
Primer rev
12_T4_1
RFC25
O146
13_T4_2
RFC25
O148
14_T4_2
O149
RFC25
Ingredients
Volume / µl
5X Phusion HF buffer
10
10 mM dNTP mix
1
forward primer: see above
2,5
reverse primer: see above
2,5
DNA Template (1:100 dilution)
1,0
DMSO
1,0
Phusion Polymerase
0,5
H2O
31,5
Total volume
50
PCR program:
Cycles
Temperature / °C
Time / s
98
60
98
15
25x
68
25
72
10
1x
72
300
Hold
4°C
Digestion of samples:
Restriction enzymes used: BamHI and PvuI.
Comment:
PCR of Vp1_ex, Vp2_ex and Vp3_ex and cloning into pSB1C3_001
Investigator: Anissa
DNA sample P72 (pkex_Vp1), P73(pkex_Vp2) and P74(pkex_Vp3) were diluted 1:100
used primers: O89 = Präfix_Vp1ex, O90= Präfix_Vp2ex, O91= Präfix_Vp2ex, O92= Suffix_Vp123ex, O120= Suffix_Vp123ex_reg
Ingredients
Volume / µl
5X Phusion HF buffer
10
10 mM dNTP mix
1
forward primer: O152
2,5
reverse primer: O188
2,5
DNA Template
2,0
DMSO
1
Phusion Polymerase
0,5
H2O
30,5
Total volume
50
PCR program:
Cycles
Temperature / °C
Time / s
1x
98
60
25x
98
15
25x
60
25
25x
72
55
1x
72
5
Hold
4
Comment:The primer O120 was firstly given to the samples. It's a different primer for the Vp123_suffix, which binds to the regulatory-sequence. Because there was no knowledge, if the Vp_ex constucts have this sequence or not, O92 was also given to the samples.
Comment:Because gel-picture showed only DNA within the gel-bags, PCR was repeated.
Digestion of plasmid backbone:
Plasmid used: pSB1C3_001 (P320)
c (pSB1C3_001) = 408 ng/ µl
Components
vector Volume/µL
DNA
2,5
BSA (10x)
1,5
Buffer no. 4 (10x)
1,5
SpeI
1
NgomIV
1
H2O
7,5
Total volume
15
incubation @ 37 C for approx. 1,5 h
Comment:Strangely, PCR-products did not run into the gel, altough the PCR was repeated, after first PCR had two reverse primers in the sample. This could be because of a complex with the polymerase.However, ligation will be continued with the DNA within the bags.
</br>
Digestion of PCR product:
Components
PCR products Volume/µL
DNA
20
BSA (10x)
3
Buffer no. 4 (10x)
3
SpeI
1
NgomIV
1
H2O
2
Total volume
30
Digestion was performed at 37 °C for 2 hours.
PCR-Purification:
was peformed according the standard-protocol.
c (P432) =-- ng/µl
Comment:THE WHOLE PCR WILL BE REPEATED ONCE AGAIN
RNA Purity Gel
Investigator: Kira
10x FA buffer: 200mM MOPS, 50mM Sodium Acetat, 10mM EDTA, RNase free water
1x FA running buffer: 10x FA buffer, 37% Formamid, RNAase free water
Agarose gel: Agarose 0,6 g; 10x FA Buffer 5ml; RNAse free water 45ml; 37% Formamid 0,9 ml
hTERT PCR Reaction
Investigator: Kira
@Kira: Do you think that we can (suggestion?) use the A431 for detection of gene expression driven by phTERT promoter?? We should keep in mind that the transduction efficieny will not be that well ;-) (Bea)
Ingredients
Volume / µl
10x PCR buffer
2
10 mM dNTP mix
0,4
50mM MgCl2
0,6
Primer 1
0,5
Primer 2
0,5
cDNA template
1
Taq Polymerase
0,1
H2O
14,9
Total volume
20
PCR program:
Cycles
Temperature / °C
Time
1x
94
3min
30x
94
45sec
51
30
72
1min 30sec
1x
72
10
Hold
4
1% agarose gel:
Evaluation: The agarose gel reveals that all samples show telomerase activity but in different levels.
Cloning of pBAD_Affibody_middlelinker_EGFP_His
Investigator: Jessica
Vector: name: pBAD_ OmpA His FluA LL Foki PGerrit
Insert: name: pSB1C3_ZEGFR:1907_middlelinker_EGFP_His P566
new vector name: pBAD_ZEGFR:1907_middlelinker_EGFP_His P
buffer used: 4
DNA concentration (vector): 97,0 ng/µl ; DNA concentration (insert): 209,5 µg/µl
components
volume of pBAD_ OmpA His FluA LL Foki  /µl
volume of pSB1C3_ZEGFR:1907_middlelinker_EGFP_His /µl
DNA
10
6
BSA (10x)
2
2
Buffer 4 (10x)
2
2
Enzyme XbaI
0,8
0,8
Enzyme AgeI
0,8
0,8
H2O
4,4
8,4
Total volume (e.g. 15,20,25,30 µl)
20
20
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 120 Volt
Loading plan for agarose gel:
Loading dye with SDS (6X), 3 µl
Marker used: GeneRuler ladder mix (Fermentas), 6 µl
Marker
Sample PGerrit, 20µl
Sample 566, 20µl
Lane
1
3
5
Fragment size
4047 bp
963 bp
Gel extraction:
pBAD
Affibody_middlelinker_EGFP_His
DNA-concentration [ng/µl]
8,3
10,2
T4 Ligation:
Volume vector:  5,06 µl
Volume insert:  2,94 µl
Transformation:
XL1B cells were used.
Picking clones of pSB1C3_CMV_VP123_inscap
Investigator: Anna
Three clones were picked (10 ml DYT, 10 µl Cm).
Cloning of ***** into pSB1C3_CFP
Investigator: Chris W.
Vector: name: pSB1C3_CFP P51,2
Insert: name: email Hybrid Oligo (O184&O185)
new vector name: pSB1C3_email
buffer used: 4
DNA concentration (vector): 151,0 ng/µl ; DNA concentration (insert): 95,3 µg/µl
Interesting, where does the promoter come from the new annotared vector?? ;-)
components
volume of pSB1C3_CFP /µl
volume of email/µl
DNA
6
15
BSA (10x)
2
2
Buffer 4 (10x)
2
2
Enzyme I
NgoMIV 1
NgoMIV 1
Enzyme II
SpeI 1
SpeI 1
H2O
8,4
-
Total volume (e.g. 15,20,25,30 µl)
20
20
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 120 Volt, running time:45
Loading plan for agarose gel:
Loading dye with SDS (6X), 4 µl
Marker used: GeneRuler ladder mix (Fermentas), 3 µl
Marker
Sample Insert, 20µl
Sample Vector, 20µl
Lane
1
3
5
Fragment size
70 bp
2070 bp
Gel extraction:
pSB1C3_CFP_cut
email
DNA-concentration [ng/µl]
7,8
27,4
T4 Ligation:
Volume vector: 8,74 µl
Volume insert: 0,26 µl
Transformation:
XL1B cells were used.
Repetition: Cloning of P5_TATAless downstream of pSB1C3_001_RC_insrepcap_KpnI_back and P5 upstream of pSB1C3_001_RC_insrepcap_KpnI_back
Investigator: Stefan
Digestion of the constructs:
P432 = pSB1C3_001_RC_insrepcap_KpnI_back c=344,5 ng/µl
P432 = pSB1C3_001_RC_insrepcap_KpnI_back c=344,5 ng/µl
P180 = pSB1C3_pTAV2 (P5)clone 1 c=108,8 ng/µl
P184 = pSB1C3_pAAV_RC (P5TATAless)clone 3 c=176,9 ng/µl
My dear friend, did you mix up the Rep-Cap constructs in the plasmid list? P432 = pAAV_RC_insRepCap_KpnI-back ;)
Components
V P432up /µL
VP180up /µL
V P432down /µL
V P184down /µL
DNA
4,5
14
4,5
11
BSA (10x)
2
2
2
2
Buffer no. 4 (10x)
2
2
2
2
Enzyme 1
XbaI 1
SpeI 1
SpeI 1
XbaI 1
Enzyme 2
EcoRI 1
EcoRI 1
PstI 1
PstI 1
H2O
9,5
9,5
-
3
Total volume
20
20
20
20
Results:
Gel-Extraction: Gel-Ex was performed according to protocol. The following concentrations were measured:
432up c= 25,73 ng/µl
180up c= 9,57 ng/µl
432down c= 27,34 ng/µl
184down c= 17,80 ng/µl
Ligation:
The following DNA mixes were used for ligation:
432up V= 6,3 µl
180up V= 1,7 µl
432down V= 6,93 µl
184down V= 1,07 µl
Transformation:
The transformation was performed according to standard protocol using BL21 cells. They were plated on agar plates containing chloramphenicol and stored in 37°C room overnight.
129. labday 24.09.2010
Sequencing results of VB 13.4
Investigator: Achim
We sent in another clone of the 587 KO Z34C motif to see if it carries the same mutations as the first one. If that had been the case, we could account the mutations to faulty oligos. However, sequencing showed that the second 587 KO Z34C clone is mutated as well, but in different positions. So we either recieved oligos containing various mutations, or the fill-in reaction was faulty in spite of the proofreading activity of the klenow fragment.
Because 587 KO Z34C is one of our more promising approaches, we sent another two clones from our second colony pcr/clone picking round for sequencing: 13.1 and 13.4
Contuniation: Cloning pSB1C3_Viral Brick 587KO-empty (B274/P542) into pSB1C3_001_VP2/3_Capins (B438/P514)
Investigator Patrick
Because of the sequencing results of clone 1 a mini-prep was performed only with clone 2 and 3 yielding the following concentrations:
pSB1C3_VP2/3_Capins_587KO-empty clone 2: 195 ng/µl
pSB1C3_VP2/3_Capins_587KO-empty clone 3: 224,74 ng/µl
Testdigestion:
pSB1C3_VP2/3_Capins_587KO-empty clone 2 (P578/B472): 3 µl Buffer 4, 0,5 µl BamHI, 0,5 µl PvuII, 22 µl DNA, 4 µl H2O
pSB1C3_VP2/3_Capins_587KO-empty clone 3 (P579/B473): 23 µl Buffer 4, 0,5 µl BamHI, 0,5 µl PvuII, 20 µl DNA, 6 µl H2O
Because somebody switched the 37°C thermoblock off i dont know the exact digestion time. Therefore i switched the thermoblock on again and waited 1 hour. I used that much DNA because the desired insert is 48 bp long.
Clone 3 was sent for sequencing.
Sequencing evaluation of VP2/3_Gsat-linker and VP2/3_cap_Gsat-linker
Investigator Kira
Comment: pSB1C3_VP2/3_cap_Gsat-linker as well as pSB1C3_VP2/3_Gsat-linker can be used for further experiments.
New CD primer ordered
Investigator Stefan
Comment: CD reverse primer was incorrect. A new primer was ordered.
Sequencing of pCerulean_Zegfr:1907_VP2/3
Investigator Hanna
Because I found out, that this plasmid was sent to GATC with the wrong sequencing primers (therefore no sequencing results were available), I sent it again with the GATC_std_CMV-F primer.
--> backbone = pCerulean and not pSB1C3!!! ;)
Repetition: PCR for biobrick production: Cloning RepCap into pSB1C3
Investigator: Stefan
Comment: Construct was cloned mixed RFC10 and RFC25. Therefore, a new approach will be performed consisting of RFC10 only.
Plasmid used:
pAAV_RC_RepCapIns_SDMKpnI (P432)  A 1:1000 dilutionof the plasmid was prepared.
PCR protocol:
Ingredients
P432 (v/µl)
5X Phusion HF buffer
10
10 mM dNTP mix
1
forward primer: O93
2,5
reverse primer: O121
2,5
DNA Template
3
DMSO
-
Phusion Polymerase
0,5
H2O
30,5
Total volume
50
PCR program:
Cycles
Temperature /°C
Time /s
1
98
60
2 (8x step 2-4)
98
15
3
59
25
4
72
70
5 (17x step 5-6)
98
15
6
72
80
71x
72
300
Hold 4°C
Digestion of the vector:
components
pSB1C3_VCK_Bla (P320) (v/µl)
DNA
3,5
BSA
2
Buffer 4 (10x)
2
SpeI
1
EcoRI
1
H2O
10,5
Total volume
20
Comment:Digestion was performed 2 hours at 37 °C.
Gel:
0,5 g Agarose,50 ml TAE (1%), 3 µl Gelred , at 110 Volt; run for ~50 minutes
Gelextraction:
The gelextraction was performed according to the standard protocol. DNA concentration of the vector:
pSB1C3 (P320): c= 8,5 ng/µl
Digestion of the PCR-products:
components
PCR product (v/µl)
DNA
30
BSA
4
Buffer 4 (10x)
4
SpeI
1
EcoRI
1
H2O
-
Total volume
40
Comment:Digestion was performed 2 hours at 37 °C.
PCR purification:
Purification was performed accoding to the standard protocol. The following concentrations were measured:
PCR of P432: c= 6,4 ng/µl
Ligation:
1 µl T4 Ligase buffer (2x) 8 µl DNA mix 1 µl T4 Ligase
Incubating for 45 minutes at room temperature.
The  Ligation was performed using the following amounts of DNA:
Vector Volume: 0,92 µl
Insert Volume: 7,08 µl
Transformation:
Trafo was performed according to the standard protocol using XL1b cells. The cells were plated on a agar plates containing chlorampenicol.
Picking clones of pSB1C3_CFP_email
Investigator: Anna
Three clones were picked (10 ml DYT, 10 µl Chloramphenicol).
MiniPreps and Test digestion of pSB1C3_CMV_VP123_capins
Investigator: Anna
Sample
Concentration [ng/µl]
pSB1C3_CMV_VP123_capins_clone1
377,29
pSB1C3_CMV_VP123_capins_clone2
369,92
pSB1C3_CMV_VP123_capins_clone3
387,57
Test digestion:
Components
Volume /µl
DNA
2
BSA
-
Buffer 4 (10x)
1
EcoRI
0,5
NgoMIV
0,5
H2O
6
Total volume
10
Expected size (CFP): 681 bp
Comment: Clone 1 was sent for sequencing with the following primers: VF2, 4000 for, 4200 rev, 2800 for and 2800 rev.
MiniPreps of 587 Z34C ViralBricks and pSB1C3_p40
Investigator: Anna
Sample
Clone
Concentration [ng/µl]
pSB1C3_587_KO_Z34C
13.1
266,97
pSB1C3_587_KO_Z34C
13.4
277,47
Comment: Both clones were sent for sequencing.
Sample
Clone
Concentration [ng/µl]
pSB1C3_p40
1
266,97
pSB1C3_p40
2
277,47
pSB1C3_p40
3
167,20
Re-Trafo of DARPin
Investigator Stefan
Comment: A Re-Trafo of the synthesized DARPin was performed using XL1b cells.
130. labday 25.09.2010
Sequencing of VB: 13.1, 13.4 (approach w. dephosphorylated vector, second clone picking round
Investigator: Achim
The two clones that were sequenced both contained a deletion in the same position in the insert. We really seem to have bad luck picking clones. Hopefully our PCR attempt works better.
Miniprep of serveral constructs
Investigator: Jessica
Glycerol stocks were prepared:
B489 = pBAD_EGFP_His clone1
B490 = pBAD_EGFP_His clone2
B491 = pBAD_EGFP_His clone3
B492 = pSB1C3_CFP_email clone 1
B493 = pSB1C3_CFP_email clone 2
B494 = pSB1C3_CFP_email clone 3
Mini-Prep was performed according to the standard protocol
P588= pBAD_EGFP_His clone1 c=71,0 ng/µl
P589 = pBAD_EGFP_His clone2 c=74,8 ng/µl
P590 = pBAD_EGFP_His clone3 c=58,3 ng/µl
P591 = pSB1C3_CFP_email clone 1 c=132,7 ng/µl
P592 = pSB1C3_CFP_email clone 2 c=168,9 ng/µl
P593 = pSB1C3_CFP_email clone 3 c=153,6 ng/µl
Test digestion:
Components
Volume P588-590 /µl
Volume P591-593 /µl
Volume P583-585 /µl
DNA
4,5
2,5
2
BSA
1
1
1
Buffer 4 (10x)
1
1
1
XbaI
0,4
0,4
0,4
PstI
0,4
0,4
0,4
H2O
2,7
4,7
5,2
Total volume
10
10
10
Expected size :  bp
Comment: test digestion of P591-593 and P583-585 (pSB1C3_P40) is just an attempt because I can't find any sequences in geneious.P583 looks well. P591, P588 and P583 will be sent for sequencing on monday.
Sequencing results of pSb1C3_Affibody_middlelinker_EGFP_His
Comment: there was anywhere a problem in the cloning process. approach will be repeated today. there is pBAD_EGFP_His, affibody_middlelinker will be cloned in, also in pSB1C3_EGFP_His
Sequencing result of pSB1C3_CMV_VP123_capins_clone1
Investigators: Volker, Anna
For test digestion go to labday 24.09.10
Comment: The following primers were used for sequencing: VF2, 4000 for, 4200 rev, 2800 for and 2800 rev. There is a point mutation in the non-coding sequence, which should not influence the performence of the construct, so it can be used for further cloning steps.
Cloning of 453 and 587 ViralBricks into pSB1C3_CMV_VP123_capins
Investigators: Achim, Anna
Vector name: pSB1C3_CMV_VP123_capins P580
Inserts: different ViralBricks (see pdf underneath), pSB1C3_VCK_Bla P320
buffer used: 4
Update: ViralBricks ready to use:
media:Freiburg10_Production of ViralBricks_new.pdf
Digestion of vector and inserts:
media:Freiburg10_Cloning of ViralBricks into pSB1C3_CMV_VP123_capins.pdf
The vector p580 was dephosphorylated.
Two gels were prepared with 1 g Agarose, 100 ml TAE (1%) and 6 µl GELRED; at 120 Volt, running time:45
Gel run:
Loading dye(6X), 6 µl
Marker used: GeneRuler ladder mix (Fermentas), 3 µl
Expected sizes of fragments: see Pdf under Gel extraction and overnight T4 Ligation
Gel extraction and overnight T4 Ligation:
media:Freiburg10_Cloning of ViralBricks into pSB1C3_CMV_VP123_capinsII.pdf
Sequencing results: pSB1C3_001_VP2/3_HSPG-KO
Investigator: Hanna
Comment: In order to knock down the natural tropism of the virus, the HSPG knock-out motif (two aa exchanges) was cloned into VP2/3. This fragment can then be used for the VP2 N-terminal fusion or VP1 insertion approaches.
Conclusion: Sequencing results looked well: The HSPG-KO ViralBrick was cloned successfully into VP2/3.
Cloning of pBAD_Affibody_middlelinker_EGFP_His and pSB1C3_Affibody_middlelinker_EGFP_His
Investigator: Jessica
Comment: There was a mistake in cloning pSB1C3_Affibody_middlelinker_EGFP_His so there is just pSB1C3_EGFP_His. in this plasmid will be cloned Affibody_middlelinker, also in pBAD_EGFP_His
Vector: name: pBAD_EGFP_His P588
Vector: name: pSB1C3_EGFP_His P565
Insert: name: pSB1C3_ZEGFR:1907_middlelinker_ P290
new vector name: pBAD_ZEGFR:1907_middlelinker_EGFP_His P
new vector name: pSb1C3_ZEGFR:1907_middlelinker_EGFP_His P
buffer used: 4
DNA concentration (vector): 71,0 ng/µl / 252,7 ng/µl; DNA concentration (insert): 227,4 µg/µl
components
volume of pBAD_EGFP_His  /µl
volume of pSB1C3_EGFP_His /µl
volume of pSB1C3_ZEGFR:1907_middlelinker /µl
DNA
15
4
7
BSA (10x)
2
2
2
Buffer 4 (10x)
2
2
2
Enzyme XbaI
0,8
0,8
0,8
Enzyme AgeI
-
-
0,8
Enzyme NgoMIV
0,8
0,8
-
H2O
-
10,4
7,4
Total volume (e.g. 15,20,25,30 µl)
20,6
20
20
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 120 Volt
Marker
P290, 20µl
P565, 20µl
P588, 20µl
Sample
4
1
2
3
Fragment size
210 bp
2800 bp
4700 bp
Gel extraction:
pBAD
pSB1C3
Affibody_middlelinker
DNA-concentration [ng/µl]
T4 Ligation pBAD:
Volume vector:  3,5 µl
Volume insert:  4,5 µl
T4 Ligation pSB1C3:
Volume vector:  2,3 µl
Volume insert:  5,7 µl
131. labday 26.09.2010
Continuation of cloning of pBAD_Affibody_middlelinker_EGFP_His and pSB1C3_Affibody_middlelinker_EGFP_His
Transformation:
BL21 cells were used - in 37°C at 8.30am
pBAD_Affibody_middlelinker_EGFP_His
pSB1C3_Affibody_middlelinker_EGFP_His
Comment:pSB1C3_Affibody_middlelinker_EGFP_His was not successful (no colonies on plate), will be repeated on monday
Picking clones of different constructs
Investigator: Anna
Clones were picked of:
pBad_Affibody_middlelinker_EGFP_His
pSB1C3_001_RC_InsRepCap_KpnIback_RFC10
Darpin (ReTrafo)
pSB1C3_453_BAP (p402, ReTrafo)
pSB1C3_587_KO_RGD (p390, ReTrafo)
pSB1C3_587_KO_His (p391, ReTrafo)
In addition, preparations for Mini-Preps of several ViralBricks were done:
B289: 587_BAP
B270: 587_KO_BAP
B271: 587_RGD
B272: 453_His
B273: 587_His
B274: 587_KO_empty
B488: 453_Z34C
Comment: Inoculating of 10 mL DYT medium containing 10µL chloramphenicol (except pBad with Amp).
There were no clones on the plate with pSB1C3_Affibody_middlelinker_EGFP_His.
Continuation of cloning of 453 and 587 inserts into pSB1C3_CMV_VP123_capins
Investigator: Achim, Anna
Transformation:
Trafo was performed according to the standard protocol using XL1b cells. The cells were plated on agar plates containing chlorampenicol and stored in the 37°C room.
Colony PCR of remaining 587 constructs
Investigator: Achim, Anna
Comment: New clones of the remaining 587 constructs were picked (from new cloning approach with T4 Ligation and dephosphorylated vector, see labdays 15.09. and 22.09.).
Constructs:
12: 587_Z34C
13: 587_KO_Z34C
14: 587_Z34C_KO_Spacer
Components
Volume /µl
10x Standard Taq Buffer (with MgCl2)
1
MgCl2 (50mM)
0,1
dNTPs
0,2
Primer 1: RFC25 for
0,2
Primer 2: RFC25 rev
0,2
Taq Polymerase
0,05
H2O
8,25
Total volume
10
Cycles
Temperature
Time
95°C
6 min
30x
95°C
25 sec
(Tm -3°C)
60°C
30 sec
1 min/kb
68°C
55 sec
1x
68°C
5 min
Hold 4°C
Clone 13.3, 14.1 and 14.3 will be sent for sequencing.
Sequencing results: pCerulean_Zegfr:1907_VP2/3
Investigator: Hanna
Sequencing results revealed that VP2/3 (unmodified!) was successfully fused to the Affibody (without linker).
Next step: Fusion of VP2/3-HSPG-KO to the C-terminus of the Affibody.
N-terminal VP2-Fusion & VP1 Insertion: Fusion of VP2/3_HSPG-KO
Investigator: Hanna
Comment: In order to target tumor cells with our AAV2 two VP2 fusion strategies were figuered out: The so called VP2-fusion and VP1 insertions. These approaches were successfully cloned and are now tested in cell culture. In order to prevent targeting of other (healthy) cells, the natural tropism of the virus needs to be knocked down. For this purpose a ViralBrick, "587-KO_empty", was designed and cloned into pSB1C3_001_VP2/3_Capins.
Today this VP2/3_HSPG-KO sequence will be fused to the N-terminus of the Affibody, CFP/mVenus and His-Tag of both, the N-terminal fusion to VP2 and the VP1 insertion, approach.
DIGESTION:
1. pCerulean_CFP_MiddleLinker (P407)
2. pCerulean_ZEGFR:1907_MiddleLinker (P408)
3. pCerulean_ZEGFR:1907_SEG (P409)
4. pCerulean_ZEGFR:1907_ShortLinker (P410)
5. pCerulean_ZEGFR:1907_LongLinker (P412)
6. pCerulean_6xHis_MiddleLinker (P374)
7. pCerulean_VP1up_NLS_ZEGFR:1907 (P422)
8. pCerulean_VP1up_NLS_6xHis (P424)
9. pCerulean_VP1up_NLS_mVenus (P426)
10.1 pSB1C3_001_Vp2/3_HSPG-KO (P579)
10.2 pSB1C3_001_Vp2/3_HSPG-KO (P579)
Incubation: 2 hours, 37°C.
Agarose gel: 0.8%
Sample
Sample/µl]
Loading dye (6x)/µl
Expected size
1
20 µl
4 µl
4710 bp
2
20 µl
4 µl
4169 bp
3
20 µl
4 µl
4254 bp
4
20 µl
4 µl
4158 bp
5
20 µl
4 µl
4181 bp
6
20 µl
4 µl
2129 bp
7
20 µl
4 µl
4583 bp
8
20 µl
4 µl
4428 bp
9
20 µl
4 µl
5124 bp
10.1
20 µl
4 µl
1937 bp
10.2
20 µl
4 µl
1937 bp
Marker: GeneRuler ladder mix
Marker /µL
Sample 1 /µl
Sample 2 /µl
Sample 3 /µl
Sample 4 /µl
Sample 5 /µl
Sample 6 /µl
Sample 7 /µl
Sample 8 /µl
Sample 9 /µl
Sample 10.1 /µl
Sample 10.2 /µl
Lane
4.5
24
24
24
24
24
24
24
24
24
36
36
LIGATION:
TRAFO:
Trafo was performed following our standard protocol. Cells: XL1b, DNA: 2µL.
Plates will be stored over night @ the 37°C room.
Next step: Picking 2 clones of each construct, colony PCR!
132. labday 27.09.2010
Sent for sequencing
Investigator: Jessica
P591 pSB1C3_email: VR2
P583 pSB1C3_P40: VR2
P588 pBAD_EGFP_His: pBAD-FP
P596 pBAD_Affibody_middlelinker_EGFP_His: pBAD-FP / pTrcHis-RP
Picking clones of VP1 insertion and VP2 fusion HSPG-KO approaches
Investigator: Hanna
2 clones were picked of each ligation approach = 18 falcons (Kanamycin).
To do: Mini-Prep and test digestion - sequencing.
MiniPreps of different constructs
Investigator: Anna
The following constructs were prepped according to the standard protocol:
pSB1C3_453_BAP (p402, ReTrafo)
pSB1C3_587_KO_RGD (p390, ReTrafo)
pSB1C3_587_KO_His (p391, ReTrafo)
B289: 587_BAP
B290: 453_RGD
B270: 587_KO_BAP
B271: 587_RGD
B272: 453_His
B273: 587_His
B274: 587_KO_empty
B488: 453_Z34C
B473:
Miniprep of serveral constructs
Investigator: Stefan
Glycerol stocks were prepared:
B489 = pSB1C3_001_RC_InsRepCap_KpnIback_RFC10  clone 1
B490 = pSB1C3_001_RC_InsRepCap_KpnIback_RFC10  clone 2
B491 = pBAD_Affibody_middlelinker_EGFP_His clone 1
B492 = pBAD_Affibody_middlelinker_EGFP_His clone 2
B493 = pBAD_Affibody_middlelinker_EGFP_His clone 3
Mini-Prep was performed according to the standard protocol
P594= pSB1C3_001_RC_InsRepCap_KpnIback_RFC10  clone 1 c= 140,9 ng/µl
P595 = pSB1C3_001_RC_InsRepCap_KpnIback_RFC10  clone 2 c= 117,5 ng/µl
P596 = pBAD_Affibody_middlelinker_EGFP_His clone 1 c= 101,4 ng/µl
P597 = pBAD_Affibody_middlelinker_EGFP_His clone 2 c= 92,9 ng/µl
P598 = pBAD_Affibody_middlelinker_EGFP_His clone 3 c= 92,1 ng/µl
Test digestion:
Components
Volume P594+P595 /µl
Volume P596-598 /µl
DNA
2,5
2,5|
BSA
1
1
Buffer 4 (10x)
1
1
XbaI
0,4
0,4
PstI
0,4
0,4
H2O
4,7
4,7
Total volume
10
10
Comment:
Re-Trafo of pSB1C3_pTAV (P5)
Investigator: Stefan
Comment:After inocculating from glycerol-stock, the plasmid showed strange results during digestion. Therefore a re-trafo will have to be performed to replace the stock.
Repetition: Cloning of P5_TATAless downstream of pSB1C3_001_RC_insrepcap_KpnI_back and P5 upstream of pSB1C3_001_RC_insrepcap_KpnI_back
Investigator: Stefan
Digestion of the constructs:
P594 = pSB1C3_001_RC_insrepcap_KpnI_back c=344,5 ng/µl
P594 = pSB1C3_001_RC_insrepcap_KpnI_back c=344,5 ng/µl
P586 = pSB1C3_pTAV2 (P5)clone 1 c=137,37 ng/µl
P587 = pSB1C3_pAAV_RC (P5TATAless)clone 3 c=191,61 ng/µl
Components
V P594up /µL
VP586up /µL
V P594down /µL
V P587down /µL
DNA
10
14
10
13
BSA (10x)
2
2
2
2
Buffer no. 4 (10x)
2
2
2
2
Enzyme 1
XbaI 1
SpeI 1
SpeI 1
XbaI 1
Enzyme 2
EcoRI 1
EcoRI 1
PstI 1
PstI 1
H2O
9,5
9,5
-
3
Total volume
20
20
20
20
Results:
Comment:P5 and P5TATAless do not fit the expected results. For P5TATAless, a new digestion using the P184 plasmid was performed. For P5 no old plasmid stock was available, therefore a re-trafo will have to be performed.
Components
V P184down /µL
DNA
10
BSA (10x)
2
Buffer no. 4 (10x)
2
XbaI
1
PstI
1
H2O
4
Total volume
20
Comment:No gel picture available because computer crashed completely...
Gel-Extraction:
Gel-Ex was performed according to protocol. The following concentrations were measured:
594down c= 29,9 ng/µl
184down c= 3,8 ng/µl
Ligation:
The following DNA mixes were used for ligation:
432down V= 4,48 µl
184down V= 3,52 µl
Transformation:
The transformation was performed according to standard protocol using BL21 cells. They were plated on agar plates containing chloramphenicol and stored in 37°C room overnight.
Picking clones of VP1 insertion and VP2 fusion HSPG-KO approaches
Investigator: Hanna
2 clones were picked of each ligation approach = 18 falcons (Kanamycin).
To do: Mini-Prep and test digestion - sequencing.
Picking clones of VP1 insertion and VP2 fusion HSPG-KO approaches
Investigator: Hanna
2 clones were picked of each ligation approach = 18 falcons (Kanamycin).
To do: Mini-Prep and test digestion - sequencing.
133. labday 28.09.2010
Cloning of the ViralBricks BAP-HSPG-ko and His-HSPG-ko into pSB1C3_VP2/3
Investigator: Bea
Comment: After discussing the finals constructs, we decided to clone HSPG-KO with BAP and His motifs into the N-terminal approaches which means that the plasmids pCerulean_Targeting_VP2/3 contains His and BAP ViralBricks integrated in the 587 loop, respectively. Therefore I started the first cloning step in order to obtain the fully assembled targeting construct which can be cotransfected with the plasmids containing rep and cap genes (with or wihtout HSPG-ko)
Digestion of the constructs:
P604 = pSB1C3_001_587-ko_BAP
P606 = pSB1C3_001_587-ko_His
P525 = pSB1C3_001_VP2/3capins
Components
vP606 /µL
vP525 upstream/µL
vP604 /µL
DNA
21
6
18
BSA (10x)
4
2
4
Buffer no. 4 (10x)
4
2
4
Enzyme 1
BamHI
BamHI
BamHI
Enzyme 2
SpeI
PvuII
PvuII
H2O
9
8
12
Total volume
40
20
40
Loading plan of a 1% agarose gel:
M    P525
Loading plan of a 2% agarose gel:
M    P604    P606
Expected sizes of the digested products are:
P525= 48,3800 bp
P604= 2216, 81bp
P606 = 2216, 108bp
Results:
After gel extraction has been performed, the 2µL of each ligated plasmids were transformed into BL-21 cells.
Test digestion: ViralBricks in pSB1C3_CMV_VP123_capins
Investigator: Achim
We digested preps of all our 453 Inserts with BamHI & SspI and all our 587 Inserts with SalI and PvuII. That way we get bands of ~50 bp difference between our loop inserts and the wt loop region. (Eg. 453 BAP: 455 with insert, 396 without insert.)
Nomenclature as in previous VB assays.
Clone 11.2 is missing , I forgot to inoculate it.
P580 (pSB1C3_CMV_VP123_capins)
The framed samples were midiprepped and will be used for transfection
Sequencing result of pSB1C3_email and pBAD_Affibody_middlelinker_EGFP_His
Investigator: Jessica
1. pSB1C3_email:
2. pBAD_Affibody_middlelinker_EGFP_His:
Preparation of PBS Buffer and PBS Buffer with Imidazole
Investigator: Jessica
The following protocol was used:
dissolve the following in 800ml ddH2O:
8g of NaCl
0.2g of KCl
1.44g of Na2HP04 (1.69 because of the crystal water in the reagent)
0.24 g of KH2PO4
adjust to pH 7.4
adjust to 1000ml
steril filtrate the solution
dissolve the following in 180ml ddH2O:
6,4g of NaCl
0,16g of KCl
1,15g of Na2HP04 (1.69 because of the crystal water in the reagent)
0,19 g of KH2PO4
3,4g of Imidazole
adjust to pH 7.4
adjust to 200ml
steril filtrate the solution
ViralBrick Sequencing results: pSB1C3_587_KO_Z34C and pSB1C3_587_KO_Z34C_Spacer
Investigator: Achim
Sequencing showed that the PCR reaction we designed to remove the mutations in the z34c insert were successful! Both  pSB1C3_587_KO_Z34C and pSB1C3_587_KO_Z34C_Spacer are correct. We hadn't sent the 587_Z34C for sequencing because the colony PCR hadn't shown any bands. But because this construct is less important for cell culuture, we won't continue working on this construct.
Cloning of Darpin into pCerulean_Vp1up_NLS and pSB1C3
Investigator: Achim
The Darpin needs to be cloned into pSB1C3 for submission to the parts registry as well as into pCerulean for expression.
Darpin ( cut with NgOMIV & SpeI
Vectors cut with AgeI & SpeI
-> Results in fusion protein
Cloning of pSB1C3_001_CMV and pSB1C3_001_Vp2/3_HSPG-Ko_CMV_ZEGFR:1907
Investigator: Anissa
Comment: By working with three-fragment-ligation, CMV and ZEGFR:1907 will each be cut out of pSB1C3, will be ligated together and into the vector pSB1C3_001_Vp2/3_HSPG-Ko. Additionally, CMV will be cloned freom pSB1C3 into pSB1C3_001
Vector: name:pSB1C3_001_Vp2/3_HSPG-Ko P613 c=273,6 ng/µl
Vector: name: pSB1C3_001_bla P320 c=408 ng/µl
Insert: name: pSB1C3_CMV P146 c=155,6 ng/µl
Insert: name: pSB1C3_ZEGFR:1907 P268 c=162,5 ng/µl
new vector name: pSB1C3_001_CMV
new vector name: pSB1C3_001_Vp2/3_HSPG-Ko_CMV_ZEGFR:1907
buffer used: 4
components
pSB1C3_CMV /µl
pSB1C3_ZEGFR:1907  /µl
volume of pSB1C3_001_bla /µl
volume of pSB1C3_001_Vp2/3_HSPG-Ko/µl
DNA
12,7
12,3
3,6
5,5
BSA (10x)
2
2
1,5
1,5
Buffer 4 (10x)
2
2
1,5
1,5
Enzyme SpeI
1
-
1
-
Enzyme EcoRI
1
-
1
1
Enzyme XbaI
-
1
-
-
Enzyme AgeI
-
1
-
-
Enzyme NgoMIV
-
-
-
1
H2O
1,3
1,7
6,4
4,5
Total volume (e.g. 15,20,25,30 µl)
20
20
15
15
0,5 g Agarose, 55 ml TAE (1%), 1,5 µl GELRED , at 115Volt
Marker
P146, 20µl
P268, 20µl
P320, 20µl
P613, 20µl
Fragment size
-
650 bp
170 bp
2100 bp
4000 bp
Gel extraction:
pSB1C3_CMV (p146)
pSB1C3_ZEGFR:1907 (p268)
pSB1C3_001_bla (p320)
pSB1C3_001_Vp2/3_HSPG-Ko (p613)
DNA-concentration [ng/µl]
28,2
13,2
22,1
43,2
T4 Ligation pSB1C3_001_CMV:
Volume vector p320: 4,63 µl
Volume insert p146: 3,37 µl
T4 Ligation pSB1C3_001_Vp2/3_HSPG-Ko_CMV_ZEGFR:1907:
Volume vector p613:  2,9 µl
Volume insert p268:  3 µl
Volume insert p146:  3 µl
Transformation:
Was performed in BL21 according the standard protocol
CD assembly with new suffix primer
Investigator: Kira
PCR program:
components
volume in µl
volume in µl
5x Phusion HF buffer
10
10
10 mM dNTP mix
1
1
0158 primer_for (1:10 dilution)
2,5
2,5
0159 primer_rev (1:10 dilution)
2,5
2,5
DNA template (1:100)
0,5
0,5
DMSO
0
0.5
Phusion polymerase
0,5
0,5
H2O
33
32,5
Total volume (e.g. 50 µl)
50
50
Cycles
Temperature
Time
98°C
1
10x
98°C
15"
60°C
25"
72°C
40"
20x
98°C
15"
65°C
25"
72°C
40"
1x
72°C
5'
Hold 4°C
no PCR bands are visible on the gel..--> PCR reaction failed.
test digestion of all CD constructs
Investigator: Kira
Comment: Because the PCR failed also with new suffix primer, we decided to check if the DNA samples are ok. 3 different CD samples were used as DNA samples.
1_original CD with iGEM binding sites p(223)
2_CD with SDM of PstI
3_CD with SDM of PstI and NgoMIV (p290)
components
volume in µl
DNA
3
BSA 10x
2,0
Buffer 4
2,0
PstI-HF
0,5
NgoMIV
1,0
H2O
11,5
Total volume (e.g. 20 µl)
20
Cloning of pSB1C3_Affibody_middlelinker_EGFP_His
Investigator: Jessica
Comment: There was a mistake in cloning pSB1C3_Affibody_middlelinker_EGFP_His so there is just pSB1C3_EGFP_His. in this plasmid will be cloned Affibody_middlelinker_EGFP_His
Vector: name: pSB1C3_EGFP_His P565
Insert: name: pBAD_Affibody_middlelinker_EGFP_His P596
new vector name: pSb1C3_ZEGFR:1907_middlelinker_EGFP_His P
buffer used: 4
DNA concentration (vector): 252,7 ng/µl; DNA concentration (insert): 101,4 µg/µl
components
volume of pBAD  /µl
volume of pSB1C3 /µl
DNA
15
4
BSA (10x)
2
2
Buffer 4 (10x)
2
2
Enzyme XbaI
0,8
0,8
Enzyme AgeI
0,8
0,8
H2O
-
10,4
Total volume (e.g. 15,20,25,30 µl)
20,6
20
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 120 Volt
Marker
P565, 20µl
P596, 20µl
Sample
Fragment size
2100 bp
1000 bp
Gel extraction:
pSB1C3
Affibody_middlelinker_EFGP_His
DNA-concentration [ng/µl]
17,5
14,2
T4 Ligation:
Volume vector:  2,9 µl
Volume insert:  5,1 µl
Transformation:
BL21 cells were used
Sequencing analysis of pSB1C3_001_RC_IRCK, pSB1C3_001_RC_IRCK_VP1-ko_P5tataless and pSB1C3_001_RC_IRCK_VP2-ko_P5tataless
Investigator: Stefan
Because all these constructs were cloned via PCR out of the pAAV vector, a complete sequencing was performed:
P548 pSB1C3_001_RC_InsRepCap_KpnIback_VP1-ko_P5tataless clone 1:
Sequence could be verified, can be used for further cloning steps!
P568 pSB1C3_001_RC_InsRepCap_KpnIback_VP2-KO_P5tataless clone 2:
Sequence could be verified, can be used for further cloning steps!
P594 pSB1C3_001_RC_InsRepCap_KpnIback_RFC10  clone 1:
Sequence could be verified, can be used for further cloning steps!
Cloning of HSPG_KO_empty into pSB1C3_001_RC_IRCK_VP1-ko_P5tataless and pSB1C3_001_RC_IRCK_VP2-ko_P5tataless
Investigator: Stefan
Vector name:
pSB1C3_001_RC_IRCK_VP1-ko_P5tataless (P548)
pSB1C3_001_RC_IRCK_VP1-ko_P5tataless (P568)
Insert name:
VB:pSB1C3_001_KO_empty(P611)
components
volume of P548  /µl
volume of P568 /µl
volume of P611 /µl
DNA
3,5
3,5
16
BSA (10x)
2
2
3
Buffer 4 (10x)
2
2
3
Enzyme BamHI
1
1
1
Enzyme PvuII
1
1
1
H2O
10,5
10,5
6
Total volume (e.g. 15,20,25,30 µl)
20
20
30
Gel:
for vector:
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 115 Volt
for insert:
1 g Agarose, 50 ml TAE (2%), 3 µl GELRED , at 105 Volt
Gel extraction:
Was performed according to protocol.
T4 Ligation:
VP1-ko:
Volume VP1-ko:  5,57 µl
Volume insert:  2,43 µl
VP2-ko
Volume VP2-ko:  5,04 µl
Volume insert:  2,96 µl
Transformation:
Was performed according to standard protocol using BL21 cells.
Cloning of 587_KO_Z34C and 587_KO_Z34C_Spacer into pSB1C3_001_CMV_VP123_capins
Investigator: Anna
Comment: Todays sequencing results of 587_KO_Z34C and 587_KO_Z34C_Spacer looked well, so they will be cloned into pSB1C3_CMV_VP123_capins. Finally produced ViralBricks:
media:Freiburg10_Production of ViralBricks_End version.pdf
Vector name: pSB1C3_CMV_VP123_capins P580
Insert name:
1. pSB1C3_587_KO_Z34C P599
2. pSB1C3_587_KO_Z34C_Spacer P600
buffer used: 4
DNA concentration (P580): 377,29 ng/µl;
DNA concentration (P599): 102,5 µg/µl
DNA concentration (P600): 90,4 µg/µl
components
volume of P580  /µl
volume of P599 /µl
volume of P600 /µl
DNA
2,7
14,7
16,7
BSA (10x)
2,5
2,5
2,5
Buffer 4 (10x)
2,5
2,5
2,5
Enzyme BamHI
1
1
1
Enzyme PvuII
1
1
1
H2O
15,3
3,3
1,3
Total volume (e.g. 15,20,25,30 µl)
25
25
25
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 120 Volt
Marker, 4 µl
P580
2: P599
3: P600
Construct
pSB1C3_CMV_VP123_cut
587_KO_Z34C
587_KO_Z34C_Spacer
Fragment size
5000 bp
194 bp
209 bp
Gel extraction:
Was performed following the standard protocol.
DNA-concentrations [ng/µl]:
c(vector)= 18,88
c(587_KO_Z34C)= 9,74
c(587_KO_Z34C_Spacer)= 1,72
T4 Ligation:
587_KO_Z34C:
Volume vector:  1,5 µl
Volume insert:  6,5 µl
587_KO_Z34C_Spacer:
Volume vector:  4,6 µl
Volume insert:  3,4 µl
Trafo:
Was performed following the standard protocol using BL21 cells.
Mini-Prep and Test-digestion of VP1 insertion and VP2 fusion HSPG-KO approaches
Investigator: Hanna
Comments: Yesterday, 2 clones of each construct were picked. Today mini-prep and test digestion will be performed of one of each approach - the other clones will be spinned down and stored at -20°C - waiting for test digestion and sequencing results.
Mini-Prep was performed, following the standard protocol.
New plasmid names:
P617: pCerulean_CFP_MiddleLinker_VP2/3_HSPG-KO
P618: pCerulean_ZEGFR:1907_MiddleLinker_VP2/3_HSPG-KO
P619: pCerulean_ZEGFR:1907_SEG_VP2/3_HSPG-KO
P620: pCerulean_ZEGFR:1907_ShortLinker_VP2/3_HSPG-KO
P621: pCerulean_ZEGFR:1907_LongLinker_VP2/3_HSPG-KO
P622: pCerulean_6xHis_MiddleLinker_VP2/3_HSPG-KO
P623: pCerulean_VP1up_NLS_ZEGFR:1907_VP2/3_HSPG-KO
P624: pCerulean_VP1up_NLS_6xHis_VP2/3_HSPG-KO
P625: pCerulean_VP1up_NLS_mVenus_VP2/3_HSPG-KO
TEST DIGESTION:
Mastermix:
BSA (10x): 10 µL
Buffer 4: 10 µL
EcoRI: 5 µL
PstI: 5 µL
H2O: 37 µL
DNA: 3.3 µL + 6.7 µL Master Mix.
Conclusion: Test digestion looked well - except of P618: There is an additional band visible, which can't be correlated to any construct.
All samples were sent for sequencing.
134. labday 29.09.2010
CD assembly with new suffix primer
Investigator: Kira
PCR program:
as DNA samples p291 and p292 with and wo DMSO were used.
components
volume in µl
volume in µl
5x Phusion HF buffer
10
10
10 mM dNTP mix
1
1
0158 primer_for (1:10 dilution)
2,5
2,5
0159 primer_rev (1:10 dilution)
2,5
2,5
DNA template (1:100)
0,5
0,5
DMSO
0
0.5
Phusion polymerase
0,5
0,5
H2O
33
32,5
Total volume (e.g. 50 µl)
50
50
Cycles
Temperature
Time
98°C
1
10x
98°C
15"
60°C
25"
72°C
40"
20x
98°C
15"
65°C
25"
72°C
40"
1x
72°C
5'
Hold 4°C
Digestion of plasmid backbone:
c (pSB1C3) = 151, 1 ng/ µl
Components
vector Volume/µL
DNA 1 µg
6,0 µl
BSA (10x)
2 µl
Buffer no. 4 (10x)
2,0 µl
Enzyme 1 XbaI
1,0 µl
Enzyme 2 AgeI
1,5 µl
H2O
7,5 µl
Total volume
20
incubation @ 37 C for approx. 2 h
1% agarose gel
Digestion of PCR product:
Components
PCR product Volume/µL
DNA
30,0 µl
BSA (100x)
0,4 µl
Buffer no. 4
4,0 µl
Enzyme 1 XbaI
1,5 µl
Enzyme 2 AgeI
2,0 µl
H2O
2,1 µl
Total volume
40
incubation @ 37 C for approx. 2 h
ligation with T4 ligase:
c(insert)= 31,35 ng/ul
c(vector)= 10,45 ng/ul
8ul DNA mix = insert 3 ul + vector 5 ul
T4 ligase 1 ul
T4 buffer 1 ul
Transformation: Anna
Miniprep of serveral constructs
Investigator: Stefan
Glycerol stocks were prepared:
B516 = pSB1C3_001_RC_ICRK_P5tataless_RFC10  clone 1
B517 = pSB1C3_pTAV (P5)
B518 = pSB1C3_001_RC_ICRK_P5tataless_RFC10  clone 2
Mini-Prep was performed according to the standard protocol
P628 = pSB1C3_001_RC_ICRK_P5tataless_RFC10  clone 1
P629 = pSB1C3_pTAV (P5)
P630 = pSB1C3_001_RC_ICRK_P5tataless_RFC10  clone 2
Test digestion:
Components
Volume P628 + P630 /µl
DNA
3
BSA
1
Buffer 4 (10x)
1
Enzyme I
0,4
Enzyme II
0,4
H2O
4,2
Total volume
10
Comment:First test digestion using XmaI and SpeI did not work out, thererfore, two different approaches using MngBI/PstI and SnaBI and PstI were performed. This verified successful cloning of P5tataless into the target vector.
Sent for sequencing:
pSB1C3_001_RC_ICRK_P5tataless_RFC10 clone 1 (P628) was sent for sequencing using VR2 primer.
mGMK_TK30 approaches
Investigator: Bea
Comment: Since we are still waiting for the TK30 gene ordered at Mr.Gene we decided to take another road to produce a BioBrick compatible thymidine kinase gene. Therefore, we are cloning the mGMK_TK30 fusion construct into the pSB1C3 and after removing the PstI site (see next topic) it is ready for submitting it to the parts registry.  Additionally, we are fusing the mGMK_TK30 to the composite part pSB1C3-leftITR_PROMOTER_betaglobin for testing it in cell culture with pur final super construct.
Digestion of the constructs:
P51.2 = pSB1C3_CFP
P81 = pAAV_RFC25_mGMK_TK30
P434 = pSB1C3_leftITR_CMV_betaglobin
P437 = pSB1C3_leftITR_phTERT_betaglobin
Components
vP51.2 /µL
vP81 upstream/µL
vP81 /µL
vP434/µL
vP437/µL
DNA
5
5
10
4,5
4,5
BSA (10x)
2
2
2
2
2
Buffer no. 4 (10x)
2
2
2
2
2
Enzyme 1
EcoRI
SpeI
SpeI 1
-
-
Enzyme 2
SpeI
XbaI
EcoRI
SpeI
SpeI
H2O
9
9
4
10,5
10,5
Total volume
20
20
20
20
20
Overview cloning plan:
Loading plan of a 0,8% agarose gel:
M    P51.2    P81 (digested with E+S)    P81 (digested with X+S)    P434    P437
Expected sizes of the digested products are:
pAAV_RFC25_mGMK_TK30= 4626, 1717bp
pAAV_RFC25_mGMK_TK30= 4626, 1717bp
pSB1C3_CFP = 2052, 756bp
pSB1C3_leftITR_CMV_betaglobin= 5094b
pSB1C3_leftITR_phTERT_betaglobin= 5094b
Results:
After Gel extraction have been performed the ligation was conducetd:
Design of mutagenesis primers for deletion of PstI in mGMK_TK30
Investigator: Bea
Comment: Since we are still waiting for the TK30 gene ordered at Mr.Gene we decided to take another road to produce a BioBrick compatible thymidine kinase gene. Therefore, new primers were designed in order to remove the PstI restriction sites within the TK30 gene. The primers will be ordered tomorrow.
Cloning of Darpin into pCerulean_Vp1up_NLS and pSB1C3 Part Two
Investigator: Achim
I accidentally plated yesterdays trafo with the wrong antibiotics and I had to digest pSB1C3 again anyway, so I prepared a new digestion of both vectors with Age & Spe.
After watching the gel picture I realized I had digested pSB1C3 with the wrong enzymes. I should have used NgOMIV and Spe in order to cut out BLA.
pCerulean_Vp1up_NLS was ligated with yesterdays Darpin fragment and plated on Ampicillin, the pSB1C3_BLA construct will be cloned correctly tomorrow
Cloning of pGGTBT7_Affibody_middlelinker_EGFP_His
Investigator: Jessica
Comment: pBAD is not really working, so we test another expression-plasmid
Vector: name: pGGTBT7_MVenus P627
Insert: name: pBAD_Affibody_middlelinker_EGFP_His 596prepared yesterday
new vector name: pGGTBT7_ZEGFR:1907_middlelinker_EGFP_His P
buffer used: 4
DNA concentration (vector): 144,1 ng/µl; DNA concentration (insert): 101,4 µg/µl
components
volume of pBAD  /µl
DNA
10
BSA (10x)
2
Buffer 4 (10x)
2
Enzyme XbaI
0,8
Enzyme AgeI
0,8
H2O
4,4
Total volume (e.g. 15,20,25,30 µl)
20
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 120 Volt
Fragment size
5200 bp
730 bp
Gel extraction:
pGGTBT7
Affibody_middlelinker_EFGP_His
DNA-concentration [ng/µl]
20,79
7,84
T4 Ligation:
Volume vector: 3,2  µl
Volume insert: 4,8  µl
Transformation:
BL21 DE3 R2L(Kana+Cm), BL21 DE3(Kana), BL21DE3 R2(Kana+Cm) cells were used
Cloning of CMV promotor and Affibody into pSB1C3_001_VP2/3_capins and pSB1C3_001_VP2/3_HSPG_KO
Investigator: Anna
Comment: Cloning of CMV promotor and Affibody into pSB1C3_001_VP2/3_HSPG_KO was repeated, because it was not clear if the first attempt worked.
Vector name:
1. pSB1C3_001_VP2/3_capins P514: 318,9 ng/µl
2. pSB1C3_001_VP2/3_HSPG_KO P613: 273,6 ng/µl
Insert name:
1. pSB1C3_CMV P146: 155,6 ng/µl
2. pSB1C3_ZEGFR:1907 P248: 146,5 ng/µl
buffer used: 4
components
volume of P514 /µl
volume of P613 /µl
volume of P146 /µl
volume of P248 /µl
DNA
3,1
3,7
12,8
13,7
BSA (10x)
2,0
2,0
2,0
2,0
Buffer 4 (10x)
2,0
2,0
2,0
2,0
Enzyme I
EcoRI  1
EcoRI   1
XbaI  1
SpeI  1
Enzyme II
NgoMIV 1
NgoMIV  1
AgeI  1
EcoRI  1
H2O
10,9
10,3
1,2
0,3
Total volume (e.g. 15,20,25,30 µl)
20
20
20
20
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 120 Volt
Marker, 4 µl
Vector 1: pSB1C3_001_VP2/3_capins
Vector 2: pSB1C3_001_VP2/3_HSPG_KO
CMV
Affibody
Fragment size
4000 bp
4000 bp
650 bp
170 bp
Gel extraction:
Was performed following the standard protocol.
DNA-concentrations [ng/µl]:
c (pSB1C3_001_VP2/3_capins) = 40.9
c (pSB1C3_001_VP2/3_HSPG_KO)= 51.94
c (CMV)= 12.67
c (Affibody)= 14.44
T4 Ligation:
Vector 1:
Volume vector:  2,8 µl
Volume Affibody:  2,0 µl
Volume CMV:  3,2 µl
Vector 2:
Volume vector:  2,5 µl
Volume Affibody:  2,0 µl
Volume CMV:  3,5 µl
Trafo:
Was performed following the standard protocol using BL21 cells.
Cloning of CFP into pSB1C3_lITR_phTERT_beta-globin
Investigator: Anissa
Vector name:
pSB1C3_lITR_phTERT_beta-globin P514: 333,46 ng/µl
Insert name:
pSB1C3_CFP P146: 151,1 ng/µl
buffer used: 4
components
volume of P51.2 /µl
volume of P437 /µl
DNA
13,2
4,5
BSA (10x)
2,0
1,5
Buffer 4 (10x)
2,0
1,5
Enzyme I
XbaI 1
SpeI 1
Enzyme II
PstI 1
PstI 1
H2O
0,8
5,5
Total volume (e.g. 15,20,25,30 µl)
20
15
0,5 g Agarose, 50 ml TAE (1%), 1,5 µl GELRED , at 115 Volt
Marker, 4 µl
Vector
Insert
Fragment size
3180 bp
750 bp
Gel extraction:
Was performed following the standard protocol.
DNA-concentrations [ng/µl]:
c (pSB1C3_CFP) = 11,6
c (pSB1C3_lITR_phTERT_beta-globin)= 24,8
T4 Ligation:
Volume vector:  3,2 µl
Volume insert:  4,8 µl
Trafo:
Was performed following the standard protocol using BL21 cells.
Sequencing results: VP1 insertion and VP2 fusion: Fusion of VP2/3_HSPG-KO
Investigator: Hanna
Sample gallery
pCerulean_CFP_MiddleLinker_ VP2/3_HSPG-KO (P617)
pCerulean_ZEGFR:1907_MiddleLinker_ VP2/3_HSPG-KO (P618)
pCerulean_ZEGFR:1907_SEG_ VP2/3_HSPG-KO (P619)
pCerulean_ZEGFR:1907_ShortLinker_ VP2/3_HSPG-KO (P620)
pCerulean_ZEGFR:1907_LongLinker_ VP2/3_HSPG-KO (P621)
pCerulean_6xHis</sub>_MiddleLinker_ VP2/3_HSPG-KO (P622)
pCerulean__VP1up_NLS_ZEGFR:1907_ VP2/3_HSPG-KO (P623)
pCerulean_VP1up_NLS_6xHis_ VP2/3_HSPG-KO (P624)
pCerulean_VP1up_NLS_mVenus_ VP2/3_HSPG-KO (P625)
Conclusion: Except of pCerulean_ZEGFR:1907_MiddleLinker_ VP2/3_HSPG-KO (P618), all sequencing results looked well. This means: The VP2/3-HSPG-KO motif was cloned successfully to the VP2-Fusion and VP1-Insertion approaches and are ready to be tested in cell culture via qPCR and ELISA. --> Co-transfection with pSB1C3_001_RC_iRCK_VP1or2-KO_HSPG-KO + pHelper + pSB1C3_GOI.
In addition to that, clone 2 of pCerulean_ZEGFR:1907_MiddleLinker_ VP2/3_HSPG-KO will be preped and test digested tomorrow.
Cloning of HSPG_KO_empty into pSB1C3_001_RC_IRCK_P5tataless
Investigator: Stefan
Vector name:
pSB1C3_001_RC_IRCK_P5tataless cl1 (P628)
pSB1C3_001_RC_IRCK_P5tataless cl2 (P630)
Insert name:
VB:pSB1C3_001_KO_empty(P611)
components
volume of P628  /µl
volume of P630  /µl
volume of P611 /µl
DNA
3,5
3,5
16
BSA (10x)
2
2
3
Buffer 4 (10x)
2
2
3
Enzyme BamHI
1
1
1
Enzyme PvuII
1
1
1
H2O
10,5
10,5
6
Total volume (e.g. 15,20,25,30 µl)
20
20
30
Gel:
for vector:
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 110 Volt
for insert:
1 g Agarose, 50 ml TAE (2%), 3 µl GELRED , at 110 Volt
Gel extraction:
Was performed according to protocol.
T4 Ligation:
Volume vector P628:  7,11 µl
Volume insert:  0,89 µl
Volume vector P630:  7,33 µl
Volume insert:  0,67 µl
Transformation:
Was performed according to standard protocol using BL21 cells.
Mass Midi-Prep II
Investigator: Chris W.
Midi-Prep of:
453_BAP =P631
587_BAP =P632
587_KO_BAP =P633
453_RGD =P634
587_RGD =P635
587_KO_RGD =P636
453_HIS =P637
587_HIS =P638
587_KO_HIS =P639
587_KO_empty =P640
453_Z34C =P641
pSB1C3_001_RC_IRCK_VP1-ko_P5tataless =P642 =B463
pSB1C3_001_RC_IRCK_VP2-ko_P5tataless =P643 =B470
The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
plasmid-no.
P631
P632
P633
P634
P635
P636
P637
P638
P639
P640
P641
P642
P643
concentration (ng/µl)
4758,69
448,12
3243,97
2231,44
1401,68
3289,15
3441,22
3058,92
4425,05
2339,06
4377,05
5045,55
5201,73
135. labday 30.09.2010
picking CD clones
Investigator: Kira The agaro plate contained many clones, 4 of them were picked and inoculated into media
ViralBricks BAP-ko and His-ko in VP23
Investigator: Bea
Comment: In order to verify the insertion of the ViralBricks BAP-ko and His-ko into the VP23 construct we digested the produced constructs with SalI and BamHI (both HF) and loaded it on a 1%a agarose gel. The constructs were used for creating the final super constructs (yay). Sequencing will verify if we subcloned the ViralBrick truly/seriously into the plasmids.
Digestion of the constructs:
P660 = pSB1C3_001_VP2/3_HSPG-ko_587_BAP
P661 = pSB1C3_001_VP2/3_HSPG-ko_587_BAP
Expected size: 464bp
P662 = pSB1C3_001_VP2/3_HSPG-ko_587_His
P663 = pSB1C3_001_VP2/3_HSPG-ko_587_His
Expected size: 437bp
P525 = pSB1C3_001_VP2/3_capins
Expected size: 404bp
Results:
The detected bands correspond to the expected sizes, but we cannot distinguish between the exact sizes because they are too similar. Therefore we sent P660 and P663 for sequencing in order to confirm if the ViralBricks have been successfully subcloned in the pSB1c3_VP2/3 construct.
Cloning of Darpin into pCerulean_Vp1up_NLS and pSB1C3 Part Three
Investigator: Achim
Clones of the pCerulean_Vp1up_NLS_Darpin construct were picked and will be prepped tonight.
pSB1C3_001_BLA (P320) and pMA_Darpin (P614) were digested with NgOMIV and SpeI. Bands were cut out accordingly, ligated with T4 ligase and plated on Chloramphenicol.
Test digestion of pCerulean_ZEGFR:1907_MiddleLinker_VP2/3_HSPG-KO clone 2
Investigator: Hanna
Comment: Yesterday's sequencing results revealed, that VP2/3_HSPG-KO was successfully cloned to the VP2 fusion and VP1 insertion approaches, except of pCerulean_ZEGFR:1907_MiddleLinker_VP2/3_HSPG-KO clone 2. The chromatogram showed that fusion worked, but there was a kind of contamination in the sample - already visible in the test digestion.
DNA = 2 µL
BSA (10x) = 1 µL
Buffer 4 = 1 µl
EcoRI = 0.5 µL
PstI = 0.5 µL
H20 = 5 µL
Incubation time = 45 minutes.
0.9 % Agarose gel.
Test digestion looked well. Sample was sent for sequencing.
Cloning VP2/3_Capins and VP2/3_HSPG-KO downstream of Middle Linker
Investigator: Hanna
Comment: Besides the Affibody we also want to test and characterize the DARPin for its binding properties referring to the VP2 fusion and VP1 insertion. Therefore some preliminary cloning steps have to be performed. Today the VP2/3_insCap and VP2/3_HSPG-KO sequence will be cloned downstream of the Middle Linker for the VP2 fusion.
Practical Cloning:
Vector: pGA14_MiddleLinker (P301)
Insert: pSB1C3_VP2/3_insCap (P525) and pSB1C3_001_VP2/3_HSPG-KO (P613)
new vector name: pGA14_MiddleLinker_VP2/3_insCap and pGA14_MiddleLinker_VP2/3_HSPG-KO
buffer used: 4 ; Restriction-enzymes used: PstI, AgeI/NgoMIV
Digestion
components
volume of P301 /µl
volume of P525 /µl
volume of P613 /µl
DNA
5
7.6
5.5
BSA (10x)
2
2
2
Buffer 4 (10x)
2
2
2
PstI
1
1
1
AgeI
1
-
-
NgoMIV
-
1
1
McsI
0.5
0.5
0.5
H2O
9
6.4
8.5
Total volume
20
20
20
Comment: After 30 minutes of digestion I remembered that it would be adventageous to digest the plasmid backbone of the VP2/3 constructs. Therefore 0.5 µL of McsI was added to each sample - fortunately McsI does not have any restriction sites in the pGA14_MiddleLinker to which it was also added by mistake.
Incubation: 20 minutes + 2 hours.
Agarose-Gel:
0.5 g Agarose, 50 mL TAE (1 %), 3 µL GELRED, at 10 Volt, running time: 45 minutes
Sample
Sample/µl]
Loading dye (6x)/µl
Expected size 1
Expected size 2
Expected size 3
P301
20 µl
4 µl
~ 2450 bp
P525
20 µl
4 µl
1951 bp
738 bp
1324bp
P613
20 µl
4 µl
1951 bp
738 bp
1324bp
Marker: GeneRuler ladder mix
Marker /µl
Sample P301 /µl
Sample P525 /µl
Sample P613 /µl
Lane
4.5
24
24
24
Gel extraction
Gel measurement:
Sample
Weight
Concentration
P301
250 mg
38.41
P525
270 mg
18.21
P613
300 mg
21.55
Ligation
Construct
Vector (µl)
Insert (µl)
T4 Ligase Buffer (µl)
T4 Ligase (µl)
pGA14_MiddleLinker_VP2/3_insCap
1.32
6.68
1
1
pGA14_MiddleLinker_VP2/3_HSPG-KO
1.52
6.48
1
1
Miniprep of serveral constructs
Investigator: Jessica
Glycerol stocks were prepared:
B527 pSB1C3_001_CMV_VP2/3_587-KO_Z34C_spacer clone3
B528 pSB1C3_001_CMV_VP123_587-KO_Z34C clone1
B529 pSB1C3_001_CMV_VP123_587-KO_Z34C clone2
B530 pSB1C3_001_CMV_VP123_587-KO_Z34C clone3
B531 pSB1C3_oo1_CMV clone1
B532 pSB1C3_oo1_CMV clone2
B533 pSB1C3_001_CMV_ZEGFR:1907_VP2/3_HSPG-KO clone1
B534 pSB1C3_001_CMV_ZEGFR:1907_VP2/3_HSPG-KO clone2
B535 pSB1C3_001_VP2/3_ins_HSPG-KO_CMV_ZEGFR:1907 clone 1
B536 pSB1C3_001_VP2/3_ins_HSPG-KO_CMV_ZEGFR:1907 clone 2
B537 pBS1C3_001_VP2/3_587_BAP_HSPG-KO clone1
B538 pBS1C3_001_VP2/3_587_BAP_HSPG-KO clone2
B539 pBS1C3_VP2/3_capins_His clone1
B540 pBS1C3_VP2/3_capins_His clone2
B541 pSB1C3_ZEGFR:1907 _middlelinker_EGFP_His clone1
B542 pSB1C3_ZEGFR:1907 _middlelinker_EGFP_His clone2
Mini-Prep was performed according to the standard protocol
P644  pSB1C3_001_RC_IRCK_VP1-ko_HSPG-ko_P5tataless cl1 c= 151,0 ng/µl
P645  pSB1C3_001_RC_IRCK_VP1-ko_HSPG-ko_P5tataless cl2 c= 115,8 ng/µl
P646  pSB1C3_001_RC_IRCK_VP2-ko_HSPG-ko_P5tataless cl1 c= 173,1 ng/µl
P647  pSB1C3_001_RC_IRCK_VP2-ko_HSPG-ko_P5tataless cl2 c= 119,9 ng/µl
P648  pSB1C3_001_CMV_VP123_587-KO_Z34C_spacer clone1 c= 226,8 ng/µl
P649  pSB1C3_001_CMV_VP123_587-KO_Z34C_spacer clone2 c= 290,3 ng/µl
P650  pSB1C3_001_CMV_VP123_587-KO_Z34C_spacer clone3 c= 284,2 ng/µl
P651  pSB1C3_001_CMV_VP123_587-KO_Z34C clone1 c= 283,8 ng/µl
P652  pSB1C3_001_CMV_VP123_587-KO_Z34C clone2 c= 290,5 ng/µl
P653  pSB1C3_001_CMV_VP123_587-KO_Z34C clone3 c= 251,7 ng/µl
P654  pSB1C3_oo1_CMV clone1 c= 199,5 ng/µl
P655  pSB1C3_oo1_CMV clone2 c= 192,7 ng/µl
P656  pSB1C3_001_CMV_ZEGFR:1907_VP2/3_HSPG-KO clone1 c= 200,3 ng/µl
P657  pSB1C3_001_CMV_ZEGFR:1907_VP2/3_HSPG-KO clone2 c= 325,0 ng/µl
P658  pSB1C3_001_CMV_ZEGFR:1907_VP2/3_HSPG-KO clone3 c= 222,9 ng/µl
P659  pSB1C3_001_CMV_ZEGFR:1907_VP2/3_HSPG-KO clone4 c= 201,3 ng/µl
P660  pBS1C3_001_VP2/3_587_BAP_HSPG-KO clone1  c= 267,1 ng/µl
P661  pBS1C3_001_VP2/3_587_BAP_HSPG-KO clone2 c= 193,1 ng/µl
P662  pBS1C3_VP2/3_capins_His clone1 c= 209,1 ng/µl
P663  pBS1C3_VP2/3_capins_His clone2 c= 252,5 ng/µl
P664  pSB1C3_ZEGFR:1907 _middlelinker_EGFP_His clone1 c= 185,7 ng/µl
P665  pSB1C3_ZEGFR:1907 _middlelinker_EGFP_His clone2 c= 192,0 ng/µl
P666  pSB1C3_CFP c= 212,7  ng/µl
Test digestion of pSB1C3_ZEGFR:1907 _middlelinker_EGFP_His and mCherry iGEM 2008
Investigator: Jessica
components
volume of P664 pSB1C3_ZEGFR:1907 _middlelinker_EGFP_His /µl
volume of P665 pSB1C3_ZEGFR:1907 _middlelinker_EGFP_His /µl
volume of P626 mCherry /µl
DNA
1,5
1,5
1,5
BSA (10x)
1
1
1
Buffer 4 (10x)
1
1
1
XbaI
0,4
0,4
0,4
AgeI
0,4
0,4
0,4
H2O
5,7
5,7
5,7
Total volume
10
10
10
Sample gallery
Test digestion of pSB1C3_001_CMV_affibody_HSPG_KO and  pSB1C3_001_CMV_VP123_587_KO_Z34C
Investigator: Anna
Comment: 587_KO_Z34C and 587_KO_Z34C_Spacer were cloned into pSB1C3_001_CMV_VP123 as already done with the other 453 and 587 constructs (see test digestion from labday 28.09.) The test digestion looks well, nevertheless one sample out of these constructs should be sent for sequencing.
components
volume of pSB1C3_001_CMV_affibody_HSPG_KO /µl
volume of pSB1C3_001_CMV_VP123_587_KO_Z34C /µl
volume of pSB1C3_001_CMV_VP123_587_KO_Z34C_Spacer /µl
DNA
3
3
3
BSA (10x)
1,5
1,5
1,5
Buffer 4 (10x)
1,5
1,5
1,5
EnzymeI
SspI 0,3
SalI 0,3
SalI 0,3
EnzymeII
EcoRI 0,3
PvuII 0,3
PvuII 0,3
H2O
4,8
4,8
4,8
Total volume
15
15
15
Sample
1
2
3
4
5
6
7
Construct
587_KO_Z34C
587_KO_Z34C
587_KO_Z34C
587_KO_Z34C_Spacer
587_KO_Z34C_Spacer
587_KO_Z34C_Spacer
p580
Clone
1
2
3
1
2
3
Sample
8
9
10
11
12
Construct
p613
CMV_EGFR:1907
CMV_EGFR:1907
CMV_EGFR:1907
CMV_EGFR:1907
Clone
1
2
3
4
Midiprep of pSB1C3_001_CMV_VP1/2/3_capins_587bla and pSB1C3_001_RC_IRCK_P5tataless clone1
Investigators: Chris W.
Midi-Preps of:
pSB1C3_001_CMV_VP1/2/3_capins_587bla =P667
pSB1C3_001_RC_IRCK_P5tataless clone1 =P668
The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
plasmid-no.
P667
P668
concentration (ng/µl)
1502,3
991,6
Cloning of VP2/3_BAP_HSPG-KO and VP2/3_His_HSPG-KO into pCerulean_Zegfr:1907_Middlelinker and VP2/3_His_HSPG-KO into pCerulean_VP1up_NLS_mVenus
Investigator: Stefan
Vector name:
pCerulean_Zegfr:1907_Middlelinker (P408)
pCerulean_VP1up_NLS_mVenus (P426)
Insert name:
pBS1C3_001_VP2/3_587_BAP_HSPG-KO clone1 (P660)
pBS1C3_001_VP2/3_587_BAP_HSPG-KO clone2 (P661)
pBS1C3_001_VP2/3_587_His_HSPG-KO clone1 (P662)
pBS1C3_001_VP2/3_587_His_HSPG-KO clone2 (P663)
components
volume for inserts (P660 - P663)  /µl
volume of P408  /µl
volume of P426 /µl
DNA
10
3
3
BSA (10x)
3
2
2
Buffer 4 (10x)
3
2
2
Enzyme NgoMIV
1
-
-
Enzyme PstI
1
1
1
Enzyme MscI
1
-
-
Enzyme AgeI
-
1
1
H2O
11
11
11
Total volume (e.g. 15,20,25,30 µl)
30
20
20
Gel:
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 115 Volt
Gel extraction:
Was performed according to protocol.
T4 Ligation:
ligation name
408 + 660
408 + 661
408 + 662
408 + 663
426 + 662
426 + 663
volume of vector
3,4
3,05
3,35
2,56
2,74
3,13
volume of insert
4,6
4,95
4,65
5,44
5,26
4,87
Transformation:
Was performed according to standard protocol using BL21 cells.
=> Go to Labjournal October part 1 (labday 136 - 145 )
Retrieved from " http://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/September2 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
