<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Bioware_NoteBook_Labjournal_May"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg_Bioware/NoteBook/Labjournal/May">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg_Bioware/NoteBook/Labjournal/May&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/May&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/May&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg_Bioware/NoteBook/Labjournal/May" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg Bioware/NoteBook/Labjournal/May</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="menu"><UL id="menu"><LI><DL><DT class="menu_button_left"><A href="https://2010.igem.org/Team:Freiburg_Bioware">Home</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Team_2010">Team</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Team_2010">Team 2010</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Photo_Gallery">Photo Gallery</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Cuckoo_Clock">Cuckoo Clock Competition</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Collaboration">Collaboration</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Example_Page">Example Page</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project">Project</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project">Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#highlights">Highlights</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#highlights">Results</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#modularisation">   Modularisation</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#targeting">   Targeting</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#arming">   Arming</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#superconstruct">   All-in-One</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Methods">Methods</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Virus_Construction_Kit">Virus Construction Kit Manual</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks">BioBricks</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks">Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/capsid_coding">Capsid Coding Parts</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/vector_plasmid">Vector Plasmid Parts</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/miscellaneous">Miscellaneous Parts</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A class="current" href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook">Notebook</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal">Labjournal</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Materials">Materials</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Methods">Methods</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Experiments">Experiments</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Meetings">Meetings</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Statistics">Statistics</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Sponsors">Sponsors</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Safety">Safety</A></DT></DL></LI><LI><DL><DT class="menu_button_right"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling">Modeling</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling">Modeling Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Virus_Production">Virus Production</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Virus_Infection">Virus Infection</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Structure_Modeling">Structure Modeling</A></DD></DL></LI></UL></DIV><DIV class="pageContent"><P><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook" class="external text" rel="nofollow">=&gt; Back to Notebook overview</A></P><DIV class="box_right"><LEFT><U1>NoteBook Navigator</U1></LEFT><UL><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/March">March (labday 1)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/April">April (labday 2 - 5)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/May">May (labday 6 - 17)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/June">June (labday 18 - 45)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/July">July (labday 46 - 75)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/August">August part 1 (labday 76 - 92)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/August2">August part 2 (labday 93 - 106)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/September">September part 1 (labday 107 - 123)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/September2">September part 2 (labday 124 - 135)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/October">October part 1 (labday 136 - 149 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/October2">October part 2 (labday 150 - 166 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/November">November  (labday 167 - 170 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/Cellculture">Cellculture</A></LI></UL></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#6._Labday_03.05.2010"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">6. Labday 03.05.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Theoretical_cloning"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Theoretical cloning</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-3"><A href="#7._Labday_07.05.2010"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">7. Labday 07.05.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Protocolls"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Protocolls</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#8._Labday_10.05.2010"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">8. Labday 10.05.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Stock_solutions"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Stock solutions</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#9._Labday_17.05.2010"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">9. Labday 17.05.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#.CE.B2-Globin"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">β-Globin</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#10._Labortag_18.05.2010"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">10. Labortag 18.05.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#pCMV_mVenus_YFP"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">pCMV_mVenus_YFP</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#11._Labortag_19.05.2010"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">11. Labortag 19.05.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Cloning_of_pCMV-MCS_.2B_pGA14_mVenus_YFP_pCMV_mVenus_YFP"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Cloning of pCMV-MCS + pGA14_mVenus_YFP pCMV_mVenus_YFP</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#12._Labortag_20.05.2010"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">12. Labortag 20.05.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Picking_clones"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Picking clones</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#13._Labortag_21.05.2010"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">13. Labortag 21.05.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#CMV-Promoter"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">CMV-Promoter</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#LB_medium_was_prepared"><SPAN class="tocnumber">8.2</SPAN><SPAN class="toctext">LB medium was prepared</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Plasmid_Mini-Prep_according_to_the_standard_protocol"><SPAN class="tocnumber">8.3</SPAN><SPAN class="toctext">Plasmid Mini-Prep according to the standard protocol</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#14._Labortag_25.05.2010"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">14. Labortag 25.05.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Cloning_of_pCMV_mVenus_YFP_.2B_pAAV_MCS"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">Cloning of pCMV_mVenus_YFP + pAAV_MCS</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Production_of_chemical_competent_E.coli"><SPAN class="tocnumber">9.2</SPAN><SPAN class="toctext">Production of chemical competent E.coli</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#Design_of_MCS-Oligos"><SPAN class="tocnumber">9.3</SPAN><SPAN class="toctext">Design of MCS-Oligos</SPAN></A></LI><LI class="toclevel-1 tocsection-23"><A href="#15._Labortag_26.05.2010"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">15. Labortag 26.05.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Repetition:_Cloning_of_pAAV_MCS_.2B_pCMV_mVenus_YFP"><SPAN class="tocnumber">10.1</SPAN><SPAN class="toctext">Repetition: Cloning of pAAV_MCS + pCMV_mVenus_YFP</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Ordering_oligos"><SPAN class="tocnumber">10.2</SPAN><SPAN class="toctext">Ordering oligos</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#Test-Trafo_of_chemical_competent_E.coli_cells"><SPAN class="tocnumber">10.3</SPAN><SPAN class="toctext">Test-Trafo of chemical competent E.coli cells</SPAN></A></LI><LI class="toclevel-1 tocsection-27"><A href="#16._Labortag_27.05.2010"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">16. Labortag 27.05.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#Cell_counting"><SPAN class="tocnumber">11.1</SPAN><SPAN class="toctext">Cell counting</SPAN></A></LI><LI class="toclevel-1 tocsection-29"><A href="#17._Labortag_31.05.2010"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">17. Labortag 31.05.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#pAAV_RC_R585A_and_R588A_transistions"><SPAN class="tocnumber">12.1</SPAN><SPAN class="toctext">pAAV_RC R585A and R588A transistions</SPAN></A></LI></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="6._Labday_03.05.2010"><P style="font-size:17px; background-color:#00dd77;">6. Labday 03.05.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Theoretical_cloning"><P style="font-size:15px; background-color:#66bbFF;"><B>Theoretical cloning</B></P></SPAN></H4><P><B>Investigators: Adrian, Hanna, Bea, Patrick, Chris W. </B></P><P>Several theoretical cloning issues need to be adressed: The modularization and modification of the Stratagene plasmids and the prodrug activating enzymes. :
1. Cap-Gen: 
</P><UL><LI> delete the PstI-restriction site
</LI><LI> insert an antibody fragment (targeting): sequence? in which part of VP1? (Patrick)
</LI><LI> add prefix &amp; suffix 
</LI><LI> disable binding of Heparan Sulphat Proteoglycan 
</LI></UL><P>
2. Rep-Gen:
</P><UL><LI> delete EcoRI (2x) and PstI (2x) 
</LI></UL><P>
3. ITRs:
</P><UL><LI> NotI and PstI (in sequence?) flank the ITRs: The question whether they can/have to be deleted arose.
</LI><LI> where exactly does the sequence start and end? (Patrick)
</LI><LI> we should check if three ITRs could be used to increase expression levels
</LI><LI> we blasted the ITR (left) of the MCS vektor (Stratagene). we got a 92% &quot;Coverage&quot; with the AAV2 genom (to 100%). from the  alignment (Stratagene ITR with ITR of the AAV2 genom) we got:</LI></UL><P>You can see, that the ITR sequence begins 4 bp after the PstI restriction site. thereupon we did a secundary structure analyse (www.dinamelt.bioinfo.rpi.edu), with the following structure we got: <A href="/wiki/images/8/8a/Freiburg10_AAV2ITR%28left%29nachBlast.pdf" class="internal" title="Freiburg10 AAV2ITR(left)nachBlast.pdf">Media:Freiburg10_AAV2ITR(left)nachBlast.pdf</A>
conclusion: the big loop of the secondary structure dosent change if we delete the PstI restriction site (cf. with other uploadet secondary structures), it should be considered that we can't add random bp that could affect the secundary structure. <B>To do: which bp, which sequence could/can be insertet? cf. the genome of AAV2</B>
4. MCS:
</P><UL><LI> replacement through the iGEM-MCS
</LI><LI> where is the beginning and ending of the MCS in the Vector? ß-globin function and sequence (search for literature and patents, which are denoted in the AAV-Helper-Free-System Manual)
</LI><LI> in this context it would be also important to find out where the beta-Globulin-Intron exactly starts or rather we can cut out the MCS.
</LI></UL><P>
5. enzymes:
</P><UL><LI> Thymidinkinase: find informations. TK30 (Bea), SR39 (Hanna)
</LI><LI> Cytosindeaminase: find informations (Adrian)
</LI></UL><P>
in addition we downloaded, added and anotated the sequence of the pHelper plasmid of Stratagene in Geneious (see &quot;Constructs&quot;).	</P><P>
insert the antibody fragment (targeting): sequence? in which part of VP1? (Patrick)
</P><H3><SPAN class="mw-headline" id="7._Labday_07.05.2010"><P style="font-size:17px; background-color:#00dd77;">7. Labday 07.05.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Protocolls"><P style="font-size:15px; background-color:#66bbFF;"><B>Protocolls</B></P></SPAN></H4><P><B>Investigators: Anissa, Kerstin </B></P><UL><LI> adaption and extension of the standart prorcolls (Cloning for Pro's)
</LI><LI> we startet to create a protokoll-mask for the praktical-cloning (short version for labwork)
</LI></UL><H3><SPAN class="mw-headline" id="8._Labday_10.05.2010"><P style="font-size:17px; background-color:#00dd77;">8. Labday 10.05.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Stock_solutions"><P style="font-size:15px; background-color:#66bbFF;"><B>Stock solutions</B></P></SPAN></H4><P><B>Investigators: Adrian, Chris W., Chris L., Bea, Achim, Patrick, Hanna, (Sven)</B></P><P>The following stock solutions were prepared: 
1. Antibiotics:
</P><UL><LI> Ampicillin: 2 g Ampicillin were dissolved in 20 mL ethanol (70%), filled into 2 mL tubes and stored at -20°C.
</LI><LI> Chloramphenicol: 0.5 g Chloramphenicol were dissolved in 20 mL ethanol (70%), filled into 2 mL tubes and stored at the -20°C.
</LI><LI> Kanamycin: 1 g Kanamycin was dissolved in 20 mL multipore-H2O, sterilized by filtration and filled into 2 mL tubes and stored at the -20°C.
</LI><LI> Tetracyclin: 0.5 g Tetracyclin were dissolved in 20 mL ethanol (70%) and filled into 2 mL tubes. The tubes were wrapped with aluminium foil (light sensitive!) and stored at -20°C.
</LI></UL><P>2. ITPG solution (1 M):
</P><UL><LI> ~ 4.766 g ITPG were dissolved in 20 mL multipore-H2O, sterilized by filtration, filled in 2 mL tubes and stored at -20°C.
</LI></UL><P>3. DYT (5 litres)
</P><UL><LI> 80 g Bactotrypton, 50 g Bactoyeast, 25 g NaCl were weight out. 
</LI><LI> 2 L multipore-H2O were added 
</LI><LI> after mixing, multipore-H2O was added -&gt; endvolume 5 litres 
</LI><LI> medium was filled into flask and was autoclaved
</LI></UL><P>4. Glycerol:
</P><UL><LI> Glycerol was filled into a flask and was then autoclaved
</LI></UL><P><B>To do: register at Mr. Gene!!!</B></P><H3><SPAN class="mw-headline" id="9._Labday_17.05.2010"><P style="font-size:17px; background-color:#00dd77;">9. Labday 17.05.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id=".CE.B2-Globin"><P style="font-size:15px; background-color:#66bbFF;"><B>β-Globin</B></P></SPAN></H4><P><B>Investigators: Bea, Chris W., Patrick, Hanna (and instructors)</B></P><P>
We blasted the sequence of the β-globin and additional nucleotides „in the pre- and suffix“.
We found only 70% coverage with the human β-globin-intron add. some parts of the exon 3.
For this reason we think that the pAAV_MCS annotation of the β-Globin-intron of Stratagene is too generously.
In addition the alignment of the human ß-globin showed that only parts of the intron 2 (5’) and exon 3 are integratet into the vector.
We assume that the informations from Stratagene are not total correctly ( intron flanket by the splicedonors and the acceptor-sequence).
We blasted the sequence between the CMV-promoter and the origin beginning of the ß-globin intron.
We found a 98,8% coverage with a synthetic CMV-promoter construct ( 1 nucleotide difference).
Literature: „Diverse plasmid DNA vectors by directed  molecular evolution of cytomegalovirus promoters. (Wright A. et al.)“
→The question arises if we can omit the ß-globin (because the exact function is unknown).
For this we should contact various Companys (GeneArt, DNA2.0, Mr. Gene,…) to get more information.
In addition we could test the expression with and without ß-globin.
</P><H3><SPAN class="mw-headline" id="10._Labortag_18.05.2010"><P style="font-size:17px; background-color:#00dd77;">10. Labortag 18.05.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="pCMV_mVenus_YFP"><P style="font-size:15px; background-color:#66bbFF;"><B>pCMV_mVenus_YFP</B></P></SPAN></H4><P><B>Investigator: Bea, Chris W., Patrick, Hanna, Anissa, Kerstin, Adrian (und Instructors)</B></P><P>
Theoretical cloning with Geneious of pCMV-MCS + pGA14_mVenus_YFP --&gt;<B>pCMV_mVenus_YFP</B></P><UL><LI>Enzyme set: <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> (iGEM)
</LI><LI> digest pGA14_mVenus_YFP (insert) with XbaI and PstI: mVenus_YFP_cut_XbaI+PstI 
</LI><LI> digest pCMV_MCS (vector) with XbaI and PstI: pCMV_MCS_cut_XbaI+PstI
</LI><LI> ligate mVenus_YFP_cut_XbaI+PstI with pCMV_MCS_cut_XbaI+PstI 
</LI></UL><H3><SPAN class="mw-headline" id="11._Labortag_19.05.2010"><P style="font-size:17px; background-color:#00dd77;">11. Labortag 19.05.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Cloning_of_pCMV-MCS_.2B_pGA14_mVenus_YFP_pCMV_mVenus_YFP"><P style="font-size:15px; background-color:#66bbFF;"><B>Cloning of pCMV-MCS + pGA14_mVenus_YFP pCMV_mVenus_YFP</B></P></SPAN></H4><P><B>Investigators: Adrian, Bea, Chris W., Hanna, Patrick</B></P><P><B>Digestion</B></P><UL><LI>plasmid: insert: pGA14_mVenus_YFP; number: P1 production date: ____ origin: ____ </LI><LI>plasmid: vector: pCMV_MCS; number: P2 production date: ____ origin: ____ </LI><LI>new vector name: pCMV_mVenus_YFP </LI><LI>buffer used:3 ; Restriction-enzymes used: Enzyme XbaI (no. Lab:___) ; Enzyme PstI(no.Lab:___)</LI><LI>DNA concentration (vector): 375 ng/µl ; DNA concentration (insert): 476 ng/µl</LI><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right"> V (pGA_mVenus_YFP)/ µl </TD><TD align="right"> V(pCMV_MCS) / µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 4</TD><TD align="right">2,7
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2</TD><TD align="right">2
</TD></TR><TR><TD> Buffer 3 (10x)</TD><TD align="right">2</TD><TD align="right">2
</TD></TR><TR><TD>Enzyme: XbaI (no.Lab:___)</TD><TD align="right">1,5</TD><TD align="right">1,5
</TD></TR><TR><TD>Enzyme: PstI (no.Lab:___)</TD><TD align="right">1</TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">9,5</TD><TD align="right">10,8
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"><B>20</B></TD><TD align="right"><B>20</B></TD></TR></TBODY></TABLE><LI> Incubation: 1 h at 37°C</LI></UL><P><B>1% Agarose gel and Gel extraction</B></P><UL><LI>prepare 1% agarose gel, run gel for 45 minutes(119 V)</LI><LI>cut out insert and vector</LI><LI>perform gel extraction following standard protocol provided by Qiagen</LI></UL><P><B>Ligation</B></P><UL><LI>Measure DNA-concentration with Nanodrop </LI><LI>c(mVenus_YFP) = 16,8 ng/µL</LI><LI>c(pCMV_MCS) = 22,8 ng/µL</LI><LI>Calculation of volume needed for ligation: 
</LI><LI>c(mVenus_YFP) = 3,66 µL</LI><LI>c(pCMV_MCS) = 5,34 µL</LI></UL><P><B>Transformation</B></P><UL><LI>Transformation has been followed the standard protocol <A href="/wiki/images/4/4e/Freiburg10_Cloning_Protocol.pdf" class="internal" title="Freiburg10 Cloning Protocol.pdf">Media:Freiburg10_Cloning Protocol.pdf</A></LI></UL><H3><SPAN class="mw-headline" id="12._Labortag_20.05.2010"><P style="font-size:17px; background-color:#00dd77;">12. Labortag 20.05.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Picking_clones"><P style="font-size:15px; background-color:#66bbFF;"><B>Picking clones</B></P></SPAN></H4><P><B>Investigators: Adrian, Bea</B></P><UL>
Clones were picked according to the standard protocol.
<UL><LI>3 approaches from each plate</LI></UL></UL><H3><SPAN class="mw-headline" id="13._Labortag_21.05.2010"><P style="font-size:17px; background-color:#00dd77;">13. Labortag 21.05.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="CMV-Promoter"><P style="font-size:15px; background-color:#66bbFF;"><B>CMV-Promoter</B></P></SPAN></H4><P><B>Investigators: Volker, Hanna</B></P><P><B>Theoretical cloning:</B></P><UL><LI> The CMV promoter (mainly the regulatory region) was further characterized: For this purpose U.S. patent no. 5,385,839 was used. <A href="/wiki/images/6/6f/Freiburg10_Patent_US5385839A.pdf" class="internal" title="Freiburg10 Patent US5385839A.pdf">Media:Freiburg10_Patent_US5385839A.pdf</A></LI><LI> Further on the CMV promoter sequence was blasted. The results delivered a 98% query coverage with the &quot;Human herpesvirus 5 strain Toledo, complete genome&quot; (accession no.: GU937742.1). Interestingly the maximal identity was just 99%. This can be explained due to a nucleotide deletion and a C-T transition (red circles), which were also marked in Geneious.   
</LI></UL><P><B>To do</B>: find &quot;+1&quot;-location (transcription start); which transcription factors bind to the regulatory region of the CMV promoter?
</P><H4><SPAN class="mw-headline" id="LB_medium_was_prepared"><P style="font-size:15px; background-color:#66bbFF;"><B>LB medium<B> was prepared</B></B></P></SPAN></H4><P>Patrick and Chris W. </P><UL><LI> 10 g Bacto-Tryptone, 5 g Bacto-Yeast, 10 g NaCl were mixed in 500 mL milipore-H2O. 
</LI><LI> Volume was adjusted to 1 L with milipore-H2O.
</LI><LI> 100 mL flasks were each filled with 50 mL medium.
</LI><LI> 0.75 g agar was added to each flask.
</LI><LI> LB was sterilized by autoclaving and is now stored at room temperature.
</LI></UL><H4><SPAN class="mw-headline" id="Plasmid_Mini-Prep_according_to_the_standard_protocol"><P style="font-size:15px; background-color:#66bbFF;"><B>Plasmid Mini-Prep according to the standard protocol</B></P></SPAN></H4><P>Adrian, Kira, Anna, Chris W., Patrick

Measure DNA-concentration with Nanodrop:</P><TABLE border="1"><TBODY><TR><TD align="right"> P3 (pCMV_mVenus_YFP)   </TD><TD align="right"> P4 (pCMV_mVenus_YFP)  </TD><TD align="right"> P5 (pCMV_mVenus_YFP)
</TD></TR><TR><TD> concentration (ng/µl)</TD><TD align="right"> 457</TD><TD align="right">470,8</TD><TD align="right"> 477,29
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="14._Labortag_25.05.2010"><P style="font-size:17px; background-color:#00dd77;">14. Labortag 25.05.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Cloning_of_pCMV_mVenus_YFP_.2B_pAAV_MCS"><P style="font-size:15px; background-color:#66bbFF;"><B>Cloning of pCMV_mVenus_YFP + pAAV_MCS</B></P></SPAN></H4><P><B>Investigators: Kira, Anna, Volker, Jessica</B></P><LI>plasmid: insert: pCMV_mVenus_YFP; number: P5 production date: 21.05.2010 origin: ____ 
</LI><LI>plasmid: vector: pAAV_MCS; number: P? production date: ____ origin: ____ 
</LI><LI>new vector name: pAAV_mVenus_YFP <B>1st try:</B></LI><LI>buffer used:3 ; Restriction-enzymes used: Enzyme NotI (no. Lab:46) 
</LI><LI>DNA concentration (vector): 360 ng/µl ; DNA concentration (insert): 477 ng/µl

<TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right"> V (pCMV_mVenus_YFP)/ µl </TD><TD align="right"> V(pAAV_MCS) / µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 4.2</TD><TD align="right">3.5
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">3</TD><TD align="right">3
</TD></TR><TR><TD> Buffer 3 (10x)</TD><TD align="right">3</TD><TD align="right">3
</TD></TR><TR><TD>Enzyme: NotI (no.Lab:46)</TD><TD align="right">1</TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">15.3</TD><TD align="right">15.3
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"><B>26.4</B></TD><TD align="right"><B>25.8</B></TD></TR></TBODY></TABLE></LI><LI> Incubation: 1 1/2 h at 37°C
<B>note: too little water was added</B>
1% Agarose gel
</LI><LI>1% agarose gel was prepared, gel ran for 45 minutes(119 V)
<B>2nd try:</B></LI><LI>buffer used:4 ; Restriction-enzymes used: Enzyme NotI (no. Lab:159) 
</LI><LI>DNA concentration (vector): 360 ng/µl ; DNA concentration (insert): 477 ng/µl

<TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right"> V (pCMV_mVenus_YFP)/ µl </TD><TD align="right"> V(pAAV_MCS) / µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 4.2</TD><TD align="right">3.5
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">3</TD><TD align="right">3
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">3</TD><TD align="right">3
</TD></TR><TR><TD>Enzyme: NotI (no.Lab:159)</TD><TD align="right">1,5</TD><TD align="right">1,5
</TD></TR><TR><TD>H2O</TD><TD align="right">18,3</TD><TD align="right">19
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"><B>30</B></TD><TD align="right"><B>30</B></TD></TR></TBODY></TABLE></LI><LI> Incubation: 1 1/2 h at 37°C


1% Agarose gel
</LI><LI>1% agarose gel was repared, gel ran for 45 minutes(119 V)
<FONT color="#FF00FF">Results: unexpected sizes of fragments (see protocol), try again next day with an ethidium bromid gel</FONT><H4><SPAN class="mw-headline" id="Production_of_chemical_competent_E.coli"><P style="font-size:15px; background-color:#66bbFF;"><B>Production of chemical competent E.coli</B></P></SPAN></H4><P><B>Investigators: Patrick and Jessica</B></P><P>
competent E.coli XL-1-blue and competent E.coli BL21 produced according to the standard-protocoll <A href="/wiki/images/9/9a/Production_of_competent_E.coli.pdf" class="internal" title="Production of competent E.coli.pdf">Media:production of competent E.coli.pdf</A></P><H4><SPAN class="mw-headline" id="Design_of_MCS-Oligos"><P style="font-size:15px; background-color:#66bbFF;"><B>Design of MCS-Oligos</B></P></SPAN></H4><P><B>Investigators: Bea, Adrian, Hanna, Sven</B></P><P>
In order to replace the multiple cloning site of the pAAV_MCS vector from Stratagene by RFC25, two oligos were designed. After their hybridization the overhangs correspond to ClaI- and BglII restriction sites. A digestion with BglII and ClaI will be performed with pAAV-MCS and then will be ligated with the designed oligos.
The oligos (see link) were ordered at Sigma-Aldrich. Estimated shipment: 31.05.2010 .</P><P><A href="/File:Freiburg10_Oligos_MCS_RFC25_for_pAAV.pdf" title="File:Freiburg10 Oligos MCS RFC25 for pAAV.pdf">File:Freiburg10 Oligos MCS RFC25 for pAAV.pdf</A></P><H3><SPAN class="mw-headline" id="15._Labortag_26.05.2010"><P style="font-size:17px; background-color:#00dd77;">15. Labortag 26.05.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Repetition:_Cloning_of_pAAV_MCS_.2B_pCMV_mVenus_YFP"><P style="font-size:15px; background-color:#66bbFF;"><B>Repetition: Cloning of pAAV_MCS + pCMV_mVenus_YFP</B></P></SPAN></H4><P><B>Investigators: Bea, Hanna, Jessica</B></P></LI><LI>Repeat cloning of pAAV-MCS + pCMV_mVenus_YFP --&gt;<B>Goal: pAAV_mVenus_YFP</B>

Cloning did not work (see lab day 25.05.2010) either with NotI or with NotI-HF. Gel did not show any proper band at the expected size. 
 There will be used EtBr instead of gelred in the agarose gel and the incubation time of digestion will be prolonged. Further details are followed. 
NOTE: There are three restriction sites of NotI in the pCMV-mVenus_YFP. If cloning with NotI, the polyA hGH signal will be deleted. therefore cloning with NotI is <B>not</B> possible .
</LI><LI>Test digestion of pCMV-mVenusYFP with PstI and MluI <P><B>Digestion</B></P></LI><LI>experiment date: 26.05.2010 </LI><LI>plasmid: pCMV_mVenus_YFP; number: P5; production date: 21.05.2010/ pCMV-MSC; number: P2; production date:  </LI><LI>buffer used: 3/4; Restriction-enzymes used: Enzyme PstI (no. Lab:48) and Enyzme MluI (no. Lab:40) / NotI HF (no. Lab:159) </LI><LI>DNA concentration plasmid: P5 477,3 ng/µl / P2 550ng/µl </LI><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right"> V (pCMV_mVenus_YFP)/ µl </TD><TD align="right"> V(pCMV_MCS) / µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 4,2</TD><TD align="right">3,6
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2</TD><TD align="right">2
</TD></TR><TR><TD> Buffer 3 (10x)</TD><TD align="right">2</TD><TD align="right">2
</TD></TR><TR><TD>Enzyme: PstI/NotI HF (no.Lab:48/159)</TD><TD align="right">1</TD><TD align="right">2
</TD></TR><TR><TD>Enzyme: MluI (no.Lab:40)</TD><TD align="right">1</TD><TD align="right">-
</TD></TR><TR><TD>H2O</TD><TD align="right">9,8</TD><TD align="right">10,4
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"><B>20</B></TD><TD align="right"><B>20</B></TD></TR></TBODY></TABLE><LI> Incubation: 1 h at 37°C


1% Agarose gel
</LI><LI>1% agarose gel was repared, gel ran for 45 minutes(119 V)

Results: Test digestion of pCMV_MCS with NotI delivered plausibel results (expected fragment size: ~ 27 kbp and 17 kbp). 
Unfortunately the digestion of pCMV_mVenus_YFP with PstI and MluI resulted in one 2.9 kbp and one 1.2 kbp fragment, which didn't correspond to the expected fragment sizes of 19 kbp and 33 kbp - but to fragment sizes of pCMV_MCS without mVenus_YFP! 
<H4><SPAN class="mw-headline" id="Ordering_oligos"><P style="font-size:15px; background-color:#66bbFF;"><B>Ordering oligos</B></P></SPAN></H4><P>Bea
Oligos have been ordered for modifying the ITRs. Deletion of PstI restriction site in ITR. </P><P><B>right ITR of pAAV_MCS</B></P><P>oligos ordered:</P><P>fwd:	5´- gcgcagctgcctgcaCGGGCGCCTGATGCGG -3´ 77 °C, 31 bp 
rev:    5´- CCGCATCAGGCGCCCGTGCAGGCAGCTGCGC -3´ </P><P><B>leftITR of pAAV_MCS</B></P><P>oligos ordered:</P><P>fwd:	5´- CCTTTTGCTCACATGTCGTGCAGGCAGCTGCGCG -3´ 74 °C, 34 bp 
rev:	5´- CGCGCAGCTGCCTGCACGACATGTGAGCAAAAGG -3´ 
For further details see link </P><P><A href="http://www.molbiotech.uni-freiburg.de/iGEM/wiki2010/images/4/44/Freiburg10_Oligos_ITR_mutagenesis_for_pAAV_delete_PstI.pdf" class="external free" rel="nofollow">http://www.molbiotech.uni-freiburg.de/iGEM/wiki2010/images/4/44/Freiburg10_Oligos_ITR_mutagenesis_for_pAAV_delete_PstI.pdf</A>&lt;/li&gt;</P><H4><SPAN class="mw-headline" id="Test-Trafo_of_chemical_competent_E.coli_cells"><P style="font-size:15px; background-color:#66bbFF;"><B>Test-Trafo of chemical competent E.coli cells</B></P></SPAN></H4><P>Hanna

The competent E.coli XL-1-blue and competent E.coli BL21 which were prepared the day before were test-transformed with pUC18 - following the standard protocol. Cells were plated on agar plates containing ampicillin and stored over night at 37°C. 
Further on two control plates containing ampicillin or kanamycin were prepared. Non-transformed cells were plated on them and also stored over night at 37°C.   
</P><H3><SPAN class="mw-headline" id="16._Labortag_27.05.2010"><P style="font-size:17px; background-color:#00dd77;">16. Labortag 27.05.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Cell_counting"><P style="font-size:15px; background-color:#66bbFF;"><B>Cell counting</B></P></SPAN></H4><P><B>Investigator: Patrick, Adrian, Chris W. Christian L.</B></P></LI><LI> Meeting
</LI><P>cell counting:
</P><UL><LI> BL21 + PUC18 Trafo 54*4 = 216
</LI><LI> XL1B + PUC18 Trafo 30*4 = 120
</LI></UL><P>digitalization of recipe-cards (Christian L.)
</P><H3><SPAN class="mw-headline" id="17._Labortag_31.05.2010"><P style="font-size:17px; background-color:#00dd77;">17. Labortag 31.05.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="pAAV_RC_R585A_and_R588A_transistions"><P style="font-size:15px; background-color:#66bbFF;"><B>pAAV_RC R585A and R588A transistions</B></P></SPAN></H4><P>In order to alter the tropism of AAV2 several modifications have to be performed.
First of all the binding to Heparan Sulfate Proteoglycan has to be disrupted. This can be done by R585A and R588A transistions:
</P><CENTER><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/June" class="external text" rel="nofollow"><B>=&gt; Go to Labjournal June (labday 18 - 45)</B></A></CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/May">http://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/May</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg_Bioware/NoteBook/Labjournal/May" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg_Bioware/NoteBook/Labjournal/May" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/May&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/May&amp;oldid=209423" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>