<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Bioware_NoteBook_Labjournal_September2"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg_Bioware/NoteBook/Labjournal/September2">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg_Bioware/NoteBook/Labjournal/September2&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/September2&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/September2&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg_Bioware/NoteBook/Labjournal/September2" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg Bioware/NoteBook/Labjournal/September2</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="menu"><UL id="menu"><LI><DL><DT class="menu_button_left"><A href="https://2010.igem.org/Team:Freiburg_Bioware">Home</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Team_2010">Team</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Team_2010">Team 2010</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Photo_Gallery">Photo Gallery</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Cuckoo_Clock">Cuckoo Clock Competition</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Collaboration">Collaboration</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Example_Page">Example Page</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project">Project</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project">Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#highlights">Highlights</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#highlights">Results</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#modularisation">   Modularisation</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#targeting">   Targeting</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#arming">   Arming</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#superconstruct">   All-in-One</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Methods">Methods</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Virus_Construction_Kit">Virus Construction Kit Manual</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks">BioBricks</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks">Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/capsid_coding">Capsid Coding Parts</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/vector_plasmid">Vector Plasmid Parts</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/miscellaneous">Miscellaneous Parts</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A class="current" href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook">Notebook</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal">Labjournal</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Materials">Materials</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Methods">Methods</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Experiments">Experiments</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Meetings">Meetings</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Statistics">Statistics</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Sponsors">Sponsors</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Safety">Safety</A></DT></DL></LI><LI><DL><DT class="menu_button_right"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling">Modeling</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling">Modeling Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Virus_Production">Virus Production</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Virus_Infection">Virus Infection</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Structure_Modeling">Structure Modeling</A></DD></DL></LI></UL></DIV><DIV class="pageContent"><P><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook" class="external text" rel="nofollow">=&gt; Back to Notebook overview</A></P><DIV class="box_right"><LEFT><U1>NoteBook Navigator</U1></LEFT><UL><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/March">March (labday 1)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/April">April (labday 2 - 5)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/May">May (labday 6 - 17)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/June">June (labday 18 - 45)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/July">July (labday 46 - 75)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/August">August part 1 (labday 76 - 92)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/August2">August part 2 (labday 93 - 106)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/September">September part 1 (labday 107 - 123)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/September2">September part 2 (labday 124 - 135)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/October">October part 1 (labday 136 - 149 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/October2">October part 2 (labday 150 - 166 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/November">November  (labday 167 - 170 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/Cellculture">Cellculture</A></LI></UL></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#September"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">September</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#124._labday_19.09.2010"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">124. labday 19.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Continuation_with_Colony-PCR_of_pSB1C3_CD_and_ViralBrick_motif_Z34C"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Continuation with Colony-PCR of pSB1C3_CD and ViralBrick motif Z34C</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Testtransduction_of_the_final_modified_capsid_coding_constructs"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Testtransduction of the final modified capsid coding constructs</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Miniprep_of_serveral_constructs"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">Miniprep of serveral constructs</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-6"><A href="#125._labday_20.09.2010"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">125. labday 20.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Picking_the_clones_and_preparing_the_over-night_culture"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Picking the clones and preparing the over-night culture</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Minipreps_and_Sequencing_of_ViralBrick_clones"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Minipreps and Sequencing of ViralBrick clones</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Cloning_of_pCerulean_Affibody_VP2.2F3_and_pSB1C3_Affibody_middlelinker_GFP_his"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">Cloning of pCerulean_Affibody_VP2/3 and pSB1C3_Affibody_middlelinker_GFP_his</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Cloning_of_P5_TATAless_downstream_of_pSB1C3_001_RC_insrepcap_KpnI_back_and_P5_upstream_of_pSB1C3_001_RC_insrepcap_KpnI_back"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">Cloning of P5_TATAless downstream of pSB1C3_001_RC_insrepcap_KpnI_back and P5 upstream of pSB1C3_001_RC_insrepcap_KpnI_back</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Test_digestion_of_VP2-N-terminal_fusion_constructs"><SPAN class="tocnumber">1.2.5</SPAN><SPAN class="toctext">Test digestion of VP2-N-terminal fusion constructs</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#BioBrick_production:_PCR_of_pAAV_RC_InsRepCap_KpnIback_VP1-ko_and_VP2-ko"><SPAN class="tocnumber">1.2.6</SPAN><SPAN class="toctext">BioBrick production: PCR of pAAV_RC_InsRepCap_KpnIback_VP1-ko and VP2-ko</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#2x_repetition_of_CD_biobrick"><SPAN class="tocnumber">1.2.7</SPAN><SPAN class="toctext">2x repetition of CD biobrick</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#126._labday_21.09.2010"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">126. labday 21.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Trafo_evalutation_of_CD_biobrick_plate"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Trafo evalutation of CD biobrick plate</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Continuation_of_cloning_of_pCerulean_Affibody_VP2.2F3_and_pSB1C3_Affibody_middlelinker_GFP_his"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Continuation of cloning of pCerulean_Affibody_VP2/3 and pSB1C3_Affibody_middlelinker_GFP_his</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Sequencing_result_of_pSB1C3_EGFP_His_upper_and_.28.21.29_lower_band"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">Sequencing result of pSB1C3_EGFP_His upper and (!) lower band</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#Midi-Prep_of_pHelper_.26_pSB1C3_leftITR_CMV_beta-globin_mVenus_hGH_rightITR"><SPAN class="tocnumber">1.3.4</SPAN><SPAN class="toctext">Midi-Prep of pHelper &amp; pSB1C3_leftITR_CMV_beta-globin_mVenus_hGH_rightITR</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#New_aliquots_.2860.C2.B5l.29_of_XL1blue_can_be_used_.21_.28Jessica.29"><SPAN class="tocnumber">1.3.5</SPAN><SPAN class="toctext">New aliquots (60µl) of XL1blue can be used !  (Jessica)</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#127._labday_22.09.2010"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">127. labday 22.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#Sequencing_results_of_ViralBrick_ligations_using_dephosphorylated_vector"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">Sequencing results of ViralBrick ligations using dephosphorylated vector</SPAN></A></LI><LI class="toclevel-3 tocsection-22"><A href="#Cloning_pSB1C3_Viral_Brick_587KO-empty_.28B274.2FP542.29_into_pSB1C3_VP2.2F3_Capins_.28B438.2FP514.29"><SPAN class="tocnumber">1.4.2</SPAN><SPAN class="toctext">Cloning pSB1C3_Viral Brick 587KO-empty (B274/P542) into pSB1C3_VP2/3_Capins (B438/P514)</SPAN></A></LI><LI class="toclevel-3 tocsection-23"><A href="#Cellculture:_Serum-free_cells"><SPAN class="tocnumber">1.4.3</SPAN><SPAN class="toctext">Cellculture: Serum-free cells</SPAN></A></LI><LI class="toclevel-3 tocsection-24"><A href="#PCR_of_P40"><SPAN class="tocnumber">1.4.4</SPAN><SPAN class="toctext">PCR of P40</SPAN></A></LI><LI class="toclevel-3 tocsection-25"><A href="#Mini-Prep_of_pSB1C3_001_RC_InsRepCap_KpnIback_VP1-ko"><SPAN class="tocnumber">1.4.5</SPAN><SPAN class="toctext">Mini-Prep of pSB1C3_001_RC_InsRepCap_KpnIback_VP1-ko</SPAN></A></LI><LI class="toclevel-3 tocsection-26"><A href="#Cloning_of_CMV_promotor_into_pSB1C3_VP123_inscap"><SPAN class="tocnumber">1.4.6</SPAN><SPAN class="toctext">Cloning of CMV promotor into pSB1C3_VP123_inscap</SPAN></A></LI><LI class="toclevel-3 tocsection-27"><A href="#Sequencing_of_Zegfr:1907-Linker_Library.2C_His-Tag_and_imaging_appoaches"><SPAN class="tocnumber">1.4.7</SPAN><SPAN class="toctext">Sequencing of Zegfr:1907-Linker Library, His-Tag and imaging appoaches</SPAN></A></LI><LI class="toclevel-3 tocsection-28"><A href="#Mass_Midi-Prep"><SPAN class="tocnumber">1.4.8</SPAN><SPAN class="toctext">Mass Midi-Prep</SPAN></A></LI><LI class="toclevel-3 tocsection-29"><A href="#RNA_isolation_and_cDNA_synthesis"><SPAN class="tocnumber">1.4.9</SPAN><SPAN class="toctext">RNA isolation and cDNA synthesis</SPAN></A></LI><LI class="toclevel-3 tocsection-30"><A href="#cDNA_synthesis"><SPAN class="tocnumber">1.4.10</SPAN><SPAN class="toctext">cDNA synthesis</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#128._labday_23.09.2010"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">128. labday 23.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-32"><A href="#PCR_of_VB_constructs_to_remove_mutations"><SPAN class="tocnumber">1.5.1</SPAN><SPAN class="toctext">PCR of VB constructs to remove mutations</SPAN></A></LI><LI class="toclevel-3 tocsection-33"><A href="#Colony_PCR_of_new_clones_from_the_last_ligations"><SPAN class="tocnumber">1.5.2</SPAN><SPAN class="toctext">Colony PCR of new clones from the last ligations</SPAN></A></LI><LI class="toclevel-3 tocsection-34"><A href="#Contuniation:_Cloning_pSB1C3_Viral_Brick_587KO-empty_.28B274.2FP542.29_into_pSB1C3_VP2.2F3_Capins_.28B438.2FP514.29"><SPAN class="tocnumber">1.5.3</SPAN><SPAN class="toctext">Contuniation: Cloning pSB1C3_Viral Brick 587KO-empty (B274/P542) into pSB1C3_VP2/3_Capins (B438/P514)</SPAN></A></LI><LI class="toclevel-3 tocsection-35"><A href="#Mini-Prep_and_test_digestion_of_pCerulean_Affibody_VP2.2F3.2C_pSB1C3_Affibody_middlelinker_EGFP_His.2C_pSB1C3_001_RC_InsRepCap_KpnIback_VP2-ko"><SPAN class="tocnumber">1.5.4</SPAN><SPAN class="toctext">Mini-Prep and test digestion of pCerulean_Affibody_VP2/3, pSB1C3_Affibody_middlelinker_EGFP_His, pSB1C3_001_RC_InsRepCap_KpnIback_VP2-ko</SPAN></A></LI><LI class="toclevel-3 tocsection-36"><A href="#PCR_for_587_Z34C_ViralBricks"><SPAN class="tocnumber">1.5.5</SPAN><SPAN class="toctext">PCR for 587 Z34C ViralBricks</SPAN></A></LI><LI class="toclevel-3 tocsection-37"><A href="#PCR_of_Vp1_ex.2C_Vp2_ex_and_Vp3_ex_and_cloning_into_pSB1C3_001"><SPAN class="tocnumber">1.5.6</SPAN><SPAN class="toctext">PCR of Vp1_ex, Vp2_ex and Vp3_ex and cloning into pSB1C3_001</SPAN></A></LI><LI class="toclevel-3 tocsection-38"><A href="#RNA_Purity_Gel"><SPAN class="tocnumber">1.5.7</SPAN><SPAN class="toctext">RNA Purity Gel</SPAN></A></LI><LI class="toclevel-3 tocsection-39"><A href="#hTERT_PCR_Reaction"><SPAN class="tocnumber">1.5.8</SPAN><SPAN class="toctext">hTERT PCR Reaction</SPAN></A></LI><LI class="toclevel-3 tocsection-40"><A href="#Cloning_of_pBAD_Affibody_middlelinker_EGFP_His"><SPAN class="tocnumber">1.5.9</SPAN><SPAN class="toctext">Cloning of pBAD_Affibody_middlelinker_EGFP_His</SPAN></A></LI><LI class="toclevel-3 tocsection-41"><A href="#Picking_clones_of_pSB1C3_CMV_VP123_inscap"><SPAN class="tocnumber">1.5.10</SPAN><SPAN class="toctext">Picking clones of pSB1C3_CMV_VP123_inscap</SPAN></A></LI><LI class="toclevel-3 tocsection-42"><A href="#Cloning_of_.2A.2A.2A.2A.2A_into_pSB1C3_CFP"><SPAN class="tocnumber">1.5.11</SPAN><SPAN class="toctext">Cloning of ***** into pSB1C3_CFP</SPAN></A></LI><LI class="toclevel-3 tocsection-43"><A href="#Repetition:_Cloning_of_P5_TATAless_downstream_of_pSB1C3_001_RC_insrepcap_KpnI_back_and_P5_upstream_of_pSB1C3_001_RC_insrepcap_KpnI_back"><SPAN class="tocnumber">1.5.12</SPAN><SPAN class="toctext">Repetition: Cloning of P5_TATAless downstream of pSB1C3_001_RC_insrepcap_KpnI_back and P5 upstream of pSB1C3_001_RC_insrepcap_KpnI_back</SPAN></A></LI><LI class="toclevel-2 tocsection-44"><A href="#129._labday_24.09.2010"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">129. labday 24.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-45"><A href="#Sequencing_results_of_VB_13.4"><SPAN class="tocnumber">1.6.1</SPAN><SPAN class="toctext">Sequencing results of VB 13.4</SPAN></A></LI><LI class="toclevel-3 tocsection-46"><A href="#Contuniation:_Cloning_pSB1C3_Viral_Brick_587KO-empty_.28B274.2FP542.29_into_pSB1C3_001_VP2.2F3_Capins_.28B438.2FP514.29"><SPAN class="tocnumber">1.6.2</SPAN><SPAN class="toctext">Contuniation: Cloning pSB1C3_Viral Brick 587KO-empty (B274/P542) into pSB1C3_001_VP2/3_Capins (B438/P514)</SPAN></A></LI><LI class="toclevel-3 tocsection-47"><A href="#Sequencing_evaluation_of_VP2.2F3_Gsat-linker_and_VP2.2F3_cap_Gsat-linker"><SPAN class="tocnumber">1.6.3</SPAN><SPAN class="toctext">Sequencing evaluation of VP2/3_Gsat-linker and VP2/3_cap_Gsat-linker</SPAN></A></LI><LI class="toclevel-3 tocsection-48"><A href="#New_CD_primer_ordered"><SPAN class="tocnumber">1.6.4</SPAN><SPAN class="toctext">New CD primer ordered</SPAN></A></LI><LI class="toclevel-3 tocsection-49"><A href="#Sequencing_of_pCerulean_Zegfr:1907_VP2.2F3"><SPAN class="tocnumber">1.6.5</SPAN><SPAN class="toctext">Sequencing of pCerulean_Zegfr:1907_VP2/3</SPAN></A></LI><LI class="toclevel-3 tocsection-50"><A href="#Repetition:_PCR_for_biobrick_production:_Cloning_RepCap_into_pSB1C3"><SPAN class="tocnumber">1.6.6</SPAN><SPAN class="toctext">Repetition: PCR for biobrick production: Cloning RepCap into pSB1C3</SPAN></A></LI><LI class="toclevel-3 tocsection-51"><A href="#Picking_clones_of_pSB1C3_CFP_email"><SPAN class="tocnumber">1.6.7</SPAN><SPAN class="toctext">Picking clones of pSB1C3_CFP_email</SPAN></A></LI><LI class="toclevel-3 tocsection-52"><A href="#MiniPreps_and_Test_digestion_of_pSB1C3_CMV_VP123_capins"><SPAN class="tocnumber">1.6.8</SPAN><SPAN class="toctext">MiniPreps and Test digestion of pSB1C3_CMV_VP123_capins</SPAN></A></LI><LI class="toclevel-3 tocsection-53"><A href="#MiniPreps_of_587_Z34C_ViralBricks_and_pSB1C3_p40"><SPAN class="tocnumber">1.6.9</SPAN><SPAN class="toctext">MiniPreps of 587 Z34C ViralBricks and pSB1C3_p40</SPAN></A></LI><LI class="toclevel-3 tocsection-54"><A href="#Re-Trafo_of_DARPin"><SPAN class="tocnumber">1.6.10</SPAN><SPAN class="toctext">Re-Trafo of DARPin</SPAN></A></LI><LI class="toclevel-2 tocsection-55"><A href="#130._labday_25.09.2010"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">130. labday 25.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-56"><A href="#Sequencing_of_VB:_13.1.2C_13.4_.28approach_w._dephosphorylated_vector.2C_second_clone_picking_round"><SPAN class="tocnumber">1.7.1</SPAN><SPAN class="toctext">Sequencing of VB: 13.1, 13.4 (approach w. dephosphorylated vector, second clone picking round</SPAN></A></LI><LI class="toclevel-3 tocsection-57"><A href="#Miniprep_of_serveral_constructs_2"><SPAN class="tocnumber">1.7.2</SPAN><SPAN class="toctext">Miniprep of serveral constructs</SPAN></A></LI><LI class="toclevel-3 tocsection-58"><A href="#Sequencing_results_of_pSb1C3_Affibody_middlelinker_EGFP_His"><SPAN class="tocnumber">1.7.3</SPAN><SPAN class="toctext">Sequencing results of pSb1C3_Affibody_middlelinker_EGFP_His</SPAN></A></LI><LI class="toclevel-3 tocsection-59"><A href="#Sequencing_result_of_pSB1C3_CMV_VP123_capins_clone1"><SPAN class="tocnumber">1.7.4</SPAN><SPAN class="toctext">Sequencing result of pSB1C3_CMV_VP123_capins_clone1</SPAN></A></LI><LI class="toclevel-3 tocsection-60"><A href="#Cloning_of_453_and_587_ViralBricks_into_pSB1C3_CMV_VP123_capins"><SPAN class="tocnumber">1.7.5</SPAN><SPAN class="toctext">Cloning of 453 and 587 ViralBricks into pSB1C3_CMV_VP123_capins</SPAN></A></LI><LI class="toclevel-3 tocsection-61"><A href="#Sequencing_results:_pSB1C3_001_VP2.2F3_HSPG-KO"><SPAN class="tocnumber">1.7.6</SPAN><SPAN class="toctext">Sequencing results: pSB1C3_001_VP2/3_HSPG-KO</SPAN></A></LI><LI class="toclevel-3 tocsection-62"><A href="#Cloning_of_pBAD_Affibody_middlelinker_EGFP_His_and_pSB1C3_Affibody_middlelinker_EGFP_His"><SPAN class="tocnumber">1.7.7</SPAN><SPAN class="toctext">Cloning of pBAD_Affibody_middlelinker_EGFP_His and pSB1C3_Affibody_middlelinker_EGFP_His</SPAN></A></LI><LI class="toclevel-2 tocsection-63"><A href="#131._labday_26.09.2010"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">131. labday 26.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-64"><A href="#Continuation_of_cloning_of_pBAD_Affibody_middlelinker_EGFP_His_and_pSB1C3_Affibody_middlelinker_EGFP_His"><SPAN class="tocnumber">1.8.1</SPAN><SPAN class="toctext">Continuation of cloning of pBAD_Affibody_middlelinker_EGFP_His and pSB1C3_Affibody_middlelinker_EGFP_His</SPAN></A></LI><LI class="toclevel-3 tocsection-65"><A href="#Picking_clones_of_different_constructs"><SPAN class="tocnumber">1.8.2</SPAN><SPAN class="toctext">Picking clones of different constructs</SPAN></A></LI><LI class="toclevel-3 tocsection-66"><A href="#Continuation_of_cloning_of_453_and_587_inserts_into_pSB1C3_CMV_VP123_capins"><SPAN class="tocnumber">1.8.3</SPAN><SPAN class="toctext">Continuation of cloning of 453 and 587 inserts into pSB1C3_CMV_VP123_capins</SPAN></A></LI><LI class="toclevel-3 tocsection-67"><A href="#Colony_PCR_of_remaining_587_constructs"><SPAN class="tocnumber">1.8.4</SPAN><SPAN class="toctext">Colony PCR of remaining 587 constructs</SPAN></A></LI><LI class="toclevel-3 tocsection-68"><A href="#Sequencing_results:_pCerulean_Zegfr:1907_VP2.2F3"><SPAN class="tocnumber">1.8.5</SPAN><SPAN class="toctext">Sequencing results: pCerulean_Zegfr:1907_VP2/3</SPAN></A></LI><LI class="toclevel-3 tocsection-69"><A href="#N-terminal_VP2-Fusion_.26_VP1_Insertion:_Fusion_of_VP2.2F3_HSPG-KO"><SPAN class="tocnumber">1.8.6</SPAN><SPAN class="toctext">N-terminal VP2-Fusion &amp; VP1 Insertion: Fusion of VP2/3_HSPG-KO</SPAN></A></LI><LI class="toclevel-2 tocsection-70"><A href="#132._labday_27.09.2010"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">132. labday 27.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-71"><A href="#Sent_for_sequencing"><SPAN class="tocnumber">1.9.1</SPAN><SPAN class="toctext">Sent for sequencing</SPAN></A></LI><LI class="toclevel-3 tocsection-72"><A href="#Picking_clones_of_VP1_insertion_and_VP2_fusion_HSPG-KO_approaches"><SPAN class="tocnumber">1.9.2</SPAN><SPAN class="toctext">Picking clones of VP1 insertion and VP2 fusion HSPG-KO approaches</SPAN></A></LI><LI class="toclevel-3 tocsection-73"><A href="#MiniPreps_of_different_constructs"><SPAN class="tocnumber">1.9.3</SPAN><SPAN class="toctext">MiniPreps of different constructs</SPAN></A></LI><LI class="toclevel-3 tocsection-74"><A href="#Miniprep_of_serveral_constructs_3"><SPAN class="tocnumber">1.9.4</SPAN><SPAN class="toctext">Miniprep of serveral constructs</SPAN></A></LI><LI class="toclevel-3 tocsection-75"><A href="#Re-Trafo_of_pSB1C3_pTAV_.28P5.29"><SPAN class="tocnumber">1.9.5</SPAN><SPAN class="toctext">Re-Trafo of pSB1C3_pTAV (P5)</SPAN></A></LI><LI class="toclevel-3 tocsection-76"><A href="#Repetition:_Cloning_of_P5_TATAless_downstream_of_pSB1C3_001_RC_insrepcap_KpnI_back_and_P5_upstream_of_pSB1C3_001_RC_insrepcap_KpnI_back_2"><SPAN class="tocnumber">1.9.6</SPAN><SPAN class="toctext">Repetition: Cloning of P5_TATAless downstream of pSB1C3_001_RC_insrepcap_KpnI_back and P5 upstream of pSB1C3_001_RC_insrepcap_KpnI_back</SPAN></A></LI><LI class="toclevel-3 tocsection-77"><A href="#Picking_clones_of_VP1_insertion_and_VP2_fusion_HSPG-KO_approaches_2"><SPAN class="tocnumber">1.9.7</SPAN><SPAN class="toctext">Picking clones of VP1 insertion and VP2 fusion HSPG-KO approaches</SPAN></A></LI><LI class="toclevel-3 tocsection-78"><A href="#Picking_clones_of_VP1_insertion_and_VP2_fusion_HSPG-KO_approaches_3"><SPAN class="tocnumber">1.9.8</SPAN><SPAN class="toctext">Picking clones of VP1 insertion and VP2 fusion HSPG-KO approaches</SPAN></A></LI><LI class="toclevel-2 tocsection-79"><A href="#133._labday_28.09.2010"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">133. labday 28.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-80"><A href="#Cloning_of_the_ViralBricks_BAP-HSPG-ko_and_His-HSPG-ko_into_pSB1C3_VP2.2F3"><SPAN class="tocnumber">1.10.1</SPAN><SPAN class="toctext">Cloning of the ViralBricks BAP-HSPG-ko and His-HSPG-ko into pSB1C3_VP2/3</SPAN></A></LI><LI class="toclevel-3 tocsection-81"><A href="#Test_digestion:_ViralBricks_in_pSB1C3_CMV_VP123_capins"><SPAN class="tocnumber">1.10.2</SPAN><SPAN class="toctext">Test digestion: ViralBricks in pSB1C3_CMV_VP123_capins</SPAN></A></LI><LI class="toclevel-3 tocsection-82"><A href="#Sequencing_result_of_pSB1C3_email_and_pBAD_Affibody_middlelinker_EGFP_His"><SPAN class="tocnumber">1.10.3</SPAN><SPAN class="toctext">Sequencing result of pSB1C3_email and pBAD_Affibody_middlelinker_EGFP_His</SPAN></A></LI><LI class="toclevel-3 tocsection-83"><A href="#Preparation_of_PBS_Buffer_and_PBS_Buffer_with_Imidazole"><SPAN class="tocnumber">1.10.4</SPAN><SPAN class="toctext">Preparation of PBS Buffer and PBS Buffer with Imidazole</SPAN></A></LI><LI class="toclevel-3 tocsection-84"><A href="#ViralBrick_Sequencing_results:_pSB1C3_587_KO_Z34C_and_pSB1C3_587_KO_Z34C_Spacer"><SPAN class="tocnumber">1.10.5</SPAN><SPAN class="toctext">ViralBrick Sequencing results: pSB1C3_587_KO_Z34C and pSB1C3_587_KO_Z34C_Spacer</SPAN></A></LI><LI class="toclevel-3 tocsection-85"><A href="#Cloning_of_Darpin_into_pCerulean_Vp1up_NLS_and_pSB1C3"><SPAN class="tocnumber">1.10.6</SPAN><SPAN class="toctext">Cloning of Darpin into pCerulean_Vp1up_NLS and pSB1C3</SPAN></A></LI><LI class="toclevel-3 tocsection-86"><A href="#Cloning_of_pSB1C3_001_CMV_and_pSB1C3_001_Vp2.2F3_HSPG-Ko_CMV_ZEGFR:1907"><SPAN class="tocnumber">1.10.7</SPAN><SPAN class="toctext">Cloning of pSB1C3_001_CMV and pSB1C3_001_Vp2/3_HSPG-Ko_CMV_ZEGFR:1907</SPAN></A></LI><LI class="toclevel-3 tocsection-87"><A href="#CD_assembly_with_new_suffix_primer"><SPAN class="tocnumber">1.10.8</SPAN><SPAN class="toctext">CD assembly with new suffix primer</SPAN></A></LI><LI class="toclevel-3 tocsection-88"><A href="#test_digestion_of_all_CD_constructs"><SPAN class="tocnumber">1.10.9</SPAN><SPAN class="toctext">test digestion of all CD constructs</SPAN></A></LI><LI class="toclevel-3 tocsection-89"><A href="#Cloning_of_pSB1C3_Affibody_middlelinker_EGFP_His"><SPAN class="tocnumber">1.10.10</SPAN><SPAN class="toctext">Cloning of pSB1C3_Affibody_middlelinker_EGFP_His</SPAN></A></LI><LI class="toclevel-3 tocsection-90"><A href="#Sequencing_analysis_of_pSB1C3_001_RC_IRCK.2C_pSB1C3_001_RC_IRCK_VP1-ko_P5tataless_and_pSB1C3_001_RC_IRCK_VP2-ko_P5tataless"><SPAN class="tocnumber">1.10.11</SPAN><SPAN class="toctext">Sequencing analysis of pSB1C3_001_RC_IRCK, pSB1C3_001_RC_IRCK_VP1-ko_P5tataless and pSB1C3_001_RC_IRCK_VP2-ko_P5tataless</SPAN></A></LI><LI class="toclevel-3 tocsection-91"><A href="#Cloning_of_HSPG_KO_empty_into_pSB1C3_001_RC_IRCK_VP1-ko_P5tataless_and_pSB1C3_001_RC_IRCK_VP2-ko_P5tataless"><SPAN class="tocnumber">1.10.12</SPAN><SPAN class="toctext">Cloning of HSPG_KO_empty into pSB1C3_001_RC_IRCK_VP1-ko_P5tataless and pSB1C3_001_RC_IRCK_VP2-ko_P5tataless</SPAN></A></LI><LI class="toclevel-3 tocsection-92"><A href="#Cloning_of_587_KO_Z34C_and_587_KO_Z34C_Spacer_into_pSB1C3_001_CMV_VP123_capins"><SPAN class="tocnumber">1.10.13</SPAN><SPAN class="toctext">Cloning of 587_KO_Z34C and 587_KO_Z34C_Spacer into pSB1C3_001_CMV_VP123_capins</SPAN></A></LI><LI class="toclevel-3 tocsection-93"><A href="#Mini-Prep_and_Test-digestion_of_VP1_insertion_and_VP2_fusion_HSPG-KO_approaches"><SPAN class="tocnumber">1.10.14</SPAN><SPAN class="toctext">Mini-Prep and Test-digestion of VP1 insertion and VP2 fusion HSPG-KO approaches</SPAN></A></LI><LI class="toclevel-2 tocsection-94"><A href="#134._labday_29.09.2010"><SPAN class="tocnumber">1.11</SPAN><SPAN class="toctext">134. labday 29.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-95"><A href="#CD_assembly_with_new_suffix_primer_2"><SPAN class="tocnumber">1.11.1</SPAN><SPAN class="toctext">CD assembly with new suffix primer</SPAN></A></LI><LI class="toclevel-3 tocsection-96"><A href="#Miniprep_of_serveral_constructs_4"><SPAN class="tocnumber">1.11.2</SPAN><SPAN class="toctext">Miniprep of serveral constructs</SPAN></A></LI><LI class="toclevel-3 tocsection-97"><A href="#mGMK_TK30_approaches"><SPAN class="tocnumber">1.11.3</SPAN><SPAN class="toctext">mGMK_TK30 approaches</SPAN></A></LI><LI class="toclevel-3 tocsection-98"><A href="#Design_of_mutagenesis_primers_for_deletion_of_PstI_in_mGMK_TK30"><SPAN class="tocnumber">1.11.4</SPAN><SPAN class="toctext">Design of mutagenesis primers for deletion of PstI in mGMK_TK30</SPAN></A></LI><LI class="toclevel-3 tocsection-99"><A href="#Cloning_of_Darpin_into_pCerulean_Vp1up_NLS_and_pSB1C3_Part_Two"><SPAN class="tocnumber">1.11.5</SPAN><SPAN class="toctext">Cloning of Darpin into pCerulean_Vp1up_NLS and pSB1C3 Part Two</SPAN></A></LI><LI class="toclevel-3 tocsection-100"><A href="#Cloning_of_pGGTBT7_Affibody_middlelinker_EGFP_His"><SPAN class="tocnumber">1.11.6</SPAN><SPAN class="toctext">Cloning of pGGTBT7_Affibody_middlelinker_EGFP_His</SPAN></A></LI><LI class="toclevel-3 tocsection-101"><A href="#Cloning_of_CMV_promotor_and_Affibody_into_pSB1C3_001_VP2.2F3_capins_and_pSB1C3_001_VP2.2F3_HSPG_KO"><SPAN class="tocnumber">1.11.7</SPAN><SPAN class="toctext">Cloning of CMV promotor and Affibody into pSB1C3_001_VP2/3_capins and pSB1C3_001_VP2/3_HSPG_KO</SPAN></A></LI><LI class="toclevel-3 tocsection-102"><A href="#Cloning_of_CFP_into_pSB1C3_lITR_phTERT_beta-globin"><SPAN class="tocnumber">1.11.8</SPAN><SPAN class="toctext">Cloning of CFP into pSB1C3_lITR_phTERT_beta-globin</SPAN></A></LI><LI class="toclevel-3 tocsection-103"><A href="#Sequencing_results:_VP1_insertion_and_VP2_fusion:_Fusion_of_VP2.2F3_HSPG-KO"><SPAN class="tocnumber">1.11.9</SPAN><SPAN class="toctext">Sequencing results: VP1 insertion and VP2 fusion: Fusion of VP2/3_HSPG-KO</SPAN></A></LI><LI class="toclevel-3 tocsection-104"><A href="#Cloning_of_HSPG_KO_empty_into_pSB1C3_001_RC_IRCK_P5tataless"><SPAN class="tocnumber">1.11.10</SPAN><SPAN class="toctext">Cloning of HSPG_KO_empty into pSB1C3_001_RC_IRCK_P5tataless</SPAN></A></LI><LI class="toclevel-3 tocsection-105"><A href="#Mass_Midi-Prep_II"><SPAN class="tocnumber">1.11.11</SPAN><SPAN class="toctext">Mass Midi-Prep II</SPAN></A></LI><LI class="toclevel-2 tocsection-106"><A href="#135._labday_30.09.2010"><SPAN class="tocnumber">1.12</SPAN><SPAN class="toctext">135. labday 30.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-107"><A href="#picking_CD_clones"><SPAN class="tocnumber">1.12.1</SPAN><SPAN class="toctext">picking CD clones</SPAN></A></LI><LI class="toclevel-3 tocsection-108"><A href="#ViralBricks_BAP-ko_and_His-ko_in_VP23"><SPAN class="tocnumber">1.12.2</SPAN><SPAN class="toctext">ViralBricks BAP-ko and His-ko in VP23</SPAN></A></LI><LI class="toclevel-3 tocsection-109"><A href="#Cloning_of_Darpin_into_pCerulean_Vp1up_NLS_and_pSB1C3_Part_Three"><SPAN class="tocnumber">1.12.3</SPAN><SPAN class="toctext">Cloning of Darpin into pCerulean_Vp1up_NLS and pSB1C3 Part Three</SPAN></A></LI><LI class="toclevel-3 tocsection-110"><A href="#Test_digestion_of_pCerulean_ZEGFR:1907_MiddleLinker_VP2.2F3_HSPG-KO_clone_2"><SPAN class="tocnumber">1.12.4</SPAN><SPAN class="toctext">Test digestion of pCerulean_Z<SUB>EGFR:1907</SUB>_MiddleLinker_VP2/3_HSPG-KO clone 2</SPAN></A></LI><LI class="toclevel-3 tocsection-111"><A href="#Cloning_VP2.2F3_Capins_and_VP2.2F3_HSPG-KO_downstream_of_Middle_Linker"><SPAN class="tocnumber">1.12.5</SPAN><SPAN class="toctext">Cloning VP2/3_Capins and VP2/3_HSPG-KO downstream of Middle Linker</SPAN></A></LI><LI class="toclevel-3 tocsection-112"><A href="#Miniprep_of_serveral_constructs_5"><SPAN class="tocnumber">1.12.6</SPAN><SPAN class="toctext">Miniprep of serveral constructs</SPAN></A></LI><LI class="toclevel-3 tocsection-113"><A href="#Test_digestion_of_pSB1C3_ZEGFR:1907_middlelinker_EGFP_His_and_mCherry_iGEM_2008"><SPAN class="tocnumber">1.12.7</SPAN><SPAN class="toctext">Test digestion of pSB1C3_ZEGFR:1907 _middlelinker_EGFP_His and mCherry iGEM 2008</SPAN></A></LI><LI class="toclevel-3 tocsection-114"><A href="#Test_digestion_of_pSB1C3_001_CMV_affibody_HSPG_KO_and_pSB1C3_001_CMV_VP123_587_KO_Z34C"><SPAN class="tocnumber">1.12.8</SPAN><SPAN class="toctext">Test digestion of pSB1C3_001_CMV_affibody_HSPG_KO and  pSB1C3_001_CMV_VP123_587_KO_Z34C</SPAN></A></LI><LI class="toclevel-3 tocsection-115"><A href="#Midiprep_of_pSB1C3_001_CMV_VP1.2F2.2F3_capins_587bla_and_pSB1C3_001_RC_IRCK_P5tataless_clone1"><SPAN class="tocnumber">1.12.9</SPAN><SPAN class="toctext">Midiprep of pSB1C3_001_CMV_VP1/2/3_capins_587bla and pSB1C3_001_RC_IRCK_P5tataless clone1</SPAN></A></LI><LI class="toclevel-3 tocsection-116"><A href="#Cloning_of_VP2.2F3_BAP_HSPG-KO_and_VP2.2F3_His_HSPG-KO_into_pCerulean_Zegfr:1907_Middlelinker_and_VP2.2F3_His_HSPG-KO_into_pCerulean_VP1up_NLS_mVenus"><SPAN class="tocnumber">1.12.10</SPAN><SPAN class="toctext">Cloning of VP2/3_BAP_HSPG-KO and VP2/3_His_HSPG-KO into pCerulean_Zegfr:1907_Middlelinker and VP2/3_His_HSPG-KO into pCerulean_VP1up_NLS_mVenus</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="September">September</SPAN></H2><H3><SPAN class="mw-headline" id="124._labday_19.09.2010"><P style="font-size:17px; background-color:#00dd77;">124. labday 19.09.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Continuation_with_Colony-PCR_of_pSB1C3_CD_and_ViralBrick_motif_Z34C"><P style="font-size:15px; background-color:#66bbFF;"><B>Continuation with Colony-PCR of pSB1C3_CD and ViralBrick motif Z34C</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="color:#66bbff;"><I>Comment</I>: Since the colony PCR of the last try did not work exactly the way we expected it, another colony PCR approach with new primers was conducted.</P><P><B>Loading plan:</B><B>upper lanes</B></P><PRE>M   CD1  CD2  CD3  CD4  CD5  CD6  CD7  CD8  CD9  CD10   +ctrl(CFP) +ctrl(BAP) -ctrl(pAAV_MCS)   11Q1  12Q1  13Q1 13Q2  M
</PRE><P><B>lower lanes</B></P><PRE>M   11T41  11T42  11T43  11T44    12T41  12T42  12T43  12T44    13T41  13T42  13T43  13T44    14T41  14T42 14T43  14T44  M
</PRE><P><B>Expected sizes of the PCR products are:</B></P><UL><LI>Cytosine deaminase (CD) = 1300bp</LI><LI>Z34C loops insertion motif = 220bp</LI><LI>Positive control 1 = CFP = 750bp</LI><LI>Positive control 2 = BAP_587 = 170bp</LI></UL><P><B>Results:</B> The Colony PCR of the Cytosine Deaminase (CD) did not work out. The detectable bands at around 700 bp correspond to CFP. Therefore, BioBrick production of the CD must be repeated. The following three bands represent the controls. Three different controls were used. The first corresponds to pSB1C3_CFP, the second to BAP_587 and third control is the negative control. Unfortunately now bands can be detected in the postive controls. A faint band can be seen in the negative control which is the only lane in which no band should be detectable.
The following bands after the three controls correspond to the approaches of the Z34C Viralbrick production. Some bands in the lower and the upper lanes show positive results. The corresponding inoculated overnight culture were centrifuged and prepared in order to perform a Mini-Prep tomorrow.
</P><P style="color:#66bbff;"><I>Comment</I>: hmm, seems that the CD-assambly didn't work at all.. Should i try it again or are we gonna skip it for good due to other priorities?(kira)
 No, we cannot skip it. Can we change something else?? Any ideas what we can alter in the protocol? Because problem is that we are still waiting for the tk... so we should aswell focus on the other prodrug activating enzmye!! (Bea)</P><H4><SPAN class="mw-headline" id="Testtransduction_of_the_final_modified_capsid_coding_constructs"><P style="font-size:15px; background-color:#66bbFF;"><B>Testtransduction of the final modified capsid coding constructs</B></P></SPAN></H4><P><B>Investigator: Adrian</B></P><P>Aim of the experiment:

During the project 22 silent nucleotide exchange mutations were introduced into the capsid coding construct to make it compatible with the RFC standards and to have single cutting restriction enzymes flanking the 453 and the 587 loop sequence. Two point mutations had to be dismissed, because either a first test transduction showed that the construct was not working anymore or the insertion of the synthesized gene posed serious problems, because the restriction enzyme did not work.<B>Finally, we have a construct that is shown to produce infectious particles comparable to the current AAV systems and carries 20 point mutations. Now we can announce that the Adeno-associated Virus is compatible to the RFC standard and the idea to replace the loop sequences via ViralBricks works! </B>
Impression from the moment:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">The construct with the insertion of the Rep and the Cap synthesis worked! Reintroduction of the KpnI restriction site recovered the functionality of the viral genome!</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Miniprep_of_serveral_constructs"><P style="font-size:15px; background-color:#66bbFF;"><B>Miniprep of serveral constructs</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P><B>Glycerol stocks were prepared:</B></P><UL><LI>B430 =	pCerulean_ZEGFR:1907_Longlinker_VP2/3</LI><LI>B431 =	pCerulean_CFP_Middlelinker_VP2/3_insCap</LI><LI>B432 =	pCerulean_ZEGFR:1907_Shortlinker_VP2/3_insCap</LI></UL><P><B>Mini-Prep was performed according to the standard protocol</B></P><UL><LI>P527 =	pCerulean_ZEGFR:1907_Longlinker_VP2/3 c = 230,61 ng/µl</LI><LI>P528 =	pCerulean_CFP_Middlelinker_VP2/3_insCap c = 290,81 ng/µl</LI><LI>P529 =	pCerulean_ZEGFR:1907_Shortlinker_VP2/3_insCap c = 269,39 ng/µl</LI><P>Several other constructs were preped, but need to be confirmed before being added to plasmid/glycerol stock.
</P></UL><H3><SPAN class="mw-headline" id="125._labday_20.09.2010"><P style="font-size:17px; background-color:#00dd77;">125. labday 20.09.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Picking_the_clones_and_preparing_the_over-night_culture"><P style="font-size:15px; background-color:#66bbff;"><B>Picking the clones and preparing the over-night culture</B></P></SPAN></H4><P><B>Investigator: Kira </B></P><P>VP2/3_Gsat-linker as well as VP2/3_cap_Gsat-linker plates contain colonies, thus 3 clones from each plate were inoculated into 5 ml DYT+ 5ul Chloramphenicol and incubated @ 37 C
</P><H4><SPAN class="mw-headline" id="Minipreps_and_Sequencing_of_ViralBrick_clones"><P style="font-size:15px; background-color:#66bbff;"><B>Minipreps and Sequencing of ViralBrick clones</B></P></SPAN></H4><P><B>Investigator: Achim </B></P><P>The following clones were prepped:
</P><UL><LI>Quickligation:
</LI><LI>11.1, c = 85,41 ng/µl
</LI><LI>13.1, c = 96,93 ng/µl
</LI></UL><LI>T4-Ligation:
</LI><LI>11.4, c = 76,42 ng/µl
</LI><LI>12.1, c = 80,36 ng/µl
</LI><LI>12.4, c = 85,43 ng/µl
</LI><LI>13.1, c = 90,06 ng/µl
</LI><LI>13.2, c = 80,87 ng/µl
</LI><LI>13.4, c = 76,62 ng/µl
</LI><LI>14.1, c = 76,50 ng/µl
</LI><LI>14.2, c = 98,52 ng/µl
</LI><P>Clones 11.4, 12.1, 13.2, 14.2 (T4) were sent for sequencing.
</P><H4><SPAN class="mw-headline" id="Cloning_of_pCerulean_Affibody_VP2.2F3_and_pSB1C3_Affibody_middlelinker_GFP_his"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of pCerulean_Affibody_VP2/3 and pSB1C3_Affibody_middlelinker_GFP_his</B></P></SPAN></H4><P><B>Investigator: Jessica </B></P><P style="font-size:13px; color:red;"><B>Comment</B>:Affibody_VP2/3 will be cloned in pCerulean and Affibody_middlelinker will be cloned together with GFP_His in pSB1C3</P><UL><LI>P407= pCerulean_CFP_middlelinker c=482,76 ng/µl
</LI><LI>P516= pSB1C3_ZEGFR:1907_VP2/3 clone 1 c= 191,6 ng/µl
</LI><LI>P290= pSB1C3_Affibody_Middlelinker clone 1 c= 227,4 ng/µl
</LI><LI>P518= pSB1C3_GFP_RFC25_upper band clone 1 c= 138,3 ng/µl
</LI><LI>P520= pSB1C3_GFP_RFC25_lower band clone 1 c= 141,6 ng/µl
</LI></UL><P><B>Digestion:</B></P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">P407 </TD><TD align="right">P516 </TD><TD align="right">P290 </TD><TD align="right">P518 </TD><TD align="right">P520
</TD></TR><TR><TD> DNA  </TD><TD align="right">3,5 </TD><TD align="right"> 10 </TD><TD align="right">7 </TD><TD align="right"> 10 </TD><TD align="right"> 10
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2</TD><TD align="right"> 2</TD><TD align="right"> 2</TD><TD align="right"> 2</TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right"> 2 </TD><TD align="right"> 2</TD><TD align="right"> 2</TD><TD align="right"> 2
</TD></TR><TR><TD>EcoRI </TD><TD align="right">0,8 </TD><TD align="right">0,8</TD><TD align="right">-</TD><TD align="right">-</TD><TD align="right">-
</TD></TR><TR><TD>AgeI  </TD><TD align="right">0,8</TD><TD align="right">0,8</TD><TD align="right">0,8</TD><TD align="right">-</TD><TD align="right">-
</TD></TR><TR><TD>NgoMIV  </TD><TD align="right">-</TD><TD align="right">-</TD><TD align="right">-</TD><TD align="right">0,8</TD><TD align="right">0,8
</TD></TR><TR><TD>SpeI  </TD><TD align="right">-</TD><TD align="right">-</TD><TD align="right">0,8</TD><TD align="right">0,8</TD><TD align="right">0,8
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">10,9 </TD><TD align="right"> 4,4</TD><TD align="right"> 7,4</TD><TD align="right"> 4,4</TD><TD align="right"> 4,4
</TD></TR><TR><TD><B>Total volume </B></TD><TD align="right">20</TD><TD align="right"> 20</TD><TD align="right"> 20</TD><TD align="right"> 20</TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><P>1,0 g Agarose,100 ml TAE (1%), 6 µl GELRED , at 120 Volt

P516, P518, P520 will be repeated tomorrow (no DNA after gel-ex)
</P><P><B>Gelextraction:</B></P><P>The gelextraction was performed according to the standard protocol. DNA concentration of the extracts:
</P><UL><LI>P407  c=  6,5 ng/µl
</LI><LI>P516  c=  -   ng/µl
</LI><LI>P290  c=  7,3 ng/µl
</LI><LI>P518  c=  -   ng/µl
</LI><LI>P520  c=  -   ng/µl
</LI></UL><P>
stands in 4°C room for comtinuation tomorrow
</P><H4><SPAN class="mw-headline" id="Cloning_of_P5_TATAless_downstream_of_pSB1C3_001_RC_insrepcap_KpnI_back_and_P5_upstream_of_pSB1C3_001_RC_insrepcap_KpnI_back"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of P5_TATAless downstream of pSB1C3_001_RC_insrepcap_KpnI_back and P5 upstream of pSB1C3_001_RC_insrepcap_KpnI_back</B></P></SPAN></H4><P><B>Investigator: Anissa</B><B>Digestion of the constructs:</B></P><UL><LI>P320 = pSB1C3_001_RC_insrepcap_KpnI_back c=408 ng/µL</LI><LI>P320 = pSB1C3_001_RC_insrepcap_KpnI_back c=408 ng/µL</LI><LI>P180 = pSB1C3_pTAV2 (P5)clone 1 c=108,8 ng/µL</LI><LI>P184 = pSB1C3_pAAV_RC (P5TATAless)clone 3 c=176,9 ng/µL</LI></UL><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>v<SUB>P320 upstream</SUB> /µL</B></TD><TD align="left"><B>v<SUB>P180 upstream</SUB>/µL</B></TD><TD align="left"><B>v<SUB>P320 downsteam</SUB> /µL</B></TD><TD align="left"><B>v<SUB>P184 downsteam</SUB>/µL</B></TD></TR><TR><TD align="left"> DNA  </TD><TD align="left"> 2,5</TD><TD align="left">13,8 </TD><TD align="left"> 2,5 </TD><TD align="left">8,5
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 1,5</TD><TD align="left">2 </TD><TD align="left"> 1,5</TD><TD align="left"> 1,5
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 1,5</TD><TD align="left">2 </TD><TD align="left"> 1,5</TD><TD align="left"> 1,5
</TD></TR><TR><TD align="left"> Enzyme 1 </TD><TD align="left">XbaI 1</TD><TD align="left">SpeI 1 </TD><TD align="left">SpeI 1</TD><TD align="left">XbaI 1
</TD></TR><TR><TD align="left"> Enzyme 2 </TD><TD align="left">EcoRI 1</TD><TD align="left">EcoRI 1 </TD><TD align="left">PstI 1</TD><TD align="left">PstI 1
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 7,5</TD><TD align="left">0,2</TD><TD align="left">7,5</TD><TD align="left">1,5
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>15</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>15</B></TD><TD align="left"><B>15</B></TD></TR></TBODY></TABLE><P><B>Loading plan: </B></P><PRE>M    P320(upstream)    P320(downstream)    P180    P184
</PRE><P><B>Results:</B></P><P>
After gel extraction has been performed, the ligation was carried out.
</P><P><B>Ligation:</B></P><UL><LI>v<SUB>=P320_up </SUB>=7,37µL </LI><LI>v<SUB>=P180_up </SUB>=0,63µL</LI><LI>v<SUB>=P320_down </SUB>=7µL </LI><LI>v<SUB>=P184_down </SUB>=1µL</LI></UL><P>The ligation mix was transformed into XL1-Blue cells and plated on agar plates containing chloramphenicol.
</P><P><B>Next steps:</B>
Picking clones and perform Mini-Prep. 
</P><H4><SPAN class="mw-headline" id="Test_digestion_of_VP2-N-terminal_fusion_constructs"><P style="font-size:15px; background-color:#66bbff;"><B>Test digestion of VP2-N-terminal fusion constructs</B></P></SPAN></H4><P><B>Investigator: Hanna</B><B>Comment:</B> On Satureday a colony PCR was performed of: 
1. pCerulean_Zegfr:1907_ShortLinker_VP2/3 (P530)
2. pCerulean_Zegfr:1907_SEG_VP2/3 (P531)
3. pCerulean_CFP_MiddleLinker_VP2/3 (P532)
4. pCerulean_6xHis_MiddleLinker_VP2/3 (P533)
5. pCerulean_6xHis_MiddleLinker_VP2/3_insCap (P534)
6. pCerulean_Zegfr:1907_MiddleLinker_VP2/3_insCap (P535)
Because all constructs (4 clones of each approach), including the negative control (pAAV_RFC25), showed positive results, one test digestion of each construct was performed.
</P><P style="font-size:15px; font-weight: bold; color: blue;">Digestion</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">Components Master Mix /µl
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right"> 7
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right"> 7
</TD></TR><TR><TD>EcoRI</TD><TD align="right"> 3.5
</TD></TR><TR><TD>SpeI</TD><TD align="right"> 3.5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right"> 35
</TD></TR></TBODY></TABLE><P>--&gt; 2 µL DNA + 8 µL Master Mix </P><UL><LI>Incubation: 1.5 h
</LI></UL><P style="font-size:15px; font-weight: bold; color: blue;">Agarose-Gel:</P><P>
0.4 g Agarose, 50 mL TAE (0.8 %), 3 µL GELRED, at 100 Volt, running time: 50 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (6x)/µl
</TH><TH>Expected size 1
</TH><TH>Expected size 2
</TH></TR><TR><TD>P 530
</TD><TD>10 µl
</TD><TD>2 µl
</TD><TD>2710 bp
</TD><TD>3937 bp
</TD></TR><TR><TD>P 531
</TD><TD>10 µl
</TD><TD>2 µl
</TD><TD>2710 bp
</TD><TD>3937 bp
</TD></TR><TR><TD>P 532
</TD><TD>10 µl
</TD><TD>2 µl
</TD><TD>2710 bp
</TD><TD>3937 bp
</TD></TR><TR><TD>P 533
</TD><TD>10 µl
</TD><TD>2 µl
</TD><TD>2710 bp
</TD><TD>3937 bp
</TD></TR><TR><TD>P 534
</TD><TD>10 µl
</TD><TD>2 µl
</TD><TD>2710 bp
</TD><TD>3937 bp
</TD></TR><TR><TD>P 535
</TD><TD>10 µl
</TD><TD>2 µl
</TD><TD>2710 bp
</TD><TD>3937 bp
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker /µL
</TH><TH>Sample P530 /µl
</TH><TH>Sample P531 /µl
</TH><TH>Sample P532 /µl
</TH><TH>Sample P533 /µl
</TH><TH>Sample P534 /µl
</TH><TH>Sample P535 /µl
</TH></TR><TR><TH>Lane
</TH><TD> 4
</TD><TD> 12
</TD><TD> 12
</TD><TD> 12
</TD><TD> 12
</TD><TD> 12
</TD><TD> 12
</TD></TR></TBODY></TABLE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Test Digestion N-terminal VP2 Fusion</DIV></DIV></DIV></DIV><P><B>Comment:</B> Test digestion delivered positive results: Expected fragment size after VP2/3 fusion to Zegfr:1907_Linker: 2180 bp - compared to failed VP2/3 insertion: ~ 220 bp. 
Expected fragment size after VP2/3 fusion to CFP_MiddleLinker: 2710 bp - compared to failed VP2/3 insertion: ~ 760 bp.

pCerulean_ZEGFR:1907_Longlinker_VP2/3 
pCerulean_CFP_Middlelinker_VP2/3_insCap
pCerulean_ZEGFR:1907_Shortlinker_VP2/3_insCap
pCerulean_ZEGFR:1907_Shortlinker_VP2/3
pCerulean_ZEGFR:1907_SEG_VP2/3
pCerulean_CFP_MiddleLinker_VP2/3
pCerulean_6xHis_MiddleLinker_VP2/3
pCerulean_6xHis_MiddleLinker_VP2/3_insCap
pCerulean_ZEGFR:1907_MiddleLinker_VP2/3_insCap
will be sent for sequencing to GATC.
</P><H4><SPAN class="mw-headline" id="BioBrick_production:_PCR_of_pAAV_RC_InsRepCap_KpnIback_VP1-ko_and_VP2-ko"><P style="font-size:15px; background-color:#66bbff;"><B>BioBrick production: PCR of pAAV_RC_InsRepCap_KpnIback_VP1-ko and VP2-ko</B></P></SPAN></H4><P><B>Investigator: Stefan </B></P><P>DNA samples were diluted 1:1000
</P><UL><LI>P455 = pAAV_RC_1.2 SDM SalI</LI><LI>P463 = pAAV_RC_CapIns_prepSDM clone 3</LI></UL><P><B>PCR progam: </B></P><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD align="right">Volume P455 / µl </TD><TD align="right">Volume P463 / µl
</TD></TR><TR><TD> 5X Phusion HF buffer </TD><TD align="right">10  </TD><TD align="right">10
</TD></TR><TR><TD> 10 mM dNTP mix</TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD> forward primer: O93 </TD><TD align="right">2,5 </TD><TD align="right">2,5
</TD></TR><TR><TD> reverse primer: O115 </TD><TD align="right">2,5 </TD><TD align="right">2,5
</TD></TR><TR><TD> DNA Template</TD><TD align="right">3</TD><TD align="right">3
</TD></TR><TR><TD> DMSO </TD><TD align="right">2 </TD><TD align="right">2
</TD></TR><TR><TD> Phusion Polymerase</TD><TD align="right">0,5 </TD><TD align="right">0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">28,5 </TD><TD align="right">28,5
</TD></TR><TR><TD>Total volume</TD><TD align="right">50 </TD><TD align="right">50
</TD></TR></TBODY></TABLE><P><B>PCR program:</B></P><TABLE border="1"><TBODY><TR><TD>Cycles</TD><TD>Temperature / °C</TD><TD>Time / s
</TD></TR><TR><TD>1</TD><TD>98</TD><TD>60
</TD></TR><TR><TD>2 ( step 2-4: 8x)</TD><TD>98</TD><TD>15
</TD></TR><TR><TD>3</TD><TD>59</TD><TD>25
</TD></TR><TR><TD>4</TD><TD>72</TD><TD>75
</TD></TR><TR><TD>5 (step 5-6: 17x)</TD><TD>98</TD><TD>15
</TD></TR><TR><TD>6</TD><TD>72</TD><TD>85
</TD></TR><TR><TD>7 </TD><TD>72</TD><TD>300
</TD></TR><TR><TD>Hold</TD><TD> 4</TD><TD> 4
</TD></TR></TBODY></TABLE><P><B>Gel:</B>
0,5 g agarose (1%), 50 ml TAE, 3µl GELRED running time: 45 minutes at 110V
</P><P><B>Gel-Extraction:</B>
Gel-extraction was performed according to standard protocol. The complete amount eluted was used for PCR digestion.</P><P><B>Digestion of PCR product:</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>PCR products</B> Volume/µL
</TD></TR><TR><TD align="left"> DNA   </TD><TD align="left"> 30
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 4
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 4
</TD></TR><TR><TD align="left"> EcoRI </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> SpeI </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> -
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>40</B></TD></TR></TBODY></TABLE><P>
Digestion was performed at 37 °C for 2 hours.</P><P><B>PCR-Purification:</B>
Gel-extraction was performed according to standard protocol.</P><UL><LI>c (P455) = 13,5 ng/µl</LI><LI>c (P463) =  ng/µl</LI></UL><P><B>Digestion of plasmid backbone:</B> 
Plasmid used: pSB1C3_VCK_Bla (P320)
c (pSB1C3_VCK_Bla) = 408,0 ng/ µl
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>vector</B> Volume/µL
</TD></TR><TR><TD align="left"> DNA  </TD><TD align="left"> 3
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> EcoRI </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> SpeI </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 11
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"> 20
</TD></TR></TBODY></TABLE><P>Digestion was performed at 37 °C for 2 hours.</P><P><B>Gel:</B>
0,5 g agarose (1%), 50 ml TAE, 3µl GELRED running time: 50 minutes at 120V
</P><P><B>Gel-Extraction:</B>
Gel-extraction was performed according to standard protocol.
c(P320) = 0,8 ng/µl
</P><P><B>Ligation:</B></P><UL><LI>1 µl T4 DNA ligase</LI><LI>1 µl 10x Buffer</LI><LI>8 µl DNA mix (vector + insert)</LI></UL><P>Amounts of DNA used:
</P><UL><LI>c(P320) = 7,73 µl</LI><LI>c(P455) = 0,27 µl</LI></UL><UL><LI>c(P320) = 7,19 µl</LI><LI>c(P463) = 0,81 µl</LI></UL><P>Incubation time: 45 minutes at room temperature. </P><P><B>Transformation:</B>
Transformation was performed according to standard protocol using XL1b cells and Chloramphenicol as antibiotic.
</P><H4><SPAN class="mw-headline" id="2x_repetition_of_CD_biobrick"><P style="font-size:15px; background-color:#66bbff;"><B>2x repetition of CD biobrick</B></P></SPAN></H4><P><B>Investigator: Kira </B></P><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD align="right">CD sample
</TD></TR><TR><TD> 5X Phusion HF buffer </TD><TD align="right">10 µl
</TD></TR><TR><TD> 10 mM dNTP mix</TD><TD align="right">1µl
</TD></TR><TR><TD> forward primer: O158 </TD><TD align="right">2,5µl
</TD></TR><TR><TD> reverse primer: O159 </TD><TD align="right">2,5 µl
</TD></TR><TR><TD> DNA Template (1:100 dil)</TD><TD align="right">0,5 µl
</TD></TR><TR><TD> DMSO </TD><TD align="right">0 µl
</TD></TR><TR><TD> Phusion Polymerase</TD><TD align="right">0,5 µl
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">33µl
</TD></TR><TR><TD>Total volume</TD><TD align="right">50 µl
</TD></TR></TBODY></TABLE><P>PCR program:
</P><TABLE border="1"><TBODY><TR><TD>Cycles</TD><TD>Temperature</TD><TD>Time
</TD></TR><TR><TD>98°C</TD><TD>1
</TD></TR><TR><TD>10x</TD><TD>98°C</TD><TD>15&quot;
</TD></TR><TR><TD>58°C</TD><TD>25&quot;
</TD></TR><TR><TD>72°C</TD><TD>40&quot;
</TD></TR><TR><TD>17x</TD><TD>98°C</TD><TD>15&quot;
</TD></TR><TR><TD>65°C</TD><TD>25&quot;
</TD></TR><TR><TD>72°C</TD><TD>40&quot;
</TD></TR><TR><TD>1x</TD><TD>72°C</TD><TD>5'
</TD></TR><TR><TD>Hold 4°C
</TD></TR></TBODY></TABLE><P>Digestion of plasmid backbone:
</P><P>c (pSB1C3) = 151, 1 ng/ µl
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>vector</B> Volume/µL
</TD></TR><TR><TD align="left"> DNA 1 µg  </TD><TD align="left"> 6,0 µl
</TD></TR><TR><TD align="left"> BSA (100x) </TD><TD align="left"> 0,2 µl
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 2,0 µl
</TD></TR><TR><TD align="left"> Enzyme 1 XbaI </TD><TD align="left"> 0,5 µl
</TD></TR><TR><TD align="left"> Enzyme 2 AgeI HF </TD><TD align="left"> 0,5 µl
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 10,8 µl
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>20</B></TD></TR></TBODY></TABLE><P>incubation @ 37 C for approx. 6 h
</P><P>1% agarose gel </P><P>According to the gel results, PCR was repeated twice.. But both tries revealed the same unexpected results. Insert from the last cloning procedure was found and used for the ligation and further transformation.
</P><P>
Ligation 
T4 ligase was used 
1 ul T4 Buffer 
1 ul T4 Ligase 
8 ul (0,9 ul vector+ 7,1 ul insert) DNA-mix </P><P>incubation @ RT for 45 min</P><P>Transformation was performed according to the standard protocol and the cells were spread on the agar plate containing Chloramphenicol.
</P><H3><SPAN class="mw-headline" id="126._labday_21.09.2010"><P style="font-size:17px; background-color:#00dd77;">126. labday 21.09.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Trafo_evalutation_of_CD_biobrick_plate"><P style="font-size:15px; background-color:#66bbff;"><B>Trafo evalutation of CD biobrick plate</B></P></SPAN></H4><P><B>Investigator: Kira</B></P><P>the agar plate was checked at approx. 4 pm and there were no colonies visible. Taking in account that the plate was incubated from 10.30 pm till 8 am @ 37C due to the electricity failure, the plate was put back @ 37C at 4.30 pm </P><P><B>Comment:</B> the plate was picked this morning by a lab member and put in the coldroom
</P><H4><SPAN class="mw-headline" id="Continuation_of_cloning_of_pCerulean_Affibody_VP2.2F3_and_pSB1C3_Affibody_middlelinker_GFP_his"><P style="font-size:15px; background-color:#66bbff;"><B>Continuation of cloning of pCerulean_Affibody_VP2/3 and pSB1C3_Affibody_middlelinker_GFP_his</B></P></SPAN></H4><P><B>Investigator: Jessica </B></P><P style="font-size:13px; color:red;"><B>Comment</B>:Affibody_VP2/3 will be cloned in pCerulean and Affibody_middlelinker will be cloned together with GFP_His in pSB1C3</P><UL><LI>P407= pCerulean_CFP_middlelinker c=482,76 ng/µl
</LI><LI>P516= pSB1C3_ZEGFR:1907_VP2/3 clone 1 c= 191,6 ng/µl
</LI><LI>P290= pSB1C3_Affibody_Middlelinker clone 1 c= 227,4 ng/µl
</LI><LI>P518= pSB1C3_GFP_RFC25_upper band clone 1 c= 138,3 ng/µl
</LI><LI>P520= pSB1C3_GFP_RFC25_lower band clone 1 c= 141,6 ng/µl
</LI></UL><P><B>Digestion:</B></P><TABLE border="1"><TBODY><TR><TD> components  </TD><TD align="right">P516 </TD><TD align="right">P518 </TD><TD align="right">P520
</TD></TR><TR><TD> DNA   </TD><TD align="right"> 10 </TD><TD align="right"> 10 </TD><TD align="right"> 10
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right"> 2</TD><TD align="right"> 2</TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x) </TD><TD align="right"> 2 </TD><TD align="right"> 2</TD><TD align="right"> 2
</TD></TR><TR><TD>EcoRI  </TD><TD align="right">0,8</TD><TD align="right">-</TD><TD align="right">-
</TD></TR><TR><TD>AgeI  </TD><TD align="right">0,8</TD><TD align="right">-</TD><TD align="right">-
</TD></TR><TR><TD>MscI  </TD><TD align="right">0,8</TD><TD align="right">-</TD><TD align="right">-
</TD></TR><TR><TD>NgoMIV  </TD><TD align="right">-</TD><TD align="right">0,8</TD><TD align="right">0,8
</TD></TR><TR><TD>SpeI  </TD><TD align="right">-</TD><TD align="right">0,8</TD><TD align="right">0,8
</TD></TR><TR><TD>H<SUB>2</SUB>O </TD><TD align="right"> 3,6</TD><TD align="right"> 4,4</TD><TD align="right"> 4,4
</TD></TR><TR><TD><B>Total volume </B></TD><TD align="right">20</TD><TD align="right"> 20</TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><P>0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 120 Volt
</P><P><B>Gelextraction:</B></P><P>The gelextraction was performed according to the standard protocol. DNA concentration of the extracts:
</P><UL><LI>P516=  c=  ng/µl
</LI><LI>P518=  c=  ng/µl
</LI><LI>P520=  c=  ng/µl
</LI></UL><P><B>T4 Ligation:</B></P><P>The  Ligation was performed as following:</P><UL><LI> Vector Volume:  µl
</LI><LI> Insert Volume:  µl
</LI></UL><UL><LI> 1µl T4-Ligase buffer (10x)
</LI><LI> 8µl (Vector + Insert) mix
</LI><LI> 1µl T4-Ligase
</LI></UL><P> Incubating for 40 minutes.
</P><P><B>Transformation:</B></P><P>Trafo was performed according to the standard protocol (XL1blue). The cells were plated on a agar plate with Kana/Cm
</P><H4><SPAN class="mw-headline" id="Sequencing_result_of_pSB1C3_EGFP_His_upper_and_.28.21.29_lower_band"><P style="font-size:15px; background-color:#66bbff;"><B>Sequencing result of pSB1C3_EGFP_His upper and (!) lower band</B></P></SPAN></H4><P><B>Investigator: Jessica </B><B>1. pSB1C3_EGFP_His suffix:</B></P><P><B>2. pSB1C3_EGFP_His prefix:</B></P><P><B>results look in both plasmids and are used for cloning of pSB1C3_Affibody_middlelinker_GFP_His</B></P><H4><SPAN class="mw-headline" id="Midi-Prep_of_pHelper_.26_pSB1C3_leftITR_CMV_beta-globin_mVenus_hGH_rightITR"><P style="font-size:15px; background-color:#66bbff;"> Midi-Prep of pHelper &amp; pSB1C3_leftITR_CMV_beta-globin_mVenus_hGH_rightITR</P></SPAN></H4><P><B>Investigators: Chris W. </B></P><P style="font-size:13px; color:#003399;"> Midi-Preps of pHelper=P540 and pSB1C3_leftITR_CMV_beta-globin_mVenus_hGH_rightITR=P541</P><P>
The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
</P><TABLE border="1"><TBODY><TR><TD> plasmid-no. </TD><TD align="right">P540</TD><TD align="right">P541
</TD></TR><TR><TD> concentration (ng/µl)</TD><TD align="right">480,45 </TD><TD align="right">1264,20
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="New_aliquots_.2860.C2.B5l.29_of_XL1blue_can_be_used_.21_.28Jessica.29"><P style="font-size:15px; background-color:#66bbFF;"><B>New aliquots (60µl) of XL1blue can be used !  (Jessica)</B></P></SPAN></H4><H3><SPAN class="mw-headline" id="127._labday_22.09.2010"><P style="font-size:17px; background-color:#00dd77;">127. labday 22.09.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Sequencing_results_of_ViralBrick_ligations_using_dephosphorylated_vector"><P style="font-size:15px; background-color:#66bbff;"><B>Sequencing results of ViralBrick ligations using dephosphorylated vector</B></P></SPAN></H4><P><B>Investigator: Achim </B></P><P>Sequencing confirmed a correct sequence for clone 11.4 (453 Z34C). Clone 12.1 (587 Z34C)has several missing bases at the 3' end of the insert, clone 13.2 has an a -&gt; c mutation and a c insertion in the insert (although the a-&gt;c might be a sequencing error.). Clone 14.2 contained a c deletion. 
</P><P>We have several new approaches:
</P><P>-Correction of the mutations via PCR: We'll use the hybridisation oligo for the side of the insert containing the mutations, the rfc 25 primer on the other side. The amplified insert will be ligated into dephosphorized vector.
</P><P>-As a backup, SDM-primers will be ordered and should be available next week.
</P><P>-Also, a new colony pcr with several clones from the last ligation plates will be carried out.
</P><P>-clone 13.4 (already prepped) will be sent for sequencing
</P><H4><SPAN class="mw-headline" id="Cloning_pSB1C3_Viral_Brick_587KO-empty_.28B274.2FP542.29_into_pSB1C3_VP2.2F3_Capins_.28B438.2FP514.29"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning pSB1C3_Viral Brick 587KO-empty (B274/P542) into pSB1C3_VP2/3_Capins (B438/P514) </B></P></SPAN></H4><P><B>Investigator Patrick </B>
After an over night digestion with BamHI and PvuII at 37°C ....
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">My dear friend - what do you mean with &quot;...&quot;?? overnight digest at which temperature?? Did you look it up whether the enzymes show star activity at longer digestions or not?? ;-)</DIV></DIV></DIV><P>
The Gelextraction with P514 was performed according to the standard protocol yielding 16,9 ng/ml. 
The P542 insert should be 48 bp long and therefore was extracted with a QIAGEN kit able to extract even very small DNA fragments (40 bp) (QiaEX II Gel Extraction Kit (150)).
</P><P>Expected size of the fragments: 
P542: 2112 &amp; 48 bp  
P514: 3956 &amp; 48 bp  </P><P>The 48 bp band of P514 can't be seen clearly on the picture due to the bad resolution.
</P><P>Ligation with T4 DNA ligase: 1 µl buffer (10x), 1 µl T4 DNA ligase, 7 µl Vector, 1 µl Insert, 70 minutes.
</P><P>The transformation with XL1B was performed according to the standard protocol and plated: pSB1C3_VP2/3_Capins_587KO-empty
</P><H4><SPAN class="mw-headline" id="Cellculture:_Serum-free_cells"><P style="font-size:15px; background-color:#66bbff;"><B>Cellculture: Serum-free cells</B></P></SPAN></H4><P><B>Investigator Patrick </B>
We want our virus production to be serum-free to simplify the following purification.  
15 ml cell-medium, 25% serum-free: 3,75 ml serum-free DMEM + 11,25 ml DMEM 
15 ml cell-medium, 50% serum-free: 7,5 ml serum-free DMEM + 7,5 DMEM </P><H4><SPAN class="mw-headline" id="PCR_of_P40"><P style="font-size:15px; background-color:#66bbff;"><B>PCR of P40</B></P></SPAN></H4><P><B>Investigator: Jessica</B>
DNA sample P432 was diluted 1:1000
</P><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD align="right">Volume / µl
</TD></TR><TR><TD> 5X Phusion HF buffer </TD><TD align="right">10
</TD></TR><TR><TD> 10 mM dNTP mix</TD><TD align="right">1
</TD></TR><TR><TD> forward primer: O152 </TD><TD align="right">2,5
</TD></TR><TR><TD> reverse primer: O188 </TD><TD align="right">2,5
</TD></TR><TR><TD> DNA Template</TD><TD align="right">3,0
</TD></TR><TR><TD> DMSO </TD><TD align="right">-
</TD></TR><TR><TD> Phusion Polymerase</TD><TD align="right">0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">31
</TD></TR><TR><TD>Total volume</TD><TD align="right">50,5
</TD></TR></TBODY></TABLE><P><B>PCR program:</B></P><TABLE border="1"><TBODY><TR><TD>Cycles</TD><TD>Temperature / °C</TD><TD>Time / s
</TD></TR><TR><TD>98</TD><TD>60
</TD></TR><TR><TD>8x</TD><TD>98</TD><TD>15
</TD></TR><TR><TD>61</TD><TD>25
</TD></TR><TR><TD>72</TD><TD>8
</TD></TR><TR><TD>17x</TD><TD>98</TD><TD>15
</TD></TR><TR><TD>69</TD><TD>25
</TD></TR><TR><TD>72</TD><TD>8
</TD></TR><TR><TD>1x</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>Hold</TD><TD> 4
</TD></TR></TBODY></TABLE><P><B>Digestion of plasmid backbone:</B>
Plasmid used: pSB1C3_CFP (P51.2)
c (pSB1C3_CFP) = 151, 1 ng/ µl
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>vector</B> Volume/µL
</TD></TR><TR><TD align="left"> DNA  </TD><TD align="left"> 10
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> XbaI </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> PstI </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 4
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>20</B></TD></TR></TBODY></TABLE><P>incubation @ 37 C for approx. 2 h
</P><P style="font-size:13px; color:red;"><B>Comment</B>: </P><P><B>Digestion of PCR product:</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>PCR products</B> Volume/µL
</TD></TR><TR><TD align="left"> DNA   </TD><TD align="left"> 40
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 6
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 6
</TD></TR><TR><TD align="left"> PstI </TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> XbaI </TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 4
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>60</B></TD></TR></TBODY></TABLE><P>
Digestion was performed at 37 °C for 2 hours.</P><P><B>PCR-Purification:</B></P><UL><LI>c (P432) = 31,8ng/µl</LI></UL><P><B>Gel-Extraction:</B>
Gel-extraction was performed according to standard protocol.
c(P51.2) = 3,7ng/µl
</P><P><B>Ligation:</B></P><UL><LI>1 µl T4 DNA ligase</LI><LI>1 µl 10x Buffer</LI><LI>8 µl DNA mix (vector + insert)</LI></UL><P>Amounts of DNA used:
</P><UL><LI>(P51.2) = 7,74 µl</LI><LI>(P432) =  0,26µl</LI></UL><P>Incubation time: 45 minutes at room temperature. </P><P><B>Transformation:</B>
Transformation was performed according to standard protocol using XL1b cells and Chloramphenicol as antibiotic.
</P><H4><SPAN class="mw-headline" id="Mini-Prep_of_pSB1C3_001_RC_InsRepCap_KpnIback_VP1-ko"><P style="font-size:15px; background-color:#66bbff;"><B>Mini-Prep of pSB1C3_001_RC_InsRepCap_KpnIback_VP1-ko</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P style="color:#66bbff;"><I>Comment:</I></P><P>
Glycerol stocks were prepared:</P><UL><LI>B463 = pSB1C3_001_RC_InsRepCap_KpnIback_VP1-ko_P5tataless clone 1</LI><LI>B464 = pSB1C3_001_RC_InsRepCap_KpnIback_VP1-ko_P5tataless clone 2</LI></UL><P>
Mini-Prep was performed according to standard protocol:
</P><UL><LI>P548 = pSB1C3_001_RC_InsRepCap_KpnIback_VP1-ko_P5tataless clone 1 c = 435,82 ng/µl</LI><LI>P549 = pSB1C3_001_RC_InsRepCap_KpnIback_VP1-ko_P5tataless clone 2 c = 409,21 ng/µl</LI></UL><H4><SPAN class="mw-headline" id="Cloning_of_CMV_promotor_into_pSB1C3_VP123_inscap"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of CMV promotor into pSB1C3_VP123_inscap</B></P></SPAN></H4><P><B>Investigator: Anna</B></P><UL><LI>Vector name: pSB1C3_VP123_inscap <B>P489</B></LI><LI>Insert name: pSB1C3_CMV <B>P193</B></LI><LI>new vector name: pSB1C3_CMV_VP123_capins <B>P580</B></LI><LI>buffer used: 4 
</LI><LI>DNA concentration (vector): 350,0 ng/µl ; DNA concentration (insert): 333,5 µg/µl
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume of pSB1C3_VP123_capins /µl</B></TD><TD align="right"><B>volume of pSB1C3_CMV /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">3 </TD><TD align="right">4,5
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">1,5 </TD><TD align="right"> 1,5
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">1,5</TD><TD align="right">1,5
</TD></TR><TR><TD>Enzyme I</TD><TD align="right">EcoI HF 1 </TD><TD align="right">EcoI HF 1
</TD></TR><TR><TD>Enzyme II</TD><TD align="right">XbaI 1 </TD><TD align="right">SpeI 1
</TD></TR><TR><TD>H2O</TD><TD align="right">7 </TD><TD align="right">5,5
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 15</TD><TD align="right">15
</TD></TR></TBODY></TABLE><P>
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 120 Volt, running time:45
</P><P><B>Loading plan for agarose gel</B>: 
Loading dye with SDS (6X), 3 µl 
Marker used: GeneRuler ladder mix (Fermentas), 4 µl</P><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>Sample 489, 18µl
</TH><TH>Sample 193, 18µl
</TH></TR><TR><TH>Lane
</TH><TD>1
</TD><TD>3
</TD><TD>5
</TD></TR><TR><TH>Fragment size
</TH><TD>4404 bp
</TD><TD>681 bp
</TD></TR></TBODY></TABLE><P><B>Gel extraction</B>: </P><TABLE border="1"><TBODY><TR><TH>pSB1C3_VP123_capins_cut
</TH><TH>CMV
</TH></TR><TR><TH>DNA-concentration [ng/µl]
</TH><TD>14,09
</TD><TD>32,61
</TD></TR></TBODY></TABLE><P><B>T4 Ligation</B>: 
Volume vector: 3,86 µl 
Volume insert: 4,14 µl <B>Transformation</B>: </P><P>Was performed following the standard protocol using XL1B cells.
</P><H4><SPAN class="mw-headline" id="Sequencing_of_Zegfr:1907-Linker_Library.2C_His-Tag_and_imaging_appoaches"><P style="font-size:15px; background-color:#66bbff;"><B>Sequencing of Zegfr:1907-Linker Library, His-Tag and imaging appoaches</B></P></SPAN></H4><P><B>Investigator: Hanna</B></P><P><B>Comment:</B> After several problems were managed (cloning difficulties, colony PCR troubles, interchanges of samples,...), sequences of the final constructs needed to be verified in order to test the constructs in cell culture.
</P><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Sample gallery</CAPTION><TBODY><TR><TD><DIV class="gallerybox" style="width: 465px;"><DIV class="gallerytext"><P>pCerulean_CFP_MiddleLinker_ VP2/3_InsCap (P528) forward
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 465px;"><DIV class="gallerytext"><P>pCerulean_CFP_MiddleLinker_ VP2/3_InsCap (P528) reverse
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 465px;"><DIV class="gallerytext"><P>pCerulean_ZEGFR:1907_ShortLinker_ VP2/3_InsCap (P527) forward
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 465px;"><DIV class="gallerytext"><P>pCerulean_ZEGFR:1907_ShortLinker_ VP2/3_InsCap (P527) reverse
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 465px;"><DIV class="gallerytext"><P>pCerulean_6xHis_MiddleLinker_ VP2/3 (P533) forward
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 465px;"><DIV class="gallerytext"><P>pCerulean_6xHis_MiddleLinker_ VP2/3 (P533) reverse
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 465px;"><DIV class="gallerytext"><P>pCerulean_6xHis_MiddleLinker_ VP2/3_insCap (P534) forward
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 465px;"><DIV class="gallerytext"><P>pCerulean_6xHis_MiddleLinker_ VP2/3_insCap (P534) reverse
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 465px;"><DIV class="gallerytext"><P>pCerulean_ZEGFR:1907_MiddleLinker_ VP2/3_insCap (P535) forward
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 465px;"><DIV class="gallerytext"><P>pCerulean_ZEGFR:1907_MiddleLinker_ VP2/3_insCap (P535) reverse
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 465px;"><DIV class="gallerytext"><P>pCerulean_ZEGFR:1907_SEG_ VP2/3_insCap (P502) reverse
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 465px;"><DIV class="gallerytext"><P>pCerulean_ZEGFR:1907_LongLinker_ VP2/3_insCap (P503) reverse
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 465px;"><DIV class="gallerytext"><P>pCerulean_ZEGFR:1907_MiddleLinker_ VP2/3 (P504) reverse
</P></DIV></DIV></TD></TR></TBODY></TABLE><P><B>Comment:</B> Sequences looked well. The 6 samples with VP2/3_insCap will be tested today in cell culture.
</P><H4><SPAN class="mw-headline" id="Mass_Midi-Prep"><P style="font-size:15px; background-color:#66bbff;">Mass Midi-Prep </P></SPAN></H4><P><B>Investigator: Chris W. </B></P><P style="font-size:13px; color:#003399;"> Midi-Prep of:</P><P>pSB1C3_001_RC_InsRepCap_KpnIback_P5tataless clone 1 =P544
pSB1C3_001_RC_InsRepCap_KpnIback_P5tataless clone 2 =P545
pSB1C3_001_P5_RC_InsRepCap_KpnIback clone 1 =P546
pSB1C3_001_P5_RC_InsRepCap_KpnIback clone 2 =P547
pHelper =P550
pHelper =P551
pSB1C3_litr_pTert_ßglobin_mvenus_hgh_ritr clone 1 =P552
pAAV_RC_RepCapIns_SDMKpnl_VP1ko clone 1 =P553
pAAV_RC_RepCapIns_SDMKpnl_VP2ko clone 1 =P554
pAAV_RC_RepCapIns_SDMKpnl_VP2ko clone 1 =P555
pCerulean_ZEGFR:1907_SEG_VP2/3_CapIns clone 3 =P556
pCerulean_ZEGFP:1907_LongLinker_VP2/3_CapIns clone 2 =P557
pCerulean_CFP_Middlelinker_VP2/3_insCap =P558
pCerulean_ZEGFR:1907_Shortlinker_VP2/3_insCap =P559
pCerulean_6xHis_MiddleLinker_VP2/3_insCap =P560
pCerulean_ZEGFR:1907_MiddleLinker_VP2/3_insCap =P561
pCerlulean_ZEGFR:1907_Shortlinker_VP2/3_insCap clone 4.8 =P562
The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
</P><TABLE border="1"><TBODY><TR><TD> plasmid-no. </TD><TD align="right">P544</TD><TD align="right">P545</TD><TD align="right">P546</TD><TD align="right">P547</TD><TD align="right">P550</TD><TD align="right">P551</TD><TD align="right">P552</TD><TD align="right">P553</TD><TD align="right">P554</TD><TD align="right">P555</TD><TD align="right">P556</TD><TD align="right">P557</TD><TD align="right">P558</TD><TD align="right">P559</TD><TD align="right">P560</TD><TD align="right">P561</TD><TD align="right">P562
</TD></TR><TR><TD> concentration (ng/µl)</TD><TD align="right">1851,91 </TD><TD align="right">1822,3 </TD><TD align="right">1783,48</TD><TD align="right">2625,57</TD><TD align="right">1524,63</TD><TD align="right">3333,48</TD><TD align="right">2641,57</TD><TD align="right">2165,22</TD><TD align="right">1818,23</TD><TD align="right">2318,58</TD><TD align="right">3255,81</TD><TD align="right">2818,15</TD><TD align="right">2761,32</TD><TD align="right">3289,23</TD><TD align="right">3134,26</TD><TD align="right">3725,27</TD><TD align="right">1965,28
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="RNA_isolation_and_cDNA_synthesis"><P style="font-size:15px; background-color:#66bbff;"><B>RNA isolation and cDNA synthesis</B></P></SPAN></H4><P><B>Investigator Kira </B></P><P>Motivation: in order to figure out if our cell lines do express telomerase, cell pellets were used for RNA isolation.
</P><P>RNeasy Kit was used for RNA isolation. The cells were harvested and pellets washed 2x with PBS. 
add RLT buffer 
homogenize the lysate with syringe ans blunt needle 
add 1 volume of 70% EtOH 
centrifuge for 15s @ 10.000 rpm 
add 700 ul Buffer RW1 and centrifuge again 
add 500 ul Buffer RPE and centrifuge 
add 500 ul Buffer RPE and centifuge for 2 min 
place the column in 1.5 ml tube and add RNase free water (50 ul) </P><P>concentrations: 
c(293)= 41, 68 ng/ul 
c(a431) = 135, 48 ng/ul </P><P>Gel apparatus (chamber, tray and comb) for RNA purity gel has to be prepared a day ahead with : water, SDS and NaOH </P><H4><SPAN class="mw-headline" id="cDNA_synthesis"><P style="font-size:15px; background-color:#66bbff;"><B>cDNA synthesis</B></P></SPAN></H4><P><B>Investigator: Kira</B></P><P>The yielded DNA was used for cDNA synthesis. 
QuantiTect Reverse Transcription (Qiagen) reaction was performed according to the manufacturer protocol. 
DNA elimination reaction mix: </P><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>a431/µl</B></TD><TD align="right"><B>293/µl</B></TD></TR><TR><TD> 7xgDNA Wipeout Buffer </TD><TD align="right">2 </TD><TD align="right">2
</TD></TR><TR><TD> Template RNA </TD><TD align="right">1,6 </TD><TD align="right"> 2
</TD></TR><TR><TD>RNase-free H2O</TD><TD align="right">10,4 </TD><TD align="right">10
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 14</TD><TD align="right">14
</TD></TR></TBODY></TABLE><P>Reverse-transcription reaction mix: </P><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>1xreaction</B></TD><TD align="right"><B>MM</B></TD></TR><TR><TD> Quantiscript Reverse Transcriptase </TD><TD align="right">1 </TD><TD align="right">3
</TD></TR><TR><TD> 5x Quantiscript RT buffer </TD><TD align="right">4 </TD><TD align="right"> 12
</TD></TR><TR><TD> RT primer Mix</TD><TD align="right">1 </TD><TD align="right">3
</TD></TR><TR><TD> DNA elimination reaction mix</TD><TD align="right"> 14</TD><TD align="right"> -
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 20</TD><TD align="right">-
</TD></TR></TBODY></TABLE><P>incubation for 15 min @ 42C 
incubation for 3 min @ 95C </P><H3><SPAN class="mw-headline" id="128._labday_23.09.2010"><P style="font-size:17px; background-color:#00dd77;">128. labday 23.09.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="PCR_of_VB_constructs_to_remove_mutations"><P style="font-size:15px; background-color:#66bbff;"><B>PCR of VB constructs to remove mutations </B></P></SPAN></H4><P><B>Investigator Achim</B></P><H4><SPAN class="mw-headline" id="Colony_PCR_of_new_clones_from_the_last_ligations"><P style="font-size:15px; background-color:#66bbff;"><B>Colony PCR of new clones from the last ligations </B></P></SPAN></H4><P><B>Investigator Achim</B></P><P>Sequencing of the previously picked clones revealed that the inserts were successfully cloned into pSB1C3. Sadly, apart from 453 Z34C every other insert contained various mutations (deletions, insertions, base exchanges in different positions). The reasons are unclear, probably difficulties with the fill in reaction or incorrectly synthesized oligos. We picked more clones from the last ligations' plates and carried out a colony pcr on them:
</P><H4><SPAN class="mw-headline" id="Contuniation:_Cloning_pSB1C3_Viral_Brick_587KO-empty_.28B274.2FP542.29_into_pSB1C3_VP2.2F3_Capins_.28B438.2FP514.29"><P style="font-size:15px; background-color:#66bbff;"><B>Contuniation: Cloning pSB1C3_Viral Brick 587KO-empty (B274/P542) into pSB1C3_VP2/3_Capins (B438/P514)</B></P></SPAN></H4><P><B>Investigator Patrick</B>
3 Clones were picked in the morning and in each case inoculated 10 ml DYT. Clone 3 didnt grew fast enough. There were not enough bacteria in the eppi even after 8 ml were centrifuged. The mini-prep of clone 1 and 2 was performed according to the standrd protocol yielding the following concentrations:  
Clone 1: 36,64 ng/µl 
Clone 2: 7,8 ng/µl 
Therefore only clone 1 was sent for sequencing  (labelling: pat1, primer:VR2) and only this clone received a plasmid number: P569 
No test-digestion was performed. 
Because all medium was used to receive a sufficient amount of bacteria from clone 1 there was no cell suspension left to prepare a glycerol stock. Therefore again each clone was used to inoculated 10 ml DYT, following a mini-prep tomorrow.
</P><P>Sequencing results:
</P><P>Obviously there is something wrong with the backbone or the sequencing.
</P><H4><SPAN class="mw-headline" id="Mini-Prep_and_test_digestion_of_pCerulean_Affibody_VP2.2F3.2C_pSB1C3_Affibody_middlelinker_EGFP_His.2C_pSB1C3_001_RC_InsRepCap_KpnIback_VP2-ko"><P style="font-size:15px; background-color:#66bbff;"><B>Mini-Prep and test digestion of pCerulean_Affibody_VP2/3, pSB1C3_Affibody_middlelinker_EGFP_His, pSB1C3_001_RC_InsRepCap_KpnIback_VP2-ko</B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><P style="color:#66bbff;"><I>Comment:</I></P><P>
Glycerol stocks were prepared:</P><UL><LI>B465 = pCerulean_ZEGFR:1907_VP23 clone1</LI><LI>B466 = pCerulean_ZEGFR:1907_VP23 clone2</LI><LI>B467 = pSB1C3_ZEGFR:1907_middlelinker_EGFP_His clone1</LI><LI>B468 = pSB1C3_ZEGFR:1907_middlelinker_EGFP_His clone2</LI><LI>B469 = pSB1C3_001_RC_InsRepCap_KpnIbackVP2-KO clone1</LI><LI>B470 = pSB1C3_001_RC_InsRepCap_KpnIbackVP2-KO clone2</LI></UL><P>
Mini-Prep was performed according to standard protocol:
</P><UL><LI>P563 = pCerulean_ZEGFR:1907_VP23 clone1 = 227,4 ng/µl</LI><LI>P564 = pCerulean_ZEGFR:1907_VP23 clone2 = 337,2 ng/µl</LI><LI>P565 = pSB1C3_ZEGFR:1907_middlelinker_EGFP_His clone1 = 252,7 ng/µl</LI><LI>P566 = pSB1C3_ZEGFR:1907_middlelinker_EGFP_His clone2 = 209,5 ng/µl</LI><LI>P567 = pSB1C3_001_RC_InsRepCap_KpnIback_VP1-KO clone1 = 304,7 ng/µl</LI><LI>P568 = pSB1C3_001_RC_InsRepCap_KpnIback_VP1-KO clone2 = 383,1 ng/µl</LI></UL><P><B>Test digestion of
</B></P><UL><LI>P563 = pCerulean_ZEGFR:1907_VP23 clone1 </LI><LI>P564 = pCerulean_ZEGFR:1907_VP23 clone2 </LI><LI>P565 = pSB1C3_ZEGFR:1907_middlelinker_EGFP_His clone1</LI><LI>P566 = pSB1C3_ZEGFR:1907_middlelinker_EGFP_His clone2 </LI></UL><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>P563-566</B> Volume/µL
</TD></TR><TR><TD align="left"> DNA   </TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> PstI </TD><TD align="left"> 0,5
</TD></TR><TR><TD align="left"> XbaI </TD><TD align="left"> 0,5
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 5
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>10</B></TD></TR></TBODY></TABLE><P><B>Results:</B></P><UL><LI> P566 = P565 ; P567 = P566 - P566 is not OK!!!
</LI><LI> P563 and P566 was sent for sequencing with primer VF2 and VF2/VR2
</LI></UL><H4><SPAN class="mw-headline" id="PCR_for_587_Z34C_ViralBricks"><P style="font-size:15px; background-color:#66bbff;"><B>PCR for 587 Z34C ViralBricks</B></P></SPAN></H4><P><B>Investigators: Achim, Anna</B></P><UL><LI>Plasmids used as template:
</LI></UL><P>12_T4_1:  c = 80,36 ng/µl 
13_T4_2:  c = 80,87 ng/µl 
14_T4_2:  c = 98,52 ng/µl </P><UL><LI>Primer used:
</LI></UL><TABLE border="1"><TBODY><TR><TD align="left"><B>Template</B></TD><TD align="left"><B>Primer for</B></TD><TD align="left"><B>Primer rev</B></TD></TR><TR><TD align="left"> 12_T4_1   </TD><TD align="left"> RFC25 </TD><TD align="left">O146
</TD></TR><TR><TD align="left"> 13_T4_2 </TD><TD align="left"> RFC25 </TD><TD align="left"> O148
</TD></TR><TR><TD align="left"> 14_T4_2 </TD><TD align="left"> O149 </TD><TD align="left">RFC25
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD align="right">Volume / µl
</TD></TR><TR><TD> 5X Phusion HF buffer </TD><TD align="right">10
</TD></TR><TR><TD> 10 mM dNTP mix</TD><TD align="right">1
</TD></TR><TR><TD> forward primer: see above</TD><TD align="right">2,5
</TD></TR><TR><TD> reverse primer: see above </TD><TD align="right">2,5
</TD></TR><TR><TD> DNA Template (1:100 dilution)</TD><TD align="right"> 1,0
</TD></TR><TR><TD> DMSO </TD><TD align="right">1,0
</TD></TR><TR><TD> Phusion Polymerase</TD><TD align="right">0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">31,5
</TD></TR><TR><TD>Total volume</TD><TD align="right">50
</TD></TR></TBODY></TABLE><P><B>PCR program:</B></P><TABLE border="1"><TBODY><TR><TD>Cycles</TD><TD>Temperature / °C</TD><TD>Time / s
</TD></TR><TR><TD>98</TD><TD>60
</TD></TR><TR><TD>98</TD><TD>15
</TD></TR><TR><TD>25x</TD><TD>68</TD><TD>25
</TD></TR><TR><TD>72</TD><TD>10
</TD></TR><TR><TD>1x</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>Hold</TD><TD> 4°C
</TD></TR></TBODY></TABLE><P><B>Digestion of samples:</B></P><P>Restriction enzymes used: BamHI and PvuI.
</P><P style="font-size:13px; color:red;"><B>Comment</B>: </P><H4><SPAN class="mw-headline" id="PCR_of_Vp1_ex.2C_Vp2_ex_and_Vp3_ex_and_cloning_into_pSB1C3_001"><P style="font-size:15px; background-color:#66bbff;"><B>PCR of Vp1_ex, Vp2_ex and Vp3_ex and cloning into pSB1C3_001</B></P></SPAN></H4><P><B>Investigator: Anissa</B></P><UL><LI>DNA sample P72 (pkex_Vp1), P73(pkex_Vp2) and P74(pkex_Vp3) were diluted 1:100
</LI><LI>used primers: O89 = Präfix_Vp1ex, O90= Präfix_Vp2ex, O91= Präfix_Vp2ex, O92= Suffix_Vp123ex, O120= Suffix_Vp123ex_reg
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD align="right">Volume / µl
</TD></TR><TR><TD> 5X Phusion HF buffer </TD><TD align="right">10
</TD></TR><TR><TD> 10 mM dNTP mix</TD><TD align="right">1
</TD></TR><TR><TD> forward primer: O152 </TD><TD align="right">2,5
</TD></TR><TR><TD> reverse primer: O188 </TD><TD align="right">2,5
</TD></TR><TR><TD> DNA Template</TD><TD align="right">2,0
</TD></TR><TR><TD> DMSO </TD><TD align="right">1
</TD></TR><TR><TD> Phusion Polymerase</TD><TD align="right">0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">30,5
</TD></TR><TR><TD>Total volume</TD><TD align="right">50
</TD></TR></TBODY></TABLE><P><B>PCR program:</B></P><TABLE border="1"><TBODY><TR><TD>Cycles</TD><TD>Temperature / °C</TD><TD>Time / s
</TD></TR><TR><TD>1x</TD><TD>98</TD><TD>60
</TD></TR><TR><TD>25x</TD><TD>98</TD><TD>15
</TD></TR><TR><TD>25x</TD><TD>60</TD><TD>25
</TD></TR><TR><TD>25x</TD><TD>72</TD><TD>55
</TD></TR><TR><TD>1x</TD><TD>72</TD><TD>5
</TD></TR><TR><TD>Hold</TD><TD> 4
</TD></TR></TBODY></TABLE><P style="font-size:13px; color:red;"><B>Comment</B>:The primer O120 was firstly given to the samples. It's a different primer for the Vp123_suffix, which binds to the regulatory-sequence. Because there was no knowledge, if the Vp_ex constucts have this sequence or not, O92 was also given to the samples.  </P><P style="font-size:13px; color:red;"><B>Comment</B>:Because gel-picture showed only DNA within the gel-bags, PCR was repeated.  </P><P><B>Digestion of plasmid backbone:</B>
Plasmid used: pSB1C3_001 (P320)
c (pSB1C3_001) = 408 ng/ µl
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>vector</B> Volume/µL
</TD></TR><TR><TD align="left"> DNA  </TD><TD align="left"> 2,5
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left">1,5
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 1,5
</TD></TR><TR><TD align="left"> SpeI </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> NgomIV </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left">7,5
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>15</B></TD></TR></TBODY></TABLE><P>incubation @ 37 C for approx. 1,5 h
</P><P style="font-size:13px; color:red;"><B>Comment</B>:Strangely, PCR-products did not run into the gel, altough the PCR was repeated, after first PCR had two reverse primers in the sample. This could be because of a complex with the polymerase.However, ligation will be continued with the DNA within the bags. </P>&lt;/br&gt;<P><B>Digestion of PCR product:</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>PCR products</B> Volume/µL
</TD></TR><TR><TD align="left"> DNA   </TD><TD align="left"> 20
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 3
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 3
</TD></TR><TR><TD align="left"> SpeI </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> NgomIV </TD><TD align="left">1
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 2
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>30</B></TD></TR></TBODY></TABLE><P>
Digestion was performed at 37 °C for 2 hours.</P><P><B>PCR-Purification:</B></P><UL><LI>was peformed according the standard-protocol.
</LI></UL><UL><LI>c (P432) =-- ng/µl</LI></UL><P style="font-size:13px; color:red;"><B>Comment</B>:THE WHOLE PCR WILL BE REPEATED ONCE AGAIN  </P><H4><SPAN class="mw-headline" id="RNA_Purity_Gel"><P style="font-size:15px; background-color:#66bbff;"><B>RNA Purity Gel</B></P></SPAN></H4><P><B>Investigator: Kira</B>
10x FA buffer: 200mM MOPS, 50mM Sodium Acetat, 10mM EDTA, RNase free water 
1x FA running buffer: 10x FA buffer, 37% Formamid, RNAase free water </P><P><B>Agarose gel</B>: Agarose 0,6 g; 10x FA Buffer 5ml; RNAse free water 45ml; 37% Formamid 0,9 ml </P><H4><SPAN class="mw-headline" id="hTERT_PCR_Reaction"><P style="font-size:15px; background-color:#66bbff;"><B>hTERT PCR Reaction</B></P></SPAN></H4><P><B>Investigator: Kira </B></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption"><P style="color:#66bbff;"> @Kira: Do you think that we can (suggestion?) use the A431 for detection of gene expression driven by phTERT promoter?? We should keep in mind that the transduction efficieny will not be that well ;-) (Bea)</P></DIV></DIV></DIV><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD align="right">Volume / µl
</TD></TR><TR><TD> 10x PCR buffer </TD><TD align="right">2
</TD></TR><TR><TD> 10 mM dNTP mix</TD><TD align="right">0,4
</TD></TR><TR><TD> 50mM MgCl2 </TD><TD align="right">0,6
</TD></TR><TR><TD> Primer 1 </TD><TD align="right">0,5
</TD></TR><TR><TD> Primer 2</TD><TD align="right">0,5
</TD></TR><TR><TD> cDNA template </TD><TD align="right">1
</TD></TR><TR><TD> Taq Polymerase</TD><TD align="right">0,1
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">14,9
</TD></TR><TR><TD>Total volume</TD><TD align="right">20
</TD></TR></TBODY></TABLE><P><B>PCR program:</B></P><TABLE border="1"><TBODY><TR><TD>Cycles</TD><TD>Temperature / °C</TD><TD>Time
</TD></TR><TR><TD>1x</TD><TD>94</TD><TD>3min
</TD></TR><TR><TD>30x</TD><TD>94</TD><TD>45sec
</TD></TR><TR><TD>51</TD><TD>30
</TD></TR><TR><TD>72</TD><TD>1min 30sec
</TD></TR><TR><TD>1x</TD><TD>72</TD><TD>10
</TD></TR><TR><TD>Hold</TD><TD> 4
</TD></TR></TBODY></TABLE><P>1% agarose gel: 
Evaluation: The agarose gel reveals that all samples show telomerase activity but in different levels.
</P><H4><SPAN class="mw-headline" id="Cloning_of_pBAD_Affibody_middlelinker_EGFP_His"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of pBAD_Affibody_middlelinker_EGFP_His</B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><UL><LI>Vector: name: pBAD_ OmpA His FluA LL Foki <B>PGerrit</B></LI><LI>Insert: name: pSB1C3_ZEGFR:1907_middlelinker_EGFP_His <B>P566</B></LI><LI>new vector name: pBAD_ZEGFR:1907_middlelinker_EGFP_His <B>P</B></LI><LI>buffer used: 4 
</LI><LI>DNA concentration (vector): 97,0 ng/µl ; DNA concentration (insert): 209,5 µg/µl
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume of pBAD_ OmpA His FluA LL Foki  /µl</B></TD><TD align="right"><B>volume of pSB1C3_ZEGFR:1907_middlelinker_EGFP_His /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">10 </TD><TD align="right">6
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2</TD><TD align="right">2
</TD></TR><TR><TD>Enzyme XbaI</TD><TD align="right">0,8 </TD><TD align="right">0,8
</TD></TR><TR><TD>Enzyme AgeI</TD><TD align="right">0,8 </TD><TD align="right">0,8
</TD></TR><TR><TD>H2O</TD><TD align="right">4,4 </TD><TD align="right">8,4
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 20</TD><TD align="right">20
</TD></TR></TBODY></TABLE><P>
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 120 Volt
</P><P><B>Loading plan for agarose gel</B>: 
Loading dye with SDS (6X), 3 µl 
Marker used: GeneRuler ladder mix (Fermentas), 6 µl</P><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>Sample PGerrit, 20µl
</TH><TH>Sample 566, 20µl
</TH></TR><TR><TH>Lane
</TH><TD>1
</TD><TD>3
</TD><TD>5
</TD></TR><TR><TH>Fragment size
</TH><TD>4047 bp
</TD><TD>963 bp
</TD></TR></TBODY></TABLE><P><B>Gel extraction</B>: </P><TABLE border="1"><TBODY><TR><TH>pBAD
</TH><TH>Affibody_middlelinker_EGFP_His
</TH></TR><TR><TH>DNA-concentration [ng/µl]
</TH><TD>8,3
</TD><TD>10,2
</TD></TR></TBODY></TABLE><P><B>T4 Ligation</B>: 
Volume vector:  5,06 µl 
Volume insert:  2,94 µl <B>Transformation</B>: </P><P>XL1B cells were used.
</P><H4><SPAN class="mw-headline" id="Picking_clones_of_pSB1C3_CMV_VP123_inscap"><P style="font-size:15px; background-color:#66bbff;"><B>Picking clones of pSB1C3_CMV_VP123_inscap</B></P></SPAN></H4><P><B>Investigator: Anna</B></P><P>Three clones were picked (10 ml DYT, 10 µl Cm).
</P><H4><SPAN class="mw-headline" id="Cloning_of_.2A.2A.2A.2A.2A_into_pSB1C3_CFP"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of ***** into pSB1C3_CFP</B></P></SPAN></H4><P><B>Investigator: Chris W.</B></P><UL><LI>Vector: name: pSB1C3_CFP P51,2
</LI><LI>Insert: name: email Hybrid Oligo (O184&amp;O185)
</LI><LI>new vector name: pSB1C3_email 
</LI><LI>buffer used: 4 
</LI><LI>DNA concentration (vector): 151,0 ng/µl ; DNA concentration (insert): 95,3 µg/µl
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Interesting, where does the promoter come from the new annotared vector?? ;-)</DIV></DIV></DIV><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume of pSB1C3_CFP /µl</B></TD><TD align="right"><B>volume of email/µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">6 </TD><TD align="right">15
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2</TD><TD align="right">2
</TD></TR><TR><TD>Enzyme I</TD><TD align="right">NgoMIV 1 </TD><TD align="right">NgoMIV 1
</TD></TR><TR><TD>Enzyme II</TD><TD align="right">SpeI 1 </TD><TD align="right">SpeI 1
</TD></TR><TR><TD>H2O</TD><TD align="right">8,4 </TD><TD align="right">-
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 20</TD><TD align="right">20
</TD></TR></TBODY></TABLE><P>
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 120 Volt, running time:45
</P><P><B>Loading plan for agarose gel</B>: 
Loading dye with SDS (6X), 4 µl 
Marker used: GeneRuler ladder mix (Fermentas), 3 µl</P><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>Sample Insert, 20µl
</TH><TH>Sample Vector, 20µl
</TH></TR><TR><TH>Lane
</TH><TD>1
</TD><TD>3
</TD><TD>5
</TD></TR><TR><TH>Fragment size
</TH><TD>70 bp
</TD><TD>2070 bp
</TD></TR></TBODY></TABLE><P><B>Gel extraction</B>: </P><TABLE border="1"><TBODY><TR><TH>pSB1C3_CFP_cut
</TH><TH>email
</TH></TR><TR><TH>DNA-concentration [ng/µl]
</TH><TD>7,8
</TD><TD>27,4
</TD></TR></TBODY></TABLE><P><B>T4 Ligation</B>: 
Volume vector: 8,74 µl 
Volume insert: 0,26 µl <B>Transformation</B>: </P><P>XL1B cells were used.
</P><H4><SPAN class="mw-headline" id="Repetition:_Cloning_of_P5_TATAless_downstream_of_pSB1C3_001_RC_insrepcap_KpnI_back_and_P5_upstream_of_pSB1C3_001_RC_insrepcap_KpnI_back"><P style="font-size:15px; background-color:#66bbff;"><B>Repetition: Cloning of P5_TATAless downstream of pSB1C3_001_RC_insrepcap_KpnI_back and P5 upstream of pSB1C3_001_RC_insrepcap_KpnI_back</B></P></SPAN></H4><P><B>Investigator: Stefan</B><B>Digestion of the constructs:</B></P><UL><LI>P432 = pSB1C3_001_RC_insrepcap_KpnI_back c=344,5 ng/µl</LI><LI>P432 = pSB1C3_001_RC_insrepcap_KpnI_back c=344,5 ng/µl</LI><LI>P180 = pSB1C3_pTAV2 (P5)clone 1 c=108,8 ng/µl</LI><LI>P184 = pSB1C3_pAAV_RC (P5TATAless)clone 3 c=176,9 ng/µl</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">My dear friend, did you mix up the Rep-Cap constructs in the plasmid list? P432 = pAAV_RC_insRepCap_KpnI-back ;)</DIV></DIV></DIV><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>V P432up /µL</B></TD><TD align="left"><B>VP180up /µL</B></TD><TD align="left"><B>V P432down /µL</B></TD><TD align="left"><B>V P184down /µL</B></TD></TR><TR><TD align="left"> DNA  </TD><TD align="left"> 4,5</TD><TD align="left">14 </TD><TD align="left"> 4,5 </TD><TD align="left">11
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 2</TD><TD align="left">2 </TD><TD align="left"> 2</TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 2</TD><TD align="left">2 </TD><TD align="left"> 2</TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> Enzyme 1 </TD><TD align="left">XbaI 1</TD><TD align="left">SpeI 1 </TD><TD align="left">SpeI 1</TD><TD align="left">XbaI 1
</TD></TR><TR><TD align="left"> Enzyme 2 </TD><TD align="left">EcoRI 1</TD><TD align="left">EcoRI 1 </TD><TD align="left">PstI 1</TD><TD align="left">PstI 1
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 9,5</TD><TD align="left">9,5</TD><TD align="left">-</TD><TD align="left">3
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>20</B></TD></TR></TBODY></TABLE><P><B>Results:</B><B>Gel-Extraction:</B>
Gel-Ex was performed according to protocol. The following concentrations were measured:
</P><UL><LI>432up c= 25,73 ng/µl</LI><LI>180up c= 9,57 ng/µl</LI><LI>432down c= 27,34 ng/µl</LI><LI>184down c= 17,80 ng/µl</LI></UL><P><B>Ligation:</B>
The following DNA mixes were used for ligation:</P><UL><LI>432up V= 6,3 µl</LI><LI>180up V= 1,7 µl</LI><LI>432down V= 6,93 µl</LI><LI>184down V= 1,07 µl</LI></UL><P><B>Transformation:</B>
The transformation was performed according to standard protocol using BL21 cells. They were plated on agar plates containing chloramphenicol and stored in 37°C room overnight.
</P><H3><SPAN class="mw-headline" id="129._labday_24.09.2010"><P style="font-size:17px; background-color:#00dd77;">129. labday 24.09.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Sequencing_results_of_VB_13.4"><P style="font-size:15px; background-color:#66bbff;"><B>Sequencing results of VB 13.4</B></P></SPAN></H4><P><B>Investigator: Achim</B></P><P>We sent in another clone of the 587 KO Z34C motif to see if it carries the same mutations as the first one. If that had been the case, we could account the mutations to faulty oligos. However, sequencing showed that the second 587 KO Z34C clone is mutated as well, but in different positions. So we either recieved oligos containing various mutations, or the fill-in reaction was faulty in spite of the proofreading activity of the klenow fragment.
</P><P>Because 587 KO Z34C is one of our more promising approaches, we sent another two clones from our second colony pcr/clone picking round for sequencing: 13.1 and 13.4
</P><H4><SPAN class="mw-headline" id="Contuniation:_Cloning_pSB1C3_Viral_Brick_587KO-empty_.28B274.2FP542.29_into_pSB1C3_001_VP2.2F3_Capins_.28B438.2FP514.29"><P style="font-size:15px; background-color:#66bbff;"><B>Contuniation: Cloning pSB1C3_Viral Brick 587KO-empty (B274/P542) into pSB1C3_001_VP2/3_Capins (B438/P514)</B></P></SPAN></H4><P><B>Investigator Patrick </B>
Because of the sequencing results of clone 1 a mini-prep was performed only with clone 2 and 3 yielding the following concentrations: 
pSB1C3_VP2/3_Capins_587KO-empty clone 2: 195 ng/µl 
pSB1C3_VP2/3_Capins_587KO-empty clone 3: 224,74 ng/µl </P><P>Testdigestion: 
pSB1C3_VP2/3_Capins_587KO-empty clone 2 (P578/B472): 3 µl Buffer 4, 0,5 µl BamHI, 0,5 µl PvuII, 22 µl DNA, 4 µl H2O 
pSB1C3_VP2/3_Capins_587KO-empty clone 3 (P579/B473): 23 µl Buffer 4, 0,5 µl BamHI, 0,5 µl PvuII, 20 µl DNA, 6 µl H2O 
Because somebody switched the 37°C thermoblock off i dont know the exact digestion time. Therefore i switched the thermoblock on again and waited 1 hour. I used that much DNA because the desired insert is 48 bp long.
</P><P>Clone 3 was sent for sequencing.
</P><H4><SPAN class="mw-headline" id="Sequencing_evaluation_of_VP2.2F3_Gsat-linker_and_VP2.2F3_cap_Gsat-linker"><P style="font-size:15px; background-color:#66bbff;"><B>Sequencing evaluation of VP2/3_Gsat-linker and VP2/3_cap_Gsat-linker</B></P></SPAN></H4><P><B>Investigator Kira</B></P><P><B>Comment:</B> pSB1C3_VP2/3_cap_Gsat-linker as well as pSB1C3_VP2/3_Gsat-linker can be used for further experiments.
</P><H4><SPAN class="mw-headline" id="New_CD_primer_ordered"><P style="font-size:15px; background-color:#66bbff;"><B>New CD primer ordered</B></P></SPAN></H4><P><B>Investigator Stefan</B></P><P style="font-size:13px; color:red;"><B>Comment</B>: CD reverse primer was incorrect. A new primer was ordered. </P><H4><SPAN class="mw-headline" id="Sequencing_of_pCerulean_Zegfr:1907_VP2.2F3"><P style="font-size:15px; background-color:#66bbff;"><B>Sequencing of pCerulean_Zegfr:1907_VP2/3</B></P></SPAN></H4><P><B>Investigator Hanna</B>
Because I found out, that this plasmid was sent to GATC with the wrong sequencing primers (therefore no sequencing results were available), I sent it again with the GATC_std_CMV-F primer.
--&gt; backbone = pCerulean and not pSB1C3!!! ;)
</P><H4><SPAN class="mw-headline" id="Repetition:_PCR_for_biobrick_production:_Cloning_RepCap_into_pSB1C3"><P style="font-size:15px; background-color:#66bbff;"><B>Repetition: PCR for biobrick production: Cloning RepCap into pSB1C3</B></P></SPAN></H4><P><B>Investigator: Stefan </B></P><P style="font-size:13px; color:red;"><B>Comment</B>: Construct was cloned mixed RFC10 and RFC25. Therefore, a new approach will be performed consisting of RFC10 only.</P><P>
Plasmid used:</P><LI>pAAV_RC_RepCapIns_SDMKpnI (P432)

A 1:1000 dilutionof the plasmid was prepared.
<B>PCR protocol:</B><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD align="right"> P432 (v/µl)
</TD></TR><TR><TD> 5X Phusion HF buffer </TD><TD align="right">10
</TD></TR><TR><TD> 10 mM dNTP mix</TD><TD align="right">1
</TD></TR><TR><TD> forward primer: O93 </TD><TD align="right">2,5
</TD></TR><TR><TD> reverse primer: O121 </TD><TD align="right">2,5
</TD></TR><TR><TD> DNA Template</TD><TD align="right">3
</TD></TR><TR><TD> DMSO </TD><TD align="right">-
</TD></TR><TR><TD> Phusion Polymerase</TD><TD align="right">0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">30,5
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"><B>50</B></TD></TR></TBODY></TABLE><P><B>PCR program:</B></P><TABLE border="1"><TBODY><TR><TD>Cycles</TD><TD>Temperature /°C</TD><TD>Time /s
</TD></TR><TR><TD>1</TD><TD>98</TD><TD>60
</TD></TR><TR><TD>2 (8x step 2-4)</TD><TD>98</TD><TD>15
</TD></TR><TR><TD>3</TD><TD>59</TD><TD>25
</TD></TR><TR><TD>4</TD><TD>72</TD><TD>70
</TD></TR><TR><TD>5 (17x step 5-6)</TD><TD>98</TD><TD>15
</TD></TR><TR><TD>6</TD><TD>72</TD><TD>80
</TD></TR><TR><TD>71x</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>Hold 4°C
</TD></TR></TBODY></TABLE><P><B>Digestion of the vector:</B></P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">pSB1C3_VCK_Bla (P320) (v/µl)
</TD></TR><TR><TD> DNA  </TD><TD align="right">3,5
</TD></TR><TR><TD> BSA  </TD><TD align="right">2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2
</TD></TR><TR><TD>SpeI </TD><TD align="right">1
</TD></TR><TR><TD>EcoRI  </TD><TD align="right">1
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">10,5
</TD></TR><TR><TD><B>Total volume </B></TD><TD align="right">20
</TD></TR></TBODY></TABLE><P style="font-size:13px; color:red;"><B>Comment</B>:Digestion was performed 2 hours at 37 °C.</P><P><B>Gel:</B>
0,5 g Agarose,50 ml TAE (1%), 3 µl Gelred , at 110 Volt; run for ~50 minutes</P><P><B>Gelextraction:</B></P><P>The gelextraction was performed according to the standard protocol. DNA concentration of the vector:
</P></LI><LI>pSB1C3 (P320): c= 8,5 ng/µl</LI><P><B>Digestion of the PCR-products:</B></P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">PCR product (v/µl)
</TD></TR><TR><TD> DNA  </TD><TD align="right">30
</TD></TR><TR><TD> BSA  </TD><TD align="right">4
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">4
</TD></TR><TR><TD>SpeI </TD><TD align="right">1
</TD></TR><TR><TD>EcoRI  </TD><TD align="right">1
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">-
</TD></TR><TR><TD><B>Total volume </B></TD><TD align="right">40
</TD></TR></TBODY></TABLE><P style="font-size:13px; color:red;"><B>Comment</B>:Digestion was performed 2 hours at 37 °C.</P><P><B>PCR purification:</B>
Purification was performed accoding to the standard protocol. The following concentrations were measured:
</P><UL><LI>PCR of P432: c= 6,4 ng/µl</LI></UL><P><B>Ligation:</B>
1 µl T4 Ligase buffer (2x)
8 µl DNA mix
1 µl T4 Ligase
 Incubating for 45 minutes at room temperature.
The  Ligation was performed using the following amounts of DNA:</P><UL><LI>Vector Volume: 0,92 µl</LI><LI>Insert Volume: 7,08 µl</LI></UL><P><B>Transformation:</B></P><P>Trafo was performed according to the standard protocol using XL1b cells. The cells were plated on a agar plates containing chlorampenicol.
</P><H4><SPAN class="mw-headline" id="Picking_clones_of_pSB1C3_CFP_email"><P style="font-size:15px; background-color:#66bbff;"><B>Picking clones of pSB1C3_CFP_email</B></P></SPAN></H4><P><B>Investigator: Anna</B></P><P>Three clones were picked (10 ml DYT, 10 µl Chloramphenicol).
</P><H4><SPAN class="mw-headline" id="MiniPreps_and_Test_digestion_of_pSB1C3_CMV_VP123_capins"><P style="font-size:15px; background-color:#66bbff;"><B>MiniPreps and Test digestion of pSB1C3_CMV_VP123_capins</B></P></SPAN></H4><P><B>Investigator: Anna</B></P><TABLE border="1"><TBODY><TR><TD> Sample   </TD><TD align="right">Concentration [ng/µl]
</TD></TR><TR><TD> pSB1C3_CMV_VP123_capins_clone1  </TD><TD align="right">377,29
</TD></TR><TR><TD>pSB1C3_CMV_VP123_capins_clone2 </TD><TD align="right">369,92
</TD></TR><TR><TD>pSB1C3_CMV_VP123_capins_clone3 </TD><TD align="right">387,57
</TD></TR></TBODY></TABLE><P><B>Test digestion:</B></P><TABLE border="1"><TBODY><TR><TD> Components   </TD><TD align="right">Volume /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right">2
</TD></TR><TR><TD> BSA  </TD><TD align="right">-
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">1
</TD></TR><TR><TD>EcoRI </TD><TD align="right">0,5
</TD></TR><TR><TD>NgoMIV  </TD><TD align="right">0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">6
</TD></TR><TR><TD><B>Total volume </B></TD><TD align="right">10
</TD></TR></TBODY></TABLE><P><B>Expected size (CFP): 681 bp</B></P><P style="font-size:13px; color:red;"><B>Comment</B>: Clone 1 was sent for sequencing with the following primers: VF2, 4000 for, 4200 rev, 2800 for and 2800 rev.</P><H4><SPAN class="mw-headline" id="MiniPreps_of_587_Z34C_ViralBricks_and_pSB1C3_p40"><P style="font-size:15px; background-color:#66bbff;"><B>MiniPreps of 587 Z34C ViralBricks and pSB1C3_p40</B></P></SPAN></H4><P><B>Investigator: Anna</B></P><TABLE border="1"><TBODY><TR><TD> Sample  </TD><TD align="right">Clone </TD><TD align="right">Concentration [ng/µl]
</TD></TR><TR><TD>pSB1C3_587_KO_Z34C </TD><TD align="right">13.1</TD><TD align="right">266,97
</TD></TR><TR><TD>pSB1C3_587_KO_Z34C </TD><TD align="right">13.4</TD><TD align="right">277,47
</TD></TR></TBODY></TABLE><P style="font-size:13px; color:red;"><B>Comment</B>: Both clones were sent for sequencing.</P><TABLE border="1"><TBODY><TR><TD> Sample   </TD><TD align="right">Clone</TD><TD align="right">Concentration [ng/µl]
</TD></TR><TR><TD>pSB1C3_p40 </TD><TD align="right">1</TD><TD align="right">266,97
</TD></TR><TR><TD>pSB1C3_p40 </TD><TD align="right">2</TD><TD align="right">277,47
</TD></TR><TR><TD>pSB1C3_p40 </TD><TD align="right">3</TD><TD align="right">167,20
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Re-Trafo_of_DARPin"><P style="font-size:15px; background-color:#66bbff;"><B>Re-Trafo of DARPin</B></P></SPAN></H4><P><B>Investigator Stefan</B></P><P style="font-size:13px; color:red;"><B>Comment</B>: A Re-Trafo of the synthesized DARPin was performed using XL1b cells.</P><H3><SPAN class="mw-headline" id="130._labday_25.09.2010"><P style="font-size:17px; background-color:#00dd77;">130. labday 25.09.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Sequencing_of_VB:_13.1.2C_13.4_.28approach_w._dephosphorylated_vector.2C_second_clone_picking_round"><P style="font-size:15px; background-color:#66bbFF;"><B>Sequencing of VB: 13.1, 13.4 (approach w. dephosphorylated vector, second clone picking round</B></P></SPAN></H4><P><B>Investigator: Achim</B></P><P>The two clones that were sequenced both contained a deletion in the same position in the insert. We really seem to have bad luck picking clones. Hopefully our PCR attempt works better.
</P><H4><SPAN class="mw-headline" id="Miniprep_of_serveral_constructs_2"><P style="font-size:15px; background-color:#66bbFF;"><B>Miniprep of serveral constructs</B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><P><B>Glycerol stocks were prepared:</B></P><UL><LI>B489 =	pBAD_EGFP_His clone1</LI><LI>B490 =	pBAD_EGFP_His clone2</LI><LI>B491 =	pBAD_EGFP_His clone3</LI><LI>B492 =	pSB1C3_CFP_email clone 1</LI><LI>B493 =	pSB1C3_CFP_email clone 2</LI><LI>B494 =	pSB1C3_CFP_email clone 3</LI></UL><P><B>Mini-Prep was performed according to the standard protocol</B></P><UL><LI>P588=	pBAD_EGFP_His clone1 c=71,0 ng/µl</LI><LI>P589 =	pBAD_EGFP_His clone2 c=74,8 ng/µl</LI><LI>P590 =	pBAD_EGFP_His clone3 c=58,3 ng/µl</LI><LI>P591 =	pSB1C3_CFP_email clone 1 c=132,7 ng/µl</LI><LI>P592 =	pSB1C3_CFP_email clone 2 c=168,9 ng/µl</LI><LI>P593 =	pSB1C3_CFP_email clone 3 c=153,6 ng/µl</LI></UL><P><B>Test digestion:</B></P><TABLE border="1"><TBODY><TR><TD> Components   </TD><TD align="right">Volume P588-590 /µl</TD><TD align="right">Volume P591-593 /µl</TD><TD align="right">Volume P583-585 /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right">4,5 </TD><TD align="right">2,5</TD><TD align="right">2
</TD></TR><TR><TD> BSA  </TD><TD align="right">1 </TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">1</TD><TD align="right">1</TD><TD align="right">1
</TD></TR><TR><TD>XbaI </TD><TD align="right">0,4 </TD><TD align="right">0,4 </TD><TD align="right">0,4
</TD></TR><TR><TD>PstI  </TD><TD align="right">0,4</TD><TD align="right">0,4</TD><TD align="right">0,4
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">2,7 </TD><TD align="right">4,7</TD><TD align="right">5,2
</TD></TR><TR><TD><B>Total volume </B></TD><TD align="right">10</TD><TD align="right">10</TD><TD align="right">10
</TD></TR></TBODY></TABLE><P><B>Expected size :  bp</B></P><P style="font-size:13px; color:red;"><B>Comment</B>: test digestion of P591-593 and P583-585 (pSB1C3_P40) is just an attempt because I can't find any sequences in geneious.P583 looks well. P591, P588 and P583 will be sent for sequencing on monday. </P><H4><SPAN class="mw-headline" id="Sequencing_results_of_pSb1C3_Affibody_middlelinker_EGFP_His"><P style="font-size:15px; background-color:#66bbFF;"><B>Sequencing results of pSb1C3_Affibody_middlelinker_EGFP_His</B></P></SPAN></H4><P style="font-size:13px; color:red;"><B>Comment</B>: there was anywhere a problem in the cloning process. approach will be repeated today. there is pBAD_EGFP_His, affibody_middlelinker will be cloned in, also in pSB1C3_EGFP_His</P><H4><SPAN class="mw-headline" id="Sequencing_result_of_pSB1C3_CMV_VP123_capins_clone1"><P style="font-size:15px; background-color:#66bbff;"><B>Sequencing result of pSB1C3_CMV_VP123_capins_clone1</B></P></SPAN></H4><P><B>Investigators: Volker, Anna</B></P><P>For test digestion go to labday 24.09.10 
</P><P style="font-size:13px; color:red;"><B>Comment</B>: The following primers were used for sequencing: VF2, 4000 for, 4200 rev, 2800 for and 2800 rev. There is a point mutation in the non-coding sequence, which should not influence the performence of the construct, so it can be used for further cloning steps.</P><H4><SPAN class="mw-headline" id="Cloning_of_453_and_587_ViralBricks_into_pSB1C3_CMV_VP123_capins"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of 453 and 587 ViralBricks into pSB1C3_CMV_VP123_capins</B></P></SPAN></H4><P><B>Investigators: Achim, Anna</B></P><UL><LI>Vector name: pSB1C3_CMV_VP123_capins <B>P580</B></LI><LI>Inserts: different ViralBricks (see pdf underneath), pSB1C3_VCK_Bla <B>P320</B></LI><LI>buffer used: 4 
</LI></UL><P style="font-size:13px; color:red;"><B>Update</B>: ViralBricks ready to use:</P><P><A href="/wiki/images/6/61/Freiburg10_Production_of_ViralBricks_new.pdf" class="internal" title="Freiburg10 Production of ViralBricks new.pdf">media:Freiburg10_Production of ViralBricks_new.pdf</A><B>Digestion of vector and inserts:</B></P><P><A href="/wiki/images/7/7f/Freiburg10_Cloning_of_ViralBricks_into_pSB1C3_CMV_VP123_capins.pdf" class="internal" title="Freiburg10 Cloning of ViralBricks into pSB1C3 CMV VP123 capins.pdf">media:Freiburg10_Cloning of ViralBricks into pSB1C3_CMV_VP123_capins.pdf</A></P><P>The vector p580 was dephosphorylated.
Two gels were prepared with 1 g Agarose, 100 ml TAE (1%) and 6 µl GELRED; at 120 Volt, running time:45
</P><P><B>Gel run</B>: 
Loading dye(6X), 6 µl 
Marker used: GeneRuler ladder mix (Fermentas), 3 µl</P><P>Expected sizes of fragments: see Pdf under <B>Gel extraction and overnight T4 Ligation</B></P><P><B>Gel extraction and overnight T4 Ligation</B>: </P><P><A href="/wiki/images/2/2e/Freiburg10_Cloning_of_ViralBricks_into_pSB1C3_CMV_VP123_capinsII.pdf" class="internal" title="Freiburg10 Cloning of ViralBricks into pSB1C3 CMV VP123 capinsII.pdf">media:Freiburg10_Cloning of ViralBricks into pSB1C3_CMV_VP123_capinsII.pdf</A></P><H4><SPAN class="mw-headline" id="Sequencing_results:_pSB1C3_001_VP2.2F3_HSPG-KO"><P style="font-size:15px; background-color:#66bbff;"><B>Sequencing results: pSB1C3_001_VP2/3_HSPG-KO</B></P></SPAN></H4><P><B>Investigator: Hanna</B><B>Comment:</B> In order to knock down the natural tropism of the virus, the HSPG knock-out motif (two aa exchanges) was cloned into VP2/3. This fragment can then be used for the VP2 N-terminal fusion or VP1 insertion approaches. 
Conclusion: Sequencing results looked well: The HSPG-KO ViralBrick was cloned successfully into VP2/3. </P><H4><SPAN class="mw-headline" id="Cloning_of_pBAD_Affibody_middlelinker_EGFP_His_and_pSB1C3_Affibody_middlelinker_EGFP_His"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of pBAD_Affibody_middlelinker_EGFP_His and pSB1C3_Affibody_middlelinker_EGFP_His</B></P></SPAN></H4><P><B>Investigator: Jessica</B><B>Comment: There was a mistake in cloning pSB1C3_Affibody_middlelinker_EGFP_His so there is just pSB1C3_EGFP_His. in this plasmid will be cloned Affibody_middlelinker, also in pBAD_EGFP_His</B></P><UL><LI>Vector: name: pBAD_EGFP_His  <B>P588</B></LI><LI>Vector: name: pSB1C3_EGFP_His  <B>P565</B></LI><LI>Insert: name: pSB1C3_ZEGFR:1907_middlelinker_ <B>P290</B></LI><LI>new vector name: pBAD_ZEGFR:1907_middlelinker_EGFP_His <B>P</B></LI><LI>new vector name: pSb1C3_ZEGFR:1907_middlelinker_EGFP_His <B>P</B></LI><LI>buffer used: 4 
</LI><LI>DNA concentration (vector): 71,0 ng/µl / 252,7 ng/µl; DNA concentration (insert): 227,4 µg/µl
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume of pBAD_EGFP_His  /µl</B></TD><TD align="right"><B>volume of pSB1C3_EGFP_His /µl</B></TD><TD align="right"><B>volume of pSB1C3_ZEGFR:1907_middlelinker /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">15 </TD><TD align="right">4  </TD><TD align="right">7
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right"> 2 </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2</TD><TD align="right">2  </TD><TD align="right"> 2
</TD></TR><TR><TD>Enzyme XbaI</TD><TD align="right">0,8 </TD><TD align="right">0,8 </TD><TD align="right">0,8
</TD></TR><TR><TD>Enzyme AgeI</TD><TD align="right">- </TD><TD align="right">- </TD><TD align="right">0,8
</TD></TR><TR><TD>Enzyme NgoMIV</TD><TD align="right">0,8 </TD><TD align="right">0,8 </TD><TD align="right">-
</TD></TR><TR><TD>H2O</TD><TD align="right">-</TD><TD align="right">10,4</TD><TD align="right">7,4
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 20,6</TD><TD align="right">20 </TD><TD align="right">20
</TD></TR></TBODY></TABLE><P>
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 120 Volt
</P><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>P290, 20µl
</TH><TH>P565, 20µl
</TH><TH>P588, 20µl
</TH></TR><TR><TH>Sample
</TH><TD>4
</TD><TD>1
</TD><TD>2
</TD><TD>3
</TD></TR><TR><TH>Fragment size
</TH><TD>210 bp
</TD><TD>2800 bp
</TD><TD>4700 bp
</TD></TR></TBODY></TABLE><P><B>Gel extraction</B>: </P><TABLE border="1"><TBODY><TR><TH>pBAD
</TH><TH>pSB1C3
</TH><TH>Affibody_middlelinker
</TH></TR><TR><TH>DNA-concentration [ng/µl]
</TH></TR></TBODY></TABLE><P><B>T4 Ligation pBAD</B>: 
Volume vector:  3,5 µl 
Volume insert:  4,5 µl <B>T4 Ligation pSB1C3</B>: 
Volume vector:  2,3 µl 
Volume insert:  5,7 µl </P><H3><SPAN class="mw-headline" id="131._labday_26.09.2010"><P style="font-size:17px; background-color:#00dd77;">131. labday 26.09.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Continuation_of_cloning_of_pBAD_Affibody_middlelinker_EGFP_His_and_pSB1C3_Affibody_middlelinker_EGFP_His"><P style="font-size:15px; background-color:#66bbff;"><B>Continuation of cloning of pBAD_Affibody_middlelinker_EGFP_His and pSB1C3_Affibody_middlelinker_EGFP_His</B></P></SPAN></H4><P><B>Transformation</B>: </P><P>BL21 cells were used - in 37°C at 8.30am
</P><UL><LI> pBAD_Affibody_middlelinker_EGFP_His
</LI><LI> pSB1C3_Affibody_middlelinker_EGFP_His</LI></UL><P style="font-size:13px; color:red;"><B>Comment</B>:pSB1C3_Affibody_middlelinker_EGFP_His was not successful (no colonies on plate), will be repeated on monday</P><H4><SPAN class="mw-headline" id="Picking_clones_of_different_constructs"><P style="font-size:15px; background-color:#66bbff;"><B>Picking clones of different constructs</B></P></SPAN></H4><P><B>Investigator: Anna </B></P><P>Clones were picked of:</P><UL><LI>pBad_Affibody_middlelinker_EGFP_His</LI><LI>pSB1C3_001_RC_InsRepCap_KpnIback_RFC10</LI><LI>Darpin (ReTrafo)</LI><LI>pSB1C3_453_BAP (p402, ReTrafo)
</LI><LI>pSB1C3_587_KO_RGD (p390, ReTrafo) 
</LI><LI>pSB1C3_587_KO_His (p391, ReTrafo) 
</LI></UL><P>In addition, preparations for Mini-Preps of several ViralBricks were done: 
</P><UL><LI>B289: 587_BAP
</LI><LI>B270: 587_KO_BAP
</LI><LI>B271: 587_RGD
</LI><LI>B272: 453_His
</LI><LI>B273: 587_His
</LI><LI>B274: 587_KO_empty
</LI><LI>B488: 453_Z34C
</LI></UL><P style="font-size:13px; color:red;"><B>Comment</B>: Inoculating of 10 mL DYT medium containing 10µL chloramphenicol (except pBad with Amp).
There were no clones on the plate with pSB1C3_Affibody_middlelinker_EGFP_His. </P><H4><SPAN class="mw-headline" id="Continuation_of_cloning_of_453_and_587_inserts_into_pSB1C3_CMV_VP123_capins"><P style="font-size:15px; background-color:#66bbff;"><B>Continuation of cloning of 453 and 587 inserts into pSB1C3_CMV_VP123_capins</B></P></SPAN></H4><P><B>Investigator: Achim, Anna</B></P><P><B>Transformation</B>: </P><P>Trafo was performed according to the standard protocol using XL1b cells. The cells were plated on agar plates containing chlorampenicol and stored in the 37°C room.
</P><H4><SPAN class="mw-headline" id="Colony_PCR_of_remaining_587_constructs"><P style="font-size:15px; background-color:#66bbff;"><B>Colony PCR of remaining 587 constructs</B></P></SPAN></H4><P><B>Investigator: Achim, Anna</B></P><P style="font-size:13px; color:red;"><B>Comment</B>: New clones of the remaining 587 constructs were picked (from new cloning approach with T4 Ligation and dephosphorylated vector, see labdays 15.09. and 22.09.). </P><P>
Constructs:
</P><UL><LI>12: 587_Z34C
</LI><LI>13: 587_KO_Z34C
</LI><LI>14: 587_Z34C_KO_Spacer
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>Components</B></TD><TD align="right"><B>Volume /µl</B></TD></TR><TR><TD> 10x Standard Taq Buffer (with MgCl2)  </TD><TD align="right">1
</TD></TR><TR><TD> MgCl2 (50mM) </TD><TD align="right">0,1
</TD></TR><TR><TD> dNTPs</TD><TD align="right">0,2
</TD></TR><TR><TD>Primer 1: RFC25 for </TD><TD align="right">0,2
</TD></TR><TR><TD>Primer 2: RFC25 rev</TD><TD align="right">0,2
</TD></TR><TR><TD>Taq Polymerase</TD><TD align="right">0,05
</TD></TR><TR><TD>H2O</TD><TD align="right">8,25
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 10
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD><B>Cycles</B></TD><TD align="right"><B>Temperature</B></TD><TD align="right"><B>Time</B></TD></TR><TR><TD align="right">95°C </TD><TD align="right">6 min
</TD></TR><TR><TD> 30x </TD><TD align="right">95°C </TD><TD align="right">25 sec
</TD></TR><TR><TD>(Tm -3°C)</TD><TD align="right">60°C</TD><TD align="right">30 sec
</TD></TR><TR><TD>1 min/kb</TD><TD align="right">68°C </TD><TD align="right">55 sec
</TD></TR><TR><TD>1x</TD><TD align="right">68°C</TD><TD align="right">5 min
</TD></TR><TR><TD>Hold 4°C</TD></TR></TBODY></TABLE><P style="font-size:13px; color:red;"> Clone 13.3, 14.1 and 14.3 will be sent for sequencing.</P><H4><SPAN class="mw-headline" id="Sequencing_results:_pCerulean_Zegfr:1907_VP2.2F3"><P style="font-size:15px; background-color:#66bbff;"><B>Sequencing results: pCerulean_Zegfr:1907_VP2/3</B></P></SPAN></H4><P><B>Investigator: Hanna </B></P><P>
Sequencing results revealed that VP2/3 (unmodified!) was successfully fused to the Affibody (without linker). 
Next step: Fusion of VP2/3-HSPG-KO to the C-terminus of the Affibody.
</P><H4><SPAN class="mw-headline" id="N-terminal_VP2-Fusion_.26_VP1_Insertion:_Fusion_of_VP2.2F3_HSPG-KO"><P style="font-size:15px; background-color:#66bbff;"><B>N-terminal VP2-Fusion &amp; VP1 Insertion: Fusion of VP2/3_HSPG-KO</B></P></SPAN></H4><P><B>Investigator: Hanna </B></P><P><B>Comment:</B> In order to target tumor cells with our AAV2 two VP2 fusion strategies were figuered out: The so called VP2-fusion and VP1 insertions. These approaches were successfully cloned and are now tested in cell culture. In order to prevent targeting of other (healthy) cells, the natural tropism of the virus needs to be knocked down. For this purpose a ViralBrick, &quot;587-KO_empty&quot;, was designed and cloned into pSB1C3_001_VP2/3_Capins. 
Today this VP2/3_HSPG-KO sequence will be fused to the N-terminus of the Affibody, CFP/mVenus and His-Tag of both, the N-terminal fusion to VP2 and the VP1 insertion, approach. </P><P><B>DIGESTION:</B>
1. pCerulean_CFP_MiddleLinker (P407)
2. pCerulean_Z<SUB>EGFR:1907</SUB>_MiddleLinker (P408)
3. pCerulean_Z<SUB>EGFR:1907</SUB>_SEG (P409)
4. pCerulean_Z<SUB>EGFR:1907</SUB>_ShortLinker (P410)
5. pCerulean_Z<SUB>EGFR:1907</SUB>_LongLinker (P412)
6. pCerulean_6xHis_MiddleLinker (P374)
7. pCerulean_VP1up_NLS_Z<SUB>EGFR:1907</SUB> (P422)
8. pCerulean_VP1up_NLS_6xHis (P424)
9. pCerulean_VP1up_NLS_mVenus (P426)
10.1 pSB1C3_001_Vp2/3_HSPG-KO (P579)
10.2 pSB1C3_001_Vp2/3_HSPG-KO (P579)</P><P>
Incubation: 2 hours, 37°C. 
Agarose gel: 0.8% </P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (6x)/µl
</TH><TH>Expected size
</TH></TR><TR><TD>1
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD>4710 bp
</TD></TR><TR><TD>2
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD>4169 bp
</TD></TR><TR><TD>3
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD>4254 bp
</TD></TR><TR><TD>4
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD>4158 bp
</TD></TR><TR><TD>5
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD>4181 bp
</TD></TR><TR><TD>6
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD>2129 bp
</TD></TR><TR><TD>7
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD>4583 bp
</TD></TR><TR><TD>8
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD>4428 bp
</TD></TR><TR><TD>9
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD>5124 bp
</TD></TR><TR><TD>10.1
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD>1937 bp
</TD></TR><TR><TD>10.2
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD>1937 bp
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker /µL
</TH><TH>Sample 1 /µl
</TH><TH>Sample 2 /µl
</TH><TH>Sample 3 /µl
</TH><TH>Sample 4 /µl
</TH><TH>Sample 5 /µl
</TH><TH>Sample 6 /µl
</TH><TH>Sample 7 /µl
</TH><TH>Sample 8 /µl
</TH><TH>Sample 9 /µl
</TH><TH>Sample 10.1 /µl
</TH><TH>Sample 10.2 /µl
</TH></TR><TR><TH>Lane
</TH><TD>4.5
</TD><TD>24
</TD><TD>24
</TD><TD>24
</TD><TD>24
</TD><TD>24
</TD><TD>24
</TD><TD>24
</TD><TD>24
</TD><TD>24
</TD><TD>36
</TD><TD>36
</TD></TR></TBODY></TABLE><P><B>LIGATION:</B></P><P><B>TRAFO:</B>
Trafo was performed following our standard protocol. Cells: XL1b, DNA: 2µL. 
Plates will be stored over night @ the 37°C room. <B>Next step:</B> Picking 2 clones of each construct, colony PCR!
</P><H3><SPAN class="mw-headline" id="132._labday_27.09.2010"><P style="font-size:17px; background-color:#00dd77;">132. labday 27.09.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Sent_for_sequencing"><P style="font-size:15px; background-color:#66bbff;"><B>Sent for sequencing</B></P></SPAN></H4><P><B>Investigator: Jessica </B></P><UL><LI>P591 pSB1C3_email: VR2
</LI><LI>P583 pSB1C3_P40: VR2
</LI><LI>P588 pBAD_EGFP_His: pBAD-FP
</LI><LI>P596 pBAD_Affibody_middlelinker_EGFP_His: pBAD-FP / pTrcHis-RP
</LI></UL><H4><SPAN class="mw-headline" id="Picking_clones_of_VP1_insertion_and_VP2_fusion_HSPG-KO_approaches"><P style="font-size:15px; background-color:#66bbff;"><B>Picking clones of VP1 insertion and VP2 fusion HSPG-KO approaches</B></P></SPAN></H4><P><B>Investigator: Hanna</B>
2 clones were picked of each ligation approach = 18 falcons (Kanamycin).
To do: Mini-Prep and test digestion - sequencing.
</P><H4><SPAN class="mw-headline" id="MiniPreps_of_different_constructs"><P style="font-size:15px; background-color:#66bbff;"><B>MiniPreps of different constructs</B></P></SPAN></H4><P><B>Investigator: Anna </B></P><P>The following constructs were prepped according to the standard protocol:
</P><UL><LI>pSB1C3_453_BAP (p402, ReTrafo)
</LI><LI>pSB1C3_587_KO_RGD (p390, ReTrafo) 
</LI><LI>pSB1C3_587_KO_His (p391, ReTrafo) 
</LI><LI>B289: 587_BAP
</LI><LI>B290: 453_RGD
</LI><LI>B270: 587_KO_BAP
</LI><LI>B271: 587_RGD
</LI><LI>B272: 453_His
</LI><LI>B273: 587_His
</LI><LI>B274: 587_KO_empty
</LI><LI>B488: 453_Z34C
</LI></UL><UL><LI>B473:
</LI></UL><H4><SPAN class="mw-headline" id="Miniprep_of_serveral_constructs_3"><P style="font-size:15px; background-color:#66bbFF;"><B>Miniprep of serveral constructs</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P><B>Glycerol stocks were prepared:</B></P><UL><LI>B489 =	pSB1C3_001_RC_InsRepCap_KpnIback_RFC10  clone 1</LI><LI>B490 =	pSB1C3_001_RC_InsRepCap_KpnIback_RFC10  clone 2</LI><LI>B491 =	pBAD_Affibody_middlelinker_EGFP_His clone 1</LI><LI>B492 =	pBAD_Affibody_middlelinker_EGFP_His clone 2</LI><LI>B493 =	pBAD_Affibody_middlelinker_EGFP_His clone 3</LI></UL><P><B>Mini-Prep was performed according to the standard protocol</B></P><UL><LI>P594=	pSB1C3_001_RC_InsRepCap_KpnIback_RFC10  clone 1 c= 140,9 ng/µl</LI><LI>P595 =	pSB1C3_001_RC_InsRepCap_KpnIback_RFC10  clone 2 c= 117,5 ng/µl</LI><LI>P596 =	pBAD_Affibody_middlelinker_EGFP_His clone 1 c= 101,4 ng/µl</LI><LI>P597 =	pBAD_Affibody_middlelinker_EGFP_His clone 2 c= 92,9 ng/µl</LI><LI>P598 =	pBAD_Affibody_middlelinker_EGFP_His clone 3 c= 92,1 ng/µl</LI></UL><P><B>Test digestion:</B></P><TABLE border="1"><TBODY><TR><TD> Components   </TD><TD align="right">Volume P594+P595 /µl</TD><TD align="right">Volume P596-598 /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right">2,5 </TD><TD align="right">2,5|
</TD></TR><TR><TD> BSA  </TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">1</TD><TD align="right">1
</TD></TR><TR><TD>XbaI </TD><TD align="right">0,4 </TD><TD align="right">0,4
</TD></TR><TR><TD>PstI  </TD><TD align="right">0,4</TD><TD align="right">0,4
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">4,7 </TD><TD align="right">4,7
</TD></TR><TR><TD><B>Total volume </B></TD><TD align="right">10</TD><TD align="right">10
</TD></TR></TBODY></TABLE><P style="font-size:13px; color:red;"><B>Comment</B>:</P><H4><SPAN class="mw-headline" id="Re-Trafo_of_pSB1C3_pTAV_.28P5.29"><P style="font-size:15px; background-color:#66bbff;"><B>Re-Trafo of pSB1C3_pTAV (P5)</B></P></SPAN></H4><P><B>Investigator: Stefan </B></P><P style="font-size:13px; color:red;"><B>Comment</B>:After inocculating from glycerol-stock, the plasmid showed strange results during digestion. Therefore a re-trafo will have to be performed to replace the stock.</P><H4><SPAN class="mw-headline" id="Repetition:_Cloning_of_P5_TATAless_downstream_of_pSB1C3_001_RC_insrepcap_KpnI_back_and_P5_upstream_of_pSB1C3_001_RC_insrepcap_KpnI_back_2"><P style="font-size:15px; background-color:#66bbff;"><B>Repetition: Cloning of P5_TATAless downstream of pSB1C3_001_RC_insrepcap_KpnI_back and P5 upstream of pSB1C3_001_RC_insrepcap_KpnI_back</B></P></SPAN></H4><P><B>Investigator: Stefan</B><B>Digestion of the constructs:</B></P><UL><LI>P594 = pSB1C3_001_RC_insrepcap_KpnI_back c=344,5 ng/µl</LI><LI>P594 = pSB1C3_001_RC_insrepcap_KpnI_back c=344,5 ng/µl</LI><LI>P586 = pSB1C3_pTAV2 (P5)clone 1 c=137,37 ng/µl</LI><LI>P587 = pSB1C3_pAAV_RC (P5TATAless)clone 3 c=191,61 ng/µl</LI></UL><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>V P594up /µL</B></TD><TD align="left"><B>VP586up /µL</B></TD><TD align="left"><B>V P594down /µL</B></TD><TD align="left"><B>V P587down /µL</B></TD></TR><TR><TD align="left"> DNA  </TD><TD align="left"> 10</TD><TD align="left">14 </TD><TD align="left"> 10 </TD><TD align="left">13
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 2</TD><TD align="left">2 </TD><TD align="left"> 2</TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 2</TD><TD align="left">2 </TD><TD align="left"> 2</TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> Enzyme 1 </TD><TD align="left">XbaI 1</TD><TD align="left">SpeI 1 </TD><TD align="left">SpeI 1</TD><TD align="left">XbaI 1
</TD></TR><TR><TD align="left"> Enzyme 2 </TD><TD align="left">EcoRI 1</TD><TD align="left">EcoRI 1 </TD><TD align="left">PstI 1</TD><TD align="left">PstI 1
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 9,5</TD><TD align="left">9,5</TD><TD align="left">-</TD><TD align="left">3
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>20</B></TD></TR></TBODY></TABLE><P><B>Results:</B></P><P style="font-size:13px; color:red;"><B>Comment</B>:P5 and P5TATAless do not fit the expected results. For P5TATAless, a new digestion using the P184 plasmid was performed. For P5 no old plasmid stock was available, therefore a re-trafo will have to be performed.</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>V P184down /µL</B></TD></TR><TR><TD align="left"> DNA  </TD><TD align="left"> 10
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> XbaI </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> PstI </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 4
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>20</B></TD></TR></TBODY></TABLE><P style="font-size:13px; color:red;"><B>Comment</B>:No gel picture available because computer crashed completely...</P><P><B>Gel-Extraction:</B>
Gel-Ex was performed according to protocol. The following concentrations were measured:
</P><UL><LI>594down c= 29,9 ng/µl</LI><LI>184down c= 3,8 ng/µl</LI></UL><P><B>Ligation:</B>
The following DNA mixes were used for ligation:</P><UL><LI>432down V= 4,48 µl</LI><LI>184down V= 3,52 µl</LI></UL><P><B>Transformation:</B>
The transformation was performed according to standard protocol using BL21 cells. They were plated on agar plates containing chloramphenicol and stored in 37°C room overnight.
</P><H4><SPAN class="mw-headline" id="Picking_clones_of_VP1_insertion_and_VP2_fusion_HSPG-KO_approaches_2"><P style="font-size:15px; background-color:#66bbff;"><B>Picking clones of VP1 insertion and VP2 fusion HSPG-KO approaches</B></P></SPAN></H4><P><B>Investigator: Hanna</B>
2 clones were picked of each ligation approach = 18 falcons (Kanamycin).
To do: Mini-Prep and test digestion - sequencing.
</P><H4><SPAN class="mw-headline" id="Picking_clones_of_VP1_insertion_and_VP2_fusion_HSPG-KO_approaches_3"><P style="font-size:15px; background-color:#66bbff;"><B>Picking clones of VP1 insertion and VP2 fusion HSPG-KO approaches</B></P></SPAN></H4><P><B>Investigator: Hanna</B>
2 clones were picked of each ligation approach = 18 falcons (Kanamycin).
To do: Mini-Prep and test digestion - sequencing.
</P><H3><SPAN class="mw-headline" id="133._labday_28.09.2010"><P style="font-size:17px; background-color:#00dd77;">133. labday 28.09.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Cloning_of_the_ViralBricks_BAP-HSPG-ko_and_His-HSPG-ko_into_pSB1C3_VP2.2F3"><P style="font-size:15px; background-color:#66bbFF;"><B>Cloning of the ViralBricks BAP-HSPG-ko and His-HSPG-ko into pSB1C3_VP2/3</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="color:#66bbff;"><I>Comment</I>: After discussing the finals constructs, we decided to clone HSPG-KO with BAP and His motifs into the N-terminal approaches which means that the plasmids pCerulean_Targeting_VP2/3 contains His and BAP ViralBricks integrated in the 587 loop, respectively. Therefore I started the first cloning step in order to obtain the fully assembled targeting construct which can be cotransfected with the plasmids containing rep and cap genes (with or wihtout HSPG-ko) </P><P><B>Digestion of the constructs:</B></P><UL><LI>P604 = pSB1C3_001_587-ko_BAP </LI><LI>P606 = pSB1C3_001_587-ko_His</LI><LI>P525 = pSB1C3_001_VP2/3capins</LI></UL><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>v<SUB>P606</SUB> /µL</B></TD><TD align="left"><B>v<SUB>P525 upstream</SUB>/µL</B></TD><TD align="left"><B>v<SUB>P604</SUB> /µL</B></TD></TR><TR><TD align="left"> DNA  </TD><TD align="left"> 21</TD><TD align="left">6 </TD><TD align="left">18
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 4</TD><TD align="left">2 </TD><TD align="left"> 4
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 4</TD><TD align="left">2 </TD><TD align="left"> 4
</TD></TR><TR><TD align="left"> Enzyme 1 </TD><TD align="left">BamHI</TD><TD align="left">BamHI </TD><TD align="left">BamHI
</TD></TR><TR><TD align="left"> Enzyme 2 </TD><TD align="left">SpeI</TD><TD align="left">PvuII</TD><TD align="left">PvuII
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left">9</TD><TD align="left">8</TD><TD align="left">12
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>40</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>40</B></TD></TR></TBODY></TABLE><P><B>Loading plan of a 1% agarose gel:</B></P><PRE>M    P525    
</PRE><P><B>Loading plan of a 2% agarose gel:</B></P><PRE>M    P604    P606
</PRE><P><B>Expected sizes of the digested products are:</B></P><UL><LI> P525= 48<B>,3800 bp</B></LI><LI> P604= 2216, <B>81bp</B></LI><LI> P606 = 2216, <B>108bp</B></LI></UL><P><B>Results:</B></P><P>
After gel extraction has been performed, the 2µL of each ligated plasmids were transformed into BL-21 cells.
</P><H4><SPAN class="mw-headline" id="Test_digestion:_ViralBricks_in_pSB1C3_CMV_VP123_capins"><P style="font-size:15px; background-color:#66bbff;"><B>Test digestion: ViralBricks in pSB1C3_CMV_VP123_capins</B></P></SPAN></H4><P><B>Investigator: Achim </B></P><UL><LI>We digested preps of all our 453 Inserts with BamHI &amp; SspI and all our 587 Inserts with SalI and PvuII. That way we get bands of ~50 bp difference between our loop inserts and the wt loop region. (Eg. 453 BAP: 455 with insert, 396 without insert.)
</LI></UL><UL><LI>Nomenclature as in previous VB assays.
</LI></UL><UL><LI>Clone 11.2 is missing , I forgot to inoculate it.
</LI></UL><UL><LI>P580 (pSB1C3_CMV_VP123_capins)
</LI></UL><DIV class="thumb tnone"><DIV class="thumbinner" style="width:472px;"><DIV class="thumbcaption">The framed samples were midiprepped and will be used for transfection</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Sequencing_result_of_pSB1C3_email_and_pBAD_Affibody_middlelinker_EGFP_His"><P style="font-size:15px; background-color:#66bbff;"><B>Sequencing result of pSB1C3_email and pBAD_Affibody_middlelinker_EGFP_His</B></P></SPAN></H4><P><B>Investigator: Jessica </B><B>1. pSB1C3_email:</B></P><P><B>2. pBAD_Affibody_middlelinker_EGFP_His:</B></P><H4><SPAN class="mw-headline" id="Preparation_of_PBS_Buffer_and_PBS_Buffer_with_Imidazole"><P style="font-size:15px; background-color:#66bbff;"><B>Preparation of PBS Buffer and PBS Buffer with Imidazole</B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><P>The following protocol was used:
</P><UL><LI>dissolve the following in 800ml ddH<SUB>2</SUB>O:
</LI><LI>8g of NaCl
</LI><LI>0.2g of KCl
</LI><LI>1.44g of Na<SUB>2</SUB>HP0<SUB>4</SUB> (1.69 because of the crystal water in the reagent)
</LI><LI>0.24 g of KH<SUB>2</SUB>PO<SUB>4</SUB></LI></UL><LI>adjust to pH 7.4
</LI><LI>adjust to 1000ml
</LI><LI>steril filtrate the solution
</LI><UL><LI>dissolve the following in 180ml ddH<SUB>2</SUB>O:
</LI><LI>6,4g of NaCl
</LI><LI>0,16g of KCl
</LI><LI>1,15g of Na<SUB>2</SUB>HP0<SUB>4</SUB> (1.69 because of the crystal water in the reagent)
</LI><LI>0,19 g of KH<SUB>2</SUB>PO<SUB>4</SUB></LI><LI>3,4g of Imidazole 
</LI></UL><LI>adjust to pH 7.4
</LI><LI>adjust to 200ml
</LI><LI>steril filtrate the solution
</LI><H4><SPAN class="mw-headline" id="ViralBrick_Sequencing_results:_pSB1C3_587_KO_Z34C_and_pSB1C3_587_KO_Z34C_Spacer"><P style="font-size:15px; background-color:#66bbff;"><B>ViralBrick Sequencing results: pSB1C3_587_KO_Z34C and pSB1C3_587_KO_Z34C_Spacer</B></P></SPAN></H4><P><B>Investigator: Achim</B></P><P>Sequencing showed that the PCR reaction we designed to remove the mutations in the z34c insert were successful! Both  pSB1C3_587_KO_Z34C and pSB1C3_587_KO_Z34C_Spacer are correct. We hadn't sent the 587_Z34C for sequencing because the colony PCR hadn't shown any bands. But because this construct is less important for cell culuture, we won't continue working on this construct. </P><H4><SPAN class="mw-headline" id="Cloning_of_Darpin_into_pCerulean_Vp1up_NLS_and_pSB1C3"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of Darpin into pCerulean_Vp1up_NLS and pSB1C3 </B></P></SPAN></H4><P><B>Investigator: Achim</B></P><UL><LI>The Darpin needs to be cloned into pSB1C3 for submission to the parts registry as well as into pCerulean for expression.
</LI></UL><UL><LI>Darpin ( cut with NgOMIV &amp; SpeI
</LI><LI>Vectors cut with AgeI &amp; SpeI
</LI></UL><UL><LI>-&gt; Results in fusion protein
</LI></UL><H4><SPAN class="mw-headline" id="Cloning_of_pSB1C3_001_CMV_and_pSB1C3_001_Vp2.2F3_HSPG-Ko_CMV_ZEGFR:1907"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of pSB1C3_001_CMV and pSB1C3_001_Vp2/3_HSPG-Ko_CMV_ZEGFR:1907</B></P></SPAN></H4><P><B>Investigator: Anissa</B><B>Comment: By working with three-fragment-ligation, CMV and ZEGFR:1907 will each be cut out of pSB1C3, will be ligated together and into the vector pSB1C3_001_Vp2/3_HSPG-Ko. Additionally, CMV will be cloned freom pSB1C3 into pSB1C3_001  </B></P><UL><LI>Vector: name:pSB1C3_001_Vp2/3_HSPG-Ko <B>P613</B> c=273,6 ng/µl
</LI><LI>Vector: name: pSB1C3_001_bla  <B>P320</B> c=408 ng/µl
</LI><LI>Insert: name: pSB1C3_CMV <B>P146</B> c=155,6 ng/µl
</LI><LI>Insert: name: pSB1C3_ZEGFR:1907 <B>P268</B> c=162,5 ng/µl
</LI><LI>new vector name: pSB1C3_001_CMV 
</LI><LI>new vector name: pSB1C3_001_Vp2/3_HSPG-Ko_CMV_ZEGFR:1907
</LI><LI>buffer used: 4 
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>pSB1C3_CMV /µl</B></TD><TD align="right"><B>pSB1C3_ZEGFR:1907  /µl</B></TD><TD align="right"><B>volume of pSB1C3_001_bla /µl</B></TD><TD align="right"><B>volume of pSB1C3_001_Vp2/3_HSPG-Ko/µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">12,7 </TD><TD align="right">12,3  </TD><TD align="right"> 3,6</TD><TD align="right">5,5
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right"> 2 </TD><TD align="right">1,5 </TD><TD align="right">1,5
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2</TD><TD align="right">2  </TD><TD align="right">1,5 </TD><TD align="right">1,5
</TD></TR><TR><TD>Enzyme SpeI</TD><TD align="right">1</TD><TD align="right">- </TD><TD align="right">1</TD><TD align="right">-
</TD></TR><TR><TD>Enzyme EcoRI</TD><TD align="right">1 </TD><TD align="right">- </TD><TD align="right">1</TD><TD align="right">1
</TD></TR><TR><TD>Enzyme XbaI</TD><TD align="right">- </TD><TD align="right">1 </TD><TD align="right">-</TD><TD align="right">-
</TD></TR><TR><TD>Enzyme AgeI</TD><TD align="right">- </TD><TD align="right">1 </TD><TD align="right">-</TD><TD align="right">-
</TD></TR><TR><TD>Enzyme NgoMIV</TD><TD align="right">-</TD><TD align="right">-</TD><TD align="right">-</TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">1,3</TD><TD align="right">1,7</TD><TD align="right">6,4</TD><TD align="right">4,5
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 20</TD><TD align="right">20 </TD><TD align="right">15</TD><TD align="right">15
</TD></TR></TBODY></TABLE><P>
0,5 g Agarose, 55 ml TAE (1%), 1,5 µl GELRED , at 115Volt
</P><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>P146, 20µl
</TH><TH>P268, 20µl
</TH><TH>P320, 20µl
</TH><TH>P613, 20µl
</TH></TR><TR><TH>Fragment size
</TH><TD> -
</TD><TD>650 bp
</TD><TD>170 bp
</TD><TD>2100 bp
</TD><TD>4000 bp
</TD></TR></TBODY></TABLE><P><B>Gel extraction</B>: </P><TABLE border="1"><TBODY><TR><TH>pSB1C3_CMV (p146)
</TH><TH>pSB1C3_ZEGFR:1907 (p268)
</TH><TH>pSB1C3_001_bla (p320)
</TH><TH>pSB1C3_001_Vp2/3_HSPG-Ko (p613)
</TH></TR><TR><TH>DNA-concentration [ng/µl]
</TH><TD>28,2
</TD><TD>13,2
</TD><TD>22,1
</TD><TD>43,2
</TD></TR></TBODY></TABLE><P><B>T4 Ligation pSB1C3_001_CMV</B>: 
Volume vector p320: 4,63 µl 
Volume insert p146: 3,37 µl <B>T4 Ligation pSB1C3_001_Vp2/3_HSPG-Ko_CMV_ZEGFR:1907</B>: 
Volume vector p613:  2,9 µl 
Volume insert p268:  3 µl 
Volume insert p146:  3 µl <B>Transformation</B>: 
Was performed in BL21 according the standard protocol </P><H4><SPAN class="mw-headline" id="CD_assembly_with_new_suffix_primer"><P style="font-size:15px; background-color:#66bbff;"><B>CD assembly with new suffix primer</B></P></SPAN></H4><P><B>Investigator: Kira</B></P><UL><LI>PCR program:
</LI></UL><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume in µl  </TD><TD align="right">volume in µl
</TD></TR><TR><TD> 5x Phusion HF buffer  </TD><TD align="right"> 10</TD><TD align="right">10
</TD></TR><TR><TD> 10 mM dNTP mix </TD><TD align="right">1</TD><TD align="right">1
</TD></TR><TR><TD> 0158 primer_for (1:10 dilution)</TD><TD align="right">  2,5</TD><TD align="right"> 2,5
</TD></TR><TR><TD>0159 primer_rev (1:10 dilution)</TD><TD align="right"> 2,5</TD><TD align="right"> 2,5
</TD></TR><TR><TD>DNA template (1:100)</TD><TD align="right">  0,5</TD><TD align="right"> 0,5
</TD></TR><TR><TD>DMSO</TD><TD align="right">  0</TD><TD align="right"> 0.5
</TD></TR><TR><TD>Phusion polymerase</TD><TD align="right"> 0,5</TD><TD align="right"> 0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right"> 33</TD><TD align="right"> 32,5
</TD></TR><TR><TD><B>Total volume (e.g. 50 µl)</B></TD><TD align="right"> 50</TD><TD align="right"> 50
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD>Cycles</TD><TD>Temperature</TD><TD>Time
</TD></TR><TR><TD>98°C</TD><TD>1
</TD></TR><TR><TD>10x</TD><TD>98°C</TD><TD>15&quot;
</TD></TR><TR><TD>60°C</TD><TD>25&quot;
</TD></TR><TR><TD>72°C</TD><TD>40&quot;
</TD></TR><TR><TD>20x</TD><TD>98°C</TD><TD>15&quot;
</TD></TR><TR><TD>65°C</TD><TD>25&quot;
</TD></TR><TR><TD>72°C</TD><TD>40&quot;
</TD></TR><TR><TD>1x</TD><TD>72°C</TD><TD>5'
</TD></TR><TR><TD>Hold 4°C
</TD></TR></TBODY></TABLE><P>no PCR bands are visible on the gel..--&gt; PCR reaction failed.</P><H4><SPAN class="mw-headline" id="test_digestion_of_all_CD_constructs"><P style="font-size:15px; background-color:#66bbFF;"><B>test digestion of all CD constructs</B></P></SPAN></H4><P><B>Investigator: Kira</B></P><P><B>Comment</B>: Because the PCR failed also with new suffix primer, we decided to check if the DNA samples are ok. 3 different CD samples were used as DNA samples.
1_original CD with iGEM binding sites p(223)
2_CD with SDM of PstI 
3_CD with SDM of PstI and NgoMIV (p290) </P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume in µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 3
</TD></TR><TR><TD> BSA 10x </TD><TD align="right">2,0
</TD></TR><TR><TD> Buffer 4</TD><TD align="right">  2,0
</TD></TR><TR><TD>PstI-HF</TD><TD align="right"> 0,5
</TD></TR><TR><TD>NgoMIV</TD><TD align="right"> 1,0
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right"> 11,5
</TD></TR><TR><TD><B>Total volume (e.g. 20 µl)</B></TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Cloning_of_pSB1C3_Affibody_middlelinker_EGFP_His"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of pSB1C3_Affibody_middlelinker_EGFP_His</B></P></SPAN></H4><P><B>Investigator: Jessica</B><B>Comment: There was a mistake in cloning pSB1C3_Affibody_middlelinker_EGFP_His so there is just pSB1C3_EGFP_His. in this plasmid will be cloned Affibody_middlelinker_EGFP_His</B></P><UL><LI>Vector: name: pSB1C3_EGFP_His  <B>P565</B></LI><LI>Insert: name: pBAD_Affibody_middlelinker_EGFP_His <B>P596</B></LI><LI>new vector name: pSb1C3_ZEGFR:1907_middlelinker_EGFP_His <B>P</B></LI><LI>buffer used: 4 
</LI><LI>DNA concentration (vector): 252,7 ng/µl; DNA concentration (insert): 101,4 µg/µl
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume of pBAD  /µl</B></TD><TD align="right"><B>volume of pSB1C3 /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">15 </TD><TD align="right">4
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2</TD><TD align="right">2
</TD></TR><TR><TD>Enzyme XbaI</TD><TD align="right">0,8 </TD><TD align="right">0,8
</TD></TR><TR><TD>Enzyme AgeI</TD><TD align="right">0,8 </TD><TD align="right">0,8
</TD></TR><TR><TD>H2O</TD><TD align="right">-</TD><TD align="right">10,4
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 20,6</TD><TD align="right">20
</TD></TR></TBODY></TABLE><P>
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 120 Volt
</P><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>P565, 20µl
</TH><TH>P596, 20µl
</TH></TR><TR><TH>Sample
</TH></TR><TR><TH>Fragment size
</TH><TD>2100 bp
</TD><TD>1000 bp
</TD></TR></TBODY></TABLE><P><B>Gel extraction</B>: </P><TABLE border="1"><TBODY><TR><TH>pSB1C3
</TH><TH>Affibody_middlelinker_EFGP_His
</TH></TR><TR><TH>DNA-concentration [ng/µl]
</TH><TD>17,5
</TD><TD>14,2
</TD></TR></TBODY></TABLE><P><B>T4 Ligation</B>: 
Volume vector:  2,9 µl 
Volume insert:  5,1 µl <B>Transformation</B>: </P><P>BL21 cells were used
</P><H4><SPAN class="mw-headline" id="Sequencing_analysis_of_pSB1C3_001_RC_IRCK.2C_pSB1C3_001_RC_IRCK_VP1-ko_P5tataless_and_pSB1C3_001_RC_IRCK_VP2-ko_P5tataless"><P style="font-size:15px; background-color:#66bbff;"><B>Sequencing analysis of pSB1C3_001_RC_IRCK, pSB1C3_001_RC_IRCK_VP1-ko_P5tataless and pSB1C3_001_RC_IRCK_VP2-ko_P5tataless</B></P></SPAN></H4><P><B>Investigator: Stefan </B></P><P style="font-size:13px; color:red;">Because all these constructs were cloned via PCR out of the pAAV vector, a complete sequencing was performed:</P><P><B>P548	pSB1C3_001_RC_InsRepCap_KpnIback_VP1-ko_P5tataless clone 1:</B>
Sequence could be verified, can be used for further cloning steps!<B>P568	pSB1C3_001_RC_InsRepCap_KpnIback_VP2-KO_P5tataless clone 2:</B>
Sequence could be verified, can be used for further cloning steps!<B>P594	pSB1C3_001_RC_InsRepCap_KpnIback_RFC10  clone 1:</B>
Sequence could be verified, can be used for further cloning steps!</P><H4><SPAN class="mw-headline" id="Cloning_of_HSPG_KO_empty_into_pSB1C3_001_RC_IRCK_VP1-ko_P5tataless_and_pSB1C3_001_RC_IRCK_VP2-ko_P5tataless"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of HSPG_KO_empty into pSB1C3_001_RC_IRCK_VP1-ko_P5tataless and pSB1C3_001_RC_IRCK_VP2-ko_P5tataless</B></P></SPAN></H4><P><B>Investigator: Stefan </B></P><P>Vector name:
</P><UL><LI>pSB1C3_001_RC_IRCK_VP1-ko_P5tataless (P548)</LI><LI>pSB1C3_001_RC_IRCK_VP1-ko_P5tataless (P568)</LI><P>Insert name:
</P><LI>VB:pSB1C3_001_KO_empty(P611)</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume of P548  /µl</B></TD><TD align="right"><B>volume of P568 /µl</B></TD><TD align="right"><B>volume of P611 /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">3,5 </TD><TD align="right">3,5  </TD><TD align="right">16
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right"> 2 </TD><TD align="right"> 3
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2</TD><TD align="right">2  </TD><TD align="right"> 3
</TD></TR><TR><TD>Enzyme BamHI</TD><TD align="right">1 </TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>Enzyme PvuII</TD><TD align="right">1 </TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">10,5</TD><TD align="right">10,5</TD><TD align="right">6
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 20</TD><TD align="right">20 </TD><TD align="right">30
</TD></TR></TBODY></TABLE><P><B>Gel:</B></P><P>for vector:
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 115 Volt
for insert:</P>
1 g Agarose, 50 ml TAE (2%), 3 µl GELRED , at 105 Volt<P><B>Gel extraction</B>: 
Was performed according to protocol.
</P><P><B>T4 Ligation</B>: </P><P><B>VP1-ko:</B></P><UL><LI>Volume VP1-ko:  5,57 µl</LI><LI>Volume insert:  2,43 µl</LI></UL><P><B>VP2-ko</B></P><UL><LI>Volume VP2-ko:  5,04 µl</LI><LI>Volume insert:  2,96 µl</LI></UL><P><B>Transformation</B>: 
Was performed according to standard protocol using BL21 cells.
</P><H4><SPAN class="mw-headline" id="Cloning_of_587_KO_Z34C_and_587_KO_Z34C_Spacer_into_pSB1C3_001_CMV_VP123_capins"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of 587_KO_Z34C and 587_KO_Z34C_Spacer into pSB1C3_001_CMV_VP123_capins</B></P></SPAN></H4><P><B>Investigator: Anna</B></P><P style="font-size:13px; color:red;">Comment: Todays sequencing results of 587_KO_Z34C and 587_KO_Z34C_Spacer looked well, so they will be cloned into pSB1C3_CMV_VP123_capins. Finally produced ViralBricks:</P><P><A href="/wiki/images/2/20/Freiburg10_Production_of_ViralBricks_End_version.pdf" class="internal" title="Freiburg10 Production of ViralBricks End version.pdf">media:Freiburg10_Production of ViralBricks_End version.pdf</A></P><UL><LI>Vector name: pSB1C3_CMV_VP123_capins  <B>P580</B></LI><LI>Insert name: 
</LI></UL><P>1. pSB1C3_587_KO_Z34C <B>P599</B>
2. pSB1C3_587_KO_Z34C_Spacer <B>P600</B></P><UL><LI>buffer used: 4 
</LI></UL><UL><LI>DNA concentration (P580): 377,29 ng/µl; 
</LI><LI>DNA concentration (P599): 102,5 µg/µl
</LI><LI>DNA concentration (P600): 90,4 µg/µl
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume of P580  /µl</B></TD><TD align="right"><B>volume of P599 /µl</B></TD><TD align="right"><B>volume of P600 /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">2,7 </TD><TD align="right">14,7  </TD><TD align="right">16,7
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2,5 </TD><TD align="right"> 2,5 </TD><TD align="right"> 2,5
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2,5</TD><TD align="right">2,5  </TD><TD align="right"> 2,5
</TD></TR><TR><TD>Enzyme BamHI</TD><TD align="right">1 </TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>Enzyme PvuII</TD><TD align="right">1 </TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">15,3</TD><TD align="right">3,3</TD><TD align="right">1,3
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 25</TD><TD align="right">25 </TD><TD align="right">25
</TD></TR></TBODY></TABLE><P>
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 120 Volt
</P><TABLE border="1"><TBODY><TR><TH>Marker, 4 µl
</TH><TH>P580
</TH><TH>2: P599
</TH><TH>3: P600
</TH></TR><TR><TH>Construct
</TH><TD>pSB1C3_CMV_VP123_cut
</TD><TD>587_KO_Z34C
</TD><TD>587_KO_Z34C_Spacer
</TD></TR><TR><TH>Fragment size
</TH><TD>5000 bp
</TD><TD>194 bp
</TD><TD>209 bp
</TD></TR></TBODY></TABLE><P><B>Gel extraction</B>: 
Was performed following the standard protocol.

DNA-concentrations [ng/µl]:
</P><P>c(vector)= 18,88
c(587_KO_Z34C)= 9,74
c(587_KO_Z34C_Spacer)= 1,72<B>T4 Ligation</B>: </P><P>587_KO_Z34C:
Volume vector:  1,5 µl 
Volume insert:  6,5 µl </P><P>587_KO_Z34C_Spacer:
Volume vector:  4,6 µl 
Volume insert:  3,4 µl </P><P><B>Trafo</B>: 
Was performed following the standard protocol using BL21 cells.
</P><H4><SPAN class="mw-headline" id="Mini-Prep_and_Test-digestion_of_VP1_insertion_and_VP2_fusion_HSPG-KO_approaches"><P style="font-size:15px; background-color:#66bbff;"><B>Mini-Prep and Test-digestion of VP1 insertion and VP2 fusion HSPG-KO approaches</B></P></SPAN></H4><P><B>Investigator: Hanna</B><B>Comments:</B> Yesterday, 2 clones of each construct were picked. Today mini-prep and test digestion will be performed of one of each approach - the other clones will be spinned down and stored at -20°C - waiting for test digestion and sequencing results. </P><P>Mini-Prep was performed, following the standard protocol. <B>New plasmid names:</B>
P617: pCerulean_CFP_MiddleLinker_VP2/3_HSPG-KO 
P618: pCerulean_ZEGFR:1907_MiddleLinker_VP2/3_HSPG-KO 
P619: pCerulean_ZEGFR:1907_SEG_VP2/3_HSPG-KO 
P620: pCerulean_ZEGFR:1907_ShortLinker_VP2/3_HSPG-KO 
P621: pCerulean_ZEGFR:1907_LongLinker_VP2/3_HSPG-KO 
P622: pCerulean_6xHis_MiddleLinker_VP2/3_HSPG-KO 
P623: pCerulean_VP1up_NLS_ZEGFR:1907_VP2/3_HSPG-KO 
P624: pCerulean_VP1up_NLS_6xHis_VP2/3_HSPG-KO 
P625: pCerulean_VP1up_NLS_mVenus_VP2/3_HSPG-KO </P><P><B>TEST DIGESTION:</B>
Mastermix: </P><UL><LI> BSA (10x): 10 µL
</LI><LI> Buffer 4: 10 µL
</LI><LI> EcoRI: 5 µL
</LI><LI> PstI: 5 µL
</LI><LI> H<SUB>2</SUB>O: 37 µL
</LI></UL><P>
DNA: 3.3 µL + 6.7 µL Master Mix.
</P><P><B>Conclusion:</B> Test digestion looked well - except of P618: There is an additional band visible, which can't be correlated to any construct. 
All samples were sent for sequencing. </P><H3><SPAN class="mw-headline" id="134._labday_29.09.2010"><P style="font-size:17px; background-color:#00dd77;">134. labday 29.09.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="CD_assembly_with_new_suffix_primer_2"><P style="font-size:15px; background-color:#66bbff;"><B>CD assembly with new suffix primer</B></P></SPAN></H4><P><B>Investigator: Kira</B></P><UL><LI>PCR program:
</LI></UL><P>as DNA samples p291 and p292 with and wo DMSO were used. </P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume in µl  </TD><TD align="right">volume in µl
</TD></TR><TR><TD> 5x Phusion HF buffer  </TD><TD align="right"> 10</TD><TD align="right">10
</TD></TR><TR><TD> 10 mM dNTP mix </TD><TD align="right">1</TD><TD align="right">1
</TD></TR><TR><TD> 0158 primer_for (1:10 dilution)</TD><TD align="right">  2,5</TD><TD align="right"> 2,5
</TD></TR><TR><TD>0159 primer_rev (1:10 dilution)</TD><TD align="right"> 2,5</TD><TD align="right"> 2,5
</TD></TR><TR><TD>DNA template (1:100)</TD><TD align="right">  0,5</TD><TD align="right"> 0,5
</TD></TR><TR><TD>DMSO</TD><TD align="right">  0</TD><TD align="right"> 0.5
</TD></TR><TR><TD>Phusion polymerase</TD><TD align="right"> 0,5</TD><TD align="right"> 0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right"> 33</TD><TD align="right"> 32,5
</TD></TR><TR><TD><B>Total volume (e.g. 50 µl)</B></TD><TD align="right"> 50</TD><TD align="right"> 50
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD>Cycles</TD><TD>Temperature</TD><TD>Time
</TD></TR><TR><TD>98°C</TD><TD>1
</TD></TR><TR><TD>10x</TD><TD>98°C</TD><TD>15&quot;
</TD></TR><TR><TD>60°C</TD><TD>25&quot;
</TD></TR><TR><TD>72°C</TD><TD>40&quot;
</TD></TR><TR><TD>20x</TD><TD>98°C</TD><TD>15&quot;
</TD></TR><TR><TD>65°C</TD><TD>25&quot;
</TD></TR><TR><TD>72°C</TD><TD>40&quot;
</TD></TR><TR><TD>1x</TD><TD>72°C</TD><TD>5'
</TD></TR><TR><TD>Hold 4°C
</TD></TR></TBODY></TABLE><P>Digestion of plasmid backbone:
</P><P>c (pSB1C3) = 151, 1 ng/ µl
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>vector</B> Volume/µL
</TD></TR><TR><TD align="left"> DNA 1 µg  </TD><TD align="left"> 6,0 µl
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 2 µl
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 2,0 µl
</TD></TR><TR><TD align="left"> Enzyme 1 XbaI </TD><TD align="left"> 1,0 µl
</TD></TR><TR><TD align="left"> Enzyme 2 AgeI</TD><TD align="left"> 1,5 µl
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 7,5 µl
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>20</B></TD></TR></TBODY></TABLE><P>incubation @ 37 C for approx. 2 h
</P><P>1% agarose gel </P><P>
Digestion of PCR product:
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>PCR product</B> Volume/µL
</TD></TR><TR><TD align="left"> DNA   </TD><TD align="left"> 30,0 µl
</TD></TR><TR><TD align="left"> BSA (100x) </TD><TD align="left"> 0,4 µl
</TD></TR><TR><TD align="left"> Buffer no. 4  </TD><TD align="left"> 4,0 µl
</TD></TR><TR><TD align="left"> Enzyme 1 XbaI </TD><TD align="left"> 1,5 µl
</TD></TR><TR><TD align="left"> Enzyme 2 AgeI  </TD><TD align="left"> 2,0 µl
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 2,1 µl
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>40</B></TD></TR></TBODY></TABLE><P>
incubation @ 37 C for approx. 2 h </P><P>ligation with T4 ligase: 
c(insert)= 31,35 ng/ul 
c(vector)= 10,45 ng/ul 
8ul DNA mix = insert 3 ul + vector 5 ul 
T4 ligase 1 ul 
T4 buffer 1 ul </P><P>Transformation: Anna
</P><H4><SPAN class="mw-headline" id="Miniprep_of_serveral_constructs_4"><P style="font-size:15px; background-color:#66bbFF;"><B>Miniprep of serveral constructs</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P><B>Glycerol stocks were prepared:</B></P><UL><LI>B516 =	pSB1C3_001_RC_ICRK_P5tataless_RFC10  clone 1</LI><LI>B517 =	pSB1C3_pTAV (P5)</LI><LI>B518 =	pSB1C3_001_RC_ICRK_P5tataless_RFC10  clone 2</LI></UL><P><B>Mini-Prep was performed according to the standard protocol</B></P><UL><LI>P628 =	pSB1C3_001_RC_ICRK_P5tataless_RFC10  clone 1</LI><LI>P629 =	pSB1C3_pTAV (P5)</LI><LI>P630 =	pSB1C3_001_RC_ICRK_P5tataless_RFC10  clone 2</LI></UL><P><B>Test digestion:</B></P><TABLE border="1"><TBODY><TR><TD> Components   </TD><TD align="right">Volume P628 + P630 /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right">3
</TD></TR><TR><TD> BSA  </TD><TD align="right">1
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">1
</TD></TR><TR><TD>Enzyme I </TD><TD align="right">0,4
</TD></TR><TR><TD>Enzyme II  </TD><TD align="right">0,4
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">4,2
</TD></TR><TR><TD><B>Total volume </B></TD><TD align="right">10
</TD></TR></TBODY></TABLE><P style="font-size:13px; color:red;"><B>Comment</B>:First test digestion using XmaI and SpeI did not work out, thererfore, two different approaches using MngBI/PstI and SnaBI and PstI were performed. This verified successful cloning of P5tataless into the target vector.</P><P><B>Sent for sequencing:</B></P><P>pSB1C3_001_RC_ICRK_P5tataless_RFC10 clone 1 (P628) was sent for sequencing using VR2 primer.
</P><H4><SPAN class="mw-headline" id="mGMK_TK30_approaches"><P style="font-size:15px; background-color:#66bbFF;"><B>mGMK_TK30 approaches</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="color:#66bbff;"><I>Comment</I>: Since we are still waiting for the TK30 gene ordered at Mr.Gene we decided to take another road to produce a BioBrick compatible thymidine kinase gene. Therefore, we are cloning the mGMK_TK30 fusion construct into the pSB1C3 and after removing the PstI site (see next topic) it is ready for submitting it to the parts registry.  Additionally, we are fusing the mGMK_TK30 to the composite part pSB1C3-leftITR_PROMOTER_betaglobin for testing it in cell culture with pur final super construct.</P><P><B>Digestion of the constructs:</B></P><UL><LI>P51.2 = pSB1C3_CFP </LI><LI>P81 = pAAV_RFC25_mGMK_TK30</LI><LI>P434 = pSB1C3_leftITR_CMV_betaglobin</LI><LI>P437 = pSB1C3_leftITR_phTERT_betaglobin</LI></UL><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>v<SUB>P51.2</SUB> /µL</B></TD><TD align="left"><B>v<SUB>P81 upstream</SUB>/µL</B></TD><TD align="left"><B>v<SUB>P81</SUB> /µL</B></TD><TD align="left"><B>v<SUB>P434</SUB>/µL</B></TD><TD align="left"><B>v<SUB>P437</SUB>/µL</B></TD></TR><TR><TD align="left"> DNA  </TD><TD align="left"> 5</TD><TD align="left">5 </TD><TD align="left">10 </TD><TD align="left">4,5</TD><TD align="left">4,5
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 2</TD><TD align="left">2 </TD><TD align="left"> 2</TD><TD align="left"> 2</TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 2</TD><TD align="left">2 </TD><TD align="left"> 2</TD><TD align="left"> 2</TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> Enzyme 1 </TD><TD align="left">EcoRI</TD><TD align="left">SpeI  </TD><TD align="left">SpeI 1</TD><TD align="left">-</TD><TD align="left">-
</TD></TR><TR><TD align="left"> Enzyme 2 </TD><TD align="left">SpeI</TD><TD align="left">XbaI</TD><TD align="left">EcoRI</TD><TD align="left">SpeI</TD><TD align="left">SpeI
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left">9</TD><TD align="left">9</TD><TD align="left">4</TD><TD align="left">10,5</TD><TD align="left">10,5
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>20</B></TD></TR></TBODY></TABLE><P><B>Overview cloning plan:</B></P><P><B>Loading plan of a 0,8% agarose gel:</B></P><PRE>M    P51.2    P81 (digested with E+S)    P81 (digested with X+S)    P434    P437

</PRE><P><B>Expected sizes of the digested products are:</B></P><UL><LI> pAAV_RFC25_mGMK_TK30= 4626, <B>1717bp</B></LI><LI> pAAV_RFC25_mGMK_TK30= 4626, <B>1717bp</B></LI><LI> pSB1C3_CFP = <B>2052</B>, 756bp</LI><LI> pSB1C3_leftITR_CMV_betaglobin= <B>5094b</B></LI><LI> pSB1C3_leftITR_phTERT_betaglobin= <B>5094b</B></LI></UL><P><B>Results:</B></P><P>
After Gel extraction have been performed the ligation was conducetd:
</P><H4><SPAN class="mw-headline" id="Design_of_mutagenesis_primers_for_deletion_of_PstI_in_mGMK_TK30"><P style="font-size:15px; background-color:#66bbFF;"><B>Design of mutagenesis primers for deletion of PstI in mGMK_TK30</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="color:#66bbff;"><I>Comment</I>: Since we are still waiting for the TK30 gene ordered at Mr.Gene we decided to take another road to produce a BioBrick compatible thymidine kinase gene. Therefore, new primers were designed in order to remove the PstI restriction sites within the TK30 gene. The primers will be ordered tomorrow. </P><H4><SPAN class="mw-headline" id="Cloning_of_Darpin_into_pCerulean_Vp1up_NLS_and_pSB1C3_Part_Two"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of Darpin into pCerulean_Vp1up_NLS and pSB1C3 Part Two </B></P></SPAN></H4><P><B>Investigator: Achim</B></P><UL><LI>I accidentally plated yesterdays trafo with the wrong antibiotics and I had to digest pSB1C3 again anyway, so I prepared a new digestion of both vectors with Age &amp; Spe. 
</LI></UL><UL><LI> After watching the gel picture I realized I had digested pSB1C3 with the wrong enzymes. I should have used NgOMIV and Spe in order to cut out BLA. 
</LI></UL><UL><LI>pCerulean_Vp1up_NLS was ligated with yesterdays Darpin fragment and plated on Ampicillin, the pSB1C3_BLA construct will be cloned correctly tomorrow
</LI></UL><H4><SPAN class="mw-headline" id="Cloning_of_pGGTBT7_Affibody_middlelinker_EGFP_His"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of pGGTBT7_Affibody_middlelinker_EGFP_His</B></P></SPAN></H4><P><B>Investigator: Jessica</B><B>Comment: pBAD is not really working, so we test another expression-plasmid</B></P><UL><LI>Vector: name: pGGTBT7_MVenus  <B>P627</B></LI><LI>Insert: name: pBAD_Affibody_middlelinker_EGFP_His <B>596</B>prepared yesterday
</LI><LI>new vector name: pGGTBT7_ZEGFR:1907_middlelinker_EGFP_His <B>P</B></LI><LI>buffer used: 4 
</LI><LI>DNA concentration (vector): 144,1 ng/µl; DNA concentration (insert): 101,4 µg/µl
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume of pBAD  /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">10
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2
</TD></TR><TR><TD>Enzyme XbaI</TD><TD align="right">0,8
</TD></TR><TR><TD>Enzyme AgeI</TD><TD align="right">0,8
</TD></TR><TR><TD>H2O</TD><TD align="right">4,4
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><P>
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 120 Volt
</P><TABLE border="1"><TBODY><TR><TH>Fragment size
</TH><TD>5200 bp
</TD><TD>730 bp
</TD></TR></TBODY></TABLE><P><B>Gel extraction</B>: </P><TABLE border="1"><TBODY><TR><TH>pGGTBT7
</TH><TH>Affibody_middlelinker_EFGP_His
</TH></TR><TR><TH>DNA-concentration [ng/µl]
</TH><TD>20,79
</TD><TD>7,84
</TD></TR></TBODY></TABLE><P><B>T4 Ligation</B>: 
Volume vector: 3,2  µl 
Volume insert: 4,8  µl <B>Transformation</B>: </P><P>BL21 DE3 R2L(Kana+Cm), BL21 DE3(Kana), BL21DE3 R2(Kana+Cm) cells were used
</P><H4><SPAN class="mw-headline" id="Cloning_of_CMV_promotor_and_Affibody_into_pSB1C3_001_VP2.2F3_capins_and_pSB1C3_001_VP2.2F3_HSPG_KO"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of CMV promotor and Affibody into pSB1C3_001_VP2/3_capins and pSB1C3_001_VP2/3_HSPG_KO </B></P></SPAN></H4><P><B>Investigator: Anna</B></P><P style="font-size:13px; color:red;">Comment: Cloning of CMV promotor and Affibody into pSB1C3_001_VP2/3_HSPG_KO was repeated, because it was not clear if the first attempt worked. </P><UL><LI>Vector name: 
</LI></UL><P>1. pSB1C3_001_VP2/3_capins <B>P514</B>: 318,9 ng/µl
2. pSB1C3_001_VP2/3_HSPG_KO <B>P613</B>: 273,6 ng/µl</P><UL><LI>Insert name: 
</LI></UL><P>1. pSB1C3_CMV <B>P146</B>: 155,6 ng/µl
2. pSB1C3_ZEGFR:1907 <B>P248</B>: 146,5 ng/µl</P><UL><LI>buffer used: 4 
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume of P514 /µl</B></TD><TD align="right"><B>volume of P613 /µl</B></TD><TD align="right"><B>volume of P146 /µl</B></TD><TD align="right"><B>volume of P248 /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">3,1 </TD><TD align="right">3,7  </TD><TD align="right">12,8 </TD><TD align="right">13,7
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2,0 </TD><TD align="right"> 2,0  </TD><TD align="right"> 2,0 </TD><TD align="right">2,0
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2,0 </TD><TD align="right">2,0   </TD><TD align="right">2,0 </TD><TD align="right">2,0
</TD></TR><TR><TD>Enzyme I</TD><TD align="right">EcoRI  1 </TD><TD align="right">EcoRI   1 </TD><TD align="right">XbaI  1</TD><TD align="right">SpeI  1
</TD></TR><TR><TD>Enzyme II</TD><TD align="right">NgoMIV 1 </TD><TD align="right">NgoMIV  1 </TD><TD align="right">AgeI  1</TD><TD align="right">EcoRI  1
</TD></TR><TR><TD>H2O</TD><TD align="right">10,9</TD><TD align="right">10,3</TD><TD align="right">1,2</TD><TD align="right">0,3
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 20</TD><TD align="right">20 </TD><TD align="right">20</TD><TD align="right">20
</TD></TR></TBODY></TABLE><P>
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 120 Volt
</P><TABLE border="1"><TBODY><TR><TH>Marker, 4 µl
</TH><TH>Vector 1: pSB1C3_001_VP2/3_capins
</TH><TH>Vector 2: pSB1C3_001_VP2/3_HSPG_KO
</TH><TH>CMV
</TH><TH>Affibody
</TH></TR><TR><TH>Fragment size
</TH><TD>4000 bp
</TD><TD>4000 bp
</TD><TD>650 bp
</TD><TD>170 bp
</TD></TR></TBODY></TABLE><P><B>Gel extraction</B>: 
Was performed following the standard protocol.

DNA-concentrations [ng/µl]:
</P><P>c (pSB1C3_001_VP2/3_capins) = 40.9 
c (pSB1C3_001_VP2/3_HSPG_KO)= 51.94
c (CMV)= 12.67
c (Affibody)= 14.44<B>T4 Ligation</B>: </P><UL><LI>Vector 1:</LI></UL><P>Volume vector:  2,8 µl 
Volume Affibody:  2,0 µl 
Volume CMV:  3,2 µl </P><UL><LI>Vector 2:</LI></UL><P>Volume vector:  2,5 µl 
Volume Affibody:  2,0 µl 
Volume CMV:  3,5 µl </P><P><B>Trafo</B>: 
Was performed following the standard protocol using BL21 cells.
</P><H4><SPAN class="mw-headline" id="Cloning_of_CFP_into_pSB1C3_lITR_phTERT_beta-globin"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of CFP into pSB1C3_lITR_phTERT_beta-globin </B></P></SPAN></H4><P><B>Investigator: Anissa</B></P><UL><LI>Vector name: 
</LI></UL><P>pSB1C3_lITR_phTERT_beta-globin <B>P514</B>: 333,46 ng/µl</P><UL><LI>Insert name: 
</LI></UL><P>pSB1C3_CFP <B>P146</B>: 151,1 ng/µl</P><UL><LI>buffer used: 4 
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume of P51.2 /µl</B></TD><TD align="right"><B>volume of P437 /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">13,2</TD><TD align="right">4,5
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2,0 </TD><TD align="right"> 1,5
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2,0 </TD><TD align="right">1,5
</TD></TR><TR><TD>Enzyme I</TD><TD align="right">XbaI 1 </TD><TD align="right">SpeI 1
</TD></TR><TR><TD>Enzyme II</TD><TD align="right">PstI 1 </TD><TD align="right">PstI 1
</TD></TR><TR><TD>H2O</TD><TD align="right">0,8</TD><TD align="right">5,5
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 20</TD><TD align="right">15
</TD></TR></TBODY></TABLE><P>
0,5 g Agarose, 50 ml TAE (1%), 1,5 µl GELRED , at 115 Volt
</P><TABLE border="1"><TBODY><TR><TH>Marker, 4 µl
</TH><TH>Vector
</TH><TH>Insert
</TH></TR><TR><TH>Fragment size
</TH><TD>3180 bp
</TD><TD>750 bp
</TD></TR></TBODY></TABLE><P><B>Gel extraction</B>: 
Was performed following the standard protocol.

DNA-concentrations [ng/µl]:
</P><P>c (pSB1C3_CFP) = 11,6 
c (pSB1C3_lITR_phTERT_beta-globin)= 24,8 <B>T4 Ligation</B>: </P><UL><LI>Volume vector:  3,2 µl </LI><LI>Volume insert:  4,8 µl </LI></UL><P><B>Trafo</B>: 
Was performed following the standard protocol using BL21 cells.
</P><H4><SPAN class="mw-headline" id="Sequencing_results:_VP1_insertion_and_VP2_fusion:_Fusion_of_VP2.2F3_HSPG-KO"><P style="font-size:15px; background-color:#66bbff;"><B>Sequencing results: VP1 insertion and VP2 fusion: Fusion of VP2/3_HSPG-KO </B></P></SPAN></H4><P><B>Investigator: Hanna</B></P><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Sample gallery</CAPTION><TBODY><TR><TD><DIV class="gallerybox" style="width: 465px;"><DIV class="gallerytext"><P>pCerulean_CFP_MiddleLinker_ VP2/3_HSPG-KO (P617)
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 465px;"><DIV class="gallerytext"><P>pCerulean_Z<SUB>EGFR:1907</SUB>_MiddleLinker_ VP2/3_HSPG-KO (P618)
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 465px;"><DIV class="gallerytext"><P>pCerulean_Z<SUB>EGFR:1907</SUB>_SEG_ VP2/3_HSPG-KO (P619)
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 465px;"><DIV class="gallerytext"><P>pCerulean_Z<SUB>EGFR:1907</SUB>_ShortLinker_ VP2/3_HSPG-KO (P620)
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 465px;"><DIV class="gallerytext"><P>pCerulean_Z<SUB>EGFR:1907</SUB>_LongLinker_ VP2/3_HSPG-KO (P621)
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 465px;"><DIV class="gallerytext"><P>pCerulean_6xHis&lt;/sub&gt;_MiddleLinker_ VP2/3_HSPG-KO (P622)
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 465px;"><DIV class="gallerytext"><P>pCerulean__VP1up_NLS_Z<SUB>EGFR:1907</SUB>_ VP2/3_HSPG-KO (P623)
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 465px;"><DIV class="gallerytext"><P>pCerulean_VP1up_NLS_6xHis_ VP2/3_HSPG-KO (P624)
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 465px;"><DIV class="gallerytext"><P>pCerulean_VP1up_NLS_mVenus_ VP2/3_HSPG-KO (P625)
</P></DIV></DIV></TD></TR></TBODY></TABLE><P><B>Conclusion:</B> Except of pCerulean_Z<SUB>EGFR:1907</SUB>_MiddleLinker_ VP2/3_HSPG-KO (P618), all sequencing results looked well. This means: The VP2/3-HSPG-KO motif was cloned successfully to the VP2-Fusion and VP1-Insertion approaches and are ready to be tested in cell culture via qPCR and ELISA. --&gt; Co-transfection with pSB1C3_001_RC_iRCK_VP1or2-KO_HSPG-KO + pHelper + pSB1C3_GOI. 
In addition to that, clone 2 of pCerulean_Z<SUB>EGFR:1907</SUB>_MiddleLinker_ VP2/3_HSPG-KO will be preped and test digested tomorrow.
</P><H4><SPAN class="mw-headline" id="Cloning_of_HSPG_KO_empty_into_pSB1C3_001_RC_IRCK_P5tataless"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of HSPG_KO_empty into pSB1C3_001_RC_IRCK_P5tataless</B></P></SPAN></H4><P><B>Investigator: Stefan </B></P><P>Vector name:
</P><UL><LI>pSB1C3_001_RC_IRCK_P5tataless cl1 (P628)</LI><LI>pSB1C3_001_RC_IRCK_P5tataless cl2 (P630)</LI><P>Insert name:
</P><LI>VB:pSB1C3_001_KO_empty(P611)</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume of P628  /µl</B></TD><TD align="right"><B>volume of P630  /µl</B></TD><TD align="right"><B>volume of P611 /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">3,5 </TD><TD align="right">3,5</TD><TD align="right">16
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right">2</TD><TD align="right"> 3
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right">2 </TD><TD align="right"> 3
</TD></TR><TR><TD>Enzyme BamHI</TD><TD align="right">1</TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>Enzyme PvuII</TD><TD align="right">1</TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">10,5</TD><TD align="right">10,5</TD><TD align="right">6
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 20</TD><TD align="right"> 20</TD><TD align="right">30
</TD></TR></TBODY></TABLE><P><B>Gel:</B>
for vector:
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 110 Volt
for insert:
1 g Agarose, 50 ml TAE (2%), 3 µl GELRED , at 110 Volt</P><P><B>Gel extraction</B>: 
Was performed according to protocol.
</P><P><B>T4 Ligation</B>: </P><UL><LI>Volume vector P628:  7,11 µl</LI><LI>Volume insert:  0,89 µl</LI></UL><UL><LI>Volume vector P630:  7,33 µl</LI><LI>Volume insert:  0,67 µl</LI></UL><P><B>Transformation</B>: 
Was performed according to standard protocol using BL21 cells.
</P><H4><SPAN class="mw-headline" id="Mass_Midi-Prep_II"><P style="font-size:15px; background-color:#66bbff;">Mass Midi-Prep II </P></SPAN></H4><P><B>Investigator: Chris W. </B></P><P style="font-size:13px; color:#003399;"> Midi-Prep of:</P><P>453_BAP =P631
587_BAP =P632
587_KO_BAP =P633
453_RGD =P634
587_RGD =P635
587_KO_RGD =P636
453_HIS =P637
587_HIS =P638
587_KO_HIS =P639
587_KO_empty =P640
453_Z34C =P641
pSB1C3_001_RC_IRCK_VP1-ko_P5tataless =P642 =B463 
pSB1C3_001_RC_IRCK_VP2-ko_P5tataless =P643 =B470</P><P>
The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
</P><TABLE border="1"><TBODY><TR><TD> plasmid-no. </TD><TD align="right">P631</TD><TD align="right">P632</TD><TD align="right">P633</TD><TD align="right">P634</TD><TD align="right">P635</TD><TD align="right">P636</TD><TD align="right">P637</TD><TD align="right">P638</TD><TD align="right">P639</TD><TD align="right">P640</TD><TD align="right">P641</TD><TD align="right">P642</TD><TD align="right">P643
</TD></TR><TR><TD> concentration (ng/µl)</TD><TD align="right">4758,69 </TD><TD align="right">448,12 </TD><TD align="right">3243,97</TD><TD align="right">2231,44</TD><TD align="right">1401,68</TD><TD align="right">3289,15</TD><TD align="right">3441,22</TD><TD align="right">3058,92</TD><TD align="right">4425,05</TD><TD align="right">2339,06</TD><TD align="right">4377,05</TD><TD align="right">5045,55</TD><TD align="right">5201,73
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="135._labday_30.09.2010"><P style="font-size:17px; background-color:#00dd77;">135. labday 30.09.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="picking_CD_clones"><P style="font-size:15px; background-color:#66bbFF;"><B>picking CD clones</B></P></SPAN></H4><P><B>Investigator: Kira</B>
The agaro plate contained many clones, 4 of them were picked and inoculated into media </P><H4><SPAN class="mw-headline" id="ViralBricks_BAP-ko_and_His-ko_in_VP23"><P style="font-size:15px; background-color:#66bbFF;"><B>ViralBricks BAP-ko and His-ko in VP23</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="color:#66bbff;"><I>Comment</I>: In order to verify the insertion of the ViralBricks BAP-ko and His-ko into the VP23 construct we digested the produced constructs with SalI and BamHI (both HF) and loaded it on a 1%a agarose gel. The constructs were used for creating the final super constructs (yay). Sequencing will verify if we subcloned the ViralBrick truly/seriously into the plasmids.</P><P><B>Digestion of the constructs:</B></P><UL><LI>P660 = pSB1C3_001_VP2/3_HSPG-ko_587_BAP</LI><LI>P661 = pSB1C3_001_VP2/3_HSPG-ko_587_BAP</LI><UL><B>Expected size: 464bp</B></UL><LI>P662 = pSB1C3_001_VP2/3_HSPG-ko_587_His</LI><LI>P663 = pSB1C3_001_VP2/3_HSPG-ko_587_His</LI><UL><B>Expected size: 437bp</B></UL><LI>P525 = pSB1C3_001_VP2/3_capins</LI><UL><B>Expected size: 404bp</B></UL></UL><P><B>Results</B>:

The detected bands correspond to the expected sizes, but we cannot distinguish between the exact sizes because they are too similar. Therefore we sent P660 and P663 for sequencing in order to confirm if the ViralBricks have been successfully subcloned in the pSB1c3_VP2/3 construct.
</P><H4><SPAN class="mw-headline" id="Cloning_of_Darpin_into_pCerulean_Vp1up_NLS_and_pSB1C3_Part_Three"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of Darpin into pCerulean_Vp1up_NLS and pSB1C3 Part Three </B></P></SPAN></H4><P><B>Investigator: Achim </B></P><UL><LI>Clones of the pCerulean_Vp1up_NLS_Darpin construct were picked and will be prepped tonight.
</LI></UL><UL><LI>pSB1C3_001_BLA (P320) and pMA_Darpin (P614) were digested with NgOMIV and SpeI. Bands were cut out accordingly, ligated with T4 ligase and plated on Chloramphenicol.
</LI></UL><H4><SPAN class="mw-headline" id="Test_digestion_of_pCerulean_ZEGFR:1907_MiddleLinker_VP2.2F3_HSPG-KO_clone_2"><P style="font-size:15px; background-color:#66bbff;">Test digestion of pCerulean_Z<SUB>EGFR:1907</SUB>_MiddleLinker_VP2/3_HSPG-KO clone 2</P></SPAN></H4><P><B>Investigator: Hanna </B><B>Comment:</B> Yesterday's sequencing results revealed, that VP2/3_HSPG-KO was successfully cloned to the VP2 fusion and VP1 insertion approaches, except of pCerulean_Z<SUB>EGFR:1907</SUB>_MiddleLinker_VP2/3_HSPG-KO clone 2. The chromatogram showed that fusion worked, but there was a kind of contamination in the sample - already visible in the test digestion. </P><UL><LI> DNA = 2 µL
</LI><LI> BSA (10x) = 1 µL
</LI><LI> Buffer 4 = 1 µl
</LI><LI> EcoRI = 0.5 µL
</LI><LI> PstI = 0.5 µL
</LI><LI> H<SUB>2</SUB>0 = 5 µL
</LI></UL><P>
Incubation time = 45 minutes. 
0.9 % Agarose gel. </P><P> 
Test digestion looked well. Sample was sent for sequencing. </P><H4><SPAN class="mw-headline" id="Cloning_VP2.2F3_Capins_and_VP2.2F3_HSPG-KO_downstream_of_Middle_Linker"><P style="font-size:15px; background-color:#66bbff;">Cloning VP2/3_Capins and VP2/3_HSPG-KO downstream of Middle Linker</P></SPAN></H4><P><B>Investigator: Hanna </B></P><P style="color:#800080;"><B>Comment:</B> Besides the Affibody we also want to test and characterize the DARPin for its binding properties referring to the VP2 fusion and VP1 insertion. Therefore some preliminary cloning steps have to be performed. Today the VP2/3_insCap and VP2/3_HSPG-KO sequence will be cloned downstream of the Middle Linker for the VP2 fusion. </P><P style="font-size:15px; font-weight: bold; color: #800080;">Practical Cloning:</P><UL><LI>Vector: pGA14_MiddleLinker (P301) 
</LI><LI>Insert: pSB1C3_VP2/3_insCap (P525) and pSB1C3_001_VP2/3_HSPG-KO (P613)
</LI></UL><LI>new vector name: pGA14_MiddleLinker_VP2/3_insCap and pGA14_MiddleLinker_VP2/3_HSPG-KO 
</LI><LI>buffer used: 4 ; Restriction-enzymes used: PstI, AgeI/NgoMIV
</LI><P style="font-size:15px; font-weight: bold; color: #800080;">Digestion</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of P301 /µl </TD><TD align="right">volume of P525 /µl </TD><TD align="right">volume of P613 /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 5 </TD><TD align="right"> 7.6 </TD><TD align="right"> 5.5
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right"> 2 </TD><TD align="right"> 2 </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right"> 2 </TD><TD align="right"> 2 </TD><TD align="right"> 2
</TD></TR><TR><TD>PstI </TD><TD align="right"> 1 </TD><TD align="right"> 1 </TD><TD align="right"> 1
</TD></TR><TR><TD>AgeI </TD><TD align="right"> 1  </TD><TD align="right"> - </TD><TD align="right"> -
</TD></TR><TR><TD>NgoMIV </TD><TD align="right"> -  </TD><TD align="right"> 1 </TD><TD align="right"> 1
</TD></TR><TR><TD>McsI </TD><TD align="right"> 0.5  </TD><TD align="right"> 0.5 </TD><TD align="right"> 0.5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right"> 9 </TD><TD align="right"> 6.4 </TD><TD align="right"> 8.5
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 20 </TD><TD align="right"> 20 </TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><UL><LI> Comment: After 30 minutes of digestion I remembered that it would be adventageous to digest the plasmid backbone of the VP2/3 constructs. Therefore 0.5 µL of McsI was added to each sample - fortunately McsI does not have any restriction sites in the pGA14_MiddleLinker to which it was also added by mistake. </LI></UL><UL><LI>Incubation: 20 minutes + 2 hours.
</LI></UL><P style="font-size:15px; font-weight: bold; color: #800080;">Agarose-Gel:</P><P>
0.5 g Agarose, 50 mL TAE (1 %), 3 µL GELRED, at 10 Volt, running time: 45 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (6x)/µl
</TH><TH>Expected size 1
</TH><TH>Expected size 2
</TH><TH>Expected size 3
</TH></TR><TR><TD>P301
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD>~ 2450 bp
</TD></TR><TR><TD>P525
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD>1951 bp
</TD><TD>738 bp
</TD><TD>1324bp
</TD></TR><TR><TD>P613
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD>1951 bp
</TD><TD>738 bp
</TD><TD>1324bp
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker /µl
</TH><TH>Sample P301 /µl
</TH><TH>Sample P525 /µl
</TH><TH>Sample P613 /µl
</TH></TR><TR><TH>Lane
</TH><TD>4.5
</TD><TD>24
</TD><TD>24
</TD><TD>24
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color: #800080;">Gel extraction</P><P>
Gel measurement:
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Sample</B></TD><TD align="left"><B>Weight</B></TD><TD align="left"><B>Concentration</B></TD></TR><TR><TD align="left"> P301
</TD><TD align="left"> 250 mg
</TD><TD align="left"> 38.41
</TD></TR><TR><TD align="left"> P525
</TD><TD align="left"> 270 mg
</TD><TD align="left"> 18.21
</TD></TR><TR><TD align="left"> P613
</TD><TD align="left"> 300 mg
</TD><TD align="left"> 21.55
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color:#800080;">Ligation</P><TABLE border="1"><TBODY><TR><TD><B>Construct</B></TD><TD><B>Vector (µl)</B></TD><TD><B>Insert (µl)</B></TD><TD><B>T4 Ligase Buffer (µl)</B></TD><TD><B>T4 Ligase (µl)</B></TD></TR><TR><TD> pGA14_MiddleLinker_VP2/3_insCap
</TD><TD> 1.32
</TD><TD> 6.68
</TD><TD> 1
</TD><TD> 1
</TD></TR><TR><TD> pGA14_MiddleLinker_VP2/3_HSPG-KO
</TD><TD> 1.52
</TD><TD> 6.48
</TD><TD> 1
</TD><TD> 1
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Miniprep_of_serveral_constructs_5"><P style="font-size:15px; background-color:#66bbFF;"><B>Miniprep of serveral constructs</B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><P><B>Glycerol stocks were prepared:</B></P><UL><LI>B527	pSB1C3_001_CMV_VP2/3_587-KO_Z34C_spacer clone3
</LI><LI>B528	pSB1C3_001_CMV_VP123_587-KO_Z34C clone1
</LI><LI>B529	pSB1C3_001_CMV_VP123_587-KO_Z34C clone2
</LI><LI>B530	pSB1C3_001_CMV_VP123_587-KO_Z34C clone3
</LI><LI>B531	pSB1C3_oo1_CMV clone1
</LI><LI>B532	pSB1C3_oo1_CMV clone2
</LI><LI>B533	pSB1C3_001_CMV_ZEGFR:1907_VP2/3_HSPG-KO clone1
</LI><LI>B534	pSB1C3_001_CMV_ZEGFR:1907_VP2/3_HSPG-KO clone2
</LI><LI>B535	pSB1C3_001_VP2/3_ins_HSPG-KO_CMV_ZEGFR:1907 clone 1
</LI><LI>B536	pSB1C3_001_VP2/3_ins_HSPG-KO_CMV_ZEGFR:1907 clone 2
</LI><LI>B537	pBS1C3_001_VP2/3_587_BAP_HSPG-KO clone1 
</LI><LI>B538	pBS1C3_001_VP2/3_587_BAP_HSPG-KO clone2
</LI><LI>B539	pBS1C3_VP2/3_capins_His clone1 
</LI><LI>B540	pBS1C3_VP2/3_capins_His clone2
</LI><LI>B541	pSB1C3_ZEGFR:1907 _middlelinker_EGFP_His clone1
</LI><LI>B542	pSB1C3_ZEGFR:1907 _middlelinker_EGFP_His clone2
</LI></UL><P><B>Mini-Prep was performed according to the standard protocol</B></P><UL><LI>P644		pSB1C3_001_RC_IRCK_VP1-ko_HSPG-ko_P5tataless cl1 c= 151,0 ng/µl
</LI><LI>P645		pSB1C3_001_RC_IRCK_VP1-ko_HSPG-ko_P5tataless cl2 c= 115,8 ng/µl
</LI><LI>P646		pSB1C3_001_RC_IRCK_VP2-ko_HSPG-ko_P5tataless cl1 c= 173,1 ng/µl
</LI><LI>P647		pSB1C3_001_RC_IRCK_VP2-ko_HSPG-ko_P5tataless cl2 c= 119,9 ng/µl
</LI><LI>P648		pSB1C3_001_CMV_VP123_587-KO_Z34C_spacer clone1 c= 226,8 ng/µl
</LI><LI>P649		pSB1C3_001_CMV_VP123_587-KO_Z34C_spacer clone2 c= 290,3 ng/µl
</LI><LI>P650		pSB1C3_001_CMV_VP123_587-KO_Z34C_spacer clone3 c= 284,2 ng/µl
</LI><LI>P651		pSB1C3_001_CMV_VP123_587-KO_Z34C clone1 c= 283,8 ng/µl
</LI><LI>P652		pSB1C3_001_CMV_VP123_587-KO_Z34C clone2 c= 290,5 ng/µl
</LI><LI>P653		pSB1C3_001_CMV_VP123_587-KO_Z34C clone3 c= 251,7 ng/µl
</LI><LI>P654		pSB1C3_oo1_CMV clone1 c= 199,5 ng/µl
</LI><LI>P655		pSB1C3_oo1_CMV clone2 c= 192,7 ng/µl
</LI><LI>P656		pSB1C3_001_CMV_ZEGFR:1907_VP2/3_HSPG-KO clone1 c= 200,3 ng/µl
</LI><LI>P657		pSB1C3_001_CMV_ZEGFR:1907_VP2/3_HSPG-KO clone2 c= 325,0 ng/µl
</LI><LI>P658		pSB1C3_001_CMV_ZEGFR:1907_VP2/3_HSPG-KO clone3 c= 222,9 ng/µl
</LI><LI>P659		pSB1C3_001_CMV_ZEGFR:1907_VP2/3_HSPG-KO clone4 c= 201,3 ng/µl
</LI><LI>P660		pBS1C3_001_VP2/3_587_BAP_HSPG-KO clone1  c= 267,1 ng/µl
</LI><LI>P661		pBS1C3_001_VP2/3_587_BAP_HSPG-KO clone2 c= 193,1 ng/µl
</LI><LI>P662		pBS1C3_VP2/3_capins_His clone1 c= 209,1 ng/µl
</LI><LI>P663		pBS1C3_VP2/3_capins_His clone2 c= 252,5 ng/µl
</LI><LI>P664		pSB1C3_ZEGFR:1907 _middlelinker_EGFP_His clone1 c= 185,7 ng/µl
</LI><LI>P665		pSB1C3_ZEGFR:1907 _middlelinker_EGFP_His clone2 c= 192,0 ng/µl
</LI><LI>P666		pSB1C3_CFP c= 212,7  ng/µl
</LI></UL><H4><SPAN class="mw-headline" id="Test_digestion_of_pSB1C3_ZEGFR:1907_middlelinker_EGFP_His_and_mCherry_iGEM_2008"><P style="font-size:15px; background-color:#66bbff;">Test digestion of pSB1C3_ZEGFR:1907 _middlelinker_EGFP_His and mCherry iGEM 2008</P></SPAN></H4><P><B>Investigator: Jessica </B></P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of P664 pSB1C3_ZEGFR:1907 _middlelinker_EGFP_His /µl </TD><TD align="right">volume of P665 pSB1C3_ZEGFR:1907 _middlelinker_EGFP_His /µl </TD><TD align="right">volume of P626 mCherry /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 1,5 </TD><TD align="right"> 1,5 </TD><TD align="right"> 1,5
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right"> 1 </TD><TD align="right"> 1 </TD><TD align="right"> 1
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right"> 1 </TD><TD align="right"> 1 </TD><TD align="right"> 1
</TD></TR><TR><TD>XbaI </TD><TD align="right"> 0,4 </TD><TD align="right"> 0,4 </TD><TD align="right"> 0,4
</TD></TR><TR><TD>AgeI </TD><TD align="right"> 0,4  </TD><TD align="right"> 0,4 </TD><TD align="right"> 0,4
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right"> 5,7 </TD><TD align="right"> 5,7 </TD><TD align="right"> 5,7
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 10 </TD><TD align="right"> 10 </TD><TD align="right"> 10
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Sample gallery</CAPTION></TABLE><H4><SPAN class="mw-headline" id="Test_digestion_of_pSB1C3_001_CMV_affibody_HSPG_KO_and_pSB1C3_001_CMV_VP123_587_KO_Z34C"><P style="font-size:15px; background-color:#66bbff;">Test digestion of pSB1C3_001_CMV_affibody_HSPG_KO and  pSB1C3_001_CMV_VP123_587_KO_Z34C</P></SPAN></H4><P><B>Investigator: Anna </B></P><P style="font-size:13px; color:red;">Comment: 587_KO_Z34C and 587_KO_Z34C_Spacer were cloned into pSB1C3_001_CMV_VP123 as already done with the other 453 and 587 constructs (see test digestion from labday 28.09.) The test digestion looks well, nevertheless one sample out of these constructs should be sent for sequencing.  </P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of pSB1C3_001_CMV_affibody_HSPG_KO /µl </TD><TD align="right">volume of pSB1C3_001_CMV_VP123_587_KO_Z34C /µl </TD><TD align="right">volume of pSB1C3_001_CMV_VP123_587_KO_Z34C_Spacer /µl
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 3 </TD><TD align="right"> 3 </TD><TD align="right"> 3
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right"> 1,5 </TD><TD align="right"> 1,5 </TD><TD align="right"> 1,5
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right"> 1,5 </TD><TD align="right"> 1,5 </TD><TD align="right"> 1,5
</TD></TR><TR><TD>EnzymeI </TD><TD align="right">SspI 0,3 </TD><TD align="right">SalI 0,3 </TD><TD align="right">SalI 0,3
</TD></TR><TR><TD>EnzymeII </TD><TD align="right">EcoRI 0,3  </TD><TD align="right">PvuII 0,3 </TD><TD align="right">PvuII 0,3
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right"> 4,8 </TD><TD align="right"> 4,8 </TD><TD align="right"> 4,8
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 15 </TD><TD align="right"> 15 </TD><TD align="right"> 15
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD> Sample   </TD><TD align="right">1 </TD><TD align="right">2</TD><TD align="right">3</TD><TD align="right">4</TD><TD align="right">5</TD><TD align="right">6</TD><TD align="right">7
</TD></TR><TR><TD> Construct  </TD><TD align="right"> 587_KO_Z34C </TD><TD align="right"> 587_KO_Z34C </TD><TD align="right"> 587_KO_Z34C</TD><TD align="right"> 587_KO_Z34C_Spacer</TD><TD align="right"> 587_KO_Z34C_Spacer</TD><TD align="right"> 587_KO_Z34C_Spacer</TD><TD align="right"> p580
</TD></TR><TR><TD> Clone  </TD><TD align="right"> 1 </TD><TD align="right"> 2 </TD><TD align="right"> 3</TD><TD align="right"> 1</TD><TD align="right"> 2</TD><TD align="right"> 3</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD> Sample   </TD><TD align="right">8</TD><TD align="right">9</TD><TD align="right">10</TD><TD align="right">11</TD><TD align="right">12
</TD></TR><TR><TD> Construct </TD><TD align="right"> p613</TD><TD align="right"> CMV_EGFR:1907</TD><TD align="right"> CMV_EGFR:1907</TD><TD align="right"> CMV_EGFR:1907</TD><TD align="right"> CMV_EGFR:1907
</TD></TR><TR><TD> Clone  </TD><TD align="right"> 1 </TD><TD align="right"> 2</TD><TD align="right"> 3</TD><TD align="right"> 4
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Midiprep_of_pSB1C3_001_CMV_VP1.2F2.2F3_capins_587bla_and_pSB1C3_001_RC_IRCK_P5tataless_clone1"><P style="font-size:15px; background-color:#66bbff;">Midiprep of pSB1C3_001_CMV_VP1/2/3_capins_587bla and pSB1C3_001_RC_IRCK_P5tataless clone1</P></SPAN></H4><P><B>Investigators: Chris W. </B></P><P style="font-size:13px; color:#003399;"> Midi-Preps of:
pSB1C3_001_CMV_VP1/2/3_capins_587bla =P667 
pSB1C3_001_RC_IRCK_P5tataless clone1 =P668</P><P>
The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
</P><TABLE border="1"><TBODY><TR><TD> plasmid-no. </TD><TD align="right">P667</TD><TD align="right">P668
</TD></TR><TR><TD> concentration (ng/µl)</TD><TD align="right">1502,3 </TD><TD align="right">991,6
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Cloning_of_VP2.2F3_BAP_HSPG-KO_and_VP2.2F3_His_HSPG-KO_into_pCerulean_Zegfr:1907_Middlelinker_and_VP2.2F3_His_HSPG-KO_into_pCerulean_VP1up_NLS_mVenus"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of VP2/3_BAP_HSPG-KO and VP2/3_His_HSPG-KO into pCerulean_Zegfr:1907_Middlelinker and VP2/3_His_HSPG-KO into pCerulean_VP1up_NLS_mVenus</B></P></SPAN></H4><P><B>Investigator: Stefan </B></P><P>Vector name:
</P><UL><LI>pCerulean_Zegfr:1907_Middlelinker (P408)</LI><LI>pCerulean_VP1up_NLS_mVenus (P426)</LI></UL><P>Insert name:
</P><UL><LI>pBS1C3_001_VP2/3_587_BAP_HSPG-KO clone1 (P660)</LI><LI>pBS1C3_001_VP2/3_587_BAP_HSPG-KO clone2 (P661)</LI><LI>pBS1C3_001_VP2/3_587_His_HSPG-KO clone1 (P662)</LI><LI>pBS1C3_001_VP2/3_587_His_HSPG-KO clone2 (P663)</LI></UL><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume for inserts (P660 - P663)  /µl</B></TD><TD align="right"><B>volume of P408  /µl</B></TD><TD align="right"><B>volume of P426 /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">10 </TD><TD align="right">3</TD><TD align="right">3
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">3 </TD><TD align="right">2</TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">3 </TD><TD align="right">2 </TD><TD align="right"> 2
</TD></TR><TR><TD>Enzyme NgoMIV</TD><TD align="right">1</TD><TD align="right">- </TD><TD align="right">-
</TD></TR><TR><TD>Enzyme PstI</TD><TD align="right">1</TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>Enzyme MscI</TD><TD align="right">1</TD><TD align="right">- </TD><TD align="right">-
</TD></TR><TR><TD>Enzyme AgeI</TD><TD align="right">-</TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">11</TD><TD align="right">11</TD><TD align="right">11
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 30</TD><TD align="right"> 20</TD><TD align="right">20
</TD></TR></TBODY></TABLE><P><B>Gel:</B>
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 115 Volt</P><P><B>Gel extraction</B>: 
Was performed according to protocol.
</P><P><B>T4 Ligation</B>: </P><TABLE border="1"><TBODY><TR><TD>ligation name </TD><TD align="right">408 + 660</TD><TD align="right">408 + 661</TD><TD align="right">408 + 662</TD><TD align="right">408 + 663</TD><TD align="right">426 + 662</TD><TD align="right">426 + 663
</TD></TR><TR><TD>volume of vector </TD><TD align="right">3,4 </TD><TD align="right"> 3,05</TD><TD align="right"> 3,35</TD><TD align="right"> 2,56</TD><TD align="right"> 2,74</TD><TD align="right"> 3,13
</TD></TR><TR><TD>volume of insert</TD><TD align="right">4,6 </TD><TD align="right">4,95</TD><TD align="right">4,65</TD><TD align="right">5,44</TD><TD align="right">5,26</TD><TD align="right">4,87
</TD></TR></TBODY></TABLE><P><B>Transformation</B>: 
Was performed according to standard protocol using BL21 cells.
</P><CENTER><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/October" class="external text" rel="nofollow"><B>=&gt; Go to Labjournal October part 1 (labday 136 - 145 )</B></A></CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/September2">http://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/September2</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg_Bioware/NoteBook/Labjournal/September2" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg_Bioware/NoteBook/Labjournal/September2" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/September2&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/September2&amp;oldid=197524" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>