<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Bioware_NoteBook_Labjournal_Cellculture"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg_Bioware/NoteBook/Labjournal/Cellculture">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg_Bioware/NoteBook/Labjournal/Cellculture&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/Cellculture&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/Cellculture&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg_Bioware/NoteBook/Labjournal/Cellculture" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg Bioware/NoteBook/Labjournal/Cellculture</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="menu"><UL id="menu"><LI><DL><DT class="menu_button_left"><A href="https://2010.igem.org/Team:Freiburg_Bioware">Home</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Team_2010">Team</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Team_2010">Team 2010</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Photo_Gallery">Photo Gallery</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Cuckoo_Clock">Cuckoo Clock Competition</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Collaboration">Collaboration</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Example_Page">Example Page</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project">Project</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project">Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#highlights">Highlights</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#highlights">Results</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#modularisation">   Modularisation</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#targeting">   Targeting</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#arming">   Arming</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#superconstruct">   All-in-One</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Methods">Methods</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Virus_Construction_Kit">Virus Construction Kit Manual</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks">BioBricks</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks">Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/capsid_coding">Capsid Coding Parts</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/vector_plasmid">Vector Plasmid Parts</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/miscellaneous">Miscellaneous Parts</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A class="current" href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook">Notebook</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal">Labjournal</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Materials">Materials</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Methods">Methods</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Experiments">Experiments</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Meetings">Meetings</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Statistics">Statistics</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Sponsors">Sponsors</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Safety">Safety</A></DT></DL></LI><LI><DL><DT class="menu_button_right"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling">Modeling</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling">Modeling Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Virus_Production">Virus Production</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Virus_Infection">Virus Infection</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Structure_Modeling">Structure Modeling</A></DD></DL></LI></UL></DIV><DIV class="pageContent"><P><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook" class="external text" rel="nofollow">=&gt; Back to Notebook overview</A></P><DIV class="box_right"><LEFT><U1>NoteBook Navigator</U1></LEFT><UL><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/March">March (labday 1)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/April">April (labday 2 - 5)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/May">May (labday 6 - 17)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/June">June (labday 18 - 45)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/July">July (labday 46 - 75)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/August">August part 1 (labday 76 - 92)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/August2">August part 2 (labday 93 - 106)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/September">September part 1 (labday 107 - 123)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/September2">September part 2 (labday 124 - 135)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/October">October part 1 (labday 136 - 149 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/October2">October part 2 (labday 150 - 166 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/November">November  (labday 167 - 170 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/Cellculture">Cellculture</A></LI></UL></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Cellculture"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Cellculture</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Date_of_first_action.2C_title_of_the_experiment"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Date of first action, title of the experiment</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#14.6_Seeding_AAV293_for_transfection"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">14.6 Seeding AAV293 for transfection</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#19.6_Seeding_AAV293_for_second_transfection_according_to_standard_protocol"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">19.6 Seeding AAV293 for second transfection according to standard protocol</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#30.6_Examination_of_different_Transduction_methods"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">30.6 Examination of different Transduction methods</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#3.7_Seeding_AAV293_for_transfection_with_different_plasmid_amounts_and_harvesting_methods"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">3.7 Seeding AAV293 for transfection with different plasmid amounts and harvesting methods</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#4.7_Transfection_with_different_amounts_of_plasmids"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">4.7 Transfection with different amounts of plasmids</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#14.7_Seeding_AAV293_for_TKGMK_constructs"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">14.7 Seeding AAV293 for TKGMK constructs</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#7.8_Seeding_AAV293_for_TKGMK_and_mVenus_stocks"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">7.8 Seeding AAV293 for TKGMK and mVenus stocks</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#14.8_Seeding_different_amounts_of_AAV293_for_transfection"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">14.8 Seeding different amounts of AAV293 for transfection</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#17.8_transduction_with_viral_stocks_20_.C2.B5g_and_40_.C2.B5g_GOI-plasmid"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">17.8 transduction with viral stocks 20 µg and 40 µg GOI-plasmid</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#16.8_Seeding_AAV293_for_transfection_with_new_R.2FC"><SPAN class="tocnumber">1.11</SPAN><SPAN class="toctext">16.8 Seeding AAV293 for transfection with new R/C</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#19.8_Seeding_AAv293_for_transfection_with_different_HBS_buffers"><SPAN class="tocnumber">1.12</SPAN><SPAN class="toctext">19.8 Seeding AAv293 for transfection with different HBS buffers</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#22.8:_Transfection_of_AAV293_with_different_amounts_of_cells.2C_and_two_plasmid_concentrations"><SPAN class="tocnumber">1.13</SPAN><SPAN class="toctext">22.8: Transfection of AAV293 with different amounts of cells, and two plasmid concentrations</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#30.8_Seeding_AAV293_for_Transfection_of_10_cm2_plates"><SPAN class="tocnumber">1.14</SPAN><SPAN class="toctext">30.8 Seeding AAV293 for Transfection of 10 cm<SUP>2</SUP> plates</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#4.9_Seeding_cells_for_checking_constructs_.28GOIs.29_with.2Fwithout_HGH_and_beta_globin_and_reassembled_as_control"><SPAN class="tocnumber">1.15</SPAN><SPAN class="toctext">4.9 Seeding cells for checking constructs (GOIs) with/without HGH and beta globin and reassembled as control</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#6.9_checking_five_ganciclovir_concentrations_for_two_day_incubation_approach_optimal_medication"><SPAN class="tocnumber">1.16</SPAN><SPAN class="toctext">6.9 checking five ganciclovir concentrations for two day incubation approach optimal medication</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#15.9_Seeding_AAV293_for_testing_our_Affibody_and_other_VP-mVenus_constructs"><SPAN class="tocnumber">1.17</SPAN><SPAN class="toctext">15.9 Seeding AAV293 for testing our Affibody and other VP-mVenus constructs</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#19.9_Seeding_AAV293_for_checking_two_different_Rep.2FCap_Constructs"><SPAN class="tocnumber">1.18</SPAN><SPAN class="toctext">19.9 Seeding AAV293 for checking two different Rep/Cap Constructs</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#22.9_Seeding_AAV293_for_transfection_with_different_mVenus_constructs_for_checking_specific_promoter_activity"><SPAN class="tocnumber">1.19</SPAN><SPAN class="toctext">22.9 Seeding AAV293 for transfection with different mVenus constructs for checking specific promoter activity</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#22.9_Testing_of_Hannas_6_VP-constructs_with_different_compositions_.2810.25.2C_25.25.2C_50.25_and_75.25_fractions.29_with_a_VP2-knockout_construct"><SPAN class="tocnumber">1.20</SPAN><SPAN class="toctext">22.9 Testing of Hannas 6 VP-constructs with different compositions (10%, 25%, 50% and 75% fractions) with a VP2-knockout construct</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#22.9_Seeding_cells_for_testing_pTERT_mVenus_in_R.2FC_P326_and_P431"><SPAN class="tocnumber">1.21</SPAN><SPAN class="toctext">22.9 Seeding cells for testing pTERT_mVenus in R/C P326 and P431</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#29.9_Checking_location_of_VP_proteins_via_fluorescence_microscopy_and_checking_fractions_of_centrifugation_steps_via_fluorescence_spectrometer"><SPAN class="tocnumber">1.22</SPAN><SPAN class="toctext">29.9 Checking location of VP proteins via fluorescence microscopy and checking fractions of centrifugation steps via fluorescence spectrometer</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#30.9_Production_of_new_Standard_mVenus_Vector"><SPAN class="tocnumber">1.23</SPAN><SPAN class="toctext">30.9 Production of new Standard mVenus Vector</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#30.9_Seeding_AAV293_for_production_of_standard_TKGMK_vector"><SPAN class="tocnumber">1.24</SPAN><SPAN class="toctext">30.9 Seeding AAV293 for production of standard TKGMK vector</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#30.9_Seeding_AAV293_for_production_of_vectors_with_mVenus_missing_beta_globin"><SPAN class="tocnumber">1.25</SPAN><SPAN class="toctext">30.9 Seeding AAV293 for production of vectors with mVenus missing beta globin</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#30.9_Seeding_AAV293_for_production_of_vectors_with_mVenus_missing_HGH"><SPAN class="tocnumber">1.26</SPAN><SPAN class="toctext">30.9 Seeding AAV293 for production of vectors with mVenus missing HGH</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#30.9_Seeding_AAV293_for_testing_RC_with_HSPG_knockout_for_functionality"><SPAN class="tocnumber">1.27</SPAN><SPAN class="toctext">30.9 Seeding AAV293 for testing RC with HSPG knockout for functionality</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#30.9_Seeding_AAV293_for_testing_RC_WITHOUT_HSPG_knockout_for_functionality"><SPAN class="tocnumber">1.28</SPAN><SPAN class="toctext">30.9 Seeding AAV293 for testing RC WITHOUT HSPG knockout for functionality</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#12.10_Seeding_AAV293_cells_for_transfection_with_VP-BAP"><SPAN class="tocnumber">1.29</SPAN><SPAN class="toctext">12.10 Seeding AAV293 cells for transfection with VP-BAP</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#12.10_Seeding_AAV293_cells_for_transfection_with_mVenus_as_gene_of_interest_and_pCerulean_VP1up_NLS_mVenus_VP2.2F3_HSPG-KO"><SPAN class="tocnumber">1.30</SPAN><SPAN class="toctext">12.10 Seeding AAV293 cells for transfection with mVenus as gene of interest and pCerulean_VP1up_NLS_mVenus_VP2/3_HSPG-KO</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#12.10_Seeding_AAV293_cells_for_transfection_with_mVenus_as_GOI_and_CFP_Middlelinker_VP2.2F3_HSPG-KO"><SPAN class="tocnumber">1.31</SPAN><SPAN class="toctext">12.10 Seeding AAV293 cells for transfection with mVenus as GOI and CFP_Middlelinker_VP2/3_HSPG-KO</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#17.10_Production_of_AAV2_with_587-His"><SPAN class="tocnumber">1.32</SPAN><SPAN class="toctext">17.10 Production of AAV2 with 587-His</SPAN></A></LI><LI class="toclevel-2 tocsection-34"><A href="#18.10_Seeding_AAV293_cells_for_transfection_Superconstructs"><SPAN class="tocnumber">1.33</SPAN><SPAN class="toctext">18.10 Seeding AAV293 cells for transfection Superconstructs</SPAN></A></LI><LI class="toclevel-2 tocsection-35"><A href="#18.10_Seeding_AAV293_cells_for_transfection_with_Cytosindeaminase"><SPAN class="tocnumber">1.34</SPAN><SPAN class="toctext">18.10 Seeding AAV293 cells for transfection with Cytosindeaminase</SPAN></A></LI><LI class="toclevel-2 tocsection-36"><A href="#22.10.2010_Transduction_with_old_mVenus_Stocks_for_taking_pictures"><SPAN class="tocnumber">1.35</SPAN><SPAN class="toctext">22.10.2010 Transduction with old mVenus Stocks for taking pictures</SPAN></A></LI><LI class="toclevel-2 tocsection-37"><A href="#23.10_Testing_the_bystander_effect_of_the_cytosine_deaminase"><SPAN class="tocnumber">1.36</SPAN><SPAN class="toctext">23.10 Testing the bystander effect of the cytosine deaminase</SPAN></A></LI><LI class="toclevel-2 tocsection-38"><A href="#25.10.2010_Transduction_with_several_constructs_for_Flow_Cytometry"><SPAN class="tocnumber">1.37</SPAN><SPAN class="toctext">25.10.2010 Transduction with several constructs for Flow Cytometry</SPAN></A></LI><LI class="toclevel-2 tocsection-39"><A href="#BLANK_EXPERIMENT"><SPAN class="tocnumber">1.38</SPAN><SPAN class="toctext">BLANK EXPERIMENT</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Citations"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Citations</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Cellculture">Cellculture</SPAN></H2><P>This subpage is for cellculture, the structure of each experiment is following:
</P><H3><SPAN class="mw-headline" id="Date_of_first_action.2C_title_of_the_experiment"><P style="font-size:17px; background-color:#00dd77;">Date of first action, title of the experiment</P></SPAN></H3><P><B>The motivation</B>
This short introduction provides an inside into our thoughts for this experiment.
<B>The plan</B>
In this section, the design of the experiment is explained.
<B>The results</B>
Interpretation and discussion of our results.
</P><H3><SPAN class="mw-headline" id="14.6_Seeding_AAV293_for_transfection"><P style="font-size:17px; background-color:#00dd77;">14.6 Seeding AAV293 for transfection</P></SPAN></H3><P><B>The motivation</B>
We want to create viral stocks with the stratagene R/C, the gene of interest is mVenus. This is a qualitative control.
<B>The plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">14.6</TH><TH width="100">15.6</TH><TH width="100">16.6</TH><TH width="100">17.6</TH><TH width="100">18.6</TH><TH width="100">19.6</TH><TH width="100">20.6</TH><TH width="100">21.6</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest + Transduction</TD><TD>Fluorescence microscopy</TD></TR></TBODY></TABLE><P>AAV 293 and HT1080 Cells have been splitted and plated out on 10 cm dishes for transfektion. After harvesting the cells according to the standard protocol the cell-pellets were resuspendiated in 15ml of DYT medium.
2,5µl of this cell suspension have been mixed with 47.5µl of trypan blue and the cells were counted by using the Neubauer-Meteringchamber.
</P><P>We counted 12,5x 10^6 cells/ml for the AAV293 cell line and 10x10^6 cells /ml for the HT1080 cell line.
</P><P>1ml and 1,5 ml of the AAV 293 cells have been seeded on four dishes with 250µl of the cell suspension.
Note that that the date is wrong on the plates and the flasks! Cells have been already plated out on the 14th. of June.
</P><P>We also seated two flasks one for each cell line. Therefor we used 1ml of the HT1080 cell suspension and 2ml of the the AAV293 cells.

1) 
</P><UL><LI> pAAV_iGEM_mVenus_YFP (glycerol stock B34, P39) P39 has the correct sequence (confirmed)
</LI><LI> pHelper
</LI><LI> pAAV_RC
</LI></UL><P>2) 
</P><UL><LI> pAAV_iGEM_mVenus_YFP (glycerol stock B33, P38) P38 (we dont know if P39 has the correct sequence!)was used because the amount of P39 <I><A href="/wiki/index.php?title=Was_not_sufficient_enough&amp;action=edit&amp;redlink=1" class="new" title="Was not sufficient enough (page does not exist)">was not sufficient enough</A></I> for four Transfections!
</LI><LI> pHelper
</LI><LI> pAAV_RC
</LI></UL><P>Plasmid concentrations: 
pAAV_RC: 1 µg/µl 
pHelper: 280 ng/µl 
pAAV_iGEM_mVenus_YFP, P39: 180,83 ng/µl 
pAAV_iGEM_mVenus_YFP, P38: 179,75 ng/µl </P><P>Cellculture clone 1 and 2 were transfected with P39. Cellculture clone 3 and 4 were transfected with P38. </P><P>
We could only pipet 3,3 µg (instead of 10 µg) from each of the 3 plasmids into the 15 ml falcons due to insufficient amount of plasmid.
</P><P>Adrian tried to examine the precipation of the CaCl2+ DNA Clusters. We have to optimize the pH of our 2xHBS at the moment it is 11.12.

The viral stocks were harvested according to standard protocol.
<B>The results</B></P><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Results of transduction, ~ 1% of the HT1080 showed expression of mVenus.</CAPTION><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>Bright light image 1
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>Fluoresce microscopy image 1
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>Bright light image 1
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>Fluoresce microscopy image 1
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>
According to the stratagene protocol, transduction efficencies about ~80% are achievable. We performed every step under guidance, so probably something is wrong with the buffers, cells or even the standard protocol. The next step is to repeat this transfection.
</P><H3><SPAN class="mw-headline" id="19.6_Seeding_AAV293_for_second_transfection_according_to_standard_protocol"><P style="font-size:17px; background-color:#00dd77;">19.6 Seeding AAV293 for second transfection according to standard protocol</P></SPAN></H3><P><B>The motivation</B>
We want to repeat the last transfection and obtain a vsibly higher GOI (mVenus) expression.
<B>The plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">19.6</TH><TH width="100">20.6</TH><TH width="100">20.6</TH><TH width="100">21.6</TH><TH width="100">22.6</TH><TH width="100">23.6</TH><TH width="100">24.6</TH><TH width="100">25.6</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest + Transduction</TD><TD>Fluorescence microscopy</TD></TR></TBODY></TABLE><P>Six transfections were performed according to the standart protocol:
Three 10 cm cellcluture dishes with 293 cells were transfected with 10 µg DNA (3,33 µg each plasmid) and the other three 10 cm cellculture dishes with 30 µg DNA (10 µg DNA each plasmid).
</P><P>Plasmids:
</P><UL><LI> pAAV_iGEM-MCS_mVenus (P41), 507 ng/µl
</LI><LI> pAAV_RC , 1000 ng/µl
</LI><LI> pHelper , 280 ng/µl
</LI></UL><UL><LI> Transduction of 2x6 wells was successfully done (14.25)
</LI><LI> one six well was transduced with 10µg the other with 30µg DNA amount </LI></UL><P><B>The results</B></P><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Results of transduction nr.2</CAPTION><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>10µg_unverd_2_(c1).JPG 48h post transduction
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>10µg_unverd_1_(c1).JPG 48h post transduction
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>30µg_unverd_2_(c1).JPG 48h post transduction
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>30µg_unverd_(c1).JPG 48h post transduction
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>30µg_unverd_4_(c1).JPG 48h post transduction
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>30µg unverd 3 (c1).JPG 48h post transduction
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>30µg_unverd_6_(c1).JPG 48h post transduction
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>30µg_unverd_5_(c1).JPG 48h post transduction, we changed the mVenus-yellow to pink, for the girls :)
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>
The Transduction efficency is still quite low! We have to optimize the stratagene protocol!
</P><H3><SPAN class="mw-headline" id="30.6_Examination_of_different_Transduction_methods"><P style="font-size:17px; background-color:#00dd77;">30.6 Examination of different Transduction methods</P></SPAN></H3><P><B>The motivation</B>
We want to check whether the amount of our produced AAV2s has an effect on the transduction rate.
Instead of 500µl, 1000µl of the AAV-stock were pipetted into each well.
<B>The plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">30.6</TH><TH width="100">1.7</TH><TH width="100">2.7</TH><TH width="100">3.7</TH><TH width="100">4.7</TH><TH width="100">5.7</TH><TH width="100">6.7</TH><TH width="100">7.7</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest + Transduction</TD><TD>Fluorescence microscopy</TD></TR></TBODY></TABLE><P>1. Plate I(A is up)
</P><TABLE border="1"><TBODY><TR><TD> 3x10^5 cells  </TD><TD align="right">3x10^5 cells </TD><TD align="right">3x10^5 cells
</TD></TR><TR><TD> 3x10^5 cells  </TD><TD align="right"> 3x10^5 cells</TD><TD align="right"> 3x10^5 cells
</TD></TR></TBODY></TABLE><P>Transduction:
</P><TABLE border="1"><TBODY><TR><TD> 1000 µl of Viral Stock  DROPPED </TD><TD align="right">1000 µl of Viral Stock gently resuspended</TD><TD align="right">1500 µl of Viral Stock rough resuspending
</TD></TR><TR><TD> 1000 µl of Viral Stock  DROPPED </TD><TD align="right"> 1000 µl of Viral Stock gently resuspended</TD><TD align="right"> no Transduction
</TD></TR></TBODY></TABLE><P><B>The results</B>
All cells died probably because we forgot to wash the cells with PBS before adding fresh medium. It was not possible to take pictures because the dead cells were all clumped.
Looks like we should not transduce 3x10^5 cells with 1000 µl (or more) of our viral solution because almost all cells died in all approaches and the medium indicator was yellow.
So we can not say if resuspendig the cells with the viral soultion is a useful alteration of the standard protocol. 
</P><H3><SPAN class="mw-headline" id="3.7_Seeding_AAV293_for_transfection_with_different_plasmid_amounts_and_harvesting_methods"><P style="font-size:17px; background-color:#00dd77;">3.7 Seeding AAV293 for transfection with different plasmid amounts and harvesting methods</P></SPAN></H3><P><B>The motivation</B>
We want to check if the amount of plasmids is the reason for the low transduction efficiency. Unfortunately the flow cytometry (FACS) is not available at the moment so the results can only be checked via fuorescence microscopy.
<B>The plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">3.7</TH><TH width="100">4.7</TH><TH width="100">5.7</TH><TH width="100">6.7</TH><TH width="100">7.7</TH><TH width="100">8.7</TH><TH width="100">9.7</TH><TH width="100">10.7</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest + Transduction</TD><TD>Fluorescence microscopy</TD></TR></TBODY></TABLE><P>Transfection with P41 (pAAV_iGEM-MCS_mVenus) was performed according standart protocol. 
Two transfections where carried out with 10µg (3,33 µg each plasmid) DNA and the other two with 20µg DNA (6,66 µg each plasmid).
We want to investigate which of the following approaches yields a better transduction efficiency.
</P><UL><LI>Half of the transfected cells were exposed 3 cycles of thawing and freezing and then centrifuged (15 ml falcon, 2100G). The supernatant was transfered into a new 15 ml falcon and used for transduction. 
</LI><LI>The other half was centrifuged at 300 G for 5min . The supernatant was transfered into another 15 ml falcon and the pellet was resuspended with 5 ml of DMEM. The content of these two falcons was also exposed to 3 cycles of thawing and freezing and then used for transduction.
</LI></UL><UL><LI> We have got 2 10 cm cellculture dishes with 10µg and 2 dishes with 20µg DNA used for transfection. 
</LI></UL><P>Approach with standart protocol (one dish with 10 and the other with 20µg Plasmids)(2100 G and 3 cycles of freezing and thawing)
</P><P>Deviations from the standart protocol:
</P><UL><LI> The cells were centrifuged at 2100 G instead of 10.000 G
</LI><LI> The viruses were harvested after 44 hours
</LI><LI> 3 cycles of freezing and thawing
</LI></UL><UL><LI> used plasmids: 10µg, 20µg GOI-plasmid (mVenus)
</LI><LI> we have 10µg and 20µg from standart protocol =&gt; standard virus
</LI><LI> we have 10µg and 20µg from standart protocol pellet =&gt; Pellet
</LI><LI> we have 10µg and 20µg from standart protocol suspension =&gt; Super
</LI><LI> amount of 500µl 
</LI></UL><P>1. Plate I(A is up)
</P><TABLE border="1"><TBODY><TR><TD> 10µg dropped Super </TD><TD align="right">10µg dropped Super gently resuspending</TD><TD align="right">10µg dropped Super gently resuspending
</TD></TR><TR><TD> 10µg Super gently resuspending Super</TD><TD align="right">20µg dropped Super gently resuspending 1:10 dilution</TD><TD align="right"> control
</TD></TR></TBODY></TABLE><P>
2. Plate I(A is up)
</P><TABLE border="1"><TBODY><TR><TD> 10µg dropped Super </TD><TD align="right">10µg dropped Super gently resuspending</TD><TD align="right">10µg dropped Super gently resuspending
</TD></TR><TR><TD> 10µg Super gently resuspending Super</TD><TD align="right">20µg dropped Super gently resuspending 1:10 dilution</TD><TD align="right"> control
</TD></TR></TBODY></TABLE><P>
3. Plate I(A is up)
</P><TABLE border="1"><TBODY><TR><TD> 10µg dropped Pellet </TD><TD align="right">10µg dropped Pellet gently resuspending</TD><TD align="right">10µg dropped Pellet gently resuspending
</TD></TR><TR><TD> 10µg Super gently resuspending Pellet</TD><TD align="right">20µg dropped Pellet gently resuspending 1:10 dilution</TD><TD align="right"> control
</TD></TR></TBODY></TABLE><P><B>The results</B></P><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Results of transduction with different harvested viral particles</CAPTION><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>plate 1 A 1
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>plate 1 A 2
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>
We only took some pictures from single cells, because there was no significant difference in fluorescence between each sample.
The main conclusion of this experiment is, that we have to get the flow cytometry started.
</P><H3><SPAN class="mw-headline" id="4.7_Transfection_with_different_amounts_of_plasmids"><P style="font-size:17px; background-color:#00dd77;">4.7 Transfection with different amounts of plasmids</P></SPAN></H3><P><B>The motivation</B>
We want to check if the mVenus expression and transduction efficiency can be improved with higher amounts of DNA.
<B>The plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">4.7</TH><TH width="100">5.7</TH><TH width="100">6.7</TH><TH width="100">7.7</TH><TH width="100">8.7</TH><TH width="100">9.7</TH><TH width="100">10.7</TH><TH width="100">11.7</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest + Transduction</TD><TD>FACS</TD></TR></TBODY></TABLE><P>The Transfection was performed with 10, 18 and 24 µg per plasmid (mVenus, pHelper and R/C).

The virus harvest was performed according to standard protocol:
</P><UL><LI> 3 Plates got transduced
</LI></UL><P>1. Plate I(A is up) completely 10 µg
</P><TABLE border="1"><TBODY><TR><TD> 300µl resuspended</TD><TD align="right">300µl resuspended</TD><TD align="right">450µl resuspended
</TD></TR><TR><TD> 600µl resuspended</TD><TD align="right">600µl resuspended</TD><TD align="right"> control
</TD></TR></TBODY></TABLE><P>
2. Plate II(A is up)completely 18 µg
</P><TABLE border="1"><TBODY><TR><TD> 300µl resuspended</TD><TD align="right">300µl resuspended</TD><TD align="right">450µl resuspended
</TD></TR><TR><TD> 600µl resuspended</TD><TD align="right">600µl resuspended</TD><TD align="right"> control
</TD></TR></TBODY></TABLE><P>
3. Plate III(A is up) completely 24 µg
</P><TABLE border="1"><TBODY><TR><TD> 300µl resuspended</TD><TD align="right">300µl resuspended</TD><TD align="right">450µl resuspended
</TD></TR><TR><TD> 600µl resuspended</TD><TD align="right">600µl resuspended</TD><TD align="right"> control
</TD></TR></TBODY></TABLE><P><B>The results</B>
FACS analysis of different treated HT1080 cells was done (18 samples)
</P><P>The first percentage refers to the living and fluorescent cells and the second percentage to the total amout of living cells.
</P><P>The data are listed this way:
</P><P>1. Plate I(A is up)
</P><TABLE border="1"><TBODY><TR><TD>A1</TD><TD align="right">A2</TD><TD align="right">A3
</TD></TR><TR><TD>B1</TD><TD align="right">B2</TD><TD align="right">B3
</TD></TR></TBODY></TABLE><P>1=  Plate 1 A1 : 0% YFP-positive Cells (there were only a few cells in this sample)
2=  Plate 1 A2 : 27,7% YFP-positve Cells  from 73,3% 
3=  Plate 1 A3 : 17,3%       &quot;            from 63,1% 
4=  Plate 2 A1 : 32,9%       &quot;            from 75,3% 
5=  Plate 2 A2 : 27,2%       &quot;            from 76,5% 
6=  Plate 2 A3 : 30,7%       &quot;            from 73,4% 
7=  Plate 3 A1 : 26,8%       &quot;            from 76,0%  
8=  Plate 3 A2 : 30,5%       &quot;            from 74,2% 
9=  Plate 3 A3 : 30,7%       &quot;            from 75,2% 
10= Plate 1 B1 : 6,25%       &quot;            from 36,1% 
11= Plate 1 B2 : 6,42%       &quot;            from 68,4% 
12= Plate 1 B3 : 0,05%       &quot;            from 89,5% 
13= Plate 2 B1 : 20,5%       &quot;            from 72,5%  
14= Plate 2 B2 : 15,7%       &quot;            from 73,6% 
15= Plate 2 B3 : 0,02%       &quot;            from 85,4% 
16= Plate 3 B1 : 19,5%       &quot;            from 73,3%  
17= Plate 3 B2 : 20,8%       &quot;            from 75,7% 
18= Plate 3 B3 : 0,07%       &quot;            from 84,8% </P><P>
Conclusions
</P><UL><LI> Higher amounts of Plasmids (24, 30µg) seems to have no obvious effect on YFP expression
</LI><LI> Theres no difference in YFP expression comparing resuspended samples to not resuspended samples
</LI><LI> The amount of dead HT cells is about 25%, it is not clear if it is either AAV induced or mechanic stress (Trypsin, washing procedure)
</LI></UL><P>Next steps:
</P><UL><LI> does the amount of GOI (ng plasmids) has an effect on YFP expression  
</LI><LI> compare transduced to non transduced cells (vitality), or in other words is the mechanic stress responsible for 25% dead cell ratio?
</LI><LI> comparison of transduction efficiency of different virus stock production routines
</LI></UL><H3><SPAN class="mw-headline" id="14.7_Seeding_AAV293_for_TKGMK_constructs"><P style="font-size:17px; background-color:#00dd77;">14.7 Seeding AAV293 for TKGMK constructs </P></SPAN></H3><P><B>The motivation</B>
We want to create the first viral particles with TKGMK as gene of interest.
<B>The plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">14.7</TH><TH width="100">15.7</TH><TH width="100">16.7</TH><TH width="100">17.7</TH><TH width="100">18.7</TH><TH width="100">19.7</TH><TH width="100">20.7</TH><TH width="100">21.7</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest + Transduction</TD><TD>FACS</TD></TR></TBODY></TABLE><P>30µg Plasmid were used (pHelper + pAAV_RC+ pAAV_RFC25_mgmk_TK30)
VIRUS no GOI :pHelper + pAAV_RC
VIRUS 1  :pHelper + pAAV_RC+ pAAV_RFC25_mgmk_TK30 clone 1 : P54
VIRUS 2  :pHelper + pAAV_RC+ pAAV_RFC25_mgmk_TK30 clone 2 : P55
VIRUS 3  :pHelper + pAAV_RC+ pAAV_RFC25_mgmk_TK30 clone 3 : P56</P><P><B>The results</B>
Unfortunately the stocks never were used (exception: see following experiment, 7. august) because later on we noticed that the TKGMK plasmids were not in line with the RFC.
</P><H3><SPAN class="mw-headline" id="7.8_Seeding_AAV293_for_TKGMK_and_mVenus_stocks"><P style="font-size:17px; background-color:#00dd77;">7.8 Seeding AAV293 for TKGMK and mVenus stocks</P></SPAN></H3><P><B>The motivation</B>
After proofing the successfull transduction of tumor cells with the reportergen mVenus the next step is to kill the cells with a prodrug approach. The prodrug of our choice is the guanosine anolog ganciclovir, once activated ganciclovir gets integrated into the growing dna chain which results in termination of dna synthesis and cell death. We transduce the tumor cell line HT1080 with viral particles packed with the thymidine kinase guanosine mono phosphate kinase fusion protein (TKGMK) as gene of interest and the incubation with ganciclovir should kill the cells.<B>The plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">7.8</TH><TH width="100">8.8</TH><TH width="100">9.8</TH><TH width="100">10.8</TH><TH width="100">11.8</TH><TH width="100">12.8</TH><TH width="100">13.8</TH><TH width="100">14.8</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest + Transduction</TD><TD>FACS and microscopy</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD> Plate  </TD><TD align="right">pAAV_RC/µg</TD><TD align="right">pHelper/µg</TD><TD align="right">GOI/µg
</TD></TR><TR><TD> 1</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> no GOI
</TD></TR><TR><TD> 2</TD><TD align="right">6,6 </TD><TD align="right"> 3,3</TD><TD align="right"> no GOI
</TD></TR><TR><TD> 3</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 3,3 TK_GMK clone1 (P54)
</TD></TR><TR><TD> 4</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 3,3 TK_GMK clone1 (P54)
</TD></TR><TR><TD> 5</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 3,3 TK_GMK clone2 (P55)
</TD></TR><TR><TD> 6</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 3,3 YFP
</TD></TR><TR><TD> 7</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 10 YFP
</TD></TR><TR><TD> 8</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 20 YFP
</TD></TR><TR><TD> 9</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 40 YFP
</TD></TR><TR><TD> 10</TD><TD align="right">3,3 </TD><TD align="right"> 3,3</TD><TD align="right"> 60 YFP
</TD></TR></TBODY></TABLE><P>The transduction was performed according to standard protocol. 
</P><P>Plate 1:</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>300µl virus 1</TD><TD>300µl virus 2</TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>600µl virus 1</TD><TD>600µl virus 2</TD></TR></TBODY></TABLE><P>
Plate 2:</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>300µl virus 3</TD><TD>300µl virus 4</TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>600µl virus 3</TD><TD>600µl virus 4</TD></TR></TBODY></TABLE><P>
Plate 3:</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>300µl virus 5</TD><TD>300µl virus 6</TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>600µl virus 5</TD><TD>600µl virus 6</TD></TR></TBODY></TABLE><P>
Plate 4:</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>300µl virus 7</TD><TD>300µl virus 8</TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>600µl virus 7</TD><TD>600µl virus 8</TD></TR></TBODY></TABLE><P>
Plate 5:</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>300µl virus 9</TD><TD>300µl virus 10</TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>600µl virus 9</TD><TD>600µl virus 10</TD></TR></TBODY></TABLE><P>
Plate 6:</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>300µl virus 3</TD><TD>300µl virus 4</TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>600µl virus 3</TD><TD>600µl virus 4</TD></TR></TBODY></TABLE><P>
Plate 7:</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>300µl virus 3</TD><TD>300µl virus 4</TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>500µl virus 3</TD><TD>500µl virus 4</TD></TR></TBODY></TABLE><P>
Plate 8:</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>300µl virus 5</TD><TD>300µl virus 5</TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>600µl virus 5</TD><TD>600µl virus 5</TD></TR></TBODY></TABLE><P>
The test for functionality for TK GMK was done with the Prodrug Cymeven ganciclovir 500mg from Roche. The final concentration was 0,05 mM in the wells. After two days pictures were taken.
</P><P><B>The results</B></P><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Death to HT1080</CAPTION><TBODY><TR><TD><DIV class="gallerybox" style="width: 335px;"><DIV class="gallerytext"><P>HT1080 with gancliclovir only
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 335px;"><DIV class="gallerytext"><P>HT1080 with TK_GMK without ganciclovir
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 335px;"><DIV class="gallerytext"><P>HT1080 with TK_GMK (300µl AAV stock) with ganciclovir
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 335px;"><DIV class="gallerytext"><P>HT1080 with TK_GMK (600µl AAV stock) with ganciclovir
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 335px;"><DIV class="gallerytext"><P>picture from the 40µg stock
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>As you can see, the TKGMK cell killing mechanism works! The samples with targeted mVenus constructs couldnt be checked because the FACS machine was not working. The fluorescence was checked with microscopy, the transduction efficiency is still low.
</P><H3><SPAN class="mw-headline" id="14.8_Seeding_different_amounts_of_AAV293_for_transfection"><P style="font-size:17px; background-color:#00dd77;"><B>14.8 Seeding different amounts of AAV293 for transfection</B></P></SPAN></H3><P><B>The motivation</B>
We want to know which confluence of AAV293 cells is optimal for AAV production.<B>The plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">14.8</TH><TH width="100">15.8</TH><TH width="100">16.8</TH><TH width="100">17.8</TH><TH width="100">18.8</TH><TH width="100">19.8</TH><TH width="100">20.8</TH><TH width="100">21.8</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest + Transduction</TD><TD>FACS</TD></TR></TBODY></TABLE><P>
We seed different amounts of cells in 10 cm<SUP>2</SUP>, and check the highest titer.</P><TABLE border="1"><TBODY><TR><TD> Plate  </TD><TD align="right">amount of seeded AAV-293 cells
</TD></TR><TR><TD> 1</TD><TD align="right">100 000
</TD></TR><TR><TD> 2</TD><TD align="right">200 000
</TD></TR><TR><TD> 3</TD><TD align="right">400 000
</TD></TR><TR><TD> 4</TD><TD align="right">500 000
</TD></TR><TR><TD> 5</TD><TD align="right">800 000
</TD></TR><TR><TD> 6</TD><TD align="right">1 000 000
</TD></TR><TR><TD> 7</TD><TD align="right">1 200 000
</TD></TR><TR><TD> 8</TD><TD align="right">1 500 000
</TD></TR><TR><TD> 9</TD><TD align="right">1 750 000
</TD></TR><TR><TD> 10</TD><TD align="right">1 750 000
</TD></TR></TBODY></TABLE><P>The cells were transfected with 40µg standard CMV_YFP, 3,3µg Rep/Cap and 3,3µg pHelper. 
</P><P>Transduction was performed according to standard protocol.

Plate 1 YFP: 200.000 cells per well</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>165µl virus (1) </TD><TD>165µl virus (2) </TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD> 165µl virus (1) </TD><TD> 165µl virus (2) </TD></TR></TBODY></TABLE><P>Plate 2 YFP: 200.000 cells per well</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>165µl virus (3) </TD><TD>165µl virus (4) </TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD> 165µl virus (3) </TD><TD> 165µl virus (4) </TD></TR></TBODY></TABLE><P>Plate 3 YFP: 200.000 cells per well</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>165µl virus (5) </TD><TD>165µl virus (6) </TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD> 165µl virus (5) </TD><TD> 165µl virus (6) </TD></TR></TBODY></TABLE><P>Plate 4 YFP: 200.000 cells per well</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>165µl virus (7) </TD><TD>165µl virus (8) </TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD> 165µl virus (7) </TD><TD> 165µl virus (8) </TD></TR></TBODY></TABLE><P>Plate 5 YFP: 200.000 cells per well</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>165µl virus (9) </TD><TD>165µl virus (10)</TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD> 165µl virus (9) </TD><TD> 165µl virus (10)</TD></TR></TBODY></TABLE><P><B>The results</B>
The FACS was still not available but we could easily detect a significant higher YFP expression (70-80%) at the following positions:
</P><UL><LI> Plate 1: A2 and B2, A3 and B3
</LI><LI> Plate 2: A2 and B2, A3 and B3
</LI></UL><P>Therefore we conclude that we have to seed between 100000 and 500000 cells for optimal virus production precedure.
</P><H3><SPAN class="mw-headline" id="17.8_transduction_with_viral_stocks_20_.C2.B5g_and_40_.C2.B5g_GOI-plasmid"><P style="font-size:17px; background-color:#00dd77;">17.8 transduction with viral stocks 20 µg and 40 µg GOI-plasmid</P></SPAN></H3><P><B>The Motivation</B>
We want to FACS our stocks to get valid data for further investigation of optimal virus assembly. Therefore created stocks with 20 and 40 µg CMV_mVenus.
<B>The Plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">17.8</TH><TH width="100">18.8</TH><TH width="100">19.8</TH><TH width="100">20.8</TH><TH width="100">21.8</TH><TH width="100">22.8</TH><TH width="100">23.8</TH><TH width="100">24.8</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest + Transduction</TD><TD>FACS</TD></TR></TBODY></TABLE><LI>1. plate 50.000 cells, YFP
<TABLE border="1"><TBODY><TR><TH>A
</TH><TD>control no cells
</TD><TD>300 µl virus (40 µg YFP)
</TD><TD>300 µl virus (20 µg YFP)
</TD></TR><TR><TH>B
</TH><TD>control no virus
</TD><TD>500µl P38 (pAAV2_mVenus)
</TD><TD>1000µl P38 (pAAV2_mVenus)
</TD></TR></TBODY></TABLE></LI><LI>2. plate 50.000 cells, YFP
<TABLE border="1"><TBODY><TR><TH>A
</TH><TD>control no cells
</TD><TD>300 µl virus (40 µg YFP)
</TD><TD>300 µl virus (20 µg YFP)
</TD></TR><TR><TH>B
</TH><TD>control no virus
</TD><TD>500µl P38 (pAAV2_mVenus)
</TD><TD>1000µl P38 (pAAV2_mVenus)
</TD></TR></TBODY></TABLE></LI><LI>3. plate 100.000 cells, YFP  
<TABLE border="1"><TBODY><TR><TH>A
</TH><TD>control no cells
</TD><TD>300 µl virus (40 µg YFP)
</TD><TD>300 µl virus (20 µg YFP)
</TD></TR><TR><TH>B
</TH><TD>control no virus
</TD><TD>500µl P38 (pAAV2_mVenus)
</TD><TD>1000µl P38 (pAAV2_mVenus)
</TD></TR></TBODY></TABLE></LI><LI>4. plate 100.000 cells, YFP
<TABLE border="1"><TBODY><TR><TH>A
</TH><TD>control no cells
</TD><TD>300 µl virus (40 µg YFP)
</TD><TD>300 µl virus (20 µg YFP)
</TD></TR><TR><TH>B
</TH><TD>control no virus
</TD><TD>500µl P38 (pAAV2_mVenus)
</TD><TD>1000µl P38 (pAAV2_mVenus)
</TD></TR></TBODY></TABLE><P><B>The Results</B>
Although we were told that the FACS would be available in time it wasn't. So the cells could unfortunately not be checked for their YFP expression =(.
</P><H3><SPAN class="mw-headline" id="16.8_Seeding_AAV293_for_transfection_with_new_R.2FC"><P style="font-size:17px; background-color:#00dd77;">16.8 Seeding AAV293 for transfection with new R/C</P></SPAN></H3><P><B>The motivation</B>
We want examine the functionality of different R/C constructs and the new eGFP gene of interest. This transfection was performed with new AAV-293 cells which were thawed.
</P><P><B>The plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">16.8</TH><TH width="100">17.8</TH><TH width="100">18.8</TH><TH width="100">19.8</TH><TH width="100">20.8</TH><TH width="100">21.8</TH><TH width="100">22.8</TH><TH width="100">23.8</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest + Transduction</TD><TD>FACS</TD></TR></TBODY></TABLE></LI><LI> 2x10<SUP>6</SUP> cells were seeded
</LI><LI> used plasmids: 
</LI><LI>P50   c= 429 ng/µl (RC)
</LI><LI>P64   c= 500 ng/µl (pHelper)
</LI><LI>P229  c= 162 ng/µl (GOI=YFP)
</LI><LI>P228  c= 540 ng/µl (GOI=eGFP)
</LI><LI>P158a c= 1800ng/µl (RC mutant)
</LI><LI>P158a c= 1770ng/µl (RC mutant) 
</LI><P>The transfection was performed according to standard protocol.
</P><UL><LI><B> 1. plate:</B> Lipo-Transfection =&gt; named Lipo
</LI><LI><B> 2. plate:</B> Lipo-Transfection =&gt; named Lipo
</LI><LI><B> 3. plate:</B> 3,3 µg (RC), 3,3 µg (pHelper), 3,3 µg (YFP) =&gt; was not used for transduction
</LI><LI><B> 4. plate:</B> 10 µg (RC), 10 µg (pHelper), 3,3 µg (YFP) =&gt; named CaCl<SUB>2</SUB></LI><LI><B> 5. plate:</B> 10 µg (RC mutant P158a), 10 µg (pHelper), 3,3 µg (YFP)=&gt; CaCl<SUB>2</SUB> 10 µg RC mutant P158a
</LI><LI><B> 6. plate:</B> 10 µg (RC mutant P158b), 10 µg (pHelper), 3,3 µg (YFP)=&gt; CaCl<SUB>2</SUB> 10 µg RC mutant P158b
</LI><LI><B> 7. plate:</B> 3,3 µg (RC mutant P158a), 3,3 µg (pHelper), 3,3 µg (YFP)=&gt; CaCl<SUB>2</SUB> 3,3 µg RC mutant P158a
</LI><LI><B> 8. plate:</B> 3,3 µg (RC mutant P158b), 3,3 µg (pHelper), 3,3 µg (YFP)=&gt; CaCl<SUB>2</SUB> 3,3 µg RC mutant P158b
</LI><LI><B> 9. plate:</B> 10 µg (RC), 10 µg (pHelper), 3,3 µg (eGFP)=&gt; CaCl<SUB>2</SUB> eGFP 3,3 µg
</LI><LI><B> 10. plate:</B> 10 µg (RC), 10 µg (pHelper), 10 µg (eGFP)=&gt; CaCl<SUB>2</SUB> eGFP 10 µg
</LI><LI><B> 11. plate:</B> 10 µg (RC), 10 µg (pHelper), 20 µg (eGFP)=&gt; CaCl<SUB>2</SUB> eGFP 20 µg
</LI><LI><B> 12. plate:</B> 10 µg (RC), 10 µg (pHelper), 40 µg (eGFP)=&gt; CaCl<SUB>2</SUB> eGFP 40 µg
</LI></UL><P><B>The results</B></P><P>As you can see the eGFP constructs worked very well. And even the transduction efficiency of the standard R/C is ok, the conclusion is that in previous transfections the AAV293 cells were to confluent so their transfection capability. The confluence of the cells is crucial! It also seems that our used amounts of plasmids had no decisive effect on the YFP expression because of the obviously variating results. A final conclusion is still due.
</P><H3><SPAN class="mw-headline" id="19.8_Seeding_AAv293_for_transfection_with_different_HBS_buffers"><P style="font-size:17px; background-color:#00dd77;">19.8 Seeding AAv293 for transfection with different HBS buffers </P></SPAN></H3><P><B>The Motivation</B>
We want to try different 2xHBS Buffers (pH) and their influence on transfection efficiency!<B>The Plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">19.8</TH><TH width="100">20.8</TH><TH width="100">21.8</TH><TH width="100">22.8</TH><TH width="100">23.8</TH><TH width="100">24.8</TH><TH width="100">25.8</TH><TH width="100">26.8</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest + Transduction</TD><TD>FACS</TD></TR></TBODY></TABLE><P>We try 5 different 2xHBS buffers:
</P><UL><LI>pH: 7.06</LI><LI>pH: 7.08</LI><LI>pH: 7.10</LI><LI>pH: 7.12</LI><LI>pH: 7.14</LI></UL><P>Used Plasmids: 10µg of each plasmid
</P><UL><LI>GOI: mVENUS =&gt; P262, conc:1148ng/µl; used amount: 8,71µl</LI><LI>GOI: TKGMK  =&gt; P82b, conc:1500ng/µl; used amount: 6,67µl</LI><LI>pHELPER =&gt; P64, conc:503ng/µl; used amount: 20µl</LI><LI>RepCap =&gt; 10.8: 1348ng/µl; used amount: 7,41µl</LI></UL><UL><LI>Transduction of three 6-well plates:
</LI></UL><P>Plate 1 YFP: 150.000 cells per well</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>500µl virus (1) </TD><TD>500µl virus (2) </TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>500µl virus (1) </TD><TD>500µl virus (2) </TD></TR></TBODY></TABLE><P>Plate 2 YFP: 150.000 cells per well</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>500µl virus (3) </TD><TD>500µl virus (4) </TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>500µl virus (3) </TD><TD>500µl virus (4) </TD></TR></TBODY></TABLE><P>Plate 3 YFP: 200.000 cells per well</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>500µl virus (5) </TD><TD>500µl virus (9) </TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD> 500µl virus (9) </TD><TD> 500µl virus (5) </TD></TR></TBODY></TABLE><UL><LI>(1)= 10µg RC, 10µg pHelper, 10µg GOI (YFP); pH(2xHBS)=7,06
</LI><LI>(2)= 10µg RC, 10µg pHelper, 10µg GOI (YFP); pH(2xHBS)=7,08
</LI><LI>(3)= 10µg RC, 10µg pHelper, 10µg GOI (YFP); pH(2xHBS)=7,112
</LI><LI>(4)= 10µg RC, 10µg pHelper, 10µg GOI (YFP); pH(2xHBS)=7,12
</LI><LI>(5)= 10µg RC, 10µg pHelper, 10µg GOI (YFP); pH(2xHBS)=7,14
</LI></UL><P><B>the Results</B>
AAV-Harvesting is at 24.8, same day Transduction and FACS will be done at 26.8
</P><P>As you can see the 2xHBS with pH 7.112 performed best, in future experiments this buffer will be used.
</P><H3><SPAN class="mw-headline" id="22.8:_Transfection_of_AAV293_with_different_amounts_of_cells.2C_and_two_plasmid_concentrations"><P style="font-size:17px; background-color:#00dd77;">22.8: Transfection of AAV293 with different amounts of cells, and two plasmid concentrations</P></SPAN></H3><P><B>The motivation</B>
We want to define the final amount of AAV293 per 10 cm<SUP>2</SUP> for optimal AAV-Production<B>The plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">22.8</TH><TH width="100">23.8</TH><TH width="100">24.8</TH><TH width="100">25.8</TH><TH width="100">26.8</TH><TH width="100">27.8</TH><TH width="100">28.8</TH><TH width="100">29.8</TH><TH width="100">30.8</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest + Transduction</TD><TD>-</TD><TD>FACS</TD></TR></TBODY></TABLE><P>The following amount of cells were seeded in 10 cm<SUP>2</SUP>:
</P><OL><LI>100.000</LI><LI>100.000</LI><LI>200.000</LI><LI>200.000</LI><LI>300.000</LI><LI>300.000</LI><LI>400.000</LI><LI>400.000</LI><LI>500.000</LI><LI>500.000</LI><LI>600.000</LI><LI>600.000</LI><LI>700.000</LI><LI>700.000</LI></OL><P>
10µg RC P158a four times mutant, 10µg pHelper, 3,3 µg mVenus P162  were pipetted on plates: 2, 4, 6, 8, 10 , 12, 14
 
3,3µg RC P158a four times mutant, 3,3µg pHelper, 3,3 µg mVenus P262 were pipetted on plates: 1, 3, 5, 7, 9, 11, 13
</P><P><B>The results</B>
There is only one value for each approach but nevertheless the average YFP expression should be higher.
We really dont know the reason for the quite low YFP expression. There are also very good news: the reassembled construct (pSB1C3_leftITR_CMV_betaglobin_mVenus_hGH_rightITR) created from the single Biobricks works as &quot;well&quot; as the AAV2 with CMV_mVenus!
</P><H3><SPAN class="mw-headline" id="30.8_Seeding_AAV293_for_Transfection_of_10_cm2_plates"><P style="font-size:17px; background-color:#00dd77;">30.8 Seeding AAV293 for Transfection of 10 cm<SUP>2</SUP> plates</P></SPAN></H3><P><B>The motivation</B>
We want to create ten hypothetical identical viral stocks to estimate the standard deviation. 
<B>The plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">30.8</TH><TH width="100">31.8</TH><TH width="100">1.9</TH><TH width="100">2.9</TH><TH width="100">3.9</TH><TH width="100">4.9</TH><TH width="100">5.9</TH><TH width="100">6.9</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest + Transduction</TD><TD>FACS</TD></TR></TBODY></TABLE><P>
Transfection will be done with:
</P><UL><LI>RepCap:P357 10µg =&gt;conc.:1274,8ng/µl used amount: 7,84µl       </LI><LI>pHelper:P356 10µg =&gt;conc.:1068ng/µl used amount: 9,36µl      </LI><LI>mVenus:P261 =&gt; excel sheet P263: 10µg =&gt;conc.:979ng/µl used amount: 10,21 µl         </LI></UL><P>
0,5 ml of each stock be stored at -80°C.
9,5 ml of each stock will be used for Transduction. 
Result:

Standard deviation: about +/- 4,98% YFP expression. All future AAV2 stocks will be pooled if produced with this same plasmids and amounts of plasmids. Unfortunately the YFP expression is still located at around 50%. We would like to have a YFP expression of at least 70% because as soon as we start to test our not yet ready modified AAV2 we suggest the YFP expression will obviously decrease.
</P><H3><SPAN class="mw-headline" id="4.9_Seeding_cells_for_checking_constructs_.28GOIs.29_with.2Fwithout_HGH_and_beta_globin_and_reassembled_as_control"><P style="font-size:17px; background-color:#00dd77;">4.9 Seeding cells for checking constructs (GOIs) with/without HGH and beta globin and reassembled as control </P></SPAN></H3><P><B>The motivation</B>
We want to check the functionality of these genes of interest
</P><P><B>The plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">4.9</TH><TH width="100">6.9</TH><TH width="100">7.9</TH><TH width="100">8.9</TH><TH width="100">9.9</TH><TH width="100">10.9</TH><TH width="100">11.9</TH><TH width="100">12.9</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest + Transduction</TD><TD>FACS</TD></TR></TBODY></TABLE><UL><LI>P269 pSB1C3_lITR_CMV_beta-globin_mVenus_HGH_rITR = 325 ng/µl =&gt; 9,66 µl </LI><LI>P270 pSB1C3_lITR_CMV_beta-globin_mVenus_HGH_rITR = 561 ng/µl =&gt; 7,26 µl</LI><LI>P377 pSB1C3_lITR_CMV_mVenus_HGH_rITR = 325,04 ng/µl =&gt; 30,76 µl   </LI><LI>P378 pSB1C3_lITR_CMV_beta-globin_mVenus_rITR = 561 ng/µl =&gt; 17,8µl  </LI></UL><P>Additionally P50: pAAV_RC R/C (26,45 µl) and pHELPER (14,61µl) pipetted to each approach. <B>The results</B></P><H3><SPAN class="mw-headline" id="6.9_checking_five_ganciclovir_concentrations_for_two_day_incubation_approach_optimal_medication"><P style="font-size:17px; background-color:#00dd77;">6.9 checking five ganciclovir concentrations for two day incubation approach optimal medication</P></SPAN></H3><P><B>The motivation</B>
We want to check the optimal ganciclovir concentration, we use the TK/GMK Stocks (10 µg from each plasmid) from 24.8 with Buffer pH 7,10 and the pH12 stock.
<B>The plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">3.9</TH><TH width="100">4.9</TH><TH width="100">5.9</TH><TH width="100">6.9</TH></TR><TR><TD>Action</TD><TD>seeding HT</TD><TD>Transduction</TD><TD>-</TD><TD>FACS</TD></TR></TBODY></TABLE><P>
We want to use five different ganciclovir concentrations (the concentrations are absolute to the volume in the wells):
</P><UL><LI>48.5µM</LI><LI>97 µM</LI><LI>0.485 mM</LI><LI>0.97 mM</LI><LI>4.85 mM</LI></UL><P><B>the results</B>
As expected the highest concentration of ganciclovir worked the best but it is remarkable that there is no significant difference between the first three ganciclovir amounts. The most remarkable note is, that in future experiments the incubation time has to be extended to 3 or 4 days.
</P><H3><SPAN class="mw-headline" id="15.9_Seeding_AAV293_for_testing_our_Affibody_and_other_VP-mVenus_constructs"><P style="font-size:17px; background-color:#00dd77;">15.9 Seeding AAV293 for testing our Affibody and other VP-mVenus constructs</P></SPAN></H3><P><B>The motivation</B>
The pCerulean_VP1up_mVenus_Vp2/3 Construct will be tested in three different compositions.<B>The plan</B>
We want to test these constructs:
</P><UL><LI>pCerulean_VP1up_NLS_Affibody_VP2/3</LI><LI>pCerulean_VP1up_His_VP2/3</LI><LI>pCerulean_VP1up_mVenus_Vp2/3</LI></UL><P>The His-tag is for purify our constructs and in this case we want to test, if the particles are still infectious. Construct with the affibody will be tested for functionality and even for their better affinity to our EGFR overexpressing cell line A431.
</P><P><B>The results</B>
Interpretation and discussion of our results.
</P><H3><SPAN class="mw-headline" id="19.9_Seeding_AAV293_for_checking_two_different_Rep.2FCap_Constructs"><P style="font-size:17px; background-color:#00dd77;">19.9 Seeding AAV293 for checking two different Rep/Cap Constructs</P></SPAN></H3><P><B>The motivation</B>
During the project 22 silent nucleotide exchange mutations were introduced into the capsid coding construct to make it compatible with the RFC standards and to have single cutting restriction enzymes flanking the 453 and the 587 loop sequence. Two point mutations had to be dismissed, because either a first test transduction showed that the construct was not working anymore or the insertion of the synthesized gene posed serious problems, because the restriction enzyme did not work.
<B>The plan</B>
Transfection and transduction was performed according to standard protocol.
</P><P><B>The results</B>
Finally, we have a construct that is shown to produce infectious particles comparable to the current AAV systems and carries 20 point mutations. Now we can announce that the Adeno-associated Virus is compatible to the RFC standard and the idea to replace the loop sequences via ViralBricks works!
</P><H3><SPAN class="mw-headline" id="22.9_Seeding_AAV293_for_transfection_with_different_mVenus_constructs_for_checking_specific_promoter_activity"><P style="font-size:17px; background-color:#00dd77;">22.9 Seeding AAV293 for transfection with different mVenus constructs for checking specific promoter activity</P></SPAN></H3><P><B>The motivation</B>
We want to check if the CMV and pTERT promoters work equaly in the AAV293, A431 and HT1080 cells. So we transfect them via standard transfection CaCl<SUB>2</SUB>. It is obvious that the individual cell lines are not equaliy competent to transfection (the AAV293 are optimised for transfection). It seems that this promoter is not that active in AAV293 cells.
<B>The plan</B>
We transfected the three cell lines A431, AAV293 and HT1080 with following three constructs:
</P><UL><LI>CMV_mVenus</LI><LI>CMV_eGFP</LI><LI>pTERT_mVenus</LI></UL><P>
Remember, that mVenus and eGFP have the exact same excitation wavelenght. We took the pictures with the same parameters, but saved the data in the wrong formate (.tif instead of .zvi), <B>so we werent able to change the YFP expression from green to yellow</B>.
</P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">22.9</TH><TH width="100">23.9</TH><TH width="100">24.10</TH><TH width="100">25.10</TH><TH width="100">26.10</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>fluorescence microscopy</TD><TD>fluorescence microscopy</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Transfection of different cell types with different promoter_reportergenes</CAPTION><TBODY><TR><TD><DIV class="gallerybox" style="width: 335px;"><DIV class="gallerytext"><P>AAV293 CMV_eGFP 16h post transfection
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 335px;"><DIV class="gallerytext"><P>AAV293 CMV_mVenus 16h post transfection
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 335px;"><DIV class="gallerytext"><P>AAV293 pTERT_mVenus 16h post transfection
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 335px;"><DIV class="gallerytext"><P>AAV293 CMV_eGFP 48h post transfection
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 335px;"><DIV class="gallerytext"><P>HT1080 CMV_mVenus 48h post transfection
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 335px;"><DIV class="gallerytext"><P>A431 CMV_mVenus 48h post transfection
</P></DIV></DIV></TD></TR></TBODY></TABLE><P><B>The results</B>
As excepted, the expression of the CMV_mVenus and CMV_eGFP was very high even at 16h post transfection. The pTERT_mVenus construct worked aswell, but significantly weaker! 
Problematic is the fact, that there was no expression in HT1080 and A431 detectable. Due the fact that these cell lines are not optimized for CaCl<SUB>2</SUB> transfection. We can't evaluate if the bad transfection efficiency or the low promoter activity of pTERT are responsible for the results.
</P><H3><SPAN class="mw-headline" id="22.9_Testing_of_Hannas_6_VP-constructs_with_different_compositions_.2810.25.2C_25.25.2C_50.25_and_75.25_fractions.29_with_a_VP2-knockout_construct"><P style="font-size:17px; background-color:#00dd77;">22.9 Testing of Hannas 6 VP-constructs with different compositions (10%, 25%, 50% and 75% fractions) with a VP2-knockout construct</P></SPAN></H3><P><B>The motivation</B> Test the following 18 constructs for functionality on A431 and HT1080 cells. Each of these constructs getting transfected in different compositions 25%, 50%, 75% and 90% with VP2_knockout 75%. In this way we are able to create viral particles with this ratios of VP proteins. We hope that the created particles are still infectious.</P><UL><LI>pCerulean_CFP_MiddleLinker_VP2/3insCap P528</LI><LI>pCerulean_Zegfr:1907_ShortlLinker_VP2/3insCap P527</LI><LI>pCerulean_6xHis_Middlelinker_VP2/3insCap P534</LI><LI>pCerulean_Zegfr:1907_Middlelinker_VP2/3insCap P535</LI><LI>pCerulean_Zegfr:1907_SEG_VP2/3_Capins 10:90 P502</LI><LI>pCerulean_Zegfr:1907_LongLinker_VP2/3_CapIns P503</LI></UL><P><B>The plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">20.9</TH><TH width="100">21.9</TH><TH width="100">22.9</TH><TH width="100">23.9</TH><TH width="100">24.9</TH><TH width="100">25.9</TH><TH width="100">26.9</TH><TH width="100">27.9</TH><TH width="100">28.9</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest + Transduction</TD><TD>-</TD><TD>FACS</TD></TR></TBODY></TABLE><P><B>The results</B>
Obviously the constructs still work!
</P><H3><SPAN class="mw-headline" id="22.9_Seeding_cells_for_testing_pTERT_mVenus_in_R.2FC_P326_and_P431"><P style="font-size:17px; background-color:#00dd77;">22.9 Seeding cells for testing pTERT_mVenus in R/C P326 and P431 </P></SPAN></H3><P><B>The motivation</B>
We want to check if the pTERT-promoter is realiable for tumor specific gene expression, we use two different kinds of R/C. <B>The plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">22.9</TH><TH width="100">23.9</TH><TH width="100">24.9</TH><TH width="100">25.9</TH><TH width="100">26.9</TH><TH width="100">27.9</TH><TH width="100">28.9</TH><TH width="100">29.9</TH><TH width="100">30.9</TH><TH width="100">1.10</TH><TH width="100">2.10</TH><TH width="100">3.10</TH><TH width="100">4.10</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest</TD><TD>-</TD><TD>-</TD><TD>Seeding HT, A293 and A431</TD><TD>Transduction</TD><TD>FACS</TD></TR></TBODY></TABLE><P><B>The results</B></P><H3><SPAN class="mw-headline" id="29.9_Checking_location_of_VP_proteins_via_fluorescence_microscopy_and_checking_fractions_of_centrifugation_steps_via_fluorescence_spectrometer"><P style="font-size:17px; background-color:#00dd77;">29.9 Checking location of VP proteins via fluorescence microscopy and checking fractions of centrifugation steps via fluorescence spectrometer</P></SPAN></H3><P><B>The motivation</B>: We want to figure out if it is possible to purify our viral stocks via centrifugation with 10.000 g and/or 20.000 g. In theory/ according to the literature the viral particles should be in the cells and attached to the HSPGs at the cellular surfaces.
<B>The plan</B>: Two different viral stocks were prepared:

Stock 1 with VP2-mVenus-fusion-Capsid loaded with TKGMK (4 wells from a 6 well plate)</P><UL><LI>R/C 1: 50% pCerulean_VP1up_NLS_mVenus_Vp2/3 P501</LI><LI>R/C 2: 50% Rep/Cap2: P449 pAAV_RC_4fachmut_VP1-ko:</LI><LI>GOI: TKGMK: P82.b</LI><LI>pHelper</LI></UL><P>
Stock 2 &quot;standard&quot; virus (4 wells from a 6 well plate)</P><UL><LI>R/C 1 P486</LI><LI>GOI: TKGMK: P82.b</LI><LI>pHelper</LI></UL><P>These two stocks got harvested according to standard protocol (scratching cells, transfer them in to 15 ml falcons, resuling 12ml solution total). After centrifugation (10 min 200g), the supernatant got transferred into two new 15 ml falcons and the pellets were resuspended with 10 ml DMEM Medium. The Stocks got freezed thawed two times so finally there were:
</P><OL><LI>DMEM as negative control</LI><LI>resuspended Pellet with mVenus-fusioned viral capsids</LI><LI>resuspended Pellet with &quot;standard virus&quot;</LI><LI>supernatant with mVenus-fusioned viral capsids</LI><LI>supernatant with &quot;standard virus&quot;</LI></OL><P>
The fluorescence of each stock was measured via spectrometer. After that, stocks 2 and 4 were centrifugated 10min with 10.000 g followed by an other fluorescence measuring. Finally the stocks got spin down with 20.000 g for 10 min
<B>The results</B>:
</P><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Transfection with VP-mVenus-fusion constructs</CAPTION><TBODY><TR><TD><DIV class="gallerybox" style="width: 335px;"><DIV class="gallerytext"><P>Picture two days post transfection.
</P></DIV></DIV></TD></TR></TBODY></TABLE><P><B>The conclusions</B>: 
The highest amount of fluorescence was measured in the supernatant, the individual centrifugation steps had <B>no decrease in fluorescence</B> as consequence!
</P><P>
Keep in mind, without further investigation it is not valid to say that we can purify our stocks via 10-20.000 g centrifugation steps, because we dont know if the viral capsids are still intact! The FACS analysis are not sufficient enough to make a decision, because our last samples had YFP as GOI (=&gt; so efficient cell sorting was not possible). We need to do qPCRs to make a valid evidence.
</P><H3><SPAN class="mw-headline" id="30.9_Production_of_new_Standard_mVenus_Vector"><P style="font-size:17px; background-color:#00dd77;">30.9 Production of new Standard mVenus Vector</P></SPAN></H3><P><B>The motivation</B>
The AAV2 with 4 point mutations, inserted rep and cap and remutated KpnI restriction has to be checked for its functionality in order to compare it with the wild type AAV2. Hopefully there will be no significant difference.<B>The plan</B>
After the &quot;delay&quot; 200.000 A431 and HT1080 cells were seeded and transduced on the following day with 0,5 ml of this AAV2 mutant.
</P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">30.9</TH><TH width="100">1.10</TH><TH width="100">2.10</TH><TH width="100">3.10</TH><TH width="100">4.10</TH><TH width="100">5.10</TH><TH width="100">6.10</TH><TH width="100">7.10</TH><TH width="100">8.10</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest</TD><TD>delayed because of cell death</TD><TD>delayed because of cell death</TD></TR></TBODY></TABLE><P><B>The results</B>
The stocks were harvested successfully
</P><H3><SPAN class="mw-headline" id="30.9_Seeding_AAV293_for_production_of_standard_TKGMK_vector"><P style="font-size:17px; background-color:#00dd77;">30.9 Seeding AAV293 for production of standard TKGMK vector  </P></SPAN></H3><P><B>The motivation</B>
We want to create a big amount of viral particles with TKGMK as gene of interest for using it as an standard vector for the MTT-assay.
<B>The plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">30.9</TH><TH width="100">1.10</TH><TH width="100">2.10</TH><TH width="100">3.10</TH><TH width="100">4.10</TH><TH width="100">5.10</TH><TH width="100">6.10</TH><TH width="100">7.10</TH><TH width="100">8.10</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest</TD><TD>delayed because of cell death</TD><TD>delayed because of cell death</TD></TR></TBODY></TABLE><P><B>The results</B>
The viral stocks were harvested efficiently and are ready to use.
</P><H3><SPAN class="mw-headline" id="30.9_Seeding_AAV293_for_production_of_vectors_with_mVenus_missing_beta_globin"><P style="font-size:17px; background-color:#00dd77;">30.9 Seeding AAV293 for production of vectors with mVenus missing beta globin</P></SPAN></H3><P><B>The motivation</B>
Beta globin is said to improve (stratagene) the expression so we expect a worse mVenus (reportergene) expression compared to the standard AAVs. Therefore the seeded A431 and HT1080 cells were transduced with a AAV2 mutant missing beta-globin. 
<B>The plan</B>
Ater the &quot;delay&quot; 200.000 HT1080 and A431 cells were seeded and transduced with 0,5 ml AAV2 mutant on the following day.
</P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">30.9</TH><TH width="100">1.10</TH><TH width="100">2.10</TH><TH width="100">3.10</TH><TH width="100">4.10</TH><TH width="100">5.10</TH><TH width="100">6.10</TH><TH width="100">7.10</TH><TH width="100">8.10</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest</TD><TD>delayed because of cell death</TD><TD>delayed because of cell death</TD></TR></TBODY></TABLE><P><B>The results</B>
Interpretation and discussion of our results.
</P><H3><SPAN class="mw-headline" id="30.9_Seeding_AAV293_for_production_of_vectors_with_mVenus_missing_HGH"><P style="font-size:17px; background-color:#00dd77;">30.9 Seeding AAV293 for production of vectors with mVenus missing HGH</P></SPAN></H3><P><B>The motivation</B>
HGH is responsible for the mRNA polyadenylation of the virus so we expect expect a worse mVenus (reportergene) expression compared to the standard AAVs. Therefore the seeded A431 and HT1080 cells were transduced with a AAV2 mutant missing HGH. 
<B>The plan</B>
After the &quot;delay&quot; 200.000 A431 and HT1080 cells were seeded and transduced on the following day with 0,5 ml AAV2 mutant missing HGH.
</P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">30.9</TH><TH width="100">1.10</TH><TH width="100">2.10</TH><TH width="100">3.10</TH><TH width="100">4.10</TH><TH width="100">5.10</TH><TH width="100">6.10</TH><TH width="100">7.10</TH><TH width="100">8.10</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest</TD><TD>delayed because of cell death</TD><TD>delayed because of cell death</TD></TR></TBODY></TABLE><P><B>The results</B>
Interpretation and discussion of our results.
</P><H3><SPAN class="mw-headline" id="30.9_Seeding_AAV293_for_testing_RC_with_HSPG_knockout_for_functionality"><P style="font-size:17px; background-color:#00dd77;">30.9 Seeding AAV293 for testing RC with HSPG knockout for functionality</P></SPAN></H3><P><B>The motivation</B>
The primary AAV2 receptor (the HSPG receptor) was knocked out so we expect a worse tranduction rate (about 0,3 to 0,4 of the standard transduction rate, assuming that there are HSPGs on the HT1080 and A431 cell surface). The worse the better.
<B>The plan</B>
After the &quot;delay&quot; 200.000 A431 and HT1080 cells were seeded and transduced on the following day with 0,5 ml AAV2 mutant with a HSPG-receptor knockout.
</P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">30.9</TH><TH width="100">1.10</TH><TH width="100">2.10</TH><TH width="100">3.10</TH><TH width="100">4.10</TH><TH width="100">5.10</TH><TH width="100">6.10</TH><TH width="100">7.10</TH><TH width="100">8.10</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest</TD><TD>delayed because of cell death</TD><TD>delayed because of cell death</TD></TR></TBODY></TABLE><P><B>The results</B></P><P>
There is a significant difference in transduction efficiency. The actual reduction is even higher than expected, according to literature it should be around 60%, we reached 80%. We measured via qPCR the amount of viral particles for each stock, and they are nearly the same the HSPG Knockout has 2*10<SUP>7</SUP>particles per ml , the stock with HSPG-binding motif has 4.5*10<SUP>6</SUP> particles per ml.
</P><H3><SPAN class="mw-headline" id="30.9_Seeding_AAV293_for_testing_RC_WITHOUT_HSPG_knockout_for_functionality"><P style="font-size:17px; background-color:#00dd77;">30.9 Seeding AAV293 for testing RC WITHOUT HSPG knockout for functionality</P></SPAN></H3><P><B>The motivation</B>
The produced vector has the intact HSPG-binding motif, so we can compare it with the vector produced in: &quot;Seeding AAV293 for testing RC with HSPG knockout for functionality&quot;
<B>The plan</B>
Transfection and transduction were performed according to standard protocol.
</P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">30.9</TH><TH width="100">1.10</TH><TH width="100">2.10</TH><TH width="100">3.10</TH><TH width="100">4.10</TH><TH width="100">5.10</TH><TH width="100">6.10</TH><TH width="100">7.10</TH><TH width="100">8.10</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest</TD><TD>delayed because of cell death</TD><TD>delayed because of cell death</TD></TR></TBODY></TABLE><P><B>The results</B>
See 30.9 Seeding AAV293 for testing RC with HSPG knockout for functionality
</P><H3><SPAN class="mw-headline" id="12.10_Seeding_AAV293_cells_for_transfection_with_VP-BAP"><P style="font-size:17px; background-color:#00dd77;">12.10 Seeding AAV293 cells for transfection with VP-BAP</P></SPAN></H3><P><B>The motivation</B>
We want to create a viral stock for purification and biotinylation, the ratio of the Rep/Cap plasmids is 1:1.<B>The plan</B>
Transfection was performed according to standard protocol:
</P><UL><LI>Gene of interest: mVenus</LI><LI>R/C: pSB1C3-001_pCMV_[AAV]VP123_587_BAP in cotransfection with pSB1C3_001_RC_IRCK_P5tataless clone 1</LI></UL><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">12.10</TH><TH width="100">13.10</TH><TH width="100">14.10</TH><TH width="100">15.10</TH><TH width="100">16.10</TH><TH width="100">17.10</TH><TH width="100">18.10</TH><TH width="100">19.10</TH><TH width="100">20.10</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest + Transduction</TD><TD>-</TD><TD>FACS</TD></TR></TBODY></TABLE><P><B>The results</B></P><H3><SPAN class="mw-headline" id="12.10_Seeding_AAV293_cells_for_transfection_with_mVenus_as_gene_of_interest_and_pCerulean_VP1up_NLS_mVenus_VP2.2F3_HSPG-KO"><P style="font-size:17px; background-color:#00dd77;">12.10 Seeding AAV293 cells for transfection with mVenus as gene of interest and pCerulean_VP1up_NLS_mVenus_VP2/3_HSPG-KO</P></SPAN></H3><P><B>The motivation</B>
We want to create a viral stock for purification and biotinylation, the ratio of the Rep/Cap plasmids is 1:1.<B>The plan</B>
Transfection was performed according to standard protocol:
</P><UL><LI>Gene of interest: mVenus</LI><LI>R/C: pCerulean_VP1up_NLS_mVenus_VP2/3_HSPG-KO in cotransfection with pSB1C3_001_RC_IRCK_VP1-ko_HSPG-ko_P5tataless cl1</LI></UL><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">12.10</TH><TH width="100">13.10</TH><TH width="100">14.10</TH><TH width="100">15.10</TH><TH width="100">16.10</TH><TH width="100">17.10</TH><TH width="100">18.10</TH><TH width="100">19.10</TH><TH width="100">20.10</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest + Transduction</TD><TD>-</TD><TD>FACS</TD></TR></TBODY></TABLE><P><B>The results</B></P><H3><SPAN class="mw-headline" id="12.10_Seeding_AAV293_cells_for_transfection_with_mVenus_as_GOI_and_CFP_Middlelinker_VP2.2F3_HSPG-KO"><P style="font-size:17px; background-color:#00dd77;">12.10 Seeding AAV293 cells for transfection with mVenus as GOI and CFP_Middlelinker_VP2/3_HSPG-KO</P></SPAN></H3><P><B>The motivation</B>
This viral stock is for live imaging and probably for western blotting, the ratio of the Rep/Cap plasmids is 1:1.<B>The plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">18.10</TH><TH width="100">19.10</TH><TH width="100">20.10</TH><TH width="100">21.10</TH><TH width="100">22.10</TH><TH width="100">23.10</TH><TH width="100">24.10</TH><TH width="100">25.10</TH><TH width="100">26.10</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest + Transduction</TD><TD>-</TD><TD>FACS</TD></TR></TBODY></TABLE><P>Transfection was performed according to standard protocol:
</P><UL><LI>Gene of interest: mVenus</LI><LI>R/C: CFP_Middlelinker_VP2/3_HSPG-KO in cotransfection with pSB1C3_001_RC_IRCK_P5tataless clone 1</LI></UL><P><B>The results</B></P><H3><SPAN class="mw-headline" id="17.10_Production_of_AAV2_with_587-His"><P style="font-size:17px; background-color:#00dd77;">17.10 Production of AAV2 with 587-His</P></SPAN></H3><P>Investigator Patrick <B>The motivation</B>
We wanted to produce AAV2 which could be purified via affinity purification. Therefore 10 x 10cm^2 were seeded with AAV-293. Five of these plates contained serum-free medium and cells accustomized to this. 
Used plasmids:
</P><UL><LI> pHelper
</LI><LI> pSB1C3_leftITR_CMV_betaglobin_mvenus_hGH_rightITR (P541)
</LI><LI> 50% pSB1C3_001_RC_IRCK_P5tataless (P712)
</LI><LI> 50% pSB1C3-001_pCMV_[AAV2]VP123_587_HIS (P638)
</LI></UL><P><B>The plan</B>
In this section, the design of the experiment is explained.
</P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">17.10</TH><TH width="100">19.10</TH><TH width="100">20.10</TH><TH width="100">21.10</TH><TH width="100">22.10</TH><TH width="100">23.10</TH><TH width="100">24.10</TH><TH width="100">25.10</TH><TH width="100">26.10</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>Transfection</TD><TD>-</TD><TD>Seeding HT</TD><TD>AAV2 harvest</TD><TD>Transduction</TD><TD>FACS</TD></TR></TBODY></TABLE><P><B>The results</B>
Interpretation and discussion of our results.
</P><H3><SPAN class="mw-headline" id="18.10_Seeding_AAV293_cells_for_transfection_Superconstructs"><P style="font-size:17px; background-color:#00dd77;">18.10 Seeding AAV293 cells for transfection Superconstructs</P></SPAN></H3><P><B>The motivation</B>
We want to create a viral stock for purification and biotinylation, the ratio of the Rep/Cap plasmids is 1:1.<B>The plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">18.10</TH><TH width="100">19.10</TH><TH width="100">20.10</TH><TH width="100">21.10</TH><TH width="100">22.10</TH><TH width="100">23.10</TH><TH width="100">24.10</TH><TH width="100">25.10</TH><TH width="100">26.10</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest + Transduction</TD><TD>-</TD><TD>FACS</TD></TR></TBODY></TABLE><P>Transfection was performed according to standard protocol:<B>Gene of interest: CMV_mVenus</B></P><UL><LI>pSB1C3_CMV_Zegfr:1907_MiddleLinker_VP2/3_587-KO_6xHis in ratio 1:1 pSB1C3_001_RC_IRCK_VP2-ko_HSPG-ko_P5tataless cl1</LI><LI>pSB1C3-001_pCMV_[AAV2]VP123_453_Z34C in ratio 1:1 pSB1C3_001_RC_IRCK_HSPG-ko_P5tataless_RFC10</LI><LI>pSB1C3_001_CMV_VP123_587-KO_Z34C in ratio 1:1 pSB1C3_001_RC_IRCK_HSPG-ko_P5tataless_RFC10</LI><LI> pSB1C3_001_CMV_VP123_587-KO_Z34C_spacer in ratio 1:1 pSB1C3_001_RC_IRCK_HSPG-ko_P5tataless_RFC10</LI></UL><P><B>Gene of interest: pTERT_TKGMK</B></P><UL><LI>pSB1C3_CMV_Zegfr:1907_MiddleLinker_VP2/3_587-KO_6xHis in ratio 1:1 pSB1C3_001_RC_IRCK_VP2-ko_HSPG-ko_P5tataless cl1</LI></UL><P><B>Gene of interest: CMV_TKGMK</B></P><UL><LI>pSB1C3_CMV_Zegfr:1907_MiddleLinker_VP2/3_587-KO_6xHis in ratio 1:1 pSB1C3_001_RC_IRCK_VP2-ko_HSPG-ko_P5tataless cl1</LI></UL><P><B>The results</B></P><H3><SPAN class="mw-headline" id="18.10_Seeding_AAV293_cells_for_transfection_with_Cytosindeaminase"><P style="font-size:17px; background-color:#00dd77;">18.10 Seeding AAV293 cells for transfection with Cytosindeaminase</P></SPAN></H3><P><B>The motivation</B>
We want to create viral stocks with CMV_Cytosindeaminase and pTERT_Cytosindeaminase as gene of interests, to  check qualitatively the bystander effect.<B>The plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">18.10</TH><TH width="100">19.10</TH><TH width="100">20.10</TH><TH width="100">21.10</TH><TH width="100">22.10</TH><TH width="100">23.10</TH><TH width="100">24.10</TH><TH width="100">25.10</TH><TH width="100">26.10</TH><TH width="100">27.10</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest + Transduction</TD><TD>-</TD><TD>FACS</TD></TR></TBODY></TABLE><P>Transfection, harvest and transduction were performed according to standard protocol:<B>Gene of interest: CMV_Cytosindeaminase</B></P><UL><LI>P712</LI></UL><P><B>Gene of interest: pTERT_Cytosindeaminase</B></P><UL><LI>P712</LI></UL><P><B>The results</B></P><H3><SPAN class="mw-headline" id="22.10.2010_Transduction_with_old_mVenus_Stocks_for_taking_pictures"><P style="font-size:17px; background-color:#00dd77;">22.10.2010 Transduction with old mVenus Stocks for taking pictures</P></SPAN></H3><P><B>The motivation</B>
We transduced HT1080 with our stocks packaged with CMV_mVenus.
<B>The plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">22.9</TH><TH width="100">23.9</TH><TH width="100">24.9</TH><TH width="100">25.9</TH></TR><TR><TD>Action</TD><TD>seeding HT1080</TD><TD>Transduction</TD><TD>-</TD><TD>Fluorescence microscopy</TD></TR></TBODY></TABLE><P><B>The results</B></P><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Transduction with vectors pSB1C3_001_RC_IRCK_P5tataless as R/C and CMV_mVenus</CAPTION></TABLE><H3><SPAN class="mw-headline" id="23.10_Testing_the_bystander_effect_of_the_cytosine_deaminase"><P style="font-size:17px; background-color:#00dd77;">23.10 Testing the bystander effect of the cytosine deaminase</P></SPAN></H3><P><B>The motivation</B>
The activated form of 5-fluorocytosine has the ability to diffuse through the plasma membrane. In this way cells get killed which were not transduced by the viral particles. We use an old yfp stock t determine if the transgene expression is on and to harvest the cells which are transduced with cytosine deaminase loaded AAV2.<B>The plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">23.10</TH><TH width="100">24.10</TH><TH width="100">25.10</TH><TH width="100">26.10</TH><TH width="100">27.10</TH><TH width="100">28.10</TH></TR><TR><TD>Action</TD><TD>Transduce HT1080 and seed HT1080</TD><TD>Transfer the transduced HT1080 onto the non transduced HT1080</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>Measure living cells via Neubauer cellchamber</TD></TR></TBODY></TABLE><P><B>Gene of interest: CMV_Cytosindeaminase</B></P><UL><LI>P712</LI></UL><P><B>Gene of interest: CMV_mVenus</B></P><UL><LI>P468</LI></UL><P>All dishes had this order:</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>Medium with prodrug</TD><TD>Medium with prodrug</TD><TD>Medium with prodrug</TD></TR><TR><TD>B</TD><TD>Medium</TD><TD>Medium</TD><TD>Medium</TD></TR></TBODY></TABLE><P>The concentration of the prodrug 5-fluorocytosine is 54 mmolarCells were seeded following this scheme:<B><FONT size="3">plate 1</FONT></B></P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>1*10<SUP>5</SUP> HT1080 transduced with CMV_Cytosindeaminase</TD><TD>1*10<SUP>5</SUP> HT1080 transduced with CMV_Cytosindeaminase</TD><TD>1*10<SUP>5</SUP> HT1080 transduced with CMV_Cytosindeaminase</TD></TR><TR><TD>B</TD><TD>1*10<SUP>5</SUP> HT1080 transduced with CMV_Cytosindeaminase</TD><TD>1*10<SUP>5</SUP> HT1080 transduced with CMV_Cytosindeaminase</TD><TD>1*10<SUP>5</SUP> HT1080 transduced with CMV_Cytosindeaminase</TD></TR></TBODY></TABLE><P><B><FONT size="3">plate 2</FONT></B></P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>2*10<SUP>5</SUP> HT1080 non transduced</TD><TD>2*10<SUP>5</SUP> HT1080 non transduced</TD><TD>2*10<SUP>5</SUP> HT1080 non transduced</TD></TR><TR><TD>B</TD><TD>2*10<SUP>5</SUP> HT1080 non transduced</TD><TD>2*10<SUP>5</SUP> HT1080 non transduced</TD><TD>2*10<SUP>5</SUP> HT1080 non transduced</TD></TR></TBODY></TABLE><P>
The next plates were seeded with non transduced cells and 1*10<SUP>5</SUP> of the cytosin deaminase transduced cells 
<B><FONT size="3">plate 3</FONT></B></P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>2*10<SUP>5</SUP> HT1080</TD><TD>2*10<SUP>5</SUP> HT1080</TD><TD>2*10<SUP>5</SUP> HT1080</TD></TR><TR><TD>B</TD><TD>2*10<SUP>5</SUP> HT1080</TD><TD>2*10<SUP>5</SUP> HT1080</TD><TD>2*10<SUP>5</SUP> HT1080</TD></TR></TBODY></TABLE><P><B>The results</B></P><H3><SPAN class="mw-headline" id="25.10.2010_Transduction_with_several_constructs_for_Flow_Cytometry"><P style="font-size:17px; background-color:#00dd77;">25.10.2010 Transduction with several constructs for Flow Cytometry</P></SPAN></H3><P><B>The motivation</B>
We want to quantify via flow cytometry the Affibody superconstruct, Darpin HSPG-KO Construct and standard R/C with HSPG-KO.
<B>The plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">25</TH><TH width="100">26</TH><TH width="100">27</TH></TR><TR><TD>Action</TD><TD>Transduction</TD><TD>-</TD><TD>FACS</TD></TR></TBODY></TABLE><P><B>The results</B></P><H3><SPAN class="mw-headline" id="BLANK_EXPERIMENT"><P style="font-size:17px; background-color:#00dd77;">BLANK EXPERIMENT</P></SPAN></H3><P><B>The motivation</B><B>The plan</B></P><TABLE border="5" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="100">Days</TH><TH width="100">-</TH><TH width="100">-</TH><TH width="100">-</TH><TH width="100">-</TH><TH width="100">-</TH><TH width="100">-</TH><TH width="100">-</TH><TH width="100">-</TH><TH width="100">-</TH></TR><TR><TD>Action</TD><TD>seeding AAV293</TD><TD>-</TD><TD>Transfection</TD><TD>-</TD><TD>-</TD><TD>Seeding HT</TD><TD>Harvest + Transduction</TD><TD>-</TD><TD>FACS</TD></TR></TBODY></TABLE><P><B>The results</B></P><H1><SPAN class="mw-headline" id="Citations">Citations</SPAN></H1><P>
[1] <A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=Freiburg10_Stratagene_AAV_helper_free_manual.pdf" class="new" title="Freiburg10 Stratagene AAV helper free manual.pdf">Media:Freiburg10_Stratagene_AAV_helper_free_manual.pdf</A>
[2] <A href="http://www.copewithcytokines.de/cope.cgi?key=A431" class="external text" rel="nofollow">Titel einfügen</A>
[3] <A href="/wiki/index.php?title=Read,_A._P.;_Strachan,_T._(1999)._%22Chapter_18:_Cancer_Genetics%22._Human_molecular_genetics_2._New_York:_Wiley._ISBN_0-471-33061-2.&amp;action=edit&amp;redlink=1" class="new" title="Read, A. P.; Strachan, T. (1999). &quot;Chapter 18: Cancer Genetics&quot;. Human molecular genetics 2. New York: Wiley. ISBN 0-471-33061-2. (page does not exist)">Read, A. P.; Strachan, T. (1999). &quot;Chapter 18: Cancer Genetics&quot;. Human molecular genetics 2. New York: Wiley. ISBN 0-471-33061-2.</A>
[4] <A href="http://www.biomol.de/dateien/infos_nr778.gif" class="external text" rel="nofollow">Titel einfügen</A>
[5] <A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=Freiburg10_Titration_of_AAV-2_particles_via_a_novel_capsid_ELISA.pdf" class="new" title="Freiburg10 Titration of AAV-2 particles via a novel capsid ELISA.pdf">Media:Freiburg10 Titration of AAV-2 particles via a novel capsid ELISA.pdf</A></P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/Cellculture">http://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/Cellculture</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg_Bioware/NoteBook/Labjournal/Cellculture" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg_Bioware/NoteBook/Labjournal/Cellculture" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/Cellculture&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/Cellculture&amp;oldid=197597" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>