<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Bioware_NoteBook_Labjournal_August2"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg_Bioware/NoteBook/Labjournal/August2">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg_Bioware/NoteBook/Labjournal/August2&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/August2&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/August2&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg_Bioware/NoteBook/Labjournal/August2" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg Bioware/NoteBook/Labjournal/August2</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="menu"><UL id="menu"><LI><DL><DT class="menu_button_left"><A href="https://2010.igem.org/Team:Freiburg_Bioware">Home</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Team_2010">Team</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Team_2010">Team 2010</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Photo_Gallery">Photo Gallery</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Cuckoo_Clock">Cuckoo Clock Competition</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Collaboration">Collaboration</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Team/Example_Page">Example Page</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project">Project</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project">Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#highlights">Highlights</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#highlights">Results</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#modularisation">   Modularisation</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#targeting">   Targeting</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#arming">   Arming</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Results#superconstruct">   All-in-One</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Methods">Methods</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Project/Virus_Construction_Kit">Virus Construction Kit Manual</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks">BioBricks</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks">Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/capsid_coding">Capsid Coding Parts</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/vector_plasmid">Vector Plasmid Parts</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/BioBricks/miscellaneous">Miscellaneous Parts</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A class="current" href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook">Notebook</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal">Labjournal</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Materials">Materials</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Methods">Methods</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Experiments">Experiments</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Meetings">Meetings</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Statistics">Statistics</A></DD></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Sponsors">Sponsors</A></DT></DL></LI><LI><DL><DT class="menu_button_center"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Safety">Safety</A></DT></DL></LI><LI><DL><DT class="menu_button_right"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling">Modeling</A></DT><DD class="first"><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling">Modeling Overview</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Virus_Production">Virus Production</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Virus_Infection">Virus Infection</A></DD><DD><A href="https://2010.igem.org/Team:Freiburg_Bioware/Modeling/Structure_Modeling">Structure Modeling</A></DD></DL></LI></UL></DIV><DIV class="pageContent"><P><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook" class="external text" rel="nofollow">=&gt; Back to Notebook overview</A></P><DIV class="box_right"><LEFT><U1>NoteBook Navigator</U1></LEFT><UL><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/March">March (labday 1)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/April">April (labday 2 - 5)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/May">May (labday 6 - 17)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/June">June (labday 18 - 45)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/July">July (labday 46 - 75)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/August">August part 1 (labday 76 - 92)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/August2">August part 2 (labday 93 - 106)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/September">September part 1 (labday 107 - 123)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/September2">September part 2 (labday 124 - 135)</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/October">October part 1 (labday 136 - 145 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/October2">October part 2 (labday 146 - 155 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/October3">October part 3 (labday 156 - 166 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/November">November  (labday 167 - 170 )</A></LI><LI><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/Cellculture">Cellculture </A></LI></UL></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#93._labday_18.08.2010"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">93. labday 18.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Continuation:_pSB1C3_SDM_SspI_Bla14FM_clone_5_SDM_.28P222.29"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Continuation: pSB1C3_SDM_SspI_Bla14FM clone 5 SDM (P222)</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Mini_Preps_pEGFP_C1.2CpSB1C3_Rep40.2CpSB1C3_Rep78.2CpMA_T_Affibody.2CpSB1C3_SspIdel_BLA_clone_2.2B3.2CpSB1C3_AAP.2C_pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR.2CpSB1C3_lITR_CMV_mVenus_hGH_rITR.2CpSK_CD_SDMPstI"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Mini Preps pEGFP_C1,pSB1C3_Rep40,pSB1C3_Rep78,pMA_T_Affibody,pSB1C3_SspIdel_BLA clone 2+3,pSB1C3_AAP, pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR,pSB1C3_lITR_CMV_mVenus_hGH_rITR,pSK_CD_SDMPstI</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Continuation_of_Repetition:_Biobrick_production_of_Rep_68_.2817.08.2010.29"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Continuation of Repetition: Biobrick production of Rep 68 (17.08.2010)</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Transfection_of_AAV293_cells"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Transfection of AAV293 cells</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Seeding_Ht1080"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Seeding Ht1080</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Cloning_of_pMA_T_affibody_and_pSB1C3_CFP"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Cloning of pMA_T_affibody and pSB1C3_CFP</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-8"><A href="#94._labday_19.08.2010"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">94. labday 19.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Continuation:_pSB1C3_SDM_SspI_Bla14FM_clone_5_SDM_.28P222.29_2"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Continuation: pSB1C3_SDM_SspI_Bla14FM clone 5 SDM (P222)</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Cloning_of_pSB1C3_CFP_and_middlelinker"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Cloning of pSB1C3_CFP and middlelinker</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Test_digestion_of_pSB1C3_Rep78"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Test digestion of pSB1C3_Rep78</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Sent_for_sequencing"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Sent for sequencing</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Test_Digestion_of_P242.2C_P243.2C_P245.2C_P246.2C_P247"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Test Digestion of P242, P243, P245, P246, P247</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Test_digestion_of_pAAV_RC-ins-rep"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">Test digestion of pAAV_RC-ins-rep</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Midiprep_of_dsAAV_CMV_EGFP_and_pAAV_iGEM_mVenus_YF"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">Midiprep of dsAAV_CMV_EGFP and pAAV_iGEM_mVenus_YF</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Harvest_viral_particles_and_Transduction_of_HT1080_cells"><SPAN class="tocnumber">2.8</SPAN><SPAN class="toctext">Harvest viral particles and Transduction of HT1080 cells</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#New_ladder_mix_and_testing"><SPAN class="tocnumber">2.9</SPAN><SPAN class="toctext">New ladder mix and testing</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#SDM_of_CD_NgoMIV"><SPAN class="tocnumber">2.10</SPAN><SPAN class="toctext">SDM of CD NgoMIV</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Picking_clones_of_pSB1C3_affibody_and_pSB1C3_Rep68"><SPAN class="tocnumber">2.11</SPAN><SPAN class="toctext">Picking clones of pSB1C3_affibody and pSB1C3_Rep68</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#95._labday_20.08.2010"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">95. labday 20.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Sequence_analysis_of_sequencing_reads_prepared_yesterday_.2819.08.2010.29"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Sequence analysis of sequencing reads prepared yesterday (19.08.2010)</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#Continuation:_pSB1C3_SDM_SspI_Bla14FM_clone_5_SDM_.28P222.29_3"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Continuation: pSB1C3_SDM_SspI_Bla14FM clone 5 SDM (P222)</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Subcloning_cap_from_pMA_RepCap_into_pAAV_RC_which_contains_four_mutations_and_the_synthesized_Rep_part"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Subcloning cap from pMA_RepCap into pAAV_RC which contains four mutations and the synthesized Rep part</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Mini-Preps_of_pSB1C3_Rep68_ex_and_pSB1C3_ZEGFR:1907"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Mini-Preps of pSB1C3_Rep68_ex and pSB1C3_ZEGFR:1907</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Hybridisation_of_the_oligos_for_NLS_and_cloning_NLS_into_pSB1C3"><SPAN class="tocnumber">3.5</SPAN><SPAN class="toctext">Hybridisation of the oligos for NLS and cloning NLS into pSB1C3</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#Seeding_HT1080_cells"><SPAN class="tocnumber">3.6</SPAN><SPAN class="toctext">Seeding HT1080 cells</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#PCR_and_ligation_for_biobrick-production_of_pCerulean"><SPAN class="tocnumber">3.7</SPAN><SPAN class="toctext">PCR and ligation for biobrick-production of pCerulean</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#Biobrick_production_of_VP1_up"><SPAN class="tocnumber">3.8</SPAN><SPAN class="toctext">Biobrick production of VP1_up</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#BioBrick_assembly_for_pSB1C3_Affibody_Linker_and_pSB1C3_YFP_His-Tag"><SPAN class="tocnumber">3.9</SPAN><SPAN class="toctext">BioBrick assembly for pSB1C3_Affibody_Linker and pSB1C3_YFP_His-Tag</SPAN></A></LI><LI class="toclevel-1 tocsection-30"><A href="#96._labday_21.08.2010"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">96. labday 21.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#Transfection_of_AAV293.2C_Harvest_viral_particles.2C_Transduction_of_HT1080_and_Fluorescent_microscopy"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Transfection of AAV293, Harvest viral particles, Transduction of HT1080 and Fluorescent microscopy</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#Midi-Prep_of_pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR_clone1"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Midi-Prep of pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR clone1</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#Re-Trafo_of_pSB1C3_SDM_SspI_Bla14FM"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Re-Trafo of pSB1C3_SDM_SspI_Bla14FM</SPAN></A></LI><LI class="toclevel-1 tocsection-34"><A href="#97._labday_22.08.2010"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">97. labday 22.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-35"><A href="#Test-digestion_of_pCerulean_.28P273.2C4.2C5.29.2C_pSB1C3_VP1up_.28P279.2C_280.29.2C_pSB1C3_NLS_.28P281.2C282.29.2C_pAAV_RC_final_.28P283.29"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Test-digestion of pCerulean (P273,4,5), pSB1C3_VP1up (P279, 280), pSB1C3_NLS (P281,282), pAAV_RC_final (P283)</SPAN></A></LI><LI class="toclevel-2 tocsection-36"><A href="#Mini-Prep_of_pCerulean"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Mini-Prep of pCerulean</SPAN></A></LI><LI class="toclevel-1 tocsection-37"><A href="#98._labday_23.08.2010"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">98. labday 23.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-38"><A href="#Primer_design:_Sequencing_primer_for_BioBrick_Assembly_of_vectorplasmid_and_phTERT_qPCR_primer"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Primer design: Sequencing primer for BioBrick Assembly of vectorplasmid and phTERT qPCR primer</SPAN></A></LI><LI class="toclevel-2 tocsection-39"><A href="#Mini-Prep_of_pSB1C3_CFP_middlelinker_and_pSB1C3_ZEGFR:1907_.2F_testdigestion_of_pSB1C3_CFP_middlelinker_and_pSB1C3_lITR_pHTERT"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Mini-Prep of pSB1C3_CFP_middlelinker and pSB1C3_ZEGFR:1907 / testdigestion of pSB1C3_CFP_middlelinker and pSB1C3_lITR_pHTERT</SPAN></A></LI><LI class="toclevel-2 tocsection-40"><A href="#Continuation_of_BioBrick_assembly_for_pSB1C3_Affibody_Linker"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">Continuation of BioBrick assembly for pSB1C3_Affibody_Linker</SPAN></A></LI><LI class="toclevel-2 tocsection-41"><A href="#Cloning_of_pSB1C3_.C3.9F-globin_YFP_and_pSB1C3_hGH_rITR"><SPAN class="tocnumber">6.4</SPAN><SPAN class="toctext">Cloning of pSB1C3_ß-globin_YFP and pSB1C3_hGH_rITR</SPAN></A></LI><LI class="toclevel-2 tocsection-42"><A href="#Repetition:_Quickchange_site-directed_mutagenesis_of_pSB1C3_SDM_SspI_Bla14FM"><SPAN class="tocnumber">6.5</SPAN><SPAN class="toctext">Repetition: Quickchange site-directed mutagenesis of pSB1C3_SDM_SspI_Bla14FM</SPAN></A></LI><LI class="toclevel-2 tocsection-43"><A href="#FACS_analysis.2C_Seeding_HT1080_and_Hek293_cells"><SPAN class="tocnumber">6.6</SPAN><SPAN class="toctext">FACS analysis, Seeding HT1080 and Hek293 cells</SPAN></A></LI><LI class="toclevel-2 tocsection-44"><A href="#Miniprep_of_pSBC13_Bla"><SPAN class="tocnumber">6.7</SPAN><SPAN class="toctext">Miniprep of pSBC13_Bla</SPAN></A></LI><LI class="toclevel-1 tocsection-45"><A href="#99._labday_24.08.2010"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">99. labday 24.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-46"><A href="#Sequence_analysis_of_pSB1C3_CFP_middlelinker"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Sequence analysis of pSB1C3_CFP_middlelinker</SPAN></A></LI><LI class="toclevel-2 tocsection-47"><A href="#Harvest_viral_particles.2C_Transduction_of_HT1080"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">Harvest viral particles, Transduction of HT1080</SPAN></A></LI><LI class="toclevel-2 tocsection-48"><A href="#Mini-prep_of_SDM_NgoMIV_CD"><SPAN class="tocnumber">7.3</SPAN><SPAN class="toctext">Mini-prep of SDM NgoMIV CD</SPAN></A></LI><LI class="toclevel-2 tocsection-49"><A href="#Sent_for_sequencing:_pSB1C3_6xHis"><SPAN class="tocnumber">7.4</SPAN><SPAN class="toctext">Sent for sequencing: pSB1C3_6xHis</SPAN></A></LI><LI class="toclevel-2 tocsection-50"><A href="#Subcloning_Cap_into_pAAV_RC_ins-rep"><SPAN class="tocnumber">7.5</SPAN><SPAN class="toctext">Subcloning Cap into pAAV_RC_ins-rep</SPAN></A></LI><LI class="toclevel-2 tocsection-51"><A href="#Test_digestion_of_pCerulean_.28p273.29"><SPAN class="tocnumber">7.6</SPAN><SPAN class="toctext">Test digestion of pCerulean (p273)</SPAN></A></LI><LI class="toclevel-2 tocsection-52"><A href="#Cloning_of_VP1up_into_PCerulean"><SPAN class="tocnumber">7.7</SPAN><SPAN class="toctext">Cloning of VP1up into PCerulean</SPAN></A></LI><LI class="toclevel-2 tocsection-53"><A href="#Sequence_analysis_of_sequencing_reads_prepared_yesterday_.2823.08.2010.29"><SPAN class="tocnumber">7.8</SPAN><SPAN class="toctext">Sequence analysis of sequencing reads prepared yesterday (23.08.2010)</SPAN></A></LI><LI class="toclevel-2 tocsection-54"><A href="#Test_digestion_of_SDM_NgoMIV_CD"><SPAN class="tocnumber">7.9</SPAN><SPAN class="toctext">Test digestion of SDM NgoMIV CD</SPAN></A></LI><LI class="toclevel-2 tocsection-55"><A href="#Picking_clones_of_pSB1C3_Affibody_linker_and_pSB1C3_.C3.9F-Globin_YFP_hGH_rITR"><SPAN class="tocnumber">7.10</SPAN><SPAN class="toctext">Picking clones of pSB1C3_Affibody_linker and pSB1C3_ß-Globin_YFP_hGH_rITR</SPAN></A></LI><LI class="toclevel-2 tocsection-56"><A href="#Test_digestion_of_pSB1C3_SDM_SspI_Bla14FM_.28P222.29.2C_pSB1C3_BLA_.28P286.2C_P288.29"><SPAN class="tocnumber">7.11</SPAN><SPAN class="toctext">Test digestion of pSB1C3_SDM_SspI_Bla14FM (P222), pSB1C3_BLA (P286, P288)</SPAN></A></LI><LI class="toclevel-2 tocsection-57"><A href="#Test_digestion_of_SDM_NgoMIV_CD_2"><SPAN class="tocnumber">7.12</SPAN><SPAN class="toctext">Test digestion of SDM NgoMIV CD</SPAN></A></LI><LI class="toclevel-2 tocsection-58"><A href="#Picking_clones_of_pSB1C3_Affibody_linker_and_pSB1C3_.C3.9F-Globin_YFP_hGH_rITR_2"><SPAN class="tocnumber">7.13</SPAN><SPAN class="toctext">Picking clones of pSB1C3_Affibody_linker and pSB1C3_ß-Globin_YFP_hGH_rITR</SPAN></A></LI><LI class="toclevel-2 tocsection-59"><A href="#Test_digestion_of_pSB1C3_SDM_SspI_Bla14FM_.28P222.29.2C_pSB1C3_BLA_.28P286.2C_P288.29_2"><SPAN class="tocnumber">7.14</SPAN><SPAN class="toctext">Test digestion of pSB1C3_SDM_SspI_Bla14FM (P222), pSB1C3_BLA (P286, P288)</SPAN></A></LI><LI class="toclevel-1 tocsection-60"><A href="#100._labday_25.08.2010"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">100. labday 25.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-61"><A href="#Cloning_of_pSB1C3_Rep40.2C_pSB1C3_Rep68_and_pMA_RC_insert"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">Cloning of pSB1C3_Rep40, pSB1C3_Rep68 and pMA_RC_insert</SPAN></A></LI><LI class="toclevel-2 tocsection-62"><A href="#Picking_clones_of_pAAV_RC-ins_rep_cap_and_pCeruleanVP1up"><SPAN class="tocnumber">8.2</SPAN><SPAN class="toctext">Picking clones of pAAV_RC-ins_rep_cap and pCeruleanVP1up</SPAN></A></LI><LI class="toclevel-2 tocsection-63"><A href="#Cloning_of_pSB1C3_6xHis_with_middlelinker_and_pSB1C3_6xHis_with_pGA14_mVenus_YFP"><SPAN class="tocnumber">8.3</SPAN><SPAN class="toctext">Cloning of pSB1C3_6xHis with middlelinker and pSB1C3_6xHis with pGA14_mVenus_YFP</SPAN></A></LI><LI class="toclevel-2 tocsection-64"><A href="#Sequence_analysis_of_pSB1C3_6xHis"><SPAN class="tocnumber">8.4</SPAN><SPAN class="toctext">Sequence analysis of pSB1C3_6xHis</SPAN></A></LI><LI class="toclevel-2 tocsection-65"><A href="#Inoculation_of_pGA14_middlelinker_and_pSB1C3_6xHis_clone1"><SPAN class="tocnumber">8.5</SPAN><SPAN class="toctext">Inoculation of pGA14_middlelinker and pSB1C3_6xHis clone1</SPAN></A></LI><LI class="toclevel-2 tocsection-66"><A href="#Cloning_of_pSB1C3_SspI_BLA_and_pSB1C3_CFP_PvuII"><SPAN class="tocnumber">8.6</SPAN><SPAN class="toctext">Cloning of pSB1C3_SspI_BLA and pSB1C3_CFP_PvuII</SPAN></A></LI><LI class="toclevel-2 tocsection-67"><A href="#Mini-Preps_and_test_digestion_of_pSB1C3_Affibody_Middle-Linker.2C_pSB1C3_Affibody_Short-Linker.2C_pSB1C3_Affibody_SEG-Linker.2C_pSB1C3_Affibody_Long-Linker_and_pSB1C3_betaglobin_mVenus_hGH_rITR"><SPAN class="tocnumber">8.7</SPAN><SPAN class="toctext">Mini-Preps and test digestion of pSB1C3_Affibody_Middle-Linker, pSB1C3_Affibody_Short-Linker, pSB1C3_Affibody_SEG-Linker, pSB1C3_Affibody_Long-Linker and pSB1C3_betaglobin_mVenus_hGH_rITR</SPAN></A></LI><LI class="toclevel-2 tocsection-68"><A href="#Sequenzing_evaluation_of_SDM_NgoMIV"><SPAN class="tocnumber">8.8</SPAN><SPAN class="toctext">Sequenzing evaluation of SDM NgoMIV</SPAN></A></LI><LI class="toclevel-1 tocsection-69"><A href="#101._labday_26.08.2010"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">101. labday 26.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-70"><A href="#Mini-Preps_of_pGA14_middle_linker_and_pSB1C3_6xHis_clone_1"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">Mini-Preps of pGA14_middle linker and pSB1C3_6xHis clone 1</SPAN></A></LI><LI class="toclevel-2 tocsection-71"><A href="#Mini-Preps_and_test_digestion_of_pAAV_RC-ins_rep_cap.2C_pCeruleanVP1up_and_pAAV_RC_ins-rep-cap."><SPAN class="tocnumber">9.2</SPAN><SPAN class="toctext">Mini-Preps and test digestion of pAAV_RC-ins_rep_cap, pCeruleanVP1up and pAAV_RC_ins-rep-cap.</SPAN></A></LI><LI class="toclevel-1 tocsection-72"><A href="#102._labday_27.08.2010"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">102. labday 27.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-73"><A href="#Sequence_analysis_of_pSB1C3_Bla_final_.28P309.29and_pCerulean_VP1up_.28P303.29"><SPAN class="tocnumber">10.1</SPAN><SPAN class="toctext">Sequence analysis of pSB1C3_Bla_final (P309)and pCerulean_VP1up (P303)</SPAN></A></LI><LI class="toclevel-2 tocsection-74"><A href="#Hybridisation_and_cloning_of_loop_insertions"><SPAN class="tocnumber">10.2</SPAN><SPAN class="toctext">Hybridisation and cloning of loop insertions</SPAN></A></LI><LI class="toclevel-2 tocsection-75"><A href="#Cloning_of_pCerulean_VP1up_NLS"><SPAN class="tocnumber">10.3</SPAN><SPAN class="toctext">Cloning of pCerulean_VP1up_NLS</SPAN></A></LI><LI class="toclevel-2 tocsection-76"><A href="#DARPin_E_01"><SPAN class="tocnumber">10.4</SPAN><SPAN class="toctext">DARPin E_01</SPAN></A></LI><LI class="toclevel-2 tocsection-77"><A href="#Update_VP1_insertion"><SPAN class="tocnumber">10.5</SPAN><SPAN class="toctext">Update VP1 insertion</SPAN></A></LI><LI class="toclevel-2 tocsection-78"><A href="#Sequence_analysis_of_pSB1C3_BLA_final_.28P309.29"><SPAN class="tocnumber">10.6</SPAN><SPAN class="toctext">Sequence analysis of pSB1C3_BLA_final (P309)</SPAN></A></LI><LI class="toclevel-2 tocsection-79"><A href="#Mini-Preps_and_test_digestion_of_pSB1C3_6xHis_middlelinker_and_pSB1C3_6xHis_mVenus"><SPAN class="tocnumber">10.7</SPAN><SPAN class="toctext">Mini-Preps and test digestion of pSB1C3_6xHis_middlelinker and pSB1C3_6xHis_mVenus</SPAN></A></LI><LI class="toclevel-2 tocsection-80"><A href="#Repetition_of_test_digestion_of_pAAV_RC_ins-rep-cap.2C_pAAV_RC_ins-rep.2C_pSB1C3_Rep40_RC_Insert_and_pSB1C3_Rep68_RC_Insert."><SPAN class="tocnumber">10.8</SPAN><SPAN class="toctext">Repetition of test digestion of pAAV_RC_ins-rep-cap, pAAV_RC_ins-rep, pSB1C3_Rep40_RC_Insert and pSB1C3_Rep68_RC_Insert.</SPAN></A></LI><LI class="toclevel-2 tocsection-81"><A href="#Midi-Prep_of_pSB1C3_VCK_Bla"><SPAN class="tocnumber">10.9</SPAN><SPAN class="toctext">Midi-Prep of pSB1C3_VCK_Bla</SPAN></A></LI><LI class="toclevel-1 tocsection-82"><A href="#103._labday_28.08.2010"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">103. labday 28.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-83"><A href="#Mini-Prep_of_pSB1C3_NLS"><SPAN class="tocnumber">11.1</SPAN><SPAN class="toctext">Mini-Prep of pSB1C3_NLS</SPAN></A></LI><LI class="toclevel-2 tocsection-84"><A href="#Cell_culture"><SPAN class="tocnumber">11.2</SPAN><SPAN class="toctext">Cell culture</SPAN></A></LI><LI class="toclevel-3 tocsection-85"><A href="#Harvest_viral_Stock_and_Transductiot_of_HT_and_A431_cells"><SPAN class="tocnumber">11.2.1</SPAN><SPAN class="toctext">Harvest viral Stock and Transductiot of HT and A431 cells</SPAN></A></LI><LI class="toclevel-3 tocsection-86"><A href="#Seeding_AAV293_for_Transfection_at_30.8"><SPAN class="tocnumber">11.2.2</SPAN><SPAN class="toctext">Seeding AAV293 for Transfection at 30.8</SPAN></A></LI><LI class="toclevel-2 tocsection-87"><A href="#Picking_clones_from_the_ViralBricks"><SPAN class="tocnumber">11.3</SPAN><SPAN class="toctext">Picking clones from the ViralBricks</SPAN></A></LI><LI class="toclevel-1 tocsection-88"><A href="#104._labday_29.08.2010"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">104. labday 29.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-89"><A href="#P282.2FB234_mini-prep"><SPAN class="tocnumber">12.1</SPAN><SPAN class="toctext">P282/B234 mini-prep</SPAN></A></LI><LI class="toclevel-2 tocsection-90"><A href="#Digestion_of_pSB1C3_NLS_.28P324.29"><SPAN class="tocnumber">12.2</SPAN><SPAN class="toctext">Digestion of pSB1C3_NLS (P324)</SPAN></A></LI><LI class="toclevel-2 tocsection-91"><A href="#Harvesting_cultures_from_the_ViralBricks"><SPAN class="tocnumber">12.3</SPAN><SPAN class="toctext">Harvesting cultures from the ViralBricks</SPAN></A></LI><LI class="toclevel-1 tocsection-92"><A href="#105._labday_30.08.2010"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">105. labday 30.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-93"><A href="#BioBrick_assembly_of_pSB1C3_leftITR_pTERT_.26_pSB1C3_.C3.9F-globin_YFP_hGH_rITR"><SPAN class="tocnumber">13.1</SPAN><SPAN class="toctext">BioBrick assembly of pSB1C3_leftITR_pTERT &amp; pSB1C3_ß-globin_YFP_hGH_rITR</SPAN></A></LI><LI class="toclevel-2 tocsection-94"><A href="#Continuation:_Digestion_of_pSB1C3_NLS_.28P324.29"><SPAN class="tocnumber">13.2</SPAN><SPAN class="toctext">Continuation: Digestion of pSB1C3_NLS (P324)</SPAN></A></LI><LI class="toclevel-2 tocsection-95"><A href="#CD_biobrick_production"><SPAN class="tocnumber">13.3</SPAN><SPAN class="toctext">CD biobrick production</SPAN></A></LI><LI class="toclevel-2 tocsection-96"><A href="#Repetition_of_test_digestion_of_pSB1C3_Rep40_RC_Insert_and_pSB1C3_Rep68_RC_Insert."><SPAN class="tocnumber">13.4</SPAN><SPAN class="toctext">Repetition of test digestion of pSB1C3_Rep40_RC_Insert and pSB1C3_Rep68_RC_Insert.</SPAN></A></LI><LI class="toclevel-2 tocsection-97"><A href="#Midi-Prep_of_pAAV_RC_insertparts_clone1"><SPAN class="tocnumber">13.5</SPAN><SPAN class="toctext">Midi-Prep of pAAV_RC_insertparts clone1</SPAN></A></LI><LI class="toclevel-2 tocsection-98"><A href="#FACS_analysis.2C_Seeding_HT1080_and_AAV293"><SPAN class="tocnumber">13.6</SPAN><SPAN class="toctext">FACS analysis, Seeding HT1080 and AAV293</SPAN></A></LI><LI class="toclevel-2 tocsection-99"><A href="#Mini-Preps_and_test_digestion_of_Loop_insertion_BioBricks"><SPAN class="tocnumber">13.7</SPAN><SPAN class="toctext">Mini-Preps and test digestion of Loop insertion BioBricks</SPAN></A></LI><LI class="toclevel-2 tocsection-100"><A href="#Repetition_of_subcloning_Cap-insert_into_pAAV_RC"><SPAN class="tocnumber">13.8</SPAN><SPAN class="toctext">Repetition of subcloning Cap-insert into pAAV_RC</SPAN></A></LI><LI class="toclevel-1 tocsection-101"><A href="#106._labday_31.08.2010"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">106. labday 31.08.2010</SPAN></A></LI><LI class="toclevel-2 tocsection-102"><A href="#Cloning_ZEGFR:1907_and_6xHis_middlelinker_into_pCerulean"><SPAN class="tocnumber">14.1</SPAN><SPAN class="toctext">Cloning ZEGFR:1907 and 6xHis_middlelinker into pCerulean</SPAN></A></LI><LI class="toclevel-2 tocsection-103"><A href="#PCR_of_SV_40_terminator"><SPAN class="tocnumber">14.2</SPAN><SPAN class="toctext">PCR of SV 40 terminator</SPAN></A></LI><LI class="toclevel-2 tocsection-104"><A href="#Cloning_of_NLS_into_pCerulean_VP1up"><SPAN class="tocnumber">14.3</SPAN><SPAN class="toctext">Cloning of NLS into pCerulean_VP1up</SPAN></A></LI><LI class="toclevel-2 tocsection-105"><A href="#Cloning_CFP_middlelinker_.28from_pSB1C3_CFP_middlelinker.2C_P276.29_into_pCerulean_.28P273.29"><SPAN class="tocnumber">14.4</SPAN><SPAN class="toctext">Cloning CFP_middlelinker (from pSB1C3_CFP_middlelinker, P276) into pCerulean (P273)</SPAN></A></LI><LI class="toclevel-2 tocsection-106"><A href="#Sequencing_results_of_pSB1C3_6xHis_middlelinker_and_pSB1C3_6xHis_mVenus"><SPAN class="tocnumber">14.5</SPAN><SPAN class="toctext">Sequencing results of pSB1C3_6xHis_middlelinker and pSB1C3_6xHis_mVenus</SPAN></A></LI><LI class="toclevel-2 tocsection-107"><A href="#Contiuation:_Repetition_of_subcloning_Cap-insert_into_pAAV_RC"><SPAN class="tocnumber">14.6</SPAN><SPAN class="toctext">Contiuation: Repetition of subcloning Cap-insert into pAAV_RC</SPAN></A></LI><LI class="toclevel-2 tocsection-108"><A href="#2nd_Repetition_of_test_digestion_of_Loop_insertion_BioBricks"><SPAN class="tocnumber">14.7</SPAN><SPAN class="toctext">2nd Repetition of test digestion of Loop insertion BioBricks</SPAN></A></LI><LI class="toclevel-2 tocsection-109"><A href="#Midi-Prep_of_pAAV_RC_.26_pHelper"><SPAN class="tocnumber">14.8</SPAN><SPAN class="toctext">Midi-Prep of pAAV_RC &amp; pHelper</SPAN></A></LI><LI class="toclevel-2 tocsection-110"><A href="#Transformation_of_Biobrick_CD"><SPAN class="tocnumber">14.9</SPAN><SPAN class="toctext">Transformation of Biobrick CD</SPAN></A></LI></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="93._labday_18.08.2010"><P style="font-size:17px; background-color:#00dd77;">93. labday 18.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Continuation:_pSB1C3_SDM_SspI_Bla14FM_clone_5_SDM_.28P222.29"><P style="font-size:15px; background-color:#66bbff;"><B>Continuation: pSB1C3_SDM_SspI_Bla14FM clone 5 SDM (P222)</B></P></SPAN></H4><P><B>Investigator: Patrick</B></P><P>There is a lawn on the plate although chloramphenicol was added. Some clones were picked and a new agar plate inoculated to hopefully receive some 'pickable' clones.
</P><P>Additionally a new SDM was performed:
PCR program:
</P><UL><LI>95°C 2 min (1x)
</LI><LI>95°C 20s, 60°C 10 s, 68°C 30 s (30s/kb) (18x)
</LI><LI>68°C 5 minutes
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD><B>Volume</B></TD></TR><TR><TD> 10x reaction buffer </TD><TD>2,5
</TD></TR><TR><TD> DNA template ( about 10 ng)</TD><TD> 1 µl diluted P222
</TD></TR><TR><TD> forward primer: </TD><TD> 0,58 µl O108 (pSB1C3 PvuII rev)
</TD></TR><TR><TD> reverse primer: </TD><TD> 0,59 µl O109 (pSB1C3 PvuII for)
</TD></TR><TR><TD> DMSO (2%)</TD><TD> 0,5
</TD></TR><TR><TD> dNTP Mix from the kit</TD><TD> 0,5 µl
</TD></TR><TR><TD> QuickSolution Reagent</TD><TD>0,75 µl
</TD></TR><TR><TD> Quickchange Lightning Enzyme (1.25U)</TD><TD>0,5 µl
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD> 18,08 µl
</TD></TR><TR><TD>Total volume</TD><TD>25 µl
</TD></TR></TBODY></TABLE><P>All further steps including the transformation were performed according to the standard protocol.
This time 2 agar plates were inoculated. One of them with the resuspended pellet and the other one with a 1:100 dilution of the resuspended pellet.
</P><H4><SPAN class="mw-headline" id="Mini_Preps_pEGFP_C1.2CpSB1C3_Rep40.2CpSB1C3_Rep78.2CpMA_T_Affibody.2CpSB1C3_SspIdel_BLA_clone_2.2B3.2CpSB1C3_AAP.2C_pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR.2CpSB1C3_lITR_CMV_mVenus_hGH_rITR.2CpSK_CD_SDMPstI"><P style="font-size:15px; background-color:#66bbff;"><B>Mini Preps pEGFP_C1,pSB1C3_Rep40,pSB1C3_Rep78,pMA_T_Affibody,pSB1C3_SspIdel_BLA clone 2+3,pSB1C3_AAP, pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR,pSB1C3_lITR_CMV_mVenus_hGH_rITR,pSK_CD_SDMPstI</B></P></SPAN></H4><P><B>Investigator: Jessica</B>
Preps and test digestions were made:<B>pSK_CD_SDMPstI</B></P><UL><LI>Given Plasmid-Number: P248; DNA concentration: 302,7 ng/µL ;</LI><LI>Given Plasmid-Number: P249; DNA concentration: 267,0 ng/µL ;</LI></UL><UL><LI><B>P248</B>, <B>P249</B> and <B>P223</B> as control digested with PstI  --&gt; no satisfying result,control was not cut but samples two times... sent for sequencing with primer: GATC_std_T7minus1 </LI></UL><P><B>pSB1C3_Rep40</B></P><UL><LI>Given Plasmid-Number: P231 (clone3); DNA concentration: 234,00 ng/µL</LI><LI>Given Plasmid-Number: P232 (clone2); DNA concentration: 234,96 ng/µL</LI><LI>Given Plasmid-Number: P233 (clone1); DNA concentration: 237,12 ng/µL</LI></UL><P><B>pSB1C3_Rep78</B></P><UL><LI>Given Plasmid-Number: P234 (clone2); DNA concentration: 277,56 ng/µL</LI><LI>Given Plasmid-Number: P235 (clone1); DNA concentration: 294,19 ng/µL</LI><LI>Given Plasmid-Number: P236 (clone3); DNA concentration: 378,87 ng/µL</LI></UL><P><B>pSB1C3_AAP</B></P><UL><LI>Given Plasmid-Number: P239 (clone1); DNA concentration: 219,82 ng/µL</LI><LI>Given Plasmid-Number: P240 (clone2); DNA concentration: 220,45 ng/µL</LI><LI>Given Plasmid-Number: P241 (clone3); DNA concentration: 209,57 ng/µL</LI></UL><P><B>has to be edited</B></P><P>@Jessica: What are these plasmid numbers?? (Bea)
</P><UL><LI>Given Plasmid-Number: P230; DNA concentration: 367,36 ng/µL ;</LI><LI>Given Plasmid-Number: P242; DNA concentration: 202,09 ng/µL ;</LI><LI>Given Plasmid-Number: P243; DNA concentration: 369,34 ng/µL ;</LI><LI>Given Plasmid-Number: P245; DNA concentration: 207,12 ng/µL ;</LI><LI>Given Plasmid-Number: P246; DNA concentration: 198,35 ng/µL ;</LI><LI>Given Plasmid-Number: P247; DNA concentration: 212,22 ng/µL ;</LI></UL><P>Test digestion</P><UL><LI>buffer used: 4 ; Restriction-enzymes used: Enzyme XbaI ; Enzyme SpeI </LI><LI>Plasmid
<TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>P231-233 Volume/µL</B></TD><TD align="left"><B>P234 Volume/µL</B></TD><TD align="left"><B>P235 Volume/µL</B></TD><TD align="left"><B>236 Volume/µL</B></TD><TD align="left"><B>P239-242 Volume/µL</B></TD></TR><TR><TD align="left"> DNA </TD><TD align="left"> 4 </TD><TD align="left"> 3.5 </TD><TD align="left"> 3 </TD><TD align="left"> 2.5 </TD><TD align="left"> 4
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 2 </TD><TD align="left"> 2 </TD><TD align="left"> 2 </TD><TD align="left"> 2 </TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> Buffer 4 (10x) </TD><TD align="left"> 2 </TD><TD align="left"> 2 </TD><TD align="left"> 2 </TD><TD align="left"> 2 </TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> SpeI </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5
</TD></TR><TR><TD align="left"> XbaI </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 11 </TD><TD align="left"> 11,5 </TD><TD align="left"> 12 </TD><TD align="left"> 12.5 </TD><TD align="left"> 11
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>20</B></TD></TR></TBODY></TABLE><P style="font-size:13px; color:#003399;"><B>Comment</B>: Test digestion of P231-233 and P239-242 contained equal volumes of ingredients. Therefore the charts were simplyfied. </P><P style="font-size:13px; color:#003399;"><B>Comment</B>: P231 and P239 will be sent for sequenzing tomorrow. We picked for additional clones of Rep78 for Mini-Prep and test digestion tomorrow.</P></LI></UL><H4><SPAN class="mw-headline" id="Continuation_of_Repetition:_Biobrick_production_of_Rep_68_.2817.08.2010.29"><P style="font-size:15px; background-color:#66bbff;"><B>Continuation of Repetition: Biobrick production of Rep 68 (17.08.2010)</B></P></SPAN></H4><P><B>Investigator: Christian L. , Stefan</B></P><P> 	
Aim of the experiment:
We want to produce biobricks from Rep 68.
</P><UL><LI>Vector: name: pSB1C3_CFP <B>P51.1</B></LI><LI>Insert: name: Rep68_SDM_PstI clone 1 <B>P224</B></LI></UL><UL><LI>buffer used: 4
</LI><LI>Restriction-enzymes used:
</LI><LI>XbaI
</LI><LI>SpeI
</LI></UL><LI>DNA concentration (P51.1): 132,1 ng/µl
</LI><LI>DNA concentration (P224): 350.29 ng/µl
</LI><P><B>Digestion of plasmid backbone and PCR product:</B></P><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>volume of pSB1C3_CFP /µl</B></TD><TD align="right"><B>volume of Rep68_SDM_PstI clone 1 /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">10 </TD><TD align="right">29
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right"> 4
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right"> 4
</TD></TR><TR><TD>SpeI </TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>XbaI </TD><TD align="right">1 </TD><TD align="right">1
</TD></TR><TR><TD>H2O</TD><TD align="right">4 </TD><TD align="right">1
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 20</TD><TD align="right">40
</TD></TR></TBODY></TABLE><P>
Incubation at 37°C, 2 h
<B>Dephoshorylation of Vector pSB1C3_CFP after digestion:</B></P><UL><LI>add 2 µl of 10x Antarctic Phopshatase Reaction Buffer to DNA cut with XbaI and SpeI
</LI><LI>add 1 µl of Antarctic Phosphatase and mix
</LI><LI>incubate for 60 minutes at 37°C</LI></UL><P><B>Gelextration of pSB1C3</B>
Gelextration of pSB1C3 was performed according to standard protocol

0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 115 Volt, running time:45 </P><P><B>PCR Purification</B>
Purification of Rep68 was performed according to the standard protocol

Concentration measured: 16,4 ng/µl
<B>Ligation</B>
For the ligation 1µl T4 buffer (10x) and 1µl T4 ligase were used.
</P><TABLE border="1"><TBODY><TR><TD align="right"><B>vector /µl</B></TD><TD align="right"><B>insert /µl</B></TD></TR><TR><TD> pSB1C3 + Rep68 </TD><TD align="right"> 3,9 </TD><TD align="right"> 4,1
</TD></TR></TBODY></TABLE><P>
Incubation at room temperature for 45 minutes.
<B>Transformation:</B>
The transformation was done following the standard protocol using XL1b cells.
</P><H4><SPAN class="mw-headline" id="Transfection_of_AAV293_cells"><P style="font-size:15px; background-color:#66bbff;"><B>Transfection of AAV293 cells</B></P></SPAN></H4><P><B>Investigator: Adrian, Kerstin</B></P><UL><LI> 2x10<SUP>6</SUP> cells were seeded (instead of 3x10<SUP>6</SUP>) --&gt; hopefully better virus-production
</LI><LI> used plasmids: 
</LI><LI>P50   c= 429 ng/µl (RC)
</LI><LI>P64   c= 500 ng/µl (pHelper)
</LI><LI>P229  c= 162 ng/µl (GOI=YFP)
</LI><LI>P228  c= 540 ng/µl (GOI=eGFP)
</LI><LI>P158a c= 1800ng/µl (RC mutant)
</LI><LI>P158a c= 1770ng/µl (RC mutant) 
</LI></UL><UL><LI><B> 1. plate:</B> Lipo-Transfection
</LI><LI><B> 2. plate:</B> Lipo-Transfection
</LI></UL><UL><LI><B> 3. plate:</B> 3,3 µg (RC), 3,3 µg (pHelper), 3,3 µg (YFP)
</LI><LI><B> 4. plate:</B> 10 µg (RC), 10 µg (pHelper), 3,3 µg (YFP)
</LI><LI><B> 5. plate:</B> 10 µg (RC mutant P158a), 10 µg (pHelper), 3,3 µg (YFP)
</LI><LI><B> 6. plate:</B> 10 µg (RC mutant P158b), 10 µg (pHelper), 3,3 µg (YFP)
</LI><LI><B> 7. plate:</B> 3,3 µg (RC mutant P158a), 3,3 µg (pHelper), 3,3 µg (YFP)
</LI><LI><B> 8. plate:</B> 3,3 µg (RC mutant P158b), 3,3 µg (pHelper), 3,3 µg (YFP)
</LI><LI><B> 9. plate:</B> 10 µg (RC), 10 µg (pHelper), 3,3 µg (eGFP)
</LI><LI><B> 10. plate:</B> 10 µg (RC), 10 µg (pHelper), 10 µg (eGFP)
</LI><LI><B> 11. plate:</B> 10 µg (RC), 10 µg (pHelper), 20 µg (eGFP)
</LI><LI><B> 12. plate:</B> 10 µg (RC), 10 µg (pHelper), 40 µg (eGFP)
</LI></UL><H4><SPAN class="mw-headline" id="Seeding_Ht1080"><P style="font-size:15px; background-color:#66bbff;"><B>Seeding Ht1080</B></P></SPAN></H4><P><B>Investigators: Adrian, Kerstin</B></P><H4><SPAN class="mw-headline" id="Cloning_of_pMA_T_affibody_and_pSB1C3_CFP"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of pMA_T_affibody and pSB1C3_CFP</B></P></SPAN></H4><P style="font-size:13px; color:#003399;"><B>Comment</B>: pMA_T_affibody (p237) from Retrafo and pSB1C3_CFP (p51.5) were digested with EcoRI and PstI.</P><P><B> Investigator: Anna</B></P><P>Vector: </P><UL><LI>pSB1C3_CFP (p51.1): c= 132,1 ng/µl</LI></UL><P>Insert: </P><UL><LI>pMA_T_affibody (p237): c= 233,96 ng/µl</LI></UL><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">Vector p51.1</TD><TD align="right">Insert p237
</TD></TR><TR><TD> DNA  </TD><TD align="right">9 </TD><TD align="right"> 8
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2</TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right"> 2
</TD></TR><TR><TD>Enzyme EcoRI </TD><TD align="right">1 </TD><TD align="right"> 1
</TD></TR><TR><TD>Enzyme PstI </TD><TD align="right">1</TD><TD align="right"> 1
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">5 </TD><TD align="right"> 6
</TD></TR><TR><TD><B>Total volume </B></TD><TD align="right">20 </TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><UL><LI>Gelextraction of pSB1C3_RFC25_CFP:
</LI></UL><P>0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 120 Volt, running time:55 
4µl loading dye (6x) for the sample, Marker: GeneRuler ladder mix (Fermentas)
</P><UL><LI>c(pSB1C3)= 5,4 ng/µl </LI><LI>c(affibody)= 5,6 ng/µl
</LI></UL><UL><LI>Ligation of PCR products and vector:
</LI></UL><P>For the Ligation 1µl T4 buffer (2x) and 1µl T4 ligase were used. Incubation time: 30 min
</P><TABLE border="1"><TBODY><TR><TD align="right"><B>vector /µl</B></TD><TD align="right"><B>insert /µl</B></TD></TR><TR><TD> pSB1C3 + affibody </TD><TD align="right">5,91 </TD><TD align="right">2,09
</TD></TR></TBODY></TABLE><UL><LI>Transformation:
</LI></UL><P>The transformation was done following the standard protocol using XL1 blue cells.
</P><P style="font-size:13px; color:#003399;"><B>Result</B>: Clones of pSB1C3_affibody will be picked tomorrow.</P><H3><SPAN class="mw-headline" id="94._labday_19.08.2010"><P style="font-size:17px; background-color:#00dd77;">94. labday 19.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Continuation:_pSB1C3_SDM_SspI_Bla14FM_clone_5_SDM_.28P222.29_2"><P style="font-size:15px; background-color:#66bbFF;"><B>Continuation: pSB1C3_SDM_SspI_Bla14FM clone 5 SDM (P222)</B></P></SPAN></H4><P><B>Investigator: Patrick</B>
Again there was a bacterial lawn on the agar plate. Therefore some clones from the overgrown agar plated were picked and diluted. The 1:10^4, 1:10^5, 1:10^6 dilution each were put onto an agar plate (37°C, over night). Additionally a transformation with XL1B cells from the remaining SDM DNA sample was performed and diluted, too. The 1:10^4 and the 1:10^6 dilution were plated. Hopefully there will be some 'pickable' clones tomorrow.
</P><H4><SPAN class="mw-headline" id="Cloning_of_pSB1C3_CFP_and_middlelinker"><P style="font-size:15px; background-color:#66bbFF;"><B>Cloning of pSB1C3_CFP and middlelinker </B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><UL><LI>buffer used: 4; Restriction-enzymes used: Enzyme 1: AgeI ; Enzyme 2: PstI ; Enzyme3: NgoMIV
</LI></UL><UL><LI>Plasmids:</LI><UL><LI>pSB1C3_CFP <B>P51.2</B></LI><LI>pGA14_middle linker <B>P65</B></LI></UL></UL><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>Mastermix</B></TD><TD align="left"><B>P51.2/µL</B></TD><TD align="left"><B>P65/µL</B></TD></TR><TR><TD align="left"> DNA  </TD><TD align="left"> -</TD><TD align="left"> 9,9</TD><TD align="left"> 10,8
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left">-</TD><TD align="left">2</TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> Buffer 4 (10x)  </TD><TD align="left"> -</TD><TD align="left"> 2</TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> Enzyme 1 AgeI   </TD><TD align="left">-</TD><TD align="left"> 1</TD><TD align="left"> -
</TD></TR><TR><TD align="left"> Enzyme 2  PstI   </TD><TD align="left"> -</TD><TD align="left"> 1</TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Enzyme 3  NgoMIV   </TD><TD align="left"> -</TD><TD align="left"> -</TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> -</TD><TD align="left"> 4,1</TD><TD align="left"> 3,2
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"> -</TD><TD align="left"> 20</TD><TD align="left"> 20
</TD></TR></TBODY></TABLE><UL><LI>Incubation: 90 minutes</LI></UL><P><B>Agarosegel</B>
0.5 g Agarose, 50 ml TAE (1 %), 3 µL GELRED, at 115 Volt, running time: 45 minutes
</P><P><B>concentrations</B> measured via NanoDrop:
</P><UL><LI>SB1C3_CFP:  5,0 ng/µl
</LI><LI>middle linker:1,7 ng/µl
</LI></UL><P><B>T4 Ligation</B> 
Volume insert: 1,24 µl
Volume vector: 6,76 µl<B>Trafo</B> was prepared with XL1blue
</P><H4><SPAN class="mw-headline" id="Test_digestion_of_pSB1C3_Rep78"><P style="font-size:15px; background-color:#66bbff;"><B>Test digestion of pSB1C3_Rep78</B></P></SPAN></H4><P><B>Investigator: Chris L., Stefan</B>
Preps and test digestions were made:<B>pSB1C3_Rep78</B></P><UL><LI>Given Plasmid-Number: P252 (clone4); DNA concentration: 352,47 ng/µL</LI><LI>Given Plasmid-Number: P253 (clone5); DNA concentration: 408,23 ng/µL</LI><LI>Given Plasmid-Number: P254 (clone6); DNA concentration: 349,18 ng/µL</LI><LI>Given Plasmid-Number: P255 (clone7); DNA concentration: 320,25 ng/µL</LI></UL><P>Test digestion</P><UL><LI>buffer used: 4 ; Restriction-enzymes used: Enzyme XbaI ; Enzyme SpeI </LI><LI>Plasmid
<TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>P252 Volume/µL</B></TD><TD align="left"><B>P253 Volume/µL</B></TD><TD align="left"><B>P254 Volume/µL</B></TD><TD align="left"><B>255 Volume/µL</B></TD></TR><TR><TD align="left"> DNA </TD><TD align="left"> 2.5 </TD><TD align="left"> 2 </TD><TD align="left"> 2.5 </TD><TD align="left"> 3
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 2 </TD><TD align="left"> 2 </TD><TD align="left"> 2 </TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> Buffer 4 (10x) </TD><TD align="left"> 2 </TD><TD align="left"> 2 </TD><TD align="left"> 2 </TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> SpeI </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5
</TD></TR><TR><TD align="left"> XbaI </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 12.5 </TD><TD align="left"> 13 </TD><TD align="left"> 12.5 </TD><TD align="left"> 12
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>20</B></TD></TR></TBODY></TABLE><P>
Incubation time: 1 h; Incubation temperature: 37°

0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , running time:5 minutes at 90 Volt and 40 minutes at 115 Volt 
4µl loading dye (6x) for the sample, Marker: GeneRuler ladder mix (Fermentas)
</P></LI></UL><H4><SPAN class="mw-headline" id="Sent_for_sequencing"><P style="font-size:15px; background-color:#66bbff;">Sent for sequencing</P></SPAN></H4><P><B>Investigator: Chris L., Stefan</B></P><P>pSB1C3_Rep52 clone3 (P202) sent for sequencing with O162 (VF2).
pSB1C3_Rep40 clone3 (P231), pSB1C3_AAP clone1 (P239), pSB1C3_Rep78 clone4 (P252) and pSB1C3_Rep78 clone7 (255) sent for sequencing with O162 (VF2) and O51 (VR2)
pAAV_RC_ins-rep clone1 (P250) and pAAV_RC_ins-rep clone2 (P251) sent for sequencing with O19 (VP1 primer for pKEX reverse)
</P><H4><SPAN class="mw-headline" id="Test_Digestion_of_P242.2C_P243.2C_P245.2C_P246.2C_P247"><P style="font-size:15px; background-color:#66bbFF;"><B>Test Digestion of P242, P243, P245, P246, P247</B></P></SPAN></H4><P><B>Investigator: Achim</B></P><P>P242: pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR clone1
</P><P>P243: pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR clone2
</P><P>P245: pSB1C3_lITR_CMV_mVenus_hGH_rITR clone1
</P><P>P246: pSB1C3_lITR_CMV_mVenus_hGH_rITR clone2
</P><P>P247: pSB1C3_lITR_CMV_mVenus_hGH_rITR clone3
</P><P>(Left picture: Test digestion from 18.8., right picture: Repetition of test digestion from 19.8.)
</P><P>Fragments sizes: 
</P><UL><LI>P242/243: 1400/3400
</LI><LI>P245/P246/P247: 1400/2900
</LI></UL><P>Bands are diffuse and size estimation is difficult because of strange marker behaviour, but at least the second digestion shows two distinct bands which look like they have the correct sizes. P242 and P245 were sent for sequencing.
</P><H4><SPAN class="mw-headline" id="Test_digestion_of_pAAV_RC-ins-rep"><P style="font-size:15px; background-color:#66bbff;"><B>Test digestion of pAAV_RC-ins-rep</B></P></SPAN></H4><P><B>Investigator: Chris L., Stefan</B>
Preps and test digestions were made:<B>pAAV_RC-ins-rep</B></P><UL><LI>Given Plasmid-Number: P250 (clone1); DNA concentration: 467,5 /µL</LI><LI>Given Plasmid-Number: P251 (clone2); DNA concentration: 108,4 ng/µL</LI></UL><P>Test digestion</P><UL><LI>buffer used: 3 ; Restriction-enzymes used: Enzyme SwaI ; Enzyme BstEII </LI><LI>Plasmid
<TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>P250 Volume/µL</B></TD><TD align="left"><B>P251 Volume/µL</B></TD></TR><TR><TD align="left"> DNA </TD><TD align="left"> 1,7 </TD><TD align="left"> 7.3
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 1 </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Buffer 3 (10x) </TD><TD align="left"> 1 </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> SpeI </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5
</TD></TR><TR><TD align="left"> XbaI </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 5.3 </TD><TD align="left"> -
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>10</B></TD><TD align="left"><B>10.3</B></TD></TR></TBODY></TABLE><P>
Incubation time with SwaI: 60 minutes; Incubation temperature: 25°
Incubation time with BstEII: 75 minutes; Incubation temperature: 60°

0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , running time: 45 minutes at 115 Volt 
4µl loading dye (6x) for the sample, Marker: GeneRuler ladder mix (Fermentas)
</P></LI></UL><H4><SPAN class="mw-headline" id="Midiprep_of_dsAAV_CMV_EGFP_and_pAAV_iGEM_mVenus_YF"><P style="font-size:15px; background-color:#66bbff;"><B>Midiprep of dsAAV_CMV_EGFP and pAAV_iGEM_mVenus_YF </B></P></SPAN></H4><P><B>Investigators: Chris W. </B></P><P style="font-size:13px; color:#003399;"><B>Comment</B>: Midi-Preps of dsAAV_CMV_EGFP (p31, B24) and pAAV_iGEM_mVenus_YFP (p39, B34)</P><P>The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
</P><TABLE border="1"><TBODY><TR><TD> plasmid-no. </TD><TD align="right">p31</TD><TD align="right">p39
</TD></TR><TR><TD> concentration (ng/µl)</TD><TD align="right">2420,46 &amp; 2586,64 </TD><TD align="right">1148,07 &amp; 978,99
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Harvest_viral_particles_and_Transduction_of_HT1080_cells"><P style="font-size:15px; background-color:#66bbff;"><B>Harvest viral particles and Transduction of HT1080 cells </B></P></SPAN></H4><P><B>Investigators: Adrian, Kerstin </B></P><LI><B> Transfectionplan </B> (16.08.2010)


<TABLE border="1"><TBODY><TR><TD> Plate  </TD><TD align="right">amount of cells
</TD></TR><TR><TD> 1</TD><TD align="right">100 000
</TD></TR><TR><TD> 2</TD><TD align="right">200 000
</TD></TR><TR><TD> 3</TD><TD align="right">400 000
</TD></TR><TR><TD> 4</TD><TD align="right">500 000
</TD></TR><TR><TD> 5</TD><TD align="right">800 000
</TD></TR><TR><TD> 6</TD><TD align="right">1 000 000
</TD></TR><TR><TD> 7</TD><TD align="right">1 200 000
</TD></TR><TR><TD> 8</TD><TD align="right">1 500 000
</TD></TR><TR><TD> 9</TD><TD align="right">1 750 000
</TD></TR><TR><TD> 10</TD><TD align="right">1 750 000 not sure
</TD></TR></TBODY></TABLE><P>The cells were transfected with 40µg YFP, 3,3µg Rep/Cap and 3,3µg pHelper.
</P></LI><LI><B> Transduction of HT1080 </B><P style="font-size:13px; color:#66bbff;"><B>Comments: We want to find out, if the efficiency of virus production can be influenced by the amount of transfected cells. </B></P><P>
Plate 1 YFP: 200.000 cells per well</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>165µl virus (1) </TD><TD>165µl virus (2) </TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD> 165µl virus (1) </TD><TD> 165µl virus (2) </TD></TR></TBODY></TABLE><P>Plate 2 YFP: 200.000 cells per well</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>165µl virus (3) </TD><TD>165µl virus (4) </TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD> 165µl virus (3) </TD><TD> 165µl virus (4) </TD></TR></TBODY></TABLE><P>Plate 3 YFP: 200.000 cells per well</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>165µl virus (5) </TD><TD>165µl virus (6) </TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD> 165µl virus (5) </TD><TD> 165µl virus (6) </TD></TR></TBODY></TABLE><P>Plate 4 YFP: 200.000 cells per well</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>165µl virus (7) </TD><TD>165µl virus (8) </TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD> 165µl virus (7) </TD><TD> 165µl virus (8) </TD></TR></TBODY></TABLE><P>Plate 5 YFP: 200.000 cells per well</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>165µl virus (9) </TD><TD>165µl virus (10)</TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD> 165µl virus (9) </TD><TD> 165µl virus (10)</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="New_ladder_mix_and_testing"><P style="font-size:15px; background-color:#66bbFF;"><B>New ladder mix and testing </B></P></SPAN></H4><P><B>Investigator: Jessica</B>
new Generuler ladder mix from Fermentas arrived
one tube (containing 100µl solution) was prepared with 100µl loading dye and 400 µl miliporewater
two test-gels were prepared:
</P></LI><LI>1% TAE agarose gel with samples of</LI><UL><LI> 0,5 µl </LI><LI> 1,0 µl </LI><LI> 2,0 µl</LI><LI>3,0 µl</LI><LI> 4,0 µl </LI><LI> 5,0 µl </LI><LI> 6,0 µl</LI><LI> 7,0 µl</LI><LI> 8,0 µl</LI><LI> 9,0 µl</LI><LI> 10,0 µl</LI><LI> sample P250 </LI></UL><UL><LI>1% TAE agarose gel containing 20% H2O with samples of ( 20% H2O was an idea I have read in a journal but it doesn't seem like a good idea</LI><UL><LI> 0,5 µl </LI><LI> 1,0 µl </LI><LI> 2,0 µl</LI><LI> 3,0 µl</LI><LI> 4,0 µl </LI><LI> 5,0 µl </LI><LI> 6,0 µl</LI><LI> 7,0 µl</LI><LI> 8,0 µl</LI><LI> 9,0 µl</LI><LI> 10,0 µl</LI><LI> sample P251 </LI></UL></UL><P style="font-size:13px; color:#66bbff;"><B>Comments:</B> an example of these laddervolumes is patches at the bench (right up)</P><H4><SPAN class="mw-headline" id="SDM_of_CD_NgoMIV"><P style="font-size:15px; background-color:#66bbFF;"><B>SDM of CD NgoMIV </B></P></SPAN></H4><P><B>Investigator:Kira</B></P><P>Comment: Lightning kit was used for quickchange reaction
</P><TABLE border="1"><TBODY><TR><TD><B>component</B></TD><TD><B>Volume/µl</B></TD></TR><TR><TD> DNA: (1:100) </TD><TD>3,3
</TD></TR><TR><TD> 10x reaction buffer: </TD><TD>2,5
</TD></TR><TR><TD> primer forward NgoMIV (1:10) </TD><TD>0,55
</TD></TR><TR><TD> primer reverse NgoMIV (1:10) </TD><TD>0,55
</TD></TR><TR><TD> dNTP </TD><TD>0,5
</TD></TR><TR><TD>DMSO </TD><TD>0
</TD></TR><TR><TD>QuickSolution Reagent </TD><TD>0,75
</TD></TR><TR><TD>H2O </TD><TD>16,35
</TD></TR><TR><TD>Lightning Polymerase </TD><TD>0,5
</TD></TR></TBODY></TABLE><P>PCR
</P><TABLE border="1"><TBODY><TR><TD><B>segment</B></TD><TD><B>cycles</B></TD><TD><B>temperature</B></TD><TD><B>time</B></TD></TR><TR><TD> 1 </TD><TD>1 </TD><TD>95 </TD><TD>2min
</TD></TR><TR><TD> 2 </TD><TD>18 </TD><TD>95 </TD><TD>20sec
</TD></TR><TR><TD>60 </TD><TD> 10 sec
</TD></TR><TR><TD>68 </TD><TD> 2 min15 sec
</TD></TR><TR><TD> 3 </TD><TD> 68</TD><TD> 5 min
</TD></TR></TBODY></TABLE><P>
sample was digested with 1 ul DpnI for 10 min at 37°C
</P><P>Transformation was performed according to the standard protocol.
</P><H4><SPAN class="mw-headline" id="Picking_clones_of_pSB1C3_affibody_and_pSB1C3_Rep68"><P style="font-size:15px; background-color:#66bbFF;"><B>Picking clones of pSB1C3_affibody and pSB1C3_Rep68</B></P></SPAN></H4><P><B>Investigator: Anna</B></P><TABLE border="1"><TBODY><TR><TD> Plasmid   </TD><TD align="right">Resistance </TD><TD align="right">Bacterial strain
</TD></TR><TR><TD> pSB1C3_affibody°  </TD><TD align="right">Chloramphenicol</TD><TD align="right">XL1 blue
</TD></TR><TR><TD> pSB1C3_Rep68°°  </TD><TD align="right">Chloramphenicol</TD><TD align="right">XL1 blue
</TD></TR></TBODY></TABLE><P>°: 2 clones
°°: 3 clones each</P><P>Clones were inoculated in 10 mL DYT medium containing 10 µL of antibiotics and were put in 37°C room on shaker.</P><H3><SPAN class="mw-headline" id="95._labday_20.08.2010"><P style="font-size:17px; background-color:#00dd77;">95. labday 20.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Sequence_analysis_of_sequencing_reads_prepared_yesterday_.2819.08.2010.29"><P style="font-size:15px; background-color:#66bbff;"><B>Sequence analysis of sequencing reads prepared yesterday (19.08.2010)</B></P></SPAN></H4><P><B>Investigator: Bea </B></P><P style="font-size:13px; color:#cc0033;"><I><B>GENERAL COMMENT:</B> pSB1C3 {Rep 52 and Rep 78{} were sent for sequencing BUT they cannot be used for subcloning the ordered rep protein because in the produced BioBricks the restriction enzyme needed for subcloning (SwaI) is NOT present in the BioBricks pSB1C3_Rep52 and rep78.</I></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: Sequence analysis of pAAV_RC containing the subcloned &quot;rep&quot; (ordered) </P><UL><LI>Primer used: VP1 primer fro pKex</LI><LI>Plasmid sequenced: P250 </LI><LI>Sequence sample: CL-9 </LI><LI>Stored in folder: </LI></UL><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>xy</CAPTION></TABLE><B>Results:</B><P><B>Next steps: xy </B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: Sequence analysis of pAAV_RC containing the subcloned &quot;rep&quot; (ordered) </P><UL><LI>Primer used: VP1 primer fro pKex</LI><LI>Plasmid sequenced: P251 </LI><LI>Sequence sample: CL-10 </LI><LI>Stored in folder: </LI></UL><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>xy</CAPTION></TABLE><B>Results:</B><P><B>Next steps: xy </B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: Sequence analysis of pSB1C3_Rep40 </P><UL><LI>Primer used: VR-2</LI><LI>Plasmid sequenced: P231 </LI><LI>Sequence sample: CL-2 </LI><LI>Stored in folder: DKFZ Rep40ex</LI></UL><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>xy</CAPTION></TABLE><B>Results:</B><P style="color:#00bbff;"> Sequence read is good (prefix is ok) but there was found a mutation in Rep52 protein: G2886A (nucleotides) --&gt; V337I. </P><P><B>Next steps: xy </B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: Sequence analysis of pSB1C3_AAP </P><UL><LI>Primer used: VR-2</LI><LI>Plasmid sequenced: P239 </LI><LI>Sequence sample: CL-3 </LI><LI>Stored in folder: DKFZ AAP</LI></UL><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>xy</CAPTION></TABLE><B>Results:</B><P style="color:#00bbff;"> xy</P><P><B>Next steps: xy </B></P><P><A href="http://www.molbiotech.uni-freiburg.de/iGEM/wiki2010/index.php/Laborjournal" class="external text" rel="nofollow">top of page</A></P><H4><SPAN class="mw-headline" id="Continuation:_pSB1C3_SDM_SspI_Bla14FM_clone_5_SDM_.28P222.29_3"><P style="font-size:15px; background-color:#66bbFF;"><B>Continuation: pSB1C3_SDM_SspI_Bla14FM clone 5 SDM (P222)</B></P></SPAN></H4><P><B>Investigator: Patrick </B></P><P>The XL1B transformation didnt work or was too much diluted because there were no clones on the plates. The dilution steps of the transformed XL10-Gold cells worked quite well. Now there are 'pickable' clones.
<A href="http://www.molbiotech.uni-freiburg.de/iGEM/wiki2010/index.php/Laborjournal" class="external text" rel="nofollow">top of page</A></P><H4><SPAN class="mw-headline" id="Subcloning_cap_from_pMA_RepCap_into_pAAV_RC_which_contains_four_mutations_and_the_synthesized_Rep_part"><P style="font-size:15px; background-color:#66bbFF;"><B>Subcloning cap from pMA_RepCap into pAAV_RC which contains four mutations and the synthesized Rep part</B></P></SPAN></H4><P><B>Investigator: Patrick </B></P><P>pAAV_RC_ins-rep clone 1 (P250) and pMA_RepCap Vector_SDM_InsPvuII clone 1 (P211) were digested with PspLI and BspMI according to the standard protocol (Subcloning_cap_into_pAAV_RC).
</P><P>Expected size of the bands:
</P><UL><LI>Insert: 920 bp (P211)
</LI><LI>Vector: 6414 bp (P250)
</LI></UL><P>The gelextraction yielded the following DNA concentrations:
</P><UL><LI> P 211 : 2,45 ng/µl
</LI><LI> P 250 : 11,15 ng/µl
</LI></UL><P>
The marked bands were cut out and the DNA was extracted. The ligation was performed accodring to the standard prtocol with a total volume of 10 µl: 2,67 µl vector backbone(P250), 5,33 µl Insert following a transformation according to the standard protocol with XL1B cells.
</P><H4><SPAN class="mw-headline" id="Mini-Preps_of_pSB1C3_Rep68_ex_and_pSB1C3_ZEGFR:1907"><P style="font-size:15px; background-color:#66bbff;"><B>Mini-Preps of pSB1C3_Rep68_ex and pSB1C3_ZEGFR:1907</B></P></SPAN></H4><P><B>Investigator: Chris L.,Stefan</B></P><UL><B>Glycerol stocks were prepared:</B><LI>B218 = pSB1C3_Rep68_ex clone1</LI><LI>B219 = pSB1C3_Rep68_ex clone2</LI><LI>B220 = pSB1C3_Rep68_ex clone3</LI><LI>B221 = pSB1C3_ZEGFR:1907 clone 1</LI><LI>B222 = pSB1C3_ZEGFR:1907 clone 2</LI><P><B>Mini-Prep following the standard protocol</B></P><LI>B218 = pSB1C3_Rep68_ex clone1 c=286.72 ng/µl</LI><LI>B219 = pSB1C3_Rep68_ex clone2 c=277.06 ng/µl</LI><LI>B220 = pSB1C3_Rep68_ex clone3 c=298.18 ng/µl </LI><LI>B221 = pSB1C3_ZEGFR:1907 clone 1 c=174.20 ng/µl
</LI><LI>B222 = pSB1C3_ZEGFR:1907 clone 2 c=162.48 ng/µl </LI></UL><P><B>Test digestion</B></P><UL><LI>buffer used: 4 ; Restriction-enzymes used: XbaI ; SpeI ; EcoRI ; PstI </LI><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>P264 Volume/µL</B></TD><TD align="left"><B>P265 Volume/µL</B></TD><TD align="left"><B>P266 Volume/µL</B></TD><TD align="left"><B>P267 Volume/µL</B></TD><TD align="left"><B>P268 Volume/µL</B></TD></TR><TR><TD align="left"> DNA (800 ng) </TD><TD align="left"> 2.8 </TD><TD align="left"> 2.9 </TD><TD align="left"> 2.7 </TD><TD align="left"> 4.6 </TD><TD align="left"> 4.9
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 1 </TD><TD align="left"> 1 </TD><TD align="left"> 1 </TD><TD align="left"> 1 </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Buffer 4 (10x) </TD><TD align="left"> 1 </TD><TD align="left"> 1 </TD><TD align="left"> 1 </TD><TD align="left"> 1 </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> SpeI </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5 </TD><TD align="left"> - </TD><TD align="left"> -
</TD></TR><TR><TD align="left"> XbaI </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5 </TD><TD align="left"> - </TD><TD align="left"> -
</TD></TR><TR><TD align="left"> PstI </TD><TD align="left"> - </TD><TD align="left"> - </TD><TD align="left"> - </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5
</TD></TR><TR><TD align="left"> EcoRI </TD><TD align="left"> - </TD><TD align="left"> - </TD><TD align="left"> - </TD><TD align="left"> 0.5 </TD><TD align="left"> 0.5
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 4.2 </TD><TD align="left"> 4.1 </TD><TD align="left"> 4.3 </TD><TD align="left"> 2.4 </TD><TD align="left"> 2.1
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>10</B></TD><TD align="left"><B>10</B></TD><TD align="left"><B>10</B></TD><TD align="left"><B>10</B></TD><TD align="left"><B>10</B></TD></TR></TBODY></TABLE><P>
Incubation time: 1 h; Incubation temperature: 37°

0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , running time:5 minutes at 90 Volt and 40 minutes at 115 Volt 
4µl loading dye (6x) for the sample, Marker: GeneRuler ladder mix (Fermentas)
</P><P style="font-size:13px; color:#003399;"><B>Comment</B>:pSB1C3_Rep68_ex clone 3 (P266) sent for sequencing with O162 (VF2) and O51 (VR2). pSB1C3_ZEGFR:1907 clone 1 (P267) sent for sequencing with O51 (VR2).</P></UL><H4><SPAN class="mw-headline" id="Hybridisation_of_the_oligos_for_NLS_and_cloning_NLS_into_pSB1C3"><P style="font-size:15px; background-color:#66bbff;"><B>Hybridisation of the oligos for NLS and cloning NLS into pSB1C3</B></P></SPAN></H4><P><B>Investigator: Anissa </B></P><UL><LI><B>Hybridisation</B>: program: Origami 1 
Hybridisation was peformed according the standard protocol </LI><LI>Digestion of pSB1C3_CFP
<TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>Vector /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right">6,62
</TD></TR><TR><TD> BSA (10X)  </TD><TD align="right">1,5
</TD></TR><TR><TD> Buffer 4  </TD><TD align="right">1,5
</TD></TR><TR><TD> Enzyme EcoRI</TD><TD align="right">1
</TD></TR><TR><TD> Enzyme PstI</TD><TD align="right">1
</TD></TR><TR><TD> H2O</TD><TD align="right">3,38
</TD></TR><TR><TD> Endvolume</TD><TD align="right">15
</TD></TR></TBODY></TABLE></LI><LI>Gel:After digestion, the sample was loaded on a 1% agarose-gel.</LI><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Digest pSB1C3_CFP</CAPTION></TABLE><LI> Gelextraction: After the gel run, the vector backbone pSB1C3 was cut and purified according to the standard protocol</LI><LI>Ligation: the original concentration of the hybridisation was 209,7 ng/µl, for the ligation a 1:10 dilution was made
<TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>Volume /µl</B></TD><TD align="right"><B>Concentration/ ng/µl</B></TD></TR><TR><TD> Insert (NLS) </TD><TD align="right">6,79</TD><TD align="right"> 2,097
</TD></TR><TR><TD> Vector (pSB1C3_cut)  </TD><TD align="right">1,21</TD><TD align="right"> 8,3
</TD></TR></TBODY></TABLE></LI><LI>Transformation: Trafo into XL1 blue was performed according the standard protocol</LI></UL><H4><SPAN class="mw-headline" id="Seeding_HT1080_cells"><P style="font-size:15px; background-color:#66bbff;"><B>Seeding HT1080 cells</B></P></SPAN></H4><P><B>Investigator: Kerstin, Adrian </B></P><UL><LI> Seeding HT1080 cells for transduction with viral stocks from 18.08. 
</LI><LI> We harvest four T75 flask (two of them were used three times (3 passages)) and seed them into eight 6er dishes (200.000 cells per well)
</LI></UL><H4><SPAN class="mw-headline" id="PCR_and_ligation_for_biobrick-production_of_pCerulean"><P style="font-size:15px; background-color:#66bbff;"><B>PCR and ligation for biobrick-production of pCerulean</B></P></SPAN></H4><P><B>Investigator: Christian L. , Stefan</B> 	
DNA template: pEGFP_C1 (P230)
C=367,4 ng/µl
<B>PCR:</B>
(was performed following the standard protocol)
</P><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD align="right"><B>Volume / µl</B></TD></TR><TR><TD> 5X Phusion HF buffer </TD><TD align="right">10
</TD></TR><TR><TD> 10 mM dNTP mix</TD><TD align="right">1
</TD></TR><TR><TD> forward primer: O163 (pEGFP-C1_rev) </TD><TD align="right">2.5
</TD></TR><TR><TD> reverse primer: O164 (pEGFP-C1_for) </TD><TD align="right">2.5
</TD></TR><TR><TD> DNA Template(1:1000) </TD><TD align="right">3 µl
</TD></TR><TR><TD> DMSO (2%)</TD><TD align="right"> 1
</TD></TR><TR><TD> Phusion Polymerase</TD><TD align="right">0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">29.5
</TD></TR><TR><TD>Total volume</TD><TD align="right">50
</TD></TR></TBODY></TABLE><P><B>PCR program:</B></P><TABLE border="1"><TBODY><TR><TD><B>PCR Program</B></TD><TD align="right"><B>temperature/ °C</B></TD><TD align="right"><B>time/ sec.</B></TD></TR><TR><TD>1</TD><TD align="right">98 </TD><TD align="right">60
</TD></TR><TR><TD>2</TD><TD align="right">98 </TD><TD align="right">15
</TD></TR><TR><TD>8x</TD><TD align="right">59 </TD><TD align="right">25
</TD></TR><TR><TD>3 </TD><TD align="right">72 </TD><TD align="right">60
</TD></TR><TR><TD>4 </TD><TD align="right">98 </TD><TD align="right">15
</TD></TR><TR><TD>17x</TD><TD align="right">66 </TD><TD align="right">25
</TD></TR><TR><TD>5</TD><TD align="right">72 </TD><TD align="right">60
</TD></TR><TR><TD>1x</TD><TD align="right">72 </TD><TD align="right">5
</TD></TR><TR><TD>Hold</TD><TD align="right">4 </TD><TD align="right"> -
</TD></TR></TBODY></TABLE><P style="font-size:13px; color:#FF0033;"><I><B>Comments</B></I>: Errors in table, please correct!!! (TB)</P><P><B>Gelextraction of PCR product</B>
Gelextraction was performed according to the standard protocol of the PCR product.
</P><P style="font-size:13px; color:#68bbff;"><I><B>Comment</B></I>: 0,4 g Agarose,50 ml TAE (0,8%), 3 µl GELRED , at 115 Volt, running time: 45 minutes</P><P><B>Digestion of PCR product and vector:</B></P><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>PCR product /µl</B></TD><TD align="right"><B>vector /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right"> 10 </TD><TD align="right"> 29
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right"> 2 </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right"> 2 </TD><TD align="right">2
</TD></TR><TR><TD>Enzyme <B>EcoRI </B></TD><TD align="right"> 1 </TD><TD align="right"> 1</TD></TR><TR><TD>Enzyme <B>PstI </B></TD><TD align="right"> 1 </TD><TD align="right"> 1</TD></TR><TR><TD>H2O</TD><TD align="right"> 4 </TD><TD align="right"> 5
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 20 </TD><TD align="right"> 40
</TD></TR></TBODY></TABLE><P>
Purification of the PCR product with PCR purification was made according to the standard protocol.
<B>Gelextraction of CFP</B>
Gelextraction was performed according to the standard protocol.
&gt;</P><P style="font-size:13px; color:#68bbff;"><I><B>Comment</B></I>: 0.5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 115 Volt, running time: 45 minutes</P><P><B>Ligation</B>
The  Ligation was performed as following:
</P><UL><LI> Vector Volume: 4,1 µl
</LI><LI> Insert Volume: 3,9 µl
</LI></UL><UL><LI> 1µl T4-Ligase buffer (10x)
</LI><LI> 8µl (Vector + Insert) mix
</LI><LI> 1µl T4-Ligase
</LI></UL><P> Incubation for 45 minutes at room temperature
<B>Transformation:</B>
performed according to the standard protocol (XL1B). The cells were plated on a agar plate with Kanamycin.
</P><H4><SPAN class="mw-headline" id="Biobrick_production_of_VP1_up"><P style="font-size:15px; background-color:#66bbff;"><B>Biobrick production of VP1_up</B></P></SPAN></H4><P><B>Investigator: Achim</B> 	
DNA template: pKex_VP1 (P27)
C=99,3 ng/µl
<B>PCR:</B>
(was performed following the standard protocol)
</P><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD align="right"><B>Volume / µl</B></TD></TR><TR><TD> 5X Phusion HF buffer </TD><TD align="right">10
</TD></TR><TR><TD> 10 mM dNTP mix</TD><TD align="right">1
</TD></TR><TR><TD> forward primer: O89 (Präfix_VP1_ex_RFC25_for) </TD><TD align="right">2.5
</TD></TR><TR><TD> reverse primer: O167 (pEGFP-C1_for) </TD><TD align="right">2.5
</TD></TR><TR><TD> DNA Template(1:100) </TD><TD align="right">1 µl
</TD></TR><TR><TD> DMSO (2%)</TD><TD align="right"> -
</TD></TR><TR><TD> Phusion Polymerase</TD><TD align="right">0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD></TR><TR><TD>Total volume</TD><TD align="right">50
</TD></TR></TBODY></TABLE><P><B>PCR program:</B></P><TABLE border="1"><TBODY><TR><TD><B>PCR Program</B></TD><TD align="right"><B>temperature/ °C</B></TD><TD align="right"><B>time/ sec.</B></TD></TR><TR><TD>1</TD><TD align="right">98 </TD><TD align="right">60
</TD></TR><TR><TD>2</TD><TD align="right">98 </TD><TD align="right">15
</TD></TR><TR><TD>8x</TD><TD align="right">60 </TD><TD align="right">25
</TD></TR><TR><TD>3 </TD><TD align="right">72 </TD><TD align="right">15
</TD></TR><TR><TD>4 </TD><TD align="right">98 </TD><TD align="right">15
</TD></TR><TR><TD>17x</TD><TD align="right">66 </TD><TD align="right">25
</TD></TR><TR><TD>1x</TD><TD align="right">72 </TD><TD align="right">15
</TD></TR><TR><TD>Hold</TD><TD align="right">4 </TD><TD align="right"> -
</TD></TR></TBODY></TABLE><P><B>Digestion of PCR product and vector:</B></P><TABLE border="1"><TBODY><TR><TD><B>components</B></TD><TD align="right"><B>PCR product /µl</B></TD><TD align="right"><B>vector /µl</B></TD></TR><TR><TD> DNA  </TD><TD align="right"> 20 </TD><TD align="right"> 5,2
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right"> 3 </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right"> 3 </TD><TD align="right">2
</TD></TR><TR><TD>Enzyme <B>NgOMIV</B></TD><TD align="right"> 1 </TD><TD align="right"> 1</TD></TR><TR><TD>Enzyme <B>SpeI </B></TD><TD align="right"> 1 </TD><TD align="right"> 1</TD></TR><TR><TD>H2O</TD><TD align="right"> 2 </TD><TD align="right"> 8,8
</TD></TR><TR><TD><B>Total volume (e.g. 15,20,25,30 µl)</B></TD><TD align="right"> 30 </TD><TD align="right">20
</TD></TR></TBODY></TABLE><P>
Ligation with T4-Ligase and transformation with XL1B were carried out. Clones have to be picked tomorrow.
</P><H4><SPAN class="mw-headline" id="BioBrick_assembly_for_pSB1C3_Affibody_Linker_and_pSB1C3_YFP_His-Tag"><P style="font-size:15px; background-color:#66bbff;"><B>BioBrick assembly for pSB1C3_Affibody_Linker and pSB1C3_YFP_His-Tag</B></P></SPAN></H4><P><B>Investigators: Volker</B></P><P>
The aim of this experiment is to assemble the affibodies with the the five linkers and the YFP with the His-Tag. All cloning is done according to <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> and will be used to test the affibody if it really binds the EGF-receptor, for the N-terminal fusions and to supply it to other groups in the registry.</P><P>This experiment went wrong on Saturday 21.08. and was repeated on Sunday 22.08. and gave the following results. Results from the first experiment are not shown but can be senn in the written Labjournal.
</P><P style="font-size:15px; font-weight: bold; color: blue;">Digestion</P><P>The following digestions were performed:
</P><TABLE border="1"><TBODY><TR><TD> components
</TD><TD> Affibody
</TD><TD> Affibody
</TD><TD> Affibody
</TD><TD> Short Linker
</TD><TD> Middle Linker
</TD><TD> Long Linker
</TD><TD> SEG Linker
</TD><TD> GSAT Linker
</TD><TD> YFP
</TD><TD> His-Tag
</TD></TR><TR><TD> DNA
</TD><TD>8.6 µl
</TD><TD>8.6 µl
</TD><TD>8.6 µl
</TD><TD>7.7 µl
</TD><TD>8.2 µl
</TD><TD>8.0 µl
</TD><TD>8.1 µl
</TD><TD>8.4 µl
</TD><TD>5.4 µl
</TD><TD>14 µl
</TD></TR><TR><TD> BSA (100x)
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>0µl
</TD><TD>0µl
</TD><TD>0µl
</TD><TD>0µl
</TD><TD>0µl
</TD><TD>2µl
</TD><TD>0µl
</TD></TR><TR><TD> Buffer (10x)
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>2µl
</TD><TD>2µl
</TD></TR><TR><TD>Enzyme 1 (1µl each)
</TD><TD>PstI
</TD><TD>PstI
</TD><TD>PstI
</TD><TD>PstI
</TD><TD>PstI
</TD><TD>PstI
</TD><TD>PstI
</TD><TD>PstI
</TD><TD>EcoRI
</TD><TD>EcoRI
</TD></TR><TR><TD>Enzyme 2 (1µl each)
</TD><TD>AgeI
</TD><TD>AgeI
</TD><TD>AgeI
</TD><TD>NgoMIV
</TD><TD>NgoMIV
</TD><TD>NgoMIV
</TD><TD>NgoMIV
</TD><TD>NgoMIV
</TD><TD>AgeI
</TD><TD>NgoMIV
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>5.4 µl
</TD><TD>5.4 µl
</TD><TD>5.4 µl
</TD><TD>8.3 µl
</TD><TD>7.8 µl
</TD><TD>8.0 µl
</TD><TD>7.9 µl
</TD><TD>7.6 µl
</TD><TD>8.6 µl
</TD><TD>2.0 µl
</TD></TR><TR><TD><B>Total volume </B></TD><TD>20µl
</TD><TD>20µl
</TD><TD>20µl
</TD><TD>20µl
</TD><TD>20µl
</TD><TD>20µl
</TD><TD>20µl
</TD><TD>20µl
</TD><TD>20µl
</TD><TD>20µl
</TD></TR></TBODY></TABLE><P>Incubation: 2 h
</P><P style="font-size:15px; font-weight: bold; color: blue;">Preparative Agarose-Gel</P><P>
1 g Agarose, 100 ml TAE (1%), 6 GELRED (3-6µl), at 120Volt, running time: 130 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Lane
</TH><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (5x/6x)/µl
</TH><TH>Expected size 1 (Geneious)
</TH><TH>Expected size 2 (Geneious)
</TH></TR><TR><TD>1
</TD><TD>Affibody
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD> bp
</TD><TD> bp
</TD></TR><TR><TD>2
</TD><TD>Affibody
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD> bp
</TD><TD> bp
</TD></TR><TR><TD>3
</TD><TD>Affibody
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD> bp
</TD><TD> bp
</TD></TR><TR><TD>10
</TD><TD>His-Tag
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD> bp
</TD><TD> bp
</TD></TR></TBODY></TABLE><P>
2 g Agarose, 100 ml TAE (2%), 6 GELRED (3-6µl), at 120 Volt, running time: 130 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Lane
</TH><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (5x/6x)/µl
</TH></TR><TR><TD>4
</TD><TD>Short Linker
</TD><TD>20 µl
</TD><TD>4 µl
</TD></TR><TR><TD>5
</TD><TD>Medium Linker
</TD><TD>20 µl
</TD><TD>4 µl
</TD></TR><TR><TD>6
</TD><TD>Long Linker
</TD><TD>20 µl
</TD><TD>4 µl
</TD></TR><TR><TD>7
</TD><TD>SEG Linker
</TD><TD>20 µl
</TD><TD>4 µl
</TD></TR><TR><TD>8
</TD><TD>GSAT Linker
</TD><TD>20 µl
</TD><TD>4 µl
</TD></TR><TR><TD>9
</TD><TD>YFP
</TD><TD>20 µl
</TD><TD>4 µl
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Sequencing results of pSB1C3_hTERT</CAPTION></TABLE><P style="font-size:15px; font-weight: bold; color: blue;">Gel extraction</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Sample</B></TD><TD align="left"><B>Weight</B></TD><TD align="left"><B>Volume</B></TD><TD align="left"><B>Concentration</B></TD></TR><TR><TD align="left"> Affibody
</TD><TD align="left"> 0.4g
</TD><TD align="left"> 1200µl
</TD><TD align="left"> 29.78
</TD></TR><TR><TD align="left"> Affibody
</TD><TD align="left"> 0.44g
</TD><TD align="left"> 1320µl
</TD><TD align="left"> 24.86
</TD></TR><TR><TD align="left"> Short-Linker
</TD><TD align="left"> 0.27 g
</TD><TD align="left"> 810µl
</TD><TD align="left"> 0.22
</TD></TR><TR><TD align="left"> Middle-Linker
</TD><TD align="left"> 0.38 g
</TD><TD align="left"> 1140µl
</TD><TD align="left"> 3.67
</TD></TR><TR><TD align="left"> Long-Linker
</TD><TD align="left"> 0.35 g
</TD><TD align="left"> 1050µl
</TD><TD align="left"> 1.31
</TD></TR><TR><TD align="left"> SEG-Linker
</TD><TD align="left"> 0.34 g
</TD><TD align="left"> 1020µl
</TD><TD align="left"> 2.13
</TD></TR><TR><TD align="left"> GSAT-Linker
</TD><TD align="left"> 0.22 g
</TD><TD align="left"> 660µl
</TD><TD align="left"> 3.58
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color: blue;">Ligation</P><TABLE border="1"><TBODY><TR><TD> Sample
</TD><TD><B>Insert</B></TD><TD><B>V(Insert)</B></TD><TD><B>Vector</B></TD><TD><B>V(Vector)</B></TD></TR><TR><TD>pSB1C3_Affibody_Short-Linker
</TD><TD>Affibody
</TD><TD>1.06µl
</TD><TD>Short-Linker
</TD><TD>6.94µl
</TD></TR><TR><TD>pSB1C3_Affibody_Middle-Linker
</TD><TD>Affibody
</TD><TD>5.25µl
</TD><TD>Middle-Linker
</TD><TD>2.75µl
</TD></TR><TR><TD>pSB1C3_Affibody_Long-Linker
</TD><TD>Affibody
</TD><TD>2.93µl
</TD><TD>Long-Linker
</TD><TD>5.07µl
</TD></TR><TR><TD>pSB1C3_Affibody_SEG-Linker
</TD><TD>Affibody
</TD><TD>2.46µl
</TD><TD>SEG-Linker
</TD><TD>5.54µl
</TD></TR><TR><TD>pSB1C3_Affibody_GSAT-Linker
</TD><TD>Affibody
</TD><TD>3.41µl
</TD><TD>GSAT-Linker
</TD><TD>4.59µl
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="96._labday_21.08.2010"><P style="font-size:17px; background-color:#00dd77;">96. labday 21.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Transfection_of_AAV293.2C_Harvest_viral_particles.2C_Transduction_of_HT1080_and_Fluorescent_microscopy"><P style="font-size:15px; background-color:#66bbff;"><B>Transfection of AAV293, Harvest viral particles, Transduction of HT1080 and Fluorescent microscopy</B></P></SPAN></H4><P><B> Investigators: Adrian, Kerstin </B></P><P><B>The Motivation</B>
Measure the Transduction efficiency from 19.8
</P><P><B>The Motivation</B>
Transfection of AAV293 cells from 19.8 (ten 10cm<SUP>2</SUP> dishes with 2.000.000 cells each) <FONT size="4" color="#F0000">with original standard protocoll!!!</FONT> We want to try different 2xHBS Buffers (pH) and their influence on transfection efficiency!<B>The Plan</B><B>Requiered goods</B>: 0,3 M CaCl<SUB>2</SUB> , 2x HBS-Buffer , autoclaved millipore water, falcons, eppis
<B>Working steps</B>:
</P><OL><LI>Remove the three plasmids to be co-transfected (the recombinant  pAAV expression plasmid or control plasmid, pAAV-RC, and pHelper)  from storage at –20°C.</LI><LI>Pipet 10 µg of each of the three plasmid DNA solutions into an eppi.</LI><LI>Add 1ml of 0.3 M CaCl<SUB>2</SUB> and mix gently</LI><LI>Pipet  1ml of 2 × HBS  into a 15-ml falcon.</LI><LI>pipet the DNA/CaCl<SUB>2</SUB> solution dropwise into the falcon (use the Pipetus).</LI><LI>Apply the DNA/CaCl<SUB>2</SUB>/HBS suspension to the plate of cells in a dropwise fashion, swirling gently to distribute the DNA suspension evenly in the medium.</LI><LI>Return the tissue culture plate to the 37°C room for 6 hours.</LI><LI>At the end of the incubation period, remove the medium from the plate and wash cells with warm PBS, replace it with 10 ml of fresh DMEM growth medium.</LI><LI>Return the plate to the 37°C incubator for an additional 72 hours.</LI></OL><P>
We try 5 different 2xHBS buffers:
</P><UL><LI>pH: 7.06</LI><LI>pH: 7.08</LI><LI>pH: 7.10</LI><LI>pH: 7.12</LI><LI>pH: 7.14</LI></UL><UL><LI>1. Plate: 10µg of Rep/Cap, pHelper and GOI=mVenus; pH(2xHBS)= 7.06
</LI><LI>2. Plate: 10µg of Rep/Cap, pHelper and GOI=mVenus; pH(2xHBS)= 7.08
</LI><LI>3. Plate: 10µg of Rep/Cap, pHelper and GOI=mVenus; pH(2xHBS)= 7.10
</LI><LI>4. Plate: 10µg of Rep/Cap, pHelper and GOI=mVenus; pH(2xHBS)= 7.12
</LI><LI>5. Plate: 10µg of Rep/Cap, pHelper and GOI=mVenus; pH(2xHBS)= 7.14
</LI><LI>6. Plate: 10µg of Rep/Cap, pHelper and GOI=TKGMK;  pH(2xHBS)= 7.08
</LI><LI>7. Plate: 10µg of Rep/Cap, pHelper and GOI=TKGMK;  pH(2xHBS)= 7.10
</LI><LI>8. Plate: 10µg of Rep/Cap, pHelper and GOI=mVenus; pH(2xHBS)= 7.14
</LI></UL><P>
The transfected cells were seeded at 19.8 (2.000.000 cells per 10 cm<SUP>2</SUP>)

Used Plasmids: Standart protocol: 10µg of each plasmid
</P><UL><LI>GOI: mVENUS =&gt; P262, conc:1148ng/µl; used amount: 8,71µl</LI><LI>GOI: TKGMK  =&gt; P82b, conc:1500ng/µl; used amount: 6,67µl</LI><LI>pHELPER =&gt; P64, conc:503ng/µl; used amount: 20µl</LI><LI>RepCap =&gt; 10.8: 1348ng/µl; used amount: 7,41µl</LI><LI><B> Harvest viral paricles from 18.08.2010 and transduction of HT1080 cells following the standard protocol </B></LI><LI><B> Analysis via Fluorescent microscopy (Transduction from 19.08.2010)</B></LI><UL><LI>Results:
</LI></UL></UL><H4><SPAN class="mw-headline" id="Midi-Prep_of_pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR_clone1"><P style="font-size:15px; background-color:#66bbff;"><B>Midi-Prep of pSB1C3_lITR_CMV_betaglobin_mVenus_hGH_rITR clone1</B></P></SPAN></H4><P><B>Investigators: Chris W. </B></P><P style="font-size:13px; color:#003399;"><B>Comment</B>: Midi-Preps of P 242, B200</P><P>The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
</P><TABLE border="1"><TBODY><TR><TD> plasmid-no. </TD><TD align="right">P269</TD><TD align="right">p270
</TD></TR><TR><TD> concentration (ng/µl)</TD><TD align="right">1035,10 </TD><TD align="right">1376,95
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Re-Trafo_of_pSB1C3_SDM_SspI_Bla14FM"><P style="font-size:15px; background-color:#66bbff;"><B>Re-Trafo of pSB1C3_SDM_SspI_Bla14FM</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P style="font-size:13px; color:#68bbff;"><I><B>Comment</B></I>: Mini-Prep of pSB1C3_SDM_SspI_Bla14FM showed extremely low plasmid concentrations. It was proposed that this could correspond to using not the standard lab cells but rather XL-10 Gold cells for trafo. Therefore a re-trafo was performed using BL21 cells. 2 µl of each clone was added to BL21 cells and trafo was performed. Clones can be picked tonight (Volker)!</P><H3><SPAN class="mw-headline" id="97._labday_22.08.2010">97. labday 22.08.2010</SPAN></H3><H4><SPAN class="mw-headline" id="Test-digestion_of_pCerulean_.28P273.2C4.2C5.29.2C_pSB1C3_VP1up_.28P279.2C_280.29.2C_pSB1C3_NLS_.28P281.2C282.29.2C_pAAV_RC_final_.28P283.29"><P style="font-size:15px; background-color:#66bbff;"><B>Test-digestion of pCerulean (P273,4,5), pSB1C3_VP1up (P279, 280), pSB1C3_NLS (P281,282), pAAV_RC_final (P283) </B></P></SPAN></H4><P><B>Investigator: Patrick</B>
Digestions:
</P><UL><LI> 0,5 µl EcoRI, 0,5 µl Enzyme PstI, 1 µl BSA (10x), 1 µl Buffer 4 (10x), 1 µl pCerulean (P73), 6 µl H2O; total volume = 10 µl
</LI><LI> 0,5 µl EcoRI, 0,5 µl Enzyme PstI, 1 µl BSA (10x), 1 µl Buffer 4 (10x), 1 µl pCerulean (P74), 6 µl H2O; total volume = 10 µl
</LI><LI> 0,5 µl EcoRI, 0,5 µl Enzyme PstI, 1 µl BSA (10x), 1 µl Buffer 4 (10x), 1 µl pCerulean (P75), 6 µl H2O; total volume = 10 µl
</LI><LI> 1 µl NotI, 1 µl BSA (10x), 1 µl Buffer 4 (10x), 1 µl pSB1C3_VP1up (P279), 6 µl H2O; total volume = 10 µl
</LI><LI> 1 µl NotI, 1 µl BSA (10x), 1 µl Buffer 4 (10x), 1 µl pSB1C3_VP1up (P280), 6 µl H2O; total volume = 10 µl
</LI><LI> 0,5 µl EcoRI, 0,5 µl Enzyme PstI, 1 µl BSA (10x), 1 µl Bufer 4 (10x), 1 µl pSB1C3_NLS (P281) , 6 µl H2O; total volume = 10 µl
</LI><LI> 0,5 µl EcoRI, 0,5 µl Enzyme PstI, 1 µl BSA (10x), 1 µl Bufer 4 (10x), 1 µl pSB1C3_NLS (P282) , 6 µl H2O; total volume = 10 µl
</LI><LI> 0,5 µl Acc65I, 0,5 µl BspMI, 1 µl BSA (10x), 1 µl Buffer 3 (10x) , 1 µl pAAV_RC_final ,  6 µl H2O; total volume = 10 µl
</LI></UL><P>Incubation time: 37°C, 1h
</P><P>Expected size of the fragments:
</P><UL><LI>pCerulean clone 1 : 771 &amp; 3915 bp
</LI><LI>pCerulean clone 2 : 771 &amp; 3915 bp
</LI><LI>pCerulean clone 3 : 771 &amp; 3915 bp
</LI><LI>pSB1C3_VP1up clone 1 : 468 &amp; 2030 bp
</LI><LI>pSB1C3_VP1up clone 2 : 468 &amp; 2030 bp
</LI><LI>pSB1C3_NLS clone 1 : 78 &amp; 2031 bp
</LI><LI>pSB1C3_NLS clone 2 : 78 &amp; 2031 bp
</LI><LI>pAAV_RC_final : 615 &amp; 6720 bp
</LI></UL><P>
pCerulean clone 1 (P273), pSB1C3_VP1up clone 2 (P280), pSB1C3_NLS clone 2 (P282) and pAAV_RC_final (P283) were sent for sequencing although the results don't look very promising. 
Labelling:
</P><UL><LI> pCerulean clone 1 (P273) : iGEM2-CMV-F
</LI><LI> pSB1C3_VP1up clone 2 (P280) : iGEM1-VF2
</LI><LI> pSB1C3_NLS clone 2 (P282) : iGEM276-O51_VR2 and iGEM3-O51_VR2 (was ordered twice due to an ordering mistake)
</LI><LI> pAAV_RC_final (P283) : iGEM4-SK
</LI><LI> pAAV_RC_final (P283) : iGEM4-VP1 pKEX rev
</LI><LI> pAAV_RC_final (P283) : iGEM4-Cap2800for
</LI><LI> pAAV_RC_final (P283) : iGEM4-Cap2800rev
</LI><LI> pAAV_RC_final (P283) : iGEM4-Cap3500for
</LI></UL><H4><SPAN class="mw-headline" id="Mini-Prep_of_pCerulean"><P style="font-size:15px; background-color:#66bbff;"><B>Mini-Prep of pCerulean</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><UL><B>Glycerol stocks were prepared:</B><LI>B225 = pCerulean clone1</LI><LI>B226 = pCerulean clone2</LI><LI>B227 = pCerulean clone3</LI></UL><UL><B>Mini-Prep following the standard protocol</B><LI>P273 = pCerulean clone1 c= 419,52 ng/µl</LI><LI>P274 = pCerulean clone2 c= 383,37 ng/µl</LI><LI>P275 = pCerulean clone3 c= 382,72 ng/µl </LI></UL><P>Test digestion will be performed tomorrow.
</P><H3><SPAN class="mw-headline" id="98._labday_23.08.2010"><P style="font-size:17px; background-color:#00dd77;">98. labday 23.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Primer_design:_Sequencing_primer_for_BioBrick_Assembly_of_vectorplasmid_and_phTERT_qPCR_primer"><P style="font-size:15px; background-color:#66bbff;"><B>Primer design: Sequencing primer for BioBrick Assembly of vectorplasmid and phTERT qPCR primer</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="font-size:13px; color:#003399;"><B>Comment</B>: Two goals will be accomplished with this primers. 
<OL><LI>The first primer pair (left picture) serve as sequencing primers for the pSB1C3 vector plasmid. Problem was that the designed VR-2/VF-2 primers cannot be used for sequencing because the ITRs flank several assembled parts, mostly CMV, beta globin, GOI and hGH. These primers were designed to bind in the flanked region, directly after/before left and right ITR repsectivel and can therfore be used to sequence most of the flanked sequence inbetween the ITRs.</LI><LI>The second primer pair (right picture) will be used for performing the qPCR with th econtruct containing the phTERT promoter. The qPCR primers used for the other qRT-PCR cannot be used because these two primers anneal in the CMV region. The new designed primers bind in the phTERT region and can therefore bee used for determinining the infectious titers.</LI></OL></P><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Primer design. Ordered at Sigma Aldrich</CAPTION></TABLE><H4><SPAN class="mw-headline" id="Mini-Prep_of_pSB1C3_CFP_middlelinker_and_pSB1C3_ZEGFR:1907_.2F_testdigestion_of_pSB1C3_CFP_middlelinker_and_pSB1C3_lITR_pHTERT"><P style="font-size:15px; background-color:#66bbff;"><B>Mini-Prep of pSB1C3_CFP_middlelinker and pSB1C3_ZEGFR:1907 / testdigestion of pSB1C3_CFP_middlelinker and pSB1C3_lITR_pHTERT</B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><P><B>Mini-Prep following the standard protocoll</B></P><UL><LI>P276 PSB1C3_CFP_middlelinker clone1 c=207,7 ng/µl
</LI><LI>P277 PSB1C3_CFP_middlelinker clone2 c=203,0 ng/µl
</LI><LI>P278 PSB1C3_CFP_middlelinker clone3 c=227,9 ng/µl
</LI><LI>P279 pSB1C3_VP1up clone1 c=183,0 ng/µl
</LI><LI>P280 pSB1C3_VP1up clone2 c=178,1 ng/µl
</LI><LI>P281 pSB1C3_NLS clone1 c=155,7 ng/µl
</LI><LI>P282 pSB1C3_NLS clone2 c=142,6 ng/µl
</LI><LI>P283 pAAV_RC_final c=147,7 ng/µl
</LI><LI>P284 pSB1C3_ZEGFR:1907 clone 1 c=146,5 ng/µl
</LI><LI>P285 pSB1C3_ZEGFR:1907 clone 1 c=161,9 ng/µl</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pSB1C3_Zegfr:1907 wurde bereits von Stefan geprept... was hast du hier gemacht Jessy? Retrafo vom Glycerolstock???</DIV></DIV></DIV><UL><B>Test digestion:</B><LI>Perfomed with 10 µL of total volume</LI></UL><TABLE border="1" cellpadding="0" cellspacing="0" width="621" style="border-collapse: collapse;table-layout:fixed;width:466pt"><TBODY><TR style="mso-height-source:userset;height:21.75pt"><TD height="29" class="xl6724191" width="143" style="height:21.75pt;width:107pt"> </TD><TD class="xl6824191" width="131" style="border-left:none;width:98pt">Mastermix/µL</TD><TD class="xl6824191" width="87" style="border-left:none;width:65pt">P256/µL</TD><TD class="xl6824191" width="84" style="border-left:none;width:63pt">P257/µL</TD><TD class="xl6824191" width="90" style="border-left:none;width:68pt">P276/µL</TD><TD class="xl6824191" width="86" style="border-left:none;width:65pt">P277/µL</TD><TD class="xl6824191" width="86" style="border-left:none;width:65pt">P278/µL</TD></TR><TR style="mso-height-source:userset;height:21.75pt"><TD height="29" class="xl6624191" width="143" style="height:21.75pt;border-top: none;width:107pt"><SPAN lang="EN-US" style="mso-ansi-language:EN-US">DNA</SPAN></TD><TD class="xl6524191" style="border-top:none;border-left:none">-</TD><TD class="xl6524191" style="border-top:none;border-left:none">5</TD><TD class="xl6524191" style="border-top:none;border-left:none">5,2</TD><TD class="xl6524191" style="border-top:none;border-left:none">4,3</TD><TD class="xl6524191" style="border-top:none;border-left:none">4,4</TD><TD class="xl6524191" style="border-top:none;border-left:none">4,0</TD></TR><TR style="mso-height-source:userset;height:21.75pt;mso-yfti-irow: 1"><TD height="29" class="xl6624191" width="143" style="height:21.75pt;border-top: none;width:107pt"><SPAN lang="EN-US" style="mso-ansi-language:EN-US">BSA (10x)</SPAN></TD><TD class="xl6524191" style="border-top:none;border-left:none">6</TD><TD rowspan="4" class="xl6524191" style="border-top:none">1</TD><TD rowspan="4" class="xl6524191" style="border-top:none">1</TD><TD rowspan="4" class="xl6524191" style="border-top:none">1</TD><TD rowspan="4" class="xl6524191" style="border-top:none">1</TD><TD rowspan="4" class="xl6524191" style="border-top:none">1</TD></TR><TR style="mso-height-source:userset;height:21.75pt;mso-yfti-irow: 2"><TD height="29" class="xl6624191" width="143" style="height:21.75pt;border-top: none;width:107pt"><SPAN lang="EN-US" style="mso-ansi-language:EN-US">Buffer
  4(10x)</SPAN></TD><TD class="xl6524191" style="border-top:none;border-left:none">6</TD></TR><TR style="mso-height-source:userset;height:21.75pt;mso-yfti-irow: 3"><TD height="29" class="xl6624191" width="143" style="height:21.75pt;border-top: none;width:107pt"><SPAN lang="EN-US" style="mso-ansi-language:EN-US">Enzyme
  XbaI</SPAN></TD><TD class="xl6524191" style="border-top:none;border-left:none">3</TD></TR><TR style="mso-height-source:userset;height:21.75pt;mso-yfti-irow: 4"><TD height="29" class="xl6624191" width="143" style="height:21.75pt;border-top: none;width:107pt"><SPAN lang="EN-US" style="mso-ansi-language:EN-US">Enzyme
 PstI</SPAN></TD><TD class="xl6524191" style="border-top:none;border-left:none">3</TD></TR><TR style="mso-height-source:userset;height:21.75pt;mso-yfti-irow: 5;mso-yfti-lastrow:yes"><TD height="29" class="xl6624191" width="143" style="height:21.75pt;border-top: none;width:107pt"><SPAN lang="EN-US" style="mso-ansi-language:EN-US">H2O</SPAN></TD><TD class="xl6524191" style="border-top:none;border-left:none">-</TD><TD class="xl6524191" style="border-top:none;border-left:none">2</TD><TD class="xl6524191" style="border-top:none;border-left:none">1,8</TD><TD class="xl6524191" style="border-top:none;border-left:none">2,7</TD><TD class="xl6524191" style="border-top:none;border-left:none">2,6</TD><TD class="xl6524191" style="border-top:none;border-left:none">3,0</TD></TR><TR style="mso-height-source:userset;height:21.75pt"><TD height="29" class="xl6624191" width="143" style="height:21.75pt;border-top: none;width:107pt">Total volume</TD><TD class="xl6524191" style="border-top:none;border-left:none">18</TD><TD class="xl6524191" style="border-top:none;border-left:none">10</TD><TD class="xl6524191" style="border-top:none;border-left:none">10</TD><TD class="xl6524191" style="border-top:none;border-left:none">10</TD><TD class="xl6524191" style="border-top:none;border-left:none">10</TD><TD class="xl6524191" style="border-top:none;border-left:none">10</TD></TR></TBODY></TABLE><UL><LI>incubation time: 50'
</LI><LI>incubation temparatur: 37°C
</LI></UL><UL><LI>5µl new ladder mix
</LI></UL><P><B> Sequencing</B>: 
</P><UL><LI> Plasmid used: P276: pSB1C3_CFP_middlelinker</LI><LI> Primer used: VR-2</LI><LI> Tube name: JG 276</LI></UL><H4><SPAN class="mw-headline" id="Continuation_of_BioBrick_assembly_for_pSB1C3_Affibody_Linker"><P style="font-size:15px; background-color:#66bbff;"><B>Continuation of BioBrick assembly for pSB1C3_Affibody_Linker</B></P></SPAN></H4><P><B> Investigator: Anna</B></P><P style="font-size:13px; color:#003399;"><B>Comment</B>: Transformation was done of pSB1C3_Affibody with Short-Linker, Middle-Linker, Long-Linker, SEG-Linker and GSAT-Linker. Unfortunately, the digestion of pSB1C3_YFP_His-Tag didn't work (see 20.08).</P><P>Transformation was done following the standard protocol using XL1 blue cells.
</P><P style="font-size:13px; color:#003399;"><B>To do:</B> Repetition of BioBrick assembly for pSB1C3_YFP_His-Tag.</P><H4><SPAN class="mw-headline" id="Cloning_of_pSB1C3_.C3.9F-globin_YFP_and_pSB1C3_hGH_rITR"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of pSB1C3_ß-globin_YFP and pSB1C3_hGH_rITR</B></P></SPAN></H4><P><B> Investigator: Anna</B></P><P style="font-size:13px; color:#003399;"><B>Comment</B>: pSB1C3_ß-globin_mVenus (p178) was digested with SpeI and PstI and pSB1C3_hGH_rITR (p186) was digested with XbaI and PstI .</P><P>Vector: </P><UL><LI>pSB1C3_ß-globin_YFP (p178): c= 237,2 ng/µl</LI></UL><P>Insert: </P><UL><LI>pSB1C3_hGH_rITR (p186): c= 105,2 ng/µl</LI></UL><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">Vector (p178)</TD><TD align="right">Insert (p186)
</TD></TR><TR><TD> DNA  </TD><TD align="right">4,2 </TD><TD align="right"> 14
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">1,5</TD><TD align="right"> 2,5
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">1,5 </TD><TD align="right"> 2,5
</TD></TR><TR><TD>Enzyme EcoRI </TD><TD align="right">1 </TD><TD align="right"> 1,5
</TD></TR><TR><TD>Enzyme PstI </TD><TD align="right">1</TD><TD align="right"> 1,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">5,8 </TD><TD align="right"> 3
</TD></TR><TR><TD><B>Total volume </B></TD><TD align="right">15 </TD><TD align="right"> 25
</TD></TR></TBODY></TABLE><UL><LI>Gelextraction of pSB1C3_RFC25_CFP:
</LI></UL><P>0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 120 Volt, running time:55 
Marker: GeneRuler ladder mix (Fermentas)
</P><UL><LI>c(pSB1C3_ß-globin_YFP)= 16.9 ng/µl; size: 3293 bp</LI><LI>c(hGH_rITR)= 7,5 ng/µl; size: 657 bp
</LI></UL><UL><LI>Ligation of PCR products and vector:
</LI></UL><P>For the Ligation 1µl T4 buffer (2x) and 1µl T4 ligase were used. Incubation time: 30 min
</P><TABLE border="1"><TBODY><TR><TD align="right"><B>vector /µl</B></TD><TD align="right"><B>insert /µl</B></TD></TR><TR><TD> pSB1C3_ß-globin_mVenus + hGH_rITR </TD><TD align="right">3,41 </TD><TD align="right">4,59
</TD></TR></TBODY></TABLE><UL><LI>Transformation:
</LI></UL><P>The transformation was done following the standard protocol using XL1 blue cells.</P><P style="font-size:13px; color:#003399;"><B>To do:</B> Picking clones of pSB1C3_ß-globin_mVenus_hGH_rITR.</P><H4><SPAN class="mw-headline" id="Repetition:_Quickchange_site-directed_mutagenesis_of_pSB1C3_SDM_SspI_Bla14FM"><P style="font-size:15px; background-color:#66bbff;"><B>Repetition: Quickchange site-directed mutagenesis of pSB1C3_SDM_SspI_Bla14FM</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P><B>Comment:</B> To delete a PvuII restriction site in the CAT marker, a SDM will be performed.</P><UL><LI><B>PCR reaction:</B></LI></UL><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD align="right"><B>v / µl</B></TD></TR><TR><TD> 10x Pfu Ultra II buffer </TD><TD align="right">2,5
</TD></TR><TR><TD> dNTP</TD><TD align="right">0,5
</TD></TR><TR><TD> forward primer: O109 (pSB1C3 PvuII for)</TD><TD align="right">0,59
</TD></TR><TR><TD> reverse primer: O108 (pSB1C3 PvuII rev) </TD><TD align="right">0,58
</TD></TR><TR><TD> DNA Template 10ng (1:100 dilution) </TD><TD align="right">4
</TD></TR><TR><TD> DMSO (1%)</TD><TD align="right"> 0,5
</TD></TR><TR><TD> PfuUltra II fusion </TD><TD align="right">0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">15,83
</TD></TR><TR><TD>Total volume</TD><TD align="right">25
</TD></TR></TBODY></TABLE><UL><LI><B>PCR program:</B></LI></UL><TABLE border="1"><TBODY><TR><TD><B>PCR Program</B></TD><TD align="right"><B>temperature/ °C</B></TD><TD><B>time / s</B></TD></TR><TR><TD>1</TD><TD align="right">95  </TD><TD align="right"> 120
</TD></TR><TR><TD>2</TD><TD align="right">95 </TD><TD align="right"> 30
</TD></TR><TR><TD>3</TD><TD align="right">55  </TD><TD align="right"> 60
</TD></TR><TR><TD>4 (Step 2-4: 20x) </TD><TD align="right">68 </TD><TD align="right"> 130
</TD></TR><TR><TD>Hold</TD><TD align="right">4
</TD></TR></TBODY></TABLE><P>
SDM was performed according to protocol.</P><P>Transformation was performed according to protocol using XL1b cells.
</P><H4><SPAN class="mw-headline" id="FACS_analysis.2C_Seeding_HT1080_and_Hek293_cells"><P style="font-size:15px; background-color:#66bbff;"><B>FACS analysis, Seeding HT1080 and Hek293 cells</B></P></SPAN></H4><P><B>Investigator: Kerstin, Adrian </B></P><UL><LI> Seeding HEK293 cells (100.000, 200.000, 300.000, 400.000, 500.000, 600.000 and 700.000 cells per dish)for transfection
</LI></UL><UL><LI>FACS analysis of transduction from 21.08.2010:
</LI></UL><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">plate/well</TH><TH width="80">transfection plan</TH><TH width="80">living cells</TH><TH width="80">YFP expression</TH><TH width="80">eGFP expression</TH></TR><TR><TD>1.1</TD><TD>control, no cells</TD></TR><TR><TD>1.2</TD><TD>non treated cells</TD><TD>86,8%</TD></TR><TR><TD>1.3</TD><TD>Lipo-Transfection I</TD><TD>83%</TD><TD>63,1%</TD></TR><TR><TD>1.4</TD><TD>Lipo-Transfection I</TD><TD>84,3%</TD><TD>67,5%</TD></TR><TR><TD>1.5</TD><TD>Lipo-Transfection I</TD><TD>86,4%</TD><TD>68,4%</TD></TR><TR><TD>1.6</TD><TD>Lipo-Transfection I</TD><TD>81,7%</TD><TD>66,5%</TD></TR><TR><TD>2.2</TD><TD>non treated cells</TD><TD>83,9%</TD></TR><TR><TD>2.3</TD><TD>Lipo-Transfection II</TD><TD>82,6%</TD><TD>65%</TD></TR><TR><TD>2.4</TD><TD>Lipo-Transfection II</TD><TD>84,7%</TD><TD>68,7%</TD></TR><TR><TD>2.5</TD><TD>Lipo-Transfection II</TD><TD>81,5%</TD><TD>62,6%</TD></TR><TR><TD>2.6</TD><TD>Lipo-Transfection II</TD><TD>83,1%</TD><TD>69,9%</TD></TR><TR><TD>3.2</TD><TD>non treated cells</TD><TD>83,3%</TD></TR><TR><TD>3.3</TD><TD>3,3µg of RC, pHelper&amp;YFP</TD><TD>81,5%</TD><TD>50,7%</TD></TR><TR><TD>3.4</TD><TD>3,3µg of RC, pHelper&amp;YFP</TD><TD>75,6%</TD><TD>40%</TD></TR><TR><TD>3.5</TD><TD>3,3µg of RC, pHelper&amp;YFP</TD><TD>84,7%</TD><TD>63,7%</TD></TR><TR><TD>3.6</TD><TD>3,3µg of RC, pHelper&amp;YFP</TD><TD>82,3%</TD><TD>61,5%</TD></TR><TR><TD>4.3</TD><TD>10µg RC <B>mut(P158a)</B>, pHelper, 3,3µg YFP</TD><TD>85,8%</TD><TD>66,5%</TD></TR><TR><TD>4.4</TD><TD>10µg RC <B>mut(P158a)</B>, pHelper, 3,3µg YFP</TD><TD>84,5%</TD><TD>68,2%</TD></TR><TR><TD>4.5</TD><TD>10µg RC <B>mut(P158b)</B>, pHelper, 3,3µg YFP</TD><TD>76,9%</TD><TD>52,4%</TD></TR><TR><TD>4.6</TD><TD>10µg RC <B>mut(P158b)</B>, pHelper, 3,3µg YFP</TD><TD>56,6%</TD><TD>31,8%</TD></TR><TR><TD>5.2</TD><TD>non treated cells</TD><TD>88,2%</TD></TR><TR><TD>5.3</TD><TD>3,3µg RC <B>mut(P158a)</B>, pHelper&amp;YFP</TD><TD>83,9%</TD><TD>57,5%</TD></TR><TR><TD>5.4</TD><TD>3,3µg RC <B>mut(P158a)</B>, pHelper&amp;YFP</TD><TD>89,1%</TD><TD>48,0%</TD></TR><TR><TD>5.5</TD><TD>3,3µg RC <B>mut(P158b)</B>, pHelper&amp;YFP</TD><TD>88,8%</TD><TD>29,4%</TD></TR><TR><TD>5.6</TD><TD>3,3µg RC <B>mut(P158b)</B>, pHelper&amp;YFP</TD><TD>85,1%</TD><TD>40,6%</TD></TR><TR><TD>6.3</TD><TD>10µg RC, pHelper&amp;<B>eGFP</B></TD><TD>82,5%</TD><TD>75,2%</TD></TR><TR><TD>6.4</TD><TD>10µg RC, pHelper, 3,3µg<B>eGFP</B></TD><TD>87,6%</TD><TD>80,1%</TD></TR><TR><TD>6.5</TD><TD>10µg RC, pHelper, 10µg<B>eGFP</B></TD><TD>81,9%</TD><TD>41,8%</TD></TR><TR><TD>6.6</TD><TD>10µg RC, pHelper, 10µg<B>eGFP</B></TD><TD>83,4%</TD><TD>27,95%</TD></TR><TR><TD>7.2</TD><TD>non treated cells</TD><TD>88,7%</TD></TR><TR><TD>7.3</TD><TD>10µg RC, pHelper, 20µg<B>eGFP</B></TD><TD>87,1%</TD><TD>45,4%</TD></TR><TR><TD>7.4</TD><TD>10µg RC, pHelper, 20µg<B>eGFP</B></TD><TD>86,6%</TD><TD>46,9%</TD></TR><TR><TD>7.5</TD><TD>10µg RC, pHelper, 40µg<B>eGFP</B></TD><TD>89,8%</TD><TD>0,5% ??? (negative control?)</TD></TR><TR><TD>7.6</TD><TD>10µg RC, pHelper, 40µg<B>eGFP</B></TD><TD>84,8%</TD><TD>54,2%</TD></TR><TR><TD>8.2</TD><TD>non treated cells</TD><TD>90,9%</TD></TR><TR><TD>8.3</TD><TD>10µg RC, pHelper, 3,3µg YFP</TD><TD>83,5%</TD><TD>41,2%</TD></TR><TR><TD>8.4</TD><TD>10µg RC, pHelper, 3,3µg YFP</TD><TD>84,9%</TD><TD>42,6%</TD></TR><TR><TD>8.5</TD><TD>negative control ?</TD><TD>86,7%</TD><TD>26,8%</TD></TR><TR><TD>8.6</TD><TD>negative control ?</TD><TD>84,3%</TD><TD>33,7%</TD></TR></TBODY></TABLE><UL><LI> Seeding HT1080 cells for transduction with viral stocks from 23.08.2010:
</LI><LI> 3x 6er wells(Transductionplan 23.08.2010):
</LI></UL><H4><SPAN class="mw-headline" id="Miniprep_of_pSBC13_Bla"><P style="font-size:15px; background-color:#66bbff;"><B>Miniprep of pSBC13_Bla</B></P></SPAN></H4><P><B>Investigator: Chris L.</B></P><P style="font-size:13px; color: blue;">Plasmid Mini-Prep</P><P>
Gycerol stock was prepared and labeled for pSB1C3_Bla clone 1.1 (B236), 1.2 (B237), 2.1 (B238) and 2.2 (B239).B167.
</P><P>Miniprep was performed according to protocol.</P><P>Samples were labeled:</P><UL><LI>pSB1C3_Bla clone 1.1 = P286
</LI><LI>pSB1C3_Bla clone 1.2 = P287
</LI><LI>pSB1C3_Bla clone 2.1 = P288
</LI><LI>pSB1C3_Bla clone 2.2 = P289
</LI></UL><P>Concentrations:
</P><UL><LI>pSB1C3_Bla clone 1.1 (P286): 95.24 ng/µl
</LI><LI>pSB1C3_Bla clone 1.2 (P287): 84.92 ng/µl
</LI><LI>pSB1C3_Bla clone 2.1 (P288): 43.66 ng/µl
</LI><LI>pSB1C3_Bla clone 2.2 (P289): 43.95 ng/µl
</LI></UL><P><B>Test digestion:</B>
pSB1C3_Bla 1.1, 1.2, 2.1 and 2.2 were digested with NotI.
pSB1C3_Bla 1.1, 1.2, 2.1 and 2.2 were digested with SspI and SalI.
pSB1C3_Bla 1.1, 1.2, 2.1 and 2.2 were digested with BamHI and PvuII.
</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of <B>P286</B>/µl </TD><TD>volume of <B>P286</B>/µl</TD><TD>volume of <B>P286/</B>µl </TD><TD>volume of <B>P287</B>/µl </TD><TD>volume of <B>P287</B>/µl</TD><TD>volume of <B>P287</B>/µl</TD><TD>volume of <B>P288</B>µl </TD><TD>volume of <B>P288</B>/µl</TD><TD>volume of <B>P288</B>µl </TD><TD>volume of <B>P289</B>/µl</TD><TD>volume of <B>P289</B>/µl</TD><TD>volume of <B>P289</B>/µl
</TD></TR><TR><TD> DNA  </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 4 </TD><TD> 4 </TD><TD> 4 </TD><TD> 4 </TD><TD> 4 </TD><TD> 4
</TD></TR><TR><TD> BSA (10x) </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1
</TD></TR><TR><TD> Buffer 4 (10x) </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1
</TD></TR><TR><TD>Enzyme NotI </TD><TD>0,5 </TD><TD>0 </TD><TD>0 </TD><TD>0,5 </TD><TD>0 </TD><TD>0 </TD><TD>0,5 </TD><TD>0 </TD><TD>0 </TD><TD>0,5 </TD><TD>0 </TD><TD>0
</TD></TR><TR><TD>Enzyme SspI </TD><TD>0 </TD><TD>0,5 </TD><TD>0 </TD><TD>0 </TD><TD>0,5 </TD><TD>0 </TD><TD>0 </TD><TD>0,5 </TD><TD>0 </TD><TD>0 </TD><TD>0,5 </TD><TD>0
</TD></TR><TR><TD>Enzyme SalI </TD><TD>0 </TD><TD>0,5 </TD><TD>0 </TD><TD>0 </TD><TD>0,5 </TD><TD>0 </TD><TD>0 </TD><TD>0,5 </TD><TD>0 </TD><TD>0 </TD><TD>0,5 </TD><TD>0
</TD></TR><TR><TD>Enzyme BamHI </TD><TD>0 </TD><TD>0 </TD><TD>0,5 </TD><TD>0 </TD><TD>0 </TD><TD>0,5 </TD><TD>0 </TD><TD>0 </TD><TD>0,5 </TD><TD>0 </TD><TD>0 </TD><TD>0,5
</TD></TR><TR><TD>Enzyme PvuII </TD><TD>0 </TD><TD>0 </TD><TD>0,5 </TD><TD>0 </TD><TD>0 </TD><TD>0,5 </TD><TD>0 </TD><TD>0 </TD><TD>0,5 </TD><TD>0 </TD><TD>0 </TD><TD>0,5
</TD></TR><TR><TD>H2O </TD><TD>5.5 </TD><TD>5 </TD><TD>5 </TD><TD>5,5 </TD><TD>5 </TD><TD>5 </TD><TD>3.5 </TD><TD>3 </TD><TD>3 </TD><TD>3.5 </TD><TD>3 </TD><TD>3
</TD></TR><TR><TD><B>Total volume /µl</B></TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10
</TD></TR></TBODY></TABLE><P>
Incubation time: 1 h, Incubation temperature: 37°

2 Gels were made:

0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , running time: 15 minutes at 90 Volt and 30 minutes at 120 Volt 
2µl loading dye (6x) for the sample, Marker: 5 µl GeneRuler ladder mix (Fermentas)
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (6x)/µl
</TH><TH>Expected size 1 (Geneious)
</TH><TH>Expected size 2 (Geneious)
</TH></TR><TR><TD>P286 NotI /µl
</TD><TD>10 µl
</TD><TD> 2 µl
</TD><TD> 905 bp
</TD><TD>2052 bp
</TD></TR><TR><TD>P286 SspI/SalI /µl
</TD><TD>10 µl
</TD><TD> 2 µl
</TD><TD> 854 bp
</TD><TD>2099 bp
</TD></TR><TR><TD>P286 BamHI/PvuII/µl
</TD><TD>10 µl
</TD><TD> 2 µl
</TD><TD> 837 bp
</TD><TD>2116 bp
</TD></TR><TR><TD>P287 NotI /µl
</TD><TD>10 µl
</TD><TD> 2 µl
</TD><TD> 905 bp
</TD><TD>2052 bp
</TD></TR><TR><TD>P287 SspI/SalI /µl
</TD><TD>10 µl
</TD><TD> 2 µl
</TD><TD> 854 bp
</TD><TD>2099 bp
</TD></TR><TR><TD>P287 BamHI/PvuII/µl
</TD><TD>10 µl
</TD><TD> 2 µl
</TD><TD> 837 bp
</TD><TD>2116 bp
</TD></TR><TR><TD>P288 NotI /µl
</TD><TD>10 µl
</TD><TD> 2 µl
</TD><TD> 905 bp
</TD><TD>2052 bp
</TD></TR><TR><TD>P288 SspI/SalI /µl
</TD><TD>10 µl
</TD><TD> 2 µl
</TD><TD> 854 bp
</TD><TD>2099 bp
</TD></TR><TR><TD>P288 BamHI/PvuII/µl
</TD><TD>10 µl
</TD><TD> 2 µl
</TD><TD> 837 bp
</TD><TD>2116 bp
</TD></TR><TR><TD>P289 NotI /µl
</TD><TD>10 µl
</TD><TD> 2 µl
</TD><TD> 905 bp
</TD><TD>2052 bp
</TD></TR><TR><TD>P289 SspI/SalI /µl
</TD><TD>10 µl
</TD><TD> 2 µl
</TD><TD> 854 bp
</TD><TD>2099 bp
</TD></TR><TR><TD>P289 BamHI/PvuII/µl
</TD><TD>10 µl
</TD><TD> 2 µl
</TD><TD> 837 bp
</TD><TD>2116 bp
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>Sample P286 NotI /µl
</TH><TH>Sample P286 SspI/SalI /µl
</TH><TH>Sample P286 BamHI/PvuII/µl
</TH><TH>Sample P287 NotI /µl
</TH><TH>Sample P287 SspI/SalI /µl
</TH><TH>Sample P287 BamHI/PvuII/µl
</TH><TH>Marker
</TH></TR><TR><TH>Lane
</TH><TD> 1
</TD><TD> 3
</TD><TD> 4
</TD><TD> 5
</TD><TD> 7
</TD><TD> 8
</TD><TD> 9
</TD><TD> 11
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>Sample P288 NotI /µl
</TH><TH>Sample P288 SspI/SalI /µl
</TH><TH>Sample P288 BamHI/PvuII/µl
</TH><TH>Sample P289 NotI /µl
</TH><TH>Sample P289 SspI/SalI /µl
</TH><TH>Sample P289 BamHI/PvuII/µl
</TH><TH>Marker
</TH></TR><TR><TH>Lane
</TH><TD> 1
</TD><TD> 3
</TD><TD> 4
</TD><TD> 5
</TD><TD> 7
</TD><TD> 8
</TD><TD> 9
</TD><TD> 11
</TD></TR></TBODY></TABLE><P style="font-size:13px; color:#66bbff;"><B>Comments:</B> Strange results, seems like the test digestion didn't work out. Repitition of test digestion tomorrow.</P><H3><SPAN class="mw-headline" id="99._labday_24.08.2010"><P style="font-size:17px; background-color:#00dd77;">99. labday 24.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Sequence_analysis_of_pSB1C3_CFP_middlelinker"><P style="font-size:15px; background-color:#66bbff;"><B>Sequence analysis of pSB1C3_CFP_middlelinker</B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><UL><LI><B>pSB1C3_CFP_middlelinker is ready</B></LI></UL><H4><SPAN class="mw-headline" id="Harvest_viral_particles.2C_Transduction_of_HT1080"><P style="font-size:15px; background-color:#66bbff;"><B>Harvest viral particles, Transduction of HT1080</B></P></SPAN></H4><P><B>Investigator: Kerstin</B></P><UL><LI>Harvest viral particles following the standard protocol
</LI></UL><UL><LI>Transduction of three 6-well plates:
</LI></UL><P>Plate 1 YFP: 150.000 cells per well</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>500µl virus (1) </TD><TD>500µl virus (2) </TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>500µl virus (1) </TD><TD>500µl virus (2) </TD></TR></TBODY></TABLE><P>Plate 2 YFP: 150.000 cells per well</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>500µl virus (3) </TD><TD>500µl virus (4) </TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>500µl virus (3) </TD><TD>500µl virus (4) </TD></TR></TBODY></TABLE><P>Plate 3 YFP: 200.000 cells per well</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>500µl virus (5) </TD><TD>500µl virus (9) </TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD> 500µl virus (9) </TD><TD> 500µl virus (5) </TD></TR></TBODY></TABLE><UL><LI>(1)= 10µg RC, 10µg pHelper, 10µg GOI (YFP); pH(2xHBS)=7,06
</LI><LI>(2)= 10µg RC, 10µg pHelper, 10µg GOI (YFP); pH(2xHBS)=7,08
</LI><LI>(3)= 10µg RC, 10µg pHelper, 10µg GOI (YFP); pH(2xHBS)=7,10
</LI><LI>(4)= 10µg RC, 10µg pHelper, 10µg GOI (YFP); pH(2xHBS)=7,12
</LI><LI>(5)= 10µg RC, 10µg pHelper, 10µg GOI (YFP); pH(2xHBS)=7,14
</LI><LI>(9)= 10µg RC, 10µg pHelper, 3,3µg GOI (<B>eGFP</B>)
</LI></UL><H4><SPAN class="mw-headline" id="Mini-prep_of_SDM_NgoMIV_CD"><P style="font-size:15px; background-color:#66bbff;"><B>Mini-prep of SDM NgoMIV CD</B></P></SPAN></H4><P><B>Investigator: Kira</B>
Motivation: Harvest DNA for further analysis, e.g. test digestion
</P><P>Mini-prep was performed on 3 colonies.
</P><P>c(clone1) = 358, 05 ng/ul
c(clone2) = 352, 10 ng/ul
c(clone 3) = 376, 53 ng/ul
</P><H4><SPAN class="mw-headline" id="Sent_for_sequencing:_pSB1C3_6xHis"><P style="font-size:15px; background-color:#66bbff;"><B>Sent for sequencing: pSB1C3_6xHis</B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><UL><LI> Plasmid: <B>P84</B> c= 107,8 ng/µl
</LI><LI> Primer: VR2
</LI><LI> tube number: JG84
</LI></UL><H4><SPAN class="mw-headline" id="Subcloning_Cap_into_pAAV_RC_ins-rep"><P style="font-size:15px; background-color:#66bbff;"><B>Subcloning Cap into pAAV_RC_ins-rep</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P>Comment: Sequencing of pAAV_RC_final showed that Cap was not inserted succsessfully. Therefore it has to be repeated.
<B>Digestion of vector and insert:</B></P><UL><LI>Insert: pMA_RepCap Vector_SDM_InsPvuII clone 1 (P211)
</LI><LI>Vector: pAAV_RC_ins-rep clone1 (P250)</LI></UL><P>
A two-step digestion was performed: <B>1st step</B></P><TABLE border="1"><TBODY><TR><TD align="right"><B>P211 / µl</B></TD><TD><B>P250 / µl</B></TD></TR><TR><TD>DNA</TD><TD align="right">5,8  </TD><TD align="right"> 5,2
</TD></TR><TR><TD>buffer 3</TD><TD align="right">2 </TD><TD align="right"> 2
</TD></TR><TR><TD> Enzyme BsiWI</TD><TD align="right">1  </TD><TD align="right"> 1
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">11,2 </TD><TD align="right"> 11,8
</TD></TR><TR><TD>total volume</TD><TD align="right">20 </TD><TD align="right">20
</TD></TR></TBODY></TABLE><P><B>2nd step</B></P><TABLE border="1"><TBODY><TR><TD align="right"><B>P211 / µl</B></TD><TD><B>P250 / µl</B></TD></TR><TR><TD>Mix obtained from step 1 </TD><TD align="right">20  </TD><TD align="right"> 20
</TD></TR><TR><TD> Enzyme BspMI</TD><TD align="right">1  </TD><TD align="right"> 1
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">1,25 </TD><TD align="right"> 1,25
</TD></TR><TR><TD>total volume</TD><TD align="right">22,25 </TD><TD align="right">22,25
</TD></TR></TBODY></TABLE><P>
Samples were loaded on a 0,8% agarose gel an run at 115V for about 60 minutes.
</P><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>pAAV RC final</CAPTION></TABLE><P><B>Gelextraction:</B></P><P>Gelex was performed according to protocol exept for elution was done with 60 µl instead of 20 µl.</P><UL><LI>c(P211)= 1,14 ng/µl
</LI><LI>c(P250)= 5,60 ng/µl
</LI></UL><P><B>T4 Ligation:</B>
T4 ligation was performed according to protocol.
used DNA amounts: </P><UL><LI>v(P211)= 2,56 µl
</LI><LI>v(P250)= 5,44 µl
</LI></UL><P><B>Transformation:</B>
Trafo was performed according to protocol using XL1b cells.
</P><H4><SPAN class="mw-headline" id="Test_digestion_of_pCerulean_.28p273.29"><P style="font-size:15px; background-color:#66bbFF;"><B>Test digestion of pCerulean (p273) </B></P></SPAN></H4><P><B>Investigator: Anissa</B></P><P style="font-size:13px; color:#66bbff;"><B>Comments:</B>Sequencing of pCerulean showed strange results, because the qualtity of sequenzing was not good. For working further with pCerulean, we started with a test digestion, to be sure, everything works.
Because the sequencing showed no AgeI, we cut one time with AgeI and EcoRI, the other time with NotI.
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>p273/µL</B></TD><TD align="left"><B>p273/µL</B></TD></TR><TR><TD align="left"> DNA  </TD><TD align="left"> 2</TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left">1,5</TD><TD align="left">1,5
</TD></TR><TR><TD align="left"> Buffer 4 (10x)  </TD><TD align="left"> 1,5</TD><TD align="left"> 1,5
</TD></TR><TR><TD align="left"> Enzyme 1 </TD><TD align="left">0,5 NotI</TD><TD align="left">0,5 EcoRI
</TD></TR><TR><TD align="left"> Enzyme 2  </TD><TD align="left">0,5 NotI </TD><TD align="left">0,5 AgeI
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 9</TD><TD align="left"> 9
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"> 15</TD><TD align="left">15
</TD></TR></TBODY></TABLE><P style="font-size:13px; color:#66bbff;"><B>Comments:</B> Results of digestion seem to be all right</P><H4><SPAN class="mw-headline" id="Cloning_of_VP1up_into_PCerulean"><P style="font-size:15px; background-color:#66bbFF;"><B>Cloning of VP1up into PCerulean </B></P></SPAN></H4><P><B>Investigator: Anissa</B>
VP1up was cut out of pSB1C3 and into pCerulean
</P><UL><LI> Digestion:
<TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>Vector/µL</B></TD><TD align="left"><B>Insert/µL</B></TD></TR><TR><TD align="left"> DNA  </TD><TD align="left"> 3,7</TD><TD align="left">8,4
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left">1,5</TD><TD align="left">2
</TD></TR><TR><TD align="left"> Buffer 4 (10x)  </TD><TD align="left"> 1,5</TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> Enzyme 1 EcoRI  </TD><TD align="left">1</TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Enzyme 2 PstI   </TD><TD align="left"> 1</TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left">6,3</TD><TD align="left"> 5,6
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left">15</TD><TD align="left"> 20
</TD></TR></TBODY></TABLE></LI><LI> Gel:

A 1% agarose-gel was made, after 30 minutes samples were cut</LI><LI> Gelextraction was performed according the standardprotocol </LI><LI> Ligation:
<TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>used volume for T4 ligation/µL</B></TD><TD align="left"><B>concentration /ng/µl</B></TD></TR><TR><TD align="left"> Vector  </TD><TD align="left"> 5,15</TD><TD align="left">27,7
</TD></TR><TR><TD align="left">Insert </TD><TD align="left">2,85</TD><TD align="left">15,4
</TD></TR></TBODY></TABLE><P>In addition 1µl T4-ligase and 1µl T4-ligase-buffer were added
</P></LI><LI>Transformation: was performes into Xl1 blue according the standard-protocol
</LI></UL><H4><SPAN class="mw-headline" id="Sequence_analysis_of_sequencing_reads_prepared_yesterday_.2823.08.2010.29"><P style="font-size:15px; background-color:#66bbff;"><B>Sequence analysis of sequencing reads prepared yesterday (23.08.2010)</B></P></SPAN></H4><P><B>Investigator: Bea </B></P><P style="font-size:13px; color:#cc0033;"><I><B>GENERAL COMMENT:</B> pSB1C3 {NLS and VP1up} were sent for sequencing. Aswell as the pAAV_RC_final which contains all four mutations in the Rep-Cap gene and the integrated and synthesiezd rep and cap gene and the plasmid pCerulean whith the CMV promoter and sv40 polyadenylation site.</I></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: Sequence analysis of pAAV_RC_final containing the subcloned &quot;rep&quot; (ordered) and &quot;Cap (ordered) plus the four mutations to delete the restriction sites PstI, BamHi and SalI. </P><UL><LI>Primer used: </LI><UL><LI>VP1 primer for pKex</LI><LI>GATC_std_SK</LI><LI>Primer Cap 2800 rev</LI><LI>Primer Cap 2800 for</LI><LI>Primer Cap 3500 for</LI></UL><LI>Plasmid sequenced: P283 </LI><LI>Sequence sample: iGEM4</LI><LI>Stored in Geneious-folder: RepCap insert ordered</LI></UL><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>pAAV RC final</CAPTION></TABLE><B>Results:</B><P style="color:#00bbff;">Sequencing of the mutation ok. Rep integration worked. Cap integration needs to be repeated.</P><P><B>Next steps: Repetition of the integration of the synthesized cap gene performed by Stefan (see topic). </B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: Sequence analysis of pSB1C3_NLS </P><UL><LI>Primer used: VR-2</LI><LI>Plasmid sequenced: P282 </LI><LI>Sequence sample: iGEM3_O51_VR-2 </LI><LI>Stored in folder: pSB1C3_NLS</LI></UL><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>pSB1C3_NLS</CAPTION></TABLE><B>Results:</B><P style="color:#00bbff;">Insertion of the nuclear localisation signal (NLS) into the pSB1C3_NLS as one step in the N-terminal insertion of targeting molecules. Sequencing results look good. prefix and suffix ok!</P><P><B>Next steps: Fuse NLS to targeting molecule. </B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: Sequence analysis of pSB1C3_VP1up</P><UL><LI>Primer used: VF-2</LI><LI>Plasmid sequenced: P280 </LI><LI>Sequence sample: iGEM1_VF-2 </LI><LI>Stored in folder: N-Terminal targeting --&gt; pSB1C3_VP1up</LI></UL><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>pSB1C3_VP1up</CAPTION></TABLE><B>Results:</B><P style="color:#00bbff;"> Prefix ok, but annotation is wrong. VP1up should start with: GC!!  Suffix ok! The additional EcoRI before the prefix standard can be due to the bad sequence read quality.</P><P><B>Next steps: Clone VP1up into pCerulean in order to obtain the plasmid with a CMV promoter and the sv40 polyadenylation site. </B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: Sequence analysis of pCerulean </P><UL><LI>Primer used: GATC_CMV-F</LI><LI>Plasmid sequenced: P273 </LI><LI>Sequence sample: iGEM2_CMV-F </LI><LI>Stored in folder: N-Terminal targeting --&gt; pCerulean</LI></UL><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>pCerulean</CAPTION></TABLE><B>Results:</B><P style="color:#00bbff;"> It seem that the suffix is not correct which means that AgeI, PstI and NotI is missing. This can be due to the bad sequence quality. Prefix is ok!
</P><P><B>Next steps: For verification a test digestion with the missing restriction sites will be perforemd and another round of sequencing will be conducted. </B></P><P><A href="http://www.molbiotech.uni-freiburg.de/iGEM/wiki2010/index.php/Laborjournal" class="external text" rel="nofollow">top of page</A></P><H4><SPAN class="mw-headline" id="Test_digestion_of_SDM_NgoMIV_CD"><P style="font-size:15px; background-color:#66bbFF;"><B>Test digestion of SDM NgoMIV CD </B></P></SPAN></H4><P><B>Investigator: Kira</B>
Motivation: In order to check if site directed mutagenesis was successfull, test digestion was performed with 3 clones, which were picked yesterday as well as with 'original' DNA, which still contains all the restriction sites
</P><LI> Test digestion:
<TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>clone 1/µL</B></TD><TD align="left"><B>clone 2/µL</B></TD><TD align="left"><B>clone 3/µL</B></TD><TD align="left"><B>original/µL</B></TD></TR><TR><TD align="left"> DNA (500ng)  </TD><TD align="left"> 1,4</TD><TD align="left">1,4</TD><TD align="left"> 1,4</TD><TD align="left"> 1,4
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 1</TD><TD align="left"> 1</TD><TD align="left"> 1</TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Buffer 4 (10x)  </TD><TD align="left"> 1</TD><TD align="left"> 1</TD><TD align="left"> 1</TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Enzyme 1 NgoMIV </TD><TD align="left">0,5</TD><TD align="left"> 0,5</TD><TD align="left">0,5</TD><TD align="left">0,5
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left">6,1</TD><TD align="left"> 6,1</TD><TD align="left"> 6,1</TD><TD align="left"> 6,1
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left">10</TD><TD align="left"> 10</TD><TD align="left"> 10</TD><TD align="left"> 10
</TD></TR></TBODY></TABLE><DIV class="thumb tright"><DIV class="thumbinner" style="width:74px;"><DIV class="thumbcaption">Bitte schneid mich aus !</DIV></DIV></DIV><P>According to the gel, NgoMIV SDM was successful in clones 1 and 2. Clone 3 well reveales 2 bands of unknown origin. Samples 1 and 3 were sent for sequencing.
</P><H4><SPAN class="mw-headline" id="Picking_clones_of_pSB1C3_Affibody_linker_and_pSB1C3_.C3.9F-Globin_YFP_hGH_rITR"><P style="font-size:15px; background-color:#66bbff;"><B>Picking clones of pSB1C3_Affibody_linker and pSB1C3_ß-Globin_YFP_hGH_rITR  </B></P></SPAN></H4><P><B>Investigator: Anna </B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: A 1:1000 dilution of pSB1C3_Affibody_GSAT-Linker was prepared. <B>To do</B>: Mini-Prep of pSB1C3_Affibody_(Short/Middle/Long and SEG-Linker) and pSB1C3_ß-Globin_YFP_hGH_rITR. Picking clones of pSB1C3_Affibody_GSAT-Linker. </P><H4><SPAN class="mw-headline" id="Test_digestion_of_pSB1C3_SDM_SspI_Bla14FM_.28P222.29.2C_pSB1C3_BLA_.28P286.2C_P288.29"><P style="font-size:15px; background-color:#66bbff;"><B>Test digestion of pSB1C3_SDM_SspI_Bla14FM (P222), pSB1C3_BLA (P286, P288)  </B></P></SPAN></H4><P><B>Investigator: Achim</B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>New test digestion of two SDM-attempts, this time we also digested the original vector containing the PvuII restriction site to see differences.</P></LI><LI>The original vector is being cut as expected: the uncut vector can be seen in different conformations, the linearized vector shows one distinct band and the digestions cutting out BLA show a band at ~900 bp. Our SDM attempts show inconsistent results, not only does the Pvu site still seem to be in the backbone, there are no more proper insert bands visible either. We therefore conclude that either the DpnI digestion or the dilution of the transformation must have gone wrong (differing plasmids...). Tomorow we'll try a new cloning approach using an old vector with deleted Pvu restriction site and MscI &amp; XbaI digestion. In case this also fails we also yet have to prep and test digest the repetition of the mutagenesis which was carried out by Stefan.
</LI><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>pSB1C3_VP1up</CAPTION></TABLE><B>Results:</B><P style="color:#00bbff;"> Prefix ok, but annotation is wrong. VP1up should start with: GC!!  Suffix ok! The additional EcoRI before the prefix standard can be due to the bad sequence read quality.</P><P><B>Next steps: Clone VP1up into pCerulean in order to obtain the plasmid with a CMV promoter and the sv40 polyadenylation site. </B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: Sequence analysis of pCerulean </P><UL><LI>Primer used: GATC_CMV-F</LI><LI>Plasmid sequenced: P273 </LI><LI>Sequence sample: iGEM2_CMV-F </LI><LI>Stored in folder: N-Terminal targeting --&gt; pCerulean</LI></UL><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>pCerulean</CAPTION></TABLE><B>Results:</B><P style="color:#00bbff;"> It seem that the suffix is not correct which means that AgeI, PstI and NotI is missing. This can be due to the bad sequence quality. Prefix is ok!
</P><P><B>Next steps: For verification a test digestion with the missing restriction sites will be perforemd and another round of sequencing will be conducted. </B></P><P><A href="http://www.molbiotech.uni-freiburg.de/iGEM/wiki2010/index.php/Laborjournal" class="external text" rel="nofollow">top of page</A></P><H4><SPAN class="mw-headline" id="Test_digestion_of_SDM_NgoMIV_CD_2"><P style="font-size:15px; background-color:#66bbFF;"><B>Test digestion of SDM NgoMIV CD </B></P></SPAN></H4><P><B>Investigator: Kira</B>
Motivation: In order to check if site directed mutagenesis was successfull, test digestion was performed with 3 clones, which were picked yesterday as well as with 'original' DNA, which still contains all the restriction sites
</P><LI> Test digestion:
<TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>clone 1/µL</B></TD><TD align="left"><B>clone 2/µL</B></TD><TD align="left"><B>clone 3/µL</B></TD><TD align="left"><B>original/µL</B></TD></TR><TR><TD align="left"> DNA (500ng)  </TD><TD align="left"> 1,4</TD><TD align="left">1,4</TD><TD align="left"> 1,4</TD><TD align="left"> 1,4
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 1</TD><TD align="left"> 1</TD><TD align="left"> 1</TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Buffer 4 (10x)  </TD><TD align="left"> 1</TD><TD align="left"> 1</TD><TD align="left"> 1</TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Enzyme 1 NgoMIV </TD><TD align="left">0,5</TD><TD align="left"> 0,5</TD><TD align="left">0,5</TD><TD align="left">0,5
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left">6,1</TD><TD align="left"> 6,1</TD><TD align="left"> 6,1</TD><TD align="left"> 6,1
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left">10</TD><TD align="left"> 10</TD><TD align="left"> 10</TD><TD align="left"> 10
</TD></TR></TBODY></TABLE><DIV class="thumb tright"><DIV class="thumbinner" style="width:74px;"><DIV class="thumbcaption">Bitte schneid mich aus !</DIV></DIV></DIV><P>According to the gel, NgoMIV SDM was successful in clones 1 and 2. Clone 3 well reveales 2 bands of unknown origin. Samples 1 and 3 were sent for sequencing.
</P><H4><SPAN class="mw-headline" id="Picking_clones_of_pSB1C3_Affibody_linker_and_pSB1C3_.C3.9F-Globin_YFP_hGH_rITR_2"><P style="font-size:15px; background-color:#66bbff;"><B>Picking clones of pSB1C3_Affibody_linker and pSB1C3_ß-Globin_YFP_hGH_rITR  </B></P></SPAN></H4><P><B>Investigator: Anna </B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: A 1:1000 dilution of pSB1C3_Affibody_GSAT-Linker was prepared. <B>To do</B>: Mini-Prep of pSB1C3_Affibody_(Short/Middle/Long and SEG-Linker) and pSB1C3_ß-Globin_YFP_hGH_rITR. Picking clones of pSB1C3_Affibody_GSAT-Linker. </P><H4><SPAN class="mw-headline" id="Test_digestion_of_pSB1C3_SDM_SspI_Bla14FM_.28P222.29.2C_pSB1C3_BLA_.28P286.2C_P288.29_2"><P style="font-size:15px; background-color:#66bbff;"><B>Test digestion of pSB1C3_SDM_SspI_Bla14FM (P222), pSB1C3_BLA (P286, P288)  </B></P></SPAN></H4><P><B>Investigator: Achim</B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>New test digestion of two SDM-attempts, this time we also digested the original vector containing the PvuII restriction site to see differences.</P></LI><LI>The original vector is being cut as expected: the uncut vector can be seen in different conformations, the linearized vector shows one distinct band and the digestions cutting out BLA show a band at ~900 bp. Our SDM attempts show inconsistent results, not only does the Pvu site still seem to be in the backbone, there are no more proper insert bands visible either. We therefore conclude that either the DpnI digestion or the dilution of the transformation must have gone wrong (differing plasmids...). Tomorow we'll try a new cloning approach using an old vector with deleted Pvu restriction site and MscI &amp; XbaI digestion. In case this also fails we also yet have to prep and test digest the repetition of the mutagenesis which was carried out by Stefan.
</LI><H3><SPAN class="mw-headline" id="100._labday_25.08.2010"><P style="font-size:17px; background-color:#00dd77;">100. labday 25.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Cloning_of_pSB1C3_Rep40.2C_pSB1C3_Rep68_and_pMA_RC_insert"><P style="font-size:15px; background-color:#66bbFF;"><B>Cloning of pSB1C3_Rep40, pSB1C3_Rep68 and pMA_RC_insert </B></P></SPAN></H4><P><B>Investigator: Chris L.</B></P><UL><LI>buffer used: 2; Restriction-enzymes used: Enzyme 1: HindIII ; Enzyme 2: BstEII
</LI></UL><P>Plasmids:
</P><UL><LI>pSB1C3_Rep40 <B>P231</B></LI><LI>pSB1C3_Rep68 <B>P266</B></LI><P>Insert:
</P><LI>pMA_RC_insert <B>P190</B></LI></UL><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>P190/µL</B></TD><TD align="left"><B>P231/µL</B></TD><TD align="left"><B>P266/µL</B></TD></TR><TR><TD align="left"> DNA  </TD><TD align="left"> 5.9 </TD><TD align="left"> 6.4 </TD><TD align="left"> 5
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 2 </TD><TD align="left">2</TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> Buffer 2 (10x)  </TD><TD align="left"> 2 </TD><TD align="left"> 2</TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> Enzyme 1 HindIII   </TD><TD align="left"> 1 </TD><TD align="left"> 1 </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Enzyme 2 BstEII   </TD><TD align="left"> 1 </TD><TD align="left"> 1 </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 8.1 </TD><TD align="left"> 7.6 </TD><TD align="left"> 9
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"> 20 </TD><TD align="left"> 20</TD><TD align="left"> 20
</TD></TR></TBODY></TABLE><UL><LI>Incubation: 90 minutes; 37°C with HindIII</LI><LI>Incubation: 90 minutes; 60°C with BstEII</LI></UL><P><B>Agarosegel</B>
0.5 g Agarose, 50 ml TAE (1 %), 3 µL GELRED, 5 min at 90 Volt, 40 min at 115 Volt
<B>Concentrations</B> measured via NanoDrop:
</P><UL><LI>pSB1C3_Rep40: 81.97 ng/µl
</LI><LI>pSB1C3_Rep68: 48,68 ng/µl
</LI><LI>RC_Insert: 3,60 ng/µl
</LI></UL><P><B>T4 Ligation of pSB1C3_Rep40 with RC_Insert</B> 
Volume insert: 6,54 µl
Volume vector: 1,46 µl<B>T4 Ligation of pSB1C3_Rep68 with RC_Insert</B> 
Volume insert: 5,46 µl
Volume vector: 2,54 µl<B>Trafo</B> was prepared with XL1blue and Cm
</P><H4><SPAN class="mw-headline" id="Picking_clones_of_pAAV_RC-ins_rep_cap_and_pCeruleanVP1up"><P style="font-size:15px; background-color:#66bbff;"><B>Picking clones of pAAV_RC-ins_rep_cap and pCeruleanVP1up</B></P></SPAN></H4><P><B>Investigator: Chris L. </B></P><P><B>To do</B>: Mini-Prep of pAAV_RC-ins_rep_cap and pCeruleanVP1up.
</P><H4><SPAN class="mw-headline" id="Cloning_of_pSB1C3_6xHis_with_middlelinker_and_pSB1C3_6xHis_with_pGA14_mVenus_YFP"><P style="font-size:15px; background-color:#66bbFF;"><B>Cloning of pSB1C3_6xHis with middlelinker and pSB1C3_6xHis with pGA14_mVenus_YFP</B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><UL><LI>buffer used: 4; Restriction-enzymes used: Enzyme 1: AgeI ; Enzyme 2: PstI ; Enzyme3: NgoMIV
</LI></UL><UL><LI>Plasmids:</LI><UL><LI>pSB1C3_6xHis <B>P84</B></LI><LI>pGA14_middle linker <B>P65</B></LI><LI>GA14_mVenus_YFP <B>P60</B></LI></UL></UL><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>Mastermix</B></TD><TD align="left"><B>P84/µL</B></TD><TD align="left"><B>P65/µL</B></TD><TD align="left"><B>P60/µL</B></TD></TR><TR><TD align="left"> DNA  </TD><TD align="left"> -</TD><TD align="left"> 13,91</TD><TD align="left"> 13,52</TD><TD align="left"> 8,93
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left">-</TD><TD align="left">2</TD><TD align="left"> 2</TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> Buffer 4 (10x)  </TD><TD align="left"> -</TD><TD align="left"> 2</TD><TD align="left"> 2</TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> Enzyme 1 AgeI   </TD><TD align="left">-</TD><TD align="left"> 1</TD><TD align="left"> -</TD><TD align="left"> -
</TD></TR><TR><TD align="left"> Enzyme 2  PstI   </TD><TD align="left"> -</TD><TD align="left"> 1</TD><TD align="left"> 1 </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Enzyme 3  NgoMIV   </TD><TD align="left"> -</TD><TD align="left"> -</TD><TD align="left"> 1</TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> -</TD><TD align="left"> 0,09</TD><TD align="left"> 0,48</TD><TD align="left"> 5,07
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"> -</TD><TD align="left"> 20</TD><TD align="left"> 20</TD><TD align="left"> 20
</TD></TR></TBODY></TABLE><UL><LI>Incubation:110 minutes; 37°C</LI></UL><P><B>Agarosegel</B>
0.5 g Agarose, 50 ml TAE (1 %), 3 µL GELRED, at 120 Volt
</P><P><B>concentrations</B> measured via NanoDrop:
</P><UL><LI>SB1C3_6xHis:  1,7 ng/µl
</LI><LI>middle linker:  3,8 ng/µl
</LI><LI>mVenus_YFP:  3,2 ng/µl
</LI></UL><P><B>T4 Ligation pSB1C3_6xHis_middlelinker</B> 
Volume insert: 0,25 µl
Volume vector: 7,75 µl<B>T4 Ligation pSB1C3_6xHis_mVenus_YFP</B> 
Volume insert: 2,64 µl
Volume vector: 5,36 µl<B>Trafo</B> was prepared with XL1blue and Cm
</P><H4><SPAN class="mw-headline" id="Sequence_analysis_of_pSB1C3_6xHis"><P style="font-size:15px; background-color:#66bbff;"><B>Sequence analysis of pSB1C3_6xHis</B></P></SPAN></H4><P><B>Investigator: Jessica</B><B>P84</B> was sequenced for continuation of working with it
</P><UL><LI><B>pSB1C3_6xHis P84 is ready to use, but is empty. a new tube will be made tomorrow</B></LI></UL><H4><SPAN class="mw-headline" id="Inoculation_of_pGA14_middlelinker_and_pSB1C3_6xHis_clone1"><P style="font-size:15px; background-color:#66bbFF;"><B>Inoculation of pGA14_middlelinker and pSB1C3_6xHis clone1 </B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><UL><LI>both tubes (<B>P65</B> and <B>P84</B>)are empty --&gt; inoculation with glycerolstocks 
</LI><LI><B>B48</B> pGA14_middlelinker was inoculated with 10ml DYT and 10µl Amp
</LI><LI><B>B64</B> pSB1C3_6xHis clone1 was inoculated with 10ml DYT and 10µl Cm
</LI></UL><H4><SPAN class="mw-headline" id="Cloning_of_pSB1C3_SspI_BLA_and_pSB1C3_CFP_PvuII"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of pSB1C3_SspI_BLA and pSB1C3_CFP_PvuII</B></P></SPAN></H4><P><B> Investigator: Achim</B></P><P style="font-size:13px; color:#003399;"><B>Comment</B>:I ligated a sequence from pSB1C3_CFP_PvuII missing the PvuII restriction site into pSB1C3_SspI_BLA to obtain a pSB1C3 vector without PvuII in the cat marker.</P><P>Vector: </P><UL><LI>pSB1C3_SspI_BLA (p222): c= 308,32 ng/µl</LI></UL><P>Insert: </P><UL><LI>pSB1C3_CFP_PvuII (p129): c= 264,86 ng/µl</LI></UL><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">Vector </TD><TD align="right">Insert
</TD></TR><TR><TD> DNA  </TD><TD align="right">3,8 </TD><TD align="right"> 4,9
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2</TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right"> 2
</TD></TR><TR><TD>Enzyme MscI </TD><TD align="right">1 </TD><TD align="right"> 1
</TD></TR><TR><TD>Enzyme XbaI </TD><TD align="right">1</TD><TD align="right"> 1
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">10,2 </TD><TD align="right"> 9,1
</TD></TR><TR><TD><B>Total volume </B></TD><TD align="right">20 </TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><UL><LI>Gelextraction:
</LI></UL><P>0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 110 Volt, running time:45 
Marker: GeneRuler ladder mix (Fermentas)
</P><UL><LI>c(Insert)= 15,66 ng/µl; size: 730 bp</LI><LI>c(Vector)= 14,42 ng/µl; size: 2200 bp
</LI></UL><UL><LI>Ligation of PCR products and vector:
</LI></UL><P>For the Ligation 1µl T4 buffer (2x) and 1µl T4 ligase were used. Incubation time: 60 min due to blunt end ligation
</P><TABLE border="1"><TBODY><TR><TD align="right"><B>vector /µl</B></TD><TD align="right"><B>insert /µl</B></TD></TR><TR><TD> pSB1C3_BLA </TD><TD align="right">0,86 </TD><TD align="right">7,14
</TD></TR></TBODY></TABLE><UL><LI>Transformation:
</LI></UL><P>The transformation was done following the standard protocol using XL1 blue cells.</P><P style="font-size:13px; color:#003399;"><B>Comment</B>: Two clones were picked, but because we already obtained our plasmid via SDM in the meantime, no prep was performed. Glycerol stocks of the two clones were created just in case, B264 and B265.</P><H4><SPAN class="mw-headline" id="Mini-Preps_and_test_digestion_of_pSB1C3_Affibody_Middle-Linker.2C_pSB1C3_Affibody_Short-Linker.2C_pSB1C3_Affibody_SEG-Linker.2C_pSB1C3_Affibody_Long-Linker_and_pSB1C3_betaglobin_mVenus_hGH_rITR"><P style="font-size:15px; background-color:#66bbff;"><B>Mini-Preps and test digestion of pSB1C3_Affibody_Middle-Linker, pSB1C3_Affibody_Short-Linker, pSB1C3_Affibody_SEG-Linker, pSB1C3_Affibody_Long-Linker and pSB1C3_betaglobin_mVenus_hGH_rITR</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><UL><B>Glycerol stocks were prepared:</B><LI>B240 = pSB1C3_Affibody_Middle-Linker clone1</LI><LI>B241 = pSB1C3_Affibody_Middle-Linker clone2</LI><LI>B242 = pSB1C3_Affibody_Short-Linker clone 1</LI><LI>B243 = pSB1C3_Affibody_Short-Linker clone 2</LI><LI>B246 = pSB1C3_Affibody_SEG-Linker clone1</LI><LI>B247 = pSB1C3_Affibody_SEG-Linker clone2</LI><LI>B248 = pSB1C3_Affibody_Long-Linker clone 1</LI><LI>B249 = pSB1C3_Affibody_Long-Linker clone 2</LI><LI>B244 = pSB1C3_betaglobin_mVenus_hGH_rITR clone 1</LI><LI>B245 = pSB1C3_betaglobin_mVenus_hGH_rITR clone 2</LI><P><B>Mini-Prep following the standard protocol</B></P><LI>P290 = pSB1C3_Affibody_Middle-Linker clone1 c= 227,36 ng/µl</LI><LI>P291 = pSB1C3_Affibody_Middle-Linker clone2 c= 219,05 ng/µl</LI><LI>P292 = pSB1C3_Affibody_Short-Linker clone 1 c= 196,63 ng/µl</LI><LI>P293 = pSB1C3_Affibody_Short-Linker clone 2 c= 224,76 ng/µl</LI><LI>P296 = pSB1C3_Affibody_SEG-Linker clone1 c= 218,68 ng/µl</LI><LI>P297 = pSB1C3_Affibody_SEG-Linker clone2 c= 229,66 ng/µl</LI><LI>P298 = pSB1C3_Affibody_Long-Linker clone 1 c= 229,73 ng/µl</LI><LI>P299 = pSB1C3_Affibody_Long-Linker clone 2 c= 202,05 ng/µl</LI><LI>P294 = pSB1C3_betaglobin_mVenus_hGH_rITR clone 1 c= 425,14 ng/µl</LI><LI>P295 = pSB1C3_betaglobin_mVenus_hGH_rITR clone 2 c= 328,28 ng/µl</LI></UL><P><B>Test digestion</B></P><UL><LI>Restriction-enzymes used: for P290-P293 and P296-P299: NotI ; for P294-P295: NgoMIV and PstI </LI></UL><P><B><B>comment:</B></B> The same amount of ingredients were used for P290-P293 and P296-P299, therefore these approaches will be merged into the chart. The same goes for P294-P295.
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>P290-P293 and P296-P299</B></TD><TD align="left"><B>P294-P295</B></TD></TR><TR><TD align="left"> DNA </TD><TD align="left"> 1 </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 1 </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Buffer 4 (10x) </TD><TD align="left"> 1 </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> NotI </TD><TD align="left"> 0,5 </TD><TD align="left"> -
</TD></TR><TR><TD align="left"> NgoMIV </TD><TD align="left"> - </TD><TD align="left"> 0,5
</TD></TR><TR><TD align="left"> PstI </TD><TD align="left"> - </TD><TD align="left"> 0,5
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 6,5 </TD><TD align="left"> 6
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>10</B></TD><TD align="left"><B>10</B></TD></TR></TBODY></TABLE><P>
Incubation time: 70 minutes; Incubation temperature: 37°

0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , running time:5 minutes at 90 Volt and 50 minutes at 115 Volt 
2µl loading dye (6x) for the sample, Marker: GeneRuler ladder mix (Fermentas)
</P><P style="font-size:13px; color:#003399;"><B>Comment</B>: Sizes of fragments look like expected. Clone 1 of each plasmid will be sent for sequencing tomorrow.</P><H4><SPAN class="mw-headline" id="Sequenzing_evaluation_of_SDM_NgoMIV"><P style="font-size:15px; background-color:#66bbff;"><B>Sequenzing evaluation of SDM NgoMIV</B></P></SPAN></H4><P><B>Investigator: Kira</B></P><P>2 samples have been sent for sequencing yesterday. According to the data, both samples do not contain any NgoMIV restriction site anymore. </P><H3><SPAN class="mw-headline" id="101._labday_26.08.2010"><P style="font-size:17px; background-color:#00dd77;">101. labday 26.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Mini-Preps_of_pGA14_middle_linker_and_pSB1C3_6xHis_clone_1"><P style="font-size:15px; background-color:#66bbff;"><B>Mini-Preps of pGA14_middle linker and pSB1C3_6xHis clone 1</B></P></SPAN></H4><P><B>Investigator: Jessica</B></P><UL>
Glycerol stocks were not prepared because I have inoculated from glycerol stocks
<B>Mini-Prep following the standard protocol</B><LI>P301 = pGA14_middle linker (= <B>P65</B>)
c=305,7</LI><LI>P300 = pSB1C3_6xHis clone 1 (= <B>P84</B>)
c=177,5</LI></UL><H4><SPAN class="mw-headline" id="Mini-Preps_and_test_digestion_of_pAAV_RC-ins_rep_cap.2C_pCeruleanVP1up_and_pAAV_RC_ins-rep-cap."><P style="font-size:15px; background-color:#66bbff;"><B>Mini-Preps and test digestion of pAAV_RC-ins_rep_cap, pCeruleanVP1up and pAAV_RC_ins-rep-cap.</B></P></SPAN></H4><P><B>Investigator: Chris L.</B></P><UL><B>Glycerol stocks were prepared:</B><LI>B250 = pCerulean_VP1up clone1</LI><LI>B251 = pCerulean_VP1up clone2</LI><LI>B252 = pCerulean_VP1up clone3</LI><LI>B253 = pCerulean_VP1up clone4</LI><LI>B254 = pSB1C3_VCK_Bla clone1</LI><LI>B255 = pSB1C3_VCK_Bla clone2</LI><LI>B256 = pSB1C3_VCK_Bla clone3</LI><LI>B257 = pSB1C3_VCK_Bla clone4</LI><LI>B262 = pAAV_RC_ins-rep-cap clone1</LI><LI>B263 = pAAV_RC_ins-rep-cap clone2</LI><P><B>Mini-Prep following the standard protocol</B></P><LI>P302 = pCerulean_VP1up clone1c=410,70 ng/µl</LI><LI>P303 = pCerulean_VP1up clone2c=387,78 ng/µl</LI><LI>P304 = pCerulean_VP1up clone3c=321,04 ng/µl</LI><LI>P305 = pCerulean_VP1up clone4c=350,34 ng/µl</LI><LI>P306 = pSB1C3_VCK_Bla clone1c=82,53 ng/µl</LI><LI>P307 = pSB1C3_VCK_Bla clone2c=92,70 ng/µl</LI><LI>P308 = pSB1C3_VCK_Bla clone3c=82,22 ng/µl</LI><LI>P309 = pSB1C3_VCK_Bla clone4c=105,34 ng/µl</LI><LI>P314 = pAAV_RC_ins-rep-cap clone1c=517,04 ng/µl</LI><LI>P315 = pAAV_RC_ins-rep-cap clone2c=444,33 ng/µl</LI></UL><P><B>Test digestion:</B></P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of <B>P302</B>/µl </TD><TD>volume of <B>P303</B>/µl</TD><TD>volume of <B>P304/</B>µl </TD><TD>volume of <B>P305</B>/µl </TD><TD>volume of <B>P306</B>/µl</TD><TD>volume of <B>P306</B>/µl</TD><TD>volume of <B>P306</B>µl </TD><TD>volume of <B>P307</B>/µl</TD><TD>volume of <B>P307</B>µl </TD><TD>volume of <B>P307</B>/µl</TD><TD>volume of <B>P308</B>/µl</TD><TD>volume of <B>P308</B>/µl </TD><TD>volume of <B>P308</B>/µl </TD><TD>volume of <B>P309</B>/µl </TD><TD>volume of <B>P309</B>/µl </TD><TD>volume of <B>P309</B>/µl</TD><TD>volume of <B>P314</B>/µl </TD><TD>volume of <B>P315</B>/µl
</TD></TR><TR><TD> DNA  </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 1 </TD><TD> 1
</TD></TR><TR><TD> BSA (10x) </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1
</TD></TR><TR><TD> Buffer 4 (10x) </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>1 </TD><TD>- </TD><TD>-
</TD></TR><TR><TD> Buffer 2 (10x) </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>-</TD><TD>- </TD><TD>1 </TD><TD>1
</TD></TR><TR><TD>Enzyme NotI </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>0,5 </TD><TD>- </TD><TD>- </TD><TD>0,5 </TD><TD>- </TD><TD>- </TD><TD>0,5 </TD><TD>- </TD><TD>- </TD><TD>0,5 </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>-
</TD></TR><TR><TD>Enzyme SspI </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>0,5 </TD><TD>- </TD><TD>- </TD><TD>0,5 </TD><TD>- </TD><TD>- </TD><TD>0,5 </TD><TD>- </TD><TD>- </TD><TD>0,5 </TD><TD>- </TD><TD>- </TD><TD>-
</TD></TR><TR><TD>Enzyme SalI </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>0,5 </TD><TD>- </TD><TD>- </TD><TD>0,5 </TD><TD>-</TD><TD>- </TD><TD>0,5 </TD><TD>- </TD><TD>- </TD><TD>0,5 </TD><TD>- </TD><TD>-
</TD></TR><TR><TD>Enzyme BamHI </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>0,5 </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>-</TD><TD>-
</TD></TR><TR><TD>Enzyme PvuII </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>0,5 </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>-</TD><TD>-
</TD></TR><TR><TD>Enzyme Acc65I </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>-</TD><TD>- </TD><TD>0,75 </TD><TD>0,75
</TD></TR><TR><TD>Enzyme XcmI </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>-</TD><TD>- </TD><TD>0,5 </TD><TD>0,5
</TD></TR><TR><TD>Enzyme PstI </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>-
</TD></TR><TR><TD>Enzyme EcoRI </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>- </TD><TD>-
</TD></TR><TR><TD>H2O </TD><TD>6 </TD><TD>6 </TD><TD>6 </TD><TD>6 </TD><TD>5,5 </TD><TD>5 </TD><TD>5 </TD><TD>5,5 </TD><TD>5 </TD><TD>5 </TD><TD>5,5 </TD><TD>5</TD><TD>5 </TD><TD>5,5 </TD><TD>5 </TD><TD>5 </TD><TD>5,75 </TD><TD>5,75
</TD></TR><TR><TD><B>Total volume /µl</B></TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10
</TD></TR></TBODY></TABLE><P>
Incubation time: 1 h, Incubation temperature: 37°

Preparation of gel:
1 g Agarose, 100 ml TAE (1%), 6 µl GELRED , at 115 Volt, running time: 50 minutes
</P><B>Results:</B><P style="color:#00bbff;">pSB1C3_VCK_Bla: Test digestion looks like expected. Clone 4 was sent for sequencing. 
pCerulean_VP1up: Gel looks well as well. clone 2 was sent for sequencing. 
pAAV_RC_ins-rep-cap: Test  digestion looks strange. Just one big band with more than 10000 bp and one very small with 150 bp. Maybe the restriction enzymes didn´t cut. (@Christian: please insert picture)</P><H3><SPAN class="mw-headline" id="102._labday_27.08.2010"><P style="font-size:17px; background-color:#00dd77;">102. labday 27.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Sequence_analysis_of_pSB1C3_Bla_final_.28P309.29and_pCerulean_VP1up_.28P303.29"><P style="font-size:15px; background-color:#66bbff;"><B>Sequence analysis of pSB1C3_Bla_final (P309)and pCerulean_VP1up (P303)</B></P></SPAN></H4><P><B>Investigator: Bea </B></P><P style="font-size:13px; color:#cc0033;"><I><B>GENERAL COMMENT:</B> pSB1C3_Bba_final (P309, SB_9) and pCerulean_Vp1up (P303; SB_10) were sent for sequencing. In the case of SB1C3_Bba_final, which means that the whole vector was assembled from the buttom,the purpose was to test the designed primers. In the other case we wanted to verify the insertion of Vp1up inot pCerulean. </I></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: Sequence analysis of pSB1C3_leftITR_CMV_betaglobin_mVenus_hGH_rightITR.</P><UL><LI>Primer used: </LI><UL><LI>pSB1C3_Bba_seq_for</LI><LI>pSB1C3_Bba_seq_rev</LI></UL><LI>Plasmid sequenced: P?? </LI><LI>Sequence sample: ??</LI><LI>Stored in Geneious-folder: BioBricks --&gt; final parts</LI></UL><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>pSB1C3_leftITR_CMV_betaglobin_mVenus_hGH_rightITR</CAPTION></TABLE><B>Results:</B><P style="color:#00bbff;">Results look good. The forward primer worked, still waiting for the results of the reverse primer. The scar inbetween the CMV promoter and the beat globin intron corresponds to the expected result.</P><P><B>Next steps: Wwait until sequencing results of the reverse primer can be confirmed aswell. If that will not be the case, order new sequensing primer.</B></P><P><A href="http://www.molbiotech.uni-freiburg.de/iGEM/wiki2010/index.php/Laborjournal" class="external text" rel="nofollow">top of page</A></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: Sequence analysis of pCerulean_Vp1up.</P><UL><LI>Primer used: CMV-F</LI><LI>Plasmid sequenced: P303 </LI><LI>Sequence sample: ??</LI><LI>Stored in Geneious-folder: N-terminal Targeting --&gt; pCerulean_Vp1up</LI></UL><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>pCerulean_Vp1up</CAPTION></TABLE><B>Results:</B><P style="color:#00bbff;">Results are good. Insertion of the Vp1up region can be confirmed.</P><P><B>Next steps: Fuse pCerulean_Vp1up to the NLS. </B></P><P><A href="http://www.molbiotech.uni-freiburg.de/iGEM/wiki2010/index.php/Laborjournal" class="external text" rel="nofollow">top of page</A></P><H4><SPAN class="mw-headline" id="Hybridisation_and_cloning_of_loop_insertions"><P style="font-size:15px; background-color:#66bbff;"><B>Hybridisation and cloning of loop insertions</B></P></SPAN></H4><P><B>Investigator: Achim, Volker</B></P><P style="font-size:13px;"><B>We hybridized the oligos for the different loop insertions and cloned them into the pSB1C3_BLA backbone.</B></P><P style="font-size:13px; color:#cc0033;"><B>Hybridisation</B></P><UL><LI>4 Inserts contained overlapping ends and had to be filled up using Klenov fragments. We therefore added Klenow buffer and dNTPs to those samples.
</LI></UL><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>453 BAP</B></TD><TD align="left"><B>587 BAP</B></TD><TD align="left"><B>587 KO BAP</B></TD><TD align="left"><B>453 RGD</B></TD><TD align="left"><B>587 RGD</B></TD><TD align="left"><B>587 KO RGD</B></TD><TD align="left"><B>453 HIS</B></TD><TD align="left"><B>587 HIS</B></TD><TD align="left"><B>587 KO HIS</B></TD><TD align="left"><B>587 KO EMPTY</B></TD><TD align="left"><B>453 Z34C</B></TD><TD align="left"><B>587 Z34C</B></TD><TD align="left"><B>587 KO Z34C</B></TD><TD align="left"><B>587 KO Z34C SPACER</B></TD></TR><TR><TD align="left"> Oligo 1</TD><TD align="left">O124: 10 </TD><TD align="left">O126: 10 </TD><TD align="left">O128: 10 </TD><TD align="left">O130: 10 </TD><TD align="left">O132: 10 </TD><TD align="left">O134: 10 </TD><TD align="left">O135: 10 </TD><TD align="left">O137: 10 </TD><TD align="left">O139: 10 </TD><TD align="left">O141: 10 </TD><TD align="left">O143: 10 </TD><TD align="left">O145: 10 </TD><TD align="left">O147: 10 </TD><TD align="left">O149: 10
</TD></TR><TR><TD align="left"> Oligo 2 </TD><TD align="left">O125: 10 </TD><TD align="left">O127: 10 </TD><TD align="left">O129: 10 </TD><TD align="left">O131: 10 </TD><TD align="left">O133: 10 </TD><TD align="left">O151: 10 </TD><TD align="left">O136: 10 </TD><TD align="left">O138: 10 </TD><TD align="left">O140: 10 </TD><TD align="left">O142: 10 </TD><TD align="left">O144: 10 </TD><TD align="left">O146: 10 </TD><TD align="left">O148: 10 </TD><TD align="left">O150: 10
</TD></TR><TR><TD align="left"> TrisHCl pH8</TD><TD align="left">4</TD><TD align="left">4</TD><TD align="left">4</TD><TD align="left">4</TD><TD align="left">4</TD><TD align="left">4 </TD><TD align="left">4</TD><TD align="left">4</TD><TD align="left">4</TD><TD align="left">4</TD><TD align="left">-</TD><TD align="left">-</TD><TD align="left">-</TD><TD align="left">-
</TD></TR><TR><TD align="left"> 5mM MgCl2</TD><TD align="left">8</TD><TD align="left">8</TD><TD align="left">8</TD><TD align="left">8</TD><TD align="left">8</TD><TD align="left">8 </TD><TD align="left">8</TD><TD align="left">8</TD><TD align="left">8</TD><TD align="left">8</TD><TD align="left">-</TD><TD align="left">-</TD><TD align="left">-</TD><TD align="left">-
</TD></TR><TR><TD align="left"> Klenow Buffer</TD><TD align="left">-</TD><TD align="left">-</TD><TD align="left">-</TD><TD align="left">-</TD><TD align="left">-</TD><TD align="left">- </TD><TD align="left">-</TD><TD align="left">-</TD><TD align="left">-</TD><TD align="left">-</TD><TD align="left">4</TD><TD align="left">4</TD><TD align="left">4</TD><TD align="left">4
</TD></TR><TR><TD align="left"> dNTP Mix</TD><TD align="left">-</TD><TD align="left">-</TD><TD align="left">-</TD><TD align="left">-</TD><TD align="left">-</TD><TD align="left">- </TD><TD align="left">-</TD><TD align="left">-</TD><TD align="left">-</TD><TD align="left">-</TD><TD align="left">1</TD><TD align="left">1</TD><TD align="left">1</TD><TD align="left">1
</TD></TR><TR><TD align="left"> H2O</TD><TD align="left">8</TD><TD align="left">8</TD><TD align="left">8</TD><TD align="left">8</TD><TD align="left">8</TD><TD align="left">8 </TD><TD align="left">8</TD><TD align="left">8</TD><TD align="left">8</TD><TD align="left">8</TD><TD align="left">15</TD><TD align="left">15</TD><TD align="left">15</TD><TD align="left">15
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>40</B></TD><TD align="left"><B>40</B></TD><TD align="left"><B>40</B></TD><TD align="left"><B>40</B></TD><TD align="left"><B>40</B></TD><TD align="left"><B>40</B></TD><TD align="left"><B>40</B></TD><TD align="left"><B>40</B></TD><TD align="left"><B>40</B></TD><TD align="left"><B>40</B></TD><TD align="left"><B>40</B></TD><TD align="left"><B>40</B></TD><TD align="left"><B>40</B></TD><TD align="left"><B>40</B></TD></TR></TBODY></TABLE><UL><LI>Hybridisation was carried out according to standard protocoll
</LI><LI>Klenov fill-in reaction:
</LI><LI>Added 1 µl of NEB Klenow fragment to samples 11,12,13,14
</LI><LI>incubated for 1h  @ 37°C
</LI></UL><P style="font-size:13px; color:#cc0033;"><B>Digestion of pSB1C3_BLA vector and Samples 11-14 </B></P><UL><LI>We digested the standard vector with the 453 (Ssp/Sal) and the 587 (Bam/Pvu) standard. The samples that were filled in were also digested to create sticky ends.
</LI></UL><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>V453</B></TD><TD align="left"><B>V587</B></TD><TD align="left"><B>11</B></TD><TD align="left"><B>12</B></TD><TD align="left"><B>13</B></TD><TD align="left"><B>14</B></TD></TR><TR><TD align="left"> DNA </TD><TD align="left"> 2,5 </TD><TD align="left"> 2,5 </TD><TD align="left"> 3,5</TD><TD align="left"> 3,5</TD><TD align="left"> 3,5</TD><TD align="left"> 3,5
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> - </TD><TD align="left"> - </TD><TD align="left"> - </TD><TD align="left"> - </TD><TD align="left"> - </TD><TD align="left"> -
</TD></TR><TR><TD align="left"> Buffer 4 (10x) </TD><TD align="left"> 2 </TD><TD align="left"> 2 </TD><TD align="left"> 2 </TD><TD align="left"> 2 </TD><TD align="left"> 2 </TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> Enzyme 1</TD><TD align="left"> Ssp: 1 </TD><TD align="left"> Bam: 1 </TD><TD align="left"> Ssp: 1 </TD><TD align="left"> Bam: 1 </TD><TD align="left"> Bam: 1 </TD><TD align="left"> Bam: 1
</TD></TR><TR><TD align="left"> Enzyme 2</TD><TD align="left"> Sal:1 </TD><TD align="left"> Pvu: 1 </TD><TD align="left"> Sal: 1 </TD><TD align="left"> Pvu: 1 </TD><TD align="left"> Pvu: 1 </TD><TD align="left"> Pvu: 1
</TD></TR><TR><TD align="left"> H2O</TD><TD align="left"> 13,5 </TD><TD align="left"> 13,5 </TD><TD align="left"> 12,5 </TD><TD align="left"> 12,5 </TD><TD align="left"> 12,5 </TD><TD align="left"> 12,5
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>20</B></TD><TD align="left"><B>20</B></TD></TR></TBODY></TABLE><P style="font-size:13px; color:#cc0033;"><B>Gel Extraction </B></P><P style="font-size:13px; color:#cc0033;"><B>Ligation</B></P><H4><SPAN class="mw-headline" id="Cloning_of_pCerulean_VP1up_NLS"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of pCerulean_VP1up_NLS</B></P></SPAN></H4><P><B>Investigator: Anissa </B></P><P style="font-size:13px; color:red;"><B>Comment</B>:Cloning did not work, because no NLS-band could be seen in the gel... Will be repeated on monday in two new approaches. One time VP1up will be cloned into pSB1C3_NLS and recloned as a fusion-product into pCerulean. Another time the oligos of NLS will be hybridisized, cut with rescriction enzymes and purificated with the Qiaex II kit.Then the NLS will be ligated into the pCerulean_VP1up  </P><UL><LI>Digestion: 
</LI></UL><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">Vector </TD><TD align="right">Insert
</TD></TR><TR><TD> DNA  </TD><TD align="right">2,6 </TD><TD align="right"> 32
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">1,5</TD><TD align="right"> -
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">1,5 </TD><TD align="right"> 4
</TD></TR><TR><TD>Enzyme  </TD><TD align="right">1 PstI </TD><TD align="right"> 2 PstI
</TD></TR><TR><TD>Enzyme  </TD><TD align="right">1 AgeI</TD><TD align="right"> 2 NgoMIV
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">7,4 </TD><TD align="right"> -
</TD></TR><TR><TD><B>Total volume </B></TD><TD align="right">15</TD><TD align="right"> 40
</TD></TR></TBODY></TABLE><UL><LI> Gel: that's only the picture of the 2% agarose gel for seperating the NLS. But no band could be seen.
</LI></UL><H4><SPAN class="mw-headline" id="DARPin_E_01"><P style="font-size:15px; background-color:#66bbff;"><B>DARPin E_01</B></P></SPAN></H4><P><B>Investigator: Bea </B></P><P style="font-size:13px; color:#66bbaa;"><B>General overview of DARPins:</B></P><P>Natural protein ankyrin repeat molecules are motifs which can be found in proteins. These motifs are mediating protein-protein interactions. This suggests that ankyrin repeat (AR) proteins can be used for designing binding molecules. In Kohl <I>et al.</I> (2003) and Binz et al. (2003) the authors designed a structural framework with fixed consensus regions and randomized positions of interacting residues. 
</P><P>The repetitive nature of the ankyrin proteins allows modifications in their variable and modular binding surface. Therefore consensus sequences of natural ankyrin proteins were used to design novel and stable scaffolds for binding proteins. 
Designed Ankyrin Proteins (DARPins) are well expressed, monomeric in solution, thermodynamically stable and have the ability to fold fast. In the publication of Steiner <I>et al.</I> (2008) screening libraries were created by useing the signal recognition particle (SRP) translocation pathway for phage display. The proteins containing the appropriate translocation signal sequence are efficiently displayed on filamentous phage particles. Screening for DARPins binding to specific target proteins like EGF-R (Erb1) were performed by phage ELISAs. The extracellular domain I-III of the receptor ErbB1 was fused to the Fc part of human IgG1 and used for selection. The previously described combinatorial N3C library was used as a template for SRP selection. After phage ELISA ans sequencing have been performed only one clone (E_01) could be selected with high-affinity binding characteristics. For broaden the diversity of ErbB1 DARPins three more clones were selected by epitope masking (E_67, E_68 and E_69). Binding and epitope localization experiments showed that all selected clones recognize a competing epitope as the dominant binder E_01, except of E_69 which recognizes an epitope not competing with E_01. The low diversity obtained for DARPins against ErbB1 suggests that the binding interfaces are well suited for dominant selection. Additionally the selected clones showed no cross-reactivity with other ErbB-family receptors (Figure 2C).
</P><P>
The dominant DARPpin E_01 has very high affinities to the target protein ErbB1 (Table 3) and can be used as a potential targeting molecule four our approach in fusing the DARPin to the N-terminal VP proteins.
</P><P style="font-size:13px; color:#66bbaa;"><B>Overview of DARPinE_01 used in our approach: </B></P><P>The designed ankyrin repeat protein used as a potential targeting molecule consists of three internal capping repeats and the C-and N-terminal capping repeats. Each internal repeat module comprises of one beta-turn and two hydrophobic alpha-helices. The potential interaction residues are located in the beta-turn the first alpha-helices of the AR-proteins. The complete nucleotide and corresponding amino acid sequence can be found in the appendix.
</P><P><A href="http://www.molbiotech.uni-freiburg.de/iGEM/wiki2010/index.php/Laborjournal" class="external text" rel="nofollow">top of page</A></P><H4><SPAN class="mw-headline" id="Update_VP1_insertion"><P style="font-size:15px; background-color:#66bbff;"><B>Update VP1 insertion</B></P></SPAN></H4><P><B>Investigator: Anissa, Bea </B></P><P style="font-size:13px; color:#cc0033;"><B>Done:</B></P><UL><LI><B>pSB1C3_VP1up</B> was succesfully created by Achim. For details see labday from 2010-22-08.</LI><LI><B>pCerulean_VP1up</B> was suceesfully created by Anissa. pCerulean was sequenced and the VP1up which contains the upstream 137 aa´s of the VP1 protein could be successfully cloned into the plasmid backbone pCerulean (obtained by performeing a PCR).</LI><LI><B>pSB1C3_NLS</B>Hybridization of the nuclear localization sequence (NLS) was successfully incorporated into the pSB1C3_CFP.  </LI><LI>Today, Anissa tried to fuse the NLS to the VP1up sequence. Cutting the pSB1C3_NLS with NgomIV and PstI leads to a 45 nt fragment which normally can be resoluted in a 2% agarose gel. But no fragment could be detected in the gel (for further details see: Cloning of <B>pCerulean_VP1up_NLS</B>). 

On monday another approach will be performed: The NLS oligos will be hybridized and digested with NgomIV and SpeI. With the Qiaex II Kit (protocol for desalting and concentrating DNA solutions),the hybrifized oligos will be purified. Parallel, another digestion approach will be conducted. </LI><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>Overview</CAPTION></TABLE></UL><P style="color:#00bbff;"> Next steps: </P><UL><LI> We obtain pSB1C3_VP23_HSPG_ko by conducting a PCR with the pAAV_RC_final (which contains the integrated &quot;cap&quot;. </LI><LI> Fuse the obtained construct pSB1C3_Targeting Molcule to the pSB1C3_VP23_HSPG_ko. </LI><UL><LI>Affibody Z<SUB>EGFR:1907</SUB></LI><LI>DARPin E_01</LI><LI>6xHis Tag</LI><LI>CFP</LI></UL><LI> Fuse pSB1C3_VP23_HSPG_ko_Targeting molecule to the construct pCerulean_VP1up_NLS.</LI><LI> Finally, for providing the possibilty to express obtained construct in trans: perform a site-directed mutagenesis with the pAAV_RC_final to remove the startcodon of VP1.</LI></UL><P><A href="http://www.molbiotech.uni-freiburg.de/iGEM/wiki2010/index.php/Laborjournal" class="external text" rel="nofollow">top of page</A></P><H4><SPAN class="mw-headline" id="Sequence_analysis_of_pSB1C3_BLA_final_.28P309.29"><P style="font-size:15px; background-color:#66bbff;"><B>Sequence analysis of pSB1C3_BLA_final (P309)</B></P></SPAN></H4><P><B>Investigator: Bea </B></P><P style="font-size:13px; color:#cc0033;"><I><B>GENERAL COMMENT:</B> pSB1C3_BLA_final (P309) was sent for sequencing. the beta lactamse was inserted and two site-directed mutagenesis have been performed in order to delete the loop insertion restriction enzymes in the CAT marker.</I></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: Sequence analysis of pSB1C3_BLA_final containing the two deleted restriction sites  (SspI and PvuII)in the CAT marker.</P><UL><LI>Primer used: </LI><UL><LI>pQE-RP</LI><LI>VF-2</LI></UL><LI>Plasmid sequenced: P309 </LI><LI>Sequence sample: SB_9</LI><LI>Stored in Geneious-folder: pSB1C3_BLA</LI></UL><TABLE class="gallery" cellspacing="0" cellpadding="0"><CAPTION>pSB1C3_BLA_final</CAPTION></TABLE><B>Results:</B><P style="color:#00bbff;">The two performed site-directed mutagenesis performed at the backbone of the pSB1C3 containing the beta-lactamase can be confrimed as it can be seen in the two pictures above. Additionally the BLA sequence was sequenced aswell as the insertion of the BLA which can be confirmed with no mutations.</P><P><B>Next steps: This new plasmid can be used for further experiments. Clone the PCR prdouct of the pAAV_RC into the pSB1C3 and subclone the loop insertion motifs into the vector.</B></P><P><A href="http://www.molbiotech.uni-freiburg.de/iGEM/wiki2010/index.php/Laborjournal" class="external text" rel="nofollow">top of page</A></P><H4><SPAN class="mw-headline" id="Mini-Preps_and_test_digestion_of_pSB1C3_6xHis_middlelinker_and_pSB1C3_6xHis_mVenus"><P style="font-size:15px; background-color:#66bbff;"><B>Mini-Preps and test digestion of pSB1C3_6xHis_middlelinker and pSB1C3_6xHis_mVenus</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><UL><B>Glycerol stocks were prepared:</B><LI>B258 = pSB1C3_6xHis_middlelinker clone1</LI><LI>B259 = pSB1C3_6xHis_middlelinker clone2</LI><LI>B260 = pSB1C3_6xHis_mVenus clone 1</LI><LI>B261 = pSB1C3_6xHis_mVenus clone 2</LI><P><B>Mini-Prep following the standard protocol</B></P><LI>P310 = pSB1C3_6xHis_middlelinker clone1
c= 185,31 ng/µl</LI><LI>P311 = pSB1C3_6xHis_middlelinker clone2
c= 178,87 ng/µl</LI><LI>P312 = pSB1C3_6xHis_mVenus clone 1
c= 247,62 ng/µl</LI><LI>P313 = pSB1C3_6xHis_mVenus clone 2
c= 268,98 ng/µl</LI></UL><P><B>Test digestion</B></P><UL><LI>Restriction-enzymes used: EcoRI and PstI</LI></UL><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>P310+311</B></TD><TD align="left"><B>P312+313</B></TD></TR><TR><TD align="left"> DNA </TD><TD align="left"> 2 </TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 1 </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Buffer 4 (10x) </TD><TD align="left"> 1 </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> EcoRI </TD><TD align="left"> 0,5 </TD><TD align="left"> 0,5
</TD></TR><TR><TD align="left"> PstI </TD><TD align="left"> 0,5 </TD><TD align="left"> 0,5
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 5 </TD><TD align="left"> 5
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>10</B></TD><TD align="left"><B>10</B></TD></TR></TBODY></TABLE><P>
Incubation time: 90 minutes; Incubation temperature: 37°

1 g Agarose, 100 ml TAE (1%), 6 µl GELRED , running time: 55 minutes at 120 Volt </P><P style="font-size:13px; color:#003399;"><B>Comment</B>: Test digestion looks good. Clones 1 of each plasmid (P310 and p312) were sent in for sequencing. </P><H4><SPAN class="mw-headline" id="Repetition_of_test_digestion_of_pAAV_RC_ins-rep-cap.2C_pAAV_RC_ins-rep.2C_pSB1C3_Rep40_RC_Insert_and_pSB1C3_Rep68_RC_Insert."><P style="font-size:15px; background-color:#66bbff;"><B>Repetition of test digestion of pAAV_RC_ins-rep-cap, pAAV_RC_ins-rep, pSB1C3_Rep40_RC_Insert and pSB1C3_Rep68_RC_Insert.</B></P></SPAN></H4><P><B>Investigator: Chris L.</B></P><UL><LI>P250 = pAAV_RC_ins-rep clone1c=467,50 ng/µl</LI><LI>P314 = pAAV_RC_ins-rep-cap clone1c=517,04 ng/µl</LI><LI>P315 = pAAV_RC_ins-rep-cap clone2c=444,33 ng/µl</LI><LI>P316 = pSB1C3_Rep40_ins clone 1c=302,22 ng/µl</LI><LI>P317 = pSB1C3_Rep40_ins clone 2c=274,37 ng/µl</LI><LI>P318 = pSB1C3_Rep68_ins clone 1c=491,69 ng/µl</LI><LI>P319 = pSB1C3_Rep68_ins clone 2c=498,71 ng/µl</LI></UL><P><B>Test digestion:</B></P><TABLE border="1"><TBODY><TR><TD> components </TD><TD>volume of <B>P250</B>/µl </TD><TD>volume of <B>P314</B>/µl </TD><TD>volume of <B>P315</B>/µl </TD><TD>volume of <B>P316</B>/µl </TD><TD>volume of <B>P317</B>/µl </TD><TD>volume of <B>P318</B>/µl </TD><TD>volume of <B>P319</B>/µl
</TD></TR><TR><TD> DNA  </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1
</TD></TR><TR><TD> BSA (10x) </TD><TD>1 </TD><TD>1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD>1
</TD></TR><TR><TD> Buffer 2 (10x) </TD><TD>1 </TD><TD>1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1
</TD></TR><TR><TD>Enzyme Acc65I </TD><TD>0,75 </TD><TD>0,75 </TD><TD>0,75 </TD><TD> - </TD><TD> - </TD><TD> - </TD><TD> -
</TD></TR><TR><TD>Enzyme XcmI </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD> - </TD><TD> - </TD><TD> - </TD><TD> -
</TD></TR><TR><TD>Enzyme HindIII </TD><TD> - </TD><TD> - </TD><TD> - </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5
</TD></TR><TR><TD>Enzyme BstEII </TD><TD> - </TD><TD> - </TD><TD> - </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5
</TD></TR><TR><TD>H2O </TD><TD>5,75 </TD><TD>5,75 </TD><TD>5,75 </TD><TD> 6 </TD><TD> 6 </TD><TD> 6 </TD><TD> 6
</TD></TR><TR><TD><B>Total volume /µl</B></TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10
</TD></TR></TBODY></TABLE><P>
Incubation time: 1 h, Incubation temperature: 37° for P250, P314 and P315

Incubation time: 45 min, Incubation temperature: 37° with HindIII for P316, P317, P318 and P319 ; 45 min, Incubation temperature: 60° with BstEII for P316, P317, P318 and P319
&gt;br /&gt;
Preparation of gel:
1 g Agarose, 100 ml TAE (1%), 6 µl GELRED , at 120 Volt, running time: 45 minutes
</P><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker
</TH><TH>Sample P250 10 /µl
</TH><TH>Sample P314 10 /µl
</TH><TH>Sample P315 10 /µl
</TH><TH>Sample P316 10 /µl
</TH><TH>Sample P317 10 /µl
</TH><TH>Sample P318 10 /µl
</TH><TH>Sample P319 10 /µl
</TH></TR><TR><TH>Lane
</TH><TD>1
</TD><TD>3
</TD><TD>4
</TD><TD>5
</TD><TD>7
</TD><TD>8
</TD><TD>9
</TD><TD>10
</TD></TR></TBODY></TABLE><P style="font-size:13px; color:#003399;"><B>Comment</B>: The P314 and P315 looks like the control vector. The RepCap Vector_SDM_InsPvuII insertion didn´t work. The P316, P317, P318 and P318 was cut with the &quot;wrong&quot; enzymes. The same fragment size as the original vector appeared. Next time cut with EcoRI. </P><H4><SPAN class="mw-headline" id="Midi-Prep_of_pSB1C3_VCK_Bla"><P style="font-size:15px; background-color:#66bbff;"><B>Midi-Prep of pSB1C3_VCK_Bla</B></P></SPAN></H4><P><B>Investigators: Chris W. </B></P><P style="font-size:13px; color:#003399;"><B>Comment</B>: Midi-Preps of P 320, B257</P><P>The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
</P><TABLE border="1"><TBODY><TR><TD> plasmid-no. </TD><TD align="right">P320
</TD></TR><TR><TD> concentration (ng/µl)</TD><TD align="right">408
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="103._labday_28.08.2010"><P style="font-size:17px; background-color:#00dd77;">103. labday 28.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Mini-Prep_of_pSB1C3_NLS"><P style="font-size:15px; background-color:#66bbff;"><B>Mini-Prep of pSB1C3_NLS</B></P></SPAN></H4><P><B>Investigator: Stefan</B>
comments: No glycerol stock and test digestion was performed because bacteria culture inocculated from glyerol stock (B233).
</P><UL><B>Mini-Prep following the standard protocol</B><LI>P321 = pSB1C3_NLS (1)
c= 180,45 ng/µl</LI><LI>P322 = pSB1C3_NLS (2)
c= 187,26 ng/µl</LI></UL><P style="font-size:13px; color:#cc0033;"><I><B>COMMENT (Patrick):</B> These mini-preps will be thrown away because B233 was used for inoculation due to a wrong entry in our plasmid excel sheet.  B233 this is NOT the sequenced clone ! We sequenced B234/P282. Therefore 2x10 ml DYT + Cm were inoculated with B234. </I></P><H4><SPAN class="mw-headline" id="Cell_culture"><P style="font-size:15px; background-color:#66bbff;"><B>Cell culture</B></P></SPAN></H4><H5><SPAN class="mw-headline" id="Harvest_viral_Stock_and_Transductiot_of_HT_and_A431_cells">Harvest viral Stock and Transductiot of HT and A431 cells</SPAN></H5><OL><LI>P169: 100.000 cells</LI><LI>P169: 200.000 cells</LI><LI>P169: 300.000 cells</LI><LI>P169: 400.000 cells</LI><LI>P169: 500.000 cells</LI><LI>P169: 600.000 cells</LI><LI>P169: 700.000 cells</LI><LI>P262: 100.000 cells</LI><LI>P262: 200.000 cells</LI><LI>P262: 300.000 cells</LI><LI>P262: 400.000 cells</LI><LI>P262: 500.000 cells</LI><LI>P262: 600.000 cells</LI><LI>P262: 700.000 cells</LI></OL><P>
200.000 cells per well</P><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>1</TD><TD>2</TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>1</TD><TD>2</TD></TR></TBODY></TABLE><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>3</TD><TD>4</TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>3</TD><TD>4</TD></TR></TBODY></TABLE><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>5</TD><TD>6</TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>5</TD><TD>6</TD></TR></TBODY></TABLE><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>7</TD><TD>8</TD></TR><TR><TD>B</TD><TD>control, no virus</TD><TD>7</TD><TD>9</TD></TR></TBODY></TABLE><TABLE border="3" rules="all" cellpadding="5" cellspacing="1" style="font-size:13pt;text-align:center"><TBODY><TR><TH width="80">1</TH><TH width="80">2</TH><TH width="80">3</TH></TR><TR><TD>A</TD><TD>control, no cells</TD><TD>11</TD><TD>13</TD></TR><TR><TD>B</TD><TD>10</TD><TD>12</TD><TD>14</TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="Seeding_AAV293_for_Transfection_at_30.8">Seeding AAV293 for Transfection at 30.8</SPAN></H5><P>
8 plates with 200.000 cells each were seeded
</P><H4><SPAN class="mw-headline" id="Picking_clones_from_the_ViralBricks"><P style="font-size:15px; background-color:#66bbff;"><B>Picking clones from the ViralBricks</B></P></SPAN></H4><P><B>Investigator: Volker</B></P><P>Clones from all 14 transformations of the ViralBricks were picked. Because the probability that the ligation worked and that the synthesized oligos do not conatin mutations is quite bad I picked 8 to 10 clones from each ligation, resulting in approximately 150 clones at all. The clones were used to inoculate 5ml of a mixture of 60%LB-media nd 40%DYP-media because our stocks were not sufficient for this experiment.
</P><H3><SPAN class="mw-headline" id="104._labday_29.08.2010"><P style="font-size:17px; background-color:#00dd77;">104. labday 29.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="P282.2FB234_mini-prep"><P style="font-size:15px; background-color:#66bbff;"><B>P282/B234 mini-prep</B></P></SPAN></H4><P><B>Investigator: Patrick</B>
... was performed according to the standard protocol. 
Labelling:
</P><UL><LI> P323, pSB1C3_NLS clone 2.1, 129 ng/µl
</LI><LI> P324, pSB1C3_NLS clone 2.2, 141 ng/µl
</LI></UL><H4><SPAN class="mw-headline" id="Digestion_of_pSB1C3_NLS_.28P324.29"><P style="font-size:15px; background-color:#66bbff;"><B>Digestion of pSB1C3_NLS (P324)</B></P></SPAN></H4><P><B>Investigator: Patrick</B></P><P>The NLS was cut out with NgoMIV and EcoRI, therefore the expected size of the hoped-for fragment is 55 bp. About 30 ng DNA can be detected in a 1% gel. According to my calculation there will be maximal 80 ng insert. 
55bp/2019bp = 0,027 
0,027x140 ng/µl = 3,81 ng/µl 
80 ng/3,81 ng/µl = 21 µl 
Digestion: 21 µl pSB1C3_NLS (P324), 2,5 µl EcoRI, 2,5 µl NgoMIV, 3 µl Buffer 4, 1 µl H2O. 
Digestion Time: 2 h 20 minutes.
</P><P>The 2% agarose gel was loaded with 36 µl digesteion product including 6 µl 30% glycerol solution. I did not use loading dye because i to prevent that the dye overlay the mutual 55 bp insert.
</P><P>The gelextraction was performed according to the the standard protocl. The resolution of the picture ist not sufficient to be able to see the band clearly as the gel was put onto the UV-light table a very thin band with the expected size could be seen.   
DNA concentration measurement:
3,28 &amp; 7,7 ng/µl (two measurements). Especially the second value is doubtable because according to my calculation above the insert conccentration cant be higher than 3,81 ng/µl. Labeled: NLS, put into 4°C room.
</P><P>New digestion over night following a gelrun tomorrow: 23 µl pSB1C3_NLS (P324), 2 µl EcoRI, 2 µl NgoMIV, 3 µl Buffer 4. </P><H4><SPAN class="mw-headline" id="Harvesting_cultures_from_the_ViralBricks"><P style="font-size:15px; background-color:#66bbff;"><B>Harvesting cultures from the ViralBricks</B></P></SPAN></H4><P><B>Investigator: Volker</B></P><P>All approximately 150 cultures were pelletted (twice 2 ml) and the supernatant thrown away. The pellets were freezed and the rest of the culture stored in the 4°C room. These pellets will be preped, test digested and sequenced subsequently until right clones are identified.
</P><H3><SPAN class="mw-headline" id="105._labday_30.08.2010"><P style="font-size:17px; background-color:#00dd77;">105. labday 30.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="BioBrick_assembly_of_pSB1C3_leftITR_pTERT_.26_pSB1C3_.C3.9F-globin_YFP_hGH_rITR"><P style="font-size:15px; background-color:#66bbff;"><B>BioBrick assembly of pSB1C3_leftITR_pTERT &amp; pSB1C3_ß-globin_YFP_hGH_rITR</B></P></SPAN></H4><P><B>Investigator: Achim </B></P><P>comment: final assembly of a vector containing all aav elements &amp; the tert promoter <B>Digestion:</B></P><TABLE border="1"><TBODY><TR><TD> components:   </TD><TD align="right"> pSB1C3_ß-globin_YFP_hGH_rITR(Vector): P294 </TD><TD align="right">pSB1C3_leftITR_pTERT (Insert): P256
</TD></TR><TR><TD> DNA  </TD><TD align="right">2,4 </TD><TD align="right">8,4
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">2 </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">2 </TD><TD align="right">2
</TD></TR><TR><TD>Enzyme1 </TD><TD align="right">1 (EcoRI) </TD><TD align="right">1 (EcoRI)
</TD></TR><TR><TD>Enzyme2 </TD><TD align="right">1 (XbaI)</TD><TD align="right">1 (SpeI)
</TD></TR><TR><TD>H2O</TD><TD align="right">11,6 </TD><TD align="right">5,6
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 20</TD><TD align="right">20
</TD></TR></TBODY></TABLE><P>
Preparation of gel:
0,5 g Agarose,50 ml TAE (1%), 3 µl GELRED , at 110 Volt, running time: 45 minutes
</P><P>Expected sizes of constructs:
</P><UL><LI>pSB1C3_ß-globin_YFP_hGH_rITR:  ~4000 bp
</LI><LI>pSB1C3_leftITR_pTERT: 2896 bp, ~640 bp
</LI></UL><P>The corresponding bands were cut out and Gel-Extraction was performed according to protocol.
</P><P>concentrations measured via NanoDrop:
</P><UL><LI>pSB1C3_ß-globin_YFP_hGH_rITR:  34.73 ng/µl
</LI><LI>pSB1C3_leftITR_pTERT: 9.48 ng/µl
</LI></UL><P><B>Ligation:</B></P><P>T4 ligation was performed according to protocol. </P><P>Volumes used:
</P><UL><LI>vector: 2,9 µl
</LI><LI>insert: 5,1 µl
</LI></UL><P><B>Transformation:</B></P><P>bacterial strain used: XL1bB</P><P>antibiotic used for plate: chlorampenicol</P><P>Transformation was performed according to protocol.
Plate was prepared and put in 37°C room.
</P><H4><SPAN class="mw-headline" id="Continuation:_Digestion_of_pSB1C3_NLS_.28P324.29"><P style="font-size:15px; background-color:#66bbff;"><B>Continuation: Digestion of pSB1C3_NLS (P324)</B></P></SPAN></H4><P><B>Investigator: Patrick</B></P><P>The gelextraction was performed with QIAEX II Gel Extraction Kit which should also allow to extract fragments smaller than 70 bp (at least 40 bp). According to the QIAGEN Gel Extraction Kit the lenght of the DNA fragment should be about 70 bp.
Unfortunately the DNA fragment was blurred so a quite large piece had to be cut out.
</P><P>The extraction yielded 4,26 and 3,77 ng/µl (two measurements) but this sample also quite polluted. ... put into freezer
</P><H4><SPAN class="mw-headline" id="CD_biobrick_production"><P style="font-size:15px; background-color:#66bbff;"><B>CD biobrick production</B></P></SPAN></H4><P><B>Investigator: Kira</B>
NA samples were diluted 1:100
</P><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD align="right">CD sample
</TD></TR><TR><TD> 5X Phusion HF buffer </TD><TD align="right">10 µl
</TD></TR><TR><TD> 10 mM dNTP mix</TD><TD align="right">1µl
</TD></TR><TR><TD> forward primer: O158 </TD><TD align="right">2,5µl
</TD></TR><TR><TD> reverse primer: O159 </TD><TD align="right">2,5 µl
</TD></TR><TR><TD> DNA Template</TD><TD align="right">0,5 µl
</TD></TR><TR><TD> DMSO </TD><TD align="right">0 µl
</TD></TR><TR><TD> Phusion Polymerase</TD><TD align="right">0,5 µl
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">33µl
</TD></TR><TR><TD>Total volume</TD><TD align="right">50 µl
</TD></TR></TBODY></TABLE><P>PCR program:
</P><TABLE border="1"><TBODY><TR><TD>Cycles</TD><TD>Temperature</TD><TD>Time
</TD></TR><TR><TD>98°C</TD><TD>1
</TD></TR><TR><TD>8x</TD><TD>98°C</TD><TD>15&quot;
</TD></TR><TR><TD>60°C</TD><TD>25&quot;
</TD></TR><TR><TD>72°C</TD><TD>60&quot;
</TD></TR><TR><TD>17x</TD><TD>98°C</TD><TD>15&quot;
</TD></TR><TR><TD>65°C</TD><TD>25&quot;
</TD></TR><TR><TD>72°C</TD><TD>60&quot;
</TD></TR><TR><TD>1x</TD><TD>72°C</TD><TD>5'
</TD></TR><TR><TD>Hold 4°C
</TD></TR></TBODY></TABLE><P>Digestion of plasmid backbone:
</P><P>c (pSB1C3) = 151, 1 ng/ µl
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>vector</B> Volume/µL
</TD></TR><TR><TD align="left"> DNA 1 µg  </TD><TD align="left"> 6,0 µl
</TD></TR><TR><TD align="left"> BSA (100x) </TD><TD align="left"> 0,2 µl
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 2,0 µl
</TD></TR><TR><TD align="left"> Enzyme 1 XbaI </TD><TD align="left"> 0,5 µl
</TD></TR><TR><TD align="left"> Enzyme 2 AgeI HF </TD><TD align="left"> 0,5 µl
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 10,8 µl
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>20</B></TD></TR></TBODY></TABLE><P>incubation @ 37 C for approx. 2 h
</P><P>
Digestion of PCR product:
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>PCR product</B> Volume/µL
</TD></TR><TR><TD align="left"> DNA   </TD><TD align="left"> 30,0 µl
</TD></TR><TR><TD align="left"> BSA (100x) </TD><TD align="left"> 0,4 µl
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 4,0 µl
</TD></TR><TR><TD align="left"> Enzyme 1 XbaI </TD><TD align="left"> 1,5 µl
</TD></TR><TR><TD align="left"> Enzyme 2 AgeI HF </TD><TD align="left"> 1,0 µl
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 3,1 µl
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>40</B></TD></TR></TBODY></TABLE><P>
incubation @ 37 C for approx. 2 h
</P><P>1% agarose gel </P><P>Ligation 
T4 ligase was used 
1 ul T4 Buffer 
1 ul T4 Ligase 
9 ul (6,9 ul vector+ 1,1 ul insert) DNA-mix </P><P>incubation over-night @ 18C </P><H4><SPAN class="mw-headline" id="Repetition_of_test_digestion_of_pSB1C3_Rep40_RC_Insert_and_pSB1C3_Rep68_RC_Insert."><P style="font-size:15px; background-color:#66bbff;"><B>Repetition of test digestion of pSB1C3_Rep40_RC_Insert and pSB1C3_Rep68_RC_Insert.</B></P></SPAN></H4><P><B>Investigator: Chris L.</B></P><UL><LI>P316 = pSB1C3_Rep40_ins clone 1c=302,22 ng/µl</LI><LI>P317 = pSB1C3_Rep40_ins clone 2c=274,37 ng/µl</LI><LI>P318 = pSB1C3_Rep68_ins clone 1c=491,69 ng/µl</LI><LI>P319 = pSB1C3_Rep68_ins clone 2c=498,71 ng/µl</LI></UL><P><B>Test digestion:</B></P><TABLE border="1"><TBODY><TR><TD> components </TD><TD>volume of <B>P316</B>/µl </TD><TD>volume of <B>P317</B>/µl </TD><TD>volume of <B>P318</B>/µl </TD><TD>volume of <B>P319</B>/µl
</TD></TR><TR><TD> DNA  </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1
</TD></TR><TR><TD> BSA (10x) </TD><TD>1 </TD><TD>1 </TD><TD> 1 </TD><TD> 1
</TD></TR><TR><TD> Buffer 4 (10x) </TD><TD>1 </TD><TD>1 </TD><TD> 1 </TD><TD> 1
</TD></TR><TR><TD>Enzyme XcmI </TD><TD>0,5 </TD><TD>0,5 </TD><TD>0,5 </TD><TD> 0,5
</TD></TR><TR><TD>H2O </TD><TD> 6,5 </TD><TD> 6,5 </TD><TD> 6,5 </TD><TD> 6,5
</TD></TR><TR><TD><B>Total volume /µl</B></TD><TD>10 </TD><TD>10 </TD><TD>10 </TD><TD>10
</TD></TR></TBODY></TABLE><P>
Incubation time: 1 h, Incubation temperature: 37°
&gt;br /&gt;
Preparation of gel:
0,5 g Agarose, 50 ml TAE (1%), 3 µl GELRED , at 120 Volt, running time: 45 minutes
</P><P style="font-size:13px; color:#003399;"><B>Comment</B>: Test digestion looks good. Clones 2 of each plasmid (P317 and P319) were sent in for sequencing with VR2 Primer. </P><H4><SPAN class="mw-headline" id="Midi-Prep_of_pAAV_RC_insertparts_clone1"><P style="font-size:15px; background-color:#66bbff;"><B>Midi-Prep of pAAV_RC_insertparts clone1</B></P></SPAN></H4><P><B>Investigators: Chris W. </B></P><P style="font-size:13px; color:#003399;"><B>Comment</B>: Midi-Preps of B208</P><P>The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
</P><TABLE border="1"><TBODY><TR><TD> plasmid-no. </TD><TD align="right">P325</TD><TD align="right">p326
</TD></TR><TR><TD> concentration (ng/µl)</TD><TD align="right">2481,04 </TD><TD align="right">2050,66
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="FACS_analysis.2C_Seeding_HT1080_and_AAV293"><P style="font-size:15px; background-color:#66bbff;"><B>FACS analysis, Seeding HT1080 and AAV293</B></P></SPAN></H4><P><B>Investigators: Kerstin, Adrian </B></P><UL><LI>FACS analysis of Transduction from 28.08.2010
</LI><LI>Seeding HT1080 cells for Transduction (31.08.2010): 5x 6-well plates, 270.000 cells per well
</LI><LI>Seeding AAV293 cells for Transfection (01.09.2010): 10 plates, 300.000 cells per well
</LI></UL><H4><SPAN class="mw-headline" id="Mini-Preps_and_test_digestion_of_Loop_insertion_BioBricks"><P style="font-size:15px; background-color:#66bbff;"><B>Mini-Preps and test digestion of Loop insertion BioBricks</B></P></SPAN></H4><P><B>Investigator: Anna</B></P><P style="font-size:13px; color:#003399;"><B>Comment</B>: Two clones of each ViralBrick were prepared and test digested. </P><P><B>Mini-Preps:</B></P><TABLE border="1"><TBODY><TR><TD align="right"><B>p331</B></TD><TD align="right"><B>p335</B></TD><TD align="right"><B>p339</B></TD></TR><TR><TD>Construct</TD><TD align="right">pSB1C3_587_KO_BAP_clone3.1  </TD><TD align="right"> pSB1C3_587_RGD_clone5.1 </TD><TD align="right"> pSB1C3_453_His_clone7.1
</TD></TR><TR><TD>DNA-Concentration / µl</TD><TD align="right">65,08 </TD><TD align="right"> 94,29</TD><TD align="right"> 73,49
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD align="right"><B>p342</B></TD><TD align="right"><B>p345</B></TD></TR><TR><TD>Construct</TD><TD align="right"> pSB1C3_587_His_clone8.3   </TD><TD align="right">pSB1C3_587_KO_empty_clone10.1
</TD></TR><TR><TD>DNA-Concentration / µl</TD><TD align="right">69,05</TD><TD align="right"> 66,73
</TD></TR></TBODY></TABLE><P><B>Test digestion:</B></P><P>1% Agarose gel: 1 g Agarose, 100 ml TAE, 6 µl Gelred 
Marker: Gene ruler ladder DNA mix, 5 µl</P><P>Used enzymes: SalI, BamHI (in general, SDS has to be added because of the strong binding of enzymes to DNA!)
</P><P style="font-size:13px; color:#003399;"><B>Comment</B>: Test digestion had no significant results. In addition, one sample was forgotten, it was not clear which one (description makes no sense). Test digestion has to be repeated.</P><H4><SPAN class="mw-headline" id="Repetition_of_subcloning_Cap-insert_into_pAAV_RC"><P style="font-size:15px; background-color:#66bbff;"><B>Repetition of subcloning Cap-insert into pAAV_RC</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P style="font-size:13px; color:#003399;"><B>Comments</B>: There are still problems with BspMI. It was assumed that longer digestion could improve the results, therefore digestion with BspMI will be performed overnight.<B>P.S: Welcome back Hanna!</B></P><P><B>Digestion of vector and insert:</B></P><UL><LI>Insert: pMA_RepCap Vector_SDM_InsPvuII clone 1 (P211)
</LI><LI>Vector: pAAV_RC_ins-rep clone1 (P250)</LI></UL><P>
A two-step digestion was performed: <B>1st step</B></P><TABLE border="1"><TBODY><TR><TD align="right"><B>P211 / µl</B></TD><TD><B>P250 / µl</B></TD></TR><TR><TD>DNA</TD><TD align="right">8,5  </TD><TD align="right"> 2
</TD></TR><TR><TD>buffer 3</TD><TD align="right">4,6 </TD><TD align="right"> 5
</TD></TR><TR><TD> Enzyme BsiWI</TD><TD align="right">1  </TD><TD align="right"> 1
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">32,4 </TD><TD align="right"> 38
</TD></TR><TR><TD>total volume</TD><TD align="right">46 </TD><TD align="right">46
</TD></TR></TBODY></TABLE><P><B>2nd step</B></P><TABLE border="1"><TBODY><TR><TD align="right"><B>P211 / µl</B></TD><TD><B>P250 / µl</B></TD></TR><TR><TD>Mix obtained from step 1 </TD><TD align="right">46  </TD><TD align="right"> 46
</TD></TR><TR><TD> Enzyme BspMI</TD><TD align="right">4  </TD><TD align="right"> 4
</TD></TR><TR><TD>total volume</TD><TD align="right">50 </TD><TD align="right">50
</TD></TR></TBODY></TABLE><P>
Digestion will be performed overnight and continued tomorrow.
</P><H3><SPAN class="mw-headline" id="106._labday_31.08.2010"><P style="font-size:17px; background-color:#00dd77;">106. labday 31.08.2010</P></SPAN></H3><H4><SPAN class="mw-headline" id="Cloning_ZEGFR:1907_and_6xHis_middlelinker_into_pCerulean"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning ZEGFR:1907 and 6xHis_middlelinker into pCerulean</B></P></SPAN></H4><P><B>Investigator: Hanna</B></P><P><FONT color="#ff00ff"><B>Comment:</B> In order to fuse the His-Tag to VP2, the His-Tag-Middlelinker construct will be cloned into the pCerulean plasmid (contains CMV promoter and SV40 terminator) today. In the next step, VP2 will be cloned behind the His-Tag-Middlelinker.

In addition to that the Affibody (ZEGFR:1907) will be also cloned into pCerulean (without linker). Also here, the next step will be fusing VP2 behind the this targeting molecule.</FONT></P><P style="font-size:15px; font-weight: bold; color: #ff00ff;"><U>Practical Cloning:</U></P><UL><LI>plasmid: 
</LI><LI>Vector: name: pCerulean; number: P273
</LI><LI>Insert: name: pSB1C3_6xHis_middlelinker (P310) <B>+</B> pSB1C3_Zegfr:1907 (because P267 was empty, P285 was used! See lab day 23.08.!)
</LI></UL><LI>new vector name: pCerulean_6xHis_middlelinker <B>+</B> pCerulean_Zegfr:1907
</LI><LI>buffer used: 4; Restriction-enzymes used: XbaI and PstI
</LI><P><B>Comments:</B> Because the P267 tube was empty (!), P285 was used which was prepared by Jessy - I don't know whether this is a trafo of the glycerol stock? Sequenced?
</P><P style="font-size:15px; font-weight: bold; color: #ff00ff;">Digestion</P><TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">volume of pCerulean/µl </TD><TD align="right">volume of pSB1C3_6xHis_middlelinker/µl </TD><TD align="right">volume of pSB1C3_ZEGFR:1907
</TD></TR><TR><TD> DNA  </TD><TD align="right"> 3.6 </TD><TD align="right"> 16.2 </TD><TD align="right"> 11.5
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right"> 3 </TD><TD align="right"> 2.5 </TD><TD align="right"> 2
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right"> 3 </TD><TD align="right"> 2.5 </TD><TD align="right"> 2
</TD></TR><TR><TD>Enzyme 1 XbaI </TD><TD align="right"> 1.5 </TD><TD align="right"> 1 </TD><TD align="right"> 1
</TD></TR><TR><TD>Enzyme 2 PstI </TD><TD align="right"> 1.5 </TD><TD align="right"> 1 </TD><TD align="right"> 1
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right"> 17.4 </TD><TD align="right"> 1.8 </TD><TD align="right"> 2.5
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"> 30 </TD><TD align="right"> 25 </TD><TD align="right"> 20
</TD></TR></TBODY></TABLE><UL><LI>Incubation: 1.5 h
</LI></UL><P style="font-size:15px; font-weight: bold; color: #ff00ff;">Agarose-Gel:</P><P>
0.8 g Agarose, 55 ml TAE (1.45 %), 3 µL GELRED, at 115 Volt, running time: 35 minutes
</P><TABLE border="1" cellspacing="0" cellpadding="2"><TBODY><TR><TH>Sample
</TH><TH>Sample/µl]
</TH><TH>Loading dye (6x)/µl
</TH><TH>Expected size 1 (Geneious)
</TH></TR><TR><TD>P285
</TD><TD>20 µl
</TD><TD>4 µl
</TD><TD>210 bp
</TD></TR><TR><TD>P273
</TD><TD>30 µl
</TD><TD>6 µl
</TD><TD>~ 3900 bp
</TD></TR><TR><TD>P310
</TD><TD>25 µl
</TD><TD>5 µl
</TD><TD>86 bp
</TD></TR></TBODY></TABLE><UL><LI>Marker: GeneRuler ladder mix
</LI></UL><TABLE border="1"><TBODY><TR><TH>Marker /µL
</TH><TH>Sample P285 /µl
</TH><TH>Sample P273 /µl
</TH><TH>Sample P310 /µl
</TH></TR><TR><TH>Lane
</TH><TD>5.5
</TD><TD>24
</TD><TD>25
</TD><TD>25
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color: #ff00ff;">Gel extraction</P><P>
Gel measurement:
</P><TABLE border="1"><TBODY><TR><TD align="left"><B>Sample</B></TD><TD align="left"><B>Weight</B></TD><TD align="left"><B>Volume</B></TD><TD align="left"><B>Concentration</B></TD></TR><TR><TD align="left"> P285
</TD><TD align="left"> 240 mg
</TD><TD align="left"> 20 µL
</TD><TD align="left"> 5.4 ng/µL
</TD></TR><TR><TD align="left"> P273
</TD><TD align="left"> 130 mg
</TD><TD align="left"> 20 µL
</TD><TD align="left"> 14.5 ng/µL
</TD></TR><TR><TD align="left"> P310
</TD><TD align="left"> 220 mg
</TD><TD align="left"> 20 µL
</TD><TD align="left"> 2.4 ng/µL
</TD></TR></TBODY></TABLE><P><B>Comment:</B> The DNA concentrations were very bad, which could be due to the fact, that the 6xHis_middlelinker construct's size is just 86 bp and the affibody's size is just 210 bp... 
</P><P style="font-size:15px; font-weight: bold; color: #ff00ff;">Ligation</P><P><B>1. pCerulean_6xHis_middlelinker</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>6xHis_middlelinker</B></TD><TD align="left"><B>pCerulean</B></TD></TR><TR><TD align="left"><B>Volume/µl</B></TD><TD align="left"> 2.57 </TD><TD align="left"> 6.43
</TD></TR></TBODY></TABLE><P><B>2. pCerulean_Zegfr:1907</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Zegfr:1907</B></TD><TD align="left"><B>pCerulean</B></TD></TR><TR><TD align="left"><B>Volume/µl</B></TD><TD align="left"> 2.72 </TD><TD align="left"> 6.28
</TD></TR></TBODY></TABLE><P style="font-size:15px; font-weight: bold; color: #ff00ff;">Trafo</P><P>
Trafo was performed following the standard protocol. Cells: XL1b (60 µL), DNA amount per sample: 3 µL; cells were plated onto Kanamycin-plates.
</P><H4><SPAN class="mw-headline" id="PCR_of_SV_40_terminator"><P style="font-size:15px; background-color:#66bbff;"><B>PCR of SV 40 terminator</B></P></SPAN></H4><P><B>Investigator: Bea</B></P><P style="color:#66bbff;"><B>Comment:</B> In order to obtain the SV40 poly adenylation site and clone it into the iGEM standard plasmid pSB1C3 a PCR needs to be performed with the pEGFP-C1 which contains the SV40 terminator.</P><UL><LI>Plasmid used for PCR: pEGFP-C1 (P230)</LI><LI>1:1000 dilution of P230 has been prepared</LI><LI>Plasmid used as vector: pSB1C3_CFP1 (P51.2)</LI></UL><P><B>PCR (was performed following the standard protocol)</B></P><TABLE border="1"><TBODY><TR><TD><B>Ingredients</B></TD><TD align="right"><B>Volume / µl</B></TD></TR><TR><TD> 5X Phusion HF buffer </TD><TD align="right">10
</TD></TR><TR><TD> 10 mM dNTP mix</TD><TD align="right">1
</TD></TR><TR><TD> forward primer: O160 </TD><TD align="right">2,5
</TD></TR><TR><TD> reverse primer: O161</TD><TD align="right">2,5
</TD></TR><TR><TD> DNA Template</TD><TD align="right">3
</TD></TR><TR><TD> Phusion Polymerase</TD><TD align="right">0,5
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">30,5
</TD></TR><TR><TD><B>Total volume</B></TD><TD align="right"><B>50</B></TD></TR></TBODY></TABLE><P><B>PCR program:</B></P><TABLE border="1"><TBODY><TR><TD><B>PCR Program</B></TD><TD align="right"><B>temperature/ °C</B></TD><TD align="right"><B>Time</B></TD></TR><TR><TD>1</TD><TD align="right">98 </TD><TD align="right">1min
</TD></TR><TR><TD>2</TD><TD align="right">98 </TD><TD align="right">15s
</TD></TR><TR><TD>8x</TD><TD align="right">59 </TD><TD align="right">25s
</TD></TR><TR><TD>3 </TD><TD align="right">72</TD><TD align="right">7s
</TD></TR><TR><TD>4 </TD><TD align="right">98</TD><TD align="right"> 15s
</TD></TR><TR><TD>17x</TD><TD align="right">67</TD><TD align="right">25s
</TD></TR><TR><TD>5</TD><TD align="right">72</TD><TD align="right">7s
</TD></TR><TR><TD>6x</TD><TD align="right">72</TD><TD align="right">5min
</TD></TR><TR><TD>Hold</TD><TD align="right">4</TD></TR></TBODY></TABLE><P>After the PCR was performed, the sample was loaded on a 1.5% agarose gel and run for 30 minutes. As it can be seen in the picture below the expected size of the PCR product (260bp) can be confirmed. The lower band corresponds to the added primers and does not seem to be an additional band. 
</P><P>
For cloning the obtained fragment into the pSB1C3 standard vector, the construct need to be digested with the same restriction enzymes as the PCR product will be cut with. In this case the primers were designed with XbaI overhang for the prefix, and with SpeI-NotI-PstI overhang for the suffix. Therefore the vector will be cut with XbaI and PstI. The sample were incubated at 37 C for approx. 1.5 h
</P><UL><LI>Plasmid used: pSB1C3_CFP (P51.2) c=150ng/µL</LI><LI>Add BSA</LI><LI>XbaI and PstI were used</LI></UL><P><B>Digestion of vector:</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>v<SUB>pSB1C3_CFP</SUB></B>/µL
</TD></TR><TR><TD align="left"> DNA   </TD><TD align="left"> 8
</TD></TR><TR><TD align="left"> BSA (100x) </TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 2
</TD></TR><TR><TD align="left"> Enzyme 1 XbaI </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> Enzyme 2 AgeI HF </TD><TD align="left"> 1
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 6
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>20</B></TD></TR></TBODY></TABLE><UL><LI>Incubation of the sample at 37°C</LI><LI>Incubation for 120 minutes</LI></UL><P>
After the PCR has been cut out of the gel and purified with the Qiagen Gel Extraction Kit obtaining a concentration of 45 ng/µl measured with the Nanodrop, the complete PCR product volume of 29 µL was digested with XbaI and PstI for 2 hours.<B>Digestion of the PCR product SV40</B></P><TABLE border="1"><TBODY><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>v<SUB>PCR product</SUB></B>/µL
</TD></TR><TR><TD align="left"> DNA   </TD><TD align="left"> 29,0 µl
</TD></TR><TR><TD align="left"> BSA (10x) </TD><TD align="left"> 4,0 µl
</TD></TR><TR><TD align="left"> Buffer no. 4 (10x) </TD><TD align="left"> 4,0 µl
</TD></TR><TR><TD align="left"> Enzyme 1 XbaI </TD><TD align="left"> 1,0 µl
</TD></TR><TR><TD align="left"> Enzyme 2 AgeI HF </TD><TD align="left"> 1,0 µl
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 1,0 µl
</TD></TR><TR><TD align="left"><B>Total volume</B></TD><TD align="left"><B>40</B></TD></TR></TBODY></TABLE><UL><LI>Incubation of the sample at 37°C</LI><LI>Incubation for 120 minutes</LI></UL><P>
The digested PCR product SV40 was purifed with the PC Purification Kit and concentration was measured.
</P><UL><LI><B>c<SUB>SV40</SUB>= 31,5 ng/µl</B></LI><LI><B>c<SUB>pSB1C3</SUB>= 7,74 ng/µL</B></LI></UL><P>For T4 ligation of the two fragments, the needed volumes of insert (SV40) and vector (pSB1C3) were calculated with the labtools program. The total volume DNA (vector/insert) mix will be 8 µL.
</P><UL><LI><B>v<SUB>SV40</SUB>= 0,67 µL</B></LI><LI><B>v<SUB>pSB1C3</SUB>= 7,33 µL</B></LI><LI><B>v<SUB>T4 buffer</SUB>= 1 µL</B></LI><LI><B>v<SUB>t4 ligase</SUB>= 1 µL</B></LI></UL><P>The ligation reaction was carried out for 40 minutes at room temperature. The ligated plasmid was transformed into <B>XL1-Blue cells</B> and plated on agar plates containing the appropriate amount of <B>chloramphenicol</B>.
</P><H4><SPAN class="mw-headline" id="Cloning_of_NLS_into_pCerulean_VP1up"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning of NLS into pCerulean_VP1up</B></P></SPAN></H4><P><B>Investigator: Anissa</B></P><P style="color:#33FF88;"><B>Comment:</B>  NLS was already 2 times cut of the pSB1C3_NLS and purified by Patrick (NLS 1: of the 29.08.10, NLS 2: of the 30.08.10) . Now, pCerulean_VP1up will be digested and NLS will be ligated into it.</P><UL><LI>Digestion of pCerulean_VP1up (p303)
<TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">Vector
</TD></TR><TR><TD> DNA  </TD><TD align="right">2,6
</TD></TR><TR><TD> BSA (10x) </TD><TD align="right">1,5
</TD></TR><TR><TD> Buffer 4 (10x)</TD><TD align="right">1,5
</TD></TR><TR><TD>Enzyme PstI </TD><TD align="right">1
</TD></TR><TR><TD>Enzyme AgeI  </TD><TD align="right">1
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">7,4
</TD></TR><TR><TD><B>Total volume </B></TD><TD align="right">15
</TD></TR></TBODY></TABLE></LI><LI>Gel:

A 1% gel was made and the sample was running 30 minutes.
Afterwards a gelextraction according the qiagen-protocol was performed.</LI><LI>Ligation: <TABLE border="1"><TBODY><TR><TD> components   </TD><TD align="right">Volume /µl </TD><TD align="right">concentration /ng/µl
</TD></TR><TR><TD> NLS 2 insert of 30.08.10  </TD><TD align="right">1,51 </TD><TD align="right">3
</TD></TR><TR><TD> NLS 1 insert of 29.08.10</TD><TD align="right">1,51</TD><TD align="right">3
</TD></TR><TR><TD> pCerulean_VP1up</TD><TD align="right">6,49 </TD><TD align="right">29
</TD></TR></TBODY></TABLE></LI><LI>Transfomation: was performed into XL1 blue on a kanamycin plate according the standard protocol.
</LI></UL><H4><SPAN class="mw-headline" id="Cloning_CFP_middlelinker_.28from_pSB1C3_CFP_middlelinker.2C_P276.29_into_pCerulean_.28P273.29"><P style="font-size:15px; background-color:#66bbff;"><B>Cloning CFP_middlelinker (from pSB1C3_CFP_middlelinker, P276) into pCerulean (P273)</B></P></SPAN></H4><P><B>Investigator: Patrick </B>
P276 digestion (207 ng/µl): 14 µl DNA sample, 2 µl Buffer 4, 2 µl BSA, 1 µl XbaI, 1 µl PstI-HF. 
P273 digestion (419 ng/µl): 4 µl DNA sample, 1 µl Buffer 4, 1 µl BSA, 1 µl XbaI, 1 µl PstI-HF. 
Digestion time: 1 h 45 min.
</P><P>Expected size of the fragments: 
P276: 786 &amp; 2058 bp 
P273: 756 &amp; 3938 bp </P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">My dear friend, where is your initial comment? Nice gel :) Yours sincerely, Hanna :)</DIV></DIV></DIV><P>see: <A href="http://www.molbiotech.uni-freiburg.de/iGEM/wiki2010/index.php/August_2010#Strategy_for_VP2_N-terminal_fusion_and_VP1_insertion_of_targeting_moelcules" class="external free" rel="nofollow">http://www.molbiotech.uni-freiburg.de/iGEM/wiki2010/index.php/August_2010#Strategy_for_VP2_N-terminal_fusion_and_VP1_insertion_of_targeting_moelcules</A><FONT color="#ff00ff">Haha, good idea! :) </FONT></P><P>
Gelextraction:
Obviously there was something wrong with the gel or the sample. The marked bands were cut out and the gelextraction was performed acording to he standard protocol, yielding the following concentrations:
P276 up: 12,6 ng/µl  
P276 down: 9,9 ng/µl 
P273: 37,7 ng/µl </P><P>All of them were polluted.
</P><P>There were two bands at a level where no bands should be. P 276 looks also strange because accidentally 6,5 µl 1kb GeneRuler were pipetted into the sample. The expected bands are all blurred. Even the expected 756 bp fragment of P273 is not at the level it should be. To ensure i dont throw away the insert i cut out both bands from P276. Now tomorrow there will be a ligation and a transformation with two samples instead of one 
</P><H4><SPAN class="mw-headline" id="Sequencing_results_of_pSB1C3_6xHis_middlelinker_and_pSB1C3_6xHis_mVenus"><P style="font-size:15px; background-color:#66bbff;"><B>Sequencing results of pSB1C3_6xHis_middlelinker and pSB1C3_6xHis_mVenus</B></P></SPAN></H4><P><B>Investigator: Hanna</B></P><P><B>1st Comment:</B> Sequencing of pSB1C3_6xHis_middlelinker looked good:</P><P><B>2nd Comment:</B> Sequencing of mVenus looked good. The His-Tag delivered 2 additional bp and the suffix 1 transition (see picture). This could be due to bad sequencing qualities at this region... but needs to be verified!</P><H4><SPAN class="mw-headline" id="Contiuation:_Repetition_of_subcloning_Cap-insert_into_pAAV_RC"><P style="font-size:15px; background-color:#66bbff;"><B>Contiuation: Repetition of subcloning Cap-insert into pAAV_RC</B></P></SPAN></H4><P><B>Investigator: Stefan</B></P><P><B>Comments:</B> Overnight digestion does not look good, therefore a new approach will be performed today using different concentrations of spermidine (0,5mM, 2,5mM and 10mM) because in Oller et al (Biochemistry, 1991) it was suggested that this improves digestion with BspMI.
</P><P>A two-step digestion was performed: <B>1st step</B></P><TABLE border="1"><TBODY><TR><TD align="right"><B>P211 / µl</B></TD><TD><B>P250 / µl</B></TD></TR><TR><TD>DNA</TD><TD align="right">8,5  </TD><TD align="right"> 2
</TD></TR><TR><TD>buffer 3</TD><TD align="right">4,6 </TD><TD align="right"> 5
</TD></TR><TR><TD> Enzyme BsiWI</TD><TD align="right">1  </TD><TD align="right"> 1
</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD align="right">32,4 </TD><TD align="right"> 38
</TD></TR><TR><TD>total volume</TD><TD align="right">46 </TD><TD align="right">46
</TD></TR></TBODY></TABLE><UL><LI>Incubation: 70 minutes; 55°C</LI></UL><P><B>2nd step</B></P><TABLE border="1"><TBODY><TR><TD align="right"><B>P211 / µl</B></TD><TD><B>P250 / µl</B></TD></TR><TR><TD>Mix obtained from step 1 </TD><TD align="right">46  </TD><TD align="right"> 46
</TD></TR><TR><TD> Enzyme BspMI</TD><TD align="right">4  </TD><TD align="right"> 4
</TD></TR><TR><TD>total volume</TD><TD align="right">50 </TD><TD align="right">50
</TD></TR></TBODY></TABLE><UL><LI>Incubation: 2,5h ; 37°C</LI></UL><P><B>Agarosegel</B>
0.4 g Agarose, 50 ml TAE (0,8 %), 3 µL GELRED, at 115 Volt
</P><P><B>Gel extraction</B> 
Gel-Ex was performed according to protocol.
NanoDrop:
</P><UL><LI>pMA_RepCap-insert: c= 7,42 ng/µl
</LI><LI>pAAV_RC_ins-rep: c=12,9 ng/µl
</LI></UL><P><B>T4 Ligation</B> 
Volume insert: 3,43 µl
Volume vector: 4,57 µl
Ligation incubated for 50 minutes at room temperature.<B>Transformation</B>
Trafo was preformed according to standard protocol using XL1blue cells on plates containing Ampicilin.
</P><H4><SPAN class="mw-headline" id="2nd_Repetition_of_test_digestion_of_Loop_insertion_BioBricks"><P style="font-size:15px; background-color:#66bbff;"><B>2nd Repetition of test digestion of Loop insertion BioBricks</B></P></SPAN></H4><P><B>Investigator: Achim, Anna </B></P><P style="font-size:13px; color:#003399;"><B>Comment</B>: First test digestion didn't work (see 31.08.10), this time a new approach was done with NotI. </P><P><B>Test digestion:</B></P><TABLE border="1"><TBODY><TR><TD> components </TD><TD>Volume for each sample /µl
</TD></TR><TR><TD> DNA  </TD><TD> 10
</TD></TR><TR><TD> BSA (10x) </TD><TD>1,5
</TD></TR><TR><TD> Buffer 4 (10x) </TD><TD>1,5
</TD></TR><TR><TD>Enzyme NotI </TD><TD>0,5
</TD></TR><TR><TD>H2O </TD><TD> 1,5
</TD></TR><TR><TD><B>Total volume /µl</B></TD><TD>15
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD align="left"><B>Sample-No.</B></TD><TD align="left"><B>1+2</B></TD><TD align="left"><B>3+4</B></TD><TD align="left"><B>5+6</B></TD><TD align="left"><B>7+8</B></TD><TD align="left"><B>9+10</B></TD><TD align="left"><B>11+12</B></TD><TD align="left"><B>13+14</B></TD><TD align="left"><B>15+16</B></TD><TD align="left"><B>17+18</B></TD><TD align="left"><B>19+20</B></TD><TD align="left"><B>21-23</B></TD><TD align="left"><B>24+25</B></TD><TD align="left"><B>26+27</B></TD><TD align="left"><B>28+29</B></TD></TR><TR><TD align="left"><B>Components</B></TD><TD align="left"><B>453 BAP</B></TD><TD align="left"><B>587 BAP</B></TD><TD align="left"><B>587 KO BAP</B></TD><TD align="left"><B>453 RGD</B></TD><TD align="left"><B>587 RGD</B></TD><TD align="left"><B>587 KO RGD</B></TD><TD align="left"><B>453 HIS</B></TD><TD align="left"><B>587 HIS</B></TD><TD align="left"><B>587 KO HIS</B></TD><TD align="left"><B>587 KO EMPTY</B></TD><TD align="left"><B>453 Z34C</B></TD><TD align="left"><B>587 Z34C</B></TD><TD align="left"><B>587 KO Z34C</B></TD><TD align="left"><B>587 KO Z34C SPACER</B></TD></TR><TR><TD align="left"> Oligo 1</TD><TD align="left">O124: 10 </TD><TD align="left">O126: 10 </TD><TD align="left">O128: 10 </TD><TD align="left">O130: 10 </TD><TD align="left">O132: 10 </TD><TD align="left">O134: 10 </TD><TD align="left">O135: 10 </TD><TD align="left">O137: 10 </TD><TD align="left">O139: 10 </TD><TD align="left">O141: 10 </TD><TD align="left">O143: 10 </TD><TD align="left">O145: 10 </TD><TD align="left">O147: 10 </TD><TD align="left">O149: 10
</TD></TR><TR><TD align="left"> Oligo 2</TD><TD align="left">O125: 10 </TD><TD align="left">O127: 10 </TD><TD align="left">O129: 10 </TD><TD align="left">O131: 10 </TD><TD align="left">O133: 10 </TD><TD align="left">O151: 10 </TD><TD align="left">O136: 10 </TD><TD align="left">O138: 10 </TD><TD align="left">O140: 10 </TD><TD align="left">O142: 10 </TD><TD align="left">O144: 10 </TD><TD align="left">O146: 10 </TD><TD align="left">O148: 10 </TD><TD align="left">O150: 10
</TD></TR></TBODY></TABLE><P><B>30</B>: pSb1C3_Bla (cut version)
<B>31</B>: pSb1C3_Bla (uncut)
</P><P>
Incubation time: 1 h, Incubation temperature: 37°
</P><P><B>Preparation of gel:</B>
1 g Agarose, 100 ml TAE (1%), 6 µl GELRED , at 115 Volt, running time: 45 minutes
</P><P><B>Expected fragment sizes /bp:</B>
pSB1C3: 2051 bp
Bla: 905
BAP: ~150
RGD: ~ 130
Z34C: ~ 200
His: ~ 125 </P><P>The following clones were sent for sequencing: 2,3,5,7,9,11,13,16,17,19 and 25.
</P><P style="font-size:13px; color:#003399;"><B>Comment</B>: Clones (see above) were sent for sequencing with VR2 Primer. In addition, 8 mL DYT medium was inoculated with each of the clones for preparation of glycerol stocks. For sequencing results go to labday 01.09.10.</P><H4><SPAN class="mw-headline" id="Midi-Prep_of_pAAV_RC_.26_pHelper"><P style="font-size:15px; background-color:#66bbff;"><B>Midi-Prep of pAAV_RC &amp; pHelper</B></P></SPAN></H4><P><B>Investigators: Chris W. </B></P><P style="font-size:13px; color:#003399;"><B>Comment</B>: Midi-Preps of pHelper=P356 and pAAV_RC=P357</P><P>The Midi-Preps were performed according to the standard protocol yielding the following concentrations:
</P><TABLE border="1"><TBODY><TR><TD> plasmid-no. </TD><TD align="right">P356</TD><TD align="right">P357
</TD></TR><TR><TD> concentration (ng/µl)</TD><TD align="right">1068,0 </TD><TD align="right">1274.80
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Transformation_of_Biobrick_CD"><P style="font-size:15px; background-color:#66bbff;"><B>Transformation of Biobrick CD</B></P></SPAN></H4><P><B>Investigator: Kira</B>
Transformation was performed according to the standard protocol.
</P><CENTER><A href="https://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/September" class="external text" rel="nofollow"><B>=&gt; Go to Labjournal September part 1 (labday 107 - 123)</B></A></CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/August2">http://2010.igem.org/Team:Freiburg_Bioware/NoteBook/Labjournal/August2</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg_Bioware/NoteBook/Labjournal/August2" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg_Bioware/NoteBook/Labjournal/August2" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/August2&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg_Bioware/NoteBook/Labjournal/August2&amp;oldid=197113" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>