Team:Sydney Australia/ProteinModel2
Team
About Us
Attributions
Project
Project- Home
Safety
Sense
Express
Keep Fresh
Applied Design
Business Plan
Notebook
Interlab
Protocols
Human Practices
Human Practices - Summary
Integrated Human Practices
Outreach
Theory of Knowledge
Collaboration
Modelling
Modelling - Home
Protein Modelling
Etnr1
Protein Modelling
Etnr2
Internal Cellular Model
Macroscopic Diffusion Model
Software
Software - Home
Primer Design App
SmartPhone App
Achievements
Parts
Medals
EtnR2: ProtParam – What does the primary structure tell us?
  
The ExPaSy ProtParam tool was used to derive basic information concerning EtnR2 based on its amino acid sequence.
It was calculated that:
      •Number of amino acids: 198
      •Molecular weight: 21803.5
      •Theoretical pI: 4.65
It was also found that EtnR2 has an instability index of 21.05, classifying the protein as stable.
EtnR2: BLAST analysis – Are there any known close homologues?
The NCBI BLAST tool was used to conduct a protein BLAST (Basic Local Alignment Search Tool) to identify proteins with close sequence similarity to the EtnR2 sequence. The SmartBLAST feature selects ‘the three best matches in the sequence database together with the two best matches from well-studied reference species, showing phylogenetic relationships based on multiple sequence alignment and conserved protein domains.’ In the case of EtnR1, there were no matches from well-studied reference species. The top matches were all hypothetical Mycobacterium proteins. Figure 1 shows the phylogenetic tree constructed based on sequence similarities and conserved domains. The individual alignments can be found at the bottom of the page.
Figure 1. Phylogenetic tree produced by SmartBLAST with query: hypothetical protein [Mycobacterium chubuense] (accession: WP_014805816.1, GI: 504618714).
Other candidates shown are hypothetical protein [Mycobacterium tusciae] (accession: WP_006247393.1, GI: 493289683), hypothetical protein [Mycobacterium sp. JS623] (accession: WP_015305843.1 GI: 505118741) and hypothetical protein [Mycobacterium rhodesiae] (accession: WP_005148477.1 GI: 491290461). The BLAST analysis also revealed a predicted MEDS domain (Figure 2). The MEDS domain has been characterised in in hydrocarbon sensing in Methylococcus and is generally fused to a histidine kinase.
Figure 2. Snapshot of BLAST results, showing putative HTH-30 domain at the C-terminal end of EtnR2.
EtnR2: PsiPred – Secondary structure prediction
The UCL Bioinformatics PsiPred tool was used to predict the secondary structure of EtnR2 based on its amino acid sequence (Figure 3). The putative helices and sheets were generally predicted with high confidence.
Figure 3. Secondary structure prediction using PsiPred for EtnR2 based on amino acid sequence.
EtnR2: SwissModel Homology Modelling
The SWISS-MODEL server [1-4] is an automated service that selects the best matching template to a submitted amino acid sequence, performs the alignment and builds a 3D model. The most important limiting factor of the quality of the model is generally the matching of the query sequence with an appropriate template. SwissProt threaded EtnR2 onto a selected template protein: flagellar accessory protein FlaH from Methanocaldococcus jannaschii. It is a conserved component of the archaeal flagellum system.
Based on the structure of the template, a homology model was constructed (Figure 4). The predicted structure consisted of a parallel beta sheet surrounded by alpha helices.
Figure 4. Homology model of EtnR2
EtnR2: BLAST alignments with three best matches and from well-studied reference species
hypothetical protein [Mycobacterium rhodesiae]
Sequence ID: WP_005148477.1 Length: 197 Number of Matches: 1
Related Information
Identical Proteins-Identical proteins to WP_005148477.1
Range 1: 1 to 197
Score
Expect
Method
Identities
Positives
Gaps
359 bits(922)
9e-133()
Compositional matrix adjust.
179/198(90%)
185/198(93%)
1/198(0%)
Query
Sequence
Query  1 - 60
Sbjct  1 - 60
MRTSDTAAVGVMGVSLAAGDHVCAFYPTLAERNEILLPYLKEGLNHGDKCICIVSDDSAD
M T+D A VGVMGVSLAAGDHVCAFYPTLAERNEILLPYLKEGL  GDKCICIVSD
MSTTDMATVGVMGVSLAAGDHVCAFYPTLAERNEILLPYLKEGLKLGDKCICIVSDSPDS
Query 61- 120
Sbjct  61- 120
SDLDSVRCLSQHTGQLEVDHSGDTYLKGGNFSSDRMLKYWNSAISQAVDDGFDFARGAGE
DSV+CLSQ TGQLEVDHSGDTYLKGGNFSSDRMLKYWNS+ISQAVDDGF+FARGAGE
DL-DSVKCLSQDTGQLEVDHSGDTYLKGGNFSSDRMLKYWNSSISQAVDDGFNFARGAGE
Query  121- 180
Sbjct  121- 180
MTWALEKMPGVENLVSYESQLNSVLRDYPAVIVCLYELGRFSGDLLVEVLKTHPKVILGG
MTWALEKMPGVENLVSYESQLNSVLR+YPAVIVCLYELGRFSGD+LVEVLKTHPKVILGG
MTWALEKMPGVENLVSYESQLNSVLREYPAVIVCLYELGRFSGDMLVEVLKTHPKVILGG
Query  181- 240
Sbjct  181- 240
IVLENPYYLEHNEYLTSR
IVLENPYYLEHNEYLTSR
IVLENPYYLEHNEYLTSR
hypothetical protein [Mycobacterium tusciae]
Sequence ID: WP_006247393.1 Length: 199 Number of Matches: 1
Related Information
Range 1: 1 to 199
Score
Expect
Method
Identities
Positives
Gaps
327 bits(838)
7e-120()
Compositional matrix adjust.
154/199(77%)
175/199(87%)
1/199(0%)
Query
Sequence
Query  1 - 59
Sbjct  1- 60
MRTSDTAAVGVMGVSLAAGDHVCAFYPTLAERNEILLPYLKEGLNHGDKCICIVSDDS-A
MRT+D A VGVMGVSLA GDHVCAFYP  AERNEIL+PYL+EGL+ GDKCICIVSD +
MRTTDAATVGVMGVSLAPGDHVCAFYPGAAERNEILIPYLQEGLDAGDKCICIVSDATEP
Query  60- 119
Sbjct 61- 120
DSDLDSVRCLSQHTGQLEVDHSGDTYLKGGNFSSDRMLKYWNSAISQAVDDGFDFARGAG
D DLDS+RC S H+ QL VD S +TYLKGG+FSSDRMLKYW+SAI QA++DG+ FARGAG
DGDLDSIRCYSHHSNQLAVDRSSETYLKGGSFSSDRMLKYWDSAIRQAIEDGYTFARGAG
Query  120- 179
Sbjct  121- 180
EMTWALEKMPGVENLVSYESQLNSVLRDYPAVIVCLYELGRFSGDLLVEVLKTHPKVILG
EMTWALE MPGVE+LV+YES+LN  L++YPAVIVCLYELGRFSG++LVEVLKTHPKVILG
EMTWALEAMPGVEDLVTYESELNDFLKEYPAVIVCLYELGRFSGEMLVEVLKTHPKVILG
Query  180- 198
Sbjct 181- 199
GIVLENPYYLEHNEYLTSR
GIVLENPYYL+  EYL SR
GIVLENPYYLDQAEYLASR
hypothetical protein [Mycobacterium sp. JS623]
Sequence ID: WP_015305843.1 Length: 199 Number of Matches: 1
Related Information
Identical Proteins-Identical proteins to WP_015305843.1
Range 1: 1 to 199
Score
Expect
Method
Identities
Positives
Gaps
325 bits(833)
4e-119()
Compositional matrix adjust.
156/199(78%)
176/199(88%)
1/199(0%)
Query
Sequence
Query  1 - 59
Sbjct  1- 60
MRTSDTAAVGVMGVSLAAGDHVCAFYPTLAERNEILLPYLKEGLNHGDKCICIVSDDSA-
M T+DTA VGVMGVSLA GDHVCAFYP  AERNEIL+PYL+EGL+ GDKCICIVSD +A
MLTTDTATVGVMGVSLAPGDHVCAFYPGAAERNEILIPYLREGLDAGDKCICIVSDATAP
Query  60- 119
Sbjct 61- 120
DSDLDSVRCLSQHTGQLEVDHSGDTYLKGGNFSSDRMLKYWNSAISQAVDDGFDFARGAG
D +LDSVRC S ++ QL VD SGDTYLKGG FS+DRML+YW+SAISQA++DG+ FARGAG
DGELDSVRCHSHNSDQLAVDRSGDTYLKGGGFSADRMLEYWDSAISQAMEDGYTFARGAG
Query  120- 179
Sbjct  121- 180
EMTWALEKMPGVENLVSYESQLNSVLRDYPAVIVCLYELGRFSGDLLVEVLKTHPKVILG
EMTWALE MPGVENLV+YE +LN  L+ YPAVIVCLYELGRFSG++LVEVLKTHPKVILG
EMTWALETMPGVENLVTYEFELNDFLKKYPAVIVCLYELGRFSGEMLVEVLKTHPKVILG
Query  180- 198
Sbjct 181- 199
GIVLENPYYLEHNEYLTSR
GIVLENPYYL+H EYL SR
GIVLENPYYLDHAEYLASR
For reference: EtnR2 Amino Acid Sequence
MRTSDTAAVGVMGVSLAAGDHVCAFYPTLAERNEILLPYLKEGLNHGDKCICIVSDDSADSDLD
SVRCLSQHTGQLEVDHSGDTYLKGGNFSSDRMLKYWNSAISQAVDDGFDFARGAGEMTWALEKM
PGVENLVSYESQLNSVLRDYPAVIVCLYELGRFSGDLLVEVLKTHPKVILGGIVLENPYYLEHN
EYLTSR
References
1. Marco Biasini, Stefan Bienert, Andrew Waterhouse, Konstantin Arnold, Gabriel Studer, Tobias Schmidt, Florian Kiefer, Tiziano Gallo Cassarino, Martino Bertoni, Lorenza Bordoli, Torsten Schwede. (2014). SWISS-MODEL: modelling protein tertiary and quaternary structure using evolutionary information. Nucleic Acids Research; (1 July 2014) 42 (W1): W252-W258; doi: 10.1093/nar/gku340.
2. Arnold K., Bordoli L., Kopp J., and Schwede T. (2006). The SWISS-MODEL Workspace: A web-based environment for protein structure homology modelling. Bioinformatics, 22,195-201.
3. Kiefer F, Arnold K, Künzli M, Bordoli L, Schwede T (2009). The SWISS-MODEL Repository and associated resources. Nucleic Acids Research. 37, D387-D392.
4. Guex, N., Peitsch, M.C., Schwede, T. (2009). Automated comparative protein structure modeling with SWISS-MODEL and Swiss-PdbViewer: A historical perspective.Electrophoresis, 30(S1), S162-S173.
School of Life and Environmental Sciences The University of Sydney City Road, Darlington 2006, New South Wales, Sydney, Australia
