Team:Sydney Australia/Sense
Team
About Us
Attributions
Project
Project- Home
Safety
Sense
Express
Keep Fresh
Applied Design
Business Plan
Notebook
Interlab
Protocols
Human Practices
Human Practices - Summary
Integrated Human Practices
Outreach
Theory of Knowledge
Collaboration
Modelling
Modelling - Home
Protein Modelling
Etnr1
Protein Modelling
Etnr2
Internal Cellular Model
Macroscopic Diffusion Model
Software
Software - Home
Primer Design App
SmartPhone App
Achievements
Parts
Medals
Background
• Two Mycobacterium genes of interest, designated EtnR1 and EtnR2, are suspected to control a two-component regulatory system for ethylene detection and metabolism in the native NBB4 strain
Aims:
• Test recombinant expression of EtnR1 and EtnR2 in E. coli.
• Characterise the interactions between ethylene, EtnR1, EtnR2 and EtnP.
Suspected Circuitry
• EtnR2 is thought to bind to ethylene, leading to phosphorylation of the putative transcription factor EtnR1.
• Binding of EtnR1 to a putative ethylene promoter (EtnP) then activates transcription of the genes responsible for ethylene metabolism.
Results
• Confirmed expression of EtnR1 and EtnR2 in E. coli.
• Characterised EtnR1 as a DNA-binding protein by confirming binding to EtnP.
BioBricks
We submitted three new BioBricks to the iGEM parts registry:
• BBa_K1996000 – EtnP sequence from Mycobacterium NBB4
• BBa_K1996001 – codon optimised EtnR1 sequence
• BBa_K1996002 – codon optimised EtnR2 sequence
Background
The first ethylene-oxidising bacterium, Mycobacterium sp. strain E20 , was isolated in 1978 (de Bont and Harder, 1978). Subsequently, the characterisation of the enzymes involved in aerobic ethylene metabolism was aimed at utilising such strains for microbial bioremediation of chlorinated ethenes, such as vinyl chloride, as many of these are environmental pollutants (Mattes et al., 2010). However, a separate stream of research has identified another potential use for ethylene-oxidising bacteria – in regulating and monitoring fruit ripening.
As the key hormone responsible for the ripening of most fruits, ethylene levels need to be monitored at every step of the fruit packaging process, and there is a promising market for a cheap, portable and convenient ethylene sensor . Hence, we turned to Mycobacterium NBB4 , the best characterised ethylene-oxidising bacterium, with the aim of designing a cell-based biosensor for ethylene.
In Mycobacterium NBB4 , the enzymes required for ethylene metabolism are contained within an operon organised in two bidirectional gene clusters. One gene cluster encodes the catabolic enzymes required to convert ethylene into acetyl-CoA, the final product of the degradation pathway which is subsequently directed to the tricarboxylic acid cycle. The other gene cluster encodes the enzymes responsible for synthesising coenzyme M, a cofactor required for the function of the second enzyme in the metabolic pathway.
While the enzymes required for ethylene metabolism have been investigated in detail (Coleman et al. 2011), the regulatory determinants of ethylene metabolism have yet to be characterised. Studies examining the expression of ethylene-oxidising genes in Mycobacterium NBB4 when grown on different substrates have demonstrated that these genes are only expressed when the strain is exposed to ethylene (Figure 1; Coleman et al. 2011). This suggests that these genes are under the control of an inducible, rather than constitutive, promoter.
Figure 1. SDS-PAGE gel showing induction of Mycobacterium NBB4 ethylene-oxidising genes when grown on ethene (adapted from Li, C. 2006). Lysates from cells grown on ethanol were used as a control. The bands labelled 1 to 3 correspond to enzymes required for ethylene degradation. Band 1 is EtnC, band 2 is epoxyalkane:coenzyme M transferase and band 3 is EtnA.
A promising candidate for regulation of this operon are two genes present in the gene clusters, designated EtnR1 and EtnR2 (Figure 2). We suspect that together, EtnR1 and EtnR2 represent a two-component regulatory system responsible for activating gene expression from the putative promoter EtnP upon exposure to ethylene. Such a two-component system is common in bacteria, involving a histidine protein kinase that phosphorylates another regulatory protein when it senses the environmental stimulus (Stock et al. 2000)
Figure 2. Schematic of the ethylene metabolism operon in Mycobacterium NBB4. The two bidirectional gene clusters are indicated by the curved brackets. Between the two clusters is the putative promoter for their transcription, EtnP. EtnR1 and EtnR2 are located in the ethylene metabolism cluster, indicated in bold.
There is strong evidence supporting such a hypothesis:
EtnR1 and EtnR2 are the only genes of unknown function in the well-characterised ethylene metabolism operon.
The regulation of the genes responsible for ethylene degradation is also unknown. This may suggest that EtnR1 and EtnR2 are involved in regulating this process.
EtnR1 and EtnR2 are highly conserved across ethylene-oxidising Mycobacterium.
All other ethylene-oxidising mycobacterial strains such as Mycobacterium rhodesiae NBB3, rhodesiae JS60 and tusciae JS617 contain homologues of EtnR1 and EtnR2 in similar positions in alkene monooxygenase clusters, that have highly conserved residues:
EtnR1 - Mouse over the images to Zoom
EtnR2
EtnP
Figure 3. Amino acid sequence similarity between EtnR1, EtnR2 and EtnP homologues from Mycobacterium NBB4, NBB3, JS60 and JS617 . * indicates fully conserved, : indicates strongly similar amino acids, . indicates weakly similar amino acids.
These EtnR1 and EtnR2 homologues all share >70% amino acid identity. Homologues in bacteria that do not oxidise ethylene, on the other hand, only share ~50% or less amino acid identity. Furthermore, Mycobacterium strains that do not oxidise ethylene lack any homologues of EtnR1 and EtnR2.
Two-component regulatory systems are commonly used in bacterial gene regulation.
Typically, this involves a membrane-bound histidine protein kinase (which we suspect to be EtnR2), which responds to environmental stimuli by autophosphorylating and self-activation, before phosphorylating a response regulator protein (Stock et al. 2000). This activates the response regulator (which we suspect to be EtnR1), enabling it to perform its effector function. In most cases, the response regulator is a transcription factor (Stock et al. 2000).
EtnR1 has DNA-binding and kinase domains and EtnR2 has a MEDS domain.
From protein domain analysis and searching of protein databases, EtnR1 was found to have a helix-turn-helix domain near the C terminus which is a DNA-binding motif commonly associated with negative repressor transcription factors in prokaryotic systems. EtnR1 was also found to have a kinase domain near the N terminus, which is commonly linked with activating/deactivating phosphorylation marks. EtnR2 has a MEDS domain, which is commonly associated with a substrate sensing-role, being found in methane-sensing for methylotrophs and methanogens.
Experimental Aims
We had a twofold curiosity to understand and characterise the EtnR1/2 system better, and also utilise it as ethylene-responsive circuitry. In order to do this, we needed to:
• Test recombinant expression of EtnR1 and EtnR2 in E. coli.
• Characterise the interactions between ethylene, EtnR1, EtnR2 and the putative promoter
We would first determine whether the BL21 DE3 strain of E. coli. could support the expression of our two mycobacterial genes. After confirming expression of EtnR1/2 and affinity purification, we can decipher the precise protein-DNA, protein-protein and protein-substrate interactions that make up our system.
This would serve as a proof of concept for a novel regulatory mechanism of ethylene degradation in Mycobacterium NBB4 and other mycobacterial strains.
Figure 4. Experimental characterisation of EtnR1 and EtnR2 interactions with ethylene, the putative promoter and each other. The various assays that would be used to characterise each interaction are listed in the boxes on the right.
Expression Constructs
Figure 5. pET28c construct with main characteristics: IPTG-inducible expression, N-terminal His-Tag for affinity purification.
We selected pET28c to use for the expression of our mycobacterial proteins as it was available in the laboratory and contained an N-terminal His6-Tag independent of insert, meaning that we could proceed to protein characterisation following expression with the same construct. pET28 also has inducible expression which was valuable as it provided an extra measure of control over EtnR1 and EtnR2 expression.
Progress and Results
Cloning
We generated coding sequences of EtnR1 and EtnR2 either native to Mycobacterium NBB4 or codon optimised for E. coli. K12 which were then cloned into the pET28c plasmid to be under the control of the T7 promoter . All expression assembly was successful and verified by colony PCR, restriction digestion and Sanger sequencing. We also cloned EtnR1, EtnR2 and the putative promoter from NBB4 into the assembly standard BioBrick pSB1C3 for parts submission verified using similar processes.
Expression assay in BL21 (DE3) E. coli.
To test expression of EtnR1 and EtnR2, we transformed the pET constructs into BL21 (DE3) E. coli. containing a pGro7 helper plasmid expressing the GroEL and GroES chaperone proteins for aiding protein folding. We selected for Km and Cm resistant colonies, induced protein expression upon addition of IPTG in liquid culture and ran the cellular lysates on SDS-PAGE (Figure 6).
Figure 6. SDS-PAGE sizing of EtnR1 and EtnR2 expressed following 6 hour IPTG induction. Cellular lysates were isolated from BL21 (DE3) E. coli. with pGro7 chaperone plasmid expressing pET-EtnR1 or pET-EtnR2, and separated electrophoretically on 12% SDS-PAGE. The bands around 66-70 kDa and 25-28 kDa respectively were trypsin-digested and analysed using mass spectrometry. (a) Isolated EtnR1 lysates. (b) Isolated EtnR2 lysates.
EtnR1 was recovered from the soluble lysate fraction at a size of roughly 66 kDa, whereas EtnR2 expression was only detected upon solubilisation with SDS at a size of roughly 25 kDa. This indicates that EtnR2 may be membrane associated or expressed in inclusion bodies. Expression of the two genes was found to be stronger using codon optimised sequences, and so were used for all subsequent applications including protein characterisation and biosensor construction.
Mass spectrometry analysis was performed to confirm the suspected SDS-PAGE bands were our proteins of interest. The amino acid sequences obtained indicated we had successfully expressed EtnR1 and EtnR2 in E. coli. from the native Mycobacterium host (Figure 7).
EtnR1
EtnR2
Finally, EtnR1 and EtnR2 expression with an N-terminal His6-Tag was confirmed via Western blotting using an anti-His antibody performed by the UNSW iGEM team (Figure 8) as part of our wet lab collaboration with bands observed at the expected sizes for each protein.
RESULT: The Mycobacterium NBB4 genes EtnR1 and EtnR2 were successfully expressed in E. coli. verified through SDS-PAGE, mass spectrometry and Western blot.
Figure 8. Western blot with anti-His antibody. Cellular lysates were isolated from BL21 (DE3) E. coli. with pGro7 chaperone plasmid following IPTG-induced protein expression, and separated using SDS-PAGE. The proteins were transferred to a nitrocellulose blot and probed with anti-His antibody. Loading order is indicated above the gel. SeeBlue Plus2 Protein Standard was used as the protein ladder. Insol. - Insoluble fraction, obtained through SDS-treatment; sol. - Soluble fraction. The bands corresponding to EtnR1 and EtnR2 are indicated by the arrows.
Characterisation
In order to isolate purified EtnR1 and EtnR2, we used affinity chromatography based off the attraction between pyridine moieties in histidine residues, and divalent metal cations such as nickel(II) and cobalt(II). Following multiple attempts using Ni2+-loaded columns, we successfully obtained EtnR1 and EtnR2 in purer fractions using Co2+-coated beads, again confirmed via mass spectrometry.
Subsequently, an electrophoretic mobility shift assay (EMSA) was performed to determine the interaction between EtnR1 and EtnP, with band retardation observed when the EtnR1-EtnP complex was run on an agarose gel. This confirms that EtnR1 acts as a DNA-binding protein for EtnP, and provides evidence of its role in transcriptional activation of ethylene metabolism in Mycobacterium NBB4 and other ethylene-oxidising bacteria.
RESULT: EtnR1 was characterised as binding to EtnP, and so functions as a likely transcription factor for regulating ethylene metabolism.
Figure 9. Electrophoretic mobility shift assay for EtnP DNA. The 250 bp region of EtnP was PCR amplified and 100 ng of DNA incubated with 6 mg of protein. The protein-DNA mixtures were run on a 3% agarose gel at 150 V for 100 minutes and stained with Gel Green overnight for visualisation. Lane 1: NEB 100 bp ladder. Lane 2: EtnP DNA only. Lane 3: EtnR1 protein only. Lane 4: EtnR1 protein and EtnP DNA. Lane 5: BSA protein and EtnP DNA.
Next Steps
Given more time, we would like to fully investigate the rest of the proposed regulatory system - specifically the interaction between EtnR2 and ethylene, and the interaction between EtnR1 and EtnR2. We would plan to use a phosphorylation assay and a pulldown assay to test these respectively, as outlined in Figure 5.
Leading on from our characterisation of EtnR1, we would aim to confirm that it acts as a transcription factor, and investigate whether it acts as a positive or negative regulator. Lastly, we would like to examine the strengths of putative promoters from other ethylene-metabolising strains of Mycobacterium, and determine whether the NBB4 system is homologous in other strains.
References
1. de Bont, J. A. M. and Harder, W. (1978). Metabolism of ethylene by Mycobacterium E20. FEMS Microbiology Letters, 3, pp.89-93.
2. Mattes, T. E., Alexander, A. K. and Coleman, N. V. (2010). Aerobic biodegradation of the chloroethenes: pathways, enzymes, ecology, and evolution. FEMS Microbiology Reviews, 34, pp.445-475.
3. Coleman, N. V., Yau, S., Wilson, N. L., Nolan, L. M., Migocki, M. D., Ly, M., Crossett, B. and Holmes, A. J. (2011). Untangling the multiple monooxygenases of Mycobacterium chubuense strain NBB4, a versatile hydrocarbon degrader. Environmental Microbiology Reports, 3(3), pp.297-307.
4. Li, C. (2006) Soluble diiron monooxygenases: the linkage between genotype and phenotype in Mycobacterium NBB4. (Honours thesis) University of Sydney, Sydney, Australia.
5. Stock, A. M., Robinson, V. L. and Goudreau, P. N. (2000). Two-component signal transduction. Annual Review of Biochemistry, 69, pp.183-215.
School of Life and Environmental Sciences The University of Sydney City Road, Darlington 2006, New South Wales, Sydney, Australia
