Team:Sydney Australia/Description
Team
About Us
Attributions
Project
Project- Home
Safety
Sense
Express
Keep Fresh
Applied Design
Business Plan
Notebook
Interlab
Protocols
Human Practices
Human Practices - Summary
Integrated Human Practices
Outreach
Theory of Knowledge
Collaboration
Modelling
Modelling - Home
Protein Modelling
Etnr1
Protein Modelling
Etnr2
Internal Cellular Model
Macroscopic Diffusion Model
Software
Software - Home
Primer Design App
SmartPhone App
Achievements
Parts
Medals
Background
amilCP is a blue chromoprotein from the coral Acropora millepora.
Aims
To improve the part by producing darker blue and/or other coloured mutants, to be used as a reporter gene in our biosensor.
Results
Produced 3 colour mutants:
• Pink
• Purple
• Mint Green
BioBricks
• BBa_K1996003 - pink colour mutant of amilCP
• BBa_K1996004 - purple colour mutant of amilCP
• BBa_K1996005 - mint green colour mutant of amilCP
Introduction
amilCP is a chromoprotein, derived from the coral Acropora millepora, that has a strong blue colour that is visible to the naked eye, with an absorbance maximum at 588nm.  The original part was developed by Team Uppsala Sweden 2011 ( BBa_K592009 ).
Motivations and use in biosensor
amilCP was chosen as the reporter gene for our biosensor as it is easily visible to the naked eye. This allows it to be easily observed by consumers in a fruit sticker version of our biosensor, or to be detected by scanners in a plate or handheld versions, without the need for extra equipment. In contrast, fluorescent proteins such as GFP would need an additional component to shine UV light before the output can be observed.
There were several reasons for our choice to improve amilCP:
• To allow for more sensitivity in the detection of the output – by creating a darker blue mutant
• To decrease response time – by creating a mutant that would turn blue faster
• To add to the spectrum of chromoproteins – by creating different coloured mutants
Error Prone PCR
To create our mutant amilCP, we used error prone PCR (EP-PCR). EP-PCR involves the addition of Mn2+, which increases the error rate of Taq DNA polymerase. This creates random mutations within the region defined by the primers, the number of which can be controlled by the number of doublings.
For shorter sequences (<100bp), chemical synthesis of the sequences is preferred. However, in our case, we were unable to use this for two reasons: firstly, amilCP, at 669bp, is too large to be chemically synthesised. Secondly, we were looking for random mutations rather than targeted mutations, as it was unknown what mutations would be needed to create different colour mutants.
A high throughput method is also needed to screen the large number of mutants produced by EP-PCR.  In our case, this involved scoring the colonies that had been transformed with the EP-PCR products based on colour. Mutants that appeared to be dark blue in colour, as well as other coloured mutants, such as purple and pink, were then selected for sequencing.
The concentration of Mn2+ plays a large role in the mutation rate. 0.1mM MnCl2 was the optimum concentration for the EP-PCR, as it resulted in roughly half the colonies remaining blue after being transformed, while the other half were white or other colours. It was found that the use of concentrations above 0.6mM MnCl2 resulted in a marked decrease in the amount of product (Figure 1). This was likely because at such concentrations the Mn2+ had decreased the fidelity of Taq to the point that it was unable to continue amplifying along the entire sequence.
Figure 1. Increasing concentrations of MnCl2 decreases the amount of amilCP product. Products were run on a 1.5% agarose gel in 1X TAE at 100V for 1h, and post stained in Gel Green for 1h.
                                                                       
Results: Number of Mutants
Following transformation, the colours of the colonies on the plates were scored and counted to assess the EP-PCR efficiency. Using 0.1mM of Mn2+, the majority of colonies (55%) were white, indicating that the amilCP coding region had experienced loss of function mutations. Subsequent sequencing indicated that these were most deletions resulting in frameshifts. A large proportion of colonies (34%) were scored as ‘blue’, indicating that they were of a similar shade to the wild type. A small number of mutants appeared to be darker blue or purple/pink (1.5% and 1.6% respectively).
Figure 2. Proportions of colonies of each colour following transformation with the mutated amilCP sequences.
Results: Common Mutants
Mutants of each colour were selected for sequencing. It was found that all coloured mutants (blue, purple, pink and green) only had single nucleotide substitutions (SNPs), whereas mutants that were white, and had thus lost amilCP function, tended to have deletions resulting in frameshift mutations. Several different SNPs were found to cause the same colour, with some common ones that result in purple, pink or green phenotypes summarised in Figure 3.
Some colonies appeared to have the same colour as the wild type, however upon sequencing, it was discovered that there were several mutations that changed the nucleotide sequence, but did not seem to alter the function of amilCP. Most of these were later found to have no effect on the amino acid sequence.
Figure 3. Common mutations of amilCP resulting in a change of colour.  The wild type amilCP amino acid sequence is shown, with the mutated residues in bold. The 3 mutants that were submitted as parts (BBa_K1996003-005) are indicated by asterisks.
We were unable to find an objectively darker blue mutant, however, we have submitted pink, purple and green mutants (Figure 4) (BBa_K1996003-005 respectively). The colour in the purple mutant, like the wild type, can be observed after incubation overnight, whereas the pink and green mutants take up to 2-3 days to express the colour.
It was observed that wild type amilCP tends to decrease in vibrancy following multiple restreaks or after storage at 4˚C, something that was also observed in the pink and green mutants. However, the purple mutant retained its colour both on plates, and when inoculated into liquid media.
Figure 4. Purple, pink and green mutants compared to the wild type.
Protein was extracted from 50mL of culture and resuspended in 0.5mL TE buffer.
Results: Spectrums
Protein was extracted and a spectrum scan from 300-750nm was performed. Maximum absorbance for wild type amilCP was at 589nm, whereas the absorbance maximum for the purple mutant was at 579nm (Figure 5), indicating that it had been shifted slightly to the left. The absorbance maximums for the pink and green mutants were also shifted as expected, to 576nm and 609nm respectively (Figure 5). When standardised for protein concentration, the peaks for the pink and green mutants were not as high as for blue or purple (Figure 5), indicating that the chromoprotein colour is not as strong.
Figure 5. Spectrum scan of the extracted protein from the different coloured amilCP mutants. Protein concentration is equal between all samples.
The spectrums were then adjusted mathematically to have the same peak absorbance of 1 (Figure 6). This revealed that the purple and pink mutants have spectrums similar to the wild type, however, the green mutant had a second peak at about 314nm (Figure 6). This is out of the visible spectrum, and likely just due to contamination of the sample by cell membranes. The main peak at 609nm accounts for the green colour, since 609nm is within the red part of the spectrum and so would lead to the sample being seen as green.
Figure 6. Spectrum scan of the extracted protein from the different coloured amilCP mutants, with the spectrums all adjusted to the same levels to show peaks.
School of Life and Environmental Sciences The University of Sydney City Road, Darlington 2006, New South Wales, Sydney, Australia
