<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Sydney_Australia_ProteinModel2 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Sydney Australia/ProteinModel2</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar navbar-fixed-top" style="margin-top:15px"><NAV class="navbar-inner"><DIV class="row-sm-height"><DIV class="col-md-10"><UL class="nav navbar-nav" style="margin-top:10px"><LI><A href="#" class="dropdown-toggle" data-toggle="dropdown">

                 Team</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Team">About Us </A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Attributions">Attributions </A></LI></UL><LI><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project </A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/ProjectHome">Project- Home</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Sense">Sense</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Express">Express</A></LI><LI class="dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Keep Fresh</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Design">Applied Design</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Entrepreneurship">Business Plan</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Interlab">Interlab</A></LI><LI><A href=" https://2016.igem.org/Team:Sydney_Australia/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Human_Practices" class="dropdown-toggle" data-toggle="dropdown">

                 Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Human_Practices">Human Practices - Summary</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Integrated_Practices">Integrated Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Engagement">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/TOK">Theory of Knowledge</A></LI><LI class="menu-item"><A href="https://2016.igem.org/Team:Sydney_Australia/Collaboration"><SPAN>Collaboration</SPAN></A></LI><LI><A href="#" class="dropdown-toggle" data-toggle="dropdown">

                 Modelling </A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Model">Modelling - Home</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/ProteinModel1">Protein Modelling  Etnr1</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/ProteinModel2">Protein Modelling  Etnr2</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/InternalCellular">Internal Cellular Model</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/MacroscopicDiffusion">Macroscopic Diffusion Model</A></LI><LI><A href="#" class="dropdown-toggle" data-toggle="dropdown">

                 Software </A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Software">Software - Home</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Primers">Primer Design App</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/SmartPhone">SmartPhone App </A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Achievements" class="dropdown-toggle" data-toggle="dropdown">

                Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Parts"> Parts</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Medals">Medals</A></LI></DIV></DIV></NAV></DIV><DIV id="page"><DIV class="boxAll"><H2> EtnR2: ProtParam – What does the primary structure tell us? </H2></DIV><P>  </P><DIV class="boxAll"><P>The <A href="http://web.expasy.org/protparam/">ExPaSy ProtParam tool</A> was used to derive basic information concerning EtnR2 based on its amino acid sequence. 
It was calculated that:       •Number of amino acids: 198       •Molecular weight: 21803.5       •Theoretical pI: 4.65 
It was also found that EtnR2 has an instability index of 21.05, classifying the protein as stable. </P><DIV class="boxAll"><H2>  EtnR2: BLAST analysis – Are there any known close homologues? </H2></DIV><DIV class="boxLeft"><P> The <A href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">NCBI BLAST tool</A> was used to conduct a protein BLAST (Basic Local Alignment Search Tool) to identify proteins with close sequence similarity to the EtnR2 sequence. The SmartBLAST feature selects ‘the three best matches in the sequence database together with the two best matches from well-studied reference species, showing phylogenetic relationships based on multiple sequence alignment and conserved protein domains.’
In the case of EtnR1, there were no matches from well-studied reference species. The top matches were all hypothetical Mycobacterium proteins. Figure 1 shows the phylogenetic tree constructed based on sequence similarities and conserved domains. The individual alignments can be found at the bottom of the page.

  
</P><P><B>Figure 1. Phylogenetic tree produced by SmartBLAST with query: hypothetical protein [Mycobacterium chubuense] (accession: WP_014805816.1, GI: 504618714).   </B></P></DIV><DIV class="boxRight"><P> Other candidates shown are hypothetical protein [Mycobacterium tusciae] (accession: WP_006247393.1, GI: 493289683), hypothetical protein [Mycobacterium sp. JS623] (accession: WP_015305843.1 GI: 505118741) and hypothetical protein [Mycobacterium rhodesiae] (accession: WP_005148477.1 GI: 491290461).
The BLAST analysis also revealed a predicted <A href="https://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?ascbin=8&amp;maxaln=10&amp;seltype=2&amp;uid=pfam14417">MEDS domain</A> (Figure 2). The MEDS domain has been characterised in in hydrocarbon sensing in Methylococcus and is generally fused to a histidine kinase. </P><P><B>Figure 2. Snapshot of BLAST results, showing putative HTH-30 domain at the C-terminal end of EtnR2. </B></P></DIV><DIV class="boxAll"><H2>  EtnR2: PsiPred – Secondary structure prediction </H2><P>The UCL Bioinformatics <A href="http://bioinf.cs.ucl.ac.uk/psipred/">PsiPred tool</A> was used to predict the secondary structure of EtnR2 based on its amino acid sequence (Figure 3). The putative helices and sheets were generally predicted with high confidence. </P><P style="text-align: center;"><B>  Figure 3. Secondary structure prediction using PsiPred for EtnR2 based on amino acid sequence. </B></P></DIV><DIV class="boxAll"><H2> EtnR2: SwissModel Homology Modelling</H2></DIV><DIV class="boxLeft"><P> The <A href="https://swissmodel.expasy.org/">SWISS-MODEL</A> server [1-4] is an automated service that selects the best matching template to a submitted amino acid sequence, performs the alignment and builds a 3D model. The most important limiting factor of the quality of the model is generally the matching of the query sequence with an appropriate template.
SwissProt threaded EtnR2 onto a selected template protein: <A href="http://www.rcsb.org/pdb/explore.do?structureId=4WIA ">flagellar accessory protein FlaH</A> from <I>Methanocaldococcus jannaschii</I>. It is a conserved component of the archaeal flagellum system. 
Based on the structure of the template, a homology model was constructed (Figure 4). The predicted structure consisted of a parallel beta sheet surrounded by alpha helices.


  </P></DIV><DIV class="boxRight"><P style="text-align: center"><B> Figure 4. Homology model of EtnR2 </B></P></DIV><DIV class="boxAll"><H2> EtnR2: BLAST alignments with three best matches and from well-studied reference species </H2></DIV><P> hypothetical protein [Mycobacterium rhodesiae] 
Sequence ID: WP_005148477.1 Length: 197 Number of Matches: 1 
Related Information 
Identical Proteins-Identical proteins to WP_005148477.1 
Range 1: 1 to 197 </P><DIV class="boxAll"><TABLE><TBODY><TR><TH>Score</TH><TH>Expect</TH><TH>Method</TH><TH>Identities</TH><TH>Positives</TH><TH>Gaps</TH></TR><TR><TD>359 bits(922)</TD><TD>9e-133()</TD><TD>Compositional matrix adjust.</TD><TD>179/198(90%)</TD><TD>185/198(93%)</TD><TD>1/198(0%)</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>Query</TH><TH>Sequence</TH></TR><TR><TD>Query  1 - 60   Sbjct  1 - 60</TD><TD>MRTSDTAAVGVMGVSLAAGDHVCAFYPTLAERNEILLPYLKEGLNHGDKCICIVSDDSAD 

 M T+D A VGVMGVSLAAGDHVCAFYPTLAERNEILLPYLKEGL  GDKCICIVSD      
MSTTDMATVGVMGVSLAAGDHVCAFYPTLAERNEILLPYLKEGLKLGDKCICIVSDSPDS
</TD></TR><TR><TD>Query 61-
120
  Sbjct  61-
120
</TD><TD>SDLDSVRCLSQHTGQLEVDHSGDTYLKGGNFSSDRMLKYWNSAISQAVDDGFDFARGAGE    DSV+CLSQ TGQLEVDHSGDTYLKGGNFSSDRMLKYWNS+ISQAVDDGF+FARGAGE DL-DSVKCLSQDTGQLEVDHSGDTYLKGGNFSSDRMLKYWNSSISQAVDDGFNFARGAGE   </TD></TR><TR><TD>Query  121-
180
           
Sbjct  121-
180
</TD><TD>MTWALEKMPGVENLVSYESQLNSVLRDYPAVIVCLYELGRFSGDLLVEVLKTHPKVILGG  MTWALEKMPGVENLVSYESQLNSVLR+YPAVIVCLYELGRFSGD+LVEVLKTHPKVILGG  MTWALEKMPGVENLVSYESQLNSVLREYPAVIVCLYELGRFSGDMLVEVLKTHPKVILGG</TD></TR><TR><TD>Query  181-
240
 Sbjct  181-
240
</TD><TD>IVLENPYYLEHNEYLTSR  IVLENPYYLEHNEYLTSR  IVLENPYYLEHNEYLTSR</TD></TR></TBODY></TABLE><P> hypothetical protein [Mycobacterium tusciae] 
Sequence ID: WP_006247393.1 Length: 199 Number of Matches: 1 
Related Information 
Range 1: 1 to 199 </P><TABLE><TBODY><TR><TH>Score</TH><TH>Expect</TH><TH>Method</TH><TH>Identities</TH><TH>Positives</TH><TH>Gaps</TH></TR><TR><TD>327 bits(838)</TD><TD>7e-120()</TD><TD>Compositional matrix adjust.</TD><TD>154/199(77%)</TD><TD>175/199(87%)</TD><TD>1/199(0%)</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>Query</TH><TH>Sequence</TH></TR><TR><TD>Query  1 - 59   Sbjct  1- 60</TD><TD>MRTSDTAAVGVMGVSLAAGDHVCAFYPTLAERNEILLPYLKEGLNHGDKCICIVSDDS-A  

MRT+D A VGVMGVSLA GDHVCAFYP  AERNEIL+PYL+EGL+ GDKCICIVSD +    
MRTTDAATVGVMGVSLAPGDHVCAFYPGAAERNEILIPYLQEGLDAGDKCICIVSDATEP
</TD></TR><TR><TD>Query  60-
119
  Sbjct 61-
120
</TD><TD>DSDLDSVRCLSQHTGQLEVDHSGDTYLKGGNFSSDRMLKYWNSAISQAVDDGFDFARGAGD DLDS+RC S H+ QL VD S +TYLKGG+FSSDRMLKYW+SAI QA++DG+ FARGAG  DGDLDSIRCYSHHSNQLAVDRSSETYLKGGSFSSDRMLKYWDSAIRQAIEDGYTFARGAG</TD></TR><TR><TD>Query  120-
179
           
Sbjct  121-
180
</TD><TD>EMTWALEKMPGVENLVSYESQLNSVLRDYPAVIVCLYELGRFSGDLLVEVLKTHPKVILGEMTWALE MPGVE+LV+YES+LN  L++YPAVIVCLYELGRFSG++LVEVLKTHPKVILG EMTWALEAMPGVEDLVTYESELNDFLKEYPAVIVCLYELGRFSGEMLVEVLKTHPKVILG   </TD></TR><TR><TD>Query  180-
198
 Sbjct 181-
199
</TD><TD>GIVLENPYYLEHNEYLTSR GIVLENPYYL+  EYL SR   GIVLENPYYLDQAEYLASR </TD></TR></TBODY></TABLE><P> hypothetical protein [Mycobacterium sp. JS623] 
Sequence ID: WP_015305843.1 Length: 199 Number of Matches: 1 
Related Information 
Identical Proteins-Identical proteins to WP_015305843.1 
Range 1: 1 to 199 </P><TABLE><TBODY><TR><TH>Score</TH><TH>Expect</TH><TH>Method</TH><TH>Identities</TH><TH>Positives</TH><TH>Gaps</TH></TR><TR><TD>325 bits(833)</TD><TD>4e-119()</TD><TD>Compositional matrix adjust.</TD><TD>156/199(78%)</TD><TD>176/199(88%)</TD><TD>1/199(0%)</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>Query</TH><TH>Sequence</TH></TR><TR><TD>Query  1 - 59   Sbjct  1- 60</TD><TD>MRTSDTAAVGVMGVSLAAGDHVCAFYPTLAERNEILLPYLKEGLNHGDKCICIVSDDSA-

M T+DTA VGVMGVSLA GDHVCAFYP  AERNEIL+PYL+EGL+ GDKCICIVSD +A  
MLTTDTATVGVMGVSLAPGDHVCAFYPGAAERNEILIPYLREGLDAGDKCICIVSDATAP
</TD></TR><TR><TD>Query  60-
119
  Sbjct 61-
120
</TD><TD>DSDLDSVRCLSQHTGQLEVDHSGDTYLKGGNFSSDRMLKYWNSAISQAVDDGFDFARGAGD +LDSVRC S ++ QL VD SGDTYLKGG FS+DRML+YW+SAISQA++DG+ FARGAG DGELDSVRCHSHNSDQLAVDRSGDTYLKGGGFSADRMLEYWDSAISQAMEDGYTFARGAG</TD></TR><TR><TD>Query  120-
179
           
Sbjct  121-
180
</TD><TD>EMTWALEKMPGVENLVSYESQLNSVLRDYPAVIVCLYELGRFSGDLLVEVLKTHPKVILGEMTWALE MPGVENLV+YE +LN  L+ YPAVIVCLYELGRFSG++LVEVLKTHPKVILG EMTWALETMPGVENLVTYEFELNDFLKKYPAVIVCLYELGRFSGEMLVEVLKTHPKVILG   </TD></TR><TR><TD>Query  180-
198
 Sbjct 181-
199
</TD><TD>GIVLENPYYLEHNEYLTSR  GIVLENPYYL+H EYL SRGIVLENPYYLDHAEYLASR </TD></TR></TBODY></TABLE><H2>
For reference: EtnR2 Amino Acid Sequence
</H2><P> MRTSDTAAVGVMGVSLAAGDHVCAFYPTLAERNEILLPYLKEGLNHGDKCICIVSDDSADSDLD
SVRCLSQHTGQLEVDHSGDTYLKGGNFSSDRMLKYWNSAISQAVDDGFDFARGAGEMTWALEKM
PGVENLVSYESQLNSVLRDYPAVIVCLYELGRFSGDLLVEVLKTHPKVILGGIVLENPYYLEHN
EYLTSR
</P><H2> References </H2><P> 
1.	Marco Biasini, Stefan Bienert, Andrew Waterhouse, Konstantin Arnold, Gabriel Studer, Tobias Schmidt, Florian Kiefer, Tiziano Gallo Cassarino, Martino Bertoni, Lorenza Bordoli, Torsten Schwede. (2014). SWISS-MODEL: modelling protein tertiary and quaternary structure using evolutionary information. Nucleic Acids Research; (1 July 2014) 42 (W1): W252-W258; doi: 10.1093/nar/gku340.  
2.	Arnold K., Bordoli L., Kopp J., and Schwede T. (2006). The SWISS-MODEL Workspace: A web-based environment for protein structure homology modelling. Bioinformatics, 22,195-201. 
3.	Kiefer F, Arnold K, Künzli M, Bordoli L, Schwede T (2009). The SWISS-MODEL Repository and associated resources. Nucleic Acids Research. 37, D387-D392. 
4.	Guex, N., Peitsch, M.C., Schwede, T. (2009). Automated comparative protein structure modeling with SWISS-MODEL and Swiss-PdbViewer: A historical perspective.Electrophoresis, 30(S1), S162-S173. 
</P></DIV></DIV><DIV id="mfooter"><DIV class="footbox" style="margin-left:5%;margin-top:1%;width: 20%; margin-right: 5%;"><B>School of Life and Environmental Sciences</B>
    The University of Sydney
    City Road, Darlington
    2006, New South Wales, Sydney, Australia</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>