<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Sydney_Australia_ProteinModel1 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Sydney Australia/ProteinModel1</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar navbar-fixed-top" style="margin-top:15px"><NAV class="navbar-inner"><DIV class="row-sm-height"><DIV class="col-md-10"><UL class="nav navbar-nav" style="margin-top:10px"><LI><A href="#" class="dropdown-toggle" data-toggle="dropdown">

                 Team</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Team">About Us </A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Attributions">Attributions </A></LI></UL><LI><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project </A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/ProjectHome">Project- Home</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Sense">Sense</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Express">Express</A></LI><LI class="dropdown-submenu"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Keep Fresh</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Design">Applied Design</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Entrepreneurship">Business Plan</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Interlab">Interlab</A></LI><LI><A href=" https://2016.igem.org/Team:Sydney_Australia/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Human_Practices" class="dropdown-toggle" data-toggle="dropdown">

                 Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Human_Practices">Human Practices - Summary</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Integrated_Practices">Integrated Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Engagement">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/TOK">Theory of Knowledge</A></LI><LI class="menu-item"><A href="https://2016.igem.org/Team:Sydney_Australia/Collaboration"><SPAN>Collaboration</SPAN></A></LI><LI><A href="#" class="dropdown-toggle" data-toggle="dropdown">

                 Modelling </A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Model">Modelling - Home</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/ProteinModel1">Protein Modelling  Etnr1</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/ProteinModel2">Protein Modelling  Etnr2</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/InternalCellular">Internal Cellular Model</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/MacroscopicDiffusion">Macroscopic Diffusion Model</A></LI><LI><A href="#" class="dropdown-toggle" data-toggle="dropdown">

                 Software </A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Software">Software - Home</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Primers">Primer Design App</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/SmartPhone">SmartPhone App </A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Achievements" class="dropdown-toggle" data-toggle="dropdown">

                Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Parts"> Parts</A></LI><LI><A href="https://2016.igem.org/Team:Sydney_Australia/Medals">Medals</A></LI></DIV></DIV></NAV></DIV><DIV id="page"><DIV class="boxAll"><H2> EtnR1: ProtParam – What does the primary structure tell us? </H2><P>  </P><DIV class="boxAll"><P>The <A href="http://web.expasy.org/protparam/">ExPaSy ProtParam tool</A> was used to derive basic information concerning EtnR1 based on its amino acid sequence. 
It was calculated that:       • Number of amino acids: 580      • Molecular weight: 63306.7      • Theoretical pI: 5.64
It was also found that EtnR1 has an instability index of 38.13, classifying the protein as stable.</P><DIV class="boxAll"><H2>  EtnR1: BLAST analysis – Are there any known close homologues? </H2></DIV><DIV class="boxLeft"><P><A href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">The NCBI BLAST tool</A> was used to conduct a protein BLAST (Basic Local Alignment Search Tool) to identify proteins with close sequence similarity to the EtnR1 sequence. The SmartBLAST feature selects ‘the three best matches in the sequence database together with the two best matches from well-studied reference species, showing phylogenetic relationships based on multiple sequence alignment and conserved protein domains.’ 
In the case of EtnR1, the top hits were Mycobacterium CdaR family transcriptional regulators, though EtnR1 also showed minimal relationship to the E. Coli. CdaR which has been more thoroughly characterised. Figure 1 shows the phylogenetic tree constructed based on sequence similarities and conserved domains. The individual alignments can be found at the bottom of the page.  &gt;    


 </P></DIV><DIV class="boxRight"><P> Other candidates shown are regulatory protein [Streptomyces coelicolor A3(2)] (accession: NP_629806.1 GI:21224027), carbohydrate diacid regulon transcriptional regulator; autoregulator [Escherichia coli str. K-12 substr. MG1655] (accession: NP_414704.4, GI: 90111093), CdaR family transcriptional regulator [Mycobacterium tusciae] (accession: WP_006247394.1, GI: 493289684), CdaR family transcriptional regulator [Mycobacterium sp. JS623] (accession: WP_015305844.1, GI: 505118742) and CdaR family transcriptional regulator [Mycobacterium rhodesiae] (accession: WP_005148475.1, GI: 491290459). 

The BLAST analysis also revealed a predicted <A href="https://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13556">PucR C-terminal helix-turn-helix domain (HTH-30)</A> at the C-terminal end (Figure 2). 
It is often found in PucR-like transcriptional regulators. This HTH-30 domain is likely to be where EtnR1 binds to DNA.
</P></DIV><DIV class="boxAll"><P><B> Figure 1. Phylogenetic tree produced by SmartBLAST with query: CdaR family transcriptional regulator [Mycobacterium chubuense] (accession: WP_014805817.1, GI:504618715). </B></P><DIV class="boxAll"><P><B> Figure 2. Snapshot of BLAST results, showing putative HTH-30 domain at the C-terminal end of EtnR1. </B></P><H2>  EtnR1: PsiPred – Secondary structure prediction </H2></DIV><DIV class="boxLeft"><P> The UCL Bioinformatics <A href="http://bioinf.cs.ucl.ac.uk/psipred/">PsiPred tool</A> was used to predict the secondary structure of EtnR1 based on its amino acid sequence (Figure 3).
Importantly, the four helices at the C-terminus show high confidence of prediction, and thus is very likely that the putative HTH-30 domain is present. Many of the other areas of secondary structure also show high levels of confidence.  </P></DIV><DIV class="boxRight"><P><B>  Figure 3. Secondary structure prediction using PsiPred for EtnR1 based on amino acid sequence. </B></P></DIV><DIV class="boxAll"><H2> EtnR1: SwissModel Homology Modelling</H2></DIV><DIV class="boxLeft"><P> The <A href="https://swissmodel.expasy.org/">SWISS-MODEL</A> server [1-4] is an automated service that selects the best matching template to a submitted amino acid sequence, performs the alignment and builds a 3D model. The most important limiting factor of the quality of the model is generally the matching of the query sequence with an appropriate template.
SwissProt threaded amino acids 1 to 313 of EtnR1 onto a selected template protein: <A href="http://www.rcsb.org/pdb/explore.do?structureId=3lfv">phosphodiesterase 5A GAF domain</A>. It is a regulatory domain of the human PDE5. Notably, GAF domains are commonly found at the N-terminus in CdaR family transcriptional regulators, along with the HTH-30 domain at the C-terminus.
Based on the structure of the template, a homology model was constructed for this region encompassing residues 1-313 (Figure 4).

  </P><P><B> Figure 4. Homology model of residues 1-313 of EtnR1 based on phosphodiesterase 5A GAF domain. </B></P></DIV><DIV class="boxRight"><P> Residues 313 to 580 were threaded onto a different template protein: <A href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3ONQ ">regulator of polyketide synthase expression</A> from <I>Bifidobacterium adolescentis</I>. This protein is a transcriptional regulator.
Based on the structure of the template, a homology model was constructed for this region encompassing residues 313-580 (Figure 5). The HTH-30 domain at the C-terminus can be visualised in this model.
</P><P><B> Figure 5. Homology model of residues 313 to 580 of EtnR1 based on regulator of polyketide synthase expression BAD_0249 from <I>Bifidobacterium adolescentis</I>. HTH-30 at C-terminus is shown in cyan. </B></P></DIV><P> CdaR family transcriptional regulator [Mycobacterium rhodesiae] 
Sequence ID: WP_005148475.1 Length: 580 Number of Matches: 1 
Related Information 
Identical Proteins-Identical proteins to WP_005148475.1 
Range 1: 1 to 580 </P><DIV class="boxAll"><H2> EtnR1: BLAST alignments with three best matches and from well-studied reference species </H2><P> CdaR family transcriptional regulator [Mycobacterium rhodesiae] 
Sequence ID: WP_005148475.1 Length: 580 Number of Matches: 1 
Related Information 
Identical Proteins-Identical proteins to WP_005148475.1 
Range 1: 1 to 580 </P><TABLE><TBODY><TR><TH>Score</TH><TH>Expect</TH><TH>Method</TH><TH>Identities</TH><TH>Positives</TH><TH>Gaps</TH></TR><TR><TD>1140 bits(2948)</TD><TD>0.0()</TD><TD>Compositional matrix adjust.</TD><TD>560/580(97%)</TD><TD>566/580(97%)</TD><TD>0/580(0%)</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>Query</TH><TH>Sequence</TH></TR><TR><TD>Query  1 - 60   Sbjct  1 - 60</TD><TD> MTATSDVAHTETLVELREQLSNLQGLLMLAMLMTQSSDENKIVQLSTTSLPAFYRCPFVG 

 MTATSDVAHTETLVELREQLSNLQGLLMLAMLMTQS DENKI+QLSTTSLPAFYRCPFVG  
MTATSDVAHTETLVELREQLSNLQGLLMLAMLMTQSGDENKIIQLSTTSLPAFYRCPFVG
</TD></TR><TR><TD>Query 61-
120
  Sbjct  61-
120
</TD><TD>IYLNDGGWQKHLGARVDYSAAAEVDAQIATLGPSGGDLALGRYARCVALPLRGLDAHIGF  IYL DGGWQK LGARVDYSA AE+DA IATLGPSGG L LGRYA CVALPLRGLDAHIGF  IYLTDGGWQKALGARVDYSATAELDASIATLGPSGGVLTLGRYAWCVALPLRGLDAHIGF </TD></TR><TR><TD>Query  121-
180
           
Sbjct  121-
180
</TD><TD>FIVASDEEPSVGEQFLLRVLVQQTGVALANARLHRKEQASTEALRDSNNALAESISALEN  FIVASDEEPSVGEQFLLRVLVQQTG ALANARLHRKEQASTEALRDSN ALAES+SALEN  FIVASDEEPSVGEQFLLRVLVQQTGAALANARLHRKEQASTEALRDSNIALAESVSALEN</TD></TR><TR><TD>Query  181-
240
 Sbjct  181-
240
</TD><TD>AARIHARLTEVAAKGNGEDGIATALHELTGLSVAIEDRFGNLRAWAGPDCPEPYPKDDAN  AARIHARLTEVAAKGNGE+GIA ALHELTGLSVAIEDRFGNLRAWAGPDCPEPYPKD+AN  AARIHARLTEVAAKGNGEEGIAIALHELTGLSVAIEDRFGNLRAWAGPDCPEPYPKDNAN</TD></TR><TR><TD>Query 241-
300
           
Sbjct  241-
300
</TD><TD>AREAMLQRCIRAGEPIRHAGRLSAVANPRVDIVGVVSLIDPDEGGGEQAKVALEHGTTILAREAMLQRCIRAGEPIRHAGRLSAVANPRVDIVGVVSLIDPDEGGGEQAKVALEHGTT+LAREAMLQRCIRAGEPIRHAGRLSAVANPRVDIVGVVSLIDPDEGGGEQAKVALEHGTTVL</TD></TR><TR><TD>Query  301-
360
 Sbjct  301-
360
</TD><TD>AMELARLRSLAEAELRLRRDLVEEVLLGTDDESALARAEALGHDLGTPHRVVIVESEGRCAMELARLRSLAEAELRLRRDLVEEVLLGTDDESALARAEALGHDLGTPHRVVIVESEGRCAMELARLRSLAEAELRLRRDLVEEVLLGTDDESALARAEALGHDLGTPHRVVIVESEGRC </TD></TR><TR><TD>Query 361-
420
Sbjct 361-
420
</TD><TD>ADMEKFFHGVRSAARHAHMGTLVVARSNTVVILSDADMNRDRFVSALVSQLGSDDCRVGV ADMEKFFHGVR AARHAHMGTLVVARSNTVVILSDADMNRDRFVSALVS LGSDDCRVGVADMEKFFHGVRRAARHAHMGTLVVARSNTVVILSDADMNRDRFVSALVSHLGSDDCRVGV</TD></TR><TR><TD>
Query  421-
480
Sbjct  421-
480
</TD><TD>GGWCDRPRHLPRSYREAQLALKMQRRVGSRDPAPVTFYDELGVYRILAEVENQDSIERFV GGWCDRPRHLPRSYREAQLALKMQRRVGSRDPA VTFYDELGVYRILAEVENQDSIERFVGGWCDRPRHLPRSYREAQLALKMQRRVGSRDPAAVTFYDELGVYRILAEVENQDSIERFV</TD></TR><TR><TD>Query  481-
540
Sbjct  481-
540
</TD><TD>RQWLGPLLDYDAAKQSQLVATLSAYLECGGHHDATTAAIFVHRSTLKYRLSRIRTLLGLDRQWLGPLLDYDAAKQSQLVATLSAYLECGGHHDATTAAIFVHRSTLKYRLSRIRTLLGLDRQWLGPLLDYDAAKQSQLVATLSAYLECGGHHDATTAAIFVHRSTLKYRLSRIRTLLGLD</TD></TR><TR><TD>Query  541 -580 Sbjct  541 - 580</TD><TD>VNDPDVRFNLQMATRAWKTLEHLGSGDGHMSHQVEIPPTL VNDPDVRFNLQMATRAWKTLEHLGSGDGHMSHQVEIPPTLVNDPDVRFNLQMATRAWKTLEHLGSGDGHMSHQVEIPPTL</TD></TR></TBODY></TABLE><P> CdaR family transcriptional regulator [Mycobacterium sp. JS623] 
Sequence ID: WP_015305844.1Length: 580Number of Matches: 1
Related Information
Identical Proteins-Identical proteins to WP_015305844.1
Range 1: 3 to 580</P><TABLE><TBODY><TR><TH>Score</TH><TH>Expect</TH><TH>Method</TH><TH>Identities</TH><TH>Positives</TH><TH>Gaps</TH></TR><TR><TD>856 bits(2212)</TD><TD>0.0()</TD><TD>Compositional matrix adjust.</TD><TD>424/579(73%)</TD><TD>493/579(85%)</TD><TD>1/579(0%)</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>Query</TH><TH>Sequence</TH></TR><TR><TD>Query  2 - 61   Sbjct  3 - 61</TD><TD> TATSDVAHTETLVELREQLSNLQGLLMLAMLMTQSSDENKIVQLSTTSLPAFYRCPFVGI 

 A  DV+ TE+ +ELREQ+SNLQGLLML++LMTQS DENKI+QLS TS+P+FYRCPFVGI  
VAQEDVS-TESYIELREQISNLQGLLMLSLLMTQSGDENKIIQLSITSVPSFYRCPFVGI  
</TD></TR><TR><TD>Query  62-
121
  Sbjct 62-
121
</TD><TD>YLNDGGWQKHLGARVDYSAAAEVDAQIATLGPSGGDLALGRYARCVALPLRGLDAHIGFF  +LND GWQ  L          E++ Q+  LGPSGG L   RY+ CVALPLRGL+AHIG+F  HLNDRGWQNPLDPHDVSLERIELEPQLKALGPSGGVLNFTRYSWCVALPLRGLEAHIGYF </TD></TR><TR><TD>Query  122-
181
           
Sbjct  122-
181
</TD><TD>IVASDEEPSVGEQFLLRVLVQQTGVALANARLHRKEQASTEALRDSNNALAESISALENA +VA++ EPS GEQFL+RVL QQTGVALANARLHRKEQASTEALR SN ALAES++ALE A VVAAEAEPSTGEQFLIRVLAQQTGVALANARLHRKEQASTEALRRSNVALAESVTALEYA  </TD></TR><TR><TD>Query  182-
241
 Sbjct 182-
241
</TD><TD>ARIHARLTEVAAKGNGEDGIATALHELTGLSVAIEDRFGNLRAWAGPDCPEPYPKDDANA  A IHAR TE+A KG GE+GIATALHELTGL VAIEDRFGNLRAWAGPDCP+PYPKDD     ANIHARFTEIATKGQGEEGIATALHELTGLPVAIEDRFGNLRAWAGPDCPDPYPKDDPAT</TD></TR><TR><TD>Query 242-
301
           
Sbjct 242-
301
</TD><TD>REAMLQRCIRAGEPIRHAGRLSAVANPRVDIVGVVSLIDPDEGGGEQAKVALEHGTTILAREAMLQRC++AGEPIR  GRLSAVANPRVDI+GV+SLIDP    G+QA+VALEHGTT+LAREAMLQRCVQAGEPIRQGGRLSAVANPRVDIMGVLSLIDPQAVAGDQAQVALEHGTTVLA</TD></TR><TR><TD>Query  302-
361
 Sbjct  302-
361
</TD><TD>MELARLRSLAEAELRLRRDLVEEVLLGTDDESALARAEALGHDLGTPHRVVIVESEGRCAMELARLRSLAEAELRLRRDLVEE+LLGT DESALARAEALGHDLG  HRV+IVE +GR AMELARLRSLAEAELRLRRDLVEELLLGTGDESALARAEALGHDLGRCHRVLIVEPQGRTA  </TD></TR><TR><TD>Query  362-
421
Sbjct 362-
421
</TD><TD>DMEKFFHGVRSAARHAHMGTLVVARSNTVVILSDADMNRDRFVSALVSQLGSDDCRVGVG DM+KFFH VR AAR+A +G+L+VAR++TVVILSDA+++R++F+SA+ + +G D+ RVGVGDMDKFFHAVRRAARNAQLGSLIVARASTVVILSDAEVDREKFLSAISTSVGDDNFRVGVG</TD></TR><TR><TD>
Query 422-
481
Sbjct 422-
481
</TD><TD>GWCDRPRHLPRSYREAQLALKMQRRVGSRDPAPVTFYDELGVYRILAEVENQDSIERFVR GWCDRP   PRSY EAQLALKMQRR G+   A V FYDELGVYRILAEVENQ SIE FVRGWCDRPEDFPRSYHEAQLALKMQRRSGTATAAGVIFYDELGVYRILAEVENQQSIESFVR</TD></TR><TR><TD>Query 482-
541
Sbjct  482-
541
</TD><TD>QWLGPLLDYDAAKQSQLVATLSAYLECGGHHDATTAAIFVHRSTLKYRLSRIRTLLGLDVQWLGPLLDYDAAK SQ+VATL+ YL+CGGH+D TTAA+++HRSTLKYRLSRIR LLG+D+QWLGPLLDYDAAKGSQMVATLAGYLQCGGHYDTTTAALYIHRSTLKYRLSRIRDLLGIDI</TD></TR><TR><TD>Query  542 -580 Sbjct  542 - 580</TD><TD>NDPDVRFNLQMATRAWKTLEHLGSGDGHMSHQVEIPPTL NDP+ RFNL++A RAW TLE L +G GHMS   E+ P++NDPEARFNLELAARAWGTLEELAAGQGHMSPPAEVDPSV</TD></TR></TBODY></TABLE><P> carbohydrate diacid regulon transcriptional regulator; autoregulator [Escherichia coli str. K-12 substr. MG1655] 
Sequence ID: NP_414704.4 
Related Information 
Gene-associated gene details 
Identical Proteins-Identical proteins to WP_000929443.1 
Range 1: 87 to 378 </P><TABLE><TBODY><TR><TH>Score</TH><TH>Expect</TH><TH>Method</TH><TH>Identities</TH><TH>Positives</TH><TH>Gaps</TH></TR><TR><TD>75.9 bits(185)</TD><TD>0.0()</TD><TD>Compositional matrix adjust.</TD><TD>85/305(28%)</TD><TD>143/305(46%)</TD><TD>35/305(11%)</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>Query</TH><TH>Sequence</TH></TR><TR><TD>Query  271 - 330   Sbjct  87 - 144</TD><TD> DIVGVVSLIDPDEGGGEQAKVALEHGTTILAMELARLRSLAEAELRLRRDLVEEVLLGTD 

 +IVGV+ L    E   +  ++     T  + +E +RL  L   + RLR +LV  ++   + 
EIVGVIGLTGEPENLRKYGELVCM--TAEMMLEQSRLMHLLAQDSRLREELVMNLIQAEE   
</TD></TR><TR><TD>Query 331-382

  Sbjct  145-
202
</TD><TD>DESALAR-AEALGHDLGTPHRVVIVESEG-------RCADMEKFFHGVRSAARHAHMGTL   +  AL   A+ LG DL  P  V IVE +          A++++  + + +  R+  +     NTPALTEWAQRLGIDLNQPRVVAIVEVDSGQLGVDSAMAELQQLQNALTTPERNNLVA--  </TD></TR><TR><TD>Query  383-
431
           
Sbjct 203-261
</TD><TD>VVARSNTVVILS--------DADMNRDRFVSALVSQL---GSDDCRVGVGGWCDRPRHLP   +V+ +  VV+          DA+ +R R V  L++++   G    RV +G +   P  +  IVSLTEMVVLKPALNSFGRWDAEDHRKR-VEQLITRMKEYGQLRFRVSLGNYFTGPGSIA   </TD></TR><TR><TD>Query 432 - 489
 Sbjct 262 - 313
</TD><TD>RSYREAQ--LALKMQRRVGSRDPAPVTFYDELGVYRILAEVENQDSIERFVRQWLGPLLD  RSYR A+  + +  QR   SR      FY +L +  +L  +          R    PL   RSYRTAKTTMVVGKQRMPESR----CYFYQDLMLPVLLDSLRGDWQANELAR----PLAR</TD></TR><TR><TD>Query 490 - 548

           
Sbjct 314-373
</TD><TD>YDAAKQSQLVA-TLSAYLECGGHHDATTAAIFVHRSTLKYRLSRIRTLLGLDVNDPDVRF        + L+  TL+A+        AT+ A+F+HR+TL+YRL+RI  L GLD+ + D R LKTMDNNGLLRRTLAAWFRHNVQPLATSKALFIHRNTLEYRLNRISELTGLDLGNFDDRL</TD></TR><TR><TD>Query 549-553
 Sbjct  374-378

</TD><TD>NLQMA L +ALLYVA  </TD></TR></TBODY></TABLE><P> regulatory protein [Streptomyces coelicolor A3(2)]  
Sequence ID: NP_629806.1 
Related Information 
Gene-associated gene details 
Identical Proteins-Identical proteins to WP_011030383.1 
Range 1: 430 to 548 </P><TABLE><TBODY><TR><TH>Score</TH><TH>Expect</TH><TH>Method</TH><TH>Identities</TH><TH>Positives</TH><TH>Gaps</TH></TR><TR><TD>71.2 bits(173)</TD><TD>1e-11()</TD><TD>Compositional matrix adjust.</TD><TD>43/124(35%)</TD><TD>69/124(55%)</TD><TD>5/124(4%)</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>Query</TH><TH>Sequence</TH></TR><TR><TD>Query  433 - 492   Sbjct  430 - 484</TD><TD>SYREAQLALKMQRRVGSRDPAPVTFYDELGVYRILAEVENQDSIERFVRQWLGPLLDYDA 

+Y++AQ AL + RR G         ++ +    +L  + + D++  F    L  L D+DA 
AYKQAQQALSVARRRGRV----CVEHEHVAAGSVLPLLAD-DAVRAFADGLLRALRDHDA  
</TD></TR><TR><TD>Query 393-552

  Sbjct  485-
544
</TD><TD>AKQSQLVATLSAYLECGGHHDATTAAIFVHRSTLKYRLSRIRTLLGLDVNDPDVRFNLQM     +  LVA++ A+L   G  DA  A + VHR TL+YR+ R+  +LG  ++DPDVR  L +  TGRGDLVASVRAWLSRHGQWDAAAADLGVHRHTLRYRMRRVEEILGRSLDDPDVRMELWL </TD></TR><TR><TD>Query  553-
556
           
Sbjct 545-548
</TD><TD>ATRA A +AALKA   </TD></TR></TBODY></TABLE><H2> For reference: EtnR1 Amino Acid Sequence </H2><P> MTATSDVAHTETLVELREQLSNLQGLLMLAMLMTQSSDENKIVQLSTTSLPAFYRCPFVGIYLN DGGWQKHLGARVDYSAAAEVDAQIATLGPSGGDLALGRYARCVALPLRGLDAHIGFFIVASDEE PSVGEQFLLRVLVQQTGVALANARLHRKEQASTEALRDSNNALAESISALENAARIHARLTEVA AKGNGEDGIATALHELTGLSVAIEDRFGNLRAWAGPDCPEPYPKDDANAREAMLQRCIRAGEPI RHAGRLSAVANPRVDIVGVVSLIDPDEGGGEQAKVALEHGTTILAMELARLRSLAEAELRLRRD LVEEVLLGTDDESALARAEALGHDLGTPHRVVIVESEGRCADMEKFFHGVRSAARHAHMGTLVV ARSNTVVILSDADMNRDRFVSALVSQLGSDDCRVGVGGWCDRPRHLPRSYREAQLALKMQRRVG SRDPAPVTFYDELGVYRILAEVENQDSIERFVRQWLGPLLDYDAAKQSQLVATLSAYLECGGHH DATTAAIFVHRSTLKYRLSRIRTLLGLDVNDPDVRFNLQMATRAWKTLEHLGSGDGHMSHQVEIPPTL</P><H2> References </H2><P> 
1.	Marco Biasini, Stefan Bienert, Andrew Waterhouse, Konstantin Arnold, Gabriel Studer, Tobias Schmidt, Florian Kiefer, Tiziano Gallo Cassarino, Martino Bertoni, Lorenza Bordoli, Torsten Schwede. (2014). SWISS-MODEL: modelling protein tertiary and quaternary structure using evolutionary information. Nucleic Acids Research; (1 July 2014) 42 (W1): W252-W258; doi: 10.1093/nar/gku340.  
2.	Arnold K., Bordoli L., Kopp J., and Schwede T. (2006). The SWISS-MODEL Workspace: A web-based environment for protein structure homology modelling. Bioinformatics, 22,195-201. 
3.	Kiefer F, Arnold K, Künzli M, Bordoli L, Schwede T (2009). The SWISS-MODEL Repository and associated resources. Nucleic Acids Research. 37, D387-D392. 
4.	Guex, N., Peitsch, M.C., Schwede, T. (2009). Automated comparative protein structure modeling with SWISS-MODEL and Swiss-PdbViewer: A historical perspective.Electrophoresis, 30(S1), S162-S173. 
</P></DIV></DIV></DIV></DIV></DIV><DIV id="mfooter"><DIV class="footbox" style="margin-left:5%;margin-top:1%;width: 20%; margin-right: 5%;"><B>School of Life and Environmental Sciences</B>
    The University of Sydney
    City Road, Darlington
    2006, New South Wales, Sydney, Australia</DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>