<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Edinburgh_UG_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Edinburgh UG/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#unique-navbar" aria-expanded="false" aria-controls="navbar" style="margin-top: 0.8em"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="unique-navbar"><UL class="nav navbar-nav navbar-right"><LI><A href="/Team:Edinburgh_UG">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team</A></LI><LI><A href="/Team:Edinburgh_UG/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_UG/Collaborations">Collaborations</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project</A></LI><LI><A href="/Team:Edinburgh_UG/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_UG/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_UG/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_UG/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_UG/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_UG/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_UG/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_UG/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_UG/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_UG/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_UG/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_UG/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_UG/HP/Gold_Integrated">Integrated and Gold</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_UG/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_UG/HP/Interdisciplinarity">Interdisciplinarity</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_UG/HP/Accessibility">Accessibility</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Awards</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_UG/Measurement">Measurement</A></LI><LI><A href="https://2017.igem.org/Team:Edinburgh_UG/Model">Model</A></LI></DIV></DIV><DIV style="text-align: center;"><DIV class="same-width" style="display: inline-block; text-align: left"><H1 style="font-size: 60px; text-align: center; margin-top: 0; padding-top: 0;"> Design </H1><P> In our project, we view a single recombinase and a pair of its target site as a functional unit for building
                dynamic switches. With this concept in mind, we designed more sophisticated composite parts such as the measurement
                devices, gene randomizer, logic gates, and pulse generator. </P><H2 class="header-subsection"> Measurement devices </H2><P>To quantify orthogonality and recombination efficiency, we have designed simple measurement devices to characterize
                our basic parts â€“ the five recombinases (Cre, Dre, VCre, SCre, Vika) and their target sites (LoxP, Rox, Vlox,
                Slox, Vox). They are a set of fifteen biobricks that have a target sites-flanked transcriptional terminator
                inserted between a constitutive promoter and a RFP:</P><P>Thus, when recombination occurred, the transcriptional terminator was, expressing the RFP. We propose to use
                this set of device and our protocol for standardized measurement of recombination progression using plate
                reader. In addition to this, the measurement devices also demonstrate the ability of recombinases to act
                as a molecular switch, as shown below:</P><FIGCAPTION style="margin: 0 auto; max-width: 850px; padding-top: 20px; font-size: 16px; text-align: justify;"> Plates each containing cells included to express recombinase and a measurement construct. These cells contain
                a measurement construct with orthogonal target sites and give little fluorescence</FIGCAPTION><FIGCAPTION style="margin: 0 auto; max-width: 850px; padding-top: 20px; font-size: 16px; text-align: justify;"> Plates each containing cells included to express recombinase and a measurement construct. Colonies 6, 9, and
                12 contain Slox-Slox, Vox-Vox, and Rox-Rox target sites respectively. These produce brilliant fluorescence,
                effectively demonstrating the recombinases' ability to function as a genetic switch.</FIGCAPTION><H2 class="header-subsection"> Pulse generator </H2><P>Aware of the potential for SSR to mediate genetic engineering in a highly dynamic way, we have designed a novel
                construct that once initiated, will express multiple genes in separated pulses of transcription and translation.</P><FIGURE style="text-align: center"><FIGCAPTION style="margin: 0 auto; max-width: 600px; padding-top: 20px; font-size: 16px; text-align: justify;"> Pulse generator steps 1 and 2. Step 1: Induction of the expression of SSR1 leads to the excision of the terminator
                    immediately downstream of the promoter. Step 2: Excision of terminator allows the expression of SSR2,
                    which in turn excises the terminator in the bottommost construct.</FIGCAPTION></FIGURE><FIGURE style="text-align: center"><FIGCAPTION style="margin: 0 auto; max-width: 600px; padding-top: 20px; font-size: 16px; text-align: justify;"> Pulse generator steps 3 and 4. Step 3: SSR3 is expressed after SSR2 excised the terminator. SSR3 perform
                    excision, stopping the expression of SSR2. Step 4: Excision of the SSR2 construct allows SSR4 to be expressed,
                    which will in turn delete the SSR3 construct.</FIGCAPTION></FIGURE><FIGURE style="text-align: center"><FIGCAPTION style="margin: 0 auto; max-width: 600px; padding-top: 20px; font-size: 16px; text-align: justify;"> Pulse generator steps 5 and 6. Step 5: Deletion of the SSR3 construct by SSR4 leads to the expression of
                    SSR5. SSR5 then deletes the SSR4 construct. Step 6: In the end, only SSR5 remained.</FIGCAPTION></FIGURE><P>The functionality of the pulse generator has been simulated in the <A href="https://2017.igem.org/Team:Edinburgh_UG/Model"> model page </A>,
                where we have performed stochastic modeling and used it to predict the behavior of the pulse generator in
                different conditions. The pulse generator may find applications in metabolic engineering, where one can temporally
                separate the expression of several genes, thus lowering the metabolic stress on a cell [1].</P><H2 class="header-subsection"> Gene randomizer </H2><P>Gene randomizer utilizes the ability of site-specific recombinases to perform dynamic DNA inversion, and has
                been utilized by researchers to develop the brainbow system [2] to label individual cells:</P><P><A target="_blank" href="https://static.igem.org/mediawiki/2017/5/57/T--Edinburgh_UG--CFP_expression.mp4"> To see how CFP is expressed, click here!</A></P><P><A target="_blank" href="https://static.igem.org/mediawiki/2017/c/cb/T--Edinburgh_UG--RFP_deletion.mp4"> Click to see the deletion of RFP! </A></P><P><A target="_blank" href="https://static.igem.org/mediawiki/2017/6/6a/T--Edinburgh_UG--YFP_Expression.mp4"> To see how YFP is expressed, click here!</A></P><P>
                Using biobricks from our toolkit, they can be assembled using standard biobrick assembly protocols. The SSR systems in our
                toolkit are also characterized in terms of orthogonality so that it is possible to have multiple gene randomizers
                working parallel within the same cell:
            </P><P>
                We envision that such a design could be used for generating a heterogeneous population that can each cell carries a distinct
                combination of gene expression, and can be screened with single cell technology.
            </P><H2 class="header-subsection"> Logic gates </H2><P>
                Biological logic gates can be very useful to create sophisticated genetic circuits [3]. We have used tyrosine recombinases
                in our toolkit to design six logic gates (OR, AND, NOR, NAND, XOR, XNOR) using only excision mechanism:
            </P><FIGURE style="text-align: center;"><FIGCAPTION style="font-size: 20px">AND gate</FIGCAPTION></FIGURE><FIGURE style="text-align: center;"><FIGCAPTION style="font-size: 20px">OR gate</FIGCAPTION></FIGURE><FIGURE style="text-align: center;"><FIGCAPTION style="font-size: 20px">NAND gate</FIGCAPTION></FIGURE><FIGURE style="text-align: center;"><FIGCAPTION style="font-size: 20px">NOR gate</FIGCAPTION></FIGURE><FIGURE style="text-align: center;"><FIGCAPTION style="font-size: 20px">XOR gate</FIGCAPTION></FIGURE><FIGURE style="text-align: center;"><FIGCAPTION style="font-size: 20px">XNOR gate</FIGCAPTION></FIGURE><P>
                This kind of construct is possible only when the recombinases and their target sites are well defined to be non-interfering
                (i.e. orthogonal), so that they can be designed according to engineering principle, building from simple
                biobricks.
            </P><H2 class="header-subsection"> References </H2><P> [1] Singha, T.K., Gulati, P., Mohanty, A., Khasa, Y.P., Kapoor, R.K., Kmuar, S. (2017) Efficient genetic approaches
                for improvement of plasmid based expression of recombinant protein in Escherichia coli: A review. Process
                biochemistry. 55: 17-31
            </P><P>
                [2] Livet, J., Weissman, T.A., Kang, H., Draft, R.W., Lu, J., Bennis, R.A., Sanes, J.R., Lichtman, J.W. (2007) Transgenic
                strategies for combinatorial expression of fluorescent proteins in the nervous system. Nature. 450: 56-62
            </P><P>
                [3] Weinberg, B.H., Hang Pham, N.T., Caraballo, L.D., Lozanoski, T., Engel, A., Bhatia, S., Wong, W.W. (2017) Large-scale
                design of robust genetic circuits with multiple inputs and outputs for mammalian cells. Nature biotechnology.
                35(5): 453-462
            </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>