Team:Edinburgh UG/HP/Accessibility
Toggle navigation
Home
Team
Team
Collaborations
Project
Description
Design
Methods
Notebook
InterLab
Model
Results
Demonstrate
Attributions
Parts
Safety
Human Practices
Silver HP
Integrated and Gold
Public Engagement
Interdisciplinarity
Accessibility
Awards
Measurement
Model
Accessibility
Introduction
We believe SMORE should be accessible to every researcher who needs it and everyone who likes to learn about                 it. An accessibility regardless of academic background is important to innovation through interdisciplinary                 research, a notion we explored in the Interdisciplinarity page.
Nonetheless, an exchange of skillset and knowledge can be difficult. Technical language forms communication barriers.                 A steep learning curve hinders the gain of new skills. How can we make SMORE an accessible platform that                 promotes interdisciplinarity?
We improved the accessibility of SMORE in five aspects: readability, teaching the principle of recombination, hardware, user experience and data. These help                 our project to become easier to understand to use, as a step to diversify the field of synthetic biology.
Readability
To provide everyone with the opportunity to understand SMORE, our first step was to increase the readability                 of our wiki.
Introduction is where most readers start from. And technical details in later paragraphs are often reserved for                 the experienced and the interested. Therefore, we decided to write introductory paragraphs on main pages in a highly                 readable language, as they are targeted to a wide audience.
The readability of the introductions was carefully set to a 12th Grade student level. This is because the iGEM                 community includes high-school students. And thus, synthetic biology topics should be available and understood                 by a high-school level audience.
To obtain an estimate on readability, we chose the Dale-Chall formula over other readability indices [1]. Unlike                 other indices, result of the Dale-Chall formula does not depend on the number of syllables. Many scientific                 terms with a high syllable count have no short and accurate substitutes. For example, “significance” has many                 syllables, but it is irreplaceable in scientific statistics. Moreover, our skill exchange survey showed that                 technical language, when used moderately and precisely, also helps understanding. Dale-Chall formula does                 not weigh heavily on syllables, and directly measures the difficulty of words. This encourages us to explain                 the difficult words, rather than to use inaccurate substitutes.
Dale-Chall formula: $0.1579*(\frac{difficult\;words}{words})*100+0.0496*(\frac{words}{sentences})$
The Dale-Chall formula includes a collection of non-difficult words. We used the updated word collection to calculate                 the readability, using an online calculator [2-3]. The score of the index can be converted to grade level                 (Table 1). Familiar jargon, i.e. SMORE, iGEM, DNA and other repeating names, are excluded, given they have                 been thoroughly introduced. A simple word in the collection is used to replace them in calculation.
Below is an example of readability change in our project description. We have referred to simple guides on writing                 readable sentences [4-5], but we also developed some tricks through the process. The change was subtle and                 perhaps hardly noticeable. But it greatly benefits the reader in a long passage, as proven by our own reading                 experience.
Original text (readability = college graduate):
“Although recombinases 1 have been widely adopted in developmental biology to study vital genes 2, 3 their potential in genetic engineering to produce dynamic constructs 2 has yet to be fully exploited.”
1: “Recombinase” is a jargon in biology. Most readers are unlikely to understand.
2: This detail is irrelevant for non-biologist readers.
3: Unnecessary use of a complex sentence further decreases readability.
High readability text (readability = 11th or 12th grade):
“For years, researchers have used recombinase proteins 1 to genetically modify organisms 2 . However 3 we noticed 4 the potential of these proteins                 in genetic engineering is not fully exploited.”
1: “Recombinase proteins” uses the least amount the word to explain that “recombinase” is a protein, a substance                 relatable to most readers.
2: Details at a right level to inform the readers about the context, i.e. scientists use them to change organisms.
3: Breaking the long sentence apart.
4: The use of “we” makes the sentence more engaging. Active voice makes the sentence easy to read. Also, the                 use of short words increases readability. This is especially important in scientific writing, where long                 words are common.
Table 1: Conversion table from Dale-Chall score to reading skill level. Note the addition of a value of 3.6365 is required to get the adjusted score, for difficult word percentage higher than 5%.
Score
Reading skill needed for easy first-time understanding
4.9 or lower
4th grade student or lower
5.0 – 5.9
5th or 6th grade student
6.0 – 6.9
7th or 8th grade student
7.0 – 7.9
9th or 10th grade student
8.0 – 8.9
11th or 12th grade student
9.0 – 9.9
College student
10 or above
College graduate
Explaining the Principle of Recombination
Even for an experienced biologist, it can be very difficult to imagine how recombinases work in three dimensions. In fact, we are often told by our lecturers in genetics that teaching genetic recombination is one of the most difficult things to do. This is because just looking at static PowerPoint slides is usually not sufficient for students to understand the dynamic, three-dimensional activity of recombination.
Given how difficult it is for even biologists to explain recombination, we thought it was essential for increasing the accessibility of our project to provide tools so that anyone interested could understand the mechanism of SSR. Indeed, we often found that a large part of communicating our project to non-biologists required a lengthy introduction of the concept of SSR.
Therefore, we have produced unique animations that demonstrate how our randomizer construct works. Furthermore, we have created a longer video where recombination is described step-by-step using three-dimensional Claymation. These resources will be useful for those just being introduced to the concept of recombination and more experienced biologists alike.
Click here to watch our three-dimensional explanation of how SSR works, using Cre/LoxP as an example!
Hardware
SMORE should be accessible to every researcher who likes to use it. One of the barrier is the use of cell sorters.                 Cell sorters are integral to our randomiser strategy. If a series of randomiser constructs were able to produce a hetergenous cell population, this heterogenous population would need to be screened and sorted in some way. The most straightforward method to do this would be to use a cell sorter. However, cell sorters are expensive and might be a deterrent                 to some researchers. For example, second-hand flow cytometers costs approximately 2,000 – 3,000 USD.
We hope to produce an affordable and accessible cell sorter. Our ideal cell sorter possesses two qualities: 1)                 can be manufactured by small scale laboratories without extensive machinery; 2) can be produced at a low                 cost.
This is a challenging goal, as we need to account for both precision and the cost of the cell sorter. Here is                 our story of our endeavour:
To achieve quality 1, we utilised two technologies: 3D printing and microfluidics. 3D printing is a popular                 technology. Objects of any shape can be “printed” by 3D printing. Using 3D printing increases accessibility,                 because the parts do not have to be produced through manufacturers, which may not serve single users. Instead                 the users can produce the parts. On the other hand, we also looked to microfluidics. Microfluidic technology                 can be viewed as the use of small amounts of liquid to control precise movements [6]. The production of microfluidic                 device (fabrication) is increasingly commonplace. For example, fabrication using a desktop cutter has been                 reported as early as 2009 [7]. This is an indication that microfluidics may become increasingly accessible                 in the near future. The use of 3D printing and microfluidic device also addresses quality 2. As these technologies                 become more commonplace, we expect them to be less expensive.
Our cell sorter is composed of two components: the syringe pump and the microfluidic chip. The syringe pumps                 control the liquid pressure in the microfluidic chip, where the movement of cells are controlled.
We have successfully assembled our syringe pumps using 3D printed parts and simple electronic components, with                 a reference to the pneumatic control syringe pump on the open-source repository Metafluidics [8]. However, we realized this pump was not as accessible as we thought it would be, due to the lack of a readily available software to control it. Therefore, in collaboration with Team Glasgow, we devised a software to control the syringe pump. We have also written an easy-to-follow protocol to aid the user, further improving the accessibility of SMORE.
With the help from the Scottish Microelectronics Centre, we have successfully decided on the design of a microfluidic                 cell sorter, which was confirmed to be viable by experts. We based our device on the designs in pre-existing studies on droplet-based                 microfluidic cell sorters and referred to their calculations [6, 9-10]. The cell sorter is coupled to a droplet                 former, which forms aqueous droplets that host individual cells. The cell-containing droplets are then sorted                 based on fluorescent or colorimetric signals, that are coupled to the metabolic pathway.
Despite the verified design, the fabrication process was error prone and too demanding in skill. Regrettably,                 the channels of the device were too shallow for our final test with living bacterial cells. However, the                 process was thought provoking and further persuaded us a democratisation of microfluidic technology will                 benefit the synthetic biology community.
Click here to see the design of the microfluidic device.
Click here to see our program for the syringe pump.
Click here to download code required to run our program!
User Experience
Using SMORE involves the synthesis of target sites, which is an arduous task even for the experienced.
This is because the small length and repetitive sequences make direct chemical synthesis of target sites unfeasible.                 An alternative is the assembly of target sites from oligonucleotides (oligos). However, the design of oligos                 presents several challenges:
The oligos cannot be palindromic.
The oligos must bind to each other in a specific order.
The oligos must be within a reasonable length for synthesis.
The learning curve of oligo design is steep. SMORE needs an easy way to design oligos, to become accessible and                 attractive to more people.
This is why we have devised an automatic oligo designer program. This program requires basic computer skills                 for installation and use, but it allows researcher with no experience to design oligos. This program is unique                 to other primer designers, as it is optimized for recombinase target site assembly.
From our Interdisciplinary study, we understood the importance of a well-defined protocol. This is why we have                 provided a step-by-step user manual, with an additional glossary. The glossary is not intended to teach the                 user from scratch. Instead, it is designed to provide a common terminology that readers can communicate with.                 It is also a useful reminder for those unfamiliar with molecular biology. The font was also set to Times                 News Roman, following the tradition of most scientific literature, and allowing a high readability in both                 print and webpage [11].
Sounds interesting? Click here to see our program. and to download the relevant code we have created Click here.
Data
To establish recombination strategies as a robust and reliable technology, characterisation and definitive information                 of the enzymes are important. In particular, experimental verification of orthogonality between recombinases                 is crucial when using multiple recombinases in parallel. However, sequence and experimental data were scattered                 between online sources, hindering research effort. SMORE compiles and share the information of recombinases                 to promote the use of recombinases.
We have encountered the following issues when researching the four recombinases: Dre, Scre, Vcre and Vika (Table                 2). For Dre and Vika, sequence discrepancies exist between sources and experimental verification was limited.                 For Scre and Vcre, we realized they were not commercially available and the information was buried in supplementary                 material that has limited access. Moreover, the online sources were different from our sequences.
The study of orthogonality of recombinases by Weinberg et al. [12] was a pleasant surprise. However, the study was                 limited in mammalian cells. Orthogonality varies between chassis and a comprehensive study is yet to be seen                 in E. coli.
By providing our definitive sequences in Biobricks supported by experimental data, SMORE improves the accessibility                 of a recombinase strategy.
Please proceed to our Parts page for more information.
Table 2: information and availability of four recombinases: Dre, Scre, Vcre and Vika.
Original Discovery
Sequence sources [chassis]
Availability
Problem
Dre
Anastassiadis et al. (2009) [13]
Original paper [E. coli / mammalian]
Addgene Plasmid #51275
Addgene
Sequence discrepancy;
Addgene plasmid has only been used in mammalian cells, without extensive characterisation
Scre
Suzuki and Nakayama (2011) [14]
Original paper[E. coli]
NCBI [E. coli]
None
No direct availability
Vcre
Suzuki and Nakayama (2011) [14]
Original paper[E. coli]
NCBI [E. coli]
None
No direct availability
Vika
Karimova et al. (2013) [15]
Original paper [E. coli / mammalian]
iGEM BBa_K1641011 [unknown]
Addgene Plasmid #79969
iGEM,
Addgene
Sequence discrepancy between sources;
Biobrick has no experimental verification
References
[1] Dale, E. and Chall, J.S. 1948. A Formula for Predicting Readability: Instructions. Source: Educational Research                 Bulletin 27(2):37–54.
[2] Chall, J.S. and Dale, E. 1995. Readability revisited: the new Dale-Chall readability formula. U.S.: Brookline                 Books.
[3]  Readability Formula. Free Dale-Chall Readability Formula with Word List - Original and Revised versions. [Accessed                 26 October 2017]. Available from: http://www.readabilityformulas.com/free-dale-chall-test.php.
[4]  Aldrige, M.D. 2004. Writing and Designing Readable Patient Education Materials. Nephrology Nursing Journal                 31(4):373–377.
[5]  Manning, D. 1981. Writing readable health messages. Public Health Reports 96(5):464–465.
[6] Ferry, M.S., Razinkov, I.A. and Hasty, J. 2010. Microfluidics for synthetic biology: from design to execution.
[7] Yuen, P.K. and Goral, V.N. 2010. Low-cost rapid prototyping of flexible microfluidic devices using a desktop                 digital craft cutter. Lab on a Chip 10(3):384–387.
[8] 3D Printed Syringe Pump for Pneumatic Control. Metafluidics. [Accessed on 17th October, 2017] Available from                 https://metafluidics.org/devices/3d-printed-syringe-pump-for-pneumatic-control/
[9] Mazutis, L., Gilbert, J., Ung, W.L., Weitz, D.A., Griffiths, A. and Heyman, J.A. 2014. Single-cell analysis                 and sorting using droplet-based microfluidics. Nat Protoc. 8(5):870–891.
[10] Sia, S.K. and Whitesides, G.M. 2003. Microfluidic devices fabricated in poly(dimethylsiloxane) for biological                 studies. Electrophoresis 24(21):3563–3576.
[11] Mohamad Ali, A.Z., Wahid, R., Samsudin, K. and Zaffwan Idris, M. 2013. Reading on the computer screen: Does                 font type has effects on Web text readability? International Education Studies 6(3):26–35.
[12] Weinberg, B.H., Pham, N.T.H., Caraballo, L.D., Lozanoski, T., Engel, A., Bhatia, S. and Wong, W.W. 2017. Large-scale design of robust genetic circuits with multiple inputs and outputs for mammalian cells. Nature Biotechnology 35(5):435–462.
[13] Anastassiadis, K., Fu, J., Patsch, C., Hu, S., Weidlich, S., Duerschke, K., Buchholz, F., Edenhofer, F. and Steward, A.F. 2009. Dre recombinase, like Cre, is a highly efficient site-specific recombinase in E. coli, mammalian cells and mice. Disease Models and Mechanisms 2:508–515.
[14] Suzuki, E. and Nakayama, M. 2011. VCre/VloxP and SCre/SloxP: new site-specific recombination systems for genome engineering. Nucleic Acid Research 39(8):e49.
[15] Karimova, M., Abi-Ghanem, J., Berger, N., Surendranath, V., Pisabarro, M.T. and Buchholz, F. 2013. Vika/vox, a novel efficient and specific Cre/loxP-like site-specific recombination system. Nucleic Acid Research 41(2):e37.
