Team:Edinburgh UG/Design
Toggle navigation
Home
Team
Team
Collaborations
Project
Description
Design
Methods
Notebook
InterLab
Model
Results
Demonstrate
Attributions
Parts
Safety
Human Practices
Silver HP
Integrated and Gold
Public Engagement
Interdisciplinarity
Accessibility
Awards
Measurement
Model
Design
In our project, we view a single recombinase and a pair of its target site as a functional unit for building                 dynamic switches. With this concept in mind, we designed more sophisticated composite parts such as the measurement                 devices, gene randomizer, logic gates, and pulse generator.
Measurement devices
To quantify orthogonality and recombination efficiency, we have designed simple measurement devices to characterize                 our basic parts â€“ the five recombinases (Cre, Dre, VCre, SCre, Vika) and their target sites (LoxP, Rox, Vlox,                 Slox, Vox). They are a set of fifteen biobricks that have a target sites-flanked transcriptional terminator                 inserted between a constitutive promoter and a RFP:
Thus, when recombination occurred, the transcriptional terminator was, expressing the RFP. We propose to use                 this set of device and our protocol for standardized measurement of recombination progression using plate                 reader. In addition to this, the measurement devices also demonstrate the ability of recombinases to act                 as a molecular switch, as shown below:
Plates each containing cells included to express recombinase and a measurement construct. These cells contain                 a measurement construct with orthogonal target sites and give little fluorescence
Plates each containing cells included to express recombinase and a measurement construct. Colonies 6, 9, and                 12 contain Slox-Slox, Vox-Vox, and Rox-Rox target sites respectively. These produce brilliant fluorescence,                 effectively demonstrating the recombinases' ability to function as a genetic switch.
Pulse generator
Aware of the potential for SSR to mediate genetic engineering in a highly dynamic way, we have designed a novel                 construct that once initiated, will express multiple genes in separated pulses of transcription and translation.
Pulse generator steps 1 and 2. Step 1: Induction of the expression of SSR1 leads to the excision of the terminator                     immediately downstream of the promoter. Step 2: Excision of terminator allows the expression of SSR2,                     which in turn excises the terminator in the bottommost construct.
Pulse generator steps 3 and 4. Step 3: SSR3 is expressed after SSR2 excised the terminator. SSR3 perform                     excision, stopping the expression of SSR2. Step 4: Excision of the SSR2 construct allows SSR4 to be expressed,                     which will in turn delete the SSR3 construct.
Pulse generator steps 5 and 6. Step 5: Deletion of the SSR3 construct by SSR4 leads to the expression of                     SSR5. SSR5 then deletes the SSR4 construct. Step 6: In the end, only SSR5 remained.
The functionality of the pulse generator has been simulated in the model page ,                 where we have performed stochastic modeling and used it to predict the behavior of the pulse generator in                 different conditions. The pulse generator may find applications in metabolic engineering, where one can temporally                 separate the expression of several genes, thus lowering the metabolic stress on a cell [1].
Gene randomizer
Gene randomizer utilizes the ability of site-specific recombinases to perform dynamic DNA inversion, and has                 been utilized by researchers to develop the brainbow system [2] to label individual cells:
To see how CFP is expressed, click here!
Click to see the deletion of RFP!
To see how YFP is expressed, click here!
Using biobricks from our toolkit, they can be assembled using standard biobrick assembly protocols. The SSR systems in our                 toolkit are also characterized in terms of orthogonality so that it is possible to have multiple gene randomizers                 working parallel within the same cell:
We envision that such a design could be used for generating a heterogeneous population that can each cell carries a distinct                 combination of gene expression, and can be screened with single cell technology.
Logic gates
Biological logic gates can be very useful to create sophisticated genetic circuits [3]. We have used tyrosine recombinases                 in our toolkit to design six logic gates (OR, AND, NOR, NAND, XOR, XNOR) using only excision mechanism:
AND gate
OR gate
NAND gate
NOR gate
XOR gate
XNOR gate
This kind of construct is possible only when the recombinases and their target sites are well defined to be non-interfering                 (i.e. orthogonal), so that they can be designed according to engineering principle, building from simple                 biobricks.
References
[1] Singha, T.K., Gulati, P., Mohanty, A., Khasa, Y.P., Kapoor, R.K., Kmuar, S. (2017) Efficient genetic approaches                 for improvement of plasmid based expression of recombinant protein in Escherichia coli: A review. Process                 biochemistry. 55: 17-31
[2] Livet, J., Weissman, T.A., Kang, H., Draft, R.W., Lu, J., Bennis, R.A., Sanes, J.R., Lichtman, J.W. (2007) Transgenic                 strategies for combinatorial expression of fluorescent proteins in the nervous system. Nature. 450: 56-62
[3] Weinberg, B.H., Hang Pham, N.T., Caraballo, L.D., Lozanoski, T., Engel, A., Bhatia, S., Wong, W.W. (2017) Large-scale                 design of robust genetic circuits with multiple inputs and outputs for mammalian cells. Nature biotechnology.                 35(5): 453-462
