Team:Edinburgh UG/Results
Toggle navigation
Home
Team
Team
Collaborations
Project
Description
Design
Methods
Notebook
InterLab
Model
Results
Demonstrate
Attributions
Parts
Safety
Human Practices
Silver HP
Integrated and Gold
Public Engagement
Interdisciplinarity
Accessibility
Awards
Measurement
Model
Results
Overview
We cloned Dre, VCre, SCre, and Vika generators into Biobrick format, removing all illegal sites when necessary.
We assembled T7-LacO-Cre generator and cloned it into Biobrick format.
We successfully assembled 12 out of 15 of measurement constructs to allow users to quantify recombinase activity                     in vivo.
We assembled 10 target sites for Cre and proved their functionality in vitro. We also assembled the target                     sites for Dre, VCre, SCre, and Vika – Rox, VLox, SLox, and Vox respectively. Their functionality is proved                     by the measurement constructs.
We have extensively quantified the recombination efficiency of the five recombinases in E. coli.
We have built software and used it to design assembly methods for six logic gates using tyrosine recombinase.
We have built deterministic and stochastic modeling to simulate the behavior of site-specific recombinase.                     We also devised an algorithm to detect potential recombination sites in a genome.
We have conducted an investigation into interdisciplinarity. This includes a survey to identify challenges                     in interdisciplinary work, and a systematic analysis of past iGEM teams, to test correlation between                     interdisciplinarity and iGEM achievement.
We have integrated the result from the interdisciplinarity study to improve accessibility of SMORE in four                     aspects: readability, hardware, user experience and data.
For the full story of what we did in the wet lab, and extensive documentation on our workflows and results, check out our Notebook page by clicking here!
Cloning Dre, VCre, SCre, Vika
Using PCR mutagenesis, we have successfully removed the illegal XbaI site from all four recombinases.
We then cloned the T7-LacO-regulated recombinases into biobrick format. For SCre, there are two illegal PstI                 sites within the coding sequence (CDS), and for VCre, there is one illegal PstI site within the CDS. We have                 successfully removed all the illegal sites.
We submitted all four recombinases as T7-LacO-regulated generator, in pSB1C3 and contain no illegal sites ( BBa_K2406081 , BBa_K2406082 , BBa_K2406083 , BBa_K2406084 ).
Restriction enzyme digestion followed by gel electrophoresis proved that our SCre and VCre generators have their illegal sites removed. SCre and VCre are the original construct; SCremut1 is after removing the first illegal PstI site, and SCremut2 are after removing the second illegal PstI site. Similarly, VCremut1a and 1b are after removing the illegal PstI site. U: uncut. XbaI: miniprep cut with XbaI.  PstI: miniprep cut with PstI.
T7-LacO-regulated Cre generator
We have successfully cloned T7-LacO out of pET28b, and used the cloned fragment to perform a five-part MoClo                 assembly. The assembled T7-LacO-regulated Cre generator was then cloned into biobrick format ( BBa_K2406080 ).
Standardized measurement constructs
We have created standardized measurement constructs to quantify recombinase activity in vivo. They are essentially                 transcriptional terminator flanked by two recombination target sites, inserted between a constitutive promoter                 and a RFP gene. Of fifteen possible combinations for five recombinases (Cre, Dre, VCre, SCre, Vika), we have                 successfully generated, sequenced, and submitted twelve of them:
The biobrick number of the measurement devices generated
We also demonstrated that the measurement devices can be used as a molecular switch, expressing RFP in the presence of correct combination of target sites and recombinases:
The measurement devices can act as a biological switch, expressing RFP in the presence of IPTG. Plates on the left have IPTG added, whereas those on the right have no IPTG. 9A: Measurement device Vox-Vox in Vika-expressing cell; 12A: Measurement device Rox-Rox in Dre-expressing cell; 6A: Measurement device Slox-Slox in SCre-expressing cell; 5A: Slox-Vlox in SCre-expressing cell, acting as a negative control.
We noticed that the RFP is expressed relatively slowly (>24 hours) in both agar plate and liquid medium in plate reader. This may find applications in situation where a delayed response is required.
Assembling and testing the 14 recombination sites
We have successfully cloned the Rox ( BBa_K2406000 ), Vox ( BBa_K2406001 ), VLox ( BBa_K2406002 ), and SLox ( BBa_K2406003 ) in to pSB1C3 biobrick for use. Their functionality is demonstrated in the measurement constructs, described above.
We have also cloned ten additional target sites that can be recognized by the Cre recombinase. They are called Lox511 ( BBa_K2406008 ), Lox2272 ( BBa_K2406009 ), Lox5171 ( BBa_K2406010 ), LoxN ( BBa_K2406011 ), M2 ( BBa_K2406012 ), M3 ( BBa_K2406013 ), M7 ( BBa_K2406014 ), M11 ( BBa_K2406015 ), Nuoya ( BBa_K2406016 ), and Zsoka ( BBa_K2406017 ). They are all proved to be functional by in vitro assay using cell lysate containing Cre recombinase.
In vitro recombination assay using cell lysate containing Cre recombinase. In the experiment, linear oligo containing a complete target site and miniprep containing the identical target site are mixed together with cell lysate (+) or water (-).  In the lanes with cell lysate (+), a smear can be seen, indicating that recombination occurred between miniprep to form concatamer. However, one can also see a brighter band within the smear, indicating that recombination has occurred between the linear oligo and the circular plasmid, thus linearizing the plasmid.
Quantitative measurement of recombinase activity
We have co-transformed both of our T7-LacO-recombinase generator and our measurement construct into E. coli BL21 (DE3). These strains are then incubated with or without IPTG, on either LB plate over 48 hours, or in LB media in plate reader. We have thoroughly characterized the recombination efficiency of the five recombinases, and determined what combinations are the most orthogonal pairs for future applications. To access all of the raw data for the measurements we carried out, and the basic statistical parameters we calculated, e.g. standard error, download the Excel file below. [Excel spreadsheet containing all raw data and high-resolution graphs]
We have determined that Dre/Rox recombinase is likely the most efficient SSR in E. coli, and that it is orthogonal to the rest of the recombinases. However, we also observed that the SCre/VCre and VCre/Vika pairs would cross-react with one another. As a result, we do not suggest using either pair to catalyse parallel recombination events in one cell. Therefore, we recommend using Dre, Cre, SCre, and Vika for parallel reactions in E. coli, as all four listed will have minimal interference with one another.
Heat map showing describing the orthogonality of different recombinase systems in E. coli. Percentage of orthogonality is normalised with Rox-Rox in Dre being 100%.
Click here to learn more about our measurement.
Sequencing results
For all parts listed above, you can access sequencing data obtained using standard biobrick verification primers VF and VR by clicking on the download-able file below. Included in the file is a "cipher" so that any particularly confusing acronyms can be deciphered. Aligning relevant sequencing results to part sequences on registry will confirm the validity of these results.
[All sequencing data for our relevant parts, including spreadsheet explaining acronyms]
Logic gates and software
We have designed two-input OR, NOR, AND, NAND, XOR, and XNOR gates using the excision property of two orthogonal tyrosine recombinases. See our designs in the design page.
As they contain a high degree of repetitiveness, we had difficulty ordering them as a single DNA block. Therefore, we designed software to break up the repetitive elements into several oligonucleotides. This allows convenient design and ordering of target sites. The details on how to download and use our software can be found in the accessibility page under the human practice section.
Modeling the behavior of site-specific recombinase
We have built deterministic and stochastic models to simulate the behavior of our E. coli strain used for measurement (BL21 (DE3) E. coli carrying T7-LacO-recombinase generator and measurement constructs). The model is able to predict that the leaky expression of recombinase can induce a significant degree of terminator excision, leading to a moderate background expression of RFP.
Furthermore, we have developed an algorithm and used it to scan through the genome of E. coli BL21 (DE3) strain, and identified five genomic regions that may potentially be a functional target site for Cre recombinase.
Click here to learn more about how we model site-specific recombination.
Human practice: Interdisciplinarity in Synthetic Biology
Due to the interdisciplinary nature of SMORE, we investigated interdisciplinarity in biology to understand its influence on how successful projects are.
We conducted a skill exchange survey with Team Bulgaria and Technion Israel. The survey identified the use of technical language as pivotal in mutual understanding in interdisciplinary collaboration.
We also measured the diversity in discipline of past iGEM teams and analyzed it with iGEM achievements. We found no significant correlation between diversity and achievements. We proposed hypotheses to explain the result.
See how we analyze the interdisciplinarity in iGEM here.
Human practice: Accessibility Improvement of SMORE
From the aforementioned study, we identified challenges in interdisciplinary work and decided an improvement of accessibility is needed to promote interdisciplinary use of SMORE. We improved accessibility in four aspects:
Readability: we wrote highly readable introductory paragraphs for a wide audience. We also provided highly readable protocols for the software and the cell sorter.
Hardware: we devised a microfluidic device with a 3D syringe pump – an alternative to the expensive cell sorter in the market – to use with SMORE’s randomizer strategy.
Software: we wrote an oligonucleotide designer programme to help the inexperienced to design oligos to use with SMORE.
Data: we compiled and experimentally verified recombinase-related sequence data to establish recombination as a convenient and reliable technology.
See how we integrate our findings on interdisciplinarity into our project here.
