Team:Santa Clara/Notebook
Santa Clara Template for iGEM wiki site
Toggle navigation
Home
Team
Project
Theory:
The Problem
Nature's Solution
Our Defense System
Future Prospects
Outcomes:
Parts
Results
Achievements
Lab Work
Lab Notebook
Protocols
Safety
Human Practices
Practices
Collaboration
Attributions
SCU iGEM Lab Notebook
May
Week 1
Week 2
Week 3
Week 4
5/4/15 - Had first official meeting to review the guidelines of the competition and begin brainstorming ideas for our project.
5/6/15 - Regrouped to discuss ideas. Entertained diverse ideas from creating a pH sensing bacterium to use as a drug delivery system to fight cancer to using an enterobacterium that would function as a probiotic to try and suppress the Celiac’s disease response to gluten.
5/9/15 - Decided that focusing on in vivo projects isn’t a good avenue for us to study because of the practical hurdles associated with trying to get an organism to live within the body and then to get that passed by the FDA. Thought about possibly trying to create some sort of helper cell that would aid in vitro screening processes, specifically for cancer research.
5/11/15 - Over the weekend we intensively studied immunotherapy and its cancer related applications. We thought about trying to create an organism that would maybe aid in the production of T and B-cells.
5/14/15 - Decided that we didn’t have the proper resources to adequately test anything related to cancer research so we shifted gears. Some of us revisited the Celiac’s disease idea and tried finding a site in the recognition of gluten in an attempt to find a spot to repress the response. The rest then went back to the pH sensing system. We saw this competition as a driving force to construct more parts to expand the registry rather than to create a stand alone project, so creating a pH sensing system could prove useful in other applications.
5/16/15 - After more investigation we found that other pH sensors had been submitted to the Registry in the past but none were directly linked to transcription factors. So our focused turned to understanding nature’s way of sensing pH.
5/20/15 - Reconvened to discuss findings. Only a few potential candidates were found but none seemed realistic. We decided to table the idea and keep brainstorming and return to this idea if we were unable to come up with anything else.
5/23/15 - Entertained the idea of biofuel production. The team went an investigated potential molecules we would produce and the research that has already been done in this competition and out.
5/28/15 - Started thinking more big picture about the goals of the synthetic biology and what kind of problems it can realistically solve at this point in time. We thought about how it is that such small organisms can have larger effects on society. This idea of scaling up biology bot us thinking about bioreactors.
June
Week 1
Week 2
Week 3
Week 4
6/2/15 - With bioreactor on our mind, several of us took a field trip to Anaerobe Systems, a local company, to see the large scale fermentors that were in use there. After seeing their setup and how beneficial his systems were we shifted from bioreactors, in general, to fermentors that degrade organic waste like anaerobe systems does. None of us were familiar with the cellulolytic pathways so we went and did more research.
6/10/15 - Met up to discuss everyone’s findings. We found that the exact mechanism for this degradation is still not well understood as it requires the use of a cellulosome, which is a conglomerate of numerous large proteins that has yet to be fully visualized. Our hope was to try and take several pivotal proteins from the complex and insert them into a high expression vector to see if by increasing the production of these proteins, we could increase the cellulolytic activity of the organism being used in the bioreactor.
6/16/15 - Studied the actual workings of the fermentors at Anaerobe systems to see what problems were actually present. We found that the rate of activity of the organism was not the problem because they were using a cellulolytic Clostridium species, which has a doubling time of about 6 minutes. We did find, however, that their continuous culture system was unable to function for longer than three hours. We hypothesized this to be the result of the high acid concentration in the culture that accumulated over the time of the culture. In order to address this issue we decided to create a two part project: one side will sense low pH which would then induce the expression of the other side which is the regulating system that would be responsible for returning the intracellular pH to neutral.
6/26/15 - We met to discuss our findings related to the sensing and regulating systems utilized in nature. The sensing systems were still eluding us but we found numerous regulating systems. We focused on either trying to actively transport the protons out of the cell or quenching them in the cytosol using amino acid decarboxylases.
July
Week 1
Week 2
Week 3
Week 4
7/1/15 - Based on these two systems we began planning out the experiments we would have to run to begin standardizing procedures to then use to test our acid resistance system while we continue to plan out exactly what that system is. We decided to use B. subtilis as our chassis because it shares close homology with Clostridia, which is the organism we aim to implement the system in. To do this we had to figure out how to transform B. subtilis and what we were going to transform with.
7/4/15 - Decided to use the pVeg promoter BioBrick as our test, constitutive promoter and then using GFP as our reporter gene. We opted to use the starvation method to induce the natural competence in B. subtilis and transform.
7/6/15 - Resuspended both pVeg and GFP BioBricks and then transformed them into Top10 competent cells.
7/7/15 - Colonies were observed on plates so the colonies were picked to make overnights. B. subtilis was ordered and preparations for its transformation were made. This included making several different solutions to act as washes and growth media. All solutions were prepped and autoclaved.
7/8/15 - DNA was extracted from overnights and quantified using a NanoDrop. DNA was then sent out for sequencing to verify the sequence posted on the Registry.
7/10/15 - Sequences came back matching the anticipated sequences. Based on that sequence we then designed and ordered primers to use PIPE cloning to insert GFP into the pVeg vector. Received the B. subtilis in the mail. An overnight culture was made from it into LB.
7/11/15 - We successfully cultured up B. subtilis so we took a small aliquot to make a glycerol stock.
7/13/15 - Primers arrived and PCRs were run. 1st round of PCR did not generate any bands on the agarose gel. We suspected that this was due to the annealing temperature being too high for proper binding. So we ran another using a gradient from 55 to 65 instead of 72 like was done previously. We saw proper amplification of the GFP insert DNA but only saw a band that represented a substantially smaller segment than what was supposed to be there. This indicated off target binding somewhere else in the vector.
7/14/15 - Another round of PCR was run on both of the sequences to try and isolate the issue with the pVeg amplification. The gel showed that GFP was successful but pVeg showed the same band that we previously saw. To better understand the what was happening we sent the pVeg DNA along with the primers that we were using to see what was being amplified.
7/15/15 - While waiting for the sequencing results we decided to move forward with the B. subtilis transformation. A plate was streaked out from the glycerol stock of the B. subtilis 168.
7/16/15 - Colonies were picked from the streak plate of B. sub 168 to make overnights to seed the cultures for the transformation. All required glassware was prepped and autoclaved. Sequence of pVeg came in and was analyzed. It was found that the primer was preferentially binding to another site in the vector. To remedy this issue we redesigned the primers to have a higher Tm and made sure that the Tm’s of the primer pair were closer together to ensure that we are getting the proper binding of both at the same annealing temperature. Primers were designed and ordered.
7/17/15 - The overnight cultures were used to run the starvation transformation protocol. The transformation was run with the pVeg plasmid as that will be the vector that the construct will be inside of.
7/18/15 - Plates from the transformation yielded no colonies indicating a failed transformation. It was anticipated that maybe the procedure run time was much longer than it was supposed to be so we thought that maybe that was the issue because all of the OD readings matched the documented procedure’s estimations. More colonies were picked to rerun the transformation. Materials were prepped again for the next day.
7/19/15 - Another starvation transformation was run and plated.
7/20/15 - No colonies were once again observed on the plates. Knowing that the procedure was followed very closely we broadened our scope to look for other potential causes for the failed transformations. We hypothesized that it could have been the detergent residue left in the glassware that was causing the cells to not take up the DNA. It was also possible that our deionized water could have been causing  issues because it was found to be quite acidic. To fix these issues we soaked the glassware in deionized water overnight to try and draw all of the remaining detergent out and to pH to neutral and autoclave deionized water. We went ahead and remade all of the solutions too. Primers also came in.
7/21/15 - Overnights were created from the B. subtilis 168 plate. PCRs were run using the new primers. the amplicons were then run on a gel and visualized. We once again saw the proper amplification of the GFP insert but no amplification of pVeg. We went back and reran the PCR but this time ran a gradient of annealing temperatures. After that was run it was run on a gel and still yielded no bands.
7/22/15 - We ran another B. subtilis starvation transformation using the, hopefully, detergent-free glassware and neutral deionized water. We also gave the PCR another shot to see if maybe there was just pipetting error or insufficient mixing of primers or buffers. Once again the result was no amplification. Confident that we were not going to get these primers to work, we re-evaluated our primers. Based on all of the literature that we read on the PIPE method and primer design we couldn’t find the issue. After burning so many days on the PIPE method we decided that it would be better to switch to the traditional scar cloning method.
7/23/15 - Yet again the transformation didn’t work. The only possibility that we could come up with was that we still were not successfully ridding the glassware of the detergents. To fix this we decided to use disposable culture flasks that have never been exposed to detergents to ensure that is not the problem. More overnights were made. All of the restriction enzymes, ligase, gel extraction, and genome extraction kits were ordered.
7/24/15 - Starvation transformation was conducted again using the overnights and disposable flasks.
7/25/15 - Still no colonies from the transformation. After long deliberations over what other parameters we could change, we came up blank. We assessed whether or not it was really necessary to transform into B. subtilis. We chose to do it in the first place because of its close homology to Clostridia, but all we really need at the moment is to show some increase in acid resistance. So after wasting numerous days, we opted to discontinue trying to transform into B. subtilis and keep on testing in E. coli.
August
Week 1
Week 2
Week 3
Week 4
8/3/15 - Being as we moved from B. subtilis to E. coli we need to choose a new promoter+RBS for expression in E. coli. Part BBa_K608002 was then resuspended and transformed into NEB 5 alpha.
8/4/15 - Transformation was successful and colonies were picked to generate overnight cultures containing the constitutive promoter.
8/5/15 - All of the scar cloning materials finally came in the mail so we could begin trying to clone again. The overnight cultures were mini prepped and quantified using a NanoDrop. The GFP plasmid was digested with EcoRI and SpeI and the constitutive promoter was digested with EcoRI and XbaI. The digests were run on a gel, showing the appropriate sized bands. The desired bands were extracted with a gel extraction kit. The pieces were then ligated together at a 1:2 molar ratio and transformed into NEB 5 alpha cells.
8/6/15 - No colonies were observed on any of the plates indicating an unsuccessful transformation. The conclusion was that the DNA was exposed to the UV for too long and so we decided to run another gel using the restriction digests from before. Gel extraction was run quickly and then transformed.
8/7/15 - Still no colonies were observed from the previous transformation. Confident that it wasn’t a procedural error we turned the focus toward vector insert ratios. We reran the digests based on the same protocol as before but this time used a spectrum of molar ratios. We tried 1:1, 1:2, 1:5, and 1:10 vector to insert then ligated and transformed.
8/8/15 - No colonies were observed showing that the transformation was unsuccessful  yet again. The hypothesis then turned to ligation time.
8/10/15 - Restriction digests were preformed based on the prior specifications. Gel followed by extraction showed proper bands. The ligation was then preformed for 2 hours at 16C to see if that would allow for proper ligation. The same spectrum of vector to insert ratios was used in the ligation. Samples were then transformed and plated.
8/11/15 - No colonies were observed again. With the cloning process taking as long as it is, we moved to trying to extract our regulator gene CFA. We requested a streak plate of E. coli K-12 from the Steven’s lab on the SCU campus to extract the DNA from.
8/13/15 - Plate was received from the Steven’s lab. Colonies were picked to generate overnights.
8/14/15 - A genome extraction was preformed on the overnights. DNA was then quantified using a NanoDrop. Primers were designed to flank the CFA synthase gene with overhangs that matched the sequence of the prefix and suffix.
8/17/15 - Received primers in the mail. PCRs were run on the genome using these primers. The product was then digested based on the standard assembly, as GFP was previously. CFA and GFP insert digests were gel extracted and ligated downstream of the constitutive promoter using the same spectrum of vector to insert ratios used previously. These products were then transformed and plated.
8/18/15 - Still no recombinants could be seen from the transformation. Thinking that we have changed all of the parameters that we could, we suspected that maybe there was an issue with the constitutive promoter sequence that was leading to neither of them working. Another promoter, BBa_J04500, was resuspended and transformed along with BBa_R0040, BBa_B0015, and BBa_B0034.
8/19/15 - Colonies were observed and picked to make overnight cultures.
8/20/15 - DNA was extracted from the overnights and glycerol stocks were made. The restriction digests were run as before. The digests were visualized on a gel and showed the appropriated sized bands. Gel extraction followed by ligation as before was conducted. The resulting DNA was transformed.
8/21/15 - No colonies were observed on the plates. Even though the bands were the right size, we decided to change the restriction digest run time to see if that would help. The digests were conducted for two hours as opposed to one. Visualized on a gel, extracted, ligated and transformed.
8/22/15 - No colonies were observed yet again. The only other step that we though could possibly be causing issues was the gel extraction step. This drops the efficiency of a ligation/transformation by several orders of magnitude. To fix this issue we changed around our gel extraction protocol so that each lane was precut out of the gel so that only the band being cut would be exposed to the UV opposed to all of them being exposed. Using this method we reran the standard assembly.
8/23/15 - Still no colonies were observed. With no other parameters to change we decided to switch back to scarless cloning, specifically PIPE. Primers pairs were designed to amplify both the CFA and GFP inserts as well as to open up the pLac promoter. Primers were then ordered.
8/25/15 - Primers arrived in mail. Primers were resuspended and used in PCRs with the GFP and pLac BioBricks along with the genomic DNA for CFA. Bands were generated for both CFA and GFP but not for pLac. The PCR for pLac was conducted again, this time with a gradient of annealing temperatures. The gel then revealed that we had a slight band in the right position. We then took that sample, along with CFA and GFP, and cleaned them in a PCR cleanup, quantified them and then ligated at a 1:2 molar ratio.
8/26/15 - Once again no colonies were seen on any of the plates. We then thought that maybe the ratio was not right. So we reran the DpnI digestion/ligation step of PIPE with the cleaned PCR products from before but with the previously used spectrum of vector to insert ratios. They were then transformed.
8/27/15 - No colonies were seen on plate.
8/31/15 - We reran the PCR on the pLac vector and this time saw much larger bands. The same PIPE procedure was conducted using this new DNA.
September
Week 1
Week 2
Week 3
Week 4
9/1/15 - Studied plates from PIPE transformation of both CFA and GFP downstream of the pLac promoter and saw isolated colonies. These colonies were picked to start overnight. More LB/LB+CAM plates were made. No more PCRs or transformations were run to avoid wasting resources.
9/2/15 - The DNA was extracted from the overnights, digested, and then run on a gel but the gel only showed the bands representing the pLac promoter on its own, without the insert for both CFA and GFP. We anticipate that the problem was with the molar ratio of vector to insert. We also have been consistently getting weak bands from our vector PCRs so we are going to try running them temperatures, as opposed to the standard 72°C
9/3/15 - PCRs were run passed on the troubleshooting from the day before. The hypothesis appeared to be right because we saw much stronger bands from the this round of PCR.
9/4/15 - DpnI digestion/ligation of the PIPE PCR products were conducted and then transformed into NEB 5 alpha cells. This time we ran a background of only vector to account for the amount of perceived colonies that were just the result of re-ligation of the vector.
9/5/15 - The plates from the transformation showed 2 colonies on both the background plate and on the recombinant plate, indicating that we did not get recombinants. Overnights were still made from the colonies to be screened the next day. We decided to revisit the primer design of the overhangs being used to see if that is where the problem lied. The knew that the problem had to be embedded in the ligation step because we were getting clear amplification of the inserts and vectors and we have standardized our transformation protocol, so that leaves the ligation. We created two different sets of primers with new overhangs to hopefully remedy the problem. Unfortunately the primers can’t be ordered until Tuesday because of the holiday.
9/6/15 - Went back and double checked the primers. Also decided it was time to integrate SAM into its own vector. We picked the pAraC vector so that we could induce transcription with arabinose separately from inducing expression of CFA with IPTG.
9/7/15 - pAraC BioBrick was resuspended and transformed into NEB 5 alpha cells.
9/8/15 - Colonies were observed from the pAraC transformation. Primers were then ordered.
9/9/15 - Primers finally arrived so numerous PCRs were run to amplify the inserts CFA and SAM and vectors pLac and pAraC. Both PCRs of the inserts worked but the vectors were unsuccessful.
9/10/15 - Transformation plates were checked but only one colony was observed. Being as there was only one colony we weren’t confident that it was a recombinant. Ran a colony PCR on the colony but the gel showed no amplicon indicating a failed PCR rather than a failed transformation. We made an overnight from the colony to miniprep and analyze the next day. A gel was also run on the PCR products that were used for PIPE the day before. The gel showed large bands for CFA, SAM and the pLac vector but the pAraC vector still wouldn’t amplify. We anticipate this being a result of incorrect sequencing for the bases near the middle of the BioBrick. The sequence available for that part stated that the sequence was inconsistent in the center which is where we designed our primers to bind so if the sequences didn’t match up, we wouldn’t get amplification which is what we are seeing.
9/11/15 - DNA extraction was run on the overnight but after visualization on the gel we saw that the product was an empty vector. So we decided to rerun the PCRs, digestion/ligation and transformation. Ran several PCRs to generate the amplicons needed for our PIPE cloning method. Two primer sets were used. One integrated a 6bp spacer between the RBS and ORF, while the other did not. Strong bands were seen from product. Digest/ligation was conducted at a 1:2 and 1:4 molar ratios of vector to insert. The DNA was then transformed.
9/12/15 - Observed no colonies on the plates from the PIPE transformations. Ran three different PCRs replicating the PCRs that were conducted the day before but for some reason no amplicon was produced on the first round. SAM was produced on the second and then nothing again in the third one of just CFA. The cycles were maintained as well as the template DNA. We used all three sets of primers that we have to extract CFA and yet was unable to amplify it. During this time we were also running a control acid shock assay to test the response of the untransformed T7 cells that we will be testing.
9/13/15 - Began by rerunning the PCRs from the previous day and for some reason it worked this time. Three different sets of PCRs were run: One with a final elongation step and 35 cycles for OEC, 35 cycles for PIPE, and 30 cycles for PIPE. It was hypothesized that our PIPE method had to be running into issues during the DpnI digestion/ligation step because we saw defined bands after PCR and high efficiency transformation so the insert must not be integrating. This could happen for a myriad of reasons but it is likely that the polymerase is going back and filling in the overhangs making it impossible to ligate. So we ran the PCR at fewer cycles in order to see if those last 5 cycles were causing the cells to fix the sequence as if there was a final elongation step.
9/14/15 - Observed plates from the three different transformations attempting to build CFA synthase gene downstream of the pLac promoter. No colonies from the OEC transformation or from the PIPE products run at 35 cycles. The 30 cycle PCR products yielded ~50 colonies on the concentrated plate with only 7 on the background plate. Colonies were picked to make overnight cultures. Another round of OEC was conducted. A gel was run to verify that the OEC procedure was not successful. It was predicted that there was not a large enough concentration of mega-primer for the second PCR which would cause little binding to the template DNA. So we ran the procedure again and adding more mega-primer. We also weren’t sure if the annealing temperature was too high for adequate binding so we ran the second PCR on a gradient. Also miniprepped the overnight cultures made from the CFA and SAM genes inserted into the pSB1C3 vector to be sent into iGEM. The DNA was quantified using a nanodrop, digested with EcoRI and SpeI, and then visualized on an agarose gel. The gel revealed that CFA and SAM were successfully built into the vectors. The CFA PstI site was then destroyed using site-directed mutagenesis. PCR product was then transformed into competent cells. DNA was sent to Sequetech for sequencing.
9/15/15 - Observed colonies from the site-directed mutagenesis tranformation. Colonies picked for overnights. OEC transformation once again yielded no colonies. Miniprep was conducted on the overnights generated from the PIPE colonies, but after digestion and visualization, it was clear that we had not inserted the CFA gene into the vector. We were running low on mini prep reagents and so we only picked four colonies and so we picked more to see if maybe we just grabbed ones with religated vectors like those we saw on the background plate. Colony PCR was run on some colonies but yielded no conclusive results. We also ran a Gibson assembly utilizing the PCR products generated by the product of the 35 cycles of PCR generated through OEC. We knew we didn’t have time to order new primers, but scarless cloning procedures use similar primers so we kept with the ones that we had. Digestion, ligation, and transformation were conducted on these segments into both NEB 5 alpha cells and T7 competent cells.
9/16/15 - Gibson assembly transformations were successful from all three primer sets constructed for the integration of CFA downstream of the pLac promoter. Colonies were picked for colony PCR using primers that lie outside of the prefix and suffix. The colony PCR only showed the band representing the pLac promoter indicating an failed integration of CFA. We decided to go back and screen more colonies. The second round of colony PCR then showed several recombinants. The overnight cultures made from those recombinant colonies were then kept and incubated overnight. ~150 LB/LB+CAM plates were poured in preparation for the acid shock trials. Sequence data for the pSB1C3+CFA and pSB1C3+SAM came in and was analyzed in comparison to the anticipated sequence. There was no discrepancies between the sequences. The site-directed mutagenesis to destroy the PstI site in CFA also proved to be successful after running a digest with PstI and EcoRI and then analysis through gel electrophoresis. Only two bands were seen in the gel showing that the PstI site had been destroyed.
Going to the Jamboree!
Achievements
Protocols
