<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Santa_Clara_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Santa Clara/Protocols</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Santa Clara Template for iGEM wiki site</TITLE></P><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse-target" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="navbar-collapse-target"><UL class="nav navbar-nav"><LI><A href="https://2015.igem.org/Team:Santa_Clara">Home</A></LI><LI><A href="https://2015.igem.org/Team:Santa_Clara/Team">Team</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project </A></LI><LI class="dropdown-header">Theory:</LI><LI><A href="https://2015.igem.org/Team:Santa_Clara/TheProblem">The Problem</A></LI><LI><A href="https://2015.igem.org/Team:Santa_Clara/NatureSolution">Nature's Solution</A></LI><LI><A href="https://2015.igem.org/Team:Santa_Clara/OurDefenseSystem">Our Defense System</A></LI><LI><A href="https://2015.igem.org/Team:Santa_Clara/FutureProspects">Future Prospects</A></LI><LI class="dropdown-header">Outcomes:</LI><LI><A href="https://2015.igem.org/Team:Santa_Clara/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Santa_Clara/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Santa_Clara/Achievements">Achievements</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Lab Work </A></LI><LI><A href="https://2015.igem.org/Team:Santa_Clara/Notebook">Lab Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Santa_Clara/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Santa_Clara/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Human Practices </A></LI><LI><A href="https://2015.igem.org/Team:Santa_Clara/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Santa_Clara/Collaboration">Collaboration</A></LI><LI><A href="https://2015.igem.org/Team:Santa_Clara/Attributions">Attributions</A></LI></DIV></DIV><DIV id="innerContent" class="container-fluid"><H1>Protocols</H1><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#AgaroseGels">Agarose Gels</A></H4></DIV><DIV id="AgaroseGels" class="panel-collapse collapse"><DIV class="panel-body"><H2>Materials</H2><UL><LI>Agarose</LI><LI>Gel Rig</LI><LI>Loading Dye</LI><LI>Gel Red™ DNA stain</LI><UL><LI>Gel Red™ is a non-toxic gel dye (used in place of Ethidium Bromide</LI><LI>10,000x in water</LI></UL><LI>DNA ladder</LI><LI>10x TAE (Tris-Acetic Acid-EDTA)</LI><UL><LI>40 mM Tris Base (48.4 g Tris Base; MW = 121 g/mol)</LI><LI>20 mM Acetic Acid (11.42 mL Glacial Acetic Acid)</LI><LI>1 mM EDTA (20 mL 0.5 M EDTA, pH 5)</LI><LI>Adjust pH to 8.3</LI></UL></UL><H2>Protocol</H2><H4>Pouring Gel</H4><OL><LI>Measure our an appropriate amount of agarose.</LI><OL type="a"><LI>1% agarose gel = 0.5 g agarose in 50 mL of 1x TAE</LI></OL><LI>Dissolve gel in 1X TAE to final volume desired.</LI><OL type="a"><LI>Microwave the solution until all the agarose has dissolved.</LI><LI>Watch carefully to ensure that the contents do not boil over.</LI></OL><LI>Let dissolved gel cool until you can hold it, but not too cool that it solidifies.</LI><LI>Once cool enough to handle add 2.5 μl Gel Red™ (0.5x Concentration) to the solution and swirl to dissolve (0.5-1x concentration works best to visualize samples)</LI><OL type="a"><LI>It is important to add the Gel Red™ while the solution is still warm to allow for proper mixing, but not too hot as to denature the dye.</LI></OL><LI>Pour the solution into the gel rig such that the gel will solidify. Add comb to create wells to be loaded.</LI><LI>Let cool until solid, approximately 15 minutes.</LI></OL><H4>Loading Gel</H4><OL><LI>Add appropriate amount of gel loading dye to your DNA sample.</LI><OL type="a"><LI>1μl dye to 5μl DNA is sufficient for PCR products and restriction digests.</LI></OL><LI>Load samples and Molecular weight marker.</LI><OL type="a"><LI>3-4μl is sufficient to visualize.</LI><LI>Record the order of which the samples were loaded.</LI></OL><LI>Run the Gel at 120V-140V until the dye is at least 3/4 way down the gel.</LI><OL type="a"><LI>Make sure the DNA samples will run the correct direction. DNA is negatively charged and will migrate towards the cathode (the phrase “Run to Red” refers to DNA samples migrating to the red cathode on the gel rig).</LI><LI>Do not run the gel too fast (≥140V). At faster speeds the resolution of the bands of DNA decreases.</LI></OL><LI>Once gel is complete image with UV light.</LI></OL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#LIC">Ligation Independent Cloning Techniques (PIPE and SLIC)</A></H4></DIV><DIV id="LIC" class="panel-collapse collapse"><DIV class="panel-body"><P>Protocol adapted from “A Practical Comparison of Ligation-Independent Cloning Techniques” by Julian Stevenson, James R. Krycer, Lisa Phan, Andrew J. Brown</P><H2>Materials</H2><H4>Reagents</H4><UL><LI>Vector primers</LI><LI>Insert primers (overhang to vector)</LI><LI>Vector DNA (known concentration)</LI><LI>Insert DNA (known concentration)</LI></UL><H4>PCR Components</H4><UL><LI>DNA polymerase (Q5 high fidelity DNA Polymerase from NEB)</LI><LI>5x Reaction buffer (comes with DNA polymerase)</LI><LI>10 mM dNTPs</LI><LI>Distilled Water (ddH<SUB>2</SUB>O)</LI><LI>Enhancer (optional)</LI></UL><H4>Cloning and transformation</H4><UL><LI>DpnI and buffer</LI><LI>Chemically competent cells</LI><LI>LB+Antibiotic plates</LI></UL><H2>Protocol</H2><H4>For Polymerase Incomplete Primer Extension (PIPE) cloning</H4><OL><LI>Combine PCR components in a PCR tube.</LI><TABLE class="table table-bordered"><THEAD><TR><TH>Component</TH><TH>Volumes for 25 μL reaction</TH><TH>Volumes for 50 μL reaction</TH><TH>Final Concentration</TH></TR></THEAD><TBODY><TR><TD>5x reaction buffer</TD><TD>5 μL</TD><TD>10 μL</TD><TD>1x</TD></TR><TR><TD>25 μM Forward Primer</TD><TD>0.5 μL</TD><TD>1 μL</TD><TD>0.5 μM</TD></TR><TR><TD>25 μM Reverse Primer</TD><TD>0.5 μL</TD><TD>1 μL</TD><TD>0.5 μM</TD></TR><TR><TD>Template DNA</TD><TD>1 μL of 1 ng/μL</TD><TD>2 μL of 1 ng/μL</TD><TD>0.04 ng/μL</TD></TR><TR><TD>10 mM dNTPs</TD><TD>0.5 μL</TD><TD>1 μL</TD><TD>200 μM</TD></TR><TR><TD>DNA polymerase</TD><TD>0.25 μL</TD><TD>0.5 μL</TD><TD>0.02 U/μL</TD></TR><TR><TD>Enhancer (optional)</TD><TD>5 μL</TD><TD>10 μL</TD><TD>1x</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>Up to 25 μL</TD><TD>Up to 50 μL</TD></TR></TBODY></TABLE><LI>Mix components well, spin down if necessary.</LI><LI>Run PCR cycle conditions</LI><TABLE class="table table-bordered"><THEAD><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR></THEAD><TBODY><TR><TD>Initial denaturation</TD><TD>98˚C</TD><TD>30 seconds</TD></TR><TR><TD>30 cycles</TD><TD>98˚C72˚C72˚C</TD><TD>10 seconds10 seconds20-30 seconds/kb</TD></TR><TR><TD>Infinite hold</TD><TD>4˚C</TD><TD>Indefinite</TD></TR></TBODY></TABLE><OL type="a"><LI>Run an agarose gel on the PCR sample to check for successful amplification.</LI><LI>If there is high off target amplification, you can gel purify the right size DNA fragment and proceed with protocol.</LI></OL><LI>Combine 5 μL of Insert DNA PCR products to 5 μL of Vector DNA PCR products. Also digest vector only and insert only samples to act as controls for transformation efficiency.</LI><LI>Add 10 U (0.5 μL) of DpnI and CutSmart NEB buffer to a final concentration of 1x. Mix well.</LI><LI>Incubate at 37˚C for 2 hours.</LI><LI>Add 6 μL of cloned and control DNA to separate vials of competent cells. Follow manufacturer's transformation protocol.</LI></OL><H4>For Sequence and Ligation-Independent Cloning (SLIC)</H4><OL><LI>PCR cleanup products before DpnI digest.</LI><LI>After DpnI digestion add 0.75 U of T4 DNA polymerase and incubate at 25˚C for 5-10 min the immediately put on ice to stop the reaction.</LI><LI>Transform competent cells according to manufacturer's directions.</LI></OL><H4>Notes</H4><UL><LI>T4 DNA polymerase acts as an exonuclease in the absence of dNTPs. For this reason PCR clean-up is extremely important.</LI><LI>Do not permit the T4 chew back reaction to progress too long because it will degrade the DNA and take away the designed overhangs.</LI></UL><H4>Troubleshooting</H4><TABLE class="table table-bordered"><THEAD><TR><TH>Problem</TH><TH>Reason</TH><TH>Solution</TH></TR></THEAD><TBODY><TR><TD>High Background</TD><TD>Too much of one DNA sample to ligate to its partner</TD><TD>Change the ratio of Vector: Insert depending on where the background is arising from. Typically high background is seen in the vector due to conserved antibiotic resistance. Increase the amount of insert added to the reaction to force ligation with the vector. Ratios range from 1:2-5. You can compare the molar ratios to get exact ratios.</TD></TR><TR><TD>No recombinants</TD><TD>Ligation of the vector and insert did not occur</TD><TD>During PCR full length, blunt products can be created which would inhibit the reaction. Either shorten the PCR cycles or/elongation length/ try SLIC cloning to manually create the overhangs.</TD></TR><TR><TD>Weak/no PCR product observed</TD><TD>Not enough target DNA addedNot enough primer addedOther</TD><TD>Make sure you are adding all the ingredients for PCR and at the right concentrations. Make sure all ingredients are thawed and mixed well.If all else fails, make fresh solutions of DNA, primers, and dNTPs.</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#RestrictionDigests">Restriction Digests</A></H4></DIV><DIV id="RestrictionDigests" class="panel-collapse collapse"><DIV class="panel-body"><H2>Materials</H2><UL><LI>DNA</LI><LI>Restriction enzymes</LI><LI>Buffer</LI><LI>Sterile Water</LI></UL><H2>Protocol</H2><OL><LI>Mix all components as follows:</LI><TABLE class="table table-bordered"><THEAD><TR><TH>Reagent\Reaction</TH><TH>Uncut</TH><TH>Single Cut</TH><TH>Double Cut</TH></TR></THEAD><TBODY><TR><TD>DNA</TD><TD>5 μl</TD><TD>5 μl</TD><TD>5 μl</TD></TR><TR><TD>Restriction Enzyme</TD><TD>0 μl</TD><TD>1 μl</TD><TD>1 μl of each enzyme</TD></TR><TR><TD>Buffer</TD><TD>2 μl</TD><TD>2 μl</TD><TD>2 μl</TD></TR><TR><TD>Water</TD><TD>13 μl</TD><TD>12 μl</TD><TD>11 μl</TD></TR><TR><TD>Final Volume</TD><TD>20 μl</TD><TD>20 μl</TD><TD>20 μl</TD></TR></TBODY></TABLE><LI>Digest samples at 37˚C for 1 hour.</LI><LI>Run samples on a gel to see results of digest.</LI></OL><H4>Notes</H4><UL><LI>Online tools, like NEB cutter, can be used to determine what restriction enzymes to use.</LI><LI>If cutting is not occurring/only some of the DNA is getting cut it is possible that too much DNA is added and the activity of the enzyme prevents it from cutting all of the DNA. Redo the digest with less DNA.</LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#SDM">Site-Directed Mutagenesis (via PIPE)</A></H4></DIV><DIV id="SDM" class="panel-collapse collapse"><DIV class="panel-body"><H2>Materials</H2><UL><LI>See PIPE cloning protocol.</LI></UL><H2>Primer Design</H2><UL><LI>Design Primers as you would for PIPE with the overhangs containing the DNA to be mutated and the primers facing to amplify the vector.</LI><LI>Make sure the point mutation is in the middle of the overhangs.</LI></UL><H2>Protocol</H2><OL><LI>Follow PIPE cloning protocol for PCR.</LI><LI>Check for proper amplification on a gel.</LI><LI>DpnI digest the PCR product.</LI><LI>Transform following manufacturer's directions.</LI><LI>Check recombinants via restriction digest (if a restriction site was destroyed) or sequencing.</LI></OL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#AcidShockAssay">Acid Shock Assay</A></H4></DIV><DIV id="AcidShockAssay" class="panel-collapse collapse"><DIV class="panel-body"><H2>Materials</H2><UL><LI>LB Broth, pH 3, 4, 5, and 7 (record actual pH)</LI><LI>T7 cells</LI><LI>Spectrophotometer</LI><LI>Culture tubes</LI><LI>Eppendorf Tubes</LI></UL><H2>Protocol</H2><OL><LI>Grow overnight of T7 cells.</LI><LI>Dilute 100 μl of T7 Cells in 4 mL of Neutral pH LB.</LI><OL type="a"><LI>Separate tubes for each pH LB (4 total).</LI></OL><LI>Grow to an OD<SUB>600</SUB> of approximately 0.7 (record exact OD).</LI><LI>Take 100 μl aliquot and add 900 μl neutral pH LB, and plate (1:100 dilution).</LI><LI>Pellet 4 mL.</LI><LI>Resuspend in 1 mL of appropriate pH LB.</LI><LI>Add to 3 mL of appropriate pH LB.</LI><LI>Record OD</LI><OL type="a"><LI>Take 100 μl aliquot and add 900 μl neutral pH, and plate (1:100 dilution).</LI></OL><LI>Take OD reading every 30 minutes.</LI><OL type="a"><LI>Take 100 μl aliquot and add 900 μl neutral pH, and plate (1:100 dilution).</LI></OL><LI>Take time points until OD levels off for 1.5 hours.</LI><LI>Count colonies on plates to see the survival of the cells throughout the time course.</LI></OL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#AciduricityTest">Aciduricity Test</A></H4></DIV><DIV id="AciduricityTest" class="panel-collapse collapse"><DIV class="panel-body"><H2>Materials</H2><UL><LI>LB Broth, pH 3 and 7 (record actual pH)</LI><LI>T7 cells</LI><UL><LI>Untransformed cells and transformed cells</LI></UL><LI>Spectrophotometer</LI><LI>Culture tubes</LI><LI>Eppendorf Tubes</LI><LI>LB agar plates</LI><UL><LI>With appropriate antibiotic if necessary</LI></UL></UL><H2>Protocol</H2><OL><LI>Grow overnight of cell lines to be tested.</LI><LI>Dilute 100 μl of cells in 4 mL of Neutral pH LB</LI><LI>Measure OD<SUB>600</SUB> until approximately 0.5 and induce the expression of gene in transformed cell line.</LI><OL><LI>0.5 mM - 5 mM IPTG for lac promoters</LI></OL><LI>At OD<SUB>600</SUB> 0.7 take 1 μl aliquot and add 99 μl of neutral pH LB and plate (1:100 dilution).</LI><OL><LI>Record the exact OD from when aliquot was taken. This is T<SUB>0</SUB></LI></OL><LI>Pellet all 4 mL of remaining cell culture.</LI><LI>Resuspend in 1 mL of appropriate LB pH 3.</LI><LI>Add resuspension to culture tube with 3 mL of pH 3.</LI><OL><LI>Record the OD and exact time after resuspension and an aliquot to be plated (see step 4). This is T<SUB>i</SUB>.</LI></OL><LI>Every 30 minutes record the OD and take an aliquot to plate (see step 4).</LI><OL><LI>a. Keep culture going for 4-5 hours (T<SUB>1-10</SUB>).</LI></OL><LI>Count colonies on the plates the next day.</LI></OL><H4>Notes</H4><UL><LI>A blank needs to be taken at each time point to account for drift in the spectrophotometer.</LI><LI>Each pH LB needs its own blank as well as the appropriate antibiotic if necessary.</LI><LI>Once done with assay cells need to be killed with Bleach or Lysol before being thrown out.</LI></UL></DIV></DIV></DIV></DIV><UL class="pager"><LI class="previous"><A href="https://2015.igem.org/Team:Santa_Clara/Notebook"> Notebook
        	</A></LI><LI class="next"><A href="https://2015.igem.org/Team:Santa_Clara/Safety">
            Safety </A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>