Team:Santa Clara/Protocols
Santa Clara Template for iGEM wiki site
Toggle navigation
Home
Team
Project
Theory:
The Problem
Nature's Solution
Our Defense System
Future Prospects
Outcomes:
Parts
Results
Achievements
Lab Work
Lab Notebook
Protocols
Safety
Human Practices
Practices
Collaboration
Attributions
Protocols
Agarose Gels
Materials
Agarose
Gel Rig
Loading Dye
Gel Red™ DNA stain
Gel Red™ is a non-toxic gel dye (used in place of Ethidium Bromide
10,000x in water
DNA ladder
10x TAE (Tris-Acetic Acid-EDTA)
40 mM Tris Base (48.4 g Tris Base; MW = 121 g/mol)
20 mM Acetic Acid (11.42 mL Glacial Acetic Acid)
1 mM EDTA (20 mL 0.5 M EDTA, pH 5)
Adjust pH to 8.3
Protocol
Pouring Gel
Measure our an appropriate amount of agarose.
1% agarose gel = 0.5 g agarose in 50 mL of 1x TAE
Dissolve gel in 1X TAE to final volume desired.
Microwave the solution until all the agarose has dissolved.
Watch carefully to ensure that the contents do not boil over.
Let dissolved gel cool until you can hold it, but not too cool that it solidifies.
Once cool enough to handle add 2.5 μl Gel Red™ (0.5x Concentration) to the solution and swirl to dissolve (0.5-1x concentration works best to visualize samples)
It is important to add the Gel Red™ while the solution is still warm to allow for proper mixing, but not too hot as to denature the dye.
Pour the solution into the gel rig such that the gel will solidify. Add comb to create wells to be loaded.
Let cool until solid, approximately 15 minutes.
Loading Gel
Add appropriate amount of gel loading dye to your DNA sample.
1μl dye to 5μl DNA is sufficient for PCR products and restriction digests.
Load samples and Molecular weight marker.
3-4μl is sufficient to visualize.
Record the order of which the samples were loaded.
Run the Gel at 120V-140V until the dye is at least 3/4 way down the gel.
Make sure the DNA samples will run the correct direction. DNA is negatively charged and will migrate towards the cathode (the phrase “Run to Red” refers to DNA samples migrating to the red cathode on the gel rig).
Do not run the gel too fast (≥140V). At faster speeds the resolution of the bands of DNA decreases.
Once gel is complete image with UV light.
Ligation Independent Cloning Techniques (PIPE and SLIC)
Protocol adapted from “A Practical Comparison of Ligation-Independent Cloning Techniques” by Julian Stevenson, James R. Krycer, Lisa Phan, Andrew J. Brown
Materials
Reagents
Vector primers
Insert primers (overhang to vector)
Vector DNA (known concentration)
Insert DNA (known concentration)
PCR Components
DNA polymerase (Q5 high fidelity DNA Polymerase from NEB)
5x Reaction buffer (comes with DNA polymerase)
10 mM dNTPs
Distilled Water (ddH2O)
Enhancer (optional)
Cloning and transformation
DpnI and buffer
Chemically competent cells
LB+Antibiotic plates
Protocol
For Polymerase Incomplete Primer Extension (PIPE) cloning
Combine PCR components in a PCR tube.
Component
Volumes for 25 μL reaction
Volumes for 50 μL reaction
Final Concentration
5x reaction buffer
5 μL
10 μL
1x
25 μM Forward Primer
0.5 μL
1 μL
0.5 μM
25 μM Reverse Primer
0.5 μL
1 μL
0.5 μM
Template DNA
1 μL of 1 ng/μL
2 μL of 1 ng/μL
0.04 ng/μL
10 mM dNTPs
0.5 μL
1 μL
200 μM
DNA polymerase
0.25 μL
0.5 μL
0.02 U/μL
Enhancer (optional)
5 μL
10 μL
1x
ddH2O
Up to 25 μL
Up to 50 μL
Mix components well, spin down if necessary.
Run PCR cycle conditions
Step
Temperature
Time
Initial denaturation
98˚C
30 seconds
30 cycles
98˚C
72˚C
72˚C
10 seconds
10 seconds
20-30 seconds/kb
Infinite hold
4˚C
Indefinite
Run an agarose gel on the PCR sample to check for successful amplification.
If there is high off target amplification, you can gel purify the right size DNA fragment and proceed with protocol.
Combine 5 μL of Insert DNA PCR products to 5 μL of Vector DNA PCR products. Also digest vector only and insert only samples to act as controls for transformation efficiency.
Add 10 U (0.5 μL) of DpnI and CutSmart NEB buffer to a final concentration of 1x. Mix well.
Incubate at 37˚C for 2 hours.
Add 6 μL of cloned and control DNA to separate vials of competent cells. Follow manufacturer's transformation protocol.
For Sequence and Ligation-Independent Cloning (SLIC)
PCR cleanup products before DpnI digest.
After DpnI digestion add 0.75 U of T4 DNA polymerase and incubate at 25˚C for 5-10 min the immediately put on ice to stop the reaction.
Transform competent cells according to manufacturer's directions.
Notes
T4 DNA polymerase acts as an exonuclease in the absence of dNTPs. For this reason PCR clean-up is extremely important.
Do not permit the T4 chew back reaction to progress too long because it will degrade the DNA and take away the designed overhangs.
Troubleshooting
Problem
Reason
Solution
High Background
Too much of one DNA sample to ligate to its partner
Change the ratio of Vector: Insert depending on where the background is arising from. Typically high background is seen in the vector due to conserved antibiotic resistance. Increase the amount of insert added to the reaction to force ligation with the vector. Ratios range from 1:2-5. You can compare the molar ratios to get exact ratios.
No recombinants
Ligation of the vector and insert did not occur
During PCR full length, blunt products can be created which would inhibit the reaction. Either shorten the PCR cycles or/elongation length/ try SLIC cloning to manually create the overhangs.
Weak/no PCR product observed
Not enough target DNA added
Not enough primer added
Other
Make sure you are adding all the ingredients for PCR and at the right concentrations. Make sure all ingredients are thawed and mixed well.
If all else fails, make fresh solutions of DNA, primers, and dNTPs.
Restriction Digests
Materials
DNA
Restriction enzymes
Buffer
Sterile Water
Protocol
Mix all components as follows:
Reagent\Reaction
Uncut
Single Cut
Double Cut
DNA
5 μl
5 μl
5 μl
Restriction Enzyme
0 μl
1 μl
1 μl of each enzyme
Buffer
2 μl
2 μl
2 μl
Water
13 μl
12 μl
11 μl
Final Volume
20 μl
20 μl
20 μl
Digest samples at 37˚C for 1 hour.
Run samples on a gel to see results of digest.
Notes
Online tools, like NEB cutter, can be used to determine what restriction enzymes to use.
If cutting is not occurring/only some of the DNA is getting cut it is possible that too much DNA is added and the activity of the enzyme prevents it from cutting all of the DNA. Redo the digest with less DNA.
Site-Directed Mutagenesis (via PIPE)
Materials
See PIPE cloning protocol.
Primer Design
Design Primers as you would for PIPE with the overhangs containing the DNA to be mutated and the primers facing to amplify the vector.
Make sure the point mutation is in the middle of the overhangs.
Protocol
Follow PIPE cloning protocol for PCR.
Check for proper amplification on a gel.
DpnI digest the PCR product.
Transform following manufacturer's directions.
Check recombinants via restriction digest (if a restriction site was destroyed) or sequencing.
Acid Shock Assay
Materials
LB Broth, pH 3, 4, 5, and 7 (record actual pH)
T7 cells
Spectrophotometer
Culture tubes
Eppendorf Tubes
Protocol
Grow overnight of T7 cells.
Dilute 100 μl of T7 Cells in 4 mL of Neutral pH LB.
Separate tubes for each pH LB (4 total).
Grow to an OD600 of approximately 0.7 (record exact OD).
Take 100 μl aliquot and add 900 μl neutral pH LB, and plate (1:100 dilution).
Pellet 4 mL.
Resuspend in 1 mL of appropriate pH LB.
Add to 3 mL of appropriate pH LB.
Record OD
Take 100 μl aliquot and add 900 μl neutral pH, and plate (1:100 dilution).
Take OD reading every 30 minutes.
Take 100 μl aliquot and add 900 μl neutral pH, and plate (1:100 dilution).
Take time points until OD levels off for 1.5 hours.
Count colonies on plates to see the survival of the cells throughout the time course.
Aciduricity Test
Materials
LB Broth, pH 3 and 7 (record actual pH)
T7 cells
Untransformed cells and transformed cells
Spectrophotometer
Culture tubes
Eppendorf Tubes
LB agar plates
With appropriate antibiotic if necessary
Protocol
Grow overnight of cell lines to be tested.
Dilute 100 μl of cells in 4 mL of Neutral pH LB
Measure OD600 until approximately 0.5 and induce the expression of gene in transformed cell line.
0.5 mM - 5 mM IPTG for lac promoters
At OD600 0.7 take 1 μl aliquot and add 99 μl of neutral pH LB and plate (1:100 dilution).
Record the exact OD from when aliquot was taken. This is T0
Pellet all 4 mL of remaining cell culture.
Resuspend in 1 mL of appropriate LB pH 3.
Add resuspension to culture tube with 3 mL of pH 3.
Record the OD and exact time after resuspension and an aliquot to be plated (see step 4). This is Ti.
Every 30 minutes record the OD and take an aliquot to plate (see step 4).
a. Keep culture going for 4-5 hours (T1-10).
Count colonies on the plates the next day.
Notes
A blank needs to be taken at each time point to account for drift in the spectrophotometer.
Each pH LB needs its own blank as well as the appropriate antibiotic if necessary.
Once done with assay cells need to be killed with Bleach or Lysol before being thrown out.
Notebook
Safety
