Team:SIAT-SCIE/Protocol
Home
Project
Description Design Results
Lab
iGEM Diary Protocol Safety
Parts
Parts Overview Primer
Modeling
Team
About us Attribution Collaborations
Human Practices
InterLab
PCR
Primestar Max PCR protocol — for plasmid
1. General Composition of PCR Reaction Mixture
NOTE: When using plasmids as the template, the mass of the mixture should not exceed 1ng.
2. PCR condition
Before cycles: 98°C 10 After cycles: 72°C 7min; 16°C ∞ (4°C is more suitable for storing PCR product, but storing 4°C overnight may result in damage of some PCR equipments)
(A) For reactions in which the quantity of template is 200 ng / 50 μl or less:*
-98°C 10 sec.
-55°C 5 sec. or 15 sec
-72°C 5 sec./kb
-30-35 cycles
(B) For reactions in which the quantity of template exceeds 200 ng / 50 μl:*
-98°C 10 sec.
-55°C 5 sec. or 15 sec.
-72°C 30 - 60 sec./kb
-30-35 cycles
OR
-98°C 10 sec.
-68°C 30 - 60 sec./kb
-30 - 35 cycles [2-step PCR]
NOTE: For amplifying fragments more than 3kb, extension time should be increased to 40 sec./kb
・Denaturing conditions: 98°C for 5 to 10 sec. is recommended. If performing denaturation at 94°C, set the denaturation step for 10 to 15 sec.
・Annealing temperature: Use 55°C as the default annealing temperature.
Annealing time:
For primers that are 25-mer or shorter:
-For primer Tm values (calculated by the formula below) of 55°C or greater, anneal for 5 sec.
-For primer Tm values (calculated by the formula below) less than 55°C, anneal for 15 sec.
For primers longer than 25-mers:
-Use an annealing time of 5 sec.
* Tm value calculation method:
Tm (°C) = 2(NA + NT) + 4(NC + NG) - 5
where N represents the number of primer nucleotides having the specified identity (A, T, C, or G)
TaKaRa Premix Taq PCR protocol — for bacterial colony
1. General reaction mixture for PCR
Premix Taq    25ul
Template    <500ng>
Primer 1      0.1-1.0uM(final conc.)
Primer 2    0.1-1.0uM(final conc.)
Sterilized distilled water     up to 50ul
When using bacterial colony as the template, we take a very small sample (a dip) from the colony
2. PCR conditions
Before cycles: 98°C 10;
After cycles: 72°C 7min; 16°C ∞
When amplifying a 1 kb DNA fragment
-98°C  10sec.
-55°C  30 sec.
-72°C  1 min.
-30  cycles
OR:
-98°C  10 sec.
-68°C  1min.
-30  cycles
Recombination
Set up the following reaction on ice. Spin briefly to bring the sample to the bottom of the tube before reaction
ddH20     up to 20ul
5x CE II Buffer      4ul
Linearized Vector     50ng-200ng
Amplified Insert     20ng-200ng
Exnase II     2ul
The optimal amount of vector for recombination is 0.03 pmol. The optimal molar ratio of vector to insertion is 1:2, which means the optimal amount of insert for recombination is 0.06 pmol. Their amount in molar can be roughly calculated according to the following formula:
The amount of vector required = [0.02 × number of base pairs] ng (0.03 pmol)
The amount of insert required = [0.04 × number of base pairs] ng (0.06 pmol)
For example, when cloning an insert of 2 kb to a vector of 5 kb, the optimal amount of vector is 0.02 × 5000 = 100 ng, and the optimal amount of insert is 0.04 × 2000 = 80 ng.
After finishing setting up, gently pipette for several times to mix thoroughly and avoid bubbles. DO NOT VOTEX! Incubate at 37°C for 30 min and immediately place the tube on ice for 5 min. The recombination products is now ready for transformation or storage at -20°C for future use.
A PCR thermo-cyclers or water bath are recommended for the reaction. The recombination efficiency can reach its peak at 30 min. Longer or shorter reaction time will decrease on the cloning efficiency.
Plasmid extraction Miniprep
1. Spin the cell culture in a centrifuge to pellet the cells, empty the supernatant (media) into a waste collection container.
2. Resuspend pelleted bacterial cells in 250 μl Buffer P1 (kept at 4 °C) and transfer to a microcentrifuge tube. No cell clumps should be visible after resuspension of the pellet.
Important: Ensure that RNase A has been added to Buffer P1.
3. Add 250 μl Buffer P2 and gently invert the tube 4–6 times to mix. Do not vortex, as this will result in shearing of genomic DNA. If necessary, continue inverting the tube until the solution becomes viscous and slightly clear.
Do not allow the lysis reaction to proceed for more than 5 min.
4. Add 350 μl Buffer N3 and invert the tube immediately and gently 4–6 times.
To avoid localized precipitation, mix the solution gently but thoroughly, immediately after addition of Buffer N3. The solution should become cloudy.
5. Centrifuge for 10 min at 13,000 rpm (~17,900 x g) in a table-top microcentrifuge. A white pellet will form.
6. Apply the supernatants from step 4 to the QIAprep spin column by decanting or pipetting.
7. Centrifuge for 30–60 s. Discard the flow-through.
8. Wash QIAprep spin column by adding 0.75 ml Buffer PE and centrifuging for 30–60 s. Spinning for 60 seconds produces good results.
9. Discard the flow-through, and centrifuge for an additional 1 min to remove residual wash buffer.
IMPORTANT: Residual wash buffer will not be completely removed unless the flow-through is discarded before this additional centrifugation. Residual ethanol from Buffer PE may inhibit subsequent enzymatic reactions. They are right about this.
10. Open the lid and wait for 10 min for residual ethanol from Buffer PE to evaporate completely.
11. Place the QIAprep column in a clean 1.5 ml microcentrifuge tube. To elute DNA, add 50 μl water to the center of each QIAprep spin column, let stand for 1 min, and centrifuge for 1 min.
Adapted from http://parts.igem.org/Help:Protocols/Miniprep
Agarose Gel Electrophoresis
With reference to the corresponding protocol provided by Joseph Sambrook and David W. Russell. 'Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, USA, 2001'
INTRODUCTION
How to pour, load, and run an agarose gel.
MATERIALS
6x Gel-loading buffer
Agarose solutions (please see Step 3)
DNA samples
DNA size standards
Samples of DNAs of known size are typically generated by restriction enzyme digestion Of a plasmid or bacteriophage DNA of known sequence. Alternatively, they are produced by ligating a monomer DNA fragment of known size into a ladder of polymeric
DNA staining solution
Electrophoresis buffer
METHOD
1. Seal the edges of a clean, dry glass plate (or the open ends of the plastic tray supplied with the electrophoresis apparatus) with tape to form a mold.
Set the mold on a horizontal section ofthe bench.
2. Prepare sufficient electrophoresis buffer (Ix TAE) to fill the electrophoresis tank and to cast the gel.
It is important to use the same batch of electrophoresis buffer in both the electrophoresis tank and the gel.
3. Prepare a solution of 1xTAE buffer. Add 1% powdered agarose to measured quantity of 1xTAE buffer in a flask.
4. Loosely plug the neck of the Erlenmeyer flask with Kimwipes. If using a glass bottle, make certain the cap is loose. Heat the slurry in a microwave oven until the agarose dissolves.
5. Use cool water to wash the flask. When the molten gel has cooled, add 1/10000 volume of SYBR-Safe stain.
6. While the agarose solution is cooling, choose an appropriate comb for forming the sample slots in the gel. Position the comb 0.5-1.0 mm above the plate so that a complete well is formed when the agarose is added to the mold.
7. Pour the warm agarose solution into the mold. The gel should he between 3 mm and 5 mm thick. Check that no air bubbles are under or between the teeth Of the comb. Air bubbles present in the molten gel can be removed easily by poking them with the corner of a Kimwipe.
8. Allow the gel to set completely (30-45 minutes at room temperature), then pour a small amount of electrophoresis buffer on the top of the gel, and carefully remove the comb. Pour off the electrophoresis buffer and carefully remove the tape. Mount the gel in the electrophoresis tank.
9. Add just enough electrophoresis buffer to cover the gel to a depth of approx. 1 mm.
10. Mix the samples of DNA with 0.50 volume of the desired 6x gel-loading buffer. The maximum amount of DNA that can be applied to a slot depends on the number of fragments in the sample and their sizes.
11. Slowly load the sample mixture into the slots of the submerged gel using a disposable micropipette. Load size standards into slots on both the right and left sides of the gel.
12. Attach the electrical leads so that the DNA will migrate toward the positive anode (red lead). Apply a voltage of 120V. If the leads have been attached correctly, bubbles should be generated at the anode and cathode (due to electrolysis), and within a few minutes, the bromophenol blue should migrate from the wells into the body of the gel. Run the gel until the bromophenol blue and xylene cyanol FF have migrated an appropriate distance through the gel.
13. When the DNA samples or dyes have migrated a sufficient distance through the gel, turn off the electric current and remove the leads and lid from the gel tank.
PCR purification
—QIAquick® PCR Purification Kit
Notes before starting
• Add ethanol (96–100%) to Buffer PE before use (see bottle label for volume).
• All centrifugation steps are carried out at 17,900 x g (13,000 rpm) in a conventional table-top microcentrifuge at room temperature.
• Add 1:250 volume pH indicator I to Buffer PB. The yellow color of Buffer PB with pH indicator I indicates a pH of ≤7.5. If the purified PCR product is to be used in sensitive microarray applications, it may be beneficial to use Buffer PB without the addition of pH indicator I. Do not add pH indicator I to buffer aliquots.
1. Add 5 volumes Buffer PB to 1 volume of the PCR reaction and mix. Optional: If the color of the mixture is orange or violet, add 10 μl 3 M sodium acetate, pH 5.0, and mix. The color of the mixture will turn yellow.
2. Place a QIAquick column in a provided 2 ml collection tube or into a vacuum manifold. For details on how to set up a vacuum manifold, refer to the QIAquick Spin Handbook.
3. To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s until all the samples have passed through the column. Discard flow-through and place the QIAquick column back in the same tube.
4. To wash, add 0.75 ml Buffer PE to the QIAquick column centrifuge for 30–60 s.Discard flowthrough and place the QIAquick column back in the same tube.
5. Centrifuge the QIAquick column once more in the provided 2 ml collection tube for 1 min to remove residual wash buffer.
6. Place each QIAquick column in a clean 1.5 ml microcentrifuge tube.
7. Open the cap and wait for 10 min for residual ethanol from Buffer PE to evaporate completely.
8. To elute DNA, add 50 μl water (pH 7.0– 8.5) to the center of the QIAquick membrane and stand for at least 10 min. Centrifuge the column for 1 min. For increased DNA concentration, add 30 μl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge.
To see Full Version of the Protocol, click the link for download
DOWNLOAD
