<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SIAT-SCIE_Project_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SIAT-SCIE/Project Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="const" style="padding-left:-30px;padding-right:-30px;"><DIV id="nav"><UL><LI><A href="https://2018.igem.org/Team:SIAT-SCIE/Home" title="Home" style="font-family:'Avenir';">Home</A></LI><LI class="dropdownlist"><A href="javascript:void(0)" class="dropbtn" style="font-family:'Avenir';">Project</A><DIV class="dropdown_content"><A href="https://2018.igem.org/Team:SIAT-SCIE/Project_Description" style="font-family:'Avenir';" alt="Description">Description</A><A href="https://2018.igem.org/Team:SIAT-SCIE/Design" style="font-family:'Avenir';" alt="Design">Design</A><A href="https://2018.igem.org/Team:SIAT-SCIE/Results" style="font-family:'Avenir';" alt="Results">Results</A></DIV></LI><LI class="dropdownlist"><A href="javascript:void(0)" class="dropbtn" style="font-family:'Avenir';">Lab</A><DIV class="dropdown_content"><A href="https://2018.igem.org/Team:SIAT-SCIE/IGEM_Diary" style="font-family:'Avenir';" alt="iGEM Diary">iGEM Diary</A><A href="https://2018.igem.org/Team:SIAT-SCIE/Protocol" style="font-family:'Avenir';" alt="Protocol">Protocol</A><A href="https://2018.igem.org/Team:SIAT-SCIE/Safety" style="font-family:'Avenir';" alt="Safety">Safety</A></DIV></LI><LI class="dropdownlist"><A href="javascript:void(0)" style="font-family:'Avenir';" class="dropbtn">Parts</A><DIV class="dropdown_content"><A href="https://2018.igem.org/Team:SIAT-SCIE/Parts_Overview" style="font-family:'Avenir';" alt="Parts Overview">Parts Overview</A><A href="https://2018.igem.org/Team:SIAT-SCIE/Primer" style="font-family:'Avenir';" alt="Primer">Primer</A></DIV></LI><LI><A href="https://2018.igem.org/Team:SIAT-SCIE/Modeling" style="font-family:'Avenir';" title="Modeling">Modeling</A></LI><LI class="dropdownlist"><A href="javascript:void(0)" style="font-family:'Avenir';" class="dropbtn">Team</A><DIV class="dropdown_content"><A href="https://2018.igem.org/Team:SIAT-SCIE/About_us" style="font-family:'Avenir';" alt="About us">About us</A><A href="https://2018.igem.org/Team:SIAT-SCIE/Attributions" style="font-family:'Avenir';" alt="Attribution">Attribution</A><A href="https://2018.igem.org/Team:SIAT-SCIE/Collaborations" style="font-family:'Avenir';" alt="Collaborations">Collaborations</A></DIV></LI><LI><A href="https://2018.igem.org/Team:SIAT-SCIE/Human_Practices" style="font-family:'Avenir';" title="Human Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:SIAT-SCIE/InterLab" style="font-family:'Avenir';" title="InterLab">InterLab</A></LI></UL></DIV></DIV><DIV class="chassis"><H1>Outer Membrane vesicles</H1><P style="font-size: 20px">Outer membrane vesicles (OMVs) are vesicles produced and used as common vehicles in the world of gram-negative-bacteria. Despite their ubiquity, they have been grievously overlooked in the past; budding out as spherical containers of 20 to 500 nm in diameter from the bacterial membrane, they are potentially capable of transporting a wide array of biomolecules that awaits the academia to divulge. As potent transporters, OMVs play an integral role in various biological phenomena, ranging from stress regulations to microbial interactions (1).</P><P style="font-size: 20px"> Seeing that the unique properties of OMVs may revolutionise traditional delivery system, our team aims to devise a technique that incorporates the wonders of OMVs and the very frontier technology in genetic engineering — Cas9 proteins — to form a OMV-CRISPR-Cas9 system, which is capable of delivering Cas-9 proteins to target the bacterial genome of bacteria inside mammals’ bodies. 
         </P><P style="font-size: 20px">As natural kins to bacterial cell membranes, OMVs can be degraded easily while preserving the shape and bioactivity of sensitive Cas9 proteins within, as well as single guide RNAs (sg-RNAs). We expect this technique would open up new possibilities of in vitro genetic engineering, thus providing substantial aid in curing and preventing illnesses such as inflammatory bowel diseases by removing virulence genes from malignant bacteria with this technique.</P><H1>Fusobacterium nucleatum</H1><P style="font-size: 20px">Our project aims to test the efficiency of our system by applying it to knock out fadA gene of Fusobacterium nucleatum, a strain of bacteria that reside in human alimentary canal. </P><P style="font-size: 20px">F. nucleatum is a gram-negative bacterium prevalently found in mammal oral cavity, and is frequently associated with oral inflammation diseases and various cancers (2).  Its virulence stems from its invasion of the human epithelial cells, which induces oncogenic gene expression. The main genetic culprit of its pathogenicity is its fadA gene, which is an adhesion virulence factor that insures the binding of F.nucleatum to the host epithelial cell, thereby enabling the bacterium to invading the host and causing inflammation and cancer. </P><P style="font-size: 20px">F. nucleatum’s fadA gene is a congenial candidate for our project since, being gram-negative, F.nucleatum possesses abundant OMVs that are central to our designed system. Through knocking out the fadA gene, we are enabled to observe the efficacy of our system’s application in realistic setting.</P></DIV><ARTICLE><DETAILS><SUMMARY>Mechanism of FadA protein</SUMMARY><P style="text-indent: 20px;font-size: 20px; padding: 10px 10px 10px 10px;margin:40px 200px 0px 200px">The FadA protein is activated when its two forms combine and become internalized. The first form is a pre-FadA that is anchored in the cell membrane, whereas the second form is the mature FadA (mFadA) that is secreted out of F. nucleatum. When the two forms combine to form a complex, the protein is capable to help F. nucleatum bind to the host epithelial cell, thus allowing F. nucleatum embark on invading the host cells.</P><H3 style="margin: 20px 200px 30px 200px;font-size:30px">Overview (Fig. 1)</H3><P style="text-indent: 20px;font-size: 20px; padding: 10px 10px 10px 10px;margin:0px 200px 0px 200px">We will first express Cas9 and sgRNA in E.coli and then transport them to E.coli’s periplasm(Step 1). By then, they may be packaged by OMVs that bud off from E.coli’s outer membrane(Step 2). Those OMVs will be collected and mixed with our target bacteria, thereby allowing them to fuse with bacteria again, to release the Cas9 proteins and sgRNA(Step 3), and cleave the target gene (Step 4 &amp; 5).  Afterwards, we will test whether the target gene is cleaved by Cas9.
             </P></DETAILS></ARTICLE><DIV class="chassis"><H1>Safety</H1><P style="font-size: 20px">Out of safety concerns, instead of using the pathogenic Fusobacterium nucleatum, we transform a section of FadA’s coding sequence — with sgRNA’s binding site — into E.coli to test our system’s efficiency.</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>