<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SIAT-SCIE_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SIAT-SCIE/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="const" style="padding-left:-30px;padding-right:-30px;"><DIV id="nav"><UL><LI><A href="https://2018.igem.org/Team:SIAT-SCIE/Home" title="Home" style="font-family:'Avenir';">Home</A></LI><LI class="dropdownlist"><A href="javascript:void(0)" class="dropbtn" style="font-family:'Avenir';">Project</A><DIV class="dropdown_content"><A href="https://2018.igem.org/Team:SIAT-SCIE/Project_Description" style="font-family:'Avenir';" alt="Description">Description</A><A href="https://2018.igem.org/Team:SIAT-SCIE/Design" style="font-family:'Avenir';" alt="Design">Design</A><A href="https://2018.igem.org/Team:SIAT-SCIE/Results" style="font-family:'Avenir';" alt="Results">Results</A></DIV></LI><LI class="dropdownlist"><A href="javascript:void(0)" class="dropbtn" style="font-family:'Avenir';">Lab</A><DIV class="dropdown_content"><A href="https://2018.igem.org/Team:SIAT-SCIE/IGEM_Diary" style="font-family:'Avenir';" alt="iGEM Diary">iGEM Diary</A><A href="https://2018.igem.org/Team:SIAT-SCIE/Protocol" style="font-family:'Avenir';" alt="Protocol">Protocol</A><A href="https://2018.igem.org/Team:SIAT-SCIE/Safety" style="font-family:'Avenir';" alt="Safety">Safety</A></DIV></LI><LI class="dropdownlist"><A href="javascript:void(0)" style="font-family:'Avenir';" class="dropbtn">Parts</A><DIV class="dropdown_content"><A href="https://2018.igem.org/Team:SIAT-SCIE/Parts_Overview" style="font-family:'Avenir';" alt="Parts Overview">Parts Overview</A><A href="https://2018.igem.org/Team:SIAT-SCIE/Primer" style="font-family:'Avenir';" alt="Primer">Primer</A></DIV></LI><LI><A href="https://2018.igem.org/Team:SIAT-SCIE/Modeling" style="font-family:'Avenir';" title="Modeling">Modeling</A></LI><LI class="dropdownlist"><A href="javascript:void(0)" style="font-family:'Avenir';" class="dropbtn">Team</A><DIV class="dropdown_content"><A href="https://2018.igem.org/Team:SIAT-SCIE/About_us" style="font-family:'Avenir';" alt="About us">About us</A><A href="https://2018.igem.org/Team:SIAT-SCIE/Attributions" style="font-family:'Avenir';" alt="Attribution">Attribution</A><A href="https://2018.igem.org/Team:SIAT-SCIE/Collaborations" style="font-family:'Avenir';" alt="Collaborations">Collaborations</A></DIV></LI><LI><A href="https://2018.igem.org/Team:SIAT-SCIE/Human_Practices" style="font-family:'Avenir';" title="Human Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:SIAT-SCIE/InterLab" style="font-family:'Avenir';" title="InterLab">InterLab</A></LI></UL></DIV></DIV><DIV class="chassis"><H1>Chassis</H1><P style="font-size: 20px">In our project, we used E.coli BW25113 strain with OmpA gene knocked out for the reason that the absence of OmpA protein would lead to hyper-vesiculation, meaning increased production of outer membrane vesicles (OMVs).</P><P style="font-size: 20px">Inside the periplasm there is a thin, rigid peptidoglycan(PG) layer attached to both outer membrane and cytoplasmic membrane by membrane anchored proteins.[Fig. 1](1) Deletion or truncation of OmpA, an abundant protein linking the outer membrane and peptidoglycan layer, will therefore result in increased vesiculation in E. coli, Salmonella, and Vibrio cholerae (2) as the Figure 1 lack of OmpA destabilises the periplasm’s structure.This will result in greater likelihood that our Cas9 protein will be encapsulated into OMVs.
         </P><P style="padding-left:100px; padding-right:100px;font-size: 15px"><B>Figure 1. A section of gram-negative’s periplasm.</B> OmpA is a porin anchored in the outer membrane. OmpA links to the PG layer, which is gram-negative bacteria’s cell wall. Absence of OmpA will destabilize periplasm’s structure, thereby promoting the formation of OMVs.</P></DIV><DIV class="chassis"><H1 style="font-size:30px">Importing Proteins into Periplasm Through TAT Pathway
         </H1><P style="font-size: 20px;">To allow proteins to be encapsulated in OMVs, they need to be imported into bacteria’s periplasm first. Compared to other common transporting pathways that transport proteins before folding (e.g. Sec and SRP-dependent pathways), twin arginine transport (TAT) pathway is an available option for transportation of fully folded proteins(3). This is crucial since some proteins, including GFP, cannot fold properly when transported into periplasm(3). 
         </P><P style="font-size: 20px">Thus, they have to complete their folding in cytoplasm before they can be transported to periplasm. Consequently, TAT pathway is the only plausible option[Fig. 2]. Regarding the Cas9 proteins, since there has been no literature discussing the possibility of their possible
             Figure 2 of folding in periplasm, we chose TAT pathway for Cas9 proteins as well out of prudence. 
         </P><P style="padding-left:100px; padding-right:100px;font-size: 18px"><B>Figure 2. Exporting GFPs to periplasm through Sec and Tat pathway. </B>Sec pathway(on the left) export proteins before they fold. Some proteins, such as GFP, cannot fold into correct shape in periplasm, thus cannot function if Sec pathway is used. In contrast, Tat pathway(on the right) export proteins after they fold properly in cytoplasm. By doing so, functional GFP can be transported to periplasm and be packaged into OMVs (An OMV is about to form at the outer membrane’s bending site). TorA(yellow box on the right) helps the export of proteins through Tat pathway.</P></DIV><SECTION style="margin-top:-130px"><ARTICLE><DETAILS><SUMMARY>Testing TorA Signal Peptide Efficiency</SUMMARY><P style="text-indent: 20px;font-size: 20px; padding: 10px 10px 10px 10px;margin:40px 200px 0px 200px">TorA is one of the signal peptides that trigger TAT pathway. The addition of TorA to the recombinant proteins can enable the transportation of protein to periplasm.</P><P style="text-indent: 20px;font-size: 20px; padding: 10px 10px 10px 10px;margin:0px 200px 0px 200px">In order to test TorA’s working efficiency, we first fused it with GFPmut3 for more convenient reading. Then we transformed plasmids with TorA-GFP into E.coli.  After cultivating the transformed E.coli overnight, osmotic shock assay would be carried out to separate cytoplasm and periplasm, thereby enabling a direct reading of the latter as high degree of fluorescent light in the periplasm indicates the robustness and efficiency of TorA. 
             </P><DIV class="chassis" style="padding-left: 200px"><H3>Four sets of samples are needed:
            </H3><UL><LI>WT with GFP
                </LI><LI>ΔOmpA with GFP
                </LI><LI>WT with TorA-GFP
                </LI><LI>ΔOmpA with TorA-GFP
                </LI></UL></DIV><P style="text-indent: 20px;font-size: 20px; padding: 10px 10px 10px 10px;margin:0px 200px 0px 200px;line-height: 25px">We used plate reader to measure the fluorescence value of both periplasmic and cytoplasmic fractions and calculate their fluorescent ratio (periplasm : cytoplasm) in order to take background noises into account. We expect the ratio to be approximately sample 3＞sample 4＞sample 2≈sample1, for the bacteria with TorA-GFP should have a larger amount of GFP in periplasm. Lower fluorescence reading in sample 4 than sample 3 indicates higher yield of OMVs in ΔOmpA strain as more GFP are encapsulated by OMVs and leave periplasm.
                </P></DETAILS></ARTICLE></SECTION><DIV class="chassis"><H1>CjCas9</H1><P style="font-size: 20px;">Since OMVs generally have a limited capacity as they only have a size ranging from 50nm to 200nm, proteins should be as small as possible for them to be encapsulated into OMVs. Recently characterized cjCas9(derived from Campylobacter.jejuni), with a sequence length of 2.95Kbp making it one of the smallest Cas9 orthologues, became our first choice. Moreover, cjCas9 has a high specificity, cleaving only a limited and particular number of sites in human and mouse genome (4), thereby making it a safer candidate for its application in mammals..</P></DIV><SECTION style="margin: 20px;padding-top:0px"><ARTICLE><DETAILS><SUMMARY>sgRNA</SUMMARY><H2>PAM</H2><P style="font-size: 20px;line-height: 25px">Due to conflicting reports on the PAM sequence of cjCas9, we adopted the most up-to-date results: 5‘-NNNVRYM-3’ PAM with the preference for T and C at positions 6 and 7 (5).  There are many ‘PAM’ sites in FadA. Among them, we chose the one closest to 5’ as the cleaving site, in hope that the introduction of indels caused by cjCas9’s cleavage should disrupt most of FadA’s reading frame, rendering it non-functional.
</P><H2>crRNA</H2><P style="font-size: 20px;line-height: 25px">The CRISPR guide RNA (crRNA) is a type of RNA that enables CRISPR proteins to locate their targets. For an enhanced efficacy and on-target rate at knocking out fadA by our cjCas9, the design of crRNA is very important.</P><P style="font-size: 20px;line-height: 25px"> Unlike most of Cas9 orthologues, cjCas9 recognizes the PAM nucleotides on both the target and non-target DNA strands. Our crRNA is designed to recognized the PAM strand.</P><P style="font-size: 20px;line-height: 25px">We copy the rest of the sgRNA[Fig.3] sequence according to Crystal Structure of the Minimal Cas9 from Campylobacter jejuni Reveals the Molecular Diversity in the CRISPR-Cas9 Systems.</P><P style="font-size: 18px;line-height: 20px;padding-left:100px; padding-right:100px;"><B>Figure 3. Target DNA and sgRNA. </B>The gray DNA strand on top is a single strand of the target DNA(FadA). The red chain, including the gray part at tetraloop, shows our sgRNA’s sequence and structure.</P></DETAILS><DETAILS><SUMMARY>Testing cjCas9’s cleavage efficiency
            </SUMMARY><P style="font-size: 20px;margin-top:20px;line-height: 25px">After cjCas9 recognizing the cleavage site on FadA gene, a double-stranded break shall be introduced to the fadA gene, thus triggering DNA repair through nonhomologous end joining(NHEJ)[Fig.4]. NHEJ causes nearly random insertion and deletion mutations (i.e. indels) at the double-stranded break site, and thus leading to gene knockout (e.g., by causing a shift in the target gene’s reading frame or mutating a critical region of the encoded protein) (6).</P><P style="font-size: 20px;line-height: 25px">To detect FadA’s frameshift, we fused a part of FadA with a mRFP protein using a GILQSTVPRARNP linker (7), creating a FadA-linker-mRFP segment. If cjCas9 successfully cleaved mRFP and resulted in a frameshift, mRFP would cease to functioning, thereby weakening the fluorescence. Subsequently, sequencing was carried out to test if cjCas9 did cause a mutation that does not affect the reading frame.</P><P style="font-size: 18px;line-height: 20px"><B>Figure 4. DNA repair through Nonhomologous end joining (NHEJ)</B>Having cleaved by Cas9 protein(not shown), a double strand break (DSB) will be created. It may leads to NHEJ. NHEJ will probably result in deletion(not shown) or addition of nucleotide pairs, disrupting the gene’s reading frame, causing serious mutations. If the deletion or addition nucleotides happens to be the multiple of three, it may also cause a serious mutation though it won't disrupt the reading frame of the gene.</P></DETAILS><DETAILS><SUMMARY>Biobrick</SUMMARY><P style="line-height: 30px;font-size: 25px;padding-top:20px;margin-top: 30px;line-height: 30px"><SPAN class="BoldLarge">CjCas9</SPAN>[Fig. 5] is under the control of ptet-tetR promoter. The backbone consists of a pSC101 origin (yellow) and a Chloramphenicol resistant gene (light green).</P><P style="line-height: 30px;font-size: 25px;padding-top:20px;margin-top: 30px;line-height: 30px"><SPAN class="BoldLarge">sgRNA</SPAN>[Fig. 6] is under the control of J23100 promoter. It is inserted in pSB1A3 vector.</P></DETAILS></ARTICLE></SECTION><DIV class="chassis"><H1>Reference</H1><P style="font-size: 15px">1. Schwechheimer, C., &amp; Kuehn, M. J. (2015). Outer-membrane vesicles from Gram-negative bacteria: biogenesis and functions. Nature Reviews Microbiology, 13(10), 605–619. doi:10.1038/nrmicro3525.</P><P style="font-size: 15px">2. Kulp, A., &amp; Kuehn, M. J. (2010). Biological Functions and Biogenesis of Secreted Bacterial Outer Membrane Vesicles. Annual Review of Microbiology, 64(1), 163–184. doi:10.1146/annurev.micro.091208.073413.</P><P style="font-size: 15px">3. Engineering of bacterial outer membrane vesicles: Potential applications for the development of vaccines.</P><P style="font-size: 15px">4. Kim, E., Koo, T., Park, S. W., Kim, D., Kim, K., Cho, H.-Y., … Kim, J.-S. (2017). In vivo genome editing with a small Cas9 orthologue derived from Campylobacter jejuni. Nature Communications, 8, 14500. doi:10.1038/ncomms14500.</P><P style="font-size: 15px">5. Yamada, M., Watanabe, Y., Gootenberg, J. S., Hirano, H., Ran, F. A., Nakane, T., … Nureki, O. (2017). Crystal Structure of the Minimal Cas9 from Campylobacter jejuni Reveals the Molecular Diversity in the CRISPR-Cas9 Systems. Molecular Cell, 65(6), 1109–1121.e3. doi:10.1016/j.molcel.2017.02.007.</P><P style="font-size: 15px">6. Wang, H., La Russa, M., &amp; Qi, L. S. (2016). CRISPR/Cas9 in Genome Editing and Beyond. Annual Review of Biochemistry, 85(1), 227–264. doi:10.1146/annurev-biochem-060815-014607.</P><P style="font-size: 15px">7. Yasuda, R., Harvey, C. D., Zhong, H., Sobczyk, A., van Aelst, L., &amp; Svoboda, K. (2006). Supersensitive Ras activation in dendrites and spines revealed by two-photon fluorescence lifetime imaging. Nature Neuroscience, 9(2), 283–291. doi:10.1038/nn1635.</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>