<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HFUT-China_Human_Practices_Software_Features skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HFUT-China/Human Practices/Software Features</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>HFUT-China Software Features</TITLE></P><DIV class="titleBar"><SPAN class="title">Biodesigner Coral</SPAN></DIV><UL class="sf-menu"><LI><A href="https://2016.igem.org/Team:HFUT-China">Home</A></LI><LI><A href="#">Project</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Proof">Validation</A></LI></UL><LI><A href="#">Software</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/UserGuide">User Guide</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Technology">Technology</A></LI><LI><A href="#">Documents</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Api">API</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/UnitTest">Unit Test</A></LI><LI><A href="#">Requirements</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Medals">Medals</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Collaborations">Collaboration</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Attributions">Attribution</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Human_Practices/Project_Review">Project Review</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Human_Practices/Software_Features">Software Features</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Human_Practices/Recapitulation">Recapitulation</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Integrated_Practices">Integrated Human Practice</A></LI><DIV class="mainCenter"><DIV class="main"><DIV class="pull-left"><DIV class="sf_part"><H3>HFUT-China-2015 BioDesigner</H3><UL><LI>Search result will be adjusted according to the function of current design because users can add a function tag to their design.</LI><LI>BioDesigner also provides other useful information including part basic information, related registry page, and related literatures.</LI><LI>Data about related proteins, compounds and genes are shown in a graph.</LI></UL></DIV><DIV class="sf_part"><H3>Michigan-Software-2015 ProtoCat 2.0</H3><UL><LI>ProtoCat 2.0 works to establish such a community by giving anyone with an internet connection or smartphone access to a repository of synthetic biology protocols collected from all over the world.</LI><LI>ProtoCat 2.0 allows users to document trails, rate, review, and edit existing methods, and easily locate related protocols.</LI></UL></DIV><DIV class="sf_part"><H3>USTC-Software-2015 BioBLESS</H3><UL><LI>If the simulation results correspond with the initial truth table, sensitivity analysis and optimization will be done to further maximize circuits’ practical realizablity, and robustness analysis will access their relative immunity to perturbations.</LI><LI>Users are free to see the concentration-time evolution of every substance in designed gene circuits individually.</LI></UL></DIV><DIV class="sf_part"><H3>Toronto-2015 ConsortiaFlux</H3><UL><LI>The visualization is generated through a process called community flux balance analysis (cFBA)</LI><LI>Providing insights in the metabolism of communities with an intuitive user interface.</LI></UL></DIV><DIV class="sf_part"><H3>Michigan-Software-2014 ProtoCat</H3><UL><LI>Users can use a rating system to allow the best protocols to rise to the top.</LI></UL></DIV><DIV class="sf_part"><H3>SYSU-Software-2014 FLAME</H3><UL><LI>Framework-based method is the core of FLAME.</LI><LI>A simulation module is designed for characterization and standardization of synthetic circuits or systems.</LI></UL></DIV><DIV class="sf_part"><H3>UESTC-Software-2014 CRISPR-X</H3><UL><LI>Design CRISPR sgRNA with minimized off-target effects and high cutting rate.</LI><LI>A part is compatible with an assembly standard, as long as its sequence has no sites of relevant restriction enzymes.</LI></UL></DIV><DIV class="sf_part"><H3>SYSU-Software-2013 CAST</H3><UL><LI>Users can design their functional network but not gene circuits thus they can focus on the ideas without the bothering of technical questions.</LI><LI>The plasmid will be presented intuitively and clearly and users can select different assembly standard as they wish.</LI></UL></DIV><DIV class="sf_part"><H3>USTC-Software-2013 gNAP</H3><UL><LI>gNAP is the first software analyzing the infection of importing new exogenous gene into engineered bacteria.</LI></UL></DIV><DIV class="sf_part"><H3>Johns-Hopkins-Software-2012 AutoGene</H3><UL><LI>The processes will be run on the cloud as a standard web server gateway interface service. A user may export their annotated plasmid in any format they'd like: genbank, fasta, or SBOL.</LI></UL></DIV><DIV class="sf_part"><H3>SYSU-Software-2012 BiArkit</H3><UL><LI>Biobrick is localized and transformed into searching kit for users to get comprehensive information conveniently.</LI><LI>The seven parts of our software, BiArkit, have been transferred into App called Clotho, and anyone can create and share new tools in it.</LI></UL></DIV><DIV class="sf_part"><H3>Wellesley-HCI-2012 MoClo planner</H3><UL><LI>The layered workspace allows for efficient and organized transfer of information from one level of the cloning method to another without distracting from the primary workspace.</LI><LI>SynFlo allows users to create useful virtual life forms from standardized genetic components, exploring common synthetic biology concepts and techniques.</LI></UL></DIV><DIV class="sf_part"><H3>ENSPS-Strasbourg-2011</H3><UL><LI>Two simulations will be possible: a behavioral simulation (binary model) will give to the biologist an idea of the behavior of the system; in another simulation biologists will see the different interaction between species, the evolution of the concentrations into the system, and have an accurate idea of the transient states. USTC-Software-2011 Lachesis</LI><LI>It can provide automatic parameter fitting of a mathematical model for a desired behavior, according to experiment data or hand drawn curves using evolution algorithm.</LI></UL></DIV><DIV class="sf_part"><H3>VT-ENSIMAG-Biosecurity-2010 GenoTHREAT</H3><UL><LI>It was developed in accordance with a draft sequence screening guidance published by the United States Government in 2009.</LI></UL></DIV></DIV><DIV class="pull-right"><DIV class="sf_part"><H3>Korea-U-Seoul-2015 Gil</H3><UL><LI>Gil can select a simple path that does not include repeated nodes among all the possible paths and better paths will be shown in comparison with “All shortest paths”</LI><LI>Users can set up a cutoff to determine the depth of search.</LI><LI>When it comes to a set of compounds, information about the maximum twelve paths will be saved into JSON and text format. When it comes to the standard biological parts, after running the Nucleotide BLAST to KEGG GENES data, 565,163 matches were found (with E-value of 1e-5). The 3972 parts were linked to 20,276 gene IDs.</LI></UL></DIV><DIV class="sf_part"><H3>SYSU-Software-2015 CORE</H3><UL><LI>When designing circuits, users will know whether your circuit is safe and the safety of the design can also be evaluated by everyone.</LI><LI>CORE provides protocol managements and experiment schedule.</LI><LI>CORE employs SBOL(v) standard for standardized deposit of genetic design, thus information retrieval and parsing become easier.</LI></UL></DIV><DIV class="sf_part"><H3>UESTC-Software-2015 MCCAP</H3><UL><LI>The construction of minimal gene set starts from the experimentally-determined essential genes.</LI><LI>A new strategy called half-retaining is used to identify essential genes conserved among over half of the reference species.</LI><LI>Users can compare how many differences are there between the two minimal gene sets and see Access number and function of the gene cluster.</LI></UL></DIV><DIV class="sf_part"><H3>AMU-Poznan-2014 Sh-miR designer 2.0</H3><UL><LI>Only the mRNA number which expression should be decreased can be provided.</LI><LI>Include off-target validation and check of immune motifs.</LI><LI>Two applications are provided to access the software, one requires siRNA sequences and the other requires transcript accession number from NCBI database.</LI></UL></DIV><DIV class="sf_part"><H3>SJTU-Software-2014 Easy BBK</H3><UL><LI>With presented biobricks connected with database, the upload form could be directly filled out in &quot;upload&quot;.</LI><LI>Users can also submit their form filled out in the software to the Registry to upload their self-created biobricks.</LI></UL></DIV><DIV class="sf_part"><H3>UCSD-Software-2014 SBiDer</H3><UL><LI>A comprehensive database is built to capture the behavior, composition, and interactions of existing genetic devices in the literature.</LI><LI>Researchers can query the database network for a circuit design expressed as logical operators, and retrieve the sub-network of genetic devices that satisfies the circuit design.</LI></UL></DIV><DIV class="sf_part"><H3>Vanderbilt-Software-2014 Darwin</H3><UL><LI>Produces a formatted file representing each ORF on its own line of text, making each edit only modify a single line of the output text.</LI><LI>Sample a section of every newly inserted ORF and compare it to nearby ORFs; if the new ORF is similar to another ORF, it is counted as “edited”.</LI><LI>Uses concurrency to help speed up the process.</LI></UL></DIV><DIV class="sf_part"><H3>UESTC-2013 Transpeeder</H3><UL><LI>There are two versions of Transpeeder: the online version and the desktop version. In both versions, users can submit an amino acid, a nucleic acid sequence,upload a local file to Transpeeder or choose parts ID of iGEM.</LI></UL></DIV><DIV class="sf_part"><H3>CBNU-Korea-2013 Brick Designer</H3><UL><LI>Brick can be saved as picture file.</LI></UL></DIV><DIV class="sf_part"><H3>SUSTC-Shenzhen-A-212 BBF RFC</H3><UL><LI>Create a default template for storing the information of BioBricks to standardize the minimum information required for a qualified BioBrick.</LI></UL></DIV><DIV class="sf_part"><H3>USTC-Software-2012 REBORN</H3><UL><LI>For the reverse engineering work, REBORN can automatically upload results on to a server and email users with a link,so they can check out the latest results anywhere.</LI></UL></DIV><DIV class="sf_part"><H3>BU-Wellesley-Software-2011</H3><UL><LI>If the software does not work, the design cycle can continue incorporating the knowledge gained at this stage to try to improve the overall process.</LI><LI>In order to integrate the five areas fully together, they are all built upon the Clotho design framework.</LI></UL></DIV><DIV class="sf_part"><H3>USTC-Software-2011 Lachesis</H3><UL><LI>It can provide automatic parameter fitting of a mathematical model for a desired behavior, according to experiment data or hand drawn curves using evolution algorithm.</LI></UL></DIV><DIV class="sf_part"><H3>Freiburg-Software-2009 SynBioWave</H3><UL><LI>It turns Google Wave into a biosynthetic software suite.There are three ways to display sequence data, a short sequence view, a gadget view and a circular view.One additional robot providing the feature of a Blast search was designed to verify it is extendable.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>