<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HFUT-China_Human_Practices_Project_Review skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HFUT-China/Human Practices/Project Review</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>HFUT-2016 Human practice</TITLE></P><DIV class="titleBar"><SPAN class="title">Biodesigner Coral</SPAN></DIV><UL class="sf-menu"><LI><A href="https://2016.igem.org/Team:HFUT-China">Home</A></LI><LI><A href="#">Project</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Proof">Validation</A></LI></UL><LI><A href="#">Software</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/UserGuide">User Guide</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Technology">Technology</A></LI><LI><A href="#">Documents</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Api">API</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/UnitTest">Unit Test</A></LI><LI><A href="#">Requirements</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Medals">Medals</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Collaborations">Collaboration</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Attributions">Attribution</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Human_Practices/Project_Review">Project Review</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Human_Practices/Software_Features">Software Features</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Human_Practices/Recapitulation">Recapitulation</A></LI><LI><A href="https://2016.igem.org/Team:HFUT-China/Integrated_Practices">Integrated Human Practice</A></LI><DIV class="mainCenter"><DIV class="mainHeader"><DIV class="header_text">
      Here is a review for all the software (tools) track projects in IGEM from 2009 to 2015. 
      They are classified into seven types,
      according to their function. 
    </DIV></DIV><DIV class="searchMain main"><TABLE class="searchTable"><TBODY><TR><TD>HFUT-China-2015</TD><TD><DIV class="software">
         BioDesigner
        </DIV></TD><TD>Help users to find their needs based on what they describe and presenting related information.</TD></TR><TR><TD>Korea-U-Seoul-2015</TD><TD>Gil</TD><TD>Help users to find pathway.</TD></TR><TR><TD>Michigan-Software-2015</TD><TD>ProtoCat 2.0</TD><TD>Help users to search for, upload and edit protocol.</TD></TR><TR><TD>SJTU-Software-2015</TD><TD>BASE</TD><TD>Help users to search for parts or devices though IDs or features of genes.</TD></TR><TR><TD>SJTU-Software-2014</TD><TD>Easy BBK</TD><TD>Help users to search for biobrick.</TD></TR><TR><TD>UESTC-Software-2014</TD><TD>CRISPR-X</TD><TD>Help users to search for CRISPR target sites.</TD></TR><TR><TD>USTC-Software-2014</TD><TD>Biopano</TD><TD>Search the route connecting two biological parts input by users. Find E.coli K-12 gene highly similar to the injected sequence by BLAST method. Help users to search for biobrick on iGEM’s official website. </TD></TR><TR><TD>SYSU-Software-2013</TD><TD>CAST</TD><TD>Help users to search all kinds of proteins related to synthetic biology, and arrange the regulatory relationship.</TD></TR><TR><TD>SUSTC-Shenzhen-A-2012</TD><TD>Biosearch</TD><TD>An iPhone app which can search biobricks in partsregistry.org.</TD></TR><TR><TD>SYSU-Software-2012</TD><TD>BiArkit</TD><TD>Allow users to surf for genomes from the typical model organisms.</TD></TR><TR><TD>UT-Tokyo-Software-2012</TD><TD>Tutorial</TD><TD>Help users to search past projects and BioBrick parts.</TD></TR><TR><TD>Wellesley-HCI-2012</TD><TD>Synbio Search</TD><TD>Help users to search for biological parts.</TD></TR><TR><TD>BU-Wellesley-Software-2011</TD><TD>G-nome Surfer Pro</TD><TD>Help users to find genes on the chromosome wheel or search for a gene name or Genbank number.</TD></TR><TR><TD>METU-BIN-Ankara-2011</TD><TD>M4B</TD><TD>Allow users to search all possible device combinations for the user defined input and output within bioparts of 2011 distribution.</TD></TR><TR><TD>NYC-Software-2011</TD><TD>Primer Designer</TD><TD>Help users to search for BioBricks based on their parts' sequence.</TD></TR><TR><TD>CBNU-Korea-2009</TD><TD>Essarker</TD><TD>Manage and retrieve required sequences of genomes. Explore the essential gene order and direction and the related orthologous genes.</TD></TR></TBODY></TABLE></DIV><DIV class="simulationMain main"><TABLE><TBODY><TR><TD>HFUT-China-2015</TD><TD>BioDesigner</TD><TD>Simulate the reaction process of the designed device.</TD></TR><TR><TD>Korea-U-Seoul-2015</TD><TD>Gil</TD><TD>Calculate the increase and decrease of the number of ATP, NADH, NADPH, and CO2, providing the maximum three output paths.</TD></TR><TR><TD>UESTC-Software-2015</TD><TD>MCCAP</TD><TD>Construct an updated minimal gene set of bacteria</TD></TR><TR><TD>USTC-Software-2015</TD><TD>BioBLESS</TD><TD>Simulate the stochastic time evolution of all the inputs and outputs.</TD></TR><TR><TD>SYSU-Software-2015</TD><TD>CORE</TD><TD>A tool that can provide mathematical modeling , plasmid design, experiment scheduling and protocol management.</TD></TR><TR><TD>AMU-Poznan-2014</TD><TD>Sh-miR designer 2.0</TD><TD>Design potentially effective RNA interference (RNAi) reagents: shmiRs</TD></TR><TR><TD>PoznanSoft-2014</TD><TD>MUFASA</TD><TD>Assemble multiple fragments for the cloning of big genetic constructs.</TD></TR><TR><TD>AMU-Poznan-2013</TD><TD>Sh-miR designer</TD><TD>Design potentially effective RNA interference (RNAi) reagents: shmiRs</TD></TR><TR><TD>Shenzhen-BGIC-0101-2013</TD><TD>Genovo</TD><TD>Assist users to create new system denovo, including editing coding DNA sequence, splitting the chromosome into equipped fragments for synthesis and assembly.</TD></TR><TR><TD>SYSU-Software-2013</TD><TD>CAST</TD><TD>Allow users to trace the change of protein or compare some curves. Allow users to view the model influenced by stochastic noise or time delay.</TD></TR><TR><TD>USTC-Software-2013</TD><TD>gNAP</TD><TD>Predict the interactions between new gene and original genes to construct a new gene regulatory network.</TD></TR><TR><TD>Wellesley-Desyne-2013</TD><TD>Eugenie</TD><TD>Allow biologists to specify biological parts, properties, and device composition rules.</TD></TR><TR><TD>CBNU-Korea-2012</TD><TD>Brick Designer</TD><TD>Help users to design new biobricks and use them.</TD></TR><TR><TD>Johns-Hopkins-Software-2012</TD><TD>AutoGene</TD><TD>Allow users to annotate look inside and manipulate the plasmids. Allow users to drag their cursor over multiple features from a plasmid and drag them into the design registry all at once.</TD></TR><TR><TD>SUSTC-Shenzhen-B-2012</TD><TD>TTEC</TD><TD>Predict the secondary structure and the efficiency of terminators. Calculates the free energy of RNA folding.</TD></TR><TR><TD>SYSU-Software-2012</TD><TD>BiArkit</TD><TD>Illustrate the given genome and the expression level of genes within it under different environment.</TD></TR><TR><TD>USTC-Software-2012</TD><TD>REBORN</TD><TD>Reversely engineer the input behavior to get the possible structures of a biological system.</TD></TR><TR><TD>BU-Wellesley-Software-2011</TD><TD>Optimus Primer</TD><TD>Allow users to create a primer design from a selected gene.</TD></TR><TR><TD>CBNU-Korea-2011</TD><TD>GOD</TD><TD>Design minimal genome.</TD></TR><TR><TD>ENSPS-Strasbourg-2011</TD><TD>Translated the biosystem into an electronic equivalent, and simulate the equivalent into a conventional electronic simulator.</TD></TR><TR><TD>HUST-China-2011</TD><TD>SEA</TD><TD>Design engineering bacteria.</TD></TR><TR><TD>UIUC-Illinois-Software-2010</TD><TD>BioMORTAR</TD><TD>Design a plasmid in-silico for the user-specified host bacteria. Simulate cell growth and chemical flux.</TD></TR><TR><TD>USTC-Software-2010</TD><TD>iGaME</TD><TD>Provide automatic modeling simulation, including compartment construction and plasmid construction. Show system behavior according to users’ assembling of parts.</TD></TR><TR><TD>Illinois-Tools-2009</TD><TD>IMPtools</TD><TD>Provide different pathways according to user defined weights.</TD></TR></TBODY></TABLE></DIV><DIV class="automaticMain main"><TABLE><TBODY><TR><TD>USTC-Software-2015</TD><TD>BioBLESS</TD><TD>Convert the truth table into Boolean Formula and find corresponding digital circuit.</TD></TR><TR><TD>UCSD-Software-2014</TD><TD>SBiDer</TD><TD>Help users to search for existing circuits.</TD></TR><TR><TD>SYSU-Software-2014</TD><TD>FLAME</TD><TD>Help users to design synthetic gene circuits.</TD></TR><TR><TD>SYSU-Software-2013</TD><TD>CAST</TD><TD>Generate default gene circuits automatically, and the circuits can be optimized by users.</TD></TR><TR><TD>UESTC-2013</TD><TD>Nebular</TD><TD>Design biological circuits.</TD></TR><TR><TD>BU-Wellesley-Software-2011</TD><TD>Trumpet</TD><TD>Aid the user in designing fully reconfigurable genetic circuit plasmids.</TD></TR><TR><TD>USTC-Software-2009</TD><TD>ABCD</TD><TD>Design biological circuits to assist the design of plasmids.</TD></TR></TBODY></TABLE></DIV><DIV class="analysisMain main"><TABLE><TBODY><TR><TD>Team</TD><TD>Software</TD><TD>Function</TD></TR><TR><TD>HFUT-China-2015</TD><TD>BioDesigner</TD><TD>Recommend what the users might use during the design process.</TD></TR><TR><TD>USTC-Software-2015</TD><TD>BioBLESS</TD><TD>Recommend most reliable circuits according to gene circuits’ fitness score.</TD></TR><TR><TD>SJTU-Software-2015</TD><TD>BASE</TD><TD>Allowing users to draw their devices and give some keywords when drag an icon to the chain to get a list of parts in recommendation interface.Giving a score and advice for each part to improve users’ devices.</TD></TR><TR><TD>HFUT-China-2014</TD><TD>BioDesigner</TD><TD>Recommending biobricks.</TD></TR><TR><TD>UESTC-Software-2014</TD><TD>CRISPR-X</TD><TD>Predict and evaluate genome-wide Cas9 potential off-target cleavage sites.</TD></TR><TR><TD>USTC-Software-2013</TD><TD>gNAP</TD><TD>Give advice to experimenters before choosing the imported gene meeting their purpose.</TD></TR><TR><TD>XMU-Software-2013</TD><TD>Brick Worker</TD><TD>Analyze and predict the strength of various biobricks.</TD></TR><TR><TD>CBNU-Korea-2012</TD><TD>Minimal Genome Designer</TD><TD>Help users to design minimal genomes by essential genes which are predicted by the developers.</TD></TR><TR><TD>SYSU-Software-2012</TD><TD>BiArkit</TD><TD>Compute out potential regulatory riboswitches and siRNA for engineering systems.</TD></TR><TR><TD>UPT-Software-2012</TD><TD>S2MT</TD><TD>Recommend Primers to make gene sequence compatible with the iGEM standard.</TD></TR><TR><TD>ENSPS-Strasbourg-2011</TD><TD>Recommend biobricks.</TD></TR><TR><TD>METU-BIN-Ankara-2011</TD><TD>M4B</TD><TD>Rank all possible composite devices.</TD></TR><TR><TD>NYC-Software-2011</TD><TD>Primer Designer</TD><TD>Take a given sequence and output the primers necessary to biobrick that part.</TD></TR><TR><TD>USTC-Software-2011</TD><TD>Lachesis</TD><TD>Evaluate the working ability or robustness of a biological reaction network.</TD></TR><TR><TD>METU-Turkey-Software-2010</TD><TD>BioGuide</TD><TD>Provide alternative pathways to construct the most reliable and functional Biobrick devices as a guide to Biobricks parts registry.</TD></TR><TR><TD>Illinois-Tools-2009</TD><TD>IMPtools</TD><TD>Provide additional information according to the resulting pathway by the software.</TD></TR></TBODY></TABLE></DIV><DIV class="sharingMain main"><TABLE><TBODY><TR><TD>Michigan-Software-2015</TD><TD>ProtoCat 2.0</TD><TD>Rating and commenting on others' protocol; Generating feedback for the author about the efficacy of his/her protocol.</TD></TR><TR><TD>SJTU-Software-2015</TD><TD>BASE</TD><TD>The database can be expanded through user-generated additions.</TD></TR><TR><TD>SYSU-Software-2015</TD><TD>CORE</TD><TD>(&quot;Synbio Hub&quot;) Help users to design and redesign, share and reuse previous projects. Providing a proof-of-concept crowd-sourcing platform for project co-development and communication.</TD></TR><TR><TD>Michigan-Software-2014</TD><TD>ProtoCat</TD><TD>Allowing users to upload protocols, comment on them.</TD></TR><TR><TD>SJTU-Software-2014</TD><TD>Easy BBK</TD><TD>Allowing users to compare the quality of biobrick. The database can be expanded through user-generated additions.</TD></TR><TR><TD>UCSD-Software-2014</TD><TD>SBiDer</TD><TD>The database can be expanded through user-generated additions.</TD></TR><TR><TD>USTC-Software-2014</TD><TD>Biopano</TD><TD>Allow users to name new biological parts and build a brand new net. Allow users to sign up with Google account and upload their network to the main database.</TD></TR><TR><TD>SUSTC-Shenzhen-B-2013</TD><TD>Circuit+</TD><TD>Build up a central database for collecting and sharing genetic circuits.</TD></TR><TR><TD>SYSU-Software-2013</TD><TD>CAST</TD><TD>Allow users to build and upload their novel biobricks. Regulation network, plasmid constructed and simulation can be stored on CAST and shared between researchers</TD></TR><TR><TD>Wellesley-Desyne-2013</TD><TD>Bac to the Future</TD><TD>Utilize Twitter to illustrate synbio ideas to the public.</TD></TR><TR><TD>Wellesley-HCI-2012</TD><TD>Synflo</TD><TD>Serve as an interactive installation that utilizes tangible interaction to illustrate core concepts of synthetic biology in outreach programs.</TD></TR><TR><TD>BU-Wellesley-Software-2011</TD><TD>eLabNotebook</TD><TD>Allow users to get an overview of current activity and progress, and share physical and digital resources in real-time.</TD></TR><TR><TD>Freiburg-Software-2010</TD><TD>SynBioWave 2.0</TD><TD>Provide real-time sharing of parts, design, documentation, and the process of creating research data.</TD></TR><TR><TD>UIUC-Illinois-Software-2010</TD><TD>BioMORTAR</TD><TD>Connect users who store their plasmids in the software.</TD></TR><TR><TD>Freiburg-Software-2009</TD><TD>SynBioWave</TD><TD>Allow users to record and share the process of creating research data. Provide client-to-server or server-to-client communication for building ui-elements, event reporting and data transmission.</TD></TR></TBODY></TABLE></DIV><DIV class="experimentalMain main"><TABLE><TBODY><TR><TD>SJTU-Software-2014</TD><TD>Easy BBK</TD><TD>Allowing users to draw their bio-system with the standardized biobrick icons, and the result can be exported.</TD></TR><TR><TD>UCSD-Software-2014</TD><TD>SBiDer</TD><TD>Provide a standalone modeling Python package that can be used to model circuits given by the online webtool.</TD></TR><TR><TD>Vanderbilt-Software-2014</TD><TD>Darwin</TD><TD>Installed on top of the existing software, parsing and formatting the biological data so that it can be used more effectively with these systems.</TD></TR><TR><TD>UESTC-2013</TD><TD>Transpeeder</TD><TD>Help users to change the component sequences to regulate translation speed and adapt to the experiment requirement.</TD></TR><TR><TD>XMU-Software-2013</TD><TD>E’NOTE</TD><TD>Serve as an assist for experiments: providing a series of intelligent templates and human-orientated data export port; doing some simple calculation.</TD></TR><TR><TD>SUSTC-Shenzhen-A-2012</TD><TD>Biodesign</TD><TD>Allow users to draw biochemical reaction schematics.</TD></TR><TR><TD>SYSU-Software-2012</TD><TD>BiArkit</TD><TD>Allow users to explore and acquire their interested gene’s relevant pathways and networks in one map.</TD></TR><TR><TD>USTC-Software-2012</TD><TD>REBORN</TD><TD>Display experimental data on a beautiful map. Provide a 3-D tool to view the structure of a genetic regulatory network.</TD></TR><TR><TD>UT-Tokyo-Software-2012</TD><TD>Tutorial</TD><TD>Help users to understand concepts that beginners frequently stumble on, and what BioBricks and Genetic Networks are.</TD></TR><TR><TD>ENSPS-Strasbourg-2011</TD><TD>Translate description of the biosystem into a block diagram of elementary functions.</TD></TR><TR><TD>VT-ENSIMAG-Biosecurity-2010</TD><TD>GenoTHREAT</TD><TD>Screen prototype sequence and tell the user if the DNA sequence they input is a dangerous sequence or if the sequence can be synthesized.</TD></TR><TR><TD>ZJU-China-2010</TD><TD>Bach</TD><TD>Calculate RiPS(Ribosomes Per Second) of any given coding sequences from any organism.</TD></TR><TR><TD>Berkeley-Software-2009</TD></TR><TR><TD>Spectacles</TD></TR><TR><TD>Kepler Integration</TD></TR><TR><TD>Eugene</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>Provide a visual editing framework for the design of composite biological devices.</TD></TR><TR><TD>Capture a number of specific design protocols related to composite part assembly.</TD></TR><TR><TD>A language which allows users to mix custom parts with predefined parts from established databases.</TD></TR></TBODY></TABLE>Minnesota-2009SynBioSSOverhaul the user interface for the SynBioSS Designer, allowing users to pull all relevant properties of any BioBrick automatically using only the brick ID.</DIV><DIV class="visualizationMain main"><TABLE><TBODY><TR><TD>Team</TD><TD>Software</TD><TD>Function</TD></TR><TR><TD>Toronto-2015</TD><TD>ConsortiaFlux</TD><TD>Visualize and manipulate community-level metabolic networks and species-specific cytoplasmic networks</TD></TR><TR><TD>UCSD-Software-2014</TD><TD>SBiDer</TD><TD>Construct and visualize the network of all synthetic genetic circuits that can interact with one another. Show literature reference, characterization data, and images of included devices of circuits through the built-in table.</TD></TR><TR><TD>USTC-Software-2014</TD><TD>Biopano</TD><TD>Show the connections of biological parts through visualization.</TD></TR><TR><TD>Wellesley-Desyne-2013</TD><TD>zTree</TD><TD>Visually represent the Registry of Standard Biological Parts.</TD></TR><TR><TD>Wellesley-HCI-2012</TD><TD>MoClo planner</TD><TD>Visualizes the Golden Gate Modular Cloning (a DNA construction method) process.</TD></TR><TR><TD>USTC-Software-2011</TD><TD>Lachesis</TD><TD>Generate biological reaction network automatically, and visualize network behavior using graphs and tables.</TD></TR><TR><TD>Johns-Hopkins-Software-2012</TD><TD>AutoGene</TD><TD>Generating an interactive visualization of the annotated plasmid.</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>