<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Saclay_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Saclay/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid nav-container"><NAV class="navbar navbar-default navbar-inverse affix-top" data-spy="affix" data-offset-top="260"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav center-block"><LI><A href="https://2016.igem.org/Team:Paris_Saclay">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Media">Media</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Acknowledgements">Acknowledgements</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Background">Background</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/CRISPR-Cas9">CRISPR/Cas9</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Strategy">Strategy</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Description">Results</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Perspective">Perspective</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Interlab_Study">Interlab Study</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Composite_Part">Composite Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Pratices </A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Human_Practices">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/HP/Silver">Societal Issues of CRISPR/Cas9</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/HP/Gold">Responsible Research and Innovation</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Human_Practices/GMO_Regulation">GMO regulation</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Safety">Safety</A></LI></DIV></DIV></NAV></DIV><DIV class="container content"><DIV id="compl"><H1 id="titreps">Experiments</H1></DIV><DIV id="main"><P>You can find here our materials and methods.
</P><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Antibiotics_concentrations"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Antibiotics concentrations</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Cloning_PCR_Products"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Cloning PCR Products</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Ligation_with_pJET_1.2.2Fblunt_vector"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Ligation with pJET 1.2/blunt vector</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Heat_shock_competent_cells"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Heat shock competent cells</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Preparation"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Preparation</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Transformation"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Transformation</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Electro-competent_cells"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Electro-competent cells</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Preparation_2"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Preparation</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Transformation_2"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Transformation</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Plasmid_DNA_extraction"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Plasmid DNA extraction</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#DNA_extraction_using_the_Invitrogen_ChargeSwitch.C2.AE-Pro_Plasmid_Miniprep_Kit"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">DNA extraction using the Invitrogen ChargeSwitch®-Pro Plasmid Miniprep Kit</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Before_Starting"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Before Starting</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Preparing_the_Sample"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Preparing the Sample</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Binding_the_DNA"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">Binding the DNA</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Washing_the_Column"><SPAN class="tocnumber">6.4</SPAN><SPAN class="toctext">Washing the Column</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Eluting_the_DNA"><SPAN class="tocnumber">6.5</SPAN><SPAN class="toctext">Eluting the DNA</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#Plasmid_digestion"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Plasmid digestion</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#gBlock_plasmid_insertion"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">gBlock plasmid insertion</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#List_of_plasmids_constructed"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">List of plasmids constructed</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#List_of_BioBrick_constructed"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">List of BioBrick constructed</SPAN></A></LI><LI class="toclevel-1 tocsection-21"><A href="#List_of_plasmids_used_for_the_characterization_of_the_previous_part.2C_BBa_K13372001"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">List of plasmids used for the characterization of the previous part, BBa_K13372001</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#DNA_electrophoresis_on_agarose_gel"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">DNA electrophoresis on agarose gel</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Gel_preparation"><SPAN class="tocnumber">12.1</SPAN><SPAN class="toctext">Gel preparation</SPAN></A></LI><LI class="toclevel-1 tocsection-24"><A href="#Polymerase_chain_reaction"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">Polymerase chain reaction</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#List_of_PCR_primers"><SPAN class="tocnumber">13.1</SPAN><SPAN class="toctext">List of PCR primers</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#PCR_with_Taq_PCRx_DNA_Polymerase_.28recombinant.29_or_DreamTaq_DNA_Polymerase_from_ThermoFisher_Scientific"><SPAN class="tocnumber">13.2</SPAN><SPAN class="toctext">PCR with Taq PCRx DNA Polymerase (recombinant) or DreamTaq DNA Polymerase from ThermoFisher Scientific</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#PCR_with_Q5.C2.AE_High-Fidelity_2X_Master_Mix_from_NEB"><SPAN class="tocnumber">13.3</SPAN><SPAN class="toctext">PCR with Q5<SUP>®</SUP> High-Fidelity 2X Master Mix from NEB</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#PCR_with_Phusion_High-Fidelity_DNA_Polymerase_from_ThermoFisher_Scientific"><SPAN class="tocnumber">13.4</SPAN><SPAN class="toctext">PCR with Phusion High-Fidelity DNA Polymerase from ThermoFisher Scientific</SPAN></A></LI><LI class="toclevel-1 tocsection-29"><A href="#PCR_clean-up_with_NucleoSpin_Gel_and_PCR_Clean-up"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">PCR clean-up with NucleoSpin Gel and PCR Clean-up</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#Adjuste_DNA_binding_condition"><SPAN class="tocnumber">14.1</SPAN><SPAN class="toctext">Adjuste DNA binding condition</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#Binding_the_DNA_2"><SPAN class="tocnumber">14.2</SPAN><SPAN class="toctext">Binding the DNA</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#Wash_silica_membrane"><SPAN class="tocnumber">14.3</SPAN><SPAN class="toctext">Wash silica membrane</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#Dry_silica_membrane"><SPAN class="tocnumber">14.4</SPAN><SPAN class="toctext">Dry silica membrane</SPAN></A></LI><LI class="toclevel-2 tocsection-34"><A href="#Elute_DNA"><SPAN class="tocnumber">14.5</SPAN><SPAN class="toctext">Elute DNA</SPAN></A></LI><LI class="toclevel-1 tocsection-35"><A href="#DNA_extraction_from_agarose_gels_with_NucleoSpin_Gel_and_PCR_Clean-up"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">DNA extraction from agarose gels with NucleoSpin Gel and PCR Clean-up</SPAN></A></LI><LI class="toclevel-2 tocsection-36"><A href="#Excise_DNA_fragment.2Fsolubilize_gel_slice"><SPAN class="tocnumber">15.1</SPAN><SPAN class="toctext">Excise DNA fragment/solubilize gel slice</SPAN></A></LI><LI class="toclevel-2 tocsection-37"><A href="#Bind_DNA"><SPAN class="tocnumber">15.2</SPAN><SPAN class="toctext">Bind DNA</SPAN></A></LI><LI class="toclevel-2 tocsection-38"><A href="#Wash_silica_membrane_2"><SPAN class="tocnumber">15.3</SPAN><SPAN class="toctext">Wash silica membrane</SPAN></A></LI><LI class="toclevel-2 tocsection-39"><A href="#Dry_silica_membrane_2"><SPAN class="tocnumber">15.4</SPAN><SPAN class="toctext">Dry silica membrane</SPAN></A></LI><LI class="toclevel-2 tocsection-40"><A href="#Elute_DNA_2"><SPAN class="tocnumber">15.5</SPAN><SPAN class="toctext">Elute DNA</SPAN></A></LI><LI class="toclevel-1 tocsection-41"><A href="#Protein_extraction_from_bacterial_liquid_culture_for_Western_Blot"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">Protein extraction from bacterial liquid culture for Western Blot</SPAN></A></LI><LI class="toclevel-1 tocsection-42"><A href="#Protein_electrophoresis_using_the_ThermoFisher_Mini_Gel_Tank.C2.AE_device"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">Protein electrophoresis using the ThermoFisher Mini Gel Tank® device</SPAN></A></LI><LI class="toclevel-2 tocsection-43"><A href="#Prepare_gel_cassette"><SPAN class="tocnumber">17.1</SPAN><SPAN class="toctext">Prepare gel cassette</SPAN></A></LI><LI class="toclevel-2 tocsection-44"><A href="#Assemble_Mini_Gel_Tank_and_start_electrophoresis"><SPAN class="tocnumber">17.2</SPAN><SPAN class="toctext">Assemble Mini Gel Tank and start electrophoresis</SPAN></A></LI><LI class="toclevel-1 tocsection-45"><A href="#Protein_transfert_using_the_ThermoFisher_iBlot_2_Dry_Blotting_System.C2.AE"><SPAN class="tocnumber">18</SPAN><SPAN class="toctext">Protein transfert using the ThermoFisher iBlot 2 Dry Blotting System®</SPAN></A></LI><LI class="toclevel-2 tocsection-46"><A href="#Assembling_the_iBlot.C2.AE_2_Transfer_Stack"><SPAN class="tocnumber">18.1</SPAN><SPAN class="toctext">Assembling the iBlot® 2 Transfer Stack</SPAN></A></LI><LI class="toclevel-2 tocsection-47"><A href="#Performing_blotting"><SPAN class="tocnumber">18.2</SPAN><SPAN class="toctext">Performing blotting</SPAN></A></LI><LI class="toclevel-2 tocsection-48"><A href="#Disassembling_the_iBlot.C2.AE_2_Transfer_Stack"><SPAN class="tocnumber">18.3</SPAN><SPAN class="toctext">Disassembling the iBlot® 2 Transfer Stack</SPAN></A></LI><LI class="toclevel-1 tocsection-49"><A href="#Western-Blot_using_the_Bind.E2.84.A2_Flex_Western_Device"><SPAN class="tocnumber">19</SPAN><SPAN class="toctext">Western-Blot using the Bind™ Flex Western Device</SPAN></A></LI><LI class="toclevel-2 tocsection-50"><A href="#Prepare_iBind.E2.84.A2_Flex_Card"><SPAN class="tocnumber">19.1</SPAN><SPAN class="toctext">Prepare iBind™ Flex Card</SPAN></A></LI><LI class="toclevel-2 tocsection-51"><A href="#Place_membrane_on_iBind.E2.84.A2_Flex_Card"><SPAN class="tocnumber">19.2</SPAN><SPAN class="toctext">Place membrane on iBind™ Flex Card</SPAN></A></LI><LI class="toclevel-2 tocsection-52"><A href="#Add_solutions_to_wells"><SPAN class="tocnumber">19.3</SPAN><SPAN class="toctext">Add solutions to wells</SPAN></A></LI></DIV><H1><SPAN class="mw-headline" id="Antibiotics_concentrations">Antibiotics concentrations</SPAN></H1><P>The following concentrations of antibiotic were used to grow bacteria (on Petri dishes or liquid cultures):
</P><TABLE class="wikitable"><TBODY><TR><TH>Antibiotic
</TH><TH>Concentration (µg/mL)
</TH></TR><TR><TD>Ampicillin
</TD><TD>50
</TD></TR><TR><TD>Chloramphenicol
</TD><TD>30
</TD></TR><TR><TD>Kanamycin
</TD><TD>50
</TD></TR><TR><TD>Spectinomycin
</TD><TD>50
</TD></TR><TR><TD>Streptomycin
</TD><TD>50
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Cloning_PCR_Products">Cloning PCR Products</SPAN></H1><H4><SPAN class="mw-headline" id="Ligation_with_pJET_1.2.2Fblunt_vector">Ligation with pJET 1.2/blunt vector</SPAN></H4><TABLE class="wikitable"><TBODY><TR><TH>Component
</TH><TH>Volume (µL)
</TH></TR><TR><TD>2x Reaction Buffer</TD><TD>10
</TD></TR><TR><TD>PCR product</TD><TD>1
</TD></TR><TR><TD>pJET 1.2/blunt cloning vector</TD><TD>1
</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1
</TD></TR><TR><TD>Nuclease-free water</TD><TD>up to 19
</TD></TR><TR><TD>Total Volume</TD><TD>20
</TD></TR></TBODY></TABLE><OL><LI>Vortex the ligation mixture.</LI><LI>Short spin (3- 5 seconds) of the same tube.</LI><LI>Incubation of the ligation at RT for 5 minutes. (Note. For PCR products &gt; 3kb, ligation can be prolonged to 30 min)</LI><LI>The ligation mixture can be used directly for transformation. (Note. Keep the ligation mix at -20°C if transformation is postponed. Thaw on ice and mix carefully before transformation).</LI></OL><H1><SPAN class="mw-headline" id="Heat_shock_competent_cells">Heat shock competent cells</SPAN></H1><H2><SPAN class="mw-headline" id="Preparation">Preparation</SPAN></H2><P><B>Day 1</B> Inoculate cells in 3.5mL LB medium.
Incubate at 37°C overnight.
</P><P><B>Day 2</B> Measure culture OD at 600nm. Dilute cells in 250mL of LB medium so OD equals to 0.12. 
Incubate overnight at 20°C and 180rpm.
</P><P><B>Day 3</B> Measure culture OD at 600nm and dilute to obtain OD<SUB>600nm</SUB>=0.6. 
Cells must be kept at 4°C during all following steps.
Put on ice for 10min and centrifuge for 10min at 3000rpm and 4°C. 
Discard supernatant and resuspend cells in 80mL of fresh TB buffer. 
Keep on ice for 10min and centrifuge for 10min at 3000rpm and 4°C.
Discard supernatant again and resuspend cells in 20mL of fresh TB buffer with 7% of DMSO.
Keep on ice for 10min.
Aliquot cells and freeze with liquid nitrogen.
Keep at -80°C.
</P><P><B>TB buffer recipe</B></P><TABLE class="wikitable"><TBODY><TR><TD>HEPES
</TD><TD>10mM
</TD></TR><TR><TD>MnCl<SUB>2</SUB></TD><TD>55mM
</TD></TR><TR><TD>CaCl<SUB>2</SUB></TD><TD>15mM
</TD></TR><TR><TD>KCl
</TD><TD>250mM
</TD></TR><TR><TD>KOH
</TD><TD>up to pH6.7
</TD></TR></TBODY></TABLE><P>Dissolve HEPES, CaCl<SUB>2</SUB> and KCl in water. Adjust pH to 6.7 with KOH. Add MnCl<SUB>2</SUB>. Filter to sterilize and keep at 4°C.
</P><H2><SPAN class="mw-headline" id="Transformation">Transformation</SPAN></H2><P>Add 1µL of plasmids to 50µL of competent cells (make a control tube without plasmid). 
Keep tubes on ice for 30min and heat shock at 42°C for 1min. 
Add 500µL of LB medium into each tube and incubate at 37°C for 1h. 
Spread cells on Petri dishes in duplicate and incubate at 37°C overnight.
</P><H1><SPAN class="mw-headline" id="Electro-competent_cells">Electro-competent cells</SPAN></H1><H2><SPAN class="mw-headline" id="Preparation_2">Preparation</SPAN></H2><P>Inoculate 15mL of LB with 200µL of an overnight cell culture. 
Incubate at 37°C and 180rpm until OD<SUB>600nm</SUB> reaches 0.6. 
Centrifuge cells for 10 minutes at 4000rpm. 
Wash twice with 10mL of 10% glycerol.
Put cells in 200µL of 10% glycerol and use for electroporation.
</P><H2><SPAN class="mw-headline" id="Transformation_2">Transformation</SPAN></H2><P>Use an Eppendorf Electroporator 2510 (tune to 2500V). 
Add 1µL of plasmids to 50µL of competent cells (make a control tube without plasmid). 
Place cells in a 1mm-cuvette, electroporate (the time constant should be close to 6ms) and recover cells with 1mL of cold LB medium.
Incubate cells for 1h at 37°C.
Spread cells on Petri dishes in two different concentrations and incubate at 37°C overnight.
</P><H1><SPAN class="mw-headline" id="Plasmid_DNA_extraction">Plasmid DNA extraction</SPAN></H1><P>Transfer 1.5mL of an overnight culture into an eppendorf tube.
Centrifuge at 13000rpm for 1min to pellet the cells. 
Discard supernatant and resuspend cells in 100µL of <A href="#TE">TE buffer</A>. 
Add 200µL of <A href="#solII">solution II</A> and mix gently by inverting the tubes until lysate appears clear.
Add 150µL of <A href="#solIII">solution III</A> and mixed gently.
Keep the solution on ice for 10min. 
Centrifuge for 10min at 13000rpm and recover the supernatant.
Add 100µL of phenol in each tubes to denature the proteins and vortex for 30s. 
Centrifuge the tubes for 7min at 13000rpm. 
Recover the aqueous phase.
Add 2 volumes (900µL) of 100% ethanol and put at -20°C for 10min. 
Centrifuge the tubes again for 10min at 13000rpm. 
Discard supernatant and wash with 800µL of 70% ethanol to remove the remaining ions. Make sure that the pellet containing DNA remains at the bottom of the tubes. 
Centrifuge the tubes for 4min at 13000rpm and remove supernatants.
Dry tubes in speedvac. 
Resuspend the pellet in 50µL of TE/RNAse. 
Keep the extracted plasmids at -20°C.
</P><B>TE buffer recipe</B><TABLE class="wikitable"><TBODY><TR><TD>Tris
</TD><TD>10mM
</TD></TR><TR><TD>EDTA
</TD><TD>55mM
</TD></TR></TBODY></TABLE><B>Solution II recipe (lysis solution)</B> Keep for maximum 3 weeks.<TABLE class="wikitable"><TBODY><TR><TD>SDS
</TD><TD>1%
</TD></TR><TR><TD>NaOH
</TD><TD>0.1M
</TD></TR></TBODY></TABLE><P><B>Solution III recipe (neutralization solution)</B> CH<SUB>3</SUB>COOK 3M at pH4.8.
</P><TABLE class="wikitable"><TBODY><TR><TD>CH<SUB>3</SUB>COOK 5M
</TD><TD>60mL
</TD></TR><TR><TD>CH<SUB>3</SUB>COOH
</TD><TD>11.5mL
</TD></TR><TR><TD>water
</TD><TD>28.5mL
</TD></TR></TBODY></TABLE><P><B>TE/RNase</B> Use only fresh solution.
</P><TABLE class="wikitable"><TBODY><TR><TD>RNase (10mg/mL)
</TD><TD>5µL
</TD></TR><TR><TD>TE
</TD><TD>1mL
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="DNA_extraction_using_the_Invitrogen_ChargeSwitch.C2.AE-Pro_Plasmid_Miniprep_Kit">DNA extraction using the Invitrogen ChargeSwitch®-Pro Plasmid Miniprep Kit</SPAN></H1><P>Follow the steps below to purify plasmid DNA from 1-5ml of fresh overnight cultures using a microcentrifuge. All steps are performed at room temperature.
</P><H2><SPAN class="mw-headline" id="Before_Starting">Before Starting</SPAN></H2><UL><LI>For a new kit, add Rnase A (provided in the kit) to the Resuspension Buffer and mix.</LI><LI>If necessary, warm the Lysis Buffer to 37°C to dissolve any precipitate.</LI></UL><H2><SPAN class="mw-headline" id="Preparing_the_Sample">Preparing the Sample</SPAN></H2><OL><LI>Pellet cells from 1-5 ml of overnight culture.</LI><LI>Resuspend in 250 µL of Resuspension Buffer premixed with Rnase A. No cell clumps should remain.</LI><LI>Add 250 µL of Lysis Buffer. Mix well by gentle inversion. Do not vortex.</LI><LI>Incubate at room temperature for 2-5 minutes. Do not incubate more than 5 minutes.</LI><LI>Add 250µL of Precipation Buffer, and mix well until a white precipitate is formed.</LI><LI>Centrifuge for 10 minutes at maximum speed to pellet the debris.</LI></OL><H2><SPAN class="mw-headline" id="Binding_the_DNA">Binding the DNA</SPAN></H2><OL><LI>Carefully transfer the supernatant from Step 2.6 above onto the ChargeSwitch®-Pro MiniPrep Colum inserted in a Collection Tube.</LI><LI>Centrifuge the column/tube at maximun speed for 30-60 seconds.</LI><LI>Remove the column from the tube and discard the flow-through. Re-insert the column in the same tube.</LI></OL><H2><SPAN class="mw-headline" id="Washing_the_Column">Washing the Column</SPAN></H2><OL><LI>Add 750µL of Wash Buffer 1 to the column.</LI><LI>Centrifuge the column/tube at the maximun speed for 30-60 seconds.</LI><LI>Remove the column from the tube and discart the flow-through. Re-insert the column in the tube;</LI><LI>Add 250µL of Wash Buffer 2 to the column.</LI><LI>Centrifuge the column/tube at maximun speed for 30-60 seconds.</LI><LI>remove the column from the tube. Discard the flow-through and the Collection Tube.</LI></OL><H2><SPAN class="mw-headline" id="Eluting_the_DNA">Eluting the DNA</SPAN></H2><OL><LI>Insert the column into an Elution Tube (provided in the kit).</LI><LI>Add 50-100µL of Elution Buffer onto the column.</LI><LI>Centrifuge the column/tube at maximun speed for 30-60 seconds.</LI><LI>Optional:Remove the Elution Tube and transfer the eluate back onto the same column. Re-insert the column in the tube and centrifugate at maximun speed for 30-60 seconds. This step is recommended for maximun recovery.</LI><LI>The eluate contains your purified plasmid DNA. Store DNA at 4°C or -20°C.</LI></OL><H1><SPAN class="mw-headline" id="Plasmid_digestion">Plasmid digestion</SPAN></H1><P>Mix 10µL of extracted plasmid with 2µL of digestion buffer and 7µL of water. 
Add 1µL of restriction enzyme.
Incubate for 1 hour at 37°C (except for fast digest enzymes which need only 5 min of incubation at 37°C).
Then digestion products were migrated on <A href="#Electrophoresis">an agarose gel</A>.
</P><H1><SPAN class="mw-headline" id="gBlock_plasmid_insertion">gBlock plasmid insertion</SPAN></H1><P>The following ratio should be over 7:
\[
\frac{vector\ size}{insert\ size}\times\frac{insert\ concentration}{vector\ concentration}
\]
</P><P>Gently centrifuge gBlocks.
Add 100µL of <A href="#TE">TE buffer</A> (final concentration should be around 10ng/µL).
Vortex and incubate at 50°C for 20min.
Mix 7µL of gBlocks with 1µL of <A href="#PlasmidExtraction">extracted</A> and <A href="#PlasmidDigestion">linearized</A> plasmid, 1µL of ligase and 1µL of ligase buffer.
Incubate for 1h at RT.
</P><H1><SPAN class="mw-headline" id="List_of_plasmids_constructed">List of plasmids constructed</SPAN></H1><TABLE class="wikitable"><TBODY><TR><TH>Plasmid name
</TH><TH>Backbone
</TH><TH>Insert
</TH><TH>Insert size (bp)
</TH></TR><TR><TD>pPS16_001</TD><TD>pUC19
</TD><TD>gBlock 1.1
</TD><TD>960
</TD></TR><TR><TD>pPS16_002</TD><TD>pUC19
</TD><TD>gBlock 1.2
</TD><TD>960
</TD></TR><TR><TD>pPS16_003</TD><TD>pUC19
</TD><TD>gBlock 2.1
</TD><TD>1023
</TD></TR><TR><TD>pPS16_004</TD><TD>pUC19
</TD><TD>gBlock 2.2
</TD><TD>808
</TD></TR><TR><TD>pPS16_005</TD><TD>pUC19
</TD><TD>gBlock 3.1
</TD><TD>960
</TD></TR><TR><TD>pPS16_006</TD><TD>pUC19
</TD><TD>gBlock 3.2
</TD><TD>960
</TD></TR><TR><TD>pPS16_007</TD><TD>pUC19
</TD><TD>gBlock 4.1
</TD><TD>706
</TD></TR><TR><TD>pPS16_008</TD><TD>pUC19
</TD><TD>gBlock 4.2
</TD><TD>1288
</TD></TR><TR><TD>pPS16_009</TD><TD>pUC19
</TD><TD>gBlock GFP 1-9
</TD><TD>862
</TD></TR><TR><TD>pPS16_010</TD><TD>pUC19
</TD><TD>gBlock ATG linker RFB
</TD><TD>374
</TD></TR><TR><TD>pPS16_011</TD><TD>pUC19
</TD><TD>gBlock detection
</TD><TD>1020
</TD></TR><TR><TD>pPS16_012</TD><TD>pUC19
</TD><TD>gBlock St sgRNA
</TD><TD>310
</TD></TR><TR><TD>pPS16_013</TD><TD>pUC19
</TD><TD>gBlock ATG linker FKBP
</TD><TD>419
</TD></TR><TR><TD>pPS16_014</TD><TD>pUC19
</TD><TD>gBlock Nm sgRNA
</TD><TD>362
</TD></TR><TR><TD>pPS16_015</TD><TD>pUC19
</TD><TD>gBlock spacer1
</TD><TD>900
</TD></TR><TR><TD>pPS16_016</TD><TD>pSB1C3
</TD><TD>gBlock 1.1 + gblock 1.2 + gblock 2.1 + gblock 2.2
</TD><TD>5715
</TD></TR><TR><TD>pPS16_017</TD><TD>pSB1C3
</TD><TD>gBlock 3.1 + gblock 3.2 + gblock 4.1 + gblock 4.2
</TD><TD>5835
</TD></TR><TR><TD>pPS16_018</TD><TD>pSB1C3
</TD><TD>gBlock ATG linker FKBP + GFP 10
</TD><TD>2784
</TD></TR><TR><TD>pPS16_019</TD><TD>pSB1C3
</TD><TD>gBlock ATG linker FRB + GFP 11
</TD><TD>2754
</TD></TR><TR><TD>pPS16_020</TD><TD>pSB1C3
</TD><TD>gBlock GFP 1-9
</TD><TD>2889
</TD></TR><TR><TD>pPS16_021</TD><TD>pSB1C3
</TD><TD>gBlock ATG linker FRB + GFP 11 + gBlock ATG linker FKBP + GFP 10
</TD><TD>3468
</TD></TR><TR><TD>pPS16_022</TD><TD>pSB1C3
</TD><TD>gBlock ATG linker FRB + GFP 11 + GFP 1.9
</TD><TD>3574
</TD></TR><TR><TD>pPS16_023</TD><TD>pSB1C3
</TD><TD>gBlock ATG linker FRB + GFP 11 + gBlock ATG linker FKBP + GFP 10 + gBlock GFP 1-9
</TD><TD>4287
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="List_of_BioBrick_constructed">List of BioBrick constructed</SPAN></H1><TABLE class="wikitable"><TBODY><TR><TH>BioBrick name
</TH><TH>Plasmid name
</TH><TH>Backbone
</TH><TH>Insert
</TH><TH>Insert size (bp)
</TH></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2039000">BBa_K2039000</A></TD><TD>pPS16_018</TD><TD>pSB1C3
</TD><TD>gBlock ATG linker FKBP + GFP 10
</TD><TD>2784
</TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2039001">BBa_K2039001</A></TD><TD>pPS16_019</TD><TD>pSB1C3
</TD><TD>gBlock ATG linker FRB + GFP 11
</TD><TD>2754
</TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2039002">BBa_K2039002</A></TD><TD>pPS16_021</TD><TD>pSB1C3
</TD><TD>gBlock ATG linker FRB + GFP 11 + gBlock ATG linker FKBP + GFP 10
</TD><TD>3468
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="List_of_plasmids_used_for_the_characterization_of_the_previous_part.2C_BBa_K13372001">List of plasmids used for the characterization of the previous part, BBa_K13372001</SPAN></H1><TABLE class="wikitable"><TBODY><TR><TH>Construction
</TH><TH>Plasmid name
</TH></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1372001">BBa_K13372001</A></TD><TD>pSB1C3
</TD></TR><TR><TD>pcl_TAA
</TD><TD>pcl99
</TD></TR><TR><TD>pcl_TAG
</TD><TD>pcl99
</TD></TR><TR><TD>pcl_Tq
</TD><TD>pcl99
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="DNA_electrophoresis_on_agarose_gel">DNA electrophoresis on agarose gel</SPAN></H1><H2><SPAN class="mw-headline" id="Gel_preparation">Gel preparation</SPAN></H2><P>Mix agarose and TAE buffer to get a final concentration of 0.8%(w/v). 
Microwave for 1-2min in order to melt the agarose.
Let cool down for 10min and add BET up to 5µg/mL.
This solution can be kept at 61°C.
Pour the solution into the casting tray.
Place the comb and let cool down for about 30min.
Place the gel in an electrophoresis unit full of TAE and load the samples plus a ladder.
Migrate at 100V for 20min.
Visualize DNA fragments with UV light.
</P><H1><SPAN class="mw-headline" id="Polymerase_chain_reaction">Polymerase chain reaction</SPAN></H1><H2><SPAN class="mw-headline" id="List_of_PCR_primers">List of PCR primers</SPAN></H2><TABLE><TBODY><TR><TH>Name
</TH><TH>Description
</TH><TH>Sequence 5'→3'
</TH></TR><TR><TD>iPS120
</TD><TD>Reverse primer amplification gBlock 1-1
</TD><TD>TCCAGGATGCGCAGGTTGTTCAGCTTG
</TD></TR><TR><TD>iPS121
</TD><TD>Forward primer amplification gBlock 1-2
</TD><TD>GCAAGGCAGCGAGAGACCCCTGAC
</TD></TR><TR><TD>iPS122
</TD><TD>Reverse primer amplification gBlock 1-2
</TD><TD>TCGCTGCCCAGCACCAGCACCTTG
</TD></TR><TR><TD>iPS123
</TD><TD>Forward primer amplification gBlock 2-1
</TD><TD>GCTGCCCTTCAGCCGCACCTGGGAC
</TD></TR><TR><TD>iPS124
</TD><TD>Reverse primer amplification gBlock 2-1
</TD><TD>TTCTGCACCTGCTCCACGCGCACGG
</TD></TR><TR><TD>iPS125
</TD><TD>Forward primer amplification gBlock 2-2
</TD><TD>GACCGGCGTGTGGGTGCGCAACC
</TD></TR><TR><TD>iPS126
</TD><TD>Reverse primer amplification gBlock 3-1
</TD><TD>GGCACGGTCAGGTTGTTCAGGTCGTTC
</TD></TR><TR><TD>iPS127
</TD><TD>Forward primer amplification gBlock 3-2
</TD><TD>CACCGAGACCAAGAAGCTGAGCAAGG
</TD></TR><TR><TD>iPS128
</TD><TD>Reverse primer amplification gBlock 3-2
</TD><TD>CTGTCGTCGAAGGTGATGCTCAGG
</TD></TR><TR><TD>iPS129
</TD><TD>Forward primer amplification gBlock 4-1
</TD><TD>GCTGCCTGTACACCGGCAAGACCATC
</TD></TR><TR><TD>iPS130
</TD><TD>Reverse primer amplification gBlock 4-1
</TD><TD>AACAGGATGCTGTCCTCGAACTCC
</TD></TR><TR><TD>iPS131
</TD><TD>Forward primer amplification gBlock 4-2
</TD><TD>CAGCTACCAGGTGGACAGCAAGTTC
</TD></TR><TR><TD>iPS132
</TD><TD>Forward primer amplification St guide
</TD><TD>TTGACAGCTAGCTCAGTCCTAGGTATAATG
</TD></TR><TR><TD>iPS133
</TD><TD>Reverse primer amplification Nm guide
</TD><TD>CTAGCACTATCAGCGTTATTTAGCG
</TD></TR><TR><TD>iPS134
</TD><TD>Forward primer sequencing Cas plasmids from Addgene
</TD><TD>TTTACGGGCATGCATAAGGCTCG
</TD></TR><TR><TD>iPS135
</TD><TD>Reverse primer sequencing Cas plasmids from Addgene
</TD><TD>GTACCTAGGACTGAGCTAGCCG
</TD></TR><TR><TD>iPS136
</TD><TD>Reverse primer sequencing DS-SPCasN- from Addgene
</TD><TD>CTGCATGAAGTTCCGGTTGGC
</TD></TR><TR><TD>iPS137
</TD><TD>Forward primer sequencing DS-SPCasN- from Addgene
</TD><TD>GAAACAGCTCAAGAGGCGCCG
</TD></TR><TR><TD>iPS138
</TD><TD>Prefix
</TD><TD>GAATTCGCGGCCGCTTCTAGAG
</TD></TR><TR><TD>iPS139
</TD><TD>Suffix
</TD><TD>CTGCAGCGGCCGCTACTAGTA
</TD></TR><TR><TD>iPS140
</TD><TD>Primer to amplify gBlock 1.1 if prefix missed
</TD><TD>GAATTCGCGGCCGCTTCTAGAGTTGACAGCTAGCTCAGTCCT
</TD></TR><TR><TD>iPS141
</TD><TD>Primer to amplify gBlock 2.2 terminator and add Gibson sequence
</TD><TD>CCAAAGCTGGACTTCTAATAACGCTGATAGTGCTAGTGTAGATCGC
</TD></TR><TR><TD>iPS142
</TD><TD>Primer to amplify 2.2 and add linker TEV
</TD><TD>ACCCTGGAAGTACAGGTTTTCACCACCAGAACCACCACCACCGCGCACAGGAGGGCGCTTCTT
</TD></TR><TR><TD>iPS143
</TD><TD>Primer to amplify 3.1 and add linker TEV
</TD><TD>TGAAAACCTGTACTTCCAGGGTGGTTCTGGTGGTGGTGGTTCTAGCGACCTGGTGCTGGGC
</TD></TR><TR><TD>iPS144
</TD><TD>Primer to amplify 4.2 and add Gibson sequence
</TD><TD>CTATCAGCGTTATTAGAAGTCCAGCTTTGGCTTGTCTCC
</TD></TR><TR><TD>iPS145
</TD><TD>Forward primer to amplify FKBP and add Gibson sequence
</TD><TD>GAGAAATACTAGATGATGGGTGTTCAGGTTGAAAC
</TD></TR><TR><TD>iPS146
</TD><TD>Reverse primer to amplify FKBP and add Gibson sequence
</TD><TD>TCCACCTCCTACATCTTCCAGTTTCAGCAGTTCAAC
</TD></TR><TR><TD>iPS147
</TD><TD>Gibson to amplify linker L30 in gBlock 2.2 and add Gibson sequence
</TD><TD>CTGCTGAAACTGGAAGATGTAGGAGGTGGAGGT
</TD></TR><TR><TD>iPS148
</TD><TD>Gibson to amplify promoter in gBlock 1.1 and add Gibson sequence
</TD><TD>CTGAACACCCATCATCTAGTATTTCTCCTCTTTATTATTGTAC
</TD></TR><TR><TD>iPS149
</TD><TD>Forward primer to amplify FRB and add Gibson sequence
</TD><TD>GAGAAATACTAGATGATCCTGTGGCACGAAATG
</TD></TR><TR><TD>iPS150
</TD><TD>Reverse primer to amplify FRB and add Gibson sequence
</TD><TD>ATCGGATCCGAGCTCTTTAGAGATACGACGGAAAACG
</TD></TR><TR><TD>iPS151
</TD><TD>Primer to amplify linker L30 in gBlock 4.2 and add Gibson sequence
</TD><TD>CCGTCGTATCTCTAAAGAGCTCGGATCCGATGTA
</TD></TR><TR><TD>iPS152
</TD><TD>Primer to amplify promoter in gBlock 3.1 and add Gibson sequence
</TD><TD>GTGCCACAGGATCATCTAGTATTTCTCCTCTTTATTATTGTAC
</TD></TR><TR><TD>iPS153
</TD><TD>Forward primer to amplify gBlock detection
</TD><TD>CGAGTCCCTAAACCCTAACCCTAACCCTAACCC
</TD></TR><TR><TD>iPS154
</TD><TD>Reverse primer to amplify gBlock detection
</TD><TD>GCTAACTTAGTTCCAATCCATAAGACACCATTACT
</TD></TR><TR><TD>iPS155
</TD><TD>Forward primer to amplify gBlock spacer
</TD><TD>TGGATTGGAACTAAGTTAGCCTAGAGAATC
</TD></TR><TR><TD>iPS156
</TD><TD>Reverse primer to amplify gBlock spacer
</TD><TD>TAGCTGTCAATCGACTGCAATTCATTATGT
</TD></TR><TR><TD>iPS157
</TD><TD>Forward primer to amplify gBlock Nm sgRNA
</TD><TD>TTGCAGTCGATTGACAGCTAGCTCAGTCCT
</TD></TR><TR><TD>iPS158
</TD><TD>Reverse primer to amplify gBlock Nm sgRNA
</TD><TD>TAGCTGTCAACACTAGCACTATCAGCGTTATTTAGC
</TD></TR><TR><TD>iPS159
</TD><TD>Forward primer to amplify gBlock St sgRNA
</TD><TD>AGTGCTAGTGTTGACAGCTAGCTCAGTCCTAGGTATA
</TD></TR><TR><TD>iPS160
</TD><TD>Reverse primer to amplify gBlock St sgRNA
</TD><TD>GATGCCTCTACGCAGAAAGGCCCACCCG
</TD></TR><TR><TD>iPS161
</TD><TD>Forward primer to amplify plasmid pZA11
</TD><TD>CCTTTCTGCGTAGAGGCATCAAATAAAACGAAA
</TD></TR><TR><TD>iPS162
</TD><TD>Reverse primer to amplify plasmid pZA11
</TD><TD>GGTTAGGGTTTAGGGACTCGAGGTGAAG
</TD></TR><TR><TD>iPS163
</TD><TD>Reverse primer for construction pZA11-detection-sg RNA St &amp; Nm
</TD><TD>GAAGCATTCCCGGGTTACAC
</TD></TR><TR><TD>iPS164
</TD><TD>Forward primer for construction pZA11-detection-sg RNA St &amp; Nm - 50 bp
</TD><TD>GTGTAACCCGGGAATGCTTCCAACCACAGGCTAGATCTGT
</TD></TR><TR><TD>iPS165
</TD><TD>Forward primer for construction pZA11-detection-sg RNA St &amp; Nm - 75 bp
</TD><TD>GTGTAACCCGGGAATGCTTCCGAGATATGGTTGCTAGATACG
</TD></TR><TR><TD>iPS166
</TD><TD>Forward primer for construction pZA11-detection-sg RNA St &amp; Nm - 100 bp
</TD><TD>GTGTAACCCGGGAATGCTTCACCTCCAGATCGATTTTATCGG
</TD></TR><TR><TD>iPS167
</TD><TD>Forward primer for construction pZA11-detection-sg RNA St &amp; Nm - 150 bp
</TD><TD>GTGTAACCCGGGAATGCTTCTGAGTCTATTACGACTTCAAGGC
</TD></TR><TR><TD>iPS168
</TD><TD>Forward primer for pSB1C3 verification (iGEM)
</TD><TD>CCACCTGACGTCTAAGAAAC
</TD></TR><TR><TD>iPS169
</TD><TD>Reverse primer for pSB1C3 verification (iGEM)
</TD><TD>GTATTACCGCCTTTGAGTGA
</TD></TR><TR><TD>iPS170
</TD><TD>Primer to verify gBlobk 2.1 - 2.2 junction
</TD><TD>GGTCACATGGAGACCGTGAAG
</TD></TR><TR><TD>iPS171
</TD><TD>Primer to verify gBlobk 4.1 - 4.2 junction
</TD><TD>TAGCCAGCTGAACCTGTGGA
</TD></TR><TR><TD>iPS172
</TD><TD>Primer to amplify pSB1C3 and add Gibson sequence
</TD><TD>TACTAGTAGCGGCCGCTG
</TD></TR><TR><TD>iPS173
</TD><TD>Primer to correct mutation 1 dCas9 St - GFP 11 and add Gibson sequence
</TD><TD>TGAACAACCTGACCGTGCCCACCGAGACCAAGAAGCTGAG
</TD></TR><TR><TD>iPS174
</TD><TD>Primer to correct mutation 1 dCas9 St - GFP 11 and add Gibson sequence
</TD><TD>CTCAGCTTCTTGGTCTCGGTGGGCACGGTCAGGTTGTTCA
</TD></TR><TR><TD>iPS175
</TD><TD>Primer to correct mutation 2 dCas9 St - GFP 11 and add Gibson sequence
</TD><TD>TCGAGGACAGCATCCTGTTCAGCTACCAGGTGGACAGCAA
</TD></TR><TR><TD>iPS176
</TD><TD>Primer to correct mutation 2 dCas9 St - GFP 11 and add Gibson sequence
</TD><TD>TTGCTGTCCACCTGGTAGCTGAACAGGATGCTGTCCTCGA
</TD></TR><TR><TD>iPS83
</TD><TD>4bases_Prefix
</TD><TD>ATTCGAATTCGCGGCCGCTTCTAGAG
</TD></TR><TR><TD>iPS84
</TD><TD>4bases_suffix
</TD><TD>AGTACTGCAGCGGCCGCTACTAGTA
</TD></TR><TR><TD>iPS41
</TD><TD>PSB1C3 amplification
</TD><TD>GCCGCTGCAGTCCGGCAAAAAA
</TD></TR><TR><TD>iPS42
</TD><TD>PSB1C3 amplification
</TD><TD>ATGAATTCCAGAAATCATCCTTAGCG
</TD></TR><TR><TD>Ptet_R
</TD><TD>Primer for Gibson assembly plasmid
</TD><TD>TTCTGCTGATGTGCTCAGTATC
</TD></TR><TR><TD>Link-TDdcas_F
</TD><TD>Primer for Gibson assembly link-TDdcas
</TD><TD>TGAAAACCTGTACTTCCAGGGTGGTTCTGGTGGTGGTGGTTCTAAGAAGGAGATCAA
</TD></TR><TR><TD>Ter_TDdcas_R
</TD><TD>Primer for Gibson assembly TDdcas-Ter
</TD><TD>CTTTAATGAATTCGGTCAGTGCGAGGAATGGTCAAGGTCCCAG
</TD></TR><TR><TD>Tet-SPdcas_F
</TD><TD>Primer for Gibson assembly Tet-SPdcas
</TD><TD>GATACTGAGCACATCAGCAGGAACTTTTACTAGAGGAGGAGGCA
</TD></TR><TR><TD>Link-SPdcas_R
</TD><TD>Primer for Gibson assembly SPdcas-link
</TD><TD>CACCCTGGAAGTACAGGTTTTCACCACCAGAACCACCACCACCGTCTCCACCGAGCT
</TD></TR><TR><TD>Ptet_F
</TD><TD>Primer for Gibson assembly plasmid
</TD><TD>CGCACTGACCGAATTCATTAAAG
</TD></TR><TR><TD>1152_pheoF
</TD><TD>Forward primer amplification pUC19 insert
</TD><TD>GAATTCGAGCTCGGTACCC
</TD></TR><TR><TD>1151_pheoR
</TD><TD>Reverse primer amplification pUC19 insert
</TD><TD>AAGCTTGCATGCCTGCAG
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="PCR_with_Taq_PCRx_DNA_Polymerase_.28recombinant.29_or_DreamTaq_DNA_Polymerase_from_ThermoFisher_Scientific">PCR with Taq PCRx DNA Polymerase (recombinant) or DreamTaq DNA Polymerase from ThermoFisher Scientific</SPAN></H2><UL><LI>Prepare enough PCR master mix for the number of colonies analyzed plus one extra. Mix the following reagents :</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>Components
</TH><TH>Volume
</TH></TR><TR><TD>10x Taq buffer with KCl
</TD><TD>2µL
</TD></TR><TR><TD>MgCl<SUB>2</SUB> (25mM)
</TD><TD>1.2µL
</TD></TR><TR><TD>dNTP (10mM)
</TD><TD>2µL
</TD></TR><TR><TD>Primers mix (10µM each)
</TD><TD>1.2µL
</TD></TR><TR><TD>Taq DNA polymerase
</TD><TD>0.1µL
</TD></TR><TR><TD>Nuclease-free water
</TD><TD>up to 20µL
</TD></TR><TR><TH>Total volume
</TH><TH>20µL
</TH></TR></TBODY></TABLE><P>or
</P><TABLE class="wikitable"><TBODY><TR><TH>Components
</TH><TH>Volume
</TH></TR><TR><TD>10X DreamTaq Green Buffer
</TD><TD>2.5µL
</TD></TR><TR><TD>dNTP (10mM)
</TD><TD>2.5µL
</TD></TR><TR><TD>Primers mix (10µM each)
</TD><TD>1µL
</TD></TR><TR><TD>DreamTaq DNA polymerase
</TD><TD>0.13µL
</TD></TR><TR><TD>Nuclease-free water
</TD><TD>up to 25µL
</TD></TR><TR><TH>Total volume
</TH><TH>25µL
</TH></TR></TBODY></TABLE><UL><LI>Mix gently and aliquot 20μL or 25μL of the mix into the PCR tubes on ice. </LI><LI>Pick up an individual colony with a sterile pipette tip and resuspend it in 20μL or 25μL of the PCR master mix (or take 1μL of liquid preculture). Make a short strike with the same tip over culture plate to save the clone for repropagation. </LI><LI>Perform PCR as follow (annealing temperature can be calculated <A rel="nofollow" class="external text" href="https://www.thermofisher.com/fr/fr/home/brands/thermo-scientific/molecular-biology/molecular-biology-learning-center/molecular-biology-resource-library/thermo-scientific-web-tools/tm-calculator.html">here</A>): </LI></UL><TABLE class="wikitable"><TBODY><TR><TH>Step
</TH><TH>Temperature
</TH><TH>Time
</TH></TR><TR><TD>Initial denaturation
</TD><TD>95°C
</TD><TD>3min
</TD></TR><TR><TD rowspan="3">30 cycles
</TD><TD>94°C
</TD><TD>30sec
</TD></TR><TR><TD>T<SUB>annealing</SUB></TD><TD>30sec
</TD></TR><TR><TD>72°C
</TD><TD>1min/kb
</TD></TR><TR><TD>Final Extension
</TD><TD>72°C
</TD><TD>7min
</TD></TR><TR><TD>Hold
</TD><TD>4°C
</TD><TD>$\infty$
</TD></TR></TBODY></TABLE><UL><LI><A href="#Electrophoresis">Analyze on a gel</A> for the presence of the PCR product of the expected length.</LI></UL><H2><SPAN class="mw-headline" id="PCR_with_Q5.C2.AE_High-Fidelity_2X_Master_Mix_from_NEB">PCR with Q5<SUP>®</SUP> High-Fidelity 2X Master Mix from NEB</SPAN></H2><UL><LI>Prepare enough PCR master mix for the number of colonies analyzed plus one extra. For each 25μl of reaction, mix the following reagents :</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>Components
</TH><TH>Volume
</TH></TR><TR><TD>Q5 High-Fidelity 2X Master Mix
</TD><TD>12.5µL
</TD></TR><TR><TD>Primers mix (10µM each)
</TD><TD>2.5µL
</TD></TR><TR><TD>Template DNA (from extractions)
</TD><TD>1µL
</TD></TR><TR><TD>Nuclease-free water
</TD><TD>up to 25µL
</TD></TR><TR><TH>Total volume
</TH><TH>25µL
</TH></TR></TBODY></TABLE><UL><LI>Mix gently and aliquot 25μl of the mix into the PCR tubes on ice. </LI><LI>Perform PCR as follow (annealing temperature can be calculated <A rel="nofollow" class="external text" href="http://tmcalculator.neb.com/#!/">here</A>): </LI></UL><TABLE class="wikitable"><TBODY><TR><TH>Step
</TH><TH>Temperature
</TH><TH>Time
</TH></TR><TR><TD>Initial denaturation
</TD><TD>98°C
</TD><TD>3min
</TD></TR><TR><TD rowspan="3">30 cycles
</TD><TD>98°C
</TD><TD>10sec
</TD></TR><TR><TD>T<SUB>annealing</SUB></TD><TD>20sec
</TD></TR><TR><TD>72°C
</TD><TD>30sec/kb
</TD></TR><TR><TD>Final Extension
</TD><TD>72°C
</TD><TD>2min
</TD></TR><TR><TD>Hold
</TD><TD>4°C
</TD><TD>$\infty$
</TD></TR></TBODY></TABLE><UL><LI><A href="#Electrophoresis">Analyze on a gel</A> for the presence of the PCR product of the expected length.</LI></UL><H2><SPAN class="mw-headline" id="PCR_with_Phusion_High-Fidelity_DNA_Polymerase_from_ThermoFisher_Scientific">PCR with Phusion High-Fidelity DNA Polymerase from ThermoFisher Scientific</SPAN></H2><UL><LI>Prepare enough PCR master mix for the number of colonies analyzed plus one extra. For each 50μl of reaction, mix the following reagents :</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>Components
</TH><TH>Volume
</TH></TR><TR><TD>Buffer 5x HF
</TD><TD>10µL
</TD></TR><TR><TD>dNTP (10mM)
</TD><TD>1µL
</TD></TR><TR><TD>Primers mix (10µM each)
</TD><TD>5µL
</TD></TR><TR><TD>Template DNA (from extractions)
</TD><TD>1µL
</TD></TR><TR><TD>Phusion DNA polymerase
</TD><TD>0,25 µL
</TD></TR><TR><TD>Nuclease-free water
</TD><TD>up to 50µL
</TD></TR><TR><TH>Total volume
</TH><TH>50µL
</TH></TR></TBODY></TABLE><UL><LI>Mix gently and aliquot 50μl of the mix into the PCR tubes on ice. </LI><LI>Perform PCR as follow (annealing temperature can be calculated <A rel="nofollow" class="external text" href="https://www.thermofisher.com/fr/fr/home/brands/thermo-scientific/molecular-biology/molecular-biology-learning-center/molecular-biology-resource-library/thermo-scientific-web-tools/tm-calculator.html">here</A>): </LI></UL><TABLE class="wikitable"><TBODY><TR><TH>Step
</TH><TH>Temperature
</TH><TH>Time
</TH></TR><TR><TD>Initial denaturation
</TD><TD>98°C
</TD><TD>30sec
</TD></TR><TR><TD rowspan="3">30 cycles
</TD><TD>98°C
</TD><TD>10sec
</TD></TR><TR><TD>T<SUB>annealing</SUB></TD><TD>30sec
</TD></TR><TR><TD>72°C
</TD><TD>30sec/kb
</TD></TR><TR><TD>Final Extension
</TD><TD>72°C
</TD><TD>7min
</TD></TR><TR><TD>Hold
</TD><TD>4°C
</TD><TD>$\infty$
</TD></TR></TBODY></TABLE><UL><LI><A href="#Electrophoresis">Analyze on a gel</A> for the presence of the PCR product of the expected length.</LI></UL><H1><SPAN class="mw-headline" id="PCR_clean-up_with_NucleoSpin_Gel_and_PCR_Clean-up">PCR clean-up with NucleoSpin Gel and PCR Clean-up</SPAN></H1><H2><SPAN class="mw-headline" id="Adjuste_DNA_binding_condition">Adjuste DNA binding condition</SPAN></H2><P>PCR sample must have 50-100µL and if not, water is added. 1 volume of sample was mixed with 2 volumes of Buffer NT1.
</P><H2><SPAN class="mw-headline" id="Binding_the_DNA_2">Binding the DNA</SPAN></H2><P>NucleoSpin gel and PCR Clean-up column were placed into collection tube (2mL). The sample are centrifuged at 11,000rpm for 30s. The flow-through is discarded and the column put back into the collection tube. 
</P><H2><SPAN class="mw-headline" id="Wash_silica_membrane">Wash silica membrane</SPAN></H2><P>700µL of Buffer NT3 is added in each column. The tubes were centrifuged at 11,000 rpm for 30s. The flow-through is discarded and the column put back into the collection tube. This step is repeated another time.
</P><H2><SPAN class="mw-headline" id="Dry_silica_membrane">Dry silica membrane</SPAN></H2><P>The tubes were centrifuged 1min at 11,000 rpm to removed completely.
</P><H2><SPAN class="mw-headline" id="Elute_DNA">Elute DNA</SPAN></H2><P>The NuleoSpin gel and PCR clean-up column were put into new tube (1.5mL) and 30*L of Buffer NE was added. The tube were left incubated for 1 at RT and then centrifuged 1min at 11,000 rpm.
</P><UL><LI><A href="#Electrophoresis">Analyze on a gel</A> for the presence of the PCR product of the expected length.</LI></UL><H1><SPAN class="mw-headline" id="DNA_extraction_from_agarose_gels_with_NucleoSpin_Gel_and_PCR_Clean-up">DNA extraction from agarose gels with NucleoSpin Gel and PCR Clean-up</SPAN></H1><H2><SPAN class="mw-headline" id="Excise_DNA_fragment.2Fsolubilize_gel_slice">Excise DNA fragment/solubilize gel slice</SPAN></H2><P>Take a clean scalpel to excise the DNA fragment from an agarose gel. Remove all excess agarose.
</P><P>Determine the weight of the gel slice and transfer it to a clean tube.
For each 100 mg of agarose gel&lt;2% add 200µL Butter NT1.
</P><P>For gels containing &gt;2%, double the volume of Buffer NT1.
</P><P>Incubate sample for 5-1à min at 50°C. Vortex the sample briefly every 2-3 min until the gel slice is completely dissolved!
</P><H2><SPAN class="mw-headline" id="Bind_DNA">Bind DNA</SPAN></H2><P>Place a NucleoSPin Gel and PCR Clean-Up Column into a Collection Tube (2mL) and load up to 700µL sample.
</P><P>Centrifuge for 30s at 11 000x g. Discard flow-through and place the column nack into the collection tube.
</P><P>PCR sample must have 50-100µL and if not, water is added. 1 volume of sample was mixed with 2 volumes of Buffer NT1.
</P><H2><SPAN class="mw-headline" id="Wash_silica_membrane_2">Wash silica membrane</SPAN></H2><P>Add 700µL Buffer NT3 to the NucleoSpin Gel and PCR Clean-up Column. Centrifuge for 30s at 11 000 x g. Discard flow-through and place the column back into the collection tube.
</P><P>Recommended: Repeat previous washing step to minimize chaotropic salt carry-over and low A260/A260.
</P><H2><SPAN class="mw-headline" id="Dry_silica_membrane_2">Dry silica membrane</SPAN></H2><P>Centrifuge for 1 min at 1 000 x g to remove Buffer NT3 completely. Make sure the spin column does not come in contact with the flow-through while removing it from the centrifuge and the collection tube.
</P><H2><SPAN class="mw-headline" id="Elute_DNA_2">Elute DNA</SPAN></H2><P>Place the NucleoSpin Gel and PCR Clean-up Column into a new 1.5mL microcentrifuge tube.
Add 15-30µL Buffer NE and incubate at room temperature (18-25°C) for 1 min. Centrifuge for 1 min at 11 000 x g.
</P><UL><LI><A href="#Electrophoresis">Analyze on a gel</A> for the presence of the PCR product of the expected length.</LI></UL><H1><SPAN class="mw-headline" id="Protein_extraction_from_bacterial_liquid_culture_for_Western_Blot">Protein extraction from bacterial liquid culture for Western Blot</SPAN></H1><P>Mesure the OD of 1 mL overnight cell diluated at 1/10 with LB (make a blank with 1 mL LB). Once the OD is obtained for 1 mL culture, centrifuge 500 µL of culture for 10 minutes at maximum speed. The flow-through is discarded and lysis buffer for Western Blot is added in order to obtain 20 OD. 
</P><UL><LI> Volume of lysis buffer to be added: V = OD<SUB>measured</SUB> x 0,5 / 20</LI><LI> Lysis buffer for Western Blot (SDS 1%, Tris-HCl, DTT, pH = 6.8, bromophenol blue)  </LI></UL><P>
Then, put the sample 3 minutes at 95°C. If the middle is too sticky (there is a lot of DNA in <I>E. coli</I>), add 1 μl of MgCl<SUB>2</SUB> and 1 μl of benzoate nuclease (Novgen) and put the middle at 37°C for 10 to 30 minutes.
</P><H1><SPAN class="mw-headline" id="Protein_electrophoresis_using_the_ThermoFisher_Mini_Gel_Tank.C2.AE_device">Protein electrophoresis using the ThermoFisher Mini Gel Tank® device</SPAN></H1><H2><SPAN class="mw-headline" id="Prepare_gel_cassette">Prepare gel cassette</SPAN></H2><P>1. Cut open the gel cassette pouch and remove the cassette. 
</P><P>2. Remove the cassette from the pouch and rinse it with deionized water.
</P><P>3. Remove the gel comb by sliding the comb up one side at a time.
</P><P>4. Remove the tape covering the slot at the lower portion of the cassette.
</P><P>5. Use a pipette and rinse the wells 3 times with 1X running buffer Bolt® LDS Sample Buffer. Invert the gel  and shake gently between rinses to remove buffer.
</P><P>6. Fill the sample wells with running buffer Bolt® LDS Sample Buffer.
</P><H2><SPAN class="mw-headline" id="Assemble_Mini_Gel_Tank_and_start_electrophoresis">Assemble Mini Gel Tank and start electrophoresis</SPAN></H2><P>1. Place the base on a flat surface and snap the electrophoresis tank into the base.
</P><P>2. Place the cassette clamp into the appropriate chamber of the electrophoresis tank with the anode connector (–) aligned to the center. 
</P><P>3. Fill the chamber with 1x running buffer Bolt® LDS Sample Buffer to the level of the cathode.
</P><P>4. Place a gel cassette into a chamber with the wells facing towards you. Hold the cassette in a raised position, and close the clamp by moving the cam handle forward to secure the cassette. 
</P><P>5. Make sure that the wells are completely filled with 1X running buffer Bolt® LDS Sample Buffer. Load your samples and markers. 
</P><P>6. Hold the cassette and release the cassette clamp. Gently lower the cassette so that it rests on the bottom of the chamber, and close the cassette clamp.
</P><P>7. Add 1X running buffer to the level of the fill line. Any excess buffer will spill over into the overflow compartment.
</P><P>8. Fit your power supply with Novex® Power Supply Adapters if your power supply is not designed for use with covered or retractable power leads.</P><P>9. If only running one gel, make sure the unused chamber does not contain a cassette clamp.
</P><P>10. Place the lid on the electrophoresis tank. The lid can only be firmly seated if all the connectors are properly aligned. If the lid is not properly seated, power will not be properly supplied to the system.
</P><P>11. With the power off, connect the electrode cords to power supply {red to (+) jack, black to (–) jack}.
</P><P>12. Turn on the power.
</P><P>13. Set the power supply according to the type of buffer and number of mini gels you are using (refer to the instructions supplied with your mini gel).
</P><H1><SPAN class="mw-headline" id="Protein_transfert_using_the_ThermoFisher_iBlot_2_Dry_Blotting_System.C2.AE">Protein transfert using the ThermoFisher iBlot 2 Dry Blotting System®</SPAN></H1><H2><SPAN class="mw-headline" id="Assembling_the_iBlot.C2.AE_2_Transfer_Stack">Assembling the iBlot® 2 Transfer Stack</SPAN></H2><P>1. Open the lid of the device using the latch. Ensure the blotting surface is clean.
</P><P>2. Unseal the iBlot® 2 Transfer Stack.
</P><P>3. Separate the Top Stack and set it to one side of the bench with the transfer gel layer facing up. Keep the Bottom Stack in the transparent plastic tray. 
</P><P>4. Remove and discard the white separator from the Top Stack.  
</P><P>5. Place the Bottom Stack with the tray directly on the blotting surface. Align the tray on the blotting surface according to the type of iBlot® 2 Transfer Stack being used. The electrical contacts on the tray should be aligned with the corresponding electrical contacts on the blotting surface of the iBlot® 2 Gel Transfer Device. 
</P><P>6. Ensure there are no bubbles between the membrane and the transfer stack. Remove any trapped air bubbles using the Blotting Roller.
</P><P>7. Open the gel cassette and immerse the pre-run gel briefly in deionized water (1–10 seconds) to facilitate easy positioning of the gel on top of the transfer membrane.
</P><P>8. Shake off excess water, and place the pre-run gel on the transfer membrane of the Bottom stack as described:
</P><P>• 1 midi gel on an iBlot® 2 Regular Transfer Stack
• 2 mini gels (head-to-head) on an iBlot® 2 Regular Transfer Stack
• 1 mini gel on an iBlot® 2 Mini Transfer Stack  
</P><P>9. Use the Blotting Roller to remove any air bubbles between the gel and the membrane.
</P><P>10. Soak the iBlot® Filter Paper (use the appropriate filter paper for the size of the gel) in a clean container of deionized water. 
</P><P>11. Place the presoaked iBlot® Filter Paper on the pre-run gel. Use the Blotting Roller to remove any air bubbles between the filter paper and gel.
</P><P>12. Take the Top Stack from the bench and place it on top of the presoaked filter paper with the copper electrode facing up (and transfer gel layer facing down). Remove any air-bubbles using the Blotting Roller.
</P><P>13. Place the iBlot®2 Absorbent Pad on top of the iBlot® 2 Transfer Stack such that the electrical contacts are aligned with the corresponding electrical contacts on the blotting surface of the iBlot® 2 Gel Transfer Device. 
</P><P>14. Use the Blotting Roller to flatten any protrusions in the transfer stack.
</P><P>15. Close the lid.
</P><H2><SPAN class="mw-headline" id="Performing_blotting">Performing blotting</SPAN></H2><P>After assembling the iBlot® 2 Gel Transfer Stack, perform blotting as described below. Perform blotting within 10–
15 minutes of assembling the stacks with the gel.
</P><P>1. Close the iBlot® 2 Gel Transfer Device lid by pressing down firmly with two hands on the sides of the lid. Make sure the latch is secure, and ensure that the correct Method is selected. 
</P><P>2. Touch the Start icon on the screen  to begin the transfer.  
</P><P>3. At the end of the transfer, the current automatically shuts off and the iBlot® 2 Gel Transfer Device signals the end of transfer with repeated beeping sounds, and a message on the digital display.  
</P><P>4. Touch the Done icon to stop the beeping. 
</P><P>5. Proceed to Disassembling the iBlot® 2 Transfer Stack.
</P><H2><SPAN class="mw-headline" id="Disassembling_the_iBlot.C2.AE_2_Transfer_Stack">Disassembling the iBlot® 2 Transfer Stack</SPAN></H2><P>To obtain good transfer and detection results, open the device and disassemble the stack within 30 minutes of ending the blotting procedure.
</P><P>1. Open the lid of the iBlot® 2 Gel Transfer Device.
</P><P>2. Discard the iBlot® 2 Absorbent Pad and Top Stack.
</P><P>3. Carefully remove and discard the gel and filter paper (if used) as shown below. Remove the transfer membrane from the stack and proceed with the blocking procedure or stain the membrane (see next page for details).  
</P><P>4. Discard the Bottom stack. 
</P><H1><SPAN class="mw-headline" id="Western-Blot_using_the_Bind.E2.84.A2_Flex_Western_Device">Western-Blot using the Bind™ Flex Western Device</SPAN></H1><H2><SPAN class="mw-headline" id="Prepare_iBind.E2.84.A2_Flex_Card">Prepare iBind™ Flex Card</SPAN></H2><P>1. Open the lid of the iBind™ Flex Western Device. 
</P><P>2. Verify the Midi Insert is inserted in the iBind™ Flex Western Device.
</P><P>3. Open the packaging for the iBind™ Flex Card. 
</P><P>4. Hold the iBind™ Flex Card by the Stack, and remove the card from the packaging.
</P><P>5. Place the iBind™ Flex Card on the Stage.
</P><P>6. Pipette 10 mL of 1X iBind™ Flex/ iBind™ Flex FD Solution evenly across the Flow Region. 
</P><P>7. Pipette 2 mL of 1X iBind™ Flex/ iBind™ Flex FD Solution so that it pools at the center of the membrane region on the iBind™ Flex Card.
</P><H2><SPAN class="mw-headline" id="Place_membrane_on_iBind.E2.84.A2_Flex_Card">Place membrane on iBind™ Flex Card</SPAN></H2><P>1. Place the membrane on top of the pooled solution with the protein-side down, and the low molecular weight protein region closest to the Stack. 
</P><P>2. Use the blotting roller to remove any air bubbles between the membrane and the iBind™ Flex Card.
</P><P>3. Make sure that the the membrane is within the boundaries of the membrane region. No part of the membrane should be directly under the Midi Insert. 
</P><P>4. Lower the lid of the iBind™ Flex Western Device and close the latch handle to lock the lid.
</P><H2><SPAN class="mw-headline" id="Add_solutions_to_wells">Add solutions to wells</SPAN></H2><P>1. Open the Well Cover and add solutions sequentially to the iBind™ Flex Wells starting with row 1:
</P><TABLE class="wikitable"><TBODY><TR><TH>Row
</TH><TH>Solution
</TH><TH>Volume per Well
</TH></TR><TR><TD>1
</TD><TD>Diluted primary antibody
</TD><TD>4 ml
</TD></TR><TR><TD>2
</TD><TD>1X iBind™ Flex/ iBind™ Flex FD Solution
</TD><TD>4 mL
</TD></TR><TR><TD>2
</TD><TD>Diluted secondary antibody
</TD><TD>4 ml
</TD></TR><TR><TD>4
</TD><TD>1X iBind™ Flex/ iBind™ Flex FD Solution
</TD><TD>12 ml
</TD></TR></TBODY></TABLE><P>2. Close the Well Cover and write the start time of incubation on the lid of the iBind™ Flex Western device.
</P><P>3. Incubate 2.5 hours or longer.
</P><P>4. Open the Well Cover to verify that row 4 is completely empty (indicating that the run is over) before releasing the latch handle and opening the lid.
</P><P>5. After incubation, rinse the membrane twice in 20 mL of distilled water for 2 minutes, and proceed to the appropriate detection protocol.
</P><P>6.Discard iBind™ Flex Card after use. 
</P></DIV></DIV><DIV class=" footer"><DIV class="container"><DIV class="adress"><ADRESS>Université Paris Sud
91405 Orsay, France</ADRESS></DIV><SMALL class="envers copyright">Copyright © 2016 iGEM Paris Saclay. <STRIKE>All rights reserved</STRIKE> All inspirations encouraged.*This wiki was conceived during long nights of work and a great coffee experience in the beautiful Orsay, France.</SMALL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>