<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Saclay_Strategy skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Saclay/Strategy</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid nav-container"><NAV class="navbar navbar-default navbar-inverse affix-top" data-spy="affix" data-offset-top="260"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav center-block"><LI><A href="https://2016.igem.org/Team:Paris_Saclay">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Media">Media</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Acknowledgements">Acknowledgements</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Background">Background</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/CRISPR-Cas9">CRISPR/Cas9</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Strategy">Strategy</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Description">Results</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Perspective">Perspective</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Interlab_Study">Interlab Study</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Composite_Part">Composite Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Pratices </A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Human_Practices">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/HP/Silver">Societal Issues of CRISPR/Cas9</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/HP/Gold">Responsible Research and Innovation</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Human_Practices/GMO_Regulation">GMO regulation</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Safety">Safety</A></LI></DIV></DIV></NAV></DIV><DIV class="container content"><DIV id="compl"><H1 id="titreps">Experimental Strategy</H1></DIV><DIV id="main"><P>We designed a bring DNA closer tool (BDC tool) and a visualization tool, as mentioned <A href="/Team:Paris_Saclay/Design#design" title="Team:Paris Saclay/Design">perviously</A>. In order to characterize our tools, we set up the experimental strategy explained bellow.
</P><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Characterization_strategy"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Characterization strategy</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Tripartite_Split-GFP_and_FRB.2A-FKBP12_dimerization_systems"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Tripartite Split-GFP and FRB*-FKBP12 dimerization systems</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Assessment_of_the_minimal_distance_to_have_fluorescence"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Assessment of the minimal distance to have fluorescence</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Assessment_of_the_DNA_regions_brought_closer"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Assessment of the DNA regions brought closer</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#Gene_expression_tests"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Gene expression tests</SPAN></A></LI></DIV><H1><SPAN class="mw-headline" id="Characterization_strategy">Characterization strategy</SPAN></H1><H2><SPAN class="mw-headline" id="Tripartite_Split-GFP_and_FRB.2A-FKBP12_dimerization_systems">Tripartite Split-GFP and FRB*-FKBP12 dimerization systems</SPAN></H2><P>First, we designed two biobricks to test the FRB*/FKBP12 interaction and the tripartite GFP system. FRB* was fused with one subunit of the gene encoding the GFP (GFP 11) and FKBP12 was fused with another subunit (GFP10) <B>[Fig1]</B>. Then, we also put the gene encoding the last subunit (GFP 1-9) in the plasmid pSB1C3 to form the tripartite GFP <B>[Fig2]</B>. 
</P><CENTER><B>Figure 1</B>: Tripartite split-GFP and FRB*/FKBP12 functional assessment. <I>Biobricks corresponding to intermediate characterization. FRB* is fused with GFP11 and FKBP12 is fused with GFP10.</I></CENTER><P>
In order to test the system, we built a plasmid containing three biobricks to express the full system <B>[Fig2]</B>. Then we transformed it in <I>E. coli</I> to assess the system. The system would be tested by measuring GFP fluorescence level under two different conditions <B>[Fig1]</B>: a growth medium containing rapalog (rapamycin analog) and a growth medium without it. We also planed to test it in bacteria containing just two parts (FRB-GFP11 and FKBP12-GFP10) instead of three (so without GFP1-9).
</P><CENTER><B>Figure 2</B>: Biobrick design containing all the parts to characterize tripartite split-GFP and FRB*/FKBP12. <I>All our biobricks in one plasmid and no interference.</I></CENTER><P>
This construction would give us the first results, validate the functionality of the tripartite GFP and the dimerization of FRB* and FKBP12.
</P><H2><SPAN class="mw-headline" id="Assessment_of_the_minimal_distance_to_have_fluorescence">Assessment of the minimal distance to have fluorescence</SPAN></H2><P>One of the goals of our project is to assess the system composed of the BDC tool and the tripartite split-GFP. To evaluate the effect of the bring DNA closer tool, we have to know the minimal distance needed to observe fluorescence emission.
</P><P>This question was also the core of our <A href="/Team:Paris_Saclay/Model#modelisation" title="Team:Paris Saclay/Model">model</A>, which answers the question: <I>What is the optimal distance between the two dCas9s to observe fluorescence?</I></P><P>This question is essential because the distance between the dCas9s may cause major problems. First, steric hindrance and dCas9 footprint may avoid GFP assembling if we target sequences that are too close. Secondly, the proteins size could prevent the GFP parts from assembling if they are too far away. As a result, fluorescence emission would be detected only if the proteins, as well as the DNA regions, are at a precise range of distance.
</P><P>To experimentally assess such distance, we decided to design different plasmids containing the visualization target sequences separated from each other by different number of base pairs <B>[Fig3]</B>. To do so, we designed specific primers to carry out reverse PCR and obtain, from a plasmid in which the target sequences are distant by 1kB, different plasmids where the number of base pairs between the target sequences is reduced. 
This plasmid was thought to be expressed with the plasmid pSB1C3 containing the BioBricks 3, 4 and 5 (cf <A href="/Team:Paris_Saclay/Design#design" title="Team:Paris Saclay/Design">design</A> page).
The target sequences would have been separated by:
</P><UL><LI>1kB</LI><LI>500bp</LI><LI>150bp</LI><LI>75bp</LI><LI>50bp</LI></UL><CENTER><B>Figure 3</B>: Plasmid design to assess the minimal distance needed to have GFP fluorescence and fully characterize the tripartite split-GFP. <I>It would be expressed with the visualization tool BioBrick. The different RT-PCRs would allow us to have different distances between the two target sequences.</I></CENTER><H2><SPAN class="mw-headline" id="Assessment_of_the_DNA_regions_brought_closer">Assessment of the DNA regions brought closer</SPAN></H2><P>In order to test our BDC tool, all the biobricks previously quoted in the <A href="/Team:Paris_Saclay/Design#design" title="Team:Paris Saclay/Design">design</A> page should be expressed in <I>E. coli</I>, as well as all the sgRNAs (corresponding to the target sequences and their cognate dCas9s). Later, the team would measure GFP fluorescence levels in growth medium with or without rapalog. The emission of any fluorescence by the tripartite split-GFP would validate our BDC tool <B>[Fig4]</B> as it would mean the two target sequences are close. 
</P><CENTER><B>Figure 4</B>: BDC tool characterization by the visualization tool mechanism. <I>Using both tools together.</I></CENTER><H1><SPAN class="mw-headline" id="Gene_expression_tests">Gene expression tests</SPAN></H1><P>In order to test a possible influence of the spatial proximity in gene expression, we would test the expression of two different reporter genes. With the aim of having more accurate variation measurements, we should use enzymes such as luciferase or beta-galactosidase.
</P></DIV></DIV><DIV class=" footer"><DIV class="container"><DIV class="adress"><ADRESS>Université Paris Sud
91405 Orsay, France</ADRESS></DIV><SMALL class="envers copyright">Copyright © 2016 iGEM Paris Saclay. <STRIKE>All rights reserved</STRIKE> All inspirations encouraged.*This wiki was conceived during long nights of work and a great coffee experience in the beautiful Orsay, France.</SMALL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>