<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Saclay_Interlab_Study skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Saclay/Interlab Study</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid nav-container"><NAV class="navbar navbar-default navbar-inverse affix-top" data-spy="affix" data-offset-top="260"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav center-block"><LI><A href="https://2016.igem.org/Team:Paris_Saclay">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Media">Media</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Acknowledgements">Acknowledgements</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Background">Background</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/CRISPR-Cas9">CRISPR/Cas9</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Strategy">Strategy</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Description">Results</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Perspective">Perspective</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Interlab_Study">Interlab Study</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Composite_Part">Composite Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Pratices </A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Human_Practices">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/HP/Silver">Societal Issues of CRISPR/Cas9</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/HP/Gold">Responsible Research and Innovation</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Human_Practices/GMO_Regulation">GMO regulation</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Safety">Safety</A></LI></DIV></DIV></NAV></DIV><DIV class="container content"><DIV id="compl"><H1 id="titreps">Interlab Study</H1></DIV><DIV id="main"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Constructions"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Constructions</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Methods"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Assessment"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Assessment</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#Results"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Discussion"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Reference"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Reference</SPAN></A></LI></DIV><H1><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H1><P>While we were at the laboratory developing our project, we also participated to the 2016 Interlab Study. The interlab study consists in measuring the fluorescence level of constructions provided by the iGEM Measurement Committee in order to compare results obtained by worldwide iGEM teams and thus study the variations of measurements among each experiments. This year it consisted on measuring the fluorescence of three test devices composed of a Green Fluorescent Protein (GFP) coding sequence under the control of promoters of different strengths. The measurements could be proceeded using a plate reader or flow cytometry. We chose to use flow cytometry which was available at the laboratory where our team worked.
</P><H2><SPAN class="mw-headline" id="Constructions">Constructions</SPAN></H2><UL><LI><B>Test Device 1</B> is composed of a strong constitutive promotor (J23101), a RBS, wild type GFP gene and a double terminator cloned into the plasmid pSB1C3.</LI><LI><B>Test Device 2</B> is composed of a medium strength constitutive promotor (J23106), a RBS, wild type GFP gene and a double terminator cloned into the plasmid pSB1C3. </LI><LI><B>Test Device 3</B> is composed of a week constitutive promotor (J23117), a RBS, wild type GFP gene and a double terminator cloned into the plasmid pSB1C3.</LI><LI><B>Positive control Device</B> is composed of a constitutive promotor (J23151), a RBS, wild type GFP gene and a double terminator cloned into the plasmid pSB1C3.</LI><LI><B>Negative control Device</B> is only composed of the repressible promotor of the TetR gene cloned into the plasmid pSB1C3.</LI></UL><H2><SPAN class="mw-headline" id="Methods">Methods</SPAN></H2><P>At the beginning of the Interlab study, we had a problem with device 1, the tube received was empty. We waited to receive another tube from iGEM.
</P><P>Constructions test devices 2 and 3 and the two controls were transformed into competent DH5α <I>E.coli</I> strain using a <A href="/Team:Paris_Saclay/Experiments#heat-shocktransformation" title="Team:Paris Saclay/Experiments">heat shock transformation protocol</A>. Transformed bacteria were plated on solid LB medium containing 30 μg/mL chloramphenicol. Petri dishes were incubated at 37°C overnight. For each device, a colony was used to inoculate 3 mL of liquid LB medium containing 30 μg/mL chloramphenicol. The cultures were incubated at 37°C at 180 rpm overnight. Then a glycerol stock was made from these overnight cultures and stored at -80°C. When we received the device 1, we used the same protocol to clone it into the DH5α E.coli strain.
</P><P>Glycerol stocks were plated on solid LB medium containing 30 μg/mL chloramphenicol, and incubated the cultures at 37°C overnight. For each construction, two colonies were randomly picked up to inoculate two different tubes containing 5 mL of liquid LB medium containing 30 μg/mL chloramphenicol.
</P><P>Then we used those tubes to perform flow cytometry. We used the &quot;Cube 8&quot; cytometer from the PARTEC Company. Cells were excited by a 488 nm laser, and we detected fluorescence emission using a 536/40 filter. For each sample, around 1 million cells were counted. Data were obtained in arbitrary units, since we did not have any calibration beads.
</P><H2><SPAN class="mw-headline" id="Assessment">Assessment</SPAN></H2><P>The size of cells (FSC) should be the same for every sample as it is the same bacterial strain and only the fluorescence emission level (FL1) should vary. We expected fluorescent emission to be correlated to promoter strength for each construction as the promoter strength has an influence over the expression level of the GFP gene and fluorescence is proportional to GFP quantity in the cell. However it is important to keep in mind that even if GFP level in the cell might be correlated to promoter strength, it exists stochasticity on such expression level (Elowitz, 2002).
</P><H1><SPAN class="mw-headline" id="Results">Results</SPAN></H1><CENTER><B>Figure 1</B>: Size of the different bacterial clones</CENTER><P><I>This figures shows cells size according to the number of cells counted for samples #1. Pink: negative control; black: positive control; yellow: test device 1; blue: test device 2; green: test device 3.</I></P><CENTER><B>Figure 2</B>: Size of the different bacterial clones</CENTER><P><I>This figure shows cells size according to the number of cells counted for samples #2. Pink: negative control; black: positive control; yellow: test device 1; blue: test device 2; green: test device 3.</I></P><P><B>Fig. 1</B> and <B>Fig. 2</B> show that cell population is homogenous between every sample. Cell size is between 10<SUP>2</SUP> and 10<SUP>3</SUP> for each sample, and the number of counted cells is almost the same for every sample.
</P><CENTER><B>Figure 3</B>: GFP fluorescence intensity</CENTER><P><I>This figure shows GFP fluorescence intensity according to the number of cells counted for samples #1. Pink: negative control; black: positive control; yellow: test device 1; blue: test device 2; green: test device 3.</I></P><CENTER><B>Fig. 4</B>: GFP fluorescence intensity</CENTER><P><I>This figure shows GFP fluorescence intensity according to the number of cells counted for samples #2. Pink: negative control; black: positive control; yellow: test device 1; blue: test device 2; green: test device 3.</I></P><P><B>Fig. 3</B> and <B>Fig. 4</B> show that fluorescence intensity is correlated to promoter strength of each device. The fluorescence emission level of device 1 is more important than device 2. Device 2 presents a more important fluorescence emission level than device 3. The positive control shows a large range of fluorescence intensity, containing two peaks. This probably means that there might be two subpopulations, expressing GFP at different levels. In other terms, two colonies were probably picked up and inoculated instead of one in the liquid medium. Those two subpopulations probably do not have the same number of plasmids inside each cell, which leads to different fluorescence emission intensity. However, positive control fluorescence level is around the same as device 2. As expected, negative control does not show any significant fluorescence emission.
</P><TABLE class="wikitable"><TBODY><TR><TH>Constructions
</TH><TH>Sample
</TH><TH># of events
</TH><TH>FL1 median
</TH></TR><TR><TD rowspan="2">Negative Control
</TD><TD>#1
</TD><TD>1 000 980
</TD><TD> 13
</TD></TR><TR><TD>#2
</TD><TD>1 005 646
</TD><TD>18
</TD></TR><TR><TD rowspan="2">Positive Control
</TD><TD># 1
</TD><TD>1 002 586
</TD><TD>2 217
</TD></TR><TR><TD> #2
</TD><TD>1 004 747
</TD><TD> 3 830
</TD></TR><TR><TD rowspan="2">Test device 1
</TD><TD># 1
</TD><TD>1 028 072
</TD><TD>23 572
</TD></TR><TR><TD>#2
</TD><TD>1 021 552
</TD><TD>23 990
</TD></TR><TR><TD rowspan="2">Test device 2
</TD><TD># 1
</TD><TD>1 003 942
</TD><TD>4 571
</TD></TR><TR><TD>#2
</TD><TD>1 006 803
</TD><TD>4 458
</TD></TR><TR><TD rowspan="2">Test device 3
</TD><TD># 1
</TD><TD>1 011 107
</TD><TD> 66
</TD></TR><TR><TD># 2
</TD><TD> 1 012 481
</TD><TD>68
</TD></TR></TBODY></TABLE><P>Table 1: Detailed cytometry data
</P><P><I>This table shows detailed measurment data for each sample. Fluorescence measures are presented in arbitrary units.</I></P><P>
Detailed data (Table I) show that fluorescence intensity results does not vary between two samples of the same construction, except for the positive control. This observation has to be linked with the fact that the positive control cells population might not be homogeneous.
</P><H1><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H1><P>We were not able to give the fluorescence measurment results in absolute units, because we did not have any calibration beads that would have allowed us to transform arbitrary units into absolute units.
</P><P>The interlab study has been a challenging experience that we have enjoyed be part of. We are hopping the results we have obtained will be useful and will help to the understanding of experiment reproducibility.
</P><H1><SPAN class="mw-headline" id="Reference">Reference</SPAN></H1><P>Elowitz MB. Stochastic Gene Expression in a Single Cell. Science. 16 août 2002;297(5584):1183‑6. <A rel="nofollow" class="external free" href="http://www.sciencemag.org/cgi/doi/10.1126/science.1070919">http://www.sciencemag.org/cgi/doi/10.1126/science.1070919</A></P></DIV></DIV><DIV class=" footer"><DIV class="container"><DIV class="adress"><ADRESS>Université Paris Sud
91405 Orsay, France</ADRESS></DIV><SMALL class="envers copyright">Copyright © 2016 iGEM Paris Saclay. <STRIKE>All rights reserved</STRIKE> All inspirations encouraged.*This wiki was conceived during long nights of work and a great coffee experience in the beautiful Orsay, France.</SMALL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>