Team:Paris Saclay/CRISPR-Cas9
Toggle navigation
Home
Team
Team
Media
Collaborations
Sponsors
Acknowledgements
Project
Background
Design
CRISPR/Cas9
Strategy
Experiments
Notebook
Results
Perspective
Interlab Study
Parts
Parts
Basic Parts
Composite Parts
Human Pratices
Overview
Societal Issues of CRISPR/Cas9
Responsible Research and Innovation
GMO regulation
Integrated Practices
Engagement
Model
Attributions
Safety
CRISPR/Cas9 system
The CRISPR/Cas9 system: a very special member of the engineered nucleases family
Programmable nucleases produce site-specific DNA double-strand breaks (DSBs). These DSBs are then repaired by two endogenous mechanisms: homology-directed repair (HDR) with the presence of a gene targeting vector or a homologous donor DNA, or non-homologous end-joining (NHEJ) without complementary DNA.  These two mechanisms have several consequences: small insertions-deletions (INDEL) in the case of NHEJ repair, substitution, gene disruption, insertion, correction and chromosomal rearrangements in the case of HDR repair. The outcome of genome editing using programmable nucleases are represented in [Fig. 1] (Kim & Kim, 2014).
Figure 1 : Outcomes of genome editing using endineered nuclease
In the field of biotechnology, three applications are particularly interesting. The first one is gene disruption – gene knock out ([Fig. 1A] – small INDEL) - through error-prone NHEJ. INDELs often cause frameshifts in the coding region, which disrupt genetic information and result in gene knock out. The second one is gene insertion – knock in ([Fig. 1A] – gene or tag insertion)- through HDR. To achieve this, the nuclease is co-transfected with a targeting vector, in which the genetic segment to be incorporated is flanked by homology arms with sequences that are identical to those near the target region. The last one is gene correction and point mutagenesis ([Fig. 1A] – gene correction or point mutagenesis) through HDR. Point mutations can be corrected or single-nucleotide variations (SNP) introduced in the genome through co-delivery of programmable nucleases and targeting vectors or single-strand oligodeoxynucleotides (ssODNs). The chromosomal rearrangements [Fig. 1B & Fig. 1C] are not very used in crop improvement, but have a therapeutic interest for chromosomal diseases (Kim & Kim, 2014).
Structure of CRISPR/Cas9
Ishino et al. reported in 1987 the presence of an intriguing stretch of DNA, close to a bacterial protein gene and composed of short direct-repeats interspaced by short unique sequences (protospacers) in E. coli genome, coming from bacteriophages of plasmids. The Cas protein is an endonuclease involved in a bacterial defense mechanism against bacteriophages by DSB. This is the work of Emmanuelle Charpentier’s teams between 2011 and 2012, by describing the molecular mechanism governing the accurate positioning of the DSB in DNA in the natural CRISPR/Cas9 system, which lead to the construction of artificial CRISPR/Cas9 engineered to produce DSB at any position chosen along the DNA sequence (Quétier, 2016). This DSB is specific thanks to the presence of the sgRNA, which is why the CRISPR/Cas9 system is considered as a RGEN (RNA-guided engineered nuclease). A representation is given on [Fig. 2].
Figure 2 : Structure of CRISP/Cas9 system
Bacteria insert protospacers into their own genome to form a CRISPR (clustered regularly interspaced short palindromic repeats). In type II CRISPR systems, the transcription of a CRISPR unit leads to a RNA called crRNA, which binds by complementary in 3’ to an other RNA in 5’, called tracrRNA. Once these two RNAs are complexed, the new structure called sgRNA is blocked in the catalytic site of the Cas9 protein, to form an active DNA endonuclease, which is often termed dualRNA–Cas9. This complex can bind to DNA and recognize a 23bp target DNA sequence that is composed of the 20bp guide sequence in the crRNA (the protospacer) and the 5’-X20NGG-3’ sequence known as protospacer adjacent motif (PAM), which is recognized by Cas9 itself (Kim & Kim, 2014).
References
Kim H, Kim J-S. A guide to genome engineering with programmable nucleases. Nature Reviews Genetics. 2 avr 2014;15(5):321‑34. http://www.nature.com/nrg/journal/v15/n5/full/nrg3686.html
Quétier F. The CRISPR-Cas9 technology: Closer to the ultimate toolkit for targeted genome editing. Plant Science. janv 2016;242:65‑76. http://www.sciencedirect.com/science/article/pii/S0168945215300613
Université Paris Sud
91405 Orsay, France
Copyright © 2016 iGEM Paris Saclay. All rights reserved All inspirations encouraged.
*This wiki was conceived during long nights of work and a great coffee experience in the beautiful Orsay, France.
