<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Saclay_Description_Background skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Saclay/Description/Background</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid nav-container"><NAV class="navbar navbar-default navbar-inverse affix-top" data-spy="affix" data-offset-top="260"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav center-block"><LI><A href="https://2016.igem.org/Team:Paris_Saclay">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Media">Media</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Acknowledgements">Acknowledgements</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Background">Background</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/CRISPR-Cas9">CRISPR/Cas9</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Strategy">Strategy</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Description">Results</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Perspective">Perspective</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Interlab_Study">Interlab Study</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Composite_Part">Composite Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Pratices </A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Human_Practices">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/HP/Silver">Societal Issues of CRISPR/Cas9</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/HP/Gold">Responsible Research and Innovation</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Human_Practices/GMO_Regulation">GMO regulation</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Safety">Safety</A></LI></DIV></DIV></NAV></DIV><DIV class="container content"><DIV id="compl"><H1 id="titreps">{{{titre}}}</H1></DIV><DIV id="main"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Promoters_and_DNA_topology.2C_towards_novel_epigenetic_engineering"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Promoters and DNA topology, towards novel epigenetic engineering</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Introduction..E2.80.A8"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext"><B>Introduction. </B></SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#DNA_binding_proteins."><SPAN class="tocnumber">3</SPAN><SPAN class="toctext"><B>DNA binding proteins.</B></SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Competition_for_binding_within_a_cell."><SPAN class="tocnumber">4</SPAN><SPAN class="toctext"><B>Competition for binding within a cell.</B></SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Transcription_initiation."><SPAN class="tocnumber">5</SPAN><SPAN class="toctext"><B>Transcription initiation.</B></SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Question"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext"><B>Question</B></SPAN></A></LI></UL></DIV><H3><SPAN class="mw-headline" id="Promoters_and_DNA_topology.2C_towards_novel_epigenetic_engineering">Promoters and DNA topology, towards novel epigenetic engineering</SPAN></H3><H1><SPAN class="mw-headline" id="Introduction..E2.80.A8"><B>Introduction. </B></SPAN></H1><P>Transcription is the first step of the Central dogma of molecular biology. It is the process in which an RNA molecule is created based on the sequence of a DNA template [Fig. 1]. In this case the template is a gene, a region of DNA who’s RNAs can be translated into proteins, which perform different functions within the cell. 
Because the environment of bacteria is constantly changing, cells have to adapt. A part of adaptation consists in using different proteins present at different times in the cell. These changes can be the result of transcriptional regulation. 
</P><P>Numerous molecules and factors are involved in the mechanism of transcription and can be involved in its regulation. Among these, RNA polymerase is the protein that performs transcription [Fig 1]. Transcription factors are proteins that enhance or prevent transcription; other factors such as small molecules, DNA sequence properties, and chromosome structure also play a role even though these mechanisms are diverse and often poorly understood (Browning 2002) [Fig. 2]. Sequences called promoters are small DNA sequences upstream of genes. RNA polymerase binds to them before starting transcription. 
</P><P>
RNA polymerase is composed of several parts, each of them called a “subunit”. In bacteria, it’s one of these subunits that binds first to the promoter. It is called “σ-factor”. But other factors can be necessary to begin transcription or to prevent it, especially when the promoter isn’t strong enough to keep the σ-factor attached to it. All of the proteins that can bind to DNA are called DNA binding protein. The binding of a multisubunit RNA polymerase or general transcription factors to a specialized transcription promoter DNA sequence is an essential step in initiating DNA transcription in all organisms (Friedman 2013).
</P><H1><SPAN class="mw-headline" id="DNA_binding_proteins."><B>DNA binding proteins.</B></SPAN></H1><P>These DNA binding proteins are able to bind to DNA with different strengths also called affinities. Some proteins bind DNA with the same affinity regardless of the DNA sequence, others can only be recruited by specific sequences. Four mechanisms by which a DNA binding protein finds its target sequence are known: sliding, when the protein slides along DNA [Fig. 3a]; hopping, when the protein unbinds a DNA segment and binds to another one [Fig. 3b]; intersegment transfer, when the protein binds two different segments of DNA before releasing one; and looping, which consists on the binding of two DNA segments leading to a loop (Bustamante 1999, Bondos 2015) [Fig. 3Cd]. Among these, only the one-dimensional sliding is independent of spatial organization in the chromosome, that’s why the structural organization of genomes is very important for transcriptional regulation. 
</P><H1><SPAN class="mw-headline" id="Competition_for_binding_within_a_cell."><B>Competition for binding within a cell.</B></SPAN></H1><P>These four mechanisms allow for dynamic binding of proteins to their target sequences, despite the fact that these proteins are not present in large concentrations. For example, the number of genes in Escherichia coli  is about 5 000 and the number of active RNA polymerases is about 200 per cell, of which only 140 transcribe 2% of the genes leaving 60 polymerases for the other 4995 genes.
Thus, there is tight competition between genes to get transcribed: competition between the same σ-factor for several promoters, competition between σ-factors for the rest of the RNA polymerase or competition between a σ-factor and other DNA binding proteins.
These competitions are fundamental in the mechanism of transcriptional regulation. They are often allowed or prevented by remodeling of the chromosome topology (Haugen 2008) [Fig. 4].
</P><P>That’s why, the link between genome topology and gene regulation is a major research topic in the subject of transcription. Although the mechanism of global gene regulation by changes in DNA organization are well established, the fine tuning of transcription is particularly hard to study and not very well understood.  Considering this gap in knowledge, new studies have to be without a priori in order to determine if fine tuning of the DNA topology takes place during transcriptional regulation and if so, how does such regulation work.
</P><H1><SPAN class="mw-headline" id="Transcription_initiation."><B>Transcription initiation.</B></SPAN></H1><P>Regulation can occur at every step on the pathway to gene expression, but transcription initiation is probably the most frequently regulated step (Haugen 2008). It is the result of 1) the recognition of a promoter by a σ-factor, 2) the recruitment of the other parts of the RNA polymerase, 3) the formation of an open complex, thus the unwinding of DNA from either side of the promoter 4) the beginning of RNA synthesis and finally 5) the release of the σ-factor (Browning 2002) [Fig. 6].
The first step, in which the σ-factor binds to the promoter, can limit the transcription rate. Moreover one σ-factor can lead to the beginning of transcription of a gene, be released and lead to the beginning of the transcription of an other one. That’s why we think that spatially close promoters can act on each others, for example by doubling the number of sequences which can recruit a σ-factor in a certain area. A strong promoter could recruit a σ-factor and after its release, this σ-factor would have a better probability of being recruited to a weaker promoter. Conversely, a strong promoter could have a better chance of recruiting a σ-factor leading to “starving” of the weaker promoters within the same area.
</P><H1><SPAN class="mw-headline" id="Question"><B>Question</B></SPAN></H1><P>     To understand transcriptional regulation, we have to study chromosome topology in 4-dimensions (3D + time) and its effect on the recruitment of DNA binding proteins. In this project, we try to answer to a fundamental question: Are proximal promoters influencing each other’s transcriptional rates? [Fig. 5].
</P></DIV></DIV><DIV class=" footer"><DIV class="container"><DIV class="adress"><ADRESS>Université Paris Sud
91405 Orsay, France</ADRESS></DIV><SMALL class="envers copyright">Copyright © 2016 iGEM Paris Saclay. <STRIKE>All rights reserved</STRIKE> All inspirations encouraged.*This wiki was conceived during long nights of work and a great coffee experience in the beautiful Orsay, France.</SMALL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>