Team:Paris Saclay/Strategy
Toggle navigation
Home
Team
Team
Media
Collaborations
Sponsors
Acknowledgements
Project
Background
Design
CRISPR/Cas9
Strategy
Experiments
Notebook
Results
Perspective
Interlab Study
Parts
Parts
Basic Parts
Composite Parts
Human Pratices
Overview
Societal Issues of CRISPR/Cas9
Responsible Research and Innovation
GMO regulation
Integrated Practices
Engagement
Model
Attributions
Safety
Experimental Strategy
We designed a bring DNA closer tool (BDC tool) and a visualization tool, as mentioned perviously . In order to characterize our tools, we set up the experimental strategy explained bellow.
Contents
1 Characterization strategy
1.1 Tripartite Split-GFP and FRB*-FKBP12 dimerization systems
1.2 Assessment of the minimal distance to have fluorescence
1.3 Assessment of the DNA regions brought closer
2 Gene expression tests
Characterization strategy
Tripartite Split-GFP and FRB*-FKBP12 dimerization systems
First, we designed two biobricks to test the FRB*/FKBP12 interaction and the tripartite GFP system. FRB* was fused with one subunit of the gene encoding the GFP (GFP 11) and FKBP12 was fused with another subunit (GFP10) [Fig1]. Then, we also put the gene encoding the last subunit (GFP 1-9) in the plasmid pSB1C3 to form the tripartite GFP [Fig2].
Figure 1: Tripartite split-GFP and FRB*/FKBP12 functional assessment. Biobricks corresponding to intermediate characterization. FRB* is fused with GFP11 and FKBP12 is fused with GFP10.
In order to test the system, we built a plasmid containing three biobricks to express the full system [Fig2]. Then we transformed it in E. coli to assess the system. The system would be tested by measuring GFP fluorescence level under two different conditions [Fig1]: a growth medium containing rapalog (rapamycin analog) and a growth medium without it. We also planed to test it in bacteria containing just two parts (FRB-GFP11 and FKBP12-GFP10) instead of three (so without GFP1-9).
Figure 2: Biobrick design containing all the parts to characterize tripartite split-GFP and FRB*/FKBP12. All our biobricks in one plasmid and no interference.
This construction would give us the first results, validate the functionality of the tripartite GFP and the dimerization of FRB* and FKBP12.
Assessment of the minimal distance to have fluorescence
One of the goals of our project is to assess the system composed of the BDC tool and the tripartite split-GFP. To evaluate the effect of the bring DNA closer tool, we have to know the minimal distance needed to observe fluorescence emission.
This question was also the core of our model , which answers the question: What is the optimal distance between the two dCas9s to observe fluorescence?
This question is essential because the distance between the dCas9s may cause major problems. First, steric hindrance and dCas9 footprint may avoid GFP assembling if we target sequences that are too close. Secondly, the proteins size could prevent the GFP parts from assembling if they are too far away. As a result, fluorescence emission would be detected only if the proteins, as well as the DNA regions, are at a precise range of distance.
To experimentally assess such distance, we decided to design different plasmids containing the visualization target sequences separated from each other by different number of base pairs [Fig3]. To do so, we designed specific primers to carry out reverse PCR and obtain, from a plasmid in which the target sequences are distant by 1kB, different plasmids where the number of base pairs between the target sequences is reduced.  This plasmid was thought to be expressed with the plasmid pSB1C3 containing the BioBricks 3, 4 and 5 (cf design page). The target sequences would have been separated by:
1kB
500bp
150bp
75bp
50bp
Figure 3: Plasmid design to assess the minimal distance needed to have GFP fluorescence and fully characterize the tripartite split-GFP. It would be expressed with the visualization tool BioBrick. The different RT-PCRs would allow us to have different distances between the two target sequences.
Assessment of the DNA regions brought closer
In order to test our BDC tool, all the biobricks previously quoted in the design page should be expressed in E. coli, as well as all the sgRNAs (corresponding to the target sequences and their cognate dCas9s). Later, the team would measure GFP fluorescence levels in growth medium with or without rapalog. The emission of any fluorescence by the tripartite split-GFP would validate our BDC tool [Fig4] as it would mean the two target sequences are close.
Figure 4: BDC tool characterization by the visualization tool mechanism. Using both tools together.
Gene expression tests
In order to test a possible influence of the spatial proximity in gene expression, we would test the expression of two different reporter genes. With the aim of having more accurate variation measurements, we should use enzymes such as luciferase or beta-galactosidase.
Université Paris Sud
91405 Orsay, France
Copyright © 2016 iGEM Paris Saclay. All rights reserved All inspirations encouraged.
*This wiki was conceived during long nights of work and a great coffee experience in the beautiful Orsay, France.
