Team:Paris Saclay/Background
Toggle navigation
Home
Team
Team
Media
Collaborations
Sponsors
Acknowledgements
Project
Background
Design
CRISPR/Cas9
Strategy
Experiments
Notebook
Results
Perspective
Interlab Study
Parts
Parts
Basic Parts
Composite Parts
Human Pratices
Overview
Societal Issues of CRISPR/Cas9
Responsible Research and Innovation
GMO regulation
Integrated Practices
Engagement
Model
Attributions
Safety
Background
Contents
1 Overview
2 Theoretical basis
2.1 The transcription and its regulation
2.2 DNA binding proteins
2.3 Competition for binding within a cell
2.4 Transcription initiation
2.5 DNA topology in bacteria
3 Question
4 References
Overview
Recent observations lead to the idea that genes, not in the same operon but spatially close, are highly co-transcribed, even in the absence of regulatory factors at their promoter regions.
The iGEM Paris-Saclay project aims at studying the effects of DNA topology on gene expression in E. coli by answering this question:
Does bringing two differentialy regulated promoters close to each others influences the expression level of genes located downstream?
We have designed a new tool based on the CRISPR/Cas9 system to bring two specific DNA regions closer. This system is composed of two different dCas9s fused with each part of the FRB / FKBP12 dimerization system. Each dCas9 will target a specific DNA sequence, one on the chromosome and one on a plasmid, whereas the dimerization system will promote the joining of the two dCas9s when rapalog is added.
In order to assess whether or not this system works, we have also designed a new tool to visualize the interaction between both dCas9s. This tool is composed of a split GFP attached to two dCas9s. These two small GFP tags will interact with the complementary GFP detector only if the two dCas9s are close enough to interact.
Theoretical basis
The transcription and its regulation
Transcription is the first step of the Central dogma of molecular biology. It is the process in which an RNA molecule is created based on the sequence of a DNA template. In this case the template is a gene, a region of DNA whose RNAs can be translated into proteins, which perform different functions within the cell.  Because the environment of bacteria is constantly changing, cells have to adapt. A part of adaptation consists in using different proteins present at different times in the cell. These changes can be the result of transcriptional regulations.
Sequences called promoters are small DNA sequences upstream of genes. RNA polymerase is the protein that performs transcription by binding to promoters before starting transcription. RNA polymerase is composed of several parts, each of them called a “subunit”. In bacteria, it is one of these subunits that binds first to the promoter. It is called “σ-factor” [Fig. 1].
Fig 1 : DNA transcription by RNA polymerase.
a. Binding of RNA polymerase on a promoter, a specific sequence on the DNA. b. Transcription of the DNA template into a RNA molecule
But other factors can be necessary to begin transcription or to prevent it, especially when the promoter is not strong enough to keep the σ-factor attached to it. All of the proteins that can bind to DNA are called DNA binding proteins. The binding of a multisubunit RNA polymerase or general transcription factors to a specialized transcription promoter DNA sequence is an essential step in initiating DNA transcription in all organisms (Friedman 2013).
Numerous molecules and factors are involved in the mechanism of transcription and can be involved in its regulation.  Transcription factors are proteins that enhance or prevent transcription; other factors such as small molecules, DNA sequence properties, and chromosome structure also play a role even though these mechanisms are diverse and often poorly understood (Browning 2002) [Fig. 2].
Fig 2 : Transcriptional regulation.
Several mechanisms play a role in transcriptional regulation. Transcription factors (blue circles) can inhibit or enhance transcription rate.
DNA binding proteins
These DNA binding proteins are able to bind to DNA with different strengths also called affinities. Some proteins bind DNA with the same affinity regardless of the DNA sequence, others can only be recruited by specific sequences. Four mechanisms by which a DNA binding protein finds its target sequence are known: sliding, when the protein slides along DNA [Fig. 3a]; hopping, when the protein unbinds a DNA segment and binds to another one [Fig. 3b]; intersegment transfer, when the protein binds two different segments of DNA before releasing one; and looping, which consists on the binding of two DNA segments leading to a loop (Bustamante 1999, Bondos 2015) [Fig. 3c,d]. Among these, only the one-dimensional sliding is independent of spatial organization in the chromosome, that’s why the structural organization of genomes is very important for transcriptional regulation.
Figure 3 : Four mechanisms allow DNA-binding proteins to find target sequences.
a. Sliding of RNA polymerase on DNA. b. Hopping of a DNA-binding protein from a site to an other. c. Intersegment transfer of  two DNA binding proteins (dark blue circles) linked by another protein (blue circle) from a DNA segment to an other. d. Looping of DNA allowed by a DNA binding protein
Competition for binding within a cell
These four mechanisms allow for dynamic binding of proteins to their target sequences, despite the fact that these proteins are not present in large concentrations. For example, the number of genes in Escherichia coli is about 5 000 and the number of active RNA polymerases is about 200 per cell, of which  140 only transcribe 2% of the genes leaving 60 polymerases for the other 4995 genes.
Thus, there is tight competition between genes to get transcribed: competition between the same σ-factor for several promoters, competition between σ-factors for the rest of the RNA polymerase or competition between a σ-factor and other DNA binding proteins. These competitions are fundamental in the mechanism of transcriptional regulation. They are often allowed or prevented by remodeling of the chromosome topology (Haugen 2008) [Fig. 3].
This is why the link between genome topology and gene regulation is a major research topic in the subject of transcription. Although the mechanisms of global gene regulation by changes in DNA organization are well established, the fine tuning of transcription is particularly hard to study and not very well understood.  Considering this gap in knowledge, new studies have to be conducted without a priori in order to determine if fine tuning of the DNA topology takes place during transcriptional regulation and if so, how does such regulation work.
Transcription initiation
Transcriptional regulation can occur at every step on the pathway to gene expression, but transcription initiation is probably the most frequently regulated step (Haugen 2008). It is the result of:
The recognition of a promoter by a σ-factor;
The recruitment of the other parts of the RNA polymerase;
The formation of an open complex, thus the unwinding of DNA from either side of the promoter;
The beginning of RNA synthesis and finally;
The release of the σ-factor (Browning 2002) [Fig. 4].
The first step, in which the σ-factor binds to the promoter, can limit the transcription rate. Moreover one σ-factor can lead to the beginning of transcription of a gene, be released and lead to the beginning of the transcription of an other one. As a result, we think that spatially close promoters can act on each others, for example by doubling the number of sequences which can recruit a σ-factor in a certain area. A strong promoter could recruit a σ-factor and after its release, this σ-factor would have a better probability of being recruited to a weaker promoter. Conversely, a strong promoter could have a better chance of recruiting a σ-factor leading to “starving” of the weaker promoters within the same area. This could be an other unstudied link between topology and transcriptional regulation.
Figure 4 : Transcription initiation mechanism.
1. Binding of a σ-factor on a promoter. 2. Recruitment of RNA polymerase subunits on the σ-factor. 3. Open complex formation and beginning of transcription. 4.  Release of the σ-factor and elongation
DNA topology in bacteria
As mentioned above, DNA topology plays a fundamental role in gene regulation and in protein interactions after their synthesis. A growing number of studies have shown a  periodically structure of bacterial chromosome in solenoid or in plectoneme at some sub levels of organization [Fig.5]. Such structures could be selected to keep gene pairs close to each other. Indeed, some authors (Wright and al. 2007) have shown that gene pairs are often distant from each others by 117 kb, resulting in possible helix-like structures with this periodicity.
Figure 5:  Possible chromosome conformation and conformational dynamism, From Wright et al. 2007.
« Curves represent genomic DNA that is compacted 21-fold and may consist of substructures including topological domains. The dimensions of the helices are 2.5 µm high with a radius of 0.3 µm, compatible with an average E. coli cell. Consecutive dots along the helices represent 1 kb of compacted genomic DNA. In this configuration the chromosome would be coiled into a single stack of 117-kb loops, with the origin and terminus at midcell. Note that several configurations are possible because of dynamical changes. The gene pair density  is indicated by increasing dot size.»
O: origin of replication, T: terminus region
Because of this periodicity, regulation could be effective in particular area of the chromosome, leading to a fine, but integral, tuning of the genes involved in the same pathway. That's why concepts such as regulons (genes that are usually geographically scattered but under the control of the same regulatory factor), who were initially described with a 2D vision of the bacterial chromosomes, could be specified with the incorporation of new researches. Theories behind these concepts focused on protein-protein interactions after translation, which occurs at the same time as transcription in bacteria. Indeed, products of gene-coding-protein are synthesized  near their gene and a pairing of genes can results in an increasing probability for the two proteins to interact with each others [Fig.6] (Dorman 2013).
Figure 6 : Nucleoid folding and gene regulation, From Dorman 2013.
« A simple regulon consisting of a regulatory gene and two structural operons, A and B, is illustrated in various conformations.  When the chromosome is represented in a one‐dimensional, linear form, the three genetic loci are separated by large distances in space. However, when the chromosome is reorganized as a solenoid (left) or as a plectoneme (right), the periodicity of these structures brings the three genes close together, between the regulatory gene and its two target operons. Moreover, the products of the A and B operons are produced in close proximity, favouring their interaction.»
In the same way a single sigma factor, specific of a group of genes, could be recruited from a promoter to another easily because of the DNA topology. Thus, gene pair associations could provide the cell with another level of regulation of transcription. These genes associations could not only be significant for protein-protein interactions after translation but could also be significant to enhance or to inhibit transcription initiation.
Question
To understand transcriptional regulation, we have to study chromosome topology in 4-dimensions (3D + time) and its effect on the recruitment of DNA binding proteins. In this project, we try to answer to a fundamental question: are proximal promoters influencing each other’s transcriptional rates? [Fig. 8].
Figure 7: Gene pairs and transcriptional regulation.
a. Binding of a sigma factor on gene A's promoter. b. Recruitment of the other parts of the RNA polymerase (red and yellow) on the sigma factor. c. Gene A's transcription begins, producing a RNA (orange) molecule. The sigma factor is released and binds an other close promoter on a pair gene, allowing the formation of an active RNA polymerase. d. Termination of gene A's transcription, an entire RNA molecule is released. Beginning of gene B's transcription, producing a RNA (dark green) molecule. The sigma factor is released.
References
Bondos, S.E., Swint-Kruse, L., and Matthews, K.S. (2015). Flexibility and Disorder in Gene Regulation: LacI/GalR and Hox Proteins. Journal of Biological Chemistry 290, 24669–24677.
Browning, D.F., and Busby, S.J.W. (2004). The regulation of bacterial transcription initiation. Nature Reviews Microbiology 2, 57–65.
Cases, I., and de Lorenzo, V. (2005). Promoters in the environment: transcriptional regulation in its natural context. Nature Reviews Microbiology 3, 105–118.
Dorman, C.J. (2013). Genome architecture and global gene regulation in bacteria: making progress towards a unified model? Nature Reviews Microbiology 11, 349–355.
Elowitz, Levine, Siggia, and Swain (2002). Stochastic Gene Expression in a Single Cell. Science.
Esvelt, K.M., Mali, P., Braff, J.L., Moosburner, M., Yaung, S.J., and Church, G.M. (2013). Orthogonal Cas9 proteins for RNA-guided gene regulation and editing. Nature Methods 10, 1116–1121.
Friedman, L.J., Mumm, J.P., and Gelles, J. (2013). RNA polymerase approaches its promoter without long-range sliding along DNA. Proceedings of the National Academy of Sciences 110, 9740–9745.
Haugen, S.P., Ross, W., and Gourse, R.L. (2008). Advances in bacterial promoter recognition and its control by factors that do not bind DNA. Nature Reviews Microbiology 6, 507–519.
Ishihama, A. (2000). Functional modulation of Escherichia coli RNA polymerase. Annu. Rev. Microbiol. 54, 499–518.
Lagomarsino, M.C., Espéli, O., and Junier, I. (2015). From structure to function of bacterial chromosomes: Evolutionary perspectives and ideas for new experiments. FEBS Letters 589, 2996–3004.
Rocha, E.P.C. (2008). The Organization of the Bacterial Genome. Annual Review of Genetics 42, 211–233.
Rud, I. (2006). A synthetic promoter library for constitutive gene expression in Lactobacillus plantarum. Microbiology 152, 1011–1019.
Sanli, I., and Feil, R. (2016). PRC-mediated interaction networks of repressed genes: emerging insights and possible roles. Epigenomics 8, 733–735.
Sobetzko, P., Travers, A., and Muskhelishvili, G. (2012). Gene order and chromosome dynamics coordinate spatiotemporal gene expression during the bacterial growth cycle. Proceedings of the National Academy of Sciences 109, E42–E50.
Travers, A., and Muskhelishvili, G. (2005). DNA supercoiling — a global transcriptional regulator for enterobacterial growth? Nature Reviews Microbiology 3, 157–169.
Wright, M.A., Kharchenko, P., Church, G.M., and Segrè, D. (2007). Chromosomal periodicity of evolutionarily conserved gene pairs. Proceedings of the National Academy of Sciences 104, 10559–10564.
Université Paris Sud
91405 Orsay, France
Copyright © 2016 iGEM Paris Saclay. All rights reserved All inspirations encouraged.
*This wiki was conceived during long nights of work and a great coffee experience in the beautiful Orsay, France.
