<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Saclay_CRISPR-Cas9 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Saclay/CRISPR-Cas9</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid nav-container"><NAV class="navbar navbar-default navbar-inverse affix-top" data-spy="affix" data-offset-top="260"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav center-block"><LI><A href="https://2016.igem.org/Team:Paris_Saclay">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Media">Media</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Acknowledgements">Acknowledgements</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Background">Background</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/CRISPR-Cas9">CRISPR/Cas9</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Strategy">Strategy</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Description">Results</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Perspective">Perspective</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Interlab_Study">Interlab Study</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Composite_Part">Composite Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Pratices </A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Human_Practices">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/HP/Silver">Societal Issues of CRISPR/Cas9</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/HP/Gold">Responsible Research and Innovation</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Human_Practices/GMO_Regulation">GMO regulation</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Safety">Safety</A></LI></DIV></DIV></NAV></DIV><DIV class="container content"><DIV id="compl"><H1 id="titreps">CRISPR/Cas9 system</H1></DIV><DIV id="main"><H1><SPAN class="mw-headline" id="The_CRISPR.2FCas9_system:_a_very_special_member_of_the_engineered_nucleases_family">The CRISPR/Cas9 system: a very special member of the engineered nucleases family</SPAN></H1><P>Programmable nucleases produce site-specific DNA double-strand breaks (DSBs). These DSBs are then repaired by two endogenous mechanisms: homology-directed repair (HDR) with the presence of a gene targeting vector or a homologous donor DNA, or non-homologous end-joining (NHEJ) without complementary DNA.  These two mechanisms have several consequences: small insertions-deletions (INDEL) in the case of NHEJ repair, substitution, gene disruption, insertion, correction and chromosomal rearrangements in the case of HDR repair. The outcome of genome editing using programmable nucleases are represented in <B>[Fig. 1]</B> (Kim &amp; Kim, 2014).
</P><CENTER><B>Figure 1</B> : Outcomes of genome editing using endineered nuclease</CENTER><P>
In the field of biotechnology, three applications are particularly interesting. The first one is gene disruption – gene knock out (<B>[Fig. 1A]</B> – small INDEL) - through error-prone NHEJ. INDELs often cause frameshifts in the coding region, which disrupt genetic information and result in gene knock out. The second one is gene insertion – knock in (<B>[Fig. 1A]</B> – gene or tag insertion)- through HDR. To achieve this, the nuclease is co-transfected with a targeting vector, in which the genetic segment to be incorporated is flanked by homology arms with sequences that are identical to those near the target region. The last one is gene correction and point mutagenesis (<B>[Fig. 1A]</B> – gene correction or point mutagenesis) through HDR. Point mutations can be corrected or single-nucleotide variations (SNP) introduced in the genome through co-delivery of programmable nucleases and targeting vectors or single-strand oligodeoxynucleotides (ssODNs). The chromosomal rearrangements <B>[Fig. 1B &amp; Fig. 1C</B>] are not very used in crop improvement, but have a therapeutic interest for chromosomal diseases (Kim &amp; Kim, 2014).
</P><H1><SPAN class="mw-headline" id="Structure_of_CRISPR.2FCas9">Structure of CRISPR/Cas9</SPAN></H1><P>Ishino et al. reported in 1987 the presence of an intriguing stretch of DNA, close to a bacterial protein gene and composed of short direct-repeats interspaced by short unique sequences (protospacers) in <I>E. coli</I> genome, coming from bacteriophages of plasmids. The Cas protein is an endonuclease involved in a bacterial defense mechanism against bacteriophages by DSB. This is the work of Emmanuelle Charpentier’s teams between 2011 and 2012, by describing the molecular mechanism governing the accurate positioning of the DSB in DNA in the natural CRISPR/Cas9 system, which lead to the construction of artificial CRISPR/Cas9 engineered to produce DSB at any position chosen along the DNA sequence (Quétier, 2016). This DSB is specific thanks to the presence of the sgRNA, which is why the CRISPR/Cas9 system is considered as a RGEN (RNA-guided engineered nuclease). A representation is given on <B>[Fig. 2]</B>.
</P><CENTER><B>Figure 2</B> : Structure of CRISP/Cas9 system</CENTER><P>
Bacteria insert protospacers into their own genome to form a CRISPR (clustered regularly interspaced short palindromic repeats). In type II CRISPR systems, the transcription of a CRISPR unit leads to a RNA called crRNA, which binds by complementary in 3’ to an other RNA in 5’, called tracrRNA. Once these two RNAs are complexed, the new structure called sgRNA is blocked in the catalytic site of the Cas9 protein, to form an active DNA endonuclease, which is often termed dualRNA–Cas9. This complex can bind to DNA and recognize a 23bp target DNA sequence that is composed of the 20bp guide sequence in the crRNA (the protospacer) and the 5’-X20NGG-3’ sequence known as protospacer adjacent motif (PAM), which is recognized by Cas9 itself (Kim &amp; Kim, 2014).
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>Kim H, Kim J-S. A guide to genome engineering with programmable nucleases. Nature Reviews Genetics. 2 avr 2014;15(5):321‑34. <A rel="nofollow" class="external free" href="http://www.nature.com/nrg/journal/v15/n5/full/nrg3686.html">http://www.nature.com/nrg/journal/v15/n5/full/nrg3686.html</A></P><P>Quétier F. The CRISPR-Cas9 technology: Closer to the ultimate toolkit for targeted genome editing. Plant Science. janv 2016;242:65‑76. <A rel="nofollow" class="external free" href="http://www.sciencedirect.com/science/article/pii/S0168945215300613">http://www.sciencedirect.com/science/article/pii/S0168945215300613</A></P></DIV></DIV><DIV class=" footer"><DIV class="container"><DIV class="adress"><ADRESS>Université Paris Sud
91405 Orsay, France</ADRESS></DIV><SMALL class="envers copyright">Copyright © 2016 iGEM Paris Saclay. <STRIKE>All rights reserved</STRIKE> All inspirations encouraged.*This wiki was conceived during long nights of work and a great coffee experience in the beautiful Orsay, France.</SMALL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>