Team:Paris Saclay/Design
Toggle navigation
Home
Team
Team
Media
Collaborations
Sponsors
Acknowledgements
Project
Background
Design
CRISPR/Cas9
Strategy
Experiments
Notebook
Results
Perspective
Interlab Study
Parts
Parts
Basic Parts
Composite Parts
Human Pratices
Overview
Societal Issues of CRISPR/Cas9
Responsible Research and Innovation
GMO regulation
Integrated Practices
Engagement
Model
Attributions
Safety
Tools Design
Contents
1 Introduction
2 Molecular tools
2.1 DNA recognition and fixation
2.2 Dimerization system
2.3 Fluorescence system
3 Bring DNA Closer tool construction
4 Visualization tool construction
5 References
Introduction
The project aims at studying how bacterial DNA organization can influence gene expression. In order to answer this question, we decided to create a new tool based on the CRISPR/Cas9 technology in order to bring closer two DNA regions separated in space. This new tool is expected to assess the effect of DNA structure on gene expression. We designed a Bring DNA closer tool (BDC tool) and a visualization tool.
The Escherichia coli bacterium was chosen to be our chassis for this project. Indeed, it is the best-known and a well documented bacterial model (Dorman, 2013) to answer fundamental questions such as the one that our project raises. It will be easier to make a proof of concept in E. coli and then optimize and expand the use of our tool to other species or other microorganisms. Furthermore, the use of E. coli, which is the most common chassis in iGEM, allows us to use biological parts that were constructed and optimized for this bacterium, such as promoters, RBS, terminators, or protein coding sequences.
In this part, we expose our final tool design. Our experimental strategy is available here .
Molecular tools
In order to design both our tools, we need molecular tools with specific functions: DNA recognition, DNA binding function, dimerization and fluorescence.
DNA recognition and fixation
Both tools have to recognize a specific target sequence on DNA and then bind to it. Several options were possible, such as a protein-DNA or a RNA-DNA recognition. First of all, we thought about using engineered nuclease proteins, such as zing-finger nuclease (ZFN), transcription activator-like effector nuclease (TALEN) or a RNA-guided engineered nuclease (RGEN) for their ability to target a specific sequence thanks to a specific proteic or nucleic sequence (for TALEN for instance, we speak about TALE code). We finally chose a CRISPR/Cas9 recognition system with an endonuclease deficient Cas9 (dCas9). For more information about CRISPR/Cas9 technology, click here . The CRISPR/Cas9 system is a recent tool for synthetic biology which has many applications, such as targeting specific DNA sequences. Nevertheless, our system allows a new application for the CRISPR/Cas9 technology. We chose the dCas9 for three reasons:
simplicity: to target a DNA sequence, we only have to design a specific sgRNA, which takes about a week.
adaptability: the tool would be adaptable to target any sequence in the genome through the sgRNA sequence design.
specificity: the use of orthologous dCas9s allows a specific one by one sgRNA/dCas9 recognition by a specific PAM sequence for each pair (Ma, Hanhui et al. 2015).
Dimerization system
Our project aims at recognizing specific sequences on the genome and then inducing a spatial proximity. So once we chose a system allowing  DNA targeting, we had to choose a system which could dimerize the dCas9/sgRNA, in order to link these two dCas9/sgRNA when they happen to be close enough. We chose the FRB/FKBP12 dimerization system. This system allows a dimerization induced by rapamycin. To avoid any problem of rapamycin toxicity in E. coli, we decided to use an analog of rapamycin called rapalog (gratefully offered by Takara Clontech). We then used a FRB mutated protein sequence (FRB*) that is recognized by rapalog (Bayle and al, 2006).
Fluorescence system
For the visualization tool, we needed a fluorescence system to asses a spatial proximity induced by our system. We chose the tripartite split-GFP, which is divided into three different parts: two parts of twenty amino acids corresponding to the 10th and 11th beta-sheets and a third part corresponding to the other beta-sheets of the GFP. This solution avoids poor folding and/or self-assembly background fluorescence. Indeed, the different subunits do not have affinity for each other but can only assemble and emit fluorescence upon excitation when they are all spatially close, which is not the case for a bipartite GFP. With this system, only two sgRNAs/dCas9s fused to their specific GFP tags are necessary to obtain an accurate fluorescence (Nguyen and al, 2013).
Bring DNA Closer tool construction
We have designed a tool based on the CRISPR/Cas9 system to target a specific DNA sequence. We imagined a system that dimerizes under an induction signal to bring two DNA sequences closer.
dCas9 is a protein which precisely recognizes a DNA sequence (contrary to Cas9, it has no nuclease activity) [Fig1]. We chose this system for its high adaptability, as it targets DNA through a sgRNA which could be easily designed. For a reason you will understand very quickly, we chose two orthologous dCas9s which are respectively expressed by T. denticola (TD) and S. pyogenes (SP). As they come from different organisms, they recognize different sgRNAs thanks to their PAM sequence. We ordered from Addgene the plasmids coding for each one of these dCas9s and their sgRNAs.
Figure 1: Specific DNA sequence recognition by our tool based on sgRNA/dCas9 assembling. Association of the heterocomplex composed of dCas9s and their sgRNAs recognizing specific target sequences. Colors correspond to two orthologous dCas9
To dimerize these two dCas9s, we chose an inducible system using the FRB and FKBP12 proteins [Figure 2]. Originally found in mammals, these two proteins form an heterodimer when rapamycin is added in the middle. It is particularly used in protein interaction studies (Cui et al., 2014). However, rapamycin is toxic for bacteria, but studies showed that a mutated FRB (FRB*) still allows dimerization with a non toxic analog of rapamycin called rapalog. The mutations are: T2098L, K2095P, W2101F (Bayle et al., 2006; Liberles, Diver, Austin, & Schreiber, 1997).
A biobrick coding for FRB with the mutation T2098L was already in the iGEM registry (iGEM Part_ J18926) but was not available. Moreover, it contains only one mutation on the 3 described in the literature. So we decided to work with the fully mutated FRB. The rapalog and the plasmid containing the mutated FRB and FKBP12 were offered from Takara Clontech. But as mentioned previously, this system is used in mammal cells, so we decided to optimize the sequences for an expression in E. coli with the Jcat platform. We finally ordered gBlocks coding for the FRB* and FKBP12 optimized sequences and a linker in prevision of the fusion with their respective dCas9s.
Using these two systems (dCas9 recognition and FRB/FKBP12 dimerization) we design our new tool [Figure 2 and Figure 3] based on the two following BioBricks:
Fig 2: Final biobrick designs for the BDC tool and dimerization mechanism. Color correspond to the two different orthologous dCas9s and to the heterodimerization system. dCas9 from Streptococcus pyogenes is fused to FRB* and dCas9 from Treponema denticola is fused to FKBP12. Addition of rapalog allowing dimerization.
These two biobricks will be assembled in the pSB1C3 plasmid giving us a BDC tool which would work as presented bellow:
Fig 3: BDC tool mechanism
Now you can understand why we needed two orthologous dCas9: it avoids us being in the situation where two dCas9-FRB* or two dCas9-FKBP12 are fixed on the DNA!
Visualization tool construction
As every system needs an efficient control, we set up a new part of the project in order to assess our Bring DNA Closer tool works properly: the visualization tool.
To build this new tool, two other orthologous dCas9s were used and respectively come from the organisms N. meningitidis and S. thermophilus. These dCas9s will be fused to subunits of a fluorescent protein [Fig5]. We also decided to use a dCas9 system for this tool in order to detect an accurate and unique sequence in the genome. It will be associated with a tripartite split-GFP, which has never been released in the iGEM competition.
The tripartite split-GFP is composed of two twenty amino-acids long GFP tags (GFP 10 and GFP 11) and a third complementary subsection (GFP 1-9). The tags will be fused to the two dCas9 previously quoted. A functional GFP would be formed when the tools would be close enough to allow the three split-GFP parts reunion and the fluorescence emission. This fluorescence system avoids poor folding and/or self-assembly background fluorescence. With this system, only two sgRNAs associated with their dCas9s fused to their specific GFP tags were necessary instead of nearly 30 with mundane GFP due to background fluorescence.
We designed three biobricks [Fig4] to achieve this part of the project:
BioBrick n°3: dcas9 from N.meningitidis fused to GFP-10 expressed by a constitutive promoter, a RBS and a double terminator
BioBrick n°4: dcas9 from S.thermophilus fused to GFP-11 expressed by a constitutive promoter, a RBS and a double terminator
BioBrick n°5: The third part of the GFP 1-9 expressed by a constitutive promoter, a RBS and a double terminator
The biobricks would then be inserted into pSB1C3 by digestion/ligation using the restriction sites EcoRI and PstI.
Fig 4: Biobrick designs for the visualization tool. Scheme for the three biobricks corresponding to the visualization tool. The three biobricks have the same promoter, RBS and terminators. dCas9 from Nesseria meningitidis is fused to GFP10 and dCas9 from Streptococcus thermophilus is fused to GFP11.
Then, we considered to establish a composite biobrick composed of the three biobricks in the same pSB1C3 plasmid (cf Fig5). This plasmid would have been build using the iGEM restriction site technique.
Fig 5: Visualization tool mechanism. Tripartite split-GFP allows fluorescence only if the three parts are maintain close enough.
We chose to express sgRNAs on another plasmid pZA11 (cf Fig6) which is compatible with pSB1C3, as it is resistant to ampicilin and do not have the same replication origin.
Fig 6: Plasmid containing the sgRNAs specific to their cognate dCas9 and that can be adaptable to target any sequences in the genome. This plasmid contain the part of the project that can be change and adapte to any sequences on DNA.
References
Bayle, J. H., Grimley, J. S., Stankunas, K., Gestwicki, J. E., Wandless, T. J., & Crabtree, G. R. (2006). Rapamycin analogs with differential binding specificity permit orthogonal control of protein activity. Chemistry and Biology, 13(1), 99–107. http://doi.org/10.1016/j.chembiol.2005.10.017
Chen, Baohui et al. 2013. “Resource Dynamic Imaging of Genomic Loci in Living Human Cells by an Optimized CRISPR / Cas System.” CELL 155(7): 1479–91. http://dx.doi.org/10.1016/j.cell.2013.12.001 .
Church, George M. 2014. “HHS Public AccessOrthogonal Cas9 Proteins for RNA-Guided Gene Regulation and Editing.” Nat. Methods 10(11): 1116–21.
Cui, B., Wang, Y., Song, Y., Wang, T., Li, C., Wei, Y., … Shen, X. (2014). Bioluminescence resonance energy transfer system for measuring dynamic protein-protein interactions in bacteria. mBio, 5(3), 1–10. http://doi.org/10.1128/mBio.01050-14
Dorman, C.J. (2013). Genome architecture and global gene regulation in bacteria: making progress towards a unified model? Nature Reviews Microbiology 11, 349–355.
Liberles, S. D., Diver, S. T., Austin, D. J., & Schreiber, S. L. (1997). Inducible gene expression and protein translocation using nontoxic ligands identified by a mammalian three-hybrid screen. Proceedings of the National Academy of Sciences of the United States of America, 94(15), 7825–7830. http://doi.org/10.1073/pnas.94.15.7825
Ma, Hanhui et al. 2015. “Multicolor CRISPR Labeling of Chromosomal Loci in Human Cells.” Proceedings of the National Academy of Sciences of the United States of America.
Nguyen, Hau B et al. 2013. “A New Protein-Protein Interaction Sensor Based on Tripartite Split-GFP Association.” Scientific Reports 10: 1–9.
Université Paris Sud
91405 Orsay, France
Copyright © 2016 iGEM Paris Saclay. All rights reserved All inspirations encouraged.
*This wiki was conceived during long nights of work and a great coffee experience in the beautiful Orsay, France.
