<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Saclay_Perspective skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Saclay/Perspective</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid nav-container"><NAV class="navbar navbar-default navbar-inverse affix-top" data-spy="affix" data-offset-top="260"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav center-block"><LI><A href="https://2016.igem.org/Team:Paris_Saclay">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Media">Media</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Acknowledgements">Acknowledgements</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Background">Background</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/CRISPR-Cas9">CRISPR/Cas9</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Strategy">Strategy</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Description">Results</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Perspective">Perspective</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Interlab_Study">Interlab Study</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Composite_Part">Composite Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Pratices </A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Human_Practices">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/HP/Silver">Societal Issues of CRISPR/Cas9</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/HP/Gold">Responsible Research and Innovation</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Human_Practices/GMO_Regulation">GMO regulation</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Paris_Saclay/Safety">Safety</A></LI></DIV></DIV></NAV></DIV><DIV class="container content"><DIV id="compl"><H1 id="titreps">Perspective</H1></DIV><DIV id="main"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Perspective"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Perspective</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Study_the_chromosome_structure_and_function"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Study the chromosome structure and function</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#changing_the_expression_of_any_genes:_epigenetic_engineering"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">changing the expression of any genes: epigenetic engineering</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Production_of_non-GMO_strains_for_the_industry"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Production of non-GMO strains for the industry</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#References"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></DIV><H1><SPAN class="mw-headline" id="Perspective">Perspective</SPAN></H1><P>In this project we design a tool to study the relation between DNA structure and gene expression. Here we propose many applications of our tool in genome study and industry. Using dCas9 in this system would give us the advantage to specifically target the desired sequence and change the structure of the DNA. Indeed, it would be possible to design specific sgRNA to target specific sequences.
</P><H1><SPAN class="mw-headline" id="Study_the_chromosome_structure_and_function">Study the chromosome structure and function</SPAN></H1><P>Our tool can be considered as a reporter for chromosome structural properties. According to Lagomarsino <I>et al.</I>, developing a stable reporter is the most important challenge in chromosome structural studies. <I> (Lagomarsino et al., 2015).</I>
One of the classic technics to study the contact between two arbitrary genomic loci is protein-fragment complementation assay (PCA). In PCA  two proteins of interest (targets) are genetically linked to two moieties of an enzyme (reporter) that can confer resistance to the bacterium. Because the reporter is fully assembled only when the two target proteins interact, the fraction of surviving bacteria after an antibiotic treatment provides quantitative information regarding the interaction energies between the two proteins. It is assumed that this assay may be naturally transposed to the problem of the contact between two genomic loci. 
In order to end this problem, Lagomarsino <I>et al.</I> suggesting to target directly two genomic loci. For instance, the CRISPR/Cas9 system with two different short sgRNAs could be used to direct the target, a modified Cas9 with no endonuclease property, to the genomic loci. The modified Cas9 would then be used in two different fused systems, each one containing one reporter moiety. <B>[Fig. 1]</B>.  In our project we chose tripartite split-GFP as a marker.
</P><P>A possible PCA-Cas9 assay to select bacteria on the basis of physical contacts between pairs of genomic loci. On the one hand, two short guide RNAs (sgRNA1 and sgRNA2) are used to recognize two specific genomic loci (in red and blue) distantly situated along the genome. On the other hand, the dCas9 protein is used in two different fused systems. Each fused system contains one moiety (X or Y) of a reporter protein (X–Y) that can confer either survival to the bacteria, as in a DHFR system <I> (Remy et al. 2007)</I> , or fluorescence, as in a FRET system <I> (Miyawaki et al. 2011)</I> . Both cases enable the selection of bacteria that have brought X and Y (i.e., the two genomic loci) into contact. See <I> (Mali et al. 2013)</I>  for a more general discussion about the engineering potential of CRISPR-Cas9 systems.
</P><H1><SPAN class="mw-headline" id="changing_the_expression_of_any_genes:_epigenetic_engineering">changing the expression of any genes: epigenetic engineering</SPAN></H1><P>The global goal of our project is to study the impact of the spatial organization of the chromosome on the genetic transcription. For that purpose, we designed a tool to characterize the effect of a strong promoter on a weak promoter in space. With our tool we are able to study how we can change the strength of one desired promoter according to DNA topology. This enhancement of the transcription of the gene controlled by the weak promoter could be very interesting in many fields and could be used for research and in the industry. Indeed, promoter strength, or activity, is important in genetic engineering and synthetic biology. Furthermore, this enhancement is based on the chromosome organization, the tool we designed correspond to an ‘’epigenetic’’ engineering tool.
</P><P>In order to increase the expression of a specific gene on the bacterial chromosome, a classic method is to choose one promoter with a suitable strength which can be employed to regulate the rate of transcription, which then leads to the required level of protein expression. For this purpose, so far, many promoter libraries have been established experimentally <I> (Li &amp; Zhang, 2014) </I>. Then the strategy is to clone the chosen promoter just before the gene sequence <B>[Fig. 2a]</B>. With our tool, we offer a new strategy which also involves a cloning (the plasmid of interest) but no insertion in the bacterial chromosome, which is an advantage!
</P><P>We believe that with our tool it is possible to increase or decrease the strength of a given promoter by changing the DNA topology. This method does not necessitate the insertion of a new promoter in the bacterial genome. In this method it is enough to design two sgRNAs to target two DNA regions which are situated in the optimal distance from the promoters of interest. Online bioinformatics tools are also available in order to design the sgRNAs<I> (Kim &amp; Kim, 2014) </I>. When the plasmids coding for our tools (customized with designed sgRNAs) is transformed into the bacteria, it is able to change the strength of a given promoter <B>[Fig. 2b]</B>.
</P><CENTER><B>Figure 2</B> : Different ways to enhance gene expression</CENTER><P>
(A)	Classic method: insert a plasmid to clone a strong promoter in front of the gene
</P><P>(B)	iJ’Aime method 1: insert a plasmid to express iJ’AIME tool
</P><P>(C)	iJ’Aime method 2: insert iJ’AIME ribonucleoprotein
</P><H1><SPAN class="mw-headline" id="Production_of_non-GMO_strains_for_the_industry">Production of non-GMO strains for the industry</SPAN></H1><P>As it was mentioned before, the wide majority of the strains used in the bio-industry are GMOs. To learn more about GMO regulation, <A rel="nofollow" class="external text" href="https://2016.igem.org/Team:Paris_Saclay/Human_Practices/GMO_Regulation">click here</A>.
There is a debate in the European Union in order to decide if organisms modified by CRISPR/Cas9 technology should be considered as GMO or not. Nevertheless, this debate is only for CRISPR/Cas9 doing a double strand break (DSB) in the chromosome. In our tool we use dCas9 proteins, the Cas9 protein in which the exonuclease (cutting) activity is enabled.
</P><P>
However, our system still can be considered as a genetic modification of bacteria (expression plasmid containing foreign DNA). But recent works on the use of CRISPR/Cas9 system help us to overcome this issue as well. It is possible to introduce the dCas9 ribonucleoproteins (RNP), which consists in a purified dCas9 protein in complex with a sgRNA, un the cell without doing a transformation of coding plasmids. They are assembled in vitro and can be delivered directly to cells, using standard electroporation or transfection techniques <I> (McDade and Waxmonsky ,2016) </I>.In conclusion, it is possible to deliver directly our tool by this method <B> [Fig. 2c] </B>. As no foreign DNA is introduced, the organisms manipulated by our tool should not be considered as GMOs!
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>Kim H, Kim J-S. A guide to genome engineering with programmable nucleases. <I> Nature Reviews Genetics. </I> 2 avr 2014;15(5):321‑34
</P><P>Lagomarsino, M., Espéli, O. and Junier, I. 2015. From structure to function of bacterial chromosomes: Evolutionary perspectives and ideas for new experiments. <I> FEBS letters. </I> 589, 20 Pt A (2015), 2996–3004.
</P><P>Li, J. and Zhang, Y. 2014. Relationship between promoter sequence and its strength in gene expression. <I> arXiv. </I> (2014).
Mali, P., Esvelt, K. and Church, G. 2013. Cas9 as a versatile tool for engineering biology. <I> Nature Methods. </I> 10, 10 (2013), 957–963.
</P><P>McDade JR, Waxmonsky NC (2016) Emerging CRISPR Technologies Available Through Resource Sharing. <I> JSM Genet Genomics</I>  3(1): 1008.
</P><P>Miyawaki, A. 2011. Development of Probes for Cellular Functions Using Fluorescent Proteins and Fluorescence Resonance Energy Transfer. <I> Annual Review of Biochemistry. </I>  80, 1 (2011), 357–373. 
</P><P>Remy, I., Campbell-Valois, F. and Michnick, S. 2007. Detection of protein-protein interactions using a simple survival protein-fragment complementation assay based on the enzyme dihydrofolate reductase. <I> Nature protocols. </I> 2, 9 (2007), 2120–5.
</P></DIV></DIV><DIV class=" footer"><DIV class="container"><DIV class="adress"><ADRESS>Université Paris Sud
91405 Orsay, France</ADRESS></DIV><SMALL class="envers copyright">Copyright © 2016 iGEM Paris Saclay. <STRIKE>All rights reserved</STRIKE> All inspirations encouraged.*This wiki was conceived during long nights of work and a great coffee experience in the beautiful Orsay, France.</SMALL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>