<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_USNA-Annapolis_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:USNA-Annapolis/Project</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:USNA-Annapolis">HOME </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:USNA-Annapolis/Project"> PROJECT </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:USNA-Annapolis/Parts"> PARTS </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:USNA-Annapolis/Safety"> SAFETY </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:USNA-Annapolis/Attributions">  ATTRIBUTIONS </A></LI><LI class="menu_item"> HUMAN PRACTICES 
			</LI><LI><A href="https://2016.igem.org/Team:USNA-Annapolis/Human_Practices"> Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:USNA-Annapolis/Human_Practices"> Silver </A></LI><LI><A href="https://2016.igem.org/Team:USNA-Annapolis/Human_Practices"> Gold </A></LI><LI><A href="https://2016.igem.org/Team:USNA-Annapolis/Human_Practices">  Integrated Practices </A></LI><LI><A href="https://2016.igem.org/Team:USNA-Annapolis/Human_Practices">Engagement </A></LI></UL><LI class="menu_item"> AWARDS 
			</LI><LI><A href="https://2016.igem.org/Team:USNA-Annapolis/Awards">  Entrepreneurship </A></LI><LI><A href="https://2016.igem.org/Team:USNA-Annapolis/Awards">  Hardware </A></LI><LI><A href="https://2016.igem.org/Team:USNA-Annapolis/Awards"> Software </A></LI><LI><A href="https://2016.igem.org/Team:USNA-Annapolis/Awards">  Measurement </A></LI><LI><A href="https://2016.igem.org/Team:USNA-Annapolis/Awards"> Model </A></LI></DIV><DIV class="content_wrapper"><DIV class="column full_size"><H2 style="text-align:center;"><B> Description </B></H2></DIV><DIV><H3 style="text-align:center;"><B><U> General </U></B></H3><P> The USNA iGEM project was separated into two main objectives. The first was to create mathematical models that can address the challenges of creating an efficient conotoxin counter measure in the respiratory microbiome, for example the rate of passage of conotoxins through the respiratory mucosa, toxicity, binding affinities of sensors, and respond modules. The mathematical model created over the summer represented a much simpler model of the rate of Potassium and Sodium ion movement through the<I> E. Coli </I> cellular membrane . This model will continue to be adjusted to meet the desired results that are needed to test.

</P><P>The second objective was to insert a two-part Arc system into a chloramphenicol vector, which is a classic sense and respond mechanism, in a manner that would allow for quick detection of presence of conotoxin.</P><H2 style="text-align:center;"><B> Lab Work </B></H2><H3 style="text-align:center;"><B><U>Background</U></B></H3><P>Conotoxins are neurotoxic peptides 10-30 amino acids long and are produced by marine cone snails. Conotoxins are a public health threat because they work by disrupting the flow of ions in cells, thus blocking the action potential in nerve, muscle, and endocrine cells, which can lead to paralysis, seizure, or even death. </P><H3 style="text-align:center;"><B><U>Overall Goal</U></B></H3><P>The primary objective of this project is to characterize the effects that conotoxins have on the microorganisms in the respiratory tract in order to create a synthetic biology toolkit for cell-based conotoxin counteraction and to understand the pathology of Conotoxins and how these toxins can be a problem for the U.S military. This is not an easy task, and the NRL and Naval Academy anticipate the entire project will take about five years. Our task for the project is to incorporate the genes discussed below into the appropriate vector, then to put the plasmid into competent cells. </P><H3 style="text-align:center;"><B><U>Goal for the summer</U></B></H3><P>The goal for this summer was to create a working two part sense and response system that can detect the presence of Conotoxins, which we could be entered into the IGEM competition this October.</P><H3 style="text-align:center;"><B><U>Conotoxins</U></B></H3><P>The membrane potential changes in the presence of Conotoxins, possibly including the membrane potential in bacteria. To counteract the changes in potential we sought to develop a sense and response system to sense the Conotoxins and activate a countermeasure.</P><P>To develop this countermeasure, we focused primarily on the arc system.</P><P>The arc system is made up of Arc A and arc B which is a two-component sense and response system, which is an aerobic respiration control in variety of different Bactria.</P><P>How this system works is that, Arc B sits in the cell membrane of a bacterium, such as E. coli, and it senses changes in the electrical potential of the membrane. When the Conotoxins change the influx of the ion channels in the cell, the membrane potential shifts and activates arc B which triggers it to phosphorylate. It then transfers the phosphate group to arc A, activating it. Arc A then binds to the bacteria’s DNA to help turn on or off the production of certain proteins.
The bacteria’s DNA mentioned above can actually be modify to make the gene activated by arcA code for a specific protein that eliminates Conotoxins. As part of the IGEM competition we isolated arcA and arcB from bacteria growing in an electrically active biofilm.</P><H4 style="text-align:center;"><B><U>Progress</U></B></H4><P>Currently we have both arcA and arcB isolated into a kanamycin vector, but they have not been combined to create the working two part system. Also, due to the time constraint we were unable to place the parts into the IGEM standard chloramphenicol vector for submission. In the coming years, we hope to combine the two parts into a working system and eventually be able to combat the effect of Conotoxins for the future warfighters in the United States military.</P><H4 style="text-align:center;"><B><U>Difficulties</U></B></H4><P>We have been making great progress, but we have run into some difficulties. There were complications in trying to amplify the gene responsible for Arc A. The gene was successfully isolated in gel filtration; however, the amplification of the gene through PCR produced poor results. In the end, we were still able to add the gene to <I> E. coli </I> and produce the Arc A protein with the small amount of gene. Several of our transformations were unsuccessful, due to difficulties in primer design and inhibited competent cell growth. There have also been issues with growing the cells in the correct vector. The chloramphenicol resistance was giving us a bit of trouble, and so, we decided to grow them on the kanamycin vector insead, for the sake of time. Another challenge comes from our team dynamic. Due to the Naval Academy’s summer schedule, each month a different group of Midshipman were at NRL, which meant that they needed to be retaught and briefed on the project each time. Needless to say this can make collaboration a bit difficult. But, through effective communication, we were able to accomplish many of our goals. </P><DIV class="column full_size"><H2 style="text-align:center;"><B> Modeling </B></H2></DIV><H3 style="text-align:center;"><B><U> Mathematical Model </U></B></H3><P>The <A href="https://static.igem.org/mediawiki/2016/c/c1/T--USNA-Annapolis--model_code.pdf"> mathematical model </A> made by the team began with the Hodgkin-Huxley model. This model uses ordinary differential equations to find the membrane potential of a neuron undergoing action potential as a function of time. Some of the parameters for the model are membrane capacitance, ionic current through membrane, and external current, but the most important feature of the model is that it is a function of time.</P><P>The Nernst equation gives the membrane potential as a function of the concentrations of ions on both sides of the membrane. One of our major assumptions for this model was that the concentration of ions outside of the cell was essentially fixed. This allowed us to isolate the term for intracellular concentration of ions. Based on literature available, the primary ions we determined of interest were sodium and potassium, both with a charge of +1.</P><P>We based the model of an <I>E. coli</I> cell with three transporters. The first two were sodium import and potassium export channels, and then in order to reverse the polarity we included a sodium-potassium pump which moved sodium and potassium out of and into the cell respectively. All transporters were assumed to be voltage gated. These major assumptions are based on the fact that the sodium and potassium are the two major ions present in <I> E. coli </I> which means they likely contribute to the membrane potential.</P><P>Based on the process of action potential, we modeled that when the cation concentration was increasing slowly, it was due to sodium-potassium pumps, but if it was increasing quickly, then it was due to an open sodium channel. When the cation concentration was decreasing, it was due to an open potassium channel </P><P>The modeled system was heuristically found to become stable for an arbitrarily long time when the voltage threshold for the switch from pump to sodium channel was at -38.0389 mV. Thus, we conjecture that a voltage gated sodium channels in <I>E. coli</I> would open when the membrane potential is greater than -38.0389 mV.</P><H3 style="text-align:center;"><B><U> Lab Notes </U></B></H3><A href="https://static.igem.org/mediawiki/2016/9/9a/T--USNA-Annapolis--Maggie_Lab.pdf"> Maggie Pana's Lab Notes </A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>