"
Page
Discussion
View source
History
teams
Log in
Â 
Team:DTU-Denmark-2/results/Proofofconcept/fungi
From 2011.igem.org
Home
About us
The Team
Supervisors
Gallery
The Project
Abstract
Introduction
The Plug 'n' Play
Assembly system
Customization
Other assembly systems
Achievements
Attributions
Results
Background
Characterization
Proof of concept
Mammalian Cells
Fungi
Data page
Submitted Biobricks
Collaboration
Safety
Safety
Safety rules
Notebook
Lab notes
Protocols
Our Partners
Sponsors
Advisors
Acknowledgements
Proof of concept - Fungi
Proof of concept
pJEJAM12
BBa_K678060
pJEJAM13
BBa_K678061
pJEJAM14
BBa_K678062
pJEJAM15
BBa_K678063
Wild type
Proof of concept
Five different plasmids were constructed with the Plug 'n' Play assembly standard in order to verify the systems function in filamentous fungi. To ensure a successful transformation the backbone plasmid pFun has two NotI restriction sites flanking the device to be inserted. Hereby, the device can be cut out of the plasmid and the linearized DNA fragments can be transformed into the fungus. The devices for proof of concept are not designed to be inserted at a specific site in the fungal genome. Therefore, the device can be integrated at any site and with a random number of copies by non-homologous end joining (NHEJ). This means that the possibility of the disruption of essential genes exists.
The fungal proof of concept plasmids, were constructed with the strong constitutive promoter PgpdA , the TtrpC terminator, and the pyrG marker cassette . Different genes encoding fluorescent proteins were included in the reporter system, where different compartments of fungi could be targeted. The reporter system was designed to target the nucleus, peroxisomes, and the mitochondria. All transformations were performed in the Aspergillus nidulans laboratory strain: argB2, pyrG89, veA1.
We here prove that the Plug 'n' Play assembly standard can easily be applied for the construction fungal plasmids. The constructed plasmids were transformed into A. nidulans and in each case the fluorescent protein was expressed. Only the strain transformed with pJEJAM16 targeting GFP to the mitochondria did not result in any transformants and due to time limitation the experiment was not repeated.
The obtained transformants with the different constructed devices integrated into the genome all had a different morphology and were growing slower than the wild type strain. Especially the strain transformed with pJEJAM14 looked sick. The fungi also had a pronounced number of undefined vacuoles that can be seen on the images obtained from differential interference contrast (DIC) microscopy. Such 'symptoms' are not uncommon when DNA is introduced into the genome by NHEJ. Below the images of the microscopy are presented.
pJEJAM12 BBa_K678060
pJEJAM12 is a plasmid intended for linearization and transformation into A. nidulans. The plasmid contains the gene encoding green fluorescent protein (GFP) under the control of the strong constitutive PgpdA promoter. The expressed GFP was evenly spread in the hyphae.
DIC image of Aspergillus nidulans with
pJEJAM12 .
Fluorescence image of Aspergillus nidulans with
pJEJAM12 .
Aspergillus nidulans with
pJEJAM12 - Shown from front
Aspergillus nidulans with
pJEJAM12 - Shown from Back
pJEJAM13 BBa_K678061
pJEJAM13 is a plasmid intended for linearization and transformation into A. nidulans. The plasmid contains the gene encoding green fluorescent protein (GFP) with the peroxisomal targeting sequence 1 (PTS1) fused to the C-terminus, under the control of the strong constitutive PgpdA promoter.
Green fluorescence can be observed in the hyphae and in clear spots. This indicates that GFP is targeted to the peroxisomes, but further investigations are required to confirm this. The 'background' fluorescence was however not expected and a replication of the experiment should be conducted in order to be able to draw any conclusions upon this.
DIC image of Aspergillus nidulans with
pJEJAM13 .
Fluorescence image of Aspergillus nidulans with
pJEJAM13 . Green fluorescence can be observed in the hyphae and in clear spots.
Aspergillus nidulans with
pJEJAM13 - Shown from front
Aspergillus nidulans with
pJEJAM13 - Shown from back
pJEJAM14 BBa_K678062
pJEJAM14 is a plasmid intended for linearization and transformation into A. nidulans. The plasmid contains the gene encoding monomeric red fluorescent protein (mRFP1) under the control of the strong constitutive PgpdA promoter. The expressed mRFP1 was evenly spread in the hyphae.
DIC image of Aspergillus nidulans with
pJEJAM14 .
Fluorescence image of Aspergillus nidulans with
pJEJAM14 .
Aspergillus nidulans with
pJEJAM14 - shown from front
Aspergillus nidulans with
pJEJAM14 - shown from back
pJEJAM15 BBa_K678063
pJEJAM15 is a plasmid intended for linearization and transformation into A. nidulans. The plasmid contains the gene encoding monomeric red fluorescent protein (mRFP1) with a nucleosomal targeting sequence (NLS) fused to the C-terminus, under the control of the strong constitutive PgpdA promoter.
Red fluorescence can be observed in clear spots. To confirm that the mRFP1 in fact is targeted to the nucleus further experiments have to be conducted.
DIC image ofAspergillus nidulans with
pJEJAM15 .
Fluorescence image of Aspergillus nidulans
pJEJAM15 .
Aspergillus nidulans with
pJEJAM15 - Shown from front
Aspergillus nidulans with
pJEJAM15 - Shown from back
Wild type
The wild type strain shows no background or auto-fluorescence.
DIC image of wild type Aspergillus nidulans .
RFP fluorescence image of wild type Aspergillus nidulans .
GFP fluorescence image of wild type Aspergillus nidulans .
Wild type Aspergillus nidulans
- Shown from front
Wild type Aspergillus nidulans
- Shown from back
Retrieved from " http://2011.igem.org/Team:DTU-Denmark-2/results/Proofofconcept/fungi "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
