<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_DTU-Denmark-2_Team_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:DTU-Denmark-2/Team/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:DTU-Denmark-2/Team/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:DTU-Denmark-2/Team/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:DTU-Denmark-2/Team/Protocols&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:DTU-Denmark-2/Team/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:DTU-Denmark-2/Team/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><UL id="nav" class="dropdown dropdown-horizontal"><LI><A href="https://2011.igem.org/Team:DTU-Denmark-2">Home</A></LI><LI class="dir">About us
               </LI><LI><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team">The Team</A></LI><LI><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/supervisors">Supervisors</A></LI><LI><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Gallery">Gallery</A></LI></UL><LI class="dir">The Project

                 </LI><LI><A href="https://2011.igem.org/Team:DTU-Denmark-2/Project/abstract">Abstract</A></LI><LI><A href="https://2011.igem.org/Team:DTU-Denmark-2/Project/introduction">Introduction</A></LI><LI class="dir" style="width: 114.7px;">The Plug 'n' Play
                            </LI><LI class="subsubmenu"><A href="https://2011.igem.org/Team:DTU-Denmark-2/Project/PlugnplayAssembly">Assembly system</A></LI><LI class="subsubmenu"><A href="https://2011.igem.org/Team:DTU-Denmark-2/Project/PlugnplayAssembly/customization">Customization</A></LI><LI><A href="https://2011.igem.org/Team:DTU-Denmark-2/Project/Other_assembly_systems">Other assembly systems</A></LI><LI><A href="https://2011.igem.org/Team:DTU-Denmark-2/Project/achievements">Achievements</A></LI><LI><A href="https://2011.igem.org/Team:DTU-Denmark-2/Project/attributions">Attributions</A></LI><LI class="dir">Results

                 </LI><LI><A href="https://2011.igem.org/Team:DTU-Denmark-2/results/background">Background</A></LI><LI><A href="https://2011.igem.org/Team:DTU-Denmark-2/results/Characterisation">Characterization</A></LI><LI class="dir" style="width: 114.7px;">Proof of concept
                            </LI><LI class="subsubmenu"><A href="https://2011.igem.org/Team:DTU-Denmark-2/results/Proofofconcept/mammalian">Mammalian Cells</A></LI><LI class="subsubmenu"><A href="https://2011.igem.org/Team:DTU-Denmark-2/results/Proofofconcept/fungi">Fungi</A></LI><LI><A href="https://2011.igem.org/Team:DTU-Denmark-2/results/data_page">Data page </A></LI><LI><A href="https://2011.igem.org/Team:DTU-Denmark-2/results/submitted_biobricks">Submitted Biobricks </A></LI><LI><A href="https://2011.igem.org/Team:DTU-Denmark-2/results/Copenhagen">Collaboration </A></LI><LI class="dir">Safety

                 </LI><LI><A href="https://2011.igem.org/Team:DTU-Denmark-2/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:DTU-Denmark-2/Safetyrules">Safety rules</A></LI><LI class="dir">Notebook
		</LI><LI><A href="https://2011.igem.org/Team:DTU-Denmark-2/Notebook/Lab%20notes">Lab notes</A></LI><LI><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols">Protocols</A></LI><LI class="dir">Our Partners 
		</LI><LI><A href="https://2011.igem.org/Team:DTU-Denmark-2/Our Partners/Sponsors">Sponsors</A></LI><LI><A href="https://2011.igem.org/Team:DTU-Denmark-2/Our Partners/Advisors">Advisors</A></LI><LI><A href="https://2011.igem.org/Team:DTU-Denmark-2/acknowledgments">Acknowledgements</A></LI></P><DIV class="text"><TABLE cellpadding="10px"><TBODY><TR><TD width="500px" height="100%" valign="top" valign="center" bgcolor="#FFFFFF"><FONT color=" #990000" face="arial" size="6"><SPAN class="red"><B>Protocols</B></SPAN></FONT><DIV id="menu"><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#Amplification of biobricks by PCR" class="h1"><B>1</B> Amplification of biobricks by PCR</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#List of primers for fungi and for mammalian cells" class="h2"><B>1.1</B> List of primers for fungi and for mammalian cells</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#PCR" class="h2"><B>1.2</B> PCR </A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#Purification of PCR Product" class="h2"><B>1.3</B> Purification of PCR Product</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#Gel electrophoresis" class="h2"><B>1.4</B> Gel electrophoresis</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#USER cloning" class="h1"><B>2</B> USER cloning</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#Tranformation in &lt;i&gt;E. coli&lt;/i&gt;" class="h2"><B>2.1</B> Tranformation in <I>E. coli</I></A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#Cultivation of transformed cells" class="h2"><B>2.2</B> Cultivation of transformed cells</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#Purification of plasmids for fungi" class="h2"><B>2.3</B> Purification of plasmids for fungi</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#Purification of plasmids for mammalian cells" class="h2"><B>2.4</B> Purification of plasmides for mammalian cells</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#Restriction enzyme analysis" class="h2"><B>2.5</B> Restriction enzyme analysis</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#Fungi" class="h1"><B>3</B> Fungi</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#Transformation in fungi" class="h2"><B>3.1</B> Transformation in fungi</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#Production of conidiospores" class="h2"><B>3.2</B> Production of conidiospores</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#Flourescence Microscopy" class="h2"><B>3.3</B> Flourescence Microscopy</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#Fluorescence detection" class="h2"><B>3.4</B> Fluorescence detection</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#Extraction of proteins" class="h2"><B>3.5</B> Extraction of proteins</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#Assays" class="h1"><B>4</B> Assays</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#ß-galactosidase assay" class="h2"><B>4.1</B> ß-galactosidase assay</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#Bradford assay" class="h2"><B>4.2</B>Bradford assay</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#Mammalian cells" class="h1"><B>5</B> Mammalian cells</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#Cell culture and reagents" class="h2"><B>5.1</B> Cell culture and reagents</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#Cultivation of cells" class="h2"><B>5.2</B> Cultivation of cells</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#Passing and maintenance of U2OS cells" class="h2"><B>5.3</B> Passing and maintenance of U2OS cells</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#Transferring cells to coverslips" class="h2"><B>5.4</B> Transferring cells to coverslips</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#Transfection of cells" class="h2"><B>5.5</B> Transfection of cells</A><A href="https://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols#Coverslips for microscopy" class="h2"><B>5.6</B> Coverslips for microscopy</A></DIV></TD><TD width="75%" height="100%" valign="top"><H2><B>Amplification of biobricks by PCR</B></H2><H3><B>List of primers for fungi and for mammalian cells</B></H3></TD></TR><TR><TH>Fungal primers</TH></TR><TR><TH>Primer name </TH><TH>Sequence </TH></TR><TR><TD>pgpd FW</TD><TD><FONT color=" #990000" id="red">ACGTCGCU</FONT>ATTCCCTTGTATCTCTACACACAGG</TD></TR><TR><TD>pgpd RV</TD><TD><FONT color=" #990000">ATCGCACU</FONT>GCGGTAGTGATGTCTGCTCAA</TD></TR><TR><TD>pAlc FW</TD><TD><FONT color=" #990000">ACGTCGCU</FONT>CTCCCCGATGACATACAGGAGG</TD></TR><TR><TD>pAlc RV</TD><TD><FONT color=" #990000">ATCGCACU</FONT>TTTGAGGCGAGGTGATAGGATTG</TD></TR><TR><TD>DMKP-P6 FW</TD><TD><FONT color=" #990000">ACGTCGCU</FONT>ATCTCACTCCACTAGAATTCCTGTC</TD></TR><TR><TD>DMKP-P6 RV</TD><TD><FONT color=" #990000">ATCGCACU</FONT>AAATGAGTCGAGAATGGCGG</TD></TR><TR><TD>GFP-module FW</TD><TD><FONT color=" #990000">AGTGCGAU</FONT>ATGGTGAGCAAGGGCGAG </TD></TR><TR><TD>GFP-module RV</TD><TD><FONT color=" #990000">ATCGGAAU</FONT>TTACTTGTACAGCTCGTCCATGC</TD></TR><TR><TD>GFP-GOI FW</TD><TD><FONT color=" #990000">AGTGCGAU</FONT>ATGGTGAGCAAGGGCGAG</TD></TR><TR><TD>GFP-GOI RV</TD><TD><FONT color=" #990000">ATCGCTCU</FONT>TTACTTGTACAGCTCGTCCATGC</TD></TR><TR><TD>GFP-TS FW</TD><TD><FONT color=" #990000">AGCGCTGGU</FONT>ATGGTGAGCAAGGGCGAG</TD></TR><TR><TD>GFP-TS RV</TD><TD><FONT color=" #990000">ATCGGAAU</FONT>TTACTTGTACAGCTCGTCCATGC</TD></TR><TR><TD>GFP-PTS1_module RV</TD><TD><FONT color=" #990000">ATCGGAAU</FONT><FONT color=" blue">TTACAGCTTGGA</FONT> CCTTGTACAGCTCGTCCATGCC</TD></TR><TR><TD>RFP-module FW</TD><TD><FONT color=" #990000">AGTGCGAU</FONT>ATGGCCTCCTCCGAGGAC </TD></TR><TR><TD>RFP-module RV</TD><TD><FONT color=" #990000">ATCGGAAU</FONT>TTAGGCGCCGGTGGAGTG</TD></TR><TR><TD>RFP-GOI FW</TD><TD><FONT color=" #990000">AGTGCGAU</FONT>ATGGCCTCCTCCGAGGAC</TD></TR><TR><TD>RFP-GOI RV</TD><TD><FONT color=" #990000">ATCGCTCU</FONT>TTAGGCGCCGGTGGAGTG</TD></TR><TR><TD>RFP-TS FW</TD><TD><FONT color=" #990000">AGCGCTGGU</FONT>ATGGCCTCCTCCGAGGAC</TD></TR><TR><TD>RFP-TS RV</TD><TD><FONT color=" #990000">ATCGGAAU</FONT>TTACTTGTACAGCTCGTCCATGC</TD></TR><TR><TD>RFP-NLS_module RV</TD><TD><FONT color=" #990000">ATCGGAAU</FONT><FONT color=" blue">TTAGACCTTGCGCTTTTTCTTGGG</FONT> GGCGCCGGTGGAGTGG</TD></TR><TR><TD>LacZ FW</TD><TD><FONT color=" #990000">AGTGCGAU</FONT>ATGACCATGATTACGGATTCACT</TD></TR><TR><TD>LacZ RV</TD><TD><FONT color=" #990000">ATCGGAAU</FONT>TTATTTTTGACACCAGACCAACT</TD></TR><TR><TD>trpC FW</TD><TD><FONT color=" #990000">ATTCCGAU</FONT>GATCCACTTAACGTTACTGAAATCA</TD></TR><TR><TD>trpC RV</TD><TD><FONT color=" #990000">ACGCAAGU</FONT>GGGCGCTTACACAGTACACGAG</TD></TR><TR><TD>PyrG FW</TD><TD><FONT color=" #990000">ACTTGCGU</FONT>CGTGGAGTTACCAGTGATTGACC</TD></TR><TR><TD>PyrG RV</TD><TD><FONT color=" #990000">AGCTTAAU</FONT>CTTGCTAGATGACTGGTAGGAATCT</TD></TR><TR><TD>bleR FW</TD><TD><FONT color=" #990000">ACTTGCGU</FONT>ATGGCCAAGTTGACCAGTG</TD></TR><TR><TD>bleR RV</TD><TD><FONT color=" #990000">AGCTTAAU</FONT>TCAGTCCTGCTCCTCGGCC</TD></TR><TR><TD>hyrR FW</TD><TD><FONT color=" #990000"> ACTTGCGU</FONT> GCTAGTGGAGGTCAACACATCAAT </TD></TR><TR><TD>hyrR RV</TD><TD><FONT color=" #990000">AGCTTAAU</FONT>CGGTCGGCATCTACTCTATTCC</TD></TR><TR><TD>argB FW</TD><TD><FONT color=" #990000">ACTTGCGU</FONT>CGCGGTTTTTTGGGGTAGT</TD></TR><TR><TD>argB RV</TD><TD><FONT color=" #990000">AGCTTAAU</FONT>GCCACCTACAGCCATTGCGAA</TD></TR><TR><TD>PyrG-DR FW</TD><TD><FONT color=" #990000">ACTTGCGU</FONT>TGGATAACCGTATTACCGCCT</TD></TR><TR><TD>PyrG-DR RV</TD><TD><FONT color=" #990000">AGCTTAAU</FONT>TGCCAAGCTTAACGCGTACC</TD></TR><TR><TD>ptrA FW</TD><TD><FONT color=" #990000">ACTTGCGU</FONT>GGAGATCGTCCGCCGATG</TD></TR><TR><TD>ptrA RV</TD><TD><FONT color=" #990000">AGCTTAAU</FONT>CTAGAATGCCCCACCGTTACATAC</TD></TR><TR><TD>Amp-cas FW</TD><TD><FONT color=" #990000">AGCTTAAU</FONT>TTACCAATGCTTAATCAGTGAGGC</TD></TR><TR><TD>Amp-cas RV</TD><TD><FONT color=" #990000">ACTTGCGU</FONT>GACGTCAGGTGGCACTTTTCG</TD></TR><TR><TD>yA FW</TD><TD><FONT color=" #990000">AGTGCGAU</FONT>ATGTACCTCTCCACGGTCCTCT</TD></TR><TR><TD>yA RV</TD><TD><FONT color=" #990000">ATCGCTCU</FONT>CTAAGAATCCCAAACATCAACCC</TD></TR><TR><TD>MTS FW</TD><TD><FONT color=" #990000">AGTGCGAU</FONT>ATGTTTACAGCGGCAGCTCG</TD></TR><TR><TD>MTS RV</TD><TD><FONT color=" #990000">ACCAGCGCU</FONT>CTTGCGCCGCGGAGC</TD></TR><TR><TD> T1-motni FW</TD><TD><FONT color=" #990000">ATTCCGAU</FONT>GGCTCCGAGGCTACTGGAGT</TD></TR><TR><TD> T1-motni FW</TD><TD><FONT color=" #990000">ACGCAAGU</FONT>GCGACGTCTGATGCCAATAT</TD></TR><TR><TD> T2-motni FW</TD><TD><FONT color=" #990000">ATTCCGAU</FONT>GCGGTTTTCAAGGATAACAGAT</TD></TR><TR><TD>T2-motni RV</TD><TD><FONT color=" #990000">ACGCAAGU</FONT>CATGACCCCGGATAACTTTAAAA</TD></TR><TR><TD>T3 motni FW</TD><TD><FONT color=" #990000">ATTCCGAU</FONT>AAGTCTTCCGTTACCCTTGCA</TD></TR><TR><TD>T3-motni RV</TD><TD><FONT color=" #990000">ACGCAAGU</FONT>TGTAATTCCTACCTACCTACCTCTT</TD></TR></TBODY></TABLE><TABLE border="3" bordercolor="#990000"><TBODY><TR><TH>Mammalian primers</TH></TR><TR><TH>Primer name </TH><TH>Sequence </TH></TR><TR><TD> PGK FW</TD><TD><FONT color=" #990000">ACGTCGCU</FONT>CCGGTAGGCGCCAACCG</TD></TR><TR><TD>PGK RV</TD><TD><FONT color=" #990000">ATCGCACU</FONT>GGCTGCAGGTCGAAAGGCC</TD></TR><TR><TD>SV40 (+ori) FW</TD><TD><FONT color=" #990000">ACGTCGCU</FONT>CTGTGGAATGTGTGTCAGTTAGG</TD></TR><TR><TD>SV40 (+ori) RV</TD><TD><FONT color=" #990000">ATCGCACU</FONT>AGCTTTTTGCAAAAGCCTAGG</TD></TR><TR><TD>CMV FW</TD><TD><FONT color=" #990000">ACGTCGCU</FONT>CGATGTACGGGCCAGATATAC</TD></TR><TR><TD>CMV-RV</TD><TD><FONT color=" #990000">ATCGCACU</FONT>ATTTCGATAAGCCAGTAAGCAGT</TD></TR><TR><TD>eGFP+K_GOI FW</TD><TD><FONT color=" #990000">AGTGCGAU</FONT>CGCCACCATGGTGAGCAA </TD></TR><TR><TD>eGFP+K_GOI RV</TD><TD><FONT color=" #990000">ATCGCTCU</FONT>TTACTTGTACAGCTCGTCCATGC</TD></TR><TR><TD>Mammalian backbone pU0020 FW</TD><TD><FONT color=" #990000">ATTAAGCU</FONT>AGTGAGTCGAATAAGGGCGACA</TD></TR><TR><TD>Mammalian backbone pU0020 RV</TD><TD><FONT color=" #990000">AGCGACGU</FONT>GAGTCGAATAAGGGCGACACC</TD></TR><TR><TD>eGFP_module FW</TD><TD><FONT color=" #990000">AGTGCGAU</FONT>CGCCACCATGGTGAGCAA</TD></TR><TR><TD>eGFP_module RV</TD><TD><FONT color=" #990000">ATCGGAAU</FONT>TTACTTGTACAGCTCGTCCATGCC</TD></TR><TR><TD>eGFP+K GOI FW</TD><TD><FONT color=" #990000">AGTGCGAU</FONT>CGCCACCATGGTGAGCAA</TD></TR><TR><TD>eGFP+K GOI-stop RV</TD><TD><FONT color=" #990000">ACCAGCGCU</FONT>CTTGTACAGCTCGTCCATGCC</TD></TR><TR><TD>eGFP_TS FW</TD><TD><FONT color=" #990000">AGAGCGAU</FONT>CGCCACCATGGTGAGCAA</TD></TR><TR><TD>eGFP_TS RV</TD><TD><FONT color=" #990000">ATCGGAAU</FONT>TTACTTGTACAGCTCGTCCATGCC</TD></TR><TR><TD>YFP_module FW</TD><TD><FONT color=" #990000">AGTGCGAU</FONT>CGCCACCATGGTGAGCAA</TD></TR><TR><TD>YFP_module RV</TD><TD><FONT color=" #990000">ATCGGAAU</FONT>TTATCTAGATCCGGTGGATCCC</TD></TR><TR><TD>YFP+K GOI FW</TD><TD><FONT color=" #990000">AGTGCGAU</FONT>CGCCACCATGGTGAGCAA</TD></TR><TR><TD>YFP+K GOI RV</TD><TD><FONT color=" #990000"> ACCAGCGCU </FONT> TCTAGATCCGGTGGATCCCG </TD></TR><TR><TD>YFP_TS FW</TD><TD><FONT color=" #990000">AGCGCTGGU</FONT>CGCCACCATGGTGAGCAA</TD></TR><TR><TD>YFP_TS RV</TD><TD><FONT color=" #990000">ATCGGAAU</FONT>TTATCTAGATCCGGTGGATCCC</TD></TR><TR><TD>CFP_module FW</TD><TD><FONT color=" #990000">AGTGCGAU</FONT>CGCCACCATGGTGAGCAA</TD></TR><TR><TD>CFP_module RV</TD><TD><FONT color=" #990000">ATCGGAAU</FONT>TTATCTAGATCCGGTGGATCCC</TD></TR><TR><TD>CFP+K GOI FW</TD><TD><FONT color=" #990000">AGTGCGAU</FONT>CGCCACCATGGTGAGCAA</TD></TR><TR><TD>CFP+K GOI-stop RV</TD><TD><FONT color=" #990000">ACCAGCGCU</FONT>TCTAGATCCGGTGGATCCCG</TD></TR><TR><TD>CFP_TS FW</TD><TD><FONT color=" #990000">AGCGCTGGU</FONT>CGCCACCATGGTGAGCAA</TD></TR><TR><TD>CFP_TS RV</TD><TD><FONT color=" #990000">ATCGGAAU</FONT>TTATCTAGATCCGGTGGATCCC</TD></TR><TR><TD>mCherry module FW</TD><TD><FONT color=" #990000">AGTGCGAU</FONT>CGCCACCATGGTGAGCAA</TD></TR><TR><TD>mCherry module RW</TD><TD><FONT color=" #990000">ATCGGAAU</FONT>CTACTTGTACAGCTCGTCCATGC</TD></TR><TR><TD>mCherry+K GOI FW</TD><TD><FONT color=" #990000">AGTGCGAU</FONT>CGCCACCATGGTGAGCAA</TD></TR><TR><TD>mCherry GOI-stop RV</TD><TD><FONT color=" #990000">ACCAGCGCU</FONT>CTTGTACAGCTCGTCCATGCC</TD></TR><TR><TD>mCherry_TS FW</TD><TD><FONT color=" #990000">AGCGCTGGU</FONT>CGCCACCATGGTGAGCAA</TD></TR><TR><TD>mCherry_TS RV</TD><TD><FONT color=" #990000">ATCGGAAU</FONT>CTACTTGTACAGCTCGTCCATGC</TD></TR><TR><TD>BGH poly A FW</TD><TD><FONT color=" #990000">ATTCCGAU</FONT>CTGTGCCTTCTAGTTGCCAGC</TD></TR><TR><TD>BGA poly A RV</TD><TD><FONT color=" #990000">ACGCAAGU</FONT>CCATAGAGCCCACCGCATC</TD></TR><TR><TD>SV40 pA FW</TD><TD><FONT color=" #990000">ATTCCGAU</FONT>AACTTGTTTATTGCAGCTTATAATGGTTAC</TD></TR><TR><TD>SV40 pA RV</TD><TD><FONT color=" #990000">ACGCAAGU</FONT>CAGACATGATAAGATACATTGATGAGTTTG</TD></TR><TR><TD>Hygromycin FW</TD><TD><FONT color=" #990000">ACTTGCGU</FONT>CCAGCAGGCAGAAGTATGCA</TD></TR><TR><TD>Hygromycin RV</TD><TD><FONT color=" #990000">AGCTTAAU</FONT>CAGGCTTTACACTTTATGCTTCC</TD></TR><TR><TD>Neomycin FW</TD><TD><FONT color=" #990000">ACTTGCGU</FONT>CTGTGGAATGTGTGTCAGTTAGG</TD></TR><TR><TD>Neomycin RV</TD><TD><FONT color=" #990000">AGCTTAAU</FONT>CAGACATGATAAGATACATTGATGAGTTTG</TD></TR></TBODY></TABLE><TABLE border="3" bordercolor="#990000"><TBODY><TR><TH>Copenhagen primers</TH></TR><TR><TH>Primer name </TH><TH>Sequence </TH></TR><TR><TD>pIPTG FW</TD><TD><FONT color=" #990000">ACGTCGCU</FONT>CAATACGCAAACCGCCTCTC</TD></TR><TR><TD>pIPTG RV</TD><TD><FONT color=" #990000">ATCGCACU</FONT>TGTGTGAAATTGTTATCCGCTCA</TD></TR><TR><TD>CYP79A1 FW</TD><TD><FONT color=" #990000">AGTGCGAU</FONT><FONT color="blue">aaagaggagaaa</FONT>ATGGCTCTGTTATTAGCAGTTTTT</TD></TR><TR><TD>CYP79A1 RV</TD><TD><FONT color=" #990000">ATCGGAAU</FONT>TTAGATGGAGATGGACGGGTA</TD></TR><TR><TD>CYP79B1 FW</TD><TD><FONT color=" #990000">AGTGCGAU</FONT><FONT color="blue">aaagaggagaaa</FONT>ATGTATTTACTTACAACGCTTCAAG</TD></TR><TR><TD>CYP79B1 RV</TD><TD><FONT color=" #990000">ATCGGAAU</FONT>TACTTCACTGTAGGGTAAAGATGT</TD></TR><TR><TD>2C9 FW</TD><TD><FONT color=" #990000">AGTGCGAU</FONT><FONT color="blue">aaagaggagaaa</FONT>ATGGCTCGACAATCTTCTGGA</TD></TR><TR><TD>2C9 RV</TD><TD><FONT color=" #990000">ATCGGAAU</FONT>TTAATGGTGATGGTGATGGACAG</TD></TR><TR><TD>Terminator FW</TD><TD><FONT color=" #990000">ATTCCGAU</FONT>CCAGGCATCAAATAAAACGAAA</TD></TR><TR><TD>Terminator RV</TD><TD><FONT color=" #990000">ACGCAAGU</FONT>TATAAACGCAGAAAGGCCCAC</TD></TR><TR><TD>Amp-cas FW</TD><TD><FONT color=" #990000">AGCTTAAU</FONT>TTACCAATGCTTAATCAGTGAGGC</TD></TR><TR><TD>Amp-cas RV</TD><TD><FONT color=" #990000">ACTTGCGU</FONT>GACGTCAGGTGGCACTTTTCG</TD></TR></TBODY></TABLE><H3>PCR</H3><TABLE border="3" bordercolor="#990000"><TBODY><TR><TH>PCR mix</TH><TH>1 x PCR mix á 50 µl </TH></TR><TR><TD> 5 x HF PCR buffer</TD><TD>10 µl</TD></TR><TR><TD> dNTP’s 2mM </TD><TD>5 µl</TD></TR><TR><TD> Primer forward 10 µM</TD><TD>5 µl</TD></TR><TR><TD> Primer reverse 10 µM</TD><TD>5 µl</TD></TR><TR><TD> Phusion DNA polymerase 5u/µl</TD><TD>0.5 µl</TD></TR><TR><TD> DNA template </TD><TD>1 µl</TD></TR><TR><TD> MilliQ water</TD><TD>23.5 µl</TD></TR></TBODY></TABLE><B><BIG>Procedure</BIG></B><LI>The following PCR mix components are mixed together: HF Buffer, dNTP, Phusion DNA polymerase, and MilliQ water. The DNA template and the corresponding primers are added subsequently (see Materials). 
Remember to multiply by the amount of PCR reactions. </LI><LI>50 µl PCR mix is added to each tube, and afterwards the DNA template and the corresponding primers are added to each PCR tubes. The solution is mixed well.</LI><LI>Often it can be advantageous to add 1 µl MgCl2 (50mM) to the reaction and reduce the volume of MilliQ water to 22.5 µl.
</LI><LI>The following PCR program is applied, and the PCR samples are run. </LI><H3>PCR Programs</H3><TABLE border="3" bordercolor="#990000"><TBODY><TR><TH>Temperature [°C]</TH><TH>Time [min] </TH><TH>Cycles</TH></TR><TR><TD>98</TD><TD>15:00</TD></TR><TR><TD> 98</TD><TD>0:20</TD><TD>35</TD></TR><TR><TD>62</TD><TD>0:30</TD><TD>-</TD></TR><TR><TD> 72</TD><TD>1:00</TD><TD>-</TD></TR><TR><TD> 72</TD><TD>5:00</TD></TR><TR><TD> 12</TD><TD>Store</TD></TR></TBODY></TABLE>
For each PCR product a gel electrophoresis is made to make sure that the PCR product has the correct size. 
7-10µl loading buffer is added to each 50µl PCR product and the samples are run in 2% agarose. The Gel electrophoresis were set on 75V for 30-60 minutes (according to the length of the product).
<H3>Purification of PCR Product</H3><BIG><B>Materials</B></BIG>
GFX PCR DNA and Gel band purification Kit from GE healthcare is used to purify the PCR product after gel electrophoresis. 
<BIG><B>Preparations</B></BIG><LI>Ethanol is added to the Wash Buffer 1 and the label is marked. Store in airtight container.</LI><BIG><B>Procedures - GE healthcare protocol</B></BIG><B>Sample Capture </B><LI> A DNase-free 1,5 ml eppendorf tube is weighed, and the weight is noted.</LI><LI> By using long wavelength (365 nm) ultraviolet light and minimal exposure time, the sample in the gel is visualized. A clean scalpel is used to cut out an the sample from the agarose gel. The agarose gel band is placed into a DNase-free 1,5 ml eppendorf tube and weighed. The weight of the agarose band is calculated.</LI><LI> 10 μl Capture buffer type 3 is added for each 10 mg of gel slice. Never use less than 300 μl Capture buffer type 3.</LI><LI> The Capture buffer type 3-sample mix is mixed by inversion and incubated at 60°C for 15–30 minutes until the agarose is completely dissolved. The sample is mixed by inversion every 3 minutes</LI><LI> For each purification a GFX MicroSpin column is placed into a Collection tube. </LI><B>Sample Binding</B><LI> The Capture buffer type 3-sample mix is centrifuged briefly to collect the liquid at the bottom of the DNase-free 1,5 ml microcentrifuge tube.</LI><LI> Up to 800 μl Capture buffer type 3- sample mix is transferred to the assembled GFX MicroSpin column.</LI><LI> The sample is incubated at room temperature for 1 minute. </LI><LI> The sample is centrifuged for 30 s at 16 000 × g. The flow-through is discarded. </LI><LI> Repeat the sample binding until all sample is loaded. </LI><B>Wash and Dry</B><LI> 500 μl Wash buffer type 1 is added to the GFX MicroSpin column. </LI><LI> The sample is centrifuged at 16 000 × g for 30 seconds.</LI><LI> Discard the Collection tube and transfer the GFX MicroSpin column to a new DNase-free 1.5 ml eppendorf tube (supplied by user). </LI><B>Elution</B><LI> 10–50 μl MilliQ water is added to the center of the membrane in the GFX MicroSpin column placed in a Collection tube. The sample is incubated at room temperature for 1 minute. </LI><LI> The GFX MicroSpin column placed in the Collection tube is centrifuged for 1 minute at 16 000 × g to recover the purified DNA. </LI><LI> The purified DNA is stored at -20°C. </LI><H3><B>Gel electrophoresis</B></H3><BIG><B>Materials</B></BIG>

Agarose
TAE Buffer (1L): <SUP>4.84 g Tris Base, 1.14 ml Glacial Acetic Acid, 2 ml 0.5M EDTA (pH 8.0), add water until volume is 1L</SUP>
10X Loading Buffer
DNA ladder standard
Electrophoresis chamber
Gel casting tray and combs<BIG><B> Procedures </B></BIG><LI>1.25 g Agarose powder is measured and added to a 500 ml flask</LI><LI>125 ml TAE Buffer is added to the flask.  </LI><LI>The agarose is melted in a microwave or hot water bath until the solution becomes clear.</LI><LI>The solution is cooled down to 50-55°C. A trick is to swirl the flask lightly to cool evenly.  </LI><LI> 20 µl of cybrsafe is added in 4-5 drops on the casting tray. The agarose gel is added and mixed evenly. The gel is left to cure, which takes 30-45 min. </LI><LI> The gel is placed in a electrophoresis chamber. Make sure the TAE buffer in the chamber is 2-3mm above the agarose gel.</LI><LI>The gel is loaded with the samples, a negative sample, and the DNA ladder.</LI><LI> The gel is run at 70V. The time depends on the bp length, but 30-60 min are usually good for 1000-3000bp.</LI><LI> The gel is visualized in a <B>-GEL EXPOSER!-.</B><H2><B>USER cloning</B></H2><H3>Materials</H3><BIG><B>Procedures </B></BIG></LI><LI> The USER mix components are mixed (Table in materials). The PCR product must be purified before used in USER cloning</LI><LI> 2 µl of the USER mix is transferred to PCR tubes.</LI><LI> The PCR products is added in equal amounts of each to a total volume of 8 µl and incubated for 40 minutes at 37°C and for 30 min at 25°C.</LI><TABLE border="3" bordercolor="#990000"><TBODY><TR><TH>USER mix</TH><TH>1 x USER mix á 10µl </TH></TR><TR><TD> USER enzyme</TD><TD>1 µl</TD></TR><TR><TD> NEB (10 x diluted)</TD><TD>0,5 µl </TD></TR><TR><TD>BSA </TD><TD>0,5 µl</TD></TR><TR><TD> PCR product(s)</TD><TD> 8 µl</TD></TR></TBODY></TABLE><H3>Tranformation in <I>E. coli</I></H3>
The preparations for the transformation can preferably be done, while the USER cloning is incubating. 
<BIG><B>Materials</B></BIG>
LB-plates with antibiotic resistance.<I>E. coli DHα5</I> cells&quot;
Grigalsky spartula
Ethanol 
USER reaction
Bunsen burner <BIG><B>Procedures</B></BIG><LI> LB- plates are taken out of the refrigerator and marked. Remember to use LB-plates with the right antibiotics. </LI><LI> 50 µl competent <I>E. coli DHα5</I> cells per USER reaction is taken from the -80C freezer and place on ice. Additionally, 1,5 ml tubes are placed on ice.</LI><LI> The USER reaction mix is added to the 50 µl competent <I>E. coli</I> cells. Mix well by pipetting. </LI><LI> The cells are kept on ice for 30 min. </LI><LI> The hot plate is set on 60C, and each transformation is heat shocked for 90 sec. The cells are put directly on ice for 2 min afterwards</LI><LI><SMALL><DL><DT><B> Plating on LB plates. </B></DT></DL></SMALL></LI><DL><DT><SMALL> With ampicillin resistence gene.</SMALL></DT><DD><LI> A Drigalski spartula is sterilized in 90% ethanol and flamed between each transformation.</LI></DD><DD><LI>The transformation mix is transferred to an LB-amp plate containing ampicillin and dispersed with the cooled drigalski.</LI></DD><DD><LI>The transformated cells are incubated over night at 37°C. </LI></DD><DD><LI>Next day; the plates are checked for visual colonies. These are used for cultivating. </LI></DD></DL><DL><DT><SMALL> Without ampicillin resistence gene</SMALL></DT><DD><LI>500 µl LB is added to the transformation mix and incubated for 30-60 min at RT. </LI></DD><DD><LI>The transformation mix is centrifuged for 1 min at 8000g. </LI></DD><DD><LI>The supernantant is removed except for approximately 50 µl.</LI></DD><DD><LI>The pellet is resuspended in the remaining LB. </LI></DD><DD><LI>The 50 µl of tranformatin mix is plated on the LB plate.</LI></DD><DD><LI> The transformated cells are incubated over night at 37°C. </LI><LI> Next day; the plates are checked for visual colonies. These are used for cultivating. </LI></DD></DL><H3>Cultivation of transformed cells</H3><BIG><B>Material</B></BIG>
Liquid LB with resistance marker
tooth pick<BIG><B>Procedure</B></BIG><LI> A couple of colonies are chosen from the LB-plate. </LI><LI> Each colony are transferred to 5 ml LB + resistance marker with a pipet tip.</LI><LI> Incubate over night at 37°C in the shaking incubator. </LI><H3>Purification of plasmids for fungi</H3><BIG><B>Materials</B></BIG>
The GenElute Plasmid Miniprep Kit from Sigma-Aldrich is used to isolate our plasmids to use in fungi. QIAGENs EndoFree Plasmid Maxi kit is used to purify plasmids for use in mammalian cells.
<BIG><B>Preparation</B></BIG><LI> The agents are thoroughly mixed. If any reagent forms a precipitate, warm at 55–65 °C until the precipitate dissolves. Cool to room temperature before use. </LI><LI> The solution is resuspended. Spin the tube of the RNase A Solution briefly to collect the solution in the bottom of the tube. Add 13 μl (for 10 prep package), 78 μl (for 70 prep package) or 500 μl (for 350 prep package) of the RNase A Solution to the Resuspension Solution prior to initial use. Store at 4 °C.</LI><LI> Wash solution; Dilute with 95-100% ethanol prior to initial use. <BIG><B>Procedure - according to the manufactures protocol.</B></BIG></LI><LI> Harvest cells by centrifugation of 5 ml of an overnight recombinant <I>E. coli</I> culture. The optimal volume of culture to use depends upon the plasmid and culture density. For best yields, follow the instructions in the note below. Transfer the appropriate volume of the recombinant <I>E. coli</I> culture to a microcentrifuge tube and pellet cells at ≥12,000 3 g for 1 minute. Discard the supernatant.</LI><LI>Completely resuspend the bacterial pellet with 200 μl of the Resuspension Solution. Vortex or pipette up and down to thoroughly resuspend the cells until homogeneous. Incomplete resuspension will result in poor recovery.</LI><LI>Lyse the resuspended cells by adding 200 μl of the Lysis Solution. Immediately mix the contents by gentle inversion (6–8 times) until the mixture becomes clear and viscous. The lyse reaction must not exceed 5 min.</LI><LI>Precipitate the cell debris by adding 350 μl of the Neutralization/Binding Solution. Gently invert the tube 4–6 times. Pellet the cell debris by centrifuging at ≥12,000*3 g or maximum speed for 10 minutes. Cell debris, proteins, lipids, SDS, and chromosomal DNA should fall out of solution as a cloudy, viscous precipitate.</LI><LI>Prepare columns
Insert a GenElute Miniprep Binding Column into a provided microcentrifuge tube, if not already assembled. Add 500 μl of the Column Preparation Solution to each miniprep column and centrifuge at ≥12,000*3 g for 30 seconds to 1 minute. Discard the flow-through liquid.</LI><LI>Transfer the cleared lysate from step 3 to the column prepared in step 4 and centrifuge at ≥12,000*3 g for 30 seconds to 1 minute. Discard the flow-through liquid.</LI><LI>Add 750 μl of the diluted Wash Solution to the column. Centrifuge at ≥12,000*3 g for 30 seconds to 1 minute. The column wash step removes residual salt and other contaminants introduced during the column load. Discard the flow-through liquid and centrifuge again at maximum speed for 1 to 2 minutes without any additional Wash Solution to remove excess ethanol.</LI><LI>Transfer the column to a fresh collection tube. Add 100 μl of MilliQ water to the column. Centrifuge at ≥12,000*3 g for 1 minute. The DNA is now present in the eluate and is ready for immediate use or storage at –20 °C.</LI><H3>Purification of plasmids for mammalian cells</H3><BIG><B>Materials</B></BIG>
EndoFree Plasmid Maxi kit from QIAGEN was used to purify plasmids for use in mammalian cells. <BIG><B>Procedure (according to manufactures protocol)</B></BIG><LI> Pick a single colony from a freshly streaked selective plate and inoculate a starter culture of 2–5 ml LB medium containing the appropriate selective antibiotic. Incubate for ~8 h at 37°C with vigorous shaking (~300 rpm). </LI><LI> Dilute the starter culture 1/500 to 1/1000 into selective LB medium. For high-copy plasmids inoculate 100 ml medium, and for low-copy plasmids, inoculate 250 ml medium. Grow at 37°C for 12–16 h with vigorous shaking (~300 rpm). </LI><LI> Harvest the bacterial cells by centrifugation at 6000 x g for 15 min at 4°C. Resuspend the bacterial pellet in 10 ml Buffer P1 </LI><LI> Add 10 ml Buffer P2, mix gently but thoroughly by inverting 4–6 times, and incubate at room temperature for 5 min. The lysis reaction time must not exceed 5 min. </LI><LI>Add 10 ml chilled Buffer P3 to the lysate, and mix immediately but gently by inverting 4–6 times. Proceed directly to next step. Do not incubate the lysate on ice. </LI><LI>Pour the lysate into the barrel of the QIAfilter Cartridge. Incubate at room temperature for 10 min. Do not insert the plunger! </LI><LI>Remove the cap from the QIAfilter Cartridge outlet nozzle. Gently insert the plunger into the QIAfilter Maxi Cartridge and filter the cell lysate into a 50 ml tube. </LI><LI>Add 2.5 ml Buffer ER to the filtered lysate, mix by inverting the tube approximately 10 times, and incubate on ice for 30 min. </LI><LI>Equilibrate a QIAGEN-tip 500 by applying 10 ml Buffer QBT, and allow the column to empty by gravity flow. </LI><LI>Apply the filtered lysate from step 9 to the QIAGEN-tip and allow it to enter the resin by gravity flow. </LI><LI>Wash the QIAGEN-tip with 2 x 30 ml Buffer QC. </LI><LI>Elute DNA with 15 ml Buffer QN. </LI><LI>Precipitate DNA by adding 10.5 ml (0.7 volumes) room-temperature isopropanol to the eluted DNA. Mix and centrifuge immediately at ≥15,000 x g for 30 min at 4°C. Carefully decant the supernatant. </LI><LI>Wash DNA pellet with 5 ml of endotoxin-free room-temperature 70% ethanol (add 40 ml of 96–100% ethanol to the endotoxin-free water supplied with the kit) and centrifuge at ≥15,000 x g for 10 min. Carefully decant the supernatant without disturbing the pellet. </LI><LI>Air-dry the pellet for 5–10 min, and redissolve the DNA in a suitable volume of endotoxin-free Buffer TE.</LI> 

To determine the yield, DNA concentration should be determined by both UV spectrophotometry and quantitative analysis on an agarose gel.
<H3>Restriction enzyme analysis</H3><H4>Materials </H4><B><BIG>Procedure</BIG></B><LI> The components for restriction analysis of the fungi plasmid are mixed. The recipe for 1 restriction analysis below. </LI><TABLE border="3" bordercolor="#990000"><TBODY><TR><TH>Ingredients </TH><TH>Volume: 10µl </TH></TR><TR><TD> Device</TD><TD>5 µl</TD></TR><TR><TD>Buffer 3</TD><TD>3 µl </TD></TR><TR><TD>Restriction enzyme </TD><TD>1,5 µl</TD></TR><TR><TD> H<SUB>2</SUB>O</TD><TD> 5,5 µl</TD></TR></TBODY></TABLE><LI>For fungal plasmides the restriction enzyme used: BgI II. </LI><LI>For mammalian plasmides the restriction enzyme used: ScaI. </LI><LI> The mixture is incubated for 1 hour and applied to gel electrophoresis</LI><H2><B>Fungi</B></H2><H3>Transformation in fungi</H3><B>Genetic Transformation of filamentous fungi – protocol from Center for Microbial Biotechnology (CMB) at DTU, Author Nielsen, J. B.</B><BIG><B>Materials</B></BIG><B>Media</B>
Minimal medium (MM) (1L): <SMALL>50 mL D-glucose 20% w/V, 20 mL 50x mineral mix, 10 mL 1 M sodium nitrate, 20 g ager.</SMALL>

Transformation media(TM)(1L): <SMALL> 342.3 g Sucrose, 20 mL 50x mineral mix, 20 g agar.</SMALL>

Mineral Mix (1L):<SMALL> 26g KCL, 26g MgSO4·7H2O, 76g KH2PO4, 50 mL Trace element solution, MilliQ water to volume 1000 mL.</SMALL>

D-glucose 20% (0.5 L): <SMALL> 100g D-glucose and MilliQ water up to 500 ml.</SMALL>

Aspergillus protoplastationbuffer (APB):<SMALL> Final conc. 1.1 M MgSO4 and 10 mM Na-phosphate buffer. pH is adjusted with 2 N NaOH to 5.8.</SMALL>

Aspergillus transformation buffer (ATB):<SMALL>Final conc: 1.2 M Sorbitol; 50 mM CaCl2·2 H2O; 20 mM Tris; and 0.6 M KCl. pH is adjusted with 2 N HCl to 7.2.</SMALL>

PCT (200ml) - Final conc: 50% w/vol PEG 8000; <SMALL>50 mM CaCl2; 20 mM Tris; and 0.6 M KCl. pH is adjusted with 2 N HCl to 7.5. Store at 4 °C.</SMALL><BIG><B>Procedure</B></BIG><LI>The host strain is grown as three-point stabs on Minimal medium plates with the require suppliants added. MM will be referred to as MM with the supplements included throughout the protocol.</LI><B>Inoculation:</B><LI>The conidia are harvested by adding 5 ml of MM and firmly rub with a sterile Grigalsky spatula. The conidial suspension is pipetted to a sterile 500 ml shake flask containing 100 ml MM. The cultures are incubated at 30 °C with 150 rpm of shaking over night (14-20 hours).</LI><B>Mycelial harvest:</B><LI>A funnel with a sterile Mira cloth (filter) is used to harvest mycelia. To remove residual glucose from mycelia the biomass are washed with Aspergillus protoplastationbuffer (APB). </LI><LI>The filtered biomass is transferred to a new Falcon tube with a sterile spoon.</LI><B>Protoplastation:</B><LI> Mycellium is resuspended in 10 ml filter-sterillized(0.45μm filters) APB containing 40 mg Glucanex/ml. The Glucanex is dissolved in APB with gentle magnetic stirring less than 100/min.</LI><LI> Mycelia with dissolved Glucanex are mixed at 30 °C with 150 rpm of shaking for 2-3 hours.</LI><LI>Protoplast solutions are diluted in APB adding up to 40 ml mark. An overlay of max. 5ml Aspergillus transformation buffer (ATB) diluted to ½x with sterile MilliQ-water is carefully placed on top of the APB. </LI><LI>Centrifuged at 13 min 3000 RCF in Sorvall centrifuge. Protoplates should be observed as a halo of with slurry in the interphase of the two liquids. </LI><LI>Withdraw of the protoplasts are done with pipette and placed in a Falcon tube. </LI><LI>ATB is added up to 40 ml mark. Centrifuge at 300 RCF in 13 min and supernatanted are discarded.</LI><LI>The protoplastes are resuspended in 1 ml ATB with a 5 ml pipette.</LI><B>Genetic transformation:</B><LI>Aliquotes of 50 μl are transferred to a 1.5 ml Eppendorf tube containgen 10 μl of DNA for transformation. 
</LI><LI>Protoplast and DNA are incubated at room temperature for at least 30 min.</LI><LI>Protoplat and DNA suspension are added to 1 ml PCT in a 15 ml tube and shake gently. </LI><LI>Incubate for 1-5 min at room temperature. </LI><LI> Dilute in 3 ml ATB. The tube is filled with molten transformation medium (TM) agar (temperature of 40-45°C) to apporeimately 12 ml. The tube is mix rapidly by inverting the tube twice. </LI><LI>Poure directly on pre-made TM plates and incubate at 37°C for 3-8 days.</LI><H3>Production of conidiospores</H3><B><BIG>Materials</BIG></B>
tooth stick 
LB plates <B><BIG>Procedure</BIG></B><LI> Gently touch the a mature colony with a tooth pick.</LI><LI> Hold the LB-plate upside down to avoid scattering of spores, and stab three different places.</LI><LI> Incubated plates at 37°C for approximately five days. After incubation spores were either harvested from the plates or they were stored in the fridge at 4°C.</LI><H3>Flourescence Microscopy</H3><B><BIG>Materials</BIG></B>
MM
Bleomycin 
4-NQO <B><BIG>Procedure</BIG></B>
Cultivation for fluorescence detection was performed by collecting the conidia from plates having three point stab colonies that have grown for 4-5 days. 5ml of MM with supplements were added to the plate and conidia were rubbed off the colonies with a sterile Drigalsky spatula. 

<LI> Inoculate the conidial suspension in 5ml MM in a sterile 50ml Falcon. </LI><LI> Add bleomycin (10mg/mL) and 4-NQO (5 mg/mL). This will cause damage to the DNA inducing gene expression</LI><LI> Incubated at 37 °C with 150 rpm of shaking overnight.
<H3>Fluorescence detection</H3><BIG><B>Materials</B></BIG>
Coverslide + overlay cover. 
Conidia Culture.<BIG><B>Procedure</B></BIG></LI><LI> Take 500µL of conidia culture (ref. till protocol for det) in a tube. This has to be done after 4, 24, 48 and 120 hours .</LI><LI> Centrifuge the tube at 16000g for 1min and 30 sec.</LI><LI>Discard the supernatant.</LI><LI>Resolute the pellet in 5µL Milli Q water and place on slide. Place the overlay cover</LI><LI> Detection microscopy was performed first with DIT filter and fluorescence filter with enlargement of 10x and 100x.<DL><DD>To check background fluorescence, comparison  of detection with yellow filter was made against detection by green and red filter.</DD><H3>Extraction of proteins</H3><BIG><B>Materials</B></BIG>
Z-buffer
AEBSF stock solution
Bradford Reagent
BSA
MilliQ water
Minimal media with aminoacids<BIG><B>Procedure</B></BIG><B>Inoculation of conidia in shaking bottle</B></DL></LI><LI>The colonies from the three point stab are harvested by adding 2mL MilliQ water to the petri dish. The colonies are rubt with a digalski spartula.</LI><LI>500µl of the colony suspension is pipette into a 500mL shaking bottle with 100mL minimal media containing amino acids</LI><LI>Incubate in 48 hours at 37°C</LI><B>Protein extract</B><LI> 2 mL culture are transferred from the shaking bottles to eppendorf tubes. Remember to do triple determination. </LI><LI> Centrifuge tubes in 1 min with 8000g</LI><LI> Remove the supernatant. </LI><LI> Transfer the mycelierne with a sterile spartula to FastPrep tubes.</LI><LI> 250 µL Z-buffer is added to the FastPrep tubes in stink cabinet.</LI><LI>Add approximately 200µL of the small glass balls to the FastPrep tubes.</LI><LI>Add 12,5µL AEBSF stock solution.</LI><LI> Place the FastPrep tubes in the FastPrep machines. Let it run with max velocity  for 30 sec.</LI><LI>Tubes are kept on ice from now on. Add 250 µL Z-buffer and mix well.</LI><LI>The extract is transferred by a 1000 µL pipette. Place the pipet at the bottom of the tube and transfer it to a new eppendorf tube.</LI><LI>Centrifuge the eppendorp tubes in 15 min at 10000g to purify the protein extract.</LI><LI>Transfer the supernatant to a new eppendorp tube.</LI><H2><B>Assays</B></H2><BIG><B>Materials</B></BIG>
Z-buffer
AEBSF stock solution
Bradford Reagent
BSA
MilliQ water
Minimal media with aminoacids
Protein extract (from above)<H3><B>ß-galactosidase assay</B></H3><BIG><B>Procedure</B></BIG><LI> Add 25 µL extract and 225 µL Z-buffer to a well in a microtiter plate. Three wells per extract for triple determination.</LI><LI> Mix enough ONPG stock solution so 50 µL can be added to each well.</LI><LI> Measure OD with the computer program Gen5; Shake the plate every 30 sec to mix the solution, rest for 10 min before measuring the OD.</LI><H3><B>Bradford assay</B></H3><BIG><B>Procedure</B></BIG><LI> Add 5 µL extract and 245 µL Bradford reagent to a well in a microtiter plate. Three wells per extract for triple determination.</LI><LI>Make a BSA standard with concentrations between 0,1-2,0mg/mL with. See table below:</LI><H3>Preparation of BSA standards</H3><TABLE border="3" bordercolor="#990000"><TBODY><TR><TH> Standard</TH><TH> µL standard </TH><TH> µL dH<SUB>2</SUB>O</TH><TH> Concentration</TH></TR><TR><TH> Std 1</TH><TH> 300 µL stock</TH><TH>1200µL</TH><TH> 2 mg/ml </TH></TR><TR><TH> Std 2 </TH><TH> 200 µL stock</TH><TH>1000µL</TH><TH> 1,67mg/ml </TH></TR><TR><TH> Std 3</TH><TH> 700 µL Std 1</TH><TH>700µL</TH><TH> 1 mg/ml </TH></TR><TR><TH> Std 4</TH><TH> 700 µL Std 2</TH><TH>700µL</TH><TH> mg/ml </TH></TR><TR><TH> Std 5</TH><TH> 700 µL Std 3</TH><TH>700µL</TH><TH> 0,5 mg/ml </TH></TR><TR><TH> Std 6</TH><TH> 700 µL Std 4</TH><TH>700µL</TH><TH> mg/ml </TH></TR><TR><TH> Std 7</TH><TH> 700 µL Std 5</TH><TH>700µL</TH><TH> 0.25mg/ml </TH></TR><TR><TH> Std 8</TH><TH> 700 µL Std 7</TH><TH>700µL</TH><TH> 0,125mg/ml </TH></TR><TR><TH> Std 9</TH><TH> - </TH><TH>700µL</TH><TH> 0 mg/ml </TH></TR></TBODY></TABLE><UL><LI>Make triple determination.</LI><LI>Incubate at 37°C for 10 min.</LI></UL><H2><B>Mammalian cells</B></H2><H3>Cell culture and reagents</H3><BIG><B>Materials</B></BIG><B>cells</B>: U2OS cells were kindly provided by The Danish Cancer Society.  U2OS cell line is derived from the bone tissue of a patient suffering from osteosarcoma. U2OS cells show epithelial adherent morphology. <B>Medium</B>:
500 ml DMEM 
50 ml Fetal Calf Serum (FCS)
5 ml Penicillin
5 ml Streptomycin <BIG><B>Procedures</B></BIG><LI> Add FCS, penicillin, and streptomycin to a new flask of DMEM. Store at 5°C.</LI>The cells are cultivated in Dulbecco’s Modified Eagle Medium (DMEM), supplemented with Penicillin, Streptomycin, and 10 % heat-inactivated foetal calf serum (FCS). Penicillin and Streptomycin are added to prevent any microbial growth. FCS is added to supply essential non-defined components, such as serum proteins and lipids. Supplemented DMEM medium is referred to as complete DMEM throughout the report. The cells are adherent and are kept in 75 cm<SUP>2</SUP> culture flask until 80-100% confluency, where they are passed on to new culture flasks or cover slips. 
<H3>Cultivation of cells</H3><BIG><B>Materials</B></BIG>
25 cm<SUP>2</SUP> culture flask 
Complete DMEM <BIG><B>Procedures</B></BIG><LI>Place the appropriate amount of EDTA-trypsin and complete DMEM in the incubator (37°C), before handling the cells (about 1-2 hours before).</LI><LI>The U2OS cells are kept at -80°C until defrosted gently at 37°.</LI><LI>Exactly when defrosted, they are mixed with 12 ml complete DMEM in the pipette. </LI><LI>The cell suspension is transferred to a 15 ml vial and centrifuged at 280 g for 10 minutes. The supernatant is discarded. </LI><LI>The cell pellet is resuspended in 5 ml complete DMEM and transferred to a 25 cm<SUP>2</SUP> culture flask.</LI><LI>The cells are kept at 37°C in a  5% CO<SUP>2</SUP> incubator until the following day, where they are passed on to larger culture flasks.</LI><H3>Passing and maintenance of U2OS cells</H3><BIG><B>Materials</B></BIG>
75 cm<SUP>2</SUP> culture flask:
50 ml vial
Complete DMEM medium
0.05 % EDTA-trypsin<BIG><B>Procedure</B></BIG><LI>The appropriate amount of EDTA-trypsin and complete DMEM in the incubator (37°C), before handling the cells.</LI><LI>The filter of a 1 ml pipette is broken off, and the pipette is attached to the vacuum suction maschine. The medium is removed, and the pipette is put in a 50 ml vial for later use.</LI><LI>1 ml 0,05% EDTA-trypsin/PBS is added to wash the cells. Tilt the flask quickly, so the EDTA-trypsin covers the cells. Remove the liquid quickly. </LI><LI>1 ml 0.05% trypsin-EDTA/PBS is added and the flask is incubated for 3 min at 37°C and 5 % CO<SUP>2</SUP>.</LI><LI>9 ml complete medium is added to inactivate the EDTA-trypsin and to wash the cells of the surface. Make sure the cells are well resuspended, before you transfer them. </LI><LI>2,5 ml (depends on the concentration of the cells) cell suspension is transferred to a new 75 cm<SUP>2</SUP> culture flask. </LI><LI>The cells are kept at 37°C in a 5% CO<SUP>2</SUP> incubator until they are 80-100% confluent. This takes about 2-3 days. Then they are passed on to a new flask or plate.</LI><H3>Transferring cells to coverslips</H3><BIG><B>Materials</B></BIG>
75 cm<SUP>2</SUP> culture flask:
50 ml vial
Complete DMEM medium
0.05 % EDTA-trypsin
6-well plate
Cover slips 
<BIG><B>Procedure</B></BIG><LI>The filter of a 1 ml pipette is broken off, and the pipette is attached to the vacuum suction maschine. The medium is removed, and the pipette is put in a 50 ml vial for later use.</LI><LI>1,5 ml 0,05% EDTA-trypsin/PBS is added to wash the cells. Tilt the flask quickly, so the EDTA-trypsin covers the cells. Remove the liquid quickly.</LI><LI>1,5 ml 0.05% trypsin-EDTA/PBS is added and the flask is incubated for 3 min at 37°C and 5 % CO<SUP>2</SUP>. </LI><LI>9 ml complete medium is added to inactivate the EDTA-trypsin and to wash the cells of the surface. Make sure the cells are well resuspended, before you transfer them.</LI><LI>Calculate the concentration of cells you want in the wells</LI><LI>Cover slips are placed in the bottom of a 6-well plate (2 cover slips/well). 2 ml cell suspension is added gently to each well.</LI><LI>The plate is placed in the incubator at 37°C and 5% CO<SUP>2</SUP> O/N.</LI><H3>Transfection of cells</H3><BIG><B>Materials</B></BIG>
Optimem
FuGENE transfection reagent
Plasmid DNA<BIG><B>Procedure</B></BIG><LI>The LAF bench and gloves is disinfected with ethanol before and after working in the LAF bench. </LI><LI>46μl optimem per is transferred to a 1,5 ml eppendorf tube per tranfection. </LI><LI>3 μl of Fugene is mixed well with the optimem. </LI><LI>The tube is flicked gently to make sure that the solution is properly mixed. Incubation for 5 min at room temperature</LI><LI>1 μl of plasmid DNA is mixed well with the optimem/Fugene mix.</LI><LI>The tube is flicked gently to make sure the mixture is properly mixed.</LI><LI>If there is any liquid remaining on the side of the tube, the tube is centrifuged shortly.</LI><LI>The plasmid mixture is incubated for 15 min at 37°C. </LI><LI>The  dish with U-2OS cells from the incubator. </LI><LI>The 50 μl transfection mixture is added dropwise to the 6 cm<SUP>2</SUP> dish containg the coverslips.</LI><LI>The mixture is gently mixed by rocking motion and the dish is placed back in the incubator.</LI><H3>Coverslips for microscopy</H3><BIG><B>Materialer</B></BIG>
PBS
MilliQ water
Formaldehyde
lint-free paper
Cover slides
Transparent nail polish<BIG><B>Procedure</B></BIG><LI> A 10X PBS dilution is made from PBS and MilliQ water. </LI><LI>The coverslips are transferred to a 24-well plate (2 coverslip/well). </LI><LI>The coverslips are washed by adding 1 ml diluted PBS to the wells. The PBS is discarded. This is done twice</LI><LI>400 μl Formaldehyde is added to each well under the fume hood, and incubated for 12 min at RT. The liquid is transferred to the waste bin </LI><LI>The coverslips are washed 3 times with the diluted PBS. </LI><LI>The coverslips are dipped in MilliQ water before laid on lint-free paper for drying. </LI><LI>4 drops each of 4 μl Vectashield is placed on a glass slide. 
</LI><LI> When a coverslip is completely dry, it is placed on a drop of Vectashield on the glass slide. The procedure is repeated for each coverslip. </LI><LI>The coverslips are fixated with transparent nail polish.
</LI></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols">http://2011.igem.org/Team:DTU-Denmark-2/Team/Protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:DTU-Denmark-2/Team/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:DTU-Denmark-2/Team/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:DTU-Denmark-2/Team/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:DTU-Denmark-2/Team/Protocols&amp;oldid=215386" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>