"
Page
Discussion
View source
History
teams
Log in
 
Team:DTU-Denmark-2/Project/introduction
From 2011.igem.org
Home
About us
The Team
Supervisors
Gallery
The Project
Abstract
Introduction
The Plug 'n' Play
Assembly system
Customization
Other assembly systems
Achievements
Attributions
Results
Background
Characterization
Proof of concept
Mammalian Cells
Fungi
Data page
Submitted Biobricks
Collaboration
Safety
Safety
Safety rules
Notebook
Lab notes
Protocols
Our Partners
Sponsors
Advisors
Acknowledgements
Introduction
What is iGEM?
Synthetic Biology
The world calls for a better Assembly System
USER cloning
Outline of the Plug 'n' Play with DNA concept
The mission of Plug 'n' Play with DNA
Applications
References
What is iGEM?
The iGEM (international Genetically Engineered Machine) competition is the world’s largest competition within synthetic biology, and is hosted by Massachussetts Institute of Technolgy (MIT). The iGEM competition is considered the most prestigious competition for students in the field of biotechnology(1).
The competition started out as a month-long course at MIT, where the students had to design a biological system. This course grew to a summer competition in 2004 with just 5 teams, and since then the competition has expanded dramatically with more than 160 teams from universities all over the world in 2011(1).
The iGEM competition provides a kit with standardized biological parts known as BioBricks, that the teams can use for building the genetic machines. The teams can also submit their own BioBricks. Information about the BioBricks and the toolkit to make and manipulate them is provided by the Registry of Standard Biological Parts . Over the summer, the teams work at their universities, where they use the provided parts plus parts of their own design to build biological systems that can operate in living cells(1).
iGEM is about manipulation of genetic material, where the only limit is ones own imagination. The iGEM competition has demonstrated a new way to arouse students interest in modern biology and to develop their independent learning skills.The project ideas are many and diverse, but overall they show great potential.
Synthetic Biology
Synthetic biology is a relatively new area of biological research that combines science and engineering. Synthetic Biology is about modification of organisms by engineering them to perform new and innovative tasks. An analogy, both conceptualizing the goal and methods of synthetic biology, is computer engineering hierarchy, were  every constituent part is embedded in a more complex system that provides its context. Starting in the bottom of the hierarchy with transistors, capacitors, and resistors; second the engineered logic gates performing binary computations and finally the more complex integrated circuits.
This is equivalent to how synthetic biology works. In synthetic biology standardized parts such as DNA, RNA, proteins, and metabolites are used as the basic components. The device layer regulates physical processes by the constructed biochemical reactions. The parts can be assembled into modules in order to assemble more complex pathways, having similarity with the integrated circuits. Connection of pathways and their integration into host cells allow researchers to extend or modify the behaviour of cells in a programmatic fashion (2).
The world calls for a better Assembly System
The Registry of Standard Biological Parts and iGEM make use of the Standard Assembly of BioBricks formulated by Tom Knight.   The BioBrick Standard Assembly makes use of the restriction recognition sites of the four restriction enzymes EcoRI, XbaI, SpeI, and PstI. The BioBricks are flanked by a prefix and a suffix containing the restriction recognition sites EcoRI, XbaI, and SpeI, PstI, respectively. To ensure a correct assembly the BioBricks cannot contain any of these four restriction recognition sites. This means that if any of these four restriction recognition sites are present, they will have to be eliminated by alterations like site-directed mutagenesis, which can be time consuming and laborious. In developing new BioBricks from natural sources and higher organism such as eukaryotes the illegal restriction sites can be a problem. Furthermore, when assembling BioBricks with the Standard Assembly System scars between the Biobricks are introduced, this can be a problem for the construction of fusion proteins. Additionally, the BioBrick Assembly Standard has the drawback that only two BioBricks can be assembled at a time (3).
All in all, the iGEM competition and the fast growing field of synthetic biology calls for a simpler, faster and more efficient assembly system that is easily applied to both bacteria, fungi, and mammalian cells.
USER cloning
In the early 1990s, uracil excision-based (USER) cloning was invented as a ligation-independent cloning technique that could substitute the conventional cloning methods, which mades use of restriction enzymes and ligase. In 2003 New England Biolabs (NEB) introduced the USER Friendly Cloning Kit. Although NEBs USER kit was simple and efficient, it was not compatible with proofreading polymerases that stalled when encountering a uracil base in the DNA template (4). This made the USER Friendly Kit unattractive, although the concept was brilliant. In recent years, proofreading polymerases have been developed that are compatible with the concept of USER cloning, since they can read through uracil (5).
The USER method applies long complementary overhangs on the PCR product(s) as well as on the destination vector. The overhangs on the PCR product are custom made, between 7-15 nucleotides long and deoxy uridine nucleotides substitute selected deoxy thymidine nucleotides. The PCR products containing the customized overhangs are treated with the USER enzyme, which is a mix of DNA glycosidase and DNA glycosylase-lyase endo VIII. This treatment results in release of the DNA sequence upstream of the deoxy uridine nucleotide and the resulting exposed overhangs can anneal to each other to form a stable hybridization product. This product can now be transformed directly into E.coli without prior ligation (4,5,6). In order to avoid template carry-over after PCR, the PCR product is usually treated with the restriction enzyme DpnI. DpnI cleaves only when its recognition site is methylated. Unmethylated PCR-derived DNA will be left intact (5).
Outline of the Plug 'n' Play with DNA concept
We introduce a standardized assembly system based on the principle of USER cloning and the USER fusion Assembly standard introduced by the 2009 DTU iGEM team. The new assembly standard called Plug 'n' Play with DNA ( BBF RFC 80 ) offers easier cloning and is a combined standard and assembly system.
The time required to construct devices and plasmids can be significantly reduced with the use of Plug 'n' Play. This is essential to be able to move beyond what we theoretically can create with synthetic biology and conducting it in practice.
The mission of Plug 'n' Play with DNA
We introduce the standardized and versatile system called "Plug ‘n’ Play with DNA", where categories of biological parts can be gathered. We imagine that BioBricks in the form of pre-produced PCR-products, can be directly mixed with a backbone vector. This will make assembly of expression vectors possible in only a few hours. All our parts in form of PCR-products, should be distributed in microtiter plates, like the original iGEM BioBrick kit. The parts in Plug 'n' Play kit are directly ready for cloning. Furthermore, the "Plug ’n’ Play" kit will contain back-up plasmids of all parts to ensure amplification from a mutation free template if needed.
Applications
The Plug ‘n’ Play assembly standard can be applied in a lot of different contexts. Use of the Plug 'n' Play assembly standard is especially advantageous for high throughput projects. In a previous study by Hussam H. Nour-Eldin et. al. (5), they were able to clone 240 genes from a library into an expression vector in only three weeks with efficiency close to 95% (5).
Below we have listed some different applications in which our standard assembly would be particularly useful.
Construction of DNA libraries.
High-troughput cloning.
Development of vector systems for gene expression analysis.
Development of genetic tools for eukaryotes.
Development of reporter systems.
To proof our concept, we developed a reporter system for mammalian cells and for Aspergillus nidulans .
References
[1] https://igem.org/About (Website, accessed 19/09/2011)
[2] Andrianantoandro, E. et. al. Synthetic biology: new engineering rules for an emerging discipline. Molecular Systems Biology: 10: 1-14 (2006).
[3] http://partsregistry.org/Assembly:Standard_assembly (Website, accessed 19/09/2011) [4] New England Biolabs, 2004
[5] Hussam H. Nour-Eldin, Fernando Geu-Flores, and Barbara A. Halkier. USER Cloning and USER Fusion: The Ideal Cloning Techniques for Small and Big Laboratories. Methods in Molecular Biology 643.
[6] Nørholm, M. H. H. A mutant Pfu DNA polymerase designed for advanced uracil-excision DNA engineering. BMC Biotechnol. 10, 21 (2010).
Retrieved from " http://2011.igem.org/Team:DTU-Denmark-2/Project/introduction "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
