"
Page
Discussion
View source
History
teams
Log in
 
Team:DTU-Denmark-2/results/Characterisation
From 2011.igem.org
Home
About us
The Team
Supervisors
Gallery
The Project
Abstract
Introduction
The Plug 'n' Play
Assembly system
Customization
Other assembly systems
Achievements
Attributions
Results
Background
Characterization
Proof of concept
Mammalian Cells
Fungi
Data page
Submitted Biobricks
Collaboration
Safety
Safety
Safety rules
Notebook
Lab notes
Protocols
Our Partners
Sponsors
Advisors
Acknowledgements
Characterization
Characterization
Genetics and USER cloning
Qualitative analysis
Quantitative analysis
References
Characterization
Two fungal promoters have been characterized by both a qualitative and a quantitative analysis. The plan was to characterize the fungal promoters; DMKP-P6 also known as PAN1122, a medium strength constitutive promoter and PalcA an inducible promoter. A simple way of analyzing promoters is by measuring the expression of a reporter gene. This was done by performing the widely used β-galactosidase assay (1) with the modifications described here .
Genetics and USER cloning
Aspergillus nidulans can integrate DNA fragments into its genome by exploitation of the natural mechanisms for double-strand break (DSB) repair. In fungi, the most widely occurring mechanisms for DSB repair are non-homologous end joining (NHEJ) and homologous recombination (HR). Integration by NHEJ will occur randomly, which means that DNA fragments will be integrated at a random site in the genome, and with alternating copy numbers. HR uses widespread homology search to repair breaks and does this without losing any of the sequence around the break (3, 4). For the characterization of the promoters it was important only to have one copy integrated in the genome. The host strain used for transformation nkuAΔ, was therefore a NHEJ deficient strain, and the integration should occur by HR (2).
p68 is a plasmid that contains a lacZ gene, a terminator, and a USER cassette. For HR to occur, the gene-targeting substrate has to contain two large sequences of around 1500 bp. Therefore, p68 also contains up- and down stream regions for targeting to the specific site called Insertion Site 1 (IS1) situated 202 bp downstream of AN6638 gene and 245 bp upstream of AN6639 gene, this ensures the targeted integration (5).
p68 was digested with AsiSI for 2 hours and afterwards nicked with Nb.BstI for 1 hour. The vector and each of the promoters were then mixed in a USER reaction. Prior to transformation of A. nidulans the plasmids were cut with NotI to obtain a linearized DNA fragment to make HR possible and thereby increase transformation efficiency. The nkuAΔ transformants containing PalcA::lacZ and the nkuAΔ transformants containing DMKP-P6::lacZ will be referred to as nkuAΔ-IS1::PalcA::lacZ::TtrpC::argB and nkuAΔ-IS1::DMKP-P6::lacZ::TtrpC::argB, respectively.
Qualitative analysis
First, the promoters were evaluated qualitatively by inoculating nkuAΔ-IS1::PalcA::lacZ::TtrpC::argB and nkuAΔ-IS1::DMKP-P6::lacZ::TtrpC::argB onto minimal media plates containing 5-bromo-4-chloro-3-indolyl-D-galactoside (X-gal). A functional promoter allows the expression of the lacZ gene and thereby production of β-galactosidase, which  results in blue colonies on X-gal plates.  The blue color is produced because β-galactosidase cleaves X-gal into 5- bromo-4-chloro-3-indolyl (blue) and D-galactose. Thus, blue colonies means that the transcription of the lacZ gene has occurred. It should be noted that the X-gal plates used for the PalcA transformants contained glycerol as carbon source and ethanol and threonine to induce the PalcA promoter.
The plates contain two positive controls that express lacZ from the constitutive promoters PgpdA with two different lengths, PgpdA 0.5kb and PgpdA 1.0kb (nkuAΔ-IS1::PgpdA 0.5kb::lacZ::TtrpC::argB and nkuAΔ-IS1::PgpdA 1.0kb::lacZ::TtrpC::argB) that are used for comparison of the intensity of the blue colour. Moreover, the reference strain nkuAΔ-IS1::PgpdA::TtrpC::argB (without the lacZ gene) was also inoculated on the plates. The PgpdA 0.5kb promoter on both plates appeared to endorse the strongest expression. When comparing the intensities of the two transformants with the DMKP-P6 promoter it is clear that the expression of lacZ is most similar to the expression with PgpdA 1.0kb. The same observation was made for the PalcA-2 promoter, however the expression of the lacZ gene in the PalcA transform ants seam to differ a bit, since the colour intensity for PalcA-1 is the lower than for PalcA-2. Thus, the qualitative analysis indicates that both promoters DMKP-P6 and PalcA are of medium strength. This is in accordance with previous data for the DMKP-P6 promoter (not published), but PalcA is known to be a strong promoter and we would have expected to see the same colour intensity of the PalcA transformants as we did for the nkuAΔ-IS1::PgpdA 0.5kb::lacZ::TtrpC::argB strain. A reason for this could be a sub-optimalt level of the inducers ethanol and threonine in the X-gal plates.
Quantitative analysis
The level of protein production was examined by performing a β-galactosidase assay. First, conidia from a three-point inoculation were grown in minimal media in shake flasks for 48 hours with the appropriate supplements. For each strain, two individual transformants were used, thus providing biological replicates.
Filamentous fungi have a tendency to growth in pellets, when circumstances are not optimal. Growth in pellets was observed for all the suspensions, but for the strains with the PalcA promoter it was more pronounced, most likely due to the different growth medium. Then proteins were extracted from the cultures and used for the β-galactosidase assay and Bradford assay (described below). All measurements were performed in triplicates.
It can be very difficult to measure the optical density of fungi, because they grow in complex structures, are heavy and not single celled like bacteria. Therefore the OD measurement that is usually performed when conducting the β-galactosidase assay would not be accurate enough. The protein concentration of the fungal samples were instead determined by a Bradford assay. For the Bradford assay a standard dilution series with known concentrations of bovine serum albumin (BSA) were made in order to determine the protein concentrations. The protein samples and BSA standards were mixed with Bradford reagent. The procedure is based on the dye, Brilliant Blue G (Sigma-Aldrich), that forms a complex with the proteins in solution. This dye-protein complex results in a shift of the absorption maximum of the dye from 465nm to 595nm, where the absorption is proportional to protein present.
For the β-galactosidase assay, a solution of o–nitrophenyl-β–D–galactoside (ONPG) was used to measure the  activity of β-galactosidase. β-galactosidase hydrolyses ONPG to o–nitrophenol at a linear rate until ONPG is completely degraded. In other words, the amount of o-nitrophenol produced is proportional to the amount of β-galactosidase present in the sample (6). This can be seen as a yellow colour that becomes more and more intense as the degradation proceeds.
Protein extracts were mixed with Z-buffer in a microtiter plate. ONPG solution was added, and OD420 was measured every minute for 20 minutes. The specific activities were calculated using the equation below.
Where:
Abs420 = the absorbance of o-nitrophenol measured.
the factor 1.7 corrects for the reaction volume.
0.0045 is the absorbance of a 1 nmol/mL o-nitrophenol solution.
[p] = the concentration of protein [mg/mL]
v = volume of culture assayed [mL]
t = the reaction time [min]
Specific activities for the three promoters were calculated according to the equation above. The specific activities of the promoters are an expression of the specific activity of β-galactosidase. All measurements in the figures are for the time: 5 minutes. The mean specific promoter activity for each sample (based on triplicates) is shown below.
The strain nkuAΔ-IS1::PalcA::lacZ::TtrpC::argB converted o-nitrophenyl-β-D-galactoside at a rate of 0.93 μmol/min/mg total protein, and the strain nkuAΔ-IS1::DMKP-P6::lacZ::TtrpC::argB converted the same compound at a rate of 1.3 μmol/min/mg total protein while the reference strain nkuAΔ-IS1::PgpdA 1.0kb::lacZ::TtrpC::argB converted it at a rate of 3.5 μmol/min/mg total protein. The negative reference did not show any detectable activity. It should be noted that the growth medium for the strain containing PalcA was different from the growth medium of the reference strains. Furthermore, the PalcA strain did not grow well in the glycerol containing media. This means that in principal PalcA can not be compared to the reference. The figure above shows that the specific activity of PgpdA 1.0 kb is more than double the specific activity of DMKP-P6. The figure also shows that the specific activity of PalcA when compared to PgpdA 1.0 kb is more than three times as high. Compared with the qualitative analysis, we would have expected that the specific activity of PalcA and DMKP-P6 would have been similar to the activity of PgpdA 1.0 kb this was however not the case.
References
(1) Miller, J. H. 1972. Experiments in molecular genetics. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY.
(2) Nielsen, Jakob B.; Michael L. Nielsen; and Uffe H. Mortensen; “Transient disruption of non-homologous end-joining facilitates targeted genome manipulation in the             filamentous fungus Aspergillus nidulans.” Elsevier, 2008.
(3) Mortensen, Uffe; Center for Mikrobiel Bioteknologi. 28 January 2008. http://www.cmb.bio.dtu.dk/Forskning/eukaryotic_molecular_biology/A,-         d,%20nidulans%20mutant%20library.aspx.
(4) Krappmann, Sven; “Gene Targeting in filamentous fungi: the benefits of impaired repair.” The British Mycological Society, 2007: 25-29.
(5) Hansen, Bjarne G.; Bo Salomonsen; Morten T. Nielsen; Jakob B. Nielsen; Niels B. Hansen; Kristian F. Nielsen; Torsten B. Regueira; Jens Nielsen; Kiran R. Patil; and Uffe H.          Mortensen; “Versatile enzyme expression and Characterization system for Aspergillus, with the Penicillium brevicompactum Polyketide Synthase Gene from the Mycophenolic           Acid Gene Cluster as a Test Case.” American Society for Microbiology, 2011, 3044-3051.
(6) Storms, Reginald; Yun Zhenga; Hongshan Li; Susan Sillaots; Amalia Martinez-Perez: and Adrian Tsanga; “Plasmid vectors for protein production, gene expression and          molecular manipulations in Aspergillus niger.” 2005: 191–204.
Retrieved from " http://2011.igem.org/Team:DTU-Denmark-2/results/Characterisation "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
