"
Page
Discussion
View source
History
teams
Log in
 
Team:UNIPV-Pavia/Project/Results
From 2009.igem.org
Home
The Project
Motivation
Solution
Results & Conclusions
References
Materials & Methods
Measurements
Instruments
Protocols
Software Developed
Parts
Submitted
Characterization
The Team
Notebook
Week-book
Freezer Management
Biological Safety
Pavia
History
University
Gallery
Sponsors
Gallery
Team Locations
Results & Conclusions
Experimental results
In this section a summary of "Ethanol? Whey not!" detailed results is reported. For more information you can visit Parts Characterization section, in which you can find a documentation for each every BioBrick involved in this project.
Ethanol fermentation
Survival of E. coli TOP10 in ethanol
Different concentrations of ethanol have been added to M9 supplemented with glycerol and LB selective media (+ Amp) and E. coli TOP10 (bearing BBa_B0015 BioBrick) survival, when grown in a microplate reader (37°C, automatic protocol ) and in a 50 ml falcon tube (30 ml of culture, 37°C, 220 rpm), has been tested.
Growth in microplate in LB medium.
Growth in microplate in M9 supplemented with glycerol medium.
Growth in 50 ml falcon tube in LB medium.
Growth in 50 ml falcon tube in M9 supplemented with glycerol medium.
Of the concentrations we tested, in the microplate reader bacteria can grow at concentrations up to 2.5%w/v (LB) and 1.5%w/v (M9), while in the 50 ml falcon tube they can grow at concentrations up to 3.5%w/v (LB) and 2.5%w/v (M9).
In general, in the falcon tube bacteria grow better than in the microplate (see Measurement section for details) and this is confirmed by these data: it seems that the toxicity threshold concentration of ethanol is higher for growth in falcon tube than in the microplate (for example, at 2.5%w/v in M9, bacteria can survive in the falcon tube, but not in the microplate).
Ethanol production
A synthetic ethanol-producing operon () has been built up by our team and has been assembled downstream of several promoters or devices:
- tetR promoter (in high copy plasmid)
- constitutive promoter (in high copy plasmid)
- constitutive promoter (in high copy plasmid)
- 3OC6-HSL inducible system (in high and low copy plasmids)
These strain's most important phenotype qualitative characteristics can be summarized as follows:
strains that had tetR promoter, J23106 or J23118 upstream of the operon gave very small colonies when plated on selective LB agar plates (with or without 2% of glucose);
strains with 3OC6-HSL inducible system upstream of the operon in low copy plasmid could grow when induced with 100 nM and 1 uM HSL;
strains with 3OC6-HSL inducible system upstream of the operon in high copy plasmid died when induced with 100 nM and 1 uM HSL, but could grow when not induced;
bacterial density of strains with 3OC6-HSL inducible system upstream of the operon in liquid cultures supplemented with 2% or 10% of glucose were higher than the density of all other tested strains grown in the same conditions.
All the other details can be found on Parts Characterization page, as well as in  Registry page.
Comments:
The growth results of strains bearing a regulated ethanol-producing operon confirm the metabolic burden given by this BioBrick when gene expression is triggered: induced strains bearing a high copy plasmid containing BBa_K173021 do not survive, while low copy ones can survive even at high induction (1 uM 3OC6-HSL). In all the presented experiments, we decided to exploit the lux promoter leakage activity for BBa_K173021 in pSB1AK3 (high copy), while we decided to induce BBa_K173021 in pSB4C5 (low copy) the with 1 uM of 3OC6-HSL.
The most important phenotype quantitative characteristics of these strain can be summarized as follows:
1 - Growth of strains bearing:
3OC6-HSL inducible system upstream of the operon in high copy plasmid () uninduced;
3OC6-HSL inducible system upstream of the operon in low copy plasmid () uninduced;
3OC6-HSL inducible system upstream of the operon in low copy plasmid () induced with 1 uM of 3OC6-HSL;
3OC6-HSL inducible system without the operon in high copy plasmid ();
a promoterless operon in high copy plasmid ().
has been monitored in selective LB + 10% glucose starting from an inoculum in the microplate reader (figure below).
Mean values on three wells filled with aliquots of the indicated strains (200 ul) in selective LB + 10% glucose.
Mean values with 95% confidence intervals for the mean on three wells filled with aliquots of the indicated strains (200 ul) in selective LB + 10% glucose.
2 - Ethanol fermentation capability of these strains in 10%w/v glucose-supplemented LB medium has been tested with three different fermentation protocols (see Fermentation experiments for a detailed description).
The main fermentation results can be summarized with the following bar plots. Ethanol and pH have been measured after 48 hours ( PROTOCOL#2 , first two measurements) or 24 hours ( PROTOCOL#3 , other measurements).
Ethanol production after fermentation of 10% glucose.
pH after fermentation of 10% glucose.
Comments:
The pH results suggest that in strains bearing expressed pdc and adhB the fermentation does not give high levels of organic acids as in negative control strains which do not have pdc and adhB. It is surprising that the pH of BBa_K173003 in pSB1AK3 (pdc and adhB without any promoter) is higher than in the negative control. This could be due to a weak spurious transcription of pdc and adhB, amplified by the high copy number plasmid. This may re-direct part of pyruvate metabolism to ethanol and not to organic acids. Of course this phenomenon should be further investigated.
The results about dynamic growth (in terms of O.D.600) are in accordance with the pH results, confirming that strains bearing BBa_K173021 in pSB1AK3 produce less organic acids than the negative controls, hopefully thanks to pdc and adhB which re-direct the metabolism of pyruvate from organic acids to ethanol, and consequently grow better. No significant difference was noticed in the growth of promoterless operon, uninduced and induced BBa_K173021 in pSB4C5, suggesting that in promoterless operon transcription could be activated in unspecific manner (i.e. without a promoter upstream).
Fermentation results showed that strains bearing expressed pdc and adhB in pSB1AK3 high copy plasmid can produce less ethanol than the negative controls in both PROTOCOL#2 and PROTOCOL#3 . Our hypothesis is that in our working conditions adhB converts part of the fermented ethanol back to acetaldehyde, in fact this enzyme is able to perform both acetaldehyde conversion to ethanol and the opposite reaction.
On the other hand, if we add 1%w/v of ethanol in the medium this strain can ferment 30 g/l of ethanol, which is about 60% of theoretical yield. This may be due to adhB behaviour, which was reported in literature to be activated by ethanol accumulation.
Moreover, It is possible that when starting from less than 1% of ethanol in fermentation medium ethanol production could be increased, because in our experiment the bacterial culture reached ethanol concentration survival threshold (40 g/l) and so it difficult to ferment more than 30 g/l.
For all these reasons, we can say that we created two ethanol producing biological systems:
BBa_K173003 in pSB1AK3
BBa_K173003 in pSB4C5
and found promising working conditions for them, even if a massive optimization work should be further completed.
Lactose metabolism
A bacterial beta-galactosidase protein generator () was assembled from the parts contained in the Registry, even if it had already been designed and submitted as , whose sequence was inconsistent according to iGEM QC.
The enzyme, encoded by lacZ gene, cleaves lactose in glucose and galactose and can be used, properly assembled downstream of a regulatory module, to speed up the lactose metabolism.
We used tetR promoter, contained in , to produce a constitutive expression of lacZ.
The correct behaviour of the expressed beta-galactosidase enzyme has been tested in TOP10 E. coli (which do not have a working lacZ gene in its genome) on X-Gal plates (more details Parts Characterization section):
Constitutively expressed beta-galactosidase ( in pSB1AK3)
Negative control: TOP10 bearing  in pSB1A2
strain, which contains a working lacZ gene
Promoterless beta-galactosidase protein generator ( in pSB1AK3)
This qualitative result shows that:
beta-galactosidase is expressed in our engineered strain (1st picture), while in wild type E. coli TOP10 (2nd picture) it is not expressed;
Surprisingly, as reported in the 4th picture, the promoterless protein generator shows blue colour. It may be due to i) spurious transcription of the protein generator in the high copy number plasmid pSB1AK3 or to ii) the recombination occurred between plasmidic lacZ and genomic lacZdeltaM15, in which the working lacZ was integrated in E. coli genome under the control of lac promoter. This phenomenon has still to be studied. Caltech iGEM 2008 team reported this phenomenon in a similar protein generator, in which beta-galactosidase assay was performed.
Anyway, further comparative tests should be done in order to see if lactose cleavage can be performed faster than in wild type E. coli, after the choice of a suitable promoter which controls this protein generator.
Promoter characterization
Several promoters and devices have been tested by our team during this summer. Our efforts have been focused on standard characterization methodologies, so we decided to try to apply the Relative Promoter Units ( Kelly J. et al., 2008 ) concept to do this. Regulation of gene expression is very important for our project as we want to optimize the expression rate of two actuators, one for lactose conversion to glucose and one for ethanol production from glucose. In our case, it essential do have a sort of "genetic regulators user's handbook", in which all the functional characteristics of promoters can be found and in which we can choose the suitable regulator for us.
We dedicated a large part of our project in this direction and here we summarize the promoter strength measurements. In the following bar plot we report:
X axis - the name and the working condition (medium, induction) of the promoter or device;
Y axis - the measured strength of the tested promoter or device reported in RPUs, as well as the minimum and the maximum measured values.
All these BioBrick parts or devices have been validated in Invitrogen TOP10 E. coli strain in pSB1A2 standard vector.
Project conclusions and perspectives
A complete ethanol-producing operon has been assembled using BioBrick parts.
A lot of work has still to be done in order to exploit the standard measurement concepts to optimize the gene expression of this operon.
We found very useful to use inducible promoters as a sort of gene expression knob, through which the optimal promoter strength can be easily estimated without performing any other assembly.
We proved that E. coli bearing an ethanol-producing operon capable of gene expression can actually convert glucose into ethanol in particular working conditions. The best set of conditions is still not clear from the literature and has to be optimized.
Further work should be done to speed up the lactose cleaving process in E. coli.
With this project we hope to have provided many useful physical parts and several knowledge about ethanol fermentation pathway and promoter characterization.
We demonstrated the feasibility of lactose conversion to ethanol through the design and the implementation of genetically engineered machines and we hope in the future to exploit Synthetic Biology concepts to optimize our machine for biofuel actual production.
Retrieved from " http://2009.igem.org/Team:UNIPV-Pavia/Project/Results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
