<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UNIPV-Pavia_New_Parts"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:UNIPV-Pavia/New_Parts">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UNIPV-Pavia/New_Parts&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UNIPV-Pavia/New_Parts&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UNIPV-Pavia/New_Parts&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UNIPV-Pavia/New_Parts" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UNIPV-Pavia/New Parts</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><TABLE width="100%" border="0"><TBODY><TR><TD valign="top" width="19%" rowspan="2"><P><FONT face="eras demi itc"><UL id="MenuBar1" class="MenuBarVertical"><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia">Home</A></LI><LI><A class="MenuBarItemSubmenu">The Project</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Project/Motivation">Motivation</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Project/Solution">Solution</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Project/Results">Results &amp; Conclusions</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Project/References">References</A></LI></UL><LI><A class="MenuBarItemSubmenu">Materials &amp; Methods </A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Methods_Materials/Measurement">Measurements</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Methods_Materials/Instruments">Instruments</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Methods_Materials/Protocols">Protocols</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Software">Software Developed</A></LI><LI><A class="MenuBarItemSubmenu">Parts</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Parts">Submitted</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Parts_Characterization">Characterization</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Team">The Team</A></LI><LI><A class="MenuBarItemSubmenu">Notebook</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Notebook">Week-book</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Freezer">Freezer Management</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Safety">Biological Safety</A></LI><LI><A class="MenuBarItemSubmenu" href="https://2009.igem.org/Team:UNIPV-Pavia/Pavia">Pavia</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Pavia#History">History</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Pavia#University">University</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Pavia#Gallery">Gallery</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Sponsors">Sponsors</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Gallery">Gallery</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Team_Locations">Team Locations</A></LI></FONT></P></TD><TD align="right"><FONT face="Pristina" size="50"><B>Our new parts</B></FONT></TD></TR><TR><TD><P><STRONG class="selflink"><B>Our new parts</B>:</STRONG></P><UL><LI><A href="/Team:UNIPV-Pavia/New_Parts#BBa_K173003_-_ethanol_producing_device" title="Team:UNIPV-Pavia/New Parts"> BBa_K173003 - ethanol producing device</A></LI><LI><A href="/Team:UNIPV-Pavia/New_Parts#BBa_K173007_-_aTc_inducible_device_with_J23100_promoter" title="Team:UNIPV-Pavia/New Parts"> BBa_K173007 - aTc inducible device with J23100 promoter</A></LI><LI><A href="/Team:UNIPV-Pavia/New_Parts#BBa_K173011_-_aTc_inducible_device_with_J23118_promoter" title="Team:UNIPV-Pavia/New Parts"> BBa_K173011 - aTc inducible device with J23118 promoter</A></LI><LI><A href="/Team:UNIPV-Pavia/New_Parts#BBa_K173010_-_lactose.2FIPTG_inducible_device_with_J23118_promoter" title="Team:UNIPV-Pavia/New Parts"> BBa_K173010 - lactose/IPTG inducible device with J23118 promoter</A></LI></UL><P><A href="/Team:UNIPV-Pavia/Rebuilt" title="Team:UNIPV-Pavia/Rebuilt"><B>Re-built existing parts (BBa_our part code/BBa_existing part code)</B></A></P><P><A href="/Team:UNIPV-Pavia/Existing" title="Team:UNIPV-Pavia/Existing"><B>Existing parts from the Registry</B></A></P><P><A href="/Team:UNIPV-Pavia/Debug" title="Team:UNIPV-Pavia/Debug"><B>Existing parts: sequence debugging</B></A></P><H2><SPAN class="mw-headline" id="BBa_K173003_-_ethanol_producing_device">BBa_K173003 - ethanol producing device</SPAN></H2><H3><SPAN class="mw-headline" id="Description">Description</SPAN></H3><P>This device takes PoPS as input and produces pyruvate decarboxylase (pdc) and alcohol dehydrogenase II (adhB) enzymes.
</P><P>Pyruvate decarboxylase (pdc, ) catalyses the decarboxylation of pyruvic acid to acetaldehyde and carbon dioxide, while alcohol dehydrogenase II (adhB, ) catalyses the acetaldehyde reduction to ethanol. The latter enzyme, as reported in the literature, is also able to work in the opposite direction, oxidizing ethanol to acetaldehyde. The two enzymes come from Zymomonas mobilis ethanologenic bacterium and constitute an essential step of alcoholic fermentation.
</P><P>So, this device contains the minimum set of genes that are required to engineer a heterologous fermentation pathway.
</P><P>The coding sequences of pdc and adhB genes have been optimized for Escherichia coli codon usage.
</P><H3><SPAN class="mw-headline" id="Characterization">Characterization</SPAN></H3><P><B>Qualitative phenotype characterization</B></P><P><I>PLATES RESULTS</I></P><P>This device has been cloned downstream of 4 different promoters, one of which in two different vectors. These assemblies were transformed in TOP10 and gave the following phenotypes when bacteria were plated on LB agar plates or on LB agar plates + 2% glucose (all the plates have been incubated at 37°C for about 11 hours, then some colonies were picked with a sterile tip and plates were incubated for additional 10 hours):
</P></TD></TR><TR><TD><B>upstream promoter</B></TD><TD><B>plasmid</B></TD><TD><B>promoter RPU in LB</B></TD><TD><B>colony phenotype</B></TD></TR><TR><TD>1.58 [1.56; 1.67]
</TD><TD>small colonies
</TD></TR><TR><TD>not estimated
</TD><TD>small colonies
</TD></TR><TR><TD>0.69 [0.64; 0.73]
</TD><TD>small colonies
</TD></TR><TR><TD>~0 without induction
</TD><TD>normal colonies
</TD></TR><TR><TD>not estimated
</TD><TD>normal colonies
</TD></TR></TBODY></TABLE><P>These results demonstrate that a constitutive expression of pdc and adhB gives a high metabolic burden to E. coli, in fact the only normal phenotype on plates was  with  non-induced device. Only this assembled part has been submitted to the Registry, as .
</P><TABLE align="center"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">BBa_F2620-BBa_K173003 in pSB1AK3 plate: example of normal size colonies.</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">BBa_K173005-BBa_K173003 in pSB1AK3 plate: example of small size colonies.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P><I>LIQUID CULTURES RESULTS</I></P><P>Grown cultures of  and  after the FIRST INOCULUM showed different phenotypes (qualitative analysis), as a function of the used protocol.  was the regulated operon,  was the promoterless operon,  was the promoter alone, pSB1AK3 was a high copy plasmid and pSB4C5 was a low copy plasmid.
</P><P><A href="/Team:UNIPV-Pavia/Parts_Characterization#PROTOCOL.231" title="Team:UNIPV-Pavia/Parts Characterization"> PROTOCOL#1</A> RESULTS (i.e. shaken anaerobic falcon tubes, no glucose):
</P><TABLE align="center" border="1"><TBODY><TR><TD><B>BioBrick</B></TD><TD><B>plasmid</B></TD><TD><B>inductionwith 3OC6-HSL</B></TD><TD><B>phenotype</B></TD></TR><TR><TD>pSB1AK3
</TD><TD>no
</TD><TD>normal growth
</TD></TR><TR><TD>pSB1AK3
</TD><TD>no
</TD><TD>normal growth
</TD></TR><TR><TD>pSB4C5
</TD><TD>no
</TD><TD>normal growth
</TD></TR><TR><TD>BBa_F2620 ALONE- negative control
</TD><TD>pSB1A2
</TD><TD>no
</TD><TD>normal growth
</TD></TR></TBODY></TABLE><P><A href="/Team:UNIPV-Pavia/Parts_Characterization#PROTOCOL.232" title="Team:UNIPV-Pavia/Parts Characterization"> PROTOCOL#2</A> RESULTS (i.e. unshaken anaerobic falcon tubes, 2% glucose):
</P><TABLE align="center" border="1"><TBODY><TR><TD><B>BioBrick</B></TD><TD><B>plasmid</B></TD><TD><B>inductionwith 3OC6-HSL</B></TD><TD><B>phenotype</B></TD></TR><TR><TD>pSB1AK3
</TD><TD>no
</TD><TD>a lot of foam in the upper side of the culture;a strangely big and cloudy pellet after 4000 rpm centrifugation.
</TD></TR><TR><TD>pSB1AK3
</TD><TD>100 nM
</TD><TD>no growth
</TD></TR><TR><TD>pSB1AK3
</TD><TD>1 uM
</TD><TD>no growth
</TD></TR><TR><TD>BBa_F2620 ALONE- negative control
</TD><TD>pSB1A2
</TD><TD>no
</TD><TD>normal growth
</TD></TR><TR><TD>BBa_F2620 ALONE- negative control
</TD><TD>pSB1A2
</TD><TD>100 nM
</TD><TD>normal growth
</TD></TR><TR><TD>BBa_F2620 ALONE- negative control
</TD><TD>pSB1A2
</TD><TD>1 uM
</TD><TD>normal growth
</TD></TR><TR><TD>pSB1AK3
</TD><TD>no
</TD><TD>normal growth
</TD></TR></TBODY></TABLE><P><A href="/Team:UNIPV-Pavia/Parts_Characterization#PROTOCOL.233" title="Team:UNIPV-Pavia/Parts Characterization"> PROTOCOL#3</A> RESULTS (i.e. shaken anaerobic falcon tubes, 2% glucose):
</P><TABLE align="center" border="1"><TBODY><TR><TD><B>BioBrick</B></TD><TD><B>plasmid</B></TD><TD><B>inductionwith 3OC6-HSL</B></TD><TD><B>phenotype</B></TD></TR><TR><TD>pSB1AK3
</TD><TD>no
</TD><TD>higher amount of bacterial pellet than in the negative control (BBa_F2620)after 4000 rpm centrifugation
</TD></TR><TR><TD>pSB1AK3
</TD><TD>100 nM
</TD><TD>no growth
</TD></TR><TR><TD>pSB1AK3
</TD><TD>1 uM
</TD><TD>no growth
</TD></TR><TR><TD>pSB4C5
</TD><TD>no
</TD><TD>a bit higher amount of bacterial pellet than in the negative control (BBa_F2620),but lower than in pSB1AK3 after 4000 rpm centrifugation
</TD></TR><TR><TD>pSB4C5
</TD><TD>1 uM
</TD><TD>a bit higher amount of bacterial pellet than in the negative control (BBa_F2620),but lower than in pSB1AK3 after 4000 rpm after 4000 rpm centrifugation
</TD></TR><TR><TD>BBa_F2620 ALONE- negative control
</TD><TD>pSB1A2
</TD><TD>no
</TD><TD>normal growth
</TD></TR><TR><TD>BBa_F2620 ALONE- negative control
</TD><TD>pSB1A2
</TD><TD>100 nM
</TD><TD>normal growth
</TD></TR><TR><TD>BBa_F2620 ALONE- negative control
</TD><TD>pSB1A2
</TD><TD>1 uM
</TD><TD>normal growth
</TD></TR><TR><TD>pSB1AK3
</TD><TD>no
</TD><TD>normal growth
</TD></TR></TBODY></TABLE><P>These results confirm the metabolic burden given by this BioBrick when gene expression is triggered: induced strains bearing a high copy plasmid containing  do not survive, while low copy ones can survive even at 1 uM HSL, which is a high induction, according to the characterization results on  (see its own page on the Registry and our <A href="https://2009.igem.org/Team:UNIPV-Pavia/Existing#BBa_F2620_-_3OC6HSL_receiver_device" class="external text" rel="nofollow">our characterization experiments on it</A>). We decided to exploit the lux promoter leakage activity for  in pSB1AK3 (high copy), while we decided to induce  in pSB4C5 (low copy) the with 1 uM of 3OC6-HSL during our fermentation experiments.
</P><P>The qualitative phenotypes of the grown cultures of  and  after fermentation in 10% glucose showed that pellets were higher in  (in both pSB1AK3 uninduced and pSB4C5 induced) than in  and in the negative control .
</P><P>
Moreover, after both inoculum steps, the grown cultures of  and  (in both pSB1AK3 uninduced and pSB4C5 induced) showed a &quot;sparkling&quot; phenotype, while  and in the negative control  did not.
</P><P><B>Quantitative characterization</B></P><P>1 - Growth of strains bearing:
</P><UL><LI>3OC6-HSL inducible system upstream of the operon in high copy plasmid () uninduced;
</LI><LI>3OC6-HSL inducible system upstream of the operon in low copy plasmid () uninduced;
</LI><LI>3OC6-HSL inducible system upstream of the operon in low copy plasmid () induced with 1 uM of 3OC6-HSL;
</LI><LI>3OC6-HSL inducible system without the operon in high copy plasmid ();
</LI><LI>a promoterless operon in high copy plasmid ().
</LI></UL><P>has been monitored in selective LB + 10% glucose starting from an inoculum in the microplate reader (figure below).
</P><P><I>Protocol:</I> 200 ul aliquots (3 replicates) were taken from a just inoculated 30 ml culture (LB + 10% glucose) of the SECOND INOCULUM of <A href="/Team:UNIPV-Pavia/Parts_Characterization#PROTOCOL.233" title="Team:UNIPV-Pavia/Parts Characterization"> PROTOCOL#3</A> and let grow in the microplate reader according to the automatic protocol described in <A href="/Team:UNIPV-Pavia/Parts_Characterization#Microplate_reader_experiments" title="Team:UNIPV-Pavia/Parts Characterization"> Microplate reader experiments</A>. The microplate was &quot;sealed&quot; with parafilm in order to create an anaerobic condition.
</P><TABLE align="center"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Mean values on three wells filled with aliquots of the indicated strains (200 ul) in selective LB + 10% glucose.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE align="center"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Mean values with 95% confidence intervals for the mean on three wells filled with aliquots of the indicated strains (200 ul) in selective LB + 10% glucose.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>2 - Ethanol fermentation capability of these strains in 10%w/v glucose-supplemented LB medium has been tested with three different fermentation protocols (see <A href="/Team:UNIPV-Pavia/Parts_Characterization#Fermentation_experiments" title="Team:UNIPV-Pavia/Parts Characterization"> Fermentation experiments</A> for a detailed description).
</P><P>The main fermentation results can be summarized with the following bar plots. Ethanol and pH have been measured after 48 hours (<A href="/Team:UNIPV-Pavia/Parts_Characterization#PROTOCOL.232" title="Team:UNIPV-Pavia/Parts Characterization"> PROTOCOL#2</A>, first two measurements) or 24 hours (<A href="/Team:UNIPV-Pavia/Parts_Characterization#PROTOCOL.233" title="Team:UNIPV-Pavia/Parts Characterization"> PROTOCOL#3</A>, other measurements).
</P><TABLE align="center"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Ethanol production after fermentation of 10% glucose.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE align="center"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">pH after fermentation of 10% glucose.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P><A href="/Team:UNIPV-Pavia/Parts_Characterization#PROTOCOL.232" title="Team:UNIPV-Pavia/Parts Characterization"> PROTOCOL#1</A> was tested only on  (in pSB1AK3 induced with 100 nM HSL) and  as a negetive control. This protocol resulted in bacterial death for  (in pSB1AK3 induced with 100 nM HSL) because of the previously explained metabolic burden when pdc and adhB gene expression was triggered at high levels in a high copy plasmid.
</P><P><I>Comments:</I></P><P>The pH results suggest that in strains bearing expressed pdc and adhB the fermentation does not give high levels of organic acids as in negative control strains which do not have pdc and adhB. It is surprising that the pH of BBa_K173003 in pSB1AK3 (pdc and adhB without any promoter) is higher than in the negative control. This could be due to a weak spurious transcription of pdc and adhB, amplified by the high copy number plasmid. This may re-direct part of pyruvate metabolism to ethanol and not to organic acids. Of course this phenomenon should be further investigated.
</P><P>The results about dynamic growth (in terms of O.D.600) are in accordance with the pH results, confirming that strains bearing BBa_K173021 in pSB1AK3 produce less organic acids than the negative controls, hopefully thanks to pdc and adhB which re-direct the metabolism of pyruvate from organic acids to ethanol, and consequently grow better (to OD600=~1). No significant difference was noticed in the growth of promoterless operon, uninduced and induced BBa_K173021 in pSB4C5, suggesting that in promoterless operon transcription could be activated in unspecific manner (i.e. without a promoter upstream). These three cultures reached OD600 values of ~0.7, while the negative control () reached only OD600=~0.4.
</P><P>Fermentation results showed that strains bearing expressed pdc and adhB in pSB1AK3 high copy plasmid can produce less ethanol than the negative controls in both <A href="/Team:UNIPV-Pavia/Parts_Characterization#PROTOCOL.232" title="Team:UNIPV-Pavia/Parts Characterization"> PROTOCOL#2</A> and <A href="/Team:UNIPV-Pavia/Parts_Characterization#PROTOCOL.233" title="Team:UNIPV-Pavia/Parts Characterization"> PROTOCOL#3</A>. Our hypothesis is that in our working conditions adhB converts part of the fermented ethanol back to acetaldehyde, in fact this enzyme is able to perform both acetaldehyde conversion to ethanol and the opposite reaction.
</P><P>On the other hand, if we add 1%w/v of ethanol in the medium this strain can ferment 30 g/l of ethanol, which is about 60% of theoretical yield. This may be due to adhB behaviour, which was reported in literature to be activated by ethanol accumulation.
</P><P>Moreover, It is possible that when starting from less than 1% of ethanol in fermentation medium ethanol production could be increased, because in our experiment the bacterial culture reached ethanol concentration survival threshold (40 g/l) and so it difficult to ferment more than 30 g/l.
</P><H3><SPAN class="mw-headline" id="Conclusions">Conclusions</SPAN></H3><P>Ethanol production in this operon has been proved for  when assembled under the control of  HSL receiver device, yielding  composite part.
</P><P>Promising working conditions have been found for it:
</P><UL><LI>BBa_K173021 in pSB1AK3 (HC) not induced with 3OC6-HSL (exploiting  basic transcriptional activity), induced with 1%w/v of ethanol;
</LI><LI>BBa_K173021 in pSB4C5 (LC) induced with 1 uM of 3OC6-HSL.
</LI></UL><P>Increased pH and growth in ethanol producing strains are also proves that the pyruvate metabolism is re-directed from organic acid fermentation to ethanol production. Further work has to be performed to fully understand pdc and adhB optimal working conditions and to optimize the environment for a high yield ethanol production.
</P><DIV align="right"><P><A href="#top">Top</A></P></DIV><H2><SPAN class="mw-headline" id="BBa_K173007_-_aTc_inducible_device_with_J23100_promoter">BBa_K173007 - aTc inducible device with J23100 promoter</SPAN></H2><H3><SPAN class="mw-headline" id="Description_2">Description</SPAN></H3><P>This is an aTc sensing device.
</P><PRE>promoter drives the constitutive production of tetR repressor (), which inhibits tetR promoter () activity. When aTc is added to the medium, it binds tetR and inhibits it. So, the PoPS output is a function of the aTc concentration.
</PRE><P>A tight regulation is expected for this inducible system because BBa_J23100 is a strong promoter and so tetR repressor should be produced at extremely high levels.
</P><P>The data below are referred to , which is the measurement system of .
</P><H3><SPAN class="mw-headline" id="Characterization_2">Characterization</SPAN></H3><P><TABLE border="1" align="center" cellpadding="5px"><TBODY><TR><TD rowspan="2" valign="bottom" align="center"><B>aTc concentration[ng/ml]</B></TD><TD colspan="2" align="center"><B>LB</B></TD><TD colspan="2" align="center"><B>M9 supplemented</B></TD></TR><TR><TD><B>Doubling time [minutes]</B></TD><TD><B>RPU mean [range]</B></TD><TD><B>Doubling time [minutes]</B></TD><TD><B>RPU mean [range]</B></TD></TR><TR align="center"><TD>     0</TD><TD>     36 </TD><TD>   0.05 [~0 ; 0.15] </TD><TD>     59 </TD><TD>   0.03 [  0.02 ;   0.04]</TD></TR><TR align="center"><TD>    25</TD><TD>     32 </TD><TD>   0.12 [~0 ; 0.19] </TD><TD>     61 </TD><TD>   0.03 [  0.02 ;   0.04]</TD></TR><TR align="center"><TD>    50</TD><TD>     34 </TD><TD>   0.17 [0.05; 0.25] </TD><TD>     57 </TD><TD>   0.04 [  0.03 ;   0.05]</TD></TR><TR align="center"><TD>    75</TD><TD>     43 </TD><TD>   0.15 [~0; 0.23] </TD><TD>     65 </TD><TD>   0.03 [  0.02 ;   0.06]</TD></TR><TR align="center"><TD>   100</TD><TD>     40 </TD><TD>   0.21 [0.19; 0.23] </TD><TD>     62 </TD><TD>   0.03 [  0.03 ;   0.05]</TD></TR><TR align="center"><TD>   200</TD><TD>     40 </TD><TD>   0.25 [0.20; 0.28] </TD><TD>     70 </TD><TD>   0.04 [  0.03 ;   0.06]</TD></TR><TR align="center"><TD>   300</TD><TD>     43 </TD><TD>   0.20 [0.18; 0.21] </TD><TD>     67 </TD><TD>   0.05 [  0.04 ;   0.08]</TD></TR></TBODY></TABLE></P><TABLE align="center"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"> Growth curves for BBa_K173009 in M9</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"> Growth curves for BBa_K173009 in LB</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">(dGFP/dt)/OD in LB</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">(dGFP/dt)/OD in M9</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"> Induction curve of BBa_K173009 in M9</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"> Induction curve of BBa_K173009 in LB</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Conclusions_2">Conclusions</SPAN></H3><P>We demonstrated that this part works as expected, sensing the aTc concentration provided in the culture medium. The transfer function of this sensor has been characterized in standard units (RPUs) in two different growth media (LB and M9 supplemented with glycerol), as well as the metabolic burden (in terms of doubling time) which affects E. coli bearing this part.
</P><P>On the other hand, we did not expect to have a higher GFP synthesis rate per cell after the exponential growth phase than in the exponential phase itself (as reported in the 3rd plot for both LB and M9).
</P><P>This part shows to have a very low leakage rate (about 0.025 RPU) but also a very low transcription rate for high concentrations of aTc. So, it can be used for tight regulation, in those systems who need very low leakage rates (no gene expressed in absence of inducer) but not for massive protein production.
</P><P>Unfortunately, the estimated transfer function is very noisy (probably because the measured fluorescence values are very close to blank values), and further experiments should be performed in order to validate these induction curves.
</P><DIV align="right"><P><A href="#top">Top</A></P></DIV><H2><SPAN class="mw-headline" id="BBa_K173011_-_aTc_inducible_device_with_J23118_promoter">BBa_K173011 - aTc inducible device with J23118 promoter</SPAN></H2><H3><SPAN class="mw-headline" id="Description_3">Description</SPAN></H3><P>This is an aTc sensing device.
</P><PRE>promoter drives the constitutive production of tetR repressor (), which inhibits tetR promoter () activity. When aTc is added to the medium, it binds tetR and inhibits it. So, the PoPS output is a function of the aTc concentration.
</PRE><P>A less tight regulation is expected for this inducible system than in  because BBa_J23118 promoter is weaker than BBa_J23100 and so tetR repressor shold be produced at lower levels than in the other sensor.
</P><P>The data below are referred to , which is the measurement system of .
</P><H3><SPAN class="mw-headline" id="Characterization_3">Characterization</SPAN></H3><P><TABLE border="1" align="center" cellpadding="5px"><TBODY><TR><TD rowspan="2" valign="bottom" align="center"><B>aTc concentration[ng/ml]</B></TD><TD colspan="2" align="center"><B>LB</B></TD><TD colspan="2" align="center"><B>M9 supplemented</B></TD></TR><TR><TD><B>Doubling time [minutes]</B></TD><TD><B>RPU</B></TD><TD><B>Doubling time [minutes]</B></TD><TD><B>RPU</B></TD></TR><TR align="center"><TD>      0 </TD><TD>     36 </TD><TD>   0.51 [0.43; 0.73] </TD><TD>     78 </TD><TD>   0.96 [0.80; 1.23]</TD></TR><TR align="center"><TD>     25 </TD><TD>     33 </TD><TD>   1.36 [1.28; 1.44] </TD><TD>     82 </TD><TD>   1.66 [1.38; 2.11]</TD></TR><TR align="center"><TD>     50 </TD><TD>     40 </TD><TD>   1.39 [1.30; 1.48] </TD><TD>     81 </TD><TD>   1.85 [1.61; 2.27]</TD></TR><TR align="center"><TD>     75 </TD><TD>     43 </TD><TD>   1.25 [1.17; 1.31] </TD><TD>     88 </TD><TD>   1.75 [1.51; 2.15]</TD></TR><TR align="center"><TD>    100 </TD><TD>     43 </TD><TD>   1.40 [1.28; 1.48] </TD><TD>     90 </TD><TD>   1.89 [1.68; 2.34]</TD></TR><TR align="center"><TD>    200 </TD><TD>     45 </TD><TD>   1.37 [  1.31 ;   1.41] </TD><TD>     92 </TD><TD>   1.75 [1.51; 2.09]</TD></TR><TR align="center"><TD>    300 </TD><TD>     48 </TD><TD>   1.45 [  1.36 ;   1.55] </TD><TD>     94 </TD><TD>   1.73 [1.53; 2.07]</TD></TR></TBODY></TABLE></P><TABLE align="center"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"> Growth curves for BBa_K173026 in M9</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"> Growth curves for BBa_K173026 in LB</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">(dGFP/dt)/OD in LB and M9</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">(dGFP/dt)/OD in LB and M9</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"> Induction curve of BBa_K173026 in M9</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"> Induction curve of BBa_K173026 in LB</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Conclusions_3">Conclusions</SPAN></H3><P>We demonstrated that this part works as expected because GFP is produced as an increasing function of the aTc concentration provided in the culture medium. The transfer function of this sensor has been characterized in standard units (RPUs) in two different growth media (LB and M9 supplemented with glycerol), as well as the metabolic burden (in terms of doubling time) which affects E. coli bearing this part.
</P><P>On the other hand, as for BBa_K173007, we did not expect to have a higher GFP synthesis rate per cell after the exponential growth phase than in the exponential phase itself (as reported in the 3rd plot).
</P><P>This part shows a high level of leakage (about 0.5 RPU in LB and 1 RPU in M9) but also high levels of induction for high concentrations of aTc (about 1.45 RPU in LB and 1.7 RPU in M9). For this reason, it can be used in those cases where an important response in terms of gene expression is required in presence of an inducer, but is not suitabe if the absence of expression has to be off when inducer is absent. Its induction range is 0.95 RPU in LB and 0.7 RPU in M9.
</P><P>The switch point of this device is between 0 ng/ml and 25 ng/ml of aTc in both LB and M9.
</P><DIV align="right"><P><A href="#top">Top</A></P></DIV><H2><SPAN class="mw-headline" id="BBa_K173010_-_lactose.2FIPTG_inducible_device_with_J23118_promoter">BBa_K173010 - lactose/IPTG inducible device with J23118 promoter</SPAN></H2><H3><SPAN class="mw-headline" id="Description_4">Description</SPAN></H3><P>This should work as a lactose/IPTG sensor.
</P><PRE>promoter drives the constitutive production of lacI repressor (), which inhibits lac promoter () activity. When lactose or IPTG is added to the medium, it binds lacI and inhibits it. So, the PoPS output is a function of lactose/IPTG concentration.
</PRE><P>Thanks to the hybrid lac promoter (), designed taking the Plambda promoter () and substituting its cI () binding sites with two lacI binding sites, the behaviour of this device is not a function of glucose concentration because the wild type CAP binding sites are not present in this artificial lac promoter.
</P><P>The test performed to characterize this device has been done with , which is the measurement system of  itself.
</P><H3><SPAN class="mw-headline" id="Characterization_4">Characterization</SPAN></H3><UL><LI>We inoculated 8 ul of  glycerol stock in 5 ml of LB + Amp and incubated it overnight (37°C, 220 rpm).
</LI><LI>In the morning we diluted 1:100 the culture in two falcon tubes and incubated them under the same conditions as before.
</LI><LI>After about 6 hours we added IPTG in one of the two cultures (final concentration = 1 mM).
</LI><LI>Fluorescence and O.D.600 were measured in the microplate reader after additional 4 hours.
</LI></UL><P>Results: both cultures had a comparable O.D.600, but also had the same RFU values, demonstrating that the induced culture did not produce any GFP.
</P><H3><SPAN class="mw-headline" id="Conclusions_4">Conclusions</SPAN></H3><P>We did not perform any standard measurement on this device because preliminary tests showed that GFP levels of induced and non induced cultures were the same and were equals to  negative control.
</P><P>Unfortunately, we did not check if the sequence of , so we do not know if it was actually correct. Another possibility is that lacI is produced at so high levels that 2 mM of IPTG is not sufficient to induce the lac promoter.
</P><P>Further tests should be done for this system.
</P><DIV align="right"><P><A href="#top">Top</A></P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:UNIPV-Pavia/New_Parts">http://2009.igem.org/Team:UNIPV-Pavia/New_Parts</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UNIPV-Pavia/New_Parts" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UNIPV-Pavia/New_Parts" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UNIPV-Pavia/New_Parts&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UNIPV-Pavia/New_Parts&amp;oldid=159079" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>