"
Page
Discussion
View source
History
teams
Log in
 
Team:UNIPV-Pavia/Existing
From 2009.igem.org
Home
The Project
Motivation
Solution
Results & Conclusions
References
Materials & Methods
Measurements
Instruments
Protocols
Software Developed
Parts
Submitted
Characterization
The Team
Notebook
Week-book
Freezer Management
Biological Safety
Pavia
History
University
Gallery
Sponsors
Gallery
Team Locations
Existing parts: registry
Our new parts
Re-built existing parts (BBa_our part code/BBa_existing part code)
Existing parts from the Registry:
BBa_J23100, BBa_J23101, BBa_J23118 - constitutive promoter family members
BBa_F2620 - 3OC6HSL receiver device
BBa_K116001 - nhaA promoter
BBa_K112808 - Enterobacteria phage T4 Lysis Device
BBa_R0011 - Plac hybrid promoter
Existing parts: sequence debugging
BBa_J23100, BBa_J23101, BBa_J23118 - constitutive promoter family members
Description
These three promoters are from the Anderson Promoter Collection, which is a library of constitutive sigma70 bacterial promoters. The strength of each promoter of the library has already been estimated in saturation growth phase cultures in LB, but here we provide the characterization of BBa_J23100 and BBa_J23118 in standard units (RPUs) in LB medium, in order to add experience and data for these BioBricks. BBa_J23101 is the reference standard promoter, so it has RPU=1 for definition.
The data shown below are referred to ,  and  that are the measurement parts of ,  and  respectively.
Characterization
Part
LB
Doubling time [minutes]
RPU
BBa_J23100
(in BBa_J61002 plasmid)
36
not computed
BBa_J23101
(in BBa_J61002 plasmid)
37
not computed
BBa_J23118
(in BBa_J61002 plasmid)
36
not computed
BBa_K173000
36
2.04 [1.99; 2.08]
BBa_K173001
36
1 (reference standard)
BBa_K173002
35
0.69 [0.64; 0.73]
Growth curves for 3OC6HSL in M9
Growth curves for 3OC6HSL in LB
Conclusions
RPU estimation of these promoters was not present in the Registry and even the doubling time of these parts was not documented. We added these data in the pages of ,  and  characterized parts, hoping that they can be useful for promoter comparison in standard units.
The estimated strength in RPU of BBa_J23100 and BBa_J23118 are in accordance with the ranking provided in , although this ranking has been estimated in saturation growth phase in TG1 strain (see Anderson Promoter Collection Registry page for details).
Top
BBa_F2620 - 3OC6HSL receiver device
Description
This device gives PoPS as output and can be induced with 3OC6-HSL autoinducer molecule: it binds luxR protein (encoded by ), which is constitutively expressed by tetR promoter (). LuxR-HSL complex can work as a transcriptional activator for lux promoter ().
Several studies have been performed on this BioBrick. Here we provide the experimental characterization we performed during this summer. The tests have been performed through  measurement system, which has a GFP protein generator downstream.
Characterization
3OC6-HSL
concentration
LB
M9 supplemented
Doubling time [minutes]
RPU mean [range]
Doubling time [minutes]
RPU mean [range]
0nM
36
~0
59
~0
0.1nM
37
~0
65
0.03 [~0; 0.19]
1nM
40
0.86 [  0.82 0.92]
65
1.19 [0.53; 3.08]
10nM
43
4.44 [3.81; 5.44]
73
4.29 [2.63; 7.83]
100nM
43
4.53 [3.88; 5.18]
76
5.23 [3.21; 9.28]
1uM
39
3.53 [3.35; 3.75]
73
4.99 [3.14; 8.45]
10uM
41
4.04 [3.56; 4.57]
62
5.40 [3.39; 9.56]
Growth curves for 3OC6HSL in M9
Growth curves for 3OC6HSL in LB
(dGFP/dt)/O.D. in LB and M9
Characterization of part in M9
Characterization of part in LB
Conclusions
The induction curve of the receiver device, reported in  page (M9 supplemented medium), was represented in PoPS units, while ours is reported in RPUs and has been obtained through a very similar protocol (see Growth conditions section). Anyway, the experiments we performed in M9 supplemented medium confirmed the induction curve shape of this device, with a switch point between 1nM and 10nM.
We also estimated the transfer function of this device in LB medium, for which no data were reported in the Registry from previous characterization. More experiments sholud be perfomed on the behaviour of  this device in M9 because the one performed show a great variation in RPU range.
Top
BBa_K116001 - nhaA promoter
from iGEM 2008 NYMU-Taipei.
We received the BioBrick measurement system (which has a GFP protein generator downstream) of  from iGEM HQ in September (it is called ). The bacterial strain that contained the plasmid was NEB 10-beta, so we decided to transform it into E. coli TOP10.
We wanted to perform some experiments to better understand how it works and if can be successfully used.
We performed several experiments with different LB medium and we got almost the same results. We used:
LBK (NaCl 0M) (pH 5.5 - 6.6 - 7.5 - 8.5)
LB NaCl 70mM (pH 5.5 - 6.6 - 7.5 - 8.5)
LB NaCl 171mM (pH 5.5 - 6.6 - 7.5 - 8.5)
LB NaCl 250mM (pH 5.5 - 6.6 - 7.5 - 8.5)
LB NaCl 600mM (pH 10 - 11.2)
Here we show just two experiments to explain our work. A complete report can be downloaded from this link .
Experiment Na+ 0M
Motivation
The working principle of the antiporter Na+/H+ channel described in [Rachel Karpel et al., Etana Padan et al., N. Dover et al.] makes the nhaA promoter a Na+ sensor and only under certain conditions (presence of Na+) a pH sensor.
Methods
We prepared LBK (potassium - 87mM - instead of sodium) and adjusted pH using KOH and HCl to values 5.5, 6.6, 7.5 and 8.5.
We inoculated 8ul of Invitrogen TOP10 containing  into 4ml of LB + Amp and incubated overnight at 37°C, 220 rpm. We did the same for TOP10 with  and  inside.
Next morning we put 50ul from each of the three falcon tubes into 5ml of LBK pH 6.6 and incubated again for about four hours and a half at 37°C, 220 rpm.
We measured the final O.D. with TECAN F200 and diluted each genetic circuit into four falcons with LBK at different pH (5.5 - 6.6 - 7.5 - 8.5) in order to obtain a same O.D. equal to 0,02 (12 falcons overall).
Then we performed a 6 hours' experiment with measures of absorbance and fluorescence every 5 minutes with TECAN F200.
Each value shown is the mean of three measurements and cultures were shaked for 15 seconds every five minutes.
Results
Absorbance
Fluorescence
pH 5,5: Fluorescence
pH 6,6: Fluorescence
pH 7,5: Fluorescence
pH 8,5: Fluorescence
Comments
As you can see from graphs  didn't produce any GFP. After a short transient in which we have some noise you can see that the GFP production rate goes to zero as well as the negative control (). Positive control () has significantly higher production rate. So we can consider it a Na+ sensor and only secondarily a pH sensor.
Experiment Na+ 250mM
Motivation
We’ll try again to make E. coli producing GFP at the variation of pH in presence of Na+.
Methods
We prepared falcons of LB NaCl 250mM and adjusted pH using KOH and HCl to values 5.5, 6.6, 7.5 and 8.5.
We inoculated 8ul of Invitrogen TOP10 containing  into 4ml of LB + Amp and incubated overnight at 37°C, 220 rpm. We did the same for TOP10 with  and  inside.
Next morning we put 50ul from each of the three falcon into 5ml of LB NaCl 250 mM pH 6.6 and incubated again for five hours and at 37°C, 220 rpm.
We measured the final O.D. with TECAN F200 and diluted each genetic circuit into four falcons with LB NaCl 250mM at different pH (5.5 - 6.6 - 7.5 - 8.5) in order to obtain a same O.D. equal to 0,02 (12 falcons overall).
Then we performed an experiment of 21 hours duration with measures of absorbance and fluorescence every 5 minutes with TECAN F200. Each value is the mean of three measurements and cultures were shaked for 15 seconds every five minutes.
Results
Absorbance
Fluorescence
pH 5,5: Fluorescence
pH 6,6: Fluorescence
pH 7,5: Fluorescence
pH 8,5: Fluorescence
Comments
We didn't expect this. Again  didn't produce any GFP as well as negative control (): after the transient due to noise their GFP production rate goes to zero while positive control () has a significantly higher one.  After looking better for a motivation in some articles ([Rachel Karpel et al.]) we think this could be because of the E. coli strain: we use TOP10 while a special strain (delta-nhaA) without some membrane proteins that regulate E. coli homeostasis is used in other experiments.
Final considerations
In our opinion this sensor (primarily sodium sensor and secondarily pH sensor) needs very particular conditions to work (first of all a specific bacterial strain) we couldn’t reproduce, so we consider it almost unusable.
Top
BBa_K112808 - Enterobacteria phage T4 Lysis Device
Description
Actually a lysis device such as  is not part of "Ethanol? Whey not!" project. Anyway, we found very interesting to characterize the activity of this BioBrick and we decided to do it by building up a specific measurement system: , in which we can modulate the expression of the lysis genes through  inducible device. During this summer we have also characterized  in standard units, so the characterization of the lysis device is promoter-independent.
receiver device controls the expression of holin () and lysozyme () genes from T4 bacteriophage. Holins form pores in the inner membrane of bacteria, while lysozymes degrade the peptidoglycan layer, passing through these pores and performing the lysis of the bacterial cell. This device also contains a constitutive expression of an antiholin (), which inhibits holin multimer formation.
Lysis can be induced with 3OC6-HSL autoinducer molecule: it binds luxR protein (encoded by ), which is constitutively expressed by tetR promoter (). LuxR-HSL complex can work as a transcriptional activator for lux promoter ().
Characterization
We performed four experiments on our measurement system () in order to characterize  activity.
All the experiments have been performed in the same conditions in the microplate reader and involved the induction of  with 3OC6-HSL 10 nM (which corresponds to a RPU value of 4.44 [3.81 ; 5.44] in LB for BBa_F2620) and 100 nM (which corresponds to a RPU value of 4.53 [3.88 ; 5.18] in LB for BBa_F2620). The basic activity of BBa_F2620 was ~0 RPU.
O.D.600 has been used to measure the amount of bacteria in the wells.
Experiment 1 results
We tested  and  as a negative control. These two parts were tested according to [ Microplate reader experiments ] section, except for  first inoculum, which was performed from single colonies in a streaked LB agar plate. Three colonies were tested separately for this part.
O.D.600 of two replicates of three colonies of  (uninduced or induced with 100 nM 3OC6-HSL) and of .
The result showed that BBa_K173010 had comparable growth curves in presence of 3OC6HSL (dashed blue lines) or not (blue lines). The cultures containing the lysis device not induced (dashed red lines) shows a slow growth compared to BBa_K173010, maybe because of leakage of the inducible system causing a weak lysis or because the device decrease bacteria’s fitness. The culture containing the induced lysis device (red lines) shows a rapid decrease of O.D. after 15 minutes from induction, followed by a O.D. increasing. Different colonies had comparable dynamic growth.
This experiment has been repeated two times (data have not been reported): the first one (Experiment 2) lysis failed for unknown reasons, while in the second one (Experiment 3) it worked as in Experiment 1.
We deduced that the slow O.D. increase in lysed cultures (as well as the apparent genetic instability of the inducible device, which was unresponsive to 3OC6-HSL in Experiment 2) could be due to a positive selection of mutant cells unresponsive to 3OC6-HSL (or defective in the lysis device) or to the shortage of free 3OC6-HSL. To test this hypotheses, we performed a fourth, independent experiment, using  inducible system,  as a negative control and a culture containing only  part (not functional because the operon lacks a promoter). The test has been performed with two different inducer concentrations (100 nM and 10nM) and following the standard protocol of [ Microplate reader experiments ]. In this experiment (Experiment 4), lysis was induced at t=0 and, in previously induced cultures, also after 4 hours of growth to test if they were still responsive to 3OC6-HSL.
O.D.600 of  (uninduced, induced with 10 nM 3OC6-HSL or 100 nM), of  and of .
The growth curves of the second experiment are in agreement with the result of Experiment 1. Moreover, our hypothesis of mutant generation for  induced culture is confirmed by the second induction: if the increase in O.D. was perhaps due to a shortage of 3OC6-HSL in the growing population, the second induction would have caused a new decline in O.D., which is not our case. This hypotesis is also confirmed by the growth curves of induced (at t=0)  cultures, which show a decrease (from 15 min to 2 hours) followed by an increasing growth, comparable with the negative controls growth and faster than  non induced cultures growth.
This new experiment also shows that both induction with 10nM or 100nM of 3OC6-HSL cause the same decrease in O.D. value and that K112808 part alone (green lines) could grow like the negative control, demonstrating that the slow growth of  was actually due to leakage activity of .
Conclusions
We built up a new measurement system to characterize a particularly interesting lysis device, already present in the Registry.
The reported results show that the lysis device works as expected when gene expression of the lysis genes is triggered. The performed experiments showed a genetic instability of this device (which was expected because a complete expression system of lysis genes is introduced in E. coli): in one of four experiments lysis could not be induced (Experiment 2) and in all the other experiments after lysis induction in  cultures showed an O.D.600 decrease, followed by a fast increase, probably due to mutations in lysis expression system.
Top
BBa_R0011 - Plac hybrid promoter
Description
The hybrid lac promoter (BBa_R0011) has been designed taking the Plambda promoter (BBa_R0051) and substituting its cI (BBa_C0051) binding sites with two lacI binding sites.
This promoter can be repressed by lacI (BBa_C0012), which can be repressed by lactose or IPTG, providing a lactose/IPTG inducible system. Differently from wild type lac promoter, this part does not have any CAP binding sites, so its behaviour is glucose-independent.
Even if lacI is not expressed in this BioBrick, strains bearing a genomic copy of lacI can repress this promoter, which acts as a glucose-independent lactose/IPTG sensor. In the other strains BBa_R0011 acts as a constitutive promoter.
Here we provide the characterization of this promoter in E. coli TOP10, which has a lacI genomic copy, constitutively expressed in a weak manner.
The data below are referred to , which is the measurement system of .
Characterization
IPTG concentration
LB
Doubling time [minutes]
RPU
0uM
42
1.52 [1.46; 1.59]
10uM
41
1.83 [1.73; 1.90]
50uM
43
1.78 [1.64 ; 1.92]
100uM
46
1.73 [1.52; 1.95]
500uM
43
1.61 [1.46 ; 1.81]
1mM
47
1.78 [1.53; 2.01]
2mM
44
1.47 [0.93; 2.05]
Growth curves for BBa_K173025 in LB
(dGFP/dt)/O.D. in LB and LB
Induction curve of BBa_K173025 in LB
Conclusions
As TOP10 strains contains an expressed lacI in its genome, we wanted to know if it was enough to repress Plac activity in a high copy number plasmid in order to use it as a lactose/IPTG sensor. It is evident from induction curve that the behaviour of this part is almost independent from IPTG concentration, probably because lacI production by genomic DNA is too low. So our characterization has shown that  can be used as a strong constitutive promoter, with mean RPU higher than 1.5.  Future work should be dedicated to the characterization of this BioBrick in low copy number plasmids and in strains containing lacIq mutation in order to buil-up new IPTG/lactose sensors.
Top
Retrieved from " http://2009.igem.org/Team:UNIPV-Pavia/Existing "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
