<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UNIPV-Pavia_Existing"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:UNIPV-Pavia/Existing">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UNIPV-Pavia/Existing&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Existing&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Existing&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UNIPV-Pavia/Existing" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UNIPV-Pavia/Existing</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><TABLE width="100%" border="0"><TBODY><TR><TD valign="top" width="19%" rowspan="2"><P><FONT face="eras demi itc"><UL id="MenuBar1" class="MenuBarVertical"><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia">Home</A></LI><LI><A class="MenuBarItemSubmenu">The Project</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Project/Motivation">Motivation</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Project/Solution">Solution</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Project/Results">Results &amp; Conclusions</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Project/References">References</A></LI></UL><LI><A class="MenuBarItemSubmenu">Materials &amp; Methods </A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Methods_Materials/Measurement">Measurements</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Methods_Materials/Instruments">Instruments</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Methods_Materials/Protocols">Protocols</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Software">Software Developed</A></LI><LI><A class="MenuBarItemSubmenu">Parts</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Parts">Submitted</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Parts_Characterization">Characterization</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Team">The Team</A></LI><LI><A class="MenuBarItemSubmenu">Notebook</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Notebook">Week-book</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Freezer">Freezer Management</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Safety">Biological Safety</A></LI><LI><A class="MenuBarItemSubmenu" href="https://2009.igem.org/Team:UNIPV-Pavia/Pavia">Pavia</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Pavia#History">History</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Pavia#University">University</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Pavia#Gallery">Gallery</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Sponsors">Sponsors</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Gallery">Gallery</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Team_Locations">Team Locations</A></LI></FONT></P></TD><TD align="right"><FONT face="Pristina" size="50"><B>Existing parts: registry</B></FONT></TD></TR><TR><TD><P><A href="/Team:UNIPV-Pavia/New_Parts" title="Team:UNIPV-Pavia/New Parts"><B>Our new parts</B></A></P><P><A href="/Team:UNIPV-Pavia/Rebuilt" title="Team:UNIPV-Pavia/Rebuilt"><B>Re-built existing parts (BBa_our part code/BBa_existing part code)</B></A></P><P><STRONG class="selflink"><B>Existing parts from the Registry</B>:</STRONG></P><UL><LI><A href="/Team:UNIPV-Pavia/Existing#BBa_J23100.2C_BBa_J23101.2C_BBa_J23118_-_constitutive_promoter_family_members" title="Team:UNIPV-Pavia/Existing"> BBa_J23100, BBa_J23101, BBa_J23118 - constitutive promoter family members</A></LI><LI><A href="/Team:UNIPV-Pavia/Existing#BBa_F2620_-_3OC6HSL_receiver_device" title="Team:UNIPV-Pavia/Existing"> BBa_F2620 - 3OC6HSL receiver device</A></LI><LI><A href="/Team:UNIPV-Pavia/Existing#BBa_K116001_-_nhaA_promoter" title="Team:UNIPV-Pavia/Existing"> BBa_K116001 - nhaA promoter</A></LI><LI><A href="/Team:UNIPV-Pavia/Existing#BBa_K112808_-_Enterobacteria_phage_T4_Lysis_Device" title="Team:UNIPV-Pavia/Existing"> BBa_K112808 - Enterobacteria phage T4 Lysis Device</A></LI><LI><A href="/Team:UNIPV-Pavia/Existing#BBa_R0011_-_Plac_hybrid_promoter" title="Team:UNIPV-Pavia/Existing"> BBa_R0011 - Plac hybrid promoter</A></LI></UL><P><A href="/Team:UNIPV-Pavia/Debug" title="Team:UNIPV-Pavia/Debug"><B>Existing parts: sequence debugging</B></A></P><H2><SPAN class="mw-headline" id="BBa_J23100.2C_BBa_J23101.2C_BBa_J23118_-_constitutive_promoter_family_members">BBa_J23100, BBa_J23101, BBa_J23118 - constitutive promoter family members</SPAN></H2><H3><SPAN class="mw-headline" id="Description">Description</SPAN></H3><P>These three promoters are from the Anderson Promoter Collection, which is a library of constitutive sigma70 bacterial promoters. The strength of each promoter of the library has already been estimated in saturation growth phase cultures in LB, but here we provide the characterization of BBa_J23100 and BBa_J23118 in standard units (RPUs) in LB medium, in order to add experience and data for these BioBricks. BBa_J23101 is the reference standard promoter, so it has RPU=1 for definition.
</P><P>The data shown below are referred to ,  and  that are the measurement parts of ,  and  respectively.
</P><H3><SPAN class="mw-headline" id="Characterization">Characterization</SPAN></H3></TD></TR><TR><TD rowspan="2" valign="bottom" align="center"><B>Part</B></TD><TD colspan="2" align="center"><B>LB</B></TD></TR><TR><TD><B>Doubling time [minutes]</B></TD><TD><B>RPU</B></TD></TR><TR align="center"><TD>BBa_J23100(in BBa_J61002 plasmid)</TD><TD>36</TD><TD>not computed</TD></TR><TR align="center"><TD>BBa_J23101(in BBa_J61002 plasmid)</TD><TD>37</TD><TD>not computed</TD></TR><TR align="center"><TD>BBa_J23118(in BBa_J61002 plasmid)</TD><TD>36</TD><TD>not computed</TD></TR><TR align="center"><TD>BBa_K173000</TD><TD>36</TD><TD> 2.04 [1.99; 2.08]
</TD></TR><TR align="center"><TD>BBa_K173001</TD><TD>36</TD><TD>1 (reference standard)</TD></TR><TR align="center"><TD>BBa_K173002</TD><TD>35</TD><TD> 0.69 [0.64; 0.73] 
</TD></TR></TBODY></TABLE><TABLE align="center"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"> Growth curves for 3OC6HSL in M9</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"> Growth curves for 3OC6HSL in LB</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Conclusions">Conclusions</SPAN></H3><P>RPU estimation of these promoters was not present in the Registry and even the doubling time of these parts was not documented. We added these data in the pages of ,  and  characterized parts, hoping that they can be useful for promoter comparison in standard units.
</P><P>The estimated strength in RPU of BBa_J23100 and BBa_J23118 are in accordance with the ranking provided in , although this ranking has been estimated in saturation growth phase in TG1 strain (see <A href="http://partsregistry.org/Promoters/Catalog/Anderson" class="external text" rel="nofollow">Anderson Promoter Collection Registry page</A> for details).
</P><DIV align="right"><P><A href="#top">Top</A></P></DIV><H2><SPAN class="mw-headline" id="BBa_F2620_-_3OC6HSL_receiver_device">BBa_F2620 - 3OC6HSL receiver device</SPAN></H2><H3><SPAN class="mw-headline" id="Description_2">Description</SPAN></H3><P>This device gives PoPS as output and can be induced with 3OC6-HSL autoinducer molecule: it binds luxR protein (encoded by ), which is constitutively expressed by tetR promoter (). LuxR-HSL complex can work as a transcriptional activator for lux promoter ().
</P><P>Several studies have been performed on this BioBrick. Here we provide the experimental characterization we performed during this summer. The tests have been performed through  measurement system, which has a GFP protein generator downstream.
</P><H3><SPAN class="mw-headline" id="Characterization_2">Characterization</SPAN></H3><P><TABLE border="1" align="center" cellpadding="5px"><TBODY><TR><TD rowspan="2" valign="bottom" align="center"><B>3OC6-HSLconcentration</B></TD><TD colspan="2" align="center"><B>LB</B></TD><TD colspan="2" align="center"><B>M9 supplemented</B></TD></TR><TR><TD><B>Doubling time [minutes]</B></TD><TD><B>RPU mean [range]</B></TD><TD><B>Doubling time [minutes]</B></TD><TD><B>RPU mean [range]</B></TD></TR><TR align="center"><TD>0nM</TD><TD>  36 </TD><TD>  ~0 </TD><TD>  59 </TD><TD>  ~0</TD></TR><TR align="center"><TD>0.1nM</TD><TD>  37 </TD><TD>  ~0 </TD><TD>  65 </TD><TD>   0.03 [~0; 0.19]</TD></TR><TR align="center"><TD>1nM</TD><TD>  40 </TD><TD>   0.86 [  0.82 0.92] </TD><TD>  65 </TD><TD>   1.19 [0.53; 3.08]</TD></TR><TR align="center"><TD>10nM</TD><TD>  43 </TD><TD>   4.44 [3.81; 5.44] </TD><TD>  73 </TD><TD>   4.29 [2.63; 7.83]</TD></TR><TR align="center"><TD>100nM</TD><TD>  43 </TD><TD>   4.53 [3.88; 5.18] </TD><TD>  76 </TD><TD>   5.23 [3.21; 9.28]</TD></TR><TR align="center"><TD>1uM</TD><TD>  39 </TD><TD>   3.53 [3.35; 3.75] </TD><TD>  73 </TD><TD>   4.99 [3.14; 8.45]</TD></TR><TR align="center"><TD>10uM</TD><TD>  41 </TD><TD>   4.04 [3.56; 4.57] </TD><TD>  62 </TD><TD>   5.40 [3.39; 9.56]</TD></TR></TBODY></TABLE></P><TABLE align="center"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"> Growth curves for 3OC6HSL in M9</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"> Growth curves for 3OC6HSL in LB</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">(dGFP/dt)/O.D. in LB and M9</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"> Characterization of part in M9</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"> Characterization of part in LB</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Conclusions_2">Conclusions</SPAN></H3><P>The induction curve of the receiver device, reported in  page (M9 supplemented medium), was represented in PoPS units, while ours is reported in RPUs and has been obtained through a very similar protocol (see Growth conditions section). Anyway, the experiments we performed in M9 supplemented medium confirmed the induction curve shape of this device, with a switch point between 1nM and 10nM.
</P><P>We also estimated the transfer function of this device in LB medium, for which no data were reported in the Registry from previous characterization.
More experiments sholud be perfomed on the behaviour of  this device in M9 because the one performed show a great variation in RPU range.
</P><DIV align="right"><P><A href="#top">Top</A></P></DIV><H2><SPAN class="mw-headline" id="BBa_K116001_-_nhaA_promoter"> BBa_K116001 - nhaA promoter </SPAN></H2><PRE>from iGEM 2008 NYMU-Taipei.
</PRE><P>We received the BioBrick measurement system (which has a GFP protein generator downstream) of  from iGEM HQ in September (it is called ). The bacterial strain that contained the plasmid was NEB 10-beta, so we decided to transform it into E. coli TOP10.
</P><P>We wanted to perform some experiments to better understand how it works and if can be successfully used.
</P><P>We performed several experiments with different LB medium and we got almost the same results.
We used:
</P><UL><LI>LBK (NaCl 0M) (pH 5.5 - 6.6 - 7.5 - 8.5)
</LI><LI>LB NaCl 70mM (pH 5.5 - 6.6 - 7.5 - 8.5)
</LI><LI>LB NaCl 171mM (pH 5.5 - 6.6 - 7.5 - 8.5)
</LI><LI>LB NaCl 250mM (pH 5.5 - 6.6 - 7.5 - 8.5)
</LI><LI>LB NaCl 600mM (pH 10 - 11.2)
</LI></UL><P>Here we show just two experiments to explain our work. A complete report can be downloaded from <A href="http://aimed11.unipv.it/iGEM2009/TestTECANInfiniteF200/BBa_K116001 characterization.pdf" target="_blank">this link</A>.
</P><H4><SPAN class="mw-headline" id="Experiment_Na.2B_0M">Experiment Na+ 0M</SPAN></H4><H5><SPAN class="mw-headline" id="Motivation">Motivation</SPAN></H5><P>The working principle of the antiporter Na+/H+ channel described in [Rachel Karpel et al., Etana Padan et al., N. Dover et al.] makes the nhaA promoter a Na+ sensor and only under certain conditions (presence of Na+) a pH sensor.
</P><H5><SPAN class="mw-headline" id="Methods">Methods</SPAN></H5><UL><LI>We prepared LBK (potassium - 87mM - instead of sodium) and adjusted pH using KOH and HCl to values 5.5, 6.6, 7.5 and 8.5.
</LI><LI>We inoculated 8ul of Invitrogen TOP10 containing  into 4ml of LB + Amp and incubated overnight at 37Â°C, 220 rpm. We did the same for TOP10 with  and  inside.
</LI><LI>Next morning we put 50ul from each of the three falcon tubes into 5ml of LBK pH 6.6 and incubated again for about four hours and a half at 37Â°C, 220 rpm. 
</LI><LI>We measured the final O.D. with TECAN F200 and diluted each genetic circuit into four falcons with LBK at different pH (5.5 - 6.6 - 7.5 - 8.5) in order to obtain a same O.D. equal to 0,02 (12 falcons overall).
</LI><LI>Then we performed a 6 hours' experiment with measures of absorbance and fluorescence every 5 minutes with TECAN F200. 
</LI><LI>Each value shown is the mean of three measurements and cultures were shaked for 15 seconds every five minutes.
</LI></UL><H5><SPAN class="mw-headline" id="Results">Results</SPAN></H5><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"> Absorbance</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"> Fluorescence</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pH 5,5: Fluorescence</DIV></DIV></DIV></DIV></TD></TR><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pH 6,6: Fluorescence</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pH 7,5: Fluorescence</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pH 8,5: Fluorescence</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="Comments">Comments</SPAN></H5><P>As you can see from graphs  didn't produce any GFP. After a short transient in which we have some noise you can see that the GFP production rate goes to zero as well as the negative control (). Positive control () has significantly higher production rate.
So we can consider it a Na+ sensor and only secondarily a pH sensor.
</P><H4><SPAN class="mw-headline" id="Experiment_Na.2B_250mM">Experiment Na+ 250mM</SPAN></H4><H5><SPAN class="mw-headline" id="Motivation_2">Motivation</SPAN></H5><P>Weâll try again to make E. coli producing GFP at the variation of pH in presence of Na+.
</P><H5><SPAN class="mw-headline" id="Methods_2">Methods</SPAN></H5><UL><LI>We prepared falcons of LB NaCl 250mM and adjusted pH using KOH and HCl to values 5.5, 6.6, 7.5 and 8.5.
</LI><LI>We inoculated 8ul of Invitrogen TOP10 containing  into 4ml of LB + Amp and incubated overnight at 37Â°C, 220 rpm. We did the same for TOP10 with  and  inside.
</LI><LI>Next morning we put 50ul from each of the three falcon into 5ml of LB NaCl 250 mM pH 6.6 and incubated again for five hours and at 37Â°C, 220 rpm.
</LI><LI>We measured the final O.D. with TECAN F200 and diluted each genetic circuit into four falcons with LB NaCl 250mM at different pH (5.5 - 6.6 - 7.5 - 8.5) in order to obtain a same O.D. equal to 0,02 (12 falcons overall).
</LI><LI>Then we performed an experiment of 21 hours duration with measures of absorbance and fluorescence every 5 minutes with TECAN F200. Each value is the mean of three measurements and cultures were shaked for 15 seconds every five minutes.
</LI></UL><H5><SPAN class="mw-headline" id="Results_2">Results</SPAN></H5><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"> Absorbance</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"> Fluorescence</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pH 5,5: Fluorescence</DIV></DIV></DIV></DIV></TD></TR><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pH 6,6: Fluorescence</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pH 7,5: Fluorescence</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pH 8,5: Fluorescence</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="Comments_2">Comments</SPAN></H5><P>We didn't expect this. Again  didn't produce any GFP as well as negative control (): after the transient due to noise their GFP production rate goes to zero while positive control () has a significantly higher one.  After looking better for a motivation in some articles ([Rachel Karpel et al.]) we think this could be because of the E. coli strain: we use TOP10 while a special strain (delta-nhaA) without some membrane proteins that regulate E. coli homeostasis is used in other experiments.
</P><H5><SPAN class="mw-headline" id="Final_considerations">Final considerations</SPAN></H5><P>In our opinion this sensor (primarily sodium sensor and secondarily pH sensor) needs very particular conditions to work (first of all a specific bacterial strain) we couldnât reproduce, so we consider it almost unusable.
</P><DIV align="right"><P><A href="#top">Top</A></P></DIV><H2><SPAN class="mw-headline" id="BBa_K112808_-_Enterobacteria_phage_T4_Lysis_Device">BBa_K112808 - Enterobacteria phage T4 Lysis Device</SPAN></H2><H3><SPAN class="mw-headline" id="Description_3">Description</SPAN></H3><P>Actually a lysis device such as  is not part of &quot;Ethanol? Whey not!&quot; project. Anyway, we found very interesting to characterize the activity of this BioBrick and we decided to do it by building up a specific measurement system: , in which we can modulate the expression of the lysis genes through  inducible device. During this summer we have also characterized  in standard units, so the characterization of the lysis device is promoter-independent.
</P><PRE>receiver device controls the expression of holin () and lysozyme () genes from T4 bacteriophage. Holins form pores in the inner membrane of bacteria, while lysozymes degrade the peptidoglycan layer, passing through these pores and performing the lysis of the bacterial cell. This device also contains a constitutive expression of an antiholin (), which inhibits holin multimer formation.
</PRE><P>Lysis can be induced with 3OC6-HSL autoinducer molecule: it binds luxR protein (encoded by ), which is constitutively expressed by tetR promoter (). LuxR-HSL complex can work as a transcriptional activator for lux promoter ().
</P><H3><SPAN class="mw-headline" id="Characterization_3">Characterization</SPAN></H3><P>We performed four experiments on our measurement system () in order to characterize  activity.
</P><P>All the experiments have been performed in the same conditions in the microplate reader and involved the induction of  with 3OC6-HSL 10 nM (which corresponds to a RPU value of 4.44 [3.81Â ; 5.44] in LB for BBa_F2620) and 100 nM (which corresponds to a RPU value of 4.53 [3.88Â ; 5.18] in LB for BBa_F2620). The basic activity of BBa_F2620 was ~0 RPU.
</P><P>O.D.600 has been used to measure the amount of bacteria in the wells.
</P><P><B>Experiment 1 results</B></P><P>We tested  and  as a negative control. These two parts were tested according to [<A href="https://2009.igem.org/Team:UNIPV-Pavia/Parts_Characterization#Microplate_reader_experiments|" class="external text" rel="nofollow">Microplate reader experiments</A>] section, except for  first inoculum, which was performed from single colonies in a streaked LB agar plate. Three colonies were tested separately for this part.
</P><TABLE align="center"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">O.D.600 of two replicates of three colonies of  (uninduced or induced with 100 nM 3OC6-HSL) and of .</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The result showed that BBa_K173010 had comparable growth curves in presence of 3OC6HSL (dashed blue lines) or not (blue lines). The cultures containing the lysis device not induced (dashed red lines) shows a slow growth compared to BBa_K173010, maybe because of leakage of the inducible system causing a weak lysis or because the device decrease bacteriaâs fitness. The culture containing the induced lysis device (red lines) shows a rapid decrease of O.D. after 15 minutes from induction, followed by a O.D. increasing. Different colonies had comparable dynamic growth.
</P><P>
This experiment has been repeated two times (data have not been reported): the first one (<I>Experiment 2</I>) lysis failed for unknown reasons, while in the second one (<I>Experiment 3</I>) it worked as in <I>Experiment 1</I>.
</P><P>
We deduced that the slow O.D. increase in lysed cultures (as well as the apparent genetic instability of the inducible device, which was unresponsive to 3OC6-HSL in <I>Experiment 2</I>) could be due to a positive selection of mutant cells unresponsive to 3OC6-HSL (or defective in the lysis device) or to the shortage of free 3OC6-HSL. To test this hypotheses, we performed a fourth, independent experiment, using  inducible system,  as a negative control and a culture containing only  part (not functional because the operon lacks a promoter). The test has been performed with two different inducer concentrations (100 nM and 10nM) and following the standard protocol of [<A href="https://2009.igem.org/Team:UNIPV-Pavia/Parts_Characterization#Microplate_reader_experiments|" class="external text" rel="nofollow">Microplate reader experiments</A>]. In this experiment (<I>Experiment 4</I>), lysis was induced at t=0 and, in previously induced cultures, also after 4 hours of growth to test if they were still responsive to 3OC6-HSL.
</P><TABLE align="center"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">O.D.600 of  (uninduced, induced with 10 nM 3OC6-HSL or 100 nM), of  and of .</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The growth curves of the second experiment are in agreement with the result of <I>Experiment 1</I>. Moreover, our hypothesis of mutant generation for  induced culture is confirmed by the second induction: if the increase in O.D. was perhaps due to a shortage of 3OC6-HSL in the growing population, the second induction would have caused a new decline in O.D., which is not our case. This hypotesis is also confirmed by the growth curves of induced (at t=0)  cultures, which show a decrease (from 15 min to 2 hours) followed by an increasing growth, comparable with the negative controls growth and faster than  non induced cultures growth.
</P><P>This new experiment also shows that both induction with 10nM or 100nM of 3OC6-HSL cause the same decrease in O.D. value and that K112808 part alone (green lines) could grow like the negative control, demonstrating that the slow growth of  was actually due to leakage activity of .
</P><H3><SPAN class="mw-headline" id="Conclusions_3">Conclusions</SPAN></H3><P>We built up a new measurement system to characterize a particularly interesting lysis device, already present in the Registry.
</P><P>The reported results show that the lysis device works as expected when gene expression of the lysis genes is triggered. The performed experiments showed a genetic instability of this device (which was expected because a complete expression system of lysis genes is introduced in E. coli): in one of four experiments lysis could not be induced (<I>Experiment 2</I>) and in all the other experiments after lysis induction in  cultures showed an O.D.600 decrease, followed by a fast increase, probably due to mutations in lysis expression system.
</P><DIV align="right"><P><A href="#top">Top</A></P></DIV><H2><SPAN class="mw-headline" id="BBa_R0011_-_Plac_hybrid_promoter">BBa_R0011 - Plac hybrid promoter</SPAN></H2><H3><SPAN class="mw-headline" id="Description_4">Description</SPAN></H3><P>The hybrid lac promoter (BBa_R0011) has been designed taking the Plambda promoter (BBa_R0051) and substituting its cI (BBa_C0051) binding sites with two lacI binding sites.
</P><P>This promoter can be repressed by lacI (BBa_C0012), which can be repressed by lactose or IPTG, providing a lactose/IPTG inducible system. Differently from wild type lac promoter, this part does not have any CAP binding sites, so its behaviour is glucose-independent.
</P><P>Even if lacI is not expressed in this BioBrick, strains bearing a genomic copy of lacI can repress this promoter, which acts as a glucose-independent lactose/IPTG sensor. In the other strains BBa_R0011 acts as a constitutive promoter.
</P><P>Here we provide the characterization of this promoter in E. coli TOP10, which has a lacI genomic copy, constitutively expressed in a weak manner.
</P><P>The data below are referred to , which is the measurement system of .
</P><H3><SPAN class="mw-headline" id="Characterization_4">Characterization</SPAN></H3><P><TABLE border="1" align="center" cellpadding="5px"><TBODY><TR><TD rowspan="2" valign="bottom" align="center"><B>IPTG concentration</B></TD><TD colspan="2" align="center"><B>LB</B></TD></TR><TR><TD><B>Doubling time [minutes]</B></TD><TD><B>RPU</B></TD></TR><TR align="center"><TD>0uM</TD><TD>     42 </TD><TD>   1.52 [1.46; 1.59] </TD></TR><TR align="center"><TD>10uM</TD><TD>     41 </TD><TD>   1.83 [1.73; 1.90] </TD></TR><TR align="center"><TD>50uM</TD><TD>     43 </TD><TD>   1.78 [1.64 ; 1.92] </TD></TR><TR align="center"><TD>100uM</TD><TD>     46 </TD><TD>   1.73 [1.52; 1.95] </TD></TR><TR align="center"><TD>500uM</TD><TD>     43 </TD><TD>   1.61 [1.46 ; 1.81] </TD></TR><TR align="center"><TD>1mM</TD><TD>     47 </TD><TD>   1.78 [1.53; 2.01] </TD></TR><TR align="center"><TD>2mM</TD><TD>     44 </TD><TD>   1.47 [0.93; 2.05] </TD></TR></TBODY></TABLE></P><TABLE align="center"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"> Growth curves for BBa_K173025 in LB</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">(dGFP/dt)/O.D. in LB and LB</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"> Induction curve of BBa_K173025 in LB</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Conclusions_4">Conclusions</SPAN></H3><P>As TOP10 strains contains an expressed lacI in its genome, we wanted to know if it was enough to repress Plac activity in a high copy number plasmid in order to use it as a lactose/IPTG sensor. It is evident from induction curve that the behaviour of this part is almost independent from IPTG concentration, probably because lacI production by genomic DNA is too low. So our characterization has shown that  can be used as a strong constitutive promoter, with mean RPU higher than 1.5. 
Future work should be dedicated to the characterization of this BioBrick in low copy number plasmids and in strains containing lacIq mutation in order to buil-up new IPTG/lactose sensors.
</P><DIV align="right"><P><A href="#top">Top</A></P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:UNIPV-Pavia/Existing">http://2009.igem.org/Team:UNIPV-Pavia/Existing</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UNIPV-Pavia/Existing" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UNIPV-Pavia/Existing" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Existing&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Existing&amp;oldid=157468" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>