<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UNIPV-Pavia_Project_Solution"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:UNIPV-Pavia/Project/Solution">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UNIPV-Pavia/Project/Solution&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Project/Solution&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Project/Solution&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UNIPV-Pavia/Project/Solution" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UNIPV-Pavia/Project/Solution</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><TABLE width="100%"><TBODY><TR><TD valign="top" width="23%" rowspan="2"><P><FONT face="eras demi itc"><UL id="MenuBar1" class="MenuBarVertical"><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia">Home</A></LI><LI><A class="MenuBarItemSubmenu">The Project</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Project/Motivation">Motivation</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Project/Solution">Solution</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Project/Results">Results &amp; Conclusions</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Project/References">References</A></LI></UL><LI><A class="MenuBarItemSubmenu">Materials &amp; Methods </A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Methods_Materials/Measurement">Measurements</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Methods_Materials/Instruments">Instruments</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Methods_Materials/Protocols">Protocols</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Software">Software Developed</A></LI><LI><A class="MenuBarItemSubmenu">Parts</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Parts">Submitted</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Parts_Characterization">Characterization</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Team">The Team</A></LI><LI><A class="MenuBarItemSubmenu">Notebook</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Notebook">Week-book</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Freezer">Freezer Management</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Safety">Biological Safety</A></LI><LI><A class="MenuBarItemSubmenu" href="https://2009.igem.org/Team:UNIPV-Pavia/Pavia">Pavia</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Pavia#History">History</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Pavia#University">University</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Pavia#Gallery">Gallery</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Sponsors">Sponsors</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Gallery">Gallery</A></LI><LI><A href="https://2009.igem.org/Team:UNIPV-Pavia/Team_Locations">Team Locations</A></LI></FONT></P></TD><TD align="right"><FONT face="Pristina" size="50"><B>Solution</B></FONT></TD></TR><TR><TD><P>Our aim is to merge and optimize the main features of naturally occurring lactose-cleaving and ethanol-producing microorganisms to build up an engineered biological system that can metabolize lactose and can ferment it in ethanol with high yield. Here we provide an overview of the main enzymes and biochemical pathways involved in these two processes.
</P><H2><SPAN class="mw-headline" id="MICROBIAL_SOURCES_OF_GALACTOSIDASE">MICROBIAL SOURCES OF GALACTOSIDASE</SPAN></H2><P>The complete lactose to ethanol transformation pathway is summarized in the following figure.
</P></TD></TR><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Lactose to ethanol transformation pathway</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The first task of conversion is provided by β-D-Galactosidase. It can be obtained from a wide variety of sources such as microorganisms, plants and animals; however, according to its source, its properties differ markedly.
</P><TABLE align="center"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:236px;"><DIV class="thumbcaption">β-galactosidase</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The products of lactose hydrolysis may inhibit β-galactosidase action. Galactose is often a competitive inhibitor but glucose is usually ineffective, except at higher concentrations when it is usually non-competitive. Complete hydrolysis is therefore very difficult to achieve unless high concentrations of the enzyme are used. Yeast enzymes are generally inhibited by galactose (competitively) and glucose (non-competitively). Aspergillus niger enzyme is strongly inhibited by galactose; however, enzyme from A. oryzae is less subject to galactose inhibition.
</P><P><FONT size="3"><B>Fungal enzymes</B></FONT></P><P>These enzymes have a pH optimum in the acidic range 2.5–5.4, which makes them suitable for processing of acid whey and its ultrafiltration permeate. These have a relatively high temperature optimum and are typically used at temperatures up to 50°C, where they are reasonably stable. The production and purification of β-D-galactosidase from different fungal sources have been carried out using a variety of purification techniques. Fungal sources are: Aspergillus niger, Beauveria bassiana, Aspergillus fonsecaeus, Rhizomucor sp., Penicillium chrysogenum, Aspergillus carbonarius.
</P><P><FONT size="3"><B>Yeast enzymes</B></FONT></P><P>Yeast has been considered an important source of β-D-galactosidase from an industrial viewpoint. Yeast sources are: Kluyveromyces fragilis, Kluyveromyces lactis, Saccharomyces lactis, Saccharomyces anamensis, Kluyveromyces marxianus, recombinant yeast Saccharomyces cerevisiae.
</P><P><FONT size="3"><B>Bacterial enzymes</B></FONT></P><P>A large number of bacteria produce β-D-galactosidase, but relatively few bacterial species are regarded as safe sources. However, Streptococcus thermophilus and Bacillus stearothermophilus can be considered as potential bacterial sources. The former is eminently suitable as a source organism from the safety viewpoint because it is used as a starter culture in yoghurt and some cheeses.
Other bacterial sources are: Escherichia coli, Propionibacterium shermanii, Streptococcus salivarius, Pseudoalteromonas sp., S. thermophilus, Thermoanaerobacter, Bacillus coagulans, Lactobacillus delbrueckii subsp. Bulgaricus.
</P><P><FONT size="3"><B>Beta Galactosidase in <I>E. coli</I> (β-gal / gene name: LacZ)</B></FONT></P><P>It is a tetrameric hydrolytic enzyme, coded by the LacZ gene and able to catalyse the hydrolysis of a disaccharide (β-galacotisedes) into two monosaccharides: it is, then, related to the reaction that injects new glucose into glycolysis.
</P><P>The <I>E. coli</I> isoform is 464K-Da weight and 1021 amino acid long. Each monomer is composed by five domains, the third of which hosts the active site. The protein can be split into two peptides, which are unable to catalyse reaction when separated. 
</P><P>This peculiarity gives β.galactosidase a primary importance in synthetic and molecular biology, for it is used as a reporter gene, by fusing protein into the β-gal chain, disrupting the first peptide’s sequence, in a test known as “blue white screening”. A well known galactose homologue, X-Gal, turns blue when hydrolysed by this enzyme and shows whether colonies have incorporated the fusion genes or not.
</P><P>The active site is associated to Glu-537, that is the main actor of a nucleophil substitution between the carboxyl group of the side chain of Glu and the ketonic oxygen that links the two mono-saccharides. Result of this reaction is the substitution of an alcoholic group to the ketonic group and the following liberation of the other monosaccharide. Thus β-gal is the gateway for glucose into the glycolytic process.
</P><H2><SPAN class="mw-headline" id="ENGINEERING_FERMENTATION_USING_Z._MOBILIS_METABOLISM">ENGINEERING FERMENTATION USING Z. MOBILIS METABOLISM</SPAN></H2><P><FONT size="3"><B><I>Zymomonas mobilis</I></B></FONT></P><P><I>Zymomonas mobilis</I> is a Gram-negative bacteria of the soil that has several appealing features  for its use in the industrial production of ethanol. <I>Zymomonas</I> is the only known microorganism capable of oxidizing glucose anaerobically, via the Entner-Doudoroff pathway, as opposed to the classical glicolytic pathway.
</P><P>Unfortunately <I>Zymomonas</I> can only catabolize simple sugars as glucose, fructose and sucrose. Since our project relies on lactose catabolism, we decided to insert two <I>Zymomonas</I> key fermenting enzymes: the Pyruvate Decarboxylase (pdc) and the Alcohol Dehydrogenase II (adhB)  in <I>E. coli</I>, which is, instead, capable of fermenting lactose.
</P><P><I>E. coli</I> does not possess a native Pyruvate Decarboxylase: although, it uses Pyruvate Formate Lyase. This enzyme has an unbalanced consumption of NADH, and the microorganism balances it by producing acetic acid and succinic acid rather than ethanol. The insertion of <I>Zymomonas</I> pdc gene should determine the production of ethanol as only final product of the fermentation pathway.
</P><P>Although <I>E. coli</I> strains have Alcohol Dehydrogenase, the activity of this native enzyme is insufficient to achieve this high yield of ethanol, while <I>Zymomonas</I>  adhB enzyme has a higher efficiency.
</P><P><FONT size="3"><B>Pyruvate decarboxylase (PDC)</B></FONT></P><P>Pyruvate decarboxylase is a homotetrameric enzyme (EC 4.1.1.1) that catalyses the decarboxylation of pyruvic acid to acetaldehyde and carbon dioxide. It is also called 2-oxo-acid carboxylase, alpha-ketoacid carboxylase, and pyruvic decarboxylase. In anaerobic conditions, this enzyme is part of an alternative pathway for fermentation,known as Entner-Dourdoff pathway, which is characterized by the bypass of Acetyl-CoA formation, and determines an irreversible loss of a CO<SUB>2</SUB>  molecule, in order to produce acetaldehyde. Pyruvate decarboxylase starts this process by converting pyruvate into acetaldehyde and carbon dioxide. To do this, two thiamine pyrophosphate (TPP) and two magnesium ions are required as cofactors. This enzyme should not be mistaken for the unrelated enzyme pyruvate dehydrogenase, an oxidoreductase (EC 1.2.4.1), that catalyzes the oxidative decarboxylation of pyruvate to acetyl-CoA.
</P><P><FONT size="3"><B>Alcohol dehydrogenase (ADH)</B></FONT></P><P>They are a class of enzymes that catalyse the conversion between alcohols and aldehydes or ketons, in order to decrease the toxicity of alcoholic products.
Though, in the glycolytic process it is used in the opposite direction, by converting aldehydes into alcohols. All the reactions involved are reduction or oxidations, with the contribution of nicotinamide dinucleotide (NAD), as a cofactor.
</P><P>ADH2 (gene name adhB) catalyses the crucial step into ethanol conversion: acetaldehyde comes from the last steps of glycolysis and the bacteria converts it into an ethanol molecule in order to gain a NAD<SUP>+</SUP>, as a reducing  power, when in anaerobic, or microaerobial environment.
</P><P>Structurally, the alcohol dehydrogenase family is composed by multimeric proteins, about 300 amino acids long (ADH2 of <I>Z.mobilis</I> is by 383 amino acids and 40,145 KDa weight). The ratio between isoforms of each monomer may vary. Moreover, each unit has two zinc binding domains, where a zinc ion is the core of the active site: it settles an hydrogen bond with NAD, helped by Cys-146, Cys-174, and His-67; while the lateral chains of Phe-319, Ala-317, His-51, Ile-269 and Val-292 bind aldehyde. 
</P><P>Deprotonation of NAD is catalysed by Ser-48, which is deprotonated by His-51. When NAD<SUP>+</SUP> loses its positive charge, it can reduce acetaldehyde to ethanol and become NADH.
</P><P>We selected <I>Zymomonas mobilis</I> as the source of the adhB transgene for implementing the conversion efficiency in the glucose-to-ethanol process, because it has been found that this microbiota follows a special glucose degradation pathway (Entner-Dourdoff pathway), that hass as only final byproducts, two ethanol molecules by each glucose, while two ATP molecules are produced.
</P><P>Moreover,<I> Z.mobilis</I> has another adh gene (adhA), which does not seem an isoform even if it accomplishes the same function. In addition, adhB seems more efficient in ethanol conversion than its homologue.
</P><H2><SPAN class="mw-headline" id="SYSTEM_OVERVIEW">SYSTEM OVERVIEW</SPAN></H2><P>Our purpose is to build up a biological system to convert lactose into ethanol. We chose <I>E. coli</I> bacterium as the system’s chassis and we have planned to engineer an efficient lactose-ethanol conversion system using three main enzymes: beta-galactosidase, pyruvate decarboxylase and alcohol dehydrogenase II.
The engineered gene network we want to build up and characterize quantitatively is shown in figure.
</P><TABLE align="center"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Full gene network</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The full characterization of this network will allow to know the efficiency and the feasibility of the entire system in cheese whey valorisation. Theoretically, this treatment will drop dramatically B.O.D.5/C.O.D. parameters of the waste.
</P><P><FONT size="3"><B>Lactose conversion in glucose through beta-galactosidase</B></FONT></P><P><I>E. coli</I> has a well known gene system to grow in a lactose-rich and glucose-poor environment, whose main enzyme is the beta-galactosidase, which performs the conversion of lactose in glucose and galactose. The latter is then metabolized by other enzymes and is converted in glucose too.
</P><P>Although <I>E. coli</I> already has this system in nature, the lactose metabolism may be improved in several ways:
</P><UL><LI>overexpressing the native E. coli beta-galactosidase;
</LI><LI>engineering the expression of a more efficient heterologous beta-galactosidase;
</LI><LI>engineering a glucose-independent lactose transport inside the cell;
</LI><LI>limiting the metabolic burden of the strain by controlling the expression of beta-galactosidase with a lactose-inducible system just like in lac operon, but without the dependence on glucose.
</LI></UL><P>In this project, we want to overexpress the native E. coli enzyme. We hope that this system will be able to convert lactose in glucose more rapidly than in wild type strains. The genetic parts which can perform this function are shown and described below.
</P><P><FONT size="3"><B>Beta-galactosidase protein generator</B></FONT></P><TABLE align="center"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Beta-galactosidase protein generator</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>The gene which encodes for beta galactosidase in <I>E. coli</I> is called lacZ and is known to the Registry as BBa_I732005. We used it in association with a strong RBS to build up a beta-galactosidase protein generator which can ensure an efficient overexpression of the enzyme (the protein generator was actually present in the Registry as BBa_I732019, but its sequence analysis gave bad results in 2008 Distribution, so we re-built it).
As already shown, beta-galactosidase performs the conversion from lactose to glucose and galactose; on the other hand, galactose is metabolized by the cell and converted in glucose through another pathway. So, the expression of this protein generator is the only needed step for the conversion of lactose in glucose, which is essential for the ethanol fermentation we plan to engineer in our project.
</P><P>In this artificial protein generator, lacZ gene could be replaced with any other beta-galactosidase genes taken from the genome of other organisms in order to compare their activity and to choose the most efficient one.
</P><P>Used BioBrick parts:
</P><UL><LI> - :  lacZ translational unit
</LI><LI> - : Double terminator
</LI></UL><P><FONT size="3"><B>Ethanol production from glucose: engineering a high efficiency fermentation in <I>E. coli</I></B></FONT></P><TABLE align="center"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:484px;"><DIV class="thumbcaption">Pyruvate decarboxylase and alcohol dehydrogenase generator</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>In this project, two genes from the fermentative bacterium <I>Zymomonas mobilis</I> are used to build up an ethanol-producing operon in <I>E. coli</I>. Genes pdc and adhB, which encode for pyruvate decarboxylase and alcohol dehydrogenase II respectively, were assembled to engineer an ethanol producing system.
</P><P>DNA synthesis technology and the Registry of Standard Biological Parts were used to optimize the characteristics of the operon, in particular:
</P><UL><LI> codon usage;
</LI><LI> ethanol yield;
</LI><LI> metabolic burden of the host strain.
</LI></UL><P>The codon usage has been optimized for <I>E. coli</I>, through DNA de novo synthesis, in order to have a high efficiency translation of pdc and adhB genes. The ethanol yield and the metabolic burden can be calibrated using different PoPS inputs.
This optimized operon can be considered as the final actuator for our project: it  converts glucose, the intermediate product, in ethanol, which can be used as a biofuel directly or can be converted in different biofuels, e.g. biodiesel through a transesterification process.
</P><P>The final version of the actuator should be controlled by a constitutive promoter in order to ensure an efficient and continuous ethanol production without any additional cost for inducer molecules. However, the strength of the promoter has to be calibrated and the assembly of the operon with a high number of promoter BioBricks can be avoided using well characterized inducible promoters. These inducible systems can be used as user-controlled knobs for gene expression.
</P><P>In order to compare the strengths of different promoters, a standard measurement system has been used: the Relative Promoter Units (RPUs), proposed by Kelly J. et al. (2008). The characterization of these systems would significantly contribute to Synthetic Biology community, because it would provide well-characterized and re-usable parts to the Registry. In particular, we have focused our attention on IPTG, aTc and 3OC6HSL inducible systems.
</P><P>Used BioBrick parts:
</P><UL><LI>pdc (): codon-optimized coding sequence from Mr Gene de novo synthesis service
</LI><LI>adhB (): codon-optimized coding sequence from Mr Gene de novo synthesis service
</LI><LI> - pSB1A2: RBS with efficiency 0.6
</LI><LI> - pSB1AK3: Double terminator
</LI><LI>A promoter to be chosen
</LI></UL><P><FONT size="3"><B>Constitutive promoters and Inducible systems</B></FONT></P><P>The final part containing pdc and adhB has to be controlled by the best promoter, i.e. the promoter with transcriptional strength that ensures an ethanol yield as close as possible to the theoretical value, which is about 52% of the weight of fermented lactose.
Therefore, we want to build up a collection of well characterized promoters both constitutive and inducible, to optimize the operon yield.
In order to characterize inducible systems behaviour, an important part of our project will be dedicated to perform quantitative experiments of these constructs.
</P><P>The systems we want to use are 
</P><UL><LI>aTc inducible device
</LI></UL><P>This device is based on the expression of tetR, a molecule which has the ability of inhibit the activity of Ptet promoter. In presence of aTc, tetR is inactivated and Ptet can be turned on.
We want to test different promoters to regulate the expression of the repressor molecule, so promoters of Anderson Promoters Collection have been assembled in order to drive tetR production.
</P><UL><LI>3OC6HSL inducible device
</LI></UL><P>This device can be induced with 3OC6-HSL autoinducer molecule: it binds luxR protein (encoded by ), which is constitutively expressed by tetR promoter (). LuxR-HSL complex can work as a transcriptional activator for lux promoter ().
</P><UL><LI>IPTG/lactose inducible device
</LI></UL><P>The hybrid lac promoter () has been designed taking the Plambda promoter () and substituting its cI () binding sites with two lacI binding sites.
This promoter can be repressed by lacI (), which can be repressed by lactose or IPTG, providing a lactose/IPTG inducible system. Differently from wild type lac promoter, this part does not have any CAP binding sites, so its behaviour is glucose-independent.
</P><P>
We used these devices as user-controlled knobs to calibrate promoter strenght, with the final goal of determining the best promoter for our ethanol producing operon and lactose cleaving module.
Of course, thanks to the standard measurement approach, our methodology can be generalized to optimize the regulation of any genetic actuator.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:UNIPV-Pavia/Project/Solution">http://2009.igem.org/Team:UNIPV-Pavia/Project/Solution</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UNIPV-Pavia/Project/Solution" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UNIPV-Pavia/Project/Solution" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Project/Solution&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Project/Solution&amp;oldid=164844" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>