Team:US AFRL CarrollHS/Results
Team
Team Mentors Collaborations
Project
Description Interlab Design Experiments Notebook Contributions Modeling Results Improve Attributions
Parts
Parts Basic Parts Composite Parts
Safety
Human Practices
Silver HP Integrated and Gold HP Public Engagement
Judging
Medal Criteria Judging Form
Jamboree
Awards Our Time in Boston Memes
Results
Results
At the conclusion of our project we had created three working plasmids, all of which were sent to the parts registry.
Part BBa_K2522000 is a plasmid developed using the csgA gene extracted from the genomic DNA of E. coli Nissle, and a double cellulose binding domain taken from the iGEM DNA Distribution Kit. Theoretically, the plasmid should allow an E. coli bacterium to produce curli fibers with a double cellulose binding domain attached to each CsgA subunit so that the bacterium can encase itself in cellulose. However, while sequencing was completed to confirm our plasmid matched our theoretical design, no tests have been performed to validate a working construct yet (Ability to bind to cellulose, ect.)
Part BBa_K2522001 is a working construct developed to produce Super Folder Green Fluorescent Protein in the presence of the quorum sensing molecule 3OC12. The DNA is confirmed to work as theoretically planned through both sequencing and experiment.
Figure 1 This picture shows our plasmid produces a visible fluorescence in the presence of 3OC12
Part BBa_K2522002 is a plasmid designed to produce sfGFP in the presence of the quorum sensing molecule Auto Inducer 2 (AI-2). While confirmed to match our theoretical design via sequencing, due to time constraints only one experiment was performed. The plasmid was grown up overnight along with supernatant from a E.coli Nissle 1917 sample (which in theory should contain AI-2); however, the test was inconclusive due to contamination (bacterial growth in pure LB broth used as our blank)
We also characterized our two choices for green fluorescent protein: GFPa1 and sfGFP. Below are our results indicating that sfGFP is best for our project due to the short time taken to fluoresce (~ 4 hours to a visible level) and it’s use in a diagnostic environment that our plasmid would be used for.
In conclusion we:
Characterized and submitted 3 parts to the registry
Compared and characterized two of our GFPs
Successfully completed Interlab
Created presentation accessible to grade schoolers and a brochure aimed toward the general public.
Gained a better understanding of the Interlab project through collaboration with Cadets2Vets
Received modelling results from the National University of Singapore.
Added protocols to the U of Michigan – Software’s website project: Protocat.
Future Plans
In the future, we hope to take a more proactive approach with the purpose of our project, producing a protein to eliminate ETEC if detected (a la Probiotics) instead of a simple visual response. This would allow a single capsule to be taken beforehand when deployed, with the probiotics quickly leaving the system if nothing is detected and eliminating the E. coli if it is detected.
