<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_US_AFRL_CarrollHS_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:US AFRL CarrollHS/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><UL class="navbar"><LI class="dropdown"><A href="javascript:void(0)" class="dropbtn">Team</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/Team">Team</A><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/Mentors">Mentors</A><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/Collaborations">Collaborations</A></DIV></LI><LI class="dropdown"><A href="javascript:void(0)" class="dropbtn">Project</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/Description">Description</A><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/Interlab">Interlab</A><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/Design">Design</A><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/Experiments">Experiments</A><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/Notebook">Notebook</A><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/Contribution">Contributions</A><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/Model">Modeling</A><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/Results">Results</A><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/Improve">Improve</A><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/Attributions">Attributions</A></DIV></LI><LI class="dropdown"><A href="javascript:void(0)" class="dropbtn">Parts</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/Parts">Parts</A><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/Basic_Part">Basic Parts</A><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/Composite_Part">Composite Parts</A></DIV></LI><LI class="navbar"><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/Safety">Safety</A></LI><LI class="dropdown"><A href="javascript:void(0)" class="dropbtn">Human Practices</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/HP/Silver">Silver HP</A><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/HP/Gold_Integrated">Integrated and Gold HP</A><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/Engagement">Public Engagement</A></DIV></LI><LI class="dropdown"><A href="javascript:void(0)" class="dropbtn">Judging</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/MedalCriteria">Medal Criteria</A><A href="https://igem.org/2017_Judging_Form?team=US_AFRL_CarrollHS">Judging Form</A></DIV></LI><LI class="dropdown"><A href="javascript:void(0)" class="dropbtn">Jamboree</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/Awards">Awards</A><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/Boston">Our Time in Boston</A><A href="https://2017.igem.org/Team:US_AFRL_CarrollHS/Memes" style="background:transparent; color: transparent; 
       height: 4px; padding:0px; ">Memes</A></DIV></LI></UL></DIV><DIV class="hero-image"><DIV class="hero-text"><H1>          Notebook          </H1></DIV></DIV></DIV><DIV class="text"><H2>Week 1: June 1-2</H2><H3>Thursday (06/01/17)</H3><P>Exciting first day orientation and safety training off-Base for the newly assembled crew. The mentors and students bonded as work started towards a common goal.</P><H3>Friday (06/02/17)</H3><P>On the second day, the students began learning and overviewing basic lab
techniques. These techniques include conversion factors, serial dilutions,
micro pipetting, and creating and pouring plates of LB agar. Under the
tender supervision of knowledgable mentors, the students learned to accurately
master lab techniques that would prove invaluable once lab work began.</P></DIV><DIV class="text2"><H2>Week 2: June 5-9</H2><H3>Monday (06/05/17) and Tuesday (06/06/17)</H3><P>The first two days of this week were spent on Wright Patterson Air Force Base
(WPAFB). The team members toured the two labs that they would be
working in and received safety training in both. The student researchers
were very excited to tour the world-class facilities on base. Multiple brainstorming
discussions were also held to finalize the project idea. After the week of training, the
group of students split up into 2 groups: one to work in the 711th Human
Performance Wing, and one to work in the Materials Directorate. The young
researchers at the 711th Human Performance Wing were tasked with sensing and
response while the researchers at the Materials Directorate were tasked with
packaging.</P><H3>Wednesday (06/07/17)</H3><P><B>Sense/Respond: </B>On the first day in the lab the members made LB broth, (formula in
protocol tab). Although a simple procedure, this was the first hands-on
experience for the students in the lab. They then did a plasma
transformation protocol using E.coli JM109 and the plasmid PRhl_GFPa1. The
bacteria created was plated onto LB chloramphenicol and left in incubator (37°C),
overnight. If there was growth on the plates in the morning there would a greater
probability that the E. coli absorbed the plasmid. The tired but excited young
researchers went home with fingers crossed.</P><P><B>Packaging: </B>The lab members attended a review meeting for the Materials Lab. This is where people gave mini presentations on how their projects were going and what problems they have run into. The Packaging members and Chia considered the advantages of using different surface proteins in order to package E. coli and decided on curli. The four Packaging students searched the iGEM site for parts (cellulose binding domains, CsgA, etc) was discussed. 
</P><H3>Thursday (06/08/17)</H3><P>The team attended a seminar given by a professor from Northwestern.</P><P><B>Sense/Respond: </B>When they entered the lab on the second day the plates showed growth
with distinctions of individual
colonies. Mentor Mike Goodson
ecstatically let the students know about the
significance of the growth and students were
relieved that their plates had grown! These
colonies were used to complete the PCR
protocol. The master mixture, (protocol tab) and
LB broth in round bottom falcon tubes were left
in the incubator (37°C), overnight. The plates
were put in the refrigerator (4°C) to preserve the
bacteria for future use. This protocol was used
to confirm that E.coli JM109 had absorbed the
plasmid PRhl_GFPa1</P><P><B>Packaging: </B> The Packaging team clarified the goals of their project by deciding to attach a cellulose binding domain to the CsgA portion of a curli protein on E. coli bacteria. In the lab, the team worked with producing agarose gel, diluting a buffer, and using these components to run gel electrophoresis on DNA samples. Additionally, the team poured LB agar plates.

</P><H3>Friday (06/09/17)</H3><P><B>Sense/Respond: </B>For the last day of the week, the students’ goal was to confirm the E. Coli
had absorbed the plasmid and then perform a MiniPrep Kit to take the
plasmid out of the bacteria. To accomplish this, they first learned how to
create a gel and how to use the electrophoresis. The electrophoresis sends the DNA
down the gel using a positive and negative charge. Under a blacklight you can see if
the DNA is present. Next, the students use a QIA prep spin Miniprep Kit to take
plasmid out of bacteria. The students placed one nanodrop onto spectrophotometer
to measure how pure the DNA extracted is. For future experiments, the plasmid DNA
would then be sent to an offsite lab to be sequenced.</P><TABLE><TBODY><TR><TH>260</TH><TH>280</TH><TH>Ratio 260/280</TH><TH>ng/μl</TH></TR><TR><TH>Tina</TH><TD>1.099</TD><TD>0.620</TD><TD>1.77</TD><TD>54.9</TD></TR><TR><TH>Hayley</TH><TD>1.594</TD><TD>0.867</TD><TD>1.84</TD><TD>79.7</TD></TR><TR><TH>Annie</TH><TD>1.366</TD><TD>0.776</TD><TD>1.76</TD><TD>98.3</TD></TR><TR><TH>Jason</TH><TD>1.599</TD><TD>0.886</TD><TD>1.81</TD><TD>79.9</TD></TR></TBODY></TABLE><P class="caption2">Table 1: Nanodrop results</P><P><B>Packaging: </B>The Packaging team prepared DNA samples and analyzed the samples with the Nanodrop device. Later, the team used restriction enzymes to slice open the DNA and then ran the samples through gel electrophoresis. The project idea continued to progress with the team finding parts form the iGEM inventory and researching DNA sequences of the various parts.

</P></DIV><DIV class="text"><H2>Week 3: June 12-16</H2><H3>Monday (06/12/17)</H3><P>The whole team met in the morning to discuss the Wiki and each group’s project.
</P><P><B>Sense/Respond: </B>The students went to a group meeting within their labs. Students
streaked bacteria onto agar plates and put back into the incubator. The
rest of the day was spent collaborating on website design and graphic
design. Two coding novices, Annie and Jason, worked on learning html, CSS, and
Javascript. Graphic designer Tina created a new mascot microbe. Assistant graphic
designer and artist Annie created a drawing of the original Wright Flyer and assisted
in Photoshop drawing. Stenographer Hayley typed up protocols and helped organize
team documents.</P><P><B>Packaging: </B> The RX team observed the preparation and running of PCR. The team made LB
broth and agar. Later the team observed the purification of the PCR results and the
analyzation of the results with Qubit.
</P><H3>Tuesday (06/13/17)</H3><P><B>Sense/Respond: </B>The students removed the agar plates with bacteria from the incubator.
Two tubes were filled with LB broth, one labeled as a blank and one
containing the bacteria. Using a sterile toothpick, a single colony was picked
from each plate, and placed in the culture tube. An unused sterile toothpick was
added to the blank tube, to verify that the process was sterile. These tubes were
then shaken in an incubator for 3 hours.</P><P>The students conducted an experiment using culture growth to see if the bacteria
will turn green with the addition of the quorum sensing molecule to the bacteria.</P><P>The next morning, the plasmid should glow green indicating it is present.</P><P><B>Packaging: </B>The RX team attended a Materials Lab meeting. Afterwards, the team poured agar plates and transferred LB broth from one liter bottles to Falcon tubes. The team went to the RH lab to obtain the double CBD (cellulose binding domain) DNA from the iGEM plates and transfer the DNA to RX.
</P><H3>Wednesday (06/14/17)</H3><P><B>Sense/Respond: </B>The students retrieved the test tubes placed in the incubator the day before
and analytically checked the optical density and the fluorescence of the
colonies. Immediately afterwards, a visual check was performed as well,
confirming that the Colony + 3OC12 glowed green. The ecstatic students celebrated
as anticipated results showed through the light.</P><TABLE><TBODY><TR><TH>Colony+3OC12</TH><TH>Colony+DMSO</TH><TH>LB+3OC12</TH><TH>LB+DMSO</TH></TR><TR><TH>Dallas</TH><TD>0.929</TD><TD>1.048</TD><TD>0.039</TD><TD>0.037</TD></TR><TR><TH>Tina</TH><TD>1.075</TD><TD>1.084</TD><TD>0.048</TD><TD>524.53</TD></TR><TR><TH>Hayley</TH><TD>1.094</TD><TD>1.079</TD><TD>0.042</TD><TD>0.041</TD></TR><TR><TH>Annie</TH><TD>1.096</TD><TD>1.079</TD><TD>0.044</TD><TD>1.227</TD></TR><TR><TH>Jason</TH><TD>1.229</TD><TD>1.213</TD><TD>0.055</TD><TD>1.227</TD></TR></TBODY></TABLE><P class="caption2">Table 2: Optical Density of PLCD-GFPa1</P><TABLE><TBODY><TR><TH>Colony+3OC12</TH><TH>Colony+DMSO</TH><TH>LB+3OC12</TH><TH>LB+DMSO</TH></TR><TR><TH>Dallas</TH><TD>2359.7</TD><TD>889.14</TD><TD>594.84</TD><TD>645.10</TD></TR><TR><TH>Tina</TH><TD>35454</TD><TD>841.14</TD><TD>524.53</TD><TD>1074.5</TD></TR><TR><TH>Hayley</TH><TD>27733</TD><TD>844.70</TD><TD>508.87</TD><TD>513.41</TD></TR><TR><TH>Annie</TH><TD>28791</TD><TD>837.42</TD><TD>906.98</TD><TD>950.71</TD></TR><TR><TH>Jason</TH><TD>35088</TD><TD>958.79</TD><TD>673.31</TD><TD>950.71</TD></TR></TBODY></TABLE><P class="caption2">Table 3: Fluorescence of LCD-GFPA1+3OC12</P><P>In the afternoon, the students were back at work again preparing a mini-prep on the
PLCD-GFPa1 (the plate streaked on Monday). The remaining DNA containing the
GFP was then placed in the -20o C freezer for storage. Having much more
experience the second time doing the mini-prep, the students had a
much better grasp and spent less time with more accuracy.</P><TABLE><TBODY><TR><TH>260</TH><TH>280</TH><TH>Ratio 260/280</TH><TH>ng/μl</TH></TR><TR><TH>Dallas</TH><TD>2.106</TD><TD>1.273</TD><TD>1.65</TD><TD>105.3</TD></TR><TR><TH>Tina</TH><TD>0.927</TD><TD>0.456</TD><TD>2.03</TD><TD>46.4</TD></TR><TR><TH>Hayley</TH><TD>0.723</TD><TD>0.766</TD><TD>2.18</TD><TD>38.8</TD></TR><TR><TH>Annie</TH><TD>0.705</TD><TD>0.357</TD><TD>1.97</TD><TD>35.2</TD></TR><TR><TH>Jason</TH><TD>0.561</TD><TD>0.276</TD><TD>2.03</TD><TD>28.0</TD></TR><TR><TH>Blank</TH><TD>-0.50</TD><TD>-0.048</TD><TD>1.04</TD><TD>-2.5</TD></TR></TBODY></TABLE><P class="caption2">Table 4: Nanodrop results</P><P><B>Packaging: </B>The RX team performed a transformation in order to induce DH5 alpha competent cells to accept a plasmid containing the DNA for the double cellulose binding domain protein. The plates containing the bacteria were left in an incubator overnight in order to allow colonies to grow.
</P><H3>Thursday (06/15/17)</H3><P><B>Sense/Respond: </B>Time to go back to work bright and early. A little after the crack of dawn,
the engaged students immediately went to work preparing a digest. The
scholars made a 15 uL digest along with a Master Mix. Interactive steps
included preparing a mini-prep, mixing with enzyme BSr61-HF, enzyme KpuI-HF, buffer Cutsmart, and water. The remaining solution was then placed in incubation for
2 hours. In addition, students were excited to read and learn about plasmid cloning
by restriction enzyme digest and restriction endonuclease. With a desire to learn
more, the students trekked back to the lab and met with another qualified scientist
to assist in the digest. Once the digest was
done, the students made a gel and
electrophoresis to analyze the DNA. The
electrophoresis sends the DNA down the
gel using a positive and negative charge
and allows the students to see the amount
of base pairs, and in this case also allows
the students to see the cut DNA and
replacement. However, the young scientists
were very disappointed to find out the the
digest did not go as expected and DNA did
not get replaced as expected. Disappointed
but not defeated, the students immediately
went back to work and retraced their steps,
beginning the process all over again.
Things do not always work out in the lab
and the students were eager to restart and achieve favorable results.</P><TABLE><TBODY><TR><TH>260</TH><TH>280</TH><TH>Ratio 260/280</TH><TH>ng/μl</TH></TR><TR><TH>Dallas- PLas1s</TH><TD>2.429</TD><TD>1.275</TD><TD>1.91</TD><TD>121.5</TD></TR><TR><TH>Dallas- PLas2</TH><TD>1.363</TD><TD>0.701</TD><TD>1.94</TD><TD>68.2</TD></TR><TR><TH>Tina- PLas1</TH><TD>2.059</TD><TD>1.065</TD><TD>1.93</TD><TD>103.0</TD></TR><TR><TH>Tina- PLas2</TH><TD>1.883</TD><TD>0.990</TD><TD>1.89</TD><TD>94.2</TD></TR><TR><TH>Hayley- PLas1</TH><TD>0.994</TD><TD>0.516</TD><TD>1.93</TD><TD>49.7</TD></TR><TR><TH>Hayley- PLas2</TH><TD>1.691</TD><TD>0.883</TD><TD>1.92</TD><TD>84.6</TD></TR><TR><TH>Annie- PLas1</TH><TD>2.186</TD><TD>1.149</TD><TD>1.90</TD><TD>109.3</TD></TR><TR><TH>Annie- PLas2</TH><TD>1.488</TD><TD>0.796</TD><TD>1.87</TD><TD>74.4</TD></TR><TR><TH>Jason- PLas1</TH><TD>1.3494</TD><TD>0.723</TD><TD>1.93</TD><TD>69.7</TD></TR><TR><TH>Jason- PLas2</TH><TD>2.160</TD><TD>1.104</TD><TD>1.96</TD><TD>108.0</TD></TR><TR><TH>Blank</TH><TD>0.024</TD><TD>0.026</TD><TD>0.91</TD><TD>1.2</TD></TR></TBODY></TABLE><P class="caption2">Table 5: Nanodrop results</P><P><B>Packaging: </B> The RX team performed a Restriction Enzyme Digest on the pBAD containing CsgA DNA and then purified the cut pBAD using  a clean-up kit. The team then analyzed the results of the clean-up using the Nanodrop. When the Nanodrop confirmed the presence of DNA, the team created a Master Mix and ran PCR on the CsgA DNA. Upon the completion of the PCR, the products were purified using a purification protocol.
</P><H3>Friday (06/16/17)</H3><P><B>Sense/Respond: </B>Immediately the next day, the eager students were right back at work. The
previous day's failure had fired them up and they were eager to do
everything carefully and quickly. The day began with a student-led mini-prep
in which the students were able to follow the protocol and do everything by
themselves. Afterwards, the students and mentors went out for a cool ice cream
treat. All 12 days of hard work deserved a celebration and the students and mentors
bonded over frozen custard and ice cream.</P></DIV><P><B>Packaging: </B>The RX team prepared agarose gel and 1X TAE Buffer in order to run the CsgA DNA PCR products on a gel. Images were taken of the results of the gel electrophoresis in order to confirm the presence of the correct DNA. Colonies began to appear from the transformation using the dCBD DNA. 

</P><DIV class="text2"><H2>Week 4: June 19-23</H2><H3>Monday (06/19/17)</H3><P><B>Sense/Respond: </B>After the weekend, the students met in the lab and initiated a second
digest. They were determined to get everything measured correctly and
leave minimal room for mistakes. This time around, the students were
much more knowledgeable about buffers and enzymes used and
proceeded carefully combining them with DNA. After the digest was complete, the
young scientists began prepping another gel for electrophoresis. Everything went
smoothly while the digest was completed and DNA was extracted and cut by the
enzymes. Next, the students prepped the DNA by adding a loading dye and then
proceeded to put it on the gel. After electrophoresis for 30 min, the gel was complete.
The LabPats were able to use this downtime for updating lab journals, creating new
and exciting logos, as well as continuing to work on the wiki. Next, another
experienced mentor helped with analyzing the gel and cutting out the DNA from the
gel. However, they were quite disappointed to learn that there were 3 bars, indicating
more base pairs than expected. The mentors ensured them that the gel still worked
and that they just needed to cut out the fluorescent band that was relevant.
Meticulous work was done to cut out the gel, and the labpats were finally able to let
out a sigh of relief once the gel was safely placed in the tube and put in the fridge to
be melted down tomorrow.</P><P><B>Packaging: </B>To start the week, the RX team ran a miniprep and restriction digest on the dCBD DNA. After the restriction digest, the team ran a gel to test the different digested DNA lengths. The gel results weren’t as expected and the team began to re-do a miniprep.
</P><H3>Tuesday (06/20/17)</H3><P><B>Sense/Respond: </B>The students quickly resumed where they had left off to begin their gel
extraction. Two different protocols were used to see which one would
achieve a higher concentration. The teams were divided into Tina and Annie
working with Svetlana on (QIAEX II Agarose Gel Extraction) and Jason and Dallas (who
unfortunately was ill, so Annie and Tina tag teamed his sample) working with Mike on
QIAEX Gel Extraction Kit. The gel extractions were performed and the nano drop test
was performed. The only successful gel extractions were performed by the girls
(Svetlana’s protocol), so they took home the victory! The boys learned a lot from their
error and submitted defeat. The day was wrapped up by working hard designing the
wiki and learning how to code.</P><P><B>Packaging: </B>The RX team finished the miniprep from Monday and ran another nanodrop test to make sure of the concentration and purity.  Later, the team prepared LB broth with Kanamycin. To increase the amount of DNA, the RXers ran a PCR and then prepared a gel to test the PCR products.
</P><H3>Wednesday (06/21/17)</H3><P><B>Sense/Respond: </B>Today the students began their second Polymerase Chain Reaction (PCR).
Under the watchful eye of Dr. Harbaugh, the students independently finished
the PCR and loaded it into the thermocycler. Another small gel was prepared
and the PCR’s were taken out and loaded on the gel. The students meticulously loaded
the PCR’s onto the slits of the gel with incredible precision that truly showed their
growing expertise.</P><P>The students received the results they had been hoping for, which showed that the
Super folded GFP had been inserted into the bacteria. They then performed a PCR
Purification, which separated the massive amounts of DNA. The DNA was then
measured using the nano drop and these results were given:</P><TABLE><TBODY><TR><TH>260</TH><TH>280</TH><TH>Ratio 260/280</TH><TH>ng/μl</TH></TR><TR><TH>Dallas</TH><TD>3.099</TD><TD>1.700</TD><TD>1.82</TD><TD>155.0</TD></TR><TR><TH>Tina</TH><TD>2.355</TD><TD>1.357</TD><TD>1.74</TD><TD>117.8</TD></TR><TR><TH>Jason</TH><TD>1.756</TD><TD>1.005</TD><TD>1.75</TD><TD>87.8</TD></TR><TR><TH>Hayley</TH><TD>2.3</TD><TD>1.3</TD><TD>1.81</TD><TD>119.6</TD></TR><TR><TH>Annie</TH><TD>3.793</TD><TD>2.050</TD><TD>1.85</TD><TD>189.6</TD></TR><TR><TH>Blank</TH><TD>0.013</TD><TD>0.058</TD><TD>0.022</TD><TD>-0.6</TD></TR></TBODY></TABLE><P class="caption2">Table 6: Nanodrop results</P><P><B>Packaging: </B>The RX team ran another gel using the PCR products of dCBD DNA from Tuesday. After the gel was complete, the DNA bands were cut from the gel and purified using a gel extraction. The results of the gel extraction were run on the previous gel in order to confirm the dCBD DNA as correct. The dCBD DNA was injected into a pCR Blunt II TOPO plasmid through a TOPO Ligation and then transformed into TOP 10 competent cells.
</P><H3>Thursday (06/22/17)</H3><P><B>Sense/Respond: </B>The day began with another PCR. Today, the students were ready to
completely do the PCR by themselves with little guidance from the mentors.
The standard protocol was followed and the PCR and ran another cycle in
the thermocycler. Meanwhile, another gel was prepared by the students. This time
around, the students were very efficient and the process went by very quickly. The gel
ended up giving too many bands, but the mentors conversed and decided to proceed
with the PCR purification anyways. After the lab work was done, the students decided
to meet with the other lab students to discuss current progress. The students created
discussion points and had a great discussion with the other lab.</P><P><B>Packaging: </B>The morning began with the RXers beginning cellulose cultures. Later the team started colonies of the TOP 10 E. coli cells in culture tubes with LB broth with Kanamycin, as well as LB agar plates with Kanamycin. Growing colonies would allow the plasmids to replicate inside the cells. The entire team met at UES in the afternoon.
</P><H3>Friday (06/23/17)</H3><P><B>Sense/Respond: </B>The day began bright and early with two PCR digests. This time a PCR
Vector Digest was conducted on the same from yesterday and a PCR GFP
digest was done on the sample from Wednesday (6/21/17). After following
the protocol, the students were able to successfully perform the digests. After
a 2 hour wait (per the protocol), the young scientists performed a purification on the
GFP digest. The goal was to cut off end bits of the base pairs to allow the plasmid
connect again. During the students' lunch break, another meeting was held with teacher
leaders who wanted to know information about progress and future proceedings.</P><TABLE><TBODY><TR><TH>260</TH><TH>280</TH><TH>Ratio 260/280</TH><TH>ng/μl</TH></TR><TR><TH>Dallas</TH><TD>2.106</TD><TD>1.273</TD><TD>1.65</TD><TD>105.3</TD></TR><TR><TH>Tina</TH><TD>0.927</TD><TD>0.456</TD><TD>2.03</TD><TD>46.4</TD></TR><TR><TH>Jason</TH><TD>1.188</TD><TD>0.719</TD><TD>1.65</TD><TD>59.6</TD></TR></TBODY></TABLE><P class="caption2">Table 7: Nanodrop results</P><TABLE><TBODY><TR><TH>DNA</TH><TD>10μl</TD></TR><TR><TH>10x Buffer</TH><TD>1.5μl</TD></TR><TR><TH>Kpn1-HF</TH><TD>1μl</TD></TR><TR><TH>BsrG1-HF</TH><TD>1μl</TD></TR><TR><TH>H2O</TH><TD>1.5μl</TD></TR></TBODY></TABLE><P class="caption2">Table 8: GFP Digest (15μl)</P><TABLE><TBODY><TR><TH>DNA</TH><TD>8.5μl</TD></TR><TR><TH>10x Buffer</TH><TD>1.5μl</TD></TR><TR><TH>Kpn1-HF</TH><TD>1μl</TD></TR><TR><TH>BsrG1-HF</TH><TD>1μl</TD></TR><TR><TH>H2O</TH><TD>3μl</TD></TR></TBODY></TABLE><P class="caption2">Table 9: Vector Digest (15μl)</P><P><B>Packaging: </B>The RXers extracted the TOPO plasmids from the E. coli and ran a restriction digest. To ensure the correct dCBD length, both cut and uncut DNA  was run on a gel. The results of the gel determined the possibility of sending in plasmids with dCBD for sequencing on Monday.
</P></DIV><DIV class="text"><H2>Week 5: June 26-30</H2><H3>Monday (06/26/17)</H3><P><B>Sense/Respond: </B>The students began another week of hard work. They made a gel to run
their previous Plasmid Digest. Next, they looked at the gel under the blue
light and found the band they were looking for at 3.2k base pairs. Next they
scalpeled out the desired band and began a gel extraction using the QIAEX ll
Agarose Gel Extraction Protocol. After the gel extraction they tested their results on the
Spectrotropsesis. The first sample was lower than preferred and the next sample had a
very uneven graph, but with the desired numerals. Afterwards, the mentors discussed
and determined to still do a legation the next day and see the results.</P><TABLE><TBODY><TR><TH>260</TH><TH>280</TH><TH>Ratio 260/280</TH><TH>ng/μl</TH></TR><TR><TH>TD+AB+JD</TH><TD>0.046</TD><TD>0.027</TD><TD>1.71</TD><TD>2.3</TD></TR><TR><TH>DM+HJ</TH><TD>0.227</TD><TD>0.183</TD><TD>1.24</TD><TD>11.3</TD></TR><TR><TH>Blank</TH><TD>0.015</TD><TD>-0.005</TD><TD>-2.86</TD><TD>0.7</TD></TR></TBODY></TABLE><P class="caption2">Table 10: Nanodrop results</P><P><B>Packaging: </B>The RX side of the team did a nanodrop on the uncut dCBD TOPO plasmid and found the concentration of each sample. Water was added to each samples to create an equal concentration at 50 nanograms per microliter and 500 ng in all. We then sent our samples to be sequenced.
</P><H3>Tuesday (06/27/17)</H3><P><B>Sense/Respond: </B>Today was the student's first ligation. The goal was to put the sticky ends
back together to form a complete plasmid. A lot of math was done to
determine the exact ratios of sfGFP and PCR vector plasmid. Luckily, Dr.
Harbaugh was able to assist the students with newly introduced lab techniques. There
was another protocol and the students were able to follow it with no problem. They
then went to the conference room and had another meeting with the other labs and
teachers. This time, both labs gave extraordinary presentations about project progress
and end goal. The RH lab presented their abundance of lab work and myriad of
website/wiki work. The RX lab presented lab work and some technical website work.
Afterwards, the groups planned out agenda and discussed necessary steps to
complete the wiki page.</P><P><B>Packaging: </B>The team removed the cellulose from the plates, rinsed the cellulose top layer and put each sample in a test tube. The team took pictures of each sample.  The tubes were then frozen in a -80༠ freezer.
</P><H3>Wednesday (06/28/17)</H3><P><B>Sense/Respond: </B>The students began the day with a new PCR. After examining the plates from
yesterday, it was determined that there indeed was green fluorescence on the
plates. However, a PCR was needed with a gel to determine whether or not it was
sfGFP or GFP. High in spirits, the students immediately went to work. First, they
created a Master Mix following the PCR Protocol for Phusion DNA Polymerase. They
did some tricky math and figured out exact proportions needed for nuclease-free water,
Phusion Buffer, DMSO, Kpn1(forward primer), BsrG1(reverse primer), as well as the
phusion DNA polymerase. They each used 6 colonies. Four students used green
colonies and then dipped it into the PCR and the LB Broth with chloramphenicol. This
method ensures that only the bacteria immune to chloramphenicol can survive. On the
gel, it should give a precise band of base pairs indicating sGFP or GFP. Another
student used 6 colonies without the green fluorescence. It acts as a control. When it is
run as a gel, there should be random bars that indicate there is no sGFP or GFP. Then
the samples were put into the thermocycler for 1.5 hours according to the protocol.
The students then had the endure a nerve raking wait as they wondered if the PCR had
gone successfully and whether the bacteria had sGFP or not. After preparing another
gel, the students meticulously put the samples on the 40 wells. After 30 minutes of
running the gel, the students came back to an ecstatic Dr. Goodson informing them
that the gel had turned out as planned and that they could proceed with the mini-prep
before sending in the DNA for sequencing. Everyone shared &quot;elbow-fives&quot; and were
relieved that all their hard work had not gone to waste!</P><P><B>Packaging: </B>The cellulose tubes were removed from the -80༠ freezer. The tubes were then put into a beaker that would be attached to a lyophilizer overnight at -105༠ in a vacuum.
</P><H3>Thursday (06/29/17)</H3><P><B>Sense/Respond: </B>Today the students began to mini-prep the samples from the previous day.
Right from the start, the students noticed fluorescent green coming from
one of the tubes. This would foreshadow what they would see later in the
day. They began the mini-prep full of hope and concentrated on following the protocol
exactly to achieve the best possible results. They decided to first put the tubes over
the light and observed a lot of fluorescence (see picture). After about 2 hours, they
were finally done. They nano dropped the sample and observed an incredible amount
of DNA. Everyone cheered as they realized that the &quot;response&quot; component of the
project was successful.</P><P><B>Packaging: </B> Sequencing results for dCBD and CsgA DNA were reviewed using the program Blast. Each of the six dCBD in TOPO plasmid samples contained no errors on either the SP6 or the T7 primer sequences; however, some samples contained the gene in the reverse orientation. Chia's CsgA in TOPO plasmid samples were also entirely correct. 
</P><H3>Friday (06/30/17)</H3><P><B>Sense/Respond: </B>Since all the lab work is currently done until the DNA sample is sent to Ohio State
University for DNA sequencing, the students had the whole day to make website
improvements and wiki renovations. New logos were also created as well as
graphics. The students will continue to add content to the wiki weekly.</P><TABLE><TBODY><TR><TH>260</TH><TH>280</TH><TH>Ratio 260/280</TH><TH>ng/μl</TH></TR><TR><TH>1</TH><TD>2.899</TD><TD>1.657</TD><TD>1.75</TD><TD>145.0</TD></TR><TR><TH>2</TH><TD>2.364</TD><TD>1.268</TD><TD>1.86</TD><TD>118.2</TD></TR><TR><TH>3</TH><TD>1.948</TD><TD>1.030</TD><TD>1.89</TD><TD>97.4</TD></TR><TR><TH>4</TH><TD>2.152</TD><TD>1.143</TD><TD>1.88</TD><TD>107.6</TD></TR><TR><TH>5</TH><TD>2.071</TD><TD>1.123</TD><TD>1.84</TD><TD>103.5</TD></TR><TR><TH>6</TH><TD>2.421</TD><TD>1.318</TD><TD>1.84</TD><TD>121.1</TD></TR><TR><TH>7</TH><TD>2.072</TD><TD>1.103</TD><TD>1.88</TD><TD>103.6</TD></TR><TR><TH>8</TH><TD>1.764</TD><TD>0.963</TD><TD>1.83</TD><TD>88.3</TD></TR><TR><TH>9</TH><TD>2.287</TD><TD>1.228</TD><TD>1.86</TD><TD>114.4</TD></TR><TR><TH>11</TH><TD>6.402</TD><TD>3.492</TD><TD>1.83</TD><TD>320.1</TD></TR><TR><TD>Blank</TD><TD>0.0003</TD><TD>0.029</TD><TD>0.11</TD><TD>0.2</TD></TR></TBODY></TABLE><P class="caption2">Table 11: Nanodrop results</P><P><B>Packaging: </B>The RXers learned how to design a restriction digest and created the digests for CsgA, dCBD, and pBAD samples. Later, the team performed the three digests, each with one sample, and left the samples to incubate in a heat block for several hours. In the afternoon, the whole iGEM team met with mentors to review the projects.
</P></DIV><DIV class="text2"><H2>Week 6: July 5-6</H2><H3>Wednesday (07/05/17)</H3><P>“Ring, ring, ring!” came from the laptop as the students huddled around
to receive a Skype call from the Singapore iGem team. They were greeted
by Wilbert from the Singapore team. They immediately began conversing
with Wilbert and exchanged project descriptions and ideas. The team was
very intrigued to learn about the Human Practices components of iGem
from Wilbert. After a whole hour of video conference, the team learned a lot from the
sagacious Wilbert. The conversation was very beneficial to the students as they
learned about the different components of iGem. Wilbert even offered to help them
with the wiki page!</P><P><B>Sense/Respond: </B>Afterwards, the students arrived back to the lab and received
presentation tips from mentors for the upcoming lab presentations the following
week.</P><P><B>Packaging: </B>The entire iGEM team convened to hold a Skype call with a representative from the Singapore National College's team. Later on Base, the RXers ran a gel on the results from the previous Friday's digests, which included pBAD, pCRBlunt-CsgA, and pCRBlunt-dCBD. Upon completion, the team members extracted DNA for each sample from the gel using a gel extraction protocol. 
</P><H3>Thursday (07/06/16)</H3><P><B>Sense/Respond: </B>The next day, the group decided to do the InterLab in order to be
considered for medal criteria. They sifted through the kit and found many
useful parts that they could use in order to help standardize fluorescence.
After reading through the protocols and laying out a plan of attack, the
students had the wait since the lab was going under routine inspection. The students
spent the rest of the day designing more graphics and made wiki edits. </P><P><B>Packaging: </B>The morning began with running a gel on the results of the gel extraction. While the gel was solidifying, the team determined the concentrations of the samples with the Nanodrop. After the gel confirmed the pBAD, <I>csgA</I> fragment, and dCBD fragment, the RXers performed a ligation in order to clone the <I>csgA</I> gene into the pBAD plasmid.</P><TABLE><TBODY><TR><TH>Gel Extraction Results</TH><TH>Concentration (ng/uL)</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TH>pBAD 1 (w/ Ncol and Kpnl)</TH><TD>6.2</TD><TD>1.64</TD><TD>0.02</TD></TR><TR><TH>pBAD 2 (w/ Ncol and Kpnl)</TH><TD>7.4</TD><TD>2.11</TD><TD>0.01</TD></TR><TR><TH>CsgA 1 (w/ Ncol and Kpnl)</TH><TD>17.0</TD><TD>1.88</TD><TD>0.03</TD></TR><TR><TH>CsgA 2 (w/ Ncol and Kpnl)</TH><TD>21.5</TD><TD>1.79</TD><TD>0.05</TD></TR><TR><TH>dCBD 1 (w/ Ncol and HindIII)</TH><TD>8.0</TD><TD>1.75</TD><TD>0.02</TD></TR><TR><TH>dCBD 2 (w/ Ncol and HindIII)</TH><TD>8.8</TD><TD>1.98</TD><TD>0.02</TD></TR></TBODY></TABLE><P class="caption2">Table 12: Nanodrop results</P><P>In the
afternoon, the whole team got together with mentors to meet at the Air Force
Museum in Dayton, Ohio in order to take self pictures and group pictures for the
wiki.</P><H3>Friday (07/07/16)</H3><P><B>Packaging: </B>To finish the week, the RXers transformed pBAD-CsgA plasmids into DH5-alpha competent cells and patched the samples onto LB agar plates with ampicillin. Additionally, the team made and poured a multitude of LB agar plates with ampicillin. The entire team met at UES in the afternoon to discuss the Wiki.</P></DIV><DIV class=""><H2>Week 7: July 10-14</H2><H3>Monday (07/10/17)</H3><P><B>Sense/Respond: </B>At 9 AM, the students went to their weekly lab meeting. Interns from their
department always have a weekly meeting where they share ideas and summer
progress. The LabPats will share their presentation next week. Afterwards, it was
finally time to begin the InterLab study for the iGem project. The students began
re suspending the wells. However, they soon found out that there were no more
plates of LB- Chloramphenicol and thus had to make more. They had to
postpone the InterLab studies and finished the day by creating a lot more plates
for future experiments.</P><P><B>Packaging: </B>To start the week, the RX team made LB Broth with Ampicillin and later transferred colonies of E. coli cells with the pBAD-CsgA plasmids to allow them to grow. To accomplish this, the RXers patched colonies of the cells onto LB agar plates with Ampicillin and then transferred the remainder of the colonies into culture tubes containing LB Broth with Ampicillin.
</P><H3>Tuesday (07/11/17)</H3><P><B>Sense/Respond: </B>The student researchers began the InterLab the next morning. They split up in 2
groups while one group worked with the competent cells kit and completed Day
1. The other group transformed the wells inside the plates provided by iGem. The
students did a great job following protocol and growing the bacteria on the agar
plates from the previous day. After they were done, the students began filling out
the necessary forms for the InterLab as well as planned out the dropdown menu and the
slideshow for the website.</P><P><B>Packaging: </B>The RXers spent the morning miniprepping twenty-one samples of DH5 alpha competent cells with pBAD-CsgA. Around five milliliters of each sample from the culture tubes of the previous day were pelleted for miniprepping and one milliliter was reserved for storage.

</P><H3>Wednesday (07/12/17)</H3><P><B>Sense/Respond: </B>The next morning, the young students attended a lecture taught by biologists
from the Purdue University. It was very interesting and higher level, and the
students were very excited to hear topics relevant to their iGem goals.
Afterwards, they returned to the lab with their mentor to examine the overnight
bacteria colonies. However, they were disappointed to learn that the competent cell test did
not work out. Instead of an expected 300+ colonies, the plates only yielded about 25 colonies.
The students spent the rest of the day redoing the protocol in hopes of achieving the needed
amount of colonies in order to proceed.
</P><P><B>Packaging: </B>The Nanodrop was used to check the concentration of several samples of the miniprep results. Then, a restriction digest was performed on the miniprepped pBAD-CsgA plasmids with NcoI and KpnI. Digested and undigested pBAD-CsgA samples were run on a gel in order to confirm the lengths of the fragments. The samples with the clearest CsgA bands were chosen to have a second restriction digest with KpnI and HindIII performed on them to prepare for cloning the dCBD gene into the plasmid. 
</P><TABLE><TBODY><TR><TH>Spot Samples of
pBAD-CsgA</TH><TH>Concentration (ng/uL)</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TH>AS2 pBAD-CsgA</TH><TD>100.0</TD><TD>1.90</TD><TD>1.91</TD></TR><TR><TH>AS6 pBAD-CsgA</TH><TD>105.2</TD><TD>1.89</TD><TD>1.93</TD></TR><TR><TH>AP1 pBAD-CsgA</TH><TD>102.1</TD><TD>1.94</TD><TD>2.00</TD></TR><TR><TH>AP2 pBAD-CsgA</TH><TD>106.0</TD><TD>1.88</TD><TD>1.71</TD></TR><TR><TH>MH1 pBAD-CsgA</TH><TD>109.7</TD><TD>1.94</TD><TD>1.92</TD></TR><TR><TH>MH7 pBAD CsgA</TH><TD>64.8</TD><TD>1.92</TD><TD>2.23</TD></TR></TBODY></TABLE><P class="caption2">Table 13: Nanodrop results</P><TABLE><TBODY><TR><TH>Chosen Samples of
pBAD-CsgA</TH><TH>Concentration (ng/uL)</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TH>AS3 pBAD-CsgA</TH><TD>129.2</TD><TD>1.96</TD><TD>1.98</TD></TR><TR><TH>MH2 pBAD-CsgA</TH><TD>83.1</TD><TD>1.94</TD><TD>1.89</TD></TR><TR><TH>MH3 pBAD-CsgA</TH><TD>114.3</TD><TD>1.96</TD><TD>1.98</TD></TR><TR><TH>MH5 pBAD-CsgA</TH><TD>93.4</TD><TD>1.94</TD><TD>1.97</TD></TR><TR><TH>MH6 pBAD-CsgA</TH><TD>108.0</TD><TD>1.96</TD><TD>2.06</TD></TR></TBODY></TABLE><P class="caption2">Table 14: Nanodrop results</P><H3>Thursday (07/13/17)</H3><P><B>Sense/Respond: </B>Continuing the next morning, the students immediately continued the Interlab.
Today they focused on completing the OD600 and Serial Dilution part of the
protocol. The students hope to test the fluorescence of the colonies tomorrow.
Afterwards, they picked colonies and left them in the incubator to grow overnight.
Afterwards, they got back on wifi and successfully implemented a background to
the wiki. Tomorrow they will have s video conference with the U.S. Navy and U.S. Army iGem
team.</P><P><B>Packaging: </B>The morning began with a &quot;gel extraction&quot; of the digest with pBAD-CsgA with KpnI and HindIII in order to purify the samples. Later, a gel was run to confirm the digested samples of pBAD-CsgA and several samples of dCBD. A number of cultures containing Top10/pCRB-dCBD and separate cultures wtih Top10/pCRB-CsgA were frozen down in a -80 degree Celsius freezer. The day ended with a ligation of the dCBD gene fragment into the pBAD-CsgA. 
</P><H3>Friday (07/14/17)</H3><P>-“I brought doughnuts!” exclaimed Andrea as she walked in the door in the
morning. The group had planned a Skype meeting with the U.S. Navy and U.S.
Army. After getting the connection set up, the students were able to have a
myriad of conversations from topics regarding to giant Jamboree to technical
components of the wiki. After some casual conversion, the students were able to
have a better understanding of the happenings at the Jamboree and how to improve the wiki
as well as Interlab components. </P><P><B>Sense/Respond: </B>
Next, the long part of the Interlab began. The students
tested for absorbance and fluorescence of the bacteria samples over 0,2,4, and 6 hours.
Later in the afternoon, the students did a dry run of their presentation in preparation for
Monday.</P><P><B>Packaging: </B>The entire iGEM team participated in a video conference with the Army and Navy iGEM teams in order to discuss similarities and issues with managing military teams. Afterwards at the lab, the RX team transformed the ligation into NEB 5-alpha competent cells and left the cells to grow in an incubator over the weekend. 
</P></DIV><DIV class="text2"><H2>Week 8: June 17-21</H2><H3>Monday (07/17/17)</H3><P><B>Sense/Respond: </B>The student sat around nervously waiting for
their presentation to begin. In a few minutes
they would be presenting to the whole lab
about their project and goals. The students
gave a phenomenal presentation that received
much praise and applause. They were relieved to be done
and went back to work on the wiki. Everything in the lab is
up to date so the young researchers dedicated the rest of
the day for wiki organization and miscellaneous tasks.</P><P><B>Packaging: </B>GoTaq Flexi PCR protocol was performed on 28 colonies of the transformants. After the PCR was completed, a gel was run in order to verify which of the chosen colonies contained both the <I>csgA</I> and the dCBD gene fragments. Based off of the PCR products, 6 of the 28 chosen colonies were grown on LB with ampicillin plates and broth in an incubator overnight. 

</P><H3>Tuesday (07/18/17)</H3><P><B>Sense/Respond: </B>Mentor Mike was finally back from vacation and the students were very excited
to see him and update him on current progress. They discussed the plan moving
forward and the layout of the rest of the
summer. After half an hour, the students had a
general idea of the &quot;end goal&quot;. They discussed
the DNA sequencing that had just come in and were like
described. Afterwards the students continued working on
collaboration and organizing wiki pages.</P><P><B>Packaging: </B>The six liquid culture samples were taken out of the incubator and 3ml from each tube was miniprepped for four restriction digests. The Nanodrop was used to check the concentration of DNA in each sample. After the digests were performed using Nco1, Kpn1, HindII, and Nco1 and HindIII, a gel was run to analyze the results. The results obtained were unexpected, but several samples, 14 and 32, were chosen as likely candidates. 
</P><TABLE><TBODY><TR><TH>Chosen Samples of
Prepped DNA</TH><TH>Concentration (ng/uL)</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TH>Sample 14</TH><TD>8.3</TD><TD>1.61</TD><TD>1.50</TD></TR><TR><TH>Sample 15</TH><TD>33.0</TD><TD>1.49</TD><TD>0.71</TD></TR><TR><TH>Sample 110</TH><TD>21.0</TD><TD>1.67</TD><TD>0.82</TD></TR><TR><TH>Sample 22</TH><TD>17.6</TD><TD>1.69</TD><TD>1.30</TD></TR><TR><TH>Sample 32</TH><TD>43.6</TD><TD>1.75</TD><TD>1.85</TD></TR><TR><TH>Sample 34</TH><TD>23.7</TD><TD>1.06</TD><TD>1.59</TD></TR></TBODY></TABLE><P class="caption2">Table 15: Nanodrop results</P><H3>Wednesday (07/19/17)</H3><P><B>Packaging: </B>
In the morning, the entire iGEM team met with mentors at RH to discuss the projects and Wiki. At the lab, the RXers miniprepped samples 14 and 32 of NEB 5-alpha/ pJAAM1. Later, restriction digests with Nco1, Kpn1, HindII, and Nco1 and HindII were performed on the miniprepped samples. Lastly, cultures of NEB 5-alpha/pJAAM 1 were diluted and spread on LB agar with ampicillin plates. 

</P><H3>Thursday (07/20/17)</H3><P><B>Sense/Respond: </B>The day started at UES and the students
began to look up literature that would help
them with the sensing part of the project goal.
They were able to find lots of useful
information referring to the two ETEC toxins, a
heat labile and heat stabile. Over the course of a couple
hours, the students were able to find and compose a list of
the possible ways to detect ETEC. In the afternoon, the
students shared the ideas with their mentor, Mike. After
lunch, the students discussed what they had gathered from
the past 3 days of researching literature. After exchanging
ideas and articles, the students and Dr. Goodson realized that being able to sense a quorum
sensing molecule autoinducer-2 (AI-2) might be the way to go. They discussed at length the
necessary components needed to be able to put the promoter into the existing plasmid so that
it could detect AI-2. After a bit of research, they decided to go with E. coli DH5a which fit
perfectly as it does not create AI-2 on its own. The students were relieved that they had taken a
massive step forward in their “sensing”
component and hope to achieve the end goal
soon.</P><P><B>Packaging: </B>A gel was run on the digested pJAAM1 and the results were again unexpected. To try a different approach, the RXers chose colonies from the diluted cultures of NEB 5-alpha/pJAAM1 and performed a PCR. The gel of the PCR products appeared correct, so the remainder of the chosen colonies were streaked onto plates and inoculated in culture tubes to grow overnight. 
</P><H3>Friday (07/21/17)</H3><P><B>Sense/Respond: </B>The students finally began the lab
work today for &quot;sensing&quot;. The
students worked on resuspending
part K18 in Well 3 and followed the
protocol to grow them up on plates.
They also transformed the GFPa1 plasmid into
DH5a E. coli cells. The plates were then left
outside overnight to grow. Meanwhile, the
young researchers were able to analyze the
plasmid and look at all the restriction sites to
find the ideal sites to cut the plasmid. Under the
guidance of Dr. Goodson, the students picked
three restriction sites and an order was put in to
order the primers. Afterwards, the RH lab held
their weekly lab meeting with the RX lab and
shared the exciting news of their new discovery.</P><P><B>Packaging: </B>The week ended with the RX team miniprepping three milliliters of the cultures from the previous day and digesting the results. However, when a gel was run on the digest, the results still appeared incorrect. Consequently, the RXers decided to return to a previous step in order to retry the ligation between pBAD-CsgA and dCBD. Previously digested samples of pBAD-CsgA with KpnI and HindIII were further cut with the same enzymes to achieve a more thorough digestion. 
</P></DIV><DIV class="text"><H2>Week 9: July 24-28</H2><H3>Monday (07/24/17)</H3><P><B>Sense/Respond: </B>The day was a bit more relaxed for the Monday morning. The weekly lab
meeting was scheduled and the students were able to learn a lot about other
researchers and their current studies. In addition, they learned how to present to
an audience and create simple but informational slides. Afterwards, they enjoyed
a team lunch together with mentors to discuss the Michigan meet up. The

ecstatic students exchanged travel ideas and a powerpoint presentation for the &quot;mini-
jamboree&quot;. Afterwards, the young researchers returned to the lab to pick colonies from Friday

to grow overnight in LB-Chl. They then left for the afternoon to get on wi-fi and worked on the
wiki and human outreach goals.</P><P><B>Packaging: </B>Lab Fun Day!
</P><H3>Tuesday (07/25/17)</H3><P><B>Sense/Respond: </B>The adventures continue the next morning as the team went back into the lab to
look at the overnight cultures. Tina and Hayley were extremely pleased to find
their cultures had grown perfectly and were ready to do the mini-prep and nano
drop. However, Dallas and Jason did not get favorable results. They were very
disappointed to find out that their cultures had not grown and would need to
restart and pick other colonies.</P><P><B>Packaging: </B>The RXers attended a branch meeting and then worked to develop presentations for the Journal Talk on Friday, and for Michigan State. Later, the team performed a gel extraction protocol on the digests from Friday and created a gel for gel electrophoresis on Wednesday.
</P><H3>Wednesday (07/25/17)</H3><P><B>Packaging: </B>The entire iGEM team met in the morning at RH to practice the Michigan State presentation and to discuss the trip. Later the RXers ran a gel of the gel extractions, as well as checking the concentrations with the Nanodrop. The results were not entirely desirable, likely due to the number of protocols performed with the samples, but a ligation was set up and performed. Vanessa helped the team gather pictures and begin work on their Friday presentation. </P><H3>Thursday (07/27/17)</H3><P>Another meeting was held to discuss the presentation and viability of the
Michigan State Trip. Students from both labs went over their combined
presentation and rehearsed thoroughly.</P><P><B>Sense/Respond: </B> Hayley and Tina prepared
cultures of GFPa1 and sfGFP from the plates. Dallas transformed the iGem
cultures but sadly failed. Dr. Goodson recommended that they regrow the plates
and proceed again tomorrow. Later in the day, the young researchers finalized their plan for
&quot;flat Stanley&quot; and the outreach survey they hope to do in the future.</P><P><B>Packaging: </B>To start the morning, the RXers spent a portion of time discussing the Friday presentation with Vanessa. Later the team performed a transformation of the ligation from the previous day and continued preparing for their presentation.
</P><H3>Friday (07/28/17)</H3><P>The morning began bright and early with
a tour of a C-17 cargo plane on base. It
was a great bonding experience for
teams from both labs as they took
pictures and learned about the C-17
aircraft. The RX group mentally prepared to present
their project later in the afternoon to some high-ups
and members of their lab. The RH Lab planned to
attend and give them support. The presentation
went very well and the RX lab members received a
stellar ovation.</P><P><B>Sense/Respond: </B>Later in the afternoon, the RH lab
would be preparing a mini-prep for the Well 3, 18K
competent cells provided by iGem. The students
had just found out yesterday that the cultures finally
worked and that they could start proceeding to the
next step, which was extremely exciting. As experts
in the lab now, the group was able to follow the
protocol with ease and completed the protocol
accurately and quickly.</P><P><B>Packaging: </B>Both the RH and the RX teams participated in a tour of a C-17. Afterwards, the RXers presented their project at a Journal Talk. Vanessa suggested troubleshooting by sending samples of pBAD-CsgA off for sequencing. Additionally, the University of of Michigan mutant strains of E. coli arrived. Once the preparation for sequencing was completed, the team met at UES for a bit before leaving for the weekend (Michigan State!)
</P></DIV><DIV class="text2"><H2>Week 10: July 31-August 4</H2><H3>Monday (07/31/17)</H3><P><B>Sense/Respond: </B>The morning started with the students performing a digest over the mini prep
from several days pervious. Two digests were preformed using different sets of
cut enzymes. Digest 1 was using the SpnI-HF and BmrI, while Digest 2 used the
cut enzymes PacI and BmrI. The Digest was then confirmed using a gel, which
showed the accurate number of base pairs.</P><P><B>Packaging: </B>The week started with the RXers streaking and growing cultures of certain colonies of the transformants from Friday. The samples of pBAD-CsgA were sent off for sequencing and a plan was drawn up to further troubleshoot and proceed with the project. 
</P><H3>Tuesday (08/01/17)</H3><P><B>Sense/Respond: </B>The students set up another run of their GFP Project. The students used a
Kinetics reading of the Plate Reader. The kinetics were taken over 24 hours, every
20 minutes. The students then worked on the wiki, trying to create a template.</P><P><B>Packaging: </B>The day began with developing a plan for troubleshooting the project throughout the week. Later, the RXers made up a stock of LB agar and LB agar with Kanamycin and miniprepped three milliliters of the liquid cultures from Monday. Restriction digests were performed on each miniprep and a gel was run, and one sample's results seemed hopeful. New restriction digests were performed on the sample, but the outcomes of the sequencing arrived and suggested that success with the new restriction digests would be unlikely. The sequencing results showed a fragment of the TOPO plasmid had cloned in before <I>csgA</I> in each pBAD-CsgA sample. Solutions of magnesium chloride, calcium chloride, and glycerol were made in order to perform a competent cell protocol on the mutant strains of E. coli, which were left in an incubator overnight to grow.
</P><H3>Wednesday (08/02/17)</H3><P>Today was an exciting day for the students as they got to perform their survey
at their high school. The iGEM students volunteered to help the high school on
schedule pick up day. As the other students were coming to grab their schedule
we would offer for them to fill out a pick survey on computers we had set up.
Over 200 students varying in with some teachers filled the survey. This was a

huge success for the students.</P><P><B>Packaging: </B>The RXers chose samples of the pBAD-CsgA containing TOPO plasmid to digest with NcoI in the hopes that the TOPO fragment would separate from the rest. A gel showed the digests to be successful, and a gel extraction was performed. The concentrations of the results from the gel extraction were checked using a Nanodrop. Lastly, the mutant strains with <I>csgA</I> and without <I>csgA</I> were each inoculated onto plates with LB and Kanamycin and with just LB respectively, and these plates were stored in an incubator overnight to grow.
</P><H3>Thursday (08/03/17)</H3><P><B>Sense/Respond: </B>Today the students got back to work on the sense plasmid. The students
performed a PCR, because finally their primers came in. The PCR product was
ran on the gel and the students got the results they were hoping for. Next, they
performed a MiniElute Reaction Cleanup Kit and then conducted a Nanodrop.
The Nanodrop were high concentrations of DNA. To end the day, the students

performed another digest using the PCR they had just purified.</P><TABLE><TBODY><TR><TH>260</TH><TH>280</TH><TH>Ratio</TH><TH>ng/ul</TH></TR><TR><TH>F1</TH><TD>7.408</TD><TD>3.974</TD><TD>1.86</TD><TD>370.4</TD></TR><TR><TH>F2</TH><TD>8.412</TD><TD>4.554</TD><TD>1.85</TD><TD>420.6</TD></TR><TR><TH>Blank</TH><TD>0.089</TD><TD>0.002</TD><TD>5.59</TD><TD>0.4</TD></TR></TBODY></TABLE><P class="caption2">Table 16: Nanodrop results</P><P><B>Packaging: </B>Using the gel extractions from Wednesday, the RXers performed four ligations. After two hours, part of the ligations were removed in order to complete several transformations, while the remaining ligations were left to incubate overnight. Colonies from the plates containing the University of Michigan strains were transferred to flasks containing LB Broth and LB Broth with Kanamycin.
</P><H3>Friday (08/04/17)</H3><P>The big day to present to the Chief Scientist of the lab had finally come. Early in
the day, the students had a dry presentation with their teachers and mentors.
They received criticism and praise for their work and presentation that they would
present later. They left in high spirits to return to the lab and begin the
purification, ligation, and transformation of the plasmid for the “sense”
component of the project.</P><P><B>Sense/Respond: </B>
The purification was then nanodroped and the students were
pleased with the results. The students then performed a ligation on the plasmid backbone
from the iGEM part and the sfGFP. The ligation seemed to go as planned and the students
hoped that the two sticky ends had connected. After completing the transformation, they got
ready to present to the Chief Scientist who got the funding for the iGem team. The
presentation went extremely well and the Chief Scientist was very impressed with their work
in the summer and the progress made. The students were finally able to let out a sigh of
relief as they finished their presentation successfully. Afterwards, they came back to the lab
to perform the transformation on the plasmid. The students followed the protocol exactly
and were able to achieve favorable results.</P><P><B>Packaging: </B>The entire iGEM team met at Carroll in the morning to explain the project to several teachers from the science department and to the principal. Later, the team presented to the chief scientist of the 711th. In the lab, a second transformation was performed using the ligations that were left overnight. Additionally, samples from the flasks containing the University of Michigan strains were prepared for storage in the negative eighty degree freezer.
</P></DIV><DIV class="text"><H2>Week 11: August 7-11</H2><H3>Monday (08/07/17)</H3><P><B>Sense/Respond: </B>Unfortunately the students were disheartened to learn that none of the plates
grew colonies. The students were at first shocked, but immediately started
tracing back to the different protocols and plasmids from the past few days.
Mentor Mike ultimately recommended them to run another gel on the PCR
product and the students diligently began the gel protocol. After a few hours, the
gel inspection seemed fine and the bands displayed exactly what the students expected.
Therefore Mentor Mike recommended that they re-do the ligation from Monday. The ligation
went very smoothly and the transformation will be done tomorrow.</P><P><B>Packaging: </B>Samples of pJAAM01 from early August were digested using Nco1 in the hope that the TOPO fragment would separate from the pBAD-CsgA-dCBD plasmid. A gel showed the digests to be successful, and a gel extraction was performed. The concentrations of the results from the gel extraction were checked using a Nanodrop. The RXers then set up ligations using the gel extractions and a gel was run to confirm the gel extractions. Colonies from the transformants of pBAD-CsgA from last week were patched on an agar plate and inoculated in culture tubes. Finally, plates were streaked using the freezer stocks of the University of Michigan strains and left to grow overnight.
</P><H3>Tuesday (08/08/17)</H3><P><B>Sense/Respond: </B>Today the LabPats were finally able to transform the plasmid from the ligation
yesterday. They followed the standard protocol and transformed the cells into E.
coli DH5a again. After that, the cells were ready to be plated like previously. This
time the students hope that the cells will grow colonies and indicate the
completion of their &quot;sense&quot; component. The students left the lab in high spirits and
headed out for wifi and collaboration space to discuss more wiki work and future work.</P><P><B>Packaging: </B> We weren't in the lab!
</P><H3>Wednesday (08/09/17)</H3><P><B>Sense/Respond: </B>The students eagerly went into the lab to check the plates from the
transformation. To the students dismay all the plates contained growth even the
blanks. The concerned students went to ask Mentor Mike, who help the student
decide to perform a PCR in order to run the product on a gel. They ran the gel
and got the desired band lengths for everything. They also noticed that some of
the colonies were already glowing green. With already putting in a long day of work the
students made overnights for the next day.</P><P><B>Packaging: </B> We weren't in the lab! 
</P><H3>Thursday (08/10/17)</H3><P><B>Sense/Respond: </B>Bright and early the students got their overnights out of the incubator. They then
made glycerol stocks and performed a mini prep, so the bacteria could be sent
for sequencing. In the afternoon, the students got to view a set of poster
presentations. This was especially helpful for the students, and gave them many
ideas on how to set up their poster.</P><P><B>Packaging: </B> We weren't in the lab! 
</P><H3>Friday (08/11/17)</H3><P><B>Sense/Respond: </B>Today was a sad day in the lab, as the summer was ending and the students
had to return to school. The team spent the morning taking their mentors to
breakfast, as a thank you for all they had done for us. The rest of the day was
spent at the lab. The students cleaned up their work benches, and made sure
everything was in order before they left the lab for the last time. The students said
their good byes to their coworkers, whom they had enjoyed many lunch breaks with.</P></DIV><P><B>Packaging: </B> We weren't in the lab!
</P></DIV><DIV class="text2"><H2>Parts Submission</H2><H3>(10/19-20/17)</H3><P>The parts were submitted according to the protocol provided by iGEM. Minipreped DNA was placed into their respective wells before being dried in a hood overnight. The next day, they were then packaged and shipped.</P></DIV></DIV></DIV></BODY></HTML>