<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tartu_TUIT_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tartu TUIT/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><HEADER class="header" data-stick="#reef"><NAV class="navigation" data-breakpoint="768"><UL class="menu menu-left"><LI class="menu-item-branch"><SPAN class="label">Project</SPAN></LI><LI><A href="/Team:Tartu_TUIT/Description">Description</A></LI><LI><A href="/Team:Tartu_TUIT/Design">Design</A></LI><LI><A href="/Team:Tartu_TUIT/Model">Modeling</A></LI><LI><A href="/Team:Tartu_TUIT/Parts">Parts</A></LI><LI><A href="/Team:Tartu_TUIT/Results">Results</A></LI></UL><LI class="menu-item-branch"><SPAN class="label">Laboratory</SPAN></LI><LI><A href="/Team:Tartu_TUIT/InterLab">InterLab</A></LI><LI><A href="/Team:Tartu_TUIT/Experiments">Experiments</A></LI><LI><A href="/Team:Tartu_TUIT/Notebook">Notebook</A></LI><LI><A href="/Team:Tartu_TUIT/Safety">Safety</A></LI><UL class="menu menu-right"><LI class="menu-item-branch"><SPAN class="label">People</SPAN></LI><LI><A href="/Team:Tartu_TUIT/Team">Team</A></LI><LI><A href="/Team:Tartu_TUIT/Attributions">Attributions</A></LI></UL><LI class="menu-item-branch"><SPAN class="label">Human Practices</SPAN></LI><LI><A href="/Team:Tartu_TUIT/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Tartu_TUIT/Public_Engagement">Public Engagement</A></LI><LI><A href="/Team:Tartu_TUIT/Collaborations">Collaborations</A></LI></NAV></HEADER></P><DIV class="title text-glow-blue">MODELING</DIV><SECTION class="article narrow"><DIV class="content"><H2>Model 1. The optimization of the shinorine and porphyra-334 yield</H2><P>The goal of our project is to obtain yeast extract enriched with MAAs shinorine and porphyra-334 by heterologous expression of the genes encoding for the pathway enzymes.  Shinorine and porphyra-334 are synthesized via four-reaction pathway catalyzed by 4 different enzymes.<A href="#ref-1">[1]</A><A href="#ref-2">[2]</A></P><P>The initial aim of the modelling was to predict the yield of the MAAs-producing pathway. To accomplish this aim, we have constructed a mathematical model using Matlab Simbiology tools (file attached).</P><P style="text-align: center">ODEs for all the reactions presented below were derived. The first reaction is monosubstrate while other three are bisubstrate:</P><OL class="fancy"><LI class=" half">sedoheptulose-7-phosphate⟶desmethyl-4-deoxygadusol + phosphate + H2O</LI><LI class=" half">desmethyl-4-deoxygadusol + S-adenosyl-L-methionine⟶4-deoxygadusol + S-adenosyl-L-homocysteine</LI><LI class=" half">4-deoxygadusol + glycine + ATP⟶4mycosporine-glycine + ADP + phosphate + H+</LI><LI class=" half">mycosporine-glycine  + ATP + serine/threonine ⟶4shinorine/porphyra-334 + ADP + phosphate+H2O</LI></OL><OL class="fancy"><LI class=" half">demethyl-4-deoxygadusol synthase</LI><LI class=" half">demethyl-4-deoxygadusol methyltransferase (O-MT family)</LI><LI class=" half">4-deoxygadusol glycyltransferase</LI><LI class=" half">D-ala-D-ala ligase (ATP-grasp)</LI></OL><P style="text-align: center">ODEs:</P><OL class="fancy"><LI class=" half">d[D4D]/dt=V<SUB>m</SUB>[S7D]/(K<SUB>m</SUB>+[S7P])</LI><LI class=" half">d[4D]/dt=V<SUB>m</SUB>[SAM][D4D]/K<SUB>s</SUB>K<SUB>m2</SUB>+K<SUB>m1</SUB>[SAM]+K<SUB>m2</SUB>[D4D]+[SAM][D4D]</LI><LI class=" half">d[M-Gly]/dt=V<SUB>m</SUB>[4D][ATP]/KsKm1+Km1[4D]+K<SUB>m2</SUB>[ATP]+[4D][ATP]</LI><LI class=" half">d[MAA]/dt=V<SUB>m</SUB>[M-Gly][ATP]/KV<SUB>s</SUB>KV<SUB>m1</SUB>+KV<SUB>m1</SUB>[M-Gly]+KV<SUB>m2</SUB>[ATP]+[M-Gly][ATP]</LI></OL><P>However, we have encountered a sudden problem: kinetic constants for the majority of our target enzymes are undetermined. We managed to find the constants for the mysA enzyme from the Nostoc punctiforme. Unfortunately, our attempts to find more information about the homological enzymes from similar organisms did not give any results. For our research, we used BRENDA, KEGG, UniProt databases. We have also contacted other researchers working in this field, but they were unable to provide the required information either <A href="https://2018.igem.org/Team:Tartu_TUIT/Human_Practices">IHP page</A>.</P><P>In the beginning, we expected the lack of information would still allow us to generate dependency graphs. However, there are too many unknown constants, and MATLAB was not able to find an analytical solution to all 4 ODEs.</P><P>However, we expect to develop this model further when our yeast strains are created, and we will measure the constants in the experiments.</P><H2>Model 1. The optimization of the shinorine and porphyra-334 yield</H2><P>One of the precursors of shinorine and porphyra-334 biosynthesis is sedoheptulose 7-phosphate (S7P). However, it was shown the concentration of S7P in the S. cerevisiae cells is low <A href="#ref-3">[3]</A>. It was reported in the literature that it is possible to increase the concentration of S7P in the yeast by genetic engineering <A href="#ref-4">[4]</A><A href="#ref-5">[5]</A>. Therefore, we have decided to model the effect of genetic manipulation on the concentration of S7P.</P><P style="text-align: center">We looked for the genes shown to affect the production of the S7P <A href="#ref-5">[5]</A>. The following information about the genes was retrieved from the open KEGG database:</P><OL class="fancy"><LI class=" half">Tal1 (transaldolase, the enzyme in the non-oxidative pentose phosphate pathway; converts sedoheptulose 7-phosphate and glyceraldehyde 3-phosphate to erythrose 4-phosphate and fructose 6-phosphate)</LI><LI class=" half"> Nqm1 (the paralog of Tal1)</LI><LI class=" half">Tkl1 and Tkl2 (Transketolase; catalyzes the conversion of xylulose-5-phosphate and ribose-5-phosphate to sedoheptulose-7-phosphate and glyceraldehyde-3-phosphate in the pentose phosphate pathway; needed for synthesis of aromatic amino acids)</LI><LI class=" half"> Shb17 (Sedoheptulose bisphosphatase involved in riboneogenesis; dephosphorylates sedoheptulose 1,7-bisphosphate)</LI><LI class=" half">Pgi1 (Phosphoglucose isomerase, catalyzes the interconversion of glucose-6-phosphate and fructose-6-phosphate)</LI><LI class=" half"> Pho13 (phosphatase, induces tal1, «PHO13 deletion-induced transcriptional activation prevents sedoheptulose accumulation during xylose metabolism in engineered Saccharomyces cerevisiae)</LI></OL>
				(figure 1)<P>Among those genes, Tal1, Nqm1, Tkl1, Tkl2 induce the consumption of S7P, while Shb17 induces the synthesis of S7P. None of these genes is vital for yeast cells. However, some double mutants and triple mutants show decreased viability or are even are non-viable (yeastgenome.org).</P><P>Our initial plan was to change the expression of these genes using promoters of different strength: weaker promoter for Tal1, Nqm1, Tkl1 and Tkl2 and stronger one for SHB17. It is also possible to use stronger promoter for the first gene from the MAA biosynthesis pathway to enhance flux towards the synthesis of shinorine and porphyra-334.</P><P>According to the literature, the zwf1 deletion increases the percentage of S7P produced by SHB17. Double null mutants tal1 nqm1 showed almost four times higher amount of S7P to be produced through SHB17. However, the triple mutant was less viable and showed decreased growth even in the rich media (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3163394/). It was also shown that tal1 pgi1 deletion significantly increases the yield of S7P. Using glucose as a carbon source further increased the S7P production<A href="#ref-4">[4]</A>. In addition, simultaneous deletion of tal1 and nqm1 increases the flux through Shb17<A href="#ref-5">[5]</A>.</P><P>Taking together all the information, in order to raise the S7P concentration in S. cerevisiae, we propose to increase the flux through SHB17 enzyme at the background of the deletion of the genes involved in the consumption of S7P. The latter can be accomplished by using promoters of different strength along with gene deletion.</P><P>Here is presented a logical pathway, that illustrates, how the expression of specific genes affects the concentration of S7P. Switches represent the presence and absence of the genes, while the red light represents the active production of the S7P.(figure2)</P><P>Between Tkl1 and Tkl2 we have put XOR gate since only the absence of one of the genes (not both) will increase the S7P concentration and simultaneously not reduce the growth rate significantly.</P><P>Pho13 and Pgi1 increase the influence of the deletion of Tal1; therefore, they are connected with the AND gate.</P><P>Nqm1 and Tal1 double mutants have a significantly changed flux of S7P; thus they are also connected with the AND gate.</P><P>However, XOR gate between these genes and Zwf1 symbolize less critical fluctuations in the triple mutant.</P><P> The following OR gate represents that both knockouts change S7P concentrations but not interrupt the influence of each other.</P><P>The last gate is AND that represents the accumulative effect of the deletions.</P><P>In our opinion, it would be beneficial to knock-down Tkl1, Tal1, Nqm1, Pho13, Pgi1.</P><P>The pathway was created with the help of an open online resource: <A href="https://sciencedemos.org.uk/logic_gates.php">https://sciencedemos.org.uk/logic_gates.php</A></P><P>Interactive version of the pathway can be opened using the file.</P><H3>References:</H3><OL class="references"><LI>Miyamoto, K. T., Komatsu, M., &amp; Ikeda, H. (2014). Discovery of gene cluster for mycosporine-like amino acid biosynthesis from Actinomycetales microorganisms and production of a novel mycosporine-like amino acid by heterologous expression. Applied and environmental microbiology, AEM-00727.</LI><LI>Katoch, M., Mazmouz, R., Chau, R., Pearson, L. A., Pickford, R., &amp; Neilan, B. A. (2016). Heterologous production of cyanobacterial mycosporine-like amino acids mycosporine-ornithine and mycosporine-lysine in E. coli. Applied and environmental microbiology, AEM-01632.</LI><LI>Senac, T., &amp; Hahn-Hägerdal, B. (1990). Intermediary metabolite concentrations in xylulose-and glucose-fermenting Saccharomyces cerevisiae cells. Applied and environmental microbiology, 56(1), 120-126.</LI><LI>Schaaff, I., Hohmann, S., &amp; Zimmermann, F. K. (1990). Molecular analysis of the structural gene for yeast transaldolase. European journal of biochemistry, 188(3), 597-603.</LI><LI>Clasquin, M. F., Melamud, E., Singer, A., Gooding, J. R., Xu, X., Dong, A., ... &amp; Rabinowitz, J. D. (2011). Riboneogenesis in yeast. Cell, 145(6), 969-980.</LI><LI>Stincone, A., Prigione, A., Cramer, T., Wamelink, M. M., Campbell, K., Cheung, E., ... &amp; Keller, M. A. (2015). The return of metabolism: biochemistry and physiology of the pentose phosphate pathway. Biological Reviews, 90(3), 927-963.</LI></OL></DIV></SECTION><H3>SPONSORS</H3></DIV></DIV></DIV></DIV></DIV></BODY></HTML>