"
Page
Discussion
View source
History
teams
Log in
 
Team:TUDelft/Color design
From 2008.igem.org
 
 
 
Contact In the news Acknowledgements References Links Downloads Protocols Brainstorm Meetings
Home
Research Proposal
Safety
Deliverables
Future Work
Temperature (input)
Overview
Analysis
Parts design I
Parts design II
Software
Testing
Results
Conclusions
Color (output)
Overview
Modeling
Parts design
Testing & Results
Conclusions
Ethics
Introduction
Methods
Macro ethics
Micro ethics
Discussion
Conclusions
References
Contents
1 Parts Design
1.1 Genes of the Color Pathway
1.2 Design
1.2.1 PCR
1.2.2 Expression system
1.3 Results
Parts Design
Genes of the Color Pathway
The enzymes necessary to produce colored Escherichia coli colonies will be isolated from E. coli genomic DNA and Saccharomyces cerevisiae cDNA. A total of eight enzymes are needed to produce FPP, for the rest of the pathway we will make use of BioBrick I742152 and I742161 to make sure colonies will turn red. Other colors (orange and yellow also see the Edinburgh 2007 wiki ) can be produced by adding other enzymes from the registry. Of the eight enzymes we will isolate three are E. coli enzymes (atoB, idi and ispA), while the other five are S. cerevisiae enzymes (ERG8, ERG12, ERG13, MVD1 and HMG2).
Design
To see all parts we are working on click here . All parts with number BBa_K115050 and higher are the parts involved in color synthesis.
PCR
To PCR the genes out of the E. coli genome, we've designed primers using the invitrogen website. These primers don't contain the biobrick prefix and suffix yet. When the first PCR on the genome has worked with Taq polymerase, we will try it with Pfx polymerase which has proofreading and is more reliable. On these products, we'll perform a new PCR, with attached biobrick prefix and suffix. This way we prevent interference of the prefix or suffix in our genomic PCR. Table 1 gives an overview of the sequences of all primers designed.
Table 1. Primers used for the PCRs during this project with the respective target genes. atoB2, idi2 and ispA2 are the primers used for touch down PCR with BioBrick pre- and suffix.
Target gene (name)
Sequence
idi2 F
GAATTCGCGGCCGCTTCTAGATGCAAACGGAACACGTCA
idi2 R
CTGCAGCGGCCGCTACTAGTATTATTATTTAAGCTGGGTAAATGCAGAT
atoB2 F
GAATTCGCGGCCGCTTCTAGATGAAAAATTGTGTCATCGTCAGT
atoB2 R
CTGCAGCGGCCGCTACTAGTATTATTAATTCAATCGTTCAATCACCAT
ispA2 F
GAATTCGCGGCCGCTTCTAGATGGACTTTCCGCAGCAA
ispA2 R
CTGCAGCGGCCGCTACTAGTATTATTATTTATTACGCTGGATGATGTAGTC
atoB F
AAATCTAGAAATGAAAAATTGTGTCATCGTCAG
atoB R
TTTACTAGTTTAATTCAATCGTTCAATCACCAT
ERG13 F
AAATCTAGAAAAGGTGCAGCATGAAACTCTC
ERG13 R
TTTACTAGTTGGGGGAAGATTATTTTTTAACATC
HMG2 F
AAATCTAGAAATGTCACTTCCCTTAAAAACGATAG
HMG2 R
TTTACTAGTTTATAATAATGCTGAGGTTTTACAGGG
ERG12 F
AAATCTAGATATGTCATTACCGTTCTTAACTTCTGC
ERG12 R
TTTACTAGTTTATGAAGTCCATGGTAAATTCGTG
ERG8 F
AAATCTAGAAATGTCAGAGTTGAGAGCCTTCA
ERG8 R
TTTACTAGTTTATTTATCAAGATAAGTTTCCGGATC
MVD1 F
AAATCTAGAAATGACCGTTTACACAGCATCC
MVD1 R
TTTACTAGTTTATTCCTTTGGTAGACCAGTCTTTG
idi F
AAATCTAGATATGCAAACGGAACACGTCA
idi R
TTTACTAGTTTATTTAAGCTGGGTAAATGCAGAT
ispA F
AAATCTAGAAATGGACTTTCCGCAGCA
ispA R
TTTACTAGTTTATTTATTACGCTGGATGATGTAGTC
BB F
TGCCACCTGACGTCTAAGAA
BB R
ATTACCGCCTTTGAGTGAGC
Expression system
As we need to drain FPP constantly (FPP is toxic to the cell in high concentrations), the idea is to produce a color or the direct precursor (Phytoene) at all temperatures. We want to express the enzymes of the FPP producing pathway constitutively, independent of temperature. There is a need however to 'tune' this expression to the consumption of the color pathway. In practice this means the eight enzymes necessary for FPP production will be expressed in one operon under the same promotor. However, the required strength of the promotor and ribosome binding site will have to be determined experimentally. The color enzymes (from FPP to lycopene, B-carotene and zeaxanthin) will be expressed seperately and under regulation of the RNA thermometers. As we will obtain three colors, this would mean two switches, for instance at 27ºC and 37ºC in the case of constitutively induced lycopene production. This means red colonies will form at all temperatures below 27ºC. However the enzyme for B-carotene production would be switched on (by loss of secondary structure of the RNA element) at 27ºC and between 27ºC and 37ºC E. coli colonies will be orange. Above 37ºC the zeaxanthin enzyme will be turned on and colonies will have a yellow color. If the basic molecule will be phytoene, there is room for three RNA thermometers, for example 27ºC, 32ºC and 37ºC. This would give the temperature scheme of:
T < 27ºC: 'E. coli color'
27ºC < T < 32ºC: Red
32ºC < T < 37ºC: Orange
37ºC < T : Yellow
Results
The first gradient PCR have been performed on the genes, the E. coli genes all worked correctly (see lab notebook entry of August 19th ), while still some work has to be done on the S. cerevisiae primers ( August 20th ). The next step will be to PCR the E. coli genes again, with the ideal annealing temperature, using Pfx polymerase instead of Taq. The Pfx enzyme has proofreading and so is less likely to make a mistake within the genes. The gradient PCR on S. cerevisiae cDNA will be repeated and optimized.
Team
Calendar
Parts
Lab notebook
Pictures
Sponsors
 
 
 
 
Visitors
 
 
Retrieved from " http://2008.igem.org/Team:TUDelft/Color_design "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
