"
Page
Discussion
View source
History
teams
Log in
 
Team:TUDelft/Color testing
From 2008.igem.org
 
 
 
Contact In the news Acknowledgements References Links Downloads Protocols Brainstorm Meetings
Home
Research Proposal
Safety
Deliverables
Future Work
Temperature (input)
Overview
Analysis
Parts design I
Parts design II
Software
Testing
Results
Conclusions
Color (output)
Overview
Modeling
Parts design
Testing & Results
Conclusions
Ethics
Introduction
Methods
Macro ethics
Micro ethics
Discussion
Conclusions
References
Contents
1 Color pathway testing
1.1 Correct genes testing
1.2 Correct activity testing
1.3 Results
Color pathway testing
Correct genes testing
The first indication of whether obtaining of the genes was a succes, is the correct size of DNA product on a gel. DNA sequencing will provide final certainty of the correct genes.
Correct activity testing
The ultimate proof of activity of the pathway would obviously be a colored Escherichia coli colony. However, to identify individual working enzymes, we'd need specific enzymatic assays. Furthermore this is useful, as some enzymatic assays could yield parameters for the modeling guys. Due to a lack of time, we've not looked any further into these assays.
Results
After we had our first succesful PCR on the genes of E. coli with Taq polymerase ( Lab notebook August 19th ) we tried to repeat it using Pfx polymerase, but this didn't work ( Lab notebook September 4th ). In the end we decided to repeat the experiment with Taq polymerase as the desired sequences were not very long, thus the chance for mutations due to the lack of proofreading of Taq polymerase was small. In the end the sequences will be verified through DNA sequencing. The gel of this PCR is displayed in figure 1. The S. cerevisiae genes showed no results.
Behind the gene names is mentioned what the size of the gene would be, idi is larger due to primer design.
Figure 1: Taq polymerase PCR of atoB (1188 bp), idi (552 bp) and ispA (903 bp)
On these PCR products, we did a touchdown PCR with new primers, which were designed to add the biobrick prefix and suffix to the gene. This is described here . The results of this experiment are displayed in figure 2.
Figure 2: Touchdown PCR with prefix/suffix primers of atoB (1188 bp), idi (552 bp) and ispA (903 bp)
The touchdown PCR product was cut with appropiate restriction enzymes and ligated into a biobrick construction vector.  The colony PCR of the transformants is displayed in figure 3 and figure 4. Behind the gene name the size + biobrick primer DNA (316 bp) is noted.
Figure 3: Colony PCR of idi (868 bp) and ispA (1219 bp). For prepping idi strain labeled 3 was used, for ispA strain labeled 4 was used.
Figure 4: Colony PCR of atoB(1504 bp). For prepping strain labeled 2 was used.
The miniprepped plasmids have been sent to Baseclear (NL) for sequencing and MIT for addition into the registry. For the sequences you can check our openwetware page.
Team
Calendar
Parts
Lab notebook
Pictures
Sponsors
 
 
 
 
Visitors
 
 
Retrieved from " http://2008.igem.org/Team:TUDelft/Color_testing "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
