"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Washington/Magnetosomes/GibsonVectors
From 2011.igem.org
Home
iGEM Homepage UW 2011 UW 2010 UW 2009 UW 2008 UW SynBio
About Us
The team Sponsors Americas Regional Jamboree
Diesel Production
Background Methods Results Summary Future Directions Parts Submitted
Gluten Destruction
Background Methods Results Summary Future Directions Parts Submitted
iGEM Toolkits
Background Gibson Assembly Toolkit Gibson Toolkit Results Magnetosome Toolkit Magnetosome Results Parts Submitted Future Directions
Data Page
Data Page Parts Registry
Protocols
Outreach
Local Outreach iGEM Collaborations
Safety
iGEM Toolkits: Gibson Assembly Vectors
About Gibson Assembly
Gibson assembly is a new synthetic biology tool that allows scar-free assembly of multiple gene-inserts in one isothermal reaction by using an exonuclease, polymerase, and heat-stable ligase to chew back, anneal, and repair gaps from homologous DNA fragments. This method is gaining popularity as it tends to be more efficient than standard restriction/ligation assembly, which can save a great amount of time in the cloning process. Overall, Gibson assembly allows teams to built large gene constructs with ease.
What happened last year?
The Gibson Cloning method is definitely not a new method to be introduced to the iGEM community.  In 2010, the Cambridge iGEM team created the BBF RFC 57 document which outlines a protocol for Gibson Assembly using standard BioBricks ( BBF RFC 10 ) that would allow many fragment inserts during a single cloning step. However, while creating the Magnetosome Toolkit, we found that this BioBrick standard was incapable of producing high yields of desired Gibson products even for two-fragment assemblies.
The "pSB" standard BioBrick vectors available through iGEM are not designed for efficient multiple-insert cloning beyond three fragments and are limited by ligation scars. The primary problem with a standard pSB vector is the self-complementarity of the two NotI sequences embedded in both the BioBrick prefix and suffix. These sequences prevent gene inserts from being incorporated efficiently, and do not produce a high yield of the Gibson product even in two-fragment assemblies. Generally, the backbone self-anneals and the recircularized plasmid has a combined prefix and suffix region that reads EcoRI-NotI-PstI.
To overcome this problem, the 2010 UW iGEM team developed new prefix and suffix regions that are based on BglBrick (BBF RFC 21 ) standard and designed to eliminate self-complementarity from the prefix and suffix of the plasmid. These vectors dramatically increase the Gibson assembly efficiency of large-scale gene assemblies and are also compatible with iGEM standard BioBrick parts.
What did we do this year?
Seeing that this is a very efficient method to do cloning, we continued to make improvements to the methods and created a Gibson Assembly Toolkit!
Creation of 5 plasmid vectors
Our new vectors for Gibson assembly follow the naming convention of pGA. To make our pGA vectors, we first amplified the backbones and the pLac GFP insert respectively. Then we performed a Gibson reaction to combine them together to make the pGA vectors.
All togeher, we created 5 pGA vectors and submitted them to the registry:
2 High copy extraction/cloning vectors
pGA1A3, pGA1C3
1 medium copy expression vector
pGA3K3
2 low copy expression vectors
pGA4A5, pGA4C5
As listed above, each of our vectors have varying copy numbers, antibiotic resistances, and purposes within the magnetosome gene assembly. However, they all appear to be very efficient will multiple gene inserts.
For experimental details comparing the efficiencies of pSB1A3 and pGA1A3, see our assay results . In addition to the characterization of assembly efficiency, we used the pGA vectors for the Magnetosome Toolkit project. First, we efficiently extracted and BioBricked all the 18 mamAB magnetosome genes. We also made super-assemblies from these BioBricks to make 10 kilobase plasmids (see mamHIEJKL and mamQRBSTUV ) from 2 kilobase pieces by assembling up to five fragments all in one cloning step!
References
1. Daniel G Gibson, Lei Young, Ray-Yuan Chuang, Craig J Venter, Clyde A Hutchison, and Hamilton O Smith. Enzymatic assembly of DNA molecules up to several hundred kilobases. Nat Methods, 6(5):343, Apr 2009.
2. Peter A Carr and George M Church. Genome Engineering. Nat Biotechnol , 27(12):1151 Dec 2009.
3. Daniel G Gibson, Hamilton O Smith, Clyde A Hutchison Iii, J Craig Venter, and Chuck Merryman. Chemical synthesis of the mouse mitochondrial genome. Nat Methods, 7(11):901, Oct 2010.
Retrieved from " http://2011.igem.org/Team:Washington/Magnetosomes/GibsonVectors "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
