<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Washington_Magnetosomes_GibsonVectors"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Washington/Magnetosomes/GibsonVectors">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Washington/Magnetosomes/GibsonVectors&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Washington/Magnetosomes/GibsonVectors&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Washington/Magnetosomes/GibsonVectors&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Washington/Magnetosomes/GibsonVectors" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Washington/Magnetosomes/GibsonVectors</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="ddwrapper"><DIV id="ddnav" align="center"><UL><LI><A href="https://2011.igem.org/Team:Washington">Home</A><DIV><A href="https://2011.igem.org/Main_Page"> iGEM Homepage </A><A href="https://2011.igem.org/Team:Washington">UW 2011</A><A href="https://2010.igem.org/Team:Washington"> UW 2010 </A><A href="https://2009.igem.org/Team:Washington"> UW 2009 </A><A href="https://2008.igem.org/Team:University_of_Washington"> UW 2008 </A><A href="http://synbio.washington.edu/"> UW SynBio </A></DIV></LI><LI><A href="https://2011.igem.org/Team:Washington/Team/Members">About Us</A><DIV><A href="https://2011.igem.org/Team:Washington/Team/Members">The team</A><A href="https://2011.igem.org/Team:Washington/Team/Sponsors"> Sponsors </A><A href="https://2011.igem.org/Team:Washington/Team/Americas"> Americas Regional Jamboree </A></DIV></LI><LI><A href="https://2011.igem.org/Team:Washington/Alkanes/Background">Diesel Production</A><DIV><A href="https://2011.igem.org/Team:Washington/Alkanes/Background">Background</A><A href="https://2011.igem.org/Team:Washington/Alkanes/Methods"> Methods </A><A href="https://2011.igem.org/Team:Washington/Alkanes/Results"> Results Summary </A><A href="https://2011.igem.org/Team:Washington/Alkanes/Future"> Future Directions </A><A href="https://2011.igem.org/Team:Washington/Alkanes/Parts"> Parts Submitted </A></DIV></LI><LI><A href="https://2011.igem.org/Team:Washington/Celiacs/Background">Gluten Destruction</A><DIV><A href="https://2011.igem.org/Team:Washington/Celiacs/Background">Background</A><A href="https://2011.igem.org/Team:Washington/Celiacs/Methods"> Methods </A><A href="https://2011.igem.org/Team:Washington/Celiacs/Results"> Results Summary</A><A href="https://2011.igem.org/Team:Washington/Celiacs/Future"> Future Directions </A><A href="https://2011.igem.org/Team:Washington/Celiacs/Parts"> Parts Submitted </A></DIV></LI><LI><A href="https://2011.igem.org/Team:Washington/Magnetosomes/Background">iGEM Toolkits</A><DIV><A href="https://2011.igem.org/Team:Washington/Magnetosomes/Background"> Background </A><A href="https://2011.igem.org/Team:Washington/Magnetosomes/GibsonVectors">Gibson Assembly Toolkit </A><A href="https://2011.igem.org/Team:Washington/Magnetosomes/GibsonResults">Gibson Toolkit Results</A><A href="https://2011.igem.org/Team:Washington/Magnetosomes/Magnet_Toolkit">Magnetosome Toolkit</A><A href="https://2011.igem.org/Team:Washington/Magnetosomes/Magnet_Results">Magnetosome Results</A><A href="https://2011.igem.org/Team:Washington/Magnetosomes/Parts"> Parts Submitted </A><A href="https://2011.igem.org/Team:Washington/Magnetosomes/Future"> Future Directions </A></DIV></LI><LI><A href="https://2011.igem.org/Team:Washington/Parts">Data Page</A><DIV><A href="https://2011.igem.org/Team:Washington/Parts">Data Page</A><A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2011&amp;group=Washington"> Parts Registry </A></DIV></LI><LI><A href="https://2011.igem.org/Team:Washington/Protocols">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Washington/Outreach">Outreach</A><DIV><A href="https://2011.igem.org/Team:Washington/Outreach">Local Outreach</A><A href="https://2011.igem.org/Team:Washington/Outreach/iGEM_Collaborations">iGEM Collaborations</A></DIV></LI><LI><A href="https://2011.igem.org/Team:Washington/Safety">Safety</A></LI></UL></DIV></DIV><CENTER><BIG><BIG><BIG><BIG><B>iGEM Toolkits: Gibson Assembly Vectors</B></BIG></BIG></BIG></BIG></CENTER><H3><SPAN class="mw-headline" id="About_Gibson_Assembly"> About Gibson Assembly</SPAN></H3><P>Gibson assembly is a new synthetic biology tool that allows scar-free assembly of multiple gene-inserts in one isothermal reaction by using an exonuclease, polymerase, and heat-stable ligase to chew back, anneal, and repair gaps from homologous DNA fragments. This method is gaining popularity as it tends to be more efficient than standard restriction/ligation assembly, which can save a great amount of time in the cloning process. Overall, Gibson assembly allows teams to built large gene constructs with ease. 
</P><H3><SPAN class="mw-headline" id="What_happened_last_year.3F"> What happened last year?</SPAN></H3><P>The Gibson Cloning method is definitely not a new method to be introduced to the iGEM community. 
In 2010, the Cambridge iGEM team created the <A href="http://www.cambridgeigem.org/RFC57.pdf" class="external text" rel="nofollow">BBF RFC 57</A> document which outlines a protocol for Gibson Assembly using standard BioBricks (<A href="http://dspace.mit.edu/bitstream/handle/1721.1/45138/BBFRFC10.txt?sequence=1" class="external text" rel="nofollow">BBF RFC 10</A>) that would allow many fragment inserts during a single cloning step. However, while creating the Magnetosome Toolkit, we found that this BioBrick standard was incapable of producing high yields of desired Gibson products even for two-fragment assemblies.
</P><P>The &quot;pSB&quot; standard BioBrick vectors available through iGEM are not designed for efficient multiple-insert cloning beyond <A href="http://partsregistry.org/Assembly:3A_Assembly" class="external text" rel="nofollow">three fragments</A> and are limited by ligation scars. The primary problem with a standard pSB vector is the self-complementarity of the two NotI sequences embedded in both the BioBrick prefix and suffix. These sequences prevent gene inserts from being incorporated efficiently, and do not produce a high yield of the Gibson product even in two-fragment assemblies. Generally, the backbone self-anneals and the recircularized plasmid has a combined prefix and suffix region that reads EcoRI-NotI-PstI. </P><P>To overcome this problem, the <A href="https://2010.igem.org/Team:Washington/Tools_Used/Next-Gen_Cloning" class="external text" rel="nofollow">2010 UW iGEM</A> team developed new prefix and suffix regions that are based on BglBrick (BBF <A href="http://dspace.mit.edu/handle/1721.1/46747" class="external text" rel="nofollow">RFC 21</A>) standard and designed to eliminate self-complementarity from the prefix and suffix of the plasmid. These vectors <A href="https://2011.igem.org/Team:Washington/Magnetosomes/GibsonResults" class="external text" rel="nofollow">dramatically</A> increase the Gibson assembly efficiency of large-scale gene assemblies and are also compatible with iGEM standard BioBrick parts. </P><H3><SPAN class="mw-headline" id="What_did_we_do_this_year.3F"> What did we do this year?</SPAN></H3><P>Seeing that this is a very efficient method to do cloning, we continued to make improvements to the methods and created a <B>Gibson Assembly Toolkit</B>! </P><H3><SPAN class="mw-headline" id="Creation_of_5_plasmid_vectors">Creation of 5 plasmid vectors</SPAN></H3><P>Our new vectors for Gibson assembly follow the naming convention of pGA. To make our pGA vectors, we first amplified the backbones and the pLac GFP insert respectively. Then we performed a Gibson reaction to combine them together to make the pGA vectors. 
</P><P>All togeher, we created 5 pGA vectors and submitted them to the registry:
</P><UL><LI> 2 High copy extraction/cloning vectors
</LI><LI> pGA1A3, pGA1C3
</LI></UL><LI> 1 medium copy expression vector
</LI><LI> pGA3K3
</LI><LI> 2 low copy expression vectors
</LI><LI> pGA4A5, pGA4C5
</LI><P>As listed above, each of our vectors have varying copy numbers, antibiotic resistances, and purposes within the magnetosome gene assembly. However, they all appear to be very efficient will multiple gene inserts.

For experimental details comparing the efficiencies of pSB1A3 and pGA1A3, see <A href="https://2011.igem.org/Team:Washington/Magnetosomes/GibsonResults" class="external text" rel="nofollow">our assay results</A>. In addition to the characterization of assembly efficiency, we used the pGA vectors for the <A href="https://2011.igem.org/Team:Washington/Magnetosomes/Magnet_Toolkit" class="external text" rel="nofollow">Magnetosome Toolkit</A> project. First, we efficiently extracted and BioBricked all the 18 <I>mamAB</I> magnetosome genes. We also made super-assemblies from these BioBricks to make 10 kilobase plasmids (see <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K590017" class="external text" rel="nofollow">mamHIEJKL</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K590019" class="external text" rel="nofollow">mamQRBSTUV</A>) from 2 kilobase pieces by assembling up to five fragments all in one cloning step! 
</P><H4><SPAN class="mw-headline" id="References">References</SPAN></H4><P>1. Daniel G Gibson, Lei Young, Ray-Yuan Chuang, Craig J Venter, Clyde A Hutchison, and Hamilton O Smith. Enzymatic assembly of DNA molecules up to several hundred kilobases. <I>Nat Methods</I>, <B>6</B>(5):343, Apr 2009.
2. Peter A Carr and George M Church. Genome Engineering. <I>Nat Biotechnol</I> , <B>27</B>(12):1151 Dec 2009.
3. Daniel G Gibson, Hamilton O Smith, Clyde A Hutchison Iii, J Craig Venter, and
Chuck Merryman. Chemical synthesis of the mouse mitochondrial genome. <I>Nat Methods</I>, <B>7</B>(11):901, Oct 2010.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Washington/Magnetosomes/GibsonVectors">http://2011.igem.org/Team:Washington/Magnetosomes/GibsonVectors</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Washington/Magnetosomes/GibsonVectors" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Washington/Magnetosomes/GibsonVectors" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Washington/Magnetosomes/GibsonVectors&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Washington/Magnetosomes/GibsonVectors&amp;oldid=197446" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>