<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Washington_Alkanes_Future_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Washington/Alkanes/Future/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Washington/Alkanes/Future/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Washington/Alkanes/Future/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Washington/Alkanes/Future/Modeling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Washington/Alkanes/Future/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Washington/Alkanes/Future/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="ddwrapper"><DIV id="ddnav" align="center"><UL><LI><A href="https://2011.igem.org/Team:Washington">Home</A><DIV><A href="https://2011.igem.org/Main_Page"> iGEM Homepage </A><A href="https://2011.igem.org/Team:Washington">UW 2011</A><A href="https://2010.igem.org/Team:Washington"> UW 2010 </A><A href="https://2009.igem.org/Team:Washington"> UW 2009 </A><A href="https://2008.igem.org/Team:University_of_Washington"> UW 2008 </A><A href="http://synbio.washington.edu/"> UW SynBio </A></DIV></LI><LI><A href="https://2011.igem.org/Team:Washington/Team/Members">About Us</A><DIV><A href="https://2011.igem.org/Team:Washington/Team/Members">The team</A><A href="https://2011.igem.org/Team:Washington/Team/Sponsors"> Sponsors </A><A href="https://2011.igem.org/Team:Washington/Team/Americas"> Americas Regional Jamboree </A></DIV></LI><LI><A href="https://2011.igem.org/Team:Washington/Alkanes/Background">Diesel Production</A><DIV><A href="https://2011.igem.org/Team:Washington/Alkanes/Background">Background</A><A href="https://2011.igem.org/Team:Washington/Alkanes/Methods"> Methods </A><A href="https://2011.igem.org/Team:Washington/Alkanes/Results"> Results Summary </A><A href="https://2011.igem.org/Team:Washington/Alkanes/Future"> Future Directions </A><A href="https://2011.igem.org/Team:Washington/Alkanes/Parts">Parts Submitted </A></DIV></LI><LI><A href="https://2011.igem.org/Team:Washington/Celiacs/Background">Gluten Destruction</A><DIV><A href="https://2011.igem.org/Team:Washington/Celiacs/Background">Background</A><A href="https://2011.igem.org/Team:Washington/Celiacs/Methods"> Methods </A><A href="https://2011.igem.org/Team:Washington/Celiacs/Results"> Results Summary</A><A href="https://2011.igem.org/Team:Washington/Celiacs/Future"> Future Directions </A><A href="https://2011.igem.org/Team:Washington/Celiacs/Parts"> Parts Submitted </A></DIV></LI><LI><A href="https://2011.igem.org/Team:Washington/Magnetosomes/Background">iGEM Toolkits</A><DIV><A href="https://2011.igem.org/Team:Washington/Magnetosomes/Background"> Background </A><A href="https://2011.igem.org/Team:Washington/Magnetosomes/GibsonVectors">Gibson Assembly Toolkit </A><A href="https://2011.igem.org/Team:Washington/Magnetosomes/GibsonResults">Gibson Toolkit Results</A><A href="https://2011.igem.org/Team:Washington/Magnetosomes/Magnet_Toolkit">Magnetosome Toolkit</A><A href="https://2011.igem.org/Team:Washington/Magnetosomes/Magnet_Results">Magnetosome Results</A><A href="https://2011.igem.org/Team:Washington/Magnetosomes/Parts"> Parts Submitted </A><A href="https://2011.igem.org/Team:Washington/Magnetosomes/Future"> Future Directions </A></DIV></LI><LI><A href="https://2011.igem.org/Team:Washington/Parts">Data Page</A><DIV><A href="https://2011.igem.org/Team:Washington/Parts">Data Page</A><A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2011&amp;group=Washington"> Parts Registry </A></DIV></LI><LI><A href="https://2011.igem.org/Team:Washington/Protocols">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Washington/Outreach">Outreach</A><DIV><A href="https://2011.igem.org/Team:Washington/Outreach">Local Outreach</A><A href="https://2011.igem.org/Team:Washington/Outreach/iGEM_Collaborations">iGEM Collaborations</A></DIV></LI><LI><A href="https://2011.igem.org/Team:Washington/Safety">Safety</A></LI></UL></DIV></DIV><CENTER><BIG><BIG><BIG><BIG><B>Diesel Production: System Modeling, Flux Balance Analysis</B></BIG></BIG></BIG></BIG></CENTER><H1><SPAN class="mw-headline" id="Background"> Background </SPAN></H1><P>While in vivo analysis is invaluable, computational modeling can help direct experiments in a way that conserves time and resources.  Current in silico models often rely on flux balance analysis (FBA) as an optimization algorithm.  FBA provides a method for tracking fluxes within a biological system and “analyz[ing] biological networks in a quantitative manner” [1].  A flow chart of FBA methodology is depicted in Figure 1 [1].
</P><P>FBA begins with the construction of an often complex biological network.  After accounting for all reactions within a system, FBA involves the construction of a stoichiometric matrix (N) where each column corresponds to a chemical reaction within the system and each row a molecular species.  The matrix is then multiplied by a vector of internal and exchange fluxes (v), shown in Equation 1.
</P><PRE>      {1}   dx/dt = Nv
</PRE><P>Since FBA is a static model, we assume a steady-state and therefore Equation 2, applies.  The overall equation simplifies to Equation 3.
</P><PRE>      {2}   dx/dt = 0   
      {3}   Nv = 0
</PRE><P>Constraints are applied to the stoichiometric matrix to limit the number of possible solutions.  These constraints often depend on the uptake rate and availability of nutrients within the media in which the cells are growing. A cell cannot take in a substrate if it is not available within its environment.  After determining the constraints, an objective is defined, such as maximized protein production or maximized cell growth.  Then, the fluxes that maximize the objective are mathematically calculated [1].
</P><H1><SPAN class="mw-headline" id="Methods"> Methods </SPAN></H1><P>The modeling project began with an existing metabolic network for the MG1655 strain of E. coli (see Appendix).  This model was obtained through the in silico organism database of Professor Bernard Palsson, University of California: San Diego [2].  The alkane producing reactions were then added to the metabolic network.  These reactions were simplified  (see below) for the FBA system in order to minimize the number of new metabolites, and thus minimize the number of dead ends in the network.  In particular, we only considered the major C16 alkane product in the model. The simplified reactions are:
</P><P>
To then limit the number of solutions, constraints were applied to the system.  The first of which concerned the uptake rate of glucose.  For this value, we referred to a PNAS article, where researchers had already determined the needed rate for E. coli cells growing in a similar media [3].  Then, using the exact media composition used for this process, the exchange rate of thiamine between the extracellular space and the cytoplasm was adjusted such that the uptake rate of glucose by mass was equal to the uptake of thiamine by mass.  Linear programming, the optimization process used by FBA, was then applied to determine the solution of fluxes for this system. For this process we used the COBRA Toolbox, which is a MATLAB application for FBA [4]. 
</P><P>
With the model working, the next step was to apply modifications to optimize alkane production.   Initial data showed that with the objective of the cell being growth, the in silico fluxes through the alkane producing pathway were zero.  We propose that this observation could be explained in one of two ways.  The hydrophobic nature of the alkane means that it makes its way out of the cell into the surrounding media and therefore does not contribute to biomass, the overall objective we used.  Also, both reactions we added rely on NADPH therefore it is possible that the model is unable to account for the additional use of NADPH and predicts a zero alkane flux. This zero flux may also be due to the scale set by the biomass function.  The growth of the cell, compared to the production of alkane, is very large.  Since this system is centered about growth, the FBA algorithm may consider the flux through the alkane producing pathway insignificant.  This observation resulted in all following modifications focusing on maximizing production of Acyl-ACP to thus increase the probability of the metabolites entering the alkane producing pathway.
</P><P>
To investigate these problems further we used GDLS, a MATLAB function that suggests possible gene knockouts to maximize a certain flux [4]. We used this technique to determined which knockouts would optimize production of Acyl-ACP.  This work proposed the deletion of gene B1249, which encodes for cardiolipin synthase 1.  The in silico deletion of this gene forced reactions that involved catabolizing of Acyl-ACP into smaller carrier proteins to zero (see Appendix for the raw data).  The deletion also significantly decreased the biomass function, which suggests further revisions should be made. However, this is a <B>prediction</B> can could be tested in the future.
</P><H1><SPAN class="mw-headline" id="Future_Modifications"> Future Modifications </SPAN></H1><P>While some preliminary in silico data was acquired, this model requires revisions, such as further integration of the alkane producing pathway into the existing network. One possibility is to modify the existing model by using other variants that have been published to investigate whether redox limitation is an issue.  Also, significant testing of this system, in particular using C13 tracers to actually determine the fluxes internal to the cell, could test the accuracy of this model.  With the needed adjustments made more predictive suggestions for maximizing alkane production could be constructed and thus a better product could be made in vivo.  Overall, while the model has not yet significantly affect experimental design, it points the way towards new experiments that would help improve our understanding and increase aklane production.
</P><H1><SPAN class="mw-headline" id="References"> References </SPAN></H1><P>[1] J. M. Lee, E. P. Gianchandani, and J. A. Papin, “Flux balance analysis in the era of metabolomics,” Briefings in bioinformatics, vol. 7, 2006, pp. 140-50.
</P><P>[2] Orth, Jeff. &quot;In Silico Organisms - Systems Biology Research Group.&quot; Systems Biology Research Group :: UCSD. Web. 28 Oct. 2011. &lt;<A href="http://gcrg.ucsd.edu/In_Silico_Organisms" class="external free" rel="nofollow">http://gcrg.ucsd.edu/In_Silico_Organisms</A>&gt;.
</P><P>[3] Causey, T. B., K. T. Shanmaugam, L. P. Yomano, and L. O. Ingram. &quot;Engineering Escherichia Coli for Efficient Conversion of Glucose to Pyruvate.&quot; Proceedings of the National Academy of Sciences 101.8 (2004): 2235-240.
</P><P>[4] S. A. Becker et al., “Quantitative prediction of cellular metabolism with constraint-based models: the COBRA Toolbox,” Nature Protocols, 2(3), 2007, pp. 727-738.
</P><H1><SPAN class="mw-headline" id="Appendix"> Appendix </SPAN></H1><P>1. E. Coli metabolic network    <A href="/wiki/images/f/f2/Ec_iJR904_GlcMM.txt" class="internal" title="Ec iJR904 GlcMM.txt">Media: Ec_iJR904_GlcMM.txt</A></P><P>2. MATLAB code for FBA    <A href="/wiki/images/c/cb/Matlab_Script.m" class="internal" title="Matlab Script.m">Media: Matlab_Script.m</A></P><P>3. Solution: all fluxes <B>without</B> gene knock out <A href="/wiki/images/f/f1/AllFluxes.txt" class="internal" title="AllFluxes.txt">Media: allFluxes.txt</A></P><P>4. Solution: nonzero fluxes <B>without</B> gene knock out <A href="/wiki/images/c/c3/NonzeroFluxes.txt" class="internal" title="NonzeroFluxes.txt">Media: nonzeroFluxes.txt</A></P><P>5. Solution: all fluxes <B>with</B> gene knock out <A href="/wiki/images/0/04/AllFluxes_geneKnockout.txt" class="internal" title="AllFluxes geneKnockout.txt">Media: allFluxes_geneKnockout.txt</A></P><P>6. Solution: nonzero fluxes <B>with</B> gene knock out <A href="/wiki/images/c/c5/NonzeroFluxes_geneKnockout.txt" class="internal" title="NonzeroFluxes geneKnockout.txt">Media: nonzeroFluxes_geneKnockout.txt</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Washington/Alkanes/Future/Modeling">http://2011.igem.org/Team:Washington/Alkanes/Future/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Washington/Alkanes/Future/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Washington/Alkanes/Future/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Washington/Alkanes/Future/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Washington/Alkanes/Future/Modeling&amp;oldid=260610" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>