"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Washington/Outreach/iGEM Collaborations
From 2011.igem.org
Home
iGEM Homepage UW 2011 UW 2010 UW 2009 UW 2008 UW SynBio
About Us
The team Sponsors Americas Regional Jamboree
Diesel Production
Background Methods Results Summary Future Directions Parts Submitted
Gluten Destruction
Background Methods Results Summary Future Directions Parts Submitted
iGEM Toolkits
Background Gibson Assembly Toolkit Gibson Toolkit Results Magnetosome Toolkit Magnetosome Results Parts Submitted Future Directions
Data Page
Data Page Parts Registry
Protocols
Outreach
Local Outreach iGEM Collaborations
Safety
Community Outreach: iGEM Collaborations
Primer Design Tool with LMU-Munich
We worked with LMU-Munich 2011 iGEM team this summer on the testing and development of their PrimerDesign tool.  To learn more about it check out their website .
An example of the output from the tool based off of input from our luxC biobrick, multiple primers with various melting temperatures are given
Software Testing
We helped LMU-Munich test their primer design software by sending our primer data and comparing it with the output of their automated designer. To do this, we compared the sequences of the primers used to amplify out parts of the LuxBrick with the primers that their software designed and concluded that their software works as expected.  Unfortunately, we did not have enough time to order the primers that the software designed to test it ourselves. Overall, we found the tool easy to use and potentially very useful for future primer design.
Implemented Bug Fixes & Features
Added cyanobacteria to available genomes
Switch coding sequences in the primer designer to allow for the user to incude a stop codon in a coding sequence. Before, software insisted that coding sequences not include the stop codon
Made primer designer default to only check for biobrick restriction enzymes. Before, Primer Designer defaulted to checking  for all restriction sites, making the user scroll to the bottom of the page for results
Added support for FASTA files with headers
Added descriptions for each column of the Primer Designer output.
Original vs. Revised Output
Original Output
The original output identified every commercially available enzyme recognition site by default.  This made it annoying to reach the bottom of the page where the primer info was.
Revised Output
New, more useful default output with only searching for BioBrick enzymes, and providing labels in the output
Retrieved from " http://2011.igem.org/Team:Washington/Outreach/iGEM_Collaborations "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
