"
Page
Discussion
View source
History
teams
Log in
 
Team:Waterloo
From 2013.igem.org
Controlled Modification and Intercellular Transmission of a DNA Message
Intercellular messaging in nature allows cells to coordinate and exhibit complex population-level behaviour. Synthetic biologists would like to build this into engineered populations.  Last year, Ortiz and Endy at Stanford University developed a method for messaging using DNA - the master information molecule - to carry information between cells using hijacked M13 bacteriophage. Their proof of principle demonstration opens up an exciting avenue for intercellular messaging and is ripe for development.   We designed methods to advance this intercellular communication system by allowing cells to CONTROL, MODIFY, and RETRANSMIT DNA messages.
)
Introduction
Intercellular Communication and DNA Messaging
Intercellular communication between cells in nature allows for coordinated population-level behavior, enabling spatial and temporal organization and complex responses to environmental stimuli.
Synthetic biology is continually expanding the range of programmable cellular characteristics and behaviors, and incorporation of intercellular communication into engineered cell populations has extended programmable behavior to a population level.
The Traditional Approach: Engineered AHL-based intercellular communication
Many bacteria naturally secrete acylated homoserine lactone molecules, or AHLs, which can be detected by other members of the bacterial population (Fig 1). Concentration-sensitive detection of AHL brings about significant qualitative changes in cell behavior via transcription regulation, including feedback on AHL production. Such natural quorum sensing systems are associated with coordinated behaviours such as biofilm formation [1] and bioluminescence [2].
These natural bacterial quorum-sensing systems have been successfully modulated to enable programmed intercellular communication in engineered bacterial populations. Genes associated with AHL production, detection, and response are “re-wired” such that they correspond to different input stimuli and output behaviors. Coordinated population-level behaviors including two-dimensional pattern formation [3], coordinated oscillations in gene expression [4], and even a system exhibiting predator-prey dynamics [5] have been demonstrated using this technique.
Fig 1. AHL molecules secreted by bacterial cells are detected by receptors in other bacterial cells
Why DNA messaging?
Ortiz and Endy suggest the use of DNA as an intercellular messaging molecule [6] (Fig 2). They identify several inherent advantages of this as compared to AHL-based messaging, which are summarized here in description of the DIVERSITY and INFORMATION CONTENT of AHL- vs DNA-mediated messaging.
DIVERSITY: Stimulation of an AHL receptor can only afford receipt of one or a few different messages, corresponding to concentrations of the receptor’s cognate AHL molecule. To enable receipt of a wider variety of messages, more receptor/AHL pairs must be included. In contrast, transmission of DNA molecules would afford receipt of a range of arbitrary different messages through sequence variation, presuming the mechanism of DNA molecule transmission does not restrict the DNA sequence [6].
INFORMATION CONTENT: AHL-based messaging acts by regulating transcription of genes that are already present in the receiver cell. Thus, the information in the message is always a specification of which genes should be regulated. In contrast, a DNA molecule transmitted to a receiver cell could introduce arbitrary new genes or genetic circuits, including, but not limited to, transcription factors that regulate expression in the receiver.
The main point here is that DNA, as the master information molecule, is inherently a better-suited molecule for information exchange than AHL.
To improve in these areas, Ortiz and Endy designed and demonstrated a “DNA Messaging” communication system [6]. These “DNA messages” are carried between E. coli cells inside a hijacked M13 bacteriophage particle: through a cunning act of trickery, M13 viral proteins are deceived into packaging the DNA message inside infectious particles instead of the viral genome (keep reading for details).
Fig 2. DNA messaging would allow DNA to be exchanged between cells as a form of communication.
The Nuts and Bolts of DNA Messaging
Ortiz and Endy [6] designed a system for effective DNA messaging wherein DNA messages are packaged in M13 bacteriophage particles, dispatched from an E. coli “sender population”, and delivered to an E. coli “receiver population” via the M13 infection mechanism.
In nature, M13 proteins package the M13 genome into bacteriophage particles through recognition of the M13 packaging sequence (M13 ori). As it turns out, any circular DNA bearing the M13 packaging sequence will be recognized by the M13 proteins and packaged into a bacteriophage particle. Removing the M13 packaging sequence from the M13 genome and placing it on a plasmid, which is then called a “phagemid”, causes the phagemid DNA to be packaged instead of the M13 DNA. A version of the M13 genome that does not get packaged itself due to a missing packaging sequence is called a “helper plasmid”.
A phagemid carrying a DNA message is referred to as a “messaging phagemid” and will be transmitted from sender cells that carry a helper plasmid
DNA of arbitrary length can be transmitted: M13 viral packaging occurs at the cell membrane, where the viral particle forms around the DNA as it is packaged and slowly secreted, forming a long filament. This allows DNA of arbitrary length to be packaged [7].
Sender cells continue to grow: M13 is not a lytic bacteriophage. Since M13 viral particles are secreted through the cell membrane, infected cells are able to continue living and dividing, albeit at ½ to ¾ their normal rate [7]. Because of this, cells sending a DNA message need not commit suicide to transmit their message!
Only F+ cells can be infected: Infection by M13 bacteriophage requires attachment to the E. coli F pilus. Therefore, only E. coli cells carrying the F plasmid (F+ cells) are susceptible, while F- cells are not.
The basic scheme of DNA messaging is seen in Figure 3. This method was established in 2012 when Ortiz and Endy demonstrated transmission and receipt of a DNA message encoding GFP and ampicillin resistance, as well as a separate message encoding T7 RNA polymerase [6]. Their proof of principle demonstration indicates the viability of DNA messaging and suggests extension of the method to diversify potential communication programs.
Figure 3. Sender cells contain a messaging phagemid and a helper plasmid, which allows them to secrete viral particles. Receiver cells must be F+.  When the two cell populations are co-cultured, DNA messaging takes place.
The Idea
We are fascinated by the idea of DNA messaging. Since this intercellular communication method [6] is so new, there is room for advancement of the method. We have identified ways to advance the system. We believe that in advanced DNA messaging, a DNA message should be:
Controllable: A DNA message should not necessarily be constitutively transmitted, but should rather be controllable by a particular stimulus.
Modifiable: Established methods of DNA recombination should be available for modification of a DNA message by senders or receivers. In particular, recent advances in DNA digital memory and logic [X,X] should be incorporated into DNA messaging.
Retransmittable: A receiver cell should be able to retransmit a DNA message following modification.
We developed designs, models, and experiments to approach these goals.
* It should be noted that modification of a DNA message was accomplished by Bonnet et al in a March publication [8]. We were not aware of this until our project was nearly complete. However, our work toward modification of a DNA message was key to our project and did allow us to contribute new BioBricks to the registry.
1. Lewis-Sauer K, Camper A, Ehrlich G, Costerton J, Davies D. Pseudomonas aeruginosa displays multiple phenotypes during development as a biofilm. Journal of Bacteriology, 2002, 184 (4) pp 1140–54.
2. Nealson K, Platt T, Hastings JW. The cellular control of the synthesis and activity of the bacterial luminescent system. Journal of Bacteriology, 1970, 104 (1) pp 313–22.
3. Basu S, Gerchman Y, Collins CH, Arnold FH, Weiss R. A synthetic multicellular system for programmed pattern formation. Nature, 2005, 434 pp 1130–1134.
4. Danino T, Mondragón-Palomino O, Tsimring L, Hasty J. A synchronized quorum of genetic clocks. Nature, 2010, 463 pp 326–330.
5. Balagaddé FK, Song H, Ozaki J, Collins CH, Barnet M, Arnold FH, Quake SR, You L. A synthetic Escherichia coli predator–prey ecosystem. Molecular Systems Biology, 2008, 4, pp 1–8.
6. Ortiz ME, Endy D. Engineered cell-cell communication via DNA messaging. Journal of Biological Engineering, 2012, 6:16.
Read References
Control
Design
Controlling Transmission of a DNA Message
In communication systems, messages may need to be sent in response to particular stimuli.  We sought to incorporate this control into DNA messaging at the level of M13 viral particle production.
Setting out to control particle production, we had the option of manipulating any of the 11 genes in the M13 genome (see the M13 Bacteriophage page). In our design, viral particle production is regulated by controlling production of M13 protein VIII, which is the major coat protein [1]. Gene VIII is removed from the helper plasmid and is placed on the messaging phagemid under control of an inducible promoter (Figure 1).
When gene VIII is induced on the messaging phagemid, we should have gene VIII complementation and DNA message transmission. When gene VIII is not induced, there will be no gene VIII present and no complementation will take place.
When gene VIII is induced on the messaging phagemid, we should have gene VIII complementation and DNA message transmission. When gene VIII is not induced, there will be no gene VIII present and no message transmission will take place.
The choice to control gene VIII was made based on several factors:
Control over bacteriophage ϕX174 particle production was accomplished using the same approach [2].
Gene VIII is the last gene in an operon on the M13 genome [1], so its removal is unlikely to affect transcription of other genes.
Gene VIII is not known to have regulatory function, so regulation of other M13 genes should not be disturbed.
Monica Ortiz (co-author on the original DNA messaging paper [3]) suggested the choice of gene VIII over other genes.
Using this design, we expect DNA messaging to be controllable through regulation of the M13 major coat protein, which is required for message transmission.
1. Sambrook J, Russell DW. Molecular Cloning: A Laboratory Manual. 3rd edition. Cold Spring Harbor Laboratory Press, Cold Spring Harbour, NY. 2001.
2. Jaschke P, Lieberman E, Rodruiguez J, Sierra A, Endy D. A fully decompressed synthetic bacteriophage øX174 genome assembled and archived in yeast.Virology, 2012, 434(2) pp 278-84.
3. Ortiz ME, Endy D. Engineered cell-cell communication via DNA messaging. Journal of Biological Engineering, 2012, 6:16.
Read References
Experiment
Regulating transmission of a DNA message.
To accomplish our goal of controlling DNA messaging, we sought to control M13 viral particle production by removing M13 gene VIII from the helper plasmid and placing it under an inducible promoter on a separate construct (See Figure 0.1). M13 protein VIII, the product of gene VIII, is the major coat protein of the M13 bacteriophage, so viral particle production and DNA messaging should be possible only when gene VIII is expressed and protein VIII is present (refer to the Design page).
As a step toward this goal, we first sought to demonstrate that viral particles could be produced when a helper plasmid missing gene VIII is complemented by a copy of gene VIII constitutively expressed on a separate construct; we chose to place gene VIII on the messaging phagemid.
1. Constructs
1a. Experimental constructs
To test the complementation of gene VIII, we used E. coli βDH10B strain containing the messaging phagemid and helper plasmid as a sender population, and we used E. coli JM109 strain containing a constitutively expressed GFP expression cassette (BBa_I20260) in pSB3K3 (Kanamycin-resistant) as a receiver population. The senders, E. coli βDH10B, require diaminopimelic acid (DAP) as supplement for growth in LB media. The receivers, E. coli JM109 are F+ and do not require this supplement. (Recall that receivers must be F+ for delivery of a DNA message by an M13 particle.) This allows counter-selection against sender cells by growing in LB media with no DAP supplementation.  Receiver cells that have received the message can be selected using chloramphenicol and RFP expression.
In order to perform the gene VIII complementation test, we made the following constructs using standard assembly and PCR methods, and then transformed them sequentially into E. coli βDH10B (F-) strain (Figure 1.1).
M13 ori – RFP cassette - Constitutive Promoter + RBS – M13 gVIII (pSB1C3) (BBa_K314110- BBa_J04450- BBa_K1039020)
HPdO Δ gVIII (BBa_K1039017)
Figure 1.1: A messaging phagemid containing M13 gVIII would complement a modified helper plasmid with gVIII deleted and create M13 viral particles. The messaging phagemid would then be packaged into an M13 viral particle due to the presence of the M13 origin on the phagemid and could then be transferred through viral infection.
An indicator plasmid containing a GFP expression cassette was transformed into E. coli JM109 (F+) receiver cells (Figure 1.2).
GFP cassette (pSB3K3) (BBa_I20260)
Figure 1.2: Receiver cells that contain the indicator plasmid could be distinguished from sender cells due to the presence of the GFP.
1b. Control constructs
As a positive control, we used senders carrying a complete helper plasmid (gene VIII present) and a messaging phagemid carrying the M13 origin of replication and an RFP expression cassette (BBa_K314110- BBa_J04450). With the helper plasmid intact and all phage proteins expressed, we expect the phage machinery to recognize the M13 origin and package the messaging phagemid to be sent to the receiver population.
M13 ori- RFP cassette (pSB1C3) (BBa_K314110- BBa_J04450)
HPdO (BBa_K1039016)
As a negative control to confirm the necessity of the M13 origin, we performed an experiment identical to the positive control using a messaging phagemid that is missing the M13 origin. Without the M13 origin, we expect no transmission of the messaging phagemid. We also performed a negative control for the gene VIII complementation experiment wherein the messaging phagemid was missing the M13 ori.
RFP cassette (pSB1C3) (BBa_J04450)
HPdO (BBa_K1039016)
AND
RFP- Constitutive promoter + RBS -  gVIII (pSB1C3) (BBa_J04450- BBa_K1039020)
HPdO Δ gVIII (BBa_K1039017)
To demonstrate the need of gene VIII for viral packaging, we added a negative control to the experiment that is identical to the gene VIII complementation experiment, but with the messaging phagemid lacking gene VIII (ie no gene VIII was present in the system).
M13 - RFP cassette (pSB1C3) (BBa_K314110- BBa_J04450)
HPdO Δ pVIII (BBa_K1039017)
In order to demonstrate the necessity of having both a helper plasmid and a messaging phagemid for successful transmission, we also included experiments wherein senders contained only a messaging phagemid or only a helper plasmid. In these cases we expected no message transmission.
HPdO (BBa_K1039016)
HPdO Δ gVIII (BBa_K1039017)
M13 ori - RFP- Constitutive promoter + RBS - gVIII (pSB1C3) (BBa_K314110- BBa_J04450- BBa_K1039020)
RFP - Constitutive promoter + RBS - gVIII (pSB1C3) (BBa_J04450- BBa_K1039020)
M13 - RFP cassette (pSB1C3) (BBa_K314110- BBa_J04450)
RFP cassette (pSB1C3) (BBa_J04450)
As a negative control to demonstrate the necessity for F+ receivers, we transformed the indicator plasmid used in the receiver cells into E. coli DH5α (F- strain) and used it as a receiver population. We expected no successful messaging with F- receivers because the F pilus, which is only present in F+ strains, is required for the mechanism of message delivery using M13 particles.
In addition, we plated senders and receivers individually on our selection plates to check for contamination.
1c. Biobricks submitted
We submitted the following Biobricks to the Standard Registry of Biological Parts.
BBa_K1039016- HPdO Helper Plasmid
BBa_K1039017- HPdO with deleted gVIII Helper Plasmid
BBa_K1039018- pVIII of M13 virus
BBa_K1039019- pVIII of M13 virus with strong RBS (B0034)
BBa_K1039020- pVIII of M13 virus with strong constitutive promoter (J23104) and RBS (B0034)
2. Experimental design for Gene VIII complementation
After making our constructs, we set out to determine if our helper plasmid with gene VIII knockout (BBa_K1039017) could be complemented with a messaging phagemid consisting of M13 origin (BBa_K314110) - RFP expression cassette (BBa_J04450) - Constitutive Promoter and Ribosome Binding site + M13 gene VIII (BBa_K1039020) in pSB1C3. We expected the expression of gene VIII from the messaging phagemid to produce a viral coat protein so it can complement the helper plasmid with gene VIII knockout. With a complete set of functional proteins, the virus should be able to assemble itself and package the messaging phagemid through recognition of the M13 ori to successfully deliver the message to the receiver population. Chloramphenicol resistance (from the pSB1C3 backbone) and RFP expression should be detectable in receiver cells that have received the DNA message.
2a. Co-culture experiment of sender and receiver populations
Message transmission was tested by co-culturing sender and receiver populations, then selecting for successful transmission. During the co-culture, M13 particles containing the DNA message should be secreted by sender cells and should deliver the message to receiver cells.
2b. Exposure of receiver population to filtrate from sender culture
Since the message is transmitted through viral particles, sender cells should not need to be in contact with receivers for message transmission; presence of message-carrying viral particles should be enough. We therefore also tested message transmission by exposing receiver cells to culture filtrate from sender cells, which should contain viral particles secreted by the senders that can deliver messages to receivers.
3. Experimental protocol
3a. Co-culture experiment (Figure 3.1)
Grow sender cells and receiver cells individually in liquid LB media with appropriate antibiotics and supplements overnight at 37 °C with shaking.
Subculture a 1:100 dilution into fresh media containing the appropriate antibiotics and supplements and grow at 37 °C with shaking until O.D ≈ 0.6 (± 0.1).
Remove the chloramphenicol antibiotic from the media of sender cells by centrifuging the cultures and washing the pellet 3 times with saline.
Resuspend the sender cells in fresh LB liquid media containing no chloramphenicol
Co-culture sender cells with receiver cells at a 2:1 volume ratio and incubate at room temperature for ~ 5 h with shaking.
Subculture the co-culture at a 1:100 dilution into fresh LB media with no DAP and the appropriate antibiotics to select for receiver cells containing the messaging phagemid. Incubate overnight at 37 °C with shaking.
Dilute cultures 10-6 in saline and plate 100 μL of the diluted culture onto LB Cm 20 Km 25 agar.
Figure 3.1 PhiC31 or Bxb1 invertible promoter switch (PB state). The invertible promoter switch consists of a strong or medium promoter (J23119 or J23118) and an upstream transcription terminator, flanked by either PhiC31 or Bxb1 attP and attB sites (PB state). These constructs were standardized by adding a standard Prefix/Suffix and submitted as BioBricks.
3b. Filtrate experiment
Grow sender cells and receiver cells individually in liquid LB media with appropriate antibiotics and supplements overnight at 37 °C with shaking.
Subculture a 1:100 dilution into fresh media containing the appropriate antibiotics and supplements and grow at 37 °C with shaking until O.D ≈ 0.6 (± 0.1).
Remove the chloramphenicol antibiotic from the media of sender cells by centrifuging the cultures and washing the pellet 3 times with saline.
Resuspend the sender cells in fresh LB liquid media containing no chloramphenicol
Co-culture sender cells with receiver cells at a 2:1 volume ratio and incubate at room temperature for ~ 5 h with shaking.
Results
Table 4.1: Results for M13 complementation co-culture experiment in liquid LB Cm Km
Plasmid Constructs in Sender Cells (βDH10B)
Biobricks
Optical Density at 600 nm (OD600)
Co-culture with JM109 (F+) Receiver Cells containing
BBa_I20260
Co-culture with DH5α (F-) Receiver Cells containing BBa_I20260
1. HPdO
2. M13 ori- RFP cassette (pSB1C3)
1. BBa_K1039016
2. BBa_K314110- BBa_J04450
0.039
0.023
1. HPdO
2. RFP cassette (pSB1C3)
1. BBa_K1039016
2. BBa_J04450
0
0
1. HPdO Δ gVIII
2. M13 ori - RFP- Constitutive promoter + RBS -gVIII (pSB1C3)
1. BBa_K1039017
2. BBa_K314110-BBa_J04450-BBa_K1039020
0.054
0.019
1. HPdO Δ gVIII
2. RFP- Constitutive promoter + RBS -gVIII (pSB1C3)
1. BBa_K1039017
2. BBa_J04450-BBa_K1039020
0.118
0
1. HPdO Δ gVIII
2. M13 - RFP cassette (pSB1C3)
1. BBa_K1039017
2. BBa_K314110-BBa_J04450
0
0
1. HPdO
1. BBa_K1039016
0
0
1. HPdO Δ gVIII
1. BBa_K1039017
0
0
1. M13 ori - RFP- Constitutive promoter + RBS - gVIII (pSB1C3)
1. BBa_K314110-BBa_J04450-BBa_K1039020
0.299
0.014
1. M13 ori - RFP cassette (pSB1C3)
1. BBa_K314110-BBa_J04450
0.370
0.011
1. RFP- Constitutive promoter + RBS - gVIII (pSB1C3)
1. BBa_J04450-BBa_K1039020
0.378
0.022
1. RFP cassette
1. BBa_J04450
0.423
0.038
None
---
0
0
In many of the test samples, growth in liquid culture was detected when the sender cells contained the messaging phagemid, regardless of whether the M13 ori was present or if the helper plasmid was present. Since the receiver cells were selected on the grounds of containing chloramphenicol resistance, transfer of the messaging phagemid was very likely due to conjugation between sender cells and receiver cells, rather than infection by the M13 bacteriophage. When the sender population contained a helper plasmid along with the messaging phagemid, the selection for receiver cells with chloramphenicol yielded much less growth. This could suggest that if M13 bacteriophage were being produced, it would slow the growth of the receiver cells. It is also likely that the message was transmitted via M13 bacteriophage but the results are inconclusive.
Monica Ortiz from the Endy Lab proposed the use of F+ receivers and senders, which may decrease chance of conjugation. The new design of the experiment would use the E. coli JM109 F+ strain as the sender population as well as the receiver population. The sender populations would have the same plasmids as was mentioned in the previous experiments but the receiver population would contain a GFP indicator in the pSB4A5 backbone so that receiver cells could be selected using ampicillin. Currently, the sender and receiver cell populations are in the process of being established so that the co-culturing experiment can be performed.
The filtrate experiment was the second experiment proposed by our team. Conjugation requires cell-cell contact so by filtering the sender cells out of the media and using the filtrate to infect receiver cells, message transfer would only occur through M13 viral infection. The filtrate experiment was performed but no colonies were observed on the plates. It is believed that the cells were too dilute when plating on agar so the experiment will be repeated and optimized, possibly using E. coli JM109 F+ strain as the sender population.
In addition to the experiments proposed above, we devised a plaque assay that allows us to visualize M13 infection of the receiver cells. Although M13 virus does not undergo the lytic cycle, its infection of a cell population slows its growthreference. If plated onto agar, an area of diminished growth would be observed at the site of M13 viral infection, creating a turbid plaque. We proposed to use E. coli JM109 F+ receiver cells containing a helper plasmid and infecting them with the filtrate of E. coli JM109 F+ sender cells as described above. Receiver cells that become infected by the M13 virus would gain the messaging phagemid and then produce more virus (since they contain the helper plasmid), which will then infect neighbouring cells, thus creating a zone of diminished growth. This experiment is still in its preliminary stages but we hope to achieve positive results.
5a. Genotypes of strains used
DH5α
F- endA1 glnV44 thi-1 recA1 relA1 gyrA96 deoR nupG Φ80dlacZΔM15 Δ(lacZYA-argF)U169, hsdR17(rK- mK+), λ–
JM109
endA1 glnV44 thi-1 relA1 gyrA96 recA1 mcrB+ Δ(lac-proAB) e14- [F' traD36 proAB+ lacIq lacZΔM15] hsdR17(rK-mK+)
βDH10B
F- endA1 recA1 galE15 galK16 nupG rpsL ΔlacX74 Φ80lacZΔM15 araD139 Δ(ara,leu)7697 mcrA Δ(mrr-hsdRMS-mcrBC) λ-
6. References
Sambrook J, Russell DW. Molecular Cloning: A Laboratory Manual. 3rd edition. Cold Spring Harbor Laboratory Press, Cold Spring Harbour, NY. 2001
Read References
Modify
DESIGN
Modifying a DNA Message
Recent work with serine integrases (see Appendix on Serine Integrases) has demonstrated their utility in implementing digital memory [1] and Boolean logic [2,3] in DNA through inversion of DNA sequences. We sought to incorporate an invertible promoter switch into a DNA message, consisting of a promoter whose orientation can be flipped and restored through action of a serine integrase and a corresponding recombination directionality factor (RDF). The switch would allow different gene expression based on its state, could function in digital memory storage, and could be flipped in sender cells before transmission or in receiver cells after receipt of the message.
As part of this goal, we designed four invertible promoter switches, corresponding to combinations of two different integrase systems, Bxb1 and Phi C31, with two different promoters, J23119 and J23118 (see the BioBricks page). Our design is directly inspired by and closely mimics the recombinase addressable data (RAD) module designed by Bonnet et al [1], which uses the Bxb1 integrase system.  The invertible promoter switch consists of a promoter flanked by att sites, with a transcription terminator upstream of the promoter. Expression of integrase alone or in concert with RDF inverts the switch through recombination of the att sites. The terminator guards against transcription of genes upstream of the promoter that should only be transcribed when the switch is in its opposite state (Figure 1). See the Experiment and Results pages for information on our implementation of invertible promoter switches and the integrase/RDF BioBricks we produced for working with the switches.
When the switch is in “PB” state, the promoter is flanked by attP and attB sites, and in “RL” state the promoter is flanked by attR and attL sites. Flipping from PB to RL is catalyzed by integrase and is irreversible in the presence of integrase alone. Flipping from RL to PB is catalyzed by integrase in conjunction with RDF, and is irreversible in the presence of these proteins together. For more on the functioning of integrase and RDF in site-specific recombination of att sites, see the appendix on Serine Integrsaes.
Since the switch in the RAD module was demonstrated to be functional [1], we replicated its architecture exactly in our switches. Intervening spacer sequences between functional elements are also the same in our switches.
Figure 1. The invertible promoter switch flips from PB to RL state in the presence of integrase, and from RL to PB state in the presence of integrase and RDF. Difference genes are expressed depending on the state of the switch.
It is important to note that any genes added to the prefix side of the switch must be inverted relative to the BioBrick convention, i.e. they are transcribed from suffix to prefix, rather than prefix to suffix. These inverted genes are expressed when the switch is in PB state.
These switches may be used as a form of passive memory storage; it holds its state after the stimulus (Int or Int+RDF) has passed. It does not require active gene expression to hold state, in contrast to switches that function through regulation of transcription (eg a mutual inhibition toggle switch [4]).
It should be noted that modification of a DNA message was accomplished by Bonnet et al in their March publication [2]. We were not aware of this until our project was nearly complete. However, our work toward modification of a DNA message was key to our project and did allow us to contribute new BioBricks to the registry.
It should also be noted that our BioBrick switch constructs contain an NheI restriction site outside of the functional switch portion of the BioBrick close to the prefix. This NheI site is an artifact of a design idea we had early in our project’s design phase and does not have a functional role. It is not expected to affect the functioning of the switch.
References
1. Bonnet J, Subsoontorn P, Endy D.  Rewritable digital data storage in live cells via engineered control of recombination directionality.  Proceedings of the National Acadamy of Science USA, 2012, 109(23) pp 8884-8889.
2. Bonnet J, Yin P, Ortiz ME, Subsoontorn P, Endy D. Amplifying genetic logic gates. Science, 2013, 340 (6132) pp 599-603.
3. Siuti P, Yazbek J, Lu TK. Synthetic circuits integrating logic and memory in living cells. Nature Biotechnology, 2013, 31 pp 448–452.
4. Gardner TS, Cantor CR, Collins JJ. Construction of a genetic toggle switch in Escherichia coli. Nature, 2000, 403(6767) pp 339-342.
Read References
EXPERIMENT
1.  Constructs
1a. Multi-copy test constructs:
Multi-copy test constructs were constructed for the following promoter switches in their PB state (promoter switches flanked by attP and attB sites):
BBa_K1039004: Bxb1 Invertible Promoter Switch (Promoter J23119) – PB State
BBa_K1039008: PhiC31 Invertible Promoter Switch (Promoter J23119) – PB State
BBa_K1039009: PhiC31 Invertible Promoter Switch (Promoter J23118) – PB State
The gene for green fluorescent protein (GFP) was placed downstream of each promoter switch in its inverted orientation (BBa_K1039015). This allows for the promoter switch to transcribe GFP in its PB state. The gene for red fluorescent protein (RFP) (BBa_J04450) was placed upstream of each promoter switch. RFP was therefore only transcribed upon a directionally regulated site-specific recombination event on the attP and attB sites flanking the promoter switch. As mentioned before, this event is mediated by the corresponding PhiC31 or Bxb1 integrases and thereby results in the reversed orientation of the promoter (Fig. 1).
The multi-copy test constructs were assembled via 3A assembly (Parts Registry Assembly Protocol) and sub-cloned into a Parts Registry plasmid, pSB4A5. Each test construct is flanked by the standard iGEM suffix and prefix. The multi-test constructs made include:
Bba_K1039025: Test construct for Bxb 1 Invertible Promoter switch (Promoter J23119).
Bba_K1039027: Test construct for PhiC31 Invertible Promoter switch (Promoter J23119).
Bba_K1039028: Test construct for PhiC31 Invertible Promoter switch (Promoter J23118).
Figure 2. Invertible promoter switch test construct (PB state): The figure above shows the test constructs made for the PhiC31 and Bxb 1 invertible promoter switches in their PB state. The gene for GFP (with its ORF in a reverse orientation) was cloned downstream of each promoter switch. The gene for RFP was cloned upstream of each promoter switch. The invertible promoter switch is flanked by attP and attB sites (as represented by the triangles). The promoter within each promoter switch would drive the expression of GFP in the PB state. Test constructs for all Invertible promoter switches are flanked with iGEM Prefix and Suffix (as indicated by the box labeled “P” and “S”).
2b. Multi-copy Non-inducible Integrase-expressing constructs:
Two constructs for the constitutive expression of PhiC31 and Bxb1 integrases respectively, were assembled via Standard Assembly (Parts Registry Assembly Protocol). Genes encoding the PhiC31 and Bxb1 integrases (BBa_K1039012 and BBa_K1039003) were separately cloned downstream of a constitutive promoter and its corresponding ribosome binding site (K608002) in pSB1C3 (Parts Registry Standard vector) (Fig. 2). Each test construct is flanked by the standard iGEM suffix and prefix. The multi-test non-inducible integrase-expressing constructs include the following:
1)  Bba_K1039030: PhiC31 integrase-expressing construct with a Promoter and RBS
2)  Bba_K1039029: Bxb1 integrase-expressing construct with a promoter and RBS.
2c. Multi-copy Non-inducible Integrase and Recombination directionality factor- expressing constructs: Constructs were assembled to allow constitutive expression of both integrase and recombination directionality factor simultaneously from a single promoter. This construct was assembled via Standard Assembly (Parts Registry Assembly Protocol) for both PhiC31 and Bxb1 integrases/RDFs respectively (Fig. 2). Each construct is flanked by the standard iGEM suffix and prefix. The multi-test non-inducible integrase and RDF producing constructs include:
Bba_K1039014: PhiC31 integrase and RDF expressing construct from a single promoter
Bba_K1039007: Bxb1 integrase and RDF expressing construct from a single promoter.
Figure 3. Multi-test non-inducible constructs for PhiC31 and Bxb1 integrase/RDF:  (A.) Construct for non-inducible expression of integrase driven from a constitutive promoter and associated RBS for both PhiC31 and Bxb1 integrase (B.) Construct for non-inducible simultaneous expression of integrase and RDF from a constitutive promoter. Both genes have an associated RBS. All constructs are flanked with iGEM Prefix and Suffix (as indicated by the boxes labeled as “P” and “S”).
1.  Experimental design for demonstrating the functionality of Bxb1 Invertible Promoter switch (Promoter J23119: PB state)
To demonstrate the functionality of the Bxb1 invertible promoter switch in its PB state, ~500 ng of the test construct was co-transformed into a competent E. coli strain (DH5α) with its corresponding constitutive Bxb1 integrase-expressing construct (~500 ng). For our negative control, ~500 ng of the Bxb1 test construct was transformed into E. coli in the absence of the integrase expressing construct The cells were subsequently plated on LB agar (with appropriate antibiotics to maintain both plasmids) and allowed to grow for two days at 37 degrees (~36h).The growth period was chosen to allow adequate expression of the integrase which would consequently mediate a recombination event resulting in the reversed orientation of the promoter (RL state) within each switch and thereby drive the expression of RFP.
Figure 4. Testing PhiC31 and Bxb1 Invertible promoter switches (PB state): To test the invertible promoter switches in their PB state (flanked by attP and attB), they were independently co-transformed with their corresponding non-inducible integrase-expressing constructs and consequently plated on LB agar with appropriate antibiotics to select for both plasmids. The resulting bacterial colonies were then analyzed for RFP expression.
2.  Flow cytometry to demonstrate functionality of “inverted” GFP
The Bxb1 invertible promoter switch in its PB state drives the expression of GFP. A single GFP-expressing colony (from the Bxb1 invertible promoter switch test construct transformation plate) was selected and inoculated in 5mL of LB (with appropriate antibiotics) and grown overnight at 37°C. Since a modified version of GFP (ORF was modified to be in the opposite orientation with respect to convention) was used for these constructs, bacterial cells producing GFP (BBa_I20260) was used as one of the controls. Flow cytometry was then used to confirm GFP expression from all cultures
RESULTS AND ANALYSIS
Functionality of the Bxb1 Invertible promoter switch
Once the Bxb1 Invertible promoter switch test construct was co-transformed with its corresponding Bxb1 integrase-expressing construct, the resulting colonies were screened for RFP and GFP expression for both the co-transfomations and the negative control respectively. Figure 5b. represents the resulting colonies screened for RFP expression after the co-transfomation. All bacterial colonies were expressing detectable amounts of RFP. These colonies were also screened for GFP expression (Fig 5a.) and showed no fluorescent protein expression, thus confirming the integrase-mediated inversion of the Bxb1 promoter switch.
The negative control consisting of our single transformation of the Bxb1 promoter switch test construct in the absence of integrase. Figure 5c. and 5d. represent the resulting colonies when screened for GFP and RFP expression repectively. As expected, fig. 5d. confirmed no expression of RFP, however, GFP expression was also undetectable. To detect GFP from our control plates, a bacterial colony from the negative control transformation plate (fig. 5e.) was inoculated in culture for ~16 hours and centrifuged at 13,000 rpm for 1 minute. The allowed the bacterial cells to be concentrated into a pellet, which was consequently screened for GFP expression. As shown in Fig. 5f., the cell pellet showed detectable GFP expression when compared to its corresponding controls (RFP expressing bacterial cells and bacterial cells expressing no fluorescent protein). The observed GFP expression confirmed the proper functioning of our switch in its PB state.
Figure 5: Characterization of PhiC31 and Bxb 1 invertible promoter switches (PB state ): Bxb1 promoter switch test construct co-transformed with Bxb1 integrase expressing construct and viewed under (a) GFP filter (b) RFP filter; The Bxb1 test construct was transformed in the absence of Bxb1 integrase and viewed under (c) GFP filter (d) RFP filter (e) no filter; RFP expression was only seen in plate(b). (e) (f) shows a pellet of cells containing the Bxb1 test construct without the integrase expressing construct. When viewed under the GFP filter, the pellet fluoresced green when compared to its corresponding controls (cells not expressing a fluorescent protein, and cells expressing RFP)
Flow cytometry
The results for flow cytometry are shown in Figure 6. GFP expression from both cultures (Figure 6b. and 6c.) result in a shift in fluorescent populations when compared to control cells expressing no fluorescent protein (Figure 6a.). These results confirm the proper functioning of our modified GFP gene and furthermore, demonstrate the functionality of the promoter switches in their PB state.
Figure 6: Flow cytometry for confirming expression of inverted GFP (A) negative control: bacterial cells expressing no fluorescent protein (B) GFP (BBa_I20260) expressing bacterial cells (C) Bacterial cells expressing inverted GFP from the invertible promoter switch test construct. GFP expression from cultures (B) and (C) result in a shift in fluorescent populations when compared to control cells expressing no fluorescent protein (A). These results confirm the proper functioning of our modified GFP gene and furthermore, characterize the promoter switches in their PB state
MODELLING
Modelling the Dynamics of Bxb1-style and PhiC31-style Invertible Promoter Switch Systems
Invertible Promoter Switch “Styles”
We made invertible promoter switches based on the Bxb1 and PhiC31 integrase systems.  In both of these systems Int dimerizes before attaching to att sites, and RDF modifies Int by binding it stoichiometrically (ie two RDF molecules bind each Int dimer). However, the details of the interaction between Int and RDF in these two systems is thought to differ. Notably, the Bxb1 RDF is thought to interact with Bxb1 Int only when Int is bound to DNA at the Bxb1 att sites [1] while PhiC31 RDF is also able to complex with Int in the cytosol independently of DNA binding [2] (Figure 1).
Figure 1. Bxb1 RDF can only complex with Int dimers that are bound to att sites, while PhiC31 RDF can complex with cytosolic Int dimers as well.
We refer to a switch system where RDF binds Int only when Int is already complexed with an att site as a “Bxb1-style” system, and we refer to a switch system where RDF can complex with Int independently of DNA as a “PhiC31-style” system. We sought to investigate the differences in the dynamics of these systems.
We first modeled switch systems where Int and RDF are driven by separate promoters (Figure 2). We should note that Bonnet et al did simulations describing the same switch systems in their original design of the RAD module [3].
Figure 2. We modeled the dynamics of Bxb1- and PhiC31-style promoter switch systems with Int and RDF driven by separate promoters.
Questioning the Model to Inform Model-based Design
We wanted to know the ranges of integrase and RDF expression that would flip switches of each style to identify potential differences in the ways they should be applied.
Model Construction
State variables (all time-dependent):
I = Integrase
R = RDF
mx = mRNA corresponding to protein X.
SPB = Switch in “PB” state, where we have attP and attB sites flanking the promoter
SRL = Switch in “RL” state, where we have attR and attL sites flanking the promoter
I2 = integrase dimer
I2SPB = integrase dimer bound to one att site in a PB state switch
I2RXSPB = integrase dimer bound to one att site in a PB state switch, with X RDF proteins also bound (X = 1 or 2)
I4SPB = one integrase dimer bound to each att site in the PB state switch
I4RXSPB = one integrase dimer bound to each att site in the PB state switch, with X RDF proteins also bound (X = 1 – 4)
In the PhiC31 style switch but NOT the Bxb1 style switch, the following complexes are also included:
I2Rx = X RDF proteins bound to an integrase dimer, X = 1 or 2
Parameters:
αI = transcription rate of Int
αR = transcription rate of RDF
δx = degradation rate of mRNA corresponding to protein X
βx = translation rate of protein X from mx
ρx = degradation rate of protein X
k1 = forward rate constant for association of a protein to a complex
k-1 = reverse rate constant for dissociation of a protein to a complex
krec = rate of switch flipping through recombination when bound to appropriate complex
Note that rate constants k1 and k-1 are assumed to be the same for formation of all complexes, since rate constants for the complexes are not known. krec is assumed to apply to flipping of the switch from PB to LR state and also from LR to PB state, when DNA is bound in the appropriate complexes.
Differential Equations: The differential equation for the abundance of an mRNA species is determined by transcription and degradation rates of mRNA:
The differential equation for the abundance of a complex X of I, R, and S accounts for the following factors:
If X is a monomer of I or R, it can be produced through translation and we add the term,
If X is formed from association of any complexes A and B in a reaction,
then we add the terms,
for association and dissociation.
If X can associate with a complex Y to form a complex Z in a reaction,
then we add the terms,
for association and dissociation.
If X is a complex that allows flipping of the switch (ie X is SPBI4 or SRLI4R4) then we add the term,
for production of X through switch flipping
Protein degradation in complexes is assumed such that individual proteins degrade at the same rate as they would outside of a complex. Degradation rate ρ is assumed to be equal for all monomers, so we assume that a complex X with N subunits degrades at a rate proportional to 1/N, since all subunits degrade together in the complex. For degradation of complexes, we add the term,
DNA is assumed to be conserved, without production or degradation. To allow protein complexes to degrade without losing DNA, degradation of any complex involving DNA is assumed to apply only to the protein; the DNA is left behind. For free DNA involved in complexes Ci, we add the term,
where Ni is the number of proteins in complex Ci.
Thus the general differential equation for abundance of a protein complex X is:
where n different protein pairs Ai and Bi can complex to form X, and X can complex with m different other complexes Yj to form m complexes Zj, and there are N proteins in complex X.
If X is a monomer of I or R then we also add the translation term, if X can be consumed or formed through switch flipping we add the appropriate production or consumption terms, and if X is naked DNA then we add terms for its formation through degradation of complexes.
Note that the only reactions involving switch flipping are:
Example:
For abundance of integrase dimer, which for a PhiC31 style system takes part in the interactions,
the differential equation is:
Estimation of Parameter Values
Promoter J32101 was reported to drive transcription at a rate of 0.03 PoPS (polymerases per second) [3]. We ranged our promoter strengths from 10-12 to 0.1 PoPS, as this was the range in which we saw interesting dynamics.
A previous study found that the average half-life of an mRNA is approximately 5 minutes [4]. We used this parameter for mRNA degradation in our model.
To find a reasonable range for translation rate, we referred to a study in which lacZ translation was found to range from roughly 15 to 50 translation initiation events per minute per mRNA [5]. We assumed translation occurred at a rate of 20 initiation events per minute per mRNA, and varied only transcription rate to change protein concentrations.
Based on a study of protein degradation rates [6], we produced protein degradation constants by assuming the half-life of all proteins to be 60 minutes.
For the forward and reverse rate constants, we did not have any data. We assumed k1, k-1, and krec all to be 1.
For all simulations discussed below, we ran the simulations with varying rate constant values to check if our assumptions had a significant impact on the results. We found that results were qualitatively similar.
Sensitivity Analysis of Bxb1- and PhiC31-style Switch Systems
We simulated the behavior of each switch with varying integrase and RDF promoter strengths. We used 50 switches in the simulations, and kept track of the percentage of switches that had flipped from PB to RL state at the system’s steady state (Figure 3). We also ran the simulations with different switch copy numbers and obtained qualitatively similar results.
Figure 3. Comparison of a) Bxb1-style and b) PhiC31-style switches flipping with various Int and RDF promoter strengths.
These graphs are qualitatively similar to those obtained by Bonnet et al using their simulation [3].
We note that the dynamics for very low Int and RDF promoter strengths (10-12 – 10-9 PoPS) are similar for the two switches, but that for higher promoter strengths they differ. In the Bxb1-style system RDF promoter strength above 10-9 PoPS rules out flipping of the switch, regardless of integrase promoter strength, while in the PhiC31-style system high RDF expression can be matched with high Int expression and the switch can still flip. Since the PhiC31-style system allows for RDF to complex with Int in solution, RDF can be sequestered by Int in solution. This allows for they dynamics we see in Figure 3.
Two-switch Systems
A promoter switch system may be subject to noise, where the switch temporarily flips accidentally due to a burst of integrase expression. If there are multiple copies of the switch present in the cell, there is a region of Int and RDF expression where some copies of the switch are flipped while others are not (the range of colours between blue and red in Figure 3). This behvaiour is undesirable, as it is not “switch-like”; since a switch is an all-or-nothing device, we would like our switches to function this way even when they are present in multiple copies.
We wanted to see if use of a two-switch system, wherein the first switch controls the second switch, could improve the “switch-like nature” of a multi-copy set of switches. If the first switch must flip to completion before the second switch flips, then the expression range where switching is only partial may be reduced in the second switch.
We modeled a two-switch system where the first switch controls expression of the Int and RDF corresponding to the second switch (RDF2 and Int2).  RDF2 is expressed when switch 1 is in its initial PB state, and Int2 is expressed when switch 1 is flipped to RL state. The Int and RDF corresponding to switch 1 are now referred to as Int1 and Int2 (Figure 4).
Figure 4. A two-switch system where the second switch is controlled by the first switch.
Modelling the Two-switch System
In this system, we have state variables for Int2 and RDF 2 (I2 and R2), their mRNAs, and complexes involving them and the second switch. The two-switch system was modelled using the same principles described for the one-switch system, with one extra detail: the transcription rate of Int2 and RDF2 genes depends on the state of switch 1. We introduce the parameter,
αS1,
the transcription rate from the promoter in switch 1. The differential equations for Int2 and RDF2 mRNAs are given by:
where Pi are complexes involving S1 in PB state, and Rj are complexes involving switch 1 in RL state.
Sensitivity Analysis of a Two-switch System
We simulated the two-switch system with varying Int1 and RDF1 promoter strengths. We used 50 switches in the simulations, and kept track of the percentage of switches that had flipped from PB to RL state at the system’s steady state for each of the two switches. The strength of the switch 1 promoter was chosen such that the sum of expression from all the switch 1 promoters was equal to the expression from the Int1 promoter. This way, similar amounts of integrase are in the system for both switches.  Figure 5 displays a two-switch system using two PhiC31-style switches. We selected the PhiC31-style switch because it offers control over switching at a wider range of parameter values than the Bxb1-style switch.
We also ran the simulations with different switch copy numbers, different switch 1 promoter strengths, and different combinations of switch 1 and switch 2 styles, and obtained qualitatively similar results.
Figure 5. Percentages of promoters flipped for a) Switch 1 and b) Switch 2 in a two-switch system. Both switches were PhiC31-style in this case.
We see that the ranges of Int1 and RDF1 promoter strengths corresponding to incomplete switch flipping (the colours between blue and red in Figure 5) are narrower for switch 2 (Figure 5b) than for switch 1 (Figure 5a). This simulation suggests that controlling one switch with another helps make a multi-switch system more switch-like, with flipping of all copies at once.
To investigate the effects of stochastic processes, we implemented our differential equations model for the two-switch system stochastically using the Gillespie algorithm and varied Int and RDF promoter strengths. The shape of the parameter regions where each switch flipped switches flipped were qualitatively similar to those found in the continuous model (Figure 7).
Figure 7. Percentages of promoters flipped for a) Switch 1 and b) Switch 2 in a two-switch system simulated stocahstically. Both switches were PhiC31-style in this case.
We note a clear reduction in the variability of the percentage of switches flipped in switch 2 compared to switch 1. By controlling switch 2 with switch 1, we have reduced the stochastic nature of switch 2 to give it more robust switch-like behavior. This complements the finding using the continuous model, which showed that a two-switch system narrowed the parameter range corresponding to partial flipping of switch 2.
Conclusions
Our goal is to use modelling to inform design decisions in a model-based design approach.  The following points are relevant to the application of these invertible promoter switches:
Our single-switch model indicates that a Bxb1-style switch has a threshold of RDF expression above which no flipping can occur, regardless of Int expression, while a PhiC31-style switch can be flipped at any RDF expression level as long as Int expression is high enough.
o The PhiC31-style switch is thus more flexible in different situations that may require performance at different ranges of Int and RDF expression levels.
o The Bxb1-style switch has the advantage that switching can generally be controlled by varying one parameter, RDF expression, past a specific threshold.
Our two-switch model shows that switch-like behavior of a switch present in multiple copies is improved if it is controlled by a second switch; the two-switch system corresponds to a narrower range of parameter values that permit partial switching.
References
1. Ghosh P, Wasil LR, Hatfull GF. (2006) Control of phage Bxb1 excision by a novel recombination directionality factor. PLoS Biology, 2006, 4:e186.
2. Khaleel T, Younger E, McEwan AR, Varghese AS, Smith MCM. A phage protein that binds PhiC31 integrase to switch its directionality. Molecular microbiology, 2011, 80(6), 1450-63.
3. Bonnet J, Subsoontorn P, Endy D. Rewritable digital data storage in live cells via engineered control of recombination directionality.  Proceedings of the National Acadamy of Science USA, 2012, 109(23) pp 8884-8889.
Read References
Retransmit
Retransmitting a Modified DNA Message
DESIGN
In intercellular communication systems, it may be useful for receiver cells to be able to retransmit a message, especially if they are able to modify the message.
We sought to combine control over transmission and modification of a DNA message to allow receiver cells to modify an original DNA message (Mori) to produce a modified DNA message (Mmod) and then retransmit it.
In the scheme we developed (Figure 1), three populations of E. coli are co-cultured (Figure 1):
The sender population transmits the original DNA message, Mori.
Mori includes an invertible promoter switch. When flipped, it leads to production of M13 gene VIII and T7 RNA polymerase (RNAP) from Mmod
The F+ primary receiver population receives Mori, and in the presence of a particular stimulus modifies and retransmits it.
Primary receivers carry a helper plasmid with gene VIII knockout, as well as inducible integrase expression
Upon induction of integrase, the switch on the Mori messaging phagemid flips and gene VIII expression complements the helper plasmid, allowing retransmission of Mmod.
The F+ secondary receiver population exhibits a detectable behavior (ie fluorescent protein production) upon receipt of Mmod.
T7 RNAP expression from Mmod causes expression of a fluorescent protein in secondary receivers.
Note that secondary receivers may also receive Mori, since all three populations are present in co-culture, and that primary receivers can also receive M¬mod.
Figure 1. The sender population transmits a DNA message, which is modified and retransmitted by primary receivers in the presence of a particular stimulus. The secondary receiver population exhibits a detectable behavior (eg fluorescent protein expression) when it receives the modified message.
The dynamics and potential modes of failure of this system are explored through mathematical modeling.
MODELLING
We sought to combine controlled transmission and modification of a DNA message to design a system wherein receiver cells are able to modify a DNA message by flipping an invertible promoter switch and then retransmit the modified message. The system is summarized in Figure 1 (see the Design section for more details).
Figure 2. The sender population transmits a DNA message, which is modified and retransmitted by primary receivers in the presence of a particular stimulus. The secondary receiver population exhibits a detectable behavior (eg fluorescent protein expression) when it receives the modified message.
The key output of this system is secondary receiver cells that have received a modified message and are fluorescing. This output can be evaluated in two different scenarios, where modification and retransmission are:
Induced: We want as many secondary receivers as possible carrying the modified message.
Uninduced: We want as few secondary receivers as possible carrying the modified message.
This system may be prone to several undesired outcomes (Figure 2) outlined below:
1. False negative: Since populations are co-cultured, secondary receivers can also receive the original unmodified message (M¬ori) that is transmitted by sender cells. When modification and retransmission are induced, there is a risk that secondary receivers might still be overwhelmed with Mori, preventing them from receiving the modified message (M-mod¬). This could result from the following modes of failure:
The majority of secondary receivers become infected with Mori during the lag time between Mori and Mmod production.
M¬ori is present in the system first, since Mmod must be produced from Mori.
There is an additional lag in Mmod production caused by the “turnaround time” between receipt of a DNA message and accumulation of copy number and protein concentration that can fuel packaging and transmission of the modified message
Inefficiency of production of Mmod vs Mori due to genomic tinkering keeps concentration of Mmod low compare to Mori.
We have picked apart the M13 genome in order to allow control over production of viral particles in primary receivers. This may result in lowered efficiency of particle production in primary receivers compared to senders.
2. False positive: When retransmission and modification are not induced, basal expression of integrase due to promoter leakiness may cause unwanted flipping of the switch, producing Mmod and leading to retransmission. Since a switch holds state after flipping, such an error would be irreversible. This problem is exacerbated by the fact that the resulting unwanted message transmission would spread to other cells. Such amplification of error could make the system effectively constitutive, even though an inducible promoter is employed.
Questioning our Model
Our goal in modeling is model-based design. We would like to use our model to guide design choices and direct lab efforts. Before producing the model, we made the following list of questions on which we hoped the model would shed light. In each case, the measure of system performance is the ratio of secondary receivers that receive Mmod vs Mori:
What is the sensitivity to reduced viral particle production efficiency in primary receivers, leading to weaker transmission of Mmod?
How does the ratio of initial populations (senders : primary receivers : secondary receivers) affect the output of the system?
What is the sensitivity to basal (uninduced) expression of integrase in primary receivers, leading to unwanted modification and retransmission, especially given amplification of error through retransmission?
To cut to the chase, skip to the “conclusions” section! If you like details, keep reading here.
Model Construction
We used the following set of state variables and parameters to model the spread of Mori and Mmod through the co-cultured populations:
State variables:
Ps(t) = population of sender cells
P1 uninfected(t)  = population of primary receiver cells that have not received a DNA message
P2 uninfected(t)  = population of secondary receiver cells that have not received a DNA message
P1 ori(t)  = population of primary receiver cells carrying the original DNA message
P2 ori (t) = population of secondary receiver cells carrying the modified DNA message
P1 mod (t,a) = population of primary receiver cells of age a at time t. Age a corresponds to how long the cell has been a P1 mod cell. This is important to know because of the turnaround time between receipt of a DNA message and accumulation of phagemid and protein to fuel viral particle production.
P2 mod(t)  = population of secondary receiver cells carrying the modified DNA message
Mori (t) = concentration of M13 viral particles carrying the original unmodified DNA message
Mmod (t) = concentration of M13 viral particles carrying the modified DNA message
Parameters:
c = carrying capacity of the liquid growth medium
g = maximum growth rate of cells that are not producing viral particles in absence of pressure due to carrying capacity
r = maximum growth rate of cells that are producing viral particles as a fraction of g
k = adsorption rate of viral particles to cells
i = induction constant. i = 1 when message modification and retransmission is induced, and i = 0 when it is not induced
S(i) = rate of switch flipping per primary receiver carrying Mori. Depends on induction.
p = rate of viral particle production through intact M13 cistrons on the helper plasmid
b = the percent rate, relative to p, of viral particle production in primary receiver cells through complemented helper plasmid with gene VIII knockout. Assumed between 0 and 1.
j1 = age at which viral particle production begins in P1 mod cells. Production starts at zero at age j1 and increases.
j2 = age at which viral particle production reaches its maximum and continues to maintain a steady rate
When modeling the growth of each population over the course of the experiment, the carrying capacity of the media had to be considered. To simplify our equations, we define:
We built a differential equations (DEs) model of the system using these state variables and parameters, which is described below.
The DE for senders is a simple logistic growth equation:
where the growth rate G is modified by r since senders produce viral particles and therefore grow more slowly. The DEs for P1 uninfected and P2 uninfected incorporate loss due to receipt of DNA messages, and the DE for P¬2 ori incorporates production due to receipt of DNA message:
Since there is a turnaround time associated with receipt, modification, and retransmission of a DNA message in primary receivers, the “age” of P1 mod¬ cells – the time since they became capable of producing viral particles – is an important aspect of the system state for determining the rate of production of M¬mod. We are therefore interested in P1 mod (t,a), where a is age, and we must use a partial differential equation to keep track of this state variable. The boundary condition for the PDE accounts for P1 mod(t,0), the production of P1 mod through flipping of the switch in P1 ori cells as well as direct infection by M1 mod.
Production of Mori by senders occurs through intact helper plasmid at rate p,
The rate of production of Mmod by P1 mod cells requires consideration of the age of the cells. The rate of production of Mmod by cells of a given age is defined by the function
where viral particle production by a P1 mod cell begins at age j1 and increases with age to maximal production rate bp at age j2, where 0 < b < 1. Production in these cells is likely slower than the rate in senders, p, because one of the M13 cistrons has been picked apart to allow complementation.
The production rate of M¬mod at time t is determined from the distribution of P1 mod cells over all ages and the rate of particle production at each age through a convolution:
Estimation of Model Parameter Values
In the absence of precise parameter values, a literature review provided reasonable ranges for the parameters that allowed us to make qualitative statements about the behavior of the system.
Carrying capacity c was estimated as 1.5*109 cells/ml [1].
Maximal growth rate g was estimated as ln(2)/20 – ln(2)/30  / min, for a doubling time between 20 and 30 minutes.
Maximal growth rate of cells producing viral particles was taken as 1/4g – 1/2g [2].
Adsorption rate k of viral particles to cells was taken as 3*10-11 ml/min [3]. .
The rate of viral particle production p from the intact helper plasmid was taken to be 33 particles per cell per minute [4].
The age at which viral particle production begins in P1 mod cells, j¬1, and the age at which viral particle production reaches a maximum, j2, are assumed to be similar to that of wildtype M13, and these values are taken to be 10 minutes and 40 minutes respectively [4]
The rate of viral particle production through complemented helper plasmid with gene VIII knockout is taken as bp, where p is the rate of production from intact helper plasmid and b is between 0 and 1. It is assumed that picking apart the M13 genome for purposes of complementation will either reduce efficiency or have no effect.
The age at which viral particle production begins in P1 mod cells, j1, and the age at which viral particle production reaches a maximum, j2, are assumed to be similar to that of wildtype M13, and these values are taken to be 10 minutes and 40 minutes respectively [4].
Analysis of Model
Sensitivity to Reduced Efficiency of Viral Particle Production
Since we are using gene VIII complementation of the helper plasmid to control viral particle production in primary receivers, it is likely that efficiency of production will be reduced in these cells relative to senders. We asked the model, “What is the sensitivity to reduced efficiency viral particle production in primary receivers, leading to weaker transmission of Mmod?”
Sensitivity to Reduced Efficiency of Viral Particle Production
Since we are using gene VIII complementation of the helper plasmid to control viral particle production in primary receivers, it is likely that efficiency of production will be reduced in these cells relative to senders. We asked the model, “What is the sensitivity to reduced efficiency viral particle production in primary receivers, leading to weaker transmission of Mmod?”
We simulated the system with varying values of b, the percent rate of viral particle production, relative to p, in primary receiver cells. The system performance measure was the ratio of secondary receivers that receive Mmod vs Mori, expressed as P¬2 mod/P2 ori, measured at the time when all secondary receiver cells have received a message (Figure 3). For these simulations, modification and retransmission were induced, and the half-life of the switch in the original message, once received by primary receivers, was assumed to be 30 mins. Initial populations of senders, primary receivers, and secondary receivers were assumed to be 1000, 10000, and 1000 cells/mL respectively.
Figure 3. Effects of genomic tinkering on system outputs. The viral particle production efficiency in primary receivers, as a percentage of efficiency in senders, plotted against the ratio of secondary receivers that received Mmod vs Mori.
We see that sensitivity is highest when efficiency is low. As expected, increasing efficiency always leads to more successful delivery of M¬mod to secondary receivers. For the parameters used, an efficiency of approximately 0.3 is required for an equal proportion of P2 mod and P2 ori.
Effect of Initial Populations on Receipt of Modified Message
The aim of this system is receipt of Mmod by secondary receiver cells. Mmod can only be produced by primary receivers that are infected with Mori. In a sense, the message must “pass through” the primary receivers before it can get to the secondary receivers. We thus suspect that an increase in the initial population of primary receivers will improve system performance.
To confirm this, we simulated the system with varied initial populations for senders and primary receivers. The system performance measure was the ratio of secondary receivers that receive Mmod vs Mori, expressed as P¬2 mod/P2 ori, measured at the time when all secondary receiver cells have received a message. We also kept track of the simulated time required (Figure 4). For these simulations, modification and retransmission were induced; the half-life of the switch in the original message, once received by primary receivers, was assumed to be 30 mins; and the rate of particle production in primary receivers was assumed to be 50% of the rate in senders.
Figure 4. Initial populations of senders and primary receivers plotted against the ratio of secondary receivers that received Mmod vs Mori.
As expected, we note that the system is most efficient in delivering Mmod to secondary receivers when the primary receiver population is high and the sender population is low, so that the majority of phage particles carry Mmod. We also note that very low initial populations of senders results in a longer time taken to “infect” all secondary receivers with DNA messages.
Sensitivity to Basal Expression of Integrase in Primary Receivers
A key concern in this system is the reliability of the switch on the messaging phagemid. The switch must flip when modification and retransmission is induced, but must hold state when uninduced.
When induced, we expect the half-life of the switch in the original message, once received by primary receivers, to be short. In absence of induction, we expect half-life to be long.
We simulated the system with varying switch half-life, for short and long half-life (Figure 5 a and b). The system performance measure was the ratio of secondary receivers that receive Mmod vs Mori, expressed as P2 mod/P2 ori, measured at the time when all secondary receiver cells have received a message. For these simulations modification and retransmission were induced, and the rate of particle production in primary receivers was assumed to be 50% of the rate in senders. Initial populations of senders, primary receivers, and secondary receivers were assumed to be 1000, 10000, and 1000 cells/mL respectively.
Figure 5. Switch half-life in primary receivers plotted against the ratio of secondary receivers that received Mmod vs Mori for a) short and b) long half-life.
We note that sensitivity to switch half-life is high for short half-life and low for long half-life. Increasing responsiveness of the switch to induction gives increasingly better returns by increasing receipt of Mmod, while reducing unwanted switch flipping due to basal integrase expression gives diminishing returns in preventing receipt of Mmod.
Conclusions
Analysis of our model indicates that,
Delivery of Mmod to secondary receivers is most sensitive to reduced M¬mod transmission efficiency that may result from tinkering with the M13 genome when efficiency is very low. As expected, increasing Mmod transmission efficiency improves delivery of Mmod.
A large initial population of primary receivers with a low initial population of senders improves delivery of Mmod.
Most interestingly, we find that sensitivity of Mmod delivery to switch flipping rate is greatest for fast switch flipping rates and least for slow switch flipping rates. This indicates that improved responsiveness of switch flipping to induction produces increasing returns for system performance, but that improved resistance to uninduced switch flipping to prevent unwanted modification and retransmission produces diminishing returns.
The most significant result for design of this system is the sensitivity of M¬mod delivery to the rate of switch flipping. Leaky expression of integrase in primary receivers must be effectively avoided. Extra measures to prevent leak, such as the T3 promoter system or post-transcriptional regulation, should be employed. In light of the issues with basal integrase expression, this system is most ideal in situations where background levels of modification and retransmission are tolerable.
References:
1. Ausubel FM, Brent R, Kingston RE, Moore DD, Seidman JG, Smith JA, Struhl K. Short Protocols in Molecular Biology. 5th ed. Vol. 1 pp. 1-6.
2. Sambrook J, Russell DW. Molecular Cloning: A Laboratory Manual. 3rd edition. Cold Spring Harbor Laboratory Press, Cold Spring Harbour, NY. 2001.
3. Tzagoloff H, Pratt D. 1964. The initial steps in infection with coliphage M13. Virology. 24:3 pp 372-380.
4. Clackson T, Lowman HB. Phage Display: A Practical Approach. Oxford University Press, NY. 2006.
Read References
Ottawa's Collaboration
1. Wet Lab Collaboration with uOttawa iGEM
The following constructs were designed by uOttawa iGEM for Waterloo iGEM
Inducible Integrase-Expressing constructs
The inducible expression of the PhiC31 and Bxb1 integrase genes was designed by incorporating riboregulators that allow post-transcriptional control of gene expression. Riboregulators include two RNA elements; a cis repressor molecule (also known as the Ribo “lock”) and a trans-activating RNA molecule (also known as the Ribo “Key”). The cis-repressor element is a sequence complementary to the RBS associated with the gene whose expression needs to be silenced (that would be the integrase gene in this case). To control expression of the integrase gene, a Ribo lock was designed to be cloned directly upstream of the integrase gene and its associated RBS (5’ UTR). The sequence encoding the Ribo lock, integrase and its associated RBS as well as a lacI cassette (BBa_Q04121) were to be placed under the control of a strong constitutive promoter (J23119). The trans-activating Ribo key (BBa_I714036) on the other hand, was cloned in the 3’ UTR of the construct and was placed under the control of the PLlac 0-1 promoter (BBa_R0011), which is both LacI-repressible and IPTG-inducible. Simultaneous expression of the LacI cassette and the integrase from a single promoter would ensure repression of Ribo key from the LacI promoter and therefore results in a controlled expression of both PhiC31 and Bxb1 integrase (Isaac et al. 2004).
Inducible Integrase and Recombination Directionality factor expressing constructs:
Inducible PhiC31 and Bxb1 Integrase and Reverse Directionality factor expressing constructs were also designed via the use of riboregulators. As mentioned in section above, controlled expression of  integrase would be achieved via a cis-repressor Ribo Lock and a trans-activating Ribo Key. The design of these constructs required the gene for RDF to be cloned under the control of a LacI promoter (BBa_R0011) with a Ribo lock cloned directly upstream (5’ UTR). For simultaneous expression of both integrase and RDF, the inducible integrase-expressing construct described above was modified to also contain RDF under inducible conditions (Isaac et al. 2004).
Figure 1. (A.) Inducible PhiC31 and Bxb1 Integrase-expressing construct: The gene for each integrase is placed under the control of a strong constitutive promoter (J23119) with a Ribo “lock” directly upstream. The trans-activating Ribo “key” is placed downstream under the control of a PLlac 0-1 promoter. The lacI cassette is also placed under the control of promoter J23119. (B.) Inducible PhiC31 and Bxb1 integrase and recombination directionality factor-expressing construct: The genes for the integrase and the RDF were placed directly downstream from their corresponding ribo “lock” with the integrase under the control of a strong constitutive promoter and the RDF under the control of a PLlac 0-1 promoter. The lacI cassette (BBa_Q04121) was placed under the control of a strong constitutive promoter (J23119).
2. Modelling Collaboration with uOttawa iGEM
We constructed a differential equations model to describe uOttawa’s feed-forward fold-change detection system and implemented it in MATLAB. Parameterization and analysis of the model was performed by uOttawa.
Accomplishments
This year Waterloo iGEM developed a comprehensive project, which attempts to improve an existing system that utilizes the use of DNA as the intracellular molecule for messaging between individual bacterial populations. The process of messaging was improved by making the DNA message 1) Modifiable, 2) Controllable and 3) Re-transmittable.
Wet Lab
Submitted 21 BioBricks to the Parts Registry
Characterized 4 BioBricks
Mathematical Modelling
Modelled the qualitative difference in the dynamics of Bxb1-based and PhiC31-based invertible promoter switch systems using continuous and stochastic methods.
Modelled a two-switch system where the first invertible promoter switch controls the second one, and showed that this improves the “switch-like” behavior of the second switch in multiple copies.
Modelled a system of DNA message re-transmission in co-culture and identified potential modes of failure.
Appendix
Serine Integrase
Our design for modification of a DNA message uses the site-specific recombinase activity of serine integrase enzymes found naturally in various bacteriophages.
In nature, serine integrases (Int) allow phages to integrate into the bacterial genome through site-specific recombination between attachment sites on the phage and bacterial genomes (attP and attB), both of which are between 30 and 60 bp depending which phage serine integrase is considered.  Recombination between attP and attB leaves behind left and right attachment sites (attL and attR) (Fig 1), which cannot be acted upon by Int alone.  Thus, in the presence of Int alone, this recombination is irreversible [1].
In order for the phage genome to excise from the bacterial genome, recombination between attL and attR sites must occur; this reproduces the original phage and bacterial genomes.  To this end, phages that exploit serine integrases also have a “recombination directionality factor” (RDF) which reverses the recombination activity of Int; that is, coexpression of Int and RDF allows recombination between attL  and attR sites to reproduce attP and attB sites, and inhibits recombination of attP and attB (Fig. 1) [1].
Figure 1.  Recombination between attP and attB leaves behind attL and attR sites and allows for integration of the phage genome into the bacterial genome in nature.  RDF reverses this activity.  Adapted from Groth and Calos, 2004 [1].
When attP and attB sites are placed in the correct orientation on the same DNA sequence, the DNA segment between them can be inverted through integrase activity.
This DNA inversion technique has been used to implement passive digital memory in live cells [2]. More recently, this technique has been extended to the implementation of two-input one-output Boolean logic gates [3,4].
References:
Groth AC, Olivares EC, Thyagarajan B, Calos MP. A phage integrase directs efficient site-specific integration in human cells. Proceedings of the National Academy of Sciences USA, 2000, 97 pp 5995-6000.
Bonnet J, Subsoontorn P, Endy D.  Rewritable digital data storage in live cells via engineered control of recombination directionality.  Proceedings of the National Acadamy of Science USA, 2012, 109(23) pp 8884-8889.
Bonnet J, Yin P, Ortiz ME, Subsoontorn P, Endy D. Amplifying genetic logic gates. Science, 2013, 340 (6132) pp 599-603.
Siuti P, Yazbek J, Lu TK. Synthetic circuits integrating logic and memory in living cells. Nature Biotechnology, 2013, 31 pp 448–452.
Read References
M13 Bacteriophage
M13 Bacteriophage
The M13 bacteriophage infects E. coli by attaching to the F pilus and injecting single-stranded DNA into the cell. Because the mechanism requires the F pilus, only E. coli cells carrying the F plasmid (F+ cells) can be infected.
The M13 genome takes on two forms: the single-stranded infective form (IF), and the double stranded replicative form (RF).  When the IF form is initially deposited into the cell, the complementary strand is synthesized by host-encoded enzymes forming the RF form. Protein II nicks the RF form and initiates rolling circle replication, producing additional single stranded copies of the genome that are again converted into RF form by host-encoded machinery. During this process, the M13 genome increases in copy number and viral proteins accumulate in the cell. When protein V reaches a critical concentration in the cell, it binds strongly and cooperatively to single stranded IF DNA, sequestering it to be packaged into viral particles and preventing further accumulation of the RF form [1] (Figure 1).
Figure 1. Life cycle of M13 bacteriophage.
M13 bacteriophage is routinely used to isolate single stranded DNA by “tricking” the M13 viral proteins into packaging heterologous DNA that carries the M13 viral packaging site. This site, called the M13 ori, is necessary and sufficient for packaging of circular DNA carrying the site into M13 viral particles [7]. Heterologous DNA packaged this way can still be delivered to an F+ cell and the cell will maintain the circular DNA as a plasmid. A plasmid carrying the M13 ori is termed a “phagemid”. This aspect of M13 bacteriophage is used in DNA messaging.
Importantly, cells infected with M13 bacteriophage do not lyse. Rather, the bacteriophage is secreted through the cell membrane and the cell continues to grow at ½ to ¾ its normal rate [7].
In our attempt to design a system that regulates DNA messaging by controlling M13 particle production, we had the option of manipulating any of the 11 genes in the M13 genome.
M13’s relatively small genome can be classified in three subsets: structural (genes III, and VI - IX), morphogenetic (genes I, IV, and XI), and replicative (genes II, V, and X). The main function behind each of the proteins can be seen in the table below:
Protein:
Function:
pI
Spans the inner membrane of infected bacteria; interacts with viral pIV and host’s thioredoxin; potentially initiates phage assembly [1, 9, 10]
pII
Nicks at specific site in intergenic region of + strand of replicative form (RF) DNA, starting rolling-circle replication; cleaves single-stranded (ss) product of rolling-circle replication; required  for the initiation  of fl RF DNA synthesis [2, 10]
pIII
Minor coat protein; required for adsorption of phage to sex pili of new hosts [1]
pIV
Resides mainly on outer membrane of infected cell; interacts with pXI and pI within the perisplasm in order to form a gate channel for secretion [1, 9, 10]
pV
Sequesters the viral strands displaced by the rolling circle replication of + strand DNA to prevent their reconversion to double strands [3-7]; needed for the accumulation of  exportable ssDNA; controls the rate of synthesis of pII and pX [8]
pVI
Minor coat protein located at proximal end of filament [1]
pVII
Coat protein that interacts with packaging signal in intergenic region [1]
pVIII
Major coat protein; ~2700 copies formed in cylindrical sheath around DNA [1, 11]
pIX
Minor coat protein, located at end of phage particle where assembly begins [1]
pX
Formed from gX, a subset of gII; required for accumulation of ssDNA; powerful repressor of phage-specific DNA synthesis in vivo; limits number of RF molecules [1]
pXI
Aka pI*; forms gate channel in cell membrane along with pI and pIV [1, 9, 10]
Figure 2: The M13 phage structure and genome. Phage M13 structure with proteins color-coded to genes in the phage genome. Note, the actual phage filament is about 150 times as long as it is wide. [12]
Design Comments
In our attempt to design a system that regulates DNA messaging by controlling M13 particle production, we had the option of manipulating any of the 11 genes in the M13 genome. We chose to remove a gene from the helper plasmid and control viral particle production through complementation. Specifically, we chose to use gene VIII (see the Control section), which was advised my Monica Ortiz. Gene VIII is not a regulatory gene and is last in its operon, so it’s removal is unlikely to interfere with regulation of other genes. Most importantly, a similar approach was previously used to control production of phage ϕX174 [13].
The strong affinity of protein V for single-stranded IF DNA is a concern for our DNA message retransmission design, wherein the primary receiver cells have a helper plasmid with gene VIII knockout and the incoming message carries gene VIII under control of a switch. Since gene V is found on the knockout helper plasmid in the primary receiver, it may be present at very high levels in the primary receiver when the DNA message is delivered. The DNA message will be introduced as single stranded DNA, and protein V may bind it, sequester it, and prevent it from replicating, breaking our messaging system. Indeed, it is known that the presence of protein V alone in a cell renders it immune to subsequent infection by M13 bacteriophage. However, it has also been shown that the presence of protein II in the system reverses this immunity, which demonstrates some regulatory role by protein II [8].
This suggests that expression of proteins II and V may be relevant to our goal of retransmission of a DNA message. Since proteins II and V are important regulatory proteins, fiddling with them might require extensive remodeling of the M13 regulatory system, which is outside the scope of what we thought was reasonable for our iGEM team. However, it might be wise for those attempting similar goals in the future to consider potential complications arising from proteins II and V.
References
[1] Sambrook J, Russell DW. Molecular Cloning: A Laboratory Manual. 3rd edition. Cold Spring Harbor Laboratory Press, Cold Spring Harbour, NY. 2001
[2] Baas, P. D. (1985). Biochim. Biophys. Acta. 825, 111-139
[3] Mazur, B. J. & Model, P. (1973). J. Mol. Biol. 78, 285-300.
[4] Webster, R. E. & Cashman, J. S. (1973). Virology. 55, 20-38.
[5] Mazur, B. J. & Zinder, N. D. (1975). Virology. 68, 490-502.
[6] Geider, K. & Kornberg, A. (1974). J. Biol. Chem. 249, 3999-4005.
[7] Salstrom, J. S. & Pratt, D. (1971). J. Mol. Biol. 61, 489-501.
[8] Fulford, W. "Bacteriophage F1 DNA Replication Genes *1II. The Roles of Gene V Protein and Gene II Protein in Complementary Strand Synthesis." Journal of Molecular Biology 203.1 (1988): 39-48. Print.
[9] Haigh, Nora G., and Robert E. Webster. "The pI and pXI Assembly Proteins Serve Separate and Essential Roles in Filamentous Phage Assembly." Journal of Molecular Biology 293 (1999): 1017–1027.
[10] Russel, Marjorie. "Interchangeability of Related Proteins and Autonomy of Function: The Morphogenetic Proteins of Filamentous Phage f1 and IKe Cannot Replace One Another." Journal of Molecular Biology 227 (1992): 453-462.
[11] Wickner, William, Gail Mandel, Craig Zwizinski, Marjorie Bates, and Teresa Killick. "Synthesis of phage M13 coat protein and its assembly into membranes in vitro." Proceedings of the National Academy of Sciences 75.4 (1978): 1754-1758.
[12] Slonczewski, Joan, and John W. Foster. Microbiology: an evolving science. New York: W.W. Norton, 2011. Web.
[13] Jaschke P, Lieberman E, Rodruiguez J, Sierra A, Endy D. A fully decompressed synthetic bacteriophage øX174 genome assembled and archived in yeast.
Read References
The University of Waterloo’s iGEM – Human Practices team is a diverse team whose goal is to raise awareness on issues regarding synthetic biology. The team also provides the student community information about the latest in the research area of synthetic biology to help the community make informed, accurate and fact-based opinions. Our goal is to strengthen the bridge between the community and their knowledge of synthetic biology while eliminating misconceptions regarding synthetic biology.
In the past year, the team gained valuable experience and information through the projects they worked on. Each project provided more insight on how informed the student community is on the topic of synthetic biology. This further helped us plan out activities that helped us achieve our goal.
One of the main purposes behind the projects this year to enrich, educate and empower the student community. To achieve this goal, various activities were planned to inform the student community about the field of synthetic biology, it’s potentials and how it affects the world around us. These activities provide fundamental knowledge of synthetic biology and it’s uses, allowing the participants and the viewers to form an informed, accurate and fact-based opinion about the topic.
T.I.L. Syn Bio
iGEM is a community of people passionate about synthetic biology – how can we best convey this while reaching out to the student community? Sometimes reading papers and textbooks doesn't quite do it for understanding an idea. As students, we know it can be difficult to grasp some concepts we’re not familiar with. So what’s a better way to communicate an idea? Could social media be the answer? That was the idea behind the VLOG series TIL: Syn Bio.
These series are a quick and effective way to convey the ideas and passion of synthetic biology. The series has many episodes that highlight various aspects of synthetic biology through a mixture of one-on-one videos and animated style videos.              The series begin with episodes explaining  “What is Synthetic Biology?, “Fundamental Advances” and “Cell-to-Cell Communication” (Waterloo iGEM’s 2013 project). This phase of the series is important to orient the viewers and provide some background information.
During the second part of the series, the team takes a fun twist. Using the TIL: Synthetic Biology outreach event footage to compare the viewpoints of students and professors on various topics relating to synthetic biology. The footage from this event is used for addressing many factors associated with the idea of synthetic biology. These factors range from the background knowledge to stigma associated with synthetic biology and from the regulations needed to its future potential.
The series begins with these six videos, leaving the rest of the series to be shaped by viewers. Ultimately, viewers engage with the team about what they want to see in future videos, ask questions they want answered and connect with information from a variety of sources.
The TIL: Synthetic Biology outreach event (used as part of the video series) was well-received. The team came prepared with questions to ask passing students. Students were also given 4-5 days notice via Facebook. The idea behind this aspect of the video was to have it be a surprise. Questions like "do you support GMOs?", "would you eat modified fruit/meat?", "who should be able to practice synthetic biology/should it be open sourced?" and many more were asked. The team was in for some surprises with the diversity of knowledge on campus! We hope incorporating the footage into our series will give participants a fun look into their experience, which they can easily share with their friends and family.             Overall, we hope that the team's work will inspire more leaders to take part and contribute to the advances in synthetic biology, regardless of their academic or professional background.
Intent to Invent
Intent to Invent was hosted on March 07, 2013 at the University of Waterloo’s Quantum Nano Center. The purpose of the event was to:
Connect the students to experts in 3 key industries that use synthetic biology in their processes: Agriculture, Health and Pharmaceuticals.
Bridge the level of discomfort a scientist has in regards to business.
To encourage entrepreneurship within the scientific community by delivering resourceful content from industry experts.
The event promoted open panel discussions of emerging technologies in biotechnology and other advanced biological fields within the 3 industries. Students got a chance to see how synthetic biology is the connected to entrepreneurship, innovation and commercialization. They learned about the industry perspectives and barriers faced by biological companies at different stages in their business model. This talk also encouraged entrepreneurship within the scientific community by delivering resourceful content form industry experts. Each speaker gave a 20-minute mini lecture on topics including: Clinical Trial Drug Development, Commercialization of Biomass and Energy Products and Entrepreneurial Barriers for Biotechnology Companies.
Steve jobs once said, “I think the biggest innovation of 21st century will be the intersection of biology and technology. A new era is beginning, just like the digital one…”. Through sessions such as Intent to Invent, Waterloo iGEM hopes to enrich the experience of science enthusiasts as well as those just curious about synthetic biology and it’s potential. By connecting these students to industry experts, we were able to gage their interests in an innovative and entrepreneurial aspect of science. Many students showed interest in learning more about the bridge between science and business in the future. iGEM received good feedback regarding Intent to Invent, as many students felt that the information they learned was very valuable. Waterloo iGEM provided many students the appropriate connection and information they need to start connecting the scientist in them with the businessman/businesswoman in them.
VeloCity Science
How do we inspire young people to eliminate the gap between science and business? The conventional education system does not provide for this overlap. There is a job unemployment crisis throughout North America, and from the past experience, this is the perfect time to turn to entrepreneurship for solutions. It is time to do what we have done to the IT industry back in the 80s, but with biotechnology  this time. The perfect storm is brewing. Economic downturns have proven to be the best time for entrepreneurship. The Canadian government has seen this and supports entrepreneurial initiatives like these. And most importantly, we have bright young people hungry to make changes to the world.
And that’s why VeloCity Science has been started, an entrepreneurship program that brings together the right business resources (networks, mentorship, legal and financial services, etc.) and the right technical resources (wet-lab space, consumables and equipment) to create kick-ass biotechnology start-ups.
But that’s not all. Above of all these resources, it is the sense of community that is crucial for the success of these entrepreneurs. The University of Waterloo has proven time and time over in providing a strong sense of community to our entrepreneurs through the programs like Accelerator Centre, Communitech Hub, and VeloCity.
That’s the story of VeloCity Science, and we are just starting to write it.
Safety
All experiments are carried out in a BL2 certified lab. Researcher safety when using E. coli, would not be compromised in safety issues due to use of M13. It poses no threat at all to humans. While, the E. coli strain used was relatively harmless, treatment of possible infections may potentially be affected by the antibiotic resistance. Furthermore, spontaneous mutations which result in increased infectivity may result.  However, measures and precautions suggested by the Canadian biosafety guidelines were taken to minimize even the slight chance of infection. Additionally, the working conditions of the lab is already above the recommended safety level of BL1 for usage of M13 viruses. Every member of the team has been trained with safety modules and went through a week of lab training and continuous oversight from the Advisors and his graduate students in the lab. All lab members, including graduate students or other students that were working in the lab, wore appropriate PPEs and disposed all consumable in appropriate biological waste boxes. All surfaces were wiped down with ethanol after use and all glassware was washed immediately after their usage.
The design of the project does not call for release into the public. Additionally, the project design does not produce any harmful products. Through it is possible that the construct could get released to the general public accidentally. But, the product of the constructs only produce fluorescent proteins and it can only be used in a controlled setting with a certain type of chemical present in the environment, thus making it ineffective when released to the public. Because safety of the public and the lab members is our utmost concern, we have ensured that all wastes are thrown out appropriately and autoclaved so that accidentally release would never occur.
There are no additional risks posed by our projects compared to other general BL1 lab concerns. Our bacteria are not pathogenic and are unable to survive outside of the lab environment, because they are unable to effectively compete with other organisms in nature. As stated above, all wastes are discarded according to the Waterloo standards and autoclaved.
There is no potential for harm to human health through use of our constructs, as described above. There is therefore no risk of malicious use.
Our constructs pose no threats to human health, as described above, and scaling up would not change this. Our project is a "fundamental advance" that contributes to the coordination of population-level cellular behavior by allowing messages to be sent between populations of E. coli cells. However, many additional layers of complexity in engineering would be required to use our method to enable pathogenic or otherwise dangerous behaviors in populations of cells.
The cell to cell communication project does include packaging viral particles. Although there are only some proteins of the M13 virus that are packaged and are therefore not a safety risk. M13 is not a safety risk even if its whole genome is packaged. Our project poses no threat to safety and thus we haven't implemented any of these mechanisms.
All the lab and design team members successfully passed the following safety training:
Employee Orientation Training Session
Workplace Violence and Harrassment Training
General Laboratory Safety
WHMIS
Laboratory Biosafety Training
Biosafety Guidelines followed by the Waterloo iGEM Team
University of Waterloo has a Biosafety Committee and can be found here: http://www.safetyoffice.uwaterloo.ca/hse/bio_safety/bsc.html. Although the project has not been discussed with the Biosafety Committee, it has been discussed with several faculty members and has been found to have no risks. Furthermore, the laboratories operating at the University of Waterloo have obtained permits from the Bio-Safety Committee in order to perform intended research. Since the Waterloo iGEM team performs all laboratory work in a parent lab under the guidance of the Masters and PhD students of that lab, the permits obtained by the parent lab cover the projects carried out in the lab.
Canada has very well established biosafety regulations and guidelines which can be found here
The laboratory we work on cell to cell communication project is rated level 1. E.coli strains that Waterloo iGEM team works with falls within the risk level 1. Additionally the laboratory we operate in is certified for work with the above listed risk group of the E.coli.
BioBricks
Bba_K1039000
Φ attP site
Bba_K1039001
Bxb1 PB switch J23119 promoter
Bba_K1039002
Bxb1 PB switch J23118 promoter
Bba_K1039003
Bxb1 integrase
Bba_K1039004
Bxb1 LR switch J23119 promoter
Bba_K1039005
Bxb1 LR switch J23118 promoter
Bba_K1039006
Bxb1 RDF
Bba_K1039007
Bxb1 integrase and RDF
Bba_K1039008
ΦC31 PB switch J23119 promoter
Bba_K1039009
ΦC31 PB switch J23118 promoter
Bba_K1039010
ΦC31 LR switch J23119 promoter
Bba_K1039011
ΦC31 LR switch J23118 promoter
Bba_K1039012
ΦC31 integrase
Bba_K1039013
ΦC31 RDF
Bba_K1039014
ΦC31 integrase and RDF
Bba_K1039015
Inverted GFP
Bba_K1039016
HPdO Helper Plasmid
Bba_K1039017
HPdO with deleted gVIII Helper Plasmid
Bba_K1039018
pVIII of M13 Bacteriophage
Bba_K1039019
pVIII of M13 Bacteriophage with strong RBS (B0034)
Bba_K1039020
pVIII of M13 Bacteriophage with strong constitutive promoter (J23104) and RBS (B0034)
Bba_K1039025
BS1TC
Bba_K1039026
BS2TC
Bba_K1039027
ΦS1TC
Bba_K1039028
ΦS2TC
Bba_K1039029
Bxb1 integrase with the Promoter and RBS
Bba_K1039030
ΦC31 integrase with the Promoter and RBS
Bba_K1039031
Bxb1 att P
Bba_K1039032
Bxb1 att B
Bba_K1039033
Bxb1 att R
Bba_K1039034
Bxb1 att L
Bba_K1039035
ΦC31 att B
Bba_K1039036
ΦC31 att R
Bba_K1039037
ΦC31 att L
Administrators
Lab & Design
Team Leaders
Team
Mathematical Modelling
Dejan Cvijanovic
Brandon Fung
Moses Wanyonyi
Magda Karski
Samantha Hirniak
John Drake
Jordan Lapointe
Tasker Hull
Human Practices
Advisors
Graduate Student Advisors
Web Developer
Acknowledgements
We would like to offer special thanks to the following groups for your help
•  Our advisors for donating your time and intellectual knowledge to the team
•  Members of the Charles lab from University of Waterloo for your generous sharing of lab space, equipment and constructive suggestions to our project.
•  Endy lab from Stanford University for proposing the use of cell-cell DNA messaging and sharing of knowledge and parts.
•  Monica Martinez from Endy lab for your supportive role in our project and expertise in the usage of M13.
•  Our IGEM collaborator this year: team UOttawa for helping us with the making of our constructs and interchange of knowledge.
•  Staff of University of Waterloo Department of Biology for your timeless support and encouragement to our team.
•  Dr. Maud Gorbet's from University of Waterloo for the sharing and expertise on flow cytometry for detecting reporter molecules.
•  Dr. Mungo Marsden from University of Waterloo for the use of fluorescent microscope
•  Susanna Zucca from Magni lab of Università degli Studi di Pavia for your sharing of parts for the single copy switch experiment.
•  Andrew Dhawan for your guidance for the Mathematical Modelling team.
•  Dr. Roderick Slavcev from pharmacy for his guidance with M13 bacteriophage.
•  Dr. Gord Surgeoner, Dr. Catherine Burns, and Nicky Arvanitis for your insight presentation at our Intent to Invent seminar.
•  All members of University of Waterloo IGEM 2013 for your sleepless nights and love for synthetic biology
© 2013 WATERLOO iGEM  |  WWW.iGEM.ORG
Retrieved from " http://2013.igem.org/Team:Waterloo "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
