<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_LZU-CHINA_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:LZU-CHINA/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV><DIV class="bg-other"><SECTION class="generic-banner-safety relative"><HEADER class="default-header"><NAV class="navbar navbar-expand-lg  navbar-light"><DIV class="container"><DIV class="collapse navbar-collapse justify-content-end align-items-center" id="navbarSupportedContent"><UL class="navbar-nav"><LI><A href="https://2018.igem.org/Team:LZU-CHINA">Home</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="https://2018.igem.org/Team:LZU-CHINA/Team" id="1" data-toggle="dropdown">Team</A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:LZU-CHINA/Team">Team</A><A class="dropdown-item" href="https://2018.igem.org/Team:LZU-CHINA/Attributions">Attribution</A><A class="dropdown-item" href="https://2018.igem.org/Team:LZU-CHINA/Collaborations">collaborations</A></DIV></LI><LI class="dropdown"><A class="dropdown-toggle" href="https://2018.igem.org/Team:LZU-CHINA/Description" id="2" data-toggle="dropdown">Project</A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:LZU-CHINA/Description">Description</A><A class="dropdown-item" href="https://2018.igem.org/Team:LZU-CHINA/Design">Design</A><A class="dropdown-item" href="https://2018.igem.org/Team:LZU-CHINA/Parts">part</A><A class="dropdown-item" href="https://2018.igem.org/Team:LZU-CHINA/Experiments">experiments</A><A class="dropdown-item" href="https://2018.igem.org/Team:LZU-CHINA/Safety">safety</A></DIV></LI><LI class="dropdown"><A class="dropdown-toggle" href="https://2018.igem.org/Team:LZU-CHINA/Results" id="3" data-toggle="dropdown">wet lab</A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:LZU-CHINA/Results">result</A><A class="dropdown-item" href="https://2018.igem.org/Team:LZU-CHINA/Futurework">future work</A><A class="dropdown-item" href="https://2018.igem.org/Team:LZU-CHINA/InterLab">interlab</A><A class="dropdown-item" href="https://2018.igem.org/Team:LZU-CHINA/Notebook">notebook</A></DIV></LI><LI><A href="https://2018.igem.org/Team:LZU-CHINA/Model">dry lab</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="https://2018.igem.org/Team:LZU-CHINA/Human_Practices" id="4" data-toggle="dropdown">HP</A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:LZU-CHINA/Human_Practices">intergrated hp</A><A class="dropdown-item" href="https://2018.igem.org/Team:LZU-CHINA/Public_Engagement">public engagement</A></DIV></LI></UL></DIV></DIV></NAV></HEADER><DIV class="container" style="height: 100%"><DIV class="row height align-items-center justify-content-center" style="height: 90%"><DIV class="col-lg-10"><DIV class="generic-banner-content"><H2 class="text-white">Model and Bioinformatic data</H2></DIV></DIV></DIV></DIV></SECTION><DIV class="main-wrapper"><DIV class="location"><P style="text-align: left;"><A href="#p1" style="font-size: 14px;color: #FFFFFF;margin-bottom: 16px;">1.Overview</A></P><P style="text-align: left;"><A href="#p2" style="font-size: 14px;color: #FFFFFF;margin-bottom: 16px;">2.Model A</A></P><P style="text-align: left;"><A href="#p3" style="font-size: 14px;color: #FFFFFF;margin-bottom: 20px;">3.Model B</A></P><P style="text-align: left;"><A href="#p6" style="font-size: 14px;color: #FFFFFF;margin-bottom: 20px;">4.Bioinformatics data </A></P><P class="scroll_top" style="text-align: center;font-size: 16px;">RETURN TOP</P></DIV><SECTION class="about-generic-area pt-100" id="p1"><DIV class="container" style="background: rgb(255,255,255)"><DIV class="col-md-12"><P style="font-size:40px;font-family: 'Segoe Script';color: darkred">Model</P><P class="para2">The model can help us predict the experimental results and guide our experimental design. We designed two models, one to simulate the effect of increasing exosome secretion. It simulates the effect of different exosomes enhancing genes and exosomes booster on the experimental results, providing guidance for our experiment. Another model was used to evaluate the effectiveness of intravenous administration of tetracycline to avoid adverse effects from high concentrations of tetracycline. Therefore, we established a drug metabolism model induced by tetracycline for intravenous administration to evaluate the efficiency of microRNA expression at different concentrations.</P></DIV><DIV class="col-md-12"><H3 class="about-title mb-30">Model A (Tetracycline iv drug model)</H3><P class="para2">To activate our tetracycline-induced promoter, we administered tetracycline intravenously. However, excessive tetracycline can affect gastrointestinal tract, liver, blood system, central nervous system and kidney to different degrees. Tetracycline can also bind with phosphate in dentin and enamel, leading to teeth becoming yellow and bone dysplasia. Therefore, we need to establish a drug metabolism model to analyze and calculate the optimal drug concentration range during intravenous injection, to reduce the impact of tetracycline administration and provide certain mathematical basis for the selection of drug injection time and drug injection amount in practical application.<B>a. The metabolism of tetracycline in the stomach conforms to the rapid perfusion model.</B>the specific calculation formula is as follows ï¼š</P><P class="para2"><B>The parameters are described below the form:</B></P><P class="para2"><B>Here comes our result:</B></P><P class="para2">Figure1. Over time, tetracycline concentrations gradually drop in the blood and enter the stomach tissue. It was predicted that tetracycline in the stomach reached its peak at 10hours, and then its concentration gradually decreased with the degradation of tetracycline.</P><P class="para2">Figure2. This diagram shows the functional relationship between tetracycline and absorption time and absorption coefficient in gastric tissue. The tetracycline concentration decreases gradually with the time of absorption, and the absorption efficiency is inversely proportional to the tetracycline concentration in the gastric tissue.</P><P class="para2"><B>b. when we give the drug intravenously, the change of blood drug concentration conforms to the following formula:</B></P><P class="para2"><B>Here comes our result:</B></P><P class="para2">Figure3. With the increase of time, tetracycline concentration in blood medicine gradually increased until reaching the plateau stage.</P></DIV><DIV class="col-md-12"><H3 class="about-title mb-30">Model B</H3><P class="para2">To increase the expression of exosomes, we added three enhanced genes to simulate the effect of increasing exosomes secretion through the model, providing guidance for our experiment.
							Equations is as follow:
							<B>a.transcriptional rate simulation of three booster genes:</B></P><P class="para2"><B>b.	Simulation of three booster genes transcriptional translation:</B></P><P class="para2"><B>Here comes our result:</B></P><P class="para2">Figure4. the promotional concentrations of the three promotional genes for the secretion of exosomes. Over time, the number of exosomes secreted by cells increased.</P><P class="para2">Figure5. Comparison between the expression levels of three promoter genes and those of non-promoter exosomes. It demonstrates that exosome booster could dramatically increase exosome secrion.</P><P class="para2">Figure6. production rate of exosomes in cell. This model predicts the production rate of exosome over time in cells.</P></DIV><P style="font-size:40px;font-family: 'Segoe Script';color: darkred">Bioinformatics data</P><DIV class="col-md-12"><P class="para2">The effects of different microRNAs on gene expression and gene pathways were analyzed and predicted in the KEGG database. We predicted miR-135b-3p, miR-769-5p and miR-942-5p, respectively, the inhibitory ability of these three microRNAs to other genes, the gene pathways that may be affected and the interactions between these gene pathways and analyzed the interaction network between the same functional proteins that are affected. We also predicted the genetic effects of all three on human cells at the same time. These bioinformatics results suggest that this is the kind of microRNA that has complex regulatory networks. Combined with our phenotypic results (cck-8 analysis, transwell test, flow cytometry analysis, scratch test), miR-135b-3p, miR-769-5p and miR-942-5p finally affected the activity, migration and DNA synthesis of gastric cancer cells through these regulatory networks. In addition, bioinformatic analysis indicates that much work remains to be done. Many regulatory mechanisms still need to be validated and interpreted experimentally. Nevertheless, as an important regulatory device in human organism, microRNA has a promising prospect in the application of cancer treatment.
						</P><H6 style="color: black">1.Biological information analysis of miR-135b-3p </H6><P class="para2">Figure7. According to bioinformatics prediction analysis, miR-135b-3p can inhibit 50 genes expression. The redder the color, the higher the gene expression, and the bluer the color, the lower the gene expression. As you can see from the diagram. When miR-135b-3p expression was at a high level, most genes were at a low level. As the expression level of miR-135b-3p gradually decreased, most genes showed high expression level. However, some gene expression levels are not linearly correlated, which may be due to the interaction between genes.
						</P><P class="para2">Figure8. The gene pathways that miR-135b-3p may affected and the interactions between these gene pathways.
						</P><P class="para2">Figure9.The interaction network between the same functional proteins affected by miR-135b-3p.
						</P><P class="para2">Figure10. GO (Gene Ontology) analysis. The molecular function affected by miR-135b-3p were analyzed by GO database, and there was a total of 20 molecular function related to miR-135b-3p. The color and size of the histogram represent the P value. The darker the color, the smaller the P value, the stronger the correlation between this function and miR-135b-3p.
						</P><H6 style="color: black">2. Biological information analysis of miR-942-5p </H6><P class="para2">Figure11. According to bioinformatics prediction analysis, miR-942-5p can inhibit 50 genes expression. The redder the color, the higher the gene expression, and the bluer the color, the lower the gene expression. As you can see from the diagram. When miR-942-5p expression was at a high level, most genes were at a low level. As the expression level of miR-942-5p gradually decreased, most genes showed high expression level. However, some gene expression levels are not linearly correlated, which may be due to the interaction between genes.
						</P><P class="para2">Figure12. The gene pathways that miR-942-5p may affected and the interactions between these genes.
						</P><P class="para2">Figure13. The interaction network between the same functional proteins affected by miR-942-5p.
						</P><P class="para2">Figure14. GO (Gene Ontology) analysis. The molecular function affected by miR-942-5p were analyzed by GO database, and there was a total of 20 molecular function related to miR-942-5p. The color and size of the histogram represent the P value. The darker the color, the smaller the P value, the stronger the correlation between this function and miR-942-5p.
						</P><H6 style="color: black">3. Biological information analysis of miR-769-5p </H6><P class="para2">Figure15. According to bioinformatics prediction analysis, miR-769-5p can inhibit 50 genes expression. The redder the color, the higher the gene expression, and the bluer the color, the lower the gene expression. As you can see from the diagram. When miR-769-5p expression was at a high level, most genes were at a low level. As the expression level of miR-769-5p gradually decreased, most genes showed high expression level. However, some gene expression levels are not linearly correlated, which may be due to the interaction between genes.
						</P><P class="para2">Figure16. The gene pathways that miR-769-5p may affected and the interactions between these genes.
						</P><P class="para2">Figure17. The interaction network between the same functional proteins affected by miR-769-5p.
						</P><P class="para2">Figure18. According to bioinformatics prediction analysis, miR-769-5p can inhibit 50 genes expression. The redder the color, the higher the gene expression, and the bluer the color, the lower the gene expression. As you can see from the diagram. When miR-769-5p expression was at a high level, most genes were at a low level. As the expression level of miR-769-5p gradually decreased, most genes showed high expression level. However, some gene expression levels are not linearly correlated, which may be due to the interaction between genes.
						</P><H6 style="color: black">4. Biological information analysis of miR-135b-3p, miR-942-5p and miR-769-5p interaction</H6><P class="para2">Figure19. The gene pathways that miR-135b-3p, miR-942-5p and miR-769-5p may affected and the interactions between these genes were predicted by Biological information analysis.
						</P><P class="para2">Figure20. The interaction network between the same functional proteins affected by miR-135b-3p, miR-942-5p and miR-769-5p.
						</P><P class="para2">Figure21. The functional molecules affected by miR-135b-3p, miR-942-5p and miR-769-5p were analyzed by GO database. The redder the color, the higher the gene expression, and the bluer the color, the lower the gene expression. As you can see from the diagram. When miR-135b-3p, miR-942-5p and miR-769-5p expression was at a high level, most genes were at a low level. As the expression level of miR-135b-3p, miR-942-5p and miR-769-5p gradually decreased, most genes showed high expression level. However, some gene expression levels are not linearly correlated, which may be due to the interaction between genes.
</P><H6 style="color: black">Reference</H6><P class="para2">1.Sipponen, P., Kekki, M., Haapakoski, J., IhamÃ¤ki, T., &amp; Siurala, M. (2010). Gastric cancer risk in chronic atrophic gastritis: statistical calculations of cross-sectional data. International Journal of Cancer,35(2), 173-177.
2.Zhu, Z. (2018). [2017 hotspots review and outlook on gastric cancer surgery in china]. Zhonghua Wei Chang Wai Ke Za Zhi, 21(1), 7-14.
3.Zhang, Z. Y., Zheng-Qi, W. U., &amp; Wang, H. J. (2009). An analysis of gastric cancer detection and epidemicity in high incidental area with gastric cancer,wuwei city,gansu province. China Cancer, 19(3), 194â€“201.
4.Chan, S. L., &amp; So, J. B. (2016). Gastric cancer therapy.
5.Barr, W. H., Gerbracht, L. M., Letcher, K., Plaut, M., &amp; Strahl, N. (1972). Assessment of the biologic availability of tetracycline products in man. Clinical Pharmacology &amp; Therapeutics, 13(1), 97â€“108.</P></DIV></DIV></SECTION><FOOTER class="footer-area section-gap"><DIV class="container"><DIV class="row"><DIV class="col-lg-6 col-md-6 col-sm-6"><DIV class="single-footer-widget"><H6>About Us</H6><P>
									We wish to extend our deepest appreciation to the Lanzhou University, The First Clinical Medical College and School of Life Science for the great support and help in this project
								</P></DIV></DIV><DIV class="col-lg-6 col-md-6 col-sm-6 social-widget"><DIV class="single-footer-widget"><H6>Contact Us</H6><P>E-mailï¼š752786236@qq.com 1098385458@qq.com</P><P>Addressï¼šNo. 222, Tianshui South Road, chengguan district, Lanzhou City, Gansu Province,730000, P. R. China.</P><P style="font-size:20px;font-family: 'Segoe Script'">We are the walker on the long journey towards science</P><P class="footer-text">Copyright Â©2018 iGEM LZU-CHINA Team All rights reserved.</P></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>