Team:UCSC/Experiments
Home
Team
Meet the Team
Collaborations
Partnership
Attributions
Sponsors
Project
Overview
Design
Applied Design
Experiments
Results
Demonstrate
Notebook
Human Practices
Outreach
Public Engagement
Safety
Notebook
Weekly Overview
Protocols
Parts
Overview
Improvement
Judging Contact
Experiments
Experiments
LipLox
Experiment 1
Experiment 2
Experiment 3
Protocols
Experiments: The Progesterone Producing Yeast Strain
Our team will use synthetic biology to address insufficient access to contraception by engineering a progesterone-producing yeast. We will construct our gene cassettes and insert them into the Yarrowia lipolytica genome in three parallel experiments. In the preliminary experiment, we will use the strain Y. lipolytica str. FKP393 which has the auxotrophic markers leu2 and ura3 for selection. We will engineer a second strain named LipLox, that will contain the gene for uracil flanked by lox sites loxP and lox71 for use in experiments 1, 2, and 3 below. This will allow us to have a yeast strain that contains lox sites, as well as a strain that now has uracil in a designed location that we can swap in or out as a selectable marker.
CREATE LIPLOX
The first step in our project was to engineer a strain of yeast with an genetic marker in the location we plan on inserting the genes for the progesterone biosynthesis pathway. We also use Cre-lox integration in later experiments, so we needed to enter lox sites into our strain as well. Because Y. lipolytica str. FKP393 is auxotrophic for uracil, we can replace this gene in a specific location as a marker for a second homologous recombination (HR) experiment. When we perform the other HR experiment, the uracil gene will be swapped out, and we can use 5-FOA media as our selection tool.
Homology Arms
We designed homology arms (HA) for homologous recombination that span 1 kb upstream and downstream of the ade2 gene in Y. lipolytica str. FKP393. To prepare HAs for this experiment, and Experiment 1, we amplified the HAs out of the Y. lipolytica str. FKP393 genome using primers flagged with Gibson overhangs (GOs) homologous with the gene cassette ends and the linearized pUC19 backbone as seen in Figure 1. This ensured that the HAs were ready for GA after being amplified out.
Figure 1: Isolation of DNA from yeast, and PCR amplification of upstream and downstream homology arms that surround ade2 out of Y. lipolytica str. FKP393. Primers with Gibson overhang flags are used to create homology arms with Gibson overhangs on each end that associate to their corresponding fragments.
We performed colony PCR to amplify the HAs. After several failed attempts, we found that we needed to extract the genomic DNA (gDNA) from Y. lipolytica str. FKP393 and then PCR-amplify the HAs out of it. To extract the gDNA from Y. lipolytica str. FKP393, we used lithium acetate and sodium dodecyl sulfate (SDS) to lyse the cells, then precipitated the gDNA with ethanol. See our Protocols section below for Yeast DNA Extraction protocol. We used touchdown PCR with the gDNA and flagged primers to amplify the arms out. We checked the success of PCR amplification using gel electrophoresis.
Enter lox sites and ura3 into Y. lipolytica str. FKP393 genome
To create LipLox, which contains the two lox sites used in experiments 2 and 3, as well as the ura3 gene that will act as a selective marker in all experiments, we will Gibson assemble the pOPPY-UC19-yXXU plasmid shown in Figure 2.
Figure 2: pOPPY-UC19-yXXU plasmid in detail. Contains UHA, terminator Tsynth8, loxP site, TEF1 promoter, ura3 gene, Tsynth8 terminator again, lox71 site, and DHA. Will be used to insert lox sites and ura3 into Y. lipolytica str. FKP393.
This plasmid contains our loxP-ura3-lox71 gene block, surrounded by the UHA and DHA to Y. lipolytica str. FKP393 genome, in the linearized pUC19 backbone. The HAs were designed with ends homologous with loxP-ura3-lox71 gene block and pUC19. This design enables GA to seamlessly stitch together all four fragments resulting in pOPPY-UC19-yXXU seen in Figure 2. Because HR requires a linear construct, we will amplify out the product from UHA to DHA. We will perform HR and the loxP-ura3-lox71 fragment will be exchanged for the ade2 gene region between the HAs in the Y. lipolytica str. FKP393 genome (Figure 3).
Figure 3: Creation of pOPPY-UC19-yXXU containing ura3, pUC19 backbone, lox sites and homologous arms using Gibson Assembly. Transformation of Y. lipolytica to create LipLox using homologous recombination.
The yeast recombinants, which were previously auxotrophic for uracil, will now contain ura3 and will grow on uracil-deficient media, allowing us to select for them. As a second method of confirmation, we will extract DNA from transformed Y. lipolytica str. FKP393 cells and use PCR to screen for successful mutants. We will know that we have successful HR results if we observe single bands at the appropriate size (4 kb) on a 1% agarose gel.
THREE PARALLEL EXPERIMENTS
Experiment 1 will use well-studied methods previously tested in Y. lipolytica str. FKP393. We will assemble the plasmid pOPPY-UC19-yP to create the construct necessary to integrate into the LipLox genome to produce Y. lipolytica str. PoPPY. We will Gibson-Assemble the linearized pUC19 plasmid backbone, the UHA and DHA amplified out of the Y. lipolytica str. FKP393 genome, and the five progesterone pathway genes to create pOPPY-UC19-yP seen in Figure 4.
Figure 4: pOPPY-UC19-yP plasmid in detail. Contains UHA, DHA, and the genes ∆7-sterol reductase, ADR, FDX1, P450scc, and 3β-HSD, each of which has a TEF1 promoter, Tsynth8 terminator, and Gibson overhangs (each denoted by GO) to assemble the fragments together, and into the plasmid backbone. Will be used to engineer LipLox with the progesterone pathway. BioBrick prefix and suffix are described in “BioBrick” section.
Figure 5: Creation of pOPPY-19-yP via GA of progesterone biosynthesis pathway genes, HAs, and linearized pUC19 backbone. Transformation of LipLox using HR to exchange gene cassette. Selection of PoPPY using 5-FOA ura+ media.
HR requires a linear fragment, so we will amplify out our construct from UHA to DHA and perform HR to recombine the construct into the LipLox genome. HR will exchange the previously inserted ura3 from LipLox with the gene cassette from pOPPY-UC19-yP. The new yeast strain, named Y. lipolytica str. PoPPY, will be uracil-deficient again, so we will test for successful integration by growing the yeast on 5-Fluoroorotic acid (5-FOA) ura+ media to select for our new strain containing our gene cassette as seen in Figure 5.
Experiment 2 will combine two well-studied experimental techniques. Yeast mediated cloning, which can assemble fragments with Gibson overhangs  inside of a yeast cell; and Cre-lox recombination, which can perform a homologous-recombination-like insertion where there are corresponding lox sites. We will use YMC in S. cerevisiae, due to that technique being well studied in that organism, to create the plasmid pOPPY-XRL2-yP (Figure 6) that contains the gene cassette required to complete the progesterone biosynthesis pathway. We will isolate this plasmid and then transform it into LipLox.
Figure 6: We will linearize the pXRL2 plasmid using inverse PCR to remove one loxP site (loxP cutout). We will re-circularize and re-linearize the plasmid with one loxP site, pOPPY-XRL2-yX. We will insert the genes ∆7-sterol reductase, ADR, FDX1, P450scc, and 3β-HSD, each of which has a TEF1 promoter, a Tsynth8 terminator, and Gibson overhangs (each denoted by GO) to assemble the fragments together, and into the plasmid backbone during YMC to create pOPPY-XRL2-yP.
We will linearize pXRL2 using PCR and remove one of the two loxP sites to produce pOPPY-XRL2-yX. We will re-circularize pOPPY-XRL2-yX before re- linearizing with PCR primers in a region that is homologous with both ends of the gene cassette. We will insert a re-linearized pOPPY-XRL2-yX with the five pathway genes into S. cerevisiae for yeast mediated cloning. We will miniprep out the pOPPY-XRL2-yP plasmid and transform into E. coli. We will insert our gene cassette into the LipLox genome using Cre-lox with lox sites entered in experiment 0 to produce Y. lipolytica str. PoPPY as shown in Figure 7.
Figure 7: Creation of pOPPY-XRL2-yP via YMC in S. cerevisiae using linearized pXRL2 and gene fragments. Transformation of Y. lipolytica str. LipLox using pOPPY-XRL2-yP followed by Cre-lox recombination and 5-FOA ura+ selection to create Y. lipolytica str. PoPPY.
We will amplify pOPPY-XRL2-yP in E. coli and isolate the plasmid using the ZyppyTM Plasmid Miniprep Kit from Zymo Research. We will transform pOPPY-XRL2-yP and the pSH47 plasmid that contains the genes for Cre recombinase into LipLox and perform Cre-lox recombination. The pXRL2 plasmid contains one loxP site upstream of the gene construct, and the downstream end of the gene construct contains the lox71 sequence necessary for Cre-lox recombination. The presence of galactose will induce the expression of Cre recombinase and trigger site-specific recombination between the incompatible lox71 and loxP sites on pOPPY-XRL2-yP and in the LipLox genome to make PoPPY as in Figure 7. The five pathway genes on pOPPY-XRL2-yP and ura3 gene in LipLoxura3 gene through non-selective screening and allow us to select for our desired cells using 5-FOA.
Experiment 3 will be a novel undertaking; while YMC has been tested in Yali, Cre-lox has not. In Experiment 3, we will use YMC in LipLox to create the plasmid pOPPY-XRL2-yP (Figure 6) to insert our gene cassette into the LipLox genome to produce Y. lipolytica str. PoPPY, shown in Figure 8.
Figure 8: LipLox will create pOPPY-XRL2-yP from the five pathway genes and pOPPY-XRL2-yX in YMC. Cre recombinase will induce recombination between the lox sites on the plasmid and the lox sites in the genome to create PoPPY.
We will insert pOPPY-XRL2-yX with the 5 pathway genes into LipLox to create pOPPY-XRL2-yP (Figure 6). The pXRL2 plasmid contains one loxP site upstream of the gene construct, and the downstream end of the gene construct contains the lox71 sequence necessary for Cre-lox recombination. We will amplify pOPPY-XRL2-yP in LipLox, wait for YMC to completely assemble the construct, then transform the Cre recombinase gene-containing pSH47 plasmid into LipLox to perform Cre-lox recombination and create PoPPY.
Riboswitch
Prior to beginning work on our own riboswitch, we decided to characterize the riboswitch biobrick part BBa_K2223004 made by the UCSC 2017 iGEM team, which is intended to detect vitamin b12. We wanted to collect this characterization data to help us better familiarize ourselves with how riboswitches work. We measured the fluorescence and OD600 of samples of E. coli in M9 minimal media and LB with various concentrations of vitamin b12 over a period of time to learn about the behavior of the promoter and the riboswitch itself.
We will amplify our pHR-D17-hrGFP plasmid by transforming it into E. coli using the NEB High Efficiency Transformation protocol for DH5-α competent E. coli cells. After we select for our successfully transformed colonies by plating them on luria broth / ampicillin (LB/ampicillin) media, we will perform a plasmid DNA isolation using the ZyppyTM Plasmid Miniprep Kit. We will measure the concentration of our plasmid isolations using NanoDropTM analysis, examine the identity using restriction digest, and will confirm them with Sanger sequencing. We will use PCR with primers RiLin(r)1 and RiLin(f)1 to linearize pHR-D17-hrGFP and simulate a blunt-end double strand break in the 3’ UTR of the hrGFP gene.
We will order five different DNA gene blocks from IDT, each containing our riboswitch expression platform, one of the five progesterone-specific aptamers describe in Contreras Jim ́enez et al. (2015), and two end sequences that will form overhangs homologous to the blunt-end sequences of the linearized plasmid (Win and Smolke, 2007; Contreras Jim ́enez et al., 2015). We will amplify these inserts using the Apt(f) and Apt(r) primers, which are specific to the HAs held in common by each of them. We will then Gibson-Assemble our riboswitch inserts into linearized pHR-D17-hrGFP.
Next, we will transform these experimental plasmids into DH5-α competent E. coli cells for amplification. After isolating these plasmids from E. coli, we will linearize the plasmids again using RiHA1(f) and RiHA(r) to create a linear portion of DNA consisting of the TEF(136) promoter, cyc1 terminator, hrGFP gene, riboswitch structure, and HAs suitable for integration into the Yali genome.
Following transformation into Yali, we will use replica plating to compare identical colonies in the absence and presence of progesterone. If any colonies show working riboswitch constructs, we will isolate and linearize plasmid DNA from these previous cultures using our yeast miniprep protocol, provided in appendix A.4, and PCR. We will then transform these linearized plasmids into Y. lipolytica str. PoPPY cells. Because of our aptamers high binding affinity, they are not suitable for measuring a relatively large dose of progesterone in a consumer’s culture. If one of our five aptamers shows functionality within our riboswitch structure, we will run trials using the column electrophoresis SELEX method described by Mosing and Bowser (2009) and random mutagenesis via error-prone PCR, which is described in a paper by McCullum et al. (2010), to identify variants that show a decreased sensitivity for progesterone which will trigger fluorescence at higher concentrations (Mosing and Bowser, 2009; McCullum et al., 2010).
Detection of Progesterone
We picked colonies from Yeast Mediated Cloning transformations of Y. lipolytica and S. cerevisiae that grew on our leucine-deficient plates. We picked 30 colonies from the leucine deficient plates and inoculated them in 5 mL of leucine-deficient liquid media, in 15 mL falcon tubes. We spun them down at 3,000 RPM for 10 minutes to pellet the cells.
After pellets formed, we dumped the excess media and added glass beads to each tube. We added hexane and MilliQ water to the tubes, and vortexed on high for five minutes. After the phase separation occurred, we extracted the hexane layer sitting above the water and beads, and transferred each sample to a new conical glass tube. We blew off the hexane to dry it, using air instead of nitrogen to avoid loss of sample. After the aqueous phase had been vaporized, the residue was resuspended in DMSO, and vortexed to ensure all material was dissolved. We then performed a dot blot using nitrocellulose blocked with BSA in TBS, and washed with TBS/TWEEN. We used anti-progesterone-HRP antibodies from Santa Cruz Biotech (sc-53423 HRP) for detection and imaged using a CCD imager.
Protocols
Designing PCR Primers
E. Coli Miniprep
Yeast Miniprep
Yeast DNA Extraction
Progesterone Detection
Sourced Procedures
Designing PCR Primers
Our teaching assistant, McKenna Hicks, taught us the following methods for primer design.
Identify the DNA sequence to amplify or stitch together with overlap extension PCR (OE-PCR). During PCR, primers anneal to their complementary sequence and recruit DNA polymerase to extend the sequence in the 5' to 3' Direction.
Design a forward (5'- 3') and a reverse primer (3'- 5') to amplify both the top and bottom DNA strands. The ideal primer length is between 18 and 23 bp, but this can be changed to accommodate other constraints.
For the forward primer, select a sequence of base pairs (bp) of optimal length.
Repeat Step 3 for the reverse primer and generate the reverse complement of the reverse primer sequence.
Analyze each sequence using a computational primer tool (such as IDT's Oligo Analyzer) and record the GC content and the melting temperature (Tm) for each primer.
The optimal GC content is between 40-60%, and the optimal Tm range is between 52-62℃. Ideally, the difference in Tm for the forward and reverse primer pair should not exceed 5℃.
If the GC content exceeds 60% for either primer, add High GC Enhancer to the PCR reaction mix.
Check that the free energy (ΔG) value for self-dimers and hetero-dimers on each primer is above -10 kcal/mol to avoid primers spontaneously self-annealing or annealing to each other.
Check that hairpin formations do not exceed 30℃ in each primer. PCR uses different temperatures for each thermocycler step; the lowest temperature used during the replication step for primer annealing is 58-60℃. If hairpins form around this temperature, the primer binding efficiency may be reduced and yield less product. However, if the primer Tm is above the hairpin formation temperature, and if the potential hairpin leaves the 3' end of the primer free for annealing, the primer should still anneal properly without reducing PCR Efficiency.
E. Coli Miniprep
The following protocol was adapted from the ZyppyTM Plasmid Miniprep Kit procedure from Zymo Research.
First prepare your cell samples: centrifuge the cell cultures (which should be 5 mL in conical tubes) for 15 minutes at 3,750 rpm and 30℃.
During waiting time, grab a 250 mL beaker and add roughly 50 mL of 10% bleach. Grab some foil, label waste, and cover the top of the beaker. Use this beaker to discard biohazardous supernatants and flow-throughs throughout your miniprep.
Collect Zyppy columns, Zyppy collection tubes, and 2 sets of 1.5 mL microcentrifuge tubes to match the number of samples you centrifuged. Place the columns over the collection tubes. Be sure to label all vessels accordingly. Place a kimwipe over the open columns.
After centrifugation, discard the supernatants into your waste beaker. Be sure to replace the foil lid afterwards.
Add 600 uL Milli-Q water into each of the conical tubes containing the cell pellets. Resuspend each by slowly pipetting up and down; use that same pipette to transfer the resuspended mixture to its respective clean microcentrifuge tube. Change tips between samples.
Add 100 uL of 7X Lysis Buffer (blue) and mix by inverting the tube 4-6 times. Proceed to next step within 2 minutes. After addition of 7X Lysis Buffer, the solution should change from opaque to clear blue, indicating complete lysis.
Add 350 uL of cold Neutralization Buffer and mix thoroughly. The sample will turn yellow when the neutralization is complete and a yellowish precipitate will form. Invert the sample an additional 2-3 times to ensure complete neutralization.
Centrifuge at 13,000 rpm for 3 minutes.
Transfer the supernatant into the provided Zymo-SpinTM IIN column, which should be on top of a collection tube. Avoid disturbing the cell debris pellet.
Centrifuge the column + collection tube for 15 seconds at 13,000 rpm (stop the centrifuge when it reaches 15 sec).
Discard the flow-through and place the column back into the same Collection Tube.
Add 200 uL of Endo-Wash Buffer to the column. Centrifuge for 30 seconds at 13,000 rpm. Discard the flow-through.
Add 400 uL of ZyppyTM Wash Buffer to the column. Centrifuge for 1 minute at 13,000 rpm. Discard flow-through. Be sure to close cap tightly on wash buffer so that ethanol does not evaporate.
Transfer the column into a clean 1.5 mL microcentrifuge tube then add 50 uL of Milli-Q water in place of Elution Buffer (EB) and add it as close to the filter as possible, being careful not to touch it (use Milli-Q instead of EB to prevent contamination from salts and EDTA). Let stand for at least 10 minutes at room temperature. Incubate longer if needed.
Centrifuge for 30 seconds at 13,000 rpm to elute the plasmid DNA.
Yeast Miniprep
The following protocol was adapted from the ZyppyTM Plasmid Miniprep Kit procedure from Zymo Research.
To prepare your cell samples: centrifuge the cell cultures (which should be 5 mL in conical tubes) for 15 minutes at 3,750 rpm and 30C.
During waiting time, grab a 250 mL beaker and add roughly 50 mL of 10%  bleach. Grab some foil, label \waste", and cover the top of the beaker. Use this  beaker to discard biohazardous supernatants and flow-throughs throughout your miniprep.
Collect Zyppy columns, Zyppy collection tubes, and 4 sets of 1.5 mL microcentrifuge tubes to match the number of samples you centrifuged (you can get the Zyppy items from the yellow miniprep kit). Place the columns over the collection tubes. Be sure to label all vessels accordingly. Place a kimwipe over the open columns.
After centrifugation, discard the supernatants into your waste beaker. Be sure to replace the foil lid afterwards.
Add 600 uL Milli-Q water into each of the conical tubes containing the cell pellets. Resuspend each by slowly pipetting up and down; use that same pipette to transfer the resuspended mixture to its respective clean microcentrifuge tube. Change tips between samples.
Add 100 uL of 7X Lysis Buffer (blue) and add 50 uL of small glass beads. Vortex for 5 min. Let stand to allow beads to settle.
Transfer the supernatant to a fresh microcentrifuge tube.
Add 350 uL of cold Neutralization Buffer and mix thoroughly. The sample will turn yellow when the neutralization is complete and a yellowish precipitate will form. Invert the sample an additional 2-3 times to ensure complete neutralization.
Centrifuge at 13,000 rpm for 10 minutes.
Transfer the supernatant into the provided Zymo-SpinTM IIN column, which should be on top of a collection tube. Avoid disturbing the cell debris pellet.
Centrifuge the column and collection tube for 45 seconds at 13,000 rpm.
Discard the flow-through and place the column back into the same Collection Tube.
Add 200 uL of Endo-Wash Buffer to the column. Centrifuge for 30 seconds at 13,000 rpm. Discard the flow-through.
Add 400 uL of ZyppyTM Wash Buffer to the column. Centrifuge for 1 minute at 13,000 rpm. Discard flow-through. Be sure to close cap tightly on wash buffer so that ethanol does not evaporate.
Transfer the column into a clean 1.5 mL microcentrifuge tube, then add 50 uL of Milli-Q water in place of Elution Buffer (EB) and add it as close to the filter as possible, being careful not to touch it (use Milli-Q instead of EB to prevent contamination from salts and EDTA). Let stand for at least 10 minutes at room temperature. Incubate longer if needed.
Centrifuge for 1 minute at 13,000 rpm to elute the plasmid DNA.
Yeast DNA Extraction
We adapted the following procedure from Looke et al. with advice from our teaching assistant, McKenna Hicks.
Add 100uL Y. lipolytica in YPD to 100uL 200mM LiOAc and 1% SDS solution. Incubate in water bath at 70℃ for 15 minutes.
Add 300uL of 96% EtOH. Incubate in freezer overnight.
Centrifuge at 8,000 rcf at 4℃ for 30 minutes. Discard supernatant.
To wash pellet: resuspend in 1ml 70% EtOH, incubate for 5 minutes then centrifuge for 10 minutes with the previous speed and temperature conditions. Discard supernatant.
Repeat wash step once.
Resuspend in 100uL 0.5X TE or Milli-Q water; if pellet is not completely resuspended add 0.5X TE or Milli-Q water in 75uL increments until completely resuspended.
Use NanoDrop to determine concentration and quality of the DNA.
Progesterone Detection
Pick 30 colonies from the leucine deficient plates and inoculate them in 5 mL of leucine-deficient liquid media, in 15 mL falcon tubes. Spin them down at 3,000 RPM for 10 minutes to pellet the cells.
After pellets form, dump the excess media and add 0.2 g of 500 um glass beads to each tube.
Add 1 mL of hexane, and 1 mL of MilliQ water to the tubes, and vortex on high for five minutes.
After the phase separation occurrs, extract the hexane layer sitting above the water and beads, and transfer each sample to a new conical glass tube.
Blow off the hexane to dry it, using air instead of nitrogen to avoid loss of sample.
After the aqueous phase has been vaporized, resuspend residue in 100 uL of DMSO, and vortex to ensure all material is dissolved.
Perform a dot blot using nitrocellulose blocked with BSA in TBS (25mg/ml BSA, 10mM Tris, HCl, 150mM NaCl), and wash with TBS/TWEEN (10mM Tris, 150mM NaCl, HCl, 0.05% TWEEN).
Use anti-progesterone-HRP antibodies from Santa Cruz Biotech (sc-53423 HRP).
Add HRP chemiluminescent substrate and image.
Sourced Procdures
The following is a list of procedures we directly sourced for our project.
Gibson Cloning
Column Electrophoresis via SELEX
Random Mutagenesis via Error-prone PCR
High Performance Liquid Chromatography and Mass Spectrometry
