Team:UCSC/Notebook
Home
Team
Meet the Team
Collaborations
Partnership
Attributions
Sponsors
Project
Overview
Design
Applied Design
Experiments
Results
Demonstrate
Notebook
Human Practices
Outreach
Public Engagement
Safety
Notebook
Weekly Overview
Protocols
Parts
Overview
Improvement
Judging Contact
Notebook
Week of
June 26th
July 2nd
July 9th
July 16th
July 23rd
July 30th
August 6th
August 13th
August 20th
August 27th
September 3rd
September 10th
September 17th
September 24th
October 1st
October 8th
October 15th
Group
GA and HR
OE-PCR
YMC and Cre-Lox
BioBrick
Riboswitch
Other
Week by Week Overview
In Detail
Week of June 26th
Overview
Day 1 was mutiny. Five students had met with a sponsor in the morning and when the two captains returned to the room, the students were milling around, whispering under their breath, and fidgeting. The problem was with motivation towards the project, as some students felt that it wasn’t novel enough. We had found a 1998 paper by Duport et al. that explained how their team developed yeast to produce pregnenolone and progesterone - the things we wanted to make. It turns out though that the paper didn’t lead to anything, the strain has been lost, and it was a good proof of concept for our idea. From the uprising we learned to communicate thoroughly.
Week of July 2nd
Overview
Our yeast strain is going to have all the genes to make progesterone, as well as the lactase enzyme so that we can feed it lactose and it break it down to glucose and galactose. We figured out after much deliberation how we could have our friends in developing nations get the accurate dose of progesterone. We learned from our mentor that we could use a technique to measure the weight of an object using a coat hanger, and depending on weight, calculate the amount of lactose inside that container. This would be the dairy that the people have on hand. Depending on weight, we will have a chart that says how much lactose is in that container. The people will be guided to portion a certain amount of the lactose into an ice-cube-like container and grow the yeast in there. Because there will be a specific amount of lactose in each section, and our promoters are sensitive to galactose, our yeast will stop producing progesterone after it consumes all of the lactose.
We have found multiple ways to insert our genes into Y.lipolytica and we made pros and cons lists for each method to decide which one would be the most effective and time-efficient. The methods are regular homologous recombination using homologous arms on our plasmid, EasyCloneYALI: CRISPR/Cas9‐based technique for engineering Y.lipolytica specifically, and Drag and Drop cloning.
Week of July 9th
Overview
We confirmed that using a constitutive promoter would be better than an inducible one, at least initially, because it will use up energy to keep producing progesterone after the growth media has been consumed. Our website has gone through several prototype designs, but it seems like we may be getting close to a final design we would like to use to represent our work. One fun feature that is a work in progress is an interactive map that will, when moused over, provide information about different countries and our outreach there. We have also been working on finalizing our logo, which will include the name of our project: PoPPY. PoPPY stands for Portable Progesterone Production in Yeast, an accurate description of the goal of our project. A low this week for the team was realizing that most of the survey responses received from one of the groups we reached out to about contraceptive accessibility were unreliable, since many answers between supposedly different women were the exact same, word for word. However, we hope that this was a problem specific to the group, and not the norm. We have some more legitimate groups who are willing to send us photos of the women filling out the surveys, and we hope to see good results. A high for the team this week was finally understanding how Cre-Lox recombination, a method that we will use in one of our experiments to integrate our gene cassette into the yeast genome, works. We now expect to run three parallel experiments to get to our end goal: a control using Gibson and homologous arms to create our gene cassette and integrate it into our Y. lipolytica genome; assembly of the plasmid inside S. cerevisiae using yeast-mediated cloning,  isolation and insertion of the plasmid into E. coli for amplification, and insertion of the gene cassette into Y. lipolytica using lox sites; and assembly of gene cassette/plasmid inside Y. lipolytica and integration into genome using lox sites. For each of the three experiments, we will begin by inserting lox sites into the Y. lipolytica genome using homologous recombination.
Week of July 16th
Overview
About half the team spent the week drafting our thesis. While some of us were writing, others were meeting with professors on campus to discuss our ideas and to pick up some S. cerevisiae strains to use as our experimental organism. We learned from Rohinton Kamakaka that we could insert Cre recombinase into our yeast using a plasmid, so now we are in pursuit of a method to both express and remove Cre recombinase once we no longer need it. Our yeast parents have been growing our Y. lipolytica and amplifying the plasmids that we have received. Our wiki boys have almost finished formatting the website to their (or our) liking and are now adding more content about our project itself. A few of our team members scheduled an interview with a representative from Family Planning 2020, but due to timing issues, had to reschedule.
Week of July 23rd
Overview
Other
This week, our team wrote a thesis! Or, more accurately, finished writing it. The majority of us put our brains together to create a draft and we chose one excellent editor to take control of the paper. She enlisted grammar police, style monitors, and LaTeX formatters to help her revise our drafted ideas into one cohesive overview of our work to-date. Each of us also wrote about our own contributions to the project and personal reflections describing our feelings towards the journey thus far.
Also, our two captains and our PI met Dean Wolf, Dean of Baskin School of Engineering at UC Santa Cruz, Roger Trippel, Senior Director of Development and Individual Giving, and Abigail Kaun, Special Assistant to the Dean. We discussed the project to-date, our outreach efforts, and shared ideas for managing both our and future iGEM UCSC team’s funds.
Created first batch of YPD medium and also created 80% glycerol freeze stocks of our Y. lipolytica strain. We attempted to miniprep our pXRL2- and pUC19-containing E. coli but we made a mistake by switching the order of wash buffers used from the Zyppy Miniprep Kit. The Nanodrop curve of the plasmid sample was ragged and thus we will need to attempt this again.
Week of July 30th
Overview
Gibson Assembly
Gene Amplification
Riboswitch
Other
This week, the captains held check-ins with each individual on the team to hear concerns and alleviate worries, since we are now halfway through the summer. While the captains were busy with meetings, the rest of the team was working hard as always. Our modeling team started a growth curve for Yarrowia lipolytica so we can better understand the yeast’s growth rate, our riboswitch team began lab work, our plasmid team started linearizing plasmids, and our outreach team continued to update the wiki page and started creating a presentation for high schoolers.
Four of our team members and our PI took a vacation to Minneapolis that wasn’t much of a vacation at all. The small group toured Medtronic's Operational Headquarters, Physiological Research Laboratories, and the University of Minnesota's Visible Heart Lab. Almost every other second of the trip was spent on the BMES Coulter College Conference, which required teams to come up with a medical solution to a need. Our team focused on hypertension in resource-constrained settings and developed a solution in the form of a blood-pressure-monitoring phone case and app. Even after the conference, the fivesome was hard at work; they met with the University of Minnesota iGEM team, toured their lab, and gave advice about human practices, outreach, and fundraising.
The first step to prepare for Gibson Assembly was to linearize pUC19 and pXRL2 via PCR. We performed PCR Over the course of this week we had bands in strange and random locations. We were led to believe our plasmids were not what we believed them to be. Eventually it was discovered that we were using an incorrect amount of Q5 2x master mix which caused the PCR to not work properly.
During the first lab week, we rehydrated our gene blocks from IDT and performed overlap extension (OE) PCR on three of the genes (delta7, ADR, and FDX1). We used touch-down and touch-up thermocycler settings which resulted in many different sized bands on an agarose gel. To reduce the amount of nonspecific primer annealing in these reactions, we decided to only use touch-down thermocycler settings and to put the primers on the side walls of the PCR tubes to limit the Q5 polymerase exposure to the primers. The exonuclease activity of the Q5 polymerase chews back the ends of our primers and causes non specific annealing, so we made sure the primers were added last before putting the reactions into the thermocycler. We also decided to run OE PCR reactions without primers to allow the overlapping regions of the genes to anneal and form the full length genes which could be used as templates for regular PCR reactions.
Since our project would eventually require separate experiments with 5 different inserts, our first goal in lab was the amplification of our experimental base plasmid. We first created a stock of ampicillin selective plates with a concentration of 100ug/mL. We carried out a transformation with NEB DH5α chemicompetent cells and created a glycerol stock of leftover cells after plating. We plated out transformants on 3 plates at original strength, 10-fold, and 100-fold dilutions using glass beads. Our platings were successful, and we used the 100-fold dilution plate to isolate single colonies and create a monoclonal culture of transformants. We also learned how to plate with a sterile wire loop in order to keep the number of plates used to a minimum. Every protocol--pouring plates, transformations, platings, and creating glycerol stocks--was new to us and required some reading of protocols from NEB and other sources.
Created 80% glycerol freeze stocks of four S. cerevisiae strains obtains from the Heinrich lab at UCSC. Did a second attempt of miniprepping the pXRL2- and pUC19-containing E. coli, but unfortunately had salt contamination in our resulting sample. We attempted this a third time by tossing out flow-throughs between each step of the miniprep procedure and obtained workable plasmid samples for pXRL2 and pUC19.
Primers came in and were rehydrated.
We performed our first attempt to PCR amplify out the ade2 homologous arms from Y. lipolytica using colony PCR. No amplification was observed. For our second amplification attempt, we used Touchdown PCR and used increasing concentrations of yeast cells in the PCR reaction mixes. Still no results.
Week of August 6th
Overview
Gibson Assembly
Gene Amplification
Riboswitch
Other
So many people are in the lab now! Because so many of our small lab teams have transitioned to lab work, the “Yeast Parents”, or team in charge of maintaining the yeast and the lab, gave everyone a tour of the space to keep everything organized (and keep us all in line).
Unfortunately, with great numbers of people in lab comes great margin for mistake. We burnt through a ridiculous amount of Q5 polymerase this week because we miscommunicated the concentrations needed to perform PCR. However, the team took this as a learning experience and practiced concentration math and master-mix-making. We are also going to begin a crowdfunding campaign to supplement the team funds when/if errors arise in the future.
Outreach is really cool! Three of us, plus our PI (of course), spoke to Martyn Smith, Managing Director of the Family Planning 2020 Secretariat, over the phone. He nearly gave us heart attacks when he offered to help us reach local agents familiar with the regulatory environments in the countries we wanted our project to extend to. Aside from the FP2020 call, two of our members have been diligently working on outreach at home. They’ve started to develop a game as an interactive way for high schoolers to join in when we present our project to them later this month.
After double checking that we were using the right volumes for our PCR reaction, we were able to linearize pUC19 and pXRL2. We then confirmed this by running it on a 1% agarose gel seen below.
We ran OE PCR reactions without primers for all five genes but still saw multiple bands for some of the genes. We discovered that the master-mix concentration was too high for all of the reactions we had performed, so we changed the amount of master-mix in each reaction to ensure the concentration was 1X rather than 2X. After changing the master-mix concentration, we ran more OE PCR reactions without primers and selected certain reactions with the least amount of nonspecific bands for templates in our regular PCR reactions.
Our over-the-weekend cultures had little growth after 24 hours, but showed adequate growth come Monday morning. After discussing this with other teammates, we found it was most likely due to improper aeration of our samples, which were stored upright with a tight cap in the incubator. We then performed a plasmid isolation followed by a restriction digest to test the identity of our isolated plasmid. Our first digest was unsuccessful, which we hypothesized was because of the age of our enzyme sample. Our next digest, performed with our most recently acquired enzyme, showed the expected band pattern after altering the contrast on the gel image. We also attempted a PCR linearization of our amplified plasmid which was unsuccessful. Through these experiments we learned how to better analyze gel images by increasing exposure, brightness, and contrast, as well as proper incubation practices for liquid cultures. We also ran into several issues with budgeting time correctly in lab, which prompted us to change how we schedule our lab procedures to create as little “dead space” as possible.
For our third attempt to amplify out ade2 homologous arms from Y. lipolytica, we instead did a serial dilution to ensure that we were not using too many cells in the PCR reaction mixes. Again, no amplification was observed.
Week of August 13th
Overview
Gibson Assembly
Gene Amplification
Riboswitch
Other
Each year, the UCSC iGEM team presents to the Bioethics class during the second Summer Session. A few team members diligently prepared a presentation about our questionnaires and project for that purpose. Several of us attended the presentation, and afterwards broke into small groups with students in the class to ask and answer questions about our ethics. Because of this presentation, we learned about the IRB at UCSC and how we needed to get our questionnaires approved, since we were technically working with “human subjects” if we sent it out.
Another small group designed a poster to be displayed at the Annual Physical and Biological Sciences Summer Research Symposium. This poster included details about our three parallel experiments, our research, and our outreach. Three team members presented the information to interested audiences at the Symposium.
After successfully linearizing pUC19 and pXRL2, we repeated the same experiment with larger volumes to get more product for future use. Using this product, we performed several DPN1s and clean and concentrate experiments. The first four tests were failures. We used both our advisor’s reagents and Zymo’s reagents. Our solution that eventually worked was to prewarm the 30 uL water we were using as the elution buffer and let it sit for several minutes, in the final step. On the fifth experiment, the clean and concentrate worked but still not very well. We tried one more time using only 10 uL water to elute and got sufficient concentrations for our Gibson reactions.
We tried many PCR reactions with the new templates but still saw multiple bands for each gene, so we decided to extract the correct bands from the gels and use those as our templates. We tried using larger reaction volumes in the gels to get more DNA from the gel extraction, but we didn’t get clear enough bands to properly extract the correct sized genes. We also tried amplifying the loxP-URA3-lox71 gene cassette with our existing primers, but the reaction didn’t work due to the difference in annealing temperatures between the forward and the reverse primer.
This was the week in which we discovered our collective error in the PCR protocol we had used up until this point. This was a lesson to us and the rest of the team about blindly following protocols without questioning them. Upon redoing our PCR experiment with the corrected protocol in a 10uL reaction, our gel showed bands indicating a successful linearization. We also went through the protocol for the creation of chemicompetent yeast cells in anticipation for our transformation into Y. lipolytica. Protocol was successful and yielded 2.5 mL of glycerol stock of our chemicompetent Y. lipolytica.
The glycerol stocks of E. coli were successfully streaked to verify viability. Contacted lab support for pH meter electrode issues.
Week of August 20th
Overview
Gibson Assembly
Gene Amplification
YMC
Riboswitch
Other
This week, we began to think about our thesis again. Whew. A few of us began reorganizing the information we needed to write about to make the paper more clear. We also had a meeting at the end of the week about expectations for writing during the coming week.
After finding out about the IRB, we applied for exemption. We then found out that we would have to revise our questionnaires. Appropriate changes were made and the application was submitted again (this happens several more times in the next week or so).
Some of us considered different design and selectable marker options to test integration of genes into the yeast genome. We learned that one of our steps lacks a reliable selectable marker test, but that replica plating may provide sufficient levels of certainty.
Our several lab groups worked tirelessly to make more media, to check for deletions in plasmids, and to design primers that actually work.
Now that we have lox-URA3-lox, linearized pUC19, and amplified homologous arms, we are able to begin Gibson Assembly. We did two gibsons this week. The first Gibson includes the parts mentioned above and the second is our linearized pXRL2 along with several gene fragments: ADR, FDX, Delta7H, p450SCC, and 3BH. After combining the neccessary components with the Gibson Master Mix, we put it in the PCR machine for an hour and then stored it on ice. The following day we inoculated the gibson products on high copy LB-AMP plates. The pUC19 plate grew normally but the pXRL2 plate lawned.
We then picked colonies from the pUC19 plate and transformed it in liquid cultures. We then performed colony PCR to check if our Gibson assembly was successful. It likely did not work. The gel showed very bright bands in the wrong areas but a faint band where our desired product would be. With this information, we made streak plates of the potential successful colonies in hopes to isolate what we believe to be our desired gibson product and threw the rest of the cultures out.
Once we created a gel with clear, distinct bands for DNA extraction, we tried using the Bioline Isolate II PCR and Gel kit to retrieve the DNA from our agarose gel. At first we used a different binding buffer than the one provided in the kit, but the final extraction had a high salt concentration that made the DNA unusable as a PCR template. We tried using the Zymogen clean and concentrator kit to further purify the DNA, but there wasn’t enough DNA to use for PCR reactions after the clean-up. We performed two more gel extractions using the reagents from the kit but still didn’t get clean enough DNA for further PCR amplification. We finally decided to order the full genes and use the primers from the OE PCRs to amplify the full length genes.
After the Gibson team removed one loxP from pXLR2, we used the KLD enzyme mix from a site-directed mutagenesis kit to recircularize the plasmid. We plated the transformants, and then made liquid cultures from colonies two days later. We designed and ordered primers for colony PCR to check that loxP was properly removed. We prepared for yeast mediated cloning by preparing 10X TE buffer and making liquid cultures of Y. lipolytica and S. cerevisiae.
After a successful test reaction at 10uL, we repeated our linearization PCR with a volume of 50uL. Our confirmation gel indicated a successful linearization, although some of the sample streaked above the 10 kb ladder.
We then attempted a transformation of our new chemicompetent Y. lipolytica using our control D17 plasmid. Our transformant cells were then incubated in liquid selective media for outgrowth. We also attempted a transformation of our linearized PCR product from the 50 uL reaction, but later realized that the linearized plasmid was unfit for expression of any genes at this point and so the transformation plate was discarded.
Created YNB+CSM-Leu medium (leucine-deficient) and YNBcasa (uracil-deficient) medium. Attempted to transform Y. lipolytica with pXRL2, ultimately to test if we can miniprep yeast with our protocol. Unfortunately there was no obvious yeast growth on the transformation plates.
Week of August 27th
Overview
Gibson Assembly
Gene Amplification
YMC
Riboswitch
Other
Thesis week is upon us! This time around, we struggled a lot less to work as a group and write as a group. The early organization probably helped. We turned in a much more cohesive paper.
We received an invitation to speak at TEDx! This is exciting. We get to tell people about our project, but we also get to bring more attention to the unmet need for modern contraceptives! Our team is really happy about the opportunity to present on such a large platform.
In lab, we had some successful yeast and bacteria transformations, some successful Gibsons, and confirmed Y. lipolytica’s antibiotic resistance.
We also finally received IRB exemption for our questionnaires! This means we can now officially use our questionnaires to gather information about the women who may not have adequate access to modern contraceptives.
This week we carried out more inoculations of pUC19 and pXRL2. After confirming growth in the liquid cultures, we miniprepped both sets of plasmids and got very good results on the nanodrop. Using the most concentrated cultures, we sent out our samples for sanger sequencing. We also carried out another Colony PCR of our old gibson in order to isolate our desired plasmid. It failed.
Once the full length genes came, we tried amplifying the genes using the forward and the reverse primers from the OE PCR reactions. We used thermocycler settings that had worked for past reactions but still saw multiple bands for some of the genes, suggesting that the primers had non-specific annealing at those annealing temperatures. We started making touch-down thermocycler settings for each gene to determine what annealing temperature would yield the least amount of non-specific bands.
We made new yeast inoculations and attempted YMC for the first time as a test for the protocol. Sadly, we forgot to include the plasmid backbone in the reaction so it failed. We performed YMC again, being sure to include all components, and allowed the transformed yeast to grow on selective (leucine-deficient) plates for several days.
After outgrowth in liquid selective media, our cultures repeatedly failed to show colonies when plated on selective media. In order to address this, we redid our protocol for making chemicompetent Y. lipolytica cells after performing a final plating with the last of our transformant cell stock. We also performed transformations to reamplify our D17 plasmid in anticipation of beginning Gibson Assembly trials.
For our second yeast transformation attempt, we accidentally used less plasmid DNA; we obtained no growth. For our third attempt, we made sure to use the sufficient amount of plasmid DNA and to eliminate the outgrowth step, which was not necessary for leucine-marker selection. This yielded a successful transformation.
Week of September 3rd
Overview
Gibson Assembly
YMC
Biobrick & Terminator
Riboswitch
Other
As the start of the school year grew closer, we saw most of our teammates busy in lab trying to finish their lab group’s tasks before the chaos of the school year starts. For those not in lab, they were busy working on our iGEM wiki. Now that progress is being made on experiments, many teammates spent the week updating their online lab notebook to later add to the wiki.
Continuing last weeks trial, we carried out several colony PCRs throughout the week in order to isolate our desired plasmid. All of our attempts failed. In response to this we decided it would be best to regrow our colonies by making streak and index plates from the leftover colonies on our gibson pOPPY-UC19-yP and pOPPY-UC19-yXXU plates. With these newly grown colonies, we did a few more colony PCR reactions and still did not get any isolated bands.
At the end of the week, our sequencing results came. We observed that our gene fragments and homologous arms were not being inserted into the plasmid correctly. Each sample, although the right size, was incorrect and it seemed as if the plasmid was binding with itself. This led to more questions as to what went wrong. We first went back and double checked our design to make sure it was correct. It was. It wasn’t till next week when we figured out the issue.
Our colony PCR primers arrived, so we were able to check whether loxP was properly cut out using colony PCR. We made inoculations of E. coli carrying our pOPPY-XLR2-yX plasmid. We also made streak plates and liquid cultures using colonies from our previous round of YMC. We tested our “yeast miniprep protocol” using these liquid cultures, but received very weird (negative) DNA concentrations when we NanoDropped the products. We also intended to perform a miniprep on the E. coli carrying our pOPPY-XLR2-yX plasmid, but we had contamination in our negative control so we repeated the inoculations. These also ended up having contamination, so we asked our TA, McKenna Hicks, to give us a refresher on aseptic technique before repeating them again. We saw growth on our streak plates. We repeated the recircularization of pOPPY-XLR2-yX using site-directed mutagenesis, and plated transformed E. coli on LB/AMP to select for transformants.
After having a sufficient amount of our genes and having received our restriction enzymes (EcoR1-HF and Sca1), we were able to begin assembling our BioBricks together. We followed the protocols provided by iGEM for our restriction digest, ligation, and transformation. We began with the restriction digest protocol and adjusted for a lack of resources. Since we did not have the ideal buffer that iGEM recommended (NEB Buffer 2), we used the buffer we had in stock - CutSmart Buffer. One drawback of the buffer was its enzymatic activity of 50% for PstI, so to account for this, we doubled the amount of enzyme used. We then double digested the Psb1C3 backbone and both the 7-dehydrocholesterol reductase (Delta7) gene and the adrenodoxin-NADP+ reductase (ADR) gene, which we then ligated together to create a plasmid.
Over the weekend our final Y. lipolytica plating showed a large amount of growth, indicating that our old stock was indeed competent. Our E. coli amplification, however, showed no growth. It was found that this was because of an error in the media used. To save time, we thawed our previous glycerol stock of transformed cells and used it to inoculate a culture for outgrowth and plasmid isolation. DNA yields of this culture were lower than previous amplifications. We reasoned that this was due to the glycerol stock not having been plated on selective media before freezing. This would have resulted in some population of non-transformants existing in the stock and lowering plasmid yields. To remedy this, we plated our glycerol stock on selective media to isolate colonies and inoculated cultures with these colonies. We left these cultures to incubate over the weekend.
We began PCR experiments to amplify our riboswitch inserts in anticipation of our Gibson Assembly trials. Trials initially showed mixed results, which was perplexing since all PCR trials used the same primers and overhangs, but were confirmed as successful on 0.8% agarose gels by the end of the week. Nanodrop analysis gave the following concentrations of PCR products in the table below.
We also began out first Gibson Assembly experiments with our amplified inserts and linearized plasmid backbone. After this, we used PCR to confirm the presence of our pOPPY_GFP plasmids by using two of our Sanger sequencing primers and running the generated amplicons against a ladder on a 1.5% agarose gel. We also attempted a series of transformations of E. coli using our 5 Gibson Assembly products.
We also sent off our first samples for Sanger sequencing at the UC Berkeley DNA sequencing facility. With results coming in at the end of the week. Our reads indicated that our GFP gene was functional with no detected SNPs that would hinder our experiments.
We did our first yeast miniprep to test the effectiveness of our outlined miniprep procedures. The resulting Nanodrop curve was ragged and lacked the attributes of plasmid DNA. We tried again to miniprep yeast, this time washing the glass beads multiple times to get rid of residual isopropanol. The results were much better: a DNA curve. Still, there was an unknown source of salt contamination in the plasmid sample.
Week of September 10th
Overview
Gibson Assembly
YMC
Biobrick & Terminator
Riboswitch
Other
Another busy week of lab work! Several teammates spent their week with our new best friends: E. coli and Y. lipolytica. We only have a little over a month until the Giant Jamboree! It’s crazy to think about how far our team has come. Speaking of, we received an invitation to speak at TEDx Oakland! We’ve been busy preparing for the TEDx San Jose talk, and we’ve even more excited now knowing that we have two opportunities to discuss our project with the community.
After testing both primers and genes, we identified that the reason our gibson was not working properly was because lox-ura-lox was not what we expected. When we ran it on a gel, we saw two bands, one correct and one incorrect, meaning that our gene was contaminated. This was a horrible realization to make so we started making preparations for a backup plan. Our backup plan would use the D17 plasmid the riboswitch team had been using as a base. With D17, we would attempt to insert lox-ura-lox via KLD reaction and eventually insert our desired plasmid into Yarrowia.
Having observed positive results of a team member using electrocompetent E. coli cells we decided to use these cells to perform our transformations with. We transformed the electrocompetent E. coli cells the DNA of our ADR, and Delta7 plasmid. We transformed with electrocompetent E. coli three times during the week and each yielded unsuccessful results, made evident in the absence of growth on the plates.
To analyze our results, we first tested the legitimacy of using electrocompetent cells and performed a transformation using the chemicompetent E. coli cells and the ligated Delta7 and ADR plasmids; the results were still unsuccessful. The next step we took was to adjust the protocol again, adding the enzymes and DNA at once within one tube, rather than adding them later in a separate enzyme mastermix.
We made inoculations using the E. coli carrying pOPPY-XLR2-yX in LB/AMP and the yeast from the previous round of YMC in YNB-CSM-Leu. Colony PCR on E. coli showed bands indicating two colonies had successful transformants. We made streak plates using the two successful transformants, and then went on to make liquid cultures from these colonies. We miniprepped the pOPPY-XLR2-yX plasmid and then performed PCR, mistakenly using Q5 polymerase rather than OneTaq. Q5 polymerase could do the job just fine, it is just much more expensive than OneTaq for simply verifying the size of a fragment!
We isolated more stock of our D17 plasmid for further Gibson Assembly trials. These were miniprepped out to concentrations of ~450 and ~500ng/uL. In order to get growth from the rest of our aptameric inserts, we redid streak plates with higher depositions of transformant. We also redid a transformation of D17 into DH5α to test if our sample of cells was properly competent. These new transformations were unsuccessful, we eventually found this was because of an error in the creation of competent cell stocks.
While more chemicompetent cell cultures were being produced, we used electrocompetent stocks to preform more transformations on E. coli with our recircularized plasmids. We wrote up a new protocol for electroporation and ran tests to optimize our procedure.
We ran another 50uL linearization PCR to restock and checked this on a gel. Then, we performed 5 more transformations to get our recircularized stocks into E. coli.
We attempted a yeast plasmid miniprep yet again, this time using S. cerevisiae containing a backbone pXRL2 connected to five progesterone pathway genes. Despite the improvement in our previous attempt, this attempt yielded an unusable sample. We concluded that our procedures may be inadequate.
Week of September 17th
Overview
Gibson Assembly
YMC
Biobrick & Terminator
Riboswitch
Other
This week, our friend at Family Planning 2020 introduced us to his friend in the Population Council named John Townsend. John scheduled a Skype call with us on September 20th to discuss our project idea as well as potential avenues for development after the iGEM competition ends. He provided us several other contacts to reach out to for an opinion on our next steps and offered to pass our project summary onto his colleagues. John seems really eager to help us move forward with development, and we are so honored to have him as a resource.
Our other teammates spent another long week in lab and, in their off-time, worked on completing their assigned website pages. We are making great progress!
In preparation to start our backup plan, this week we linearized D17. We also attempted to linearize our contaminated lox-ura-lox in an attempt to clean it for future use. As a backup, we reordered lox-ura-lox from IDT in case our clean up failed.
Using the newly revised restriction digest protocol, we performed new rounds of restriction digests with the PSb1C3 backbone, the P450scc gene, and the 3BH gene. Then we ligated the results from the restriction digests and performed a transformation with chemicompetent and electrocompetent E. coli cells. None of the transformations yielded successful results.
The terminators we received (from Hal Alper) were synthetic, which allowed for it to carry a lower amount of base pairs. After discussing our procedure with our lab teaching assistant, we found a fitting method for both using bacterial stabs and for plating them afterwards.
On luria broth high copy plates, we plated Tsynth8, Tsynth30, and TEF1 in E. coli which were left to incubate over the span of nine hours. We also followed this method with our terminators when growing them in S. cerevisiae on a Histidine plate. We were presented with successful results the following day and observed growth of all three terminators on our E. coli plate and then inoculated them in liquid cultures. We then performed a miniprep procedure and then inoculated them in a falcon tube with luria broth and ampicillin to get to the proper ratio of 100 ug/mL in the 5 mL of luria broth.
We then performed another miniprep of the liquid E. coli cultures and continuously tailored the iGEM protocol to fit our experiments. We then analyzed each terminator’s absorbance via nanodrop which yielded us the results necessary to analyze whether or not each terminator would be fit to continue with based on their DNA concentrations; we gathered that TEF1 and Tsynth8 would be the most promising. We then transformed the plasmids from the miniprep procedure from E. coli into Y. lipolytica, with two hour incubation periods after vortexing.
We attempted many trials to confirm successful site directed mutagenesis. We failed many times do to bad polymerase, conducting colony PCR on yeast as well as bad thermal cycler settings.We also conducted a PCR with Q5 polymerase to relinearize the plasmid in preparation for yeast mediated cloning. The results of that were great! All of the bands were at the expected size of the full plasmid.
Over the course of this week we were able to get platings of E. coli transformed with all 5 of our pOPPY_GFP series plasmids. Growth of these transformants, however, was inconsistent, with multiple platings being required to get colonies for each sample. By the end of the week, only transformants carrying pOPPY_GFP(2), (4), and (5) were able to be isolated enough to grow monoclonal liquid cultures. Upon plasmid isolation and PCR testing with the Rib(f)2 and Rib(r)1 primers our gel images suggested the presence of D17 plasmid in our pOPPY_GFP(3) transformants. Considering these troubling results as well as the fact that liquid cultures were failing to grow, we opted to discard these samples and begin the next week with new transformations.
We conducted a 4th yeast miniprep attempt with an identical sample, but instead followed a protocol given to us by UCSC Professor Kamakaka. We obtained an improved Nanodrop curve that indicated the presence of DNA. However, there was still an unknown source of salt contamination.
Week of September 24th
Overview
Gibson Assembly
YMC
Biobrick & Terminator
Riboswitch
Other
This was a busy week for our iGEM team! Not only was this the first week of the school year, it was also the busiest week of our public engagement work thus far! On Monday, we juggled making preparations for the annual UCSC OPERs Festival, preparing for the TEDx talk in San Jose, and making slides for a school presentation to San Lorenzo Valley High School all happening on the same day. Thankfully, we worked together as a team and accomplished all our goals with flying colors.
This week we performed a clean and concentrate on D17. This was successful meaning we could begin to use D17 when lox-ura-lox was ready. In addition we also ran our linearized lox-ura-lox gene and the plasmid lox-ura-lox gene on a gel. It failed.
With an absence of successful results from the transformations, we decided to alter the restriction digest protocol again. One of the edits included using the 3.1 NEB buffer, rather than the CutSmart buffer. Using the NEB buffer, PstI had 100% enzymatic activity, whereas EcoRI had 50%, so we used twice as much of EcoRI. Using the newly made protocol, we performed a double restriction digest and ligation to get the plasmids for Cholesterol side-chain cleavage enzyme (P450scc) and 3-beta-hydroxy-Delta(5)-steroid dehydrogenase (3BH). We transformed the ligations with electrocompetent and chemicompetent E. coli cells, and we then got four colonies on the electrocompetent E. coli plate for the plasmid of P450scc. The other plates, which were our transformed chemicompetent E. coli, and our electrocompetent E. coli with 3BH ligation failed. Since we had four colonies that contained our P450scc plasmid, we index plated them, inoculated them into liquid media, and miniprepped out the plasmid. To confirm if the E. coli actually had our P450scc Biobrick plasmid, we performed a colony PCR, and a restriction digest. Both the colony PCR and restriction digest had inconclusive results, and we were not able to tell if the plasmid was actually our P450scc BioBrick.
None of the transformations were working, so we tried to alter our restriction digest protocol again. This time instead of using the buffer CutSmart, we used the 3.1 NEB buffer.  In this buffer Pst1 has 100% activity, but EcoR1 has 50%, so we used twice as much EcoR1 this time instead of twice as much Pst1. With this new restriction digest protocol we performed a double restriction digest and ligation to get us the plasmids for P450scc and 3BH.  We transformed these ligations with electrocompetent and chemicompetent E. coli cells, and this time we got four colonies on the electrocompetent E.coli plate for the BioBrick plasmid of P450scc! The other plates which were our transformed chemicompetent E. coli, and our electrocompetent E. coli with 3BH ligation failed. Since we had four colonies that contained our P450scc plasmid we index plated them, inoculated them into liquid media and miniprepped out the plasmid. To confirm if the E. coli actually had our P450scc Biobrick, we performed a colony PCR, and a restriction digest. Both the colony PCR and restriction digest had inconclusive results, and we were not able to tell if the plasmid was actually our P450scc BioBrick.
Using newly procured T4 DNA ligase and buffer to prevent any implications, since the buffer has a limited, low amount of freeze-thaw cycles. Then we chose colonies from the plates which were isolated and ran a yeast transformation for TEF1, but found that overgrowth had occurred and the results were no longer fit to continue with.
To ensure fluorescence, we recognized that since the S. cerevisiae strain contains the mStrawberry and yCitrene fluorescent proteins, we figured out a way to grow it in a liquid culture to log phase. We would then centrifuge the cells and place the pellet in new YPD with galactose to induce the GAL1 promoter. Then we would let it grow for two to four hours and follow up by testing the fluorescence levels of mStrawberry to yCitrene for the Tsynth8, Tsynth30, and TEF1 in S. cerevisiae.
We then restreaked colonies from the overgrown Tsynth8 and Tsynth30 plates from isolated colonies. Then we transformed the TEF1 plates again into Y. lipolytica because the overgrown plates contained zero isolated colonies without signs of contamination. We used 3 uL of the plasmid from the miniprep procedure and transformed Y. lipolytica and 100 uL of it and plated on leucine deficient plates; the plates were left in incubation for two days.
We clean and concentrated the PCR products and quantified them with a nanodrop with good results. We then prepared the samples for Sanger sequencing and sent them out to Sequetech. In the meantime of waiting for results, we made more liquid stocks of our samples. We received the results and found out that we had successfully removed loxP in sequence alignment. We conducted a trial of yeast mediated cloning as well and incubated them over the weekend.
New platings for all 5 transformant population showed successful growth, although colonies were still too clustered to be isolated. We also discovered that our final homologous arm primers had not been ordered, and so attempted to isolate our gene inserts using some restriction sites that were near the binding site of our designed primers. These transformations were unsuccessful, though we were able to produce a glycerol stock of E. coli containing our pOPPY_GFP(1) plasmid.
Our second attempt to transform E.coli with a yeast-derived plasmid (pXRL2 + 5 pathway genes) involved the use of electroporation. Results are visible in the more detailed section.
Week of October 1st
Overview
Gibson Assembly
YMC
Biobrick & Terminator
Riboswitch
Other
This week, our team started preparing for the Giant Jamboree by creating our presentation slides. With this being the first full week of school, attendance in lab has been spotty. We are all still trying to get back in the routine of school work while also juggling iGEM tasks!
After receiving the lox-ura-lox from IDT, we began our KLD reactions. We ran our KLD reaction with our new clean D17 in the PCR machine. Using the product, we plated the mix on a high copy plate and let it incubate overnight. The next day, we confirmed its growth and began preparation for colony PCR.
We also attempted to transform our pXRl2 gibson with our desired genes and perform PCR on p450. Both failed.
Since we were successful in making a colony that contained the Psb1C3 backbone with the electrocompetent E. coli,  we did another double digest, ligation, and transformation with the genes 3BH and Delta7. We also did another restriction digest to confirm if the P450scc plasmid that was miniprepped was correct, and we did not get the bands we expected. We had bands at 2.8kb and 1.2kb when we expected bands at 3.2kb and 800bp. Since we were not getting our BioBricks, we consulted one of our PI’s, as to why our transformations were not working. He assumed there was too much salt when we electroporated our cells, which would kill them and explained why we may not get growth on our plates. We did the math and he was right;.There was way too much salt in the DNA that we were transforming our electrocompetent E. coli with and would explain why we rarely ever got growth on transformations performed with electrocompetent E. coli cells. From then on we only transformed with chemicompetent E. coli cells. We tried to perform a transformation with chemicompetent E.coli cells with a ligated plasmid of 3BH. We did these transformations with different concentrations of DNA to see if the amount of DNA that we transformed with was the problem. There was no growth on the plates, so it may not be that the amount of DNA that was the problem.
We conducted another trial of yeast mediated cloning with many more colonies due to lack of growth in some of the samples and low DNA concentrations. Then, we tried a colony PCR on the samples but failed. We finally realized that colony PCR did not work on yeast cells due to the presence of exonucleases that degrade the primers.
Considering our poor results using Gibson Assembly, we reasoned that our experiments could be improved now that our new set of primers had come in, including one that would allow us to linearize D17 without the 1bp mismatch we had discovered earlier. Repeating the Gibson Assembly trial and amplifying with PCR suggested the presence of both pOPPY_GFP as well as D17 in each GA sample, despite digest with a 5,000-fold excess of DpnI.
No growth observed on transformation plates. By this time, other team members were doing yeast minipreps. Subsequent transformations would use the resulting miniprep samples.
Week of October 8th
Overview
Gibson Assembly
YMC
Biobrick & Terminator
Riboswitch
Other
We sent out our improved terminator BioBrick for sequencing on October 10th. This is so exciting!  We’ve all been working so hard that the little victories mean the most. Both our yeast-mediated cloning team and our Gibson Assembly team accomplished big tasks this week and set themselves up for an even busier next week! The competition is less than a month away and lab work is high priority now.
We did several rounds of colony PCRs, most of which failed. On the last round, colony #4 and colony #17 of 30 both showed up on the gel at the right height. We then created liquid cultures of our successful colonies and grew them up over night. The cultures didn’t grow so we performed a few experiments to try to get it to grow. We created several tests that included longer incubation time, leaving the caps slightly open, and tilting it while it shakes. Nothing worked.
Since the transformation protocols were not working, we tried a new technique to create our plasmid. We decided to do a KLD (Kinase, Ligase and DpnI enzymes) reaction and hope that our plasmid was made. We performed a KLD transformation because other members of our team were successful in making plasmids this way. When we performed a transformation with this technique with our gene P450scc, there were 15 colonies. We proceeded to inoculate five of the biggest E. coli colonies into liquid media in preparation for a miniprep. We performed a double digest of the miniprepped plasmid and ran it on a gel along with the non-digested miniprepped plasmid to confirm if the plasmids were our BioBrick. For some reason there were no bands in the restriction digest lanes, but in the undigested plasmid lanes there were bands at around 4kb which means that there is some insert in our plasmid. We will perform more double digest to confirm if the P450scc plasmids are correct.
In comparing Tsynth8 and Tsynth30 for use in yeast, both demonstrated increased heterologous protein expression and transcript levels greater than 2-fold over the commonly used CYC1 terminator [cite study]. However, specifically for use in Y. lipolytica, Tsynth8 showed 3-fold greater protein expression than Tsynth30. In addition, Tsynth8 had slightly lower cryptic promoter activity and a lower tendency for transcript read-through, which would signal  insufficient transcript termination. Most significantly, Tsynth8 could be ordered as a sequence from IDT; whereas Tsynth30, with it’s extended TA efficiency region, could not.
We miniprepped the yeast mediated cloning products and received high DNA concentrations in all of our samples. Afterwards, we attempted multiple PCRs with different primers to check if our genes assembled correctly to the background. We received a lot of blank gels.
We then decided to run the miniprep products on a gel to determine if they are larger than the expected size of pOPPY-XLR2-yX. In our first attempt, we diluted the miniprep products to about 1ng/ul and saw no bands because there was not enough DNA to be visible on the gel. We then repeated with miniprep products diluted to about 50ng/ul, and saw bands for several samples above the 10kb band on the ladder. We submitted 3 of these for Sanger sequencing through Sequetech, using primers for various points on the gene cassette.
All 5 transformant populations with our new Gibson Assembly product showed growth after overnight incubation. We selected 10 isolated colonies from each plate as well as 2 from a D17 control plate to create an index plate as well as perform colony PCR. After imaging, we selected colonies 2, 16, 27, 32, 43, and 51 as likely carriers of pOPPY_GFP(1)--pOPPY_GFP(5) and D17, respectively. All continued experiments up to this point have been focused on growing cultures of these colonies, isolating their plasmids, sequencing, and transforming into our Yali cell line.
Created some more leucine deficient media for our team.
Week of October 15th
Overview
This is the make or break week for our iGEM team! The wiki freeze was on Wednesday on our team cut it close to the deadlines. Thanks to the tireless work of our teammates, we finished on time!
Our team decided to start work on a milk growth curve since our target organism will survive on dairy waste.
As of October 16th at 8:00pm, OUR TEAM DID IT!! We made progesterone!!!
Gibson Assembly and Homologous Recombination
OEPCR and Gene Amplification
Yeast Mediated Cloning and Cre-Lox
Biobrick and Terminator
Riboswitch
Other
