Team:UCSC/Notebook-3
Home
Team
Meet the Team
Collaborations
Partnership
Attributions
Sponsors
Project
Overview
Design
Applied Design
Experiments
Results
Demonstrate
Notebook
Human Practices
Outreach
Public Engagement
Safety
Notebook
Weekly Overview
Protocols
Parts
Overview
Improvement
Judging Contact
Notebook
Week by Week Overview
In Detail
Week of
June 26th
July 2nd
July 9th
July 16th
July 23rd
July 30th
August 6th
August 13th
August 20th
August 27th
September 3rd
September 10th
September 17th
September 24th
October 1st
October 8th
October 15th
Week of June 26th
Overview
Gibson Assembly
Modeling
Gene Amplification
YMC
Biobrick & Terminator
Riboswitch
Other
Day 1 was mutiny. Five students had met with a sponsor in the morning and when the two captains returned to the room, the students were milling around, whispering under their breath, and fidgeting. The problem was with motivation towards the project, as some students felt that it wasn’t novel enough. We had found a 1998 paper by Duport et al. that explained how their team developed yeast to produce pregnenolone and progesterone - the things we wanted to make. It turns out though that the paper didn’t lead to anything, the strain has been lost, and it was a good proof of concept for our idea. From the uprising we learned to communicate thoroughly.
Week of July 2nd
Overview
Gibson Assembly
Modeling
Gene Amplification
YMC
Biobrick & Terminator
Riboswitch
Other
Our yeast strain is going to have all the genes to make progesterone, as well as the lactase enzyme so that we can feed it lactose and it break it down to glucose and galactose. We figured out after much deliberation how we could have our friends in developing nations get the accurate dose of progesterone. We learned from our mentor that we could use a technique to measure the weight of an object using a coat hanger, and depending on weight, calculate the amount of lactose inside that container. This would be the dairy that the people have on hand. Depending on weight, we will have a chart that says how much lactose is in that container. The people will be guided to portion a certain amount of the lactose into an ice-cube-like container and grow the yeast in there. Because there will be a specific amount of lactose in each section, and our promoters are sensitive to galactose, our yeast will stop producing progesterone after it consumes all of the lactose.
We have found multiple ways to insert our genes into Y.lipolytica and we made pros and cons lists for each method to decide which one would be the most effective and time-efficient. The methods are regular homologous recombination using homologous arms on our plasmid, EasyCloneYALI: CRISPR/Cas9‐based technique for engineering Y.lipolytica specifically, and Drag and Drop cloning.
Week of July 9th
Overview
Gibson Assembly
Modeling
Gene Amplification
YMC
Biobrick & Terminator
Riboswitch
Other
We confirmed that using a constitutive promoter would be better than an inducible one, at least initially, because it will use up energy to keep producing progesterone after the growth media has been consumed. Our website has gone through several prototype designs, but it seems like we may be getting close to a final design we would like to use to represent our work. One fun feature that is a work in progress is an interactive map that will, when moused over, provide information about different countries and our outreach there. We have also been working on finalizing our logo, which will include the name of our project: PoPPY. PoPPY stands for Portable Progesterone Production in Yeast, an accurate description of the goal of our project. A low this week for the team was realizing that most of the survey responses received from one of the groups we reached out to about contraceptive accessibility were unreliable, since many answers between supposedly different women were the exact same, word for word. However, we hope that this was a problem specific to the group, and not the norm. We have some more legitimate groups who are willing to send us photos of the women filling out the surveys, and we hope to see good results. A high for the team this week was finally understanding how Cre-Lox recombination, a method that we will use in one of our experiments to integrate our gene cassette into the yeast genome, works. We now expect to run three parallel experiments to get to our end goal: a control using Gibson and homologous arms to create our gene cassette and integrate it into our Y. lipolytica genome; assembly of the plasmid inside S. cerevisiae using yeast-mediated cloning,  isolation and insertion of the plasmid into E. coli for amplification, and insertion of the gene cassette into Y. lipolytica using lox sites; and assembly of gene cassette/plasmid inside Y. lipolytica and integration into genome using lox sites. For each of the three experiments, we will begin by inserting lox sites into the Y. lipolytica genome using homologous recombination.
Week of July 16th
Overview
Gibson Assembly
Modeling
Gene Amplification
YMC
Biobrick & Terminator
Riboswitch
Other
About half the team spent the week drafting our thesis. While some of us were writing, others were meeting with professors on campus to discuss our ideas and to pick up some S. cerevisiae strains to use as our experimental organism. We learned from Rohinton Kamakaka that we could insert Cre recombinase into our yeast using a plasmid, so now we are in pursuit of a method to both express and remove Cre recombinase once we no longer need it. Our yeast parents have been growing our Y. lipolytica and amplifying the plasmids that we have received. Our wiki boys have almost finished formatting the website to their (or our) liking and are now adding more content about our project itself. A few of our team members scheduled an interview with a representative from Family Planning 2020, but due to timing issues, had to reschedule.
Week of July 23rd
Overview
Gibson Assembly
Modeling
Gene Amplification
YMC
Biobrick & Terminator
Riboswitch
Other
This week, our team wrote a thesis! Or, more accurately, finished writing it. The majority of us put our brains together to create a draft and we chose one excellent editor to take control of the paper. She enlisted grammar police, style monitors, and LaTeX formatters to help her revise our drafted ideas into one cohesive overview of our work to-date. Each of us also wrote about our own contributions to the project and personal reflections describing our feelings towards the journey thus far.
Also, our two captains and our PI met Dean Wolf, Dean of Baskin School of Engineering at UC Santa Cruz, Roger Trippel, Senior Director of Development and Individual Giving, and Abigail Kaun, Special Assistant to the Dean. We discussed the project to-date, our outreach efforts, and shared ideas for managing both our and future iGEM UCSC team’s funds.
Created first batch of YPD medium and also created 80% glycerol freeze stocks of our Y. lipolytica strain. We attempted to miniprep our pXRL2- and pUC19-containing E. coli but we made a mistake by switching the order of wash buffers used from the Zyppy Miniprep Kit. The Nanodrop curve of the plasmid sample was ragged and thus we will need to attempt this again.
Week of July 30th
Overview
Gibson Assembly
Modeling
Gene Amplification
YMC
Biobrick & Terminator
Riboswitch
Other
This week, the captains held check-ins with each individual on the team to hear concerns and alleviate worries, since we are now halfway through the summer. While the captains were busy with meetings, the rest of the team was working hard as always. Our modeling team started a growth curve for Yarrowia lipolytica so we can better understand the yeast’s growth rate, our riboswitch team began lab work, our plasmid team started linearizing plasmids, and our outreach team continued to update the wiki page and started creating a presentation for high schoolers.
Four of our team members and our PI took a vacation to Minneapolis that wasn’t much of a vacation at all. The small group toured Medtronic's Operational Headquarters, Physiological Research Laboratories, and the University of Minnesota's Visible Heart Lab. Almost every other second of the trip was spent on the BMES Coulter College Conference, which required teams to come up with a medical solution to a need. Our team focused on hypertension in resource-constrained settings and developed a solution in the form of a blood-pressure-monitoring phone case and app. Even after the conference, the fivesome was hard at work; they met with the University of Minnesota iGEM team, toured their lab, and gave advice about human practices, outreach, and fundraising.
The first step to prepare for Gibson Assembly was to linearize pUC19 and pXRL2 via PCR. We performed PCR Over the course of this week we had bands in strange and random locations. We were led to believe our plasmids were not what we believed them to be. Eventually it was discovered that we were using an incorrect amount of Q5 2x master mix which caused the PCR to not work properly.
During the first lab week, we rehydrated our gene blocks from IDT and performed overlap extension (OE) PCR on three of the genes (delta7, ADR, and FDX1). We used touch-down and touch-up thermocycler settings which resulted in many different sized bands on an agarose gel. To reduce the amount of nonspecific primer annealing in these reactions, we decided to only use touch-down thermocycler settings and to put the primers on the side walls of the PCR tubes to limit the Q5 polymerase exposure to the primers. The exonuclease activity of the Q5 polymerase chews back the ends of our primers and causes non specific annealing, so we made sure the primers were added last before putting the reactions into the thermocycler. We also decided to run OE PCR reactions without primers to allow the overlapping regions of the genes to anneal and form the full length genes which could be used as templates for regular PCR reactions.
Since our project would eventually require separate experiments with 5 different inserts, our first goal in lab was the amplification of our experimental base plasmid. We first created a stock of ampicillin selective plates with a concentration of 100ug/mL. We carried out a transformation with NEB DH5α chemicompetent cells and created a glycerol stock of leftover cells after plating. We plated out transformants on 3 plates at original strength, 10-fold, and 100-fold dilutions using glass beads. Our platings were successful, and we used the 100-fold dilution plate to isolate single colonies and create a monoclonal culture of transformants. We also learned how to plate with a sterile wire loop in order to keep the number of plates used to a minimum. Every protocol--pouring plates, transformations, platings, and creating glycerol stocks--was new to us and required some reading of protocols from NEB and other sources.
Created 80% glycerol freeze stocks of four S. cerevisiae strains obtains from the Heinrich lab at UCSC. Did a second attempt of miniprepping the pXRL2- and pUC19-containing E. coli, but unfortunately had salt contamination in our resulting sample. We attempted this a third time by tossing out flow-throughs between each step of the miniprep procedure and obtained workable plasmid samples for pXRL2 and pUC19.
Primers came in and were rehydrated.
We performed our first attempt to PCR amplify out the ade2 homologous arms from Y. lipolytica using colony PCR. No amplification was observed. For our second amplification attempt, we used Touchdown PCR and used increasing concentrations of yeast cells in the PCR reaction mixes. Still no results.
Week of August 6th
Overview
Gibson Assembly
Modeling
Gene Amplification
YMC
Biobrick & Terminator
Riboswitch
Other
So many people are in the lab now! Because so many of our small lab teams have transitioned to lab work, the “Yeast Parents”, or team in charge of maintaining the yeast and the lab, gave everyone a tour of the space to keep everything organized (and keep us all in line).
Unfortunately, with great numbers of people in lab comes great margin for mistake. We burnt through a ridiculous amount of Q5 polymerase this week because we miscommunicated the concentrations needed to perform PCR. However, the team took this as a learning experience and practiced concentration math and master-mix-making. We are also going to begin a crowdfunding campaign to supplement the team funds when/if errors arise in the future.
Outreach is really cool! Three of us, plus our PI (of course), spoke to Martyn Smith, Managing Director of the Family Planning 2020 Secretariat, over the phone. He nearly gave us heart attacks when he offered to help us reach local agents familiar with the regulatory environments in the countries we wanted our project to extend to. Aside from the FP2020 call, two of our members have been diligently working on outreach at home. They’ve started to develop a game as an interactive way for high schoolers to join in when we present our project to them later this month.
After double checking that we were using the right volumes for our PCR reaction, we were able to linearize pUC19 and pXRL2. We then confirmed this by running it on a 1% agarose gel seen below.
We ran OE PCR reactions without primers for all five genes but still saw multiple bands for some of the genes. We discovered that the master-mix concentration was too high for all of the reactions we had performed, so we changed the amount of master-mix in each reaction to ensure the concentration was 1X rather than 2X. After changing the master-mix concentration, we ran more OE PCR reactions without primers and selected certain reactions with the least amount of nonspecific bands for templates in our regular PCR reactions.
Our over-the-weekend cultures had little growth after 24 hours, but showed adequate growth come Monday morning. After discussing this with other teammates, we found it was most likely due to improper aeration of our samples, which were stored upright with a tight cap in the incubator. We then performed a plasmid isolation followed by a restriction digest to test the identity of our isolated plasmid. Our first digest was unsuccessful, which we hypothesized was because of the age of our enzyme sample. Our next digest, performed with our most recently acquired enzyme, showed the expected band pattern after altering the contrast on the gel image. We also attempted a PCR linearization of our amplified plasmid which was unsuccessful. Through these experiments we learned how to better analyze gel images by increasing exposure, brightness, and contrast, as well as proper incubation practices for liquid cultures. We also ran into several issues with budgeting time correctly in lab, which prompted us to change how we schedule our lab procedures to create as little “dead space” as possible.
For our third attempt to amplify out ade2 homologous arms from Y. lipolytica, we instead did a serial dilution to ensure that we were not using too many cells in the PCR reaction mixes. Again, no amplification was observed.
Week of August 13th
Overview
Gibson Assembly
Modeling
Gene Amplification
YMC
Biobrick & Terminator
Riboswitch
Other
Each year, the UCSC iGEM team presents to the Bioethics class during the second Summer Session. A few team members diligently prepared a presentation about our questionnaires and project for that purpose. Several of us attended the presentation, and afterwards broke into small groups with students in the class to ask and answer questions about our ethics. Because of this presentation, we learned about the IRB at UCSC and how we needed to get our questionnaires approved, since we were technically working with “human subjects” if we sent it out.
Another small group designed a poster to be displayed at the Annual Physical and Biological Sciences Summer Research Symposium. This poster included details about our three parallel experiments, our research, and our outreach. Three team members presented the information to interested audiences at the Symposium.
After successfully linearizing pUC19 and pXRL2, we repeated the same experiment with larger volumes to get more product for future use. Using this product, we performed several DPN1s and clean and concentrate experiments. The first four tests were failures. We used both our advisor’s reagents and Zymo’s reagents. Our solution that eventually worked was to prewarm the 30 uL water we were using as the elution buffer and let it sit for several minutes, in the final step. On the fifth experiment, the clean and concentrate worked but still not very well. We tried one more time using only 10 uL water to elute and got sufficient concentrations for our Gibson reactions.
We tried many PCR reactions with the new templates but still saw multiple bands for each gene, so we decided to extract the correct bands from the gels and use those as our templates. We tried using larger reaction volumes in the gels to get more DNA from the gel extraction, but we didn’t get clear enough bands to properly extract the correct sized genes. We also tried amplifying the loxP-URA3-lox71 gene cassette with our existing primers, but the reaction didn’t work due to the difference in annealing temperatures between the forward and the reverse primer.
This was the week in which we discovered our collective error in the PCR protocol we had used up until this point. This was a lesson to us and the rest of the team about blindly following protocols without questioning them. Upon redoing our PCR experiment with the corrected protocol in a 10uL reaction, our gel showed bands indicating a successful linearization. We also went through the protocol for the creation of chemicompetent yeast cells in anticipation for our transformation into Y. lipolytica. Protocol was successful and yielded 2.5 mL of glycerol stock of our chemicompetent Y. lipolytica.
The glycerol stocks of E. coli were successfully streaked to verify viability. Contacted lab support for pH meter electrode issues.
Week of August 20th
Overview
Gibson Assembly
Modeling
Gene Amplification
YMC
Biobrick & Terminator
Riboswitch
Other
This week, we began to think about our thesis again. Whew. A few of us began reorganizing the information we needed to write about to make the paper more clear. We also had a meeting at the end of the week about expectations for writing during the coming week.
After finding out about the IRB, we applied for exemption. We then found out that we would have to revise our questionnaires. Appropriate changes were made and the application was submitted again (this happens several more times in the next week or so).
Some of us considered different design and selectable marker options to test integration of genes into the yeast genome. We learned that one of our steps lacks a reliable selectable marker test, but that replica plating may provide sufficient levels of certainty.
Our several lab groups worked tirelessly to make more media, to check for deletions in plasmids, and to design primers that actually work.
Now that we have lox-URA3-lox, linearized pUC19, and amplified homologous arms, we are able to begin Gibson Assembly. We did two gibsons this week. The first Gibson includes the parts mentioned above and the second is our linearized pXRL2 along with several gene fragments: ADR, FDX, Delta7H, p450SCC, and 3BH. After combining the neccessary components with the Gibson Master Mix, we put it in the PCR machine for an hour and then stored it on ice. The following day we inoculated the gibson products on high copy LB-AMP plates. The pUC19 plate grew normally but the pXRL2 plate lawned.
We then picked colonies from the pUC19 plate and transformed it in liquid cultures. We then performed colony PCR to check if our Gibson assembly was successful. It likely did not work. The gel showed very bright bands in the wrong areas but a faint band where our desired product would be. With this information, we made streak plates of the potential successful colonies in hopes to isolate what we believe to be our desired gibson product and threw the rest of the cultures out.
Once we created a gel with clear, distinct bands for DNA extraction, we tried using the Bioline Isolate II PCR and Gel kit to retrieve the DNA from our agarose gel. At first we used a different binding buffer than the one provided in the kit, but the final extraction had a high salt concentration that made the DNA unusable as a PCR template. We tried using the Zymogen clean and concentrator kit to further purify the DNA, but there wasn’t enough DNA to use for PCR reactions after the clean-up. We performed two more gel extractions using the reagents from the kit but still didn’t get clean enough DNA for further PCR amplification. We finally decided to order the full genes and use the primers from the OE PCRs to amplify the full length genes.
After a successful test reaction at 10uL, we repeated our linearization PCR with a volume of 50uL. Our confirmation gel indicated a successful linearization, although some of the sample streaked above the 10 kb ladder.
We then attempted a transformation of our new chemicompetent Y. lipolytica using our control D17 plasmid. Our transformant cells were then incubated in liquid selective media for outgrowth. We also attempted a transformation of our linearized PCR product from the 50 uL reaction, but later realized that the linearized plasmid was unfit for expression of any genes at this point and so the transformation plate was discarded.
Created YNB+CSM-Leu medium (leucine-deficient) and YNBcasa (uracil-deficient) medium. Attempted to transform Y. lipolytica with pXRL2, ultimately to test if we can miniprep yeast with our protocol. Unfortunately there was no obvious yeast growth on the transformation plates.
Week of August 27th
Overview
Gibson Assembly
Modeling
Gene Amplification
YMC
Biobrick & Terminator
Riboswitch
Other
Thesis week is upon us! This time around, we struggled a lot less to work as a group and write as a group. The early organization probably helped. We turned in a much more cohesive paper.
We received an invitation to speak at TEDx! This is exciting. We get to tell people about our project, but we also get to bring more attention to the unmet need for modern contraceptives! Our team is really happy about the opportunity to present on such a large platform.
In lab, we had some successful yeast and bacteria transformations, some successful Gibsons, and confirmed Y. lipolytica’s antibiotic resistance.
We also finally received IRB exemption for our questionnaires! This means we can now officially use our questionnaires to gather information about the women who may not have adequate access to modern contraceptives.
Once the full length genes came, we tried amplifying the genes using the forward and the reverse primers from the OE PCR reactions. We used thermocycler settings that had worked for past reactions but still saw multiple bands for some of the genes, suggesting that the primers had non-specific annealing at those annealing temperatures. We started making touch-down thermocycler settings for each gene to determine what annealing temperature would yield the least amount of non-specific bands.
After outgrowth in liquid selective media, our cultures repeatedly failed to show colonies when plated on selective media. In order to address this, we redid our protocol for making chemicompetent Y. lipolytica cells after performing a final plating with the last of our transformant cell stock. We also performed transformations to reamplify our D17 plasmid in anticipation of beginning Gibson Assembly trials.
For our second yeast transformation attempt, we accidentally used less plasmid DNA; we obtained no growth. For our third attempt, we made sure to use the sufficient amount of plasmid DNA and to eliminate the outgrowth step, which was not necessary for leucine-marker selection. This yielded a successful transformation.
Week of September 3rd
Overview
Gibson Assembly
Modeling
Gene Amplification
YMC
Biobrick & Terminator
Riboswitch
Other
Over the weekend our final Y. lipolytica plating showed a large amount of growth, indicating that our old stock was indeed competent. Our E. coli amplification, however, showed no growth. It was found that this was because of an error in the media used. To save time, we thawed our previous glycerol stock of transformed cells and used it to inoculate a culture for outgrowth and plasmid isolation. DNA yields of this culture were lower than previous amplifications. We reasoned that this was due to the glycerol stock not having been plated on selective media before freezing. This would have resulted in some population of non-transformants existing in the stock and lowering plasmid yields. To remedy this, we plated our glycerol stock on selective media to isolate colonies and inoculated cultures with these colonies. We left these cultures to incubate over the weekend.
We began PCR experiments to amplify our riboswitch inserts in anticipation of our Gibson Assembly trials. Trials initially showed mixed results, which was perplexing since all PCR trials used the same primers and overhangs, but were confirmed as successful on 0.8% agarose gels by the end of the week. Nanodrop analysis gave the following concentrations of PCR products in the table below.
We also began out first Gibson Assembly experiments with our amplified inserts and linearized plasmid backbone. After this, we used PCR to confirm the presence of our pOPPY_GFP plasmids by using two of our Sanger sequencing primers and running the generated amplicons against a ladder on a 1.5% agarose gel. We also attempted a series of transformations of E. coli using our 5 Gibson Assembly products.
We also sent off our first samples for Sanger sequencing at the UC Berkeley DNA sequencing facility. With results coming in at the end of the week. Our reads indicated that our GFP gene was functional with no detected SNPs that would hinder our experiments.
We did our first yeast miniprep to test the effectiveness of our outlined miniprep procedures. The resulting Nanodrop curve was ragged and lacked the attributes of plasmid DNA. We tried again to miniprep yeast, this time washing the glass beads multiple times to get rid of residual isopropanol. The results were much better: a DNA curve. Still, there was an unknown source of salt contamination in the plasmid sample.
Week of September 10th
Overview
Gibson Assembly
Modeling
Gene Amplification
YMC
Biobrick & Terminator
Riboswitch
Other
We attempted a yeast plasmid miniprep yet again, this time using S. cerevisiae containing a backbone pXRL2 connected to five progesterone pathway genes. Despite the improvement in our previous attempt, this attempt yielded an unusable sample. We concluded that our procedures may be inadequate.
Week of September 17th
Overview
Gibson Assembly
Modeling
Gene Amplification
YMC
Biobrick & Terminator
Riboswitch
Other
We conducted a 4th yeast miniprep attempt with an identical sample, but instead followed a protocol given to us by UCSC Professor Kamakaka. We obtained an improved Nanodrop curve that indicated the presence of DNA. However, there was still an unknown source of salt contamination.
Week of September 24th
Overview
Gibson Assembly
Modeling
Gene Amplification
YMC
Biobrick & Terminator
Riboswitch
Other
Our second attempt to transform E.coli with a yeast-derived plasmid (pXRL2 + 5 pathway genes) involved the use of electroporation. Results are visible in the more detailed section.
Week of October 1st
Overview
Gibson Assembly
Modeling
Gene Amplification
YMC
Biobrick & Terminator
Riboswitch
Other
Week of October 8th
Overview
Gibson Assembly
Modeling
Gene Amplification
YMC
Biobrick & Terminator
Riboswitch
Other
Week of October 15th
Overview
Gibson Assembly
Modeling
Gene Amplification
YMC
Biobrick & Terminator
Riboswitch
Other
Group
GA and HR
Modeling
OE-PCR
YMC and Cre-Lox
BioBrick
Riboswitch
Other
Gibson Assembly and Homologous Recombination
Modeling
OEPCR and Gene Amplification
Yeast Mediated Cloning and Cre-Lox
Biobrick and Terminator
Riboswitch
Other
