<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCSC_Experiments-2 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCSC/Experiments-2</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="nav-links-container"><A href="https://2018.igem.org/Team:UCSC">Home</A><DIV class="dropdown"><A href="javascript:void(0);" class="dropbtn dropdown-icon" onclick="toggleDropdownTeam()">Team
        </A><UL class="dropdown-content" id="ddTeam"><LI><A href="https://2018.igem.org/Team:UCSC/Team">Meet the Team</A></LI><LI><A href="https://2018.igem.org/Team:UCSC/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:UCSC/Partnership">Partnership</A></LI><LI><A href="https://2018.igem.org/Team:UCSC/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:UCSC/Sponsors">Sponsors</A></LI></UL></DIV><DIV class="dropdown"><A href="javascript:void(0);" class="dropbtn dropdown-icon" onclick="toggleDropdownProject()">Project
       </A><UL class="dropdown-content" id="ddProject"><LI><A href="https://2018.igem.org/Team:UCSC/Description">Overview</A></LI><LI><A href="https://2018.igem.org/Team:UCSC/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:UCSC/Applied_Design">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:UCSC/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:UCSC/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:UCSC/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:UCSC/Notebook">Notebook</A></LI></UL></DIV><DIV class="dropdown"><A href="javascript:void(0);" class="dropbtn dropdown-icon" onclick="toggleDropdownHumanPractices()">Human Practices
       </A><UL class="dropdown-content" id="ddHumanPractices"><LI><A href="https://2018.igem.org/Team:UCSC/Human_Practices">Outreach</A></LI><LI><A href="https://2018.igem.org/Team:UCSC/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:UCSC/Safety">Safety</A></LI></UL></DIV><DIV class="dropdown"><A href="javascript:void(0);" class="dropbtn dropdown-icon" onclick="toggleDropdownNotebook()">Notebook
       </A><UL class="dropdown-content" id="ddNotebook"><LI><A href="https://2018.igem.org/Team:UCSC/Notebook">Weekly Overview</A></LI><LI><A href="https://2018.igem.org/Team:UCSC/Experiments#Expe_pro">Protocols</A></LI></UL></DIV><DIV class="dropdown"><A href="javascript:void(0);" class="dropbtn dropdown-icon" onclick="toggleDropdownParts()">Parts
       </A><UL class="dropdown-content" id="ddParts"><LI><A href="https://2018.igem.org/Team:UCSC/Parts">Overview</A></LI><LI><A href="https://2018.igem.org/Team:UCSC/Improve">Improvement</A></LI></UL></DIV><A href="https://2018.igem.org/Team:UCSC/Judging">Judging</A><A href="https://2018.igem.org/Team:UCSC/Contact">Contact</A></DIV><DIV class="content toc-body toc-right"><DIV class="header-image" id="header_image_attributions" style="background-image: url(https://static.igem.org/mediawiki/2018/4/4b/T--UCSC--Experiment_Cover_Photo.jpg); background-position: 50% 0%"><H1>
     Experiments
    </H1></DIV><DIV class="float-page-list"><LI><H3> Project Pages</H3></LI><LI><A href="https://2018.igem.org/Team:UCSC/Description"> Description </A></LI><LI><A href="https://2018.igem.org/Team:UCSC/Design"> Design </A></LI><LI><A href="https://2018.igem.org/Team:UCSC/Target_Organism"> Target Organism </A></LI><LI><A href="https://2018.igem.org/Team:UCSC/Model"> Modeling </A></LI><LI><A href="https://2018.igem.org/Team:UCSC/Experiments"> Experiments </A></LI><LI><A href="https://2018.igem.org/Team:UCSC/Improve"> Improvement </A></LI><LI><A href="https://2018.igem.org/Team:UCSC/Results"> Results </A></LI><LI><A href="https://2018.igem.org/Team:UCSC/Demonstrate"> Demonstrate </A></LI><LI><A href="https://2018.igem.org/Team:UCSC/Notebook"> Notebook </A></LI></DIV><DIV class="float-toc"><DIV class="toc-container"><LI><H3 id="top_anchor" class="centered"> Experiments </H3></LI><LI><A href="#Expe_lip"> LipLox </A></LI><LI><A href="#Expe_ex1"> Experiment 1 </A></LI><LI><A href="#Expe_ex2"> Experiment 2 </A></LI><LI><A href="#Expe_ex3"> Experiment 3 </A></LI><LI><A href="#Expe_pro"> Protocols </A></LI></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><H2>Experiments: The Progesterone Producing Yeast Strain</H2></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><P>
Experiments: the progesterone producing yeast strain
Our team will use synthetic biology to address insufficient access to contraception by engineering a progesterone-producing yeast. We will construct our gene cassettes and insert them into the <I>Yarrowia lipolytica</I> genome in three parallel experiments. In the preliminary experiment, we will use the strain <I>Y. lipolytica str. FKP393</I> which has the auxotrophic markers <I>leu2</I> and <I>ura3</I> for selection. We will engineer a second strain named Lip<I>Lox</I>, that will contain the gene for uracil flanked by lox sites <I>loxP</I> and <I>lox71</I> for use in experiments 1, 2, and 3 below. This will allow us to have a yeast strain that contains lox sites, as well as a strain that now has uracil in a designed location that we can swap in or out as a selectable marker.</P></DIV></DIV><DIV class="flex-row"><DIV class="flex-col hex-header hex0 center"><H2 id="Expe_lip">CREATE LIPLOX</H2></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><P>The first step in our project was to engineer a strain of yeast with an genetic marker in the location we plan on inserting the genes for the progesterone biosynthesis pathway. We also use Cre-<I>lox</I> integration in later experiments, so we needed to enter <I>lox</I> sites into our strain as well. Because <I>Y. lipolytica str. FKP393</I>  is auxotrophic for uracil, we can replace this gene in a specific location as a marker for a second homologous recombination (HR) experiment. When we perform the other HR experiment, the uracil gene will be swapped out, and we can use 5-FOA media as our selection tool. </P></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><H3>Homology Arms</H3></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><P>We designed homology arms (HA) for homologous recombination that span 1 kb upstream and downstream of the <A href="">ade2 gene</A> in Y. lipolytica str. FKP393 . To prepare HAs for this experiment, and experiment 1, we amplified the HAs out of the <I>Y. lipolytica str. FKP393</I>  genome using primers flagged with Gibson overhangs (GOs) homologous with the gene cassette ends and the linearized pUC19 backbone as seen in Figure 1. This ensured that the HAs were ready for GA after being amplified out.</P></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><P>Figure 1: Isolation of DNA from yeast, and PCR amplification of upstream and downstream homology arms that surround <I>ade2</I> out of <I>Y. lipolytica str. FKP393</I>. Primers with Gibson overhang flags are used to create homology arms with Gibson overhangs on each end that associate to their corresponding fragments.</P></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><P>We performed colony PCR to amplify the HAs. After several failed attempts, we found that we needed to extract the genomic DNA (gDNA) from <I>Y. lipolytica str. FKP393</I> and then PCR-amplify the HAs out of it. To extract the gDNA from <I>Y. lipolytica str. FKP393</I>, we used lithium acetate and sodium dodecyl sulfate (SDS) to lyse the cells, then precipitated the gDNA with ethanol. See our <A href="">Protocols section</A> below for Yeast DNA Extraction protocol. We used touchdown PCR with the gDNA and flagged primers to amplify the arms out. We checked the success of PCR amplification using gel electrophoresis.</P></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><H3>Enter <I>lox</I> sites and <I>ura3</I> into <I>Y. lipolytica str. FKP393</I> genome</H3></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><P>To create Lip<I>Lox</I>, which contains the two <I>lox</I> sites used in experiments 2 and 3, as well as the <I>ura3</I> gene that will act as a selective marker in all experiments, we will Gibson assemble the pOPPY-UC19-yXXU plasmid shown in Figure 2.</P></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><P>Figure 2: pOPPY-UC19-yXXU plasmid in detail. Contains UHA, terminator Tsynth8, <I>loxP</I> site, TEF1 promoter, <I>ura3</I> gene, Tsynth8 terminator again, <I>lox71</I> site, and DHA. Will be used to insert <I>lox</I> sites and <I>ura3</I> into <I>Y. lipolytica str. FKP393</I>.</P></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><P>This plasmid contains our <I>loxP-ura3-lox71</I> gene block, surrounded by the UHA and DHA to <I>Y. lipolytica str. FKP393</I> genome, in the linearized pUC19 backbone. The HAs were designed with ends homologous with <I>loxP-ura3-lox71</I> gene block and pUC19. This design enables GA to seamlessly stitch together all four fragments resulting in pOPPY-UC19-yXXU seen in Figure 2. Because HR requires a linear construct, we will amplify out the product from UHA to DHA. We will perform HR and the <I>loxP-ura3-lox71</I> fragment will be exchanged for the <I>ade2</I> gene region between the HAs in the <I>Y. lipolytica str. FKP393</I> genome (Figure 3).</P></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><P>Figure 3: Creation of pOPPY-UC19-yXXU containing <I>ura3</I>, pUC19 backbone, <I>lox</I> sites and homologous arms using Gibson Assembly. Transformation of <I>Y. lipolytica</I> to create Lip<I>Lox</I> using homologous recombination.</P></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><P>The yeast recombinants, which were previously auxotrophic for uracil, will now contain <I>ura3</I> and will grow on uracil-deficient media, allowing us to select for them. As a second method of confirmation, we will extract DNA from transformed <I>Y. lipolytica str. FKP393</I> cells and use PCR to screen for successful mutants. We will know that we have successful HR results if we observe single bands at the appropriate size (4 kb) on a 1% agarose gel.</P></DIV></DIV><DIV class="flex-row"><DIV class="flex-col center"><H2>THREE PARALLEL EXPERIMENTS</H2></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><P>Experiment 1 will use well-studied methods previously tested in <I>Y. lipolytica str. FKP393</I>. We will assemble the plasmid pOPPY-UC19-yP to create the construct necessary to integrate into the Lip<I>Lox</I> genome to produce <I>Y. lipolytica str. PoPPY</I>. We will Gibson-Assemble the linearized pUC19 plasmid backbone, the UHA and DHA amplified out of the <I>Y. lipolytica str. FKP393</I> genome, and the five progesterone pathway genes to create pOPPY-UC19-yP seen in Figure 4.</P></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><P>Figure 4: pOPPY-UC19-yP plasmid in detail. Contains UHA, DHA, and the genes ∆7-sterol reductase, ADR, FDX1, P450scc, and 3β-HSD, each of which has a TEF1 promoter, Tsynth8 terminator, and Gibson overhangs (each denoted by GO) to assemble the fragments together, and into the plasmid backbone. Will be used to engineer Lip<I>Lox</I> with the progesterone pathway. BioBrick prefix and suffix are described in “BioBrick” section.</P></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><P>
Figure 5: Creation of pOPPY-19-yP via GA of progesterone biosynthesis pathway genes, HAs, and linearized pUC19 backbone. Transformation of Lip<I>Lox</I> using HR to exchange gene cassette. Selection of PoPPY using 5-FOA ura+ media.</P></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><P>HR requires a linear fragment, so we will amplify out our construct from UHA to DHA and perform HR to recombine the construct into the Lip<I>Lox</I> genome. HR will exchange the previously inserted <I>ura3</I> from Lip<I>Lox</I> with the gene cassette from pOPPY-UC19-yP. The new yeast strain, named <I>Y. lipolytica str. PoPPY</I>, will be uracil-deficient again, so we will test for successful integration by growing the yeast on 5-Fluoroorotic acid (5-FOA) ura+ media to select for our new strain containing our gene cassette as seen in Figure 5.</P></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><P>Experiment 2 will combine two well-studied experimental techniques. Yeast mediated cloning, which can assemble fragments with Gibson overhangs  inside of a yeast cell; and Cre-<I>lox</I> recombination, which can perform a homologous-recombination-like insertion where there are corresponding <I>lox</I> sites. We will use YMC in <I>S. cerevisiae</I>, due to that technique being well studied in that organism, to create the plasmid pOPPY-XRL2-yP (Figure 6) that contains the gene cassette required to complete the progesterone biosynthesis pathway. We will isolate this plasmid and then transform it into Lip<I>Lox</I>.</P></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><P>Figure 6: We will linearize the pXRL2 plasmid using inverse PCR to remove one <I>loxP</I> site (<I>loxP</I> cutout). We will re-circularize and re-linearize the plasmid with one <I>loxP</I> site, pOPPY-XRL2-yX. We will insert the genes ∆7-sterol reductase, ADR, FDX1, P450scc, and 3β-HSD, each of which has a TEF1 promoter, a Tsynth8 terminator, and Gibson overhangs (each denoted by GO) to assemble the fragments together, and into the plasmid backbone during YMC to create pOPPY-XRL2-yP.</P></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><P>We will linearize pXRL2 using PCR and remove one of the two loxP sites to produce pOPPY-XRL2-yX. We will re-circularize pOPPY-XRL2-yX before re- linearizing with PCR primers in a region that is homologous with both ends of the gene cassette. We will insert a re-linearized pOPPY-XRL2-yX with the five pathway genes into <I>S. cerevisiae</I> for yeast mediated cloning. We will miniprep out the pOPPY-XRL2-yP plasmid and transform into <I>E. coli</I>. We will insert our gene cassette into the Lip<I>Lox</I> genome using Cre-<I>lox</I> with <I>lox</I> sites entered in experiment 0 to produce <I>Y. lipolytica str. PoPPY</I> as shown in Figure 7.</P></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><P>Figure 7: Creation of pOPPY-XRL2-yP via YMC in <I>S. cerevisiae</I> using linearized pXRL2 and gene fragments. Transformation of <I>Y. lipolytica str. LipLox</I> using pOPPY-XRL2-yP followed by Cre-<I>lox</I> recombination and 5-FOA ura+ selection to create <I>Y. lipolytica str. PoPPY</I>.</P></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><P>We will amplify pOPPY-XRL2-yP in <I>E. coli</I> and isolate the plasmid using the ZyppyTM Plasmid Miniprep Kit from Zymo Research. We will transform pOPPY-XRL2-yP and the pSH47 plasmid that contains the genes for Cre recombinase into Lip<I>Lox</I> and perform Cre-<I>lox</I> recombination. The pXRL2 plasmid contains one <I>loxP</I> site upstream of the gene construct, and the downstream end of the gene construct contains the <I>lox71</I> sequence necessary for Cre-<I>lox</I> recombination. The presence of galactose will induce the expression of Cre recombinase and trigger site-specific recombination between the incompatible <I>lox71</I> and <I>loxP</I> sites on pOPPY-XRL2-yP and in the Lip<I>Lox</I> genome to make PoPPY as in Figure 7. The five pathway genes on pOPPY-XRL2-yP and <I>ura3</I> gene in Lip<I>Loxura3</I> gene through non-selective screening and allow us to select for our desired cells using 5-FOA.</P></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><P>Experiment 3 will be a novel undertaking; while YMC has been tested in Yali, Cre-<I>lox</I> has not. In Experiment 3, we will use YMC in Lip<I>Lox</I> to create the plasmid pOPPY-XRL2-yP (Figure 6) to insert our gene cassette into the LipLox genome to produce <I>Y. lipolytica str. PoPPY</I>, shown in Figure 8.</P></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><P>Figure 8: Lip<I>Lox</I> will create pOPPY-XRL2-yP from the five pathway genes and pOPPY-XRL2-yX in YMC. Cre recombinase will induce recombination between the <I>lox</I> sites on the plasmid and the <I>lox</I> sites in the genome to create PoPPY.</P></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><P>We will insert pOPPY-XRL2-yX with the 5 pathway genes into LipLox to create pOPPY-XRL2-yP (Figure 6). The pXRL2 plasmid contains one <I>loxP</I> site upstream of the gene construct, and the downstream end of the gene construct contains the <I>lox71</I> sequence necessary for Cre-<I>lox</I> recombination. We will amplify pOPPY-XRL2-yP in Lip<I>Lox</I>, wait for YMC to completely assemble the construct, then transform the Cre recombinase gene-containing pSH47 plasmid into Lip<I>Lox</I> to perform Cre-<I>lox</I> recombination and create PoPPY.</P></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><H2 id="Expe_pro">Protocols</H2></DIV></DIV><DIV class="flex-row"><DIV class="flex-col"><BUTTON class="buttonnotebook2" onclick="openInfoBlock('DPP', this, '#e91b63')" id="defaultOpen">Designing PCR Primers</BUTTON><BUTTON class="buttonnotebook2" onclick="openInfoBlock('ECM', this, '#e91b63')"><I>E. Coli</I> Miniprep</BUTTON><BUTTON class="buttonnotebook2" onclick="openInfoBlock('YM', this, '#e91b63')">Yeast Miniprep</BUTTON><BUTTON class="buttonnotebook2" onclick="openInfoBlock('YDE', this, '#e91b63')">Yeast DNA Extraction</BUTTON><BUTTON class="buttonnotebook2" onclick="openInfoBlock('DSP', this, '#e91b63')">Sourced Procedures</BUTTON><DIV id="DPP" class="buttoncontent"><H6> Designing PCR Primers </H6><P>Our teaching assistant, McKenna Hicks, taught us the following methods for primer design. </P><OL><LI>Identify the DNA sequence to amplify or stitch together with overlap extension PCR (OE-PCR). During PCR, primers anneal to their complementary sequence and recruit DNA polymerase to extend the sequence in the 5' to 3' Direction.</LI><LI>Design a forward (5'- 3') and a reverse primer (3'- 5') to amplify both the top and bottom DNA strands. The ideal primer length is between 18 and 23 bp, but this can be changed to accommodate other constraints.</LI><LI>For the forward primer, select a sequence of base pairs (bp) of optimal length.</LI><LI>Repeat Step 3 for the reverse primer and generate the reverse complement of the reverse primer sequence.</LI><LI>Analyze each sequence using a computational primer tool (such as IDT's Oligo Analyzer) and record the GC content and the melting temperature (Tm) for each primer.</LI><LI>The optimal GC content is between 40-60%, and the optimal Tm range is between 52-62℃. Ideally, the difference in Tm for the forward and reverse primer pair should not exceed 5℃.</LI><LI>If the GC content exceeds 60% for either primer, add High GC Enhancer to the PCR reaction mix.</LI><LI>Check that the free energy (ΔG) value for self-dimers and hetero-dimers on each primer is above -10 kcal/mol to avoid primers spontaneously self-annealing or annealing to each other.</LI><LI>Check that hairpin formations do not exceed 30℃ in each primer. PCR uses different temperatures for each thermocycler step; the lowest temperature used during the replication step for primer annealing is 58-60℃. If hairpins form around this temperature, the primer binding efficiency may be reduced and yield less product. However, if the primer Tm is above the hairpin formation temperature, and if the potential hairpin leaves the 3' end of the primer free for annealing, the primer should still anneal properly without reducing PCR Efficiency.</LI></OL></DIV><DIV id="ECM" class="buttoncontent"><H6><I>E. Coli</I> Miniprep </H6><P>The following protocol was adapted from the ZyppyTM Plasmid Miniprep Kit procedure from Zymo Research<SUP class="ref"><A href="#ref-0" title="Reference">[3]</A></SUP>. </P><OL><LI>First prepare your cell samples: centrifuge the cell cultures (which should be 5 mL in conical tubes) for 15 minutes at 3,750 rpm and 30℃.</LI><LI>During waiting time, grab a 250 mL beaker and add roughly 50 mL of 10% bleach. Grab some foil, label \waste&quot;, and cover the top of the beaker. Use this beaker to discard biohazardous supernatants and flow-throughs throughout your miniprep.</LI><LI>Collect Zyppy columns, Zyppy collection tubes, and 2 sets of 1.5 mL microcentrifuge tubes to match the number of samples you centrifuged. Place the columns over the collection tubes. Be sure to label all vessels accordingly. Place a kimwipe over the open columns. </LI><LI>After centrifugation, discard the supernatants into your waste beaker. Be sure to replace the foil lid afterwards.</LI><LI>Add 600 L Milli-Q water into each of the conical tubes containing the cell pellets. Resuspend each by slowly pipetting up and down; use that same pipette to transfer the resuspended mixture to its respective clean microcentrifuge tube. Change tips between samples.</LI><LI>Add 100 L of 7X Lysis Buffer (blue) and mix by inverting the tube 4-6 times. Proceed to next step within 2 minutes. After addition of 7X Lysis Buffer, the solution should change from opaque to clear blue, indicating complete lysis. </LI><LI>Add 350 L of cold Neutralization Buffer and mix thoroughly. The sample will turn yellow when the neutralization is complete and a yellowish precipitate will form. Invert the sample an additional 2-3 times to ensure complete neutralization.</LI><LI>Centrifuge at 13,000 rpm (NOT \rcf&quot; or \x g&quot;) for 3 minutes.</LI><LI>Transfer the supernatant into the provided Zymo-SpinTM IIN column, which should be on top of a collection tube. Avoid disturbing the cell debris pellet.</LI><LI>Centrifuge the column + collection tube for 15 seconds at 13,000 rpm (stop the centrifuge when it reaches 15 sec). </LI><LI>Discard the flow-through and place the column back into the same Collection Tube.</LI><LI>Add 200 L of Endo-Wash Buffer to the column. Centrifuge for 30 seconds at 13,000 rpm. Discard the flow-through.</LI><LI>Add 400 L of ZyppyTM Wash Buffer to the column. Centrifuge for 1 minute at 13,000 rpm. Discard flow-through. Be sure to close cap tightly on wash buffer so that ethanol does not evaporate. </LI><LI>Transfer the column into a clean 1.5 mL microcentrifuge tube then add 50 L of Milli-Q water in place of Elution Buffer (EB) and add it as close to the filter as possible, being careful not to touch it (use Milli-Q instead of EB to prevent contamination from salts and EDTA). Let stand for at least 10 minutes at room temperature. Incubate longer if needed. </LI><LI>Centrifuge for 30 seconds at 13,000 rpm to elute the plasmid DNA. </LI></OL></DIV><DIV id="YM" class="buttoncontent"><H6> Yeast Miniprep </H6><P>The following protocol was adapted from the ZyppyTM Plasmid Miniprep Kit procedure from Zymo Research<SUP class="ref"><A href="#ref-0" title="Reference">[3]</A></SUP>. </P><OL><LI>To prepare your cell samples: centrifuge the cell cultures (which should be 5 mL in conical tubes) for 15 minutes at 3,750 rpm and 30C.</LI><LI>During waiting time, grab a 250 mL beaker and add roughly 50 mL of 10%  bleach. Grab some foil, label \waste&quot;, and cover the top of the beaker. Use this
 beaker to discard biohazardous supernatants and flow-throughs throughout your miniprep.</LI><LI>Collect Zyppy columns, Zyppy collection tubes, and 4 sets of 1.5 mL microcentrifuge tubes to match the number of samples you centrifuged (you can get the Zyppy items from the yellow miniprep kit). Place the columns over the collection tubes. Be sure to label all vessels accordingly. Place a kimwipe over the open columns.</LI><LI>After centrifugation, discard the supernatants into your waste beaker. Be sure to replace the foil lid afterwards.</LI><LI>Add 600 L Milli-Q water into each of the conical tubes containing the cell pellets. Resuspend each by slowly pipetting up and down; use that same pipette to transfer the resuspended mixture to its respective clean microcentrifuge tube. Change tips between samples.</LI><LI>Add 100 L of 7X Lysis Buffer (blue) and add 50 L of small glass beads. Vortex for 5 min. Let stand to allow beads to settle.</LI><LI>Transfer the supernatant to a fresh microcentrifuge tube.</LI><LI>Add 350 L of cold Neutralization Buffer and mix thoroughly. The sample will turn yellow when the neutralization is complete and a yellowish precipitate will form. Invert the sample an additional 2-3 times to ensure complete neutralization.</LI><LI>Centrifuge at 13,000 rpm (NOT \rcf&quot; or \x g&quot;) for 10 minutes.</LI><LI>Transfer the supernatant into the provided Zymo-SpinTM IIN column, which should be on top of a collection tube. Avoid disturbing the cell debris pellet.</LI><LI>Centrifuge the column and collection tube for 45 seconds at 13,000 rpm. </LI><LI>Discard the flow-through and place the column back into the same Collection Tube.</LI><LI>Add 200 L of Endo-Wash Buffer to the column. Centrifuge for 30 seconds at 13,000 rpm. Discard the flow-through.</LI><LI>Add 400 L of ZyppyTM Wash Buffer to the column. Centrifuge for 1 minute at 13,000 rpm. Discard flow-through. Be sure to close cap tightly on wash buffer so that ethanol does not evaporate.</LI><LI>Transfer the column into a clean 1.5 mL microcentrifuge tube, then add 50 L of Milli-Q water in place of Elution Buffer (EB) and add it as close to the filter as possible, being careful not to touch it (use Milli-Q instead of EB to prevent contamination from salts and EDTA). Let stand for at least 10 minutes at room temperature. Incubate longer if needed.</LI><LI>Centrifuge for 1 minute at 13,000 rpm to elute the plasmid DNA.</LI></OL></DIV><DIV id="YDE" class="buttoncontent"><H6> Yeast DNA Extraction </H6><P>We adapted the following procedure from Looke et al.<SUP class="ref"><A href="#ref-0" title="Reference">[4]</A></SUP> with advice from our teaching assistant, McKenna Hicks.</P><OL><LI>Add 100L Y. lipolytica in YPD to 100L 200mM LiOAc and 1% SDS solution. Incubate in water bath at 70C for 15 minutes.</LI><LI>Add 300L of 96% EtOH. Incubate in freezer overnight.</LI><LI>Centrifuge at 8,000 rcf at 4C for 30 minutes. Discard supernatant.</LI><LI>To wash pellet: resuspend in 1ml 70% EtOH, incubate for 5 minutes then centrifuge for 10 minutes with the previous speed and temperature conditions. Discard supernatant.</LI><LI>Repeat wash step once.</LI><LI>Resuspend in 100L 0.5X TE or Milli-Q water; if pellet is not completely resuspended add 0.5X TE or Milli-Q water in 75L increments until completely resuspended.</LI><LI>Use NanoDrop to determine concentration and quality of the DNA.</LI></OL></DIV><DIV id="DSP" class="buttoncontent"><H6> Sourced Procdures </H6><P>The following is a list of procedures we directly sourced for our project.</P><UL><LI> Gibson Cloning<SUP class="ref"><A href="#ref-0" title="Reference">[5]</A></SUP></LI><LI> Column Electrophoresis via SELEX<SUP class="ref"><A href="#ref-0" title="Reference">[1]</A></SUP></LI><LI> Random Mutagenesis via Error-prone PCR<SUP class="ref"><A href="#ref-0" title="Reference">[6]</A></SUP></LI><LI> High Performance Liquid Chromatography and Mass Spectrometry<SUP class="ref"><A href="#ref-0" title="Reference">[7]</A></SUP></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>