Team:UCSC/Results
Home
Team
Meet the Team
Collaborations
Partnership
Attributions
Sponsors
Project
Overview
Design
Applied Design
Experiments
Results
Demonstrate
Notebook
Human Practices
Outreach
Public Engagement
Safety
Notebook
Weekly Overview
Protocols
Parts
Overview
Improvement
Judging Contact
Results
Results
Overview
LipLox
Yeast Mediated Cloning
Riboswitch
Quantification
Project Achievements
OVERVIEW
We constructed the biosynthetic pathway for progesterone, consisting of 5 genes ( Δ7 Red, P450scc, FDX1, ADR, 3Β-HSD). In addition, we created a riboswitch that will allow us to measure the concentration of progesterone.
LIPLOX
Homology Arms
To amplify out the HAs of Y. lipolytica str. FKP393, we lysed the cells and extracted the gDNA. We then performed touchdown PCR with the gDNA and confirmed the amplification of the HAs with gel electrophoresis. The UHA is 1014 bases long, and the DHA is 1000 bases long, so we expected bands at ~1 kb for both PCR samples. Figure 1, shows the expected bands just above 1 kb. It is likely that the bands are above 1 kb because the loaded samples contain Q5 reaction buffer and ran slower than the 2-log DNA ladder. We concluded that the bands produced were from the HAs. We sent the PCR products to be sequenced with primers that would identify regions of the homology arms and were able to confirm that our product were the HAs.
Figure 1: Gel of Upstream and Downstream HAs, 1% TBE gel electrophoresis of the upstream and downstream HA PCR amplifications. We dyed each 5 µL sample with 1 µL of 6X loading dye and loaded 5 µL of this mix into each well. Lane 1 contains the 2-log DNA ladder, lane 2 contains the amplified UHA, and lane 3 contains the amplified DHA.
Enter lox sites and ura3 into Y. lipolytica str. FKP393 genome
We attempted to assemble the loxP-URA3-lox71 geneblock with homology arms from the Y. lipolytica genome using Gibson Assembly (GA) to create pOPPY-UC19-yXXU. We performed GA using 1:10 molar ratio of plasmid backbone to insert, and transformed NEB 5-alpha E. coli with the product, and grew them on high copy LB/AMP plates for 14 hours at 37ºC. The E. coli plates grew colonies, shown below.
Figure 1: pOPPY-UC19-yX transformed in NEB 5-alpha E.coli grown on LB/AMP high copy plate.
We picked 12 colonies, with which we performed colony PCR, and created an index plate. Our colony PCR gel results indicated that the colonies we chose were not homogenous; while some colonies showed light bands at the appropriate size for pOPPY-UC19-yXXU (~4 kb), they also contained other bands around 0.5kb which is the size expected if there is no insert, and just pUC19 backbone.
Figure 1: 1% agarose gel containing samples from colony PCR of pOPPY-UC19-yXXU attempting to detect loxP-ura3-lox71 geneblock with homology arms to ade2.
The gel shows that we may have a mixed population in our colony picks. While there are light bands at 4kb, the bright bands at 0.5kb show that our insert was not included in the majority of the DNA we performed the PCR on. We created streak plates with the colonies on the index plate to attempt to screen out the mutants who had not successfully assembled our insert into the backbone. Unfortunately, after PCR of those, the results returned the same information that our colonies did not include the insert. This is most likely due to the plasmid somehow re-circularizing to itself, or DPN1 not completely digesting the original pUC19 plasmid after isolation from E.coli. Because we were not able to isolate pOPPY-UC19-yXXU, we chose a different route.
The Riboswitch team has been using a plasmid called D17, generously gifted from Cory Schwartz at UC Irvine, which contains homology arms for the pseudogene D17 in Y. lipolytica. We decided to try and insert loxP-ura3-lox71 into that plasmid. If successful, we would then amplify out to obtain a linear construct including the homology arms to D17, which we could perform homologous recombination into the Y. lipolytica genome with. We PCR amplified D17, cutting out plasmid backbone between the homology arms to create a construct with homology arms at each end of the linear product. We ordered primers with Gibson Overhangs (GO) to the D17 plasmid to amplify loxP-ura3-lox71 with. The PCR results were unsuccessful every try. Due to these issues, we decided to use KLD to insert the loxP-URA3-lox71 gene block into D17. We performed KLD treatment with linearized D17 backbone, and the loxP-URA3-lox71 gene block and transformed NEB 5-alpha E.coli. We saw many colonies, and we picked 15 for colony PCR to check if integration occurred.
Figure 1: 1% agarose gel containing colony PCR results from loxP-URA3-lox71 geneblock in between D17 homology arms.
The gel shows in lane 2 that we achieved a clean, single band at ~5kb. The primers bound to the D17 plasmid at the beginning and end of the D17 homologous arms, including the arms and our loxP-URA3-lox71 insert. The expected size of our product is 3.6 kb. The band is slightly higher than this size, which we believe is due to the salts in the cell causing it to run a bit slower than it should. Because of this we are confident that this band is our desired product. We have grown a liquid culture of Colony 2, miniprepped it, and performed PCR using Q5 polymerase and the same primers used for colony PCR to attempt to amplify out loxP-URA3-lox71 sandwiched between the D17 homology arms. The results have not yet shown any bands from this.
Once we are able to amplify loxP-URA3-lox71 with D17 homology arms, we will transform Y. lipolytica with this linear product and confirm success by growing the colonies on uracil-deficient media. Once the loxP and lox71 sites are integrated into the genome we can perform Recombinase mediated cassette exchange with the plasmid created by YMC with all pathway genes using the corresponding lox sites.
YEAST MEDIATED CLONING
We concluded the successful removal of loxP from pXRL2 via site directed mutagenesis. The results of our colony PCR (Figure 1) suggested that the product of the reaction was slightly smaller than the original plasmid. We sequenced to confirm our success and found that the product was missing the desired region in an alignment with the original plasmid. We also conducted attempts of yeast mediated cloning with successful growth in leucine deficient media. This result would suggest that we had a successful transformation and possible assembly of our gene cassette on to the plasmid backbone.
Figure 1: With the primers used in this colony PCR reaction, we should see a fragment of about 270bp if one loxP site was successfully removed from pXLR2 and a 340bp fragment if the loxP site was not removed. We initially believed that the bright bands between 300bp and 400bp in colony 2 and 5 (marked here in green rectangles) indicated failures to remove loxP because they appeared to be the 340bp fragments. We later realized that the gel did not run evenly, as can be seen in the arc of the dark line representing the dye. This resulted in the bands appearing higher than they should have, and these two bright bands likely indicated successful removal of loxP.
We conducted a colony PCR (Figure 1) to check the successful site directed mutagenesis of loxP from pXRL2. We used a pair of primers: cloxP F and cloxP R to amplify the region that we removed and ran the samples on a gel. A successful trial would be a band 300 bp or lower since the fragment including the lox site would be 343 bp. Our results were puzzling due to the band being higher than 300 bp but we thought that this had occurred because the sample ran slower than the other lanes evident in the arc of the dye. We conducted another colony PCR with monoclonal colonies. Strangely, our positive control appeared to be much larger than the other samples but we concluded that it was due to issues with gel electrophoresis. We resolved to use the samples with the lowest bands or smallest size which were 5C, 5D, 21B and 23A. Upon further investigation by sequencing (Figure 2), we concluded that we had succeeded due to the region being missing in an alignment with the original plasmid. However, in sample A, more bases were deleted than expected so we did not use it because of risk of a frameshift. We, then relinearized the four sequenced samples for gene assembly via yeast mediated cloning. The results of this suggested a success since the bands resulting from gel electrophoresis were at the expected 8-10 kb.
Figure 2: We sequenced our relinearized samples of pOPPY-XRL2-yX and aligned the results with the original pXRL2 plasmid or negative control. The green lines indicate the successful removal of loxP by site directed mutagenesis while the red lines show the deletion of extra bases in plasmid A and 21.
Figure 3: After creating a set of liquid cultures of the two colonies we believed to contain plasmids with one loxP site removed (2 and 5), we miniprepped and then ran a PCR using the same primers used in the previous colony PCR (see figure 1). A band at 270bp indicates that loxP was removed, while a band at 340bp would indicate that loxP was not removed. The PCR products of “Miniprep C from Colony 5 liquid culture” (lane 1) showed a band just below the 300bp mark, suggesting that loxP was successfully removed from the plasmid in colony 5. The band in lane 4 also appeared promising, while we felt that the slightly higher bands appearing between 300bp and 400bp in lanes 5 and 9 seemed to indicate that loxP may not have been removed from the plasmid in colony 2. Our negative control, the PCR run on the unaltered pXLR2 plasmid backbone, surprisingly showed a band between 400bp and 500bp rather than the expected 340bp. We think this could potentially be due to errors made in addition of PCR reagents to this specific sample.
Figure 4: In this gel, we ran our samples that were relinearized with inverse PCR subsequent to confirming that loxP was removed. The PCR product resulting from the original plasmid shown in lane 5 appeared to be slightly higher than the pOPPY-XRL2-yX samples in lane 2, 3 and 4. This was expected since the linearization theoretically removed 230bp from the plasmid which was originally 6693bp.
We performed yeast mediated cloning to assemble delta 7-sterol reductase, ADR, FDX1, P450scc and 3ꞵ-HSD into linearized pOPPY-XRL2-yX. We plated the transformants and saw many colonies. We grew 30 colonies selected from leucine-deficient plates in leucine-deficient liquid media to mini-prep them and check for successful plasmid assembly. We rescued the plasmid and ran 250ng dilutions of our samples on a gel (Figure 5). The expected size of the desired plasmid with all pathway genes, pOPPY-XRL2-yP, was approximately 15kb in comparison to pXRL2 which is 6693bp.
Figure 5: All of the lanes contained circularized plasmids. Samples 1, 2, 9 and 14 had bands higher than the circular plasmid backbone control and above 10kb, suggesting they may have contained some number of genes. We selected the plasmid in Lane 2, 9, and 14 for sequencing to confirm their success.
RIBOSWITCH
For our part of the project, we intended to create a one-gene reporter system for progesterone concentration consisting of: a Renilla GFP gene (hrgfp), a cyc1 terminator, TEF(136) promoter, and a hammerhead ribozyme expression platform attached to a progesterone-specific aptamer.
Our first task in this project was to create a large stock of our pHR_D17_hrGFP plasmid (from here on referred to as D17), which we did by transforming a portion of our IDT D17 stock into chemicompetent E. coli, running a selection on an LB agar plate with a 100 ug/L concentration of ampicillin, and then growing out selected colonies in liquid LB medium. We then isolated our samples of D17 from the bacterial culture by using a Zymo plasmid miniprep kit.
In order to confirm the sequence of our hrgfp gene as well as the cyc1 terminator, we also designed a series of primers meant to be used in Sanger sequencing of our D17 plasmid. These primers were labeled Rib(f)-1 through Rib(f)-3 and Rib(r)-1 through Rib(r)-3 and encompassed the entire hrgfp gene as well as the cyc1 terminator with several hundred base pairs of overhang on either side. Sequencing data from the UC Berkeley DNA sequencing facility showed a successful primer design and an intact DNA sequence in our D17 plasmid.
Figure 1: Agarose gel confirming the presence of our linearized D17 plasmid at around the 9kb mark.
In order to carry out Gibson Assembly, we also had to amplify our stock of riboswitch inserts. To do this, we designed the Apt(f) and Apt(r) primers to anneal to the overhangs of all 5 different inserts. Running a series of PCR trials and imaging on an agarose gel in Figure 2 showed strong bands at the expected bp length, indicating successful amplification of our inserts.
Figure 2:  Agarose gel confirming the amplification of riboswitch oligomers on lanes 2 through 5.
We then performed a Gibson assembly trial with our newly amplified D17 vector and riboswitch inserts. We attempted a confirmation PCR to check if our inserts were successfully integrated into the D17 backbone with the ampGB(f) and ampGB(r) primers. A successful integration event would result in amplicons approximately 4.2 kb in length, while a circular D17 template would create an amplicon 4.0 kp in length. Imaging of our gel in Figure 3 suggested the continued presence of D17 circular template along with our new pOPPY_GFP plasmids, even after digestion with >500-fold excess of DpnI.
Figure 3:  Agarose gel comparing the presence of linear vs. circular D17 template amplification after Gibson assembly.
To remedy this, we performed a series of transformations and selective platings with our Gibson assembly product on 100 ug/mL LB-ampicillin medium and selected a total of 52 colonies--10 from each Gibson transformation and 2 from a D17 control plate--for colony PCR and spot plating on an index plate. The imaged gels in Figure 4 from the colony PCR trials allowed us to select single colonies we believed to be carrying our riboswitch insert.
Figure 4: Agarose gels of index plate colonies 1-52, 1-10 of pOPPY_GFP 1, 11-20 of pOPPY_GFP 2, and so on. Colonies 51 was our negative D17 control. We looked for a 200 base pair difference in each pOPPY_GFP variant to confirm presence of our riboswitch insert.
From these gels we selected colonies 2, 16, 27, 32, 43, as likely carriers of pOPPY_GFP(1)--pOPPY_GFP(5), respectively, and colony 51 as a control carrier of D17. We then used our previously grown index plate to create monoclonal liquid cultures of these colonies in LB medium. After overnight incubation, we then performed a plasmid isolation with all 6 cultures and transformed the isolated plasmids into our competent Y. lipolytica strain, which we created using the Zymo Frozen EZ-yeast transformation protocol.
QUANTIFICATION
From the pOPPY-XRL2-yP plasmids that YMC had assembled, supposedly including all of the progesterone pathway genes, we chose six colonies to inoculate and test for progesterone production. We chose the colonies YL 10, YL 11, YL 20, YL 21, YL 22, and YL 25, shown in Figure X. We chose these both arbitrarily and due to the presence of apparent plasmid bands at the appropriate size.
Figure 3:  Colonies of Y. lipolytica transformed with progesterone pathway genes grown on leucine-deficient media. The pOPPY-XRL2-yP plasmid contains the gene completing the  leucine pathway, acting as our auxotrophic marker.
After purification of progesterone according to the Progesterone Extraction protocol, we performed “dot blotting” on nitrocellulose membranes, and detected using anti-progesterone antibodies conjugated to HRP (sc-53423 HRP), gifted from Santa Cruz Biotech (Santa Cruz Biotechnology, Inc., TX, USA).
Figure 3:  Dot blot of progesterone re-suspended in DMSO, isolated from colonies grown on leucine-deficient media after YMC with progesterone pathway genes. Negative control circled in black and marked with a dash (-). Positive controls circled in blue. Experimental samples circled in pink. Primary antibody used was anti-progesterone-HRP, blocked with BSA in TBS, washed with TBS/TWEEN. Positive controls are 1mg/100uL progesterone in DMSO with 1:100 dilutions. Negative control is Y. lipolytica transformed with pD17 with no pathway genes which underwent same extraction protocol as all other samples.
Our dot blot shows that we have high luminescence in multiple colonies, YL 10, YL 20, YL 21, and YL 25. We performed the extraction protocol on +1 positive control, and our negative control. The +1 positive is 1mg of pure progesterone extract from Sigma Aldrich (P8783 Sigma) that we added 1mL of MilliQ water and 1mL of hexane to, and then followed the rest of the extraction protocol for. The positive controls +3, +4 and +5 are 1:10 dilutions from +2 positive control of 1mg/1mL of pure progesterone in DMSO. These positives were not subjected to the extraction protocol. We believe that they may show no luminescence due to +2 being one order of magnitude less than detection that is possible for the antibodies. The negative control is pD17 transformed into Y. lipolytica with no pathway genes, and then subjected to the same extraction protocol.
We believe that these results suggest that we have at least four progesterone producing Y. lipolytica colonies. The positive control shows the level of luminescence for 1mg of progesterone, and the colonies that show luminescence as well are of similar magnitude. We inoculated 5 mL cultures with our colonies, so we find it a bit suggestive that we have similar levels of luminescence in our 1 mg positive control as we do in our colonies.
Due to the ability of our hexane extraction protocol to apparently purify our samples so greatly, we will perform mass spectrometry on these samples in later work.
PROJECT ACHIEVEMENTS
Successes
Failures
Successful Results
The following tasks were successfully completed by our team.
LoxP removal from pXRL2
Lox-URA3-Lox in D17
Amplification of HAs
Gibson Assembly of pOPPY_GFP plasmids
Once we received a set of improved primers, our D17 plasmid had overhangs that allowed for more efficient Gibson Assembly. We confirmed this by choosing 50 colonies from a series of transformations with our pOPPY_GFP plasmids for colony PCR. Running these individual PCRs on a gel allowed allowed us to select colonies that showed bands consistent with an insert of the correct size in the same location as our riboswitch insert.
Amplification of Δ7, P450scc, ADR, and 3ΒH genes
Successful Yeast-Mediated Cloning in Y. lipolitica
Made Progesterone
Unsuccessful Results
The following bullets include experiments and tasks that were not successful during the wetlab period.
Overlap Extension PCR
After many touchdown PCR trials, we were unable to assemble the full genes without seeing non-specific sequences in our PCR reactions. We tried using PCR without primers to anneal the overlapping regions of the gene fragments to create better DNA templates, but there were always non-specific bands alongside the full-length genes. Even after extracting the correct sized bands from agarose gels and using that DNA as a template resulted in the same non-specific banding patterns.
Attaching Homologous Arms to Lox-URA3-Lox
Gibson Assembly of Genes into pXRL2
P450 into PoPPY and D17 Plasmid
Lox-URA3-Lox into PoPPY Plasmid
Amplification of FDX1 gene
Ferrodoxin (FDX1) was one of the five genes in the progesterone pathway that we tried to amplify for Gibson Assembly and Yeast Mediated Cloning. After many touchdown PCR trials, we found a few thermocycler conditions that seemed to properly amplify FDX1. When we tried creating larger PCR reaction volumes, we saw bands that were much larger than the expected gene size for FDX1, and we couldn’t get the same amplification like in earlier PCR reactions.
Isolating pOPPY_GFP plasmids from D17 backbone
We had repeatedly tried to separate our pOPPY_GFP plasmids from their original D17 backbone upon completion of Gibson Assembly using both DpnI digests as well as gel electrophoresis. The small length difference between the plasmids (~200bp) did not allow for enough separation to perform a gel purification, and PCR repeatedly created amplicons indicative of both pOPPY_GFP plasmids as well as D17 backbone.
