"
Page
Discussion
View source
History
teams
Log in
 
Team:SYSU-China/file/Project/Result/Mutagenesis.html
From 2014.igem.org
Contents
1 Mutagenesis·RESULT
1.1 Obtaining functional genes
1.2 Expression test of mutagenic genes
1.3 Test of evolutional mutation
1.4 Module and simulation of the mutation rate
Mutagenesis·RESULT
The work of mutagenesis group focuses on the construction and tests of mutagenic device. The mutagenesis plasmid’s structure has been described in <a href=" https://2014.igem.org/Team:SYSU-China/content.html#Project/Design/Mutagenesis " target="_blank">Design-Mutagenesis</a> and we specifically aim to confirm that:
1）All functional genes mutagenic genes can be expressed in our system.
2）This device can remarkably increase the mutation rate of target sequence.
3）(In the future) The functionality of this device can be activated or deactivated at proper stages, and work in tune with B2H system and M13 proliferation cycle, further conducting directed revolution.
Obtaining functional genes
The four primitive genes dnaQ, umuD, umuC, and recA are obtained from genome of E. coli BL21 (DE3) by PCR and transferred into pMD-19T. The reason for using pMD-19T is to make the following procedures easier to be carried out. Using quick-change PCR (see Fig-1), we conducted point-mutations on dnaQ and recA and obtained dnaQ926 (D12A, E14A) and recA730 (E39K). For recA730, two synonymous mutations were performed additionally to delete the inherent restricted enzyme recognition sites EcorI and PstI. UmuD’ was the N-term (4-72bp) truncated umuD.
<a class="fancybox" rel="group" href="
"><img src="
" alt="" /></a>
Figure 1 &nbspProcess of quick-change PCR
We checked the plasmids by PCR to make sure functional genes have been inserted. The agarose gel electrophoresis result is showed as below. All genes have been confirmed by sequencing.
<a class="fancybox" rel="group" href="
"><img src="
" style="width:450px; heigth:auto;margin-left:125px" alt="" /></a>
Figure 2 &nbspMutagenic genes PCR result
Expression test of mutagenic genes
After we had got the four mutagenic genes, plac (BBa_R0010) was chosen as the promoter in our expression vectors. RFP was used to test whether genes were expressed. Specific operation method is to add RFP (BBa_E1010) behind single mutagenic gene and transfer the plasmid into BL21. Testing RFP fluorescence (480nm/510nm), the positive index indicated the successful expression of our genes.
The plasmid structures are as follows:
&nbsp&nbsp&nbsp&nbspplac + RBS (BBa_B0034) + mutagenic gene + RBS + RFP (BBa_E1010)
We planned to express four mutagenic genes on one single plasmid, so we also constructed a test vector as follows:
&nbsp&nbsp&nbsp&nbspplac + RBS + mutagenic gene + RBS + mutagenic gene+ RBS + mutagenic gene+ RBS + RFP
Pure liquid LB broth was chosen as blank control.
<a class="fancybox" rel="group" href="
"><img src="
" alt="" /></a>
Figure 3 &nbspFluorescence intensity of RFP
In the future, we plan to test these mutagenic genes combined with more inducible promoter pbad (BBa_I13453) and RBS with different intensities.
Test of evolutional mutation
All together, the four mutagenic genes should induce obvious DNA mutation. In order to test such mutation, we use RFP as mutation target. As showed in Fig-4, we constructed the mutagenesis plasmid, pSB1A2 with similar Plac, RBS and all four genes as polycistron.The reporter plasmid to be mutated is pSB1C3 with constructive promoter BBa_J23100, RBS and RFP. Then we co-transformed both plasmids into E. coli ER2738, a strain able to be infected by M13. After cultivation of positive red strains in LB (with Amp and Cm)solution, the diluted bacteria was spread on LB plate with Amp and Cm. If white strains, rather than red, are observed (which indicates RFP chromophore is mutated), pick out the white strains and sequence the promoter and RFP to confirm whether it is exactly mutated.The test of mutation rate is still undergoing.
<a class="fancybox" rel="group" href="
"><img src="
" alt="" /></a>
Figure 4 &nbspTest of evolutional mutation
Module and simulation of the mutation rate
For the DNA polymerase III proof-readingε-subunit, DNAQ926 works by competition mechanism. We have constructed a module to simulate the mutation rate when the two kinds of subunits (wild-type and mutant-type) both exist in E.coli. As reported, the mutation rate of well-functioned DNA polymerase is 10^(-10), while that of the proofreading-defected one is 10^(-5).For there is no significant functional difference between the defective and non-defective polymerase but mutation rates,we calculate the average mutation rate with formula as follows:
<a class="fancybox" rel="group" href="
"><img src="
" style="width:500px; heigth:auto;margin-left:100px" alt="" /></a>
We assume that the number of DNAPIII is 15 as it’s ranged from 10 to 20 in E.coli. The mutation rate changes as below when the number of defective polymerase varies from 1 to 40. <a class="fancybox" rel="group" href="
"><img src="
" style="width:550px; heigth:auto;margin-left:125px" alt="" /></a>
Figure 5 &nbspAverage mutation rate increases with number of DNAQ926
Retrieved from " http://2014.igem.org/Team:SYSU-China/file/Project/Result/Mutagenesis.html "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
