"
Page
Discussion
View source
History
teams
Log in
 
Team:SYSU-China/file/Project/Result/M13.html
From 2014.igem.org
M13·RESULT
Overall, our job is to construct a suitable modified M13 phage vector for IgEM. The ideal M13 vector for our system is the one which is breeding-defective but can be rescued by exogenous gene supplement. So we design a rescue experiment to verify it.
The rescue experiment involves in two plasmids. The first one is the modified M13 phage vector, which fails to produce progeny phage. The other one is the rescue plasmid, an inducible gene expression vector being able to exogenously supply the deleted gene of M13 phage vector. Both vectors would be co-transformed into the host and we would check whether phage progeny can be generated after induction.
Modified M13 phage vector construction
Our basic phage vector, named M13KE (Fi. 1A), is derived from the commercial Phage Display system of New England Biolabs.
We modified the M13KE vector in several aspects.
First, kanamycin resistance cassette (BBa_P1003), as a selective marker, has been inserted into LacZ locus for convenience of further selection. Because deleting an essential gene from M13 bacteriophage would lose its infectivity, generating no virion to infect the host. Also, the deficient M13 vector could not be maintained by the bacteria after transformation. This excessive vector is named M13KdZ (K represents the kanamycin resistance, and dZ means the lacZ is defected) (Fig. 1B).
To ensure that this primarily engineered M13 vector has a chance to be rescued after additional modification, we verified M13KdZ’s ability of generating progeny phage using Infection Assay. Because the fully package phage genome contains the kanamycin resistance gene, the host E.coli ER2738 would gain the capability to form colonies on kanamycin plates (Fig. 1D).
Second, delete a core gene in order to prevent the vector from breeding itself. The gene III of M13 phage is partly deleted from the M13KdZ vector as previous described[1] using PCR. An XhoI restriction site is left inside the residual gene III fragment with some extra bases on the primer (Figure 1C). This infect-defeated phage vector is named M13KdZdpIII (dpIII means the gene III protein is defected).
All vectros above are confirmed by restriction analysis (Fig. 1A,B,C) and sequencing. Though we didn’t fully sequence the whole vector, the changed part of the vector is verified by sequencing result.
<a class="fancybox" rel="group" href="
"><img src="
" alt="" /></a>
Figure 1. Plasmid maps and verification of M13-based vectors.
<p1> The restriction sites on the plasmid maps are partly shown, which are all unique sites. Because all vectors contains the basic elements of a virus, a band cannot be digested is visible after plasmid extraction, possibly the mediate DNA form of the phage. (A) Plasmid map and restriction analysis result of M13KE. M13KE is 7222bp in length totally and should generate a 3.6kb and a 4.6 fragment after digestion of KpnI and PstI. (B) Plasmid map and restriction analysis result of M13KdZ. M13KdZ is 8214bp in length totally and should generate a 1.0kb and a 7.2kb fragment after digestion of EcoRI and PstI. (C) Plasmid map and restriction analysis result of M13KdZdpIII. M13KdZdpIII is 7026bp in length totally. Because the KpnI site is inside the deleted gene III, even EcoRI and KpnI digest together can only cut the plasmid once. Also, XhoI is introduced into the vector during PCR, so a 4.5kb and a 2.5kb fragment should be generated after digestion of EcoRI and XhoI. (D) Infection Assay result of M13KdZ. On the left plate, only phage is added into Top Agar. On the right plate, phage and ER2738 are added into Top Agar after 5 minutes’ incubation at room temperature. </p1>
After all, the gene yet to be evolved should be inserted properly when IgEM is completed established.
Rescue plasmid construction
To fulfil the rescue experiment, gene III is cloned from M13KE, with additional FLAG tag on the C-terminal onto the pSB1C3 vector. Then, araC (BBa_I13458), pBAD promoter (BBa_I13453) and an RBS (BBa_B0034) is constructed in front of the gene III (BBa_K1333002) in order, completing the G3P expression vector (Figure 2A). The rescue plasmid is verified by both restriction analysis (Data not shown) and sequencing.
The expression efficiency is tested in the E. coli strain ER2738, which is a suitable host for M13 bacteriophage. Using arabinose to induce G3P to express, the expression level shows well relevance to the arabinose concentration (Figure 2B).
<a class="fancybox" rel="group" href="
"><img src="
" alt="" /></a>
Figure 2. Plasmid map and G3P expression efficiency of rescue plasmid.
<p1> A. Plasmid map of rescue plasmid. B. Western-blot result of induced G3P protein expression from rescue plasmid in ER2738. The G3P is induced by arabinose (Ara). Minus sign means the group without induction and the number shows the final concentration of arabinose. Sample preparation is described in protocol. </p1>
Rescue experiment
In an ideal rescue experiment, after co-transformed the M13KdZdpIII and rescue plasmid into E. coli ER2738, plenty of progeny phage can be observed when arabinose is added to induce the G3P expression from rescue plasmid, while few progeny can be obtained when arabinose is not added.
Firstly, we tested whether rescue plasmid can be induced after co-transformation (Protocol described in B2H and Mutation section). By selecting different clones from the double-antibiotic plate to induce G3P expression, it can be known that different clones indeed perform differently while being induced (Fig. 3A). According to this preliminary result, clone 4 and 5 would be selected out to fulfil the rescue experiment.
<a class="fancybox" rel="group" href="
"><img src="
" style="width:600px; heigth:auto;margin-left:50px" alt="" /></a>
Figure 3. Preliminary result of rescue experiment.
<p1> (A) Western-blot result of co-transformed ER2738 G3P expression from different clones. 1-5 are the number of randomly selected clones. Minus sign means the group without induction, which is from clone 1. (B) Result of rescue experiment. There should be the figure showing the correlation between arabinose concentration and infectious phage number progeny. </p1>
Next, screening out the clone with best performance, the complete rescue experiment would be operated after Wiki frozen. Hopefully, the correlation between arabinose concentration and infectious phage number progeny in rescue experiment could be confirmed soon.
Reference
[1]Dübel S. et. al., A helper phage to improve single-chain antibody presentation in phage display, Nat Biotechnol. 2001 Jan;19(1):75-8.
Retrieved from " http://2014.igem.org/Team:SYSU-China/file/Project/Result/M13.html "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
