<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SYSU-China_file_Project_Design_Mutagenesis_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SYSU-China/file/Project/Design/Mutagenesis.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SYSU-China/file/Project/Design/Mutagenesis.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SYSU-China/file/Project/Design/Mutagenesis.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SYSU-China/file/Project/Design/Mutagenesis.html&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SYSU-China/file/Project/Design/Mutagenesis.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SYSU-China/file/Project/Design/Mutagenesis.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><H1><SPAN class="mw-headline" id="Mutagenesis.C2.B7DESIGN">Mutagenesis·DESIGN</SPAN></H1><P>
In traditional directed evolution, a mutation library is achieved mostly by PCR and DNA shuffling. Differently, we introduced a mutagenesis module in the IgEM system to make the diversification automatic and more controllable. In order to generate enough mutants of the interested sequence carried by the phage, an inducible mutagenesis plasmid was constructed (see Fig.1), inspired by the work of Kevin M. Esvelt [1].
</P><P>&lt;a class=&quot;fancybox&quot; rel=&quot;group&quot; href=&quot;&quot;&gt;&lt;img src=&quot;&quot; style=&quot;width:400px; heigth:600px; margin-left:160px&quot; alt=&quot;&quot; /&gt;&lt;/a&gt;</P><DIV style="text-align: center;">Figure 1  &amp;nbspStructure of the mutagenesis plasmid</DIV><P>
The increased mutation rate is achieved by suppressing the proofreading ability of DNA polymerase Ⅲ and enhancing the translesion synthesis in the SOS response.
</P><H2>
Suppressing the proofreading
</H2><P>
The dnaQ gene encodes the ε-subunit of DNA polymerase III holoenzyme, which acts as the proofreading 3' exonuclease. During DNA synthesis, mispaired bases are removed by the ε-subunit soon after they are incorporated. By changing two amino acid residues, we get dnaQ926, a defective mutant whose products interfere proofreading by competing with wild-type ε-subunits [2].
</P><P>&lt;a class=&quot;fancybox&quot; rel=&quot;group&quot; href=&quot;&quot;&gt;&lt;img src=&quot;&quot; alt=&quot;&quot; /&gt;&lt;/a&gt;</P><DIV style="text-align: center;">Figure 2 &amp;nbspDefective proofreading in DNA synthesis</DIV><P>&lt;p1&gt;
(a)An adenine nucleotide has been mistakenly added to the newly-synthesized DNA strand;
&lt;/p1&gt;&lt;p1&gt;
(b)Different from wild-type Pol III, the mutant one with defective ε-subunit (ε’), without 3’→5’exonuclease function, fails to remove the mispaired nucleotide;
&lt;/p1&gt;&lt;p1&gt;
(c)The mistake is retained in the progeny DNA strands.
&lt;/p1&gt;</P><H2><SPAN class="mw-headline" id="Enhancing_the_translesion_synthesis">Enhancing the translesion synthesis</SPAN></H2><P>
When a cell experiences DNA damage, RecA proteins will bind to single-stranded DNA formed consequentially by the damage, and induce the SOS response by inactivating the repressor of umuD and umuC. The UmuD protein then undergoes an auto-digestion and produces UmuD’ which associates with the umuC product to form a complex UmuD’2C, the form active in translesion synthesis [3]. To accelerate this progress, recA730, umuD’ and umuC are introduced in the mutagenesis plasmid. RecA730 is one of the mutants of recA whose product is the most proficient in the competition for single-stranded DNA and thereby more activated for the repressor inactivation that leads to SOS induction [4].The over-expressed umuD’ and umuC will enhance error-prone lesion bypass and inhibit RecA-mediated homologous recombination, an additional mechanism by which the umuDC gene products help <EM>E. coli</EM> tolerate DNA damage [3].
</P><P>&lt;a class=&quot;fancybox&quot; rel=&quot;group&quot; href=&quot;&quot;&gt;&lt;img src=&quot;&quot; alt=&quot;&quot; /&gt;&lt;/a&gt;</P><DIV style="text-align: center;">Figure 3  &amp;nbspProcess of translesion synthesis</DIV><P>
Soon after infection, phage DNA replicates in a speed much faster than <EM>E. coli</EM>’s genome. Progeny M13 phages are then released from the hosts and infect other uninfected bacterium. Owing to this fast speed of the phage life cycle, mutations take place mostly in the phage DNA and the mutation of host cells should therefore have little impact on the selection. With the help of inducible promoter, the mutagenesis module will work only when it is necessary and the mutation rate can be adjusted according to needs with different inductor concentrations, making this module more controllable. 
</P><H2><SPAN class="mw-headline" id="References:">References:</SPAN></H2><P>&lt;p1&gt;
[1].	Esvelt, K.M., J.C. Carlson and D.R. Liu, A system for the continuous directed evolution of biomolecules. Nature, 2011. 472(7344): p. 499-503.
[2].	Fijalkowska, I.J. and R.M. Schaaper, Mutants in the Exo I motif of Escherichia coli dnaQ: defective proofreading and inviability due to error catastrophe. Proc Natl Acad Sci U S A, 1996. 93(7): p. 2856-61.
[3].	Opperman, T., et al., A model for a umuDC-dependent prokaryotic DNA damage checkpoint. Proc Natl Acad Sci U S A, 1999. 96(16): p. 9218-23.
[4].	Lavery, P.E. and S.C. Kowalczykowski, Biochemical basis of the constitutive repressor cleavage activity of recA730 protein. A comparison to recA441 and recA803 proteins. J Biol Chem, 1992. 267(29): p. 20648-58.&lt;/p1&gt;</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:SYSU-China/file/Project/Design/Mutagenesis.html">http://2014.igem.org/Team:SYSU-China/file/Project/Design/Mutagenesis.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SYSU-China/file/Project/Design/Mutagenesis.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SYSU-China/file/Project/Design/Mutagenesis.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SYSU-China/file/Project/Design/Mutagenesis.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SYSU-China/file/Project/Design/Mutagenesis.html&amp;oldid=387577" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>