"
Page
Discussion
View source
History
teams
Log in
 
Team:SYSU-China/file/Project/Discussion.html
From 2014.igem.org
Discussion & Future Work
It can be said that evolution is the most fascinating phenomena in biology. Evolution has the simplest strategy (mutation, selection, evolution), which, nevertheless, is so beautifully complex and wonderful. Ever since the birth of modern biology, humans continually try to mimic the evolution, hope to get a wide variety of products, including proteins, nucleic acids, and peptides and so on. However, the traditional method of artificial Evolution is time-consuming and has limited ability. This year, iGEM-SYSU China Team designed the IgEM, expecting that the various tedious work in artificial evolution can be integrated as one and to realize automatic directed evolution.
The current system
In early experiments, we have accomplished the key step in integration: the testing and verification of the Bacterial two-hybrid system. Using ligands and receptors with different interaction strength, we succeeded in producing the different intensities of reporter gene expression, which indicating that there is a correlation between the ligand-receptor interaction and reporter gene expression. Meanwhile, we also tried to integrate RNAT with Bacterial two-hybrid system, and have built the plasmid we want. In the future, we hope to use the similar way of IgEM to achieve a wider variety of directed evolution. For example, by imitating a variety of logic gates【1】 (Shown as figure1), we can construct different interaction way of the Bacterial two-hybrid system, so that we can implement different types of interaction evolution and not just the protein-protein binding.
<img src="
" style="width:600px;margin-left:50px"></a>
<p1 style="text-align: center">Fig.1 Different logic gates which can be used in IgEM </p1>
Meanwhile, we successfully knocked out gene3 which is closely related to phage packaging, and the rescue experiments is in progress. Now, we are trying to integrate B2H with M13 geneⅢ to test whether the geneⅢ producing by B2H system can rescue the defective phage. Next, we plan to knockout other phage genes, including the geneⅧ, which has a similar function with geneⅢ and the geneⅡ, which is associated with phage genomes replicate, and then, we will choose the best way to apply.
In addition, for the continual evolution, on the one hand we introduced the mutation mechanism by damaging the proofreading activity of the DNA polymerase, on the other hand we enhanced the error-prone repair ability of the E.coli, so that we can extend the follow-up directed evolution library. However, there is a huge problem: the mutation in our system is random, which can result in the collapse of the entire system. To avoid this problem, now we replace the infected E.coli with fresh E.coli to prevent mutations incorrectly appear in bacterial genomes. We consider, whether we can achieve the site-directed mutagenesis in the future.
<img src="
" style="width:500px;margin-left:100px"></a>
<p1 style="text-align: center">Fig.2 The sketch of site-directed mutagenesis realized by CRISPR </p1>
Finally, in order to achieve the controllability of our device, we introduced the RNAT. Preliminary experiments have shown the RNAT function correctly. At present, the integration of RNAT and B2H has been completed, and need the further validation experiments.
From the above we can know that our design has been part work. Once successfully integrate all parts together, we hope to use some cases to test whether our system can evolve different kind of things including scFv【2】【4】, enzyne【3】, and so on.
The Future System
Due to the lab conditions and time limits , at present our experimental design is not the best  one. In current designs, one bacteria contains too many plasmids, which is a disadvantage for growth and protein expression of bacteria. We therefore consider whether we can split our system into two types of bacteria. So that the mutagenesis module can less effect to other part of the system.
In addition, because we couldn't find a suitable laboratory, we have to limit our ideas into prokaryotic organisms. In fact, the IgEM in prokaryotes has many big limitations. We know that many proteins come from eukaryotic cannot be folded correctly in prokaryotes. And also, prokaryotes lack the capacity of the post-translational modification of proteins, which is essential for protein function (such as antibodies). Therefore, in the future, we want to apply our IgEM in more organisms, such as yeast or mammalian cells. In this way, we can evolve more proteins, and can also solve the problems of protein post-translational modification and folding.
Reference
[1] Silva-Rocha, Rafael, and Víctor de Lorenzo. "Mining logic gates in prokaryotic transcriptional regulation networks." FEBS letters 582.8 (2008): 1237-1244.
[2] Ueda, Masashi, et al. "Gallium-68-Labeled Anti-HER2 Single-Chain Fv Fragment: Development and In Vivo Monitoring of HER2 Expression." Molecular Imaging and Biology (2014): 1-9.
[3] Tawfik, Dan S., and Andrew D. Griffiths. "Man-made cell-like compartments for molecular evolution." Nature biotechnology 16.7 (1998): 652-656.
[4] Graff, Christilyn P., et al. "Directed evolution of an anti-carcinoembryonic antigen scFv with a 4-day monovalent dissociation half-time at 37 C." Protein Engineering Design and Selection 17.4 (2004): 293-304.
Retrieved from " http://2014.igem.org/Team:SYSU-China/file/Project/Discussion.html "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
