<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_HKUST-Hong_Kong_Characterization"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:HKUST-Hong_Kong/Characterization">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:HKUST-Hong_Kong/Characterization&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:HKUST-Hong_Kong/Characterization&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:HKUST-Hong_Kong/Characterization&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:HKUST-Hong_Kong/Characterization" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:HKUST-Hong Kong/Characterization</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>Team:HKUST-Hong Kong - 2012.igem.org</TITLE></P><DIV id="Navigation_top"><DIV class="Navigation_Buttons" id="Team" align="center"><H3><P>TEAM</P></H3></DIV><DIV class="Navigation_Buttons" id="Project" align="center"><H3><P>PROJECT</P></H3></DIV><DIV class="Navigation_Buttons" id="Wet_lab" align="center"><H3><P>WET LAB</P></H3></DIV><DIV class="Navigation_Buttons" id="Human_practice" align="center"><H3><P>HUMAN PRACTICE</P></H3></DIV><DIV class="Navigation_Buttons" id="Extras" align="center"><H3><P>EXTRAS</P></H3></DIV><DIV class="Navigation_Content" id="Team_Content"><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Introduction">Introduction</A></P></DIV><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Supervisor">Supervisor</A></P></DIV><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Instructor">Instructor</A></P></DIV><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Members">Members</A></P></DIV><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Advisors">Advisors</A></P></DIV></DIV><DIV class="Navigation_Content" id="Project_Content"><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Project_Abstraction">Abstract</A></P></DIV><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Background_and_Motive">Motive</A></P></DIV><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Design_Overview">Design - Overview</A></P></DIV><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Design_Module">Design - Module</A></P></DIV><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Design_Chassis">Design - Chassis</A></P></DIV></DIV><DIV class="Navigation_Content" id="Wet_lab_Content" align="center"><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Parts_and_Device">Parts and Devices</A></P></DIV><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Notebook">Notebook</A></P></DIV><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Characterization">Characterization</A></P></DIV><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Achievement">Achievement</A></P></DIV><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Future_Work">Future Work</A></P></DIV></DIV><DIV class="Navigation_Content" id="Human_practice_Content" align="center"><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Overview">Overview</A></P></DIV><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Interview">Interview</A></P></DIV><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Presentation">Presentation</A></P></DIV><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Calendar">Calendar</A></P></DIV></DIV><DIV class="Navigation_Content" id="Extras_Content" align="center"><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Medal_Requirements">Medal Requirements</A></P></DIV><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Safety">Safety</A></P></DIV><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Attribution">Attribution</A></P></DIV><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Acknowledgement">Acknowledgement</A></P></DIV><DIV class="Content_Buttons"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Glossary">Glossary</A></P></DIV></DIV></DIV><DIV><P align="center"><FONT size="20">Characterization</FONT></P></DIV><DIV id="paragraph1" class="bodyParagraphs"><DIV align="left"><H1>Introduction</H1></DIV><P>In our project, we have characterized two promoters and the cell death device using different methods. The results indicate that our parts are functional and we can quantitatively control their activities by changing the experimental conditions.</P></DIV><DIV id="paragraph2" class="bodyParagraphs"><DIV align="center"><H1>Low Efficiency Constitutive Promoter <EM>Ptms</EM></H1></DIV><P><STRONG>Background  Information <U><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Module/Regulation_and_control" target="_blank">(link to Regulation and Control Module)</A></U></STRONG>




 



  The key reason for using this low efficiency constitutive promoter in our construct is to enable our bacteria to express a low level of antitoxin so that the bacterial cell can only tolerate a certain amount of toxin. As the expression of BMP2 is tightly linked to the toxin, its expression can be regulated accordingly.</P><P><STRONG>Objective</STRONG>
  Our objective in characterizing this promoter is to test whether <EM>Ptms</EM> works in <I>E. coli</I> DH10B strain and determine its relative promoter unit (RPU) compared to the standard constitutive promoter (a promoter whose activity is arbitrarily valued at 1.0 by partsregistry.org).</P><P><STRONG>Intended Result</STRONG>
  1.	<EM>Ptms</EM> should work in <I>E. coli</I>. This is supported by previous research (Moran et al., 1982). 
2.	The activity of <EM>Ptms</EM> should be relatively low.
</P><P><STRONG>Method</STRONG>
  Instead of using the absolute promoter activity as the final result, our characterization was based on obtaining the in vivo activity of this constitutive promoter. Adopting this method enables us to eliminate errors caused by different experimental conditions and give a more convincing result.
  By linking the promoter with GFP (BBa_E0240), the promoter activity was represented by the GFP synthesis rate which can be easily measured.<I> E. coli </I>carrying the right construct was then cultured to log phase. At a time point around the mid-log phase, the GFP intensity and OD595 values were measured to obtain the Relative Promoter Units (RPU).</P><P><STRONG>Characterization Procedure</STRONG></P><OL><LI>Constructing BBa_K733009-pSB3K3 (<EM>Ptms</EM>-BBa_E0240-pSB3K3); Transforming BBa_I20260-pSB3K3 (Standard Constitutive Promoter/Reference Promoter) from the 2012 Distribution Kit;</LI><LI>Preparing supplemented M9 medium (see below); </LI><LI>Culturing <I>E. coli</I> DH10B strain carrying BBa_K733009-pSB3K3 and <I>E. coli</I> carrying BBa_I20260-pSB3K3 in supplemented M9 medium and measuring the respective growth curves;</LI><LI>Measuring the GFP intensity and OD595 values every 15 minutes after the above mentioned<I> E. coli</I> strains are cultured to mid-log phase;</LI><LI>Calculating the Relative Promoter Units (RPU) using the obtained data;</LI><LI>Compiling the results.</LI></OL><P><STRONG>Data Processing</STRONG></P><OL><LI>After <I>E. coli</I> carrying the right construct was grown to mid-log phase, GFP intensity and OD595 were measured every 15 minutes (up to 60 mins);</LI><LI>For GFP intensity, curve reflecting GFP expression change was plotted; for OD595, average values were taken;</LI><LI>GFP synthesis rate was then obtained by calculating the slope of linear regression line of the above mentioned curve;</LI><LI>Absolute promoter activity of <EM>Ptms</EM> and BBa_I20260 were calculated by dividing the corresponding GFP synthesis rate over the average OD595 value;</LI><LI>Averaged absolute promoter activity was then obtained by averaging the respective sets of absolute promoter activity values;</LI><LI>Finally, R.P.U was calculated by dividing the averaged <EM>Ptms</EM> absolute promoter activity over the averaged BBa_I20260 absolute promoter activity.</LI></OL><P><STRONG>Result</STRONG></P><DIV style="margin-left: 25px">
  1.	Suggested by the GFP expression curve we plotted, <EM>Ptms</EM> functions in <I>E.coli </I>DH10B strain.
   <I>            * Above: GFP expression curve for one set of data</I>
2.	The RPU of <EM>Ptms</EM> obtained was 0.046497. This shows that <EM>Ptms</EM> has a very low promoter efficiency in <I>E.coli</I> DH10B strain.
</DIV><P><STRONG>Discussion</STRONG>
  Compared to BBa_I20260, it seems that <I>E. coli</I> carrying <EM>Ptms</EM>-GFP has a rather low GFP expression. This may cause some difficulties in deciding whether <EM>Ptms</EM> functions in <I>E. coli</I> or not. However, referring to the curve (for GFP Intensity), the GFP expression for <EM>Ptms</EM> increased gradually with respect to time. This suggests that <EM>Ptms</EM> functions in <I>E. coli</I> DH10B strain, although with a low efficiency. A possible reason for its low efficiency could be that <EM>Ptms</EM> was originally from <I>B. subtilis</I> but was functioning in a heterologous system when placed in <I>E. coli</I>. Due to time constraints, we were unable to characterize the promoter in <I>B. subtilis</I>, and we still hope that we can address this in the future.
</P><P><STRONG>Reference</STRONG>
 Moran, C., Lang, N., LeGrice, S., Lee, G., Stephens, M., Sonenshein, A., et al. (1982). Nucleotide sequences that signal the initiation of transcription and translation in <I>Bacillus subtilis</I>..<I>Molecular and General Genetics MGG,186,</I> 339-346.
</P><P><STRONG>Supplemented M9 Medium Composition</STRONG>
1.  5X M9 Salt Composition (1L) </P><DIV style="margin-left:25px">
(1) 64g Na<FONT size="1"><SMALL>2</SMALL></FONT>HPO<FONT size="1"><SMALL>4</SMALL></FONT>﹒7H<FONT size="1"><SMALL>2</SMALL></FONT>O
(2) 15g KH<FONT size="1"><SMALL>2</SMALL></FONT>PO<FONT size="1"><SMALL>4</SMALL></FONT>
(3) 2.5g NaCl
(4) 5.0g NH<FONT size="1"><SMALL>4</SMALL></FONT>CL</DIV>
2.   Minimal 1X M9 medium (1L) <DIV style="margin-left:25px">
(1) 200ml of 5X M9 Salt  
    (2) 2ml of 1M MgSO<FONT size="1"><SMALL>4</SMALL></FONT>
    (3) 100μl of 1M CaCl<FONT size="1"><SMALL>2</SMALL></FONT>
    (4) 5ml of 40% glycerol</DIV>
3.   Supplement (for the final medium) <DIV style="margin-left:25px">
(1) 1mM thiamine hydrochloride
    (2) 0.2% casamino acids
</DIV></DIV><DIV id="paragraph3" class="bodyParagraphs"><DIV align="right"><H1>Xylose Inducible Promoter</H1></DIV><P><STRONG>Background  Information <U><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Module/Regulation_and_control" target="_blank">(link to Regulation and Control Module)</A></U></STRONG>
 

  The reason for using the xylose inducible  promoter is to enable control on the expression of toxin and BMP2. Xylose is not toxic and normally not present in the human colon. This provides us an easy way to induce BMP2 expression without disrupting normal human body function.</P><P><STRONG>Objective</STRONG>
  On characterization, we want to test whether the promoter works in <I>E. coli</I> DH10B strain and if it works, what is the absolute promoter activity under varied experimental condition (i.e. xylose concentration).</P><P><STRONG>Intended Result</STRONG>
  
1.	Xylose inducible promoter is functional in <I>E. coli</I>.

2.	After the inducer concentration has reached a certain level, a relatively stationary GFP expression level (expression upper-limit) should be observed.

</P><P><STRONG>Method</STRONG>
  The absolute promoter activity was measured with respect to xylose concentration. 
The same reporter gene (BBa_E0240) was used to indicate promoter activity. <I>E. coli</I> carrying the right construct was cultured to log phase. Following the addition of xylose at various predetermined concentrations, at a time point around the mid-log phase, the GFP intensity and OD595 were measured for every 30 mins (up to 120 mins). Independent curves indicating the GFP intensity units (of various xylose concentrations) with respect to time were then plotted, following which the respective absolute promoter activities were calculated.
</P><P><STRONG>Characterization Procedure</STRONG></P><OL><LI>Constructing <EM>xylR-PxylA</EM>-BBa_E0240-pSB1A2</LI><LI>Preparing supplemented M9 medium (see below);</LI><LI>Culturing <I>E. coli</I> carrying <EM>xylR-PxylA</EM>-BBa_E0240-pSB1A2 and <I>E. coli</I> without constructs in supplemented M9 medium and measuring the growth curve respectively;</LI><LI>Culturing the above mentioned bacteria in supplemented M9 medium to log phase;</LI><LI>Adding xylose at different concentrations to different sets of bacterial culture;</LI><LI>Measuring the GFP intensity and OD595 values across time for every set of bacterial culture containing different xylose concentrations;</LI><LI>Plotting independent curves showing the GFP intensity units of various xylose concentrations with respect to time;</LI><LI>Plotting a graph to demonstrate the absolute promoter activity under different inducer concentrations;</LI><LI>Compiling the results.</LI></OL><P><STRONG>Data Processing</STRONG></P><OL><LI>After <I>E. coli</I> carrying the right construct was grown to mid-log phase, GFP intensity and OD595 were measured every 30 minutes (up to 120 mins);</LI><LI>For GFP intensity, curve reflecting GFP expression change was plotted; for OD595, average value was taken;</LI><LI>GFP synthesis rate was then obtained by calculating the slope of linear regression line of the above mentioned curve;</LI><LI>Absolute promoter activity for the promoter under different inducer concentrations were calculated by dividing the corresponding GFP synthesis rate over the average OD595 value;</LI><LI>Averaged absolute promoter activity was then obtained by averaging the respective sets of absolute promoter activity values.</LI></OL><P><STRONG>Result</STRONG></P><OL><LI>Shown in the figure below, with the addition of xylose, GFP expression increased. This tells us that the xylose inducible promoter is functional in <I>E. coli</I> DH10B strain.</LI><LI>When no xylose was added, a limited amount of GFP was expressed. This suggests that the xylose inducible promoter is to some extent leaky.</LI><LI>A relatively stationary GFP expression level was observed at xylose concentrations of 1% to 5%. Despite some other variables (see discussion for more details), the data suggests that the minimum inducer concentration for triggering a full induction should lie somewhere between 0% and 1%</LI><LI>For 10% inducer concentration, the GFP expression was relatively lower. There could be several reasons for this occurence, such as suboptimal growth conditions due to high osmotic pressure. (see discussion for more details)</LI></OL><P><STRONG>Discussion</STRONG></P><OL><LI>It is quite obvious that addition of xylose induces GFP expression in this construct. However, a slight issue remains: even when no xylose was added, a minute but detectable amount of GFP was still expressed. This shows that the xylose inducible promoter is leaky. It should be noteworthy that this version of the xylose inducible promoter has undergone mutagenesis on 3 different sites on the repressive gene for standardization purposes. Even so, the mutagenesis is done while preserving an identical codon translation. As such, similar to the <EM>Ptms</EM> promoter, activity of the <EM>xylR-PxylA</EM> promoter might be different in <I>E. coli</I> from that of <I>B. subtilis</I> due to expression in a heterologous system. Further characterization on this promoter in <I>B. subtilis</I> would be our future aim as time was limited.</LI><LI>Another interesting fact to note is that the <I>E. coli</I> strain used is one in which it's xylose metabolic operon remains active. As such, one might assume that the observed GFP expression upper-limit (&quot;stationary expression level&quot;),particularly at 1% or 2% is not the &quot;true&quot; upper-limit, since there should be underlying metabolism of xylose. In order to eliminate this possible error, higher concentrations of xylose was used and the promoter activity does not appear to vary greatly, suggesting that the stationary expression level reflects the maximum promoter activity. While this problem is solved, due to the relatively large concentration difference of inducer, it appears that the maximum promoter activity was achieved at 1% xylose, implying that we are unable to determine the exact minimum inducer concentration requirement for maximum activity. Based on the current result, it is safe for us to make a conjecture that the minimum concentration required lies between 0% and 1% xylose.</LI><LI>For the decreased GFP expression at 10% xylose concentration, one possible reason is that the high osmotic pressure caused by the xylose in the medium may inhibit the growth and metabolism of bacteria, thus reducing the bacterial population and/or its GFP expression. Another possible, but unlikely reason could be that the over-expression of induced GFP expression may disturb the normal bacterial function, leading to a low overall GFP expression.</LI></OL><P><STRONG>Supplemented M9 Medium Composition</STRONG>
1.  5X M9 Salt Composition (1L) </P><DIV style="margin-left:25px">
(1) 64g Na<FONT size="1"><SMALL>2</SMALL></FONT>HPO<FONT size="1"><SMALL>4</SMALL></FONT>﹒7H<FONT size="1"><SMALL>2</SMALL></FONT>O
(2) 15g KH<FONT size="1"><SMALL>2</SMALL></FONT>PO<FONT size="1"><SMALL>4</SMALL></FONT>
(3) 2.5g NaCl
(4) 5.0g NH<FONT size="1"><SMALL>4</SMALL></FONT>CL</DIV>
2.   Minimal 1X M9 medium (1L) <DIV style="margin-left:25px">
(1) 200ml of 5X M9 Salt  
    (2) 2ml of 1M MgSO<FONT size="1"><SMALL>4</SMALL></FONT>
    (3) 100μl of 1M CaCl<FONT size="1"><SMALL>2</SMALL></FONT>
    (4) 5ml of 40% glycerol</DIV>
3.   Supplement (for the final medium) <DIV style="margin-left:25px">
(1) 1mM thiamine hydrochloride
    (2) 0.2% casamino acids
</DIV></DIV><DIV id="paragraph4" class="bodyParagraphs"><DIV align="center"><H1>YdcDE Growth Inhibition Device</H1></DIV><P><STRONG>Background  Information <U><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Module/Regulation_and_control" target="_blank">(link to Regulation and Control Module)</A></U></STRONG>
The rationale for including this growth inhibition device is that over-dose of BMP2 can possibly cause unexpected proliferation of healthy colon epithelial cells (Zhang et al., 2012). Thus, a growth inhibition device is introduced and will be characterized.
</P><P><STRONG>Objective</STRONG>
The objective of this characterization is to determine the minimal concentration of xylose necessary to inhibit the growth of our B. hercules.
</P><P><STRONG>Method</STRONG><I>I. Construct </I></P><DIV style="margin-left: 30px"><EM>xylR</EM>: The transcriptional regulator for the xylose inducible promoter. <EM>PxylA</EM>: The xylose inducible promoter. <EM>ydcE (ndoA)</EM>: The toxin gene encoding EndoA. <EM>Ptms</EM>: The low efficiency constitutive promoter. <EM>ydcD (endB)</EM>: The antitoxin gene encoding YdcD. </DIV><I>II. Culture Medium </I>
Supplemented M9 minimal medium (M9 salt, 1 mM thiamine hydrochloride, 0.2% casamino acids, 0.1 M MgSO<FONT size="1"><SMALL>4</SMALL></FONT>, 0.5 M CaCl<FONT size="1"><SMALL>2</SMALL></FONT>, 0.4% glycerol) was used for our characterization. The reason for using this medium with 0.4% glycerol (instead of glucose) as a sole carbon source is that glucose can repress the induction of xylose (Kim, Mogk &amp; Schumann,. 1996). 25 mg/mL chloramphenicol stock was diluted 100 times and added to the medium to select for bacterial cells carrying our intended vectors. The final concentration gradient of xylose in the supplemented M9 minimal medium is: 0.05%, 0.10%, 0.15%, 0.20% and 0.25%.<I>III. Control and Experiment Group </I>
Control Group: <I>E. coli</I> DH10B without any transformed plasmid was used as the control. It was inoculated in the supplemented M9 minimal medium with xylose concentrations: 0.00%, 0.05%, 0.10%, 0.15%, 0.20%, 0.25%. Note that in the control group, the medium does not contain chloramphenicol.
Experiment Group: <I>E. coli</I> DH10B cells containing our cell growth inhibition device were inoculated in the supplemented M9 minimal medium with the following xylose concentrations: 0.00%, 0.05%, 0.10%, 0.15%, 0.20%, 0.25%. The total volume of the culture was 2mL for each test tube.<I>IV. Experiment </I>
The bacterial cultures were incubated at 37 degree Celsius, shaken at 200 rpm for exactly 16 hours. After the 16-hour incubation, the turbidity each culture tube was checked and photographed. Later on, 50uL of culture from one set of the experiments was aliquoted and spread on LB plate containing chloramphenicol (25 ug/mL) for overnight incubation. <P><STRONG>Result</STRONG></P><DIV style="margin-left" :40px="">
Note that after spreading the bacterial liquid cultures on the plates, bacterial growth was observed in all the tubes belonging to the experiment groups. However, there was distinguishable difference in the extent of bacterial growth between cultures given 0.00%, 0.05% and 0.10% of xylose and those given 0.15%, 0.20% and 0.25% of xylose. That is, in the left three tubes (see picture), the number of bacterial cells (as indicated by culture turbidity) was much greater than that in the right three ones. This result indicates that our<I> E. coli </I> had its growth inhibited but did not die from the xylose- induced toxin expression.</DIV><P><STRONG>Reference</STRONG>
Pellegrini O, Mathy N, Gogos A, Shapiro L, and Condon C. &quot;The<I> Bacillus subtilis </I>ydcDE operon encodes an endoribonuclease of the MazF/PemK family and its inhibitor..&quot; <I>Molecular microbiology.</I> 56.5 (2005): 1139-1148. Print.
Kim, L., Mogk, A., &amp; Schumann, W. (1996). A xylose-inducible <I>Bacillus subtilis</I> integration vector and its application.. <I>Gene, 181</I>(1-2), 71-76. 
Zhang J, Ge Y, Sun L, Cao J, Wu Q, Guo L, Wang Z. Effect of Bone Morphogenetic Protein-2 on Proliferation and Apoptosis of Gastric Cancer Cells.<I> Int J Med Sci </I>2012; 9(2):184-192.
</P></DIV><DIV id="paragraph5" class="bodyParagraphs"><DIV align="left"><H1>Colon Tumor Binding System</H1></DIV><P><STRONG>Abstract <U><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Module/Target_binding" target="_blank">(link to Target Binding Module)</A></U></STRONG>
A phage displaying peptide, RPMrel, was reported to have the ability to bind to poorly differentiated colon cancer cells while having significantly lower binding affinity to well-differentiated colon cancer cell and other normal tissues including lung, liver and stomach.  (Kelly &amp; Jones, 2003). Displaying RPMrel peptide on <EM>B. subtilis</EM> cell wall is expected to enable vegetative <EM>B. subtilis</EM> binding to colon cancer cell line specifically without adhering to other normal epithelial cell. In this characterization, <EM>B. subtilis</EM> was transfected with integration plasmid pDG1661-BBa_K733007 in order to display RPMrel on its cell wall. Empty vector pDG1661 is also transformed into <EM>B. subtilis</EM> as negative control. HT-29 (cancer adenocarcinoma) and HBE16 (Human Bronchial epithelial cell) are utilized for adherence test to compare the binding affinity of experiment and control <EM>B. subtilis</EM> on cancer cell. The binding specificity of RPMrel displaying <EM>Bacillus subtilis</EM> was also investigated in this characterization.</P><STRONG>Materials and Method</STRONG><LI><STRONG>Constructing part for <EM>B. subtilis</EM> characterization</STRONG>
In order to characterize biobrick BBa_K733007 and demonstrate that it functions as expected in <EM>Bacillus subtilis</EM>, the recombinant DNA constructed needs to be inserted into integration plasmid pDG1661. Therefore, K733007 was first inserted into pBluescript II KS+ through <EM>EcoRI</EM> and <EM>PstI</EM> site. The construct in pBluescript II KS+ was further digested with <EM>EcoRI</EM> and <EM>BamHI</EM> site in order to insert into the integration vector pDG1661. pDG1661-K733007 was transformed into <I>E. coli</I> DH10B and selected on Ampicillin plate (150 μg/ml). After replicating in <I>E. coli</I>, pDG1661-K733007 was extracted and transformed into <EM>B. subtilis</EM>. LB plate containing 5μg/ml Chloramphenicol is used to select transformed <EM>B. subtilis</EM>. All <I>E. coli</I> and <EM>B. subtilis</EM> transformation were performed strictly follow the protocol uploaded in the protocol section. </LI><LI><STRONG>Mammalian cell culture</STRONG>Colon adenocarcinoma (HT-29) was cultured in McCoy’s 5A medium supplemented with 10% FBS in 5% CO2, 37℃ incubator. Normal human bronchial epithelial cell (HBE16) was cultured in MEM medium supplemented with 10% FBS in 5% CO2, 37℃ incubator. 12-well plate was used to culture both cells to confluent. </LI><LI><STRONG><I>Bacillus subtilis</I> culture</STRONG><EM>Bacillus subtilis</EM> transformed with pDG1661-K733007 and the one transformed with pDG1661 empty vector were cultured separately in LB medium. Overnight cultures were diluted to OD650=0.1 and sub-cultured for another 3 hours until OD650 reached 1. Bacteria were washed in 0.1M PBS 3 times and then re-suspended in McCoy's 5A (without FBS) and MEM medium (without FBS) respectively.</LI><LI><STRONG>Co-culture <I>Bacillus subtilis</I> with mammalian cell: </STRONG>
Confluent mammalian cells were washed with 0.1M PBS once before adding bacteria. 1ml <EM>B. subtilis</EM> suspensions were added into each well of mammalian cell culture plate. After co-culturing the mammalian cell with bacteria, 5 times washing with 0.1M PBS was then performed to wash away the free unattached bacteria and proceed further in characterization.</LI><LI><STRONG>Gram staining for detecting the binding between <I>Bacillus subtilis</I> and colon cancer cell:</STRONG>
Cells were fixed with 1% paraformaldehyde for 15 minutes at room temperature. After fixation, 0.0007 % crystal violet was used to stain cells overnight. Stains were washed away the next day with water and observed under inverted microscope with 400X magnification.</LI><LI><STRONG>Adherence assay through CFU calculation</STRONG>
0.1% Triton X-100 in 0.1M PBS was added into the well after PBS washing and incubated for 10 minutes. Cells were then pelleted and re-suspended in LB medium, plating on LB plate to determine CFU. (Sheng et al, 2011)</LI><STRONG>Result</STRONG><P><LI><STRONG>Gram staining </STRONG>
Co-cultured <I>Bacillus subtilis</I> with RPMrel peptide on HT-29 and controlled <I>Bacillus subtilis</I> on HT-29 were stained and observed under inverted microscope with 400X magnification. As shown in Figure 1, <I>Bacillus subtilis</I> and nucleus of HT-29 cell were stained purple and both types of <I>B. subtilis</I> can be detected on HT -29 cell.</LI></P><P style="clear:both">
Figure 1: Gram stain of <I>B. subtilis</I> on confluent HT-29 cell.
A: Gram stain of HT-29 co-cultured with <I>B. subtilis</I> with RPMrel peptide.
B: Gram stain of HT-29 co-cultured with control bacteria, <I>B. subtilis</I> without RPMrel peptide.</P><P><LI><STRONG>CFU calculation </STRONG>
Serial dilutions were performed before plating and plates with colonies between 25~300 were counted to calculate the CFU/ml. As shown in Table 1 and Figure 1, more <I>Bacillus subtilis</I> with empty vector pDG1661 was detected on HT-29 cell line than the one of <I>B. subtilis</I> with RPMrel peptide on HT-29. Comparing the number of <I>Bacillus subtilis</I> with RPMrel retained on HT-29 and HBE16, more <I>Bacillus subtilis</I> can be detected on HBE16 cell. Two rounds of T-tests were further performed and it showed that no significant differences in the binding affinity between <I>Bacillus subtilis</I> with or without RPMrel to HT-29 cell (P&gt;0.05) but the number of <I>Bacillus subtilis</I> with RPMrel peptide binding to HBE16 cells is significantly higher than the same bacteria bind to HT-29.</LI></P>

Table 1: Number of <I>Bacillus subtilis</I> retained on different mammalian cell line.
Figure 2: Number of Bacillus subtilis retained on different mammalian cell line<P><STRONG>Discussion</STRONG></P><P><LI><STRONG>Experiment results to some extent reject our hypothesis but no solid conclusion can be made at this stage:</STRONG></LI></P><P>Based on the results from Table 1 and Figure 2 and the two t-tests performed, it can be stated that <I>B. subtilis</I> with RPMrel displaying have no significant increase in binding ability to cancer cell line and the peptide displaying results in significant improvement in binding to normal epithelial tissue. </P><P>However, the statement above is just based on the incomplete evidence we have currently. Because of time limitations, we haven’t demonstrated that LytC system can facilitate the display of RPMrel on the cell wall successfully. Therefore, we can hardly tell whether the unexpected result is caused by the unexpected structural changes of RPMrel when displaying on cell wall or because of the failure of the LytC system in transporting and localizing RPMrel on the cell wall.</P><P>In addition, the Gram staining method used in our characterization is not consistent during our one-month characterization work. Over staining happened from time to time and the cell layers were easily detached from the bottom of the well after adding crystal violet.</P><P>Besides, more proper control group is needed for our experiment. Because of the limitation of resources and time we had, no normal epithelial cell line from the digestive system can be obtained and used in our work. Substituting with HBE16 cells in the experiment is the best compromise we have so far. If possible, we will try to utilize other cell lines in our experiment in order to draw solid conclusions in the future. </P><P><LI><STRONG>Future work</STRONG></LI></P><OL><LI>Using BBa_K733008 (LytC system with FLAG-tag on its C terminus) to verify the proper functioning of the cell wall binding system.</LI><LI>Developing some more reliable staining method to demonstrate the adherence between <I>B. subtilis</I> and mammalian cell.</LI><LI>Proper control bacteria and proper control cell line need to be added in our characterization in order to obtain reliable and solid conclusions. </LI></OL><P><STRONG>Reference</STRONG></P><P>Haiqing Sheng, Wang Jing, Lim Ji Youn, Davitt Christine, Minnich Scott A. &amp; Hovde Carolyn J. 2012. Internalization of <I>Escherichia coli</I> O157:H7 by bovine rectal epithelial cells. <I>Frontiers of Microbiology.</I> (2012.2.32)</P><P>Kimberly A. Kelly &amp; Jones David A.2003. Isolation of a Colon Tumor Specific Binding Peptide Using Phage Display Selection. <I>Neoplasia.</I> 5: 437 – 444</P></DIV><DIV id="paragraph6" class="bodyParagraphs"><DIV align="left"><H1>Anti-tumor Molecule Secretion</H1></DIV><STRONG>Abstract <U><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Module/Anti_tumor" target="_blank">(link to Anti-tumor Molecule Secretion Module)</A></U></STRONG>
  This characterization is intended to  demonstrate that <EM>B. subtilis </EM>transformed  with BMP2 expression and secretion construct can trigger the apoptosis of colon  cancer. In this characterization, <EM>B.  subtilis</EM> was co-cultured with HT-29 cell. MTT assay which is widely used as a cell proliferation and viability assay was performed in our experiment to investigate  the growth suppression effect on colon cancer cell.<P><STRONG>Material  and Method</STRONG></P><UL><LI><STRONG>Constructing part for <EM>B. subtilis</EM> characterization  </STRONG></LI></UL><P align="left">In order to characterize this  construct in <EM>B. subtilis</EM>, the K733017 was inserted into integration vector  pDG1661. Detailed methods can be referred to the characterization of  BBa_K733007. </P><UL><LI><STRONG>Mammalian cell culture</STRONG></LI></UL><P>Colon adenocarcinoma (HT-29) was cultured in McCoy’s 5A medium  supplement with 10% FBS in 5% CO2, 37℃ incubator. They were seeded in 96-well plates for further characterization.</P><UL><LI><STRONG><EM>Bacillus  subtilis</EM></STRONG><STRONG> culture </STRONG></LI></UL><P><EM>Bacillus subtilis</EM> transformed with pDG1661-K733017 and another  transformed with pDG1661 empty vector were cultured separately in LB medium.  Overnight cultures were diluted to OD650=0.1 and subculture to OD650  reaching 1.0.<EM> B. subtilis</EM> were washed  three times with 0.1M PBS and diluted to OD=0.1, 0.01, 0.001 and 0.0001 in  McCoy’s 5A medium with 10% FBS supplementation. </P><UL><LI><STRONG>Co-culture <EM>Bacillus subtilis</EM> with mammalian cell </STRONG></LI></UL><P>3000 HT-29 cells in 100μl were seeded in 96-well plates. After overnight incubation, 100μl <EM>B. subtilis</EM> suspension is added into each well and co-cultured with  HT-29 cell for 48 hours. </P><UL><LI><STRONG>MTT assay </STRONG></LI></UL><P>Cells were washed 5 times with 0.1M PBS. 0.5% MTT solution was  added and incubated at 37℃ for 4 hours. After incubation, 100ul DMSO was  added into each well, shaking for 5 minutes in order to obtain a homogenized  solution. A570 was measured then to reflect the  viability of cells. Percentage of viability was calculated through the equation <STRONG>Result </STRONG>
  A declining trend of cell  proliferation can be clearly observed when more bacteria were co-cultured with HT-29  cell as shown in Figure 1. Comparing the proliferation rate of HT-29 cells which  were co-cultured with BMP2 producing<EM> B.  subtilis</EM> and non-BMP2 producing <EM>B.  subtilis</EM>, a significant decline can be detected when initial OD650 of <EM>B. subtilis </EM>equaled to 0.1 and 0.01.  (P&lt;0.05 ). No significant difference can be detected when OD650 decreased to  0.001. When the OD650 reaches 0.0001, significant increase in cell  proliferation rate can be detected in the experiment group. </P><P>Figure 1: The viability of HT-29 cell after co-cultured with <I>B. subtilis</I></P><P style="clear:both"><STRONG>Discussion </STRONG>
In this characterization, <EM>Bacillus subtilis</EM> transformed with plasmid pDG161-K733017 exert  significant growth inhibition effect when OD650 is above 0.01. However, no extra supporting experiments like Western Blot has been successfully carried out to  confirm the expression of BMP2 in <EM>B.  subtilis</EM>. Therefore, more experiments need to be done in order to fully  characterize this BioBrick.</P></DIV><DIV id="Sitemap"><DIV id="Sitemap_Home" align="center"><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong"><B>Home</B></A></P></DIV><DIV class="Sitemap_Content"><LI><P><B>Team</B></P></LI><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Introduction">Introduction</A></P></LI><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Supervisor">Supervisor</A></P></LI><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Instructor">Instructor</A></P></LI><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Members">Members</A></P></LI><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Advisors">Advisors</A></P></LI></DIV><DIV class="Sitemap_Content"><LI><P><B>Project</B></P></LI><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Project_Abstraction">Abstract</A></P></LI><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Background_and_Motive">Motive</A></P></LI><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Design_Overview">Design - Overview</A></P></LI><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Design_Module">Design - Module</A></P></LI><P>-- <A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Module/Target_binding">Target Binding Module</A></P><P>-- <A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Module/Anti_tumor">Anti-tumor Molecule Secretion Module</A></P><P>-- <A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Module/Regulation_and_control">Regulation and Control Module</A></P><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Design_Chassis">Design - Chassis</A></P></LI></DIV><DIV class="Sitemap_Content"><LI><P><B>Wet Lab</B></P></LI><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Parts_and_Device">Parts and Devices</A></P></LI><P>-- <A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Parts_and_Device">Overview</A></P><P>-- <A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Construction">Construction</A></P><P>-- <A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Assembly">Assembly</A></P><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Notebook">Notebook</A></P></LI><P>-- <A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Notebook/Logbook">Logbook</A></P><P>-- <A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Notebook/Protocol">Protocol</A></P><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Characterization">Characterization</A></P></LI><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Achievement">Achievement</A></P></LI><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Future_Work">Future Work</A></P></LI></DIV><DIV class="Sitemap_Content"><LI><P><B>Human Practice</B></P></LI><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Overview">Overview</A></P></LI><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Interview">Interview</A></P></LI><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Presentation">Presentation</A></P></LI><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Calendar">Calendar</A></P></LI></DIV><DIV class="Sitemap_Content"><LI><P><B>Extras</B></P></LI><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Medal_Requirements">Medal Requirements</A></P></LI><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Safety">Safety</A></P></LI><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Attribution">Attribution</A></P></LI><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Acknowledgement">Acknowledgement</A></P></LI><LI><P><A href="https://2012.igem.org/Team:HKUST-Hong_Kong/Glossary">Glossary</A></P></LI></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:HKUST-Hong_Kong/Characterization">http://2012.igem.org/Team:HKUST-Hong_Kong/Characterization</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:HKUST-Hong_Kong/Characterization" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:HKUST-Hong_Kong/Characterization" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:HKUST-Hong_Kong/Characterization&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:HKUST-Hong_Kong/Characterization&amp;oldid=227613" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>