"
Page
Discussion
View source
History
teams
Log in
 
Team:HKUST-Hong Kong/Module/Target binding
From 2012.igem.org
Team:HKUST-Hong Kong - 2012.igem.org
TEAM
PROJECT
WET LAB
HUMAN PRACTICE
EXTRAS
Introduction
Supervisor
Instructor
Members
Advisors
Abstract
Motive
Design - Overview
Design - Module
Design - Chassis
Parts and Devices
Notebook
Characterization
Achievement
Future Work
Overview
Interview
Presentation
Calendar
Medal Requirements
Safety
Attribution
Acknowledgement
Glossary
Target Binding Module
<<< Back to Modules
Overview
Our decision to pursue colorectal carcinoma suppression arose from two key points obtained from preliminary  research: 1) bone morphogenetic protein 2 (BMP2) suppresses the growth of colon cancer cell growth in vivo, and 2) the phage display peptide  RPMrel confers specific and preferential binding to non-differentiated colon cancer cells.
With these two pieces of knowledge we have respectively: 1) our carcinoma suppression drug, and 2) a tool for  specifically targeting cancerous cells. Thus the objective of this module was to identify and then construct a suitable mechanism making use of the RPMrel peptide to target the delivery of BMP2.
Design
Considering limitations.
Design of a solution starts with considering existing limitations. Since this is iGEM, the clearest limitation was that the solution must be a biological one and thus must involve a living component. Only a certain set of living organisms lie within our reasonable capacity to engineer them, and of these we decided on Bacillus subtilis (see Chassis page).
We then examined the treatment environment. Carcinomas of the colon protrude into the digestive tract and items within the tract can interact with them directly. We decided on the concept that our biological system would be ingested, then produce and release the drug in the vicinity of the tumor for direct action.
As a ligand in the Transforming Growth Factor β (TGF-β) signalling pathway, BMP2 had to be expressed in mature form to have any effect. It was thus decided that it would be synthesized within our chassis and then released into the external environment by secretion. Since the polypeptide sequence and conformation of BMP2 must be preserved, it was decided to bring the drug to the vicinity of the carcinoma by conferring the binding ability of RPMrel to the chassis as a whole.
To do this, RPMrel had to be expressed on the cell surface in a functional form. We conducted research into several methods to do this on B. subtilis and concluded that cell wall expression of the peptide was ideal. Imperial College London’s 2010 team had performed that same task using the cell wall binding domain of the hydrolase lytC as their peptide anchor and a helical linker of their design. We decided to employ that same system for surface expression of RPMrel.
RPMrel, and colon tumor specific binding.
Using phage display to compile peptide libraries that confer specific binding to certain antigens is a now common way to come up with useful peptides. RPMrel, a 9 amino acid disulfide-constrained peptide, was screened out of the New England Biolabs Ph.D.-C7C library for positive binding  to poorly differentiated HT-29 cells, and negative binding to well  differentiated HCT 116 cells. All peptides in the Ph.D.-C7C library have random sequences of 7 amino acids bounded by cystines at the N- and C- terminals.  Further screening of the peptides was done by performing 6 successive  incubation-wash-elution cycles against HT-29. See Kelly & Jones  (2003) . ‘RPM’ - for an arginine-proline-methionine amino acid sequence immediately  before the C-terminal cystine - emerged as a consensus motif for late selection of high-affinity peptides, thus giving the peptide’s name. RPMrel’s full amino  acid sequence is n-CPIEDRPMC-c.
The binding properties of RPMrel  were identified during Kelly & Jones’ study when it was fused to the  surface-exposed p3 minor coat protein of the bacteriophage M13KE. This module  will lead to its novel fusion to the cell wall binding domain of lytC, exposing  it to the extracellular environment.
LytC, and its cell wall binding domain.
LytC, a cell surface hydrolase, is native to B. subtilis and binds non-covalently to its cell wall  interacting with it electrostatically. This property was previously determined  to make it superior for exposure of bound peptides. Furthermore, as compared to  other surface expression methods we investigated (including peptide expression  on an engineered S-layer), the LytC model was much better studied.
According to the study conducted by Yamamoto et al (2003) LytC is localized uniformly on B. subtilis cells grown past log phase, making it more ideal than the more specifically localized LytE and LytF for expression of RPMrel. It was further found that - compared to the others - LytC  was particularly resistant to degradation by the cell surface protease WprA and extracellular protease Epr, both products of B. subtilis . In this study, 3xFLAG (a standard peptide epitope designed by Sigma-Aldrich Corp.) was successfully fused to the protein via a short linker and was successfully exposed to specific antibodies.
The full sequence of the lytC gene is 1488bp, but its cell wall binding domain was isolated by Imperial College London’s 2010 team as the region encoded by the first 954bp. This means the natural function of LytC - cell wall turnover and autolysis for cell growth and separation - is removed from the recombinant protein we will use.
References
Kelly, Kimberly A., and David A. Jones. "Isolation of a Colon Tumor Specific Binding Peptide Using Phage Display Selection." Neoplasia 5.5 (2003): 437-444. Print.
Yamamoto, Hiroki, Shin-ichirou Kurosawa, and Junichi Sekiguchi. "Localization of the Vegetative Cell Wall Hydrolases LytC, LytE, and LytF on the Bacillus subtilis Cell Surface and Stability of These Enzymes to Cell Wall-Bound or Extracellular Proteases." Journal of Bacteriology 185.22 (2003): 6666-6677. Print.
Anti-tumor Molecule Secretion Module
Regulation and Control Module
Home
Team
Introduction
Supervisor
Instructor
Members
Advisors
Project
Abstract
Motive
Design - Overview
Design - Module
-- Target Binding Module
-- Anti-tumor Molecule Secretion Module
-- Regulation and Control Module
Design - Chassis
Wet Lab
Parts and Devices
-- Overview
-- Construction
-- Assembly
Notebook
-- Logbook
-- Protocol
Characterization
Achievement
Future Work
Human Practice
Overview
Interview
Presentation
Calendar
Extras
Medal Requirements
Safety
Attribution
Acknowledgement
Glossary
p>
Retrieved from " http://2012.igem.org/Team:HKUST-Hong_Kong/Module/Target_binding "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
