"
Page
Discussion
View source
History
teams
Log in
 
Team:HKUST-Hong Kong/Construction
From 2012.igem.org
Team:HKUST-Hong Kong - 2012.igem.org
TEAM
PROJECT
WET LAB
HUMAN PRACTICE
EXTRAS
Introduction
Supervisor
Instructor
Members
Advisors
Abstract
Motive
Design - Overview
Design - Module
Design - Chassis
Parts and Devices
Notebook
Characterization
Achievement
Future Work
Overview
Interview
Presentation
Calendar
Medal Requirements
Safety
Attribution
Acknowledgement
Glossary
Construction
<<< Back to Parts and Devices
Three modules in our project are constructed separately and later assembled together to function as a whole in our project.
Target Binding
In order to facilitate targeting of colon tumor cells, the colon tumor recognition peptide RPMrel is displayed on the Bacillus subtilis cell wall via the LytC cell wall displaying system. lytC gene encodes N-acetylmuramoyl-L-alanine amidase, a type of autolysin on cell wall. It contains a 24 amino acid signaling peptide and three repetitive cell wall binding regions on its N-terminus all of which facilitate the translocation and insertion of LytC to cell wall. The catalytic domain of LytC is located at the C-terminus. In our project, we only introduce the first 318 amino acid of LytC in our construct to signal the secretion and retention in cell wall. A helical linker with three repetitive sequence - (EAAAK)3 - is fused to the C-terminus of LytC, separating cell wall binding domain and RPMrel peptide in order to maintain the structure and function of both parts. RPMrel peptide is then fused to the C-terminus of the linker, hanging on cell wall, pointing to extracellular environment, to recognize colon tumor.
The expression of this fusion protein is driven by a high efficiency constitutive promoter Pveg. An endogenous ribosome binding site spoVG is linked to the downstream of Pveg promoter, stimulating translation.
Two parts were built for this module. One part ( BBa_K733007 ) has RPMrel is fused to the LytC cell wall binding system. Another part ( BBa_K733008 ) has FLAG™ fused to LytC for confirmation of the correct expression and translocation of LytC cell wall binding system on cell wall.
To build our parts, we utilized the BioBrick BBa_K316037 built by Imperial College London's iGEM 2010 team. Fusion protein was obtained by adding RPMrel or FLAG™ tag coding sequence to the C-terminus of helical linker through PCR.
Anti-tumor Molecule Secretion
In this module, two main parts with 5 intermediate parts were constructed. To synthesize and secrete active BMP2 from the cytosol to the extracellular environment, mature BMP2 is expressed with a secretory signal peptide fused to its N-terminus. Composed of a positively charged lysine or arginine residue at its N-terminus, a hydrophobic H domain and type I signal peptide peptidase (SPPase) cleavage site A-X-A at its C-terminus, sec-type signaling peptides can carry BMP2 to the cytoplasmic membrane and cleaved at A-X-A site, releasing BMP2 to the extracellular environment. While BMP2 is secreted out and folded to its functional form, the retained signaling peptide in the cytoplasmic membrane will be degraded by SPPase.
Since the secretion efficiency and cleavage accuracy for secreted BMP2 in prokaryotic system has never been investigated, we built two constructs with BMP2 fused with two different signal peotide, YbdN and YdjM respectively based on previous studies in signalling peptides in B. subtilis.
Considering the lack of proteases in B. subtilis for modifying proteins BMP2 after translation, we directly amplified the DNA sequence encoding mature BMP2 from the mouse genome. We further fused a signaling peptide to its N-terminus and recombined the fusion protein product for expression under the constitutive promoter Pveg and a strong ribosome binding site for further characterization.
In addition, due to the existence of EcoRI cutting site in BMP2 coding region, a point mutation is done to eliminate this illegal cutting site without changing the amino acid sequence.
Regulation and Control
To regulate the timing of BMP2 expression and the dosage of BMP2, two regulatory systems are built in this module. 9 BioBricks were constructed to meet this need or to further engage in characterization.
Ptms promoter, ( BBa_K733001 ) which is used to drive the expression of antitoxin, YdcD, was constructed through Gibson assembly. With a very low transcription efficiency, the low amount of antitoxin accumulated in bacteria can provide a threshold supporting a certain amount of BMP2 expression. Meanwhile over-induction from xylose can also lead to the inhibition of growth when the toxin outrange the antitoxin.
Xylose inducible promoter, ( BBa_K733002 ) which was used to control the timing of BMP2 synthesis was obtained from the integration plasmid ‘pAX01’ provided by the Bacillus Genetic Stock Center. Two XbaI and one EcoRI cutting site in the sequence had to be mutated in order to convert the part into a BioBrick. Antitoxin ( BBa_K733003 ) and toxin ( BBa_K733004 ) coding gene used in the cell growth inhibition device were both obtained from the B. subtilis genome through PCR.
In order to build the cell growth inhibition device ( BBa_K733012 ), toxin ( BBa_K733004 ) is ligated after xylose inducible promoter ( BBa_K733002 ) to generate BioBrick ( BBa_K733011 ) while antitoxin ( BBa_K733003 ) is linked to the downstream of Ptms promoter ( BBa_K733001 ) to obtain BBa_K733010 . Finally, BBa_K733011 and BBa_K733010 were assembled together to generate the cell growth inhibition device that was taken forward for further characterization.
While characterizing the cell inhibition device as a whole, we further investigated the transcription efficiency of constitutive promoter Ptms and xylose inducible promoter through the construction of BBa_K733009 and BBa_K733018 . BioBrick BBa_E0240 , which includes the B. subtilis consensus ribosome binding site, a GFP reporter gene and double terminator, is ligated to the downstream of Ptms and xylose inducible promoter respectively.
Although BioBricks for each module were built separately, they will be assembled together in an integration plasmid pDG1661 to function together in our B. hercules. For more information on our detailed assembly method, please go to Assembly .
Home
Team
Introduction
Supervisor
Instructor
Members
Advisors
Project
Abstract
Motive
Design - Overview
Design - Module
-- Target Binding Module
-- Anti-tumor Molecule Secretion Module
-- Regulation and Control Module
Design - Chassis
Wet Lab
Parts and Devices
-- Overview
-- Construction
-- Assembly
Notebook
-- Logbook
-- Protocol
Characterization
Achievement
Future Work
Human Practice
Overview
Interview
Presentation
Calendar
Extras
Medal Requirements
Safety
Attribution
Acknowledgement
Glossary
Retrieved from " http://2012.igem.org/Team:HKUST-Hong_Kong/Construction "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
