Team:Stanford-Brown/SB16 BioMembrane AptamerPurification
Stanford-Brown 2016
Toggle navigation
Stanford-Brown
Home
Attributions
BioMembrane
Overview
Collagen
Elastin
p-Aramid
Latex
UV Protection
Float
Gas production
BioSensor
Overview
Chromoproteins
Fluorophore-Quencher
Aptamer purification
Practices
Overview
Integrated Human Practices
Outreach
Life Beyond the Lab
Collaborations
InterLab Study
Team Collaborations
BioBricks
Protocols
Protocols
Software Design
Modeling
Notebooks
Collagen & Elastin
P-aramid
Latex
UV protection
Aptamer Purification
Gas Production
Chromoproteins
Fluorophore-Quencher
Nylon
Interlab Study
Team
Requirements
Medals
Safety
Team member Julia introduces the Aptamer Purification components of the project
Introduction
An aptamer is a nucleic acid molecule that binds with  high specificity and affinity to a particular target. Aptamers have been likened to antibodies in their “lock-and-key” binding specificity. Currently there are hundreds of documented aptamers, some of which can be accessed and purchased through the Aptagen database [1]. As an example, one very well characterized example of an aptamer is the theophylline aptamer (pictured below, courtesy of the Aptagen website ).
Figure 1: Folded Theophylline RNA aptamer. This picture is from the Aptagen Database.
In our biosensing projects we utilized the fact that these aptamers tend to dramatically alter their 3-D shape upon binding their targets. In our more experimentally successful Fluorophore-Quencher system this conformational change displaced a quencher from a synthetically constructed fluorescent aptamer reporter -- read about that project here. In our secondary biosensing system, the conformational change restored the integrity of an intentionally compromised Internal Ribosome Entry Site (IRES), allowing for restored translation of a reporter gene -- read about that project here.
In addition to our biosensing projects using aptamers, we realized that aptamers could have applications in biomanufacturing. Specifically, if we could generate a novel aptamer against pABA (the constituent monomer in p-aramid), we could biotinylate it, attach it to a solid streptavidin surface, and use the resulting  surface to concentrate the pABA out of cell lysate for further processing. To that end we set out to create an aptamer against pABA. To read about our team's efforts to over express pABA and its relevance to Kevlar, see here.
The main problem was pABA's size. The normal process by which aptamers are generated is called Systematic Evolution of Ligands by Exponential Enrichment or SELEX. In it the binding target is immobilized to a surface, a large library of about 1015 or more unique candidate aptamer oligos are flowed over it, and motifs that bind are iteratively selected and enriched [2]. From there directed mutagenesis and characterization of the most represented motifs can be done if desired to further enhance both the specificity and the affinity of the isolated aptamer. This generic SELEX process works well for targets that can be immobilized easily. pABA is not such a target. Even by comparison to other small molecules that have published aptamers, pABA is extremely small, and thus not readily able to be adhered to a solid surface for selection.
Accordingly, we searched literature for alternate selection techniques, and settled on one called capture SELEX. We picked capture SELEX because it  has been used successfully by other groups to isolate aptamers against very small targets: nucleic acids, other small biological substrates [3] and even perchlorate [4]. The difference between capture SELEX and normal SELEX is that the former uses a candidate aptamer library with a specific constant region in the middle, so that the candidates themselves can be attached to streptavidin beads via a biotinylated capture oligo. In this setup, free target is added to the bound oligos, and whatever oligos elute off the column to bind the target are PCR amplified, PAGE purified, and iteratively enriched. To generate our candidate motifs, the library was sequenced after round 10. Although the timespan of this project exceeded the time available in the summer, in the next section we've published our detailed experimental protocol for this procedure in hopes that future iGEM teams can build on this research.
References:
1) Aptamer-Index Aptamer Database by Aptagen LLC. (2016). Aptagen.com. Retrieved 7 October 2016, from http://www.aptagen.com/aptamer-index/aptamer-list.aspx
2) Lang, T. (2012). In Vitro Selection of Aptamers for Perchlorate and Melamine. (Ph.D). University of Illinois at Urbana-Champaign.
3) Famulok, M. (1999). Oligonucleotide aptamers that recognize small molecules. Current Opinion In Structural Biology, 9(3), 324-329. http://dx.doi.org/10.1016/s0959-440x(99)80043-8
4) Stoltenburg, R., Nikolaus, N., & Strehlitz, B. (2012). Capture-SELEX: Selection of DNA Aptamers for Aminoglycoside Antibiotics. Journal Of Analytical Methods In Chemistry, 2012, 1-14. http://dx.doi.org/10.1155/2012/415697
5) Gu, H., Duan, N., Wu, S., Hao, L., Xia, Y., Ma, X., & Wang, Z. (2016). Graphene oxide-assisted non-immobilized SELEX of okdaic acid aptamer and the analytical application of aptasensor. Sci. Rep., 6, 21665. http://dx.doi.org/10.1038/srep21665
Capture SELEX Protocol
Figure 2: This is a graphical overview of the process of Capture SELEX. In brief the candidate aptamers from the random library are attached to beads via their conserved internal domain. Then the target small molecule ligand is introduced with enough time to allow any oligos that bind to dissociate from the beads and bind the ligand. Sequences that come off the beads are iteratively enriched and re-selected for a number of rounds to generate a pool of sequences that express demonstrated binding affinity to the target small molecule.
Acknowledgements and Further Resources:
Our team is immensely thankful to Dr. Mark Ditzler, and Stanford Professor Christina Smolke for giving us feedback on our proposed SELEX protocols. Furthermore, there has been very valuable previous iGEM work done on aptamers that we benefited from greatly when considering what aptamers could do and designing our experiments. Notably, in 2015 the team  at Heidelberg University began developing software that can generate novel aptamer sequences in silico (MAWS). If fully realized, this software would be a huge advance in reducing the experimental time and effort associated with developing new aptamers. In addition, the work done by the 2014 Team at the Technical University of Denmark around using Spinach aptamers as a way to measure promoter efficiency was of great use in helping us find and explore relevant atpamer literature. Another innovative iGEM use of aptamer technology was the 2012 Zhe Jiang University team's development of an aptamer based RNA scaffold to help localize products together to increase the efficiency of engineered biosynthetic pathways. Finally, iGEM has a small section of parts relating to DNA aptamers , and we looked to many of them for inspiration as well.
Pyrroloquinoline quinone (PQQ) Purification
The second section of this project was aimed at using just the capturing properties of aptamers, independent of sensing applications, to concentrate trace amounts of desired small molecules out of dilute solutions. Here we built on both previous work [1] and our own observations from the FQ project that immobilized aptamers appear to retain functionality when bound to solid surfaces. Accordingly we used streptavidin magnetic beads to bind two candidate biotinylated aptamers which captured PQQ from both pre-purified (HPLC) and a comparable concentration from the supernatant of an over-the-counter supplement. We used both racemic lysine and peptidyl lysine (in the form of a nickel column purified collagen construct) with copper(II) sulfate and demonstrated activity using an Amplex® Red Hydrogen Peroxide/Peroxidase Assay Kit. Importantly, the bound PQQ retained its catalytic activity, with comparable results seen for HPLC-purified and supplement-sourced PQQ. This makes the beads solid surfaces functionalized by addition of DNAzymes, which can be used to purify their own cofactors in functional amounts (none of the original PQQ solution was used for the H2O2 assay). See the chart below for positive results. At the time of the assay, we were working with limited resources; with more time I would certainly like to repeat the analysis and use a rigorous set of controls, but this is at least a very positive indication that this concept merits further study and this application in particular could have interesting extensions for elastin or collagen nanostructuring.
Conditions
Assay Readout (average from 30 to 60 min post Amplex® Red addition)
Corresponding [H2O2] (μM)
Aptamer 1, racemic lysine, pure PQQ
2966.75
0.087
Aptamer 1, racemic lysine, supplement PQQ
2967.25
0.087
Aptamer 2, racemic lysine, pure PQQ
3110.25
0.092
Aptamer 2, racemic lysine, supplemental PQQ
2606.5
0.077
Aptamer 1, peptidyl lysine, pure PQQ
1117.75
0.033
Aptamer 1, peptidyl lysine, supplemental PQQ
1503.00
0.044
Aptamer 2, peptidyl lysine, pure PQQ
1320.25
0.039
Aptamer 2, peptidyl lysine, supplemental PQQ
1222.75
0.036
Figure 3: Illustration of what we're trying to accomplish: concentrating particular products of interest out of dilute solutions using the binding specificity of aptamers.
Initial binding
Figure 4: Pictures of initial and final binding, showing the PQQ aptamer binding its target. To see the full procedure, see our Protocols. .
Figure 5: Standard Curve depicting the hydrogen peroxide concentration dependent RFU.
References:
1) Emahi, I., Mulvihill, I., & Baum, D. (2015). Pyrroloquinoline quinone maintains redox activity when bound to a DNA aptamer. RSC Adv., 5(10), 7450-7453. http://dx.doi.org/10.1039/c4ra11052h
