Team:Stanford-Brown/SB16 BioMembrane Elastin
Stanford-Brown 2016
Toggle navigation
Stanford-Brown
Home
Attributions
BioMembrane
Overview
Collagen
Elastin
p-Aramid
Latex
UV Protection
Float
Gas production
BioSensor
Overview
Chromoproteins
Fluorophore-Quencher
Aptamer Purification
Practices
Overview
Integrated Human Practices
Outreach
Life Beyond the Lab
Collaborations
InterLab Study
Team Collaborations
BioBricks
Protocols
Protocols
Software Design
Modeling
Notebooks
Collagen & Elastin
P-aramid
Latex
UV protection
Aptamer Purification
Gas Production
Chromoproteins
Fluorophore-Quencher
Nylon
Interlab Study
Team
Requirements
Medals
Safety
BioMembranes team member Charlie introduces the collagen and elastin subprojects
Making an Elastin Mimetic
Elastin fibers are insoluble even in hot, basic solution, but they are made up of identical, soluble tropoelastin monomers. These monomers are defined by a pentapeptide repetitive motive, Val-Pro-Gly-X-Gly, which is responsible for many of elastin’s properties, in addition to lysine-based cross-linking domains. The intrinsically random and modular structure of tropoelastin makes it suitable for production via circular mRNA. While most mRNA represents the entire sequence of a protein and proceeds linearly from start codon to stop codon, this is not an absolute requirement for translation. Work by the 2014 and 2015 Gifu IGEM teams showed the efficacy of an mRNA circularization device, which could be used to produce translated protein product when a complete coding sequence was enclosed. In addition, they demonstrated that without a stop codon translation would occur for multiple rounds of the same coding sequence with eventual random termination of translation. This process was adapted to produce long tropoelastin-like monomers using a subset of human tropoelastin exons representing alternating cross-linking and coacervation domains, replicating many of the properties which make tropoelastin both interesting and useful.
Figure 1: Elastin extracted from bovine aorta
Circular mRNA
Where does circular mRNA come from?
While most natural examples of circular mRNA formation do not appear to function as infinite translation templates, circularization machinery has been adapted from an intron splicing system from the thymidilate synthase gene in bacteriophage T4 to produce both long, repetitive proteins1 and circularized aptamers more stable to exonucleases2. The circular RNA produced by Perriman and Ares was 795 bp in length,1, so the 750 bp construct we aim to synthesize should be well within the realm of possibility. The mechanism is described in detail by the Gifu team , but essentially the native functionality depends only on attack of free GTP to remove the intron by circularization and subsequent degradation; this process is shown in Figure 2.
Figure 2: Native functioning of the intron results in a desired linear exon product and a circular intron which is eventually degraded
Figure 3: When the ends of the original intron are rearranged around a new gene, the same mechanism acts to circularize that functional RNA when it is transcribed; if it has a ribosome binding site it can then be translated.
However, if the 5' and 3' ends of the intron (with a small accompanying portion of the native exon sequence) are conjugated to the 3' and 5' ends of a coding sequence with ribosome binding sites, the reverse process can happen with th exon circularized and the intron pieces removed; This process is detailed in Figure 3. Multiple sources have shown that these products can be translated, but the 2015 Gifu team estimated yield of the circularization product at a mere 2.5%. It is possible that additional proteins are involved, with some indication that initiation of translation by the ribosome encourages proper splicing of the native gene and other proteins can rescue splicing.3 Clearly, the possibility of tuning the response further remains. However, the potential use of this technique to produce a useful naturally repetitive protein remains promising.
Elastin-Like Polypeptides (ELPs)
This is not the first time an iGEM team has worked with ELPs: the 2014 Bordeaux Team studied the characteristics of different numbers of characteristic pentapeptide repeats as potential purification tags and the 2008 Imperial Collage team added a Bacillus subtilis secretion tag to the (EP) 20-24-24 construct from the work of Bellingham et al.4 That work demonstrated the complexity of function dependence on sequence of different ELPs, but did not rule out the generation of alternative interesting constructs. By including an additional cross-linking domain and placing the resulting construct into the circularization parts developed by Team Gifu, an interesting and functional ELP might be produced for inclusion in a collagen-elastin composite material.
Figure 4: Location of RBS within secondary structure predicted by RNA Fold before (a) and after (b) redesign. Note the bases nearly on top of each other within the RBS in the first structure.
Design Concerns
While the original aim was to quickly clone the construct into T7 Express Escherichia coli and focus on optimizing expression, extraction, and functional characterization problems arose rapidly. The design phase was relatively rapid while remaining well-reasoned. Perriman and Ares emphasized the importance of a downstream box structure for optimal translation of circular constructs. Since the ribosome entry site can theoretically be located anywhere along the construct due to the circular nature of the construct, a sequence alignment of a typical downstream box sequence suggested movement of the ribosome binding site from the beginning of the sequence to the second border between exon 24 and exon 21, between a hydrophobic and cross-linking domain. Estimates of RNA secondary structure made by Geneious RNA fold predicted a fairly inaccessible start codon and ribosome binding site (RBS); the insertion of two synonymous T->A mutations in the immediate area without creating any rare codons was able to create a much more accessible RBS based on this level of prediction. Synthesis by IDT of the entire construct was predicted to result in synthesis complexities, but the general convenience of Gibson assembly of 3 fragments made it possible to instead order two shorter sequences with an appropriate overlap (location at the border of two exons made the overlap area unique among possible sequences in the overlap created by the exonuclease). No difficulties were predicted.
Cloning Difficulties
Receipt of the two DNA strands was delayed by a week, the first indication of difficulties due to the repetitive sequence still being a problem. Gibson assembly and transformation yielded a number of colonies, but every colony sequenced had major deletions or insertion. Tuning of Gibson timing and parameters did not result in significant improvement through multiple iterations. PCR amplification of the initial synthesis products became necessary as the intial supply began to run out. An attempt at self-primed PCR rather than Gibson assembly, in case some particular component of the specific procedure was responsible for the difficulties, resulted in a smear potentially indicative of nonspecific binding. Ultimately, brute force identified two candidates from Gibson assembly with perfect sequence-verified ligation regions but deletions in the middle of the sequence, likely from initial synthesis or mutation. One of these had only a single G deletion; however, the repetitive nature of the sequence made site directed mutagenesis (SDM) seem an unlikely candidate. Elastin synthesis was temporarily relegated to the back burner. Trevor! Kalkus, one of our mentors, believed that SDM primers could provide the necessary specificity to correct the defect, and assisted with design of primers and execution of the process. Unfortunately, no corrected candidate was identified in either of two rounds of SDM after sequencing multiple resultant colonies. Ultimately, the result of this project was a densely nested folder containing 245 sequence files with increasingly confused folder names. While the particular assembly approach was clearly not optimal, the lack of results to date does not invalidate the design work done or the concept. Obviously, a smoother cloning process would have been ideal.
Salvage
Fortunately, all was not lost. Some of the collagen constructs contain N-terminal elastin cross-linking domains. Although this was not intentional and largely an artifact of other design concerns, the particular design of the circular elastin part allows for insertion of a coding sequence with an N-terminal elastin cross-linking domain, producing as many as ten or more repeats of this sequence concatenated together. While this construct, BBa_K2027046 may not produce any functional fibers, it contains a number of tags for both purification and sensitive detection of protein production. The longer region to be circularized may result in lower circularization yields, but any protein produced should be much easier to manipulate. This allows some characterization of the circularization device while avoiding the problem of the deletion, located within the coding region being replaced.
References:
Perriman, R. & Ares, M. Jr. (1998). Circular mRNA can direct translation of extremely long repeating-sequence proteins in vivo. RNA, Vol. 4, pp. 1047-1054
Uehara, T., Fujita, Y., Suzuki, H., & Kikuchi, Y. (2012). In Vivo Circular RNA Expression by the Permuted Intron-Exon Method. In E. C. Agbo (Author) & S. Umekage (Ed.), Innovations in biotechnology (pp. 75-90). Rijeka, Croatia: InTech. Retrieved from InTechOpen.
Semrad K, Schroeder R. 1998. Ribosomal function is necessary for efficient splicing of the T4 phage thymidylate synthase intron in vivo. Genes Dev 12: 1327–1337.
Bellingham CM, Woodhouse KA, Robson P, Rothstein SJ, Keeley FW. Self-aggregation characteristics of recombinantly expressed human elastin polypeptides. Biochim Biophys Acta - Protein Struct Mol Enzymol. 2001;1550:6–19
