<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stanford-Brown_SB16_Software skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stanford-Brown/SB16 Software</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Stanford-Brown 2016</TITLE></P><DIV class="container"><NAV class="navbar navbar-inverse navbar-fixed-top"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" id="BioballoonLabel" href="https://2016.igem.org/Team:Stanford-Brown">Stanford-Brown</A><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Stanford-Brown">Home</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Attributions">Attributions</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">BioMembrane</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_Collagen">Collagen</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_Elastin">Elastin</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_p-Aramid">p-Aramid</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_Latex">Latex</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_UV">UV Protection</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Float</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Float_Gas">Gas production</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">BioSensor</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioSensor_Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioSensor_Chromoproteins">Chromoproteins</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioSensor_FQsensor">Fluorophore-Quencher</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_AptamerPurification">Aptamer purification</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Practices</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Human_Practices">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Integrated_Practices">Integrated Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Engagement">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Practices_Interviews">Interviews</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Practices_Exploration">Exploration</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Collaborations_Interlab">InterLab Study</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Collaborations">Team Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Parts">BioBricks</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle active" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Software">Software Design</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Modeling">Modeling</A></LI><LI class="dropdown-header">Notebooks</LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_CE">Collagen &amp; Elastin</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_p-Aramid">P-aramid</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_Latex">Latex</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_UV">UV protection</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_AptamerPurification">Aptamer Purification</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_Gas">Gas Production</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_Chromoproteins">Chromoproteins</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_FQsensor">Fluorophore-Quencher</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_Nylon">Nylon</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_Interlab">Interlab Study</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Team">Team</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Requirements</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_MedalRequirements">Medals</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Safety">Safety</A></LI></DIV></DIV></NAV></DIV><DIV class="container"><DIV class="row rowT"><DIV class="col-sm-12 col-PT noHead"><H1 class="sectionTitle-L firstTitle">Protein Optimization &amp; Gibson Assembly Primer design</H1></DIV></DIV><H2 class="subHead">Overview</H2><DIV class="row"><DIV class="col-sm-12 pagetext">
		    The purpose of this script is to expedite the process of protein DNA sequence optimization and Gibson Primer design for Gibson assembly reactions.  The files for the script can be found either <A href="https://github.com/gsun1729/Prot-Optimization-Gibson-Assembly">here on github</A> or be downloaded individually from the IGEM database below.  Note because IGEM does not support .py file extensions, you will need to remove .txt from the extensions of all of the downloaded files from the links below and append &quot;.py&quot; to the end of (1) seq_analyzer, (2) seq_tools, (3) input_tools, and (4) format_tools (i.e. &quot;seq_tools.txt&quot; becomes &quot;seq_tools.py.&quot;) No further formatting is needed for the github link download.<UL><LI><A href="https://static.igem.org/mediawiki/2016/1/1f/T--Stanford-Brown--POGAS_seq_analyzer.txt">seq_analyzer</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/b/b7/T--Stanford-Brown--POGAS_seq_tools.txt">seq_tools</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/e/e5/T--Stanford-Brown--POGAS_input_tools.txt">input_tools</A></LI><LI><A href="https://2016.igem.org/File:T--Stanford-Brown--POGAS_format_tools.txt">format_tools</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/d/d1/T--Stanford-Brown--POGAS_NT_Lib.txt">NT_Lib</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/6/60/T--Stanford-Brown--POGAS_codonlib.txt">Codon_Lib</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/9/90/T--Stanford-Brown--POGAS_restriction_enzymes.txt">restriction_enzymes</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/d/d5/T--Stanford-Brown--POGAS_ecoli.txt">ecoli</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/4/41/T--Stanford-Brown--POGAS_Ecolik12.txt">ecoliK12</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/f/f5/T--Stanford-Brown--POGAS_yeast.txt">yeast</A></LI></UL></DIV></DIV><H2 class="subHead">Input File Formatting</H2><DIV class="row"><DIV class="col-sm-12 pagetext">
			In order to run the program, the user has to provide a text file containing the sequences in need of optimization and/or primer design. Each sequence should be included on a separate line; any blank newline entries and spaces will result in a processing error. The sequences should also contain no other characters other than that representing nucleotides (A,T,G,C) or amino acids (G,A,L,M,F,W,K,Q,E,S,P,V,I,C,Y,H,R,N,D,T,*).For protein optimization, sequences in the input file are not limited to only amino acid or nucleotide sequences--the user can input either and the program will recognize the sequence type and process it accordingly. This is limited however in cases where the only amino acids in the sequence are alanine, threonine, cysteine, and glycine, since the single letter code for each amino acid is also found in the single letter nucleotide representations. For these cases, the program will be unable to distinguish between an amino acid or nucloetide sequence--this however can be corrected by putting a &quot;*&quot; at the end of an amino acid sequence.<OL><LI>First sequence contains at least 50nt of the 3' end of the backbone where the 5' end of the first fragment will join to</LI><LI>N number of sequences to be assembled ...</LI><LI>Last sequence contains the 5' end of the backbone where the the 3' end of the last fragment will join to This is illustrated in the following figure, where the first sequence in the file is &quot;Backbone front&quot;, last sequence is &quot;Backbone rear&quot;, and the middle sequences in the file would be the N fragments listed in order of assembly.</LI></OL>Sample input and output files are also included in the repository and are listed below:Input:<UL><LI><A href="https://static.igem.org/mediawiki/2016/b/b9/T--Stanford-Brown--POGAS_sample_protopt_input.txt">Sample protein optimization Input</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/d/d8/T--Stanford-Brown--POGAS_sample_gibson_assembly_input.txt">Sample Gibson primer design Input</A></LI></UL>Output:<UL><LI><A href="https://static.igem.org/mediawiki/2016/b/b8/T--Stanford-Brown--POGAS_sample_protopt_input.output.txt">Sample protein optimization Output</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/1/15/T--Stanford-Brown--POGAS_sample_gibson_assembly_input.output.txt">Sample Gibson primer design Output</A></LI></UL>Once the user input files have been formatted properly, the script can be run.</DIV></DIV><H2 class="subHead">Running the Program</H2><DIV class="row"><DIV class="col-sm-12 pagetext">
			Before the script can be run, make sure you have downloaded the following files to a directory of your choosing:<UL><LI>seq_analyzer.py</LI><LI>seq_tools.py</LI><LI>input_tools.py</LI><LI>format_tools.py</LI><LI>Codon_Lib</LI><LI>NT_Lib</LI><LI>restriction_enzymes</LI></UL>These files contain the functions needed by the main script to run the algorithms, and also contain libraries for the program to read from that contain codon and nucleotide pairing maps, and restriction site sequences.Additionally, the user should download<UL><LI>ecoli <A href="http://www.sci.sdsu.edu/~smaloy/MicrobialGenetics/topics/in-vitro-genetics/codon-usage.html">[1]</A></LI><LI>ecoliK12 <A href="http://openwetware.org/wiki/Escherichia_coli/Codon_usage">[2]</A></LI><LI>yeast <A href="http://www.kazusa.or.jp/codon/cgi-bin/showcodon.cgi?species=4932">[3]</A></LI></UL>which are codon frequency use tables for their respective organisms.  If a different codon table is desired, the user can create a textfile with each row (codon) arranged as such:<P style="margin-left: 50px">3nt_codon, \t, single_letter_aminoacid_abbreviation, \t, frequency, #/1000</P>Example:<P style="margin-left: 50px">GCT A   0.16    15.34
			GCC A   0.27    25.51
			GCA A   0.21    20.28
			GCG A   0.36    33.66
			…</P>To run the script, you will need to open terminal or command prompt on your computer.  For windows, press Windows+R, and then type “cmd” into the run bar, and hit enter.  On mac, hit Command+Space, type “terminal” and then hit enter.  This should open a command line interface in which the program can be run.Commands are input into the program in the following format:<PRE><CODE class="language-python">python seq_analyzer.py -<SPAN class="hljs-keyword">if</SPAN> sequence_file -m (g)ibson/(p)rotoptimization -c organism_codon_file -o output_file -t speed
			</CODE></PRE>
			in which the arguments are:<UL><LI>-if = input filename, note the input file should be in the same directory</LI><LI>-m = mode, p for protein optimization, and g for gibson primer design</LI><LI>-c = organism codon frequency usage</LI><LI>-o = output filename</LI><LI>-t = speed at which you want to run the program–a larger number is 0.</LI></UL>The required arguments are -if, -m, and -c.  If no output filename is designated, the output file will be named after the input file with “.output” concatenated on the end.  if no speed is designated, the default speed is 1 second between each step.Example:<PRE><CODE class="language-python">python seq_analyzer.py -<SPAN class="hljs-keyword">if</SPAN> sequence_file -m g -c ecoli -o output -t <SPAN class="hljs-number">10</SPAN></CODE></PRE><PRE><CODE class="language-python">python seq_analyzer.py -<SPAN class="hljs-keyword">if</SPAN> sequence_file -m g -c ecoli <SPAN class="hljs-number">10</SPAN></CODE></PRE>
			Both commands above do the same thing, except the first command designates an output file called output and makes the operation run at 10 seconds between each step.The script will then run and depending on the operation selected ask for appropriate user input and at the end output a text file with the results.</DIV></DIV><H2 class="subHead">Sequence processing</H2><DIV class="row"><DIV class="col-sm-12 pagetext">
			When running the protein optimization module, the program will process each sequence individually.  For each sequence, the program will convert the sequence to its codon optimized DNA form (for amino acid sequences, it will convert it directly to an optimized DNA sequence; for DNA sequences, it will convert it into an amino acid sequence to verify length, and then conver it to an optimized DNA sequence).Each sequence will then be scanned for restriction sites–if a restriction site is found, the program will isolate that area, and iterate over optimal codons until it finds an iteration that does not contain any restriction sites.  This newly optimized area will then replace the original code.When running the gibson optimization module, the program will ask the user for a melting temperature and a salt concentration.  The recommended gibson assembly temperature is 48C from NEB, while the standard salt concentration for a gibson reaction is around 0.05M, or 50mM.  The user can tune either parameter to their specifications, however, note a high salt concentration and/or low melting temperature input can result in primers being more than several degrees apart in melting temperature.  The melting temperature of the primer is calculated via these <A href="http://biotools.nubic.northwestern.edu/OligoCalc.html">formulas</A>.The script will then generate the junction between the fragments to calculate the optimal homology overlap and primer extension to match the temperature and salt requirement.  The process of gibson primer design is indicated in the figure below, adapted from <A href="https://www.neb.com/~/media/Catalog/All-Products/0AA961B294E444AFBEDD5C4A904C76E6/Datacards%20or%20Manuals/ManualE2611.pdf">NEB</A>.  These will then be output in a text file in a readable format.</DIV></DIV><H2 class="subHead">Output Files</H2><DIV class="row"><DIV class="col-sm-12 pagetext">
		Sample output files are included for <A href="https://github.com/gsun1729/Prot-Optimization-Gibson-Assembly/blob/master/sample_input_output_files/sample_protopt_input.output">Protein Optimization</A>and <A href="https://github.com/gsun1729/Prot-Optimization-Gibson-Assembly/blob/master/sample_input_output_files/sample_gibson_assembly_input.output">Gibson processes</A>on github, and linked in iGEM under &quot;Input File Formatting&quot;.If there are any questions and/or comments concerning the program and its usage, please let us know!Thanks!</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>