Team:Stanford-Brown/SB16 Notebooks Latex
Stanford-Brown 2016
Toggle navigation
Stanford-Brown
Home
Attributions
BioMembrane
Overview
Collagen
Elastin
p-Aramid
Latex
UV Protection
Float
Gas production
BioSensor
Overview
Chromoproteins
Fluorophore-Quencher
Aptamer purification
Practices
Overview
Integrated Human Practices
Outreach
Life Beyond the Lab
Collaborations
InterLab Study
Team Collaborations
BioBricks
Protocols
Protocols
Software Design
Modeling
Notebooks
Collagen & Elastin
P-aramid
Latex
UV protection
Aptamer Purification
Gas Production
Chromoproteins
Fluorophore-Quencher
Nylon
Interlab Study
Team
Requirements
Medals
Safety
Latex · Benchling
Latex
Made with Benchling
Project: iGEM 2016
Authors: Gordon Sun, Taylor Sihavong, Elias Robinson
Dates: 2016-07-18 to 2016-10-15
Monday, 7/18
●
Discussed possibility of polymerizing Styrene-Butadiene Rubber
●
Emulsion Polymerization most common, but Anionic "Living" Poymerization (S-BRR) easier in solution
●
Materials needed: styrene, butadiene, n-butyl lithium, cyclohexene
Tuesday, 7/19
ELIAS'S LAB NOTEBOOK
●
ELIM orders #234915
○
Ordered 9 single stranded oligos
○
Primers for linearization of pSB1C3 and fixing Gibson tails
Wednesday, 7/20
ELIAS'S LAB NOTEBOOK
●
Fragment #2 came
●
PCR fun on BBa_5176017 to amplify pSB1C3
●
Took biobrick directly from well 65 plate #6, 2016
○
Diluted with 10ul H2O, used 2ul in PCR
●
PCR with Q5 high fidelity --> will gel tomorrow
Wednesday, 7/27
ELIAS'S LAB NOTEBOOK
●
Gibson overlapping ends: 200nM (A5 hot start, 2xMM)
○
P1 & P2: 32bp, Tm = 72˚C
○
P2 & P3: 39bp, Tm = 71˚C
●
Design for Charlie backbone PCR:
○
For primer: 32bp, Tm = 71˚C
○
Rev primer: 38bp, Tm = 71˚C
○
Cut out backbone from rAIP plasmid:
■
5' flanking base: 2079
■
3' flanking base: 3725
○
Designed with SnapGene at target 55˚C Tm
■
Uploaded to "sheet 2" of "melanin" spread on Google Drive
■
Will order today
●
Design for amplifying Gibsoned latex operon:
○
Used SnapGene with target Tm = 60˚C so primers would be long enough to anneal to gibson product & so there would be extra nucleotides outside of prefix and suffix to anneal to
●
ELIM order:
○
Ordering 4 primers for latex construct & one primer for melanin binding
○
Latex: rAIP_latF & rAIP_latR (linearize rAIP backbone for latex operon)
○
gib_latF & gib_latR (amplify gib product without adding gibson overlaps)
○
gib_ex_latF & gib_ex_latR (amplify gib product with adding gibson overlaps
Friday, 7/29
●
mbPCR cleanup done on undiluted Gibson product from Thursday (Taylor)
○
DNA reading on Nanodrop was wonky and low concentration (~2ng/ul)
Monday, 8/1
●
Ran gel with rAIP linearized backbone for latex operon. Backbone PCRed correctly, but PCR amplification of latex operon post-gibson looked questionable
●
Gibson:
○
2pt: PCR latex operon + lin rAIP
■
8ul of ~12ng/ul
■
2ul of ~37.5ng/ul
○
4pt: P1/3, P2/3, P3/3, lin rAIP
■
3ul rAIP --> 112ng, 86fmol
■
1ul each frag --> 100ng, 180fmol
●
Plated both gibson products at 1:10 and 1:100 dilutions. Used NEB5alpha cells, will miniprep plasmid if successful
Tuesday, 8/2
●
PCR + PCR cleanup done, products sent in for sequencing -- 5 colonies picked
ELIAS'S LAB NOTEBOOK
●
Transform:
○
Ran colony PCR on 5 colonies from 1:10 dilutions of 2pt gibson on latex operon. Will get sequencing back tomorrow.
○
*Made liquid cultures of all 5 colonies, sent out for sequencing --> will miniprep the best one(s) tomorrow based on sequencing data
●
Sequencing:
○
Premix template ELIM
■
1: 4ul and 10ul H2O and 1ul primer
■
2: 3ul + 11ul H2O + 1ul primer
■
3: 2ul + 12ul H2O + 1ul primer
■
4: 2ul + 12ul H2O + 1ul primer
■
5: 3ul + 11ul H2O + 1ul primer
○
Ten total tubes, one with VF2 and one with VR for each colony
Thursday, 8/11
●
DXS Synthase gBlocks arrive
○
Hydrated dry DNA with 20 µl D.I. H2O
○
Stored in 4°C fridge for later Gibson reaction
●
Digested linear pSB1A3 according to iGEM protocol found at:
○
http://parts.igem.org/Help:Protocols/Linearized_Plasmid_Backbones
○
Corrects ends of pSB1A3 backbone to have the correct length Gibson overlap with the ends of the DXS synthase gBlocks
●
Performed Gibson Assembly using NEBuilder master mix to insert two gBlocks comprised of DXS Synthase gene into pSB1A3
●
Transformed DH5-α with Gibson product
○
Plated 100 µl undiluted onto LB+Amp plate
○
Centrifuged transformed cells at 3000rpm for 3 min to pellet, removed 450 µl SOC media supernatant then resuspended cells (2x concentrated )
○
plated 100 µl of 2:1 concentrated cells on LB+Amp plate
Sunday, 8/14
●
Colony pCR of DXS synthase with 4 colonies
●
PCR cleanup nanodrops
○
C1: 84.7ng/ul
○
C2: 76.2ng/ul
○
C3: 76.0ng/ul
○
C4: 62.0ng/ul
●
8 Sq tubes: F&R for each colony
●
Mix: 1ul DNA and 1ul primer and 1.3ul H2O
○
Except 1.5ul DNA for CH
Monday, 8/22
●
1 plasmid seqeuence confirmed--"Latex Plasmid" (2 rubber transferases). Operon with 3 proteins
○
PAGE gel revealed not all proteins were present (proteins being cleaved by latex product)
-->Lyse cells in different way (4 C on plate "Latex Operon").
○
Lyse in a different way: sonicate? Ask Jessica
●
Sequence verified colonies from "Latex Operon 8/8" (2 and 5) and "DXS Synthase" (2 and 3)
Wednesday, 8/24
●
Grow up six liquid cultures overnight at 37˚C in 1.5ml eppendorfs (LB + Chl)
○
1 colony/tube for 3 tubes, from cells expressing the latex operon (Gordon's plates)
○
1 colony/tube for 3 tubes, from cells expressing DXS synthase (Elias's plates)
Thursday, 8/25
●
Grow up 6 liquid cultures overnight for eventual large-scale production in 15ml tubes (6ml LB + Chl, 10ul from each small culture from 8/24)
●
Spin down small cultures from 8/24 at 7000rpm for 15 minutes, remove all supernatant and put in -20˚C freezer for protein extraction the next day
●
Realisation that DXS enzyme-expressing cells should only grow in LB+Amp and on 8/25 they were grown in LB+Chl
○
4 more colonies picked from Elias's DXS plates and put in 1ml LB+Amp overnight to see if the original cells were contamination or are just Chl resistant for whatever reason (?)
Friday, 8/26
●
4 DXS colonies grown in LB+Amp had healthy cell growth --> 1ul IPTG added to these 4 liquid cultures plus the 3 latex operon cultures from 8/25
○
3 DXS cultures from 8/25 kept in -20˚C freezer just in case we need them later
○
Realised that the pelleted cultures were dead anyway so we're just going to do protein extraction.
●
6 6ml cultures from 8/25 put at room temp with 6ul IPTG, glucose, and MgSO4; incubating with additives over the weekend
●
Protein extraction done on 6 1ml liquid cultures (latex 1, 2, 3, DXS 2, 3, 4 grown up in Amp)
○
Soluble fractions collected for all 6 cultures, insoluble fractions collected for DXS 2, 3, 4
Monday, 8/29
●
Lumio gel run with 1ul ladder, 5ul lumio buffer/tube, 0.2ul lumio green/tube, 2ul in-gel detection enhancer/tube, 15ul sample/tube
○
Run with 3 soluble latex, 3 soluble DXS, 3 insoluble DXS
○
Latex bands came out ok, DXS bands all messed up
IMG_4831.JPG.jpeg
The ladder, then latex 1, latex 2, latex 3, DXS soluble 1, DXS insoluble 1, DXS soluble 2, DXS insoluble 2, DXS soluble 3, DXS insoluble 3
●
For latex, expecting bands at: 24.97kDa (SRPP), 35.79kDa (HRT1), 35.31kDa (HRT2): between rungs 6 and 5 on the ladder
●
Ran another gel with only DXS samples: 3 soluble and 3 insoluble, separated by a lane, and then Intern Ryan's sample separated by another lane
IMG_4832.JPG.jpeg
Intern Ryan's sample, then an empty lane, then DXS insoluble 3, DXS insoluble 2, DXS insoluble 1, then an empty lane, then DXS soluble 3, DXS soluble 2, DXS soluble 1, and the ladder
IMG_4834.JPG.jpeg
Additional picture with some brightness settings changed. Should be expecting 70.25kDa band for DXS, all bands around that range are very light (between rungs 2 and 3 on the ladder)
Monday, 9/12
●
PCR done on pUC19 and DXS synthase with new primers that Trevor! and Amy ordered
○
Annealing temp 57˚C for pUC19, ~2600bp
○
Annealing temp 54˚C for DXS, ~600bp
○
Stored in fridge overnight for cleanup the next morning
Tuesday, 9/13
●
PCR cleanup done on pUC19 and DXS synthase: 44.2ng/ul and 33.1ng/ul, respectively
●
Gibson done on PCR cleanup products
○
1.95ul vector used, 1.78ul insert used
○
Gibson results: 1139.5ng/ul
●
Colony from old latex operon plates picked and grown in 5ml LB and 5ul Chl overnight
Thursday, 9/15
●
Miniprep done of latex operon culture: 76.7ng/ul
●
Transformation into T7 cells done with latex operon and DXS plasmids (0.65ul latex operon, 3.5ul DXS)
○
Plated on Chl + Kan plates
Tuesday, 9/20
●
Thursday transformation failed: realised mistake: puc19 is resistant to Amp, not Kan
●
Testing transformation done with T7 cells: latex operon into T7 on Chl plates, and DXS synthase into T7 on Amp plates and Kan plate
Wednesday, 9/21
●
As expected, all plates grew except for the DXS on Kan
●
Final transformation done with both plasmids into T7 cells (0.65ul latex operon, 3.5ul DXS) on Chl + Amp plates
Thursday, 9/22
●
Two total colonies grew successfully in double antibiotic resistance. Both grown up overnight in Liquid Culture (5mL LB +Amp +Chl)
Monday, 9/26
●
Small liquid cultures taken to Stanford and grown in 1000ml LB + Amp + Chl
●
Culture is split into 3 500ml containers with 333ml of culture
Tuesday, 9/27
●
After liquid cultures are grown up, two are taken, centrifuged at 20min, 4˚C, 1200rpm
○
The pellets are kept frozen in the fridge as backups for further characterisation and protein purification
●
The third liquid culture has 350ul of IPTG, glucose, and MgSO4 added
Friday, 9/30
●
The third liquid culture is placed at room temperature to shake and a further round of IPTG, glucose, and MgSO4 is added
Friday, 10/7
●
Latex extraction is performed with the protocol from Gordon's lab notebook
○
Chloroform and methanol are used to lyse the bacterial cells and precipitate the latex
○
A rubbery substance is extracted and put in an eppendorf for later testing
●
Rubbery substance is burned as a quick test to see if it is latex
IMG_5069-1.JPG
IMG_5071.JPG
Saturday, 10/15
●
Follow-up tests done on latex-induced culture to see if latex is produced internally or externally
●
Same protocol as previous latex test, but LB and cell pellet were separated before chloroform was added to both
○
Results inconclusive
