Team:Stanford-Brown/SB16 Notebooks Chromoproteins
Stanford-Brown 2016
Toggle navigation
Stanford-Brown
Home
Attributions
BioMembrane
Overview
Collagen
Elastin
p-Aramid
Latex
UV Protection
Float
Gas production
BioSensor
Overview
Chromoproteins
Fluorophore-Quencher
Aptamer purification
Practices
Overview
Integrated Human Practices
Outreach
Life Beyond the Lab
Collaborations
InterLab Study
Team Collaborations
BioBricks
Protocols
Protocols
Software Design
Modeling
Notebooks
Collagen & Elastin
P-aramid
Latex
UV protection
Aptamer Purification
Gas Production
Chromoproteins
Fluorophore-Quencher
Nylon
Interlab Study
Team
Requirements
Medals
Safety
Chromoproteins · Benchling
Chromoproteins
Made with Benchling
Project: iGEM 2016
Authors: Cynthia Hale-Phillips, Theresa Sievert, Taylor Pullinger
Dates: 2016-06-06 to 2016-10-18
Monday, 6/6/16
●
Students arrive! Orientation to Rothschild Astrobiology/Synthetic Biology Lab at NASA Ames Research Center
●
Plan to continue research on chromoproteins to catalogue the denaturation temperature of each protein
●
If we do so, perhaps we can employ it in a biological thermometer
Tuesday, 6/7/16
●
E. coli expressing producing different chromoproteins were placed into 15 mL tubes with LB and incubated at 37 °C for 24 hours.
A
B
1
Chromoprotein
Observations
2
RFP
Clear red color, fluorescent
3
amil GFP (yellow)
Yellow color, fluorescent
4
amil CP (indigo)
Purple blue color, not fluorescent
5
meff (blue)
No significant color production
iGEM Chromoprotein Color and Fluorescent Observations
●
The RFP was examined for whether it was heat or light inactivated. Our rudimentary setup to analyze this was to extract a portion of the cells and smear them onto a clear plate. This was then heated to 50 °C, at which point the cells dried and also lost their color. In addition, it was no longer fluorescent.
●
The cells were allowed to sit for a short period of time until the temperature had dropped, upon which the RFP appeared to slowly regain pink color and became fluorescent again.
●
These preliminary results into the nature of RFP suggest that it is heat-inactivated, meaning that there is definitely potential for application in a bio-thermometer.
Wednesday, 6/8/16
●
Pellet the liquid cultures started yesterday by centrifuging
●
Transferred portions of each pellet into PCR tubes, then heated them at different temperatures in thermal cycler to see where the color loss for each chromoprotein was
○
RFP denatures at 90˚C
○
Indigo denatures at 90˚C
Thursday, 6/9/16
●
The heated RFP did not renature overnight, remained discolored
●
Tested fluorescence for RFP and GFP
●
Performed Gibson assembly of the 4 chromoproteins for iGEM (AmiGFP Yellow, RFP Red, meffblue - Blue, AmilCP - Indigo) with primers ordered last week
●
Then transformed these into chemically competent E. Coli and plated these Gibson assemblies onto LB+chlor plates
Friday, 6/10/16
●
Gibson assembly iGEM chromoproteins: No growth on any of the LB+chlor plates
○
Due to a mistake in the sequence when we ordered the primers
○
New, correct primers have been ordered
Monday, 6/13/16
●
Used new primers to PCR our 4 iGEM chromoprotein and his-tag to get them ready for a Gibson assembly tomorrow
Tuesday, 6/14/16
●
Performed Gibson assembly on the 4 iGEM chromoproteins and transformed and plated and placed into the 37 °C incubator overnight and will hopefully see growth on plates tomorrow
Wednesday, 6/15/16
●
No growth on any of the plates with the new Gibson assembly
○
Perhaps the chromoprotein sequence is incorrect
Thursday, 6/16/16
●
Cell competency test done on 3 different DNA concentrations (0.5pg/ul, 10pg/ul, 50pg/ul) with 2 different cell types (T7, 5-alpha)
●
Protocol given for the competency test wasn't compatible with T7, but we got colonies on all of our 5-alpha plates (Charlie, Taylor)
Friday, 6/17/16
●
Started new Gibson assembly of amilCP and the flag lumio-His tags
●
Then trasnformed and plated this Gibson assembly
Monday, 6/20
●
Checked last week's Gibson assembly on chromoprotein and flag lumio-His tags. No colonies across all 3 tested plates.
Tuesday, 6/21
●
Gibson redo #3
○
Diluted tag from 577 ng/ul to 150 ng/ul to get 2.9 pmol/ul
○
1 ul of dilution and 1ul of vector added together
○
2 types of incubation: 50 ˚C for 15 min and stepped down from 52 ˚C (0.5 ˚C/5 min) until temp reached 48 ˚C
■
Stepped down temperature labeled as #2
○
Transformed with NEB 5-alpha and plated with 1:4 and 1:40 dilution
●
Gibson: discovered we initially didn't ligate the strands properly with buffer, Cynthia is doing a redo
Wednesday, 6/22
●
Gibson assembly re done today and strands properly ligates with buffer
Thursday, 6/23
●
Gibson worked!!!!!!!!!! Colony growth seen on plates
Monday, 6/27
●
We obtained chromoproteins from DNA 2.0 in the paintbox set
●
Here is a link to more information on the different chromoproteins contained in the paintbox: https://www.dna20.com/eCommerce/catalog/datasheet/348.
●
For our purposes we used the kanamycin resistance.
●
We plan on inserting these chromoproteins into a psB1C3 backbone to standardize to iGEM
A
B
C
1
Chromoprotein Name
Shorthand Given
DNA 2.0 name
2
Blitzen Blue
BB
CBP-33-444
3
Dreidel Teal
DT
CBP-34-444
4
Virginia Violet
VV
CBP-35-444
5
Vixen Purple
VP
CBP-36-444
6
Prancer Purple
PP
CBP-37-444
7
Tinsel Purple
TP
CBP-38-444
8
Maccabee Purple
MP
CBP-39-444
9
Donner Magenta
DM
CBP-40-444
10
Cupid Pink
CP
CBP-41-444
11
Seraphina Pink
SP
CBP-44-444
12
Scrooge Orange
SO
CBP-45-444
13
Leor Orange
LO
CBP-46-444
Table2
Here are the 12 DNA2.0 Chromoproteins we will be working with and the abbreviations we will be using throughout the summer for each of them
●
We created primers for all of these chromoproteins for the vector (all the same), its promoter, the specific chromoprotein, and the cellulose binding domains that will be included into the chromoproteins.
●
Because the chromoproteins shared many cellulose binding domain primers they were named to show which one was shared and could be used multiple times.
A
B
C
1
Shorthand
Sequence
Used For
2
CBD1
GTCATCACAAAACCGGCGGT
BB
3
CBD2
TCATAACACCGGCGGTCCG
DT
4
CBD3
TCATCACACCGGCGGTCCG
VV, DM, LO
5
CBD4
AGCGACGACCGGCGGTCCG
VP, PP, TP, CP
6
CBD5
AGCGACCACCGGCGGTCCG
SP, MP
7
CBD6
AGAAACGACCGGCGGTCCG
SO
Table1
Table3: shows the shorthand used for the forward cellulose binding domain primers
Wednesday, 6/29
●
Started 5 ml liquid cultures from the iGEM biobrick chromoproteins to see if we could induce the color we expected
●
Did PCR for the Gibson Assembly tomorrow on the linearized pSB1C3, cellulose binding domain (CBD), flag lumio His-tag (His-tag) and the DNA2.0 chromoproteins. The primers for these reactions are in the chromoprotein excel spreadsheet
Thursday, 6/30
●
Ran a gel with the 12 chromoproteins that had been PCRed and CBD1-6
●
Performed gel extraction on all 18 samples
○
Chromoproteins MP and LO did not show bands of DNA in the right place on the gel for this reason we decided to redo the primers for them
Tuesday, 7/5
●
We performed a Gibson assembly on the DNA 2.0 chromoproteins to add a CBD and His-tag all together
A
B
C
D
E
F
G
H
I
J
1
Gibson Assembly Amounts Added
2
Chromoprotein
ul
Plasmid
ul
CBD to add
ul
His tag
ul
Water ul
Matermix ul
3
BB
2
AD
1
CBD1
1.5
all the same
1
4.5
10
4
DT
1.5
AD
1
CBD2
1.5
all the same
1
5
10
5
VV
2
AD
1
CBD3
1.5
all the same
1
4.5
10
6
VP
1.5
AE
2
CBD4
1.5
all the same
1
4
10
7
PP
1.5
AE
2
CBD4
1.5
all the same
1
4
10
8
TP
1.5
AE
2
CBD4
1.5
all the same
1
4
10
9
DM
2
AD
1
CBD3
1.5
all the same
1
4.5
10
10
CP
1.5
AE
2
CBD4
1.5
all the same
1
4
10
11
SP
1.5
AD
1
CBD5
1.5
all the same
1
5
10
12
SO
1.5
AF
2
CBD6
1.5
all the same
1
4
10
Gibson Assembly of DNA 2.0 Chromoprotein-CBD-His-tag
Table1: Shows the fragments and vector added to the Gibson Assembly procedure performed
●
The new primers for MP and LO came in today! We will PCR and gel extract them tomorrow
●
We do not have enough NEB 5-alpha cells to transform so we will wait until those come in to transform them
IMG_0779.JPG
We spun down the chromoproteins from iGEM that had been grown up to see what colors we would get and were happy to see they expressed different distinct colors. We have resuspended the cells now
Wednesday, 7/6
●
Performed PCR on MP and LO chromoproteins using the new primers received yesterday
●
Made gel and ran the PCR products for MP and LO and a DNA ladder
●
We performed a gel extraction and nano dropped our samples they were both in the 20 ng/ul range. We will use these PCR products to perform a Gibson assembly tomorrow
Thursday, 7/7
●
Performed Gibson on MP and LO proteins using the table below as a guideline for how much to add
●
Instead of the Gibson Master Mix we used NEB Builder HiFi DNA Assembly Mix (2X) for the LO chromoprotein
●
Placed Gibson tubes into freezer with other Gibson samples and plan to transform on Monday
●
Poured plates (LB+Chlor) for our transformation of the Gibson assembly products on Monday
A
B
C
D
E
F
G
H
I
J
1
Gibson Assembly Amounts Added
2
Chromoprotein
ul
Plasmid
ul
CBD to add
ul
His tag
ul
Water ul
Matermix ul
3
MP
2
AD
1
CBD1
1.5
all the same
1
4.5
10
4
LO
1.5
AD
1
CBD2
1.5
all the same
1
5
10
Table3
Friday, 7/8
●
Inoculated new liquid cultures of iGEM chromoproteins from previous liquid cultures
●
Put iGEM chromoproteins from room temperature oscillator into the 4 °C fridge to store (they store for up to a month like this)
Monday, 7/11
●
We transformed and plated all 12 of our Gibson products of DNA2.0 Chromoprotein-CBD-His-tag at a 1:4 dilution and a 1:40 dilution on LB+Chlor plates and will grow them up overnight in the incubator
●
We ran a PCR on AE blue so that we could run a Gibson on AE blue vector and CBD
Tuesday, 7/12
●
Ran the gel on AE blue PCR product and performed a gel extraction on the CBD, because the vector did not show up on the gel we re ordered primers
●
Took our plates from the transformation yesterday out of the incubator and 10 of them had significant growth. The other 2 plates with little growth (LO and SP) were placed back into the incubator for a longer incubation. We colony PCRed 30 samples (3 from each plate that worked) and will sequence them in the future if our liquid cultures work (when we see colored expressed)
●
We also started 10 liquid cultures, 1 from each plate that worked of the chromoproteins
●
The other 2 plates that did not grow as well grew up throughout the day in the incubator and we created liquid cultures of them and will colony PCR them tomorrow
Wednesday, 7/13
●
Ran colony PCR on SP and LO (the plates that grew up throughout the day yesterday)
●
New primers came in for the vector of AE blue
●
Used the new primers to run PCR on the AE blue vector
●
The cultures we grew up from Friday with the iGEM chromoproteins we spun down to see if their color expressed. We picked the ones that expressed color best to move forward with and they are:
○
spisPink
○
amajLime
○
tsPurple
○
gfasPurple
○
asPink
○
scOrange
IMG_0860.JPG
spisPink iGEM Chromoprotein
IMG_0861.JPG
amajLime iGEM chromoprotein
IMG_0862.JPG
tsPurple iGEM chromoprotein
IMG_0863.JPG
gfasPurple iGEM chromoprotein
IMG_0864.JPG
asPink iGEM chromoprotein
IMG_0865.JPG
scOrange iGEM chromoprotein
●
We also kept aeBlue and meffRed to work with in the future but did not spin them down
Thursday, 7/14
●
Ran PCR on AE Blue vector with new primers (25 ul reaction) from our liquid culture we had started
●
Ran gel on the AE blue vector with new primers
●
When we performed the gel extraction the band was faint and not enough protein was present so we are doing a 50 ul reaction tomorrow in hopes of getting a larger band
Friday, 7/15
●
Ran new PCR reaction of AE blue vector at 50 ul to hopefully increase protein yield
●
Ran a gel of the PCR from AE blue and performed a gel extraction
●
Low to little DNA after gel extraction so we plan to use AE blue from our cryostock next week to run a PCR again in hopes that the low yield is a result of the AE blue culture having sat incubating too long
Monday, 7/18
●
Ran PCR of AE Blue from cryostock. Running it from the cryostock will hopefully increase yield of vector in gel so that we can perform a gel extraction on that
●
Ran a gel on AE blue PCR product and also on 5 samples of colony PCR we created last week from DNA 2.0 Gibsoned product. The 5 samples we ran from the Gibson product were:
○
MP-28 (Chromoprotein-Colony number)
○
SO-21
○
DT-12
○
DM-3
○
BB-24
●
Looking at the gel, the AE blue bands were too faint to perform a gel extraction on and means a step is going wrong either in out PCR, with our primers, or taking directly from the cryostock to run a PCR. We are going to mini prep from the cryostock and then run the PCR and gel again to see if this solves the issue of the cryostock now working directly in a PCR and to see if this is the issue with our gel
●
The Gibsoned DNA 2.0 chromoproteins showed up on the gel in the correct spot around 1000 base pairs on the ladder. This is the size that we would expect the chromoprotein plus the cellulose binding domain plus the his tag plus the vector. The only Gibsoned product that did not work of the five we ran was the DT-12
●
Our liquid stocks we grew from the Gibsoned DNA 2.0 chromoproteins are still on the room temperature shaker. After spinning a couple of them down this morning to see if their color was expressing we did not see any color. We are thinking that the cellulose binding domain may be affecting the color expression of the chromoproteins or they are just taking a long time to express (the iGEM chromoproteins also took a long time to express)
Tuesday, 7/19
●
We sent in some of the Gibsoned DNA 2.0 chromoproteins for sequencing
●
We added the following for each sequencing reaction and labeled according to the below table:
○
2 ul PCRed Gibsoned product
○
1 ul primers (VF2 or VR)
○
12 ul of water
A
B
1
Reaction #
Reaction Name
2
1
MP29_F
3
2
MP29_R
4
3
SO20_F
5
4
SO20_R
6
5
DT10_F
7
6
DT10_R
8
7
DM1_F
9
8
DM1_R
10
9
BB22_F
11
10
BB22_R
Sequencing Orders
Table1: This table shows the reactions and what they were named
●
Put a culture of LB+chlor into the shaker incubator overnight from our cryostock of the iGEM chromoprotein AEBlue
●
We covered the Gibsoned DNA 2.0 liquid cultures that were shaking at room temperature with aluminum foil. Chromoprotein cells were not growing as well in light last year because the light helped crate reactive oxygens that stunted the growth of the chromoproteins and caused less expression of the chromoproteins.
Wednesday, 7/20
●
Mini prepped the culture that had grown up overnight of the AE blue iGEM chromoprotein
Thursday, 7/21
●
Ran PCR on mini prepped AE blue
●
Ran gel of AE blue mini prepped vector
●
Performed gel extraction even though band on gel was weak and nano drop showed no DNA at the end of the extraction
Friday, 7/22
●
Ran PCR again on mini prepped AE blue
●
Ran PCR clean up on mini prepped AE blue to confirm a DNA product after the mini prep reaction
●
After the PCR cleanup we nano dropped our sample and there was only a 9.3 ng/ul concentration of DNA. This means that running the gel and performing the gel extraction is not the issue. The issues that may be affecting the PCR are:
○
Temperatures of PCR not allowing proper annealing
○
Primers not right
○
Vector not what expected
Monday, 7/25
●
Spun down DNA 2.0 Chromoprotein liquid cultures that have been on the room temperature oscillator. Color was still not observed which makes us think the chromoproteins were not the correct sequence when we got them from DNA 2.0
●
Started a liquid culture of AE Blue from the cryostock. We plan to PCR it using VF2 and VR primers and then perform a gel extraction to then sequence that DNA to see what the sequence of the chromoprotein actually is
Tuesday, 7/26
●
Mini prepped AE blue from cryostock
●
Ran PCR of AE Blue that was mini prepped using VF2 and VR primers
●
Ran gel of AE blue PCR product and biobrick of cellulose binding domain (CBD) which should contain the cellulose binding domain and GFP.
●
The gel worked for both the PCR product and the CBD but two bands were seen in the CBD when we expected only one.
●
Gel extracted the AE blue PCR product and the two bands from the CBD and sent them in for sequencing to help us better understand what we are working with and how to order primers to properly linearize the AE blue and CBD
Wednesday, 7/27
●
Sent 6 samples in for sequencing (AE blue, GFP, and CBD both forward and backward primers)
●
The sequencing data for the DNA 2.0 chromoproteins reveals that there is probably and error in the CBD biobrick which leads to the DNA 2.0 chromoproteins not being expressed properly
●
We found another CBD in the iGEM distribution kit which we rehydrated and PCRed and performed a PCR clean up on. We will send it in for sequencing tomorrow to see if it another option for a CBD
●
We found out that the DNA 2.0 chromoproteins we received did not express color when grown up from the stock and only grew when IPTG was added. This means that our chromoproteins did not initially express color and was the wrong sequence so that is why our Gibsoned product did not express color when getting grown up in liquid culture. We have another set of DNA 2.0 chromoproteins in lab the have been shown to grow up in liquid culture and express color so we will be using those for the next phase since they are correct.
Thursday, 7/28
●
Sent in the new CBD from the iGEM distribution kit for sequencing (we will call CBD2)
●
Looked at sequencing data from yesterday for the AE blue iGEM chromoprotein, CBD, and GFP.
○
For the AE blue chromoprotein it looks like the sequence is what we expected which means for the future we need to redesign primers and see if we can get the PCR and gel extraction to produce product
○
The CBD is the expected sequence and we are going to determine which CBD to use after we get sequencing back from CBD2 we sent in today
●
Plated on a glass plate three of the iGEM chromoproteins and let them dry out overnight so we can observe their color change over time and temperature. We plated three chromoproteins:
○
AE blue
○
gfas Purple
○
ts Purple
●
We used our previous chromoproteins from iGEM that had been dried on a plastic dish and took videos of them over time while increasing the temperature and then decreasing the temperature to observe their color loss
A
B
C
D
E
1
Time to warm
Temperature (degrees C)
Time to cool
Temperature (degrees C)
2
0:00
51
0:00
100
3
1:00
56
1:00
103
4
2:00
60
2:00
97
5
3:00
64
3:00
92
6
4:00
68
4:00
89
7
5:15
72
5:00
86
8
6:00
75
6:00
83
9
7:00
75
7:00
82
10
8:00
76
8:00
79
11
9:00
80
9:00
74
12
10:00
82
10:00
73
13
11:00
87
11:00
70
14
12:00
92
12:00
69
15
13:00
99
13:00
66
16
14:00
101
14:00
65
17
15:00
63
18
16:00
61
19
17:00
59
20
18:00
57
21
19:00
55
22
20:00
55
23
21:00
52
24
22:00
51
Table5
This table shows the temperature the chromoproteins were at over time for both heating them and cooling them. The time stamps correspond to the video time.
Friday, 7/29
●
Rehydrated the DNA 2.0 chromoproteins that actually work which we got from Trevor with 20 ul of Milli Q water
●
Transformed DNA 2.0 chromoproteins using half of a normal transformation protocol and plated them to grow over the weekend
●
Put a black X on top of the chromoproteins that did not work
●
Took videos of the color change of the three plated chromoproteins over time we plated yesterday
○
The temperature went up to 83 °C and the chromoproteins lost their color
○
When losing temperature the chromoproteins did not change back so we need to find at what temperature the chromoproteins can go up to without being damaged
7.29.16 iGEM Chromoproteins before Heat.jpg
Here are the chromoproteins before we added heat to them
●
Created liquid cultures of all 12 iGEM Chromoproteins and put them in the incubator/shaker over the weekend to get color expression out of these chromoproteins so we can perform further heat tests on them
Monday, 8/1
●
Looked at the two liquid cultures of DNA 2.0 Chromoproteins (MP and DT) and saw color only in the DT culture. Letting both of them incubate more
●
Took out iGEM chromoproteins that we started as liquid cultures on Friday and are not expressing color so put on room temperature on oscillator
●
Found two different AE blue plasmids and trying a Gibson assembly on each with already ordered primers to see if it works and then will transform. We nano dropped the Gibsoned product and found there was a high amount of DNA present. Transformed the AE blue plasmids onto LB plates.
●
From the plates we put DNA 2.0 chromoproteins on we found two were expressing color and started liquid cultures from those and put all plated into incubator to see if color will express with more incubator. These plates had 1 ul of .4 M IPTG per ml of media
●
Started a new liquid culture of AE blue from the cryostock
Tuesday, 8/2
●
Started liquid cultures from colonies on transformation plates from the Gibson yesterday of the AE blue, CBD, and His tag
●
Started new culture of chromoproteins using a 1 ul from previous cultures of DNA 2.0 chromoproteins Cupid Pink, Prancer Purple, Donner Teal. We spun down the remainder of the cultures and put the cells onto a plate to dry and then do light testing today.
●
We created a setup to more consistently video and provide temperature to the chromoproteins. Pictures below show the set up
IMG_0198.jpg
IMG_0199.jpg
IMG_0200.jpg
●
Light testing on the DNA 2.0 Chromoproteins using a water bath this time to try and keep the temperature consistent over time
●
TRIAL 1: In video in order Cupid Pink, Prancer Purple, Donner Teal from left to right
A
B
1
Time
Temperature (C)
2
0:00
24.2
3
5:00
28.6
4
10:00
40
5
12:00
43.6
6
15:00
50.2
7
18:00
56
8
19:00
59
9
End (20:00)
60
Table6
This table shows the temperature over time of heating up the chromoproteins
●
After the chromoproteins lost their color due to heat added we took them off the heat and they did not regain color but when we added water to the chromoproteins (10 ul) we saw the color come back to what is was originally before it lost the color
○
This might suggest the chromoproteins are losing their color because they are being dehydrated by the added heat and this causes the color change
●
Time Lapse videos are in the google drive
8.2.16 Before Heat iGEM CP.jpg
The chromoproteins before the heat was added
8.2.16 After Heat iGEM CP.jpg
The chromoproteins lost color after heat was added
8.2.16 Rehydrated iGEM CP
Here is a picture of the chromoproteins once they were rehydrated. Their color is back to what it looked like before it was exposed to heat. The order from top to bottom: cupid pink, prancer purple, and donner teal
●
When spinning the cultures down from the iGEM Chromoproteins the AE blue one looked purple which is interesting. The other iGEM chromoproteins looked normal when spun down except for amilCP, amilGFP, and fwYellow. Those three were not expressing any color or minimal color so we are not going to use them in the future and will move forward with the ones that worked
●
Using the DNA 2.0 chromoprotein VP (Vixen purple) we spun a liquid culture of it down and then used a q tip which we placed in the oven to see how the color change would occur. The color was lost with the added heat and when the q tip was placed into water the color came back
●
TRIAL 2: Once the rehydrated iGEM chromoproteins dried out we tried heating them again and took a video. We observed a loss of color.
●
TRIAL 3: We the rehydrated them again and put them back onto the hot plate and the color disappeared once again.
Wednesday, 8/3
●
We are testing more chromoproteins and their response to heat. We are testing CP, AE blue, PP, TS purple (TS), spis pink (SP), scrooge orange (SO). CP and PP are the DNA 2.0 chromoproteins and the rest are the iGEM Chromoproteins. This test will help show the difference (if any) between the DNA 2.0 chromoprotein and iGEM chromoprotein response to heat
●
We plated these 6 chromoproteins in the configuration shown below
8.3.16 Chromoprotein Test.JPG
The configuration of the chromoproteins we tested against heat today. CP and PP are DNA 2.0 chromoproteins and the rest are iGEM chromoproteins
●
We took a video over time of the chromoproteins reacting to heat and this time our set up included a thermometer that is in the heat bath to monitor the temperature throughout to make sure the heat bath was the same as the thermometer gun was telling us
●
The chromoproteins are being put directly into a constant heat (74-75 °C) instead of being heated over time to see their reaction. This test is being performed so we can see how fast the chromoproteins react and change color at a constant temperature. At the 4.5 minute mark the temperature peaked at 76 °C and held steady after that. The color also held steady at the constant temperature and dried out over time. The time lapse video and full length video are in the google drive
●
We then rehydrated these chromoproteins and retested them while heating over time to see if there was noticeable color change. The initial temperature of the water bath was 55 °C
A
B
1
Time
Temperature (degrees C)
2
0:00
55
3
2:00
63
4
4:00
67
5
5:00
68.5
6
6:00
70
Table7
Change over time of the temperature of the water bath the chromoprotein plate was sitting in
●
This test showed a change/ loss of color over time but also it revealed that we need to find a better way to plate the chromoproteins on the glass so they are evenly spread and have an even thickness so the color loss is consistent
●
The chromoproteins are still in liquid form at time 0 so they may not react to the change in temperature right away
●
We made more LB+Kan media and LB+Chloro media to use for the iGEM chromoprotein cultures
●
The AE blue from the Gibson assembly is expressing color in the liquid culture which looks like the Gibson was successful. We will test further the Gibsoned AE blue today and we will start a 5 ml culture from a the current liquid culture for a mini prep tomorrow
●
We are looking into using ImageJ in the future to better quantify the changes in color we are seeing in the chromoproteins and create a way to consistently quantify our results
●
Currently thinking about ways to better plate the chromoproteins for consistency and for better ways of testing
Thursday, 8/4
●
Mini prep on the culture from the Gibson Assembly of AE Blue, Cellulose Binding Domain, and His Tag. After the mini prep, a PCR was done using VF2 and VR. The PCR product was then run on the gel which revealed a band at the 1000 bp mark (this suggests that the Gibson Assembly was successful). I cut out the band from the gel and did a gel extraction. I'll send this in for sequencing tomorrow.
●
The second AE Blue Gibson that grew cells did not express a blue color so that culture (which was entirely independent of the previously mentioned culture) was disguarded. This is likely because the AE blue vector that was linearized for the Gibson Assembly had some errors.
●
I spun down the chromoproteins I incubated yesterday to check for color. While most had a faint color, Cupid Pink (from DNA2.0), Prancer Purple (from DNA2.0), ScOrange (IGEM), AE Blue (iGEM), and dsPink (iGEM) had the strongest showing color. These five were selected to do temperature tests on while the remaining cultures were placed back on the room temperature incubator. Replacement cultures from five selected were made using 2 uL of the culture into fresh media and set to incubate at 37 °C overnight in the shaker.
●
After checking back at the plates (stored in the 4 °C fridge) made from transforming the DNA2.0 Chromoprotein plasmids, I diiscovered that others looked like they were expressing color, so I made liquid cultures of the other ten DNA2.0 chromoproteins and set them to incubate at 37 °C overnight in the shaker.
●
While checking the plates from the DNA2.0 transformation, I also looked at the plates from the Gibson Assembly we did 7/5/15. Although the majority of them did not show any colored colonies, there were two blue colonies on the Bilzen Blue plate. There were several colonies on this plate that did not express any color, so it might be possible that these two colonies were successful outcomes of our Gibson Assembly from a month ago. I started a liquid culture of these two colonies and will check for color expression tomorrow.
●
In order to test the selected chromoproteins, I tried plating one of the colors in on of the wells in a 96 well plate, but discovered that the plastic of the 96 well plate was not good for conducting heat to the sample (the liquid culture in the well did not dry even when it was on the heating block for 10 minutes, something that typically happens). Thus, I returned to the idea of plating the chromoproteins on a glass petri dish.
●
In order to attempt to standardize the process of plating the chromoproteins on the petri dish, I first spun down the cultures (4500 rpm for 2 minutes) then decanted out the supernatant. Then, I spun the tubes again to further concentrate the cells on the bottom of the tube which allowed me to remove the remaining supernatant from the tube. Lastly, I resuspended the cells clumped at the bottom of the tube in 15 uL of DI water then plated 15 uL of the cells onto the petri dish. Because Cupid Pink and Prancer Purple had been studied in previous experiments, I moved on to look at ScOrange, AE Blue, and dsPink--these were the ones that were plated. The clumped cells of Cupid Pink and Prancer Purple were kept in a tube rack to study tomorrow.
●
A video from the temperature test of ScOrange, AE Blue, and dsPink was uploaded to the google drive. From this experiment, it appears that AE blue turns clear then dries purple (it also rehydrates to purple) when heated to temperatures above 70 °C. This color change was not rescuable, but the now purple AE Blue becomes clear when heated. dsPink does not appear to change in color in response to heat even to temperatures exceeding 90 °C. ScOrange (which looks yellow when plated as described) becomes a fainter yellow when heated then rehydrated.
IMG_1865.JPG
To the left are scOrange (top), dsPink (middle), and AE blue (bottom) which were plated in the middle of the petri dish heated to 90 °C, then rehydrated. The patch of chromproteins to the right of scOrange is from the cell culture that the scOrange (in the top center) was orginally taken from. The patches of chromoproteins to the left of the center row are also from their respective original cultures. This was done in order to compare the loss of color the chromoproteins experience after heating and rehydrating. As it is clear to see, AE blue is now purple/pink.
●
In order to explore other applications for these chromoproteins, I wanted to test if their color was UV sensitve. In order to do this, I plated Prancer Purple and dsPink in 6 wells in a 96 well plate then placed that plate in a UV light box. After twenty minutes, there was no detectable change in response to UV. An additional ten minutes also yielded the same result. Futher tests will be conducted with longer time scales, but it seems safe to conclude that these chromoproteins do not have a short term response to UV light.
IMG_1860.JPG
To the left is an image of the chromoproteins under UV light
IMG_1861.JPG
A picture of the chromoproteins before UV exposure (Prancer Purple is the most visible). 5 uL dots of the dsPink chromoproteins were placed in the wells to the right of Prancer Purple to see if volume was affecting the UV sensitiviy.
IMG_1862.JPG
To the left is a picture of the chromoproteins after being exposed to UV light for 30 minutes. No detectable changes.
Friday, 8/5
●
Sent AE Blue-CBD-His Tag in for sequencing
●
Checking the chromoproteins for color after 1 day of incubation in the 37 °C shaker:
○
Vixen Purple: very dark purple
○
Biltzen Blue: tealish blue
○
Cupid Pink (made from culture of Cupid Pink): very faint pink (interestingly paler than I've noticed in the past)
○
Cupid Pink (made from plated cupid pink): very bright pink
○
tsPurple: pale purple (but strong enough to defintely tell that this is purple. It's just less than vixen purple)
○
scOrange: light orange
○
Prancer Purple (from the plated prancer purple): bright purple (slightly different color than vixen purple)
○
Loer Orange: light orangey/pinkish hue
○
Scrooge orange: very pale pink/orange color (almost identical to color that cells normally are)
○
Virginia Violet: very pale color, not even sure this looks purple (normal color cells with a darkish tint)
○
Donner Magenta: pastel pink with a purple tint (ie magenta)
○
dsPink: pale pink
○
Prancer Purple (made from the liquid culture of Prancer purple): pale purple
○
spis Pink: bright pink (only slightly less bright than cupid pink from the plated culture)
○
Seraphina Pink: super pale pink
○
tinsel Purple:  pale pink
●
Chromoproteins after three days of incubation (one day at 37 °C and 2 days at room temp)
○
AE Blue-CBD-His tag: pale blue/green
○
Vixen Purple: never really showed color
○
AE Blue-: pale blue/green
●
Chromoproteins from cultures start on 7/29
○
gfas purple: very pale color
○
meff blue: very pale blue
○
meff red: no color
○
amaj Lime: slight tint of lime color
●
Conclusions from above color observations:
○
Disguard: gfas purple, meff blue (obtaining a new culture from Danny-his have color), meff red, amaj Lime, Vixen purple (from 8/3), AE blue (from 8/2--going to start a fresh culture from a plate), prancer purple (from 8/4 liquid culture), cupid pink (from 8/4 liquid culture), vixen violet (too pale- maintaing plate), leor orange (too pale), scrooge  orange (the one from iGEM, too pale), seraphina pink (too pale)
○
Keeping: ds Pink, donner magenta, blizten blue, prancer purple, ts Purple, vixen purple, spis pink, cupid pink, sc Orange (previous referred to as scrooge orange, on 8/4 but this is actually scOrange from the iGEM registry),  and tinsel purple. Also maintaing AE blue-CBD-His tag.
○
Before disguarding the chromoproteins, I looked at them under the UV light to test for fluorescence. AmajLime glows bright green under the UV light (pictured below on the right with a control culture that does not glow on the left). I'll do a peak excitation test next week.
IMG_1870[1].JPG
○
Plated the 12 selected chromoproteins on a single petri dish and heated them up to 85 °C then rehydrated and heated them up to 70 °C then rehydrated.
○
The three blue chromoproteins (Blitzen Blue, Drenzel Teal, and AE blue) used appear to be more sensitive to temperature: they lose their blue color after being heated and eventually become irreversibly clear. Future experiments will not exceed temperatures of 70 °C.
IMG_1871.JPG
Top row: VP Second row: dsPink, tsPurple, AE Blue Third row: TP, CP, PP,DT Fourth row: spisPink, scOrange, BB Fifth row: DM
IMG_1878.JPG
After heat
IMG_1880.JPG
Rehydrated
Monday, 8/8
●
Checked the sequencing data from last week and confirmed that the Gibson Assembly of AE blue, cellulose binding domain, and the flag-lumino-his tag (AE Blue-CBD-His) was successful! (ready to biobrick)
●
New liquid cultures of Prancer Purple, Tinsel Purple, Cupid Pink, Donner Magneta, Scrooge Orange, Blitzen Blue, Drediel Teal, Vixen Violet, spis Pink, asPink, scOrange, AE blue, AE blue-CBD-His, meff Red (for mini prep tomorrow), and tsPurple were started from existing plates.
●
Using a 5 mL culture of the AE Blue-CBD-His tag, I tried to test the CBD. First, I used the sonicator to lyse 2 mL aliquots of the AE blue-CBD-his tag culture then I added 1 mM  of EDTA (3.5 uL added from the .138 mM stock solution). After this, I dropped 15 uL of the lysate onto a cellose sheet and onto a glass fiber sheet. The intent was to wash each sheet with water after dropping 15 uL and then to check how much of the protein had stuck to the sheet, but I determined that the color wasn't concentrated enough for me to see if the CBD was working. Next time, I'll either use extracted protein or spin the culture down and resuspend in far less media (like 1 or 2 mL).
●
After allowing the chromoproteins tested on Friday to dry on the benchtop over the weekend, I put them on the heating block and heated them to 70 °C for 30 min then they were let to cool on the lab bench instead of rehydrating. The color still came back.
IMG_1881.JPG
Chromoproteins that had been left out on the benchtop over the weekend
IMG_1883.JPG
After being heated
IMG_1885.JPG
After being allowed to cool for 1.5 hours.
●
To further investigate whether or not the observed color change in the chromoproteins comes from dehydration or from the heat (or both). I spun down cultures of Prancer Purple and Scrooge Orange then put 20 uL of each type of color into two PCR tubes. I then added 30 uL of water to one of the two PCR tubes per color. This was done to test if the chromoproteins would lose their color in the presence of water. I placed these PCR tubes in a PCR machine and set it to cycle up from 50 °C to 75 °C every 5 minutes at 5 °C intervals. After this, I observed no color change in any of the tubes (those that had the additional 30 uL of water looked the same as those that were just the cell clumps). After this, I replaced the PCR tubes into the PCR and set the machine to run at 80 °C for an hour. When I removed the tubes, again I found that there was no color change.
IMG_1882.JPG
Chromoproteins before being placed in the PCR
IMG_1884.JPG
Chromoproteins after being placed in the PCR machine for 1 hour at 80 °C
●
To further investigate if dehyration is purely the cause for the color change I have been observing, I placed these same chromoproteins (Prancer Purple and Scrooge Orange) onto a small piece of filter paper which I then put in the -80 °C. Once the samples had been allowed to freeze for 20 min, I removed the filter paper and placed it in a lyophilizer (a device that dries a wet sample using a vacuum pump system) for an hour, taking pictures as time went on. I observed that Prancer Purple appeared pink when totally dehydrated and Scrooge Orange was almost impossible to see (it was practially white). After I removed them, I reydrated the colors and saw that while Scrooge Orange came back to its original color (as it does during the heat tests), Prancer Purple also comes back to its original color (this is not observed in the heat tests, it usually comes back as pink). These findings suggest that a combination of dehydration and heat contribute to the color change observed.
IMG_1886.JPG
Prancer purple and Scrooge Orange in the lyophilizer
IMG_1889.JPG
Prancer Purple ( on the left, both spots) and Scrooge Orange ( barely visible on the right) after they had been removed from the lyophilizer.
IMG_1890.JPG
Prancer Purple (top left) and Scrooge Orange (top right) that have been rehydrated. Note: the bottom two spots are also Prancer Purple (bottom left) and Scrooge Orange (bottom right) that were from a more diluted amount of chromoprotein cells. These bottom cells were not rehydrated.
●
A protein extraction of a 1 mL culture of AE blue - CBD- His Tag was performed on an existing culture from last week. This was done to verify that the lumio tag was expressing the fluorescence we were expecting. From this I also determined that the protein is in the soluble fraction (I ran both the soluble and insoluble fraction on a gel and saw a band at the correct size in the soluble fraction lane).
●
A large 1 liter culture of the AE blue-CBD-His tag was started using 4 mL of the 5 mL cultured started earlier in the day. This was put inside a shaker at 37 °C and will remain there until it looks like it's expressing enough of the chromoprotein to merit a large scale protein extraction. 50 uL of the remaining 5 mL culture was plated with 150 uL of LB media + Chloro in order to maintain a plate culture of that particular strain. 4 mL of LB was added to the remaining 1 mL original AEblue-CBD-his tag culture and allowed to incubate at 37 °C for the night.
●
Primers to Biobrick the DNA2.0 chromoproteins were designed and ordered using elimbio.
Tuesday, 8/9
●
In preparation for Gibson Assembly of the DNA2.0 chromoproteins, I did a miniprep on meffRed (iGEM chromoprotein) in order to have a plasmid to linearize with the primer I ordered yesterday. I also did a restriction digestion of 1 ug (of a 1 ug/uL solution) of each DNA2.0 chromoprotein in order to use the isolated protein sequence for the Gibson Assembly. The DNA2.0 restriction sites are designed so that EcoR1 and Nde1 can be used to remove the whole protein sequence from the plasmid.
●
I spun down the liquid cultures I started yesterday and replaced them using the cultures from plates. I put 20 uL (and 1 uL of .134 M EDTA) of each of the clumped cells into a PCR tube and heated them in the thermal cycler for 1 hour at 70 °C. Pictures are below, but most notebably Cupid Pink which has been shown when dry to lose its color in response to heat then regain it when cooled, now turned a dark purple color after being heated at 70 °C for 1 hour. Other colors showed mild responses to the heat treatment, but the orange and purple chromoproteins remained unchanged (as was observed yesterday).
IMG_1891.JPG
Proteins shown (before 1 hour heat treatment): asPink, Donner Magenta, Prancer Purple, TsPurple Scrooge Orange, Cupid Pink, Tinsel Purple, Ae Blue (appears green) scOrange, spisPink, Vixen Purple, Dreidel Teal
IMG_1895.JPG
Proteins after 1 hour heating treatment. Configuration the same as above. Note that Cupid Pink is now purple, AE Blue now appears blue (instead of green). Dreidel Teal now appears green. Spis Pink now appears yellow. Scrooge Orange now appears Orange (not yellow). tsPurple now appears darker purple. These color changes are not reversible.
●
Because the results of the above experiment were surprising, I repeated some of it using cells that had been pelleted from 8/4 and 8/3 (AE Blue only) and had remained on the lab bench in a sealed falcon tube. As I did before, I put 20 uL of clumped cells and 1 uL of .134M EDTA into a PCR tube. As before I heated the PCR tubes to 70 °C for 1 hour, and as before Cupid Pink turned purple. (pictured below)
IMG_1896.JPG
Proteins before heat treatment: Donner Magenta, Prancer Purple, Blitzen Blue (appears blue) asPink, Cupid Pink, AE Blue (appears blue) spisPink, Vixen Purple, Dreidel Teal (appears green)
IMG_1897.JPG
Proteins pictured after the heat treatment (same configuration as above). asPink appears darker pink, spis Pink appears to have lost color. Prancer Purple appears darker purple. Cupid Pink appears purple. Vixen Purple appears the same. All the blues appear lighter, especailly Dreidel Teal which looks lime green. Note: the blue that both AE blue and Blitzen Blue now appear looks identical to the blue the AE Blue on the previous test turned. These color changes are not reversible.
●
While it is now unclear what is causing the chromoproteins to change their color, it is appearing more likely that it has something to do with the denaturation of the protein.  This makes sense in context of some of the papers I have been reading about chromoprotein structure. For instance, http://www.jbc.org/content/281/49/37813.long (Structural Characterization of a Blue Chromoprotein and Its Yellow Mutant from the Sea AnemoneCnidopus Japonicus. Mitchell C. Y. Chan‡,1, Satoshi Karasawa§¶∥,1, Hideaki Mizuno§, Ivan Bosanac‡, Dona Ho‡, Gilbert G. Privé‡, Atsushi Miyawaki§,2 and Mitsuhiko Ikura, Holds Canada Research Chair in Cancer Structural Biology‡,3) state that chromoproteins are stablized by water molecules that congregate around hydrogen bonds with in the barrel portion of the protein.
●
The 1 liter culture of AE blue-CBD-Flag-Lumino-His was removed from the 37 °C shaker (set to 100rpm) after 32 hours of incubation and placed on a stir plate to incubate at room temperature for 15 hours. Because this culture was not showing the expected color, I made a new culture of LB + Chloro and innoculated it using a large amount of the cultures grown from the plate made on 8/7 of the AE blue-CBD-His tag. This was set on a stir plate at room temperature. According to Geneious, the size of this fusion protein is 39.6 kDa.
Wednesday, 8/10
●
A dark blue ring formed around the outside of the culture of AE Blue-CBD-Tag that was started Monday night. This culture was spun down at 4000 rpm for 10 min. The supernatant was discarded then the remaining cells were placed in the -20 °C.
●
New plates (Kan + IPTG) of the DNA2.0 chromoproteins were started using 25 uL of the liquid culture made yesterday, diluted with 175 uL of LB then set in the 37 °C.  500uL of these same cultures was used to cryostock the DNA2.0 proteins (BB, DM, VV, SP, LO, DT, VP, PP, TP, CP, SO--missing MP) and placed in the -80 °C box label iGEM 2016.
●
The PCR tubes that were left to dry yesterday did not appear to have regained any of the color they lost yesterday (pictured below)
IMG_1898.JPG
Chromoproteins pictured before heating (starting with the top left and working down) Scrooge Orange, scOrange, Blitzen Blue, Tinsel Purple ,tsPurple AE Blue, Cupid Pink, Prancer Purple, asPink,Donner Magenta, spisPink, Dreidel Teal (both the second and third rows are the same chromoprotein).
Untitled.png
Chromoproteins after 30 min in the PCR at 70 °C (note, the top row has been shifted to the right by one well position, otherwise it is the same order as above). The color appears unchanged.
●
Using the remaining cultures from the DNA2.0 chromoproteins after plating and cyrostocking, I  selected the brightest colors (prancer purple, cupid pink, dreidel teal, blitzen blue, vixen purple) and added 1 uL of .134 M EDTA to the clumped cells after discarding the supernatant. The remaining cultures that did not express color strongly enough to merit testing were placed on the shaker at room temperature.
●
The five chromoproteins selected above were placed in a PCR at 40 °C for 5 minutes then photographed. After this, they were placed in the PCR again, but at 45 °C for 5 minutes and photographed. This process repeated by stepping up the temperature by 5 degrees and photographing the chromoproteins after 5 minutes at that temperature until the final run at 100 °C.
IMG_1902.JPG
The five chromoproteins tested before undergoing the heat test.       Vixen Purple    Prancer Purple, Cupid Pink, Blitzen Blue Dreidel Teal
●
A powerpoint compling all of these images was made and placed in the Google Drive labeled Chromoproteins PCR test.
IMG_1914.JPG
After the 75 °C run, the Dreidel Teal looked as green as Blitzen Blue appears in the first image, and Cupid Pink turned dark purple.
IMG_1916.JPG
After 80 °C, Vixen Purple starts to lose its purple color. The two blue are completely yellow.
IMG_1918.JPG
After 85 °C, Vixen Purple is yellow as well.
IMG_1919.JPG
After 95 °C, all colors but Cupid Pink have turned yellow.
IMG_1920.JPG
After 100 °C, all chromoproteins have turned yellow
●
The same chromoproteins used in the PCR test above were placed on a piece of filter paper and heated in the oven. Only a small color change was detected in response to heat (left at 70 °C for two hours)
IMG_1904.JPG
Chromoproteins before heat was added: Vixen Purple    Prancer Purple, Cupid Pink, Blitzen Blue Dreidel Teal
IMG_1913.JPG
Chromoproteins after 2 hours at 70 °C. The filter paper could be helping to stablize the structure for some of these which is why we don't observe the same color change (?)
●
A large scale proteins extraction was done on the culture of AE Blue-CBD-Tag started on Monday and spun down earlier this morning and placed in the -20 °C. Using a small amount of LB and vigorous vortexing, the clumped cells were transfered from the four individual bottles they were in into one 50 mL Falcon Tube. The empty tube weighed 3.123 g.
●
The Falcon tube containing the cells was balanced then spun at 500 g (4500 RPM ) for 10 minutes. The supernatant was discarded and the falcon tube was weighed again: 14.034 g. Subtracting from the empty weight: 1.811 g of cells.
●
18 mL of Cell lytic B was added to the falcon tube  (add cell lytic B 10 ml/(g of cell)). 180 uL DNAse (10uL/(ml total solution)),  36 uLRNAse (2uL/(ml total solution)), and 2 uL lysozyme (10 mg/mL -> want final concetration to be .2 mg/mL).
●
After those four things were added, the falcon tube was taped to a vortex and shaken for 10 minutes at max speed then spun down for 15 minutes at max speed (12000 RPM). The supernatant was discarded and the clumped cells were used in the protein purification process according to the Protocol detailed elsewhere in this notebook (under label: Protocols > Large Scale Protein Extraction). Interestingly, the color of the solution during this process was a bright purple color.
●
An SDS page gel was run on each of the collected fractions during the protein extraction process (EQ, Wash 1, Wash 2, Wash 3, Elution Buffer) then stained with RUBY. In the Washes 1-3 and Elution Buffer, there appeared to be equal amounts of the protein we are attempting to purify at each stage, but the amount of other proteins present in the gel decreased as the washes increased and were minimal in the Elution Buffer fraction.  Dialysis to will be done tomorrow to concentrate the purer fractions.
●
Primers arrived for the Gibson assembly of the DNA2.0 chromoproteins into pSB1C3 with a purification tag (flag-lumino-his). PCR was done using these primers and left at 10 °C for the night.
Thursday, 8/11
●
The extracted protein which appeared purple yesterday now looks bright blue. The reason for this is unknown.
●
PCR extraction was done on the PCR from yesterday. The chromoproteins from the restriction digest done on Tuesday was run on a gel to ensure that the digest was sucessful. The gel showed two bands (as expected) with the larger part of the fragments were much brighter. Gel extraction proved unsuccessful. Another restriction digest was done and the Gibson Assembly to add a purfication tag and to move the DNA2.0 chromoproteins into the iGEM standard plasmid will be done tomorrow using just the product of the restriction digest. The concentration of the restriction digest insert should be 1 ug/50uL or .02 ug/uL or .043 pmol/uL which will be sufficent for the Gibson Assembly.
●
Cultures from DNA2.0 chromoproteins were spun down and put in PCR tubes in a similar fashion as yesterday. Starting at 40 °C in the thermal cycler for 5 minutes and increasing the temperature by 5 degrees until 100 °C. Pictured below:
IMG_1923.JPG
Chromoproteins pictured (before heat) Prancer Purple, Vixen Purple, Cupid Pink, Donner Magenta, Seraphina Pink,  Virginia Violet, Scrooge Orange, Leor Orange, Dreidel Teal, Blitzen Blue
●
Donner Magenta turns purple around 85 °C. Cupid Pink turned purple at 75 °C. Serphina Pink and Virginia Violet turn yellow at 90 °C. Leor Orange and Scrooge Orange turn yellow at 95 °C. Dreidel Teal and Blitzen Blue turn yellow at 80 °C.
●
The plates for the DNA2.0 chromoproteins that were done from liquid cultures yesterday were taken out of the 37 °C and placed in the 4 °C fridge. Some of the plates did not grow colonies: Prancer Purple, Dreidel Teal, Vixen Purple, Cupid Pink, Scrooge Orange. New cultures of these colors were started and will be plated tomorrow.
●
The culture of AE Blue-CBD-Tag that was started on Tuesday was spun down at 4000 rpm for 10 minutes. The supernatant was disgarded and put in the -20 °C for two hours. Then protein extraction was done on these cells following the protocol detailed in the protocols section of this notebook. 2.515 g of cells were extracted then suspended in 25 mL of cell lytic B, 250 uL of DNAse, 50 uL of RNAse, and 2.5 uL of lysozyme then shaken for 10 minutes on the vortex. After this, 25 mL of Equilibration Buffer was added to the cell lysate then it was poured into 3 seperation columns.  3 washes were done using Equilibration Buffer then the extracted protein was eluted in 3 mL of elution buffer. This is a slight change in protocol which was done in order to prevent the protein from washing through th column too soon as happened yesterday. Minimal amounts of extracted protein were visible in the washes of Equilibration buffer.
●
Snake skin dialysis was used to concentrate the extracted protein using a 1 L solution of 1 x TBS.
Friday, 8/12
●
The extracted protein that was dialysized yesterday was removed from the buffer and placed in 15 mL falcon tubes. 3 mL of the extracted protein from Monday's culture was concentrated using a filter column (spun at max in the mini centrifuge for 10 minutes then the flow through is disguarded and the column is placed upside down in a new tube then spun at 3.8 rpm for two minutes-- .5 mL of extracted protein yields 20 uL of concentrated protein).
●
The concentrated protein was then pippeted onto cellulose sheets that had wax wells printed onto them. These sheets were then placed in an oven at 75 °C and were observed to change from bright blue to purple in less than 30 seconds (more accurate time tests will be conducted next week). The sheets were then removed and allowed to cool to room temperature. The purple color lessened, but was still obviously more purple than the orginial color. After adding 2 uL of water to each well, the blue color came back. This blue color again changed back to purple when put in the oven at 75 °C. The same sheet was then placed in the oven at 95-100 °C and the protein turned yellow. When water was added too the protein, the yellow color became red. This red color turned back to yellow when put in the oven at 85 °C.
IMG_1956.JPG
AE Blue-CBD fusion protein pipetted onto a cellulose sheet before being exposed to 75 °C heat.
IMG_1959.JPG
AE Blue- CBD fusion protein after being heated to 75 °C
IMG_1965.JPG
Same AE Blue-CBD fusion protein as in the last two images after 2 uL of water have been added to each well.
IMG_1968.JPG
Crude image (from the flowthrough of pipetting the protein into the wells above) clearly comparing the color difference of hydrated (blue dot) vs. non hydrated (purple sections) AE Blue-CBD fusion protein after being exposed to 75 °C heat.
IMG_1970.JPG
The blue dot from above is now purple again after being placed back in the oven at 75 °C.
IMG_1971.JPG
Same sheet as above after being heated to 95-100 °C (the range is given because the oven was not quite preheated to 100 °C--further tests next week will clarify this).
IMG_1972.JPG
The yellow from above turns red in the presence of water
IMG_1974.JPG
The red color from above becomes fainter at 75 °C.
●
The results depicted above are promising. Next week will be spent better characterizing the color change observed. Currently, the wetness of the concentrated protein is making it hard to evenly coat the cellulose paper with it. As can be seen in the pictures above, the protein color tends to be concentrated around the ridges of the droplets instead of being evenly coated. To address this, I have placed a thin strip of cellulose paper in a epindorph tube containing the concentrated protein at the bottom and left it close for the weekend to allow the color to saturate the paper as it is drawn up through capillary action. Another approach will be to freeze dry the extracted protein and rehydrate it in a extremely small amount of water. This approach could run into issues because it has been documented (http://www.pharmtech.com/freeze-drying-protein-formulations) that in the process of freeze drying proteins that have been buffered in a phosphate salt solution, pH changes can occur to damage the proteins. As a result, only a small amount of the extracted protein will be freeze dried to ensure that this isn't a problem for our protein.
●
Additional tests on the cellulose binding domain on the fusion protein will be conducted next week. According to  http://www.ncbi.nlm.nih.gov/pubmed/10962023, the paper from which the sequence for the cellulose binding domain obtained from the iGEM registry originally came from, the cellulose binding domain has stronger binding capabilities when they incubated proteins and cellulose together overnight at 4 °C in 50 mM Tris HCl, pH 7.0 containing 10 mM CaCl2. After 6 days, they saw that the cellulose was permanently bound to their cellulose binding domains. This will be attempted next week to ensure that the protein is properly bound.
●
Further tests on the UV resistance and pH sensitivity of the AE blue-CBD fusion protein will also be conducted next week.
●
20 uL of the concentrated AE blue-CBD protein was placed in a PCR tube and heated for 5 minutes at 75 °C then for 5 minutes at 80 °C. While no color change was seen at the 75 °C heat treatment in the thermal cycler, the 80 °C treatment turned the protein green.
IMG_1961.JPG
Before any heat has been adminstered.
IMG_1964.JPG
After being at 75 °C for five minutes (note there are now bubbles in the tube because I turned the tube on its cap in order to try to take a picture similar to those taken on previous days this week, but the protein was too dilute and did not remain at the bottom like the cell lysate does, resulting in the formation of bubbles--there was no observable color change.)
IMG_1967.JPG
After 80 °C for 5 minutes, the protein now appears green.
●
Gibson assembly using the PCR products and restriction digests from last week was done on all DNA2.0 chromoproteins except for Tinsel Purple. The reason Tinsel Purple was excluded is because I ran out of PCR product for the meff Red vector that was designed to overlap with it because this vector was also used for Cupid pink, vixen purple, and prancer purple. I determined to leave tinsel purple out for now because it expresses color the least of these four and thus will be harder to tell if the Gibson assembly was sucessful or not. The Gibson product was transformed into NEB 5 alpha cells then plated on LB plates with chloramphenicol and left on the lab bench for the weekend.
●
New plates were made from liquid cultures of Prancer Purple, Dreidel Teal, Vixen Purple, Cupid Pink, Scrooge Orange were made and left on the lab bench to grow over the weekend.
Monday, 8/15
●
Performed heat testing on 11 DNA 2.0 chromoproteins using the thermal cycler starting at 55 °C and raising the temperature 5 °C every 5 minutes. Each PCR tube had 30 ul of the chromoprotein and 1 uL of .134 M EDTA. The 11 chromoproteins we tested were:
○
VixenPurple (VP)
○
PrancerPurple (PP)
○
TinselPurple (TP)
○
DonnerMagenta (DM)
○
CupidPink (CP)
○
SeraphinaPink (SP)
○
VirginiaViolet (VV)
○
LeorOrange (LO)
○
ScroogeOrange (SO)
○
DreidelTeal (DT)
○
BlitzenBlue (BB)
●
Here are pictures from our testing of the chromoproteins:
Before Heating
Before Heating                                               Order from left to right: BB, DT, SO, LO, VV, SP, CP, DM, TP, PP, Vp
After 55 degrees C
After 55 °C for 5 minutes                 Same order as above
After 60 degrees C
After 60 °C for 5 minutes          Same order as above
After 65 degrees C
After 65 °C for 5 minutes          Same order as above
After 70 degrees C
After 70 °C for 5 minutes          Same order as above
After 75 degrees C
After 75 °C for 5 minutes          Same order as above
After 80 degrees C
After 80 °C for 5 minutes          Same order as above
After 85 degrees C
After 85 °C for 5 minutes          Same order as above
After 90 degrees C
After 90 °C for 5 minutes          Same order as above
After 95 degrees C
After 95 °C for 5 minutes          Same order as above
After 100 degrees C
After 100 °C for 5 minutes          Same order as above
●
We will let these chromoproteins sit overnight to see if their color comes back we expect them to remain this yellow color though
●
Successful Gibsons from Friday:
○
VixenPurple
○
DonnerMagenta
○
LeorOrange
○
ScroogeOrange
○
PrancerPurple
●
We will be re doing the other Gibsons that did not work from Friday (All but the 5 above)
●
Created 13 liquid cultures to incubate overnight from plates of the following by adding 5 ul LB+Kan and 5 ul of IPTG to each and a stab from a plate:
○
VV
○
BB
○
SP
○
LO
○
SO
○
TP
○
CP
○
PP
○
DM
○
CP
○
VP
○
MP
○
AE Blue-CBD-His Tag (We used LB+chlor for this and no IPTG)
●
We did a PCR using the primers we designed for the Gibson on the meffRed plasmid and we performed PCR extraction and the results looked contaminated and had a very low yield so we are running the Gibson assembly from Friday on a gel to see if any of the product ligated together
●
The gel revealed some confusing results. It was difficult to tell exactly what size things were because too much ladder was loaded, but it looked like maybe two pieces of the vector could have ligated together? It appeared as though there was a band at the 4000 bp mark.
●
Looked at the heat sensitivity of a concentrated version of AE blue-CBD-Tag
IMG_1989.JPG
Before Heat
IMG_1996.JPG
Color change occurs at 55 °C and gets more pronounced at temperature is increased
IMG_1999.JPG
At 80 °C
IMG_2002.JPG
Color has returned to blue once put at room temperature
Tuesday, 8/16
●
Performed PCR on HisTag using 5 different primers which will allow us to use the His tag to overlap with the different DNA 2.0 chromoprotein sequences
○
2.5 ul of each primer (F and R)
○
1 ul of 1ng/ul His tag
○
25 ul Masterix
○
Bring Volume up to 50 ul with water (19 ul)
●
Performed PCR on meffRed vector to linearize the vector using 4 different primers which will allow us to overlap the linearized vector with different DNA 2.0 chromoprotein sequences
○
PCR was run on at two different annealing temperatures (70 °C and 71 °C) to see which was most effective
●
A PCR cleanup was performed on both the PCR from the Histag and the meffRed and the nano drop showed concentrations of DNA in each that showed the PCR was effective. The annealing temperature of 70 °C produced higher concentrations of DNA after the PCR extraction so we decided to move forward with this as the annealing temperature if we need it in the future and we are using the PCR extraction products from the 70 °C to perform a Gibson assembly with.
●
Because our HisTag and meffRed PCR were effective (nano drop showed significant concentrations after PCR Cleanup) we performed a Gibson assembly using the 12 different DNA 2.0 chromoproteins and the meffRed linearized backbone and the HisTag. The Gibson assembly prodcut will be transformed into T7 express LysS cells and plated on LB +Chloro plates.
After Sitting Overnight
After sitting overnight the chromoproteins did not regain any of their color. This means the color change was irreversible
Wednesday, 8/17
●
The plates from the transformation of the Gibson product grew a lot of colonies and over grew which hopefully means the Gibson was successful. Because of the over growth of the colonies we made new plates with LB+chloro and plated a transformation again in a 1:40 dilution and placed into the 37 °C incubator overnight to grow
Thursday, 8/18
●
The plates we transformed grew overnight and less dense than yesterday but still pretty dense. We think the reason for the large growth is because we used T7 express LysS competent cells instead of the regular T7 express competent cells which increases growth. Individual colonies were picked and started liquid cultures with these colonies. The plates were then placed in the 4 °C fridge for storage
●
Spun the 13 cultures down that were made Monday. The following cultures worked:
○
CupidPink
○
Prancerpruple (barley worked)
●
Because not all of the cultures worked we redid all but CupidPink and placed the cultures to incubate overnight the cultures we redid in LB+Chlor with IPTG were:
○
VV
○
BB
○
SP
○
LO
○
SO
○
TP
○
PP
○
DM
○
VP
○
MP
○
AE Blue-CBD-His Tag (We used LB+chlor for this and no IPTG)
●
We performed 12 restriction digests on the Gibson products of the DNA 2.0 chromoproteins from Tuesday 8/16. The protocol we used was:
○
1 ul EcoR1 Prefix
○
1 ul Pst1 Suffix
○
1 ul Gibsoned product (12 different of these)
○
5 ul Cutsmart
○
42 ul MilliQ water
●
We than ran these restriction digests at 37 °C for an hour and then 65 °C for 20 minutes.
●
We will run both the restriction digest and Gibson product on a gel to compare them and see if our Gibson was successful which we will determine if the restriction digest band is at the proper length and the Gibson product is at the proper length
●
After looking at our gel we do not think the restriction digest worked because the band was in the same location as the band on the Gibson product that was run without being digested. We think this occurred because there is something wrong with the Cutsmart we are using because other people in lab are having issues with outsmart
●
Tried to get specter readings from CupidPink that was spun down this morning from liquid culture. Unfortunately the specter readings were not successful because we need to do extracted protein instead of the cells straight fro culture
○
We also heated some of the cupidPink to the color change of purple to try and compare the two specter readings. Instead of taking 5 min at 75 °C to make it purple so we turned up the heat to 80 °C for 10 min until the color changed was observed
○
We then heated new aliquots of the same Cupid Pink cultures and heated them for 15 minutes at 75 °C and observed the color change we expected. This makes it seem like we might be able to detect color change if we treat with heat for a long time.
●
We then decided to do more heat testing with CupidPink because we wanted to see if we left the chromoprotein in the thermal cycler longer than 5 minutes but at a lower temperature if the same color change from pink to purple would occur.
○
We saw that after an hour at a lower temperature the chromoproteins did not reach as much of a purple color as at a higher temperature for less time but it did reach a more purple color than the original CupidPink
CupidPink Color Test
CupidPink chromoproteins that had been sitting for longer at a lower temperature did not fully change color like the ones that sat at a higher temperature for less time
Friday, 8/19
●
Ran colony PCR from the plates with Gibson product on them and picked same colony that we picked from for the culture yesterday
●
Mini prep of the cultures started from the Gibson product yesterday
●
Ran colony PCR on the gel to see if the transformation was successful
●
Created NASA sticker using the AE blue fusion protein to coat a cellulose sheet as the blue background of the NASA sticker and added the lettering and red swoosh with stickers.
NASA Sticker
Here is a picture of the sticker we created with our AE Blue fusion protein. The blue background was created with a cellulose sheet and pipetting the AE blue protein onto it
●
We placed our sticker onto a beaker and poured boiling water into it. The water changed the sticker color from blue to purple and then we poured out the hot water and poured cold ice water in instead and the color changed back to blue. This sticker prototype shows a reversible color change in the presence of heat. Videos of this process are uploaded in the googleDrive
●
The gel we ran of the colony PCR ran off the end so we re made and re ran a gel
●
The re run gel showed only one band in the correct number of base pairs on the gel so we think something might have been wrong with our PCR and we pipetted the wrong things and will re do on Monday
chromoprotien_8_19_16.jpg
Here is our gel from today. The only band that kind of shows up is on the third chromoprotein which is VP (VirginiaPurple). Because this band shows up but is pretty faint we believe there was something wrong with the PCR we ran today and will re run it on Monday
Monday, 8/22
●
Created 12 cultures with LB+Kan+IPTG of the 12 DNA 2.0 chromoproteins and places them into the incubator shaker overnight to grow up the 12 cultures were
○
VV
○
BB
○
SP
○
LO
○
SO
○
TP
○
PP
○
DM
○
VP
○
MP
○
DT
●
Spun down the cultures made from Friday with the DNA 2.0 chromoproteins and 4 of them showed color. The 4 chromoproteins that showed color were:
○
TP
○
PP
○
VV
○
CP
●
Created 12 cultures with LB+chloro of the 12 DNA 2.0 chromoproteins that were Gibsoned with a flag-lumio-his-tag. We put these in the incubator shaker overnight and we hope to see color out of these cultures tomorrow.
●
Ran the same PCR from the mini prep of the plasmid run on Friday because of the interesting gel results and only seeing one band we thought maybe there was a mistake in the PCR we ran on Friday.
●
We found out that our plates we plated from the Gibson transformation were not showing color because we used T7 LysS cells instead of T7 or NEB 5 alpha so we decided to re transform our cells in NEB 5 alpha to hopefully show our color when plated and show our Gibson worked. We labeled our tubes 1-12 and the table below shows what the number corresponds to in terms of the chromoprotein
A
B
1
Tube number
Chromoprotein Code
2
1
BB
3
2
MP
4
3
DM
5
4
VP
6
5
VV
7
6
TP
8
7
SP
9
8
LO
10
9
CP
11
10
DT
12
11
SO
13
12
PP
Chromoprotein Table
●
The new PCR product we ran today was run on a gel an strong bands were seen where we expected to see them. This helps to confirm our Gibson was successful and we hope to send in these DNA 2.0 chromoproteins with the flag lumio his-tag for sequencing tomorrow
8.22.16_Chromoprotein Gel.jpg
The gel from the re done PCR shows the strong band at the correct number of base pairs helping to strengthen the thought that our Gibson was successful on these chromoproteins
●
We started looking at how pH changes affected our chromoproteins. We performed a serial dilution to obtain a pH range from 1 to 14 at each integer value. We then pipetted some of our purified AE-blue+CBD+tag onto a well plate lid and mixed with each of the pH solutions in different wells. The pH of 1 and pH of 14 changed the blue chromoprotein to a clear/yellowish tint but the other chromoproteins did not change color they faded a bit because of the added water diluting them. We think because we did not see any color change other than at pH of 1 and 14 that the serial dilution was maybe not correct and we will look into that tomorrow. One interesting thing to note is that when the chromoprotein is mixed pH 1 HCl it loses its color but when pH 14 NaOH is added it regains its color. This could be an interesting observation to explore in the future.
Before pH testing
Here is a picture of the chromoproteins before we added acid or base to them. The number in each well indicate the value of pH we are adding to each well
After pH Testing
This pictures shows what happened after we added the pH indicated in each circle to the chromoprotein. We plated 20 ul of each chromoprotein and added 10 ul of the pH solution. At the pH of 1 and 14 changes were seen but not at the other pH's this makes us think our serial dilution was wrong or we needed to change how much of the pH solution we added to the chromoprotein.
●
Started 500 mL liquid culture of CP
Tuesday, 8/23
●
The plates from the transformation yesterday do not have much growth at the 1:40 dilution they were plated at. We put them into the incubator longer to see if they would show growth over a longer time. 6 of the plates grew colonies and we will grow liquid cultures from these and perform a colony PCR. The other 6 plates did not have colony growth but the plates were bubbly so we think it may just be hard to see the growth. The SOC transformation from yesterday were placed into the incubator this morning and then moved to room temperature in case we need to replate some of our transformations. 7 of the plates showed colonies after extra incubation and we started liquid cultures from them and also run a colony PCR from them. 5 of the plates did not grow and will be replated in a 1:4 dilution to hopefully see growth tomorrow. The following 5 colors were replated:
○
PP
○
SO
○
TP
○
VV
○
MP
●
After we ran the colony PCR we ran a gel on the product to see where the bands were and to hopefully confirm we did our Gibson and transformation correctly
●
The gel showed the correct size for the colony PCR we ran which confirms we had performed looks like the Gibson and transformation were done correctly
●
We spun down the cultures we started yesterday from the DNA 2.0 chromoproteins and saw color in most of the cultures. The following cultures showed cell growth but did not show color:
○
VV
○
TP
○
MP
○
SP
○
DM
●
Both SO and LO are expected to be orange in color and it is hard to tell wether or not they worked or did not show color.
●
The remaining 7 cultures that showed color and the 5 that didn't we took about 20 ul of the pellet and placed into a PCR tube to perform heat testing on. We also placed 1 uL of .134 M EDTA into each PCR tube. The heat testing was done in the thermal cycler by putting the PCR tubes in for 30 minutes at 55 °C and increasing the temperature 5 °C after each 30 minutes and taking pictures after each temperature step to see if there was a change in color. Because of the interesting results seen that when adding heat the chromoproteins that had no color gained the color they were supposed to have we decided to try and recreate this process again tomorrow.
Before Heating
This picture shows the chromoproteins before any heat was applied to them. The top row is the chromoproteins that expressed color from culture and in order from left to right are: BB, DT, VP, PP, CP, SO, LO. The bottom row are the 5 chromoproteins that did not express color from culture and in order from left to right they are: VV, MP, SP, TP, DM.
After 30 minutes at 55 degrees C
This pictures sows the chromoprotins after their first heating step at 55 °C for 30 minutes. Same order as above picture. A color change is not seen after this temperature step.
After 30 minutes of 60 degrees C
This picture shows the chromoproteins after the 60 °C time step for 30 minutes. Same order as above. CP has changed from pink to purple after this temperature step. Also the bottom row which are the chromoproteins that originally did not have color now have gained the color they should have had in culture. This is an interesting observation and result and could give rise to a way to gain color in chromoproteins even if they do not have color in an initial liquid culture.
After 30 minutes at 65 degrees C
This picture shows the chromoproteins after the 65 °C time step for 30 minutes. Same order as above. CP has darkened in its purple color. The bottom row has intensified in color to the color that the original cultures were expected to grow to. This reveals that with some added heat sometimes the cultures can gain color even if they did not originally have it.
After 30 minutes at 70 degrees C
This picture shows the chromoproteins after the 70 °C time step for 30 minutes. Same order as above. CP has continued to darken in color but no other changes in the other chromoproteins are seen.
After 30 minutes at 75 degrees C
This picture shows the chromoproteins after the 75 °C time step for 30 minutes. Same order as above. Little change in color is seen after this time step. DM has started to lose some of the color it had gained in previous steps.
After 30 minutes at 80 degrees C
This picture shows the chromoproteins after the 80 °C time step for 30 minutes. Same order as above. DM has lost all its color that it had previously gained, and VP is getting lighter in color.
After 30 minutes at 85 degrees C
This picture shows the chromoproteins after the 85 °C time step for 30 minutes. Same order as above. VP has lost its color and PP is beginning to lighten in color. MP changed from a previously purple color to a pink color.
After 30 minutes at 90 degrees C
This picture shows the chromoproteins after the 90 °C time step for 30 minutes. Same order as above. Every chromoprotein has lost color except for CP which is fading in its purple color.
After 30 minutes at 95 degrees C
This picture shows the chromoproteins after the 95 °C time step for 30 minutes. Same order as above. At this point all the colors have lost their color.
●
This temperature experiment we will compare to the previous ones to see if the temperature for a longer period of time changes the color at a lower temperature or not. We will also repeat this experiment tomorrow to see if the colors in culture that were initially colorless can gain color when heated like we saw today.
Wednesday, 8/24
●
Colonies grew on the transformations that were replated from the transformation at the 1:4 dilution
●
The liquid cultures started from the transformation plates grew cells but without color so they were placed into the 25 °C incubator/shaker to see if this would help color growth
●
Performed small scale protein extraction on the CP culture that was started two days ago but taking 1 mL out of the 500 mL culture
●
We ran both the soluble and insoluble fraction of the protein extraction on a gel and saw a band at the correct size in the soluble fraction lane
082316_CupidPink.jpg
This gel has a dark band on the soluble fraction lane in the first row which is where our chromoprotein plus his tag should be. This is promising because it confirms our chromoproteins have the his tag present in them we Gibsoned in.
●
We now performed a large scale protein extraction on the CP culture. We spun them down in the centrifuge at 4000 rpm for 10 minutes (make sure tubes are balanced). We then poured of the supernatant and poured a bit of LB into one of the tubes and vortexed it. Then once all the pellet was off the bottle the LB and cells were transferred to the next bottle and so on until the LB and cells were poured into a 50 mL tube. Spin in centrifuge at 4500 rpm for 10 minutes. Weighed pellet after and added 12 mL of cell lysate because pellet was 1.2 g (add 10 mL/g of cell lysate). Then added 100 ul of lysozyme and 50 units/mL of benzonase (can also add RNAse and DNAse). Then vortex until pellet is resuspended and place onto vortex for 10 minutes (remember to tape so it doesn't fly off). Spin down at 11,200 rpm for 10 minutes in centrifuge. Then follow the protocol on Jesica's lab bench to wash the column and add the protein.
○
After we did the three washes we noticed there was not a pink color in them so we decided to re run the gel from this morning but instead with the extracted protein washes to see if the chromoprotein was the protein we extracted. The gel shows that our chromoprotein is present in our protein extraction (large dark band) which is exciting but also interesting because there was no color seen in the protein extraction
082316_CupidPink2.jpg
Here is the gel from the washes and elutions. The order on the gel is ladder, wash 1, wash 1, wash 2, wash 3, elution. The dark band in the elution well is the right length for our chromoprotein plus tag which confirms our chromoprotein is present in our extraction.
●
Sent in the forward for all 12 chromoprotein sequences from DNA 2.0 with both the chromoprotein and flag-lumio his-tag.
After sitting overnight
Here is a picture of the 12 chromoproteins we ran yesterday after sitting overnight. They did not regain or change color.
●
We are re running the temperature experiments on our 12 DNA 2.0 chromoproteins today. We kept the liquid cultures from yesterday at room temperature and added EDTA to keep them overnight. Today we re spun them down to get a pellet and then we pipetted 21 ul (because EDTA was already added) into each PCR tube and ran temperature tests starting at 55 °C to 95 °C in 5 °C increments for 30 minutes on each temperature. We also re ran the tubes from yesterday to see if anything interesting would happen with their color and if they would regain any of it. We don't expect them to because they went to 95 °C yesterday and lost their color which we think means they were denatured. Pictures of the experiment are shown below:
Before heat new run of 12 chromoproteins
This picture is taken before any heat is added. The 7 tubes in the top row and 5 tubes in the bottom row are the 12 chromoproteins we are running from the culture from yesterday. The top row is the chromoproteins that expressed color from culture and in order from left to right are: BB, DT, VP, PP, CP, SO, LO. The bottom row are the 5 chromoproteins that did not express color from culture and in order from left to right they are: VV, MP, SP, TP, DM.
Before heat all chromoproteins
This picture shows each set of 12 chromoproteins. The top two rows are the ones we are running today to see if they have a color change similar to the ones we ran yesterday. The bottom two rows are the ones we ran yesterday and we are seeing if they change colors at all (we do not expect them too because we believe they were denatured at the high temperature we ran them at yesterday). The top row in each set are the chromoproteins that showed color from culture and are in the following order from left to right: BB, DT, VP, PP, CP, SO, LO. The bottom row in each set are the 5 chromoproteins that did not express color from culture and in order from left to right they are: VV, MP, SP, TP, DM.
After 30 minutes at 55 degrees C new 12 chromoproteins
This picture shows the 12 chromoproteins we started today after the 55 °C time step for 30 minutes. Same order as above. The bottom row of chromoproteins that did not have color from the original culture are beginning to gain the color they were expected to have.
After 30 minutes at 55 degrees C all chromoproteins
This picture shows all the chromoproteins after the 55 °C time step for 30 minutes. Same order as above. The bottom set have not regained any color as we expected.
After 30 minutes at 60 degrees C new 12 chromoproteins
This picture shows the 12 chromoproteins we started today after the 60 °C time step for 30 minutes. Same order as above. The bottom row has gained color even more than before which replicates the findings we made yesterday that temperature can help cells express their color if they do not show it initially in culture.
After 30 minutes at 60 degrees C all chromoproteins
This picture shows all the chromoproteins after the 60 °C time step for 30 minutes. Same order as above. The bottom set have not regained any color as we expected.
After 30 minutes at 65 degrees C new 12 chromoproteins
This picture shows the 12 chromoproteins we started today after the 65 °C time step for 30 minutes. Same order as above. CP has turned to purple.
After 30 minutes at 65 degrees C all chromoproteins
This picture shows all the chromoproteins after the 65 °C time step for 30 minutes. Same order as above. The bottom set have not regained any color as we expected and we no longer tested them after this temperature.
After 30 minutes at 70 degrees C new 12 chromoproteins
This picture shows the 12 chromoproteins we started today after the 70 °C time step for 30 minutes. Same order as above.
After 30 minutes at 75 degrees C new 12 chromoproteins
This picture shows the 12 chromoproteins we started today after the 75 °C time step for 30 minutes. Same order as above. Bottom row is losing color they had gained.
After 30 minutes at 80 degrees C new 12 chromoproteins
This picture shows the 12 chromoproteins we started today after the 80 °C time step for 30 minutes. Same order as above. VP is fading in color and changing to pink.
After 30 minutes at 85 degrees C new 12 chromoproteins
This picture shows the 12 chromoproteins we started today after the 85 °C time step for 30 minutes. Same order as above. VP has lost all its color and PP is almost colorless.
After 30 minutes at 90 degrees C new 12 chromoproteins
This picture shows the 12 chromoproteins we started today after the 90 °C time step for 30 minutes. Same order as above. Everything has lost color except for CP.
After 30 minutes at 95 degrees C new 12 chromoproteins
This picture shows the 12 chromoproteins we started today after the 95 °C time step for 30 minutes. Same order as above. Everything has lost color.
Thursday, 8/25
After sitting overnight
Here is a picture of the chromoproteins we tested yesterday after they have been sitting overnight. They did not regain any of their color because they were denatured at the high temperature.
●
The liquid cultures we grew up overnight from the transformations that had been plate with the DNA 2.0 chromoproteins plus the flag-lumio his-tag did not show any color but when spun down did show cell growth
●
The protein extraction we performed yesterday does not show any of the color that is should. We believe this is because the flag-lumio-his-tag was inserted onto the wrong terminus (C terminus) on the chromoprotein which might be where the chromophore (what gives the chromoprotein color) is attached and so the tag attaches here and doesn't allow the color to show.
○
Because we believe this tag is interacting with the chromophore on every DNA 2.0 chromoprotein this would be why the cultures we grew up overnight were not showing any color
●
Because we think the tag is on the wrong terminus we are going to use Enterokinase to cleave off the flag-lumio his-tag and see if the color then appears.
Friday, 8/26
●
Discovered there was a secondary restriction site in our DNA 2.0 chromoproteins that had not shown up on one of our softwares. 2 of the restriction sites are the same so we were cutting out ~50 base pairs of our chromoprotein unintentionally which is why the color was not being seen when we did the Gibson assembly. We will now use this information to create new primers and insert the fragment of the chromoprotein directly into our backbone and use the his-tag to PCR into next to our chromoprotein. Hopefully we will then see color when the Gibson assembly is performed.
●
Ordered primers for our Gibson assembly of our DNA 2.0 chromoproteins and flag-lumino his-tag. The primers are for the vector forward and reverse and the his-tag forward and reverse. We also ordered primers to linearize our chromoprotein with its ribosome binding site attached which then can be directly inserted into our backbone using Gibson assembly.
Monday, 8/29
●
Performed spectra readings the extracted and purified and concentrated AE blue - CBD - His-Tag. Blanked nano drop with the flow through from the concentration of the extracted protein. The blank did not work so instead we blanked with 1X TBS buffer and this worked a lot better. We saw a peak at ~ 590 nm which is the color orange. It is showing a peak here because it is absorbing blue and reflecting orange which is the reading we are seeing.
●
We then heated our AE blue-CBD-His-tag for an hour at 50 °C in order to gain a color change and perform spectra readings on this
○
After and hour no color change was seen so they were put in for another hour at the same temperature and no color change was observed
○
They were then put in at 70 °C for an hour and when taken out one of the AE blue was murkier blue and the other was a green color
●
We created more TBS to use in dialysis when purifying the chromoproteins
●
The primers we ordered on Friday came in so ran a PCR of all 12 chromoproteins to isolate them as a fragment and also ran a PCR on the His-tag to add the Gibson tails
●
Created a liquid culture of meffRed which we will mini prep tomorrow for use as our backbone
Tuesday, 8/30
●
Mini prepped the meffRed and then ran a PCR and PCR cleanup to isolate the vector and promoter we wanted
○
The PCR cleanup when nano dropped showed no product so something went wrong with the PCR or extraction because DNA was present after the mini prep.
○
Because of the no yield we created a liquid culture again of meffRed which we will mini prep and PCR again tomorrow
●
Ran gel on our 12 chromoprotein PCRs and 5 His tag PCRs
○
First gel well order: Ladder, MP, SP, SO, LO, BB, DT, TP, CP, VP, PP, DM
○
Second gel well order: Ladder, VV, 2, B1, B2, 1, R
■
two gibsons to be done with 2 and B1 because not sure if we can
●
When we went to perform a gel extraction the chromoproteins did not have bands at the correct place on the ladder and the his tag PCR products were not even on the gel. We think the settings were not right on the PCR yesterday which resulted in these gel results. We re did all 12 chromoprotein PCRs and the 5 his tag PCRs and will run them on a gel and extract them tomorrow. We ran two His-tag PCRs and increased the annealing temperature by 1 degree for the second set to see if this would help it work better because we did not even see bands for the his-tag on our gel
Wednesday, 8/31
●
We ran a gel on our PCR of our chromoproteins and his-tags from yesterday. The order on the gel was:
○
First gel well order: Ladder, PP, LO, SP, MP, VV, VP, SO, BB, CP, TP, DM
○
Second gel well order: Ladder, DT, H1-H5 for lower temperature PCR, H1-H5 for higher temperature PCR
●
Re did our PCR on meffRed and then ran a PCR clean up which gave us product unlike the last time we ran it.
●
Performed gel extraction on our chromoproteins and on the 5 His-tags twice. The nano drop showed concentrations that were reasonable and had decent 260/280 ratios. We then performed a Gibson assembly using our meffRed backbone, inserting the chromoproteins directly as a fragment, and the PCred his-tag.
●
Concentrations for Gibson assembly and what we added:
A
B
C
D
E
F
1
Name
Concentration (ng/ul)
BP
pmol/ul
ul of each
Water
2
MP
12.6
700
0.027
5
0
3
SP
13.6
700
0.029
4
1
4
LO
14.4
700
0.031
4
3
5
VV
7.7
700
0.017
6
1
6
DM
9.4
700
0.020
5
2
7
BB
16.1
700
0.035
3
4
8
SO
10.3
700
0.022
5
1
9
VP
25.7
700
0.056
2
5
10
PP
19.6
700
0.042
3
4
11
TP
24
700
0.052
2
5
12
CP
28.3
700
0.061
2
5
13
DT
5.7
700
0.012
10
-6
14
H1
42.9
100
0.650
1
15
H2
16.7
100
0.253
2
16
H3
26.6
100
0.403
1
17
H4
35.3
100
0.535
1
18
H5
30
100
0.455
1
19
MR_ALL
9.5
2070
0.007
5
20
MR_ALL
23.4
2070
0.017
2
21
MR_ALL
23.5
2070
0.017
2
22
MR_ALL
16.6
2070
0.012
3
23
MR_SO
18.4
2070
0.013
3
Table8
●
Our Gibson nano drop showed the concentration of DNA to be extremely high which is what we would expect from a Gibson assembly. We then took our Gibson product and transformed it into competent E. Coli cells and plated it. Hopefully our colonies tomorrow will show color.
Thursday, 9/1
●
The plates from the transformation grew colonies but they have not shown color yet so they were placed back into the incubator to see if color would result
●
Ran colony PCR and created liquid cultures from the same colony from each transformation plate. VR and VF2 was used to amplify the chromoprotein+His-tag. The colony PCR may have been messed up because too much of each primer was added to each of the tubes. We will run a gel of the colony PCR to image and see if the correct region was amplified.
○
Gel order: Ladder, TP, MP, PP, VP, VV, LO, SO, BB, DT, SP, CP, DM
●
The gel showed interesting results with the thing we amplified at around 300 bp. We are thinking something went wrong with the PCR so we will re do the PCR tomorrow and see if our results differ
09-01-16-chromor-pcr.jpg
Here is the gel we ran of our colony PCR. From left to right the order is:  Ladder, TP, MP, PP, VP, VV, LO, SO, BB, DT, SP, CP, DM. Most of the samples look to be around 300 base pairs (bp) where they are amplified. The other 3 samples were around 1,200 bp. Both of these numbers of bp are not what we were expecting to see from the PCR. We were expecting our chromoprotein plus his-tag to be around 1000 bp.
●
One plate (TP) began to show colorful colonies so we made a liquid culture from one of the colored colonies to see if that would also show color
●
The 12 plates were placed at room temperature overnight to see if color would result
Friday, 9/2
●
We spun down the 12 liquid cultures of the DNA 2.0 chromoproeins + his-tag and these cultures did not show color but cells were present. They will be placed at room temperature oscillating to see if color growth will be seen after a couple more days
●
The culture we made for TP was spun down and color was seen. This helps to confirm our Gibsoned work but we will colony PCR from a colorful colony today to see if that also helps confirm our results.
TP Colored Cells
Here is the result when TP+His-tag was spun down. Color is seen in the cells that grew from the colorful colony we picked from yesterday.
●
After sitting out overnight 3 more plates in addition to TP gained color. Colored colonies were also seen on VP, CP, and PP.
○
These 4 colors (TP, VP, CP and PP) all used the same primers for both the vector and the his-tag and chromoprotein so there may be something wrong with the other 8 primers that we need to fix to get them to show color.
●
We will start liquid cultures from the colored colonies and also run the colony PCR from the colored colonies. We used VF2 and VR as our primers in the PCR because our backbone is a iGEM backbone and VF2 and VR should amplify the region in this backbone which should be our chromoprotein and hi-tag. We ran out of chloro when making the liquid cultures so only 8 were made. LO, SO, DM, and VV did not get made into liquid cultures but will be once we make more chloro. We put the liquid cultures in the incubator for today but will take them out before we leave and put them at room temperature to incubate over the weekend. We made more chloro and were able to get all 12 chromoproteins in the incubator to grow up in liquid culture
●
We ran the colony PCR on a gel to see if the correct region was amplified with our chromoprotein and his-tag
○
Gel order: Ladder, DM, VV, SO, LO, DT, VP, SP, CP, TP, PP, MP, BB
●
Our gel showed our colony PCR was successful as the fragment amplified was around 1000 bp.
9-2-16-chromo-his-2.jpg
This is the gel scan of the colony PCR we ran. The order is Ladder, DM, VV, SO, LO, DT, VP, SP, CP, TP, PP, MP, BB.
●
We then ran a PCR clean up on the four that showed color and did a PCR clean up
Tuesday, 9/6
●
Nano dropped the PCR clean up from Friday on the 4 chromoproteins that had shown color (TP, PP, CP, TP) and it showed significant concentrations
●
Will now send in these 4 PCR cleanups for sequencing to confirm we got the right sequence when we Gibsoned the product together. Sent in using VF2 for sequencing
●
Looked at the plates that had been left to grow on the bench over the three day weekend. The color on the 4 plates (VP, CP, PP, TP) was more intense but the other plates showed no color. Also spun down the liquid cultures made on Friday and color was seen in the 4 (VP, PP, CP, TP) only.
●
Ran small scale protein extraction on 1.5 ml of the liquid cultures of the 4 chromoproteins that worked. We then ran a lumio green detection kit on the chromoproteins and ran them on an SDS gel. When we imaged them we saw our protein which helps confirm our flag-limi his-tag was successfully Gibsoned into our chromoprotein
9-6-16-chromo-4-color.jpg
Here is the lumio gel we ran. the order from left to right: ladder, CP, PP, VP, TP. The proteins are at ~25 kDa which is where we would expect our chromoproteins to be and they show up which helps confirm our flag lumio his-tag is present
●
Because this gel showed our flag-lumio his-tag was present we created 250 ml liquid cultures of each of the 4 color producing chromoproteins (VP, CP, TP, PP) which we put into the 37 °C incubator/shaker overnight and will move to the 25 °C incubator/shaker for another day before doing a large scale protein extraction on these 4 chromoproteins
●
Created liquid cultures of TP, VP, CP, and PP that will grow for 6 hours and then we will mini prep this afternoon
●
Mini prepped cultures created this morning and got ~40 ng/ul when nano dropped
Wednesday, 9/7
●
Because only 4 of our chromoproteins showed color and on plates and in liquid cultures and these 4 chromoproteins were Gibsoned with the same His-tag and backbone we decided to re do the Gibsons for the other 8 chromoproteins that did not work.
●
We PCRed the meffRed backbone with the primers previously designed and also PCRed the His-tag depending on the primers for each chromoprotein
●
We then performed a PCR clean up on the meffRed PCRs and His-tag PCRs
●
We wanted to see if we needed to re do our Gibson or our primers so we took 1.5 ml of our liquid cultures for the 8 that did not work and performed a small scale protein extraction and then ran a lumio green detection test and gel. If the lumio gel shows our protein that this means our Gibson was successful but we need to re order primers because they were not correct and somehow made the color our our chromoproteins go away. If this is the case then we will order sequencing to see what exactly is going. If our lumio gel shows no protein fluorescing then we know our Gibson failed and we should re do it.
9-7-16-8chromo-failed-real-1.jpg
Here is the lumio gel we ran today of our 8 chromoproteins that had not shown color in culture or on plates. The order from left to right is: Ladder, SO, DM, VV, DT, LO, SP, BB, MP. Because there was protein present in the gel once images we think our Gibson was successful but something went wrong with out primers which removed or inhibited the color producing part of our chromoprotein.
●
We are running heat tests on the cell lysate of the 12 chromoproteins + His-tag to see if the 4 colored chromoproteins lose color at a different temperature now that they have a His-tag and are in psb1C3. We are heat testing the other 8 chromoproteins to see if they gain color like we saw before they were Gibsoned where they did not have color originally in culture but when heated gained color. We added 30 ul of each pelleted chromoprotein to a PCR tube and 1 ul of EDTA to each tube. We started heating at 55 °C in 5 minutes increments and would raise the temperature 5 °C after each 5 minute increment.
Before Heat Added
These are the 12 chromoproteins before heat is added. The 4 chromoproteins on top are the ones that showed color in culture and from left to right are: VP, CP, TP, PP. The bottom 8 chromoproteins are the ones that did not show color in culture or on a plate and in order from left to right are: DT, DM, MP, VV, SP, LO, SO, and BB.
After 55 degrees C
These are the chromoproteins after being heated for 5 minutes at 55 °C. Same order as above. Bottom row has not gained color.
After 60 degrees C
These are the chromoproteins after being heated for 5 minutes at 60 °C. Same order as above. Bottom row still has not gained color. At this point in the last heat tests color was starting to be seen so we hypothesize color will not come back in these.
After 65 degrees C
These are the chromoproteins after being heated for 5 minutes at 65 °C. Same order as above.
After 70 degrees C
These are the chromoproteins after being heated for 5 minutes at 70 °C. Same order as above.
After 75 degrees C
These are the chromoproteins after being heated for 5 minutes at 75 °C. Same order as above.
After 80 degrees C
These are the chromoproteins after being heated for 5 minutes at 80 °C. Same order as above.
After 85 degrees C
These are the chromoproteins after being heated for 5 minutes at 85 °C. Same order as above. CP has turned to purple as we expected and VP and PP are beginning to fade in color.
After 90 degrees C
These are the chromoproteins after being heated for 5 minutes at 90 °C. Same order as above. VP has completely lost its color and TP and PP are continuing to fade in color. CP is now purple and this purple color is beginning to fade.
After 95 degrees C
These are the chromoproteins after being heated for 5 minutes at 95 °C. Same order as above. The top 4 chromoproteins have lost their color completely. The bottom 8 chromoproteins did not gain color or lose any.
●
We looked at the primers we ordered to PCR our chromoprotein and realized we did not use the reverse compliment for the reverse primer. For this reason our Gibson and everything else failed. It is interesting 4 of our chromoproteins did express color because the primers were not accurately created. We will re make and order primers tomorrow to hopefully correct this mistake and allow colors to be see in the future after our Gibson assemblies.
Thursday, 9/8
●
We now performed a large scale protein extraction on the 4 liquid cultures of VP, PP, TP, and CP we started on Tuesday. We spun them down in the centrifuge at 4000 rpm for 10 minutes (make sure tubes are balanced). The pellets were colored which was a promising first step.
Chromoprotein Pellets out of liquid cultures
4 pellets spun down from liquid cultures. Clockwise from the top: PP, CP, VP, TP.
●
We then poured of the supernatant and poured a bit of LB into one of the tubes and vortexed it. Then once all the pellet was off the bottle the LB and cells were transferred to the next bottle and so on until the LB and cells were poured into a 50 mL tube. Spin in centrifuge at 4500 rpm for 10 minutes. Weighed pellet after and added 10 mL/g of cell lysate (CellLytic B). Then added 100 ul of lysozyme and 50 units/mL of benzonase (can also add RNAse and DNAse). Then vortex until pellet is resuspended and place onto vortex for 10 minutes (remember to tape so it doesn't fly off). Spin down at 11,200 rpm for 10 minutes in centrifuge. Then follow the protocol on Jesica's lab bench to wash the column and add the protein. After each elution wash we got fractions that showed color. This confirms our His-tag was present and we were able to purify our proteins.
●
Once we had our purified protein we put them into snake skin dialysis tubing and into 1.75L of  1X TBS buffer overnight
●
Tomorrow we plan to concentrate the product of our dialysis
●
We ordered primers that were the reverse compliment so we will use these tomorrow to perform a PCR of our chromoprotein and then Gibson and transform
Friday, 9/9
●
Took our four chromoproteins out of their dialysis tubing and put them into 50 ml tubes. Because we see color it confirms our His-tag is present because we were able to purify them. This also gives us protein to work with that we can concentrate and use to perform heat testing on.
Chromoproteins after purification and dialysis
Here are our proteins after they have been purified and dialyzed. From left to right they are: TP, PP, CP, and VP.
●
We then concentrated some of each chromoprotein. We did this using centrifugal filter units and spinning them at max speed for 10 minutes.
●
The PCR and PCR clean up we ran Wednesday did not give high enough concentrations of meffRed vector so we re did the PCR today using Q5
●
We then performed a PCR clean up of the meffRed PCR and saw okay concentrations with the nano drop
●
Our new reverse primers for our chromoproteins came in and we ran a PCR and PCR clean up on the 8 chromoproteins that had not shown color.
●
We then ran a Gibson to insert our chromoprotein and His-tag into our vector.
●
Below is the table of the amounts we used for our Gibson:
A
B
C
D
E
F
G
1
Name
Concentration (ng/ul)
BP
pmol/ul
ul of each
Water
MR to use
2
MP
3.2
700
0.007
14
-15
MR_ALL_1
3
SP
23.6
700
0.051
2
2
MR_ALL_2
4
LO
27
700
0.058
2
2
MR_ALL_2
5
VV
11.3
700
0.024
4
0
MR_ALL_3_2
6
DM
24.3
700
0.053
2
2
MR_ALL_3_2
7
BB
30
700
0.065
2
-3
MR_ALL_1_2
8
SO
30.1
700
0.065
2
4
MR_SO_2
9
DT
58.5
700
0.127
1
0
MR_ALL_2_2
10
H1
33.5
100
0.508
1
11
H2
38
100
0.576
1
12
H3
50.5
100
0.765
1
13
H5
35.7
100
0.541
1
14
MR_ALL_1
4.3
2070
0.003
10
15
MR_ALL_2
9.3
2070
0.007
5
16
MR_ALL_3
3.9
2070
0.003
10
17
MR_SO
13
2070
0.010
3
18
MR_ALL_1_2
4.7
2070
0.003
10
19
MR_ALL_2_2
5.7
2070
0.004
8
20
MR_ALL_3_2
8.1
2070
0.006
5
21
MR_SO_2
14.4
2070
0.011
3
Gibson Assembly for 8 remaining chromoproteins
●
After our Gibson we nano dropped and saw extremely high concentrations which is to be expected. We then transformed our 8 Gibson products into chemically competent E. Coli and plated them onto LB+chlor plates at a 1:4 dilution. We covered them and placed them at room temperature to grow up over the weekend. If our PCR and Gibson were successful we should see colorful colonies on Monday.
Monday, 9/12
●
The plates did not grow colonies over the weekend so they were placed into the incubator to see if this would help growth
●
4 plates began to grow colonies after sitting in the incubator. The four plates were SP, BB, LO, and SO. These plates all used the same meffRed for the Gibson so maybe this is why these worked and the others did not. We will still re do the Gibson and transformation in case the growth is contamination and not actually growth
●
Because our plates did not grow colonies we redid the PCR of meffRed back bone, His-tag and chromoproteins
●
We then ran a PCR clean up of all our PCR products
●
We ordered primers for the 4 chromoproteins that expressed color (TP, PP, CP, VP) in order to Gibson in the CBD to our chromoprotein-His-tag. The CBD will be Gibsoned in between the chromoprotein and His-tag
●
Once we ran the PCR clean up of our PCRs we created this chart for our Gibson assemblies and used NEBuilder Hifi DNA Assembly Master Mix to run 8 Gibson assemblies
A
B
C
D
E
F
1
Name
Concentration (ng/ul)
BP
pmol/ul
ul of each
Water
2
MP
15.4
700
0.033
3
4
3
SP
47.2
700
0.102
1
6
4
LO
35.3
700
0.076
1.5
3.5
5
VV
25.8
700
0.056
2
3
6
DM
43.4
700
0.094
1.5
0.5
7
BB
54.2
700
0.117
1
5
8
SO
64.2
700
0.139
1
5
9
DT
48
700
0.104
1
0
10
H1
10
100
0.152
3
11
H2
38
100
0.576
1
12
H3
50.5
100
0.765
1
13
H5
35.7
100
0.541
1
14
MR
21
2070
0.015
2
15
MR
5.1
2070
0.004
8
16
MR
2.3
2070
0.002
15
17
MR
8.1
2070
0.006
5
18
MR
3.8
2070
0.003
10
19
MR
4
2070
0.003
10
20
MR
3.2
2070
0.002
15
21
MR-SO
14.4
2070
0.011
3
Table9
●
Once we had our Gibson assemblies we transformed them into chemically competent E. Coli and then plated them at a 1:4 dilution on LB+chlor plates
●
These plates were placed into the 37 °C incubator overnight and we hope to see growth tomorrow on the plates
●
The plates from the previous transformation continued to grow colonies but we believe these colonies were contamination because they were a different almost pink color but not the color the colonies were supposed to be on each plate
Tuesday, 9/13
●
Checked the plates from the transformation yesterday and there was no growth seen on them.
●
Because we are not seeing any growth on the plates there may be an issue with the Gibson reaction itself or the transformation. We realized we needed to gel extract our chromoprotein instead of doing just a PCR clean up because our primers are not necessarily specific binding so we need to extract only our chromoprotein from the gel.
●
We ran a gel with our meffRed backbones and chromoproteins and performed a gel extraction. The gel extraction resulted in very low if no DNA present when nano dropped so we will re run a PCR and gel extraction on the chromoproteins and meffRed tomorrow
●
We will then Gibson and transform if we have enough products from our extractions
●
The primers for Gibsoning the CBD came in today and we used these to run a PCR of TP, VP, PP, and CP and then the CBD. We then performed a PCR clean up and will Gibson and transform tomorrow
Wednesday, 9/14
●
Re did PCR of meffRed backbone, chromoproteins, and His-tags
●
Ran a gel of the meffRed and chromoproteins
○
Gel Order: Ladder, SP, MP, DM, BB, LO, VV, DT, SO, MR, MR, MR, MR, MR, MR, MR,MRSO
○
We tried to do a gel extraction on these chromoproteins and the vector, but the chromoproteins other than DT were not amplified at the correct number of base pairs. the meffRed vector has some weak bands which we extracted.
○
Because they gel revealed our chromoproteins did not properly PCR we will re do it tonight and then hopefully successfully gel extract tomorrow
●
Ran a PCR clean up of the His-tags
●
Gibsoned the 4 chromoproteins that had shown color and added a cellulose binding domain (CBD) into them. We used the following chart for the Gibson:
A
B
C
D
E
F
1
Name
Concentration (ng/ul)
BP
pmol/ul
ul of each
Water
2
CP
113.7
2800
0.062
2
7
3
TP
195.4
2800
0.106
1
8
4
VP
79.2
2800
0.043
2
7
5
PP
113.3
2800
0.061
2
7
6
CBD
95.5
333
0.435
0.5
7
CBD
53
333
0.241
1
8
CBD
93.5
333
0.425
0.5
Gibson for CBD of 4 chromoproteins
●
Transformed our Gibson product into chemically competent E. Coli and then plated on LB+chloro plates
Thursday, 9/15
●
The transformed cells (chromoprotein-CBD-His-tag) had colonies on the plates. VP had ones colored colony but sometimes the color can take some time to show in the colonies
●
Because of colonies present we are colony PCRing and create liquid cultures from each of the plates today
●
Ordered sequencing for tomorrow for both forward and reverse set (total of 8) to be picked up tomorrow
●
We ran a gel with some of the product from our colony PCR and the main amplified band was not around 1100 base pairs like we would expect but instead was at about 300 base pairs. Sequencing data will help us to see what is going on in our sample
CHROMOpcr-4COLORS-9-15-16.jpg
The gel we rn on 2 ul of our colony PCR product. The order is: ladder, CP, PP, TP, VP. The main amplified band is around 300 base pairs which indicated the PCR is amplifying only the CBD or the construct only has the CBD
●
We then ran a PCR clean up on our colony PCR and will use this product to send to sequencing tomorrow
●
Ran gel of the chromoproteins and meffRed backbone that were PCRed yesterday
○
Gel order: Ladder, MR, MR, MR, MR, MR, MR, MR, DT, LO, VV, DM, BB, SO, MP, SP
●
We then performed a gel extraction of the chromoproteins at the correct number of base pairs (~700) and also a gel extraction on the meffRed backbone (~2070 bp).
●
We then used these products and products from our gel extraction yesterday and PCR clean up of our His-tags to do a Gibson assembly. We had enough of each fragment to do a Gibson for all but VV. We used the following amounts for each Gibson:
A
B
C
D
E
F
G
1
Name
Concentration (ng/ul)
BP
pmol/ul
ul of each
Water
MR
2
MP
9
700
0.019
5
0
3
SP
7.1
700
0.015
7
-2
4
LO
5.8
700
0.013
7
-1
5
VV
0
700
0.000
0
5
6
DM
4.5
700
0.010
10
-4
7
BB
4.1
700
0.009
10
-4
8
SO
6.8
700
0.015
7
-1
9
DT
17.9
700
0.039
2
-1
10
H1
25.6
100
0.388
1
11
H2
31.7
100
0.480
1
12
H3
22.4
100
0.339
1
13
H5
32.7
100
0.495
1
14
MR
10.8
2070
0.008
4
15
MR
15.4
2070
0.011
3
16
MR
5.3
2070
0.004
8
17
MR-SO
13.5
2070
0.010
3
Table10
●
Because we did not get any VV from either of our gel extractions we will try another PCR and gel extraction on it tomorrow
Friday, 9/16
●
Checked plates from transformation of the 8 chromoproteins with His-tags and no colony growth was seen. Because they were plated last night they were put back into the incubator to see if any growth would occur with more incubation time
○
Because of no growth we went through over our primers again and realized that we reverse complimented the forward primer instead of the reverse primer and this is why nothing was working and our gels were only showing weak bands where our chromoproteins should be
○
We then reverse complimented the correct primers and re ordered. On Monday we will be able to hopefully complete the PCR, gel extraction, Gibson, and transformation and see colonies on our plates on Tuesday.
●
The liquid cultures were spun down and VP showed color in culture, but the colony PCR gel we ran yesterday did not show a band at the correct length of ~1200 base pairs so we are re doing the colony PCR today and running another gel to see if the correct insert is amplified and if its not then something probably went wrong with the Gibson assembly
●
The plates from the 4 chromoprotein-CBD-His-tag began to show color and so we used these colonies to re do the colony PCR and liquid cultures
●
If this gel shows the correct amplification we will send the PCR in for sequencing.
●
The gel showed bands at the correct length ~1200 base pairs but also many other bands at different lengths and the primary band was at ~300 base pairs. This means we have some of the correct product getting amplified but also other things as well. We will send in for sequencing and see if this helps reveal what is going on
●
We concentrated our 4 purified chromoproteins (TP, VP, CP, PP) by using centrifugal filter units and spinning at max speed for 10 minutes, discarding the flow through, and then flipping the unit over and spinning at 3.8 rpm for 3 minutes
●
We began heat testing on the 4 concentrated chromoproteins
●
First we pipetted 3 ul of the concentrated protein onto a sheet of cellulose paper with wax wells printed on it. After it dried we pipetted another 3 ul into the same well. Once it was dry for the second time we put it into the oven that was pre heated to 55 °C. CP, VP, and TP lost their color at this temperature but after 2 minutes PP still has some of its color so the temperature was raised to 60 °C. It took 1.5 minutes for the over to heat up to this temperature and by the time it got up to 60 °C PP has lost its color. We then took the chromoproteins out of the oven and let them sit on the bench. After sitting for 3 minutes they had regained none of their color, but when rehydrated with 1 ul of milliQ water they regained their color. These chromoproteins were then heated again but instead at 40 °C for one minute. No color change was observed and so the temperature was raise to 45 °C and CP, TP, and VP lost their color in between these temperatures around 42 °C. PP still had its color at 45 °C after a minute it was raised to 50 °C and it began to lose its color. The chromoproteins were then rehydrated with 1 ul of miiliQ water and they gained their color back. This test shows a reversible color change that is seen at a relatively low temperature. Interestingly CP did not change to purple and then lose its color like we say in cell lysate testing.
Before Heat
Here are the chromoproteins before heat is added. Order from top to bottom: CP, PP, TP, VP.
After first heat test and losing color
Here are the chromoproteins after they lost their color and before they were rehydrated. Same order as above.
After Rehydration
Here are the chromoproteins after they have been rehydrated with 1 ul of water. They have gained their color back. Same order as above. These were then put back into heat.
After heat 2nd time
Here are the chromoproteins after the second heating where PP lost its color at 50 degrees C and CP, TP, and VP lost their color between 40 an 45 degrees. Same order as above.
After rehydration
Here are the chromoproteins after they were rehydrated again with 1 ul of milliQ water they have gained their color back. Same order as above.
Monday, 9/19
●
Ran a PCR of the meffRed backbone and also of the His-tags to be used in the Gibson assembly of the 8 chromoproteins that we ordered new primers for on Friday
○
We then did a PCR clean up on the His-tag and got decent concentrations
○
We ran a gel and planned to do a gel extraction on the meffRed but the band was not in the correct place so we could not extract anything
●
Then once our new primers came in we ran a PCR of the 8 of our chromoproteins that are still not expressing color and re ran the meffRed PCR
○
We then ran a gel on these PCR products and then performed a gel extraction on the bands at the correct base pairs (~700 base paris for chromoproteins and ~2000 base pairs for meffRed)
○
High concentrations were seen after the gel extraction of the chromoproteins and meffRed had a pretty low concentration
●
We then planned to run a Gibson assembly but were out of the masterMix for this so instead we re ran the meffRed PCR to hopefully get higher concentrations to run a Gibson tomorrow
●
Concentrated more chromoproteins that had the his-tag (PP, TP, CP) by spinning at full speed for 10 minutes, discarding the supernatant, then flipping the column and spinning at 3.8 rpm for 2 minutes
Tuesday, 9/20
●
Ran gel and performed a gel extraction on meffRed that we had run a PCR on yesterday
●
Got more Gibson assembly master mix and ran a gibson assembly to insert a chromoprotein and His-tag into the meffRed backbone using the following amounts:
A
B
C
D
E
F
1
Name
Concentration (ng/ul)
BP
pmol/ul
ul of each
Water
2
MP
101.3
700
0.219
1
4
3
SP
120.5
700
0.261
1
0
4
LO
125.9
700
0.273
1
3
5
VV
88.5
700
0.192
1
3
6
DM
113
700
0.245
1
3
7
BB
80.4
700
0.174
1
3
8
SO
75.6
700
0.164
1
5
9
DT
120.3
700
0.260
1
0
10
H1
22.5
100
0.341
2
11
H2
41.7
100
0.632
1
12
H3
36.5
100
0.553
1
13
H5
35.4
100
0.536
1
14
MR
9.3
2070
0.007
4
15
MR
3.2
2070
0.002
10
16
MR
4
2070
0.003
8
17
MR
2.3
2070
0.002
10
18
MR
3.8
2070
0.003
8
19
MR
7.7
2070
0.006
4
20
MR-SO
14.4
2070
0.011
3
Table11
●
Transformed this Gibson into chemically competent E. Coli and then plated onto LB+chlor plates
●
Ran a colony PCR from the 4 chromoprotein plates with chromoprotein-CBD-His-tag and using VF2 and VR as the primers. We then ran 2 ul of each reaction on a gel to see if the correct region was being amplified (in the past we hadn't seen a strong enough band at the correct spot). We also did a PCR clean up of the remaining colony PCR product
○
When we imaged the gel we saw the correct band amplifies around 1200 base pairs and we will send the PCR clean ups in tomorrow to confirm our sequences
Wednesday, 9/21
●
Checked the plates from the Gibson assembly and transformation yesterday. 4 of the plates (LO, BB, VV, SO) showed colonies, although no color, after the overnight incubation and the 4 other plates did not show colonies even after extra incubation time in the incubator.
○
We will place this plates overnight on the bench and hope to see color on the 4 plates that had colonies
●
We want to re do the Gibson of the 4 plates that did not work. We think the meffRed backbone was not inserted in high enough concentrations so we will try and get higher concentrations by redoing the meffRed PCR and gel/gel extraction. We re did the PCR of the meffRed backbone and then ran a gel of it to perform a gel extraction. When we imaged the gel the bands were not at the correct length and were instead around 700 base pairs which is where the chromoprotein should be. Because meffRed has been not correctly getting amplified many of the times we run PCR on it we decided to re miniprep and PCR the meffRed backbone.
○
We made a liquid culture of meffRed and placed it into the 37 °C  incubator/shaker overnight which we will mini prep tomorrow
●
We ran a colony PCR from the 4 plates that worked to see if the correct region is amplified with VF2 and VR
○
We will run part of this product on a gel tomorrow and then send in for sequencing if the correct number of base paris (~100 base pairs) is amplified with VF2 and VR
●
We sent the sequences in for the 4 chromoproteins with CBDs using both VF2 and VR for a total of 8 reactions
●
Made liquid cultures of the 4 CBD chromoproteins and placed them into the 37°C incubator shaker overnight to eventually mini prep tomorrow
●
Created 4 100ml liquid cultures of the 4 chromoprotein-CBD-His-tag and placed them into the 37 °C incubator/shaker overnight and will love them to the 25 °C incubator/shaker for another day before performing a protein extraction on Friday
Thursday, 9/22
●
We ran 2 ul of the colony PCR from yesterday from the 4 plates that worked of chromoprotein-His-tag on a gel to see if a region around 1000 base pairs is amplified because this is the length of our chromoprotein and His-tag
○
The first gel we ran turned out weird and we could not see the ladder well
9-22-16-his-tag-PCR.jpg
Here is the gel we ran from our colony PCR. The ladder is on the far left and not well separated to it is hard to see what the length of the band is that is the amplified region. Gel order: ladder, LO, BB, VV, SO. Because the ladder was not well separated or seen we decided to re do the gel to see if we could get a better image of it.
9-22-16-CHROMOPCR-REDO.jpg
Here is the gel we re ran of the 4 colony PCRs. Order: ladder, BB, LO, VV, SO. The amplified region is seen around 1000 base pairs which is where we would expect it. Because of this result we ordered sequencing and will use the rest of the colony PCR to do a PCR clean up and use this to send in for the sequencing with both VF2 and VR.
●
We then mini prepped the 4 liquid cultures from the CBD-His-tag chromoproteins and also of meffRed
○
The 4 minipreps of the chromoprotein-CBD-His-tags will be used to send in for biobricks
●
The sequencing data of the from the 4 CBD-His-tag chromoproteins came back and we were not able to tell if our sequencing was off or it was messy data but the correct sequence
●
We created 4 more liquid cultures, 1 from each of the CBD-His-tag chromoproteins to grow up overnight and will do a lumio green detection on it tomorrow to confirm our his-tag is present
●
We also created 4 more cultures of the chromoprotein-his-tag plates that showed colonies on plates (BB, VV, LO, SO) and plan to test for the presence of the flag lumio his-tag tomorrow using a lumino green detection and running it on a SDS page gel
●
We used the mini prep of meffRed to run a PCR and then ran the PCR product on a gel and performed a gel extraction. The gel showed strong bands (stronger than we had seen in previous gels using the previously prepped meffRed) at around 2070 base pairs and after the gel extraction we ended with a higher nano drop concentration than previously seen.
●
We used this meffRed backbone and previously PCRed his-tags and chromoproteins to re do the Gibson assembly of the 4 plates that did not show colonies (MP, SP, DM, DT).
●
Here is the table of the amount of each chromoprotein, backbone, and His-tag we used.
A
B
C
D
E
F
1
Name
Concentration (ng/ul)
BP
pmol/ul
ul of each
Water
2
MP
101.3
700
0.219
1
5.5
3
SP
120.5
700
0.261
1
5.5
4
DM
113
700
0.245
1
4.5
5
DT
120.3
700
0.260
1
5
6
H1
22.5
100
0.341
2
7
H2
41.7
100
0.632
1
8
H3
36.5
100
0.553
1
9
MR
18.4
2070
0.013
2.5
10
MR
13.1
2070
0.010
3
Gibson Assembly Table 4 Chromoproteins
●
We then took this Gibson assembly and transformed it into chemically competent E. Coli and then plated it onto LB+chloro plates
●
We moved the 4 100ml cultures of the 4 chromoprotein-CBD-His-tags from the 37 °C incubator/shaker to the 25 °C incubator/shaker overnight
●
Cleaned 8 columns for protein extraction tomorrow
Friday, 9/23
●
Looked at plates from transformation yesterday. MP, SP, and DM all had colony growth but DT did not. On the plates from the transformation earlier this week DT began to grow colonies. We are unsure if this is contamination or actually a colony. We then used the three plates that worked and the older DT plate to run a colony PCR and create liquid cultures. We will then run 2 ul of the colony PCR on a gel to determine if the correct number of base pairs (1000 base pairs) is being amplified with VF2 and VR and we performed the Gibson correctly.
●
The gel revealed that all but DT worked using the PCR and the correct band ~1000 base pairs was amplified. Because of this result we ordered sequencing on all 4 of these samples to see how they compare to the sequence we think we are getting. We PCR clean used the colony PCR and used VF2 and VR for sequencing.
9-23-16-ChromoPCR-4his.jpg
This gel was run on our 4 chromoprotein-His-tag colony PCR products. The gel order from left to right: ladder, SP, DM, MP, DT. The first three columns of chromoproteins see a correctly amplified region around 1000 base pairs. DT does not have this band present and we believe the sequencing data will show we do not have the correct product.
●
We ran a small scale protein extraction on the liquid cultures we started yesterday. 4 of them are CP, PP, TP, and VP with both cellulose binding domains and his-tags and 4 of them are BB, VV, LO, SO which are chromoproteins with His-tags. We then performed a lumio greed detection and ran on a SDS page gel and imaged to see if our his-tag was present in both sets of these chromoproteins.
●
The gel showed that the his-tag was present in only 1 of our proteins. The other 3 had lumio present but not at the correct number of kDa (~2.5) this means maybe our his-tag is present just not as part of our chormoprotein. Because we started the large liquid cultures from different colonies it is unclear whether or not our large scale protein extraction will work today.
9-23-16-lumio-4cbdchromo.jpg
Here is the gel we ran of the luminous of the 4 chromoprotein-CBD-His-tag proteins. Order from left to right: benchmark ladder, CP, PP, TP, VP. The dark band in the 4th column is TP and at around 2.6 kDA which is where we would expect our chromoproteins to be. This shows that TP was correctly Gibsoned with the his-tag. The other 3 it is unclear if the colony picked was not a good colony and representative of the Gibson product or if they just did not work.
●
We ran a lumio gel of the 4 cultures that has his-tags that were present yesterday (BB, VV, LO, SO) and saw the correct bands for each of these proteins. This is exciting because it means our his-tag is present in the chromoprotein but we still have yet to see color on the plates or in culture so we will check for color again on Monday
9-23-16-lumio-4chis-tagchromo.jpg
Here is the gel we ran of the 4 lumio tests. From left to right: Benchmark Ladder, LO, SO, BB, VV. The right two bands are around 2.6 kDA and the left two chromoprotein bands are around 2.5 kDA which is where we would expect. This gel adds another form of verification to our chromoprotein + his-tag being correctly Gibsoned.
●
We then performed a large scale protein extraction on CP, PP, TP, VP CBD and His-tag liquid cultures. The elutions showed color in PP, TP, and VP which means the his-tag worked properly because the proteins were bound to the nickel column with the his-tag and then eluted off. CP did not elute or show color in any of the washes or elutions so we may have to re do this protein extraction
●
These 4 chromoproteins were then placed into TBE buffer over the weekend for dialysis. The three elutions for each protein were placed into separate dialysis tubing and for CP the initial equilibration fraction was also placed into the tubing because it had color in it.
●
Started liquid cultures of SP, MP, DT, DM to use for lumio testing on Monday
Monday, 9/26
●
Sequencing data from Friday came back for MP, SP, DM, and DT. All but DT had perfect or close to perfect sequencing which helps us confirm we have the correct construct on our plates.
●
Plates showed color for DM, SP, BB, LO, VV, SO. DT and MP are not currently showing color but maybe they will in the next couple days. Because we saw color on these plates we ordered primers to Gibson in the CBD into these constructs.
●
DT from the transformation Thursday now has colonies on it so we ran colony PCR again on this because the sequencing on it from Friday was not correct.
●
We ran 2 ul of the colony PCR on a gel so that we would know if it was the correct sequence being amplified (~1000 base pairs)
9-26-16-ChromoPCR-1his.jpg
This is a gel run of the DT colony PCR. From left to right: ladder, DT. The region amplified in DT is around 200 base pairs which is not the product we were expecting. We will send in for sequencing and see if this can help us answer what we are doing wrong with this particular chromoprotein.
●
We then ran a PCR clean up and sent in DT for sequencing using both forward and reverse primers (VF2 and VR)
●
We used 1.5 ml of the liquid cultures started from Friday to perform a small scale protein extraction. We used the soluble fraction to run a lumio green detection on. Once we ran the lumio green detection protocol we then ran it on an SDS gel and imaged it on the scanner
9-26-16-lumio-chromo.jpg
This gel shows the lumio green product ran on the 4 his-tag chromoprotein cultures started Friday. Order from left to right: benchmark ladder, SP, DM, DT, MP. The primary bands on the SP and DM columns are around 30 kDa which is higher than would be expected for a chromoprotein. All but DT appear to have bands around 25 kDA which is where we would expect to see larger bands showing our product is present.
●
This gel confirms our his-tag is present but it is unclear on whether or not it is part of our construct. We will start liquid cultures tomorrow and perform protein extraction using nickel columns to see if the cultures are properly extracted with their his-tags
●
Started 100 ml liquid cultures of VV, BB, LO, and SO from the colorful colonies on the plates and placed into the 37 °C incubator/shaker overnight. Tomorrow we will transfer to the 25 °C incubator/shaker for another night and perform a protein extraction on Thursday on these 4 chromoproteins
Tuesday, 9/27
●
The sequencing data from DT came back and showed we did not have the correct sequence. We will keep troubleshooting this today by picking a different colony from the DT plate and seeing if a colony PCR works on a different colony.
●
We ran part of the colony PCR on a gel saw a band around 200 base pairs like we did yesterday. We will not send this colony PCR in for sequencing because the gel did not work. Because this construct is not the correct one we will re do the Gibson and transformation and plating and hopefully get the correct construct.
9-27-dt.jpg
Order from left to right: Ladder, DT. The amplified band of DT is around 200 base pairs which is not at all close to 1000 base pairs which is where it should be.
●
We moved the 4 100 ml liquid cultures from the 37 °C incubator/shaker to the 25 °C incubator/shaker and will perform a protein extraction on these chromoproteins tomorrow. When transferring the cultures the BB erlenmeyer flask broke and the liquid culture went everywhere so we need to restart this liquid culture.
●
We started 4 100 ml liquid cultures for protein extraction on Friday. We started cultures of BB again, SP, MP, and VV. We placed these cultures into the 37 °C incubator/shaker and then will transfer to the 25 °C incubator/shaker tomorrow and perform a protein extraction on Friday.
●
We cleaned the columns needed from tomorrows protein extraction
Wednesday, 9/28
●
We prepared the 3 (VV, LO, SO) 100 ml liquid cultures for protein extraction by first spinning them down at 4000 rpm for 10 minutes and then discarding the supernatant. We then resuspended the pellet in minimal LB and poured into separate 50 ml tubes. We then spun these tubes down at 4500 rpm for 10 minutes. We removed the supernatant and weighed the pellet. We added 10 ml/g of cell lysate (CellyLyticB), lysozyme at a  final concentration of 0.2 mg/mL, and benzonase at 50units/ml. We then vortexed this until the pellet was mixed and then placed onto the vortex for 10 minutes. We then spun it at 11,200 rpm for 10 minutes in the centrifuge. We then removed the soluble fraction and placed it into a separate tube because we were out of one of the chemicals needed for the column extraction.
●
We moved the 4 100 ml liquid cultures we started yesterday from the 37 °C incubator/shaker to the 25 °C incubator/shaker and will extract them tomorrow
Thursday, 9/29
●
We found the chemical we needed for the protein extraction so we finished the protein column extraction today with VV, LO, and SO using the soluble fraction from yesterday.
●
Once we had our purified protein we put them into snake skin dialysis tubing and into .5L of  1X TBE buffer overnight
●
We also then took the 4 100 ml cultures out of the incubator/shaker and prepped them for the column purification in the same way as yesterday. The BB liquid culture did not show color in it so we discarded this one and will move forward with the MP, SP, and DM liquid cultures. Once we got to the part of separating the soluble fraction from the insoluble fraction we kept the soluble fraction for the column purification tomorrow.
●
Created 5 ml cultures of VV, BB, SP, MP, LO, SO, and DM using LB+chlor and we will grow them up in the 37 °C incubator/shaker overnight and mini prep them tomorrow morning. We will then use this mini prep product to perform our Gibson assembly of adding CBDs to all out constructs. We did not mini prep DT because we have to still troubleshoot and get the his-tag to work. Also CP, PP, TP, and VP are already mini prepped.
Friday, 9/30
●
Mini prepped the 5 ml liquid cultures we started yesterday.
●
We then used these mini preps of our chromoprotein+his-tag in a psb1c3 backbone to PCR open this construct so we could add our CBD in. Once we ran the PCR of our 11 chromoproteins+his-tag (minus DT because it didn't work) we ran it on a gel and performed a gel extraction of the one part we wanted which was amplified around 3000 base pairs. We did a gel extraction because we were not sure how specific our primers would be and wanted to only get our backbone+chromoprotein+his-tag
●
We also ran a PCR of the cellulose binding domains for each chromoprotein and performed a PCR extraction on these. After our PCR extraction we saw little to no DNA present via the nano drop so we could not perform our Gibson today. It looked like our sample may be contaminated with ethanol which would make the concentration look different than it actually is so we will test the concentration again after the weekend to see if ethanol has evaporated off and left us with cleaner results.
●
We ran a DT PCR so that we could re do the Gibson assembly of adding a his-tag again. We also ran a PCR of the meffRed backbone and ran these both on a gel for a gel extraction. On the gel there were no bands at around 2000 base pairs for meffRed or around 700 base pairs for DT so we will re do these PCRs on Monday to then Gibson and transform
●
We purified our 3 proteins we had prepared yestereday (MP, SP, DT) using nickel column purification and then placed the colored fractions together into snake skin dialysis tubing into 1X TBE buffer to sit over the weekend
Monday, 10/3
●
We took the 3 chromoproteins+his-tag purified proteins (MP, SP, DM) our of the dialysis tubing and placed them into 50 ml eppendorf tubes.
●
We ran a PCR of DT and meffRed which we then gel extracted. These will be used to re do the gibson and transformation and plating. If this works then we should see colored colonies of DT-his-tag on the plate and be able to sequence, lumio test, mini prep, and then protein extract and purify.
●
Another PCR was run of VV-his-tag because it did not work on Friday and we need it for our CBD Gibson and we performed a gel extraction on this so we would only have the band we needed (~3000 base pairs)
●
We also ran a PCR of the 5 CBDs and PCR clean upped them for use for our CBD Gibson
●
We ran a Gibson assembly for adding a CBD to 11 of our chromoproteins using the following amounts for each Gibson assembly.
A
B
C
D
E
F
1
Name
Concentration (ng/ul)
BP
pmol/ul
ul of each
Water
2
MP
11.9
700
0.026
2
6
3
SP
12.2
700
0.026
2
6
4
LO
7
700
0.015
2
7.5
5
VV
23.2
700
0.050
1
8.5
6
DM
8.1
700
0.018
2
7.5
7
BB
19.1
700
0.041
1
8.5
8
SO
11.7
700
0.025
2
7.5
9
VP
13.4
700
0.029
2
7.5
10
PP
13.9
700
0.030
1
8.5
11
TP
20.8
700
0.045
2
7.5
12
CP
6.6
700
0.014
2
7.5
13
CBD MP/SP
32.1
300
0.162
2
14
CBD LO-BB
91
300
0.460
0.5
15
CBD SO
95.1
300
0.480
0.5
16
CBD VP-CP
92.5
300
0.467
0.5
Table12
●
We also did a Gibson assembly on DT and the his-tag to hopefully get this construct working
A
B
C
D
E
F
G
1
Name
Concentration (ng/ul)
BP
pmol/ul
ul of each
Water
2
DT
54.9
700
0.119
1
6.5
3
H3
22.4
100
0.339
0.5
4
meffRed
21.1
2070
0.015
2
Table13
●
After our Gibson assembly of the 11 chromoprotein+CBDs+his-tags and DT+his-tag we transformed these into chemically competent E. Coli. We then plated these onto LB+chloro plates and placed them into the 37 °C incubator overnight
Tuesday, 10/4
●
There was cell growth on all our plates except for SO, and SP. We will re do these Gibson and transformations when we also do DT. Although the colonies did not have colors yet we have seen it takes a couple days for them to gain colors so we are not concerned at this point.
●
Because of the cell growth we colony PCRed from the 10 plates that worked using VF2 and VR and also started 5 ml liquid cultures from these colonies.
●
We plan to run 2 ul of the colony PCR on a gel to confirm the sequence is the correct number of base pairs and then order sequencing tomorrow.
●
The 5 ml liquid cultures we started we will mini prep tomorrow morning and use as biobricks if our sequencing is clean.
Wednesday, 10/5
●
We ran 2 ul of the colony PCR on a gel and saw bands in the correct spot (~1500 base pairs for chromoprotein-CBD-his-tag proteins and ~1000 base pairs for DT+his-tag)
10-5-16-PCR-CBD
Here is the gel we ran of our chromoproteins with CBDs and DT with just a his-tag. The order from left to right: ladder, PP, VP, CP, VV, BB, LO, TP, MP, DM, DT. The chromoproteins with the CBDs show up around 1500 base pairs which is what we would expect. VV and LO do not appear to have the bands at all so we do not think enough DNA was loaded into the gel. DT (in the last well) shows a band around 1000 base pairs which is what we would expect for a chromoprotein+his-tag. This gel helps to confirm our constructs and we will send in for sequencing.
●
We ordered sequencing and performed a PCR clean up and then sent in both forward and reverse sequencing
●
If the sequencing comes back clean tomorrow we will start 100 ml liquid cultures to perform protein extraction on on Friday
●
We also mini prepped the 10 liquid cultures we had started last night and will use these as biobricks and then the DT one to perform the CBD gibson on if the sequencing looks correct.
Thursday, 10/6
●
The sequencing data came back for the 9 constructs of chromoproteins+CBD+his-tags and for DT-His-tag and all the sequencing looks like we have the correct construct
●
Because of this we started 100 ml liquid cultures of all these CBD constructs and DT, BB, and VP plus his-tags . We will use these liquid cultures to run a protein extraction tomorrow afternoon.
Friday, 10/7
●
Moved the liquid cultures from the 37 °C incubator/shaker to the 25 °C incubator/shaker in the morning and plan to protein extract in the afternoon
●
We nano dropped our mini preps from Wednesday and then used the DT-his-tag mini prep to start a PCR with the CBD primers we had designed and ordered last week
●
We also ran a PCR of the CBD for SP and SO but we already had the CBD for DT ready
●
We then gel extracted DT and did a PCR clean up of the CBDs we saw minimal to no DNA present when nano dropped. We will re do the PCR of DT-his-tag and its gel extraction next week to hopefully Gibson in the CBD to DT, SO, and SP
●
We then ran a lumio green of the 9 CBD chromoprotens and the 1 his-tag chromoprotein to confirm the precesnse of a tag. We used 1.5ml from the large liquid cultures we had started to perform a small scale protein extraction and then the lumio green detection and ran this product on a page gel.
lumio-cbd.jpg
Here is a lumio gel we ran of our 9 chromoproteins+CBDs and DT+his-tag. The order from left to right: benchmark ladder, LO, MP, BB, VV, PP, TP, DM, VP, CP, DT, benchmark ladder. DT (far right)
●
We prepared the 12 liquid cultures for protein extraction by spinning down at 4000 rpm for 10 minutes and then discarding the supernatant, resuspending in minimal LB, and then pouring into a 50 ml eppendorf tube, spinning down at 4000 rpm for 10 minutes and weighing the pellet after. We then froze these pellets in the -20 °C freezer and will protein extract them on another day.
Saturday, 10/8
●
We added 10 ml/g of cell lysate (CellyLyticB), lysozyme at a  final concentration of 0.2 mg/mL, and benzonase at 50units/ml. We then vortexed this until the pellet was mixed and then placed onto the vortex for 10 minutes. We then spun it at 11,200 rpm for 10 minutes in the centrifuge. We then removed the soluble fraction and placed it into a separate tube because we were out of one of the chemicals needed for the column extraction.
Friday, 10/14
●
Ran a PCR of DT-his-tag, SO-his-tag, and SP-his-tag from the minipreps to get ready for a Gibson assembly. We then ran a gel and did a gel extraction on these samples.
●
We also ran a PCR of the 3 different CBD+primer combinations we need for our Gibson assembly and performed a PCR clean up on these
●
Performed Gibson assembly on DT, SO, SP and CBDs using the following table:
A
B
C
D
E
F
1
Name
Concentration (ng/ul)
BP
pmol/ul
ul of each
Water
2
SP
11.7
700
0.025
2
7
3
SO
19.4
700
0.042
1
8
4
DT
3.7
700
0.008
4
3
5
CBD MP/SP
43.6
300
0.220
1
6
CBD SO
35.6
300
0.180
1
7
CBD DT
12.9
300
0.065
3
Table14
●
Transformed the Gibson assemblies into chemically competent E. Coli and plated these
●
Clean columns for protein extraction and protein extract 10 chromoproteins. Some of the chromoproteins with the CBDs did not have colored soluble fractions which is where our chromoprotein should be so the only two chromoproteins-CBD-his-tag that worked were PP and LO
●
We put our extracted chromoproteins into dialysis tubing and into 1X TBE buffer to sit overnight
Saturday, 10/15
●
Took 10 chromoproteins out of dialysis tubing
●
Checked plates and saw growth on all three plates. No color seen yet but that typically takes a couple days to show
●
We inoculated 5 ml liquid cultures which we will mini prep tomorrow to be used as biobricks
●
We ran a colony PCR with VF2 and VR which we will use to send in for sequencing
●
We also placed 3 ul of PP-CBD-His-tag purified and concentrated protein onto a wax well sheet and also LO-CBD-his-tag
IMG_1630.jpg
Here are the chromoproteins we pipetted onto the cellulose sheet. The top row is PP-CBD-his-tag and the bottom row is LO-CBD-his-tag
●
We let this to sit overnight to see if there would be a color change in an open air environment
Sunday, 10/16
●
Colony PCR did not properly run last night so we re did today
●
We then used 2 ul of this colony PCR for DT, SO, SP and ran it on a gel to confirm our constructs were of the correct length ~1500 base pairs
stanford 2016-10-17 01hr 24min.tif
Here is the gel we ran of our chromoprotein-CBD-his-tag constructs that have been PCRed out using VF2 and VR. Order from left to right: Ladder SP, DT, DO, Ladder. The chromoproteins all have bands around 1500 base pairs which is where we would expect so we will send in for sequencing
●
We ordered sequencing for these constructs for both forward and reverse sequencing
●
Miniprep liquid cultures and will use these for biobricks
●
Started 50 ml liquid cultures of all but PP and LO chromoproteins-CBD-his-tag constructs to perform protein extraction on tomorrow and placed into the 37 °C incubator/shaker
●
The plates for SO and SP now are showing color but DT is still without color (DT-his-tag construct plate is also still without color)
●
We looked at LO and PP on the cellulose sheet from yesterday and saw that it had lost a lot of its color probably due to evaporation of water
IMG_1635.jpg
Here are our chromoproteins after sitting out overnight on the bench top. They have faded in color probably due to evaporation of water.
Monday, 10/17
●
Spun down the 50 ml cultures at 4000 rpm for 10 min and did not see colored pellets so we resuspended and placed back into the 37 °C incubator/shaker overnight to see if color would result
●
Sent in sequencing for SO, DT, and SP
●
We performed a test in the lyophilizer with the chromoprotein-CBD-his-tag chromoproteins we placed on the cellulose sheet Saturday.
●
We started off by freezing our cellulose sheet at -80 °C for an hour and then placed into the lyophilizer for an hour. This is done to remove any excess water and see if the color change is a result of water loss or heat or both
IMG_1638.jpg
Here is our cellulose sheet with PP on the top row and LO on the bottom row before placing in the lyophilizer
IMG_0120.JPG
Here are the chromoproteins after an hour in the lyophilizer. They have lost a little of their color but not all of it. Which helps to see water loss is not the only thing contributing to color change but also heat.
IMG_0121.JPG
Here are the chromoproteins after they have been removed from the lyophilizer. They have gained a little color back just naturally from the water in the air.
●
We next performed heat testing today on the 12 chromoprotein-his-tag constructs
●
First we concentrated our purified proteins by using spin columns and placing 500 ul into each column and spinning at max speed for 10 minutes. We then discarded the supernatant and flipped the column over and spun for 2 minutes at 3800 rmp. We then used this concentrated protein for our heat tests.
●
We pipetted 3 ul of each chromoprotein into a wax well, let dry, and then added another 3 ul to each well.
IMG_1639.jpg
Here is a picture right after the chromoproteins were pipetted onto the cellulose sheet in the wax wells. Order is seen across the bottom.
●
After the chromoproteins sat on the bench top and dried for a couple minutes they lost a little bit of their color
IMG_1641.jpg
Here is the chromoproteins after they have sat and dried for a couple minutes on the lab bench. They have lost some of their color but they are still colored.
●
We then placed this sheet into the oven and starting at 20 °C raised the temperature degree by degree until all chromoproteins had lost their color
IMG_0122.JPG
Here are the chromoproteins after they have been heated. They were heated to 60 °C before all of them lost their color. A video is in the googleDrive to show the color loss
●
We then rehydrated each chromoprotein with 3 ul of milliQ water and the color came back to what it originally was
IMG_0123.JPG
Here are the chromoproteins after they have been rehydrated with 3 ul of milliQ water. They have regained their color
●
We then repeated this process by adding another row of these chromoproteins to see if there was a difference in the heat response after they have been heated multiple times
IMG_0124.JPG
The top row is a new set of chromoproteins that have been pipetted (as before 3 ul let dry, then another 3 ul) onto the cellulose sheet. The bottom row is the previously tested chromoproteins that have been rehydrated.
●
We then heated these starting at 40  °C until all the colors were gone again (around 60  °C)
IMG_1644.jpg
Here are the chromoproteins after being heated. The bottom row was heated a second time and the top row was heated for the first time. They have all lost their color.
●
We rehydrated these again with 3 ul of milliQ water
IMG_1646.jpg
Here are the chromoproteins after they have been rehydrated with 3 ul of water. Both the top and bottom row look the same even though the bottom row has been heated more. This helps to show that the number of times of heating does not affect the outcome of rehydration. This also shows a reversible color change in our chromoproteins at relatively low temperatures compared to those in the cell lysate tests.
Tuesday, 10/18
●
We first looked at pelleted cultures that have been maintained by EDTA for over a month. They maintained their color which means the chromoproteins were not degrading and the heat testing and heat loss is not a speeding up a degradation process and instead they are sensitive to heat. The EDTA is a protease inhibitor so the chromoproteins have not been producing more chormoproteins over this time period but maintained their color.
IMG_0126 2.jpg
Pellets still colored over 1 month later using EDTA as a protease inhibitor.
●
We wanted to test to see if the CBD worked on our PP and LO chromoproteins that had been placed in wax wells on Saturday. We put the cellulose sheet under running water from the faucet for a minute and took pictures before and after (show below). There was no color change seen which means our chromoprotein stayed on the cellulose sheet adn the cellulose binding domain successfully worked and binded to the cellulose
IMG_1648.JPG
Here are the chromoproteins with CBDs before we tested their CBDs with water. On the top row is PP and the bottom row is LO
IMG_1649.JPG
This was taken right after the chromoproteins were put under water for a minute. They still are on the cellulose sheet and appear to be bound well to it.
IMG_1650.JPG
Here is after the chromoproteins were given time to dry. They still have their color the same as before which helps to show the cellulose binding domain inserted is working properly to bind the chromoproteins to the cellulose paper.
●
We also looked again at our 12 chromoproteins we heat tested on yesterday to see the affect of sitting overnight on the bench top. We saw that they lost some of their color due to evaporation of water but maintained the color seen before heat testing.
IMG_1647.jpg
Here are the chromoproteins after sitting overnight. They have lost some of their color because of water evaporation but still retain the color they had before heat testing.
