Team:Stanford-Brown/SB16 BioMembrane p-Aramid
Stanford-Brown 2016
Toggle navigation
Stanford-Brown
Home
Attributions
BioMembrane
Overview
Collagen
Elastin
p-Aramid
Latex
UV Protection
Float
Overview
Gas production
BioSensor
Overview
Chromoproteins
Fluorophore-Quencher
Aptamer purification
Practices
Overview
Integrated Human Practices
Outreach
Life Beyond the Lab
Collaborations
InterLab Study
Team Collaborations
BioBricks
Protocols
Protocols
Software Design
Modeling
Notebooks
Collagen & Elastin
P-aramid
Latex
UV protection
Aptamer Purification
Gas Production
Chromoproteins
Fluorophore-Quencher
Nylon
Interlab Study
Team
Requirements
Medals
Safety
p-Aramid team member Anna introduces the p-aramid project
Why aramids?
For our exploration purposes, it is important to have a membrane that can withstand harsh environments without bursting. While our collagen-elastin fiber and latex polymer display promising properties of elasticity and strength, their ability to withstand high physical stress is limited. Due to its rigidity (70,500 MPa [1]), toughness (2,920 MPa [1]), and low density (1.44 g/cm³ [1]), Kevlar®  is highly capable of meeting the unique needs of inflatable modules and habitats. It can act as both a lightweight balloon reinforcement and shield against atmospheric debris. In addition, it is also highly resistant to chemical, thermal, and physical wear--making it ideal for durability in a harsh environment.  Today, a Kevlar® composite is used as a protective outer covering for the Bigelow Space Module on the International Space Station for exactly these reasons. We wanted to make an aramid fiber similar in structure to Kevlar so that it can exhibit those same desirable properties that Kevlar possesses.
Cartoon depiction of the detrimental impacts choosing of a weak balloon material.
Current industrial production of Kevlar® is based on materials which are derived from petrochemicals. The long term use of fossil fuels is hazardous to the environment and will become unsustainable as our fuel sources are exhausted. By replacing the monomers 1,4-phenylene-diamine and terephthaloyl chloride with a single biologically produced monomer, para-aminobenzoic acid (pABA), our team can manufacture poly-pABA, a cost-efficient and ecologically sound p-aramid Kevlar® analogue.  Our primary goal is to optimize the production of pABA by identifying and isolating genes relevant for pABA synthesis and transforming them in Escherichia coli (E. coli).
Experimental Design
Source: http://www-abell.ch.cam.ac.uk/research/Shikimate.html
Our p-aramid production is focused on the manipulation of E. coli’s native Shikimate metabolic pathway to maximize the synthesis of our monomer p-aminobenzoic acid. In this pathway, pABA is an intermediate for folate synthesis [2].
First, we wanted to enhance the activity of the first enzyme of the Shikimate pathway. This enzyme is 3-deoxy-D-arabinoheptulosonate 7-phosphate (DAHP) synthase. By introducing more copies of the gene and overexpressing this enzyme in cells, an increased amount of carbon will be funneled into the pathway. The pathway then proceeds and produces (after several intermediate products) chorismate, which is partially funneled to produce pABA. Blitzen blue plasmid was used to transform DAHP synthase into E. coli.
Second, our introduction of genes PabA, PabB and PabC facilitates the production of enzymes that are responsible for the conversion of chorismate to pABA. Chorismate is transformed into para-aminobenzoic acid through chemical reactions involving the enzymes 4-amino-4-deoxychorismate (ADC) synthase and 4-amino-4-deoxychorismate (ADC) lyase, with the former encoded by PabA, and the latter encoded by PabB and PabC. Our approach was to introduce extra copies of genes PabA, PabB, and PabC into our cells. PabA and PabB together produce ADC synthase, which is responsible for the conversion of chorismate to 4-amino-4-deoxychorismate. Expression of PabC results in ADC lyase, the enzyme that would convert 4-amino-4deoxychorismate to pyruvate and our desired product, pABA.
Since pABA is an intermediate in the folic acid pathway, pABA will be consumed in the pathway’s downstream chemical reactions. Dihydropteroate (DHP) synthase catalyzes the reaction that converts pABA to 7,8-dihydropteroate, a precursor to 7,8-dihydrofolate [3]. By knocking out the gene that expresses dihydropteroate synthase, folate production is halted, thus preventing the conversion of pABA to subsequent folate precursors. To carry this out, we planned to use CRISPR-Cas9 for excision of the gene and to transform into E. coli. To verify that the cells were successfully transfected, cells were grown in medium with selection markers, such as chloramphenicol and folate. Afterwards, the growing cells would be replated on medium containing only the selection marker. Colonies that did not survive on the replated medium are successful, because they are folate-dependent since their folate pathway was disrupted by the DHP synthase knockout.
We also wanted to take into account previous work that has been done on pABA production. Previously, the ETH Zurich 2012 team [4] had produced a biobrick containing pabA and pabB from E. coli (BBa_K909014), but did not see large increase in the amount of pABA being produced. However, we also wanted a biobrick with pabC from E. coli in order to investigate whether or not the pabC enzyme was limiting the rate of pABA production. We also introduced used the DAHP synthase and Cas9 gene knockout to affect other parts of the Shikimate pathway besides just the steps involving chorismate to pABA. Further theoretical work was done on optimizing pABA yield, which can be seen on our modelling page.
Previously, the primary motivation for pABA production was for UV protection. However, our focus on pABA was due to its value as a monomer for a biomaterial. Thus, we also considered more issues for post-production processing of pABA in order to allow it to better fulfill its purpose. We drafted a procedure to chemically process pABA, given that a sufficient quantity of pABA had been accumulated. After accumulating pABA, we then planned to polymerize it using trimethyl borate, after which we would use the standard Kevlar spinning procedure with sulfuric acid to produce a fiber.
Chemical structure of polymerized polyamide formed from pABA
Results
We attempted to introduce PabA and PabB into E. coli using the Biobrick part BBa_K909014 from the 2015 iGEM DNA Distribution Kit. From that, we planned to extract these genes in order to linearize them and combine them with the gene PabC through Gibson Assembly. However, our colony polymerase chain reactions (CPCR) of our transformed colonies were ~300 base pairs (bp) when we used the primers VF2 and VR, but we expected the plasmid to be approximately 2,700 bp. In empty pSB1C3 plasmid, the amplified region, using primers VF2 and VR, would be approximately 300 bp. After performing polymerase chain reaction (PCR) and sequencing the plasmid part, we confirmed that the plasmid was empty and could not be used for this crucial part of our project. In Figure 1, lanes 5 and 6 are 2 colonies picked from the transformation with BBa_K909014. Lanes 9 and 10 is NEB 2-Log DNA Ladder . The green arrow indicates the desired band size (~2,000-3,000 bp) and the red arrow indicates the actual band size (~300 bp).
Figure 1:  Gel Electrophoresis Results of Cells Transformed with BBa_K909014 after CPCR
To implement more copies of DAHP enzyme into E. coli, we used BBa_K1060000 . When we performed CPCR, we had a product of band size ~1,000bp, as shown in lane 8 and by the blue arrow of Figure 1. According to the BioBrick page, the product should be 1,053 bp. This was promising; however when we sequenced it, the results did not overlap with the published sequence.
References:
http://www.dupont.co.uk/content/dam/dupont/products-and-services/fabrics-fibers-and-nonwovens/fibers/documents
/Kevlar%C2%AE%20Technical%20Guide.pdf
Herrmann, K. (1995). The Shikimate Pathway: Early Steps in the Biosynthesis of Aromatic Compounds. THE PLANT CELL ONLINE, 7(7), 907-919.
http://dx.doi.org/10.1105/tpc.7.7.907
Dihydropteroate Synthase(DHPS). (2016). Biology.kenyon.edu. Retrieved 19 July 2016, from
http://biology.kenyon.edu/BMB/jsmol2012/Lyra_Rina/index.htm
PABA Results. (2016). 2012.igem.org (Team ETH Zurich). Retrieved 19 June 2016, from https://2012.igem.org/Team:ETH_Zurich/PABA/Results
