Team:Stanford-Brown/SB16 Notebooks p-Aramid
Stanford-Brown 2016
Toggle navigation
Stanford-Brown
Home
Attributions
BioMembrane
Overview
Collagen
Elastin
p-Aramid
Latex
UV Protection
Float
Gas production
BioSensor
Overview
Chromoproteins
Fluorophore-Quencher
Aptamer purification
Practices
Overview
Integrated Human Practices
Outreach
Life Beyond the Lab
Collaborations
InterLab Study
Team Collaborations
BioBricks
Protocols
Protocols
Software Design
Modeling
Notebooks
Collagen & Elastin
P-aramid
Latex
UV protection
Aptamer Purification
Gas Production
Chromoproteins
Fluorophore-Quencher
Nylon
Interlab Study
Team
Requirements
Medals
Safety
pABA future · Benchling
pABA future
Made with Benchling
Project: iGEM 2016
Authors: Eric Liu, Anna Le
Date: 2016-08-30
Tuesday, 8/30
**First few steps of each project are definitely doable and prob wont take too much time!** Currently there are 3 subprojects under pABA. In the end we hope to consolidate these subprojects into one. Our subprojects are:
1.
Introducing extra copies of pab,A,B,C to increase pABA production
2.
Knocking out the last enzyme of the pathway to stop pABA from being further converted into folate
3.
Overproducing the first enzyme in the pathway
Inventory
●
ELIM DNA tubes marked with blue X's along bottom row = don't use
●
Pink 1.5 mL tubes with "25" on them = miniprepped pab(A+B) cultures
●
WHITE "Psb1c3 bb __ PCR 9/1" tubesand BLUE "9/2 1/2/1b/2b" tubes are trash lol.
●
"pcas9 478" tube = undigested pcas9 plasmid that needs to be digested for Project #2. Plasmid size is 10k
●
Primers
○
"psb1c3 backbone"
○
"c gibson" = primers for pabC to make gibson ends
○
"dahp" - primers for DAHP synthase gene to make gibson ends
●
pc9 PCR tube is probably useless.
●
General sizes
○
pcas9 = 10k
○
psb1c3 backbone (const. promoter + terminator) = 2.7kb
○
blitzen blue backbone = 4kb
○
dahp synthase = 1k
To do
Project 1: Extra copies of pabA,B,C to increase pABA production
Biobrick pabC
DAY 1 Transform PabA+B (DNA is in iGEM 2015 distribution kit #2) under 13P into NEB 5alpha cells. (Not making protein, so no need to use T7 cells. Also 5-alpha is more efficient anyways.) --- AL - I've done this already, I'm leaving the plates out over the weekend (9/3-9/5). They should be by the RT shaker)
■
Amy redid this 9/6 because 9/2 transformation failed.
■
Redid transformation 9/14.
●
Sequenced colonies, ran gels - gels did not have bands 2.7kb!! And sequencing results were trash as well!!
DAY 2 If transformation is successful (if not, we need to retransform lol)
If plates are lawns.. we'll have to replate. Scrape some colonies and inoculate a 1mL LB-chlor culture. Replate them after 2-3 hours.
If plates AREN'T lawns... Pick ~3 isolated colonies and do cPCR (use VF2 and VR. Size of plasmid should be 2.7kbp. Annealing T of VF2 and VR = 66C). After cPCR ends, run a gel to confirm the size of the PCR product and perform PCR cleanup. Send out for sequencing (use both forward and reverse, because the plasmid is so big). Grow 5 mL liquid cultures of the picked colonies and leave overnight (use 5 mL LB + 5 uL chlor media)
Don't forget to put them in the fridge after you're done!
●
Redid transformation 9/20!! 4th time??
●
9/21 - transformation successful. Sent sequencing, ran gels. Bands showed 1.2kb?? Still wrong size.
●
Trevor! suggested sequencing the DNA from the actual transformation kit. Will run gels/PCR cleanup tomorrow.
DAY 3 Mini prep the overnight liquid cultures if the sequences are correct.
Get the linearized pSB1c3 backbone by doing PCR on the pabA+B minipreps. Use purple 1.5 mL "pSB1c3 backbone" primers in the box. Run PCR cleanup.
Gibson Assembly with pabC (with Gibson ends) and linearized pSB1c3 backbone and transform into 5-alpha.
NOTE: PCR tubes "p1 and p2" and "g1 and g2" are unknown ???? One set is the amplified pabC with ends for gibson. So try gibsoning with both, but if you're not feelin it - try g1 and g2 tubes and we can troubleshoot from there. Both pairs of tubes should be in the box.
DAY 4 If Gibson fails and we can't successfully determine which PCR tubes contain the pabC with Gibson ends, then we have to remake it.
PCR the 1.5 mL tubes of pabC labeled "p2" and "p4". The primers for this PCR are labeled "pabC Gibson primer fwd/rev".
Alternative DAY 4 If Gibson's successful!!!
Do colony PCR and sequence (using VF2, VR or the pSB1c3 BB primers in the box). Run gel, do PCR cleanup.
Make liquid cultures of colonies with the right size (~2k bp)
DAY 5 Biobrick pabC to submit
Biobricking process == Dilute miniprepped DNA to 25 ng/uL using MQ water. Put it in the foil tray..?? (Unclear what we put it in.) DNA needs to be dried, so leave it for 4-6 hours in laminar flow hood.
Actual instructions are google-able -- google "Biobrick submission" lol
Make a construct with ABC using restriction digest. (Ask Trevor?? Ask if there are certain primers, enzymes etc. Look for protocol.) - I don't expect this to be done this week - so no worries
Project 3: Overproduction of DAHP (enzyme in 1st committed step to making chorismate)
DAY 1 Find where DAHP synthase liquid culture is lol
Mike might know where it is
??? Might have to regrow if can't be found
Unclear what to do. Regrow from DAHP plates in 4C fridge. Otherwise if plates don't exist, bug eric lol
●
DAHP synthase/chorismate ++ cell are found in a plastic gray box w/ yellow tape in the 4C fridge. 9/14/16
●
Incubated DAHP liquid culture after picking from agar stab 9/14/16
●
9/16 - made a DAHP plate for storage. Miniprep liquid culture and ran PCR using the DAHP primers to grab gene from psb1c3 plasmid. Sequenced.
●
9/20 - sequencing was good, will use with blitzen blue backbone
Get the backbone from Blitzen Blue cells
DAY 1 Pick 2 colonies from the blitzen blue plate. Make cultures - grow overnight in LB-chlor. I made LB-chlor plates with "blitzen blue 9/2 paba" on them by the RT shaker - use this plate.
If it's a lawn - replate
●
Incubation of blitzen blue culture (picked from plate) @11 am. 9/14/16
DAY 2 Miniprep Blitzen Blue cultures. Perform PCR using Blitzen Blue backbone primers which haven't been diluted yet. You'll need to dilute the ELIM DNA tubes labeled "bbbb" lol.
I think there is a 1.5 mL white tube labeled "blitzen blue backbone 9/1/16" -- I forgot what it is. I think its a miniprep I did last week but was really low in concentration. Disregard this tube!
●
9/20 realized blitzen blue is meant to be grown in LB-amp. Wondering why there are cells in my LB culture and what I miniprepped. We don't have time to figure that out, so I'm using Theresa/Cynthia's blitzen blue synthesized DNA plasmid
DAY after we find DAHP Put DAHP synthase gene into the chromoprotein vector
Miniprep DAHP synthase culture and PCR with diluted primers in the box labeled "dahp".. think they're yellow/white 1.5mL tubes?
Gibson two fragments together (DAHP + blitzen blue). Transform into 5alpha cells. PLATES SHOULD BE LB-AMP.
If successful, pick 2 colonies/grow culture
Induce with IPTG, measure production.
Project 2: Knock out dihydropteroate synthase/DHP (gene that controls enzyme changing pABA to dihydropteroate) using CRISPRCas9
DAY 1 Make LB +chlor/+folate plates
Recipe for making LB chlor/folate plates can be googled. It should be ~1.6mg/L.
Basically follow https://www.addgene.org/static/data/07/52/50a35908-85d2-11e2-92ba-003048dd6500.pdf for your protocol. It's multiple steps so you don't have to feel pressured to complete them all??
DAY 1 Digest pCas9 with BSA1 site, run gel, there should be a big band, do gel purification (run all 50 uL of it onto the gel! USE QIAGEN GEL EX KIT). The undigest pcas9 plasmid tube should be labeled "pcas9 478" in the box, use it for ur digestion!!
Anneal oligos - done already. Tube labeled "pcas9 anneal oligos". Use 2 uL for the ligation step.
DAY whatever - Transform into NEB5alpha in (regular) LB-chlor plates
If successful, perform replicates (LB-chlor and folate) - choose ones that die, because it means these cells are folate independent/the cells we want.
After cells have grown, make liquid cultures from these colonies and then run the assay the liquid cultures (accounting for opitcal density and stuff/aka they should be very similar)
We can perform thin layer chromatography using cell lysate/media.
------------------
Additional Notes
ok so there are 3 (and 1/2) different subsubprojects on the aramid project right now
1- so we are trying to see if extra copies of pabA,B,C will increase the amount of paba we make
2-also if the knocking out the gene that controls enzyme changing pABA to dihydropteroate
3- overproducing dahp (precursor to chorismate)
1: (make sure to make 2 of each thing, check part 3 for deets)
the biobrick pabA+B already has pabA and pabB in a construct - need to test amt
*I have already isolated the pabC from the E. coli and just have gibson left
-miniprep the 4 tubes from pabA+B, steal the backbone with PCR, then gibson
-we will then get the pabC by itself in biobrick format
-submit the biobrick
-THEN: make a construct with ABC using the restriction digest
-----------------------
2:
crispr cas9 to knockout the gene
PROTOCOL: (if you need to do it again ask mike/trevor first, then jesica if all else fails)
-----
digest pCas9 with bsa1 site
cut out the big vector of gel (10 kb)
do gel purification on big band
have my ordered oligos
phosphorylate them (add phosphate to 5' end)
then anneal
-add 2 ul of annealed oligo
transform into t7, neb5a, e. coli w
p cas9 chlor resistant
plate transformation
do replicates
choose ones that die!!
---
make +folate + chlor plates, regular chlor plates
---
after u get the compound (paba), thin layer chromatography to make sure. use cell lysate/cell culture
--
use culture samples with same optical density
-----
what has been done is possibly all up until the transformation. redo digestion/pcas9 + gex
need to plate transformation on folate + plates.
to make the folate + plates, try different concentrations, but first try with saturation. (ask charlie if need help?)
after plating onto the folate + plates, then use the velvet thing to make a duplicate plate onto a REGULAR plate
-identify colonies that were unable to grow on the regular chlor, (-)folate plate, these are the successful transformations
- make liquid cultures from these colonies and then run the assay the liquid cultures (accounting for o-density and stuff)
check online for saturation ~1.6mg/L
3: dahp/chorismate overproduction (dahp synthase/chorismate ++)
-we already have the gene as part of a biobrick
-have pcr'ed out the gene and we want to put it into the vector that came from the chromoproteins
bb - backbone, bbbb = blitzen blue backbone primers
regular BB = psb1c3 + temr + promotor
-after, we will gibson the dahp synthase gene into the blitzen blue backbone
-its IPTG inducible, so make sure to induce when we stick it in culture
-what the labwork actually is:
-all primers are ordered
-miniprep the blitzen blue backbone and then pcr with the bb bb primers
-miniprep the dahp synthase culture and then pcr with the primers
-gibson the two fragments together
-take the gibson stuff and transform it into cells and make sure to induce it!
-measure production
****-make two of every single liquid culture that was used for part 1
-make one regular, other one make it electrocompetent, then transform in the gibson product and induce dahp synthase
-measure and compare the differences with dahp overproduction
