Team:Stanford-Brown/SB16 Notebooks FQsensor
Stanford-Brown 2016
Toggle navigation
Stanford-Brown
Home
Attributions
BioMembrane
Overview
Collagen
Elastin
p-Aramid
Latex
UV Protection
Float
Gas production
BioSensor
Overview
Chromoproteins
Fluorophore-Quencher
Aptamer purification
Practices
Overview
Integrated Human Practices
Outreach
Life Beyond the Lab
Collaborations
InterLab Study
Team Collaborations
BioBricks
Protocols
Protocols
Software Design
Modeling
Notebooks
Collagen & Elastin
P-aramid
Latex
UV protection
Aptamer Purification
Gas Production
Chromoproteins
Fluorophore-Quencher
Nylon
Interlab Study
Team
Requirements
Medals
Safety
Biosensing (IRES, FQ) · Benchling
Biosensing (IRES, FQ)
Made with Benchling
Project: iGEM 2016
Authors: Michael Becich, Amy Weissenbach, Julia Gross
Dates: 2016-06-24 to 2016-10-03
Friday, 6/24
-Transformed NEB5a cells with BBa_E0040, the GFP plasmid to use as the backbone for the IRES construct (Julia + Amy)
Monday, 6/27
-GFP plates had no colonies. Realized we had the wrong antibiotic, so we re-plated with the correct plates. (Julia + Amy)
Tuesday, 6/28
-Checked GFP plates, they had non fluorescent colonies on them. (Julia + Amy)
-Debugging Heidelberg MAWS Software Issues
-Designing Initial Fluorophore-Quencher Aptamer System
-Intramolecular quenching with PEG linker? Fluorophore on 5' end, Quencher on 3' end
-Tau Day--Considering Biosignatures (ATP)/Toxins (Pb 2+)/Minerals for Biomining to sense
-Pivoting to other systems: Quantum Dots, Augmenting Atomic Force Microscopy, Mechanisms of attachment
Wednesday, 6/29
-Did colony PCR on colonies from the GFP plates, and ran the products on a gel confirming that they had transformed with the correct insert.
-Ordered primers for linearizing the GFP backbone with ends compatible with Gibson assembly of the IRES construct.
-Started liquid cultures of GFP colonies, and of the Heidelberg BioBricks that came in today: BBa_K1614019, BBa_K1614002, and BBa_K1614016. (Julia + Amy)
-Plan to talk with Christina Smolke for ideas about aptamer applications and members of Stanley Qi's Lab for mechanisms of attachment (dCas9, Gal4/UAS)
Thursday, 6/30
-Cryostocked the Heidelberg BioBricks, as well as the GFP colonies.
-Minipreped both the Heidelberg BioBricks and the GFP plasmids.
-PCR amplified the GFP plasmids with the primers that have IRES overhangs, in preparation for doing a Gibson assembly of the IRES fragment into the GFP backbone. (Julia + Amy)
Friday, 7/1
-Planned FQ Constructs too expensive--> Need to pivot to cheaper, 2-piece system
-Use freeze-drying as a positive control for testing attachment of biosensors
-Discussed idea of BioFoundry
Tuesday, 7/5
-Flexible linkers (GGSGGS) superior to rigid ones
-Engineered cheaper solution to FQ design using 2 oligonucleotides rather than 1 (drawback--sensor must be in solution initially for mass action)
Various_FQ_Designs.jpg
Source: http://www.sciencedirect.com/science/article/pii/S1046202305001039
Wednesday, 7/6
-Linearized GFP plasmid with forward and reverse IRES overlap primers through PCR. (JLG/AW)
-FQ Design tradeoff=sterically decreasing non-specific interactions vs. flexibility/separation
-Don't use high GC content
-A/T are susceptible to UV radiation
-No Guanine Quadruplexes
Thursday, 7/7
-Previous GFP plasmid linearizing PCR failed, tried again with a higher annealing temp and using Q5 instead of One-Taq (TP/AW)
-Dilution Assay with 5' Biotin Fluorescein to guage brightness of fluorophores-->how will our sensing platform be sensed once glowing?
-Detectable at mM with naked eye, uM-nM with fluorometers
IMG_2632.JPG
-Idea: denature aptamer once bound to recover target in as high of a concentration possible, measure with Nanodrop
Friday, 7/8
IRES
●
Higher annealing PCR of 7/7 was successful: 394ng/ul of fairly pure DNA. Ran the product out on a gel, and observed a large band at 3kb, with two smaller bands further forward (probably primer-dimer/non specific amplification). Cut the 3kb band out, and ran it through the Epoch gel extraction kit according to manufacturer instructions. The extraction kit did not work (yield was 6ng/ul of highly impure DNA). Set up new linearizing PCR with the same DNA, primers, Q5, and thermocycler settings as 7/7. (JLG/AW)
FQ Sensors
●
Tested various concentrations of ATP (100mM - .001nM) on the spectrophotometer. Determined that at concentrations that low, comparing absorption spectra was not a viable way to distinguish between different concentrations. Decided as a result to order the fluorescently labeled ATP DNA aptamer construct, rather than ordering ATP aptamer alone and attempting to gauge efficacy via comparing before and after incubation ATP concentrations.
●
Put in the order for biotin-ATP aptamer-flourophore oligo, and hybridizing sequence-quencher oligo, both of which should arrive on 7/12. (MB)
ATP_Fluorophore_Biotinylated A new nucleotide sequence entered manually
ATP_Quencher A new nucleotide sequence entered manually
-Consider radio-labeled ATP to differentiate (cheaper assays include dNTP's vs. NTP's)
-Preliminary Plate Layout for 1:1 Fluorophore-Quencher Ratio (1:1 biotinylated fluorophore to Streptavidin-coated plate)
A
B
C
D
E
F
G
H
I
J
1
-Control (No Sensor)
10mM ATP
1mM ATP
100uM ATP
10uM ATP
1uM ATP
100M ATP
10nM ATP
1nM ATP
No ATP
2
10uM Sensor->
10mM ATP
1mM ATP
100uM ATP
10uM ATP
1uM ATP
100M ATP
10nM ATP
1nM ATP
No ATP
3
1uM Sensor->
10mM ATP
1mM ATP
100uM ATP
10uM ATP
1uM ATP
100M ATP
10nM ATP
1nM ATP
No ATP
4
10uM Sensor->
10mM GTP
1mM GTP
100uM GTP
10uM GTP
1uM GTP
100M GTP
10nM GTP
1nM GTP
No GTP
5
1uM Sensor->
10mM GTP
1mM GTP
100uM GTP
10uM GTP
1uM GTP
100M GTP
10nM GTP
1nM GTP
No GTP
Table1
Monday, 7/11
-IRES linearizing PCR of 7/8 was successful. Preformed PCR extraction on the results,   . Preformed Gibson assembly with the backbone, and the linearized construct. Transformed the Gibson plasmids into NE5a chemical competent E. coli cells, plated at 1:10 and 1:100 dilution factors on Amp plates. (AW/JLG)
Tuesday, 7/12
●
1..5uL of Plasmids obtained from Dave Dingal/Tony Gao of Stanley Qi Lab for possible attachment mechanisms:
○
pSLQ3604 --> pcDNA3-hNECD1-Gal4
○
pSLQ3829--> pEV-UAS-hNECD-dCas9-VPR
IRES gel:
IRES_7_12_16_PNG.PNG
Wednesday, 7/13
●
When oligo's arrive, suspend both to 100 uM Stock
●
Prepare Wash Buffer: Tris-Buffered Saline (25mM Tris, 150 mM NaCL)  and 0.1% BSA, 0.05% Tween -20
●
Adjusted concentration of biotinylated FQ to 10 ug/mL in wash buffer
●
Excitation/Emission 495/520 for FAM-6
IRES_7_13_16_PNG.PNG
Thursday, 7/14
●
Prepare 5x SSCT, 2x SSCT, and PBST Buffers according to https://tools.thermofisher.com/content/sfs/manuals/MAN0011250_Pierce_StreptavidinCoat_96Well_UG.pdf, https://www.thermofisher.co.nz/Uploads/file/Scientific/Applications/Lab-Plasticware-Glassware-Supplies/Thermo-Scientific-Nunc-Immobilizer-Streptavidin-Application-example-PCR-ELISA.PDF (ELISA Protocol Nunc immbolizer)
●
Wash wells 3-times with 5x SSCT Buffer
IRES Gel:
IRES_7_14_16_PNG.PNG
Friday, 7/15
●
Dilute fluorophore and quencher to 100uM in EB
●
Dilute to 2 uM Stocks in 5x SSCT
●
Add 100uL to each well: https://www.sigmaaldrich.com/content/dam/sigma-aldrich/docs/Sigma/Bulletin/s6940bul.pdf
●
Incubate for 1 hour (room temp/shaking/dark) to let binding of biotin/streptavidin to occur and let F/Q hybridize
●
Add ATP gradually and measure fluorescence over time period
IRES Gel:
IRES_7_15_16_PNG.PNG
Monday, 7/18
●
Strong signal observed in solution. Focus on proof of concept for first iteration
●
Weak signal for wells that are not attached-->troubleshoot?
●
Test different buffers? F/Q combinations? Incubation times?
●
Signal spike seems to be instantaneous, so how to better measure live?
Tuesday, 7/19
●
Results from first FQ fluorescence measurements inconclusive.
●
Back to drawing board. Rather than simulating ELISA conditions, research better conditions for FQ System: http://onlinelibrary.wiley.com/doi/10.1002/anie.200501214/full, http://www.sciencedirect.com/science/article/pii/S1046202305001039
Wednesday, 7/20
●
Refined experiment included a hybridization buffer (Tris, NaH2PO4, Na2HPO4) to first let F and Q bind.
●
Crucial improvement was 1x Rxn Buffer for ATP sensing (300mM NaCl, 5mM MgCl2, 25mM HEPES)
●
Concession--must accept 30 second delay between introduction of ATP and Plate Reader (limited by equipment imaging capabilities)
Thursday, 7/21
FQ Sensor
●
ATP vs. GTP focused study (triplicate of wells)
●
Performed stepwise concentration plate reader assay to confirm fluorophore-quencher system
○
Incubated 20nM fluorophore and 20nM quencher in hybridization buffer for 1 hour at room temp. with gentle agitation
○
Took 11 readings across 5 minute intervals, gradually increasing ATP/NTP concentration according to (limited by stocks of NTP's):
A
B
C
1
Time (min.)
μL 100mM ATP Added
μL 100mM NTP Added
2
0-5
1.5
1.5 GTP
3
6-10
1.5
1.5 UTP
4
11-15
1.5
1.5 CTP
5
16-20
10
1.5 GTP
6
21-25
10
1.5 UTP
7
26-30
10
1.5 CTP
8
31-35
15
15 UTP
9
36-40
15
15 CTP
Table2
●
Troubleshooting: Photobleaching, Further Buffer Optimization, Attached Sensor Still not as successful-->how to capture inital jump?
Friday, 7/22
FQ Sensor
●
Analyzed Fluorescent Plate Reader Data for Sensing Capabilities (observed 9-fold change in fluorescence at 30mM and linear trend with ATP target concentration)--Confirms Literature
FQ_Stepwise_Graph_FoldChange.JPG
●
Reperformed assay with dATP, dGTP, ATP, GTP
●
Attempt to video tape fluoresence peak as added
Sunday, 7/24
●
Video unsuccessful
●
Contacted author Yingfu Li on responsiveness of aptamer signaling in nanoengine paper to get insights on other imaging techniques
Tuesday, 7/26
●
Design a more useful FQ Assay-->Utilize plate reader at Stanford that can measure at certain heights in well
Wednesday, 7/27
IRES gel:
IRES_7-27.PNG
IRES_rAIP_7-27.PNG
Friday, 7/29
●
Use two identical plates (one Streptavidin-coated)
●
Achieve following concentrations in each well: 1 uL F, 1 uL Q in 2x SSCT buffer
●
Incubate gently while configuring plate reader to optimal fluorescence settings (Gain=65, Height=6.50mm), readjusted before each round of measurment
●
Took baseline at 12:50pm and every 5 minutes recorded fluorescence. NTP was added to each well to appropriately achieved 10-fold serial dilutions from 1 mM to 1 nM (target total volume=150uL)
IMG_3245.JPG
●
Inconclusive results were achieved for both the attached and unattached variations.
●
Next steps: troubleshoot binding with ITC, fix buffers (1x rxn buffer has lower signal, but better performance)
Saturday, 7/30
●
Considerations for Biosensing: FQ Improvement, IRES Characerizaion, SELEX for Perchlorate/p-ABA Aptamer pool enrichment, PQQ DNAzyme
●
Building a biodevice for better characterization of FQ Activity:
IMG_4543[1].JPG
Tuesday, 8/2
●
FQ Final Validation:
1.
92 uL Rxn Buffer + 4 uL F/ 4 uLQ (1:1 F:Q)
2.
92 uL Hybridization Buffer + 4 uL F/ 4 uLQ (1:1 F:Q)
3.
94 uL Rxn Buffer + 2 uL F/ 4 uLQ (1:2 F:Q)
4.
94 uL Hybridization Buffer + 2 uL F/ 4 uLQ (1:2 F:Q)
●
Incubate for 2 hours.
●
Pipette into plates and dilute with 100uL 2x Rxn Buffer
●
Incubate in plate gentl shaking for 30 minutes before preparing to image
●
Settings (Height=6.25mm, Gain=70)
●
Stepwise increase of ATP/GTP/dATP/dGTP concentrations
○
1.5 uL of 1 mM, 15uL of 1 mM, 1.5 u of 100mM, 15uL of 100mM, 15uL of 100mM, 15uL of 100mM, 15uL of 100mM (*ran out in some trials)
Final Data:
FQ_FINAL_GRAPHS_UNATTACHED_ATTACHED_10-13-2016.xlsx
1
2
3
4
5
6
7
8
9
10
11
12
A
B
C
D
E
Control
ATP 1F:1Q
ATP 1F:2Q
F
dATP  1F:1Q
dATP  1F:2Q
G
GTP 1F:1Q
GTP 1F:2Q
H
dGTP 1F:1Q
dGTP 1F:2Q
Plate Layout
A
B
C
D
E
1
dATP Added
[dATP]
AVG_RFU (Gain 70) Normalized to first trial
F/F0
SD (n=2)
2
0
0
167
1
0
3
0.0000015
0.00990099
168
1.005988024
0.243938
4
0.0000165
0.0990991
173
1.035928144
0.654097
5
0.0001665
0.99107143
207
1.239520958
0.169981
6
0.0016665
9.10655738
383
2.293413174
0.410408
7
0.0031665
15.9924242
820
4.910179641
0.966051
8
9
10
ATP Added
[ATP]
11
0
0
33
1
0
12
0.0000015
0.00990099
33
1
0.388719
13
0.0000165
0.0990991
38
1.151515152
0.965383
14
0.0001665
0.99107143
230
6.96969697
1.894411
15
0.0016665
9.10655738
551
16.6969697
1.547684
16
0.0031665
15.9924242
808
24.48484848
2.436957
17
18
19
dGTP Added
[dGTP]
20
0
0
134
1
0
21
5
0.00990099
134
1
0.064052
22
0.0000165
0.0990991
132
0.985074627
0.108625
23
0.0001665
0.99107143
130
0.970149254
0.032115
24
0.0016665
9.10655738
139
1.037313433
0.114406
25
0.0031665
15.9924242
121
0.902985075
0.120107
26
27
28
GTP Added
[GTP]
29
0
0
135
1
0
30
5
0.00990099
134
1
0.097803
31
0.0000165
0.0990991
132
0.985074627
0.045836
32
0.0001665
0.99107143
138
0.970149254
0.011335
33
0.0016665
9.10655738
143
1.037313433
0.033559
34
0.0031665
15.9924242
163
0.902985075
0.095841
35
Unattached FQ Results
A
B
C
D
E
1
dATP Added (mmoles)
[dATP]
AVG_RFU (Gain 70) Normalized to first trial
F/F0
SD (n=2)
2
0
0
45
1
0
3
0.0000015
0.00990099
43
0.955555556
0.243937944
4
0.0000165
0.099099099
43
0.955555556
0.654097158
5
0.0001665
0.991071429
66
1.466666667
0.169981119
6
0.0016665
9.106557377
255
5.666666667
0.410407614
7
0.0031665
15.99242424
460
10.22222222
0.96605097
8
0.0046665
21.9084507
579
12.86666667
1.109325242
9
0.0061665
27.04605263
602
13.37777778
0.670327385
10
ATP Added
[ATP]
11
0
0
33
1
0
12
0.0000015
0.00990099
33
1
0.388719421
13
0.0000165
0.099099099
38
1.151515152
0.96538261
14
0.0001665
0.991071429
230
6.96969697
1.894411086
15
0.0016665
9.106557377
551
16.6969697
1.5476843
16
0.0031665
15.99242424
808
24.48484848
2.436957161
17
0.0046665
21.9084507
923
27.96969697
1.990514531
18
0.0061665
27.04605263
1094
33.15151515
3.210473445
19
dGTP Added
[dGTP]
20
0
0
28
1
0
21
0.0000015
0.00990099
35
1.25
0.064051714
22
0.0000165
0.099099099
35
1.25
0.108624815
23
0.0001665
0.991071429
32
1.142857143
0.032114939
24
0.0016665
9.106557377
33
1.178571429
0.114405538
25
0.0031665
15.99242424
44
1.571428571
0.120106536
26
0.0046665
21.9084507
43
1.535714286
0.053652181
27
0.0061665
27.04605263
47
1.678571429
0.06508307
28
GTP Added
[GTP]
29
0
0
0
0
30
0.0000015
0.00990099
31
1.107142857
0.097802637
31
0.0000165
0.099099099
33
1.178571429
0.045835897
32
0.0001665
0.991071429
35
1.25
0.011335169
33
0.0016665
9.106557377
37
1.321428571
0.033558944
34
0.0031665
15.99242424
44
1.571428571
0.095841188
35
0.0046665
21.9084507
50
1.785714286
0.157668473
36
0.0061665
27.04605263
61
2.178571429
0.126588149
37
Attached Sensor
Unattached_Sensor_Final_Image_Legend.PNG
Attached_Sensor_Final_Image_Legend.PNG
Wednesday, 8/3
●
Transformed http://parts.igem.org/Part:BBa_K1499004 from previous year's distribution  kit into NEB5-alpha and plated on Cm plate
●
IRES PAGE gel:
IRES_PAGE_8-3.PNG
Thursday, 8/4
●
Miniprepped 2 colonies from CBD-Streptavidin part
●
Contacted Alaina Schumate from 2014 iGEM team to confirm that this part has a promoter, terminator, and a FlagLumioHisTag as ambiguously specifed-->Never confirmed
○
We told her that we would purify and confirm with our fluorescent assay we developed
●
Sent in for sequencing to verify expected sequence on Wiki with VF2/VR
●
Grew up miniprepped DNA in 2 cultures
IRES PAGE gel:
Friday, 8/5
●
Induced cultures with IPTG in 50mL flasks for large scale protein extraction to begn next week
●
Confirmed presence in soluble fraction with small protein extraction (see duplication of lanes in middle at desired length of 39 kDa protein):
IRES_8-5.PNG
●
Protein from BBa_K1499 004 indeed has FlagLumioHisTag as shown by luminescent gel image after HisTag column purification.
●
Thus, we have already improved this part by verifying unknown design aspects, now onto functional verification!
Monday, 8/8
IRES PAGE gel:
IRES_8-8.PNG
Wednesday, 8/10
IRES PAGE gel:
IRES_8-10.PNG
Friday, 8/12
IRES_8-12.PNG
Monday, 8/15
●
Extracted proteins from soluble fraction, ran gel to confirm presence, and purified:
Thursday, 9/22
●
Took extracted CBD-Streptavidin Part (purified earlier in August by Julia) and incubated (shaking, room temp, 2 hours) in 2x B&W Buffer with equimolar amounts of biotinylated fluorophore-quencher
○
1190ng/uL starting concentration, 100uM Biotinylated Fluorophore and Quencher (Recipe: 100 uL Protein, 31.7 uL Fluorophore)
●
Pipetted 25 uL onto 9 wax-based wells on cellulose filter paper. Let sit over night (CBD takes 2-3 days to set in)
Friday, 9/23
●
Re-imaged Plate on reader and Typhoon scanner (Abmax 495 nm Emmax 520 nm) with Trevor's help
●
Took picture with iPhone under gel scanner
Pre-Wash
Monday, 9/26
●
(Stanford School resumes)
●
Imaged in morning
●
Washed each square with 5x 1mL MilliQ Water (w/ 1mM ATP)
●
Imaged again on Typhoon Scanner and took picture:
Day 0
Tuesday, 9/27
●
Washed again 5x 1mL MilliQ Water (w/ 1mM ATP) and re-imaged
1 Day
Friday, 9/30
Low signal traces were still apparent on day  after 4 washes
4 Days
Monday, 10/3
After repeating wash each morning for a week, the signal had all but disappeard from even the experimental square:
1 Week
