Team:Stanford-Brown/SB16 MedalRequirements
Stanford-Brown 2016
Toggle navigation
Stanford-Brown
Home
Attributions
BioMembrane
Overview
Collagen
Elastin
p-Aramid
Latex
UV Protection
Float
Gas production
BioSensor
Overview
Chromoproteins
Fluorophore-Quencher
Aptamer purification
Practices
Overview
Integrated Human Practices
Outreach
Life Beyond the Lab
Collaborations
InterLab Study
Team Collaborations
BioBricks
Protocols
Protocols
Software Design
Modeling
Notebooks
Collagen & Elastin
P-aramid
Latex
UV protection
Aptamer Purification
Gas Production
Chromoproteins
Fluorophore-Quencher
Nylon
Interlab Study
Team
Requirements
Medals
Safety
Medal Requirements
Bronze medal requirements :
• Register for iGEM and attend the Giant Jamboree
• Provide all required deliverables on time: team wiki, poster, presentation, project attribution, registry part pages, sample submissions, safety forms, and judging form. Meet all deliverables on the Requirements page (section 3) .
• Create an attribution page on the team wiki that details who worked on each aspect of our project. This page must clearly attribute work done by the students and distinguish it from work done by others, including host labs, advisors, instructors, sponsors, professional website designers, artists, and commercial services. Required link: https://2016.igem.org/Team:Stanford-Brown/Attributions
• Document at least one new standard BioBrick Part or Device related to our project and submit the part to the iGEM Registry (submissions must adhere to the iGEM Registry guidelines ), or document a new application of a BioBrick part from a previous iGEM year. See part BBa_K2027000 .
Silver medal requirements :
• Experimentally validate that at least one new BioBrick Part or Device of your own design and construction works as expected (see BBa_K2027001 ). Document the characterization of this part in the Main Page section of the Registry entry for that Part/Device. This working part must be different from the part you documented in Bronze medal criterion #4. Submit this part to the iGEM Parts Registry.
• Collaborate with another registered iGEM team in a significant way. Learn about our collaboration with Exeter here .
• Identify, investigate, and address human practices issues in the context of our project. iGEM projects involve important questions beyond the bench, for example relating to (but not limited to) ethics, sustainability, social justice, safety, security, and intellectual property rights. We refer to these activities as Human Practices in iGEM. Demonstrate how your team has identified, investigated and addressed one or more of these issues in the context of your project (see the Human Practices Hub for more information). Learn about our sustainability work here .
Gold medal requirements:
• Expand upon silver medal human practices activity by integrating the investigated issues into the design and/or execution of our project.  Learn about our sustainability work here .
• Improve the function or characterization of an existing BioBrick Part of Device and enter it in the Registry. The part cannot be from our 2016 part number range. Learn how we improved 2014 Stanford-Brown-Spelman's Cellulose Cross-linker .
• Demonstrate functional proof of concept of our project; must consist of a BioBrick device, not a single BioBrick part (Remember, biological materials may not be taken outside the lab ). Check out our biodevice for a cellulose-based ATP Biosensor .
• Show our project working under real-world conditions by demonstrating that our functional proof of concept can work under simulated lab conditions (Remember, biological materials may not be taken outside the lab ). Learn about how we tested our Chromoproteins in the upper-atmosphere here .
iGEM Prizes:
All teams are eligible for special prizes at the Jamborees. Your team will be evaluated by the judges if (1) you have documented your special prize activity on your wiki on the specified page, AND (2) you have explained on this form why you think your team is eligible for this prize.
•Integrated Human Practices - Required link: https://2016.igem.org/Team:Stanford-Brown/Integrated_Practices
We evaluated both planetary protection and environmental sustainability concerns associated with our project.  At the beginning of the summer, we reached out to NASA and Brown astrobiologists who helped us brainstorm a balloon design that would both aid scientific research while mitigating the risk of contaminating other planets.  These conversations framed our project: we focused on developing components that would not contribute any live material to the final design.  We also looked deeper into the real environmental sustainability of our proposed methods of materials production.   We met with representatives from Mango Materials, Stanford’s Civil and Environmental Engineering department, the Joint BioEnergy Institute, World View, and Genentech, among other groups, to help us develop a clearer understanding of whether and how our biomaterials might provide “cleaner” alternatives to traditional manufacturing materials.  Hopefully our published findings will help inform other student teams’ critical perspectives on synthetic biology’s contribution to environmental sustainability.
•Education and Public Engagement - Required link: https://2016.igem.org/Team:Stanford-Brown/Engagement
We hosted teaching events at Bay Area and New York Maker Faires, where we held interactive activities that demonstrated the importance of utilizing synthetic biology in society. These activities were aimed for wide ranges of both age and experience levels with synthetic biology; children, parents, and other biology teachers all gave positive feedback on our educational activities. Through explaining physical models of DNA helices, teaching parents how to do fruit DNA extractions with their children, and explaining our summer work in an easily digestible manner, we were able to make synthetic biology concepts tangible and enjoyable to the several hundred thousand attendees at the faires. To further expand upon our synthetic biology teachings, we also created an interactive crowdsourcing board regarding the field’s applications, where we asked participants how a biologically synthesized balloon would best be used.
•Model - Required link: https://2016.igem.org/Team:Stanford-Brown/SB16_Modeling
In order to augment our capabilities within the lab, we used mathematical modelling to get our bioballoon off the ground. Our model simulates balloon payload and flight capabilities in various atmospheric conditions with the ability to modulate membrane material, mass, gas production, and other balloon features. Earth allows us only a small range of the atmospheric properties necessary to fly an exploration balloon in a habitat like Mars; simulation is necessary to predict the behavior under the extremes. Modelling also played a smaller role in the metabolic engineering of the Shikimate pathway in Escherichia coli for para-aminobenzoic acid and melanin production for UV protection and kevlar polymerization. Using elementary flux analysis, we were able to get a better understanding of E. coli production methods and examine the effects of knocking out certain genes to increase pABA production.
•Measurement - Required link: https://2016.igem.org/Team:Stanford-Brown/Measurement
Proteins have long been known to respond differently to thermal fluctuations in their environment.  Through our research, we identified a cassette of chromoproteins that by either losing or changing their color, specifically indicate the temperature of their environment.  This color change is also highly sensitive in a range of temperatures for each chromoprotein, providing a fast method for measuring the temperature in a sample.  By characterizing the color degradation of each chromoprotein, we are able to produce a novel biological tool for measuring temperature in a standardized manner. Additionally, no extensive research has characterized these color-temperature profiles before. Having quantified the color expression profile dependency on temperature of each chromoprotein, we incorporated unique chromoproteins into a cassette of multiple chromoproteins.  Through this cassette, we are able to expand beyond a single chromoprotein’s range of detection to measure a wider range of temperatures via using each chromoprotein’s characteristic color-temperature degradation curve.
•Applied Design - Required link: https://2016.igem.org/Team:Stanford-Brown/Design
Our chromoprotein research not only characterized the way in which chromogenic proteins lose or change colors in response to heat, but also was expansive enough to identify a large enough span of chromogenic proteins that differentially respond to temperature. Using this differential response, we were able to construct a biological thermometer made up of six different chromogenic proteins. Each of the proteins in this set loses or changes color at a specific and distinct temperature. Lastly, we added a cellulose binding domain to these chromogenic proteins to allow them to bind to cellulose, allowing for the creation of a biologically-based paper thermometer. Having a toolset of thirteen differently-colored chromoproteins is like a functional paint set for a biological artist. We were able to use wax-based printer paper to create images like the NASA logo with our chromoproteins and even carry one experiment up into the upper atmosphere with the permission of iGEM HQ and the US EPA.
•Software Tool - Required link: https://2016.igem.org/Team:Stanford-Brown/SB16_Software
We created a software program for automated protein optimization and gibson assembly primer design.  When supplied a list of DNA or amino acid sequences representing proteins, the program will not only codon optimize the sequence for a user-defined species, but also remove any common restriction sites from the sequence.  The program can also process multiple sequences, allowing the user to process a large number of sequences quickly and identifying which sequences have issues.  Under the gibson assembly primer module, our program accepts a list of fragments to be stitched together, calculates the homology needed to gibson the fragments together, and designs primers for fragment homology PCR extension matching a user-defined melting temperature.  These optimized protein DNA sequences or Gibson primers are then provided to the user in a text file.  For ease of use, our program also reports errors and procedures in terminal and the output file.
Team Parts:
To help the judges evaluate your parts, please identify your highest quality part for each of the following prizes:
•Best New Composite Part: BBa_K2027039
Learn what makes our Latex Operon special here .
•Best New Collection: Chromogenic Protein Collection
( BBa_K2027016 , BBa_K2027017 , BBa_K2027018 , BBa_K2027019 , BBa_K2027020 , BBa_K2027021 , BBa_K2027022 , BBa_K2027023 , BBa_K2027024 , BBa_K2027025 , BBa_K2027026 , BBa_K2027036 , BBa_K2027001 , BBa_K2027027 , BBa_K2027028 , BBa_K2027029 , BBa_K2027030 , BBa_K2027031 , BBa_K2027032 , BBa_K2027033 , BBa_K2027034 , BBa_K2027043 , BBa_K2027035 , BBa_K2027042 , BBa_K2027041 )
This summer, we explored the heat-dependent nature of chromogenic proteins. Chromogenic proteins are popularly used in a variety of scientific applications, but are commonly used only as color indicators. No extensive research has been done to demonstrate how temperature alters the color profile of these proteins. Our research not only characterized how chromogenic proteins lose or change colors in response to heat, but also identified a large enough span of chromogenic proteins that differentially respond to temperature. By taking advantage of this differential response, we were able to construct a biological thermometer consisting of six unique chromogenic proteins. Each of the proteins in this set loses or changes color at a specific temperature. Lastly, we added a cellulose binding domain to these chromogenic proteins to allow them to bind to cellulose, allowing for the creation of a biologically-based paper thermometer.
