<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stanford-Brown_SB16_BioSensor_Chromoproteins skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stanford-Brown/SB16 BioSensor Chromoproteins</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Stanford-Brown 2016</TITLE></P><DIV class="container"><NAV class="navbar navbar-inverse navbar-fixed-top"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" id="BioballoonLabel" href="https://2016.igem.org/Team:Stanford-Brown">Stanford-Brown</A><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Stanford-Brown">Home</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Attributions">Attributions</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">BioMembrane</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_Collagen">Collagen</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_Elastin">Elastin</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_p-Aramid">p-Aramid</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_Latex">Latex</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_UV">UV Protection</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Float</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Float_Gas">Gas production</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle active" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">BioSensor</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioSensor_Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioSensor_Chromoproteins">Chromoproteins</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioSensor_FQsensor">Fluorophore-Quencher</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_AptamerPurification">Aptamer purification</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Practices</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Human_Practices">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Integrated_Practices">Integrated Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Engagement">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Practices_Exploration">Life Beyond the Lab</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Collaborations_Interlab">InterLab Study</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Collaborations">Team Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Parts">BioBricks</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Software">Software Design</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Modeling">Modeling</A></LI><LI class="dropdown-header">Notebooks</LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_CE">Collagen &amp; Elastin</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_p-Aramid">P-aramid</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_Latex">Latex</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_UV">UV protection</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_AptamerPurification">Aptamer Purification</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_Gas">Gas Production</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_Chromoproteins">Chromoproteins</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_FQsensor">Fluorophore-Quencher</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_Nylon">Nylon</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_Interlab">Interlab Study</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Team">Team</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Requirements</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_MedalRequirements">Medals</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Safety">Safety</A></LI></DIV></DIV></NAV></DIV><DIV class="figure-legend">Chromoproteins team member Taylor introduces the chromoproteins subproject</DIV><DIV class="container"><DIV class="row rowT"><DIV class="col-sm-12 col-PT"><H1 class="sectionTitle-L firstTitle">Why Monitor Temperature?</H1></DIV></DIV><DIV class="row"><DIV class="col-sm-12 pagetext">In planetary exploration, monitoring temperature changes in the atmosphere is important for predicting weather patterns and atmospheric stability. At different heights above the planet’s surface, the temperature can vary drastically. The way that the temperature changes from hotter to cooler than back to hotter temperatures can be used to predict how weather and wind patterns will move across that region. This is helpful to know in planetary exploration because it can help predict the weather that various rovers or exploration outposts would have to deal with without relying on expensive satellite equipment. Additionally, the vertical temperature profile of a certain region of a planet’s atmosphere can be used to predict how easily a payload will fall through the atmosphere and land on the planet’s surface [1].

On Earth, vertical temperature profiles can be obtained using electronic temperature sensors mounted on weather balloons. These balloons go up to different heights in the atmosphere and relay real-time measurements of the atmospheric temperatures. While these electronics are easy to obtain and replace if they are broken on Earth, in the context of planetary exploration, they are irreplaceable and unreliable because it would take years to send replacements if they break and are impossible to fix without also sending replacement parts along the mission. Thus, a biological temperature sensor was developed in order to address these concerns. This biological temperature sensor can be expressed in a small culture of E. coli and be easily replaced simply by maintaining a culture of that particular bacteria.
</DIV></DIV><DIV class="row"><DIV class="col-sm-6 pagetext-L"><DIV class="text">In order to build a biological temperature sensor, we used chromogenic proteins from the <A href="https://www.dna20.com/products/protein-paintbox#3">DNA2.0 paintbox</A>and from the iGEM registry (Figure 1). Chromogenic proteins or chromoproteins are colored proteins that have been extracted from a variety of organisms such as the sea anemone, <I>Actinia equine</I> [2] or have been synthetically produced (as was the case for the chromoproteins from DNA2.0). Like all proteins, chromogenic proteins are sensitive to heat; however, their sensitivity to heat is coupled with a visible color loss or change. Our research has shown that different chromoproteins respond to different temperatures by either losing their color or changing to a different color. These differential responses to temperature allowed us to build a biological temperature sensor.</DIV></DIV><DIV class="col-sm-6 img-R"><FIGURE class="fig"><FIGCAPTION>Figure 1: This shows the colorful liquid cultures of some of the DNA2.0 chromoproteins.</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-sm-12 pagetext-L"><DIV class="text">
The basic structure of a chromogenic protein consists of a large beta barrel and a chromophore which is supported by the beta barrel. The chromophore interacts with light and gives the protein its distinctive color. Additionally, the shape of the beta barrel is supported by water molecules that interact with the side chains of the amino acids that the beta barrel comprises of. These water molecules are essential for the chromogenic protein to maintain its structure. In response to heat, our hypothesis is that these water molecules are released from the beta barrel, thus allowing the barrel to collapse and the chromophore to no longer be supported enough to produce color [3]. Because planetary exploration requires a sensor that can function in a variety of environments, we tested our chromogenic proteins in a sealed environment and in an open air environment. The following sections details those experiments.</DIV></DIV></DIV><DIV class="row rowT"><DIV class="col-sm-12 col-PT"><H1 class="sectionTitle-L firstTitle">Closed System: chromoproteins in vivo</H1></DIV></DIV><DIV class="row"><DIV class="col-sm-12 pagetext"><DIV class="text">To test our chromoproteins in a closed system that did not allow them to interact with anything in the air, we spun down 5 mL cultures of each of our chromoproteins and added 30 µL of the pellet cells to a PCR tube along with 1 µL of 0.165 M EDTA. We then heated these PCR tubes for 5 min at a starting temperature of 40 °C, removed them from the thermal cycler, took a picture, then returned them to the thermal cycler for 5 more minutes at a temperature that was 5 °C higher than the previous. We repeated this process until we reached 100 °C. As can be seen in Figure 2, groups of chromoproteins lose their color at specific temperatures. Additionally, CupidPink and DonnerMagenta change from pink to purple at 70 °C and 80 °C respectively.</DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-8 img-L"><FIGURE class="fig"><FIGCAPTION>Figure 2: This is all of the chromoproteins at a variety of different temperatures. As the temperature gets higher they all lose their color at different temperatures.</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-sm-12 pagetext">Interestingly, we also observed that if there were certain cultures that were not showing color on a particular day, the heat treatments as described above were able to cause the color we would have initially expected to appear (Figure 3, 4, and 5).
</DIV></DIV><DIV class="row"><DIV class="col-sm-4 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 3: As can be seen above, the bottom row of five PCR tubes containing 30 µL of the cell pellet does not show any color. The chromoproteins in the bottom row are from DNA2.0: VirginiaViolet (left), MaccabeePurple, SeraphinaPink, TinselPurple, and DonnerMagenta.</FIGCAPTION></FIGURE></DIV><DIV class="col-sm-4 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 4: After these previously colorless cell pellets have been treated at 55 °C for 30 min, 60 °C for 30 min, and 60 °C for 30 min, color can be observed. This color then proceeds to disappear again as the temperature continues to increase. </FIGCAPTION></FIGURE></DIV><DIV class="col-sm-4 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 5: This time, the color does not return after heat treatments. </FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-sm-4 pagetext">Using the results of the experiment above, it was possible to build a thermometer from the select groups of chromoproteins that respond to different temperatures. Shown in Figures 6 and 7 is a prototype of how that would work. Until the temperature reaches the temperature indicated on the right, the color remains constant. Here is an example of how the thermometer would look at 80 °C (Figure 6) and at 100 °C (Figure 7). 	
</DIV><DIV class="col-sm-4 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 6</FIGCAPTION></FIGURE></DIV><DIV class="col-sm-4 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 7</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-sm-8 pagetext">After the cell lysate heat tests, we moved towards testing the cell lysate in an exposed air environment. This was done based on the idea that the color change we were seeing was dependent on the interaction of water molecules with the barrel of the chromoproteins. Therefore, we hypothesized that in the exposed air system, the water molecules would not be trapped with the chromoproteins and we might see color change occur sooner. To test this we set up an apparatus that would heat a glass petri dish evenly and allow us to take a video of the cell lysate as it was being heated up as shown in Figure 8 to the right. 
Twelve chromoproteins that were expressing good color in our cultures were selected for this assay and can be seen below in Figure 9. After this, the glass petri dish was removed from heat and 20 µL of DI water were added to each of the chromoproteins. This was done on the theory that the chromoproteins had not denatured, but rather had lost their color because all of the water molecules which were supporting the structure of their barrel had been driven out by the heat. As can be seen in the image below, some of the chromoproteins regained color when water was added, but this color is different from the starting color. This could be due to the fact that the heating caused the chromoprotein structure to destabilize to a lower energy structure which causes it to have a different color when the water is added to support the barrel.
</DIV><DIV class="col-sm-4 imgcol-R"><FIGURE class="fig"><FIGCAPTION>Figure 8: Setup for conducting temperature testing on chromoproteins.</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-sm-4 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 9: 30 µL droplets of cell lysate taken from spun down 5 mL cultures of cells expressing chromoproteins were placed on a glass petri dish. Order of Chromoproteins: 
Top row: VixenPurple
Second row: asPink, tsPurple, AE Blue
Third row: TinselPurple, CupidPink, PrancerPurple, DreidelTeal
Fourth row: spisPink, scOrange, BlitzenBlue
Fifth row: DonnerMagenta </FIGCAPTION></FIGURE></DIV><DIV class="col-sm-4 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 10: The glass petri dish above was heated to 85 °C which caused all of the chromoproteins to lose their color as is seen below.  </FIGCAPTION></FIGURE></DIV><DIV class="col-sm-4 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 11: Chromoproteins from figure 10 after being brought back to room temperature. </FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-sm-12 pagetext">In order to investigate the role that water molecules play in the color change we were observing in our chromoproteins in response to heat, we used a lyophilizer to remove the water from the chromoproteins without heating them. A lyophilizer is a device that operates at sub freezing temperatures under a vacuum which causes water to move directly from the solid phase to the vapor phase. This vapor is then pumped out of the system. Below is an image of two chromoproteins (PrancerPurple and ScroogeOrange) inside the lyophilizer chamber (Figure 12). After an hour in the lyophilizer, the filter paper was removed from the chamber and photographed (Figure 13). Water was then added to to the right column of the filter paper which caused the color to return (Figure 14) as can be seen by comparing the top and bottom rows of this image. These results show that while color loss does occur when water is removed from the system, it does not have the same effect as heating the sample. In the heat testing experiments, PrancerPurple changed irreversibly to pink when heated whereas in the lyophilization experiment, it regains its color fully. <I>This supports our hypothesis that the color changing in response to heat is dependent on both water loss and a structural change of the protein itself due to heat.</I></DIV></DIV><DIV class="row"><DIV class="col-sm-4 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 12 </FIGCAPTION></FIGURE></DIV><DIV class="col-sm-4 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 13: Two replicates of PrancerPurple (top) and ScroogeOrange (bottom) are shown after 1 hour of lyophilization. </FIGCAPTION></FIGURE></DIV><DIV class="col-sm-4 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 14</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-sm-12 pagetext"><DIV class="text">Similar experiments were done with the AE Blue-CBD fusion protein. With this protein, we saw that the blue color changed from blue to purple. This purple color immediately went away when the sample was removed from the chamber, suggesting that our hypothesis that the AE Blue chromoprotein’s color changing capabilities are dependent on the presence of water inside the barrel. To further confirm this, we tested how the AE Blue-CBD fusion protein would respond to heat in a closed PCR tube. Since the PCR tube prevents water from being released from the system, the water molecules that support the barrel of the chromoprotein cannot be driven out when the tube is heated. Because of this, we no longer see the color change from purple to blue. Instead the chromoprotein turns a bluish-green color, likely indicative of the protein simply denaturing in response to heat. This is a picture of the AE Blue-CBD protein before heat (Figure 15) and this is a picture after the PCR tube was heated to 80 °C for 5 mins (Figure 16). </DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-6 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 15 </FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 16 </FIGCAPTION></FIGURE></DIV></DIV><DIV class="row rowT"><DIV class="col-sm-12 col-PT"><H1 class="sectionTitle-L firstTitle">Open System: purified chromoproteins </H1></DIV></DIV><DIV class="row"><DIV class="col-sm-6 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 17: The Gibson Assembly Process for creating tagged chromoprotein constructs.</FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6 pagetext-R">
	To delve further into the question of what exactly is happening when these chromoproteins are heated, we used Gibson Assembly to clone the gene coding for the iGEM biobrick AE Blue and the 12 DNA2.0 chromoproteins into pSB1C3. We also added a FLAG, lumino and 6x histidine tag in order be able to extract the expressed protein using nickel column purification. The lumino tag is a specific six amino acid sequence that binds to the <A href="https://www.thermofisher.com/order/catalog/product/LC6090">Lumio<SUP>TM</SUP> Green</A> [4] which allows the fusion of the lumino tag and the chromoprotein to be detected on an SDS-Page Gel without having to run a staining protocol. The FLAG tag allows for anything after its sequence to be cleaved off of the protein when the extracted protein is incubated with enterokinase [5].This was done in case the lumino or histidine tag interfered with the chromoprotein structure.
                 </DIV></DIV><DIV class="row"><DIV class="col-sm-12 pagetext">Additionally, a cellulose binding domain was added to each of the chromoproteins. This cellulose binding domain was from the BioBrick <A href="http://parts.igem.org/Part:BBa_K1321366">BBa_K1321366</A> which also has a GFP fused to it. For the purposes of our experiments, we isolated the cellulose binding domain from this part using PCR. The cellulose binding domain sequence was isolated from <I>Cellulomonas fimi</I> and has been shown to bind irreversibly to cellulose [6]. In order to ensure that the addition of this larger protein did not interfere with the structure of the chromoproteins, we extracted both the chromoprotein without the cellulose binding domain and with the domain and ran the same heat testing experiments.
The extracted protein was concentrated using microfiltration tubes then allowed to dry on cellulose sheets which had wax wells printed on them. These cellulose sheets with the dried chromoprotein were then placed in a preheated oven and the temperature was increased at five minute intervals until a temperature change was observed.
</DIV></DIV><DIV class="row rowT"><DIV class="col-sm-12 col-PT"><H1 class="sectionTitle-L firstTitle">Prototyping a biological thermometer</H1></DIV></DIV><DIV class="row"><DIV class="col-sm-4 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 18: Gel run of our chromoproteins and Flag-Lumino His tag. Ladder(left), DonnerMagenta, VirginiaViolet, ScroogeOrange, LeorOrange, DreidelTeal, VixenPurple, SeraphinaPink, CupidPink, TinselPurple, PrancerPurple, MaccabeePurple, BlitzenBlue </FIGCAPTION></FIGURE></DIV><DIV class="col-sm-8 pagetext-R">After performing a Gibson Assembly to place the DNA sequence for the DNA2.0 chromoprotein and the sequence for the Flag-Lumino-His Tag, we did a colony PCR of the cells expressing color for all twelve of the DNA2.0 chromoproteins. Pictured to the left is a picture of the gel from this colony PCR using the VF and VR iGEM primers (Figure 18). As can be seen in the image, there is a strong band near the 1000 bp mark where we would expect to see it. Next, we set up a large (250-500 mL) liquid culture for each of the cells expressing the chromoprotein then extracted the expressed protein using 3 mL <A href="https://www.thermofisher.com/order/catalog/product/88221">HisPur Ni-NTA nickel columns</A>. The extracted protein was then dialyzed and concentrated using microfiltration centrifuge tubes in order to perform heat tests. 
</DIV></DIV><DIV class="row"><DIV class="col-sm-6 pagetext-L">To heat test the extracted protein, we pipetted 3 ul of the concentrated protein onto a sheet of cellulose paper with wax wells printed on it. After it dried we pipetted another 3 µl into the same well (Figure 19). Once it was dry for the second time we put it into the oven that was preheated to 55 °C. CupidPink, VixenPurple, and TinselPurple lost their color at this temperature but after 2 minutes PrancerPurple still has some of its color so the temperature was raised to 60 °C. It took 1.5 minutes for the oven to heat up to this temperature and by the time it got up to 60 °C PrancerPurple has lost its color (Figure 20). We then took the chromoproteins out of the oven and let them sit on the bench. After sitting for 3 minutes they had regained none of their color, but when rehydrated with 1 µl of milliQ water they regained their color (Figure 21). These chromoproteins were then heated again but instead at 40 °C for one minute. No color change was observed and so the temperature was raised to 45 °C and CupidPink, TinselPurple, and VixenPurple lost their color in between these temperatures around 42 °C. PrancerPurple still had its color at 45 °C after a minute it was raised to 50 °C and it began to lose its color. The chromoproteins were then rehydrated with 1 µl of milliQ water and they gained their color back. This test shows a reversible color change that is seen at a relatively low temperature. Interestingly CupidPink did not change to purple and then lose its color like we saw in cell lysate testing.
</DIV><DIV class="col-sm-2 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 19: Chromoproteins before heat. Order is (top to bottom), CupidPink, PrancerPurple, TinselPurple, and VixenPurple </FIGCAPTION></FIGURE></DIV><DIV class="col-sm-2 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 20: Chromoproteins after heat </FIGCAPTION></FIGURE></DIV><DIV class="col-sm-2 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 21: Chromoproteins after water has been added </FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-sm-12 pagetext">
We were then able to move into heat testing of all 12 DNA2.0 chromoproteins with FLAG lumino his-tags. We pipetted 3 µl of extracted protein into each well and then let dry and added another 3 µl as seen in Figure 22. Once this was dry we placed it into the oven that was preheated at 20 °C. We then began raising the temperature a degree C at a time until all 12 chromoproteins had lost their color Figure 16. This process took an hour and the temperature was raised from 20 °C to 60 °C over this time period. The chromoprotein and the temperature it lost its color at is: PrancerPurple (55 °C), CupidPink (47 °C), TinselPurple (48 °C), VixenPurple (50 °C), MaccabeePurple (55 °C), SeraphinaPink (47 °C), LeorOrange (57 °C), ScroogeOrange (56 °C), BlitzenBlue (60 °C), DreidelTeal (Not colored in this experiment so no color changed observed), DonnerMagenta (55 °C), VirginiaViolet (53 °C). The chromoproteins were then taken out of the oven and rehydrated with 3 µl of milliQ water and they regained color Figure 23. We then added a second set of wells with chromoproteins by again pipetting 3 µl into a well, letting it dry and adding another 3 µl to the well. We tested the previously tested chromoproteins against the newly pipetted chromoproteins to see if the temperature change would occur at a different temperature once a chromoprotein has already been heated. We placed these chromoproteins into a 40 °C oven and raised the temperature by a degree C over 6 minutes up to 60 °C. The chromoproteins all lost their color again and lost their color at the same temperature as before and compared to the already once heat tested chromoproteins. We then again rehydrated all 12 chromoproteins with 3 µL of milliQ water and the color came back. This test shows a reversible color change for all 12 chromoproteins at lower temperatures than seen in cell lysate tests and also shows this heat testing can be done multiple times with consistent results. Because of the range of temperatures the chromoproteins lose color at there is the application for a paper base thermometer. The temperature results were slightly different for the 4 chromoproteins we had previously tested most likely because the 4 chromoproteins had been sitting longer on the bench top after purification than the other 8 chromoproteins.
</DIV></DIV><DIV class="row"><DIV class="col-sm-4 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 22: 12 chromoproteins on the cellulose sheet before heat is added. PrancerPurple (left), CupidPink, TinselPurple, VixenPurple, MaccabeePurple, SeraphinaPink, LeorOrange, ScroogeOrange, BlitzenBlue, DreidelTeal, DonnerMagenta, VirginiaViolet.</FIGCAPTION></FIGURE></DIV><DIV class="col-sm-4 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 23: 12 chromoproteins after being heated to 60 °C. All lost color. </FIGCAPTION></FIGURE></DIV><DIV class="col-sm-4 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 24: 12 chromoproteins after being rehydrated. Color regained. </FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-sm-12 pagetext">
	In the interest of moving towards a biologically based paper thermometer, we also did a Gibson Assembly to add a cellulose binding domain to the end of our chromoprotein. This was first done successfully with AE Blue from the iGEM registry. Once this construct was successfully transformed and expressed in bacteria, the protein was extracted and dialyzed. The concentrated protein was then pipetted onto cellulose sheets and underwent a series of heat tests. It was found that, while this protein initially appeared blue (Figure 25), it begins to turn purple at 55 °C (Figure 26) and this purple color becomes more pronounced as temperature is increased (Figure 27). Interestingly, when allowed to cool back to room temperature, the blue color returned (Figure 28). 
</DIV></DIV><DIV class="row"><DIV class="col-sm-3 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 25</FIGCAPTION></FIGURE></DIV><DIV class="col-sm-3 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 26</FIGCAPTION></FIGURE></DIV><DIV class="col-sm-3 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 27</FIGCAPTION></FIGURE></DIV><DIV class="col-sm-3 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 28</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-sm-12 pagetext">
	We then created NASA sticker using the AE blue fusion protein to coat a cellulose sheet as the blue background of the NASA logo and added the lettering and red swoosh with stickers. This sticker was then fixed to a glass container using double sided tape. Hot water was added to the glass container and after 1 minute of exposure to heat, the AE blue-CBD fusion protein turned purple (Video 1). Cold water was then added to the glass container and the AE blue-CBD fusion protein returned to its original blue color (Video 2). This testing shows a reversible, color changing, temperature sensitive fusion protein and is the prototype of a biologically based paper thermometer.
	</DIV></DIV><DIV class="row"><DIV class="col-sm-6"><DIV class="figure-legend">Video 1: Heat being added to our NASA sticker made from AE Blue-CBD fusion protein.</DIV></DIV><DIV class="col-sm-6"><DIV class="figure-legend">Video 2: Cold water being added to our NASA sticker.</DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-6 pagetext">	
We also added a cellulose binding domain to the DNA2.0 chromoproteins using Gibson assembly. The sequence for these proteins were transformed into bacteria, expressed, extracted, and fix to cellulose sheets. Figure 29 shows the 2 cellulose binding domain chromoproteins (PrancerPurple and LeorOrange) on cellulose sheets. After being allowed to set for three days, we tested the binding capabilities of the cellulose binding domain by washing the cellulose sheet the chromoproteins were bound to with a steady stream of water for 1 minute. After this washing, no color change was detected, indicating that the cellulose binding domain was effective at fixing the chromoproteins to the sheet (Figure 30).
</DIV><DIV class="col-sm-3"><FIGURE class="fig"><FIGCAPTION>Figure 29: Before cellulose binding domain testing. PrancerPurple-cellulose binding domain (top) and LeorOrange-cellulose binding domain (bottom. </FIGCAPTION></FIGURE></DIV><DIV class="col-sm-3"><FIGURE class="fig"><FIGCAPTION>Figure 30: After cellulose binding domain testing. PrancerPurple-cellulose binding domain (top) and LeorOrange-cellulose binding domain (bottom).</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row rowT"><DIV class="col-sm-12 col-PT"><H1 class="sectionTitle-L firstTitle">Testing the prototype</H1></DIV></DIV><DIV class="row"><DIV class="col-sm-8 pagetext-L">
In collaboration with the <A href="http://stanfordssi.org/teams/balloons">Stanford High Altitude Ballooning Team</A>, we field tested our AE Blue-CBD fusion protein at a high altitude. This AE Blue-CBD fusion protein was extracted from the cells that were expressing it and purified using dialysis to ensure that only the fusion protein remained for this experiment. Prior to the flight we obtained permission from the US Environmental Protection Agency and iGEM HQ. It is important to note that nothing living was sent into the environment to be tested and that our sample was recovered. No environmental contamination occurred as a result of this experiment. 

Using our extracted fusion protein, we plated 20 µL drops onto a cellulose sheet that had had wax wells printed onto it (Figure 31). Every well seen in this image is from the same chromoprotein. This chromoprotein is the same as was seen earlier in the NASA logo heat test. Because there are higher levels of UV radiation in the atmosphere than are in our lab environment, we used UV-opaque and UV-transparent glass to cover the top and bottom row of the grid. The center row was left exposed to the elements as a control. These were then mounted onto the payload of the Stanford High Altitude Balloon Team’s balloon (Details in Figures 32, 33, and 34) and were readied for launch!
</DIV><DIV class="col-sm-4 imgcol-R"><FIGURE class="fig"><FIGCAPTION>Figure 31</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-sm-4 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 32</FIGCAPTION></FIGURE></DIV><DIV class="col-sm-4 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 33</FIGCAPTION></FIGURE></DIV><DIV class="col-sm-4 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 34</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-sm-7 pagetext">
A GoPro was mounted on the payload as well in order to catch inflight data. In the image to the right, taken at 10000 feet from the surface, the chromoproteins can be seen starting to change their color from blue to purple (Figure 35). A tracking device placed on the balloon allowed for the payload to be recovered once the balloon had burst. The image below was taken of the fusion proteins immediately after payload recovery (Figure 36). As can be seen, the once-blue fusion proteins are now purple. This purple color was persistent for six hours before eventually returning back to blue which is unlike previous heat tests in that usually the blue color comes back rather quickly.
</DIV><DIV class="col-sm-5"><FIGURE class="fig"><FIGCAPTION>Figure 35</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-sm-5"><FIGURE class="fig"><FIGCAPTION>Figure 36</FIGCAPTION></FIGURE></DIV><DIV class="col-sm-7 pagetext">
This could have occurred for a variety of reasons. The simplest explanation is that the exposure time to heat stress was much longer in the field test than in the lab. Usually our heat test experiments last anywhere from 5 to 30 minutes of constant heat exposure. As a result, it might be easier for the chromoproteins to regain their structure afterwards. Because all three of the levels of UV-exposure that were tested had the same degree of color change, it seems unlikely that the exposure to UV was responsible for the prolonged color change we witnessed. However, it is possible that even the UV-resistant glasses we used were not enough to block out trace amounts of UV and that these trace amounts of UV destabilized the structure of the chromoproteins in such a way that the prolonged color change occurred. 
Below is a video capturing the flight (Video 3).
</DIV></DIV><DIV class="row"><DIV class="col-sm-8"><DIV class="figure-legend">Video 3: A video documenting our chromoproteins' flight.</DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-12 pagetext references last">
References<OL><LI> Schlatter, Thomas W. &quot;Atmospheric Composition and Vertical Structure.&quot; National Oceanic and Atmospheric Administration (2009): n. pag. Web.</LI><LI>Shkrob, Maria A., Yurii G. Yanushevich, Dmitriy M. Chudakov, Nadya G. Gurskaya, Yulii A. Labas, Sergey Y. Poponov, Nikolay N. Mudrik, Sergey Lukyanov, and Konstantin A. Lukyanov. &quot;Far-red Fluorescent Proteins Evolved from a Blue Chromoprotein from Actinia Equina.&quot; Biochem. J. Biochemical Journal 392.3 (2005): 649-54. Web.</LI><LI>Langan, P.s., D.w. Close, L. Coates, R.c. Rocha, K. Ghosh, C. Kiss, G. Waldo, J. Freyer, A. Kovalevsky, and A.r.m. Bradbury. &quot;Corrigendum to “Evolution and Characterization of a New Reversibly Photoswitching Chromogenic Protein, Dathail” [J. Mol. Biol., 428, (2016), 1776–89].&quot; Journal of Molecular Biology 428.20 (2016): 4244. Web.</LI><LI>&quot;Lumio Green Detection Kit - Thermo Fisher Scientific.&quot;; N.p., n.d. Web. 2 Oct. 2016.</LI><LI>&quot;FLAG Tag Peptide|Versatile Fusion Tag|CAS# 98849-88-8.&quot; FLAG Tag Peptide|Versatile Fusion Tag|CAS# 98849-88-8. N.p., n.d. Web. 02 Oct. 2016.</LI><LI>Carrard, G., A. Koivula, H. Soderlund, and P. Beguin. &quot;Cellulose-binding Domains Promote Hydrolysis of Different Sites on Crystalline Cellulose.&quot; Proceedings of the National Academy of Sciences 97.19 (2000): 10342-0347. Web.</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>