Team:Stanford-Brown/SB16 Notebooks UV
Stanford-Brown 2016
Toggle navigation
Stanford-Brown
Home
Attributions
BioMembrane
Overview
Collagen
Elastin
p-Aramid
Latex
UV Protection
Float
Gas production
BioSensor
Overview
Chromoproteins
Fluorophore-Quencher
Aptamer purification
Practices
Overview
Integrated Human Practices
Outreach
Life Beyond the Lab
Collaborations
InterLab Study
Team Collaborations
BioBricks
Protocols
Protocols
Software Design
Modeling
Notebooks
Collagen & Elastin
P-aramid
Latex
UV protection
Aptamer Purification
Gas Production
Chromoproteins
Fluorophore-Quencher
Nylon
Interlab Study
Team
Requirements
Medals
Safety
Melanin/Radiation · Benchling
Melanin/Radiation
Made with Benchling
Project: iGEM 2016
Authors: Michael Becich, Elias Robinson, Taylor Sihavong
Dates: 2016-06-28 to 2016-10-03
Tuesday, 6/28
●
Cultured cells from former SB-iGEM team's plate with melanin-producing cells (Andy Weir-signed plate)
Wednesday, 6/29
●
Cells were dead -- need to pivot and find new way to produce melanin
●
Julia's old lab has melanocytes that overexpress melanin
●
Copper and tyrosine supplemented to growth media boosted pigmentation of melanin
●
Cambridge team from registry has version of MelA Tyrosinase necessary (Eumelanin superior to pheomelanin and neuromelanin)
●
Coating with Titanium Dioxide to produce a solar cell still an idea down the road
Tuesday, 7/12
●
Ordered mutMelA gene (tyrosinase) for synthesis from IDT based on Biobrick BBa_K274001 from iGEM Team Cambridge 2009 that was not in stock at the registry
○
This biobrick already contained one substituted nucleotide (C to T at the 1000th nucleotide) resulting in a Pro334 to Ser mutation that was shown to decrease time to onset of melanin production (Santos et al. 2008)
○
Changed Asp535 to Gly by substituting a single nucleotide --> this mutation was also shown in a more recent paper to shorten the time to onset of melanin production in E. coli (Chávez-Béjar et al. 2013)
●
Began designing primers to insert mutMelA into pSB1C3 backbone with appropriate promoters and RBS
●
Continued research on small melanin binding proteins
○
Many short polypeptides shown to be effective in binding melanin, linked to a radioactive subunit and tested in vitro and in vivo (in mice) for human melanoma cell localization and radiation delivery
○
One decapeptide discovered through the use of a phage display library called 4B4 shown to be effective at binding melanin (Ballard et al. 2011)
○
Many phage display library derived heptapeptides shown to be effective (Howell et al. 2007)
●
Discussed possibility of designing CBD-Melanin-Binding Peptide connector
ELIAS'S LAB NOTEBOOK
●
Ordered MelA gene from iGEM registry
○
Next: design plasmid with inducible promoter
○
Next: Order primers for linearisation and Gibson
●
Melanin binding peptides:
○
4B4: Fungal melanin-binding decapeptide
■
Sequence: YERKFWHGRH
■
Designed oligonucleotide sequences for repetetive sequence assembly (Kosuke's method)
■
Planning on assembling as follows:
Screen Shot 2016-08-19 at 9.43.32 am.png
■
Should also order & assemble irrelevant decapeptide PA1 as control
●
Sequence: LQYTPSWMLV
○
Set of heptapeptides: (Howel 2007)
■
Phage 4D had highest melanised/non-melanised absorbance ratio, expressed peptide
●
His-Thr-Thr-His-His-Arg-Asn
●
HTTHHRN
■
Phage 3D: Asn-Pro-Asn-Trp-Gly-Pro-Arg (NPNWGPR)
■
Phage 1C: Thr-Thr-His-Gln-Phe-Pro-Phe (TTHQFPF)
■
Irrelevant control: Asp-Gly-Ala-Trp-Met-Gly-Ala (DGAWMGA)
●
Optimising melanin production in E. coli
○
Cultivation media & growth conditions
■
Melanin producing cell media
●
50ml per culture of...
○
M9 minimal salts medium
○
Supplemented with 2 or 10 g/L glucose
○
0.1 mM IPTG
○
Required antibiotic (strain-specific)
○
0.4 g/L of L-tyrosine
○
20 ug/mL
Wednesday, 7/13
●
Decided to engineer a repetitive motif of the melanin binding decapeptide 4B4 as well as the most effective melanin binding heptapeptide documented in Howell et al. 2007 (will be refered to as 4D based on cell strain in paper).
○
Based on the sizes of the genes that would code for the two candidate polypeptides (30mer and 21mer), we decided to use the methods detailed in Fujishima et al 2015 for overhang based randomized DNA block assembly
○
Decided to attach melanin binding repetitive motif to cellulose binding domain
■
Plan on demonstrating proof of concept for all in one melanin binding/membrane binding linker to be used as a mechanism for even and resiliant melanin coating to increase UV radiation resistance
■
Constructed 45mer GS linker in silico to separate and preserve the functions of the CBD and the melanin binding domain
○
Designed single stranded oligonucleotides for DNA building blocks of 4B4 and 4D, as well as non-melanin binding control decapeptide PA1 and heptapeptide P601G from Ballard et al. 2011 and Howell et al. 2007 respectively
●
Designed full plasmid construct for melanin binding repetitive motif connected at its C-terminus by a 45mer GS linker to the cellulose binding domain in Biobrick BBa_K1321357, which is followed by a FLAG-lumio-HIS tag for purification, and finally a double terminator
●
Decided to linearize pSB1C3 backbone from meffRed BBa_K1033920, including BBa_J23110 constitutive promoter, Elowitz Represilator RBS BBa_B0034, and meffRed start codon since no start codon was included in repetitive melanin binding DNA block
●
Began designing primers for linearization of plasmid backbone and PCR amplification of biobricked CBD
○
Used SnapGene Gibson Assembly software for primer design
○
Included primer tail extensions to create identical overlaps with fragment inserts for later Gibson assembly
○
Ran into issues with inconsistent Tm values due to high GC content, as well as secondary annealing sites due to redundancies in pSB1C3 prefix and suffix
ELIAS'S LAB NOTEBOOK
●
Assembly of Kosuke fragments:
○
Needed enzymes:
■
T4 DNA ligase (NEB)
●
Cat #: M0202S (400,000 units/ml)
●
Cat #: M0202T (2,000,000 units/ml)
●
Size: 20,000 units
●
Cost: $64
■
T4 polynucleotide kinase (NEB)
●
Cat #: M0201S (500 units), $56
●
Cat #: M0201L (2,500 units), $224
●
Concentration: 10,000 units/ml (price: $224)
●
ELIM sequencing preparation:
○
Premix recipe:
■
Total volume: 15ul (in water)
■
Template quantity:
●
PCR products:
○
100-200bp
○
200-500bp --> 2-3ng
○
500-1000bp --> 6-8ng
○
1000-2000bp --> 10-15ng
○
>=2000bp --> 60-100ng
●
Single stranded: 100-150ng
●
Double stranded (plasmids): 500ng
■
Primer: 8pmol
●
Upregulate carbon flow to aromatic/L-tyrosine
○
Got cells with genomic deletions for shuttling carbon into L-tyrosine/aromatics pathway
Screen Shot 2016-08-19 at 10.07.59 am.png
Thursday, 7/14
●
Obtained melanocytes from Julia's former lab at VA Hospital and lysed in water to obtain melanin raw source
●
Ordered ten single stranded oligonucleotides from ELIM
○
Top and bottom strands for 4B4, 4D, PA1, P601G, and GS linker
○
Designed so that once annealed, each double stranded oligo would have a six nucleotide double Gly overhang for assembly and ligation
●
Continued designing PCR primers for linearization of intended backbone and amplification of CBD for Gibson assembly
○
Secondary annealing site issues solved by using NEBuilder online software for primer design: created longer primary annealing regions on primers and increased initial Tm so that secondary annealing would be outweighed by correct annealing of primers
ELIAS'S NOTEBOOK
●
Oligos arrived (not linker)
○
Fri 7/23: anneal oligos
○
Mon 7/26: Phosphorylate & ligate
○
Tues 7/27: run gel and extract correct band
●
DNA annealing:
○
Template:
■
Oligo forward (200uM) - 10ul
■
Oligo reverse (200uM) - 10ul
■
10x H Buffer - 2ul
■
DI H2O - 28ul
■
Total - 50ul
■
Final oligo concentration: 80mM
○
Mixed 4B4, 4D, PA1, P6016 in above formula
○
Ran in thermal cycler on "anneal" in Kosuke's folder
■
Let sit at 4˚C for 15 min then put in -20˚C
■
All oligos in orange iGEM 2016 box in -20˚C
Screen Shot 2016-08-19 at 10.13.20 am.png
●
Kosuke's paper: "an overhand-based ... random library" (2015)
○
Paper suggests annealing of oligos in thermal cycler by decreasing 10˚C every 5 minutes (we used 1-2 minutes)
○
Phosphorylation: used T4 kinase in 10x ligase buffer at 37˚C for 1h
○
Ligation: T4 DNA ligase added directly to above mix, incubated at room temperature for 2h
○
Adapter (blunt ends): phosphorylated, then mixed with phosphorylated DNA blocks in 1:8:1 ratio in 1x T4 DNA ligase buffer (5' adapter : DNA block : 3' adapter)
○
Can also incubate at 16˚C overnight to increase efficiency
Friday, 7/15
●
Eight of the oligos arrived  (everything but the GS linker)
○
Annealed oligos using the "Annealing of Template DNA" protocol
○
Stored in -20°C fridge for repetitive assembly upon arrival of GS linker
Saturday, 7/16
●
Prepared and assembled buffers and enzymes for overnight phosphorylation and ligation of repetitive DNA building blocks following "Phosphorylation and Ligation of Repetitive DNA Building Blocks"
○
Used T4 Polynucleotide Kinase and T4 DNA Ligase
○
Awaiting arrival of blunt ended GS linker for construction of repetitive motif
●
No oligos came, will wait until Monday for reaction
●
Monday, 7/18
ELIAS'S LAB NOTEBOOK
●
ELIM: melanin binding constructu linker arrived
○
Annealed single stranded oligos
○
Ready for Kosuke's repetitive DNA block assembly
●
Small repetitive fragment assembly:
○
6 PCR tubes:
A
B
C
D
E
F
G
1
Tube #
1
2
3
4
5
6
2
Contents
4B4
4D
4B4 + 4D
PA1
P6016
link
3
Label
10
7
10/7
d10
d7
link
Table1
○
*7 PCR tubes (2 runs of linker, used H buffer that was not entirely thawed for first)
○
Used protocol taped to next page
○
Let ligation mixes sit at 16˚C overnight
■
Started thermal cycler at 5:40pm
Screen Shot 2016-08-19 at 10.23.04 am.png
●
ELIM orders #234915
○
Ordered 9 single-stranded oligos
■
Melanin: primers for assembly of plasmid using CBD from BBa_K1321357 and FLAG-lumio-His tag
Tuesday, 7/19
ELIAS'S LAB NOTEBOOK
●
Removed previous day's ligation mixes from thermal cycler and put in -20˚C at 9:30am
●
Realised no linker was pit in DNA assembled block mixes --> will re-phosphorylate and ligate overnight
●
Running gel to test concentratin of loaded DNA to concentration of gel extracted bands
●
Lost gel --> no extractio
Screen Shot 2016-08-19 at 10.35.51 am.png
ELIM primers:
Screen Shot 2016-08-19 at 10.36.50 am.png
●
Ran PCR on CBD from Cynthia --> will gel tomorrow
●
Reran fragments in Danny's 3% gel
○
100ng, 115V --> had weird tail
●
Kosuke assembly pt 2
○
Will try two different link:DNA block ratios
■
1:10 and 1:20 block/link mixes labelled in blue, ligations of said mixes labeled in black with a blue dot
○
Gel extraction calibration
■
Aim for 0.2pmol product
●
600bp --> want >30ng/ul
●
300bp --> want >15ng/ul
○
Forgot to run mixed 4B4 and 4D + link
Wednesday, 7/20
Mel_7-19.PNG
melanin_binding_assembly1.gel
ELIAS'S LAB NOTEBOOK
●
Made 3% large gel: 12 lane wide comb
IRES_7-27.PNG
Screen Shot 2016-08-19 at 10.46.00 am.png
○
1.5ul ladder
○
~175ng DNA per well (besides CBD)
○
Mixed 1ul dye + 1.2-2ul DNA + x H2O
■
6ul per DNA
■
4B4: 222ng/ul --> 1.58ul + 3.42 H2O
■
PA1: 1.58ul --> ""
■
4D: 166ng/ul --> 2.11ul + 2.89ul H2O
■
P601G "" --> ""
●
PAGE gel:
A
B
C
D
E
F
1
1.1
1.2
2
3
4.1
4.2
2
ladder
4B4 1:10
-
-
4B4 1:20
4D 1:20
Table2
●
Ligation of mixes liquid cultured with meffRed E. coli from cryostock
○
Two 5ml liquid cultures, 37˚C ~16hrs
Thursday, 7/21
ELIAS'S LAB NOTEBOOK
●
Gel #3: melanin binding
A
B
C
D
E
F
G
H
1
1
2
3
4
5
6
7
2
ladder
4B4
PA1
4D
P601G
Mix
Ladder
Table3
●
Future gels run with a linker only
○
Lane --> compare to see if linker bound
●
Miniprepped meffRed cultures --> ~30ng/ul per
●
Ran PCR on miniprepped samples
○
4 tubes:
■
4B4 (50ul Q5 rxn)
■
4D (50ul Q5 rxn)
■
PA1 (25ul Q5 rxn)
■
P601G (25ul Q5 rxn)
○
Diluted ~29ng/ul miniprep product 1:40 with H2O, used 1ul per PCR tube
Screen Shot 2016-08-19 at 11.06.25 am.png
Friday, 7/22
ELIAS'S LAB NOTEBOOK
●
PAGE - polyacrylamide gel electrophoresis
○
SDS-PAGE used for proteins (sodium dodecyl sulfide)
○
DNA PAGE:
■
Buffers and solutions:
●
Ammonium persulfate (10% w/v) (TEMED catalyst?)
●
Acrylamide:bisacrylamide (29:1) (30% w/v)
●
5x TBE --> run in 0.5x or 1.0x TBE
■
Volume of reagents:
A
B
C
D
E
F
1
Gel %
30% acrylamide (29:1)
H2O (ml)
5x TBE (ml)
10% APS (ul)
TEMED (ul)
2
8%
3.2
6.4
2.4
200
10
3
10%
4
5.6
2.4
200
10
4
12%
4.8
4.8
2.4
200
10
Table4
●
CBD PCR round #2:
○
Diluted ~52ng/ul CBD template 1:60
■
Used 1ul in PCR mix
○
50ul Q5 recipe
●
Fridge gel:
A
B
C
D
E
F
G
H
I
J
K
L
1
1
2
3
4
5
6
7
8
9
10
11
12
2
Ladder
-
4B4
-
PA1
-
Mix
-
4D
-
P601G
Ladder
3
15ul
15ul
17.5ul
20ul
20ul
4
+3ul dye
+3ul dye
+3.5ul dye
+4ul dye
+4ul dye
Table5
●
Melanin binding gel extraction:
○
Plasmid backbone for 4 types of DNA blocks
■
1 = 4B4 - 50ul
■
2 = PA1 - 25ul
■
3 = 4D - 50ul
■
4 = P601G - 25ul
■
5 = CBD - 50ul
Monday, 7/25
●
Gel time:
○
Made 1% large agarose gel - running at 90V for 2hrs
○
Aiming for 1ug DNA of 4B4, 4D, and mix
○
Used big 12-well comb --> order:
A
B
C
D
E
F
G
H
I
J
K
L
1
1
2
3
4
5
6
7
8
9
10
11
12
2
Ladder
-
4B4
4B4
-
Mix
Mix
-
4D
4D
-
Ladder
Table6
○
500ng/well --> 1ng for 2 wells a piece
○
Tried using lab tape to link adjacent wells but agar and TAE solution made tape come undone
○
4B4 ligated product: 222ng/ul --> 2.25ul/well + 1ul dye + 2.75ul H2O
○
Mix: 194ng/ul --> 2.6 ug/well + 1ul dye + 2.4ul H2O
○
4D: 166 ng/ul --> 3ul/well + 1ul dye + 2ul H2O
●
The gel that works:
○
1% gel, ran in fridge, 500ng/well, 90V, 2hrs
○
Premix to load wells with: (in PCR tubes)
■
4B4: 4.5ul ligation product
●
2ul dye
●
5.5ul H2O
■
Mix: 5.2ul ligation product
●
2ul dye
●
4.8ul H2O
■
4D: 5ul ligation product
●
2ul dye
●
4ul H2O
Mel_7-25.PNG
●
Extractions of the gel that works:
○
4B4A: cut from ~280-400mer
○
4B4B: cut from ~500mer-700mer
○
*Same for 4DA and 4DB, Mix A and Mix B
■
Expecting enough DNA from gel extractions A to perform Gibson, unsure of thos efrom B
■
Will perform Gibson on all 5 GEX products
■
Will tarnsform and plate all 6 with a 1:4 and 1:40 dilution
○
Nanodrop showed fairly poor concentrations
Tuesday, 7/26
●
Ordered primers for Gibson Assembly of MutMelA into pSB1C3
ELIAS'S LAB NOTEBOOK
●
Gibson:
○
CBD: 73.4ng/ul --> 368bp --> 0.307pmol/ul
○
Lin backbones: ~2180bp
■
4B4: 91.6ng/ul --> 0.065pmol/ul
■
PA1: 51.8ng/ul --> 0.037pmol/ul
■
4D: 93.1ng/ul --> 0.066pmol/ul
■
P601G: 60.9ng/ul --> 0.043pmol/ul
○
Repetitive insert:
■
4B4Long: 1.4ng/ul
■
4B4Short: 4.2ng/ul
■
4DLong: 3.2ng/ul
■
4DShort: 7.7ng/ul
■
MixLong: 3.1ng/ul
■
MixShort: 4.5ng/ul
■
*4B4Short: ~350mer --> 0.019pmol/ul
■
4B4Long: ~600mer --> 0.003pmol/ul
■
MixShort: --> 0.020pmol/ul
●
NEBuilder mix:
○
Will only run short segments --> too dilute
■
2ul vector
■
5ul mix
■
1ul CBD
■
1ul Tag
●
PCR confirmation on GEX:
○
4DL - 4D long GEX (500-700mer)
○
4DS - 4D short GEX (250-400mer)
○
+ - positive control
○
4B4 - 4B4 short GEX (250-400mer)
○
Mix B - Mix short
Mel_7-26.PNG
●
Gel on PCR product (above)
○
Expect results in third row, weird results --> gel didn't work
A
B
C
D
E
F
G
1
1
2
3
4
5
6
7
2
Ladder
4B4
Mix
4DS
4DL
+
Ladder
3
~1000bp
~1000bp
~1000bp
~1300bp
All frags
Table7
●
Re-annealed all DNA blocks: used "anneal" protocol on thermocycler
●
Phosphorylation and overnight ligation
○
Use Kosuke's protocol for 4B4, 4D, PA1, P601G, Mix
■
20ul product + just link
■
Used 1:10 linker:DNA block ratio
○
Buffer MM: 6.5 x each tube --> add 15ul per tube
■
4B4: 4.55ul 4B4 + 0.46ul link
■
PA1: 4.55ul PA1 + 0.46ul link
■
4D: 4.55ul 4D + 0.46ul link
■
P601G: 4.55ul P601G + 0.46ul link
■
Mix: 2.27ul 4B4 + 2.27ul 4D + 0.46ul link
■
Link: 5ul link
●
PCR cleanup on ligated samples:
○
Running PCR cleanup on ligated samples to remove oligos from mix --> increase desired length:total DNA fragment ratio by only retaining DNA strands >100bp
○
Nanodrop showed OK results, will run gel tomorrow for extraction
Wednesday, 7/27
●
PCR Cleanup of Ligations: 4B4L, PA1L, 4DL, PG01GL, MixL [2000ng] (MB)
○
4DL - 173.1 ng/uL [11.56uL]
○
4B4L - 136.2 ng/uL [14.68uL]
○
PAIL - 200.2 ng/uL [10.00 uL]
○
MixL - 180.9 ng/uL [11.04 uL]
○
P601L - 305.6 uL [6.54 uL]
●
Gosset promised to express ship strain VH33deltatyrR (kanamicin 30 ug/ml and chloramphenicol 30 ug/ml) and plasmid pMmelAtyrCpheACM transformed in strain W3110 (tetracicline 30 ug/ml and chloramphenicol 30 ug/ml). Samples are sent in a sterile paper disc that you must place in liquid medium and grow overnight with the required antibiotics.
ELIAS'S LAB NOTEBOOK
●
PCR cleanup concentrations: ~30ul in each tube
○
4B4: 105.5ng/ul
○
PA1: 108.4ng/ul
○
4D: 44.8ng/ul
○
P601G: 87.2ng/ul
○
Mix: 56.8ng/ul
○
Want >1ng DNA for extraction
●
ELIM order: mel_meffRedF (forward primer for gibson of repetitive strand directly to pSB1C3)
Screen Shot 2016-08-19 at 2.52.49 pm.png
Mel_7-27_4B4-4D.PNG
Mel_7-27_P601+Mix_taped.PNG
Thursday, 7/28
Gel Extraction Masses:
Mel_Gels.jpg
Friday, 7/29
ELIAS'S LAB NOTEBOOK
●
Run gel on 4 backbones linearized for only repetitive melanin insert
○
rAIP backbone
○
*Only changed gibson tail of forward primer --> worked with previous gibson tail
○
*mel_meffRedF also matches rAIP plasmid --> will try linearizing this
■
Uses T7 promoter instead of BBa_J... constitutive promoter in meffRed
Monday, 8/1
Based on Gosset's Response, plan going forward will be to BioBrick:
●
AroG_fbr (2 mutants via site-directeed mutagenesis)
●
AroF_fbr (synthesize with IDT)
●
MutMelA (synthesize with IDT)
Keep attempting to do electroporesis to get MutMelA and DAHP Synthase Form into the cell strain with the tyrR repressor knockout (VH33tyrR)
Tuesday, 8/2
ELIAS'S LAB NOTEBOOK
●
Liquid cultures from Monday both grew (optimization melanin product) --> will cryostock Cm + Kn culture, suspect only the Cm culture of contamination from other cells
●
Mike got Tetracycline from Stanford --> will dilute and aliquot (10mL milliQ water 70% ethanol with 100mg/mL tet)
○
Will make new culture of Cm + Tc cells to confirm successful culture was not attributable to contamination
●
Melanin binding domain:
○
melmeffRed primer also anneals to Charlie's rAIP plasmid (at RBS) --> will run PCR on this
○
Linearized backbone will have T7 promoter instead of constitutive
●
PCR
A
B
C
D
1
6 tubes
template
for
rev
2
r4B4
rAIP
melmeffRedF
Vec_4B4R
3
rPA1
rAIP
melmeffRedF
Vec_PA1R
4
r4D
rAIP
melmeffRedF
Vec_4DR
5
4P6
rAIP
melmeffRedF
Vec_P601GR
6
rOld
rAIP
Vec_for
Vec_4B4
7
m4B4
rAIP
mel_meffRedF
Vec_4Br
Table8
Wednesday, 8/3
●
Run gel on melanin binding backbone
○
6 PCR tubes + 2 control plasmid + 2 ladder = 10 lanes
Mel_8-3.PNG
A
B
C
D
E
F
G
H
I
J
1
1
2
3
4
5
6
7
8
9
10
2
lad
r4B4
rPA1
r4D
rP6
rOld
rAIP control
m4B4
meffRed
lad
Table9
○
PCR products: 1ul prod + 1ul dye + 4ul H2O
○
Backbone: ~30ng
○
Lad: 1ul 2-log
●
Mel binding 3pt gibson:
Screen Shot 2016-08-22 at 9.42.41 am.png
●
Gosset sent purified pMelAtyrCpheACm plasmid
●
Thursday, 8/4
●
Transform pMelAtyrCpheACm plasmid into VH33tyrR strain
●
Added RNAase A to MX1
Mel_8-4.PNG
Friday, 8/5
●
VH33deltatyrR and pMmelAtyrCpheACM transformed with first PEG transformation protocol (TS, ER)
○
2x TSS made with LB broth, 10% PEG, 5% DMSO, 50mM Mg2+
○
31mL LB, 3.15g PEG, 1.6mL DMSO, 0.375g hydrated MgSO4
●
Cells incubated at 37˚C until they reached an OD of 0.2-0.4 at 600nm, then protocol was followed with TSS
●
5 plates made with Chl + Tet
Monday, 8/8
●
PEG transformation failed, new protocol followed from Griffin on a cell culture taken from our cryostock
●
2 plates made with Chl + Tet
Tuesday, 8/9
●
PEG transformation failed again
●
Decision to wait for second plasmid to come in before starting more PEG transformations
Thursday, 8/18
●
After speaking with Kosuke, it is determined that electroporation is a more reliable method of transforming our cells than a PEG transformation
○
Liquid cultures grown overnight in preparation (VH33 transform with pMelAtyrCpheACm onto Kan+Cm plates)
Friday, 8/19
●
Kosuke's electroporation protocol is followed on 3 liquid cultures of cells
○
Cells growing black spots in some parts of the pellets -- melanin production?
○
Cells added to LB, SOC, and SOC
○
3 Chl + Tet plates made with 1:1 and 1:10 dilutions
○
Split plate with BH33 and TyrR and pMelA with LC#1 and LC#2, then 1 plate with a 1:1 dilution and 1 plate with a 1:10 dilution
Monday, 8/22
●
1 colony grew on the 1:1 plate!
●
4 liquid cultures made:
○
1 with Tet
○
1 with Tet, Chl, Kan
○
1 with Chl, Tet
○
1 with Chl
●
PCR:
○
So the point of all of these PCR rxns is to linearize out the pSB1C3 backbone, promoter, RBs, and start codon from either meffRed or rAIP in a format that is compatible with the assembly of this melanin binding construct:
○
There are two different decapeptides and two heptapeptides. One of each of them binds melanin (decapeptide 4B4, heptapeptide 4D), and one is a dumb control(decapeptide P601G, heptapeptide PA1). P601G is sometimes labeled P601 or P6 due to the size of the tube caps. In the most recent gel extraction that yielded the best results, we extracted pure repetitive sequences of 4B4, 4D, and P601G, as well as a hybrid of 4B4 and 4D (which we are most interested in). We did not extract PA1. So we will need to make linearized backbone with its 3’ blunt end compatible with the beginning of each of the three types of melanin binding sequences, and its 5’ blunt end compatible with the flag/lumio/his tag we have been using. To do so, we will run these PCR rxns--
Linearize Backbone at Terminator with Gibson Tails
A
B
C
D
E
1
Rxn #
Template DNA
For Primer
Rev Primer
Ta for PCR
2
1
rAIP Plasmid
gib_term_F3
Vec_4B4
61°C for 10 cycles72°C for 30 cycles
3
2
rAIP Plasmid
gib_term_F3
Vec_4D
61°C for 10 cycles72°C for 30 cycles
4
3
rAIP Plasmid
gib_term_F3
Vec_P601G
61°C for 10 cycles72°C for 30 cycles
5
4
rAIP Plasmid
gib_term_F1
Vec_4B4
61°C for 10 cycles72°C for 30 cycles
6
5
rAIP Plasmid
gib_term_F1
Vec_4D
61°C for 10 cycles72°C for 30 cycles
7
6
rAIP Plasmid
gib_term_F1
Vec_P601G
61°C for 10 cycles72°C for 30 cycles
Table10
Linearize Backbone without Gibson Tails
A
B
C
D
E
1
Rxn #
Template DNA
For Primer
Rev Primer
Ta for PCR
2
7
rAIP Plasmid
lin_meff_F
lin_meff_R
61°C
3
8
rAIP Plasmid
lin_term_F
lin_meff_R
62°C
Table11
●
Use Q5 for all rxns
●
Make sure to use >1ng of template DNA
●
1:20 dilution of primers
Ran on gel (twice), lanes 1-7 verified for 2250bp linearized plasmid:
Mel_8-22.PNG
Tuesday, 8/23
●
MutMelA transformation failed
●
Regather with master plan on FQ, PQQ, SELEX, Melanin, Latex, SSI
Wednesday, 8/24
●
Discussed performing CRISPR-Cas9 knockouts with Jesica
○
Clean Digest w/ BsaI, Gel Purify Band, Transform dCsa9 into T7 on folate Selection medium
Mel_8-24.PNG
14800340_1415545208460097_575863617_o.jpg
Friday, 8/26
14787592_1415545161793435_1485148049_o.jpg
Mel_Lanes1-17_8-26.PNG
Mel_Lanes18-24_8-26.PNG
Tuesday, 8/30
●
Made Electrocomp Wash (4.55g Sorbitol, 4.55g Manitol, 5mL Glycerol up to 50mL water) for preparing 6 aliqots of Electrocompetent cell
Tuesday, 9/6
AroF Site-Directed Mutagenesis (Above 78 C)
●
AroG_Mut1_Pro150Leu_F: CACCCTACAATATCTCGCTGACCTGATGAGCTGGG (77 C)
●
AroG_Mut1_Pro150Leu_R: CCAGCGGCAGGTGAGTTTCTCGATATGATCACCCTACAATATC
○
Reverse Complement: GATATTGTAGGGTGATCATATCGAGAAACTCACCTGCCGCTGG (77 C)
●
AroG_Mut2_Leu175Asp_F: CCGCGAAGATGCATCAGGGCTTTCTTGTCCGG (81 C)
●
AroG_Mut2_Leu175Asp_R: CGAATCGCAGGTGCACCGCGAAGATGCATC
○
Reverse Complement: GATGCATCTTCGCGGTGCACCTGCGATTCG (79 C)
MutMelA Gibson Primers
●
Gibson 1st Overlap: GAGAAATACTAGATGGCG (56 C)
●
Gibson MelA F Primer: GAGAAATACTAGATGGCG TGGCTGGTCG (Annealing 69 C, Elongation 72 C)
●
Gibson Vector R Primer: CTACTAGAGAAAGAG GAGAAATACTAGATGGCG (Annealing 61 C, Elongation 72 C)
○
Reverse Complement to-order: CGCCATCTAGTATTTCTC CTCTTTCTCTAG
●
Gibson 2nd Overlap:  GTGTCCGCCTAATAC (55 C)
●
Gibson Vector F Primer: GTGTCCGCCTAATAC TAGTAGCGGCCG (60 C)
●
Gibson MelA R Primer:  GCTAGAAATAGCCATA GTGTCCGCCTAATAC (70 C)
○
Reverse Complement to-order: GTATTAGGCGGACAC TATGGCTATTTCTAGC (70 C)
Thursday, 9/8
●
Only 1st primer submitted, other 7 will arrive Friday
Friday, 9/9
●
Miniprepped Colonies 9 and 13-->Added to BioBrick Box for 4D (melanin-binding heptapeptide) and P601 (control peptide) in pSB1C35
●
Linearized 1pt assembly plasmids from 4D (Colony #9) and P601  (Colony #13)
●
PCR Cleanup of 4D and P601 products
●
Gibson assembly of 4D/P601 linearized pSB1C3 backbones with CBD and HisTag
●
Transformed into NEB 5-alpha
●
Plated on warmed Cm Plates and stored at room temp over weekend (labeled 4D_3pt and P601_3pt 9/9)
Monday, 9/12
●
Colonies grew on clean 4D_3pt plate and (potentially contaminated) 4D_3pt plate
○
No growth on either P601_3pt plates
○
Put in 37˚C incubator for several hours, as colony growth was small after the weekend at room temp
●
Colonies picked from each plate and grown up in 5ml LB + 5ul Chl overnight
Tuesday, 9/13
●
Miniprep done on both cultures: 51.8ng/ul for the clean 4D_3pt plate
Thursday, 9/15
●
4D_3pt miniprep transformed into T7 (1.45ul 4D_3pt) and grown on Chl plates
Saturday, 9/17
●
4D_3pt T7 transformation grew successfully on plates-->Liquid Culture grown up again overnight
○
Need to send in plasmid for sequencing
●
The P601_3pt transformation turned out to be unsuccessful.  We will instead use a different sheet (e.g. nylon/cellulose) as a control rather than the binder itself
●
Site-directed mutagenesis was started for AroG Mut1 and Mut2
○
DpnI digest and left overnight
●
MutMelA 2 linearizations begun
Sunday, 9/18
●
Sent in P601_1pt, 4D_1pt, 4D_3pt for VF/VR sequence verification
●
Started large scale extraction of 4D_3pt protein in T7
●
PCR cleanup of AroG Mut1 and Mut2 PCR products plus linearized vector and melA for Gibson
●
Nanodropped concentrations of 4 constructs
●
Set up 2-piece Gibson Assembly of MelA with Vec
●
Transformed all 4 into NEB5-alpha
Monday, 9/19
●
All Transformations were successful
●
8 colonies picked (3 from AroG_Mut1, 3 from AroG_Mut2, 2 from MutMelA) and grown up overnight plus PCR cleanup and sent in for sequencing
●
Began Large Scale Extraction for 4D_3pt protein (soluble and insoluble fractions frozen overnight)-->Need to run SDS page gel in morning to find out which it is in]
Tuesday, 9/20
●
MutMelA (colony 1), AroG_Mut1 (colony 2, sketchy sequencing but mutation confirmed), and AroG_Mut2 (colony 2, codon fixed) Miniprep liquid cultures
●
Paste extracted protein on paper after incubating with melanin
●
Measure UV resistant properties just like Fluorophore assay
●
4D4 Protein Confirmed on PAGE gel (27.16 kDa) in soluble fraction
●
Miniprepped and Biobricked AroG_Mut1, AroG_Mut2, MutMelA
Wednesday, 9/28
●
SDS Page Gel to confirm melanin binding peptide's presence (Orange Stain):
●
Lane 1 (far left), lanes 2-3 wash, lanes 4-6 elutions v1, lanes 7-8 wash, lanes 9-11 elutions v2
9-28-16-CBD-Mel_BindingElutions.jpg
●
Ladder on far left got truncated from this flipped image, but dark band confirms desired peptide was produced and is split between elution fractions #2 and #3 (to be used going forward). Same conclusion drawn from post-stain image.
Monday, 10/3
●
Prepared 50mL of Phosphate-Buffered Saline
●
Melanin is insoluble in water, so melanin (125mg) was first dissolved in a polar aprotic solvent (12.5 mL of 100% EtOH) for 10 mg/mL concentration
●
After heating and shaking, melanin (2 mL) was added to 4 mL PBS and 2mL of purified linker. Done in duplicate with melanin (no linker) as negative control
●
After 1 hour of shaking incubation at 37 C, 1mL of the bound solution was pipetted onto cellulose sheets (3 patches) and ____ for positive control
Phosphorylation and Ligation of Repetitive DNA Building Blocks
Introduction
Use this protocol to phosphorylate and repetitive DNA buidling blocks, and ligate them into randomized long repetitive strands.
Materials
Materials per Tube (Phosphorylation) Volume
DNA building block mixture (80 µM)                           5 µl
Ligase 10x Buffer                                                             2 µl
DI Water                                                                          11.5 µl
T4 Polynucleotide Kinase (PNK)                                  1.5 µl
Total Volume (Phosphorylation)                   20 µl
Materials per Tube (Ligation)
Phosphorylation mix (above)                                       20 µl
T4 DNA Ligase                                                                  1 µl
Total Volume (Phosphorylation)                    21 µl
Procedure
Phosphorylation
Add materials listed above with desired 80 µM DNA building block mix to PCR tube
Incubate in thermal cycler at 37°C for 1 hr
Ligation
Remove tubes from thermal cycler and add 1 µl T4 DNA Ligase to each tube
Incubate at 16°C overnight
Annealing of Template DNA
Introduction
Use this protocol to anneal complimentary single stranded oligonucleotides.
Materials
Measurements per Tube:
Forward Seq (200 µM)              10 µl
Reverse Seq (200 µM)               10 µl
10x H Buffer                                 2 µl
D.I. Water                                     28 µl
Total Volume per Tube:      50 µl
Final Concentration:           80 mM
Procedure
Anneal Strands
Heat mixed tubes at 93°C for 1 min in thermal cycler
Decrease heat of thermal cycler by 5°C every 2 min with small ∆T/sec in between steps
When the temp falls more than 20°C below the Tm of the strands to anneal, change steps to decreasing 10°C every min
Once at room temperature, hold samples at 4°C for 15 min, then store samples at -30°C
