<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stanford-Brown_SB16_BioSensor_IRES skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stanford-Brown/SB16 BioSensor IRES</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Stanford-Brown 2016</TITLE></P><DIV class="container"><NAV class="navbar navbar-inverse navbar-fixed-top"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" id="BioballoonLabel" href="https://2016.igem.org/Team:Stanford-Brown">Stanford-Brown</A><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Stanford-Brown">Home</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Vision">Vision</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle active" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">BioMembrane</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_Collagen">Collagen</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_Elastin">Elastin</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_p-Aramid">p-Aramid</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_Latex">Latex</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_UV">UV Protection</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_AptamerPurification">Aptamer purification</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Float</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Float_Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Float_Gas">Gas production</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">BioSensor</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioSensor_Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioSensor_Chromoproteins">Chromoproteins</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioSensor_FQsensor">Fluorophore-Quencher</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Practices</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Human_Practices">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Integrated_Practices">Integrated Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Engagement">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Practices_Interviews">Interviews</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Collaborations_Interlab">InterLab Study</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Collaborations">Team Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Parts">BioBricks</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Attributions">General Attributions</A></LI><LI class="dropdown-header">Notebooks</LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_CE">Collagen &amp; Elastin</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_p-Aramid">P-aramid</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_Latex">Latex</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_UV">UV protection</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_AptamerPurification">Aptamer Purification</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_Gas">Gas Production</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_Chromoproteins">Chromoproteins</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_FQsensor">Fluorophore-Quencher</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Team">Team</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Requirements</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_MedalRequirements">Medal Requirements</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Safety">Safety</A></LI></DIV></DIV></NAV></DIV><DIV class="figure-legend">Biosensor team member Julia introduces the IRES subproject</DIV><DIV class="container"><DIV class="row rowT"><DIV class="col-sm-12 col-PT"><H1 class="sectionTitle-L firstTitle">Introduction</H1></DIV></DIV><DIV class="row"><DIV class="col-sm-12 pagetext">A fully functional sensor needs both a way to recognize the small molecule it’s supposed to be sensing (a recognition element) and a detectable signal that reliably results from a positive sensing interaction (an expression platform).</DIV><DIV class="col-sm-12 pagetext">An aptamer is defined by Sun et al as a “nucleic acid with high specificity and affinity for its target”. [1] As such, aptamers can be used to build  highly discriminating recognition elements for their target ligands. However, alone this is not enough -- molecular conformational changes are no easier to detect than the target small molecules themselves.</DIV><DIV class="col-sm-12 pagetext">Inspired by a 2011 paper from a group at Ehime University [2], we sought to connect the recognition functionality of specific aptamers with in vitro expression of a particular reporter gene (for our constructs we used GFP). To do this, we synthesized an “IRES sensor” construct (see picture below). IRES stands for internal ribosome entry site. It does exactly what the name implies: allows transcription and thus translation to happen. The key for our purposes is that IRES has a particular active 3-D confirmation: if the IRES  is folded correctly it facilitates protein translation, if not it doesn’t so no translation will occur.</DIV><DIV class="col-sm-12 pagetext">Ogawa’s work showed that disrupting  a particular region of the IRES (shown here in blue) by binding it with its complement,  anti-IRES (aIRES, shown here in pink), rendered the entire IRES sequence inactive and unable to translate. In the “off” state when the aptamer is unbound, IRES is bound by a-IRES and no reporter gene expression is observed. In this state the complement to aIRES, anti-anti-IRES (aaIRES, shown here in red), is trapped in an inaccessible hairpin by the end of the aptamer and the optimized modulator region.</DIV><DIV class="col-sm-12 pagetext">When the target molecule is introduced, it binds to the aptamer. As a result, the aptamer changes conformation, and releases the aaIRES sequence. The released aaIRES sequence binds to aIRES (out-competing IRES), freeing the IRES complex to refold into its active conformation. Once IRES is correctly folded, this allows the sensor to translate reporter protein. The plasmid we created to accomplish this is shown in Figure 1, with the resulting molecules pictured in Figure 2.</DIV></DIV><DIV class="row rowT"><DIV class="col-sm-12 col-PT"><H1 class="sectionTitle-R">Conceptual Design of Test Sensor</H1></DIV></DIV><DIV class="row"><DIV class="col-sm-5 imgcol-L"><DIV class="figure-legend">Figure 1: Annotated Geneious Map of the IRES system. </DIV></DIV><DIV class="col-sm-7 pagetext-R"><DIV class="text">This construct contains the iGEM standard prefix, a constitutive promoter (BBa_J23119), a 5’ loop that Ogawa found to be helpful in stabilizing the construct, a “modulator region” that determines how much of the aptamer is bound to aaIRES, the aptamer of interest, aIRES, a naturally occurring viral IRES, the translation start site at the very end of it, and finally the GFP coding sequence. </DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-7 pagetext-R"><DIV class="text">Figure 2 shows how the sensor works at the molecular level. On the left (OFF): the aptamer is unbound, and the functional IRES (blue) is bound to the aIRES (red). Because of this, the functional 3-D confirmation of the IRES complex is disrupted, blocking the translation start site, preventing transcription of the reporter gene. On the right (ON): aIRES (red) is bound in this case to aaIRES (pink) rather than IRES (blue). This allows the functional IRES complex to form, and translation of the reporter protein to occur. </DIV></DIV><DIV class="col-sm-5 imgcol-L"><DIV class="figure-legend">Figure 2: Schematic of the IRES sensor system. Modified from figure 5, “Rational Design of Artificial Riboswitches…” (Ogawa, 2011)</DIV></DIV></DIV><DIV class="row"><DIV class="row rowT"><DIV class="col-sm-12 col-PT"><H1 class="sectionTitle-L firstTitle">Conclusion</H1></DIV></DIV><DIV class="col-sm-12 pagetext last">We attempted to create an IRES based sensor for theophylline -- a test small molecule with a very well characterized aptamer. Our initial approach was to order a gene block with the constitutive promoter, stabilizing loop, modulator, aaIRES, theophylline aptamer, aIRES, and IRES sequence (in total 310 bp) from IDT, and clone that into baseline iGEM GFP plasmid BBa_E0040. To this, we ordered the following primers that were auto-generated in SnapGene. 

We spent weeks trying unsuccessfully to amplify the fragment and the backbone with these primers, before realizing that they were too short. We also realized in the interim that we had failed to include a T7 primer in the initial design and that T7 was required for our in vitro translation system to work. In light of these two observations, we redesigned our primers to include the T7 primer and to be longer. The gel below shows the backbone plasmid and the amplified fragment using the redesigned primers. </DIV><DIV class="row"><DIV class="col-sm-5 imgcol-L"><DIV class="figure-legend">Figure 3: This gel confirms the 310 bp piece and the 3 kb backbone were successfully amplified by our primers. </DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-12 pagetext last">With these we were able to successfully get the both the gene block and the plasmid backbone expressed. Figure 3 is the gel confirmation of that success. From there we used the amplified fragments to Gibson assemble together the plasmid of interest. Once assembled, we sequence confirmed the plasmid, the results of which can be found on the Bio Brick page for this construct.</DIV></DIV><DIV class="row"><DIV class="col-sm-7 pagetext-L"><DIV class="text">After we had the whole plasmid, we used it to express the RNA sensor, and then challenged the RNA with various concentrations of theophylline. In our first test we only used theophylline conditions -- we didn't want to waste expensive in vitro translation reagents -- but we saw that the .01 mM, 1 mM, and 10 mM theophylline concentrations all expressed the same baseline level of fluorescence (Figure 4).</DIV></DIV><DIV class="col-sm-5 imgcol-R"><DIV class="figure-legend">Figure 4: Initial IRES sensor trial. The far right is 0.1mM theophylline, the middle is 1mM theophylline, and the far right is 10 mM theophylline.      </DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-12 pagetext">We next postulated that the observed  inactivity may be due to the fact that our engineered plasmid completely lacked a gap between the translation start site and the AUG start codon of GFP.  Accordingly we linearized the plasmid at the point where the TSS meets GFP, amplified this template with primers that incorporated a 6nt gap region, and performed blunt-end ligation to re circularize the plasmid. Post recircularization, we re-transfected E Coli with the gap-containing plasmid, and once again tested in vitro transcription with varying levels of theophylline. </DIV></DIV><DIV class="row"><DIV class="col-sm-5 imgcol-L"><DIV class="figure-legend">Figure 5: Modified IRES with gap sensor trial. In order these tube are 0.1mM Theophylline, 1mM Theophylline, 10mM Theophylline and 30mM Theophylline, and on the the far right is the negative control (no theophylline).</DIV></DIV><DIV class="col-sm-7 pagetext-R"><DIV class="text"> Even with the  gap introduced, and utilizing sequence confirmed DNA for the sensor, we did not observe differing levels of fluorescence when treated with different levels of theophylline. </DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-12 pagetext last">Ultimately we concluded based on this data that we were not able to replicate the IRES sensor behavior of the Ogawa paper. We suspect that there is some important experimental parameter that we unknowingly did not replicate (Mg concentration? RNA cleaning process? In vitro translation time? etc.). Nonetheless we believe that this system could be an extremely powerful in vitro synthetic tool, because if it could be optimized to work as well as the data Ogawa reported this could be an easy &quot;plug and play&quot; sensing platform that allows maximal user design flexibility to customize both the desired target and the desired response. In the interests of ensuring that the broader iGEM community has the opportunity to benefit from this work and the potential of this system, we BioBricked it (<A href="http://parts.igem.org/Part:BBa_K2027015">BBa_K2027015</A>), in hopes that in hopes that it might be helpful to teams in the future interested in creating maximally flexible biosensors.<I>References</I>

1) Sun, H., Zhu, X., Lu, P., Rosato, R., Tan, W., &amp; Zu, Y. (2014). Oligonucleotide Aptamers: New Tools for Targeted Cancer Therapy. Molecular Therapy—Nucleic Acids, 3(8), e182. http://dx.doi.org/10.1038/mtna.2014.32
2) Ogawa, A. (2011). Rational design of artificial riboswitches based on ligand-dependent modulation of internal ribosome entry in wheat germ extract and their applications as label-free biosensors. RNA, 17(3), 478-488. http://dx.doi.org/10.1261/rna.2433111
 </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>