Team:Stanford-Brown/SB16 Protocols
Stanford-Brown 2016
Toggle navigation
Stanford-Brown
Home
Attributions
BioMembrane
Overview
Collagen
Elastin
p-Aramid
Latex
UV Protection
Float
Gas production
BioSensor
Overview
Chromoproteins
Fluorophore-Quencher
Aptamer purification
Practices
Overview
Integrated Human Practices
Outreach
Life Beyond the Lab
Collaborations
InterLab Study
Team Collaborations
BioBricks
Protocols
Protocols
Software Design
Modeling
Notebooks
Collagen & Elastin
P-aramid
Latex
UV protection
Aptamer Purification
Gas Production
Chromoproteins
Fluorophore-Quencher
Nylon
Interlab Study
Team
Requirements
Medals
Safety
Amide Synthesis with trimethyl borate · Benchling
Amide Synthesis with trimethyl borate
Introduction
Synthesis of amide bonds with trimethyl borate as the coupling reagent. Used to produce polymers from p-aminobenzoic acid monomers.
Materials
2 equiv. P-aminobenzoic acid (pABA)
2 equiv. trimethyl borate
Nonpolar/aprotic solvent (MeCN)
Procedure
Amide synthesis
Add pABA (1 equiv) to MeCN (2 mL, 0.5 M)
Add 1 equiv trimethyl borate to mixture. Stir at 80C (100C?) for 5-24 hours.
Solid Extraction
Dilute reaction mixture with CH2Cl2 or EtOAc (organic solvent) and H2O (polar solvent).
Add MgSO4 (anhydrous) to remove H2O.
Annealing of Template DNA · Benchling
Annealing of Template DNA
Introduction
Use this protocol to anneal complimentary single stranded oligonucleotides.
Materials
Measurements per Tube:
Forward Seq (200 µM)              10 µl
Reverse Seq (200 µM)               10 µl
10x H Buffer                                 2 µl
D.I. Water                                     28 µl
Total Volume per Tube:      50 µl
Final Concentration:           80 mM
Procedure
Anneal Strands
Heat mixed tubes at 93°C for 1 min in thermal cycler
Decrease heat of thermal cycler by 5°C every 2 min with small ∆T/sec in between steps
When the temp falls more than 20°C below the Tm of the strands to anneal, change steps to decreasing 10°C every min
Once at room temperature, hold samples at 4°C for 15 min, then store samples at -30°C
BCA Protein Assay · Benchling
BCA Protein Assay
Introduction
For determining concentration of protein using Nanodrop. summarized from http://www.nanodrop.com/Library/T085-NanoDrop-2000-2000c-BCA%20Protocol.pdf
Materials
Working reagent (50A:1B)
Pierce BCA Protein Assay Reagent A
Pierce BCA Protein Assay Reagent B
BSA/Albumin Standard (2mg/mL)
PCR tubes/PCR plate
Procedure
Sample Prep
Let all reagents/samples/standards equilibrate to RT.
Prepare enough fresh working reagent (WR) for all standards and samples to be measured using a 50:1 ratio of the kit reagents A:B.
If using high-range assay for 5 samples, you'll need approximately 1000 uL of WR total ==> So, you'll need about 1000 uL of A and 20 uL of B (ratio would be 1000:20).
Add reagent volume to each PCR tube.
Microassay (1:1 sample to WR) Add 10 uL of WR to each standards and sample tube/well.
High range assay (1:20 sample to WR) Add 200 uL of WR to each standards and sample tube/well.
Add 10 uL of standards or samples to the appropriate tube. Mix well by gentle vortexing.
Incubate standard and sample tubes at:
37 C for 30 min OR 60 C for ~5 min. Cool to room temp after.
Cleaning Electrophoresis Gel · Benchling
Cleaning Electrophoresis Gel
Introduction
Cleaning electrophoresis gel with gel extraction kit (purple).
Materials
100% ethanol
WN + WS buffers
Procedure
Gel Cleaning
Prepare WN + WS buffers by adding appropriate amount of 100% ethanol as shown on bottle.
Carefully cut out piece of gel containing fragment of interest under longwave UV.
Add 300ul GEX for every 100mg gel collected to centrifuge tube and gel piece.
Incubate at 55˚C for 5-10min until gel has dissolved. Can smash with a pipette tip.
Place extraction column into collection tube and load 0.7ul mixture into column. Centrifuge at 5000rpm for 30s. Discard flow-through. Repeat until all mixture is used.
Wash column with 0.5ml (500ul) of WN buffer by centrifuging at 5000rpm for 30s. Discard flow-through.
Wash column with 0.5ml WS buffer by centrifuging for 60s at 5000rpm. Discard flow-through.
Centrifuge column at 12000rpm for 3min to remove residual ethanol.
Place column in new (storage) centrifuge tube. Add 15-30ul elution buffer onto center of membrane. Allow to stand for 2min.
Centrifuge for 50s at 12000rpm to elute DNA. Store at -20˚C. Find concentration collected with Nanodrop.
Competent Cell Test Kit (iGEM) · Benchling
Competent Cell Test Kit (iGEM)
Introduction
Cell competency test on E.Coli strains with DNA kit provided by iGEM
Materials
Purified DNA (BBa_J04450 RFP construct with plasmid backbone pSB1C3)  at 0.5pg/ul, 5pg/ul, 10pg/ul, 20pg/ul, 50pg/ul (50ul aliquots)
Competent cell aliquots
Agar plates with chloramphenicol
SOC media
Procedure
Transformation
Spin down DNA tubes from Competent Cell Test Kit at 8,000-10,000rpm for 20-30 seconds
Thaw competent cells on ice for 10 minutes. Label one 2.0ml microcentrifuge tube for each concentration and pre-chill by placing empty tubes on ice.
Pipet 1uL of DNA into each microcentrifuge tube. Use a separate tube for each concentration.
Pipet 50uL of competent cells into each tube, flicking tube gently to mix. Incubate on ice for 30 minutes. Preheat waterbath to 42˚C.
Heat-shock cells by placing into waterbath for 1 minute.
Immediately transfer tubes back to ice and incubate on ice for 5 minutes.
Add 200uL of SOC per tube and incubate at 37˚C for 2 hours. Prepare and label agar plates.
Pipet 20uL from each tube onto appropriate plate and evenly spread mixture across the plate. Do triplicates of each tube if possible to calculate an average colony yield.
Incubate at 37˚C overnight or for 16 hours.
Count number of colonies against a dark background, using an average cell colony count if done in triplicates.
To measure competent cell efficiency: (colonies on plate) / ng of DNA plated x 1000ng/ug
To measure ng of DNA plated: 1uL x concentration of DNA in original vial x (volume plated / total reaction volume)
Competent cells have an efficiency of 1.5x10^8 to 6x10^8 cfu/ug DNA (cfu = colony-forming unit)
Cryostock Glycerol -80°C · Benchling
Cryostock Glycerol -80°C
Introduction
Clonal bacterial freezer stocks are important for preserving plasmid constructs (adapted from Stanford BioE 44 F2015). Make a glycerol freezer stock from your bacterial culture as follows:
Materials
50%/40% glycerol
Overnight culture of cells
-80°C Freezer
Procedure
Making a clonal glycerol freezer stock
Write down in your notebook the identifier and the plasmid strain you plan onstoring in the tube.
Transfer 0.5 mL of 50% (or 40%) glycerol into each tube.
Briefly vortex your bacterial culture to ensure that it is well mixed. Add 0.5 mLof the mixed culture to the appropriately labeled tube containing the glycerol
Tightly cap the tube. Gently vortex the tube or invert multiple times to mix.
Access the computer with the 2D barcode scanning hardware attached.Open the “BIOE44 F2014 glycerol stock” Excel file in the BIOE44 folder onthe computer.
Record your unique identifier into the tube/notebook/spreadshseet and input the appropriate plasmid ID (e.g. pSB1C3, p70417, pComet, pX330-gRNA, orpJ1201-03c, etc.) into the plasmid ID column. Add into the notes column anynotes you deem appropriate. Remember to save the file when you are finished!
Store the glycerol stock in a labeled box in the -80°C freezer.
Cryostock glycerol revival · Benchling
Cryostock glycerol revival
Introduction
Revive your frozen culture stocks
Materials
Cryostock
Pipette
Procedure
Use a pipette tip to scrape off a little bit of the semi-thawed glycerol. Put tip with cells into media
Put your glycerol stock back into the box in the -80°C freezer. Do not let it thaw completely! Thawing and freezing your stock repeatedly will kill your cells.
Put inoculated culture into shaking incubator to let culture grow.
Capture SELEX · Benchling
Capture SELEX
Introduction
Written by Michael Becich, Julia Gross, Amy Weissenbach Adapted from Lan et al. 2012  (https://www.ideals.illinois.edu/bitstream/handle/2142/42441/Tian_Lan.pdf?sequence=1, http://www.hindawi.com/journals/jamc/2012/415697/)
Materials
Streptavidin Beads
10mg/mL concentration (2 mL, 20mg total, 0.7-1.2X10^9 beads per mg)
Random Library
ATA CCA GCT TAT TCA ATT NNN NNN NNN NTG AGG CTC GAT CNN NNN NNN NNN NNN NNN NNN NNN NNN NNN NNN NNN NNN NNA GAT AGT AAG TGC AAT CT
Biotinylated Capture Oligo's (25 nmole)
/5Biosg/TA CCG CAA AAA AAA AGA TCG AGC CTC A
Use 750 pmol oligo for 150uL (1.5mg) beads, as rounds go on should consider cutting down
PCR Primers (Forward, Reverse, and Elongated Reverse) (25 nmole)
FWD: ATACCAGCTTATTCAATT
RVS: GATTGCACTTACTATCT
RVS_plus_spacer:  AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA/iSp18/GATTGCACTTACTATCT
Selection Buffer (SB) - make 200 mL
100 mM NaCl  - 0.005mol, about 0.3g/50mL
20 mM Tris-HCl pH7.6
2 mM MgCl2 -- 0.095g/50mL
5 mM KCl -- .0186 g/ 50mL
1 mM CaCl2 --- 0.005549g / 50mL
Binding & Washing Buffer (B&W) - make 120 mL
10 mM Tris-HCl pH7.5
1 mM EDTA
2 M NaCl
TE Buffer
10mM Tris-HCl pH 7.4
1 mM EDTA
TBE Buffer
90 mM Tris-Hcl
90 mM boric acid
2 mM EDTA
Sodium Acetate Soln
2mM EDTA, 300 mM sodium acetate, and pH 7.8 (5.2)
PCR Rxn Buffer (don't think we need this, Q5 Master Mix)
80 mM Tris-HCl pH9.5
20 mM (NH4)2SO4
0.02% Tween20
PCR Cleanup Kit
TBD: Need one that will work well when amplifying very small DNA sequences
Procedure
1. Coupling of Capture Oligos to Streptavidin-Coated Magnetic Beads
1000 pmol oligo mixture for 200uL magnetic beads (500pmol/mg)
(Calculate appropriate volume of beads to capture oligo DNA (*dependent on bead type. 500 pmol ss-oligonucleotides per mg bead for Dynabeads® MyOne™ Streptavidin C1))
In a 1.5mL Eppendorf Tube, add 200 μL beads (2.0mg), wash 3X w/ 500 μL B&W
Separate beads by placing the tube in a magnet stand.
Resuspend in B&W to a concentration of 2mg per mL (2×109 beads/mL)
So 2.0mg in 1000uL
Add equal volume of oligo (1000pmol) and incubate at room temperature for 1 h with gentle rotation
Separate and wash 3X w/ 500 μL B&W
Wash  3X w/ 500 μL SB
Nanodrop 3rd fraction to confirm no DNA in flow-through (if DNA, wash and repeat)
Resuspend in selection buffer 1×109 beads/mL (1×108 beads/mL for each subsequent round), so 2.0 mg in 2mL
Store at 4C.  When use, resuspend completely (i.e. vortex etc.)
Use 1mL for first round
100uL for following rounds
2. Preparation of Target
Dilute Target to 1mM in selection buffer
Sterile filter (used pore size of 0.22um), aquilot and store at -18 °C
3. Thermal equilibration of oligonucleotide pool
DNA library (2-3nmol 1st round, all in following rounds) heated to 90 °C for 8min
Immediately cooled and kept at 4 °C for 10min
At same time, wash streptavidin bead-capture oligo complexes (1x10^9 and 1x10^8 in following) around 3X w/ 500uL SB
Use Nanodrop to confirm no more DNA washing off
Bead-capture oligo complexes resuspended in 300uL of the pretreated DNA library
Incubate overnight at 21 °C w/ mild shaking
4. Capture-SELEX round
Wash beads around 9X w/ 500uL SB to remove unbound DNA
Nanodrop to determine fraction coming off, place this fraction aside for comparison
Incubate DNA-bead complexes in 500uL SB at 28 °C for 15min w/ mild shaking
Wash around 7X w/ 500uL SB  (to remove all remaining unhybridized or weakened DNA structures)
Nanodrop to determine fraction coming off
Save flow-through for comparison, label
Suspend DNA-bead complexes in 300uL SB
Incubate at 21 °C for 45min w/ mild shaking
Wash around 7 X with 500uL SB
Nanodrop to measure fraction coming off
Save flow-through, label
Suspend DNA-bead complexes in 300uL target mixture
Incubate at 21°C for 45min w/ mild shaking
Use a magnet to separate out the beads, saving the remaining liquid
Save 10% of the flow-through as stock solution (labeling as necessary), use rest for PCR amplification in the following step.
5. PCR Step
Nanodrop, determine appropriate number of parallel PCR reactions (e.g. 15, 100uL each)
FOR Q5 High Fidelity Master Mix, 50uL rxn:
Q5 Master Mix: 25uL
10uM forward primer: 2.5uL
10uM reverse primer: 2.5uL
DNA, properly diluted: 20uL
For Q5 Master Mix, set thermocycler to
98°C for 30s
(98°C for 5s
51°C for 10s
72°C for 20s) repeat X times
72°C for 2min
4-10°C for ∞
<Gel electrophoresis on 2.5% agarose gel to confirm proper length of DNA product (if feel compelled to do so)>
Pool PCR product and precipitate with ethanol in presence of linear polyacrylamide [19]
Add ethanol 2.5-3-fold to solution
Linear polyacrylamide ($30/tube) 10-20 ug/mL
Centrifuge for 45min (4-10C)
Pellet will be on outside edge (always have one side, eg hinge facing out, so know where pellet it)
Pour most supernatant off, then pipet rest (could save in case don't end up with anything)
Use kimwipe to touch top of tube
If still stuff left, use micropipettor to get last bit out
leave open so that rest can evaporate, til looks dry
(COULD put in vaccum centrifuge, but be careful bc DNA can get lodged to side)
Resuspend product in 100uL TE buffer
Run denaturing PAGE in TBE buffer
-Run with two extra lanes dedicated to 1) a DNA ladder and 2) a PCR product sample that you will later stain as a reference (making the sample unusable)
-When finish running, slice out the lanes you intend to stain.  Follow gel-staining protocol. (We used Gel Red.)
-Line the stained gel next to the unstained gel in a UV box.  First, identify the confirm the presence of the correct DNA bands in the stained PCR product lane.  There should be one 100bp fragment and one slightly longer fragment. Next, use the stained gel as a reference to cut out the unstained (and therefore invisible) 100bp fragments in the adjacent lanes.  Use these unstained gel pieces in the following step.
Elute DNA from gel with 2mM EDTA, 300 mM sodium acetate, and pH 7.8 (5.2)(around 5mL of this solution) -- adding acetic acid
Set at 80C for 150min with mild shaking
Precipitate DNA with ethanol in presence of linear polyacrylamide [19]
Break into 1.7mL tubes
Add ethanol 2.5-3-fold sodium acetate solution
Linear polyacrylamide ($30/tube) 10-20 ug/mL
Centrifuge for 45min (4-10C)
Pellet will be on outside edge (always have one side, eg hinge facing out, so know where pellet it)
Pour most supernatant off, then pipet rest (could save in case don't end up with anything)
use kimwipe to touch top of tube
if still stuff left, use micropipettor to get last bit out
leave open so that rest can evaporate, til looks dry
COULD put in vaccum centrifuge, but be careful bc DNA can get lodged to side
Resuspend DNA in SB.
7. TEST - TBD
<goal of around 13 rounds>
cloning+transformation into chemically competent cells
+sequencing -- talk to Mark
could do rounds 7,8
high throughput sequencing
Discuss SPR with Michaela
Archive roughly 10% of each round and save as stock solution
Distribution Kit Hydration · Benchling
Distribution Kit Hydration
Introduction
Get started by giving your protocol a name and editing this introduction.
Materials
milliQ Water
Procedure
To use the DNA in the Distribution Kit, follow these instructions:
Note: There is an estimated 2-3ng of DNA in each well, following this protocol, assume that you are transforming with 200-300pg/ul
With a pipette tip, punch a hole through the foil cover into the corresponding well of the part that you want. Make sure you have properly oriented the plate. Do not remove the foil cover, as it could lead to cross contamination between the wells.
Pipette 10uL of dH2O (distilled water) into the well. Pipette up and down a few times and let sit for 5 minutes to make sure the dried DNA is fully resuspended. The resuspension will be red, as the dried DNA has cresol red dye. We recommend that you do not use TE to resuspend the dried DNA.
Transform 1ul of the resuspended DNA into your desired competent cells, plate your transformation with the appropriate antibiotic* and grow overnight.
Pick a single colony and inoculate broth (again, with the correct antibiotic) and grow for 16 hours.
Use the resulting culture to miniprep the DNA AND make your own glycerol stock (for further instruction on making a glycerol see this page ). We recommend using the miniprepped DNA to run QC tests, such as restriction digests and sequencing.
* To know which antibiotics to use, look at the plasmid that the part is in. The naming scheme for plasmids is specifically designed to indicate antibiotic resistance.
DNA Miniprep · Benchling
DNA Miniprep
Introduction
Protocol for miniprep comes from Epoch kit in lab.
Materials
Epoch miniprep kit
Procedure
Rinse 500ml cylinder with water then ethanol.
Measure 120ml ethanol, add to wN buffer.
Add 180ml ethanol to WS buffer.
Centrifuge liquid culture tubes at 4500rpm (max) for 2min. Be sure to balance tubes.
Resuspend cell pellet in 200ul MX1 buffer kept in fridge.
Add 250ul MX2 buffer and gently mix. Incubate at room temp for 1-5 min, don't vortex.
Add 350ul MX3 and gently mix immediately.
Centrifuge for 5-10 min at 14,500rpm.
Transfer supernatant carefully to column.
Centrifuge 30-60s at 5000rpm. Discard flow-through.
Wash column once with 500ul WN buffer by centrifuging for 30s at 9000rpm. Discard flow-through.
Wash column once with 700ul WS buffer by centrifuging for 30s at 9000rpm. Discard flow-through.
Centrifuge column at 13000rpm for 2min to remove ethanol residue.
Place column into new 1.5ml centrifuge tube, add 50ul Elution Buffer onto center of membrane.
Stand for 2min at room temp, then centrifuge for 30s at 13000rpm to elute DNA.
Store plasmid DNA at 4˚C or -20˚C.
Electrophoresis · Benchling
Electrophoresis
Introduction
Standard gel electrophoresis.
Materials
DNA
DI water
Loading dye
Procedure
Electrophoresis
Load 1ul molecular ladder into first row.
Mix 1.5ul purple loading dye, 1ul DNA, 7.5ul DI water on a sheet of parafilm.
Load each row you're going to run.
Plug in cables and switch on machine to +/- 100V for +/- 40min. Too high voltage will decrease resolution of results but will run more quickly.
Ensure it doesn't run off the end.
Ensure cables are the right way around (black - near DNA, red + far from DNA)
Gel recipe · Benchling
Gel recipe
Introduction
Making gels
Materials
TAE
Agarose powder
Box molds
Procedure
Gel
Mix TAE and agarose (~1g powder per 200ml buffer (slightly less for long DNA)).
Microwave and mix until completely dissolved, pause microwaving when it bubbles.
Cool until can touch it. Add gel red and mix.
Carefully pour into box mold and cool at room temperature, covered with plastic wrap. Insert comb.
Before running gel, ensure TAE in box is just above gel (+/- 400ml).
Gibson Assembly · Benchling
Gibson Assembly
Introduction
Get started by giving your protocol a name and editing this introduction.
Materials
Gibson Master Mix 2x (exonuclase, polymerase, ligase)
His tags (annealed)
Digested vectors
H2O
LB
Agar
Procedure
Gibson ratios
Gibson Master Mix 2x: 5ul x 3 = 15ul
His tags: 1ul x 3 = 3ul
Digested vectors: 1ul
H2O: 3ul x 3 = 9ul
25g/L LB, 15g/L agar
Gibson Times
Optional: 48˚C for 5min, 5 @ 45
50˚C for 15min, 10 @ 50
Optional: 52˚C for 5min, 5 @ 55
4˚C forever
Gibson Assembly on Geneious · Benchling
Gibson Assembly on Geneious
Introduction
Make plasmids on the internet!
Materials
Procedure
Linearise vector plasmid where you want to insert new gene (eg using a restriction site).
Manually replace start codon if lost.
Select the gene you want to insert excl. stop codon and extract.
Select vector and extracted gene and click tools --> cloning --> Gibson assembly.
Ensure backbone = vector, box contains gene fragment. Choose min overlap length (18-22), min overlap Tm (48), calculation of Tm is based on protocol.
* Can also insert multiple genes simultaneously or a subfolder (eg promoter) to make several different products.
* We want G-C at primer ends, particularly on 5' side.
* Make sure primer code is written 5'-3' from L-->R for synthesis.
* If % GC is high then higher annealing T, lower % GC has lower annealing T. High Tm means less likely to anneal in funny places but don't want too high that doesn't get a chance to anneal before enzyme runs at ~72˚C.
Gluconobacter media · Benchling
Gluconobacter media
Introduction
This might optimize cellulose production.
Materials
Yeast extract (5g/L)
Peptone (3g/L)
Mannitol (25g/L)
DI water
HCl (to taste)
Procedure
After performing the necessary calculations, weigh out an appropriate amount of each component and fill with water. Adjust the pH to about 5.5, make sure the liquid is in a large enough container to prevent any boiling over, and autoclave.
Golden Gate Procedure without Master Mix · Benchling
Golden Gate Procedure without Master Mix
Introduction
This may not be as optimized as the NEB formulation, but it has worked before. Good for removing a fragment via restriction digest with a Type IIS Restriction Enzyme or an actual Golden Gate assembly (but try not to use SapI for real Golden Gate because it only has a 3bp overhang).
Materials
The DNA You Need (It's not a bad idea to linearize plasmids)
DI H2O
PCR Tube
Thermal Cycler
Everything Important but DNA (It's a good master mix candidate)
5 μL T4 DNA Ligase
2 μL T4 DNA Ligase Buffer
1 μL Type IIS Restriction Enzyme (we have SapI and BsaI)
0.2 μL 100X or 1 μL 20X BSA (Bovine Serum Albumin)
Procedure
Everything
Add the non-DNA components to a new PCR tube
Add an appropriate amount of DNA (75 ng per component is recommended)
Fill to 20 μL with DI H2O
For 1-4 fragments, incubate at 37°C for 1 hr and 55°C for 5 min. For 5-10 fragments, alternate between 37°C and 16°C for one minute each for 30 cycles, then  incubate for 5 min at 55°C. For more than 10 fragments, consider going that extra mile and buying the optimized master mix.
Proceed directly to transfomation.
H2O2/Peroxidase (Amplex Red) Assay · Benchling
H2O2/Peroxidase (Amplex Red) Assay
Introduction
Used to detect presence of hydrogen peroxide (H2O2) or peroxidase activity in biological samples. Specifically used to detect the activity of lysyl oxidase in coupling monomers.
Materials
H2O2/Peroxidase/Amplex Red Assay kit from Thermofisher
96 well plate
plate reader
Procedure
Prep for stock solutions
Let all reagents in the kit to completely warm to RT before opening.
10 mM Amplex Red reagent stock soln: It is AIR SENSITIVE, use the reagent on the same day. Also light sensitive. Use on the same day it's prepared.
Let Amplex Red reagent (Component A, blue cap) and DMSO (Component B, green cap) warm to room temp. Just before using, dissolve Amplex Red reagent into 60 uL of DMSO.
1X Rxn buffer: Add 4 mL of 5x Reaction buffer (Component C, white cap) to 16 mL of dH2O.
10 U/mL Horseradish peroxidase stock soln: Dissolve HRP (Component D, yellow cap) in 1.0 mL of 1x Rxn buffer.
Divide any unused HRP stock soln into single use aliquots and store frozen at -20C.
20 mM H2O2 working soln: Dilute ~3% H2O2 (Component E, red cap) into appropriate vol of 1X Rxn buffer to make 20 mM H2O2. Actual concentration of H2O2 is indicated on the label.
20 mM working solution eeds to be used within a few hours of prep
H2O2 Assay
Prepare H2O2 standard curve. Dilute H2O2 containing samples in 1X Rxn buffer to produce H2O2 concentrations of 0-10 uM.
Use volume of 50 uL for each rxn. Pipet into individual wells of microplate.
Prep working soln of 100 uM Amplex Red Reagent/HRP. Mix 50 uL of 10 mM Amplex Red reagent stock soln + 100 uL of 10 U/mL HRP stock soln, 4.85 mL of 1X Rxn buffer.
Begin rxn. Add50 uL of Amplex Red reagent/HRP working soln to each microplate well containing standards/controls/samples.
Incubate rxns at room temp for 30 mins, protected from light.
Use plate reader to detect fluorescence (excitation 530-560 nm, emission 590nm, absorbance 560nm).
For each point subtract value derived from no-H2O2 control
Inclusion Body Solubilization Reagent · Benchling
Inclusion Body Solubilization Reagent
Introduction
Recover insoluble proteins expressed in inclusion bodies (protein aggregates)
Materials
B-PER Bacterial Protein Extraction Reagent
Inclusion Body Solubilization Reagent
Procedure
Protein solubilization
Estimate amount of inclusion body prep by subtracting weight of centifuge tube from total weight. Use 8 mL of Inclusion Body Solubilization Reagent per 1 g of wet inclusion body pellet.
Suspend pellet in appropriate amount of Inclusion Body Solubilization by vortexing or pipetting. Shake suspension of 30 min.
Remove cell debris by centrifugation at 27k x g (15k rpm) for 15 min.
Collect supernatant which contains the solubilized protein. For protein assay, use Coomassie Plus (Bradford) Protein Assay Kit. For SDS-PAGE analysis, remove denaturant by dialysis.
Large Scale Extraction · Benchling
Large Scale Extraction
Introduction
Get started by giving your protocol a name and editing this introduction.
Materials
1 g of wet cell paste or 250 mL bacterial culture with OD600 of ~2.0
CelLytic B
Procedure
Collect bacterial cells that express protein of interest by centrifuging at 5k x g for 10 min.
Carefully remove the spent medium from the cell pellet. The cell pellet may be frozen or used fresh. Frozen cell pellet will give slightly higher yield of protein
Add CelLytic B at ratio of 10-20 mL per gram of cell paste. Mix well to completely resuspend the cells.
Add lysozyme (final concentration = 0.2 mg/mL) to enhance cell lysis
Benzonase (final amount of 50 units/mL) to decrease viscosity of solution.
Protease inhibitors to prevent proteolytic degradation
Incubate extraction suspension with vortexing/shaking at room temperature for 10-15 minutes to fully extract soluble proteins from cells
After cell extraction, centrifuge the extract at 16k x g for 10 mins to pellet insoluble material.
Carefully remove supernatant containing soluble protein fraction. Another round of extraction will yield more soluble protein if required - but will result in more dilute soluble protein sample
Analyze supernatant and insoluble fraction by SDS-PAGE and/or Western  blot to determine which fraction contains protein of interest. For SDS-PAGE use 5-15 uL of each sample for gel.
Ligating Complementary Strands (Gibson) · Benchling
Ligating Complementary Strands (Gibson)
Introduction
Ligating complementary strands before Gibson Assembly and transformation.
Materials
5ul T4 DNA ligase buffer (in freezer in 348 with other restriction enzymes)
43ul DI water
1ul forward strands (undiluted)
1ul reverse strands (undiluted)
2.5ul 1M NaCl
PCR tubes
Procedure
Ligation
Thaw ligase buffer in hand and vortex, then put immediately on ice.
Mix materials in PCR tubes with a total volume of 52.5ul
Set at 95˚C and cool at 0.1˚C on Jessica's thermocycler for 19 minutes
Dilute 1:10 if doing Gibson
Do Gibson protocol
Liquid Culture · Benchling
Liquid Culture
Introduction
Liquid culture of cells
Materials
Antibiotic
LB broth
Cell culture
Procedure
Liquid Culture
Add 5ml LB broth to 15ml tube.
Add 5ul antibiotic.
Add +/- 1ul/scrape of cells.
Shake up and incubate at 37˚C at 180rpm.
Lumio Green Detection · Benchling
Lumio Green Detection
Introduction
It's pretty great. You can do this even before inducing.
Materials
Procedure
Making LB Agar Plates · Benchling
Making LB Agar Plates
Introduction
Making LB agar plates.
Materials
Agar powder
LB broth
1ml antibiotic/L of liquid
Procedure
Making Broth
Add agar powder (15g/L) to LB broth.
Autoclave.
Cool to 55˚C then add antibiotic. Pour into petri dishes.
Allow to harden and store inverted at 4˚C in the dark (antibiotic is light sensitive).
If agar solidifies too early, possible to microwave it but it may explode.
Nanodrop Reader · Benchling
Nanodrop Reader
Introduction
Reading DNA concentration with a Nanodrop.
Materials
Tube of DNA
Procedure
Reading DNA
Read blank of 1ul elution buffer.
Run each sample, name at Sample ID, hit measure.
Want 260/280 + 260/280 ~ 2.
Record concentrations on stored tubes.
Native PAGE · Benchling
Native PAGE
Introduction
Actually this is on a card I don't possess, should be in the lab though. I'll look it up at some point soon. Good luck Anna!
Materials
Procedure
PCR · Benchling
PCR
Introduction
Get started by giving your protocol a name and editing this introduction. *Primer Tm 50˚C target*
Materials
Taq master mix
Forward primer
Reverse primer
DI water
Template DNA
PCR tubes
Procedure
Polymerase Chain Reaction
In a PCR tube, mix:
12.5ul Taq master mix (green)
1ul 10uM forward primer (1:20 dilution may be needed)
1ul 10uM reverse primer (1:20 dilution may be needed)
10.5ul super DI water
Touch of template DNA
Maintain all on ice throughout process.
Gently mix tube.
Place in thermocycler and cycle at:
94˚C (30s)
Circle 30 times: 94˚C (15-30s), 45-68˚C (15-60s), 68˚C (1min/kb)
68˚C (5min)
Final hold 4-10˚C.
Phosphorylation and Ligation of Repetitive DNA Building Blocks · Benchling
Phosphorylation and Ligation of Repetitive DNA Building Blocks
Introduction
Use this protocol to phosphorylate and repetitive DNA buidling blocks, and ligate them into randomized long repetitive strands.
Materials
Materials per Tube (Phosphorylation) Volume
DNA building block mixture (80 µM) 5 µl
Ligase 10x Buffer 2 µl
DI Water 11.5 µl
T4 Polynucleotide Kinase (PNK) 1.5 µl
Total Volume (Phosphorylation) 20 µl
Materials per Tube (Ligation)
Phosphorylation mix (above) 20 µl
T4 DNA Ligase 1 µl
Total Volume (Phosphorylation) 21 µl
Procedure
Phosphorylation
Place all materials on ice to thaw. If needed, thaw Ligase 10x Buffer in hands and vortex until all white precipitate has disolved. Place back on ice for 2 min before adding to rxn mixture.
Add materials listed above with desired 80 µM DNA building block mix to PCR tube
Incubate in thermal cycler at 37°C for 1 hr
Ligation
Add 1 µl T4 DNA Ligase to each tube -- vortex and spin down to mix if
Incubate at 16°C overnight
Scanning a Gel · Benchling
Scanning a Gel
Introduction
Scanning gels.
Materials
Gel to read
Procedure
Carefully remove gel from box and place on scanner.
Select area on scanner.
Run scanner.
Sequencing Order · Benchling
Sequencing Order
Introduction
https://www.elimbio.com/
Materials
1 uL primer_F (VF) diluted 1:20
1 1uL primer_R (VR) diluted 1:20
30ng DNA
15 uL total volume water
Procedure
Small/Trial Scale Extraction · Benchling
Small/Trial Scale Extraction
Introduction
Determine the fraction in which the protein of interest will be found
Materials
Bacterial culture OD600 0.5-1.0
CelLytic B
Procedure
Use 1.5 mL of bacterial culture with OD600 of 0.5-1.0 and centrifuge the cells at full speed/7k for 2 min.
Remove spun medium and resuspend the cell pellet in 0.4 CelLytic B.
Carefully remove the soluble protein fraction from the cell debris. Additional extractions may be performed if required, but will make your soluble protein sample more dilute.
Analyze supernatant and insoluble fraction by SDS-PAGE and/or Western blot to determine which fraction contains the protein of interest.Fir SDS-PAGE, use 5-15 uL of each sample for gel.
T7 Transformations · Benchling
T7 Transformations
Introduction
High efficiency T7 protocol.
Materials
See general transformation materials
Procedure
Transformations
Thaw cells on ice for 10min.
Add 1-5ul contianing 1pg-100ng plasmid DNA (eg from Gibson assembly).
Place mixture on ice for 30min. Do not mix.
Heat shock at 42˚C for 10s exactly.
Place on ice for 5min.
Pipette 950ul room temp SOC.
Place at 37˚C for 60min, shaking at 250rpm.
Warm selection plates to 37˚C.
Mix cells and perform dilutions.
Spread 50-100ul of cell dilution onto plates. Incubate overnight at 37˚C.
Transformation: Chemically Competent Cells · Benchling
Transformation: Chemically Competent Cells
Introduction
Transformation protocol for chemically competent cells.
Materials
SOC (super optimal broth with catabolite repression)
NED 5-alpha competent E.coli cells (Half a stock)
Purified DNA
Agar plates with chloramphenicol
Procedure
Transformation
Thaw NEB 5-alpha competent E.coli on ice for 10 minutes (Half a stock is fine, half each subsequent amount)
Add 1-5ul of 1pg-100ng plasmid DNA to cell mixture. Flick tube to mix, do not vortex.
Place on ice for 30min. Do not mix.
Heat shock at 42˚C for 30s.
Place on ice for 5min.
Pipette 950ul room temp SOC into mixture (475 if using half stock).
Place at 37˚C for 60 minutes, shaking at 250rpm.
Perform several 10-fold dilutions (1:10, 1:100) of cells.
Plate 50-100ul of each dilution onto an antibiotic selection plate.
Incubate overnight at 37˚C.
Transformation: Electrocompetent Cells · Benchling
Transformation: Electrocompetent Cells
Introduction
Transformation protocol for electrocompetent cells. To make electrocompetent cells, refer to: http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3939052/.
Materials
Agar plates with chloramphenicol
NED 5-alpha competent E.coli cells
SOC media
Purified DNA
Procedure
Transformation
Place electroporation cuvettes, microcentrifuge, and cuvette holder on ice.
Thaw NEB 5-alpha electrocompetent E.coli on ice for 10min.
Transfer 25ul of cells and 1ul DNA into microcentrifuge tube.
Transfer mixture to cuvette. Make sure all parts are cold to avoid cells exploding.
Electroporate with the following conditions:
Add 975ul SOC in cell box to cuvette, mixing gently, then place in round bottom culture tube.
Incubate at 37˚C, shaking at 250rpm for 1hr.
Dilute cells (1:10, 1:100) and spread 100-200ul onto antibiotic selection plate.
Incubate cells overnight at 37˚C.
