<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stanford-Brown_SB16_BioMembrane_Latex skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stanford-Brown/SB16 BioMembrane Latex</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Stanford-Brown 2016</TITLE></P><DIV class="container"><NAV class="navbar navbar-inverse navbar-fixed-top"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" id="BioballoonLabel" href="https://2016.igem.org/Team:Stanford-Brown">Stanford-Brown</A><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Stanford-Brown">Home</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Attributions">Attributions</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle active" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">BioMembrane</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_Collagen">Collagen</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_Elastin">Elastin</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_p-Aramid">p-Aramid</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_Latex">Latex</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_UV">UV Protection</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Float</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Float_Gas">Gas production</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">BioSensor</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioSensor_Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioSensor_Chromoproteins">Chromoproteins</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioSensor_FQsensor">Fluorophore-Quencher</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_BioMembrane_AptamerPurification">Aptamer Purification</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Practices</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Human_Practices">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Integrated_Practices">Integrated Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Engagement">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Practices_Exploration">Life Beyond the Lab</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Collaborations_Interlab">InterLab Study</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Collaborations">Team Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Parts">BioBricks</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Software">Software Design</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Modeling">Modeling</A></LI><LI class="dropdown-header">Notebooks</LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_CE">Collagen &amp; Elastin</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_p-Aramid">P-aramid</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_Latex">Latex</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_UV">UV protection</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_AptamerPurification">Aptamer Purification</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_Gas">Gas Production</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_Chromoproteins">Chromoproteins</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_FQsensor">Fluorophore-Quencher</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_Nylon">Nylon</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_Notebooks_Interlab">Interlab Study</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Team">Team</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Requirements</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/SB16_MedalRequirements">Medals</A></LI><LI><A href="https://2016.igem.org/Team:Stanford-Brown/Safety">Safety</A></LI></DIV></DIV></NAV></DIV><DIV class="container"><DIV class="row rowT"><DIV class="col-sm-12 col-PT noHead"><H1 class="sectionTitle-L firstTitle">Metabolic Engineering of <I>E. coli</I> for Cis 1,4-polyisoprene (Latex) Production</H1></DIV></DIV><H2 class="subHead">Abstract</H2><DIV class="row"><DIV class="col-sm-7 pagetext-L"><DIV class="text">With rising costs in synthetic rubber chemical synthesis, environmental blight, and deforestation diminishing the annual yield of natural rubber plantations, a new alternative for latex production is needed to address its global demand shortfall.  To address this issue, we sought to transform the latex synthesis pathway into a single cell organism that could be grown in bioreactors, such as <I>Escherichia coli</I>.  Due to its low doubling time and ability to be cultured in bulk, genetically modified <I>E. coli</I> capable of producing latex offer a promising solution for fast, high yield latex production. Through genetic manipulation of the endogenous methylerythritol phosphate (MEP/DOXP) pathway and transformation with rubber production genes from <I>Hevea brasiliensis</I>, we developed a transgenic single cell organism capable of converting glucose into cis-1,4-polyisoprene, the primary chemical constituent in latex.  Not only is our modified organism capable of producing cis-polyisoprenes quickly, but also in high yield.</DIV></DIV><DIV class="col-sm-5 imgcol-R"><FIGURE class="fig"><FIGCAPTION>Figure 1: MEP/DOXP pathway. The rate limiting step is the first step, in which pyruvate and G3P are converted into DXP. Through constitutive expression of DXS we are able to accelerate the process and push the equilibrium of the reaction towards favoring the products, IPP and DMAPP.[20]</FIGCAPTION></FIGURE></DIV></DIV><H2 class="subHead">The problem with production</H2><DIV class="row"><DIV class="col-sm-12 pagetext">Produced by the rubber tree <I>Hevea brasiliensis</I>, natural rubber is an emulsion consisting of numerous proteins, starches, sugars, oils, resins, and alkaloids.  From this emulsion latex is perhaps the most important product.  Used in a wide variety of applications, latex accounts for the highest fraction of technically used elastomers, besides polyesters that consist of medium chain length hydroxyalkanoates (PHA<SUB>MCL</SUB>).[1, 2] Additionally, latex exhibits a large stretch ratio and high resilience to repeated stress, which makes it an ideal material for constructing flexible yet durable structures.[3] Because of its structural properties, latex is an ideal material for constructing flexible structures that need to adjust to variable mechanical stresses.
		</DIV></DIV><DIV class="row"><DIV class="col-sm-12 pagetext">Currently the only source of commercially usable natural rubber that can be processed into latex is available from the rubber tree <I>H. brasiliensis</I>.  While other plants are capable of producing rubber particles, these particles when processed are weaker, requiring less extension to break, compared to natural rubbers produced by <I>H. brasiliensis</I>.[4] In fact, <I>H. brasiliensis</I> is responsible for almost all of the world's natural rubber production through mostly rubber plantations or tree tapping.[2]  Rubber farming however in recent years has been threatened by production shortfalls owing to diseases such as South American Leaf blight. <I>H. brasiliensis</I>’ narrow genetic base also signifies most large acreage farms plant genetically identical trees, making them prone to large crop failure.[5, 6] This problem is further exacerbated by deforestation and the growing land need for agriculture, which both decrease the amount of land available for rubber tree plantations, and consequently limit rubber production.[7, 8]

		Due to the difficulties of harvest and acreage demand on latex plantations, chemical synthesis of synthetic latex is appealing alternative to natural latex. Although natural latex and synthetic latex have different chemical and physical properties, both materials are largely comprised of cis-1,4-polyisoprene polymers. While natural latex is difficult to handle and has diminished durability, resilience, and elasticity without vulcanization, synthetic latex does not require vulcanization and can be prepared using different proportions of isoprene monomers to yield a wide range of physical, mechanical, and chemical properties.[9, 10] By varying the mixture of isoprene and styrene butadiene polymers, synthetic rubbers have a unique advantage in that they can be tuned to a particular use.  However, synthetic rubber lacks the mechanical and low temperature performance of its natural counterpart.  Despite characteristic differences, both natural latex and synthetic latex rely heavily on cis-polyisoprene polymers as their primary constituent.

		With global consumption of latex at over 11 million metric tons per annum, latex is an essential raw material worldwide.[11] Currently, natural rubber accounts for 40% of the global rubber demand, with the remaining 60% supplied from synthetic rubber.[12] However, the increasing price of petroleum has elevated prices in the synthetic rubber industry and consequently exacerbated the current market shortfall of natural rubbers.  Additionally, butadiene, the primary monomer used in synthetic rubber synthesis, is facing a global shortage which is increasing the cost of synthetic rubber synthesis.[13] With an increasing demand of 5-6% per annum, the global latex economy cannot be sustained by the elevating cost synthetic rubber synthesis and dependence on shrinking latex farms.[8]  For these reasons, an alternative method for latex production is desperately needed to sustain global demand and undercut material shortages. 

		Through our project, we sought to address the need for a faster and more economical alternative latex production system to mitigate the accumulating economic demand shortfall.  To achieve our objective, we sought to design a transgenic organism capable of mimicking the natural rubber production process found in <I>H. brasiliensis</I>.  Not only would a transgenic single cell organism allow for optimization of polymer synthesis, but also permit a high yield in polymer producing cells.  A scaled up cell culture bioreactor would yield large volumes of both latex (cis-polyisoprene) producing cells and cis-polyisoprene polymers with minimal growth media input.  For these reasons, we were motivated towards metabolic engineering <I>Escherichia coli</I> for high yield production of cis-polyisoprene compounds which could be used as a synthetic and natural latex replacement or precursor.

		Our objective through this project was to enable <I>E. coli</I> to produce isoprene polymers (when supplied basic cell growth media and nutrients, such as glucose, magnesium, and nitrogen compounds.  Our invention must also be scalable in culture to allow for high volume production of polyisoprene compounds.  To develop the latex synthesis pathway in <I>E. coli</I>, we first examined the chemical constituents and the metabolic processes responsible for latex production.
		</DIV></DIV><H2 class="subHead">Rubber (cis-1,4-polyisoprene) synthesis pathway</H2><DIV class="row"><DIV class="col-sm-12 pagetext">
		Although there are a variety of polyisoprene polymer forms, natural rubber is largely composed of cis-1,4-polyisoprene, which is a polymer of isoprene units joined at the first and fourth carbon by a double bond.  In order to produce the cis-1,4-polyisoprene polymer, rubber trees such as <I>H. brasiliensis</I> employ cis-prenyltransferase enzymes (also commonly referred to as rubber transferases) to link individual isoprene monomers into a polymer.  The monomer used primarily in a cis-1,4-polyisoprene chain is isopentenyl pyrophosphate (IPP), which is also the isomer of its more reactive counterpart, dimethyl allyl pyrophosphate (DMAPP).  In order to initialize chain elongation, prenyltransferase enzymes begin single DMAPP molecule and iteratively add IPP onto the chain, allowing for extension of the polymer.  However, to be enzymatically active, cis-prenyltransferase enzymes require a magnesium(II) supplement and small rubber particle protein (SRPP) coenzyme that serve as activators.[14] Without a magnesium (II)  supplement and SRPPs, prenyltransferases are incapable of polymer chain extension.  Therefore, in order to polymerize rubber chains, the key xenogeneic components our host organism will need include (1) prenyltransferases, (2) IPP substrate, (3) DMAPP substrate, (4) a magnesium(II) supplement (such as MgSO<SUB>4</SUB>), and (5) small rubber particle protein coenzymes.
	</DIV></DIV><H2 class="subHead">DXS synthase optimization</H2><DIV class="row"><DIV class="col-sm-12 pagetext">
		Although IPP and DMAPP could be supplied to the cell by a media supplement, a more appealing and system incorporated approach is to produce both compounds within the host organism using only glucose.  Since IPP and DMAPP are endogenous to many species, we identified the two IPP biosynthesis pathways: the mevalonate (MVA) pathway and methylerythritol phosphate (MEP/DOXP) pathway.  While pathways generate IPP and DMAPP from ; however, only the MEP/DOXP pathway is endogenous in <I>E. coli</I>.[15] Through a six step process, the MEP/DOXP pathway converts molecules of pyruvate and glyceraldehyde 3-phosphate (G3P) into IPP and DMAPP (Figure 1).  While incorporating the MVA pathway into E. coli would further enhance IPP and DMAPP output, we opted for optimizing the MEP/DOXP pathway to minimize the step count from converting a substrate into IPP.[16, 17] Additionally, since G3P and pyruvate are products of glycolysis, providing a glucose supplement to <I>E. coli</I> would allow for G3P and pyruvate use by the MEP/DOXP pathway without increasing cell stress.[18]</DIV></DIV><DIV class="row"><DIV class="col-sm-5 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 2: Plasmid map of DXS synthase vector.  DXS synthase gene is linked to a T7 Elowitz IPTG inducible constitutive promoter that will allow for high expression of DXS.</FIGCAPTION></FIGURE></DIV><DIV class="col-sm-7 pagetext-R"><DIV class="text">To optimize the MEP/DOXP pathway, we overexpressed reaction rate limiting enzymes in the pathway, namely DXP synthase (DXS).  Although we could have overexpressed all of the enzymes in the MEP/DOXP pathway, from prior research DXS was determined to be the rate limiting step in the metabolic pathway.[13] To control for DXS expression, we codon optimized and reintroduced the DXS enzyme into <I>E. coli</I> under an IPTG inducible constitutive promoter (DXS plasmid), allowing for regulated production of DXS (Figure 2).  A supplement of IPTG would then activate high level expression of DXS, which in turn would override the pathway bottleneck at the beginning of the MEP/DOXP pathway.  In accelerating the process limiting chemical conversion of G3P and pyruvate to 1-Deoxy-D-xylulose 5-phosphate (DXP or DOXP), we can subsequently receive a greater output of IPP and DMAPP downstream. </DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-12 pagetext">
		In order to design our optimized DXS synthase, we obtained the sequence for the DXS gene from EcoGene (EG13612).  To allow for future protein purification and characterization, we also attached a FLAG, tetracysteine, and hexahistidine tag to the DXS gene.  A double terminator, (BBa_B0010 and BBa_B0012) was also used to prevent RNA polymerase leak through.[19] Because the construct was too large to synthesize as one part, we fragmented the gene into two parts and used Gibson Assembly to insert them into linearized pSB1A3.  This resulted in a plasmid with the full DXS Synthase gene and ampicillin resistance, which was then transformed into T7 express cells.
		</DIV></DIV><H2 class="subHead">Incorporating rubber synthesis into <I>E. coli</I></H2><DIV class="row"><DIV class="col-sm-5 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 3: Construct containing prenyltransferases HRT1, HRT2, and SRPP cofactor.  All components are driven by a T7 IPTG inducible constitutive promoter, with unique ribosome binding spots.</FIGCAPTION></FIGURE></DIV><DIV class="col-sm-7 pagetext-R"><DIV class="text">
			While DXS synthase optimization can increase the amount of IPP and DMAPP substrate needed for cis-1,4-polyisoprene formation, we also need to introduce the xenogeneic components of <I>H. brasiliensis</I> into <I>E. coli</I> to enable synthesis of isoprene polymers.  To do so, we introduced a second component into our system—the prenyltransferases and SRPP coenzyme needed for isoprene extension.  We identified well characterized and isolated cDNAs from <I>H. Brasiliensis</I> for prenyltransferases participating in natural rubber biosynthesis.  Two enzymes were selected, HRT1 and HRT2, which were two cis-prenyl chain elongating enzymes isolated from <I>Hevea</I> latex.[21] These proteins are responsible for the synthesis of new rubber molecules and are also found expressed predominantly in fresh <I>Hevea</I> latex.  Because either HRT1 or HRT2 could be used for rubber synthesis, we incorporated both genes into a plasmid construct under a IPTG inducible constitutive promoter to allow for maximum expression of enzyme.</DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-12 pagetext">
		Since prenyltransferase requires an SRPP and various chemical cofactors for functionality, we identified the most commonly expressed SRPP in <I>Hevea</I> latex as a possible cofactor for HRT1 or HRT2 (Accession No. AB061234.2 and AB064661.2).[11, 22] However, because SRPPs can play a variety of roles in <I>Hevea</I> latex production, we targeted SRPPs that had also been identified as rubber elongation factors.[23] Our criterion for SRPP selection were that it had to be (1) a rubber elongation factor, (2) highly expressed in Hevea latex, and (3) of the small variety, since large rubber particle proteins would be difficult to produce in <I>E. coli</I> due to their size.[24] The gene of our SRPP (AF051317) was subsequently incorporated into the prenyltransferase cassette.[16] Since prenyltransferase activity depends on SRPP presence and gene order in an operon can influence expression level, we inserted the SRPP gene before the two prenyltransferases to account for the possibility the first protein in the cassette would be expressed the most.  This cassette was the linked to a IPTG inducible T7 Elowitz high copy promoters to allow for regulated expression.  All proteins were also tagged with a FLAG, tetracysteine, and hexahistidine tag to allow for protein purification assays (Figure 3). A double terminator, (BBa_B0010 and BBa_B0012) was also used to prevent RNA polymerase leak through.[14]

		In order to produce our plasmid, we synthesized SRPP, HRT1, HRT2 on three different fragments.  These fragments were then linked together via Gibson Assembly, which yielded a cassette of genes.  This cassette was then Gibson’d into pSB1C3, which granted the construct chloroamphenicol (Chl) resistance.  The construct (referred henceafter as latex operon) was subsequently transformed into T7 express cells.
	</DIV></DIV><H2 class="subHead">Bringing it all together</H2><DIV class="row"><DIV class="col-sm-12 pagetext">
		Although our system is located on two plasmids, all four components (HRT1, HRT2, SRPP, and DXS synthase) could be included on the same construct for ease of transfection.  However, under our present schema, transfection of <I>E. coli</I> with the DXS plasmid would allow for an organism that produces IPP and DMAPP in high volume potentially for alternative applications, such as terpenoid synthesis.  For full realization of high throughput IPP and cis-1,4-polyisoprene production in <I>E. coli</I>, both plasmids need to be transfected into the host to allow for high activity of all pathway processes.  To verify all elements are property transfected, each plasmid had a distinct selection marker such that with each subsequent transfection, un-transfected cells would be screened against.  By transfecting one plasmid at a time into a population of cells with different selection assays, we ensured that the final population of cells at the end of the process contained both plasmid constructs.

		An item of concern is the metabolic strain each of these plasmids will place upon <I>E. coli</I>.  Since IPP and DMAPP are byproducts of glycolysis and cellular respiration; enhanced conversion of pyruvate and G3P or acetyl -CoA to IPP will decrease the amount of substrate available to the host for ATP production.  In order to address this, we included an IPTG inducible promoter into the plasmids to allow for IPTG induced expression of protein.  Introducing IPTG into the cell media would activate transcription and subsequently production of protein.
	</DIV></DIV><DIV class="row"><DIV class="col-sm-5 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 4: An example of a lumio gel done on a DXS synthase precursor experiment.</FIGCAPTION></FIGURE></DIV><DIV class="col-sm-7 pagetext-R"><DIV class="text">
			For an initial product test, we scraped three colonies from our plates containing bacteria with the latex operon and three colonies from our plates containing bacteria with our DXS Synthase product. Protein extraction was done on the soluble fractions for all 6 cultures, as well as the insoluble fractions for all DXS colonies. Our protein extract was run on a LUMIO gel, with DXS Synthase and the Latex Operon both appearing within the soluble fraction(Figure 4).

			Although we confirmed the synthesis of both proteins in our initial constructs, we quickly realized that we needed to Gibson Assembly our DXS Synthase gene into pUC19 as an alternative backbone.  This is owing to the fact that both pSB1C3 and pSB1A3 share the same ORI, which could result in homologous recombination of the plasmids when both are transfected into the same cell.  Consequently, both the latex operon and DXS plasmid needed to have a different ORI. Primers were reordered to complete PCR, Gibson Assembly, and PCR cleanup for pUC19 and our DXS synthase gene.  The finished pUC19+DXS construct was then miniprepped and transformed into the T7 express cells containing the latex operon.  Colonies grew successfully on LB plates with ampicillin (pUC19) and chloramphenicol (pSB1C3) resistance, as expected. These colonies were then grown up in liquid culture for latex extraction.</DIV></DIV></DIV><H2 class="subHead">Growing latex in culture</H2><DIV class="row"><DIV class="col-sm-5 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 5: HRT1, HRT2, SRPP, and DXS expression is induced by IPTG presence.  Once produced, HRT1 and HRT2 alone are inactive(red).  When bound with both SRPP and MgSO4, it becomes enzymatically activate(green).  DXS synthase does not require any additional cofactors/activators for activity.</FIGCAPTION></FIGURE></DIV><DIV class="col-sm-7 pagetext-R"><DIV class="text">
			After <I>E. coli</I> cells have been transfected with both plasmids, a supplement of magnesium sulfate, IPTG, and glucose is necessary to induce protein expression, activity, and cis-polyisoprene polymer synthesis.  IPTG is necessary for expression of both the DXS plasmid and latex operon proteins; magnesium sulfate is needed as a cofactor for HRT1 and HRT2 to be enzymatically active; and glucose is needed as a substrate (Figure 5,6).</DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-7 pagetext-L"><DIV class="text">
			To induce latex production, IPTG (400 mM), glucose (500 mM), and MgSO<SUB>4</SUB>(500 mM) were added directly to 350 ml liquid cultures growing at 37˚C at 100 uL per 1 mL. After being kept overnight, a second round of IPTG, glucose, and MgSO<SUB>4</SUB> was added. Latex extraction was performed on 350 ml of our cells containing the plasmids for the Latex Operon and DXS Synthase. Cells were spun down and fractionated into the cell pellet and LB media. In order to lyse the cells, chloroform was used—however because chloroform is a nonpolar solvent, it dissolves polyisoprenes in solution.[25, 26]  After the cells were agitated and lysed with chloroform, the solution was filtered to yield chloroform and its solutes, and methanol was added to precipitate the latex.  The LB fraction was also subject to a similar procedure except the chloroform layer was obtained from the LB fraction by separation of the solvent layers.</DIV></DIV><DIV class="col-sm-5 imgcol-R"><FIGURE class="fig"><FIGCAPTION>Figure 6: DXS synthase accelerates the conversion of pyruvate and G3P to DXP in the endogenous MEP/DOXP pathway.  Downstream, the activated HRT1 and HRT2 complexes can extend IPP (left) into polyisoprene (right) via dephosphorization.</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row"><DIV class="col-sm-12 pagetext">
		After this procedure, we precipitated 1.5 ml of a rubbery, off-white polymer from the methanol as expected. For a quick characterization test, the polymer was dried and burned, which resulted in a smell of burning rubber and a strong black smoke—typical of rubber fires.  Additionally, our extract (when dried) is highly compressible and elastic.  Further tests with mass spectrometry, acetone characterization, degradation, and physical stress tests will be needed to confirm the composition and validity of our latex.
	</DIV></DIV><DIV class="row rowT"><DIV class="col-sm-5 imgcol-L"><FIGURE class="fig"><FIGCAPTION>Figure 7: A close-up of our manufactured latex: an off-white, rubbery substance precipitated in our methanol experiments.</FIGCAPTION></FIGURE></DIV><DIV class="col-sm-5 imgcol-R"><FIGURE class="fig"><FIGCAPTION>Figure 8: A vial of latex dissolved in methanol(70%).  Note the white fibers throughout the solution</FIGCAPTION></FIGURE></DIV></DIV><H2 class="subHead">Our work vs prior iGEM work</H2><DIV class="row"><DIV class="col-sm-12 pagetext">
		Prior iGEM teams that have experimented with latex include the 2015 Brazil USP team and the 2013 Denmark Team.The 2015 Brazil USP Team targeted devulcanization and decomposition of latex in landfills.  Through expressing RoxA, a rubber deoxygenase, and Lcp, a latex clearing protein, their project focused on developing an organism that could compost rubber in landfills to reduce landfill acreage demand.[27]
	</DIV></DIV><DIV class="row"><DIV class="col-sm-7 pagetext"><DIV class="text">
			The 2013 Denmark team focused on latex production through similar pathways as our own.[28]  However, our project has the novelty of incorporating regulatory elements and additional prenyltransferases and their cofactors to de-regulate latex production at the network scale.  In particular, our process is novel in that we supplement our cells with magnesium, glucose, and SRPP protein cofactors to augment the enzymatic activity of both HRT1 and HRT2 prenyltransferases.  Furthermore, our system also produces latex in high yield, as demonstrated in Figure 7.</DIV></DIV><DIV class="col-sm-5 imgcol-R"><FIGURE class="fig"><FIGCAPTION>Figure 9: Denmark 2013 Latex production- unlike our system, the 2013 Denmark <I>E. coli</I> appear to have a lower production of latex.[28]</FIGCAPTION></FIGURE></DIV></DIV><H2 class="subHead">Conclusions</H2><DIV class="row"><DIV class="col-sm-12 pagetext">
		As of present, we have filed a provisional patent for our invention with the USPTO.  We are currently in the process of conducting material and chemical verification tests to evaluate the characteristics of our rubber.  Additionally, we are conducting tests on the average length of polymer yield.On the materials aspect, we are also experimenting with vulcanization of our rubber with sulfur to see if we can alter the material properties of our rubber.Through metabolic engineering, we were able to successfully produce an organism capable of producing polyisoprenes in large yield.  By optimizing the endogenous MEP/DOXP pathway in <I>E. coli</I> through overexpression of rate-limiting proteins and coupling it with expression of rubber transferases (prenyltransferases) used by <I>H. brasiliensis</I> for polyisoprene production, we were able to obtain a rubber-like substance that exhibits the basic properties of latex.<I>Unique/Novel Features</I><UL><LI>Use two main rubber transferases, HRT1, HRT2</LI><LI>Employs the use of SRPP coenzyme and magnesium (II) cofactor to maximize prenyltransferase activity</LI><LI>IPTG inducible system</LI><LI>SRPP and prenyltransferase gene expression are linked, as they are on the same plasmid.</LI><LI>Glucose supplement to cells for a larger amount of substrate to convert to latex polymer.</LI><LI>Magnesium sulfate supplement is included in cell culture to enhance prenyltransferase activity</LI><LI>Can be combined into one construct</LI></UL><I>Development Timeframe:</I>
		June 15th, 2016-Present<I>Development History:</I><UL><LI>July 10th, 2016 – Idea was conceptualized and experiments were planned.</LI><LI>July 20th, 2016 – Amplification of pSB1C3 backbone for plasmid construction</LI><LI>July 27th, 2016 – Gibson assembly of Latex operon gene fragments prior to incorporation into vector</LI><LI>August 1-2nd, 2016 – Successful cloning and assembly of latex operon (rubber transferases and small rubber particle protein genes).  Gene construct was successfully transformed into <I>E. coli</I> DH10B cells.</LI><LI>August 14th, 2016 – Successful assembly of DXS gene construct and transfection of aforementioned construct into <I>E. coli</I> cells.</LI><LI>August 26th, 2016 – Large liquid cultures of DXS plasmid <I>E. coli</I> and latex operon <I>E. coli</I> for large scale plasmid extraction</LI><LI>September 13th, 2016 – DXS gene gibsoned into pUC19 backbone, cells with Latex operon were transfected with DXS + pUC19 construct</LI><LI>September 22nd, 2016 – Successful double transfection of <I>E. coli</I> with both DXS and latex operon constructs.</LI><LI>October 7th ,2016 – Successful extraction of cis-1,4-polyisoprene (latex) polymers from genetically modified <I>E. coli</I> cells</LI><LI>October 10th, 2016 – Successful burn test of latex polymers from cell extract – black smoke and pungent smell verified the presence of organic rubber compounds.</LI></UL><I>Contributions:</I><UL><LI>Gordon Sun – Conceived, designed, and conducted the experiments for the development of a latex (cis-1,4-polyisoprene) producing synthetic organism.  Designed, assembled, and transfected transfection vectors for both latex and DXS operons.   Wrote website manuscript and documentation of experimental design.</LI><LI>Elias Robinson – Optimized gene constructs for dual plasmid expression and enhanced operon efficiency. Assembled transfection vectors and transformed host cells. </LI><LI>Taylor Sihavong – Conducted and optimized latex extraction and purification tests.  Optimized and designed DXS, pUC19 construct assembly and troubleshooted plasmid design.</LI></UL></DIV></DIV><DIV class="row"><DIV class="col-sm-12 pagetext references"><I>References:</I><OL><LI>National Weather Service Weather Forecast Service; Weather Balloons; 8 Oct 2012.</LI><LI>Klinke S, Ren Q, Witholt B, Kessler B. Production of medium-chain-length poly(3-hydroxyalkanoates) from gluconate by recombinant Escherichia coli. Appl Environ Microbiol. 1999;65(2):540-8.</LI><LI>Steinbuchel, Alexander.  Production of Rubber-Like polymers by microorganisms. Current Opinion in Microbiology 2003, 6:261-270</LI><LI>Cornish, Katrina and Blakeslee, Joshua J. Rubber Biosynthesis in Plants. AOCS Lipid Library.  2 Nov 2011.</LI><LI>Lange BM, Rujan T, Martin W, Croteau R. Isoprenoid biosynthesis: the evolution of two ancient and distinct pathways across genomes. Proc Natl Acad Sci USA. 2000;97(24):13172-7.</LI><LI>Thomas M. Lewinsohn (1991). &quot;The geographical distribution of plant latex&quot;. Chemoecology 2 (1): 64–68</LI><LI>UN projects the global population to grow from 6.1 billion (2000) to 8.9 billions in 2050.</LI><LI>Li, H., et al: Demand for rubber is causing the loss of high diversity rain forest in SW China. Biodivers Conserv (2007) 16:1731–1745</LI><LI>&quot;Natural Vs Synthetic Polymers-Gelfand - Carnegie Mellon University&quot;. Leonard Gelfand Center. N.p., 2016. Web. 1 Oct. 2016.</LI><LI>Smith, Emily, T. &quot;Rubber That's So Tough it Goes the Extra Mile.&quot; Business Week (February 11, 1991): 80.</LI><LI>Black, Thomas and Jack Kaskey. &quot;Tire Raw-Material Shortage Triggers Race For Answer&quot;. Bloomberg.com. N.p., 2014. Web. 1 Oct. 2016.</LI><LI>Tanaka, Y., &amp; J. T. Sakdapipanich. 2001. Chemical structure and occurrence of natural polyisoprenes, p. 1-25. In T. Koyama &amp; A. Steinbüchel (ed.), Biopolymers, vol. 2. Polyisoprenoids. Wiley-VCH, Weinheim, Germany.</LI><LI>“World Butadiene supply to tighten.” Oil and Gas Journal. 22 Jan 2001. Web.</LI><LI>Epping, J. et al. A rubber transferase activator is necessary for natural rubber biosynthesis in dandelion. NPLANTS Nature Plants 1, 15048 (2015).</LI><LI>Wiemer, AJ; Hsiao, CH; Wiemer, DF (2010). &quot;Isoprenoid metabolism as a therapeutic target in gram-negative pathogens.&quot;. Current topics in medicinal chemistry 10 (18): 1858–71.</LI><LI>Buhaescu I, Izzedine H. Mevalonate pathway: A review of clinical and therapeutical implications. 2007. ClinBiochem 40:575–584.</LI><LI>Martin VJ, Pitera DJ, Withers ST, Newman JD, Keasling JD. Engineering a mevalonate pathway in Escherichia coli for production of terpenoids. Nat Biotechnol. 2003;21(7):796-802.</LI><LI>Zhao Y, Yang J, Qin B, et al. Biosynthesis of isoprene in Escherichia coli via methylerythritol phosphate (MEP) pathway. Appl Microbiol Biotechnol. 2011;90(6):1915-22.</LI><LI>Nielsen, Alec. “Genetic Circuits.” BioE 393. Stanford University. 2 May 2016. Conference.</LI><LI>Yikrazuul. Non-mevalonate Pathway. Digital image. Wikipedia, 24 Nov. 2016. Web. 20 June 2016. <HTTPS:>.</HTTPS:></LI><LI>Asawatreratanakul K, Zhang YW, Wititsuwannakul D, et al. Molecular cloning, expression and characterization of cDNA encoding cis-prenyltransferases from Hevea brasiliensis. A key factor participating in natural rubber biosynthesis. Eur J Biochem. 2003;270(23):4671-80.</LI><LI>Oh SK, Kang H, Shin DH, et al. Isolation, characterization, and functional analysis of a novel cDNA clone encoding a small rubber particle protein from Hevea brasiliensis. J Biol Chem. 1999;274(24):17132-8.</LI><LI>Xiang Q, Xia K, Dai L, et al. Proteome analysis of the large and the small rubber particles of Hevea brasiliensis using 2D-DIGE. Plant Physiol Biochem. 2012;60:207-13.</LI><LI>Rojruthai P, Sakdapipanich JT, Takahashi S, et al. In vitro synthesis of high molecular weight rubber by Hevea small rubber particles. J Biosci Bioeng. 2010;109(2):107-14.</LI><LI>Liengprayoon, S., Bonfils, F., Sainte-Beuve, J., Sriroth, K., Dubreucq, E. and Vaysse, L. (2008), Development of a new procedure for lipid extraction from Hevea brasiliensis natural rubber. Eur. J. Lipid Sci. Technol., 110: 563–569. doi:10.1002/ejlt.200700287</LI><LI>Ibrahim EM, Arenskötter M, Luftmann H, Steinbüchel A. Identification of poly(cis-1,4-Isoprene) degradation intermediates during growth of moderately thermophilic actinomycetes on rubber and cloning of a functional lcp homologue from Nocardia farcinica strain E1. Appl Environ Microbiol. 2006;72(5):3375-82.</LI><LI>IGEM 2015 Brasil-USP Webpage. https://2015.igem.org/Team:Brasil-USP#</LI><LI>IGEM 2013 SDU-Denmark Webpage. https://2013.igem.org/Team:SDU-Denmark/</LI></OL><I>Additional References (uncited):</I><OL><LI>&quot;Preparation of Synthetic Polyisoprene Latex and Its Use in Coagulant Dipping.&quot; Cariflex Polyisoprene Products. Kraton Performance Polymers, 2014. Web. 19 June 2016.</LI><LI>Berthelot K, Lecomte S, Estevez Y, et al. Rubber particle proteins, HbREF and HbSRPP, show different interactions with model membranes. Biochim Biophys Acta. 2014;1838(1 Pt B):287-99.</LI><LI>Durfee T, Nelson R, Baldwin S, et al. The complete genome sequence of Escherichia coli DH10B: insights into the biology of a laboratory workhorse. J Bacteriol. 2008;190(7):2597-606.</LI><LI>Farhi M, Kozin M, Duchin S, Vainstein A. Metabolic engineering of plants for artemisinin synthesis. Biotechnol Genet Eng Rev. 2013;29:135-48.</LI><LI>Han KH, Shin DH, Yang J, Kim IJ, Oh SK, Chow KS. Genes expressed in the latex of Hevea brasiliensis. Tree Physiol. 2000;20(8):503-510.</LI><LI>Stubbe J, Tian J, He A, Sinskey AJ, Lawrence AG, Liu P. Nontemplate-dependent polymerization processes: polyhydroxyalkanoate synthases as a paradigm. Annu Rev Biochem. 2005;74:433-80.</LI><LI>Thakor N, Trivedi U, Patel KC. Biosynthesis of medium chain length poly(3-hydroxyalkanoates) (mcl-PHAs) by Comamonas testosteroni during cultivation on vegetable oils. Bioresour Technol. 2005;96(17):1843-50.</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>