Team:Stanford-Brown/SB16 Notebooks CE
Stanford-Brown 2016
Toggle navigation
Stanford-Brown
Home
Attributions
BioMembrane
Overview
Collagen
Elastin
p-Aramid
Latex
UV Protection
Float
Gas production
BioSensor
Overview
Chromoproteins
Fluorophore-Quencher
Aptamer purification
Practices
Overview
Integrated Human Practices
Outreach
Life Beyond the Lab
Collaborations
InterLab Study
Team Collaborations
BioBricks
Protocols
Protocols
Software Design
Modeling
Notebooks
Collagen & Elastin
P-aramid
Latex
UV protection
Aptamer Purification
Gas Production
Chromoproteins
Fluorophore-Quencher
Nylon
Interlab Study
Team
Requirements
Medals
Safety
Collagen/Elastin/rAIP · Benchling
Collagen/Elastin/rAIP
Made with Benchling
Project: iGEM 2016
Authors: Anna Le, Charles Gleason
Dates: 2016-06-27 to 2016-10-15
Monday, 6/27
-Performed a Gibson for rAIP and elastin fragments, followed standard protocol and used the same timing for both (5 minutes, 15 minutes, 5 minutes) and transformed the result into NEB5α cells and plated
Tuesday, 6/28
-Made liquid cultures for 4 colonies from the elastin and rAIP 4X dilution plates. 40X plates had no useful colonies for both, even after they were left for the morning in the incubator. rAIP plate is relatively streaky, so the 40X plate might have produced replacement rAIP colonies
-Performed colony PCR, ran a gel, looked confusing under UV and technical difficulties prevented a good scan, so gel was saved for the next day with the hopes of more success visualizing it
Wednesday, 6/29
-Tried a gel extraction on the presumed location of amplified DNA from colony 3;  NanoDrop showed no DNA so the extraction product was discarded.
- The elastin 4X dilution plate from the day before was left out on the lab bench and appears to have bacteria expressing RFP contaminating it? Weird.
-Miniprepped and made new dilutions of the 4 liquid cultures from the day before
-Nanodropped the miniprep product and determined the high DNA concentration  and purity in c1 for elastin and c4 for rAIP
-Transformed c1 and c4 DNA into T7 cells for eventual protein expression
-Ran PCR on the c1/c4 miniprep for gel extraction and hopefully sequencing tomorrow
Thursday, 6/30
- Ran colony PCR on 2 colonies from elastin 40x and and 2 from rAIP plates
Friday, 7/1
-Found out the T7 liquid cultures weren't inoculated
-Prepared 5-alpha elastin and rAIP for sequencing
Tuesday, 7/5
- Checked protein expression in elastin circular mRNA expressing T7 cells; took 1 mL of each colony's 50 mL liquid culture preinduction after 2 hours of growth at 37°C  and used 20 μL to measure OD600 at 100x dilution (colony 1 corresponded to 0.5 and colony 2 to 0.4). Induced remaining 49 mL with 49 μL of 1000x IPTG solution (40 mg/mL) and waited 2.5 hrs. In the meantime, centrifuged remaining 980 μL of the pre-induction samples, removed supernatant, and added 50 μL CellLytic buffer to pellet. Resuspended and froze to perform extraction for comparison together. After 2.5 hrs, tested 20 μL of each culture; found both to be 0.6. Used 833 μL of colony 1 and 667 μL of colony 2 to get equivalent cell concentration, centrifuged, added CellLytic, and performed small-scale protein extraction on both. Additionally solubilized the insoluble fraction. In addition, anticipating existence of product, performed same procedures at large scale for the remaining 48 mL of each culture. Froze the soluble and insoluble fractions.
Wednesday, 7/6
- Lysyl oxidase plate showed growth, ran colony PCR and gel electrophoresis to see band sizes (CG/EL/AL)
- Made liquid cultures
-Ran SDS-PAGE on 10 μL samples from the small scale extraction products for comparison. Used 20 μL of the ~40 μL made for each so as not to overfill the wells, and put in 10 μL of ladder on either side. Stained with CYPRO Ruby, technology prevented rapid staining procedure from being effective so the gel was stored in stain overnight with shaking.
Thursday, 7/7
- Lysyl oxidase band sizes were approximately 1.5kb-2kb which is what we anticipated (CG/EL/AL)
- Made 2 cryostocks from 2 colonies on lysyl oxidase plate labeled "NEB5-alpha rAIP Chlor AL 7/7/16"
- Performed gel extraction on the lysyl oxidase for sequencing
-Technology continues to be a struggle, but:
(Charlie plz insert picture)
Friday, 7/8
- Sequencing looks rather accurate
- Transformed rAIP into T7 cells at 4x and 40x dilution, plated and left in room temperature for the weekend. (CG/EL/AL)
Monday, 7/11
- Plate for rAIP transformation from 7/7 did not work. No growth present
- Transformed rAIP plates. Used 100 uL without dilution when plating.
Tuesday, 7/12
rAIP plates worked; made two 5 mL liquid cultures
●
Redid Gibson for elastin; fragment 2 probably didn't actually add
Wednesday, 7/13
rAIP
●
Ran PAGE gel with Lumio Green detecon to verify protein product; colony 1 looked better
●
Made a ~400 mL culture of colony 1 for large scale protein extraction
Elastin
●
Transform
Thursday, 7/14
●
Spun down the culture in 2 sections; got 1 gram of wet mass of cells; performed large scale extraction procedure and solubilized the insoluble fraction
●
Made a cryostock
●
Miniprepped 5 mL of culture
●
Tried transforming onto 3x Chlor plates due to mistakes, saved some liquid culture in anticipation of failure
Friday, 7/15
●
3x plates failed at all 3 possible dilutions, Eric made a 1x plate with 100 μL of undiluted culture
Monday, 7/18
rAIP
●
Full density culture made a lawn pretty much everywhere; took what was hopefully a single colony on the edge and inoculated a liquid culture with it for the day, after 5 hours in a 1.5 mL tube this was used to replate
Tuesday, 7/19
rAIP
●
Transformation plates worked; we put them in the fridge
Thursday, 7/21
Elastin
●
Ran PCR on the two fragments and the plasmid backbone with Q5 (pSB1C3 for 2:15 elongation at 64.9°C, Elastin fragment 1 and 2 in the same thermal cycler for 1 minute extension with fragment 1 at 71.5°C and fragment 2 at 72.0°C
Friday, 7/22
Elastin
●
Ran a recycled 1% gel of the PCR products from yesterday
●
Gibson with 1 μL of pSB1C3, 1 μL of fragment 1, and 1.5 μL of fragment 2 for 10 minutes at 52°C and 50 minutes at 50°C trying to account for potential hairpin at fragment 1/2 overlap site
Monday, 7/25
Elastin
●
Picked 5 colonies from 7/22 transformation. Grew liquid cultures from these 5 colonies for 5 hours and ran PCR.
Tuesday, 7/26
Elastin
●
Ran gel on PCR, performed PCR cleanup
●
Miniprepped the 5 colonies for plasmids.
●
Continued liquid cultures of the 5 colonies.
Wednesday, 7/27
Elastin
●
Sent 10 samples in for sequencing: miniprepped elastin plasmid samples 1-3, and Gibson PCR products 4 and 5
Thursday, 7/28
Elastin
●
Ran SDS PAGE gel to determine if elastin polymer was expressed.
●
Stained with SYPRO Red protein stain overnight
Friday, 7/29
Tuesday, 8/2
Elastin
●
Performed gel extraction/gel clean up on the elastin fragment 1. Band on the gel was between 600-700 bp. Concentration was 13.6 ng/uL, purity 260/280 2.16.
○
elastin fragment 1 most likely ligated incorrectly originally in the previous attempt, off by 1 bp.
●
Performed Gibson assembly and transformed T7 cells (overall 6th try).
●
Made a 40x plate using elastin colony #5 liquid culture.
rAIP
●
Induced reaction between purified protein samples with racemic lysine. H2O2 is a side product of the reaction between lysine and lysyl oxidase. We want to characterize the performance of lysyl oxidase.
●
Performed the Amplex Red hydrogen peroxide assay on protein purification fractions. We had a standard curve of H2O2. Initial concentrations were 0, 1, 2, 3... etc to 10 uM H2O2 (before adding Amplex red working solution).
●
We predicted that the Elution buffer fraction would contain lysyl oxidase. The plate data showed that this prediction was correct, because the other fractions displayed less H2O2 activity.
○
Plate reader data saved under file name: rAIP_h2o2assay
○
Forgot to take picture of plate
○
Stock solutions of horseradish peroxidase (HRP) are in the -20C freezer (Elastin/collagen box).
○
All remaining solutions (purified protein samples, 1X Rxn buffer, 1M lysine) left in the fridge.
H2O2 working concentrations (total volume = 50 uL)
A
B
C
1
Concentration
20 uM H2O2 stock
1X Rxn buffer
2
0
0
50
3
1
2.5
47.5
4
2
5
45
5
3
7.5
42.5
6
4
10
40
7
5
12.5
37.5
8
6
15
35
9
7
17.5
32.5
10
8
20
30
11
9
22.5
27.5
12
10
25
25
Table1
Wednesday, 8/3
Elastin
●
40X colony 5 plate was a lawn. Had to replate using 1:1000 dilution.
●
Gibson assembly/transformation worked. Plate showed ~7 colonies. woo
Monday, 8/8
●
Elastin colonies #8-16 sent out for sequencing after performing colony pcr and gel electrophoresis
●
Plated collagen (c1-3, CN) 1x 100x 1000x, made liquid cultures
●
Collagen/collagen sap1 digested fragments had pcr clean up and nanodrop
●
Collagen construct plated showed growth. 5 colonies were picked from 3X plates.
○
performed colony pcr
○
nanodropped to show concentrations 300-500 ng/uL
○
ran gel electrophoresis with 2 log DNA ladder
Wednesday, 8/10
Collagen
●
Grew large culture (250 mL) of CG's collagen construct 2.2 colony to use for large scale extraction.
●
Performed colony PCR + PCR cleanup + ran gel on 24 collagen colonies. Colonies were picked from gel
Thursday, 8/11
Elastin
Collagen
●
Performed protein large scale extraction on a 250 mL liquid culture of th collagen construct 2.2 (CG's weekend plate 2 colony #2). Soluble fraction was put into fridge. We plan on performing protein purification with nickel spin columns with the extraction sample.
○
For 2 g of wet cell pellet: .4 mL DNAse, 40 uL RNAse, 200 uL lysozyme
●
Sent out 24 collagen colonies for sequencing: C1 (all colonies/samples denoted as 1.1, 1.2, 1.3.. etc.), C2, C3, CN, S1, S2, S3, SN
rAIP
●
Did another H2O2 assay on lysyl oxidase. Standard curve was not accurate because there was probably a pipetting/dilution issue/all concentrations in the curve showed max activity w peroxidase
Monday, 8/15
A
B
1
C1
C1.1
2
C2
C2.2
3
C3
C3.3
4
CN
CN.1
Table2
Wednesday, 8/17
Collagen
●
Forgot to induce the large C1/C3/CN cultures with IPTG in the morning. Induced them and left them on a RT shaker with IPTG in the PM.
●
Made new cultures from the lawns that are collagen plates. Picked one colony from each construct (C1/2/3/N)
Elastin
●
Plated elastin colonies 14, 15, 16 on LB chlor.
Thursday, 8/18
Collagen
●
Large scale extraction
○
Spun down the 250 mL C1.1 and C3.3 cultures and 500 mL CN.1 culture to get the cell pellet.
○
Froze cell pellet in the -20 C in 15 mL falcon tubes. Spun down multiple times, bc didn't see insoluble fraction precipitate out. Would ***not*** recommend centrifuging > 10k rpm in large centrifuge. Tubes were slightly warped.
○
Wet cell pellet yields
■
1 = 1.411 g
■
3 = 0.932 g
■
N= 1.902 g
A
B
C
D
1
CellLytic
Lysozyme (1 mg /mL)
Benzonate (250 u/mL)
2
C1.1
14 mL
1.4 mL
14 uL
3
C3.3
9 mL
.9 mL
9 uL
4
CN.1
19 mL
1.9 mL
1.9 uL
Table3
●
Golden Gate assembly
○
Used the miniprep of C1.1 as the linear backbone
○
PCR'ed and PCR cleanup of the C2 and C3 constructs
○
Plated using T7
●
Plated collagen from the inoculated culltures from 8/17
Golden Gate protocol from JBEI
1) Measure the DNA concentration (ng/µl) of each assembly piece.
2) Add 100 ng of the linearized vector backbone and equimolar amounts of the other assembly pieces to a 15 µl total volume assembly reaction mixture as follows:
●
+ linearized vector backbone (100 ng)
●
+ each additional assembly piece (to equimolar with backbone)
●
+ 1.5 µl 10X NEB T4 Buffer
●
+ 0.15 µl 100X BSA*
●
+ 1 µl BsaI
●
+ 1 µl NEB T4 Ligase, 2 million cohesive end units / mL
●
+ dH20
●
TOTAL 5 µl
○
NOTE: It is essential to use a High Concentration Ligase *
○
BsaI is only 10% active at 37 C without the addition of BSA.
3) Perform the assembly reaction in a thermocycler as follows:
either (following Engler 2009):
3 min @ 37 C
} 4 min @ 16 C
} 25 cycles 5 min @ 50 C
} 5 min @ 80 C } 1 cycle
or, alternatively (modified from Engler 2008)
1 hour @ 37 C 1 cycle
5 min @ 50 C
} 5 min @ 80 C  1 cycle
NOTE: If any of the assembly pieces contain an internal BsaI site(s), it would be first preferable to silence the internal BsaI site(s) through point mutation(s); a second option, if the internal BsaI site overhang(s) are not cohesive with the other assembly overhangs, is to adjust the thermocycling parameters to terminate after a ligation step (e.g. skip the final cycle at 50 and then 80 C).
OR According to NEB, can incubate at 37 C for 1 hr and incubate at 55 for 5 min.
4) Transform 5 µl of the assembly reaction into 100 µl of competent E. coli and/or run a diagnostic agarose gel to check for successful assembly.
Elastin
●
Plates from yesterday grew ok. 8 was nearly a lawn and 14/15 had many isolated colonies. Left them in the fridge.
●
Ran PAGE gel for post-48 hour induction of elastin colonies 8, 14, and 15. Previously we ran a PAGE on post-2 hour induction but we didn't see the predicted size. Based on Cynthia's experiences with her PAGEs, we may just have to induce for a longer period of time to have more protein produced.
PABA
1 - c1 pabB
2- c1 pabC
3 - c2 pabB
4 - c2 pabC
PCR genes from E. coli for pabB, pabC from selected colonies.
Friday, 8/19
Collagen
●
Golden Gate worked on both plates - almost too well. Had to replate - will leave in RT over the weekend
●
Protein purification
○
Washed nickel columns to begin. Yesterday we took the soluble fraction only because that's where we determined the collagenlike proteins would remain
○
Thursday, 8/25
Collagen
●
Golden Gate assembly #2 worked, picked 5 colonies total from each plates. Ran colony PCR and PCR cleanup. Ran a gel to determine the band sizes. #1-3 had the desired size, #4-5 was much smaller. Assembly probably did not work for #4 and 5.
●
Ran Native PAGE gel
Elastin
●
according to sequencing, #15 - our promising colony - has a deletion, causing a frame shift. We have to find another working colonies.
Friday, 8/26
Collagen
●
Sent out GG2 PCR products #1-3 for sequencing. VF2 + VR were the primers.
Monday, 8/29
BCA Protein assay
●
Used on purified protein extracts: collagen 1.1, 2.2, 3.3, N.1 and lysyl oxidase (rAIP)
●
Refer to uploaded protocol
●
Protein concentration for 1.1, 2.2, N.1 were too low (out of range) - we're going to do protein extraction again with an initial incubation step tomorrow. 3.3 was~12 mg/mL and rAIP was ~150 mg/mL!
A
B
C
1
2 mg/mL BSA (uL)
Working reaction (uL)
desired BSA concentration (ug/mL)
2
0
10
0
3
0.625
9.375
125
4
1.25
8.75
250
5
2.5
7.5
500
6
3.75
6.25
750
7
5
5
1000
8
7.5
2.5
1500
9
10
0
2000
Table4
Elastin
ADJADJADA
Collagen
DADJADJADJAJAD
Tuesday, 8/30
BCA Protein assay
●
Used on purified protein extracts: collagen 1.1, 2.2, 3.3, N.1 from a second protein extraction where cell pellets were frozen + incubated for 30 min step.
○
Refer to uploaded protocol
●
Protein concentration for C2 and N were too low (out of range and 0.329 respectively). C1 and C3 were 52 and 59 mg/mL respectively
Wednesday, 9/14
Collagen
●
Ran large scale protein extraction on collagen (C2 and CN).  Collagen cultures' cell pellets were frozen for ~1 week. Cell pellet yield for our 2L culture was 11.3g -- need to figure out what the culture was.
Cell pellet yield (from 3mL volume)
●
c2 = .279 g
●
cn = .5 g
A
B
C
D
1
CellLytic
Lysozyme (1 mg /mL)
Benzonate (250 u/mL)
2
C2
2.8 mL
.28 mL
2.8 uL
3
CN
5 mL
.5 mL
5 uL
Table5
Elastin
●
Selected colonies from plates. Grew liquid cultures in 37 incubator - selected 2 colonies from each plate (1-3). Miniprepped after 6 hrs.
●
Ran overnight PCR w/ VF2 and VR.
Thursday, 9/15
Collagen
●
GG Sap1 digest
○
5 uL T4 Ligase
○
2 uL T4 ligase buffer
○
1 uL BSA (from 1 ul 1:5)
○
1 uL Sap1
○
DNA
○
H2O
○
Tota vol 20 uL
Elastin
●
Sent out sequencing for elastin mutagenesis PCR products - use VF2 and VR (12 samples overall.)
○
Ran gel and PCR clean up
clipboard_2016-09-19_16:16:57.png
L->R: 2log ladder, elastin mut 1.1, 1.2, 2.1, 2.2, 3.1, 3.2
Elastin should be ~2kbut there are bands at 1.5k.
Friday, 9/16
Collagen
●
Started anti-flag magnetic bead extraction method of CN and C2 protein with Kosuke.
●
Performed Golden gate digestion/ligation with Sap1 for constructs C1, C2 and C3 to remove the elastin crosslink domains. Transformed into T7, plated with 4x dilution. Left S1, S2, S3 plates in RT over the weekend.
○
C# = collagen construct with xlink domain
○
S# = collagen construct that has been Sap1 digested (w/o xlink domain)
●
Ran BCA assay on nickel-method purified protein C2 and CN. COncentrations were negative (probably bc the proteins couldve been eluted in earlier washes. NEED TO SAVE ALL WASHES to run on a gel for confirmation!!!)
Elastin
●
Sequencing ended up being trash - but Jesica suggested gel extraction.
Monday, 9/19
Collagen
●
GG Sap1 plates are super robust - should have plated with a lower dilution.
●
Ran CPCR #1 - picked 2 isolated colonies from each construct using VF2 and VR. Ran gel.
clipboard_2016-09-19_17:06:26.png
●
L->R: 2 log ladder, S1.1, S1.2, S2.1, S2.2, S3.1, S3.2, 2 log ladder
○
The desired band size should be 893 bp (~900 bp), but the prominent bands in the gel are around 1.1-1.2 kbp..... Most likely the digestion was not successful.
●
Picked more colonies from each plate - 3 additional replicates per plate. RERan CPCR to check if there are successful digested plasmids. Ran gel.
clipboard_2016-09-20_10:50:38.png
●
L->R row 1: 2 log ladder, S1.3, S1.4, S1.5, S2.3, S2.4, S2.5, S3.3, S3.4, S3.5
●
L->R row 2: elastin 1.4, 2.3, 2.4, 3.3,  paba 3, paba 4, dahp synthase, psb1c3 backbone, 2log ladder, 2log ladder (2 uL)
Protein extraction
●
Magnetic antiflag extraction - finished procedure with Kosuke - will run Lumio page gel tomorrow along with the SN protein extraction.
●
Made 3 250 mL cultures to grow and induce C2, CN and SN. Will put in the 6 hr old 5 mL cultures of C2 and CN into the 250 mL cultures (+ place into shaker) and inoculate a new 5 mL culture for SN by the end of the day.
○
General procedure for induction
■
Grow 5 mL culture
■
Use 5 mL culture to inoculate 250 mL
■
Grow 250 mL culture overnight
■
Leave growing at RT for a few hours preinduction
■
To induce, add 1 IPTG:1000 culture.
■
Wait a couple of hours and spin down
Elastin
●
Ran gel extraction of the 9/15 elastin gel's 4th lane (2.1) band but when I nanodropped it the concentration was negative. Threw it out :(
●
Picked more colonies for CPCR, ran gel. (1 replicate for elastin mut 1, 2 replicates for elastin mut 2, 1 for elastin mut 3.) - Gel results are above - gel bands are totally small ~400-500bp. Charlie sent me the geneious files (most uptodate) - I realized that elastin is NOT actually 2k, it is actually 3.32kb!!! MUST REDO PCR
Todo List
Large scale protein extractions for frozen cell cultures (250 mL + 2L cultures)
FIND OUT WHAT THE 2L CULTURE WAS (full construct C = C1 + C2 + C3)
need to find enteropeptidase to remove the his-tag
Find concentration (BCA assay). Run native page if at least 50 mg/mL.
Find native protein ladder
Grow SN and do protein extraction.
Run Golden Gate style sap1 digest w/ linearized plasmids + transform into t7 for C1/2/3 ==> remove xlink domains to make S1/2/3.
Sequence.. etc
Run Golden Gate on successful/correct transformations (the S1/2/3's) using BSA1 + transform.
Elastin mutagenesis - sequence/run gel. Grow overnight cultures
BUST
Tuesday, 9/20
Elastin
Redid CPCR for elastin site directed mutagenesis colonies...... Changed the cycling time to be appropriate for 3.32kb -- used 100 secs, but gel bands still show 300 bp only. ---- Trevor! thinks its an empty plasmid, which is probably valid considering VF2->VR is ~300 bp as well.
clipboard_2016-09-20_14:43:26.png
Will 100% have to redo elastin site directed mutagenesis or troubleshoot this.
●
Took the elastin mut 2.1 MP tube because that one seemed promising in the gel run on 9/15.
●
CORRECTION  - PLASMID IS 3.3K. But PCR product should be ~1.2k lol bc takes out part from prefix->suffix only.
Collagen
Sent out GG Sap1 S# colonies for sequencing. Names on sequencing order not actually representative of the PCR colonies
1.
S1.1
2.
S1.2
3.
S2.1 -> S1.4
4.
S2.2 -> S1.5
5.
S3.1 -> S2.1
6.
S3.2 -> S2.3
2nd order for sequencing:
1.
S3.4 fwd
2.
S3.4 rev
3.
S3.5 fwd
4.
S3.5 rev
●
Induced SN at 4pm and left at 37C, spun down the culture at 730pm. Froze in a bottle.
●
Induced CN and C2 cultures at 7pm, but left them in the RT shaker overnight.
Wednesday, 9/21
Collagen
●
Sequencing for S3.4 and S3.5 show that elastin xlink domain has been cut out (for the most part), will use S3.5 for Golden Gate assembly of construct collagen S.
●
S1.1, S1.2, S1.4, S1.5, S2.1 - bust
●
S2.3 looks like elastin xlink got digested. Will use for GG for collagen S.
●
Sent out S2.2
●
Spun down CN and C2 cultures and froze.
Cell pellet yields
.5
A
B
C
D
1
CellLytic
Lysozyme (1 mg /mL)
Benzonate (250 u/mL)
2
C2
5 ml
1 ml
14 uL
3
CN
9 mL
.9 mL
9 uL
4
SN
19 mL
1.9 mL
1.9 uL
Table6
Elastin
●
Took PCR product  from yesterday and ran PCR cleanup and a gel. The bands look more promising, since it is not actually an empty plasmid.
○
Sent out for sequencing.
●
Did site directed mutagenesis
Thursday, 9/22
Wednesday, 10/12
AL: I sent in sequencing for biobrick sequence verification - needed to have reverse sequencing for S1 and S2.
Also resequenced SN and full construct C minipreps, just to make sure the tubes were what we labeled them as. Full construct C tube doesn't match with the Collagen with Crosslinking domain geneious file. It probably isn't what we want. SN had 100% pairwise alignment so its OK.
Ran PCR reactions to linearize constructs. The miniprep construct 1's (S1//C1) were linearized with linearization primers previously designed at the beginning of the summer. The miniprep constructs 2's and 3's were linearized with VF2 and VR primers.
●
These PCR's will be used for Golden Gate Assembly to make FULL COLLAGEN CONSTRUCTS. Full construct C, the miniprep we currently have, is not right so we have to redo that. Full construct S does not exist yet, so thats why we're running GG.
Thursday, 10/13
Ran PCR cleanups on the previous PCR reaction tubes and ran the Golden Gate Assembly with Bsa1-HF reaction in the thermocycler (protocol recorded in the protocols records). 2 reactions were done: C1+2+3 (to make full C construct)
●
Realized I used TBE instead of DI water (!!!!) while I was doing the transformation in NEB 5alpha. had to scrap the transformation.........
○
TBE stops dna editing enzymes from working
●
Reran GG assembly reaction, DNA tubes stored in freezer labeled S and C.
Friday, 10/14
Made LB-chlor plates because we were out. Theresa and I (as in AL) will probably need them next week.
Used GG assembly BSA1-HF reactions for full S and C and transformed in T7, plated on LB-chlor plates. Forgot to put in SOC before the 1 hr incubation period, did that after 30 min of incubation. Yikes. I kept the transformation tubes after plating in case transformation wasnt successful.
Because our elastin mRNA circularization method has failed multiple times, Charlie designed primers so that we could demonstrate circularization in collagen. We used CN with the "CN For circularization" primers (Ta = 59.6) to get the coding sequence for collagen (coiled coil homotrimerization domain and the bacterial collagen domain) and circularize it. We use elastin (e15) with the "partial backbone" primers in order to replicate only the elastin backbone of the plasmid in order to have a backbone for Gibson assembly.
●
Ran PCR reactions w/ these primers.
●
Didn't have Gibson assembly masterm mix at the base lab. Couldnt do transformation ughhhhhh
clipboard_2016-10-15_21:06:43.png
clipboard_2016-10-15_21:07:36.png
Saturday, 10/15
Transformation from yesterday didnt work! I replated the transformations for full S and full C again on different plates, and kept the old 10/14 plates in the 37C incubator still.
Used the GG Assembly reaction tubes to run another 2 set of transformations for full S and full C. I used NEB 5 alpha cells. In case the T7 transformations wouldn't work (because of that SOC incubation flaw), I wanted to make sure we would get colonies by tomorrow.
Began cross-linking experiment for collagen
We wanted to characterize crosslinked collagen and running that possibly in a page gel.
●
Used PQQ (Pyrroloquinoline quinone) diluted 1:100, copper sulfate, with eluted protein.
○
Protein is in elution buffer (contains phosphate buffer)
○
Copper sulfate is needed to
○
PQQ acts as catalyst
○
NaOH used to neutralize acidic reaction between phosphate and copper in order to maintain neutral pH
●
Used a pH meter to monitor the solution pHs
●
Tested experiment on CN only so far
○
Left tube is pH 8, we can see that there is precipitate which is not what we want!! Precipitate must be copper hydroxide
○
Right tube is pH 6.8 ~ 7, blue color is probably due to copper phosphate
●
Hope to run a PAGE tomorrow and hope to see different bands in comparison to uncrosslinked CN
clipboard_2016-10-15_22:21:16.png
