Team:UChile OpenBio-CeBiB/Parts
Document
Project
Description Parts Procedures Results Attributions
HP
HP Overview Collaborations Silver Gold Engagement
Modelling
Modelling Overview Procedure Results
Safety
Biosafety Lab Protocols
Notebook(current)
Team(current)
Giant Jamboree
Our experience Judging Feedback Photos
Home(current)
Parts
FBP/SBP-851: BBa_K2425000
Open reading frame for Fructose 1,6/sedoheptulose 1,7 bisphosphatase from cyanobacteria Synechococcus sp. (WH8103 strain ) with a 32-amino acid (96 bp) N-terminal signal peptide for chloroplast destination and codon optimized for expression in Chlamydomonas reinhardtii. This protein is involved in the Calvin cycle, which is part of the carbohydrate biosynthesis pathway. The catalytic activities are:
D-fructose 1,6-bisphosphate + H2O = D-fructose 6-phosphate + phosphate
Sedoheptulose 1,7-bisphosphate + H2O = sedoheptulose 7-phosphate + phosphate.
This protein has been expressed in chloroplasts of tobacco plant cells using the same chloroplast signal peptide and demonstrated to improve carbon fixation and growth (Miyagawa et al., Nature Biotechnology 19(10):965-9, 2001). Transgenic tobacco plants showed enhanced photosynthetic efficiency and growth characteristics. Compared with wild-type tobacco, final dry matter and photosynthetic CO2 fixation of the transgenic plants were 1.5-fold and 1.24-fold higher, respectively. Levels of intermediates in the Calvin cycle and accumulation of carbohydrates were also higher than those in wild-type plants. The signal peptide corresponds to the 32 amino acid chloroplast transit sequence of the ribulose bisphosphate carboxylase/oxygenase activase preprotein from Chlamydomonas reinhardtii, required for translocation through the envelope of the chloroplast (Krimm et al., Eur. J. Biochem. 265:171-180,1999).
Figure I: Scheme of the BBa_K2425000 construct. It is important to know that the number of  bases pairs is just the coding sequence.
This sequence is crucial in the design of Greenhardtii Project, since it has the theoretical function to generate an indirect overexpression of Rubisco activase, who activate RuBisCO and in higher levels (not induced by artificial methods) can produce higher levels in the activity of RuBisCO and through this way, have an optimization of Calvin Cycle (Miyagawa et al., Nature Biotechnology 19(10):965-9, 2001) but the mechanism is not well known. This is the first time that this assay is made and adapted for Chlamydomonas reinhardtii and it has a critical importance for the industrial use of this species.
Sequence: http://parts.igem.org/Part:BBa_K2425000
Another Biobrick designed (Not in the iGEM registry):
We have designed another biobrick for the competence but unfortunately we could not send by the deadline. So we describe here the METEp-850 sequence.  This code corresponds to the sequence of the adjustable promoter of the METE gene, which codes for Cobalamin-independent methionine synthase protein. This promoter is regulated by B12 vitamin which has shown the capacity to reduce the transcriptional levels of METE gen (Helliwell et al.,Plant physiology, 165(1), 388-397, 2014). The construct has 574 bp and contains the 5’ UTR and the sequence of the promoter. All the sequences related to METE promoter has been kindly shared by Dra. Alison Smith from the Departament of Plant Sciences of the University of Cambridge. The objective for this sequence is make deeper the characterization of this promoter using our model and at the same time gather data to use this promoter in the production of recombinant protein in the optimized strain of Chlamydomonas reinhardtii. We think that if Chlamydomonas reinhardtii is though like a production platform is very attractive the idea of have the control of its behavior.
Figure II: Scheme of the METEp 850 construct. It is important to know that the number of bases pair is just the coding sequence.
The sequence for the construct is (including suffix and prefix for biobricks):
5’GAATTCGCGGCCGCTTCTAGAGGTAGGTCAGGACCAGAGCCTACAACATTGTAAGGCCATGTATCCTCCAGTGCCATTTGCGACGT
TACACGACGAGACAATCTGCGCTGTGTTCAAGCACACACATCCGGGACTCGTGTGATGTCTTAACATCTGCAAGGAAACATAAGGT
GGCCGGCGTCACGCAAGATGACGCGACGGACTCGAATGCTTGCTTCGTCCAGCGACAAATAAGGACAGGCAGGGCGCCTGCATT
GCGTGCGCTTAGAGGCGAGGCGCTCAAGACATTTCGGCAGCAATAATTGGTATTGAGGCACATTCTGCACCAGGATGCCAAGAGG
TGAACGTTGCTGCCTTAATGTATATCTGCACAGCTGGCCGGTTACTGCGAACCGGGTGCCTTTTGGAGTCGCTCCTAAAAGACGTCA
CCGGCGCAACGTCGTCGTGCGGCCGTACAGGTATGCAAAATTGGCCCGGTTGCGCGCAGGCCGATTATTTAAGTGACATTTGGTAC
CATGCATGAGCATCCGATTGTTGTTTTGCAATTCTGTCTGACTGCACAAGGAACCCGTCCTTGGGAATCAACATCTGTCAACTACTAGT
AGCGGCCGCTGCAG ‘3
Code:
Red = EcoRI
White=NotI
Aqua = XbaI
Pink= PstlI
Blue= SpeI
Purple=NdeI
Construct designed to use in binary vectors for assays in alga (Not Biobricks):
For the assays in algae we have decided to use the binary vector pBC1 which is shown in the Figure III. In this vector, we have ligated three diferents constructs. For all the next constructs a codon bias for Chlamydomonas reinhardtii has been made.
Figure III: Shows pBC1-CrGFP vector.
a) Seq 2_2 (1890 bp):
Open reading frame for fusion mCherry-fructose 1,6/sedoheptulose 1,7 bisphosphatase from cyanobacteria Synechococcus sp. (strain WH8103) with a 32-amino acid (96 bp) N-terminal signal peptide for chloroplast destination. This construct replaces the CrGFP site of the vector, and by this way is under the control of the constitutive promoter psaD and have its 3’ UTR TpsaD. A 8 nucleotide linker has been put to allow the right folding of the two proteins.  The code for this sequence is the next:
5’CATATGATGCAGGTGACGATGAAGTCCTCGGCGGTGTCGGGTCAGCGCGTGGGTGGTGCCCGGGTGGCTACCCGCAGCGTGCGCCG
CGCCCAGCTGCAGGTGGTGTCCAAGGGTGAGGAGCTGTTTACGGGCGTGGTGCCGATTCTGGTCGAGCTGGACGGCGACGTGAACGG
TCATAAGTTCTCGGTCAGCGGTGAGGGTGAGGGTGACGCCACCTACGGCAAGCTGACCCTCAAGTTTATTTGCACCACGGGCAAGCT
CCCGGTGCCGTGGCCCACCCTGGTGACCACGCTCACGTACGGTGTCCAGTGCTTTTCCCGCTACCCTGACCACATGAAGCAGCACGA
CTTTTTCAAGTCCGCTATGCCTGAGGGTTACGTGCAGGAGCGGACCATTTTCTTCAAGGACGACGGCAACTACAAGACGCGGGCTGA
GGTCAAGTTCGAGGGTGATACGCTCGTGAACCGGATCGAGCTGAAGGGTATCGACTTTAAGGAGGATGGTAACATCCTGGGTCACAA
GCTCGAGTACAACTACAACTCGCATAACGTGTACATTATGGCCGACAAGCAGAAGAACGGCATCAAGGTGAACTTCAAGATTCGGCA
CAACATTGAGGACGGTTCCGTCCAGCTGGCCGATCATTACCAGCAGAACACCCCTATTGGTGATGGCCCCGTCCTGCTCCCCGATAA
CCATTACCTCAGCACGCAGTCGGCCCTGTCGAAGGATCCTAACGAGAAGCGCGACCACATGGTCCTCCTCGAGTTCGTCACCGCGGC
GGGTATCACCCTCGGCATGGATGAGCTCTACAAGGGTGGTAGCGGCGGCTCGGGGGGCTCCGGGGGCGAGAAGACCATCGGTCTCGA
GATTATCGAGGTGGTGGAGCAGGCGGCTATTGCCTCGGCCCGCCTCATGGGTAAGGGGGAGAAGAACGAGGCTGATCGGGTCGCCGT
CGAGGCTATGCGCGTGCGGATGAACCAGGTGGAGATGCTCGGGCGCATCGTCATTGGGGAGGGGGAGCGCGATGAGGCCCCGATGCT
GTACATTGGGGAGGAGGTGGGGATCTACCGCGACGCGGACAAGCGGGCCGGGGTCCCGGCGGGTAAGCTCGTCGAGATCGACATTGC
TGTGGACCCCTGCGAGGGGACCAACCTGTGCGCTTACGGCCAGCCTGGTTCCATGGCTGTGCTCGCGATCAGCGAGAAGGGTGGGCT
GTTCGCCGCTCCTGACTTTTACATGAAGAAGCTCGCGGCGCCGCCGGCTGCCAAGGGCAAGGTCGATATTAACAAGAGCGCGACGGA
GAACCTCAAGATTCTGAGCGAGTGCCTCGATCGGGCGATTGATGAGCTGGTCGTGGTCGTGATGGATCGCCCGCGGCACAAGGAGCT
GATCCAGGAGATCCGGCAGGCTGGCGCTCGCGTGCGCCTCATCTCCGACGGTGACGTCTCCGCCGCTATCTCGTGCGGTTTTGCTGG
CACGAACACGCATGCGCTGATGGGTATTGGTGCTGCGCCGGAGGGGGTGATTTCCGCCGCGGCTATGCGGTGCCTGGGCGGTCATTT
CCAGGGTCAGCTGATTTACGACCCGGAGGTCGTCAAGACGGGGCTCATCGGGGAGTCGCGCGAGTCCAACATTGCCCGCCTCCAGGA
GATGGGCATTACGGATCCCGACCGCGTGTACGATGCGAACGAGCTGGCGAGCGGCCAGGAGGTGCTGTTCGCTGCCTGCGGGATCAC
GCCCGGTCTGCTCATGGAGGGTGTGCGGTTCTTTAAGGGGGGCGCCCGGACGCAGTCCCTCGTGATTAGCTCCCAGAGCCGCACGGC
TCGCTTTGTGGATACGGTGCACATGTTTGATGACGTCAAGACCGTCTCCCTCCGCTAAGAATTC-3’
b) Seq 4 (1357 bp):
Construct composed by METE promoter, an open reading frame for RFP and 3’ UTR of METE promoter. This construct has the objective to serve like a proof of concept for the production of recombinant protein under the control of this promoter. Also, by this way, we can gather data for the model of our project and make deeper the kinetic characterization of METE promoter. It is a good test for in the future, with the optimized strain of Chlamydomonas reinhardtii try to generate proteins or metabolites that could help to solved the problems of communities like Puchuncavi (data shown in HP section). The sequences is the next one:
5’GCGGCCGCGTAGGTCAGGACCAGAGCCTACAACATTGTAAGGCCATGTATCCTCCAGTGCCATTTGCGACGTTACACGACGAGACA
ATCTGCGCTGTGTTCAAGCACACACATCCGGGACTCGTGTGATGTCTTAACATCTGCAAGGAAACATAAGGTGGCCGGCGTCACGCAA
GATGACGCGACGGACTCGAATGCTTGCTTCGTCCAGCGACAAATAAGGACAGGCAGGGCGCCTGCATTGCGTGCGCTTAGAGGCGAGG
CGCTCAAGACATTTCGGCAGCAATAATTGGTATTGAGGCACATTCTGCACCAGGATGCCAAGAGGTGAACGTTGCTGCCTTAATGTAT
ATCTGCACAGCTGGCCGGTTACTGCGAACCGGGTGCCTTTTGGAGTCGCTCCTAAAAGACGTCACCGGCGCAACGTCGTCGTGCGGCC
GTACAGGTATGCAAAATTGGCCCGGTTGCGCGCAGGCCGATTATTTAAGTGACATTTGGTACCATGCATGAGCATCCGATTGTTGTTT
TGCAATTCTGTCTGACTGCACAAGGAACCCGTCCTTGGGAATCAACATCTGTCAACATGGTGTCCAAGGGGGAGGAGGATAACATGGC
GATCATCAAGGAGTTCATGCGGTTTAAGGTCCACATGGAGGGCTCGGTCAACGGGCATGAGTTCGAGATCGAGGGCGAGGGCGAGGGT
CGGCCTTACGAGGGCACGCAGACGGCGAAGCTCAAGGTCACCAAGGGGGGCCCTCTCCCTTTTGCGTGGGACATTCTGAGCCCCCAGT
TCATGTACGGCTCCAAGGCTTACGTGAAGCATCCGGCTGATATCCCCGATTACCTCAAGCTGAGCTTTCCGGAGGGTTTCAAGTGGGA
GCGCGTGATGAACTTCGAGGATGGTGGTGTCGTGACCGTGACCCAGGATAGCTCCCTCCAGGATGGCGAGTTCATTTACAAGGTGAAG
CTCCGCGGCACGAACTTCCCCAGCGATGGGCCTGTCATGCAGAAGAAGACGATGGGTTGGGAGGCGTCCAGCGAGCGGATGTACCCCG
AGGACGGGGCGCTCAAGGGGGAGATTAAGCAGCGGCTCAAGCTCAAGGACGGCGGGCATTACGACGCTGAGGTCAAGACCACGTACAA
GGCCAAGAAGCCTGTGCAGCTCCCCGGTGCTTACAACGTCAACATCAAGCTGGACATCACCAGCCACAACGAGGATTACACCATTGTC
GAGCAGTACGAGCGCGCGGAGGGTCGGCATTCCACGGGCGGGATGGATGAGCTGTACAAGTAAAATGCCCGAATGTTGGGTATCTAGC
TCACACGGCAGTTTGTAAGGTGCTGAGGCGTTGCTCGAG-3
c) Seq 1_2: Same sequence of the BBa_K2425000 but without prefix and suffix for Biobricks
Figure IV:  Shows construc Seq 2_2 (up) and Seq 4 (down).
We also optimized a previous part (BBa_K515100, from iGEM11_Imperial_College_London (2011) for its use in C.  reinhardtii.
AAT TTT GTC AAG ATC ATT CTG CTG ACC ACG AGC TAC TGA CGG TGA TAC AAC CTC AAC TTT TTT TAG CAG AAG TGG GCT CGC GTG GTC CAG TAA ATG AGCGCC TGC GGC CCG GTC GCT CTC ACG ACG AAC TCG CCG GAG GGT ATT TAC GTG TGG ACG GGT TTC CCG GTG TGC ATT TAG TGA CTG GAC CGC ACC TCG TAACGG ACC TGG GTG AAC TCC GGG CGG AAG TCG CGG CGG TGA GGT TGC AAC TAC CGG GGT CGG CAT TGC CGG CTG TCC TCG AGC ATC TAA ATC AAC GAG GACGGC AGC GAG ACC TGC AGC CTC TAG GGT CTC CAG AAC GGC CGC TCC CTG ACG GTG ACG GGG ATC TGA CGG TCG GAG GGG AAC TGC TGC GGT ACG GGC CGGTAC GCC CTG AGC TCC CTG CTG CAC TCG GTG CTG TCC CTC CGC TGA TGA ACC GGT CCC TAG AAG TCG TCC GTG CCC TGA CCC TTT GAC GGC GGC CAG TGGCAG CAC GGC GAC TAA CCC GGT CGG TAC CAC GTC CTC TGC ACG GAC GTC ATC CGC TCG CCG TCC CCT GTC CCT CGG TCG GGG GGC TGC ATG GGT CGG CGGCTC CGC ATC CGG CTG AGC ATC TGG CGC TAC TCC ACG GGT CAC AGC TAG TTC TAA CGG ACC GCG AGC CAG GGC ACC GTG GAG CAG ACC CGC GCG TTT GTCGGG CGC AGC CAC TTC CTC TAA CTC TGC AGC GAT CTC AAG GGT CTC TGC GAG ATC AAG CTC CTC CCG TCG TAG GGG TTC CGG ACG AGC TGG TTC TGG TACCGC GGT GTG GGG TTT TAA TTC TCC AAG TAG TAC GTC CGG AAC CTG TCC TGC TGC GAC GAT GAG CTC TGA TGA GCC CCG GCC AGC GAC TGC TGG TGG CGGACC GCG TCG GCG TGC CGG CCT GTC GTG GCC CGG GCC TAG ACC CTG CGG ACC CTG GCG AAG GGC TAC GTC AGC TTT CTC ACC AGC TCG GGC TGC CCC GCGCCC CGC TGC CAG ACG TAC TGC ACC TGC TAA CGG TGG TTC ATC TAC GGT TAC CGG TAA CTG GGC CGG TAC ACG TCG ATT CGC TGC CGG TTT GAT AAC CTCTCC ATT GTC GCC GCG TAC AAC GCG GAT TAA ATG TAG GGC ATC TTC GTC TTT ACC GAG GAC GTC GAC GGT GCG GGC AGC CAT GCT CTG CAC GCT GTG TTTTAA AAC TTT TGC GAC GGC TAA AGC GCT GTG CTG GAG GGG CAG CGG TCG GGG AAC GGG TCC CGG TCG TAC GTG CAC GAC CCC CAT CGC AGC CTC AAC GCGCGC CAC ATT CTC ATC TAA TAA TGG CGG CGC CAG GCC CGC CGC TAC CTG CCT GAG CTC CTG ATG GAT GAG TGA CGC TTC AAG AAC GTC ACC GCT ACC GACCGC GAG ACC CGC CAG ATT AGC TTT CGG TGC CCG TAA AAG GAT CTC CCG GAG TCC GGG TAC CGC CGG ACG TAC TAC TGG TAA CCC GAC TAC TGC GAG CTCGGG TCG GGT AGC GCC TTT CCG GGG GGT ATC TAG CGG CGC GTG ACC GGC CCC CTG CCT CTC CAG CCT GCT CAC GTC CGC GCG TTT TAC GCG GCC GGC CACGCT GTG CGC ACC GCC TGG CAT GTG TAC TGC CGG TGA TAG TGC ATC CTC GAC ACG GGT ATG GGT TAA GGC AGC TCC ACC AAC GTC ATT GAG CGG GGC GTCGGT CAT CAT GAG CCT TTC CGG CGC GAG AAC CCT CGC CGC GAG TCC GGC TCG TGG CGG CGG TTC CCT TGA GAC TGG ACG GAT CGC GCT GGT GGC CTG ATCATT GCC GGG ACC CCG GCC CCT CGG AAG GGC GGG CTG AAG ATG CGC GAG ATG ATC ACC CTG GAG AGC CTC TGC CGG GCG CTC GCC GAC GAG CAG ATT GCTGCT GAG GAG CTC CGG GAG CGC GCT CTC GAT ACC GAG GCC CGG CTC ACC CTG CTG AAC TGC TTT ATT CGG GAG GGG GAT GCC GTG TCG CAG TTC GGT GAGGCC GAC CAG GCG CGG AAG GGG ACG AGC CTC TGG GGT GTG CCG GTC AGC TTT AAG GAC AAC ATC TGC GTC CGG GGT CTC CCC CTG ACC GCC GGC ACC CGGGGG ATG AGC GGT TTT ATC GCC GAC CAG GAT GCG GCC ATT GTG AGC CAG CTG AAG GCG CTG GGC GCT GTG GTC GCT GGG AAG AAC AAC ATG CAC GAG CTGTCG TTT GGG GTG ACG TCG ATT AAC CCC CAC TGG GGT GCG GTC GGT AAC CCT GTC GCC CCT GGG TAC TGC GCG GGT GGT AGC TCC GGT GGG AGC GCTGCCGCT GTC GCT TCC GGG ATT GTG CCG CTG AGC GTC GGC ACC GAC ACG GGC GGC TCG ATC CGG ATT CCG GCT GCT TTC TGC GGC ATT ACC GGT TTC CGC CCTACC ACG GGG CGG CTG TCG ACC GCC GGC ATC ATT CCC GTC TCC CAC ACC AAG GAC TGC GTC GGT CTG CTG ACC CGC ACG GCC GGT GAT GCC GAG TTT GTGTAC GGG CTC CTC TCC GGC AAG CAG CAG AGC TTC CCC CTG AAC CGC ACG GGC CCC TGC CGC ATC GGC CTC CCG GTC TCG ATG TGG TCC GAC CTG GAT GGGGAG GTG GAG CGG GCG TGC ATC AAC GCT CTC AGC CTG CTC CGG AAG ACG GGG TTC GAG TTT GTC GAG ATC GAC GAC GCT GAT ATT GTG GAG CTC AAC CAGACC CTC ACC TTT ACG GTC CCG CTC TAC GAG TTC TTT GCG GAC TTC GCG CAG TCC CTG CTC TCC CTG GGT TGG AAG CAT GGT ATT CAT CAT ATT TTC GCTCAG GTC GAT GAT GCT AAC GTG AAG GGT ATC ATC AAC CAC CAT CTC GGT GAG GGT GCT ATC AAG CCC GCT CAC TAC CTC AGC TCC CTC CAG AAC GGG GAGCTC CTC AAG CGC AAG ATG GAT GAG CTG TTC GCC CGC CAC CAC ATC AAG CTG CTC GGC TAC CCT ACG GTG CCT TGC CGC GTG CCG CAT CTG GAT CAT GCGGAC CGG CCG GAG TTC TTT TCC CAG GCG ATC CGG AAC ACG GAC CTG GCC AGC AAC GCT ATG CTC CCG AGC ATT ACG ATT CCT GTC GGC CCC GAG GGT CGCCTG CCT GTC GGT CTG TCG TTC GAT GCG CCC CGC GCG CGC GAT GCC TTT CTG CTC TCG AAC GTC TCC CTC ATC GAG AAG GTC CTC AAG GGG TGA TAA
References
1)Miyagawa, Y., Tamoi, M., & Shigeoka, S. (2001). Overexpression of a cyanobacterial fructose-1,6-/sedoheptulose-1, 7-bisphosphatase in tobacco enhances photosynthesis and growth. Nature biotechnology 19(10), 965-969.
2)Krimm, I., Gans, P., Hernandez, J. F., Arlaud, G. J., & Lancelin, J. M. (1999). A coil–helix instead of a helix–coil motif can be induced in a chloroplast transit peptide from Chlamydomonas reinhardtii. The FEBS Journal, 265(1), 171-180.
3) Helliwell, K. E., Scaife, M. A., Sasso, S., Araujo, A. P. U., Purton, S., & Smith, A. G. (2014). Unraveling vitamin B12-responsive gene regulation in algae. Plant physiology, 165(1), 388-397.
