<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UChile_OpenBio-CeBiB_Plant skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UChile OpenBio-CeBiB/Plant</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Document</TITLE></P><DIV class="collapse navbar-collapse" id="navbarNavDropdown"><UL class="navbar-nav ml-auto"><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/Description" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
          Project
        </A><DIV class="dropdown-menu item2" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/Description">Description</A><A class="dropdown-item" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/Parts">Parts</A><A class="dropdown-item" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/Experiments">Procedures</A><A class="dropdown-item" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/Results">Results</A><A class="dropdown-item" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/Attributions">Attributions</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/HP" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
          HP
        </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/HP">HP Overview</A><A class="dropdown-item" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/Collaborations">Collaborations</A><A class="dropdown-item" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/HP/Silver">Silver</A><A class="dropdown-item" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/HP/Gold_Integrated">Gold</A><A class="dropdown-item" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/Engagement">Engagement</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/Model" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
          Modelling
        </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/Model">Modelling Overview</A><A class="dropdown-item" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/Model/Procedure">Procedure</A><A class="dropdown-item" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/Model/Results">Results</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/Safety" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
          Safety
        </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/Safety/Biosafety">Biosafety</A><A class="dropdown-item" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/Safety/LabProtocols">Lab Protocols</A></DIV></LI><LI class="nav-item"><A class="nav-link" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/Notebook">Notebook<SPAN class="sr-only">(current)</SPAN></A></LI><LI class="nav-item"><A class="nav-link" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/Team">Team<SPAN class="sr-only">(current)</SPAN></A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/Jamboree" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"> Giant Jamboree </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/Jamboree/OurExperience">Our experience</A><A class="dropdown-item" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/Jamboree/JudgingFeedback">Judging Feedback</A><A class="dropdown-item" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB/Jamboree/Photos">Photos</A></DIV></LI><LI class="nav-item"><A class="nav-link" href="https://2017.igem.org/Team:UChile_OpenBio-CeBiB">Home<SPAN class="sr-only">(current)</SPAN></A></LI></UL></DIV><DIV class="container" style="margin-top:150px;"><H1> Plant </H1><H3>Description</H3><P>The non-vascular <I>Chlamydomonas reinhardtii</I> is easily found in many environments like freshwater, saltwater or even in mountain snow. <I>Chlamydomonas reinhardtii</I> is characterized by being a model microorganism for the study of various physiological processes like photosynthesis, cell division, flagellar biogenesis (Funes et al, 2007) and lipid synthesis for biofuel (Siaut et al, 2011).</P><P>Among some of the characteristics of <I>Chlamydomonas reinhardtii</I> are to be a green microalgae, eukaryotes, haploids, biflagellated, of a single coupled chloroplast and with ability to sense light product from a stigma. Besides, they have the ability to grow both in heterotrophic and autotrophic conditions and they possess a short life cycle (close to 5 hours) Also, they can adopt an anaerobic metabolism, which generates ethanol as a secondary metabolite.
Given the great potential in research and apllication that <I>Chlamydomonas</I> present, diverse techniques related to its culture and transformation have been developed, highlighting in the latter associated methods to nuclear transformation such as transformation via <I>A. tumefaciens</I> or glass beads and chloroplast and mitochondrial transformation by biolistics. With the development of such techniques many uses were given to <I>Chlamydomonas reinhardtii</I>, among them in the biofuel industry (lipid synthesis); in the bioremediation industry where it has been used to extract heavy metals from water bodies; in the production of carotenoids, and in a more recent use, in the generation of recombinant protein, with the microalgae having clear advantages over bacteria like <I>E. coli</I>, among them the better product quality, increased scaling capacity and decreased risk of contamination stand out. Also, it possesses a variable storage cost (depending on the expression being in chloroplast or nucleus) that could be less than the associated cost with bacteria (Rivera et al, 2011) Another advantage related with the production of recombinant protein in the possibility of obtaining glycosylated protein.</P><P>The great amount of interest placed in this microorganism converts it in an attractive target for improvement through genetic engineering techniques, being the main objective of <B>Greenhardtii Project</B> the generation of a strain of <I>Chlamydomonas reinhardtii</I> with an optimized Calvin cycle.</P><P>For achieving this, as a team we focused in the study of the Calvin cycle, as a fundamental cycle in the primary metabolism of plants and algae. It is expected that better yields in this process bring improved strains that could grow more in less time, along with having a increased biomass accumulation. This could provide a strain that could, in one hand, serve as alternative for the generation of CO2 emissions biofilter, and on the other hand, serve as a platform for production of recombinant protein and other metabolites of interest in an optimized manner.</P><P>Our strategy is centered around the nuclear insertion of the cyanobacterial enzyme Fructose 1,6/Sedoheptulose 1,7 bisphosphatase in a wall-deficient strain of <I>Chlamydomonas reinhardtii</I> through the method of transformation by Glass Beads. This strategy was chosen over one involving transformation of a strain with normal cell wall (for example with <I>A. tumefaciens</I>) because this is faster and this part of the project only seeks to validate our hypothesis. For a future scale-up and with the certainty that the use of the enzyme is functional through this method, more resources will be given to a more complex nuclear transformation that generates strains with wall cell. On the other hand, the catalytic activities of this protein are:</P><P style="font-size:25px; text-align:center;">
D-fructose 1,6-bisphosphate + H2O = D-fructose 6-phosphate + phosphate
Sedoheptulose 1,7-bisphosphate + H2O = sedoheptulose 7-phosphate + phosphate.</P><P>This enzyme comes from the <I>Synechococcus sp</I>. cyanobacteria (strain WH8103) which is coded by a module of 1005 bp. (<A href="http://www.genome.jp/">www.genome.jp</A>) Given that photosynthesis and Calvin cycle are processes compartmentalized in the chloroplast, the production of the enzyme must be directed to that organel. For this, it was decided to use a chloroplast transit peptide coming from the mayor subunit of RuBisCo from <I>Chlamydomonas reinhardtii</I> composed of 32 amino acids (Krimm et al, 1999) To study the subcellular location of the protein and to prove that it is actually being sent to the stroma of the chloroplast a construct with the enzyme fused to GFP was designed. In this way, in addition of the transformed strains to express the protein FBP/SBPase, it is desired to generate other strains that contain such protein fused to GFP. Finally, all of the expression of this construct is subordinated to the strong constitutive promoter psaD (and its respective terminator) The detail of these construct are shown as follows in the next image:</P><DIV align="center"><P id="FigureText">Fig. 1: Scheme that shows the fusion protein construct to make the first transformation of <I>C. reinhardtii</I>.</P></DIV><P>If we could obtain transformant strains with a greater growth rate in relation to the non-transformed ones, the next step is to prove that these strains produce more recombinant protein than the ones without the FBP/SBPase construct. For this, a second transformation over the strains with increased growth will be done, seeking that these express an easily measurable protein to compare the expression between strains with and without FBP/SBPase. This protein will be mCherry. Furthermore, to get more control over the expression of recombinant protein its production will be commanded with the use of a regulable promoter. For this, as a team we have built a biobrick that contains the codification sequence for the pMETE promoter inducible by presence of B12 vitamin (Helliwell et al, 2014) which was kindly donated by PhD. Alison Smith of Cambridge University. Also, as said promoter has not been kinetically characterized, a proof of concept was designed, in which mCherry is expressed under the command of this promoter. The construct is detailed in the next image:</P><DIV align="center"><P id="FigureText">Fig. 2: Scheme that shows the designed constructs for the proof of concept that evaluates the production of the protein and the kinetics of the promoter.</P></DIV><P>These experimental strategies will be joined by the tests made from the mathematical model, delivering a solid characterization of the behavior of the regulable promoter METE.
All of our transformation will be made over the strains UVM4 and UVM11 using the binary plasmid pBC1-CrGFP kindly donated by the PhD. Myra Chávez of the Pontifical Catholic University of Chile. These strains have proven to avoid gene silencing and show good rates of recombinant protein expression.</P><P>Finally, as a conclusion, our project is summarized to the next genetic circuit:</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>