"
Page
Discussion
View source
History
teams
Log in
 
Team:OU-Norman OK/Project/Notebook
From 2013.igem.org
Home
Team
Undergraduates
Graduate Students
Advisors
Project
Notebook
Submitted Parts
Safety
Attributions
09/13/13
Gel of Pptb-
Row 1
Lane Number
Sample
1
Ladder (1kb plus)
2
Pptb- #9A
3
Pptb- #9 B
4
Empty
5
Empty
6
Empty
7
Empty
8
Empty
Back to top
09/13/13
Digest of Pptb- Sample
500ng X (μL/178.1ng) = 2.81μL
500ng X (μL/110.1ng) = 4.54μL
Sample
EcoR1
Pst1
10x Buffer
DNA
PCR dH2O
Total
Pptb- #9A
2μL
2μL
2μL
2.81μL
11.19μL
20μL
Pptb- #9B
2μL
2μL
2μL
4.54μL
9.46μL
20μL
Back to top
09/11/13
Quantification and DNA Extraction of 3A
QIAprep Spin Miniprep Kit (250)
Back to top
09/10/13
Gel of Pptb- + Clos Ori
Row #1
Lane Number
Sample
1
Ladder (1kb plus)
2
Pptb- #1
3
pptb- #2
4
Pptb- #3
5
Pptb- #4
6
Pptb- #6
7
Empty
8
Empty
Expected insert size of 125bp
No valid inserts found
Back to top
09/11/13
Digest of Pptb-
Pptb- #1
500ng X (μL/81.1ng) = 6.17μL
Pptb- #2
500ng X (μL/86.4ng) = 5.79μL
Pptb- #3
500ng X (μL/70.9ng) = 7.05μL
Pptb- #4
500ng X (μL/147.7ng) = 3.39μL
Pptb- #6
500ng X (μL/169.3ng) = 2.95μL
Sample
EcoR1
Pst1
10x Buffer
DNA
PCR dH2O
Total
Pptb- #1
2μL
2μL
2μL
6.17μL
9.83μL
20μL
Pptb- #2
2μL
2μL
2μL
5.79μL
8.21μL
20μL
Pptb- #3
2μL
2μL
2μL
7.05μL
6.95μL
20μL
Pptb- #4
2μL
2μL
2μL
3.39μL
10.61μL
20μL
Pptb- #5
2μL
2μL
2μL
2.95μL
11.05μL
20μL
Back to top
09/11/13
Quantification and DNA Extraction of 3A
QIAprep Spin Miniprep Kit (250) Back to top
09/10/13
Gel of Pptb- + Clos Ori
Row #1
Lane Number
Sample
1
Ladder (1kb plus)
2
Pptb- #1
3
pptb- #2
4
Pptb- #3
5
Pptb- #4
6
Pptb- #5
7
Pptb- #7
8
Pptb- #14
9
Pptb- #15
10
Empty
11
Empty
12
Empty
13
Empty
14
Empty
15
EMpty
16
Empty
Row #2
Lane Number
Sample
1
Ladder (1kb plus)
2
Pptb- #16
3
Pptb- #17
4
Pptb- #18
5
Pptb- #21
6
Pptb- #22
7
Pptb- #23
8
Pptb- #24
9
Pptb- #25
10
Empty
11
Empty
12
Empty
13
Empty
14
Empty
15
EMpty
16
Empty
Back to top
09/6/13
Digest of Pptb- and Clos Ori
3 #17
500ng X (μL/123.2ng) = 4.06μL
Pptb- #1
500ng X (μL/104.6ng) = 4.78μL
Pptb- #2
500ng X (μL/123.1ng) = 4.06μL
Pptb- #3
500ng X (μL/92.7ng) = 5,39μL
Pptb- #4
500ng X (μL/74.5ng) = 6.71μL
Pptb- #5
500ng X (μL/114.6ng) = 4.36μL
Pptb- #7
500ng X (μL/154.8ng) = 3.23μL
Pptb- #14
500ng X (μL/127.9ng) = 3.91μL
Pptb- #15
500ng X (μL/110.4ng) = 4.53μL
Pptb- #16
500ng X (μL/112.4ng) = 4.45μL
Pptb- #17
500ng X (μL/124.5ng) = 4.01μL
Pptb- #18
500ng X (μL/219.8ng) = 2.27μL
Pptb- #21
500ng X (μL/157.6ng) = 3.17μL
Pptb- #22
500ng X (μL/145.6ng) = 3.43μL
Pptb- #23
500ng X (μL/134.2ng) = 3.73μL
Pptb- #24
500ng X (μL/114.6ng) = 4.36μL
Pptb- #25
500ng X (μL/189.8ng) = 2.63μL
Sample
EcoR1
Pst1
10x Buffer
DNA
PCR dH2O
Total
Pptb- #1
2μL
2μL
2μL
4.78μL
9.22μL
20μL
Pptb- #2
2μL
2μL
2μL
4.06μL
9.94μL
20μL
Pptb- #3
2μL
2μL
2μL
5.39μL
8.61μL
20μL
Pptb- #4
2μL
2μL
2μL
6.71μL
7.29μL
20μL
Pptb- #5
2μL
2μL
2μL
4.36μL
9.64μL
20μL
Pptb- #7
2μL
2μL
2μL
3.23μL
10.77μL
20μL
Pptb- #14
2μL
2μL
2μL
3.91μL
10.09μL
20μL
Pptb- #815
2μL
2μL
2μL
4.53μL
9.47μL
20μL
Pptb- #16
2μL
2μL
2μL
4.45μL
9.55μL
20μL
Pptb- #17
2μL
2μL
2μL
4.02μL
9.98μL
20μL
Pptb- #18
2μL
2μL
2μL
2.27μL
11.73μL
20μL
Pptb- #21
2μL
2μL
2μL
3.17μL
10.83μL
20μL
Pptb- #22
2μL
2μL
2μL
3.43μL
10.57μL
20μL
Pptb- #23
2μL
2μL
2μL
3.73μL
10.27μL
20μL
Pptb- #24
2μL
2μL
2μL
4.36μL
9.64μL
20μL
Pptb- #25
2μL
2μL
2μL
2.63μL
11.37μL
20μL
Heat shock for 45 minutes in water bath and 15 minutes at 80°C on heat block to inactivate EcoR1
Back to top
09/6/13
Quantification and DNA Extraction of 3A
QIAprep Spin Miniprep Kit (250)
" alt="HTML tutorial" width="300" height="300" >
Back to top
09/5/13
Gel of Plep #1
Row 1
Lane Number
Sample
1
Ladder (1kb plus)
2
Plep #1
3
Empty
4
Empty
5
Empty
6
Empty
7
Empty
8
Empty
Back to top
09/5/13
Digest of Plep #1
Amount
Component
6.67μL
Plep
2μL
10x Buffer
2μL
EcoR1
2μL
Pst1
7.33μL
PCR Water
500ng X (μL/75ng) = 6.67μL
Back to top
09/5/13
Quantification and DNA Extraction of 3A
QIAprep Spin Miniprep Kit (250)
Back to top
09/4/13
Gel of Colony PCR
Row #1
Lane Number
Sample
1
Ladder (1kb plus)
2
T2 #1
3
T2 #2
4
T2 #3
5
T2 #4
6
Ter #5
7
T2 #16
8
T2 #17
9
T2 #18
10
T2 #19
11
T2 #20
12
Empty
13
Empty
14
Empty
15
EMpty
16
Empty
Row #2
Lane Number
Sample
1
Ladder (1kb plus)
2
Pveg2 #1
3
Pveg2 #2
4
Pveg2 #3
5
Pveg2 #4
6
Pveg2 #5
7
Pveg2 #16
8
Pveg2 #17
9
Pveg2 #18
10
Pveg2 #19
11
Pveg2 #20
12
Empty
13
Empty
14
Empty
15
EMpty
16
Empty
Row #1
Lane Number
Sample
1
Ladder (1kb plus)
2
Pptb- #1
3
Pptb- #2
4
Pptb- #3
5
Pptb- #4
6
Pptb- #5
7
Pptb- #16
8
Pptb- #17
9
Pptb- #18
10
Pptb- #19
11
Pptb- #20
12
Empty
13
Empty
14
Empty
15
EMpty
16
Empty
Row #2
Lane Number
Sample
1
Ladder (1kb plus)
2
Plep #1
3
Plep #2
4
Plep #3
5
Plep #4
6
Plep #5
7
Plep #16
8
Plep #17
9
Plep #18
10
Plep #19
11
Plep #20
12
Empty
13
Empty
14
Empty
15
EMpty
16
Empty
Back to top
09/1/13
3A Ligation
1) Clos Ori +Ptb- + pSB1A3
1) Clos Ori +Pveg1 + pSB1A3
1) Clos Ori +Pveg2 + pSB1A3
1) Clos Ori +Pliag + pSB1A3
1) mlSR + Plep + pSB1A3
1) mlSR +Terminator2 (4P) + pSB1A3
08/30/13
Gel of 3A Assembly
Row #1
Lane Number
Sample
1
2 #8
2
2 #10
3
2 #12
4
3 #15
5
3 #17
6
4 #1
7
4 #3
8
5 #1
9
6 #1
10
6 #3
11
Ladder (1kb plus)
12
Empty
13
Empty
14
Empty
15
EMpty
16
Empty
Back to top
08/30/13
Digest of 3A Assembly
3 #17
500ng X (μL/123.2ng) = 4.06μL
6 #1
500ng X (μL/98.7ng) = 5.07μL
6 #3
500ng X (μL/110.2ng) = 4.54μL
5 #1
500ng X (μL/162.6ng) = 3.08μL
2 #12
500ng X (μL/78.8ng) = 6.35μL
3 #15
500ng X (μL/131.8ng) = 3.79μL
4 #1
500ng X (μL/146.5ng) = 3.41μL
2 #8
500ng X (μL/117.7ng) = 4.25μL
2 #10
500ng X (μL/87.3ng) = 5.73μL
4 #3
500ng X (μL/82.9ng) = 6.03μL
Sample
EcoR1
Pst1
10x Buffer
DNA
PCR dH2O
Total
3 #17
2μL
2μL
2μL
4.06μL
9.94μL
20μL
6 #1
2μL
2μL
2μL
5.07μL
8.93μL
20μL
6 #3
2μL
2μL
2μL
4.54μL
9.46μL
20μL
5 #1
2μL
2μL
2μL
3.08μL
10.92μL
20μL
2 #12
2μL
2μL
2μL
6.35μL
7.65μL
20μL
3 #15
2μL
2μL
2μL
3.79μL
10.21μL
20μL
4 #1
2μL
2μL
2μL
3.41μL
10.59μL
20μL
2 #8
2μL
2μL
2μL
4.25μL
9.75μL
20μL
2 #10
2μL
2μL
2μL
5.73μL
8.27μL
20μL
4 #3
2μL
2μL
2μL
6.03μL
7.97μL
20μL
Heat shock for 45 minutes in water bath and 15 minutes at 80°C on heat block to inactivate EcoR1
Back to top
08/30/13
Quantification and DNA Extraction of 3A
QIAprep Spin Miniprep Kit (250)
Back to top
08/27/13
Gel of 3A Assembly
Row #1
Lane Number
Sample
1
Ladder (1kb plus)
2
1 #9
3
2 #22
4
2 #24
5
2 #26
6
3 #3
7
4 #2
8
5 #3
9
5 #11
10
6 #2
11
6 #19
12
6 #24
13
Empty
14
Empty
15
EMpty
16
Empty
Back to top
08/26/13
Digest of Ter and Adh6
1 #9
500ng X (μL/124.7ng) = 4μL
ter #18
500ng X (μL/118.9ng) = 4.21μL
2 #22
500ng X (μL/76.1ng) = 6.57μL
2 #24
500ng X (μL/96.4ng) = 5.19μL
2 #26
500ng X (μL/99.4ng) = 5.03μL
3 #3
500ng X (μL/101ng) = 4.95μL
4 #2
500ng X (μL/104ng) = 4.81μL
5 #3
500ng X (μL/84.6ng) = 5.91μL
5 #11
500ng X (μL/124.6ng) = 4.01μL
6 #2
500ng X (μL/78.8ng) = 6.35μL
6 #19
500ng X (μL/141.4ng) = 3.54μL
6 #24
500ng X (μL/100.2ng) = 4.99μL
Sample
EcoR1
Pst1
10x Buffer
DNA
PCR dH2O
Total
1 #9
2μL
2μL
2μL
4.21μL
9.79μL
20μL
2 #22
2μL
2μL
2μL
6.57μL
7.43μL
20μL
2 #24
2μL
2μL
2μL
5.19μL
8.81μL
20μL
2 #26
2μL
2μL
2μL
5.03μL
8.97μL
20μL
3 #3
2μL
2μL
2μL
4.95μL
9.05μL
20μL
4 #2
2μL
2μL
2μL
4.81μL
9.19μL
20μL
5 #3
2μL
2μL
2μL
5.91μL
8.09μL
20μL
5 #11
2μL
2μL
2μL
4.01μL
9.99μL
20μL
6 #2
2μL
2μL
2μL
6.35μL
7.65μL
20μL
6 #19
2μL
2μL
2μL
3.54μL
10.46μL
20μL
6 #24
2μL
2μL
2μL
4.99μL
9.01μL
20μL
Back to top h1>
08/27/13
Quantification of 3A Assembly
QUIAPREP/Spin Miniprep Kit < (250)
Back to top
08/26/13
Gel of ter and Adh6
Row 1
Lane Number
Sample
1
Ladder (1kb plus)
2
ter #6
3
ter #18
4
ter #24
5
Adh6 #8
6
Adh6 #14
7
Adh6 #15
8
Empty
Back to top
08/26/13
Digest of Ter and Adh6
ter #6
500ng X (μL/124.7ng) = 4μL
ter #18
500ng X (μL/108ng) = 4.63μL
ter #24
500ng X (μL/315ng) = 1.59μL
Adh6 #8
500ng X (μL/300.3ng) = 12.41μL
Adh6 #14
500ng X (μL/95.4ng) = 5.24μL
Adh6 #15
500ng X (μL/328.3ng) = 1.52μL
Sample
EcoR1
Pst1
10x Buffer
DNA
PCR dH2O
Total
ter #6
2μL
2μL
2μL
4.0μL
10.0μL
20μL
ter #18
2μL
2μL
2μL
4.63μL
9.37μL
20μL
ter #24
2μL
2μL
2μL
1.59μL
12.41μL
20μL
Adh6 #8
2μL
2μL
2μL
1.67μL
12.33μL
20μL
Adh6 #14
2μL
2μL
2μL
5.24μL
11.76μL
20μL
Adh6 #15
2μL
2μL
2μL
1.52μL
12.48μL
20μL
Back to top
08/26/13
DNA Extraction of Adh6 and Ter
QIAprep Spin Miniprep Kit (250)
Back to top
08/22/13
Transformation of 3A Assembly
Transform 4μL of ligation product into 100μL aliquots of competent E. coli cells
Incubate on ice for 30 minutes
Heat shock in water bath for 60 seconds
Incubate on ice for 5 minutes
Add 600μL of psi broth
Incubate at 37°C for 2 hours at 200 rpm
Make necessary dilutions and plate out
Back to top
08/22/13
3A Ligation
1) Clos Ori +Pveg2 (20G) + pSB1A3
1) Clos Ori +Plep (20E) + pSB1A3
1) Clos Ori +Pveg1 (23B) + pSB1A3
1) Clos Ori +Pliag (20A) + pSB1A3
1) mlSR +Terminator1 (6D) + pSB1A3
1) mlSR +Terminator2 (4P) + pSB1A3
Amount
Sample
2μL
Upstream Part
2μL
Downstream Part
2μL
Destination Backbone
2μL
10x T4 DNA Ligase Buffer
0.5μL
T4 DNA Ligase
10μL
PCR Water
Incubate at room temperature for 10 minutes, then at 80°C for 20 minutes to inactivate EcoR1
Back to top
08/15/13
Clos Ori
500ng X (440bp/2000bp) X (μL/190.5ng) = 0.577μL 1:105.7μL
mlSR
500ng X (750bp/2000bp) X (μL/260.4ng) = 0.908μL 1:109.08μL
Pliag
500ng X (121bp/2000bp) X (μL/150.1ng) = 0.201μL 1:102.01μL
Pveg1
500ng X (100bp/2000bp) X (μL/90.0ng) = 0.278μL 1:102.78μL
Plep
500ng X (157bp/2000bp) X (μL/235.0ng) = 0.167μL 1:101.67μL
Pveg2
500ng X (237bp/2000bp) X (μL/192.3ng) = 0.308μL 1:103.08μL
Terminator
500ng X (50bp/2000bp) X (μL/108.6ng) = 0.115μL 1:101.15μL
Terminator
500ng X (50bp/2000bp) X (μL/131.8ng) = 0.095μL 1:100.95μL
pSB1A3
500ng X (μL/31.8ng) = 2μL
Sample
EcoR1
Pst1
Xba1
Spe1
100x Bovine Serum Albumin
10x Buffer
DNA
PCR dH2O
Total
Clos Ori
1μL
1μL
-
-
1μL
5μL
5.7μL
36.3μL
50μL
mlSR
1μL
1μL
-
-
1μL
5μL
9.08μL
32.92μL
50μL
Pliag
-
-
1μL
1μL
1μL
5μL
5.7μL
39.99μL
50μL
Pveg1
-
-
1μL
1μL
1μL
5μL
2.78μL
39.22μL
50μL
Pveg2
-
-
1μL
1μL
1μL
5μL
1.67μL
40.33μL
50μL
Plep
-
-
1μL
1μL
1μL
5μL
3.08μL
38.92μL
50μL
Terminator 6D
-
-
1μL
1μL
1μL
5μL
1.15μL
40.85μL
50μL
Temrinator 4P
-
-
1μL
1μL
1μL
5μL
0.95μL
41.05μL
50μL
pSB1A3
1μL
1μL
-
-
1μL
5μL
16μL
26μL
50μL
Back to top
08/21/13
PCR of BktB Fragments 1 & 2 used for SDM Step1
Previous attempt produced a lot of extraneous artifacts.
Primer concentration was too high and so was the template concentrations
Used Grad program previously used to run PCR machine
50°C to 60°V used 4 and 0
Back to top
08/21/13
Quantification of Backbones Post-SPRI Based Size Selection
The concentrations of pSB1A3 is low. Therefore, the SPRI based size selection was conducted on it once more. The gel results are displayed below.
Back to top
08/21/13
Transformation of Ter + pSB1C3 and Adh6 + pSB1C3
Thaw competent Top10 cell of E. coli from -80°C (100μL aliquots)
Add 20μL of ligation product to Top 10 E. coli
Incubate on ice for 30 minutes
Heat shock in water bath for exactly 60 seconds
Incubate on ice for 5 minutes
Add 600μL of psi broth
Incubate at 37°C for 2 hours at 200 rpm
Make dilutions using 90μL of psi broth: 1:10, 1:100, 1:1000
Plate out 25μL undiluted and diluted solutions onto a petri dish
Row 1
Lane Number
Sample
1
Ladder (1kb plus)
2
pSB1A3
3
Empty
4
Empty
5
Empty
6
Empty
7
Empty
8
Empty
Back to top
08/21/13
Gel Post SPRI Based Size Selection
Row 1
Lane Number
Sample
1
Ladder (1kb plus)
2
pSB1A3
3
pSB1C3
4
pSB1K3
5
Empty
6
Empty
7
Empty
8
Empty
Back to top
08/21/13
SPRI Based Size Selection
Use the instructions labeled “left side size slection.” This kit will hopefully remove the primer dimer from our product. Remove the primer dimer from the backbone PCR clean-up.
Back to top
08/21/13
Gel of Refined Backbone PCR
Row 1
Lane Number
Sample
1
Ladder (1kb plus)
2
pSB1A3
3
pSB1C3
4
pSB1K3
5
Empty
6
Empty
7
Empty
8
Empty
Back to top
08/21/13
PCR of pSB1A3, pSB1K3, pSB1K3 (refined)
Sample G in the primer matrix will be used as a template for this particular PCR
Label
Amount
Component
PCR # 1
3μL
SBprep-3P-1
1μL
SBprep-2Ea
2.0μL
pSB1A3
94μL
PCR Supermix High Fidelity
PCR #3
3μL
SBprep-3P-1
1μL
SBprep-2Ea
2.0μL
pSB1C3
PCR #2
3μL
SBprep-3P-1
1μL
SBprep-2Ea
2.0μL
pSB1K3
94μL
PCR Supermix High Fidelity
Back to top
08/20/13
Gel of Fifth PCR of Backbones
Row 1
Lane Number (1kb plus)
Sample
1
Ladder
2
E
3
G
4
Empty
5
Empty
6
Empty
7
Empty
8
Empty
Back to top
08/15/13
Use the two concentrations of E and G on 20-August
Back to top
08/20/13
PCR of Backbone Discussion
After discussing the results with Dr. Wawrik, it has been decided to cut the number of cycles down to thirty, and to increase the template used by two.
The best concentrations of primers to be used are indicated by the results of the primer matrix on letters E and G. F appeared to have been a probable pipetting error.
List of reactants and amount for primer matrix PCR optimization
Label
Amount
Component
E
0.5μL
SBprep-3P-1 (1:2 dilutions) use 1μL
1.5μL
SBprep-2Ea
1.0μL
Template
46.5μL
PCR Supermix High Fidelity
G
1.5μL
SBprep-3P-1
0.5μL
SBprep-2Ea (1:2 dilutions) use 1μL
1.0μL
Template
46.5μL
PCR Supermix High Fidelity
Back to top
08/19/13
Gel of Ter and Adh6 Post Digest
Row 1
Lane Number (1kb plus)
Sample
1
Ladder
2
Adh6 #19
3
Adh6 #3
4
Ter #5
5
Ter #23
6
Ter #26
7
Ter #9
8
Empty
Back to top
08/14/13
Gel of pSB1K3, pSB1C3, pSB1A3 Stocks from 9-May
Row 1
Lane Number (1kb plus)
Sample
1
Ladder
2
pSB1A3
3
pSB1C3
4
pSB1K3
5
Empty
6
Empty
7
Empty
8
Empty
Back to top
08/14/13
PCR Clean-up Quanitification
Although the concentrations are good, the purity is still in question. It is an improvement on the PCR from 13-August
One possibility is that the original stocks used as templates in the PCR may not be pure enough. Therefore, a gel will be run on these original stocks
Back to top
08/14/13
Gel of pSB1K3, pSB1C3, pSB1A3 Post Clean-up
Row 1
Lane Number (1kb plus)
Sample
1
Ladder
2
pSB1A3
3
pSB1C3
4
pSB1K3
5
Empty
6
Empty
7
Empty
8
Empty
Back to top
08/14/13
PCR Clean-up of the Three Backbones
QIAquick PCR purification kit (250) is used before viewing the post clean-up product on a gel
Back to top
08/14/13
Gel of BktB Fragments
Row 1
Lane Number (1kb plus)
Sample
1
Ladder
2
50°C
3
4
5
60°C
6
Empty
7
Empty
8
Empty
What happened here? What is in the lanes above the comb?
Many have some product and we will use the forward and reverse primers to get the whole gene, if present
Back to top
08/14/13
Gel of pSB1K3, pSB1C3, pSB1A3 Pre Clean-up
Row 1
Lane Number (1kb plus)
Sample
1
Ladder
2
pSB1A3
3
pSB1C3
4
pSB1K3
5
Empty
6
Empty
7
Empty
8
Empty
Back to top
08/14/13
Quantified Backbone Stocks for PCR
The following stock are to be used for backbone amplification in the PCR procedure illustrated on 14-August
Back to top
08/14/13
Preparing an Equimolar solution of BktB Fragments for PCR to Stitch
Fragment 1: 340bp – 125.6ng/μL
Fragment 1: 660bp – 66.3ng/μL
Fragment 1: 300bp – 231.7ng/μL
Use 1μL of fragment 3 = 231.7ng/μL
Fragment 2:
(221.7) X (660/300) = 520ng x (1μL/66.3ng) = 7.7μL
Fragment 3:
(221.7) X (340/300) = 263ng x (1μL/125.6ng) = 2.1μL
Added to a 1.5mL epi tube and diluted to 200μL
1004ng of DNA/ 200μL = 5ng/μL final concentration of DNA
Back to top
08/14/13
PCR of i-GEM Plasmid Backbones Continued
PCR #1
Amount (μL)
Component
100
PCR Supermix High Fidelity
0.7
SB-prep-3P-1
0.7
SB-prep-2Ea
0.6
pSB1A3 (1ng/μL)
PCR #2
Amount (μL)
Component
100
PCR Supermix High Fidelity
0.7
SB-prep-3P-1
0.7
SB-prep-2Ea
0.6
pSB1C3 (1ng/μL)
PCR #3
Amount (μL)
Component
100
PCR Supermix High Fidelity
0.7
SB-prep-3P-1
0.7
SB-prep-2Ea
0.6
pSB1K3 (1ng/μL)
Back to top
08/14/13
Adjusted PCR of iGEM Plasmid Backbones
Diluting concentrations of pSB1A3, pSB1C3, pSB1K3 to 1ng/μL
Quantification of current backbones to confirm concentrations before conducting dilution
pSB1A3
Vf = (27.5ng/μL) X (1μL) X (1μL/1ng) = 27.5μL – 1μL of template = 26.5μL of PCR water
pSB1C3
Vf = (39.9ng/μL) X (1μL) X (1μL/1ng) = 39.9μL – 1μL of template = 38.9μL of PCR water
pSB1K3
Vf = (29.4ng/μL) X (1μL) X (1μL/1ng) = 29.4μL – 1μL of template = 28.4μL of PCR water
Back to top
08/13/13
Quantification of PCR Clean-up
The concentrations are good, however the purity of the product is not. Actions taken to remedy this problem are to run another PCR using 1ng/μL of template and running the third step of the PCR cycle at 68°C for 5 minutes.
Back to top
08/13/13
Gel of pSB1K3, pSB1C3, pSB1A3 Post Clean-up
Row 1
Lane Number
Sample
1
Ladder
2
pSB1A3
3
pSB1C3
4
pSB1K3
5
Empty
6
Empty
7
Empty
8
Empty
Back to top
08/13/13
PCR Clean-up of pSB1K3, pSB1C3, pSB1A3
The three backbones were cleaned up using QIAquick PCR Purification Kit (250) so that the next gel will much more clearly display the backbone.
Back to top
08/13/13
Gel of pSB1K3, pSB1C3, pSB1A3
Row 1
Lane Number
Sample
1
Ladder
2
pSB1K3
3
pSB1C3
4
pSB1A3
5
Empty
6
Empty
7
Empty
8
Empty
Back to top h1>
08/13/13
Plasmid Extraction of AaDc-, AaDc+, Thl+, Thl-, and Ptb+
QUIAPREP/Spin Miniprep Kit < (250)
Cells were centrifuged from an inoculated broth
None of the concentrations obtained are high enough to work with. Must go about finding a more efficient plasmid extraction for these samples.
Back to top
08/13/13
PCR of i-GEM Plasmid Backbones
PCR #1
Amount (μL)
Component
100
PCR Supermix High Fidelity
0.7
SB-prep-3P-1
0.7
SB-prep-2Ea
0.6
pSB1C3 (43ng/μL)
PCR #2
Amount (μL)
Component
100
PCR Supermix High Fidelity
0.7
SB-prep-3P-1
0.7
SB-prep-2Ea
0.6
pSB1C3 (74.8ng/μL)
PCR #3
Amount (μL)
Component
100
PCR Supermix High Fidelity
0.7
SB-prep-3P-1
0.7
SB-prep-2Ea
0.6
pSB1C3 (55.1ng/μL)
Back to top
08/14/13
Post-Cleanup of BktB PCR Products
All Fragment 1’s, lanes 8 and 9 of Fragment 2, and all Fragment
Back to top
08/13/13
PCR of BktB from PCR Tempalte
Back to top
08/9/13
Gel of BktB Fragments
Row #1
Lane Number
Sample
Temperature (°C)
1
Ladder
2
Fragment 1
50
3
Fragment 1
4
Fragment 1
5
Fragment 1
60
6
Fragment 2
50
7
Fragment 2
8
Fragment 2
9
Fragment 2
60
10
Fragment 3
50
11
Fragment 3
12
Fragment 3
13
Fragment 3
60
14
Empty
15
EMpty
16
Empty
Based on 2% agarose gel, 115V for about 30 minutes
Going to PCR the whole gene to see if I can reduce non-specific products in fragment #2
Back to top
08/8/13
PCR of BktB Fragments
Preformed gradient PCR for all 3 BktB fragments
Undiluted template was used
Back to top
08/7/13
Transformation of Top10 with ter in pSB1C3
Thaw competent 100μL aliquot of E. coli cells from -80°C freezer
Add 20μL of ligation poduct to Top10 E. coli
Incubate on ice for 30 minutes
Heat shock in water bath for 60 seconds
Incubate on ice for 5 minutes
Add 600μL of psi broth
Incubate at 37°C for 2 hours at 200 rpm
Make a dilution series of 1:10, 1:100, 1:1000, and 1:10000
Back to top
08/7/13
Ligation of pSB1C3 and Adh6
Into 1.5mL microcentrifuge tubes
Amount (μL)
Component
20
Heat killed digest of pSB1C3
20
Heat killed digest of Adh6
1.0
T4 Ligase
5.0
10X T4 Ligase Buffer
4.0
Nuclease Free PCR Water
50
Total
Incubate on bench for 45 minutes
Back to top
08/9/13
digestion of ter
Sample
EcoR1
Pst1
10x Buffer
DNA
PCR dH2O
Total
ter #23
1μL
1μL
1μL
2.5μL
14.5μL
20μL
ter #7
1μL
1μL
1μL
1.5μL
15.5μL
20μL
ter #26
1μL
1μL
1μL
2μL
15μL
20μL
ter #5
1μL
1μL
1μL
2μL
15μL
20μL
pSB1C3
1μL
1μL
1μL
2.5μL
14.5μL
20μL
Back to top
08/9/13
Plasmid Extraction of AaDc- and Ptb-
Back to top
08/7/13
Restriction Digest of Ccell Adh1 and Adh2
Adh1 #5
500ng X (μL/297.2ng) = 1.68μL
Adh1 #19
500ng X (μL/89.3ng) = 5.60μL
Adh1 #24
500ng X (μL/261.6ng) = 1.91μL
Adh1 #4
500ng X (μL/264.0ng) = 1.89μL
Adh1 #13
500ng X (μL/588.6ng) = 0.85μL
Sample
EcoR1
Pst1
10x Buffer
DNA
PCR dH2O
Total
Adh1 #5
2μL
2μL
2μL
1.5μL
12.5μL
20μL
Adh1 #19
2μL
2μL
2μL
5.5μL
8.5μL
20μL
Adh1 #24
2μL
2μL
2μL
2μL
12μL
20μL
Adh1 #4
2μL
2μL
2μL
2μL
12μL
20μL
Adh1 #13
2μL
2μL
2μL
1μL
13μL
20μL
Sample
EcoR1
Pst1
10x Buffer
DNA
PCR dH2O
Total
Adh6 #5
1μL
1μL
1μL
5.0μL
12.0μL
20μL
Adh6 #19
1μL
1μL
1μL
2.0μL
15.0μL
20μL
Adh6 #3
1μL
1μL
1μL
3.0μL
14.0μL
20μL
Adh6 #15
1μL
1μL
1μL
3.0μL
14.0μL
20μL
pSB1C3
1μL
1μL
1μL
2.5μL
14.5μL
20μL
Back to top
08/7/13
Gel for Post-Digest of Promoters and Terminators
Using 2% agarose in the gel
Gel 1
Lane Number
Sample
1
Ladder
2
#16-6D
3
#16-23B
4
#9-4P
5
#9-6D
6
#3-23B
7
#14-20E
8
Empty
Gel 2
Lane Number
Sample
1
Ladder
2
#15-20A
3
#7-20G
4
#7-4D
5
#14-20A
6
#10-4P
7
#11-20E
8
Empty
Samples display a possible insert, and will be sent off for sequencing
3-23B, 16-23B, 14-20E, 15-20A, 16-6D, 14-20A, and 10-4P
Back to top
08/7/13
Modified Restriction Digest of Promoters and Terminators
Sample
EcoR1
Pst1
10x Buffer
DNA
PCR dH2O
Total
#16-6D
1μL
1μL
1μL
10.0μL
7.0μL
20μL
#16-23B
1μL
1μL
1μL
10.0μL
7.0μL
20μL
#9-4P
1μL
1μL
1μL
10.0μL
7.0μL
20μL
#9-6D
1μL
1μL
1μL
10.0μL
7.0μL
20μL
#3-23B
1μL
1μL
1μL
10.0μL
7.0μL
20μL
#14-20E
1μL
1μL
1μL
10.0μL
7.0μL
20μL
#15-20A
1μL
1μL
1μL
10.0μL
7.0μL
20μL
#7-20G
1μL
1μL
1μL
10.0μL
7.0μL
20μL
#7-4D
1μL
1μL
1μL
10.0μL
7.0μL
20μL
#14-20A
1μL
1μL
1μL
10.0μL
7.0μL
20μL
#10-4P
1μL
1μL
1μL
10.0μL
7.0μL
20μL
#11-20E
1μL
1μL
1μL
10.0μL
7.0μL
20μL
Back to top
08/7/13
Plasmid Extraction of Ccell of Adh1 and Adh2
QIAPrep Spin Miniprep Kit (250)
The gene’s origin is unknown, and will be found upon sequencing
Back to top
07/18/13
Plasmid Prep of Transformed Top10 Cells with pSB1C3 + mlsR and puc19 + KIVD
Back to top
07/17/13
DNA Extraction from Ca, Cs, Cb, and Cc using MoBio Power Soil DNA Extraction Kit
Back to top
07/11/13
Dual Digest of pSB1C3 and mlsR Part
Back to top
07/25/13
Gel of Aadc-/+, thl+/-, Ptb-/+ from 1 Aug
Row #1
Lane Number
Sample
1
Ladder
2
Aadc- #6
3
Aadc- #23
4
Aadc+ #20
5
thl- #4
6
Aadc+ #15
7
Aadc+ #2
8
thl+ #17
9
thl+ #3
10
thl+ #10
11
thl- #14
12
thl- #21
13
Empty
14
Empty
15
EMpty
16
Empty
Row #2
Lane Number
Sample
1
Ladder
2
Empty
3
Empty
4
Empty
5
Empty
6
Empty
7
Ladder
8
Ptb- #5
9
Ptb+ #9
10
Ptb+ #1
11
Ptb+ #26
12
Ptb- #18
13
Ptb- #8
14
Empty
15
Empty
16
Empty
Back to top
08/02/13
Plasmid Extraction of Ccell Adh and Csac Adh
QIAPrep Spin Miniprep Kit (250)
Back to top
08/02/13
Restriction Digest of Aadc-/+ and Ptb-/+
#2 Aadc+ 500ng  (1&#956L⁄85.9ng) = 6.0μL
#15 Aadc+ 500ng  (1&#956L⁄117.5ng) = 4.5μL
#20 Aadc+ 500ng  (1&#956L⁄92.1ng) = 5.5μL
#23 Aadc- 500ng  (1&#956L⁄184.8ng) = 3.0μL
#6 Aadc- 500ng  (1&#956L⁄218.1ng) = 2.5μL
#17 thl+ 500ng  (1&#956L⁄100.0ng) = 5.0μL
#3 thl+ 500ng  (1&#956L⁄84.4ng) = 6.0μL
#10 thl+ 500ng  (1&#956L⁄90.4ng) = 5.5μL
#14 thl- 500ng  (1&#956L⁄85.7ng) = 6.0μL
#21 thl- 500ng  (1&#956L⁄89.1ng) = 5.5μL
#4 thl- 500ng  (1&#956L⁄131.8ng) = 4.0μL
#9 Ptb+ 500ng  (1&#956L⁄81.4ng) = 6.0μL
#1 Ptb+ 500ng  (1&#956L⁄124.2ng) = 4.0μL
#26 Ptb+ 500ng  (1&#956L⁄206.7ng) = 2.5μL
#18 Ptb- 500ng  (1&#956L⁄103.7ng) = 5.0μL
#8 Ptb- 500ng  (1&#956L⁄150.1ng) = 3.5μL
#5 Ptb- 500ng  (1&#956L⁄216.0ng) = 2.5μL
Sample
EcoR1
Pst1
10x Buffer
DNA
PCR dH2O
Total
#2 Aadc+
2μL
2μL
2μL
6.0μL
8.0μL
20μL
#15 Aadc+
2μL
2μL
2μL
4.5μL
9.5μL
20μL
#20 Aadc+
2μL
2μL
2μL
5.5μL
8.5μL
20μL
#23 Aadc-
2μL
2μL
2μL
3.0μL
11.0μL
20μL
#6 Aadc-
2μL
2μL
2μL
2.5μL
11.5μL
20μL
#17 thl+
2μL
2μL
2μL
5.0μL
9.0μL
20μL
#3 thl+
2μL
2μL
2μL
6.0μL
8.0μL
20μL
#10 thl+
2μL
2μL
2μL
5.5μL
8.5μL
20μL
#14 thl-
2μL
2μL
2μL
6.0μL
8.0μL
20μL
#21 thl-
2μL
2μL
2μL
5.5μL
8.5μL
20μL
#4 thl-
2μL
2μL
2μL
4.0μL
10.0μL
20μL
#9 Ptb+
2μL
2μL
2μL
6.0μL
8.0μL
20μL
#1 Ptb+
2μL
2μL
2μL
4.0μL
10.0μL
20μL
#26 Ptb+
2μL
2μL
2μL
2.5μL
11.5μL
20μL
#18 Ptb-
2μL
2μL
2μL
5.0μL
9.0μL
20μL
#8 Ptb-
2μL
2μL
2μL
3.5μL
10.5μL
20μL
#5 Ptb-
2μL
2μL
2μL
2.5μL
11.5μL
20μL
Back to top
08/02/13
Plasmid Extraction of Aadc-/+, Ptb-/+, thl-/+
QIAPrep Miniprep Kit (250)
Back to top
08/01/13
Gel of Regulatory Promoters and Terminators (1 Aug)
Row 1
Lane Number
Sample
1
Ladder
2
3:23B
3
10:4P
4
12:20G
5
9:6D
6
16:6D
7
8:20G
8
11:20E
9
14:20E
10
15:20A
11
14:20A
12
9:4P
13
16:23B
14
9:20G
15
7:20G
Back to top
08/1/13
Restriction Digest using EcoR1 and Pst1 (26 July Quantification)
#16-6D 500ng  (1&#956L⁄108.6ng) = 4.5μL
#14-20E 500ng  (1&#956L⁄235.2ng) = 2.0μL
#12-20G 500ng  (1&#956L⁄135.1ng) = 3.5μL
#8-20G 500ng  (1&#956L⁄192.3ng) = 2.5μL
#14-20A 500ng  (1&#956L⁄156.5ng) = 3.0μL
#9-6D 500ng  (1&#956L⁄`99.7ng) = 2.5μL
#10-4P 500ng  (1&#956L⁄131.8ng) = 4.0μL
#7-20G 500ng  (1&#956L⁄232.8ng) = 2.0μL
#3-23B 500ng  (1&#956L⁄90.0ng) = 5.5μL
#16-23B 500ng  (1&#956L⁄123.5ng) = 4.0μL
#9-4P 500ng  (1&#956L⁄145.2ng) = 3.5μL
#9-20G 500ng  (1&#956L⁄106.8ng) = 4.5μL
#15-20A 500ng  (1&#956L⁄150.1ng) = 3.5μL
#11-20E 500ng  (1&#956L⁄126.2ng) = 4.0μL
Sample
EcoR1
Pst1
10x Buffer
DNA
PCR dH2O
Total
#16-6D
2μL
2μL
2μL
4.5μL
9.5μL
20μL
#14-20E
2μL
2μL
2μL
2.0μL
12.0μL
20μL
#12-20G
2μL
2μL
2μL
3.5μL
10.5μL
20μL
#8-20G
2μL
2μL
2μL
2.5μL
11.5μL
20μL
#14-20A
2μL
2μL
2μL
3.0μL
11.0μL
20μL
#9-6D
2μL
2μL
2μL
2.5μL
11.5μL
20μL
#10-4P
2μL
2μL
2μL
4.0μL
10.0μL
20μL
#7-20G
2μL
2μL
2μL
2.0μL
12.0μL
20μL
#3-23B
2μL
2μL
2μL
5.5μL
8.5μL
20μL
#16-23B
2μL
2μL
2μL
4.0μL
10.0μL
20μL
#9-4P
2μL
2μL
2μL
3.5μL
10.5μL
20μL
#9-20G
2μL
2μL
2μL
4.5μL
9.5μL
20μL
#15-20A
2μL
2μL
2μL
3.5μL
10.5μL
20μL
#11-20E
2μL
2μL
2μL
4.0μL
10.0μL
20μL
Back to top
08/1/13
Plasmid Extraction of Terminators and Promoters
QIAPrep Spin Miniprep Kit (250)
Back to top
08/01/13
Ampicillin 1x Preparation
To make 40mL of ampicillin 1x antibiotic
2.0g ampicillin 1000x stock from 4°C refrigerator
Pour the amipicillin ino a 50mL falcon
Fill flacon wih dH2O to 40mL mark
Use 10mL syringe with puradisc 25mm filter to dispense 10mL of ampicillin 1x into four 10mL falcons (leaves alternative stocks if one is contaminated)
Ampicillin 1000x = 50ng/mL
50ng/mL x 40mL = 2.0g Ampicillin 1000x
Back to top
07/31/13
Promoter and Terminator Culture Preparation
Used 2XYT liquid medium instead of LB to increase carbon amount in order to enhance growth of cultures.
-3mL of 2XYT with Chloramphenicol antibiotic
-Pellet of culture
Incubated at 37°C with shaking at 220rpm
Back to top
07/31/13
2XYT Medium Preparation
(Agar: 10g)
Bacto Tryptone: 16g
Yeast Extract: 10g
Sodium Chloride: 5g
Distilled Water: 1000mL
Fill 1L jar with 500mL of dH2O. Pour components and magnetic stirring rod into jar. Fill jar with another 500mL of dH2O until the shoulder of the jar is reached.
Autoclave for 20 minutes at slow exhaust (Setting #2)
Back to top
07/31/13
Gel of DNA Preps from R. eutropha
Run a gel with 3μL of dye and 5μL of DNA extract for each of our R. eutropha extracts.
Row #1
Lane Number
Sample
1
Ladder
2
Phenol Chloroform (27 July)
3
R. eutropha 1 (24 July)
4
R. eutropha 2 (24 July)
5
Power Soil (25 July)
6
Phenol Chloroform (25 July)
7
Empty
8
Empty
NOTE: Lane six shows chewed up DNA by nucleases
Disposed all samples except for Power Soil because it has the highest efficacy
Back to top
07/30/13
Plasmid Purification
Plasmid purification of top 10 cells transformed with ligation of pSB1C3 and Ca promoters (Ptb-RBS, Ptb+RBS, thl+RBS, thl-RBS, Aadc+RBS, AaDc-RBS)
Back to top
07/30/13
Plasmid Extraction of Terminators and Promoters
Used QIAPrep Spin Miniprep Kit (250)
Back to top
07/27/13
Dual Re gel of Regulatory Terminators and Promoters
Row 1
Lane Number
Sample
1
Ladder
2
#14-20A
3
#17-20G
4
#14-20E
5
#11-20E
Back to top
07/27/13
Dual Digest of pSB1C3 and mlsR Part
Row 1
Lane Number
Sample
1
Ladder
2
#10-4p
3
#14-20E
4
#14-20A
5
#7-20G
6
#11-20E
7
#16-20E
8
#9-4P
9
#15-20A
Back to top
07/11/13
Dual Digest of pSB1C3 and mlsR Part
500ng in 20μL with 2μL for each digest
#11-20E 500ng  (1&#956L⁄58.1ng) = 8.5μL
#9-4P 500ng  (1&#956L⁄40.2ng) = 12.5μL
#14-20A 500ng  (1&#956L⁄43.5ng) = 12.5μL
#7-20G 500ng  (1&#956L⁄59.4ng) = 12.5μL
#14-20E 500ng  (1&#956L⁄37.5ng) = 12.5μL
#10-4P 500ng  (1&#956L⁄43.6ng) = 12.5μL
#15-20A 500ng  (1&#956L⁄51.6ng) = 12.5μL
#16-23B 500ng  (1&#956L⁄52.5ng) = 12.5μL
Sample
EcoR1
Pst1
10x Buffer
DNA
PCR dH2O
Total
#11-20E
2μL
2μL
2μL
8.5μL
5.5μL
20μL
#9-4P
2μL
2μL
2μL
12.5μL
1.5μL
20μL
#14-20A
2μL
2μL
2μL
11.5μL
2.5μL
20μL
#7-20G
2μL
2μL
2μL
8.5μL
5.5μL
20μL
#14-20E
2μL
2μL
2μL
13.5μL
0.5μL
20μL
#10-4P
2μL
2μL
2μL
11.5μL
2.5μL
20μL
#15-20A
2μL
2μL
2μL
9.5μL
4.5μL
20μL
#16-23B
2μL
2μL
2μL
9.5μL
4.5μL
20μL
Back to top
07/26/13
Nano Drop of Isolated DNA from July 25
Back to top
07/26/13
Gel of BktB piece 3 PCR using DNA Extracts
Row #1
Lane Number
Sample
1
Ladder
2
R. eutropha extract (27 July)
3
R. eutropha 1 (24 July)
4
R. eutropha 2 (24 July)
5
Power Soil (25 July)
6
Phenol Chloroform (25 July)
7
Empty
8
Empty
Back to top
07/26/13
Gel Repeat of BktB PCR Gradient (25 July)
Repeated gel with larger loading of DNA. Removing only loaded 5μL of dye pls products. Wanted to see if bands appear brighter.
Row #1
Lane Number
Sample
Temperature (°c)
1
Kb Plus Ladder
2
Piece 1
50
3
Piece 1
52.7
4
Piece 1
54.5
5
Piece 1
56.4
6
Piece 1
60
7
Piece 2
50
8
Piece 2
52.7
9
Piece 2
54.5
10
Piece 2
56.4
11
Piece 2
60
12
Ladder
13
Empty
14
Empty
15
EMpty
16
Empty
Row #1
Lane Number
Sample
Temperature (°c)
1
Kb Plus Ladder
2
Piece 3
50
3
Piece 3
52.7
4
Piece 3
54.5
5
Piece 3
56.4
6
Piece 3
60
7
Whole gene
50
8
Whole gene
52.7
9
Whole gene
54.5
10
Whole gene
56.4
11
Whole gene
60
12
Ladder
13
Empty
14
Empty
15
Empty
16
Empty
Back to top
07/11/13
PCR of BktB piece 3 using Different DNA Extraction
Wanted to identify which extracts may have DNA present
Previous gradient seems to have produced a PCR product for 3 piece. Used temperature 55°C, since this is the predicted optimal temperature for the SDM2 forward and the BktB_Rev primers
Templates used:
R. eutropha = 27 July_ 16.9
R. eutropha = 24 July_ 25
R. eutropha = 24 July_ 38.9
Power Soil Extraction = 25 July_ 28.3
Phenol Chloroform Extraction = 25 July_ 637
Back to top
07/25/13
Gel of Gradient PCR for BktB Site-Directed Mutagenesis
Row #1
Lane Number
Sample
Temperature (°c)
1
Kb Plus Ladder
2
Forward + SDM1 Reverse
50
3
Forward + SDM1 Reverse
52.7
4
Forward + SDM1 Reverse
54.5
5
Forward + SDM1 Reverse
56.4
6
Forward + SDM1 Reverse
60
7
SDM1 Forward + SDM2 Reverse
50
8
SDM1 Forward + SDM2 Forward
52.7
9
SDM1 Forward + SDM2 Reverse
54.5
10
SDM1 Forward + SDM2 Reverse
56.4
11
SDM2 Forward + Reverse
60
12
Empty
13
Empty
14
Empty
15
EMpty
16
Empty
Row #2
Lane Number
Sample
Temperature (°C)
1
Ladder
2
SDM2 Forward + Reverse
50
3
SDM2 Forward + Reverse
52.7
4
SDM2 Forward + Reverse
54.5
5
SDM2 Forward + Reverse
56.4
6
SDM2 Forward + Reverse
60
7
Forward + Reverse
50
8
Forward + Forward
52.7
9
Forward + Reverse
54.5
10
Forward + Reverse
56.4
11
Forward + Reverse
60
12
Empty
13
Empty
14
Empty
15
Empty
16
Empty
Back to top
07/25/13
DNA Extraction from R. eutropha
If DNA template is the problem with the PCR, then one of the following procedures may correct it.
1) Phenol Chloroform extraction using the protocol from July 24, 2013 with the following modification: 2μL of lysozyme added in the 3rd step with the 10% SDS, to encourage membrane disolution
2)Using the MoBio Power Soil DNA extraction kit and included protocol.
Back to top
07/25/13
Transformation of Ca Promoters in pSB1C3
Most plates had growth. Plates are going to be incubated at 37°c  for another night. Tomorrow a library will be made.
Back to top
07/26/13
Transformation of Ca Promoters in pSB1C3
Colonies were larger. Transformant libraries were made of Ca promoters in addition to ter, Adh6, and C. cell Adh. Plates were parafilmed and placed in 37°C incubator to prevent drying out over the weekend. Monday, plates will be streaked, Tuesday, plasmids will be extracted, and Wednesday, plasmids will be sent for sequencing.
Back to top
07/25/13
PCR of BktB for Site-Directed Mutagenesis
Repeated PCR of BktB using primers for sie-directed mutagenesis
Also included one PCR series using the Forward and Revers primers for the whole gene
Used only undiluted templates
Temperature gradient was selected to identify correct temperature for each pair of primers (assuming template is present)
Used wells:
1: T = 50&#176:c
4: T = 52.7&#176:c
6: T = 54.5&#176:c
8: T = 56.4&#176:c
12: T = 60&#176:c
Back to top
07/25/13
Gel of BktB Site-Directed Mutagenesis PCR
Row #1
Lane Number
Sample
Dilution
1
Empty
2
Kb plus ladder
3
Forward + SDM1 Reverse
1:10
4
Forward + SDM1 Reverse
1:100
5
Forward + SDM1 Reverse
1:1000
6
Forward + SDM1 Reverse
1:10000
7
SDM1 Forward + SDM2 Reverse
1:10
8
SDM1 Forward + SDM2 Forward
1:100
9
SDM1 Forward + SDM2 Reverse
1:1000
10
SDM1 Forward + SDM2 Reverse
1:10000
11
SDM2 Forward + Reverse
1:10
12
SDM2 Forward + Reverse
1:100
13
SDM2 Forward + Reverse
1:1000
14
SDM@ Forward + Reverse
1:10000
15
Kb plus Ladder
16
Empty
No PCR products.
Will run a temperature gradient for each sample to see if temperature is affecting enzyme activity
Will preform Power Soil DNA Extraction and bead beaten with phenol: chloroform extraction and isopropanol precipitation to see if template is the problem.
Back to top
07/24/13
Site-Directed Mutagenesis PCR for BktB
Into each tube:
Mix: 98μL
Template: 2μL
Total: 100μL
Dilutions used: 1:10, 1:100, 1:1000, 1:10000
Mix:
2x Master Mix: 250μL
Forward Primer: 3μL
Reverse Primer: 3μL
PCR dH2O: 244μL
Total: 500μL
PCR Primers Used:
Forward + SDM1 Reverse
340 base pairs
SDM1 Forward + SDM2 Revers
660 base pairs
SDM2 Forward + Reverse
300 base pairs
Back to top
07/24/13
DNA Extraction of R. eutropha
Re suspend cells in 500μL 1x STE buffer
Vortex until mixture is homogenous
Add 50μL of 10% Sodium-dodecyl sulfate (SDS)
IF cells insufficiently lysed, 2 minutes in bead beater
Add 500μL of Chloroform:Phenol pH 8
Extract bottom layer (2 phases present)
Minimize time the chloroform is out of fridge
Vortex
Centrifuge for 30 seconds
14000 rpm
Transfer top phase to 1.5mL microcentrifuge tube
Don't take up white precipitant from interphase of solution
Add 500μL of isoamyl acetate (CIAA 24:1) to top phase
Vortex
Centrifuge for 30 seconds
14000 rpm
Transfer top phase to 1.5mL microcentrifuge tube
Don't take up white precipitant from interphase of solution
REPEAT STEPS 9-12
Note: Keep track of amount finally transfered in step 12
Add (2/10) volume (amount of top phase) of 3M Sodium Acetate
Add (7/10) volume (step 14 amount) of Isopropanol
Note: Mix solutions with pipette
Centrifuge for 25 minutes
14000 rpm
Dispose of liquid portion
Don't disturb invisible pellet
Dry pellet on heat block at 60°C for 10 minutes
Leave caps open for evaporation of isopropanol
Re suspend pellet in 50μL of distilled water
Back to top
07/24/13
Repeat of Transformation of Ca Promoters into Top 10 Cells
Preformed as outlined on February 11, 2013
-Last time we used 2μL for transformation this time since our ligations will contain a maximum of 100ng of insert (recommended amount for transformation of top 10 cells), we are going to use the entire ligation volume (20μL)
Back to top
07/23/13
Making Beef Extract Medium
Component
Amount
Beef Extract
3g
Peptone
5g
Agar
15g
Distilled Water
1000mL
Pour components and one magnetic mixer into empty jar. Fill jar with distilled water to 1000mL mark.
Autoclave at setting number "2" for 20 minutes at slow exhaust
Back to top
07/23/13
PCR of Ca AdhE
Component
Amount
2X Master Mix
125μL
Forward Primer
4μL
Reverse Primer
4μL
PCR Water
117μL
TOTAL
250μL
1 Reaction 50μL = 49μL Mix + 1μL Template
Back to top
07/23/13
Digest of Cc Adh and Cs Adh
500ng DNA for each reaction
500ng  (1&#956L⁄39.7ng) = 12.5 μL Cc DNA
500ng  (1&#956L⁄36.0ng) = 14 μL Cs DNA
Component
Cc
Cs
pSB1C3
10X Buffer
2μL
2μL
2μL
EcoR1
2μL
2μL
2μL
Pst1
2μL
2μL
2μL
DNA
12.5μL
14μL
2.5μL
PCR Water
1.5μL
0μL
11.5μL
Total
20μL
20μL
20μL
500ng  (1&#956L⁄202ng) = 2.5μL pSB1C3 backbone
Incubate at 37°C water bath for 45 minutes
Back to top
07/11/13
Repeat of Cohesive End CLoning of gBloeu Ca Promoters Ptb, AaDc, and thl into pSB1C3 Vector
1. Digest
A. In 1 tube contain
Backbone: 2μL
EcoR1: 1μL
Pst1: 1μL
10x Buffer: 2μL
Water: 14μL
Total: 20μL
B. In each 6 tubes combine
gBloeu Resuspension: 10μL
EcoR1: 1μL
Pst1: 1μL
10x Buffer: 2μL
Water: 6μL
Total: 20μL
Incubate in 37°C water bath for 1 hour
Clean with QIAquick PCR Purification Kit
Nano drop
Aadc quantification appears to be good with a reading of 1.9ng/μL, but is the lowest limit of detection for the nano drop 1000, so we can't be sure
Ligation
Since the vector is 31.7 ng/μL and the protocol calls for 50ng of DNA and the concentration of insert might be 1.5 to 2.0ng/μL due to the need of 20ng.
pSB1C3: 2μL
Promoter: 13μL
Buffer: 2μL
Ligase: 1μL
Water: 2μL
Total: 20μL
Incubate at room temperature overnight
Back to top
07/23/13
Gel of AdhE and AdhE1
Row #1
Lane Number
Sample
Dilution
Temperature(μ)
1
Ladder
2
Ca AdhE
1:100
53
3
Ca AdhE
1:100
50
4
Ca AdhE
1:100
56
5
Ca AdhE
1:100
60
6
Ca AdhE
1:10
50
7
Ca AdhE
1:10
53
8
Ca AdhE
1:10
56
9
Ca AdhE
1:10
60
10
Ca AdhE
1:1000
50
11
Ca AdhE
1:1000
53
12
Ca AdhE
1:1000
56
13
Ca AdhE
1:1000
60
14
Empty
15
Empty
16
Empty
Row #2
1
Ladder
2
Ca AdhE1
1:100
50
3
Ca AdhE1
1:10
50
4
Ca AdhE1
1:100
53
5
Ca AdhE1
1:100
50
6
Ca AdhE1
1:100
56
7
Ca AdhE1
1:10
53
8
Ca AdhE1
1:10
56
9
Ca AdhE1
1:10
60
10
Ca AdhE1
1:1000
50
11
Ca AdhE1
1:1000
53
12
Ca AdhE1
1:1000
56
13
Ca AdhE1
1:1000
60
14
Empty
15
Empty
16
Empty
Back to top
07/22/13
PCR of Ca AdhE and AdhE1
Sample
Dilution
50°
53°
56°
60°
Empty
Ca AdhE
1:10
✓
✓
✓
✓
Ca AdhE
1:100
✓
✓
✓
✓
Ca AdhE
1:1000
✓
✓
✓
✓
Empty
Ca AdhE1
1:10
✓
✓
✓
✓
Ca AdhE1
1:100
✓
✓
✓
✓
Ca AdhE1
1:1000
✓
✓
✓
✓
Back to top
07/22/13
PCR of Ca AdhE/AdhE1 using pSOL
Used Qia Quick Prep Plasmid Extraction Kit
AdhE and AdhE1 are supposed to be located on pSOL plasmid using pSOL as template (1:10, 1:100, 1:1000)
Back to top
07/22/13
Plasmid  Extraction of Ca pSOL
Used Qia Quick Prep Plasmid Extraction Kit
Back to top
07/11/13
Dual Gel of mlsR Xformants
Row #1
Well #
Sample
1
Ladder
2
mlsR 5
3
mlsR 9
4
mlsR 10
5
mlsR 12
6
mlsR 22
7
mlsR 24
Back to top
07/22/13
Dual Digest of mlsR Xformants
EcoR1 and Pst1 Dual Digest of mlsR colonies
5, 9, 10, 12, 22, 24
500ng  (1&#956L⁄112.2ng) = 4.45 μL → 4.5 μL
500ng  (1&#956L⁄124.5ng) = 4.02 μL → 4.0 μL
500ng  (1&#956L⁄117.7ng) = 4.25 μL → 4.5 μL
500ng  (1&#956L⁄162.0ng) = 3.09 μL → 3.0 μL
500ng  (1&#956L⁄188.7ng) = 2.65 μL → 2.5 μL
500ng  (1&#956L⁄206.4ng) = 2.42 μL → 2.5 μL
Colony Number
EcoR1
Pst1
10x Buffer
DNA
PCR dH2O
Total
5
2μL
2μL
2μL
4.5μL
9.5μL
20μL
9
2μL
2μL
2μL
4.0μL
10.0μL
20μL
10
2μL
2μL
2μL
4.5μL
9.5μL
20μL
12
2μL
2μL
2μL
3.0μL
11.0μL
20μL
22
2μL
2μL
2μL
2.5μL
11.5μL
20μL
24
2μL
2μL
2μL
2.5μL
11.5μL
20μL
Incubated at 37°L for 45 minutes
Transformed to 80°L heat block for 1 hour
Back to top
07/22/13
Transformation of Terminators/Promoters
Transforming the following parts from the iGEM distribution for use in shuttle vector
BBa_B1006 from 2013 Plate 3, Well 6D (in C3)
BBa_Bl003 from 2013 Plate 3, Well 4P (in C3)
Punctured wells with pipette tip
Re suspended DNA in 10μL  of PCR water
1μL of DNA added to 50μL of competent cells
100μL of cells  Used 2μL of DNA
Incubated on ice for 30 minutes
Heat shocked cells for 1 minutes
Incubated on ice for 5 minutes
Add 600μL of Psi broth
Incubated at 37°C with shaking for 2 hours
Also Xformed
BBa_K148012 Pveg
BBa_K823000 P?
BBa_K823002 P?
BBa_K823003 Pveg
BBa_K090504 ?
Plated 90μL of undiluted 1:10, 1:100, 1:1000 of BBa_B1006 and BBa_B1003
Back to top
07/22/13
PCR Cleanup of Cs Adh PCR Products
Back to top
07/22/13
Plasmid Extraction of Xformants
mlsR library colonies 5, 9, 10, 12, 22, 24
Extracted using Qiagen plasmid prep kit
Back to top
07/19/13
PCR of Ccel Adh
Preformed PCR to amplify Adh out of C. cellulolyticum
Back to top
07/19/13
PCR of Cb SAdh
Choose lower temperature (50°C) for annealing.
Still have primer dimer, or primers may be sitting down at another place on the genomic DNA
Back to top
07/19/13
Promoter and Transformation Trouble Shooting
No colonies have grown on the plates incubated with transformed Top 10 cells with pSB1C3 and various promoters plated on July 16 2013. We did a nanodrop quantification of the ligationproducts.
While the 260/280 ratios seem high, Dr. wawrick says that they aren’t troubling. However, the amount of DNA quantified is unrealistic. Since, we only started with 100ng of insert and 200ng of vector. We ran a gel to see if what we quantified is DNA and not RNA.
Well 1-Ladder
Well 2-Aadc – RBS
Well 3- thl + RBS
From this gel, it doesn’t look like we actually have any DNA (although there may be some really faint bands near the bottom). This means that the ligation didn’t actually occur (or we would have seen a band at 2000bp + 125-75bp). However, if the ligation didn’t work we would still expect to see 2 bands. One is the size of the vector and the other is the size of the insert. Perhaps, something went wrong in the phenol/chloroform extraction step.
Back to top
07/19/13
Restreak of mlsR + pSB1C3 Top 10
Transferred six colonies from library to individual plates. Numbers 5, 9, 10, 12, 22, and 24 from library.
PCR Purification of July 18, 2013 Cs
Back to top
07/18/13
PCR of C sacch and Cs Adh
Row 1
Lane Number
Sample
Dilution
Temperature (°C)
1
Ladder
2
Cs
Undiluted
55
3
Cs
Undiluted
62
4
Cs
Undiluted
65
5
Cs
1:10
55
6
Cs
1:10
62
7
Cs
1:10
65
8-16
Empty
Row 2
Lane Number
Sample
Dilution
Temperature (°C)
1
Ladder
2
Cb
Undiluted
55
3
Cb
Undiluted
62
4
Cb
Undiluted
65
5
Cb
1:10
55
6
Cb
1:10
62
Cb
Cs
1:10
65
8-16
Empty
Back to top
07/18/13
Gel of Isolated Plasmids
Row 1
Lane Number
Sample
1
Ladder
2
pSB1C3 + mlsR1
3
pSB1C3 + mlsR7
4
pSB1C3 + mlsR13
5
pSB1C3 + mlsR13
6
pSB1C3 + mlsR21
7
pSB1C3 + mlsR23
8
pSB1C3 + mlsR26
9
puc19 + KIVD
Back to top
07/18/13
Dual Digests of Isolated Plasmids (18 July)
500ng of plasmid for digest
puc19 + KIVD
500ng X (μL/516ng) = 0.89μL
pSB1C3 + mlsR21
500ng X (μL/99.4ng) = 5.03μL
pSB1C3 + mlsR13
500ng X (μL/81.0ng) = 6.17μL
pSB1C3 + mlsR1
500ng X (μL/101.0ng) = 4.95μL
pSB1C3 + mlsR7
500ng X (μL/65.1ng) = 7.68μL
pSB1C3 + mlsR23
500ng X (μL/84.1ng) = 5.95μL
pSB1C3 + mlsR26
500ng X (μL/86.1ng) = 5.81μL
Gene
EcoR1
Pst1
10x Buffer
DNA
PCR Water
Total
KIVD
2μL
2μL
2μL
1μL
13μL
mlsR21
2μL
2μL
2μL
5μL
9μL
mlsR13
2μL
2μL
2μL
6μL
8μL
mlsR1
2μL
2μL
2μL
5μL
9μL
mlsR7
2μL
2μL
2μL
7.5μL
6.5μL
mlsR23
2μL
2μL
2μL
6μL
8μL
mlsR26
2μL
2μL
2μL
6μL
8μL
Back to top
07/18/13
Plasmid Prep of Transformed Top10 Cells with pSB1C3 + mlsR and puc19 + KIVD
Back to top
07/17/13
DNA Extraction from Ca, Cs, Cb, and Cc using MoBio Power Soil DNA Extraction Kit
Back to top
07/10/13
C. acetobutylicum Shuttle Vector Design
The following sequences have been verified:
Clos Ori      BBa_J238001
Pthl             BBa_J238002
Paadc         BBa_J238003
Ptb             BBa_J238004
mlsR           BBa_J238005
Use Terminators for test testing
Long artificial terminator (T>90%)
BBa_B1006
39 base pairs
2013 p3w6D in C3
Small artificial terminator (T = 85%)
BBa_B1002
34 base pairs
2013 p5w4D in Ak3
Back to top
07/10/13
1% Agarose Gel Electrophoresis of 9 July PCR Products
Procedure followed as on pg. 50
Row 1
Lane
Sample
Dilution
Temperature (°C)
1
Ladder
2
C. sacc
1:10
50.8
3
C. sacc
1:100
50.8
4
C. sacc
1:1000
50.8
5
mlsR
1:10
50.8
6
mlsR
1:100
50.8
7
mlsR
1:1000
50.8
8
C beij
1:10
50.8
9
C beij
1:100
50.8
10
empty
11
empty
12
empty
13
empty
14
empty
15
empty
16
empty
Row 2
1
Ladder
2
C. sacc
1:10
52.8
3
C. sacc
1:100
52.8
4
C. sacc
1:1000
52.8
5
mlsR
1:10
52.8
6
mlsR
1:100
52.8
7
mlsR
1:1000
52.8
8
C beij
1:10
52.8
9
C beij
1:100
52.8
10
empty
11
empty
12
empty
13
empty
14
empty
15
empty
16
empty
Row 3
1
Ladder
2
C. beij
1:10
54.1
3
C. beij
1:100
54.1
4
mlsR
1:10
54.1
5
mlsR
1:100
54.1
6
mlsR
1:1000
54.1
7
C. sacc
1:10
54.1
8
C. sacc
1:100
54.1
9
C. sacc
1:1000
54.1
10
empty
11
empty
12
empty
13
empty
14
empty
15
empty
16
empty
Row 2
1
Ladder
2
C. beij
1:10
58.8
3
C. beij
1:100
58.8
4
mlsR
1:10
58.8
5
mlsR
1:100
58.8
6
mlsR
1:1000
58.8
7
C. sacc
1:10
58.8
8
C. sacc
1:100
58.8
9
C. sacc
1:1000
58.8
10
empty
11
empty
12
empty
13
empty
14
empty
15
empty
16
empty
Back to top
07/9/13
DNA Extraction for C sacc, C aceto, and C beij
Procedure followed as on pg. 50
Back to top
07/9/13
Gradient PCR of C.  scc, mlsR, and C. beij
Column Number
Temperature (°C)
3
50
5
52.8
6
54.1
10
58.8
Lane Number
Sample
1
C sacc 1:10
2
C sacc 1:100
3
C sacc 1:1000
4
mlsR 1:10
5
mlsR 1:100
6
mlsR 1:1000
7
C beij 1:10
8
C beij 1:100
Vf Preparation
Component
Amount
2x Mastermix
125μL
Forward
2μL
Reverse
2μL
PCR dH2O
121μL
Total
250μL
Back to top
07/3/13
Gel of (2 July) PCR Products
Row 1
Lane
Sample
1
Ladder
2
C. sacc 1:10
3
C. sacc 1:100
4
C. sacc 1:1000
5
C. sacc 1:10000
6
Negative
7
mlsR  1:10
8
mlsR 1:100
9
mlsR 1:1000
10
mlsR 1:10000
11
Negative
12
ladder
13
empty
14
empty
15
empty
16
empty
Row 2
1
Ladder
2
C. sacc 1:10
3
C. sacc  1:100
4
C. sacc 1:1000
5
C. sacc 1:10000
6
Negative
7
C. beij 1:10
8
C. beij 1:100
9
C. beij 1:1000
10
C. beij  1:10000
11
empty
12
empty
13
empty
14
empty
15
empty
16
empty
Back to top
07/2/13
PCR of mlsR and Redo of C. sacc and C. beij PCR
Back to top
07/2/13
Ralstonia eutropha DNA Extraction
Due to previous low extraction efficiency
Attempted to extract higher [DNA]
For 5mL tubes, each containing 3mL of meat extract broth inoculated with R. eutropha cells, which were in stationary phase
1mL aliquots transferred stepwise to two 15mL epi tubes, spinning at 14,000 rpm for five minutes to pellet cells
Supernatant discarded and additional 1mL aliquot added, spun until all cells pelleted
Cells re suspended in 500μL if 1x STE buffer in each of the two epi tubes
Vortex
50μL of 10% SDS added to each tube
500μL  of TE saturated phenol pH8 added to each tube
Vortex well and centrifuge for 30 seconds
Aqueous recovered
Phenol extraction
Steps 7-9 repeated
500μL chloroform added to each tube and vortexed
Aqueous discarded
Chloroform extraction repeated
400μL split into each of four tubes
100μL of 3M NaOH added to each tube
350μL isopropanol added to each tube
Centrifuge at 14000 rpm for 25 minutes
Liquid discarded
Tubes places on 80°C heating block for 15 minutes
Contents of each tube re suspended in 500μL of PCR dH2O
Back to top
07/2/13
Gel Electrophoresis of PCR Product (1 July)
Lane
Sample
1
Ladder
2
C. sacc Undiluted
3
C. sacc 1:10
4
C. sacc 1:100
5
Negative
6
C. cell undiluted
7
C. cell 1:10
8
C. cell 1:100
Lane
Sample
1
Ladder
2
Negative Control
3
1:10 Diluted C. beij
4
C. beij 1:100
5
C. beij 1:1000
6
C. beij 1:10000
7
Empty
8
Empty
Back to top
07/1/13
16S Gel Repeat
16S gel of C. acetobutylicum, R. eutropha, C. sacchaolyticum, and C. beijerinki to confirm wheather or not we have what we think we have
16S Gel
Lane Number
Sample
1
Ladder
2
C. beijerinki
3
R. eutropha
4
C. sacchrolyticum
C. acetobutylicum
Back to top
06/28/13
1:4 Loading Dye
1 part loading dye
2 parts 40% glycerol
2 parts PCR DI water
Back to top
06/27/13
DNA Extraction of R. eutropha, C. saccharolyticum, and C. beijerinki
Procedure performed as outline on pg. 42
16S PCR of C. acetobutylicum, R. eutropha, C. sacchrolyticum, and C. beijerinki to confirm wheather or not we have what we think
PCR Solution
Component
Amount (μL)
2x PCR Mastermix
350
Forward primer (27F)
Reverse primer (1525R
PCR water
344
Total
700
50μL for each reaction, three reactions for each organism with different template dilutions
Undiluted
1:10
1:100
Place 1μL template into each tube
Negative control won’t have a template
Row Number
Colum Number
Sample
3
7
Negative Control
4
4
C. beijerinki
4
5
C. beijerinki 1:10
4
6
C. beijerinki 1:100
4
7
C. sacchrolyticum
4
8
C. sacchrolyticum 1:10
4
9
C. sacchrolyticum 1:100
5
4
C. acetobutylicum
5
5
C. acetobutylicum 1:10
5
6
C. acetobutylicum 1:100
5
7
R. eutropha
5
8
R. eutropha 1:10
5
9
R. eutropha 1:100
Gel of PCR product
Row Number
Colum Number
Sample
1
1
Ladder
1
2
C. beijerinki
1
3
C. beijerinki 1:10
1
4
C. beijerinki 1:100
1
5
C. sacchrolyticum
1
6
C. sacchrolyticum 1:10
1
7
C. sacchrolyticum 1:100
2
1
Ladder
2
2
C. acetobutylicum
2
3
C. acetobutylicum 1:10
2
4
C. acetobutylicum 1:100
2
5
R. eutropha
2
6
R. eutropha 1:10
2
7
R. eutropha 1:100
2
8
Negative Control
We have bands of the correct length for Cb, Cs, and Ca, they are too faint for sequencing. So, we are going to redo the PCR without template dilutions and do 35 cycles instead of 30 cycles
Back to top
06/25/13
Modified VM Medium
C. cellulolyticum may be grown at 37°L
VM medium to reduce precicpitation
AEM: 77, 2727-2733 (2011)
Modified ? (per liter)
Component
Amount
KH2PO4
1g
KH2PO4
3.4g
Urea
2.14g
MgCl2 - 6H2O
1g
CaCl2 - 2H2O
0.15g
FeSO2 - 6H2
1.25g
MOPS 3-(N-morpholino)propanesulfuric acid
10g
Rezazurin
2mg
Vitamin solution
10mL
Yeast Extract
2g
Oligoelement solution
1mL
Cysteine – HCl
1g
Cellobiose
5.1345
100x Vitamin Solution
Component
Amount
Biotin
0.08μM
Pyridoxamine
0.02μM
Cyanocobalamine
0.001μM
p-aminobenzoic acid
0.15μM
Thiamine
0.9μM
L-alaning
0.22μM
1000x Oligoelement Solution (per liter)
Component
Amount
FeSO4 - 7H2O
5g
ZnSO4 - 7H2O
1.44g
MnSO4 - 7H2O
1.12g
CuSO4 - 5H2O
0.25g
Na2B4O7
0.2g
(Mo)7(NH4)6O24 - 4H2O
1g
NiCl2
0.04g
CoCl2
0.02g
HBO3
0.03g
Na2SeO3
0.02g
10M HCl
50mL
VM Medium placement
100mL in bottles
20mL in vials
Back to top
06/25/13
Re constituting Strains in Anaerobic Chamber
Clean glass vial with acetone
In aerobic Chamber:
Chemwipes
Sterile Needles
Bring into chamber
Heavy duty aluminum foil to work over
Glass cutter
File
Cycle chamber airlock; twice
Clean gloves and chamber with 1N HCl (x2)
???????????????????
Use sterile syringe to add 0.5mL media to inner vial
Mix with syringe
Add 0.5 of inoculums to each of the two media vials
Transfer an inoculum from these vials to a second vial to get one heavy and one light inoculum
Back to top
06/24/13
Media Preparation for C. acetobutylicum
Media components were added to a 2000mL Erlenmeyer flask
0.1mL of 1g/L Rezarin was added to the medium and was degassed under N2 for 45 minutes
Back to top
06/21/13
Media Preparation for C. acetobutylicum
C. acetobutylicum will grow on defined media:
AEM 50:1238 (1985) or the following
Dr. Tanner’s recipe of C .acetobutylicum (per liter)
Component
Amount
Yeast Extract
1g
Glucose/Dextrose
20g
RST minerals
10mL
RST vitamins
10mL
RST metals
5mL
RST Minerals (per liter)
Component
Amount
NaCl
80g
NH4Cl
100g
KCl
10g
KH2PO4
10g
MgSO4 - 7H2O
20g
CaCl2 - 2H2O
4g
RST Minerals (per liter)
Component
Amount
Nitriloacetic Acid
2g
MnSO4 - H2O
1g
Fe(NH4)2(SO4)2 - 6H2O
0.8g
CoCl2 - 6H2O
0.2g
2nSO4 - 7H2
0.2g
CuCl2 - 2H2O
0.02g
NiCl2 - 6H2O
0.02g
Na2MoO4 - 2H2O
0.02
Na2SeO4
0.02g
Na2WO4
0.02g
RST Vitamins
Components
Amount
Light Sensitivity (Y/N)
Pyridoxine HCl
10mg
Y
Thiamine HCl
5mg
Y
Riboflavin
5mg
Y
Calcium Pantothenate
5mg
Y
Thioctic Acid
5mg
N
p-Aminobenzoic Acid
5mg
Y
Nicotinic Acid
5mg
N
B12 = Cyanocobalamine
5mg
N
Biotin
2mg
N
Folic Acid
2mg
Y
Mercaptopethane Sulfuric Acid
10mg
N
Back to top
06/17/13
Competent pAN1 containing DH5 E. coli
Suspended a unit sized inoculums of pAN1 containing DH5 E. coli in LB with chloramphenicol resistance.
Incubated overnight at 37°C and at 220 rpm for 2.5 hours
One culture has an OD600 of 0.38 and another is 0.475
Spun at 12000 g for 20 minutes
Discarded supernatant and re suspended in 15mL of TFBl
Incubated on ice for 60 minutes
Centrifuged at 12000 g for 20 minutes
Back to top
06/14/13
Gel of puc19 with KIVD Digest (EcoR1 and Pst1)
Back to top
06/14/13
Digestion of pUC19 containing KIVD with EcoR1-Hf and Pst1-Hf
Component
Amount (μL)
EcoR1
1
Pst1-Hf
1
10x Buffer
2
DNA
2.5 (500ng)
PCR H2
13.5
Total
20
Place in 37°C water bath for 45 minutes
Back to top
06/13/13
Gel of PCR
Row 1
Well Number
Sample
Temperature (°C)
1
Ladder
2
AdhE1
50.8
3
AdhE1
52.8
4
AdhE1
54.1
5
AdhE1
58.8
Row 2
Well Number
Sample
Temperature (°C)
1
Ladder
2
AdhE
50.8
3
AdhE
52.8
4
AdhE
54.1
5
AdhE
58.8
The gel results of this PCR are similar to the last one. The bands indicate smaller than 250 base pairs, which is smaller than the size of the genes we intended to amplify (2000 base pairs).
AdhE = 2589 base pairs
AdhE1 = 2577 base pairs
Back to top
06/12/13
PCR redo of AdhE and AdhE1
Temperature gradient and dilution up to 1:10000
Column Number
Temperature (°C)
3
50.8
5
52.8
6
54.1
10
58.8
Back to top
06/12/13
Plasmid Isolation of pUC19 from BBa_k273006
Back to top
06/10/13
Amplification of AdhE and AdhE1 in C. acetobutylicum
Testing our primers using the template prepared on June/4
Forward primer used (AdhE)
Forward primer sequence
Reverse primer used (AdhE)
Reverse primer Sequence
Forward primer used (AdhE1)
Forward primer sequence
Reverse primer used (AdhE1)
Reverse primer sequence
Preparation of solution 1:
Into two separate 1.5mL centrifuge tubes
Component
Amount (μL)
PCR DI Water
146
2x PCR Mastermix
150
Forward primer
2
Reverse primer
2
Total
300
50μL of solution 1 were pipette into 5 PCR tubes for each solution (AdhE and AdhE1)
Template dilutions were prepared for 1:10, 1:100, 1:1000, and 1:10000
1mL of template was loaded into 2 each of 4 PCR tubes containing 50μL of solution 1
The 5th PCR tube containing solution 1 will serve as negative control
Back to top
06/4/13
Place pxy plates in glove box to degrass
Extracted Ca DNA
Re suspend cells in 500μL 1x STE buffer
Vortex
Add 1/10 volume of 10% SDS
2 minutes in bead beater
Add 500μL TE saturated phenol pH 8
Be sure to go down far enough in bottle to reach phenol
Put directly back in fridge when done
Vortex
Centrifuge for 30 seconds
Recover aqueous phase
Don’t recover white material at interphase
Throw away
Add 500μL Chloroform: Isoamyl acetate
Vortex
Centrifuge for 30 seconds
Recover and throw away aqueous phase
Repeat Steps 8-12
Add 2/10 volumes 3M sodium acetate
Add 7/10 volumes isopropanol
Mix well
Centrifuge for 25 minutes
Remove liquid and don’t disturb invisible pellet
Dry pellet at 60°C in heat block for 10 minutes
Leave caps off to allow isopropanol to evaporate
Re suspend pellet in 50μL of dH2
Back to top
05/31/13
Plasmid Extraction/Purification of Top 10 E. coli  Transformed with pSB1C3 and Clostridial Origin
Digestion as preformed on pg. 47
Gel of Digestion
Row 1
Well Number
Sample
1
Ladder
2
1
3
3
4
5
5
8
6
11
7
12
Row 2
Well Number
Sample
1
Ladder
2
16
3
18
4
20
5
22
6
24
Wells 2 and 3 like like what we need.
Sequencing primers are in the process of being made to comfirm
2 AdhE1 50.8 3 AdhE1 52.8 4 AdhE1 54.1 5 AdhE1 58.8
Row 2
Well Number
Sample
Temperature (°C)
1
Ladder
2
AdhE
50.8
3
AdhE
52.8
4
AdhE
54.1
5
AdhE
58.8
The gel results of this PCR are similar to the last one. The bands indicate smaller than 250 base pairs, which is smaller than the size of the genes we intended to amplify (2000 base pairs).
AdhE = 2589 base pairs
AdhE1 = 2577 base pairs
Back to top
06/12/13
PCR redo of AdhE and AdhE1
Temperature gradient and dilution up to 1:10000
Column Number
Temperature (°C)
3
50.8
5
52.8
6
54.1
10
58.8
Back to top
06/12/13
Plasmid Isolation of pUC19 from BBa_k273006
Back to top
06/10/13
Amplification of AdhE and AdhE1 in C. acetobutylicum
Testing our primers using the template prepared on June/4
Forward primer used (AdhE)
Forward primer sequence
Reverse primer used (AdhE)
Reverse primer Sequence
Forward primer used (AdhE1)
Forward primer sequence
Reverse primer used (AdhE1)
Reverse primer sequence
Preparation of solution 1:
Into two separate 1.5mL centrifuge tubes
Component
Amount (μL)
PCR DI Water
146
2x PCR Mastermix
150
Forward primer
2
Reverse primer
2
Total
300
50μL of solution 1 were pipette into 5 PCR tubes for each solution (AdhE and AdhE1)
Template dilutions were prepared for 1:10, 1:100, 1:1000, and 1:10000
1mL of template was loaded into 2 each of 4 PCR tubes containing 50μL of solution 1
The 5th PCR tube containing solution 1 will serve as negative control
Back to top
06/4/13
Place pxy plates in glove box to degrass
Extracted Ca DNA
Re suspend cells in 500μL 1x STE buffer
Vortex
Add 1/10 volume of 10% SDS
2 minutes in bead beater
Add 500μL TE saturated phenol pH 8
Be sure to go down far enough in bottle to reach phenol
Put directly back in fridge when done
Vortex
Centrifuge for 30 seconds
Recover aqueous phase
Don’t recover white material at interphase
Throw away
Add 500μL Chloroform: Isoamyl acetate
Vortex
Centrifuge for 30 seconds
Recover and throw away aqueous phase
Repeat Steps 8-12
Add 2/10 volumes 3M sodium acetate
Add 7/10 volumes isopropanol
Mix well
Centrifuge for 25 minutes
Remove liquid and don’t disturb invisible pellet
Dry pellet at 60°C in heat block for 10 minutes
Leave caps off to allow isopropanol to evaporate
Re suspend pellet in 50μL of dH2
Back to top
05/31/13
Plasmid Extraction/Purification of Top 10 E. coli  Transformed with pSB1C3 and Clostridial Origin
Digestion as preformed on pg. 47
Gel of Digestion
Row 1
Well Number
Sample
1
Ladder
2
1
3
3
4
5
5
8
6
11
7
12
Row 2
Well Number
Sample
1
Ladder
2
16
3
18
4
20
5
22
6
24
Wells 2 and 3 like like what we need.
Sequencing primers are in the process of being made to comfirm
Back to top
05/30/13
Plasmid Extraction/Purification of Top 10 E. coli  Transformed with pSB1C3 and Clostridial Origin
Purified six tubes of DNA (plasmid)
Quantification with Nanodrop
Digestion
Component
Amount (μL)
EcoR1
2
Pst1
2
10x Buffer
1
DNA
5
Incubate in 37°C water bath for 1 hour
Load all 10μL onto gel
Gel of Digestion
Well Number
Sample
1
Ladder
2
Screen #3
3
Screen #6
4
Screen #10
5
Screen #19
6
Screen #19
7
Screen #26
Note: Screen # corresponds to quantification Id
Since our insert is about 700 base pairs and our vector is about 2000 base pairs. Well 2 looks like what we want. We are going to re streak colonies isolated from plate with screen #3 , make a freezer stock, and take more colonies from screen #10
Freezer stocks of Top 10, pAN1, pIKM1, and p34KM were made by suspending cells in 1mL of 10% glycerol.
Back to top
05/29/13
Made 80 stocks of C. acetobutylicum (2mL of culture into 2mL of glycerol)
Streaked three more Top 10 pSB1C3 and Clostridial Origin plates
Back to top
05/28/13
Inoculated cultures  of C.  acetobutylicum and a transfer was made
Six plates (LB + Chloramphenicol) were streaked with transformation of pSB1C3 and Clostridial Origin in Top 10 E. coli cells in preparation for plasmid extraction.
Back to top
05/23/13
b>
After 824: possible growth Back to top
05/22/13
Re Dual Digest of pSB1C3 and Insert
Procedure as preformed on pg. 38
Gel of pSB1C3 and Insert Digestion
Well Number
Sample
1
Ladder
2
pSB1C3
3
Clostridial Origin
Gel of pSB1C3 and Insert Digestion after Quanitification
Well Number
Sample
1
Ladder
2
pSB1C3
3
Clostridial Origin
Ligation:
Insert, Vector: 21μL
Ligase Buffer: 5μL
Ligase: 3&#956:L
Let reaction sit on bench for 90 minutes
OR
For overnight ligation: 14°C  water bath
Back to top
05/9/13
Repeat of Dual Digest of pSB1C3 and Clostridial Origin Fragment
We’re repeating this procedure, due to that fact that days between a digestion and ligation shows to be ineffective. Hence, a ligation should be completed soon after digestion.
pSB1C3
Component
Amount (μL)
EcoR1
2
Pst1
2
10x Buffer
2
DNA
2.5
PCR H2
11.5
Total
20
Clostridial Origin Fragment
Component
Amount (μL)
EcoR1
2
Pst1
2
10x Buffer
2
DNA
10
PCR H2
4
Total
20
Place in 37°C water bath for 15 minutes
Ran gel
Well Number Sample 1 Ladder 2 Clostridial Origin 3 pSB1C3
Used wrong kit (Spin mini prep). Suppose to use PCR purification kit.
\
Cleaned/purified digests
Clostridial Origin
pSB1C3
Ligation:
5μL 10x Buffer
2μL ligase (quick)
?μL vector
?μL insert
Up to 50μL dH2
[(ng vector X size insert in Kb)/Size vector in Kb] X molar ratio of insert : vector = ng of insert
Since we used the wrong kit, we’re going to precipitate the DNA out (used digestion on May/9)
Phenol Chloroform Protocol pH 8:
Turn on heating block
Add equal amounts of phenol and whatever volume DNA is in (20μL)
Phenol : Pipette below top surface
Vortex
Centrifuge for 30 seconds
Two layers appear
Protein is inactive and at interphase
Pipette out top layer of both and combine aqueous
Need to get rid of phenol because it absorbs same wavelength as does water and interferes with enzyme activity
Add 50μL of CIAA 24:1
Removes Phenol
Vortex
Centrifuge for 30 seconds
Remove top layer (aqueous)
Leave a little behind so not to contaminate aqueous phase with bottom layer
Add 50μL of CIAA 24:1
Vortex
Centrifuge for 30 seconds
Remove top layer (aqueous)
Leave a little behind so not to contaminate aqueous phase with bottom layer
Now we have 50μL of DNA
Add 1/10 volume sodium acetate 3M pH 5.2
Add 7/10 volume (total) of isopropanol
Vortex
Centrifuge for 20 minutes at 14,000 rpm
Dispose of liquid, leaving pellet inside tube
Add 50μL of 10% ethanol to get rid of isopropanol
Centrifuge for 30 seconds
Dispose of liquid
Use heat block to dry
60°C for 15 minutes
Leave tube open
Add 10μL  of PCR water to re suspend pellet
Use 2μL  to quantify via nandrop
Add 1μL Ligase (quick)
Add 1μL 10x Buffer = 10μL reaction
Let reaction sit at room temperature for 2 hours
Back to top
05/9/13
Dual Digest of pSB1C3 and Clostridial Origin Fragment
pSB1C3 = 2000 base pairs
Costridial Origin = 700 base pairs
Want to digest 500ng of backbone and 1:1 ratio of backbone to insert
pSB1C3 linearized backbone = 202.1 ng/μL
500ng of pSB1C3:
500ng x μL/202ng  = 2.5μL
1:1 mole ratio of backbone to insert
500ng x (2000 bp/mol /700 bp/mol)  = 175ng insert
175ng of insert:
175ng x μL/17ng  = 10μL
pSB1C3
Component
Amount (μL)
EcoR1
2
Pst1
2
10x Buffer
2
DNA
2.5
PCR H2
11.5
Total
20
Clostridial Origin Fragment
Component
Amount (μL)
EcoR1
2
Pst1
2
10x Buffer
2
DNA
10
PCR H2
4
Total
20
Place in 37°C water bath for 15 minutes
Back to top
05/9/13
PCR Cleanup of Linearized Backbone of Quantification
Used QIAquick PCR Cleanup Kit and processed pSB1K3 and pSB1A3 linearized plasmid backbone PCR products
Nanodrop Quantification
Back to top
05/9/13
Gel of New Linearized Backbone using Provided Supermix
Well Number
Sample
1
Ladder
2
pSB1K3 (blue Hp) (fail: pipette tip fell into well)
3
pSB1K3
4
pSB1A3 (blue Hp)
5
pSB1A3
Back to top
05/8/13
Linearized Backbone PCR
Dilute plasmid backbones to 10ng/μL
Two reactions for pSB1K3 and pSB1A3
Component
Amount (μL)
PCR High Fidelity Supermix
97
Primer 1: SB prep 3P-1
1
Primer 2: SB prep 2Ea
1
Template
10ng
pSB1K3
10ng x μL/75ng  = 1 : 7.5μL Dilution     10μL : 65μL PCR H2O
pSB1A3
10ng x μL/40ng = 1 : 4μL Dilution     10μL : 30μL PCR H2O
Thermo Cycler  (x38)
95°C for 2 minutes
95°C for 30 seconds
55°C for 30 seconds
68°C for 3 minutes
69°C for 10 minutes
Back to top
02/12/13
stuff stuff stuff stuff
Back to top
05/1/13
PCR Supermix High Fidelity Preparation
To have eight reaction vials with 50μL supermix each. Extra is made for potential pipetting error.
Component
Amount (μL)
Amplification Buffer 10x
50
dNTP Mixture (10 mM)
15
50 mM MgSO4
10
Pfx DNA Polymerase (Hf) (Spin down)
10
dH2O
415
Total (two different backbones)
500
PCR reaction of linearized backbone in each tube
Component
Amount (μL)
PCR Supermix HF
50
SB prep 3P-1
0.4
SB prep 2Ea
0.4
DNA
5ng
pSB1K3
5ng x μL/28ng  =0 .18μL  => 0.2μL
pSB1A3
5ng x μL/135.8ng& = 0.037μL => 0.04μL
For our eight reactions, multiply all values (except supermix) above by 10 and add/mix into 250μL of supermix
Final Reaction Tube Composition
Component
Amount (μL)
PCR Supermix
250
SB prep 3P-1
4
SB prep 2Ea
4
pSB1K3 DNA
50ng = 2μL
pSB1A3 DNA
50ng = 0.4μL
Alliquot 55μL of supermix , primers, and template into each tube
To prepare dNTP mix
10μL of each NYP
60μL of dH2O
PCR for linearized backbone (actual)
Four reaction of 1 backbone
Component
Amount (μL)
10x Buffer
25
Polymerase Hf (spin)
5
Mg
5
dNTP mix
7.5
DI H2O
203
Forward Primer
2
Reverse Primer
2
Template
1 (may need to dilute)
Total
250
To dilute pSB1A3:(135.8 ng/μL)
1μL Template
4μL PCR dH2O
We didn’t get what we were trying to amplify
Back to top Back to top
04/28/13
PCR Reaction with Primer Set #3
Preformed as on pg. 15 with the following modifactions
Dilutions
1:10
1:100
1:1000
95°C for 45 seconds
52°C for 45 seconds
72°C for 2 minutes
Repeat 30x
32ng/μL of DNA template (pIKM1) successfully produced the Clostridial origin
PCR product was digested using QIAquick PCR purification kit
Note: If solution turns pick, the pH needs to be adjusted with 5μL of sodium acetate
Back to top
04/24/13
Anaerobic Media Preparation
For 0.5L
500mL  dH2
15g TSB mix
1g glucose
Adjust pH to 7.3
Place sponge stopper in place
Open silver valve and black valve
Set degassing station to 20 psi
Switch to nitrogen and run for 30 seconds to flush oxygen out of head space
Place largest needle into media and turn on spin located on stir-plate, then set time for 45 minutes
Disinfect top of bottle with alchol wipe
Add sterile wipe filter with new needle
Need steady stream of bubbles
Add 0.1mL Resasrin (will be purple until autoclaved)
Post autoclaved
Pink color signifies media has been exposed to oxygen
No color in media signifies media is anaerobic
Oxygen scavenger
If black precipitant forms after autoclave and before transfer into media bottles, then too much oxygen present.
Use acid washed vials
Place stoppers in ice and water to shrink pores
Flush bottles and vials with nitrogen
Place 35mL in each using automated pipette
Angle stopper with needle (gas still within)
Leave for 10 seconds
Bring needle out of bottle and push stopper in
Crimp top
Add 15mL to vials
Flush head space
Put regular needle in and flushing needle in
Flush for 3 minutes
Pull both needles out at same times so you don’t put pressure on tubes
OR
Switch degassing station to vacuum
Alternate between vacuum and pressure 10 times
Let it settle back to position before going back to pressure
Make sure all needles are in black stopper
Tubes now have 20 psi and poke with needle to vent
Turn off tank
Autoclave immediately
Back to top
04/19/13
PCR of Clostridial Origin with New Primers
95°C for 45 seconds
52°C for 45 seconds
72°C for 2 minutes
Repeat 30x
well 1: 1:10 dilution of template
well 2: 1:100 dilution
well 3: 1:1000 dilution
well 4: negative control
The band we get from the gel is too small to be the origin that we want. We think that we have just been given something other than what we thought we had
Back to top
04/19/13
Making Bulk Amounts of Linearized Plasmid Backbones
PCR Supermix High Fidelity
9.6mL
Primer SB-prep 2Eb
65μL
Primer SB-prep 3P-1
65μL
DNA Template
100ng
pSB1K3
100ng x μL/40ng  = 9.8μL  => 2.5μL + 7.5μL dH2O
pSB1A3
100ng x μL/75ng& = 1.6μL => 1.5μL + 8.5μL dH2O
pSB1C3
100ng x μL/43ng& = 1.6μL => 2.5μL + 7.5μL dH2O
High fidelity aliquots of 100μL for PCR
PCR
95°C for 2 minutes
95°C for 30 seconds
55°C for 30 seconds
68°C for 3 minutes
68°C for 10 minutes
PCR Cleanup: Use Quiagen
Back to top
04/9/13
Gel of 8 Apr 2013 Digest Purifications
Row 1
Lane
Sample
1
1 Kb Plus Ladder
2
pIKM1 + RSA1 digest
3
pIKM1
4
pAN1 + RSA1 digest
5
pAN1
85 Volts for 1 hour
Back to top
04/8/13
Verification of pIKM1/pANi
Because we may have accidently stimulated the plasmid and mislabeled them previously. Since, no digestion of the plasmid pIKM1 from last weeks experiment is consistent with a methylating plasmid, today we are digesting both plasmids and running a gel.
This should verify that either we mixed up our plasmids, or we received the wrong plasmid.
1) Plasmid preparation of pAN1 and pIKM1 via Qiagen QiaPrep Spin Miniprep Unit instructions
2) Nanodrop quantification of plasmid DNA
3) Digest of pIKM1 and pAN1 with RSA1
pAN1
500ng;times;μL/51ng  = 9.8μL  => 10.0μL
pIKM1
500ng;times;μL/320ng& = 1.6μL => 2μ
Compnents
pAN1
pIKM1
DNA
10μL
2μL
RSA1
2μL
2μL
10x Buffer
2μL
2μL
PCR Water
6μL
14μL
TOTAL
20μL
20μL
Placed in 42°C water bath for 15 minutes
Back to top
04/5/13
Row 1
Lane
Sample
1
1 Kb Plus Ladder
2
pSB1C3 + RSA1 + Xba + Invitrogen Buffer
3
pSB1C3 + RSA1 + Xba + Fermentas Buffer
4
pSB1C3 + RSA1 + Xba + Invitrogen Buffer + Fermentas Buffer
5
pSB1A3 + RSA1 + Xba + Invitrogen Buffer
6
pSB1A3 + RSA1 + Xba + Fermentas Buffer
7
pSB1A3 + RSA1 + Xba + Invitrogen Buffer + Fermentas Buffer
8
PIKM1 + RSA1
9
pSB1K3 + Hind1 + Xba
10
Negative Control
11
PIKM1
12
pSB1K3
Back to top
04/3/13
Since we couldn’t find information about the Fermentas buffer needed for Xba1 and Hind3, we are going to set up our experiment in such a way that tests to see if using only the buffer from Fermentas, using the buffer from Invitrogen, or using a 1:1 mixture of the two  will allow our digestion to occur in reactions where enzymes from different companies are being used.
Tubes will be labeled with the plasmid name, which enzyme used, and which buffer was used. For example
pSB1C3
RSA1 and Xba1
2μL of Invitrogen buffer
pSB1C3
RSA1 and Xba1
2μL of Fermentas buffer
pSB1C3
RSA1 and Xba1
1μL of Fermentas buffer and 1μL of Invitrogen buffer
We incubated digests in 42°C water bath for 15 minutes
Back to top
04/1/13
Restriction Digest of Biobrick and PIKM1 with AluI
PPIKM1
5μL Plasmid
2μL RSA1
2μL 10x Buffer
11μL PCR Water
Biobrick
5μL Plasmid
2μL RE1(XbaI)
2μL RE2(Hind 3 or RSA1
2μL 10x Buffer
9μL PCR Water
Use a gel of 2% agarose
0.8g of agarose
40mL of TAE 1x
Regarding a digest, you typically want to match the company that produced the enzyme you’re using to the same company for the buffer. If they’re not the same, look up components and concentrations.
Back to top
03/28/13
This looks to be the same size no matter the temperature. We hypothesize that either our primers are laying down non-specifically or they are oriented in the wrong direction. If it turns out our primers are fine, then we may have a different organis, than expected.
Back to top
03/27/13
PCR repeat of Clostridial origin repeat
Preformed as outline on page 15 with the following modifications
Only a 1:100 dilution was used since our DNA template had a concentration of 164.6 ng/μL, which is a good value to use for genomic DNA.
A temperature gradient for the annealing step was setup where different columns in thermocycler are a different temperature during the annealing process.
Column
Temperature (μC)
2
50.2μC
3
50.8μC
4
51.7μC
5
52.8μC
6
54.1μC
7
55.4μC
8
56.7μC
9
57.9μC
10
58.8μC
11
59.5μC
To set up temperature gradient on thermocycler
Highlight stage of interest (in this case; annealing)
Selection "options"
Select "Show Gradient"
Back to top
03/25/13
Gel of Clostridial Origin PCR Product
PCR analysis was completed March 13, 2013
well 1:Ladder
well 2:10 Dilution
well 3:100 Dilution
well 4:1000 Dilution
well 5:10000 Dilution
well 6:Control
From these gels, we can see a band between 250 and 500 bases, which isn't the size of our clostridial origin. Assuming PCR worked correctly, we should see a band of approximately 1200 base pairs. Since we ran multiple cells with the same result, we are hypothesizing that an error was in the PCR reaction. Therefore, we are going to repeat PCR before we move forward by adjusting the annealing temperature.
Back to top
03/15/13
Quantification of PCR Product and Linearized Vector with Chloramphenicol
Nanodrop Protocol
Click icon with ND-1000 on computer
Click "Nucleic Acids"
Wipe off pedestal with chemwipe
Load 3°L DI water to initialize and click blank
Click "Measure" to verify flat line
Load 3°L of sample
Click "Measure"
Click "Print" screen after sample ID has been typed in
Back to top
03/13/13
Making and Preparing Agarose Gel
In a 125mL erlenmyer flask, combine 0.4g ultra pure agarose and 40mL 1x TAE buffer, which makes a 1% gel
Swirl to mix. Place in microwave for 40 seconds. If all agarose isn't dissolved, heat again in 7 second increments
Run flask bottom under water to cool agarose
Pour into gel rig with comb inserted
Let gel cool until opaque
Running Gel of Post-Digested pSB1C3 and pSB1A3
Move gel into gel rig container, pour 1x TAE buffer until it covers the gel surface
Remove comb slowly
Mix 5-10μL of digest with 3μL of 1:4 EZ Vision dye (Note: if using undigested DNA, only use 2μL
Load samples into wells after 1 kb DNA ladder is loaded into far left lane
well 1:Ladder
well 2:pSB1C3 EcoR1
well 3:pSB1C3 EcoR1 Pst1
well 4:pSB1C3 Pst1
well 5:pSB1A3 EcoR1
well 6:pSB1A3 EcoR1 Pst1
well 7:pSB1A3 Pst1
well 8:pSB1C3 UNDIGESTED
Run gel for 45 minutes-1.5 hours at 85 volts
Run gel for 45 minutes-1.5 hours at 85 volts
PCR Reaction Protocol
Combine the following
150μL 2x master mix (polymerase,buffer)
1μL forward primer
1μL reverse primer
148μL DI water (PCR water, UV prior to use)
Make 1:10, 1:100, 1:1000, 1:10000 dilutions of template
In four tubes, combine 50μL of step 1 solution and 1μL of diluted template
in fifth tube, only put in step 1 solution as a negative control
Regular PCR Cycle
95°C for 45 seconds
55°C for 1 minute
75°C for 1.5 minutes
These three steps are cycled 35 times
Back to top
03/12/13
Preformed the following restriction digest of pSB1A3 and pSB1C3.
500ng DNA in 20μL
pSB1A3 [DNA] = 135.8ng/μL
pSB1C3 [DNA] = 74.7ng/μL
500ng x μL/135.8μg = 3.68μL
500ng x μL/74.7μg = 6.69μL
Sample
EcoRI
PstI
10X Buffer
DNA
PCR Water
TOTAL
pSB1A3
2μL
0μL
2μL
4μL
12μL
20μL
pSB1A3
2μL
2μL
2μL
4μL
10μL
20μL
pSB1C3
2μL
0μL
2μL
7μL
9μL
20μL
pSB1C3
2μL
2μL
2μL
7μL
7μL
20μL
Back to top
03/8/13
Nanodrop DNA Quanitification
Protocol
Open Nanodrop 7000 V.6.0
Select Nucleic Acid
Blank via 3μL of PCR water
Verify 0ng/&#;L DNA in PCR water
Load 3μL of sample
Record DNA concentration in ng/μL
Print Screen
pSB1K3
p34KM
pIKM1
pSB1A3
pSB1C3
Back to top
02/13/13
Colonies were counted
1:1000 dilution of AmpicillinR pSB1A3 wasn't properly plated
Dilution
Plasmid
Colony Count
Cells/μg DNA
1:1000
pSB1C3
58
8.24e6
1:1000
pSB1K3
79
1.12e7
1:1000
p34KM
157
2.22e7
1:100
pSB1A3
1521
2.16e7
Note: 1:100 estimates by counting colonies in 1/3 of plate and multiplying by 3
Back to top
02/12/13
Plates were removed from 37°C incubator, wrapped in parafilm, and stored in 4°C refrigerator overnight
Back to top
02/13/13
pSB1C3
Dilution 1:1000
58 Colonies
Back to top
02/11/13
Transforming Competent Cells
We transformed TOP10 chemically competent cells using plasmids.
Resistance
Plasmid ID
Original Concentration (ng/μL)
Kanamycin
pSB1K3
62
Kanamycin
p34KM
40
Chloramphenicol
pSB1C3
43
Ampicillin
pSB1A3
75
Protocol
TOP10 competent cells in 100μL alliquots (x5) were thawed on ice and resuspended.
100ng of plasmid were added to cells
Cells were placed on ice for 20 minutes
Cells were transformed to a 42°C waterbath for 60 seconds
After 60 seconds in waterbath, add 600μL of Psi proth IMMEDIATELY to clls
Cells were incubated for 60 minutes at 37°C while shaking at 200rpm
Dilutions of transformation mixture were made at 1:10, 1:100, and 1:1000
50mL of each dilution was plated on an LB + Antibiotic plate
Plates were incubated at 37°C overnight
Plasmids were diluted to 20μL of 10μg/μL
p34KM
10ng/μL×μL/62ng×20μL = 3.2μL (original plasmid concentration) + 16.8μL (dH2O)
pSB1C3
10ng/μL×μL/43ng×20μL = 4.65μL (original plasmid concentration) + 15.35μL (dH2O)
pSB1A3
10ng/μL×μL/75ng×20μL = 2.6μL (original plasmid concentration) + 17.4μL (dH2O)
pSB1K3
10ng/μL×μL/40ng×20μL = 5.0μL (original plasmid concentration) + 15.0μL (dH2O)
Back to top
Buffers for Preparing Competent E. coli
02/4/13
TFB1: pH: 5.8/ Sterile Filter
Chemicals
Concentration (mM)
RbCl
100
MnCl2
50
Potassium Acetate
30
CaCl2
10
Glycerol
15% by Weight
TFB2 ph:6.8 (Use KOH to adjust)/ Sterile Filter
Chemicals
Concentrations (mM)
MOPS
10
RbCl
10
CaCl2
75
Glycerol
15% by Weight
TFB1
Chemicals
Mass (g)
RbCl
3.02965
MnCl2
1.56996
Potassium Acetate
0.74392
CaCl2
0.27806
Glycerol
37.5463
TFB2
Chemicals
Mass (g)
MOPS
0.53398
RbCl
0.30225
CaCl2
2.88320
Glycerol
37.5457
Back to top
Making Antibiotic Stocks
02/1/13
Antibiotics are used to isolate organisms into which plasmids containing antibiotic resistance genes have been transformed.
Make antibiotic plates with the following specs.
Antibiotic
Concentration (µg/mL)
Color Code
Ampicillin
100
Orange
Chloramphenicol
35
Green
Kanamycin
50
Red
Tetracycline
15
Yellow
Stocks should be made at 1000x concentration, so that making 1L of plate medium will require only 1mL of stock antibiotic solution.
Stocks of antibiotics are made at the following concentrations
Ampicillin
100 mg/mL
Chloramphenicol
35 mg/mL
Kanamycin
30 mg/mL
Tetracycline
15 mg/mL
50mL of stock were prepared and split into several 15mL tubes
Stocks should be stored in the refrigerator at 4°C
Back to top
01/25/13
Restreaked TOP10 cells on LB plates for isolation of single colonies.
Back to top
01/23/13
Sealed TOP10 E. coli cells on LB agar plate. Stored in 37°C incubator
Back to top
01/18/13
Poured LB agar plates
Back to top
01/16/13
Making Agar Plates
LB agar is used to grow our stocks of Escherichia coli
Recipe: Per 1000 mL
10g Bacto Tryptone
5g Yeast Extract
10g Sodium Chloride (NaCl)
15g Agarose
Mix Components in 1L of dH2O until dissolved. Spilt 1000mL solution into two flask. Cap flask with aluminum to prevent spilling of solution. Autoclave on slow exhaust for 20 minutes. Keep liquid in 65°C water bath to prevent setting until 18-Jan.
Back to top
Retrieved from " http://2013.igem.org/Team:OU-Norman_OK/Project/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
