Team:Aix-Marseille/Part Collection
MENU
KILL XYL
People
Human Practices
Project
Notebook
Parts
Interlab
Judging form
Part Collection
Our collection of parts is designed to allow the creation of proteins, or phage-like particles, that target a wide range of different Gram-negative bacteria. Each part corresponds to the domains 1 (D1) and 2 (D2) from the p3 protein of filamentous phages targeting the different organisms. The parts in the collection are expected to be used as fusion proteins and so conform to the Rfc25 standard. To have more detail about the design of these parts, and how we used them to make phage-like particles in our project, you can check out our design page .
This part collection contains the following biobricks all in Rfc25 format:
p3_E.coli - BBa_K2255008
p3_N.gonorrhoeae - BBa_K2255009
p3_P.aeruginosa - BBa_K2255010
p3_R.solanacearum_RSM1 - BBa_K2255011
p3_R.solanacearum_RSS1 - BBa_K2255012
p3_V.Cholerae_CTXΦ - BBa_K2255013
p3_V.Cholerae_fs2 - BBa_K2255014
p3_V.Cholerea_VGJΦ - BBa_K2255015
p3_X.campestris - BBa_K2255016
p3_X.fastidiosa - BBa_K2255017
p3_X.fuscans - BBa_K2255018
Usage
These biobricks will be used to engineer phage-like particles targeting different bacteria.  Check out our design page to know more about their design.
Design
The domain 3 (D3) and the signal sequence are the best conserved parts of the innophage attachment protein p3.  Using a global protein sequence alignment (Needleman-Wunsch and MUSCLE alignments ), using two or three sequence at a time, we were able create a good alignment. We used this to determine the domain 1 (D1) and domain 2 (D2) sequence for the attachment proteins of each phages.
We were able to find these domains because they are separated by flexible sequences, often composed of Glycine and Serine [1] .  Then we then retrotranslated these protein sequences to obtain gene sequences and we used the iDT tool to optimise this for production in E.coli.
Pathogene
Filamentous phage
GI
Part ID
Escherichia coli
M13 (fd,ff) [2]
927334
BBa_K2255008
Neisseria gonorrheae
NgoΦ6 [3]
1260906
BBa_K2255009
Pseudomonas aeruginosa
Pf3 [4]
215374
BBa_K2255010
Ralstonia solanacearum
RSM1Φ [5]
5179368
BBa_K2255011
RSS1Φ [5]
4525385
BBa_K2255012
Vibrio Cholerea
CTXΦ [6]
26673076
BBa_K2255013
VFJΦ(fs2) [7]
1261866
BBa_K2255014
VGJΦ [8]
1260523
BBa_K2255015
Xanthomonas campestris
ΦLf [9]
3730653
BBa_K2255016
Xylella fastidiosa
XfasM23 [10]
6203562
BBa_K2255017
Xanthomonas fucans
XacF1 [11]
17150318
BBa_K2255018
Table showing the attachment proteins from various filamentous phages.
↑ Heilpern, A. J. & Waldor, M. K. pIIICTX, a predicted CTXphi minor coat protein, can expand the host range of coliphage fd to include Vibrio cholerae. J. Bacteriol. 185, 1037–1044 (2003).
↑ Smeal, S. W., Schmitt, M. A., Pereira, R. R., Prasad, A. & Fisk, J. D. Simulation of the M13 life cycle I: Assembly of a genetically-structured deterministic chemical kinetic simulation. Virology 500, 259–274 (2017).
↑ Piekarowicz, A. et al. Neisseria gonorrhoeae Filamentous Phage NgoΦ6 Is Capable of Infecting a Variety of Gram-Negative Bacteria. J Virol 88, 1002–1010 (2014).
↑ Luiten, R. G., Schoenmakers, J. G. & Konings, R. N. The major coat protein gene of the filamentous Pseudomonas aeruginosa phage Pf3: absence of an N-terminal leader signal sequence. Nucleic Acids Res 11, 8073–8085 (1983).
↑ 5.0 5.1 T, K. et al. Genomic characterization of the filamentous integrative bacteriophages {phi}RSS1 and {phi}RSM1, which infect Ralstonia solanacearum., Genomic Characterization of the Filamentous Integrative Bacteriophages φRSS1 and φRSM1, Which Infect Ralstonia solanacearum. J Bacteriol 189, 189, 5792, 5792–5802 (2007).
↑ Heilpern, A. J. & Waldor, M. K. pIIICTX, a predicted CTXphi minor coat protein, can expand the host range of coliphage fd to include Vibrio cholerae. J. Bacteriol. 185, 1037–1044 (2003).
↑ Ikema, M. & Honma, Y. A novel filamentous phage, fs-2, of Vibrio cholerae O139. Microbiology 144, 1901–1906 (1998).
↑ Campos, J. et al. VGJφ, a Novel Filamentous Phage of Vibrio cholerae, Integrates into the Same Chromosomal Site as CTXφ. J. Bacteriol. 185, 5685–5696 (2003).
↑ Tseng, Y.-H., Lo, M.-C., Lin, K.-C., Pan, C.-C. & Chang, R.-Y. Characterization of filamentous bacteriophage ΦLf from Xanthomonas campestris pv. campestris. Journal of general virology 71, 1881–1884 (1990).
↑ Chen, J. & Civerolo, E. L. Morphological evidence for phages in Xylella fastidiosa. Virology Journal 5, 75 (2008).
↑ Ahmad, A. A., Askora, A., Kawasaki, T., Fujie, M. & Yamada, T. The filamentous phage XacF1 causes loss of virulence in Xanthomonas axonopodis pv. citri, the causative agent of citrus canker disease. Front. Microbiol. 5, (2014).
