<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aix-Marseille_M13_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aix-Marseille/M13 Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_button" id="display_menu_control">MENU</DIV><UL id="menu_content"><LI><A href="/Team:Aix-Marseille" title="Kill Xyl Homepage">KILL XYL</A></LI><LI><A href="/Team:Aix-Marseille/People">People</A></LI><LI><A href="/Team:Aix-Marseille/Human Practices">Human Practices</A></LI><LI><A href="/Team:Aix-Marseille/Project">Project</A></LI><LI><A href="/Team:Aix-Marseille/Notebook">Notebook</A></LI><LI><A href="/Team:Aix-Marseille/Parts">Parts</A></LI><LI><A href="/Team:Aix-Marseille/InterLab">Interlab</A></LI><LI><A href="https://igem.org/2017_Judging_Form?team=Aix-Marseille">Judging form</A></LI></UL><DIV class="igem_2017_content_wrapper"><DIV id="pageTitle"><H1>M13 Design</H1></DIV><H2><SPAN class="mw-headline" id="Engineering_M13_helper_phage">Engineering M13 helper phage</SPAN></H2><P><A href="/Team:Aix-Marseille/M13" title="Team:Aix-Marseille/M13">M13</A> is a phage that targets <I>E.coli</I>. Our goal is to create an engineered M13 phage-like particle that will be specific to <A href="/Team:Aix-Marseille/Xylella_fastidiosa" title="Team:Aix-Marseille/Xylella fastidiosa"><I>Xylella fastidiosa</I></A>. To do so, we look into the attachment protein of <A href="/Team:Aix-Marseille/M13" title="Team:Aix-Marseille/M13">M13</A>. This protein contains three domains (D1, D2, and D3) and a signal sequence. In filamentous phages, only D1 and D2 are crucial for target attachment while the signal sequence is crucial for the excretion of p3 in the periplasm<SUP id="cite_ref-Heilpern_1-0" class="reference"><A href="#cite_note-Heilpern-1">[1]</A></SUP> and D3 is important for phage formation.
</P><P>In our design, we chose to use M13KO7 from New England BioLab. <B>M13KO7</B> is an M13 derivative which carries the mutation Met40Ile in gII, with the origin of replication from P15A and the kanamycin resistance gene from Tn903 both inserted within the <A href="/Team:Aix-Marseille/M13" title="Team:Aix-Marseille/M13">M13</A> origin of replication.
</P><P>In order to engineer multiple phages to infect various pathogens, we used two strategies. 
</P><P>The first one is to remove D1 and D2 and replace them with the D1 and D2 of <A href="/Team:Aix-Marseille/Xylella_fastidiosa" title="Team:Aix-Marseille/Xylella fastidiosa"><I>X. fastidiosa</I></A>'s filamentous phage<SUP id="cite_ref-2" class="reference"><A href="#cite_note-2">[2]</A></SUP>. Thus we designed the biobrick with the attachment domains for <I>E.coli</I><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255008">BBa_K2255008</A> and <A href="/Team:Aix-Marseille/Xylella_fastidiosa" title="Team:Aix-Marseille/Xylella fastidiosa"><I>X. fastidiosa</I></A><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255018">BBa_K2255018</A>. In our design, we initially wanted to keep the signal sequence and D3 of M13, because there are crucial for the formation of the phage. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">A) Agarose gel in which we can see PCR amplification of M13KO7 before (WT) and after (mut) deletion of p3. As you can see, the mutant has a loss the p3 gene. B) Agarose gel in which we can see the digestion by AvrII and BspI of M13KO7 WT and M13KO7 after mutation (M13KO7AB). As you can see, only the mutant has been digested.</DIV></DIV></DIV><P>To enable the D1-D2 switch, we inserted two restriction sites (AvrII and BspI) in the p3 gene which are compatible with XbaI and AgeI. With this modification, we could remove D1 and D2 from the p3 gene, but we also removed the signal sequence. As we removed it with our construction, we had to put another one. We therefore designed <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255007">BBa_K2255007</A>. We couldn't swap only D1 and D2 because of the need to substitute only 3 nucleotides to create the new restriction site.
</P><P>The second strategy to engineer <A href="/Team:Aix-Marseille/M13" title="Team:Aix-Marseille/M13">M13</A> is to entirely remove the protein III from the phage genome and to reconstruct it in another plasmid. Using this method, we created another part, <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255005">BBa_K2255005</A>, which is the domain involved in the assembly and release of M13 particles.
</P><P>We have succeeded to engineered both design, with de-insertion of AvrII and BspI(M13KO7AB) and the deletion of the whole p3 (M13KO7Δp3). Verification has been done with the sequencing of both designs.
</P><H2><SPAN class="mw-headline" id="Phagemid">Phagemid</SPAN></H2><P>As the M13KO7 genome is only poorly assembled into phage, we decided to use a phagemid to carry a toxin gene and be assembled into our engineered phage-like particles. 
</P><P>We first thought about using the SuperNova toxin (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1491017">BBa_K1491017</A>), made by the iGEM team Carnegie Mellon in 2014. This toxin has multiple benefits: because of the production of ROS occurring only in a yellow-orange light, we could produce phages-like particles carrying this toxin without killing <I>E.coli</I>. However, when this toxin would be produced in <A href="/Team:Aix-Marseille/Xylella_fastidiosa" title="Team:Aix-Marseille/Xylella fastidiosa"><I>X. fastidiosa</I></A> with the light coming from the sun, the bacterium will be harmed, even if it is in the Xylem vessels.
To optimize the production of this toxin in <A href="/Team:Aix-Marseille/Xylella_fastidiosa" title="Team:Aix-Marseille/Xylella fastidiosa"><I>X. fastidiosa</I></A> we tried to find a strong and constitutive promoter in this bacterium.
</P><P>To deliver our toxin, we either created a phagemid that contains the <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1445000">M13 origin</A> (oriM13) which gives it the opportunity to be used in the phage construction. As the part <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1445000">BBa_K1445000</A> didn't work for us, we choose to follow a new strategy to creates our PLPs.
</P><P>We also thought about the used of pBluescript II KS(+) phagemid. Both of those phagemid contain a M13 origin of replication and a gene for antibiotic resistance. We inserted it in <I>E.coli</I> (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K608002">BBa_K608002</A>) or <I>X. fastidiosa</I> (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255004">BBa_K2255004</A>) promoter along with the SuperNova gene.
</P><P>The verification of the concept has been conducted in <I>E. coli</I>, with the Super Nova toxin gene. The Toxin + plate are <I>E. coli</I> illuminated with a 16,5 Candela 595nm Diode (lumex ssl-lx5093syc/g) for 2 hours and the Toxin - plate are the same bacteria but put in the dark for two hours before incubation.
</P><P>This test was conducted with PLPs that contain a pBluescript II KS(+) phagemid. This experiment shows us that PLPs can be used as vehicles in toxin gene delivery. With this in mind, we can imagine that we could use a more powerful toxin gene in order to increase the efficiency of our PLPs.
</P><P>
We made a <A href="/Team:Aix-Marseille/SN" title="Team:Aix-Marseille/SN">Supernova activity test</A> to count the remaining colonies after illumination.
</P><P>An <A href="/Team:Aix-Marseille/Amelioration_part:BBa_K1445000" title="Team:Aix-Marseille/Amelioration part:BBa K1445000">amelioration</A> of the <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1445000">BBa_K1445000</A>part was done : we used the pBlueScript II instead of the BBa_K1445000 because the phagemid encapsidation didn't work.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><OL class="references"><LI id="cite_note-Heilpern-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-Heilpern_1-0">↑</A></SPAN><SPAN class="reference-text">Heilpern, A. J. &amp; Waldor, M. K. pIIICTX, a predicted CTXphi minor coat protein, can expand the host range of coliphage fd to include <I>Vibrio cholerae</I>. J. Bacteriol. 185, 1037–1044 (2003).</SPAN></LI><LI id="cite_note-2"><SPAN class="mw-cite-backlink"><A href="#cite_ref-2">↑</A></SPAN><SPAN class="reference-text">Chen, J. &amp; Civerolo, E. L. Morphological evidence for phages in <I>Xylella fastidiosa</I>. Virology Journal 5, 75 (2008).</SPAN></LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>