<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aix-Marseille_References skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aix-Marseille/References</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_button" id="display_menu_control">MENU</DIV><UL id="menu_content"><LI><A href="/Team:Aix-Marseille" title="Kill Xyl Homepage">KILL XYL</A></LI><LI><A href="/Team:Aix-Marseille/People">People</A></LI><LI><A href="/Team:Aix-Marseille/Human Practices">Human Practices</A></LI><LI><A href="/Team:Aix-Marseille/Project">Project</A></LI><LI><A href="/Team:Aix-Marseille/Notebook">Notebook</A></LI><LI><A href="/Team:Aix-Marseille/Parts">Parts</A></LI><LI><A href="/Team:Aix-Marseille/InterLab">Interlab</A></LI><LI><A href="https://igem.org/2017_Judging_Form?team=Aix-Marseille">Judging form</A></LI></UL><DIV class="igem_2017_content_wrapper"><DIV id="pageTitle"><H1>References</H1></DIV><OL><LI>Salmond, G. P. C. &amp; Fineran, P. C. A century of the phage: past, present and future. Nat Rev Micro 13, 777–786 (2015).</LI><LI>Ikema, M. &amp; Honma, Y. A novel filamentous phage, fs-2, of Vibrio cholerae O139. Microbiology 144, 1901–1906 (1998).</LI><LI>Honey, S., Schneider, B. L., Schieltz, D. M., Yates, J. R. &amp; Futcher, B. A novel multiple affinity purification tag and its use in identification of proteins associated with a cyclin–CDK complex. Nucleic Acids Res 29, e24 (2001).</LI><LI>Menouni, R., Hutinet, G., Petit, M.-A. &amp; Ansaldi, M. Bacterial genome remodeling through bacteriophage recombination. FEMS Microbiol. Lett. 362, 1–10 (2015).</LI><LI>Rutherford, S. T. &amp; Bassler, B. L. Bacterial Quorum Sensing: Its Role in Virulence and Possibilities for Its Control. Cold Spring Harb Perspect Med 2, a012427 (2012).</LI><LI>Buttimer, C. et al. Bacteriophages and Bacterial Plant Diseases. Front. Microbiol. 8, (2017).</LI><LI>Kim, W. S. &amp; Geider, K. Characterization of a Viral EPS-Depolymerase, a Potential Tool for Control of Fire Blight. Phytopathology 90, 1263–1268 (2000).</LI><LI>Tseng, Y.-H., Lo, M.-C., Lin, K.-C., Pan, C.-C. &amp; Chang, R.-Y. Characterization of filamentous bacteriophage ΦLf from Xanthomonas campestris pv. campestris. Journal of general virology 71, 1881–1884 (1990).</LI><LI>Ahern, S. J., Das, M., Bhowmick, T. S., Young, R. &amp; Gonzalez, C. F. Characterization of Novel Virulent Broad-Host-Range Phages of Xylella fastidiosa and Xanthomonas. J Bacteriol 196, 459–471 (2014).</LI><LI>S, K., J, M., A, C. &amp; D, K. Characterizations of highly expressed genes of four fast-growing bacteria., Characterizations of Highly Expressed Genes of Four Fast-Growing Bacteria. J Bacteriol 183, 183, 5025, 5025–5040 (2001).</LI><LI>Pa, V. &amp; Rl, C. Cloning and Expression in Escherichia coli of the Polysaccharide Depolymerase Associated with Bacteriophage-Infected Erwinia amylovora., Cloning and Expression in Escherichia coli of the Polysaccharide Depolymerase Associated with Bacteriophage-infected Erwinia amylovora. Appl Environ Microbiol 51, 51, 862, 862–864 (1986).</LI><LI>Araújo, W. L. et al. Diversity of Endophytic Bacterial Populations and Their Interaction with Xylella fastidiosa in Citrus Plants. Appl. Environ. Microbiol. 68, 4906–4914 (2002).</LI><LI>Chopin, M.-C., Rouault, A., Ehrlich, S. D. &amp; Gautier, M. Filamentous Phage Active on the Gram-Positive Bacterium Propionibacterium freudenreichii. J. Bacteriol. 184, 2030–2033 (2002).</LI><LI>A. Lukyanov, K., O. Serebrovskaya, E., Lukyanov, S. &amp; M. Chudakov, D. Fluorescent proteins as light-inducible photochemical partners. Photochemical &amp; Photobiological Sciences 9, 1301–1306 (2010).</LI><LI>T, K. et al. Genomic characterization of the filamentous integrative bacteriophages {phi}RSS1 and {phi}RSM1, which infect Ralstonia solanacearum., Genomic Characterization of the Filamentous Integrative Bacteriophages φRSS1 and φRSM1, Which Infect Ralstonia solanacearum. J Bacteriol 189, 189, 5792, 5792–5802 (2007).</LI><LI>Hodyra, K. &amp; Dąbrowska, K. Molecular and chemical engineering of bacteriophages for potential medical applications. Arch. Immunol. Ther. Exp. (Warsz.) 63, 117–127 (2015).</LI><LI>Chen, J. &amp; Civerolo, E. L. Morphological evidence for phages in Xylella fastidiosa. Virology Journal 5, 75 (2008).18.Piekarowicz, A. et al. Neisseria gonorrhoeae Filamentous Phage NgoΦ6 Is Capable of Infecting a Variety of Gram-Negative Bacteria. J Virol 88, 1002–1010 (2014).</LI><LI>Vandenbergh, P. A., Wright, A. M. &amp; Vidaver, A. K. Partial Purification and Characterization of a Polysaccharide Depolymerase Associated with Phage-Infected Erwinia amylovora. Appl. Environ. Microbiol. 49, 994–996 (1985).</LI><LI>Heilpern, A. J. &amp; Waldor, M. K. pIIICTX, a predicted CTXphi minor coat protein, can expand the host range of coliphage fd to include Vibrio cholerae. J. Bacteriol. 185, 1037–1044 (2003).</LI><LI>Bevan, M. Plant pathology: The bugs from Brazil. Nature 406, 140–141 (2000).</LI><LI>Czapar, A. E. &amp; Steinmetz, N. F. Plant viruses and bacteriophages for drug delivery in medicine and biotechnology. Current Opinion in Chemical Biology 38, 108–116 (2017).</LI><LI>Ionescu, M. et al. Promiscuous Diffusible Signal Factor Production and Responsiveness of the Xylella fastidiosa Rpf System. mBio 7, e01054-16 (2016).</LI><LI>Smeal, S. W., Schmitt, M. A., Pereira, R. R., Prasad, A. &amp; Fisk, J. D. Simulation of the M13 life cycle I: Assembly of a genetically-structured deterministic chemical kinetic simulation. Virology 500, 259–274 (2017).</LI><LI>Smeal, S. W., Schmitt, M. A., Pereira, R. R., Prasad, A. &amp; Fisk, J. D. Simulation of the M13 life cycle II: Investigation of the control mechanisms of M13 infection and establishment of the carrier state. Virology 500, 275–284 (2017).</LI><LI>Matsumoto, A. &amp; Igo, M. M. Species-Specific Type II Restriction-Modification System of Xylella fastidiosa Temecula1. Appl. Environ. Microbiol. 76, 4092–4095 (2010).</LI><LI>Ryan, R. P., An, S., Allan, J. H., McCarthy, Y. &amp; Dow, J. M. The DSF Family of Cell–Cell Signals: An Expanding Class of Bacterial Virulence Regulators. PLOS Pathogens 11, e1004986 (2015).</LI><LI>Ahmad, A. A., Askora, A., Kawasaki, T., Fujie, M. &amp; Yamada, T. The filamentous phage XacF1 causes loss of virulence in Xanthomonas axonopodis pv. citri, the causative agent of citrus canker disease. Front. Microbiol. 5, (2014).</LI><LI>Simpson, A. J. G. et al. The genome sequence of the plant pathogen Xylella fastidiosa. Nature 406, 151–158 (2000).</LI><LI>Luiten, R. G., Schoenmakers, J. G. &amp; Konings, R. N. The major coat protein gene of the filamentous Pseudomonas aeruginosa phage Pf3: absence of an N-terminal leader signal sequence. Nucleic Acids Res 11, 8073–8085 (1983).</LI><LI>Amari, D. T., Marques, C. N. H. &amp; Davies, D. G. The Putative Enoyl-Coenzyme A Hydratase DspI Is Required for Production of the Pseudomonas aeruginosa Biofilm Dispersion Autoinducer cis-2-Decenoic Acid. J. Bacteriol. 195, 4600–4610 (2013).</LI><LI>Dt, A., Cn, M. &amp; Dg, D. The putative enoyl-coenzyme A hydratase DspI is required for production of the Pseudomonas aeruginosa biofilm dispersion autoinducer cis-2-decenoic acid., The Putative Enoyl-Coenzyme A Hydratase DspI Is Required for Production of the Pseudomonas aeruginosa Biofilm Dispersion Autoinducer cis-2-Decenoic Acid. J Bacteriol 195, 195, 4600, 4600–4610 (2013).</LI><LI>Li, Y. et al. Type I and type IV pili of Xylella fastidiosa affect twitching motility, biofilm formation and cell–cell aggregation. Microbiology 153, 719–726 (2007).</LI><LI>Campos, J. et al. VGJφ, a Novel Filamentous Phage of Vibrio cholerae, Integrates into the Same Chromosomal Site as CTXφ. J. Bacteriol. 185, 5685–5696 (2003).35.Roldão, A., Silva, A. C., Mellado, M. C. M., Alves, P. M. &amp; Carrondo, M. J. T. Viruses and Virus-Like Particles in Biotechnology: Fundamentals and Applications. in Reference Module in Life Sciences (Elsevier, 2017). doi:#1016/B978-0-12-809633-8.09046-4</LI><LI>Wells, J. M. et al. Xylella fastidiosa gen. nov., sp. nov: Gram-Negative, Xylem-Limited, Fastidious Plant Bacteria Related to Xanthomonas spp. International Journal of Systematic and Evolutionary Microbiology 37, 136–143 (1987).</LI><LI>Hopkins, D. L. Xylella Fastidiosa: Xylem-Limited Bacterial Pathogen of Plants. Annual Review of Phytopathology 27, 271–290 (1989).</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>