Team:Aix-Marseille/Experiments/Protocols
MENU
KILL XYL
People
Human Practices
Project
Notebook
Parts
Interlab
Judging form
Protocols
Agarose Gel electrophoresis
1- Preparing the agarose gel
Measure 0.8 g of agarose powder and add 100 mL of TAE 1X
Melt in a micro oven until solution becomes clear
Let it cool; swirl occasionally to cool evenly until warm to the touch
Pour the melted agarose solution into the casting tray and let cool until it is solid
2- Loading the gel
Add enough TAE 0.5 buffer so that there is about 2-3 mm of buffer over the gel
Add 1X loading dye to each PCR reaction
Record the order each sample will be loaded on the gel
Pipette each sample into separate wells in the gel
Pipette 5µL of  DNA ladder standard into at least one well
3- Running the gel
Turn on the power supply to about 135 volts let it run for about 25 minutes
Check to make sure that the current is running in the correct direction by observing the movement of the dye
4- Revelation
Soak into gel red for 10 minutes
Put the gel under UV
Make a copy of the image
BioBrick assembly protocol
Digestion protocol for biobrick assembly
prepare the compounds as mentioned separately in 1.5 mL micro-centrifuge tubes
Upstream Part digest :
Upstream Part plasmid
500 ng
EcoRI-HF
0.5 µL
SpeI
0.5 µL
10X NE Buffer 2.1
5 µL
H2O
up to 50 µL
Downstream part digest :
Downstream Part plasmid
500 ng
XbaI
0.5 µL
PstI
0.5 µL
10X NE Buffer 2.1
5 µL
H2O
up to 50 µL
Destination plasmid digest :
Destination Plasmid DNA
500 ng
EcoRI-HF
0.5 µL
PstI
0.5 µL
10X NE Buffer 2.1
5 µL
H2O
up to 50 µL
Incubate all three-restriction digest reactions at 37 ˚C for 30 minutes and then heat inactivate at 80 ˚C for 20 minutes. Short spin to recuperate water condensation
Ligation protocol for biobrick assembly
Upstream Part digest
2 µL
Downstream part digest
2 µL
Destination plasmid digest
2 µL
10X T4 DNA ligase buffer
2 µL
T4 DNA ligase
1 µL
H2O
up to 11 µL
Incubate at room temperature for 10 minutes  Transform 2 µL of the ligation product into 50 µL of competent bacterial cells
Cloning protocol for IDT sequences
Re-suspending gene fragments
Centrifuge for 5 seconds at 3000 g
Add 20 µL of H2O to the tube for a final concentration of 10 ng/µL
Briefly vortex and centrifuge
Tube can be stored at -20 ˚C
Digest with the restriction endonucleases
Add the following components
Product
Gene Fragment
Vector
DNA
100 ng
500 ng
Buffer
5 µL
5 µL
Restriction Enzyme
1 µL(each)
1 µL(each)
H2O
up to 50µL
up to 50µL
Incubate for 45 minutes at 37 ˚C
Incubate for 20 minutes at 80 ˚C
Gene Fragment and vector digested separately
Ligation
Add the following components in the order listed below
Component
Amount
Gene fragment
ratio 3 to 1
T4 buffer 1X
2 µL
T4 ligase
1 µL
H2O
up to 20 µL
calculate insert to vector quantities by this equation: required mass insert (ng) = desired insert/vector molar ratio x mass of vector (ng) x ratio of insert to vector lengths
Centrifuge for 5 seconds at 3000 g
Incubate at 16 ˚C overnight
Transformation
Reapeat protocol as mentioned previously
Creation of phage-flurescence with carboxyfluorescein
Requirement: 200 µL of Napi 0.5 M - 500 µL of DMSO (diméthyl sulfoxyde) - 100 mg of carboxyfluorescein - 28 mL of PBS 1X
Prepare a solution of 1 mL of purified phages with a concentration of 7x10^8
Dilute 100 mg of carboxyfluorescein in 500 µL of DMSO in a 1.5 mL micro-centrifuge tuve wrap up with foil.
Add 20 µL of fluorophore solution in phages solution
Keep at 4 °C during 1 hour
Transfer the complete solution in a dialyse membrane (cutoff 30 kDa) set in a 15 mL centrifugation tube containing 14 mL of PBS.
Agitate for 1h30 then change the buffer solution. Let it dialyze overnight. The excess of fluorochrome that will not be fixed on phages will pass the membrane.
Gel and PCR Clean up
Wash buffer NT3 preparation
Number of preps
Wash buffer NT3 concentrate
ethanol
10 preps
6 mL
24 mL
50 preps
20 mL
80 mL
250 preps
2 X 50 mL
2 X 200 mL
1- Adjust DNA binding condition/Excise DNA fragment
Mix one volume of sample with 2 volumes of buffer NT1 for PCR sample
Add 200 µL of NT1 buffer for each 100 mg of agarose gel
Incubate sample for 5-10 minutes at 50 ˚C. Vortex if needed to completely dissolve gel
2- Bind DNA
Place PCR Clean up column into collection tube 2 mL
Load up to 700 µL of sample with NT1 buffer
Centrifuge for 30 seconds at 11 000 g
Discard flow through and place column back
3- Wash silica membrane
Add 700 µL of NT3 buffer
Centrifuge for 30 seconds at 11000 g
Discard flow through and place column back
4- Dry silica membrane
Centrifuge for 1 minute at 11 000 g
Discard flow through
5- Elute DNA
Place column into new 1.5 mL micro centrifuge tube
Add 15-30 µL of ddH2O
let it settle for 1 minutes
Centrifuge for 1 minutes at 11 000 g
Note don’t discard flow through
5’- Elute DNA > 1000 bp
Heat ddH2O to 70˚C
Incubate ddH2O on column for 5 minutes
Add 20-30µL of ddH2O
Centrifuge for 1 minute at 30-50 g
Centrifuge for 1 minute at 11 000 g
Repeat elution 2 to 3 times for the best result
Mesure of biofilm density
Add a drop of violet cristal in each well (pipette) and let colore 10 minutes at 30 °C
Vaccum the liquide
Rince twice with water MiliQ
Add 250 µL physiologic water (0.9 % NaCl) + ethanol 60 %
Agitation during 2 minutes
Mesure OD570 nm and OD600 nm plaque bottom TECAN
Agitation during 2 minutes (amplitude 3 mm orbitale)
Mesure OD600 above through the plaque (25 flashes)
The OD600 is proportional to biofilm production
Mini-preparation of plasmid DNA (MiniPrep) protocol
1-Material preparation
Centrifuge Lyse bleu and RNase samples
Add 20 µL of Lyse to 20 mL of Buffer P1
Add 200 µL of RNase to previous mix
This mix is considered as buffer P1
2- Culture preparation
Centrifuge for 2 minutes at 13000 rpm bacterial culture, remove supernatant
Re-suspend pellet with 250 µL of P1 buffer, pipette up-down
Add 250 µL of P2 buffer mix thoroughly by inverting the tubes 4-6 times
Add 350 µL of N3 buffer mix immediately and thoroughly by inverting the tubes 4-6 times
Centrifuge at 9000 rpm for 10 minutes
3- DNA material separation process
Transfer 800µL of supernatant to the spin column that should be in a tube
Centrifuge at 9000 rpm for 45 seconds, discard flow through
Add 0.5 mL of PB buffer
Centrifuge at 9000 rpm for 45 seconds ,discard flow through
Add 0.75 mL of PE buffer
Centrifuge at 9000 rpm for 45 seconds ,discard flow through
Centrifuge at 9000 rpm for 1 minute , discard collection tube
Transfer column to a new 1.5 mL micro-centrifuge tube
Add 50 µL of H2O let stand for 1 minute
Centrifuge at 9000 rpm for 1 minute
Keep flow through, discard column
No special condition while preparation. After adding P2, mix N3 in less than 5 minutes
One-step Sequence and Ligation-Independent Cloning (SLIC)
Procedures
1. Digest the vector with 1 µL/µg of plasmid with restriction enzyme(s) 2 h, and purify the linearized vector with a commercial "clean-up" purification kit. Elute the DNA with UltraPure water. Measure the concentration of the vector.
2. Amplify your gene of interest by PCR using primer with ≥ 15 per homology extension to the linearized vector end. We usually use 15bp homology for single fragment cloning, and 20 bp homology for multiple fragment cloning. Purify the linearized vector with a commercial PCR purification kit. Elute the DNA with elution buffer or 10 mM TrisCl, pH 8.0-8.5. Do not elute the DNA with water or TE. Measure the concentration of the insert(s).
3. Mix the linearized vector and insert at a molar ratio of 1:2 in a 1.5 mL tube. An example is shown as follows. (Vector to insert molar ratio of 1:1 to 1:7 works well, but we usually use 1:2 for single fragment cloning 1:2:2 for multiple fragments cloning. An example of 3 fragment cloning is shown below with vector: insert 1: insert 2 molar ratio is 1:2:2 as shown
Stock concentration
Volume added
Final concentration
Linearized Vector
0.03 pmol/µL
1 µL
0.003 ng/µL
Insert 1
0.15 pmol/µL
1 µL
0.015 pmol/µL
Insert 2
0.15 pmol/µL
1 µL
0.015 pmol/µL
10X NEB Buffer 2.1
10 X
1 µL
1 X
H2O
Up to 9.6 µL
4. Add 0.4 µL of T4 DNA polymerase (3 U/µL; NEB) to the mixture and incubate at room temperature (<27 °C) for 2.5 min.
5. Put the reaction mixture on ice immediately to stop the reaction and incubate on ice for 10 min and thaw chemically competent E. coli cells on ice.
6. Gently mix 100 µL of the competent cells with the reaction mixture and incubate on ice for 40 min.
7. Heat shock the cells at 42 ˚C for 2 minutes.
8. Incubate the cells on ice for 5 min.
9. Add 650 µL of LB broth and transfer the reaction in a 15 mL round-bottom tube.
10. Incubate the cells at 37 ˚C for 1 hour
11. Plate the cells on agar plates containing suitable antibiotics (eg. 100 µg/mL ampicillin).  We usually spread 250 µL of cells onto an agar plate to get optimal number of colonies per plate for single fragment cloning.
12. Incubate the plates at 37 ˚C for 16 hours and analyze the colonies
Phages in soil duration protocol
Preparation :
We need a 2 OD of F+ E. coli :  Dilute bacteria 10x to read and incubate till getting 0.2 DO.  Our phage concentration have to be lower than the bacterial concentration, we’ll use a 2.10⁸ pfu of bacterias. So we will use 10µL of 100x times diluted phage mix : 10⁷ pfu phages per use.
Bacteriophages counting test :
Add 1 mL of bacterias solution + 100 µL of phage 10⁸ pfu/mL solution. Incubate for 1 h at room temperature (25°C) + 300 rpm agitation. Spread 150 µL of mix on Kanamycin plate (LBA) and let it grow on. Count colonies.
Soil preparation :
Pickup ~250 g of soil. Sift it to get only soil and discard rocks and heavier objects : keep only 200 g. Store it in a 500 mL erlenmeyer with a cap that allows air to go through and place it near sunlight to imitate natural conditions. Use the diluted virus mix (1,42 µL of 7.10⁸ phages in 50 mL) and add it to soil erlenmeyer, agitate to mix.
T+ / T- :
T⁺ : soil extract (without phage) + added phage right before incubating T⁻ : soil extract (without phage) T- : Water
Recuperation :
Add 10 g of soil + 20 mL of sterile water in a 50 mL centrifugation tube. Spin 1 500 g : 10 min and transfer supernatant into an ultracentrifugation tube. Spin for 10 min at 10 000 g and transfer supernatant into filter unit (0,22 µm). In 2 1,5 mL tubes, put 500 µL of the filtrated solution and 500 µL of DO=2 GM1 bacteria. Incubate for 30 min at 25 °C with slow shaking. Spin at 5000 rpm for 10 min. Discard 925 µL of supernatant. Resuspend pellet of the first tube. Transfer into the second and resuspend. Spray on kanamycin dish. Incubate overnight.
Polymerase Chain Reaction (PCR)
Reaction setup
Add the following component as listed below
Component
25 µL reaction
50 µL reaction
100 µL reaction
Final concentration
EconoTaq PLUS GREEN 2X Master Mx
12.5 µL
25 µL
50 µL
1x
10 µM Forward primer
2.5 µL
5 µL
10 µL
1 µM
10 µM reverse primer
2.5 µL
5 µL
10 µL
1 µM
DNA Template (10 ng/µL)
Variable
Variable
Variable
ddH2O
To 2.5 µL
To 50 µL
To 100 µL
Assemble all reaction components on ice and quickly transfer the reactions to thermocycle preheated to 94 °C and begin thermocycling
Hint: make a premix of the components mentioned without DNA template and allocate appropriately 12 µL in each pcr tube
Thermocycling conditions for a routine PCR
Cycling step
Temperature
Time
Number of cycle
Initial Denaturation
94 °C
2 minutes
1
Denaturation
Annealing
Extension
94 °C
55 °C
72 °C
20 seconds
20 seconds
1 minute/Kb
29
Final extension
72 °C
7 minutes
1
Hold
16 °C
Indefinitely
1
Thermocycling conditions for a "Rolling-Circle" PCR
the pfu DNA polylerase is used in this case with adapted composition of the reaction mix.
Cycling step
Temperature
Time
Number of cycle
Initial Denaturation
94 °C
2 minutes
1
Denaturation
Annealing
Extension
94 °C
55 °C
68 °C
20 seconds
20 seconds
2 minute/Kb
29
Final extension
72 °C
7 minutes
1
Hold
16 °C
Indefinitely
1
Preparation of competent Bacterial cells
Recuperate the overnight bacterial culture
Determine the OD600 in a 1 mL Tank Spectro, dilute a 100 µL of culture in 900 µL of distilled LB
In 1 L Erlenmeyer add 200 mL of LB and an appropriate volume of culture to have an OD600 = 0.1
Put the Erlenmeyer in an incubator for about an hour at 37 ˚C.
Test the OD600 for the new culture
The OD600 should be between 0.4 and 0.6
All the handling done in a 15 cm radius of an open flame for optimal sterility
Preparation of Buffer Tbf1 and Tbf2
Tbf1 buffer
Total volume 80 mL
KAc 1 M
2.4 mL
MnCl2 0.5 M
8 mL
KCl 1 M
8 mL
CaCl2 0.1 M
8 mL
Gly 80 %
15mL
H2O
38.6 mL
Tbf2 buffer
Total volume 8 mL
NaMOPS 0.2 M
400 µL
CaCl2 0.1 M
6 mL
Gly 80 %
1.5 mL
KCl 1 M
80 µL
H2O
500 µL
Some solution aren’t available directly and are prepared by dissolving the solid compound
Volume (mL)
Mass (g)
Molecular weight (g/mol)
MnCl2 0.5 M
200
19.791
197.91
KCl 1 M
200
17.91
74.55
NaMOPS 0.2 M
50
2.0926
209.26
KAc 1 M
50
4.9
98.15
Sterilize the flask prepared after weighing in an autoclave. When preparing Tbf1 and Tbf2 should be in proximity of an open flame
Preparation of Competent Bacterial cells
Transfer the Bacterial culture in 50 mL centrifugation tubes
Centrifuge for 10 minutes at 3500 rpm in cold
Remove the supernatant then re-suspend the pellet in 20 mL of Tbf1 for each centrifugation tube
Poll in all bacterial culture in two centrifugation tubes with 40 mL each
Centrifuge for 5 minutes at 3500 rpm in cold
Remove the supernatant then re-suspend the pellet in 4 mL of Tbf2 for each centrifugation tube
Allocate 220 µL of competent Bacterial cells in each 1.5 micro-centrifuge tube
Instant freeze the 1.5 micro-centrifuge tubes in liquid nitrogen
Conserve the tubes at -80 ˚C
NOTE: all handling done in a cold room. To re-suspend the pellet a hard jerking action applied on the tube. Alternatively, put on wheel for 10 minutes.
Production of large quantities of helper phage
Requirement :
110 mL PEG/NaCl (from powder : 16 g NaCl + 22 g PEG 8000) - 14 mL PBS - 3 mL H-top Agar - LB - 2,5 mL Kanamycin 10 - 1 plate without antibiotics.
Prepare a TG1 starter from glycerol or plate
Overnight
Needed DO = 0,4-0,5 : dilute in LB and put at 37 °C for 20 min In a 1.5 mL micro-centrifuge tube, infect 200 µL bacteria with 10 µL phage solution  Incubate for 30 min without shaking In a 15 mL centrifugation tube, add 3 mL of H-top agar (42 °C) Spread on LBA plate (no antibiotics)
Overnight
Make a 0,4 DO TG1 5 mL starter, pick a small plaque at let it grow for 2 h at 37 °C with shaking in the starter In a 2 L flask : pour the starter and add 500 mL of 2YT (TB) Add 2,5 mL of kanamycin
Overnight
In a 500 mL tube, spin for 45 min at 5000 g (10800 15 min) Recover 400 mL of supernatant in an 1 L erlenmeyer  Add 100 mL of PEG/NaCl  From now, every step has to be on ice or at 4 °C Spin for 1h30 at 5000 g in the 500 mL tubes (10800 g 30 min) Throw away supernatant Resuspend pellet in 8 mL PBS and add 2 mL of PEG/NaCl Vortex for 30 s Put in a 15 mL centrifugation tube Let in ice for 30 min Spin for 30 min at 3300 g  Remove supernatant  Add 5 mL of PBS Resuspend pellet Spin for 30 min at 5000 g (11000 g 10 min)  Add 500µL of chloroform Spin for 30 min at 5000 g Keep supernatant Add 10 µL of chloroform
Storage : 15 % glycerol at -80°C
Transformation protocol
Sensitivity test of competent bacterial cells
Heat up LB-agar in micro oven at 300 W for 19 minutes for 400 mL of LB-agar loosen cap before heating to let steam out
Transfer 80 mL of LB-agar in to Erlenmeyer one for every antibiotic
For 20 mL and spread the LB-agar in petri dishes, let it dry out
Prepare petri dishes:
25µL of competent strain spread with glass beads in all four antibiotic and one without antibiotic as negative test
Antibiotic stock
Desired concentration
Volume of antibiotic per mL of LB-agar
Volume in Erlenmeyer
Ampicillin
25 mg/mL
100 µg/mL
4 µL/mL
320 µL
Kanamycin
10 mg/mL
50 µg/mL
5 µL/mL
400 µL
Tetracycline
15 mg/mL
15 µg/mL
1 µL/mL
80 µL
Chloramphenicol
30 mg/mL
50 µg/mL
1.6 µL/mL
128 µL
Handling is done near an open flame, sterile environment
Preparation of DNA material from iGEM kit
Pierce plate at the desired location
Hydrate by injecting 10 µL of ddH2O
Let it soak for 5 minutes until red coloration is highly visible
pipet up-down to homogenize the solution
Transfer the solution into a 1.5 mL micro-centrifuge tube set it on ice
No special conditions needed when manipulating just respect sterility
Transformation of bacterial cells with plasmid/ligation
Add an adequate quantity of DNA in a 1.5 mL micro-centrifuge tube
Add 100 µL (50 µL) of competent bacterial cells
Put on ice for 20 minutes
Put tubes in thermomixer at 42 ˚C for 45 seconds
Put on ice for 5 minutes
Add 450 µL (900 µL) of LB
Incubate tube for 1 hour at 37 ˚C
Centrifuge for 5 minutes at 5000 rpm (normal centrifuge will work)
Remove 400 µL (850 µL) of supernatant
Re-suspend pellet in the 150 µL of remaining medium pipet up-down
Spread total volume on petri dish with the appropriate antibiotic
Put petri dishes in an incubator overnight at 37 ˚C
For a negative control repeat procedure without adding plasmid
Handling is done near an open flame, sterile environment
