<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aix-Marseille_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aix-Marseille/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_button" id="display_menu_control">MENU</DIV><UL id="menu_content"><LI><A href="/Team:Aix-Marseille" title="Kill Xyl Homepage">KILL XYL</A></LI><LI><A href="/Team:Aix-Marseille/People">People</A></LI><LI><A href="/Team:Aix-Marseille/Human Practices">Human Practices</A></LI><LI><A href="/Team:Aix-Marseille/Project">Project</A></LI><LI><A href="/Team:Aix-Marseille/Notebook">Notebook</A></LI><LI><A href="/Team:Aix-Marseille/Parts">Parts</A></LI><LI><A href="/Team:Aix-Marseille/InterLab">Interlab</A></LI><LI><A href="https://igem.org/2017_Judging_Form?team=Aix-Marseille">Judging form</A></LI></UL><DIV class="igem_2017_content_wrapper"><DIV id="pageTitle"><H1>Notebook</H1></DIV><DIV class="row-icons white-icons"><UL><LI><SPAN class="legend">Protocols</SPAN></LI></UL></DIV><P><DETAILS><SUMMARY>WEEK 1 : 06/06 → 09/06</SUMMARY></DETAILS></P><P>Cleaning the laboratory, then recovery and installation of the equipment.
Starting culture of strains TG1 and DH5α. We launched precultures from cryotubes (sampling with 1 rod and then deposition in an erlenmeyer containing LB media). Incubation at 37 °C. 
</P><H3><SPAN class="mw-headline" id="Strains_cultivation:">Strains cultivation:</SPAN></H3><P>Recovery of strains cultivated the day before: DH5α and TG1.
To check the concentrations, the OD is measured. For doing so, the culture must be diluted because the optimum measurement of the apparatus is between 0.1 and 0.8. The cultures in the tank are diluted to 1/10 with distilled water. 
</P><P><B>Our results :</B> TG1: 5.16 / DH5α: 5.21
</P><P>To calculate the volume to be taken for our 100 mL we do: $$\frac{\text{OD wanted}}{\text{OD obtained before dilution}} \times \text{Volume} = \frac{0.1}{5.16} \times 100 \simeq 2 \text{ mL}$$
</P><P><I>E. coli</I> being aerobic, the solution is stored in LB in a Erlenmeyer flask of 5 × volume, so 500 mL and incubate for 1 h.
</P><H3><SPAN class="mw-headline" id="Glycerolization_of_strains:">Glycerolization of strains:</SPAN></H3><P>Putting the strains in glycerol protect them from the cold, necessary when one will freeze at -80 °C so that the cells do not collapse.
40 % Glycerol in 1.8 mL cryotubes → 0.9 of glycerol and 0.9 of strains.
We will freeze the cells in exponential phases in order to make our cells competent, is able to incorporate DNA.
</P><H3><SPAN class="mw-headline" id="Competent_bacteria:">Competent bacteria:</SPAN></H3><P>To make competent bacteria, it is necessary to take them in exponential growth phase. A quantity of bacterium is thus taken which is placed in LB medium and then incubated for 1 hour at 37 degree Celsius.
An OD value of 0.1 is desired in 100 mL.
For TG1-&gt; 1.94 mL
For DH5α-&gt; 1.92 mL
</P><P>Mesuring OD after 1h40:
TG1 - 0.7
DH5α - 0.5
</P><P>The cultures were thus recovered and were divided into centrifuge tubes (50 ml each) and placed in ice. (For each of the two strains).
</P><P>Preparation of the buffers: Solution TBF1 and TBF2
Preparation of buffer solutions:
KAc: 50 mL → 4.9 g
MnCl2: 200 mL → 19.79 g
KCl: 200 mL → 14.91 g
MOPS: 50 mL → 2.09 g
Each solution was autoclaved.
Then we made <A href="/Team:Aix-Marseille/Experiments/Protocols#Preparation_of_Buffer_Tbf1_and_Tbf2" title="Team:Aix-Marseille/Experiments/Protocols">two solutions necessary for making the competent cells</A>.
</P><P>To make competent cells, we ALWAYS work at low temperature (4 °C) and in sterile medium. The buffers thus prepared were stored in ice.
Centrifugation of the cell culture 10 min at 3500 rpm at 4 ° C. The pellet must then be resuspended (gently) in 80 ml of Tbf1 (20 ml as 50 ml of culture in each centrifuge tube).
3500 rpm centrifugation for 5 minutes. 
</P><P>Then resuspension of the pellet in 2 mL of Tbf2. 
</P><P>Let incubate for 15 minutes in ice and then aliquot 200 μL of bacteria solution in a cryotube and store at -80 ° C.
These aliquots are at I8 for competent DH5 alpha and I9 for competent TG1.
</P><P>The remainder is stored at -20 °C in labeled centrifuge tubes.
</P><P>After the first centrifugation was carried out in a cold (non-sterile) chamber, we re-started precultures from those in the morning: 100 μL of culture were placed in 10 ml of LB medium and then incubated at 37 ° C. overnight.
</P><P>The buffer Tbf1 and Tbf2 have also been prepared to be ready so we can restart the manipets in case of failure.
</P><P>-
</P><H3><SPAN class="mw-headline" id="Preparation_for_transformation_:_In_four_steps">Preparation for transformation : In four steps</SPAN></H3><P>For the petri dish LB agar, take a bottle of LB agar  ( 400 mL for 20 boxes ) and put it in the microwave to liquefy.
Heat with microwave 300 W ( no more ) for 19 minutes with unscrewed plug.
</P><H3><SPAN class="mw-headline" id="Organizational_axis">Organizational axis</SPAN></H3><P>Subjects organisation
</P><TABLE><TBODY><TR><TD style="background-color:#eec6aa;font-weight:bold;"> Killer red
</TD><TD style="background-color:#eee8aa;font-weight:bold;"> Crispr Cas9
</TD><TD style="background-color:#d2eeaa;font-weight:bold;"> P3
</TD><TD style="background-color:#bae1ff;font-weight:bold;"> Génome M13
</TD><TD style="background-color:#c7b1b6;font-weight:bold;"> Measurement
</TD><TD style="background-color:#ffffba;font-weight:bold;"> DEPS
</TD><TD style="background-color:#679187;font-weight:bold;"> QS
</TD></TR><TR><TD> Robin
</TD><TD> Flora/Soraya
</TD><TD> Camille
</TD><TD> Camille/Thibault
</TD><TD> Lisa
</TD><TD> Hussein
</TD><TD> JeremyFlora
</TD></TR></TBODY></TABLE><P>Small theoretical session of Sandra on the manipulations:
</P><H3><SPAN class="mw-headline" id="1._Transformation">1. Transformation</SPAN></H3><P>Transformation is a technic used to make the DNA &quot;enter&quot; into the competent cell by destabilizing the membrane:
</P><OL><LI><B>Contact</B>, the cells and plasmids (which are recovered in the iGEM kit)</LI><LI><B>Heat shock</B>, more permeable membrane that will make the DNA enter: 45 seconds at 42 ° C</LI><LI><B>Expression</B>, cells are allowed to express the inserted genes (including antibiotic resistance which serve as a test). There are two types of antibiotics: bacteriostatic (ampicillin, which freezes growth), bactericidal (which kills)</LI><LI><B>Spread on antibiotic</B> (petri dish), that will eliminate the bacteria which have not assimilated the gene</LI></OL><H3><SPAN class="mw-headline" id="2._Verification_of_contamination">2. Verification of contamination</SPAN></H3><P>Negative control: 3 antibiotics, if the bacteria resist: contaminated
</P><H3><SPAN class="mw-headline" id="3._Cloning">3. Cloning</SPAN></H3><P>We want to integrate an insert, for this, we need the restriction enzymes
</P><H3><SPAN class="mw-headline" id="4._Biobrick_-_iGEM">4. Biobrick - iGEM</SPAN></H3><P>S and X are compatible
</P><P>
B. Re-culture 
</P><P>In order to have uncontaminated crops (cultures - -)  in case the one made yesterday were.
C. Biobrick
Putting GFP into solution from the iGEM Kit (Item 5: 16K): 10 μL of H2O miliQ (pure without DNases or RNAses) are deposited and incubated for 10 minutes. The 10 μL of DNA is recovered taking care to take the edge of the cuvée.
</P><P>Opening of the stopper (in the loom) 19min to 300W MAXIMUM and left at 50 ° C in an incubator.
</P><P>Recovery of antibiotics and dosages. Here are the different products to respect
Ampicillin -&gt; 4 μL / mL Ampi [25mg / mL] -&gt; here 320μL
Kanamycin -&gt; 5 μL / mL Kana [10mg / mL] -&gt; here 400 μL
Tetracycline -&gt; 1 μL / mL Tetra [15mg / mL] -&gt; here at 80μL
Chloramphenicol -&gt; 1.6 μL / mL Chloran [30mg / mL] here 128μL
</P><P>(Re) preparation of competent bacterial cells: Error of the previous day.
</P><P>Recovery of precision and OD measurements from diluted sample to tenth:
OD TG1 = 5.24
OD DH5α = 4.27
</P><P>A quantity of bacterium is then taken which has been placed in an LB medium and then incubated for approximately 1 hour at 37 ° C.
1.9 mL of TG1 in 100 mL of LB
2.3 mL of DH5α in 100 mL of LB
Then measurement of OD TG1 = 0.474 / DH5a = 0.895 (a little high compared to the desired value).
</P><P>To make competent cells, it is imperative to work at low temperature (4 ° C) and in sterile medium.
The buffers thus prepared were kept and the eppendorf tubes must be cold (cold room and ice cube tray). First, 10 min centrifugation at 3500 rpm at 4 ° C. of the two bacterial strains from the culture (OD 0.4-0.6). The pellet should then be gently resuspended in 80 mL of Tbf1 (20 mL in our case because we do not have 200 mL of culture). 3500 rpm centrifugation with cold pendant 5 mn and resuspension of the new pellet in 2 mL of Tbf2. Incubation of 15mn in ice. Manufacture of 220μL aliquots in annotated eppendorfs STERILE and bath tubes in liquid nitrogen to freeze them quickly. Storage at -80 ° C
<DETAILS><SUMMARY>WEEK 2 : 12/06 → 16/06</SUMMARY></DETAILS></P><P>Mini préparation.
</P><P>Before starting
Centrifugate 20μL Blue lyse and 200μL de Rnase in the initial tubes.
Add 20μL of blue lyse to P1 buffer. (Initial buffer quantity : P1 = 20 ml).
Then add 200μL of Rnase 1 to &quot;Buffer P1 + blue lyse&quot;.
The ethanol solution is kept in ice.
Ethanol (96-100%) &quot;add to PE before using&quot; was already added. So we used buffer PE.
</P><P>Starting mini prep : 
Filling eppendorfs with the conserved bacteria.
Order
-Tube 1: C -
-Tube 2: C +
-Tube 3: TD1
-Tube 4: TD2
-Tube 5: TD5
-Tube 6: TD6
-Tube 7: RBS
-Tube 8: ADN
-Tube 9: Eprc
-Tube 10: Epr1
-Tube 11: MB
-Tube 12: KR
-Tube 13: Gfp
</P><P>Vector Digestion:
</P><P>We prepared 6 eppendorf tubes, we add in first the buffer ( with the same cone ) and H2O ( with a second cone ).
Afterward we add the DNA material for a total mass of 200 ng in each tube. When the initial preparation was done, a short spin was necessary. We extracted the enzyme carefully with a small up action on the edge of the enzymes tubes. 
Then carefully inject the enzymes in the eppendorf tubes then mix with an up down action. Then we incubate for half an hour at 37° degrees celsius. 
Afterward, a heat shock is applied at 80°C for 20 minutes to inactivate all enzyme activity. 
</P><P>Plasmid Digestion:
</P><P>We had 4 tubes of plasmid with each a specific antibiotic resistant gene insert. With a total of volume of 45µL in each.
We added 10µL of NE buffer, 41µL of H2O at the end we added our enzymes. ( 2µL for ECORI HF and 2µL for PST1 ).
Then we incubate for half an hour at 37° degrees celsius. 
Afterward, a heat shock is applied at 80°C for 20 minutes to inactivate all enzyme activity. 
</P><TABLE style="border-spacing:0;width:6.4375in;"><TBODY><TR><TD style="border:1pt solid #000000;padding:0.0694in;"> </TD><TD style="background-color:#cccccc;border:1pt solid #000000;padding:0.0694in;color:#000000;"> Volume
</TD><TD style="background-color:#cccccc;border:1pt solid #000000;padding:0.0694in;color:#000000;"> ECORI HF
</TD><TD style="background-color:#cccccc;border:1pt solid #000000;padding:0.0694in;color:#000000;"> SPE1
</TD><TD style="background-color:#cccccc;border:1pt solid #000000;padding:0.0694in;color:#000000;"> XBA1
</TD><TD style="background-color:#cccccc;border:1pt solid #000000;padding:0.0694in;color:#000000;"> PST1
</TD><TD style="background-color:#cccccc;border:1pt solid #000000;padding:0.0694in;color:#000000;"> Buffer
</TD><TD style="background-color:#cccccc;border:1pt solid #000000;padding:0.0694in;color:#000000;"> H2O
</TD></TR><TR><TD style="background-color:#679187;border:1pt solid #000000;padding:0.0694in;color:#000000;"> M13 cut up
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 3
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 0,5
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 0,5
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> /
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> /
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 2
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 14
</TD></TR><TR><TD style="background-color:#679187;border:1pt solid #000000;padding:0.0694in;color:#000000;"> EPRI cut up
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 3,3
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 0,5
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 0,5
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> /
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> /
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 2
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 13,7
</TD></TR><TR><TD style="background-color:#679187;border:1pt solid #000000;padding:0.0694in;color:#000000;"> EPRC cut up
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 4,5
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 0,5
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 0,5
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> /
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> /
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 2
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 12,5
</TD></TR><TR><TD style="background-color:#679187;border:1pt solid #000000;padding:0.0694in;color:#000000;"> EPRC cut down
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 4,5
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> /
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> /
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 0,5
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 0,5
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 2
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 12,5
</TD></TR><TR><TD style="background-color:#679187;border:1pt solid #000000;padding:0.0694in;color:#000000;"> GFP cut down
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 2,2
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> /
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> /
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 0,5
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 0,5
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 2
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 14,8
</TD></TR><TR><TD style="background-color:#679187;border:1pt solid #000000;padding:0.0694in;color:#000000;"> RBS cut down
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 1,1
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> /
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> /
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 0,5
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 0,5
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 2
</TD><TD style="border:1pt solid #000000;padding:0.0694in;color:#000000;"> 15,9
</TD></TR></TBODY></TABLE><P>Ligation:
</P><P>We prepared 3 tubes for ligation for which we added 2 µL of T4 DNA ligase buffer 10x. Then 11µL of ddH2O, afterward the DNA material ( 2µL upstream 2µL downstream 2µL destination plasmid ). Finally we add T4 DNA ligase and let it incubate at room temperature for 10mn.
When the ligation was done, we transformed 5µL of ligation product into 150µL of bacterial cell. ( Previously mentioned ).
</P><P>
Electrophoresis: 
</P><P>As mentioned previously, for each ligation product we added in an eppendorf tube 5µL of DNA 10µL of H2O, 3µL of loading purple dye. In the gel, the order was EPRI cut up, EPRC cut up, EPRC cut down, DNA ladder, RBS cut down, GFP cut down, M13 cut up.
We let it migrate for than 30 minutes, at 135 volt which was the limit of the migration, we should have put it for only 25.
All fragment under 200 Pb weren't displayed because it had surpassed the migration cap.
For EPRI, EPRC cut up and down, we should have seen one bar for the linear plasmid and not for the digested factors.
</P><P>We also re-did a transformation for RBS which failed previously. This time we used new competent cells and we had a positive result because there were multiple colonies on the gel.
</P><H3><SPAN class="mw-headline" id="Pr.C3.A9paration_of_DNA_and_oligonucleotids:">Préparation of DNA and oligonucleotids:</SPAN></H3><P>We recieved 6 tubes of DNA. (P3RsS, p3VcC, p3VcF, p3VcV, p3Xc and p3Xfa). All these tubes are at 1000ng. To be able to prepare them at 50ng / ul we followed the iDT protocol. Centrifuge the tubes 2 min at 3000rpm Addition of 20ul of H20 Vortex Incubation of 20 minutes at 50 ° C Vortex Short spin centrifugation DNA is stored in the freezer -20 ° C
</P><P>For oligonucleotides: We received the 4 oligonucleotides. To prepare them we followed the recommendations of the producer (Eurogentec). Centrifugation 2 min at 3000rpm Addition of Water to reach a concentration of 100uM
Transformation protocol as mentioned before for TD4, Cas9 and dCas9 with the new competent cells prepared on 13/06.
</P><P>Preparation of competent bacterial cells protocol as mentioned before was done for 100mL of bacterial cells at a OD of 0.478
</P><P>Resuspension of genomic material from the iGEM plates #4 at the positions as mentioned : 2B, 2D, 2F, 2H, 2G, 2L, 4B, 4D, 4F, 6B, 6D, 6F, 6H, 8B, 8D, 10B, 12B, 12D, 14B
This genomic material is a plasmid that codes for a RFP protein with different antibiotic resistances.
</P><P>A transformation protocol to test competency of bacterial cells was done for the ligation product that was done yesterday.
This test was done with the competent bacterial cells from 13/06 and supercompetent cells for a different lab and normal competent cells from another lab. 
To test the transformation capability we used GFP from our miniprep for a volume of 2 µL.
</P><P>All transformations were spread out on petri dishes with the specific antibiotic.
</P><P>
-
</P><P>Casting of petri dishes (10 amp / 10 kana / 20 chloramphenicol)
</P><P>PCR checking:
Colonies from RBS, TD4, 2B, 2D, 2F, 2H, 2G, 2L, 4B, 4D, 4F, 6B, 6D, 6F, 6H, 8B, 8D, 10B, 12B, 12D and 14B. Box of 15/06 except RBS 14/06.
We forgot to put the antibiotics immediately but added them few hours later.
</P><P>Transformation:
Dna from the kit (cas9, 8B and 14B)
</P><P>Digestion-ligation:
The results being negative for the first attempt, we resumed the digestion-ligation of 14/06.
</P><P>SLIC:
We must insert the iDT sequences into the plasmids, which is why it was decided to make a SLIC so that these fragments are inserted into a plasmid with resistance to chloramphenicol (pSB1C3).
The DNA selected for SLIC is p3VcV, p3VcF, p3Xc, p3RsS, p3Xfa and pVcC.
The ratio used is 3 insert for 1 vector.
The number of inserts was calculated using the NEBioCalculator software.
The DNA was then transformed into DH5α cells.
</P><P><DETAILS><SUMMARY>WEEK 3 : 19/06 → 23/06</SUMMARY></DETAILS></P><P>The PCR done on the 6/16 wasn't used because the plugs were not closed. For that reason, our tubes were dry.
Despite a rehydratation, we couldn't get good results for our electrophoresis, so we made a new PCR.
</P><P>
-
</P><P>RBS, TD4, 2B, 2D, 2F, 2H, 2G, 2L, 4B, 4D, 4F, 6B, 6D, 6F, 6H, 8B, 8D, 10B, 12B, 12D and 14B miniprep.
Verification PCR to verify the 3 ligations on Kana that did not give colonies, to know if it was ligation or transformation that was not effective. Electrophoresis.
</P><P>A) Preparation of the DNA:
We received 3 tubes. (P3Xfu, MultiTag and Xpr). All these tubes are at 1000ng. To be able to prepare them at 50ng / ul we followed the iDT protocol. The DNA is stored in the freezer -20 ° C.
</P><P>B) Infection by phage:
Since the infection of 17/06 did not work, we start again with a concentration 1000 times higher. 100 mL of Ecoli F + bacteria from the preculture, and addition of 1 μL phage to 10 ^ 8 pfu / μL
</P><P>C) Transformation of phage DNA
Since the infection of 17/06 did not work, we check that it is not DNA. Transformation of 0.5 μl of phage DNA (10 μl pfu / ml) into 50 μL of DH5α. 
</P><P>-
</P><P>Amplification protocol for cas9 dcas9 8B and 14B.
We were running out of DNA material for these 4. So an amplification was doe.
We added in 4 tubes, 50 µL of reaction compounds, composed of Q5 High fidelity 2X Master mix, forward and reverse primer, template DNA and ddH2O.
The cycling routine for PCR was 98°C for initial denaturation, then 30 cycles at 98°C, 60, 72, afterward a final extension for 2mn at 72°C. Then hold at 6°C.
We verified our PCR, with an electrophoresis. We added 5µL of DNA template. 10µL of H2O, 3µL of loading purple dye.
The electrophoresis was set for 25 minutes at 135 volts.
The gel was put in gel red for 10 minutes, then revealed using UV.
The result are shown as is:
</P><P>
For cas9 / dcas9 we had a negative result, no bounds at 4000 to 5000 pb.
And a positive result for 8B and 14B a distinct bound at a thousand pb.
A PCR clean up was then done to recuperate the DNA material.
We followed the recommendation of the supplier. To our 45µL of DNA, we added 90µL of NT1 buffer.
We placed the solution on the column then added 700µL of NT1 buffer.
30 seconds centri at 11 000g.
Then 700µL of NT3, centrifuge then discard.
A 1mn centri to dry out the column.
To recuperate the DNA, 20µL of ddH2O at 70°C was let to incubate on the column for 5mn.
Afterward, 20µL of normal H2O. A centri for 1mn at 30 to 50g then another at 11kg. We repeated two times and kept the flow through. We measured the DNA concentration using nanodrop the values mentioned as is.
</P><P>Cas9 -&gt;	4,48ng/µL
dcas9 -&gt; 4,24ng/µL
8B -&gt; 9,54ng/µL
14B -&gt; 11,88ng/µL
</P><P>
We also did a PCR for Cas9 and dCas9. 
We have been trying to PCR this two several time in order to be able to use them for mini-prep.
( See PCR protocol ).
Once the PCR  was done, we injected our samples on the gel and began electrophoresis.
Once the electrophoresis was done we putted the gel in red for 15mn and then studied it under UV.
Unfortunately no result could be observed on the gel. We decide to stop trying to use Cas9 and dCas9 for the moment.
</P><P>Lisa and jeremy made 19 petri dishes with Kanamycin.
</P><P>Transplantation of 4 plates of phage from 20/01 in a Kanamicin Petri Dish.
</P><P>The competence test reveled that our super competent bacteria are approximately three times more competent than Sandra's one, and approximately 1,5 times more competent than 
Le test de compétence a révélé que nos bactéries super compétente sont en moyenne 3 fois plus compétentes que les compétentes de Sandra, et environ 1.5 fois plus compétentes que les super compétentes de l’équipe voisine.
</P><P>Le test de fonctionnement de la ligase montre que la ligase n’est pas efficace, et a perdu son activité. Cela est très probablement dû à une mauvaise conservation de celle-ci et aux nombreux chocs thermique qu’elle a subis.
</P><P>Then we made new competent bacteria.
From the overnight bacteria, we made new competent bacteria. Our first dilution resulted in a OD of 0.54 ( 5.4 because the DO was post dilution ). We then added our culture in 200µL of LB in order to get a OD of 0.1.
To do that we had to dilute 1/50:
Solution: 
-200mL of LB
-4mL of culture.
The OD measured post 1h incubation was 0.7
The rest was the same as in the protocol.
Putted in Petri dish overnight.
</P><P>We had to check RBS because of last night.
RBS vérification by PCR. ( RBS used : 681 ng/µL ) Height awaited ~ 3000 pd on mini prep (0.2 µL of DNA ).
The results from PCR were putted in an electrophoresis and revealed that we failed. 5 stripes when we awaited only 1 ).
We remade digestion because we had suspicious results on our electrophoresis. 
Remaking digestions M13
Making digestion of “red” vector ( RFP ) in order to get linearized plasmids cutted at EP extremity. 
</P><P>Check digestion in gel of EprC / EprI. 
Digestion of RFP vectors - Purify on agarose gel - CmR et KmR (respectively C3 and K3 - Digest maximum quantity of vectors ~1000ng with 2x1µL of enzymes dans un volume total de 50µl. 
Let it migrate a long time (40/45 min) to have a good separation.
Once the migration is done, revelation with UV and Gel red.  we cut the gel containing the linear plasmid, and put it in eppendorf tubes. Weight of the final gels (K3: 0.4g ; C3: 0.3g), We then made our Clean up.
From this PCR, 60µl of linear plasmid was given, at a very low concentration ( 2-5 ng/µl).
</P><P>We made an amplification of Cas9 dCas9 with Pfu turbo which was long graciously with Eric Durand.
This protocol consisted of adding 1mL of our DNA template, to 2.5 mL of primers and 1mL of Dntp 5mL of DMSO and 5mL of PFU mix and to add a sufficient quantity of H2O to complete to 50mL.
The program for the PCR was set by Eric: 70°C for lengthening, 68° for denaturation, and 58° for oligomerization. 
The PCR thermocycling took 4h and 44mn.
</P><P>We did the digestion of 50 ng  of iDT DNA sequences p3 VCV, p3 VCC, p3VcF, p3Xfa ,p3Xfu, p3RsS, p3Xc, MultiTag, Xpr in 15µL. We ligate those fragment in pSB1C3 and we did the transformation in DH5Alpha. For the ligation we put 6µL of insert and 2µL of plasmid with 1µL of T4 DNA ligase
Then we made an amplification PCR of idt sequences (same one). We used 10ng of DNA for the amplification.
</P><P>On the petri dish with GM1 <I>E.coli</I> infected with M13 phage, coming from the 20/06, we seen multiple infection site. Thus we transferred 4 of  them to a kanamycin petri dish.
</P><P>
The M13 infected <I>E.coli</I> Has grown so we pick some colonies in order to make 4 starters. Some of the digestion and ligation of iDT sequence work (p3 VCC, p3 VcV and Xpr) so we make starters. 
In the temoin plane none colonies were visible. 
</P><P>
Transplanting GFP, killer red, EprC, EprI, M13. Preparation of starter.
Jérémy makes 18 petri dishes Chloram
</P><P>We received the iDT sequence of p3_Ng and p3_RSM so we eluate them in water (20µL) at the concentration of 5ng/µL.
</P><P>Camille and Thibault used the starters from MG1 cultures with M13KO7 phages in it. Then, we did mini preps with 4 extracts from different spots on the petri box. GR for “grande”, TP for “tres petit” p1 and p2.
</P><P>After the minipreps, we measured DNA concentration with nanodrops, our results were : 
tp :248 ng/µL,     GR : 51 ng/µL     p1 : 48 ng/µL    p2 : 154 ng/µL    RBS : 338 ng/µL
</P><P>In order to verify that we purify M13KO7 genome we digest the phagemid with BamHI and PstI-HF. We digest 100ng of plasmid in 20µL. Thus we observed 2 bands one at 5859 pb and the other at 2830 pb.
</P><P>With theses concentrations, we made a electrophoresis gel, here are the results : 
</P><P> 
From left to right : 
1 - 2  - 3 - Ladder - Eprc up E/S - M13 up ES - GFP down XP - Epr1 down XP - KillerRed down XP - ladder - 8B - 14B
</P><P>Digestion:
</P><P>Vector RFP Ampicillin resistant in order to pick up a linear vector digested on one side by EcorI and on the other side by PftI.
The goal is to be able to construct afterward.
To do so, we took RFP vector including a gene resistant to ampicillin.
We prepared 100µL 
-&gt; 6µL de plasmide RFP Amp R ( to get approximately 2000 ng )
-&gt; 2µL EcoRI HF  
-&gt; 2µL Pst I 
-&gt; 80 H2O 
-&gt; 10µL of buffer 2.1 
</P><P>30 min 37°C then 20 min 80°C.
</P><P>We made it migrate on an 1% agarose gel. The gel must be long enough to allow a good separation to see a good split between stripes. Then we revealed it with the usual gel RED+UV protocol. 
As our stripes were revealed, we cut the corresponding stripes and transfer them in two separate eppendorf tubes.
The two tubes weight were 280 and 120 mg respectively. 
Then we followed the gel clean up protocol :
we add 200µL of NT1 to 100 mg of gel, the tube was heated and vortex to dilute the gel in the liquid, after liquification of the gel we poured the solution on a column and let it soak of 5 minute a centrifuge at 10000 g was done to flow through the mix and afterward discarded. 700µL of NT3 was then added, a centrifuge was put in use to pass through NT3 and discarded also. To dry out the column a 1 minute centrifuge was done. To recuperate the DNA we let the column soak for 5 minute with hot water. We eluted the DNA with 30µL of water after centrifuge at 10000g the flow through was kept in a newly added 1.5 mL eppendorf tube. We did the same thing a second time with 20µL of hot water. We measured the DNA concentration using a nanodrop. The concentration were 10 and 12 ng/µL respectively. We finished with an electrophoresis to verify if we actually recuperated the DNA, the extra were kept at -20 degrees.
digestion
we digested KR, EprC, M13, EprI, and GFP
EprC and M13 were digested using the enzymes EcoRi and SpeI
KR, EprI and GFP were digested using XbaI and PstI
500 ng of DNA template were digested following the protocol to the letter.
to verify an electrophoresis was done, the results are shown and described as follow.
</P><P>For the binding : 
Up stream : EprC ; M13 ( EcoRI and Spe I )
Down stream KR ; EprI ; GFP ( XbaI and PstI )
</P><P>EprC -&gt; 11µL to get 500ng ( 33µL of H2O ) cutsmart
M13 -&gt; 7,6µL to get 500ng ( 36,5 of H2O )
</P><P>KR -&gt; 3,8µL to get 500ng ( 40µL of H2O )
EprI -&gt; 8,5 µL to get 500ng ( 35,5µL of H2O )
GFP -&gt; 5,7 µL to get 500ng ( 38,5 µL of H2O ).
</P><P><DETAILS><SUMMARY>WEEK 4 : 26/06 → 30/06</SUMMARY></DETAILS></P><P>Camille and Thibault made minipreps and nanodrops of : Xpr : 4ng/mL -- p3VcV 8ng/mL -- M13 15ng/mL -- EprI 14ng/mL -- GFP 25ng/mL -- KR 57ng/mL -- EprC 18ng/mL.
</P><P>Jérémy makes 43 competent cell DH5Alpha 
</P><P>Camille did the digestion and ligation of p3Ng and p3RsM in pSB1C3 at a concentration of 10uM. She transform those fragment in DH5a (2µL of dna for 50µL of competent cell). Other ligation where transform by Thibault (6µL of dna for 50µL of competent cell), the fragment were : p3Xc p3Xfu, p3Xfa, MultiT, p3RsS, p3VcF.
</P><P>Quick changes were made. We wanted to put a mutation on M13KO7 in order to insert a punctual mutation. Thus this mutation will permit the apparition of a restriction site. The Quick Change is a PCR ON.  We have two slot : 1 → M13KO7Tp and 2 → M13KO7p2. Firstly, we used the primers Bdw and Bup to insert the BspI. 
</P><P>Quick change :
Plasmid : 1µL
dNTP(10mM) : 6µL
DMSO : 1µL
Primer 1 : 2,5 µL
Primer 2 : 2,5 µL
Tampon pfu turbo : 5 µL 
pfu Turbo : 1 µL 
H2O qsp 50 µL
</P><P>I put primers with a concentration of 100mM. This might be to concentrate…
</P><P>Camille and Thibault did minipreps and nanodrops of : Xpr : 4ng/mL -- p3VcV 8ng/mL -- M13 15ng/mL -- EprI 14ng/mL -- GFP 25ng/mL -- KR 57ng/mL -- EprC 18ng/mL.
</P><P>Transformation of:
	-controls : 2uL RFP Amp vector + 100 uL of DH5a
		   2uL RFP Amp vector cut E/P + 100 uL of DH5a
		   2uL RFP Amp vector cut E/P and ligate
</P><P>Quick change :
Plasmid : 1µL
dNTP(10mM) : 6µL  
DMSO : 1µL
Primer 1 : 2,5 µL
Primer 2 : 2,5 µL
Tampon pfu turbo : 5 µL 
Pfu Turbo : 1 µL 
H2O qsp 50 µL
</P><P>I put primers with a concentration of 100mM. This might be to concentrate….
</P><P>Quick change Program
</P><P>
98°C
5 min
</P><P>
95°C
1 min
29 cycles
55°C
1 min
68°C
20 min (2 min per kb)
68°C
20 min
</P><P>
16°C
hold
</P><P>Measurements:
</P><P>Calibrage of TECAN:
</P><P>First we need to calibrate the TECAN using LUDOX-S40 as a point reference to    obtain    a    ratiometric    conversion    factor    to    transform absorbance    data    into    a    standard    OD600    measurement. We added   100    μl    LUDOX    into    wells    A1,    B1,    C1,    D1    (or    1    mL    LUDOX    into    cuvette). Then we added   100    μl    of    H2O    into    wells    A2,    B2,    C2,    D2    (or    1    mL    H2O    into    cuvette). We measured   absorbance    600    nm    of    all    samples    in    all    standard    measurement    modes    in    instrument. We recorded  the  data  into Excel.
</P><P>
LUDOX 100%    H2O	
Replicate 1	0.1                     0.0896
Replicate 2        0.099                 0.0895
Replicate 3        0.1007               0.0915
Replicate 4	0.1028               0.0933
</P><P>With C+, C-, TD1, TD2, TD3, TD4, TD5, TD6, we made starters over night. 
</P><P>
Quick change M13KO7
</P><P>Since the PCR is over we have two slot : 1 → M13KO7Tp and 2 → M13KO7p2. In order to remove methylated DNA we put in each 1µL of DpnI enzyme and we incubate 4h at 37°C.
After that, we transform 5µL of the mix in 50µL of super competent DH5a.
</P><P>A miniprep of VcV, VcC and Xpr has been done. The protocol has been followed but the centrifugation has been done at 9000 rpm and not 13 000.
In the end, the nanodrop results were : 
VcV : 27,23 ng/µL,	Vcc : 52,80 ng/µL,	Xpr : 23,63 ng/µL
</P><P>D/L of Deps (10uM), p3SS (5uM) and p3Ec (10uM) with EcoRI and PstI in 2.1 tampon. Ligation with pSBIC3 plasmid.
</P><P>Transformation : In DH5a competent cell with 5µL of digestion with 50µL of cell.
</P><P>Hussein and Thibault did : 
Digestion -  Electrophoresis - Nanodrop - Gel extract from the 2kb strip - PCR cleanup 
</P><P>Digested with EP → 
nanodrop results from minipreps : 
psb1c3 : 249,7 ng/µL
psb3c5 135,2 ng/µL
psb4c5 : 585,2 ng/µL
</P><P>After PCR cleanup :
</P><P>psb3c5 : 8,03 ng/µL
psb1c3 : 12ng/µL
psb4c5 : 15,66 ng/µL
</P><P>
Soraya and Jérémy did digestion/ligation and transformation :
</P><P>- 3 controls :
Control AmpR, check plasmids and transformation =&gt; cellular colony 
Control Plasmid AmpR cut, check digestion =&gt; no cellular colony
Control Plasmid AmpR cut/lig =&gt; cellular colony
</P><P>- 3 ligations :
Lig EprC-KR =&gt; no cellular colony
Lig M13-EprI =&gt; no cellular colony
Lig EprC-GFP =&gt; no cellular colony
</P><P>Measurements:
Fluorescein    fluorescence    standard    curve    
We prepared    a    dilution    series    of    fluorescein    in    4    replicates    and    measure    the    fluorescence    in    a    96    well    plate    in    a    plate    reader in order to  generate    a    standard    curve    of    fluorescence    for    fluorescein    concentration.
</P><P>We prepared    2x   fluorescein  stock    solution    (100    μM)    by    resuspending  fluorescein   	 in    1    mL    of    1xPBS.   Then, we diluted    the    2x   	 fluorescein   	 stock    solution    with    1xPBS    to    make    a    1x    fluorescein   	 solution    and    resulting    concentration    of   	 fluorescein   	 stock    solution    50    μM    (500μL    of    2x    fluorescein    in    500    μL    1x    PBS    will    make    1    mL    of   	 50    μM    (1x)    fluorescein    solution.)   
</P><P>
Preparing ~20 pétri dish of ampicillin 
</P><P>Vérification PCR from RBS and VcF starters 
</P><P>Jérémy did the transformation of M13 with TG1 competent cells
Jérémy did the transformation of of RPF puc19 with TG1 competent cells
Thibault did verification PCR and minipreps from starters of RBS and VcF. 
Here are the results : from left to right : RBS / empty / ladder / empty / VcF
RBS was expected around 300 kb and VcF 1098 kb 
That’s what we have (+300 for VcF)
For the minipreps, the nanodrops results are : RBS : 256 ng/µL and 25 ng/µL for VcF 
</P><P>6 starters where done of p3_VcV, Xpr, p3_VcF two of each. 
We’ve got colonies from the d/l of 27/06 Deps1 Deps2, p3_Ng, p3SS, p3_Xc, p3Ec et p3MT.
</P><P>Camille did the transformation, of p3_Xfa, p3_Xfu, p3_RsS et p3_RsM with 5uL of ligation and 50uL DH5a.
</P><P>
M13-EprC not good &gt; 1000pb to electro PCR
M13-EprI seems to be good find the miniprep
</P><P>
We did the mini prep for M13 ( 9,35 ),KR (169 ), RBS (291) and GFP 
(71), P3VCF (117), EprC (55), Xpr (72) P3Xc (62), Ligation M13-EprC (19), EprI (65), P3VcV (328)  (ng/microlitre)
</P><P>
Thibault repicked the petri dishes of : psb1c3 / psb1k3 / psb3c5 / psb3t5 / psb4c5 / psb2k3 / psb3k3 / AC3 / psb4k5 
Repicked it on pétri dishes and made starters that will incubate ON for futures minipreps
</P><P>PCR of verification from iDT sequence ligation onto pSB1C3 and  repiquage of them.
Séquences vérifiées:  
_p3Ng : 3 colonies			       _Deps: 2 colonies
</P><P>		         
_p3VcC: 3 colonies			_p3Ec :3 colonies
</P><PRE>_p3Xc: 3colonies				_p3VcF: colonies
</PRE><P>		        
_Xpr: 2 colonies				_p3Ss: 1 colonies
</P><P>		        
_p3VcV: 3 colonies
</P><P>The electrophorèse shows that p3Ec 1 and 2, and  p3Xc1, p3Xc2 and p3Xc3, might be good.
</P><P>We’ve got p3Pa from iDT, and we add water until we reach the concentration of 50ng/µl. 
</P><P>Digestion of p3_Pa with SpeI and PstI.
3uL p3_Pa (150ng)
5uL 2.1 Buffer
0,5uL PstI
0,5uL SpeI
H2O qsp 150uL
30 min at 37°C then 20 min at 80°C
This is a error from us because we should cut p3_pa w/ EcoRI rather than SpeI.
</P><P>p3Xfu, p3Xfa,p3RsS, p3RsM and p3Pa were ligate onto pSB1C3 (6uL of digestion and 2uL of linearised and cut plasmid).
</P><P>Deps, p3VcF, Xpr, p3Ss, p3VcC p3Xfu, p3Xfa,p3RsS and p3RsM  were transformed.  
</P><P>5 starter where done: p3Ec (1); p3Ec (2); p3Xc (1); p3Xc(2); p3Xc (3).
</P><P>Repiquage and PCR of p3_VcV. The electrophoresis shows us that those colonies didn’t have the right fragment.
</P><P>
PCR Amplification of all expected M13 on the gel
Verification PCR of M13 we used yesterday 
</P><P>And miniprep of p3E1, p3E2, p3Xc1, p3Xc2, p3Xc3.
</P><P>Results from nanodrop : psb2k3 : 31 ng/µL     psb3k3 : 71: ng/µL     psb4k5 : 72 ng/µL      psb3c5 : 107 ng/µL
psb1k3 : 96 ng/µL      psb3T5 : 63 ng/µL      psb4c5 : 47 ng/µL      AC3 : 62 ng/µL    psb1c3 : 250 ng/µL    p3Ec1 : 41 ng/µL      p3Ec2 : 48 ng/µL      p3Xc1 : 16 ng/µL      p3Xc2 : 49 ng/µL    p3Xc2 : 28 ng/µL    
</P><P>The concentration of p3Xc1 et p3Xc2 is too low for the sequencing, thus we make 2 staters of each ON. 
</P><P>Transformation of M13, GFP, KR and RBS.
</P><P>Transformation des biobricks measurements TD1, TD2, TD3, TD4, TD5, TD6, C+, C-, et de 2B.
PCR d’amplification des p3.
</P><P>Miniprep of M13, concentration 42,47 ng/uL.
</P><P>
PCR amplification of sequence IDT p3Vcc, p3 vcf, Mtag, p3 Pa, p3 Rsm, Xpr, p3Xfu, p3 Xca, p3 Ng, p3 Vcv, p3 Ec, p3SS, p3 Rss.
</P><P>Digestion of p3_Pa with EcoRI and PstI.
3uL p3_Pa (150ng)
5uL 2.1 Buffer
0,5uL EcoRI
0,5uL SpeI
H2O qsp 150uL
30 min at 37°C then 20min at 80°C
</P><P>Ligation of p3_Pa and p3_VcF with pSB1C3 
For p3_Pa 2uL of digestion and for p3_VcF 6uL of DNA. For both 2uL of Buffer and 1uL of T4 DNA ligase with H20 qsp 50uL.
</P><P>Transformation of p3_Pa and p3_VcF in DH5a (5uL of DNA w/ 50 of cells). 
</P><P>PCR amplification and clean up of all p3 iDT sequence amplified with Q5 enzymes.
p3Ec = 54 ng/uL, Xpr = 28 ng/uL, p3Ng = 89 ng/uL, p3Pa = 72 ng/uL, p3VcC = 93 ng/uL, p3Xc = 103 ng/uL, 
p3RsM = 134 ng/uL, MT = 33 ng/uL, p3SS = 29ng/uL, p3_Xfa = 86ng/uL, p3RsS = 98ng/uL, p3Xfu = 98 ng/uL, 
p3 VcF = 96ng/uL et p3VcV = 64ng/uL.
</P><P>Minipreps of all the psb1A10/psb1AX3/psb4A5/psb1A3/psb1AX3/psb6A1/psb1A3
And miniprep of p3Xc1, p3Xc3.
</P><P>Results from nanodrop : psb1A10 : 157 ng/µL     psb1AX3 : 85 ng/µL     psb4A5 : 89 ng/µL     psb1A3 : 48 ng/µL
psb1AX3 : 68 ng/µL      psb6A1 : 277 ng/µL      psb1A3 : 308 ng/µL      p3Xc1 : 133 ng/µL      and    p3Xc2 : 108 ng/µL.520  
</P><P>Digestion and gel extract of GFP, KR and M13ori. Result from nanodrop :
pSB1C3 (1) E/S = 7ng/uL, pSB1C3 (2) E/S = 8ng/uL, pSB1C3 (1) X/P = 10ng/uL, pSB1C3 (2) X/P = 13ng/uL, pSB1C3 (3) X/P = 9ng/uL, pSB1C3 (4) X/P = 9ng/uL, M13ori (1) E/S = 4ng/uL, M13ori (2) E/S = 5ng/uL, KR(1) X/P = 5ng/uL, KR(2) X/P = 13ng/uL, GFP(1) X/P = 7ng/uL and GFP(2) X/P = 7ng/uL
</P><P>Transformation de la biobrick Contrôle positif. 
Gel interpretation for M13 : 800~1000 pb, M13 is 529 + 300 = 529 ( the results are ok )
Soraya manips : 
Digestion-ligation of EprC, EprI and RBS are little sequences, respectively 55, 61 and 15 Bp. This explains why it is so hard to bind them with other vectors. ( They are lost in the huge mix of other strands ).
Ori M13 -EprC / EprC-Kr | EprC-GFP
Ori M13 -EprI / EprI-RBS / RBS-KR
EprI-RBS / RBS-GFP
</P><P>
Thibault : Elaboration of the “Phages In Soil protocol”
<DETAILS><SUMMARY>WEEK 5 : 03/07 → 07/07</SUMMARY></DETAILS></P><P> 
EprC, EprI and RBS are  short sequences (respectively 55pb, 61pb and 15pb),  which can explain why it is so hard to bind them with other strand into a vector.
That is why we chose to try another protocol, that consist in cut the original vector of those short sequences and directly insert in the second sequence that is longer.  
-For the upstream part : (Ori M13, 529pb) 
We want to cut about 1500ng of OriM13 plasmid, we have the miniprep from the 13/06/17, at 66ng/µL. So we have to use 22,7 µL of this miniprep by EcoRI and SpeI.
-For the downstream parts : (Killer Red(KR), 723pb; GFP, 711pb)
In the same way as before we want to cut about 1500ng of each plasmid :  
	 -For KR, we have the miniprep from the 13/06/17 at 130ng/µL, so we have to use 	11,5µL of this miniprep.
	- For GFP, we have the miniprep from the 13/06/17 at 88ng/µL, so we have to use 	17µL of this miniprep.
Each one will be cut by XbaI and PstI.
Restriction : (Total volume = 100 µL)
</P><TABLE><TBODY><TR><TD>OriM13</TD><TD>Killer Red</TD><TD>GFP</TD></TR><TR><TD>MiniPrep µL</TD><TD>23</TD><TD>12</TD><TD>17</TD></TR><TR><TD>Buffer µL</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR><TD>Enzyme 1 (µL)</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR><TD>Enzyme2 (µL)</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR><TD>H2O (µL)</TD><TD>63</TD><TD>74</TD><TD>69</TD></TR></TBODY></TABLE><P>==&gt; 1h at 37°C.
</P><P>Agarose gel (1%) electrophoresis (long gel for best separation):
</P><P>
Gel extract :
Cut the gel as closely as possible to each band, make a Gel-Clean up (see protocols).
Restriction and gel extract of GFP, KR and M13ori. Result from nanodrop :
</P><TABLE><TBODY><TR><TD>pSB1C3 (1) E/S = 7ng/uL</TD><TD>pSB1C3 (4) X/P = 9ng/uL</TD></TR><TR><TD>pSB1C3 (2) E/S = 8ng/uL</TD><TD>M13ori (1) E/S = 4ng/uL</TD></TR><TR><TD>pSB1C3 (1) X/P = 10ng/uL</TD><TD>KR(1) X/P = 5ng/uL</TD></TR><TR><TD>pSB1C3 (2) X/P = 13ng/uL</TD><TD>M13ori (1) E/S = 4ng/uL</TD></TR><TR><TD>pSB1C3 (3) X/P = 9ng/uL</TD><TD>KR(2) X/P = 13ng/uL</TD></TR><TR><TD>GFP(1) X/P = 7ng/uL </TD><TD>GFP(2) X/P = 7ng/uL</TD></TR></TBODY></TABLE><P>
at -20°C.
</P><P>
OriM13-EprI is definitely dead... (after the disappearance of the starter, the Petri dish  containing the only one colony violently fell on the floor...tragic.)
PCR of verification of the RBS's minipreps (23/06; 28/06) : This is not the right size. The minipreps are not good.
</P><P>
Cloning:
Verification by electrophoresis of the gel extract show that we can continue so, we want do make :		EprC-KR		EprC-GFP
Upstream part: Eprc (miniprep from the 13/06/17 at 45ng/µL). We need to cut about 200ng. So we will use 4.5 µL of this miniprep, with 2 µL of buffer 2.1 + 0.2 µL of SpeI +0.2 µL of PstI and load up with H2O to 20 µL.
==&gt; 1h at 37°C, then inactivate enzymes 20 min at 80°C.
For the ligations:
</P><TABLE><TBODY><TR><TD>EprC-KR</TD><TD>EprC-GFP</TD></TR><TR><TD>&quot;Upstream&quot;</TD><TD>EprC cut S/P, 2 µL</TD><TD>EprC cut S/P, 2 µL</TD></TR><TR><TD>&quot;Downstream&quot; (previously prepared)</TD><TD>KR cut X/P, 2 µL</TD><TD>GFP cut X/P, 2 µL</TD></TR><TR><TD>T4DNA ligase buffer</TD><TD>2 µL</TD><TD>2 µL</TD></TR><TR><TD>T4DNA ligase</TD><TD>1 µL</TD><TD>1 µL</TD></TR><TR><TD>H2O</TD><TD>13 µL</TD><TD>13 µL</TD></TR></TBODY></TABLE><P>==&gt; 3h at room temperature.
</P><P>
- Transformation of the cloning of the day before, into competent DH5α ( 3 µL of ligation mix with 50 µL of bacteria, on chloramphénicol) .
(Help Camille to make SLIC during the rest of the day)
-Preparation of more linear Psb1C3, using Psb1C3 vector containing RFP (transplanting of the 29/06/17). Launch a starter.
</P><P>Colonies on Petri dishes are too small to transplant them and make a PCR, so we just made the transplant and  let them grow up. (1,2 and 3 are EprC-GFP colonies 1,2 and 3; 11 to the end are EprC-KR).
</P><P>Digestion and clean-up:
In order to obtain more linear Psb1C3, we previously launch two big starters of transformed bacteria with RFP into Psb1C3 vector. Then we made  minipreps which had  very low concentrations. So we used a device which centrifuge at hot temperature to have better concentrations to simplify the digestion and gel extract.
</P><P>
-PCR of verification on colonies, for the ligations  EprC-GFP and EprC-KR.
</P><P>
Only the assembly EprC-KR succeed. ==&gt; launch a starter. 
RBS failed again.
</P><P>Digestion of 2000ng of Psb1C3 containing RFP.
We never succeeded cloning EprI and RBS so we decided to use LacI-RBS instead of these biobricks.
</P><P>More informations about week : (details)
</P><P>PCR of verification of the clone coming from the ligation between iDT sequence and pSBIC3. For each ligation we had multiple colonies that we tested. Finally it seems that only a few got the right length : 
p3Pa1, p3Pa2 p3Pa4, p3Pa6, p3Xfa4, p3RSM2, p3RSM3, p3RSM4, p3RSM6, p3RSM7 and p3RSM8.
</P><P>As DH5alpha infected with M13KO7mutB are really slow to grow, we transformed 5uL M13KO7mutB with 50uL of TG1 <I>E. coli</I> bactgerium.
</P><P>Thibault : 
Large quantities of helper phage  : 
200µL of TG1 cells with a 0,4 DO, added 10µL of M13KO7 helper φ (10^9 pfu/mL). 
After 30min 37°C growing, spread on pétri dish without antibiotics
</P><P>
Morning after : I made a 5mL starter of TG1 and added a small plaque of the yesterday culture.
After 2 hours of shaking at 37°C, I transfered that in a 2L flask with 500mL 2YT and 50µg/mL kanamycin.
</P><P>The morning after, I spinned for 15min at 10800g, kept the supernatant and added 100mL of PEG/NaCl and made another spin to be able to resuspend the pellet in PBS.
</P><P>We did the miniprep of all the good ligation between iDT and pSBIC3. p3RsM7 = 116ng/uL, p3Xfa = 92ng/uL, p3RsM2 = 67ng/uL, p3Pa4 = 76ng/uL, p3RsM3 = 75ng/uL, p3RsM4 = 80ng/uL, p3RsM8 = 102ng/uL, p3Pa1 = 104ng/uL, p3Pa2 = 85ng/uL, p3RsM6 = 67ng/uL and p3Pa6 = 103ng/uL
</P><P>We prepared the petri dishes: 16 of Chloramphenicol and 19 of Kanamycin.
</P><P>We did a competency test for the DH5α that we prepared earlier.
</P><P>We did a transformation for strong promoter medium RBS, strong promoter weak RBS, medium promoter and medium RBS.
We did a transformation for strong promoter medium RBS RFP, strong promoter weak RBS RFP, medium promoter and medium RBS RFP.
We did a transformation for strong promoter medium RBS GFP, strong promoter weak RBS GFP, medium promoter and medium RBS GFP, HisTag double stop, HisTag RFP.
</P><P>Amplification with PCR of the smallest iDT sequences = MT, Xpr and p3SS. As the elongation times are different between each fragment we choose to amplify only the smallest (between 120 and 150bp) in order to improve the success of the amplification.
In each tube we put = 	25uL Q5 master Mix
			2,5uL of OGC36
			2,5uL of OGC37
1uL of DNA
19uL H2O (qsp 50uL)
</P><P>98°C
30 sec
</P><P>
95°C
15 sec
29 cycles
55°C
30 sec
72°C
10 sec (30 sec per kb)
72°C
2 min
</P><P>16°C
hold
</P><P>We prepared the petri dishes: 20 of Chloramphenicol.
</P><P>We did a competency test for the TG1 that we prepared earlier. And we realised a transformation for RBS and Kanamycin resistance.
</P><P>Sequençing p3Pa1, p3Xfa4, p3RsM3. 
</P><P>PCR clean up of Xpr = 60ng/uL, p3SS = 53ng/uL and MT = 60 ng/uL
</P><P>We did a transformation for MT, Deps, Xpr, p3SS, p3Ng, p3Xfu, p3VcC, p3VcV, p3RsS, p3VcF in DH5α with Chloramphenicol antibiotics petri dishes. We put 5μL of DNA for 50μL of bacterial cells.
</P><P>We digest RFP pSB1C3 in E/P to do a gel extract.
</P><P>the IDT kit for DEPS present some mutation this was previously observed after transferring the sequence into a plasmid and realising a pcr verification on colony.
</P><P>to countermeasure this problem a pcr amplification is done with the setting as mentioned in the protocols with the single different where the oligomerization duration was set to a minute and 15 seconds.
an electrophoresis verified that the desired band between 1500 and 2000 was actually present
to clean the solution from any and all contaminant we did a pcr clean up afterward the concentration was measured with a nanodrop a concentration of 140 ng/μL was retrieved.
later this day a SLIC was attempted using a circular plasmid digested with EcoRI-HF and PstI. Deps was digested with the same enzymes as well. a small difference to the original protocol was done when digesting an appropriate quantity of DNA for the vector and insert was present but they weren’t digested separately. this small change gave us a negative result  the next day after the transformation with competent cells.
</P><P>The IDT kit for DEPS present some mutation this was previously observed after transferring the sequence into a plasmid and realising a PCR verification on colony.
</P><P>to countermeasure this problem a pcr amplification is done with the setting as mentioned in the protocols with the single different where the oligomerization duration was set to a minute and 15 seconds.
an electrophoresis verified that the desired band between 1500 and 2000 was actually present
to clean the solution from any and all contaminant we did a pcr clean up afterward the concentration was measured with a nanodrop a concentration of 140 ng/μL was retrieved.
later this day a SLIC was attempted using a circular plasmid digested with EcoRI-HF and PstI. Deps was digested with the same enzymes as well. a small difference to the original protocol was done when digesting an appropriate quantity of DNA for the vector and insert was present but they weren’t digested separately. this small change gave us a negative result  the next day after the transformation with competent cells.
</P><P>We digest psb1 rfp vector and then made a gel extract to isolate empty vector rfp. 
</P><P>
We made TG1 + phage pétri dish + htop agar to be able to get colonies of infected TG1 ( on kanamycin pétri dishies)
</P><P><DETAILS><SUMMARY>WEEK 6 : 10/07 → 14/07</SUMMARY></DETAILS></P><P>
Electrophoresis of what supposed to be Psb1C3-RFP cut E/P. (The size are wrong, this is not Psb1C3)
We remade it from the new miniprep.
</P><P>
-Miniprep of the assembly EprC-KR.
-Cloning:
	We tried to obtain EprC-GFP, OriM13-EprC-KR, LacI-RBS-KR and LacI-RBS-GFP.
</P><P>digestion:
- EprC-KR : 500ng=&gt; 14.5µL of miniprep 07/07/17 , H2O=29.5 µL  (see protocol)
- LacI-RBS: 500ng=&gt; 18.5 µL of miniprep 07/07/17 , H2O=25.5 µL (see protocol)
</P><TABLE><TBODY><TR><TD>EprC-KR</TD><TD>LacI-RBS</TD></TR><TR><TD>Miniprep (µL)</TD><TD>14,5</TD><TD>18,5</TD></TR><TR><TD>Buffer (µL)</TD><TD>5 (2.1)</TD><TD>5 (cutsmart)</TD></TR><TR><TD>Enzyme 1 (µL)</TD><TD>0.5 XbaI</TD><TD>0.5 EcoRI</TD></TR><TR><TD>Enzyme2 (µL)</TD><TD>0.5 PstI</TD><TD>0.5 SpeI</TD></TR><TR><TD>H2O (µL)</TD><TD>29.5 </TD><TD>25.5 </TD></TR></TBODY></TABLE><P>1h at 37°C, then inactivate 20min at 80°C.
</P><TABLE><TBODY><TR><TD>EprC-GFP</TD><TD>OriM13-EprC-KR</TD><TD>LacI-RBS-KR</TD><TD>LacI-RBS-GFP</TD></TR><TR><TD>Upstream</TD><TD>EprC cut S/P (2µL) </TD><TD>OriM13 cut E/S (2µL) </TD><TD>LacI-RBS cut E/S (2µL) </TD><TD>LacI-RBS cut E/S (2µL) </TD></TR><TR><TD>Downstream</TD><TD>GFP cut X/P(2µL)</TD><TD>eprC-KR cut X/P (2µL)</TD><TD>KR cut X/P (2µL)</TD><TD>GFP cut X/P (2µL)</TD></TR><TR><TD>Destination plasmid</TD><TD>P AmpR cutE/P (2uL)</TD><TD>P AmpR cut E/P (2uL)</TD><TD>P.AmpR cut E/P (2µL)</TD></TR><TR><TD>T4 DNA Ligase Buffer( uL)</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR><TD>T4 DNA Ligase (uL)</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TD>H2O (uL)</TD><TD>13</TD><TD>11</TD><TD>11</TD><TD>11</TD></TR></TBODY></TABLE><P>3h at room temperature.
</P><P>Transformation : 3 µL of ligation mix with 50 µL of bacteria (be careful, EprC-GFP on chloramphenicol, the others on ampicillin).
</P><P>- No colony on Amp Petri dishes ... But there is one on Petri dish Chloram ! (EprC-GFP) And when we put it in the MAGIC BOX, we could saw its green light ! (Happy!!!). ==&gt;  PCR control and starter.
</P><P>-Digestion (restriction):
100ng of each.
</P><TABLE><TBODY><TR><TD>Ori M13</TD><TD>Lac I RBS</TD></TR><TR><TD>Miniprep ( µL)</TD><TD>2.2</TD><TD>2.5</TD></TR><TR><TD>Buffer ( µL)</TD><TD>1 ( 2.1 )</TD><TD>1 (2.1 )</TD></TR><TR><TD>Enzyme 1 ( uL)</TD><TD>0.1 SpeI</TD><TD>0.1 SpeI</TD></TR><TR><TD>Enzyme 2 ( uL)</TD><TD>0.1 PstI</TD><TD>0.1 PstI</TD></TR><TR><TD>H2O</TD><TD>6.4 ( µL )</TD><TD>6.4 µL</TD></TR></TBODY></TABLE><P>1h at 37°C, then inactivate 20min at 80°C.
</P><P>Ligation:
</P><TABLE><TBODY><TR><TD>OriM13-EprC-KR</TD><TD>LacI-RBS-KR</TD><TD>LacI-RBS-GFP</TD></TR><TR><TD>Upstream</TD><TD>OriM13 cut S/P(2µL)</TD><TD>LacI-RBS cut E/S (2µL)</TD><TD>LacI-RBS cut S/P (2µL)</TD></TR><TR><TD>Downstream</TD><TD>EprC-KR cut X/P (2µL)</TD><TD>KR cut X/P (2µL)</TD><TD>KR cut X/P (2µL)</TD></TR><TR><TD>T4 DNA ligase buffer</TD><TD>(2µL)</TD><TD>(2µL)</TD><TD>(2µL)</TD></TR><TR><TD>T4 DNA ligase</TD><TD>(1µL)</TD><TD>(1µL)</TD><TD>(1µL)</TD></TR><TR><TD>H2O</TD><TD>(13µL)</TD><TD>(13µL)</TD><TD>(13µL)</TD></TR></TBODY></TABLE><P>3h at room temperature.
</P><P>
Transformation : 3 µL of ligation mix with 50 µL of bacteria, on chloramphenicol.
</P><P>
RBS are not good again...
1: miniprep 23/06
2: miniprep 28/06
</P><P>But the results of LacI-RBS are right.
3: miniprep 07/07
4: miniprep 09/07
6: directly from the kit.
</P><TABLE><TBODY><TR><TD>P3RsM</TD><TD>P3Ec</TD><TD>GFP</TD></TR><TR><TD>Miniprep (µL)</TD><TD>1.5</TD><TD>0.5</TD><TD>9</TD></TR><TR><TD>Buffer (µL)</TD><TD>1 (2.1)</TD><TD>1 (2.1)</TD><TD>1 (2.1)</TD></TR><TR><TD>Enzyme 1 (µL)</TD><TD>0.1(AgeI)</TD><TD>0.1(AgeI)</TD><TD>0.1(NgoMIV)</TD></TR><TR><TD>Enzyme2 (µL)</TD><TD>0.1 (PstI)</TD><TD>0.1(PstI)</TD><TD>0.1(PstI)</TD></TR><TR><TD>H2O (µL)</TD><TD>7.3</TD><TD>8.3</TD><TD>0</TD></TR></TBODY></TABLE><P>==&gt; 1h at 37°C.
==&gt;electrophoresis to control these enzyme (first time I use them).
</P><P>
We can't see anything, probably not enough DNA. 
</P><P>More informations about the week : ( details )
</P><P>With the negative results of the transformation another attempt was launched this time by following the protocol to the letter so we used linearized plasmid and digested DEPS in a separate tube.
The construction were as followed 3 SLIC with deps in three different plasmids psb1C3, psb1A3 and psb1K3.
and a standard digestion/ligation was done also with the three different plasmids. This was done just for fun and no practical outcome was intended but maybe this was done to increase our chances to succeed.
The different preparation are as mentioned below:
</P><P>
Digestion of deps for ligation 
500 ng of Deps
0.5 μL of EcoRI-HF
0.5 μL of PstI
5 μL of buffer 2.1
39 μL of H2O
</P><P>
ligation of Deps AmpR
ligation of Deps KanaR
ligation of Deps ChloranR
150 ng of digested Deps 
150 ng of digested Deps 
150 ng of digested Deps 
2μL of linearized psb1A3
1μL of linearized psb1K3
2μL of linearized psb1C3
2μL of T4 buffer
2μL of T4 buffer
2μL of T4 buffer
1μL of T4 ligase
1μL of T4 ligase
1μL of T4 ligase
10 μL of H2O
10 μL of H2O
10 μL of H2O
</P><P>
Deps digestion for Slic X2
500 ng of Deps 
0.5μL of EcoRI-HF
0.5μL of PstI
5μLof buffer 2.1
9μL of H2O
</P><P>
Slic Deps with AmpR
Slic Deps with KanaR
Slic Deps with ChloranR
3μL of linearized psb1A3
2μL of linearized psb1k3
3μL of linearized psb1C3
250 ng of Deps 
250 ng of Deps 
250 ng of Deps 
2μL of buffer 2.1
2μL of buffer 2.1
2μL of buffer 2.1
0.5μL of T4 polymerase
0.5μL of T4 polymerase
0.5μL of T4 polymerase
4.5μL of H2O
5.5μL of H2O
4.5μL of H2O
</P><P>Once everything was done, transformations were performed.
</P><P>We made a starter of fresh TG1 that we repicked with the infected TG1.
We put the mix in 500mL of 2YT + 2,5 ml of kanamycin and let it ON at 30°C with shaking.
</P><P>After an overnight incubation, the petri dishes presented a number of colonies but no fireworks wet a PCR verification on colony was necessary so we followed the protocol and began some starters for tomorrow in the process.
The results of the electrophoresis of the PCR were very promising we had the suitable bands between 1500 and 2000. But, we will be 100 % sure if the sequence is intact only after sequencing.  
</P><P>We made a repiquage of TG1 M13KO7mutB (33 colony). 
</P><P>With the ON flasque at 30°C with shaking we made a 6000 rpm spin for 30min
Then we put it on ice for an hour and added 100 ml of PEG/Nacl (22g of PEB 6000 + 16,1g Nacl)
Another 6000g spin for an hour.
</P><P>Added 8 mL PBS and 2 ml PEG and spinned.
</P><P>We made starters of TG1 and GM1 for future tests with phages.
</P><P>We did a transformation of M13KO7mutB in TG1 in a petri dish with Kanamicin antibiotic as well for p3RSS, p3SS, p3VcC, p3Xc, P3Ng, p3Xfa, MT, p3Xpr, p3Xfu, p3Pa, p3VcF in DH5α in petri dishes with Chloramphenicol antibiotics. We put all of these in a 37°C incubator.
We also did the transformation of M13KO7mutB two more times and put one in a 30 °C incubator and the other one in a 42 °C incubator.
</P><P>
Today we have done a digestion of: PSB1C3, P3xfa, Deps, P3VcF, Xpr, P3RSS, P3xc, P3vcc, P3xfu, P3Ng, P3ss, P3vcv, MT.
We have used buffer 2.1, PST1 et ecor1 for enzymes. We wanted a final concentration of 10 ng/µL.
For all the vectors we have used 0,5 µL of eco1 and pst1 and 5 µL of 2.1.
</P><P>
PSB1C3: 
Vector 2uL (500ng)
EcoRI 0,5 uL
PstI 0,5 uL
Buffer 2.1 5uL
H2O qsp  50uL
</P><P>P3xfa: 
Insert 2uL (200ng)
EcoRI 0,5 uL
PstI 0,5 uL
Buffer 2.1 5uL
H2O qsp  20uL
</P><P>P3vcf: 
Insert 2uL (200ng)
EcoRI 0,5 uL
PstI 0,5 uL
Buffer 2.1 5uL
H2O qsp  20uL
</P><P>P3rss: 
Insert 2uL (200ng)
EcoRI 0,5 uL
PstI 0,5 uL
Buffer 2.1 5uL
H2O qsp  20uL
</P><P>Xpr: 
Insert 6uL (200ng)
EcoRI 0,5 uL
PstI 0,5 uL
Buffer 2.1 5uL
H2O qsp  20uL
</P><P>P3Xc:
Insert 2uL (200ng)
EcoRI 0,5 uL
PstI 0,5 uL
Buffer 2.1 5uL
H2O qsp  20uL
</P><P>P3vcc: Insert 3uL (200ng)
EcoRI 0,5 uL
PstI 0,5 uL
Buffer 2.1 5uL
H2O qsp  20uL
</P><P>MT: 
Insert 2uL (200ng)
EcoRI 0,5 uL
PstI 0,5 uL
Buffer 2.1 5uL
H2O qsp  20uL
</P><P>P3vcv: 
Insert 3uL (200ng)
EcoRI 0,5 uL
PstI 0,5 uL
Buffer 2.1 5uL
H2O qsp  20uL
</P><P>P3Ng:
</P><PRE>Insert 2uL (200ng)
</PRE><P>EcoRI 0,5 uL
PstI 0,5 uL
Buffer 2.1 5uL
H2O qsp  20uL
</P><P>Deps: 
Insert 2uL (200ng)
EcoRI 0,5 uL
PstI 0,5 uL
Buffer 2.1 5uL
H2O qsp  20uL
</P><P>P3xfu: 
Insert 2uL (200ng)
EcoRI 0,5 uL
PstI 0,5 uL
Buffer 2.1 5uL
H2O qsp  20uL
</P><P>P3ss : 
Insert 6uL (200ng)
EcoRI 0,5 uL
PstI 0,5 uL
Buffer 2.1 5uL
H2O qsp  20uL   
</P><P>After it we have incubated all of this tubes at 37 °C for 2 hour and 20 minutes at 80 °C
</P><P>We did a mini prep of p3RSM which was in PSB1C3. We obtained a concentration of 132.66 ng/µL and 127.35 ng/µL.
</P><P>We did a transformation of Deps in 100 µL of DH5α.
Ligation of iDT sequence w/ pSBIC3. We choose a ratio of 1 vector of 2 insert.
P3xfa: 
Insert 2.5 µL
Vector 5 µL (50 ng) 
T4 DNA ligase 0.1 µL
Buffer 1 µL
H2O qsp  10 µL 
</P><P>P3vcf: 
Insert 3 µL
Vector 5 µL (50ng) 
T4 DNA ligase 1 µL
Buffer 1 µL  
H2O qsp  10 µL 
</P><P>P3rss: 
Insert 2 µL
Vector 5 µL (50 ng) 
T4 DNA ligase 1 µL
Buffer 1 µL
H2O qsp  10 µL 
</P><P>Xpr: 
Insert 0,5 µL
Vector 5 µL (50 ng) 
T4 DNA ligase 1 µL
Buffer 1 µL
H2O qsp  10 µL 
</P><P>P3Xc:
Insert 1,5 µL
Vector 5uL (50 ng) 
T4 DNA ligase 0,1 µL
Buffer 1 µL
H2O qsp  10 µL 
</P><P>P3vcc: 
Insert 2 µL
Vector 5 µL (50 ng) 
T4 DNA ligase 1 µL
Buffer 1 µL
H2O qsp  10 µL 
</P><P>MT: 
Insert 0,5 µL
Vector 5 µL (50 ng) 
T4 DNA ligase 1 µL
Buffer 1 µL
H2O qsp  10 µL 
</P><P>P3vcv: 
Insert 3 µL
Vector 5 µL (50 ng) 
T4 DNA ligase 1 µL
Buffer 1 µL
H2O qsp  10 µL 
</P><P>P3Ng:
Insert 3 µL
Vector 5 µL (50 ng) 
T4 DNA ligase 1 µL
Buffer 1 µL
H2O qsp  10 µL 
</P><P>P3xfu: 
Insert 1,5 µL
Vector 5 µL (50 ng) 
T4 DNA ligase 1 µL
Buffer 1 µL
H2O qsp  10 µL 
</P><P>P3ss: 
Insert 0,5 µL
Vector 5 µL (50 ng) 
T4 DNA ligase 1 µL
Buffer 1 µL
H2O qsp  10uL 
</P><P>Ligation 1H and transformation in DH5alpha (5uL of DNA and 50uL of cells), incubation at 37°C ON. We make a negative control (bacteria only) and a positive controle (pB1C3 rfp).
</P><P>Quick Change: 2 tubes in order to have 2 different test
1uL pfu turbo
6uL dNTP (10uM)
2,5uL PrimerAup
2,5uL PrimerAup
5uL pfu buffer
1uL DMSO
32uL H2O (qsp 50uL) 
</P><P>95°C
2 min
</P><P>
95°C
30 sec
29 cycles
55°C or 58°C
30 sec
68°C
20 min(2 min per kb)
68°C
2 min
</P><P>
16°C
hold
</P><P>Transformation of SuperNova (Bba).  
</P><P>Phage crossing over test : 
Yesterday we transformed TG1 with oriM13 and put two petri dishes incubation. 
We used the culture to make new starters we will use to mix up with phages later.  
</P><P>Phage : TG1 serial dilution results: too much colonies, need to dilute more.
Dilution: 200µL TG1 (O,4 OD) + 10µL phage solution (10^8 pfu/mL)
Spread on petri dish with htop agar
</P><P>Quick change :
We’ve got the result of the quick change of yesterday (M13KO7mutA1 &amp; 2). We put 1µL of DpnI in each tube and we did a 4h incubation at 37°C. 
The gel electrophoresis shows us no amplification but Valerie told me to transform anyway. 
With the advise of Gauthier I put 50µL of the PCR mix w/ 100µL of TG1 cells.
1H of incubation on ice then 45sec in 42°C then 1h at 37°C. Then we put on 4 kanamycine petri dish ON at 37°C for each.
</P><P>Starters of p3EC and p3RsM for glycerolization
</P><P>Colonies PCR of SuperNova.
</P><P>We did a repiquage of New Lig p3Pa4-GFP DH5alpha.
</P><P>
Result of the Quick change :  none of the colonies coming from M13KO7mutA2 grown, but we’ve got 13 colonies for M13KO7mutA1. We decided to grow 2 starters for 3 first colonies : M13KO7mutA1.1, M13KO7mutA1.2 and M13KO7mutA1.3.
</P><P>PCR on colonies (vérification of the ligation between pSB1C3 and iDT sequences). We choose to only verify 4 colonies per petri dish.
</P><P>p3VcC1, p3VcC2, p3VcC3, p3VcC4, p3RsS1, p3RsS2, p3RsS3, p3RsS4, p3Xc1, p3Xc2, p3Xc3, p3Xc4, Xpr1, Xpr2 , Xpr3, Xpr4, p3SS1, p3SS2, p3SS3, p3SS4, p3VcF1, p3VcF2, p3VcF3, p3VcF4, p3Xfu1, p3Xfu2, p3Xfu3, p3Xfu4,  p3Ng1, p3Ng2, p3Ng3, p3Ng4, p3Xfa1, p3Xfa2, p3Xfa3, p3Xfa4, MT1, MT2, MT3, MT4, p3VcV1, p3VcV2, p3VcV3, p3VcV4, Deps1, Deps2, Deps3, Deps4.
</P><P>But none of the colonies were good.
</P><P><DETAILS><SUMMARY>WEEK 7 : 17/07 → 21/07</SUMMARY></DETAILS></P><P>Cultures from infection grew, results : TG1+ phages : lysis + irregular growth (I’ll make another one and a strike control  to make sure it’s not a TG1 normal growth.
Negative control and positive are ok 
GM1 + phages grew correctly, less than the GM1 (due to the phage infection).
</P><P>Beginning of the <A href="/Team:Aix-Marseille/Experiments/Protocols#Production_of_large_quantities_of_helper_phage" title="Team:Aix-Marseille/Experiments/Protocols">Large quantities of helper phage</A> protocol.
</P><P>PCR Clean up pa0744 / pa0745 / thioesterase (quorum sensing)
</P><P>Marie: I did a mini prep of M13KO7mutA1.1,  M13KO7mutA1.2 et  M13KO7mutA1.3. I obtained 210ng/µL, 277ng/µL and 236ng/µL.
</P><P>Myriam/Husein:
The program for the day was to try to insert the different IDT sequences that did work in a plasmid.
The sequences were Mt, P3Xfu, Deps, p3Xc, P3Ng, Pa0744, Xpr, P3Pa4, Pa0745, P3SS, P3Xfa, thioesterase, P3VCV, p3vcc, P3RSS, P3VCF.
</P><P>Firstly we did a digestion of Psb1C3 (460ng/µL and 2070 bp)
we wanted 2000 ng in 100 µL so we have taken 4 µL of Psb1C3, 0,5µL of ecoRI, 0,5µL of Xba, 10 µL of buffer 2.1, 0,5 µL PST1 and 84,5 µL of H20 qq.
</P><P>After it, we done a ligation of these sequences with the plasmid Psb1C3 (3µL per ligation)
</P><P>
Plasmid
lenght (ng)
quantity for ligation (µL)
H20qq (µL)
MT
7,5
1
13
P3SS
6,04
1
13
Xpr
7
1
13
Xc
31,74
3
11
Xfu
27,10
3
11
Ng
53,48
5
9
Vcc
39,86
4
10
Xfa
47,25
5
9
Vcv
54,06
5
9
Rss
45,51
5
9
Vcf
53,04
5
9
Deps 2.0
76,81
3
11
PA0744
55,31
6
8
PA0745
42,66
4
10
Thiostérase
40,77
4
10
</P><P>
After it we did a transformation of it and P3PA4 in DH5 alpha with 3 µL of plasmid and 50 µL of bacteria.
</P><P>  
We did a verification PCR of New Lig p3Pa4-GFP DH5alpha and an electrophoresis which showed that the ligation wasn't successful.
</P><P><DETAILS><SUMMARY>WEEK 8 : 21/07 → 28/07</SUMMARY></DETAILS></P><P>PCR control (ligations) on others colonies :
	positive results :  OriM13-EprC-KR (number 20 and 26) =&gt; starter for miniprep.
	negative results : for LacI-RBS-GFP, P3Ec-GFP, P3RsM-GFP and P3Pa-GFP.
Miniprep of LacI-RBS-KR.
Digestion test : GFP cut E/A and GFP cut N/P using the same protocol as usual. Electrophoresis shown positive results. (see the book).
</P><P>Amplification by PCR of the sequences of GFP and LacI-RBS-KR using the primer VF2 and VR, but at the end we had only about 8ng/µL =&gt; no amplification.
We tried a second time, and again no amplification. The third time, using another enzyme (phusion instead of Q5) succeeded.
</P><P>PCR clean-up of these sequences. ==&gt; 56ng/µL for LacI-RBS-KR (column 2) and 57ng/µL for GFP(column 1). (left gel)
</P><P>PCR control on colonies (transformation of biobricks): the results were good, so we could launch starters (and obtain our new killer red, &quot;super-nova&quot;). (super-nova is in the two first column.) (right gel)
</P><P>Biobricks assembly:
-Digestion:
LacI-RBS-KR and GFP used were from PCR clean-up
P3Pa : miniprep from the 04/07/17 (76 ng/µL)
P3Ec : miniprep from the 08/07/17 ( 219ng/µL)
P3RsM : miniprep from the12/07/17 (127ng/µL)
</P><TABLE><TBODY><TR><TD>LacI-KR</TD><TD>GFP</TD><TD>GFP</TD><TD>P3Pa</TD><TD>P3Ec</TD><TD>P3Rsm</TD></TR><TR><TD>Miniprep (µL) </TD><TD>1.8</TD><TD>1.8</TD><TD>8.8</TD><TD>1.3</TD><TD>0.5</TD><TD>0.8</TD></TR><TR><TD>Buffer 2.1 (µL)</TD><TD>1</TD><TD>1</TD><TD>5</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TD>Enzyme 1 (µL)</TD><TD>0.1 XbaI </TD><TD>0.1 XbaI </TD><TD>0.5 NgoMIV</TD><TD>0.1AgeI</TD><TD>0.1AgeI</TD><TD>0.1AgeI</TD></TR><TR><TD>Enzyme2 (µL)</TD><TD>0.1 (PstI)</TD><TD>0.1(PstI)</TD><TD>0.5(PstI)</TD><TD>0.1(PstI)</TD><TD>0.1(PstI)</TD><TD>0.1(PstI)</TD></TR><TR><TD>H2O (µL)</TD><TD>7</TD><TD>7</TD><TD>35.2</TD><TD>7.5</TD><TD>8.75</TD><TD>8.9</TD></TR></TBODY></TABLE><P>==&gt; 3h at 37°C. (best results), then inactivate 20min at 80°C.
</P><P>Ligation:
</P><TABLE><TBODY><TR><TD>OriM13-LacI-RBS-KR</TD><TD>LacI-RBS-GFP</TD><TD>P3Pa-GFP</TD><TD>P3Ec-GFP</TD><TD>P3RsM-GFP</TD></TR><TR><TD>Upstream (µL) </TD><TD>OriM13 cut S/P(2µL)</TD><TD>LacI-RBS cut S/P (2µL)</TD><TD>P3Pa cut A/P (2µL)</TD><TD>P3Ec cut A/P (2µL)</TD><TD>P3RsM cut A/P (2µL)</TD></TR><TR><TD>Downstream</TD><TD>LacI-RBS-KR cut X/P (2µL)</TD><TD>GFP cut X/P (2µL)</TD><TD>GFP cut N/P (2µL)</TD><TD>GFP cut N/P (2µL)</TD><TD>GFP cut N/P (2µL)</TD></TR><TR><TD>T4 DNA ligase buffer (µL)</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR><TD>T4 DNA ligase (µL)</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TD>H2O (µL)</TD><TD>13</TD><TD>13</TD><TD>13</TD><TD>13</TD><TD>13</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>Pa0745</TD><TD>P3XFa</TD><TD>P3Xc</TD><TD>P3Ec</TD><TD>D3</TD><TD>Thio</TD><TD>Xpr-SNV</TD><TD>Pa0744</TD></TR><TR><TD>Miniprep (µL)</TD><TD>2</TD><TD>2.4</TD><TD>1.7</TD><TD>0.9</TD><TD>2.8</TD><TD>1.1</TD><TD>1.3</TD><TD>1.9</TD></TR><TR><TD>Downstream</TD><TD>LacI-RBS-KR cut X/P (2µL)</TD><TD>GFP cut X/P (2µL)</TD><TD>GFP cut N/P (2µL)</TD><TD>GFP cut N/P (2µL)</TD><TD>GFP cut N/P (2µL)</TD></TR><TR><TD>T4 DNA ligase buffer (µL)</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR><TD>T4 DNA ligase (µL)</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TD>H2O (µL)</TD><TD>13</TD><TD>13</TD><TD>13</TD><TD>13</TD><TD>13</TD><TD>13</TD></TR></TBODY></TABLE><P><DETAILS><SUMMARY>WEEK 9 : 28/07 → 05/07</SUMMARY></DETAILS></P><P>PCR control (ligations) on others colonies :
	positive results :  OriM13-EprC-KR (number 20 and 26) =&gt; starter for miniprep.
	negative results : for LacI-RBS-GFP, P3Ec-GFP, P3RsM-GFP and P3Pa-GFP.
</P><P>creening results of LacI-RBS-KR, EprC-GFP, EprC-KR and OriM13-EprC-KR are okay.
</P><P>Control of plasmid minipreps by digestion (E/P) and electrophoresis.
~The roller coster~
All minipreps were supposed to be Psb1C3 plasmid, so we expected about 2000pb for the band corresponding to the plasmid.
1,3,5,7,and 8 are not Psb1C3
2,4 and 6 are good.
</P><P>
PCR control on colonies (ligations classics and over week-end):
-1 to 6: P3Ec-GFP (1700 pb): 1,2,3,4 and 6 okay (miniprep 3)
-7 to 12: P3RsM-GFP (2000pb): 7,9 and 10 okay (miniprep 7)
-13 to 18: P3Pa-GFP (2160pb): 16 and 18 okay (miniprep 16)
-19 to24: LacI-RBS-GFP (1200pb) : all are positives (miniprep 20)
-25 to 30: OriM13- LacI-RBS-KR (1700pb) : all are positives  (miniprep 30).
-31 = negative control.
</P><P>Compared to ligation over week-end at 16°C we have the same rate of positives. So we will use ligation overnight only when we will not have enough time.
week 14 to 18/08/17
Ligations at 3 parts, using purified parts (purified by PCR and Gel clean-up) in order to obtain LacI-RBS-MT and LacI-RBS-P3ss didn't work.
We decided to use the 2 parts protocol by purifying  LacI-RBS cut X/P with gel clean-up. (=&gt;need a great quantity so made more miniprep).
</P><P><DETAILS><SUMMARY>WEEK 10 : 05/07 → 12/08</SUMMARY></DETAILS></P><P>Over night at 16°C.
P3Pa and P3RsM cut A/P from the 21/07/17 were used for ligation with D3.
Transformation of ligation as usual. (3µL of ligation with 50µL of bacteria, and positive and negative control). 
PCR control of ligation:
1 to 6 = P3Xfu-D3 ; 7 to 12 = P3Xc-D3; 13 to 18 = P3Ec-D3; 19 to 24 = P3Pa-D3 ; 25 to 30= P3VcV-D3 ; 31 and 32 =P3VcF-D3 ; 33 to 36= P3RsS-D3 ; 37 to 42= P3RsM-D3 ; 43 to 48= P3Xfa-D3 ; 49 to 54= LacI-RBS-Xpr-SNV ; 55 to 60= LacI-RBS-Thio ; 61 to 66= LacI-RBS-Pa0744  ; 67 to 72 =Pa0745-Thio.
</P><P>Positives results for : P3Pa-D3 21 and 23; P3RsM-D3 39 to 42; P3Xfa-D3 43 to 48; and Pa0745-Thio 67,70 and 71.
</P><P><DETAILS><SUMMARY>WEEK 11/12 : 12/07 → 28/08</SUMMARY></DETAILS></P><P>Some others colonies were tested of the ligations that show negatives results.
</P><P>Minipreps of P3Pa-D3, P3RsM-D3, P3Xfa-D3 and Pa0745-Thio were made.
</P><P>News digestions and ligations were made for P3VcF-D3, P3VcV-D3, P3RsS-D3, P3Xfu-D3, P3Ec-D3, LacI-RBS-Thio, LacI-RBS-Pa0744, P3VcC-D3 and P3Ng-D3, using the same protocol as usual.
- 1 to 5:P3VcF-D3: 2,3 and 4 ok
-6 to 10: P3Ng-D3: 6,9and 10 ok
- 11 to 15:P3VcC-D3: 11,13,14,   15 ok
-16 to 20: P3Ec-D3: 16 ok
-21 to 25: P3VcF-D3: 21,23,25ok
-26 to 30: LacI-RBS-Thio: all are ok
-31 to 35: P3Xfu-D3:31 to 34 ok 
-36 to 40: LacI-RBS-Pa0744 : 37 to 40 ok
								    - 41 to 44:P3RsS-D3: 41 ok
PCR amplification (primers Vf2 and VR) of ligations to continu assembly. (P3Xc-GFP, LacI-RBS-Pa0744, P3RsS-D3, P3Xfu-D3, P3Ec-D3, P3VcV-D3, P3VcF-D3, P3Ng-D3, P3VcC-D3)
</P><P>Ligations of biobricks assembly into P.Bluescript to replace OriM13 worked for LacI-SNV, EprC-KR, EprC-SNV and Xpr-SNV.
</P><P>Screening of the firsts LacI-RBS-MT and LacI-RBS-P3ss show bad results so we tried the 2 parts assembly protocol, using gel extract for the purification of LacI-RBS.
week 28 to 31/08/17
</P><P>Gel extract and clean-up of LacI-RBS, then ligation with MT and P3Ss.
</P><P>1 and 2:LacI-MT
3 and 4: LacI-Deps
rest: LacI-RBS-Pa0744-Pa0745-Thio.
</P><P>We succeeded in obtain the final biobricks assembly for quorum sensing, and obtain Deps with LacI promotor.
</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>