<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aix-Marseille_Part_Collection skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aix-Marseille/Part Collection</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_button" id="display_menu_control">MENU</DIV><UL id="menu_content"><LI><A href="/Team:Aix-Marseille" title="Kill Xyl Homepage">KILL XYL</A></LI><LI><A href="/Team:Aix-Marseille/People">People</A></LI><LI><A href="/Team:Aix-Marseille/Human Practices">Human Practices</A></LI><LI><A href="/Team:Aix-Marseille/Project">Project</A></LI><LI><A href="/Team:Aix-Marseille/Notebook">Notebook</A></LI><LI><A href="/Team:Aix-Marseille/Parts">Parts</A></LI><LI><A href="/Team:Aix-Marseille/InterLab">Interlab</A></LI><LI><A href="https://igem.org/2017_Judging_Form?team=Aix-Marseille">Judging form</A></LI></UL><DIV class="igem_2017_content_wrapper"><DIV id="pageTitle"><H1>Part Collection</H1></DIV><P>
Our collection of parts is designed to allow the creation of proteins, or phage-like particles, that target a wide range of different Gram-negative bacteria. Each part corresponds to the domains 1 (D1) and 2 (D2) from the p3 protein of filamentous phages targeting the different organisms. The parts in the collection are expected to be used as fusion proteins and so conform to the <A rel="nofollow" class="external text" href="http://parts.igem.org/Assembly_standard_25">Rfc25</A> standard. To have more detail about the design of these parts, and how we used them to make phage-like particles in our project, you can check out our <A href="/Team:Aix-Marseille/M13_Design" title="Team:Aix-Marseille/M13 Design">design page</A>. 
</P><P>This part collection contains the following biobricks all in <A rel="nofollow" class="external text" href="http://parts.igem.org/Assembly_standard_25">Rfc25</A> format:
</P><UL><LI>p3_E.coli - <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255008">BBa_K2255008</A></LI><LI>p3_N.gonorrhoeae - <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255009">BBa_K2255009</A></LI><LI>p3_P.aeruginosa - <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255010">BBa_K2255010</A></LI><LI>p3_R.solanacearum_RSM1 - <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255011">BBa_K2255011</A></LI><LI>p3_R.solanacearum_RSS1 - <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255012">BBa_K2255012</A></LI><LI>p3_V.Cholerae_CTXΦ - <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255013">BBa_K2255013</A></LI><LI>p3_V.Cholerae_fs2 - <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255014">BBa_K2255014</A></LI><LI>p3_V.Cholerea_VGJΦ - <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255015">BBa_K2255015</A></LI><LI>p3_X.campestris - <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255016">BBa_K2255016</A></LI><LI>p3_X.fastidiosa - <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255017">BBa_K2255017</A></LI><LI>p3_X.fuscans - <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255018">BBa_K2255018</A></LI></UL><H2><SPAN class="mw-headline" id="Usage">Usage</SPAN></H2><P>These biobricks will be used to engineer phage-like particles targeting different bacteria. 
Check out our <A href="/Team:Aix-Marseille/M13_Design" title="Team:Aix-Marseille/M13 Design">design page</A> to know more about their design. 
</P><H2><SPAN class="mw-headline" id="Design">Design</SPAN></H2><P>The domain 3 (D3) and the signal sequence are the best conserved parts of the innophage attachment protein p3. 
Using a global protein sequence alignment (Needleman-Wunsch and <A rel="nofollow" class="external text" href="https://static.igem.org/mediawiki/parts/5/52/T--Aix-Marseille--alignement.pdf">MUSCLE alignments</A>), using two or three sequence at a time, we were able create a good alignment.
We used this to determine the domain 1 (D1) and domain 2 (D2) sequence for the attachment proteins of each phages. 
</P><P>We were able to find these domains because they are separated by flexible sequences, often composed of Glycine and Serine <SUP id="cite_ref-1" class="reference"><A href="#cite_note-1">[1]</A></SUP>. 
Then we then retrotranslated these protein sequences to obtain gene sequences and we used the iDT tool to optimise this for production in <I>E.coli</I>.
</P><TABLE><TBODY><TR><TH scope="col">Pathogene
</TH><TH scope="col">Filamentous phage
</TH><TH scope="col">GI
</TH><TH scope="col">Part ID
</TH></TR><TR><TD><I>Escherichia coli</I></TD><TD>M13 (fd,ff)<SUP id="cite_ref-Smeal_2-0" class="reference"><A href="#cite_note-Smeal-2">[2]</A></SUP></TD><TD>927334
</TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255008">BBa_K2255008</A></TD></TR><TR><TD><I>Neisseria gonorrheae</I></TD><TD>NgoΦ6<SUP id="cite_ref-3" class="reference"><A href="#cite_note-3">[3]</A></SUP></TD><TD>1260906
</TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255009">BBa_K2255009</A></TD></TR><TR><TD><I>Pseudomonas aeruginosa</I></TD><TD>Pf3<SUP id="cite_ref-4" class="reference"><A href="#cite_note-4">[4]</A></SUP></TD><TD>215374
</TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255010">BBa_K2255010</A></TD></TR><TR><TD rowspan="2"><I>Ralstonia solanacearum</I></TD><TD>RSM1Φ<SUP id="cite_ref-T.2CK_5-0" class="reference"><A href="#cite_note-T.2CK-5">[5]</A></SUP></TD><TD>5179368
</TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255011">BBa_K2255011</A></TD></TR><TR><TD>RSS1Φ<SUP id="cite_ref-T.2CK_5-1" class="reference"><A href="#cite_note-T.2CK-5">[5]</A></SUP></TD><TD>4525385
</TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255012">BBa_K2255012</A></TD></TR><TR><TD rowspan="3"><I>Vibrio Cholerea</I></TD><TD>CTXΦ<SUP id="cite_ref-Heilpern_6-0" class="reference"><A href="#cite_note-Heilpern-6">[6]</A></SUP></TD><TD>26673076
</TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255013">BBa_K2255013</A></TD></TR><TR><TD>VFJΦ(fs2)<SUP id="cite_ref-7" class="reference"><A href="#cite_note-7">[7]</A></SUP></TD><TD>1261866
</TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255014">BBa_K2255014</A></TD></TR><TR><TD>VGJΦ<SUP id="cite_ref-8" class="reference"><A href="#cite_note-8">[8]</A></SUP></TD><TD>1260523
</TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255015">BBa_K2255015</A></TD></TR><TR><TD><I>Xanthomonas campestris</I></TD><TD>ΦLf<SUP id="cite_ref-9" class="reference"><A href="#cite_note-9">[9]</A></SUP></TD><TD>3730653
</TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255016">BBa_K2255016</A></TD></TR><TR><TD><I>Xylella fastidiosa</I></TD><TD>XfasM23<SUP id="cite_ref-10" class="reference"><A href="#cite_note-10">[10]</A></SUP></TD><TD>6203562
</TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255017">BBa_K2255017</A></TD></TR><TR><TD><I>Xanthomonas fucans</I></TD><TD>XacF1<SUP id="cite_ref-11" class="reference"><A href="#cite_note-11">[11]</A></SUP></TD><TD>17150318
</TD><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2255018">BBa_K2255018</A></TD></TR></TBODY></TABLE>
Table showing the attachment proteins from various filamentous phages.<OL class="references"><LI id="cite_note-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-1">↑</A></SPAN><SPAN class="reference-text">Heilpern, A. J. &amp; Waldor, M. K. pIIICTX, a predicted CTXphi minor coat protein, can expand the host range of coliphage fd to include <I>Vibrio cholerae</I>. J. Bacteriol. 185, 1037–1044 (2003).</SPAN></LI><LI id="cite_note-Smeal-2"><SPAN class="mw-cite-backlink"><A href="#cite_ref-Smeal_2-0">↑</A></SPAN><SPAN class="reference-text">Smeal, S. W., Schmitt, M. A., Pereira, R. R., Prasad, A. &amp; Fisk, J. D. Simulation of the M13 life cycle I: Assembly of a genetically-structured deterministic chemical kinetic simulation. Virology 500, 259–274 (2017).</SPAN></LI><LI id="cite_note-3"><SPAN class="mw-cite-backlink"><A href="#cite_ref-3">↑</A></SPAN><SPAN class="reference-text">Piekarowicz, A. et al. <I>Neisseria gonorrhoeae</I> Filamentous Phage NgoΦ6 Is Capable of Infecting a Variety of Gram-Negative Bacteria. J Virol 88, 1002–1010 (2014).</SPAN></LI><LI id="cite_note-4"><SPAN class="mw-cite-backlink"><A href="#cite_ref-4">↑</A></SPAN><SPAN class="reference-text">Luiten, R. G., Schoenmakers, J. G. &amp; Konings, R. N. The major coat protein gene of the filamentous <I>Pseudomonas aeruginosa</I> phage Pf3: absence of an N-terminal leader signal sequence. Nucleic Acids Res 11, 8073–8085 (1983).</SPAN></LI><LI id="cite_note-T.2CK-5"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-T.2CK_5-0">5.0</A></SUP><SUP><A href="#cite_ref-T.2CK_5-1">5.1</A></SUP></SPAN><SPAN class="reference-text">T, K. et al. Genomic characterization of the filamentous integrative bacteriophages {phi}RSS1 and {phi}RSM1, which infect <I>Ralstonia solanacearum</I>., Genomic Characterization of the Filamentous Integrative Bacteriophages φRSS1 and φRSM1, Which Infect <I>Ralstonia solanacearum</I>. J Bacteriol 189, 189, 5792, 5792–5802 (2007).</SPAN></LI><LI id="cite_note-Heilpern-6"><SPAN class="mw-cite-backlink"><A href="#cite_ref-Heilpern_6-0">↑</A></SPAN><SPAN class="reference-text">Heilpern, A. J. &amp; Waldor, M. K. pIIICTX, a predicted CTXphi minor coat protein, can expand the host range of coliphage fd to include <I>Vibrio cholerae</I>. J. Bacteriol. 185, 1037–1044 (2003).</SPAN></LI><LI id="cite_note-7"><SPAN class="mw-cite-backlink"><A href="#cite_ref-7">↑</A></SPAN><SPAN class="reference-text">Ikema, M. &amp; Honma, Y. A novel filamentous phage, fs-2, of <I>Vibrio cholerae</I> O139. Microbiology 144, 1901–1906 (1998).</SPAN></LI><LI id="cite_note-8"><SPAN class="mw-cite-backlink"><A href="#cite_ref-8">↑</A></SPAN><SPAN class="reference-text">Campos, J. et al. VGJφ, a Novel Filamentous Phage of <I>Vibrio cholerae</I>, Integrates into the Same Chromosomal Site as CTXφ. J. Bacteriol. 185, 5685–5696 (2003).</SPAN></LI><LI id="cite_note-9"><SPAN class="mw-cite-backlink"><A href="#cite_ref-9">↑</A></SPAN><SPAN class="reference-text">Tseng, Y.-H., Lo, M.-C., Lin, K.-C., Pan, C.-C. &amp; Chang, R.-Y. Characterization of filamentous bacteriophage ΦLf from <I>Xanthomonas campestris</I> pv. campestris. Journal of general virology 71, 1881–1884 (1990).</SPAN></LI><LI id="cite_note-10"><SPAN class="mw-cite-backlink"><A href="#cite_ref-10">↑</A></SPAN><SPAN class="reference-text">Chen, J. &amp; Civerolo, E. L. Morphological evidence for phages in <I>Xylella fastidiosa</I>. Virology Journal 5, 75 (2008).</SPAN></LI><LI id="cite_note-11"><SPAN class="mw-cite-backlink"><A href="#cite_ref-11">↑</A></SPAN><SPAN class="reference-text">Ahmad, A. A., Askora, A., Kawasaki, T., Fujie, M. &amp; Yamada, T. The filamentous phage XacF1 causes loss of virulence in <I>Xanthomonas axonopodis</I> pv. citri, the causative agent of citrus canker disease. Front. Microbiol. 5, (2014).</SPAN></LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>