<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aix-Marseille_Experiments_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aix-Marseille/Experiments/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_button" id="display_menu_control">MENU</DIV><UL id="menu_content"><LI><A href="/Team:Aix-Marseille" title="Kill Xyl Homepage">KILL XYL</A></LI><LI><A href="/Team:Aix-Marseille/People">People</A></LI><LI><A href="/Team:Aix-Marseille/Human Practices">Human Practices</A></LI><LI><A href="/Team:Aix-Marseille/Project">Project</A></LI><LI><A href="/Team:Aix-Marseille/Notebook">Notebook</A></LI><LI><A href="/Team:Aix-Marseille/Parts">Parts</A></LI><LI><A href="/Team:Aix-Marseille/InterLab">Interlab</A></LI><LI><A href="https://igem.org/2017_Judging_Form?team=Aix-Marseille">Judging form</A></LI></UL><DIV class="igem_2017_content_wrapper"><DIV id="pageTitle"><H1>Protocols</H1></DIV><P><DETAILS><SUMMARY>Agarose Gel electrophoresis</SUMMARY></DETAILS></P><P>1- Preparing the agarose gel
</P><UL><LI> Measure 0.8 g of agarose powder and add 100 mL of TAE 1X</LI><LI> Melt in a micro oven until solution becomes clear</LI><LI> Let it cool; swirl occasionally to cool evenly until warm to the touch</LI><LI> Pour the melted agarose solution into the casting tray and let cool until it is solid</LI></UL><P>2- Loading the gel
</P><UL><LI> Add enough TAE 0.5 buffer so that there is about 2-3 mm of buffer over the gel</LI><LI> Add 1X loading dye to each PCR reaction</LI><LI> Record the order each sample will be loaded on the gel</LI><LI> Pipette each sample into separate wells in the gel</LI><LI> Pipette 5µL of  DNA ladder standard into at least one well</LI></UL><P>3- Running the gel 
</P><UL><LI> Turn on the power supply to about 135 volts let it run for about 25 minutes</LI><LI> Check to make sure that the current is running in the correct direction by observing the movement of the dye</LI></UL><P>4- Revelation 
</P><UL><LI> Soak into gel red for 10 minutes</LI><LI> Put the gel under UV</LI><LI> Make a copy of the image</LI></UL><P><DETAILS><SUMMARY>BioBrick assembly protocol</SUMMARY></DETAILS></P><H3><SPAN class="mw-headline" id="Digestion_protocol_for_biobrick_assembly">Digestion protocol for biobrick assembly</SPAN></H3><UL><LI>prepare the compounds as mentioned separately in 1.5 mL micro-centrifuge tubes </LI></UL><P>Upstream Part digest :
</P><TABLE><TBODY><TR><TH>Upstream Part plasmid
</TH><TD>500 ng
</TD></TR><TR><TH>EcoRI-HF
</TH><TD>0.5 µL
</TD></TR><TR><TH>SpeI
</TH><TD>0.5 µL
</TD></TR><TR><TH>10X NE Buffer 2.1
</TH><TD>5 µL
</TD></TR><TR><TH>H<SUB>2</SUB>O
</TH><TD>up to 50 µL
</TD></TR></TBODY></TABLE><P>Downstream part digest :
</P><TABLE><TBODY><TR><TH>Downstream Part plasmid
</TH><TD>500 ng
</TD></TR><TR><TH>XbaI
</TH><TD>0.5 µL
</TD></TR><TR><TH>PstI
</TH><TD>0.5 µL
</TD></TR><TR><TH>10X NE Buffer 2.1
</TH><TD>5 µL
</TD></TR><TR><TH>H<SUB>2</SUB>O
</TH><TD>up to 50 µL
</TD></TR></TBODY></TABLE><P>Destination plasmid digest :
</P><TABLE><TBODY><TR><TH>Destination Plasmid DNA
</TH><TD>500 ng
</TD></TR><TR><TH>EcoRI-HF
</TH><TD>0.5 µL
</TD></TR><TR><TH>PstI
</TH><TD>0.5 µL
</TD></TR><TR><TH>10X NE Buffer 2.1
</TH><TD>5 µL
</TD></TR><TR><TH>H<SUB>2</SUB>O
</TH><TD>up to 50 µL
</TD></TR></TBODY></TABLE><P>Incubate all three-restriction digest reactions at 37 ˚C for 30 minutes and then heat inactivate at 80 ˚C for 20 minutes. Short spin to recuperate water condensation
</P><H3><SPAN class="mw-headline" id="Ligation_protocol_for_biobrick_assembly">Ligation protocol for biobrick assembly</SPAN></H3><TABLE><TBODY><TR><TH>Upstream Part digest
</TH><TD>2 µL
</TD></TR><TR><TH>Downstream part digest
</TH><TD>2 µL
</TD></TR><TR><TH>Destination plasmid digest
</TH><TD>2 µL
</TD></TR><TR><TH>10X T4 DNA ligase buffer
</TH><TD>2 µL
</TD></TR><TR><TH>T4 DNA ligase
</TH><TD>1 µL
</TD></TR><TR><TH>H<SUB>2</SUB>O
</TH><TD>up to 11 µL
</TD></TR></TBODY></TABLE><P>Incubate at room temperature for 10 minutes 
Transform 2 µL of the ligation product into 50 µL of competent bacterial cells
</P><P><DETAILS><SUMMARY>Cloning protocol for IDT sequences</SUMMARY></DETAILS></P><H3><SPAN class="mw-headline" id="Re-suspending_gene_fragments">Re-suspending gene fragments</SPAN></H3><OL><LI>Centrifuge for 5 seconds at 3000 g</LI><LI>Add 20 µL of H<SUB>2</SUB>O to the tube for a final concentration of 10 ng/µL</LI><LI>Briefly vortex and centrifuge</LI><LI>Tube can be stored at -20 ˚C</LI></OL><H3><SPAN class="mw-headline" id="Digest_with_the_restriction_endonucleases">Digest with the restriction endonucleases</SPAN></H3><UL><LI>Add the following components</LI></UL><TABLE><TBODY><TR><TH>Product
</TH><TH>Gene Fragment
</TH><TH>Vector
</TH></TR><TR><TH>DNA
</TH><TD>100 ng
</TD><TD>500 ng
</TD></TR><TR><TH>Buffer
</TH><TD>5 µL
</TD><TD>5 µL
</TD></TR><TR><TH>Restriction Enzyme
</TH><TD>1 µL(each)
</TD><TD>1 µL(each)
</TD></TR><TR><TH>H<SUB>2</SUB>O
</TH><TD>up to 50µL
</TD><TD>up to 50µL
</TD></TR></TBODY></TABLE><UL><LI>Incubate for 45 minutes at 37 ˚C</LI><LI>Incubate for 20 minutes at 80 ˚C</LI></UL><P>Gene Fragment and vector digested separately
</P><H3><SPAN class="mw-headline" id="Ligation">Ligation</SPAN></H3><UL><LI> Add the following components in the order listed below</LI></UL><TABLE><TBODY><TR><TH>Component
</TH><TH>Amount
</TH></TR><TR><TH>Gene fragment
</TH><TD>ratio 3 to 1
</TD></TR><TR><TH>T4 buffer 1X
</TH><TD>2 µL
</TD></TR><TR><TH>T4 ligase
</TH><TD>1 µL
</TD></TR><TR><TH>H<SUB>2</SUB>O
</TH><TD>up to 20 µL
</TD></TR></TBODY></TABLE><UL><LI>calculate insert to vector quantities by this equation: required mass insert (ng) = desired insert/vector molar ratio x mass of vector (ng) x ratio of insert to vector lengths</LI></UL><UL><LI>Centrifuge for 5 seconds at 3000 g</LI><LI>Incubate at 16 ˚C overnight</LI></UL><H3><SPAN class="mw-headline" id="Transformation">Transformation</SPAN></H3><UL><LI>Reapeat protocol as mentioned previously</LI></UL><P><DETAILS><SUMMARY>Creation of phage-flurescence with carboxyfluorescein</SUMMARY></DETAILS></P><P>Requirement:
200 µL of Napi 0.5 M - 500 µL of DMSO (diméthyl sulfoxyde) - 100 mg of carboxyfluorescein - 28 mL of PBS 1X
</P><P>Prepare a solution of 1 mL of purified phages with a concentration of 7x10^8 
Dilute 100 mg of carboxyfluorescein in 500 µL of DMSO in a 1.5 mL micro-centrifuge tuve wrap up with foil.
Add 20 µL of fluorophore solution in phages solution
Keep at 4 °C during 1 hour
Transfer the complete solution in a dialyse membrane (cutoff 30 kDa) set in a 15 mL centrifugation tube containing 14 mL of PBS.
Agitate for 1h30 then change the buffer solution. Let it dialyze overnight.
The excess of fluorochrome that will not be fixed on phages will pass the membrane.
</P><P><DETAILS><SUMMARY>Gel and PCR Clean up</SUMMARY></DETAILS></P><P>Wash buffer NT3 preparation
</P><TABLE><TBODY><TR><TH>Number of preps
</TH><TH>Wash buffer NT3 concentrate
</TH><TH>ethanol
</TH></TR><TR><TH>10 preps
</TH><TD>6 mL
</TD><TD>24 mL
</TD></TR><TR><TH>50 preps
</TH><TD>20 mL
</TD><TD>80 mL
</TD></TR><TR><TH>250 preps
</TH><TD>2 X 50 mL
</TD><TD>2 X 200 mL
</TD></TR></TBODY></TABLE><P>1- Adjust DNA binding condition/Excise DNA fragment
</P><UL><LI> Mix one volume of sample with 2 volumes of buffer NT1 for PCR sample</LI><LI> Add 200 µL of NT1 buffer for each 100 mg of agarose gel</LI><LI> Incubate sample for 5-10 minutes at 50 ˚C. Vortex if needed to completely dissolve gel</LI></UL><P>2- Bind DNA
</P><UL><LI> Place PCR Clean up column into collection tube 2 mL</LI><LI> Load up to 700 µL of sample with NT1 buffer</LI><LI> Centrifuge for 30 seconds at 11 000 g</LI><LI> Discard flow through and place column back</LI></UL><P>3- Wash silica membrane 
</P><UL><LI> Add 700 µL of NT3 buffer</LI><LI> Centrifuge for 30 seconds at 11000 g</LI><LI> Discard flow through and place column back</LI></UL><P>4- Dry silica membrane 
</P><UL><LI> Centrifuge for 1 minute at 11 000 g</LI><LI> Discard flow through</LI></UL><P>5- Elute DNA
</P><UL><LI> Place column into new 1.5 mL micro centrifuge tube</LI><LI> Add 15-30 µL of ddH2O</LI><LI> let it settle for 1 minutes</LI><LI> Centrifuge for 1 minutes at 11 000 g</LI></UL><P>Note don’t discard flow through 
</P><P>5’- Elute DNA &gt; 1000 bp
</P><UL><LI> Heat ddH2O to 70˚C</LI><LI> Incubate ddH2O on column for 5 minutes</LI><LI> Add 20-30µL of ddH2O</LI><LI> Centrifuge for 1 minute at 30-50 g</LI><LI> Centrifuge for 1 minute at 11 000 g</LI><LI> Repeat elution 2 to 3 times for the best result</LI></UL><P><DETAILS><SUMMARY>Mesure of biofilm density</SUMMARY></DETAILS></P><UL><LI>Add a drop of violet cristal in each well (pipette) and let colore 10 minutes at 30 °C</LI><LI>Vaccum the liquide</LI><LI>Rince twice with water MiliQ</LI><LI>Add 250 µL physiologic water (0.9 % NaCl) + ethanol 60 %</LI><LI>Agitation during 2 minutes</LI><LI>Mesure OD570 nm and OD600 nm plaque bottom TECAN</LI><LI>Agitation during 2 minutes (amplitude 3 mm orbitale)</LI><LI>Mesure OD600 above through the plaque (25 flashes)</LI><LI>The OD600 is proportional to biofilm production</LI></UL><P><DETAILS><SUMMARY>Mini-preparation of plasmid DNA (MiniPrep) protocol</SUMMARY></DETAILS></P><P>1-Material preparation
</P><UL><LI> Centrifuge Lyse bleu and RNase samples</LI><LI> Add 20 µL of Lyse to 20 mL of Buffer P1</LI><LI> Add 200 µL of RNase to previous mix</LI></UL><P>This mix is considered as buffer P1
</P><P>2- Culture preparation
</P><UL><LI> Centrifuge for 2 minutes at 13000 rpm bacterial culture, remove supernatant </LI><LI> Re-suspend pellet with 250 µL of P1 buffer, pipette up-down</LI><LI> Add 250 µL of P2 buffer mix thoroughly by inverting the tubes 4-6 times</LI><LI> Add 350 µL of N3 buffer mix immediately and thoroughly by inverting the tubes 4-6 times</LI><LI> Centrifuge at 9000 rpm for 10 minutes </LI></UL><P>3- DNA material separation process
</P><UL><LI> Transfer 800µL of supernatant to the spin column that should be in a tube</LI><LI> Centrifuge at 9000 rpm for 45 seconds, discard flow through</LI><LI> Add 0.5 mL of PB buffer </LI><LI> Centrifuge at 9000 rpm for 45 seconds ,discard flow through</LI><LI> Add 0.75 mL of PE buffer</LI><LI> Centrifuge at 9000 rpm for 45 seconds ,discard flow through</LI><LI> Centrifuge at 9000 rpm for 1 minute , discard collection tube</LI><LI> Transfer column to a new 1.5 mL micro-centrifuge tube</LI><LI> Add 50 µL of H2O let stand for 1 minute</LI><LI>Centrifuge at 9000 rpm for 1 minute </LI><LI> Keep flow through, discard column</LI></UL><P>No special condition while preparation. After adding P2, mix N3 in less than 5 minutes
</P><P><DETAILS><SUMMARY>One-step Sequence and Ligation-Independent Cloning (SLIC)</SUMMARY></DETAILS></P><P>Procedures
</P><P>1. Digest the vector with 1 µL/µg of plasmid with restriction enzyme(s) 2 h, and purify the linearized vector with a commercial &quot;clean-up&quot; purification kit. Elute the DNA with UltraPure water. Measure the concentration of the vector.
</P><P>2. Amplify your gene of interest by PCR using primer with ≥ 15 per homology extension to the linearized vector end. We usually use 15bp homology for single fragment cloning, and 20 bp homology for multiple fragment cloning. Purify the linearized vector with a commercial PCR purification kit. Elute the DNA with elution buffer or 10 mM TrisCl, pH 8.0-8.5. Do not elute the DNA with water or TE. Measure the concentration of the insert(s).
</P><P>3. Mix the linearized vector and insert at a molar ratio of 1:2 in a 1.5 mL tube. An example is shown as follows. (Vector to insert molar ratio of 1:1 to 1:7 works well, but we usually use 1:2 for single fragment cloning 1:2:2 for multiple fragments cloning. An example of 3 fragment cloning is shown below with vector: insert 1: insert 2 molar ratio is 1:2:2 as shown
</P><TABLE><TBODY><TR><TH scope="col">Stock concentration
</TH><TH scope="col">Volume added
</TH><TH scope="col">Final concentration
</TH></TR><TR><TH scope="col">Linearized Vector
</TH><TD>0.03 pmol/µL
</TD><TD>1 µL
</TD><TD>0.003 ng/µL
</TD></TR><TR><TH scope="col">Insert 1
</TH><TD>0.15 pmol/µL
</TD><TD>1 µL
</TD><TD>0.015 pmol/µL
</TD></TR><TR><TH scope="col">Insert 2
</TH><TD>0.15 pmol/µL
</TD><TD>1 µL
</TD><TD>0.015 pmol/µL
</TD></TR><TR><TH scope="col">10X NEB Buffer 2.1
</TH><TD>10 X
</TD><TD>1 µL
</TD><TD>1 X
</TD></TR><TR><TH scope="col">H<SUB>2</SUB>O
</TH><TD>Up to 9.6 µL
</TD></TR></TBODY></TABLE><P>4. Add 0.4 µL of T4 DNA polymerase (3 U/µL; NEB) to the mixture and incubate at room temperature (&lt;27 °C) for 2.5 min.
</P><P>5. Put the reaction mixture on ice immediately to stop the reaction and incubate on ice for 10 min and thaw chemically competent <I>E. coli</I> cells on ice.
</P><P>6. Gently mix 100 µL of the competent cells with the reaction mixture and incubate on ice for 40 min. 
</P><P>7. Heat shock the cells at 42 ˚C for 2 minutes.
</P><P>8. Incubate the cells on ice for 5 min.
</P><P>9. Add 650 µL of LB broth and transfer the reaction in a 15 mL round-bottom tube.
</P><P>10. Incubate the cells at 37 ˚C for 1 hour
</P><P>11. Plate the cells on agar plates containing suitable antibiotics (eg. 100 µg/mL ampicillin). 
We usually spread 250 µL of cells onto an agar plate to get optimal number of colonies per plate for single fragment cloning.
</P><P>12. Incubate the plates at 37 ˚C for 16 hours and analyze the colonies</P><P><DETAILS><SUMMARY>Phages in soil duration protocol</SUMMARY></DETAILS></P><P><B>Preparation :</B></P><P>We need a 2 OD of F+ <I>E. coli</I> : 
Dilute bacteria 10x to read and incubate till getting 0.2 DO. 
Our phage concentration have to be lower than the bacterial concentration, we’ll use a 2.10⁸ pfu of bacterias.
So we will use 10µL of 100x times diluted phage mix : 10⁷ pfu phages per use.
</P><P><B>Bacteriophages counting test :</B></P><P>Add 1 mL of bacterias solution + 100 µL of phage 10⁸ pfu/mL solution.
Incubate for 1 h at room temperature (25°C) + 300 rpm agitation.
Spread 150 µL of mix on Kanamycin plate (LBA) and let it grow on.
Count colonies.
</P><P><B>Soil preparation :</B></P><P>Pickup ~250 g of soil.
Sift it to get only soil and discard rocks and heavier objects : keep only 200 g.
Store it in a 500 mL erlenmeyer with a cap that allows air to go through and place it near sunlight to imitate natural conditions.
Use the diluted virus mix (1,42 µL of 7.10⁸ phages in 50 mL) and add it to soil erlenmeyer, agitate to mix.
</P><P><B>T+ / T- :</B></P><P>T⁺ : soil extract (without phage) + added phage right before incubating
T⁻ : soil extract (without phage)
T- : Water
</P><P><B>Recuperation :</B></P><P>Add 10 g of soil + 20 mL of sterile water in a 50 mL centrifugation tube.
Spin 1 500 g : 10 min and transfer supernatant into an ultracentrifugation tube.
Spin for 10 min at 10 000 g and transfer supernatant into filter unit (0,22 µm).
In 2 1,5 mL tubes, put 500 µL of the filtrated solution and 500 µL of DO=2 GM1 bacteria.
Incubate for 30 min at 25 °C with slow shaking.
Spin at 5000 rpm for 10 min.
Discard 925 µL of supernatant.
Resuspend pellet of the first tube.
Transfer into the second and resuspend.
Spray on kanamycin dish.
Incubate overnight.
</P><P><DETAILS><SUMMARY>Polymerase Chain Reaction (PCR)</SUMMARY></DETAILS></P><H3><SPAN class="mw-headline" id="Reaction_setup">Reaction setup</SPAN></H3><UL><LI> Add the following component as listed below</LI></UL><TABLE><TBODY><TR><TH scope="col">Component
</TH><TH scope="col">25 µL reaction
</TH><TH scope="col">50 µL reaction
</TH><TH scope="col">100 µL reaction
</TH><TH scope="col">Final concentration
</TH></TR><TR><TH scope="col">EconoTaq PLUS GREEN 2X Master Mx
</TH><TD>12.5 µL
</TD><TD>25 µL
</TD><TD>50 µL
</TD><TD>1x
</TD></TR><TR><TH scope="col">10 µM Forward primer
</TH><TD>2.5 µL
</TD><TD>5 µL
</TD><TD>10 µL
</TD><TD>1 µM
</TD></TR><TR><TH scope="col">10 µM reverse primer
</TH><TD>2.5 µL
</TD><TD>5 µL
</TD><TD>10 µL
</TD><TD>1 µM
</TD></TR><TR><TH scope="col">DNA Template (10 ng/µL)
</TH><TD>Variable
</TD><TD>Variable
</TD><TD>Variable
</TD></TR><TR><TH scope="col">ddH<SUB>2</SUB>O
</TH><TD>To 2.5 µL
</TD><TD>To 50 µL
</TD><TD>To 100 µL
</TD></TR></TBODY></TABLE><UL><LI> Assemble all reaction components on ice and quickly transfer the reactions to thermocycle preheated to 94 °C and begin thermocycling</LI></UL><P>Hint: make a premix of the components mentioned without DNA template and allocate appropriately 12 µL in each pcr tube
</P><H3><SPAN class="mw-headline" id="Thermocycling_conditions_for_a_routine_PCR">Thermocycling conditions for a routine PCR</SPAN></H3><TABLE><TBODY><TR><TH scope="col">Cycling step
</TH><TH scope="col">Temperature
</TH><TH scope="col">Time
</TH><TH scope="col">Number of cycle
</TH></TR><TR><TH scope="col">Initial Denaturation
</TH><TD style="text-align: center;"> 94 °C
</TD><TD>2 minutes
</TD><TD style="text-align: center;"> 1
</TD></TR><TR><TH scope="col">DenaturationAnnealingExtension
</TH><TD style="text-align: center;"> 94 °C55 °C72 °C
</TD><TD>20 seconds20 seconds1 minute/Kb
</TD><TD style="text-align: center;"> 29
</TD></TR><TR><TH scope="col">Final extension
</TH><TD style="text-align: center;"> 72 °C
</TD><TD>7 minutes
</TD><TD style="text-align: center;"> 1
</TD></TR><TR><TH scope="col">Hold
</TH><TD style="text-align: center;"> 16 °C
</TD><TD>Indefinitely
</TD><TD style="text-align: center;"> 1
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Thermocycling_conditions_for_a_.22Rolling-Circle.22_PCR">Thermocycling conditions for a &quot;Rolling-Circle&quot; PCR</SPAN></H3><P>the pfu DNA polylerase is used in this case with adapted composition of the reaction mix.
</P><TABLE><TBODY><TR><TH scope="col">Cycling step
</TH><TH scope="col">Temperature
</TH><TH scope="col">Time
</TH><TH scope="col">Number of cycle
</TH></TR><TR><TH scope="col">Initial Denaturation
</TH><TD style="text-align: center;"> 94 °C
</TD><TD>2 minutes
</TD><TD style="text-align: center;"> 1
</TD></TR><TR><TH scope="col">DenaturationAnnealingExtension
</TH><TD style="text-align: center;"> 94 °C55 °C68 °C
</TD><TD>20 seconds20 seconds2 minute/Kb
</TD><TD style="text-align: center;"> 29
</TD></TR><TR><TH scope="col">Final extension
</TH><TD style="text-align: center;"> 72 °C
</TD><TD>7 minutes
</TD><TD style="text-align: center;"> 1
</TD></TR><TR><TH scope="col">Hold
</TH><TD style="text-align: center;"> 16 °C
</TD><TD>Indefinitely
</TD><TD style="text-align: center;"> 1
</TD></TR></TBODY></TABLE><P><DETAILS><SUMMARY>Preparation of competent Bacterial cells</SUMMARY></DETAILS></P><OL><LI> Recuperate the overnight bacterial culture </LI><LI> Determine the OD600 in a 1 mL Tank Spectro, dilute a 100 µL of culture in 900 µL of distilled LB</LI><LI> In 1 L Erlenmeyer add 200 mL of LB and an appropriate volume of culture to have an OD600 = 0.1 </LI><LI> Put the Erlenmeyer in an incubator for about an hour at 37 ˚C.</LI><LI> Test the OD600 for the new culture</LI><LI> The OD600 should be between 0.4 and 0.6 </LI></OL><P>All the handling done in a 15 cm radius of an open flame for optimal sterility
</P><H3><SPAN class="mw-headline" id="Preparation_of_Buffer_Tbf1_and_Tbf2">Preparation of Buffer Tbf1 and Tbf2</SPAN></H3><TABLE><TBODY><TR><TH scope="row">Tbf1 buffer
</TH><TH scope="row">Total volume 80 mL
</TH></TR><TR><TH scope="row">KAc 1 M
</TH><TD>2.4 mL
</TD></TR><TR><TH scope="row">MnCl2 0.5 M
</TH><TD>8 mL
</TD></TR><TR><TH scope="row">KCl 1 M
</TH><TD>8 mL
</TD></TR><TR><TH scope="row">CaCl<SUB>2</SUB> 0.1 M
</TH><TD>8 mL
</TD></TR><TR><TH scope="row">Gly 80 %
</TH><TD>15mL
</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O
</TH><TD>38.6 mL
</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH scope="row">Tbf2 buffer
</TH><TH scope="row">Total volume 8 mL
</TH></TR><TR><TH scope="row">NaMOPS 0.2 M
</TH><TD>400 µL
</TD></TR><TR><TH scope="row">CaCl<SUB>2</SUB> 0.1 M
</TH><TD>6 mL
</TD></TR><TR><TH scope="row">Gly 80 %
</TH><TD>1.5 mL
</TD></TR><TR><TH scope="row">KCl 1 M
</TH><TD>80 µL
</TD></TR><TR><TH scope="row">H<SUB>2</SUB>O
</TH><TD>500 µL
</TD></TR></TBODY></TABLE><P>Some solution aren’t available directly and are prepared by dissolving the solid compound
</P><TABLE><TBODY><TR><TH scope="col">Volume (mL)
</TH><TH scope="col">Mass (g)
</TH><TH scope="col">Molecular weight (g/mol)
</TH></TR><TR><TH scope="row">MnCl<SUB>2</SUB> 0.5 M
</TH><TD>200
</TD><TD>19.791
</TD><TD>197.91
</TD></TR><TR><TH scope="row">KCl 1 M
</TH><TD>200
</TD><TD>17.91
</TD><TD>74.55
</TD></TR><TR><TH scope="row">NaMOPS 0.2 M
</TH><TD>50
</TD><TD>2.0926
</TD><TD>209.26
</TD></TR><TR><TH scope="row">KAc 1 M
</TH><TD>50
</TD><TD>4.9
</TD><TD>98.15
</TD></TR></TBODY></TABLE><P>Sterilize the flask prepared after weighing in an autoclave. When preparing Tbf1 and Tbf2 should be in proximity of an open flame
</P><H3><SPAN class="mw-headline" id="Preparation_of_Competent_Bacterial_cells">Preparation of Competent Bacterial cells</SPAN></H3><OL><LI> Transfer the Bacterial culture in 50 mL centrifugation tubes</LI><LI> Centrifuge for 10 minutes at 3500 rpm in cold</LI><LI> Remove the supernatant then re-suspend the pellet in 20 mL of Tbf1 for each centrifugation tube</LI><LI> Poll in all bacterial culture in two centrifugation tubes with 40 mL each</LI><LI> Centrifuge for 5 minutes at 3500 rpm in cold</LI><LI> Remove the supernatant then re-suspend the pellet in 4 mL of Tbf2 for each centrifugation tube</LI><LI> Allocate 220 µL of competent Bacterial cells in each 1.5 micro-centrifuge tube</LI><LI> Instant freeze the 1.5 micro-centrifuge tubes in liquid nitrogen</LI><LI> Conserve the tubes at -80 ˚C</LI></OL><P>NOTE: all handling done in a cold room. To re-suspend the pellet a hard jerking action
applied on the tube. Alternatively, put on wheel for 10 minutes.
</P><P><DETAILS><SUMMARY>Production of large quantities of helper phage</SUMMARY></DETAILS></P><P><B>Requirement :</B></P><P>110 mL PEG/NaCl (from powder : 16 g NaCl + 22 g PEG 8000) - 14 mL PBS - 3 mL H-top Agar - LB - 2,5 mL Kanamycin 10 - 1 plate without antibiotics.
</P><P>Prepare a TG1 starter from glycerol or plate 
</P><P><B>Overnight</B></P><P>Needed DO = 0,4-0,5 : dilute in LB and put at 37 °C for 20 min
In a 1.5 mL micro-centrifuge tube, infect 200 µL bacteria with 10 µL phage solution 
Incubate for 30 min without shaking
In a 15 mL centrifugation tube, add 3 mL of H-top agar (42 °C)
Spread on LBA plate (no antibiotics)
</P><P><B>Overnight</B></P><P>Make a 0,4 DO TG1 5 mL starter, pick a small plaque at let it grow for 2 h at 37 °C with shaking in the starter
In a 2 L flask : pour the starter and add 500 mL of 2YT (TB)
Add 2,5 mL of kanamycin
</P><P><B>Overnight</B></P><P>In a 500 mL tube, spin for 45 min at 5000 g (10800 15 min)
Recover 400 mL of supernatant in an 1 L erlenmeyer 
Add 100 mL of PEG/NaCl 
From now, every step has to be on ice or at 4 °C
Spin for 1h30 at 5000 g in the 500 mL tubes (10800 g 30 min)
Throw away supernatant
Resuspend pellet in 8 mL PBS and add 2 mL of PEG/NaCl
Vortex for 30 s
Put in a 15 mL centrifugation tube
Let in ice for 30 min
Spin for 30 min at 3300 g 
Remove supernatant 
Add 5 mL of PBS
Resuspend pellet
Spin for 30 min at 5000 g (11000 g 10 min) 
Add 500µL of chloroform
Spin for 30 min at 5000 g
Keep supernatant
Add 10 µL of chloroform
</P><P><B>Storage :</B> 15 % glycerol at -80°C
</P><P><DETAILS><SUMMARY>Transformation protocol</SUMMARY></DETAILS></P><H3><SPAN class="mw-headline" id="Sensitivity_test_of_competent_bacterial_cells">Sensitivity test of competent bacterial cells</SPAN></H3><OL><LI>Heat up LB-agar in micro oven at 300 W for 19 minutes for 400 mL of LB-agar loosen cap before heating to let steam out</LI><LI>Transfer 80 mL of LB-agar in to Erlenmeyer one for every antibiotic</LI><LI>For 20 mL and spread the LB-agar in petri dishes, let it dry out </LI><LI>Prepare petri dishes:</LI></OL><P>25µL of competent strain spread with glass beads in all four antibiotic and one without antibiotic as negative test 
</P><TABLE><TBODY><TR><TH scope="col">Antibiotic stock
</TH><TH scope="col">Desired concentration
</TH><TH scope="col">Volume of antibiotic per mL of LB-agar
</TH><TH scope="col">Volume in Erlenmeyer
</TH></TR><TR><TD><STRONG style="color:CornflowerBlue">Ampicillin</STRONG>25 mg/mL
</TD><TD>100 µg/mL
</TD><TD>4 µL/mL
</TD><TD>320 µL
</TD></TR><TR><TD><STRONG style="color:crimson">Kanamycin</STRONG>10 mg/mL
</TD><TD>50 µg/mL
</TD><TD>5 µL/mL
</TD><TD>400 µL
</TD></TR><TR><TD><STRONG>Tetracycline</STRONG>15 mg/mL
</TD><TD>15 µg/mL
</TD><TD>1 µL/mL
</TD><TD>80 µL
</TD></TR><TR><TD><STRONG style="color:SeaGreen">Chloramphenicol</STRONG>30 mg/mL
</TD><TD>50 µg/mL
</TD><TD>1.6 µL/mL
</TD><TD>128 µL
</TD></TR></TBODY></TABLE><P>Handling is done near an open flame, sterile environment
</P><H3><SPAN class="mw-headline" id="Preparation_of_DNA_material_from_iGEM_kit">Preparation of DNA material from iGEM kit</SPAN></H3><OL><LI>Pierce plate at the desired location</LI><LI>Hydrate by injecting 10 µL of ddH2O</LI><LI>Let it soak for 5 minutes until red coloration is highly visible</LI><LI>pipet up-down to homogenize the solution</LI><LI>Transfer the solution into a 1.5 mL micro-centrifuge tube set it on ice</LI></OL><P>No special conditions needed when manipulating just respect sterility
</P><H3><SPAN class="mw-headline" id="Transformation_of_bacterial_cells_with_plasmid.2Fligation">Transformation of bacterial cells with plasmid/ligation</SPAN></H3><OL><LI>Add an adequate quantity of DNA in a 1.5 mL micro-centrifuge tube</LI><LI>Add 100 µL (50 µL) of competent bacterial cells</LI><LI>Put on ice for 20 minutes</LI><LI>Put tubes in thermomixer at 42 ˚C for 45 seconds</LI><LI>Put on ice for 5 minutes </LI><LI>Add 450 µL (900 µL) of LB</LI><LI>Incubate tube for 1 hour at 37 ˚C </LI><LI>Centrifuge for 5 minutes at 5000 rpm (normal centrifuge will work)</LI><LI>Remove 400 µL (850 µL) of supernatant</LI><LI>Re-suspend pellet in the 150 µL of remaining medium pipet up-down</LI><LI>Spread total volume on petri dish with the appropriate antibiotic</LI><LI>Put petri dishes in an incubator overnight at 37 ˚C </LI></OL><P>For a negative control repeat procedure without adding plasmid
</P><P>Handling is done near an open flame, sterile environment
</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>