Team:Aix-Marseille/References
MENU
KILL XYL
People
Human Practices
Project
Notebook
Parts
Interlab
Judging form
References
Salmond, G. P. C. & Fineran, P. C. A century of the phage: past, present and future. Nat Rev Micro 13, 777–786 (2015).
Ikema, M. & Honma, Y. A novel filamentous phage, fs-2, of Vibrio cholerae O139. Microbiology 144, 1901–1906 (1998).
Honey, S., Schneider, B. L., Schieltz, D. M., Yates, J. R. & Futcher, B. A novel multiple affinity purification tag and its use in identification of proteins associated with a cyclin–CDK complex. Nucleic Acids Res 29, e24 (2001).
Menouni, R., Hutinet, G., Petit, M.-A. & Ansaldi, M. Bacterial genome remodeling through bacteriophage recombination. FEMS Microbiol. Lett. 362, 1–10 (2015).
Rutherford, S. T. & Bassler, B. L. Bacterial Quorum Sensing: Its Role in Virulence and Possibilities for Its Control. Cold Spring Harb Perspect Med 2, a012427 (2012).
Buttimer, C. et al. Bacteriophages and Bacterial Plant Diseases. Front. Microbiol. 8, (2017).
Kim, W. S. & Geider, K. Characterization of a Viral EPS-Depolymerase, a Potential Tool for Control of Fire Blight. Phytopathology 90, 1263–1268 (2000).
Tseng, Y.-H., Lo, M.-C., Lin, K.-C., Pan, C.-C. & Chang, R.-Y. Characterization of filamentous bacteriophage ΦLf from Xanthomonas campestris pv. campestris. Journal of general virology 71, 1881–1884 (1990).
Ahern, S. J., Das, M., Bhowmick, T. S., Young, R. & Gonzalez, C. F. Characterization of Novel Virulent Broad-Host-Range Phages of Xylella fastidiosa and Xanthomonas. J Bacteriol 196, 459–471 (2014).
S, K., J, M., A, C. & D, K. Characterizations of highly expressed genes of four fast-growing bacteria., Characterizations of Highly Expressed Genes of Four Fast-Growing Bacteria. J Bacteriol 183, 183, 5025, 5025–5040 (2001).
Pa, V. & Rl, C. Cloning and Expression in Escherichia coli of the Polysaccharide Depolymerase Associated with Bacteriophage-Infected Erwinia amylovora., Cloning and Expression in Escherichia coli of the Polysaccharide Depolymerase Associated with Bacteriophage-infected Erwinia amylovora. Appl Environ Microbiol 51, 51, 862, 862–864 (1986).
Araújo, W. L. et al. Diversity of Endophytic Bacterial Populations and Their Interaction with Xylella fastidiosa in Citrus Plants. Appl. Environ. Microbiol. 68, 4906–4914 (2002).
Chopin, M.-C., Rouault, A., Ehrlich, S. D. & Gautier, M. Filamentous Phage Active on the Gram-Positive Bacterium Propionibacterium freudenreichii. J. Bacteriol. 184, 2030–2033 (2002).
A. Lukyanov, K., O. Serebrovskaya, E., Lukyanov, S. & M. Chudakov, D. Fluorescent proteins as light-inducible photochemical partners. Photochemical & Photobiological Sciences 9, 1301–1306 (2010).
T, K. et al. Genomic characterization of the filamentous integrative bacteriophages {phi}RSS1 and {phi}RSM1, which infect Ralstonia solanacearum., Genomic Characterization of the Filamentous Integrative Bacteriophages φRSS1 and φRSM1, Which Infect Ralstonia solanacearum. J Bacteriol 189, 189, 5792, 5792–5802 (2007).
Hodyra, K. & Dąbrowska, K. Molecular and chemical engineering of bacteriophages for potential medical applications. Arch. Immunol. Ther. Exp. (Warsz.) 63, 117–127 (2015).
Chen, J. & Civerolo, E. L. Morphological evidence for phages in Xylella fastidiosa. Virology Journal 5, 75 (2008).18.Piekarowicz, A. et al. Neisseria gonorrhoeae Filamentous Phage NgoΦ6 Is Capable of Infecting a Variety of Gram-Negative Bacteria. J Virol 88, 1002–1010 (2014).
Vandenbergh, P. A., Wright, A. M. & Vidaver, A. K. Partial Purification and Characterization of a Polysaccharide Depolymerase Associated with Phage-Infected Erwinia amylovora. Appl. Environ. Microbiol. 49, 994–996 (1985).
Heilpern, A. J. & Waldor, M. K. pIIICTX, a predicted CTXphi minor coat protein, can expand the host range of coliphage fd to include Vibrio cholerae. J. Bacteriol. 185, 1037–1044 (2003).
Bevan, M. Plant pathology: The bugs from Brazil. Nature 406, 140–141 (2000).
Czapar, A. E. & Steinmetz, N. F. Plant viruses and bacteriophages for drug delivery in medicine and biotechnology. Current Opinion in Chemical Biology 38, 108–116 (2017).
Ionescu, M. et al. Promiscuous Diffusible Signal Factor Production and Responsiveness of the Xylella fastidiosa Rpf System. mBio 7, e01054-16 (2016).
Smeal, S. W., Schmitt, M. A., Pereira, R. R., Prasad, A. & Fisk, J. D. Simulation of the M13 life cycle I: Assembly of a genetically-structured deterministic chemical kinetic simulation. Virology 500, 259–274 (2017).
Smeal, S. W., Schmitt, M. A., Pereira, R. R., Prasad, A. & Fisk, J. D. Simulation of the M13 life cycle II: Investigation of the control mechanisms of M13 infection and establishment of the carrier state. Virology 500, 275–284 (2017).
Matsumoto, A. & Igo, M. M. Species-Specific Type II Restriction-Modification System of Xylella fastidiosa Temecula1. Appl. Environ. Microbiol. 76, 4092–4095 (2010).
Ryan, R. P., An, S., Allan, J. H., McCarthy, Y. & Dow, J. M. The DSF Family of Cell–Cell Signals: An Expanding Class of Bacterial Virulence Regulators. PLOS Pathogens 11, e1004986 (2015).
Ahmad, A. A., Askora, A., Kawasaki, T., Fujie, M. & Yamada, T. The filamentous phage XacF1 causes loss of virulence in Xanthomonas axonopodis pv. citri, the causative agent of citrus canker disease. Front. Microbiol. 5, (2014).
Simpson, A. J. G. et al. The genome sequence of the plant pathogen Xylella fastidiosa. Nature 406, 151–158 (2000).
Luiten, R. G., Schoenmakers, J. G. & Konings, R. N. The major coat protein gene of the filamentous Pseudomonas aeruginosa phage Pf3: absence of an N-terminal leader signal sequence. Nucleic Acids Res 11, 8073–8085 (1983).
Amari, D. T., Marques, C. N. H. & Davies, D. G. The Putative Enoyl-Coenzyme A Hydratase DspI Is Required for Production of the Pseudomonas aeruginosa Biofilm Dispersion Autoinducer cis-2-Decenoic Acid. J. Bacteriol. 195, 4600–4610 (2013).
Dt, A., Cn, M. & Dg, D. The putative enoyl-coenzyme A hydratase DspI is required for production of the Pseudomonas aeruginosa biofilm dispersion autoinducer cis-2-decenoic acid., The Putative Enoyl-Coenzyme A Hydratase DspI Is Required for Production of the Pseudomonas aeruginosa Biofilm Dispersion Autoinducer cis-2-Decenoic Acid. J Bacteriol 195, 195, 4600, 4600–4610 (2013).
Li, Y. et al. Type I and type IV pili of Xylella fastidiosa affect twitching motility, biofilm formation and cell–cell aggregation. Microbiology 153, 719–726 (2007).
Campos, J. et al. VGJφ, a Novel Filamentous Phage of Vibrio cholerae, Integrates into the Same Chromosomal Site as CTXφ. J. Bacteriol. 185, 5685–5696 (2003).35.Roldão, A., Silva, A. C., Mellado, M. C. M., Alves, P. M. & Carrondo, M. J. T. Viruses and Virus-Like Particles in Biotechnology: Fundamentals and Applications. in Reference Module in Life Sciences (Elsevier, 2017). doi:#1016/B978-0-12-809633-8.09046-4
Wells, J. M. et al. Xylella fastidiosa gen. nov., sp. nov: Gram-Negative, Xylem-Limited, Fastidious Plant Bacteria Related to Xanthomonas spp. International Journal of Systematic and Evolutionary Microbiology 37, 136–143 (1987).
Hopkins, D. L. Xylella Fastidiosa: Xylem-Limited Bacterial Pathogen of Plants. Annual Review of Phytopathology 27, 271–290 (1989).
