<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CIEI-BJ_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CIEI-BJ/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="backtop-wrapper"><A href="#" class="back-link"><DIV class="back-txt">Backtop</DIV></A></DIV><HEADER class="header-navigation" id="header"><DIV class="navi_body"><DIV class="navi_head"><DIV style="width:100%; margin-left:auto; margin-right:auto;"><SPAN><DIV class="navi_title"><A href="https://2017.igem.org/Team:CIEI-BJ">Home</A></DIV></SPAN><SPAN><DIV class="navi_title">Project</DIV><DIV><A href="https://2017.igem.org/Team:CIEI-BJ/Description">Description</A></DIV><DIV><A href="https://2017.igem.org/Team:CIEI-BJ/Background">Background</A></DIV><DIV><A href="https://2017.igem.org/Team:CIEI-BJ/Design">Our Design</A></DIV><DIV><A href="https://2017.igem.org/Team:CIEI-BJ/Results">Results</A></DIV><DIV><A href="https://2017.igem.org/Team:CIEI-BJ/Parts">Parts</A></DIV></SPAN><SPAN><DIV class="navi_title">Human Practice</DIV><DIV><A href="https://2017.igem.org/Team:CIEI-BJ/HP/Overview">Overview</A></DIV><DIV><A href="https://2017.igem.org/Team:CIEI-BJ/HP/Silver">Silver</A></DIV><DIV><A href="https://2017.igem.org/Team:CIEI-BJ/HP/Gold_Integrated">Gold</A></DIV><DIV><A href="https://2017.igem.org/Team:CIEI-BJ/Collaborations">Collaboration</A></DIV></SPAN><SPAN><DIV class="navi_title"><A href="https://2017.igem.org/Team:CIEI-BJ/Attributions">Attribution</A></DIV></SPAN><SPAN><DIV class="navi_title"><A href="https://2017.igem.org/Team:CIEI-BJ/Achievement">Achievement</A></DIV></SPAN><SPAN><DIV class="navi_title">Notebook</DIV><DIV><A href="https://2017.igem.org/Team:CIEI-BJ/Protocol">Protocol</A></DIV><DIV><A href="https://2017.igem.org/Team:CIEI-BJ/Notebook">Notebook</A></DIV><DIV><A href="https://2017.igem.org/Team:CIEI-BJ/Safety">Safety</A></DIV></SPAN><SPAN><DIV class="navi_title"><A href="https://2017.igem.org/Team:CIEI-BJ/Team">Team</A></DIV></SPAN></DIV></DIV></DIV></HEADER><DIV id="my-container"><DIV id="my-sidebar"><DIV class="sidebar__inner"><UL class="page-anchors"><LI><A href="#a1"><I>PdGES</I></A></LI><LI><A href="#a2">Optimization of PdGES</A></LI><LI><A href="#a3">Amplification of  <I> PdGES </I></A></LI><LI><A href="#a4">Construction of vectors</A></LI></UL><LI><A href="#a5"><I> PcGES </I></A></LI><LI><A href="#a6">Optimization of PcGES</A></LI><LI><A href="#a7">HbOYE</A></LI><LI><A href="#a8">Sequence of HbOYE</A></LI><LI><A href="#a9">Amplification of HbOYE</A></LI><LI><A href="#a10">HbOYE + pET32a vector</A></LI><LI><A href="#a11"><I>HbOYE</I> + pPIC9K vector</A></LI><LI><A href="#a12"><I>HbOYE</I>+<I> 2A </I>+<I>GES </I>to pPIC9K vector and pET32a vector</A></LI><LI><A href="#a13">ScOYE</A></LI><LI><A href="#a14">nucleotide sequence of ScOYE</A></LI><LI><A href="#a15">Amplification of ScOYE</A></LI><LI><A href="#a16"><I>ScOYE</I>in  <I> E.Coli </I></A></LI><LI><A href="#a17"><I>ScOYE</I> in Yeast</A></LI><LI><A href="#a18">Model</A></LI><LI><A href="#a19">Citronellol’s Effect in Repelling Mosquitoes</A></LI></DIV></DIV><DIV id="my-adjust-content"><DIV class="first-level" id="a1"><I>PdGES</I></DIV><P class="my-content">PdGES, Phyla dulcis geraniol synthase (GES), is an enzyme which bolsters the production of geraniol. Through the MVA or DXP pathway, Geranyl Diphosphate（GPP）can be  produced by glucose. The function of <I>PdGES</I> is to convert GPP into geraniol. This enzyme is one of the essential proteins in our process, since it produces geraniol in the first place.</P><DIV class="second-level" id="a2">Optimization of PdGES</DIV><DIV class="third-level">The nucleotide sequence before optimization</DIV><P class="my-content"><B>(GenBank: GU136162.1)</B></P><P class="my-content">ATGGCGAGTGCAAGAAGCACCATATCTTTGTCCTCACAGTCATCTCATCATGGGTTCTCCAAAAACTCATTTCCATGGCAACTGAGGCATTCCCGCTTTGTTATGGGTTCTCGAGCACGTACCTGCGCATGCATGTCATCATCAGTATCACTGCCTACTGCAACGACGTCGTCCTCAGTCATTACAGGCAACGATGCCCTCCTCAAATACATACGTCAGCCTATGGTAATTCCTTTGAAAGAAAAGGAGGGCACGAAGAGACGAGAATATCTGCTGGAGAAAACTGCAAGGGAACTGCAGGGAACTACGGAGGCAGCGGAGAAACTGAAATTCATTGATACAATCCAACGGCTGGGAATCTCTTGCTATTTCGAGGATGAAATCAACGGCATACTGCAGGCGGAGTTATCCGATACTGACCAGCTTGAGGACGGCCTCTTCACAACGGCTCTACGCTTCCGTTTGCTCCGTCACTACGGCTACCAAATCGCTCCCGACGTCTTCCTAAAATTCACGGACCAAAATGGAAAATTCAAAGAATCCTTAGCGGATGACACACAAGGATTAGTCAGCTTATACGAAGCATCATATATGGGAGCAAACGGAGAAAACATATTAGAAGAAGCTATGAAATTCACCAAAACTCATCTCCAAGGAAGACAACATGCGATGAGAGAAGTGGCTGAAGCCTTGGAGCTTCCGAGGCATCTGAGAATGGCCAGGTTAGAAGCAAGAAGATACATCGAACAATATGGTACAATGATTGGACATGATAAAGACCTCTTGGAGCTAGTAATATTGGACTATAACAATGTCCAGGCTCAGCACCAAGCGGAACTCGCCGAAATTGCCAGATGGTGGAAGGAGCTTGGTCTAGTTGACAAGTTAACTTTCGCGCGAGATAGACCATTGGAGTGCTTTTTGTGGACTGTCGGTCTTCTACCTGAACCCAAATACTCTGCTTGCCGAATCGAGCTCGCAAAAACAATAGCCATTCTATTGGTAATCGATGATATCTTCGATACCTATGGGAAAATGGAAGAACTCGCTCTTTTCACGGAGGCAATTAGAAGATGGGATCTTGAAGCTATGGAAACCCTTCCCGAGTACATGAAAATATGCTATATGGCATTGTACAATACCACCAACGAGATATGCTACAAAGTCCTCAAGAAAAATGGATGGAGTGTTCTCCCATACCTAAGATATACGTGGATGGACATGATAGAAGGTTTTATGGTGGAGGCAAAGTGGTTCAATGGTGGAAGTGCTCCAAACTTGGAAGAGTACATAGAGAATGGAGTCTCAACGGCTGGGGCATACATGGCTTTGGTGCATCTCTTCTTTCTAATTGGGGAAGGTGTCAGTGCGCAAAATGCCCAAATATTACTGAAGAAACCCTATCCTAAGCTCTTCTCGGCTGCCGGTCGAATTCTTCGCCTTTGGGATGATCTTGGAACGGCTAAGGAGGAGGAAGGAAGAGGTGATCTTGCATCGAGCATACGTTTATTCATGAAAGAAAAGAACCTAACAACGGAAGAGGAAGGGAGAAATGGTATACAGGAGGAGATATATAGCTTATGGAAAGACCTAAACGGAGAGCTCATTTCTAAAGGTAGGATGCCATTGGCCATCATCAAAGTGGCACTTAACATGGCTAGAGCTTCTCAAGTGGTGTACAAGCATGACGAGGACTCTTATTTTTCATGTGTAGACAATTATGTGGAGGCCCTGTTCTTCACTCCTCTCCTTTGA</P><DIV class="third-level">Optimization</DIV><P class="my-content">In order to assure that <I> PdGES </I> ligate to the multiple cloning sites of the vector successfully, we avoid all the common restriction enzyme cutting sites （<I>Eco</I>R I, <I> Xba </I> I, <I> Spe </I> I, <I> Pst </I> I, <I>Bam</I>H I, <I> Bgl </I> II, <I>Hin</I>d III, <I> Kpn </I> I, <I> Nco </I> I, <I> Nde </I> I, <I> Not </I> I, <I> Sal </I> I, <I> Xho </I> I, Polymerase slippage site, Polymerase slippage site, Frameshift element, ribosome binding site）when the synthesis is being performed.</P><P class="my-content">To make the parts established by genes satisfy the need of constructing the vector pSBIC3, we avoided the restriction enzyme cutting sites, which are  <I>Eco</I>R Ⅰ, <I> Xba </I> I, <I> Spe </I> I, <I> Pst </I> I.</P><P class="my-content">To ligate genes directly to the standard vector, we added the restriction enzyme cutting sites  of  <I> Xba </I> Ⅰ and  <I> Spe </I> Ⅰ to the upstream and downstream of target gene.</P><DIV class="third-level">The optimized sequence</DIV><P class="my-content">ATGGCGAGCGCGCGTAGCACCATCAGCCTGAGCAGCCAAAGCAGCCACCACGGTTTCAGCAAAAACAGCTTTCCGTGGCAGCTGCGTCACAGCCGTTTCGTTATGGGCAGCCGTGCGCGTACCTGCGCGTGCATGAGCAGCAGCGTGAGCCTGCCGACCGCGACCACCAGCAGCAGCGTGATTACCGGTAACGACGCGCTGCTGAAGTATATCCGTCAGCCGATGGTGATTCCGCTGAAGGAGAAAGAGGGCACCAAGCGTCGTGAATACCTGCTGGAGAAAACCGCGCGTGAGCTGCAAGGCACCACCGAGGCGGCGGAAAAGCTGAAATTCATCGACACCATTCAGCGTCTGGGTATCAGCTGCTACTTTGAGGATGAAATCAACGGCATTCTGCAAGCGGAACTGAGCGACACCGATCAGCTGGAGGATGGTCTGTTCACCACCGCGCTGCGTTTTCGTCTGCTGCGTCACTACGGCTATCAAATCGCGCCGGACGTTTTCCTGAAATTTACCGATCAGAACGGCAAGTTCAAAGAAAGCCTGGCGGACGATACCCAAGGCCTGGTGAGCCTGTACGAAGCGAGCTATATGGGTGCGAACGGCGAGAACATTCTGGAGGAAGCGATGAAGTTTACCAAAACCCACCTGCAAGGTCGTCAGCACGCGATGCGTGAGGTTGCGGAAGCGCTGGAGCTGCCGCGTCACCTGCGTATGGCGCGTCTGGAAGCGCGTCGTTACATCGAGCAGTATGGCACCATGATTGGCCACGACAAGGATCTGCTGGAGCTGGTGATCCTGGACTACAACAACGTTCAGGCGCAACACCAGGCGGAACTGGCGGAGATTGCGCGTTGGTGGAAGGAGCTGGGTCTGGTTGACAAACTGACCTTCGCGCGTGATCGTCCGCTGGAATGCTTTCTGTGGACCGTGGGTCTGCTGCCGGAACCGAAATATAGCGCGTGCCGTATCGAGCTGGCGAAGACCATCGCGATTCTGCTGGTTATCGACGATATTTTCGACACCTACGGTAAAATGGAGGAACTGGCGCTGTTTACCGAAGCGATTCGTCGTTGGGATCTGGAAGCGATGGAGACCCTGCCGGAGTATATGAAAATCTGCTACATGGCGCTGTATAACACCACCAACGAAATTTGCTACAAGGTGCTGAAGAAAAACGGTTGGAGCGTTCTGCCGTACCTGCGTTATACCTGGATGGATATGATCGAAGGCTTCATGGTGGAGGCGAAGTGGTTTAACGGTGGCAGCGCGCCGAACCTGGAGGAATATATTGAGAACGGTGTGAGCACCGCGGGCGCGTACATGGCGCTGGTTCACCTGTTCTTTCTGATCGGTGAAGGCGTGAGCGCGCAAAACGCGCAGATTCTGCTGAAGAAACCGTATCCGAAACTGTTCAGCGCGGCGGGTCGTATCCTGCGTCTGTGGGACGATCTGGGCACCGCGAAAGAGGAAGAGGGTCGTGGCGACCTGGCGAGCAGCATTCGTCTGTTTATGAAGGAGAAAAACCTGACCACCGAAGAGGAAGGTCGTAACGGCATCCAAGAGGAAATTTACAGCCTGTGGAAGGATCTGAACGGTGAACTGATCAGCAAAGGCCGTATGCCGCTGGCGATCATTAAGGTGGCGCTGAACATGGCGCGTGCGAGCCAGGTGGTTTACAAACACGACGAAGATAGCTATTTCAGCTGCGTGGACAACTACGTTGAGGCGCTGTTCTTTACCCCGCTGCTGTAA</P><DIV class="third-level">Codon Adaptation Index (CAI)</DIV><P class="my-content">PdGES</P><P class="my-content">CAI: 0.97</P><P class="my-content">Fig. 1-1 The distribution of codon usage frequency along the length of the gene sequence. A CAI of 1.0 is considered to be perfect in the desired expression organism, and a CAI of &gt; 0.8 is regarded as good, in terms of high gene expression level.</P><DIV class="second-level" id="a3">Amplification of  <I> PdGES </I></DIV><DIV class="third-level">carrier</DIV><DIV class="forth-level">pUC57 vector</DIV><P class="my-content">Introduction of pUC57:</P><P class="my-content">pUC57 and it is a cloning vector with size of 2710 bp 2710 bp. The promoter of pUC57 is lac and the replicon is ColE1 origin and the resistance of pUC57 is the ampicillin in prokaryotic. In addition, the aerobic environment with 37 degree Celsius in LB medium is optimal condition for pUC57 cultivation. In our experiment, we got gene  PdGES and PcGES from pUC57 as our initial materials.</P><P class="my-content">The sequences of common primer for pUC57:</P><P class="my-content">5' sequencing primer M13R: CAGGAAACAGCTATGACC</P><P class="my-content">3' sequencing primer M13F: TGTAAAACGACGGCCAGT</P><P class="my-content">Fig. 1-2 The plasmid portfolio of pUC57</P><DIV class="third-level">Amplification of  <I> PdGES </I></DIV><P class="my-content">To obtain the target gene  <I> PdGES </I>, by using the specific primers to clone the targeted gene, our team amplified the template----vector pUC57 with  <I> PdGES </I> ---- through PCR.</P><P class="my-content">The sequence of forward primer is: AAAAACAACTAATTATTCGAAGATGGCGAGCGCGCGTAG</P><P class="my-content">The sequence of reverse primer is: AGGCGAATTAATTCGCGGCGCTTTACAGCAGCGGGGTAA</P><P class="my-content">As shown is figure 1-3, the size of PdGES matches with the theoretical size of PdGES which is 1755 bp, The results proved that we had  successfully cloned the gene <I> PdGES </I>.</P><P class="my-content">Fig. 1-3 Electrophoresis of  <I> PdGES </I> PCR product.</P><P class="my-content">The left column is the marker, and the two strips on the right are PCR products of  <I> PdGES </I>.</P><P class="my-content">The sizes of the two strips are both about 2000 bp.</P><DIV class="second-level" id="a4">Construction of vectors</DIV><DIV class="third-level">Construction of yeast expression vector pPIC9K</DIV><P class="my-content">pPIC9K is a carrier for protein expression of in yeast of of 9276bp. The promoter of pPIC9K is AOX1 and the resistance of pPIC9K is the Ampicillin.</P><P class="my-content">The sequences of common primer for pPIC9K:</P><P class="my-content">5' sequencing primer is 5´-GACTGGTTCCAATTGACAAGC-3´</P><P class="my-content">3' sequencing primer is 5´-GCAAATGGCATTCTGACATCC-3´</P><P class="my-content">Fig. 1-4 The plasmid portfolio of pPIC9K</P><P class="my-content">The vector pPIC9K used in our experiment doesn’t have the S part, because the S part is the secretion signal which can remove the protein we produce in the experiment. Since our purpose is to obtain the protein, we have to cut this part to make our experiment successful.</P><P class="my-content">As shown in the diagram below, PdGES has been ligated to the yeast vector pPIC9K successfully. The process is achieved by digestion and ligation to  <I>Bam</I>HI and  <I>Not</I>I.  Finally, we transformed the ligation product  into competent cell yeast  <I> GS115</I>.</P><P class="my-content">Fig. 1-5 Electrophoresis of  <I> PdGES </I> -contained pPIC9K vector.</P><P class="my-content">The left column is the marker and the strips on the right are the PCR products of ligation of PdGES.</P><DIV class="third-level">Construction of  <I> E. Coli </I> expression vector pET32a</DIV><P class="my-content">pET32a is a carrier for the expression of protein in  <I> E.Coli </I> of 5900bp. The promoter of pET32a is T7, which is a strong promotor that can express at any time, any cell under any circumstance. The resistance of pET32a is the Ampicillin.</P><P class="my-content">The sequences of primer for pET32a:</P><P class="my-content">5' sequencing primer is 5´-TAATACGACTCACTATAGGG-3´</P><P class="my-content">3' sequencing primer is 5´-GCTAGTTATTGCTCAGCGG-3´</P><P class="my-content">Fig. 1-6 The plasmid portfolio of pET32a</P><P class="my-content">We tried to ligate  <I> PdGES</I> gene into pET32a vector, but after three trails, we failed to ligate the intracellular device in pET32a. The possible explanation is that the condition for ligation is not suitable or the sequence of this gene.</P><DIV class="first-level" id="a5"><I> PcGES </I></DIV><P class="my-content">PcGES, Pogostemon cablin geraniol synthase (GS1), is an also enzyme which bolsters the production of geraniol. Through the MVA or DXP pathway, Geranyl Diphosphate（GPP）can be  produced. The fuction of  <I> PcGES </I> is to convert GPP to geraniol. PcGES functions the same as PdGES, the only difference is that they are from different species. By applying both GESs in our experiments, we can find out which one is more efficient by analyzing the efficiency of production.</P><DIV class="second-level" id="a6">Optimization of PcGES</DIV><DIV class="third-level">The nucleotide sequence before optimization</DIV><P class="my-content">(GenBank: KF926075.1)</P><P class="my-content">ATGTCTTGTGCTAGAAGCTACACCATTCCTTTGTCCTTCCCCAAAACCTCTAATTCACCATTGCAACTAAAGAATCTCCTCCCTTTCCCCGCCGCCCGGTCGCGTTTTTCCGTGCGCATGTCCACCTCGTCCCTTCCCGTTGGCAACGAAGCCCTACTAAAATACATTCGACAACCCGTGGTGTTGCCTACGGAAGAAGATGAGAGCATCAAGAGGAGAGATTATTTGCTCGAAAAAACTGCGAGGAGGTTAAGGACGAGTACGGGTAGTGTGGAGAAGCTTAAGCTTATCGATACGATCCAACGACTAGGAATCGATTATTATTTGGAGGACGATATAAACGTAGTACTTAGAAATGAGTACTATAATGGTAGGTCTAGTGAAGAAGACCTCTTCACTACAGCTCTAAGATTTCGTTTGCTTCGTCACAACGGCTTCCAAATTAGTTCTGATGTATTCATGAAATTCAAGGACAAAAATGGAAAATTCAAAAAATGGATAGCTGAGGATACAATAGGGTTACTGAGCTTATATGAGGCGTCGTATATGGGAGCTCATGGTGAAAAAATATTGGAAGAAGCGATGGAATTCTCGAGGTCACTCCTCAAGAGATCGCTTCCTCAGTTGCCTCCGAAGCTCCACGGGCAGGTGGCTCAAGCCTTGGAGCTTCCGAGACACCTGAGGATGGCTAGATTAGAAGCTCGACGGTTTGTCCAGCAATACGCTAAACAAAGTGACTGCGATCGTGACCTTTTGAACCTAGCAACATTGGATTATAACAAGGTTCAATTGCAGCACCAATTTGAACTTGCCGAAATTACAAGGTGGTGGCAACAGCTTGGTCTAGTAGAAAAGTTAACATTTGCACGAGATAGACCGTTGGAGTGTTTTTTGTGGACCGTTGGATTACTCCCAGAACCCAGATATTCCACGTGTCGAATTGAGATGGCCAAGACCATTGCTATTTTATTAGTCATTGACGATATTTTCGACACGTATGGCAAAATGGATGAACTCCTCCTCTTCACCCAAGCTATTAGAAGATGGGATCTTGAAGCAATGGATATCCTTCCGGAATACATGAAAATATGTTACATGGCATTATACAACACAACTAATGAAATATGCTACAAAGTGCTCAAACTCAATGGATGGAGTGTCCTTTCTTACCTTAAATCTACGTGGATAGATATGATAGAAGGTTTTATGGTGGAGGCAAAATGGTTGAATGGTGGTGGTGCACCAAACTTGGAAGAGTACCTTGAGAATGGAGTGTCTACGGCGGGTGCATACATGGCTCTGGTGCACCTCTTTTTCCTAATTGGAGAAGGCGTTAATGACCAAAATGCCCCACTCTTGACCAAAAAACCATACCCCAAGCTCTTCTCCGCCGCTGGCCGGATTCTTCGCCTCTGGGACGACCTCGGAACTGCCAAGGAGGAGCAAGAGCGAGGAGACGTAGCATCGAGCATCGAGTTTGTGATGAGAGAGAAAGAGGTGGAAAGTGAAGAAGAGGGAAGAAGATACATATTGGGAGAGATATATGAGTTATGGAAGGATTTGAATGGGGAGTTGATGTCCAAGAATGGAATGCCATTAGCGATTATTAAAGTCGCACTTAACATGGCACGAGCTTCCCAAGTGGTGTACAAGCATGAAGAGGACACTTATTTCTCCAGCGTGGATAATTACGTGGAAGCCCTCTTCTTCACTCCTCTTCCTTCATCCATCTAA</P><DIV class="third-level">Optimization</DIV><P class="my-content">In order to assure that PcGES ligate to the multiple clone sites of the vector successfully, we avoid all the common restriction enzyme cutting sites （EcoR I, Xba I, Spe I, Pst I, BamH I, Bgl II, Hind III, Kpn I, Nco I, Nde I,Not I,Sal I, Xho I, Polymerase slippage site, Polymerase slippage site, Frameshift element, ribosome binding site）when the synthesis is being performed.</P><P class="my-content">To make the parts established by genes satisfy the need of constructing the vector pSBIC3, we avoid the restriction enzyme cutting sites, which are  <I>Eco</I>R Ⅰ, <I> Xba </I> I, <I> Spe </I> I, <I> Pst </I> I.</P><P class="my-content">To ligate genes directly to the standard vector, we added the restriction enzyme cutting sites of  <I> Xba </I> Ⅰ and  <I> Spe </I> Ⅰ to the upstream and downstream of target gene.</P><DIV class="third-level">The optimized sequence</DIV><P class="my-content">ATGTCTTGTGCTAGATCATACACTATCCCATTATCTTTTCCAAAGACATCTAATTCACCATTACAATTGAAAAATTTGTTACCATTTCCAGCTGCAAGATCAAGATTTTCTGTTAGAATGTCTACATCTTCATTACCAGTTGGTAACGAAGCATTGTTGAAGTACATCAGACAACCAGTTGTTTTGCCAACAGAAGAAGATGAATCAATTAAAAGAAGAGATTATTTGTTGGAAAAGACTGCAAGAAGATTAAGAACTTCTACAGGTTCAGTTGAAAAATTGAAATTGATCGATACTATCCAAAGATTAGGTATCGATTACTACTTGGAAGATGATATCAACGTTGTTTTGAGAAACGAATACTACAACGGTAGATCATCTGAAGAAGATTTGTTTACTACAGCTTTGAGATTCAGATTGTTGAGACATAACGGTTTCCAAATTTCTTCAGATGTTTTTATGAAGTTTAAAGATAAGAATGGTAAATTCAAGAAATGGATTGCTGAAGATACAATCGGTTTGTTATCTTTATACGAAGCATCTTACATGGGTGCACATGGTGAAAAGATTTTGGAAGAAGCTATGGAATTTTCAAGATCATTGTTGAAGAGATCATTGCCACAATTGCCACCAAAATTACATGGTCAAGTTGCTCAAGCATTAGAATTGCCAAGACATTTGAGAATGGCTAGATTGGAAGCAAGAAGATTTGTTCAACAATACGCTAAACAATCTGATTGTGATAGAGATTTGTTGAATTTGGCAACTTTGGATTACAATAAGGTTCAATTACAACATCAATTCGAATTGGCTGAAATCACTAGATGGTGGCAACAATTGGGTTTGGTTGAAAAATTGACATTTGCAAGAGATAGACCATTGGAATGTTTCTTGTGGACTGTTGGTTTGTTACCAGAACCAAGATACTCTACTTGTAGAATCGAAATGGCTAAGACAATTGCAATCTTGTTAGTTATTGATGATATCTTCGATACTTATGGTAAAATGGATGAATTGTTGTTGTTTACACAAGCTATCAGAAGATGGGATTTGGAAGCAATGGATATCTTGCCAGAATACATGAAAATTTGTTATATGGCTTTATACAATACTACAAACGAAATTTGTTACAAGGTTTTGAAATTGAACGGTTGGTCTGTTTTGTCATATTTGAAATCTACATGGATCGATATGATCGAAGGTTTTATGGTTGAAGCTAAATGGTTGAATGGTGGTGGTGCTCCAAATTTGGAAGAATACTTAGAAAACGGTGTTTCAACTGCTGGTGCATACATGGCTTTAGTTCATTTGTTTTTCTTGATCGGTGAAGGTGTTAACGATCAAAACGCACCATTATTGACTAAGAAACCATACCCAAAATTGTTTTCTGCTGCTGGTAGAATTTTAAGATTGTGGGATGATTTGGGTACTGCTAAAGAAGAACAAGAAAGAGGTGACGTTGCATCTTCAATCGAATTTGTTATGAGAGAAAAAGAAGTTGAATCAGAAGAAGAAGGTAGAAGATACATCTTGGGTGAAATATATGAATTGTGGAAAGATTTGAACGGTGAATTGATGTCTAAAAATGGTATGCCATTGGCTATTATTAAGGTTGCATTGAACATGGCTAGAGCATCACAAGTTGTTTACAAGCATGAAGAAGATACATACTTTTCTTCAGTTGATAACTACGTTGAAGCATTGTTTTTCACTCCATTGCCATCTTCTATTTGA</P><DIV class="third-level">Codon Adaptation Index (CAI)</DIV><P class="my-content">PcGES</P><P class="my-content">CAI: 0.93</P><P class="my-content">Fig. 2-1 The distribution of codon usage frequency along the length of the gene sequence. A CAI of 1.0 is considered to be perfect in the desired expression organism, and a CAI of &gt; 0.8 is regarded as good, in terms of high gene expression level.</P><DIV class="third-level">Amplification of  <I> PcGES </I></DIV><DIV class="third-level">carrier-pUC57 vector</DIV><P class="my-content"> The information of this vector is the same as  <I> PdGES </I> (above 1.2.1.1)</P><DIV class="third-level">verification of  <I> PcGES </I> product</DIV><P class="my-content">To obtain the target gene  <I> PcGES </I>, by using the specific primers to clone the target gene, we amplified the template is the vector pUC57 with PcGES through PCR.</P><P class="my-content">The sequence of forward primer is: AAAAACAACTAATTATTCGAAGATGTCTTGTGCTAGATCAT</P><P class="my-content">The sequence of reverse primer is: AGGCGAATTAATTCGCGGCGCTTTCAAATAGAAGATGGCAAT</P><P class="my-content">As shown is figure 1-3, we successfully cloned gene  <I> PcGES </I>, because the size of PcGES shown on the picture matches with the theoretical size of PcGES which is 1734 bp.</P><P class="my-content">Fig. 2-2 Electrophoresis of  <I> PcGES </I> PCR product.</P><P class="my-content">The left column is the marker, and the two strips on the right are PCR products of PcGES.</P><P class="my-content">The sizes of the two strips are about 2000 bp.</P><P class="my-content">The We tried to ligate  <I> PcGES</I> gene into pPIC9K and pET32a vector, but after three trails, we failed to ligate the vector. The possible explanation is the condition of ligation or the sequence of this gene is not suitable. But we successfully ligated <I> PcGES</I> gene with OYE and 2A in pPIC9K and pET32a vector, the specific information will be show the following text.(3.)</P><DIV class="first-level" id="a7">HbOYE</DIV><DIV class="second-level" id="a8">Sequence of HbOYE</DIV><P class="my-content">ATCTCTCTAATAATCATCTTCGTCAGCAGCAGCACAGCAGCAGCAGAAGAAAGTCATATATATATATATATCTTAGATTCATTTCAGTTCTAGATAAGAAGCAACAATGGCTGAAACTGGAACAGAAGGGACCGGGATCACCACCCTATTTTCTCCTTACAAAATGGGCAAGTTCAGCCTCTCCCACAGGGTGGTGCTGGCTCCCATGACTAGATGCAGAGCGTTGAACGGGATACCAAACGCAGCGCTGGTGGATTACTACACGCAGAGATCAACTCCCGGCGGATTTCTCATCACGGAAGGCACTCTGGTTTCCCCTACTGCCCCTGGGTTTCCTCATGTCCCTGGAATTTATACAGAAGAACAAGCGGAGGCATGGAAGAGGGTGGTGGATGCAGTTCATGCCAAAGGGAGCATCATATTCTGTCAATTATGGCATGTTGGCCGCGCATCTCATCAGGTTTATCAACCTAATGGAGCGCACCAATATCATCGACGGGCAAGGCCATCTCAAACAGATGGAGAATTCTCATGCCAGATGGATCATATGGGAAATACCCAACACCTAGGCCCTTGGAAACACCTGAAATACTAGAGGTAGTGAAGAATTATCGCCAGTCAGCCTTGAATGCCATTCGAGCAGGCTTTGATGGAATTGAGGTCCACGGGGCTCATGGTTACCTTATTGATCAATTCTTAAAAGACGGGATCAATGACCGAACAGATGAGTATGGTGGATCAATCAACAATCGATGCAGATTCCTAATGCAGGTGATTCAGGCAGTAGTTGCAGCTATTGGTGCTGATCGAGTTGGTTTCAGAATGTCACCGGCAATTGATCACCTAGATGCCATAGATTCTGATCCGCTCAACTTGGGTCTTGCTGTAATCGAGAGACTTAACAAACTTCAGTTGAACCTTGGATCAAAACTCACTTATCTCCATGTCACTCAGCCTCGCTACACAGCTTATGGCCAAACAGAATCAGGCAGACATGGTACTGAAGAAGAGGAAGCTAGATTAATGAGAACTTGGAGAAGGGCTTATAAGGGAACTTTCATCTGTAGCGGTGGGTTCACGAGGGAGCTAGGAATGGAAGCTATAGCTCAAGATGATGCAGATTTGGTATCTTATGGCCGACTTTTTATTTCAAACCCAGACTTAGTCTTGAGATTTAAGCTCAATGCGCCCTTGAATAAGTATGTCAGGAAAACATTCTACACCCAAGATCCTGTTGTTGGGTACACAGACTACCCATTTTTCAGAAAAGTAGACGGGAGCCAGGAGCCACGATCACGCCTTTGAATCTGTAGTCTTTGGAATAATTATACATGTATTTAAACATATGATATTTTGTTTGAACTTTATTATATCAGTGAACATCTTGATTAAAAGACAACATCCTCTAAGAAGAGGATTTTTTGAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA</P><DIV class="second-level" id="a9">Amplification of HbOYE</DIV><DIV class="third-level">carrier-pUC57 vector</DIV><P class="my-content">The information of this vector is the same as  <I> PdGES </I> (above 1.2.1.1)</P><DIV class="third-level">verification of HbOYE product</DIV><P class="my-content">To obtain the target gene HbOYE, by using the specific primers to clone the targeted gene，our team amplified the template is the vector pUC57 with HbOYE through PCR.</P><P class="my-content">The sequence of forward primer is: TTTTATTTCAAACCCAGAC</P><P class="my-content">The sequence of reverse primer is: GGTGGTGGTGCTCGAGTCTTTTAATTGATTCATCT</P><P class="my-content">As shown is figure 1-3, we successfully cloned gene HbOYE, because the size of HbOYE shown on the picture matches with the theoretical size of HbOYE which is 1455 bp.</P><P class="my-content">Fig. 3-1 Electrophoresis of  <I> HbOYE </I> PCR products.</P><P class="my-content">The left column is the marker, and the two strips on the right are PCR products of HbOYE.</P><P class="my-content">The sizes of the two strips are about 1200 bp.</P><DIV class="second-level" id="a10">Ligation of HbOYE fragment to pET32a vector</DIV><DIV class="third-level">pET32a vector</DIV><P class="my-content">pET32a is a carrier of <I> E.Coli </I> of 5900bp. The promoter of pET32a is T7 and the resistance of pET32a is the Ampicillin. The information of this vector is the same as  <I> PdGES </I> (above 1.3.2)</P><P class="my-content">3.3.2 Ligation of HbOYE fragment to pET32a vector</P><P class="my-content">we use the restriction endonuclease <I>Bam</I>H I and <I>Xho</I> I to cut the  <I> E. Coli </I> expression vector pET32a and make it ligated with HbOYE (1.2 kb), and the result has shown to be positive  through PCR.</P><P class="my-content">Then we ligate the product with T4 vector for sequencing. Eventually, the product has been transformed into the competent cell  <I> E. Coli </I> complement cell  <I> DH5α</I>, and the result of sequencing is 100% matched.</P><P class="my-content">Fig. 3-2 Electrophoresis of  <I> HbOYE </I> -contained pET32a vector.</P><P class="my-content">The left column is the marker and the strips on the right are all the samples of combination of HbOYE and pFT32a. According to the graph, the size is between 1000 bp and 750 bp, so the result is positive.</P><DIV class="second-level" id="a11">Ligation of <I>HbOYE</I> fragment to pPIC9K vector</DIV><DIV class="third-level">pPIC9K vector</DIV><P class="my-content">pPIC9K is a carrier for the expression of protein in yeast of 9276bp. The promoter of pPIC9K is AOX1 and the resistance of pPIC9K is the Ampicillin. The information of this vector is the same as  <I> PdGES </I> (above 1.3.1)</P><P class="my-content">3.4.2 Ligation of  <I> HbOYE </I> fragment to pPIC9K vector</P><P class="my-content">We utilized restriction endonuclease <I> Bam </I>H I and  <I>Not</I>I to cut the yeast expression vector PIC9K and make it ligated with HbOYE (about 1.2 kb), and the result has been shown positive through PCR.</P><P class="my-content">Fig. 3-3 Electrophoresis of  <I> HbOYE </I> -contained pPIC9K vector.</P><P class="my-content">The left column is the marker and the strips on the right are all the samples of combination of  <I> HbOYE </I> and pPIC9K. According to the graph, the size is between 1000 bp and 2000 bp, so the result is positive.</P><DIV class="second-level" id="a12">Ligation of HbOYE with 2A and GES to pPIC9K vector and pET32a vector</DIV><P class="my-content">To improve the efficiency of production while keeping the two proteins independent at the same time, we decide to combine  <I> OYE </I> and  <I> GES </I> together. It turns out that we choose 2A as a ‘bridge’ to link the two genes because the protein translated by 2A can be cut by certain enzymes automatically. 2A, known as CHYSEL polypeptides, contains the peptide bond to grow the peptide chain which helps to link two genes. The presence of the conserved CHYSEL residues in the peptides contributes to form a torsion which causes the peptide chain to be released later, and then the two genes can express in a cell separately. Furthermore, we employ two kinds of 2A, which are F2A and P2A. By doing so, we can compare the efficiencies of them and choose one which is more efficient.</P><P class="my-content">F2A sequence:</P><P class="my-content">CAGCTGTTGAATTTTGACCTTCTTAAGCTTGCGGGAGACGTCGAGTCCAACCCTGGGCCC</P><P class="my-content">P2A sequence:</P><P class="my-content">GGAAGCGGAGCGACGAATTTTAGTCTACTGAAACAAGCGGGAGACGTGGAGGAAAACCCTGGACCT</P><DIV class="third-level">HbOYE+2A</DIV><P class="my-content"> By using Net-PCR, we combined the HbOYE with 2A. 2A is a piece of peptide which can combine two genes together and be cut at 3’ prime end when the DNA is translated into mRNA. After being cut, the two genes can still perform their functions correctly in the latter steps. The picture of electrophoresis following shows the positive result of the combination. This product will be used in the final step of our experiment. Because of the function domain of  <I> GES </I> is on the 3’ end，the 2A which can express 20 amino acids which may influence the function of GES protein, so we ligated OYE with 2A first, the ligate GES after 2A.</P><P class="my-content">Fig. 3-4 Eelectrophoresis of  <I> HbOYE +2A </I> -contained pPIC9K vector. The left column is the marker and the strips on the right are all the samples of combination of HbOYE and 2A. According to the graph, the size of <I> HbOYE +2A </I>  is between 1000 bp and 2000 bp, so the result is positive.</P><P class="my-content">Fig. 3-5 Eelectrophoresis of HbOYE+2A-contained pET32a  vector. The M stands for  the marker and the 1,2,3 stand for samples of combination of HbOYE and 2A. lane 1 and 2 is the HbOYE + F2A,3 stands for HbOYE + P2A. According to the graph, we use a part of sequence to measure, so the result is positive.</P><DIV class="third-level">HbOYE+2A+PcGES</DIV><P class="my-content">By using Net-PCR, we combined the previous product (HbOYE with 2A) and PcGES. The picture of electrophoresis following shows the positive result of the combination. This product is our final product of the whole experiments.</P><P class="my-content">Fig. 3-6 Electrophoresis of <I>HbOYE+2A+PcGES</I>-contained  pET32a vector.</P><P class="my-content">The M stands for  the marker and the 1,2,3 stand for samples of combination of <I>HbOYE +2A</I> and <I>PcGES</I>. 1 and 2 is the  <I>HbOYE+2A+PcGES</I>, 3 and 4 stands for <I>HbOYE + P2A+ PcGES</I>. According to the graph, we use a part of sequence to measure, so the result is positive.</P><P class="my-content">Fig. 3-7 Electrophoresis of HbOYE+F2A+PcGES-contained  pPIC9K vector.</P><P class="my-content">The M stands for  the marker and the 1,2,3 stand for samples of combination of HbOYE +2A and PcGES. 1 and 2 is the HbOYE + F2A+PcGES, According to the graph, we use a part of sequence to measure, so the result is positive.</P><P class="my-content">After ultrasonication, we used the SDS-PAGE electrophoresis to detect our target protein (HbOYE+F2A+GES2 and HbOYE+P2A+GES2), and the result showed that we succesfully produced our the recombinant protein (Fig ).</P><P class="my-content">To make sure the expression of SmCPSI goes successfully with our instruments, we performed sonication on E.Coli cells and smashed them. Then we collect the supernatant fluid and protein to conduct the SDS protein electrophoresis. The size of HbOYE is about 1.2 Kb, and the encoding protein is about 44 KD; The size of PcGES is about 1.8 Kb, and the encoding protein is about 67 KD. There are about 22 KD fusion tags  on the carrier skeleton of pET32a on the end N of HbOYE. F2A and P2A can cut the fusion protein into two parts and the each new protein will be about 67 KD which cannot be separated by electropherosis. The result of SDS-PAGE shows that the protein can be expressd fluently through our instruments with the induction of IPTG. The positive result meets our expectation.</P><P class="my-content">Fig 15 Th e SDS-PAGE electrophoresis of recombinant protein</P><P class="my-content">Lane M, protein marker; Lane 1, total protein with IPTG induction of HbOYE+F2A+PcGES-contained pET32a vector ; Lane 2, total protein with IPTG induction of HbOYE+P2A+PcGES-contained pET32a vector; Lane CK, protein produced by empty pET32a vector. Black arrow shows the targeted recombinant proteins. The condition for protein induction is under 30℃ with the IPTG concentration of 0.4 mM for 3 hours.</P><P class="my-content">Next we committed Western Blot method to detect the exitence of HIS tag by using HIS antibody. The results showed that, under both situations, bacteria successfully expressed our recombinant protein. It also showed that, the concentration of soluble HbOYE+F2A+GES2 and HbOYE+P2A+GES2，duced by the IPTG-induced bacteria is great, while there is no protein produced by empty pET32a vector (Fig 16).</P><P class="my-content">Fig 16 Western blot result by using HIS antibody. Lane M, protein marker; Lane 1, total protein with IPTG induction of HbOYE+F2A+PcGES-contained pET32a vector; Lane 2, total protein with IPTG induction of HbOYE+P2A+PcGES-contained pET32a vector; Lane CK, protein produced by empty pET32a vector. The condition for protein induction is under 30℃ with the IPTG concentration of 0.4 mM for 3 hours.；The first antibody used in Western blotting is kangwei biocompany-CW0304S and the second antibody is kangwei biocompany-CW0102S</P><DIV class="first-level" id="a13">ScOYE</DIV><DIV class="second-level" id="a14">nucleotide sequence of ScOYE</DIV><P class="my-content">ATGCCATTTGTTAAGGACTTTAAGCCACAAGCTTTGGGTGACACCAACTTATTCAAACCAATCAAAATTGGTAACAATGAACTTCTACACCGTGCTGTCATTCCTCCATTGACTAGAATGAGAGCCCAACATCCAGGTAATATTCCAAACAGAGACTGGGCCGTTGAATACTACGCTCAACGTGCTCAAAGACCAGGAACCTTGATTATCACTGAAGGTACCTTTCCCTCTCCACAATCTGGGGGTTACGACAATGCTCCAGGTATCTGGTCCGAAGAACAAATTAAAGAATGGACCAAGATTTTCAAGGCTATTCATGAGAATAAATCGTTCGCATGGGTCCAATTATGGGTTCTAGGTTGGGCTGCTTTCCCAGACACCCTTGCTAGGGATGGTTTGCGTTACGACTCCGCTTCTGACAACGTGTATATGAATGCAGAACAAGAAGAAAAGGCTAAGAAGGCTAACAACCCACAACACAGTATAACAAAGGATGAAATTAAGCAATACGTCAAAGAATACGTCCAAGCTGCCAAAAACTCCATTGCTGCTGGTGCCGATGGTGTTGAAATCCACAGCGCTAACGGTTACTTGTTGAACCAGTTCTTGGACCCACACTCCAATAACAGAACCGATGAGTATGGTGGATCCATCGAAAACAGAGCCCGTTTCACCTTGGAAGTGGTTGATGCAGTTGTCGATGCTATTGGCCCTGAAAAAGTCGGTTTGAGATTGTCTCCATATGGTGTCTTCAACAGTATGTCTGGTGGTGCTGAAACCGGTATTGTTGCTCAATATGCTTATGTCTTAGGTGAACTAGAAAGAAGAGCTAAAGCTGGCAAGCGTTTGGCTTTCGTCCATCTAGTTGAACCTCGTGTCACCAACCCATTTTTAACTGAAGGTGAAGGTGAATACAATGGAGGTAGCAACAAATTTGCTTATTCTATCTGGAAGGGCCCAATTATTAGAGCTGGTAACTTTGCTCTGCACCCAGAAGTTGTCAGAGAAGAGGTGAAGGATCCTAGAACATTGATCGGTTACGGTAGATTTTTTATCTCTAATCCAGATTTGGTTGATCGTTTGGAAAAAGGGTTACCATTAAACAAATATGACAGAGACACTTTCTACAAAATGTCAGCTGAGGGATACATTGACTACCCTACGTACGAAGAAGCTCTAAAACTCGGTTGGGACAAAAATTAA</P><DIV class="second-level" id="a15">Amplification of ScOYE</DIV><DIV class="third-level">carrier-pUC57</DIV><P class="my-content">The information of this vector is the same as  <I> PdGES </I> (above 1.2.1.1)</P><DIV class="third-level">verification of ScOYE product</DIV><P class="my-content">To obtain the target gene  <I> ScOYE </I>, by using the specific primers to clone the targeted gene,we amplified the template is the vector pUC57 with HbOYE through PCR.</P><P class="my-content">The sequence of forward primer is:AAAAACAACTAATTATTCGAAGATGCCATTTGTTAAGGACT</P><P class="my-content">The sequence of reverse primer is: AGGCGAATTAATTCGCGGCGCTTTTTTGTCCCAACCG</P><P class="my-content">As shown is figure 1-3, we successfully cloned gene  <I> ScOYE </I>, because the size of  <I> ScOYE </I> shown on the picture matches with the theoretical size of  <I> ScOYE </I> which is 1203 bp.</P><P class="my-content">Fig. 4-1 The picture of electrophoresis of ScOYE.</P><P class="my-content">The left column is the marker, and the two strips on the right are PCR products of ScOYE.</P><P class="my-content">The sizes of the two strips are more than 1000 bp.</P><DIV class="second-level" id="a16">Ligation of ScOYE in  <I> E.Coli </I></DIV><DIV class="third-level">Ligation of  <I> ScOYE </I> fragment to pET32a vector</DIV><P class="my-content">We uses the restriction endonuclease  <I>Bam</I>H I and  <I> Xho </I> I to cut the  <I> E. Coli </I> expression vector pET32a and make it ligated with ScOYE, and the result has been shown positive through PCR.</P><P class="my-content">Then we ligate the product with T4 vector for sequencing. Eventually, the product has been transformed into the E. Coli competent cell DH5α, and the result of sequencing is 100% matched.</P><P class="my-content">Fig. 4-2 Electrophoresis of  <I> ScOYE </I> -contained pET32a vector. The left column is the marker and the strips on the right are all the samples of combination of  <I> ScOYE </I> and pET32a. According to the graph, the size is between 1000 bp and 2000 bp, so the result is positive.</P><DIV class="second-level" id="a17">Ligation of <I>ScOYE</I> in Yeast</DIV><P class="my-content">We plan to striction endonuclease <I>BamH I and NotI to cut the yeast expression vector PIC9K and make it ligated with ScOYE. Then combined the previous product (ScOYE with 2A) and PcGES. but the time is limitated.</I></P><DIV class="first-level" id="a18">Model</DIV><DIV class="second-level" id="a19">Results of the Experiment to Test Citronellol’s Effect in Repelling Mosquitoes</DIV><P class="my-content">To test the repellent effectiveness of citronellol, we chose the common and safe <I>Aedes（Stegomyia）albopictus</I> as subject. We bought bottled citronellol from 3 companies (each sample was tested independently). We used citronellol with concentration ranging from 95%-99% for test outdoors, but the odor was too strong for people to accept. The indoor mosquito repelling experiment used the following 4 solutions:</P><P class="my-content">Citronellol 60%; glycerin 30%; ddH2O 10%</P><P class="my-content">Citronellol 40%; glycerin 30%; ddH2O 30%</P><P class="my-content">Citronellol 20%; glycerin 30%; ddH2O 50%</P><P class="my-content">Citronellol 10%; glycerin 30%; ddH2O 60%</P><DIV class="third-level">1.Applying on Hands:</DIV><P class="my-content">Apply the Solution evenly to a circular area with a diameter of 4cm on the back of left hand and record the number of bites on the area. Take the back of the right hand as control. Stop the experiment when the effect of repellent decreases. Results showed direct proportional relationships between the concentration of citronellol and the time of repelling. The highest concentration of citronellol was 60%, the 100% protection time was 2.5h. The lowest concentration of citronellol was 10%, the 100% protection time was 2h. The results demonstrated citronellol as a highly effective mosquito repellent. The protection effectiveness does not have a significant difference when the concentration of citronellol is 20% or 10%. If the concentration of citronellol exceeds 20%, the effective protection time increases as the concentration of citronellol increases. The odor is more acceptable when the concentration of citronellol is lower.</P><P class="my-content">Figure 1: Control experiment for citronellol application on hands</P><P class="my-content">Figure2: Hands without citronellol solution applied</P><P class="my-content"><B>The Solution When the Concentration of Citronellol is 10%</B></P><TABLE><TBODY><TR><TD>The Observation Time (h)</TD><TD>Control(number of bites)</TD><TD>Treatment (number of bites)</TD><TD>Protection Rate%</TD></TR><TR><TD rowspan="3">0</TD><TD>12</TD><TD>0</TD><TD>100</TD></TR><TR><TD>6</TD><TD>0</TD><TD>100</TD></TR><TR><TD>15</TD><TD>0</TD><TD>100</TD></TR><TR><TD rowspan="3">1</TD><TD>6</TD><TD>0</TD><TD>100</TD></TR><TR><TD>6</TD><TD>0</TD><TD>100</TD></TR><TR><TD>13</TD><TD>0</TD><TD>100</TD></TR><TR><TD rowspan="2">2</TD><TD>12</TD><TD>0</TD><TD>100</TD></TR><TR><TD>20</TD><TD>1</TD><TD>95%</TD></TR></TBODY></TABLE><P class="my-content">Figure 3: the result with the solution when the concentration of citronellol is 10%</P><P class="my-content">Control: number of bites when solution composed of glycerin and dd H2 O is applied to the right hand</P><P class="my-content">Treatment: number of bites when solution composed of Citronellol 10%, glycerin 30% and ddH2O 60% is applied to the left hand. The number of bites is an average of 3 kinds of purchased citronellol. The protection equals: (bites on control – bites on treatment)/ (bites on control) *100%.</P><P class="my-content"><B>The Solution When the Concentration of Citronellol is 20%</B></P><TABLE><TBODY><TR><TD>The Observation Time (h)</TD><TD>Control(number of bites)</TD><TD>Treatment(number of bites)</TD><TD>Protection Rate%</TD></TR><TR><TD rowspan="3">0</TD><TD>6</TD><TD>0</TD><TD>100</TD></TR><TR><TD>9</TD><TD>0</TD><TD>100</TD></TR><TR><TD>16</TD><TD>0</TD><TD>100</TD></TR><TR><TD rowspan="3">1</TD><TD>9</TD><TD>0</TD><TD>100</TD></TR><TR><TD>0</TD><TD>0</TD><TD>100</TD></TR><TR><TD>5</TD><TD>0</TD><TD>100</TD></TR><TR><TD rowspan="3">2</TD><TD>13</TD><TD>1</TD><TD>92</TD></TR><TR><TD>15</TD><TD>2</TD><TD>87</TD></TR><TR><TD>8</TD><TD>0</TD><TD>100</TD></TR></TBODY></TABLE><P class="my-content">Figure 4: the result with the solution when the concentration of citronellol is 20%</P><P class="my-content">Control: number of bites when solution composed of glycerin and ddH<SUB>2</SUB> O is applied to the right hand</P><P class="my-content">Treatment: number of bites when solution composed of Citronellol 20%, glycerin 30% and ddH<SUB>2</SUB>O 50% is applied to the left hand. The number of bites is an average of 3 kinds of purchased citronellol. The protection equals: (bites on control – bites on treatment)/ (bites on control) *100%.</P><P class="my-content"><B>The Solution When the Concentration of Citronellol is 40%</B></P><TABLE><TBODY><TR><TD>The Observation Time（h）</TD><TD>Control(number of bites)</TD><TD>Treatment (number of bites)</TD><TD>Protection Rate%</TD></TR><TR><TD rowspan="3">0</TD><TD>9</TD><TD>0</TD><TD>100</TD></TR><TR><TD>9</TD><TD>0</TD><TD>100</TD></TR><TR><TD>8</TD><TD>0</TD><TD>100</TD></TR><TR><TD rowspan="3">1</TD><TD>1</TD><TD>0</TD><TD>100</TD></TR><TR><TD>16</TD><TD>0</TD><TD>100</TD></TR><TR><TD>10</TD><TD>0</TD><TD>100</TD></TR><TR><TD rowspan="3">2</TD><TD>15</TD><TD>0</TD><TD>100</TD></TR><TR><TD>9</TD><TD>0</TD><TD>100</TD></TR><TR><TD>7</TD><TD>0</TD><TD>100</TD></TR><TR><TD rowspan="3">2.5</TD><TD>13</TD><TD>0</TD><TD>100</TD></TR><TR><TD>8</TD><TD>2</TD><TD>75</TD></TR><TR><TD>11</TD><TD>2</TD><TD>81</TD></TR></TBODY></TABLE><P class="my-content">Figure 5: the result with the solution when the concentration of citronellol is 40%</P><P class="my-content">Control: number of bites when solution composed of glycerin and dd H<SUB>2</SUB> O is applied to the right hand</P><P class="my-content">Treatment: number of bites when solution composed of Citronellol 40%, glycerin 30% and ddH<SUB>2</SUB>O 30% is applied to the left hand. The number of bites is an average of 3 kinds of purchased citronellol. The protection equals: (bites on control – bites on treatment)/ (bites on control) *100%.</P><P class="my-content"><B>The Solution When the Concentration of Citronellol is 40%</B></P><TABLE><TBODY><TR><TD>The Observation Time(h)</TD><TD>Control(number of bites)</TD><TD>Treatment (number of bites)</TD><TD>Protection Rate%</TD></TR><TR><TD rowspan="3">0</TD><TD>6</TD><TD>0</TD><TD>100</TD></TR><TR><TD>10</TD><TD>0</TD><TD>100</TD></TR><TR><TD>5</TD><TD>0</TD><TD>100</TD></TR><TR><TD rowspan="3">1</TD><TD>7</TD><TD>0</TD><TD>100</TD></TR><TR><TD>9</TD><TD>0</TD><TD>100</TD></TR><TR><TD>11</TD><TD>0</TD><TD>100</TD></TR><TR><TD rowspan="3">2</TD><TD>8</TD><TD>0</TD><TD>100</TD></TR><TR><TD>8</TD><TD>0</TD><TD>100</TD></TR><TR><TD>10</TD><TD>0</TD><TD>100</TD></TR><TR><TD rowspan="3">2.5</TD><TD>7</TD><TD>0</TD><TD>100</TD></TR><TR><TD>9</TD><TD>0</TD><TD>100</TD></TR><TR><TD>16</TD><TD>0</TD><TD>100</TD></TR><TR><TD rowspan="3">3</TD><TD>8</TD><TD>1</TD><TD>88</TD></TR><TR><TD>13</TD><TD>2</TD><TD>85</TD></TR><TR><TD>15</TD><TD>2</TD><TD>87</TD></TR></TBODY></TABLE><P class="my-content">Figure 6: the result with the solution when the concentration of citronellol is 60%</P><P class="my-content">Control: number of bites when solution composed of glycerin and ddH<SUB>2</SUB> O is applied to the right hand</P><P class="my-content">Treatment: number of bites when solution composed of Citronellol 60%, glycerin 30% and ddH<SUB>2</SUB> O 10% is applied to the left hand. The number of bites is an average of 3 kinds of purchased citronellol. The protection equals: (bites on control – bites on treatment)/ (bites on control) *100%.</P><DIV class="third-level">Fuming Experiments:</DIV><P class="my-content">Prepare 2 cubic mosquito cages. Separate each mosquito cage with a piece of gauze. Mosquitoes were put in the left part of the cage and citronellol fume was put in the right part. Put both hands in the left part of the cube without citronellol fume. Then record the number of times mosquitoes stayed on or collided with the gauze in 5min. Repeat each set of experiment 3 times. Record the average number of bites and calculate the protection rate: (number of mosquitoes without citronellol fume – the number with citronellol fume)/ (number without citronellol fume) * 100%. The results showed different concentration of citronellol did not have apparent distinctions; whether there was citronellol fume or not influence on the number of mosquitoes staying on the gauze. It was natural for mosquitoes to stay on the gauze as there had been no disturbance. We learnt from Dr. Xu that citronellol only works as a repellent when in contact with skin. This is consistent with the experimental results. Citronellol blocks the scent that attracts mosquitoes. Therefore, if citronellol is not applied on skin, it can be reasonably hypothesized that mosquitoes will still be attracted.</P><P class="my-content">Figure 7: Left side of cage in citronellol fuming experiment</P><P class="my-content"><B>The Solution When the Concentration of Citronellol is 10%</B></P><TABLE><TBODY><TR><TD>The Observation Time (h)</TD><TD>Control(number of bites)</TD><TD>Number of Mosquitoes Resting on Gauze</TD><TD>Treatment (number of bites)</TD><TD>Protection Rate%</TD></TR><TR><TD rowspan="3">0</TD><TD>7</TD><TD>15</TD><TD>5</TD><TD>29</TD></TR><TR><TD>17</TD><TD>10</TD><TD>10</TD><TD>41</TD></TR><TR><TD>15</TD><TD>8</TD><TD>7</TD><TD>53</TD></TR></TBODY></TABLE><P class="my-content">Figure 8: the result with the solution when the concentration of citronellol is 10%</P><P class="my-content">Control: number of bites in designated area with sample without citronellol fume;</P><P class="my-content">Treatment: number of bites in designated area with the solution of composition of: citronellol 10%; glycerin 30%; dd H<SUB>2</SUB>O 60%.</P><P class="my-content">Number of bites is the average calculated from 3 kinds of citronellol purchased. The protection rate (%) is (control number of bites – Treatment number of bites)/ Control number of bites*100%.</P><P class="my-content"><B>The Solution When the Concentration of Citronellol is 20%</B></P><TABLE><TBODY><TR><TD>The Observation Time (h)</TD><TD> Control(number of bites)</TD><TD> Number of Mosquitoes Resting on Gauze</TD><TD>Treatment (number of bites)</TD><TD>Protection Rate%</TD></TR><TR><TD rowspan="3">0</TD><TD>8</TD><TD>10</TD><TD>6</TD><TD>25</TD></TR><TR><TD>9</TD><TD>12</TD><TD>8</TD><TD>11</TD></TR><TR><TD>14</TD><TD>10</TD><TD>7</TD><TD>50</TD></TR></TBODY></TABLE><P class="my-content">Figure 9: the result with the solution when the concentration of citronellol is 20%</P><P class="my-content">Control: Control: number of bites in designated area with sample without citronellol fume;</P><P class="my-content">Treatment: number of bites in designated area with the solution of composition of: citronellol 20%; glycerin 30%; dd H2O 50%.</P><P class="my-content">Number of bites is the average calculated from 3 kinds of citronellol purchased. The protection rate (%) is (control number of bites – Treatment number of bites)/ Control number of bites*100%.</P><P class="my-content"><B>The Solution When the Concentration of Citronellol is 40%</B></P><TABLE><TBODY><TR><TD>The Observation Time (h)</TD><TD>Control(number of bites)</TD><TD>Number of Mosquitoes Resting on Gauze</TD><TD>Treatment (number of bites)</TD><TD>Protection Rate%</TD></TR><TR><TD rowspan="3">0</TD><TD>5</TD><TD>5</TD><TD>6</TD><TD>-20</TD></TR><TR><TD>8</TD><TD>8</TD><TD>10</TD><TD>-25</TD></TR><TR><TD>10</TD><TD>6</TD><TD>17</TD><TD>-70</TD></TR></TBODY></TABLE><P class="my-content">Figure 10: the result with the solution when the concentration of citronellol is 40%</P><P class="my-content">Control: Control: number of bites in designated area with sample without citronellol fume;</P><P class="my-content">Treatment: number of bites in designated area with the solution of composition of: citronellol 40%; glycerin 30%; ddH<SUB>2</SUB>O 30%.</P><P class="my-content">Number of bites is the average calculated from 3 kinds of citronellol purchased. The protection rate (%) is (control number of bites – Treatment number of bites)/ Control number of bites*100%.</P><P class="my-content"><B>The Solution When the Concentration of Citronellol is 60%</B></P><TABLE><TBODY><TR><TD>The Observation Time (h)</TD><TD>Control(number of bites)</TD><TD>Number of Mosquitoes Resting on Gauze</TD><TD>Treatment (number of bites)</TD><TD>Protection Rate%</TD></TR><TR><TD rowspan="3">0</TD><TD>8</TD><TD>7</TD><TD>7</TD><TD>13</TD></TR><TR><TD>8</TD><TD>5</TD><TD>9</TD><TD>-13</TD></TR><TR><TD>13</TD><TD>15</TD><TD>6</TD><TD>54</TD></TR></TBODY></TABLE><P class="my-content">Figure 11: the result with the solution when the concentration of citronellol is 60%</P><P class="my-content">Control: Control: number of bites in designated area with sample without citronellol fume;</P><P class="my-content">Treatment: number of bites in designated area with the solution of composition of: citronellol 60%; glycerin 30%; ddH<SUB>2</SUB>O 10%.</P><P class="my-content">Number of bites is the average calculated from 3 kinds of citronellol purchased. The protection rate (%) is (control number of bites – Treatment number of bites)/ Control number of bites*100%.</P></DIV></DIV><FOOTER><DIV class="sponsor"><DIV class="sponsor-txt">Sponsors</DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>