"
Page
Discussion
View source
History
teams
Log in
 
Team:Cornell/project/wetlab/fungal toolkit/biosafety
From 2013.igem.org
Cornell University Genetically Engineered Machines
Previous
Tour Page
Next
Tour Page
Start
Tour!
Home
Project
Overview
Background
Wet Lab
Dry Lab
Human Practices
Future Applications
Safety
Outreach
Overview
Research Presentations
Panel Discussions
Public Awareness
Notebook
Overview
Protocols
Plasmids
Team
Overview
Profile
Bios
Sponsors
Attributions
Gallery
2013: Organofoam
Team Website
2012: SAFE BET
2011: Biofactory
2010: OMG OMVs
2009: Cadmium Sensor
Wet Lab
Overview
Chassis
Fungal Toolkit
Regulatory Elements
Selectable Markers
Characterization
Homologous Constructs
Biosafety
Protoplasting
Fungal Transformation
Carotenoids
Limonene
Antifungals
BioBricks
Animation
Future Work
Biosafety
Cre-Lox System
A major biosafety concern when genetically engineering organisms is horizontal gene transfer.  This transfer of genetic material between organisms is most prevalent in prokaryotic organisms where plasmid DNA can be easily expelled and absorbed.  When working with eukaryotes like fungi, every engineered gene is placed directly into chromosomal DNA, which is neither expelled nor absorbed by other organisms easily due to its size; however, cases of horizontal gene transfer between fungi have been reported [1], so preventing the transfer of potentially harmful genes is still very important to our project.
We identified the resistances to geneticin and hygromycin as the two most important genes in our toolkit to control.  To prevent horizontal transfer of these genes, we chose to use a Cre-Lox recombination system.  In this system, each gene is flanked by short sequences called lox sites, which are recognized and cleaved by the activated Cre recombinase protein [2].  We designed a biobrick containing two lox sites with unique non-biobrick assembly standard cut sites between them, allowing us to easily insert all our resistance genes between the lox sites.
We plan to implement the Cre recombinase protein behind the pelA inducible promoter from Aspergillus nidulans. This promoter is both repressed by glucose and induced by polygalacturonic acid, allowing for full control over the expression of the recombinase.  When the pelA promoter is activated, the recombinase will be expressed and the antibiotic resistances will be spliced out of the genome.
We anticipate that this system is very well fit for fungal genetic engineering and horizontal gene transfer control.  If this system were utilized in a prokaryote with plasmid DNA, then the removal of the antibiotic resistance (and thus the selective pressure) could also remove any useful genetic tool implemented in parallel.  The rationale is that expressing extra genes on plasmid DNA decreases the overall fitness of the organism, so if no selective pressure is used, then the organisms can expel the plasmid [3]. We hypothesize, however, that the lack of selective pressure will not be as big of an issue in fungi, as organism evolution is much slower and integration into the full genome will increase the stability of inserted genes.  Therefore, after the initial selection of successfully transformed organisms, the Cre-Lox system could then be activated, splicing out the antibiotic resistances and leaving only benign genes behind to be expressed, preventing environmental exposure to harmful genes and lowering production costs due to antibiotic use.  Success of this system has already been demonstrated in fungal eukaryotes [2], and it should be effective in basidiomycetes as well.
Kill Switch
Background
As detailed further in the future applications section, Ecovative hopes to one day develop their product into a living, self-healing material.  Their current products are killed and desiccated before distribution, but the implementation of a living, genetically modified product raises many more biosafety concerns.  To address some of these concerns, we’ve developed a kill switch system compatible with Ecovative’s living material ideas and any other fungal genetic engineering applications.  When implemented, the user would be able to both suppress and activate the kill switch, allowing a high level of control over life and death of the fungal cells.
Construction and Implementation
The system involves two stages that work together to degrade the complex fungal cell boundary.  The first stage involves the expression of a lambda bacteriophage holin designed by UC Berkeley in 2008 (part BBa_K112306).  Bacteriophage holin proteins are well documented in their ability to greatly increase cell membrane permeability, leading to lysis [4].  In the case of fungi, the increase in membrane permeability itself would not lead to lysis because the cell wall would remain intact; however, the increased membrane permeability would allow the second stage—a chitinase protein—access to the cell wall.  The organic polymer chitin composes a significant fraction of filamentous fungal cell walls and has been identified as a promising target for antifungal agents because of its structural importance [5].  The rationale behind this kill switch system is that once the holin proteins open pores, the chitinases can access the cell wall and degrade it, rendering it structurally useless. The chitinase utilized in our system is the Cht1_2 gene from the diatom Thalassiosira pseudonana, and it is hypothesized to act in either cleavage of chitin fibers or in pathogenic defense [6].  To successfully implement and control this system, we plan to place both the Cht1_2 gene and the lambda holin behind PpelA, an inducible promoter from the fungi Aspergillus nidulans. This promoter is both repressed by glucose and induced by polygalacturonic acid, allowing for full control over both expression and repression of the holin and chitinase genes [7].
Notes: Cht1_2 was synthesized by IDT and the pelA promoter was generously provided by the Turgeon lab at Cornell University.
This figure depicts our fungal kill switch system.  The first step is the holin gene that blasts holes in the phospholipid bilayer membrane, allowing the chitinase gene to pass through this membrane and degrade the chitin in the external cell wall.
References
1. Fukatsu, T. (2010). A fungal past to insect color. Science,328(5978), 574-575. doi: 10.1126/science.1190417
2. Sauer, B. (1987). Functional expression of the cre-lox site-specific recombination system in the yeast saccharomyces cerevisiae. Molecular and Cellular Biology, 7(6), 2087-2096. doi: 10.1128/MCB.7.6.2087
3. Corchero, J. L., & Villaverde, A. (1998). Plasmid maintenance in Escherichia coli recombinant cultures is dramatically, steadily, and specifically influenced by features of the encoded proteins. Biotechnology and Bioengineering, 58(6), 625-632. doi: 10.1002/(SICI)1097-0290(19980620)58:6<625::AID-BIT8>3.0.CO;2-K
4. Wang, I. N., Smith, D. L., & Young, R. (2000). Holins: The protein clocks of bacteriophage infections. Annual Review of Microbiology, 54, 799-825. doi: 10.1146/annurev.micro.54.1.799
5. Bowman, S. M. & Free, S. J. (2006), The structure and synthesis of the fungal cell wall. Bioessays, 28, 799–808. doi: 10.1002/bies.20441
6. Armbrust, E. V. et al. (2004). The genome of the diatom thalassiosira pseudonana: Ecology, evolution, and metabolism. Science, 306(5693), 79-86. doi: 10.1126/science.1101156
7. Yang, G., Turgeon, B. G., & Yoder, O. C. (1994). Toxin-deficient mutants from a toxin-sensitive transformant of cochliobolus heterostrophus . Genetics, 137(3), 751-757. Retrieved from http://www.genetics.org/content/137/3/751.long
Retrieved from " http://2013.igem.org/Team:Cornell/project/wetlab/fungal_toolkit/biosafety "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
