"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Cornell/project/wetlab/fungal toolkit/regulatory elements
From 2013.igem.org
Cornell University Genetically Engineered Machines
Previous
Tour Page
Next
Tour Page
Start
Tour!
Home
Project
Overview
Background
Wet Lab
Dry Lab
Human Practices
Future Applications
Safety
Outreach
Overview
Research Presentations
Panel Discussions
Public Awareness
Notebook
Overview
Protocols
Plasmids
Team
Overview
Profile
Bios
Sponsors
Attributions
Gallery
2013: Organofoam
Team Website
2012: SAFE BET
2011: Biofactory
2010: OMG OMVs
2009: Cadmium Sensor
Wet Lab
Overview
Chassis
Fungal Toolkit
Regulatory Elements
Selectable Markers
Characterization
Homologous Constructs
Biosafety
Protoplasting
Fungal Transformation
Carotenoids
Limonene
Antifungals
BioBricks
Animation
Future Work
Regulatory Elements
In designing our genetic circuits, we identified several desired properties:
Industrialized production of biomolecules for the modification of fungal characteristics
Continuous expression in fungi
Modularity
Thus, regulatory elements were determined to be an essential component for the controlled expression of genes we transformed into fungi. As part of our fungal toolkit, we developed a standardized registry of promoters that could be interchangeably used in our genetic circuit.
The T7 promoter was identified as an ideal promoter for property 1 due to its high affinity for T7 RNA polymerase (RNAP) and the resulting high levels of expression of downstream genes. However, the T7 RNAP consequently consumes resources heavily (i.e rNTPs), which could stop growth in E. coli [1]. Therefore, we designed our genetic circuit to regulate the expression of T7 RNAP in our fungal chassis. Constitutive fungal promoters were selected to allow for constant but regulated expression of T7 polymerase, to achieve continuous production of biomolecules.
Our genetic circuit is comprised of two parts. The first consists of an interchangeable native fungal promoter upstream of the T7 polymerase gene; the second consists of the T7 promoter upstream of an interchangeable gene we would like the fungi to express. Thus, T7 polymerase is produced in a regulated manner for the strong expression of various genes under the control of the T7 promoter.
To make our genetic circuit modular, we built our parts registry so that our promoters could be cloned into the promoter slot, and our genes (i.e. resistances) could be cloned into the gene slot by using the standardized BioBrick restriction system. Thus, if a gene needs to be expressed in fungi for a desired characteristic, it can easily be incorporated and the promoter tailored for a specific chassis, granting us modularity.
Since the use of homologous promoters can enhance expression levels, we selected two homologous glyceraldehyde-3-phosphate dehydrogenase (gpdA) promoters for expression in Cochliobolus, specifically the Aspergillus nidulans gpdA promoter and the Ganoderma lucidum gpdA promoter. The gpdA promoters have been previously characterized in producing industrial enzymes so we predicted that they could be effectively utilized in our promoter library and in several regulatory constructs. The A. nidulans trpC promoter (PtrpC) and trpC terminator (TtrpC) were also utilized for strong homologous and heterologous gene expression [2]. The Anderson promoter (Pc) and the A. nidulans pelA promoter (PpelA) were selected for characterization of certain constructs, currently still in progress. Pc is a highly constitutive promoter and, in place of a fungal promoter, could allow for construct expression and characterization in bacteria [3]. PpelA is induced by polygalacturonic acid and repressed in the presence of preferred carbon sources, such as glucose, which makes it appropriate for utilization in our kill switch constructs (designed to be inducible and repressible for practical use during manufacturing processes) [4].
A multitude of laboratory techniques were employed to generate our constructs, including PCR, DNA digestion, gel extraction, DNA ligation, and electroporation.
We are also collaborating with Wageningen's iGEM team to characterize Aspergillus nigers-based regulatory elements in our chassis organisms to further expand our library of regulatory elements. We have received the pkiA promoter and the xlnD terminator from them and are in the process of characterizing their strength and efficiency using fluorescence.
References
1. Promoters/Catalog/T7. iGEM Registry of Standard Biological Parts. Accessed from http://parts.igem.org/Promoters/Catalog/T7
2. Moore, Margo M. Genetic Engineering of Fungal Cells. Biotechnology, 3 (Encyclopedia of Life Support Systems). Accessed from http://www.eolss.net/Sample-Chapters/C17/E6-58-03-02.pdf
3. Promoters/Catalog/Anderson. iGEM Registry of Standard Biological Parts. Accessed from http://parts.igem.org/Promoters/Catalog/Anderson
4. Ho MC, Whitehead MP, Cleveland TE, and Dean RA (1995). Sequence analysis of the Aspergillus nidulans pectate lyase pelA gene and evidence for binding of promoter regions to CREA, a regulator of carbon catabolite repression. Current Genetics, 27(2), 142-149.
Retrieved from " http://2013.igem.org/Team:Cornell/project/wetlab/fungal_toolkit/regulatory_elements "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
