"
Page
Discussion
View source
History
teams
Log in
 
Team:Cornell/project/wetlab/fungal toolkit/characterization
From 2013.igem.org
Cornell University Genetically Engineered Machines
Previous
Tour Page
Next
Tour Page
Start
Tour!
Home
Project
Overview
Background
Wet Lab
Dry Lab
Human Practices
Future Applications
Safety
Outreach
Overview
Research Presentations
Panel Discussions
Public Awareness
Notebook
Overview
Protocols
Plasmids
Team
Overview
Profile
Bios
Sponsors
Attributions
Gallery
2013: Organofoam
Team Website
2012: SAFE BET
2011: Biofactory
2010: OMG OMVs
2009: Cadmium Sensor
Wet Lab
Overview
Chassis
Fungal Toolkit
Regulatory Elements
Selectable Markers
Characterization
Homologous Constructs
Biosafety
Protoplasting
Fungal Transformation
Carotenoids
Limonene
Antifungals
BioBricks
Animation
Future Work
Characterization
Flourescence
The green fluorescent protein (GFP) from the jellyfish Aequorea victoria has long been used as a molecular marker and reporter for eukaryotes and prokaryotes. Purified GFP absorbs blue light at a peak of 395 nm and emits green light at 509 nm, producing a stable fluorescence with little to no photobleaching [2]. GFP proved to be a valuable alternative to previous markers used in fungal research such as beta-glucuronidase, which was plagued with problems due to substrate uptake and leakiness [3]. Leaky gene expression is a result of initiation of transcription without the proper activator proteins. GFP was ideal because it does not require an exogenous substrate and does not negatively affect fungal tissue [3].
In our studies, GFP and monomeric red fluorescent protein (mRFP) were utilized as markers for gene expression. In developing our fungal toolkit , we placed GFP and mRFP downstream of numerous promoters including the T7 promoter and fungal promoters PtrpC and A. nidulans PgpdA. The level of fluorescent activity would be indicative of promoter strength.
Experimentation was done on the BL21 bacterial strain, which is the most common bacterial gene expression host and ideal for protein expression. BL21 utilizes the T7 promoter expression system, which is more effective at expressing proteins than any other system because it lacks two proteases, Ion and ompT, that degrade proteins before and after cell lysis [1]. DH5α E. coli was used as a control. In addition, a ribosome binding site (RBS) was added to genetic constructs of the promoter and the fluorescent protein in order to control translation initiation and rate of translation [4]. Hygromycin-resistance gene (hph) and kanamycin resistance gene (nptII) were added to form genetic constructs of promoter, fluorescent protein, and resistance gene.
A fluorescence assay was run using a 96 well plate as described in the protocol section. T7+GFP+RBS and T7+GFP constructs in E. coli BL21 were examined for their relative fluorescence (fluorescence/OD600). A T7 construct in E. coli BL21 without downstream elements was utilized as a control along with the use of the aforementioned constructs in E. coli DH5α. As can be seen from the graph there were evident differences in peak fluorescence between the T7+GFP+RBS construct and the T7+GFP construct and controls. The T7+GFP+RBS construct appears to have a 20 fold higher relative fluorescence than the T7+GFP construct or the controls.
Preliminary fluorescent data as described above was very promising. Further work examining other fluorescent constructs and in a fungal chassis is currently in progress. In particular, of high priority is the combination of fluorescent and resistance markers in one construct. This will facilitate the introduction of our constructs into fungi and allow further characterization.
We are also collaborating with Wageningen's iGEM team to characterize their team's actin-GFP fusion construct in our chassis organisms.
References
1. Biomol. (2004). Ampliqon III Competent Cells. Guide to Gene Expression in BL21, 1-12 Retrieved from http://www.biomol.de/dateien/infos_nr353.pdf
2. Chalfie, M., Tu, Y., Euskirchen, G.,  Ward, W.W., & Prasher, DC. (1994). Green Fluorescent Protein as a Marker for Gene Expression. Science, 263. Retrieved from http://www.bio.davidson.edu/courses/molbio/restricted/02GFPwow/GFPwowpg1.html
3. Maor, R., Puyesky, M., Horwitz, B. A., & Sharon, A. (1998). Use of green fluorescent protein (GFP) for studying development and fungal-plant interaction in Cochliobolus heterostrophus. Mycology Research, 102(4), 491-496. Retrieved from http://www2.tau.ac.il/lifesci/plantsci/as/articles/chetgfp1.pdf
4. Salis, H.M., Mirsky, E.A., & Voigt, C.A. (2010). Automated Design of Synthetic Ribosome Binding Sites to Precisely Control Protein Expression, Nat. Biotechnol. 27(10), 946-950. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2782888/pdf/nihms145791.pdf
Retrieved from " http://2013.igem.org/Team:Cornell/project/wetlab/fungal_toolkit/characterization "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
