<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cornell_notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cornell/notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cornell/notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cornell/notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cornell/notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cornell/notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cornell/notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>Cornell University Genetically Engineered Machines</TITLE></P><DIV class="tour" style="left:0;display:none"><A id="tourPrev" class="button radius"> PreviousTour Page </A></DIV><DIV class="tour" style="right:0;display:none"><A id="tourNext" class="button radius"> NextTour Page </A></DIV><DIV class="tour" style="right:0;display:none"><A id="tourStart" class="button radius"> StartTour! </A></DIV><DIV class="row header off-canvas"><DIV class="twelve columns centered header no-pad" id="header-wrapper"><UL class="nav-bar"><LI class="pg-home"><A href="https://2013.igem.org/Team:Cornell">Home</A></LI><LI class="has-flyout pg-project pg-project_background pg-project_wetlab pg-project_drylab pg-project_hprac pg-project_safety pg-project_future"><A href="https://2013.igem.org/Team:Cornell/project">Project</A></LI><LI class="pg-project"><A href="https://2013.igem.org/Team:Cornell/project">Overview</A></LI><LI class="pg-project_background"><A href="https://2013.igem.org/Team:Cornell/project/background">Background</A></LI><LI class="pg-project_wetlab"><A href="https://2013.igem.org/Team:Cornell/project/wetlab">Wet Lab</A></LI><LI class="pg-project_drylab"><A href="https://2013.igem.org/Team:Cornell/project/drylab">Dry Lab</A></LI><LI class="pg-project_hprac"><A href="https://2013.igem.org/Team:Cornell/project/hprac">Human Practices</A></LI><LI class="pg-project_future"><A href="https://2013.igem.org/Team:Cornell/project/future">Future Applications</A></LI><LI class="pg-project_safety"><A href="https://2013.igem.org/Team:Cornell/project/safety">Safety</A></LI></UL><LI class="has-flyout pg-outreach pg-outreach_presentations pg-outreach_panels pg-outreach_awareness"><A href="https://2013.igem.org/Team:Cornell/outreach">Outreach</A></LI><LI class="pg-outreach"><A href="https://2013.igem.org/Team:Cornell/outreach">Overview</A></LI><LI class="pg-outreach_presentations"><A href="https://2013.igem.org/Team:Cornell/outreach/presentations">Research Presentations</A></LI><LI class="pg-outreach_panels"><A href="https://2013.igem.org/Team:Cornell/outreach/discussions">Panel Discussions</A></LI><LI class="pg-outreach_awareness"><A href="https://2013.igem.org/Team:Cornell/outreach/awareness">Public Awareness</A></LI><LI class="has-flyout pg-notebook pg-notebook_protocols pg-notebook_plasmids"><A href="https://2013.igem.org/Team:Cornell/notebook">Notebook</A></LI><LI class="pg-notebook"><A href="https://2013.igem.org/Team:Cornell/notebook">Overview</A></LI><LI class="pg-notebook_protocols"><A href="https://2013.igem.org/Team:Cornell/notebook/protocols">Protocols</A></LI><LI class="pg-notebook_plasmids"><A href="https://2013.igem.org/Team:Cornell/notebook/plasmids">Plasmids</A></LI><LI class="has-flyout pg-team pg-team_bios pg-team_sponsors pg-team_gallery pg-team_attributions"><A href="https://2013.igem.org/Team:Cornell/team">Team</A></LI><LI class="pg-team"><A href="https://2013.igem.org/Team:Cornell/team">Overview</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Cornell" target="_blank">Profile</A></LI><LI class="pg-team_bios"><A href="https://2013.igem.org/Team:Cornell/team/bios">Bios</A></LI><LI class="pg-team_sponsors"><A href="https://2013.igem.org/Team:Cornell/team/sponsors">Sponsors</A></LI><LI class="pg-team_attributions"><A href="https://2013.igem.org/Team:Cornell/team/attributions">Attributions</A></LI><LI class="pg-team_gallery"><A href="https://2013.igem.org/Team:Cornell/team/gallery">Gallery</A></LI><LI class="pg-badge"><A href="https://2013.igem.org/Team:Cornell" style="margin-top:-50px">2013: Organofoam</A></LI><LI class="pg-badge_home"><A href="http://igem.engineering.cornell.edu">Team Website</A></LI><LI class="pg-badge_2012"><A href="https://2012.igem.org/Team:Cornell">2012: SAFE BET</A></LI><LI class="pg-badge_2011"><A href="https://2011.igem.org/Team:Cornell">2011: Biofactory</A></LI><LI class="pg-badge_2010"><A href="https://2010.igem.org/Team:Cornell">2010: OMG OMVs</A></LI><LI class="pg-badge_2009"><A href="https://2009.igem.org/Team:Cornell" style="font-size:20px !important">2009: Cadmium Sensor</A></LI></DIV></DIV><DIV class="row"><DIV class="two columns"><UL class="side-nav" style="margin-left:15px"><LI><H6> Jump to: </H6></LI><LI class="nb-month"><DIV style="top:67px;margin-left:-6px">Spring</DIV></LI><LI id="weekA" class="active" onclick="weekFilter(this)"><A class="nb-week" href="#A">January</A></LI><LI id="weekB" onclick="weekFilter(this)"><A class="nb-week" href="#B">February</A></LI><LI id="weekC" onclick="weekFilter(this)"><A class="nb-week" href="#C">March</A></LI><LI id="weekD" onclick="weekFilter(this)"><A class="nb-week" href="#D">April</A></LI><LI id="weekE" onclick="weekFilter(this)"><A class="nb-week" href="#E">May</A></LI><LI class="nb-month"><DIV style="top:23px">June</DIV></LI><LI id="week1" onclick="weekFilter(this)"><A class="nb-week" href="#1">Week 1</A></LI><LI id="week2" onclick="weekFilter(this)"><A class="nb-week" href="#2">Week 2</A></LI><LI class="nb-month"><DIV style="top:65px;margin-left:2px">July</DIV></LI><LI id="week3" onclick="weekFilter(this)"><A class="nb-week" href="#3">Week 3</A></LI><LI id="week4" onclick="weekFilter(this)"><A class="nb-week" href="#4">Week 4</A></LI><LI id="week5" onclick="weekFilter(this)"><A class="nb-week" href="#5">Week 5</A></LI><LI id="week6" onclick="weekFilter(this)"><A class="nb-week" href="#6">Week 6</A></LI><LI id="week7" onclick="weekFilter(this)"><A class="nb-week" href="#7">Week 7</A></LI><LI class="nb-month"><DIV style="top:52px;margin-left:-9px">August</DIV></LI><LI id="week8" onclick="weekFilter(this)"><A class="nb-week" href="#8">Week 8</A></LI><LI id="week9" onclick="weekFilter(this)"><A class="nb-week" href="#9">Week 9</A></LI><LI id="week10" onclick="weekFilter(this)"><A class="nb-week" href="#10">Week 10</A></LI><LI id="week11" onclick="weekFilter(this)"><A class="nb-week" href="#11">Week 11</A></LI><LI class="nb-month"><DIV style="top:52px;margin-left:-22px">September</DIV></LI><LI id="week12" onclick="weekFilter(this)"><A class="nb-week" href="#12">Week 12</A></LI><LI id="week13" onclick="weekFilter(this)"><A class="nb-week" href="#13">Week 13</A></LI><LI id="week14" onclick="weekFilter(this)"><A class="nb-week" href="#14">Week 14</A></LI><LI id="week15" onclick="weekFilter(this)"><A class="nb-week" href="#15">Week 15</A></LI><LI class="nb-month"><DIV style="top:55px;margin-left:-12px">October</DIV></LI><LI id="week16" onclick="weekFilter(this)"><A class="nb-week" href="#16">Week 16</A></LI><LI id="week17" onclick="weekFilter(this)"><A class="nb-week" href="#17">Week 17</A></LI><LI id="week18" onclick="weekFilter(this)"><A class="nb-week" href="#18">Week 18</A></LI><LI id="week19" onclick="weekFilter(this)"><A class="nb-week" href="#19">Week 19</A></LI></UL></DIV><DIV class="ten columns team-bios-container"><DIV class="row" id="ugd-members"><DIV class="twelve columns"><H2 class="centered">Notebook</H2></DIV></DIV><DIV class="row last-ele" style="padding-bottom:0px"><H4 style="margin-left:15px;text-align:left">Filter by subteam: 
					<DIV id="nb-all" class="nb-all" onclick="showAll()">show all subteams</DIV><DIV id="nb-alltech" class="nb-all-i" onclick="showAllTech()" style="margin-right:15px">show technical details</DIV></H4></DIV><DIV class="row last-ele nb-labels" style="padding-bottom:0px"><DIV style="opacity:0.00;width:65px">&quot;_&quot;</DIV><DIV>Wet Lab</DIV><DIV>Dry Lab</DIV><DIV>Human Practices</DIV><DIV>Wiki</DIV><DIV>Presenters</DIV></DIV><DIV class="row"><DIV class="two columns nb-filter"><DIV id="wet-only" class="nb-only" onclick="onlyFilter(this)">only</DIV></DIV><DIV class="two columns nb-filter"><DIV id="dry-only" class="nb-only" onclick="onlyFilter(this)">only</DIV></DIV><DIV class="two columns nb-filter"><DIV id="hprac-only" class="nb-only" onclick="onlyFilter(this)">only</DIV></DIV><DIV class="two columns nb-filter"><DIV id="wiki-only" class="nb-only" onclick="onlyFilter(this)">only</DIV></DIV><DIV class="two columns nb-filter"><DIV id="anim-only" class="nb-only" onclick="onlyFilter(this)">only</DIV></DIV></DIV><DIV id="weekAentry" class="nb-week" style="display: block"><DIV class="twelve columns"><H3 class="centered">January</H3><DIV class="entry nb-wet"><DIV><H6>January 5th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Applications for new Cornell iGEM members were due today-- we've got 57! 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>January 16th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		After an eternity of reading and a gigantic spreadsheet of evaluations, we're finally ready to start interviewing applicants-- 36 of them! 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>January 25th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		I'm really excited about having these new members on the team. There were many awesome interviews, but now we have to sit down and make hard decisions on who to take, as most of last year's members are graduating, so there's a limit to how many new members we can train. This is going to be a tough weekend. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>January 27th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		After several hours of discussion, we decided on our 21 new members! Congratulations, Tim Abbott, Hannah Ajmani, Eric Appel, Ryan Ashley, Nupur Bhatt, Arun Chakravorty, Rebecca Chew, Sharlene Dong, Sara Gregg, Alex Han, Eric Holmes, Daniel Leach, Oat Luengvarinkul, Jeffrey Ly, Ritvik Sarkar, Mac Sennett, Prashant Sharma, Olya Spassibojko, Yoshiko Toyoda, and Kyle Wheeler! 
		-Rafael
</P></DIV></DIV></DIV></DIV><DIV id="weekBentry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">February</H3><DIV class="entry nb-wet"><DIV><H6>February 1st
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Speed-taboodunit is a mashup of speed-dating, taboo, and whodunit; the idea is that each person writes on a piece of paper something unusual about themselves, underlining the key words, then these pieces of paper are randomly redistributed. Everyone organizes in a speed-dating format, with two lines of people, and you have a minute to talk to them and try to find out if they're the person described on their piece of paper-- with the catch that they can't use any of the underlined words. People who successfully find their person leave the group, but otherwise after the minute everyone moves down the line. As for why we came up with this specific icebreaker, we wanted to hit two key requirements of icebreakers: they have to be fun, and they have to make specific accommodations for being shy or quiet. Because each person has the specific objective of finding their person, this helps them overcome the shyness, and as for the fun, everyone loves taboo!</DIV><P>		We had our first full team meeting today with all the new members, introducing them to the iGEM competition, and did an icebreaker we call speed-taboodunit! For the next week, they have the assignment of analyzing a previous iGEM project (in groups), and explaining what the project was trying to accomplish, whether synthetic biology was a good approach for this, and reasons for their success or lack thereof. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>February 7th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Our first social event tonight went well! Hopefully with a few more of these we'll have some great team comradery and feel more comfortable around each other. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>February 9th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		At our second team meeting we went over the past project analyses, as well as some biology review-- mostly molecular cloning. For the next week, we've assigned more past projects for them to outline in more detail. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>February 16th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Today we went over literature review, went over the past project outlines, and gave a two-week problem-oriented brainstorming and project outline assignment. Now that each member has looked at several past projects and analyzed them, we're moving into brainstorming for this year's project and practicing the development of project pitches. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wiki"><DIV><H6>February 21st
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		We just started working on the wiki! Our first assignment: figuring out how wiki templates work. With templates the website would become much easier to edit, but for some reason HTML and wiki text don't seem to work very well together... 
		-Nupur
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>February 23th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		For the second week of this assignment, we compiled the brainstormed ideas and had members pick which ones they wanted to explore further. We also had a movie night planned, but it fell through :( 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wiki"><DIV><H6>February 28th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Finally got templates to work! The trick was ending HTML right before wiki text, and starting HTML again right afterwards. Not very elegant, but it works. 
		-Nupur
</P></DIV></DIV></DIV></DIV><DIV id="weekCentry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">March</H3><DIV class="entry nb-wiki"><DIV><H6>March 1st
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Check out our <A href="https://2012.igem.org/wiki/index.php?title=Team:Cornell/Sandbox/notebook&amp;oldid=299789" target="_blank"> notebook</A> before anything worked!</DIV><P>		We want to revamp the notebook page with a filter system that lets you sort by subteam. We set up the basic layout of the page, but writing the javascript is going to be another story. 
		-Nupur
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>March 2nd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Groups over the previous week investigated phage therapy, piezoelectric bacteria, &quot;sticky paper&quot;, logic gates / bistable switches, phytases, mycelia, and pressure-adaptive materials. We're keeping the results from those, but doing a second round of brainstorming over the next few weeks-- the idea is to allow the exploration of any ideas that arose while researching the previous round of ideas. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wiki"><DIV><H6>March 5th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		We just finished the javascript for filtering. Rafael also made cool subteam icons for the filter buttons. 
		-Nupur
</P></DIV></DIV><DIV class="entry nb-wiki"><DIV><H6>March 6th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		I made a template for notebook entries and added styling to each entry. 
		-Nupur
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>March 9th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		We compiled our second round of ideas, including topics like methanotrophs, a CRISPR knockout system, and suppressor tRNA inducible expression systems. We also took our first team picture, and probably the only one that will have all of last year's and this year's members together! 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>March 29rd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">A lot of iGEM teams we've looked at don't have much beyond the biology, so we really try hard to do interdisciplinary projects, as they provide a great opportunity for learning to work with different kinds of people.</DIV><P>		Spring break knocked out a couple of meetings, but it looks like some research was accomplished nonetheless! The phytase project idea is looking pretty promising-- it incorporates a lot of different disciplines, with stuff for our mechanical and electrical engineers to do, as well as modeling. 
		-Rafael
</P></DIV></DIV></DIV></DIV><DIV id="weekDentry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">April</H3><DIV class="entry nb-wet"><DIV><H6>April 6th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		We've narrowed it down to four project ideas: smokestack reactors (capturing pollutants with bioreactors), incorporation of phytases into anaerobic digesters, mycelial materials, and using bacteria to address issues with artificial implantable kidneys. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>April 10th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		We had a call today with Ecovative Design, a company that uses mushrooms to make a biodegradable styrofoam analogue-- our mycelial materials project idea would involve working with them to improve their product. It seems like our interests are aligned and that we should be able to work with them. Exciting! 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>April 13th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		This meeting was dedicated to project pitches from each of the four ideas, followed by voting on which project we should pursue. Some of our graduate advisors also attended to provide a more seasoned perspective on the viability of each project. After some deliberation and spreadsheet magic, we decided on mycelial materials! 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>April 20th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Today we worked on team structure for the summer; Swati will be the overall team leader, with Mark and I coordinating wetlab, Olya taking meeting minutes, Mac running drylab, and Hannah will be managing human practices. Later on in the summer, we will also develop task groups. For the next week, we've also split into groups to research various aspects of the mycelial materials project. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>April 26th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		We've compiled initial research on carotenoid pigments, transformation protocols, DNA that may be useful for a basidiomycete toolkit, and a few other topics. Over the next week we'll meet with several professors on campus who work with fungi to see what resources and advice they can provide. 
		-Rafael
</P></DIV></DIV></DIV></DIV><DIV id="weekEentry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">May</H3><DIV class="entry nb-wet"><DIV><H6>May 4th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		We met with Professor Gillian Turgeon, who does fungal transformation with Cochliobolus heterostrophus, an ascomycotic corn pathogen. We can get a few of the genes we've been looking for from her, yay! We also met with Professor Teresa Pawlowska, who directed us towards some other fungi and resources for fungi, as well as provided some background on fungal biology. As for the rest of the semester, we're letting everyone off the hook-- we don't want anyone to fail their exams because of iGEM! 
		-Rafael
</P></DIV></DIV></DIV></DIV><DIV id="week1entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 1</H3><H5 class="centered">(06/17 - 06/23)</H5><DIV class="entry nb-dry"><DIV><H6>June 17th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		The drylab team met briefly on June 17 to begin discussing some ideas.  We went over some of the growing methods for Ganoderma lucidum, brainstormed some non-packaging related ideas for a fungal biomaterial, and discussed how we could test some of the physical characteristics of the final product.  We thought it might be a good idea to test the thermal conductivity if it were to be used as a drinking cup.  We also determined that people probably do not want to drink out of a fungus cup. 
		-Mac
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 18th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We transformed parts containing carotenoid pathway genes <I>crtE</I>, <I>crtI</I>, <I>crtB</I>, and <I>crtY</I> (BBa_K523022, BBa_K539119, BBa_K145001), as well as the T7 promoter (both in pIVEX2.3d as well as BBa_I712074), the T7 polymerase gene (BBa_K145001), and the transformation efficiency kit part (BBa_J04450).</DIV><P>		We started bootcamp today, doing our first transformations with kit plate biobricks. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 19th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">From the plasmid pUCATPH (pAh13G), we amplified the <I>trpC</I> promoter (P<I>trpC</I>, a constitutive fungal promoter) and the <I>trpC</I> terminator (T<I>trpC</I>). From pNG (pAg13H), we amplified <I>nptII</I>, the geneticin (a plant/fungal antibiotic) resistance gene.</DIV><P>		The second day of bootcamp had more activity-- we PCR-amplified a variety of parts to biobrick them and get ready for making composite parts later. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 20th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We miniprepped the <I>crtEIB</I> (BBa_K523022), <I>crtY</I> (BBa_K539119), and T7 (BBa_I712074) biobricks. The gels confirmed our PCRs for P<I>trpC</I>, T<I>trpC</I>, and <I>nptII</I>, which we then digested to insert into pSB1C3.</DIV><P>		On our third day of bootcamp, we taught new members how to run gels, do minipreps, and run digests, continuing our preparation of biobrick parts from the previous two days. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wiki"><DIV><H6>June 20th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Check out the <A href="https://static.igem.org/mediawiki/2013/2/21/CornellStillGrowing.gif" target="_blank">gif</A> if it's not on our homepage anymore. Mycodraw simulates mycelium growth, using an image as food. We were inspired by the work we found on <A href="http://onecm.com/projects/mycelium/" target="_blank">this site</A>. Originally Rafael converted his mycodraw program from java to javascript (which took a VERY long time), but the script only worked on some browsers :( We had to settle for the gif instead.</DIV><P>		I finally started moving pages to the 2013 wiki! It's looking very empty right now, but Rafael made a gif of our logo with his mycodraw program to use as a homepage placeholder. We also decided to add a technical details section to each of our notebook entries so that we can separate the jargon from daily summaries. 
		-Nupur
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 21st
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">P<I>trpC</I>, T<I>trpC</I>, and <I>nptII</I> were column-purified, digested pSB1C3 backbone was gel-purified and dephosphorylated, and each was ligated into the pSB1C3 backbone.</DIV><P>		Continuing with bootcamp, we column-purified the digests of PCR products and gel-purified the digested pSB1C3 backbone, dephosphorylated, and ligated, resulting in our first batch of biobricks or pre-biobrick parts. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wiki"><DIV><H6>June 21st
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		The javascript for toggling technical details took a bit longer than expected, but we finished it up today. 
		-Nupur
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 22nd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We transformed T<I>trpC</I> (pC13Q), P<I>trpC</I> (pC13P), and <I>nptII</I> (pCg13R) constructs into DH5α via electroporation.</DIV><P>		Our penultimate day of bootcamp was a light one-- we only transformed cells with the constructs we'd made over the past four days. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wiki"><DIV><H6>June 22nd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		I added a flyout to notebook entries authors with a picture, short description, and link to their bio on the team page. 
		-Nupur
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 23rd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We used Q5 PCRs with the standard biobrick primers VF2 and VR to attempt to confirm the presence of T<I>trpC</I> (pC13Q), P<I>trpC</I> (pC13P), and <I>nptII</I> (pCg13R).</DIV><P>		We concluded bootcamp with colony PCRs to test whether each of the constructs was present. Unfortunately, the gels had no bands, so we grew overnight cultures and will screen the minipreps instead. 
		-Rafael
</P></DIV></DIV></DIV></DIV><DIV id="week2entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 2</H3><H5 class="centered">(06/24 - 06/30)</H5><DIV class="entry nb-wet"><DIV><H6>June 24th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Minipreps of T<I>trpC</I> (pC13Q), P<I>trpC</I> (pC13P), and <I>nptII</I> (pCg13R) were performed with an EZNA kit, and afterwards we quantified with a nanodrop.</DIV><P>		We miniprepped our three constructs. Then we went back to sleep. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>June 24th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		The drylab team met to update each other on some of the preliminary research.  The main focus of this meeting was the development of a growing protocol.  Sara found a professor that we could contact in the material science department for help regarding the testing of material properties.  Rebecca and Manny looked into what different aspects of the fungi we could model, such as growth, decay, etc.  Ritvik brainstormed some more ideas regarding the potential uses of this material, and came up with the idea of using it as insulation. 
		-Mac
</P></DIV></DIV><DIV class="entry nb-wiki"><DIV><H6>June 24th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		I started working on a hypertexting script that will automatically recognize words on a page and add a flyout with a description. We can use this to recognize names for the author flyouts I implemented yesterday, as well as other keywords such as plasmid names and protocols. 
		-Nupur
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 25th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">The Q5 PCRs of T<I>trpC</I> (pC13Q), P<I>trpC</I> (pC13P), and <I>nptII</I> (pCg13R) showed bands of the correct size, so sequencing with VF2 and VR was submitted to make sure nothing strange had happened.</DIV><P>		We did another PCR from the miniprepped plasmids to confirm the right sizes of inserts, then sent them in for sequencing. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wiki"><DIV><H6>June 25th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		I fixed a problem with the hypertext script not looping, but broke something else that causes everything to crash. I'm going to sleep. I'll figure it out tomorrow. 
		-Nupur
</P></DIV></DIV><DIV class="entry nb-wiki"><DIV><H6>June 26th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		The script is now working on all browsers except for Internet Explorer. 
		-Nupur
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 27th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">pC13B (<I>crtY</I>) was digested with XbaI and PstI, and pAK13D (T7 promoter) with SpeI and PstI (standard <A href="http://tools.ietf.org/html/rfc10" class="external mw-magiclink-rfc">RFC 10</A> cloning). However, the gel showed only one band for each (what one would expect for pAK13D, but pC13B should have had two bands).</DIV><P>		We started cloning crt genes downsteam of the T7 promoter. Because the T7 promoter is too small to extract as a fragment, we're placing crt genes downstream, then transferring the promoter+gene fragments back to pSB1C3. Today we digested <I>crtY</I> and the T7 promoter plasmid, but the digestion of <I>crtY</I> failed. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 28th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">pC13B (<I>crtY</I>) was digested with XbaI and PstI, and pAK13D (T7 promoter) with SpeI and PstI (standard <A href="http://tools.ietf.org/html/rfc10" class="external mw-magiclink-rfc">RFC 10</A> cloning). The gel this time had two bands at the correct locations for pC13B, so we're hoping it's a success this time. We also started overnight LB cultures of  pAb13T (pBARGPE1).</DIV><P>		We redid the digestion of <I>crtY</I> and the T7 promoter plasmid from yesterday, apparently successfully this time. We also started liquid cultures of pBARGPE1, a plasmid we got from the Fungal Genetics Stock Center. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 29th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">If successful, the ligation will form pAK13AD. pC13K (<I>crtI</I>, BBa_K118003) and pC13M (<I>crtB</I>, BBa_K118002) were digested with XbaI and PstI for insertion into pAK13D, and separated on a gel. pAb13T (pBARGPE1) was miniprepped and quantified by nanodrop.</DIV><P>		Crossing our fingers, we gel extracted and ligated the ostensibly successful digests of <I>crtY</I> and the T7 promoter plasmid, and then transformed. We also digested <I>crtI</I> and <I>crtB</I>, other carotenoid genes that we will also be putting under the T7 promoter, and miniprepped the pBARGPE1 that had been grown up last night. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 30th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">pC13K (<I>crtI</I>, BBa_K118003) and pC13M (<I>crtB</I>, BBa_K118002) were digested with XbaI and PstI, pAK13D with SpeI and PstI, pC13Q (<I>TtrpC</I>) with EcoRI and XbaI, and pCg13R (<I>nptII</I>) with EcoRI and SpeI, and all were separated on a gel; <I>crtI</I> looked faint, and pAK13D was smeared, but all fragments were cut out for purification tomorrow.</DIV><P>		Unfortunately, there were no colonies on the transformation plate for <I>T7</I> + <I>crtY</I>, so we redid the digestions for those. We also digested <I>TtrpC</I> and <I>nptII</I> for the eventual cloning of a <I>PtrpC</I>+<I>nptII</I>+<I>TtrpC</I> construct (we have to put the parts together in reverse because <I>nptII</I> contains a PstI site, which we will be trying to remove as well). Gel extractions of digested <I>crtI</I> and <I>crtB</I> were also empty (insignificant concentrations), so these digestions were also redone, and all were run on a gel, which looked mostly okay. 
		-Rafael
</P></DIV></DIV></DIV></DIV><DIV id="week3entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 3</H3><H5 class="centered">(07/01 - 07/07)</H5><DIV class="entry nb-wet"><DIV><H6>July 1st
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Digested pAK13D was dephosphorylated, then <I>crtI</I> , <I>crtB</I>, and <I>crtY</I> were ligated with pAK13D to put them under the T7 promoter, hopefully producing pAK13AB, pAK13AC, and pAK13AD, respectively. pCg13R was also ligated with pC13Q, hopefully producing pCg13U. All four constructs were transformed into DH5α via electroporation.</DIV><P>		Yesterday's digestions were gel extracted, ligated, and transformed. Hopefully it worked this time! 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-hprac"><DIV><H6>July 1st
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Price of one 12 oz Styrofoam cup: 4.09 cents</DIV><P>		Two of our team members participated in SILS Skills Night on June 20th. 
		-Hannah
</P></DIV></DIV><DIV class="entry nb-wiki"><DIV><H6>July 1st
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Now we just have to start filling out entries...</DIV><P>		We finished the hypertexting script and fixed some miscellaneous problems. I think we can call this a wrap. The notebook page is officially done! 
		-Nupur
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 2nd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech"><I>hph</I> was amplified with Gibson primers from pAh13G, and the pSB1C3 backbone was amplified with Gibson primers from pC13F (BBa_J04450). When assembled, they will form pCh13V.</DIV><P>		There were no transformants, again :(. On the other hand, we received our tube of <I>Ganoderma Lucidum</I> from Dr. David Hibbett today! Once more with feeling, we redid the four digestion-ligations for <I>crtI</I>, <I>crtB</I>, <I>crtY</I>, and <I>nptII</I>. We also amplified <I>hph</I>, the hygromycin resistance gene, from one of the plasmids we got from Dr. Turgeon's lab, as well as pSB1C3 backbone, and we will use Gibson assembly to put them together (as <I>hph</I> has both EcoRI and PstI sites)-- the gels looked good for both of these. Finally, we submitted pAb13T (pBARGPE1) for sequencing to determine the sequences of the <I>Aspergillus nidulans</I><I>gpdA</I> promoter and the <I>bar</I> gene, which confers phosphonothricin resistance. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 3rd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">:( ;_; T_T D: (tears for the transformations lost). The <I>bar</I> PCR and pC13F were digested with EcoRI and PstI, and ligated to hopefully form pCp13X. Site-directed mutagenesis was performed both with and without DpnI to digest template DNA.</DIV><P>		All transformations failed yet again, and were repeated once more. Four tears shed for the lost. <I>hph</I> was Gibson-assembled, and we ran a Q5 PCR of the <I>bar</I> gene, under the assumption it matches other published <I>bar</I> sequences, and the product digested and ligated with pSB1C3. We also ran site-directed mutagenesis on <I>nptII</I> (pCg13R) to remove its internal PstI site. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 4th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">The previous days ligations were all desalted and transformed, yet again, into E. Coli and plated, along with non-GC trials of <I>nptII</I> site-directed mutagenesis. Digestions on <I>bar</I> and pSB1C3 were purified and run on a gel. They appeared successful, and pSB1C3 was gel-extracted for ligation with the PCR-cleanup of <I>bar</I>. The digests of pSB1C3 and <I>bar</I> were quantified and ligated for submission as a Biobrick. Finally, liquid broth cultures of the hygromycin Gibson transformants were created.</DIV><P>		The previous days ligations were all desalted and transformed, yet again. 
		-Kyle
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 5th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">The site directed mutagenesis cultures of <I>nptII</I> were put into liquid cultures from plates. One liquid culture of a transformant of <I>crtY</I> in pAK13D was also made, along with a reference plate. A ligation of <I>bar</I> in pSB1C3 was desalted and transformed. Only one of the Gibson assemblies appeared successful, failing to express rfp, and a glycerol stock was made of that culture. After further incubation, the successful culture was miniprepped and PCR'ed.  When run on a gel, that culture appeared ineffective.</DIV><P>		All of yesterdays transformations failed again, in addition to the <I>hph</I> Gibson :-( The ligase was tested by ligating DNA ladder: All were discovered to be non-functional. Why life? Why? 
		-Kyle
</P></DIV></DIV><DIV class="entry nb-wiki"><DIV><H6>July 5th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Rafael added badges to the headers of all of our wikis, and implemented a badge flyout that links them all together. 
		-Nupur
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 6th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Colonies of <I>nptII</I>, <I>nptII</I> digested with DpnI, <I>hph</I>, and <I>crtY</I> were miniprepped. <I>nptII</I>+ DpnI sample 4 was accidentally pipetted into the sample 3 column, so contamination is possible. Concentrations were good. A digest screen was performed on <I>nptII</I> using PstI, and <I>crtY</I> was digested with EcoRI and PstI. The <I>nptII</I> digested with DpnI appeared successful, while the undigested samples failed. Gels of <I>hph</I>, and <I>crtY</I> were also run, showing that the <I>crtY</I> ligations failed, as well as the <I>hph</I> Gibson.</DIV><P>		The <I>crtY</I> ligation was run on a gel and shown to be unsuccessful. The <I>hph</I> PCR also came up blank, so the Gibson was also unsuccessful. On the bright side, a digestion screen on <I>nptII</I> shows that site-directed mutagenesis on that site most likely worked! 
		-Kyle
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 7th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">A Miniprep was run on three <I>bar</I> cultures in preparation for later ligation into pSB1C3. These were then run on a gel along with Gibson-ready pSB1C3. The gel was successful, <I>bar</I> was confirmed, and a gel extraction was performed on pSB1C3 and frozen.</DIV><P>		A pSB1C3 backbone was prepared for Gibson assembly and gel extracted; however, we later were told by a representative from New England Biolabs that gel extraction might mess up a Gibson assembly, rendering todays extractions useless. 
		-Kyle
</P></DIV></DIV></DIV></DIV><DIV id="week4entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 4</H3><H5 class="centered">(07/08 - 07/14)</H5><DIV class="entry nb-wet"><DIV><H6>July 8th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Digested <I>nptII</I> in pCg13S with EcoRI+XbaI, and TtrpC with EcoRI+SpeI. The <I>crtY</I> in pSB1C3 was digested four different ways: One with just XbaI &amp; PstI; one with XbaI, PstI, and SacI; one with XbaI, PstI, and ApaLI; and one with XbaI, PstI, SacI, and ApaLI. Although unlikely, this method is intended to prevent re-ligation of the backbone of <I>crtY</I> during digestion. More <I>crtY</I>, <I>crtI</I>, <I>crtB</I> was inoculated in liquid cultures for future miniprepping.</DIV><P>		Several constructs were ligated. The mutated <I>nptII</I> (pCg13S) was ligated with <I>TtrpC</I>. Assembled <I>bar</I> in pSB1C3 and submitted for sequencing. The Gibson of <I>hph</I> into pSB1C3 was rerun. An experimental method of ligating <I>crtY</I> into pAK13D upstream of the T7 promoter was tried as an alternative to gel purification and PCR purification, since we lack the primers to PCR and gel purification interferes with ligation. 
		-Kyle
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 9th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><P>		All yesterdays ligations were electroporated and plated. The Gibson of <I>hph</I> into pSB1C3 grew red colonies, showing it was unsuccessful, so the procedure was performed again. The sequencing of the <I>bar</I> ligation came back successful! 
		-Kyle
</P></DIV></DIV><DIV class="entry nb-wiki"><DIV><H6>July 9th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><P>		Our mircroryza crowdfunding site was launched today. We spent a long time filming and putting together a <A href="http://www.youtube.com/watch?v=dvBgyJckozs" target="_blank">video</A> to let everyone know about Cornell iGEM! 
		-Nupur
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 10th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">The <I>nptII</I>+<I>TtrpC</I> ligation was checked by doing a colony PCR on the ligation colonies and a subsequent gel. Possible gel outcomes were plasmid self ligation (200bp) in which <I>TtrpC</I> was randomly cut out of pC13Q, 800bp if the plasmid has only <I>TtrpC</I>, ~1kb indicating <I>nptII</I> noise from PCR template, and ~1.7 kb indicating successful construct formulation. The gel showed two bars at ~800bp, indicating a failed ligation. The third Gibson attempt of <I>hph</I> into pSB1C3 was also plated, and a 10mL culture was made of the successful <I>nptII</I> DpnI mutagenesis plate, evidently for future great justice.</DIV><P>		The <I>nptII</I> ligation with <I>TtrpC</I> grew colonies was unsuccessful. In other news, much lab organization was done today! 
		-Kyle
</P></DIV></DIV><DIV class="entry nb-hprac"><DIV><H6>July 10th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Five team members visited a STEM camp in Canandaigua, NY. We spent two hours making DNA necklaces with kids ranging from ages 7-14. 
		-Hannah
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 11th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Minipreps were made from the <I>crtY</I> plates, from which a digest screen was prepared using NotI to see if the ligation worked.  The digestions were run on a gel. Samples with XbaI, PstI, and SpeI; XbaI, PstI, and ApaLI numbers 2 and 3; and XbaI, PstI, SpeI, and ApaLI numbers 2 and 3 all worked.  A miniprep of <I>nptIIBB</I> was performed. The concentration was extremely low, so an overnight culture was made from the glycerol stock to be miniprepped tomorrow.  Another transformation was done using remaining previous minipreps of <I>nptII BB</I>.  Cultures were made of the third Gibson attempt, and a colony PCR performed. When run on a gel, the non-GC fragments were too small, but some GC colonies looked good.  Finally, <I>nptII BB</I> was digested with XbaI and PstI, then column purified. <I>PtrpC</I> was digested with SpeI and PstI and gel purified.</DIV><P>		Digest screens of previous <I>crtY</I> ligations showed promising results. Gibson cloning was continued, and further cloning of <I>nptIIBB</I>, to place it in front of <I>PtrpC</I>, was prepared. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>July 11th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		After several brainstorming sessions, we decided to build an intelligent incubation chamber to facilitate the growth of our mushrooms in a controlled environment. We hope to achieve this by using a temperature and humidity sensor with a feedback control loop. 
		-Rebecca
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 12th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">In the morning, the digestions of <I>nptIIBB</I> and <I>PtrpC</I> were quantified. A miniprep of <I>nptIIBB</I> transformants was performed to replenish DNA, and another glycerol stock of <I>nptII BB</I> was made from a leftover culture.  The promising mutagenesis GC colonies from yesterday's gel were prepared for sequencing. A ligation was performed on <I>nptIIBB</I> and <I>PtrpC</I>. More cultures of the successful <I>crtY</I> ligations were made.  CYM medium was created, and <I>PC13Z</I> was transformed from a kit plate.</DIV><P>		Cloning with <I>nptIIBB</I> and <I>PtrpC</I> was continued, as well as cloning with the Gibson Assembly method. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 13th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">The ligation of <I>nptIIBB</I> with <I>PtrpC</I> was desalted, transformed into E. coli using electroporation, and plated.  Antibiotic stocks of geneticin and phosphinothricin were made at 100x concentration.  <I>G. lucidum</I> cultures were inoculated with varying levels of antibiotic to test the strain's sensitivity for future selection purposes.</DIV><P>		Today we worked with fungi! We applied different amounts of antibiotic to <I>G. lucidum</I>, to test the sensitivity of our strain of fungi.  In addition, cloning of pCg13Y was nearly completed. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 14th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Colony PCRs and cultures were completed to determine the success of cloning <I>nptIIBB</I> downstream of <I>PtrpC</I>.  Unfortunately, a gel revealed that the DNA fragments appeared at 0.6kb, shorter than the expected 1.4kb of a successful ligation. pC13Z and pAK13AD were miniprepped, but had to be thrown out due to contamination of our EB.  New cultures were made to be miniprepped again.  A glycerol stock of pC13Z was also made.</DIV><P>		Cloning of <I>nptIIBB</I> in front of <I>PtrpC</I> was unsuccessful. Plasmids pC13Z and pAK13AD were prepped for cloning. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>July 14th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		This week, drylab split up so members could focus on their strengths.   Rebecca and Sara began designing our feedback controller, with some much appreciated advice from the previous drylab team leader, Dan Levine.  Ritvik, Manny, and Mac began coming up with design considerations for the incubation chamber.  We were most concerned with size; too small and it wouldn't be useful, too big and it would be difficult to control any variables.  We eventually decided that a 1'x1'x2' chamber would probably be a good compromise.  Rebecca, Sara, Ritvik, and Mac also attended machine shop training this week.  In order to complete our training, we each had to construct a small aluminum lamp using a milling machine and a lathe.  Fortunately, everyone emerged from the training session one lamp richer, and with a full set of fingers on each hand. 
		-Mac
</P></DIV></DIV></DIV></DIV><DIV id="week5entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 5</H3><H5 class="centered">(07/15 - 07/21)</H5><DIV class="entry nb-wet"><DIV><H6>July 15th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Due to the contaminated EB found yesterday, our EB supplies were checked to ensure no further delays. To begin, the Gibson <I>hph</I> was submitted for sequencing, and pAK13AD and pC13Z were miniprepped. A PCR of pC13Z, pC13F, pCP13X, pC13M, and pC13E was then run with primers 31 and 32, the standard pSB1C3 primers.  A gel was run to check the PCRs. All the PCRs looked successful except pC13M and pC13E, which had nonspecific bands around 1kb.  A new technique was used on a few of the digests involving a restriction cocktail of enzymes to cut the DNA. Plasmid pAK13AD was digested with EcoRI, SpeI, SacI, and ClaI so that it could be placed in pSB1C3. We used SpeI instead of PstI due to the small distance between the PstI and the SacI cut sites.  Plasmid pC13F was digested with EcoRI, SpeI, and KpnI so pAK13AD could be placed in it.  Plasmid pAK13D was digested with SpeI and PstI so <I>crtI</I> and <I>crtB</I> could be placed in it.  All the digests were then column purified rather than gel purified--a move which has improved our ligation efficiencies. The purifications were quantified.  Cultures were made for pC13P and pC13K and, finally, pAK13AD was dephosphorylated then ligated into pC13F in order to make pC13AH.</DIV><P>		Gibson <I>hph</I> was submitted for sequencing, and ALL of the cloning was performed. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>July 15th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Rebecca looked into the components of the temperature feedback control, namely, the temperature sensor, the microcontroller, and the heating element.  We put in an order for the temperature sensor, microcontroller (an Arduino Fio), and Xbee ports for wireless communications. 
		-Rebecca
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 16th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Sequencing results came back positive for the Gibson cloning of <I>hph</I>, so a culture of the successful Gibson colony was made.  pC13AH was transformed and plated, and pC13K were miniprepped.  pC13X, pC13F, pC13Z were digested with SpeI and PstI along with pC13P (XbaI/PstI), and all were column purified.  Reran a PCR of pC13M and pC13E, and one of pC13K with standard biobrick primers.  All PCRs were cleaned, then run on a gel along with pBARGPE1 which had been digested with NotI. pC13E and pC13M looked good, but pC13K was too short. pBARGPE1 had a 4kb fragment (super-coiled) and pBARGPE1 with NotI had a 5-6kb fragment, meaning it has a NotI site. Ligations of pAK13D to inserts pC13Z, pC13X, and pC13F along with vector pC13P to inserts pC13Z, pC13X, and pC13F were performed.  Also, overnight digestions of pC13M, pC13E, and pC13K with XbaI and PstI were set up.  Transformation was attempted for pC13AI and pA13AJ, but  the cuvettes weren't cleaned properly so it failed.  Finally, our dwindling supplies prompted the ordering of more SpeI and membrane filters.</DIV><P>		The Gibson sequencing came back positive! We then digested pC13X, pC13F, pC13Z and pC13P, and ligated pAK13D to inserts pC13Z, pC13X and pC13F, and pC13P to inserts to inserts pC13Z, pC13X and pC13F. We ran a PCR for pC13M, pC13E and pC13K. Sadly pC13K did not work… Transformation was attempted for pC13AI and pA13AJ but they both failed--we suspected that the cuvettes weren't cleaned properly. 
		-Danielle
</P></DIV></DIV><DIV class="entry nb-hprac"><DIV><H6>July 16th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Price of one 12 oz Styrofoam cup: 4.95 cents</DIV><P>		One iGEM member spoke at a panel for pre-freshmen biology students interested in pursuing research positions in college (PSP). 
		-Hannah
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 17th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">A glycerol stock of pCh13V (<I>hph</I> Gibson) was created, then miniprepped for later site directed mutagenesis of <I>hph</I> (which has both E and P cut sites).  Digestions of pC13M (<I>crtB</I>) and pC13E (T7 polymerase) were column purified and quantified.  The PCR of pC13K with GC and non-GC trials was redone, since the fragment was of the incorrect length during gel imaging yesterday - goal is to ligate to pAK13D.  Following the PCR, a gel was run which showed no bands. Perhaps the polymerase was forgotten? Another PCR of pC13K was performed and run on gel. A band showed at exactly 1.8kb as expected.  pC13M was ligated to pAK13D, placing <I>crtB</I> downstream of the T7 promoter.  pC13E was ligated to pC13P, placing the <I>T7</I> polymerase gene downstream of <I>PtrpC</I> (a native fungal promoter) - this is the first part of our genetic circuit with the second being the <I>T7</I> promoter upstream of various genes (<I>GFP</I> and <I>RFP</I> for diagnostic purposes - to quantify promoter functionality and the carotenoid genes for proof of concept).  Plates of pC13AH (T7 + crtY) had many colonies, so they were restreaked, and colony PCRs were run while cultures were made of some of the colonies.  A gel of the PCRs was run (image name was pC13AHcolonypcr7-17-13); gel was inconclusive (correct band was accompanied by strong incorrect band at 1kb). Will test again using digest screen of minipreps tomorrow.  Heat shock transformed <I>pT7</I> + <I>GFP</I> (pAK13D + pC13Z), <I>PtrpC</I> + <I>GFP</I> (pC13P + pC13Z), <I>pT7</I> + <I>BAR</I> (pAK13D + pCp13X), <I>PtrpC</I> + <I>BAR</I> (pC13P + pCp13X) ligations into chemically competent E. coli.  Heat shock transformed pC13AI (lox site), pA13AJ (lox site), pC13AK (mRFP coding sequence), and pK13AL (mRFP coding sequence) from Kit Plates 3 and 5 into chemically competent E. coli.  Heat shock transformed today's earlier ligations: pAK13D + pC13M, and pC13P + pC13E and plated all transformations in accordance with the selection marker on the plasmid.</DIV><P>		We reran a PCR for pC13K and the band was as expected on the gel! pC13M was ligated to pC13D, and pC13E was ligated to pC13P. The resulting constructs were used to transform <I>E. Coli</I> cultures. The following transformations were also done: <I>PT7</I> + <I>GFP</I>, <I>PtrpC</I> + <I>GFP</I>, <I>PT7</I> + <I>BAR</I>, <I>PtrpC</I> + <I>BAR</I>. A colony PCR was run for pC13AH. 
		-Danielle
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 18th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">All 18 (yes, 18!) of the <I>pC13AH</I> colonies that had been screened by colony PCR yesterday were miniprepped.  We digested the miniprepped colonies of pC13AH clones (T7 promoter with crtY in pSB1C3) with <I>NotI</I> to see if the restriction cocktail method of cloning worked.  On the gel that was subsequently run, bands were expected at ~2.1 for pSB1C3 and ~1.2 for <I>crtY</I>+<I>pT7</I>.  The restriction cocktail method worked! The bands appeared at the correct lengths on the gel for <I>pC13AH</I>.  There was not enough miniprepped DNA for pC13AH to send for sequencing, so the colonies were cultured again for another miniprep.  None of the ligations into pAK13D (pSB1AK8 + PT7) worked -- pC13Z (<I>GFP</I>), pC13M (<I>crtB</I>), and pCp13X (<I>bar</I>) all failed =&gt; try cloning only using restriction cocktail method, instead of attempting PCR with VF2 &amp; VR.  Digested pC13Z, pC13M, pCp13X, pC13K (<I>crtI</I>) with XbaI and PstI in addition to ApaLI and SacI for later ligation with pAK13. pAK13D had been previously digested with SpeI and PstI, column purified, and dephosphorylated for insert ligation.  More pAK13D was grown to acquire more DNA to be miniprepped tomorrow.  Nupur performed colony PCR and made cultures of transformants from yesterday. All transformations appeared to have very low efficiency--perhaps an issue with heat shock stocks?  Transformations of <I>AI-AL</I> (mRFP cds and lox site parts) all yielded colonies, though at really low efficiencies compared to most transformations off of kit plates. These were screened by colony PCR to verify.  Ligations into pC13P all yielded colonies, but at very low efficiencies. These were pC13E (T7 pol), pCp13X (bar), and pC13Z. All were screened by colony PCR as well.  Taq was used to perform 5 colony PCRs, then all colony PCRs were run on a gel.  Also, pCg13S (nptIIBB)was digested with XbaI, PstI, ApaLI, and SacI for later ligation to pC13P, which had been previously digested with SpeI and PstI, column purified, and dephosphorylated. Overnight ligations of pC13Z, pC13M, pCp13X, pC13K, pCg13S with vector pAK13D were set up, along with overnight ligations of pC13K and pCg13S in vector pC13P.</DIV><P>		The restriction cocktail method of cloning worked! Cloning of pC13AH was shown to work using this method.  We are also continuing our cloning of the <I>crtI</I> and <I>crtB</I> pathways. 
		-Danielle
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 19th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Made glycerol stocks of pC13AT, pC13AY, pC13AV, pC13AI, pA13AJ, pC13AK, pK13AL, and pAK13D. Miniprepped pC13AH cultures and sent in for sequencing. Ran a digest screen on pC13AI, pA13AJ, and pC13AL using NotI. Reran a PCR of pC13AV and pC13AT with Q5 polymerase. Attempted transformation of yesterdays ligations, with success only on pAK13AN and pAK13AD. A gel was run of pC13AL, pC13AJ, pC13AI (all digested w/ NotI), as well as the PCR of pC13AV and pC13AT. Plasmid pC13AT appeared to have worked, as well as pC13AI and pC13AJ, which had strong bands in the ~2kb range and weak bands in the ~1.5kb range, about the size of the whole plasmid.</DIV><P>		Sent pC13AH in for sequencing, and checked several plasmids for the presence of the correct genes using digestion screens and gel analysis. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 20th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Plasmid pC13AY was prepared for sequencing. Transformed pC13BD, pCg13AY, pAK13AC, pAK13BB, pAK13AB using heat shock. Recovered for ~1hr 10min, then plated on ampicillin plates made by spreading antibiotic on a sterile plate. More ampicillin plates were then created from scratch.</DIV><P>		Plasmid pC13AY was prepared for sequencing, and several transformations using heat shock were performed. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 21st
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Miniprepped pAK13AN, pAK13AD and ran a digest screen using NotI. The gel was inconclusive. The previous days transformants (pC13BD, pCg13AY, pAK13AC, pAK13BB, and pAK13AB) were cultured.</DIV><P>		Made cultures of the previous days transformants, and ran an inconclusive digest screen of pAK13AN and pAK13AD. A slow day. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>July 21st
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		This week, the fabrication group focused on what materials to construct our chamber from.  At first we considered a single wall design, but this was eventually scrapped, as we were unable to find any material that would provide sufficient insulation at a reasonable thickness (and cost).  After putting our heads together, we decided on a double wall with an insulating layer in between them.  Mac began drawing up the chamber in Solidworks. 
		-Mac
</P></DIV></DIV></DIV></DIV><DIV id="week6entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 6</H3><H5 class="centered">(07/22 - 07/28)</H5><DIV class="entry nb-wet"><DIV><H6>July 22nd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Cultures pAK13AB number 5, pCg13Y number 1, pAK13AC number 1, and pCg13Y number 3 from yesterday didn't grow for some reason. Ran a NotI digest screen of all the pAK13D ligations, as well as colony PCRs of the pC13P ligations. Plasmid pAK13AB numbers 1, 3, 4 and possibly pAK13BB number 4 looked like they worked on the gel. The gel of the colony PCRs looked weird, possibly because they were run on wrong extension time for a while. Going to rerun PCRs. We also re-ran a gel of the NotI digest screens from yesterday, pAK13AP and pAK13AN. All the pAK13AP colonies had blank colonies, but pAK13AN colony 1 had the correct bands. Sent pAK13AN colony 1 for sequencing but accidentally sent in the wrong primers. Primers for pSb1AK8 should have been used, but pSB13C primers were used instead. Transformed pC13AV ligation and pC13AL. The transformation arced, but the time constant was still 3.8 so the cells were plated anyway. Finally, 2 250mL solutions of cym agar were made and autoclaved. Began drying mycelium to isolate the genomic DNA.</DIV><P>		Sent in pC13AH, pC13AY, and pC13AT for sequencing. In addition, the digest screens of all the pAK13D ligations were redone, along with the colony PCRs of the pC13P ligations. Transformed pC13AV. Attempted to dry mycelium to isolate genomic DNA. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 23rd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">The plate of pK13AL transformation had no colonies, so it was given more time to grow. The plate of pC13AV had a lawn, but also lots of small isolated colonies. Plate was re-streaked to get fewer colonies before culturing. In addition, miniprepped and made a glycerol stock of pAK13AN and re-ran a colony PCR of pC13BD and pCg13Y with Taq polymerase. Ran a gel to see if ligation worked, but the bands were too short. This may be due to using Taq instead of Q5, so the PCR was run again with Q5. Retried cloning pAK13AP and PAK13AC, digesting pAK13D with SpeI and PstI, and pC13X and pC13K with XbaI and PstI. Overnight ligations were created, after column purification and dephosphorylation. We will redo a digestion of pC13AK tomorrow because we digested with SpeI and PstI instead of XbaI and PstI.  We began cloning of pC13AQ, pC13BC and pC13AF, digesting backbone and insert with EcoRI and PstI. We made more DH5alpha electrocompetent cells to try to make heat shock. To replenish, TAE and LB were made. <I>G. lucidum</I> finished drying.</DIV><P>		Miniprepped and made a glycerol stock of pAK13AN. Also retried the cloning for pAK13AP and pAK13AC and began cloning for pAK13AO. Plasmid pC13AH was sequence confirmed! 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>July 23rd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		The parts have arrived and Rebecca configured them. Met up with Paras who was on last year's team to get a detailed explanation of how their project had worked as well. 
		-Rebecca
</P></DIV></DIV><DIV class="entry nb-hprac"><DIV><H6>July 23rd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Five team members gave a two hour presentation including a lab tour to a group of 25 high school biology teachers (CIBT) 
		-Hannah
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 24th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Ran a gel of yesterday's Q5 PCR of pCg13Y and pC13BD, but only one sample of the latter looked good and was sent in for sequencing. Isolating <I>G. lucidum</I> DNA did not work because it did not precipitate. Next time, we will freeze the cells with liquid nitrogen. Re-tried cloning of pC13AY, pC13AT, and pCg13Y. Plasmids pC13E, pC13Z and pCg13S were digested with XpaI, PstI, SacI, and ApaII, while a previous digest of pC13P was used. Subsequently, pC13E, pC13Z, and pCg13S were column purified and the plasmids left to ligate overnight. Tried to redo cloning of pC13AU and pAK13AO because pC13AK was digested with the wrong enzymes yesterday, but failed once again. Finally, pC13I was cloned and pC13BD and pC13Z were submitted for sequencing.</DIV><P>		Ran a gel of the Q5 PCR of the pCg13Y and pC13BD. One sample of pC13BD looked good, so it was sent in for sequencing. Isolating <I>G. lucidum</I>'s DNA didn't work. Will try flash freezing the cells with liquid nitrogen next time. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 25th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">After growing up more pC13AK, attempted site directed mutagenesis to get rid of the PstI internal cut site in pCh13V. After running a gel against an unmutated control, found that GC worked while non-GC didn't. Afterwards, both were digested with DpnI and column purified. Attempted to clone pC13I. Also transformed ALL of the constructs, including: pC13AK, pK13AL, pA13AM, pCp13Y + pAK13AD, pAK13AC, pAK13AO, pCg13Y, pC13AY, pC13AU, pC13AV, pC13AT, pC13AQ, pC13AF, pC13I(crtE in psB1C3), and pCH13V GC with PstI mutation.</DIV><P>		We tried the site directed mutagenesis of pCh13V to remove the Pst1 internal cut site. Looking at the gel, it looks like the site directed mutagenesis worked! In addition, we cloned <I>crtE</I> into pSB1C3 and performed a ton of transformations! 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 26th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Sequencing results showed that pC13BD had the correct reverse sequencing, but the forward sequencing is being re-run. In addition, all 20 transformations failed due to incompetent stocks. Miniprepped and made glycerol stocks of pC13AK for more DNA and made many new plates. Also submitted the <I>lox</I> site for sequencing and ran a PCR of pC13F to append Kpn1 and BamH1 restriction sites to PSB1C3 for cloning of <I>PpelA</I>. We also ran PCRs using primers 27 and 28 to amplify <I>A. nidulans</I><I>PgpdA</I> out of pAb13T, and ran a PCR using primers 29 and 33 to amplify <I>crtE</I> + <I>rbs</I> for pC13J out of pC13A. When all was said and done, the gel showed that only the PCR of <I>A. nidulans</I> looked promising, so we cloned this into pC13F.</DIV><P>		Sadly, all 20 transformants failed and the stocks are incompetent. We received new cuvettes to retry all of them.  We submitted <I>lox</I> site for sequencing of pBARGPE1 and ran a PCR of pC13F to append Kpn1 and BamHI restriction sites to pSB1C3 for cloning of “PpelA”. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 27th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Prepped pC13BD for sequencing, and retried the PCR of pC13F to append KpnI and BamHI restriction sites to clone <I>PpelA</I>. Ran 63 colony PCRs and made 97 cultures of all the previous ligations/transformations from two days ago. Also ran gels of the colony PCRs, but forgot to run the pC13F ones.</DIV><P>		We retried the PCR of pC13F, ran 63 colony PCRs, and made 97 cultures! 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 28th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Ran PCRs of pC13F on a gel to check for <I>PpelA</I>. The Non-GC PCR products looked good, but the products in which we used the GC Enhancer did not work. Cleaned up the non-GC enhancer PCR of pC13F and digested it with Kpn1 and BamH1. Also digested pHNU with Kpn1, BamH1, and Sac1. Then the double digest products were ligated overnight. The cultures of pA13AM (they did not grow before) were also retried. Miniprepped the cultures from yesterday whose colony PCR gel screens looked promising. Prepped the colony PCRs for: pC13AQ number 1, pC13AV number 5, pC13V  PstI mutant numbers 2 and 3, pC13AF 4.3 and 4.5, and pCg13Y number 1. Also retried the questionable colony PCRs with Q5 off of the minipreps: pC13AY numbers 1-3, pC13AU numbers 1 and 5, pC13I numbers 4 and 5, pC13AT numbers 2 and 3, pAK13BC numbers 1 and 5, and pC13AF numbers 3.2 and 3.3. We digest screened all non-chloramphenicol minipreps using NotI-HF for pAK13AC numbers 1-5, and pAK13AO from July 23rd numbers 1-5. For everything else EcoRI and PstI was used.</DIV><P>		We digested pC13F and miniprepped cultures of yesterday's successful colony PCRs. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>July 28th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		This week, the fabrication group finalized the incubator design and ordered the construction materials!  We decided to construct the outer and inner layers out of HDPE plastic in the rather lab appropriate color of white.  For our insulation, we decided to use high density polystyrene.  After comparing the K-factors for various types of insulation, we determined that this material gave us the most bang for our buck, so to speak. 
		-Mac
</P></DIV></DIV></DIV></DIV><DIV id="week7entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 7</H3><H5 class="centered">(07/29 - 08/04)</H5><DIV class="entry nb-wet"><DIV><H6>July 29th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Ran a gel of the questionable PCRs from yesterday. Plasmids pC13AY number 1, pC13AU number 5, pC13I number 5, pAK13BC number 5, and pC13AF number 3.2 looked good, but the gel was smeary so will re-run tomorrow. From the digest screens, pK13AL numbers 1-3, pAK13AP numbers 1-3 and 5, and pAK13AC number 1 looked good on a gel. The site-directed mutagenesis of pCh13V was continued, with a PCR of pCh13V run to mutate the EcoRI site. Sample number 2 of the non-GC samples appeared to have worked when run on a gel, so a DpnI digest and transformation were performed. Miniprepped pA13AM and ran a colony PCR, revealing that it was not in pSB1C3 as expected and thus was thrown out. Ran a digest screen of pA13AM with NotI to be checked on a gel tomorrow. <I>PpelA</I> ligated into pSB1C3 and transformed. Finally, after a long day, retried the PCR of pC13A to obtain <I>crtE</I>.</DIV><P>		Reran many PCRs and other procedures. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 30th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Submitted pC13BD, pC13AT, pC13AV, pCh13V mutant and pAK13AB for forward sequencing. Yesterdays transformations failed. A digest screen of pC13A and pC13AM revealed that the primers for pC13A were not functioning, and still no pC13J. Reran colony PCRs from gel yesterday. Digested pC13AL with XbaI, PstI-HF, ClaI, and SacI-HF and ligated into pAK13D, as well as digested pAK13AP and pAK13AC with XbaI and PstI-HF and ligated both into pSB1C3. Re-ran the ligation of <I>PpelA</I> into pSB1C3, as well as the mutagenesis PCR of pCh13V. Fungal DNA was successfully extracted!</DIV><P>		Yesterdays transformations all failed, and other mishaps. But fungal DNA extraction was successful! 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 31st
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Column purified pC13W DpnI digests: 3 GC and 3 nGC transformed and plated. Dephosphorylated pA13AM and ligated with pC13E. Transformed ligations from yesterday: pAK13AO, pC13AS number 3, pC13AG number 1, and pC13BF. Successful returns from sequencing of pC13BD, pCg13Y, pAK13AN and pC13V PstI mutant. Transformed and plated <I>Aspergillus</I><I>Pgpd</I> in pC13F GC and non-GC.</DIV><P>		Successful returns from sequencing of pC13BD, pCg13Y, pAK13AN and pC13V PstI mutant. In other news, the PCRs of <I>Ganoderma</I> DNA looked good!! 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>July 31st
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Sara and Rebecca met up with our advisor Dr Bruce Land and he provided lots of helpful advice with regard to the design considerations (where the heater should be in our incubation chamber, convection currents, type of heating plate to use). He also suggested that we use a proportional-integral-derivative controller (PID controller) to regulate the heating. 
		-Rebecca
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 1st
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We did colony PCRs with biobrick primers 31 and 32. We also made cultures of yesterday's transformations: pC13D+pK13AL, pC13D+pCp13X #3 in pSB1C3, pC13D+pC13K #1 in pSB1C3, pC13W with GC #3, pC13W no GC #3, and <I>PpelA</I> in pSB1C3. A gel was run to see if the ligations were successful. pC13D+pC13X,  PpelA, pC13W GC 3.1 and 3.5, and pC13W nGC 3.1 all looked like they worked. The pA13AM+pC13E ligation from yesterday was transformed and plated. <I>G. lucidum</I> GPD homology regions were cloned, column purified, and digested with EcoRI and Pst.  pC13F was also digested with EcoRI and Pst. Unfortunately, digestions of the homology regions yielded no DNA, but digestion of pC13F did yield some and thus was dephosphorylated. There was not enough template from the PCR cleanup present for homology regions, so PCRs were redone for each homology region.</DIV><P>		Ligation of the <I>PpelA promoter</I>, hygromycin resistance, and <I>T7 promoter</I> in front of the herbicide-resistance bar gene worked. We continued cloning of our other constructs and repeated PCRs of <I>G. lucidum</I> GPD homology regions. 	-Tina
</P></DIV></DIV><DIV class="entry nb-hprac"><DIV><H6>August 1st
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Team visited Ecovative Design's facilities in Troy, NY. Co-founder and Chief Scientist gave us a tour and tips on working with fungi. 
		-Hannah
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 2nd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We cloned <I>G. Lucidum</I>GPD homology regions (pC13BJ, pc13BK, pC13BL, pC13BM). Yesterday's PCR reruns were run on a gel to make sure the right regions were amplified. All have correct bands! We column purified the reruns and digested with EcoRI and PstI. We also digested more <I>pC13F</I> with EcoRI and PstI and dephosphorylated the vector. We purified and ligated the reruns with pC13F. The cultures that looked good from yesterday's colony PCRs were miniprepped. We then made cultures and did a colony PCR of A.Pgpd in pSB1C3 GC from July 31st. A gel of the colony PCRs was run - colonies 1, 3, 4, and 5 looked successful and had the correct band insert length. A transformation of pA13AM+pC13E ligation did not work since the plate was contaminated. Sequencing of pC13AS #1, 3, 5 and pCh13W (hph BB) GC 3.1, 3.5, nGC 3.1 was prepared. An overnight culture of PpelA #2,3,5 was set up for miniprepping tomorrow and sequencing. We transformed pC13P+pC13E #3 and plated for cultures tomorrow, which we will then miniprep and sequence.</DIV><P>		We continued working with the PCRs of the GPD homology regions and the successful ligations of the previous day's PpelA promoter, hygromycin resistance, and T7 promoter in front of the herbicide-resistance bar gene, as well as T7 polymerase behind a constitutive promoter and <I>Aspergillus</I> promoter. 
		-Tina
</P></DIV></DIV><DIV class="entry nb-hprac"><DIV><H6>August 2nd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Video conference with Tuft's iGEM team to discuss collaboration on their human practices approach. 
		-Hannah
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 3rd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We inoculated five colonies of pC13AY and five colonies of pA13AM+pC13E. We performed colony PCR of pC13AY transformants. A gel was run, but we realized that the PCR failed due to the use of Taq instead of Q5 and a 1 min shorter cycling time than the protocol required. We grew a culture of pC13P for miniprepping, digesting with Spe and PstI, and ligation with pC13Z and pK13AL.  A digest screen of pAK13AO minipreps was performed - they were incorrect. pC13M, pC13K, pC13Z, pK13AL, and pCg13S were digested with Xba, Pst, Sac, ApaLI. An overnight ligation of pC13M + pAK13D , pCg13S + pAK13D, pC13K + pAK13D was performed. PpelA and A.Pgpd cultures were miniprepped. pC13F+pC13BJ, pC13F+pC13BK, pC13F+pC13BM, and pC13F+pC13BL were all electroporated and plated.</DIV><P>		We continued cloning of T7 polymerase behind a constitutive promoter, transforming the GPD homology regions, and biobricking the carotenoid and geneticin resistance genes. 
		-Tina
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 4th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We miniprepped pC13AY, pC13AM+pC13E colonies, and pC13P. The minipreps were stored in the fridge, and glycerol stocks were made in case one of them works. We grew overnight cultures of five colonies from pC13F+pC13BJ, pC13F+pC13BK, pC13F+pC13BM, and pC13F+pC13BL from yesterday's transformations. 
We transformed and plated pC13M + pAK13D, pCg13S +pAK13D, and pC13K +pAK13D. More LB agar and ampicillin plates were also made today.</DIV><P>		We grew cultures of the GPD homology region transformations and transformed yesterday's carotenoid genes and geneticin resistance constructs. 
		-Tina
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>August 4th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Mac went to visit Ecovative Design with Swati, Rafael, Alex, and Hannah.  We were able to meet with Gavin McIntyre, one of the founders of the company, who was kind enough to give us a tour of their facilities.  He also gave our wetlab members some pointers for our own project.  In other news, we are still waiting for our materials to arrive. 
		-Mac
</P></DIV></DIV></DIV></DIV><DIV id="week8entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 8</H3><H5 class="centered">(08/05 - 08/11)</H5><DIV class="entry nb-wet"><DIV><H6>August 5th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We electroporated pA13AM into <I>DH5α</I> and plated on ampicillin infused plates. We will know if the transformation was successful by the presence of red colonies as a result of <I>mRFP</I> in the plasmid. We found that our construction of pC13AY failed after running a digest screen as none of the lanes displayed bands at around 3kb, which corresponds to inserted <I>PtrpC</I> and <I>T7 polymerase</I> and 2kb corresponding to the vector. However, the gel may have been showing unique banding patterns for the construct, so another gel will be run to check for any supercoiling or linearized plasmids. We also submitted our constructs pC13BF and pC13BG for sequencing. Colony PCRs were done for pAK13AB, pAK13AC, and pAK13BB. A few pAK13AC and pAK13BB colonies were successful.</DIV><P>		Our previous construction of pC13AY failed. We submitted pC13BF and pC13BG for  sequencing. We ran a colony PCR for pAK13AB, pAK13AC and pAK13BB. A few pAK13AC and pAK13BB colonies were successful. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-hprac"><DIV><H6>August 5th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Eric and Arun presented for a group of scientific researchers (SILS) 
		-Hannah
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 6th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Yesterday’s transformation of pA13AM yielded no transformants, so we retried the transformation today. We ran a digest screen of <I>PtrpC</I> and <I>T7 polymerase</I> parts and found that the <I>PtrpC</I> used for our previous construction of pC13AY was of poor quality. We then set up another ligation of <I>PtrpC</I> and <I>T7 polymerase</I> using successfully screened parts. For fluorescence characterization, we double digested pAK13D and pC13P with SpeI and PstI for subsequent ligation with <I>GFP</I> and <I>mRFP</I> inserts. We then quantified our construct containing the <I>G. lucidum</I> homologous regions in pSB1C3, but we had a low yield. We will wait for a new miniprep kit to grow new cultures and perform the miniprep again. We redid a colony PCR for pAK13AB for extra colonies. They all failed so we need to reconstruct pAK13AB.</DIV><P>		After the previous failure of pC13AY construction, we recreated it with successfully screened parts. We redid a colony PCR for pAK13AB, which failed again. We will need to reconstruct pAK13AB. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 7th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We designed primers for <I>lox</I> Gibson assembly and appended cut sites to genes flanked by <I>lox</I>. We also found a possible viable kill switch in the Holin/Chitinase system. Our second try at transforming pA13AM was successful as shown by the presence of red colonies and we set up LB cultures for subsequent cloning. Sequencing showed that construction of pC13BG was successful and cultures were made for further use. However, sequencing of pC13BF was unsuccessful and we reran the PCR of pC13F in order to append BamHI and KpnI cut sites for ligation with pA13BE. We ran another digest screen for pC13AY but many lanes did not have bands so we will retry it tomorrow. We also amplified <I>GFP</I> from pC13Z and <I>mRFP</I> from pC13AK using primers 23 and 24 in a Q5 PCR. Afterwards, a gel screen on PCR products was run to verify that the correct genes were amplified. The gel image indicated consistent bands with lengths of 0.7 kb, thus the PCR to obtain <I>GFP</I> and <I>mRFP</I> inserts was successful! We made a rough draft of our protoplasting protocol to attempt in the near future, too. We then rummaged through previous notebook entries to figure out the problems with <I>crtB</I> and <I>crtI</I>.</DIV><P>		We received sequencing back for pC13BF and pC13BG and only the later was successful, so we restarted construction for pC13BF. We also ran a PCR to amplify <I>GFP</I> and <I>mRFP</I> inserts out of their respective plasmids. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>August 7th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		The sensor has been connected! 
		-Rebecca
</P></DIV></DIV><DIV class="entry nb-wiki"><DIV><H6>August 7th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		We helped <A href="/Team:Ciencias-UNAM" title="Team:Ciencias-UNAM">Ciencias-UNAM</A> get set to transfer their HTML website to the wiki. Hopefully we'll get up a tutorial for what to look out for on here soon! 
		-Nupur
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 8th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We ran a PCR of pA13BW. The <I>CRE</I> gene amplified correctly and the sample was column purified, yielding 54.8ng/uL (with GC enhancer) and 49.9ng/uL (with non GC enhancer). We ran out of supplies to miniprep, so we pelleted pA13AM and pC13BG cultures and began to not so patiently await the arrival of a new miniprep kit. Our PCR of pC13F to append BamHI and KpnI cut sites yesterday failed completely, so we digested pC13F with KpnI to linearize it with an internal cut site in <I>mRFP</I> for another PCR to be run tomorrow. Another gel was run of the pC13AY digests and all lanes were blank. There may have been an issue in quantification by nanodrop that was reading over 1000 ng/μL that mislead us into using low volumes of DNA for digestions. We decided to electroporate with the remaining pC13AY to miniprep and redo gel screening. On a brighter note, our restriction cocktail digestion of pA13BE with BamHI, KpnI and SacI showed a band at 500bp, which is the approximate length of <I>PpelA</I>. We also searched for a pAK13AN miniprep in the DNA storage box and for a glycerol stock of the same plasmid. We couldn’t find either. However, we did obtain a plate of colonies that had been transformed with pAK13AN. We picked colonies off this plate to miniprep and send off to sequencing. In addition, we column purified our inserts from the PCR run yesterday of pC13Z and pK13AL. We then quantified inserts after column purification. After, we digested backbones pAK13D and pC13P with SpeI and PstI and vector dephosphorylated both. The inserts, <I>GFP</I> in pC13Z and <I>mRFP</I> in pK13AL were then digested with XbaI and PstI. We ran a gel of backbones and inserts for confirmation of digests. Unfortunately, the results showed no bands from DNA. Perhaps there was not enough DNA from backbones for proper visualization. We also received materials for protoplasting today, and autoclaved or sterilized the supplies in whatever way we could. We made a filtration system such that all the filtering process during <I>G. lucidum</I> protoplasting will remain sterile. We made another stock of <I>G. lucidum</I> in CYM media, as well as 25 mL cultures of pC13F+pC13BK, pC13BL, pC13BJ, and pC13BM. We made glycerol stocks of pC13M and pC13K from glycerol stocks. We submitted pAK13AC for sequencing and ran a Q5 PCR for amplifying insert for constructing pC13I from pC13A.</DIV><P>		Cloning of the <I>Cre</I> gene and <I>PpelA</I> is going well, and we submitted <I>PT7</I> with <I>GFP</I> for sequencing, as well as pC13AS, pC13W, pAK13BB, and pAK13AC. 
		-Jonlin
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 9th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We digested both the PCR products of <I>Cre</I> and pC13F with EcoRI and SpeI. We may have found a limonene synthase gene we can use. We ran another PCR to append cut sites to the linearized pC13F that ultimately failed with no bands visible on a gel. As our previous construction of pC13AY likely failed since no colonies being present on the new plate, we redid digestions of pC13E and pC13P to redo the construction. We also redigested both pAK13D and pC13P minipreps with SpeI and PstI since the gel from yesterday was inconclusive. We then ligated pAK13D with <I>GFP</I>, pAK13D with <I>mRFP</I>, pC13P with <I>GFP</I> and pC13P with <I>mRFP</I>. We prepared pellets from our cultures of pC13F+pC13BK, pC13BL, pC13BJ, and pC13BM cells which were stored in the fridge as we wait for our miniprep kit to arrive. Sequencing of pAK13AC has failed, and we will resubmit a miniprep of it. We plated stocks of pC13M and pC13K.</DIV><P>		Cloning of the Cre gene is going well, and we may have found a useful limonene synthase gene! We are continuing the previously problematic cloning of <I>GFP</I> with T7 promoter-hopefully it works this time! Sequencing of pAK13AC unfortunately failed. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 10th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We did a fast ligation (30 minutes, room temperature) of <I>Cre</I> with pC13F and electroporated. The time constant was low, so the transformation may not have worked. We continued the ligation overnight. We were still waiting on a miniprep kit so we pelleted our pA13AM and pC13BG cultures to prevent further growth. We transformed with our new pC13AY ligation to hopefully get some colonies this time. We also ran a PCR of pC13F to append homologous regions to <I>G. lucidum</I><I>PgpdA</I> for Gibson assembly to create pC13BH and another PCR to retry appending BamHI and KpnI cut sites to pC13F using half to reaction volume to see if efficiency would be improved. We also electroporated the 4 ligations for fluorescence from yesterday (pAK13D with <I>GFP</I>, pAK13D with <I>mRFP</I>, pC13P with <I>GFP</I> and pC13P with <I>mRFP</I>) along with a control of pC13F to make sure that the electrocompetent cell stocks made yesterday are good. We plated the electroporated cells onto selective antibiotic resistance plates and incubated in 37 degree incubator overnight. We made our first attempt at protoplasting <I>G. lucidum</I> today. However we were having difficulty forming pellets of protoplasts, so we viewed the samples under a microscope. It appeared that the protoplasting attempt was unsuccessful. pC13M and pC13K plates were lawns on their plates. We set up overnight cultures of pC13AS, pC13W, and pC13Y, planning to miniprep and make glycerol stocks.</DIV><P>		We began constructing pC13BH with Gibson assembly, as well as making our first attempt at protoplasting <I>G. lucidum</I>. Electroporations of fluorescence constructs was also done today. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-hprac"><DIV><H6>August 10th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Five team members presented at the Ithaca Sustainability Center for citizens of Tompkins County. 
		-Hannah
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 11th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">The electroporation from the 10th did not work (there were no colonies). Electroporation was redone on the overnight ligation of <I>Cre</I> and pC13F and a better time constant was observed (even with poor cuvette drying). We transformed limonene synthase from 2013 kit plate 4. We discovered that we need spectinomycin to use the holin parts registry plates. We got colonies from the pC13AY transformation and cultured them for gel screening. For fluorescence, no colonies were found on pAK13AN and pAK13AO plates. We centrifuged potential pC13AQ cultures for 10 min at 5000g and poured off supernatant. We kept the cell pellets in the fridge for mini prepping tomorrow. We also found two colonies each on the <I>PtrpC</I> and <I>GFP</I> plate and <I>PtrpC</I> and <I>mRFP</I> plate. We picked off all the colonies and grew each up in 5 ml liquid LB with CM antibiotic. pC13M and pC13K plates are now ready for miniprepping. We could not miniprep the cultures of pC13AS, pC13W, and pC13Y due to running out of our miniprepping supplies.</DIV><P>		We are starting cloning of limonene synthase and continuing cloning of the <I>Cre</I> gene. 
Cloning of fluorescent constructs is going well. 
		-Rafael
</P></DIV></DIV></DIV></DIV><DIV id="week9entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 9</H3><H5 class="centered">(08/12 - 08/18)</H5><DIV class="entry nb-wet"><DIV><H6>August 12th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We still didn’t receive a miniprep kit so we pelleted the pC13AY cultures. The PCRs we ran on pC13F to append homologous regions and append cut sites may have been successful. DNA was present but in low concentrations that we could not check on a gel. We had transformants for pC13BX and pC13CJ so we ran a colony PCR and made cultures. The colony PCRs showed little to nothing on the gel, perhaps because Taq polymerase lost activity. We will miniprep the culture and use Q5. We also attempted a restriction cocktail digest today for both <I>mRFP</I> and <I>GFP</I> inserts. We cut pC13Z and pC13AK with Xba, PstI, SacI, and ApaLI. We also performed a colony PCR of pC13AT cultures and pC13AU cultures using primer 23 and 24 to verify length of inserts. The lengths should be around .7kb for each insert. When we ran a gel of the PCR products, the colony PCR showed a .7kb insert for pC13AT colony 1 and a .3kb insert for pC13AT colony 2. The bands for pC13AU were inconclusive. We will miniprep the potential pC13AT colony 1 and send for sequencing once the miniprep kit arrives. We digested our plasmid pAh13G with SphI in order to create linear DNA for fungal transformation. We attempted to transform this into what <I>G. lucidum</I> protoplasts we had generated, but the electroporation lead to arcing. Arching most likely resulted from the sodium citrate in the cell solution which we were instructed to use by the literature. Nevertheless, we plated the cells on hygromycin cym agar.</DIV><P>		Cloning of <I>PtrpC</I> and <I>GFP</I> appears to be going well. Our initial attempt at protoplasting and transforming <I>G. lucidum</I> does not appear to have been successful. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 13th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We spun down the pC13BX and pC13CJ cultures and put them in the fridge to be miniprepped. We also ligated <I>T7</I> with <I>mRFP</I>, <I>PtrpC</I> with <I>GFP</I>, and <I>PtrpC</I> with <I>mRFP</I> since restriction cocktail digests looked good from the gel yesterday. We then electroporated the ligations and plated them. The plates were left overnight in the incubator. We created another culture of <I>G. lucidum</I> this time in an effort to obtain solid pieces at the top of the media instead of mycelium floating throughout.</DIV><P>		We continued cloning of <I>T7</I> with <I>mRFP</I>, <I>PtrpC</I> with <I>GFP</I>, and <I>PtrpC</I> with <I>mRFP</I>. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 14th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We ran a PCR to append homologous regions to <I>G. lucidum</I><I>PgpdA</I> for Gibson assembly with pC13F that succeeded as verified by a gel screen, but we accidentally used a RNA binding tube during purification and lost all the DNA. Needless to say, that mistake won’t happen again. Luckily, when we ran the same PCR again it was successful. We ran a colony PCR from the grown cultures of pC13BX and pC13CJ to visualize the insert using Q5. We got a positive insert for pC13BX. So this will be miniprepped and sent for sequencing once the miniprep kit is in. Unfortunately, no colonies appeared on the potential pAK13AO plate. We also decided to transform RBS (pA13CZ) off of a kit plate for subsequent miniprepping/digestion/ligation with <I>GFP</I> and <I>mRFP</I>. The <I>T7 promoter</I> needs a ribosomal binding site in order to express in <I>E. coli</I>. We then retried the ligation of pAK13D with <I>mRFP</I> using a 6:1 insert to vector ratio. We attempted to protoplast a huge chunk of mycelium, but had difficulty forming a pellet again. At this point, we decided to enlist the help of Professor Gillian Turgeon, who performs fungal transformations in her lab. We made some hygromycin plates to test transformants, and digested pAh13G with Sph1 to linearize, but this time we column purified the product.</DIV><P>		Homologous regions were successfully appended to <I>PgpdA</I>, and cloning of the Cre recombinase appeared to be successful. We made another attempt at protoplasting, which was again unsuccessful. For characterization, we began cloning RBS into our constructs containing <I>T7</I>, since the promoter need a ribosomal binding site to express in <I>E. coli</I>. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 15th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We ran a Gibson assembly of pC13F and <I>G. lucidum</I><I>PgpdA</I> to create pC13BH and then transformed and plated the new construct. As our previous attempts to append BamHI and KpnI cut sites to pC13F failed, we again ran another PCR, but tried using four times the amount of dNTPs. We ran a PCR of pC13CJ once again from the cultures. The gel looks good for colony 4 and okay for colony 3. We will miniprep colony 4 and send it in for sequencing. We also made five 5ml liquid LB cultures of pA13CZ and miniprepped pC13AQ. We ran a digest screen of select pC13F+pC13BJ, pC13F+pC13BK, pC13F+pC13BL, pC13F+pc13BM colonies with EcoRI and SpeI. Our initial gel didn’t show inserts (lengths of 130 and 230 base pairs) so we ran a second gel with 2% agarose. We still couldn’t see inserts, so we ran digest screens on all colonies (1-5) and ran a new gel. We saw very faint bands for some of them, and nothing for others. Additionally, Arun created a device to use the shaking power of our vortex to agitate a sample in the waterbath. We made LB cultures of pC13A, pC13D, pC13F, pC13K, pC13L, pC13M, pC13N, pAK13AC and pAK13BB. We set up new overnight cultures of pC13AS, pC13W, and pC13Y.</DIV><P>		We ran a Gibson assembly to create pC13BH and transformed the construct. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>August 15th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Rebecca looked more into the details of the pulse-width modulation and PID controller so that the heater can be turned on for certain periods of time for more even heating. 
		-Rebecca
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 16th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">The miniprep kit arrived, so we finally were able to miniprep our pelleted pC13BG, pC13AY and pA13AM cultures. We got transformants for the Gibson assembly of pC13BH and we cultured them. The PCR to append cut sites to pC13F again failed, so we need to extensively troubleshoot this PCR. We made large cultures of pC13BX colony 1 and pC13CJ colony 4 to miniprep and sequence. We made electrocompetent stocks of BL21. The symbol is a purple star. We later met with Bradford Condon, a graduate student who works with Professor Turgeon. He gave us a few tips on our method, including using young mycelia, and avoiding the hydrophobic disc that forms on the top of our cultures for protoplasting. The advice was that we should blend <I>G. lucidum</I> which has been growing for an extended period of time, and make new cultures for a short period of time with the fungal &quot;smoothie&quot;. We also miniprepped pC13BM, pC13BJ, pC13BL, and pC13BK 6 &amp; 7. The LB stock of pAK13AC was made with the wrong antibiotic so we remade it. We also remade LB stock of pAK13AB. We miniprepped pC13A, pC13D, pC13F, pC13K, pC13L, pC13M, pC13N, pAK13BB, pC13AS, pC13W, and pC13Y. We plated pC13A, pC13D, pC13F, pC13K, pC13L.</DIV><P>		The miniprep kit finally arrived so we were able to continue progress on our pC13BG, pC13AY and pC13AW constructs. To aid in our protoplasting, we decided to seek help from the Turgeon Lab. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 17th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Today, we made glycerol stocks of pC13AT and prepared them for sequencing. We then miniprepped pAK13AC. However, the miniprep qualities are poor so we will redo with a larger stock of cells. We ran a PCR of pAK13AC, pAK13BB, pAK13A (for both <I>RBS</I> + <I>crtE</I> and <I>crtE</I>). pAK13AC and pAK13BB worked but pAK13A failed. Thus, we retried PCR of pAK13A inserts from a fresh batch of minipreps. We prepared cultures of pC13S, pC13P, pC13Q, and pC13BG.</DIV><P><I>PtrpC</I> with <I>GFP</I> will be sent in for sequencing. We ran PCR of pAK13AC, pAK13BB, pAK13A (for both <I>rbs</I> + <I>crtE</I> and <I>crtE</I>). pAK13AC and pAK13BB worked but pAK13A both failed. We ran PCR of pAK13AC, pAK13BB, pAK13A (for both <I>rbs</I> + <I>crtE</I> and <I>crtE</I>). pAK13AC and pAK13BB worked but both pAK13A failed. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 18th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Today we miniprepped pC13AT along with pA13CZ. We also ran a gel of PCR products of pAK13AO (5 colonies) in pSB1C3. Insert lengths should be around .7kb, but no bands showed up. We will rerun a gel using more DNA from the same PCR. We ran digest screens of pC13BM, pC13BJ, pC13BL, and pC13BK minipreps 6 &amp; 7, which appear to have all worked. We miniprepped pAK13AB, pAK13AC, pAK13BB, pC13A and plasmid concentration was much higher this time around. We miniprepped pC13S, pC13P, pC13Q, and pC13BG, to prepare for cloning.</DIV><P>		Cloning of <I>T7 promoter</I> with <I>mRFP</I> has been problematic - but we will keep trying. Minipreps of large stocks of pAK13AB, pAK13AC and pAK13BB yielded better results! 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-anim"><DIV><H6>August 18
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><P>		Today, the dastardly trio of Arun, Hannah, and Eric were chosen to present at the regional iGEM competition! 
		-Swati
</P></DIV></DIV></DIV></DIV><DIV id="week10entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 10</H3><H5 class="centered">(08/19 - 08/25)</H5><DIV class="entry nb-wet"><DIV><H6>August 19th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We ran a PCR of the pC13BH minipreps to verify the success of the Gibson assembly on a gel. We also digested pC13AY in order to gel screen it and check if it is successful. We began constructing pA13AW by digesting and ligating pA13AM and pC13E. We also reran a gel of a colony PCR of pAK13AO. Next, we prepped pC13AT and pC13AQ for sequencing. We sent in pC13BJ, pC13BK, pC13BL, pC13BM minipreps 6 &amp; 7, pC13AS, pC13W, and pC13Y for sequencing. We ran a gel for the PCR products from Aug. 17 and Aug. 18. Everything is good: pAK13AB, pAK13AC, pAK13BB, pC13A (both <I>rbs</I> + <I>crtE</I> and <I>crtE</I>). We ran Q5 PCR of pC13N, pC13L, pC13S, pC13Q, and pC13W for creating inserts. We digested pC13S, pC13Q, and pC13W with X and P, and digested pC13BG, pC13D, pC13Y, and pC13P with S and P. We dephosphorylated and set up the ligations of those inserts and vectors to make pC13BQ, pC13BR, pC13CC, and pC13CD.</DIV><P>		We are beginning construction of <I>T7 polymerase</I> with an <I>Anderson promoter</I>, and prepared <I>PtrpC</I> with <I>mRFP</I> and <I>PtrpC</I> with <I>GFP</I> for sequencing. Gel electrophoresis showed that pAK13AB, pAK13AC, pAK13BB and pC13A (both <I>rbs</I> + <I>crtE</I> and <I>crtE</I>) are good! 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 20th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We transformed and plated the pA13AW ligation from yesterday. We also retried the PCR to append KpnI and BamHI cut sites to pC13F again, using the newly received Q5 mastermix and higher annealing temperatures. We ran a massive gel screen and found that the cut site PCR and Gibson assembly may have been successful, but that the construction pC13AY most likely failed. We then digested pC13F with cut sites appended with BamHI and KpnI  for ligation with pC13BE to make pC13BF. We digested pA13CZ with SpeI and PstI, which is to be ligated with <I>GFP</I> from pC13Z and <I>mRFP</I> from pC13AK. We ligated the Xpa1, Pst1, Spe1, Apa digested pC13AK and Spe1 and Pst1 digested and dephosphorylated pC13BG to make pC13CG, which we transformed into BL21-A1 cells to check transformation efficiency. We also tranformed into DH5α and plated both for comparison. The sequencing results of pC13BJ 7, pC13BK 6, pC13BL 7, and pC13BM 6 confirmed that we were successful, so we made new cultures to create glycerol stocks from, but the initial cultures were thrown out and the plates were difficult to re-pick from, so we hope that cells made it into the culture and have the correct plasmid. We did a PCR clean-up for pAK13AB, pAK13AC, pC13A, pC13A, pC13N, pC13L. We digested pC13I, pC13J, pC13N, pC13L with XbaI &amp; PstI, pAK13AB &amp; pAK13AC with EcoRI &amp; PstI. We also digested pC13F with XbaI and PstI, also with EcoRI and PstI, as well as pAK13D with XbaI and PstI. We transformed ligations of pC13BQ, pC13BR, pC13CC, and pC13CD into <I>E. coli</I>. We also interpreted sequencing results of pC13AS, pC13W, and pC13Y - the latter two look good, and pC13AS may have an overlap, suggesting a mixed population of plasmid.</DIV><P>		Cloning of <I>PtrpC</I> with <I>T7 polymerase</I> continues to be problematic. We are continuing cloning of <I>PpelA</I>, last time’s <I>T7 polymerase</I> with an <I>Anderson promoter</I>, and putting a strong RBS with <I>GFP</I> and <I>mRFP</I>. We digested pC13I, pC13J, pC13N, pC13L, pAK13AB, pAK13AC, pC13F and pAK13D in preparation for ligating new products for the carotenoid pathways. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 21st
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We made cultures of the successfully transformed pA13AW and ran a gel screen of the digests of pC13F and pC13BE and found that they did not have enough DNA to be visible. The <I>G. lucidum</I><I>PgpdA</I> and pC13F parts with homologous regions needed for Gibson assembly and found them successful. We then ligated and transformed pC13BF although it sparked during electroporation, so we will most likely have to retransform. A sequence confirmation came back for pC13AQ! pC13AT turned out to be <I>PtrpC</I> and <I>mRFP</I>.Thus, we redigested <I>PtrpC</I> in pC13P and pCg13Y with SpeI and Pst1-HF. pCg13Y will be used to create pC13CB. We also redigested pC13Z, pC13AK, and pC13AQ with Xba, Pst1-HF, ApaI and Sac1-HF in order to get necessary inserts. We threw away the cultures of <I>pC13BJ&quot; 7, </I>pC13BK&quot; 6, <I>pC13BL&quot; 7, and </I>pC13BM&quot; 6 from yesterday and didn’t make glycerol stocks because we were not sure if we got the correct colonies. We will transform the sequence confirmed minipreps when more DNA is needed. We attempted site directed mutagenesis of <I>PtrpC</I> + <I>hph</I> + <I>TtrpC</I> in  pAh13G. The extension time was a little short for pAh13G so it might not work. We ran gel a gel, which had a band at 100 base pairs,  meaning it didn’t work. We will attempt this again tomorrow with a longer extension time. Colonies of BL21-A1 and DH5α with pC13CG grew. BL21-A1 seems to have a higher transformation efficiency.We started cultures and performed a colony PCR from the DH5α plate. We ran a gel of the colony PCR and pC13CG 1, 3, and 4 had the correct band at about 1.8 kb. pC13A and pC13F (XbaI,PstI) did not have enough DNA. As a result, ligations for creating pC13I and pC13J are on hold. We ligated and electroporated pC13AF, pC13AG, pAK13CR, pAK13CS from other digestions, and plated them overnight. We found the the transformations of pC13BR and pC13BQ were too efficient and had to restreak the plates in order to isolate single colonies. We ran colony PCRs of pC13BU, pC13CC, and pC13CD, and at least two copies of each construct seem to be successful!</DIV><P>		Weird happenings in lab: what we thought was <I>PtrpC</I> with <I>GFP</I> turns out to be <I>PtrpC</I> with <I>mRFP</I>! Good news: <I>PgpdA</I> and <I>pC13F</I> with homologous regions for Gibson are good! We tested the transformation efficiency of our new T7-ready bacteria, BL21-A1 as well. We ligated and electroporated pC13AF, pC13AG, pAK13CR, pAK13CS from other digestions, and plated them overnight. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 22nd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We redid the Gibson assembly of pC13BH trying 1:2 and 1:6 vector to insert ratios and  transformed it along with pC13BF as it sparked when we tried yesterday. We also miniprepped our pA13AW cultures. After, we vector dephosphorylated pA13CZ, pC13P, and pCg13Y for fluorescent characterization constructs. We then ligated <I>PgpdA</I> with <I>GFP</I>, pC13AQ with pC13Y, pC13P and <I>mRFP</I>, pC13P with <I>GFP</I>, pA13CZ with <I>GFP</I>, and pA13CZ with <I>mRFP</I>. All ligations were then electroporated and incubated overnight. We sent pC13BX and pC13CJ to sequencing. We digested and ligated pC13CJ and pAK13D and electroporated into DH5a cells. We miniprepped and made glycerol stocks of pC13CG 1,3,4 (labeled TEMP). We will make sure to go back and throw out extras and failed attempts after sequencing results return. We sent pC13CG 3 in for sequencing, using the last of our DNA, meaning if we need more it will have to come from the glycerol stock. We performed our second attempt at site directed mutagenesis of pAh13G with a longer extension time. We recreated and digested inserts pC13A and pC13F (XbaI, PstI). We ligated pC13I and pC13J. pC13AF electroporation did not yield any colonies. We made glycerol stocks of pC13AG, pAK13CR, pAK13CS, pAK13BU, pC13CC, and pC13CD and put them at -80 degrees C. We miniprepped the successful pAK13BU, pC13CC, and pC13CD and prepared them for sequencing. We also ran colony PCRs of pC13BR and pC13BQ, and set up overnight cultures of the successful constructs.</DIV><P>		We did a bunch of ligations and are continuing cloning of <I>PgpdA</I>, <I>Anderson promoter</I> with <I>T7 Polymerase</I>, <I>GFP</I>, geneticin resistance, and a strong RBS. We ligated pC13I and pC13AF. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 23rd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Today we made liquid cultures of and ran colony PCRs of pA13DC, pA13DB, pC13AU, pC13AT, and pC13CB. pC13CF did not grow any colonies. We then ran a gel of the PCR products. pC13CB colony 5, pC13AT colonies 1-3 and 5, pC13DC colonies 2-5, pC13DB colonies 1-5, and pC13AU colonies 1 and 5 are looked like they had the proper insert lengths from the gel image. Sequencing results came back confirming that pC13CG 3 worked We digested our site directed mutagenesis attempt from yesterday with DpnI and stored it in the fridge. We ran a gel combined with other DNA from other team members, which resulted in a smeared band that was in the correct place. We believe this will be successful, and we will transform our site directed mutagenesis product as soon as desalting membranes arrive. We mini-prepped and plated pC13AG, pAK13CR, pAK13CS. We miniprepped pC13BQ and pC13BR, and submitted pC13CC, pC13CD, pC13BQ, and pC13BR for sequencing. We ran PCRs of the inserts in pAK13BB and pAK13BU to prepare for further cloning.</DIV><P>		Cloning of RBS + <I>GFP</I>, RBS + <I>mRFP</I>, <I>PT7</I> + <I>GFP</I> + <I>PtrpC</I> + geneticin resistance, <I>PtrpC</I> + <I>mRFP</I>, and <I>PtrpC</I> + <I>GFP</I> appear to be going well. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 24th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We cultured transformants for our Gibson assembly of pC13BH using 1:2 vector to insert ratios, but got no colonies for the 1:6 ratio Gibson and the pC13BF ligation. We also digested our pA13AW minipreps for gel screening. Our PCR confirmed that pC13AG and pAK13CR worked. We picked more colonies of pAK13CS from electroporation plate to screen more colonies.</DIV><P>		Running into some problems with <I>PgpdA</I> cloning. PCR confirmed that pC13AG and pAK13CR worked! We picked more colonies of pAK13CS for screening. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wiki"><DIV><H6>August 24th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><P>		I finished the sponsors page today. 
		-Nupur
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 25th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">The extra colonies for pAK13CS that we made yesterday all failed.  We will need to restart pAK13CS. We are recreating insert from pAK13AB for recreating pC13AF.</DIV><P>		The extra colonies of pAK13CS from yesterday failed. We will have to restart pAK13CS. We are also recreating inserts from pAK13AB to make pC13AF. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>August 25th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Christine's back! She started looking at the components for a humidity sensor and placed an order for a mistmaker which would be able to control humidity levels. In the meantime, Sara studied the details of the heating circuit which utilizes a transistor to amplify signals and switches it such that the resistor could be used as the heating element. 
		-Rebecca
</P></DIV></DIV><DIV class="entry nb-wiki"><DIV><H6>August 25th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Wondering why our bios are so awesome? Everyone picked partners and spent a long time writing the perfect bio for their teammate.</DIV><P>		Finished the team bios page today! 
		-Nupur
</P></DIV></DIV></DIV></DIV><DIV id="week11entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 11</H3><H5 class="centered">(08/26 - 09/01)</H5><DIV class="entry nb-wet"><DIV><H6>August 26th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">The transformed colonies of pC13BH had the <I>PgpdA</I> insert amplified via colony PCR and gel screened to check for the correct length. The gel however was blurry, so the cultures were miniprepped for digest screening. Gel screen pA13AW digested with XbaI and PstI. Gel results indicate that the cloning was successful! We then amplified the <I>T7 polymerase</I> gene for more insert DNA. We ran a PCR and digested pC13L to prepare insert for recreating pAK13CS. We recreated ligation of pC13AF. We looked at sequencing results for pC13BQ, pC13BR, pC13CC, and pC13CD - all looked good, other than some minor differences between our copies of the trpC promoter and terminators and the sequences in the online database.</DIV><P>		Cloning of pA13AW seems to be successful and will test to see if pC13BH also worked. We ran a PCR and digested pC13L to prepare insert for recreating pAK13CS. We recreated ligation of pC13AF. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>August 26th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Our materials are here and everyone in the fabrication group has returned to Ithaca!  This week, Manny and Mac focused their efforts on getting the plastic cut to the proper dimensions.  Unfortunately, this proved more to be more difficult than anyone could have imagined.  After transporting a rather hefty 48&quot;x48&quot; piece of plastic across campus from Weill Hall to the Emerson Lab machine shop, we were told that we would have to take it elsewhere to have it cut (plastic is bad for blades).  Disappointed, but never defeated, we decided to try our luck with another machine shop.  Also, carrying plastic around campus isn't much fun, so we held it on the roof of a car instead (not recommended).  Fortunately, no traffic violations were received, and the Clark hall machine shop was more than happy to cut up our plastic for us. 
		-Mac
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 27th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We made cultures of pC13BX colony 1 and pC13CJ colony 4 to make glycerol stocks. We ran PCR of <I>lox</I> sites to amplify by Gibson. We ligated pAK13D with limonene. We preppred <I>cre</I> for reverse sequencing. We transformed <I>holin</I> off of kit plate and plated it. Digested pC13BH preps using EcoRI and PstI and screened alongside <I>T7 polymerase</I>. Results indicate that only the <I>T7</I> PCR worked. The backbone pC13F and pA13AM were digested with XbaI and PstI, retrieving the <I>T7 polymerase</I> gene from the latter. Gel results for the digestions revealed correct banding patterns only for pC13F. We also miniprepped and prepared for sequencing pC13AU, pC13AT, pC13CB, pA13DC, and pA13DB. We prepared insert from pAK13CR for creating pC13CV. We submitted pC13AG for sequencing. We ran PCRs of pAK13BB and pAK13BU, and digested the inserts with EcoRI and PstI. We also digested pC13CD with SpeI and PstI.</DIV><P>		Digest screens indicate that cloning pC13F and the <I>T7 polymerase</I> PCR worked. We are working on the <I>cre</I> and <I>lox</I> constructs. pC13AU, pC13AT, pC13CB, pA13DC, pC13AG, and pA13DB were miniprepped and prepared for sequencing. We prepared insert from pAK13CR for creating pC13CV. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 28th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">One colony of pC13BH had a good banding pattern and was submitted for sequencing. Digested <I>T7 polymerase</I> PCR with XbaI and PstI and pC13F with BamHI and KpnI (sites were appended via PCR). pC13F that was digested with XbaI and PstI as well as with BamHI and KpnI were dephosphorylated for ligation. After confirming <I>T7 polymerase</I> PCR on a gel, it was ligated into pA13AM and pC13P. Additionally, <I>Pc + T7 polymerase</I> was ligated into pC13F and <I>PpelA</I> was ligated into pC13F. We ran those PCR's for the Gibson of pC13BN on a gel.  <I>lox</I> 2 worked, but <I>lox</I> 1 did not, so I purified <I>lox</I> 2 and reran the PCR of <I>lox</I> 1 (it is in the small fridge labeled <I>lox</I> 1).  I also made glycerol stocks of pC13BX and pC13CJ so that we have those in case we need to do more minipreps on them. We also had transformants for pC13CN so I made cultures with spectinomycin for all of them and ran a colony PCR on them.  The colony PCR did not work due to incorrect protocol. We do not have the correct primers to do a PCR on this plasmid because its in Bbb format, so we just have to assume the plasmid is present because the cultures are growing in Spec. Tomorrow we need to miniprep those cultures that are in the incubator. We blended <I>G. lucidum</I> and made two 100mL cultures with 75mL cells and 25 mL cells. We will now let these grow for 2 days before attempting protoplasting once more. We created large batches of vectors (pC13F (XbaI,PstI) and (EcoRI-HF,PstI), pC13D (XbaI,PstI)). pC13AG sequence is sequence confirmed! We tried to dephosphorylate, but mistakenly used DNA Ligase on pC13F and pC13Q instead, so we’ll have to redo that tomorrow.</DIV><P>		Sequenced pC13BH and ligated <I>T7 polymerase</I> into a variety of vectors. <I>lox</I> 2 worked with Gibson, but <I>lox</I> 1 did not. We created large batch of vectors pC13F and pC13D. The sequence of pC13AG is confirmed! 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 29th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Today we electroporated <I>Pc + T7 polymerase</I> in pC13F, <I>PpelA</I> in pC13F, and <I>T7 polymerase</I> in pA13AM and pC13P. We prepared pAK13AB and pC13AG for glycerol stock. We ligated pAK13CS and pC13CV. We electroporated and plated pC13I, pC13J, pAK13CS, pC13CV. pC13J arced, so we ran a new Q5 PCR to begin preparation of insert from pC13A. We re-digested pC13Q with XbaI and PstI and pC13F with EcoRI and PstI, and subsequently dephosphorylated pC13CD and pC13F. We set up ligations to create pC13BC, pC13BV, and pC13CE.</DIV><P>		Transformed our <I>T7 polymerase</I> ligations. We ligated pAK13CS and pC13CV. We electroporated and plated pC13I, pC13J, pAK13CS and pC13CV -- pC13J unfortunately arced… 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 30th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Set up cultures for <I>Pc + T7 polymerase</I> in pC13F, <I>PpelA</I> in pC13F, and <I>T7 polymerase</I> in pA13AM and pC13P and ran a colony PCR for each. Each PCR was visualized on a gel; only <I>PpelA</I> in pC13F appeared to work. We quantified DNA: <I>lox</I> 1: 72ng/uL, <I>lox</I> 2: 34ng/uL, pC13CN 640.9 &amp; 820.9ng/uL. We ran Gibson on the <I>lox</I> PCRs, transformed them into DH5a cells. We made glycerol stocks of pC13CN. We have pC13CK transformants, ran colony PCR and made cultures. We got reverse sequencing on cre and it looks good. We inoculated a culture of DH5a for electrocompetent stocks. pC13AU and pC13AT was sequence confirmed today. However, for pC13CB, <I>PtrpC</I> and <I>nptII</I> BB re-ligated back to itself pA13DB colony 1 and pA13DB colony 4, pA13DC colony 3 and pA13DC colony 4 all failed to sequence. We discarded all of our pC13CG glycerol stocks except for the successful pC13CG 3, and met with members of Professor Turgeon's lab. We also blended our 25 mL culture from 2 days prior, and made new 100ml cultures with (75 mL and 25 mL, again.) We prepared pAK13AB and pC13AG for glycerol stock. We made glycerol stock of pAK13AB and pC13AG. We redid a PCR with Phusion polymerase in an attempt to amplify <I>rbs</I> + <I>crtE</I> (pC13J). We then made overnight E. Coli cultures of pC13I, pC13AF and pC13CV. We transformed the ligations of pC13BC, pC13BV, and pC13CE into <I>E. coli</I>, but both attempts for pC13CE arced, so we started another ligation for pC13CE.</DIV><P>		Cultures from yesterday’s <I>T7 polymerase</I> ligation transformations were made. We transformed the <I>lox</I> Gibson products. pC13AU and pC13AT were successfully sequenced today. We did a PCR with Phusion polymerase in an attempt to amplify the insert in pC13J. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 31st
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We ran colony PCR of pAK13CK on gel and nothing appeared. This could be because we used a janky Vent protocol. We miniprepped liquid cultures of pC13CK. A digest screen was run on the minipreps, but they did not look good. We will redo cloning for the construct. In other news, we got pC13BN transformants! We ran a colony PCR and grew up liquid cultures of pC13BN. The colony PCR looks good, and we will miniprep liquid cultures of pC13BN tomorrow. We made electrocompetent DH5a stocks, which are marked with blue circles with a line through it. Lastly, we double digested pC13Z and pC13AK with Xba, PstI, Sac, and ApaLI since we were running low on insert DNA. The gel for we ran for the double digestion looked good since we got 4 distinct bands for <I>GFP</I> and <I>mRFP</I> each. Next, we column purified pC13Z and pC13AK double digests and stored them in the freezer for later use. Today we attempted the first day of protoplasting and transforming <I>G. lucidum</I> in the Turgeon Lab. We used both our 3 day old and 1 day old cultures of <I>G. lucidum</I> in this attempt. Our initial pellet of both cultures appeared about the same size, though the younger one may have been slightly larger. We observed that the cells didn’t look like protoplasts, as they weren’t round and were smaller/very few in number. We looked at the solution after filtering and were able to see protoplasts, so it appears that something didn’t work with buffer/centrifuging. We miniprepped and made glycerol stock of the overnight cultures from yesterday--pC13I, pC13AF and pC13CV. We then digested these products. We electroporated for pAK13CS and plated the culture on Ampicillin plates. In addition, the PCR of pC13J worked! The transformations for pC13BC and pC13BV were too efficient, so we had to restreak them out to single colonies.</DIV><P>		We are continuing cloning of the carotenoid genes and <I>lox</I> sites. The pC13BN construct looks good. We double digested pC13Z and pC13AK with Xba, PstI, Sac, and ApaLI since we were running low on insert DNA for <I>GFP</I> and <I>mRFP</I>. We began a new attempt at protoplasting, now with the help of members of the Turgeon Lab. We digested the following miniprep products: pC13I, pC13AF and pC13CV. We also electroporated pAK13CS. The PCR of pC13J from yesterday is good! 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 1st
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We ran a PCR for amplification of pC13Y, pC13BQ, pC13BR, and pC13CD to put between the <I>lox</I> sites. The PCR products were then cleaned and quantified. We redid the ligation of pC13D with pC13CJ to form pC13CK. We electroporated the ligation into DH5a. We miniprepped the pC13BN 2 cultures. We then ran a Vent colony PCR of pC13CB colonies 1-5. We also grew up 5ml cultures of pC13CB colonies 1-5 and left them in the incubator overnight. We finished our attempt at protoplasting <I>G. lucidum</I>  with the help of members of the Turgeon lab, with hygromycin as our selection marker, and we left the plates in the Turgeon lab incubator.  We then combined our three blended cultures and blended again to make a new culture for tomorrow. We planned out a protoplasting schedule for this week to try running the protocol under different conditions since it takes a long time to get results. We believe there might be bacteria growing in the fungal cultures we are using for protoplasting since phosphinothricin does nothing to bacteria and we could not blend the fungi under sterile conditions. We will put ampicillin in later cultures to kill bacteria, which shouldn’t affect <I>G. lucidum</I> because only bacteria will have peptidoglycan cell walls. We did a PCR clean up for pC13A. We ran a digest screen for pC13I, pC13AF and pC13CV, but only pC13AF (colony #2) appeared to work. We then ran a PCR with Vent for pC13I, pC13AF, pC13CV and pC13L, but they all failed…</DIV><P>		We are putting <I>PgpdA</I> with geneticin resistance, <I>PtrpC</I> with geneticin resistance, and <I>PtrpC</I> with hygromycin resistance between the <I>lox</I> sites. We miniprepped pC13BN. A Vent colony PCR of pC13CB colonies 1-5 was run to confirm successful cloning. We finished our attempt at protoplasting, once again unsuccessful, likely due to contamination on the fungal cultures. We ran a digest screen for pC13I, pC13AF and pC13CV, but only pC13AF (colony #2) appeared to work. We then ran a PCR with Vent for pC13I, pC13AF, pC13CV and pC13L, but they all failed. We transformed the new ligation for <I>pC13CE</I> into <I>E. coli</I>. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>September 1st
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Lydia's back too! Discussed what we have been doing for the past few weeks and immediate goals. 
		-Rebecca
</P></DIV></DIV></DIV></DIV><DIV id="week12entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 12</H3><H5 class="centered">(09/02 - 09/08)</H5><DIV class="entry nb-wet"><DIV><H6>September 2nd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We got colonies for pAK13CK and ran a colony PCR. The PCR didn’t show anything, but the primers are not ideal, so we will miniprep and digest screen. We digested PCRs of pC13BR, pC13BQ, pC13CD, and pC13Y with KpnI, HindIII. They were all cleaned. We digested miniprep of pC13BN with KpnI and HindIII. This was cleaned and dephosphorylated. We prepped pC13BN miniprep for sequencing with primer 31. Miniprepped pC13AY, pC13BF, pA13AM, and pA13AW cultures and will quantify tomorrow. We prepared pA13DC colonies 3 and 4 and pA13DB colonies 1 and 4 for sequencing Tuesday morning. We then reran a Vent PCR for pC13CB and grew up liquid cultures of the same colonies. We ran a gel for pC13A which showed that the PCR failed. We then redid a Vent PCR with the addition of MgSO4 for pC13AF, pC13CV, pC13I and minipreps of pC13L. We made 5 pAh13G cultures from glycerol stocks, and we electroporated pC13BJ, pC13BK, pC13BL, and pC13BM to make glycerol stocks. All transformations except pC13BM arced, but we are going to plate them anyway. We also attempted protoplasting again, using fresh enzyme osmoticum and STC. We couldn't really see mycelium at the beginning, and we could not see cells when we attempted to count them, possibly due to our mycelium being too old or having depleted its media. We made a new culture with a blended cell mass and ampicillin. We also made more CYM plates and media, and prepared a plan to test the sensitivity of <I>G. lucidum</I> to antibiotics and antifungals.</DIV><P>		Miniprepped <I>PtrpC</I> + <I>T7 polymerase</I>, the <I>PpelA</I>, the <I>Anderson promoter</I>, and the <I>Anderson promoter</I> + <I>T7 polymerase</I>. We are preparing to send pC13BN to sequencing. A Vent PCR was done on pC13CB to verify successful cloning. We began a new transformation attempt, now with fresh solutions to work with. However, this also seemed unsuccessful. We also ran a Vent PCR with the addition of MgSO4 for pC13AF, pC13CV, pC13I and pC13L minipreps--they all failed. Another Vent PCR was run to check inserts in pC13BC, pC13BV, and pC13CE, but resulted in no bands. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 3rd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We quantified digestions and ligated pC13BQ + pC13BN, pC13Y + pC13BN, pC13BR + pC13BN, and pC13CD + pC13BN. We electroporated the ligations. We miniprepped pC13CK cultures and ran digest screen, which was successful. The construct is correct. We ran a PCR to append cut sites to pSB1C3 for digestion and ligation with <I>holin</I>, which was unsuccessful. The pC13AY, pA13AM, and pA13AW preps were digested with EcoRI and PstI and run on a gel. Results indicate that pC13AY and some of the pA13AW colonies worked. The gel of Vent PCR of pC13CB did not turn out too well. We got a bunch of lanes with smeared rather than distinct bands. We then ligated and transformed <I>T7 promoter</I> with <I>mRFP</I> to obtain pAK13AO and <I>PgpdA</I> with <I>GFP</I> to create pC13CF. We ran a Q5 PCR and gel for pC13AF, pC13CV, pC13I and pC13L. Sadly, they all failed except pC13AF from colony #2. We are concerned with what’s going on. The PCR for pC13A failed again too. We miniprepped our pAh13G cultures. Our new fungal culture doesn’t have nearly as much mass in it, leading us to believe that the old cultures were almost definitely contaminated with bacteria. Our transformed pC13BK and pC13BM grew into masses covering the plates rather than colonies, and the others didn’t grow. We restreaked pC13BK and pC13BM hoping for single colonies. We attempted a diagnostic protoplasting, determining that pellets formed are likely from contaminations, as the new cultures with ampicillin did not produce pellets. We also tried to manually extract mycelium and enzyme digest it, but microscope visualization did not show signs of protoplasting having occurred. Even after filtering through one layer of cheesecloth and nylon, there was no appearance of protoplasts.</DIV><P>		Digest screen of <I>PtrpC</I> + <I>T7 polymerase</I> and the <I>Anderson promoter</I> + <I>T7 polymerase</I> showed good expected banding pattern. The pC13CK construct is correct. A gel of pC13CB PCR products was run and was inconclusive. We repeated the PCR from yesterday for pC13AF, pC13CV, pC13I and pC13L but using the Q5 polymerase. Except for pC13AF from colony #2, all else failed. We attempted to protoplast again, this time with emphasis on determining what our issue is in hopes that we can fix it. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>September 3rd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Christine, Rebecca and Sara met up with Dr. Land again and we learnt more about setting up PID controls. Went to the lab next where Christine and Sara set up the heating element circuit, Rebecca soldered pins onto the arduino and Lydia drew up the circuit design. 
		-Rebecca
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 4th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We got transformants for all <I>lox</I> constructs. We got sequencing back for pC13BN and it was incorrect, so we will redo the Gibson transformation and all the subsequent <I>lox</I> constructs. We also electroporated pC13BF and pC13AZ and digested pC13F and pA13AW with XbaI and PstI for ligation to make pC13AZ. pC13F and pC13BE KpnI and BamHI digests were then ligated to make pC13BH. pA13DB colony 1 and pA13DC colony 4 were sequence confirmed today. Lastly, we made cultures from restreaked pC13BM and pC13BK plates.</DIV><P>		Transformed the <I>PpelA</I> and <I>Anderson promoter</I> + <I>T7 polymerase</I> into pSB1C3 as well as digest and ligate other promoters. The pC13BN construct was incorrect, so we will have to redo all of the Gibson transformations for the <I>lox</I> constructs. On a brighter note, sequence confirmation of pA13DB and pA13DC came back today! 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 5th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">The Gibson was digested with DpnI and electroporated into DH5a. We are growing DH5a for electrocompetent stocks. We did a PCR of pSB1C3 to append cut sites in order to ligate with <I>holin</I>. We digested pC13Y, pC13BQ, pC13BR, and pC13CN for ligation with pC13BN and made cultures of three pC13AZ colonies. Gel screen of pC13F and pA13AW appeared successful and were then column purified. We also gel screened pC13P miniprep, SpeI and PstI digested pC13P, PCR of <I>PgpdA</I> for homologous regions, PCR of pC13F for homologous regions, and BamHI/KpnI digested pA13BE to verify the viability of old minipreps. We then amplified <I>T7 polymerase</I> from pC13E. We did a Q5 PCR for pC13CV, pC13I and pC13L. Our plates from transformation on Saturday showed conidia growth on the controls with no hygromycin. None of the plates with hygromycin showed any growth. We decided that we will now use <I>Cochliobolus</I> to protoplast and transform after meeting with Professor Turgeon again. To determine if the DNA is in the fungal genome and stable, we will plate on non-antibiotic and antibiotic plates, then replate transformants from the antibiotic plate onto non-antibiotic plates and back onto antibiotic plates. Since protoplasting is failing, we will turn to agrobacterium for <I>G. lucidum</I> transformation in the future. We will also check for homology between the promoter gpd in <I>G. lucidum</I> and <I>Cochliobolus</I> to make sure we are using DNA that can be expressed. Our new strategy will be to insert genes after <I>Cochliobolus</I> gpd by cutting into that region in the genomic promoter to transform. We also created glycerol stocks of pC13BM and pC13BK cultures and miniprepped them to submit for sequencing. We set up new cultures of colonies from transformation of pC13BC, pC13BV, and pC13CE, and ran colony PCRs of them (using our brand new tube of Q5 polymerase!) to check for inserts.</DIV><P>		Today in lab we gel screened a bunch of old minipreps to see if they were still viable and made cultures of the <I>Anderson promoter</I> + <I>T7 polymerase</I> in pSB1C3. We digested the <I>lox</I> site constructs in order to ligate with pC13BN. In a meeting with Professor Turgeon we decided that we would leave <I>G. lucidum</I> behind for now and work with <I>Cochliobolus</I>, which her lab transforms regularly and has more expertise with. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 6th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We digested, ligated, and transformed to create pC13DD. We ran a PCR to append cut sites to pSB1C3 for digestion and ligation with <I>holin</I>. Gel results indicate that pC13P miniprep, pC13P SpeI/PstI digests, <I>PgpdA</I> PCR for Gibson and the pA13BE BamHI/KpnI/SacI digest are not viable, but the pC13F PCR for Gibson might be okay. pC13BF transformation failed as well. The pC13AZ cultures were digest screened; results were good, so they were sent for sequencing. pC13P was plated and pC13E was cultured. We submitted pC13AF (colony #2) for sequencing. The Q5 PCR from yesterday--pC13CV, pC13I and pC13L--was a failure. We suspected that something is wrong with our reagents. Use the new dNTPs from now on! We ran PCRs to prepare the necessary parts for our Gibson assembly to produce parts with flanking <I>G. lucidum</I> homologous regions. We spun down the cultures of pC13BC, pC13BV, and pC13CE and stored them in the fridge.</DIV><P>		Most of old minipreps were no good and the <I>PpelA</I> transformation failed. The <I>Anderson promoter+T7 polymerase</I> gel results looked good and were submitted for sequencing. Oh no, the PCR of pC13CV, pC13I and pC13L failed again--we suspect something is wrong with our reagents. Use the new dNTPs from now on! 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-anim"><DIV><H6>September 6th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><P>		We met at Flora Rose house to decide on an outline of the presentation and also decide a timeline.We met for the first time to discuss a preliminary outline for the presentation.  Hannah will be handling all the human practices elements, Eric will be explaining relevant biological background and strategies, and Arun will be presenting all of our characterization data (hopefully!). We made a detailed structure which will include an introduction about styrofoam pollution and ecovative. Then we will move on to a brief overview of our wetlab work. It was clear that the wetlab portion should be divided into 3 main components. The components would be fungal toolkit, carotenoid pathway, and antifungals. Finally, we will end with results, human practices, and acknowledgments to all our beautiful advisors and sponsors. 
		-Arun
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 7th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We did Q5 PCR to amplify pSB1C3 for Gibson with <I>afp1</I> and <I>Cht1_2</I>. 3uL of PCR product was run on the gel, found to correct, and was cleaned. We ran a Gibson of pSB1C3 with both pC13CL and pC13BA. Both of these were transformed, <I>Cht1_2</I> into DH5a (time constant 4.82) and <I>afp1</I> into BL21 (time constant 4.84). We made glycerol stock of pC13BN. We ran a gel of pSB1C3 PCR from yesterday, which was successful, and performed a PCR clean up. We digested pSB1C3 and pC13CN with HindIII and BglII. We ligated these together to create pC13CO and transformed it into DH5a (time constant 4.82). We miniprepped pC13BN culture and did a double digest on pC13BN culture with KpnI and HindIII to see if internal cut sites were present. There was some digestion that occurred when it was run on a gel, so we are confident the sequence is correct. We double digested pC13BN again with KpnI and HindIII. We ligated it with pC13Y (time constant 4.86), pC13BR (4.86), pC13BQ (4.90), pC13CD (4.80). We got transformants for pC13DD. A colony PCR was run and pC13DD colony 3 looked good. This colony was grown in liquid culture for miniprepping and sequencing. We grew cultures of pC13BX, pC13CK, and pC13BN for submission (pC13BX submitted directly; pC13CK and pC13BN need sequence confirmation first). pC13BF transformants were cultured, pC13AZ cultures were digested with EcoRI/PstI and gel screened; results were good, so they were submitted for sequencing. Digested pK13AL and pA13AW with XbaI/PstI for ligation to make pK13AX. pC13P and pC13E were digested with EcoRI/PstI and ligated to make pC13AY. Cultures of pC13Q and pC13BE were made from glycerol stocks. We electroporated pC13AU, pC13AT, pAK13CY, and pC13AQ. We also ran a Q5 PCR of pC13CB and pAK13AO using standard BioBrick primers 23 and 24 in order to confirm lengths of inserts. We redid the Q5 PCR with the new dNTPs for pC13CV, pC13I, pC13AF  and pC13L--pC13CV, pC13I and pC13AF were bad, but pC13L was good! We suspect that electroporation did not go well to begin with. We reran the 2-step Q5 PCR for pC13A. We ran a gel of our PCR to create Gibson assembly parts, and it appeared unsuccessful. We ran the colony PCRs of pC13BC, pC13BV, and pC13CE on a gel, and only observed bands for a few pC13BC colonies.</DIV><P>		Digested and ligated the <I>Anderson promoter+T7 polymerase</I> into pSB1C3 and pSB2K3 as well as start cultures from glycerol stocks and transformation and gel screened the <I>Anderson promoter</I> + <I>T7 polymerase</I>. We transformed <I>afp1</I> and <I>Cht1_2</I>. For fluorescence, we electroporated pC13AU, pC13AT, pAK13CY, and pC13AQ to obtain more plasmid DNA. We redid a Q5 PCR for pC13CV, pC13I, pC13AF and pC13L with the new dNTPs--PCR products were all bad except for pC13L. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-hprac"><DIV><H6>September 7th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Five team members went to the Ithaca Sciencenter and taught a group of 18 children and 17 parents about the power of DNA. 
		-Hannah
</P></DIV></DIV><DIV class="entry nb-anim"><DIV><H6>September 7th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><P>		After researching we found out that the powerpoint presentation should be centered around design and images rather than content. We decided to meet tomorrow to discuss possible design layouts and schemes. We looked at many examples of presentations and considering our project, we think we should have a very modern and green theme. Arun will explore with this later on. 
		-Arun
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 8th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We got transformants from the <I>afp1</I> and <I>Cht1_2</I> Gibsons. There were a lot of red colonies and very few white ones, and an attempted pick and streak of the white colonies was performed. We got transformants for pC13CO, pC13BN + pC13CD, pC13BN + pC13Y, pC13BN + pC13BQ, and pC13BN + pC13BR. A colony PCR was performed, and a gel of the PCR products looked good for only pC13CO colony 2. No positive inserts were present, so maybe the <I>lox</I> construct is not correct. To check, we will sequence it tomorrow. Afterwards, we miniprepped pC13BF, pC13P, and pC13Q and digested pC13BF. We ran the gel for pC13A and did a PCR clean up for it, as well as for pC13J and pC13L. We also digested pSB1C3 with XbaI and PstI, and miniprepped pC13CV, pC13I, pC13AH and pC13AG. We attempted to run another gel of our PCR product to create Gibson assembly parts, which looked even worse than yesterday. We also did a PCR cleanup of pC13BJ, pC13BK, pC13BL, and pC13BM.</DIV><P>		We miniprepped <I>PpelA</I>, <I>PtrpC</I>, and <I>TtrpC</I>. We got transformants with <I>afp1</I> and <I>Cht1_2</I> and for all of the <I>lox</I> site constructs. However, only the pC13CO looked good based on a gel screen. We will sequence the <I>lox</I> construct. We also miniprepped pC13CV, pC13I, pC13AH and pC13AG. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-anim"><DIV><H6>September 8th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><P>		We met super early in the morning (10 am!) for brunch and some delicious cake.  We also got work done and made an entire storyboard for the presentation.  Our golden (or perhaps orange) idea of the day was a growing carrot representation for the carotenoid pathway. 
		-Eric, Arun
</P></DIV></DIV></DIV></DIV><DIV id="week13entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 13</H3><H5 class="centered">(09/09 - 09/15)</H5><DIV class="entry nb-wet"><DIV><H6>September 9th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We ran a digest screen on pC13F with KpnI (mRFP has 2 internal KpnI cut sites). No digestion occurred since the enzyme was degraded. We submitted sequencing for pC13CK and pC13BN. We miniprepped pC13CO, pC13BA, pC13CL cultures and need to quantify and prep for sequencing. We ran a colony PCR on streaked pC13BA and pC13C plates and got a definite correct insert on pC13BA, and a lot of nonspecific amplification on pC13CL. We will run a digest screen and sequence. pC13BF colonies were quantified and gel screened and <I>PgpdA</I> was amplified from <I>G. lucidum</I> DNA, digested, and gel screened. The results were good, thus it will be Gibson assembled. We redid the PCR for pC13A with new materials but it unfortunately failed again. We also digested pC13L with XbaI and PstI and dephosphorylated the pSB1C3 digest from yesterday. We made an overnight culture of <I>Cochliobolus</I> in the Turgeon lab, as well as preparing fresh solutions of enzyme osmoticum and regeneration medium. We miniprepped the cultures of pC13BC, pC13BV, and pC13CE from the weekend. We added antibiotics to CYM plates, and intended to inoculate them for the growth experiment for testing <I>G. lucidum</I>’s antibiotic sensitivity, but the plates were too wet for reliable inoculation (that would spread in a circle of constant diameter), and inoculation was postponed so the plates could dry.</DIV><P>		PpelA preps were quantified and the <I>G. lucidum</I> promoter was amplified and gel screened. We are preparing our pC13CO, pC13BA, and pC13CL cultures to sequence. The correct insert is in pC13BA based on a gel screen. We digested pC13L and dephosphorylated the pSB1C3 digest from yesterday. We prepared an overnight culture of <I>Cochliobolus</I> and necessary solutions for a protoplasting attempt tomorrow. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>September 9th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		After picking up our pieces from the Clark Hall machine shop, we focused on cutting our insulation to the proper dimensions.  Initially, we tried cutting it with a razor blade, but after some rather sloppy results, we decided we needed a more efficient cutting method.  Our next attempt involved heating a thin wire and melting through the insulation.  This produced better results than the razor blade, but was way too slow.  Finally, we decided it would be best if we just resorted to power tools.  We bought a jigsaw from Home Depot and cut the insulation like it was cheese.  After, we gazed fondly upon the jigsaw and reminisced about high school woodshop. 
		-Mac
</P></DIV></DIV><DIV class="entry nb-anim"><DIV><H6>September 9th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><P>		We met with our design master Oat to discuss design considerations for the presentation. Oat will be making many designs, including a kill switch diagram, homologous recombination figure, our carotenoid image, a toolkit representation, and of course lots of mushrooms. 
		-Eric
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 10th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We prepared pC13CO, pC13CL, pC13BA for sequencing. We made glycerol stocks of pC13CL and pC13BA. We got a sequence confirmation  for pC13CK. We got sequencing for pC13BN and had a portion of the lac operon inside the prefix and suffix...a weird recombination must have occurred. pC13AZ sequencing results were good! That and pA13AM and pA13AW were miniprepped for glycerol stocks and biobrick submission. A PCR to add homologous regions to pC13F for Gibson assembly was done. The gel of Q5 PCR of pAK13AO colonies 1,2,3 and pC13CB 1,3,4 all had correct lengths. Thus, we miniprepped the colonies with correct plasmid insert lengths and submitted them for sequencing. We ran a PCR and electroporation for pC13CV  and pC13I, which turned out to be bad. We made overnight E. Coli culture for pC13AF (colony #2). An overnight ligation was done for <I>rbs</I> + <I>crtI</I> (from pC13L) at 4C. We ran a PCR to check the inserts in pC13A, pC13I and pC13J. We protoplasted and transformed <I>Cochliobolus</I> with Dongliang, a post-doctoral research associate in the Turgeon Lab. We used our constructs of pAh13G / pC13BR linearized with SphI, as well as the highly efficient <I>Cochliobolus</I> vector PNG from the Turgeon lab as a &quot;positive&quot; control. We submitted pC13BM 1 and pC13BK 1 for sequencing. We homogenized a dilute culture of <I>G. lucidum</I> and inoculated each of the plates for the antibiotic sensitivity test with 20 μL aliquots, without spreading.</DIV><P><I>Anderson promoter</I> + <I>T7 polymerase</I>, in pSB1C3 and BBa_J61002 were miniprepped, glycerol stocked, and prepared for biobrick submission. For fluorescent constructs, a Q5 PCR of pAK13AO colonies 1,2,3 and pC13CB 1,3,4 all had correct lengths. The pC13BN construct was incorrect. An overnight culture was prepared for <I>rbs</I> + <I>crtI</I> (from pC13L) at 4C. We made our first attempt at protoplasting and transforming <I>Cochliobolus</I> with the help of the Turgeon Lab. We set up cultures of pC13X and pC13AV in preparation for part submission. We started the growth experiment for testing <I>G. lucidum</I>’s antibiotic sensitivity. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-anim"><DIV><H6>September 10th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><P>		Oat, Eric, Arun, and Hannah sat down to make figures for the presentation. Arun also chose his first color scheme: a baby blue flanked by a serene green.  We got many designs done including the carotenoids, homologous regions, kill-switch, and the construct pieces. 
		-Arun, Eric
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 11th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We submitted pC13CO, pC13CL, and pC13BA for sequencing. After, we miniprepped and made stocks of pC13AY and pC13AZ. Also, we submitted pA13AW for sequencing. Transformation of pK13AX was unsuccessful because the wrong resistance plate was used. pC13BF and pK13AX were transformed today. We miniprepped the overnight culture of pC13AF. Gel electroporation showed that pC13I samples are good, but none of the pC13J samples are positive. We ran a PCR for the inserts in pAK13CR. We were surprised to know that <I>rbs</I> + <I>crtE</I> (pC13J) already exists! We made 1000x hygromycin and 100x geniticin. We checked plates from our protoplasting and transforming attempt under the microscope. It appeared unsuccessful so next time we will use all the protoplasts, at least 300 µL. We plated a layer of cym agar and the respective antibiotic on top of the regeneration agar gel as well as inoculating 4 new plates of <I>Cochliobolus</I>. We also ran our Gibson assembly of homology regions in <I>G. lucidum</I>. We miniprepped pC13X and pC13AV in anticipation of part submission, and ran a PCR (to check for inserts) from minipreps of pC13BC, pC13BV, and pC13CE, as well as pC13X and pC13AV (just in case).</DIV><P>		Redid transformations of <I>PpelA</I> and the <I>Anderson promoter+T7 polymerase</I>. We sent the <I>Anderson promoter</I>, pC13CO, pC13CL, and pC13BA to sequencing. Gel electroporation for pC13I showed that it was good, but nothing for pC13J. We were surprised to know that <I>rbs</I> + <I>crtE</I> (pC13J) already exists in iGEM! We continued with protoplasting <I>Cochliobolus</I> in the Turgeon Lab, and we ran a Gibson assembly of homology regions in <I>G. lucidum</I>. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 12th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We prepped pC13BX and pC13BA for BioBrick submission and got sequencing back. pC13BA was correct, and pC13CO and pC13CL were incorrect. We redid the Gibson for pC13CL and transformed it. We redid pC13CO digestions, ligation, and transformations. We transformed the redone <I>lox</I> Gibson from yesterday. We streaked out CYM plates with <I>Aspergillus Niger</I>, ran the Gibson assembly for <I>PgpdA</I> in pC13F, and transformed. We also digested pC13BR with SpeI and PstI and digested pC13AT with Xba and PstI, ran a gel of digestions to verify lengths, and column purified the inserts from digestion. We did an electroporation with pC13J and it was successful! We ran a gel for pAK13CR for confirmation and then we did a PCR clean up. We digested pAK13CR with EcoRI-HF and PstI, pC13I with XbaI and PstI, and then we ligated <I>rbs</I> + <I>crtB</I> (from pAK13CR) into pSB1C3, <I>crtE</I> (from pC13I) into pSB1C3. We prepared pC13BJ pC13BK pC13BL pC13BM and pC13BR for Biobrick submission. It seems our Gibson assemblies from yesterday were left in the water bath all night, so we reran them (pC13BK + pC13BM + pC13CD, pC13BQ, or pC13Y) before we transformed and plated the new assembled plasmids. We made a culture of pC13CG from glycerol stock. We ran a gel of the PCRs of pC13BC, pC13BV, pC13CE, pC13X, and pC13AV - there were correct bands for at least two of everything, and successful pC13BC, pC13BV, and pC13CE constructs were prepared for sequencing. We also set up a culture of pC13BQ in anticipation of part submission.</DIV><P>		Ran Gibson assembly for the <I>G. lucidum</I> promoter into pSB1C3. pC13BA is now a BioBrick! We also began cloning a construct with both fluorescence and resistance genes by ligating pC13BR and pC13AT. We successfully electroporated <I>E.Coli</I> with pC13J! We digested pAK13CR, and then we ligated <I>rbs</I> + <I>crtB</I> into pSB1C3, <I>crtE</I> into pSB1C3. After finding our Gibson assembly from last night left in the water bath, we attempted it again as well as preparing some of our Biobricks for submission. We checked pC13BC, pC13BV, and pC13CE and prepared the successful constructs for sequencing. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 13th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We submitted pC13BC, pC13BV, and pC13CE for sequencing. We reinoculated cultures of pC13BN, pC13CL, pC13CO. Then, we conducted a preliminary test on activity of <I>afp1</I>. We cut circles of various sizes from dialyzing membranes and dipped in grown <I>afp1</I> in BL21 culture. The membrane pieces were then placed on the fully grown <I>Aspergillus</I> plate. This preliminary test will see whether or not there is activity of <I>afp1</I> on the <I>Aspergillus</I>.  We are hoping for a ring of death on the <I>Aspergillus</I> plate. Unfortunately, there were no colonies for the <I>PgpdA</I> assembly. We then ran a gel for some of the remaining assembled DNA and no bands appeared. We also digested pC13AU with Xba and PstI to ligate downstream of pC13BR and pC13BQ. We then column purified pC13BQ, pC13BR, pC13AT, pAK13AO, pC13AU, pC13F. After, we dephosphorylated pC13BQ, pC13BR, pC13F, and ligated pC13BQ with pC13AT, pC13BQ with pC13AU, pC13BR with pC13AU, and pAK13AO with pC13F. Next, we electroporated all ligations into both BL21 and DH5α cells. Hopefully, a fluorescence trial can be done tomorrow with these colonies! We made overnight culture for pC13J. We did electroporation for pC13I, pC13CV and pAK13CS. The last one unfortunately arced. Our Gibson-transformed plates had too many colonies so we restreaked them. We miniprepped pC13CG and prepared it for Biobrick submission. We also checked our plates in the Turgeon lab. The control plate is growing well, and the others appear to have something very small starting to grow. We will give them more time before checking again. We also received our sequencing results from pC13BM 1, pC13BK 1, pC13BC, pC13BV, and pC13CE, which all appear to have been successful. We set up a culture of pC13CD from its glycerol stock to re-miniprep and submit as a biobrick.</DIV><P>		Gibson assembly for the <I>G. lucidum</I> promoter yielded no colonies. Numerous ligations and electroporations were done for fluorescent constructs. We are starting the <I>afp1</I><I>Aspergillus</I> growth assay. We did electroporation for pC13I, pC13CV and pAK13CS but pAK13CS arced. We prepared numerous Biobricks for submission to iGEM HQ. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>September 13th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Managed to get the arduino to control the heating circuit! 
		-Rebecca
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 14th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Electroporated pK13AX, but both arced, so outlook is grim. We also picked 12 colonies from potential pAK13CY colonies and grew corresponding colonies in liquid culture. We then prepped pC13AQ, pC13AT, and pC13AU for biobrick submission. Later that day, we picked colonies for pC13BN and pC13CL from plates from Thursday, did a colony PCR, and grew cultures for each. We re-electroporated the pC13CO construct, and plated four different versions of it. We ran a gel of all 14 colonies of pC13BN and pC13CL. The pC13CL one came out blank and the pC13BN had bands, but not where they were expected. We miniprepped pC13J and made glycerol stock. We ran PCR to check the following inserts: pC13AF, pC13P, pC13CV, pC13I and pC13BG. We ran colony PCRs on the restreaked plates containing our Gibson assembly products of homology regions. We miniprepped pC13BQ and pC13CD for part submission.</DIV><P>		The <I>Anderson promoter</I> + <I>T7 polymerase</I> were electroporated, but both ligations arced, so transformations are very likely to not succeed. We are unsuccessful with pC13CL and pC13BN. We miniprepped pC13J cultures. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 15th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We plated 100μL of <I>afp1</I> culture on fresh <I>Aspergillus</I> plates to demonstrate its effectiveness. We ran a PCR for pC13AI with primers 45/52 and 48/51 to add homologous regions for Gibson. The PCR protocol was a 1-step PCR with a 60C annealing temperature and 1:10 extension using GC buffer. We picked colonies from new pC13CO plates. We miniprepped pC13CL colonies 3 and 4, and pC13BN colonies 5 and 6 although the results of the gel was not clear. We redid the PCR for pC13BN. We also redid the PCR to append cut sites to pC13F for creation of pC13BF. We did a PCR clean up for the inserts in pC13BG, pC13P and pC13AF. We ran a PCR for pC13J and pC13BG. We prepared a double digestion for pC13AF and pC13P, and redid the ligation of <I>rbs</I> + <I>crtI</I> (from pC13L) and vector pSB1AK8. We ran gel of the colony PCR of our Gibson products, looking 4-5kb fragments. It appears that pC13BK + pC13BM + pC13BQ colony 1 was the only possibly successful reaction, so we made cultures of it, as well as various other cultures. We prepared pC13BC, pC13BQ, pC13BV, pC13CD, and pC13CE for part submission.</DIV><P>		We are continuing cloning of <I>Lambda Holin</I>, <I>chitinase</I>, and the <I>lox</I> sites and adding homologous regions to the <I>lox</I> sites for Gibson. We started the <I>afp1</I> assay with <I>Aspergillus</I>. We did a digestion for pC13AF and pC13P, and redid the ligation for insert in pC13L and vector pSB1AK8. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-anim"><DIV><H6>September 15th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><P>		We once again got brunch and ate some school famous chocolate peanut butter cake. We agreed that Becker brunch has the best brunch.  Then we sat down all day and put together the majority of the presentation including graphics and content. In addition, Hannah and Eric started writing the script for their part of the presentation. Arun will start writing his script as soon as characterization data starts coming in. 
		-Arun, Eric
</P></DIV></DIV></DIV></DIV><DIV id="week14entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 14</H3><H5 class="centered">(09/16 - 09/22)</H5><DIV class="entry nb-wet"><DIV><H6>September 16th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We had no successful fungal transformants of pC13BR, pC13CD, pUCATPH, and pC13AY. We started taking measurements of <I>G. lucidum</I> growth on CYM plates (plated on September 10) through tracing the central mass of fungus--the differences between the plates are really interesting, but there’s a small contamination on one plate. We transplanted mycelia from the G418 plates. We are slightly concerned that there’s still no growth on the antifungal plates. We submitted pC13CL and pC13BN for sequencing. We ran the gel for the colony PCR of pC13CO but they all failed. We will repick the white colonies from pC13CO plates. We had successful sequencing for pC13AY and we made cultures for glycerol stocks. We also retired the PCR to append cut sites to pC13F using a higher concentration of dNTPs and began a digestion of pA13BE in order to gel extract <I>PpelA</I>. We electroporated <I>E. Coli</I> with pAK13CS, which unfortunately arced again… We performed gel electrophoresis for inserts in pC13CV, pC13I, pC13J and pC13BG. Only pC13J colonies are positive. We also did digestions for the following vectors: pC13BG with SpeI and PstI, pC13AG with EcoRI-HF and XbaI, and pC13K with EcoRI-HF and XbaI.</DIV><P>		We started taking measurements of <I>G. lucidum</I> growth on CYM plates by tracing the central mass of fungus--the differences between the plates are really interesting! Sequencing for pC13AY was successful so we prepared more culture. We did gel electrophoresis for inserts in pC13CV, pC13I, pC13J and pC13BG, but only pC13J colonies are positive. We then did digestions for the following vectors: pC13BG, pC13AG, and pC13KI. We still do not have any growth on antifungal plates. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>September 16th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Compared with last week’s jigsaw induced excitement, this week was relatively tame.  With all of the necessary pieces in our possession, we set out to assemble our incubator.  After sifting through the leftover drylab supplies from last year’s project, we came up with a couple tubes of silicone adhesive that we could use.  Insulation pieces were adhered to their respective plastic backings and were allowed to dry for a couple days. 
		-Mac
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 17th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We attempted transforming <I>Cochliobolus</I> with pC13BR, pC13CD, pUCATPH and pC13AY. We ran a gel of the pA13BE digest and extracted <I>PpelA</I>. We also reran a PCR to append cut sites to pC13F again using a different tube of dNTPs and re-diluted primers. We ran a colony PCR and screened it on a gel, but no bands appeared. Some of the cultures were miniprepped anyways, and they will be screened for presence of the insert (~340bp). We detected some growth on the afp1 assay plates. We redid the Gibson of <I>Cht1_2</I> and pSB1C3 PCR. We electroporated this Gibson into DH5a cells and plated it on chloramphenicol LB plates. We reran the PCR of pC13AI with primers 45/52 and 48/51, annealing temperature of 61.5C and extended for 30s. We did a double digest of pC13CO with EcoRI-HF &amp; PstI. We ran a 1% TAE gel of pC13AI PCR products for 30 minutes at 100V. The bands for pC13AI with 45/53 were seen at 1.5kb and 1.1kb; bands for pC13AI with 48/51 were seen at 1.5kb and 1kb. We still have a weird banding pattern. We may need to do a gradient PCR to find a better temperature for the primers and to test the PCR without the GC buffer. We purified and dephosphorylated the vectors we digested yesterday--pC13BG, pC13AG and pC13K. We also purified inserts of pC13AF with EcoRI-HF and SpeI, and pC13P with EcoRI-HF and SpeI.</DIV><P>		We tried transformation of <I>Cochliobolus</I> again. We ran a PCR to append cut sites to pC13F but no band was shown on a gel. We redid the Gibson of <I>Cht1_2</I> and pSB1C3. When we ran a PCR fo pC13AI, weird band patterning was observed. We purified the vectors we digested yesterday: pC13BG, pC13AG and pC13K. In addition, we purified inserts of pC13AF and pC13P. We continued recording measurements of <I>G. lucidum</I>’s growth in the presence of antibiotics. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-anim"><DIV><H6>September 17th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><P>		We met for our first run through of our presentation.  The script needs some work, but it’s coming along nicely. It seems that the only part lacking in the presentation is the characterization data and charts. They coming soon! 
		-Arun, Eric
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 18th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We used less hygromycin for transformation because Cochliobolus concentrations are less than <I>G. lucidum</I> concentrations. We are recording more and more data with regards with fungal resistance! We found a new contaminant on another plate but we will keep an eye out for it and make sure our measurements will not include its small area of contact with <I>G. lucidum</I>. We ran the gel extracted <I>PpelA</I> and pC13F PCRs on a gel but there were no bands. However, they may still be successful as the concentration were fairly low and may not have appeared on the gel. We ran the pC13CO digests on the gel and looked good. We ran a PCR of pC13AI to amplify <I>lox</I> sites for Gibson with harsh conditions to remove the nonspecific amplification.</DIV><P>		We are getting more data with regards to fungal growth on resistance plates! We ran the pC13CO digests on a gel and they looked good. We ran a PCR of pC13AI to amplify <I>lox</I> sites for Gibson. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 19th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We quantified the holin minipreps. Both were around 48ng/uL, which is too low to send to sequencing. We will reculture and miniprep when LB is ready. We ran a PCR of pC13AI with primers 45/52 and 48/51. We ran the PCR products on a 1% gel and found bands at 1.5kb and 1.2kb for <I>lox1</I> and 1.5kb and 1kb for <I>lox2</I>. There may be something wrong with the pC13AI plasmid. We also ran a 1 step PCR of <I>lox1</I> and <I>lox2</I> with a 59C annealing temperature. We submitted pC13J for sequencing with Primer 31. We also ligated the constructs from Tue--in short, <I>crtB</I> + <I>crtI</I>, <I>crtI</I> + <I>PtrpC</I>. We ran a PCR with Q5 polymerase for inserts in pC13M, pC13J and pAK13CR and ran a gel for them--pC13J was good, pC13M had a faint band but nothing for pAK13CR. We took photos of the plates for the experiment on <I>G. lucidum</I>’s growth in the presence of antibiotics, to capture the differences in appearance in addition to colony size.</DIV><P>		Something may be wrong with the pC13AI plasmid based on gel screens. We ligated the constructs from Tuesday--<I>crtB</I> with <I>crtI</I> and <I>crtI</I> with <I>PtrpC</I>. We continued recording measurements of <I>G. lucidum</I>’s growth in the presence of antibiotics. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 20th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We began another growth assay because the previous ones showed no results. The cells may bave been dead by the time the cells reached them. This method of spreading the <I>afp1</I> culture like an antibiotic and plating mycelium smoothie on the top should give more consistent results. We retried electroporation with pAK13CS and it sadly arced again… We also ran a Q5 PCR for the insert in pAK13CR. We prepared and autoclaved CYM + agar, for making plates.</DIV><P>		We will start a new growth assay to test afp1. We ran a Q5 PCR for pAK13CR. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wiki"><DIV><H6>September 20th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><P>		I started populating the pages with a lot of content. Wiki freeze is coming up! 
		-Nupur
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 21st
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We ran a Gibson of the DpnI digest of pC13AI PCRs. In terms of characterization, we ran a huge Q5 PCR on pC13BQ with pC13AT, pC13BQ with pC13AU, pC13BR with pC13AU, pC13BR with pC13AT, and pAK13AO with pC13F using standard biobrick forward and reverse primers 23 and 24. We set up the PCR to append KpnI and BamHI cut sites to pC13F again with different annealing temperatures and ran a gel to test the PCRs. We ran a gel for the PCR products--insert of pAK13CR--from yesterday, and then we did a PCR clean up of the inserts of pAK13CR and pC13I. We then did electroporation for pC13DJ and pC13BD. The latter arced... We digested pSB1C3 with EcoRI-HF and PstI, pSB1C3 with XbaI and PstI, pAK13CR with EcoRI-HF and PstI, pC13I with XbaI and PstI, pSB1AK8 with SpeI and PstI, pC13J with XbaI and PstI, pC13M with XbaI and PstI. We then dephosphorylated digested pSB1C3 and pSB1AK8. Ligation was performed for the following: <I>RBS</I> + <I>crtE</I> (from pC13J) into pSB1AK8 to create pAK13CQ, <I>RBS</I> + <I>crtB</I> (from pAK13CR) into pSB1C3 to create pC13CV, <I>ctrB</I> (from pC13M) into pC13BG to create pC13DP, <I>RBS</I> + <I>crtI</I> (from pC13L) into pSB1AK8 to create pAK13CS. We transformed <I>T7</I> + <I>hph</I> (pC13BV) into BL21 cells and plated on chloramphenicol plates. Unfortunately they arced, so we will need to repeat that with clean electroporation cuvettes. We poured more CYM plates.</DIV><P>		We set up the PCR to append KpnI and BamHI cut sites to pC13F again with different annealing temperatures and ran a gel to test the PCRs. Ran Gibson for pC13AI. A large Q5 PCR was done on fluorescent constructs pC13BQ with pC13AT, pC13BQ with pC13AU, pC13BR with pC13AU, pC13BR with pC13AT, and pAK13AO with pC13F. We ran the gel for pAK13CR from yesterday. We ligated the following: <I>RBS</I> + <I>crtE</I> to make pAK13CQ, <I>RBS</I> + <I>crtB</I> to make pC13CV, <I>ctrB</I> + pC13BG to make pC13DP, <I>RBS</I> + <I>crtI</I> to make pAK13CS. We attempted to transform <I>pC13BV</I> into BL21 <I>E. coli</I>, but failed. We then poured CYM plates. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-anim"><DIV><H6>September 21st
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><P>		We ran through our slides for the full team.  We received a lot of good advice, and we still have a lot of work to do, but most of it will have to wait until after tuesday so we don’t fail our orgo exam! The main suggestion would be to change the theme of the presentation as the baby blue and serene green are too subtle to captivate attention. In addition, there are little glitches to the images. 
		-Arun , Eric
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 22nd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We did a PCR cleanup of the pC13AI Gibson pieces, electroporated (time constant was 2.88 ), and plated the cells. We continued taking masses for <I>Aspergillus</I> with <I>afp1</I> growth assay and began a growth assay with inoculating discs. We measured the colonies of <I>G. lucidum</I> (control, and in the presence of various antibiotics) one final time, as they reached the maximum possible sizes for the petri dishes they were growing in.</DIV><P>		We plated pC13AI. We are continuing <I>Aspergillus</I> growth assay, and completed the <I>G. lucidum</I> growth assay. 
		-Rafael
</P></DIV></DIV></DIV></DIV><DIV id="week15entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 15</H3><H5 class="centered">(09/23 - 09/29)</H5><DIV class="entry nb-wet"><DIV><H6>September 23rd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Two gels, each with 20 wells, were run simultaneously for fluorescence. The DNA samples loaded were from a Q5 PCR on pC13BQ with pC13AT, pC13BQ with pC13AU, pC13BR with pC13AU, pC13BR with pC13AT, and pAK13AO with pC13F. pC13BV was transformed into electrocompetent BL21 <I>E. coli</I>. A culture of <I>G. lucidum</I> was diluted and homogenized, and 150 μL aliquots were spread on plates and let dry. Paper discs were soaked in 100X solutions of geniticin, hygromycin, ampicillin, kanamycin, and chloramphenicol and placed in the center of each plate. We miniprepped <I>holin</I> 8 &amp; 9. The results from the growth assay do not look good. However, we found that the T7 polymerase in BL21 is controlled with an IPTG inducible promoter, so more pC13BA was grown with IPTG and plates were all re-inoculated with the corresponding amounts. An appropriate volume of LB was also added to some plates to account for mass changes. The zone of inhibition tests were also restarted with recultured (with IPTG) pC13BA. The pC13BN transformations yielded no colonies.</DIV><P>		Today we ran a gel to verify insert lengths from Q5 PCR products of fluorescent constructs. We also transformed our T7 + hygromycin resistance construct into <I>E. coli</I> for characterization, and set up a zone of inhibition experiment to test the efficacy of antibiotics and antifungals on <I>G. lucidum</I>. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>September 23rd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		This week, the incubator box was completed.  The fabrication and electrical groups came together to begin combining and testing.  Initial tests with the temperature sensor yielded very positive results.  We were able to successfully set and reach a target temperature with our sensor/feedback system.  Unfortunately, our humidity sensor malfunctioned while we were running tests, however, we were still able to gather some data.  Initial humidity tests indicate that our misting apparatus is able to produce a 12% increase in humidity over a time span of 8 minutes.  Using a hotter water supply, as well as allowing the misting apparatus to run for longer would certainly allow us to increase humidity even more. 
		-Mac
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 24th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We miniprepped the pC13AI cultures. We amplified with primers 45-52 and primers 48-51 and ran a gel (expected size at 1.5kb). Nothing appeared on the gel, not even the ladder. The EtBr should be replaced. There were too many transformants for pC13BV in the BL21 strain, so those cells were restreaked onto a new plate. Additionally, we transformed pC13BC into BL21 as well, and characterize that part as well.</DIV><P>		The gel of pC13AI showed nothing - the EtBr (for gel staining) needs to be replaced. We also continued putting our T7+resistances constructs into <I>E. coli</I> for part characterization. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 25th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">The sequencing of pC13CO came back as <I>PtrpC</I> + <I>GFP</I>. We are not sure why or how. We either miniprepped the wrong thing or put the wrong thing into sequencing. We continued to take measurements for growth assays. We put a lot of <I>Aspergillus</I> around the inoculating discs for gathering more data. We set up cultures of BL21, with no plasmid and with pC13BC and pC13BV, and then standardized the optical densities of the cells between the three cultures. We spread 50 μL aliquots onto LB, adding discs dipped in 10X, 1X, and 0X concentrations of geneticin and hygromycin.</DIV><P>		Sequencing of pC13CO did not turn out to be positive. Still trying to get data from <I>Aspergillus</I> and growth assay, and characterizing the <I>T7 promoter</I> and antibiotic resistances constructs. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-anim"><DIV><H6>September 25th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><P>		We made a lot of changes to the presentation tonight, and it is an almost polished form now.  Arun switched to his next color scheme: a deep Cornell red with baby blue banners. Eric has gotten his part of the script memorized. He will be speaking about the three components of our project. The fungal toolkit, and its implementation through the carotenoid pathway and antifungals expression. Hannah has also prepared her part on human practices. In addition she has prepared an analysis on the market trends towards bioplastics. 
		-Arun, Eric
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 26th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">There was a clear zone of inhibition around the discs dipped in 10X hygromycin, no inhibition around the discs dipped in sterile water, for the wild type BL21 <I>E. coli</I>. However, there was normal growth around the 10X hygromycin discs for the BL21 <I>E. coli</I> transformed with our pC13BV construct. For the geneticin tests, the antibiotic appeared relatively ineffective, possibly due to degradation, and that experiment will need to be repeated.</DIV><P>		The <I>T7 promoter</I> and hygromycin resistance construct appears functional in <I>E. coli</I>. 
		-Rafael
</P></DIV></DIV><DIV class="entry nb-anim"><DIV><H6>September 27th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><P>		Today we met again with Oat to make some changes to our images.  Arun moved onto his next color scheme: the same deep maroon with elephant grey banners.  In addition, the data, and ending slides will follow a baby blue and off white color scheme. We also made a few of our slides (our styrofoam map, standoff figure, and toolkit image) fancier and more attractive.  Arun also experimented with backgrounds of clouds, dirt, and toolkit to make the introduction slides very sharp. It look like we now have a solid design for our presentation. 
		-Arun,Eric
</P></DIV></DIV></DIV></DIV><DIV id="week16entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 16</H3><H5 class="centered">(09/30 - 10/06)</H5><DIV class="entry nb-wet"><DIV><H6>September 30th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Ran 1% TAE gel of pC13AI amplified to add <I>lox</I> sites upstream and downstream. Expected lengths were about 1kb, but results showed approximately 1.3 kb and 1 kb. We ligated ‘’crtI’’ (from pC13I) with <I>PtrpC</I> (from pC13BG) to create pC13BD. Finally, a digestion was done for pC13I with XbaI and PstI.</DIV><P>		The Gel of pSB1C3 with appended <I>lox</I> sites showed bands close to the expected length, so assembly of pC13BN can continue. We also did a ligation to create pC13BD and a digestion of pC13I. 
		-Kyle
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 1st
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Today, parts for pC13BN were DpnI digested, purified and Gibson assembled. We also performed a PCR of pC13F to append BamHI and KpnI restriction sites to the pSB1C3 backbone. The goal was to digest pC13F with BamHI and KpnI and ligate <I>PpelA</I> in place of ‘’mRFP’’ using BamHI/KpnI digested pA13BE (part pHNU3pelA). pA13BE is a plasmid generously given to us by the Turgeon lab at Cornell and contains the ‘’PpelA’’ flanked by BamHI and KpnI restriction sites. Thus, transferring ‘’PpelA’’ to the pSB1C3 backbone (creating the part pC13BF) would allow constructs made with ‘’PpelA’’ to be BioBrick submissible. All 4 of the PCRs (in addition to the BamHI/KpnI digestion of pA13BE) seemed successful upon gel electrophoresis. We also received the <I>A. nigers</I><I>pkiA</I> promoter (‘’PpkiA’’) and <I>xlnD</I> terminator (‘’TxlnD’’) from the Wageningen team, which we plan on characterizing by cloning ‘’GFP’’ and ‘’mRFP’’ (in separate constructs) between the two.</DIV><P>		pC13BN was Gibson assembled. 
		-Daniel
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 2nd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Today, we miniprepped 5 extra-cloudy colony cultures of pC13CQ with strong yields. We also made overnight cultures of pC13CS (#’s 2,3,10,11,12) and pC13DJ (#’s 1,2,3,4,5), although none of the colonies looked good. We column purified the pC13F PCRs, all of which had fair concentrations (30, 100, and 150 ng/ml). We then digested them using BamHI and KpnI. The gel screen of the digestions was inconclusive, but we decided to proceed with the ligation regardless. We also made liquid cultures of ‘’PpkiA’’ and ‘’TxlnD’’.</DIV><P>		We miniprepped the overnight cultures of pAK13CQ colonies and regrew new cultures for pAK13CS and pC13DJ. 
		-Daniel
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 3rd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">No pC13DJ grew, but we were able to run a PCR of all pC13CQ and pC13CS colonies, and a PCR cleanup of pC13I. The PCR showed that pC13CQ cultures 1,2,4, and 5 were all good. The ligation was likely at fault for failed pC13CS, as those appeared to have only ‘’PT7’’. We also electroporated and plated pC13DJ, pC13DP, pC13CV, and pC13BD without any arcing!</DIV><P>		We ran a PCR, and confirmed probable creation of pC13CQ. 
		-Kyle
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 4th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We redid the Gibson assembly of ‘’pC13BN’’ pieces and submitted a sequencing order for pC13CQ numbers 2 and 5.</DIV><P>		The Gibson assembly of pC13BN disappeared and needed to be redone. 
		-Kyle
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 5th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We desalted and electroporated the pC13BN construct into DH5α. The time constant was 4.2. The transformants recovered for 1 hr and were then plated on chloramphenicol LB agar.</DIV><P>		The pC13BN Gibson assembly was transformed. 
		-Ryan
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 6th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Three colonies of the pC13BN transformation were picked, streaked on reference plate, and grown in 5 mL of LB.</DIV><P>		The pC13BN transformation yielded a few colonies, some of which were chosen for culturing. 
		-Ryan
</P></DIV></DIV></DIV></DIV><DIV id="week17entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 17</H3><H5 class="centered">(10/07 - 10/13)</H5><DIV class="entry nb-wet"><DIV><H6>October 7th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Today, we made LB stocks of pC13BD, pC13CV, pC13DJ, and pC13DP. We also dephosphorylated the BamHI/KpnI digested pC13F PCRs in preparation for ligation. We then column purified the BamHI/KpnI digested pA13BE, after which we set up an overnight ligation with pA13BE and pC13F to create the part pC13BF.</DIV><P>		Today, we set up an overnight ligation to create the part pC13BF. 
		-Daniel
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 8th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Today, we miniprepped pC13BN colonies, submitting all for sequencing. We also miniprepped and ran PCRs of pC13BD (#’s 1-5), pC13CV (#’s 1-5), pC13DJ (all except 4), and pC13DP (#’s 2,3). Additionally, we remade liquid cultures of ‘’PpkiA’’ and ‘’TxlnD’’. The cultures made on 10/2/13 had not been miniprepped and had to be thrown out because we had to prioritize preparation for the regionals competition. We also gel extracted ‘’PpelA’’ from the BamHI/KpnI digested pA13BE for another ligation with the BamHI/KpnI digested pC13F. This method of ligation was pursued because the ligation done in the traditional manner on 10/7/13 did not seem to be successful.</DIV><P>		Colonies of pC13BN were miniprepped and submitted for sequencing, and creation of pC13CQ was confirmed by sequencing. 
		-Daniel
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 9th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Grew an ‘’afp1’’ culture with induction by arabinose overnight, and began a growth assay of <I>A.niger</I> with LB &amp; arabinose, BL21-AI &amp; arabinose, and ‘’afp1’’ &amp; arabinose inoculated plates in triplicate. Ran a gel of all yesterdays PCRs, only pC13DJ looked like it might be successful on a gel. Created PCR inserts for pC13BC (5), pC13CR, pC13L, and pC13A (1). Ran PCR of ‘’A3044’’ and ‘’A(poly)’’. Found that pC13CV #1, pC13DJ #1, and pC13DP #2 are bad, and everything else is fine.</DIV><P>		An ‘’afp1’’ culture was grown overnight. Growth assays of <I>A. niger</I>, BL21-AI, and ‘’afp1’’ were started. 
		-Kyle
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 10th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Made LB and LB agar. Made a gel of all yesterdays PCRs: pC13CR and ‘’A3044’’ failed. Redoing them using pC13CV miniprep instead of pC13CR. Ran a PCR of pC13CV. Quantified minipreps of pC13BD (#’s 1 and 2), pC13BD (#’s 3-5), pC13CV (#’s 2,3, and 5), pC13DP (# 3), pC13DJ (#’s 2-5). Column purified and quantified pC13BC, pC13L and ‘’A(poly)’’ to fine results. Ran another ‘’A3044’’ PCR with new primer dilutions. Digested pC13F with XbaI and PstI, pC13D with SpeI and PstI, pC13BC with XbaI and PstI, pC13L with XbaI and PstI, ‘’A(poly)’’ with XbaI and PstI, pC13DJ with SpeI and PstI, and pC13CQ with SpeI and PstI. In terms of characterization, pAK13CY was successfully sequence confirmed today. We will be able to run another fluorescence assay soon!</DIV><P>		We found out today that pAK13CY was successfully sequence confirmed. 
		-Jonlin
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 11th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Miniprepped pC13Y, pC13BA, pC13BG, pCh13V, and pCg13S. pC13BG was digested with SpeI and PstI and pC13BA was digested with XbaI and PstI overnight. The carotenoids subteam found that pC13CV is bad. We think that we had the wrong miniprep to start with, so we ran a PCR to create pC13CV again. The gel of pC13A and pC13L looked good, so the PCR products were column purified and quantified. We also digested pC13D with SpeI and PstI, and pC13A with XbaI and PstI.</DIV><P>		Several constructs, including <I>PgpdA</I> and <I>afp1</I>, were miniprepped today and <I>PgpdA</I> and <I>afp1</I> were digested for ligation. In Carotenoids, we unfortunately discovered that pC13CV is also bad, and we suspect that we have the wrong miniprep. Gel electrophoresis showed that pC13A and pC13L are good. 
		-Tim
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>October 11
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">It was decided that a reptile heating pad would be ordered to heat the incubation chamber more rapidly and evenly.  Other plans were made to order pcb boards for both circuits to replace the breadboards, which proved unreliable.</DIV><P>		Today a meeting was held to determine the timeline of events up until the world competition.  The temperature and humidity circuits were set as high priority, particularly the integration of electronics with the incubator.  Low priority were setting up a wireless connection, implementing a power switch, performing a cost analysis, and adding ventilation to the incubator. 
		-Sara
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 12th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Heat killed, purified, and dephosphorylated pC13BG and pC13BA digests. Reference plates of ‘’PpkiA’’ and ‘’TxlnD’’ were made from the glycerol stocks sent to us by the Wageningen team. We once again did not have time to miniprep the cultures inoculated on 10/8/13 due to exams. We also column purified the ‘’PpelA’’ gel extract and ligated it into pC13F overnight for electroporation tomorrow.</DIV><P>		Plasmids pC13BG and pC13BA were prepared for ligation. 
		-Daniel
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>October 12
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		The electronics subteam met and worked on troubleshooting the temperature sensor.   The problem was solved by changing how each circuit was grounded.  Also, the arduino code was improved upon. 
		-Sara
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 13th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Electroporated 10 uL of pC13P and pC13BA ligation into DH5a as well as 10 uL of pAK13CY (time constants 3.88 &amp; 4.66 respectively). Recovered for 1hr before plating on chloramphenicol  and kanamycin plates respectively. No growth for pC13P+pC13BA but pC13CY showed strong growth. Three colonies of the latter were cultured in LB with kanamycin antibiotic. The pC13BF ligation created using the <I>PpelA</I> gel extract into DH5a <I>E. coli</I> was electroporated and plated on chloramphenicol plates. As the reference plates did not produce single colonies, we re-streaked <I>PpkiA</I> and <I>TxlnD</I> as well.</DIV><P>		The <I>PtrcpC</I> and <I>afp1</I> construct as well as pC13CY were transformed. The former failed, but the latter succeeded. Three colonies of pC13CY were chosen for culturing. 
		-Tim
</P></DIV></DIV></DIV></DIV><DIV id="week18entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 18</H3><H5 class="centered">(10/14 - 10/20)</H5><DIV class="entry nb-wet"><DIV><H6>October 14th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech"><I>Psal</I> samples were electroporated with time constants 4.70 and 4.72.</DIV><P>		Various <I>Psal</I> samples were transformed. 
		-Tim
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 15th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">There weren’t any colonies of <I>Psal</I>, but there were a few colonies for the <I>PtrpC</I> + ‘’afp1’’ construct. We picked 3 to grow for sequencing. The reference plate colonies for pC13CY grew together and can’t be used. If sequencing comes back good for any one of the colonies, then we will need to re-transform. We cultured the pC13BF transformants and single colonies of ‘’PpkiA’’ and ‘’TxlnD’’. On carotenoids, we performed ligations from scratch for pAK13CS, pAK13AA, pC13DT, as well as putting ‘’PT7’’ and ‘’rbs’’ with ‘’crtEB’’, and ‘’PT7’’ with ‘’crtBI’’.</DIV><P>		No colonies for ‘’Psal’’ grew, but a few for <I>PtrpC</I> and ‘’afp1’’ did and were cultured. Additionally, the reference plate for pC13CY is really messy. For the carotenoids pathway, we redid many of our ligations. 
		-Jeffrey
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 16th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We set up new cultures of pC13BC, pC13BV, and wild type BL21-AI cells. Then, we ran PCRs of the left and right homology regions of <I>G. lucidum</I> along with pC13Y, pC13BQ and pC13BR and gel screened them. Three colonies from pC13P and pC13BA construct transformation were cultured in 5 mL of LB, followed by miniprepping and sequencing. We miniprepped the cultured pC13BF transformants and the liquid cultures of ‘’PpkiA’’ and ‘’TxlnD’’. Unfortunately, the concentrations of the minipreps were too low to submit for sequencing.</DIV><P>		Prepared to repeat the characterization of resistance parts pC13BC and pC13BV in BL21-AI <I>E. coli</I>. We also began working on agrobacterium mediated transformation! Plates of pC13P and pC13BA grew, so we picked three colonies from each plate and made cultures. We also miniprepped pC13P &amp; pC13BA and submitted for sequencing. 
		-Olya
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 17th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Standardized the ODs of resistance-characterization cultures to approximately 1.27, and spread 100 µL on LB and LB/Cm plates. Left plates out to dry before adding discs dipped in antifungals. We also cultured pC13BF, pC13F, pC13Z, ‘’PpkiA’’, and ‘’TxlnD’’.</DIV><P>		An experiment for characterization of resistance parts pC13BC and pC13BV was set up. The sequencing results of pC13P + pC13BA were not good. 
		-Olya
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 18th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Placed antifungals-dipped discs (10X and 2X of each Geneticin and Hygromycin) onto plates of pC13BC, pC13BV, and wild type BL21-AI, but there appeared to have been a significant amount of growth on them already, which may affect results. We ran Gibson assemblies of the left and right homology regions of <I>G. lucidum</I> with pC13Y, pC13BQ and pC13BR and transformed them. The sequencing of <I>Psal</I> was weird: the primer appeared to fall off after amplifying the cut sites. We will resubmit for sequencing after looking at a digest screen. We miniprepped pC13Z, pC13F, ‘’PpkiA’’, and ‘’TxlnD’’. ‘’PpkiA’’ and ‘’TxlnD’’ were submitted for sequencing to verify Wageningen had sent the correct parts. For some reason, the pC13BF cultures did not grow so we decided to give them another day to grow before miniprepping and sequencing (to confirm the success of the ligation). Finally, on carotenoids we miniprepped pAK13CS, pAK13AA, pC13DT, as well as ‘’PT7’’ and ‘’rbs’’ with ‘’crtEB’’, and ‘’PT7’’ with ‘’crtBI’’. After quantification, a PCR with extended an annealing time was run to confirm the sequence.</DIV><P>		Sequencing results of <I>Psal</I> were inconclusive and will be resubmitted.  On carotenoids, we miniprepped the components from yesterday and ran a PCR for confirmation. 
		-Olya
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>October 18
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		The heating pad which was ordered arrived, but integrating it with the arduino circuit did not seem to be a realistic goal within the time frame before competition.  Instead, multiple resistors were connected to the circuit, and it was found that a greater power supply was needed, along with more resistors of lower resistance.  Transistors, resistors, and breadboard adapter plugs were ordered. 
		-Sara
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 19th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Since even growth of the wild type BL21-AI cells appeared uninhibited by antifungals, the characterization attempt for pC13BC and pC13BV was inconclusive, likely due to some combination of antifungals having gone bad or to too much growth prior to addition of antifungals. We set up PCRs of inserts from pC13AT, pC13AU, pC13AV, pC13AY, pC13CC, and pC13CE constructs to append HindIII and KpnI cut sites using new primers. The plan is to transfer the inserts into pAg13H and pAh13EA vectors. Next, we digested pAg13H and the successful PCRs of pC13AT, pC13AU, and ‘’pC13AV’’ with HindIII and KpnI and ligated overnight. The successful transformants from yesterday’s Gibson assemblies were cultured, and we ran a digest screen of all Psal minipreps with low yields: the image showed correct bands. Ran a PCR overnight of the <I>Psal</I> F and <I>Psal</I> R primers that just came in to amplify out Psal with its activator (nahR with P constitutive promoter). The SpeI cut site on will be appended by the reverse primer, while the forward primer is just amplifying the region already containing the BioBrick prefix. We also grew 10 ml overnight cultures of pC13P &amp; pC13BA in LB and Chloramphenicol. Overall a productive day.</DIV><P>		Characterization of pC13BC and pC13BV was unsuccessful. We began cloning of completed constructs into new vectors for <I>C. heterosporus</I> transformation, as well as cloning of <I>Psal</I>, and grew cultures of <I>PtrpC</I> with ‘’afp1’’. Gel electrophoresis showed that none of  the components of the carotenoid pathway yesterday work. We realized that the forward primer we used is wrong and designed a new one. 
		-Olya
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 20th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We transformed the ligations of pC13AT, pC13AU, and pC13AV (in pAg13H) into <I>E. coli</I>.  We miniprepped pC13DU, pC13DV and pC13DW and gel screened them, but the results were inconclusive. We also miniprepped pC13P &amp; pC13BA digested with PstI and EcoRI to run a digest screen. The pC13BF cultures still have not grown, causing us to surmise that the bacteria in the culture are contaminants that somehow acquired antibiotic resistance. Due to our time constraints and the amount of troubleshooting that would be required to successfully construct pC13BF at this point, we decided to abandon submitting pC13BF as a BioBrick. As a result, we prioritized our remaining time to characterize the parts sent to us by Wageningen. Thus, we digested our ‘’PpkiA’’ minipreps with SpeI and PstII and also digested ‘’TxlnD’’, pC13Z, and pC13F with XbaI and PstII, the goal of which was to successively ligate ‘’GFP’’ and ‘’mRFP’’ (in separate constructs) downstream of the ‘’PpkiA’’ promoter followed by ‘’TxlnD’’ downstream of the fluorescence genes.</DIV><P>		We continued the transfer of completed constructs into <I>C. heterosporus</I> transformation vectors. The <I>PtrpC</I> and ‘’afp1’’ constructs were tested for correct length of insert. 
		-Ryan
</P></DIV></DIV></DIV></DIV><DIV id="week19entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 19</H3><H5 class="centered">(10/21 - 10/28)</H5><DIV class="entry nb-wet"><DIV><H6>October 21st
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We set up cultures and colony PCRs of pC13AT (in pAg13H), but none of the other plates had colonies. We checked the colony PCRs, and were puzzled by the unexpected-sized bands of 300kb and 600kb - we guessed that the primers may be annealing oddly to the new vector, and decided that we’d try to transform those into <I>C. heterosporus</I> anyway. We also set up new ligations of pC13AU and pC13AV into pAg13H. We transformed the remains of pAh13EA into <I>E. coli</I>. We ran a PCR for pOSCAR for future Gibson assembly with pC13DU, pC13DV and pC13DW. We then ran a gel screen of the pC13DU, pC13DV, pC13DW and pS13EB and they looked ok. We also prepped the pC13P &amp; pC13BA for sequencing and digested pC13P and pC13AI with SpeI and PstI and pC13BA and pC13CY with XbaI and PstI. We gel screened the digestions performed on 10/21/13 and all 4 digestions (‘’PpkiA’’ SP, ‘’TxlnD’’ XP, pC13Z XP, and pC13F XP) appeared successful. Progress was halted due to the lack of columns for DNA purification.</DIV><P>		We continued the transfer of completed constructs into <I>C. heterosporus</I> transformation vectors. Submitted <I>PtrpC</I> and ‘’afp1’’ construct for sequencing and digested other plasmids and inserts. 
		-Olya
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 22nd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We miniprepped colonies 1 and 2 of the construct that hopefully is pC13AT in pAg13H, and linearized half of each with HindIII before transforming into protoplasts of <I>C. heterosporus</I>. We cultured pAh13EA as a vector in insert more of our constructs into for characterization in <I>C. heterosporus</I>, and planned out a timeline for the rest of our cloning into the two new vectors. We ran new PCRs of the inserts from pC13AT, pC13AU, pC13AV, pC13CC, and pC13CE, and they all appeared successful this time! We miniprepped the cultures of pAh13EA and digested that and pAg13H with EcoRI and HindIII, in attempt to get a cleaner digest. We transformed the redone ligations of pC13AU and pC13AV in pAg13H into <I>E. coli</I>. We submitted pC13DU, pC13DV and pC13DW for sequencing and then ran a Gibson assembly of pS13EB with pC13DU, pC13DV and pC13DW and transformed.</DIV><P>		We continued transferring constructs into <I>C. heterosoporus</I> vectors, and transformed the first completed one into the fungus. We electroporated pC13P + pC13BA ligation into BL21-AI and then plated it on an LB agar plate with chloramphenicol. 
		-Olya
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 23rd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We plated an overlay of 1% agar and 1000µg/mL geneticin onto <I>C. heterosporus</I>. Yesterday’s digest of pAh13EA resulted in strange bands due to not knowing the sequence, and we redigested the vector with KpnI and HindIII instead. We also repeated the EcoRI/HindIII digest for pAg13H, since it seemed to be incompletely digested. Then we digested the newest PCRs with each of those sets of restriction enzymes, and dephosphorylated the vectors. We ligated pC13AT, pC13AU, and pC13AV into both pAg13H and pAh13EA, as well as pC13CE into pAg13H, and pC13CC in pAh13EA. Gibson assemblies of pC13DU, pC13DV and pC13DW were plated with Spectinomycin. Made 1 L of minimal media and 1 L of induction media. We also transformed pC13P + pC13BA which resulted in very dense colony growth. Attempted to pick 5 colonies and culture them. Also column purified pC13CY. Carotenoids received the right forward primer for amplifying ‘’crtE+crtB+crtI’’ in pC13A, and we ran a long, two-step PCR with an extended annealing time and 40 cycles.</DIV><P>		Today we completed the transformation of <I>C. heterosporus</I> and continued inserting our constructs into the new vectors for the last round of fungal transformations coming up. Transformation of <I>PtrpC</I> and ‘’afp1’’ was really successful! Picked colonies for culture and sequencing. On carotenoids, we finally got the right primer and ran a PCR for pC13A. 
		-Olya
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 24th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We transformed the completed ligations into <I>E. coli</I>. We checked for colonies in the evening, but there weren’t any yet. Any bacteria containing the new vectors have been growing very slowly under selective pressures, since they might be low-copy compared to our usual plasmids. We ran a PCR of pS13EB to prepare for Gibson assembly. Ran a gel of pS13EB, bands at 10,000 and 2,000 indicating probable correct product. Also, pC13DU, pC13DV, and pC13DW were miniprepped. We also digested pC13AI with SpeI and PstI. We then dephosphorylated, purified, and ligated pC13CY into pC13AI. We then miniprepped 5 colonies of pC13P+pC13BA, column purified the Psal, and quantified each. We column purified the SpeI/PstII digested ‘’PpkiA’’, the XbaI/PstII digested ‘’TxlnD’’, the XbaI/PstII digested pC13Z,  and the XbaI/PstII digested pC13F.  Next, we vector dephosphorylated ‘’PpkiA’’ and subsequently ligated ‘’mRFP’’ (from pC13F) and ‘’GFP’’ (from pC13Z) downstream of ‘’PpkiA’’. The carotenoids team ran a gel for the PCR products of pC13A and observed 2~3kb bands for all the lanes.</DIV><P>		We continued work on inserting our constructs into the new vectors for the upcoming fungal transformation. Plasmids pC13CY and pC13AI were digested and ligated. Additionally, many colonies and other constructs were quantified. Gel electrophoresis showed that the carotenoids team successfully amplified the construct in pC13A! 
		-Kyle
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>October 24
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		A few of the electronic components arrived so construction of the humidity circuit was begun. 
		-Sara
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 25th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Colonies appeared on the plate from the ligation of pC13AU in pAg13H, so we set up cultures of those and ran a colony PCR. Colonies appeared on the other ligated pAg13H plates a little later, and those were cultured (with corresponding colony PCRs) as well. We miniprepped the cultures of pC13AU in pAg13H, and the other cultures of vector pAh13EA. Sequencing for pC13DU came back successful. We ran a Gibson assembly pC13DU and pS13EB then transformed with 5x Spectinomycin to check for selection. We also made a lot more LB and LB/agar! For characterization, we retransformed pAK13D, pC13AQ, and pAK13CY into BL21-AI cells by electroporation in order to run another assay tomorrow. We electroporated ‘’PpkiA’’ + pC13Z and PpkiA + pC13F into DH5a <I>E. coli</I>. Next, we plated the bacteria on chloramphenicol plates for selection. After column purifying the PCR products of pC13A, the carotenoids team performed a double digestion of pC13A and pAK13D using enzymes XbaI and PstI.</DIV><P>		We cultured constructs ligated into the pAg13H vector for the upcoming fungal transformation. Hopefully they were successful! We also began to prepare liquid cell cultures for another fluorescence assay. The carotenoids team did a double digestion of pC13A and pAK13D. 
		-Olya
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 26th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We ran a gel of the colony PCRs, but the bands looked very blurred, so we re-ran another gel of them, which gave clear results - 3 of the 4 inserts appeared to successfully be in pAg13H! Ran a PCR of pC13DU for use in a final Gibson attempt. None of the ‘’PpkiA’’ + pC13Z transformants were present. We cultured ‘’PpkiA’’ + pC13F transformants for sequencing. Unfortunately, to test the strength of the promoter using fluorescence, we would have to transfer ‘’PpkiA’’ + pC13F from the pSB1C3 backbone into homologous constructs for transformation into fungi, which would require several days to grow. Thus, we would not have enough time to collect data before the World Jamboree.</DIV><P>		We checked our last set of inserts ligated into pAg13H, and the constructs with pC13AT, pC13AV, and pC13CE appeared successful! The Gibson of pC13DU failed to grow, indicating a failure of our Gibson assembly. We planned to try one more time. 
		-Olya
</P></DIV></DIV><DIV class="entry nb-dry"><DIV><H6>October 26
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Arduino code was further improved.  In particular, closing the serial port was improved. 
		-Sara
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 27th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">We miniprepped two of each pC13AT, pC13AU (though unconfirmed in the colony PCR screen), pC13AV, and pC13CE in pAg13H, then linearized them to use in the final <I>C. heterosporus</I> transformation. There were no transformations after the enzyme osmoticum digestions. After dephosphorylating the backbone pAK13D, the carotenoids team ligated the digested product of ‘’crtEBI’’ (from pC13A) with the ‘’PT7’’ promoter in pAK13D.</DIV><P>		We attempted a final round of <I>C. heterosporus</I> protoplasting and transformation; however, we didn’t get any protoplasts. The carotenoids team ligated the digested product of pC13A with ‘’PT7’’ in pAK13D. 
		-Olya
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 28th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">The carotenoids team performed electroporation with <I>E. coli</I> for ‘’crtEBI’’ downstream of ‘’PT7’’ in the backbone of pAK13D. The transformants were grown on Amp plates overnight.</DIV><P>		We got successful <I>C. heterospous</I> transformants of pC13AT!! The carotenoids team transformed <I>E. coli</I> with ‘’crtEBI’’ in pAK13D. 
		-Kyle
</P></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Cornell/notebook">http://2013.igem.org/Team:Cornell/notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cornell/notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cornell/notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cornell/notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cornell/notebook&amp;oldid=360799" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>