<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_MIT_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:MIT/Results/">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:MIT/Results/&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:MIT/Results/&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:MIT/Results/&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:MIT/Results/" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:MIT/Results/</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="contentSub">(Redirected from <A href="/wiki/index.php?title=Team:MIT/Results&amp;redirect=no" title="Team:MIT/Results">Team:MIT/Results</A>)</DIV><P><UL class="navbar"><LI><A href="https://2011.igem.org/Team:MIT">HOME</A></LI><LI><A href="https://2011.igem.org/Team:MIT/Project/">PROJECT</A></LI><LI><A href="https://2011.igem.org/Team:MIT/Results/">DATA</A></LI><LI><A href="https://2011.igem.org/Team:MIT/Notebook/">NOTEBOOK</A></LI><LI><A href="https://2011.igem.org/Team:MIT/Tools/">MODEL</A></LI><LI><A href="https://2011.igem.org/Team:MIT/Partners/">PARTNERS</A></LI><LI><A href="https://2011.igem.org/Team:MIT/Team/Undergraduates">TEAM</A></LI><LI><A id="long" href="https://2011.igem.org/Team:MIT/HumanPractices/">HUMAN PRACTICES</A></LI></UL></P><DIV id="col_nav"><DIV class="col_list"><H2>Navigation</H2><UL><LI id="ov">Overview</LI><LI id="parts">BioBricks</LI><LI id="char">Transcription Factor Characterizations</LI><LI id="multi">Multi-input Logic Characterization</LI><LI id="patt">Patterning Modeling Results</LI><LI id="patte">Patterning Experimental Results</LI><LI id="adh">Mammalian Cell Adhesion Results</LI><LI id="deliv">DNA Delivery</LI><LI id="attr">Attributions</LI></UL></DIV></DIV><DIV id="col_left"><DIV class="content" id="ovcontent"><CENTER><B>Communication between engineered HEK Notch senders and CHO Delta receivers.</B> Top: fluorescent. Bottom: brightfield. Note that due to high red-fluorescence bleed-through in the imaging process, the red fluorescence does not delineate a regular cell's morphology. However, the activation effect of delta-expressing cells communicating with notch-expressing cells is shown by the yellow fluorescence of adjacent receiver cells.</CENTER><CENTER><B>Timelapse of CHO cells transfected with CMV:tTA and TRE:Delta-mCherry with dox</B></CENTER><P> The video is a timelapse of CHO cells transfected with CMV:tetR and CMV-TO:Delta-mCherry. At t=0, 100ug/mL dox is added to the media. The movie is a timelapse sped up such that one second of movie time is one hour of real time. </P>See detailed results and controls by navigating on the left!</DIV><DIV class="content" id="delivcontent"><H1>DNA Delivery Systems and Data Collection</H1><H2>Transfection Using Lipofectamine 2000 (Invitrogen)</H2>
To introduce our engineered genetic parts into mammalian cells, we employed the Lipofectamine 2000 reagent from Invitrogen, and obtained at best an 80% transfection efficiency for Hek293 cells and 10% transfection efficiency for CHO cells.

<H4>Figure 1: Hek293 Lipofectamine Transfection Results</H4><P>A</P><P>B</P><P>C</P><P>(1A) This scatter plot shows the distribution of the Hek293 population after it was transfected with the following DNA parts: Hef1a-LacO:eYFP and Hef1a:mKate, both of which were constitutively active, expressing yellow and red fluorescent proteins respectively. We observe a distinct shift of approximately 83% of the population in their eYFP fluorescence (FITC channel), while we observe a 69% shift in mKate fluorescence (PE-TexasRed channel). (1B) Histogram of signal intensities of the above data. (1C) The negative control sample was transfected with plasmids containing no functional promotor-gene pairs.

</P><P>We were then interested in the efficiency of multiple plasmid transfections. Previous work suggests that the probabilities of uptaking different plasmids is not independent. Cells that have been succesfully transfected with one plasmid are more likely to also have been transfected with another. To show this correlation, we decided to see what percentage of cells that took up the Hef1a:mKate plasmid also took up the Hef1a-LacO:eYFP plasmid. In order to evaluate the correlation, we took all events above the observed basal PE-TexasRed fluorescence level and examined their FITC profile. We observe in the below graph that <B>more than 99%</B> of the red population display yellow fluorescence above basal levels (2A). We make use of this high level of correlation later.
</P><H4>Figure 2: Hek293 Lipofectamine Multiplasmid Transfection Results</H4><P>A</P><P>B</P><P>C</P><P> Figure 2: Hek293 cells were transfected with equal amounts by weight of Hef1a:mKate and Hef1a-LacO:eYFP plasmids. 
(2A) Scatterplot displaying fluorescence levels of events as measured by flow cytometry. Events falling below the basal threshold for red fluorescence were omitted to observe &quot;red&quot; population. We observe that along the FITC axis, more than 99% of the red population fall above the threshold. (2B) Histogram of the same data. (2C) A three plasmid cotransfection.
 
</P><P>Many of our circuit designs involve multiple prmoter gene pairs which means multiple plasmid transfections. We run into the issue that we cannot be sure which cells received the plasmids of interest. From this excellent correlation we can be confident that most cells with one plasmid have received others. Throughout our project, we include constitutively fluorescence expressing plasmids in transfections, allowing us to gate our data based on this constitutive color. Although this method is far from perfect, it allows us to remove untransfected background from our data and obtain a reasonable profile of our tranfected population. 


</P><H4>Figure 3: CHO Lipofectamine Transfection Results</H4><P>A
</P><P>B
</P><P>C
</P><P>D
</P><P>Figure 3: In the flow cytometry scatter plot and histogram above we see the fluorescence distribution of a population of CHO cells transfected with Lipofectamine 2000 and Hef1a_eBFP2, a constitutive blue color. We note that CHOs naturally emit autofluorescence, as seen in the small tail pointing upwards in the DAPI (blue) channel. This tail accounts for 8% of the population, so subtracting this from the 14.5% we observe in the transfected cells, we obtain approximately 6.5% transfection efficiency.


</P><H2>Transfection By Nucleofection</H2>
Hek293 cells display good levels of transfection efficiencies with Lipofectamine 2000. However, we found during the summer that cadherins are endogenously expressed in Hek cells, and this limits their experimental flexibility when it comes to cell-cell adhesion. CHO cells, however, do not naturally express significant levels of cadherins. Seeing also that the Notch-Delta system was previously characterized by the Elowitz group using CHO cells, we decided to port our parts into CHO cells. As documented above (3A), however, we were unable to obtain acceptable efficiency with lipofectamine in CHO cells. We then investigated alternative methods of transient transfection.
<P> Nucleofection was suggested as a good alternative. We made us of the Lonza Nucleofection technology to transfect our CHO cells. The results are shown below: 

</P><P>The above shows the fluorescent images of CHO cells transfected with constitutive EYFP and constitutive GFP.</P></DIV><DIV class="content" id="partscontent"><H1>BioBricks</H1>
Total Parts Count: 68
<H2>Our Favorite Parts</H2><DIV class="part"><SPAN class="name"><B>Name:</B> Hef1a-LacOid MammoBlock</SPAN><SPAN class="id"><B>ID:</B><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511000">
      BBa_K511000
    </A></SPAN><SPAN class="type"><B>Type:</B> Regulatory</SPAN><SPAN class="length"><B>Length:</B> 1275</SPAN><DIV class="desc"><P>This part encodes a promoter with low-level, constitutive activity that can be repressed by variants of the LacI transcriptional repressor. Repression by LacI-KRAB through chromatin packing is quite effective.</P></DIV><B>Description/Usage:</B><DIV class="end">
  See Transcription Factor Characterizations for characterization!
  </DIV></DIV><DIV class="part"><SPAN class="name"><B>Name:</B> UAS-Gal4 Promoter MammoBlock</SPAN><SPAN class="id"><B>ID:</B><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511003">
      BBa_K511003
    </A></SPAN><SPAN class="type"><B>Type:</B> Regulatory</SPAN><SPAN class="length"><B>Length:</B> 220</SPAN><DIV class="desc"><P>This part encodes a promoter that is inducible by variants of the Gal4 transactivator and off otherwise.</P></DIV><B>Description/Usage:</B><DIV class="end">
  See Transcription Factor Characterizations for characterization!
  </DIV></DIV><DIV class="part"><SPAN class="name"><B>Name:</B> Delta-mCherry MammoBlock</SPAN><SPAN class="id"><B>ID:</B><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511302">
      BBa_K511302
    </A></SPAN><SPAN class="type"><B>Type:</B> Signaling</SPAN><SPAN class="length"><B>Length:</B> 2853</SPAN><DIV class="desc"><P>This part encodes a fusion between the Delta-3 ligand and the monomeric mCherry red fluorescent protein.</P></DIV><B>Description/Usage:</B><DIV class="end">
  See Multi-input Logic Characterization for characterization!
  </DIV></DIV><H2>Other Parts We Like</H2><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511001"><B>BBa_K511001</B></A> minCMV-1xCI434 Promoter MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511002"><B>BBa_K511002</B></A> minCMV-4xMnt Promoter MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511004"><B>BBa_K511004</B></A> TRE-Tight-LacOid Promoter MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511100"><B>BBa_K511100</B></A> EBFP2 MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511101"><B>BBa_K511101</B></A> EYFP-FF4x4 MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511102"><B>BBa_K511102</B></A> mKate MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511103"><B>BBa_K511103</B></A> amCyan-miRFF4 MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511200"><B>BBa_K511200</B></A> CI434-VP16 Transactivator MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511201"><B>BBa_K511201</B></A> Mnt-VP16 Transactivator MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511202"><B>BBa_K511202</B></A> rtTA-DD Transactivator MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511203"><B>BBa_K511203</B></A> rtTA3 Tetracycline-Inducible Transactivator MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511204"><B>BBa_K511204</B></A> LacI Repressor MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511205"><B>BBa_K511205</B></A> LacI-KRAB Repressor MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511206"><B>BBa_K511206</B></A> LacI-miRFF4 Repressor MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511207"><B>BBa_K511207</B></A> Gal4-VP16 Transactivator MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511300"><B>BBa_K511300</B></A> Notch-Gal4-ESN Juxtacrine Signaling Receptor MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511301"><B>BBa_K511301</B></A> Notch-Gal4-VP16 Juxtacrine Signaling Receptor MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511303"><B>BBa_K511303</B></A> Delta-mCherry-4xFF4 Juxtacrine Signaling Ligand MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511400"><B>BBa_K511400</B></A> AVPR2-TEVs-Gal4-VP16 Signaling Receptor MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511401"><B>BBa_K511401</B></A> HRH4-TEVs-GV16 Signaling Receptor MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511500"><B>BBa_K511500</B></A> pDisplay-Vasopressin-MYC Ligand MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511600"><B>BBa_K511600</B></A> N-Cadherin-2A-EYFP Cell Adhesion MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511700"><B>BBa_K511700</B></A> TEV Protease MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511701"><B>BBa_K511701</B></A> B-Arrestin-2-TEV-Protease MammoBlock</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511800"><B>BBa_K511800</B></A> Blue Fluorescent Protein Generator (Hef1a-EBFP2) MammoBlock Device</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511801"><B>BBa_K511801</B></A> Red Fluorescent Protein Generator (Hef1a-mKate) MammoBlock Device</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511802"><B>BBa_K511802</B></A> Red Fluorescent Delta Ligand Generator (Hef1a-Delta-mCherry) MammoBlock Device</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511803"><B>BBa_K511803</B></A> Repressible Fluorescence/Repression Generator (Hef1a-LacO-AmCyan-miRFF4) MammoBlock Device</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511804"><B>BBa_K511804</B></A> Repressible Enhanced Blue Fluorescent Protein Generator (Hef1a-LacO-EBFP2) MammoBlock Device</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511805"><B>BBa_K511805</B></A> Repressible Yellow Fluorescence Generator (Hef1a-LacO-EYFP-4xFF4) MammoBlock Device</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511806"><B>BBa_K511806</B></A> Repressible Red Fluorescent Ligand Generator (Hef1a-LacO-Delta-mCherry) MammoBlock Device</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511807"><B>BBa_K511807</B></A> Repressible Red Fluorescent Protein Generator (Hef1a-LacO-mKate) MammoBlock Device</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511808"><B>BBa_K511808</B></A> Repressible Yellow Fluorescent Protein Generator (Hef1a-LacO-EYFP) MammoBlock Device</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511809"><B>BBa_K511809</B></A> Repressible Tango Ligand Generator (Hef1a-LacO-pDisplay-Vasopressin-MYC) MammoBlock Device</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511810"><B>BBa_K511810</B></A> Repressible Tango Receptor Generator (Hef1a-LacO-AVPR2-TEVs-Gal4-VP16) MammoBlock Device</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511811"><B>BBa_K511811</B></A> Notch-Gal4-ESN Juxtacrine Receptor Generator (CMV-Notch-Gal4-ESN) MammoBlock Device</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511812"><B>BBa_K511812</B></A> CI434-VP16-Inducible Red Fluorescence Generator (minCMV-1xCI434-mKate) MammoBlock Device</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511813"><B>BBa_K511813</B></A> Mnt-VP16-Inducible Red Fluorescence Generator (minCMV-4xMnt-mKate) MammoBlock Device</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511814"><B>BBa_K511814</B></A> Inducible Red Fluorescent Ligand Generator (TRE-Tight-Delta-mCherry) MammoBlock Device</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511815"><B>BBa_K511815</B></A> Inducible Red Fluorescent Protein Generator (TRE-Tight-mKate) MammoBlock Device</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511816"><B>BBa_K511816</B></A>Inducible Blue Fluorescent Protein Generator (TRE-Tight-EBFP2) MammoBlock Device</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511817"><B>BBa_K511817</B></A> Inducible Yellow Fluorescent Protein Generator (TRE-Tight-EYFP-FF4x4) MammoBlock Device</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511818"><B>BBa_K511818</B></A> Inducible Notch-Gal4-ESN Receptor Generator (TRE-Tight-Notch-Gal4-ESN) MammoBlock Device</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511819"><B>BBa_K511819</B></A> Inducible Yellow Fluorescent Protein Generator (UAS-Gal4-Citrine) MammoBlock Device	</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511820"><B>BBa_K511820</B></A> Inducible Blue Fluorescent Protein Generator (UAS-Gal4-EBFP2) MammoBlock Device</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511821"><B>BBa_K511821</B></A> Inducible Fluorescence/Repression Generator (UAS-Gal4-amCyan-miRFF4) MammoBlock Device	</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511822"><B>BBa_K511822</B></A> Inducible Yellow Fluorescent Protein Generator (UAS-Gal4-EYFP-FF4x4) MammoBlock Device</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511823"><B>BBa_K511823</B></A> Inducible Rd Fluorescent Protein Generator (UAS-Gal4-mKate) MammoBlock Device	</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511824"><B>BBa_K511824</B></A> Inducible Red Fluorescent Ligand Generator (UAS-Gal4-Delta-mCherry) MammoBlock Device	</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511825"><B>BBa_K511825</B></A>Inducible Notch-Gal4-ESN Receptor Generator (UAS-Gal4-Notch-Gal4-ESN) MammoBlock Device	</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511826"><B>BBa_K511826</B></A> Inducible N-Cadherin-2A-EYFP Generator (UAS-Gal4-N-Cadherin-2A-EYFP) MammoBlock Device	</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511900"><B>BBa_K511900</B></A> Gal4-VP16 Transactivator Generator (Hef1a-Gal4-VP16) MammoBlock Device</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511901"><B>BBa_K511901</B></A> LacI Repressor Generator (Hef1a-LacI) MammoBlock Device	</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511902"><B>BBa_K511902</B></A> rtTA3 Transactivator Generator (Hef1a-rtTA3) MammoBlock Device	</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511903"><B>BBa_K511903</B></A> rtTA-DD Transactivator Generator (Hef1a-rtTA-DD) MammoBlock Device	</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511904"><B>BBa_K511904</B></A> Repressible LacI Repressor Generator (Hef1a-LacO-LacI) MammoBlock Device	</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511905"><B>BBa_K511905</B></A> Repressible rtTA3 Transactivator Generator (Hef1a-LacO-rtTA3) MammoBlock Device	</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511906"><B>BBa_K511906</B></A> Inducible CI434-VP16 Transactivator Generator (TRE-Tight-CI434-VP16) MammoBlock Device	</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511907"><B>BBa_K511907</B></A> Inducible Mnt-VP16 Transactivator Generator (TRE-Tight-Mnt-VP16) MammoBlock Device	</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511908"><B>BBa_K511908</B></A> Inducible Gal4-VP16 Transactivator Generator (TRE-Tight-Gal4-VP16) MammoBlock Device	</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511909"><B>BBa_K511909</B></A> Inducible LacI Repressor Generator (TRE-Tight-LacI) MammoBlock Device	</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511910"><B>BBa_K511910</B></A> Inducible LacI-KRAB Repressor Generator (TRE-Tight-LacI-KRAB) MammoBlock Device</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511911"><B>BBa_K511911</B></A> Inducible rtTA Transactivator Generator (TRE-Tight-rtTA3) MammoBlock Device	</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511912"><B>BBa_K511912</B></A> Inducible LacI Repressor Generator (UAS-Gal4-LacI) MammoBlock Device	</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511913"><B>BBa_K511913</B></A> Inducible LacI Repressor Generator (UAS-Gal4-LacI-KRAB) MammoBlock Device	</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511914"><B>BBa_K511914</B></A> Inducible rtTA3 Transactivator Generator (UAS-Gal4-rtTA3) MammoBlock Device	</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K511915"><B>BBa_K511915</B></A> Inducible rtTA-DD Transactivator Generator (UAS-Gal4-rtTA-DD) MammoBlock Device	</P></DIV><DIV class="content" id="charcontent"><H1>Transcription Factor Characterizations</H1><P>The characterization of newly constructed biological parts as described below.</P><H2>List of characterizations</H2><UL><LI><A href="#tre">rtTA3/TRE promoter</A></LI><LI><A href="#gal4">Gal4/UAS promoter</A></LI><LI><A href="#laci">LacI/Hef1a-LacO repressor</A></LI><LI><A href="#cmv-to">CMV-TetO promoter</A></LI><LI><A href="#mnt">Mnt-VP16/Mnt promoter</A></LI><LI><A href="#ci434">CI434-VP16/CI434 promoter</A></LI></UL><H3><A id="tre">rtTA3/TRE promoter</A></H3><H4>Explanation:</H4><P>The rtTA3 system relies on two elements: the rtTA3 gene which encodes an activator and the TRE promoter which rtTA3 binds to. Upon doxcycline binding to rtTA3, it is able to to bind to TRE and activate expression of downstream genes. In this particular experiment, we transfected Hek293 cells with 266 ng each of Hef1a:eBFP2, Hef1a:rtTA3, and TRE:delta-mCherry constructs. Cells were then induced with varying levels of doxycycline for 48 hours before FACS. Because we generally observe &gt;50% transfection efficiency with Lipofectamine in Hek293 cells, we took only the top 50% of cells by TxRed signal for the displayed data. We display the mean TxRed intensity for the top 50% of events verus doxycycline concentration. Control cells were given the same set of DNA but without Hef1a:rtTA3.</P><P> We can clearly observe a steep increase in red signal which levels off at dox levels above 0.5 ug/mL. Meanwhile the control cells remain at baseline, showing that doxycyline itself does not affect fluoresence levels.</P><P> The video is a timelapse of CHO cells transfected with CMV:tTA and TRE:Delta-mCherry. At t=0, 100ug/mL dox is added to the media. The movie is a timelapse sped up such that one second of movie time is one hour of real time. 

</P><H3><A id="gal4">Gal4/UAS promoter</A></H3><H4>Explanation:</H4><P>Another activator promoter pair we make use of is the Gal4-UAS system. Gal4VP16 is an activator capable of binding to UAS promoter site and activating expression of downstream genes. Here we characterize this interaction with a set of transfections. Cells were transfected with UAS:(mKate, eYFP-FF4, eBFP2, or H2B-citrine) and Hef1a:GV16. Controls lack the Hef1a:GV16 plasmid. The above graph displays mean fluoresence in the denoted channels. </P><P>As can be clearly seen, Hef1a:GV16 results in significant activation of the UAS promoter. This results in the observed increases in mean fluoresence compared to control populations. With the exception of UAS:eBFP2, we see more than 20-fold increase in mean fluorescence.  

</P><H3><A id="laci">LacI/Hef1a-LacO repressor</A></H3><H4>Explanation:</H4><P>In order to implement inhibition into our circuit designs, we made use of the lacO/lacI system. lacI binds a lacO site and can inhibit transcription of downstream genes. This inhibition can be relieved by binding lacI with IPTG. In order to test this part, we constructed a Hef1a-lacO promoter which is normally constitutively active until inhibition by lacI. 
</P><P>
As can be seen in the figure above, we see significant inhibition of eYFP expression with addition of a Hef1a:lacI construct (around 20 fold decrease). Finally, as we increase IPTG levels, we see mean yellow fluorescence increase as lacI inhibition is relieved. We also incorporate our UAS/Gal4 system by putting lacI expression under control of UAS, thus giving us a different constitutive level of lacI production. Again we observe inhibition of lacO by lacI relieved by IPTG.
</P><H3><A id="mnt">Mnt-VP16/minCMV-4xMnt promoter</A></H3><H4>Explanation:</H4><P>Mnt is similar in role to Gal4-VP16. It binds to Mnt sites and can activate expression of downstream genes. We made a minCMV-4xMnt promoter upstream of the mKate gene. Mnt presence should activate expression of mKate and thus an increase in red fluorescence. We transfected Hek293 cells with Hef1a:rtTA3, TRE:Mnt-VP16, and minCMV-4xMnt:mKate. 
</P><P>From the data shown, we can clearly see an increase in mean red fluorescence upon dox addition 

</P><H3><A id="ci434">CI434-VP16/minCMV-1xCI434 promoter</A></H3><H4>Explanation:</H4><P>CI434 is another transcriptional activator that we characterized. In order to test it, we transfected Hek293 cells with Hef1a:rtTA3, TRE:CI434-VP16, and minCMV-1xCI434:mKate. This allowed us to use dox induction to activate expression of CI434 which in turn should activate mKate expression. 
</P><P>
The results above show a clear increase in red fluorescent signal upon dox addition. We observe around a 1.25 fold increase. We believe this is an underestimate of the actual activation strength due to lack of gating and low transfection efficiency for this experiment.
</P></DIV><DIV class="content" id="multicontent"><H1>Multi-input Logic Characterization</H1><P>Here we show an experimental result of an example of what we call our internal processing module, which is essentially a 2 input chemical AND gate. Taking the individual functional transcription factor systems characterized above, we integrated them into an AND gate with two inputs: IPTG and doxycycline levels. The system integrates the UAS/GV16, rtTA3/TRE, and lacI/lacO systems described above. As discussed earlier, IPTG can bind to lacI and relieve its repression of lacO. Thus in this particular circuit, IPTG relieves inhibition of rtTA3 expression. Then, only in the presence of doxycycline can rtTA3 activate expression of the TRE controlled reporter. Therefore, we require both IPTG and dox to achieve expression of our final Delta-mCherry reporter.
</P><P>The data show the desired 2 input AND behavior. We see maximum Delta-mCherry expression at the highest levels of both dox and IPTG. We also observe significant activation at high dox levels even in the absence of IPTG. This suggests some leakiness in the lacI inhibition, allowing for some rtTA3 to be produced and activated by dox. This behavior can be addressed using a different inhibitor such as lacI-KRAB which exhibits a much stronger repression.</P><H3><A id="cascade">2-input AND Gate: Gal4-VP16âLacIÂ¬rtTA3</A></H3><P>The above graph displays flow cytometry measuring mCherry fluorescence of transfected Hek293 cells in the presence of different IPTG and doxycycline concentrations. We observe highest reporter levels at max IPTG and dox concentrations. Meanwhile, we see decreasing levels of reporter expression as we decrease either dox or IPTG levels. Note the high background, especially at high dox levels due to leakiness of lacI inhibition. </P></DIV><DIV class="content" id="pattcontent"><H1>Patterning Modeling Results</H1><P>All simulations were done using our modeling framework, <EM><A href="http://igem.mit.edu/mcell.zip">mcell</A></EM>. For more information on <EM>mcell</EM>, please look in the Tools section. Some sample runs can be downloaded from here: <A href="http://igem.mit.edu/simulations.zip">here</A>.

</P><P>Most of our modeling was done on the following circuit:</P><P>MODERATE is a promoter that has moderate activity under the experimental conditions; for instance, TRE without any rtTA3 in the system would work. All cells contain this circuit; the values of all proteins are 0 at start. </P><P>We followed the model of Delta-Notch interaction proposed by Elowitz[1]. In it, Delta and Notch in the same cell cis-inhibit at a rate proportional to their amounts; Delta in one cell can bind to Notch in another cell (at a rate proportional to the available Notch and Delta), destroying both proteins and releasing the Notch Intracellular Domain (NICD), which then goes on bind to a promoter and stimulated production of genes.</P><P>Applying this model, along with standard mass action to model protein binding events and Hill equations to model promoter stimulation (along with the standard set of simplifying assumptions that go along with these methods), we arrive at the following series of differential equations:
<PRE>
d[NCis]/dt = n_prod - k_cis*[NCis]*[DCis] - k_trans*[NCis]*[DTrans]
d[DCis]/dt = d_min + d_prod/(1+[LacI]*b_L) - k_cis*[NCis]*[DCis]
                                           - k_trans*[NTrans]*[DCis]
d[NICD]/dt = k_trans*[NCis]*[DTrans]
d[Reporter]/dt = d[NCAD]/dt = d[LacI]/dt = V_max*(Gal4^h)/(k_d+Gal4^h)
</PRE>
DTrans is the amount of Delta in neighboring cells touching the current cell; likewise, NTrans is the amount of Notch in neighboring cells touching the current cell. We assumed that Delta and Notch are all located on the cell surface and are evenly distributed. NCis is the amount of Notch inside the given cell; DCis is the amount of Delta inside the given cell. The values for V_max, h, and k_d were taken from [1]; for the simulations shown on this page, n_prod = 5, d_min = 1, d_prod = 9 (so that the Delta production rate varies between 1 and 10); k_cis = 1, k_trans = 0.04, and b_L = 10.

</P><H2>Bimodality in HEK cells</H2><P>Running this circuit in HEK cells (modeled by setting the energy of contact between cells in CompuCell3D to -10.0 and initializing the cells with no gaps) produces stunning bimodality.</P><H3>Video</H3><H3>Analysis</H3><P>What happens here is that, initially, all the cells develop a large amount of Delta. However, some Notch still remains in the cells, and is strongly activated; Delta production rates begin to drop because of the LacI. Some cells gain significant amounts of Notch before others, due to the random movements of the cells. Once a cell gains a significant amount of Notch, it's Delta level drops off sharply, and it ceases to activate cells in its vincinity. Thus, once a cell is fully &quot;activated&quot;, other cells around it are much less likely to be. The stable state of such dynamics is a mosaic pattern, in which single cells are activated and are surrounded by a sea of inactivated cells.</P><H2>But what if they are not sticky?</H2><P>There is a very large discrepancy between the behavior of the circuit in HEK cells in CHO cells, which are modeled with no adhesion and are initialized with gaps in between them. In CHO cells, the following ensues:
</P><H3>Movie</H3><H3>Analysis</H3>
Bimodality almost completely drops off in this model. This is clear from the video; looking at the histograms, we can see that only an extremely small hump develops at the end in place of where the hump for the activated mode should be. The cells drift around randomly, most of the far away from others and thus inactivated; sometimes they bump in to one another, and a cell or two begins the road to activation, but they never stay together for long enough for the dominance of the activated cell to be established and for bimodality to be enforced.

<H2>Adding in NCAD</H2><P>Now, we add in the NCad. Using the AdhesionFlex plugin, we set the adhesion strength of NCad to 1 (modifying this within reasonable bounds does not really change things much) and set the amount of NCad in CompuCell3D to equal the amount of Reporter in our BionetSolver circuit. This created the following results:
</P><H3>Movie</H3><H3>Analysis</H3><P>This circuit never really settles down into any clear behavior. Looking at the histograms, we can see that there is a fairly large gap between most &quot;activated&quot; and &quot;non-activated&quot; cells, but the &quot;activated&quot; cells have a much wider range of Reporter values and the range drifts over time. Looking at the video, one can see certain cells that end up always being unactivated, certain cells that are always activated, and certain cells with a moderate Reporter level that oscillates in a complicated fashion. What seems to be happening is that as a cell randomly becomes activated, it causes other cells to stick to it; this creates enough cells in close proximity to one another to cause more cells to activate, which weakens the aggregate structure, causing the cells to start drifting apart. Analysis of this model is ongoing. We tried clustering the cells, but standard algorithms did not do a very good job in clustering cells into groups that share the same behavior. This may be because cells seem to change behaviors -- a cell that looks like it is always off can, in the middle of the simulation, become an always on cell.
</P><P>[1] Sprinzak, et al. Cis-interactions between Notch and Delta generate mutually exclusive signalling states. <EM>Nature</EM> 465:86-90</P></DIV><DIV class="content" id="pattecontent"><H1>Patterning Experimental Results</H1>

One of the key modules in our self-organizing cell project is intercellular communication. We achieved this through the use of the Notch/Delta signalling system. As described under the Project page, Notch/Delta is a cell-cell contact based signalling system with Delta acting as the sending ligand and Notch as the receiver.
The Notch we used was a fusion of the Notch extracellular domain with an intracellular Gal4 domain. Upon Delta binding, the intracellular domain is cleaved and Gal4 can migrate to the nucleus and activate UAS. In the following experiments, we tested out this interaction 
----

<P><B>TO-DMC (Sender) cells</B></P><P>CHO T-REx stably integrated with: pcDNA5/TOâDeltamcherry
</P><P>Purpose: These cells normally inhibit expression of Delta-mCherry. However, doxycyline can relieve this repression and start expression of Delta. These cells can only send using their Delta ligand, so we termed them &quot;senders&quot;.

</P><P><B>hN1G4esnâNoâDelta (Receiver Type)</B></P><P>CHO T-REx stably integrated with: CMV-Cerulean, UAS-H2B-Citrine, pcDNA3âhNECDâGal4esn
</P><P>Purpose: These cells express Notch constitutively and have a Histone-Citrine fusion under UAS control as the readout. We termed these cells as receivers.

</P><P><B>hN1G4esn (Cis-inhibitor Type)</B></P><P>CHO T-REx stably integrated with: CMV-Cerulean, UAS-H2B-Citrine, pcDNA3âhNECDâGal4esn, pcDNA5/TOâDeltamcherry
</P><P>Purpose: These cells express Notch constitutively and inhibit Delta expression in the absence of doxycycline. In this state, they are effectively receiver type. However, as dox levels increase, we see more and more Delta expression and due to cis-inhibition, we see a switch to sender state. Because these cells allow us to observe and modulate cis-inhibition, we termed them &quot;cis-inhibitor type&quot;.


</P><H4>Explanation:</H4><P>We decided to test our Notch/Delta constructs by performing a co-culture experiment. We transfected our Hef1a:Delta-mCherrry construct into Hek293 cells. We mixed these Hek sender cells with receiver CHO cells in the specified ratios. The results of flow cytometry are displayed above.
</P><P> From the microscopy images above, we can see clear expression of Delta-mCherry by our HEK cells. In addition, we can see activation of Notch on adjacent CHO receiver cells as seen by the presence of Citrine (yellow signal). This spatial correlation is strong evidence showing Notch activation by Delta. </P><P>The above bar chart displays average FITC signal of our cells as determined by flow cytometry. We can see an increase in fluorescence in the test samples compared to the controls. This suggests that our Delta construct is also capable of signaling to Notch.</P></DIV><DIV class="content" id="adhcontent"><H1>Mammalian Cell Adhesion Results</H1><P>Cadherins are a medically important calcium-dependent surface protein that can bind to same-type cadherins on other cells. The existence of different types of cadherins allows for the development of highly organized tissues, as the differential expression of different types of cadherins in various cells drives the development of complex three-dimensional structures. </P><P>We incorporated cadherins into our existing characterized signaling work, with the goal of achieving control of cadherin expression in mammalian cells. By integrating cadherins into our pattern-formation mechanism, we will be able to create patterns based on differential adhesion strengths between cells.</P><CENTER>Constitutive expression of NCad-eGFP.</CENTER><P>The construct shown above uses a Ncadherin eGFP fusion protein that allows us to visualize localization of Ncadherin. Using confocal microscopy, we were able to observe membrane localization of our Ncadherin protein as expected. Additionally, we can also observe very bright signal at the cell interfaces, showing clustering of Ncad at these interfaces and thus adhesion.</P><CENTER>Constitutive expression of eGFP.</CENTER><P>Meanwhile, in our control, we transfected a constitutive GFP construct. We can see diffuse cytoplasmic localization versus membrane localization. </P><P> This series of experiments show that our Ncad construct does display expected behavior and provides a promising start to incorporating adhesion into our designs. We observe membrane localization and sticking which is what we were looking for.</P></DIV><DIV class="content" id="attrcontent"><H1>Attributions</H1><P> Our instructors were very helpful not only in giving feedback on our designs, cloning strategies, and data, but also in training us for lab work.
The training for tissue culture work was conducted by <B>Linda Stockdale</B> in the Griffiths lab.
Gibson assembly techniques and FACS training from <B>Deepak Mishra</B>, one of our instructors. 
<B>Jonathan Babb</B>, another one of our instructors helped us with setting up the robot. 
</P><P>Other than the initial training, all programming, DNA, and transfection work was done by our undergraduate team. </P><P>
Special credit belongs to <B>Semon Rezchikov</B> for simulations and modeling, <B>Jenny Cheng</B> for animations, and <B>Tiffany Huang</B> and <B>Jenny Cheng</B> for wiki design.
</P><P>Special thanks to <B>Michael Elowitz</B> for generously providing not only the CMV:hN1ECD-Gal4esn, CMV-TO:Delta-mCherry, and UAS:H2B-Citrine constructs, but also the stably integrated cell lines. We could not have implemented the Notch-Delta system without these valuable contributions.
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:MIT/Results/">http://2011.igem.org/Team:MIT/Results/</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:MIT/Results/" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:MIT/Results/" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:MIT/Results/&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:MIT/Results/&amp;oldid=262548" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>