<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_MIT_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:MIT/Notebook/">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:MIT/Notebook/&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:MIT/Notebook/&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:MIT/Notebook/&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:MIT/Notebook/" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:MIT/Notebook/</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><UL class="navbar"><LI><A href="https://2011.igem.org/Team:MIT">HOME</A></LI><LI><A href="https://2011.igem.org/Team:MIT/Project/">PROJECT</A></LI><LI><A href="https://2011.igem.org/Team:MIT/Results/">DATA</A></LI><LI><A href="https://2011.igem.org/Team:MIT/Notebook/">NOTEBOOK</A></LI><LI><A href="https://2011.igem.org/Team:MIT/Tools/">MODEL</A></LI><LI><A href="https://2011.igem.org/Team:MIT/Partners/">PARTNERS</A></LI><LI><A href="https://2011.igem.org/Team:MIT/Team/Undergraduates">TEAM</A></LI><LI><A id="long" href="https://2011.igem.org/Team:MIT/HumanPractices/">HUMAN PRACTICES</A></LI></UL></P><DIV id="col_nav"><DIV class="col_list"><H2>Navigation</H2><UL><LI id="ov">Overview</LI><LI id="w1">Week 1</LI><LI id="w2">Week 2</LI><LI id="w3">Week 3</LI><LI id="w4">Week 4</LI><LI id="w5">Week 5</LI><LI id="w6">Week 6</LI><LI id="w7">Week 7</LI><LI id="w8">Week 8</LI><LI id="w9">Week 9</LI><LI id="w10">Week 10</LI><LI id="w11">Week 11</LI><LI id="w12">Week 12</LI></UL></DIV></DIV><DIV id="col_left"><DIV class="content" id="ovcontent"><H1><SPAN class="mw-headline" id="Overview"> Overview </SPAN></H1><P>We put a lot of hard work into our project. Here is a weekly summary of what we accomplished in our research over summer. Our full team of 13 students, including one high school students and one undergraduate from another university, worked throughput the summer, while a smaller group of 6 students continued beyond August as UROPs.
</P></DIV><DIV id="col_left"><DIV class="content" id="w1content"><H1>Summary of Week 1: June 6 - June 11</H1>
In the first week we began constructing basic reporter DNA constructs that have various combinations of promoters and genes. The mix-and-matching of promoters and genes was done using the LR reaction following the Gateway protocol. Our promoters included TRE, minCMV, Hef1a, Hefl1a-LacO, and our genes included eYFP, mKate, and eBFP2. Transformed and inoculated colonies from the LR reactions and performed restriction digests. Approximately half of the LR reactions were successful and became DNA parts that we could use later on.

<H3>June 7, 2011</H3><DIV class="table-wrap"><TABLE id="minimalist"><TBODY><TR><TH class="confluenceTh"> Date</TH><TH class="confluenceTh"> Assignee </TH><TH class="confluenceTh"> DEST_R4R2 </TH><TH class="confluenceTh"> L4-R1 Promoter </TH><TH class="confluenceTh"> L1-L2 
Gene </TH><TH class="confluenceTh"> LR? </TH><TH class="confluenceTh"> Tube Number </TH><TH class="confluenceTh"> Colony Number, 
Antibiotic, Person </TH><TH class="confluenceTh"> Protocol Used </TH></TR><TR><TD class="confluenceTd"> 6.7.2011 </TD><TD class="confluenceTd"> Michelle 
Louis </TD><TD class="confluenceTd"> 2-3 </TD><TD class="confluenceTd"> Tre </TD><TD class="confluenceTd"> C1434VP16 </TD><TD class="confluenceTd"> Yes. 6.7.2011, 5pm </TD><TD class="confluenceTd"> 11 </TD><TD class="confluenceTd"> 5, AMP, KH </TD><TD class="confluenceTd"> Standard, 30C for 16h </TD></TR><TR><TD class="confluenceTd"> 6.7.2011 </TD><TD class="confluenceTd"> Michelle 
Louis </TD><TD class="confluenceTd"> 2-3 </TD><TD class="confluenceTd"> Tre </TD><TD class="confluenceTd"> Mnt1VP16 </TD><TD class="confluenceTd"> Yes. 6.7.2011, 5pm </TD><TD class="confluenceTd"> 12 </TD><TD class="confluenceTd"> 4, AMP, KH </TD><TD class="confluenceTd"> Standard, 30C for 16h </TD></TR><TR><TD class="confluenceTd"> 6.7.2011 </TD><TD class="confluenceTd"> Tyler </TD><TD class="confluenceTd"> 2-3 </TD><TD class="confluenceTd"> Tre </TD><TD class="confluenceTd"> LacIKrab </TD><TD class="confluenceTd"> Yes. 6.7.2011, 5pm </TD><TD class="confluenceTd"> 6 </TD><TD class="confluenceTd"> 30, AMP, KH </TD><TD class="confluenceTd"> Standard, 30C for 16h </TD></TR><TR><TD class="confluenceTd"> 6.7.2011 </TD><TD class="confluenceTd"> Jenny 
Divya </TD><TD class="confluenceTd"> 3-4 </TD><TD class="confluenceTd"> minCMV-7xMnt1 </TD><TD class="confluenceTd"> eYFP </TD><TD class="confluenceTd"> Yes. 6.7.2011, 5pm </TD><TD class="confluenceTd"> 9 </TD><TD class="confluenceTd"> 0, AMP, KH, needs to be redone </TD><TD class="confluenceTd"> Standard, 30C for 16h </TD></TR><TR><TD class="confluenceTd"> 6.7.2011 </TD><TD class="confluenceTd"> Jenny 
Divya </TD><TD class="confluenceTd"> 2-3 </TD><TD class="confluenceTd"> Tre </TD><TD class="confluenceTd"> LexAVP16 </TD><TD class="confluenceTd"> Yes. 6.7.2011, 5pm </TD><TD class="confluenceTd"> 10 </TD><TD class="confluenceTd"> 2, AMP, KH, needs to be redone </TD><TD class="confluenceTd"> Standard, 30C for 16h </TD></TR><TR><TD class="confluenceTd"> 6.7.2011 </TD><TD class="confluenceTd"> Mariola 
Kenneth </TD><TD class="confluenceTd"> 3-4 </TD><TD class="confluenceTd"> minCMV 1xC1434 </TD><TD class="confluenceTd"> eYFP </TD><TD class="confluenceTd"> Yes. 6.7.2011, 5pm </TD><TD class="confluenceTd"> 7 </TD><TD class="confluenceTd"> 3, AMP, KH, needs to be redone </TD><TD class="confluenceTd"> Standard, 30C for 16h </TD></TR><TR><TD class="confluenceTd"> 6.7.2011 </TD><TD class="confluenceTd"> Mariola 
Kenneth </TD><TD class="confluenceTd"> 3-4 </TD><TD class="confluenceTd"> minCMV 4xLexA </TD><TD class="confluenceTd"> eYFP </TD><TD class="confluenceTd"> Yes. 6.7.2011, 5pm </TD><TD class="confluenceTd"> 8 </TD><TD class="confluenceTd"> 0, AMP, KH, needs to be redone </TD><TD class="confluenceTd"> Standard, 30C for 16h </TD></TR></TBODY></TABLE></DIV><P>LR Reactions/p&gt;



</P><DIV class="table-wrap"><TABLE id="minimalist"><TBODY><TR><TH class="confluenceTh"> Date </TH><TH class="confluenceTh"> Assignee </TH><TH class="confluenceTh"> DEST_R4R2 </TH><TH class="confluenceTh"> L4-R1 Promoter </TH><TH class="confluenceTh"> L1-L2 
Gene </TH><TH class="confluenceTh"> LR? </TH><TH class="confluenceTh"> Tube Number </TH><TH class="confluenceTh"> Colony Number, 
Antibiotic, Person </TH><TH class="confluenceTh"> Protocol Used </TH></TR><TR><TD class="confluenceTd"> 6.9.2011 </TD><TD class="confluenceTd"> Grant </TD><TD class="confluenceTd"> 3-4 </TD><TD class="confluenceTd"> minCMV 4xLexA </TD><TD class="confluenceTd"> eYFP </TD><TD class="confluenceTd"> Yes. 6.9.2011, 3pm </TD><TD class="confluenceTd"> 12 </TD><TD class="confluenceTd"> 10, AMP, GR  </TD><TD class="confluenceTd"> Standard + Grown at 37C in LB not SOC. </TD></TR><TR><TD class="confluenceTd"> 6.9.2011 </TD><TD class="confluenceTd"> Jon C </TD><TD class="confluenceTd"> 3-4 </TD><TD class="confluenceTd"> minCMV-4xMnt1 </TD><TD class="confluenceTd"> eYFP </TD><TD class="confluenceTd"> Yes. 6.9.2011, 3:45pm </TD><TD class="confluenceTd"> 2 </TD><TD class="confluenceTd"> 13, AMP, GR </TD><TD class="confluenceTd"> Standard + Grown at 37C in LB not SOC. </TD></TR><TR><TD class="confluenceTd"> 6.9.2011 </TD><TD class="confluenceTd"> Jon C </TD><TD class="confluenceTd"> 3-4 </TD><TD class="confluenceTd"> minCMV-4xCI434 </TD><TD class="confluenceTd"> eYFP </TD><TD class="confluenceTd"> Yes. 6.9.2011, 3:45pm </TD><TD class="confluenceTd"> 1 </TD><TD class="confluenceTd"> 7, AMP, GR </TD><TD class="confluenceTd"> Standard + Grown at 37C in LB not SOC. </TD></TR><TR><TD class="confluenceTd"> 6.9.2011 </TD><TD class="confluenceTd"> Jon C </TD><TD class="confluenceTd"> 3-4 </TD><TD class="confluenceTd"> Hef1a-LacO </TD><TD class="confluenceTd"> eYFP </TD><TD class="confluenceTd"> Yes, 6.9.2011, 4:20pm </TD><TD class="confluenceTd"> 5 </TD><TD class="confluenceTd"> 25, AMP, GR </TD><TD class="confluenceTd"> Standard + Grown at 37C in LB not SOC. </TD></TR><TR><TD class="confluenceTd"> 6.9.2011 </TD><TD class="confluenceTd"> Semon </TD><TD class="confluenceTd"> 4-5 </TD><TD class="confluenceTd"> Tre </TD><TD class="confluenceTd"> mKate </TD><TD class="confluenceTd"> Yes, 6.9.2011 4:27pm </TD><TD class="confluenceTd"> 3 </TD><TD class="confluenceTd"> 6, AMP, GR </TD><TD class="confluenceTd"> Standard + Grown at 37C in LB not SOC. </TD></TR><TR><TD class="confluenceTd"> 6.9.2011 </TD><TD class="confluenceTd"> Semon </TD><TD class="confluenceTd"> 1-2 </TD><TD class="confluenceTd"> Hef1a </TD><TD class="confluenceTd"> eBFP2 </TD><TD class="confluenceTd"> Yes, 6.9.2011, 4.27pm </TD><TD class="confluenceTd"> 4 </TD><TD class="confluenceTd"> 3, AMP, GR </TD><TD class="confluenceTd"> Standard + Grown at 37C in LB not SOC. </TD></TR></TBODY></TABLE></DIV><P>Transformation
</P><DIV class="table-wrap"><TABLE id="minimalist"><TBODY><TR><TH class="confluenceTh"> Date </TH><TH class="confluenceTh"> Assignee </TH><TH class="confluenceTh"> DEST_R4R2 </TH><TH class="confluenceTh"> L4-R1 Promoter </TH><TH class="confluenceTh"> L1-L2 
Gene </TH><TH class="confluenceTh"> LR? </TH><TH class="confluenceTh"> </TH><TH class="confluenceTh"> Colony Number, 
Antibiotic </TH><TH class="confluenceTh"> Protocol Used </TH></TR><TR><TD class="confluenceTd"> 6.9.2011 </TD><TD class="confluenceTd"> Mariola </TD><TD class="confluenceTd"> eYFP </TD><TD class="confluenceTd"> no. Replication stock. </TD><TD class="confluenceTd"> </TD><TD class="confluenceTd"> 2, AMP </TD><TD class="confluenceTd"> Transformed using LB instead of SOC </TD></TR></TBODY></TABLE></DIV><P>Miniprep</P><DIV class="table-wrap"><TABLE id="minimalist"><TBODY><TR><TH class="confluenceTh"> Date </TH><TH class="confluenceTh"> Assignee </TH><TH class="confluenceTh"> DNA </TH><TH class="confluenceTh"> Quantity </TH><TH class="confluenceTh"> Time collected </TH></TR><TR><TD class="confluenceTd"> 6.9.2011 </TD><TD class="confluenceTd"> Divya-Jenny </TD><TD class="confluenceTd"> pEXPR_2-3_Tre:LexAVP16 </TD><TD class="confluenceTd"> 52.2 ng/uL </TD><TD class="confluenceTd"> 12pm </TD></TR><TR><TD class="confluenceTd"> 6.9.2011 </TD><TD class="confluenceTd"> Kenneth </TD><TD class="confluenceTd"> pEXPR_3-4_minCMV-CI434:eYFP (A) </TD><TD class="confluenceTd"> 70 ng/ul </TD><TD class="confluenceTd"> 12pm </TD></TR><TR><TD class="confluenceTd"> 6.9.2011 </TD><TD class="confluenceTd"> Kenneth </TD><TD class="confluenceTd"> pEXPR_3-4_minCMV-CI434:eYFP (B) </TD><TD class="confluenceTd"> 174 ng/uL </TD><TD class="confluenceTd"> 12pm </TD></TR><TR><TD class="confluenceTd"> 6.9.2011 </TD><TD class="confluenceTd"> Louis </TD><TD class="confluenceTd"> pEXPR_2-3_Tre:C1434VP16 </TD><TD class="confluenceTd"> 234.6 ng/uL </TD><TD class="confluenceTd"> 12pm </TD></TR><TR><TD class="confluenceTd"> 6.9.2011 </TD><TD class="confluenceTd"> Tyler </TD><TD class="confluenceTd"> pEXPR_2-3_Tre:Lac/Krab </TD><TD class="confluenceTd"> 130.7 ng/uL </TD><TD class="confluenceTd"> 12pm </TD></TR><TR><TD class="confluenceTd"> 6.9.2011 </TD><TD class="confluenceTd"> Michelle </TD><TD class="confluenceTd"> pEXPR_2-3_Tre:Mnt1VP16 </TD><TD class="confluenceTd"> 110.0 ng/uL </TD><TD class="confluenceTd"> 12pm </TD></TR><TR><TD class="confluenceTd"> 6.9.2011 </TD><TD class="confluenceTd"> Michelle </TD><TD class="confluenceTd"> pDEST_2-3_ccdB </TD><TD class="confluenceTd"> 117.6 ng/uL </TD><TD class="confluenceTd"> 12pm </TD></TR></TBODY></TABLE></DIV><P>Restriction Digests</P><DIV class="table-wrap"><TABLE id="minimalist"><TBODY><TR><TH class="confluenceTh"> Assignee </TH><TH class="confluenceTh"> DNA </TH><TH class="confluenceTh"> Enzyme </TH><TH class="confluenceTh"> Expected Results </TH><TH class="confluenceTh"> Picture of Gel </TH><TH class="confluenceTh"> Time Incubated </TH><TH class="confluenceTh"> Comments </TH></TR><TR><TD class="confluenceTd"> Louis </TD><TD class="confluenceTd"> pEXPR_2-3_Tre:C1434VP16 </TD><TD class="confluenceTd"> NdeI </TD><TD class="confluenceTd"> 6700 bp 
800 bp </TD><TD class="confluenceTd"> a.3 </TD><TD class="confluenceTd"> 6/9/11 
3:50 PM </TD><TD class="confluenceTd"> </TD><TD class="confluenceTd"> </TD></TR><TR><TD class="confluenceTd"> Kenneth </TD><TD class="confluenceTd"> pEXPR_3-4_minCMV-CI434:eYFP (A+B) </TD><TD class="confluenceTd"> NcoI and SacII </TD><TD class="confluenceTd"> 2450 bp 
4650 bp </TD><TD class="confluenceTd"> A: a.6 
B: a.7 </TD><TD class="confluenceTd"> 6/9/11 
3:00 PM </TD><TD class="confluenceTd"> A did not cut as expected. B looks good. 
Digestion Mix: 2 uL NEB4, 0.5 uL of each enzyme, A: 7.1 uL/B: 2.9 uL of DNA, 10.4 uL/14.6 uL of H20 </TD><TD class="confluenceTd"> </TD><TD class="confluenceTd"> </TD></TR><TR><TD class="confluenceTd"> Michelle </TD><TD class="confluenceTd"> pEXPR_2-3_Tre:Mnt1VP16 </TD><TD class="confluenceTd"> BglI </TD><TD class="confluenceTd"> 3700 bp 
2200 bp 
1300 bp </TD><TD class="confluenceTd"> a.1 </TD><TD class="confluenceTd"> 6/9/11 
4:00PM </TD><TD class="confluenceTd"> DNA appeared to be uncut. Need to redo, possibly with a new enzyme and more DNA. </TD><TD class="confluenceTd"> </TD><TD class="confluenceTd"> </TD></TR><TR><TD class="confluenceTd"> Michelle </TD><TD class="confluenceTd"> pDEST_2-3_ccdB </TD><TD class="confluenceTd"> NcoI and NheI </TD><TD class="confluenceTd"> 6100 bp  1700 bp </TD><TD class="confluenceTd"> a.2 </TD><TD class="confluenceTd"> 6/9/2011 
4:25 PM </TD><TD class="confluenceTd"> Attained bands at correct positions. Other band represents partially cut DNA in double digest. </TD><TD class="confluenceTd"> </TD><TD class="confluenceTd"> </TD></TR><TR><TD class="confluenceTd"> Divya </TD><TD class="confluenceTd"> pEXPR_2-3_Tre:LexAVP16 </TD><TD class="confluenceTd"> HincII </TD><TD class="confluenceTd"> </TD><TD class="confluenceTd"> a.4 </TD><TD class="confluenceTd"> 6/9/2011 
4:45 PM </TD><TD class="confluenceTd">- Bands didn't show up. Needs to be redone. - DNA may have degraded. Need to redo Nanodrop as well. </TD><TD class="confluenceTd"> </TD><TD class="confluenceTd"> </TD></TR><TR><TD class="confluenceTd"> Tyler </TD><TD class="confluenceTd"> pEXPR_2-3_Tre:Lac/Krab </TD><TD class="confluenceTd"> SalI </TD><TD class="confluenceTd"> 3 bands: 
5288 bp 
1510 bp 
1241 bp 
cuts gene </TD><TD class="confluenceTd"> a.5 </TD><TD class="confluenceTd"> 6/9/2011 
3:00 PM </TD><TD class="confluenceTd"> Digestion: 
4 uL * 130.7 ng/uL = 522.8 ng DNA 
2 uL NEB3 
2 uL BSA 
11 uL H2O 
1 uL SalI 
Total: 20 uL 
Only saw one band at 3500 bp 
Needs to be redone possibly with another restriction enzyme </TD><TD class="confluenceTd"> </TD><TD class="confluenceTd"> </TD></TR></TBODY></TABLE></DIV><P>Gels</P><DIV class="table-wrap"><TABLE id="minimalist"><TBODY><TR><TH class="confluenceTh"> Label </TH><TH class="confluenceTh"> Picture </TH></TR><TR><TD class="confluenceTd"> a </TD></TR></TBODY></TABLE></DIV><H3>June 10, 2011</H3><P>Restriction Gels</P><DIV class="table-wrap"><TABLE id="minimalist"><TBODY><TR><TH class="confluenceTh"> Assignee </TH><TH class="confluenceTh"> DNA </TH><TH class="confluenceTh"> Enzyme </TH><TH class="confluenceTh"> Expected result </TH><TH class="confluenceTh"> Time Incubated </TH><TH class="confluenceTh"> Comments </TH></TR><TR><TD class="confluenceTd"> Louis </TD><TD class="confluenceTd"> pEXPR_2-3_Tre:C1434VP16 </TD><TD class="confluenceTd"> EcoRI (Buffer 2) </TD><TD class="confluenceTd"> 5500 bp 
1050 bp 
650 bp 
360 bp </TD><TD class="confluenceTd"> 6/10/11 
11:15 AM </TD><TD class="confluenceTd"> </TD></TR><TR><TD class="confluenceTd"> Michelle </TD><TD class="confluenceTd"> pEXPR_2-3_TRE:Mnt1VP16 (A) </TD><TD class="confluenceTd"> NcoI and NheI </TD><TD class="confluenceTd"> 5700 bp 
1300 bp </TD><TD class="confluenceTd"> 6/10/11 
11:40 AM </TD><TD class="confluenceTd"> Digestion: 
6 uL DNA 
1 uL NcoI 
1 uL Nhe1 
2 uL NEB2 
2 uL BSA 
8 uL H2O 
Failed. </TD></TR><TR><TD class="confluenceTd"> Michelle </TD><TD class="confluenceTd"> pEXPR_2-3_TRE:Mnt1VP16 (B) </TD><TD class="confluenceTd"> NheI and SphI </TD><TD class="confluenceTd"> 1700 bp 
5300 bp </TD><TD class="confluenceTd"> 6/10/11 
11:40 AM </TD><TD class="confluenceTd"> Digestion: 
6 uL DNA 
1 uL NheI 
1 uL SphI 
2 uL NEB2 
2 uL BSA 
8 uL H2O 
Failed. Redoing LR on Monday June 13th. </TD></TR><TR><TD class="confluenceTd"> Tyler </TD><TD class="confluenceTd"> pEXPR_2-3_Tre:Lac/Krab </TD><TD class="confluenceTd"> SalI </TD><TD class="confluenceTd"> 3 bands: 
5288 bp 
1510 bp 
1241 bp 
cuts gene </TD><TD class="confluenceTd"> 6/10/2011 
3:00 PM </TD><TD class="confluenceTd"> Digestion: 4 uL * 130.7 ng/uL = 522.8 ng DNA 
2 uL NEB3 
2 uL BSA 
11 uL H2O 
1 uL SalI 
Total: 20 uL ***Worked (see gel below - b.1) 
Stored in -80C freezer </TD></TR></TBODY></TABLE></DIV><P>Comments:</P><P>pEXPR_2-3_Tre:LexAVP16 and pEXPR_3-4_minCMV-7xMnt1:eYFP are being entirely redone.</P><P>pEXPR_2-3_Tre:LexAVP16 (post-miniprep) was lost. pEXPR_3-4_minCMV-7xMnt1:eYFP LR was unsuccessful (no bacteria grew).</P><P>LR and transformation will be done on Saturday. Miniprep and restriction mapping will be done on Sunday</P><DIV class="table-wrap"><TABLE id="minimalist"><TBODY><TR><TH class="confluenceTh"> Label </TH><TH class="confluenceTh"> Gel </TH><TH class="confluenceTh"> Legend </TH></TR><TR><TD class="confluenceTd"> b </TD><TD class="confluenceTd"> Column 0: HyperLadder 
Column 1: pEXPR_2-3_Tre:Lac/Krab 
Column 4: pEXPR_2-3_Tre:C1434VP16 
Column 6: pEXPR_2-3_TRE:Mnt1VP16 (A) 
Column 7: pEXPR_2-3_TRE:Mnt1VP16 (B) </TD></TR></TBODY></TABLE></DIV><H3>June 11, 2011</H3><P>Set of LRs from 6/9 (GR, JC, and SR) cell counts taken; inoculated by GR/JC. 3 cells taken per plate.</P><DIV class="table-wrap"><TABLE id="minimalist"><TBODY><TR><TH class="confluenceTh"> Assignee </TH><TH class="confluenceTh"> DNA </TH><TH class="confluenceTh"> Enzyme </TH><TH class="confluenceTh"> Expected Results </TH><TH class="confluenceTh"> Time Incubated </TH><TH class="confluenceTh"> Comments </TH></TR><TR><TD class="confluenceTd"> Sam </TD><TD class="confluenceTd"> pEXPR_1-2_Hef:eBFP </TD><TD class="confluenceTd"> NcoI &amp; ApaL1 </TD><TD class="confluenceTd"> 3750bp, 3180bp, 1250bp </TD><TD class="confluenceTd"> 45 min </TD><TD class="confluenceTd"> If it doesn't work there should be a whole mess of fragments (including a small 600bp one) 
1mL each enz; 2 mL Buf; 1 mL BSA; 2mL DNA; 13mL H2O (Standard Protocol with extra BSA) </TD></TR><TR><TD class="confluenceTd"> Sam </TD><TD class="confluenceTd"> pEXPR_4-5_Tre:mKate </TD><TD class="confluenceTd"> Bgl1 </TD><TD class="confluenceTd"> 1270bp, 2630bp, 3380bp </TD><TD class="confluenceTd"> 45 min </TD><TD class="confluenceTd"> 1mL enz; 2mL DNA; 2mL BSA; 2mL Buf; 15mL H2O (Standard Protocol with extra BSA and water - not quite 10x) </TD></TR></TBODY></TABLE></DIV></DIV><DIV class="content" id="w2content"><H1>Summary of Week 2: June 12 - June 18</H1>
In the second week we attempted midipreps instead of minipreps of our available cell stocks. Midipreps were unsuccessful, likely due to centrifuge limitations. LR reactions to generate more usable promoter-gene pair parts continued, expanding to include the CI434-VP16, LexA-VP16, and Mnt-VP16 genes, whose expressed proteins activate the appropriate corresponding minCMV promoters. Most of the LRs done were not successful, reason unknown at this time.
<H3>June 13, 2011</H3><P>Status</P><DIV class="table-wrap"><TABLE id="minimalist"><TBODY><TR><TH class="confluenceTh"> DNA </TH><TH class="confluenceTh"> Status </TH><TH class="confluenceTh"> Comment </TH></TR><TR><TD class="confluenceTd"> Mntcmv/1x cI434:eYFP </TD><TD class="confluenceTd"> good </TD><TD class="confluenceTd"> </TD></TR><TR><TD class="confluenceTd"> Mnt cmv/4x LexA:eYFP </TD><TD class="confluenceTd"> bad </TD><TD class="confluenceTd"> </TD></TR><TR><TD class="confluenceTd"> HefIa/LacO:eYFP </TD><TD class="confluenceTd"> good </TD><TD class="confluenceTd"> </TD></TR><TR><TD class="confluenceTd"> Tre:LacI Krab </TD><TD class="confluenceTd"> good </TD><TD class="confluenceTd"> </TD></TR><TR><TD class="confluenceTd"> HefIaL EBFP2 </TD><TD class="confluenceTd"> bad </TD><TD class="confluenceTd"> </TD></TR></TBODY></TABLE></DIV><DIV class="table-wrap"><TABLE id="minimalist"><TBODY><TR><TH class="confluenceTh"> Date </TH><TH class="confluenceTh"> Assignee </TH><TH class="confluenceTh"> DEST_R4R2 </TH><TH class="confluenceTh"> L4-R1 Promoter </TH><TH class="confluenceTh"> L1-L2 
Gene </TH><TH class="confluenceTh"> LR? </TH><TH class="confluenceTh"> Tube Number </TH><TH class="confluenceTh"> Colony Number, 
Antibiotic, Person </TH><TH class="confluenceTh"> Protocol Used </TH></TR><TR><TD class="confluenceTd"> 6.13.2011 </TD><TD class="confluenceTd"> Jon </TD><TD class="confluenceTd"> 2-3 </TD><TD class="confluenceTd"> Tre </TD><TD class="confluenceTd"> C1434-VP16 </TD><TD class="confluenceTd"> Yes. 6-13, 11am </TD><TD class="confluenceTd"> 10 </TD><TD class="confluenceTd"> about 50, AMP, JC </TD><TD class="confluenceTd"> No </TD></TR><TR><TD class="confluenceTd"> 6.13.2011 </TD><TD class="confluenceTd"> Tyler </TD><TD class="confluenceTd"> 2-3 </TD><TD class="confluenceTd"> Tre </TD><TD class="confluenceTd"> LexA-VP16 </TD><TD class="confluenceTd"> Yes. 6-13, 11am </TD><TD class="confluenceTd"> 9 </TD><TD class="confluenceTd"> about 30, AMP, GR </TD><TD class="confluenceTd"> No </TD></TR><TR><TD class="confluenceTd"> 6.13.2011 </TD><TD class="confluenceTd"> Mariola </TD><TD class="confluenceTd"> 2-3 </TD><TD class="confluenceTd"> Tre </TD><TD class="confluenceTd"> Mnt1-VP16 </TD><TD class="confluenceTd"> Yes. 6-13, 11am </TD><TD class="confluenceTd"> Eppendorf </TD><TD class="confluenceTd"> 0, AMP, GR </TD><TD class="confluenceTd"> No </TD></TR><TR><TD class="confluenceTd"> 6.13.2011 </TD><TD class="confluenceTd"> Semon </TD><TD class="confluenceTd"> 4-5 </TD><TD class="confluenceTd"> Tre </TD><TD class="confluenceTd"> mKate </TD><TD class="confluenceTd"> Yes. 6-13, 11am </TD><TD class="confluenceTd"> 11 </TD><TD class="confluenceTd"> about 10, AMP, SR </TD><TD class="confluenceTd"> No </TD></TR><TR><TD class="confluenceTd"> 6.13.2011 </TD><TD class="confluenceTd"> Divya </TD><TD class="confluenceTd"> 3-4 </TD><TD class="confluenceTd"> minCMV-7xMnt1 </TD><TD class="confluenceTd"> eYFP </TD><TD class="confluenceTd"> Yes 6/13, 1:30pm </TD><TD class="confluenceTd"> ? </TD><TD class="confluenceTd"> 1000s, AMP, GR </TD><TD class="confluenceTd"> No </TD></TR><TR><TD class="confluenceTd"> 6.13.2011 </TD><TD class="confluenceTd"> Jenny </TD><TD class="confluenceTd"> 3-4 </TD><TD class="confluenceTd"> minCMV-4xMnt1 </TD><TD class="confluenceTd"> eYFP </TD><TD class="confluenceTd"> Yes 6/13, 1:30pm </TD><TD class="confluenceTd"> ? </TD><TD class="confluenceTd"> 1000s, AMP, GR </TD><TD class="confluenceTd"> No </TD></TR><TR><TD class="confluenceTd"> 6.13.2011 </TD><TD class="confluenceTd"> Jenny </TD><TD class="confluenceTd"> 3-4 </TD><TD class="confluenceTd"> minCMV-4xC1434 </TD><TD class="confluenceTd"> eYFP </TD><TD class="confluenceTd"> Yes 6/13,1:30pm </TD><TD class="confluenceTd"> ? </TD><TD class="confluenceTd"> 1000s, AMP, GR </TD><TD class="confluenceTd"> No </TD></TR></TBODY></TABLE></DIV><H3>June 14, 2011</H3><P>Highlights:</P><H6><FONT color="#000000">~10:30 AM: Miniprep of pDEST23-ccdB commenced and completed. DNA concentrations of the two samples were 175 and 310 ng/uL</FONT></H6><P><FONT color="#000000">~1:00 PM: Inoculation of cultures for pEXPR-23-TRE-CI434-VP16, pEXPR-23-TRE-LexA-VP16, pEXPR-45-TRE-mKate, pEXPR-34-minCMV-7xMnt-eYFP, pEXPR-34-minCMV-4xMnt-eYFP, and pEXPR-minCMV-4xCI434 were conducted. Colony counts from the previous transformations are noted above.</FONT></P><P>~1:30 PM: LR Gateway reactions were performed for the failed pEXPR-23-TRE-Mnt-VP16 and the new pEXPR-12-Hef1A-rtTA.</P><P>~5:00 PM: Inoculation of midiprep cultures for EXPR-34-minCMV4xLexA-eYFP, two samples of pEXPR-34-Hef1ALacO-eYFP, pEXPR-12-Hef1A-EBFP, pEXPR-23-TRE-LacIKrab, and pEXPR-34-minCMV1xcI434-eYFP commenced. We plan to midiprep the cells at about 10:00 AM.</P><P>~6:00 PM: Transformation of the aforementioned LR Gateway reactions commenced. Transformation completed around 9:00 PM.</P><P>~7:00 PM: Design for the TANGO primers complete.</P><H3>June 15, 2011</H3><P>~4AM: Grant did more LRs – 3-4_minCMV-4xLexA:eYFP, 1-2_Hef1a:rttA3, 1-2_Hef1a:eBFP2</P><P>~10AM: Minipreps done on the 6 LRs that succeeded from yesterday by Grant</P><H4>Nanodrops</H4><DIV class="table-wrap"><TABLE id="minimalist"><TBODY><TR><TH class="confluenceTh"> Date </TH><TH class="confluenceTh"> Assignee </TH><TH class="confluenceTh"> Vector </TH><TH class="confluenceTh"> Concentrations (ng/uL) *denotes 260/280 below 1.8 </TH></TR><TR><TD class="confluenceTd"> 6.14 11AM </TD><TD class="confluenceTd"> Jon/Michelle/Clara </TD><TD class="confluenceTd"> 2-3_Tre:C1434-VP16 </TD><TD class="confluenceTd"> A: 284.0* (1.78), B: 391.3, C: 380.9, D: 205.5 </TD></TR><TR><TD class="confluenceTd"> 6.14 11AM </TD><TD class="confluenceTd"> Jon/Michelle/Clara </TD><TD class="confluenceTd"> 2-3_Tre:LexA-VP16 </TD><TD class="confluenceTd"> A: 361.4, B: 193.9* (1.64), C: 212.9* (1.58), D: 390.6 </TD></TR><TR><TD class="confluenceTd"> 6.14 11AM </TD><TD class="confluenceTd"> Jon/Michelle/Clara </TD><TD class="confluenceTd"> 4-5_Tre:mKate </TD><TD class="confluenceTd"> A: 176.9, B: 180.3, C: 188.7, D: 208.8 </TD></TR><TR><TD class="confluenceTd"> 6.14 11AM </TD><TD class="confluenceTd"> Jon/Michelle/Clara </TD><TD class="confluenceTd"> 3-4_minCMV-4xC1434:eYFP </TD><TD class="confluenceTd"> A: 173.8, B: 152.5* (1.60), C: 319.5, D: 192.5 </TD></TR><TR><TD class="confluenceTd"> 6.14 11AM </TD><TD class="confluenceTd"> Jon/Michelle/Clara </TD><TD class="confluenceTd"> 3-4_minCMV-7xMnt1:eYFP </TD><TD class="confluenceTd"> A: 147.7, B: 251.5, C: 147.6, D: 178.2 </TD></TR><TR><TD class="confluenceTd"> 6.14 11AM </TD><TD class="confluenceTd"> Jon/Michelle/Clara </TD><TD class="confluenceTd"> 3-4_minCMV-4xMnt1:eYFP </TD><TD class="confluenceTd"> A: 195.1, B: 241.2* (1.75), C: 193.7, D: 157.0 </TD></TR></TBODY></TABLE></DIV><H4>Restriction Maps, 6.14 5PM</H4><DIV class="table-wrap"><TABLE id="minimalist"><TBODY><TR><TH class="confluenceTh"> Assignee </TH><TH class="confluenceTh"> DNA </TH><TH class="confluenceTh"> Enzyme </TH><TH class="confluenceTh"> Expected Results </TH><TH class="confluenceTh"> Comments </TH></TR><TR><TD class="confluenceTd"> Jon </TD><TD class="confluenceTd"> 3-4_minCMV-4xMnt1:eYFP </TD><TD class="confluenceTd"> SalI-HF </TD><TD class="confluenceTd"> 5288bp, 2044bp if works; 5288bp, 1600bp, 977bp if doesn't </TD><TD class="confluenceTd"> All: 1uL SalI, 2uL Buf 4, 2uL BSA; A,B,C: 3uL DNA, D: 4uL DNA; DI up to 20 uL </TD></TR><TR><TD class="confluenceTd"> Jon </TD><TD class="confluenceTd"> 3-4_minCMV-7xMnt1:eYFP </TD><TD class="confluenceTd"> SalI-HF </TD><TD class="confluenceTd"> 5288bp, 2257bp if works; 5288bp, 1600bp, 977bp if doesn't </TD><TD class="confluenceTd"> All: 1uL SalI, 2uL Buf 4, 2uL BSA; A,B,C: 3uL DNA, D: 4uL DNA; DI up to 20 uL </TD></TR><TR><TD class="confluenceTd"> Jon </TD><TD class="confluenceTd"> 4-5_Tre:mKate </TD><TD class="confluenceTd"> SalI-HF </TD><TD class="confluenceTd"> 5288bp, 1995bp if works; 5288bp, 1600bp, 977bp if doesn't </TD><TD class="confluenceTd"> All: 1uL SalI, 2uL Buf 4, 2uL BSA; A,B,C: 3uL DNA, D: 4uL DNA; DI up to 20 uL </TD></TR><TR><TD class="confluenceTd"> Mariola </TD><TD class="confluenceTd"> 2-3_Tre:C1434-VP16 </TD><TD class="confluenceTd"> SalI-HF </TD><TD class="confluenceTd"> 5288, 1862, 412 bp if works; 5288bp, 1600bp, 977bp if doesn't </TD><TD class="confluenceTd"> All: 1uL SalI, 2uL Buf 4, 2uL BSA; A: 1.8uL DNA, B,C: 1.3uL DNA, D: 2.4uL DNA; DI up to 20 uL </TD></TR><TR><TD class="confluenceTd"> Mariola </TD><TD class="confluenceTd"> 3-4_minCMV-4xC1434:eYFP </TD><TD class="confluenceTd"> BamHI </TD><TD class="confluenceTd"> 3757, 1855, 1166, 478 bp if works; no cut if doesn't </TD><TD class="confluenceTd"> All: 1uL BamHI, 2uL Buf 4, 2uL BSA; A 2.9uL DNA, B: 3.2uL DNA, C: 1.6uL DNA, D: 2.6uL DNA; DI up to 20 uL </TD></TR><TR><TD class="confluenceTd"> Tyler </TD><TD class="confluenceTd"> 2-3_Tre:LexA-VP16 </TD><TD class="confluenceTd"> ScaI </TD><TD class="confluenceTd"> 2 cuts: 4464 and 2729 bp if works; 3 cuts: 3535, 2729, 1589 if doesn't </TD><TD class="confluenceTd"> All: 1uL SalI, 2uL Buf 4, 2uL BSA; A: 3uL DNA, B,D: 1.5uL DNA, C: 2.5uL DNA; DI up to 20 uL 
From Gel Below: 
A and C show the entry vector, B failed as well, D seems to have worked. There is uncut DNA around 7000 bp, but the other bands seem correct. D will be sent for sequencing. </TD></TR></TBODY></TABLE></DIV><P>Midipreps were done by Kenneth and Tyler for sequenced verified constructs, 2-3_Tre:LacI-Krab, 3-4_minCMV 1xCI434:eYFP, and 3-4_Hef1ALac0:eYFP. Unfortunately DNA yields were low, as shown in the table below, so minipreps will be performed tomorrow, June 16, 2011.</P><DIV class="table-wrap"><TABLE id="minimalist"><TBODY><TR><TH class="confluenceTh"> Date </TH><TH class="confluenceTh"> Assignee </TH><TH class="confluenceTh"> Vector </TH><TH class="confluenceTh"> DNA Concentration (ng/uL) </TH></TR><TR><TD class="confluenceTd"> 6.15.2011 </TD><TD class="confluenceTd"> Tyler </TD><TD class="confluenceTd"> 2-3_Tre:LacI-Krab </TD><TD class="confluenceTd"> 54 </TD></TR><TR><TD class="confluenceTd"> 6.15.2011 </TD><TD class="confluenceTd"> Kenneth </TD><TD class="confluenceTd"> 3-4_minCMV 1xCI434:eYFP </TD><TD class="confluenceTd"> 216 </TD></TR><TR><TD class="confluenceTd"> 6.15.2011 </TD><TD class="confluenceTd"> Kenneth/Tyler </TD><TD class="confluenceTd"> 3-4_Hef1ALac0:eYFP </TD><TD class="confluenceTd"> failed<FONT color="#660000"><B> </B></FONT></TD></TR></TBODY></TABLE></DIV><H3>June 16, 2011</H3><P>~10am Minipreps</P><DIV class="table-wrap"><TABLE id="minimalist"><TBODY><TR><TH class="confluenceTh"> Date </TH><TH class="confluenceTh"> Assignee </TH><TH class="confluenceTh"> Vector </TH><TH class="confluenceTh"> DNA Concentration (ng/uL) </TH></TR><TR><TD class="confluenceTd"> 6.16.2011 </TD><TD class="confluenceTd"> Tyler </TD><TD class="confluenceTd"> 2-3_Tre:LacI-Krab </TD><TD class="confluenceTd"> 509.1 </TD></TR><TR><TD class="confluenceTd"> 6.16.2011 </TD><TD class="confluenceTd"> Kenneth </TD><TD class="confluenceTd"> 3-4_minCMV 1xCI434:eYFP </TD><TD class="confluenceTd"> 89.1 </TD></TR><TR><TD class="confluenceTd"> 6.16.2011 </TD><TD class="confluenceTd"> Kenneth/Tyler </TD><TD class="confluenceTd"> 3-4_Hef1ALac0:eYFP </TD><TD class="confluenceTd"> 385 </TD></TR></TBODY></TABLE></DIV><H3>June 17-18, 2011 Construction</H3><H6><FONT color="#000000">Protocol for all Cells: Added DNA 5 minutes after removing cells from freezer. Incubated for 30 minutes. Heat shocked for 30 seconds at 42 Celsius. Incubated on ice for two minutes. Added 1 mL of SOC. Grew at 30 C for 2 hours. Plated 100 uL from total reaction on one half of plate &amp; the rest of the reaction on the other half of plate. </FONT></H6><DIV class="table-wrap"><TABLE id="minimalist"><TBODY><TR><TH class="confluenceTh"><FONT color="#660000">Date </FONT></TH><TH class="confluenceTh"><FONT color="#660000">Assignee</FONT></TH><TH class="confluenceTh"><FONT color="#660000">DEST_R4R2</FONT></TH><TH class="confluenceTh"><FONT color="#660000">Colonies (low/high conc.)</FONT></TH></TR><TR><TD class="confluenceTd"><FONT color="#000000">6.18.2011</FONT></TD><TD class="confluenceTd"><FONT color="#000000">Grant</FONT></TD><TD class="confluenceTd"><FONT color="#000000">pEXPR12-Hef1A-eBFP2</FONT></TD><TD class="confluenceTd"><FONT color="#000000">Order of &lt;10</FONT></TD></TR><TR><TD class="confluenceTd"><FONT color="#000000">6.18.2011</FONT></TD><TD class="confluenceTd"><FONT color="#000000">Grant</FONT></TD><TD class="confluenceTd"><FONT color="#000000">pEXPR12-Hef1A-rtTA3</FONT></TD><TD class="confluenceTd"><FONT color="#000000">Order of &lt;10</FONT></TD></TR><TR><TD class="confluenceTd"><FONT color="#000000">6.18.2011</FONT></TD><TD class="confluenceTd"><FONT color="#000000">Grant</FONT></TD><TD class="confluenceTd"><FONT color="#000000">pEXPR23-TRE-CI434.VP16</FONT></TD><TD class="confluenceTd"><FONT color="#000000">Order of 100-1000</FONT></TD></TR><TR><TD class="confluenceTd"><FONT color="#000000">6.18.2011</FONT></TD><TD class="confluenceTd"><FONT color="#000000">Grant</FONT></TD><TD class="confluenceTd"><FONT color="#000000">pEXPR23-TRE-LexA.VP16</FONT></TD><TD class="confluenceTd"><FONT color="#000000">Order of 100-1000</FONT></TD></TR><TR><TD class="confluenceTd"><FONT color="#000000">6.18.2011</FONT></TD><TD class="confluenceTd"><FONT color="#000000">Grant</FONT></TD><TD class="confluenceTd"><FONT color="#000000">pEXPR23-TRE-Mnt.VP16</FONT></TD><TD class="confluenceTd"><FONT color="#000000">Order of 100-1000</FONT></TD></TR><TR><TD class="confluenceTd"><FONT color="#000000">6.18.2011</FONT></TD><TD class="confluenceTd"><FONT color="#000000">Grant </FONT></TD><TD class="confluenceTd"><FONT color="#000000">pEXPR34-minCMV.4xCI434-EYFP</FONT></TD><TD class="confluenceTd"><FONT color="#000000">Order of 1000-10000</FONT></TD></TR><TR><TD class="confluenceTd"><FONT color="#000000">6.18.2011</FONT></TD><TD class="confluenceTd"><FONT color="#000000">Grant</FONT></TD><TD class="confluenceTd"><FONT color="#000000">pEXPR34-minCMV.4xLexA-EYFP</FONT></TD><TD class="confluenceTd"><FONT color="#000000">Order of 1000-10000</FONT></TD></TR><TR><TD class="confluenceTd"><FONT color="#000000">6.18.2011</FONT></TD><TD class="confluenceTd"><FONT color="#000000">Grant</FONT></TD><TD class="confluenceTd"><FONT color="#000000">pEXPR34-minCMV.4xMnt-EYFP</FONT></TD><TD class="confluenceTd"><FONT color="#000000">Order of 1000-10000</FONT></TD></TR><TR><TD class="confluenceTd"><FONT color="#000000">6.18.2011</FONT></TD><TD class="confluenceTd"><FONT color="#000000">Grant</FONT></TD><TD class="confluenceTd"><FONT color="#000000">pEXPR34-minCMV.7xMnt-EYFP</FONT></TD><TD class="confluenceTd"><FONT color="#000000">Order of 1000-10000</FONT></TD></TR><TR><TD class="confluenceTd"><FONT color="#000000">6.18.2011</FONT></TD><TD class="confluenceTd"><FONT color="#000000">Grant</FONT></TD><TD class="confluenceTd"><FONT color="#000000">pEXPR45-TRE-mKate</FONT></TD><TD class="confluenceTd"><FONT color="#000000">Order of &lt;10</FONT></TD></TR></TBODY></TABLE></DIV><H6><FONT color="#000000">Conclusion: pDEST12 appears to be growing slowly in all cases, as does pDEST45. pDEST23 grows at the rate typically expected by Gateway reaction standards while pDEST34 grows at a rate much faster than these standards. Corroborated in Weiss lab?</FONT></H6></DIV><DIV class="content" id="w3content"><H1>Summary of Week 3: June 20 - June 24</H1>
In the third week we continued expanding our library of usable DNA parts by creating more combinations of genes and promoters using the LR reaction.


<H3>June 20, 2011</H3><P>Minipreps and nanodrops done on the plasmids from yesterday by Clara and Grant.
Restriction digesting done on the 10 transformations.</P><DIV class="table-wrap"><TABLE id="minimalist"><TBODY><TR><TH class="confluenceTh"> Assignee </TH><TH class="confluenceTh"> DNA </TH><TH class="confluenceTh"> Enzyme </TH><TH class="confluenceTh"> Expected Results </TH><TH class="confluenceTh"> Picture of Gel </TH><TH class="confluenceTh"> Time Incubated </TH><TH class="confluenceTh"> Comments </TH></TR><TR><TD class="confluenceTd"> Jon </TD><TD class="confluenceTd"> 1-2_Hef1a-rtTA3 </TD><TD class="confluenceTd"> SalI-HF </TD><TD class="confluenceTd"> 5288, 2877bp if works; 5288, 1600, 977bp if it doesn't </TD><TD class="confluenceTd"> Gel 1 Lanes 2-4 </TD><TD class="confluenceTd"> 2 hrs </TD><TD class="confluenceTd"> Being redone </TD></TR><TR><TD class="confluenceTd"> Jon </TD><TD class="confluenceTd"> 1-2_Hef1a-eBFP2 </TD><TD class="confluenceTd"> SalI-HF </TD><TD class="confluenceTd">5288, 2889bp if works; 5288, 1600, 977bp if it doesn't </TD><TD class="confluenceTd"> Gel 1 Lanes 5-7 </TD><TD class="confluenceTd"> 2 hrs </TD><TD class="confluenceTd"> Being redone </TD></TR><TR><TD class="confluenceTd"> Jon </TD><TD class="confluenceTd"> 3-4_minCMV-7xMnt1:eYFP </TD><TD class="confluenceTd"> SalI-HF </TD><TD class="confluenceTd"> 5288bp, 2257bp if works; 5288bp, 1600bp, 977bp if doesn't </TD><TD class="confluenceTd"> Gel 1 Lanes 8-10 </TD><TD class="confluenceTd"> 2 hrs </TD><TD class="confluenceTd"> A and C sent off for sequencing </TD></TR><TR><TD class="confluenceTd"> Jon </TD><TD class="confluenceTd"> 2-3_Tre_Mnt1-VP16 </TD><TD class="confluenceTd"> ScaI </TD><TD class="confluenceTd"> 4440, 2729bp if works; 3535, 2729, 1589bp if doesn't </TD><TD class="confluenceTd"> Gel 1 Lane 11, Gel 2 Lanes 2-3 </TD><TD class="confluenceTd"> 2 hrs </TD><TD class="confluenceTd"> A sent for sequencing; also being redone </TD></TR><TR><TD class="confluenceTd"> Michelle </TD><TD class="confluenceTd"> 4-5_Tre_mKate </TD><TD class="confluenceTd"> SalI-HF </TD><TD class="confluenceTd"> 5288bp, 1995bp if works; 5288bp, 1600bp, 977bp if doesn't </TD><TD class="confluenceTd"> Gel 2 Lanes 10-11, Gel 3 Lane 2 </TD><TD class="confluenceTd"> 2 hrs </TD><TD class="confluenceTd"> B sent off for sequencing; also being redone </TD></TR><TR><TD class="confluenceTd"> Michelle </TD><TD class="confluenceTd"> 2-3_Tre:CI434-VP16 </TD><TD class="confluenceTd"> SalI-HF </TD><TD class="confluenceTd"> 5288, 2270bp if works; 5288, 1600, 977bp if doesn't </TD><TD class="confluenceTd"> Gel 2 Lanes 7-9 </TD><TD class="confluenceTd"> 2 hrs </TD><TD class="confluenceTd"> Being redone </TD></TR><TR><TD class="confluenceTd"> Michelle </TD><TD class="confluenceTd"> 2-3_Tre:LexA-VP16 </TD><TD class="confluenceTd"> SalI-HF </TD><TD class="confluenceTd"> 5288, 1913bp if works; 5288, 1600, 977bp if doesn't </TD><TD class="confluenceTd"> Gel 2 Lanes 4-6 </TD><TD class="confluenceTd"> 2 hrs </TD><TD class="confluenceTd"> B sent off for sequencing </TD></TR><TR><TD class="confluenceTd"> Charles </TD><TD class="confluenceTd"> 3-4_minCMV 4xLexA:eYFP </TD><TD class="confluenceTd"> SalI-HF </TD><TD class="confluenceTd"> 5288, 1849bp if works; 5288bp, 1600bp, 977bp if doesn't </TD><TD class="confluenceTd"> Gel 3 Lanes 6-8 </TD><TD class="confluenceTd"> 2 hrs </TD><TD class="confluenceTd"> B sent off for sequencing; also being redone </TD></TR><TR><TD class="confluenceTd"> Charles </TD><TD class="confluenceTd"> 3-4_minCMV 4xMntI:eYFP </TD><TD class="confluenceTd"> SalI-HF </TD><TD class="confluenceTd"> 5288bp, 2044bp if works; 5288bp, 1600bp, 977bp if doesn't </TD><TD class="confluenceTd"> Gel 3 Lanes 3-5 </TD><TD class="confluenceTd"> 2 hrs </TD><TD class="confluenceTd"> Being redone </TD></TR><TR><TD class="confluenceTd"> Charles </TD><TD class="confluenceTd"> 3-4_minCMV 4xCI434:eYFP </TD><TD class="confluenceTd"> SalI-HF </TD><TD class="confluenceTd"> 5288, 1960bp if works; 5288bp, 1600bp, 977bp if doesn't </TD><TD class="confluenceTd"> Gel 3 Lanes 9-11 </TD><TD class="confluenceTd"> 2 hrs </TD><TD class="confluenceTd"> B sent off for sequencing </TD></TR></TBODY></TABLE></DIV><P>Key: Ladder, 3x Hef1a rtTa3, 3x Hef1a eBFP2, 3x minCMV 7xMnt1:eYFP, Tre Mnt1 VP16 A, Ladder; Tre Mnt1 VP16 B+C, 3x Tre LexA VP16, 3x Tre CI434 VP16, Tre mKate A+B, Ladder</P><P>Key: Ladder, Tre mKate C, 3x minCMV 4xMnt1, 3x minCMV 4xLexA, 3x minCMV 4xCI434, Ladder</P><H3>June 23, 2011</H3><H6><FONT color="#000000">Colony Counts</FONT></H6><P><FONT color="#000000">These results are based on plating 100 uL of the outgrowth culture.</FONT></P><DIV class="table-wrap"><TABLE id="minimalist"><TBODY><TR><TH class="confluenceTh"> Part Shorthand </TH><TH class="confluenceTh"> Colonies this Time </TH><TH class="confluenceTh"> Average Colonies Before </TH><TH class="confluenceTh"> Notes </TH></TR><TR><TD class="confluenceTd"> TRE:mKate </TD><TD class="confluenceTd">~10 </TD><TD class="confluenceTd">~3 </TD><TD class="confluenceTd"> Using new TRE, Dest 23. </TD></TR><TR><TD class="confluenceTd"> TRE:MntVP16 </TD><TD class="confluenceTd">~50 </TD><TD class="confluenceTd">~10 </TD><TD class="confluenceTd"> Using new TRE, Dest 23. </TD></TR><TR><TD class="confluenceTd"> TRE:cI434VP16 </TD><TD class="confluenceTd">~30 </TD><TD class="confluenceTd">~1 </TD><TD class="confluenceTd"> Using new TRE, Dest 23. </TD></TR><TR><TD class="confluenceTd"> Hef1A:rtTA3 </TD><TD class="confluenceTd">~50 </TD><TD class="confluenceTd">~1 </TD><TD class="confluenceTd"> Using new Hef1A, Dest 23. </TD></TR><TR><TD class="confluenceTd"> Hef1A: eBFP2 </TD><TD class="confluenceTd">~30 </TD><TD class="confluenceTd">~3 </TD><TD class="confluenceTd"> Using new Hef1A, Dest 23. </TD></TR><TR><TD class="confluenceTd"> 4xLexA:eYFP </TD><TD class="confluenceTd">~1000s </TD><TD class="confluenceTd">~1000s </TD><TD class="confluenceTd"> Using Dest 23. </TD></TR><TR><TD class="confluenceTd"> 4xMnt:eYFP </TD><TD class="confluenceTd">~1000s </TD><TD class="confluenceTd">~1000s </TD><TD class="confluenceTd"> Using Dest 23. </TD></TR></TBODY></TABLE></DIV><H6><FONT color="#000000">Conclusion: pENTR vectors probably weren't properly aliquoted the first time around, strongly reducing recombination efficiency.</FONT></H6></DIV><DIV class="content" id="w4content"><H1>Summary of Week 4: June 27 - July 1</H1>
In the fourth week we implemented a color-coded box system in our -20 freezer to accommodate for the growing need of organization of the growing number of DNA parts. We began learning and using the Gibson reaction to create the gene part AVPR2-TEVs-GV16, which is expressed to produce a vasopressin receptor bound to Gal4-VP16 by a TEV sequence that can be recognized by the TEV Protease. Gibson reaction results were not great, and it appeared that our AVPR2 DNA was not of sufficient quality, so we re-PCRed the AVPR2 DNA segment.


<H3>June 28, 2011</H3><H3>June 29, 2011</H3><P>~ Miniprepped at 10:30 AM</P><DIV class="table-wrap"><TABLE id="minimalist"><TBODY><TR><TH class="confluenceTh"> Date </TH><TH class="confluenceTh"> Assignee </TH><TH class="confluenceTh"> Vector </TH><TH class="confluenceTh"> DNA Concentration </TH></TR><TR><TD class="confluenceTd"> 6.29.2011 </TD><TD class="confluenceTd"> Mariola </TD><TD class="confluenceTd"> pENTR_UAS </TD><TD class="confluenceTd"> A: 98.7 ng/ul 
B: 88.1 ng/ul </TD></TR><TR><TD class="confluenceTd"> 6.29.2011 </TD><TD class="confluenceTd"> Mariola </TD><TD class="confluenceTd"> pENTR_Tet-LacO </TD><TD class="confluenceTd"> A: 84.7 ng/ul 
B: 83.2 ng/ul </TD></TR><TR><TD class="confluenceTd"> 6.29.2011 </TD><TD class="confluenceTd"> Jenny </TD><TD class="confluenceTd"> pENTR_TEV-Protease </TD><TD class="confluenceTd"> A: 97.5 ng/ul 
B: 50.4 ng/ul </TD></TR><TR><TD class="confluenceTd"> 6.29.2011 </TD><TD class="confluenceTd"> Jenny </TD><TD class="confluenceTd"> pENTR_LacI-miRRF4 </TD><TD class="confluenceTd"> A: 58.2 ng/ul 
B: 161.4 ng/ul </TD></TR><TR><TD class="confluenceTd"> 6.29.2011 </TD><TD class="confluenceTd"> Michelle </TD><TD class="confluenceTd"> pENTR_GV16 </TD><TD class="confluenceTd"> A: 49.9 ng/ul 
B: 52.9 ng/ul </TD></TR></TBODY></TABLE></DIV><H3>June 30, 2011</H3><P>11:45AM ~ 12:45PM (Charles, Clara)</P><P>Miniprep AVPR2 for TANGO: 270.9 ng/ul (successful)</P><P>3:00PM ~ 5:00 PM (Charles, Clara)</P><P>PCR primer # 13, 14, 17, 18, 19, 20</P><P>- AVPR2 - TEVs</P><P>Annealing temperature: 58.9 C (13 - 58.6 C; 14 - 55.9 C)</P><P>Extension time: 18 seconds (~1200 bp)</P><P>- Gal4 - VP16</P><P><FONT color="#000000">Ta: 62.4 C (17 - 62.4 C; 18 - 59.4 C)</FONT></P><P><FONT color="#000000">Ext: 13 seconds (~740 bp)</FONT></P><P><FONT color="#000000">- L1L2 Backbone</FONT></P><P><FONT color="#000000">Ta: 65.4 C (19 - 62.4 C; 62.4 C)</FONT></P><P><FONT color="#000000">Ext: 30 seconds (~2500 bp)</FONT></P><P>5:00PM ~ 5:30PM (Charles, Clara)</P><P>Gibson assembly cont'd: inoculated and incubating samples overnight</P><H3>July 1, 2011</H3><DIV class="table-wrap"><TABLE id="minimalist"><TBODY><TR><TH class="confluenceTh"> Time </TH><TH class="confluenceTh"> Notes </TH></TR><TR><TD class="confluenceTd"> 8:00 AM </TD><TD class="confluenceTd"> Color-Coded inventory implemented in the -20 freezer. 
To be implemented in the -80 freezer. See below for details. </TD></TR><TR><TD class="confluenceTd"> 9:00 AM </TD><TD class="confluenceTd"> Morning meeting. </TD></TR><TR><TD class="confluenceTd"> 11:00 AM </TD><TD class="confluenceTd"> Design meeting. Significant amount of information to be 
added to wiki in terms of construction plans and proposed 
circuits. Do ASAP. </TD></TR><TR><TD class="confluenceTd"> 1:30 PM </TD><TD class="confluenceTd"> LR reactions of pDEST_45, pENTR EYFP-FF6, and the five 
inducible minCMV promoters complete. </TD></TR><TR><TD class="confluenceTd"> 2:00 PM </TD><TD class="confluenceTd"> Kenneth's next PCR of Delta started. </TD></TR><TR><TD class="confluenceTd"> 2:00 PM </TD><TD class="confluenceTd"> Cell stocks of the working pEXPR transactivators, TRE:mKate, 
and Hef1a:EBFP2 made. </TD></TR><TR><TD class="confluenceTd"> 2:00 PM </TD><TD class="confluenceTd"> Inoculation of the aforementioned pEXPR vectors for later 
transfections completed. </TD></TR></TBODY></TABLE></DIV><P>1:00 PM ~ <B>AVPR2 - TEVs - Gal4 VP16</B> cont'd:</P><P>Clara did PCR purification and nanodrop to check the concentrations.</P><P>The concentrations were relatively low:</P><P>AVPR2: 20.8 ng/uL</P><P>Gal4-VP16: 30ng/uL</P><P>L1L2 Gibson: 16 ng/uL</P><P>Used 10uL of each sample to run the gel.</P><P>The following are the pictures of the gel (Same picture but different exposure levels), in order of AVPR2; Gal4-VP16; and L1L2 Gibson:</P><P>Expected Length - AVPR2: ~1200 bp; Gal4-VP16: ~740 bp; L1L2 Gibson: ~2500 bp</P><P>Gal4-VP16 and L1L2 Gibson seem to be successful. Should PCR AVPR2 again. </P><P>As for the Gibson assembly practice, the practice samples were over-incubated so I had to throw them away. (Sorry, Charles!)</P><P>Color-Coded Inventory implemented in the -20 freezer.</P><P>Purple: Destination Vectors
Red: Gene Vectors
Green: Promoter Vectors
Yellow: Source/Ordered Vectors
Orange: Expression Vectors
Blue: Primers (5 uM)
Pink: Primer Stocks (100 uM)</P><P>Not yet ported to the -80 freezer cell stocks.</P></DIV><DIV class="content" id="w5content"><H1>Summary of Week 5: July 4 - July 10</H1>
We began looking into using the Goldengate assembly method, but two gene elements contained a cut site that needed to be mutated out. We performed Site-Directed Mutagenesis using the Lightning Kit in the hopes of mutating out the cut site, but results were not successful due to mishandling during the protocol, so the procedure was set to be repeated. Gibson assembly of AVPR2-TEVs-GV16 continues as we re-PCRed the AVPR2 DNA segment and re-run the entire Gibson protocol, picking 20 colonies in a determined attempt to obtain a successful result.


<H3>July 10</H3><P>Jenny and Jon - SDM Colony counts were approximately 50 for the control, 20 for the TEV, and 0 for the Gal4VP16. There is a chance the TEV and Gal4VP16 labels were switched, which we'll find out after sequencing. Six colonies were picked from the TEV plate and inoculations were accidentally run for 22 hours at 37C.</P><H3>July 9</H3><P>Jenny and Jon redid the SDM for Control, Gal4VP16, and TEV using the Lightning Kit, following pages 7 and 8 of this manual: www.qcbio.com/stratagene/210518.pdf
We used 1.5uL 100mM stock of the previously designed primers. The final reaction volume was only approximately 45uL, as we only had about 5uL of 5ng/uL dsDNA for both the Gal4VP16 and TEV reactions. We got the DNA from the dilutions that Divya and Tyler previously used.</P><P>For transformation, we followed the standard protocol instead of the one outlined in the manual and outgrew at 37C and incubated at 30C for 16 hrs.</P><H3>July 8, 2011</H3><DIV class="table-wrap"><TABLE id="minimalist"><TBODY><TR><TH class="confluenceTh"> Sample </TH><TH class="confluenceTh"> Assignees </TH><TH class="confluenceTh"> Procedures </TH><TH class="confluenceTh"> Enzyme(s) Used </TH><TH class="confluenceTh"> Expected Bands </TH><TH class="confluenceTh"> Results </TH></TR><TR><TD class="confluenceTd"> EXPR 1xCI434 (A-C) </TD><TD class="confluenceTd"> Grant/Michelle/Charles </TD><TD class="confluenceTd"> Miniprepped 
Nanodropped 
Restriction Digest </TD><TD class="confluenceTd"> ScaI </TD><TD class="confluenceTd"> 5000 
2000 </TD><TD class="confluenceTd"> </TD></TR><TR><TD class="confluenceTd"> EXPR 4xCI434 (A-C) </TD><TD class="confluenceTd"> Grant/Michelle/Charles </TD><TD class="confluenceTd"> Miniprepped 
Nanodropped 
Restriction Digest </TD><TD class="confluenceTd"> ScaI </TD><TD class="confluenceTd"> 5000 
2000 </TD><TD class="confluenceTd"> </TD></TR><TR><TD class="confluenceTd"> EXPR 4xLexA (A-C) </TD><TD class="confluenceTd"> Grant/Michelle/Charles </TD><TD class="confluenceTd"> Miniprepped 
Nanodropped 
Restriction Digest </TD><TD class="confluenceTd"> ScaI </TD><TD class="confluenceTd"> 5000 
2000 </TD><TD class="confluenceTd"> </TD></TR><TR><TD class="confluenceTd"> EXPR 4xMnt1 (A-C) </TD><TD class="confluenceTd"> Grant/Michelle/Charles </TD><TD class="confluenceTd"> Miniprepped 
Nanodropped 
Restriction Digest </TD><TD class="confluenceTd"> ScaI </TD><TD class="confluenceTd"> 5000 
2000 </TD><TD class="confluenceTd"> </TD></TR><TR><TD class="confluenceTd"> EXPR 7xMnt1 (A-C) </TD><TD class="confluenceTd"> Grant/Michelle/Charles </TD><TD class="confluenceTd"> Miniprepped 
Nanodropped 
Restriction Digest </TD><TD class="confluenceTd"> ScaI </TD><TD class="confluenceTd"> 5000 
2000 </TD><TD class="confluenceTd"> </TD></TR><TR><TD class="confluenceTd"> EXPR UAS:EBFP2 (A-C) </TD><TD class="confluenceTd"> Grant/Michelle/Charles </TD><TD class="confluenceTd"> Miniprepped 
Nanodropped 
Restriction Digest </TD><TD class="confluenceTd"> ScaI </TD><TD class="confluenceTd"> 5000 
2000 </TD><TD class="confluenceTd"> </TD></TR><TR><TD class="confluenceTd"> EXPR AVPR2 A (1-3) </TD><TD class="confluenceTd"> Grant/Michelle/Charles </TD><TD class="confluenceTd"> Miniprepped 
Nanodropped 
Restriction Digest </TD><TD class="confluenceTd"> NcoI </TD><TD class="confluenceTd"> 1356 
3000 </TD><TD class="confluenceTd"> </TD></TR><TR><TD class="confluenceTd"> EXPR AVPR2 B (1-3) </TD><TD class="confluenceTd"> Grant/Michelle/Charles </TD><TD class="confluenceTd"> Miniprepped 
Nanodropped 
Restriction Digest </TD><TD class="confluenceTd"> NcoI </TD><TD class="confluenceTd"> 1356 
3000 </TD><TD class="confluenceTd"> </TD></TR><TR><TD class="confluenceTd"> EXPR AVPR2 C (1-3) </TD><TD class="confluenceTd"> Grant/Michelle/Charles </TD><TD class="confluenceTd"> Miniprepped 
Nanodropped 
Restriction Digest </TD><TD class="confluenceTd"> NcoI </TD><TD class="confluenceTd"> 1356 
3000 </TD><TD class="confluenceTd"> </TD></TR><TR><TD class="confluenceTd"> EXPR AVPR2 D (1-3) </TD><TD class="confluenceTd"> Grant/Michelle/Charles </TD><TD class="confluenceTd"> Miniprepped 
Nanodropped 
Restriction Digest </TD><TD class="confluenceTd"> NcoI </TD><TD class="confluenceTd"> 1356 
3000 </TD><TD class="confluenceTd"> </TD></TR></TBODY></TABLE></DIV><H3>July 7, 2011</H3><P>Charles and Clara @ Knight lab: All four Kan plates had colonies, 40-50 colonies except for one of Deepak's gibson mix reactions, which had ~400 colonies. Inoculated 5 colonies from each reaction into LB-KAN. 20 Falcon tubes in 37C spinning. To be taken out at 2 AM by Grant.</P><H3>July 6, 2011</H3><P>Charles and Clara @ Weiss lab: Ran Gibson reaction with both DM and CH Gibson mixes each, total of 4 reactions run. 0.21 uL of 25.6 ng/uL AVPR2, 0.11 uL of 30 ng/uL GV16, 0.73 uL of 16 ng/uL L1L2 backbone added to each 15 uL Gibson mix. Incubated in thermocycler at 50 C for 1 hour. (Used formula to obtain required volume for 7 fmol: 7 x 0.65/(conc ng/uL) x length / 1000.</P><P>Charles and Clara @ Knight lab: Gibsons transformed into 10G cells (0.9mL SOC), incubated spinning for 1 hour at 37C. Centrifuged for 3 min max speed (14000), removed 500uL of supernatant, resuspended the remaining &amp; Plated 100 uL of each tube on KAN. Incubating plates 37C overnight (6:30pm ~ right after morning meeting tmr). The leftover tubes (Gibson + transformation) are in the iGEM fridge.</P><P>Divya and Tyler's SDM:</P><P>We used the Agilent QuikChange II SDM Kit and followed the protocol in its manual (<SPAN class="nobr"><A href="http://www.genomics.agilent.com/files/Manual/200523.pdf).&amp;nbsp" class="external-link" rel="nofollow">http://www.genomics.agilent.com/files/Manual/200523.pdf).&amp;nbsp</A></SPAN>;</P><P><INS><B>GV16 Mutagenesis</B></INS> (to remove BsaI site)</P><DIV class="table-wrap"><TABLE id="minimalist"><TBODY><TR><TH class="confluenceTh"> Amount of dsDNA </TH><TH class="confluenceTh"> 10 ng [A] </TH><TH class="confluenceTh"> 20ng [B] </TH><TH class="confluenceTh"> 50ng [C] </TH></TR><TR><TD class="confluenceTd"> dsDNA template </TD><TD class="confluenceTd"> 1.89 uL </TD><TD class="confluenceTd"> 3.78 uL </TD><TD class="confluenceTd"> 9.45 uL </TD></TR><TR><TD class="confluenceTd"> 10x Reaction Buffer </TD><TD class="confluenceTd"> 5 uL </TD><TD class="confluenceTd"> 5 uL </TD><TD class="confluenceTd"> 5 uL </TD></TR><TR><TD class="confluenceTd"> GV16 Mutagenesis Primer (Forward) </TD><TD class="confluenceTd"> 1.25 uL </TD><TD class="confluenceTd"> 1.25 uL </TD><TD class="confluenceTd"> 1.25 uL </TD></TR><TR><TD class="confluenceTd"> GV16 Mutagenesis Primer (Reverse) </TD><TD class="confluenceTd"> 1.25 uL </TD><TD class="confluenceTd"> 1.25 uL </TD><TD class="confluenceTd"> 1.25 uL </TD></TR><TR><TD class="confluenceTd"> dNTP mix </TD><TD class="confluenceTd"> 1 uL </TD><TD class="confluenceTd"> 1 uL </TD><TD class="confluenceTd"> 1 uL </TD></TR><TR><TD class="confluenceTd"> H2O (to get to 50 uL) </TD><TD class="confluenceTd"> 39.61 uL </TD><TD class="confluenceTd"> 37.72 uL </TD><TD class="confluenceTd"> 32.05 uL </TD></TR></TBODY></TABLE></DIV><P>Then added 1uL HF DNA Polymerase</P><P><INS><B>TEV Protease Mutagenesis</B></INS> (to remove BsaI site)</P><DIV class="table-wrap"><TABLE id="minimalist"><TBODY><TR><TH class="confluenceTh"><FONT color="#660000">Amount of dsDNA</FONT></TH><TH class="confluenceTh"><FONT color="#660000"><DEL>10 ng [A]</DEL></FONT></TH><TH class="confluenceTh"><FONT color="#660000">20ng [B]</FONT></TH><TH class="confluenceTh"><FONT color="#660000">50ng [C]</FONT></TH></TR><TR><TD class="confluenceTd"><FONT color="#000000">dsDNA template</FONT></TD><TD class="confluenceTd"><FONT color="#000000"><DEL>1.98 uL</DEL></FONT></TD><TD class="confluenceTd"><FONT color="#000000">3.96 uL</FONT></TD><TD class="confluenceTd"><FONT color="#000000">9.92 uL</FONT></TD></TR><TR><TD class="confluenceTd"><FONT color="#000000">10x Reaction Buffer</FONT></TD><TD class="confluenceTd"><FONT color="#000000"><DEL>5 uL</DEL></FONT></TD><TD class="confluenceTd"><FONT color="#000000">5 uL</FONT></TD><TD class="confluenceTd"><FONT color="#000000">5 uL</FONT></TD></TR><TR><TD class="confluenceTd"><FONT color="#000000">TEV Protease Mutagenesis Primer (Forward)</FONT></TD><TD class="confluenceTd"><FONT color="#000000"><DEL>1.25 uL</DEL></FONT></TD><TD class="confluenceTd"><FONT color="#000000">1.25 uL</FONT></TD><TD class="confluenceTd"><FONT color="#000000">1.25 uL</FONT></TD></TR><TR><TD class="confluenceTd"><FONT color="#000000">TEV Protease Mutagenesis Primer (Reverse)</FONT></TD><TD class="confluenceTd"><FONT color="#000000"><DEL>1.25 uL</DEL></FONT></TD><TD class="confluenceTd"><FONT color="#000000">1.25 uL</FONT></TD><TD class="confluenceTd"><FONT color="#000000">1.25 uL</FONT></TD></TR><TR><TD class="confluenceTd"><FONT color="#000000">dNTP mix</FONT></TD><TD class="confluenceTd"><FONT color="#000000"><DEL>1 uL</DEL></FONT></TD><TD class="confluenceTd"><FONT color="#000000">1 uL</FONT></TD><TD class="confluenceTd"><FONT color="#000000">1 uL</FONT></TD></TR><TR><TD class="confluenceTd"><FONT color="#000000">H2O (to get to 50 uL)</FONT></TD><TD class="confluenceTd"><FONT color="#000000"><DEL>39.52 uL</DEL></FONT></TD><TD class="confluenceTd"><FONT color="#000000">37.54 uL</FONT></TD><TD class="confluenceTd"><FONT color="#000000">31.58 uL</FONT></TD></TR></TBODY></TABLE></DIV><P>Then added 1uL HF DNA Polymerase</P><P>I accidentally added too much Primer to the 10ng tube, so it was scrapped.</P><P>After going through the Thermal Cycler, 1uL of DpnI was added to each remaining sample. The samples were then put in the incubator at 37 degrees C for 1 hour, then moved to the freezer.</P><H3>July 5, 2011</H3><P><FONT color="#000000">AVPR2 re-PRC: Charles and Clara re-PCR-ed the AVPR2 using primers 13 and 14. This time, however, we used 20s extension time instead of 18s. </FONT></P><P><FONT color="#000000">Tyler: Did LR at 4:30 pm for pEXPR 4-5 UAS:EBFP2. Will transform tomorrow morning. Plan on also getting LacI (without the Krab) from Weiss lab for use as a weak repressor.</FONT></P></DIV><DIV class="content" id="w6content"><H1>Summary of Week 6: July 11 - July 17</H1>
In the sixth week we continued re-attemping previous failures using modified protocols or higher quality DNA in hopes of obtaining successful reactions. We also began looking at Cadherins and the possibility of using them as a clumping mechanism for mammalian cells. In order to visualize Cadherins, we needed some sort of fluorescent color, so construction of NCadherin-EGFP (NCadherin is one of many types of cadherin) began. DNA for NCad-EGFP was ordered, but needed to be in a format such that we could LR react it with the different promoters we have. This is done by attaching attB sites to flank the NCad-EGFP gene and then performing the BP reaction, which generates LR reaction-compatible parts. PCR of the attB sites was successful. By this week we have also begun work with mammalian cell cultures. To explore the limitless possibilities of synthetic biology, a few of us took it upon themselves to look into other interesting gene components, such as the Caspase gene and the FF4 tag.

<H3> July 17</H3><B>Grant:</B> Worked on Evoware robot software. Close to perfecting the restriction map protocol and data passage between subroutines and main programs.
<B>Divya and Michelle:</B> Began a miniprep of many LR constructs.

<H3> July 16</H3><B>Charles:</B> re-PCR-ed attB sites onto NCad-EGFP. Also PCR-ed previously PCR-ed NCad-EGFP.&amp;nbsp; (20 uL Phusion mix, 15 ul sH2O, 2 uL forward, 2 uL reverse, 1 ul template). Program: 45 sec elongation, 54.9 C anneal. Used iGEM 34 and 35 primers.
Ran 1% agarose gel for 1 hour at 116 V with the following: Lane 2 5 uL Hyperladder I, Lane 3 100 ng pDON221, Lane 4 100 ng plasmid NCad-EGFP, Lane 5 100 ng re-PCR product of previous PCR product, Lane 6 100 ng new PCR product from plasmid, Lane 7 100 ng previous PCR product from plasmid, Lane 8 5 uL Hyperladder I.
Gel picture rather strange. Showed no bands for NCad-EGFP PCR products. One big but dim band for pDON221, seems to be greater than 10kb. Gel got photobleached.
<B>Divya:</B> Inoculated LRs from 7/14 and 7/13.
<B>Tyler and Mariola:</B> Passaged cells. Imaged Dox induced cells.
<B>Kenneth:</B> Transfected for dox stable level experiment. Transformed 15 LR's. 

<H3> July 15</H3><B>Grant:</B> Miniprepped the TEV Protease and Gal4-VP16 SDM samples with Ken. Set up sequencing reactions for the same SDM samples with Sam. Ran a second restriction mapping to verify the pDisplay-Vasopressin construct, shown [here|pDisplay-MYC-Vasopressin Assembly].
<B>Kenneth: </B> Passaged cells for transfection.
<B>Michelle &amp; Divya: </B>Transformed LRs from 7/14.
<B>Mariola and Tyler: </B>Imaged tri color experimental results, and added Dox to wells that required induction.

<H3> July 14</H3><B>Grant: </B>Miniprepped and restriction mapped 30 samples (Duplicates of 9 LR constructs, 8 AVPR2 Gibson samples, and 4 Cadherin BP reactions). Note that digestion with SalI appears to generate effective restriction maps for essentially all LR constructs. An image of the gel combined with a Geneious virtual gel of the expected bands can be seen [here|Restriction Maps]. Submitted the five apparently good colonies for sequencing.
<B>Jenny and Jon:</B> Inoculations done at 11AM. Colony count of TEV protease: 3, GAL4-VP16: 32. Six colonies picked from GAL4-VP16, all 3 picked form TEV.
<B>Mariola and Tyler:</B> Transfected cells ~4pm for Constitutive Color and LacI-KRAB repression experiments.
<B>Divya &amp; Michelle:</B> (in Weiss Lab)
Transformed the following cells:
<TABLE id="minimalist"><TBODY><TR><TH>Tube</TH><TH>Destination</TH><TH>Promoter</TH><TD>Gene</TD></TR><TR><TD>1</TD><TD>3-4 (702:9)</TD><TD>minCMV-1xCI434</TD><TD>mKate (701:22)</TD></TR><TR><TD>2</TD><TD>3-4 (702:9)</TD><TD>minCMV-4xCI434</TD><TD>mKate (701:22)</TD></TR><TR><TD>3</TD><TD>3-4 (702:9)</TD><TD>minCMV-4xMnt1</TD><TD>mKate (701:22)</TD></TR><TR><TD>4</TD><TD>3-4 (702:9)</TD><TD>minCMV-7xMnt1</TD><TD>mKate (701:22)</TD></TR><TR><TD>5</TD><TD>3-4 (702:9)</TD><TD>minCMV-4xLexA</TD><TD>mKate (701:22)</TD></TR><TR><TD>6, 7</TD><TD>4-5 (702:10)</TD><TD>UAS-Gal4 (700:10)</TD><TD>mKate (701:22)</TD></TR><TR><TD>8</TD><TD>4-5 (702:10)</TD><TD>UAS-Gal4 (700:10)</TD><TD>eBFP2 (703:12)</TD></TR></TBODY></TABLE>



Did the following LRs:

<TABLE id="minimalist"><TBODY><TR><TH>Tube</TH><TH>Destination</TH><TH>Promoter</TH><TH>Gene</TH></TR><TR><TD>1</TD><TD>2-3</TD><TD>Hef1a</TD><TD>Notch-Gal4</TD></TR><TR><TD>2</TD><TD>2-3</TD><TD>Hef1a-LacO</TD><TD>Notch-Gal4</TD></TR><TR><TD>3</TD><TD>2-3</TD><TD>Hef1a</TD><TD>Delta-mCherry</TD></TR><TR><TD>4</TD><TD>2-3</TD><TD>Hef1a</TD><TD>mKate</TD></TR><TR><TD>5</TD><TD>2-3</TD><TD>TRE</TD><TD>rtTA3</TD></TR><TR><TD>6</TD><TD>2-3</TD><TD>UAS-Gal4</TD><TD>rtTA3</TD></TR><TR><TD>7</TD><TD>2-3</TD><TD>TRE</TD><TD>LacI-miRFF4</TD></TR><TR><TD>8</TD><TD>3-4</TD><TD>TRE</TD><TD>Notch-Gal4</TD></TR><TR><TD>9</TD><TD>3-4</TD><TD>UAS-Gal4</TD><TD>Notch-Gal4</TD></TR><TR><TD>10</TD><TD>3-4</TD><TD>minCMV-7xMnt1</TD><TD>Notch-Gal4</TD></TR><TR><TD>11</TD><TD>3-4</TD><TD>UAS-Gal4</TD><TD>LacI-miRFF4</TD></TR><TR><TD>12</TD><TD>3-4</TD><TD>UAS-Gal4</TD><TD>Delta-mCherry</TD></TR><TR><TD>13</TD><TD>4-5</TD><TD>UAS-Gal4</TD><TD>Mnt-VP16</TD></TR><TR><TD>14</TD><TD>3-4</TD><TD>UAS-Gal4</TD><TD>LacI-KRAB</TD></TR><TR><TD>15</TD><TD>4-5</TD><TD>UAS-Gal4</TD><TD>eYFP</TD></TR><TR><TD>16</TD><TD>4-5</TD><TD>4xLexA</TD><TD>eBFP2</TD></TR><TR><TD>17</TD><TD>4-5</TD><TD>4xMnt1</TD><TD>mKate</TD></TR></TBODY></TABLE><H3> July 13</H3>
Did the following LRs at the Weiss Lab.
<I>LR Reaction:</I>
1uL 10fm Destination
1uL 5fm Promoter
1uL 5fm Gene
0.5uL LR Clonase

<TABLE id="minimalist"><TBODY><TR><TH>Tube #</TH><TH>Destination</TH><TH>Promoter</TH><TH>Gene</TH></TR><TR><TD>1</TD><TD>3-4 (702:9)</TD><TD>minCMV-1xCI434</TD><TD>mKate (701:22)</TD></TR><TR><TD>2</TD><TD>3-4 (702:9) \\</TD><TD>minCMV-4xCI434 \\</TD><TD>mKate (701:22) \\</TD></TR><TR><TD>3</TD><TD>3-4 (702:9) \\</TD><TD>minCMV-4xMnt1 \\</TD><TD>mKate (701:22) \\</TD></TR><TR><TD>4</TD><TD>3-4 (702:9) \\</TD><TD>minCMV-7xMnt1 \\</TD><TD>mKate (701:22) \\</TD></TR><TR><TD>5</TD><TD>3-4 (702:9) \\</TD><TD>minCMV-4xLexA \\</TD><TD>mKate (701:22) \\</TD></TR><TR><TD>6, 7</TD><TD>4-5 (702:10)</TD><TD>UAS-Gal4 (700:10)</TD><TD>mKate (701:22) \\</TD></TR><TR><TD>8</TD><TD>4-5 (702:10) \\</TD><TD>UAS-Gal4 (700:10) \\</TD><TD>eBFP2 (703:12)</TD></TR></TBODY></TABLE><B>Kenneth:</B> important stuff. Prepped cells in 24 well plates for transfection \~2x10^5 cells/well. Ready for transfection tomorrow.<B>Sam:</B> wrote up Caspase Test, TRE:rtTA propagator, TRE/UAS High Pass, and EYFP Pulse. Ordered Mnt-VP16-FF4 primers. Thought of a cool derivation of the change in repression that a DD tag offers using the data Clontech provides. Got CC3D to make videos.<B>Clara: </B>B-arrestin 2/TEV-protease Gibson assembly. B-arrestin 2 has not arrived yet, to be ordered from another source. ~8 days delay in gibson<B>Grant:</B> Miniprepped twenty samples from 11 AM to 1 PM with Ken and Tyler. Miniprepped and restriction mapped pDisplay-Vasopressin construct; more details about the construction and the verification can be found [here|pDisplay-MYC-Vasopressin Assembly]. Finally restriction mapped inoculations done by Kenneth and myself; the gel results are shown below.

﻿ 

A first glance at the gel implies that almost all of the additional colonies picked failed, although one construct (pEXPR_45_minCMV-7xMnt_mKate) has only two bands, save for the roughly 500 base pair deletion that has been observed in the past. SDM redone using LightingKit again. 50ng DNA used following pages 7 and 8 of this manual: www.qcbio.com/stratagene/210518.pdf
We used 1.5uL 100mM stock of the previously designed primers. DNA was taken from the A vials of our stocks in the freezer. During the thermocycler phase, some random buffoon that was looking for an ID label on the side of the machine randomly turned off the thermocycler and didn't turn it back on. We think based on time that the cycles should have finished though. We did the DpnI digest for 10 minutes as opposed to 5 to ensure the template strand was digested. We transformed following our standard protocol with a 60min outgrowth in SOC at 37C. Jon plated and incubated it at 37C overnight.

<H3>July 12</H3>
Re-PCRed AVPR2, GV16, L1L2 backbone. @ 10 AM PCR purified. Concentrations: AVPR2 (31.3), GV16 (60.4), L1L2 (28.5). Ran Gel three Gibson parts, using 2 uL DNA, 2 uL loading buffer, 6 uL H2O in each lane. Lanes 2 to 5: DNA ladder, AVPR2, GV16, L1L2. Volume of each to add into Gibson: uL for 10femtomoles = 10*0.65 / (concentration in ng/uL) * (exact length of PCR product)/1000
AVPR2: 10*0.65/(31.3)*(1.2) = 0.25 uL
GV16: 10*0.65/(60.4)*(0.74) = 0.08 uL
L1L2: 10*0.65/(28.5)*(2.5) = 0.57 uL
Gibson reaction run at 11 AM.<H3>July 11</H3>
Re-doing PCR of AVPR2. New AVPR2 and PCR-ed NCad-EGFP look good.&amp;nbsp;Gel result as below. New AVPR2 and PCR-ed NCad-EGFP look good.
Lanes 2 and 4: 10kb DNA ladder. Lane 5: Old AVPR2 (still plasmid). Lane 6: New PCR-ed AVPR2. Matches expected length of \~1.2kb. Lane 7: NCad-EGFP miniprepped DNA (from stab, Deepak). Lane 8: PCR-ed NCad-EGFP. Matches expected length of \~3.4kb.&amp;nbsp;Lanes 2 and 4: 10kb DNA ladder. Lane 5: Old AVPR2 (still plasmid). Lane 6: New PCR-ed AVPR2. Matches expected length of \~1.2kb. Lane 7: NCad-EGFP miniprepped DNA (from stab, Deepak). Lane 8: PCR-ed NCad-EGFP. Matches expected length of \~3.4kb.&amp;nbsp;
</DIV><DIV class="content" id="w7content"><H1>Summary of Week 7: July 18 - July 24</H1>
Part of the team worked on creating protocols for a robot liquid handler to run the usual lab reactions that we run. We are hoping that the robot can replace us and do our liquid-related lab work for us. Various dry test runs were done. We transfected various DNA parts that we have into Hek293 cells, and results show that most of our DNA works. We investigated the TRE-rtTA3 system as well as the UAS-Gal4 system. Both seemed to be functional.

<H3>July 24</H3>
Co-cultured the delta notch again. other tissue culture stuff. miniprepped LR's 1 and 6 and Hef1alaco: pdisplay vasopressin. Aliquotted DNA for tricolor experiment. miniprepped AVPR2. Dox induced Mnt Part Characterization experiment. Drank lots of tea. Added .9ug Dox to Tre:Gal4, UAS:mKate experiment to get 1.5 ug/mL. Also did tri-color experiment transfections. Imaged previous day's transfections. AmCyan failed. Very good transfection efficiency for Dox induced TRE: Gal4, UAS: mKate experiment







<H3>July 23</H3>



Kenneth - inoculated cultures from redone LR's. Organized the lab's DNA stocks. did some transfections for Delta Notch co-culture experiment. investigated caspase 3 sequence. other clerical stuff. updated wiki.



Grant - worked on organization of parts with Kenneth, but who can say?



Mariola - transfected Mnt Part Characterization experiment. 



Tyler - transfected Tre:Gal4, UAS:mKate experiment, as well as Delta, Notch, UAS:AmCyan





<H3>July 22</H3>



Jenny - [Computer generated circuits with three components and with behavior descriptions.|iGEM2011:Jenny's Notebook] Needs human pruning/refining, 'cause most seems garbage&amp;nbsp; (computer is sad about intelligence).





Inoculated Charles' plates at 5:00pm. Restriction mapped 6 LRs.



Clara - Ran 3 Gibson reactions: DRD2-TEVs-GV16, pDisplay-CCL5, and pDisplay-IL8



Kenneth - So apparently the LR's from yesterday were put in the freezer. So, they have to be incubated a little longer for today. Transform at 8. FACS and passed cells for transfection.&amp;nbsp;



Mariola - Passed cells for transfx. Shenanigans.





Tiffany - robot stuff



Divya, Michelle - Miniprepped and nanodropped samples which can be found in&amp;nbsp;{color:#003333}\[{color}{color:#003333}Divya's Personal Notebook{color}\|iGEM2011:Divya's Notebook\]. Restriction mapped these plasmids for digests to be done on Monday.



Jon - Restriction mapped our 30 miniprep samples on Geneious (reannotated screwed up att sites on all the DNA too.. will actually run on Monday when we can decide to send out for sequencing). Sent protocol to Kenneth for knockdown.



Tyler - prepared cells for FACS and passed cells to prepared for transfections tomorrow and Sunday











<H3>July 21</H3>



Kenneth: miniprepped again, again bad yields. I believe there was some contamination with an ampicillin resistant microbe. It outcompeted the e coli and thus we get no DNA from them. Will redo LR's for LR 1 and 6. Also did LR for Hef1a-lacO:pDisplay-vasopressin.&amp;nbsp;



Charles: Re-transformed and plated AVPR2-TEVs-GV16 and NCad-EGFP at 12:30 PM (Total of 4 plates). Should be ready for inoculation tonight. Reorganizing LR page and tracking down owners for actual LR statuses. Did Sam's gel extraction and sent him home to rest. 3 samples (600 bp at ng/uL, 2.8 kb at ng/uL, 3kb at ng/uL) in black/silver fridge. Inoculated Sam's 8-1 to 8-3 and 9-1 to 9-3, discarded 9-1 because it leaked. In 37 incubator. Inoculated AVPR2 2-part gibson, the only one that grew colonies (~50). Two will be rushed for sequencing this morning.





Tyler: Prepared cells for FACS. Will prepared more cells for transfection tomorrow. Hopefully we have more Lipofectamine because we finally have some more LR reactions that worked\! In the afternoon, did minipreps of Hef1A:EBFP2 and Hef1A-LacO:Delta-mCherry for Mariola. fun. Also updated the [iGEM2011:LacI.Krab Experiments]&amp;nbsp;(pretty pictures).&amp;nbsp;Will have a ton of cells ready for transfection Saturday.&amp;nbsp;



Louis: Modified Robot Script to handle multiple samples. &amp;nbsp;Should be ready for testing tomorrow morning. &amp;nbsp;Next up is Enzyme Choice and BSA Support.



Divya, Michelle, Clara: Miniprepped, nanodropped, and inoculated several samples. Tables of all samples as well as timings for when each activity was performed are located in ﻿[Divya's Personal Notebook|iGEM2011:Divya's Notebook]. I additionally emailed out about the contamination of the water that was being used as a blank for the Nanodrops. PCRed.



Jenny - Helped Charles inoculate



Grant - Did PCRs of the low-yield products from Wednesday in addition to PCRs for a number of orthogonal pDisplay constructs. After gel extraction, will attempt Gibson and Golden Gate reactions of the PCR products, ideally forming three new ENTR vectors.







<H3>July 20</H3>



Kenneth-re-inoculated LR's 1 and 6 due to bad minipreps. Will miniprep tomorrow and rest digest and send LR's 1-7 off for sequencing.&amp;nbsp;



Grant: Various PCRs and purifications of parts for the Golden Gate version of AVPR2. Issues with concentrations / yield. Will be redoing certain parts again with different conditions today. Starting a new primer for a potentially easier Golden Gate.



Charles - AVPR2 Gibson plates had 0 colonies, because they were erroneously plated on Amp. PCR Reaction for NCad-EGFP. Gel extraction looks perfect for NCad-EGFP. Clara performed BP reaction, then I transformed both AVPR2 gibsons and NCad-EGFP and plated onto Kan plates at 8 PM.



Clara - Gel extraction for NGFP. Nanodrop concentrations were 11.6 ng/uL for NGFP and 14.8 for NGFP PCR with DMSO. BP reaction for both samples.&amp;nbsp;



Louis - Everyday I'm programming. &amp;nbsp;Did dry runs and wet runs of the digest protocol. &amp;nbsp;Should be ready after a bit more front-end management and hopefully support for multiple samples.



Tyler - miniprepped DNA with Mariola. Imaged cells for LacI-Krab Experiment. Added Dox (.825ug for 550uL of cells = 1.5 ug/mL) unfortunately to all cells, so we wont have a \-Dox control for FACS.



Mariola - miniprepped with Tyler: 5/6 success rate. Reinnoculated Hef1a:EBFP2 along with Hef1a:Delta_mCherry. To be miniprepped for tomorrow. Went to EBICS Multi-institutional meeting\- talked up iGEM. Preparing powerpoint for tomorrow's EBICS progress presentations.





Michelle, Divya\- Reinoculated expression vectors that had a low miniprep concentration or whose inoculation previously didn't work, prepped 5 samples for sequencing and sent them to genewiz with Jon, did digest reactions for expression vectors from gel that failed yesterday and Sam ran the gel for them\- same issue occurred with gel today (images can be found [here|^7-20-11 Gel2.tif] and [here|^July20gel.tif].) Miniprepped yesterdays four inoculations of sequence verified Weiss Lab expression vectors. Tomorrow we plan on miniprepping and digesting inoculations from today and starting new LRs for the failed attempts which were due to a&amp;nbsp;lack of gene and&amp;nbsp;promoter entry vectors.



Sam: miniprepped LR colonies that failed to miniprep last time. Sent out correct-looking LRs. Did all the PCRing for Delta-FF4x4, Delta-FF4x1, Mnt-FF4x4; will do finish the job and send the ENTRs out for sequencing tomorrow.



Jon - Colony counts from yesterday's transformations:

<TABLE id="minimalist"><TBODY><TR><TH>Part</TH><TH>Count</TH></TR><TR><TD>﻿A pEXPR_3-4_UAS_eYFP-FF4x4</TD><TD>35 \\</TD></TR><TR><TD>B pEXPR_3-4_TRE_Caspase3</TD><TD>high 100s \\</TD></TR><TR><TD>C pEXPR_2-3_TRE_AmCyan-miRFF4</TD><TD>12 \\</TD></TR><TR><TD>D pEXPR_2-3_TRE_rtTA-DD</TD><TD>100-200 \\</TD></TR><TR><TD>E pEXPR_2-3_TRE_LacI</TD><TD>50</TD></TR><TR><TD>F pEXPR_2-3_TRE_CDH2-2A-eYFP</TD><TD>\~1000 \\</TD></TR></TBODY></TABLE>



Inoculated 5 colonies from each.

Helped Divya and Michelle prep samples for sequencing.



Jenny - Helped Jon inoculate









<H3>July 19</H3>
AVPR2 Gibson attempt #3.&amp;nbsp;﻿Running PCR (AVPR2 and GV16, L1L2, GV16-L1L2). Retrying PCR with 1.5 uL DMSO added. Moar Robot Programming. &amp;nbsp;Moar Debugging. &amp;nbsp;(Mostly debugging)
Transfected cells for LacI-Krab repression experiments for FACS. Innoculated 2-3 TRE:LacIKRAB, Hef1a:rtTA3, 1-2 Hef1A:eBFP2, 3-4 Hef1A-lacO:eYFP, 4-5 TRE:mKate B, 2-3 Tre:Delta_mCherry from cell stocks.
Multiple PCRs for the Golden Gate assembly method for AVPR2-TEVs-Gal4-VP16. Did a gel extraction of the two PCR'ed parts\- one had low yield and was redone, will be gel extracted in the morning. Did a transformation of two Gibson AVPR2-TEVs-Gal4-VP16 attempt and five additional LR reactions. 
Golden Gate reaction on the AVPR2-TEVs-Gal4.VP16 construct (with additional ligation at the end) and hope for the best. Miniprep Golden Gate reactions and send off for sequencing. (Optimally, we might be able to have someone run the DNA to Genewiz?) 
Also obtained CHO cells and thawed. Will let them expand. Miniprepped inoculations from 7/18. Nanodropped and digested them as well. Gels were run. Inoculated colonies from plates from Weiss lab. Poured 8 gels. Tomorrow, we plan on sequencing the five successful plasmids, LRing or re-inoculating those that failed, and miniprepping inoculations from today.
Transformed Jenny's 6 LRs from yesterday. Reaction F (NCad-2A-eYFP) didn't seem like it was the full reaction volume (had trouble getting 1 uL even after tapping down) and Reaction A was bubbly/soapy while plating for some reason.







<H3>July 18</H3>



Louis - Working on getting a basic Restriction Digest protocol working on the Robot



Grant - Miniprepped the &quot;bad&quot; overnight cultures discussed in the morning meeting (Michelle and Divya's inoculations from 7/16/11). Did restriction mapping of the ten samples that had concentrations over 100 ng/uL with Michelle using SalI-HF. Results of the restriction mapping can be found [here|Restriction Maps]; many of the constructs were apparently successful and will be sent for sequencing tomorrow. Finalized the virtual Restriction Digestion protocol for use on the Weiss Lab robot.



Charles - Troubleshooting AVPR2 primer design. Suspect a shady 10 bp 60% identity overlap between two primers not meant to Gibson together. Looking at sequencing from before, this seems to be what's happening. New primers proposed for this Gibson, running it by Deepak.



Divya, Michelle, Clara - Re-inoculated cultures that were left overnight using colonies from plates that Deepak had&amp;nbsp; re-transformed. Will miniprep tomorrow. Nanodropping.



Jenny - Did 6 LR reactions, will transform tomorrow.



Tiffany - Did work with the plate scanner. Work with the restriction mapping in vitro.



Ken and Sam: Miniprepped new samples (non-overnight samples). Ken did tissue culture work. Nanodropping (entry vector propagations).



Tyler: Did tissue culture work: Imaged LacI-Krab Experiment, 72 hours after adding Dox (see&amp;nbsp;[iGEM2011:LacI.Krab Experiments]). Prepared cells for transfection. Will transfect on Tuesday, add Dox on Wednesday, FACS on Thursday and Friday.



Mariola: same as Tyler.



Jon - Suspecting that the melting points for the primers are too low as the kit recommends at least 78C and the current temperatures are 67C and 72C. Considering that the extension temperature is at 68C, the primer bind is probably just not efficient enough. Designed new primers to be ordered in Geneious.


</DIV><DIV class="content" id="w8content"><H1>Summary of Week 8: July 25 - July 31</H1>
This week we underwent a momentous drive to create more LRs. A large list of promoter-gene pairs was conceived of and we began to run through LR reactions in somewhat of a factory manner. This occupied much of our time. We also received and prepared DNA parts from Elowitz's group in Caltech. We also got trained on using the FACS machine and began to get quantitative data on our transfections.

<H3>July 31</H3>
Today we passed cells for transfections tomorrow and imaged cells from 7/29 transfection with free 
delta. We did 14 PCR reactions to generate parts for the CCR5-TEVs-GV16,  HRH4-TEVs-GV16, and 
B-Arrestin-2-TEV-Protease ENTR vectors. Gel extracted, completed Gibson reaction and transformed 
Gibson reactions. Miniprepped and restriction mapped several  previous LRs, including four samples 
from Tyler's LRs (All of which were  Notch-Gal4 under different promoters). Only the  
pEXPR_23_Hef1a-LacO_Notch-Gal4 construct had proper bands, and will be  sent for sequencing. 
Inoculated quintuples of DRD2-TEVs-GV16 and  pDisplay-CCL5-MYC. Was disappoint. [Ken and I noted a 
possible issue with the UAS promoter from Elowitz.|Sequence Alignments] FACS of Testing 
UAS:LacIKRAB (day 2), and of Knockdown Pulse (day 1). Also imaged Pulse. #tired

<H3>July 30</H3>
Prepped samples for FACS. Added free delta to cells. Passed more cells. Discovered that 
pDisplay-IL8-MYC looks good. FACS of Mnt and CI434 and Testing UAS: LacIKRAB (day 1) experiments. 
(6pm) Added Shield, Dox and Free Delta in various combinations to 3 INPUT AND Gate (8pm), 
cocultured cells for Knockdown Pulse. #funfunfun

<H3>July 29</H3>
Did extensive lab cleanup! Inoculated four new parts (one LR and three Gibson reactions\- 
pEXPR_23_Hef1a-LacO_TEV-Protease, pDisplay-CCL5, pDisplay-IL-8, and DRD2-TEVs-GV16, respectively).
<I>Note: Colony counts were roughly 10, 50, 50, and 50, respectively. 
Gibson reaction was conducted at 52 C rather than 50 C due to previously low or zero colony counts; 
this may be worth generalizing to future Gibson reactions, assuming 40 bp of overlap.</I> We 
restriction digested 11 samples (x 2 each). Incubating in 37C 1:10 ~ 4:10. Sent off Hef1a-lacO:pdisplay-vaso-myc
for sequencing. Also did second set of transfections to test free delta activation of Notch. 
Transfected miRNA Knockdown Pulse, 3 input AND gate. Imaged Mnt and CI434. Got SHIELD from Noah. 
1ug/mL to fully stabilize.<H3>July 28</H3>
Transformed Divya's 4 LRs started on 7/27. Used 1.5 uL of each LR reaction to transform. Started 
miniprepping propagations EXPR-Tyler plasmids. Inoculated Divya's LRs started on 7/26. 7 out of 
13 plates had no colonies. The 6 that had colonies had fewer than 10 colonies, may possibly be 
background. pEXPR_2-3_UAS_amCyan-miRFF4 looks good based on sequencing results. Made a cell stock 
and put it into the \-80C. Also made cell stock of the pEXPR_2-3_TRE_Caspase3 despite sequencing 
issues, as the sequence came back consistent. Appears to be two forward TEV protease primers in new 
SDM, so it's being put on hold temporarily. Passaged cells. Cocultured UAS: LacIKRAB experiments. Induced Mnt and CI434 with Dox.

<H3>July 27</H3>
Sent off sequencing samples for last of LR's in personal pipeline. Miniprepped 
hef1a-lacO:pDisplay-vaso-myc LR. Transfected Testing:UAS LacIKRAB, Mnt (Attempt #2) and CI434 Parts
Characterization Experiments. Inoculated additional pDisplay-Myc-Vasopressin and two AVPR2-TEVs-GV16 
attempts via Golden Gate. Did an LR of pEXPR_23_Hef1a-LacO_TEV-Protease for a future Tango test.
Spun down samples from the day at 5 AM and inoculated colonies from six of the 13 LR reactions
(all plates that had colonies had very low colony counts between 1 and 7 colonies per plate).
Finalized entries for the [Gene Synthesis Challenge]. Transformation for Divya's LRs 
(numbered 1-19) from yesterday, except for the ones that were incomplete due to lack of promoters
(# 1, 2, 5, 10, 14, 15). Total of 13 plates are being incubated at 37 C, 2:30 pm ~
Inoculated transformations from 7/26 of BP reactions, minCMV-1xCI434, minCMV-4xCI434, Hef1a, 
and minCMV-7xMnt 12 PM (Jon). Transformed LRs from 7/26, plated 37C at 2:30 PM (Clara). Cell stocks 
of 7 samples including UAS citrine, CMV-Notch-Gal4 1 and 2, CMV-TO Delta-mCherry 1 and 2, and UAS
mKate to be made by Divya. Made cell stocks of samples provided by Deepak and Charles. It turns out 
Grant/Ken had disposed of missing plasmids during an earlier cleaning session. I haven't been 
able to find any duplicates, so all 4 will have to be re-made. Inoculated 22 colonies from yesterday's
transformations. Pyevo stuff.&amp;nbsp; Downloaded new version.

<H3>July 26</H3>
Inoculated 3 colonies from 1-2 Hef1a:display-vasopressin LR. Transfected CHO cells. Thawed out
new Hek293 cells free of contamination. Verified DNA that did not have cell stocks, re-organized 
Expression vector box, combining aliquots to leave two final aliquots for each expression vector.
Genewiz read \~1.2 kb, GV16 perfect, and about 200 bp of solid AVPR2 read before sequencing died 
off. AVPR2 is successful, should sequence again with forward primer to verify complete sequence.
6 AM BP reaction on NCad-EGFP, duplicates done of each of three successfully gel extracted PCR reactions. 
Introducing new workflow efficiencies. pEXPR_2-3_TRE_rtTA-DD sequencing results look good\! 
Made a cell stock and put in the \-80C. pEXPR_3-4_TRE_Caspase3 sequencing results look very
odd. Big region of partial match, but it looks like our Caspase3 sequence on Geneious is 
incorrect or our actual DNA is weird.&amp;nbsp;Grew up C2 and E4 again (incubated at 4:20pm) to prep
for sequencing as they have the potential to be good. Transformed 6 BPs, an expression 
vector, and 4 promoters for propagation. Got pyevo working.&amp;nbsp; Read through pyevo several 
times.&amp;nbsp; Attempted to implement a Transfer Labware command.&amp;nbsp; Maybe it even worked.
Restriction digest for UAS: mKate, CMV-Notch, CMV-TO: Delta-mCherry, UAS: citrine (X2 samples each).
Updated [Computer Generated Circuits |iGEM2011:Computer Generated Circuits](fixed assumptions, etc.)

<TABLE id="minimalist"><TBODY><TR><TH>Lane</TH><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH><TH>7</TH><TH>8</TH><TH>9</TH><TH>10</TH><TH>11</TH></TR><TR><TD>Part</TD><TD>Ladder</TD><TD>UAS:Citrine 1</TD><TD>UAS:mKate 1</TD><TD>CMV Notch 1</TD><TD>CMV-TO:Delta_mCherry 1</TD><TD>Ladder</TD><TD>UAS: Citrine 2</TD><TD>UAS:mkate 2</TD><TD>CMV Notch 2</TD><TD>CMV TO dmC 2</TD><TD>Ladder</TD></TR><TR><TD>Expected</TD><TD> 1 kb,

1.7 kb

3.8 kb</TD><TD>1.9kb

5.3 kb</TD><TD>3.3 kb

8.5 kb</TD><TD>400 bp

870 bp

2.2 kb

5 kb</TD><TD>| 1 kb,

1.7 kb

3.8 kb</TD><TD>1.9kb

5.3 kb</TD><TD>3.3 kb

8.5 kb</TD><TD>400 bp

870 bp

2.2 kb

5 kb</TD></TR></TBODY></TABLE><H3>July 25</H3>
Passed cells and dealt with possible contamination. Poor transfection efficiency for tri-color experiment 
after transfection efficiency of almost 100% on previous Hef1A:rtTA3, TRE:Gal4-VP16, UAS:mKate 
experiment (see [iGEM2011:TRE.GAL4 UAS Activation]). Also it
looks like the UAS is leaky because even the control without rtTA had some background red.
Did sequencing of eighteen samples and restriction mapped AVPR2 and other constructs. Transformed
three AVPR2-TEVs-GV16 Golden Gate attempts, all of which yielded 50\+ colonies within 10 hours, and 
plated an additional LR attempt with pDisplay-Vasopressin. Expression vector propagation. Ran FACS on the tricolor experiment, Notch Delta experiment, and Gal4 experiments. Data looks somewhat consistent with expectations but nowhere near ideal. 
Imaged Mnt Parts Characterization experiment. Fought with pyevo and lost. Divya ran FACS on Tricolor, Gal4, and Notch-Delta experiments.
Nanodropped all set of 6 LR samples, restriction digested A-E with SalI.  Restriction mapping: Lane contents are as follows:
Gel1: Ladder \| A3/A4 \| A5 \| B2 \| B3 \| B5 \| C2 \| C3 \| C5 \| D1 \| D2 \| Ladder
Gel2: Ladder \| D4 \| D5 \| E3 \| E2 \| E5 \| 3 \| 4 \| 5 \| 6 \| 7 \| Ladder
See 7/20 for the key. Sent off A5, B5, C3, D1, E3, and F2 off for sequencing.

</DIV><DIV class="content" id="w9content"><H1>Summary of Week 9: August 1 - August 7</H1>
In the ninth week we did a lot of internal re-organization to increase our overall efficiency in work. This involved re-organizing an internal wiki that we use for management of our available DNA as well as a log of transfections needed to be done. Lots of samples were FACS-ed, generating lots of results that we can make graphs out of. Many of our parts were successfully characterized.

<H3>August 7</H3>
21 FACS samples for rtTA-DD experiment and TRE:mKate. Preliminary results: Hef1a:eBFP2 did not show up. Added dox to the TRE:EBFP experiment. Also &quot;procured&quot; formaldehyde and BSA for immunostaining. Fixed and blocked cells for staining tomorrow. Also passed 4 plates of cells using new technique which should fix the clumping issue that is leading to non-uniform transfection efficiency. The key is to dilute cells to 4x10^5 and then add 0.5 mL to well, as opposed to adding a small concentrated amount of cells into well, then diluting.
Inoculations of the EPHB2, EFNB1, and CXCR1-TEVs-GV16 were done a little late in the day, but will still be miniprepped after the morning meeting on Monday. Restriction mapping not started\- once SalI-HF is in the lab, will commence. Designed new primers for Notch engineering.

<H3>August 6</H3>
8 samples for FACS. Transfected TRE:EBFP2 experiment. Mariola had 40% transfection efficiency. Tyler's had around 10%. Ken's stuff had no fluorescence.
Completed gel extraction, Gibson ligation, and transformation of the EPHB2, EPNB1, and CXCR1 ENTR vectors; inoculated duplicates of the ten LR reactions and the pDisplay-CCL5-MYC Gibson redo. 
FACSed Delta characterization experiment. Induced TRE: EYFP experiment with dox ladder. FACS for some colors. DOX for some TRE:Laci-Krab.

<H3>August 5</H3>
 Transfected pDisplay-vaopressin expression vector; will do immuno-staining on Sunday. Added dox and
 shield to &amp;nbsp;cells.&amp;nbsp; all Robot LR reactions failed while Divya had a 3/8 success rate
 and Grant had a 11/13 success rate. Based on alignment of the failed sequences, possibly the minCMV
 promoters we are using may be compromised. Transformed eleven constructs (the ten LR reactions referenced yesterday and the pDisplay-CCL5-MYC Gibson redo), completed PCR reactions for EPHB2, EPNB1, and CXCR1 (twice...). BBC application video and edits with Jon and Tiffany.

<H3>August 4</H3>
Imaged all of the tri-color cells before preparing them for FACS ([Tri-Color Experiment v2|iGEM2011:Tri-Color Experiment v2]).
Note that Hef1A:mKate works. Used Matlab to generate histograms for 7.25.11 FACS data ([iGEM2011:TRE.GAL4 UAS Activation]). These experiments all failed because of bad TRE:GV16, but note a couple things: 1.) UAS:mKate is relatively leaky (about 10% of cells). Edited wiki&amp;nbsp;
Redid the pDisplay-CCL5-MYC Gibson reaction at 55C based on previous success at this temperature. Started new LRs of multiple Tango and characterization parts, referenced here. Did significant in-silico cloning and database updating.

<H3>August 3</H3>
Did Tri-color transfections as well as TRE:LacI-Krab Repression of LacO experiments. Will need to FACS colors tomorrow with Charles and add Dox to LacI-Krab wells and FACS those on Friday. The TRE:GV16 was resequenced by Grant and failed, so all of my experiments from 8.2.11 found here ([iGEM2011:TRE.GAL4 UAS Activation]) failed as well, although we did prove that TRE:mKate works. 
Restriction digest &amp; gel by robot attempt #1 with Divya. 11 samples included variants of 3 DNAs--TRE:LacI, minCMV4xLexA:eBFP2, Hef1a:Notch. TRE and Hef1a looked good, but all 6 of minCMVs had consistent but different from expected band length.
Transfected Hef1a-lacO:Delta-mCherry verification experiments. Fixed auto_facs.m code. Multiple LRs, including Tango system LRs, will commence today, along with restriction mappings of the 50 plasmids.

<H3>August 2</H3>
Planned out future experiments, did research on protocols. Went down to image cells and induce with delta and dox. It does seem that &quot;our&quot; UAS is not very leaky at all as seen with UAS:mKate. Of course, could be effect of citrine having high stability compare to mKate. Also, we have H2B citrine, thus explaining the nuclear signal we keep observing. Also we can test for pDisplay-vasopressin-myc on cell surface with my proposed Immunofluorescence protocol. 
Added Dox to cells tranfected yesterday ([iGEM2011:TRE.GAL4 UAS Activation]&amp;nbsp;\- see bottom). Did not see any UAS leakage with UAS:mKate. Ken suggested that our UAS is different for that Elowitz used with UAS:citrine. Also imaged monoculture cis-inhibition. Ali did a repeat. Transformed 4 LRs (by robot yesterday) and 8 of Divya's LRs. Plates are in 37 C from 3pm. 
BP Reaction using 1.0 ul and 2.0 ul of BP clonase instead of 0.5 ul. Discovered that sequences sent off on Monday failed because the wrong primers were used, thereby saving Grant five to ten minutes of time aligning the sequences. Transformed thirteen LR reactions and the redo of HRH4-TEVs-GV16. 

<H3>August 1</H3>
Rest Dig of Hef1a:eYFP-4xFF4. Also transfected set 3 of free delta experiment and co-culture, and TRE modulation of Delta. 
Miniprepped and restriction mapped quintuples of DRD2-TEVs-GV16 and pDisplay-CCL5-MYC. Inoculated quintuples of CCR5-TEVs-GV16 and B-Arrestin-2-TEV-Protease Gibsons; HRH4-TEVs-GV16 did not form colonies and a Gibson was redone at a different temperature. Completed thirteen of twenty-one planned LRs before running out of LR Clonase.
@ Woods Hole Oceanographic Institute with EBICS REU touring microtubule and cytoskeletal filament research laboratories. 
Transfected cells for Gal4 experiments, also repeated cis-inhibition experiment with just Delta and Notch. Added Dox to monoculture cis-inhibition experiment. Prepared cells for transfection tomorrow. Hopefully we have Hef1a:GV16.
Ran 4 LR reactions by robot with Louis at the Weiss lab. LR reactions are at room temperature since 6:30 pm and will be ready for transformation after 8 - 16 hours since then.
Ran gels of two sets of rest digs. Sent 1 set of good plasmids off for sequencing. Experiment table organization/planning. And other stuff all around the wiki. Updates everywhere! Online interface for robot service almost up.


</DIV><DIV class="content" id="w10content"><H1>Summary of Week 10: August 8 - August 14</H1>
Computer simulations of the Notch-Delta interactions were presented in our group this week, and we became convinced of the possibility of creating self-patterning mammalian cells. On the DNA side of things, we are trying to create more and more DNA using miniprep, because midipreps have for some reason not been successful for us or the 2010 iGEM team. On the transfection side, a lot of new DNA parts were transfected, observed under the microscope, and FACS-ed to quantify their effect/behavior.

<H3>August 14</H3>
Good CP LRs, as well as re-inoculations of poor CP LRs. Ran FACS on 50 samples for Tyler and Ken. 
Sequencing of Tango and other plasmids as well as propagations of a handful of failed plasmids / spinning down of cells for Charles' miniprep.

<H3>August 13</H3>
Added delta to cells. Co-cultured AVPR and pDisplay Vasopressin cells. Froze down and passed the Elowitz CHO cell lines and put in \-80. Also passed wells and flasks.

<H3>August 12</H3>
Transfected cells for AVPR2 experiment as well as another free delta induction experiment, this time with const. color gating.
Attended Ken's orientation session and did 1 transfection just to try things out. Also obtained 
my new culture of cells. Cell stocked CP LR 4,5,6,7,8,11,12, of which 4,5, and 7 are tentative (repeat submitted).
Also propagated a whole bunch of plasmids, including the successful and tentatively successful CP LR's.
Hopefully the color palette will be complete by Tuesday. Tissue culture orientation &amp; inoculated SDM transformations from yesterday. Colony counts around ﻿20 for TEV and 3 for GV16.

<H3>August 11</H3>
Miniprepped and nanodropped 12 LRs from Aug 9. Hef1a: Delta-mCherry has esp. low concentration. 
Did FACS on&amp;nbsp;[iGEM2011:Confluency Experiment]&amp;nbsp;which shows best starting value around 1.5*10^5 cells/mL starting concentration. FACS machine showed around 40% efficiency. Matlab shows around 30%. In either case, we should start transfections with cells less confluent than we have been. TRE:LacI-Krab experiment seems to have worked, but there is only about 2x repression. Also put up FACS from AmCyan which is more green than blue.
Worked on Circuit Diagrams and write ups for the public wiki. 42 FACS in less than 1 hour. Very fast because Ken's samples were concentrated.

<H3>August 10</H3>
I updated the wiki (first (per usual(winning))). Prepared for FACS. Transfected Confluency Experiment ([iGEM2011:Confluency Experiment]). Today Jenny FACSed some DRD2 (apparently a failure from pictures) and some Delta-Notch stuff (also a failure). 
Completed Tyler's Hef1a_GV16 page with pretty FACS images. Some problems with either no cells in sample or visibly clumped cells. Started LR reactions for a few more Tango parts and the Ephrin parts. Shockingly did nothing else.

<H3>August 9</H3>
Worked on MATLAB stuff more. Treated TRE:mKate and TRE:eBFP2 experiments with dox. Transfected for pDisplay immunofluoresence experiment again. Passed cells into 60 mm plates for N-cad knockdown experiment. 
Removed propagations of various gene/promoter entry vectors and inoculated a single colony from each. Did alignments of sequencing data for samples sent out yesterday. One LR reaction failed unexpectedly, but the other failures involving minCMV promoters appear to have occurred due to contamination from a Hef1a variant promoter based on a BLAST of the sequencing results. (READ: The &quot;minCMV&quot; promoter DNA is not actually minCMV promoter DNA\- at least this is the case for the tubes without stickers.) All parts currently being used for experiments are verified. On track for \~15-20 new LR reactions for Wednesday afternoon.
12 LR reactions, left at room temperature @ 12:30pm~. Transformed Clara's LR reactions. Need to redo all of the FACS data tomorrow that Jenny did today because the red and blue colors are switched. Mariola's program works so I will use that. Tomorrow I will do transfections of a consistuitive color into wells with different confluencies that I set up today. I added Dox and Dopamine to wells today that we will FACS tomorrow.&amp;nbsp;

<H3>August 8</H3>
Planned out and did transfections for TRE eBFP2, TRE:mKate and AVPR2 validation experiments. Also threw in a co-culture of Notch/Delta cells, only with UAS:eYFP instead of UAS:citrine from Elowitz. Not expecting anything from that, but just covering all bases. Treated pDisplay-vasopressin-myc cells with anti-myc FITC Ab and prepared slide. Results suggestive. Cells were too confluent at time of fixation, many were washed off and the remaining look bad. Probably from my bootleg formaldehyde solution as well. However, remaining cells were green on the surface as well as red from delta-mcherry, while controls are not green. Seems to suggest it is being displayed...
 Because we need a number of promoters propagated and it would be nice to do LRs with the Rheoswitch system (and possibly the new Notch constructs) concurrently, I will probably wait to begin these. Cleaned the lab before the start of the work day, like a boss, or perhaps a janitor. Started propagations of a few promoters and genes that we had &lt; 1 uL of DNA left of (and no cell stock), because if you want something done...
Aliquoted and Transfected UAS:EGFP and TRE:eyfp-4xFF4 experiments. Dox ladder of each. Writing 8pg REU paper and presentation.

</DIV><DIV class="content" id="w11content"><H1>Summary of Week 11: August 15 - August 21</H1>
In our experimental attempt to characterize the Notch-Delta interaction, we used co-culture of CHO and Hek293 cells as well as stable cell lines of Notch-containing and Delta-containing cells from Elowitz to observe the trans-activation of Notch by Delta. Other experiments using CHO cells, which are more difficult to transfect than Hek293 using Lipofectamine, were carried out to observe the behavior of CHO transfected cells.

<H3>August 21</H3><B> Kenneth </B>- <I>Updates on Experiments:</I>

Latest iteration of base FACS code: removes the mean line on the histograms. Moves the quadrant lines to 200 for everything, since I've been seeing 200 as pretty much the max for the Blank samples. Can be easily changed.
Results are in for dox ladder of Elowitz CMV-TO CHO cells [iGEM2011:Elowitz CHO Dox ladder], free delta induction of Elowitz receiver CHO cells.&amp;nbsp;[iGEM2011:Free Delta Induction of Elowitz Receiver CHO]

FACS data is in for TRE:delta mcherry and TRE:mKate experiments TRE_mKate dox ladder part 2|iGEM2011:TRE_mKate dox ladder part 2], however, due to loss of blue channel, cannot gate. Instead, using the top X% percent method to process data. The MATLAB code takes in the data and only keeps the top X% of data. In this case I used 50%.  

Co-Culture of Elowitz CHO cells: taken pictures also at 48 hours. Will bring to Weiss lab to process images. Redoing the experiment again, this time with 80:20 ratio as well. Will take pictures and FACS on Tuesday

CO-Culture of HEK and CHO cells:

-Delta Heks and receiver CHO's: co-cultured Friday. added dox saturday. Will FACS Monday

-Delta CHO's and receiver Hek's: same as above

-TRE:delta Hek senders and CHO receivers: didn't have the resources to do just yet. Set up for experiment on Tuesday.<B> Tyler: </B> I just put up&amp;nbsp;[iGEM2011:CHO Experiments]. I will try to do some analysis tomorrow, but some outside analysis would also be helpful.<B>Charles: </B>FACS of various things. CHO Experiment from Tyler had very unhealthy cells. Kens cocultures have yet to be analyzed.<H3> August 20</H3><B>Divya </B>- Miniprepped, Nanodropped, Inoculated. See Personal Notebook for details.

<H3> August 19</H3><B>Divya </B>- Transformed, Inoculated. See Personal Notebook for details.

<H3> August 18</H3><B>Charles </B>- FACS 100 samples in just under 90 minutes. Tyler's stuff had low concentrations so it ran slow, but data looks promising. Ken did Heks and Chos and both look promising. My stuff failed due to higher than intended initial cell concentration. Basically cells overgrew ran out of nutrients got sick and failed. Also made 20,000 ng of Dest 4-5. 

Other things I did today:

-FACS for TRE:LacI and TRE:LacI-Krab (I'm generating graphs now)

-Transfected CHO cells for Notch-Delta Co-culture

-I was going to repeat Mariola's Mnt and CI434 activation experiments but I couldn't find the DNA, so I asked Divya to do some LR reactions <H3> August 17</H3><B>Jon </B>- Prepped Notch for full sequencing, minipreps, cell stocking, nanodropping, prepping trash for pickup.

<B>Divya </B>- Inoculated and LR'd NCADs, Nanodropped CPLRs. Will transform tomorrow. See personal notebook for details.

<H3>August 16</H3><B>Tiffany: </B>Public wiki work forever. Don't use IE please.<B>Divya/Jon </B>- In lab until 2am doing 26 minipreps, nanodrops, cell stocks, cleaning. <B>Charles: </B>Transfected cells with&amp;nbsp;[EXP1-CH|iGEM2011:EXP1-CH]. Made graphs. <H3> August 15 </H3><B>Tyler: </B>Prepared for transfections and updated the wiki. Will prepare CHO cells for NCAD phenotype experiment tomorrow. NCAD with HEK-293 cells failed. Hef1A:GV16 experiments looked very good. TRE:LacI-EYFP-FF4 shows a lot of EYFP, about 25% increase if I remember correctly, but the LacI repression looks very weak. We have a Hef1A:mKate that is bad, showing about 0-2.5% fluorescence. I will repeat these experiments tomorrow. The DRD2 shows very small levels of activation, but I think it is actually there (see&amp;nbsp;[iGEM2011:GPCR Experiments]). Plan for tomorrow's transfections: 1.)TRE:LacI-Krab 2.)TRE:LacI 3.)miRFF4 4.)Lipo Ladder. Wednesday: CHO transfections.

Kenneth: Made the line chart for TRE:eBFP experiment, looks good. Processed data from TRE:mKate experiment. No increase in red, but increase in blue, almost like TRE:eBFP2. Not quite sure what went wrong here. Maybe I aliquoted the wrong DNA, maybe there's some mixup? Either way, has to be redone. Future directions: continue working on the matlab code, process FACS data for co-cultures and AVPR2 testing as well as free Delta (all done Sunday). Experiments planned:

1) Elowitz CHO cells [Initial CHO images - No Fluorescence] (so the cells don't express Delta-mCHerry without dox.) will FACS all of this Wednesday with proper scatter gating for CHO's
a. Dox induction of Delta (get a good curve showing conc of dox to amt of delta on surface
b. Addition of Free Delta to Notch-Gal4 cells and Notch+Delta cells
c. Co-culture with varying dox levels
d. If we see successful dox induction of Delta-mCherry and delta induction of Notch-Gal4/UAS, we can do the 2D ladder experiment where we vary both dox and delta levels for the Notch+Delta CHOs and see how much citrine, thus giving us a &quot;surface&quot;

2)redo TRE:Delta-mCherry dox ladder
3)TRE:eYFP and TRE:mKate need to be redone, I don't know if Charles wants to do this or not...
4)Immunostaining using no-coverslip technique for pdisplay-vasopressin as well as AVPR2 for troubleshooting
5)still waiting on free vasopressin for the AVPR2 tests.
6)test HRH4 with histamine
7)siRNA knockdown of N-cad.

Divya: Restriction Digested Clara's LRs. Send samples for sequencing.  Labeled pRESC tubes. Got items from Weiss Lab. 



Grant: Designed quite a few  primers\- this time around, for a Tango redesign and colored  transactivators. Digested and gel extracted Sam's 4xFF4 gene entry  backbone, pushing the Mnt-VP16-4xFF4 and Delta-mCherry-4xFF4 assemblies  forward a bit. DNA work seems to have slowed considerably over the past  few days\- we should make more (non-Tango) LRs. Ephrin stuff will  hopefully be verified tomorrow. Takers?
Charles: Passed cells. Doing bunch of wiki work now.
Jon - Miniprepped and nanodropped 19 samples. Prepped SDMs for sequencing. Analyzed odd sequence results. Reinoculated Rescue 28 plasmids; put into incubator at 5:40pm, although many are already at high OD (wildly inconsistent though) so I'd check after a couple hours to make sure the E. coli don't reach stationary phase. </DIV><DIV class="content" id="w12content"><H1>Summary of Week 12: August 22 - Later</H1><H3> September 1</H3><B>Grant:</B> Inoculations of the aforementioned LR constructs.

<H3> August 29 </H3><B>Charles</B> - CHLRs see CH notebook. H, HL, T, U and delta-ff4, H and T for ncad-2a-eyfp. inoculated propagations for HLNcad2Aeyfp, UASNcad2aeyfp, and Tre:delta.

<H3> August 28 </H3><B>Grant:</B> Restarted LR reactions of 9 of the constructs listed on the Workflow (only the pEXPR_12_Hef1a-LacO_Notch-CI434-VP16 construct succeeded, possibly due to incubation time for LR reactions). Started PCRs for Tango constructs using new padding for TEV cut site.

<H3> August 25 </H3><B>Divya:</B> Won.
(See Personal Notebook for details).

<H3> August 23 </H3><B>Charles</B> - FACS Kens 50 50 and 80 20 Hek TRE-delta senders and CHO receivers. FACS was clean. Cells were all healthy. Lowered voltage of DAPI channel from 300 to 240 to move blank cells basal fluorescence to the bottom left corner. Laser was on, everything went according to plan. Preliminary results look promising, with increasing dox for each coculture, there was increasing FITC channel signal. Preliminary results suggest that our delta works.

<H3> August 22 </H3><B>Charles</B> - FACS of Kens coculture stuff. Ambiguous results.

</DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>