<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_MIT_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:MIT/Project/">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:MIT/Project/&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:MIT/Project/&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:MIT/Project/&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:MIT/Project/" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:MIT/Project/</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><UL class="navbar"><LI><A href="https://2011.igem.org/Team:MIT">HOME</A></LI><LI><A href="https://2011.igem.org/Team:MIT/Project/">PROJECT</A></LI><LI><A href="https://2011.igem.org/Team:MIT/Results/">DATA</A></LI><LI><A href="https://2011.igem.org/Team:MIT/Notebook/">NOTEBOOK</A></LI><LI><A href="https://2011.igem.org/Team:MIT/Tools/">MODEL</A></LI><LI><A href="https://2011.igem.org/Team:MIT/Partners/">PARTNERS</A></LI><LI><A href="https://2011.igem.org/Team:MIT/Team/Undergraduates">TEAM</A></LI><LI><A id="long" href="https://2011.igem.org/Team:MIT/HumanPractices/">HUMAN PRACTICES</A></LI></UL></P><DIV id="col_nav"><DIV class="col_list"><H2>Navigation</H2><UL><LI id="ov">Overview</LI><LI id="de">Device Designs</LI><LI id="no">Delta/Notch Interactions</LI><LI id="tr">Transcription Factors</LI><LI id="ca">N-Cadherin Adhesion</LI><LI id="sa">Safety</LI></UL></DIV></DIV><DIV id="col_left"><DIV class="content" id="ovcontent"><H1><SPAN class="mw-headline" id="Project_Description"> Project Description </SPAN></H1><P>Current medical practices are only able to scratch the surface of tissue engineering. Fortunately, nature has provided us with robust, cellular systems capable of governing the autonomous formation of complex structures and patterns. To gain control over emergent behaviors in multicellular systems, we made use of nature's genetic resources and engineered three modules that could give us control over autonomous pattern formation when integrated. These modules include are: <B> cell-cell signaling</B>, <B>internal logic processing</B>, and <B>cadherin output</B>.
</P><P>A. <B>Cell-Cell Interaction.</B> For cell-cell interaction, we decided were attracted to the Notch-Delta juxtacrine signaling pathway first discovered in Drosophila due to its ubiquity and critical role for the breaking of symmetry in cell populations in the development of many organisms. We engineered the Notch-Delta signaling pathway such that when a Notch protein is trans-activated by a Delta protein on a neighboring cell, an engineered Gal4-VP16 intracellular domain would be cleaved and enter the nucleus to activate our designed internal processing module.</P><P>B. <B>Internal Processing Module.</B> Upon receiving the Gal4-VP16 signal from cell-cell interactions, an artificial circuit of design is activated, leading to a variety of results, such as perhaps the up- or down-regulation of either Notch or Delta proteins, or the expression of other proteins such as cadherin. In our internal processing module we use primarily an engineered Hef1a-LacO promoter, which is strong and constitutive but negatively regulated by LacI, a UAS promoter activated by Gal4-VP16 signal, and a TRE promoter which is activated by rtTA3 in the presence of doxycycline.</P><P>C. <B>N-Cadherin.</B> We decided to use neuronal calcium-dependent adhesion proteins &quot;N-Cadherin&quot; as our output to see how tying in cell adhesion would affect cell-cell interaction and investigate the possibility of forming more stable patterns. Using a N-Cadherin-GFP fusion protein, we were able to visualize the functionality of N-Cadherin.</P><P>To understand and predict multicellular behavior, we developed a simulation framework based on the Synthetic Biology Open Language (SMOL) and CompuCell 3D modelling environment. Our models motivated several circuit designs we subsequently tested in the laboratory. Altogether, our developments establish a paradigm for manipulation of intercellular communication systems to drive self-assembly of ever more complex patterns and tissues.</P></DIV><DIV class="content" id="nocontent"><H1><SPAN class="mw-headline" id="Delta.2FNotch_Interactions"> Delta/Notch Interactions </SPAN></H1><P>The Delta/Notch ligand-receptor system is a signaling pathway that is conserved in the overwhelming majority of eukaryotes and in all metazoans, or animals. Both proteins are single-pass transmembrane proteins that can interact with one another to form a heteromeric complex capable of activating transcriptional cascades, specifically in the cell bearing the Notch receptor.
</P><P>In the mammalian cells we engineered, there are typically between one and four Delta ligands known as the Delta-Like (DLL) family in addition to several Serrate ligands that are homologous to the Jagged family. While both the Delta-Like and Serrate families are relevant in endogenous systems, the mechanics of Serrate/Notch signaling are typically more variable and dependent on interacting proteins than is the Delta-Like family in mammalian cells. We specifically selected only the Notch-1 receptor and Delta-Like-3 ligand for our project in order to optimize our resources for multiple Delta/Notch experiments. In the future, we intend to probe the usefulness of Serrate engineering and explore how modifier proteins like the Fringe fucosyltransferase enzymes or the Mind Bomb ubiquitin ligase, all of which are capable of modifying multiple aspects of Delta/Notch signaling.
</P><P>The Delta/Notch signaling system has been implicated to have fundamental roles in many cell fate and developmental decisions, including neuronal differentiation, immune cell differentiation, and binary fate decisions between different germ cell layers. Because of its wide application to tissue differentiation of all varieties and our ability to engineer the intracellular domain of Notch to activate gene transcription from promoters of our choosing, we ultimately selected the Delta/Notch system as our cell-cell signaling mechanism.
</P></DIV><DIV class="content" id="trcontent"><H1><SPAN class="mw-headline" id="Transcription_Factors"> Transcription Factors </SPAN></H1><P>In order to actuate expression from any signaling pathway, logical elements are critical for the manipulation and modification of transcriptional activity. The most fundamental of these are transcription factors, proteins that serve to upregulate or downregulate expression from promoters. In order to ultimately perform complex behavior in response to multiple signaling inputs, we characterized and MammoBlocked several different transcription factors. 
</P><P>Factors that upregulate transcription all consist of a DNA-binding domain and the VP16 transactivation domain isolated from the herpes simplex virus. This domain recruits the Mediator transcriptional complex and other proteins for high level expression in mammalian cells. 
</P><P>All factors that downregulate transcription include the LacI repressor isolated from Escherichia coli; the differences include concatenation of the miRFF4 micro RNA capable of downregulating translational activity and the Kruppel-associated box, or KRAB domain, which recruits histone deacetylase complexes and causes chromatin silencing, resulting in significant repression of transcriptional activity.
</P></DIV><DIV class="content" id="cacontent"><H1><SPAN class="mw-headline" id="N-Cadherin_Adhesion"> N-Cadherin Adhesion </SPAN></H1><P>N-Cadherin is a neurally-derived, calcium-dependent adhesion protein of significant importance to tissue construction in endogenous eukaryotic systems. In cell culture, differential cadherin expression has been shown to result in specific aggregation such that cells expressing one cadherin variant are significantly more likely to cluster with cells expressing identical cadherins. 
</P><P>Cadherins are known to contain five extracellular repeats eponymously called &quot;cadherin repeats&quot; and an intracellular domain with several scaffolding motifs. When two N-Cadherin monomers on one cell are exposed to calcium, the extracellular repeats can dimerize, and these activated cadherin dimers are then able to interact with dimers on adjacent cells, functioning as a sort of extracellular glue. 
</P><P>At the same time, the intracellular domain links with a large complement of other proteins via the scaffolding motifs presented on the cytoplasmic face, interacting with the actin cytoskeleton and possibly a variety of signaling proteins. 
</P><P>When combined, the extracellular dimerization and intercellular recruitment serve to form stable yet dynamic links between adjacent cells. Because of this integrative capacity,  we selected N-Cadherin to be our cell-cell adhesion actuator.
</P></DIV><DIV class="content" id="decontent"><H1><SPAN class="mw-headline" id="Device_Designs"> Device Designs </SPAN></H1><H3><SPAN class="mw-headline" id="Hef1a:Gal4-VP16.2C_UAS:Reporter">Hef1a:Gal4-VP16, UAS:Reporter</SPAN></H3><P>This circuit represents the behavior of the Gal4 activator. Hef1a is a constitutive promoter that will express Gal4-VP16 that can later bind to a UAS promoter, leading to the expression of downstream reporter genes.
</P><H3><SPAN class="mw-headline" id="Hef1a:LacI.2C_Hef1a-LacOid:eYFP">Hef1a:LacI, Hef1a-LacOid:eYFP</SPAN></H3><P>Here we represent the activity of the LacI inhibitor. LacI binds to the upstream LacO site and can inhibit expression of downstream reporter genes.
</P><H3><SPAN class="mw-headline" id="Hef1a:Gal4-VP16.2C_UAS:LacI.2C_Hef1a-LacOid:eYFP">Hef1a:Gal4-VP16, UAS:LacI, Hef1a-LacOid:eYFP</SPAN></H3><P>Here we see a circuit that integrates the two systems described above. By tying LacI expression to the Gal4/UAS system, we can activate expression of lacI by GV16 and thus inhibit expression of a reporter gene. This results in overall repression of reporter expression.
</P><H3><SPAN class="mw-headline" id="Gal4-VP16.2C_UAS_NCad-2A-eYFP">Gal4-VP16, UAS_NCad-2A-eYFP</SPAN></H3><P>Here we created a part that is activated by the Gal4-VP16 transactivator to produce N-Cadherin joined to eYFP by a 2A tag, which is cleaved to result in the expression of both proteins.
</P><H3><SPAN class="mw-headline" id="Hef1a:rtTA3.2C_TRE:Mnt-VP16.2C_minCMV-4xMnt:mKate">Hef1a:rtTA3, TRE:Mnt-VP16, minCMV-4xMnt:mKate</SPAN></H3><P>Here we represent a circuit that has an upstream rtTA3 system feeding into the Mnt activator system. When bound by doxycycline, rtTA3 is capable of inducing expression at the TRE promoter. This in turn leads to expression of the activator Mnt-VP16 which can bind to Mnt sites and activate expression of the reporter gene (mKate, a red fluorescent protein, in this case).
</P><H3><SPAN class="mw-headline" id="Hef1a:rtTA3.2C_TRE:CI434-VP16.2C_minCMV-1xCI434:mKate">Hef1a:rtTA3, TRE:CI434-VP16, minCMV-1xCI434:mKate</SPAN></H3><P>This circuit is very similar in design to the above, instead using a different orthogonal activator-promoter pair: CI434. In this schematic, doxycycline induced expression of CI434 leads to activation of reporter expression downstream of the CI434 binding site (minCMV-1xCI434).

</P></DIV><DIV class="content" id="sacontent"><H1><SPAN class="mw-headline" id="Safety"> Safety </SPAN></H1><P><B>Would any of your project ideas raise safety issues in terms of:</B></P><UL><LI> researcher safety,
</LI><LI> public safety, or
</LI><LI> environmental safety?
</LI></UL><P>This summer, our team worked on cell patterning in mammalian cells. Part of our team
worked with E. Coli in a BL1 lab, and a smaller group worked with mammalian cells in
a BL2 lab. Both groups within the team adhered to national and local safety protocols.
Extra care was taken not to cross-contaminate lab spaces. Cross contamination from these
settings was minimized by designating specific equipment for mammalian cells and for
bacteria, as well as immediately changing personal protective equipment when moving between
the different lab spaces.
</P><P><B>Do any of the new BioBrick parts (or devices) that you made this year raise any safety issues? If yes,</B></P><UL><LI> did you document these issues in the Registry?
</LI><LI> how did you manage to handle the safety issue?
</LI><LI> how could other teams learn from your experience?
</LI></UL><P>Our bacteria and our mammalian cells do not contain any BioBrick parts that code for
hazardous proteins or molecules. We also determined that none of our circuits would
survive if released outside the lab and thus, the circuits pose no safety concerns to researchers, the public, or
the environment.
</P><P><B>Is there a local biosafety group, committee, or review board at your institution?</B></P><UL><LI> If yes, what does your local biosafety group think about your project?
</LI></UL><P>The EHS (Environment, Health, and Safety) Office is MIT's biosafety group that enforces
lab safety in all labs on campus. They provide safety training, waste management services,
and resources for safe lab practices. The safety training included General Biosafety for Researchers, Managing Hazardous Waste, General Chemical Hygiene, lab-specific training,
and Bloodborne Pathogen Training. All undergraduates were trained by the EHS to work
safely in BL1 labs, and students working with mammalian cells received BL2 lab safety
training. EHS is an integral part of our biosafety system and continues to be
involved by conducting daily lab safety inspections.
</P><P><B>Do you have any other ideas how to deal with safety issues that could be useful for future iGEM competitions? How could parts, devices and systems be made even safer through biosafety engineering?</B></P><P>Our MammoBlock construction standard not only aids in the construction of mammalian
parts but also facilitates safer and easier storage of these parts. The MammoBlock standard
uses bacterial entry vectors which allows mammalian parts to be stored in BL1 conditions.
</P><P>Our team is greatly concerned with the safety issues regarding future iGEM research, especially relating to mammalian cells. This standard has allowed us to enter mammalian parts into the registry, and will facilitate the safe shipping and submission of mammalian parts constructed by future iGEM teams. 
and competitions. 

</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:MIT/Project/">http://2011.igem.org/Team:MIT/Project/</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:MIT/Project/" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:MIT/Project/" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:MIT/Project/&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:MIT/Project/&amp;oldid=261941" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>