<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Utah_State_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Utah State/Experiments</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>No Sidebar - Arcana by HTML5 UP</TITLE></P><DIV id="page-wrapper"><DIV id="header"><H2>Utah State Universtiy <EM>iGEM 2015</EM></H2><NAV id="nav"><UL><LI><A href="https://2015.igem.org/Team:Utah_State">Home</A></LI><LI><A href="https://2015.igem.org/Team:Utah_State/Team">Team</A></LI><LI><A href="#">Project</A></LI><LI><A href="https://2015.igem.org/Team:Utah_State/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Utah_State/Experiments">Experiments &amp; Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Utah_State/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Utah_State/Design">Design</A></LI></UL><LI><A href="https://2015.igem.org/Team:Utah_State/Interlab">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:Utah_State/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Utah_State/Parts">All Parts</A></LI><LI><A href="https://2015.igem.org/Team:Utah_State/Basic_Part">Favorite Parts</A></LI><LI><A href="https://2015.igem.org/Team:Utah_State/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Utah_State/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Utah_State/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Utah_State/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Utah_State/Standards">Medal Criteria</A></LI></NAV></DIV><SECTION class="wrapper style1"><DIV class="container"><DIV id="content"><ARTICLE><HEADER><H2>Experiments and Protocols</H2><P>The nuts and bolts</P></HEADER><H2>Experiments</H2><H2><EM>Fluorescent Protein Protocol</EM></H2><OL><LI>Plate out 1 plate of bacteria for each of the fluorescent protein constructs as well as one plate with a standard fluorescent protein construct as a positive control and one plate with non-transformed competent cells as a negative control</LI><UL style="list-style-type:disc"><LI>Ideally, there should be one positive control for each fluorescent protein used in the constructs being examined</LI></UL><LI>16-18 hours after plating the bacteria, pick 3 colonies for each construct and place each picked colony in 5 mL of LB media in a 15 mL culture tube with the appropriate antibiotics</LI><LI>Allow these cultures to grow for 14-16 hours in an incubator at 220 rpm and 37 °C</LI><LI>Determine the OD600 for each culture, using LB media as a blank</LI><LI>Inoculate one 25 mL culture of LB media at 0.1 OD600 from each of the 5 mL cultures</LI><LI>Allow the 25 mL cultures to grow to an OD600 of 0.5 in an incubator at 220 rpm and 37 °C, checking the OD approximately every 0.5 hours</LI><LI>When each culture reaches an OD600 of 0.5, place it inside the 4°C refrigerator to slow cell growth so that the cultures will be processed at similar phases of growth</LI><LI>Place 3 mL of each culture in 15 mL centrifuge tubes and centrifuge for 15 min at 3500 rpm</LI><LI>Discard supernatant and wash each cell pellet with 3 mL of 0.15 M NaCl solution</LI><LI>Centrifuge for 15 min at 3500 rpm</LI><LI>Discard supernatant and resuspend each cell pellet in 3 mL of 0.15 M NaCl solution</LI><LI>Place 200 µL of each culture in three individual wells of a 96-well plate (triplicates of 200 µL samples). Also place 200 µL of 0.15 M NaCl solution in three individual wells to serve as a blank</LI><LI>Open a GFP fluorescence reading protocol in the BioTek Gen5 Data Analysis Software that is connected to the microplate reading instrument</LI><LI>Adjust the excitation and emission wavelengths to the appropriate wavelength peaks for each fluorescent protein. In our project with the instrument we had available, these were as follows:</LI><OL><LI>GFP: 360/40, 460/40</LI><LI>sfGFP(Bs): 485/20, 528/20</LI><LI>mCherry: 530/25, 590/35</LI></OL><LI>Read the microplate using the microplate reading instrument and the Gen5 Data Analysis Software</LI><LI>Export the data to Excel</LI><LI>Average the 0.15 M NaCl well values together and create a separate table with the resulting average value subtracted of from each of the fluorescent protein culture measurement values</LI><LI>Average the resulting values for each construct and compare them to the positive and negative controls as well as the similar fluorescent protein constructs (i.e. those with the same fluorescent protein but with different promoters)</LI></OL><H2>Protocols</H2><H2><EM>E. coli</EM></H2><H4>Bacterial Transformations</H4><P> Transformations are any procedure used to insert DNA into a bacteria (if you use a virus, the term becomes transfection). Electroporation uses a pulse of electricity to disrupt the cell membrane and create holes that would allow the DNA to enter the cell. Cells need to be made competent before doing this procedure, in order for them to efficiently take up the DNA. Transformations generally utilize millions to billions of cells and DNA molecules and, for a transformation to be successful, only one molecule of DNA needs to enter into one cell, which then grows into a colony. One issue with transformations is selecting and verifying which colonies have the desired DNA. This is usually done using a marker, a characteristic possessed by cells that have the DNA (or lost by those cells) that distinguishes it from the rest of the colonies that grow up. Commonly, this is the expression of an antibiotic resistance gene included on the transformed DNA, which allows only the cells that have taken up the DNA to survive on a plate in the presence of that antibiotic. Sometimes pigment producing or fluorescent/luminescent proteins can also be used in place of antibiotic resistance to allow visual determination of transformed colonies. Other ways of selection exist, but will not be discussed here. </P><OL><LI>Turn on ice machine</LI><LI>Thaw DNA solutions</LI><LI>Clean and sterilize the electroporation cuvettes by washing with double distilled water (ddH2O) twice and then fill the cuvettes with ethanol.</LI><LI>Let cuvettes sit with ethanol for 5-10 minutes, then wash 4-8 times with ddH2O</LI><LI>Place cuvettes on ice</LI><LI>Take competent cells out of the -80 °C freezer, and thaw them on ice</LI><LI>Add 3 µL of DNA to the cell solution. (This should be around 100-250 ng of total DNA, too much DNA causes arcing, too little gives few transformed colonies).</LI><LI>Incubate on ice for 5 minutes.</LI><LI>Add 60 µL WB buffer (10% glycerol). This helps reduce arcing, although too much can lower numbers of transformed colonies.</LI><LI>Set the electroporation machine to 2500 V, 200 O, and 25 µF for E. coli.</LI><LI>Transfer the cell/DNA/WB solution into the cuvettes by pipetting up and down in the 1.5 mL tube first to mix. Make sure the pipette tip is between the metal plates on the cuvette before ejecting the solution. Keep the cuvettes on ice.</LI><LI>Before electroporating, dry the cuvettes of with a KimWipe, to ensure no liquid on the surface that could create other paths for the electric pulse (and could cause arcing).</LI><LI>Pulse the cells and return cuvette to the ice. Check the time constant on the machine, a constant of 4.5+ is a very good transformation, and will yield many colonies. A constant of 2.5-4.5 is okay, and will still work. Constants below 2.5 will yield very low colony numbers, and may need to be redone. NOTE: addition of extra WB or lower amounts of DNA will reduce the time constant as well, so it is only a rough measure.</LI><LI>To remove the cells from the cuvette add 1 mL LB media or SOC media (no antibiotic in this media) to the cuvette. Pipette up and down a few times to mix.</LI><LI>Remove the solution to just above the two plates in the first removal pipetting (~1/2 of the volume) and transfer it to the original cell tube (NOT THE DNA TUBE). Then, tip the cuvette on its side so that the space between the plates is vertical, place the 1000 µL pipette tip between the plates, and slowly draw up the solution, while tipping the cuvette further. This should draw up the rest of the liquid in the cuvette.</LI><LI>Incubate the cell solutions at 37 °C for 1-2 hours (can go up to three, but try to avoid doing it for that long).</LI><LI>Plate the cells on plates containing the correct antibiotic. Each transformation requires two plates. Add 500 µL of solution to one plate, spread with the spreading stick, and then spread the spreading stick on the second plate without adding any solution to it. This creates a dilution plate in case you have thousands of colonies on the first plate. It is roughly a 1:100 to 1:200 dilution.</LI><LI>Grow the plates up overnight at 37 °C. Do not leave for longer than 24 hours, as contaminants might have a chance to grow and the plates could dry out.</LI></OL><P>If your cuvette arcs (bright flash and loud popping noise during electroporation):</P><OL><LI>Clean the electroporator lid.</LI><LI>Wash out and sterilize the cuvette with ethanol – the cells have been pretty much killed and will not be usable in plating, so you need to restart.</LI><LI>Add less DNA to the cells (reduce by 25%-33%).</LI><LI>Add an additional 15 µL of WB buffer to the solution.</LI></OL><H4>Glycerol Stock Preparation</H4><P>Glycerol Stocks are one of the best ways to store cells and DNA for long periods of time. These stocks are prepared so that the cells in the tubes are still alive, and capable of creating new liquid cultures from small amounts of frozen stock. This allows for quick growth and extraction of important DNA without spending an extra day and extra supplies re-transforming a construct every time it is needed. In order to freeze cells and still keep them alive, a cryoprotectant is added to the cultures. Cyroprotectants function by reducing the freezing point of the solution and act to reduce the formation of large ice crystals inside cells that could rupture membranes. Cryoprotectants are non-toxic to the cells, and are generally able to pass through the membranes into the cells.</P><OL><LI>Take 1 mL of overnight E. coli culture and add it to a clean, labeled 1.5 mL tube.</LI><LI>Add 200 µL of 80% glycerol to the tube (this creates a roughly 15% glycerol solution).</LI><LI>Mix well by inverting the tubes (unmixed glycerol will tend to stay separate from the cell solution.</LI><LI>Immediately place the cells into a -80 °C freezer box. There are often special freezing boxes that let the freezing occur more slowly, but for E. coli these are generally not needed.</LI><LI>To use the glycerol stock to establish a new culture, either scrape a very small amount of the frozen culture off with a pipette tip and add it to a culture (if it is still frozen), or add 10-20 µL of the liquid glycerol stock to the culture (if it is somewhat thawed). AVOID THAWING GLYCEROL STOCKS COMPLETELY – take what you need and quickly return them to the -80 °C freezer. If they do thaw completely, they can be re-frozen, but repeated thawing may reduce the number of live cells in the stock tube. </LI></OL><H4> DNA Extraction using CTAB</H4><OL><LI>Preparation step: Make sure Lysozyme (50 mg/mL) and RNase A (10 mg/mL) stock solutions have been prepared.</LI><LI>Preparation step: Make sure CTAB solution is dissolved. If crystals of CTAB are on the bottom of the bottle, place in 37oC incubator and shake occasionally to re-suspend. CTAB is just above its soluble conditions at normal room temperature, and if the atmospheric pressure is low the day you are extracting DNA, it might come out of solution.</LI><LI>Prepare two water baths, one boiling and the other 68C (this can also be accomplished with just the digital waterbath, by bringing the temp to 99/100C, then adding some water to the tank to help it cool down to 68C by the time that water temp is needed)</LI><LI>Centrifuge the 12 ml tubes containing the 5 ml cultures in the large centrifuge at 3K RPM for 10 min.  Discard supernatant liquid.  </LI><LI>Re-suspend cells in 200 ul of “STET for CTAB” buffer. Transfer to 1.5 ml tubes.</LI><LI>Add 5-10 ul (10 uL if older preparation) Lysozyme (50 mg/ml) and incubate at room temperature for 5 min. </LI><LI>Boil for 45 seconds and centrifuge for 20 min at 13K RPM (or until pellet gets tight).</LI><LI>Use a pipette tip to remove the pellet by dragging it (it should be somewhat slimy, but if pelleted well enough, it will hold together), if it doesn’t hold together, re-centrifuge and retry.</LI><LI>Add 5 ul RNase A (10 mg/ml) and incubate at 68C for 10 minutes.</LI><LI>Add 10 ul of 5% CTAB and incubate at room temperature for 3 min.</LI><LI>Centrifuge for 5 min at 13K RPM, discard supernatant, and re-suspend in 300 ul of 1.2 M NaCl by vortexing.</LI><LI>Add 750 ul of ethanol </LI><LI>Optional  Step (but gives better yield): incubate for 30 mins in -20oC freezer to help DNA precipitate</LI><LI>Centrifuge for 5 min at 13K RPM to compact DNA pellet. Make sure the hinge is away from the center of the rotor, this will make the pellet form on the bottom of the tube on the side of the hinge (it might be hard to see or invisible, so this way you know where it should be)</LI><LI>Discard supernatant, rinse pellet in 80% ethanol, and let tubes dry upside down with caps open.</LI><LI>Re-suspend DNA in ddH2O (50 uL). Vortex or pipette up and down to ensure re-suspension of DNA.</LI></OL><H4> Making Agarose Gels</H4><P>
		Agarose gels are useful for DNA purification and analysis of DNA sizes. The gels are made up of an agarose matrix composed of long strands of agarose, and gaps of various sizes between the strands. The larger the DNA molecule, the longer it takes to fit through the gaps, making its progress through a gel slower than a small DNA molecule. The DNA is drawn through the gel using an electric current; the negatively charged phosphates on the DNA backbone being attracted toward the cathode (“Run towards red” is a helpful mnemonic as the cathode is generally red colored). By varying the concentration of the agarose gel, it is also possible to increase the separation of bands of certain sizes on the gel. A 1% agarose gel is generally used as it provides separation of bands from 200 bp to 3000 bp. For separating larger bands, a 0.7% gel is typically used and the smaller DNA fragments are run completely off the bottom of the gel. For separating smaller bands, a 2% or 1.5% gel can be used, and run normally. Gels are useful for purifying DNA bands of a particular size from restriction digests (to prevent multiple products from forming during ligations) and also for removing proteins from a DNA sample (such as restriction enzymes that are not inactivated by heat). Gel purification has the downside of losing some DNA, and reducing overall DNA concentration (a 120 bp band of DNA in a 2000 bp plasmid will only give .06 µg of DNA if 1 µg of total DNA digest is added to the lane). For small band sizes (&lt; 200 bp), it may be necessary to use CIP or TAP dephosphorylation and ligation using the digested DNA solution without gel purification.</P><OL><LI>Determine the number of lanes you wish to run. Always plan for 1-2 lanes of the DNA ladder (2 especially if this gel will be cut up and DNA removed from it), or another suitable control. Most lanes can hold 20-25 µL of sample, so larger samples may need to be run on two lanes, or use the larger lane combs (40-50 µL capacity). The small gel box can hold 6 lanes of large capacity or 10 lanes of smaller capacity. The larger gel box can hold 12 lanes of large capacity or 20 lanes of small capacity. The large gel box is also capable of having 2 combs at a time (the second placed ½ down the box), and so its capacity can double at the cost of distance over which it can separate bands.</LI><LI>Once your gel box is selected, determine the concentration of gel you wish to make (see description for details). The concentration is the mass of agarose/mL of gel x 100%. The small gel box supports gels of 50 mL (potentially up to 75 mL, but 50 is easier to use) and the large gel box supports gels of 200 mL.</LI><LI>Set up the gel box by removing the gel tray. Make sure the rubber seals are still in their grooves. Apply a small amount of water to the inside of the side walls of the gel box and to the ends of the gel tray that have the rubber seals. Slide the gel tray into the gel box so that the open ends of the tray are against the box walls, and so that the rubber seals have not rolled up out of their grooves (if the seals moved, return them to their grooves and try sliding the tray in again).</LI><LI>Add the correct gel combs to the gel. The main comb for both gels goes into the first notch on the gel tray (should be 1-2 cm from an end), the secondary comb for the large gel is placed in the notch in the middle of the tray. The small gel combs have two sides (one thinner than the other) the thinner side has about 2/3 of the capacity volume of the thick side (which has the capacities listed in #1), so choose what you need.</LI><LI>In a flask that can hold at least 4x your gel’s volume, add the correct volume of 1x TAE buffer. DO NOT ADD WATER – the gel will not work correctly.</LI><LI>Weigh out the correct mass of agarose (NOT AGAR) and add it to the flask.</LI><LI>Microwave the solution until it boils. There are two stages of boiling – where small white bubbles form (frothy) and where large clear bubbles form. You want to let it boil a bit past the frothy stage and into the clear bubble stage. These bubbles will pop naturally, and will keep you from having a bubble filled gel. It is necessary when using higher gel concentrations (and recommended for all other concentrations) that the microwaving occur in 30 second increments, with the solution being stirred by GENTLE rotation (wear protective heat gloves) after each 30 second period, to ensure proper agarose distribution.</LI><LI>After microwaving add Ethidium Bromide to the gel solution. WARNING – carcinogen, glove use is advised (if you get it on yourself, wash your skin with water for 5 minutes – its very water soluble). Add 1 µL of Ethidium bromide for EACH 10 mL of gel volume (5 µL for small gel, 20 µL for large gel).</LI><LI>Mix Ethidium Bromide into the solution by GENTLE swirling (to avoid bubbles).</LI><LI>The gel solution can be allowed to cool slightly before pouring into the gel box (pouring boiling solution into gel box can cause it to warp and bend over time).</LI><LI>Gently pour the gel into the gel tray by leaning it on the gel box wall farthest from the top comb, and slowly tipping it into the box. This prevents bubbles from forming in the gel, and if they do form, forms them near the bottom of the gel.</LI><LI>If additional bubbles form in the gel box, while the gel is still liquid take a pipette tip and push the bubbles to the bottom edge of the gel, where they won’t interfere with DNA movement.</LI><LI>Allow gel to cool for 40 minutes to 1 hour. Test solidification by gently pressing on the bottom corner of the gel with a finger, it should feel solid and gel-like (not liquid). </LI></OL><H4> Restriction Digests</H4><P>
		All restriction digests were carried out using Fermentas reagents and restriction enzymes. The Fermentas FastDigest® (FD) reagents were used. </P><P>In a tube the following was added:
			<UL style="list-style-type:disc"><LI>4 µg DNA</LI><LI>4µl 10x FD Green buffer</LI><LI>1 µl FD restriction enzyme 1</LI><LI>1 µl FD restriction enzyme 2</LI><LI>Volume brought up to 40µl total with nuclease free water</LI></UL></P><H4> Gel Purification with Qiagen Gel Extraction Kit</H4><P> This procedure was carried out using the Qiagen QIAquick Gel Extraction Kit.</P><H4> Ligation </H4><P>Ligation reactions are used to combine two linear fragments of DNA into a circular plasmid. The ligation procedure can be modified based on how much backbone DNA you have and how much insert you have, as well as if a phosphatase such as CIP (calf intestinal phosphatase) or TAP (thermo-sensitive alkaline phosphatase) was used in preparing the linear DNA molecules. Example protocol for ligation (to be added to a PCR tube)</P><OL><LI>10µl Insert DNA</LI><LI>3µl Vector DNA</LI><LI>2µl 10X ligation buffer</LI><LI>34µl H2O</LI><LI>1µl T4 DNA ligase</LI></OL><H4> PCR </H4><P>	Add the following reagents to a tube (50 µl reaction) in the following volumes and order:</P><UL style="list-style-type:disc"><LI>32µl sterile H2O</LI><LI>5 µl 10X buffer</LI><LI>2µl dNTP Mix</LI><LI>3µl MgCl2</LI><LI>6µl cells/DNA</LI><LI>0.25µl Taq Polymerase</LI><LI>1µl Primer 1</LI><LI>1µl Primer 2</LI></UL><P> The thermocyler is setup beforehand with the desired protocol. Typically: 94°C for denaturing, 50-60°C for primer annealing, and 72°C for polymerase extending. </P><P>Example setup: </P><OL><LI>94°C 2min 1x</LI><LI> 94°C 45sec</LI><LI> 55°C 45 sec</LI><LI> 72°C 1min 15 sec</LI><LI> 72°C 5min 1x</LI><LI> 4°C indefinitely</LI></OL><P>Repeat Steps 2. and 4. 35x</P><H4> DNA Mutagenesis </H4><P>Use the <A href="https://www.genomics.agilent.com/CollectionSubpage.aspx?PageType=Tool&amp;SubPageType=ToolQCPD&amp;PageID=15">QuikChange II Primer Design Program</A> available online to create mutagenesis primers.</P><P>Use the procedures in the <A href="http://www.chem.agilent.com/Library/usermanuals/Public/200521.pdf"> QuikChange II XL Site-Directed Mutagenesis Kit Manual </A>. </P><H2><EM>L. lactis</EM></H2><H4>Media</H4><UL style="list-style-type:disc"><LI>M17-G (liquid media) = M17 broth + 0.5% glucose (added after autoclaving M17)</LI><LI>SMC-M17-G (recovery media for transformations) = 5ml 2X M17-G + 5 ml 1 M sucrose + 100 µL 2 M MgCl2 + 40 µL 0.5 M CaCl2</LI><LI>SM17-G agar (solid media) = M17 + 0.5 M sucrose + 0.5% glucose + 1.5% agar</LI></UL><H4>Electrocompetent Cell Prep</H4><OL><LI>Overnight culture of L. lactis in M17-G at 30°C and no shaking</LI><LI>Measure the OD600 of the M17-G media (or blank with this)</LI><LI>Inoculate overnight cells at 2% to 100 ml (or less) fresh M17-G liquid media</LI><LI>Place the M17-G culture into 30°C incubator, cells will grow for next 2-4 hours</LI><LI>While waiting, turn on ice machine, turn on centrifuge and set temperature to 4°C</LI><LI>Place 4x 50mL falcon tubes in the -20°C freezer</LI><LI>Place 4x 50 mL falcon tubes of double distilled water into 4°C fridge</LI><LI>Place 2x 50 mL falcon tubes of sterile 10% glucose into 4°C fridge</LI><LI>Place approximately 60 sterile bullet tubes (lids closed) into the -20°C</LI><LI>Measure the OD600 of the culture every 30 min for the first 2 hours. When OD600 reaches 0.4, record every 15 min. When OD600 reaches between 0.5 and 0.6, take culture out of incubator and place immediately on ice and swirl the contents of the flask to cool</LI><LI>Start centrifuging down the cells in the 4x 50mL pre-chilled falcon tubes. Have the settings of the centrifuge at 3500 rpm, 25 min, and 4°C. Ensure centrifuge is balanced</LI><LI>Pour off supernatant and re-suspend cells with 50mL ice cold double distilled water; centrifuge at same settings</LI><LI>Repeat steps 11 and 12; make sure the cells stay chilled and work quickly</LI><LI>After the last re-suspension, combine tubes, centrifuge, and then re-suspend in 30mL cold, sterile 10% glucose</LI><LI>Centrifuge and then re-suspend final pellet in about 1mL 10% glucose</LI><LI>Aliquot 50 µL (or up to 100 µL) of cells into pre-chilled bullet tubes</LI><LI>Immediately place all bullet tubes into -80°C freezer </LI></OL><H4>Electroporation Transformation</H4><OL><LI>Follow the sterilization method for E. coli transformations described above</LI><LI>	Use 50 µL, or up to 100 µL prepared cells, add 1-5 µg DNA</LI><UL style="list-style-type:disc"><LI>Can use less DNA if you don’t have that much</LI><LI>Can use less for most pTRKH2 transformations (especially positive control, just need a couple hundred nanograms), typically want to have close to 1 µg DNA </LI></UL><LI>Transfer cell suspensions to a pre-chilled and sterile 0.2 cm gap electroporation cuvette </LI><LI>Electroporate at 2000 V, 400 O, and 25 µF</LI><LI>Add 0.9 mL SMC-M17-G and transfer to 1.5 mL tubes, keep on ice 5-10 min</LI><LI>Incubate at 30°C for 2 hours</LI><LI>Plate on SM17-G agar + antibiotic (pTRKH2: 2.5 µg/mL erythromycin (Erm))</LI><LI>Incubate at 30°C, preferably anaerobically, but should still grow in aerobic conditions (will just take a day or so longer</LI></OL><H4>Glycerol Stock Prep</H4><OL><LI>Take 1 mL of overnight L. lactis culture and add 500 µL to two sterile, labeled 1.5 mL bullet tube.</LI><LI>Add 500 µL sterile 20% glycerol to each tube</LI><LI>Mix well by either inverting the tube or pipetting up and down several times</LI><LI>Immediately place the cells into the -80°C freezer for later use</LI></OL><H4>DNA Extraction Using Qiagen</H4><P>Our lab normally uses the CTAB DNA extraction method, however this method does not work with L. lactis, therefore we have to use the Qiagen alkaline lysis kit.</P><OL><LI>Grow up 5 mL culture overnight in LB (6 mL if you want glycerol stock – 1 mL will be used for the frozen stock, which should be removed before pelleting the cells in the next step)</LI><LI>Pellet 5-10 minutes in centrifuge at 3000-3500 rpm</LI><LI>Pour off supernatant</LI><LI>Check P1 solution for the checked RNase added box</LI><LI>Add 250 µL solution P1 to the 10 mL culture tube</LI><LI>Suspend the pellet in the P1 by pipetting up and down</LI><LI>Transfer suspended pellet to a 1.5 mL tube</LI><LI>Add 250 µL solution P2 to the tube</LI><LI>Mix by inverting the tube by hand 10-20 times. Solution should turn blue throughout, if not, continue inverting until blue throughout</LI><LI>Allow lysis to occur for 3-4 minutes (no more than 5 minutes)</LI><LI>Add 350 µL solution N3 to the tube</LI><LI>Mix by inverting tube by hand 10-20 times. Solution should lose all blue color, if not, continue inverting until all blue is gone.</LI><LI>Centrifuge at 13,000 rpm for 10 minutes (keep the hinge out to get the pellet to form correctly)</LI><LI>Using a pipette, remove the supernatant from the tubes, and apply to a labeled blue spin column from the kit</LI><LI>Centrifuge at 13,000 rpm for 1 minute</LI><LI>Pour flow-through BACK into the column and centrifuge at 13,000 rpm for 1 minute 17. Discard flow through</LI><LI>Add 750 µL PE solution to tube and centrifuge at 13,000 rpm for 1 minute</LI><LI>Discard flowthrough and centrifuge again at 13,000 rpm for 1 minute</LI><LI>Transfer blue column to a fresh 1.5 mL tube (labeled)</LI><LI>Add 30-50 µL ddH2O (depending on how concentrated you want your final DNA). Buffer EB (supplied in the kit) can also be used.</LI><LI>Incubate on benchtop for 10-15 minutes</LI><LI>Centrifuge at 13,000 rpm for 1 minute and discard column</LI><LI>Nanodrop to determine DNA concentration </LI></OL><H2><EM>Phage</EM></H2><H4>Propagation</H4><OL><LI>Grow <I>Lactococcus</I> strain overnight in M17-G liquid media at 30°C.</LI><LI>The next day, add 1% overnight culture to 5 mL fresh M17-G, and let it grow for 3-4 hours, until you see light growth when you swirl the tube</LI><LI>At that point add sterile 1M CaCl2 to a final concentration of 10mM.</LI><LI>Add 250 μL phage solution.</LI><LI>The culture should continue to grow for a while, and then clear.</LI><LI>The cleared cultured in then centrifuged and filtered with a 10 mL syringe and 0.45 μm syringe filter into a sterile tube.</LI><LI>The phage can be stored in the refrigerator and re-propagated as necessary.</LI></OL></ARTICLE></DIV></DIV></SECTION><DIV class="container"><SECTION><H3 style="text-align: center">Links to Stuff</H3><TABLE class="links"><TBODY><TR><TD><A href="https://2015.igem.org/Team:Utah_State">Home</A></TD><TD><A href="https://2015.igem.org/Team:Utah_State/Team">Team</A></TD><TD><A href="https://2015.igem.org/Team:Utah_State/Standards">Medal Criteria</A></TD><TD><A href="https://2015.igem.org/Team:Utah_State/Attributions">Attributions</A></TD></TR><TR><TD><A href="https://2015.igem.org/Team:Utah_State/Description">Project Description</A></TD><TD><A href="https://2015.igem.org/Team:Utah_State/Experiments">Experiments and Protocols</A></TD><TD><A href="https://2015.igem.org/Team:Utah_State/Results">Results</A></TD><TD><A href="https://2015.igem.org/Team:Utah_State/Design">Design</A></TD></TR><TR><TD><A href="https://2015.igem.org/Team:Utah_State/Interlab">Interlab Study</A></TD><TD><A href="https://2015.igem.org/Team:Utah_State/Practices">Human Practices</A></TD><TD><A href="https://2015.igem.org/Team:Utah_State/Collaborations">Collaborations</A></TD><TD><A href="https://2015.igem.org/Team:Utah_State/Safety">Safety</A></TD></TR><TR><TD><A href="https://2015.igem.org/Team:Utah_State/Parts">All Parts</A></TD><TD><A href="https://2015.igem.org/Team:Utah_State/Basic_Part">Favorite Parts</A></TD></TR></TBODY></TABLE></SECTION><DIV class="copyright"><UL class="menu" style="text-align:center;font-size:70%"><LI>© Untitled. All rights reserved</LI><LI>Design: <A href="http://html5up.net">HTML5 UP</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>