"
Page
Discussion
View source
History
teams
Log in
 
Team:Paris Bettencourt/Collaboration
From 2013.igem.org
PROJECT
OVERVIEW
DETECT
TARGET
INFILTRATE
SABOTAGE
ACHIEVEMENTS
PARTS
RESULTS
MEDAL FULFILLMENT
HUMAN PRACTICE
OVERVIEW
GENDER STUDY
TECHNOLOGY TRANSFER
TB FACTS
TB IN FRANCE
TB GALLERY
SAFETY
COLLABORATION
COLLABORATION
SENSIGEM
NOTEBOOK
DAY NOTES
PROTOCOLS
BIBLIOGRAPHY
TEAM
OUR TEAM
GALLERY
ATTRIBUTIONS
ACKNOWLEDGEMENTS
@iGEM_Paris
Skip to Calgary
Skip to BGU
Skip to WAX Science
Skip to Tips
In Science, it is all about collaborating. Working together makes better science and is fun! Throughout our iGEM summer, we have been in contact with several teams and successfully collaborated with 3 teams from 3 different continents! As part of our Biosensor project , we collaborated with the Calgary team, Canada , and built a biosensor iGEM database – sensiGEM . With the BGU Israel team, we collaborated in an experimental aspect. They sent us parts of them to clone and in return they did a western blot for us, which supports the work of the Infiltrate subproject . The Braunschweig Team from Germany was supported from us with literature they needed to continue working on their project. Below you can find a description as well as the outcome of our collaborations! As we have collaborated successfully with our partners we further provide some tips for how we think iGEM Teams can successfully collaborate.
Collaboration with the Calgary iGEM Team from Canada
   The Paris-Bettencourt-Calgary iGEM collaboration started last June when a few members from each team met at the SB6.0 synbio conference in London. After a few beers and lab stories, we learned that despite coming from the opposite sides of the globe, we were using synthetic biology to build biosensors to sense DNA. While our systems were targeted solve different problems, we were struck by a number of commonalities between these projects (Figure 1). As we shared our projects, we recalled how there was a lack of DNA biosensor parts in the Parts Registry. Moreover, we complained about the lack of organization of biosensors in the registry. The veteran iGEMers on each team mentioned that biosensors had consistently finished as grand prize winners in previous years of iGEM.
We were curious how biosensors have evolved since the beginning of iGEM and how our projects fit into the context of the iGEM Parts Registry.
Figure 1. The Calgary and Paris Bettencourt biosensors both sense DNA, albeit with some differences in how they function mechanistically.
SensiGEM - A biosensor database
   We decided to collaborate to answer these questions. Since our initial meeting in London, members of each team have conferenced weekly on Skype. After accustoming ourselves to the eight hour time difference, we developed SensiGEM, a collaborative database in which we catalogued all the biosensors in the history of iGEM.
Before studying the past Wikis, we realized that we had different definitions of biosensors. We asked each other a fundamental question: What is a biosensor? We developed the following definition: A biosensor is an engineered system that relies on biological systems or components to detect and report a condition. The condition(s) detected and reported could encompass an environmental, biological, chemical or synthetic aspect or compound in the sensor’s environment or surroundings.  Once agreeing on the nature of biosensors, we split up the Wikis from 2007 onward between Calgary and Paris-Bettencourt. We analyzed 936 project Wikis from 2007 to 2013 by hand, incorporating the projects which matched our biosensor definition into the collaborative SensiGEM database. We included 229 projects on
the database, some of which were biosensors as per the definition, as well as other projects containing biosensor elements that aligned with our definition.
We designed this database with future iGEM teams in mind, with tools for efficient navigation biosensors according to inputs, outputs, and their intended application. We made both SensiGEM’s source code and underlying data available under the permissive MIT license. This means that other teams can either collaborate with us on our version of the database or host their own independent copies. We foresee SensiGEM as a resource where future iGEM teams can showcase their biosensors.
Lessons from SensiGEM
   We conducted some preliminary analysis of the database in SensiGEM to see how our projects stand in the current iGEM biosensor landscape as well as to get an overview of what types of biosensors have been developed for iGEM.
In Figure 2, we can see the portion of biosensors of all iGEM projects since 2007. There is a clear linear increase in number of iGEM projects, but the number of biosensors per year varies and doesn’t follow a trend.
How successful Biosensor were in the previous years (Advance to Championship, Awards, Finalists) can be seen in Figure 3. Something that also interested us was to see how many biosensors are in the Track Health and Medicine as we developed a biosensor for that Category (Figure 4). Other facts we wanted get from the database are, how many biosensors already targeted DNA (Figure 5) and what is the distribution of other inputs, sensed by previous biosensors (Figure 6).
Figure 5. Number of iGEM biosensor projects in comparison to how many of these targeted DNA like the Calgary team and we did.
Figure 6. Overview of the different types of inputs for the biosensors in iGEM and how many teams used those inputs since 2007.
Figure 2. Number of iGEM projects since 2009 in comparison to number of Biosensor iGEM projects since 2007.
Figure 3. Success of Biosensor iGEM projects since 2007. Listed are the numbers of teams that advanced to the Championship (since 2009), that won awards or were finalists (since 2007).
Figure 4. Number of iGEM projects since 2007 in the Track Helath and Medicine and how many of these are/were biosensors
Looking toward finals
   Given the similarities between each of our systems in overall function, we have begun development of BioBricks to apply each system to the other team's problem. By testing each system on a different problem, we intend to show how each system can be deployed as a modular, platform technology.
We also intend to further improve our database to more easily get information of the actual sensing system (sensitive/inducible promoter, other targeting methods like CRISPRs, TALEs,…).
Collaboration with the BGU iGEM Team from Israel
A critical part of BGU's project requires a recombination of a linear DNA cassette into the bacterial chromosome.  They sent a cassette required for the recombination as well as the helper plasmid required for it to function.  They sent us their plasmid pUC57amp-cI which is responsible for the repression of the holin and lysozyme genes in their kill switch.  As the repression of these genes is essential to prevent the kill switch from being lethal prematurely, we decided to characterize the promoter units produced by the lac/ara-1 promoter that the cI is controlled by.
The first thing we needed to do was to biobrick the promoter from pUC57amp-cI.  We PCR'd biobrick restriction sites onto the lac/ara-1 promoter along with the RBS.  We then cloned the lac/ara-1 promoter into pSB1C3 backbone.  We extracted the promoter from pSB1C3 with EcoRI and SpeI cut sites, and a GFP reporter E0240 with XbaI and PstI and the backbone was prepared with EcoRI and PstI as per the protocol for characterizing promoter activity on the biobrick website.  At time of writing we are currently cloning the lac/ara-1 GFP construct for characterization.
Collaboration with the Braunschweig iGEM Team from Germany
Jonas Zantow an advisor of the Braunschweig iGEM Team from Germany visited us in Paris. We we showed him our lab and we gave him a quick overview of our projects and what we achieved so far. Also Jonas explained us what the Braunschweig 2013 iGEM Team
is working on and we could help him out with some Literature from our Bibliography we started during our Brainstorming sessions earlier the year. His visit ended - how can it be different, when we have German visitors - over a glass of beer!
Collaboration with the BGU iGEM Team from Israel
A critical part of BGU's project requires a recombination of a linear DNA cassette into the bacterial chromosome.  They sent a cassette required for the recombination as well as the helper plasmid required for it to function.  They sent us their plasmid pUC57amp-cI which is responsible for the repression of the holin and lysozyme genes in their kill switch.  As the repression of these genes is essential to prevent the kill switch from being lethal prematurely, we decided to characterize the promoter units produced by the lac/ara-1 promoter that the cI is controlled by.
The first thing we needed to do was to biobrick the promoter from pUC57amp-cI.  We PCR'd biobrick restriction sites onto the lac/ara-1 promoter along with the RBS.  We then cloned the lac/ara-1 promoter into pSB1C3 backbone.  We extracted the promoter from pSB1C3 with EcoRI and SpeI cut sites, and a GFP reporter E0240 with XbaI and PstI and the backbone was prepared with EcoRI and PstI as per the protocol for characterizing promoter activity on the biobrick website.  At time of writing we are currently cloning the lac/ara-1 GFP construct for characterization.
Collaboration with the association WAX Science
The Gender Study was realized in collaboration with the association WAX Science (http://wax-science.fr/). WAX Science is a French Association that promotes science without stereotypes. The iGEM team Paris Bettencourt and WAX Science worked together on what was the best way to treat the subject of  mixity and diversity in iGEM and in Synthetic Biology. .
The collaboration started with the definition of the project and continued throughout its realization, from the statistics to the analysis of the results
Tips for a successful collaboration
Important for the success of our cooperation was to regularly see each other on Skype. It is way easier to talk in person than to communicate by mail as misunderstandings and questions could be solved directly. We used Asana.com to set up tasks for each team that completed until the next week. Setting up those weekly tasks was a good idea as we were not overwhelmed by a lot of work but having those small tasks we could process them in time. Also the splitting up of work, as we did for example for the Wikis, was very helpful and part of the success to reach our goal in time.
For future collaborations, setting up weekly tasks and skyping every week worked well for us! Also important is that both teams have the same idea of the aim of the collaboration as well as how to achieve it. With the Calgary Team 2013 we found a great partner that fit to what we imagined!
Centre for Research and Interdisciplinarity (CRI) Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor Paris Descartes University 24, rue du Faubourg Saint Jacques 75014 Paris, France
+33 1 44 41 25 22/25
team2013@igem-paris.org
Copyright (c) 2013 igem.org. All rights reserved.
Retrieved from " http://2013.igem.org/Team:Paris_Bettencourt/Collaboration "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
