<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Project_Target"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Paris_Bettencourt/Project/Target">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris_Bettencourt/Project/Target&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Project/Target&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Project/Target&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris_Bettencourt/Project/Target" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris Bettencourt/Project/Target</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="navmenubkg"><DIV id="navmenubox"><UL id="nav"><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Overview" title="Project Overview">PROJECT</A></LI><LI style="margin-left:14.5px"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Overview" title="Project Overview">OVERVIEW</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" title="Detect">DETECT</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" title="Target">TARGET</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" title="Infiltrate">INFILTRATE</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" title="Sabotage">SABOTAGE</A></LI></UL></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Results" title="Achievements">ACHIEVEMENTS</A></LI><LI style="margin-left:141px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Parts" title="Submitted Parts">PARTS</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Results" title="Results">RESULTS</A></LI><LI style="width:167px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Medal_Fulfillment" title="Medal Fulfillment">MEDAL FULFILLMENT</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Overview" title="Human Practice Overview">HUMAN PRACTICE</A></LI><LI style="margin-left:141px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Overview" title="Human Practice Overview">OVERVIEW</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Gender_Study" title="Gender Study">GENDER STUDY</A></LI><LI style="width:168px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Technology_Transfer" title="Technology_Transfer">TECHNOLOGY TRANSFER</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/TB_Facts" title="TB Tacts">TB FACTS</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/TB_France" title="TB in France">TB IN FRANCE</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/TB_Gallery" title="TB Gallery">TB GALLERY</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Safety" title="Safety">SAFETY</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Collaboration" title="Collaboration">COLLABORATION</A></LI><LI style="margin-left:500px"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Collaboration" title="Collaboration">COLLABORATION</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/SensiGEM" title="SensiGEM">SENSIGEM</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Notes" title="Notebook">NOTEBOOK</A></LI><LI style="margin-left:649.5px"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Notebook" title="Day Notes">DAY NOTES</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Protocols" title="Protocols">PROTOCOLS</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Bibliography" title="Bibliography">BIBLIOGRAPHY</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Team" title="Team">TEAM</A></LI><LI style="margin-left:522.5px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Team" title="Team">OUR TEAM</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Gallery" title="Team Gallery">GALLERY</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Attributions" title="Attributions">ATTRIBUTIONS</A></LI><LI style="width:167px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Acknowledgements" title="Acknowledgements">ACKNOWLEDGEMENTS</A></LI><DIV id="twnoticon" class="sidelogos"><A class="twitter-timeline" width="250" height="400" data-dnt="true" href="https://twitter.com/iGEM_Paris" data-widget-id="368144501119795201">
            @iGEM_Paris</A></DIV><DIV id="page"><DIV class="overbox"><DIV class="bkgr"><H2>Background</H2><P>SirA is an essential gene in latent tuberculosis infections</P></DIV><DIV class="results"><H2>Results</H2><UL><LI>Produced an <I>E. coli</I> strain which relies upon mycobacterial sirA, fprA and fdxA genes to survive in M9 minimal media</LI><LI>Demonstrated that <I>E. coli</I> can survive with mycobacterial sulfite reduction pathway with Flux Balance Analysis</LI><LI>Realized in sillico modeling and identified experimentally a potential anti-TB activity of Pyridoxine at high doses.</LI><LI>Performed  a high throughput drug screening and identified 10 new potential anti-TB drug candidates.</LI></UL></DIV><DIV class="biocriks"><H2>BioBricks</H2><OL><LI><A href="http://parts.igem.org/Part:BBa_K1137000">BBa_K1137000 (SirA)</A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1137001">BBa_K1137001 (FprA)</A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1137002">BBa_K1137002 (FdxA)</A></LI></OL></DIV><DIV class="aims"><H2>Aims</H2><P>To perform a drug screen targeted at the sirA gene from mycobacteria</P></DIV><A href="#Introduction"><DIV class="hlink"><H2>Skip to Introduction</H2></DIV></A><A href="#Model"><DIV class="hlink"><H2>Skip to Modeling</H2></DIV></A><A href="#Design"><DIV class="hlink"><H2>Skip to Design</H2></DIV></A><A href="#Results"><DIV class="hlink" style="margin-right:0px"><H2>Skip to Results</H2></DIV></A></DIV><H2>Introduction</H2><DIV class="leftparagraph"><P>  
	SirA is essential for <I>M. tuberculosis</I> persistence phenotype as sulfur containing amino acids are particularly sensitive to oxidative stress within the macrophage and must regularly be replaced <A href="#Reference">(Pinto <I>et al</I> 2007)</A>. Currently, there are no drug candidates that are known to specifically inhibit SirA and conventional drug screens involve do not provide information regarding the mechanism of drug action nor do compounds that inhibit exponential growth necessarily have an effect on persistent TB. We designed a working drug screen assay to specifically target the mycobacterial sulfite reductase protein SirA. To this end we cloned Ito <I>E. coli </I><SPAN style="font-style: normal;">the sulfite reduction pathway</SPAN> of <I>M. smegmatis</I>, a non-pathogenic mycobacterial relative of <I>M. Tuberculosis</I>. Our model overcomes the problem of long doubling time of <I>M. tuberculosis</I>. Specific inhibition of the sulfite reduction pathway is scored by comparing a drug screen of our <I>E. coli</I> construct <I>vs.</I> wild-type. Any drug candidates that have activity against both the wild-type <I>E. coli</I> and our construct are non-specific inhibitors of <I>E. coli</I> growth. However, any drug candidates that inhibit only the growth of our <I>E. coli </I>construct will be <SPAN style="font-style: normal;">SirA</SPAN><SPAN style="font-style: normal;">pathway specific.</SPAN></P></DIV><DIV class="rightparagraph"><P><B>Figure 1: Overview of Targeted Drug Screen Design</B></P></DIV><H2>Flux Balance Analysis of Sulfite Reduction Pathway</H2><DIV class="leftparagraph"><P>  
	We used an <I>E. coli</I> model (iJR904) obtained from the <A href="http://bigg.ucsd.edu/bigg/main.pl">BiGG database</A> as a starting model to obtain wild-type growth rate (f = 0.9129 divisions/hour). We then deleted the reaction ‘SULR’ which encodes for the sulphite reduction pathway involving cysI and obtained a f=  -8e-13=0 divisions/hour indicating that the sulphite reduction pathway is essential for growth. Finally we introduced two new reactions for sirA and fprA and a new species fdxA. We found that growth with the mycobacteria pathway reverts the growth phenotype back to wild-type levels (f = 0.9105 divisions/hour).  We then wanted to expand our model to find new pathways that we could utilize for a targeted drug screen approach.  We wrote a matlab script that finds all the essential reactions in <I>M. tuberculosis</I> and all the essential reactions in <I>E. coli</I>, and then tries to complement the essential reactions in the <I>E. coli</I> model with the essential reactions from <I>M. tuberculosis</I>.  The model identified <A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target/FBA">100 metabolic reactions</A> that we could target.  Additionally, due to the modular nature of the model, it can be used to find target-able metabolic reactions in any SBML file.  The Matlab scripts can be found <A href="https://2013.igem.org/File:TargetFBA.zip">here</A> and requires <A href="http://opencobra.sourceforge.net/openCOBRA/Welcome.html">Cobra Toolbox 2.0</A> to function.  Please visit the FBA page for a detailed list of <A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target/FBA">results</A>.
      </P></DIV><DIV class="rightparagraph"><P><B>Figure 2: Biomass Flux through <I>E. coli</I> and mycoSIR E. coli</B></P><DIV style="font-size: 90%">Flux balance analysis was run using Cobra Toolbox 2.0 on <I>E. coli</I> sbml model iJR904 with and without SULR reaction.  Additionally an <I>E. coli</I> sbml model was built with the SULR reaction replaced with a reaction representing the mycobacterial SirA reaction and FprA reaction, as well as ferredoxin FdxA as an additional species.  The Biomass flux is restored to 99.75% of the wild-type level with the synthetic mycobacterial system.</DIV></DIV><H2>Structural Analysis of SirA</H2><DIV class="leftparagraph"><P>  
	Superimposing the structures of <I>M.tuberculosis</I> SirA and <I>E.coli </I> CysI reveals high homology, in particular of the active sites. Both proteins have the same symmetry (psuedo 2 fold) indicative of a common evolutionary origin. Our analysis highlighted important conserved residues, involved in substrate binding to be Arg97, Arg130, Arg166, Lys207. These positively charged residues are conserved in the sulphite/nitrite reductase family. In addition, 4 Cys residues are conserved for iron-sulphur binding. </P><P>The most profound structural differences between the two enzymes are found in the ferredoxin binding site and SirA's most C terminal residues and several surface loop regions due to deletions or insertions. A stark difference is a covalent bond formed between Cys161 (thiolate) and Tyr69 (C carbon atom) found adjacent to the redox center (Cu ions) in SirA. The covalently bound residues act as a secondary cofactor in tyrosyl radical stabilization. </P></DIV><DIV class="rightparagraph"><P><B>Figure 3: The superimposed 3D protein structures of SirA and CysI.</B></P><DIV style="font-size: 90%"> 303 amino acids are involved in superimposition with an rsmd of 1.41Å. All domains and loops of CysI are coloured purple, whilst SirA is coloured according to structural similarity with CysI: Red indicates poor alignment whilst blue indicates good alignment.</DIV></DIV><H2>Identification of potential drug target binding sites</H2><DIV class="leftparagraph"><P>  
	Our structural analysis provided the basis for our drug target prediction. Using Chembl and swiss pdb, we have shown a predicted drug target site. Our calculation gives strong favour for a drug to be effective at this site. The calculation reflects the suitability of small molecules to the binding site under the Lipinski's Rule of 5.</P><P>The drug target is located at the interface of the three domains. This binding pocket exhibits a dense hydrophobic region. Our analysis targets 48 amino acids of SirA within 6Å of a modelled small drug molecule. Of these residues, only 6 amino acids are charged: His409, Asp453, Asp474, His500, Asp504 and Arg541.
      </P></DIV><DIV class="rightparagraph"><P><B>Figure 4 Drug target locations in SirA </B></P><DIV style="font-size: 90%">A domain located in SirA, identified as a drug target through Chembl analysis.</DIV></DIV><H2>Structure based pharmacophore modelling of mycobacterial Fpra</H2><DIV class="leftparagraph"><P>  
	Using LigandScount 3.1, we searched over 8100 drug compounds from the BindingDB and Chembl databases for drugs targetting mycobacterial Fpra. Our search revealed Riboflavin (Vitamin B2) and Pyridoxine to be drug targets for Fpra. We used NADP interacting with the active site as the model of the pharmacore. Results showed pyridoxin to be a competitive inhibitor to NADP.  Pyridoxin is a synthetic compound currently available as a prescribed drug. </P><P>Chembl analysis of Pyridoxine (vitamin B6) show that it's properties fulfill Lipinski's criteria of being an orally active drug in humans. These properties state that any small drug molecule must have:  no more than 5 H bond donors, no more 10 H bond acceptors (N or O atoms), mol mass of less than 500 dalts and octanol-water partition coefficient log P of no greater than 5).</P><P>We have shown the proposed properties of Pyridoxine's interaction with Fpra as a competitive inhibitor to NADP at Fpra's active site. The key amino acids at the active site are Ala205, GLN204 and Thr208. GLN204 and Ala205 act as hydrogen bond acceptors whilst Thr208 interacts with a H via van der waals forces. Pyridoxin is a smaller, more lipid soluble molecule than NADP, thus more fitting to Lipinski's criteria. </P></DIV><DIV class="rightparagraph"><P><B>Figure 5: </B></P><DIV style="font-size: 90%">Our 3D model shows the structure of FNR where negative residues are coloured in blue, positive residues in red and NAD in purple (ball and stick representation).  The key amino acids at the active site are Glu211, Gly 366, Arg 110, Arg 199, Arg 200 and Asn155. Glu211 acts as a hydrogen acceptor whilst the latter four residues act as hydrogen donors.</DIV><P><B>Figure 6:</B></P><DIV style="font-size: 90%"> The interaction of Pyridoxine to its active site residues.</DIV></DIV><H2>Synthetic Mycobacteria Pathway</H2><DIV class="leftparagraph"><P>  
	We designed a synthetic <I>M.smegmatis-</I> derived sulfite reduction pathway containing sirA - the sulfite reductase, and two supporting genes that are required for its function in <I>E.coli</I>: fdxA and fprA. FdxA is a mycobacterial Ferredoxin cofactor which is oxidised by SirA during the sulfite reduction reaction and FprA is a Ferredoxin-NADPH reductase use replenish the reduced Fdx pool. The genes' sequences were taken from previous work describing their expression <A href="#Reference">(Pinto <I>et al</I> 2007)</A> in <I>E.coli</I> for purification and in vitro characterization; we removed restriction sites and codon optimized for expression in <I>E. coli</I>. The genes were then cloned into two Duet expression vectors, one containing sirA and one containing the supporting genesand were transformed into our knock-out mutant strains of <I>E. coli</I>.  Data on Growth curves can be found <A href="https://2013.igem.org/Team:Paris_Bettencourt/Notebook#target_Monday_30th_September.html">here</A>. 
      </P></DIV><DIV class="rightparagraph"><P><B>Figure 7: Growth curves of <I>E. coli</I> mycoSIR</B></P><DIV style="font-size: 90%">BL21 (DE3) ΔcysI containing the MycoSIR pathway (MycoSIR <I>E. coli</I>) were grown in liquid minimal media containing Various concentration of IPTG. (A) Replicates of each strain were measured for absorbance in a spectrophotometer every 10 minutes for 14 hours. Growth was observed for the WT BL21 <I>E. coli</I>, (blue), and the MycoSIR <I>E. coli</I> (red). No growth was detected for uninduced MycoSIR <I>E. coli</I> (purple) or for the BL21 (DE3) ΔcysI that did not contain the synthetic pathway (Orange) . (B) Mean Final ODs of all replicates, measured after 14 hours of growth. Growth was detected in zmSIR <I>E. coli</I> and WT BL21 but not in uninduced zmSIR strain.</DIV></DIV><H2>Creation of Knock out Mutants</H2><DIV class="leftparagraph"><P>  
	We prepared two strains of <I>E. coli</I> which have the sulfite reduction pathway deleted: BL21 (DE3) <I>ΔCysI Δfpr ΔydbK</I> and BL21 (AI) <I>ΔCysI</I>. CysI is responsible for sulfite reduction in <I>E. coli</I>, while <I>fpr and ydbK</I> are two non-essential genes that consume ferredoxin. These two genes are deleted, as sulfite reduction in mycobacteria is ferredoxin dependent in comparison to<I> E. coli</I> in which it is NADPH dependant. These genes were also removed to ensure that they do not interfere with our system. 
    </P></DIV><H2>Synthetic Corn Pathway</H2><DIV class="leftparagraph"><P>  
	Additionally we prototyped the system with a reconstruction of a sulphite reduction pathway previously designed and published by the silver group <A href="#Reference">(2011 Barstow et al)</A>. In place of CysI, a corn (Zea mays) derived sulfite reductase (zmSIR) was used. Two additional genes were included: Spinach ferredoxin (soFD),  and  corn derived ferredoxin NADP+ reductase (zmFNR). These genes, respectively, are required for production of the ferredoxin cofactor and the NADP+ ferredoxin reductase and are required for sulfite reductase (zmSIR) to function within <I>E. coli</I>.
      </P></DIV><DIV class="rightparagraph"><P><B>Figure 8:</B> Growth curves of <I>E. coli</I> maizeSIR</P><DIV style="font-size: 90%"> BL21 (DE3) ΔcysI containing the MaizeSIR pathway (MaizeSIR <I>E. coli</I>) were grown in liquid minimal media containing Various concentration of IPTG. (A) Replicates of each strain were measured for absorbance in a spectrophotometer every 10 minutes for 14 hours. Growth was observed for the WT BL21 <I>E. coli</I>, (blue), and the MaizeSIR <I>E. coli</I> (red). No growth was detected for uninduced MaizeSIR <I>E. coli</I> (purple) or for the BL21 (DE3) ΔcysI that did not contain the synthetic pathway (Orange) . (B) Mean Final ODs of all replicates, measured after 14 hours of growth. Growth was detected in zmSIR <I>E. coli</I> and WT BL21 but not in uninduced zmSIR strain.</DIV></DIV><H2>Results</H2><DIV class="leftparagraph"><P>  
	Upon successful cloning of the three genes into our <I>E. coli</I> deletion strain, we continued to confirm that all three genes are required for growth on minimal media. Our two synthetic pathways were found to rescue growth on a sulfurless amino acid supplemented minimal media.  We hope that this technique of using synthetic biology to overcome problems faced in naturally occurring systems will be both a large boon to the pursuit of finding novel drug candidates in <I>M. tuberculosis</I> and more broadly as this technique can be used for high-throughput screening of any pathway that can be constructed to be essential for growth in <I>E. coli</I>.
      </P><P><B>Figure 9:</B> Growth of zmSIR <I>E. coli</I> on minimal media. </P><DIV style="font-size: 90%">BL21 (DE3) ΔcysI cells transformed with 1, 2 and 3 genes of the 3-gene zmSIR synthetic pathway were grown for 24 hours on minimal media supplemented with 25 uM IPTG (see methods), along with a WT BL21 (DE3) serving as a negative control, and an untransformed BL21 (DE3) ΔcysI, as negative control. Rescue of growth required all genes of the synthetic pathway (SIR, FNR and FD). </DIV></DIV><H2>Z-score</H2><DIV class="leftparagraph"><P>  
	The Z-score is a statistical measurement aimed at assessing the &quot;hit effect&quot; in a drug screen high throughput screening. It is a commonly used measurement that shows how well did the drug effect the growth of the assay strain and how significant is the decrease in growth.</P><P>
	To calculate the Z-score we used our experimental <I>E. coli</I> strain BL21 (AI) ΔcysI that carries all three genes of the synthetic pathway (sirA, fprA, fdxA). We grew it in the M9 minimal media supplemented with amino acid sulfur dropout powder, in a 96 well plate.  Four of the wells were &quot;spiked&quot; with antibiotics (Amp, Gent, Kan, and Spect). </P></DIV><DIV class="rightparagraph"><P>  
	This served as a simulation of the drug screen without the actual drug library. Only the drug screen controls are used: growth in M9 as a negative control (no drugs) and growth in M9 + antibiotics as a positive control (a sure hit). We then compared the distribution of the growth (OD) in the negative control with the distribution of growth (OD) in the positive control. The Z-score shows the distance of the negative control mean from the positive control mean in negative control standard deviation units.</P><P><B>Our Z-score is: -10.2.</B></P></DIV><H2>Z-factor</H2><DIV class="leftparagraph"><P>  
	Z-factor is a measurement complementary to the Z-score. It measures the assay's quality based on the same data  extracted from the same experiment made for the Z-score. This calculation gives an estimation of how far the negative controls are from the positive controls. It is a comparison of the two distributions which assumes that both distributions are normal and calculate how far 99% of the data points of each distribution are from each other.</P></DIV><DIV class="rightparagraph"><P>  
	Z-factor is given on a scale from 0 to 1. Scores between 0.5 and 1 show that the assay is good and will enable testing in High throughput screens.</P><P><B>Our Z-factor score is 0.58.</B></P></DIV><H2> MycoSir growth assays reveal the potential anti TB activity of Pyridoxine </H2><DIV class="leftparagraph"><P>  Motivated by the results of our computational analysis, we attemped to use MycoSIR <I>E. coli</I> to assay the activity of pyridoxine, our candidate FprA inhibitor and a potential anti-TB compound. Briefly, we added both pyridoxine and a control compound, riboflavin, to growing cultures of both WT <I>E. coli</I> and MycoSIR <I>E. coli</I>. In all cases, cells were grown in minimal media, where our previous  work demonstrates that the MycoSIR pathway is essential for viability.
</P></DIV><DIV class="rightparagraph"><P>  Our growth assays indicate that pyridoxine, at high doses, specifically inhibits the growth of MycoSIR <I>E. coli</I> and therefore acts specifically on the mycobacterial sulfur pathway. While the observed affinity is low, it could in principle be expanded through derivitivizaion and further screening.
These results indicate that our MycoSIR <I>E. coli</I> are a practical tool for measuring drug activities. We have ordered several small drug libraries to assay with our strain, and we look forward to finding more candidate anti-TB drugs!</P></DIV><DIV class="leftparagraph"><P><B>Figure 10:</B>Riboflavin has no effect on the growth of WT or synthetic MycoSIR <I>E. coli.</I></P>  The indicated quantities of riboflavin were dissolved in water and added to cultures of WT or MycoSIR <I>E. coli</I> in 4 biological replicates. No significant growth effects were observed. </DIV><DIV class="rightparagraph"><P><B>Figure 11:</B> MycoSIR <I>E. coli</I> growth assays reveal a potential anti-TB acitivity of pyridoxine at high doses. </P><DIV style="font-size: 90%"> The indicated quantities of pyridoxine were dissolved in water and added to cultures of WT or MycoSIR <I>E. coli</I> in 4 biological replicates.  Both strains were grown in defined minimal media, where MycoSIR <I>E. coli</I> require our synthetic pathway for growth. Low pyridoxine doses had no detectable effects. However, a very high dose of pyridoxine (10 mg/mL) substantially inhibited the growth of MycoSIR <I>E. coli</I> yet showed no effect on WT growth.  This suggests pyridoxine  specifically inhibits the activity of the Mycobacterial SirA pathway.  Derivativization or other methods could be used to further enhance the affinity and specificity of this compound.</DIV></DIV><H2>High throughput screening : 10 new potential drug candidates</H2><DIV class="leftparagraph"><P>  
	Using MycoSirA we performed a high throughput drug screening. We screened two libraries from the NIH, Diversity Set IV and Natural Product Set II. 
Good drug candidates are the ones in which wild type <I> E. Coli </I> grow well but not <I> MycoSirA </I>. We found 10 compound which ODs  differ clearly from the general distribution (Fig 12). Their Z-Score are all higher than 3. Those ten compounds are potential new drug candidates.
Among those 10 potential drug candidates, six share structural similarities. Those structural similarities are also shared with pyrodoxine.
      </P></DIV><DIV class="rightparagraph"><P><B>Figure 12</B> High throughput drug screening performed on the library &quot;NIH Diversity Set IV&quot;. <B> A.  </B>Scatter plot ; in red, compounds that differ from the general distribution : low growth for MycoSiRA, normal growth for E; Coli <B> B.  </B> OD Ratios of all the tested compound and the best candidates..</P></DIV></DIV><H2>Literature</H2><DIV class="leftparagraph"><UL><LI>Global Alliance for TB Drug Development, Tuberculosis. Scientific blueprint for tuberculosis drug development, Tuberculosis (Edinb) 81 Suppl 1, 1–52 (2001).</LI><LI>World Health Organization, Global Tuberculosis Report 2012 (2012).</LI><LI>K. Raman, K. Yeturu, N. Chandra, targetTB: A target identification pipeline for Mycobacterium tuberculosis through an interactome, reactome and genome-scale structural analysis, BMC Syst Biol 2, 109 (2008).</LI><LI>R. Pinto, J. S. Harrison, T. Hsu, W. R. Jacobs, T. S. Leyh, Sulfite Reduction in Mycobacteria, Journal of Bacteriology 189, 6714–6722 (2007).</LI><LI>B. Barstow C. M. Agapakis, P. M. Boyle, G. Grandl, P. A. Silver, E. H. Wintermute, A synthetic system links FeFe-hydrogenases to essential E. coli sulfur metabolism, J Biol Eng 5, 7 (2011).</LI></UL></DIV><DIV class="rightparagraph"><UL><LI>Schellenberger J, Que R, Fleming RMT, Thiele I, Orth JD, Feist AM, Zielinski DC, Bordbar A, Lewis NE, Rahmanian S, Kang J, Hyduke DR, Palsson BØ. 2011 Quantitative prediction of cellular metabolism with constraint-based models: the COBRA Toolbox v2.0. Nature Protocols 6:1290-1307.</LI><LI>Schellenberger, J., Park, J. O., Conrad, T. C., and Palsson, B. Ø., BiGG: a Biochemical Genetic and Genomic knowledgebase of large scale metabolic reconstructions, BMC Bioinformatics, 11:213, (2010).</LI><LI>S. G. Franzblau et al., Comprehensive analysis of methods used for the evaluation of compounds against Mycobacterium tuberculosis, Tuberculosis 92, 453–488 (2012).</LI><LI>D. J. Payne, M. N. Gwynn, D. J. Holmes, D. L. Pompliano, Drugs for bad bugs: confronting the challenges of antibacterial discovery, Nat Rev Drug Discov 6, 29–40 (2006).</LI><LI>M. Nakayama, T. Akashi, T. Hase, Plant sulfite reductase: molecular structure, catalytic function and interaction with ferredoxin, J. Inorg. Biochem. 82, 27–32 (2000).</LI></UL></DIV><H2>Attributions</H2><DIV class="leftparagraph"><UL><LI>Strains NEBTurbo, BL21 (DE3) KO20, BL21 AI were provided by INSERM U1001.</LI><LI>Plasmids pET Duet, pACYC Duet, pACYC zmSIR, pACYC soFD zmSIR, pCDF FNR were provided by INSERM U1001.</LI></UL></DIV><DIV class="rightparagraph"><UL><LI>Genes msSirA, msFprA, msFdxA were synthesized by IDT.</LI><LI>Project was designed by Idonnya Aghoghogbe, Yonatan Zegman, Matthew Deyell and Edwin Wintermute.  All experiments and modelling were performed by Idonnya Aghoghogbe, Yonatan Zegman, Matthew Deyell. </LI></UL></DIV><DIV class="mfooter"><DIV class="footbox" style=""><B>Centre for Research and Interdisciplinarity (CRI)</B>
    Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor
    Paris Descartes University
    24, rue du Faubourg Saint Jacques
    75014 Paris, France +33 1 44 41 25 22/25<A href="mailto:team2013@igem-paris.org">team2013@igem-paris.org</A></DIV><CENTER>
    Copyright (c) 2013 igem.org. All rights reserved.
  </CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Paris_Bettencourt/Project/Target">http://2013.igem.org/Team:Paris_Bettencourt/Project/Target</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Bettencourt/Project/Target" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Bettencourt/Project/Target" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Project/Target&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Project/Target&amp;oldid=362050" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>