<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Project_Sabotage"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Paris_Bettencourt/Project/Sabotage">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris_Bettencourt/Project/Sabotage&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Project/Sabotage&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Project/Sabotage&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris_Bettencourt/Project/Sabotage" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris Bettencourt/Project/Sabotage</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="navmenubkg"><DIV id="navmenubox"><UL id="nav"><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Overview" title="Project Overview">PROJECT</A></LI><LI style="margin-left:14.5px"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Overview" title="Project Overview">OVERVIEW</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" title="Detect">DETECT</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" title="Target">TARGET</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" title="Infiltrate">INFILTRATE</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" title="Sabotage">SABOTAGE</A></LI></UL></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Results" title="Achievements">ACHIEVEMENTS</A></LI><LI style="margin-left:141px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Parts" title="Submitted Parts">PARTS</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Results" title="Results">RESULTS</A></LI><LI style="width:167px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Medal_Fulfillment" title="Medal Fulfillment">MEDAL FULFILLMENT</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Overview" title="Human Practice Overview">HUMAN PRACTICE</A></LI><LI style="margin-left:141px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Overview" title="Human Practice Overview">OVERVIEW</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Gender_Study" title="Gender Study">GENDER STUDY</A></LI><LI style="width:168px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Technology_Transfer" title="Technology_Transfer">TECHNOLOGY TRANSFER</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/TB_Facts" title="TB Tacts">TB FACTS</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/TB_France" title="TB in France">TB IN FRANCE</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/TB_Gallery" title="TB Gallery">TB GALLERY</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Safety" title="Safety">SAFETY</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Collaboration" title="Collaboration">COLLABORATION</A></LI><LI style="margin-left:500px"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Collaboration" title="Collaboration">COLLABORATION</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/SensiGEM" title="SensiGEM">SENSIGEM</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Notes" title="Notebook">NOTEBOOK</A></LI><LI style="margin-left:649.5px"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Notebook" title="Day Notes">DAY NOTES</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Protocols" title="Protocols">PROTOCOLS</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Bibliography" title="Bibliography">BIBLIOGRAPHY</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Team" title="Team">TEAM</A></LI><LI style="margin-left:522.5px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Team" title="Team">OUR TEAM</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Gallery" title="Team Gallery">GALLERY</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Attributions" title="Attributions">ATTRIBUTIONS</A></LI><LI style="width:167px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Acknowledgements" title="Acknowledgements">ACKNOWLEDGEMENTS</A></LI><DIV id="twnoticon" class="sidelogos"><A class="twitter-timeline" width="250" height="400" data-dnt="true" href="https://twitter.com/iGEM_Paris" data-widget-id="368144501119795201">
            @iGEM_Paris</A></DIV><DIV id="page"><DIV class="overbox"><DIV class="bkgr"><H2>Background</H2><P>One of the main concerns about tuberculosis today is the emergence of antibiotic resistant strain.</P></DIV><DIV class="results"><H2>Results</H2><UL><LI>Construction and characterization of phagemids coding for small RNA targeting antibiotic resistance proteins.</LI><LI>Showed theoretically burden of a device is critical for the maintenance of a genetic element in a population.</LI><LI>Successful conversion of antibiotic resistant population of <I>E. coli</I> to a sensitive state.</LI></UL></DIV><DIV class="biocriks"><H2>BioBricks</H2><OL><LI><A href="http://parts.igem.org/Part:BBa_K1137009">BBa_K1137009 (sRNA anti Kan)    Characterized </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1137010">BBa_K1137010 (sRNA anti Cm)    Characterized</A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1137011">BBa_K1137011 (sRNA anti Lac)   </A></LI></OL></DIV><DIV class="aims"><H2>Aims</H2><P>Our objective is to make an antibiotic-resistant bacterial population sensitive again to the same antibiotic.</P></DIV><A href="#Design"><DIV class="hlink"><H2>Skip to Design</H2></DIV></A><A href="#Results"><DIV class="hlink"><H2>Skip to Results</H2></DIV></A><A href="#Perspectives"><DIV class="hlink" style="margin-right:0px"><H2>Skip to Perspectives</H2></DIV></A></DIV><H2>Introduction</H2><DIV class="leftparagraph"><P>  
	
	As antibiotic-resistant bacterial strains are rising and pharmaceutical pipelines are drying up, we decided to invest in a new strategy based on specific silencing of the genes responsible for resistance through bio-engineered stealth bacteriophages.
	The silencing of the genes is obtained with a simple and modular system of tailor-made small RNAs. The spreading of this construct in a bacterial population is based on an autonomous phagemid/helper phage system. We demonstrated the validity of this trojan horse strategy by converting back to a sensitive state populations of bacteria initially resistant to antibiotics as chloramphenicol or kanamycin.
	
      </P></DIV><DIV class="rightparagraph"><B>Figure 1.</B> The Trojan Horse Strategy : Infecting the cells with a phage bearing a sRNA silencing the expression of antibiotic resistant genes converting it back to a sensitive state.
    </DIV><H2>Design</H2><DIV class="leftparagraph"><P>  <B>Designing a silencing device</B></P><P>
	Most antibiotic-resistance result from specific bacterial proteins such as efflux pump that extract the antibiotics from the cells or enzymes that quickly metabolize the antibiotic molecules. We choose a straightforward strategy to force bacteria to stop producing those proteins by targeting their cognate mRNA with tailor-made small RNAs. In order to design our silencing device, we used the protocol described in <A href="https://2013.igem.org/Team:Paris_Bettencourt/Bibliography#Na2013" target="_blank"><I>Na et al </I><SPAN style="font-style: normal;">(</SPAN>2013)</A> and inserted a 24bp sequence complementary to the RBS area of the target genes inside the published scaffold sequence that allow the stabilization of the hybridization through recruitment of the Hfq protein. <A href="https://2013.igem.org/Team:Paris_Bettencourt/Notebook#sabotage_Friday_28th_June.html" target="_blank">We designed</A> three sRNA respectively targeting Kanamycin resistance gene located on a pCOLA Duet Vector, Chloramphenicol resistance on pACYC Duet Vector and lacZ on E. coli's chromosome. </P></DIV><DIV class="rightparagraph"><P><B>Figure 2.</B> Structure of the Chloramphenicol resistance silencing module </P></DIV><DIV class="leftparagraph"><P>  <B>Designing a genetic element that spread in a population</B>
	We adopted non-lytic filamentous bacteriophages as our vectors of choice as they are good at spreading genetic elements in bacterial populations. However, being infected by a phage represents a huge burden for an individual bacteria which we thought would be detrimental for our construct to be able to maintain itself long enough in a population for this system to be clinically relevant. 
	Therefore we chose a phagemid/helper system which is composed of two mobile interacting elements. The “helper” elements of this system is an M13 phage mutant that provides packaging proteins but largely lacks the capacity to be packaged; the “light” element called phagemid is a normal plasmid harbouring the packaging signal and can thus propagate from hosts carrying the “helper” phage.
	As a result a cell containing both elements will produce and secrete many encapsulated phagemids together with a small minority of helper phages.
      </P></DIV><DIV class="rightparagraph"><P>   We expect such a system to infect with phagemids the majority of cells in a population and to be able to spread and maintain itself thanks to a small number of co-infected cells harboring helper phages, transforming them into phage-producing cells.

	Our silencing device is cloned on the light phagemid element in order to spread it efficiently. As the post-transcriptional regulation we are using only rely on RNA, it does not require protein synthesis, resulting in a very low cost for the cell. Moreover producing protective proteins against antibiotics is costly for the cell and lowers its fitness in an antibiotic free environment, we thus expect our silencing device to be a temporary relief for the infected cell which should avoid early counter-selection dynamics.
      </P></DIV><DIV class="leftparagraph"><P>  <B>Design of a sequential killing strategy</B>
	To perform our proof of concept experiments, our targeted genes were different antibiotic resistance genes on commercial plasmids that were transformed in <I>E. coli</I> MG1655 cells. Just as in nature where resistance genes are often found on mobile elements, our model respects the constraint of having multiple copies of the target gene to silence which would not have been the case with a chromosome located target. 
	The experimental set up consisted of a sequential protocol: (I) Phage infection leads to the silencing of the antibiotic-resistance genes and (II) cells are plated on antibiotics to kill the sensitive phenotype and quantify the efficiency of the silencing.
      </P></DIV><DIV class="rightparagraph"><B>Figure 3.</B> A sequential killing strategy  : <DIV style="font-size:80%"> From a  overnight culture of Chloramphenicol resistant cells (MG1665 + pACYc Duet Vector),  different sample of cells were grown to OD 0.7, and infected (or not) with sRNA phages ( isolated from supernatant from overnight culture of cells producing phages) with a volume ratio of phage vs cells of 1 to a 100). The mix was incubated  at  37°C  for  45  minutes.    Appropriate  dilutions were then plated on Chloramphenicol plates and LB plates. Colonies were screened after overnight incubation at 37°C.</DIV></DIV><H2>Characterization of the phagemid system</H2><DIV class="leftparagraph"><P>  
	The phagemid Helper system (kindly provided by Monica Ortiz of D. Endy lab) consists of a helper plasmid (M13K07) bearing Kanamycin resistance and a phagemid (Litmus-28) bearing GFP and an Ampicilin resistance cassette. Using those different markers, <A href="https://2013.igem.org/Team:Paris_Bettencourt/Notebook#sabotage_Wednesday_24th_July.html" target="_blank">we characterized</A> the system by infecting the cells with a ratio of phage of 1 to 100 and plating appropriate dilutions on different medias (LB, Kan, Amp, Kan/Amp). We show that the phagemid is spread a thousand fold more than the helper plasmid. There are as many cells infected with the helper as cells infected by both helper and phagemid, that is to say, phages producing cells. Those results show the validity of using this system to spread a genetic element in a bacterial population. Most of the cells bear the desired genetic element and the spread is carried on by a minor quantity of cells producing the vector.
      </P></DIV><DIV class="rightparagraph"><B>Figure 4.</B> Characterization of the phagemid helper system
    </DIV><H2>Effect of device burden</H2><DIV class="leftparagraph"><P>Every introduced system to a cell will consume resources and compete for internal molecular machinery. In case the device provides no direct benefit for the organism, the burden of the device will lead to a decrease in growth rate, and therefore a lower <I>fitness</I>. One way to avoid out-competition due to lower growth rate is by horizontal gene-transfer. We studied the behavior of a the <I>phagemid/helpel</I> system, which is a derived from from the non-lytic bacteriophage <I>M13</I>. This systems propagate through <I>E. coli F+</I> populations via two vectors: a <I>phagemid</I> carrying the desired device, and a <I>helper</I> carrying the necessary machinery for production of the bacteriophage. Only when both vectors are present in a cell it will produce bacteriophage (see diagram on the right). For more details please go to <A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage/Model">here</A>.</P></DIV><DIV class="rightparagraph"><P><B>Figure 5:</B></P><DIV style="font-size:90%"> Left: scheme representing the regular non-lytic M13 bacteriophage horizontal spread. Right: scheme representing the main processes of the phagemid/helper system. Cell populations are denoted by circles, and transferable elements by pentagons. Other processes such as cell division, substrate consumption, and dilution are not represented.</DIV></DIV><DIV class="leftparagraph"><P>We found that our system should be stable in time, even under chemostat conditions. At common parameters found in the literature, we found that the burden of the device has no strong effect. However, relatively low variations (one order of magnitude reduction), makes small variations in growth rate extremely relevant for the time the infected populations remain in the system. We expect that the phage production rates under limiting conditions such as the lungs to be smaller than those described in liquid cultures, making desirable to reduce growth rate as much as possible.</P></DIV><DIV class="rightparagraph"><P><B>Figure 6:</B> Al lower phage production rates, relatively low differences in burden matter</P><DIV style="font-size:90%"> Blue, green, red, and magenta, represent recipient (<I>R</I>), phagemid infected (<I>P</I>), helper infected (<I>H</I>), and cells infected with both (factory) respectively. In the top phagemid cells grow at 99% of recipient cells, whereas in the bottom they grow at 98%.</DIV></DIV><H2>Making a Chloramphenicol resistant E. Coli population sensitive to Chloramphenicol</H2><DIV class="leftparagraph"><P>  
	We used our designed phagemid, bearing the sRNA anti-Cm, to infect a population of chloramphenicol resistant cells. <A href="https://2013.igem.org/Team:Paris_Bettencourt/Notebook#sabotage_Tuesday_10th_September.html" target="_blank">By applying the sequential killing strategy</A> described in Figure 3, we efficiently killed 99,1% of the bacterial population (chloramphenicol concentration 1000ug/ml). We tested this strategy with different concentrations of chloramphenicol. We show that the rise of chloramphenicol concentration directly lead to drastic reduction of the survival of the infected cells.
      </P><P>  
	In order to show the general application of our system, we then applied the same strategy on MG1665 transformed with pCOLA Duet. Those kanamycin resistant cells were infected with phages bearing our anti-Kan sRNA.  <A href="https://2013.igem.org/Team:Paris_Bettencourt/Notebook#sabotage_Wednesday_18th_September.html" target="_blank">99,87% of the population was killed</A> (standard deviation 0,23 %) at 1000ug/mL of kanamycine
      </P></DIV><DIV class="rightparagraph"><P><B>Figure 7.</B> Killing  Chloramphenicol resistant and Kanamycin resistant <I>E. Coli</I> population with antibiotic </P></DIV><H2>Charaterization of the origin of resistance </H2><DIV class="leftparagraph"><P>  
	One of the main limits of our system could be the development of resistances. Indeed, there are at least two ways to become resistant to our system: by avoiding infection or by developing a resistance to the silencing. In order to get a better idea of what is the main cause of resistance we screened the survivor colonies for GFP expression. GFP positive colonies correspond to infected cells with malfunctioning silencing. <A href="https://2013.igem.org/Team:Paris_Bettencourt/Notebook#sabotage_Wednesday_11th_September.html" target="_blank">We show</A> that 70% of resistance is caused by a resistance towards the sRNA. This resistance could be due to a mismatch between the sRNA sequence and the target sequence resulting from a single point mutation. 
      </P></DIV><DIV class="rightparagraph"><P><B>Figure 8.</B> Origin of resistance to the sequential killing </P></DIV><H2>Conclusion and perspectives</H2><DIV class="leftparagraph"><P>  <B>Several sRNA on the same phagemid : </B> 
	Although our silencing small RNAs were assembled in phagemids with Gibson assembly, the Biobricks cutting sites included in our BBG adapter method provide full modularity. In our case a direct application is the assembly of multiple silencing modules (promotor+sRNA+terminator) on a single phagemid vector. As the main clinical problem with bacteria is multi or even pan-resistance this modularity provide an interesting strategy to explore as it allows simultaneous downregulation of multiple target genes. 
      </P><P>  <B>Building a library of sRNA : </B> 
	As our silencing strategy rely on base pair complementarity, a main concern from the clinical point of view would be the fast emergence of resistances to silencing through single point mutations. Indeed even if the majority of mutations that would decrease the efficiency of the silencing would also lead to an impaired or non functionnal protective protein, some neutral mutations will still be made possible by the redundancy of the genetic code. To tackle this issue an interesting solution would be to build a library of small RNA containing single point mutation on those particular neutral positions and load it on our phagemids. By being one move ahead of evolution we expect that such an anticipating strategy would considerably slow down the emergence of resistance to silencing.
      </P></DIV><DIV class="rightparagraph"><P>  <B>Possible applications : </B> As most bacteria species have their own phages, our strategy could be adapted to a broad range of diseases. Tuberculosis of course is particularly concerned because the number of panresistance cases is exploding in hospitals but potential targets could also be Staphylococcus aureus infections or even dangerous strains of E. coli as suggested in our model. 
      </P><P>  <B>sRNAs as bioricks, an adaptable silencing tools : </B>  We biobricked the sRNAs described in that study with their scaffold. Since the targeting sequence of the system is only 24 bp, it is very easy to engineer new sRNA by using a classic mutagenic PCR technique (Seung Min et al, 2013). This means that every iGEM team is now one PCR away from silencing any gene it wants . 
      </P></DIV><H2>Literature</H2><DIV class="leftparagraph"><UL><LI> Na D, Yoo SM, Chung H, Park H, Park JH, Lee SY. Metabolic engineering of Escherichia coli using synthetic small regulatory RNAs. Nature Biotechnology(2013).
	</LI><LI> Monica E Ortiz and Drew Endy. Engineered cell-cell communication via DNA messaging. Journal of Biological Engineering (2012).
	</LI></UL></DIV><DIV class="rightparagraph"><UL><LI> Seung Min Yoo, Dokyun Na and Sang Yup Lee. Design and use of synthetic regulatory small RNAs to control gene expression in Escherichia coli. Nature Protocols 8, 1694–1707 (2013).
	</LI><LI> Woodford N, Wareham DW; UK Antibacterial Antisense Study Group. Tackling antibiotic resistance: a dose of common antisense? The Journal of Antimicrobial Chemotherapy. 2009 Feb;63(2):225-9.
	</LI></UL></DIV><H2>Attributions</H2><P>The project itself was designed and accomplished by Clovis Basier, Aude Bernheim and Vincent Libis in consultation with Edwin Wintermute and Ariel Lindner. Modeling was done by Sebastian Jaramillo-Riveri.</P><P>Most of the strains (MG1655, MG, MGZ1, NED turbo) and plasmids (Duet vectors, pUC18) used for this project were kindly provided by the INSERM U1001 lab.</P><P>The phagemid template as well as the helper plasmid were provided by Monica Ortiz from the Endy Lab, Stanford.</P><P>We want to thank each member of the team Paris Bettencourt who worked with us and permit this project ! </P></DIV><DIV class="mfooter"><DIV class="footbox" style=""><B>Centre for Research and Interdisciplinarity (CRI)</B>
    Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor
    Paris Descartes University
    24, rue du Faubourg Saint Jacques
    75014 Paris, France +33 1 44 41 25 22/25<A href="mailto:team2013@igem-paris.org">team2013@igem-paris.org</A></DIV><CENTER>
    Copyright (c) 2013 igem.org. All rights reserved.
  </CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage">http://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Bettencourt/Project/Sabotage" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Bettencourt/Project/Sabotage" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Project/Sabotage&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Project/Sabotage&amp;oldid=361096" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>