"
Page
Discussion
View source
History
teams
Log in
 
Team:Paris Bettencourt/Project/Phage Sensor
From 2013.igem.org
PROJECT
OVERVIEW
DETECT
TARGET
INFILTRATE
SABOTAGE
ACHIEVEMENTS
PARTS
RESULTS
MEDAL FULFILLMENT
HUMAN PRACTICE
OVERVIEW
GENDER STUDY
TECHNOLOGY TRANSFER
TB FACTS
TB IN FRANCE
TB GALLERY
SAFETY
COLLABORATION
COLLABORATION
SENSIGEM
NOTEBOOK
DAY NOTES
PROTOCOLS
BIBLIOGRAPHY
TEAM
OUR TEAM
GALLERY
ATTRIBUTIONS
ACKNOWLEDGEMENTS
@iGEM_Paris
<body>
DETECT
We develop a sensor to target antibiotic resistances in tuberculosis to test if a specific strain carries a certain antibiotic resistance. Our sensor system consists out of a phagemid with a CRISPR/Cas system and LacZ as a reporter under the control of a pREC promoter (the promoter of the RecA protein that is involved in the stress response of bacteria). If our CRISPR/Cas system can now bind to the target (antibiotic resistance gene), the Cas9 generates at this specific target site a double strand break, which then starts the expression of our reporter, as the promoter gets active at stress that results from double strand breaks. Because our system is on a phagemid, the sensor system will be spread all over the population, which will give a clear color output if the target has been detected. This means, depending on what target sequence our system carries, we can identify the different antibiotic resistances that a strain might carry. This is a novel way of detecting resistances in bacterial strains. As a proof of concept we will use E.coli to target KanR. The used phage will be M13.  This sensor could potentially be used to test if a patient has TB and what type of resistance genes the specific strain contains to adapt drug treatment.
Tuberculosis (TB) remains a major global health problem. While the treatment of this disease in countries with adequate medical care is fairly easy to detect and treat, it remains hard to treat and diagnostic in poorer countries. Up to now, a high quality lab that uses modern diagnostics is a prerequisite for early, rapid and accurate detection of TB. Therefore, diagnosis of TB and drug resistant TB remains a particular challenge for laboratory systems, especially in developing countries.
The lack of cheap, quick and accurate tests make it hard to control the Tuberculosis epidemic, which claims millions of lives every year in developing countries.
Setting up a cheap, fast and culture-based method could therefore decrease diagnostic time and facilitate patient treatment.
The most common method for diagnosing TB nowadays is sputum smear microscopy, in which bacteria are observed in sputum samples of patients under the microscope. However, this cannot be used to identify paucibacillary (containing just a few bacteria) or extrapulmonary (outside of the lungs) TB.
Diagnosis methods using culture methods require laboratory infrastructure that is not widely available in countries with a high burden of TB and results are only available after a few weeks.
Other conventional methods used to diagnose multidrug-resistant TB (MDR-TB) also rely on the culturing of specimens followed by drug susceptibility testing (DST). Results take weeks to obtain and not all laboratories have the capacity to perform DST of first-line or of second-line drugs.
MTB/RIF is a new rapid molecular test that can diagnose TB and rifampicin-resistant TB within hours. Molecular tests, such as GeneXpert, unlike culture-based methods, are fast, accurate and can detect drug-resistant strains. But the high costs and need for laboratories make access an issue for developing countries.
The new method, published in the Journal of Applied Microbiology, uses a microcalorimeter to detect heat produced by Mycobacterium tuberculosis, the bacterium that causes TB, on a growth medium. The study showed that detection takes 4–5 days but more sensitive microcalorimeters could detect tuberculosis in 24 hours.
Aim
To test more specific for drug resistance in Mycobacterium tuberculosis, we develop a new type of sensor test. We develop a sensor to target antibiotic resistances in tuberculosis to test if a specific strain carries a certain antibiotic resistance. Our sensor system consists out of a phagemid with a CRISPR/Cas system and LacZ as a reporter under the control of a pREC promoter (the promoter of the RecA protein that is involved in the stress response of bacteria). If our CRISPR/Cas system can now bind to the target (antibiotic resistance gene), the Cas9 generates at this specific target site a double strand break, which then starts the expression of our reporter, as the promoter gets active at stress that results from double strand breaks. Because our system is on a phagemid, the sensor system will be spread all over the population, which will give a clear color output if the target has been detected. This means, depending on what target sequence our system carries, we can identify the different antibiotic resistances that a strain might carry. This is a novel way of detecting resistances in bacterial strains. As a proof of concept we will use E.coli to target KanR. The used phage will be M13. Our final sensor could potentially be used to test if a patient has TB and what type of resistance genes the specific strain contains to adapt drug treatment.
Background
CRISPRs
The mechanism we are using for our sensor is based on the CRISPR/Cas System. The CRISPR/Cas genome editing method, which recently became very popular, is derived from the bacterial “immune system”. CRISPRs are Clustered Regularly Interspaced Short Palindromic Repeats that are part of the bacterial genome. These loci contain multiple short repeats. In between the repeats are so called spacers that are sequences derived from extrachromosomal DNA, e.g. from invading viruses or plasmids. Within bacteria, those CRISPRs are naturally used to detect and destroy foreign DNA that is saved in the spacers.
In total the CRISPRs and the spacers form a so-called CRISPR array that is transcribed as one unit. CRISPR associated proteins (Cas proteins) are involved in further processing and action steps of the CRISPR system. There are many different regulation systems, here we describe the Cas9 system that will be used for our purposes.
The transcribed mRNA is processed into so called crRNA (CRISPR RNA), which includes the spacer sequence of foreign DNA. Together with a transactingCRISPR RNA (tracrRNA), the crRNA forms a duplex that is cleaved by RNaseII. The resulting hybrid serves as guide for the Cas9 protein that generates double strand breaks at the position the RNAs guide it to. By this double strand break the invading DNA is destroyed.
During the last years, researchers discovered the CRISPRs as a method for genome editing. By designing the spacer sequence, specific sequences in the genome can be targeted. With the provision of a sequence with homologous sequences, easy insertion can be done. This method can be used to insert mutations or new sequences nearly everywhere in the genome.
Recently many papers have been published for genome editing in bacteria (Jinek et al., 2012, Jiang et al., 2013), yeast (DiCarlo et al., 2013) and mammalian cells (Mali et al., 2013, Cong et al., 2013). They describe how to use the CRIPRs for different purposes. One requirement to target a sequence is a NGG at the end of the sequence. This NGG sequence is called PAM - protospacer adjacent motif, while the target sequence itself is called the protospacer, which should have a length of around 12 to 20 bp.
In our approach we test two different systems, the systems of DiCarlo et al. 2013 and Jiang et al. 2013. The system of Jiang is a system developed for bacteria and especially for E.coli. The system of DiCarlo is based on a paper of () and adapted for yeast.
Different from the Jiang paper, the DiCarlo paper uses a gRNA (guideRNA) to guide the Cas9. The gRNA is actually the RNA that results after transcribing and folding is the same complex as you get after the processing of the tracrRNA with the crRNA. It is hence an improvement, which makes the design and expression easier. But as the system hasn’t been tested before in bacteria, we will use both systems to make sure we get the desired results.
RecA promoter
Our reporter system consists out of the RecA promoter and LacZ. Originally, the RecA promoter of Escherichia coli has its main role in the activation of the SOS repair system. It is regulated by the LexA repressor, which binds to the SOS box sequence of the promoter. RecA expression is regulated in a way that the number RecA proteins is maintained between 1 000 and 10 000 monomers per cell. DNA damage leads to an inducing signal, which then activates the RecA protein. In the SOS response the LexA repressor is cleaved by the RecA protein, so the full RecA expression can be reached. The RecA protein can then repair both single stranded and double stranded DNA breaks.
In our project RecA promoteris used to detect double stranded DNA breaks in the region of antibiotic resistance genes caused by the CIRSPR/Cas system. To get the desired activation of the promoter, it is important to assure that no other stressing agents might activate the RecA promoter. Those agents could be UV light, X-ray, ionizing radiation and different types of DNA breaking compounds. The SOS response in E. coli could also get induced by engineered M13 phage infection that are defective in the minus-strand origin, and so unable to form the double stranded replicative stage.
In this case the single-stranded DNA would be the SOS-inducing signal. The wild type M13 phage doesn’t induce the SOS response (Higashitani et al., 1992). In another study genomes of phage M13 infected and uninfected E. coli strains were compared by oligonucleotide microarrays, where no stress response genes were scored as upregulated (Karlsson et al., 2005). Regarding the phagemids, an A UV-damaged oriC phagemid did not induce SOS response in a recipient cells oriF phagemids on the other hand did (Sommer et al. 1991). To make sure that our system functions reliably, we will test the activation of the RecA promoter at different stress inputs (UV, phage infection,…) by measuring the fluorescence of YFP driven by the RecA promoter.
LacZ
As a reporter we are using LacZ in our system. LacZ is the part of the lac operon in E.coli and its gene product in an enzyme, the β-galactosidase. This enzyme has 3 catalytic functions: to cleave lactose into glucose and galactose, catalyze the trangalacosylation of lactose to allactose and the cleavage of allactose into monosaccharides (Juers et al., 2012). X-gal (5-bromo-4-chloro-3-indoyl-b-D-galactopyranoside) contains a galactose fused to a substituted indole and is a very common used compound in Molecular Biology. It is colorless, soluble and non-toxic to cells. The β-galactosidase recognizes the galactose of the molecule and hydrolyzes the X-gal. In this process the indole is released which dimerizes. The dimerized indole is insoluble and has a blue color. Due to this color output it is often used to confirm cloning. Only small numbers of enzyme and X-gal are needed to actually get a color detection, which makes it very attractive for our system.
Cloning Strategy
For our approach we use Gibson assembly with Biobrick Cloning as a backup. As we want to test our systems separately as well as combined we need a modular cloning, which allows us to independently combine the different parts. Therefore we developed our parts with specific overhangs that allows us to assemble the parts combined or separately.
Therefore we designed 4 different overhangs followed by Biobrick cut sites that were added by PCR to our genes or synthesized together with the gene.
Gibson assembly is a fairly new cloning method that allows the assembly of several DNA sequences within one step. To be able to do so, the neighboring sequences need to have overlaps of 20-40bp. A T5 exonuclease then chews the DNA from the 5’ end, so that the resulting matching single stranded sequences can anneal. DNA polymerase then fills up nucleotide gaps and Taq ligase covalently joints the DNA fragments.
Literature
James E. DiCarlo, Julie E. Norville, Prashant Mali, Xavier Rios, John Aach and George M. Church (2013): Genome engineering in Saccharomyces cerevisiae using CRISPR-Cas systems. Nucleic Acids Research, 2013, 1–8.
Antonio R. FernaÂndez de Henestrosa, Tomoo Ogi, Sayura Aoyagi, David Chafin, Jeffrey J. Hayes, Haruo Ohmori and Roger Woodgate (2000): Identification of additional genes belonging to the LexA regulon in Escherichia coli. Molecular Microbiology (2000) 35(6), 1560±1572.
Higashitani N., Higashitani I.A., Roth A., Horiuchi A.K. (1992): SOS Induction in Escherichia coli by Infection with Mutant Filamentous Phage That Are Defective in Initiation of Complementary-Strand DNA Synthesis. Journal of Bacteriology 174 (5): 1612-1618.
Wenyan Jiang, David Bikard, David Cox, Feng Zhang & Luciano A Marraffini (2013):  RNA-guided editing of bacterial genomes using CRISPR-Cas systems. Nature Biotechnology 31 (3),  233-239.
Douglas H. Juers, Brian W. Matthews, and Reuben E. Huber (2012): LacZ b-galactosidase: Structure and function of an enzyme of historical and molecular biological importance. PROTEIN SCIENCE, 2012, 21,1792—1807.
Karlsson F., Malmborg-Hager A.C., Albrekt A.S., Borrebaeck C.A.K. (2005): Genome-wide comparison of phage M13-infected vs. uninfected Escherichia coli. Can. J. Microbiol. 51: 29–35.
Kimberly L. Keller, Terri L. Overbeck-Carrick, Doris J. Beck (2001): Survival and induction of SOS in Escherichia coli treated with cisplatin, UV irradiation, or mitomycin C are dependent on the function of the RecBC and RecFOR pathways of homologous recombination. Mutation Research 486, 2001, 21–29.
Kobayashi, Ichizo; and Handa, Naofumi (December 2009) DNA Doublestrand Breaks and Their Consequences in Bacteria. In: Encyclopedia of Life Sciences (ELS). John Wiley & Sons, Ltd: Chichester. Kobayashi, Ichizo; and Handa, Naofumi (December 2009) DNA Doublestrand Breaks and Their Consequences in Bacteria. In: Encyclopedia of Life Sciences (ELS). John Wiley & Sons, Ltd: Chichester.
Anders Norman, Lars Hestbjerg Hansen, and Søren J. Sørensen (2004): Construction of a ColD cda Promoter-Based SOS-Green Fluorescent Protein Whole-Cell Biosensor with Higher Sensitivity toward Genotoxic Compounds than Constructs Based on recA, umuDC, or sulA Promoters. APPLIED AND ENVIRONMENTAL MICROBIOLOGY, May 2005, p. 2338–2346.
Sommer S., Leitao A., Bernardi A., Bailone A., Devoret R. (1991): Introduction of a UV-damaged replicon into a recipient cell is not a sufficient condition to produce an SOS-inducing signal. 254(2):107-17.
Producer bacteria
Phagemid is packed into the capsid, produced by the helper plasmid
Phagmids are isolated
Phagmids are mixed with target bacteria
Right strain? Yes, Phagemids enter the cell
Expression of CrRNA, tracerRNA and Cas9 Processing of CrRNA + tracerRNA
Fusion of Cas9 and hybrid RNA complex
Antibiotic resistance of interest there?
Yes : Cas9 - RNA complex bind to target sequence
Cas9 generates double strand break
pRec-SOS response activated
LacZ is expressed
ß-galactosidase hydrolisis Xgal : blue cells appear
Reporter (pRecA+LacZ): with overhang 1/2
Cas 1 (tracrRNA+Cas9): with overhang 3/2, 3/1
Cas 2 (Cas9): with overhang 3/1, 3/2
crRNA: with overhang 2/4
gRNA: with overhang 2/4
backbone pSB1A3 (AmpR): with overhang 3/2, 1/2
backbone pSB1C3 (ChlR): with overhang 3/4, 1/3, 2/3
Centre for Research and Interdisciplinarity (CRI) Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor Paris Descartes University 24, rue du Faubourg Saint Jacques 75014 Paris, France
+33 1 44 41 25 22/25
team2013@igem-paris.org
Copyright (c) 2013 igem.org. All rights reserved.
Retrieved from " http://2013.igem.org/Team:Paris_Bettencourt/Project/Phage_Sensor "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
