"
Page
Discussion
View source
History
teams
Log in
 
Team:Paris Bettencourt/Results
From 2013.igem.org
PROJECT
OVERVIEW
DETECT
TARGET
INFILTRATE
SABOTAGE
ACHIEVEMENTS
PARTS
RESULTS
MEDAL FULFILLMENT
HUMAN PRACTICE
OVERVIEW
GENDER STUDY
TECHNOLOGY TRANSFER
TB FACTS
TB IN FRANCE
TB GALLERY
SAFETY
COLLABORATION
COLLABORATION
SENSIGEM
NOTEBOOK
DAY NOTES
PROTOCOLS
BIBLIOGRAPHY
TEAM
OUR TEAM
GALLERY
ATTRIBUTIONS
ACKNOWLEDGEMENTS
@iGEM_Paris
Detect
Target
Infiltrate
Sabotage
Modelling
Human Practice
Detect
Background
CRISPR/Cas systems generate site-specific double strand breaks and have recently been used for genome editing.
Aims
Building a genotype sensor based on CRISPR/Cas that reports existance of an antibiotic resistance gene.
Results
Successfully cloned gRNA anti-KAN, crRNA anti-KAN, tracrRNA-Cas9 and pRecA-LacZ into Biobrick backbones and therefore generated four new BioBricks.
Confirmation of site-specific binding and DNA double-strand breaks generated by the gRNA-Cas9 complex in the kanamycin resistance casette.
Testing the new assembly standard for our cloning.
CRISPR anti-Kan plasmids target specifically kanamycin resistant E. coli. We introduced our CRISPR-based DNA cleavage system to two strains of E. coli : one WT (blue bars) and one carrying a genomically integrated kanamycin resistance casette (KanR, blue bars). The strains were co-transformed with two plasmids. One with the Cas9 construct, the other with the anti-Kanamycin gRNA. WT E. coli could be efficiently transformed with one or both plasmids, as determined by selective plating. However, KanR E. coli could not be efficiently transformed with both. We attribute this to Cas9-induced cleavage of the chromosome specifically at the KanR casette, with about 99% efficiency.
Target
Background
SirA is an essential gene in latent tuberculosis infections.
Aims
To perform a drug screen targeted at the sirA gene from mycobacteria.
Results
Produced an E. coli strain which relies upon mycobacterial sirA, fprA and fdxA genes to survive in M9 minimal media.
Demonstrated that E. coli can survive with mycobacterial sulfite reduction pathway with Flux Balance Analysis.
Located drug target sites on sirA as well as identified high structural similarity between cysI and sirA through structural analysis.
Identified a potential anti-TB activity of Pyridoxine at high doses.
MycoSIR E. coli depend on our synthetic pathway for growth. E. coli strain BL21(DE3) was deleted for cysI and transformed with the three genes of the mycoSIR pathway expressed from IPTG-inducible T7 promoters (red). Wild-type (blue), uninduced (purple) and pathway-minus (gold) strains were used as controls. Both time course growth curves (A) and final ODs (B) reveal that the complete, induced pathway is required for growth
Infiltrate
Background
Latent tuberculosis persists inside macrophages of the lungs, where it is partially protected from both the host immune system and conventional antibiotics.
Aims
To create an E. coli strain capable of entering the macrophage cytosol and delivering a lytic enzyme to kill mycobacteria.
Results
We expressed the enzyme Trehalose Dimycolate Hydrolase (TDMH) in E.coli and showed that it is highly toxic to mycobacteria in culture.
We expressed the listeriolyin O (LLO) gene in E. coli and showed that it is capable of entering the macrophage cytosol.
We co-infected macrophages with both mycobacteria and our engineered E. coli to characterize the resulting phagocytosis and killing.
TDMH expression kills mycobacteria in culture. We mixed E. coli and WT M.smegmatis in LB media. Plating assays were used to count specifically M.smegmatis after the indicated times. When TDMH-expression was fully induced, more than 98% of mycobacteria were killed after 6 hours (red line). Populations of mycobacteria alone (black line) and mycobacteria mixed with uninduced E. coli (blue line) were stable.
Sabotage
Background
One of the main concerns about tuberculosis today is the emergence of antibiotic resistant strains.
Aims
Our objective is to make an antibiotic-resistant bacterial population sensitive again to the selfsame antibiotics.
Results
Construction and characterization of phagemids coding for small RNA targeting antibiotic resistance proteins.
Showed theoretically burden of a device is critical for the maintenance of a genetic element in a population.
Successful conversion of antibiotic resistant population of E. coli to a sensitive state (on two different antibiotic resistances).
Our synthetic phage conveys antibiotic-sensitivity to an antbiotic-resistant population. The anti-Chloramphenicol phage system effectively killed 99.1% of the Chloramphenicol resistant population and the anti- Kanamycine phage effictevely killed 99,5% of the Kanamycine resistant population.
Modelling
Population Dynamics Model
This model investigates the propagation of horizontal gene-transfer via phagemid/helper system. We study the effect of the burden of a desired device on the maintenance of the system in time.
Left: scheme representing the regular non-lytic M13 bacteriophage horizontal spread. Right: scheme representing the main processes of the phagemid/helper system.
Structure Based Modelling
SirA: Using Swiss pdb we demonstrated the superimposed 3D structures of Mycobacterium tuberculosis SirA and Escherichia coli CysI highlighting their similarities and differences. Both proteins are important in their respective sulphite reduction pathways. We then predicted the effect of a small drug target based on SirA's structure.
FprA: Using LigandScout 3.1, we searched over 8100 drug compounds from the BindingDB and Chembl databases for drugs targetting mycobacterial Ferredoxin NADP reductase (FprA). We have modelled Pyridoxine's interaction to its active site.
Flux Balance Analysis
We used an E. coli model iJR904 obtained from BiGG database as a starting model and obtained a growth rate represented by the f value of 0.9129. We then deleted the reaction which encodes for the sulphite reduction pathway and obtained a f value of -8.63e-13 indicating that the sulphite reduction pathway is essential. We wrote a program which finds all essential reactions in M. tuberculosis and E. coli SBML models and attempts to restore growth for each essential E. coli model with essential reactions from M. tuberculosis to identify other metabolic pathways we could apply a targeted drug screen to.  We identified 100 other essential reactions we can target.
Human Practice
Technology  Transfer
An essay that addresses the issue of designing a technology aimed at "developing" countries, rather than at “developed” ones: a typical case of technology transfer.
TB in France
Analysis of the social, medical and political aspects of the management of tuberculosis in France. Synthetic biology may help in many ways such as treatments, drug development, diagnostic. We also give advice on how to introduce it in clinics.
Gender Study
A comprehensive and quantitative study of gender (in)equality in iGEM and synthetic biology. A database was gathered and statistically analysed in order to depict sex ratio of iGEM teams' students and supervisors.
Gender balance and success in iGEM
Winning teams have a significantly higher number of women and are more Gender balanced.
TB Facts
Infographics page containing TB data and facts that captures all you need to know at a glance.
TB Gallery
A gallery of famous historic figures who had tuberculosis, made to raise awareness to its prevalence of in the past and present .
Collaboration
SensiGEM
SensiGEM is the iGEM Biosensor database generated by the teams Paris Bettencourt 2013 and Calgary 2013. In this database you can find fast and easy what biosensor projects were already done by past iGEM Teams. To be able to select the projects that fit into the database, we also collaborated to compose a joint definition a biosensor.
BGU iGEM Team from Israel
A mutual part characterization.  We characterize the promoter units produced by the lac/ara-1 promoter of cI, a repressor of their constructed kill switch. In return, BGU characterizes our TDMH biobrick protein expression levels by Western Blot.
Braunschweig iGEM Team
Idea, bibliography, and beer sharing!
Centre for Research and Interdisciplinarity (CRI) Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor Paris Descartes University 24, rue du Faubourg Saint Jacques 75014 Paris, France
+33 1 44 41 25 22/25
team2013@igem-paris.org
Copyright (c) 2013 igem.org. All rights reserved.
Retrieved from " http://2013.igem.org/Team:Paris_Bettencourt/Results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
