<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Paris_Bettencourt/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris_Bettencourt/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Protocols&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris_Bettencourt/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris Bettencourt/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="navmenubkg"><DIV id="navmenubox"><UL id="nav"><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Overview" title="Project Overview">PROJECT</A></LI><LI style="margin-left:14.5px"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Overview" title="Project Overview">OVERVIEW</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" title="Detect">DETECT</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" title="Target">TARGET</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" title="Infiltrate">INFILTRATE</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" title="Sabotage">SABOTAGE</A></LI></UL></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Results" title="Achievements">ACHIEVEMENTS</A></LI><LI style="margin-left:141px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Parts" title="Submitted Parts">PARTS</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Results" title="Results">RESULTS</A></LI><LI style="width:167px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Medal_Fulfillment" title="Medal Fulfillment">MEDAL FULFILLMENT</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Overview" title="Human Practice Overview">HUMAN PRACTICE</A></LI><LI style="margin-left:141px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Overview" title="Human Practice Overview">OVERVIEW</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Gender_Study" title="Gender Study">GENDER STUDY</A></LI><LI style="width:168px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Technology_Transfer" title="Technology_Transfer">TECHNOLOGY TRANSFER</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/TB_Facts" title="TB Tacts">TB FACTS</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/TB_France" title="TB in France">TB IN FRANCE</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/TB_Gallery" title="TB Gallery">TB GALLERY</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Safety" title="Safety">SAFETY</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Collaboration" title="Collaboration">COLLABORATION</A></LI><LI style="margin-left:500px"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Collaboration" title="Collaboration">COLLABORATION</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/SensiGEM" title="SensiGEM">SENSIGEM</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Notes" title="Notebook">NOTEBOOK</A></LI><LI style="margin-left:649.5px"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Notebook" title="Day Notes">DAY NOTES</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Protocols" title="Protocols">PROTOCOLS</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Bibliography" title="Bibliography">BIBLIOGRAPHY</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Team" title="Team">TEAM</A></LI><LI style="margin-left:522.5px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Team" title="Team">OUR TEAM</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Gallery" title="Team Gallery">GALLERY</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Attributions" title="Attributions">ATTRIBUTIONS</A></LI><LI style="width:167px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Acknowledgements" title="Acknowledgements">ACKNOWLEDGEMENTS</A></LI><DIV id="twnoticon" class="sidelogos"><A class="twitter-timeline" width="250" height="400" data-dnt="true" href="https://twitter.com/iGEM_Paris" data-widget-id="368144501119795201">
            @iGEM_Paris</A></DIV><DIV id="page"><H1 style="margin-top:20px;margin-bottom:5px">Menu</H1><DIV class="leftside"><UL><LI><A onclick="showprot('#prot1')">Heat Shock Transformation</A></LI><LI><A onclick="showprot('#prot2')">CaCl2 Competent Cells</A></LI><LI><A onclick="showprot('#prot3')">Glycerol Stocks</A></LI><LI><A onclick="showprot('#prot4')">Electroporation</A></LI><LI><A onclick="showprot('#prot5')">Miniprep</A></LI><LI><A onclick="showprot('#prot6')">PCR Purification</A></LI><LI><A onclick="showprot('#prot7')">Gel Purification</A></LI><LI><A onclick="showprot('#prot8')">P1 Transduction</A></LI><LI><A onclick="showprot('#prot9')">Colony PCR</A></LI><LI><A onclick="showprot('#prot10')">Preparation of LB liquid broth</A></LI><LI><A onclick="showprot('#prot11')">Preparation of LB Agar</A></LI><LI><A onclick="showprot('#prot12')">Gel Electrophoresis</A></LI><LI><A onclick="showprot('#prot13')">Making LB Agar plates</A></LI><LI><A onclick="showprot('#prot14')">J774 macrophages cell culture</A></LI></UL></DIV><DIV class="rightside"><DIV id="prot1" class="protocolactive"><H2> Heat Shock Transformation of <I>E. coli</I></H2><P>This protocol can be used to transform chemically competent (i.e. from CaCl2) with a miniprepped plasmid or a ligation product.</P><H5>Note: Never vortex competent cells. Mix cells by gentle shaking.</H5><P><OL><LI>Thaw competent cells on ice. These can be prepared using the CaCl2 protocol.</LI><LI>Place 20 ul of cells in a pre-chilled Eppendorf tube.
	    </LI><LI><U>For an Intact Vector:</U> Add 0.5 ul or less to the chilled cells</LI><LI><U>For a Ligation Product:</U> Add 2-3 ul to the chilled cells.</LI><LI>Mix gently by flicking the tube.</LI><LI>Chill on ice for 10 minutes. <EM>This step is optional, but can improve yields when transforming a ligation product.</EM></LI><LI>Heat shock at 42 °C for 30 seconds.</LI><LI>Return to ice for 2 minutes.</LI><LI>Add 200 ul LB medium and recover the cells by shaking at 37 °C.
	    Another rich medium can substitute for the recovery.
	    The recovery time varies with the antibiotic selection.
	    Ampicillin: 15-30 minutes
	    Kanamycin or Spectinomycin: 30-60 minutes
	    Chloramphenicol: 60-120 minutes 
	  </LI><LI>Plate out the cells on selective LB.
	    Use glass beads to spread the cells.
	    The volume of cells plated depends on what is being transformed.</LI><LI><U>For an Intact Vector:</U> High transformation efficiencies are expected. Plating out 10 ul of recovered cells should produce many colonies.</LI><LI><U>For a Ligation Product:</U> Lower transformation efficiencies are expected. Therefore you can plate the entire 200 ul volume of recovered cells.</LI>
	    Note: 200 ul is the maximum volume of liquid that an LB plate can absorb.
	  <LI>Incubate at 37 °C. Transformants should appear within 12 hrs.</LI></OL></P></DIV><DIV id="prot2" class="protocol"><H2> CaCl2 Competent Cells </H2><P>This protocol makes 4 ml of competent cells, and can be easily scaled up to make more. The cells are typically stored in 110 ul aliquots, so this will make about 35 tubes. A typical transformation uses 20 ul of cells.</P><H5> Note: Never vortex competent cells. Resuspend by pipetting with large Pasteur pipettes.</H5><P><OL><LI><B><U>The night before</U></B>, inoculate a 5 ml culture and grow overnight with selection.</LI><LI><B><U>The day of</U></B> the experiment dilute cells ~ 1:200 into selective media.For this example add 250 ul to 50 ml of selective media.Note: The protocol is easily scaled to increase the number of cells.</LI><LI> Grow the cells to an OD600 of 0.6 – 0.7.
	    Use a large flask, 500ml, for good aeration.
	    Use a baffled flask for fastest growth.
	    This takes about 3 hours depending on the cells.
	    Medium-heavy cloudiness by eye is fine.</LI><LI>Spin down the cells at 4 ºC, 4000 rpm, 15 minutes.
	    Note: Keep the cells at 4 ºC from now on.</LI><LI>Resuspend cells in 15 ml, ice-cold 100 mM CaCl2. 
	    Leave on ice 4 hours to overnight.</LI><LI>Spin down the cells at 4 ºC, 4000 rpm, 15 minutes.</LI><LI>Resuspend cells in 4 ml, ice-cold 100 mM CaCl2 + 15% glycerol.</LI><LI>Aliquot into pre-chilled Eppendorf tubes. Use immediately or store at -80ºC.
	    Note: Frozen cells are only good once.Do not refreeze cells once thawed.</LI></OL></P></DIV><DIV id="prot3" class="protocol"><H2>Glycerol Stocks </H2><P><OL><LI>Pick Single colonies from agar plates
	  </LI><LI>Innoculate 5ml LB broth overnight.
	  </LI><LI>Add 750ml of overnight culture to 250ml of 60% glycerol in a cryotube.
	  </LI><LI>Make two sets of Glycerol stocks freeze one at -20ºC and the other at -80ºC.
      </LI></OL></P></DIV><DIV id="prot4" class="protocol"><H2> Electroporation </H2><H3>Preparation of Electrocompetent Cells</H3><P>
	Competent cells should never be vortexted, as this will cause them to lyse
	and release salts into the media. Resuspend cells by pipeting up and down with a large
	pasteur pipet. Once they are chilled, cells should be continuously cold.
	<OL><LI>The night before the transformation, start an overnight culture of cells.
	    5 ml LB Amp.
	  </LI><LI>The day of the transformation, dilute the cells 100X.
	    100 ml LB Amp.
	    Grow at 30°C for about 90 minutes.
	  </LI><LI>Harvest the cells.
	    When the cells reach an OD600 of between 0.6 and 0.8.
	    Split the culture into 2x 50 ml falcon tubes, on ice.
	    Centrifuge at 4 °C for 10 min at 4000 rpm.
	  </LI><LI>Wash and combine the cells.
	    Remove the supernatant.
	    Resuspend the cells in 2x 25 ml of ice cold water.
	    Combine the volumes in a single 50 ml falcon tube.
	  </LI><LI>Wash the cells 2 more times.
	    Centrifuge at 4 °C for 10 min at 4000 rpm.
	    Resuspend in 50 ml of ice cold water.
	    Repeat.
	  </LI><LI>Wash and concentrate the cells for electroporation.
	    Centrifuge at 4 °C for 10 min at 4000 rpm.
	    Resuspend in 1-2 ml of ice cold water.
	    We will use 200 ul of washed cells per transformation.
	  </LI></OL></P><H3>Dialysis of PCR or Digestion Products</H3><P>
	DNA for electroporation must be free of salts to avoid arcing.
	<OL><LI>Float a filter in a Petri dish filled with water.
	    Millipore membrane filter 0.025 uM.</LI><LI>Pipet one drop of PCR product onto the filter.
	    200 ng is needed per transformation.
	    20 - 100 ul fits well on one filter.</LI><LI>Collect the drop after 30 - 45 minutes.
	    The volume will change, but the DNA is not lost.
	  </LI></OL></P></DIV><DIV id="prot5" class="protocol"><H2> Miniprep </H2><H3>Miniprep using <I>Thermo Scientific GeneJET Plasmid Miniprep Kit</I></H3><P>All centrifugations should be carried out in a table-top microcentrifuge at sup12000 x g
	<OL><LI>Resuspend the pelleted cells in 250 ul of the Resuspension Solution. Transfer the cell suspension to a microcentrifuge tube. The bacteria should be resuspended completely by vortexing or pipetting up and down until no cell clumps remain.<B><I>Note</I></B>. Ensure RNase A has been added to the Resuspension Solution.
	  </LI><LI>Add 250 ul of the Lysis Solution and mix thoroughly by inverting the tube 4-6 times until the solution becomes viscous and slightly clear.<B><I>Note</I></B>. Do not vortex to avoid shearing of chromosomal DNA. Do not incubate for more than 5 min to avoid denaturation of supercoiled plasmid DNA.</LI><LI>Add 350 ul of the Neutralization Solution and mix immediately and thoroughly by inverting the tube 4-6 times.<B><I>Note</I></B>.
	    It is important to mix thoroughly and gently after the addition of the Neutralization Solution to avoid localized precipitation of bacterial cell debris. The neutralized bacterial lysate should become cloudy.
	  </LI><LI>Centrifuge for 5 min to pellet cell debris and chromosomal DNA. 
	  </LI><LI>Transfer the supernatant to the supplied GeneJET spin column by decanting or pipetting. Avoid disturbing or transferring the white precipitate. 
	  </LI><LI>Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube.<B><I>Note</I></B>.Do not add bleach to the flow-through.
	  </LI><LI>Add 500 ul of the Wash Solution (diluted with ethanol) to the GeneJET spin column. Centrifuge for 30-60 seconds and discard the flow-through. Place the column back into the same collection tube. 
	  </LI><LI>Repeat the wash procedure (step 7) using 500 ul of the Wash Solution. 
	  </LI><LI>Discard the flow-through and centrifuge for an additional 1 min to remove residual Wash Solution. This step is essential to avoid residual ethanol in plasmid preps. 
	  </LI><LI>Transfer the GeneJETspin column into a fresh 1.5 ml microcentrifuge tube. Add 50 ul of the Elution Bufferto the center of GeneJET spin column membrane to elute the plasmid DNA. Take care not to contact the membrane with the pipette tip. Incubate for 2 min at room tempera ture and centrifuge for 2 min.<B><I>Note</I></B>.
	    An additional elution step (<I>optional</I>) with Elution Buffer or water will recover residual DNA from the membrane and increase the overall yield by 10-20%. For elution of plasmids or cosmids sup20 kb, prewarm Elution Buffer to 70°C before applying to silica membrane.
	  </LI><LI>Discard the column and store the purified plasmid DNA at -20°C. 
	  </LI></OL></P></DIV><DIV id="prot6" class="protocol"><H2>PCR Purification </H2><H3>PCR purification using <I>Thermo Scientific GeneJET PCR Purification Kit</I></H3><P>All centrifugations should be carried out in a table-top microcentrifuge at sup12000 x g
	<OL><LI>Add a 1:1 volume of Binding Buffer to completed PCR mixture (e.g. for every 100 uL of reaction mixture, add 100 uL of Binding Buffer). Mix thoroughly. Check the color of the solution. A yellow color indicates an optimal pH for DNA binding. If the color of the solution is orange or violet, add 10 uL of 3 M sodium acetate, pH 5.2 solution and mix. The color of the mix will become yellow.
	  </LI><LI><I>Optional</I>: if the DNA fragment is inf 500 bp, add a 1:2 volume of 100% isopropanol(e.g., 100 uL of isopropanol should be added to 100 uL of PCR mixture combined with 100 uL of Binding Buffer). Mix thoroughly.<B><I>Note</I></B>. If PCR mixture contains primer-dimers, purification without isopropanol is recommended. However, the yield of the target DNA fragment will be lower.
	  </LI><LI>Transfer up to 800 uL of the solution from step 1(or optional step 2)to the GeneJET
	    purification column. Centrifuge for 30-60 s. Discard the flow-through.<B><I>Note</I></B>. If the total volume exceeds 800 uL, the solution can be added to the column in stages. After the addition of 800 uL of solution, centrifuge the column for 30-60 s and discard flow-through. Repeat until the entire solution has been added to the column membrane.
	  </LI><LI>Add 700 uL of Wash Buffer to the GeneJET purification column. Centrifuge for 30-60 s. Discard the flow-through and place the purification column back into the collection tube.
	  </LI><LI>Centrifuge the empty GeneJET purification column for an additional 1 min to completely remove any residual wash buffer.<B><I>Note</I></B>.This step is essential as the presence of residual ethanol in the DNA sample may inhibit subsequent reactions
	  </LI><LI>Transfer the GeneJET purification column to a clean 1.5 mL microcentrifuge tube (not included).Add 50 uL of Elution Buffer to the center of the GeneJET purification column membrane and centrifuge for 1 min.<B><I>Note</I></B>. For low DNA amounts the elution volumes can be reduced to increase DNA concentration. An elution volume between 20-50 uL does not significantly reduce the DNA yield. However, elution volumes less than 10 uL are not recommended. If DNA fragment is inf 10 kb, prewarm Elution Buffer to 65°C before applying to column. If the elution volume is 10 uL and DNA amount is inf 5 ug, incubate column for 1 min at room temperature before centrifugation.
	  </LI><LI>Discard the GeneJET purification column and store the purified DNA at -20°C. 
	</LI></OL></P></DIV><DIV id="prot7" class="protocol"><H2>Gel Purification </H2><H3>Gel purification using <I>Thermo Scientific GeneJET Gel Extraction Kit</I></H3><P>
	All centrifugations should be carried out in a table-top microcentrifuge at sup12000 x g<OL><LI>Excise gel slice containing the DNA fragment using a clean scalpel or razor blade. Cut as close to the DNA as possible to minimize the gel volume. Place the gel slice into a pre-weighed 1.5 ml tube and weigh. Record the weight of the gel slice.<B><I>Note</I></B>.
	    If the purified fragment will be used for cloning reactions, avoid damaging the DNA through UV light exposure. Minimize UV exposure to a few seconds or keep the gel slice on a glass or plastic plate during UV illumination.
	  </LI><LI>Add 1:1 volume of Binding Buffer to the gel slice (volume: weight)(e.g., add 100 ul of Binding Buffer for every 100 mg of agarose gel).
	    <B><I>Note</I></B>.
	    For gels with an agarose content greater than 2%, a dd 2:1 volumes of Binding Buffer to the gel slice.
	  </LI><LI>Incubate the gel mixture at 50-60°C for 10 min or until the gel slice is completely dissolved. Mix the tube by inversion every few minutes to facilitate the melting process. Ensure that the gel is completely dissolved. Vortex the gel mixture briefly before loading on the column. Check the color of the solution. A yellow color indicates an optimal pH for DNA binding. If the color of the solution is orange or violet, add 10 ul of 3 M sodium acetate, pH 5.2 solution and mix. The color of the mix will become yellow.
	  </LI><LI><I>Optional</I>: use this step only when DNA fragment is inf 500 bp or sup10 kb long. If the DNA fragment is inf 500 bp, add a 1:2 volume of 100% isopropanol to the so lubilized gel solution (e.g. 100 ul of isopropanol should be added to 100 mg gel slice solubilized in 100 ul of Binding Buffer). Mix thoroughly. If the DNA fragment is sup10 kb , add a 1:2 volume of water to the solubilized gel solution (e.g. 100 ul of water should be added to 100 mg gel slice solubilized in 100 ul of Binding Buffer). Mix thoroughly. 
	  </LI><LI>Transfer up to 800 ul of the solubilized gel solution (from step 3 or 4) to the GeneJET purification column. Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube.<B><I>Note</I></B>. If the total volume exceeds 800 ul, the solution can be added to the column in stages. After each application, centrifuge the column for 30-60 s and discard the flow-through aftereach spin. Repeat until the entire volume has been applied to the column membrane. Do not exceed 1 g of total agarose gel per column.
	  </LI><LI><I>Optional</I>: use this additional binding step only if the purified DNA will be used for sequencing. Add 100 ul of Binding Buffer to the GeneJET purification column. Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube.
	  </LI><LI>Centrifuge the empty GeneJET purification column for an additional 1 min to completely remove residual wash buffer.<B><I>Note</I></B>. This step is essential to avoid residual ethanol in the purified DNA solution. The presence of ethanol in the DNA sample may inhibit downstream enzymatic reactions.
	  </LI><LI>Transfer the GeneJET purification column into a clean 1.5 ml microcentrifuge tube (not included). Add 50 ul of Elution Buffer to the center of the purification column membrane. Centrifuge for 1 min.<B><I>Note</I></B>. For low DNA amounts the elution volumes can be reduced to increase DNA concentration. An elution volume between 20-50 ul does not significantly reduce the DNA yield. However, elution volumes less than 10 ul are not recommended. If DNA fragment is sup10 kb, prewarm Elution Buffer to 65°C before applying to column. If the elution volume is 10 ul and DNA amount is inf5 ug, incubate column for 1 min at room temperature before centrifugation.
	  </LI><LI>Discard the GeneJET purification column and store the purified DNA at -20°C.
	  </LI></OL></P></DIV><DIV id="prot8" class="protocol"><H2> P1 Transduction</H2><P>
	This protocol uses a phage to transfer a marker from a donor strain to a recipient strain. The phage head packages about 90 kb of DNA, so donor DNA near
	the marker is also transferred. Note that this can cause problems if you are working with several markers that are very close together.
      </P><H3>Lysate preparation</H3><P>Note: P1 phage should be stored at 4 C. It can't be frozen.<OL><LI>The night before, start a 5 mL culture of the donor strain in selective LB.</LI><LI>The day of, dilute the donor strain 1:100 into Phage Lysis medium.
	    </LI><LI>50 ul of cells in 5 mL LB</LI><LI>50 uL of 20% Glucose</LI><LI>50 uL of 1M MgSO4</LI><LI>25 ul of 1M CaCl2</LI><LI>No antibiotics</LI><LI>Incubate at 37 C for 1 hour.</LI><LI>Add 50 ul of P1 phage lysate
	    Monitor the culture for 1-3 hours.
	    The culture should become cloudy, then clear following lysis.
	  </LI><LI>Add 500 ul of chloroform to the lysate and vortex.</LI><LI>Centrifuge at max speed for 1 minute to clear the cell debris.
	    Collect the supernatant.
	  </LI><LI>Phage lysate can be stored indefinitely at 4 C. Freezing will destroy the phage.</LI></OL></P><H3>Transduction</H3><P><OL><LI>The night before, start a 5 ml culture of the recipient strain in selective LB.</LI><LI>The day of, harvest the cells by centrifugation.
	    6000 rpm for 2 min.
	  </LI><LI>Resuspend in original culture volume in 5 mL Phage Infection LB.</LI><LI>5 mL LB</LI><LI>50 uL of 1M MgSO4</LI><LI>25 ul of 1M CaCl2</LI><LI>Transfer 100 uL of donor P1 lysate per transformation to a 1.5 mL tube.
	    Incubate at 37 C for 30 minutes.
	    This allows the residual chloroform to evaporate.
	  </LI><LI>Set up 4 reactions for each transduction
	    </LI><LI>100 uL Donor Lysate</LI><LI>10 uL Donor Lysate</LI><LI>100 uL Donor Lysate</LI><LI>100 uL Plain Lb</LI></OL>
	    100 uL Recipient Cells 190 uL Recipient Cells 100 uL Plain LB 100 uL Recipient Cells
	  <LI>Incubate at 37 C for 30 minutes.</LI><LI>Stop the infection with 200 uL of 1 M Sodium Citrate (pH 5.5).</LI><LI>Add 1 mL LB and recover the cells for 1-2 hours.</LI><LI>Spin the cells down and resuspend for plating.
	    100 ul LB + 10 uL of 1 M Sodium Citrate (pH 5.5)
	  </LI><LI>Plate on selective LB.</LI></P></DIV><DIV id="prot9" class="protocol"><H2>Colony PCR</H2><P><OL><LI>Pick a single colony into 30ul of nuclease-free H20. (Fresh colonies grown that day work best, but they can also come from 4 C).</LI><LI>Boil for 10 minutes at 100C.</LI><LI>Centrifuge (find G) for 1 min. 1ul of this can be used directly for PCR. Best if used directly, but can also be stored at 4C for a few days.</LI></OL></P><H3>PCR Reaction</H3><P>Keep all the reagents at 4C while preparing the mixture. Pre-heat the thermocycler to 95C and transfer your reaction directly from 4 C.<CENTER><TABLE id="retable" class="tftable" border="1"><TBODY><TR><TH>Reagent</TH><TH>Volume (ul)</TH></TR><TR><TD>Forward Primer</TD><TD>1.0</TD></TR><TR><TD>Reverse Primer</TD><TD>1.0</TD></TR><TR><TD>Template DNA</TD><TD>2.0</TD></TR><TR><TD>Quick-Load Taq 2x Master Mix</TD><TD>10</TD></TR><TR><TD>Nuclease-free water</TD><TD>6</TD></TR><TR><TD>Total Volume</TD><TD>20</TD></TR></TBODY></TABLE></CENTER></P><DIV style="float:right;font-size:10px;margin-right:5%;">Created with the <A href="http://www.textfixer.com/html/html-table-generator.php" target="_blank">HTML Table Generator</A></DIV><H3> Thermocycler Protocol: NEB Quick-Load</H3><CENTER><TABLE id="retable" class="tftable" border="1"><TBODY><TR><TH>Temperature (°C)</TH><TH>Time</TH></TR><TR><TD>Start</TD><TD>95</TD><TD>30 s</TD><TD>Melt</TD></TR><TR><TD>Cycle 1</TD><TD>95</TD><TD>15 s</TD><TD>Melt</TD><TD>30 Cycles</TD></TR><TR><TD>Cycle 2</TD><TD>60</TD><TD>30 s</TD><TD>Anneal</TD><TD>30 Cycles</TD></TR><TR><TD>Cycle 3</TD><TD>72</TD><TD>1 min/kb</TD><TD>Extend</TD><TD>30 Cycles</TD></TR><TR><TD>Finish</TD><TD>72</TD><TD>5 min</TD><TD>Extend</TD></TR><TR><TD>Store</TD><TD>10</TD><TD>Forever</TD><TD>Store</TD></TR></TBODY></TABLE></CENTER></DIV><DIV id="prot10" class="protocol"><H2>Preparation of LB liquid broth</H2><P>LB is our standard rich media. Directions are on the bottle.<OL><LI>Add 12.5g of LB broth powder to a 1L glass bottle.</LI><LI>Add 500ml of osmotic water.</LI><LI>Autoclave</LI></OL></P></DIV><DIV id="prot11" class="protocol"><H2>Preparation of LB Agar</H2><P>LB agar is our standard rich media for plating. Directions are taken from the bottle.
	<OL><LI>Add 20 g of LB agar powder to a 1 L glass bottle</LI><LI>Add 500 ml of osmotic water.</LI><LI>Autoclave</LI></OL></P></DIV><DIV id="prot12" class="protocol"><H2>Making LB Agar plates</H2><P><OL><LI>Melt LBA in microwave (~8 min).</LI><LI>Don’t burn yourself. Most Antibiotics are thermo sensitive so allow agar to cool to approximately 56º C before adding antibiotics.</LI><LI>Add antibiotics if required</LI><LI>Add 15-25 ml of LBA into each plate. You can get more plates with less agar in each plate, but the plates will desiccate faster and cannot be stored as
	    long.</LI><LI>Allow the plates to set.</LI><LI>Dry them 45' under the flow hood, with the lid open.</LI></OL></P></DIV><DIV id="prot13" class="protocol"><H2>Gel Electrophoresis</H2><H3>Standard 1% agarose gel</H3><P>Agarose gels are commonly used in concentrations between 0.7 to 2% depending on the size of bands needed to be separated. Simply adjust the amount of
        starting agarose to %g/100mL TAE e.g. 2g/100mL will give 2%.
	<OL><LI>Measure out 1g of agarose</LI><LI>Pour agarose powder into a microwavable flask along with 100mL of 1xTAE</LI><LI>Microwave for 1-3mins (until the agarose has dissolved completely and there is a nice rolling boil). Caution HOT! Be careful stirring, eruptive
        boiling can occur.</LI><LI>Let agarose solution cool down for 5min.</LI><LI>Pour the agarose into a gel tray with the suitable well comb in place (pour slowly to avoid bubbles which will disrupt the gel).</LI><LI>Place newly poured gel at 4°C for 10-15 minutes or let sit at room temperature for 20-30 minutes, until the gel has completely solidified.</LI><LI>Once solidified, remove the comb and place the gel into the electrophoresis unit (gel box).</LI><LI>Fill the gel box with 1xTAE buffer.</LI><LI>Carefully load GeneRuler 100bp Plus DNA weight ladder into the first lane of the gel.</LI><LI>Carefully load your samples into the additional wells of the gel.</LI><LI>Run the gel at 50-150V until the dye line is approximately 75-80% of the way down the gel.</LI><LI>Turn off the power, disconnect the electrodes from the power source and then carefully remove the gel from the gel box.</LI><LI>Place the gel into a container filled with 100ml of 1xTAE running buffer and 5μL of EtBr for 5 minutes.</LI><LI>Use any device that has UV light to visualize the DNA fragments.</LI></OL></P></DIV><DIV id="prot14" class="protocol"><H2>J774 macrophages cell culture</H2><P>We cultured the cells in T75 flasks for cell culture. Cells were incubated at 37oC in a humidified, 5% CO2.</P><H3>Preparation of a complete media</H3><P><OL><LI>Take RPMI 1640 medium supplemented with Glutamine</LI><LI>Add 10% of foetal bovine serum</LI><LI>Add 1% of antibiotics (penicillin/streptomycin/Amphotercin B)</LI><LI>Add 1% HEPES</LI><LI>Add 1% pyruvate sodium.</LI></OL></P><H3>Passaging the cells</H3><P>Passage the J774 cells 2 times a week using a scratcher.
	<OL><LI>Pipet out the media from the flask</LI><LI>Add a 5 mL of a fresh complete media</LI><LI>Scratch the cells</LI><LI>Count the cells them under the microscope</LI></OL></P><H3>Froze cells for conservation</H3><P><OL><LI>Pipet the media out of the Flask</LI><LI>Add 5 mL of new media</LI><LI>Scratch the cells</LI><LI>Centrifuge the scratched cells at 4000 rpm during 5 minutes</LI><LI>Resuspend the pellet in DMSO 1/5, Foetal bovine serum 4/5.</LI><LI>Put in cryotubes at - 80°C.</LI></OL></P><H3>Cells counting</H3><P><OL><LI>Take 10µL of cells</LI><LI>Add 90µL of trypan blue</LI><LI>Count the cells under the microscope(if you count all the cells in the grid it corresponds to 1µL) If there is so many cells, dilute them in RPMI.</LI><LI>Calculate your amount of cells/µL (don't forget to multiplicate by 10 to take into account the dilution factor)</LI></OL></P></DIV></DIV></DIV><DIV class="mfooter"><DIV class="footbox" style=""><B>Centre for Research and Interdisciplinarity (CRI)</B>
    Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor
    Paris Descartes University
    24, rue du Faubourg Saint Jacques
    75014 Paris, France +33 1 44 41 25 22/25<A href="mailto:team2013@igem-paris.org">team2013@igem-paris.org</A></DIV><CENTER>
    Copyright (c) 2013 igem.org. All rights reserved.
  </CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Paris_Bettencourt/Protocols">http://2013.igem.org/Team:Paris_Bettencourt/Protocols</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Bettencourt/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Bettencourt/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Protocols&amp;oldid=319750" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>