<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Project_Detect"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Paris_Bettencourt/Project/Detect">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris_Bettencourt/Project/Detect&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Project/Detect&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Project/Detect&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris_Bettencourt/Project/Detect" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris Bettencourt/Project/Detect</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="navmenubkg"><DIV id="navmenubox"><UL id="nav"><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Overview" title="Project Overview">PROJECT</A></LI><LI style="margin-left:14.5px"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Overview" title="Project Overview">OVERVIEW</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" title="Detect">DETECT</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" title="Target">TARGET</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" title="Infiltrate">INFILTRATE</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" title="Sabotage">SABOTAGE</A></LI></UL></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Results" title="Achievements">ACHIEVEMENTS</A></LI><LI style="margin-left:141px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Parts" title="Submitted Parts">PARTS</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Results" title="Results">RESULTS</A></LI><LI style="width:167px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Medal_Fulfillment" title="Medal Fulfillment">MEDAL FULFILLMENT</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Overview" title="Human Practice Overview">HUMAN PRACTICE</A></LI><LI style="margin-left:141px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Overview" title="Human Practice Overview">OVERVIEW</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Gender_Study" title="Gender Study">GENDER STUDY</A></LI><LI style="width:168px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Technology_Transfer" title="Technology_Transfer">TECHNOLOGY TRANSFER</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/TB_Facts" title="TB Tacts">TB FACTS</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/TB_France" title="TB in France">TB IN FRANCE</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/TB_Gallery" title="TB Gallery">TB GALLERY</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Safety" title="Safety">SAFETY</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Collaboration" title="Collaboration">COLLABORATION</A></LI><LI style="margin-left:500px"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Collaboration" title="Collaboration">COLLABORATION</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/SensiGEM" title="SensiGEM">SENSIGEM</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Notes" title="Notebook">NOTEBOOK</A></LI><LI style="margin-left:649.5px"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Notebook" title="Day Notes">DAY NOTES</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Protocols" title="Protocols">PROTOCOLS</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Bibliography" title="Bibliography">BIBLIOGRAPHY</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Team" title="Team">TEAM</A></LI><LI style="margin-left:522.5px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Team" title="Team">OUR TEAM</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Gallery" title="Team Gallery">GALLERY</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Attributions" title="Attributions">ATTRIBUTIONS</A></LI><LI style="width:167px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Acknowledgements" title="Acknowledgements">ACKNOWLEDGEMENTS</A></LI><DIV id="twnoticon" class="sidelogos"><A class="twitter-timeline" width="250" height="400" data-dnt="true" href="https://twitter.com/iGEM_Paris" data-widget-id="368144501119795201">
            @iGEM_Paris</A></DIV><DIV id="page"><DIV class="overbox"><DIV class="bkgr"><H2>Background</H2><P>CRISPR/Cas systems generate site-specific double-strand breaks and have recently been used for genome editing.  </P></DIV><DIV class="results"><H2>Results</H2><UL><LI>Successfully cloned gRNA anti-KAN, crRNA anti-KAN, tracrRNA-Cas9 and pRecA-LacZ into Biobrick backbones and therefore generated four new BioBricks. </LI><LI> Confirmation of site-specific binding and DNA double-strand breaks generated by the gRNA-Cas9 complex in the kanamycin resistance casette. </LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Assembly_Standard">Testing the new assembly standard for our cloning.</A></LI></UL></DIV><DIV class="biocriks"><H2>BioBricks</H2><OL><LI><A href="http://parts.igem.org/Part:BBa_K1137012">BBa_K1137012(gRNA anti-KAN)</A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1137013">BBa_K1137013(crRNA anti-KAN)</A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1137014">BBa_K1137014(tracrRNA-Cas9)</A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1137015">BBa_K1137015(pRecA-LacZ)</A></LI></OL></DIV><DIV class="aims"><H2>Aims</H2><P>Building a genotype sensor based on CRISPR/Cas that reports on the existence of an antibiotic resistance gene.</P></DIV><A href="#Aim"><DIV class="hlink"><H2>Skip to Aim</H2></DIV></A><A href="#Design"><DIV class="hlink"><H2>Skip to Design</H2></DIV></A><A href="#Background"><DIV class="hlink"><H2>Skip to Background</H2></DIV></A><A href="#Results"><DIV class="hlink" style="margin-right:0px"><H2>Skip to Results</H2></DIV></A></DIV><H2>Aim</H2><DIV class="leftparagraph"><P>  
	We developed a sensor to detect whether a particular bacterial strain carries a specific antibiotic resistance gene. Our sensor system consists of <I>E.coli </I> lab strain and of a synthetic phagemid with a CRISPR/Cas system and LacZ as a reporter under the control of a pRecA promoter (SOS response promoter).  If our CRISPR/Cas system can bind to the target (antibiotic resistance gene), the Cas9 generates at this specific target site a double strand break, which will induce the expression of our double-strand break SOS sensor (Figure. 1). Having the system on a phagemid, the sensor system will spread all over a population, to get a clear color output if the target has been detected. Depending on target sequence the system carries, we can identify different antibiotic resistances in a strain. This is a novel approach of detecting genes in bacterial strains. We used <I>E.coli</I> and a M13 phagemid to target the kanamycin resistance gene.  This sensor is a proof of concept for a similar system in <I>Mycobacterium tuberculosis</I>. Such a system could potentially be used to test if a patient has TB and what type of resistance genes the specific strain contains to adapt the patient’s drug treatment.
    </P></DIV><DIV class="rightparagraph"><P><B>Figure 1:</B> Detection and reporting of an antibiotic resistance gene with a CRISPR/Cas system. </P><DIV style="font-size:90%">After expression of the Cas9 and gRNA, the gRNA guides the Cas9 to the target sequence, the kanamycin resistance. There, the Cas9 generates a double strand break. This activates the SOS response. The reporter LacZ is under the pRECA promoter, which gets activated during the SOS response and we get a blue cell, if the resistance gene has successfully been detected. </DIV></DIV><H2>Motivation and existing TB sensors/tests</H2><DIV class="leftparagraph"><P>  
	Tuberculosis (TB) remains a major global health problem. While the treatment of this disease in countries with adequate medical care is fairly easy to detect and treat, it remains hard to treat and diagnose in poorer countries. 
	Up to now, a high quality lab that uses modern diagnostics is a prerequisite for early, rapid and accurate detection of TB. Therefore, diagnosis of TB and drug resistant TB remains a particular challenge for laboratory systems, especially in developing countries.
	The lack of cheap, quick and accurate tests make it hard to control the Tuberculosis epidemic, which claims millions of lives every year in developing countries.
	Setting up a cheap, fast and culture-based method could therefore decrease diagnostic time and facilitate patient treatment.
	The most common method for diagnosing TB nowadays is sputum smear microscopy, in which bacteria are observed in sputum samples of patients under the microscope. However, this cannot be used to identify paucibacillary (containing just a few bacteria) or extrapulmonary (outside of the lungs) TB.
      </P></DIV><DIV class="rightparagraph"><P>  
	Diagnosis methods using culture methods require laboratory infrastructure that is not widely available in countries with a high burden of TB and results are only available after a few weeks.
	Other conventional methods used to diagnose multidrug-resistant TB (MDR-TB) also rely on the culturing of specimens followed by drug susceptibility testing (DST). Results take weeks to obtain and not all laboratories have the capacity to perform DST of first-line or of second-line drugs.
	MTB/RIF is a new rapid molecular test that can diagnose TB and rifampicin-resistant TB within hours. Molecular tests, such as GeneXpert, unlike culture-based methods, are fast, accurate and can detect drug-resistant strains. But the high costs and need for laboratories make access an issue for developing countries.
	The new method, published in the Journal of Applied Microbiology, uses a microcalorimeter to detect heat produced by Mycobacterium tuberculosis, the bacterium that causes TB, on a growth medium. The study showed that detection takes 4–5 days but more sensitive microcalorimeters could detect tuberculosis in 24 hours.
      </P></DIV><H2>System Design</H2><DIV class="leftparagraph"><P>  
	By taking advantage of the state of the art CRISPR/Cas research and combining it with phage diagnostics, we designed a sensor that is able to screen for a specific target gene. Our producer strain carries a helper plasmid that codes for the phage capsid proteins of M13 as well as our phagemid plasmid with our sensor system. Our sensor system consists out of three parts: the Cas9 gene under a constitutive promoter, the anti-KAN gRNA also under a constitutive promoter and the reporter element LacZ under the control of the pRecA promoter. As the helper plasmid doesn’t have a packaging sequence, only our sensor system phagemid is packed into the M13 capsids. As M13 is a lysogenic phage, the phagemid particles are exported into the media where they can be easily isolated and are ready to use to sense if an other strain contains our target sequence. 
      </P></DIV><DIV class="rightparagraph"><P>  
	As the phagemid is derived from M13, it infects only F+ (conjugated) cells. In our case, we conjugated our cells so that they can be easily infected. </P><P> Within the target strain, the Cas9 protein, as well as the gRNA get expressed. The gRNA then attaches to the Cas9 protein and guides the Cas9 to the target sequence (kanamycin resistance gene). Once bound to the sequence, the Cas9 protein generates a target specific double strand break. This double strand break leads to the cleavage of LexA. LexA is a protein bound to the pREC promoter that inhibits the expression of genes under the control of the pREC promoter. With the cleavage of LexA, the expression of our reporter LacZ is started, which leads in the presence of X-gal to a blue color output. 
	</P></DIV><DIV style="margin-left:10%"><P><B>Figure 2:Course of the detection and reporting of an antibiotic resistance gene with the CRISPR/Cas system. </B></P><DIV style="font-size:90%">After the plasmid has been released into the target cells, gRNA and Cas9 get expressed. The gRNA guides the Cas9 to the target, where it generates a double strand break. This activates the SOS response. As a results, LexA is cleaved which allows the expression of the reporter. In the presence of X-gal the cells turn blue. </DIV></DIV><DIV class="leftparagraph"><P>  
	Our system is highly modular as with the simple change of the gRNA target sequence (20bp), any gene of interest can be identified and targeted. This means that our system can be used for different applications and purposes, as for example a mutation screening. It is also not specified only for <I>E.coli</I> and the phage M13. Our whole system can be easily transferred to other phage systems that target other host bacteria. In our case, as we want to develop a sensor to test <I>Mycobacterium tuberculosis</I>, we could in principle transfer our system to mycobacteria phages. With what we are presenting here, we have a proof of concept in <I>E.coli</I>. By using host specific phages, we can easy identify specific bacteria and genes they carry.  By building a library of phages with different gRNAs we could screen for many different genes in parallel including other resistances or pathogenic genes.

    </P></DIV><DIV class="rightparagraph"><P>  
	For future perspectives, we plan to further test and improve the system so that in theory, we will generate a sensor that can be used without using high-tech laboratory facilities. This is very important, as easily to use sensors are needed in the developing countries in which tuberculosis is still a prevalent disease.</P><P>
	In a prospective design study we imagine the development of a handkerchief, in which our phagemid system can be integrated. By spitting on the tissue, we get a bacterial sample that will be tested by phages. Depending on which phagemid is integrated into the handkerchief, we can test for the <I>Mycobacterium tuberculosis</I> in general as well as for the different antibiotic resistances. A handkerchief is easy to use and the blue color output easy to read out. As phages are acting very fast, an easily usable and fast sensor system is developed.
    </P></DIV><DIV style="font-size:90%"><B>Figure 3: Design of future perspectives of how to use the sensor in daily life</B><P> The test itself will be provided as a handkerchief that is delivered in a bag with a pre stamped envelope. The bag contains the handkerchief, woven out of phage-integrated thread. The person that takes the test has to give a sputum sample on the handkerchief. To start the test, the tissue has to be put back into the bag and closed. By gently squeezing the bag, the integrated liquid vessel breaks and releases the X-gal solution which is needed to give an color output of the reporter. If the handkerchief turns blue, this means, that a specific gene e.g. a specific gene from <I>Mycobacterium tuberculosis</I> has been detected. If it doesn't turn blue, this means the target gene is not present and hence in the case of the mycobacterium gene the person doesn't have tuberculosis. This sensor and test can in principle be used for every gene and organism and can also be modified to sense other pathogenic bacteria or specific genes like antibiotic resistances a strain carries. </P></DIV><H2>Background</H2><H3>CRISPRs</H3><DIV class="leftparagraph"><P>  
	The mechanism we are using for our sensor is based on the CRISPR/Cas System. The CRISPR/Cas genome editing method, which recently became very popular, is derived from the bacterial “immune system”. CRISPRs are Clustered Regularly Interspaced Short Palindromic Repeats that are part of the bacterial genome. These loci contain multiple short repeats. In between the repeats are so called spacers that are sequences derived from extrachromosomal DNA, e.g. from invading viruses or plasmids (Figure 4). Within bacteria, those CRISPRs are naturally used to detect and destroy foreign DNA that is saved in the spacers.</P><P>In total the CRISPRs and the spacers form a so-called CRISPR array that is transcribed as one unit. CRISPR associated proteins (Cas proteins) are involved in further processing and action steps of the CRISPR system. There are many different regulation systems, here we describe the Cas9 system that will be used for our purposes (Figure 4).</P><P>The transcribed mRNA is processed into so called crRNA (CRISPR RNA), which includes the spacer sequence of foreign DNA. Together with a transactingCRISPR RNA (tracrRNA), the crRNA forms a duplex that is cleaved by RNaseII. The resulting hybrid serves as guide for the Cas9 protein that generates double strand breaks at the position the RNAs guide it to (Figure 3). By this double strand break the invading DNA is destroyed. The last years, researchers discovered the CRISPRs as a method for genome editing. </P><P><B>Figure 4:</B> CRISPR/Cas technology description (<A href="http://www.addgene.org/CRISPR/guide/" target="_blank">Addgene.org</A>).</P></DIV><DIV class="rightparagraph"><P>   By designing the spacer sequence, specific sequences in the genome can be targeted. With the provision of a sequence with homologous sequences, easy insertion can be done. This method can be used to insert mutations or new sequences nearly everywhere in the genome.
	Recently many papers have been published for genome editing in bacteria (Jinek et al., 2012, <A href="https://2013.igem.org/Team:Paris_Bettencourt/Bibliography#Jiang2013" target="_blank">Jiang <I>et al</I>. 2013</A>), yeast (<A href="https://2013.igem.org/Team:Paris_Bettencourt/Bibliography#DiCarlo2013" target="_blank">DiCarlo <I>et al</I>. 2013</A>) and mammalian cells (Mali et al., 2013, Cong et al., 2013). They describe how to use the CRISPRs for different purposes. One requirement to target a sequence is a NGG at the end of the sequence. This NGG sequence is called PAM - protospacer adjacent motif, while the target sequence itself is called the protospacer, which should have a length of around 12 to 20 bp.</P><P>In our approach we test two different systems, the systems of DiCarlo et al. 2013 and Jiang et al. 2013. The system of Jiang is a system developed for bacteria and especially for <I>E.coli</I>. The system of DiCarlo is based on a paper of Mali et al. 2013 and adapted to yeast.</P><P>Different from the Jiang paper, the DiCarlo paper uses a gRNA (guideRNA) to guide the Cas9 (Figure 5). The gRNA is actually the RNA that results after transcribing and folding is the same complex as you get after the processing of the tracrRNA with the crRNA. It is hence an improvement, which makes the design and expression easier. But as the system hasn’t been tested before in bacteria, we will use both systems to make sure we get the desired results.</P><P><B>Figure 5:</B>Cas9 protein interacting with CRISPR gRNA</P><DIV style="font-size:90%">Illustration of Cas9 protein interacting with CRISPR gRNA to direct endonuclease activity proximal to the PAM sequence (<A href="https://2013.igem.org/Team:Paris_Bettencourt/Bibliography#DiCarlo2013" target="_blank">DiCarlo et al. 2013</A>) </DIV></DIV><H3>RecA promoter</H3><DIV class="leftparagraph"><P>  
	Our reporter system consists out of the RecA promoter and LacZ. Originally, the RecA promoter of <I>E.coli</I> has its main role in the activation of the SOS repair system. It is regulated by the LexA repressor, which binds to the SOS box sequence of the promoter. DNA damage leads to an inducing signal, which then activates the RecA protein. In the SOS response the LexA repressor is cleaved by the RecA protein, so the full RecA expression can be reached. The RecA protein can then repair both single stranded and double stranded DNA breaks.</P><P>In our project RecA promoters used to detect double stranded DNA breaks in the region of antibiotic resistance genes caused by the CIRSPR/Cas system. To get the desired activation of the promoter, it is important to assure that no other stressing agents might activate the RecA promoter. Those agents could be UV light, X-ray, ionizing radiation and different types of DNA breaking compounds. </P></DIV><DIV class="rightparagraph"><P>  
	The SOS response in <I>E.coli</I> could also get induced by engineered M13 phage infection that are defective in the minus-strand origin, and so unable to form the double stranded replicative stage.</P><P>In this case the single-stranded DNA would be the SOS-inducing signal. The wild type M13 phage doesn’t induce the SOS response (<A href="https://2013.igem.org/Team:Paris_Bettencourt/Bibliography#Higashitani1992" target="_blank">Higashitani <I>et al</I>. 1992</A>). In another study genomes of phage M13 infected and uninfected <I>E.coli</I> strains were compared by oligonucleotide microarrays, where no stress response genes were scored as upregulated (<A href="https://2013.igem.org/Team:Paris_Bettencourt/Bibliography#Karlsson2005" target="_blank">Karlsson <I>et al</I>. 2005</A>). Regarding the phagemids, an A UV-damaged oriC phagemid did not induce SOS response in a recipient cells oriF phagemids on the other hand did (Sommer et al. 1991). To make sure that our system functions reliably, we will test the activation of the RecA promoter at different stress inputs (UV, phage infection,…) by measuring the fluorescence of YFP driven by the RecA promoter.</P></DIV><H2>Progress and preliminary Results</H2><H3>Parts</H3><DIV class="leftparagraph"><P>  Up to this point we cloned each of our parts into BioBrick vectors. We cloned the <A href="https://2013.igem.org/Team:Paris_Bettencourt/Notebook/Phage_Sensor/Monday_30th_September.html">gRNA anti-Kan </A> into pSB1C3 and pSB1A3.We used for this the assembly standard BioBrick cloning as well the new proposed <A href="https://2013.igem.org/Team:Paris_Bettencourt/Assembly_Standard">Assembly Standard.</A>. Both assemblies were successful and proved by colony PCR as well as by sequencing. We also used both standards for cloning the <A href="https://2013.igem.org/Team:Paris_Bettencourt/Notebook/Phage_Sensor/Monday_30th_September.html">crRNA.</A>. 
    </P></DIV><DIV class="rightparagraph"><P>   
	Also this is verified by colony PCR and by sequencing. For the cloning of the <A href="https://2013.igem.org/Team:Paris_Bettencourt/Notebook#detect_Monday_30th_September.html">tracrRNA-Cas9.</A> part we used standard BioBrick cloning, inserted into pSB1A3 and sequence verified it. We achieved the same for the  <A href="https://2013.igem.org/Team:Paris_Bettencourt/Notebook#detect_Monday_30th_September.html">pRecA-LacZ.</A> part (pSB1C3 and pSB1A3 backbones) that was submitted to the parts registry. </P></DIV><H3>Killing assay to verify targeting specificity</H3><DIV class="leftparagraph"><P>   For the characterization of the Cas9 and the gRNA anti-KAN we performed a <A href="https://2013.igem.org/Team:Paris_Bettencourt/Notebook/Phage_Sensor/Thursday_3rd_October.html">killing assay </A> . Therefore we co-transformed the Cas9 with gRNA anti-KAN into host bacteria that contain the KAN resistance gene cassette as well as without the cassette. In the KAN resistance bacteria, the CRISPR/Cas complex generated generate double strand breaks that killed the bacteria. After co-transforming the plasmids, we plated the bacteria on the one hand on the single antibiotic where the resistance is carried by the Cas9 plasmid as well as on the single antibiotic which resistance is present on the gRNA plasmid. As the two plasmids contain the same origin of replication, this will selesct for only the single plasmid and hence a non-functional CRISPR/Cas system. To get a working system, we plated the co-transformed bacteria on both antibiotics, of which the reistances were on the two plasmids. As we can see in Figure 6, if we select for both plasmids, we get a reduced number of colonies in the strain that contains the target sequence, the kanamycine resistance cassette. We attribute this reduced number to Cas9-induced cleavage of the chromosome specifically at the KanR casette, with about 99% efficiency. This leads after repeatedly double strand breaks generated by the CRISPR/Cas system to cell death which explaines the reduced colony number.
    </P></DIV><DIV class="rightparagraph"><P>  </P><P><B>Figure 6: </B> CRISPR anti-Kan plasmids target specifically kanamycin resistant <I>E. coli</I>.</P><DIV style="font-size:90%"> We introduced our CRISPR-based DNA cleavage system to two strains of <I>E. coli</I>: one WT (blue bars) and one carrying a genomically integrated kanamycin resistance casette (KanR, blue bars). The strains were co-transformed with two plasmids. One with the Cas9 construct, the other with the anti-Kanamycin gRNA. WT <I>E. coli</I> could be efficiently transformed with one or both plasmids, as determined by selective plating. However, KanR <I>E. coli</I> could not be efficiently transformed with both. We attribute this to Cas9-induced cleavage of the chromosome specifically at the KanR casette, with about 99% efficiency.</DIV></DIV><H3>Characterization of the reporter</H3><DIV class="leftparagraph"><P>  To characterize the pRecA –LacZ construct, we induced double strand breaks with Mytomycin C (MMC) and analyzed the resulting LacZ expression levels with the Miller assay. The same time, we wanted to characterize our final system: our target strain with the kanamycin resistance gene that also carries the reporter and the Cas9. The gRNA is delivered by a phage. As positive control, we used MG1655, which has a constitutive expression of LacZ, as negative control the parental strain of the target strain, which doesn’t have a kanamycin resistance cassette but also has the reporter and Cas9. As it can be seen in Figure 7, we didn’t get any expression of LacZ beside in the positive control, the MG1655 strain.
      </P><P>  </P><P><B>Figure 7: Miller assay</B> Using the Miller assay to quantify LacZ expression levels in different strains: MG1655 which has a constitutive expression, the target strain + KAN, the control strain -KAN, with and without phages as well as the target strain only with the reporter, induced with Mitomycin C.</P></DIV><DIV class="rightparagraph"><P>  
	As we worry, that our reporter might be not functional, we started to characterize a reporter constructed by Ariel Lindner. This reporter is YFP under the control of pREC. We analyzed its expression rates by inducing double strand breaks with Mitomycin C and  Niprofloxacin. The data generated by FACS analysis of the MMC induced strain over time can be seen in Figure 8. We can see an induction of the pREC promoter due to induction with MMC in the left figure. The shift to the right of the induced strain shows that we can see higher expression levels of our reporter YFP due to double strand breaks with MMC. Unfortunately this shift was not visible in an second experiment (right Figure). But we can see, that more cells show higher expression levels in comparison to the unindexed strain. As we have a high leakiness of the YFP reporter and due to time constraints, we were not able to successfully adapt our system to use this pREC-YFP reporter in our target strain and to test it. But, in previous iGEM years, there is already a <A href="http://parts.igem.org/Part:BBa_K862000?title=Part:BBa_K862000">pREC-LacZ Biobrick </A> constructed (Heidelberg 2012) with convincing functional analysis. In principle, our system could be completed, as a functional pREC-LacZ already exists. By adding the reporter of Heidelberg 2012 to our target strain with the Cas9, we should be able to get a functional sensor. This sensor can detect and report a specific DNA sequence due to a phagemid delivered gRNA that guides the present Cas9 to a target where the Cas9 generates a double strand break, which is reported by pREC-LacZ.</P><P><B>Figure 8: FACS analysis of the pREC-YFP strain.</B> Fluorescence against counts for WT as well as pREC strain unindexed and induced with 10ug/ml MMC after 2h of incubation.
    </P></DIV><H3>Next steps</H3><DIV class="leftparagraph"><P>As previously mentioned, the reporter didn’t work as expected. This can have several reasons. As our other positive control (beside MG1655), the MMC induced strains carrying the reporter didn’t show any expression, it stands to reason that the reporter itself is not functional. Another reason could be, that the phagemid didn’t deliver the gRNA or not in enough amounts to show an effect of the complete working system.
      </P></DIV><DIV class="rightparagraph"><P>    
	To test, if the phagemid delivers the gRNA correctly, we will perform another killing assay as we have done it before with the difference that the gRNA is delivered by the phagemid. The strains are then plated to see, if there is a reduced number of colonies of the target strain with phagemid in comparison to a control strain and without added phages. We expect the results before the end of iGEM.
      </P></DIV><H2>Literature</H2><DIV class="leftparagraph"><UL><LI>James E. DiCarlo, Julie E. Norville, Prashant Mali, Xavier Rios, John Aach and George M. Church (2013).<I>  Genome engineering in Saccharomyces cerevisiae using CRISPR-Cas systems.</I> Nucleic Acids Research. 2013, 1–8.</LI><LI id="">Antonio R. Fernandez de Henestrosa, Tomoo Ogi, Sayura Aoyagi, David Chafin, Jeffrey J. Hayes, Haruo Ohmori and Roger Woodgate (2000). <I>Identification of additional genes belonging to the LexA regulon in Escherichia coli.</I> Molecular Microbiology .2000 35(6), 1560-1572.</LI><LI id="">Higashitani N., Higashitani I.A., Roth A., Horiuchi A.K. (1992). <I>SOS Induction in Escherichia coli by Infection with Mutant Filamentous Phage That Are Defective in Initiation of Complementary-Strand DNA Synthesis.</I> Journal of Bacteriology. 174:1612-1618.</LI><LI id="">Wenyan Jiang, David Bikard, David Cox, Feng Zhang &amp; Luciano A Marraffini (2013). <I>RNA-guided editing of bacterial genomes using CRISPR-Cas systems.</I> Nature Biotechnology. 31 (3), 233-2398.</LI><LI id="">Douglas H. Juers, Brian W. Matthews, and Reuben E. Huber (2012). <I>LacZ b-galactosidase: Structure and function of an enzyme of historical and molecular biological importance.</I> Protein Science. 2012, 21,1792—1807.</LI><LI id="">Karlsson F., Malmborg-Hager A.C., Albrekt A.S., Borrebaeck C.A.K. (2005). <I>Genome-wide comparison of phage M13-infected vs. uninfected Escherichia coli. </I> Can. J. Microbiol. 51:29-35.</LI></UL></DIV><DIV class="rightparagraph"><UL><LI id="">Kimberly L. Keller, Terri L. Overbeck-Carrick, Doris J. Beck (2001). <I>Survival and induction of SOS in Escherichia coli treated with cisplatin, UV irradiation, or mitomycin C are dependent on the function of the RecBC and RecFOR pathways of homologous recombination. </I> Mutation Research. 486, 2001, 21–29.</LI><LI id="">Kobayashi, Ichizo; and Handa, Naofumi (2009). <I>DNA Doublestrand Breaks and Their Consequences in Bacteria. </I> Encyclopedia of Life Sciences (ELS). John Wiley &amp; Sons, Ltd: Chichester.</LI><LI id=""> Timothy K. Lu, Jayson Bowers, and Michael S. Koeris (2013). <I> Advancing bacteriophage-based microbial diagnostics with synthetic biology. </I>Trends in Biotechnology. April 2013, 31 (6), 325-227.</LI><LI id="">Anders Norman, Lars Hestbjerg Hansen, and Søren J. Sørensen (2004). <I>Construction of a ColD cda Promoter-Based SOS-Green Fluorescent Protein Whole-Cell Biosensor with Higher Sensitivity toward Genotoxic Compounds than Constructs Based on recA, umuDC, or sulA Promoters. </I> APPLIED AND ENVIRONMENTAL MICROBIOLOGY. May 2005, p. 2338–2346.</LI><LI id="">Sommer S., Leitao A., Bernardi A., Bailone A., Devoret R. (1991). <I>Introduction of a UV-damaged replicon into a recipient cell is not a sufficient condition to produce an SOS-inducing signal.</I> Mutation Research/DNA Repair. 254:107-17.</LI></UL></DIV><H2>Attributions</H2><P> Most of the strains and plasmids used for this project were kindly provided by the INSERM U1001 lab.</P><P> The Cas9 plasmid as well as the CRISPR plasmid were ordered from Addgene. The crRNA was designed after consultation with David Bikard.</P><P> The sequence of LacZ used in this project was taken from pBAC-BA-lacZ from Addgene. The corresponding pREC sequence is the natural sequence. The source for the sequence is EcoCyc.</P><P> The phaegmid template as well as the helper plasmid were provided by Monica Ortiz from the Endy Lab, Stanford.</P><P> The project itself was designed and accomplished by Nicolas Koutsoubelis, Anne Loechner and Marguerite Benony with consultation with Edwin Wintermute, Stanislas and Ariel Lindner.</P></DIV><DIV class="mfooter"><DIV class="footbox" style=""><B>Centre for Research and Interdisciplinarity (CRI)</B>
    Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor
    Paris Descartes University
    24, rue du Faubourg Saint Jacques
    75014 Paris, France +33 1 44 41 25 22/25<A href="mailto:team2013@igem-paris.org">team2013@igem-paris.org</A></DIV><CENTER>
    Copyright (c) 2013 igem.org. All rights reserved.
  </CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Paris_Bettencourt/Project/Detect">http://2013.igem.org/Team:Paris_Bettencourt/Project/Detect</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Bettencourt/Project/Detect" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Bettencourt/Project/Detect" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Project/Detect&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Project/Detect&amp;oldid=361817" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>