"
Page
Discussion
View source
History
teams
Log in
 
Team:Paris Bettencourt/BioBricks
From 2013.igem.org
PROJECT
OVERVIEW
DETECT
TARGET
INFILTRATE
SABOTAGE
ACHIEVEMENTS
PARTS
RESULTS
MEDAL FULFILLMENT
HUMAN PRACTICE
OVERVIEW
GENDER STUDY
TECHNOLOGY TRANSFER
TB FACTS
TB IN FRANCE
TB GALLERY
SAFETY
COLLABORATION
COLLABORATION
SENSIGEM
NOTEBOOK
DAY NOTES
PROTOCOLS
BIBLIOGRAPHY
TEAM
OUR TEAM
GALLERY
ATTRIBUTIONS
ACKNOWLEDGEMENTS
@iGEM_Paris
Part: BBa_K1137008 (TDMH)
   TDMH is a trehalose 6,6’ dimycolate (TDM) esterase serine esterase superfamily that can hydrolyze purified TDM from various mycobacterial species.  TDM is one of the most potent immunomodulatory and granulomatogenic surface glycolipids and it is highly abundant in at least ten species of pathogenic and non-pathogenic mycobacteria.  Additonally TDM is crucial to the structural integrity of the mycobacterial envelope.  As such, TDMH triggers rapid and extensive lysis of mycobacterial species due to the release of free mycolic acids from the non-covalently associated mycolyl-containing glycolipids.
We biobricked TDMH as a method for lysing Mycobacteria including M. smegmatis and M. Tuberculosis.  We have demonstrated experimentally that TDMH is effective at killing M. smegmatis when secreted by E. coli.  We expressed TDMH on a T7 promoter within lacO sites inserted such that it could be induced by IPTG (Invitrogen Duet Vector) at a concentration of 1mM and found that TDMH was effective when the M. smegmatis to E. coli ratio 1:1 or 10:1.  Higher ratios of M. smegmatis to E. coli were no longer effective.
Part: BBa_K1137009 (sRNA anti Kan)
   This complex part contains everything necessary to express small RNA to inhibit expression of the Kanamycin resistance cassette encoding Aminoglycoside N6'-acetyltransferase.  The biobrick contains a Pr promoter, target sRNA, a scaffold derived from MicC lacking the ompC binding sequence, and a T1/TE terminator.  The scaffold forms the RNA secondary structure while the target sRNA of the sequence CGTTTCCCGTTGAATATGGCT binds to the target sequence of ATGAGCCATATTCAACGGGAAACG including the start codon and the first 24 bp of the kanamycin ORF within the mRNA.  It is vital that the sequences are complementary for proper repression to occur.  If there are point mutations within a copy of the kanamycin resistance cassette at this location, or a different kanamycin cassette is targeted the biobrick will not function correctly.
Part: BBa_K1137010 (sRNA anti Cm)
This complex part contains everything necessary to express small RNA to inhibit expression of the Chloramphenicol resistance cassette cat encoding chloramphenicol acetyltransferase.  Very similar to BBa_K1137010, the biobrick contains a Pr promoter, target sRNA, a scaffold derived from MicC lacking the ompC binding sequence, and a T1/TE terminator.  The scaffold forms the RNA secondary structure while the target sRNA of the sequence ATATCCAGTGATTTTTTTCTC binds to the target sequence of ATGGAGAAAAAAATCACTGGATAT which includes the start codon and the first 24 bp of the ORF within the chloramphenicol mRNA.  As above, it is vital that the sequences are complementary for proper repression to occur.  If there are point mutations within a copy of the cat cassette at this location, or a different mechanism provides chloramphenicol resistance the biobrick will not function correctly.
Part: BBa_K1137011 (sRNA anti Lac)
   Highly similar to BBa_K1137009 and BBa_K1137010, this part contains everything necessary to express small RNA to inhibit expression of lacZ.  The biobrick contains a Pr promoter, target sRNA, a scaffold derived from MicC lacking the ompC binding sequence and a T1/TE terminator.  The scaffold forms the RNA secondary structure while the target sRNA of the sequence CAGTGAATCCGTAATCATGGT bind to the target sequence of ATGACCATGATTACGGATTCACTG which includes the start codon and the first 24 bp of the ORF within the lacZ mRNA.  As in the previously described biobricks, it is vital that the sequences are complementary for proper repression to occur.  If there are point mutations within the lacZ gene or biobrick sequence the biobrick will not function correctly.  Additionally, there will be some leakage of β-Galactosidase which will cause cleavage of X-Gal, however we observed that edges of colonies tended to be white while the center of colonies tended to be blue.
Part:BBa_K1137000 (M. Smegmatis SirA)
   This biobrick encodes a Ferredoxin-dependent sulfite reductase SirA, which reduces sulfite to hydrogen sulfide in the cysteine metabolism pathway.  Due to the sensitivity of sulfur contain amino acids to oxidative stress; they must constantly be replenished in mycobacteria while inside the phagosome.  Thus sirA is one of the few genes that is upregulated even in latent mycobacterial infections.  Additionally, SirA differs from E. coli and Human sulfite reductases in its use of ferredoxin as an electron done instead of NADPH.  As a result sirA has been previously identified as a drug target candidate for M. Tuberculosis infections. SirA reduces sulfite in the following reaction (EC=1.8.7.1):
Thus this gene can be used to provide cysteine in E. coli strains that lack a native cysI gene and allow for growth on minimal media.  However, to do so it requires co-expression of biobricks BBa_K1137001 and BBa_K1137002.  Even with minimal induction in a duet vector expression system, we were able to obtain growth in M9 media supplemented with glucose as a carbon source when co-expressed with BBa_K1137001 and BBa_K1137002, however the parent strain which lacked these biobricks was unable to grow.  M. Smegmatis sirA  has high homology (about 89%) to sirA from M. Tuberculosis.
Part:BBa_K1137001 (M. Smegmatis FprA)
   This biobrick encodes a NADPH-ferredoxin reductase FprA.  This replenishes FdxA by reducing it with NADPH as an electron donor and is required for the proper function of BBa_K1137000 along with BBa_K1137001 in the following reaction (EC=1.18.1.2):
FprA in addition to SirA has low homology to mammalian proteins and contains one tightly bound FAD.  Additonally it has been well characterized and it’s kinetics determined experimentally.  As mentioned under BBa_K1137000, even with minimal induction in a duet vector expression system, we were able to obtain growth in M9 media supplemented with glucose as a carbon source when co-expressed with BBa_K1137001 and BBa_K1137002, however the parent strain which lacked these biobricks was unable to grow.  FprA from M. Smegmatis has a homolog in M. Tuberculosis.
Part:BBa_K1137002 (M. Smegmatis FdxA)
   This biobrick encodes a 7Fe ferredoxin which is an orthologue of ferredoxin FdxC in M. Tuberculosis.  It contains both one 3Fe-4S and one 4Fe-4S cluster.  FdxA is the preferred substrate of FprA and is required for the proper functioning of SirA for sulfite reduction during cysteine metabolism.
Thus it required for growth on minimal media.  We were able to obtain growth from an E. coli strain lacking native cysI in M9 media supplemented with glucose as a carbon source when co-expressed with BBa_K1137000 and BBa_K1137001.
Part:BBa_K1137012 (gRNA anti KAN)
   This rather simple part contains the UG6 promoter that controls the gRNA anti KAN cassette. This cassette consists of a seed sequence of the Kanamycin resistance gene (Aminoglycoside N6’-acetyltransferase) and the gRNA scaffold. This part is ready to use and can generate together with the CAS9 a double strand break in the kanamycin resistance gene.  By definition, the gRNA is hybrid RNA out of the crRNA and tracrRNA. The gRNA is necessary to guide the CAS9 to the specific site and to cause there a double strand break. The design of the gRNA is taken from the DiCarlo et al. 2013 paper.
Part:BBa_K1137013 (crRNA anti KAN)
   This rather simple part contains the crRNA. The crRNA contains the target sequence to which the Cas9 protein is guided. The crRNA part can be cloned in front of any promoter. Note, that no RBS should be between the promoter and the crRNA. The crRNA is one of the two RNAs (crRNA and tracrRNA) that are needed to build the RNA dimer complex that fuses with the Cas9. The crRNA and the tracrRNA dimerize and are processed. This complex then binds to the CAS9. The RNA hybrid guides the Cas9 to its target and the Cas9 generates a site-specific double strand break. This crRNA here encodes for a sequence of the Kanamycin resistance gene cassette. The design of the crRNA is taken from the Jian et al. 2013 paper.
Part:BBa_K1137014 (tracRNA-CAS9 )
   This complex part contains the tracrRNA and the CAS9 under the control of constitutive promoters. Note that there is no RBS before the tracRNA.  The tracrRNA-CAS9 part is the heart of the CRISPR system. The tracrRNA and the corresponding target crRNA dimerize and are processed and hybridize before binding to the CAS9 protein. The tracrRNA has no specific part that is responsible for binding to the targeted DNA. Therefor this part can be used together with any crRNA. The CAS9 gene encodes for the CRISPR associated protein 9 that is needed to generate a double strand break at a target site. CAS9 binds to the tracRNA-crRNA dimer/gRNA and is then guided by them/it to the target site where it generate a double strand break. This part can be used together with the crRNA anti KAN or the gRNA anti KAN and cause double strand breaks. It can of course also be used together with new generated crRNAs/gRNAs. The tracRNA-CAS9 part is taken from the CRISPER plasmid of the Jiang et al. 2013 paper.
Part:BBa_K1137015 (pRecA-LacZ )
   This part is a reporter element of our sensor system that senses double strand breaks and consists of the native pRecA promoter and RBS sequence like in EcoCyc. The promoter and the RBS are fused to the full LacZ gene (I732017) that encodes for ß-galactosidase.  The RecA protein is a recombinase that is active during the SOS response after double strand breaks.  RecA and its promoter are well characterized and used in many SOS studies. ß-galcatosidase cleaves lactose and its derivate X-Gal. X-Gal is cleaved into galactose and an insoluble blue dye. Therefore the LacZ gene is a classical reporter gene.
Centre for Research and Interdisciplinarity (CRI) Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor Paris Descartes University 24, rue du Faubourg Saint Jacques 75014 Paris, France
+33 1 44 41 25 22/25
team2013@igem-paris.org
Copyright (c) 2013 igem.org. All rights reserved.
Retrieved from " http://2013.igem.org/Team:Paris_Bettencourt/BioBricks "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
