<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Paris_Bettencourt/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris_Bettencourt/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris_Bettencourt/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris Bettencourt/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="navmenubkg"><DIV id="navmenubox"><UL id="nav"><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Overview" title="Project Overview">PROJECT</A></LI><LI style="margin-left:14.5px"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Overview" title="Project Overview">OVERVIEW</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" title="Detect">DETECT</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" title="Target">TARGET</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" title="Infiltrate">INFILTRATE</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" title="Sabotage">SABOTAGE</A></LI></UL></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Results" title="Achievements">ACHIEVEMENTS</A></LI><LI style="margin-left:141px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Parts" title="Submitted Parts">PARTS</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Results" title="Results">RESULTS</A></LI><LI style="width:167px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Medal_Fulfillment" title="Medal Fulfillment">MEDAL FULFILLMENT</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Overview" title="Human Practice Overview">HUMAN PRACTICE</A></LI><LI style="margin-left:141px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Overview" title="Human Practice Overview">OVERVIEW</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Gender_Study" title="Gender Study">GENDER STUDY</A></LI><LI style="width:168px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Technology_Transfer" title="Technology_Transfer">TECHNOLOGY TRANSFER</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/TB_Facts" title="TB Tacts">TB FACTS</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/TB_France" title="TB in France">TB IN FRANCE</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/TB_Gallery" title="TB Gallery">TB GALLERY</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Safety" title="Safety">SAFETY</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Collaboration" title="Collaboration">COLLABORATION</A></LI><LI style="margin-left:500px"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Collaboration" title="Collaboration">COLLABORATION</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/SensiGEM" title="SensiGEM">SENSIGEM</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Notes" title="Notebook">NOTEBOOK</A></LI><LI style="margin-left:649.5px"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Notebook" title="Day Notes">DAY NOTES</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Protocols" title="Protocols">PROTOCOLS</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Bibliography" title="Bibliography">BIBLIOGRAPHY</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Team" title="Team">TEAM</A></LI><LI style="margin-left:522.5px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Team" title="Team">OUR TEAM</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Gallery" title="Team Gallery">GALLERY</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Attributions" title="Attributions">ATTRIBUTIONS</A></LI><LI style="width:167px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Acknowledgements" title="Acknowledgements">ACKNOWLEDGEMENTS</A></LI><DIV id="twnoticon" class="sidelogos"><A class="twitter-timeline" width="250" height="400" data-dnt="true" href="https://twitter.com/iGEM_Paris" data-widget-id="368144501119795201">
            @iGEM_Paris</A></DIV><P style="font=size:18px">Please be patient. It may take a few seconds to load</P><DIV id="page"><DIV id="leftscroll"><UL class="treeView"><LI>
  DAY NOTES
  </LI><LI class=" collapsibleListClosed">
      DETECT
      </LI><LI class=" collapsibleList">
	  July
	  </LI><LI class=""><A href="#detect_Saturday_13th_July.html"> Saturday 13<SUP>th</SUP> July</A></LI><LI class=""><A href="#detect_Sunday_14th_July.html"> Sunday 14<SUP>th</SUP> July</A></LI><LI class=""><A href="#detect_Monday_15th_July.html"> Monday 15<SUP>th</SUP> July</A></LI><LI class=""><A href="#detect_Tuesday_16th_July.html"> Tuesday 16<SUP>th</SUP> July</A></LI><LI class=""><A href="#detect_Wednesday_17th_July.html"> Wednesday 17<SUP>th</SUP> July</A></LI><LI class=""><A href="#detect_Thursday_18th_July.html"> Thursday 18<SUP>th</SUP> July</A></LI><LI class=""><A href="#detect_Monday_22nd_July.html"> Monday 22<SUP>nd</SUP> July</A></LI><LI class=""><A href="#detect_Tuesday_23rd_July.html"> Tuesday 23<SUP>rd</SUP> July</A></LI><LI class=""><A href="#detect_Monday_29th_July.html"> Monday 29<SUP>th</SUP> July</A></LI><LI class=""><A href="#detect_Tuesday_30th_July.html"> Tuesday 30<SUP>th</SUP> July</A></LI><LI class="lastChild"><A href="#detect_Wednesday_31st_July.html"> Wednesday 31<SUP>st</SUP> July</A></LI></UL><LI class=" collapsibleList">
	  August
	  </LI><LI class=""><A href="#detect_Thursday_1st_August.html"> Thursday 1<SUP>st</SUP> August</A></LI><LI class=""><A href="#detect_Friday_2nd_August.html"> Friday 2<SUP>nd</SUP> August</A></LI><LI class=""><A href="#detect_Monday_5th_August.html"> Monday 5<SUP>th</SUP> August</A></LI><LI class=""><A href="#detect_Tuesday_6th_August.html"> Tuesday 6<SUP>th</SUP> August</A></LI><LI class=""><A href="#detect_Wednesday_7th_August.html"> Wednesday 7<SUP>th</SUP> August</A></LI><LI class=""><A href="#detect_Thursday_8th_August.html"> Thursday 8<SUP>th</SUP> August</A></LI><LI class=""><A href="#detect_Friday_9th_August.html"> Friday 9<SUP>th</SUP> August</A></LI><LI class=""><A href="#detect_Saturday_10th_August.html"> Saturday 10<SUP>th</SUP> August</A></LI><LI class=""><A href="#detect_Sunday_11th_August.html"> Sunday 11<SUP>th</SUP> August</A></LI><LI class=""><A href="#detect_Monday_12th_August.html"> Monday 12<SUP>th</SUP> August</A></LI><LI class=""><A href="#detect_Tuesday_13th_August.html"> Tuesday 13<SUP>th</SUP> August</A></LI><LI class=""><A href="#detect_Wednesday_14th_August.html"> Wednesday 14<SUP>th</SUP> August</A></LI><LI class=""><A href="#detect_Thursday_15th_August.html"> Thursday 15<SUP>th</SUP> August</A></LI><LI class=""><A href="#detect_Friday_16th_August.html"> Friday 16<SUP>th</SUP> August</A></LI><LI class=""><A href="#detect_Monday_19th_August.html"> Monday 19<SUP>th</SUP> August</A></LI><LI class=""><A href="#detect_Tuesday_20th_August.html"> Tuesday 20<SUP>th</SUP> August</A></LI><LI class=""><A href="#detect_Wednesday_21st_August.html"> Wednesday 21<SUP>st</SUP> August</A></LI><LI class=""><A href="#detect_Thursday_22nd_August.html"> Thursday 22<SUP>nd</SUP> August</A></LI><LI class=""><A href="#detect_Friday_23rd_August.html"> Friday 23<SUP>rd</SUP> August</A></LI><LI class=""><A href="#detect_Sunday_25th_August.html"> Sunday 25<SUP>th</SUP> August</A></LI><LI class=""><A href="#detect_Monday_26th_August.html"> Monday 26<SUP>th</SUP> August</A></LI><LI class=""><A href="#detect_Tuesday_27th_August.html"> Tuesday 27<SUP>th</SUP> August</A></LI><LI class=""><A href="#detect_Wednesday_28th_August.html"> Wednesday 28<SUP>th</SUP> August</A></LI><LI class="lastChild"><A href="#detect_Friday_30th_August.html"> Friday 30<SUP>th</SUP> August</A></LI><LI class=" collapsibleList">
	  September
	  </LI><LI class=""><A href="#detect_Tuesday_3rd_September.html"> Tuesday 3<SUP>rd</SUP> September</A></LI><LI class=""><A href="#detect_Wednesday_4th_September.html"> Wednesday 4<SUP>th</SUP> September</A></LI><LI class=""><A href="#detect_Thursday_5th_September.html"> Thursday 5<SUP>th</SUP> September</A></LI><LI class=""><A href="#detect_Friday_6th_September.html"> Friday 6<SUP>th</SUP> September</A></LI><LI class=""><A href="#detect_Saturday_7th_September.html"> Saturday 7<SUP>th</SUP> September</A></LI><LI class=""><A href="#detect_Sunday_8th_September.html"> Sunday 8<SUP>th</SUP> September</A></LI><LI class=""><A href="#detect_Monday_16th_September.html"> Monday 16<SUP>th</SUP> September</A></LI><LI class=""><A href="#detect_Tuesday_17th_September.html"> Tuesday 17<SUP>th</SUP> September</A></LI><LI class=""><A href="#detect_Wednesday_18th_September.html"> Wednesday 18<SUP>th</SUP> September</A></LI><LI class=""><A href="#detect_Sunday_22nd_September.html"> Sunday 22<SUP>nd</SUP> September</A></LI><LI class=""><A href="#detect_Monday_23rd_September.html"> Monday 23<SUP>rd</SUP> September</A></LI><LI class=""><A href="#detect_Tuesday_24th_September.html"> Tuesday 24<SUP>th</SUP> September</A></LI><LI class=""><A href="#detect_Wednesday_25th_September.html"> Wednesday 25<SUP>th</SUP> September</A></LI><LI class=""><A href="#detect_Thursday_26th_September.html"> Thursday 26<SUP>th</SUP> September</A></LI><LI class=""><A href="#detect_Friday_27th_September.html"> Friday 27<SUP>th</SUP> September</A></LI><LI class=""><A href="#detect_Saturday_28th_September.html"> Saturday 28<SUP>th</SUP> September</A></LI><LI class="lastChild"><A href="#detect_Monday_30th_September.html"> Monday 30<SUP>th</SUP> September</A></LI><LI class=" collapsibleList">
	  October
	  </LI><LI class=""><A href="#detect_Tuesday_1st_October.html"> Tuesday 1<SUP>st</SUP> October</A></LI><LI class=""><A href="#detect_Wednesday_2nd_October.html"> Wednesday 2<SUP>nd</SUP> October</A></LI><LI class=""><A href="#detect_Thursday_3rd_October.html"> Thursday 3<SUP>rd</SUP> October</A></LI><LI class=""><A href="#detect_Monday_14th_October.html"> Monday 14<SUP>th</SUP> October</A></LI><LI class=""><A href="#detect_Tuesday_15th_October.html"> Tuesday 15<SUP>th</SUP> October</A></LI><LI class=""><A href="#detect_Wednesday_16th_October.html"> Wednesday 16<SUP>th</SUP> October</A></LI><LI class=""><A href="#detect_Thursday_17th_October.html"> Thursday 17<SUP>th</SUP> October</A></LI><LI class=""><A href="#detect_Friday_18th_October.html"> Friday 18<SUP>th</SUP> October</A></LI><LI class=""><A href="#detect_Saturday_19th_October.html"> Saturday 19<SUP>th</SUP> October</A></LI><LI class=""><A href="#detect_Tuesday_22nd_October.html"> Tuesday 22<SUP>nd</SUP> October</A></LI><LI class=""><A href="#detect_Thursday_24th_October.html"> Thursday 24<SUP>th</SUP> October</A></LI><LI class=""><A href="#detect_Friday_25th_October.html"> Friday 25<SUP>th</SUP> October</A></LI><LI class=""><A href="#detect_Saturday_26th_October.html"> Saturday 26<SUP>th</SUP> October</A></LI><LI class="lastChild"><A href="#detect_Sunday_27th_October.html"> Sunday 27<SUP>th</SUP> October</A></LI><LI class=" collapsibleListClosed">
      TARGET
      </LI><LI class=" collapsibleList">
	  June
          </LI><LI class=""><A href="#target_Monday_10th_June.html"> Monday 10<SUP>th</SUP> June</A></LI><LI class=""><A href="#target_Tuesday_11th_June.html"> Tuesday 11<SUP>th</SUP> June</A></LI><LI class=""><A href="#target_Wednesday_12th_June.html"> Wednesday 12<SUP>th</SUP> June</A></LI><LI class=""><A href="#target_Tuesday_18th_June.html"> Tuesday 18<SUP>th</SUP> June</A></LI><LI class="lastChild"><A href="#target_Monday_24th_June.html"> Monday 24<SUP>th</SUP> June</A></LI><LI class=" collapsibleList">
	  July
          </LI><LI class=""><A href="#target_Monday_1st_July.html"> Monday 1<SUP>st</SUP> July</A></LI><LI class=""><A href="#target_Tuesday_2nd_July.html"> Tuesday 2<SUP>nd</SUP> July</A></LI><LI class=""><A href="#target_Wednesday_3rd_July.html"> Wednesday 3<SUP>rd</SUP> July</A></LI><LI class=""><A href="#target_Thursday_4th_July.html"> Thursday 4<SUP>th</SUP> July</A></LI><LI class=""><A href="#target_Friday_5th_July.html"> Friday 5<SUP>th</SUP> July</A></LI><LI class=""><A href="#target_Tuesday_16th_July.html"> Tuesday 16<SUP>th</SUP> July</A></LI><LI class=""><A href="#target_Wednesday_17th_July.html"> Wednesday 17<SUP>th</SUP> July</A></LI><LI class=""><A href="#target_Thursday_18th_July.html"> Thursday 18<SUP>th</SUP> July</A></LI><LI class=""><A href="#target_Friday_19th_July.html"> Friday 19<SUP>th</SUP> July</A></LI><LI class="lastChild"><A href="#target_Wednesday_24th_July.html"> Wednesday 24<SUP>th</SUP> July</A></LI><LI class=" collapsibleList">
	  August
	  </LI><LI class=""><A href="#target_Monday_5th_August.html"> Monday 5<SUP>th</SUP> August</A></LI><LI class=""><A href="#target_Tuesday_6th_August.html"> Tuesday 6<SUP>th</SUP> August</A></LI><LI class=""><A href="#target_Monday_19th_August.html"> Monday 19<SUP>th</SUP> August</A></LI><LI class=""><A href="#target_Tuesday_20th_August.html"> Tuesday 20<SUP>th</SUP> August</A></LI><LI class=""><A href="#target_Wednesday_21st_August.html"> Wednesday 21<SUP>st</SUP> August</A></LI><LI class=""><A href="#target_Thursday_22nd_August.html"> Thursday 22<SUP>nd</SUP> August</A></LI><LI class=""><A href="#target_Friday_23rd_August.html"> Friday 23<SUP>rd</SUP> August</A></LI><LI class=""><A href="#target_Saturday_24th_August.html"> Saturday 24<SUP>th</SUP> August</A></LI><LI class=""><A href="#target_Sunday_25th_August.html"> Sunday 25<SUP>th</SUP> August</A></LI><LI class=""><A href="#target_Monday_26th_August.html"> Monday 26<SUP>th</SUP> August</A></LI><LI class=""><A href="#target_Tuesday_27th_August.html"> Tuesday 27<SUP>th</SUP> August</A></LI><LI class=""><A href="#target_Wednesday_28th_August.html"> Wednesday 28<SUP>th</SUP> August</A></LI><LI class=""><A href="#target_Thursday_29th_August.html"> Thursday 29<SUP>th</SUP> August</A></LI><LI class="lastChild"><A href="#target_Friday_30th_August.html"> Friday 30<SUP>th</SUP> August</A></LI><LI class=" collapsibleList">
	  September
	  </LI><LI class=""><A href="#target_Monday_9th_September.html"> Monday 9<SUP>th</SUP> September</A></LI><LI class=""><A href="#target_Tuesday_10th_September.html"> Tuesday 10<SUP>th</SUP> September</A></LI><LI class=""><A href="#target_Monday_16th_September.html"> Monday 16<SUP>th</SUP> September</A></LI><LI class=""><A href="#target_Tuesday_17th_September.html"> Tuesday 17<SUP>th</SUP> September</A></LI><LI class=""><A href="#target_Wednesday_18th_September.html"> Wednesday 18<SUP>th</SUP> September</A></LI><LI class=""><A href="#target_Thursday_19th_September.html"> Thursday 19<SUP>th</SUP> September</A></LI><LI class=""><A href="#target_Saturday_21st_September.html"> Saturday 21<SUP>st</SUP> September</A></LI><LI class=""><A href="#target_Sunday_22nd_September.html"> Sunday 22<SUP>nd</SUP> September</A></LI><LI class=""><A href="#target_Friday_27th_September.html"> Friday 27<SUP>th</SUP> September</A></LI><LI class=""><A href="#target_Saturday_28th_September.html"> Saturday 28<SUP>th</SUP> September</A></LI><LI class=""><A href="#target_Sunday_29th_September.html"> Sunday 29<SUP>th</SUP> September</A></LI><LI class="lastChild"><A href="#target_Monday_30th_September.html"> Monday 30<SUP>th</SUP> September</A></LI><LI class=" collapsibleList">
	  October
	  </LI><LI class=""><A href="#target_Tuesday_1st_October.html"> Tuesday 1<SUP>st</SUP> October</A></LI><LI class=""><A href="#target_Wednesday_2nd_October.html"> Wednesday 2<SUP>nd</SUP> October</A></LI><LI class="lastChild"><A href="#target_Thursday_3rd_October.html"> Thursday 3<SUP>rd</SUP> October</A></LI><LI class=" collapsibleListClosed">
      INFILTRATE
      </LI><LI class=" collapsibleList">
	  July
	  </LI><LI class=""><A href="#infiltrate_Monday_22nd_July.html"> Monday 22<SUP>nd</SUP> July</A></LI><LI class=""><A href="#infiltrate_Tuesday_23rd_July.html"> Tuesday 23<SUP>rd</SUP> July</A></LI><LI class=""><A href="#infiltrate_Wednesday_24th_July.html"> Wednesday 24<SUP>th</SUP> July</A></LI><LI class=""><A href="#infiltrate_Thursday_25th_July.html"> Thursday 25<SUP>th</SUP> July</A></LI><LI class=""><A href="#infiltrate_Friday_26th_July.html"> Friday 26<SUP>th</SUP> July</A></LI><LI class=""><A href="#infiltrate_Monday_29th_July.html"> Monday 29<SUP>th</SUP> July</A></LI><LI class=""><A href="#infiltrate_Tuesday_30th_July.html"> Tuesday 30<SUP>th</SUP> July</A></LI><LI class="lastChild"><A href="#infiltrate_Wednesday_31st_July.html"> Wednesday 31<SUP>st</SUP> July</A></LI><LI class=" collapsibleList">
	  August
	  </LI><LI class=""><A href="#infiltrate_Thursday_1st_August.html"> Thursday 1<SUP>st</SUP> August</A></LI><LI class=""><A href="#infiltrate_Friday_2nd_August.html"> Friday 2<SUP>nd</SUP> August</A></LI><LI class=""><A href="#infiltrate_Monday_5th_August.html"> Monday 5<SUP>th</SUP> August</A></LI><LI class=""><A href="#infiltrate_Tuesday_6th_August.html"> Tuesday 6<SUP>th</SUP> August</A></LI><LI class=""><A href="#infiltrate_Wednesday_7th_August.html"> Wednesday 7<SUP>th</SUP> August</A></LI><LI class=""><A href="#infiltrate_Thursday_8th_August.html"> Thursday 8<SUP>th</SUP> August</A></LI><LI class=""><A href="#infiltrate_Monday_12th_August.html"> Monday 12<SUP>th</SUP> August</A></LI><LI class=""><A href="#infiltrate_Tuesday_13th_August.html"> Tuesday 13<SUP>th</SUP> August</A></LI><LI class=""><A href="#infiltrate_Wednesday_14th_August.html"> Wednesday 14<SUP>th</SUP> August</A></LI><LI class=""><A href="#infiltrate_Thursday_15th_August.html"> Thursday 15<SUP>th</SUP> August</A></LI><LI class=""><A href="#infiltrate_Friday_16th_August.html"> Friday 16<SUP>th</SUP> August</A></LI><LI class=""><A href="#infiltrate_Saturday_17th_August.html"> Saturday 17<SUP>th</SUP> August</A></LI><LI class=""><A href="#infiltrate_Sunday_18th_August.html"> Sunday 18<SUP>th</SUP> August</A></LI><LI class=""><A href="#infiltrate_Monday_19th_August.html"> Monday 19<SUP>th</SUP> August</A></LI><LI class=""><A href="#infiltrate_Tuesday_20th_August.html"> Tuesday 20<SUP>th</SUP> August</A></LI><LI class=""><A href="#infiltrate_Wednesday_21st_August.html"> Wednesday 21<SUP>st</SUP> August</A></LI><LI class=""><A href="#infiltrate_Thursday_22nd_August.html"> Thursday 22<SUP>nd</SUP> August</A></LI><LI class=""><A href="#infiltrate_Monday_26th_August.html"> Monday 26<SUP>th</SUP> August</A></LI><LI class=""><A href="#infiltrate_Tuesday_27th_August.html"> Tuesday 27<SUP>th</SUP> August</A></LI><LI class=""><A href="#infiltrate_Wednesday_28th_August.html"> Wednesday 28<SUP>th</SUP> August</A></LI><LI class=""><A href="#infiltrate_Thursday_29th_August.html"> Thursday 29<SUP>th</SUP> August</A></LI><LI class=""><A href="#infiltrate_Friday_30th_August.html"> Friday 30<SUP>th</SUP> August</A></LI><LI class="lastChild"><A href="#infiltrate_Saturday_31st_August.html"> Saturday 31<SUP>st</SUP> August</A></LI><LI class=" collapsibleList">
	  September
	  </LI><LI class=""><A href="#infiltrate_Monday_2nd_September.html"> Monday 2<SUP>nd</SUP> September</A></LI><LI class=""><A href="#infiltrate_Tuesday_3rd_September.html"> Tuesday 3<SUP>rd</SUP> September</A></LI><LI class=""><A href="#infiltrate_Wednesday_4th_September.html"> Wednesday 4<SUP>th</SUP> September</A></LI><LI class=""><A href="#infiltrate_Thursday_5th_September.html"> Thursday 5<SUP>th</SUP> September</A></LI><LI class=""><A href="#infiltrate_Friday_6th_September.html"> Friday 6<SUP>th</SUP> September</A></LI><LI class=""><A href="#infiltrate_Saturday_7th_September.html"> Saturday 7<SUP>th</SUP> September</A></LI><LI class=""><A href="#infiltrate_Monday_9th_September.html"> Monday 9<SUP>th</SUP> September</A></LI><LI class=""><A href="#infiltrate_Tuesday_10th_September.html"> Tuesday 10<SUP>th</SUP> September</A></LI><LI class=""><A href="#infiltrate_Wednesday_11th_September.html"> Wednesday 11<SUP>th</SUP> September</A></LI><LI class=""><A href="#infiltrate_Thursday_12th_September.html"> Thursday 12<SUP>th</SUP> September</A></LI><LI class=""><A href="#infiltrate_Friday_13th_September.html"> Friday 13<SUP>th</SUP> September</A></LI><LI class=""><A href="#infiltrate_Saturday_14th_September.html"> Saturday 14<SUP>th</SUP> September</A></LI><LI class=""><A href="#infiltrate_Sunday_15th_September.html"> Sunday 15<SUP>th</SUP> September</A></LI><LI class=""><A href="#infiltrate_Monday_16th_September.html"> Monday 16<SUP>th</SUP> September</A></LI><LI class=""><A href="#infiltrate_Tuesday_17th_September.html"> Tuesday 17<SUP>th</SUP> September</A></LI><LI class=""><A href="#infiltrate_Wednesday_18th_September.html"> Wednesday 18<SUP>th</SUP> September</A></LI><LI class="lastChild"><A href="#infiltrate_Friday_20th_September.html"> Friday 20<SUP>th</SUP> September</A></LI><LI class=" collapsibleList">
	  October
	  </LI><LI class=""><A href="#infiltrate_Tuesday_15th_October.html"> Tuesday 15<SUP>th</SUP> October</A></LI><LI class=""><A href="#infiltrate_Wednesday_16th_October.html"> Wednesday 16<SUP>th</SUP> October</A></LI><LI class=""><A href="#infiltrate_Thursday_17th_October.html"> Thursday 17<SUP>th</SUP> October</A></LI><LI class=""><A href="#infiltrate_Friday_18th_October.html"> Friday 18<SUP>th</SUP> October</A></LI><LI class=""><A href="#infiltrate_Saturday_19th_October.html"> Saturday 19<SUP>th</SUP> October</A></LI><LI class=""><A href="#infiltrate_Sunday_20th_October.html"> Sunday 20<SUP>th</SUP> October</A></LI><LI class=""><A href="#infiltrate_Monday_21st_October.html"> Monday 21<SUP>st</SUP> October</A></LI><LI class=""><A href="#infiltrate_Tuesday_22nd_October.html"> Tuesday 22<SUP>nd</SUP> October</A></LI><LI class=""><A href="#infiltrate_Saturday_26th_October.html"> Saturday 26<SUP>th</SUP> October</A></LI><LI class=""><A href="#infiltrate_Sunday_27th_October.html"> Sunday 27<SUP>th</SUP> October</A></LI><LI class="lastChild"><A href="#infiltrate_Monday_28th_October.html"> Monday 28<SUP>th</SUP> October</A></LI><LI class=" collapsibleListClosed">
      SABOTAGE
      </LI><LI class=" collapsibleList">
	  June
	  </LI><LI class=""><A href="#sabotage_Monday_24th_June.html"> Monday 24<SUP>th</SUP> June</A></LI><LI class=""><A href="#sabotage_Tuesday_25th_June.html"> Tuesday 25<SUP>th</SUP> June</A></LI><LI class="lastChild"><A href="#sabotage_Friday_28th_June.html"> Friday 28<SUP>th</SUP> June</A></LI><LI class=" collapsibleList">
	  July
	  </LI><LI class=""><A href="#sabotage_Tuesday_2nd_July.html"> Tuesday 2<SUP>nd</SUP> July</A></LI><LI class=""><A href="#sabotage_Wednesday_3rd_July.html">Wednesday 3<SUP>rd</SUP> July</A></LI><LI class=""><A href="#sabotage_Thursday_4th_July.html"> Thursday 4<SUP>th</SUP> July</A></LI><LI class=""><A href="#sabotage_Monday_15th_July.html"> Monday 15<SUP>th</SUP> July</A></LI><LI class=""><A href="#sabotage_Tuesday_16th_July.html"> Tuesday 16<SUP>th</SUP> July</A></LI><LI class=""><A href="#sabotage_Wednesday_17th_July.html"> Wednesday 17<SUP>th</SUP> July</A></LI><LI class=""><A href="#sabotage_Thursday_18th_July.html"> Thursday 18<SUP>th</SUP> July</A></LI><LI class=""><A href="#sabotage_Friday_19th_July.html"> Friday 19<SUP>th</SUP> July</A></LI><LI class=""><A href="#sabotage_Saturday_20th_July.html"> Saturday 20<SUP>th</SUP> July</A></LI><LI class=""><A href="#sabotage_Monday_22nd_July.html"> Monday 22<SUP>nd</SUP> July</A></LI><LI class=""><A href="#sabotage_Tuesday_23rd_July.html"> Tuesday 23<SUP>rd</SUP> July</A></LI><LI class=""><A href="#sabotage_Wednesday_24th_July.html"> Wednesday 24<SUP>th</SUP> July</A></LI><LI class=""><A href="#sabotage_Thursday_25th_July.html"> Thursday 25<SUP>th</SUP> July</A></LI><LI class=""><A href="#sabotage_Friday_26th_July.html"> Friday 26<SUP>th</SUP> July</A></LI><LI class=""><A href="#sabotage_Tuesday_30th_July.html"> Tuesday 30<SUP>th</SUP> July</A></LI><LI class="lastChild"><A href="#sabotage_Wednesday_31st_July.html"> Wednesday 31<SUP>st</SUP> July</A></LI><LI class=" collapsibleList">
	  August
	  </LI><LI class=""><A href="#sabotage_Thursday_1st_August.html"> Thursday 1<SUP>st</SUP> August</A></LI><LI class=""><A href="#sabotage_Monday_12th_August.html"> Monday 12<SUP>th</SUP> August</A></LI><LI class=""><A href="#sabotage_Tuesday_13th_August.html"> Tuesday 13<SUP>th</SUP> August</A></LI><LI class=""><A href="#sabotage_Wednesday_14th_August.html"> Wednesday 14<SUP>th</SUP> August</A></LI><LI class=""><A href="#sabotage_Thursday_15th_August.html"> Thursday 15<SUP>th</SUP> August</A></LI><LI class=""><A href="#sabotage_Friday_16th_August.html"> Friday 16<SUP>th</SUP> August</A></LI><LI class=""><A href="#sabotage_Saturday_17th_August.html"> Saturday 17<SUP>th</SUP> August</A></LI><LI class=""><A href="#sabotage_Monday_19th_August.html"> Monday 19<SUP>th</SUP> August</A></LI><LI class=""><A href="#sabotage_Tuesday_20th_August.html"> Tuesday 20<SUP>th</SUP> August</A></LI><LI class=""><A href="#sabotage_Wednesday_21st_August.html"> Wednesday 21<SUP>st</SUP> August</A></LI><LI class=""><A href="#sabotage_Thursday_22nd_August.html"> Thursday 22<SUP>nd</SUP> August</A></LI><LI class=""><A href="#sabotage_Friday_23rd_August.html"> Friday 23<SUP>rd</SUP> August</A></LI><LI class=""><A href="#sabotage_Monday_26th_August.html"> Monday 26<SUP>th</SUP> August</A></LI><LI class=""><A href="#sabotage_Tuesday_27th_August.html"> Tuesday 27<SUP>th</SUP> August</A></LI><LI class=""><A href="#sabotage_Wednesday_28th_August.html"> Wednesday 28<SUP>th</SUP> August</A></LI><LI class=""><A href="#sabotage_Thursday_29th_August.html"> Thursday 29<SUP>th</SUP> August</A></LI><LI class="lastChild"><A href="#sabotage_Friday_30th_August.html"> Friday 30<SUP>th</SUP> August</A></LI><LI class=" collapsibleList">
	  September
	  </LI><LI class=""><A href="#sabotage_Tuesday_3rd_September.html"> Tuesday 3<SUP>rd</SUP> September</A></LI><LI class=""><A href="#sabotage_Wednesday_4th_September.html"> Wednesday 4<SUP>th</SUP> September</A></LI><LI class=""><A href="#sabotage_Thursday_5th_September.html"> Thursday 5<SUP>th</SUP> September</A></LI><LI class=""><A href="#sabotage_Saturday_7th_September.html"> Saturday 7<SUP>th</SUP> September</A></LI><LI class=""><A href="#sabotage_Tuesday_10th_September.html"> Tuesday 10<SUP>th</SUP> September</A></LI><LI class=""><A href="#sabotage_Wednesday_11th_September.html"> Wednesday 11<SUP>th</SUP> September</A></LI><LI class=""><A href="#sabotage_Saturday_14th_September.html"> Saturday 14<SUP>th</SUP> September</A></LI><LI class=""><A href="#sabotage_Sunday_15th_September.html"> Sunday 15<SUP>th</SUP> September</A></LI><LI class=""><A href="#sabotage_Wednesday_18th_September.html"> Wednesday 18<SUP>th</SUP> September</A></LI><LI class="lastChild"><A href="#sabotage_Friday_20th_September.html"> Friday 20<SUP>th</SUP> September</A></LI></DIV><DIV id="rightscroll"><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Saturday 13<SUP>th</SUP> July</H3><P><B><EM>
PCR circular and linear, Conjugation of XL-10 (with sSP011), Patches, Digestion M13 backbone, RFP insert, Gel of PCR 
</EM></B></P><B>PCR circular and linear, 40.5°C, 60°</B>
of pSB1A3, pSB1C3<TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD><B>1x</B></TD></TR><TR><TD>Nuclease-free water</TD><TD>37.25 ul</TD></TR><TR><TD>5x Phusion HF Buffer</TD><TD>10 ul</TD></TR><TR><TD>10 mM dNTPs</TD><TD>1 ul</TD></TR><TR><TD>Forward Primer (10 uM)</TD><TD>0.5 ul</TD></TR><TR><TD>Reverse Primer (10 uM)</TD><TD>0.5 ul</TD></TR><TR><TD>Template Plasmid</TD><TD>0.25 ul</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>0.5 ul</TD></TR><TR><TD><B>Total Volume</B></TD><TD>50 ul</TD></TR></TBODY></TABLE><TABLE border=""><TBODY><TR><TD><B>Thermocycler Protocol: NEB Phusion</B></TD></TR><TR><TD>Temp</TD><TD>Time</TD></TR><TR><TD>Start</TD><TD>98°C</TD><TD>30 sec</TD><TD>Melt</TD></TR><TR><TD>Cycle 1</TD><TD>98°C</TD><TD>5 sec</TD><TD>Melt</TD><TD> 35 cycles </TD></TR><TR><TD>Cycle 2</TD><TD>40.5°C / 60°C </TD><TD>25 sec</TD><TD>Anneal</TD></TR><TR><TD>Cycle 3</TD><TD>72°C</TD><TD>5 min</TD><TD>Extend</TD></TR><TR><TD>Finish</TD><TD>72°C</TD><TD>5 min</TD><TD>Extend</TD></TR><TR><TD>Store</TD><TD>10°C</TD><TD>Forever</TD><TD>Store</TD></TR></TBODY></TABLE><B>Conjugation of XL-10 (with sSP011)</B>
1)   From O/N cultures Dilute strains 1/100 in LB
2)   Wait for OD to reach O,2
3)   Prepare  tube (in BD tubes) :
-       Tube = 0,5mL LB with Strain (sSP011) + 0,5mL LB with Strain (XL-10 Kan)
4)   Incubate 2 hours at 37°C (actually not in the shaker, but we accidently kept them in the shaker...)
5)   Plate 20ul  for mixed tube on LB antiobiotics (Tet, Kan)
6)   Incubate overnight at 37°C<B>Patches</B>
check and make new ones<B>Digestion M13 backbone, RFP insert</B>
for Backbone (M13mp18 plasmid): 3ug
7,58 ul plasmid (c=395ng/ul)
3 ul EcoRI
3 ul PstI
3ul 10x Fast Digest
13,42 ul H20
incubate for 12 min on 37°
heat inactivation: 80° 5 min
for insert (BBa_J04450): 5ug
31,4 ul plasmid
5 ul EcoRi
5 ul PstI
3,6 ul H20
incubate for 20 min at 37°
heat inactivation: 80° 5min<B>Gel of PCR </B> (Amp 40.5 circ, lin, Chl 60° lin, circ)
100V, 20 min 1% gel</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Sunday 14<SUP>th</SUP> July</H3><P><B><EM>
Gel of PCR, Gel Extraction, DpnI digest of PCR products and PCR purification
</EM></B></P><B>Gel of PCR</B> (Amp 60° circ, lin, Chl 40.5 circ, lin)
100V, 20 min 1% gel<B>Gel of Digest</B>
100V, 20 min 1% gel<B>Gel Extraction</B>
Excise the DNA fragment from the agarose gel with a clean, sharp scalpel. Minimize the size of the gel slice by removing extra agarose.
Weigh the gel slice in a colorless tube. Add 3 volumes of Buffer QG to 1 volume of gel (100 mg ~ 100 μl). To help dissolve gel, mix by vortexing the tube every 2–3 min during the incubation.
After the gel slice has dissolved completely, check that the color of the mixture is yellow (similar to Buffer QG without dissolved agarose).
Add 1 gel volume of isopropanol to the sample and mix (actually only needed for very small products or very big products)
To bind DNA, apply the sample to the QIAquick column, and centrifuge for 1 min.
Discard flow-through and place QIAquick column back in the same collection tube.
To wash, add 0.75 ml of Buffer PE to QIAquick column and centrifuge for 1 min.
11. Discard the flow-through and centrifuge the QIAquick column for an additional 1 min at 17,900 x g (13,000 rpm).
12. Place QIAquick column into a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 30 μl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge for 1 min.<B>DpnI digest of PCR products</B>:
SPCR4 (40ul) - add 1 ul DpnI
SPCR 7 (150ul) - add 3 ul of DpnI
SPCR8 (60 ul) - add 3 ul of DpnI
SPCR 9 (150ul) - add 3 ul of DpnI
SPCR10 (110ul) - add 3 ul of DpnI
Incubate for 15 min at 37°C<B>PCR purification</B>
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 50 μl Buffer EB  to the center of the QIAquick membrane and centrifuge the column for 1 min.</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Monday 15<SUP>th</SUP> July</H3><P><B><EM>
Plating and control of antibiotic resistance.
</EM></B></P>
We made a 1:1000 dilution of KAN (stand.c) in an LB solid media.
Pouring the plates. Around 20 mL/plate
	1. KEIOΔPYR KANR	Growth
	2. ME1655 KANR	No growth
Conclusion : 
The KEIOΔPYRF growed as expected. Negative control did not growth.
Plates have KAN antibiotic
KEIO has KAN R.</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Tuesday 16<SUP>th</SUP> July</H3><P><B><EM>
We prepared a colony PCR to send the KAN for sequencing, to know exactly the sequence of the KAN.
</EM></B></P>

KEIOΔMPYRF has a deletion of PYRF to be replaced by KAN
	We pitched 4 singles colonies into 50µL of H2O
	Boil 5 minutes at 95°C
	1,5 µL of this can be used directly for PCR
PCR reaction
Keep all the regents at 4°C while preparing the mixture<TABLE border=""><TBODY><TR><TD>Reagent</TD><TD>Volume</TD></TR><TR><TD>JW 182 (10 uM)
JW 183 (10 uM)
Template DNA
Quick-load Tag 2x Master Mix
Nuclease free water</TD><TD>0,5 µL
0,5 µL
1,5 µL
12,5 µL
10 µL</TD></TR><TR><TD>Total volume</TD><TD>25 µL</TD></TR></TBODY></TABLE>
Thermocycler Protocol : NEB Quick-Load<TABLE border="1"><TBODY><TR><TD> Temperature </TD><TD> Time </TD></TR><TR><TD>Start
Cycle1
Cycle 2
Cycle 3
Finish
Store</TD><TD>95°C
95°C
50°C
68°C
68°C
10°C</TD><TD>30 seconds
15 seconds
30 seconds
1 minute/kB
5 minutes
Forever</TD><TD>melt
melt
anneal
extend - 35 cycles
extend
store</TD></TR></TBODY></TABLE>
Gel electrophoresis
We do it to prove that our colony PCR was successful. We expect a band at about 800 base pairs.
Our gel is 1%, therefore we used 0,5g agarose in 100µL TAE buffer.
As a ladder, we used a 1kB plus gene ruler of fermentor. 
We l the gel with 5 µL sample ans we kept the sample at 4°C.
PICTURE
We see that the band are as expected. THis is an indicator that KAN is at the right place. The sequence has then been sent for sequencing
Transformation of PAUC 18 into a NEBΔturbo clearing cells
	We are transforming PAUC 18 that consist in a low ORI and ampicillin resistance
	We will use commercialized NEB turbo competent cells as well as freshly made chemical competent cells
	We did this to check the competency of our fresh competent cells
NOTE : EVERYTHING HAS TO BE KEPT ON ICE AND NO VORTEX
	Throw competent cells on ice. Those can be prepared using the CaCl2 protocol
	Place 20 µL of cells in a pre-chilled Eppendorf tube
	For an intert vector, add 0,5 µL or less to the chilled cells
	For a ligation product, add 2-3 µL to the chilled cells
	Mix gently by flicking the tube
	Chill on ice for 10 minutes - this step is optional but can improve yields when transforming a ligation product
	Heat shock at 42°C for 30 seconds
	Return on ice for 2 minutes
	Add 200µL LB medium and recover the cells by shaking at 37°C
Another rich medium can substitute for the recovery. The recovery time varies with the antibiotic selection
		Ampicillin : 15 - 30 minutes
	Place out the cells on selective LB
	Use glass beads to spread the cells. The volume of cells plated depends on what is being transformed
For an intact vector
High transformation efficiencies are expected. Plating out 10 µL of recovered cells should produce many colonies.
Note : 200 µL is the maximum volume of liquid that an LB plate can absorb
	Incubate at 37°C. Transformants should appear within 12 hours
Conclusion
The BL 21 DE3 strain showed IP colonies
For the new NEB turbo cells, we see no colony
The old NEB turbo cells worked fine</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Wednesday 17<SUP>th</SUP> of July</H3><P><B><EM>
PCR purification
</EM></B></P>
We are purifying the colony PCR from yesterday to send it away for sequencing
Procedure
	Add 1.1 volume and Binding Buffer
	Transfer up to 800 µL. Centrifuge for 30-60 seconds. Descend the flow through
	Add 700 µL of washed Buffer. Centrifuge for 30-60 seconds
+ 1 min centrifuge to completely remove any residual wash buffer
Add 50 µL of Elution Buffer to the center of the Gene JT purification column membrane and centrifuge for 1 minute
Discard the Gene JET purification column and store the purified DNA at -20°C.	</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Thursday 18<SUP>th</SUP> July</H3><P><B><EM>
Calculating the transformation efficiency for the E.coli NEB and E.coli NNEB strains.
</EM></B></P>
·         20 µL of cell culture was inoculated in 200 µL of LB, where 0.5 µL of DNA sample (?) was added, giving the total volume of 220.5 µL.
·         After the heat shock transformation 10 µL of the culture was diluted in 90 µL of LB,  giving the 10-1 dilution (d); then 10 µL of this dilution was transferred in 90 µL of LB, giving the 10-2 dilution. 10 µL of each dilution (v) was plated on ampicillin media and left for incubation (24 h, 37  ̊C).
·         After incubation, colonies were counted:
<TABLE border="1"><TBODY><TR><TD> NEB </TD><TD> NNEB </TD></TR><TR><TD> 10<SUP>-¹</SUP></TD><TD> 0 </TD><TD> 433 </TD></TR><TR><TD> 0 </TD><TD> 368 </TD></TR><TR><TD> Average </TD><TD> 0 </TD><TD><B>400.5</B></TD></TR><TR><TD> 10<SUP>-¹</SUP></TD><TD> 0 </TD><TD> 62 </TD></TR><TR><TD> 0 </TD><TD> 40 </TD></TR><TR><TD> Average </TD><TD> 0 </TD><TD><B>51</B></TD></TR></TBODY></TABLE>
Obviously, NEB strain was not transformed and only NNEB was included in further calculations.
·         The number of the transformed colony forming units per mL of the culture was calculated following the formula: colony number / d*v.
Calculation from the 10-1 dilution: 0.4 * 109 CFU/mL
Calculation from the 10-2 dilution: 0.51 * 109 CFU/mL
·         The transformation efficiency was calculated by dividing the number of transformed colony forming units (CFU) by the amount of the DNA used (mDNA).
CFU1 = 0.4 * 109 CFU/mL  *  0.22 mL (total V of the transformation culture) = 0.088*109 CFU
CFU2 = 0.51 * 109 CFU/mL * 0.22 mL = 0.1122*109 CFU
The concentration of the DNA solution which was used was 0.323 µg/µL, while the total DNA mass which was added to the transformation culture was: 0.323 µg/µL * 0.5 µL = 0.1615 µg.
So the transformation efficiency, calculated from the 10 times diluted (TE1) and 100 times diluted (TE2) cultures is:
TE1 = CFU1/mDNA = 0.55 * 109 CFU/µg
TE2 = CFU2/mDNA = 0.69 * 109 CFU/µg
TE1 and TE2 are almost the same</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Monday 22<SUP>nd</SUP> July</H3><P><B><EM>
Culture purification by streak plate method and preparing the glycerol stock for the sSP001 (TN 57) and sSP002 (NEC) strains.
</EM></B></P>
Place your note here
</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Tuesday 23<SUP>rd</SUP> July</H3><P><B><EM>
Colony PCR
</EM></B></P>
·         Strain: Keio ΔpyrF
·         Primers: JW182 and JW183
·         Amplified region: kanamycine resistance gene</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Monday 29<SUP>th</SUP> July</H3>
Liquid culture of:
pCOLA DUET (from glycerol stock of Sabotage Group) - 20 ml culture
pSSB1 (from plate) - 5 ml culture
pSSB2 (from plate ) - 5 ml culture
Plating of addgene bacteria containing the plasmids we want:
streak out bacteria from addgene to get single colonies:
pBAC-BA-lacZ
pCRISPR
pCas9
pCRIPSR::rpsL</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Tuesday 30<SUP>th</SUP> July</H3><P><B><EM>
Liquid culture, Miniprep and Glycerol stock
</EM></B></P><B>Liquid culture of:</B>
pCOLA DUET (from liquid culture (29.07.13)) - 20 ml culture
pSSB1 (from liquid culture (29.07.13)) - 5ml culture
pSSB2 (from liquid culture (29.07.13)) - 5 ml culture
pBAC-BA-lacZ (from plate ) - 5 ml culture
pCRISPR (from plate ) - 5 ml culture
pCas9 (from plate ) - 5 ml culture
pCRIPSR::rpsL (from plate ) - 5 ml culture
F+ strain - (from glycerol stock Sabotage) - 10 ml culture
Keio Keio delta pyrF (from plate ) - 10 ml culture<B>Miniprep (pCOLA-DUET = sSP007) using the Thermo Scientific Miniprep Kit</B>
1) Pellet 2x9 ml of liquid culture (4000 rpm, 10 min)
2) Discard supernatant
3) resuspend the cells in 250 µL of resuspension solution
4) add 250 µL of lysis solution, mix by inverting 4-6 times
5) add 350 µL of neutralization solution
4) centrifuge for 5 min
5) transfer supernatant to spin column
6) centrifuge for 1 min
7) discard flow through
8) add 500 µL wash solution and centrifuge for 1 min , discard flow through(repeat this step)
9) centrifuge for 1 min to remove left over liquid
10) transfer the column on a 1.5 ml tube
11) add 50 µL of elution buffer and incubate for 2 min
12) centrifuge for 2 min
13) Nanodrop the concentration and freeze at -20°
pSP001= pCOLA-DUET: 65 ng/ul<B>Glycerol Stock</B>
 from overnight culture of MG1655-pCOLA-DUET (sSP007)
Centrifuge 4000 rpm, 10 minutes,
take out liquid
resuspend cells in 0,5 mL glycerol (60%) , 2 mL LB
freeze in -80°C</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Wednesday 31<SUP>st</SUP> July</H3><P><B><EM>
Conjugation, Miniprep addgene plasmids,Glycerol Stock of addgene plasmids
</EM></B></P><B>Conjugation</B>
sSP001, SP002, keio Delta PyrF
F+ comes from XL1 Blue (glycerol stock Sabotage) =&gt; F+ is tetR
Protocol :
1)   From O/N cultures Dilute strains 1/100 in LB
2)   Wait for OD to reach O,2
3)   Prepare 4 tubes (in BD tubes) :
-       Tube 1 = 0,5 mL LB with Strain 1 (sSP001) ,5 mL LB = control
-       Tube 2 = 0,5 mL LB with Strain 2 (sP001) + 0,5 mL LB = control
-       Tube 3 = 0,5 mL LB with Strain 3 (keio Delta pyrF) + 0,5 mL LB = control
-       Tube 4 = 0,5 mL LB with Strain 4 (XL1) + 0,5 mL LB = control
-       Tube 5 = 0,5 mL LB with Strain 1 (sSP001) + 0,5 mL LB with Strain 4 (XL1)
-       Tube 6 = 0,5 mL LB with Strain 2 (sSP002) + 0,5 mL LB with Strain 4 (XL1)
-       Tube 7 = 0,5 mL LB with Strain 3 (keio Delta pyrF) + 0,5 mL LB with Strain 4 (XL1)
4)   Incubate 2 hours at 37°C (actually not in the shaker, but we accidently kept them in the shaker...)
5)   Plate 10ul for controls, 100uL, 10ul  for mixed tubes on LB antibiotics (Tube 5: Chl + Tet, Tube 6: Chl + Tet, Tube 7: Kan + Tet)
6)   Incubate overnight at 37°C<B>Miniprep addgene plasmids</B>
1) Pellet  4 ml of liquid culture (4000 rpm, 10 min)
2) Discard supernatant
3) resuspend the cells in 250 µL of resuspension solution
4) add 250 µL of lysis solution, mix by inverting 4-6 times
5) add 350 µL of neutralization solution
4) centrifuge for 5 min
5) transfer supernatant to spin column
6) centrifuge for 1 min
7) discard flow through
8) add 500 µL wash solution and centrifuge for 1 min , discard flow through(repeat this step)
9) centrifuge for 1 min to remove leftover liquid
10) transfer the column on a 1.5 ml tube
11) add 50 µL of elution buffer and incubate for 2 min
12) centrifuge for 2 min
13) Nanodrop the concentration and freeze at -20° 
See Database for plasmid reference
pCas9: 
pCRISPR:
pBac-LacZ:
pCRISPR::rpsL:<B>Glycerol Stock of addgene plasmids</B> (see Database for strain reference)
 from overnight culture
Centrifuge 4000 rpm, 10 minutes,
take out liquid
resuspend cells in 0,5 mL glycerol (60%) , 2 mL LB
freeze in -80°C</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Thursday 1<SUP>st</SUP> August</H3><P><B><EM>
PCR (gradient + usual) of SPCR1, SPCR7, SPCR8, SPCR10, SPCR9, SPCR4
</EM></B></P><TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD><B>1x</B></TD></TR><TR><TD>Nuclease-free water</TD><TD>37.25 µL</TD></TR><TR><TD>5x Phusion HF Buffer</TD><TD>10 µL</TD></TR><TR><TD>10 mM dNTPs</TD><TD>1 µL</TD></TR><TR><TD>Forward Primer (10 µM)</TD><TD>0.5 µL</TD></TR><TR><TD>Reverse Primer (10 µM)</TD><TD>0.5 µL</TD></TR><TR><TD>Template Plasmid</TD><TD>0.25 µL</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>0.5 µL</TD></TR><TR><TD><B>Total Volume</B></TD><TD>50 µL</TD></TR></TBODY></TABLE>
Gradient:
Pos 1: 40°
Pos 2: 40,2°
Pos 3: 41,3°
Pos 4: 43,1°
Pos 5: 45,4°
Pos 6: 48°
Pos 7: 50,7°
Pos 8: 53,5°
Pos 9: 56,0°
Pos 10: 58,1°
Pos 11: 59,8°C
(Pos 12: 60,6°)
Usual PCR: 52°</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Friday 2<SUP>nd</SUP> August</H3><P><B><EM>
Place your twit here
</EM></B></P>
Gels with PCR products: 1% Agar
100V 20 Min
10 min staining in BET</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Monday 5<SUP>th</SUP> August</H3><P><B><EM>
Place your twit here
</EM></B></P><B>5ml liquid culture with proper antibiotics of:</B>
SSp001 + F+
sSP002 + F+
keio delta pyrF + F+<B>Gelelectrophoresis:</B>
1% Agar, 100 V 20 min</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Tuesday 6<SUP>th</SUP> August</H3><P><B><EM>
Liquid cultures of sSP008-sSP0011
</EM></B></P></DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Wednesday 7<SUP>th</SUP> August</H3><P><B><EM>
Glycerol Stock of sSP008-sSP0011, M13 Test, Single Colonies, Liquid cultures, PCR and Transformation
</EM></B></P><B>Glycerol Stock of sSP008-sSP0011</B>
Take 1.5ml of ON culture + 500ul Glycerol (60%)
Freeze at -80°C<B>M13 Test</B>
Plate from ON culture:
1) 1:1000 - sSP012: sSP009 (plate 200ul)
2) 200 ul sSP012
3) 200ul sSp009<B>Single Colonies</B>
streak out ON culture of sSP012 to get single colonies<B>Liquid cultures</B> of pir+ E.coli and pKD13 from Jake<B>PCR of SPCR4, SPCR7, SPCR8, SPCR9, SPCR10 from circuar backbone, directly taken from Biobrick plate</B>
Protocol<TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD><B>1x</B></TD></TR><TR><TD>Nuclease-free water</TD><TD>37,25µL</TD></TR><TR><TD>5x Phusion HF Buffer</TD><TD>10µL</TD></TR><TR><TD>10 mM dNTPs</TD><TD>1µL</TD></TR><TR><TD>Forward Primer (10 µM)</TD><TD>0.5µL</TD></TR><TR><TD>Reverse Primer (10 µM)</TD><TD>0.5µL</TD></TR><TR><TD>Template Plasmid</TD><TD>0.25µL</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>0.5µL</TD></TR><TR><TD><B>Total Volume</B></TD><TD>50µL</TD></TR></TBODY></TABLE><TABLE border=""><TBODY><TR><TD><B>Thermocycler Protocol: NEB Phusion</B></TD></TR><TR><TD>Temp</TD><TD>Time</TD></TR><TR><TD>Start</TD><TD>98°C</TD><TD>30 sec</TD><TD>Melt</TD></TR><TR><TD>Cycle 1</TD><TD>98°C</TD><TD>5 sec</TD><TD>Melt</TD><TD>35 cycles</TD></TR><TR><TD>Cycle 2</TD><TD>40.5°C / 60°C</TD><TD>25 sec</TD><TD>Anneal</TD></TR><TR><TD>Cycle 3</TD><TD>72°C</TD><TD>30 sec per kb</TD><TD>Extend</TD></TR><TR><TD>Finish</TD><TD>72°C</TD><TD>5 min</TD><TD>Extend</TD></TR><TR><TD>Store</TD><TD>10°C</TD><TD>Forever</TD><TD>Store</TD></TR></TBODY></TABLE><B>Transformation</B> Biobricks BBa_K649304 (with pSB1C3 backbone) and BBa_J04450 (with pSB1A3 backbone) into NEB Turbo (heat shock trafo)
1) Thaw competent cells on ice.
2) Place 20 ul of cells in a pre-chilled Eppendorf tube.
3) Add 2.5 ul of plasmid (from Biobrick stock)
4) Mix gently by flicking the tube.
5) Chill on ice for 10 minutes.
4) Heat shock at 42 °C for 30 seconds 
5) Return to ice for 2 minutes.
6) Add 200 ul LB medium and recover the cells by shaking at 37 °C.
                Ampicillin: 15-30 minutes
                Chloramphenicol: 60-120 minutes
7) Plate out the cells on selective LB (10ul)
8) Incubate at 37 °C. Transformants should appear within 12 hrs.<B>Result</B>:  BBa_J04450: many colonies, BBa_K649304 no colonies, plate rest</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Thursday 8<SUP>th</SUP> August</H3><P><B><EM>
Liquid culture of FR-433 (M13)
</EM></B></P>
Gel of PCR products. 100V 20min 1%
4-7-8-9-10 (40.5°C)- 4-7-8-9-10 (60°C)</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Friday 9<SUP>th</SUP> August</H3><P><B><EM>
Liquid culture, Gel, 
Trafo of BBa_E1010 (pSB1C3 backbone) into NEB turbo, Glycerol Stock, Making electro competent cells of pir+ E.coli, Transforming pKD13 into pir+ E.coli, Miniprep NEB turbo with BBa-J04450:, Liquid culture and Patch plates. 
</EM></B></P><B>Liquid culture</B>
1:1000 from liquid: FR-433 (M13), 10 ml
pir+ E.coli: from plate, 25 ml
XL10: from plate 10 ml<B>Gel</B>: rest of circular backbone PCR product, 1% Agar, 100V, 20 min
SPCR4-7-8-9-10 (40.5) SPCR4-7-8-9-10 (60.5)
Gel picture: 
didn’t work<B>Trafo of BBa_E1010 (pSB1C3 backbone) into NEB turbo</B>
1) Thaw competent cells on ice.
2) Place 20 µl of cells in a pre-chilled Eppendorf tube.
3) Add 2.5 µl of plasmid (from Biobrick stock)
4) Mix gently by flicking the tube.
5) Chill on ice for 10 minutes.
4) Heat shock at 42 °C for 30 seconds 
5) Return to ice for 2 minutes.
6) Add 200 µl LB medium and recover the cells by shaking at 37 °C.
        	Chloramphenicol: 60-120 minutes
7) Plate out the cells on selective LB (10ul)
8) Incubate at 37 °C. Transformants should appear within 12 hrs.<B>Glycerol Stock FR-433 (M13) -&gt; sSP012</B>
Glycerol Stock pir+ -&gt; sSP013
Glycerol Stock pKD13 -&gt; sSP014
Glycerol Stock NEB turbo with BBa-J04450 -&gt; sSP015
Take 1.5 ml of ON culture + 500ul Glycerol (60%)
Freeze at -80°C<B>Making electro competent cells of pir+ E.coli</B>
Preparation of Electrocompetent Cells
        	Note: Competent cells should never be vortexed, as this will cause them to lyse and release salts into the media. Resuspend cells by pipetting up and down with a large pasteur pipet. Once they are chilled, cells should be continuously cold.
1) The night before the transformation, start an overnight culture of cells.
        	pir+ E.coli
2) The day of the transformation, dilute the cells 100X.
        	100 ml LB
        	Grow at 30°C for about 90 minutes. 
3) When the cells reach an OD600 of 0.2. 
4) Harvest the cells.
        	When the cells reach an OD600 of between 0.6 and 0.8.
        	Split the culture into 2x 50 ml falcon tubes, on ice.
        	Centrifuge at 4 °C for 10 min at 4000 rpm.
5) Wash and combine the cells.
        	Remove the supernatant.
        	Resuspend the cells in 2x 25 ml of ice cold water.
        	Combine the volumes in a single 50 ml falcon tube.
6) Wash the cells 2 more times.
        	Centrifuge at 4 °C for 10 min at 4000 rpm.
        	Resuspend in 50 ml of ice cold water.
        	Repeat.
7) Wash and concentrate the cells for electroporation.
        	Centrifuge at 4 °C for 10 min at 4000 rpm.
        	Resuspend in 1-2 ml of ice cold water.
        	We will use 200 µl of washed cells per transformation.<B>Transforming pKD13 into pir+ E.coli</B>
1) Prepare BD tubes with a pipette filled with LB at the interior of each tube (pipette supplied with the electroporation cuvette)
2) Test the purity of the electrocompetent cells.
        	Add 200 ul of washed cells to a cuvette.
3) Mix the cells and DNA in a cuvette.
        	200 ul of washed cells with 200 ng of PCR product.
        	Keep the cuvette on ice until just before the electroporation.
4) Preload a pipette with 1 ml of LB.
5) Pulse the cuvette with voltage.
        	Dry the electrodes with a kimwipe prior to loading.
        	Use the EC2 setting.
6) Listen for arcing.
        	A cracking sound means all the cells are dead.
        	Note the time constant: 5 is good, 5.8 is great.
7) Immediately recover the cells.
        	Add the 1 ml of preloaded LB and pipet up and down to mix.
        	Collect 1 ml of cells, some volume is lost in the cuvette.
8) Incubate 2 h at 37 °C with shaking.
9) Plate 100 ul of recovered cells on selective plates.
        	Use antibiotic appropriate to the part being integrated.
        	Let the other 900 ul rest overnight at room temperature.
10) Concentrate and plate the remaining cells
        	Spin down quickly and resuspend in 100 ul LB before plating.
Transformed cells should be incubated at 37 °C.
Colonies should appear 24-48 h after plating.<B>Miniprep NEB turbo with BBa-J04450:</B>
1) Pellet 2x 9 ml of liquid culture (4000rpm, 10 min)
2) Discard supernatant
3) resuspend the cells in 250 µl of resuspension solution
4) add 250 µl of lysis solution, mix by inverting 4-6 times
5) add 350 µl of neutralization solution
4) centrifuge for 5 min
5) transfer supernatant to spin column
6) centrifuge for 1 min
7) discard flow through
8) add 500 µl wash solution and centrifuge for 1 min , discard flow through(repeat this step)
9) centrifuge for 1 min to remove left over liquid
10) transfer the column on a 1.5 ml tube
11) add 50ul of elution buffer and incubate for 2 min
12) centrifuge for 2 min
13) Nanodrop the concentration and freeze at -20°
J04450: 0ng/ul<B>Liquid culture  of:</B>
sSP012
sSP008
sSP009
sSP010
sSP011
sSP001
sSP002
XL-10 = sSP016
sSP017= Litmus<B>Patch plates of:</B>
sSP010
sSP013
sSP009s
sSP014
sSP012</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Saturday 10<SUP>th</SUP> August</H3><P><B><EM>
Test of M13 and Liquid culture
</EM></B></P><B>Test of M13</B>
1) 100 μ l of plating bacteria per tube
2) Prepare tenfold serial dilutions (10-6  to 10-9 ) of the bacteriophage stock in LB
3) Put 10 μ l/ 100 μ l of each dilution to the bacteria
4)  Mix the bacteriophage particles with the bacterial culture by vortexing gently.
5) Add 40 μ l of X-gal solution (working conc) and IPTG (working conc) solution to each of the tubes
6) Add 3ml of Agar to each tube and gently vortexing for 3 seconds
7) Pour the mixture onto  plates containing LB medium supplemented with 5 mM MgCl2 
8) Swirl the plate gently to ensure an even distribution of bacteria and agar.
9) Incubate them at 37°C.<B>Liquid culture:</B>
E1010
J04550</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Sunday 11<SUP>th</SUP> August</H3><P><B><EM>
Patches of sSP001-sSP008, sSP015-18 and Stress experiment (Plate reader)
</EM></B></P><B>Patches of sSP001-sSP008, sSP015-18</B><B>Stress experiment (Plate reader)</B>
sSP001, sSP002, sSP010, sSP011, sSP012 (M13)
1:1000 bacteria in M9 Glucose
Mitomycin C concentrations: 3,3*10^-3, 1,6*10°-3, 6,6*10°-4, 3,3*10°-4
Quadruplets of bacteria (+phage), Triplets of MMC
See also pipetting scheme in folder Experiments on Dropbox<B>On culture of FR-433, XL-10, delta pyrF </B>
1:1000 culture at 30°</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Monday 12<SUP>th</SUP> August</H3><P><B><EM>
Glycerol Stock, Miniprep, M13 plate test, Patches
</EM></B></P><B>Glycerol Stock</B>
sSP018 - NEB with E1010
sSP017
sSP016
Take 1.5ml of ON culture + 500ul Glycerol (60%)
Freeze at -80°C<B>Miniprep</B>
E1010
J04450
1) Pellet 2x 2ml of liquid culture (4000rpm, 10 min)
2) Discard supernatant
3) resuspend the cells in 250ul of resuspension olution
4) add 250ul of lysis solution, mix by inverting 4-6 times
5) add 350ul of neutralization solution
4) centrifuge for 5 min
5) transfer supernatant to spin column
6) centrifuge for 1 min
7) discard flow through
8) add 500 ul wash solution and centrifuge for 1 min , discard flow through(repeat this step)
9) centrifuge for 1 min to remove left over liquid
10) transfer the column on a 1.5ml tube
11) add 50ul of elution buffer and incubate for 2 min
12) centrifuge for 2 min
13) Nanodrop the concentration and freeze at -20°
J04450: ng/ul
E1010: ng/ul<B>M13 plate test</B>
1) 100 μ l of plating bacteria per tube
2) Prepare tenfold serial dilutions (10-6  to 10-9 ) of the bacteriophage stock in LB
3) Put 10 μ l/ 100 μ l of each dilution to the bacteria
4)  Mix the bacteriophage particles with the bacterial culture by vortexing gently.
5) Add 40 μ l of X-gal solution (working conc) and IPTG (working conc) solution to each of the tubes
6) Add 3ml of Agar to each tube and gently vortexing for 3 seconds
7) Pour the mixture onto  plates containing LB medium supplemented with 5 mM MgCl2 
8) Swirl the plate gently to ensure an even distribution of bacteria and agar.
9) Incubate them at 37°C.<B>Patches</B>
check them and prepare new ones of those that haven’t been done yet</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Tuesday 13<SUP>th</SUP> August</H3><P><B><EM>
PCR circular and linear 40.5°C/ 60°C, Conjugation of XL-10 (with sSP011), Patches, Digestion M13 backbone, RFP insert and Gel of PCR
</EM></B></P><B>PCR circular and linear, 40.5°C, 60°</B>
of pSB1A3, pSB1C3<TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD><B>1x</B></TD></TR><TR><TD>Nuclease-free water</TD><TD>37,25µL</TD></TR><TR><TD>5x Phusion HF Buffer</TD><TD>10µL</TD></TR><TR><TD>10 mM dNTPs</TD><TD>1µL</TD></TR><TR><TD>Forward Primer (10 µM)</TD><TD>0.5µL</TD></TR><TR><TD>Reverse Primer (10 µM)</TD><TD>0.5µL</TD></TR><TR><TD>Template Plasmid</TD><TD>0.25µL</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>0.5µL</TD></TR><TR><TD><B>Total Volume</B></TD><TD>50µL</TD></TR></TBODY></TABLE><TABLE border=""><TBODY><TR><TD><B>Thermocycler Protocol: NEB Phusion</B></TD></TR><TR><TD>Temp</TD><TD>Time</TD></TR><TR><TD>Start</TD><TD>98°C</TD><TD>30 sec</TD><TD>Melt</TD></TR><TR><TD>Cycle 1</TD><TD>98°C</TD><TD>5 sec</TD><TD>Melt</TD><TD>35 cycles</TD></TR><TR><TD>Cycle 2</TD><TD>40.5°C / 60°C</TD><TD>25 sec</TD><TD>Anneal</TD></TR><TR><TD>Cycle 3</TD><TD>72°C</TD><TD>30 sec per kb</TD><TD>Extend</TD></TR><TR><TD>Finish</TD><TD>72°C</TD><TD>5 min</TD><TD>Extend</TD></TR><TR><TD>Store</TD><TD>10°C</TD><TD>Forever</TD><TD>Store</TD></TR></TBODY></TABLE><B>Conjugation of XL-10 (with sSP011)</B>
1)   From O/N cultures Dilute strains 1/100 in LB
2)   Wait for OD to reach O,2
3)   Prepare  tube (in BD tubes) :
-       Tube = 0,5mL LB with Strain (sSP011) + 0,5mL LB with Strain (XL-10 Kan)
4)   Incubate 2 hours at 37°C (actually not in the shaker, but we accidently kept them in the shaker...)
5)   Plate 20ul  for mixed tube on LB antiobiotics (Tet, Kan)
6)   Incubate overnight at 37°C<B>Patches</B>
check and make new ones<B>Digestion M13 backbone, RFP insert</B>
for Backbone (M13mp18 plasmid): 3ug
7,58 ul plasmid (c=395ng/ul)
3 ul EcoRI
3 ul PstI
3ul 10x Fast Digest
13,42 ul H20
incubate for 12 min on 37°
heat inactivation: 80° 5 min
for insert (BBa_J04450): 5ug
31,4 ul plasmid
5 ul EcoRi
5 ul PstI
3,6 ul H20
incubate for 20 min at 37°
heat inactivation: 80° 5min<B>Gel of PCR (Amp 40.5 circ, lin, Chl 60° lin, circ)</B>
100V, 20 min 1% gel</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Wednesday 14<SUP>th</SUP> August</H3><P><B><EM>
Gel of PCR, Gel of Digest, Gel Extraction, DpnI digest of PCR products, PCR purification
</EM></B></P><B>Gel of PCR</B> (Amp 60° circ, lin, Chl 40.5 circ, lin)
100V, 20 min 1% gel<B>Gel of Digest</B>
100V, 20 min 1% gel<B>Gel Extraction</B>
Excise the DNA fragment from the agarose gel with a clean, sharp scalpel. Minimize the size of the gel slice by removing extra agarose.
Weigh the gel slice in a colorless tube. Add 3 volumes of Buffer QG to 1 volume of gel (100 mg ~ 100 μl). To help dissolve gel, mix by vortexing the tube every 2–3 min during the incubation.
After the gel slice has dissolved completely, check that the color of the mixture is yellow (similar to Buffer QG without dissolved agarose).
Add 1 gel volume of isopropanol to the sample and mix (actually only needed for very small products or very big products)
To bind DNA, apply the sample to the QIAquick column, and centrifuge for 1 min.
Discard flow-through and place QIAquick column back in the same collection tube.
To wash, add 0.75 ml of Buffer PE to QIAquick column and centrifuge for 1 min.
11. Discard the flow-through and centrifuge the QIAquick column for an additional 1 min at 17,900 x g (13,000 rpm).
12. Place QIAquick column into a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 30 μl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge for 1 min.<B>DpnI digest of PCR products:</B>
SPCR4 (40ul) - add 1 ul DpnI
SPCR 7 (150ul) - add 3 ul of DpnI
SPCR8 (60 ul) - add 3 ul of DpnI
SPCR 9 (150ul) - add 3 ul of DpnI
SPCR10 (110ul) - add 3 ul of DpnI
Incubate for 15 min at 37°C<B>PCR purification</B>
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 50 μl Buffer EB  to the center of the QIAquick membrane and centrifuge the column for 1 min.</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Thursday 15<SUP>th</SUP> August</H3><P><B><EM>
Liquid cultures
</EM></B></P><B>Liquid cultures</B>:
sSp009
sSP012
sSP016
sSp020
Xgal Bacteria from Aude</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Friday 16<SUP>th</SUP> August</H3><P><B><EM>
Liquid M13 Test, Glycerol Stock, Miniprep
</EM></B></P><B>Liquid M13 Test</B>
OD (sSP020=XL10 KanR with F+)=0,557
OD (sSP016=XL10 KanR)=0,515
OD (sSP009= delta pyrF with F+)=0,561
OD(sSP012=FR-433=M13)=0,695
Prepare Falcons containing 500ul of each strain, adding 500ul of 10^-6 dilution of FR-433
add 1ul of IPTG and 1ul of Xgal
Incubate at 37° on shaker<B>Glycerol Stock</B>
sSP019 (pir+ E.coli transformed with pKD13)
sSP020 (XL10 KanR with F+)<B>Miniprep</B>
sSP019
sSP017
1) Pellet 2x 2ml of liquid culture (4000rpm, 10 min)
2) Discard supernatant
3) resuspend the cells in 250ul of resuspension olution
4) add 250ul of lysis solution, mix by inverting 4-6 times
5) add 350ul of neutralization solution
4) centrifuge for 5 min
5) transfer supernatant to spin column
6) centrifuge for 1 min
7) discard flow through
8) add 500 ul wash solution and centrifuge for 1 min , discard flow through(repeat this step)
9) centrifuge for 1 min to remove left over liquid
10) transfer the column on a 1.5ml tube
11) add 50ul of elution buffer and incubate for 2 min
12) centrifuge for 2 min
13) Nanodrop the concentration and freeze at -20°
pKD13: 311ng/ul
Litmus 28i: 93ng/ul</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Monday 19<SUP>th</SUP> August</H3><P><B><EM>
Restreak XL-1 Blue Litmus28i, PCR, PCR purification Backbone (M13mp18), Ligation (RFP into M13mp18 backbone), Trafo of Ligation product (RFP in M13mp18 plasmid) into NEB turbo and Test Xgal
</EM></B></P><B>Restreak XL-1 Blue Litmus28i</B><B>PCR</B>
SPCR5, SPCR6, 49,3°C<TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD><B>1x</B></TD></TR><TR><TD>Nuclease-free water</TD><TD>37,25µL</TD></TR><TR><TD>5x Phusion HF Buffer</TD><TD>10µL</TD></TR><TR><TD>10 mM dNTPs</TD><TD>1µL</TD></TR><TR><TD>Forward Primer (10 µM)</TD><TD>0.5µL</TD></TR><TR><TD>Reverse Primer (10 µM)</TD><TD>0.5µL</TD></TR><TR><TD>Template Plasmid</TD><TD>0.25µL</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>0.5µL</TD></TR><TR><TD><B>Total Volume</B></TD><TD>50µL</TD></TR></TBODY></TABLE><TABLE border=""><TBODY><TR><TD><B>Thermocycler Protocol: NEB Phusion</B></TD></TR><TR><TD>Temp</TD><TD>Time</TD></TR><TR><TD>Start</TD><TD>98°C</TD><TD>30 sec</TD><TD>Melt</TD></TR><TR><TD>Cycle 1</TD><TD>98°C</TD><TD>5 sec</TD><TD>Melt</TD><TD>35 cycles</TD></TR><TR><TD>Cycle 2</TD><TD>40.5°C / 60°C</TD><TD>25 sec</TD><TD>Anneal</TD></TR><TR><TD>Cycle 3</TD><TD>72°C</TD><TD>30 sec per kb</TD><TD>Extend</TD></TR><TR><TD>Finish</TD><TD>72°C</TD><TD>5 min</TD><TD>Extend</TD></TR><TR><TD>Store</TD><TD>10°C</TD><TD>Forever</TD><TD>Store</TD></TR></TBODY></TABLE><B>PCR purification Backbone (M13mp18)</B>
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 50 μl Buffer EB  to the center of the QIAquick membrane and centrifuge the column for 1 min.<B>Ligation (RFP into M13mp18 backbone)</B>
3ul Backbone
14 ul Insert
2ul 10x T4 DNA Ligase HC Buffer
1 ul T4 DNA Ligase
Incubate for at least 10min at 22°C<B>Trafo of Ligation product (RFP in M13mp18 plasmid)  into NEB turbo</B>
1) Thaw competent cells on ice.
2) Place 20 ul of cells in a pre-chilled Eppendorf tube.
3) Add 3 ul of plasmid (from Biobrick stock)
4) Mix gently by flicking the tube.
5) Chill on ice for 10 minutes.
4) Heat shock at 42 °C for 30 seconds 
5) Return to ice for 2 minutes.
6) Add 200 ul LB medium and recover the cells by shaking at 37 °C.
 7) Plate out the cells on LB (200ul)
8. Incubate at 37 °C. Transformants should appear within 12 hrs.<B>Test Xgal</B>
3ml of Liquid culture with 
1) Xgal from iGEM 2012 (Jake) (+IPTG)
2) Xgal from last week (in DMF) (+IPTG)</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Tuesday 20<SUP>th</SUP> August</H3><P><B><EM>
Gel of PCR and Xgal Test
</EM></B></P><B>Gel of PCR</B>
1%, 100V, 20min
Result: PCR didn’t work<B>Xgal Test</B>
MG1655 (of Jake) in liquid culture with Xgal of iGEM 2012, Aude and new one (Anne)
Result:
All cultures turn blue (Aude, less blue)</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Wednesday 21<SUP>st</SUP> August</H3><P><B><EM>
Gradient PCR SPCR5, SPCR6, M13 Test, Electrocompetent Cells (sSP017-XL-1 Blue, Litmus 28i), Transformation
</EM></B></P><B>Gradient PCR SPCR5, SPCR6</B><TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD><B>1x</B></TD></TR><TR><TD>Nuclease-free water</TD><TD>37,25µL</TD></TR><TR><TD>5x Phusion HF Buffer</TD><TD>10µL</TD></TR><TR><TD>10 mM dNTPs</TD><TD>1µL</TD></TR><TR><TD>Forward Primer (10 µM)</TD><TD>0.5µL</TD></TR><TR><TD>Reverse Primer (10 µM)</TD><TD>0.5µL</TD></TR><TR><TD>Template Plasmid</TD><TD>0.25µL</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>0.5µL</TD></TR><TR><TD><B>Total Volume</B></TD><TD>50µL</TD></TR></TBODY></TABLE>
Gradient:
Pos 1: 40.1°
Pos 2: 40,8°
Pos 3: 42.2°
Pos 4: 44.2°
Pos 5: 46.7°
Pos 6: 49.4°
Pos 7: 52.1°
Pos 8: 54.7°
Pos 9: 57.1°
Pos 10: 59.0°
Pos 11: 60.2°C<B>M13 Test</B>
Liquid cultures of sSP012 and sSP020
1) 100 μ l of plating bacteria per tube
2) Prepare tenfold serial dilutions (10-6  to 10-9 ) of the bacteriophage stock in LB
3) Put 10 μ l/ 100 μ l of each dilution to the bacteria
4)  Mix the bacteriophage particles with the bacterial culture by vortexing gently.
5) Add 40 μ l of X-gal solution (working conc) and IPTG (working conc) solution to each of the tubes
6) Add 3ml of Agar to each tube and gently vortexing for 3 seconds
7) Pour the mixture onto  plates containing LB medium supplemented with 5 mM MgCl2 
8) Swirl the plate gently to ensure an even distribution of bacteria and agar.
9) Incubate them at 37°C.<B>Electrocompetent Cells (sSP017-XL-1 Blue, Litmus 28i)</B>
1) The night before the transformation, start an overnight culture of cells.
        	sSP017 (XL1 Blue Litmus 28i)
2) The day of the transformation, dilute the cells 100X.
        	100 ml LB Amp.
        	Grow at 30°C for about 90 minutes. 
3) When the cells reach an OD600 of 0.2. 
4) Harvest the cells.
        	When the cells reach an OD600 of between 0.6 and 0.8 (OD: 0,683)
        	Split the culture into 2x 50 ml falcon tubes, on ice.
        	Centrifuge at 4 °C for 10 min at 4000 rpm.
5) Wash and combine the cells.
        	Remove the supernatant.
        	Resuspend the cells in 2x 25 ml of ice cold water.
        	Combine the volumes in a single 50 ml falcon tube.
6) Wash the cells 2 more times.
        	Centrifuge at 4 °C for 10 min at 4000 rpm.
        	Resuspend in 50 ml of ice cold water.
        	Repeat.
7) Wash and concentrate the cells for electroporation.
        	Centrifuge at 4 °C for 10 min at 4000 rpm.
        	Resuspend in 1-2 ml of ice cold water.
        	We will use 200 ul of washed cells per transformation.<B>Transforming pKD13 into pir+ E.coli</B>
1) Prepare BD tubes with a pipette filled with LB at the interior of each tube (pipette supplied with the electroporation cuvette)
2) Test the purity of the electrocompetent cells.
        	Add 200 ul of washed cells to a cuvette.
3) Mix the cells and DNA in a cuvette.
        	200 ul of washed cells with 200 ng of PCR product.
        	Keep the cuvette on ice until just before the electroporation.
4) Preload a pipette with 1 ml of LB.
5) Pulse the cuvette with voltage.
        	Dry the electrodes with a kimwipe prior to loading.
        	Use the EC2 setting.
6) Listen for arcing.
        	A cracking sound means all the cells are dead.
        	Note the time constant: 5 is good, 5.8 is great.
7) Immediately recover the cells.
        	Add the 1 ml of preloaded LB and pipet up and down to mix.
        	Collect 1 ml of cells, some volume is lost in the cuvette.
8) Incubate 2 h at 37 °C with shaking.
9) Plate 100 ul of recovered cells on selective plates.
        	Use antibiotic appropriate to the part being integrated.
        	Let the other 900 ul rest overnight at room temperature.
10) Concentrate and plate the remaining cells
        	Spin down quickly and resuspend in 100 ul LB before plating.
Transformed cells should be incubated at 37 °C.
Colonies should appear 24-48 h after plating.</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Thursday 22<SUP>nd</SUP> August</H3><P><B><EM>

Gel of gradient PCR product (Pictures 6,5,5), PCR purification and pooling, Digestion M13 backbone, RFP insert, Test Digestion in Gel, Gel Extraction, Liquid culture, Purification of Backbone Cloning and Miniprep sSP015.

</EM></B></P><B>Gel of gradient PCR product </B>
1%, 100V, 20 min<B>PCR purification and pooling</B>
Pool all samples that worked
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 50 μl Buffer EB  to the center of the QIAquick membrane and centrifuge the column for 1 min.
c(SPCR5)= ng/ul
c(SPCR6)= ng/ul<B>Digestion M13 backbone, RFP insert</B>
Backbone/Insert (double the amount)<TABLE border="1"><TBODY><TR><TD>Regent</TD><TD>Volume</TD></TR><TR><TD>Purified Plasmid</TD><TD>20 ul</TD></TR><TR><TD>H2O</TD><TD>65 ul</TD></TR><TR><TD>10x FastDigest Buffer</TD><TD>10 ul</TD></TR><TR><TD>FastDigest Enzyme 1 (EcoRI)</TD><TD>2,5 ul</TD></TR><TR><TD>FastDigest Enzyme 2 (PstI)</TD><TD>2,5 ul</TD></TR><TR><TD>FastAP Phosphatase</TD><TD>0 ul</TD></TR><TR><TD><B>Total Volume</B></TD><TD><B>100ul</B></TD></TR></TBODY></TABLE>
Digest for 2.5  hours at 37 °C with shaking.<B>Test Digestion in Gel</B>
1%, 100V, 20 min<B>Gel Extraction</B>
Excise the DNA fragment from the agarose gel with a clean, sharp scalpel. Minimize the size of the gel slice by removing extra agarose.
Weigh the gel slice in a colorless tube. Add 3 volumes of Buffer QG to 1 volume of gel (100 mg ~ 100 μl). To help dissolve gel, mix by vortexing the tube every 2–3 min during the incubation.
After the gel slice has dissolved completely, check that the color of the mixture is yellow (similar to Buffer QG without dissolved agarose).
Add 1 gel volume of isopropanol to the sample and mix (actually only needed for very small products or very big products)
To bind DNA, apply the sample to the QIAquick column, and centrifuge for 1 min.
Discard flow-through and place QIAquick column back in the same collection tube.
To wash, add 0.75 ml of Buffer PE to QIAquick column and centrifuge for 1 min.
11. Discard the flow-through and centrifuge the QIAquick column for an additional 1 min at 17,900 x g (13,000 rpm).
12. Place QIAquick column into a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 30 μl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge for 1 min.
c=13,6ng/ul
Liquid culture
sSP015<B>Purification of Backbone Cloning</B>
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 30 μl Buffer EB  to the center of the QIAquick membrane and centrifuge the column for 1 min.<B>Miniprep sSP015</B>
1) Pellet 2x 2ml of liquid culture (4000rpm, 10 min)
2) Discard supernatant
3) resuspend the cells in 250ul of resuspension solution
4) add 250ul of lysis solution, mix by inverting 4-6 times
5) add 350ul of neutralization solution
4) centrifuge for 5 min
5) transfer supernatant to spin column
6) centrifuge for 1 min
7) discard flow through
8) add 500 ul wash solution and centrifuge for 1 min , discard flow through(repeat this step)
9) centrifuge for 1 min to remove left over liquid
10) transfer the column on a 1.5ml tube
11) add 50ul of elution buffer and incubate for 2 min
12) centrifuge for 2 min
13) Nanodrop the concentration and freeze at -20°</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Friday 23<SUP>rd</SUP> August</H3><P><B><EM>
Stress Experiment (sSP001, sSP002, sSP010, sSP011, M13, MMC), Microscope, Gradient PCR of SPCR2, Digestion of SPCR 5 and SPCR7 and Purification of Digestion.
</EM></B></P><B>Stress Experiment (sSP001, sSP002, sSP010, sSP011, M13, MMC)</B>
Dilute ON culture 1:100
Centrifuge (1 min, 10rpm)
Wash 1x with 70% of Volume M9 Glucose
Centrifuge again
Take up in 70% of the VOlume M9 Gluo
100ul in each 96 well plate (see pipetting file)
Add MMC 10ug/ml in corresponding wells and incubate in the plate reader
After 90 min add phages into correscponding wells
put back on the plate reader<B>Microscope</B>
look at the shape and fluorescence at t=0 of sSP001
look at the shape and fluorescence at=90 mi of sSP001 + 10ug/ml MMC
take photos<B>Gradient PCR of SPCR2</B>
iSP001 (gblock) and iS002<TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD><B>1x</B></TD></TR><TR><TD>Nuclease-free water</TD><TD>37,25µL</TD></TR><TR><TD>5x Phusion HF Buffer</TD><TD>10µL</TD></TR><TR><TD>10 mM dNTPs</TD><TD>1µL</TD></TR><TR><TD>Forward Primer (10 µM)</TD><TD>0.5µL</TD></TR><TR><TD>Reverse Primer (10 µM)</TD><TD>0.5µL</TD></TR><TR><TD>Template Plasmid</TD><TD>0.25µL</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>0.5µL</TD></TR><TR><TD><B>Total Volume</B></TD><TD>50µL</TD></TR></TBODY></TABLE><TABLE border=""><TBODY><TR><TD><B>Thermocycler Protocol: NEB Phusion</B></TD></TR><TR><TD>Temp</TD><TD>Time</TD></TR><TR><TD>Start</TD><TD>98°C</TD><TD>30 sec</TD><TD>Melt</TD></TR><TR><TD>Cycle 1</TD><TD>98°C</TD><TD>5 sec</TD><TD>Melt</TD><TD>35 cycles</TD></TR><TR><TD>Cycle 2</TD><TD>40.5°C / 60°C</TD><TD>25 sec</TD><TD>Anneal</TD></TR><TR><TD>Cycle 3</TD><TD>72°C</TD><TD>30 sec per kb</TD><TD>Extend</TD></TR><TR><TD>Finish</TD><TD>72°C</TD><TD>5 min</TD><TD>Extend</TD></TR><TR><TD>Store</TD><TD>10°C</TD><TD>Forever</TD><TD>Store</TD></TR></TBODY></TABLE><B>Digestion of SPCR 5 and SPCR7</B><TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Purified PCR Product</TD><TD>16 ul</TD></TR><TR><TD>H2O</TD><TD>0 ul</TD></TR><TR><TD>10x FastDigest Buffer</TD><TD>2 ul</TD></TR><TR><TD>FastDigest Enzyme 1</TD><TD>1 ul</TD></TR><TR><TD>FastDigest Enzyme 2</TD><TD>1 ul</TD></TR><TR><TD><B>Total Volume</B></TD><TD><B>20 ul</B></TD></TR></TBODY></TABLE>
Incubate at 37°C for 1h shaking<B>Purification of Digestion</B>
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 30 μl Buffer EB  to the center of the QIAquick membrane and centrifuge the column for 1 min.</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Sunday 25<SUP>th</SUP> August</H3><P><B><EM>
Gel of Gradient PCR, Liquid Cultures, Ligation M13 + RFP, Ligation SPCR5, SPCR7, Electrocompetent cells, Gibson Assembly, Transformation (Gibson)and Transformation (Gibson, Ligations.
</EM></B></P><B>Gel of Gradient PCR</B>
1%, 100V, 20min<B>Liquid Cultures</B>
NEB Turbo
XL10<B>Ligation</B><B>Ligation M13 + RFP</B>
0.5ul Vector (M13)
2.6ul Insert (RFP)
5.4ul H20
1.0ul T4 buffer
0.5ul T4 Ligase<B>Ligation SPCR5, SPCR7</B>
1.0ul Vector (SPCR5)
1.5ul Insert (SPCR7)
6.0ul H20
1.0ul T4 buffer
0.5ul T4 Ligase
Incubate ~30min at 37 C<B>Electrocompetent cells</B>
1) Start Culture
2) When the cells reach an OD600 of 0.2. 
3) Harvest the cells.
        	When the cells reach an OD600 of between 0.6 and 0.8 (OD: 0,683)
        	Centrifuge at 4 °C for 10 min at 4000 rpm.
4) Wash and combine the cells.
        	Remove the supernatant.
        	Resuspend the cells in 25 ml of ice cold water
5) Wash the cells 2 more times.
        	Centrifuge at 4 °C for 10 min at 4000 rpm.
        	Resuspend in 50 ml of ice cold water.
        	Repeat.
6) Wash and concentrate the cells for electroporation.
        	Centrifuge at 4 °C for 10 min at 4000 rpm.
        	Resuspend in 1-2 ml of ice cold water.
        	We will use 200 ul of washed cells per transformation.<B>Gibson Assembly</B>
Amount per Reaction
Positive Control**
PCR Fragment(s)
+ linearized vector
2-10 μl (0.02–0.5 pmols)*
Gibson
Assembly Master
Mix (2X)
10 μl
Deionized H2O
XX μl
Total Volume
20 μl
Incubate samples in a thermocycler at 50°C for 15 minutes. After incubation, store the samples on ice or at -20°C for subsequent transformation
Note: Extended incubation up to 60 minutes may help to improve assembly efficiency in some cases (for further details see FAQ section).
Transform NEB 5-alpha Competent E. coli cells (provided with the kit) with 2 μl of the assembly reaction, following the transformation protocol.<B>Transformation (Gibson)</B>
Thaw competent cells on ice.
Add 2 μl of the chilled assembly product to the competent cells. Mix gently by pipetting up and down or by flicking the tube 4–5 times. Do not vortex.
Place the mixture on ice for 30 minutes. Do not mix.
Heat shock at 42°C for 30 seconds. Do not mix.
Transfer tubes to ice for 2 minutes.
Add 950 μl of room-temperature SOC media to the tube.
Incubate the tube at 37°C for 60 minutes. Shake vigorously (250 rpm) or rotate.
Warm selection plates to 37°C.
Spread 100 μl of the cells onto the selection plates. Use Amp plates for positive control sample.
Incubate overnight at 37°C.<B>Transformation (Gibson, Ligations)</B>
1) Prepare BD tubes with a pipette filled with LB at the interior of each tube (pipette supplied with the electroporation cuvette)
2) Test the purity of the electrocompetent cells.
        	Add 200 ul of washed cells to a cuvette.
3) Mix the cells and DNA in a cuvette.
        	200 ul of washed cells with 200 ng of PCR product.
        	Keep the cuvette on ice until just before the electroporation.
4) Preload a pipette with 1 ml of LB.
5) Pulse the cuvette with voltage.
        	Dry the electrodes with a kimwipe prior to loading.
        	Use the EC2 setting.
6) Listen for arcing.
        	A cracking sound means all the cells are dead.
        	Note the time constant: 5 is good, 5.8 is great.
7) Immediately recover the cells.	Add the 1 ml of preloaded LB and pipet up and down to mix.
        	Collect 1 ml of cells, some volume is lost in the cuvette.
8) Incubate 2 h at 37 °C with shaking.
9) Plate 100 ul of recovered cells on selective plates.
        	Use antibiotic appropriate to the part being integrated.
        	Let the other 900 ul rest overnight at room temperature.
10) Concentrate and plate the remaining cells
        	Spin down quickly and resuspend in 100 ul LB before plating.
Transformed cells should be incubated at 37 °C.
Colonies should appear 24-48 h after plating.</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Monday 26<SUP>th</SUP> August</H3><B>Picking colonies (Trafo Gibson, BBC SPCR5/7)</B>
chosing 5-7 colonies and start liquid cultures
</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Tuesday 27<SUP>th</SUP> August</H3><P><B><EM>
Starting liquid cultures and Miniprep of pS.005 and pS.006.
</EM></B></P><B>Starting liquid cultures</B>
sSP001, sSP002, sSP010, sSP011, sSP012, sSP020<B>Miniprep of pS.005 and pS.006</B>
1) Pellet 2x 9ml of liquid culture (4000rpm, 10 min)
2) Discard supernatant
3) resuspend the cells in 250ul of resuspension solution
4) add 250ul of lysis solution, mix by inverting 4-6 times
5) add 350ul of neutralization solution
4) centrifuge for 5 min
5) transfer supernatant to spin column
6) centrifuge for 1 min
7) discard flow through
8) add 500 ul wash solution and centrifuge for 1 min , discard flow through(repeat this step)
9) centrifuge for 1 min to remove left over liquid
10) transfer the column on a 1.5ml tube
11) add 50ul of elution buffer and incubate for 2 min
12) centrifuge for 2 min
13) Nanodrop the concentration and freeze at -20°<B>Colony PCR</B><TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Forward Primer (10 uM)</TD><TD>0,5 ul</TD></TR><TR><TD>Reverse Primer (10 uM)</TD><TD>0,5 ul</TD></TR><TR><TD>Template DNA  (Miniprep)</TD><TD>0,5 ul</TD></TR><TR><TD>Quick-Load® Taq 2X Master Mix</TD><TD>12,5 ul</TD></TR><TR><TD>Nuclease-free water</TD><TD>10,0 ul</TD></TR><TR><TD><B>Total volume</B></TD><TD>25 ul</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD><B>Thermocycler Protocol: Dream Taq Green</B></TD></TR><TR><TD>Temp</TD><TD>Time</TD></TR><TR><TD>Start</TD><TD>95°C</TD><TD>30 sec1</TD><TD>Melt</TD></TR><TR><TD> Cycle 1</TD><TD>95°C</TD><TD>15 sec</TD><TD>Melt</TD><TD> 35 cycles </TD></TR><TR><TD> Cycle 2</TD><TD>46,8°C </TD><TD> 30 sec </TD><TD> Anneal </TD></TR><TR><TD> Cycle 3</TD><TD> 68°C </TD><TD> 1 min per kb </TD><TD>Extend</TD><TD>cellule 2</TD></TR><TR><TD> Finish</TD><TD> 68°C </TD><TD> 5 min </TD><TD> Extend </TD></TR><TR><TD> Store</TD><TD> 10°C </TD><TD>Forever </TD><TD>Store </TD></TR></TBODY></TABLE></DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Wednesday 28<SUP>th</SUP> August</H3><P><B><EM>
Gel Colony PCR, Liquid Cultures (1:100), Analytical Digest, Gel Ligation/Gibson, Gel Analytical digest, DpnI digest of SPCR5, SPCR6, SPCR7, Gel DpnI digest, PCR purification of DpnI digested SPCR5, SPCR6, SPCR7, Stress test microscope and M13 Test. 
</EM></B></P><B>Gel Colony PCR</B>
pS005 (C1-C5) - pS006 (C1-C7)
1% 100V 20 min<B>Liquid Cultures (1:100)</B>
sSP001, sSP002, sSP010, sSP011, sSP012, sSP020<B>Analytical Digest</B><TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD> Plasmid Miniprep </TD><TD>5 ul</TD></TR><TR><TD>H2O</TD><TD>12 ul</TD></TR><TR><TD>c10x FastDigest Buffer</TD><TD>2ul</TD></TR><TR><TD>FastDigest Enzyme 1</TD><TD>0,5 ul</TD></TR><TR><TD>FastDigest Enzyme 2</TD><TD>0,5 ul</TD></TR><TR><TD><B>Total Volume</B></TD><TD> 20,0 ul</TD></TR></TBODY></TABLE>
Digest for 15 minutes at 37 °C with shaking<B>Gel Ligation/Gibson</B>
pS005 - pS006 - M13/RFP
1% 100V 20min<B>Gel Analytical digest</B>
pS005 (C1-C5)-pS006 (C1-C7)
1% 100V 20min<B>DpnI digest of SPCR5, SPCR6, SPCR7</B>
add 4ul of DpnI into each tube, mix and incubate for 1h at 37°C<B>Gel DpnI digest</B>
let each 5ul run on a gel
1%, 100V 20min<B>PCR purification of DpnI digested SPCR5, SPCR6, SPCR7</B>
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 30 μl Buffer EB  to the center of the QIAquick membrane and centrifuge the column for 1 min.<B>Stress test microscope</B>
Dilute ON cultures of sSP001, sSP002, sSP010, sSP011, sSP012 1:100 and incubate up to an OD of 0.3
Take out 2ml of sSP001, sSP002, sSP010, sSP011
add:
sSP001: 10ug/ml MMC
sSP002: 10ug/ml MMC
sSP010: 20ul sSP012 supernatant
sSP011: 20ul sSP012 supernatant
incubate for 90 min
take pictures under the microscope (trans and YFP) of (un-)treated samples<B>M13 Test</B>
Liquid cultures of sSP012 and sSP020
1) 100 μ l of plating bacteria per tube (OD 0.6)
2) Prepare tenfold serial dilutions (10-6  to 10-9 ) of the bacteriophage stock in LB
3) Put 10 μ l/ 100 μ l of each dilution to the bacteria
4)  Mix the bacteriophage particles with the bacterial culture by vortexing gently.
5) Add 40 μ l of X-gal solution (working conc) and IPTG (working conc) solution to each of the tubes
6) Add 3ml of Agar to each tube and gently vortexing for 3 seconds
7) Pour the mixture onto  plates containing LB medium supplemented with 5 mM MgCl2 and 2x IPTG and Xgal
8) Swirl the plate gently to ensure an even distribution of bacteria and agar.
9) Incubate them at 37°C.</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Friday 30<SUP>th</SUP> August</H3><P><B><EM>
M13 test, DpnI digest (SPCR5,6,7), Gel SPCR5,6,7 and PCR purification of DpnI digested SPCR5, SPCR6, SPCR7
</EM></B></P><B>M13 test</B>
dilute ON cultures (NEB turbo, FR-433) 1: 100, grow up to an OD of ~0.6
1) 100 μ l of plating bacteria per tube (OD 0.6)
2) Prepare tenfold serial dilutions (10-6  to 10-9 ) of the bacteriophage stock in LB
3) Put 10 μ l/ 100 μ l of each dilution to the bacteria
4)  Mix the bacteriophage particles with the bacterial culture by vortexing gently.
5) Add 40 μ l of X-gal solution (working conc) and IPTG (working conc) solution to each of the tubes
6) Add 3ml of Agar to each tube and gently vortexing for 3 seconds
7) Pour the mixture onto  plates containing LB medium supplemented with 5 mM MgCl2 and 2x IPTG and Xgal
8) Swirl the plate gently to ensure an even distribution of bacteria and agar.
9) Incubate them at 37°C.<B>DpnI digest (SPCR5,6,7)</B>
add 8ul DpnI per tube and digest for 1h at 37°, shaking<B>Gel SPCR5,6,7</B>
1%, 100V, 20min<B>PCR purification of DpnI digested SPCR5, SPCR6, SPCR7</B>
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 30 μl Buffer EB  to the center of the QIAquick membrane and centrifuge the column for 1 min.</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Day Number<SUP>suffix</SUP> Month</H3><P><B><EM>
Place your twit here
</EM></B></P>
Place your note here
</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Wednesday 4<SUP>th</SUP> September</H3><P><B><EM>
Starting Liquid culture, PCRs, Gel, DpnI digest, Digestion of SPCR5,7, Purification of Digestion, Ligation of SPCR5,7, Gibson Assembly, Making Electrocompetent Cells, Transformation, Redo SPCR12/13 , 
</EM></B></P><B>Starting Liquid culture</B>
NEB turbo, grow up to an OD of 0.6<B>PCRs</B>
SPCR1, SPCR2, SPCR4, SPCR12, SPCR13<TABLE border="1"><TBODY><TR><TD><B>Reagant</B></TD><TD><B>Volume</B></TD></TR><TR><TD><B>1x (SPCR2)</B></TD><TD><B>11x (SPCR12)</B></TD><TD><B>11x (SPCR13)</B></TD><TD><B>8x (SPCR1)</B></TD><TD><B>8x (SPCR4)</B></TD></TR><TR><TD>Nuclease-free water</TD><TD>37.25 µl</TD><TD>447 µl</TD><TD>447 µl</TD><TD>335,25µl</TD><TD>335,25µl</TD></TR><TR><TD>5x Phusion HF Buffer</TD><TD>10 µl</TD><TD>120 µl</TD><TD>120 µl</TD><TD>90 µl</TD><TD>90 µl</TD></TR><TR><TD>10 mM dNTPs</TD><TD>1 µl</TD><TD>12 µl</TD><TD>12 µl</TD><TD>9 µl</TD><TD>9 µl</TD></TR><TR><TD>Forward Primer (10 uM)</TD><TD>0.5 µl</TD><TD>6 µl</TD><TD>6 µl</TD><TD>4.5 µl</TD><TD>4.5 µl</TD></TR><TR><TD>Reverse Primer (10 uM)</TD><TD>0.5 µl</TD><TD>6 µl</TD><TD>6 µl</TD><TD>4.5 µl</TD><TD>4.5 µl</TD></TR><TR><TD>Template Plasmid</TD><TD>0.25 µl</TD><TD>3 µl</TD><TD>3 µl</TD><TD>2.25 µl</TD><TD>2.25 µl</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>0.5 µl</TD><TD>6 µl</TD><TD>6 µl</TD><TD>4,5 µl</TD><TD>4,5 µl</TD></TR><TR><TD>Total Volume</TD><TD>50 µl</TD><TD>600 µl</TD><TD>600 µl</TD><TD>450 µl</TD><TD>450 µl</TD></TR></TBODY></TABLE>
for SPCR 12/13 gradient, 2,5 min
for SPCR3, SPCR11 51°, 45s
for SPCR 1, SPCR2, 48,8°, 2 min
for SPCR4, 40,8°, 1 min<B>Gel</B>
1%, 100V, 20 min
1.small gel (SPCR1,2,3,4,11)
2. big gel (SPCR12)
3. big Gel (SPCR13)<B>DpnI digest</B>
SPCR 1: add 10ul DpnI
SPCR4: add 10ul DpnI
SPCR2,3,11: add 1 ul DpnI
incubate for 1h at 37°<B>Digestion of SPCR5,7</B><TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Purified PCR Product</TD><TD>16 µl</TD></TR><TR><TD>H2O</TD><TD>0 µl</TD></TR><TR><TD>10x FastDigest Buffer</TD><TD>2 µl</TD></TR><TR><TD>XbaI</TD><TD>1 µl</TD></TR><TR><TD>SpeI</TD><TD>1 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>20.0 µl</TD></TR></TBODY></TABLE>
Incubate for 1h at 37°, shaking<B>Purification of Digestion</B>
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 30 μl Buffer EB  to the center of the QIAquick membrane and centrifuge the column for 1 min.<B>Ligation of SPCR5,7</B><TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Vector</TD><TD>1  µl</TD></TR><TR><TD>Insert</TD><TD>3  µl</TD></TR><TR><TD>H2O</TD><TD>4.5  µl</TD></TR><TR><TD>Fermentas T4 Ligase Buffer</TD><TD>1  µl</TD></TR><TR><TD>Fermentas T4 Ligase Enzyme</TD><TD>0.5  µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>10  µl</TD></TR></TBODY></TABLE>
Incubate at room temperature for ~1h<B>Gibson Assembly</B><B>Amount per Reaction :</B> Positive Control**<B>PCR Fragment(s)+ linearized vector :</B> 2-10 μl (0.02–0.5 pmols)*<B>Gibson Assembly Master Mix (2X):</B> 10 μl<B>Deionized H2O :</B> XX μl<B>Total Volume :</B> 20 μl
Incubate samples in a thermocycler at 50°C for 15 minutes. 
Put on Ice
Insert: 2,4 ul
Vector: 0,6 ul<B>Making Electrocompetent Cells</B>
1)  Start a liquid culture of cells. 
                NEB turbo
2)  dilute the cells 100X.
        	100 ml LB.
        	Grow at 30°C for about 90 minutes. 
3) Harvest the cells.
        	When the cells reach an OD600 of between 0.6 and 0.8
        	Split the culture into 2x 50 ml falcon tubes, on ice.
        	Centrifuge at 4 °C for 10 min at 4000 rpm.
5) Wash and combine the cells.
        	Remove the supernatant.
        	Resuspend the cells in 2x 25 ml of ice cold water.
        	Combine the volumes in a single 50 ml falcon tube.
6) Wash the cells 2 more times.
        	Centrifuge at 4 °C for 10 min at 4000 rpm.
        	Resuspend in 50 ml of ice cold water.
        	Repeat.
7) Wash and concentrate the cells for electroporation.
        	Centrifuge at 4 °C for 10 min at 4000 rpm.
        	Resuspend in 1-2 ml of ice cold water.
        	We will use 200 ul of washed cells per transformation.<B>Transformation</B>
pS005, pS006, SPCR3 (IDT plasmid) M13mp18
1) Prepare BD tubes with a pipette filled with LB at the interior of each tube (pipette supplied with the electroporation cuvette)
2) Test the purity of the electrocompetent cells.
        	Add 200 ul of washed cells to a cuvette.
3) Mix the cells and DNA in a cuvette.
        	200 ul of washed cells with 200 ng of PCR product.
        	Keep the cuvette on ice until just before the electroporation.
4) Preload a pipette with 1 ml of LB.
5) Pulse the cuvette with voltage.
        	Dry the electrodes with a kimwipe prior to loading.
        	Use the EC2 setting.
6) Listen for arcing.
        	A cracking sound means all the cells are dead.
        	Note the time constant: 5 is good, 5.8 is great.
7) Immediately recover the cells.
        	Add the 1 ml of preloaded LB and pipet up and down to mix.
        	Collect 1 ml of cells, some volume is lost in the cuvette.
8) Incubate 2 h at 37 °C with shaking.
9) Plate 10/200 ul of recovered cells on selective plates.
        	Use antibiotic appropriate to the part being integrated.
        	Let the other 900 ul rest overnight at room temperature.
10) Concentrate and plate the remaining cells
        	Spin down quickly and resuspend in 100 ul LB before plating.
Transformed cells should be incubated at 37 °C.
Colonies should appear 24-48 h after plating.<B>Redo SPCR12/13</B><TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD><B>11x (SPCR12)</B></TD><TD><B>11x (SPCR13)</B></TD></TR><TR><TD>Nuclease-free water</TD><TD>447 µl</TD><TD>447 µl</TD></TR><TR><TD>5x Phusion HF Buffer</TD><TD>120 µl</TD><TD>120 µl</TD></TR><TR><TD>10 mM dNTPs</TD><TD>12 µl</TD><TD>12 µl</TD></TR><TR><TD>Forward Primer (10 uM)</TD><TD>6 µl</TD><TD>6 µl</TD></TR><TR><TD>Reverse Primer (10 uM)</TD><TD>6 µl</TD><TD>6 µl</TD></TR><TR><TD>Template Plasmid</TD><TD>3 µl</TD><TD>3 µl</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>6 µl</TD><TD>6 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>600 µl</TD><TD>600 µl</TD></TR></TBODY></TABLE>Cycling protocol as usual + gradient + topdown</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Thursday 5<SUP>th</SUP> September</H3><P><B><EM>
Gel, Colony PCR, Topdown PCR and gels
</EM></B></P><B>Gel</B>
SPCR12, SPCR13
1%, 100V, 20 min
Pictures: 13/09/05_SPCR12_corresponding primers
	   13/09/05_SPCR13_corresponding primers<B>Colony PCR</B><TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Forward Primer (10 uM)/td&gt;
    </TD><TD>0.5 ul</TD></TR><TR><TD>Reverse Primer (10 uM)</TD><TD>0.5 ul</TD></TR><TR><TD>Template DNA (from above)</TD><TD>1.5 ul</TD></TR><TR><TD>Quick-Load® Taq 2X Master Mix</TD><TD>12.5 ul</TD></TR><TR><TD><B>Total Volume</B></TD><TD>25 ul</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD><B>Thermocycler Protocol: DreamTaq Green PCR MM</B></TD></TR><TR><TD>Temp</TD><TD>Time</TD></TR><TR><TD>Start</TD><TD>95°C</TD><TD>1 min </TD><TD>Melt</TD></TR><TR><TD>Cycle 1</TD><TD>95°C</TD><TD>30 sec</TD><TD>Melt</TD><TD>35 cycles</TD></TR><TR><TD>Cyle 2</TD><TD>52.6/46.8 °C</TD><TD>30 sec</TD><TD>Anneal</TD></TR><TR><TD>Cycle 3</TD><TD>72°C </TD><TD>1 min per kb</TD><TD>Extend</TD></TR><TR><TD>Finish</TD><TD>72°C</TD><TD>10 min</TD><TD>Extend</TD></TR><TR><TD>Store</TD><TD>10°C</TD><TD>Forever</TD><TD>Store</TD></TR></TBODY></TABLE><B>Topdown PCR</B>
SPCR 1,4,12,13<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD><B>x1&gt;/b&gt;</B></TD></TR><TR><TD>Nuclease-free water</TD><TD>37.25 µl</TD></TR><TR><TD>5x Phusion HF Buffer</TD><TD>10 µl</TD></TR><TR><TD>10 mM dNTPs</TD><TD>1 µl</TD></TR><TR><TD>Forward Primer (10 uM)</TD><TD>0.5 µl</TD></TR><TR><TD>Reverse Primer (10 uM)</TD><TD>0.5 µl</TD></TR><TR><TD>Template Plasmid</TD><TD>0.25 µl</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>50 µl</TD></TR></TBODY></TABLE>
increases every cycle 1°C, starting from 40°<B>Gels</B>
1%, 100V, 20+ Min
Pictures:
13/09/05_colony PCR_pS005_C1-C8_BB verification primers
13/09/05_colony PCR_pS005_C9-C16_BB verification primers
13/09/05_colony PCR_pS006_C1-C8_BB verification primers
13/09/05_colony PCR_pS006_C9-C16_BB verification primers
13/09/05_top down PCR_SPCR1,4,12,13</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Friday 6<SUP>th</SUP> September</H3><P><B><EM>
Place your twit here
</EM></B></P>
Digestions: linearized pSB1C3, linearized pSB1A3, SPCR2, SPCR3, SPCR11<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Purified PCR Product</TD><TD>16 µl</TD></TR><TR><TD>H2OH2O</TD><TD>0 µl</TD></TR><TR><TD>10x FastDigest Buffer</TD><TD>2 µl</TD></TR><TR><TD>FastDigest Enzyme 1</TD><TD>1 µl</TD></TR><TR><TD>cFastDigest Enzyme 2</TD><TD>1 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>4 µl</TD></TR></TBODY></TABLE>
SPCR5, pSB1C3: XbaI/SpeI
SPCR11, pSB1A3, pSB1C3: EcoRI/PstI
SPCR13, pSB1A3, pSB1C3: EcoRI/PstI
SPCR2, pSB1A3, pSB1C3: EcoRI/PstI
1h at 37° shaking</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Saturday 7<SUP>th</SUP> September</H3><P><B><EM>
Ligation, Making Electrocompetent Cells, Gibson Assembly, Transformation, Liquid culture of E1010
</EM></B></P><B>Ligation</B><TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Vector</TD><TD>1 µl</TD></TR><TR><TD>Insert</TD><TD>3 µl</TD></TR><TR><TD>H2O</TD><TD>4.5 µl</TD></TR><TR><TD>Fermentas T4 Ligase Buffer</TD><TD>1</TD></TR><TR><TD>Fermentas T4 Ligase Enzyme</TD><TD>0.5</TD></TR><TR><TD><B>Total Volume</B></TD><TD>10 µl</TD></TR></TBODY></TABLE>
SPCR5, pSB1C3
SPCR11, pSB1A3, pSB1C3
SPCR13, pSB1A3, pSB1C3
SPCR2, pSB1A3, pSB1C3
Incubate at room temperature for 10 minutes.<B>Making Electrocompetent Cells</B>
1)  Start a liquid culture of cells.
        NEB turbo
2)  dilute the cells 100X.
        	100 ml LB.
        	Grow at 30°C for about 90 minutes. 
3) Harvest the cells.
        	When the cells reach an OD600 of between 0.6 and 0.8
        	Split the culture into 2x 50 ml falcon tubes, on ice.
        	Centrifuge at 4 °C for 10 min at 4000 rpm.
5) Wash and combine the cells.
        	Remove the supernatant.
        	Resuspend the cells in 2x 25 ml of ice cold water.
        	Combine the volumes in a single 50 ml falcon tube.
6) Wash the cells 2 more times.
        	Centrifuge at 4 °C for 10 min at 4000 rpm.
        	Resuspend in 50 ml of ice cold water.
        	Repeat.
7) Wash and concentrate the cells for electroporation.
        	Centrifuge at 4 °C for 10 min at 4000 rpm.
        	Resuspend in 1-2 ml of ice cold water.
        	We will use 200 ul of washed cells per transformation.<B>Gibson Assembly</B>
Amount per Reaction
Positive Control**
PCR Fragment(s)
+ linearized vector
2-10 μl (0.02–0.5 pmols)*
Gibson
Assembly Master
Mix (2X)
10 μl
Deionized H2O
XX μl

Total Volume
20 μl
Incubate samples in a thermocycler at 50°C for 15 minutes. 
Put on Ice<B>Transformation</B>
pS002 (Chl), pS002 (Amp), pS003 (Amp), pS004 (Chl), pS004 (Amp), pS005 (Chl), pS006-7 (Chl), pS007 (Chl), pS008 (Amp), pS009 (Chl), pS012 (Amp), pS013 (Chl), pS013 (Amp) 
1) Prepare BD tubes with a pipette filled with LB at the interior of each tube (pipette supplied with the electroporation cuvette)
2) Test the purity of the electrocompetent cells.
        	Add 200 ul of washed cells to a cuvette.
3) Mix the cells and DNA in a cuvette.
        	200 ul of washed cells with 200 ng of PCR product.
        	Keep the cuvette on ice until just before the electroporation.
4) Preload a pipette with 1 ml of LB.
5) Pulse the cuvette with voltage.
        	Dry the electrodes with a kimwipe prior to loading.
        	Use the EC2 setting.
6) Listen for arcing.
        	A cracking sound means all the cells are dead.
        	Note the time constant: 5 is good, 5.8 is great.
7) Immediately recover the cells.
        	Add the 1 ml of preloaded LB and pipet up and down to mix.
        	Collect 1 ml of cells, some volume is lost in the cuvette.
8) Incubate 1/2 h at 37 °C with shaking.
9) Plate 10/200 ul of recovered cells on selective plates.
        	Use antibiotic appropriate to the part being integrated.
        	Let the other 900 ul rest overnight at room temperature.
10) Concentrate and plate the remaining cells
        	Spin down quickly and resuspend in 100 ul LB before plating.
Transformed cells should be incubated at 37 °C.
Colonies should appear 24-48 h after plating.<B>Liquid culture of E1010</B></DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Sunday 8<SUP>th</SUP> September</H3><P><B><EM>
Replating of transformation and Miniprep of E1010
</EM></B></P><B>Replating of Transformation</B><B>Miniprep of E1010</B>
1) Pellet liquid culture (4000rpm, 10 min)
2) Discard supernatant
3) resuspend the cells in 250ul of resuspension olution
4) add 250ul of lysis solution, mix by inverting 4-6 times
5) add 350ul of neutralization solution
4) centrifuge for 5 min
5) transfer supernatant to spin column
6) centrifuge for 1 min
7) discard flow through
8) add 500 ul wash solution and centrifuge for 1 min , discard flow through(repeat this step)
9) centrifuge for 1 min to remove left over liquid
10) transfer the column on a 1.5ml tube
11) add 50ul of elution buffer and incubate for 2 min
12) centrifuge for 2 min
13) Nanodrop the concentration and freeze at -20°</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Monday 16<SUP>th</SUP> September</H3><P><B><EM>
Result Cloning, Protocol: E. coli Colony PCR, Extraction of DNA, PCR Reaction and Gels
</EM></B></P><B>Result Cloning:</B>
Colonies?
pS002 (Chl): no
pS002 (Amp): yes
pS003 (Amp):yes
 pS004 (Chl): no
pS004 (Amp): yes
pS005 (Chl): yes
pS006-7 (Chl): yes
pS006 (Chl): no
pS007 (Chl): yes
pS008 (Amp): yes
pS009 (Chl): yes
pS012 (Amp): yes
pS013 (Chl): no
pS013 (Amp): yes<B>Protocol: E. coli Colony PCR</B>
        	Using the Dream Taq Master Mix<B>Extraction of DNA</B>
1) Pick a single colony into 50 ul of H20.
2) Boil for 5 minutes.<B>PCR Reaction (pS002 (Chl-Lig), pS005 (Chl-G), pS006-7 (Chl-Lig), pS007 (Chl-G), pS008 (Amp-G), pS009 (Chl-G))</B>
Keep all the reagents at 4 °C while preparing the mixture.
Pre-heat the thermocycler to 95 °C and transfer your reaction directly from 4 °C.<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Forward Primer (10 uM)</TD><TD>0.5 ul</TD></TR><TR><TD>Reverse Primer (10 uM)</TD><TD>0.5 ul</TD></TR><TR><TD>Template DNA (from above)</TD><TD>1.5 ul</TD></TR><TR><TD>Quick-Load® Taq 2X Master Mix</TD><TD>12.5 ul</TD></TR><TR><TD>Nuclease-free water</TD><TD>10 ul</TD></TR><TR><TD><B>Total Volume</B></TD><TD>25 ul</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD><B>Thermocycler Protocol: NEB Quick-Load</B></TD></TR><TR><TD>Temp</TD><TD>Time</TD></TR><TR><TD>Start</TD><TD>95°C </TD><TD>30 sec</TD><TD>Melt</TD></TR><TR><TD>Cycle 1</TD><TD>95°C</TD><TD>15 sec</TD><TD>Melt</TD><TD>35 cycles</TD></TR><TR><TD>Cycle 2</TD><TD>45.8 °C</TD><TD>30 sec</TD><TD>Anneal</TD></TR><TR><TD>Cycle 3</TD><TD>72°C</TD><TD>1 min per kb</TD><TD>Extend</TD></TR><TR><TD>Finish</TD><TD>72 °C</TD><TD>15 min</TD><TD>Extand</TD></TR><TR><TD>Store</TD><TD>10°C</TD><TD>Forever</TD><TD>Store</TD></TR></TBODY></TABLE><B>Gels</B>
100V, 25-30 min, 1%
Pictures: 
13/09/16_colony PCR_ps007.BMP
13/09/16_colony PCR_pS006-7.BMP
13/09/16_colony PCR_ps005.BMP
Result: didn’t work</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Tuesday 17<SUP>th</SUP> September</H3><P><B><EM>
PCR Reaction (pS003 (Amp-G), pS004 (Amp-Lig), pS012 (Amp-G), pS013 (Chl-Lig))
</EM></B></P><B>PCR Reaction (pS003 (Amp-G), pS004 (Amp-Lig), pS012 (Amp-G), pS013 (Chl-Lig))</B> 
Keep all the reagents at 4 °C while preparing the mixture.
Pre-heat the thermocycler to 95 °C and transfer your reaction directly from 4 °C.<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Forward Primer (10 uM)</TD><TD>0.5 ul</TD></TR><TR><TD>Reverse Primer (10 uM)</TD><TD>0.5 ul</TD></TR><TR><TD>Template DNA (from above)</TD><TD>1.5 ul</TD></TR><TR><TD>Quick-Load® Taq 2X Master Mix</TD><TD>12.5 ul</TD></TR><TR><TD>Nuclease-free water</TD><TD>10 ul</TD></TR><TR><TD><B>Total Volume</B></TD><TD>25 ul</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD><B>Thermocycler Protocol: NEB Quick-Load</B></TD></TR><TR><TD>Temp</TD><TD>Time</TD></TR><TR><TD>Start</TD><TD>95°C </TD><TD>30 sec</TD><TD>Melt</TD></TR><TR><TD>Cycle 1</TD><TD>95°C</TD><TD>15 sec</TD><TD>Melt</TD><TD>35 cycles</TD></TR><TR><TD>Cycle 2</TD><TD>45.8 °C</TD><TD>30 sec</TD><TD>Anneal</TD></TR><TR><TD>Cycle 3</TD><TD>72°C</TD><TD>1 min per kb</TD><TD>Extend</TD></TR><TR><TD>Finish</TD><TD>72 °C</TD><TD>15 min</TD><TD>Extand</TD></TR><TR><TD>Store</TD><TD>10°C</TD><TD>Forever</TD><TD>Store</TD></TR></TBODY></TABLE><B>Gels</B>
100V, 30 min, 1%<B>Pictures:</B>
13/09/17_colony PCR_pS009.JPG
13/09/17_colony PCR_pS008_pS002.jpg
13/09/17_colony PCR_pS003.JPG
13/09/17_colony PCR_pS004.JPG
13/09/17_colony PCR_pS012.JPG
13/09/17_colony PCR_pS013.JPG<B>Result: </B>
pS003: row 3,5 might have worked -&gt; analytical digest
pS004: all might have worked -&gt; analytical digest
pS013: all might have worked -&gt; analytical digest
pS012: row 3,4,5 might have workes -&gt; analytical digest<B>Sequencing</B>
Bringing primers/plasmids away for sequencin
Bacbbone, SPCR2 with iS002, iS032, iS028, iS029, iS030</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Wednesday 16<SUP>th</SUP> September</H3><P><B><EM>
Ligations
</EM></B></P><B>Ligation</B>
SPCR2 - Chl <TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>SPCR2</TD><TD>1 µl</TD></TR><TR><TD>Chl</TD><TD>2,95 µl</TD></TR><TR><TD>H2O</TD><TD>4.55 µl</TD></TR><TR><TD>Fermentas T4 Ligase Buffer</TD><TD>1.0 µl</TD></TR><TR><TD>Fermentas T4 Ligase Enzyme</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>10 µl</TD></TR></TBODY></TABLE>
SPCR2 - Amp<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>SPCR2</TD><TD>1 µl</TD></TR><TR><TD>Amp</TD><TD>3.4 µl</TD></TR><TR><TD>H2O</TD><TD>4.1 µl</TD></TR><TR><TD>Fermentas T4 Ligase Buffer</TD><TD>1.0 µl</TD></TR><TR><TD>Fermentas T4 Ligase Enzyme</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>10 µl</TD></TR></TBODY></TABLE>
SPCR3 - Chl<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>SPCR3</TD><TD>1 µl</TD></TR><TR><TD>Amp</TD><TD>1.79 µl</TD></TR><TR><TD>H2O</TD><TD>5.71 µl</TD></TR><TR><TD>Fermentas T4 Ligase Buffer</TD><TD>1.0 µl</TD></TR><TR><TD>Fermentas T4 Ligase Enzyme</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>10 µl</TD></TR></TBODY></TABLE>
SPCR3 - Amp7
<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Amp</TD><TD>1 µl</TD></TR><TR><TD>SPCR 3</TD><TD>1.56 µl</TD></TR><TR><TD>H2O</TD><TD>5.94 µl</TD></TR><TR><TD>Fermentas T4 Ligase Buffer</TD><TD>1.0 µl</TD></TR><TR><TD>Fermentas T4 Ligase Enzyme</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>10 µl</TD></TR></TBODY></TABLE>
SPCR5 - Chl<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>SPCR 5</TD><TD>0.5 µl</TD></TR><TR><TD>Chl</TD><TD>4.55 µl</TD></TR><TR><TD>H2O</TD><TD>3.45 µl</TD></TR><TR><TD>Fermentas T4 Ligase Buffer</TD><TD>1.0 µl</TD></TR><TR><TD>Fermentas T4 Ligase Enzyme</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>10 µl</TD></TR></TBODY></TABLE>
SPCR11 - Chl<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Chl</TD><TD>1 µl</TD></TR><TR><TD>SPCR11</TD><TD>1.3 µl</TD></TR><TR><TD>H2O</TD><TD>6.2 µl</TD></TR><TR><TD>Fermentas T4 Ligase Buffer</TD><TD>1.0 µl</TD></TR><TR><TD>Fermentas T4 Ligase Enzyme</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>10 µl</TD></TR></TBODY></TABLE>
SPCR2 - Amp<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Amp</TD><TD>1 µl</TD></TR><TR><TD>SPCR11</TD><TD>1.13 µl</TD></TR><TR><TD>H2O</TD><TD>6.37 µl</TD></TR><TR><TD>Fermentas T4 Ligase Buffer</TD><TD>1.0 µl</TD></TR><TR><TD>Fermentas T4 Ligase Enzyme</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>10 µl</TD></TR></TBODY></TABLE>
at 16° for 1h<B>Transformation</B>
1) Thaw competent cells on ice (Gibson Kit competent cells)
2) Place 20 ul of cells in a pre-chilled Eppendorf tube.
3) Add 2.5 ul of plasmid (from Biobrick stock)
4) Mix gently by flicking the tube.
5) Chill on ice for 10 minutes.
4) Heat shock at 42 °C for 30 seconds 
5) Return to ice for 2 minutes.
6) Add 200 ul LB medium and recover the cells by shaking at 37 °C.
        	Ampicillin: 15-30 minutes
        	Chloramphenicol: 60-120 minutes
 7) Plate out the cells on selective LB (10ul)
8. Incubate at 37 °C. Transformants should appear within 12 hrs.<B>Gel of Ligation products</B>
1%, 100V, 40 min</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Sunday 22<SUP>nd</SUP> September</H3><P><B><EM>
PCR and Gels
</EM></B></P><B>PCR</B>
M13 backbone
SPCR16,17<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD><B>x1</B></TD></TR><TR><TD>Nuclease-free water</TD><TD>37.25 µl</TD></TR><TR><TD>5x Phusion HF Buffer</TD><TD>10 µl</TD></TR><TR><TD>10 mM dNTPs</TD><TD>1 µl</TD></TR><TR><TD>Forward Primer (10 uM)</TD><TD>0.5 µl</TD></TR><TR><TD>Reverse Primer (10 uM)</TD><TD>0.5 µl</TD></TR><TR><TD>Template Plasmid</TD><TD>0.25 µl</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>50 µl</TD></TR></TBODY></TABLE>
Protocol see fotos<B>Gel</B>
100V, 1%, 1,5h</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Monday 23<SUP>rd</SUP> September</H3><P><B><EM>
Gels and PCR
</EM></B></P><B>Gel</B>
100V, 1%, 1,5h
Picture: 13/09/23_SPCR16_17_corresponding primers - PCRs didn't worked<B>PCR</B>
SPCR 16 (normal phusion at 48°C)<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD><B>x1</B></TD></TR><TR><TD>Nuclease-free water</TD><TD>37.25 µl</TD></TR><TR><TD>5x Phusion HF Buffer</TD><TD>10 µl</TD></TR><TR><TD>10 mM dNTPs</TD><TD>1 µl</TD></TR><TR><TD>Forward Primer (10 uM)</TD><TD>0.5 µl</TD></TR><TR><TD>Reverse Primer (10 uM)</TD><TD>0.5 µl</TD></TR><TR><TD>Template Plasmid</TD><TD>0.25 µl</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>50 µl</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD><B>Thermocycler Protocol: NEB Phusion</B></TD></TR><TR><TD>Temp</TD><TD>Time</TD></TR><TR><TD>Start</TD><TD>98°C </TD><TD>30 sec</TD><TD>Melt</TD></TR><TR><TD>Cycle 1</TD><TD>98°C</TD><TD>5 sec</TD><TD>Melt</TD><TD>35 cycles</TD></TR><TR><TD>Cycle 2</TD><TD>72°C</TD><TD>30 sec</TD><TD>Anneal</TD></TR><TR><TD>Cycle 3</TD><TD>72°C</TD><TD>30 sec per kb</TD><TD>Extend</TD></TR><TR><TD>Finish</TD><TD>72 °C</TD><TD>5 min</TD><TD>Extand</TD></TR><TR><TD>Store</TD><TD>10°C</TD><TD>Forever</TD><TD>Store</TD></TR></TBODY></TABLE></DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Tuesday 24<SUP>th</SUP> September</H3><P><B><EM>
Overlap PCR, PCR of Overlap, Digestion, Gel extraction, Gels, PCR Purification,  PCR (SPCR2, SPCR3, SPCR5, SPCR6, SPCR7, SPCR8, SPCR9, SPCR10, SPCR11, linearize pSB1C3) and liquid culture
</EM></B></P><B>Overlap PCR</B>
SPCR5-SPCR2<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD><B>x1</B></TD></TR><TR><TD>Nuclease-free water</TD><TD>37.25 µl</TD></TR><TR><TD>5x Phusion HF Buffer</TD><TD>10 µl</TD></TR><TR><TD>10 mM dNTPs</TD><TD>1 µl</TD></TR><TR><TD>Template Plasmid(10 uM)</TD><TD>1 µl</TD></TR><TR><TD>Template Plasmid (10 uM)</TD><TD>1 µl</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>50 µl</TD></TR></TBODY></TABLE><B>PCR of Overlap</B><B>Digestion</B>
M13 (EcoRI, PstI), pSB1C3 (EcoRI, PstI), psB1C3 (XbaI, SpeI)<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Purified Plasmid</TD><TD>20 µl</TD></TR><TR><TD>H2O</TD><TD>63 µl</TD></TR><TR><TD>10x FastDigest Buffer</TD><TD>10 µl</TD></TR><TR><TD>FastDigest Enzyme 1</TD><TD>2.5 µl</TD></TR><TR><TD>FastDigest Enzyme 2</TD><TD>2.5 µl</TD></TR><TR><TD>FastAP Phosphatase</TD><TD>2 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>100 µl</TD></TR></TBODY></TABLE>
SPCR2 (EcoRI, PstI), SPCR5 (XbaI, SpeI)<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Purified PCR Product</TD><TD>16 µl</TD></TR><TR><TD>H2O</TD><TD>0 µl</TD></TR><TR><TD>10x FastDigest Buffer</TD><TD>2 µl</TD></TR><TR><TD>FastDigest Enzyme 1</TD><TD>1 µl</TD></TR><TR><TD>FastDigest Enzyme 2</TD><TD>1 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>20 µl</TD></TR></TBODY></TABLE><B>Gel</B>
pSB1C3
1%, 100V, 1h
Picture: 13/09/24_pSB1C3 digest_EcoRI/PstI_XbaI/SpeI - digest worked<B>Gel extraction</B>
1. Excise the DNA fragment from the agarose gel with a clean, sharp scalpel. Minimize the size of the gel slice by removing extra agarose.
2. Weigh the gel slice in a colorless tube. Add 3 volumes of Buffer QG to 1 volume of gel (100 mg ~ 100 μl). To help dissolve gel, mix by vortexing the tube every 2–3 min during the incubation.
3. After the gel slice has dissolved completely, check that the color of the mixture is yellow (similar to Buffer QG without dissolved agarose).
4. Add 1 gel volume of isopropanol to the sample and mix (actually only needed for very small products or very big products)
5. To bind DNA, apply the sample to the QIAquick column, and centrifuge for 1 min.
6. Discard flow-through and place QIAquick column back in the same collection tube.
7. To wash, add 0.75 ml of Buffer PE to QIAquick column and centrifuge for 1 min.
8. Discard the flow-through and centrifuge the QIAquick column for an additional 1 min at 17,900 x g (13,000 rpm).
9. Place QIAquick column into a clean 1.5 ml microcentrifuge tube.
10. To elute DNA, add 30 μl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge for 1 min.<B>Gel</B>
PCR Overlap
1%, 100V, 1h
Picture: 13/09/24_Overlap PCR PCR SCPR5_SPCR2 - PCR didn't work<B>PCR Purification (M13 (EcoRI, PstI), SPCR2 (EcoRI, PstI), SPCR5 (XbaI, SpeI))</B>
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 50 μl Buffer EB  to the center of the QIAquick membrane and centrifuge the column for 1 min.<B>PCR (SPCR2, SPCR3, SPCR5, SPCR6, SPCR7, SPCR8, SPCR9, SPCR10, SPCR11, linearize pSB1C3)</B>
each 8x<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD><B>1x</B></TD></TR><TR><TD>Nuclease-free water</TD><TD>37.25 µl</TD></TR><TR><TD>5x Phusion HF Buffer</TD><TD>10 µl</TD></TR><TR><TD>10 mM dNTPs</TD><TD>1 µl</TD></TR><TR><TD>Forward Primer (10 uM)</TD><TD>0.5 µl</TD></TR><TR><TD>Reverse Primer (10 uM)</TD><TD>0.5 µl</TD></TR><TR><TD>Template Plasmid</TD><TD>0.25 µl</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>50 µl</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD><B>Thermocycler Protocol: Fermentas Phusion</B></TD></TR><TR><TD>Temp</TD><TD>Time</TD></TR><TR><TD>Start</TD><TD>98°C </TD><TD>30 sec</TD><TD>Melt</TD></TR><TR><TD>Cycle 1</TD><TD>98°C</TD><TD>5 sec</TD><TD>Melt</TD><TD>35 cycles</TD></TR><TR><TD>Cycle 2</TD><TD>25 sec</TD><TD>Anneal</TD></TR><TR><TD>Cycle 3</TD><TD>72°C</TD><TD>30 sec per kb</TD><TD>Extend</TD></TR><TR><TD>Finish</TD><TD>72 °C</TD><TD>5 min</TD><TD>Extand</TD></TR><TR><TD>Store</TD><TD>10°C</TD><TD>Forever</TD><TD>Store</TD></TR></TBODY></TABLE><B>Liquid culture</B>
NEB, sSP004, Fr-433</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Wednesday 25<SUP>th</SUP> September</H3><P><B><EM>
Gel of PCR, PCR (SPCR5,6) and Miniprep Cas9, M13mp18
</EM></B></P><B>Gel of PCR</B>
1%, 100V, 1h
Picture: 13/09/25_SPCR2-11 - SPCR2: worked, SPCR3: worked, SPCR5: didn't work, SPCR6: didn't work, SPCR7: didn't work, SPCR8: didn't work, SPCR9: didn't work, SPCR10: didn't work, SCPR11: worked <B>PCR (SPCR5,6)</B><TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD><B>x1</B></TD></TR><TR><TD>Nuclease-free water</TD><TD>37.25 µl</TD></TR><TR><TD>5x Phusion HF Buffer</TD><TD>10 µl</TD></TR><TR><TD>10 mM dNTPs</TD><TD>1 µl</TD></TR><TR><TD>Forward Primer (10 uM)</TD><TD>1 µl</TD></TR><TR><TD>Reverse Primer (10 uM)</TD><TD>1 µl</TD></TR><TR><TD>Template Plasmid)</TD><TD>0.25 µl</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>50 µl</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD><B>Thermocycler Protocol: Fermentas Phusion</B></TD></TR><TR><TD>Temp</TD><TD>Time</TD></TR><TR><TD>Start</TD><TD>98°C </TD><TD>30 sec</TD><TD>Melt</TD></TR><TR><TD>Cycle 1</TD><TD>98°C</TD><TD>5 sec</TD><TD>Melt</TD><TD>35 cycles</TD></TR><TR><TD>Cycle 2</TD><TD>25 sec</TD><TD>Anneal</TD></TR><TR><TD>Cycle 3</TD><TD>72°C</TD><TD>30 sec per kb</TD><TD>Extend</TD></TR><TR><TD>Finish</TD><TD>72 °C</TD><TD>5 min</TD><TD>Extand</TD></TR><TR><TD>Store</TD><TD>10°C</TD><TD>Forever</TD><TD>Store</TD></TR></TBODY></TABLE><B>Miniprep Cas9, M13mp18</B>
1) Pellet liquid culture (4000rpm, 10 min)
2) Discard supernatant
3) resuspend the cells in 250ul of resuspension olution
4) add 250ul of lysis solution, mix by inverting 4-6 times
5) add 350ul of neutralization solution
4) centrifuge for 5 min
5) transfer supernatant to spin column
6) centrifuge for 1 min
7) discard flow through
8) add 500 ul wash solution and centrifuge for 1 min , discard flow through(repeat this step)
9) centrifuge for 1 min to remove left over liquid
10) transfer the column on a 1.5ml tube
11) add 50ul of elution buffer and incubate for 2 min
12) centrifuge for 2 min
13) Nanodrop the concentration and freeze at -20°</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Thursday 26<SUP>th</SUP> September</H3><P><B><EM>
Gel PCR 5,6, Digestion, PCR (SPCR7-10) and PCR purification
</EM></B></P><B>Gel PCR 5,6</B>
1%, 100V, 1h
Picture: 13/09/26_SPCR5_6 -  PCR didn't work<B>Digestions: SPCR2 (EcoRI, PstI), SPCR3 (EcoRI, PstI), SPCR4 (EcoRI, PstI), SPCR11 (EcoRI, PstI), lin pSB1C3 (EcoRI, PstI), lin pSB1A3 (EcoRI, PstI),  SPCR5 (XbaI, SpeI), SPCR7 (XbaI, SpeI), lin pSB1C3 (XbaI, SpeI), in pSB1A3 (XbaI, SpeI)</B><TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Purified PCR Product</TD><TD>16 µl</TD></TR><TR><TD>H2O</TD><TD>0 µl</TD></TR><TR><TD>10x FastDigest Buffer</TD><TD>2 µl</TD></TR><TR><TD>FastDigest Enzyme 1</TD><TD>1 µl</TD></TR><TR><TD>FastDigest Enzyme 2</TD><TD>1 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>20 µl</TD></TR></TBODY></TABLE><B>PCR (SPCR7-10)</B><TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD><B>x1</B></TD></TR><TR><TD>Nuclease-free water</TD><TD>37.25 µl</TD></TR><TR><TD>5x Phusion HF Buffer</TD><TD>10 µl</TD></TR><TR><TD>10 mM dNTPs</TD><TD>1 µl</TD></TR><TR><TD>Forward Primer (10 uM)</TD><TD>1 µl</TD></TR><TR><TD>Reverse Primer (10 uM)</TD><TD>1 µl</TD></TR><TR><TD>Template Plasmid)</TD><TD>0.25 µl</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>50 µl</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD><B>Thermocycler Protocol: Fermentas Phusion</B></TD></TR><TR><TD>Temp</TD><TD>Time</TD></TR><TR><TD>Start</TD><TD>98°C </TD><TD>30 sec</TD><TD>Melt</TD></TR><TR><TD>Cycle 1</TD><TD>98°C</TD><TD>5 sec</TD><TD>Melt</TD><TD>35 cycles</TD></TR><TR><TD>Cycle 2</TD><TD>25 sec</TD><TD>Anneal</TD></TR><TR><TD>Cycle 3</TD><TD>72°C</TD><TD>30 sec per kb</TD><TD>Extend</TD></TR><TR><TD>Finish</TD><TD>72 °C</TD><TD>5 min</TD><TD>Extand</TD></TR><TR><TD>Store</TD><TD>10°C</TD><TD>Forever</TD><TD>Store</TD></TR></TBODY></TABLE><B>PCR Purification: (SPCR2 (EcoRI, PstI), SPCR3 (EcoRI, PstI), SPCR4 (EcoRI, PstI), SPCR11 (EcoRI, PstI), lin pSB1C3 (EcoRI, PstI), lin pSB1A3 (EcoRI, PstI),  SPCR5 (XbaI, SpeI), SPCR7 (XbaI, SpeI), lin pSB1C3 (XbaI, SpeI), in pSB1A3 (XbaI, SpeI)</B>
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 50 μl Buffer EB  to the center of the QIAquick membrane and centrifuge the column for 1 min.</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Friday 27<SUP>th</SUP> September</H3><P><B><EM>
Gel SPCR7-10, NEB liquid culture, Ligation, Electrocompetent cells and transformation
</EM></B></P><B>Gel SPCR7-10</B>
1%, 100V, 1h
Picture: 13/09/27_SPCR7_8_9_10 - PCRs worked<B>NEB liquid culture</B>
1:1000 of ON culture<B>Ligation</B>
pS006<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>SPCR5</TD><TD>1 µl</TD></TR><TR><TD>SPCR7</TD><TD>0,28 µl</TD></TR><TR><TD>H2O</TD><TD>7.22 µl</TD></TR><TR><TD>Fermentas T4 Ligase Buffer</TD><TD>1 µl</TD></TR><TR><TD>Fermentas T4 Ligase Enzime</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>10 µl</TD></TR></TBODY></TABLE>
pS006’
<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>SPCR5</TD><TD>1 µl</TD></TR><TR><TD>lin pSB1C3 (XbaI/SpeI)</TD><TD>0,47 µl</TD></TR><TR><TD>H2O</TD><TD>7.03 µl</TD></TR><TR><TD>Fermentas T4 Ligase Buffer</TD><TD>1 µl</TD></TR><TR><TD>Fermentas T4 Ligase Enzime</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>10 µl</TD></TR></TBODY></TABLE>
pS006*
<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>SPCR5</TD><TD>1 µl</TD></TR><TR><TD>lin pSB1A3 (XbaI/SpeI)</TD><TD>0,67 µl</TD></TR><TR><TD>H2O</TD><TD>6.83 µl</TD></TR><TR><TD>Fermentas T4 Ligase Buffer</TD><TD>1 µl</TD></TR><TR><TD>Fermentas T4 Ligase Enzime</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>10 µl</TD></TR></TBODY></TABLE>
pS013
<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>SPCR4</TD><TD>1 µl</TD></TR><TR><TD>SPCR11</TD><TD>0,24 µl</TD></TR><TR><TD>H2O</TD><TD>7.26 µl</TD></TR><TR><TD>Fermentas T4 Ligase Buffer</TD><TD>1 µl</TD></TR><TR><TD>Fermentas T4 Ligase Enzime</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>10 µl</TD></TR></TBODY></TABLE>
pS013'
<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>lin pSB1C3 (EcoRI, SpeI)</TD><TD>1 µl</TD></TR><TR><TD>SPCR11</TD><TD>0,28 µl</TD></TR><TR><TD>H2O</TD><TD>7.22 µl</TD></TR><TR><TD>Fermentas T4 Ligase Buffer</TD><TD>1 µl</TD></TR><TR><TD>Fermentas T4 Ligase Enzime</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>10 µl</TD></TR></TBODY></TABLE>
pS013*
<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>lin pSB1A3 (EcoRI, PstI))</TD><TD>1 µl</TD></TR><TR><TD>SPCR11</TD><TD>0,3 µl</TD></TR><TR><TD>H2O</TD><TD>7.2 µl</TD></TR><TR><TD>Fermentas T4 Ligase Buffer</TD><TD>1 µl</TD></TR><TR><TD>Fermentas T4 Ligase Enzime</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>10 µl</TD></TR></TBODY></TABLE>
pS004
<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>SPCR4</TD><TD>1 µl</TD></TR><TR><TD>SPCR3</TD><TD>0,2 µl</TD></TR><TR><TD>H2O</TD><TD>7.3 µl</TD></TR><TR><TD>Fermentas T4 Ligase Buffer</TD><TD>1 µl</TD></TR><TR><TD>Fermentas T4 Ligase Enzime</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>10 µl</TD></TR></TBODY></TABLE>
pS004'
<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>lin pSB1C3 (EcoRI, PstI)</TD><TD>1 µl</TD></TR><TR><TD>SPCR3</TD><TD>0,3 µl</TD></TR><TR><TD>H2O</TD><TD>7.2 µl</TD></TR><TR><TD>Fermentas T4 Ligase Buffer</TD><TD>1 µl</TD></TR><TR><TD>Fermentas T4 Ligase Enzime</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>10 µl</TD></TR></TBODY></TABLE>
pS004*
<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>lin pSB1A3 (EcoRI, PstI)</TD><TD>1 µl</TD></TR><TR><TD>SPCR3</TD><TD>0,27 µl</TD></TR><TR><TD>H2O</TD><TD>7.23 µl</TD></TR><TR><TD>Fermentas T4 Ligase Buffer</TD><TD>1 µl</TD></TR><TR><TD>Fermentas T4 Ligase Enzime</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>10 µl</TD></TR></TBODY></TABLE>
pS002'
<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>SPCR2</TD><TD>1 µl</TD></TR><TR><TD>lin pSB1C3 (EcoRI, PstI)</TD><TD>0,8 µl</TD></TR><TR><TD>H2O</TD><TD>6.7 µl</TD></TR><TR><TD>Fermentas T4 Ligase Buffer</TD><TD>1 µl</TD></TR><TR><TD>Fermentas T4 Ligase Enzime</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>10 µl</TD></TR></TBODY></TABLE>
pS002*
<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>SPCR2</TD><TD>1 µl</TD></TR><TR><TD>lin pSB1A3 (EcoRI, PstI)</TD><TD>0,6 µl</TD></TR><TR><TD>H2O</TD><TD>6.9 µl</TD></TR><TR><TD>Fermentas T4 Ligase Buffer</TD><TD>1 µl</TD></TR><TR><TD>Fermentas T4 Ligase Enzime</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>10 µl</TD></TR></TBODY></TABLE>
45min at 22°, 15 min at 16°<B>Electrocompetent cells</B>
1) Start Culture
2) When the cells reach an OD600 of 0.2. 
3) Harvest the cells.
        	When the cells reach an OD600 of between 0.6 and 0.8 (OD: 0,683)
        	Centrifuge at 4 °C for 10 min at 4000 rpm.
4) Wash and combine the cells.
        	Remove the supernatant.
        	Resuspend the cells in 25 ml of ice cold water
5) Wash the cells 2 more times.
        	Centrifuge at 4 °C for 10 min at 4000 rpm.
        	Resuspend in 50 ml of ice cold water.
        	Repeat.
6) Wash and concentrate the cells for electroporation.
        	Centrifuge at 4 °C for 10 min at 4000 rpm.
        	Resuspend in 1-2 ml of ice cold water.
        	We will use 200 ul of washed cells per transformation.<B>Transformation</B>
pS006, pS006’, pS006*, pS013, pS013’, pS013*, pS004, pS004’, pS004*, pS002’, pS002*
1) Prepare BD tubes with a pipette filled with LB at the interior of each tube (pipette supplied with the electroporation cuvette)
2) Test the purity of the electrocompetent cells.
        	Add 200 ul of washed cells to a cuvette.
3) Mix the cells and DNA in a cuvette.
        	200 ul of washed cells with 200 ng of PCR product.
        	Keep the cuvette on ice until just before the electroporation.
4) Preload a pipette with 1 ml of LB.
5) Pulse the cuvette with voltage.
        	Dry the electrodes with a kimwipe prior to loading.
        	Use the EC2 setting.
6) Listen for arcing.
        	A cracking sound means all the cells are dead.
        	Note the time constant: 5 is good, 5.8 is great.
7) Immediately recover the cells.
        	Add the 1 ml of preloaded LB and pipet up and down to mix.
        	Collect 1 ml of cells, some volume is lost in the cuvette.
8) Incubate 1/2 h at 37 °C with shaking.
9) Plate 10/200 ul of recovered cells on selective plates.
        	Use antibiotic appropriate to the part being integrated.
        	Let the other 900 ul rest overnight at room temperature.
10) Concentrate and plate the remaining cells
        	Spin down quickly and resuspend in 100 ul LB before plating.
Transformed cells should be incubated at 37 °C.
Colonies should appear 24-48 h after plating.</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Saturday 28<SUP>th</SUP> September</H3><P><B><EM>
Extraction of Genomic DNA, PCR Reaction and Restreak plates 
</EM></B></P><B>Protocol: E. coli Colony PCR (pS006’, pS006*, pS013, pS004, pS004*, pS002*)</B><B>Extraction of Genomic DNA</B>
1) Pick a single colony into 50 ul of H20.
        	Fresh colonies (grown that day) work best, but they can also come from 4 C.
	Pipet 2ul onto plates to have C1-C8
	Pipet 2 ul into 5ml Media with antibiotics to make liquid cultures
2) Boil for 5 minutes.
        	1.5 ul of this can be used directly for PCR.
        	Best if used directly, but can also be stored at 4 C for a few days.<B>PCR Reaction</B>
Keep all the reagents at 4 °C while preparing the mixture.
Pre-heat the thermocycler to 95 °C and transfer your reaction directly from 4 °C.<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD><TD><B>9x Volume</B></TD></TR><TR><TD>Forward Primer (10 uM)</TD><TD>0.5 µl</TD><TD>4.5 µl</TD></TR><TR><TD>Reverse Primer (10 uM)</TD><TD>0.5 µl</TD><TD>4.5 µl</TD></TR><TR><TD>Template DNA (from above)</TD><TD>1.5 µl</TD><TD>13.5 µl</TD></TR><TR><TD>Quick-Load® Taq 2X Master Mix</TD><TD>12.5 µl</TD><TD>112.5 µl9x Volume</TD></TR><TR><TD>Nuclease-free water</TD><TD>10 µl</TD><TD>90 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>25 µl</TD><TD>125 µl</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD><B>Thermocycler Protocol: Green Dream Taq</B></TD></TR><TR><TD>Temp</TD><TD>Time</TD></TR><TR><TD>Start</TD><TD>95°C </TD><TD>30 sec</TD><TD>Melt</TD></TR><TR><TD>Cycle 1</TD><TD>95°C</TD><TD>15 sec</TD><TD>Melt</TD><TD>35 cycles</TD></TR><TR><TD>Cycle 2</TD><TD>46.8°C </TD><TD>30 sec</TD><TD>Anneal</TD></TR><TR><TD>Cycle 3</TD><TD>72°C</TD><TD>1 min per kb</TD><TD>Extend</TD></TR><TR><TD>Finish</TD><TD>72 °C</TD><TD>10 min</TD><TD>Extand</TD></TR><TR><TD>Store</TD><TD>10°C</TD><TD>Forever</TD><TD>Store</TD></TR></TBODY></TABLE><B>Restreak plates</B>
Some of the plates were too full grown -&gt; Rstreak for single colonies</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Monday 30<SUP>th</SUP> September</H3><P><B><EM>
Extraction of Genomic DNA, PCR reaction, Gels, Miniprep, PCR: SPCR5, SPCR6  and Liquid culture of pS004’ and pS013’
</EM></B></P><B>Protocol: E. coli Colony PCR (pS006, pS013’,pS013*,  pS004’, pS002’)</B><B>Extraction of Genomic DNA</B>
1) Pick a single colony into 50 ul of H20.
        	Fresh colonies (grown that day) work best, but they can also come from 4 C.
	Pipet 2ul onto plates to have C1-C8	
2) Boil for 5 minutes.
        	1.5 ul of this can be used directly for PCR.
        	Best if used directly, but can also be stored at 4 C for a few days.<B>PCR Reaction</B>
Keep all the reagents at 4 °C while preparing the mixture.
Pre-heat the thermocycler to 95 °C and transfer your reaction directly from 4 °C.<TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD><TD><B>9x Volume</B></TD></TR><TR><TD>Forward Primer (10 uM)</TD><TD>0.5 µl</TD><TD>4.5 µl</TD></TR><TR><TD>Reverse Primer (10 uM)</TD><TD>0.5 µl</TD><TD>4.5 µl</TD></TR><TR><TD>Template DNA (from above)</TD><TD>1.5 µl</TD><TD>13.5 µl</TD></TR><TR><TD>Quick-Load® Taq 2X Master Mix</TD><TD>12.5 µl</TD><TD>112.5 µl9x Volume</TD></TR><TR><TD>Nuclease-free water</TD><TD>10 µl</TD><TD>90 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>25 µl</TD><TD>125 µl</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD><B>Thermocycler Protocol: Green Dream Taq</B></TD></TR><TR><TD>Temp</TD><TD>Time</TD></TR><TR><TD>Start</TD><TD>95°C </TD><TD>30 sec</TD><TD>Melt</TD></TR><TR><TD>Cycle 1</TD><TD>95°C</TD><TD>15 sec</TD><TD>Melt</TD><TD>35 cycles</TD></TR><TR><TD>Cycle 2</TD><TD>46.8°C </TD><TD>30 sec</TD><TD>Anneal</TD></TR><TR><TD>Cycle 3</TD><TD>72°C</TD><TD>1 min per kb</TD><TD>Extend</TD></TR><TR><TD>Finish</TD><TD>72 °C</TD><TD>10 min</TD><TD>Extand</TD></TR><TR><TD>Store</TD><TD>10°C</TD><TD>Forever</TD><TD>Store</TD></TR></TBODY></TABLE><B>Gels</B>
1%, 100V, 45 min

Pictures + Results
13/09/30_colony pcr_pS004_pS013 - 1-8: pS004 (gRNA - Amp), 9-16: pS013 (crRNA - Amp) - seems to have worked beside pS013 C7
13/09/30_colony PCR_pS004*_pS002* - 1-8: pS004* (gRNA - Amp), 9-16: pS002 (reporter - Amp) - pS004* seems to have worked (C1, C2, C3, C6)
13/09/30_colony PCR_pS006'_pS006* - 1-8: pS006' (Cas9 - Chl), 9-16: pS006* (Cas9 - Amp) - didn’t work
13/09/30_Colony PCR_pS013'_pS013* - 1-8: pS013' (crRNA - Chl), 9-16: pS013* (crRNA - Amp) - Seems to have worked beside 13’ C7, C8
13/09/30_colony PCR_pS004' - 1-8: pS004' (gRNA - Chl), 9-16 - pS004' seems to have worked beside C6
13/09/30_digestion of pS006 and pS002 (Cas9, reporter): seem to have worked

other gel images that proof that the reporter as well as the Cas9 construct worked will follow soon!
<B>Miniprep</B>
pS013, pS004
1) Pellet liquid culture (4000rpm, 10 min)
2) Discard supernatant
3) resuspend the cells in 250ul of resuspension olution
4) add 250ul of lysis solution, mix by inverting 4-6 times
5) add 350ul of neutralization solution
4) centrifuge for 5 min
5) transfer supernatant to spin column
6) centrifuge for 1 min
7) discard flow through
8) add 500 ul wash solution and centrifuge for 1 min , discard flow through(repeat this step)
9) centrifuge for 1 min to remove left over liquid
10) transfer the column on a 1.5ml tube
11) add 50ul of elution buffer and incubate for 2 min
12) centrifuge for 2 min
13) Nanodrop the concentration and freeze at -20°<B>PCR: SPCR5, SPCR6 (gradient + touch down 55-75, -1°/cycle)</B><TABLE border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD><B>11x</B></TD></TR><TR><TD>Nuclease-free water</TD><TD>37.25 µl</TD></TR><TR><TD>5x Phusion HF Buffer</TD><TD>10 µl</TD></TR><TR><TD>10 mM dNTPs</TD><TD>1 µl</TD></TR><TR><TD>Forward Primer (10 uM)</TD><TD>0.5 µl</TD></TR><TR><TD>Reverse Primer (10 uM)</TD><TD>0.5 µl</TD></TR><TR><TD>Template Plasmid)</TD><TD>0.25 µl</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>0.5 µl</TD></TR><TR><TD><B>Total Volume</B></TD><TD>50 µl</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TD><B>Thermocycler Protocol: Fermentas Phusion</B></TD></TR><TR><TD>Temp</TD><TD>Time</TD></TR><TR><TD>Start</TD><TD>98°C </TD><TD>30 sec</TD><TD>Melt</TD></TR><TR><TD>Cycle 1</TD><TD>98°C</TD><TD>5 sec</TD><TD>Melt</TD><TD>35 cycles</TD></TR><TR><TD>Cycle 2</TD><TD>25 sec</TD><TD>Anneal</TD></TR><TR><TD>Cycle 3</TD><TD>72°C</TD><TD>30 sec per kb</TD><TD>Extend</TD></TR><TR><TD>Finish</TD><TD>72 °C</TD><TD>5 min</TD><TD>Extand</TD></TR><TR><TD>Store</TD><TD>10°C</TD><TD>Forever</TD><TD>Store</TD></TR></TBODY></TABLE><B>Liquid culture of pS004’ and pS013’</B></DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Tuesday 1<SUP>st</SUP> October</H3><P><B><EM>
Miniprep
pS013’, pS004’
</EM></B></P>

1) Pellet liquid culture (4000rpm, 10 min)
2) Discard supernatant
3) resuspend the cells in 250ul of resuspension olution
4) add 250ul of lysis solution, mix by inverting 4-6 times
5) add 350ul of neutralization solution
4) centrifuge for 5 min
5) transfer supernatant to spin column
6) centrifuge for 1 min
7) discard flow through
8) add 500 ul wash solution and centrifuge for 1 min , discard flow through(repeat this step)
9) centrifuge for 1 min to remove left over liquid
10) transfer the column on a 1.5ml tube
11) add 50ul of elution buffer and incubate for 2 min
12) centrifuge for 2 min
13) Nanodrop the concentration and freeze at -20°</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3> Wednesday 2<SUP>nd</SUP> October</H3><P><B><EM>
Preparation killing assay
</EM></B></P><B>Liquid cultures</B>
R16, R26, R26f, delta PyrF, parental delta PyrF, NEB, M13, Litmus + Helper, Litmus GFP + helper
</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Wedsnesday 3<SUP>rd</SUP> October</H3><P><B><EM>
Killing assay to test specifity of CRISPR Cas
</EM></B></P><B>Electrocompetent cells (delta PyrF, parental, NEB)</B>
1) Start Culture

2) When the cells reach an OD600 of 0.2.

3) Harvest the cells.

           When the cells reach an OD600 of between 0.6 and 0.8 (OD: 0,683)

           Centrifuge at 4 °C for 10 min at 4000 rpm.

4) Wash and combine the cells.

           Remove the supernatant.

           Resuspend the cells in 25 ml of ice cold water

5) Wash the cells 2 more times.

           Centrifuge at 4 °C for 10 min at 4000 rpm.

           Resuspend in 50 ml of ice cold water.

           Repeat.

6) Wash and concentrate the cells for electroporation.

           Centrifuge at 4 °C for 10 min at 4000 rpm.

           Resuspend in 1-2 ml of ice cold water.

           We will use 200 ul of washed cells per transformation.
<B>Transformation </B>
Cotransform into parental/keio delta pyRF: pS006* (Cas9 in Amp BB) + pS004'(gRNA in Chl BB).
1) Prepare BD tubes with a pipette filled with LB at the interior of each tube (pipette supplied with the electroporation cuvette)

2) Test the purity of the electrocompetent cells.

           Add 200 ul of washed cells to a cuvette.

3) Mix the cells and DNA in a cuvette.

           200 ul of washed cells with 200 ng of PCR product.

           Keep the cuvette on ice until just before the electroporation.

4) Preload a pipette with 1 ml of LB.

5) Pulse the cuvette with voltage.

           Dry the electrodes with a kimwipe prior to loading.

           Use the EC2 setting.

6) Listen for arcing.

           A cracking sound means all the cells are dead.

           Note the time constant: 5 is good, 5.8 is great.

7) Immediately recover the cells.

           Add the 1 ml of preloaded LB and pipet up and down to mix.

           Collect 1 ml of cells, some volume is lost in the cuvette.

8) Incubate 1/2 h at 37 °C with shaking.

9) Plate 10/200 ul of recovered cells on selective plates.

           Use antibiotic appropriate to the part being integrated.

           Let the other 900 ul rest overnight at room temperature.

10) Concentrate and plate the remaining cells

           Spin down quickly and resuspend in 100 ul LB before plating.

Transformed cells should be incubated at 37 °C.

Colonies should appear 24-48 h after plating.

Plating of 10/100ul on 
1. Amp plates -&gt; selection for only Cas9
2. Chl plates -&gt; selection for only gRNA
3. Amp/Chl plates -&gt; selection for theoretically functional system</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Monday 14<SUP>th</SUP> October</H3><P><B><EM>
Liquid cultures sSP008, sSP009, R16, R26, R26f, sT007, sSP021, SCPR2-A3, SPCE2-C3
</EM></B></P></DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Tuesday 15<SUP>th</SUP> October</H3><P><B><EM>
Digestion Reporter and SPCR3
</EM></B></P>

BB: SpeI, PstI
SPCR: XbaI, PstI
2h 37°, heat inactivation
Electrocompetent cells: sSP008, sSP009
Transform SPCR2-A3/C3 both into sSP008/9</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Wednesday 16<SUP>th</SUP> October</H3><P><B><EM>
Miniprep of pS002, pS002’, pS006*
Liquid culture sSP017, pS004, pS004’
Glycerols
</EM></B></P></DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Thursday 17<SUP>th</SUP> October</H3><P><B><EM>
Minipreps, Digestions, Ligation and Transformation
</EM></B></P>
Miniprep sSP017, pS004, pS004’
Digest Litmus pS018 EcoRI/PstI, EcoRI/XbaI 2h 37° shaking
PCR purify
Ligation:
1. gRNA+phagmid (10,4+6)
2.phagemid+gRNA+reporter (9,6+1,86+2)
3.phagemid+gRNA+Cas9 (10,2+2,04+3)
1h at 22°
Transform
Heat shock, cells matt
Recover, plate</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Friday 18<SUP>th</SUP> October</H3><P><B><EM>
Miller assay and Liquid phagemid
</EM></B></P><B> Miller assay</B><LI> Grow liquid cultures up to an OD 600 of 0.3-0.7</LI><LI> Meassure the OD 600 </LI><LI> Start the specific cultures you want to test under the specific conditions</LI><LI> Take each 600 ul of culture and add 400ul of Z buffer (see below)</LI><LI> Add 100ul of Chloroform and 50ul of 0.1% SDS and vortex intensively</LI><LI> Incubate the tubes for 5min at 28°C</LI><LI> Add 200ul of ONPG (see below) and note the time point of adding</LI><LI> Incubate for up to 90 min or until a yellow/green color appears</LI><LI> To stop the reaction, add 500ul of Na2CO3 and note the time point of stopping</LI><LI> Centrifuge for 3 min</LI><LI> Measure the OD at 420</LI>
The expression of beta-galactosidase can now be calculated with the formula: Miller unit= (1000*OD420)/(delta t*V(culture taken)*OD600)

Z-buffer:
<LI>Na2HPO4 7H2O 60 mM - 16.1g </LI><LI>NaH4PO4 H2O 40 mM - 5.5g </LI><LI>KCl 10 mM - 0.75g </LI><LI>MgSO4 7H2O 1 mM - 0.246g </LI><LI>beta mercaptoethanol 50 mM - 2.7 ml </LI>
ONPG:

4mg/ml in Z-buffer

Na2CO3 1M
</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Saturday 19<SUP>th</SUP> October</H3><P><B><EM>
Miller assay, Digestions
</EM></B></P><B>Miller
Miniprep phagemid
Digest phagemid E/P, E/X, SPCR3 E,P</B></DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Tuesday 22<SUP>nd</SUP> October</H3><P><B><EM>
Liquid for Miller and Digest&amp;purify SPCR3 E,P
</EM></B></P></DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Thursday 24<SUP>th</SUP> October</H3><P><B><EM>
Digestion, Ligation, Transformation Ligation, Transformation
</EM></B></P><B>Digest phagemid E/P</B>
Purify it<B>Ligation:</B>
1. SPCR3 E/P + phagemid E/P
2. SPCR3 E/P + SPCR5 P/X+ phagemid E/X
2. SPCR3 E/P + SPCR2 P/X+ phagemid E/X<B>trafo Ligations
electrocmpetent cells parental+sSP009+pS002’
trafo:</B>
1. parental+pS002
2. parental+pS002’
3. parental+pS006*
4. parental+pS002’+pS006*
5. sSP009 pS002’ + pS006*
trafo helper into NEB
liquid cultures: sSP009, pRECA strains Ariel, sSP008+pS002, sSP008+pS002’

</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Friday 25<SUP>th</SUP> October</H3><P><B><EM>
Liquid cultures, Transformation, Conjugation
</EM></B></P><B>Colony PCR
Liquid Cloning</B>
Liquid: NEB helper, parental pS002, parental pS006*, sSP008+pS002, sSP008+pS002’
Miniprep phagemid gRNA
Trafo: NEB helper+phageid-gRNA, parental ps006*+pS002’, pyrfF F+ pS002’+pS006*
conjugation of parental pS006* with pS009
Liquid library
Restreak for library<B> Functionality Test of reporter</B>
1) sSP008+pS002/pS002’ +Xgal
2) sSP008+pS002/pS002’ +Xgal + MMC
3) sSP008+pS002/pS002’ +MMC</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Saturday 26<SUP>th</SUP> October</H3><P><B><EM>
Liquid cultures, Transformation, Conjugation
</EM></B></P><B>Liquid cultures</B>
sSP008, transformed parental strains, Kan strains, NEB helper-gRNA, sSP009
Conjugate at OD 0.2 parental + pS002'+pS006* with sSP009 and plate 50ul on Amp/Tet/Chl plates<B> Electrocompetent cells+ transformation </B>
Trafo:sSP008+Cas9 Addgene+ phagemid-gRNA -&gt; killing assay to confirm gRNA on phagemid works , pyrfF F+ pS002’+pS006*</DIV><DIV class="tbnote"><H2>Detect</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" target="_blank" class="tbnotelogo PSlogo"> ASDF </A><H3>Sunday 27<SUP>th</SUP> October</H3><P><B><EM>
Miller assay final strain/construct with controls
</EM></B></P><B>Miller assay</B><LI> MG1655 as positive control </LI><LI> parental + reporter + Cas9 + F+ </LI><LI> Keio delta pyrF + reporter + Cas9 + F+ </LI><LI> parental + reporter + Cas9 + F+ + phagemid with gRNA</LI><LI> Keio delta pyrF + reporter + Cas9 + F+ + phagemid with gRNA</LI><LI> Keio delta pyrF + reporter + MMC</LI><LI> parental + reporter + MMC </LI><B>Liquid culture functionality test</B><LI> parental + reporter + Cas9 + F+ + Xgal</LI><LI> parental + reporter + Cas9 + F+ + Xgal + phagemid with gRNA</LI><LI> parental + reporter + Cas9 + F+ + phagemid with gRNA</LI><LI> Keio delta pyrF + reporter + Cas9 + F+ + Xgal</LI><LI> Keio delta pyrF + reporter + Cas9 + F+ + Xgal + phagemid with gRNA</LI><LI> Keio delta pyrF + reporter + Cas9 + F+ +  phagemid with gRNA</LI><B>Killing assay</B>
Plate:
<LI> parental + reporter + Cas9 + F+ </LI><LI> Keio delta pyrF + reporter + Cas9 + F+ </LI><LI> parental + reporter + Cas9 + F+ + phagemid with gRNA</LI><LI> Keio delta pyrF + reporter + Cas9 + F+ + phagemid with gRNA</LI></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Monday 10<SUP>th</SUP> June</H3><P><B><EM>
Design of SirA genes for synthesis
</EM></B></P><B>SirA</B><P><EM>Mycobacterium tuberculosis</EM>Reference: Pinto et al. 2007. Sulfite Reduction in Mycobacteria doi:  10.1128/JB.00487-07
Primers:
TBsirA F      GGAATTCCATATGTCCGCGAAGGAGAACCCC
TBsirA R      AAGGAAAAAAGCGGCCGCTCATCGCAGGTCGTCCTCCTCGGCCCGGAT 
We blasted the above primers used to clone sirA from M. tuberculosis into E. coli.  We then extracted the sequence from genbank to retrieve the following sequence which includes the restriction sites introduced by Pinto et al.</P><P style="font-family: courier">ggaattccat atgtccgcga aggagaaccc ccaaatgacc actgcacgtc ccgccaaggc tcgaaatgag
ggccagtggg cgctgggaca tcgcgagcca ctcaacgcca acgaagagct gaagaaggcc
ggcaacccgc tcgacgtgcg ggagcgcatc gaaaacatct acgccaaaca gggtttcgac
agcatcgaca agaccgacct gcgagggcgc tttcgctggt ggggcctgta cacccagcgt
gagcagggct acgacggcac ctggaccggt gacgacaaca tcgacaagct cgaggccaaa
tacttcatga tgcgggtgcg ttgcgacggc ggcgcgctct cggctgccgc gctgcgcacg
ctgggccaga tctcgacgga gttcgcgcgc gataccgccg atatctccga ccggcagaac
gtgcaatacc actggatcga agtggaaaac gtccctgaaa tctggcgacg gttagacgat
gtcggactgc agaccaccga ggcgtgcggt gactgcccgc gggtagtgct gggctcgccg
ttggccggcg agtcgctcga cgaagtgctc gacccgacct gggcgatcga ggagatcgtg
cgtcgctaca tcggcaagcc cgacttcgcc gacttgccgc gcaagtacaa gaccgccatc
tctggcctgc aggacgtcgc gcacgagatc aacgacgtcg ccttcatcgg cgtcaaccat
cccgagcacg gaccaggcct ggatctgtgg gtgggcggtg gactgtcgac caacccgatg
ctggcccagc gggtcggcgc ctgggttcca ctgggcgaag tgcccgaggt gtgggcggcg
gtcacctcgg tgtttcgcga ctacggctac cggcgactgc gcgccaaggc ccggctgaaa
tttctgatca aagactgggg catagcgaag ttccgcgaag tgctcgaaac cgagtacctc
aagcgtccgc tgatcgacgg tccggccccc gaaccggtca agcatccgat cgaccacgtc
ggggtgcaac gactcaagaa cgggctcaac gccgtcggag tcgcccccat cgccgggcgg
gtatcgggca ccatcctcac ggcggtcgcc gacctgatgg cgcgggccgg ttccgaccgg
atccggttca ccccctacca gaagctggtc atcctcgaca ttccggacgc cttgctcgac
gacttgatcg ccggtctgga cgcgctgggg ctgcagtcgc gcccgtcgca ttggcgccgg
aacttgatgg cgtgcagcgg gattgagttc tgcaagttgt cattcgccga aacccgggtt
cgagcacagc atttggtgcc cgagctggaa cgccggcttg aggacatcaa ctcgcagctc
gacgtaccga tcaccgtcaa catcaacggc tgcccgaact catgtgcgcg aattcaaatc
gccgacatcg gattcaaggg acagatgatc gacgacggac acggcggctc cgtcgaaggc
ttccaggtgc atctgggcgg acacctcggc ctggatgccg gattcggccg caaactgcgc
cagcacaagg tcaccagtga cgaactcggc gactacatcg accgggtggt gcgcaacttc
gtcaaacacc gcagcgaagg tgaacgcttc gcgcagtggg tcatccgggc cgaggaggac
gacctgcgat gagcggccgc ttttttcctt</P><P>This sequence was then inserted into Geneious.  The PstI and EcoRI sites were manually removed and replaced with silent mutations.  A NcoI site was inserted at the start codon and an extra glycine was added after the start codon for the NcoI site to function properly.  We chose glycine as it is small an unlikely to interfere with either polar or non-polar domains within the protein.  We used GGC as the codon as it is the highest used glycine codon in e. coli.  We also added an AvrII site on the 3’ end of the site.  We then added biobrick restriction sites on the ends of the sequence (EcoRI, XbaI, SpeI, PstI).   This will allow us to clone into our expression vector pACYCDuet-1 with NcoI and AvrII enzymes and into a biobrick registry with EcoRI and PstI.  The insertion into pACYCDuet-1 removes both protein tags and cuts out the second MCS resulting in vector pACYCDuet-TBsirA.  The resulting sequence for TB SirA follows:</P><P style="font-family: courier">catggaattc tagaccatgg gctccgcgaa ggagaacccc caaatgacca ctgcacgtcc cgccaaggct cgaaatgagg gccagtgggc gctgggacat cgcgagccac tcaacgccaa cgaagagctg aagaaggccg gcaacccgct cgacgtgcgg gagcgcatcg aaaacatcta cgccaaacag ggtttcgaca gcatcgacaa gaccgacctg cgagggcgct ttcgctggtg gggcctgtac acccagcgtg agcagggcta cgacggcacc tggaccggtg acgacaacat cgacaagctc gaggccaaat acttcatgat gcgggtgcgt tgcgacggcg gcgcgctctc ggctgccgcg ctgcgcacgc tgggccagat ctcgacggag ttcgcgcgcg ataccgccga tatctccgac cggcagaacg tgcaatacca ctggatcgaa gtggaaaacg tccctgaaat ctggcgacgg ttagacgatg tcggacttca gaccaccgag gcgtgcggtg actgcccgcg ggtagtgctg ggctcgccgt tggccggcga gtcgctcgac gaagtgctcg acccgacctg ggcgatcgag gagatcgtgc gtcgctacat cggcaagccc gacttcgccg acttgccgcg caagtacaag accgccatct ctggccttca ggacgtcgcg cacgagatca acgacgtcgc cttcatcggc gtcaaccatc ccgagcacgg accaggcctg gatctgtggg tgggcggtgg actgtcgacc aacccgatgc tggcccagcg ggtcggcgcc tgggttccac tgggcgaagt gcccgaggtg tgggcggcgg tcacctcggt gtttcgcgac tacggctacc ggcgactgcg cgccaaggcc cggctgaaat ttctgatcaa agactggggc atagcgaagt tccgcgaagt gctcgaaacc gagtacctca agcgtccgct gatcgacggt ccggcccccg aaccggtcaa gcatccgatc gaccacgtcg gggtgcaacg actcaagaac gggctcaacg ccgtcggagt cgcccccatc gccgggcggg tatcgggcac catcctcacg gcggtcgccg acctgatggc gcgggccggt tccgaccgga tccggttcac cccctaccag aagctggtca tcctcgacat tccggacgcc ttgctcgacg acttgatcgc cggtctggac gcgctggggc ttcagtcgcg cccgtcgcat tggcgccgga acttgatggc gtgcagcggg attgagttct gcaagttgtc attcgccgaa acccgggttc gagcacagca tttggtgccc gagctggaac gccggcttga ggacatcaac tcgcagctcg acgtaccgat caccgtcaac atcaacggct gcccgaactc atgtgcgcga gttcaaatcg ccgacatcgg attcaaggga cagatgatcg acgacggaca cggcggctcc gtcgaaggct tccaggtgca tctgggcgga cacctcggcc tggatgccgg attcggccgc aaactgcgcc agcacaaggt caccagtgac gaactcggcg actacatcga ccgggtggtg cgcaacttcg tcaaacaccg cagcgaaggt gaacgcttcg cgcagtgggt catccgggcc gaggaggacg acctgcgatg agcggccgcc taggactact agtctgcagt ttttcctt</P><P><EM>Mycobacterium smegmatis</EM>This reference also contains the cloning of sirA from T. smegmatis.  We used the primers below taken from Pinto et al, blasted against the T. smegmatis genome and extracted the sirA sequence from genbank as above.
Primers:	smegsirA F      TACAGCTGATGCTCGAAGACGAGTACTTCAT
		smegsirA R      CCCAAGCTTTCACGTTGCCTACCTCAAATCCGCTTCGTC</P><P style="font-family: courier">
tacagctgat gctcgaagac gagtacttca tgctgcgcgt gcgctgcgat 
ggtggcgcgc tgaccactgc agcgctgcgc acgctcggcg gcatctcgac
 cgagttcgcg cgcgacaccg ccgacatctc cgaccgcgag aacgtccagt
 accactggat ccaggtcgag aacatgcccg agatctggaa gcgcctcgac
 gccgtcggcc tgcagaccac cgaggcgtgc ggcgactgcc cgcgtgtggt
 cctcggctcg ccgctggccg gtgagtccct cgacgaggtg atcgacggga
 cccccgcgat cgacgagatc gtgcgccgct acatcggcaa gcccgagtac
 tcgaacctgc cgcgcaagtt caagaccgcg atctcggggc ttcaggacgt
 ggtccacgag gtcaacgacg tcgcgttcat cggcgtcaac caccccgagc
 acggcccggg cttcgacctg tgggtcggtg gcggcctgtc gaccaacccg
 atgctggccc agcgcgtcgg ggtgtgggtg ccgctcgacg aggtgcccga
 cgtctgggag ggcgtcgtca gcatcttccg cgactacggt taccggcgtc
 tgcggtcgaa ggcgcggctg aagttcctga tcaaggactg gggcgtcgaa
 aagttcaggg aagtgctgga aaccgagtac ctcaagcgcc ccctgatcga
 cggcccggca cccgaaccgg tgacccgccc catcgaccac gtcggtgtgc
 agaagctcaa gaacggcctc aacgccgtgg gcgtcgcccc gatcgcgggt
 cgcgtctcgg gcacgatcct gaccaaggtg gccgatctcg ccgaggccgc
 cgggtccgac cggatccgct tcacgccgta ccagaagctg atcatcctcg
 acgtgcccga cgacaagatc gacgaactgc gcgctggcct cgacgcgctc
 ggactgccgt cgacgccgtc gcactggcgc cgcaacctca tggcgtgcac
 gggtatcgag ttctgcaagc tgagcttcgc cgagacccgc aagcgtgccc 
aggtgctggt tcccgagctg gagaaacggc tcgacgacat caacgcccag
 ctcgacgtgc ccatcacggt caacatcaac ggctgcccca actcgtgcgc
 ccgtatccag gtcgccgaca tcgggttcaa gggccagatg gtcgacgatg
 gcaacggccc cgaggagggt ttccaggtgc atctgggcgg cagcctgggc
 ctggacagcg ggttcggccg caagctgcgc cagcacaagg tgctctcgtc
 cgagctcggc gactacatcg agcgcgtcgt gcgcaacttc gtgaaacaac
 gcgaggacgg cgagcgtttc gcccagtggg ccgtgcgggc cgacgaagcg
 gatttgaggt aggcaacgtg aaagcttggg</P><P>As above, the sequence was imported into geneious, the biobrick restriction sites were manually removed from the sequence and restriction sites were added to the 5’ and 3’ ends in the same fashion as TB sirA.  This resulted in the following sequence for synthesis:</P><P style="font-family: courier">
tacagaattc tagaccatgg gcctcgaaga cgagtacttc atgctgcgcg 
tgcgctgcga tggtggcgcg ctgaccacta cagcgctgcg cacgctcggc
 ggcatctcga ccgagttcgc gcgcgacacc gccgacatct ccgaccgcga 
gaacgtccag taccactgga tccaggtcga gaacatgccc gagatctgga 
agcgcctcga cgccgtcggc ctacagacca ccgaggcgtg cggcgactgc 
ccgcgtgtgg tcctcggctc gccgctggcc ggtgagtccc tcgacgaggt 
gatcgacggg acccccgcga tcgacgagat cgtgcgccgc tacatcggca 
agcccgagta ctcgaacctg ccgcgcaagt tcaagaccgc gatctcgggg 
cttcaggacg tggtccacga ggtcaacgac gtcgcgttca tcggcgtcaa 
ccaccccgag cacggcccgg gcttcgacct gtgggtcggt ggcggcctgt 
cgaccaaccc gatgctggcc cagcgcgtcg gggtgtgggt gccgctcgac 
gaggtgcccg acgtctggga gggcgtcgtc agcatcttcc gcgactacgg 
ttaccggcgt ctgcggtcga aggcgcggct gaagttcctg atcaaggact 
ggggcgtcga aaagttcagg gaagtgctgg aaaccgagta cctcaagcgc 
cccctgatcg acggcccggc acccgaaccg gtgacccgcc ccatcgacca 
cgtcggtgtg cagaagctca agaacggcct caacgccgtg ggcgtcgccc 
cgatcgcggg tcgcgtctcg ggcacgatcc tgaccaaggt ggccgatctc 
gccgaggccg ccgggtccga ccggatccgc ttcacgccgt accagaagct 
gatcatcctc gacgtgcccg acgacaagat cgacgaactg cgcgctggcc 
tcgacgcgct cggactgccg tcgacgccgt cgcactggcg ccgcaacctc 
atggcgtgca cgggtatcga gttctgcaag ctgagcttcg ccgagacccg 
caagcgtgcc caggtgctgg ttcccgagct ggagaaacgg ctcgacgaca 
tcaacgccca gctcgacgtg cccatcacgg tcaacatcaa cggctgcccc 
aactcgtgcg cccgtatcca ggtcgccgac atcgggttca agggccagat 
ggtcgacgat ggcaacggcc ccgaggaggg tttccaggtg catctgggcg 
gcagcctggg cctggacagc gggttcggcc gcaagctgcg ccagcacaag 
gtgctctcgt ccgagctcgg cgactacatc gagcgcgtcg tgcgcaactt 
cgtgaaacaa cgcgaggacg gcgagcgttt cgcccagtgg gccgtgcggg 
ccgacgaagc ggatttgagg taggcaacgt gaaagcctag gtactagtct 
gcagtggg
</P></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Tuesday 11<SUP>th</SUP> June</H3><P><B><EM>
Design of FdrA genes for synthesis
</EM></B></P><B>FdrA</B><P><EM>Mycobacterium tuberculosis</EM>Reference: McLean et al. 2003. Kinetic, spectroscopic and thermodynamic characterization of the Mycobacterium tuberculosis adrenodoxin reductase homologue FprA. doi:  10.1042/BJ20021692
Primers:	MtbfdrA F     	TATGACTCATATGCGTCCCTATTACATCGCCATCG
		MtbfdrA R	     ATCCGGGATCCTCAGCCGAGCCCAATCCGCAACAGC
The above primers were blasted to retrieve sequence of FprA from genebank resulting in the following sequence:</P><P style="font-family: courier">tatgactcat atgcgtccct attacatcgc catcgtgggc tccgggccgt 
cggcgttctt cgccgcggca tccttgctga aggccgccga cacgaccgag 
gacctcgaca tggccgtcga catgctggag atgttgccga ctccctgggg 
gctggtgcgc tccggggtcg cgccggatca ccccaagatc aagtcgatca 
gcaagcaatt cgaaaagacg gccgaggacc cccgcttccg cttcttcggc 
aatgtggtcg tcggcgaaca cgtccagccc ggcgagctct ccgagcgcta 
cgacgccgtg atctacgccg tcggcgcgca gtccgatcgc atgttgaaca 
tccccggtga ggacctgccg ggcagtatcg ccgccgtcga tttcgtcggc 
tggtacaacg cacatccaca cttcgagcag gtatcacccg atctgtcggg 
cgcccgggcc gtagttatcg gcaatggaaa cgtcgcgcta gacgtggcac 
ggattctgct caccgatccc gacgtgttgg cacgcaccga tatcgccgat 
cacgctttgg aatcgctacg cccacgcggt atccaggagg tggtgatcgt 
cgggcgccga ggtccgctgc aggccgcgtt caccacgttg gagttgcgcg 
agctggccga cctcgacggg gttgacgtgg tgatcgatcc ggcggagctg 
gacggcatta ccgacgagga cgcggccgcg gtgggcaagg tctgcaagca 
gaacatcaag gtgctgcgtg gctatgcgga ccgcgaaccc cgcccgggac 
accgccgcat ggtgttccgg ttcttgacct ctccgatcga gatcaagggc 
aagcgcaaag tggagcggat cgtgctgggc cgcaacgagc tggtctccga 
cggcagcggg cgagtggcgg ccaaggacac cggcgagcgc gaggagctgc 
cagctcagct ggtcgtgcgg tcggtcggct accgcggggt gcccacgccc 
gggctgccgt tcgacgacca gagcgggacc atccccaacg tcggcggccg 
aatcaacggc agccccaacg aatacgtcgt cgggtggatc aagcgcgggc 
cgaccggggt gatcgggacc aacaagaagg acgcccaaga caccgtcgac 
accttgatca agaatcttgg caacgccaag gagggcgccg agtgcaagag 
ctttccggaa gatcatgccg accaggtggc cgactggcta gcagcacgcc 
agccgaagct ggtcacgtcg gcccactggc aggtgatcga cgctttcgag 
cgggccgccg gcgagccgca cgggcgtccc cgggtcaagt tggccagcct 
ggccgagctg ttgcggattg ggctcggctg aggatcccgg at</P><P>As previously, the sequence was imported into geneious, the biobrick restriction sites were manually removed from the sequence and restriction sites were added to the 5’ and 3’ ends in the same fashion as TB sirA.  However, in the case of FdrA, the restriction sites added for cloning into the vector pCOLAduet-1 were NdeI and AvrII.  As a result, there was no need to add an additional glycine after the start codon.  The resulting sequence is as follows:</P><P style="font-family: courier">
tatgaattct agatcatatg cgtccctatt acatcgccat cgtgggctcc 
gggccgtcgg cgttcttcgc cgcggcatcc ttgctgaagg ccgccgacac 
gaccgaggac ctcgacatgg ccgtcgacat gctggagatg ttgccgactc 
cctgggggct ggtgcgctcc ggggtcgcgc cggatcaccc caagatcaag 
tcgatcagca agcaattcga aaagacggcc gaggaccccc gcttccgctt 
cttcggcaat gtggtcgtcg gcgaacacgt ccagcccggc gagctctccg 
agcgctacga cgccgtgatc tacgccgtcg gcgcgcagtc cgatcgcatg 
ttgaacatcc ccggtgagga cctgccgggc agtatcgccg ccgtcgattt 
cgtcggctgg tacaacgcac atccacactt cgagcaggta tcacccgatc 
tgtcgggcgc ccgggccgta gttatcggca atggaaacgt cgcgctagac 
gtggcacgga ttctgctcac cgatcccgac gtgttggcac gcaccgatat 
cgccgatcac gctttggaat cgctacgccc acgcggtatc caggaggtgg 
tgatcgtcgg gcgccgaggt ccgcttcagg ccgcgttcac cacgttggag 
ttgcgcgagc tggccgacct cgacggggtt gacgtggtga tcgatccggc 
ggagctggac ggcattaccg acgaggacgc ggccgcggtg ggcaaggtct 
gcaagcagaa catcaaggtg ctgcgtggct atgcggaccg cgaaccccgc 
ccgggacacc gccgcatggt gttccggttc ttgacctctc cgatcgagat 
caagggcaag cgcaaagtgg agcggatcgt gctgggccgc aacgagctgg 
tctccgacgg cagcgggcga gtggcggcca aggacaccgg cgagcgcgag 
gagctgccag ctcagctggt cgtgcggtcg gtcggctacc gcggggtgcc 
cacgcccggg ctgccgttcg acgaccagag cgggaccatc cccaacgtcg 
gcggccgaat caacggcagc cccaacgaat acgtcgtcgg gtggatcaag 
cgcgggccga ccggggtgat cgggaccaac aagaaggacg cccaagacac 
cgtcgacacc ttgatcaaga atcttggcaa cgccaaggag ggcgccgagt 
gcaagagctt tccggaagat catgccgacc aggtggccga ctggctagca 
gcacgccagc cgaagctggt cacgtcggcc cactggcagg tgatcgacgc 
tttcgagcgg gccgccggcg agccgcacgg gcgtccccgg gtcaagttgg 
ccagcctggc cgagctgttg cggattgggc tcggctgagg atccctagga 
ctagtctgca gttacg</P><P><EM>Mycobacterium smegmatis</EM>We didn't find a reference of M. Smegmatis FprA being cloned into E. coli so we used the following sequenced data:</P><P style="font-family: courier">atgcgcccgt accacgtagc gatcgtcggc tcaggaccct ccggattctt 
tgctgccgca tcgctgctga agtttgccga ctctcaaccc gaccgcgatg
 tccgcgtgga catgctcgag atgctgccga ccccgtgggg tctggtgcgc 
tccggcgtcg ctcccgacca tccgaagatc aagtcgatca gcgcccagtt 
cgagaagacc gcggccgacc cccgcttccg gttcttcggc aacgtccggg 
tcggtgagca cgtccaaccg ggcgaactcg ccgaacgcta cgacgccgtc 
gtctacgcca ccggcgcaca gtccgaccgc gcgctgaaca tccccggcga 
ggagctgccg ggcagcgtcg cggccgtcga cttcgtgggt tggtacaacg 
cgcatccgca tttccgcgag atggcgcccg atctctccgg cgggcgcgcc 
gtggtggtcg gcaacggcaa cgtcgcactc gatgtcgcac gcatcctggt 
cagcgatccg aaggcgttgg ccaacaccga cattgccgac catgcgctgg 
acaagctgga cacgcgcggt gtggacgagg tcgtggtgct cggccggcgt 
ggtccgctgc aggcgacgtt caccacgctg gagttgcgtg aactcggcga 
catggagggc ctcggcgacg tcgacgtgat cctggatccg gccgatttcg 
ccgacatcac cgacgaggat ctcgaagccg cgggcaagac cgtcaagcag 
aacatcaagg tgctgcgcgg ttacgccgag cgggaaccgc gaggcgccaa 
gcggcgcatc gtgttccggt tcttcacctc accgatcgag ctgcgcggcg 
aggaccgcgt ggaatcgatc gtgttgggac gcaacgaact cgtgcgcggc 
gccgacggca ggatggtggc caaggacacc ggtgagcgcg aagaactgcc 
ggcacagctg gtggtgcgtg cggtcggata ccgcggcgtg ccgactccag 
gcctgccgtt cgacgagcgt tcgggcacca tcccgcacac cgacggccgc 
gtcgagggca gcgccaacga gtacgtggtg ggctggatca agcgcggacc 
gtccggcgtc atcggcagca acaagaagga ctcacaggac accgtcaaca 
ccctgctcga cgacctcgcc gcggcccggc tgccccagcg cgggcccgac 
cactccgaga agctcgcggc gtggttgctg gaacgtcagc ccaaggtggt 
cacgggcgag cactggaagc tgatcgacga ctacgagcgc gccgcgggcg 
aaccgaccgg ccgaccccgg gtgaagctgg cgagcgtggc tgaactgctg 
cgcgtgggcc acggctga</P><P>As above, the sequence was imported into geneious, the biobrick restriction sites were manually removed from the sequence and restriction sites were added to the 5’ and 3’ ends in the same fashion as TB sirA.  Again, the restriction sites added for cloning into the vector pCOLAduet-1 were NdeI and AvrII.  As a result, there was no need to add an additional glycine after the start codon.  The resulting sequence is as follows:</P><P style="font-family: courier">tatgaattct agatcatatg cgcccgtacc acgtagcgat cgtcggctca 
ggaccctccg gattctttgc tgccgcatcg ctgctgaagt ttgccgactc 
tcaacccgac cgcgatgtcc gcgtggacat gctcgagatg ctgccgaccc 
cgtggggtct ggtgcgctcc ggcgtcgctc ccgaccatcc gaagatcaag 
tcgatcagcg cccagttcga gaagaccgcg gccgaccccc gcttccggtt 
cttcggcaac gtccgggtcg gtgagcacgt ccaaccgggc gaactcgccg 
aacgctacga cgccgtcgtc tacgccaccg gcgcacagtc cgaccgcgcg 
ctgaacatcc ccggcgagga gctgccgggc agcgtcgcgg ccgtcgactt 
cgtgggttgg tacaacgcgc atccgcattt ccgcgagatg gcgcccgatc 
tctccggcgg gcgcgccgtg gtggtcggca acggcaacgt cgcactcgat 
gtcgcacgca tcctggtcag cgatccgaag gcgttggcca acaccgacat 
tgccgaccat gcgctggaca agctggacac gcgcggtgtg gacgaggtcg 
tggtgctcgg ccggcgtggt ccgcttcagg cgacgttcac cacgctggag 
ttgcgtgaac tcggcgacat ggagggcctc ggcgacgtcg acgtgatcct 
ggatccggcc gatttcgccg acatcaccga cgaggatctc gaagccgcgg 
gcaagaccgt caagcagaac atcaaggtgc tgcgcggtta cgccgagcgg 
gaaccgcgag gcgccaagcg gcgcatcgtg ttccggttct tcacctcacc 
gatcgagctg cgcggcgagg accgcgtgga atcgatcgtg ttgggacgca 
acgaactcgt gcgcggcgcc gacggcagga tggtggccaa ggacaccggt 
gagcgcgaag aactgccggc acagctggtg gtgcgtgcgg tcggataccg 
cggcgtgccg actccaggcc tgccgttcga cgagcgttcg ggcaccatcc 
cgcacaccga cggccgcgtc gagggcagcg ccaacgagta cgtggtgggc 
tggatcaagc gcggaccgtc cggcgtcatc ggcagcaaca agaaggactc 
acaggacacc gtcaacaccc tgctcgacga cctcgccgcg gcccggctgc 
cccagcgcgg gcccgaccac tccgagaagc tcgcggcgtg gttgctggaa 
cgtcagccca aggtggtcac gggcgagcac tggaagctga tcgacgacta 
cgagcgcgcc gcgggcgaac cgaccggccg accccgggtg aagctggcga 
gcgtggctga actgctgcgc gtgggccacg gctgacctag gactagtctg 
cagttacg
</P></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Wednesday 12<SUP>th</SUP> June</H3><P><B><EM>
Design of FdxC and FdxA genes for synthesis
</EM></B></P><B>FdxC</B><P><EM>Mycobacterium tuberculosis</EM>Reference: Cole, S.T. et al. 1998. Deciphering the biology of Mycobacterium tuber-
culosis from the complete genome sequence. doi:10.1038/31159.

We couldn’t find a reference of FdxC being cloned into E. coli and so we took the sequence data from genbank which is as follows:</P><P style="font-family: courier">
gtgacgtaca cgatcgccga accctgtgtc gacatcaagg acaaggcatg 
cattgaggag tgcccggtcg attgcatcta cgagggcgcc cggatgctgt 
atatccaccc cgacgaatgc gtcgactgtg gggcttgcga gccggtctgc 
cccgttgaag ctatcttcta cgaagacgat gtgcccgaac agtggagcca 
ttacacccag atcaacgccg atttcttcgc cgagctggga tcgccgggcg 
gtgcggccaa ggttggcatg accgagaacg acccgcaagc ggtcaaggat 
ctggcgccgc agagcgagga cgcctga</P><P>The Gtg was replaced with the NcoI cut site and a glycine codon (ccatgggc) and a HindIII site was added to the end of the sequence.  The biobrick restriction sites were then added to the ends of the sequence resulting in the following sequence:</P><P style="font-family: courier">
cttagaattc tagaccatgg gcacgtacac gatcgccgaa ccctgtgtcg 
acatcaagga caaggcatgc attgaggagt gcccggtcga ttgcatctac 
gagggcgccc ggatgctgta tatccacccc gacgaatgcg tcgactgtgg 
ggcttgcgag ccggtctgcc ccgttgaagc tatcttctac gaagacgatg 
tgcccgaaca gtggagccat tacacccaga tcaacgccga tttcttcgcc 
gagctgggat cgccgggcgg tgcggccaag gttggcatga ccgagaacga 
cccgcaagcg gtcaaggatc tggcgccgca gagcgaggac gcctgaagct 
tgactagtct gcagaaatg</P><B>FdxA</B><P><EM>Mycobacterium smegmatis</EM>Reference: Ricagno S. et al. 2008. The crystal structure of FdxA, a 7Fe ferredoxin from Mycobacterium smegmatis. http://dx.doi.org/10.1016/j.bbrc.2007.06.013
http://www.metacyc.org/MSME246196/NEW-IMAGE?type=GENE-IN-MAP&amp;object=GJ4Y-1124
Sequence:</P><P style="font-family: courier">
atgacatacg tgatcggccg gccgtgcgtg gacgtcaaag accgcgcatg 
cgtggatgag tgcccggtcg actgcatcta cgagggcgcg cggatgctct 
acatccaccc cgacgaatgt gtcgactgtg gcgcgtgtga acccgtgtgc 
ccggtggagg cgatctacta cgaggacgac ctgcccgagg atctgcagcc 
gtaccaggag gagaacgcga agttcttcac cgatgtcctg cccgggcgtg 
cccaaccgct cggatcaccg ggcggtgccg cgaaactcgg tgtcgtggac 
gccgatacgc ccatggttgc ggagctaccg ccgcaggggg attga</P><P>The NcoI cut site and a glycine codon (ccatgggc) replaced the atg in green and a HindIII site was added to the end of the sequence.  The biobrick restriction sites were then added to the ends of the sequence resulting in the following sequence:</P><P style="font-family: courier">
cttagaattc tagaccatgg gcacatacgt gatcggccgg ccgtgcgtgg 
acgtcaaaga ccgcgcatgc gtggatgagt gcccggtcga ctgcatctac 
gagggcgcgc ggatgctcta catccacccc gacgaatgtg tcgactgtgg 
cgcgtgtgaa cccgtgtgcc cggtggaggc gatctactac gaggacgacc 
tgcccgagga tcttcagccg taccaggagg agaacgcgaa gttcttcacc 
gatgtcctgc ccgggcgtgc ccaaccgctc ggatcaccgg gcggtgccgc 
gaaactcggt gtcgtggacg ccgatacgcc catggttgcg gagctaccgc 
cgcaggggga ttgagcctag gtactagtct gcagtggg</P></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Tuesday 18<SUP>th</SUP> June</H3><P><B><EM>
In Silico Cloning
</EM></B></P>
We used geneious to perform in silico cloning on our constructed gene sequences to ensure that they were properly made.  SirA and pACYCDuet-1 were both cut with NcoI and AvrII and then the large fragments were extracted and ligated together to form vector pD001.  FdxA and pETDuet-1 were both cut with NcoI and HindIII and the large fragments were extracted and ligated together to form vector pD002.  FprA and pD002 were both cut with NdeI and AvrII and the large fragments were extracted and ligated together to form vector pD003.  Please see Geneious files on Google Drive (or send request for files) for exact sequences.
</DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>24<SUP>th</SUP> June</H3><P><B><EM>
Codon Optimization
</EM></B></P>
The ORF of M. Smegmatis SirA, FprA, and FdxA were codon optimized for E. coli K-12 using JCat (www.jcat.de) with options to avoid using rho-independent terminators, prokaryotic RBS, and restriction sites used in vector construction (AvrII, HindIII, NdeI, NcoI) and Biobrick restriction sites (EcoRI, PstI, SpeI, XbaI).  The optimization charts are shown below:
</DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Monday 1<SUP>st</SUP> July</H3><P><B><EM>
Media was made, Strains sD001-sD004 were received, inoculated, and put into stock and catalog.
</EM></B></P><P>
4 strains were received from Jake: 
<UL><LI>E. coli: BL21 (DE3) ko20 ΔcysI, Δfpr, ΔydbK
</LI><LI>E. coli: NEBTurbo zmSIR Chloramphenicol
</LI><LI>E. coli: NEBTurbo zmFNR Spectinomycin
</LI><LI>E. coli: NEBTurbo soFD, zmSIR Chloramphenicol
</LI></UL></P><H5>Media and Glycerol stock were prepared</H5><H6>Media Preparation</H6>
3 500ml bottles of LB broth and LB agar were prepared by standard methods.  12.5g/500ml powder/water for broth and 20g/500ml powder/water for agar.  Bottles were autoclaved.  Additionally 1 flask of 500ml of LB broth was made in the same fashion.
<H6>Glycerol Stocks</H6>
Single colonies from agar plates were picked and used to innoculate 5ml LB broth overnight.  750ml of overnight culture was added to 250ml of 60% glycerol in a cryotube.  Two sets of Glycerol stocks were used for each of sD001, sD002, sD003, sD004; one set was frozen at -20ºC and the other set was frozen at -80ºC.
</DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" class="tbnotelogo DSlogo"> ASDF </A><H3>Tuesday<SUP>2nd</SUP> July</H3><P><B><EM>
MiniPrep: Plasmids pD004, pD005 and pD006 were extracted from NEBturbo vectors
</EM></B></P>
Plasmid Extraction
Plasmids pD004 (pCDF.ew12 zmFNR Spectinomycin), pD005 (pACYC.ew13 soFD, zmSIR Chloramphenicol), pD006 (pACYC.ew17 zmSIR Chloramphenicol) were extracted from NEBTurbo cells using a Thermo Scientific GeneJet Plasmid mini prep kit as described in protocol ().  Lacking Resuspention solution we used some from a different mini prep upstairs.  Plasmids were eluted in 100ul of nanopure water and frozen at -20ºC.

More notes here tra la lalal
</DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Wednesday 3<SUP>rd</SUP> July</H3><P><B><EM>
Chemically Competent Cells : BL21 (DE3) dCysI dFpr dydbk, NEBturbo stocks 
</EM></B></P><P>
Chemically Competent BL21 (DE3) Cells were made
We grew 5ml of sD001 cells overnight from a single colony.  This 5ml was used to innoculate 500ml of LB broth.   Broth was incubated for 1.5h and optical density was read at 0.62 OD600.  Cells were alloquoted into 10 falcon tubes (50ml each) and centrifuged at 4000xg for 20 minutes at 4C.  Supernatant was removed and cells were resuspended in 200ml of Buffer 1 (4x 50ml) and left on ice for 20 minutes.  Cells were centrifuged again for 20 minutes at 4000xg.  Supernatant was removed and cells were resuspended in 12ml Buffer 2.  Cells were alloquoted at 500ul into microcentifuge tubes and frozen at -80C.

Buffer 1:  50mM CaCl2
Buffer 2: 	<UL><LI>0.53ml 2M CaCl2</LI><LI>2.8ml 60% Glycerol</LI><LI>8.67ml sterile H2O
</LI></UL>This is the correct Chemical Competent Cells protocol (Similar to Protocol 2) , that we should have used instead of the one above:

<OL><LI> Take a single colony of E. coli cells and inoculate 5ml of LB broth.   Incubate over night at 37C and 200rpm and innoculate about 500ml to 1L sterile LB broth.  DO NOT ADD antibiotics since these cells do not have a plasmid in them.   Work as sterile as possible.

</LI><LI> Inoculate about 300-400ml sterile LB broth with 500ul of preculture.

</LI><LI> Grow the cells on a shaker at 37C until they reach an OD @ 600nm of 0.3 to 0.4 (1cm pathlength of the cuvette).

</LI><LI> Centrifuge at 3000xg for 10 minutes at 4C.  Ice down 100mM CaCl2 and 100mM MgCl2 solutions at this point.

</LI><LI> Gently resuspend the bacteria pellet on ice in 50ml of ice cold 100mM MgCl2, taking 3-5 minutes for this procedure.   Centrifuge the cell suspension at 3000xg for 10 minutes at 4C.

</LI><LI> Resuspend the bacteria pellet on ice 50ml of ice cold 100mM CaCl2, incubate the cells for 10 minutes on ice.

</LI><LI> Centrifuge the cell suspension at 3000xg for 10 minutes at 4C and resuspend the cell pellet in 4ml of ice cold, sterile 100mM CaCl2 in 15%(w/v) glycerol.  Dispense in 50uL aliquots and freeze cells at -80C.
</LI></OL>Chemically competent cells (NEB Turbo)
Prepared via iGEM chemically competent protocol with following changes.  500ul of preculture used to inoculate 500ml LB broth.  Grown at 37C for 3h hours to an OD600 of 0.536.  Cells centrifuged at 3200 rpm on old centrifuge for 10m at 4C.  Pellets resuspended in 80ml of CCMB80 buffer.  Incubated on ice for 20m.  Centrifuged for 10m at 3200 rpm for 10m at 4C.  Pellets resuspended in 5ml of CCMB80 buffer.  Aliquoted into Microcentrifuge tubes and frozen at -80C.

</P></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Thursday 4<SUP>th</SUP> July</H3><P><B><EM>
Heat Shock Transformation
</EM></B></P><B>Preparation of Agar Plates</B>

LB Agar was melted; 100ml was used to make 8 ampicillin plates, 100ml was used to make 8 spectinomycin plates and the remaining was used to make 20 chloramphenicol plates.  Antibiotic concentrations are: Ampicillin 50ug/ml, Spectinomycin 50ug/ml, Chloramphenicol 34ug/ml.
<B>Heat Shock Transformation</B>
 sD001: BL21 (DE3) with deletions, was transformed with the following plasmids and plasmid combinations:<UL><LI> sD001+pD004 ( FNR, Spectinomycin)
</LI><LI> sD001+pD005 ( SIR, Fdx, Chloramphenicol)
</LI><LI> sD001+pD006 (Sir, Chloramphenicol)
</LI></UL>
Transformation was done according to protocol 1 with the following specifications:
BL21 (DE3) chemically competent cells were thawed on Ice.  1ul of plasmid DNA was added to a tube of BL21 (DE3) cells.  Cells were incubated on ice for 30 minutes.  Cells were transfered to heating block at 42C for 45s.  Cells were returned to ice for 2 minutes.  1ml of LB Broth was added to the cells and they were incubated at 37C for 1h. Cells were then plated on agar with appropriate antibiotics (specified previously).
These transformations worked; single colonies were lifted from plates, entered to catalog and stocked in glycerol stock (protocol 3). 
Catalog names:
<UL><LI> sD006= sD001+pD004 ( FNR, Spectinomycin)
</LI><LI> sD007= sD001+pD005 ( SIR, Fdx, Chloramphenicol)
</LI><LI> sD008= sD001+pD006 (Sir, Chloramphenicol)
</LI></UL></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Friday 5<SUP>th</SUP> July</H3><P><B><EM>
Planning transformations for the next week
</EM></B></P><P style="margin-bottom: 0in; line-height: 114%"><FONT color="#222222"><SPAN style="text-decoration: none"><FONT face="Arial"><FONT size="2" style="font-size: 11pt"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: #ffffff">We
decided to partly reproduce Figure 3C from:
</SPAN></SPAN></SPAN></FONT></FONT></SPAN></FONT><A href="http://www.jbioleng.org/content/5/1/7"><FONT color="#005400"><SPAN style="text-decoration: none"><FONT face="Verdana"><FONT size="2" style="font-size: 11pt"><SPAN style="font-style: normal"><U><SPAN style="font-weight: normal"><SPAN style="background: #ffffff">http://www.jbioleng.org/content/5/1/7</SPAN></SPAN></U></SPAN></FONT></FONT></SPAN></FONT></A><FONT color="#222222"><SPAN style="text-decoration: none"><FONT face="Arial"><FONT size="2" style="font-size: 11pt"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: #ffffff">:</SPAN></SPAN></SPAN></FONT></FONT></SPAN></FONT></P><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 114%; text-decoration: none"><FONT color="#222222"><FONT face="Arial"><FONT size="2" style="font-size: 11pt"><SPAN style="background: #ffffff">So,
we would pursue to grow the following strains on a minimal media M9
plate, supplemented with all amino acids</SPAN></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 114%; text-decoration: none"><FONT color="#222222"><FONT face="Arial"><FONT size="2" style="font-size: 11pt"><SPAN style="background: #ffffff">aside
from the sulfur containing ones, and sulfite ans the only sulfur
source:</SPAN></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 114%; text-decoration: none"><FONT color="#222222"><FONT face="Arial"><FONT size="2" style="font-size: 11pt"><SPAN style="background: #ffffff">We
already thought we have all of the strains we need, however, an
overnight culture was launched of a few single colonies from this
double plasmid strain (sD001+pD004+pD005, which supposedly had all 3
genes SIR FNR and Fdx) and it didn't grow. </SPAN></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-weight: normal; line-height: 114%"><FONT size="2" style="font-size: 11pt"><FONT color="#222222"><SPAN style="text-decoration: none"><FONT face="Arial"><SPAN style="font-style: normal"><SPAN style="background: #ffffff">This
most likely occurred because the strains didn't have </SPAN></SPAN></FONT></SPAN></FONT><FONT color="#222222"><SPAN style="text-decoration: none"><FONT face="Arial"><I><SPAN style="background: #ffffff">Spec
</SPAN></I></FONT></SPAN></FONT><FONT color="#222222"><SPAN style="text-decoration: none"><FONT face="Arial"><SPAN style="font-style: normal"><SPAN style="background: #ffffff">antibiotics
resistance, as the other spec resistant strain containing</SPAN></SPAN></FONT></SPAN></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 114%; text-decoration: none"><FONT color="#222222"><FONT face="Arial"><FONT size="2" style="font-size: 11pt"><SPAN style="background: #ffffff">the
FNR gene (pD004), was found not to be resistant as well. </SPAN></FONT></FONT></FONT></P><P style="margin-top: 0.08in; margin-bottom: 0.08in"><FONT size="3"><I>Illustration
	1: Strains will be grown on a minimal media M9 plate, supplemented
	with all amino acids aside from the sulfur containing ones; plates
	will have increased glucose, and sulfite as the only sulfur source.
	Growth is expected only for WT and the strain containing all three
	alternative genes of sulfur reduction.</I></FONT></P><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 114%; text-decoration: none"><FONT color="#222222"><FONT face="Arial"><FONT size="2" style="font-size: 11pt"><SPAN style="background: #ffffff">Therefore,
we would  re-make: </SPAN></FONT></FONT></FONT></P><UL><LI><P style="margin-right: 0.15in; margin-bottom: 0in; background: transparent; font-style: normal; font-weight: normal; line-height: 114%; text-decoration: none"><FONT color="#000000"><FONT face="Arial"><FONT size="2" style="font-size: 11pt"><FONT color="#222222"><B><SPAN style="background: #ffffff">sD001+pD004+pD005</SPAN></B></FONT><FONT color="#222222"><SPAN style="background: #ffffff">
	E.coli:</SPAN></FONT><FONT color="#ff9900"><SPAN style="background: #ffffff">ΔcysI,
	Δfpr, ΔydbK</SPAN></FONT><SPAN style="background: #ffffff">::
	</SPAN><FONT color="#0000ff"><B><SPAN style="background: #ffffff">soFD</SPAN></B></FONT><SPAN style="background: #ffffff">,
	</SPAN><FONT color="#ff0000"><B><SPAN style="background: #ffffff">zmSIR
	</SPAN></B></FONT><SPAN style="background: #ffffff">;Chloramphenicol
	,</SPAN><FONT color="#ff00ff"><B><SPAN style="background: #ffffff">zmFNR</SPAN></B></FONT><FONT color="#222222"><SPAN style="background: #ffffff">;
	Spectinomycin- </SPAN></FONT><FONT color="#222222"><U><SPAN style="background: #ffffff">with
	all three genes.</SPAN></U></FONT></FONT></FONT></FONT></P></LI><LI><P style="margin-right: 0.15in; margin-bottom: 0in; background: transparent; font-style: normal; font-weight: normal; line-height: 114%; text-decoration: none"><FONT color="#000000"><FONT face="Arial"><FONT size="2" style="font-size: 11pt"><FONT color="#222222"><U><B><SPAN style="background: #ffffff">sD001+pD004:
	</SPAN></B></U></FONT><FONT color="#222222"><SPAN style="background: #ffffff">E.coli:</SPAN></FONT><FONT color="#ff9900"><SPAN style="background: #ffffff">ΔcysI,
	Δfpr, ΔydbK</SPAN></FONT><SPAN style="background: #ffffff">::
	</SPAN><FONT color="#ff00ff"><B><SPAN style="background: #ffffff">zmFNR</SPAN></B></FONT><FONT color="#222222"><SPAN style="background: #ffffff">;
	Spectinomycin</SPAN></FONT></FONT></FONT></FONT></P></LI></UL><P style="margin-right: 0.15in; margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 114%; text-decoration: none"><FONT color="#222222"><FONT face="Arial"><FONT size="2" style="font-size: 11pt"><SPAN style="background: #ffffff">and
make</SPAN></FONT></FONT></FONT></P><UL><LI><P style="margin-right: 0.15in; margin-bottom: 0in; background: transparent; line-height: 114%"><FONT size="2" style="font-size: 11pt"><FONT color="#222222"><SPAN style="text-decoration: none"><FONT face="Arial"><SPAN style="font-style: normal"><B><SPAN style="background: #ffffff">sD001+pD004+pD006</SPAN></B></SPAN></FONT></SPAN></FONT><FONT color="#222222"><SPAN style="text-decoration: none"><FONT face="Arial"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: #ffffff">E.coli:</SPAN></SPAN></SPAN></FONT></SPAN></FONT><FONT color="#ff9900"><SPAN style="text-decoration: none"><FONT face="Arial"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: #ffffff">ΔcysI,
	Δfpr, ΔydbK</SPAN></SPAN></SPAN></FONT></SPAN></FONT><FONT color="#000000"><SPAN style="text-decoration: none"><FONT face="Arial"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: #ffffff">::</SPAN></SPAN></SPAN></FONT></SPAN></FONT><FONT color="#ff00ff"><SPAN style="text-decoration: none"><FONT face="Arial"><SPAN style="font-style: normal"><B><SPAN style="background: #ffffff">zmFNR</SPAN></B></SPAN></FONT></SPAN></FONT><FONT color="#000000"><SPAN style="text-decoration: none"><FONT face="Arial"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: #ffffff">;
	Spectinomycin, </SPAN></SPAN></SPAN></FONT></SPAN></FONT><FONT color="#ff0000"><SPAN style="text-decoration: none"><FONT face="Arial"><SPAN style="font-style: normal"><B><SPAN style="background: #ffffff">zmSIR</SPAN></B></SPAN></FONT></SPAN></FONT><FONT color="#000000"><SPAN style="text-decoration: none"><FONT face="Arial"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: #ffffff">;
	Chloramphenicol</SPAN></SPAN></SPAN></FONT></SPAN></FONT></FONT></P></LI></UL><P style="margin-right: 0.15in; margin-bottom: 0in; background: transparent; font-style: normal; line-height: 114%; text-decoration: none"><FONT color="#000000"><FONT face="Arial"><FONT size="2" style="font-size: 11pt"><SPAN style="background: #ffffff">These
transformations will be done by using two strains of competent cells
prepared using Protocol 2 :</SPAN></FONT></FONT></FONT></P><UL><LI><P style="margin-bottom: 0in; background: transparent; font-style: normal; line-height: 114%; text-decoration: none"><FONT color="#000000"><FONT face="Arial"><FONT size="2" style="font-size: 11pt"><SPAN style="background: #ffffff">sD001
	competent cells were transformed with:</SPAN></FONT></FONT></FONT></P></LI><LI><P style="margin-bottom: 0in; background: transparent; font-style: normal; line-height: 114%; text-decoration: none"><FONT color="#000000"><FONT face="Arial"><FONT size="2" style="font-size: 11pt"><SPAN style="background: #ffffff">pD004</SPAN></FONT></FONT></FONT></P></LI><LI><P style="margin-bottom: 0in; background: transparent; font-style: normal; line-height: 114%; text-decoration: none"><FONT color="#000000"><FONT face="Arial"><FONT size="2" style="font-size: 11pt"><SPAN style="background: #ffffff">Double
		transformation: pD005 +pD004</SPAN></FONT></FONT></FONT></P></LI></UL><LI><P style="margin-bottom: 0in; background: transparent; font-style: normal; line-height: 114%; text-decoration: none"><FONT color="#000000"><FONT face="Arial"><FONT size="2" style="font-size: 11pt"><SPAN style="background: #ffffff">sD007
	(sD001+pD005) competent cells were transformed with:</SPAN></FONT></FONT></FONT></P></LI><LI><P style="margin-bottom: 0in; background: transparent; font-style: normal; line-height: 114%; text-decoration: none"><FONT color="#000000"><FONT face="Arial"><FONT size="2" style="font-size: 11pt"><SPAN style="background: #ffffff">pD004</SPAN></FONT></FONT></FONT></P><P style="margin-bottom: 0in; background: transparent; font-style: normal; line-height: 114%; text-decoration: none"><FONT color="#000000"><FONT face="Arial"><FONT size="2" style="font-size: 11pt"><SPAN style="background: #ffffff">These
transformations will be done by heat shock using protocol 1.</SPAN></FONT></FONT></FONT></P></LI></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Tuesday 16<SUP>th</SUP> July</H3><P><B><EM>
Transformation efficiency measurements
</EM></B></P>
Transformation of Puc18 to sD001 (BL21 ED3) competent cells.
We have made a transformation of pUC18 into our sD001 BL21(DE3) strain  of competent cells. Transformation was done by heat shock protocol 1 with the following specifications:
200 ul Chemically Competent Cells were thawed on Ice. 0.5 ul DNA was added and incubate on ice for 30 minutes. HEAT SHOCK cells were Incubated at 42C for 45 seconds then incubated on ice for 2 minutes. 200 ul of LB broth was added and cells were incubated at 37C for 1 hour. 10 ul of cells were plated on agar supplemented with Ampicillin. During the night 18 colonies grew. 

</DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Wednesday 17<SUP>th</SUP> July</H3><P><B><EM>
Making sD001+pD004 (Spect), sD001+pD005 (Chl31) competent
</EM></B></P>
Making sD001+pD004 (Spect), sD001+pD005 (Chl31) competent, by using protocol 2 with these specifications:
The night before, cells were inoculated in a 5 ml culture and grown overnight with selection.
0.05 ml of the cell culture was diluted ~ 1:200 into 10 ml of selective media (Chl31,Spect) and grew to an OD600 of 0.6 – 0.7 (sD001+pD004 didn’t grow and protocol was continued for sD001+pD005  - see comment). Cells were Spun down at 4 ºC, 4000 rpm, 15 minutes. Then were resuspended in 15 ml, ice-cold 100 mM CaCl2 and left for 3 hours.  Next, cells were spun down at 4 ºC, 4000 rpm, for 15 minutes, Resuspended in 4 ml, ice-cold 100 mM CaCl2 + 15% glycerol  and aliquoted into pre-chilled Eppendorf tubes. Tubes were immediately  stored at -80ºC.
10 ml CaCl2 100nM + glycerol 15% was made from:<UL><LI>3.33 ml Glycerol 60% solution.
</LI><LI>6.17 ml Sterile water
</LI><LI>0.5 ml CaCl2 2 M solution.
</LI></UL><B>Comment:</B> sD001+pD004 did not grow overnight (and possibly always) in Streptavidin instead of Spectinomycin so they most likely lost their resistance and we will make them again tomorrow by tranforming the BL21(DE3) cells with pD004.


</DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Thursday 18<SUP>th</SUP> July</H3><P><B><EM>
 Heat shock Transformation
</EM></B></P><B> Transformations made: </B><UL><LI>sD001+pD004+pD005 E.coli:ΔcysI, Δfpr, ΔydbK:: soFD, zmSIR ;Chloramphenicol ,zmFNR; Spectinomycin- with all three genes
</LI><LI>sD001+pD004 E.coli:ΔcysI, Δfpr, ΔydbK:: ,zmFNR; Spectinomycin
</LI><LI>sD001+pD004+pD006 E.coli: ΔcysI, Δfpr, ΔydbK::zmFNR; Spectinomycin, zmSIR; Chloramphenicol
</LI></UL>
Transformations were made with protocol 1 with the following specifications:
 BL21 (DE3) chemically competent cells were thawed on Ice.  2ul of plasmid DNA was added to 20ul  of BL21 (DE3) cells.  Cells were incubated on ice for 30 minutes.  Cells were transfered to heating block at 42C for 45s.  Cells were returned to ice for 2 minutes.   200ul of LB Broth was added to the cells and they were incubated at 37C for 1h.  After incubations cells of each transformation were plated on two plates: one with 5 ul , and the other with 100 ul of the post recovery cells culture. </DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Friday 19<SUP>th</SUP> July</H3><P><B><EM>
TRANSFORMATIONS WORKED!!! cells were stocked and entered in catalog.
</EM></B></P><UL><LI><FONT color="#EE0F29">sD012/sD013</FONT>=sD001+pD004+pD005 E.coli:ΔcysI, Δfpr, ΔydbK:: soFD, zmSIR ;Chloramphenicol ,zmFNR; Spectinomycin- with all three genes
</LI><LI><FONT color="#EE0F29">sD016/sD017</FONT>=sD001+pD004 E.coli:ΔcysI, Δfpr, ΔydbK:: ,zmFNR; Spectinomycin
</LI><LI><FONT color="#EE0F29">sD014/sD015</FONT>=sD001+pD004+pD006 E.coli: ΔcysI, Δfpr, ΔydbK::zmFNR; Spectinomycin, zmSIR; Chloramphenicol
</LI></UL><B>Glycerol Stocks were made of the above mentioned strains</B> 2 Single colonies from each agar plate of above mentioned strains were picked and used to inoculate 5ml LB broth overnight.  750ml of overnight culture was added to 250ml of 60% glycerol in a cryotube.  Two sets of Glycerol stocks were used for each of sD001+pD004, sD001+sD004+sD005, sD001+sD004+sD006; one set was frozen at -20ºC and the other set was frozen at -80ºC.

</DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Wednesday 24<SUP>th</SUP> July</H3><P style="margin-bottom: 0in; font-style: normal; line-height: 100%; text-decoration: none"><FONT color="#000000"><FONT face="Arial"><FONT size="2" style="font-size: 11pt"><B><SPAN style="background: transparent">24/7/13</SPAN></B></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-weight: normal; line-height: 100%"><FONT color="#000000"><SPAN style="text-decoration: none"><FONT face="Arial"><FONT size="2" style="font-size: 11pt"><SPAN style="font-style: normal"><SPAN style="background: transparent">Made
500 ml of </SPAN></SPAN></FONT></FONT></SPAN></FONT><FONT color="#000000"><SPAN style="text-decoration: none"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="font-style: normal"><SPAN style="background: transparent">M9
Minimal Plates with Sulfur Dropout Powder accouding to the following
recipe: </SPAN></SPAN></FONT></FONT></SPAN></FONT></P><P style="margin-bottom: 0in; font-style: normal; line-height: 100%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><B><SPAN style="background: transparent">Recipe:
M9 Minimal Plates with Sulfur Dropout Powder</SPAN></B></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 100%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">These
stock solutions can be pre-sterilized and stored indefinitely:</SPAN></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; line-height: 100%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><B><SPAN style="background: transparent">5x
M9 Salts (from scratch)</SPAN></B></FONT></FONT></FONT></P><TABLE cellpadding="2" cellspacing="2"><TBODY><TR><TD><P align="CENTER" style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><B><SPAN style="background: transparent">Reagent</SPAN></B></FONT></FONT></FONT></P></TD><TD><P align="CENTER" style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><B><SPAN style="background: transparent">Quantity</SPAN></B></FONT></FONT></FONT></P></TD></TR><TR><TD><P align="CENTER" style="border: 1px solid #000000; padding: 0.02in 0.07in"><FONT color="#000000"><SPAN style="text-decoration: none"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: #ffffff">Na</SPAN></SPAN></SPAN></FONT></FONT></SPAN></FONT><FONT color="#000000"><SPAN style="text-decoration: none"><FONT face="Helvetica Neue"><FONT size="1"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: #ffffff">2</SPAN></SPAN></SPAN></FONT></FONT></SPAN></FONT><FONT color="#000000"><SPAN style="text-decoration: none"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: #ffffff">HPO</SPAN></SPAN></SPAN></FONT></FONT></SPAN></FONT><FONT color="#000000"><SPAN style="text-decoration: none"><FONT face="Helvetica Neue"><FONT size="1"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: #ffffff">4</SPAN></SPAN></SPAN></FONT></FONT></SPAN></FONT></P></TD><TD><P align="CENTER" style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: #ffffff">17
			g</SPAN></FONT></FONT></FONT></P></TD></TR><TR><TD><P align="CENTER" style="border: 1px solid #000000; padding: 0.02in 0.07in"><FONT color="#000000"><SPAN style="text-decoration: none"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: #ffffff">KH</SPAN></SPAN></SPAN></FONT></FONT></SPAN></FONT><FONT color="#000000"><SPAN style="text-decoration: none"><FONT face="Helvetica Neue"><FONT size="1"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: #ffffff">2</SPAN></SPAN></SPAN></FONT></FONT></SPAN></FONT><FONT color="#000000"><SPAN style="text-decoration: none"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: #ffffff">PO</SPAN></SPAN></SPAN></FONT></FONT></SPAN></FONT><FONT color="#000000"><SPAN style="text-decoration: none"><FONT face="Helvetica Neue"><FONT size="1"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: #ffffff">4</SPAN></SPAN></SPAN></FONT></FONT></SPAN></FONT></P></TD><TD><P align="CENTER" style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: #ffffff">7.5
			g</SPAN></FONT></FONT></FONT></P></TD></TR><TR><TD><P align="CENTER" style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: #ffffff">NaCl</SPAN></FONT></FONT></FONT></P></TD><TD><P align="CENTER" style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: #ffffff">1.25
			g</SPAN></FONT></FONT></FONT></P></TD></TR><TR><TD><P align="CENTER" style="border: 1px solid #000000; padding: 0.02in 0.07in"><FONT color="#000000"><SPAN style="text-decoration: none"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: #ffffff">(NH</SPAN></SPAN></SPAN></FONT></FONT></SPAN></FONT><FONT color="#000000"><SPAN style="text-decoration: none"><FONT face="Helvetica Neue"><FONT size="1"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: #ffffff">4</SPAN></SPAN></SPAN></FONT></FONT></SPAN></FONT><FONT color="#000000"><SPAN style="text-decoration: none"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: #ffffff">)</SPAN></SPAN></SPAN></FONT></FONT></SPAN></FONT><FONT color="#000000"><SPAN style="text-decoration: none"><FONT face="Helvetica Neue"><FONT size="1"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: #ffffff">2</SPAN></SPAN></SPAN></FONT></FONT></SPAN></FONT><FONT color="#000000"><SPAN style="text-decoration: none"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: #ffffff">SO</SPAN></SPAN></SPAN></FONT></FONT></SPAN></FONT><FONT color="#000000"><SPAN style="text-decoration: none"><FONT face="Helvetica Neue"><FONT size="1"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: #ffffff">4</SPAN></SPAN></SPAN></FONT></FONT></SPAN></FONT></P></TD><TD><P align="CENTER" style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: #ffffff">3.0
			g</SPAN></FONT></FONT></FONT></P></TD></TR></TBODY></TABLE><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 100%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Bring
to a total volume of 500 ml.</SPAN></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 100%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Adjust
pH to 7.4 with NaOH.</SPAN></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 100%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Sterilize
by autoclaving or filtration</SPAN></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; line-height: 100%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><B><SPAN style="background: transparent">or</SPAN></B></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; line-height: 100%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><B><SPAN style="background: transparent">5x
M9 Salts (from Difco Powder)</SPAN></B></FONT></FONT></FONT></P><TABLE cellpadding="2" cellspacing="2"><TBODY><TR><TD><P align="CENTER" style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><B><SPAN style="background: transparent">Reagent</SPAN></B></FONT></FONT></FONT></P></TD><TD><P align="CENTER" style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><B><SPAN style="background: transparent">Quantity</SPAN></B></FONT></FONT></FONT></P></TD></TR><TR><TD><P align="CENTER" style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: #ffffff">M9
			Salts</SPAN></FONT></FONT></FONT></P></TD><TD><P align="CENTER" style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: #ffffff">28.2
			g</SPAN></FONT></FONT></FONT></P></TD></TR></TBODY></TABLE><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 100%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Bring
to a total volume of 500 mL.</SPAN></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 100%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Sterilize
by autoclaving or filtration</SPAN></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; line-height: 100%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><B><SPAN style="background: transparent">2x
Yeast Plate Agar</SPAN></B></FONT></FONT></FONT></P><TABLE cellpadding="2" cellspacing="2"><TBODY><TR><TD><P align="CENTER" style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><B><SPAN style="background: transparent">Reagent</SPAN></B></FONT></FONT></FONT></P></TD><TD><P align="CENTER" style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><B><SPAN style="background: transparent">Quantity</SPAN></B></FONT></FONT></FONT></P></TD></TR><TR><TD><P align="CENTER" style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: #ffffff">Agar</SPAN></FONT></FONT></FONT></P></TD><TD><P align="CENTER" style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: #ffffff">30
			g</SPAN></FONT></FONT></FONT></P></TD></TR></TBODY></TABLE><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 100%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Bring
to a total volume of 500 ml.</SPAN></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 100%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Sterilize
by autoclaving</SPAN></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; line-height: 100%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><B><SPAN style="background: transparent">5x
Amino Acid Dropout Powder</SPAN></B></FONT></FONT></FONT></P><TABLE cellpadding="2" cellspacing="2"><TBODY><TR><TD><P align="CENTER" style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><B><SPAN style="background: transparent">Reagent</SPAN></B></FONT></FONT></FONT></P></TD><TD><P align="CENTER" style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><B><SPAN style="background: transparent">Quantity</SPAN></B></FONT></FONT></FONT></P></TD></TR><TR><TD><P style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: #ffffff">Sulfur
			dropout powder</SPAN></FONT></FONT></FONT></P></TD><TD><P style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: #ffffff">0.5
			g</SPAN></FONT></FONT></FONT></P></TD></TR></TBODY></TABLE><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 100%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Bring
to a total volume of 200 ml.</SPAN></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 100%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Sterilize
by autoclaving</SPAN></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 100%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Sulfur
dropout powder is a rich supplement mix with cysteine and methionine
omitted:</SPAN></FONT></FONT></FONT></P><TABLE cellpadding="2" cellspacing="2"><TBODY><TR><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><B><SPAN style="background: transparent">Amino
			acids</SPAN></B></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><B><SPAN style="background: transparent">Nucleotide
			bases</SPAN></B></FONT></FONT></FONT></P></TD></TR><TR><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Alanine</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">2.0
			g</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Leucine</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">10.0
			g</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Adenine</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">0.5
			g</SPAN></FONT></FONT></FONT></P></TD></TR><TR><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Arginine</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">2.0
			g</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Lysine</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">2.0
			g</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Uracil</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">2.0
			g</SPAN></FONT></FONT></FONT></P></TD></TR><TR><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Asparagine</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">2.0
			g</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Methionine</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">0.0
			g</SPAN></FONT></FONT></FONT></P></TD></TR><TR><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Aspartic
			acid</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">2.0
			g</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Phenylalanine</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">2.0
			g</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><B><SPAN style="background: transparent">Vitamins</SPAN></B></FONT></FONT></FONT></P></TD></TR><TR><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Cysteine</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">0.0
			g</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Proline</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">2.0
			g</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">p-Aminobenzoic
			acid</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">0.2
			g</SPAN></FONT></FONT></FONT></P></TD></TR><TR><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Glutamic
			acid</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">2.0
			g</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Serine</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">2.0
			g</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Inositol</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">2.0
			g</SPAN></FONT></FONT></FONT></P></TD></TR><TR><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Histidine</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">2.0
			g</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Tyrosine</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">2.0
			g</SPAN></FONT></FONT></FONT></P></TD></TR><TR><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Isoleucine</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">2.0
			g</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Valine</SPAN></FONT></FONT></FONT></P></TD><TD bgcolor="#ffffff"><P style="border: 1px solid #000000; padding: 0.02in 0.08in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">2.0
			g</SPAN></FONT></FONT></FONT></P></TD></TR></TBODY></TABLE><P style="margin-bottom: 0in; font-style: normal; line-height: 100%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><B><SPAN style="background: transparent">Recipe:
M9 Minimal Plates with Sulfur Dropout Powder</SPAN></B></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 100%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">The
plates can be made fresh from pre-sterilized stock solutions.</SPAN></FONT></FONT></FONT></P><TABLE cellpadding="2" cellspacing="2"><TBODY><TR><TD><P align="CENTER" style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><B><SPAN style="background: transparent">Reagent</SPAN></B></FONT></FONT></FONT></P></TD><TD><P align="CENTER" style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><B><SPAN style="background: transparent">Quantity</SPAN></B></FONT></FONT></FONT></P></TD></TR><TR><TD><P style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: #ffffff">5x
			M9 Salts</SPAN></FONT></FONT></FONT></P></TD><TD><P style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: #ffffff">200
			mi</SPAN></FONT></FONT></FONT></P></TD></TR><TR><TD><P style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">5x
			Amino Acid Dropout Powder</SPAN></FONT></FONT></FONT></P></TD><TD><P style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: #ffffff">200
			mi</SPAN></FONT></FONT></FONT></P></TD></TR><TR><TD><P style="border: 1px solid #000000; padding: 0.02in 0.07in"><FONT color="#000000"><SPAN style="text-decoration: none"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: transparent">1
			M MgSO</SPAN></SPAN></SPAN></FONT></FONT></SPAN></FONT><FONT color="#000000"><SPAN style="text-decoration: none"><FONT face="Helvetica Neue"><FONT size="1"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: transparent">4</SPAN></SPAN></SPAN></FONT></FONT></SPAN></FONT></P></TD><TD><P style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">2
			ml</SPAN></FONT></FONT></FONT></P></TD></TR><TR><TD><P style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">20%
			Glucose</SPAN></FONT></FONT></FONT></P></TD><TD><P style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">20
			ml</SPAN></FONT></FONT></FONT></P></TD></TR><TR><TD><P style="border: 1px solid #000000; padding: 0.02in 0.07in"><FONT color="#000000"><SPAN style="text-decoration: none"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: transparent">1M
			CaCl</SPAN></SPAN></SPAN></FONT></FONT></SPAN></FONT><FONT color="#000000"><SPAN style="text-decoration: none"><FONT face="Helvetica Neue"><FONT size="1"><SPAN style="font-style: normal"><SPAN style="font-weight: normal"><SPAN style="background: transparent">2</SPAN></SPAN></SPAN></FONT></FONT></SPAN></FONT></P></TD><TD><P style="border: 1px solid #000000; padding: 0.02in 0.07in; font-style: normal; font-weight: normal; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">0.1
			ml</SPAN></FONT></FONT></FONT></P></TD></TR></TBODY></TABLE><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 100%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Bring
to a total volume of 500 mL</SPAN></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 115%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Stir
and warm on a hot plate to about 60 C (hot to the touch)</SPAN></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 115%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Melt
500 ml 2x Yeast Plate Agar in the microwave for ~8 minutes on high</SPAN></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 115%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Cool
to about 60 C (hot to the touch)</SPAN></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 115%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">This
takes about 30 minutes on the bench or 5 minutes in a water bath.</SPAN></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 115%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">Add
the 2x Yeast Plate agar to the other reagents to make 1 L of media.</SPAN></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 115%; text-decoration: none"><FONT color="#000000"><FONT face="Helvetica Neue"><FONT size="3"><SPAN style="background: transparent">This
recipe will prepare 20 plates at 25 ml / plate.</SPAN></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 115%; text-decoration: none"><FONT color="#000000"><FONT face="Cambria"><FONT size="3"><SPAN style="background: transparent">made
7 plates without antibiotics and 20 with Chloramphenicol and
Spectinomycin - by mistake.</SPAN></FONT></FONT></FONT></P><P style="margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 115%; text-decoration: none"><FONT color="#000000"><FONT face="Cambria"><FONT size="3"><SPAN style="background: transparent">Used
one plate without AB to plate the following strains:</SPAN></FONT></FONT></FONT></P><UL><LI><P style="margin-bottom: 0in; background: transparent; font-style: normal; font-weight: normal; line-height: 115%; text-decoration: none"><FONT color="#000000"><FONT face="Cambria"><FONT size="3"><SPAN style="background: transparent">sD007=sD001+pD005</SPAN></FONT></FONT></FONT></P></LI><LI><P style="margin-bottom: 0in; background: transparent; font-style: normal; font-weight: normal; line-height: 115%; text-decoration: none"><FONT color="#000000"><FONT face="Cambria"><FONT size="3"><SPAN style="background: transparent">sD008=sD001+pD006</SPAN></FONT></FONT></FONT></P></LI><LI><P style="margin-bottom: 0in; background: transparent; font-style: normal; font-weight: normal; line-height: 115%; text-decoration: none"><FONT color="#000000"><FONT face="Cambria"><FONT size="3"><SPAN style="background: transparent">sD013=
	sD001+pD005+ pD004</SPAN></FONT></FONT></FONT></P></LI><LI><P style="margin-bottom: 0in; background: transparent; font-style: normal; font-weight: normal; line-height: 115%; text-decoration: none"><FONT color="#000000"><FONT face="Cambria"><FONT size="3"><SPAN style="background: transparent">sD015=
	sD001+pD006+ pD004</SPAN></FONT></FONT></FONT></P></LI></UL></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Monday 5<SUP>th</SUP>August</H3><P><B><EM>
Confirmation of strains SD002 - SD018
</EM></B></P><DIV dir="ltr">
    Strains sd002-sd018 were grown overnight in 5ml LB.
</DIV></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Monday 5<SUP>th</SUP>August</H3><P><B><EM>
Confirmation of strains SD002 - SD018 using colony PCR
</EM></B></P><DIV dir="ltr">
Fresh single colonies of strains SD002-SD018 were prepared for PCR using the colony pcr protocol. The primers used for were FD_F, FD_R, FNR_F, FNR_R and SirA_F and SirA_R for each strain.
</DIV><DIV dir="ltr">
Strains SD002-SD018 were confirmed.</DIV></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Day Number<SUP>suffix</SUP> Month</H3><P><B><EM>
Place your twit here
</EM></B></P>
Received FrpA and FdxA from IDT.
Started cloning strategy:
Ligated FdxA G block into pJET plasmid for stock purposes, using the following protocol:<P>
    1. Set up the blunting reaction on ice:
</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="308" valign="top"><P>
                    Component
                </P></TD><TD width="308" valign="top"><P>
                    Volume
                </P></TD></TR><TR><TD width="308" valign="top"><P>
                    2X Reaction Buffer
                </P></TD><TD width="308" valign="top"><P>
                    10 µl
                </P></TD></TR><TR><TD width="308" valign="top"><P>
                    Non-purified PCR product
                </P><P>
                    or
                </P><P>
                    purified PCR product/other sticky-end DNA fragment
                </P></TD><TD width="308" valign="top"><P>
                    1 µl
                </P><P>
                    0.15 pmol ends
                </P></TD></TR><TR><TD width="308" valign="top"><P>
                    Water, nuclease-free
                </P></TD><TD width="308" valign="top"><P>
                    to 17 µl
                </P></TD></TR><TR><TD width="308" valign="top"><P>
                    DNA Blunting Enzyme
                </P></TD><TD width="308" valign="top"><P>
                    1 µl
                </P></TD></TR><TR><TD width="308" valign="top"><P>
                    Total volume
                </P></TD><TD width="308" valign="top"><P>
                    18 µl
                </P></TD></TR></TBODY></TABLE><P>
    Vortex briefly and centrifuge for 3-5 s.
</P><P>
    2. Incubate the mixture at 70°C for 5 min. Chill on ice.
</P><P>
    3. Set up the ligation reaction on ice. Add the following to the blunting reaction mixture:
</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="308" valign="top"><P>
                    Component
                </P></TD><TD width="308" valign="top"><P>
                    Volume
                </P></TD></TR><TR><TD width="308" valign="top"><P>
                    pJET1.2/blunt Cloning Vector (50 ng/µl)
                </P></TD><TD width="308" valign="top"><P>
                    1 µl (0.05 pmol ends)
                </P></TD></TR><TR><TD width="308" valign="top"><P>
                    T4 DNA Ligase
                </P></TD><TD width="308" valign="top"><P>
                    1 µl
                </P></TD></TR><TR><TD width="308" valign="top"><P>
                    Total volume
                </P></TD><TD width="308" valign="top"><P>
                    20 µl
                </P></TD></TR></TBODY></TABLE><P>
    Vortex briefly and centrifuge for 3-5 s to collect drops.
</P><P>
    4. Incubate the ligation mixture at room temperature (22°C) for 5 min.
</P></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Tuesday 20<SUP>th</SUP> August</H3><P><B><EM>
Cloning of FdxA and FprA
</EM></B></P>
Digestion of FprA with NdeI and AvrII
Gel Clean up of FprA fragment
Digestion of pETDuet-1 with NdeI and AvrII
Digestion of FdxA with NcoI and HindIII
Digestion of pETDuet-1 NcoI and HindIII

</DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Wednesday 21<SUP>st</SUP> August</H3><P><B><EM>
Ligation and Transformations
</EM></B></P>
Ligation of FprA NdeI AvrII with pET NdeI AvrII
Ligation of FdxA NcoI HindIII with pET NcoI HindIII
Transformation of pET FprA into NEBTurbo
Transformation of pET FdxA into NEBTurbo
</DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Thursday 22<SUP>nd</SUP> August</H3><P><B><EM>
Nano drop
</EM></B></P>
Plasmid mini-prep of transformations
Nano dropped
<OL><LI>pETDuet: 59.1ng/ul</LI><LI>pETDuet: 45.9ng/ul</LI><LI>pJET Fdx: 54.2ng/ul</LI><LI>pLS FRP: 19.7ng/ul</LI></OL>Digest pLS FrpA with NdeI and AvrII
Digest pETDuet with NdeI and AvrII
Digest FdxA gBlock with NcoI and HindIII
Digest pETDuet with NcoI and HindIII
Ligation of FprA with pETDuet
Ligation of FdxA with pETDuet
Transformation of pET FprA and pET FdxA into NEBTurboPCR of 
<UL><LI>FdxA gBlock with FdxA F/R</LI><LI>pLS FprA with FprA F/R</LI><LI>pET duet with MCSI F/R</LI><LI>pACYC duet with MCSI F/R</LI></UL></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Friday 23<SUP>rd</SUP> August</H3><P><B><EM>
Patching
</EM></B></P>
Patched colonies of pET FprA and colonies of pJET FdxA
Transformations of pET Fdx didn't work
Checking strains containing plasmids fpr to make sure genes are there.
Results from Thursdays PCR:
<UL><LI>no band for fdxA</LI><LI>band for fprA</LI><LI>band for pET</LI><LI>no band for pACYC</LI><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Saturday 24<SUP>th</SUP> August</H3><P><B><EM>
Weekend warriors!
</EM></B></P>
Nanodrop <UL><LI>FdxA 1 3.1ng/ul</LI><LI>FdxA 2 4.3ng/ul</LI><LI>pET Duet 6.2ng/ul</LI><LI>pET Duet 4.9ng/ul</LI></UL>Checking FdxA gBlock by PCR
<UL><LI>1 ul Fdx F</LI><LI>1 ul Fdx R</LI><LI>1 ul Template</LI><LI>7 ul H20</LI><LI>10 ul dreamTaq 2x master mix</LI>
PCR failed.
<DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Sunday 25<SUP>th</SUP> August</H3><P><B><EM>
Ligations
</EM></B></P>
For FprA and pET Duet:
<OL><LI>Miniprep more plasmid</LI><LI>Digest plasmid and vector backbone</LI><LI>Check DNA concentration</LI><LI>Ligate</LI><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Monday 26<SUP>th</SUP> August</H3><P><B><EM>
P1 phage transduction
</EM></B></P>  The p1 phage lysate was prepared as described in the P1 phage transduction protocol.
  BL21 (DE3) <I>ΔCysI Δfpr ΔydbK </I>was grown in 5ml Lb.
One colony was discovered in transformed plate.  This colony was re-streaked and grown overnight.
the ligation product from overnight ligation was used to transorm NEB cc cells by heatshock
5ul plate gave a few colonies
10ul plate gave many colonies
</DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Tuesday 27<SUP>th</SUP> August</H3><P><B><EM>
Sequencing FdxA
</EM></B></P>
pJET FdxA sent for sequecing

<DIV dir="ltr">
    Steps 2-7 of the P1 phage transduction: Lysate preparation were carried out.
</DIV><DIV dir="ltr">
    BL21(AI) was picked and placed into 5ml LB and incubated at 37C overnight.
</DIV></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Wednesday 28<SUP>th</SUP> August</H3><P><B><EM>
P1 phage transduction
</EM></B></P><P>
    The transduction section of the P1 phage transduction protocol was carried out.
    Cells were plated on Kanamyocin plates to select for resistance and grown overnight.
</P></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Thursday 29<SUP>th</SUP> August</H3><P>
    Colonies from the P1 phage transduction grew on the Kanomycin plates.
</P><DIV>
    Selected colonies were patched onto Kan and LBA plates and left to grow overnight.
</DIV></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Friday 30<SUP>th</SUP> August</H3><P><B><EM>
Confirmation of P1 phage transduction:
</EM></B></P><P>
    Colony PCR was used on patched colonies from the p1 phage transduction. The primers used for each colony were CysI F/R and KanR F/R.
</P><DIV>
    PCR confirmed that CysI was deleted in BL21(AI) and replaced with a Kanomycin resistance gene.</DIV><DIV>
2 Single colonies from each agar plate of the patched p1 phage transduced plates were picked and used to inoculate 5ml LB broth overnight. 
750ml of overnight culture was added to 250ml of 60% glycerol in a cryotube.
</DIV></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Monday 9<SUP>th</SUP> September</H3><P><B><EM>
Sevres Workshop
</EM></B></P>
Sevres Workshop
</DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Tuesday 10<SUP>th</SUP> September</H3><P><B><EM>
Sevres Workshop
</EM></B></P>
Sevres Workshop
</DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Monday 16<SUP>th</SUP> September</H3><P><B><EM>
Cloning gBlock FdxA and Biobricking FprA and SirA
</EM></B></P><P>FprA and SirA were PCRed with FprA BB F, FprA BB R, SirA BB F and SirA BB R primers respectfully.  The PCR was run with an annealing temperature of 60C with an extension time of 1m40s using Phusion polymerase.  The band of ~1400bp was cut out and cleaned up using a Quagen Gel Extraction Kit.</P><P>Restriction Digestion of gBlock FdxA with NcoI and HindIII.  pET FprA was also digested with NcoI and HindIII. Digestion lasted 30 minutes at 37C afterwhich enzymes were heat inactivated at 80C for 20 minutes.  Digestions were ligated together using 6.5ul of FdxA insert and 2ul of FprA into a total volume of 10ul.</P><P>PCR products for FprA and SirA along with linearized pSB1c3 backbone were digested with EcoRI and PstI for 30m at 37C.  Restriction enzymes were then heat inactivated at 80C for 20 minutes.  Digestions were then ligated together using equal amounts of 2ul insert to 2ul of backbone.</P></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Tuesday 17<SUP>th</SUP> September</H3><P><B><EM>
Transforming gBlock FdxA and Biobricking FprA and SirA
</EM></B></P>
Ligations were transformed into E. coli NEBTurbo Chemically Competent Cells by Heat Shock transformation. Cells were recovered for 1.5 hours before being plated.  pSB1c3 fprA and pSB1c3 SirA were plated onto Chloramphenicol plates (34ug/ul) and pET FprA FdxA were plated onto Ampicillin (50ug/ul) plates.
</DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Wednesday 18<SUP>th</SUP> September</H3><P><B><EM>
Patching gBlock FdxA and Biobricking FprA and SirA
</EM></B></P>
Colonies from Transformation were patched onto appropriate antibiotic plates, 15 colonies from SirA were patched, 6 colonies from FprA were patched and 4 colonies from FdxA were patched.
</DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Thursday 19<SUP>th</SUP> September</H3><P><B><EM>
PCR Verification of vectors
</EM></B></P><P>Vectors (pET FprA FdxA, pSB1c3 SirA, and pSB1c3 FprA) were used for colony PCR using DreamTaq.  Primers used were:</P><P>pET FprA FdxA: MCSI F/R, MCSII F/R, FprA F/R, FdxA F/R</P><P>pSB1c3 SirA: SB-prep-3P-1/SB-prep-2Ea, SirA F/R</P><P>pSB1c3 FprA: SB-prep-3P-1/SB-prep-2Ea, FprA F/R</P></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Saturday 21<SUP>st</SUP> September</H3><P><B><EM>
Weekend Rush
</EM></B></P><P>Gibson assembly of pSB9S1, pSB10K1, and pSB3C7.</P><P>Digestion of sirA, fprA, fdxA and linearized backbone with XbaI and SpeI</P><P>Digestion of Linearized pSB1C3 with XbaI, SpeI, and DpnI</P><P>Ligation of sirA, fprA, and fdxA with pSB1C3</P><P>Heat Shock transformation of pSB1C3 sirA, pSB1C3 fprA and pSB1C3 fdxA into NEBTurbo</P><P>Heat Shock transformation of pSB9S1, pSB10K1 and pSB3C7 into NEBTurbo</P></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Sunday 22<SUP>nd</SUP> September</H3><P><B><EM>
Verifying shipment vectors
</EM></B></P><P>Growing Culture of pSB1C3 sirA, pSB1C3 fprA and pSB1C3 fdxA.</P><P>Mini-prep of pSB1C3 sirA, pSB1C3 fprA and pSB1C3 fdxA vectors </P><P>Colony PCR of pSB1C3 sirA, pSB1C3 fprA and pSB1C3 fdxA with sirA F/R, fprA F/R, fdxA F/R, and biobrick verification primers F/R</P></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Friday 27<SUP>th</SUP> September</H3><P><B><EM>
Growth Curves
</EM></B></P><P>IPTG Concentrations by column: 1 0nM, 2 10nM, 3 50nM, 4 100nM, 5 500nM, 6 1uM, 7 5uM, 8 10uM, 9 50uM, 10 100uM, 11 500uM, 12 1mM
</P><P>Arabinose Concentrations by column: 1 0%, 2 0.01%, 3 0.02%, 4 0.03%, 5 0.04%, 6 0.05%, 7 0.06%, 8 0.07%, 9 0.08%, 10 0.09%, 11 0.1%, 12 0.2%

</P><P>Row A: BL21 AI dcysI pACYC SirA pET FprA FdxA 1
Row B: BL21 AI dcysI pACYC SirA pET FprA FdxA 1
Row C: BL21 AI dcysI pACYC SirA pET FprA FdxA 2
Row D: BL21 AI dcysI pACYC SirA pET FprA FdxA 2
</P><P>Row E: BL21 (DE3) dcysI pACYC SirA pET FprA FdxA 1
Row F: BL21 (DE3) dcysI pACYC SirA pET FprA FdxA 1
Row G: BL21 (DE3) dcysI pACYC SirA pET FprA FdxA 2
Row H: BL21 (DE3) dcysI pACYC SirA pET FprA FdxA 2

10mL 5x M9 Salts, 10mL 5x Amino Acid Sulfur dropout, 0.5mL MgSO4, 2mL 20% Glucose, 5ul CaCl2, 27.5mL H2O
</P></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Saturday 28<SUP>th</SUP> September</H3><P><B><EM>
Growth Curves
</EM></B></P><P>Growth curves of BL21 (DE3) dcysI pACYC SirA pET FprA FdxA and BL21 AI dcysI pACYC SirA pET FprA FdxA.

Column 1: Positive control WT E. coli BL21 (DE3), 2: Negative Control BL21 AI dcysI, 3: BL21 AI dcysI pACYC SirA pET FprA FdxA 0 Arabinose 0 IPTG, 4: 0.1% Arabinose 10uM IPTG, 5: 0.05% Arabinose 50uM IPTG, 6: 0.01% Arabinose 100uM IPTG, 7: 0.2% Arabinose 1mM IPTG, 8: Negative Control BL21 (DE3) dcysI dfpr dydbf, 9: BL21 (DE3) dcysI pACYC SirA pET FprA FdxA 0 IPTG, 10: 50nM IPTG, 11: 1uM IPTG, 12: 50uM IPTG</P><P>Grown in M9 Media with Galactose.</P><P>10mL 5x M9 Salts, 10mL 5x Amino Acid Sulfur dropout, 0.5mL MgSO4, 2mL 20% Galactose, 5ul CaCl2, 27.5mL H2O
</P></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Sunday 29<SUP>th</SUP> September</H3><P><B><EM>
PCR's and more!
</EM></B></P><P>Colony PCR on new Backbones.  RFP and Biobrick verification primers
This is for pSB2C and pSB3K

</P><P>Colony PCR on SirA, FprA, and FdxA Biobricked parts
Using Biobrick primers and SirA, FprA and FdxA primers respectfully
https://2013.igem.org/File:20130929_BioB.jpg
https://2013.igem.org/File:20130929_RFP.jpg
https://2013.igem.org/File:20190929_pSB3K.jpg

</P><P>Phusion PCR on Duet vectors and pSB1C3 using standard cloning primers

</P><P>Gel Clean up

</P><P>Digestion with NcoI and AvrII

</P><P>Standard cloning of new backbones (1h ligation at 16C)

</P><P>Transform into NEBTurbo

</P><P>Cutting Biobrick SirA, FprA, FdxA and Linearized backbone with EcoRI and SpeI

</P><P>Standard clonging of Biobrick parts (1h ligation at 16C)

</P><P>Transform into NEBTurbo

</P><P>Treat XbaI SpeI cut Backbone with TSAP 1h.

</P><P>Standard cloning of Biobrick parts (cut with XbaI, SpeI) into TSAP'd backbone (1h @ 16C)

</P><P>Colony PCR on BL21 (DE3) dcysI pACYC SirA pET FprA FdxA and BL21 AI dcysI pACYC SirA pET FprA FdxA
</P></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Monday 30<SUP>th</SUP> September</H3><P><B><EM>
Growth Curves
</EM></B></P>
We performed Growth Curves with the following strains in each column:
<OL><LI>WT BL21 AI</LI><LI>BL21 AI dcysI</LI><LI>BL21 AI dcysI pD002 pD003 0uM IPTG 0% Arabinose</LI><LI>BL21 AI dcysI pD002 pD003 10uM IPTG 0.05% Arabinose</LI><LI>BL21 AI dcysI pD002 pD003 100uM IPTG 0.1% Arabinose</LI><LI>BL21 AI dcysI pD002 pD003 1mM IPTG 0.2% Arabinose</LI><LI>WT BL21 DE3</LI><LI>BL21 DE3 dcysI dfpr dydbf</LI><LI>BL21 DE3 dcysI dfpr dydbf pD002 pD003 0uM IPTG</LI><LI>BL21 DE3 dcysI dfpr dydbf pD002 pD003 10uM IPTG</LI><LI>BL21 DE3 dcysI dfpr dydbf pD002 pD003 100uM IPTG</LI><LI>BL21 DE3 dcysI dfpr dydbf pD002 pD003 1mM IPTG</LI>

Results are https://2013.igem.org/File:20130930_GC_Glucose.xls
<DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Tuesday 1<SUP>st</SUP> October</H3><P><B><EM>
Israeli Collaboration
</EM></B></P><P>Plasmids arrived.  PCR CI plasmid with CI biobrick F/R primers.</P><P>Gel Clean-up of CI promoter PCR</P><P>Z score of BL21 dCysI pD002 pD003:<B>Didn't Grow, forgot Sucrose in M9</B></P><P>Transform pSB1A3 E0240 and pSB3K3 into NEB Turbo</P></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Wednesday 2<SUP>nd</SUP> October</H3><P><B><EM>
Race to finish stuff
</EM></B></P><P>Grew up 5ml cultures of pSB3K3 and pSB1A3 E0240 from transformations</P><P>Plasmid mini preps of pSB3K3 and pSB1A3 E0240</P><P>Restriction Digest of CI promoter PCR product with XbaI and PstI</P><P>Restriction Digest of Linearized Backbone pSB1C3 with XbaI, PstI, and DpnI</P><P>Make new NEB Turbo Chemically Competent Cells: Modified protocol.  4mL of NEBTurbo into 2 x 2mL tubes.  Centrifuge at 14000 rpm for 1 minute at 4C.  Resuspend in 500uL of 100mM MgCl2 Ice cold.  Centrifuge at 14000 rpm for 1 minute at 4C.  Resuspend in 500uL of iGEM cc cell buffer Ice cold.  Centrifige for 1 minute at 14000 rpm at 4C.  resuspend in 100ul of iGEM cc cell buffer Ice cold.</P><P>Heat Shock Transformation of pSB1C3 SirA, pSB1C3 FprA, pSB1C3 FdxA into NEBTurbo (new chemically competent cells)</P><P>Ligation of CI promoter with pSB1C3 backbone.</P><P>Heat Shock Transformation of pSB1C3 CI into NEB Turbo</P></DIV><DIV class="tbnote"><H2>Target</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" target="_blank" class="tbnotelogo DSlogo"> ASDF </A><H3>Thursday 3<SUP>rd</SUP> October</H3><P><B><EM>
Race to Finish stuff
</EM></B></P><P>Plated new SirA, FprA, and FdxA transformations on CHL</P><P>Picked pSB1C3 CI and grew in LB broth</P><P>Z Score of BL21 dCysI pD002 pD003 with Gentamicin as positive hit control</P><P>Analytical Digestions of New Vectors (pAD1S, pAD2C, pAD3K, pAD4A)</P><P>Digest pSB1C3 CI with EcoRI and SpeI</P><P>Digest pSB3K3 with EcoRI and PstI</P><P>Digest pSB1A3 E0240 with XbaI and PstI</P><P>Ligation of CI promoter with GFP reporter and pSB3K3 backbone</P><P>Heat Shock Transformation of pSB3K3 CI GRP into NEBTurbo</P><P>PCR verify SirA, FprA, FdxA, and CI biobrick constructs</P></DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Monday 22<SUP>nd</SUP> July</H3>
Transforming the NEB TURBO strain with the pET21b and the pUC18 plasmids. Competent cells were prepared by the CaCl2 protocol. The transformation cultures were plated on ampicilline plates. Transformation was unsuccessful.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Tuesday 23<SUP>rd</SUP> July</H3>
Transforming the new NEB strain with the pET21b and the pUC18 plasmids (as yesterday). We’ve also plated the leftovers from the yesterdays' transformation (190 µL of culture on ampicilline plates).
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Wednesday 24<SUP>th</SUP> July</H3>
Transformation form yesterday didn’t work. We plated the leftovers on new ampicillin plates.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Thursday 25<SUP>th</SUP> July</H3>
The transformation worked, probably it failed the last two times because the recovery time was to short or the plates were wrong. We did a streak plate to get single colonies.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Friday 26<SUP>th</SUP> July</H3>
Single colonies were plated into LB + ampicillin. 4 liquid cultures were prepared: NEB + pET21b (1),  NEB + pET21b (2), jNEB + pET21b (1), jNEB + pET21b (2). After 6 h of incubations the liquid cultures were used for
•	Glycerol stock preparation.
•	Plasmid purification (the protocol from the box).
DNA concentrations were measured by the nanodrop (ng/µL):
NEB1	134.4
NEB2	55.6
jNEB1	186.6
jNEB2	138.6
	

</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Monday 29<SUP>th</SUP> July</H3>
We did an analytical digest of pET 21b that we purified yesterday. We used Nedl and Xhol so our fragments should be: the backbone of the plasmid (around 5000 bp long) and the cloned MSMEG_1529 gene (around 600 bp long). On the following picture is our plasmid and we suppose that the MSMEG_1592 gene is cloned somewhere in between the two restriction sites that we have chosen. But we still don't really know that because we need the full sequence of the plasmid.

The protocol used for preparing the digestion:
<TABLE border="1"><TBODY><TR><TD>Reagent</TD><TD>Volume</TD><TD>10x</TD></TR><TR><TD>Plasmid Miniprep</TD><TD>5µl</TD></TR><TR><TD>H2O</TD><TD>12 µl</TD><TD>120 µl</TD></TR><TR><TD>10x FastDigest Buffer</TD><TD>2 µl</TD><TD>20 µl</TD></TR><TR><TD>10x FastDigest Buffer</TD><TD>2 µl</TD><TD>20 µl</TD></TR><TR><TD>FastDigest Enzyme 1</TD><TD>0.5µl</TD><TD>5 µl</TD></TR><TR><TD>FastDigest Enzyme 2</TD><TD>0.5µl</TD><TD>5 µl</TD></TR><TR><TD>Total Volume</TD><TD>20.0 µl</TD><TD>200 µl</TD></TR></TBODY></TABLE>
Digest for 1 h at 37 °C with shaking.

NEB1 and 2 are plasmids isolated from two different liquid cultures of the NEB strain; jNEB1 and 2 are plasmids isolated from two different liquid cultures of the NEB strain which was made competent by Jake. The (-) stands for a negative control: only DNA was added, without any digestion enzymes. As you can see it on the gel there are 2 bands - one is around 5000 bp long (fits the length of our plasmids backbone), and the other one is around 700 bp long (fits the length of our gene).
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Tuesday 30<SUP>th</SUP> July</H3>
Transformed the bl21 (DE3) strain with the pET21b (plated the transformation culture on 2 amp plates).
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Wednesday 31<SUP>st</SUP> July</H3>
Transformation worked. The number of transformed colonies is: 78 on plate 1 and 284 on plate. We prepared the liquid cultures and glycerol stocks of bl21 (DE3) + pET21b strain.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Thursday 1<SUP>st</SUP> August</H3>
We want to see if 1 mM solution of IPTG induces the over expression in our strain. 
We prepared 4 liquid cultures (no antibiotics added): 
•	bl21 (DE3) + pET21b strain induced with 1 mM IPTG
•	uninduced bl21 (DE3) + pET21b strain
•	untransformed bl21 (DE3) + 1 mM IPTG
•	untransformed bl21 (DE3)

10 mL of culture had 500 µL of the overnight culture (O.D.600 was set to 0.2) and 100 µL of 100 mM IPTG.
We measured the O.D. of the cultures (600 nm) for time 0 and after 6 h. We also diluted our cultures to calculate the CFU/mL (for time 0 10-1 and 10-2 dilutions were prepared, and 100 µL of each dilution was plated on LBA plates; for 6 h 10-3, 10-4 and 10-5 for the induced culture and 10-7, 10-8 and 10-9 for all the others, 10 µL of each were plated). We also saved 2 mL for each group in the fridge.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Friday 2<SUP>nd</SUP> August</H3>
We repeated the time 6 measurement (the cultures were over diluted, so this time we did 10-5, 10-6 and 10-7). We also did the 24 h measurement (we used the 10-6, 10-7 and 10-8 dilutions).
The final results are:

Growth inhibition could be noticed for the induced strain, which is probably a consequence of the over expression. The induced culture recovered and reached a greater CFU/mL value than in the previous groups. Maybe because we didn’t add any antibiotics in our media, the plasmid was lost and the culture could recover after achieving the over expression.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Monday 5<SUP>th</SUP> August</H3>
We want to repeat our previous experiment, but now we will add ampicillin in the media containing the transformed strains.  We set everything as previously, did the 0 h (100 µL of 10<SUP>-1</SUP> and 10<SUP>-2</SUP>) and 6 h measurement (100 µL of 10<SUP>-1</SUP> and 10<SUP>-2</SUP> for the induced strain; 100 µL of 10<SUP>-1</SUP> and 10<SUP>-6</SUP> for the others).
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Tuesday 6<SUP>th</SUP> August</H3>
We did the 24 h measurement (100 µL of 10<SUP>-6</SUP> and 10<SUP>-7</SUP>), and repeated the 0 h measurement (100 µL of 10<SUP>-3</SUP> dilution).
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Wednesday 7<SUP>th</SUP> August</H3>
Repeated the 0 h measurement for uninduced bl21 (DE3) + pET21b strain (100 µL of 10<SUP>-3</SUP> dilution), and the 24 h measurement for the induced strain (100 µL of 10<SUP>-5</SUP>, 10<SUP>-4</SUP>, 10<SUP>-3</SUP> and 10<SUP>-2</SUP>).

</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Thursday 8<SUP>th</SUP> August</H3>
The growth curve for uninduced and induced bl21 (DE3), with antibiotics in the liquid culture:

The O.D. grows again after 24 h for the induced strain, but the CFU stays the same. Probably the O.D. grows because of the protein.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Monday 12<SUP>th</SUP> August</H3>
In order to grow M. smegmatis we should add Tween 80 (0.2 % / v.v) into the LB media (it stops the cells from forming clumps). In some papers we found that Tween 80 could inhibit the growth of E. coli, so we did a growth curve to check that. 500 μL of BL21 + pET21b culture (O.D. 600 nm = 1.00) was added to 10 mL of LB supplemented with different concentrations of Tween 80 – 0.2, 0.1 and 0.05 %. There were two sets of these cultures – one was induced with IPTG and the other one was not. O.D. was measured after 3 and 6 h. At these time periods cultures were also diluted to calculate the CFU.
No growth inhibition was observed, so we can use Tween 80 in our E.coli – M. smegmatis co-cultures.
We have also made some plates with malachite green (40, 4 and 0.4 μg/mL) and we have plated E. coli on them.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Tuesday 13<SUP>th</SUP> August</H3>
We finally got M. smegmatis today!!! We made liquid cultures in LB + 0.2% Tween 80 and we also plated it on LBA.

E. coli doesn’t grow on plates with 40 and 4  μg/mL of malachite green, but it does on  0.4 μg/mL.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Wednesday 14<SUP>th</SUP> August</H3>
Streak plate and liquid cultures for M. smegmatis and E. coli BL21 + pET21b.
We started a plate reading experiment in order to see which concentrations of nalidixic acid should we use in our selective media. The tested concentrations were: 50, 25, 12.5, 6.25,3.125, 1.5625, 0.78125, 0.390625, 0.1953125, 0.09765625, 0.048828125 μg/mL. We also tested different concentrations of IPTG, to see which ones induce the strain without killing it. The tested concentrations were: 1, 0.5, 0.25, 0.125, 0.0625, 0.03125, 0.015625, 0.0078125, 0.00390625, 0.001953125, 0.0009765625  μg/mL. The concentration of bacteria in each well was 1 % and the bl21 culture of O.D. = 0.5 was used.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Thursday 15<SUP>th</SUP> August</H3>
Another plate reading experiment, with M. smegmatis for nalidixic acid (50, 25, 12.5, 6.25,3.125, 1.5625, 0.78125, 0.390625, 0.1953125, 0.09765625, 0.048828125 μg/mL) and with malachite green (400, 200, 100, 50, 25, 12.5, 6.25, 3.125, 1.5625, 0.78125, 0.390625  μg/mL); The concentration of bacteria was 1 % in LB with 0.2 % of Tween 80 and the culture with O.D. of 0.5 was used.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Friday 16<SUP>th</SUP> August</H3>
We started a growth curve for M. smegatis. In 20 mL of LBA with 0.2 % of Tween 20 μL of the liquid culture was added (O.D. Of the culture which was inoculated was 1). O.D. And CFU were determined each 3 hours. Dilutions which were used are: 10-1 and 10-2 for time 0, 10-2 and 10-3 for 3 h, 10-5 and 10-6 for 9 h.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Saturday 17<SUP>th</SUP> August</H3>
Because we suspect that the culture was contaminated we repeated the experiment with 3 separate cultures. n 20 mL of LBA with 0.2 % of Tween 20 μL of the liquid culture was added (O.D. Of the culture which was inoculated was 1). O.D. And CFU were determined each 3 hours. Dilutions which were used are: 10-1 and 10-2 for time 0, 10-2 and 10-3 for 3 h, 10-5 and 10-6 for 9 h, 10-7 for 24 h. We saved some plates so we could perform acid fast staining and check if its really M. smegmatis. 
We have transformed bl21 by electroporation with pET21b and with pSB1C3, so we could get RFP E. coli expressing our protein. The transformed culture was plated on plates containing ampicilline and chloramphenicol.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Sunday 18<SUP>th</SUP> August</H3>
Made a liquid culture of BL21 + pET21b + pSB1C3.
Plated the strain which contains the LLO plasmids (pACYZ184) on LBA + tetracycline + chloramphenicol, LBA +  tetracycline and LBA +  chloramphenicol because we don’t know which resistance the plasmid really has.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Monday 19<SUP>th</SUP> August</H3>

LLO plasmid containing bacteria are growing only on chloramphenicol containing plates, so tetracyclin resistance was lost due to the cloning of the hly downstream of the tet promoter. 

The cells we transformed with pET21b (ampiciline resistance) and with the pSB1C3 plasmid (chloramphenicol resistance) are not growing on amp and chl containing plates because the origins of the plasmids are not compatible. So we are repeating the transformation with the pSB3C5 plasmid, which has a different origin.

We have started a co-culture with M. smegmatis and E. coli NEB turbo + pSB1C3 to see if our selective media with nalidixic acid is really working. Two cultures in their stationary phase were mixed (O.D. for M. smegmatis was 1.5 and for E. coli was 1): in 10 mL of LB 100 µL of each culture was added. We did dilution series for the start cultures and for the co-culture. Each was plated on: LBA plates where both strains should grow; on LBA + 3 µg/mL nalidixic acid plates, where only M. smegmatis should grow; on chl plates where only E. coli should grow.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Tuesday 20<SUP>th</SUP> August</H3>
We did a miniprep for the LLO plasmid (we had to do dialysis to purify the plasmid).

We observed the RFP producing cells (BL21 + pET21b + pSB3C5) under the fluorescent microscope and the cells produce RFP. We made a liquid culutre of the strain with 1 mM IPTG. We also plated them on LBA + 1 mM IPTG.

Liquid culture of M. smegmatis was made, but in the liquid culture 2 µL/mL of malachite green was added. We plated the culture on LBA + 3 µL/mL of nalidixic acid and on LBA to compare the number of colonies.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Wednesday 21<SUP>st</SUP> August</H3>
Our macrophages arrived today and we are staring a cell culture!
The cells with pET21b and pSB3C5 can't survive the induction with 1 mM IPTG because they have two IPTG inducable promoters. We want to use a different insert, one with an uninducable promoter, such as BBa_J23100-118.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Thursday 22<SUP>nd</SUP> August</H3>
Counting the colonies from Monday; our plates with nalidixic acid are selective. We also noticed that the colonies showed up faster on the plates for the liquid culture supplemented with malachite green.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Monday 26<SUP>th</SUP> August</H3>
We started a killing assay for M. smegmatis: we had two groups; in the first one the ratio between M. smegmatis and E. coli BL21 + pET21b was 1:1 and in the other it was 1:2. BL21 was induced with 1 mM IPTG in the co-culture (10 mL of LB, 0.1 mL of each strain and 0.1 mL of 100 mM IPTG). For each group a control was made - same ratio between the strains but no induction. Also we had one group with only M. smegmatis. We did dilution series for time 0, after 3 and after 6 h. We also measured the O.D. (600 nm) for each time point.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Tuesday 27<SUP>th</SUP> August</H3>
We did a digestion of J23102 and J23104 with EcoRI and with and Pstl and we gel purified the RFP gene with a constitutive promoter. We also did a digestion for pSB3C5 with the same enzymes and we did gel purification for the backbone. The ligation product was then transformed into NEB Turbo.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Wednesday 28<SUP>th</SUP> August</H3>
Cloning from yesterday didn't work, so we will repeat everything with a different backbone. On gels for the digested pSB3C5 we couldn't see the part which was cut out, so we guess there is something wrong with this plasmid and we want to use a new one - pColaDuet-1 with a colA origin. We amplified the insert form J23102 and J23104 with biobrick verification primers and then we did PCR purification. We digested the insert and the backbone with EcoRI and Pstl and we did PCR purification for both of them. After measuring the concentration for both, we did a ligation in molar ration insert:backbone = 3:1 on 16 deg C for 1 h. The ligation product was transformed into NEB Turbo.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Thursday 29<SUP>th</SUP> August</H3>
We did a phagocytosis assay for our macrophages. First we passaged our cells and we determined their number (colored them with trypan blue and counted them): 2.68* 10<SUP>6</SUP>, 3.72* 10<SUP>6</SUP> and 1.32* 10<SUP>6</SUP>. To the first one we added 40, to the second 30 and to the last one 90 µl of NEB + pSB1A3 cell suspension. After 1 h we made this awesom picture:

Cloning from yesterday didn't work, so we repeated everything with longer digestion time for the backbone.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Friday 30<SUP>th</SUP> July</H3>
Cloning worked. Made liquid cultures. Counted the colonies for the killing assay.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Saturday 31<SUP>st</SUP> July</H3>
miniprep for pColADuet-1 + J23102/104. PCR for pET21b and the LLO plasmid (it didn't work).
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Monday 2<SUP>nd</SUP> September</H3>

Transformed BL21 + pET21b with pColA + J23102/104 (electroporation and heat shock), transformed BL21 + pET21b with pColA + J23102/104 and with the LLO plasmid.

The PCR for the pET21b worked and we've sent it for sequencing; the PCR for the LLO plasmid didn't work :(
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Tuesday 3<SUP>rd</SUP> September</H3>
Liquid cultures for BL21 + pET21b + pColA and for BL21 + pET21b + pColA + the LLO plasmid. We passaged the macrophages and we ordered new primers for sequencing the LLO plasmid.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Wednesday 4<SUP>th</SUP> September</H3>
The killing assay for M. smegamtis, with more time points. Now we can also follow the number of E. coli because it has RFP. In 20 mL of LB we added 100 µL of the M. smegmatis liquid culture (O.D. = 1.2) and 100 µL of the E. coli liquid culture (O.D. = 1) so we get 1:1 ratio. We measured the O.D. and the CFU for time 0, 1, 2, 3 and 6 h. To do the CFU for E. coli we plate the dilution on LBA + Kan + Amp, and for M. smegmatis we plate on LBA + 30 µL.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Thursday 5<SUP>th</SUP> September</H3>
Counted the colonies for E. coli and prepared everything for another killing assay. Liquid cultures for E. coli BL21 + pET21b + the LLO plasmid + the RFP plasmid; BL21 + pET21b + the RFP plasmid; M. smegmatis.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Friday 6<SUP>th</SUP> September</H3>
New killing assay with different E. coli and M.smegmatis ratio --&gt; 0.1 : 1
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Saturday 7<SUP>th</SUP> September</H3>
New killing assay with a new ratio --&gt; E. coli : M. smegmatis = 0.01 : 1
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Monday 9<SUP>th</SUP> September</H3>
Counting the colonies from the killing assay and doing the Bradford assay for each time point.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Tuesday 10<SUP>th</SUP> September</H3>• PCR for the linearized backbone: template - pSB1C3 (1 µL), primers (SB-prep-2Eb, SB-prep-3P-1; 2.5 µL), HE buffer (10 µL), dNTPs (1 µL), polymerase (0.5 µL), dH2O (32.5 µL).

program: 94.0 (30 s) - 94.0 (10 s), 55.0 (1 min), 72.0 (2 min), 72.0 (3 min) - 30 cycles. 
• touchdown PCR for the new backbones:

<TABLE border="1"><TBODY><TR><TD>Template</TD><TD>Primers</TD></TR><TR><TD>pACYC</TD><TD>pDCA2 and pDCA</TD></TR><TR><TD>pET</TD><TD>pDCo4-F and pDCo4-R</TD></TR><TR><TD>pCDF</TD><TD>pDCA1 and pDCA</TD></TR><TR><TD>pColA</TD><TD>pDCA3 and pDCA</TD></TR></TBODY></TABLE>
program: 95.0 (10 s) - 95.0 (30 s), 60.0 (30 s), 59.0 (30 s),58.0 (30 s), 56.0 (30 s), 55.0 (30 s), 72.0 (30 s) - 72.0 (10 min)- 30 cycles.

</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Wednesday 11<SUP>th</SUP> September</H3>
PCR for biobricking TDMH, for sequencing the LLO plasmid and for adding biobrick cutsites to pET21b; PCR purification of the products. PCR for the linear backbone and PCR purification. PCR for the new backbones gel slice and purification.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Thursday 12<SUP>th</SUP> September</H3>• linear bacbone PCR: Phusion protocol from the 10th of the September, new PCR program: 98.0 (30 s) - 98.0 (10 s), 58.0 (15 s), 72.0 (1:10) - 72.0 (3 min). This time the PCR worked fine - no unspecific binding; the band was 2 kb long.
• PCR for biobricking TDMH: we designed primers to add biobrick cutsites on our TDMH gene. Template - pET21b+TDMH miniprep (150 ng/µL), primers - TDMH F (gaattcgcggccgcttctagATGATTTCCCTCCGGAAGCC) and TDMH R (ctgcagcggccgctactagtaTCAGTGGTGGTGGTGGTGGT). Program: 98.0 (30 s) - 98.0 (10 s), 62.0 - 58.0 (30 s), 72.0 (30 s) - 72.0 (10 min). It worked, bend size - 700 bp.
• Gibson assembly for the new backbones:  PCR products from the 10th of September were used (the same PCR reaction was done for the biobrick pSB2C3 backbone with all primers, so each PCR product for the Duet backbones was mixed with the corresponding pSB product. Protocol: 5 µL of each product, 10 µL of the Gibson master mix, 15 min on 50.0.
• PCR for RFP - we wanna add biobrick cut sites on the RFP gene from pSB1C3 (Phusion protocol). Primers: gaattcgcggccgcttctagATGGCTTCCTCCGAAGACGT(RFP F), ctgcagcggccgctactagtaGCGATCTACACTAGCACTAT (RFP R).
• PCR to biobrick pET, so we could clone RFP in it and put it under the T7 promoter: template is pET21b+TDMH (Phusion protocol). Primers: tactagtagcggccgctgcagGATCCGGCTGCTAACAAAGC (pET R), ctagaagcggccgcgaattcATGTATATCTCCTTCTTAAA (pET F)
• miniprep for DB3.1 + pColA-pSB3C5 and for DB3.1 + pCDF-pSB3C5. Then we did colony PCR for the minipreps with the corresponding primers (pDCA/pDCA3 and pDCA/pDCA1) to check if the insert from the Duet vectors is inside. The bends were the right size.
• Digestion of TDMH (biobricked) and the linear backbone:

For the backbone: 10 µL of the plasmid, 10 µL of the Green buffer, 3 µL for EcoRI and Pstl, 14 µL of dH2O.

For TDMH: 10 µL of the pcr product, 4 µL of the Green buffer, 1.5 µL for EcoRI and Pstl, 23 µL od dH2O.
• PCR Ligation for biobricking TDMH: dH20 4.5 µL, TDMH 1 µL, linear backbone 3 µL, T4 ligation buffer 1 µL, T4 ligase 0.5 µL; 16.0 fr 1 h. 
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Friday 13<SUP>th</SUP> September</H3>
A lot of macrophages died (in suspension) because we forgot to add glutamine and pyruvate sodium. We repeated the experiment with the complete media (but again without antibiotics).
• Results for the GA: because the Chl plates were off, I have to repeat everything for biobricking TDMH and for the pACYC derived backbone. The cloning for pCDF and pColA worked - patched the colonies, start liquid cultures.
• Launch NEB Turbo to make competent cells.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Saturday 14<SUP>th</SUP> September</H3>
Fresh liquid cultures for: M smegmatis pMyC-GFP (kanamycine resistance) , M smegmatis, E.coli Bl21 carrying pET21b-TDMH and pColA-RFP (ampicilin and kanamycine resistances) and E.coli Bl21 carrying pET21b-TDMH, pColA RFP and pACyC-hly (ampicilin, kanamycine and chloramphenicol resistances)
• CaCl2 competent cells for NEB Turbo - 25 mL of the culture (O.D. 1), 3 washing steps (1o min, 4 000 rpm, on 4.0). For the first one - 100 mM MgCl2, for the second - 100 mM CaCl 2, after the last one the cells were resuspended in CaCl2 + glycerol. I've also made electrocompetent cells (3 washes in dH20).  
• cloning biobricked TDMH in the linear backbone - protocol from the 12th of September. New competent cells and new Chl plates. 
• pET+pSB3C5 worked - miniprep; repeated the Gibson for pACYC
• PCR for confirming pET+pSB3C5 - tamplet is the miniprep and the primers are pDCo4 R/F. pET Duet as the positive controle (Phusion protocol).
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Monday 15<SUP>th</SUP> September</H3>• Liquid culture for TDMH+pSB1C3.
• Colony PCR for confirming TDMH+pSB1C3 with biobrick verification primers. 
• A new setup for testing if E. coli kills M. smegmatis in the macrophages: 2 wells with M. smegmatis, 1 well with M. smeg + E. coli uninduced, 1 well with only E. coli, 1 well with M. smeg + E. coli induced. First we infect the macropheges with the liquid culture of M. smeg GFP - we set the final O.D. of the culture in the well to 0.03 (100 µL of the liquid culture with O.D.  0.3 to 900 µL of RPMI with attached macrophages. After 2 h of incubation in the CO2 incubator we wash the cells with PBS 10 times and we add E. coli induced or uninduced (induced overnight with 10 µL IPTG).
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Monday 16<SUP>th</SUP> September</H3>• I have to repeat the biobricking of TDMH because I didn't use the DpnI and because the linear backbone which I was using didnt have the RFP cassette. So I've repeated the PCR with the right backbone. 
• Digesting the linear backbone: 10 µL plasmid, 10 µL fast digest buffer, 3 µL for EcoRI and Pstl, 1 µL od DpnI, 73 µL of dH20. 1 h, 37.0
• Ligation of the digested linear backbone and TDMH - backbone 3 µL, insert 1 µL, ligation buffer 1 µL, ligation enzyme 0.5 µL, dH2O 4.5 µL.
•A new setup for the macrophages; the protocol from the 15th of September; the groups were: macrophages alone, 2 x macrophages infected with M. smegmatis, 2 x macrophages infected with E. coli, 2 x M. smeg + E. coli uninduced, 2 x M. smeg + E. coli induced overnight, 2 x M. smeg + E. coli induced for 2 h. We also scratched the macrophages from yesterdays' setup and tried to get CFUs for them - 1 mL of the scratch (in PBS)was centrifuged (1 500 rpm, 10 min) to remove any cells which were not phagocytosed. The pallet was resuspended in 1 mL of PBS and the membranes of the macrophages were lysed with 5 µL of TritonX. 100 µL of the suspension was plated on LBA plates.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Tuesday 17<SUP>th</SUP> September</H3>• A new setup (as yesterday); plated the scratches form yesterday. This time we washed our cells after the infection with e. coli.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Wednesday 18<SUP>th</SUP> September</H3>
Made pictures for the yesterdays' setup. The cloning from yesterday didn't work, so I plated the leftovers of the transformation culture.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Friday 20<SUP>th</SUP> September</H3>• The cloning for biobricking TDMH worked, so we have to check if it has the insert. We run a PCR with the TDMH primers and we get a bend of around 700 bp, which is fine. We also did a PCR with the biobrick verification primers and the bend was around 1 kb and it was different then the band for the pSB1C3 (bigger, but the difference is not that obvious). So there is probably something inside. 
• We did a new setup where we also tried to use M. smegmatis which is not GFP and stain it with a fluorescent die for Mycobacteria, but its not working. After a few h the level of contamination was hughe so we are repeating everything tomorrow with adding antibiotics in the media.
</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Tuesday 15<SUP>th</SUP> October</H3>
Start fresh cultures for E.coli bl21 and M.smegmatis-GFP. 
Change the media of the macrophages J774. 

</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Tuesday 16<SUP>th</SUP> October</H3>
We want to investigate the stability of the protein TDMH at different pH.
To do this, I prepared buffers at different pH using monosodium phosphate and disodium phosphate. 
We used a calculator (http://home.fuse.net/clymer/buffers/phos2.html) to have an idea of how much miligrams should we add of each to get the right pH. Buffer strenght was 10mM.
We prepared a range of pH and checked them using a pH meter.
The pH were: 4,54 ; 5,15; 6,43; 7,36; 8,25 and 8,84 
pH lower than 4,5 and higher than 8,8 couldn't be reached (because of the pKa as it is a weak acid).
We wanted to test another pH around 3, so we add HCl to reach pH 3.
The buffer were made with osmosed water and have been autoclaved after the pH measurement.
We also made glycerol stock of E.coli (the one with 3 plasmids carrying TDMH,LLO and RFP genes)from the overnight as described in the protocol.


</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Thursday 17<SUP>th</SUP> October</H3>

To study the stability of TDMH at 7 different pH we extracted it from E.coli using the kit B-PER.
We have 4 groups: 
- M.smegmatis alone
- M.smegmatis + E.coli's control (producing another protein) induced
- M.smegmatis + E.coli's Bl21 (with TDMH gene) uninduced proteins
- M.smegmatis+ E.coli's Bl21 (with TDMH gene) induced proteins
We plate at time 0, 3h and 6 h using selective plates (nalidixic acid 3 micrograms/mL) and making 2 different dilutions
We had 3 wells for each condition.

We plated bacteria on 504 plates


</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Friday 18<SUP>th</SUP> October</H3>

Making fresh cultures to repeat the experiment. 


</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Saturday 19<SUP>th</SUP> October</H3>

Repeating the pH experiment using smaller volume of detergent (3 microliters) because we were afraid that the detergent will affect M.smegmatis in the previous experiment.

</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Tuesday 16<SUP>th</SUP> October</H3>
Counting the colonies of M.smegmatis on the plates from Thursday.
We stopped because we noticed no significant difference between what was tested and the controls.
Maybe the protein wasn't extracted properly or it was extracted and inactive (denaturated)...
Also we noticed a decrease of 10-20% of the number of M.smegmatis colonies growing in all the wells with the detergent (and the proteins) (around 50microliters per well). 
The very low pH (3 and 4,5) affected the growth of M.smegmatis by 20-30% </DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Tuesday 21<SUP>st</SUP> October</H3>
Counting the colonies from the second experiment ==&gt; unsuccessful experiment. </DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Tuesday 16<SUP>th</SUP> October</H3>
Starting new cultures and making glycerol stocks for M.smegmatis-GFP.


</DIV><DIV class="tbnote"><H2>Infiltrate</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" target="_blank" class="tbnotelogo TClogo"> ASDF </A><H3>Tuesday 16<SUP>th</SUP> October</H3>
We repeated the pH study using the plate reader and the OD this time. 
Same controls and conditions, the only difference is the protein extraction.  
We decided that we will just overexpressed E.coli (10mM IPTG as previously) and collect the supernatant. 
We centrifuged 5minutes at 4000 rpm and took the supernatant. Then, the supernatant was filter sterilized 
We add 200 microliters of M.smegmatis (OD 0,05) resuspended in the appropriate buffer and 20 microliters of E.coli (OD 0,5).
To know precisely the initial number of bacteria, we made CFU for each pH (resuspended in a different buffer). 
We launched the plate reader for 24 hours. 

</DIV><DIV class="tbnote"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Infiltrate"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Infiltrate</SPAN></A></LI><LI class="toclevel-2"><A href="#Sunday_27th_October"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Sunday 27<SUP>th</SUP> October</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Infiltrate_2"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Infiltrate</SPAN></A></LI><LI class="toclevel-2"><A href="#Monday_28th_October"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Monday 28<SUP>th</SUP> October</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Infiltrate">Infiltrate</SPAN></H2><P>&lt;a href=&quot;<A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" class="external free" rel="nofollow">https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate</A>&quot; target=&quot;_blank&quot; class=&quot;tbnotelogo TClogo&quot;&gt; ASDF &lt;/a&gt;</P><H3><SPAN class="mw-headline" id="Sunday_27th_October"> Sunday 27<SUP>th</SUP> October</SPAN></H3><P><B><EM>&lt;/br&gt;</EM></B></P><P>launched a plate reader experiment repeating the pH study (design as previously).</P><P><I>M.smegmatis</I> was resuspended in the buffers. The proteins of E.coli were collected by taking the supernatant (centrifugation 4000 rpm, 5 minutes). The supernatant was filtersterilized. </P><P> Preparation of new buffers for the pH 3 and the pH 4,5. Former buffers weren't &quot;strong&quot; enough (pka = 7,2 for this couple). By adding chlorhydric acid we move on the second couple (pKa = 2,12) and have a sronger buffer. </P></DIV><P>&lt;/html&gt;</P><DIV class="tbnote"><H2><SPAN class="mw-headline" id="Infiltrate_2">Infiltrate</SPAN></H2><P>&lt;a href=&quot;<A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" class="external free" rel="nofollow">https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate</A>&quot; target=&quot;_blank&quot; class=&quot;tbnotelogo TClogo&quot;&gt; ASDF &lt;/a&gt;</P><H3><SPAN class="mw-headline" id="Monday_28th_October"> Monday 28<SUP>th</SUP> October</SPAN></H3><P><B><EM>&lt;/br&gt;</EM></B></P><P>Interpreting the plate reader experiment which shows many things (see results) including that pH doesn't affect TDMH action (except a small decrease for basic pH like 8,2)</P><P>Grow the macrophages in a 12 well plate. </P><P>We began the experiment to test the killing inside the macrophage. We will infect them with <I>M.smegmatis</I>-GFP then wash (5 times) and infect them with <I>E.coli</I> RFP producing TDMH (controls wells with E.coli uninduced and also an induced E.coli producing something else) and wash (5 times). Wait 3 hours and collect the bacteria from the macrophage using 2 different techniques: </P><P>- Igepal detergent</P><P>- Cold water for injectable preparations</P><P>&lt;/br&gt;&lt;/br&gt;</P></DIV><P>&lt;/html&gt;</P><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>24<SUP>th</SUP> July</H3><P lang="en-US">
    starting bacterial culture of MG1655-6300 (Chantal’s glycerol)
</P><P lang="en-US">
    -streaking 2 Agar plate with it
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>25<SUP>th</SUP> July</H3><P lang="en-US">
    choosing a nice colony and launching overnight culture 37° in LB
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>28<SUP>th</SUP> July</H3><P lang="en-US">
    sRNA design
</P><P lang="en-US">
    red sequences:
</P><P lang="en-US">
    pACYCDuet-1 (CmR) : ATATCCAGTGATTTTTTTCTCCAT
</P><P lang="en-US">
    pCOLADuet-1 (KanR) : CGTTTCCCGTTGAATATGGCTCAT
</P><P lang="en-US">
    We decided that targeting only two different antibiotic resistance will be enough for a proof of concept. We give up AmpR because otherwise they might be
    experimental issues with AmpR that is also on phagemid litmus28i.
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>2<SUP>nd</SUP> July</H3><P lang="en-US"><EM>Aude and Vincent</EM></P><P lang="en-US"><STRONG>glycerol stock</STRONG></P><P lang="en-US">
    from overnight culture of MG1655-6300 O/N : T001
</P><UL><LI><P lang="en-US">
            Centrifuge 4000rpm, 10 minutes,
        </P></LI><LI><P lang="en-US">
            take out liquid
        </P></LI><LI><P lang="en-US">
            resuspend cells in 1mL glycerol, 2mL LB
        </P></LI><LI><P lang="en-US">
            separate in two cryotubes, one for the -80°C, one for the -20°C
        </P></LI></UL><P lang="en-US"><STRONG>Electroporation</STRONG></P><P lang="en-US">
    - making MG1655-6300 competent (Aude’s protocol for electrocompetent cells &amp; electroporation [ref needed])
</P><P lang="en-US">
    - test of the competent cells (negative
</P><P lang="en-US">
    -transforming with pCOLADuet-1 : Ec2 =&gt;</P><P lang="en-US">
    -transforming with pACYCDuet-1 : Ec2 =&gt;</P><P lang="en-US">
    - plating 3 different quantity of cells (20ul, 50uL, 100uL) respectively on Cm and Kan plates (dilution 1000x for the antibiotics)
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>3<SUP>rd</SUP> July</H3><P lang="en-US"><EM>Aude and Vincent</EM></P><P lang="en-US">
    Transfo results :
</P><UL><LI><P lang="en-US">
            negative controls are negative.
        </P></LI><LI><P lang="en-US">
            Transformed cells grew on plates
        </P></LI></UL><UL><LI><P lang="en-US">
            in the evening launch of 5mL Lb cultures from clones on the plates
        </P></LI></UL><P lang="en-US"><STRONG>Igem Buffer for chemical competent cells</STRONG></P><P lang="en-US">
    Igem protocol for chemical competent cells
</P><P lang="en-US"><STRONG>Launch overnight culture of NEB turbo</STRONG></P><P lang="en-US">
    5mL LB inoculated with a clone from plate of drug team
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>4<SUP>th</SUP> July</H3><P lang="en-US"><EM>Aude</EM></P><UL><LI><P lang="en-US">
            Glycerols of the overnight cultures of the transformation
        </P></LI><LI><P lang="en-US">
            sT002 : MG1655 pCOLADuet-1
        </P></LI><LI><P lang="en-US">
            sT003 : MG1655 pACYCDuet-1
        </P></LI></UL><P lang="en-US"><EM>Aude and Clovis</EM></P><P lang="en-US"><STRONG>Making stocks of electro competent NEB turbo</STRONG></P><P lang="en-US">
    Prepare 10% glycerol solution
</P><P lang="en-US">
    Put on Ice 1L of Sterile water, 10% glycerol solution and 10 falcon tubes
</P><P lang="en-US">
    Cool down centrifuge to 4°C
</P><P lang="en-US">
    Prepare a rack with 50 microtubes at -80°C
</P><UL><LI><P lang="en-US">
            Dilute in 500 mL LB (in a 1000mL flask) 500 ul of NEB turbo pre cultured
        </P></LI><LI><P lang="en-US">
            Check the OD until it reached 0,5
        </P></LI><LI><P lang="en-US">
            Put 500mL cultures in 10 50mL falcon tubes in ice
        </P></LI><LI><P lang="en-US">
            Wait for 20 minutes for the culture to cool down
        </P></LI><LI><P lang="en-US">
            Centrifuge at 3400 rpm 4°C for 10 minutes
        </P></LI><LI><P lang="en-US">
            Take out liquid and resuspend in 50 mL ice cold water (resuspend content of two tubes in one tube)
        </P></LI><LI><P lang="en-US">
            Centrifuge 10 minutes, 4°C , 3400 rpm
        </P></LI><LI><P lang="en-US">
            Take out liquid and resuspend in 25 mL ice cold water
        </P></LI><LI><P lang="en-US">
            Centrifuge 10 minutes, 4°C, 3400 rpm
        </P></LI><LI><P lang="en-US">
            Resuspend all the content of the tubes in 20mL ice cold glycerol
        </P></LI><LI><P lang="en-US">
            Centrifuge 10 minutes, 4°C, 3400rpm
        </P></LI><LI><P lang="en-US">
            Resuspend in 5mL ice cold 10°C glycerol solution
        </P></LI><LI><P lang="en-US">
            Aliquot in 50 tubes (100uL per microtubes)
        </P></LI><LI><P lang="en-US">
            Store at -80°C
        </P></LI></UL></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>15<SUP>th</SUP> July</H3><P lang="en-US"><EM>Clovis and Anne</EM></P><P lang="en-US"><STRONG>Primer Design</STRONG></P><P lang="en-US">
    Design primers to PCR KanR out of Duet and LacZalpha out of pUC18 to then clone them into the DUET containing Chloramphenicol
</P><P lang="en-US">
    Geneious files in the dropbox: LacZ primers, KanR primers
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>16<SUP>th</SUP> July</H3><P lang="en-US"><EM>Anne with Nico, Marguerite and Yonatan</EM></P><P lang="en-US"><STRONG>Transformation of pUC18 (NEB, NEB CC, BL21)</STRONG></P><P lang="en-US">
    1) Thaw competent cells on ice.
</P><P lang="en-US">
    2) Place 20 ul of cells in a pre-chilled Eppendorf tube.
</P><P lang="en-US">
    3) add o.5ul of plasmid to the cells
</P><P lang="en-US">
    4) Mix gently by flicking the tube.
</P><P lang="en-US">
    5) Chill on ice for 10 minutes.
</P><P lang="en-US">
    4) Heat shock at 42 °C for 30 seconds.
</P><P lang="en-US">
    5) Return to ice for 2 minutes.
</P><P lang="en-US">
    6) Add 200 ul LB medium and recover the cells by shaking at 37 °C for 30 min (AmpR)
</P><P lang="en-US">
    7) Plate out the cells (10 ul) on Amp LB plates as well as a control (untransformed cells)
</P><P lang="en-US">
    8. Incubate at 37 °C. Transformants should appear within 12 hrs.
</P><P lang="en-US">
    Result: Colonies (around 10)
</P><P lang="en-US"><EM>Vincent and Aude</EM></P><P lang="en-US"><STRONG>In silico cloning: sRNA gBlocks into pUC18 and litmus28-GFP</STRONG></P><P lang="en-US">
    Successful with both Gibson assembly and regular cloning.
</P><P lang="en-US">
    See file: “in silico cloning trojan horse.geneious”</P><P lang="en-US"><EM>Vincent </EM></P><P lang="en-US">
    streaking Ortiz’s strain ELS-41 and ELS-13 containing Litmus28i_J23115-B0032-GFP and M13K07, respectively.
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>17<SUP>th</SUP> July</H3><P lang="en-US"><EM>Anne</EM></P><P lang="en-US"><STRONG>Redsign of Primers</STRONG></P><P lang="en-US">
    Design primers to PCR KanR out of Duet and LacZalpha out of pUC18 to then clone them into the DUET containing Chloramphenicol
</P><P lang="en-US">
    Geneious files in the dropbox: LacZ primers, KanR primers
</P><P lang="en-US"><EM>Clovis and Anne</EM></P><P lang="en-US"><STRONG>Liquid Cultures (for Miniprep)</STRONG></P><P lang="en-US">
    7ml LB with proper antibiotics
</P><P lang="en-US">
    adding: 1) NEB with pUC18 (AmpR)
</P><P lang="en-US">
    2) sT002 : MG1655 pCOLADuet-1 (KanR)
</P><P lang="en-US">
    3) sT003 : MG1655 pACYCDuet-1 (ChlaR)
</P><P lang="en-US">
    Cultivate over night in the 37°C shaker
</P><P lang="en-US"><EM>Vincent</EM></P><P lang="en-US">
    Check: GFP expressed as expected in ELS-41
</P><P lang="en-US"><STRONG>Lauching O/N</STRONG>
    of ELS-41 and ELS-13 with Amp and Kan, respectively.
</P><P lang="en-US">
    Aude
</P><P lang="en-US"><STRONG>Lauching O/N of MGZ1 </STRONG>
    (Glycerol from lab upstairs) with Spec (500X)
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>18<SUP>th</SUP> July</H3><P lang="en-US"><EM>Vincent and Aude</EM></P><P lang="en-US"><STRONG>Glycerol stock</STRONG>
    of ELS-41 and ELS-13 and MGZ1 now renamed sT004, sT005, sT006 respectively:
</P><P lang="en-US">
    sT004: RP437, F+ Litmus28i_J23115-B0032-GFP AmpR
</P><P lang="en-US">
    sT005: XL1-Blue, M13K07 KanR
</P><P lang="en-US"><STRONG>Making electrocompetent cells</STRONG>
    from strain sT004, sT005 and MGZ1 =&gt; Bad old water =&gt; could not obtain competent cells.
</P><P lang="en-US"><STRONG>Conjugation :</STRONG></P><P lang="en-US">
    sT004: RP437, F+ Litmus28i_J23115-B0032-GFP + conjugate with MGZ1.
</P><P lang="en-US">
    F+ comes from XL1 Blue (Ortiz paper) =&gt; F+ is tetR
</P><P lang="en-US">
    Protocol :
</P><OL><LI><P lang="en-US">
            From O/N cultures Dilute both strains 1/100 , in LB
        </P></LI><LI><P lang="en-US">
            Wait for OD to reach O,2
        </P></LI><LI><P lang="en-US">
            Prepare 4 tubes (in BD tubes) :
        </P></LI></OL><UL><LI><P lang="en-US">
            Tube 1 = 0,5mL LB with Strain 1 (Here MGZ1) + O,5mL LB = control
        </P></LI><LI><P lang="en-US">
            Tube 2 = 0,5mL LB with Strain 2 (here sT004) + 0,5mL LB = control
        </P></LI><LI><P lang="en-US">
            Tube 3 and 4= 0,5mL LB with Strain 1 (Here MGZ1) + 0,5mL LB with Strain 2 (here sT004
        </P></LI></UL><OL start="4"><LI><P lang="en-US">
            Incubate 2 hours at 37°C
        </P></LI><LI><P lang="en-US">
            Plate 100ul for controls, 100uL, 50ul, 20ul for mixed tubes on LB antiobiotics (here tet and spec)
        </P></LI><LI><P lang="en-US">
            Incubate overnight at 37°C
        </P></LI></OL><P lang="en-US"><EM>Aude</EM></P><P lang="en-US">
    Miniprep pT005 (Litmus28i_J23115-B0032-GFP) pT006 (M13K07) (<STRONG>Thermo Scientific Miniprep Kit)</STRONG></P><P lang="en-US">
    pT005 : 280 ng /uL
</P><P lang="en-US">
    pT006 : 61 ng/ul
</P><P lang="en-US"><EM>Aude Vincent Sebastian and Yonatan</EM></P><P lang="en-US">
    Work on the model
</P><P lang="en-US">
    Define the players, the processes in place (more in Modeling file)
</P><P lang="en-US"><EM>Anne and Clovis</EM></P><P lang="en-US"><STRONG>Miniprep (sT002, sT003, NEB+pUC18) using the Thermo Scientific Miniprep Kit</STRONG></P><P lang="en-US">
    1) Pellet 5ml of liquid culture (max speed, 1 min)
</P><P lang="en-US">
    2) Discard supernatant
</P><P lang="en-US">
    3) resuspend the cells in 250ul of resuspension olution
</P><P lang="en-US">
    4) add 250ul of lysis solution, mix by inverting 4-6 times
</P><P lang="en-US">
    5) add 350ul of neutralization solution
</P><P lang="en-US">
    4) centrifuge for 5 min
</P><P lang="en-US">
    5) transfer supernatant to spin column
</P><P lang="en-US">
    6) centrifuge for 1 min
</P><P lang="en-US">
    7) discard flow through
</P><P lang="en-US">
    8) add 500 ul wash solution and centrifuge for 1 min , discard flow through(repeat this step)
</P><P lang="en-US">
    9) centrifuge for 1 min to remove left over liquid
</P><P lang="en-US">
    10) transfer the column on a 1.5ml tube
</P><P lang="en-US">
    11) add 50ul of elution buffer and incubate for 2 min
</P><P lang="en-US">
    12) centrifuge for 2 min
</P><P lang="en-US">
    13) Nanodrop the concentration and freeze at -20°</P><P lang="en-US">
    pT.002: 53.9ng/ul -&gt; redo with more liquid culture, it is only a low copy plasmid
</P><P lang="en-US">
    pT.003: 53.8ng/ul -&gt; redo with more liquid culture, it is only a low copy plasmid
</P><P lang="en-US">
    pT.007: 214ng/ul
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>19<SUP>th</SUP> July</H3><DL><DT><B>Conjugation results :</B></DT><DD>Controls =&gt; no colonies</DD><DD>Mix = <I>tapis</I></DD></DL>
Ie worked really well.
To be sure to start only with one clone =&gt; restreak some colonies on tet/spec plates<DL><DT><B>Transformation by electroporation : goal = get a<I> phage factory</I></B>.</DT><DT>We do it three different ways in order to see which one works (best). </DT><DD>Transform strain containing helper plasmid with phagemid</DD><DD>Transform strain containing helper plasmid with phagemid</DD><DD>Co Transform strain with both plasmids at the same time</DD></DL><DL><DT><B>From strain sT004, sT005, sT006</B></DT><DD>Transform sT004  with pT006</DD><DD>Transform sT005 with pT005</DD><DD>Co Transform sT006 with pT005 and pT006</DD><B>Tubes :</B><OL><LI>sM13KO7 (control) : 5,60
</LI><LI>sM13KO7 + pLitmus : 5,70
</LI><LI>sM13KO7 + pLitmus : 5,70
</LI><LI>sLITMUS (control) : 5,60
</LI><LI>sLITMUS + pM13K07 : 5,60
</LI><LI>sLITMUS + pM13K07 : 5,60
</LI><LI>MG16165 : control = 5.70
</LI><LI>MG16165 + p (M13K07) and p(LITMUS) = 5.60
</LI><LI>MG16165 + p (M13K07) and p(LITMUS)= 5.60
</LI></OL>
Recovery 2h
Plate everything on Kan/ Amp<DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>20<SUP>th</SUP> July</H3><B>Results of transformation</B><UL><LI>Controls : no colonies
</LI><LI>Colonies on plates : every transformation worked
</LI></UL>
Put plates of conjugation and transformation in the fridge.</DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>22<SUP>th</SUP> July</H3><P><B><EM>
Place your twit here
</EM></B></P>
Check Colonies of transformation under iLumina tools: colonies are fluorescent.
Launch culture.</DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>23<SUP>th</SUP> July</H3><B>Infectiveness characterization experiments</B><UL><LI>The night before: prepare O/N of F+ cells, prepare O/N of phagemid producing cells (in this case : sT007 )
</LI><LI>Centrifugate phagemid producing cells
</LI><LI>Filter the supernatant 0.45um, stock it at 4°C (it contains the phages)
</LI><LI>Dilute 200x MGZ1, F+  from O/N
</LI><LI>Wait until OD600 = 0.7
</LI><LI>Immediately mix MGZ1 and the supernatant in different proportions
</LI></UL>
Here we tried 1/1 (vol supernatant/vol cells), 1/10 and 1/100
<UL>Immediately dilute each of them to 10-3, 10-4, 10-5, 10-6
<LI>Plate 100uL of those dilutions on LB-agar, AMP, KAN, AMP+KAN
</LI><LI>Incubate O/N
</LI><LI>Count colonies
</LI><LI>Interpretation:
Colonies on AMP: infected by litmus28i
Colonies on KAN: infected by M13K07
Colonies on AMP+KAN: coinfected by litmus28i and M13K07 phagemid producing cells</LI></UL><P>Results</P>
Conclusions: 
<UL><LI>Cells that were mixed with 1/1 supernatant didn't survive the excess of antibiotics that were present in the supernatant.
</LI><LI>Even at low ratio like 1/100 the infection rate of litmus28i-GFP is about 100%. GFP detection confirmed this result: almost 100% of cells were fluorescent.
</LI><LI>Co-infection did occur. It would be interesting to test if those colonies are indeed producing phages.
</LI></UL>
Start O/N culture of M13K07
Start O/N culture of MGZ1,F+
</DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>24<SUP>th</SUP> July</H3>
Glycerol stock:  
<UL><LI>sT008: MGZ1, F+
</LI><LI>sT009: XL1-Blue Litmus28i
</LI></UL><B>Infectiveness characterization experiments</B><UL><LI>Dilute 1/100 MGZ1, F+ O/N culture in 5mL LB =&gt; wait OD(600) = 0.7
</LI><LI>Centrifuge M13K07 4000rpm, 10 minutes
</LI><LI>Retrieve surnageant and filter it with 0.2 filter
</LI></UL><UL><LI>Once OD 0.7 is reached, make 4 tubes
	</LI><LI> 1mL MGZ1,F+ + 100ul LB (not done, not enough cells left ofter OD test)
	</LI><LI> 1mL MGZ1,F+ + 100ul surnageant diluted 1/100
	</LI><LI> 1mL MGZ1,F+ + 100ul surnageant diluted 1/1000
	</LI><LI> 1mL MGZ1,F+ + 100ul surnageant diluted 1/1000
	</LI><LI>Incube 37°C for 45 minutes
</LI><LI>Serial dilute the tubes 10^-1, 10^-2, 10^-3, 10^-4, 10^-5, 10^-6
	</LI><LI>4 times 6 tubes with 900uL LB, add 100ul of one previously made tube, mix and transfert 100ul to next tube.
	</LI></UL><LI>Plate on LB (10^-6), Kan(10^-2 and 10^-3), Amp(10^-6), Kan and Amp(10^-2 and 10^-3)

Experiment fail due to manipulation error (wrong dilution of the supernageant and plating of wrong dilutions)

Start O/N culture of MGZ1,F+ (T008) + Tet 1/500
Start O/N culture of M13K07 (T007) + Amp and Kan 1/1000</LI></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>25<SUP>th</SUP> July</H3>
O/N culture of T007 didn't grow (100 ul instead of 10ul of antibiotics, Kan and Amp)
Start O/N culture of MGZ1,F+ (T008) + Tet 1/500
Start O/N culture of M13K07 (T007) + Amp and Kan 1/1000</DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>26<SUP>th</SUP> July</H3><B>Infectiveness characterization experiments</B><UL><LI>Dilute 1/100 MGZ1, F+ O/N culture in 10mL LB =&gt; wait OD(600) = 0.7
</LI><LI>Centrifuge M13K07 4000rpm, 10 minutes
</LI><LI>Retrieve surnageant and filter it with 0.2 filter
</LI></UL><UL><LI>Once OD 0.7 is reached, make 4 tubes
	</LI><LI> 1mL MGZ1,F+ + 100ul LB
	</LI><LI> 1mL MGZ1,F+ + 100ul surnageant diluted 1/100
	</LI><LI> 1mL MGZ1,F+ + 100ul surnageant diluted 1/1000
	</LI><LI> 1mL MGZ1,F+ + 100ul surnageant diluted 1/1000
	</LI><LI>Incube 37°C for 45 minutes
</LI><LI>Serial dilute the tubes 10^-1, 10^-2, 10^-3, 10^-4, 10^-5, 10^-6
	</LI><LI>4 times 6 tubes with 900uL LB, add 100ul of one previously made tube, mix and transfert 100ul to next tube.
	</LI></UL><LI>Plate on LB (10^-6), Kan(10^-2), Amp(10^-6), Kan and Amp(10^-3)
</LI></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>Tuesday 30<SUP>th</SUP> July</H3>
Transformed the bl21 (DE3) strain with the pET21b (plated the transformation culture on 2 amp plates).
</DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>31<SUP>th</SUP> July</H3><P lang="en-US"><EM>Vincent and Aude</EM></P><P lang="en-US"><STRONG>Conjugation MGZ1, F+ &amp; MG1655:</STRONG></P><P lang="en-US">
    Protocol :
</P><OL><LI><P lang="en-US">
            From O/N cultures Dilute both strains 1/100 , in LB
        </P></LI><LI><P lang="en-US">
            Wait for OD to reach O,2
        </P></LI><LI><P lang="en-US">
            Prepare 4 tubes (in BD tubes) :
        </P></LI></OL><UL><LI><P lang="en-US">
            Tube 1 = 0,5mL LB with Strain 1 (Here MGZ1) + O,5mL LB = control
        </P></LI><LI><P lang="en-US">
            Tube 2 = 0,5mL LB with Strain 2 (here sT004) + 0,5mL LB = control
        </P></LI><LI><P lang="en-US">
            Tube 3 and 4= 0,5mL LB with Strain 1 (Here MGZ1) + 0,5mL LB with Strain 2 (here sT004
        </P></LI></UL><OL start="4"><LI><P lang="en-US">
            Incubate 2 hours at 37°C
        </P></LI><LI><P lang="en-US">
            Plate 50ul for controls, 50uL, 20ul, 10ul for mixed tubes on LB antiobiotics (here tet and spec)
        </P></LI><LI><P lang="en-US">
            Incubate overnight at 37°C
        </P></LI></OL><P lang="en-US"><EM>Vincent</EM></P><P lang="en-US"><STRONG>Infectiveness characterization experiments</STRONG></P><P lang="en-US">
    -the night before: prepare O/N of F+ cells, prepare O/N of phagemid producing cells (in this case : sT007 )
</P><P lang="en-US">
    -centrifugate phagemid producing cells
</P><P lang="en-US">
    -filter the supernatant 0.45um, stock it at 4°C (it contains the phages)
</P><P lang="en-US">
    -dilute 200x MGZ1, F+ from O/N
</P><P lang="en-US">
    -wait until OD600 = 0.7
</P><P lang="en-US">
    -immediately mix MGZ1 and the supernatant in different proportions
</P><P lang="en-US">
    here we planned to try 1/100 (vol supernatant/vol cells), 1/1000 and 1/10000
</P><P lang="en-US">
    Unfortunately the supernatant was dropped on the bench.
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>1<SUP>st</SUP> August</H3><P lang="en-US"><EM>Aude &amp; Vincent</EM></P><P lang="en-US"><STRONG>Cloning Kanamycin resistance gene into pACYCDuet-1:</STRONG></P><P lang="en-US">
    -PCR of Kanamycin resistance gene from pCOLADuet-1 with primer iT.0003 and iT.0004
</P><P lang="en-US">
    -electrophoresis of PCR product: 1 band between 900 and 1000bp: coherent with KanR length
</P><P lang="en-US">
    -cutting KanR bands:
</P><P lang="en-US">
    -digestion with ECOR1 and Xho1:
</P><P lang="en-US">
    PCR product digest for cloning
</P><TABLE width="232" cellpadding="1" cellspacing="0"><TBODY><TR><TD width="158"><P lang="en-US"><STRONG>Reagent</STRONG></P></TD><TD width="68"><P lang="en-US"><STRONG>Volume</STRONG></P></TD></TR><TR><TD width="158"><P lang="en-US">
                    Purified PCR Product
                </P></TD><TD width="68"><P lang="en-US">
                    16 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    H<SUB>2</SUB>O
                </P></TD><TD width="68"><P lang="en-US">
                    0 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    10x FastDigest Buffer
                </P></TD><TD width="68"><P lang="en-US">
                    2 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    FastDigest Enzyme 1
                </P></TD><TD width="68"><P lang="en-US">
                    1 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    FastDigest Enzyme 2
                </P></TD><TD width="68"><P lang="en-US">
                    1 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US"><STRONG>Total Volume</STRONG></P></TD><TD width="68"><P lang="en-US">
                    20.0 ul
                </P></TD></TR></TBODY></TABLE><P lang="en-US">
    Digest for 0.5 - 1 hours at 37 °C with shaking.
</P><P lang="en-US">
    -digestion of pACYCDuet-1 with ECOR1 and Xho1:
</P><P lang="en-US">
    Vector backbone digest for cloning</P><TABLE width="232" cellpadding="1" cellspacing="0"><TBODY><TR><TD width="158"><P lang="en-US"><STRONG>Reagent</STRONG></P></TD><TD width="68"><P lang="en-US"><STRONG>Volume</STRONG></P></TD></TR><TR><TD width="158"><P lang="en-US">
                    Purified Plasmid
                </P></TD><TD width="68"><P lang="en-US">
                    20 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    H<SUB>2</SUB>O
                </P></TD><TD width="68"><P lang="en-US">
                    65 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    10x FastDigest Buffer
                </P></TD><TD width="68"><P lang="en-US">
                    10 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    FastDigest Enzyme 1
                </P></TD><TD width="68"><P lang="en-US">
                    2.5 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    FastDigest Enzyme 2
                </P></TD><TD width="68"><P lang="en-US">
                    2.5 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    FastAP Phosphatase
                </P></TD><TD width="68"><P lang="en-US">
                    0 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US"><STRONG>Total Volume</STRONG></P></TD><TD width="68"><P lang="en-US">
                    100.0 ul
                </P></TD></TR></TBODY></TABLE><P lang="en-US">
    Digest for 2 - 4 hours at 37 °C with shaking.
</P><P lang="en-US">
    -electrophoresis of pACYCDuet-1 digestion products:
</P><P lang="en-US">
    column 1: 1kb plus marker
</P><P lang="en-US">
    column 2 &amp; 3: digestion product
</P><P lang="en-US">
    column 4: undigested pACYCDuet-1
</P><P lang="en-US">
    nothing appear on gel after digestion, therefore we will miniprep pACYCDuet-1 and try again to digest it.
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>12<SUP>th</SUP> August</H3><P lang="en-US">
    Clovis
</P><P lang="en-US">
    Start 2 cultures of MG1655 pACYCDuet-1 strain (ST003), 5mL each for further miniprep
</P><P lang="en-US">
    Phusion PCR of lacZalpha out of pUC18 (pT007)
</P><P lang="en-US">
    with primers :
</P><P lang="en-US">
    iT.0001 (Fw, added NcoI cut site) CCATGGCTGGCTTAACTATGCGG
</P><P lang="en-US">
    iT.0002 (Rv, added BglII cutsite) AGATCTGTGAGTTAGCTCACTCATTAGG
</P><P lang="en-US">
    Column 1; 100kb plus ladder
</P><P lang="en-US">
    Column 2,3,4,5,6; PCR product
</P><P lang="en-US">
    Gel purification of column 5 and 6 → 19,2 ng/ul
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>13<SUP>th</SUP> August</H3><P lang="en-US">
    Clovis
</P><P lang="en-US">
    Digestion of the PCR product (LacZalpha) with NcoI and BglII
</P><P lang="en-US">
    PCR purification using Thermo Scintific Kit of the digestion → 6,2 ng/ul
</P><P lang="en-US">
    Miniprep of pACYADuet-1 (pT002) using Thermo Scientific Kit → 132,7 ng/ul
</P><P lang="en-US">
    Start O/N sT005 and sT010
</P><P lang="en-US"><EM>Vincent</EM></P><P lang="en-US"><STRONG>PCR of pUC18 (pT007) with iT0007 &amp; iT0008 / PCR of litmus28i-GFP (pT005) with iT0005 &amp; iT0006 </STRONG></P><P lang="en-US">
    PCR Gradient of 8 tubes +-5°C around 54°C for both PCRs (pUC18 and Litmus18i-GFP)
</P><P lang="en-US">
    Failure, no bands appear on the gel.
</P><P lang="en-US"><STRONG>Infectiveness characterization experiments</STRONG></P><P lang="en-US">
    -the night before: prepare O/N of F+ cells, prepare O/N of phagemid producing cells (in this case : sT007 )
</P><P lang="en-US">
    -centrifugate phagemid producing cells
</P><P lang="en-US">
    -filter the supernatant 0.45um, stock it at 4°C (it contains the phages)
</P><P lang="en-US">
    -dilute 200x MGZ1, F+ from O/N
</P><P lang="en-US">
    -wait until OD600 = 0.7
</P><P lang="en-US">
    -immediately mix MGZ1 and the supernatant in different proportions
</P><P lang="en-US">
    here we planned to try 1/100 (vol supernatant/vol cells), 1/1000 and 1/10000
</P><OL><LI><P lang="en-US">
            1mL MGZ1,F+ + 100ul LB
        </P></LI><LI><P lang="en-US">
            1mL MGZ1,F+ + 100ul surnageant diluted 1/100
        </P></LI><LI><P lang="en-US">
            1mL MGZ1,F+ + 100ul surnageant diluted 1/1000
        </P></LI><LI><P lang="en-US">
            1mL MGZ1,F+ + 100ul surnageant diluted 1/1000
        </P></LI></OL><P lang="en-US">
    -incubate 45 minutes at 37°C for the protein to be expressed.
</P><UL><LI><P lang="en-US">
            Serial dilute the tubes 10^-1, 10^-2, 10^-3, 10^-4, 10^-5, 10^-6
        </P></LI><LI><P lang="en-US">
                    4 times 6 tubes with 900uL LB, add 100ul of one previously made tube, mix and transfert 100ul to next tube.
                </P></LI><LI><P lang="en-US">
            Plate on LB (10^-6), Kan( 10^-3), Amp(10^-5 - 10^-6), Kan and Amp(10^-2 - 10^-3)
        </P></LI></UL></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>14<SUP>th</SUP> August</H3><P lang="en-US"><STRONG>PCR of pUC18 (pT007) with iT0007 &amp; iT0008 / PCR of litmus28i-GFP (pT005) with iT0005 &amp; iT0006 (Aude):</STRONG></P><P lang="en-US"><STRONG>Gel Purif (Aude)</STRONG></P><P lang="en-US">
    Using QiaGen Kit
</P><P lang="en-US">
    3 tubes (combined the band)
</P><P lang="en-US">
    1 : 13,2 ng/uL
</P><P lang="en-US">
    2 : 14 ng/uL
</P><P lang="en-US">
    3 : 2O ng/uL
</P><P lang="en-US"><STRONG>Digestions (</STRONG><EM><STRONG>Aude</STRONG></EM><STRONG>)</STRONG></P><P lang="en-US">
    Digestion pACY par Xho EcoRI (pour le clonage de Kan dans ACY)
</P><TABLE width="232" cellpadding="1" cellspacing="0"><TBODY><TR><TD width="158"><P lang="en-US"><STRONG>Reagent</STRONG></P></TD><TD width="68"><P lang="en-US"><STRONG>Volume</STRONG></P></TD></TR><TR><TD width="158"><P lang="en-US">
                    Purified Plasmid
                </P></TD><TD width="68"><P lang="en-US">
                    10 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    H<SUB>2</SUB>O
                </P></TD><TD width="68"><P lang="en-US">
                    75 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    10x FastDigest Green Buffer
                </P></TD><TD width="68"><P lang="en-US">
                    10 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    EcoRI
                </P></TD><TD width="68"><P lang="en-US">
                    2.5 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    XhoI
                </P></TD><TD width="68"><P lang="en-US">
                    2.5 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    FastAP Phosphatase
                </P></TD><TD width="68"><P lang="en-US">
                    0 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US"><STRONG>Total Volume</STRONG></P></TD><TD width="68"><P lang="en-US">
                    100.0 ul
                </P></TD></TR></TBODY></TABLE><P lang="en-US">
    Digestion pACY par NcoI BgIII (pour le clonage de lacZalpha dans ACY)
</P><TABLE width="232" cellpadding="1" cellspacing="0"><TBODY><TR><TD width="158"><P lang="en-US"><STRONG>Reagent</STRONG></P></TD><TD width="68"><P lang="en-US"><STRONG>Volume</STRONG></P></TD></TR><TR><TD width="158"><P lang="en-US">
                    Purified Plasmid
                </P></TD><TD width="68"><P lang="en-US">
                    15 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    H<SUB>2</SUB>O
                </P></TD><TD width="68"><P lang="en-US">
                    75 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    10x FastDigest Green Buffer
                </P></TD><TD width="68"><P lang="en-US">
                    10 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    EcoRI
                </P></TD><TD width="68"><P lang="en-US">
                    2.5 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    XhoI
                </P></TD><TD width="68"><P lang="en-US">
                    2.5 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    FastAP Phosphatase
                </P></TD><TD width="68"><P lang="en-US">
                    0 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US"><STRONG>Total Volume</STRONG></P></TD><TD width="68"><P lang="en-US">
                    100.0 ul
                </P></TD></TR></TBODY></TABLE><P lang="en-US">
    =&gt; 2h30 à 37°C
</P><UL><LI><P lang="en-US">
            Run on a a gel.
        </P></LI></UL><UL><LI><P lang="en-US">
            only see a smear, Everything was digested ,
        </P></LI><LI><P lang="en-US">
            redigest but just 25 minutes as said on the thermoscientific protoc.
        </P></LI></UL><P lang="en-US"><STRONG>Digestions (</STRONG><EM><STRONG>CLovis</STRONG></EM><STRONG>)</STRONG></P><UL><LI><P lang="en-US">
            Digestion pACY par Xho EcoRI (pour le clonage de Kan dans ACY)
        </P></LI></UL><TABLE width="232" cellpadding="1" cellspacing="0"><TBODY><TR><TD width="158"><P lang="en-US"><STRONG>Reagent</STRONG></P></TD><TD width="68"><P lang="en-US"><STRONG>Volume</STRONG></P></TD></TR><TR><TD width="158"><P lang="en-US">
                    Purified Plasmid
                </P></TD><TD width="68"><P lang="en-US">
                    10 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    H<SUB>2</SUB>O
                </P></TD><TD width="68"><P lang="en-US">
                    75 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    10x FastDigest Green Buffer
                </P></TD><TD width="68"><P lang="en-US">
                    10 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    EcoRI
                </P></TD><TD width="68"><P lang="en-US">
                    2.5 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    XhoI
                </P></TD><TD width="68"><P lang="en-US">
                    2.5 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    FastAP Phosphatase
                </P></TD><TD width="68"><P lang="en-US">
                    0 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US"><STRONG>Total Volume</STRONG></P></TD><TD width="68"><P lang="en-US">
                    100.0 ul
                </P></TD></TR></TBODY></TABLE><UL><LI><P lang="en-US">
            Digestion pACY par NcoI BgIII (pour le clonage de lacZalpha dans ACY)
        </P></LI></UL><TABLE width="232" cellpadding="1" cellspacing="0"><TBODY><TR><TD width="158"><P lang="en-US"><STRONG>Reagent</STRONG></P></TD><TD width="68"><P lang="en-US"><STRONG>Volume</STRONG></P></TD></TR><TR><TD width="158"><P lang="en-US">
                    Purified Plasmid
                </P></TD><TD width="68"><P lang="en-US">
                    15 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    H<SUB>2</SUB>O
                </P></TD><TD width="68"><P lang="en-US">
                    75 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    10x FastDigest Green Buffer
                </P></TD><TD width="68"><P lang="en-US">
                    10 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    EcoRI
                </P></TD><TD width="68"><P lang="en-US">
                    2.5 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    XhoI
                </P></TD><TD width="68"><P lang="en-US">
                    2.5 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    FastAP Phosphatase
                </P></TD><TD width="68"><P lang="en-US">
                    0 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US"><STRONG>Total Volume</STRONG></P></TD><TD width="68"><P lang="en-US">
                    100.0 ul
                </P></TD></TR></TBODY></TABLE><P lang="en-US"><STRONG>25 minutes at 37°C</STRONG></P><P lang="en-US">
    It worked!
</P><P lang="en-US"><STRONG>Gel purification (Clovis)</STRONG></P><P lang="en-US">
    ACY Nco Dig : 6,3 ng/ul
</P><P lang="en-US">
    ACY Xho Dig : 7,8 ng/uL
</P><P lang="en-US">
    PCR Gradient of 8 tubes +10°C around 54°C for both PCRs (Litmus18i-GFP) but start not from the miniprep ut from a fresh culture of litmus GFp
</P><P lang="en-US">
    Only see the primer =&gt; try again with new miniprep
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>15<SUP>th</SUP> August</H3><P lang="en-US"><STRONG>Miniprep (Clovis)</STRONG></P><P lang="en-US">
    pACY (pT002) : 109ng/uL
</P><P lang="en-US">
    litmus (pT005) : 610 ng/Ul
</P><P lang="en-US"><STRONG>Ligation (Aude, Clovis ,Vincent)</STRONG></P><P lang="en-US">
    masse insert = ration (svt 10) * (taille insert / taille vecteur) * masse vecteur (50-100ng)
</P><P lang="en-US">
    CHosen Vector mass = 50 ng
</P><P lang="en-US"><U>lacZ : </U></P><P lang="en-US">
    Digestion vector : 6,3 ng/ul
</P><P lang="en-US">
    Digestion insert : 6,2
</P><P lang="en-US">
    masse insert = 10*449/3770*50 = 59 ng
</P><P lang="en-US">
    Protocol
</P><P lang="en-US">
    Vector : 8 uL
</P><P lang="en-US">
    Insert : 10 uL
</P><P lang="en-US">
    Buffer : 2 uL
</P><P lang="en-US">
    T4 DNA ligase : 0,2 uL
</P><P lang="en-US">
    Let incubate at 22°C for 30 min
</P><P lang="en-US"><STRONG>Kan : </STRONG></P><P lang="en-US">
    Digestion vector : 7,8 ng/ul
</P><P lang="en-US">
    Digestion insert : 12ng/ul
</P><P lang="en-US">
    Insert mass = 10*1052/3770*50 = 140 ng
</P><P lang="en-US">
    Protocole Ligation
</P><P lang="en-US">
    Vector : 7 uL
</P><P lang="en-US">
    Insert : 11 uL
</P><P lang="en-US">
    Buffer : 2 uL
</P><P lang="en-US">
    T4 DNA ligase : 0,2 uL
</P><P lang="en-US"><STRONG>Transformation : (in chemical competent NEB turbo (made on the …) </STRONG><EM>Aude Clovis</EM></P><P lang="en-US">
    1) Thaw competent cells on ice.
</P><P lang="en-US">
    These can be prepared using the CaCl2 protocol.
</P><P lang="en-US">
    2) Add 5 ul of ligation product
</P><P lang="en-US">
    4) Mix gently by flicking the tube.
</P><P lang="en-US">
    5) Chill on ice for 30 minutes.
</P><P lang="en-US">
    4) Heat shock at 42 °C for 30 seconds.
</P><P lang="en-US">
    5) Return to ice for 2 minutes.
</P><P lang="en-US">
    6) Add 300 ul LB medium and recover the cells by shaking at 37 °C for one hour
</P><P lang="en-US">
    7) Plate on selective media
</P><P lang="en-US">
    Plate on :
</P><P lang="en-US">
    Tube 1 : negative control Xgal /IPTG /Cm ; Kan/Cm
</P><P lang="en-US">
    Tube 2 : ligation lacZ-pacy (5uL) Xgal /IPTG /Cm
</P><P lang="en-US">
    Tube 3 : ligation Kan-pACY (5uL) Kan Cm
</P><P lang="en-US">
    Tube 4 : native plasmid (pACY) (1uL) Xgal /IPTG /Cm ; Kan/Cm
</P><P lang="en-US"><STRONG>PCR (litmus = Backbone) Vincent</STRONG></P><P lang="en-US">
    8 tubes, gradient 10°C around 64°C , started from a new miniprep
</P><P lang="en-US">
    We see the plasmid =&gt; too concentrated
</P><P lang="en-US"><STRONG>PCR (litmus pour la gibbson) Clovis</STRONG></P><P lang="en-US">
    sTOO5 primer 005 006 protocol phusion (gradient around 64 + or – 5 °C)
</P><P lang="en-US">
    Elongaiton time 2min10. Started with the new miniprep diluted.
</P><P lang="en-US"><STRONG>Infectiveness characterization experiments (Vincent)</STRONG></P><P lang="en-US">
    -the night before: prepare O/N of F+ cells, prepare O/N of phagemid producing cells (in this case : sT007 )
</P><P lang="en-US">
    After checking GFP fluorescence of sT007 and its negative result, we decided to make a new sT007 glycerol stock. We used a colony from the first successful
    Infectiveness characterization experiments that was both AMP+KAN resistant and expressed GFP.
</P><P lang="en-US">
    The following experiment was conducted with this new sT007 (strain MGZ1).
</P><P lang="en-US">
    -centrifugate phagemid producing cells
</P><P lang="en-US">
    -filter the supernatant 0.45um, stock it at 4°C (it contains the phages)
</P><P lang="en-US">
    -This experiment was conducted both on MGZ1(F+) and MG1655(F+)
</P><P lang="en-US">
    -dilute 200x MGZ1 and MG1655 from O/N
</P><P lang="en-US">
    -wait until OD600 = 0.7
</P><P lang="en-US">
    -immediately mix MGZ1 and the supernatant in different proportions
</P><P lang="en-US">
    here we planned to try 1/10 (vol supernatant/vol cells)
</P><OL><LI><P lang="en-US">
            1mL MGZ1,F+ + 100ul LB
        </P></LI><LI><P lang="en-US">1mL MGZ1,F+ + 100ul surnageant diluted 1/100
        </P></LI></OL><P lang="en-US">
    -incubate 45 minutes at 37°C for the protein to be expressed.
</P><UL><LI><P lang="en-US">
            Serial dilute the tubes 10^-1, 10^-2, 10^-3, 10^-4, 10^-5, 10^-6
        </P></LI><LI><P lang="en-US">
                    6 tubes with 900uL LB, add 100ul of one previously made tube, mix and transfert 100ul to next tube.
                </P></LI></UL><P lang="en-US">
    Plate on LB (10^-4 10^-5 10^-6), Kan(10^-1 10^-2 10^-3), Amp(10^-4 10^-5 - 10^-6), Kan and Amp(10^-1 10^-2 - 10^-3)
    </P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>16<SUP>th</SUP> August</H3><P lang="en-US"><STRONG>Results from Infectiveness characterization experiments (Vincent)</STRONG></P><P lang="en-US"> See file “phage infectivness 15_08.xlsx”</P><P lang="en-US"><STRONG>Results</STRONG></P><P lang="en-US"><EM>Transformations : (Aude)</EM></P><P lang="en-US">
    Lac : “tapis” on all the plates even the negative control =&gt; plates were not done correctly, antibio added to early =&gt; do another
    transformation
</P><P lang="en-US">
    Kan : Some colonies on the plates , but also colonies on the negative control =&gt; launch few colonies on liquid media (with Kan, Cm)
</P><P lang="en-US"><EM>Phage infectiveness : (Vincent)</EM></P><P lang="en-US"><STRONG>Run PCR</STRONG>
    (Litmus-Gibbson) (Aude)
</P><P lang="en-US">
    Small point around 3000-3500bp
</P><P lang="en-US">
    Run another gel with large holes to check if it is correct and purify =&gt; Not correct, nothing on the new gel
</P><P lang="en-US"><STRONG>Transformation : (in chemical competent NEB turbo )</STRONG><EM>Aude Clovis</EM></P><P lang="en-US">
    Rest of ligation product
</P><P lang="en-US">
    1) Thaw competent cells on ice.
</P><P lang="en-US">
    These can be prepared using the CaCl2 protocol.
</P><P lang="en-US">
    2) Add 5 ul of ligation product
</P><P lang="en-US">
    4) Mix gently by flicking the tube.
</P><P lang="en-US">
    5) Chill on ice for 30 minutes.
</P><P lang="en-US">
    4) Heat shock at 42 °C for 30 seconds.
</P><P lang="en-US">
    5) Return to ice for 2 minutes.
</P><P lang="en-US">
    6) Add 300 ul LB medium and recover the cells by shaking at 37 °C for one hour
</P><P lang="en-US">
    7) Plate on selective media
</P><P lang="en-US">
    Plate on :
</P><P lang="en-US">
    Tube 1 : negative control Xgal /IPTG /Cm ; Kan/Cm
</P><P lang="en-US">
    Tube 2 : ligation lacZ-pacy (5uL) Xgal /IPTG /Cm
</P><P lang="en-US">
    Tube 3 : ligation Kan-pACY (5uL) Kan Cm
</P><P lang="en-US">
    Tube 4 : native plasmid (pACY) (1uL) Xgal /IPTG /Cm ; Kan/Cm
</P><P lang="en-US"><STRONG>Gel from PCR litmus (Clovis)</STRONG></P><P lang="en-US">
    1kB ladder / triple puit (3*50uL) from column 2,3,4 of PCR
</P><P lang="en-US">
    50V
</P><P lang="en-US"><STRONG>Infectiveness characterization experiments (Aude)</STRONG></P><P lang="en-US">
    -the night before: prepare O/N of F+ cells, prepare O/N of phagemid producing cells (in this case : sT007 )
</P><P lang="en-US">
    -centrifugate phagemid producing cells
</P><P lang="en-US">
    -filter the supernatant 0.45um, stock it at 4°C (it contains the phages)
</P><P lang="en-US">
    -dilute 200x MG, F+ from O/N
</P><P lang="en-US">
    -wait until OD600 = 0.7
</P><P lang="en-US">
    -immediately mix MG and the supernatant in different proportions
</P><P lang="en-US">
    here we planned to try 1/100 (vol supernatant/vol cells), 1/1000 and 1/10000
</P><OL><LI><P lang="en-US">
            1mL MG,F+ + 100ul LB
        </P></LI><LI><P lang="en-US">
            1mL MG,F+ + 100ul surnageant diluted 1/10
        </P></LI><LI><P lang="en-US">
            1mL MG,F+ + 100ul surnageant diluted 1/100
        </P></LI><LI><P lang="en-US">
            1mL MG,F+ + 100ul surnageant diluted 1/1000
        </P></LI><LI><P lang="en-US">
            1mL MG,F+ + 100ul surnageant diluted 1/10000
        </P></LI></OL><P lang="en-US">
    -incubate 45 minutes at 37°C for the protein to be expressed.
</P><UL><LI><P lang="en-US">
            Serial dilute the tubes 10^-1, 10^-2, 10^-3, 10^-4, 10^-5
        </P></LI><LI><P lang="en-US">
                    4 times 5 tubes with 900uL LB, add 100ul of one previously made tube, mix and transfert 100ul to next tube.
                </P></LI></UL><P lang="en-US">
    Plate on LB (10^-5), Kan( 10^-2), Amp(10^-5), Kan and Amp(10^-2)
    </P><P lang="en-US"><STRONG>Ligation (Vincent)</STRONG></P><P lang="en-US">
    masse insert = ration (svt 10) * (taille insert / taille vecteur) * masse vecteur (50-100ng)
</P><P lang="en-US">
    CHosen Vector mass = 50 ng
</P><P lang="en-US"><U>lacZ : </U></P><P lang="en-US">
    Digestion vector : 6,3 ng/ul
</P><P lang="en-US">
    Digestion insert : 6,2
</P><P lang="en-US">
    masse insert = 10*449/3770*50 = 59 ng
</P><P lang="en-US">
    Protocol
</P><P lang="en-US">
    Vector : 8 uL
</P><P lang="en-US">
    Insert : 10 uL
</P><P lang="en-US">
    Buffer : 2 uL
</P><P lang="en-US">
    T4 DNA ligase : 0,2 uL
</P><P lang="en-US">
    Let incubate at 22°C for 30 min
</P><P lang="en-US"><STRONG>Kan : </STRONG></P><P lang="en-US">
    Digestion vector : 7,8 ng/ul
</P><P lang="en-US">
    Digestion insert : 12ng/ul
</P><P lang="en-US">
    Insert mass = 10*1052/3770*50 = 140 ng
</P><P lang="en-US">
    Protocole Ligation
</P><P lang="en-US">
    Vector : 7 uL
</P><P lang="en-US">
    Insert : 11 uL
</P><P lang="en-US">
    Buffer : 2 uL
</P><P lang="en-US">
    T4 DNA ligase : 0,2 uL
</P><P lang="en-US"><STRONG>Transformation : (in chemical competent NEB turbo (made on the …) </STRONG><EM>vincent</EM></P><P lang="en-US">
    1) Thaw competent cells on ice.
</P><P lang="en-US">
    These can be prepared using the CaCl2 protocol.
</P><P lang="en-US">
    2) Add 5 ul of ligation product
</P><P lang="en-US">
    4) Mix gently by flicking the tube.
</P><P lang="en-US">
    5) Chill on ice for 30 minutes.
</P><P lang="en-US">
    4) Heat shock at 42 °C for 30 seconds.
</P><P lang="en-US">
    5) Return to ice for 2 minutes.
</P><P lang="en-US">
    6) Add 300 ul LB medium and recover the cells by shaking at 37 °C for one hour
</P><P lang="en-US">
    7) Plate on selective media
</P><P lang="en-US">
    Plate on :
</P><P lang="en-US">
    Tube 1 : negative control Xgal /IPTG /Cm ; Kan/Cm
</P><P lang="en-US">
    Tube 2 : ligation lacZ-pacy (10uL) Xgal /IPTG /Cm
</P><P lang="en-US">
    Tube 3 : ligation Kan-pACY (10uL) Kan Cm
</P><P lang="en-US">
    Tube 4 : native plasmid (pACY) (1uL) Xgal /IPTG /Cm ; Kan/Cm
</P><P lang="en-US">
    PCR pUC18 (pT007)
</P><P lang="en-US">
    5*50uL
</P><P lang="en-US">
    34 H2O
</P><P lang="en-US">
    10 Buffer 5x
</P><P lang="en-US">
    1 dNTP
</P><P lang="en-US">
    2.5 FW iT007
</P><P lang="en-US">
    2.5 RV iT008
</P><P lang="en-US">
    0.2 pUC18
</P><P lang="en-US">
    0.5 phusion
</P><P lang="en-US">
    50uL Tot
</P><P lang="en-US">
    Tm = 55°C
</P><P lang="en-US">
    te = 2min10
</P><P lang="en-US">
    no bands 
</P><P lang="en-US">
    PCR Litmus (Clovis):
</P><P lang="en-US">
    Trouble shooting : since the temperature does not help, that it was not the mini prer = but that we see homodimere between the primers =&gt; try with less
    primers
</P><P lang="en-US">
    PCR litmus (pT005)
</P><P lang="en-US">
    5*50ul
</P><P lang="en-US">
    40 H2O
</P><P lang="en-US">
    10 Buffer 5x
</P><P lang="en-US">
    1 dNTP
</P><P lang="en-US">
    0.5 FW iT005
</P><P lang="en-US">
    0.5 RV iT006
</P><P lang="en-US">
    0.2 litmus
</P><P lang="en-US">
    0.5 phusion
</P><P lang="en-US">
    50ul Tot
</P><P lang="en-US">
    Tm = 64°C
</P><P lang="en-US">
    te = 2min10
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>17<SUP>th</SUP> August</H3><P lang="en-US"><STRONG>Results</STRONG></P><UL><LI><P lang="en-US">
            Transformation worked but no clone with the ligation product =&gt;restart from the beginning, problem must be with one of the digested products.
        </P></LI><LI><P lang="en-US">
            Phage infectiveness : Experiment worked =&gt; results here
        </P></LI></UL><P lang="en-US"><STRONG>Repcr inserts / redigest vector</STRONG></P><P lang="en-US">
    PCR KanR out of pCOLA (pT003)
</P><P lang="en-US">
    5*50ul
</P><P lang="en-US">
    34 H2O
</P><P lang="en-US">
    10 Buffer 5x
</P><P lang="en-US">
    1 dNTP
</P><P lang="en-US">
    2.5 FW iT003
</P><P lang="en-US">
    2.5 RV iT004
</P><P lang="en-US">
    0.2 pCOLA
</P><P lang="en-US">
    0.5 phusion
</P><P lang="en-US">
    50ul Tot
</P><P lang="en-US">
    Tm = 54°C
</P><P lang="en-US">
    te = 45sec
</P><P lang="en-US">
    PCR LacZ out of pUC18 (pT007)
</P><P lang="en-US">
    5*50ul
</P><P lang="en-US">
    34 H2O
</P><P lang="en-US">
    10 Buffer 5x
</P><P lang="en-US">
    1 dNTP
</P><P lang="en-US">
    2.5 FW iT001
</P><P lang="en-US">
    2.5 RV iT002
</P><P lang="en-US">
    0.2 pUC18
</P><P lang="en-US">
    0.5 phusion
</P><P lang="en-US">
    50ul Tot
</P><P lang="en-US">
    Tm = 54°C
</P><P lang="en-US">
    te = 45sec
</P><P lang="en-US">
    Gel of PCR Kan and LacZ
</P><P lang="en-US">
    5 puits LacZ / 100bP+ / 5 puits Kan
</P><P lang="en-US"><STRONG>Preparaiton of the vectors to clone the sRNA</STRONG></P><P lang="en-US">
    PCR litmus (pT005)
</P><P lang="en-US">
    5*50ul
</P><P lang="en-US">
    40 H2O
</P><P lang="en-US">
    10 Buffer 5x
</P><P lang="en-US">
    1 dNTP
</P><P lang="en-US">
    0.5 FW iT003
</P><P lang="en-US">
    0.5 RV iT004
</P><P lang="en-US">
    0.2 pCOLA (pT007)
</P><P lang="en-US">
    0.5 phusion
</P><P lang="en-US">
    50ul Tot
</P><P lang="en-US">
    Tm = 64°C
</P><P lang="en-US">
    te = 2min10
</P><P lang="en-US"><STRONG>Results</STRONG></P><UL><LI><P lang="en-US">
            Transformation worked but no clone with the ligation product =&gt;restart from the beginning, problem must be with one of the digested products.
        </P></LI><LI><P lang="en-US">
            Phage infectiveness : Experiment worked =&gt; results here
        </P></LI></UL><P lang="en-US"><STRONG>Repcr inserts / redigest vector</STRONG></P><P lang="en-US">
    PCR KanR out of pCOLA (pT003)
</P><P lang="en-US">
    5*50ul
</P><P lang="en-US">
    34 H2O
</P><P lang="en-US">
    10 Buffer 5x
</P><P lang="en-US">
    1 dNTP
</P><P lang="en-US">
    2.5 FW iT003
</P><P lang="en-US">
    2.5 RV iT004
</P><P lang="en-US">
    0.2 pCOLA
</P><P lang="en-US">
    0.5 phusion
</P><P lang="en-US">
    50ul Tot
</P><P lang="en-US">
    Tm = 54°C
</P><P lang="en-US">
    te = 45sec
</P><P lang="en-US">
    PCR LacZ out of pUC18 (pT007)
</P><P lang="en-US">
    5*50ul
</P><P lang="en-US">
    34 H2O
</P><P lang="en-US">
    10 Buffer 5x
</P><P lang="en-US">
    1 dNTP
</P><P lang="en-US">
    2.5 FW iT001
</P><P lang="en-US">
    2.5 RV iT002
</P><P lang="en-US">
    0.2 pUC18
</P><P lang="en-US">
    0.5 phusion
</P><P lang="en-US">
    50ul Tot
</P><P lang="en-US">
    Tm = 54°C
</P><P lang="en-US">
    te = 45sec
</P><P lang="en-US">
    Gel of PCR Kan and LacZ
</P><P lang="en-US">
    5 puits LacZ / 100bP+ / 5 puits Kan
</P><P lang="en-US">
    --&gt;img 622&lt;--
</P><P lang="en-US"><STRONG>Preparaiton of the vectors to clone the sRNA</STRONG></P><P lang="en-US">
    PCR litmus (pT005)
</P><P lang="en-US">
    5*50ul
</P><P lang="en-US">
    40 H2O
</P><P lang="en-US">
    10 Buffer 5x
</P><P lang="en-US">
    1 dNTP
</P><P lang="en-US">
    0.5 FW iT003
</P><P lang="en-US">
    0.5 RV iT004
</P><P lang="en-US">
    0.2 pCOLA (pT007)
</P><P lang="en-US">
    0.5 phusion
</P><P lang="en-US">
    50ul Tot
</P><P lang="en-US">
    Tm = 64°C
</P><P lang="en-US">
    te = 2min10
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>19<SUP>th</SUP> August</H3><P lang="en-US"><STRONG>CLONING the target Plasmids (Kan and lacZ)</STRONG></P><P lang="en-US"><EM>Gel purification of LacZ using Thermo Scientific Kit</EM></P><P lang="en-US">
    lacZ : 83 ng/ul
</P><P lang="en-US"><EM>Gel purification of Kan using Thermo Scientific Kit</EM></P><P lang="en-US">
    Kan : 80ng/ul
</P><P lang="en-US"><EM>Digestion of pACYC (pT002) with EcoRI and XhoI</EM></P><P lang="en-US">
    10 H2O
</P><P lang="en-US">
    2 Buffer
</P><P lang="en-US">
    4 pACYC
</P><P lang="en-US">
    2 EcoRI
</P><P lang="en-US">
    2 XhoI
</P><P lang="en-US">
    20uL Tot
</P><P lang="en-US">
    37°C
</P><P lang="en-US">
    20min
</P><P lang="en-US"><EM>Digestion of pACYC (pT002) with BglII and NcoI</EM></P><P lang="en-US">
    10 H2O
</P><P lang="en-US">
    2 Buffer
</P><P lang="en-US">
    4 pACYC
</P><P lang="en-US">
    2 BglII
</P><P lang="en-US">
    2 NcoI
</P><P lang="en-US">
    20uL Tot
</P><P lang="en-US">
    37°C
</P><P lang="en-US">
    20min
</P><P lang="en-US"><EM>Digestion of Kan insert with EcorI and XhoI</EM></P><P lang="en-US">
    10 H2O
</P><P lang="en-US">
    15 Kan insert
</P><P lang="en-US">
    3 Buffer
</P><P lang="en-US">
    1 EcoRI
</P><P lang="en-US">
    1 XhoI
</P><P lang="en-US">
    30uL Tot
</P><P lang="en-US">
    37°C
</P><P lang="en-US">
    20min
</P><P lang="en-US"><EM>Digestion of LacZ insert with BglII and NcoI</EM></P><P lang="en-US">
    10 H2O
</P><P lang="en-US">
    15 LacZ insert
</P><P lang="en-US">
    3 Buffer
</P><P lang="en-US">
    1 BglII
</P><P lang="en-US">
    1 NcoI
</P><P lang="en-US">
    30uL Tot
</P><P lang="en-US">
    37°C
</P><P lang="en-US">
    20min
</P><P lang="en-US"><EM>Gel of digestions</EM></P><P lang="en-US">
    100Bp+ / pACYC+EcoRI+XhoI / pACYC+EcoRI+XhoI / pACYC+BglII+NcoI /pACYC+BglII+NcoI
</P><P lang="en-US"><EM>Colum purification of Kan insert digestions using Thermo Scientific Kit</EM></P><P lang="en-US"><EM>Column purification of LacZ insert digestions using Thermo Scientific Kit</EM></P><P lang="en-US"><EM>Colum purification of pACYC digestions using Thermo Scientific Kit</EM></P><P lang="en-US"><EM>Heat inactivation of Kan digestion</EM></P><P lang="en-US">
    65°C
</P><P lang="en-US">
    20min
</P><P lang="en-US"><EM>Ligation Kan insert purif + pACYC (digestion EcoRI + XhoI)</EM></P><P lang="en-US">
    Digestion vector : 6.8 ng/ul
</P><P lang="en-US">
    Digestion insert : 14
</P><P lang="en-US">
    masse insert = 10*1052/3770*50 = 140 ng
</P><P lang="en-US">
    Vector : 7.35 uL
</P><P lang="en-US">
    Insert : 10 uL
</P><P lang="en-US">
    Buffer : 2 uL
</P><P lang="en-US">
    T4 DNA ligase : 0,2 uL
</P><P lang="en-US">
    Let incubate at 22°C for 30 min
</P><P lang="en-US"><EM>Ligation Kan insert heat desactivated + pACYC (digestion EcoRI + XhoI)</EM></P><P lang="en-US">
    Digestion vector : 6.8 ng/ul
</P><P lang="en-US">
    Digestion insert : 40/2 = 20
</P><P lang="en-US">
    masse insert = 10*1052/3770*50 = 140 ng
</P><P lang="en-US">
    Vector : 7.35 uL
</P><P lang="en-US">
    Insert : 7 uL
</P><P lang="en-US">
    Buffer : 2 uL
</P><P lang="en-US">
    T4 DNA ligase : 0,2 uL
</P><P lang="en-US">
    H2O : 4 uL
</P><P lang="en-US">
    Let incubate at 22°C for 30 min
</P><P lang="en-US"><EM>Ligation LacZ insert purif + pACYC (pigestion BglII + NcoI)</EM></P><P lang="en-US">
    Digestion vector : 10? ng/ul
</P><P lang="en-US">
    Digestion insert : 33
</P><P lang="en-US">
    masse insert = 10*449/3770*50 = 59 ng
</P><P lang="en-US">
    Vector : 5 uL
</P><P lang="en-US">
    Insert : 1.8 uL
</P><P lang="en-US">
    Buffer : 2 uL
</P><P lang="en-US">
    T4 DNA ligase : 0,2 uL
</P><P lang="en-US">
    H2O : 11 uL
</P><P lang="en-US">
    Let incubate at 22°C for 30 min
</P><P lang="en-US"><EM>Chemical transformation in NEB turbo</EM></P><P lang="en-US">
    Reference protocol
</P><P lang="en-US">
    Recovery 1 hour.
</P><P lang="en-US">
    Plate on
</P><P lang="en-US">
    Tube 1 = negative control
</P><P lang="en-US">
    Tube 2 : Kan (purif) Kan / Cm
</P><P lang="en-US">
    Tube 3 : Kan (heat inactivated) Kan/ Cm
</P><P lang="en-US">
    Tube 4 : lac(purify, enzyme cannot be heat inactivated) Cm/ IPTG/ Xgal
</P><P lang="en-US"><STRONG>Cloning the Weapon (ie sRNA into Litmus and pUC 18)</STRONG></P><P lang="en-US"><EM>Gel of litmus PCR</EM></P><P lang="en-US"><EM>Gel purification of litmus PCR using Thermo Scientific Kit</EM></P><P lang="en-US"><EM>PCR from mini Gene sRNA anti Kan</EM></P><P lang="en-US">
    35 H2O
</P><P lang="en-US">
    10 Buffer 5x
</P><P lang="en-US">
    1 dNTP
</P><P lang="en-US">
    2 FW iT009
</P><P lang="en-US">
    2 RV iT0010
</P><P lang="en-US">
    0.2 minigene-plasmid amp from IDT
</P><P lang="en-US">
    0.5 phusion
</P><P lang="en-US">
    PCR Phusion Gradient around 53 °C
</P><P lang="en-US">
    Time : 30 secondes
</P><P lang="en-US"><EM><STRONG>Gibson Assembly</STRONG></EM></P><P lang="en-US">
    Protocol from the Kit, quantities are divided by 2: final volume of reaction = 10uL
</P><P lang="en-US">
    Assembly of pUC18-SRNA_KAN : 5uL MasterMix, 0.5uL H2O, 3uL pUC18 with overhang from PCR (nanodrop = 13,4), 1,5uL sRNA from PCR of minigene (nanodrop =
    26,7)
</P><P lang="en-US">
    Assembly of Litmus28i-GFP-SRNA_KAN : 5uL MasterMix, 3,6uL H2O, 0,5uL Litmus28i-GFP with overhang from PCR (nanodrop = 85), 0,9uL sRNA from PCR of minigene
    (nanodrop = 26,7)
</P><P lang="en-US">
    Heat shock Transformation into provided competent cells. 60 min recovery, plate on AMP.
</P><P lang="en-US"><EM><STRONG>Regular cloning using biobricks</STRONG></EM></P><P lang="en-US"><EM>Digestion</EM></P><P lang="en-US">
    Insert sRNA cut by XbaI &amp; PstI
</P><P lang="en-US">
    Vector Litmus cut by PstI &amp; SpeI
</P><P lang="en-US">
    Insert sRNA cut by EcoRI &amp; SpeI
</P><P lang="en-US">
    Vector Litmus cut by EcoRI &amp; SpeI
</P><P lang="en-US"><EM>Digestion of Vectors</EM></P><P lang="en-US">
    10 H2O
</P><P lang="en-US">
    2 Buffer
</P><P lang="en-US">
    4 Vector
</P><P lang="en-US">
    2 Enz 1
</P><P lang="en-US">
    2 Enz 2
</P><P lang="en-US">
    20uL Tot
</P><P lang="en-US">
    37°C
</P><P lang="en-US">
    20min
</P><P lang="en-US"><EM>Digestion of Inserts</EM></P><P lang="en-US">
    10 H2O
</P><P lang="en-US">
    15 insert
</P><P lang="en-US">
    3 Buffer
</P><P lang="en-US">
    1 Enz 1
</P><P lang="en-US">
    1 Enz 2
</P><P lang="en-US">
    30uL Tot
</P><P lang="en-US">
    37°C
</P><P lang="en-US">
    20min
</P><P lang="en-US">
    Heat inactivation before ligation
</P><P lang="en-US">
    80°C for 20 minutes
</P><P lang="en-US"><EM>Ligation</EM></P><P lang="en-US">
    pUC= 40ng in 20 uL =&gt; 3ng/ul
</P><P lang="en-US">
    litmus 340ng =&gt;17ng /ul
</P><P lang="en-US">
    insert 420 ng =&gt; 14 ng / uL
</P><P lang="en-US"><U>Litmus : </U></P><P lang="en-US">
    7uL vector
</P><P lang="en-US">
    8 ul insert
</P><P lang="en-US">
    2 uL Buffer
</P><P lang="en-US">
    O,2 uLT4 DNA ligase
</P><P lang="en-US">
    3ul Water
</P><P lang="en-US">
    Vtot = 20 uL
</P><P lang="en-US">
    pUC
</P><P lang="en-US">
    20 uL vector
</P><P lang="en-US">
    9 uL insert
</P><P lang="en-US">
    4 uL Buffer
</P><P lang="en-US">
    0,4 T4 DNA
</P><P lang="en-US">
    7 uL water
</P><P lang="en-US">
    Vtot = 40uL
</P><P lang="en-US"><EM>Chemical transformation in NEB turbo</EM></P><P lang="en-US">
    Tube 1 : negative control
</P><P lang="en-US">
    Tube 2 lig litmus
</P><P lang="en-US">
    Tube 3 lig pUC
</P><UL><LI><P lang="en-US">
            plate everything on amp
        </P></LI></UL></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>20<SUP>th</SUP> August</H3><P lang="en-US"><STRONG>CLONING the target Plasmids (Kan and lacZ)</STRONG></P><P lang="en-US"><EM>Gel purification of LacZ using Thermo Scientific Kit</EM></P><P lang="en-US">
    lacZ : 83 ng/ul
</P><P lang="en-US"><EM>Gel purification of Kan using Thermo Scientific Kit</EM></P><P lang="en-US">
    Kan : 80ng/ul
</P><P lang="en-US"><EM>Digestion of pACYC (pT002) with EcoRI and XhoI</EM></P><P lang="en-US">
    10 H2O
</P><P lang="en-US">
    2 Buffer
</P><P lang="en-US">
    4 pACYC
</P><P lang="en-US">
    2 EcoRI
</P><P lang="en-US">
    2 XhoI
</P><P lang="en-US">
    20uL Tot
</P><P lang="en-US">
    37°C
</P><P lang="en-US">
    20min
</P><P lang="en-US"><EM>Digestion of pACYC (pT002) with BglII and NcoI</EM></P><P lang="en-US">
    10 H2O
</P><P lang="en-US">
    2 Buffer
</P><P lang="en-US">
    4 pACYC
</P><P lang="en-US">
    2 BglII
</P><P lang="en-US">
    2 NcoI
</P><P lang="en-US">
    20uL Tot
</P><P lang="en-US">
    37°C
</P><P lang="en-US">
    20min
</P><P lang="en-US"><EM>Digestion of Kan insert with EcorI and XhoI</EM></P><P lang="en-US">
    10 H2O
</P><P lang="en-US">
    15 Kan insert
</P><P lang="en-US">
    3 Buffer
</P><P lang="en-US">
    1 EcoRI
</P><P lang="en-US">
    1 XhoI
</P><P lang="en-US">
    30uL Tot
</P><P lang="en-US">
    37°C
</P><P lang="en-US">
    20min
</P><P lang="en-US"><EM>Digestion of LacZ insert with BglII and NcoI</EM></P><P lang="en-US">
    10 H2O
</P><P lang="en-US">
    15 LacZ insert
</P><P lang="en-US">
    3 Buffer
</P><P lang="en-US">
    1 BglII
</P><P lang="en-US">
    1 NcoI
</P><P lang="en-US">
    30uL Tot
</P><P lang="en-US">
    37°C
</P><P lang="en-US">
    20min
</P><P lang="en-US"><EM>Gel of digestions</EM></P><P lang="en-US">
    100Bp+ / pACYC+EcoRI+XhoI / pACYC+EcoRI+XhoI / pACYC+BglII+NcoI /pACYC+BglII+NcoI
</P><P lang="en-US"><EM>Colum purification of Kan insert digestions using Thermo Scientific Kit</EM></P><P lang="en-US"><EM>Column purification of LacZ insert digestions using Thermo Scientific Kit</EM></P><P lang="en-US"><EM>Colum purification of pACYC digestions using Thermo Scientific Kit</EM></P><P lang="en-US"><EM>Heat inactivation of Kan digestion</EM></P><P lang="en-US">
    65°C
</P><P lang="en-US">
    20min
</P><P lang="en-US"><EM>Ligation Kan insert purif + pACYC (digestion EcoRI + XhoI)</EM></P><P lang="en-US">
    Digestion vector : 6.8 ng/ul
</P><P lang="en-US">
    Digestion insert : 14
</P><P lang="en-US">
    masse insert = 10*1052/3770*50 = 140 ng
</P><P lang="en-US">
    Vector : 7.35 uL
</P><P lang="en-US">
    Insert : 10 uL
</P><P lang="en-US">
    Buffer : 2 uL
</P><P lang="en-US">
    T4 DNA ligase : 0,2 uL
</P><P lang="en-US">
    Let incubate at 22°C for 30 min
</P><P lang="en-US"><EM>Ligation Kan insert heat desactivated + pACYC (digestion EcoRI + XhoI)</EM></P><P lang="en-US">
    Digestion vector : 6.8 ng/ul
</P><P lang="en-US">
    Digestion insert : 40/2 = 20
</P><P lang="en-US">
    masse insert = 10*1052/3770*50 = 140 ng
</P><P lang="en-US">
    Vector : 7.35 uL
</P><P lang="en-US">
    Insert : 7 uL
</P><P lang="en-US">
    Buffer : 2 uL
</P><P lang="en-US">
    T4 DNA ligase : 0,2 uL
</P><P lang="en-US">
    H2O : 4 uL
</P><P lang="en-US">
    Let incubate at 22°C for 30 min
</P><P lang="en-US"><EM>Ligation LacZ insert purif + pACYC (pigestion BglII + NcoI)</EM></P><P lang="en-US">
    Digestion vector : 10? ng/ul
</P><P lang="en-US">
    Digestion insert : 33
</P><P lang="en-US">
    masse insert = 10*449/3770*50 = 59 ng
</P><P lang="en-US">
    Vector : 5 uL
</P><P lang="en-US">
    Insert : 1.8 uL
</P><P lang="en-US">
    Buffer : 2 uL
</P><P lang="en-US">
    T4 DNA ligase : 0,2 uL
</P><P lang="en-US">
    H2O : 11 uL
</P><P lang="en-US">
    Let incubate at 22°C for 30 min
</P><P lang="en-US"><EM>Chemical transformation in NEB turbo</EM></P><P lang="en-US">
    Reference protocol
</P><P lang="en-US">
    Recovery 1 hour.
</P><P lang="en-US">
    Plate on
</P><P lang="en-US">
    Tube 1 = negative control
</P><P lang="en-US">
    Tube 2 : Kan (purif) Kan / Cm
</P><P lang="en-US">
    Tube 3 : Kan (heat inactivated) Kan/ Cm
</P><P lang="en-US">
    Tube 4 : lac(purify, enzyme cannot be heat inactivated) Cm/ IPTG/ Xgal
</P><P lang="en-US"><STRONG>Cloning the Weapon (ie sRNA into Litmus and pUC 18)</STRONG></P><P lang="en-US"><EM>Gel of litmus PCR</EM></P><P lang="en-US"><EM>Gel purification of litmus PCR using Thermo Scientific Kit</EM></P><P lang="en-US"><EM>PCR from mini Gene sRNA anti Kan</EM></P><P lang="en-US">
    35 H2O
</P><P lang="en-US">
    10 Buffer 5x
</P><P lang="en-US">
    1 dNTP
</P><P lang="en-US">
    2 FW iT009
</P><P lang="en-US">
    2 RV iT0010
</P><P lang="en-US">
    0.2 minigene-plasmid amp from IDT
</P><P lang="en-US">
    0.5 phusion
</P><P lang="en-US">
    PCR Phusion Gradient around 53 °C
</P><P lang="en-US">
    Time : 30 secondes
</P><P lang="en-US"><EM><STRONG>Gibson Assembly</STRONG></EM></P><P lang="en-US">
    Protocol from the Kit, quantities are divided by 2: final volume of reaction = 10uL
</P><P lang="en-US">
    Assembly of pUC18-SRNA_KAN : 5uL MasterMix, 0.5uL H2O, 3uL pUC18 with overhang from PCR (nanodrop = 13,4), 1,5uL sRNA from PCR of minigene (nanodrop =
    26,7)
</P><P lang="en-US">
    Assembly of Litmus28i-GFP-SRNA_KAN : 5uL MasterMix, 3,6uL H2O, 0,5uL Litmus28i-GFP with overhang from PCR (nanodrop = 85), 0,9uL sRNA from PCR of minigene
    (nanodrop = 26,7)
</P><P lang="en-US">
    Heat shock Transformation into provided competent cells. 60 min recovery, plate on AMP.
</P><P lang="en-US"><EM><STRONG>Regular cloning using biobricks</STRONG></EM></P><P lang="en-US"><EM>Digestion</EM></P><P lang="en-US">
    Insert sRNA cut by XbaI &amp; PstI
</P><P lang="en-US">
    Vector Litmus cut by PstI &amp; SpeI
</P><P lang="en-US">
    Insert sRNA cut by EcoRI &amp; SpeI
</P><P lang="en-US">
    Vector Litmus cut by EcoRI &amp; SpeI
</P><P lang="en-US"><EM>Digestion of Vectors</EM></P><P lang="en-US">
    10 H2O
</P><P lang="en-US">
    2 Buffer
</P><P lang="en-US">
    4 Vector
</P><P lang="en-US">
    2 Enz 1
</P><P lang="en-US">
    2 Enz 2
</P><P lang="en-US">
    20uL Tot
</P><P lang="en-US">
    37°C
</P><P lang="en-US">
    20min
</P><P lang="en-US"><EM>Digestion of Inserts</EM></P><P lang="en-US">
    10 H2O
</P><P lang="en-US">
    15 insert
</P><P lang="en-US">
    3 Buffer
</P><P lang="en-US">
    1 Enz 1
</P><P lang="en-US">
    1 Enz 2
</P><P lang="en-US">
    30uL Tot
</P><P lang="en-US">
    37°C
</P><P lang="en-US">
    20min
</P><P lang="en-US">
    Heat inactivation before ligation
</P><P lang="en-US">
    80°C for 20 minutes
</P><P lang="en-US"><EM>Ligation</EM></P><P lang="en-US">
    pUC= 40ng in 20 uL =&gt; 3ng/ul
</P><P lang="en-US">
    litmus 340ng =&gt;17ng /ul
</P><P lang="en-US">
    insert 420 ng =&gt; 14 ng / uL
</P><P lang="en-US"><U>Litmus : </U></P><P lang="en-US">
    7uL vector
</P><P lang="en-US">
    8 ul insert
</P><P lang="en-US">
    2 uL Buffer
</P><P lang="en-US">
    O,2 uLT4 DNA ligase
</P><P lang="en-US">
    3ul Water
</P><P lang="en-US">
    Vtot = 20 uL
</P><P lang="en-US">
    pUC
</P><P lang="en-US">
    20 uL vector
</P><P lang="en-US">
    9 uL insert
</P><P lang="en-US">
    4 uL Buffer
</P><P lang="en-US">
    0,4 T4 DNA
</P><P lang="en-US">
    7 uL water
</P><P lang="en-US">
    Vtot = 40uL
</P><P lang="en-US"><EM>Chemical transformation in NEB turbo</EM></P><P lang="en-US">
    Tube 1 : negative control
</P><P lang="en-US">
    Tube 2 lig litmus
</P><P lang="en-US">
    Tube 3 lig pUC
</P><UL><LI><P lang="en-US">
            plate everything on amp
        </P></LI></UL></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>21<SUP>st</SUP> August</H3><P><B><EM>
Place your twit here
</EM></B></P><P lang="en-US"><STRONG>Results of assemblies</STRONG>
    (Colonies on the transformation plates):
</P><P lang="en-US">
    sRNA –Kan into Litmus by Gibson: Success
</P><P lang="en-US">
    sRNA – Kan into Litmus by biobrick assembly: Success
</P><P lang="en-US">
    sRNA – Kan into pUC18 by Gibson: Failure
</P><P lang="en-US">
    sRNA – Kan into pUC18 by biobrick assembly: Success
</P><P lang="en-US">
    Colony PCR to check if sRNA-KAN is in our Litmus phagemid: positive band between 400 and 500: (expected length 419bp)
</P><P lang="en-US">
    Check the design of primers to clone out Kan =&gt; problem =&gt; redesign them
</P><P lang="en-US">
    Design of the primers to sequence all the cloning
</P><P lang="en-US">
    Launch 3 colonies of each successful cloning
</P><P lang="en-US"><STRONG>Glycerol and mini prep of all the launched colonies / need 3 before we have</STRONG></P><P lang="en-US"><STRONG>Construction of target strain</STRONG></P><P lang="en-US">
    Electroporation of pCOLA in MGZ1 F+
</P><P lang="en-US"><STRONG>Construction of weapon strains</STRONG></P><P lang="en-US">
    Electroporation of litmus-GFP-sRNA in XL1 Blue-M13KO7-Helper
</P><P lang="en-US">
    Electroporation of Litmus-GFP-sRNA and M13K07 in MGZ1 F+
</P><P lang="en-US"><STRONG>Chemical transformation of minigene+vector-sRNA anti Kan in NEB turbo</STRONG></P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>22<SUP>nd</SUP> August</H3><P lang="en-US"><STRONG>Results : </STRONG></P><P lang="en-US">
    All the transformation worked
</P><P lang="en-US">
    Launch cultures (to do experiment)
</P><UL><LI><P lang="en-US">
            mGZ1 F+ -Cola
        </P></LI><LI><P lang="en-US">
            MGZ1 F+ litmus + Helper
        </P></LI></UL><P lang="en-US">
    Launch overnigh culture to do glycerols.
</P><P lang="en-US"><STRONG>exploring the parameters in play in the phage infectiveness of our new phagemid construct </STRONG></P><P lang="en-US">
    -diluting 100x O/N of MGZ1, F+
</P><P lang="en-US">
    -mix with 1/10 vol of supernatant containing phagemids with the cells with the following combination of parameters:
</P><P lang="en-US">
    OD600 : 0.3 0.6 0.9
</P><P lang="en-US">
    Plate at different time (min) after first contact with the supernatant: 2, 30, 60
</P><P lang="en-US"><STRONG>Silencing experiments</STRONG></P><P lang="en-US">
    - Launch O/N culture of producing phages cells ; centrifuge ; take supernatant and filter it.
</P><P lang="en-US">
    -diluting 100x O/N of MGZ1, F+ -pCola in LB-Kan
</P><P lang="en-US">
    - At OD 0,7 : 4 tubes : Ctrl, P1, P2 ,P3. Put 1 mL of cells in each tube. Centrifuge. Take out supernatant, resuspend in 1mL of LB. Add in tubes
    respectively : nothing, 100uL phages of Miniprep 1, Miniprep 2, miniprep 3.
</P><P lang="en-US">
    - Incubate 45 minutes at 37°C
</P><P lang="en-US">
    - Serial Dilute every tube until 10-5. Plate for each tube 100ul of 10-5 dilutions on Kan and LB and 100 ul of 10-2 and 10-3 dilution on Kan.
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>23<SUP>th</SUP> August</H3><P lang="en-US">
    Results: failure: as much GFP+ colonies growing on Kan than on LB
</P><P lang="en-US"><EM>PCR from mini Gene sRNA anti Cm</EM></P><P lang="en-US">
    35 H2O
</P><P lang="en-US">
    10 Buffer 5x
</P><P lang="en-US">
    1 dNTP
</P><P lang="en-US">
    2 FW iT009
</P><P lang="en-US">
    2 RV iT0010
</P><P lang="en-US">
    0.2 minigene-plasmid amp from IDT
</P><P lang="en-US">
    0.5 phusion
</P><P lang="en-US">
    PCR Phusion Gradient around 53 °C
</P><P lang="en-US">
    Time : 30 secondes
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>26<SUP>th</SUP> August</H3><P lang="en-US">
    Digestion of litmus miniprep (C=280ng/uL)with PstI and SpeI
</P><P lang="en-US">
    13 H2O
</P><P lang="en-US">
    3 litmus miniprep
</P><P lang="en-US">
    2 Buffer
</P><P lang="en-US">
    1 PstI
</P><P lang="en-US">
    1 SpeI
</P><P lang="en-US">
    20uL Tot
</P><P lang="en-US">
    37°C
</P><P lang="en-US">
    20min
</P><P lang="en-US">
    Digestion of sRNA_Cm insert (C=31ng/uL) with XbaI and PstI
</P><P lang="en-US">
    10 H2O
</P><P lang="en-US">
    15 sRNA_Cm insert
</P><P lang="en-US">
    3 Buffer
</P><P lang="en-US">
    1 XbaI
</P><P lang="en-US">
    1 PstI
</P><P lang="en-US">
    30uL Tot
</P><P lang="en-US">
    37°C
</P><P lang="en-US">
    20min
</P><P lang="en-US">
    PCR purification of litmus digestion using Thermo Scientific Kit
</P><P lang="en-US">
    PCR purification of sRNA_Cm insert digestion using Thermo Scientific Kit
</P><P lang="en-US">
    Ligation sRNA_Cm insert purif + litmus
</P><P lang="en-US">
    Digestion vector : 13,3 ng/ul
</P><P lang="en-US">
    Digestion insert : 10
</P><P lang="en-US">
    masse insert = 10*376/3626*50 = 52 ng
</P><P lang="en-US">
    Vector : 3.8 uL
</P><P lang="en-US">
    Insert : 5.2 uL
</P><P lang="en-US">
    Buffer : 2 uL
</P><P lang="en-US">
    T4 DNA ligase : 1 uL
</P><P lang="en-US">
    H2O : 8 uL
</P><P lang="en-US">
    Let incubate at 22°C for 30 min
</P><P lang="en-US">
    Ligation control litmus
</P><P lang="en-US">
    Vector : 3.8 uL
</P><P lang="en-US">
    Insert : 0 uL
</P><P lang="en-US">
    Buffer : 2 uL
</P><P lang="en-US">
    T4 DNA ligase : 1 uL
</P><P lang="en-US">
    H2O : 13,2 uL
</P><P lang="en-US">
    Let incubate at 22°C for 30 min
</P><P lang="en-US">
    Digestion of pUC18 PCR (C=13ng/uL) with EcorI and SpeI
</P><P lang="en-US">
    10 H2O
</P><P lang="en-US">
    15 pUC18 PCR
</P><P lang="en-US">
    3 Buffer
</P><P lang="en-US">
    1 EcorI
</P><P lang="en-US">
    1 SpeI
</P><P lang="en-US">
    30uL Tot
</P><P lang="en-US">
    37°C
</P><P lang="en-US">
    20min
</P><P lang="en-US">
    Digestion of sRNA_Cm insert (C=31ng/uL) with SpeI and EcorI
</P><P lang="en-US">
    10 H2O
</P><P lang="en-US">
    15 sRNA_Cm insert
</P><P lang="en-US">
    3 Buffer
</P><P lang="en-US">
    1 EcorI
</P><P lang="en-US">
    1 SpeI
</P><P lang="en-US">
    30uL Tot
</P><P lang="en-US">
    37°C
</P><P lang="en-US">
    20min
</P><P lang="en-US">
    PCR purification of pUC18 digestion using Thermo Scientific Kit
</P><P lang="en-US">
    PCR purification of sRNA_Cm insert digestion using Thermo Scientific Kit
</P><P lang="en-US">
    Ligation sRNA_Cm insert purif + pUC18
</P><P lang="en-US">
    Digestion vector : 4,3 ng/ul
</P><P lang="en-US">
    Digestion insert : 9,4
</P><P lang="en-US">
    masse insert = 10*362/2175*50 = 83 ng
</P><P lang="en-US">
    Vector : 11,6 uL
</P><P lang="en-US">
    Insert : 8,8 uL
</P><P lang="en-US">
    Buffer : 3 uL
</P><P lang="en-US">
    T4 DNA ligase : 1,5 uL
</P><P lang="en-US">
    H2O : 7,1 uL
</P><P lang="en-US">
    Let incubate at 22°C for 30 min
</P><P lang="en-US">
    Ligation control pUC18
</P><P lang="en-US">
    Vector : 11,6 uL
</P><P lang="en-US">
    Insert : 0 uL
</P><P lang="en-US">
    Buffer : 3 uL
</P><P lang="en-US">
    T4 DNA ligase : 1,5 uL
</P><P lang="en-US">
    H2O : 15,9 uL
</P><P lang="en-US">
    Let incubate at 22°C for 30 min
</P><P lang="en-US">
    Chemical transformation in NEB-Turbo only (control -)
</P><P lang="en-US">
    Chemical transformation in NEB-Turbo of ligation sRNA_Cm insert purif + litmus
</P><P lang="en-US">
    Chemical transformation in NEB-Turbo of ligation control litmus
</P><P lang="en-US">
    Chemical transformation in NEB-Turbo of ligation sRNA_Cm insert purif + pUC18
</P><P lang="en-US">
    Chemical transformation in NEB-Turbo of ligation control pUC18
</P><P lang="en-US">
    Chemical transformation in NEB-Turbo of litmus (control +)
</P><P lang="en-US">
    Plate all 5 on Amp
</P><P lang="en-US"><EM>PCR from mini Gene sRNA anti LacZ</EM></P><P lang="en-US">
    35 H2O
</P><P lang="en-US">
    10 Buffer 5x
</P><P lang="en-US">
    1 dNTP
</P><P lang="en-US">
    2 FW iT009
</P><P lang="en-US">
    2 RV iT0010
</P><P lang="en-US">
    0.2 minigene-plasmid amp from IDT
</P><P lang="en-US">
    0.5 phusion
</P><P lang="en-US">
    PCR Phusion Gradient around 53 °C
</P><P lang="en-US">
    Time : 30 secondes
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>27<SUP>th</SUP> August</H3><P><B><EM>
Place your twit here
</EM></B></P><P lang="en-US"><STRONG>Building anti LacZ weapons:</STRONG></P><P lang="en-US"><STRONG>Gibson assembly of litmus (sT005) and sRNA_LacZ</STRONG></P><P lang="en-US">
    Digestion vector : 56 ng/ul
</P><P lang="en-US">
    Digestion insert : 9,6
</P><P lang="en-US">
    masse insert = 5*376/3626*50 = 26 ng
</P><P lang="en-US">
    Vector : 1 uL
</P><P lang="en-US">
    Insert : 3 uL
</P><P lang="en-US">
    Master Mix : 5 uL
</P><P lang="en-US">
    H2O : 1
</P><P lang="en-US">
    Heat shock following Gibson assembly kit protocole.
</P><P lang="en-US"><STRONG>Biobrick assembly</STRONG><STRONG>of sRNA_LacZ in litmus-GFP and pUC18</STRONG></P><P lang="en-US">
    Using digested vectors (litmus-GFP and pUC18) from 26/08
</P><P lang="en-US">
    Digestion of sRNA_LacZ insert (C=48ng/uL) with XbaI and PstI
</P><P lang="en-US">
    10 H2O
</P><P lang="en-US">
    15 sRNA_LacZ insert
</P><P lang="en-US">
    3 Buffer
</P><P lang="en-US">
    1 PstI
</P><P lang="en-US">
    1 XbaI
</P><P lang="en-US">
    30uL Tot
</P><P lang="en-US">
    37°C
</P><P lang="en-US">
    20min
</P><P lang="en-US">
    Digestion of sRNA_LacZ insert (C=45ng/uL) with SpeI and EcorI
</P><P lang="en-US">
    10 H2O
</P><P lang="en-US">
    15 sRNA_LacZ insert
</P><P lang="en-US">
    3 Buffer
</P><P lang="en-US">
    1 EcorI
</P><P lang="en-US">
    1 SpeI
</P><P lang="en-US">
    30uL Tot
</P><P lang="en-US">
    37°C
</P><P lang="en-US">
    20min
</P><P lang="en-US">
    Purification on column
</P><P lang="en-US">
    ligation:
</P><P lang="en-US">
    Ligation sRNA_LacZ insert purif + pUC18
</P><P lang="en-US">
    Digestion vector : 4,3 ng/ul
</P><P lang="en-US">
    Digestion insert : 18
</P><P lang="en-US">
    masse insert = 10*362/2175*50 = 83 ng
</P><P lang="en-US">
    Vector : 11,6 uL
</P><P lang="en-US">
    Insert : 4,6 uL
</P><P lang="en-US">
    Buffer : 3 uL
</P><P lang="en-US">
    T4 DNA ligase : 1,5 uL
</P><P lang="en-US">
    H2O : 11 uL
</P><P lang="en-US">
    Let incubate at 22°C for 30 min
</P><P lang="en-US">
    Ligation sRNA_LacZ insert purif + litmus
</P><P lang="en-US">
    Digestion vector : 13,3 ng/ul
</P><P lang="en-US">
    Digestion insert : 11,4
</P><P lang="en-US">
    masse insert = 10*376/3626*50 = 52 ng
</P><P lang="en-US">
    Vector : 3.8 uL
</P><P lang="en-US">
    Insert : 4,56 uL
</P><P lang="en-US">
    Buffer : 2 uL
</P><P lang="en-US">
    T4 DNA ligase : 1 uL
</P><P lang="en-US">
    H2O : 8,64 uL
</P><P lang="en-US">
    Let incubate at 22°C for 30 min
</P><P lang="en-US"><STRONG>Preparing experiments of LacZ silencing</STRONG></P><P lang="en-US">
    Plating MGZ1, F+ ; MG1655, F+ ; NEB Turbo on IPTG+Xgal
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>28<SUP>th</SUP> August</H3><P lang="en-US">
    as expected MGZ1 and MG1655 turn blue.
</P><P lang="en-US"><STRONG>Result from assemblies:</STRONG></P><P lang="en-US">
    Gibson sRNA LacZ in litmus-GFP: failure
</P><P lang="en-US">
    Biobrick assembly sRNA LacZ in Litmus-GFP : success
</P><P lang="en-US">
    Biobrick assembly sRNA LacZ in pUC18 : success
</P><P lang="en-US"><STRONG>Miniprep of Litmus-GFP-sRNACm ,Litmus-GFP-sRNALacZ, pUC18-sRNACm , pUC18-sRNALacZ</STRONG></P><P lang="en-US"><STRONG>
        Building phagemid weapon producing cells: Transformation of electrocompetent MGZ1, F+ with M13K07 + Litmus-GFP-sRNALacZ / Litmus-GFP-sRNACm
    </STRONG></P><P lang="en-US">
    Transforming:
</P><P lang="en-US">
    -1) negative control : 5,2
</P><P lang="en-US">
    -2) IDT plasmid with sRNALacZ ( should form white colonies of MGZ1 on Xgal+IPTG):5,1
</P><P lang="en-US">
    -3) Litmus-GFP-sRNALacZ1 ( should form white colonies of MGZ1 on Xgal+IPTG): 5,2
</P><P lang="en-US">
    -4) Litmus-GFP-sRNALacZ3 ( should form white colonies of MGZ1 on Xgal+IPTG): 5,2
</P><P lang="en-US">
    -5) Litmus-GFP-sRNACm1 (control XGal ) : 5,2
</P><P lang="en-US">
    -6) Litmus-GFP-sRNACm1 + M13K07: 5,2
</P><P lang="en-US">
    -7) Litmus-GFP-sRNACm2 + M13K07 : 5,2
</P><P lang="en-US">
    -8) Litmus-GFP-sRNACm3 + M13K07: 5,2
</P><P lang="en-US">
    -9) Litmus-GFP-sRNALacZ1 + M13K07 : 5,2
</P><P lang="en-US">
    -10) Litmus-GFP-sRNALacZ3 + M13K07: 5,2
</P><P lang="en-US">
    plate on AMP+IPTG+Xgal ( silencing of LacZ on MGZ1 strain) and KAN+AMP (phage factories cells)
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>29<SUP>th</SUP> August</H3><P><B><EM>
Place your twit here
</EM></B></P><P lang="en-US">
    silencing of LacZ failed on MGZ1: the sequence of LacZ might have mutation compared to MG1655 for which the sRNA was designed or the test is not
    quantitative enough (one sRNA =&gt; several proteins). Try to find a way to get quantitative data about the silencing =&gt; Miller Assay
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>30<SUP>th</SUP> August</H3><P><B><EM>
Place your twit here
</EM></B></P><P lang="en-US">
    attempt to silence LacZ on MG1655, F+ with litmusGFP-LacZ1 &amp; litmusGFP-LacZ2
</P><P lang="en-US">
    XXXXX
</P><P lang="en-US"><STRONG>Silencing experiments (LacZ)</STRONG></P><P lang="en-US">
    - Launch O/N culture of producing phages cells ; centrifuge ; take supernatant and filter it.
</P><P lang="en-US">
    -diluting 100x O/N of MGZ1, F+ -pCola in LB-Kan
</P><P lang="en-US">
    - At OD 0,7 : 5 tubes. Put 1 mL of cells in each tube. Add in tubes respectively : nothing, 100uL phages of litmusGFP-LacZ1 &amp; litmusGFP-LacZ2,
    litmusGFP-Cm1 &amp; litmusGFP-Cm2
</P><P lang="en-US">
    litmusGFP-Cm1 &amp; litmusGFP-Cm2 are used as negative control.
</P><P lang="en-US">
    - Incubate 45 minutes at 37°C
</P><P lang="en-US">
    - Serial Dilute every tube until 10-5. Plate for each tube 100ul of 10-5 and 10-3 dilutions on IPTG+Xgal and LB and 100uL on AMP+IPTG+Xgal and LB.
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>3<SUP>rd</SUP> September</H3><P lang="en-US"><STRONG>Silencing experiments (Cm)</STRONG></P><P lang="en-US">
    - Launch O/N culture of producing phages cells ; centrifuge ; take supernatant and filter it.
</P><P lang="en-US">
    -diluting 100x O/N of MGZ1, F+ -pACYc in LB-Cm (25 ug/ul)
</P><P lang="en-US">
    - At OD 0,7 : 5 tubes. Put 900 uL of cells in each tube. Add in tubes respectively : nothing, 100uL phages of litmusGFP-LacZ1 &amp; litmusGFP-LacZ2,
    litmusGFP-Cm1 &amp; litmusGFP-Cm2
</P><P lang="en-US">
    litmusGFP-lacZ1 &amp; litmusGFP-lacZ2 are used as negative control.
</P><P lang="en-US">
    - Incubate 45 minutes at 37°C
</P><P lang="en-US">
    - Serial Dilute every tube until 10-5. Plate 100ul for each tube according to this dilution table
</P><TABLE width="500" cellpadding="5" cellspacing="0"><TBODY><TR valign="BOTTOM"><TD width="90"><P lang="en-US">
                    LB
                </P></TD><TD width="90"><P lang="en-US">
                    Cm 25 ug/ul
                </P></TD><TD width="90"><P lang="en-US">
                    Cm 250ug/ul
                </P></TD><TD width="90"><P lang="en-US">
                    Cm 500 ug/ul
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    non infected
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA anti lacz
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA Cm1
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA Cm2
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA Cm3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA Cm4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD></TR></TBODY></TABLE></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>4<SUP>th</SUP> September</H3><P><B>Results of silencing experiments</B></P><P>It works !</P><P lang="en-US"><EM><STRONG>Miller assay</STRONG></EM></P><P lang="en-US"><STRONG>XXXXXX</STRONG></P><P lang="en-US"><EM><STRONG>Redo the silencing experiment</STRONG></EM></P><P lang="en-US">
    - Launch O/N culture of producing phages cells ; centrifuge ; take supernatant and filter it.
</P><P lang="en-US">
    -diluting 100x O/N of MGZ1, F+ -pACYc in LB-Cm (25 ug/ul)
</P><P lang="en-US">
    - At OD 0,7 : 5 tubes. Put 900 uL of cells in each tube. Add in tubes respectively : nothing, 100uL phages of litmusGFP-LacZ1, litmusGFP-Cm1 &amp;
    litmusGFP-Cm2
</P><P lang="en-US">
    litmusGFP-lacZ1 &amp; litmusGFP-lacZ2 are used as negative control.
</P><P lang="en-US">
    - Incubate 45 minutes at 37°C
</P><P lang="en-US">
    - Serial Dilute every tube until 10-5. Plate 100ul for each tube according to this dilution table
</P><TABLE width="600" cellpadding="5" cellspacing="0"><TBODY><TR valign="BOTTOM"><TD width="90"><P lang="en-US">
                    LB
                </P></TD><TD width="90"><P lang="en-US">
                    Cm 25 ug/ul
                </P></TD><TD width="90"><P lang="en-US">
                    Cm 125 ug/ul
                </P></TD><TD width="90"><P lang="en-US">
                    Cm 250ug/ul
                </P></TD><TD width="90"><P lang="en-US">
                    Cm 500 ug/ul
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    non infected
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA anti lacz
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">    10^-3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA Cm1
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA Cm2
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">              sRNA Cm3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA Cm4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD></TR></TBODY></TABLE></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>5<SUP>th</SUP> September</H3><P lang="en-US"><STRONG>Confirmation of silencing (CmR)</STRONG></P><P lang="en-US"><STRONG>Cloning biobricks : ie clone sRNAs (Cm, Kan, lacZ) in pB1C3</STRONG></P><TABLE width="232" cellpadding="1" cellspacing="0"><TBODY><TR><TD width="158"><P lang="en-US"><STRONG>Reagent</STRONG></P></TD><TD width="68"><P lang="en-US"><STRONG>Volume</STRONG></P></TD></TR><TR><TD width="158"><P lang="en-US">
                    Purified PCR Product
                </P></TD><TD width="68"><P lang="en-US">
                    16 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    H<SUB>2</SUB>O
                </P></TD><TD width="68"><P lang="en-US">
                    0 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    10x FastDigest Buffer
                </P></TD><TD width="68"><P lang="en-US">
                    2 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    FastDigest Enzyme 1
                </P></TD><TD width="68"><P lang="en-US">
                    1 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US">
                    FastDigest Enzyme 2
                </P></TD><TD width="68"><P lang="en-US">
                    1 ul
                </P></TD></TR><TR><TD width="158"><P lang="en-US"><STRONG>Total Volume</STRONG></P></TD><TD width="68"><P lang="en-US">
                    20.0 ul
                </P></TD></TR></TBODY></TABLE><P lang="en-US">
    Digest for 0.5 - 1 hours at 37 °C with shaking
</P><P lang="en-US">
    Ligation sRNA into biobrick vector
</P><P lang="en-US">
    masse insert = 10*419/2000*100 = 210 ng
</P><P lang="en-US">
    Vector : 5 uL
</P><P lang="en-US">
    Insert : 10 uL
</P><P lang="en-US">
    Buffer : 2 uL
</P><P lang="en-US">
    T4 DNA ligase : 1 uL
</P><P lang="en-US">
    H2O : 2uL
</P><P lang="en-US">
    Let incubate at 22°C for 30 min
</P><P lang="en-US">
    Transformation in NEB turbo chemical competent . Plate on Cm
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>7<SUP>th</SUP> September</H3><P><B><EM>
Place your twit here
</EM></B></P><P lang="en-US"><STRONG>07/09/13</STRONG></P><P lang="en-US"><EM>Results Transformation</EM></P><P lang="en-US">
    Colonies on the different paltes (no colonies on the negative controls)
</P><P lang="en-US"><EM>PCR Colonies</EM></P><P lang="en-US">
    Primer from Phage sensor team : XXXXX
</P><P lang="en-US">
    Gel :
</P><P lang="en-US">
    XXXXX
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>10<SUP>th</SUP> September</H3><P><B><EM>
Place your twit here
</EM></B></P><P lang="en-US"><EM><STRONG>Experiment Silencing Cm (try to reduce the noise, aka the variability)</STRONG></EM></P><P lang="en-US">
    - Launch O/N culture of producing phages cells ; centrifuge ; take supernatant and filter it.
</P><P lang="en-US">
    -diluting 100x O/N of MGZ1, F+ -pACYc in LB-Cm (25 ug/ul)
</P><P lang="en-US">
    - At OD 0,7 :. Put 900 uL of cells in each tube. Add in tubes respectively : nothing, 100uL phages of litmusGFP-LacZ1, litmusGFP-Cm1 &amp; litmusGFP-Cm2
</P><P lang="en-US">
    -3 replicates for each condition. Plate each Dilution 2 times to limit the variability
</P><P lang="en-US">
    Non infected cells (1,2,3)
</P><P lang="en-US">
    Cells infected with anti lacZ (1,2,3)
</P><P lang="en-US">
    Cells infected with anti CmA (1,2,3)
</P><P lang="en-US">
    Cells infected with antiCmB (1,2,3
</P><P lang="en-US">
    - Incubate 45 minutes at 37°C
</P><P lang="en-US">
    - Serial Dilute every tube until 10-5. Plate 100ul for each tube according to this dilution table
</P><TABLE width="500" cellpadding="5" cellspacing="0"><TBODY><TR valign="BOTTOM"><TD width="90"><P lang="en-US">
                    LB
                </P></TD><TD width="90"><P lang="en-US">
                    Cm 25 ug/ul
         </P></TD><TD width="90"><P lang="en-US">
                    Cm 250ug/ul
                </P></TD><TD width="90"><P lang="en-US">
                    Cm 500 ug/ul
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    non infected 1
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    non infected 2
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    non infected 3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA lacZ 1
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/2
                </P></TD><TD width="90"><P lang="en-US">
                    10^-2/1
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA lacZ 2
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/2
                </P></TD><TD width="90"><P lang="en-US">
                    10^-2/1
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA lacZ 3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/2
                </P></TD><TD width="90"><P lang="en-US">
                    10^-2/1
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA Cm A 1
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-2/1
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA Cm A 2
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-2/1
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA Cm A 3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-2/1
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA Cm B 1
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-2/1
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA Cm B 2
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-2/1
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA Cm B 3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-2/1
                </P></TD></TR></TBODY></TABLE></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>11<SUP>th</SUP> September</H3><P lang="en-US">
    Results experiment Silencing
</P><P lang="en-US">
    AT a concentration of 500mg/ul, we get <STRONG>99,1% of killing</STRONG> (on 12 replicates) with a standard déviation of 1,19
</P><UL><LI><P lang="en-US">
            When you try to understand why are the cells surviving, it is mainly due to the non silencing of the cells
        </P></LI><LI><P lang="en-US">
            This problem can be due to the necessity of the sRNA to match to the plasmid sequence perfectly (and therefore problemsof mutations, especially on
            phage).
        </P></LI></UL></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>14<SUP>th</SUP> September</H3><P><B><EM>
Place your twit here
</EM></B></P><P lang="en-US"><EM>Cloning two sRNA on the same phagemid</EM></P><P lang="en-US">
    P009 (litmus Kan) + sRNA Cm :
</P><P lang="en-US">
    P027 (litmus Cm) + sRNA Kan :
</P><P lang="en-US"><STRONG>Inserts</STRONG></P><P lang="en-US">
    sRNA Cm : 35 ng/uL Final (16*35/20) = 28 ng/uL
</P><P lang="en-US">
    sRNA Kan : 16 ng/uL Final (16*16/20) = 12,8 ng/uL
</P><CENTER><TABLE width="232" cellpadding="1" cellspacing="0"><TBODY><TR><TD width="158"><P lang="en-US"><STRONG>Reagent</STRONG></P></TD><TD width="68"><P lang="en-US"><STRONG>Volume</STRONG></P></TD></TR><TR><TD width="158"><P lang="en-US">
                        Purified PCR Product
                    </P></TD><TD width="68"><P lang="en-US">
                        16 ul
                    </P></TD></TR><TR><TD width="158"><P lang="en-US">            H<SUB>2</SUB>O
                    </P></TD><TD width="68"><P lang="en-US">
                        0 ul
                    </P></TD></TR><TR><TD width="158"><P lang="en-US">
                        10x FastDigest Buffer
                    </P></TD><TD width="68"><P lang="en-US">
                        2 ul
                    </P></TD></TR><TR><TD width="158"><P lang="en-US">
                        XbaI
                    </P></TD><TD width="68"><P lang="en-US">
                        1 ul
                    </P></TD></TR><TR><TD width="158"><P lang="en-US">
                        Pst1
                    </P></TD><TD width="68"><P lang="en-US">
                        1 ul
                    </P></TD></TR><TR><TD width="158"><P lang="en-US"><STRONG>Total Volume</STRONG></P></TD><TD width="68"><P lang="en-US">
                        20.0 ul
                    </P></TD></TR></TBODY></TABLE></CENTER><P lang="en-US"><STRONG>Vectors</STRONG></P><P lang="en-US">
    Litmus Kan : 350 ng /uL Final (3*350/20) = 52,5 75 ng/ul
</P><P lang="en-US">
    Litmus Cm : 750 ng /ul Final (2*750/20) = 75 ng/uL
</P><CENTER><TABLE width="232" cellpadding="1" cellspacing="0"><TBODY><TR><TD width="158"><P lang="en-US"><STRONG>Reagent</STRONG></P></TD><TD width="68"><P lang="en-US"><STRONG>Volume</STRONG></P></TD></TR><TR><TD width="158"><P lang="en-US">
                        Purified PCR Product
                    </P></TD><TD width="68"><P lang="en-US">
                        3
                    </P></TD></TR><TR><TD width="158"><P lang="en-US">
                        H<SUB>2</SUB>O
                    </P></TD><TD width="68"><P lang="en-US">
                        13 ul
                    </P></TD></TR><TR><TD width="158"><P lang="en-US">
                        10x FastDigest Buffer
                    </P></TD><TD width="68"><P lang="en-US">
                        2 ul
                    </P></TD></TR><TR><TD width="158"><P lang="en-US">
                        Pst1
                    </P></TD><TD width="68"><P lang="en-US">
                        1 ul
                    </P></TD></TR><TR><TD width="158"><P lang="en-US">
                        SpeI
                    </P></TD><TD width="68"><P lang="en-US">
                        1 ul
                    </P></TD></TR><TR><TD width="158"><P lang="en-US"><STRONG>Total Volume</STRONG></P></TD><TD width="68"><P lang="en-US">
                        20.0 ul
                    </P></TD></TR></TBODY></TABLE></CENTER><P lang="en-US">
    Ligation sRNA into biobrick vector
</P><P lang="en-US">
    Litmus Kan + sRNA anti Cm
</P><P lang="en-US">
    masse insert = 10*419/4000*100 = 110 ng
</P><P lang="en-US">
    Vector : 3 uL
</P><P lang="en-US">
    Insert : 4 uL
</P><P lang="en-US">
    Buffer : 2 uL
</P><P lang="en-US">
    T4 DNA ligase : 1 uL
</P><P lang="en-US">
    H2O : 10uL
</P><P lang="en-US">
    Litmus Cm + sRNA anti Kan
</P><P lang="en-US">
    masse insert = 10*419/4000*100 = 110 ng
</P><P lang="en-US">
    Vector : 1,5 uL
</P><P lang="en-US">
    Insert : 10 uL
</P><P lang="en-US">
    Buffer : 2 uL
</P><P lang="en-US">
    T4 DNA ligase : 1 uL
</P><P lang="en-US">
    H2O : 5,5uL
</P><P lang="en-US">
    Let incubate at 22°C for 30 min
</P><P lang="en-US"><EM><STRONG>Launch cultures : </STRONG></EM></P><P lang="en-US">
    Glycerol of the different biobricks : sT040 ; sT041 ; sT042 sT043 ; sT044 ; sT045 ; sT046 ; sT047 on Cm
</P><P lang="en-US">
    MG F+ + pCola sT027 on Kan :
</P><P lang="en-US">
    MGZ1, F+ Litmus-GFP-sRNA antiKan + M13KO7 Helper col : sT022 sT023 sT024 on Kan Amp
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>15<SUP>th</SUP> September</H3><P lang="en-US"><EM>Exp Silencing Kan</EM></P><P lang="en-US">
    - Launch O/N culture of producing phages cells ; centrifuge ; take supernatant and filter it.
</P><P lang="en-US">
    -diluting 100x O/N of MGZ1, F+ -pCola in LB-Kan (25 ug/ul)
</P><P lang="en-US">
    - At OD 0,7 :. Put 900 uL of cells in each tube. Add in tubes respectively : nothing, 100uL phages of litmusGFP-LacZ1, litmusGFP-Kan1 &amp; litmusGFP-Kan2
</P><P lang="en-US">
    -2 replicates for each condition. Plate each Dilution 2 times to limit the variability
</P><P lang="en-US">
    Non infected cells (1,2)
</P><P lang="en-US">
    Cells infected with anti lacZ (1,2)
</P><P lang="en-US">
    Cells infected with anti KanA (1,2)
</P><P lang="en-US">
    Cells infected with anti KanB (1,2,)
</P><P lang="en-US">
    - Incubate 70 minutes at 37°C
</P><P lang="en-US">
    - Serial Dilute every tube until 10-5. Plate 100ul for each tube according to this dilution table
</P><TABLE width="500" cellpadding="5" cellspacing="0"><TBODY><TR valign="BOTTOM"><TD width="90"><P lang="en-US">
                    LB
                </P></TD><TD width="90"><P lang="en-US">
                    Kan 25 ug/ul
                </P></TD><TD width="90"><P lang="en-US">
                    Kan 250ug/ul
                </P></TD><TD width="90"><P lang="en-US">
                    Kan 500 ug/ul
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    non infected 1
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    non infected 2
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA lacZ 1
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3/2
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA lacZ 2
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
            </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3/2
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA Kan A 1
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3/2
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA Kan A 2
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3/2
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA Kan B 1
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3/2
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA Kan B 2
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3/2
                </P></TD></TR></TBODY></TABLE><P lang="en-US"><EM>Miniprep biobricks</EM></P><P lang="en-US">
    Cm1 / Cm2 / Kan 1 / Kan 2 / Lac1 / Lac2
</P><P lang="en-US"><EM>Cloning 2 sRNA on the same phage : Ligation sRNA into biobrick vector</EM></P><P lang="en-US">
    Litmus Kan + sRNA anti Cm
</P><P lang="en-US">
    masse insert = 10*419/4000*100 = 110 ng
</P><P lang="en-US">
    Vector : 3 uL
</P><P lang="en-US">
    Insert : 4 uL
</P><P lang="en-US">
    Buffer : 2 uL
</P><P lang="en-US">
    T4 DNA ligase : 1 uL
</P><P lang="en-US">
    H2O : 10uL
</P><P lang="en-US">
    Litmus Cm + sRNA anti Kan
</P><P lang="en-US">
    masse insert = 10*419/4000*100 = 110 ng
</P><P lang="en-US">
    Vector : 1,5 uL
</P><P lang="en-US">
    Insert : 10 uL
</P><P lang="en-US">
    Buffer : 2 uL
</P><P lang="en-US">
    T4 DNA ligase : 1 uL
</P><P lang="en-US">   H2O : 5,5uL
</P><P lang="en-US">
    Let incubate at 22°C for 30 min
</P><P lang="en-US">
    Transformation in chemical competent 10G cells : plate on Amp;
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>18<SUP>th</SUP> September</H3><P lang="en-US"><EM>Exp Silencing Kan</EM></P><P lang="en-US">
    - Launch O/N culture of producing phages cells ; centrifuge ; take supernatant and filter it.
</P><P lang="en-US">
    -diluting 100x O/N of MGZ1, F+ -pCola in LB-Kan (25 ug/ul)
</P><P lang="en-US">
    - At OD 0,7 :. Put 900 uL of cells in each tube. Add in tubes respectively : nothing, 100uL phages of litmusGFP-LacZ1, litmusGFP-Kan1 &amp; litmusGFP-Kan2
</P><P lang="en-US">
    -2 replicates for each condition. Plate each Dilution 2 times to limit the variability
</P><P lang="en-US">
    Non infected cells (1,2)
</P><P lang="en-US">
    Cells infected with anti lacZ (1,2)
</P><P lang="en-US">
    Cells infected with anti KanA (1,2)
</P><P lang="en-US">
    Cells infected with anti KanB (1,2,)
</P><P lang="en-US">
    - Incubate 70 minutes at 37°C
</P><P lang="en-US">
    - Serial Dilute every tube until 10-5. Plate 100ul for each tube according to this dilution table
</P><TABLE width="500" cellpadding="5" cellspacing="0"><TBODY><TR valign="BOTTOM"><TD width="90"><P lang="en-US">
                    LB
                </P></TD><TD width="90"><P lang="en-US">
                    Kan 25 ug/ul
                </P></TD><TD width="90"><P lang="en-US">
                    Kan 250ug/ul
                </P></TD><TD width="90"><P lang="en-US">
                    Kan 500 ug/ul
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    non infected 1
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    non infected 2
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA lacZ 1
                </P></TD><TD width="90"><P lang="en-US">                  10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3/2
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA lacZ 2
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3/2
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA Kan A 1
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3/2
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA Kan A 2
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3/2
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA Kan B 1
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3/2
                </P></TD></TR><TR valign="BOTTOM"><TD height="7" width="90"><P lang="en-US">
                    sRNA Kan B 2
                </P></TD><TD width="90"><P lang="en-US">
                    10^-5/4
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-4/3
                </P></TD><TD width="90"><P lang="en-US">
                    10^-3/2
                </P></TD></TR></TBODY></TABLE><P lang="en-US"><EM>Miniprep biobricks</EM></P><P lang="en-US">
    Cm1 / Cm2 / Kan 1 / Kan 2 / Lac1 / Lac2
</P><P lang="en-US"><EM>Cloning 2 sRNA on the same phage : Ligation sRNA into biobrick vector</EM></P><P lang="en-US">
    Litmus Kan + sRNA anti Cm
</P><P lang="en-US">
    masse insert = 10*419/4000*100 = 110 ng
</P><P lang="en-US">
    Vector : 3 uL
</P><P lang="en-US">
    Insert : 4 uL
</P><P lang="en-US">
    Buffer : 2 uL
</P><P lang="en-US">
    T4 DNA ligase : 1 uL
</P><P lang="en-US">
    H2O : 10uL
</P><P lang="en-US">
    Litmus Cm + sRNA anti Kan
</P><P lang="en-US">
    masse insert = 10*419/4000*100 = 110 ng
</P><P lang="en-US">
    Vector : 1,5 uL
</P><P lang="en-US">
    Insert : 10 uL
</P><P lang="en-US">
    Buffer : 2 uL
</P><P lang="en-US">
    T4 DNA ligase : 1 uL
</P><P lang="en-US">
    H2O : 5,5uL
</P><P lang="en-US">
    Let incubate at 22°C for 30 min
</P><P lang="en-US">
    Transformation in chemical competent 10G cells : plate on Amp;
</P></DIV><DIV class="tbnote"><H2>Sabotage</H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" target="_blank" class="tbnotelogo THlogo"> ASDF </A><H3>20<SUP>th</SUP> September</H3><P lang="en-US">
    results from sequencing:
</P><P lang="en-US">
    Litmus-Anti LacZ: OK
</P><P lang="en-US">
    Litmus-Anti Kan: OK
</P><P lang="en-US">
    Litmus-Anti Cm: OK
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Paris_Bettencourt/Notebook">http://2013.igem.org/Team:Paris_Bettencourt/Notebook</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Bettencourt/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Bettencourt/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Notebook&amp;oldid=361475" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DL></DIV></OL></DIV></OL></DIV></UL></DIV></UL></DIV></DIV></DIV></BODY></HTML>