<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Paris_Bettencourt/Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris_Bettencourt/Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Results&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris_Bettencourt/Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris Bettencourt/Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="navmenubkg"><DIV id="navmenubox"><UL id="nav"><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Overview" title="Project Overview">PROJECT</A></LI><LI style="margin-left:14.5px"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Overview" title="Project Overview">OVERVIEW</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect" title="Detect">DETECT</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target" title="Target">TARGET</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate" title="Infiltrate">INFILTRATE</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage" title="Sabotage">SABOTAGE</A></LI></UL></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Results" title="Achievements">ACHIEVEMENTS</A></LI><LI style="margin-left:141px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Parts" title="Submitted Parts">PARTS</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Results" title="Results">RESULTS</A></LI><LI style="width:167px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Medal_Fulfillment" title="Medal Fulfillment">MEDAL FULFILLMENT</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Overview" title="Human Practice Overview">HUMAN PRACTICE</A></LI><LI style="margin-left:141px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Overview" title="Human Practice Overview">OVERVIEW</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Gender_Study" title="Gender Study">GENDER STUDY</A></LI><LI style="width:168px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Technology_Transfer" title="Technology_Transfer">TECHNOLOGY TRANSFER</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/TB_Facts" title="TB Tacts">TB FACTS</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/TB_France" title="TB in France">TB IN FRANCE</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/TB_Gallery" title="TB Gallery">TB GALLERY</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Safety" title="Safety">SAFETY</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Collaboration" title="Collaboration">COLLABORATION</A></LI><LI style="margin-left:500px"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Collaboration" title="Collaboration">COLLABORATION</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/SensiGEM" title="SensiGEM">SENSIGEM</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Notes" title="Notebook">NOTEBOOK</A></LI><LI style="margin-left:649.5px"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Notebook" title="Day Notes">DAY NOTES</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Protocols" title="Protocols">PROTOCOLS</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Bibliography" title="Bibliography">BIBLIOGRAPHY</A></LI></DIV><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Team" title="Team">TEAM</A></LI><LI style="margin-left:522.5px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Team" title="Team">OUR TEAM</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Gallery" title="Team Gallery">GALLERY</A></LI><LI><A href="https://2013.igem.org/Team:Paris_Bettencourt/Attributions" title="Attributions">ATTRIBUTIONS</A></LI><LI style="width:167px;"><A href="https://2013.igem.org/Team:Paris_Bettencourt/Acknowledgements" title="Acknowledgements">ACKNOWLEDGEMENTS</A></LI><DIV id="twnoticon" class="sidelogos"><A class="twitter-timeline" width="250" height="400" data-dnt="true" href="https://twitter.com/iGEM_Paris" data-widget-id="368144501119795201">
            @iGEM_Paris</A></DIV><DIV style="height:68px"><A href="#Detect"><DIV class="hlink" style="width:355px"><H2>Detect</H2></DIV></A><A href="#Target"><DIV class="hlink" style="width:355px"><H2>Target</H2></DIV></A><A href="#Infiltrate"><DIV class="hlink" style="width:355px;margin-right:0"><H2>Infiltrate</H2></DIV></A></DIV><DIV style="height:68px"><A href="#Sabotage"><DIV class="hlink" style="width:355px"><H2>Sabotage</H2></DIV></A><A href="#Modelling"><DIV class="hlink" style="width:355px"><H2>Modelling</H2></DIV></A><A href="#Human_Practice"><DIV class="hlink" style="width:355px;margin-right:0"><H2>Human Practice</H2></DIV></A></DIV><DIV id="page"><DIV id="Detect"><H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Detect">Detect</A></H2><DIV class="overbox"><DIV class="subbox1"><DIV class="bkgr"><H2>Background</H2><P>CRISPR/Cas systems generate site-specific double strand breaks and have recently been used for genome editing.  </P></DIV><DIV class="aims"><H2>Aims</H2><P>Building a genotype sensor based on CRISPR/Cas that reports existance of an antibiotic resistance gene.</P></DIV></DIV><DIV class="subbox2"><DIV class="results"><H2>Results</H2><UL><LI>Successfully cloned gRNA anti-KAN, crRNA anti-KAN, tracrRNA-Cas9 and pRecA-LacZ into Biobrick backbones and therefore generated four new BioBricks. </LI><LI> Confirmation of site-specific binding and DNA double-strand breaks generated by the gRNA-Cas9 complex in the kanamycin resistance casette. </LI><LI>Testing the new assembly standard for our cloning.</LI></UL></DIV></DIV><DIV class="mainfig"><P style="margin-top:-60px;font-size:13px"><B>CRISPR anti-Kan plasmids target specifically kanamycin resistant <I>E. coli</I>. </B>We introduced our CRISPR-based DNA cleavage system to two strains of <I>E. coli</I> : one WT (blue bars) and one carrying a genomically integrated kanamycin resistance casette (KanR, blue bars). The strains were co-transformed with two plasmids. One with the Cas9 construct, the other with the anti-Kanamycin gRNA. WT <I>E. coli</I> could be efficiently transformed with one or both plasmids, as determined by selective plating. However, KanR <I>E. coli</I> could not be efficiently transformed with both. We attribute this to Cas9-induced cleavage of the chromosome specifically at the KanR casette, with about 99% efficiency.
	</P></DIV></DIV><H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target">Target</A></H2><DIV class="overbox"><DIV class="subbox1"><DIV class="bkgr"><H2>Background</H2><P>SirA is an essential gene in latent tuberculosis infections.  </P></DIV><DIV class="aims"><H2>Aims</H2><P>To perform a drug screen targeted at the sirA gene from mycobacteria.</P></DIV></DIV><DIV class="subbox2"><DIV class="results"><H2>Results</H2><UL><LI>Produced an <I>E. coli</I> strain which relies upon mycobacterial sirA, fprA and fdxA genes to survive in M9 minimal media.</LI><LI>Demonstrated that <I>E. coli</I> can survive with mycobacterial sulfite reduction pathway with Flux Balance Analysis.</LI><LI>Located drug target sites on sirA as well as identified high structural similarity between cysI and sirA through structural analysis.</LI><LI>Identified a potential anti-TB activity of Pyridoxine at high doses.</LI></UL></DIV></DIV><DIV class="mainfig"><P style="font-size:13px"><B>MycoSIR <I>E. coli</I> depend on our synthetic pathway for growth.</B><I>E. coli</I> strain BL21(DE3) was deleted for cysI and transformed with the three genes of the mycoSIR pathway expressed from IPTG-inducible T7 promoters (red). Wild-type (blue), uninduced (purple) and pathway-minus (gold) strains were used as controls. Both time course growth curves (A) and final ODs (B) reveal that the complete, induced pathway is required for growth 
	</P></DIV></DIV><DIV id="Infiltrate"><H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate">Infiltrate</A></H2><DIV class="overbox"><DIV class="subbox1"><DIV class="bkgr"><H2>Background</H2><P>Latent tuberculosis persists inside macrophages of the lungs, where it is partially protected from both the host immune system and conventional antibiotics.  </P></DIV><DIV class="aims"><H2>Aims</H2><P>To create an <I>E. coli</I> strain capable of entering the macrophage cytosol and delivering a lytic enzyme to kill mycobacteria.</P></DIV></DIV><DIV class="subbox2"><DIV class="results"><H2>Results</H2><UL><LI>We expressed the enzyme Trehalose Dimycolate Hydrolase (TDMH) in <I>E.coli</I> and showed that it is highly toxic to mycobacteria in culture.</LI><LI>We expressed the listeriolyin O (LLO) gene in <I>E. coli</I> and showed that it is capable of entering the macrophage cytosol.</LI><LI>We co-infected macrophages with both mycobacteria and our engineered <I>E. coli</I> to characterize the resulting phagocytosis and killing.</LI></UL></DIV></DIV><DIV class="mainfig"><P style="font-size:13px"><B>TDMH expression kills mycobacteria in culture.</B> We mixed <I>E. coli</I> and WT <I>M.smegmatis</I> in LB media. Plating assays were used to count specifically <I>M.smegmatis</I> after the indicated times. When TDMH-expression was fully induced, more than 98% of mycobacteria were killed after 6 hours (red line). Populations of mycobacteria alone (black line) and mycobacteria mixed with uninduced <I>E. coli</I> (blue line) were stable.
	</P></DIV></DIV><DIV id="Sabotage"><H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage">Sabotage</A></H2><DIV class="overbox"><DIV class="subbox1"><DIV class="bkgr"><H2>Background</H2><P>One of the main concerns about tuberculosis today is the emergence of antibiotic resistant strains.</P></DIV><DIV class="aims"><H2>Aims</H2><P>Our objective is to make an antibiotic-resistant bacterial population sensitive again to the selfsame antibiotics.</P></DIV></DIV><DIV class="subbox2"><DIV class="results"><H2>Results</H2><UL><LI>Construction and characterization of phagemids coding for small RNA targeting antibiotic resistance proteins.</LI><LI>Showed theoretically burden of a device is critical for the maintenance of a genetic element in a population.</LI><LI>Successful conversion of antibiotic resistant population of <I>E. coli</I> to a sensitive state (on two different antibiotic resistances).</LI></UL></DIV></DIV><DIV class="mainfig"><P style="font-size:13px"><B>Our synthetic phage conveys antibiotic-sensitivity to an antbiotic-resistant population. </B>The anti-Chloramphenicol phage system effectively killed 99.1% of the Chloramphenicol resistant population and the anti- Kanamycine phage effictevely killed 99,5% of the Kanamycine resistant population.
	</P></DIV></DIV><DIV id="Modelling"><H2><A href="">Modelling</A></H2><DIV class="overbox" style=""><DIV class="projtile" style=""><H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Sabotage">Population Dynamics Model</A></H2><P>This model investigates the propagation of horizontal gene-transfer via phagemid/helper system. We study the effect of the burden of a desired device on the maintenance of the system in time.</P><P style="font-size:13px">Left: scheme representing the regular non-lytic M13 bacteriophage horizontal spread. Right: scheme representing the main processes of the phagemid/helper system.</P></DIV><DIV class="projtile" style=""><H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target">Structure Based Modelling</A></H2><P><B>SirA: </B>Using Swiss pdb we demonstrated the superimposed 3D structures of <I>Mycobacterium tuberculosis</I> SirA and <I>Escherichia coli</I> CysI highlighting their similarities and differences. Both proteins are important in their respective sulphite reduction pathways. We then predicted the effect of a small drug target based on SirA's structure. </P><P><B>FprA: </B>Using LigandScout 3.1, we searched over 8100 drug compounds from the BindingDB and Chembl databases for drugs targetting mycobacterial Ferredoxin NADP reductase (FprA). We have modelled Pyridoxine's interaction to its active site. </P></DIV><DIV class="projtile" style="margin-right:0;"><H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target">Flux Balance Analysis</A></H2><P>We used an <I>E. coli</I> model iJR904 obtained from BiGG database as a starting model and obtained a growth rate represented by the f value of 0.9129. We then deleted the reaction which encodes for the sulphite reduction pathway and obtained a f value of -8.63e-13 indicating that the sulphite reduction pathway is essential. We wrote a program which finds all essential reactions in <I>M. tuberculosis</I> and <I>E. coli</I> SBML models and attempts to restore growth for each essential <I>E. coli</I> model with essential reactions from M. tuberculosis to identify other metabolic pathways we could apply a targeted drug screen to.  We identified <A href="https://2013.igem.org/Team:Paris_Bettencourt/Project/Target/FBA">100 other essential reactions</A> we can target.</P></DIV></DIV><DIV id="Human_Practice"><H2><A href="https://2013.igem.org/wiki/index.php?title=Team:Paris_Bettencourt/Human_Practice/Overview">Human Practice</A></H2><DIV class="overbox"><DIV class="subbox3"><DIV class="bkgr"><H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Technology_Transfer">Technology  Transfer  </A></H2><P> An essay that addresses the issue of designing a technology aimed at &quot;developing&quot; countries, rather than at “developed” ones: a typical case of technology transfer.</P></DIV><DIV class="aims"><H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/TB_France">TB in France  </A></H2><P>Analysis of the social, medical and political aspects of the management of tuberculosis in France.
Synthetic biology may help in many ways such as treatments, drug development, diagnostic. We also give advice on how to introduce it in clinics. </P></DIV></DIV><DIV class="subbox4"><DIV class="aims" style="height:100%"><H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/Gender_Study">Gender Study  </A></H2><P>A comprehensive and quantitative study of gender (in)equality in iGEM and synthetic biology. A database was gathered and statistically analysed in order to depict sex ratio of iGEM teams' students and supervisors.</P><P style="margin-top:-10px;font-size:13px"><B>Gender balance and success in iGEM</B>Winning teams have a significantly higher number of women and are more Gender balanced.</P></DIV></DIV><DIV class="subbox3" style="margin-right:0"><DIV class="bkgr"><H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/TB_Facts">TB Facts  </A></H2><P> Infographics page containing TB data and facts that captures all you need to know at a glance.</P></DIV><DIV class="aims"><H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Human_Practice/TB_Gallery"> TB Gallery </A></H2><P> A gallery of famous historic figures who had tuberculosis, made to raise awareness to its prevalence of in the past and present .</P></DIV></DIV></DIV><DIV id="Collaboration"><H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Collaboration">Collaboration</A></H2><DIV class="overbox" style="height:300px"><DIV class="projtile" style="height:300px"><H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/SensiGEM">SensiGEM</A></H2><P>SensiGEM is the iGEM Biosensor database generated by the teams Paris Bettencourt 2013 and Calgary 2013. In this database you can find fast and easy what biosensor projects were already done by past iGEM Teams. To be able to select the projects that fit into the database, we also collaborated to compose a joint definition a biosensor.
        </P></DIV><DIV class="projtile" style="height:300px"><H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Collaboration">BGU iGEM Team from Israel</A></H2><P> A mutual part characterization.  We characterize the promoter units produced by the lac/ara-1 promoter of cI, a repressor of their constructed kill switch. In return, BGU characterizes our TDMH biobrick protein expression levels by Western Blot.  
        </P></DIV><DIV class="projtile" style="height:300px;margin-right:0;"><H2><A href="https://2013.igem.org/Team:Paris_Bettencourt/Collaboration">Braunschweig iGEM Team</A></H2><P>Idea, bibliography, and beer sharing!</P></DIV></DIV></DIV><DIV class="mfooter"><DIV class="footbox" style=""><B>Centre for Research and Interdisciplinarity (CRI)</B>
    Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor
    Paris Descartes University
    24, rue du Faubourg Saint Jacques
    75014 Paris, France+33 1 44 41 25 22/25<A href="mailto:team2013@igem-paris.org">team2013@igem-paris.org</A></DIV><CENTER>
    Copyright (c) 2013 igem.org. All rights reserved.
  </CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Paris_Bettencourt/Results">http://2013.igem.org/Team:Paris_Bettencourt/Results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Bettencourt/Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Bettencourt/Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Results&amp;oldid=360927" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>