"
Page
Discussion
View source
History
teams
Log in
 
Team:Paris Bettencourt/Notebook
From 2013.igem.org
PROJECT
OVERVIEW
DETECT
TARGET
INFILTRATE
SABOTAGE
ACHIEVEMENTS
PARTS
RESULTS
MEDAL FULFILLMENT
HUMAN PRACTICE
OVERVIEW
GENDER STUDY
TECHNOLOGY TRANSFER
TB FACTS
TB IN FRANCE
TB GALLERY
SAFETY
COLLABORATION
COLLABORATION
SENSIGEM
NOTEBOOK
DAY NOTES
PROTOCOLS
BIBLIOGRAPHY
TEAM
OUR TEAM
GALLERY
ATTRIBUTIONS
ACKNOWLEDGEMENTS
@iGEM_Paris
Please be patient. It may take a few seconds to load
DAY NOTES
DETECT
July
Saturday 13th July
Sunday 14th July
Monday 15th July
Tuesday 16th July
Wednesday 17th July
Thursday 18th July
Monday 22nd July
Tuesday 23rd July
Monday 29th July
Tuesday 30th July
Wednesday 31st July
August
Thursday 1st August
Friday 2nd August
Monday 5th August
Tuesday 6th August
Wednesday 7th August
Thursday 8th August
Friday 9th August
Saturday 10th August
Sunday 11th August
Monday 12th August
Tuesday 13th August
Wednesday 14th August
Thursday 15th August
Friday 16th August
Monday 19th August
Tuesday 20th August
Wednesday 21st August
Thursday 22nd August
Friday 23rd August
Sunday 25th August
Monday 26th August
Tuesday 27th August
Wednesday 28th August
Friday 30th August
September
Tuesday 3rd September
Wednesday 4th September
Thursday 5th September
Friday 6th September
Saturday 7th September
Sunday 8th September
Monday 16th September
Tuesday 17th September
Wednesday 18th September
Sunday 22nd September
Monday 23rd September
Tuesday 24th September
Wednesday 25th September
Thursday 26th September
Friday 27th September
Saturday 28th September
Monday 30th September
October
Tuesday 1st October
Wednesday 2nd October
Thursday 3rd October
Monday 14th October
Tuesday 15th October
Wednesday 16th October
Thursday 17th October
Friday 18th October
Saturday 19th October
Tuesday 22nd October
Thursday 24th October
Friday 25th October
Saturday 26th October
Sunday 27th October
TARGET
June
Monday 10th June
Tuesday 11th June
Wednesday 12th June
Tuesday 18th June
Monday 24th June
July
Monday 1st July
Tuesday 2nd July
Wednesday 3rd July
Thursday 4th July
Friday 5th July
Tuesday 16th July
Wednesday 17th July
Thursday 18th July
Friday 19th July
Wednesday 24th July
August
Monday 5th August
Tuesday 6th August
Monday 19th August
Tuesday 20th August
Wednesday 21st August
Thursday 22nd August
Friday 23rd August
Saturday 24th August
Sunday 25th August
Monday 26th August
Tuesday 27th August
Wednesday 28th August
Thursday 29th August
Friday 30th August
September
Monday 9th September
Tuesday 10th September
Monday 16th September
Tuesday 17th September
Wednesday 18th September
Thursday 19th September
Saturday 21st September
Sunday 22nd September
Friday 27th September
Saturday 28th September
Sunday 29th September
Monday 30th September
October
Tuesday 1st October
Wednesday 2nd October
Thursday 3rd October
INFILTRATE
July
Monday 22nd July
Tuesday 23rd July
Wednesday 24th July
Thursday 25th July
Friday 26th July
Monday 29th July
Tuesday 30th July
Wednesday 31st July
August
Thursday 1st August
Friday 2nd August
Monday 5th August
Tuesday 6th August
Wednesday 7th August
Thursday 8th August
Monday 12th August
Tuesday 13th August
Wednesday 14th August
Thursday 15th August
Friday 16th August
Saturday 17th August
Sunday 18th August
Monday 19th August
Tuesday 20th August
Wednesday 21st August
Thursday 22nd August
Monday 26th August
Tuesday 27th August
Wednesday 28th August
Thursday 29th August
Friday 30th August
Saturday 31st August
September
Monday 2nd September
Tuesday 3rd September
Wednesday 4th September
Thursday 5th September
Friday 6th September
Saturday 7th September
Monday 9th September
Tuesday 10th September
Wednesday 11th September
Thursday 12th September
Friday 13th September
Saturday 14th September
Sunday 15th September
Monday 16th September
Tuesday 17th September
Wednesday 18th September
Friday 20th September
October
Tuesday 15th October
Wednesday 16th October
Thursday 17th October
Friday 18th October
Saturday 19th October
Sunday 20th October
Monday 21st October
Tuesday 22nd October
Saturday 26th October
Sunday 27th October
Monday 28th October
SABOTAGE
June
Monday 24th June
Tuesday 25th June
Friday 28th June
July
Tuesday 2nd July
Wednesday 3rd July
Thursday 4th July
Monday 15th July
Tuesday 16th July
Wednesday 17th July
Thursday 18th July
Friday 19th July
Saturday 20th July
Monday 22nd July
Tuesday 23rd July
Wednesday 24th July
Thursday 25th July
Friday 26th July
Tuesday 30th July
Wednesday 31st July
August
Thursday 1st August
Monday 12th August
Tuesday 13th August
Wednesday 14th August
Thursday 15th August
Friday 16th August
Saturday 17th August
Monday 19th August
Tuesday 20th August
Wednesday 21st August
Thursday 22nd August
Friday 23rd August
Monday 26th August
Tuesday 27th August
Wednesday 28th August
Thursday 29th August
Friday 30th August
September
Tuesday 3rd September
Wednesday 4th September
Thursday 5th September
Saturday 7th September
Tuesday 10th September
Wednesday 11th September
Saturday 14th September
Sunday 15th September
Wednesday 18th September
Friday 20th September
Detect
ASDF
Saturday 13th July
PCR circular and linear, Conjugation of XL-10 (with sSP011), Patches, Digestion M13 backbone, RFP insert, Gel of PCR
PCR circular and linear, 40.5°C, 60° of pSB1A3, pSB1C3
Reagent
Volume
1x
Nuclease-free water
37.25 ul
5x Phusion HF Buffer
10 ul
10 mM dNTPs
1 ul
Forward Primer (10 uM)
0.5 ul
Reverse Primer (10 uM)
0.5 ul
Template Plasmid
0.25 ul
Phusion DNA Polymerase
0.5 ul
Total Volume
50 ul
Thermocycler Protocol: NEB Phusion
Temp
Time
Start
98°C
30 sec
Melt
Cycle 1
98°C
5 sec
Melt
35 cycles
Cycle 2
40.5°C / 60°C
25 sec
Anneal
Cycle 3
72°C
5 min
Extend
Finish
72°C
5 min
Extend
Store
10°C
Forever
Store
Conjugation of XL-10 (with sSP011)
1)   From O/N cultures Dilute strains 1/100 in LB
2)   Wait for OD to reach O,2
3)   Prepare  tube (in BD tubes) :
-       Tube = 0,5mL LB with Strain (sSP011) + 0,5mL LB with Strain (XL-10 Kan)
4)   Incubate 2 hours at 37°C (actually not in the shaker, but we accidently kept them in the shaker...)
5)   Plate 20ul  for mixed tube on LB antiobiotics (Tet, Kan)
6)   Incubate overnight at 37°C
Patches
check and make new ones
Digestion M13 backbone, RFP insert
for Backbone (M13mp18 plasmid): 3ug
7,58 ul plasmid (c=395ng/ul)
3 ul EcoRI
3 ul PstI
3ul 10x Fast Digest
13,42 ul H20
incubate for 12 min on 37°
heat inactivation: 80° 5 min
for insert (BBa_J04450): 5ug
31,4 ul plasmid
5 ul EcoRi
5 ul PstI
3,6 ul H20
incubate for 20 min at 37°
heat inactivation: 80° 5min
Gel of PCR (Amp 40.5 circ, lin, Chl 60° lin, circ)
100V, 20 min 1% gel
Detect
ASDF
Sunday 14th July
Gel of PCR, Gel Extraction, DpnI digest of PCR products and PCR purification
Gel of PCR (Amp 60° circ, lin, Chl 40.5 circ, lin)
100V, 20 min 1% gel
Gel of Digest
100V, 20 min 1% gel
Gel Extraction
Excise the DNA fragment from the agarose gel with a clean, sharp scalpel. Minimize the size of the gel slice by removing extra agarose.
Weigh the gel slice in a colorless tube. Add 3 volumes of Buffer QG to 1 volume of gel (100 mg ~ 100 μl). To help dissolve gel, mix by vortexing the tube every 2–3 min during the incubation. After the gel slice has dissolved completely, check that the color of the mixture is yellow (similar to Buffer QG without dissolved agarose).
Add 1 gel volume of isopropanol to the sample and mix (actually only needed for very small products or very big products)
To bind DNA, apply the sample to the QIAquick column, and centrifuge for 1 min.
Discard flow-through and place QIAquick column back in the same collection tube.
To wash, add 0.75 ml of Buffer PE to QIAquick column and centrifuge for 1 min.
11. Discard the flow-through and centrifuge the QIAquick column for an additional 1 min at 17,900 x g (13,000 rpm). 12. Place QIAquick column into a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 30 μl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge for 1 min.
DpnI digest of PCR products:
SPCR4 (40ul) - add 1 ul DpnI
SPCR 7 (150ul) - add 3 ul of DpnI
SPCR8 (60 ul) - add 3 ul of DpnI
SPCR 9 (150ul) - add 3 ul of DpnI
SPCR10 (110ul) - add 3 ul of DpnI
Incubate for 15 min at 37°C
PCR purification
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 50 μl Buffer EB  to the center of the QIAquick membrane and centrifuge the column for 1 min.
Detect
ASDF
Monday 15th July
Plating and control of antibiotic resistance.
We made a 1:1000 dilution of KAN (stand.c) in an LB solid media.
Pouring the plates. Around 20 mL/plate
1. KEIOΔPYR KANR Growth
2. ME1655 KANR No growth
Conclusion :
The KEIOΔPYRF growed as expected. Negative control did not growth.
Plates have KAN antibiotic
KEIO has KAN R.
Detect
ASDF
Tuesday 16th July
We prepared a colony PCR to send the KAN for sequencing, to know exactly the sequence of the KAN.
KEIOΔMPYRF has a deletion of PYRF to be replaced by KAN
We pitched 4 singles colonies into 50µL of H2O
Boil 5 minutes at 95°C
1,5 µL of this can be used directly for PCR
PCR reaction
Keep all the regents at 4°C while preparing the mixture
Reagent
Volume
JW 182 (10 uM)
JW 183 (10 uM)
Template DNA
Quick-load Tag 2x Master Mix
Nuclease free water
0,5 µL
0,5 µL
1,5 µL
12,5 µL
10 µL
Total volume
25 µL
Thermocycler Protocol : NEB Quick-Load
Temperature
Time
Start
Cycle1
Cycle 2
Cycle 3
Finish
Store
95°C
95°C
50°C
68°C
68°C
10°C
30 seconds
15 seconds
30 seconds
1 minute/kB
5 minutes
Forever
melt
melt
anneal
extend - 35 cycles
extend
store
Gel electrophoresis
We do it to prove that our colony PCR was successful. We expect a band at about 800 base pairs.
Our gel is 1%, therefore we used 0,5g agarose in 100µL TAE buffer.
As a ladder, we used a 1kB plus gene ruler of fermentor.
We l the gel with 5 µL sample ans we kept the sample at 4°C.
PICTURE
We see that the band are as expected. THis is an indicator that KAN is at the right place. The sequence has then been sent for sequencing
Transformation of PAUC 18 into a NEBΔturbo clearing cells
We are transforming PAUC 18 that consist in a low ORI and ampicillin resistance
We will use commercialized NEB turbo competent cells as well as freshly made chemical competent cells
We did this to check the competency of our fresh competent cells
NOTE : EVERYTHING HAS TO BE KEPT ON ICE AND NO VORTEX
Throw competent cells on ice. Those can be prepared using the CaCl2 protocol
Place 20 µL of cells in a pre-chilled Eppendorf tube
For an intert vector, add 0,5 µL or less to the chilled cells
For a ligation product, add 2-3 µL to the chilled cells
Mix gently by flicking the tube
Chill on ice for 10 minutes - this step is optional but can improve yields when transforming a ligation product
Heat shock at 42°C for 30 seconds
Return on ice for 2 minutes
Add 200µL LB medium and recover the cells by shaking at 37°C
Another rich medium can substitute for the recovery. The recovery time varies with the antibiotic selection
Ampicillin : 15 - 30 minutes
Place out the cells on selective LB
Use glass beads to spread the cells. The volume of cells plated depends on what is being transformed
For an intact vector
High transformation efficiencies are expected. Plating out 10 µL of recovered cells should produce many colonies.
Note : 200 µL is the maximum volume of liquid that an LB plate can absorb
Incubate at 37°C. Transformants should appear within 12 hours
Conclusion
The BL 21 DE3 strain showed IP colonies
For the new NEB turbo cells, we see no colony
The old NEB turbo cells worked fine
Detect
ASDF
Wednesday 17th of July
PCR purification
We are purifying the colony PCR from yesterday to send it away for sequencing
Procedure
Add 1.1 volume and Binding Buffer
Transfer up to 800 µL. Centrifuge for 30-60 seconds. Descend the flow through
Add 700 µL of washed Buffer. Centrifuge for 30-60 seconds
+ 1 min centrifuge to completely remove any residual wash buffer
Add 50 µL of Elution Buffer to the center of the Gene JT purification column membrane and centrifuge for 1 minute
Discard the Gene JET purification column and store the purified DNA at -20°C.
Detect
ASDF
Thursday 18th July
Calculating the transformation efficiency for the E.coli NEB and E.coli NNEB strains.
·         20 µL of cell culture was inoculated in 200 µL of LB, where 0.5 µL of DNA sample (?) was added, giving the total volume of 220.5 µL.
·         After the heat shock transformation 10 µL of the culture was diluted in 90 µL of LB,  giving the 10-1 dilution (d); then 10 µL of this dilution was transferred in 90 µL of LB, giving the 10-2 dilution. 10 µL of each dilution (v) was plated on ampicillin media and left for incubation (24 h, 37  ̊C).
·         After incubation, colonies were counted:
NEB
NNEB
10-¹
0
433
0
368
Average
0
400.5
10-¹
0
62
0
40
Average
0
51
Obviously, NEB strain was not transformed and only NNEB was included in further calculations.
·         The number of the transformed colony forming units per mL of the culture was calculated following the formula: colony number / d*v.
Calculation from the 10-1 dilution: 0.4 * 109 CFU/mL
Calculation from the 10-2 dilution: 0.51 * 109 CFU/mL
·         The transformation efficiency was calculated by dividing the number of transformed colony forming units (CFU) by the amount of the DNA used (mDNA).
CFU1 = 0.4 * 109 CFU/mL  *  0.22 mL (total V of the transformation culture) = 0.088*109 CFU
CFU2 = 0.51 * 109 CFU/mL * 0.22 mL = 0.1122*109 CFU
The concentration of the DNA solution which was used was 0.323 µg/µL, while the total DNA mass which was added to the transformation culture was: 0.323 µg/µL * 0.5 µL = 0.1615 µg.
So the transformation efficiency, calculated from the 10 times diluted (TE1) and 100 times diluted (TE2) cultures is:
TE1 = CFU1/mDNA = 0.55 * 109 CFU/µg
TE2 = CFU2/mDNA = 0.69 * 109 CFU/µg
TE1 and TE2 are almost the same
Detect
ASDF
Monday 22nd July
Culture purification by streak plate method and preparing the glycerol stock for the sSP001 (TN 57) and sSP002 (NEC) strains.
Place your note here
Detect
ASDF
Tuesday 23rd July
Colony PCR
·         Strain: Keio ΔpyrF
·         Primers: JW182 and JW183
·         Amplified region: kanamycine resistance gene
Detect
ASDF
Monday 29th July
Liquid culture of:
pCOLA DUET (from glycerol stock of Sabotage Group) - 20 ml culture
pSSB1 (from plate) - 5 ml culture
pSSB2 (from plate ) - 5 ml culture
Plating of addgene bacteria containing the plasmids we want:
streak out bacteria from addgene to get single colonies:
pBAC-BA-lacZ
pCRISPR
pCas9
pCRIPSR::rpsL
Detect
ASDF
Tuesday 30th July
Liquid culture, Miniprep and Glycerol stock
Liquid culture of:
pCOLA DUET (from liquid culture (29.07.13)) - 20 ml culture
pSSB1 (from liquid culture (29.07.13)) - 5ml culture
pSSB2 (from liquid culture (29.07.13)) - 5 ml culture
pBAC-BA-lacZ (from plate ) - 5 ml culture
pCRISPR (from plate ) - 5 ml culture
pCas9 (from plate ) - 5 ml culture
pCRIPSR::rpsL (from plate ) - 5 ml culture
F+ strain - (from glycerol stock Sabotage) - 10 ml culture
Keio Keio delta pyrF (from plate ) - 10 ml culture
Miniprep (pCOLA-DUET = sSP007) using the Thermo Scientific Miniprep Kit
1) Pellet 2x9 ml of liquid culture (4000 rpm, 10 min)
2) Discard supernatant
3) resuspend the cells in 250 µL of resuspension solution
4) add 250 µL of lysis solution, mix by inverting 4-6 times
5) add 350 µL of neutralization solution
4) centrifuge for 5 min
5) transfer supernatant to spin column
6) centrifuge for 1 min
7) discard flow through
8) add 500 µL wash solution and centrifuge for 1 min , discard flow through(repeat this step)
9) centrifuge for 1 min to remove left over liquid
10) transfer the column on a 1.5 ml tube
11) add 50 µL of elution buffer and incubate for 2 min
12) centrifuge for 2 min
13) Nanodrop the concentration and freeze at -20°
pSP001= pCOLA-DUET: 65 ng/ul
Glycerol Stock
from overnight culture of MG1655-pCOLA-DUET (sSP007)
Centrifuge 4000 rpm, 10 minutes,
take out liquid
resuspend cells in 0,5 mL glycerol (60%) , 2 mL LB
freeze in -80°C
Detect
ASDF
Wednesday 31st July
Conjugation, Miniprep addgene plasmids,Glycerol Stock of addgene plasmids
Conjugation
sSP001, SP002, keio Delta PyrF
F+ comes from XL1 Blue (glycerol stock Sabotage) => F+ is tetR
Protocol :
1)   From O/N cultures Dilute strains 1/100 in LB
2)   Wait for OD to reach O,2
3)   Prepare 4 tubes (in BD tubes) :
-       Tube 1 = 0,5 mL LB with Strain 1 (sSP001) ,5 mL LB = control
-       Tube 2 = 0,5 mL LB with Strain 2 (sP001) + 0,5 mL LB = control
-       Tube 3 = 0,5 mL LB with Strain 3 (keio Delta pyrF) + 0,5 mL LB = control
-       Tube 4 = 0,5 mL LB with Strain 4 (XL1) + 0,5 mL LB = control
-       Tube 5 = 0,5 mL LB with Strain 1 (sSP001) + 0,5 mL LB with Strain 4 (XL1)
-       Tube 6 = 0,5 mL LB with Strain 2 (sSP002) + 0,5 mL LB with Strain 4 (XL1)
-       Tube 7 = 0,5 mL LB with Strain 3 (keio Delta pyrF) + 0,5 mL LB with Strain 4 (XL1)
4)   Incubate 2 hours at 37°C (actually not in the shaker, but we accidently kept them in the shaker...)
5)   Plate 10ul for controls, 100uL, 10ul  for mixed tubes on LB antibiotics (Tube 5: Chl + Tet, Tube 6: Chl + Tet, Tube 7: Kan + Tet)
6)   Incubate overnight at 37°C
Miniprep addgene plasmids
1) Pellet  4 ml of liquid culture (4000 rpm, 10 min)
2) Discard supernatant
3) resuspend the cells in 250 µL of resuspension solution
4) add 250 µL of lysis solution, mix by inverting 4-6 times
5) add 350 µL of neutralization solution
4) centrifuge for 5 min
5) transfer supernatant to spin column
6) centrifuge for 1 min
7) discard flow through
8) add 500 µL wash solution and centrifuge for 1 min , discard flow through(repeat this step)
9) centrifuge for 1 min to remove leftover liquid
10) transfer the column on a 1.5 ml tube
11) add 50 µL of elution buffer and incubate for 2 min
12) centrifuge for 2 min
13) Nanodrop the concentration and freeze at -20°
See Database for plasmid reference
pCas9:
pCRISPR:
pBac-LacZ:
pCRISPR::rpsL:
Glycerol Stock of addgene plasmids
(see Database for strain reference)
from overnight culture
Centrifuge 4000 rpm, 10 minutes,
take out liquid
resuspend cells in 0,5 mL glycerol (60%) , 2 mL LB
freeze in -80°C
Detect
ASDF
Thursday 1st August
PCR (gradient + usual) of SPCR1, SPCR7, SPCR8, SPCR10, SPCR9, SPCR4
Reagent
Volume
1x
Nuclease-free water
37.25 µL
5x Phusion HF Buffer
10 µL
10 mM dNTPs
1 µL
Forward Primer (10 µM)
0.5 µL
Reverse Primer (10 µM)
0.5 µL
Template Plasmid
0.25 µL
Phusion DNA Polymerase
0.5 µL
Total Volume
50 µL
Gradient:
Pos 1: 40°
Pos 2: 40,2°
Pos 3: 41,3°
Pos 4: 43,1°
Pos 5: 45,4°
Pos 6: 48°
Pos 7: 50,7°
Pos 8: 53,5°
Pos 9: 56,0°
Pos 10: 58,1°
Pos 11: 59,8°C
(Pos 12: 60,6°)
Usual PCR: 52°
Detect
ASDF
Friday 2nd August
Place your twit here
Gels with PCR products: 1% Agar
100V 20 Min
10 min staining in BET
Detect
ASDF
Monday 5th August
Place your twit here
5ml liquid culture with proper antibiotics of:
SSp001 + F+
sSP002 + F+
keio delta pyrF + F+
Gelelectrophoresis: 1% Agar, 100 V 20 min
Detect
ASDF
Tuesday 6th August
Liquid cultures of sSP008-sSP0011
Detect
ASDF
Wednesday 7th August
Glycerol Stock of sSP008-sSP0011, M13 Test, Single Colonies, Liquid cultures, PCR and Transformation
Glycerol Stock of sSP008-sSP0011
Take 1.5ml of ON culture + 500ul Glycerol (60%)
Freeze at -80°C
M13 Test
Plate from ON culture:
1) 1:1000 - sSP012: sSP009 (plate 200ul)
2) 200 ul sSP012
3) 200ul sSp009
Single Colonies
streak out ON culture of sSP012 to get single colonies
Liquid cultures of pir+ E.coli and pKD13 from Jake
PCR of SPCR4, SPCR7, SPCR8, SPCR9, SPCR10 from circuar backbone, directly taken from Biobrick plate
Protocol
Reagent
Volume
1x
Nuclease-free water
37,25µL
5x Phusion HF Buffer
10µL
10 mM dNTPs
1µL
Forward Primer (10 µM)
0.5µL
Reverse Primer (10 µM)
0.5µL
Template Plasmid
0.25µL
Phusion DNA Polymerase
0.5µL
Total Volume
50µL
Thermocycler Protocol: NEB Phusion
Temp
Time
Start
98°C
30 sec
Melt
Cycle 1
98°C
5 sec
Melt
35 cycles
Cycle 2
40.5°C / 60°C
25 sec
Anneal
Cycle 3
72°C
30 sec per kb
Extend
Finish
72°C
5 min
Extend
Store
10°C
Forever
Store
Transformation Biobricks BBa_K649304 (with pSB1C3 backbone) and BBa_J04450 (with pSB1A3 backbone) into NEB Turbo (heat shock trafo)
1) Thaw competent cells on ice.
2) Place 20 ul of cells in a pre-chilled Eppendorf tube.
3) Add 2.5 ul of plasmid (from Biobrick stock)
4) Mix gently by flicking the tube.
5) Chill on ice for 10 minutes.
4) Heat shock at 42 °C for 30 seconds
5) Return to ice for 2 minutes.
6) Add 200 ul LB medium and recover the cells by shaking at 37 °C.
Ampicillin: 15-30 minutes
Chloramphenicol: 60-120 minutes
7) Plate out the cells on selective LB (10ul)
8) Incubate at 37 °C. Transformants should appear within 12 hrs.
Result:  BBa_J04450: many colonies, BBa_K649304 no colonies, plate rest
Detect
ASDF
Thursday 8th August
Liquid culture of FR-433 (M13)
Gel of PCR products. 100V 20min 1%
4-7-8-9-10 (40.5°C)- 4-7-8-9-10 (60°C)
Detect
ASDF
Friday 9th August
Liquid culture, Gel,  Trafo of BBa_E1010 (pSB1C3 backbone) into NEB turbo, Glycerol Stock, Making electro competent cells of pir+ E.coli, Transforming pKD13 into pir+ E.coli, Miniprep NEB turbo with BBa-J04450:, Liquid culture and Patch plates.
Liquid culture
1:1000 from liquid: FR-433 (M13), 10 ml
pir+ E.coli: from plate, 25 ml
XL10: from plate 10 ml
Gel: rest of circular backbone PCR product, 1% Agar, 100V, 20 min
SPCR4-7-8-9-10 (40.5) SPCR4-7-8-9-10 (60.5)
Gel picture:
didn’t work
Trafo of BBa_E1010 (pSB1C3 backbone) into NEB turbo
1) Thaw competent cells on ice.
2) Place 20 µl of cells in a pre-chilled Eppendorf tube.
3) Add 2.5 µl of plasmid (from Biobrick stock)
4) Mix gently by flicking the tube.
5) Chill on ice for 10 minutes.
4) Heat shock at 42 °C for 30 seconds
5) Return to ice for 2 minutes.
6) Add 200 µl LB medium and recover the cells by shaking at 37 °C.
Chloramphenicol: 60-120 minutes
7) Plate out the cells on selective LB (10ul)
8) Incubate at 37 °C. Transformants should appear within 12 hrs.
Glycerol Stock FR-433 (M13) -> sSP012
Glycerol Stock pir+ -> sSP013
Glycerol Stock pKD13 -> sSP014
Glycerol Stock NEB turbo with BBa-J04450 -> sSP015
Take 1.5 ml of ON culture + 500ul Glycerol (60%)
Freeze at -80°C
Making electro competent cells of pir+ E.coli
Preparation of Electrocompetent Cells
Note: Competent cells should never be vortexed, as this will cause them to lyse and release salts into the media. Resuspend cells by pipetting up and down with a large pasteur pipet. Once they are chilled, cells should be continuously cold.
1) The night before the transformation, start an overnight culture of cells.
pir+ E.coli
2) The day of the transformation, dilute the cells 100X.
100 ml LB
Grow at 30°C for about 90 minutes.
3) When the cells reach an OD600 of 0.2.
4) Harvest the cells.
When the cells reach an OD600 of between 0.6 and 0.8.
Split the culture into 2x 50 ml falcon tubes, on ice.
Centrifuge at 4 °C for 10 min at 4000 rpm.
5) Wash and combine the cells.
Remove the supernatant.
Resuspend the cells in 2x 25 ml of ice cold water.
Combine the volumes in a single 50 ml falcon tube.
6) Wash the cells 2 more times.
Centrifuge at 4 °C for 10 min at 4000 rpm.
Resuspend in 50 ml of ice cold water.
Repeat.
7) Wash and concentrate the cells for electroporation.
Centrifuge at 4 °C for 10 min at 4000 rpm.
Resuspend in 1-2 ml of ice cold water.
We will use 200 µl of washed cells per transformation.
Transforming pKD13 into pir+ E.coli
1) Prepare BD tubes with a pipette filled with LB at the interior of each tube (pipette supplied with the electroporation cuvette)
2) Test the purity of the electrocompetent cells.
Add 200 ul of washed cells to a cuvette.
3) Mix the cells and DNA in a cuvette.
200 ul of washed cells with 200 ng of PCR product.
Keep the cuvette on ice until just before the electroporation.
4) Preload a pipette with 1 ml of LB.
5) Pulse the cuvette with voltage.
Dry the electrodes with a kimwipe prior to loading.
Use the EC2 setting.
6) Listen for arcing.
A cracking sound means all the cells are dead.
Note the time constant: 5 is good, 5.8 is great.
7) Immediately recover the cells.
Add the 1 ml of preloaded LB and pipet up and down to mix.
Collect 1 ml of cells, some volume is lost in the cuvette.
8) Incubate 2 h at 37 °C with shaking.
9) Plate 100 ul of recovered cells on selective plates.
Use antibiotic appropriate to the part being integrated.
Let the other 900 ul rest overnight at room temperature.
10) Concentrate and plate the remaining cells
Spin down quickly and resuspend in 100 ul LB before plating.
Transformed cells should be incubated at 37 °C.
Colonies should appear 24-48 h after plating.
Miniprep NEB turbo with BBa-J04450:
1) Pellet 2x 9 ml of liquid culture (4000rpm, 10 min)
2) Discard supernatant
3) resuspend the cells in 250 µl of resuspension solution
4) add 250 µl of lysis solution, mix by inverting 4-6 times
5) add 350 µl of neutralization solution
4) centrifuge for 5 min
5) transfer supernatant to spin column
6) centrifuge for 1 min
7) discard flow through
8) add 500 µl wash solution and centrifuge for 1 min , discard flow through(repeat this step)
9) centrifuge for 1 min to remove left over liquid
10) transfer the column on a 1.5 ml tube
11) add 50ul of elution buffer and incubate for 2 min
12) centrifuge for 2 min
13) Nanodrop the concentration and freeze at -20°
J04450: 0ng/ul
Liquid culture  of: sSP012
sSP008
sSP009
sSP010
sSP011
sSP001
sSP002
XL-10 = sSP016
sSP017= Litmus
Patch plates of:
sSP010
sSP013
sSP009s
sSP014
sSP012
Detect
ASDF
Saturday 10th August
Test of M13 and Liquid culture
Test of M13
1) 100 μ l of plating bacteria per tube
2) Prepare tenfold serial dilutions (10-6  to 10-9 ) of the bacteriophage stock in LB
3) Put 10 μ l/ 100 μ l of each dilution to the bacteria
4)  Mix the bacteriophage particles with the bacterial culture by vortexing gently.
5) Add 40 μ l of X-gal solution (working conc) and IPTG (working conc) solution to each of the tubes
6) Add 3ml of Agar to each tube and gently vortexing for 3 seconds
7) Pour the mixture onto  plates containing LB medium supplemented with 5 mM MgCl2
8) Swirl the plate gently to ensure an even distribution of bacteria and agar.
9) Incubate them at 37°C.
Liquid culture:
E1010
J04550
Detect
ASDF
Sunday 11th August
Patches of sSP001-sSP008, sSP015-18 and Stress experiment (Plate reader)
Patches of sSP001-sSP008, sSP015-18
Stress experiment (Plate reader)
sSP001, sSP002, sSP010, sSP011, sSP012 (M13)
1:1000 bacteria in M9 Glucose
Mitomycin C concentrations: 3,3*10^-3, 1,6*10°-3, 6,6*10°-4, 3,3*10°-4
Quadruplets of bacteria (+phage), Triplets of MMC
See also pipetting scheme in folder Experiments on Dropbox
On culture of FR-433, XL-10, delta pyrF
1:1000 culture at 30°
Detect
ASDF
Monday 12th August
Glycerol Stock, Miniprep, M13 plate test, Patches
Glycerol Stock sSP018 - NEB with E1010
sSP017
sSP016
Take 1.5ml of ON culture + 500ul Glycerol (60%)
Freeze at -80°C
Miniprep
E1010
J04450
1) Pellet 2x 2ml of liquid culture (4000rpm, 10 min)
2) Discard supernatant
3) resuspend the cells in 250ul of resuspension olution
4) add 250ul of lysis solution, mix by inverting 4-6 times
5) add 350ul of neutralization solution
4) centrifuge for 5 min
5) transfer supernatant to spin column
6) centrifuge for 1 min
7) discard flow through
8) add 500 ul wash solution and centrifuge for 1 min , discard flow through(repeat this step)
9) centrifuge for 1 min to remove left over liquid
10) transfer the column on a 1.5ml tube
11) add 50ul of elution buffer and incubate for 2 min
12) centrifuge for 2 min
13) Nanodrop the concentration and freeze at -20°
J04450: ng/ul
E1010: ng/ul
M13 plate test
1) 100 μ l of plating bacteria per tube
2) Prepare tenfold serial dilutions (10-6  to 10-9 ) of the bacteriophage stock in LB
3) Put 10 μ l/ 100 μ l of each dilution to the bacteria
4)  Mix the bacteriophage particles with the bacterial culture by vortexing gently.
5) Add 40 μ l of X-gal solution (working conc) and IPTG (working conc) solution to each of the tubes
6) Add 3ml of Agar to each tube and gently vortexing for 3 seconds
7) Pour the mixture onto  plates containing LB medium supplemented with 5 mM MgCl2
8) Swirl the plate gently to ensure an even distribution of bacteria and agar.
9) Incubate them at 37°C.
Patches
check them and prepare new ones of those that haven’t been done yet
Detect
ASDF
Tuesday 13th August
PCR circular and linear 40.5°C/ 60°C, Conjugation of XL-10 (with sSP011), Patches, Digestion M13 backbone, RFP insert and Gel of PCR
PCR circular and linear, 40.5°C, 60° of pSB1A3, pSB1C3
Reagent
Volume
1x
Nuclease-free water
37,25µL
5x Phusion HF Buffer
10µL
10 mM dNTPs
1µL
Forward Primer (10 µM)
0.5µL
Reverse Primer (10 µM)
0.5µL
Template Plasmid
0.25µL
Phusion DNA Polymerase
0.5µL
Total Volume
50µL
Thermocycler Protocol: NEB Phusion
Temp
Time
Start
98°C
30 sec
Melt
Cycle 1
98°C
5 sec
Melt
35 cycles
Cycle 2
40.5°C / 60°C
25 sec
Anneal
Cycle 3
72°C
30 sec per kb
Extend
Finish
72°C
5 min
Extend
Store
10°C
Forever
Store
Conjugation of XL-10 (with sSP011)
1)   From O/N cultures Dilute strains 1/100 in LB
2)   Wait for OD to reach O,2
3)   Prepare  tube (in BD tubes) :
-       Tube = 0,5mL LB with Strain (sSP011) + 0,5mL LB with Strain (XL-10 Kan)
4)   Incubate 2 hours at 37°C (actually not in the shaker, but we accidently kept them in the shaker...)
5)   Plate 20ul  for mixed tube on LB antiobiotics (Tet, Kan)
6)   Incubate overnight at 37°C
Patches
check and make new ones
Digestion M13 backbone, RFP insert
for Backbone (M13mp18 plasmid): 3ug
7,58 ul plasmid (c=395ng/ul)
3 ul EcoRI
3 ul PstI
3ul 10x Fast Digest
13,42 ul H20
incubate for 12 min on 37°
heat inactivation: 80° 5 min
for insert (BBa_J04450): 5ug
31,4 ul plasmid
5 ul EcoRi
5 ul PstI
3,6 ul H20
incubate for 20 min at 37°
heat inactivation: 80° 5min
Gel of PCR (Amp 40.5 circ, lin, Chl 60° lin, circ)
100V, 20 min 1% gel
Detect
ASDF
Wednesday 14th August
Gel of PCR, Gel of Digest, Gel Extraction, DpnI digest of PCR products, PCR purification
Gel of PCR (Amp 60° circ, lin, Chl 40.5 circ, lin) 100V, 20 min 1% gel
Gel of Digest
100V, 20 min 1% gel
Gel Extraction
Excise the DNA fragment from the agarose gel with a clean, sharp scalpel. Minimize the size of the gel slice by removing extra agarose.
Weigh the gel slice in a colorless tube. Add 3 volumes of Buffer QG to 1 volume of gel (100 mg ~ 100 μl). To help dissolve gel, mix by vortexing the tube every 2–3 min during the incubation.
After the gel slice has dissolved completely, check that the color of the mixture is yellow (similar to Buffer QG without dissolved agarose).
Add 1 gel volume of isopropanol to the sample and mix (actually only needed for very small products or very big products)
To bind DNA, apply the sample to the QIAquick column, and centrifuge for 1 min.
Discard flow-through and place QIAquick column back in the same collection tube.
To wash, add 0.75 ml of Buffer PE to QIAquick column and centrifuge for 1 min.
11. Discard the flow-through and centrifuge the QIAquick column for an additional 1 min at 17,900 x g (13,000 rpm). 12. Place QIAquick column into a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 30 μl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge for 1 min.
DpnI digest of PCR products:
SPCR4 (40ul) - add 1 ul DpnI
SPCR 7 (150ul) - add 3 ul of DpnI
SPCR8 (60 ul) - add 3 ul of DpnI
SPCR 9 (150ul) - add 3 ul of DpnI
SPCR10 (110ul) - add 3 ul of DpnI
Incubate for 15 min at 37°C
PCR purification
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 50 μl Buffer EB  to the center of the QIAquick membrane and centrifuge the column for 1 min.
Detect
ASDF
Thursday 15th August
Liquid cultures
Liquid cultures:
sSp009
sSP012
sSP016
sSp020
Xgal Bacteria from Aude
Detect
ASDF
Friday 16th August
Liquid M13 Test, Glycerol Stock, Miniprep
Liquid M13 Test
OD (sSP020=XL10 KanR with F+)=0,557
OD (sSP016=XL10 KanR)=0,515
OD (sSP009= delta pyrF with F+)=0,561
OD(sSP012=FR-433=M13)=0,695
Prepare Falcons containing 500ul of each strain, adding 500ul of 10^-6 dilution of FR-433
add 1ul of IPTG and 1ul of Xgal
Incubate at 37° on shaker
Glycerol Stock sSP019 (pir+ E.coli transformed with pKD13)
sSP020 (XL10 KanR with F+)
Miniprep
sSP019
sSP017
1) Pellet 2x 2ml of liquid culture (4000rpm, 10 min)
2) Discard supernatant
3) resuspend the cells in 250ul of resuspension olution
4) add 250ul of lysis solution, mix by inverting 4-6 times
5) add 350ul of neutralization solution
4) centrifuge for 5 min
5) transfer supernatant to spin column
6) centrifuge for 1 min
7) discard flow through
8) add 500 ul wash solution and centrifuge for 1 min , discard flow through(repeat this step)
9) centrifuge for 1 min to remove left over liquid
10) transfer the column on a 1.5ml tube
11) add 50ul of elution buffer and incubate for 2 min
12) centrifuge for 2 min
13) Nanodrop the concentration and freeze at -20°
pKD13: 311ng/ul
Litmus 28i: 93ng/ul
Detect
ASDF
Monday 19th August
Restreak XL-1 Blue Litmus28i, PCR, PCR purification Backbone (M13mp18), Ligation (RFP into M13mp18 backbone), Trafo of Ligation product (RFP in M13mp18 plasmid) into NEB turbo and Test Xgal
Restreak XL-1 Blue Litmus28i
PCR
SPCR5, SPCR6, 49,3°C
Reagent
Volume
1x
Nuclease-free water
37,25µL
5x Phusion HF Buffer
10µL
10 mM dNTPs
1µL
Forward Primer (10 µM)
0.5µL
Reverse Primer (10 µM)
0.5µL
Template Plasmid
0.25µL
Phusion DNA Polymerase
0.5µL
Total Volume
50µL
Thermocycler Protocol: NEB Phusion
Temp
Time
Start
98°C
30 sec
Melt
Cycle 1
98°C
5 sec
Melt
35 cycles
Cycle 2
40.5°C / 60°C
25 sec
Anneal
Cycle 3
72°C
30 sec per kb
Extend
Finish
72°C
5 min
Extend
Store
10°C
Forever
Store
PCR purification Backbone (M13mp18)
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 50 μl Buffer EB  to the center of the QIAquick membrane and centrifuge the column for 1 min.
Ligation (RFP into M13mp18 backbone)
3ul Backbone
14 ul Insert
2ul 10x T4 DNA Ligase HC Buffer
1 ul T4 DNA Ligase
Incubate for at least 10min at 22°C
Trafo of Ligation product (RFP in M13mp18 plasmid)  into NEB turbo
1) Thaw competent cells on ice.
2) Place 20 ul of cells in a pre-chilled Eppendorf tube.
3) Add 3 ul of plasmid (from Biobrick stock)
4) Mix gently by flicking the tube.
5) Chill on ice for 10 minutes.
4) Heat shock at 42 °C for 30 seconds
5) Return to ice for 2 minutes.
6) Add 200 ul LB medium and recover the cells by shaking at 37 °C.
7) Plate out the cells on LB (200ul)
8. Incubate at 37 °C. Transformants should appear within 12 hrs.
Test Xgal
3ml of Liquid culture with
1) Xgal from iGEM 2012 (Jake) (+IPTG)
2) Xgal from last week (in DMF) (+IPTG)
Detect
ASDF
Tuesday 20th August
Gel of PCR and Xgal Test
Gel of PCR
1%, 100V, 20min
Result: PCR didn’t work
Xgal Test
MG1655 (of Jake) in liquid culture with Xgal of iGEM 2012, Aude and new one (Anne)
Result:
All cultures turn blue (Aude, less blue)
Detect
ASDF
Wednesday 21st August
Gradient PCR SPCR5, SPCR6, M13 Test, Electrocompetent Cells (sSP017-XL-1 Blue, Litmus 28i), Transformation
Gradient PCR SPCR5, SPCR6
Reagent
Volume
1x
Nuclease-free water
37,25µL
5x Phusion HF Buffer
10µL
10 mM dNTPs
1µL
Forward Primer (10 µM)
0.5µL
Reverse Primer (10 µM)
0.5µL
Template Plasmid
0.25µL
Phusion DNA Polymerase
0.5µL
Total Volume
50µL
Gradient:
Pos 1: 40.1°
Pos 2: 40,8°
Pos 3: 42.2°
Pos 4: 44.2°
Pos 5: 46.7°
Pos 6: 49.4°
Pos 7: 52.1°
Pos 8: 54.7°
Pos 9: 57.1°
Pos 10: 59.0°
Pos 11: 60.2°C
M13 Test
Liquid cultures of sSP012 and sSP020
1) 100 μ l of plating bacteria per tube
2) Prepare tenfold serial dilutions (10-6  to 10-9 ) of the bacteriophage stock in LB
3) Put 10 μ l/ 100 μ l of each dilution to the bacteria
4)  Mix the bacteriophage particles with the bacterial culture by vortexing gently.
5) Add 40 μ l of X-gal solution (working conc) and IPTG (working conc) solution to each of the tubes
6) Add 3ml of Agar to each tube and gently vortexing for 3 seconds
7) Pour the mixture onto  plates containing LB medium supplemented with 5 mM MgCl2
8) Swirl the plate gently to ensure an even distribution of bacteria and agar.
9) Incubate them at 37°C.
Electrocompetent Cells (sSP017-XL-1 Blue, Litmus 28i)
1) The night before the transformation, start an overnight culture of cells.
sSP017 (XL1 Blue Litmus 28i)
2) The day of the transformation, dilute the cells 100X.
100 ml LB Amp.
Grow at 30°C for about 90 minutes.
3) When the cells reach an OD600 of 0.2.
4) Harvest the cells.
When the cells reach an OD600 of between 0.6 and 0.8 (OD: 0,683)
Split the culture into 2x 50 ml falcon tubes, on ice.
Centrifuge at 4 °C for 10 min at 4000 rpm.
5) Wash and combine the cells.
Remove the supernatant.
Resuspend the cells in 2x 25 ml of ice cold water.
Combine the volumes in a single 50 ml falcon tube.
6) Wash the cells 2 more times.
Centrifuge at 4 °C for 10 min at 4000 rpm.
Resuspend in 50 ml of ice cold water.
Repeat.
7) Wash and concentrate the cells for electroporation.
Centrifuge at 4 °C for 10 min at 4000 rpm.
Resuspend in 1-2 ml of ice cold water.
We will use 200 ul of washed cells per transformation.
Transforming pKD13 into pir+ E.coli
1) Prepare BD tubes with a pipette filled with LB at the interior of each tube (pipette supplied with the electroporation cuvette)
2) Test the purity of the electrocompetent cells.
Add 200 ul of washed cells to a cuvette.
3) Mix the cells and DNA in a cuvette.
200 ul of washed cells with 200 ng of PCR product.
Keep the cuvette on ice until just before the electroporation.
4) Preload a pipette with 1 ml of LB.
5) Pulse the cuvette with voltage.
Dry the electrodes with a kimwipe prior to loading.
Use the EC2 setting.
6) Listen for arcing.
A cracking sound means all the cells are dead.
Note the time constant: 5 is good, 5.8 is great.
7) Immediately recover the cells.
Add the 1 ml of preloaded LB and pipet up and down to mix.
Collect 1 ml of cells, some volume is lost in the cuvette.
8) Incubate 2 h at 37 °C with shaking.
9) Plate 100 ul of recovered cells on selective plates.
Use antibiotic appropriate to the part being integrated.
Let the other 900 ul rest overnight at room temperature.
10) Concentrate and plate the remaining cells
Spin down quickly and resuspend in 100 ul LB before plating.
Transformed cells should be incubated at 37 °C.
Colonies should appear 24-48 h after plating.
Detect
ASDF
Thursday 22nd August
Gel of gradient PCR product (Pictures 6,5,5), PCR purification and pooling, Digestion M13 backbone, RFP insert, Test Digestion in Gel, Gel Extraction, Liquid culture, Purification of Backbone Cloning and Miniprep sSP015.
Gel of gradient PCR product
1%, 100V, 20 min
PCR purification and pooling
Pool all samples that worked
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 50 μl Buffer EB  to the center of the QIAquick membrane and centrifuge the column for 1 min.
c(SPCR5)= ng/ul
c(SPCR6)= ng/ul
Digestion M13 backbone, RFP insert
Backbone/Insert (double the amount)
Regent
Volume
Purified Plasmid
20 ul
H2O
65 ul
10x FastDigest Buffer
10 ul
FastDigest Enzyme 1 (EcoRI)
2,5 ul
FastDigest Enzyme 2 (PstI)
2,5 ul
FastAP Phosphatase
0 ul
Total Volume
100ul
Digest for 2.5  hours at 37 °C with shaking.
Test Digestion in Gel
1%, 100V, 20 min
Gel Extraction
Excise the DNA fragment from the agarose gel with a clean, sharp scalpel. Minimize the size of the gel slice by removing extra agarose.
Weigh the gel slice in a colorless tube. Add 3 volumes of Buffer QG to 1 volume of gel (100 mg ~ 100 μl). To help dissolve gel, mix by vortexing the tube every 2–3 min during the incubation.
After the gel slice has dissolved completely, check that the color of the mixture is yellow (similar to Buffer QG without dissolved agarose).
Add 1 gel volume of isopropanol to the sample and mix (actually only needed for very small products or very big products)
To bind DNA, apply the sample to the QIAquick column, and centrifuge for 1 min.
Discard flow-through and place QIAquick column back in the same collection tube.
To wash, add 0.75 ml of Buffer PE to QIAquick column and centrifuge for 1 min.
11. Discard the flow-through and centrifuge the QIAquick column for an additional 1 min at 17,900 x g (13,000 rpm). 12. Place QIAquick column into a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 30 μl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge for 1 min.
c=13,6ng/ul
Liquid culture
sSP015
Purification of Backbone Cloning
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 30 μl Buffer EB  to the center of the QIAquick membrane and centrifuge the column for 1 min.
Miniprep sSP015
1) Pellet 2x 2ml of liquid culture (4000rpm, 10 min)
2) Discard supernatant
3) resuspend the cells in 250ul of resuspension solution
4) add 250ul of lysis solution, mix by inverting 4-6 times
5) add 350ul of neutralization solution
4) centrifuge for 5 min
5) transfer supernatant to spin column
6) centrifuge for 1 min
7) discard flow through
8) add 500 ul wash solution and centrifuge for 1 min , discard flow through(repeat this step)
9) centrifuge for 1 min to remove left over liquid
10) transfer the column on a 1.5ml tube
11) add 50ul of elution buffer and incubate for 2 min
12) centrifuge for 2 min
13) Nanodrop the concentration and freeze at -20°
Detect
ASDF
Friday 23rd August
Stress Experiment (sSP001, sSP002, sSP010, sSP011, M13, MMC), Microscope, Gradient PCR of SPCR2, Digestion of SPCR 5 and SPCR7 and Purification of Digestion.
Stress Experiment (sSP001, sSP002, sSP010, sSP011, M13, MMC)
Dilute ON culture 1:100
Centrifuge (1 min, 10rpm)
Wash 1x with 70% of Volume M9 Glucose
Centrifuge again
Take up in 70% of the VOlume M9 Gluo
100ul in each 96 well plate (see pipetting file)
Add MMC 10ug/ml in corresponding wells and incubate in the plate reader
After 90 min add phages into correscponding wells
put back on the plate reader
Microscope
look at the shape and fluorescence at t=0 of sSP001
look at the shape and fluorescence at=90 mi of sSP001 + 10ug/ml MMC
take photos
Gradient PCR of SPCR2
iSP001 (gblock) and iS002
Reagent
Volume
1x
Nuclease-free water
37,25µL
5x Phusion HF Buffer
10µL
10 mM dNTPs
1µL
Forward Primer (10 µM)
0.5µL
Reverse Primer (10 µM)
0.5µL
Template Plasmid
0.25µL
Phusion DNA Polymerase
0.5µL
Total Volume
50µL
Thermocycler Protocol: NEB Phusion
Temp
Time
Start
98°C
30 sec
Melt
Cycle 1
98°C
5 sec
Melt
35 cycles
Cycle 2
40.5°C / 60°C
25 sec
Anneal
Cycle 3
72°C
30 sec per kb
Extend
Finish
72°C
5 min
Extend
Store
10°C
Forever
Store
Digestion of SPCR 5 and SPCR7
Reagent
Volume
Purified PCR Product
16 ul
H2O
0 ul
10x FastDigest Buffer
2 ul
FastDigest Enzyme 1
1 ul
FastDigest Enzyme 2
1 ul
Total Volume
20 ul
Incubate at 37°C for 1h shaking
Purification of Digestion
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 30 μl Buffer EB  to the center of the QIAquick membrane and centrifuge the column for 1 min.
Detect
ASDF
Sunday 25th August
Gel of Gradient PCR, Liquid Cultures, Ligation M13 + RFP, Ligation SPCR5, SPCR7, Electrocompetent cells, Gibson Assembly, Transformation (Gibson)and Transformation (Gibson, Ligations.
Gel of Gradient PCR
1%, 100V, 20min
Liquid Cultures
NEB Turbo
XL10
Ligation
Ligation M13 + RFP
0.5ul Vector (M13)
2.6ul Insert (RFP)
5.4ul H20
1.0ul T4 buffer
0.5ul T4 Ligase
Ligation SPCR5, SPCR7
1.0ul Vector (SPCR5)
1.5ul Insert (SPCR7)
6.0ul H20
1.0ul T4 buffer
0.5ul T4 Ligase
Incubate ~30min at 37 C
Electrocompetent cells
1) Start Culture
2) When the cells reach an OD600 of 0.2.
3) Harvest the cells.
When the cells reach an OD600 of between 0.6 and 0.8 (OD: 0,683)
Centrifuge at 4 °C for 10 min at 4000 rpm.
4) Wash and combine the cells.
Remove the supernatant.
Resuspend the cells in 25 ml of ice cold water
5) Wash the cells 2 more times.
Centrifuge at 4 °C for 10 min at 4000 rpm.
Resuspend in 50 ml of ice cold water.
Repeat. 6) Wash and concentrate the cells for electroporation.
Centrifuge at 4 °C for 10 min at 4000 rpm.
Resuspend in 1-2 ml of ice cold water.
We will use 200 ul of washed cells per transformation.
Gibson Assembly
Amount per Reaction
Positive Control**
PCR Fragment(s)
+ linearized vector
2-10 μl (0.02–0.5 pmols)*
Gibson
Assembly Master
Mix (2X)
10 μl
Deionized H2O
XX μl
Total Volume
20 μl
Incubate samples in a thermocycler at 50°C for 15 minutes. After incubation, store the samples on ice or at -20°C for subsequent transformation
Note: Extended incubation up to 60 minutes may help to improve assembly efficiency in some cases (for further details see FAQ section).
Transform NEB 5-alpha Competent E. coli cells (provided with the kit) with 2 μl of the assembly reaction, following the transformation protocol.
Transformation (Gibson)
Thaw competent cells on ice.
Add 2 μl of the chilled assembly product to the competent cells. Mix gently by pipetting up and down or by flicking the tube 4–5 times. Do not vortex.
Place the mixture on ice for 30 minutes. Do not mix.
Heat shock at 42°C for 30 seconds. Do not mix.
Transfer tubes to ice for 2 minutes.
Add 950 μl of room-temperature SOC media to the tube.
Incubate the tube at 37°C for 60 minutes. Shake vigorously (250 rpm) or rotate.
Warm selection plates to 37°C.
Spread 100 μl of the cells onto the selection plates. Use Amp plates for positive control sample.
Incubate overnight at 37°C.
Transformation (Gibson, Ligations)
1) Prepare BD tubes with a pipette filled with LB at the interior of each tube (pipette supplied with the electroporation cuvette)
2) Test the purity of the electrocompetent cells.
Add 200 ul of washed cells to a cuvette.
3) Mix the cells and DNA in a cuvette.
200 ul of washed cells with 200 ng of PCR product.
Keep the cuvette on ice until just before the electroporation.
4) Preload a pipette with 1 ml of LB.
5) Pulse the cuvette with voltage.
Dry the electrodes with a kimwipe prior to loading.
Use the EC2 setting.
6) Listen for arcing.
A cracking sound means all the cells are dead.
Note the time constant: 5 is good, 5.8 is great.
7) Immediately recover the cells.
Add the 1 ml of preloaded LB and pipet up and down to mix.
Collect 1 ml of cells, some volume is lost in the cuvette.
8) Incubate 2 h at 37 °C with shaking.
9) Plate 100 ul of recovered cells on selective plates.
Use antibiotic appropriate to the part being integrated.
Let the other 900 ul rest overnight at room temperature.
10) Concentrate and plate the remaining cells
Spin down quickly and resuspend in 100 ul LB before plating.
Transformed cells should be incubated at 37 °C.
Colonies should appear 24-48 h after plating.
Detect
ASDF
Monday 26th August
Picking colonies (Trafo Gibson, BBC SPCR5/7)
chosing 5-7 colonies and start liquid cultures
Detect
ASDF
Tuesday 27th August
Starting liquid cultures and Miniprep of pS.005 and pS.006.
Starting liquid cultures
sSP001, sSP002, sSP010, sSP011, sSP012, sSP020
Miniprep of pS.005 and pS.006
1) Pellet 2x 9ml of liquid culture (4000rpm, 10 min)
2) Discard supernatant
3) resuspend the cells in 250ul of resuspension solution
4) add 250ul of lysis solution, mix by inverting 4-6 times
5) add 350ul of neutralization solution
4) centrifuge for 5 min
5) transfer supernatant to spin column
6) centrifuge for 1 min
7) discard flow through
8) add 500 ul wash solution and centrifuge for 1 min , discard flow through(repeat this step)
9) centrifuge for 1 min to remove left over liquid
10) transfer the column on a 1.5ml tube
11) add 50ul of elution buffer and incubate for 2 min
12) centrifuge for 2 min
13) Nanodrop the concentration and freeze at -20°
Colony PCR
Reagent
Volume
Forward Primer (10 uM)
0,5 ul
Reverse Primer (10 uM)
0,5 ul
Template DNA  (Miniprep)
0,5 ul
Quick-Load® Taq 2X Master Mix
12,5 ul
Nuclease-free water
10,0 ul
Total volume
25 ul
Thermocycler Protocol: Dream Taq Green
Temp
Time
Start
95°C
30 sec1
Melt
Cycle 1
95°C
15 sec
Melt
35 cycles
Cycle 2
46,8°C
30 sec
Anneal
Cycle 3
68°C
1 min per kb
Extend
cellule 2
Finish
68°C
5 min
Extend
Store
10°C
Forever
Store
Detect
ASDF
Wednesday 28th August
Gel Colony PCR, Liquid Cultures (1:100), Analytical Digest, Gel Ligation/Gibson, Gel Analytical digest, DpnI digest of SPCR5, SPCR6, SPCR7, Gel DpnI digest, PCR purification of DpnI digested SPCR5, SPCR6, SPCR7, Stress test microscope and M13 Test.
Gel Colony PCR
pS005 (C1-C5) - pS006 (C1-C7)
1% 100V 20 min
Liquid Cultures (1:100)
sSP001, sSP002, sSP010, sSP011, sSP012, sSP020
Analytical Digest
Reagent
Volume
Plasmid Miniprep
5 ul
H2O
12 ul
c10x FastDigest Buffer
2ul
FastDigest Enzyme 1
0,5 ul
FastDigest Enzyme 2
0,5 ul
Total Volume
20,0 ul
Digest for 15 minutes at 37 °C with shaking
Gel Ligation/Gibson
pS005 - pS006 - M13/RFP
1% 100V 20min
Gel Analytical digest
pS005 (C1-C5)-pS006 (C1-C7)
1% 100V 20min
DpnI digest of SPCR5, SPCR6, SPCR7
add 4ul of DpnI into each tube, mix and incubate for 1h at 37°C
Gel DpnI digest
let each 5ul run on a gel
1%, 100V 20min
PCR purification of DpnI digested SPCR5, SPCR6, SPCR7
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 30 μl Buffer EB  to the center of the QIAquick membrane and centrifuge the column for 1 min.
Stress test microscope
Dilute ON cultures of sSP001, sSP002, sSP010, sSP011, sSP012 1:100 and incubate up to an OD of 0.3
Take out 2ml of sSP001, sSP002, sSP010, sSP011
add:
sSP001: 10ug/ml MMC
sSP002: 10ug/ml MMC
sSP010: 20ul sSP012 supernatant
sSP011: 20ul sSP012 supernatant
incubate for 90 min
take pictures under the microscope (trans and YFP) of (un-)treated samples
M13 Test
Liquid cultures of sSP012 and sSP020
1) 100 μ l of plating bacteria per tube (OD 0.6)
2) Prepare tenfold serial dilutions (10-6  to 10-9 ) of the bacteriophage stock in LB
3) Put 10 μ l/ 100 μ l of each dilution to the bacteria
4)  Mix the bacteriophage particles with the bacterial culture by vortexing gently.
5) Add 40 μ l of X-gal solution (working conc) and IPTG (working conc) solution to each of the tubes
6) Add 3ml of Agar to each tube and gently vortexing for 3 seconds
7) Pour the mixture onto  plates containing LB medium supplemented with 5 mM MgCl2 and 2x IPTG and Xgal
8) Swirl the plate gently to ensure an even distribution of bacteria and agar.
9) Incubate them at 37°C.
Detect
ASDF
Friday 30th August
M13 test, DpnI digest (SPCR5,6,7), Gel SPCR5,6,7 and PCR purification of DpnI digested SPCR5, SPCR6, SPCR7
M13 test
dilute ON cultures (NEB turbo, FR-433) 1: 100, grow up to an OD of ~0.6
1) 100 μ l of plating bacteria per tube (OD 0.6)
2) Prepare tenfold serial dilutions (10-6  to 10-9 ) of the bacteriophage stock in LB
3) Put 10 μ l/ 100 μ l of each dilution to the bacteria
4)  Mix the bacteriophage particles with the bacterial culture by vortexing gently.
5) Add 40 μ l of X-gal solution (working conc) and IPTG (working conc) solution to each of the tubes
6) Add 3ml of Agar to each tube and gently vortexing for 3 seconds
7) Pour the mixture onto  plates containing LB medium supplemented with 5 mM MgCl2 and 2x IPTG and Xgal
8) Swirl the plate gently to ensure an even distribution of bacteria and agar.
9) Incubate them at 37°C.
DpnI digest (SPCR5,6,7)
add 8ul DpnI per tube and digest for 1h at 37°, shaking
Gel SPCR5,6,7
1%, 100V, 20min
PCR purification of DpnI digested SPCR5, SPCR6, SPCR7
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 30 μl Buffer EB  to the center of the QIAquick membrane and centrifuge the column for 1 min.
Detect
ASDF
Day Numbersuffix Month
Place your twit here
Place your note here
Detect
ASDF
Wednesday 4th September
Starting Liquid culture, PCRs, Gel, DpnI digest, Digestion of SPCR5,7, Purification of Digestion, Ligation of SPCR5,7, Gibson Assembly, Making Electrocompetent Cells, Transformation, Redo SPCR12/13 ,
Starting Liquid culture
NEB turbo, grow up to an OD of 0.6
PCRs
SPCR1, SPCR2, SPCR4, SPCR12, SPCR13
Reagant
Volume
1x (SPCR2)
11x (SPCR12)
11x (SPCR13)
8x (SPCR1)
8x (SPCR4)
Nuclease-free water
37.25 µl
447 µl
447 µl
335,25µl
335,25µl
5x Phusion HF Buffer
10 µl
120 µl
120 µl
90 µl
90 µl
10 mM dNTPs
1 µl
12 µl
12 µl
9 µl
9 µl
Forward Primer (10 uM)
0.5 µl
6 µl
6 µl
4.5 µl
4.5 µl
Reverse Primer (10 uM)
0.5 µl
6 µl
6 µl
4.5 µl
4.5 µl
Template Plasmid
0.25 µl
3 µl
3 µl
2.25 µl
2.25 µl
Phusion DNA Polymerase
0.5 µl
6 µl
6 µl
4,5 µl
4,5 µl
Total Volume
50 µl
600 µl
600 µl
450 µl
450 µl
for SPCR 12/13 gradient, 2,5 min
for SPCR3, SPCR11 51°, 45s
for SPCR 1, SPCR2, 48,8°, 2 min
for SPCR4, 40,8°, 1 min
Gel
1%, 100V, 20 min
1.small gel (SPCR1,2,3,4,11)
2. big gel (SPCR12)
3. big Gel (SPCR13)
DpnI digest
SPCR 1: add 10ul DpnI
SPCR4: add 10ul DpnI
SPCR2,3,11: add 1 ul DpnI
incubate for 1h at 37°
Digestion of SPCR5,7
Reagent
Volume
Purified PCR Product
16 µl
H2O
0 µl
10x FastDigest Buffer
2 µl
XbaI
1 µl
SpeI
1 µl
Total Volume
20.0 µl
Incubate for 1h at 37°, shaking
Purification of Digestion
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 30 μl Buffer EB  to the center of the QIAquick membrane and centrifuge the column for 1 min.
Ligation of SPCR5,7
Reagent
Volume
Vector
1  µl
Insert
3  µl
H2O
4.5  µl
Fermentas T4 Ligase Buffer
1  µl
Fermentas T4 Ligase Enzyme
0.5  µl
Total Volume
10  µl
Incubate at room temperature for ~1h
Gibson Assembly
Amount per Reaction : Positive Control**
PCR Fragment(s)+ linearized vector : 2-10 μl (0.02–0.5 pmols)*
Gibson Assembly Master Mix (2X): 10 μl
Deionized H2O : XX μl
Total Volume : 20 μl
Incubate samples in a thermocycler at 50°C for 15 minutes.
Put on Ice
Insert: 2,4 ul
Vector: 0,6 ul
Making Electrocompetent Cells
1)  Start a liquid culture of cells.                  NEB turbo
2)  dilute the cells 100X.
100 ml LB.
Grow at 30°C for about 90 minutes.
3) Harvest the cells.
When the cells reach an OD600 of between 0.6 and 0.8
Split the culture into 2x 50 ml falcon tubes, on ice.
Centrifuge at 4 °C for 10 min at 4000 rpm.
5) Wash and combine the cells.
Remove the supernatant.
Resuspend the cells in 2x 25 ml of ice cold water.
Combine the volumes in a single 50 ml falcon tube.
6) Wash the cells 2 more times.
Centrifuge at 4 °C for 10 min at 4000 rpm.
Resuspend in 50 ml of ice cold water.
Repeat.
7) Wash and concentrate the cells for electroporation.
Centrifuge at 4 °C for 10 min at 4000 rpm.
Resuspend in 1-2 ml of ice cold water.
We will use 200 ul of washed cells per transformation.
Transformation
pS005, pS006, SPCR3 (IDT plasmid) M13mp18
1) Prepare BD tubes with a pipette filled with LB at the interior of each tube (pipette supplied with the electroporation cuvette)
2) Test the purity of the electrocompetent cells.
Add 200 ul of washed cells to a cuvette.
3) Mix the cells and DNA in a cuvette.
200 ul of washed cells with 200 ng of PCR product.
Keep the cuvette on ice until just before the electroporation.
4) Preload a pipette with 1 ml of LB.
5) Pulse the cuvette with voltage.
Dry the electrodes with a kimwipe prior to loading.
Use the EC2 setting.
6) Listen for arcing.
A cracking sound means all the cells are dead.
Note the time constant: 5 is good, 5.8 is great.
7) Immediately recover the cells.
Add the 1 ml of preloaded LB and pipet up and down to mix.
Collect 1 ml of cells, some volume is lost in the cuvette.
8) Incubate 2 h at 37 °C with shaking.
9) Plate 10/200 ul of recovered cells on selective plates.
Use antibiotic appropriate to the part being integrated.
Let the other 900 ul rest overnight at room temperature.
10) Concentrate and plate the remaining cells
Spin down quickly and resuspend in 100 ul LB before plating.
Transformed cells should be incubated at 37 °C.
Colonies should appear 24-48 h after plating.
Redo SPCR12/13
Reagent
Volume
11x (SPCR12)
11x (SPCR13)
Nuclease-free water
447 µl
447 µl
5x Phusion HF Buffer
120 µl
120 µl
10 mM dNTPs
12 µl
12 µl
Forward Primer (10 uM)
6 µl
6 µl
Reverse Primer (10 uM)
6 µl
6 µl
Template Plasmid
3 µl
3 µl
Phusion DNA Polymerase
6 µl
6 µl
Total Volume
600 µl
600 µl
Cycling protocol as usual + gradient + topdown
Detect
ASDF
Thursday 5th September
Gel, Colony PCR, Topdown PCR and gels
Gel
SPCR12, SPCR13
1%, 100V, 20 min
Pictures: 13/09/05_SPCR12_corresponding primers
13/09/05_SPCR13_corresponding primers
Colony PCR
Reagent
Volume
Forward Primer (10 uM)/td>
0.5 ul
Reverse Primer (10 uM)
0.5 ul
Template DNA (from above)
1.5 ul
Quick-Load® Taq 2X Master Mix
12.5 ul
Total Volume
25 ul
Thermocycler Protocol: DreamTaq Green PCR MM
Temp
Time
Start
95°C
1 min
Melt
Cycle 1
95°C
30 sec
Melt
35 cycles
Cyle 2
52.6/46.8 °C
30 sec
Anneal
Cycle 3
72°C
1 min per kb
Extend
Finish
72°C
10 min
Extend
Store
10°C
Forever
Store
Topdown PCR
SPCR 1,4,12,13
Reagent
Volume
x1>/b>
Nuclease-free water
37.25 µl
5x Phusion HF Buffer
10 µl
10 mM dNTPs
1 µl
Forward Primer (10 uM)
0.5 µl
Reverse Primer (10 uM)
0.5 µl
Template Plasmid
0.25 µl
Phusion DNA Polymerase
0.5 µl
Total Volume
50 µl
increases every cycle 1°C, starting from 40°
Gels
1%, 100V, 20+ Min
Pictures:
13/09/05_colony PCR_pS005_C1-C8_BB verification primers
13/09/05_colony PCR_pS005_C9-C16_BB verification primers
13/09/05_colony PCR_pS006_C1-C8_BB verification primers
13/09/05_colony PCR_pS006_C9-C16_BB verification primers
13/09/05_top down PCR_SPCR1,4,12,13
Detect
ASDF
Friday 6th September
Place your twit here
Digestions: linearized pSB1C3, linearized pSB1A3, SPCR2, SPCR3, SPCR11
Reagent
Volume
Purified PCR Product
16 µl
H2OH2O
0 µl
10x FastDigest Buffer
2 µl
FastDigest Enzyme 1
1 µl
cFastDigest Enzyme 2
1 µl
Total Volume
4 µl
SPCR5, pSB1C3: XbaI/SpeI
SPCR11, pSB1A3, pSB1C3: EcoRI/PstI
SPCR13, pSB1A3, pSB1C3: EcoRI/PstI
SPCR2, pSB1A3, pSB1C3: EcoRI/PstI
1h at 37° shaking
Detect
ASDF
Saturday 7th September
Ligation, Making Electrocompetent Cells, Gibson Assembly, Transformation, Liquid culture of E1010
Ligation
Reagent
Volume
Vector
1 µl
Insert
3 µl
H2O
4.5 µl
Fermentas T4 Ligase Buffer
1
Fermentas T4 Ligase Enzyme
0.5
Total Volume
10 µl
SPCR5, pSB1C3
SPCR11, pSB1A3, pSB1C3
SPCR13, pSB1A3, pSB1C3
SPCR2, pSB1A3, pSB1C3
Incubate at room temperature for 10 minutes.
Making Electrocompetent Cells
1)  Start a liquid culture of cells.
NEB turbo
2)  dilute the cells 100X.
100 ml LB.
Grow at 30°C for about 90 minutes.
3) Harvest the cells.
When the cells reach an OD600 of between 0.6 and 0.8
Split the culture into 2x 50 ml falcon tubes, on ice.
Centrifuge at 4 °C for 10 min at 4000 rpm.
5) Wash and combine the cells.
Remove the supernatant.
Resuspend the cells in 2x 25 ml of ice cold water.
Combine the volumes in a single 50 ml falcon tube.
6) Wash the cells 2 more times.
Centrifuge at 4 °C for 10 min at 4000 rpm.
Resuspend in 50 ml of ice cold water.
Repeat.
7) Wash and concentrate the cells for electroporation.
Centrifuge at 4 °C for 10 min at 4000 rpm.
Resuspend in 1-2 ml of ice cold water.
We will use 200 ul of washed cells per transformation.
Gibson Assembly
Amount per Reaction
Positive Control**
PCR Fragment(s)
+ linearized vector
2-10 μl (0.02–0.5 pmols)*
Gibson
Assembly Master
Mix (2X)
10 μl
Deionized H2O
XX μl
Total Volume
20 μl
Incubate samples in a thermocycler at 50°C for 15 minutes.
Put on Ice
Transformation pS002 (Chl), pS002 (Amp), pS003 (Amp), pS004 (Chl), pS004 (Amp), pS005 (Chl), pS006-7 (Chl), pS007 (Chl), pS008 (Amp), pS009 (Chl), pS012 (Amp), pS013 (Chl), pS013 (Amp)
1) Prepare BD tubes with a pipette filled with LB at the interior of each tube (pipette supplied with the electroporation cuvette)
2) Test the purity of the electrocompetent cells.
Add 200 ul of washed cells to a cuvette.
3) Mix the cells and DNA in a cuvette.
200 ul of washed cells with 200 ng of PCR product.
Keep the cuvette on ice until just before the electroporation.
4) Preload a pipette with 1 ml of LB.
5) Pulse the cuvette with voltage.
Dry the electrodes with a kimwipe prior to loading.
Use the EC2 setting.
6) Listen for arcing.
A cracking sound means all the cells are dead.
Note the time constant: 5 is good, 5.8 is great.
7) Immediately recover the cells.
Add the 1 ml of preloaded LB and pipet up and down to mix.
Collect 1 ml of cells, some volume is lost in the cuvette.
8) Incubate 1/2 h at 37 °C with shaking.
9) Plate 10/200 ul of recovered cells on selective plates.
Use antibiotic appropriate to the part being integrated.
Let the other 900 ul rest overnight at room temperature.
10) Concentrate and plate the remaining cells
Spin down quickly and resuspend in 100 ul LB before plating.
Transformed cells should be incubated at 37 °C.
Colonies should appear 24-48 h after plating.
Liquid culture of E1010
Detect
ASDF
Sunday 8th September
Replating of transformation and Miniprep of E1010
Replating of Transformation
Miniprep of E1010
1) Pellet liquid culture (4000rpm, 10 min)
2) Discard supernatant
3) resuspend the cells in 250ul of resuspension olution
4) add 250ul of lysis solution, mix by inverting 4-6 times
5) add 350ul of neutralization solution
4) centrifuge for 5 min
5) transfer supernatant to spin column
6) centrifuge for 1 min
7) discard flow through
8) add 500 ul wash solution and centrifuge for 1 min , discard flow through(repeat this step)
9) centrifuge for 1 min to remove left over liquid
10) transfer the column on a 1.5ml tube
11) add 50ul of elution buffer and incubate for 2 min
12) centrifuge for 2 min
13) Nanodrop the concentration and freeze at -20°
Detect
ASDF
Monday 16th September
Result Cloning, Protocol: E. coli Colony PCR, Extraction of DNA, PCR Reaction and Gels
Result Cloning:
Colonies?
pS002 (Chl): no
pS002 (Amp): yes
pS003 (Amp):yes
pS004 (Chl): no
pS004 (Amp): yes
pS005 (Chl): yes
pS006-7 (Chl): yes
pS006 (Chl): no
pS007 (Chl): yes
pS008 (Amp): yes
pS009 (Chl): yes
pS012 (Amp): yes
pS013 (Chl): no
pS013 (Amp): yes
Protocol: E. coli Colony PCR
Using the Dream Taq Master Mix
Extraction of DNA
1) Pick a single colony into 50 ul of H20.
2) Boil for 5 minutes.
PCR Reaction (pS002 (Chl-Lig), pS005 (Chl-G), pS006-7 (Chl-Lig), pS007 (Chl-G), pS008 (Amp-G), pS009 (Chl-G))
Keep all the reagents at 4 °C while preparing the mixture.
Pre-heat the thermocycler to 95 °C and transfer your reaction directly from 4 °C.
Reagent
Volume
Forward Primer (10 uM)
0.5 ul
Reverse Primer (10 uM)
0.5 ul
Template DNA (from above)
1.5 ul
Quick-Load® Taq 2X Master Mix
12.5 ul
Nuclease-free water
10 ul
Total Volume
25 ul
Thermocycler Protocol: NEB Quick-Load
Temp
Time
Start
95°C
30 sec
Melt
Cycle 1
95°C
15 sec
Melt
35 cycles
Cycle 2
45.8 °C
30 sec
Anneal
Cycle 3
72°C
1 min per kb
Extend
Finish
72 °C
15 min
Extand
Store
10°C
Forever
Store
Gels
100V, 25-30 min, 1%
Pictures:
13/09/16_colony PCR_ps007.BMP
13/09/16_colony PCR_pS006-7.BMP
13/09/16_colony PCR_ps005.BMP
Result: didn’t work
Detect
ASDF
Tuesday 17th September
PCR Reaction (pS003 (Amp-G), pS004 (Amp-Lig), pS012 (Amp-G), pS013 (Chl-Lig))
PCR Reaction (pS003 (Amp-G), pS004 (Amp-Lig), pS012 (Amp-G), pS013 (Chl-Lig))
Keep all the reagents at 4 °C while preparing the mixture.
Pre-heat the thermocycler to 95 °C and transfer your reaction directly from 4 °C.
Reagent
Volume
Forward Primer (10 uM)
0.5 ul
Reverse Primer (10 uM)
0.5 ul
Template DNA (from above)
1.5 ul
Quick-Load® Taq 2X Master Mix
12.5 ul
Nuclease-free water
10 ul
Total Volume
25 ul
Thermocycler Protocol: NEB Quick-Load
Temp
Time
Start
95°C
30 sec
Melt
Cycle 1
95°C
15 sec
Melt
35 cycles
Cycle 2
45.8 °C
30 sec
Anneal
Cycle 3
72°C
1 min per kb
Extend
Finish
72 °C
15 min
Extand
Store
10°C
Forever
Store
Gels
100V, 30 min, 1%
Pictures: 13/09/17_colony PCR_pS009.JPG
13/09/17_colony PCR_pS008_pS002.jpg
13/09/17_colony PCR_pS003.JPG
13/09/17_colony PCR_pS004.JPG
13/09/17_colony PCR_pS012.JPG
13/09/17_colony PCR_pS013.JPG
Result:
pS003: row 3,5 might have worked -> analytical digest
pS004: all might have worked -> analytical digest
pS013: all might have worked -> analytical digest
pS012: row 3,4,5 might have workes -> analytical digest
Sequencing
Bringing primers/plasmids away for sequencin
Bacbbone, SPCR2 with iS002, iS032, iS028, iS029, iS030
Detect
ASDF
Wednesday 16th September
Ligations
Ligation
SPCR2 - Chl
Reagent
Volume
SPCR2
1 µl
Chl
2,95 µl
H2O
4.55 µl
Fermentas T4 Ligase Buffer
1.0 µl
Fermentas T4 Ligase Enzyme
0.5 µl
Total Volume
10 µl
SPCR2 - Amp
Reagent
Volume
SPCR2
1 µl
Amp
3.4 µl
H2O
4.1 µl
Fermentas T4 Ligase Buffer
1.0 µl
Fermentas T4 Ligase Enzyme
0.5 µl
Total Volume
10 µl
SPCR3 - Chl
Reagent
Volume
SPCR3
1 µl
Amp
1.79 µl
H2O
5.71 µl
Fermentas T4 Ligase Buffer
1.0 µl
Fermentas T4 Ligase Enzyme
0.5 µl
Total Volume
10 µl
SPCR3 - Amp7
Reagent
Volume
Amp
1 µl
SPCR 3
1.56 µl
H2O
5.94 µl
Fermentas T4 Ligase Buffer
1.0 µl
Fermentas T4 Ligase Enzyme
0.5 µl
Total Volume
10 µl
SPCR5 - Chl
Reagent
Volume
SPCR 5
0.5 µl
Chl
4.55 µl
H2O
3.45 µl
Fermentas T4 Ligase Buffer
1.0 µl
Fermentas T4 Ligase Enzyme
0.5 µl
Total Volume
10 µl
SPCR11 - Chl
Reagent
Volume
Chl
1 µl
SPCR11
1.3 µl
H2O
6.2 µl
Fermentas T4 Ligase Buffer
1.0 µl
Fermentas T4 Ligase Enzyme
0.5 µl
Total Volume
10 µl
SPCR2 - Amp
Reagent
Volume
Amp
1 µl
SPCR11
1.13 µl
H2O
6.37 µl
Fermentas T4 Ligase Buffer
1.0 µl
Fermentas T4 Ligase Enzyme
0.5 µl
Total Volume
10 µl
at 16° for 1h
Transformation
1) Thaw competent cells on ice (Gibson Kit competent cells)
2) Place 20 ul of cells in a pre-chilled Eppendorf tube.
3) Add 2.5 ul of plasmid (from Biobrick stock)
4) Mix gently by flicking the tube.
5) Chill on ice for 10 minutes.
4) Heat shock at 42 °C for 30 seconds
5) Return to ice for 2 minutes.
6) Add 200 ul LB medium and recover the cells by shaking at 37 °C.
Ampicillin: 15-30 minutes
Chloramphenicol: 60-120 minutes
7) Plate out the cells on selective LB (10ul)
8. Incubate at 37 °C. Transformants should appear within 12 hrs.
Gel of Ligation products
1%, 100V, 40 min
Detect
ASDF
Sunday 22nd September
PCR and Gels
PCR
M13 backbone
SPCR16,17
Reagent
Volume
x1
Nuclease-free water
37.25 µl
5x Phusion HF Buffer
10 µl
10 mM dNTPs
1 µl
Forward Primer (10 uM)
0.5 µl
Reverse Primer (10 uM)
0.5 µl
Template Plasmid
0.25 µl
Phusion DNA Polymerase
0.5 µl
Total Volume
50 µl
Protocol see fotos
Gel 100V, 1%, 1,5h
Detect
ASDF
Monday 23rd September
Gels and PCR
Gel
100V, 1%, 1,5h
Picture: 13/09/23_SPCR16_17_corresponding primers - PCRs didn't worked
PCR
SPCR 16 (normal phusion at 48°C)
Reagent
Volume
x1
Nuclease-free water
37.25 µl
5x Phusion HF Buffer
10 µl
10 mM dNTPs
1 µl
Forward Primer (10 uM)
0.5 µl
Reverse Primer (10 uM)
0.5 µl
Template Plasmid
0.25 µl
Phusion DNA Polymerase
0.5 µl
Total Volume
50 µl
Thermocycler Protocol: NEB Phusion
Temp
Time
Start
98°C
30 sec
Melt
Cycle 1
98°C
5 sec
Melt
35 cycles
Cycle 2
72°C
30 sec
Anneal
Cycle 3
72°C
30 sec per kb
Extend
Finish
72 °C
5 min
Extand
Store
10°C
Forever
Store
Detect
ASDF
Tuesday 24th September
Overlap PCR, PCR of Overlap, Digestion, Gel extraction, Gels, PCR Purification,  PCR (SPCR2, SPCR3, SPCR5, SPCR6, SPCR7, SPCR8, SPCR9, SPCR10, SPCR11, linearize pSB1C3) and liquid culture
Overlap PCR
SPCR5-SPCR2
Reagent
Volume
x1
Nuclease-free water
37.25 µl
5x Phusion HF Buffer
10 µl
10 mM dNTPs
1 µl
Template Plasmid(10 uM)
1 µl
Template Plasmid (10 uM)
1 µl
Phusion DNA Polymerase
0.5 µl
Total Volume
50 µl
PCR of Overlap
Digestion
M13 (EcoRI, PstI), pSB1C3 (EcoRI, PstI), psB1C3 (XbaI, SpeI)
Reagent
Volume
Purified Plasmid
20 µl
H2O
63 µl
10x FastDigest Buffer
10 µl
FastDigest Enzyme 1
2.5 µl
FastDigest Enzyme 2
2.5 µl
FastAP Phosphatase
2 µl
Total Volume
100 µl
SPCR2 (EcoRI, PstI), SPCR5 (XbaI, SpeI)
Reagent
Volume
Purified PCR Product
16 µl
H2O
0 µl
10x FastDigest Buffer
2 µl
FastDigest Enzyme 1
1 µl
FastDigest Enzyme 2
1 µl
Total Volume
20 µl
Gel
pSB1C3
1%, 100V, 1h
Picture: 13/09/24_pSB1C3 digest_EcoRI/PstI_XbaI/SpeI - digest worked
Gel extraction
1. Excise the DNA fragment from the agarose gel with a clean, sharp scalpel. Minimize the size of the gel slice by removing extra agarose.
2. Weigh the gel slice in a colorless tube. Add 3 volumes of Buffer QG to 1 volume of gel (100 mg ~ 100 μl). To help dissolve gel, mix by vortexing the tube every 2–3 min during the incubation.
3. After the gel slice has dissolved completely, check that the color of the mixture is yellow (similar to Buffer QG without dissolved agarose).
4. Add 1 gel volume of isopropanol to the sample and mix (actually only needed for very small products or very big products) 5. To bind DNA, apply the sample to the QIAquick column, and centrifuge for 1 min.
6. Discard flow-through and place QIAquick column back in the same collection tube.
7. To wash, add 0.75 ml of Buffer PE to QIAquick column and centrifuge for 1 min.
8. Discard the flow-through and centrifuge the QIAquick column for an additional 1 min at 17,900 x g (13,000 rpm).
9. Place QIAquick column into a clean 1.5 ml microcentrifuge tube.
10. To elute DNA, add 30 μl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge for 1 min.
Gel
PCR Overlap
1%, 100V, 1h
Picture: 13/09/24_Overlap PCR PCR SCPR5_SPCR2 - PCR didn't work
PCR Purification (M13 (EcoRI, PstI), SPCR2 (EcoRI, PstI), SPCR5 (XbaI, SpeI))
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 50 μl Buffer EB  to the center of the QIAquick membrane and centrifuge the column for 1 min.
PCR (SPCR2, SPCR3, SPCR5, SPCR6, SPCR7, SPCR8, SPCR9, SPCR10, SPCR11, linearize pSB1C3)
each 8x
Reagent
Volume
1x
Nuclease-free water
37.25 µl
5x Phusion HF Buffer
10 µl
10 mM dNTPs
1 µl
Forward Primer (10 uM)
0.5 µl
Reverse Primer (10 uM)
0.5 µl
Template Plasmid
0.25 µl
Phusion DNA Polymerase
0.5 µl
Total Volume
50 µl
Thermocycler Protocol: Fermentas Phusion
Temp
Time
Start
98°C
30 sec
Melt
Cycle 1
98°C
5 sec
Melt
35 cycles
Cycle 2
25 sec
Anneal
Cycle 3
72°C
30 sec per kb
Extend
Finish
72 °C
5 min
Extand
Store
10°C
Forever
Store
Liquid culture
NEB, sSP004, Fr-433
Detect
ASDF
Wednesday 25th September
Gel of PCR, PCR (SPCR5,6) and Miniprep Cas9, M13mp18
Gel of PCR
1%, 100V, 1h
Picture: 13/09/25_SPCR2-11 - SPCR2: worked, SPCR3: worked, SPCR5: didn't work, SPCR6: didn't work, SPCR7: didn't work, SPCR8: didn't work, SPCR9: didn't work, SPCR10: didn't work, SCPR11: worked
PCR (SPCR5,6)
Reagent
Volume
x1
Nuclease-free water
37.25 µl
5x Phusion HF Buffer
10 µl
10 mM dNTPs
1 µl
Forward Primer (10 uM)
1 µl
Reverse Primer (10 uM)
1 µl
Template Plasmid)
0.25 µl
Phusion DNA Polymerase
0.5 µl
Total Volume
50 µl
Thermocycler Protocol: Fermentas Phusion
Temp
Time
Start
98°C
30 sec
Melt
Cycle 1
98°C
5 sec
Melt
35 cycles
Cycle 2
25 sec
Anneal
Cycle 3
72°C
30 sec per kb
Extend
Finish
72 °C
5 min
Extand
Store
10°C
Forever
Store
Miniprep Cas9, M13mp18
1) Pellet liquid culture (4000rpm, 10 min)
2) Discard supernatant
3) resuspend the cells in 250ul of resuspension olution
4) add 250ul of lysis solution, mix by inverting 4-6 times
5) add 350ul of neutralization solution
4) centrifuge for 5 min
5) transfer supernatant to spin column
6) centrifuge for 1 min
7) discard flow through
8) add 500 ul wash solution and centrifuge for 1 min , discard flow through(repeat this step)
9) centrifuge for 1 min to remove left over liquid
10) transfer the column on a 1.5ml tube
11) add 50ul of elution buffer and incubate for 2 min
12) centrifuge for 2 min
13) Nanodrop the concentration and freeze at -20°
Detect
ASDF
Thursday 26th September
Gel PCR 5,6, Digestion, PCR (SPCR7-10) and PCR purification
Gel PCR 5,6 1%, 100V, 1h
Picture: 13/09/26_SPCR5_6 -  PCR didn't work
Digestions: SPCR2 (EcoRI, PstI), SPCR3 (EcoRI, PstI), SPCR4 (EcoRI, PstI), SPCR11 (EcoRI, PstI), lin pSB1C3 (EcoRI, PstI), lin pSB1A3 (EcoRI, PstI),  SPCR5 (XbaI, SpeI), SPCR7 (XbaI, SpeI), lin pSB1C3 (XbaI, SpeI), in pSB1A3 (XbaI, SpeI)
Reagent
Volume
Purified PCR Product
16 µl
H2O
0 µl
10x FastDigest Buffer
2 µl
FastDigest Enzyme 1
1 µl
FastDigest Enzyme 2
1 µl
Total Volume
20 µl
PCR (SPCR7-10)
Reagent
Volume
x1
Nuclease-free water
37.25 µl
5x Phusion HF Buffer
10 µl
10 mM dNTPs
1 µl
Forward Primer (10 uM)
1 µl
Reverse Primer (10 uM)
1 µl
Template Plasmid)
0.25 µl
Phusion DNA Polymerase
0.5 µl
Total Volume
50 µl
Thermocycler Protocol: Fermentas Phusion
Temp
Time
Start
98°C
30 sec
Melt
Cycle 1
98°C
5 sec
Melt
35 cycles
Cycle 2
25 sec
Anneal
Cycle 3
72°C
30 sec per kb
Extend
Finish
72 °C
5 min
Extand
Store
10°C
Forever
Store
PCR Purification: (SPCR2 (EcoRI, PstI), SPCR3 (EcoRI, PstI), SPCR4 (EcoRI, PstI), SPCR11 (EcoRI, PstI), lin pSB1C3 (EcoRI, PstI), lin pSB1A3 (EcoRI, PstI),  SPCR5 (XbaI, SpeI), SPCR7 (XbaI, SpeI), lin pSB1C3 (XbaI, SpeI), in pSB1A3 (XbaI, SpeI)
Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix.
To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through. Place the QIAquick column back into the same tube
To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
Discard flow-through and place the QIAquick column back in the same tube.
Centrifuge the column for an additional 1 min.
Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 50 μl Buffer EB  to the center of the QIAquick membrane and centrifuge the column for 1 min.
Detect
ASDF
Friday 27th September
Gel SPCR7-10, NEB liquid culture, Ligation, Electrocompetent cells and transformation
Gel SPCR7-10
1%, 100V, 1h
Picture: 13/09/27_SPCR7_8_9_10 - PCRs worked
NEB liquid culture
1:1000 of ON culture
Ligation
pS006
Reagent
Volume
SPCR5
1 µl
SPCR7
0,28 µl
H2O
7.22 µl
Fermentas T4 Ligase Buffer
1 µl
Fermentas T4 Ligase Enzime
0.5 µl
Total Volume
10 µl
pS006’
Reagent
Volume
SPCR5
1 µl
lin pSB1C3 (XbaI/SpeI)
0,47 µl
H2O
7.03 µl
Fermentas T4 Ligase Buffer
1 µl
Fermentas T4 Ligase Enzime
0.5 µl
Total Volume
10 µl
pS006*
Reagent
Volume
SPCR5
1 µl
lin pSB1A3 (XbaI/SpeI)
0,67 µl
H2O
6.83 µl
Fermentas T4 Ligase Buffer
1 µl
Fermentas T4 Ligase Enzime
0.5 µl
Total Volume
10 µl
pS013
Reagent
Volume
SPCR4
1 µl
SPCR11
0,24 µl
H2O
7.26 µl
Fermentas T4 Ligase Buffer
1 µl
Fermentas T4 Ligase Enzime
0.5 µl
Total Volume
10 µl
pS013'
Reagent
Volume
lin pSB1C3 (EcoRI, SpeI)
1 µl
SPCR11
0,28 µl
H2O
7.22 µl
Fermentas T4 Ligase Buffer
1 µl
Fermentas T4 Ligase Enzime
0.5 µl
Total Volume
10 µl
pS013*
Reagent
Volume
lin pSB1A3 (EcoRI, PstI))
1 µl
SPCR11
0,3 µl
H2O
7.2 µl
Fermentas T4 Ligase Buffer
1 µl
Fermentas T4 Ligase Enzime
0.5 µl
Total Volume
10 µl
pS004
Reagent
Volume
SPCR4
1 µl
SPCR3
0,2 µl
H2O
7.3 µl
Fermentas T4 Ligase Buffer
1 µl
Fermentas T4 Ligase Enzime
0.5 µl
Total Volume
10 µl
pS004'
Reagent
Volume
lin pSB1C3 (EcoRI, PstI)
1 µl
SPCR3
0,3 µl
H2O
7.2 µl
Fermentas T4 Ligase Buffer
1 µl
Fermentas T4 Ligase Enzime
0.5 µl
Total Volume
10 µl
pS004*
Reagent
Volume
lin pSB1A3 (EcoRI, PstI)
1 µl
SPCR3
0,27 µl
H2O
7.23 µl
Fermentas T4 Ligase Buffer
1 µl
Fermentas T4 Ligase Enzime
0.5 µl
Total Volume
10 µl
pS002'
Reagent
Volume
SPCR2
1 µl
lin pSB1C3 (EcoRI, PstI)
0,8 µl
H2O
6.7 µl
Fermentas T4 Ligase Buffer
1 µl
Fermentas T4 Ligase Enzime
0.5 µl
Total Volume
10 µl
pS002*
Reagent
Volume
SPCR2
1 µl
lin pSB1A3 (EcoRI, PstI)
0,6 µl
H2O
6.9 µl
Fermentas T4 Ligase Buffer
1 µl
Fermentas T4 Ligase Enzime
0.5 µl
Total Volume
10 µl
45min at 22°, 15 min at 16°
Electrocompetent cells
1) Start Culture
2) When the cells reach an OD600 of 0.2.
3) Harvest the cells.
When the cells reach an OD600 of between 0.6 and 0.8 (OD: 0,683)
Centrifuge at 4 °C for 10 min at 4000 rpm.
4) Wash and combine the cells.
Remove the supernatant.
Resuspend the cells in 25 ml of ice cold water
5) Wash the cells 2 more times.
Centrifuge at 4 °C for 10 min at 4000 rpm.
Resuspend in 50 ml of ice cold water.
Repeat.
6) Wash and concentrate the cells for electroporation.
Centrifuge at 4 °C for 10 min at 4000 rpm.
Resuspend in 1-2 ml of ice cold water.
We will use 200 ul of washed cells per transformation.
Transformation
pS006, pS006’, pS006*, pS013, pS013’, pS013*, pS004, pS004’, pS004*, pS002’, pS002*
1) Prepare BD tubes with a pipette filled with LB at the interior of each tube (pipette supplied with the electroporation cuvette)
2) Test the purity of the electrocompetent cells.
Add 200 ul of washed cells to a cuvette.
3) Mix the cells and DNA in a cuvette.
200 ul of washed cells with 200 ng of PCR product.
Keep the cuvette on ice until just before the electroporation.
4) Preload a pipette with 1 ml of LB.
5) Pulse the cuvette with voltage.
Dry the electrodes with a kimwipe prior to loading.
Use the EC2 setting.
6) Listen for arcing.
A cracking sound means all the cells are dead.
Note the time constant: 5 is good, 5.8 is great.
7) Immediately recover the cells.
Add the 1 ml of preloaded LB and pipet up and down to mix.
Collect 1 ml of cells, some volume is lost in the cuvette.
8) Incubate 1/2 h at 37 °C with shaking.
9) Plate 10/200 ul of recovered cells on selective plates.
Use antibiotic appropriate to the part being integrated.
Let the other 900 ul rest overnight at room temperature.
10) Concentrate and plate the remaining cells
Spin down quickly and resuspend in 100 ul LB before plating.
Transformed cells should be incubated at 37 °C.
Colonies should appear 24-48 h after plating.
Detect
ASDF
Saturday 28th September
Extraction of Genomic DNA, PCR Reaction and Restreak plates
Protocol: E. coli Colony PCR (pS006’, pS006*, pS013, pS004, pS004*, pS002*)
Extraction of Genomic DNA
1) Pick a single colony into 50 ul of H20.
Fresh colonies (grown that day) work best, but they can also come from 4 C.
Pipet 2ul onto plates to have C1-C8
Pipet 2 ul into 5ml Media with antibiotics to make liquid cultures
2) Boil for 5 minutes.
1.5 ul of this can be used directly for PCR.
Best if used directly, but can also be stored at 4 C for a few days.
PCR Reaction
Keep all the reagents at 4 °C while preparing the mixture.
Pre-heat the thermocycler to 95 °C and transfer your reaction directly from 4 °C.
Reagent
Volume
9x Volume
Forward Primer (10 uM)
0.5 µl
4.5 µl
Reverse Primer (10 uM)
0.5 µl
4.5 µl
Template DNA (from above)
1.5 µl
13.5 µl
Quick-Load® Taq 2X Master Mix
12.5 µl
112.5 µl9x Volume
Nuclease-free water
10 µl
90 µl
Total Volume
25 µl
125 µl
Thermocycler Protocol: Green Dream Taq
Temp
Time
Start
95°C
30 sec
Melt
Cycle 1
95°C
15 sec
Melt
35 cycles
Cycle 2
46.8°C
30 sec
Anneal
Cycle 3
72°C
1 min per kb
Extend
Finish
72 °C
10 min
Extand
Store
10°C
Forever
Store
Restreak plates
Some of the plates were too full grown -> Rstreak for single colonies
Detect
ASDF
Monday 30th September
Extraction of Genomic DNA, PCR reaction, Gels, Miniprep, PCR: SPCR5, SPCR6  and Liquid culture of pS004’ and pS013’
Protocol: E. coli Colony PCR (pS006, pS013’,pS013*,  pS004’, pS002’)
Extraction of Genomic DNA
1) Pick a single colony into 50 ul of H20.
Fresh colonies (grown that day) work best, but they can also come from 4 C.
Pipet 2ul onto plates to have C1-C8
2) Boil for 5 minutes.
1.5 ul of this can be used directly for PCR.
Best if used directly, but can also be stored at 4 C for a few days.
PCR Reaction
Keep all the reagents at 4 °C while preparing the mixture.
Pre-heat the thermocycler to 95 °C and transfer your reaction directly from 4 °C.
Reagent
Volume
9x Volume
Forward Primer (10 uM)
0.5 µl
4.5 µl
Reverse Primer (10 uM)
0.5 µl
4.5 µl
Template DNA (from above)
1.5 µl
13.5 µl
Quick-Load® Taq 2X Master Mix
12.5 µl
112.5 µl9x Volume
Nuclease-free water
10 µl
90 µl
Total Volume
25 µl
125 µl
Thermocycler Protocol: Green Dream Taq
Temp
Time
Start
95°C
30 sec
Melt
Cycle 1
95°C
15 sec
Melt
35 cycles
Cycle 2
46.8°C
30 sec
Anneal
Cycle 3
72°C
1 min per kb
Extend
Finish
72 °C
10 min
Extand
Store
10°C
Forever
Store
Gels
1%, 100V, 45 min
Pictures + Results
13/09/30_colony pcr_pS004_pS013 - 1-8: pS004 (gRNA - Amp), 9-16: pS013 (crRNA - Amp) - seems to have worked beside pS013 C7
13/09/30_colony PCR_pS004*_pS002* - 1-8: pS004* (gRNA - Amp), 9-16: pS002 (reporter - Amp) - pS004* seems to have worked (C1, C2, C3, C6)
13/09/30_colony PCR_pS006'_pS006* - 1-8: pS006' (Cas9 - Chl), 9-16: pS006* (Cas9 - Amp) - didn’t work
13/09/30_Colony PCR_pS013'_pS013* - 1-8: pS013' (crRNA - Chl), 9-16: pS013* (crRNA - Amp) - Seems to have worked beside 13’ C7, C8
13/09/30_colony PCR_pS004' - 1-8: pS004' (gRNA - Chl), 9-16 - pS004' seems to have worked beside C6
13/09/30_digestion of pS006 and pS002 (Cas9, reporter): seem to have worked
other gel images that proof that the reporter as well as the Cas9 construct worked will follow soon!
Miniprep
pS013, pS004
1) Pellet liquid culture (4000rpm, 10 min)
2) Discard supernatant
3) resuspend the cells in 250ul of resuspension olution
4) add 250ul of lysis solution, mix by inverting 4-6 times
5) add 350ul of neutralization solution
4) centrifuge for 5 min
5) transfer supernatant to spin column
6) centrifuge for 1 min
7) discard flow through
8) add 500 ul wash solution and centrifuge for 1 min , discard flow through(repeat this step)
9) centrifuge for 1 min to remove left over liquid
10) transfer the column on a 1.5ml tube
11) add 50ul of elution buffer and incubate for 2 min
12) centrifuge for 2 min
13) Nanodrop the concentration and freeze at -20°
PCR: SPCR5, SPCR6 (gradient + touch down 55-75, -1°/cycle)
Reagent
Volume
11x
Nuclease-free water
37.25 µl
5x Phusion HF Buffer
10 µl
10 mM dNTPs
1 µl
Forward Primer (10 uM)
0.5 µl
Reverse Primer (10 uM)
0.5 µl
Template Plasmid)
0.25 µl
Phusion DNA Polymerase
0.5 µl
Total Volume
50 µl
Thermocycler Protocol: Fermentas Phusion
Temp
Time
Start
98°C
30 sec
Melt
Cycle 1
98°C
5 sec
Melt
35 cycles
Cycle 2
25 sec
Anneal
Cycle 3
72°C
30 sec per kb
Extend
Finish
72 °C
5 min
Extand
Store
10°C
Forever
Store
Liquid culture of pS004’ and pS013’
Detect
ASDF
Tuesday 1st October
Miniprep pS013’, pS004’
1) Pellet liquid culture (4000rpm, 10 min)
2) Discard supernatant
3) resuspend the cells in 250ul of resuspension olution
4) add 250ul of lysis solution, mix by inverting 4-6 times
5) add 350ul of neutralization solution
4) centrifuge for 5 min
5) transfer supernatant to spin column
6) centrifuge for 1 min
7) discard flow through
8) add 500 ul wash solution and centrifuge for 1 min , discard flow through(repeat this step)
9) centrifuge for 1 min to remove left over liquid
10) transfer the column on a 1.5ml tube
11) add 50ul of elution buffer and incubate for 2 min
12) centrifuge for 2 min
13) Nanodrop the concentration and freeze at -20°
Detect
ASDF
Wednesday 2nd October
Preparation killing assay
Liquid cultures
R16, R26, R26f, delta PyrF, parental delta PyrF, NEB, M13, Litmus + Helper, Litmus GFP + helper
Detect
ASDF
Wedsnesday 3rd October
Killing assay to test specifity of CRISPR Cas
Electrocompetent cells (delta PyrF, parental, NEB)
1) Start Culture
2) When the cells reach an OD600 of 0.2.
3) Harvest the cells.
When the cells reach an OD600 of between 0.6 and 0.8 (OD: 0,683)
Centrifuge at 4 °C for 10 min at 4000 rpm.
4) Wash and combine the cells.
Remove the supernatant.
Resuspend the cells in 25 ml of ice cold water
5) Wash the cells 2 more times.
Centrifuge at 4 °C for 10 min at 4000 rpm.
Resuspend in 50 ml of ice cold water.
Repeat.
6) Wash and concentrate the cells for electroporation.
Centrifuge at 4 °C for 10 min at 4000 rpm.
Resuspend in 1-2 ml of ice cold water.
We will use 200 ul of washed cells per transformation.
Transformation
Cotransform into parental/keio delta pyRF: pS006* (Cas9 in Amp BB) + pS004'(gRNA in Chl BB).
1) Prepare BD tubes with a pipette filled with LB at the interior of each tube (pipette supplied with the electroporation cuvette)
2) Test the purity of the electrocompetent cells.
Add 200 ul of washed cells to a cuvette.
3) Mix the cells and DNA in a cuvette.
200 ul of washed cells with 200 ng of PCR product.
Keep the cuvette on ice until just before the electroporation.
4) Preload a pipette with 1 ml of LB.
5) Pulse the cuvette with voltage.
Dry the electrodes with a kimwipe prior to loading.
Use the EC2 setting.
6) Listen for arcing.
A cracking sound means all the cells are dead.
Note the time constant: 5 is good, 5.8 is great.
7) Immediately recover the cells.
Add the 1 ml of preloaded LB and pipet up and down to mix.
Collect 1 ml of cells, some volume is lost in the cuvette.
8) Incubate 1/2 h at 37 °C with shaking.
9) Plate 10/200 ul of recovered cells on selective plates.
Use antibiotic appropriate to the part being integrated.
Let the other 900 ul rest overnight at room temperature.
10) Concentrate and plate the remaining cells
Spin down quickly and resuspend in 100 ul LB before plating.
Transformed cells should be incubated at 37 °C.
Colonies should appear 24-48 h after plating.
Plating of 10/100ul on
1. Amp plates -> selection for only Cas9
2. Chl plates -> selection for only gRNA
3. Amp/Chl plates -> selection for theoretically functional system
Detect
ASDF
Monday 14th October
Liquid cultures sSP008, sSP009, R16, R26, R26f, sT007, sSP021, SCPR2-A3, SPCE2-C3
Detect
ASDF
Tuesday 15th October
Digestion Reporter and SPCR3
BB: SpeI, PstI
SPCR: XbaI, PstI
2h 37°, heat inactivation
Electrocompetent cells: sSP008, sSP009
Transform SPCR2-A3/C3 both into sSP008/9
Detect
ASDF
Wednesday 16th October
Miniprep of pS002, pS002’, pS006* Liquid culture sSP017, pS004, pS004’ Glycerols
Detect
ASDF
Thursday 17th October
Minipreps, Digestions, Ligation and Transformation
Miniprep sSP017, pS004, pS004’
Digest Litmus pS018 EcoRI/PstI, EcoRI/XbaI 2h 37° shaking
PCR purify
Ligation:
1. gRNA+phagmid (10,4+6)
2.phagemid+gRNA+reporter (9,6+1,86+2)
3.phagemid+gRNA+Cas9 (10,2+2,04+3)
1h at 22°
Transform
Heat shock, cells matt
Recover, plate
Detect
ASDF
Friday 18th October
Miller assay and Liquid phagemid
Miller assay
Grow liquid cultures up to an OD 600 of 0.3-0.7
Meassure the OD 600
Start the specific cultures you want to test under the specific conditions
Take each 600 ul of culture and add 400ul of Z buffer (see below)
Add 100ul of Chloroform and 50ul of 0.1% SDS and vortex intensively
Incubate the tubes for 5min at 28°C
Add 200ul of ONPG (see below) and note the time point of adding
Incubate for up to 90 min or until a yellow/green color appears
To stop the reaction, add 500ul of Na2CO3 and note the time point of stopping
Centrifuge for 3 min
Measure the OD at 420
The expression of beta-galactosidase can now be calculated with the formula: Miller unit= (1000*OD420)/(delta t*V(culture taken)*OD600)
Z-buffer:
Na2HPO4 7H2O 60 mM - 16.1g
NaH4PO4 H2O 40 mM - 5.5g
KCl 10 mM - 0.75g
MgSO4 7H2O 1 mM - 0.246g
beta mercaptoethanol 50 mM - 2.7 ml
ONPG:
4mg/ml in Z-buffer
Na2CO3 1M
Detect
ASDF
Saturday 19th October
Miller assay, Digestions
Miller
Miniprep phagemid
Digest phagemid E/P, E/X, SPCR3 E,P
Detect
ASDF
Tuesday 22nd October
Liquid for Miller and Digest&purify SPCR3 E,P
Detect
ASDF
Thursday 24th October
Digestion, Ligation, Transformation Ligation, Transformation
Digest phagemid E/P
Purify it
Ligation:
1. SPCR3 E/P + phagemid E/P
2. SPCR3 E/P + SPCR5 P/X+ phagemid E/X
2. SPCR3 E/P + SPCR2 P/X+ phagemid E/X
trafo Ligations
electrocmpetent cells parental+sSP009+pS002’
trafo:
1. parental+pS002
2. parental+pS002’
3. parental+pS006*
4. parental+pS002’+pS006*
5. sSP009 pS002’ + pS006*
trafo helper into NEB
liquid cultures: sSP009, pRECA strains Ariel, sSP008+pS002, sSP008+pS002’
Detect
ASDF
Friday 25th October
Liquid cultures, Transformation, Conjugation
Colony PCR
Liquid Cloning
Liquid: NEB helper, parental pS002, parental pS006*, sSP008+pS002, sSP008+pS002’
Miniprep phagemid gRNA
Trafo: NEB helper+phageid-gRNA, parental ps006*+pS002’, pyrfF F+ pS002’+pS006*
conjugation of parental pS006* with pS009
Liquid library
Restreak for library
Functionality Test of reporter
1) sSP008+pS002/pS002’ +Xgal
2) sSP008+pS002/pS002’ +Xgal + MMC
3) sSP008+pS002/pS002’ +MMC
Detect
ASDF
Saturday 26th October
Liquid cultures, Transformation, Conjugation
Liquid cultures
sSP008, transformed parental strains, Kan strains, NEB helper-gRNA, sSP009
Conjugate at OD 0.2 parental + pS002'+pS006* with sSP009 and plate 50ul on Amp/Tet/Chl plates
Electrocompetent cells+ transformation
Trafo:sSP008+Cas9 Addgene+ phagemid-gRNA -> killing assay to confirm gRNA on phagemid works , pyrfF F+ pS002’+pS006*
Detect
ASDF
Sunday 27th October
Miller assay final strain/construct with controls
Miller assay
MG1655 as positive control
parental + reporter + Cas9 + F+
Keio delta pyrF + reporter + Cas9 + F+
parental + reporter + Cas9 + F+ + phagemid with gRNA
Keio delta pyrF + reporter + Cas9 + F+ + phagemid with gRNA
Keio delta pyrF + reporter + MMC
parental + reporter + MMC
Liquid culture functionality test
parental + reporter + Cas9 + F+ + Xgal
parental + reporter + Cas9 + F+ + Xgal + phagemid with gRNA
parental + reporter + Cas9 + F+ + phagemid with gRNA
Keio delta pyrF + reporter + Cas9 + F+ + Xgal
Keio delta pyrF + reporter + Cas9 + F+ + Xgal + phagemid with gRNA
Keio delta pyrF + reporter + Cas9 + F+ +  phagemid with gRNA
Killing assay
Plate:
parental + reporter + Cas9 + F+
Keio delta pyrF + reporter + Cas9 + F+
parental + reporter + Cas9 + F+ + phagemid with gRNA
Keio delta pyrF + reporter + Cas9 + F+ + phagemid with gRNA
Target
ASDF
Monday 10th June
Design of SirA genes for synthesis
SirA
Mycobacterium tuberculosis
Reference: Pinto et al. 2007. Sulfite Reduction in Mycobacteria doi:  10.1128/JB.00487-07
Primers:
TBsirA F      GGAATTCCATATGTCCGCGAAGGAGAACCCC
TBsirA R      AAGGAAAAAAGCGGCCGCTCATCGCAGGTCGTCCTCCTCGGCCCGGAT
We blasted the above primers used to clone sirA from M. tuberculosis into E. coli.  We then extracted the sequence from genbank to retrieve the following sequence which includes the restriction sites introduced by Pinto et al.
ggaattccat atgtccgcga aggagaaccc ccaaatgacc actgcacgtc
ccgccaaggc tcgaaatgag ggccagtggg cgctgggaca tcgcgagcca
ctcaacgcca acgaagagct gaagaaggcc ggcaacccgc tcgacgtgcg
ggagcgcatc gaaaacatct acgccaaaca gggtttcgac agcatcgaca
agaccgacct gcgagggcgc tttcgctggt ggggcctgta cacccagcgt
gagcagggct acgacggcac ctggaccggt gacgacaaca tcgacaagct
cgaggccaaa tacttcatga tgcgggtgcg ttgcgacggc ggcgcgctct
cggctgccgc gctgcgcacg ctgggccaga tctcgacgga gttcgcgcgc
gataccgccg atatctccga ccggcagaac gtgcaatacc actggatcga
agtggaaaac gtccctgaaa tctggcgacg gttagacgat gtcggactgc
agaccaccga ggcgtgcggt gactgcccgc gggtagtgct gggctcgccg
ttggccggcg agtcgctcga cgaagtgctc gacccgacct gggcgatcga
ggagatcgtg cgtcgctaca tcggcaagcc cgacttcgcc gacttgccgc
gcaagtacaa gaccgccatc tctggcctgc aggacgtcgc gcacgagatc
aacgacgtcg ccttcatcgg cgtcaaccat cccgagcacg gaccaggcct
ggatctgtgg gtgggcggtg gactgtcgac caacccgatg ctggcccagc
gggtcggcgc ctgggttcca ctgggcgaag tgcccgaggt gtgggcggcg
gtcacctcgg tgtttcgcga ctacggctac cggcgactgc gcgccaaggc
ccggctgaaa tttctgatca aagactgggg catagcgaag ttccgcgaag
tgctcgaaac cgagtacctc aagcgtccgc tgatcgacgg tccggccccc
gaaccggtca agcatccgat cgaccacgtc ggggtgcaac gactcaagaa
cgggctcaac gccgtcggag tcgcccccat cgccgggcgg gtatcgggca
ccatcctcac ggcggtcgcc gacctgatgg cgcgggccgg ttccgaccgg
atccggttca ccccctacca gaagctggtc atcctcgaca ttccggacgc
cttgctcgac gacttgatcg ccggtctgga cgcgctgggg ctgcagtcgc
gcccgtcgca ttggcgccgg aacttgatgg cgtgcagcgg gattgagttc
tgcaagttgt cattcgccga aacccgggtt cgagcacagc atttggtgcc
cgagctggaa cgccggcttg aggacatcaa ctcgcagctc gacgtaccga
tcaccgtcaa catcaacggc tgcccgaact catgtgcgcg aattcaaatc
gccgacatcg gattcaaggg acagatgatc gacgacggac acggcggctc
cgtcgaaggc ttccaggtgc atctgggcgg acacctcggc ctggatgccg
gattcggccg caaactgcgc cagcacaagg tcaccagtga cgaactcggc
gactacatcg accgggtggt gcgcaacttc gtcaaacacc gcagcgaagg
tgaacgcttc gcgcagtggg tcatccgggc cgaggaggac gacctgcgat
gagcggccgc ttttttcctt
This sequence was then inserted into Geneious.  The PstI and EcoRI sites were manually removed and replaced with silent mutations.  A NcoI site was inserted at the start codon and an extra glycine was added after the start codon for the NcoI site to function properly.  We chose glycine as it is small an unlikely to interfere with either polar or non-polar domains within the protein.  We used GGC as the codon as it is the highest used glycine codon in e. coli.  We also added an AvrII site on the 3’ end of the site.  We then added biobrick restriction sites on the ends of the sequence (EcoRI, XbaI, SpeI, PstI).   This will allow us to clone into our expression vector pACYCDuet-1 with NcoI and AvrII enzymes and into a biobrick registry with EcoRI and PstI.  The insertion into pACYCDuet-1 removes both protein tags and cuts out the second MCS resulting in vector pACYCDuet-TBsirA.  The resulting sequence for TB SirA follows:
catggaattc tagaccatgg gctccgcgaa ggagaacccc caaatgacca
ctgcacgtcc cgccaaggct cgaaatgagg gccagtgggc gctgggacat
cgcgagccac tcaacgccaa cgaagagctg aagaaggccg gcaacccgct
cgacgtgcgg gagcgcatcg aaaacatcta cgccaaacag ggtttcgaca
gcatcgacaa gaccgacctg cgagggcgct ttcgctggtg gggcctgtac
acccagcgtg agcagggcta cgacggcacc tggaccggtg acgacaacat
cgacaagctc gaggccaaat acttcatgat gcgggtgcgt tgcgacggcg
gcgcgctctc ggctgccgcg ctgcgcacgc tgggccagat ctcgacggag
ttcgcgcgcg ataccgccga tatctccgac cggcagaacg tgcaatacca
ctggatcgaa gtggaaaacg tccctgaaat ctggcgacgg ttagacgatg
tcggacttca gaccaccgag gcgtgcggtg actgcccgcg ggtagtgctg
ggctcgccgt tggccggcga gtcgctcgac gaagtgctcg acccgacctg
ggcgatcgag gagatcgtgc gtcgctacat cggcaagccc gacttcgccg
acttgccgcg caagtacaag accgccatct ctggccttca ggacgtcgcg
cacgagatca acgacgtcgc cttcatcggc gtcaaccatc ccgagcacgg
accaggcctg gatctgtggg tgggcggtgg actgtcgacc aacccgatgc
tggcccagcg ggtcggcgcc tgggttccac tgggcgaagt gcccgaggtg
tgggcggcgg tcacctcggt gtttcgcgac tacggctacc ggcgactgcg
cgccaaggcc cggctgaaat ttctgatcaa agactggggc atagcgaagt
tccgcgaagt gctcgaaacc gagtacctca agcgtccgct gatcgacggt
ccggcccccg aaccggtcaa gcatccgatc gaccacgtcg gggtgcaacg
actcaagaac gggctcaacg ccgtcggagt cgcccccatc gccgggcggg
tatcgggcac catcctcacg gcggtcgccg acctgatggc gcgggccggt
tccgaccgga tccggttcac cccctaccag aagctggtca tcctcgacat
tccggacgcc ttgctcgacg acttgatcgc cggtctggac gcgctggggc
ttcagtcgcg cccgtcgcat tggcgccgga acttgatggc gtgcagcggg
attgagttct gcaagttgtc attcgccgaa acccgggttc gagcacagca
tttggtgccc gagctggaac gccggcttga ggacatcaac tcgcagctcg
acgtaccgat caccgtcaac atcaacggct gcccgaactc atgtgcgcga
gttcaaatcg ccgacatcgg attcaaggga cagatgatcg acgacggaca
cggcggctcc gtcgaaggct tccaggtgca tctgggcgga cacctcggcc
tggatgccgg attcggccgc aaactgcgcc agcacaaggt caccagtgac
gaactcggcg actacatcga ccgggtggtg cgcaacttcg tcaaacaccg
cagcgaaggt gaacgcttcg cgcagtgggt catccgggcc gaggaggacg
acctgcgatg agcggccgcc taggactact agtctgcagt ttttcctt
Mycobacterium smegmatis
This reference also contains the cloning of sirA from T. smegmatis.  We used the primers below taken from Pinto et al, blasted against the T. smegmatis genome and extracted the sirA sequence from genbank as above.
Primers:
smegsirA F      TACAGCTGATGCTCGAAGACGAGTACTTCAT
smegsirA R      CCCAAGCTTTCACGTTGCCTACCTCAAATCCGCTTCGTC
tacagctgat gctcgaagac gagtacttca tgctgcgcgt gcgctgcgat
ggtggcgcgc tgaccactgc agcgctgcgc acgctcggcg gcatctcgac
cgagttcgcg cgcgacaccg ccgacatctc cgaccgcgag aacgtccagt
accactggat ccaggtcgag aacatgcccg agatctggaa gcgcctcgac
gccgtcggcc tgcagaccac cgaggcgtgc ggcgactgcc cgcgtgtggt
cctcggctcg ccgctggccg gtgagtccct cgacgaggtg atcgacggga
cccccgcgat cgacgagatc gtgcgccgct acatcggcaa gcccgagtac
tcgaacctgc cgcgcaagtt caagaccgcg atctcggggc ttcaggacgt
ggtccacgag gtcaacgacg tcgcgttcat cggcgtcaac caccccgagc
acggcccggg cttcgacctg tgggtcggtg gcggcctgtc gaccaacccg
atgctggccc agcgcgtcgg ggtgtgggtg ccgctcgacg aggtgcccga
cgtctgggag ggcgtcgtca gcatcttccg cgactacggt taccggcgtc
tgcggtcgaa ggcgcggctg aagttcctga tcaaggactg gggcgtcgaa
aagttcaggg aagtgctgga aaccgagtac ctcaagcgcc ccctgatcga
cggcccggca cccgaaccgg tgacccgccc catcgaccac gtcggtgtgc
agaagctcaa gaacggcctc aacgccgtgg gcgtcgcccc gatcgcgggt
cgcgtctcgg gcacgatcct gaccaaggtg gccgatctcg ccgaggccgc
cgggtccgac cggatccgct tcacgccgta ccagaagctg atcatcctcg
acgtgcccga cgacaagatc gacgaactgc gcgctggcct cgacgcgctc
ggactgccgt cgacgccgtc gcactggcgc cgcaacctca tggcgtgcac
gggtatcgag ttctgcaagc tgagcttcgc cgagacccgc aagcgtgccc
aggtgctggt tcccgagctg gagaaacggc tcgacgacat caacgcccag
ctcgacgtgc ccatcacggt caacatcaac ggctgcccca actcgtgcgc
ccgtatccag gtcgccgaca tcgggttcaa gggccagatg gtcgacgatg
gcaacggccc cgaggagggt ttccaggtgc atctgggcgg cagcctgggc
ctggacagcg ggttcggccg caagctgcgc cagcacaagg tgctctcgtc
cgagctcggc gactacatcg agcgcgtcgt gcgcaacttc gtgaaacaac
gcgaggacgg cgagcgtttc gcccagtggg ccgtgcgggc cgacgaagcg
gatttgaggt aggcaacgtg aaagcttggg
As above, the sequence was imported into geneious, the biobrick restriction sites were manually removed from the sequence and restriction sites were added to the 5’ and 3’ ends in the same fashion as TB sirA.  This resulted in the following sequence for synthesis:
tacagaattc tagaccatgg gcctcgaaga cgagtacttc atgctgcgcg
tgcgctgcga tggtggcgcg ctgaccacta cagcgctgcg cacgctcggc
ggcatctcga ccgagttcgc gcgcgacacc gccgacatct ccgaccgcga
gaacgtccag taccactgga tccaggtcga gaacatgccc gagatctgga
agcgcctcga cgccgtcggc ctacagacca ccgaggcgtg cggcgactgc
ccgcgtgtgg tcctcggctc gccgctggcc ggtgagtccc tcgacgaggt
gatcgacggg acccccgcga tcgacgagat cgtgcgccgc tacatcggca
agcccgagta ctcgaacctg ccgcgcaagt tcaagaccgc gatctcgggg
cttcaggacg tggtccacga ggtcaacgac gtcgcgttca tcggcgtcaa
ccaccccgag cacggcccgg gcttcgacct gtgggtcggt ggcggcctgt
cgaccaaccc gatgctggcc cagcgcgtcg gggtgtgggt gccgctcgac
gaggtgcccg acgtctggga gggcgtcgtc agcatcttcc gcgactacgg
ttaccggcgt ctgcggtcga aggcgcggct gaagttcctg atcaaggact
ggggcgtcga aaagttcagg gaagtgctgg aaaccgagta cctcaagcgc
cccctgatcg acggcccggc acccgaaccg gtgacccgcc ccatcgacca
cgtcggtgtg cagaagctca agaacggcct caacgccgtg ggcgtcgccc
cgatcgcggg tcgcgtctcg ggcacgatcc tgaccaaggt ggccgatctc
gccgaggccg ccgggtccga ccggatccgc ttcacgccgt accagaagct
gatcatcctc gacgtgcccg acgacaagat cgacgaactg cgcgctggcc
tcgacgcgct cggactgccg tcgacgccgt cgcactggcg ccgcaacctc
atggcgtgca cgggtatcga gttctgcaag ctgagcttcg ccgagacccg
caagcgtgcc caggtgctgg ttcccgagct ggagaaacgg ctcgacgaca
tcaacgccca gctcgacgtg cccatcacgg tcaacatcaa cggctgcccc
aactcgtgcg cccgtatcca ggtcgccgac atcgggttca agggccagat
ggtcgacgat ggcaacggcc ccgaggaggg tttccaggtg catctgggcg
gcagcctggg cctggacagc gggttcggcc gcaagctgcg ccagcacaag
gtgctctcgt ccgagctcgg cgactacatc gagcgcgtcg tgcgcaactt
cgtgaaacaa cgcgaggacg gcgagcgttt cgcccagtgg gccgtgcggg
ccgacgaagc ggatttgagg taggcaacgt gaaagcctag gtactagtct
gcagtggg
Target
ASDF
Tuesday 11th June
Design of FdrA genes for synthesis
FdrA
Mycobacterium tuberculosis
Reference: McLean et al. 2003. Kinetic, spectroscopic and thermodynamic characterization of the Mycobacterium tuberculosis adrenodoxin reductase homologue FprA. doi:  10.1042/BJ20021692
Primers:
MtbfdrA F      TATGACTCATATGCGTCCCTATTACATCGCCATCG
MtbfdrA R      ATCCGGGATCCTCAGCCGAGCCCAATCCGCAACAGC
The above primers were blasted to retrieve sequence of FprA from genebank resulting in the following sequence:
tatgactcat atgcgtccct attacatcgc catcgtgggc tccgggccgt
cggcgttctt cgccgcggca tccttgctga aggccgccga cacgaccgag
gacctcgaca tggccgtcga catgctggag atgttgccga ctccctgggg
gctggtgcgc tccggggtcg cgccggatca ccccaagatc aagtcgatca
gcaagcaatt cgaaaagacg gccgaggacc cccgcttccg cttcttcggc
aatgtggtcg tcggcgaaca cgtccagccc ggcgagctct ccgagcgcta
cgacgccgtg atctacgccg tcggcgcgca gtccgatcgc atgttgaaca
tccccggtga ggacctgccg ggcagtatcg ccgccgtcga tttcgtcggc
tggtacaacg cacatccaca cttcgagcag gtatcacccg atctgtcggg
cgcccgggcc gtagttatcg gcaatggaaa cgtcgcgcta gacgtggcac
ggattctgct caccgatccc gacgtgttgg cacgcaccga tatcgccgat
cacgctttgg aatcgctacg cccacgcggt atccaggagg tggtgatcgt
cgggcgccga ggtccgctgc aggccgcgtt caccacgttg gagttgcgcg
agctggccga cctcgacggg gttgacgtgg tgatcgatcc ggcggagctg
gacggcatta ccgacgagga cgcggccgcg gtgggcaagg tctgcaagca
gaacatcaag gtgctgcgtg gctatgcgga ccgcgaaccc cgcccgggac
accgccgcat ggtgttccgg ttcttgacct ctccgatcga gatcaagggc
aagcgcaaag tggagcggat cgtgctgggc cgcaacgagc tggtctccga
cggcagcggg cgagtggcgg ccaaggacac cggcgagcgc gaggagctgc
cagctcagct ggtcgtgcgg tcggtcggct accgcggggt gcccacgccc
gggctgccgt tcgacgacca gagcgggacc atccccaacg tcggcggccg
aatcaacggc agccccaacg aatacgtcgt cgggtggatc aagcgcgggc
cgaccggggt gatcgggacc aacaagaagg acgcccaaga caccgtcgac
accttgatca agaatcttgg caacgccaag gagggcgccg agtgcaagag
ctttccggaa gatcatgccg accaggtggc cgactggcta gcagcacgcc
agccgaagct ggtcacgtcg gcccactggc aggtgatcga cgctttcgag
cgggccgccg gcgagccgca cgggcgtccc cgggtcaagt tggccagcct
ggccgagctg ttgcggattg ggctcggctg aggatcccgg at
As previously, the sequence was imported into geneious, the biobrick restriction sites were manually removed from the sequence and restriction sites were added to the 5’ and 3’ ends in the same fashion as TB sirA.  However, in the case of FdrA, the restriction sites added for cloning into the vector pCOLAduet-1 were NdeI and AvrII.  As a result, there was no need to add an additional glycine after the start codon.  The resulting sequence is as follows:
tatgaattct agatcatatg cgtccctatt acatcgccat cgtgggctcc
gggccgtcgg cgttcttcgc cgcggcatcc ttgctgaagg ccgccgacac
gaccgaggac ctcgacatgg ccgtcgacat gctggagatg ttgccgactc
cctgggggct ggtgcgctcc ggggtcgcgc cggatcaccc caagatcaag
tcgatcagca agcaattcga aaagacggcc gaggaccccc gcttccgctt
cttcggcaat gtggtcgtcg gcgaacacgt ccagcccggc gagctctccg
agcgctacga cgccgtgatc tacgccgtcg gcgcgcagtc cgatcgcatg
ttgaacatcc ccggtgagga cctgccgggc agtatcgccg ccgtcgattt
cgtcggctgg tacaacgcac atccacactt cgagcaggta tcacccgatc
tgtcgggcgc ccgggccgta gttatcggca atggaaacgt cgcgctagac
gtggcacgga ttctgctcac cgatcccgac gtgttggcac gcaccgatat
cgccgatcac gctttggaat cgctacgccc acgcggtatc caggaggtgg
tgatcgtcgg gcgccgaggt ccgcttcagg ccgcgttcac cacgttggag
ttgcgcgagc tggccgacct cgacggggtt gacgtggtga tcgatccggc
ggagctggac ggcattaccg acgaggacgc ggccgcggtg ggcaaggtct
gcaagcagaa catcaaggtg ctgcgtggct atgcggaccg cgaaccccgc
ccgggacacc gccgcatggt gttccggttc ttgacctctc cgatcgagat
caagggcaag cgcaaagtgg agcggatcgt gctgggccgc aacgagctgg
tctccgacgg cagcgggcga gtggcggcca aggacaccgg cgagcgcgag
gagctgccag ctcagctggt cgtgcggtcg gtcggctacc gcggggtgcc
cacgcccggg ctgccgttcg acgaccagag cgggaccatc cccaacgtcg
gcggccgaat caacggcagc cccaacgaat acgtcgtcgg gtggatcaag
cgcgggccga ccggggtgat cgggaccaac aagaaggacg cccaagacac
cgtcgacacc ttgatcaaga atcttggcaa cgccaaggag ggcgccgagt
gcaagagctt tccggaagat catgccgacc aggtggccga ctggctagca
gcacgccagc cgaagctggt cacgtcggcc cactggcagg tgatcgacgc
tttcgagcgg gccgccggcg agccgcacgg gcgtccccgg gtcaagttgg
ccagcctggc cgagctgttg cggattgggc tcggctgagg atccctagga
ctagtctgca gttacg
Mycobacterium smegmatis
We didn't find a reference of M. Smegmatis FprA being cloned into E. coli so we used the following sequenced data:
atgcgcccgt accacgtagc gatcgtcggc tcaggaccct ccggattctt
tgctgccgca tcgctgctga agtttgccga ctctcaaccc gaccgcgatg
tccgcgtgga catgctcgag atgctgccga ccccgtgggg tctggtgcgc
tccggcgtcg ctcccgacca tccgaagatc aagtcgatca gcgcccagtt
cgagaagacc gcggccgacc cccgcttccg gttcttcggc aacgtccggg
tcggtgagca cgtccaaccg ggcgaactcg ccgaacgcta cgacgccgtc
gtctacgcca ccggcgcaca gtccgaccgc gcgctgaaca tccccggcga
ggagctgccg ggcagcgtcg cggccgtcga cttcgtgggt tggtacaacg
cgcatccgca tttccgcgag atggcgcccg atctctccgg cgggcgcgcc
gtggtggtcg gcaacggcaa cgtcgcactc gatgtcgcac gcatcctggt
cagcgatccg aaggcgttgg ccaacaccga cattgccgac catgcgctgg
acaagctgga cacgcgcggt gtggacgagg tcgtggtgct cggccggcgt
ggtccgctgc aggcgacgtt caccacgctg gagttgcgtg aactcggcga
catggagggc ctcggcgacg tcgacgtgat cctggatccg gccgatttcg
ccgacatcac cgacgaggat ctcgaagccg cgggcaagac cgtcaagcag
aacatcaagg tgctgcgcgg ttacgccgag cgggaaccgc gaggcgccaa
gcggcgcatc gtgttccggt tcttcacctc accgatcgag ctgcgcggcg
aggaccgcgt ggaatcgatc gtgttgggac gcaacgaact cgtgcgcggc
gccgacggca ggatggtggc caaggacacc ggtgagcgcg aagaactgcc
ggcacagctg gtggtgcgtg cggtcggata ccgcggcgtg ccgactccag
gcctgccgtt cgacgagcgt tcgggcacca tcccgcacac cgacggccgc
gtcgagggca gcgccaacga gtacgtggtg ggctggatca agcgcggacc
gtccggcgtc atcggcagca acaagaagga ctcacaggac accgtcaaca
ccctgctcga cgacctcgcc gcggcccggc tgccccagcg cgggcccgac
cactccgaga agctcgcggc gtggttgctg gaacgtcagc ccaaggtggt
cacgggcgag cactggaagc tgatcgacga ctacgagcgc gccgcgggcg
aaccgaccgg ccgaccccgg gtgaagctgg cgagcgtggc tgaactgctg
cgcgtgggcc acggctga
As above, the sequence was imported into geneious, the biobrick restriction sites were manually removed from the sequence and restriction sites were added to the 5’ and 3’ ends in the same fashion as TB sirA.  Again, the restriction sites added for cloning into the vector pCOLAduet-1 were NdeI and AvrII.  As a result, there was no need to add an additional glycine after the start codon.  The resulting sequence is as follows:
tatgaattct agatcatatg cgcccgtacc acgtagcgat cgtcggctca
ggaccctccg gattctttgc tgccgcatcg ctgctgaagt ttgccgactc
tcaacccgac cgcgatgtcc gcgtggacat gctcgagatg ctgccgaccc
cgtggggtct ggtgcgctcc ggcgtcgctc ccgaccatcc gaagatcaag
tcgatcagcg cccagttcga gaagaccgcg gccgaccccc gcttccggtt
cttcggcaac gtccgggtcg gtgagcacgt ccaaccgggc gaactcgccg
aacgctacga cgccgtcgtc tacgccaccg gcgcacagtc cgaccgcgcg
ctgaacatcc ccggcgagga gctgccgggc agcgtcgcgg ccgtcgactt
cgtgggttgg tacaacgcgc atccgcattt ccgcgagatg gcgcccgatc
tctccggcgg gcgcgccgtg gtggtcggca acggcaacgt cgcactcgat
gtcgcacgca tcctggtcag cgatccgaag gcgttggcca acaccgacat
tgccgaccat gcgctggaca agctggacac gcgcggtgtg gacgaggtcg
tggtgctcgg ccggcgtggt ccgcttcagg cgacgttcac cacgctggag
ttgcgtgaac tcggcgacat ggagggcctc ggcgacgtcg acgtgatcct
ggatccggcc gatttcgccg acatcaccga cgaggatctc gaagccgcgg
gcaagaccgt caagcagaac atcaaggtgc tgcgcggtta cgccgagcgg
gaaccgcgag gcgccaagcg gcgcatcgtg ttccggttct tcacctcacc
gatcgagctg cgcggcgagg accgcgtgga atcgatcgtg ttgggacgca
acgaactcgt gcgcggcgcc gacggcagga tggtggccaa ggacaccggt
gagcgcgaag aactgccggc acagctggtg gtgcgtgcgg tcggataccg
cggcgtgccg actccaggcc tgccgttcga cgagcgttcg ggcaccatcc
cgcacaccga cggccgcgtc gagggcagcg ccaacgagta cgtggtgggc
tggatcaagc gcggaccgtc cggcgtcatc ggcagcaaca agaaggactc
acaggacacc gtcaacaccc tgctcgacga cctcgccgcg gcccggctgc
cccagcgcgg gcccgaccac tccgagaagc tcgcggcgtg gttgctggaa
cgtcagccca aggtggtcac gggcgagcac tggaagctga tcgacgacta
cgagcgcgcc gcgggcgaac cgaccggccg accccgggtg aagctggcga
gcgtggctga actgctgcgc gtgggccacg gctgacctag gactagtctg
cagttacg
Target
ASDF
Wednesday 12th June
Design of FdxC and FdxA genes for synthesis
FdxC
Mycobacterium tuberculosis
Reference: Cole, S.T. et al. 1998. Deciphering the biology of Mycobacterium tuber- culosis from the complete genome sequence. doi:10.1038/31159.
We couldn’t find a reference of FdxC being cloned into E. coli and so we took the sequence data from genbank which is as follows:
gtgacgtaca cgatcgccga accctgtgtc gacatcaagg acaaggcatg
cattgaggag tgcccggtcg attgcatcta cgagggcgcc cggatgctgt
atatccaccc cgacgaatgc gtcgactgtg gggcttgcga gccggtctgc
cccgttgaag ctatcttcta cgaagacgat gtgcccgaac agtggagcca
ttacacccag atcaacgccg atttcttcgc cgagctggga tcgccgggcg
gtgcggccaa ggttggcatg accgagaacg acccgcaagc ggtcaaggat
ctggcgccgc agagcgagga cgcctga
The Gtg was replaced with the NcoI cut site and a glycine codon (ccatgggc) and a HindIII site was added to the end of the sequence.  The biobrick restriction sites were then added to the ends of the sequence resulting in the following sequence:
cttagaattc tagaccatgg gcacgtacac gatcgccgaa ccctgtgtcg
acatcaagga caaggcatgc attgaggagt gcccggtcga ttgcatctac
gagggcgccc ggatgctgta tatccacccc gacgaatgcg tcgactgtgg
ggcttgcgag ccggtctgcc ccgttgaagc tatcttctac gaagacgatg
tgcccgaaca gtggagccat tacacccaga tcaacgccga tttcttcgcc
gagctgggat cgccgggcgg tgcggccaag gttggcatga ccgagaacga
cccgcaagcg gtcaaggatc tggcgccgca gagcgaggac gcctgaagct
tgactagtct gcagaaatg
FdxA
Mycobacterium smegmatis
Reference: Ricagno S. et al. 2008. The crystal structure of FdxA, a 7Fe ferredoxin from Mycobacterium smegmatis. http://dx.doi.org/10.1016/j.bbrc.2007.06.013 http://www.metacyc.org/MSME246196/NEW-IMAGE?type=GENE-IN-MAP&object=GJ4Y-1124
Sequence:
atgacatacg tgatcggccg gccgtgcgtg gacgtcaaag accgcgcatg
cgtggatgag tgcccggtcg actgcatcta cgagggcgcg cggatgctct
acatccaccc cgacgaatgt gtcgactgtg gcgcgtgtga acccgtgtgc
ccggtggagg cgatctacta cgaggacgac ctgcccgagg atctgcagcc
gtaccaggag gagaacgcga agttcttcac cgatgtcctg cccgggcgtg
cccaaccgct cggatcaccg ggcggtgccg cgaaactcgg tgtcgtggac
gccgatacgc ccatggttgc ggagctaccg ccgcaggggg attga
The NcoI cut site and a glycine codon (ccatgggc) replaced the atg in green and a HindIII site was added to the end of the sequence.  The biobrick restriction sites were then added to the ends of the sequence resulting in the following sequence:
cttagaattc tagaccatgg gcacatacgt gatcggccgg ccgtgcgtgg
acgtcaaaga ccgcgcatgc gtggatgagt gcccggtcga ctgcatctac
gagggcgcgc ggatgctcta catccacccc gacgaatgtg tcgactgtgg
cgcgtgtgaa cccgtgtgcc cggtggaggc gatctactac gaggacgacc
tgcccgagga tcttcagccg taccaggagg agaacgcgaa gttcttcacc
gatgtcctgc ccgggcgtgc ccaaccgctc ggatcaccgg gcggtgccgc
gaaactcggt gtcgtggacg ccgatacgcc catggttgcg gagctaccgc
cgcaggggga ttgagcctag gtactagtct gcagtggg
Target
ASDF
Tuesday 18th June
In Silico Cloning
We used geneious to perform in silico cloning on our constructed gene sequences to ensure that they were properly made.  SirA and pACYCDuet-1 were both cut with NcoI and AvrII and then the large fragments were extracted and ligated together to form vector pD001.  FdxA and pETDuet-1 were both cut with NcoI and HindIII and the large fragments were extracted and ligated together to form vector pD002.  FprA and pD002 were both cut with NdeI and AvrII and the large fragments were extracted and ligated together to form vector pD003.  Please see Geneious files on Google Drive (or send request for files) for exact sequences.
Target
ASDF
24th June
Codon Optimization
The ORF of M. Smegmatis SirA, FprA, and FdxA were codon optimized for E. coli K-12 using JCat (www.jcat.de) with options to avoid using rho-independent terminators, prokaryotic RBS, and restriction sites used in vector construction (AvrII, HindIII, NdeI, NcoI) and Biobrick restriction sites (EcoRI, PstI, SpeI, XbaI).  The optimization charts are shown below:
Target
ASDF
Monday 1st July
Media was made, Strains sD001-sD004 were received, inoculated, and put into stock and catalog.
4 strains were received from Jake:
E. coli: BL21 (DE3) ko20 ΔcysI, Δfpr, ΔydbK
E. coli: NEBTurbo zmSIR Chloramphenicol
E. coli: NEBTurbo zmFNR Spectinomycin
E. coli: NEBTurbo soFD, zmSIR Chloramphenicol
Media and Glycerol stock were prepared
Media Preparation
3 500ml bottles of LB broth and LB agar were prepared by standard methods.
12.5g/500ml powder/water for broth and 20g/500ml powder/water for agar.
Bottles were autoclaved.  Additionally 1 flask of 500ml of LB broth was made in the same fashion.
Glycerol Stocks
Single colonies from agar plates were picked and used to innoculate 5ml LB broth overnight.
750ml of overnight culture was added to 250ml of 60% glycerol in a cryotube.
Two sets of Glycerol stocks were used for each of sD001, sD002, sD003, sD004; one set was frozen at -20ºC and the other set was frozen at -80ºC.
Target
ASDF
Tuesday2nd July
MiniPrep: Plasmids pD004, pD005 and pD006 were extracted from NEBturbo vectors
Plasmid Extraction
Plasmids pD004 (pCDF.ew12 zmFNR Spectinomycin), pD005 (pACYC.ew13 soFD, zmSIR Chloramphenicol), pD006 (pACYC.ew17 zmSIR Chloramphenicol) were extracted from NEBTurbo cells using a Thermo Scientific GeneJet Plasmid mini prep kit as described in protocol ().  Lacking Resuspention solution we used some from a different mini prep upstairs.  Plasmids were eluted in 100ul of nanopure water and frozen at -20ºC.  More notes here tra la lalal
Target
ASDF
Wednesday 3rd July
Chemically Competent Cells : BL21 (DE3) dCysI dFpr dydbk, NEBturbo stocks
Chemically Competent BL21 (DE3) Cells were made
We grew 5ml of sD001 cells overnight from a single colony.  This 5ml was used to innoculate 500ml of LB broth.
Broth was incubated for 1.5h and optical density was read at 0.62 OD600.
Cells were alloquoted into 10 falcon tubes (50ml each) and centrifuged at 4000xg for 20 minutes at 4C.
Supernatant was removed and cells were resuspended in 200ml of Buffer 1 (4x 50ml) and left on ice for 20 minutes.
Cells were centrifuged again for 20 minutes at 4000xg.
Supernatant was removed and cells were resuspended in 12ml Buffer 2.
Cells were alloquoted at 500ul into microcentifuge tubes and frozen at -80C.
Buffer 1:  50mM CaCl2
Buffer 2:
0.53ml 2M CaCl2
2.8ml 60% Glycerol
8.67ml sterile H2O
This is the correct Chemical Competent Cells protocol (Similar to Protocol 2) , that we should have used instead of the one above:
Take a single colony of E. coli cells and inoculate 5ml of LB broth.
Incubate over night at 37C and 200rpm and innoculate about 500ml to 1L sterile LB broth.
DO NOT ADD antibiotics since these cells do not have a plasmid in them.
Work as sterile as possible.
Inoculate about 300-400ml sterile LB broth with 500ul of preculture.
Grow the cells on a shaker at 37C until they reach an OD @ 600nm of 0.3 to 0.4 (1cm pathlength of the cuvette).
Centrifuge at 3000xg for 10 minutes at 4C.  Ice down 100mM CaCl2 and 100mM MgCl2 solutions at this point.
Gently resuspend the bacteria pellet on ice in 50ml of ice cold 100mM MgCl2, taking 3-5 minutes for this procedure.
Centrifuge the cell suspension at 3000xg for 10 minutes at 4C.
Resuspend the bacteria pellet on ice 50ml of ice cold 100mM CaCl2, incubate the cells for 10 minutes on ice.
Centrifuge the cell suspension at 3000xg for 10 minutes at 4C and resuspend the cell pellet in 4ml of ice cold, sterile 100mM CaCl2 in 15%(w/v) glycerol.
Dispense in 50uL aliquots and freeze cells at -80C.
Chemically competent cells (NEB Turbo)
Prepared via iGEM chemically competent protocol with following changes.
500ul of preculture used to inoculate 500ml LB broth.  Grown at 37C for 3h hours to an OD600 of 0.536.
Cells centrifuged at 3200 rpm on old centrifuge for 10m at 4C.  Pellets resuspended in 80ml of CCMB80 buffer.
Incubated on ice for 20m.  Centrifuged for 10m at 3200 rpm for 10m at 4C.
Pellets resuspended in 5ml of CCMB80 buffer.  Aliquoted into Microcentrifuge tubes and frozen at -80C.
Target
ASDF
Thursday 4th July
Heat Shock Transformation
Preparation of Agar Plates
LB Agar was melted; 100ml was used to make 8 ampicillin plates,
100ml was used to make 8 spectinomycin plates and the remaining was used to make 20 chloramphenicol plates.
Antibiotic concentrations are: Ampicillin 50ug/ml, Spectinomycin 50ug/ml, Chloramphenicol 34ug/ml.
Heat Shock Transformation
sD001: BL21 (DE3) with deletions, was transformed with the following plasmids and plasmid combinations:
sD001+pD004 ( FNR, Spectinomycin)
sD001+pD005 ( SIR, Fdx, Chloramphenicol)
sD001+pD006 (Sir, Chloramphenicol)
Transformation was done according to protocol 1 with the following specifications:
BL21 (DE3) chemically competent cells were thawed on Ice.  1ul of plasmid DNA was added to a tube of BL21 (DE3) cells.
Cells were incubated on ice for 30 minutes.  Cells were transfered to heating block at 42C for 45s.
Cells were returned to ice for 2 minutes.  1ml of LB Broth was added to the cells and they were incubated at 37C for 1h.
Cells were then plated on agar with appropriate antibiotics (specified previously). These transformations worked; single colonies were lifted from plates, entered to catalog and stocked in glycerol stock (protocol 3).
Catalog names:
sD006= sD001+pD004 ( FNR, Spectinomycin)
sD007= sD001+pD005 ( SIR, Fdx, Chloramphenicol)
sD008= sD001+pD006 (Sir, Chloramphenicol)
Target
ASDF
Friday 5th July
Planning transformations for the next week
We decided to partly reproduce Figure 3C from: http://www.jbioleng.org/content/5/1/7 :
So, we would pursue to grow the following strains on a minimal media M9 plate, supplemented with all amino acids
aside from the sulfur containing ones, and sulfite ans the only sulfur source:
We already thought we have all of the strains we need, however, an overnight culture was launched of a few single colonies from this double plasmid strain (sD001+pD004+pD005, which supposedly had all 3 genes SIR FNR and Fdx) and it didn't grow.
This most likely occurred because the strains didn't have Spec antibiotics resistance, as the other spec resistant strain containing
the FNR gene (pD004), was found not to be resistant as well.
Illustration  1: Strains will be grown on a minimal media M9 plate, supplemented  with all amino acids aside from the sulfur containing ones; plates  will have increased glucose, and sulfite as the only sulfur source.  Growth is expected only for WT and the strain containing all three  alternative genes of sulfur reduction.
Therefore, we would  re-make:
sD001+pD004+pD005 E.coli:ΔcysI,  Δfpr, ΔydbK:: soFD, zmSIR ;Chloramphenicol  ,zmFNR;  Spectinomycin- with  all three genes.
sD001+pD004: E.coli:ΔcysI,  Δfpr, ΔydbK:: zmFNR;  Spectinomycin
and make
sD001+pD004+pD006 E.coli: ΔcysI,  Δfpr, ΔydbK::zmFNR;  Spectinomycin, zmSIR;  Chloramphenicol
These transformations will be done by using two strains of competent cells prepared using Protocol 2 :
sD001  competent cells were transformed with:
pD004
Double   transformation: pD005 +pD004
sD007  (sD001+pD005) competent cells were transformed with:
pD004
These transformations will be done by heat shock using protocol 1.
Target
ASDF
Tuesday 16th July
Transformation efficiency measurements
Transformation of Puc18 to sD001 (BL21 ED3) competent cells. We have made a transformation of pUC18 into our sD001 BL21(DE3) strain  of competent cells.
Transformation was done by heat shock protocol 1 with the following specifications:
200 ul Chemically Competent Cells were thawed on Ice. 0.5 ul DNA was added and incubate on ice for 30 minutes.
HEAT SHOCK cells were Incubated at 42C for 45 seconds then incubated on ice for 2 minutes. 200 ul of LB broth was added and cells were incubated at 37C for 1 hour.
10 ul of cells were plated on agar supplemented with Ampicillin. During the night 18 colonies grew.
Target
ASDF
Wednesday 17th July
Making sD001+pD004 (Spect), sD001+pD005 (Chl31) competent
Making sD001+pD004 (Spect), sD001+pD005 (Chl31) competent, by using protocol 2 with these specifications:
The night before, cells were inoculated in a 5 ml culture and grown overnight with selection.
0.05 ml of the cell culture was diluted ~ 1:200 into 10 ml of selective media (Chl31,Spect) and grew to an OD600 of 0.6 – 0.7
(sD001+pD004 didn’t grow and protocol was continued for sD001+pD005  - see comment). Cells were Spun down at 4 ºC, 4000 rpm, 15 minutes.
Then were resuspended in 15 ml, ice-cold 100 mM CaCl2 and left for 3 hours.
Next, cells were spun down at 4 ºC, 4000 rpm, for 15 minutes, Resuspended in 4 ml, ice-cold 100 mM CaCl2 + 15% glycerol
and aliquoted into pre-chilled Eppendorf tubes. Tubes were immediately  stored at -80ºC.
10 ml CaCl2 100nM + glycerol 15% was made from:
3.33 ml Glycerol 60% solution.
6.17 ml Sterile water
0.5 ml CaCl2 2 M solution.
Comment: sD001+pD004 did not grow overnight (and possibly always) in Streptavidin
instead of Spectinomycin so they most likely lost their resistance and we will make them again tomorrow by tranforming the BL21(DE3) cells with pD004.
Target
ASDF
Thursday 18th July
Heat shock Transformation
Transformations made:
sD001+pD004+pD005 E.coli:ΔcysI, Δfpr, ΔydbK:: soFD, zmSIR ;Chloramphenicol ,zmFNR; Spectinomycin- with all three genes
sD001+pD004 E.coli:ΔcysI, Δfpr, ΔydbK:: ,zmFNR; Spectinomycin
sD001+pD004+pD006 E.coli: ΔcysI, Δfpr, ΔydbK::zmFNR; Spectinomycin, zmSIR; Chloramphenicol
Transformations were made with protocol 1 with the following specifications:
BL21 (DE3) chemically competent cells were thawed on Ice.
2ul of plasmid DNA was added to 20ul  of BL21 (DE3) cells.  Cells were incubated on ice for 30 minutes.
Cells were transfered to heating block at 42C for 45s.  Cells were returned to ice for 2 minutes.
200ul of LB Broth was added to the cells and they were incubated at 37C for 1h.
After incubations cells of each transformation were plated on two plates: one with 5 ul , and the other with 100 ul of the post recovery cells culture.
Target
ASDF
Friday 19th July
TRANSFORMATIONS WORKED!!! cells were stocked and entered in catalog.
sD012/sD013=sD001+pD004+pD005 E.coli:ΔcysI, Δfpr, ΔydbK:: soFD, zmSIR ;Chloramphenicol ,zmFNR; Spectinomycin- with all three genes
sD016/sD017=sD001+pD004 E.coli:ΔcysI, Δfpr, ΔydbK:: ,zmFNR; Spectinomycin
sD014/sD015=sD001+pD004+pD006 E.coli: ΔcysI, Δfpr, ΔydbK::zmFNR; Spectinomycin, zmSIR; Chloramphenicol
Glycerol Stocks were made of the above mentioned strains
2 Single colonies from each agar plate of above mentioned strains were picked and used to inoculate 5ml LB broth overnight.
750ml of overnight culture was added to 250ml of 60% glycerol in a cryotube.
Two sets of Glycerol stocks were used for each of sD001+pD004, sD001+sD004+sD005, sD001+sD004+sD006;
one set was frozen at -20ºC and the other set was frozen at -80ºC.
Target
ASDF
Wednesday 24th July
24/7/13
Made 500 ml of M9 Minimal Plates with Sulfur Dropout Powder accouding to the following recipe:
Recipe: M9 Minimal Plates with Sulfur Dropout Powder
These stock solutions can be pre-sterilized and stored indefinitely:
5x M9 Salts (from scratch)
Reagent
Quantity
Na2HPO4
17    g
KH2PO4
7.5    g
NaCl
1.25    g
(NH4)2SO4
3.0    g
Bring to a total volume of 500 ml.
Adjust pH to 7.4 with NaOH.
Sterilize by autoclaving or filtration
or
5x M9 Salts (from Difco Powder)
Reagent
Quantity
M9    Salts
28.2    g
Bring to a total volume of 500 mL.
Sterilize by autoclaving or filtration
2x Yeast Plate Agar
Reagent
Quantity
Agar
30    g
Bring to a total volume of 500 ml.
Sterilize by autoclaving
5x Amino Acid Dropout Powder
Reagent
Quantity
Sulfur    dropout powder
0.5    g
Bring to a total volume of 200 ml.
Sterilize by autoclaving
Sulfur dropout powder is a rich supplement mix with cysteine and methionine omitted:
Amino    acids
Nucleotide    bases
Alanine
2.0    g
Leucine
10.0    g
Adenine
0.5    g
Arginine
2.0    g
Lysine
2.0    g
Uracil
2.0    g
Asparagine
2.0    g
Methionine
0.0    g
Aspartic    acid
2.0    g
Phenylalanine
2.0    g
Vitamins
Cysteine
0.0    g
Proline
2.0    g
p-Aminobenzoic    acid
0.2    g
Glutamic    acid
2.0    g
Serine
2.0    g
Inositol
2.0    g
Histidine
2.0    g
Tyrosine
2.0    g
Isoleucine
2.0    g
Valine
2.0    g
Recipe: M9 Minimal Plates with Sulfur Dropout Powder
The plates can be made fresh from pre-sterilized stock solutions.
Reagent
Quantity
5x    M9 Salts
200    mi
5x    Amino Acid Dropout Powder
200    mi
1    M MgSO4
2    ml
20%    Glucose
20    ml
1M    CaCl2
0.1    ml
Bring to a total volume of 500 mL
Stir and warm on a hot plate to about 60 C (hot to the touch)
Melt 500 ml 2x Yeast Plate Agar in the microwave for ~8 minutes on high
Cool to about 60 C (hot to the touch)
This takes about 30 minutes on the bench or 5 minutes in a water bath.
Add the 2x Yeast Plate agar to the other reagents to make 1 L of media.
This recipe will prepare 20 plates at 25 ml / plate.
made 7 plates without antibiotics and 20 with Chloramphenicol and Spectinomycin - by mistake.
Used one plate without AB to plate the following strains:
sD007=sD001+pD005
sD008=sD001+pD006
sD013=  sD001+pD005+ pD004
sD015=  sD001+pD006+ pD004
Target
ASDF
Monday 5thAugust
Confirmation of strains SD002 - SD018
Strains sd002-sd018 were grown overnight in 5ml LB.
Target
ASDF
Monday 5thAugust
Confirmation of strains SD002 - SD018 using colony PCR
Fresh single colonies of strains SD002-SD018 were prepared for PCR using the colony pcr protocol. The primers used for were FD_F, FD_R, FNR_F, FNR_R and SirA_F and SirA_R for each strain.
Strains SD002-SD018 were confirmed.
Target
ASDF
Day Numbersuffix Month
Place your twit here
Received FrpA and FdxA from IDT.
Started cloning strategy:
Ligated FdxA G block into pJET plasmid for stock purposes, using the following protocol:
1. Set up the blunting reaction on ice:
Component
Volume
2X Reaction Buffer
10 µl
Non-purified PCR product
or
purified PCR product/other sticky-end DNA fragment
1 µl
0.15 pmol ends
Water, nuclease-free
to 17 µl
DNA Blunting Enzyme
1 µl
Total volume
18 µl
Vortex briefly and centrifuge for 3-5 s.
2. Incubate the mixture at 70°C for 5 min. Chill on ice.
3. Set up the ligation reaction on ice. Add the following to the blunting reaction mixture:
Component
Volume
pJET1.2/blunt Cloning Vector (50 ng/µl)
1 µl (0.05 pmol ends)
T4 DNA Ligase
1 µl
Total volume
20 µl
Vortex briefly and centrifuge for 3-5 s to collect drops.
4. Incubate the ligation mixture at room temperature (22°C) for 5 min.
Target
ASDF
Tuesday 20th August
Cloning of FdxA and FprA
Digestion of FprA with NdeI and AvrII
Gel Clean up of FprA fragment
Digestion of pETDuet-1 with NdeI and AvrII
Digestion of FdxA with NcoI and HindIII
Digestion of pETDuet-1 NcoI and HindIII
Target
ASDF
Wednesday 21st August
Ligation and Transformations
Ligation of FprA NdeI AvrII with pET NdeI AvrII
Ligation of FdxA NcoI HindIII with pET NcoI HindIII
Transformation of pET FprA into NEBTurbo
Transformation of pET FdxA into NEBTurbo
Target
ASDF
Thursday 22nd August
Nano drop
Plasmid mini-prep of transformations
Nano dropped
pETDuet: 59.1ng/ul
pETDuet: 45.9ng/ul
pJET Fdx: 54.2ng/ul
pLS FRP: 19.7ng/ul
Digest pLS FrpA with NdeI and AvrII
Digest pETDuet with NdeI and AvrII
Digest FdxA gBlock with NcoI and HindIII
Digest pETDuet with NcoI and HindIII
Ligation of FprA with pETDuet
Ligation of FdxA with pETDuet
Transformation of pET FprA and pET FdxA into NEBTurbo
PCR of
FdxA gBlock with FdxA F/R
pLS FprA with FprA F/R
pET duet with MCSI F/R
pACYC duet with MCSI F/R
Target
ASDF
Friday 23rd August
Patching
Patched colonies of pET FprA and colonies of pJET FdxA
Transformations of pET Fdx didn't work
Checking strains containing plasmids fpr to make sure genes are there.
Results from Thursdays PCR:
no band for fdxA
band for fprA
band for pET
no band for pACYC
Target
ASDF
Saturday 24th August
Weekend warriors!
Nanodrop
FdxA 1 3.1ng/ul
FdxA 2 4.3ng/ul
pET Duet 6.2ng/ul
pET Duet 4.9ng/ul
Checking FdxA gBlock by PCR
1 ul Fdx F
1 ul Fdx R
1 ul Template
7 ul H20
10 ul dreamTaq 2x master mix
PCR failed.
Target
ASDF
Sunday 25th August
Ligations
For FprA and pET Duet:
Miniprep more plasmid
Digest plasmid and vector backbone
Check DNA concentration
Ligate
Target
ASDF
Monday 26th August
P1 phage transduction
The p1 phage lysate was prepared as described in the P1 phage transduction protocol.
BL21 (DE3) ΔCysI Δfpr ΔydbK was grown in 5ml Lb.
One colony was discovered in transformed plate.  This colony was re-streaked and grown overnight.
the ligation product from overnight ligation was used to transorm NEB cc cells by heatshock
5ul plate gave a few colonies
10ul plate gave many colonies
Target
ASDF
Tuesday 27th August
Sequencing FdxA
pJET FdxA sent for sequecing
Steps 2-7 of the P1 phage transduction: Lysate preparation were carried out.
BL21(AI) was picked and placed into 5ml LB and incubated at 37C overnight.
Target
ASDF
Wednesday 28th August
P1 phage transduction
The transduction section of the P1 phage transduction protocol was carried out.     Cells were plated on Kanamyocin plates to select for resistance and grown overnight.
Target
ASDF
Thursday 29th August
Colonies from the P1 phage transduction grew on the Kanomycin plates.
Selected colonies were patched onto Kan and LBA plates and left to grow overnight.
Target
ASDF
Friday 30th August
Confirmation of P1 phage transduction:
Colony PCR was used on patched colonies from the p1 phage transduction. The primers used for each colony were CysI F/R and KanR F/R.
PCR confirmed that CysI was deleted in BL21(AI) and replaced with a Kanomycin resistance gene.
2 Single colonies from each agar plate of the patched p1 phage transduced plates were picked and used to inoculate 5ml LB broth overnight.  750ml of overnight culture was added to 250ml of 60% glycerol in a cryotube.
Target
ASDF
Monday 9th September
Sevres Workshop
Sevres Workshop
Target
ASDF
Tuesday 10th September
Sevres Workshop
Sevres Workshop
Target
ASDF
Monday 16th September
Cloning gBlock FdxA and Biobricking FprA and SirA
FprA and SirA were PCRed with FprA BB F, FprA BB R, SirA BB F and SirA BB R primers respectfully.  The PCR was run with an annealing temperature of 60C with an extension time of 1m40s using Phusion polymerase.  The band of ~1400bp was cut out and cleaned up using a Quagen Gel Extraction Kit.
Restriction Digestion of gBlock FdxA with NcoI and HindIII.  pET FprA was also digested with NcoI and HindIII. Digestion lasted 30 minutes at 37C afterwhich enzymes were heat inactivated at 80C for 20 minutes.  Digestions were ligated together using 6.5ul of FdxA insert and 2ul of FprA into a total volume of 10ul.
PCR products for FprA and SirA along with linearized pSB1c3 backbone were digested with EcoRI and PstI for 30m at 37C.  Restriction enzymes were then heat inactivated at 80C for 20 minutes.  Digestions were then ligated together using equal amounts of 2ul insert to 2ul of backbone.
Target
ASDF
Tuesday 17th September
Transforming gBlock FdxA and Biobricking FprA and SirA
Ligations were transformed into E. coli NEBTurbo Chemically Competent Cells by Heat Shock transformation. Cells were recovered for 1.5 hours before being plated.  pSB1c3 fprA and pSB1c3 SirA were plated onto Chloramphenicol plates (34ug/ul) and pET FprA FdxA were plated onto Ampicillin (50ug/ul) plates.
Target
ASDF
Wednesday 18th September
Patching gBlock FdxA and Biobricking FprA and SirA
Colonies from Transformation were patched onto appropriate antibiotic plates, 15 colonies from SirA were patched, 6 colonies from FprA were patched and 4 colonies from FdxA were patched.
Target
ASDF
Thursday 19th September
PCR Verification of vectors
Vectors (pET FprA FdxA, pSB1c3 SirA, and pSB1c3 FprA) were used for colony PCR using DreamTaq.  Primers used were:
pET FprA FdxA: MCSI F/R, MCSII F/R, FprA F/R, FdxA F/R
pSB1c3 SirA: SB-prep-3P-1/SB-prep-2Ea, SirA F/R
pSB1c3 FprA: SB-prep-3P-1/SB-prep-2Ea, FprA F/R
Target
ASDF
Saturday 21st September
Weekend Rush
Gibson assembly of pSB9S1, pSB10K1, and pSB3C7.
Digestion of sirA, fprA, fdxA and linearized backbone with XbaI and SpeI
Digestion of Linearized pSB1C3 with XbaI, SpeI, and DpnI
Ligation of sirA, fprA, and fdxA with pSB1C3
Heat Shock transformation of pSB1C3 sirA, pSB1C3 fprA and pSB1C3 fdxA into NEBTurbo
Heat Shock transformation of pSB9S1, pSB10K1 and pSB3C7 into NEBTurbo
Target
ASDF
Sunday 22nd September
Verifying shipment vectors
Growing Culture of pSB1C3 sirA, pSB1C3 fprA and pSB1C3 fdxA.
Mini-prep of pSB1C3 sirA, pSB1C3 fprA and pSB1C3 fdxA vectors
Colony PCR of pSB1C3 sirA, pSB1C3 fprA and pSB1C3 fdxA with sirA F/R, fprA F/R, fdxA F/R, and biobrick verification primers F/R
Target
ASDF
Friday 27th September
Growth Curves
IPTG Concentrations by column: 1 0nM, 2 10nM, 3 50nM, 4 100nM, 5 500nM, 6 1uM, 7 5uM, 8 10uM, 9 50uM, 10 100uM, 11 500uM, 12 1mM
Arabinose Concentrations by column: 1 0%, 2 0.01%, 3 0.02%, 4 0.03%, 5 0.04%, 6 0.05%, 7 0.06%, 8 0.07%, 9 0.08%, 10 0.09%, 11 0.1%, 12 0.2%
Row A: BL21 AI dcysI pACYC SirA pET FprA FdxA 1 Row B: BL21 AI dcysI pACYC SirA pET FprA FdxA 1 Row C: BL21 AI dcysI pACYC SirA pET FprA FdxA 2 Row D: BL21 AI dcysI pACYC SirA pET FprA FdxA 2
Row E: BL21 (DE3) dcysI pACYC SirA pET FprA FdxA 1 Row F: BL21 (DE3) dcysI pACYC SirA pET FprA FdxA 1 Row G: BL21 (DE3) dcysI pACYC SirA pET FprA FdxA 2 Row H: BL21 (DE3) dcysI pACYC SirA pET FprA FdxA 2  10mL 5x M9 Salts, 10mL 5x Amino Acid Sulfur dropout, 0.5mL MgSO4, 2mL 20% Glucose, 5ul CaCl2, 27.5mL H2O
Target
ASDF
Saturday 28th September
Growth Curves
Growth curves of BL21 (DE3) dcysI pACYC SirA pET FprA FdxA and BL21 AI dcysI pACYC SirA pET FprA FdxA.  Column 1: Positive control WT E. coli BL21 (DE3), 2: Negative Control BL21 AI dcysI, 3: BL21 AI dcysI pACYC SirA pET FprA FdxA 0 Arabinose 0 IPTG, 4: 0.1% Arabinose 10uM IPTG, 5: 0.05% Arabinose 50uM IPTG, 6: 0.01% Arabinose 100uM IPTG, 7: 0.2% Arabinose 1mM IPTG, 8: Negative Control BL21 (DE3) dcysI dfpr dydbf, 9: BL21 (DE3) dcysI pACYC SirA pET FprA FdxA 0 IPTG, 10: 50nM IPTG, 11: 1uM IPTG, 12: 50uM IPTG
Grown in M9 Media with Galactose.
10mL 5x M9 Salts, 10mL 5x Amino Acid Sulfur dropout, 0.5mL MgSO4, 2mL 20% Galactose, 5ul CaCl2, 27.5mL H2O
Target
ASDF
Sunday 29th September
PCR's and more!
Colony PCR on new Backbones.  RFP and Biobrick verification primers This is for pSB2C and pSB3K
Colony PCR on SirA, FprA, and FdxA Biobricked parts Using Biobrick primers and SirA, FprA and FdxA primers respectfully https://2013.igem.org/File:20130929_BioB.jpg https://2013.igem.org/File:20130929_RFP.jpg https://2013.igem.org/File:20190929_pSB3K.jpg
Phusion PCR on Duet vectors and pSB1C3 using standard cloning primers
Gel Clean up
Digestion with NcoI and AvrII
Standard cloning of new backbones (1h ligation at 16C)
Transform into NEBTurbo
Cutting Biobrick SirA, FprA, FdxA and Linearized backbone with EcoRI and SpeI
Standard clonging of Biobrick parts (1h ligation at 16C)
Transform into NEBTurbo
Treat XbaI SpeI cut Backbone with TSAP 1h.
Standard cloning of Biobrick parts (cut with XbaI, SpeI) into TSAP'd backbone (1h @ 16C)
Colony PCR on BL21 (DE3) dcysI pACYC SirA pET FprA FdxA and BL21 AI dcysI pACYC SirA pET FprA FdxA
Target
ASDF
Monday 30th September
Growth Curves
We performed Growth Curves with the following strains in each column:
WT BL21 AI
BL21 AI dcysI
BL21 AI dcysI pD002 pD003 0uM IPTG 0% Arabinose
BL21 AI dcysI pD002 pD003 10uM IPTG 0.05% Arabinose
BL21 AI dcysI pD002 pD003 100uM IPTG 0.1% Arabinose
BL21 AI dcysI pD002 pD003 1mM IPTG 0.2% Arabinose
WT BL21 DE3
BL21 DE3 dcysI dfpr dydbf
BL21 DE3 dcysI dfpr dydbf pD002 pD003 0uM IPTG
BL21 DE3 dcysI dfpr dydbf pD002 pD003 10uM IPTG
BL21 DE3 dcysI dfpr dydbf pD002 pD003 100uM IPTG
BL21 DE3 dcysI dfpr dydbf pD002 pD003 1mM IPTG
Results are https://2013.igem.org/File:20130930_GC_Glucose.xls
Target
ASDF
Tuesday 1st October
Israeli Collaboration
Plasmids arrived.  PCR CI plasmid with CI biobrick F/R primers.
Gel Clean-up of CI promoter PCR
Z score of BL21 dCysI pD002 pD003:Didn't Grow, forgot Sucrose in M9
Transform pSB1A3 E0240 and pSB3K3 into NEB Turbo
Target
ASDF
Wednesday 2nd October
Race to finish stuff
Grew up 5ml cultures of pSB3K3 and pSB1A3 E0240 from transformations
Plasmid mini preps of pSB3K3 and pSB1A3 E0240
Restriction Digest of CI promoter PCR product with XbaI and PstI
Restriction Digest of Linearized Backbone pSB1C3 with XbaI, PstI, and DpnI
Make new NEB Turbo Chemically Competent Cells: Modified protocol.  4mL of NEBTurbo into 2 x 2mL tubes.  Centrifuge at 14000 rpm for 1 minute at 4C.  Resuspend in 500uL of 100mM MgCl2 Ice cold.  Centrifuge at 14000 rpm for 1 minute at 4C.  Resuspend in 500uL of iGEM cc cell buffer Ice cold.  Centrifige for 1 minute at 14000 rpm at 4C.  resuspend in 100ul of iGEM cc cell buffer Ice cold.
Heat Shock Transformation of pSB1C3 SirA, pSB1C3 FprA, pSB1C3 FdxA into NEBTurbo (new chemically competent cells)
Ligation of CI promoter with pSB1C3 backbone.
Heat Shock Transformation of pSB1C3 CI into NEB Turbo
Target
ASDF
Thursday 3rd October
Race to Finish stuff
Plated new SirA, FprA, and FdxA transformations on CHL
Picked pSB1C3 CI and grew in LB broth
Z Score of BL21 dCysI pD002 pD003 with Gentamicin as positive hit control
Analytical Digestions of New Vectors (pAD1S, pAD2C, pAD3K, pAD4A)
Digest pSB1C3 CI with EcoRI and SpeI
Digest pSB3K3 with EcoRI and PstI
Digest pSB1A3 E0240 with XbaI and PstI
Ligation of CI promoter with GFP reporter and pSB3K3 backbone
Heat Shock Transformation of pSB3K3 CI GRP into NEBTurbo
PCR verify SirA, FprA, FdxA, and CI biobrick constructs
Infiltrate
ASDF
Monday 22nd July
Transforming the NEB TURBO strain with the pET21b and the pUC18 plasmids. Competent cells were prepared by the CaCl2 protocol. The transformation cultures were plated on ampicilline plates. Transformation was unsuccessful.
Infiltrate
ASDF
Tuesday 23rd July
Transforming the new NEB strain with the pET21b and the pUC18 plasmids (as yesterday). We’ve also plated the leftovers from the yesterdays' transformation (190 µL of culture on ampicilline plates).
Infiltrate
ASDF
Wednesday 24th July
Transformation form yesterday didn’t work. We plated the leftovers on new ampicillin plates.
Infiltrate
ASDF
Thursday 25th July
The transformation worked, probably it failed the last two times because the recovery time was to short or the plates were wrong. We did a streak plate to get single colonies.
Infiltrate
ASDF
Friday 26th July
Single colonies were plated into LB + ampicillin. 4 liquid cultures were prepared: NEB + pET21b (1),  NEB + pET21b (2), jNEB + pET21b (1), jNEB + pET21b (2). After 6 h of incubations the liquid cultures were used for
• Glycerol stock preparation.
• Plasmid purification (the protocol from the box).
DNA concentrations were measured by the nanodrop (ng/µL):
NEB1 134.4
NEB2 55.6
jNEB1 186.6
jNEB2 138.6
Infiltrate
ASDF
Monday 29th July
We did an analytical digest of pET 21b that we purified yesterday. We used Nedl and Xhol so our fragments should be: the backbone of the plasmid (around 5000 bp long) and the cloned MSMEG_1529 gene (around 600 bp long). On the following picture is our plasmid and we suppose that the MSMEG_1592 gene is cloned somewhere in between the two restriction sites that we have chosen. But we still don't really know that because we need the full sequence of the plasmid.
The protocol used for preparing the digestion:
Reagent
Volume
10x
Plasmid Miniprep
5µl
H2O
12 µl
120 µl
10x FastDigest Buffer
2 µl
20 µl
10x FastDigest Buffer
2 µl
20 µl
FastDigest Enzyme 1
0.5µl
5 µl
FastDigest Enzyme 2
0.5µl
5 µl
Total Volume
20.0 µl
200 µl
Digest for 1 h at 37 °C with shaking.
NEB1 and 2 are plasmids isolated from two different liquid cultures of the NEB strain; jNEB1 and 2 are plasmids isolated from two different liquid cultures of the NEB strain which was made competent by Jake. The (-) stands for a negative control: only DNA was added, without any digestion enzymes. As you can see it on the gel there are 2 bands - one is around 5000 bp long (fits the length of our plasmids backbone), and the other one is around 700 bp long (fits the length of our gene).
Infiltrate
ASDF
Tuesday 30th July
Transformed the bl21 (DE3) strain with the pET21b (plated the transformation culture on 2 amp plates).
Infiltrate
ASDF
Wednesday 31st July
Transformation worked. The number of transformed colonies is: 78 on plate 1 and 284 on plate. We prepared the liquid cultures and glycerol stocks of bl21 (DE3) + pET21b strain.
Infiltrate
ASDF
Thursday 1st August
We want to see if 1 mM solution of IPTG induces the over expression in our strain.  We prepared 4 liquid cultures (no antibiotics added):
• bl21 (DE3) + pET21b strain induced with 1 mM IPTG
• uninduced bl21 (DE3) + pET21b strain
• untransformed bl21 (DE3) + 1 mM IPTG
• untransformed bl21 (DE3)
10 mL of culture had 500 µL of the overnight culture (O.D.600 was set to 0.2) and 100 µL of 100 mM IPTG. We measured the O.D. of the cultures (600 nm) for time 0 and after 6 h. We also diluted our cultures to calculate the CFU/mL (for time 0 10-1 and 10-2 dilutions were prepared, and 100 µL of each dilution was plated on LBA plates; for 6 h 10-3, 10-4 and 10-5 for the induced culture and 10-7, 10-8 and 10-9 for all the others, 10 µL of each were plated). We also saved 2 mL for each group in the fridge.
Infiltrate
ASDF
Friday 2nd August
We repeated the time 6 measurement (the cultures were over diluted, so this time we did 10-5, 10-6 and 10-7). We also did the 24 h measurement (we used the 10-6, 10-7 and 10-8 dilutions). The final results are:
Growth inhibition could be noticed for the induced strain, which is probably a consequence of the over expression. The induced culture recovered and reached a greater CFU/mL value than in the previous groups. Maybe because we didn’t add any antibiotics in our media, the plasmid was lost and the culture could recover after achieving the over expression.
Infiltrate
ASDF
Monday 5th August
We want to repeat our previous experiment, but now we will add ampicillin in the media containing the transformed strains.  We set everything as previously, did the 0 h (100 µL of 10-1 and 10-2) and 6 h measurement (100 µL of 10-1 and 10-2 for the induced strain; 100 µL of 10-1 and 10-6 for the others).
Infiltrate
ASDF
Tuesday 6th August
We did the 24 h measurement (100 µL of 10-6 and 10-7), and repeated the 0 h measurement (100 µL of 10-3 dilution).
Infiltrate
ASDF
Wednesday 7th August
Repeated the 0 h measurement for uninduced bl21 (DE3) + pET21b strain (100 µL of 10-3 dilution), and the 24 h measurement for the induced strain (100 µL of 10-5, 10-4, 10-3 and 10-2).
Infiltrate
ASDF
Thursday 8th August
The growth curve for uninduced and induced bl21 (DE3), with antibiotics in the liquid culture:
The O.D. grows again after 24 h for the induced strain, but the CFU stays the same. Probably the O.D. grows because of the protein.
Infiltrate
ASDF
Monday 12th August
In order to grow M. smegmatis we should add Tween 80 (0.2 % / v.v) into the LB media (it stops the cells from forming clumps). In some papers we found that Tween 80 could inhibit the growth of E. coli, so we did a growth curve to check that. 500 μL of BL21 + pET21b culture (O.D. 600 nm = 1.00) was added to 10 mL of LB supplemented with different concentrations of Tween 80 – 0.2, 0.1 and 0.05 %. There were two sets of these cultures – one was induced with IPTG and the other one was not. O.D. was measured after 3 and 6 h. At these time periods cultures were also diluted to calculate the CFU. No growth inhibition was observed, so we can use Tween 80 in our E.coli – M. smegmatis co-cultures. We have also made some plates with malachite green (40, 4 and 0.4 μg/mL) and we have plated E. coli on them.
Infiltrate
ASDF
Tuesday 13th August
We finally got M. smegmatis today!!! We made liquid cultures in LB + 0.2% Tween 80 and we also plated it on LBA.
E. coli doesn’t grow on plates with 40 and 4  μg/mL of malachite green, but it does on  0.4 μg/mL.
Infiltrate
ASDF
Wednesday 14th August
Streak plate and liquid cultures for M. smegmatis and E. coli BL21 + pET21b. We started a plate reading experiment in order to see which concentrations of nalidixic acid should we use in our selective media. The tested concentrations were: 50, 25, 12.5, 6.25,3.125, 1.5625, 0.78125, 0.390625, 0.1953125, 0.09765625, 0.048828125 μg/mL. We also tested different concentrations of IPTG, to see which ones induce the strain without killing it. The tested concentrations were: 1, 0.5, 0.25, 0.125, 0.0625, 0.03125, 0.015625, 0.0078125, 0.00390625, 0.001953125, 0.0009765625  μg/mL. The concentration of bacteria in each well was 1 % and the bl21 culture of O.D. = 0.5 was used.
Infiltrate
ASDF
Thursday 15th August
Another plate reading experiment, with M. smegmatis for nalidixic acid (50, 25, 12.5, 6.25,3.125, 1.5625, 0.78125, 0.390625, 0.1953125, 0.09765625, 0.048828125 μg/mL) and with malachite green (400, 200, 100, 50, 25, 12.5, 6.25, 3.125, 1.5625, 0.78125, 0.390625  μg/mL); The concentration of bacteria was 1 % in LB with 0.2 % of Tween 80 and the culture with O.D. of 0.5 was used.
Infiltrate
ASDF
Friday 16th August
We started a growth curve for M. smegatis. In 20 mL of LBA with 0.2 % of Tween 20 μL of the liquid culture was added (O.D. Of the culture which was inoculated was 1). O.D. And CFU were determined each 3 hours. Dilutions which were used are: 10-1 and 10-2 for time 0, 10-2 and 10-3 for 3 h, 10-5 and 10-6 for 9 h.
Infiltrate
ASDF
Saturday 17th August
Because we suspect that the culture was contaminated we repeated the experiment with 3 separate cultures. n 20 mL of LBA with 0.2 % of Tween 20 μL of the liquid culture was added (O.D. Of the culture which was inoculated was 1). O.D. And CFU were determined each 3 hours. Dilutions which were used are: 10-1 and 10-2 for time 0, 10-2 and 10-3 for 3 h, 10-5 and 10-6 for 9 h, 10-7 for 24 h. We saved some plates so we could perform acid fast staining and check if its really M. smegmatis.  We have transformed bl21 by electroporation with pET21b and with pSB1C3, so we could get RFP E. coli expressing our protein. The transformed culture was plated on plates containing ampicilline and chloramphenicol.
Infiltrate
ASDF
Sunday 18th August
Made a liquid culture of BL21 + pET21b + pSB1C3. Plated the strain which contains the LLO plasmids (pACYZ184) on LBA + tetracycline + chloramphenicol, LBA +  tetracycline and LBA +  chloramphenicol because we don’t know which resistance the plasmid really has.
Infiltrate
ASDF
Monday 19th August
LLO plasmid containing bacteria are growing only on chloramphenicol containing plates, so tetracyclin resistance was lost due to the cloning of the hly downstream of the tet promoter.
The cells we transformed with pET21b (ampiciline resistance) and with the pSB1C3 plasmid (chloramphenicol resistance) are not growing on amp and chl containing plates because the origins of the plasmids are not compatible. So we are repeating the transformation with the pSB3C5 plasmid, which has a different origin.
We have started a co-culture with M. smegmatis and E. coli NEB turbo + pSB1C3 to see if our selective media with nalidixic acid is really working. Two cultures in their stationary phase were mixed (O.D. for M. smegmatis was 1.5 and for E. coli was 1): in 10 mL of LB 100 µL of each culture was added. We did dilution series for the start cultures and for the co-culture. Each was plated on: LBA plates where both strains should grow; on LBA + 3 µg/mL nalidixic acid plates, where only M. smegmatis should grow; on chl plates where only E. coli should grow.
Infiltrate
ASDF
Tuesday 20th August
We did a miniprep for the LLO plasmid (we had to do dialysis to purify the plasmid).
We observed the RFP producing cells (BL21 + pET21b + pSB3C5) under the fluorescent microscope and the cells produce RFP. We made a liquid culutre of the strain with 1 mM IPTG. We also plated them on LBA + 1 mM IPTG.
Liquid culture of M. smegmatis was made, but in the liquid culture 2 µL/mL of malachite green was added. We plated the culture on LBA + 3 µL/mL of nalidixic acid and on LBA to compare the number of colonies.
Infiltrate
ASDF
Wednesday 21st August
Our macrophages arrived today and we are staring a cell culture! The cells with pET21b and pSB3C5 can't survive the induction with 1 mM IPTG because they have two IPTG inducable promoters. We want to use a different insert, one with an uninducable promoter, such as BBa_J23100-118.
Infiltrate
ASDF
Thursday 22nd August
Counting the colonies from Monday; our plates with nalidixic acid are selective. We also noticed that the colonies showed up faster on the plates for the liquid culture supplemented with malachite green.
Infiltrate
ASDF
Monday 26th August
We started a killing assay for M. smegmatis: we had two groups; in the first one the ratio between M. smegmatis and E. coli BL21 + pET21b was 1:1 and in the other it was 1:2. BL21 was induced with 1 mM IPTG in the co-culture (10 mL of LB, 0.1 mL of each strain and 0.1 mL of 100 mM IPTG). For each group a control was made - same ratio between the strains but no induction. Also we had one group with only M. smegmatis. We did dilution series for time 0, after 3 and after 6 h. We also measured the O.D. (600 nm) for each time point.
Infiltrate
ASDF
Tuesday 27th August
We did a digestion of J23102 and J23104 with EcoRI and with and Pstl and we gel purified the RFP gene with a constitutive promoter. We also did a digestion for pSB3C5 with the same enzymes and we did gel purification for the backbone. The ligation product was then transformed into NEB Turbo.
Infiltrate
ASDF
Wednesday 28th August
Cloning from yesterday didn't work, so we will repeat everything with a different backbone. On gels for the digested pSB3C5 we couldn't see the part which was cut out, so we guess there is something wrong with this plasmid and we want to use a new one - pColaDuet-1 with a colA origin. We amplified the insert form J23102 and J23104 with biobrick verification primers and then we did PCR purification. We digested the insert and the backbone with EcoRI and Pstl and we did PCR purification for both of them. After measuring the concentration for both, we did a ligation in molar ration insert:backbone = 3:1 on 16 deg C for 1 h. The ligation product was transformed into NEB Turbo.
Infiltrate
ASDF
Thursday 29th August
We did a phagocytosis assay for our macrophages. First we passaged our cells and we determined their number (colored them with trypan blue and counted them): 2.68* 106, 3.72* 106 and 1.32* 106. To the first one we added 40, to the second 30 and to the last one 90 µl of NEB + pSB1A3 cell suspension. After 1 h we made this awesom picture:
Cloning from yesterday didn't work, so we repeated everything with longer digestion time for the backbone.
Infiltrate
ASDF
Friday 30th July
Cloning worked. Made liquid cultures. Counted the colonies for the killing assay.
Infiltrate
ASDF
Saturday 31st July
miniprep for pColADuet-1 + J23102/104. PCR for pET21b and the LLO plasmid (it didn't work).
Infiltrate
ASDF
Monday 2nd September
Transformed BL21 + pET21b with pColA + J23102/104 (electroporation and heat shock), transformed BL21 + pET21b with pColA + J23102/104 and with the LLO plasmid.  The PCR for the pET21b worked and we've sent it for sequencing; the PCR for the LLO plasmid didn't work :(
Infiltrate
ASDF
Tuesday 3rd September
Liquid cultures for BL21 + pET21b + pColA and for BL21 + pET21b + pColA + the LLO plasmid. We passaged the macrophages and we ordered new primers for sequencing the LLO plasmid.
Infiltrate
ASDF
Wednesday 4th September
The killing assay for M. smegamtis, with more time points. Now we can also follow the number of E. coli because it has RFP. In 20 mL of LB we added 100 µL of the M. smegmatis liquid culture (O.D. = 1.2) and 100 µL of the E. coli liquid culture (O.D. = 1) so we get 1:1 ratio. We measured the O.D. and the CFU for time 0, 1, 2, 3 and 6 h. To do the CFU for E. coli we plate the dilution on LBA + Kan + Amp, and for M. smegmatis we plate on LBA + 30 µL.
Infiltrate
ASDF
Thursday 5th September
Counted the colonies for E. coli and prepared everything for another killing assay. Liquid cultures for E. coli BL21 + pET21b + the LLO plasmid + the RFP plasmid; BL21 + pET21b + the RFP plasmid; M. smegmatis.
Infiltrate
ASDF
Friday 6th September
New killing assay with different E. coli and M.smegmatis ratio --> 0.1 : 1
Infiltrate
ASDF
Saturday 7th September
New killing assay with a new ratio --> E. coli : M. smegmatis = 0.01 : 1
Infiltrate
ASDF
Monday 9th September
Counting the colonies from the killing assay and doing the Bradford assay for each time point.
Infiltrate
ASDF
Tuesday 10th September
• PCR for the linearized backbone: template - pSB1C3 (1 µL), primers (SB-prep-2Eb, SB-prep-3P-1; 2.5 µL), HE buffer (10 µL), dNTPs (1 µL), polymerase (0.5 µL), dH2O (32.5 µL).
program: 94.0 (30 s) - 94.0 (10 s), 55.0 (1 min), 72.0 (2 min), 72.0 (3 min) - 30 cycles.
• touchdown PCR for the new backbones:
Template
Primers
pACYC
pDCA2 and pDCA
pET
pDCo4-F and pDCo4-R
pCDF
pDCA1 and pDCA
pColA
pDCA3 and pDCA
program: 95.0 (10 s) - 95.0 (30 s), 60.0 (30 s), 59.0 (30 s),58.0 (30 s), 56.0 (30 s), 55.0 (30 s), 72.0 (30 s) - 72.0 (10 min)- 30 cycles.
Infiltrate
ASDF
Wednesday 11th September
PCR for biobricking TDMH, for sequencing the LLO plasmid and for adding biobrick cutsites to pET21b; PCR purification of the products. PCR for the linear backbone and PCR purification. PCR for the new backbones gel slice and purification.
Infiltrate
ASDF
Thursday 12th September
• linear bacbone PCR: Phusion protocol from the 10th of the September, new PCR program: 98.0 (30 s) - 98.0 (10 s), 58.0 (15 s), 72.0 (1:10) - 72.0 (3 min). This time the PCR worked fine - no unspecific binding; the band was 2 kb long.
• PCR for biobricking TDMH: we designed primers to add biobrick cutsites on our TDMH gene. Template - pET21b+TDMH miniprep (150 ng/µL), primers - TDMH F (gaattcgcggccgcttctagATGATTTCCCTCCGGAAGCC) and TDMH R (ctgcagcggccgctactagtaTCAGTGGTGGTGGTGGTGGT). Program: 98.0 (30 s) - 98.0 (10 s), 62.0 - 58.0 (30 s), 72.0 (30 s) - 72.0 (10 min). It worked, bend size - 700 bp.
• Gibson assembly for the new backbones:  PCR products from the 10th of September were used (the same PCR reaction was done for the biobrick pSB2C3 backbone with all primers, so each PCR product for the Duet backbones was mixed with the corresponding pSB product. Protocol: 5 µL of each product, 10 µL of the Gibson master mix, 15 min on 50.0.
• PCR for RFP - we wanna add biobrick cut sites on the RFP gene from pSB1C3 (Phusion protocol). Primers: gaattcgcggccgcttctagATGGCTTCCTCCGAAGACGT(RFP F), ctgcagcggccgctactagtaGCGATCTACACTAGCACTAT (RFP R).
• PCR to biobrick pET, so we could clone RFP in it and put it under the T7 promoter: template is pET21b+TDMH (Phusion protocol). Primers: tactagtagcggccgctgcagGATCCGGCTGCTAACAAAGC (pET R), ctagaagcggccgcgaattcATGTATATCTCCTTCTTAAA (pET F)
• miniprep for DB3.1 + pColA-pSB3C5 and for DB3.1 + pCDF-pSB3C5. Then we did colony PCR for the minipreps with the corresponding primers (pDCA/pDCA3 and pDCA/pDCA1) to check if the insert from the Duet vectors is inside. The bends were the right size.
• Digestion of TDMH (biobricked) and the linear backbone:
For the backbone: 10 µL of the plasmid, 10 µL of the Green buffer, 3 µL for EcoRI and Pstl, 14 µL of dH2O.
For TDMH: 10 µL of the pcr product, 4 µL of the Green buffer, 1.5 µL for EcoRI and Pstl, 23 µL od dH2O.
• PCR Ligation for biobricking TDMH: dH20 4.5 µL, TDMH 1 µL, linear backbone 3 µL, T4 ligation buffer 1 µL, T4 ligase 0.5 µL; 16.0 fr 1 h.
Infiltrate
ASDF
Friday 13th September
A lot of macrophages died (in suspension) because we forgot to add glutamine and pyruvate sodium. We repeated the experiment with the complete media (but again without antibiotics).
• Results for the GA: because the Chl plates were off, I have to repeat everything for biobricking TDMH and for the pACYC derived backbone. The cloning for pCDF and pColA worked - patched the colonies, start liquid cultures.
• Launch NEB Turbo to make competent cells.
Infiltrate
ASDF
Saturday 14th September
Fresh liquid cultures for: M smegmatis pMyC-GFP (kanamycine resistance) , M smegmatis, E.coli Bl21 carrying pET21b-TDMH and pColA-RFP (ampicilin and kanamycine resistances) and E.coli Bl21 carrying pET21b-TDMH, pColA RFP and pACyC-hly (ampicilin, kanamycine and chloramphenicol resistances)
• CaCl2 competent cells for NEB Turbo - 25 mL of the culture (O.D. 1), 3 washing steps (1o min, 4 000 rpm, on 4.0). For the first one - 100 mM MgCl2, for the second - 100 mM CaCl 2, after the last one the cells were resuspended in CaCl2 + glycerol. I've also made electrocompetent cells (3 washes in dH20).
• cloning biobricked TDMH in the linear backbone - protocol from the 12th of September. New competent cells and new Chl plates.
• pET+pSB3C5 worked - miniprep; repeated the Gibson for pACYC
• PCR for confirming pET+pSB3C5 - tamplet is the miniprep and the primers are pDCo4 R/F. pET Duet as the positive controle (Phusion protocol).
Infiltrate
ASDF
Monday 15th September
• Liquid culture for TDMH+pSB1C3.
• Colony PCR for confirming TDMH+pSB1C3 with biobrick verification primers.
• A new setup for testing if E. coli kills M. smegmatis in the macrophages: 2 wells with M. smegmatis, 1 well with M. smeg + E. coli uninduced, 1 well with only E. coli, 1 well with M. smeg + E. coli induced. First we infect the macropheges with the liquid culture of M. smeg GFP - we set the final O.D. of the culture in the well to 0.03 (100 µL of the liquid culture with O.D.  0.3 to 900 µL of RPMI with attached macrophages. After 2 h of incubation in the CO2 incubator we wash the cells with PBS 10 times and we add E. coli induced or uninduced (induced overnight with 10 µL IPTG).
Infiltrate
ASDF
Monday 16th September
• I have to repeat the biobricking of TDMH because I didn't use the DpnI and because the linear backbone which I was using didnt have the RFP cassette. So I've repeated the PCR with the right backbone.
• Digesting the linear backbone: 10 µL plasmid, 10 µL fast digest buffer, 3 µL for EcoRI and Pstl, 1 µL od DpnI, 73 µL of dH20. 1 h, 37.0
• Ligation of the digested linear backbone and TDMH - backbone 3 µL, insert 1 µL, ligation buffer 1 µL, ligation enzyme 0.5 µL, dH2O 4.5 µL.
•A new setup for the macrophages; the protocol from the 15th of September; the groups were: macrophages alone, 2 x macrophages infected with M. smegmatis, 2 x macrophages infected with E. coli, 2 x M. smeg + E. coli uninduced, 2 x M. smeg + E. coli induced overnight, 2 x M. smeg + E. coli induced for 2 h. We also scratched the macrophages from yesterdays' setup and tried to get CFUs for them - 1 mL of the scratch (in PBS)was centrifuged (1 500 rpm, 10 min) to remove any cells which were not phagocytosed. The pallet was resuspended in 1 mL of PBS and the membranes of the macrophages were lysed with 5 µL of TritonX. 100 µL of the suspension was plated on LBA plates.
Infiltrate
ASDF
Tuesday 17th September
• A new setup (as yesterday); plated the scratches form yesterday. This time we washed our cells after the infection with e. coli.
Infiltrate
ASDF
Wednesday 18th September
Made pictures for the yesterdays' setup. The cloning from yesterday didn't work, so I plated the leftovers of the transformation culture.
Infiltrate
ASDF
Friday 20th September
• The cloning for biobricking TDMH worked, so we have to check if it has the insert. We run a PCR with the TDMH primers and we get a bend of around 700 bp, which is fine. We also did a PCR with the biobrick verification primers and the bend was around 1 kb and it was different then the band for the pSB1C3 (bigger, but the difference is not that obvious). So there is probably something inside.
• We did a new setup where we also tried to use M. smegmatis which is not GFP and stain it with a fluorescent die for Mycobacteria, but its not working. After a few h the level of contamination was hughe so we are repeating everything tomorrow with adding antibiotics in the media.
Infiltrate
ASDF
Tuesday 15th October
Start fresh cultures for E.coli bl21 and M.smegmatis-GFP.
Change the media of the macrophages J774.
Infiltrate
ASDF
Tuesday 16th October
We want to investigate the stability of the protein TDMH at different pH.
To do this, I prepared buffers at different pH using monosodium phosphate and disodium phosphate.
We used a calculator (http://home.fuse.net/clymer/buffers/phos2.html) to have an idea of how much miligrams should we add of each to get the right pH. Buffer strenght was 10mM.
We prepared a range of pH and checked them using a pH meter.
The pH were: 4,54 ; 5,15; 6,43; 7,36; 8,25 and 8,84
pH lower than 4,5 and higher than 8,8 couldn't be reached (because of the pKa as it is a weak acid).
We wanted to test another pH around 3, so we add HCl to reach pH 3.
The buffer were made with osmosed water and have been autoclaved after the pH measurement.
We also made glycerol stock of E.coli (the one with 3 plasmids carrying TDMH,LLO and RFP genes)from the overnight as described in the protocol.
Infiltrate
ASDF
Thursday 17th October
To study the stability of TDMH at 7 different pH we extracted it from E.coli using the kit B-PER.
We have 4 groups:
- M.smegmatis alone
- M.smegmatis + E.coli's control (producing another protein) induced
- M.smegmatis + E.coli's Bl21 (with TDMH gene) uninduced proteins
- M.smegmatis+ E.coli's Bl21 (with TDMH gene) induced proteins
We plate at time 0, 3h and 6 h using selective plates (nalidixic acid 3 micrograms/mL) and making 2 different dilutions
We had 3 wells for each condition.
We plated bacteria on 504 plates
Infiltrate
ASDF
Friday 18th October
Making fresh cultures to repeat the experiment.
Infiltrate
ASDF
Saturday 19th October
Repeating the pH experiment using smaller volume of detergent (3 microliters) because we were afraid that the detergent will affect M.smegmatis in the previous experiment.
Infiltrate
ASDF
Tuesday 16th October
Counting the colonies of M.smegmatis on the plates from Thursday.
We stopped because we noticed no significant difference between what was tested and the controls.
Maybe the protein wasn't extracted properly or it was extracted and inactive (denaturated)...
Also we noticed a decrease of 10-20% of the number of M.smegmatis colonies growing in all the wells with the detergent (and the proteins) (around 50microliters per well).
The very low pH (3 and 4,5) affected the growth of M.smegmatis by 20-30%
Infiltrate
ASDF
Tuesday 21st October
Counting the colonies from the second experiment ==> unsuccessful experiment.
Infiltrate
ASDF
Tuesday 16th October
Starting new cultures and making glycerol stocks for M.smegmatis-GFP.
Infiltrate
ASDF
Tuesday 16th October
We repeated the pH study using the plate reader and the OD this time.
Same controls and conditions, the only difference is the protein extraction.
We decided that we will just overexpressed E.coli (10mM IPTG as previously) and collect the supernatant.
We centrifuged 5minutes at 4000 rpm and took the supernatant. Then, the supernatant was filter sterilized
We add 200 microliters of M.smegmatis (OD 0,05) resuspended in the appropriate buffer and 20 microliters of E.coli (OD 0,5).
To know precisely the initial number of bacteria, we made CFU for each pH (resuspended in a different buffer).
We launched the plate reader for 24 hours.
Contents
1Infiltrate
1.1Sunday 27th October
2Infiltrate
2.1Monday 28th October
Infiltrate
<a href=" https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate " target="_blank" class="tbnotelogo TClogo"> ASDF </a>
Sunday 27th October
</br>
launched a plate reader experiment repeating the pH study (design as previously).
M.smegmatis was resuspended in the buffers. The proteins of E.coli were collected by taking the supernatant (centrifugation 4000 rpm, 5 minutes). The supernatant was filtersterilized.
Preparation of new buffers for the pH 3 and the pH 4,5. Former buffers weren't "strong" enough (pka = 7,2 for this couple). By adding chlorhydric acid we move on the second couple (pKa = 2,12) and have a sronger buffer.
</html>
Infiltrate
<a href=" https://2013.igem.org/Team:Paris_Bettencourt/Project/Infiltrate " target="_blank" class="tbnotelogo TClogo"> ASDF </a>
Monday 28th October
</br>
Interpreting the plate reader experiment which shows many things (see results) including that pH doesn't affect TDMH action (except a small decrease for basic pH like 8,2)
Grow the macrophages in a 12 well plate.
We began the experiment to test the killing inside the macrophage. We will infect them with M.smegmatis-GFP then wash (5 times) and infect them with E.coli RFP producing TDMH (controls wells with E.coli uninduced and also an induced E.coli producing something else) and wash (5 times). Wait 3 hours and collect the bacteria from the macrophage using 2 different techniques:
- Igepal detergent
- Cold water for injectable preparations
</br></br>
</html>
Sabotage
ASDF
24th July
starting bacterial culture of MG1655-6300 (Chantal’s glycerol)
-streaking 2 Agar plate with it
Sabotage
ASDF
25th July
choosing a nice colony and launching overnight culture 37° in LB
Sabotage
ASDF
28th July
sRNA design
red sequences:
pACYCDuet-1 (CmR) : ATATCCAGTGATTTTTTTCTCCAT
pCOLADuet-1 (KanR) : CGTTTCCCGTTGAATATGGCTCAT
We decided that targeting only two different antibiotic resistance will be enough for a proof of concept. We give up AmpR because otherwise they might be     experimental issues with AmpR that is also on phagemid litmus28i.
Sabotage
ASDF
2nd July
Aude and Vincent
glycerol stock
from overnight culture of MG1655-6300 O/N : T001
Centrifuge 4000rpm, 10 minutes,
take out liquid
resuspend cells in 1mL glycerol, 2mL LB
separate in two cryotubes, one for the -80°C, one for the -20°C
Electroporation
- making MG1655-6300 competent (Aude’s protocol for electrocompetent cells & electroporation [ref needed])
- test of the competent cells (negative
-transforming with pCOLADuet-1 : Ec2 =>
-transforming with pACYCDuet-1 : Ec2 =>
- plating 3 different quantity of cells (20ul, 50uL, 100uL) respectively on Cm and Kan plates (dilution 1000x for the antibiotics)
Sabotage
ASDF
3rd July
Aude and Vincent
Transfo results :
negative controls are negative.
Transformed cells grew on plates
in the evening launch of 5mL Lb cultures from clones on the plates
Igem Buffer for chemical competent cells
Igem protocol for chemical competent cells
Launch overnight culture of NEB turbo
5mL LB inoculated with a clone from plate of drug team
Sabotage
ASDF
4th July
Aude
Glycerols of the overnight cultures of the transformation
sT002 : MG1655 pCOLADuet-1
sT003 : MG1655 pACYCDuet-1
Aude and Clovis
Making stocks of electro competent NEB turbo
Prepare 10% glycerol solution
Put on Ice 1L of Sterile water, 10% glycerol solution and 10 falcon tubes
Cool down centrifuge to 4°C
Prepare a rack with 50 microtubes at -80°C
Dilute in 500 mL LB (in a 1000mL flask) 500 ul of NEB turbo pre cultured
Check the OD until it reached 0,5
Put 500mL cultures in 10 50mL falcon tubes in ice
Wait for 20 minutes for the culture to cool down
Centrifuge at 3400 rpm 4°C for 10 minutes
Take out liquid and resuspend in 50 mL ice cold water (resuspend content of two tubes in one tube)
Centrifuge 10 minutes, 4°C , 3400 rpm
Take out liquid and resuspend in 25 mL ice cold water
Centrifuge 10 minutes, 4°C, 3400 rpm
Resuspend all the content of the tubes in 20mL ice cold glycerol
Centrifuge 10 minutes, 4°C, 3400rpm
Resuspend in 5mL ice cold 10°C glycerol solution
Aliquot in 50 tubes (100uL per microtubes)
Store at -80°C
Sabotage
ASDF
15th July
Clovis and Anne
Primer Design
Design primers to PCR KanR out of Duet and LacZalpha out of pUC18 to then clone them into the DUET containing Chloramphenicol
Geneious files in the dropbox: LacZ primers, KanR primers
Sabotage
ASDF
16th July
Anne with Nico, Marguerite and Yonatan
Transformation of pUC18 (NEB, NEB CC, BL21)
1) Thaw competent cells on ice.
2) Place 20 ul of cells in a pre-chilled Eppendorf tube.
3) add o.5ul of plasmid to the cells
4) Mix gently by flicking the tube.
5) Chill on ice for 10 minutes.
4) Heat shock at 42 °C for 30 seconds.
5) Return to ice for 2 minutes.
6) Add 200 ul LB medium and recover the cells by shaking at 37 °C for 30 min (AmpR)
7) Plate out the cells (10 ul) on Amp LB plates as well as a control (untransformed cells)
8. Incubate at 37 °C. Transformants should appear within 12 hrs.
Result: Colonies (around 10)
Vincent and Aude
In silico cloning: sRNA gBlocks into pUC18 and litmus28-GFP
Successful with both Gibson assembly and regular cloning.
See file: “in silico cloning trojan horse.geneious”
Vincent
streaking Ortiz’s strain ELS-41 and ELS-13 containing Litmus28i_J23115-B0032-GFP and M13K07, respectively.
Sabotage
ASDF
17th July
Anne
Redsign of Primers
Design primers to PCR KanR out of Duet and LacZalpha out of pUC18 to then clone them into the DUET containing Chloramphenicol
Geneious files in the dropbox: LacZ primers, KanR primers
Clovis and Anne
Liquid Cultures (for Miniprep)
7ml LB with proper antibiotics
adding: 1) NEB with pUC18 (AmpR)
2) sT002 : MG1655 pCOLADuet-1 (KanR)
3) sT003 : MG1655 pACYCDuet-1 (ChlaR)
Cultivate over night in the 37°C shaker
Vincent
Check: GFP expressed as expected in ELS-41
Lauching O/N of ELS-41 and ELS-13 with Amp and Kan, respectively.
Aude
Lauching O/N of MGZ1 (Glycerol from lab upstairs) with Spec (500X)
Sabotage
ASDF
18th July
Vincent and Aude
Glycerol stock of ELS-41 and ELS-13 and MGZ1 now renamed sT004, sT005, sT006 respectively:
sT004: RP437, F+ Litmus28i_J23115-B0032-GFP AmpR
sT005: XL1-Blue, M13K07 KanR
Making electrocompetent cells from strain sT004, sT005 and MGZ1 => Bad old water => could not obtain competent cells.
Conjugation :
sT004: RP437, F+ Litmus28i_J23115-B0032-GFP + conjugate with MGZ1.
F+ comes from XL1 Blue (Ortiz paper) => F+ is tetR
Protocol :
From O/N cultures Dilute both strains 1/100 , in LB
Wait for OD to reach O,2
Prepare 4 tubes (in BD tubes) :
Tube 1 = 0,5mL LB with Strain 1 (Here MGZ1) + O,5mL LB = control
Tube 2 = 0,5mL LB with Strain 2 (here sT004) + 0,5mL LB = control
Tube 3 and 4= 0,5mL LB with Strain 1 (Here MGZ1) + 0,5mL LB with Strain 2 (here sT004
Incubate 2 hours at 37°C
Plate 100ul for controls, 100uL, 50ul, 20ul for mixed tubes on LB antiobiotics (here tet and spec)
Incubate overnight at 37°C
Aude
Miniprep pT005 (Litmus28i_J23115-B0032-GFP) pT006 (M13K07) (Thermo Scientific Miniprep Kit)
pT005 : 280 ng /uL
pT006 : 61 ng/ul
Aude Vincent Sebastian and Yonatan
Work on the model
Define the players, the processes in place (more in Modeling file)
Anne and Clovis
Miniprep (sT002, sT003, NEB+pUC18) using the Thermo Scientific Miniprep Kit
1) Pellet 5ml of liquid culture (max speed, 1 min)
2) Discard supernatant
3) resuspend the cells in 250ul of resuspension olution
4) add 250ul of lysis solution, mix by inverting 4-6 times
5) add 350ul of neutralization solution
4) centrifuge for 5 min
5) transfer supernatant to spin column
6) centrifuge for 1 min
7) discard flow through
8) add 500 ul wash solution and centrifuge for 1 min , discard flow through(repeat this step)
9) centrifuge for 1 min to remove left over liquid
10) transfer the column on a 1.5ml tube
11) add 50ul of elution buffer and incubate for 2 min
12) centrifuge for 2 min
13) Nanodrop the concentration and freeze at -20°
pT.002: 53.9ng/ul -> redo with more liquid culture, it is only a low copy plasmid
pT.003: 53.8ng/ul -> redo with more liquid culture, it is only a low copy plasmid
pT.007: 214ng/ul
Sabotage
ASDF
19th July
Conjugation results :
Controls => no colonies
Mix = tapis
Ie worked really well.
To be sure to start only with one clone => restreak some colonies on tet/spec plates
Transformation by electroporation : goal = get a phage factory.
We do it three different ways in order to see which one works (best).
Transform strain containing helper plasmid with phagemid
Transform strain containing helper plasmid with phagemid
Co Transform strain with both plasmids at the same time
From strain sT004, sT005, sT006
Transform sT004  with pT006
Transform sT005 with pT005
Co Transform sT006 with pT005 and pT006
Tubes :
sM13KO7 (control) : 5,60
sM13KO7 + pLitmus : 5,70
sM13KO7 + pLitmus : 5,70
sLITMUS (control) : 5,60
sLITMUS + pM13K07 : 5,60
sLITMUS + pM13K07 : 5,60
MG16165 : control = 5.70
MG16165 + p (M13K07) and p(LITMUS) = 5.60
MG16165 + p (M13K07) and p(LITMUS)= 5.60
Recovery 2h
Plate everything on Kan/ Amp
Sabotage
ASDF
20th July
Results of transformation
Controls : no colonies
Colonies on plates : every transformation worked
Put plates of conjugation and transformation in the fridge.
Sabotage
ASDF
22th July
Place your twit here
Check Colonies of transformation under iLumina tools: colonies are fluorescent.
Launch culture.
Sabotage
ASDF
23th July
Infectiveness characterization experiments
The night before: prepare O/N of F+ cells, prepare O/N of phagemid producing cells (in this case : sT007 )
Centrifugate phagemid producing cells
Filter the supernatant 0.45um, stock it at 4°C (it contains the phages)
Dilute 200x MGZ1, F+  from O/N
Wait until OD600 = 0.7
Immediately mix MGZ1 and the supernatant in different proportions
Here we tried 1/1 (vol supernatant/vol cells), 1/10 and 1/100
Immediately dilute each of them to 10-3, 10-4, 10-5, 10-6
Plate 100uL of those dilutions on LB-agar, AMP, KAN, AMP+KAN
Incubate O/N
Count colonies
Interpretation: Colonies on AMP: infected by litmus28i
Colonies on KAN: infected by M13K07
Colonies on AMP+KAN: coinfected by litmus28i and M13K07 phagemid producing cells
Results
Conclusions:
Cells that were mixed with 1/1 supernatant didn't survive the excess of antibiotics that were present in the supernatant.
Even at low ratio like 1/100 the infection rate of litmus28i-GFP is about 100%. GFP detection confirmed this result: almost 100% of cells were fluorescent.
Co-infection did occur. It would be interesting to test if those colonies are indeed producing phages.
Start O/N culture of M13K07 Start O/N culture of MGZ1,F+
Sabotage
ASDF
24th July
Glycerol stock:
sT008: MGZ1, F+
sT009: XL1-Blue Litmus28i
Infectiveness characterization experiments
Dilute 1/100 MGZ1, F+ O/N culture in 5mL LB => wait OD(600) = 0.7
Centrifuge M13K07 4000rpm, 10 minutes
Retrieve surnageant and filter it with 0.2 filter
Once OD 0.7 is reached, make 4 tubes
1mL MGZ1,F+ + 100ul LB (not done, not enough cells left ofter OD test)
1mL MGZ1,F+ + 100ul surnageant diluted 1/100
1mL MGZ1,F+ + 100ul surnageant diluted 1/1000
1mL MGZ1,F+ + 100ul surnageant diluted 1/1000
Incube 37°C for 45 minutes
Serial dilute the tubes 10^-1, 10^-2, 10^-3, 10^-4, 10^-5, 10^-6
4 times 6 tubes with 900uL LB, add 100ul of one previously made tube, mix and transfert 100ul to next tube.
Plate on LB (10^-6), Kan(10^-2 and 10^-3), Amp(10^-6), Kan and Amp(10^-2 and 10^-3)
Experiment fail due to manipulation error (wrong dilution of the supernageant and plating of wrong dilutions)
Start O/N culture of MGZ1,F+ (T008) + Tet 1/500
Start O/N culture of M13K07 (T007) + Amp and Kan 1/1000
Sabotage
ASDF
25th July
O/N culture of T007 didn't grow (100 ul instead of 10ul of antibiotics, Kan and Amp)
Start O/N culture of MGZ1,F+ (T008) + Tet 1/500
Start O/N culture of M13K07 (T007) + Amp and Kan 1/1000
Sabotage
ASDF
26th July
Infectiveness characterization experiments
Dilute 1/100 MGZ1, F+ O/N culture in 10mL LB => wait OD(600) = 0.7
Centrifuge M13K07 4000rpm, 10 minutes
Retrieve surnageant and filter it with 0.2 filter
Once OD 0.7 is reached, make 4 tubes
1mL MGZ1,F+ + 100ul LB
1mL MGZ1,F+ + 100ul surnageant diluted 1/100
1mL MGZ1,F+ + 100ul surnageant diluted 1/1000
1mL MGZ1,F+ + 100ul surnageant diluted 1/1000
Incube 37°C for 45 minutes
Serial dilute the tubes 10^-1, 10^-2, 10^-3, 10^-4, 10^-5, 10^-6
4 times 6 tubes with 900uL LB, add 100ul of one previously made tube, mix and transfert 100ul to next tube.
Plate on LB (10^-6), Kan(10^-2), Amp(10^-6), Kan and Amp(10^-3)
Sabotage
ASDF
Tuesday 30th July
Transformed the bl21 (DE3) strain with the pET21b (plated the transformation culture on 2 amp plates).
Sabotage
ASDF
31th July
Vincent and Aude
Conjugation MGZ1, F+ & MG1655:
Protocol :
From O/N cultures Dilute both strains 1/100 , in LB
Wait for OD to reach O,2
Prepare 4 tubes (in BD tubes) :
Tube 1 = 0,5mL LB with Strain 1 (Here MGZ1) + O,5mL LB = control
Tube 2 = 0,5mL LB with Strain 2 (here sT004) + 0,5mL LB = control
Tube 3 and 4= 0,5mL LB with Strain 1 (Here MGZ1) + 0,5mL LB with Strain 2 (here sT004
Incubate 2 hours at 37°C
Plate 50ul for controls, 50uL, 20ul, 10ul for mixed tubes on LB antiobiotics (here tet and spec)
Incubate overnight at 37°C
Vincent
Infectiveness characterization experiments
-the night before: prepare O/N of F+ cells, prepare O/N of phagemid producing cells (in this case : sT007 )
-centrifugate phagemid producing cells
-filter the supernatant 0.45um, stock it at 4°C (it contains the phages)
-dilute 200x MGZ1, F+ from O/N
-wait until OD600 = 0.7
-immediately mix MGZ1 and the supernatant in different proportions
here we planned to try 1/100 (vol supernatant/vol cells), 1/1000 and 1/10000
Unfortunately the supernatant was dropped on the bench.
Sabotage
ASDF
1st August
Aude & Vincent
Cloning Kanamycin resistance gene into pACYCDuet-1:
-PCR of Kanamycin resistance gene from pCOLADuet-1 with primer iT.0003 and iT.0004
-electrophoresis of PCR product: 1 band between 900 and 1000bp: coherent with KanR length
-cutting KanR bands:
-digestion with ECOR1 and Xho1:
PCR product digest for cloning
Reagent
Volume
Purified PCR Product
16 ul
H2O
0 ul
10x FastDigest Buffer
2 ul
FastDigest Enzyme 1
1 ul
FastDigest Enzyme 2
1 ul
Total Volume
20.0 ul
Digest for 0.5 - 1 hours at 37 °C with shaking.
-digestion of pACYCDuet-1 with ECOR1 and Xho1:
Vector backbone digest for cloning
Reagent
Volume
Purified Plasmid
20 ul
H2O
65 ul
10x FastDigest Buffer
10 ul
FastDigest Enzyme 1
2.5 ul
FastDigest Enzyme 2
2.5 ul
FastAP Phosphatase
0 ul
Total Volume
100.0 ul
Digest for 2 - 4 hours at 37 °C with shaking.
-electrophoresis of pACYCDuet-1 digestion products:
column 1: 1kb plus marker
column 2 & 3: digestion product
column 4: undigested pACYCDuet-1
nothing appear on gel after digestion, therefore we will miniprep pACYCDuet-1 and try again to digest it.
Sabotage
ASDF
12th August
Clovis
Start 2 cultures of MG1655 pACYCDuet-1 strain (ST003), 5mL each for further miniprep
Phusion PCR of lacZalpha out of pUC18 (pT007)
with primers :
iT.0001 (Fw, added NcoI cut site) CCATGGCTGGCTTAACTATGCGG
iT.0002 (Rv, added BglII cutsite) AGATCTGTGAGTTAGCTCACTCATTAGG
Column 1; 100kb plus ladder
Column 2,3,4,5,6; PCR product
Gel purification of column 5 and 6 → 19,2 ng/ul
Sabotage
ASDF
13th August
Clovis
Digestion of the PCR product (LacZalpha) with NcoI and BglII
PCR purification using Thermo Scintific Kit of the digestion → 6,2 ng/ul
Miniprep of pACYADuet-1 (pT002) using Thermo Scientific Kit → 132,7 ng/ul
Start O/N sT005 and sT010
Vincent
PCR of pUC18 (pT007) with iT0007 & iT0008 / PCR of litmus28i-GFP (pT005) with iT0005 & iT0006
PCR Gradient of 8 tubes +-5°C around 54°C for both PCRs (pUC18 and Litmus18i-GFP)
Failure, no bands appear on the gel.
Infectiveness characterization experiments
-the night before: prepare O/N of F+ cells, prepare O/N of phagemid producing cells (in this case : sT007 )
-centrifugate phagemid producing cells
-filter the supernatant 0.45um, stock it at 4°C (it contains the phages)
-dilute 200x MGZ1, F+ from O/N
-wait until OD600 = 0.7
-immediately mix MGZ1 and the supernatant in different proportions
here we planned to try 1/100 (vol supernatant/vol cells), 1/1000 and 1/10000
1mL MGZ1,F+ + 100ul LB
1mL MGZ1,F+ + 100ul surnageant diluted 1/100
1mL MGZ1,F+ + 100ul surnageant diluted 1/1000
1mL MGZ1,F+ + 100ul surnageant diluted 1/1000
-incubate 45 minutes at 37°C for the protein to be expressed.
Serial dilute the tubes 10^-1, 10^-2, 10^-3, 10^-4, 10^-5, 10^-6
4 times 6 tubes with 900uL LB, add 100ul of one previously made tube, mix and transfert 100ul to next tube.
Plate on LB (10^-6), Kan( 10^-3), Amp(10^-5 - 10^-6), Kan and Amp(10^-2 - 10^-3)
Sabotage
ASDF
14th August
PCR of pUC18 (pT007) with iT0007 & iT0008 / PCR of litmus28i-GFP (pT005) with iT0005 & iT0006 (Aude):
Gel Purif (Aude)
Using QiaGen Kit
3 tubes (combined the band)
1 : 13,2 ng/uL
2 : 14 ng/uL
3 : 2O ng/uL
Digestions (Aude)
Digestion pACY par Xho EcoRI (pour le clonage de Kan dans ACY)
Reagent
Volume
Purified Plasmid
10 ul
H2O
75 ul
10x FastDigest Green Buffer
10 ul
EcoRI
2.5 ul
XhoI
2.5 ul
FastAP Phosphatase
0 ul
Total Volume
100.0 ul
Digestion pACY par NcoI BgIII (pour le clonage de lacZalpha dans ACY)
Reagent
Volume
Purified Plasmid
15 ul
H2O
75 ul
10x FastDigest Green Buffer
10 ul
EcoRI
2.5 ul
XhoI
2.5 ul
FastAP Phosphatase
0 ul
Total Volume
100.0 ul
=> 2h30 à 37°C
Run on a a gel.
only see a smear, Everything was digested ,
redigest but just 25 minutes as said on the thermoscientific protoc.
Digestions (CLovis)
Digestion pACY par Xho EcoRI (pour le clonage de Kan dans ACY)
Reagent
Volume
Purified Plasmid
10 ul
H2O
75 ul
10x FastDigest Green Buffer
10 ul
EcoRI
2.5 ul
XhoI
2.5 ul
FastAP Phosphatase
0 ul
Total Volume
100.0 ul
Digestion pACY par NcoI BgIII (pour le clonage de lacZalpha dans ACY)
Reagent
Volume
Purified Plasmid
15 ul
H2O
75 ul
10x FastDigest Green Buffer
10 ul
EcoRI
2.5 ul
XhoI
2.5 ul
FastAP Phosphatase
0 ul
Total Volume
100.0 ul
25 minutes at 37°C
It worked!
Gel purification (Clovis)
ACY Nco Dig : 6,3 ng/ul
ACY Xho Dig : 7,8 ng/uL
PCR Gradient of 8 tubes +10°C around 54°C for both PCRs (Litmus18i-GFP) but start not from the miniprep ut from a fresh culture of litmus GFp
Only see the primer => try again with new miniprep
Sabotage
ASDF
15th August
Miniprep (Clovis)
pACY (pT002) : 109ng/uL
litmus (pT005) : 610 ng/Ul
Ligation (Aude, Clovis ,Vincent)
masse insert = ration (svt 10) * (taille insert / taille vecteur) * masse vecteur (50-100ng)
CHosen Vector mass = 50 ng
lacZ :
Digestion vector : 6,3 ng/ul
Digestion insert : 6,2
masse insert = 10*449/3770*50 = 59 ng
Protocol
Vector : 8 uL
Insert : 10 uL
Buffer : 2 uL
T4 DNA ligase : 0,2 uL
Let incubate at 22°C for 30 min
Kan :
Digestion vector : 7,8 ng/ul
Digestion insert : 12ng/ul
Insert mass = 10*1052/3770*50 = 140 ng
Protocole Ligation
Vector : 7 uL
Insert : 11 uL
Buffer : 2 uL
T4 DNA ligase : 0,2 uL
Transformation : (in chemical competent NEB turbo (made on the …) Aude Clovis
1) Thaw competent cells on ice.
These can be prepared using the CaCl2 protocol.
2) Add 5 ul of ligation product
4) Mix gently by flicking the tube.
5) Chill on ice for 30 minutes.
4) Heat shock at 42 °C for 30 seconds.
5) Return to ice for 2 minutes.
6) Add 300 ul LB medium and recover the cells by shaking at 37 °C for one hour
7) Plate on selective media
Plate on :
Tube 1 : negative control Xgal /IPTG /Cm ; Kan/Cm
Tube 2 : ligation lacZ-pacy (5uL) Xgal /IPTG /Cm
Tube 3 : ligation Kan-pACY (5uL) Kan Cm
Tube 4 : native plasmid (pACY) (1uL) Xgal /IPTG /Cm ; Kan/Cm
PCR (litmus = Backbone) Vincent
8 tubes, gradient 10°C around 64°C , started from a new miniprep
We see the plasmid => too concentrated
PCR (litmus pour la gibbson) Clovis
sTOO5 primer 005 006 protocol phusion (gradient around 64 + or – 5 °C)
Elongaiton time 2min10. Started with the new miniprep diluted.
Infectiveness characterization experiments (Vincent)
-the night before: prepare O/N of F+ cells, prepare O/N of phagemid producing cells (in this case : sT007 )
After checking GFP fluorescence of sT007 and its negative result, we decided to make a new sT007 glycerol stock. We used a colony from the first successful     Infectiveness characterization experiments that was both AMP+KAN resistant and expressed GFP.
The following experiment was conducted with this new sT007 (strain MGZ1).
-centrifugate phagemid producing cells
-filter the supernatant 0.45um, stock it at 4°C (it contains the phages)
-This experiment was conducted both on MGZ1(F+) and MG1655(F+)
-dilute 200x MGZ1 and MG1655 from O/N
-wait until OD600 = 0.7
-immediately mix MGZ1 and the supernatant in different proportions
here we planned to try 1/10 (vol supernatant/vol cells)
1mL MGZ1,F+ + 100ul LB
1mL MGZ1,F+ + 100ul surnageant diluted 1/100
-incubate 45 minutes at 37°C for the protein to be expressed.
Serial dilute the tubes 10^-1, 10^-2, 10^-3, 10^-4, 10^-5, 10^-6
6 tubes with 900uL LB, add 100ul of one previously made tube, mix and transfert 100ul to next tube.
Plate on LB (10^-4 10^-5 10^-6), Kan(10^-1 10^-2 10^-3), Amp(10^-4 10^-5 - 10^-6), Kan and Amp(10^-1 10^-2 - 10^-3)
Sabotage
ASDF
16th August
Results from Infectiveness characterization experiments (Vincent)
 See file “phage infectivness 15_08.xlsx”
Results
Transformations : (Aude)
Lac : “tapis” on all the plates even the negative control => plates were not done correctly, antibio added to early => do another     transformation
Kan : Some colonies on the plates , but also colonies on the negative control => launch few colonies on liquid media (with Kan, Cm)
Phage infectiveness : (Vincent)
Run PCR (Litmus-Gibbson) (Aude)
Small point around 3000-3500bp
Run another gel with large holes to check if it is correct and purify => Not correct, nothing on the new gel
Transformation : (in chemical competent NEB turbo )Aude Clovis
Rest of ligation product
1) Thaw competent cells on ice.
These can be prepared using the CaCl2 protocol.
2) Add 5 ul of ligation product
4) Mix gently by flicking the tube.
5) Chill on ice for 30 minutes.
4) Heat shock at 42 °C for 30 seconds.
5) Return to ice for 2 minutes.
6) Add 300 ul LB medium and recover the cells by shaking at 37 °C for one hour
7) Plate on selective media
Plate on :
Tube 1 : negative control Xgal /IPTG /Cm ; Kan/Cm
Tube 2 : ligation lacZ-pacy (5uL) Xgal /IPTG /Cm
Tube 3 : ligation Kan-pACY (5uL) Kan Cm
Tube 4 : native plasmid (pACY) (1uL) Xgal /IPTG /Cm ; Kan/Cm
Gel from PCR litmus (Clovis)
1kB ladder / triple puit (3*50uL) from column 2,3,4 of PCR
50V
Infectiveness characterization experiments (Aude)
-the night before: prepare O/N of F+ cells, prepare O/N of phagemid producing cells (in this case : sT007 )
-centrifugate phagemid producing cells
-filter the supernatant 0.45um, stock it at 4°C (it contains the phages)
-dilute 200x MG, F+ from O/N
-wait until OD600 = 0.7
-immediately mix MG and the supernatant in different proportions
here we planned to try 1/100 (vol supernatant/vol cells), 1/1000 and 1/10000
1mL MG,F+ + 100ul LB
1mL MG,F+ + 100ul surnageant diluted 1/10
1mL MG,F+ + 100ul surnageant diluted 1/100
1mL MG,F+ + 100ul surnageant diluted 1/1000
1mL MG,F+ + 100ul surnageant diluted 1/10000
-incubate 45 minutes at 37°C for the protein to be expressed.
Serial dilute the tubes 10^-1, 10^-2, 10^-3, 10^-4, 10^-5
4 times 5 tubes with 900uL LB, add 100ul of one previously made tube, mix and transfert 100ul to next tube.
Plate on LB (10^-5), Kan( 10^-2), Amp(10^-5), Kan and Amp(10^-2)
Ligation (Vincent)
masse insert = ration (svt 10) * (taille insert / taille vecteur) * masse vecteur (50-100ng)
CHosen Vector mass = 50 ng
lacZ :
Digestion vector : 6,3 ng/ul
Digestion insert : 6,2
masse insert = 10*449/3770*50 = 59 ng
Protocol
Vector : 8 uL
Insert : 10 uL
Buffer : 2 uL
T4 DNA ligase : 0,2 uL
Let incubate at 22°C for 30 min
Kan :
Digestion vector : 7,8 ng/ul
Digestion insert : 12ng/ul
Insert mass = 10*1052/3770*50 = 140 ng
Protocole Ligation
Vector : 7 uL
Insert : 11 uL
Buffer : 2 uL
T4 DNA ligase : 0,2 uL
Transformation : (in chemical competent NEB turbo (made on the …) vincent
1) Thaw competent cells on ice.
These can be prepared using the CaCl2 protocol.
2) Add 5 ul of ligation product
4) Mix gently by flicking the tube.
5) Chill on ice for 30 minutes.
4) Heat shock at 42 °C for 30 seconds.
5) Return to ice for 2 minutes.
6) Add 300 ul LB medium and recover the cells by shaking at 37 °C for one hour
7) Plate on selective media
Plate on :
Tube 1 : negative control Xgal /IPTG /Cm ; Kan/Cm
Tube 2 : ligation lacZ-pacy (10uL) Xgal /IPTG /Cm
Tube 3 : ligation Kan-pACY (10uL) Kan Cm
Tube 4 : native plasmid (pACY) (1uL) Xgal /IPTG /Cm ; Kan/Cm
PCR pUC18 (pT007)
5*50uL
34 H2O
10 Buffer 5x
1 dNTP
2.5 FW iT007
2.5 RV iT008
0.2 pUC18
0.5 phusion
50uL Tot
Tm = 55°C
te = 2min10
no bands
PCR Litmus (Clovis):
Trouble shooting : since the temperature does not help, that it was not the mini prer = but that we see homodimere between the primers => try with less     primers
PCR litmus (pT005)
5*50ul
40 H2O
10 Buffer 5x
1 dNTP
0.5 FW iT005
0.5 RV iT006
0.2 litmus
0.5 phusion
50ul Tot
Tm = 64°C
te = 2min10
Sabotage
ASDF
17th August
Results
Transformation worked but no clone with the ligation product =>restart from the beginning, problem must be with one of the digested products.
Phage infectiveness : Experiment worked => results here
Repcr inserts / redigest vector
PCR KanR out of pCOLA (pT003)
5*50ul
34 H2O
10 Buffer 5x
1 dNTP
2.5 FW iT003
2.5 RV iT004
0.2 pCOLA
0.5 phusion
50ul Tot
Tm = 54°C
te = 45sec
PCR LacZ out of pUC18 (pT007)
5*50ul
34 H2O
10 Buffer 5x
1 dNTP
2.5 FW iT001
2.5 RV iT002
0.2 pUC18
0.5 phusion
50ul Tot
Tm = 54°C
te = 45sec
Gel of PCR Kan and LacZ
5 puits LacZ / 100bP+ / 5 puits Kan
Preparaiton of the vectors to clone the sRNA
PCR litmus (pT005)
5*50ul
40 H2O
10 Buffer 5x
1 dNTP
0.5 FW iT003
0.5 RV iT004
0.2 pCOLA (pT007)
0.5 phusion
50ul Tot
Tm = 64°C
te = 2min10
Results
Transformation worked but no clone with the ligation product =>restart from the beginning, problem must be with one of the digested products.
Phage infectiveness : Experiment worked => results here
Repcr inserts / redigest vector
PCR KanR out of pCOLA (pT003)
5*50ul
34 H2O
10 Buffer 5x
1 dNTP
2.5 FW iT003
2.5 RV iT004
0.2 pCOLA
0.5 phusion
50ul Tot
Tm = 54°C
te = 45sec
PCR LacZ out of pUC18 (pT007)
5*50ul
34 H2O
10 Buffer 5x
1 dNTP
2.5 FW iT001
2.5 RV iT002
0.2 pUC18
0.5 phusion
50ul Tot
Tm = 54°C
te = 45sec
Gel of PCR Kan and LacZ
5 puits LacZ / 100bP+ / 5 puits Kan
-->img 622<--
Preparaiton of the vectors to clone the sRNA
PCR litmus (pT005)
5*50ul
40 H2O
10 Buffer 5x
1 dNTP
0.5 FW iT003
0.5 RV iT004
0.2 pCOLA (pT007)
0.5 phusion
50ul Tot
Tm = 64°C
te = 2min10
Sabotage
ASDF
19th August
CLONING the target Plasmids (Kan and lacZ)
Gel purification of LacZ using Thermo Scientific Kit
lacZ : 83 ng/ul
Gel purification of Kan using Thermo Scientific Kit
Kan : 80ng/ul
Digestion of pACYC (pT002) with EcoRI and XhoI
10 H2O
2 Buffer
4 pACYC
2 EcoRI
2 XhoI
20uL Tot
37°C
20min
Digestion of pACYC (pT002) with BglII and NcoI
10 H2O
2 Buffer
4 pACYC
2 BglII
2 NcoI
20uL Tot
37°C
20min
Digestion of Kan insert with EcorI and XhoI
10 H2O
15 Kan insert
3 Buffer
1 EcoRI
1 XhoI
30uL Tot
37°C
20min
Digestion of LacZ insert with BglII and NcoI
10 H2O
15 LacZ insert
3 Buffer
1 BglII
1 NcoI
30uL Tot
37°C
20min
Gel of digestions
100Bp+ / pACYC+EcoRI+XhoI / pACYC+EcoRI+XhoI / pACYC+BglII+NcoI /pACYC+BglII+NcoI
Colum purification of Kan insert digestions using Thermo Scientific Kit
Column purification of LacZ insert digestions using Thermo Scientific Kit
Colum purification of pACYC digestions using Thermo Scientific Kit
Heat inactivation of Kan digestion
65°C
20min
Ligation Kan insert purif + pACYC (digestion EcoRI + XhoI)
Digestion vector : 6.8 ng/ul
Digestion insert : 14
masse insert = 10*1052/3770*50 = 140 ng
Vector : 7.35 uL
Insert : 10 uL
Buffer : 2 uL
T4 DNA ligase : 0,2 uL
Let incubate at 22°C for 30 min
Ligation Kan insert heat desactivated + pACYC (digestion EcoRI + XhoI)
Digestion vector : 6.8 ng/ul
Digestion insert : 40/2 = 20
masse insert = 10*1052/3770*50 = 140 ng
Vector : 7.35 uL
Insert : 7 uL
Buffer : 2 uL
T4 DNA ligase : 0,2 uL
H2O : 4 uL
Let incubate at 22°C for 30 min
Ligation LacZ insert purif + pACYC (pigestion BglII + NcoI)
Digestion vector : 10? ng/ul
Digestion insert : 33
masse insert = 10*449/3770*50 = 59 ng
Vector : 5 uL
Insert : 1.8 uL
Buffer : 2 uL
T4 DNA ligase : 0,2 uL
H2O : 11 uL
Let incubate at 22°C for 30 min
Chemical transformation in NEB turbo
Reference protocol
Recovery 1 hour.
Plate on
Tube 1 = negative control
Tube 2 : Kan (purif) Kan / Cm
Tube 3 : Kan (heat inactivated) Kan/ Cm
Tube 4 : lac(purify, enzyme cannot be heat inactivated) Cm/ IPTG/ Xgal
Cloning the Weapon (ie sRNA into Litmus and pUC 18)
Gel of litmus PCR
Gel purification of litmus PCR using Thermo Scientific Kit
PCR from mini Gene sRNA anti Kan
35 H2O
10 Buffer 5x
1 dNTP
2 FW iT009
2 RV iT0010
0.2 minigene-plasmid amp from IDT
0.5 phusion
PCR Phusion Gradient around 53 °C
Time : 30 secondes
Gibson Assembly
Protocol from the Kit, quantities are divided by 2: final volume of reaction = 10uL
Assembly of pUC18-SRNA_KAN : 5uL MasterMix, 0.5uL H2O, 3uL pUC18 with overhang from PCR (nanodrop = 13,4), 1,5uL sRNA from PCR of minigene (nanodrop =     26,7)
Assembly of Litmus28i-GFP-SRNA_KAN : 5uL MasterMix, 3,6uL H2O, 0,5uL Litmus28i-GFP with overhang from PCR (nanodrop = 85), 0,9uL sRNA from PCR of minigene     (nanodrop = 26,7)
Heat shock Transformation into provided competent cells. 60 min recovery, plate on AMP.
Regular cloning using biobricks
Digestion
Insert sRNA cut by XbaI & PstI
Vector Litmus cut by PstI & SpeI
Insert sRNA cut by EcoRI & SpeI
Vector Litmus cut by EcoRI & SpeI
Digestion of Vectors
10 H2O
2 Buffer
4 Vector
2 Enz 1
2 Enz 2
20uL Tot
37°C
20min
Digestion of Inserts
10 H2O
15 insert
3 Buffer
1 Enz 1
1 Enz 2
30uL Tot
37°C
20min
Heat inactivation before ligation
80°C for 20 minutes
Ligation
pUC= 40ng in 20 uL => 3ng/ul
litmus 340ng =>17ng /ul
insert 420 ng => 14 ng / uL
Litmus :
7uL vector
8 ul insert
2 uL Buffer
O,2 uLT4 DNA ligase
3ul Water
Vtot = 20 uL
pUC
20 uL vector
9 uL insert
4 uL Buffer
0,4 T4 DNA
7 uL water
Vtot = 40uL
Chemical transformation in NEB turbo
Tube 1 : negative control
Tube 2 lig litmus
Tube 3 lig pUC
plate everything on amp
Sabotage
ASDF
20th August
CLONING the target Plasmids (Kan and lacZ)
Gel purification of LacZ using Thermo Scientific Kit
lacZ : 83 ng/ul
Gel purification of Kan using Thermo Scientific Kit
Kan : 80ng/ul
Digestion of pACYC (pT002) with EcoRI and XhoI
10 H2O
2 Buffer
4 pACYC
2 EcoRI
2 XhoI
20uL Tot
37°C
20min
Digestion of pACYC (pT002) with BglII and NcoI
10 H2O
2 Buffer
4 pACYC
2 BglII
2 NcoI
20uL Tot
37°C
20min
Digestion of Kan insert with EcorI and XhoI
10 H2O
15 Kan insert
3 Buffer
1 EcoRI
1 XhoI
30uL Tot
37°C
20min
Digestion of LacZ insert with BglII and NcoI
10 H2O
15 LacZ insert
3 Buffer
1 BglII
1 NcoI
30uL Tot
37°C
20min
Gel of digestions
100Bp+ / pACYC+EcoRI+XhoI / pACYC+EcoRI+XhoI / pACYC+BglII+NcoI /pACYC+BglII+NcoI
Colum purification of Kan insert digestions using Thermo Scientific Kit
Column purification of LacZ insert digestions using Thermo Scientific Kit
Colum purification of pACYC digestions using Thermo Scientific Kit
Heat inactivation of Kan digestion
65°C
20min
Ligation Kan insert purif + pACYC (digestion EcoRI + XhoI)
Digestion vector : 6.8 ng/ul
Digestion insert : 14
masse insert = 10*1052/3770*50 = 140 ng
Vector : 7.35 uL
Insert : 10 uL
Buffer : 2 uL
T4 DNA ligase : 0,2 uL
Let incubate at 22°C for 30 min
Ligation Kan insert heat desactivated + pACYC (digestion EcoRI + XhoI)
Digestion vector : 6.8 ng/ul
Digestion insert : 40/2 = 20
masse insert = 10*1052/3770*50 = 140 ng
Vector : 7.35 uL
Insert : 7 uL
Buffer : 2 uL
T4 DNA ligase : 0,2 uL
H2O : 4 uL
Let incubate at 22°C for 30 min
Ligation LacZ insert purif + pACYC (pigestion BglII + NcoI)
Digestion vector : 10? ng/ul
Digestion insert : 33
masse insert = 10*449/3770*50 = 59 ng
Vector : 5 uL
Insert : 1.8 uL
Buffer : 2 uL
T4 DNA ligase : 0,2 uL
H2O : 11 uL
Let incubate at 22°C for 30 min
Chemical transformation in NEB turbo
Reference protocol
Recovery 1 hour.
Plate on
Tube 1 = negative control
Tube 2 : Kan (purif) Kan / Cm
Tube 3 : Kan (heat inactivated) Kan/ Cm
Tube 4 : lac(purify, enzyme cannot be heat inactivated) Cm/ IPTG/ Xgal
Cloning the Weapon (ie sRNA into Litmus and pUC 18)
Gel of litmus PCR
Gel purification of litmus PCR using Thermo Scientific Kit
PCR from mini Gene sRNA anti Kan
35 H2O
10 Buffer 5x
1 dNTP
2 FW iT009
2 RV iT0010
0.2 minigene-plasmid amp from IDT
0.5 phusion
PCR Phusion Gradient around 53 °C
Time : 30 secondes
Gibson Assembly
Protocol from the Kit, quantities are divided by 2: final volume of reaction = 10uL
Assembly of pUC18-SRNA_KAN : 5uL MasterMix, 0.5uL H2O, 3uL pUC18 with overhang from PCR (nanodrop = 13,4), 1,5uL sRNA from PCR of minigene (nanodrop =     26,7)
Assembly of Litmus28i-GFP-SRNA_KAN : 5uL MasterMix, 3,6uL H2O, 0,5uL Litmus28i-GFP with overhang from PCR (nanodrop = 85), 0,9uL sRNA from PCR of minigene     (nanodrop = 26,7)
Heat shock Transformation into provided competent cells. 60 min recovery, plate on AMP.
Regular cloning using biobricks
Digestion
Insert sRNA cut by XbaI & PstI
Vector Litmus cut by PstI & SpeI
Insert sRNA cut by EcoRI & SpeI
Vector Litmus cut by EcoRI & SpeI
Digestion of Vectors
10 H2O
2 Buffer
4 Vector
2 Enz 1
2 Enz 2
20uL Tot
37°C
20min
Digestion of Inserts
10 H2O
15 insert
3 Buffer
1 Enz 1
1 Enz 2
30uL Tot
37°C
20min
Heat inactivation before ligation
80°C for 20 minutes
Ligation
pUC= 40ng in 20 uL => 3ng/ul
litmus 340ng =>17ng /ul
insert 420 ng => 14 ng / uL
Litmus :
7uL vector
8 ul insert
2 uL Buffer
O,2 uLT4 DNA ligase
3ul Water
Vtot = 20 uL
pUC
20 uL vector
9 uL insert
4 uL Buffer
0,4 T4 DNA
7 uL water
Vtot = 40uL
Chemical transformation in NEB turbo
Tube 1 : negative control
Tube 2 lig litmus
Tube 3 lig pUC
plate everything on amp
Sabotage
ASDF
21st August
Place your twit here
Results of assemblies (Colonies on the transformation plates):
sRNA –Kan into Litmus by Gibson: Success
sRNA – Kan into Litmus by biobrick assembly: Success
sRNA – Kan into pUC18 by Gibson: Failure
sRNA – Kan into pUC18 by biobrick assembly: Success
Colony PCR to check if sRNA-KAN is in our Litmus phagemid: positive band between 400 and 500: (expected length 419bp)
Check the design of primers to clone out Kan => problem => redesign them
Design of the primers to sequence all the cloning
Launch 3 colonies of each successful cloning
Glycerol and mini prep of all the launched colonies / need 3 before we have
Construction of target strain
Electroporation of pCOLA in MGZ1 F+
Construction of weapon strains
Electroporation of litmus-GFP-sRNA in XL1 Blue-M13KO7-Helper
Electroporation of Litmus-GFP-sRNA and M13K07 in MGZ1 F+
Chemical transformation of minigene+vector-sRNA anti Kan in NEB turbo
Sabotage
ASDF
22nd August
Results :
All the transformation worked
Launch cultures (to do experiment)
mGZ1 F+ -Cola
MGZ1 F+ litmus + Helper
Launch overnigh culture to do glycerols.
exploring the parameters in play in the phage infectiveness of our new phagemid construct
-diluting 100x O/N of MGZ1, F+
-mix with 1/10 vol of supernatant containing phagemids with the cells with the following combination of parameters:
OD600 : 0.3 0.6 0.9
Plate at different time (min) after first contact with the supernatant: 2, 30, 60
Silencing experiments
- Launch O/N culture of producing phages cells ; centrifuge ; take supernatant and filter it.
-diluting 100x O/N of MGZ1, F+ -pCola in LB-Kan
- At OD 0,7 : 4 tubes : Ctrl, P1, P2 ,P3. Put 1 mL of cells in each tube. Centrifuge. Take out supernatant, resuspend in 1mL of LB. Add in tubes     respectively : nothing, 100uL phages of Miniprep 1, Miniprep 2, miniprep 3.
- Incubate 45 minutes at 37°C
- Serial Dilute every tube until 10-5. Plate for each tube 100ul of 10-5 dilutions on Kan and LB and 100 ul of 10-2 and 10-3 dilution on Kan.
Sabotage
ASDF
23th August
Results: failure: as much GFP+ colonies growing on Kan than on LB
PCR from mini Gene sRNA anti Cm
35 H2O
10 Buffer 5x
1 dNTP
2 FW iT009
2 RV iT0010
0.2 minigene-plasmid amp from IDT
0.5 phusion
PCR Phusion Gradient around 53 °C
Time : 30 secondes
Sabotage
ASDF
26th August
Digestion of litmus miniprep (C=280ng/uL)with PstI and SpeI
13 H2O
3 litmus miniprep
2 Buffer
1 PstI
1 SpeI
20uL Tot
37°C
20min
Digestion of sRNA_Cm insert (C=31ng/uL) with XbaI and PstI
10 H2O
15 sRNA_Cm insert
3 Buffer
1 XbaI
1 PstI
30uL Tot
37°C
20min
PCR purification of litmus digestion using Thermo Scientific Kit
PCR purification of sRNA_Cm insert digestion using Thermo Scientific Kit
Ligation sRNA_Cm insert purif + litmus
Digestion vector : 13,3 ng/ul
Digestion insert : 10
masse insert = 10*376/3626*50 = 52 ng
Vector : 3.8 uL
Insert : 5.2 uL
Buffer : 2 uL
T4 DNA ligase : 1 uL
H2O : 8 uL
Let incubate at 22°C for 30 min
Ligation control litmus
Vector : 3.8 uL
Insert : 0 uL
Buffer : 2 uL
T4 DNA ligase : 1 uL
H2O : 13,2 uL
Let incubate at 22°C for 30 min
Digestion of pUC18 PCR (C=13ng/uL) with EcorI and SpeI
10 H2O
15 pUC18 PCR
3 Buffer
1 EcorI
1 SpeI
30uL Tot
37°C
20min
Digestion of sRNA_Cm insert (C=31ng/uL) with SpeI and EcorI
10 H2O
15 sRNA_Cm insert
3 Buffer
1 EcorI
1 SpeI
30uL Tot
37°C
20min
PCR purification of pUC18 digestion using Thermo Scientific Kit
PCR purification of sRNA_Cm insert digestion using Thermo Scientific Kit
Ligation sRNA_Cm insert purif + pUC18
Digestion vector : 4,3 ng/ul
Digestion insert : 9,4
masse insert = 10*362/2175*50 = 83 ng
Vector : 11,6 uL
Insert : 8,8 uL
Buffer : 3 uL
T4 DNA ligase : 1,5 uL
H2O : 7,1 uL
Let incubate at 22°C for 30 min
Ligation control pUC18
Vector : 11,6 uL
Insert : 0 uL
Buffer : 3 uL
T4 DNA ligase : 1,5 uL
H2O : 15,9 uL
Let incubate at 22°C for 30 min
Chemical transformation in NEB-Turbo only (control -)
Chemical transformation in NEB-Turbo of ligation sRNA_Cm insert purif + litmus
Chemical transformation in NEB-Turbo of ligation control litmus
Chemical transformation in NEB-Turbo of ligation sRNA_Cm insert purif + pUC18
Chemical transformation in NEB-Turbo of ligation control pUC18
Chemical transformation in NEB-Turbo of litmus (control +)
Plate all 5 on Amp
PCR from mini Gene sRNA anti LacZ
35 H2O
10 Buffer 5x
1 dNTP
2 FW iT009
2 RV iT0010
0.2 minigene-plasmid amp from IDT
0.5 phusion
PCR Phusion Gradient around 53 °C
Time : 30 secondes
Sabotage
ASDF
27th August
Place your twit here
Building anti LacZ weapons:
Gibson assembly of litmus (sT005) and sRNA_LacZ
Digestion vector : 56 ng/ul
Digestion insert : 9,6
masse insert = 5*376/3626*50 = 26 ng
Vector : 1 uL
Insert : 3 uL
Master Mix : 5 uL
H2O : 1
Heat shock following Gibson assembly kit protocole.
Biobrick assemblyof sRNA_LacZ in litmus-GFP and pUC18
Using digested vectors (litmus-GFP and pUC18) from 26/08
Digestion of sRNA_LacZ insert (C=48ng/uL) with XbaI and PstI
10 H2O
15 sRNA_LacZ insert
3 Buffer
1 PstI
1 XbaI
30uL Tot
37°C
20min
Digestion of sRNA_LacZ insert (C=45ng/uL) with SpeI and EcorI
10 H2O
15 sRNA_LacZ insert
3 Buffer
1 EcorI
1 SpeI
30uL Tot
37°C
20min
Purification on column
ligation:
Ligation sRNA_LacZ insert purif + pUC18
Digestion vector : 4,3 ng/ul
Digestion insert : 18
masse insert = 10*362/2175*50 = 83 ng
Vector : 11,6 uL
Insert : 4,6 uL
Buffer : 3 uL
T4 DNA ligase : 1,5 uL
H2O : 11 uL
Let incubate at 22°C for 30 min
Ligation sRNA_LacZ insert purif + litmus
Digestion vector : 13,3 ng/ul
Digestion insert : 11,4
masse insert = 10*376/3626*50 = 52 ng
Vector : 3.8 uL
Insert : 4,56 uL
Buffer : 2 uL
T4 DNA ligase : 1 uL
H2O : 8,64 uL
Let incubate at 22°C for 30 min
Preparing experiments of LacZ silencing
Plating MGZ1, F+ ; MG1655, F+ ; NEB Turbo on IPTG+Xgal
Sabotage
ASDF
28th August
as expected MGZ1 and MG1655 turn blue.
Result from assemblies:
Gibson sRNA LacZ in litmus-GFP: failure
Biobrick assembly sRNA LacZ in Litmus-GFP : success
Biobrick assembly sRNA LacZ in pUC18 : success
Miniprep of Litmus-GFP-sRNACm ,Litmus-GFP-sRNALacZ, pUC18-sRNACm , pUC18-sRNALacZ
Building phagemid weapon producing cells: Transformation of electrocompetent MGZ1, F+ with M13K07 + Litmus-GFP-sRNALacZ / Litmus-GFP-sRNACm
Transforming:
-1) negative control : 5,2
-2) IDT plasmid with sRNALacZ ( should form white colonies of MGZ1 on Xgal+IPTG):5,1
-3) Litmus-GFP-sRNALacZ1 ( should form white colonies of MGZ1 on Xgal+IPTG): 5,2
-4) Litmus-GFP-sRNALacZ3 ( should form white colonies of MGZ1 on Xgal+IPTG): 5,2
-5) Litmus-GFP-sRNACm1 (control XGal ) : 5,2
-6) Litmus-GFP-sRNACm1 + M13K07: 5,2
-7) Litmus-GFP-sRNACm2 + M13K07 : 5,2
-8) Litmus-GFP-sRNACm3 + M13K07: 5,2
-9) Litmus-GFP-sRNALacZ1 + M13K07 : 5,2
-10) Litmus-GFP-sRNALacZ3 + M13K07: 5,2
plate on AMP+IPTG+Xgal ( silencing of LacZ on MGZ1 strain) and KAN+AMP (phage factories cells)
Sabotage
ASDF
29th August
Place your twit here
silencing of LacZ failed on MGZ1: the sequence of LacZ might have mutation compared to MG1655 for which the sRNA was designed or the test is not     quantitative enough (one sRNA => several proteins). Try to find a way to get quantitative data about the silencing => Miller Assay
Sabotage
ASDF
30th August
Place your twit here
attempt to silence LacZ on MG1655, F+ with litmusGFP-LacZ1 & litmusGFP-LacZ2
XXXXX
Silencing experiments (LacZ)
- Launch O/N culture of producing phages cells ; centrifuge ; take supernatant and filter it.
-diluting 100x O/N of MGZ1, F+ -pCola in LB-Kan
- At OD 0,7 : 5 tubes. Put 1 mL of cells in each tube. Add in tubes respectively : nothing, 100uL phages of litmusGFP-LacZ1 & litmusGFP-LacZ2,     litmusGFP-Cm1 & litmusGFP-Cm2
litmusGFP-Cm1 & litmusGFP-Cm2 are used as negative control.
- Incubate 45 minutes at 37°C
- Serial Dilute every tube until 10-5. Plate for each tube 100ul of 10-5 and 10-3 dilutions on IPTG+Xgal and LB and 100uL on AMP+IPTG+Xgal and LB.
Sabotage
ASDF
3rd September
Silencing experiments (Cm)
- Launch O/N culture of producing phages cells ; centrifuge ; take supernatant and filter it.
-diluting 100x O/N of MGZ1, F+ -pACYc in LB-Cm (25 ug/ul)
- At OD 0,7 : 5 tubes. Put 900 uL of cells in each tube. Add in tubes respectively : nothing, 100uL phages of litmusGFP-LacZ1 & litmusGFP-LacZ2,     litmusGFP-Cm1 & litmusGFP-Cm2
litmusGFP-lacZ1 & litmusGFP-lacZ2 are used as negative control.
- Incubate 45 minutes at 37°C
- Serial Dilute every tube until 10-5. Plate 100ul for each tube according to this dilution table
LB
Cm 25 ug/ul
Cm 250ug/ul
Cm 500 ug/ul
non infected
10^-4
10^-4
10^-4
10^-4
sRNA anti lacz
10^-4
10^-4
10^-3
10^-3
sRNA Cm1
10^-4
10^-4
10^-3
10^-3
sRNA Cm2
10^-4
10^-4
10^-3
10^-3
sRNA Cm3
10^-4
10^-4
10^-3
10^-3
sRNA Cm4
10^-4
10^-4
10^-3
10^-3
Sabotage
ASDF
4th September
Results of silencing experiments
It works !
Miller assay
XXXXXX
Redo the silencing experiment
- Launch O/N culture of producing phages cells ; centrifuge ; take supernatant and filter it.
-diluting 100x O/N of MGZ1, F+ -pACYc in LB-Cm (25 ug/ul)
- At OD 0,7 : 5 tubes. Put 900 uL of cells in each tube. Add in tubes respectively : nothing, 100uL phages of litmusGFP-LacZ1, litmusGFP-Cm1 & litmusGFP-Cm2
litmusGFP-lacZ1 & litmusGFP-lacZ2 are used as negative control.
- Incubate 45 minutes at 37°C
- Serial Dilute every tube until 10-5. Plate 100ul for each tube according to this dilution table
LB
Cm 25 ug/ul
Cm 125 ug/ul
Cm 250ug/ul
Cm 500 ug/ul
non infected
10^-4
10^-4
10^-4
10^-4
10^-4
sRNA anti lacz
10^-4
10^-4
10^-3
10^-3
10^-3
sRNA Cm1
10^-4
10^-4
10^-3
10^-3
10^-3
sRNA Cm2
10^-4
10^-4
10^-3
10^-3
10^-3
sRNA Cm3
10^-4
10^-4
10^-3
10^-3
10^-3
sRNA Cm4
10^-4
10^-4
10^-3
10^-3
10^-3
Sabotage
ASDF
5th September
Confirmation of silencing (CmR)
Cloning biobricks : ie clone sRNAs (Cm, Kan, lacZ) in pB1C3
Reagent
Volume
Purified PCR Product
16 ul
H2O
0 ul
10x FastDigest Buffer
2 ul
FastDigest Enzyme 1
1 ul
FastDigest Enzyme 2
1 ul
Total Volume
20.0 ul
Digest for 0.5 - 1 hours at 37 °C with shaking
Ligation sRNA into biobrick vector
masse insert = 10*419/2000*100 = 210 ng
Vector : 5 uL
Insert : 10 uL
Buffer : 2 uL
T4 DNA ligase : 1 uL
H2O : 2uL
Let incubate at 22°C for 30 min
Transformation in NEB turbo chemical competent . Plate on Cm
Sabotage
ASDF
7th September
Place your twit here
07/09/13
Results Transformation
Colonies on the different paltes (no colonies on the negative controls)
PCR Colonies
Primer from Phage sensor team : XXXXX
Gel :
XXXXX
Sabotage
ASDF
10th September
Place your twit here
Experiment Silencing Cm (try to reduce the noise, aka the variability)
- Launch O/N culture of producing phages cells ; centrifuge ; take supernatant and filter it.
-diluting 100x O/N of MGZ1, F+ -pACYc in LB-Cm (25 ug/ul)
- At OD 0,7 :. Put 900 uL of cells in each tube. Add in tubes respectively : nothing, 100uL phages of litmusGFP-LacZ1, litmusGFP-Cm1 & litmusGFP-Cm2
-3 replicates for each condition. Plate each Dilution 2 times to limit the variability
Non infected cells (1,2,3)
Cells infected with anti lacZ (1,2,3)
Cells infected with anti CmA (1,2,3)
Cells infected with antiCmB (1,2,3
- Incubate 45 minutes at 37°C
- Serial Dilute every tube until 10-5. Plate 100ul for each tube according to this dilution table
LB
Cm 25 ug/ul
Cm 250ug/ul
Cm 500 ug/ul
non infected 1
10^-5/4
10^-5/4
10^-5/4
10^-5/4
non infected 2
10^-5/4
10^-5/4
10^-5/4
10^-5/4
non infected 3
10^-5/4
10^-5/4
10^-5/4
10^-5/4
sRNA lacZ 1
10^-4
10^-4
10^-4/2
10^-2/1
sRNA lacZ 2
10^-4
10^-4
10^-4/2
10^-2/1
sRNA lacZ 3
10^-4
10^-4
10^-4/2
10^-2/1
sRNA Cm A 1
10^-4/3
10^-4/3
10^-3
10^-2/1
sRNA Cm A 2
10^-4/3
10^-4/3
10^-3
10^-2/1
sRNA Cm A 3
10^-4/3
10^-4/3
10^-3
10^-2/1
sRNA Cm B 1
10^-4/3
10^-4/3
10^-3
10^-2/1
sRNA Cm B 2
10^-4/3
10^-4/3
10^-3
10^-2/1
sRNA Cm B 3
10^-4/3
10^-4/3
10^-3
10^-2/1
Sabotage
ASDF
11th September
Results experiment Silencing
AT a concentration of 500mg/ul, we get 99,1% of killing (on 12 replicates) with a standard déviation of 1,19
When you try to understand why are the cells surviving, it is mainly due to the non silencing of the cells
This problem can be due to the necessity of the sRNA to match to the plasmid sequence perfectly (and therefore problemsof mutations, especially on             phage).
Sabotage
ASDF
14th September
Place your twit here
Cloning two sRNA on the same phagemid
P009 (litmus Kan) + sRNA Cm :
P027 (litmus Cm) + sRNA Kan :
Inserts
sRNA Cm : 35 ng/uL Final (16*35/20) = 28 ng/uL
sRNA Kan : 16 ng/uL Final (16*16/20) = 12,8 ng/uL
Reagent
Volume
Purified PCR Product
16 ul
H2O
0 ul
10x FastDigest Buffer
2 ul
XbaI
1 ul
Pst1
1 ul
Total Volume
20.0 ul
Vectors
Litmus Kan : 350 ng /uL Final (3*350/20) = 52,5 75 ng/ul
Litmus Cm : 750 ng /ul Final (2*750/20) = 75 ng/uL
Reagent
Volume
Purified PCR Product
3
H2O
13 ul
10x FastDigest Buffer
2 ul
Pst1
1 ul
SpeI
1 ul
Total Volume
20.0 ul
Ligation sRNA into biobrick vector
Litmus Kan + sRNA anti Cm
masse insert = 10*419/4000*100 = 110 ng
Vector : 3 uL
Insert : 4 uL
Buffer : 2 uL
T4 DNA ligase : 1 uL
H2O : 10uL
Litmus Cm + sRNA anti Kan
masse insert = 10*419/4000*100 = 110 ng
Vector : 1,5 uL
Insert : 10 uL
Buffer : 2 uL
T4 DNA ligase : 1 uL
H2O : 5,5uL
Let incubate at 22°C for 30 min
Launch cultures :
Glycerol of the different biobricks : sT040 ; sT041 ; sT042 sT043 ; sT044 ; sT045 ; sT046 ; sT047 on Cm
MG F+ + pCola sT027 on Kan :
MGZ1, F+ Litmus-GFP-sRNA antiKan + M13KO7 Helper col : sT022 sT023 sT024 on Kan Amp
Sabotage
ASDF
15th September
Exp Silencing Kan
- Launch O/N culture of producing phages cells ; centrifuge ; take supernatant and filter it.
-diluting 100x O/N of MGZ1, F+ -pCola in LB-Kan (25 ug/ul)
- At OD 0,7 :. Put 900 uL of cells in each tube. Add in tubes respectively : nothing, 100uL phages of litmusGFP-LacZ1, litmusGFP-Kan1 & litmusGFP-Kan2
-2 replicates for each condition. Plate each Dilution 2 times to limit the variability
Non infected cells (1,2)
Cells infected with anti lacZ (1,2)
Cells infected with anti KanA (1,2)
Cells infected with anti KanB (1,2,)
- Incubate 70 minutes at 37°C
- Serial Dilute every tube until 10-5. Plate 100ul for each tube according to this dilution table
LB
Kan 25 ug/ul
Kan 250ug/ul
Kan 500 ug/ul
non infected 1
10^-5/4
10^-4/3
10^-4/3
10^-4/3
non infected 2
10^-5/4
10^-4/3
10^-4/3
10^-4/3
sRNA lacZ 1
10^-5/4
10^-4/3
10^-4/3
10^-3/2
sRNA lacZ 2
10^-5/4
10^-4/3
10^-4/3
10^-3/2
sRNA Kan A 1
10^-5/4
10^-4/3
10^-4/3
10^-3/2
sRNA Kan A 2
10^-5/4
10^-4/3
10^-4/3
10^-3/2
sRNA Kan B 1
10^-5/4
10^-4/3
10^-4/3
10^-3/2
sRNA Kan B 2
10^-5/4
10^-4/3
10^-4/3
10^-3/2
Miniprep biobricks
Cm1 / Cm2 / Kan 1 / Kan 2 / Lac1 / Lac2
Cloning 2 sRNA on the same phage : Ligation sRNA into biobrick vector
Litmus Kan + sRNA anti Cm
masse insert = 10*419/4000*100 = 110 ng
Vector : 3 uL
Insert : 4 uL
Buffer : 2 uL
T4 DNA ligase : 1 uL
H2O : 10uL
Litmus Cm + sRNA anti Kan
masse insert = 10*419/4000*100 = 110 ng
Vector : 1,5 uL
Insert : 10 uL
Buffer : 2 uL
T4 DNA ligase : 1 uL
H2O : 5,5uL
Let incubate at 22°C for 30 min
Transformation in chemical competent 10G cells : plate on Amp;
Sabotage
ASDF
18th September
Exp Silencing Kan
- Launch O/N culture of producing phages cells ; centrifuge ; take supernatant and filter it.
-diluting 100x O/N of MGZ1, F+ -pCola in LB-Kan (25 ug/ul)
- At OD 0,7 :. Put 900 uL of cells in each tube. Add in tubes respectively : nothing, 100uL phages of litmusGFP-LacZ1, litmusGFP-Kan1 & litmusGFP-Kan2
-2 replicates for each condition. Plate each Dilution 2 times to limit the variability
Non infected cells (1,2)
Cells infected with anti lacZ (1,2)
Cells infected with anti KanA (1,2)
Cells infected with anti KanB (1,2,)
- Incubate 70 minutes at 37°C
- Serial Dilute every tube until 10-5. Plate 100ul for each tube according to this dilution table
LB
Kan 25 ug/ul
Kan 250ug/ul
Kan 500 ug/ul
non infected 1
10^-5/4
10^-4/3
10^-4/3
10^-4/3
non infected 2
10^-5/4
10^-4/3
10^-4/3
10^-4/3
sRNA lacZ 1
10^-5/4
10^-4/3
10^-4/3
10^-3/2
sRNA lacZ 2
10^-5/4
10^-4/3
10^-4/3
10^-3/2
sRNA Kan A 1
10^-5/4
10^-4/3
10^-4/3
10^-3/2
sRNA Kan A 2
10^-5/4
10^-4/3
10^-4/3
10^-3/2
sRNA Kan B 1
10^-5/4
10^-4/3
10^-4/3
10^-3/2
sRNA Kan B 2
10^-5/4
10^-4/3
10^-4/3
10^-3/2
Miniprep biobricks
Cm1 / Cm2 / Kan 1 / Kan 2 / Lac1 / Lac2
Cloning 2 sRNA on the same phage : Ligation sRNA into biobrick vector
Litmus Kan + sRNA anti Cm
masse insert = 10*419/4000*100 = 110 ng
Vector : 3 uL
Insert : 4 uL
Buffer : 2 uL
T4 DNA ligase : 1 uL
H2O : 10uL
Litmus Cm + sRNA anti Kan
masse insert = 10*419/4000*100 = 110 ng
Vector : 1,5 uL
Insert : 10 uL
Buffer : 2 uL
T4 DNA ligase : 1 uL
H2O : 5,5uL
Let incubate at 22°C for 30 min
Transformation in chemical competent 10G cells : plate on Amp;
Sabotage
ASDF
20th September
results from sequencing:
Litmus-Anti LacZ: OK
Litmus-Anti Kan: OK
Litmus-Anti Cm: OK
Retrieved from " http://2013.igem.org/Team:Paris_Bettencourt/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
