Team:UChicago/Notebook
Home
Team
Team Members Collaborations
Project
Description Design Experiments Notebook InterLab Results Improve Attributions
Parts Safety Human Practices
Notebook
June
6/14/18 -Got familiar with lab -Made LB and normal YPAUD liquid media         -Labelled liquid media and left on bench shelf -Make chloramphenicol plates         -250mg chloramphenicol; marked by CAM 6/15/18 -Made Arg- YPAUD liquid media        -Used CSM -Arg, -Ura; added uracil in a 1:1 ratio with CSM (1.7g of CSM -Arg -Ura, 1.7g of uracil)        -Marked by 3 blue stripes -Made Arg- YPAUD plates     -Used CSM -Arg, -Ura; added uracil in a 1:1 ratio with CSM     -Marked by three blue stripes -PCR amplify arg-ars sequence     -Total volume: 50µL         -35µL water         -10µL 5X phusion buffer         -1µL dNTPs         -2.5µL 10µM primer stock (forward and reverse arg-ars primers)         -1µL pOW1 (17.6 ng/µL) DNA stock (0.8µL DNA, 4.2µL H2O)         -0.5µL phusion         -Stored in refrigerator next to PCR machine (labeled iGEM arg-ars PCR) 6/18/18 -Gel electrophoresis of arg-ars sequence (PCR amplified on 6/15/18)     -Check if PCR was successful     -Agarose gel:         -40 mL 1X SB buffer, 0.32g agarose         -Mixed and microwaved for 35 seconds         -Added 2µL EtBr to agarose gel in flask     -Agarose gel opaque and gray when fully polymerized     -Sat in chamber, completely covered in buffer, until ready to run the gel -Gel was unsuccessful, neither the ladder band nor the PCR product band could be observed     -Suspected problem: samples not loaded into wells properly         -Large smear of DNA across wells observed -PCR amplified arg-ars sequence with 1 min per Kb (5 min since Arg-ars sequence is 4.86 Kb)     -Labeled in blue marker 6/19/18 -PCR amplified arg-ars sequence with 30 seconds per Kb (2.5 minutes since Arg-ars sequence is 4.86 Kb)     -Labeled in black marker -Gel purification Arg-ars PCR products     -Made 0.8% agarose gel         -0.32 g agarose; 40 mL 1X SB buffer; 1µL EtBr         -Each well has 25µL of sample     -Well 1 loaded with 1 min per Kb arg-ars PCR product     -Well 3 loaded with DNA ladder---     -Well 5 loaded with 30 sec per Kb arg-ars PCR product     -Ran at 120V for 25 min (4.86 Kb sequence) -Well 5 came out clear, however wells 1 and 3 were diffuse to be observed     -Suspected problem: holding micropipette to the bottom of the well when pushing to the second stop, when pulling the micropipette out, dye doesn’t remain 6/20/18 -Gel purification Arg-ars PCR products     -Made 0.8% agarose gel         -0.32 g agarose; 40 mL 1X SB buffer; 1µL EtBr     -Well 1 loaded with 1 min per Kb arg-ars PCR product     -Well 4 loaded with DNA ladder     -Well 7 loaded with 30 sec per Kb arg-ars PCR product     -Ran at 120V for 25 min (4.86 Kb sequence) Bands were smeared     -Michael: voltage likely too high, run at 90V; ladder lane did not have clear bands, try running the ladders against each other; the PCR product with 30 seconds per Kb produced very faint bands, use the protocol with 1 min per Kb -Identified the (large and rather smeared) region where the arg-ars sequence was likely to be and excised it     -Placed in eppendorf tube and added 450 µL of GQ buffwe     -Placed in 50º bath for 10 minutes     -Added 450µL of GQ buffer (solubilization buffer), centrifuged at 5.0k rpm for 1 min     -Added 200 µL of PE buffer (wash buffer), centrifuge at 13.0k rpm for 30 seconds     -Added 200µL of PE buffer (wash buffer), centrifuged at 13.0k rpm for 3 minutes     -Inverted and set to dry for 3 minutes     -Nanodrop detected no DNA (no peak at wavelength of 260 nm) -PCR amplified arg-ars sequence (1 min per Kb)     -Made two PCR products (each 50µL)     -Used protocol outlined above -Cleaned lab area 6/21/18 -Gel purification of arg-ars PCR product     -0.8% agarose gel with 1X SB buffer, 2µL EtBr     -Run at 105V for 30 minutes     -Lane 1: arg-ars PCR product; lane 3: 100 bp ladder     -Bands observed in gel were too smeared and also too short to be the desired sequence -Simone’s suggestions     -Not trying to PCR the entirety of POW1 (which is 4.86 Kb), only trying to amplify a 2.5 Kb section that contains ScARG4     -Run PCR with annealing temp at 58ºC instead of 55ºC 6/22/18 -PCR amplified arg-ars sequence with an annealing temp of 58ºC -Gel electrophoresis of arg-ars PCR product     -0.8% agarose gel with 1X SB buffer and 2µL EtBr     -105V for 35 minutes     -Lane 1: 1 Kb ladder; lane 3: arg-ars PCR product; lane 5: arg-ars PCR product     -Lane 5 did not come out clear     -Ladder too bright, added too great a volume to the well (20 µL instead of 5µL) -Gel purify arg-ars PCR product -No DNA detected by the nanodrop 6/22/18 -Did diagnostic work on the PCR reaction performed by RF     -Suspected cause 1: Template was at 18.2 ng/µL instead of the 96.5 labeled on the tube         -Sample also has a 260/230 of 1.27 indicating EtOH contamination inflating this value         -Likely that there is not enough template to successfully create PCR product         -Only 2µL remain anyway, so need new pOW1 to proceed anyway     -Suspected Cause 2: Primers are not specific enough. Forward primer’s last 5 3’ bases match to 7 locations. One of these forms a 600 bp product that is consistent with the lower band. Can extend the primer 5 bases to create a specific primer that only binds to its target site. -To grow more pOW1, prepared an overnight culture of PPY12 as Allison instructed the plasmid to be grown in yeast 6/23/18 -Talked with Dr. Glick     -Indicated that cannot grow more plasmid in the PPY12 overnights as they need to be grown in bacteria     -Transformed 1µL of remaining pOW1 plasmid into DH5α cells. Plated onto an LB+Amp plate (See below) -Created 20 LB+Amp plates following Glick lab plate recipe. Plated both my sample on 1 and a negative control on another. Did not have a strain that I knew had Amp resistance already, so I was unable to do a complete positive control. 6/24/18 -LB+ Amp Plate result     -Negative control had no growth indicating the antibiotics are in sufficient quantity to kill non antibiotic resistant bacteria     -pOW1 sample had too many colonies to count indicating a successful transformation was likely (Had no positive control, so cannot be fully certain until attempt to isolate the plasmid is performed)         -Inoculated 5 overnight liquid cultures (4 for plasmid harvesting and one to make a glycerol stock and future positive Amp control) 6/25/18 -Miniprep Liquid cultures A→ D     -Concentrations         -A: 748.9 ng/µL         -B: 573.3 ng/µL         -C: 527.9 ng/µL         D: 554.2 ng/µL -Made glycerol stock of Liquid culture E 6/26/18 -PCR amplified arg-ars sequence from pOW1 samples B-D     -Want ~20 ng of the plasmid DNA (these dilutions result in pOW1 concentrations of 20 ng/µL)         -A: 0.8µL pOW1 A, 29.2 µL H2O         -B: 0.8µL pOW1 A, 23.0 µL H2O         -C: 0.8µL pOW1 A, 21.1 µL H2O         -D: 0.8µL pOW1 A, 22.2 µL H2O -Made glycerol stock of liquid pOW1 culture E -Gel electrophoresis of arg-ars pOW1 A-D PCR products     -Lane 1: 1 Kb ladder; lane 2: pOW1 B; lane 4: pOW1 C; lane 6: pOW1 D     -Ran at 110V for 30 minutes         -Gel imaging showed only a smaller band around 100 bp for lane 2 (pOW1 sample B)             -Suggests primer issue; will address in team meeting tomorrow -PCR amplified arg-ars sequences from pOW1 sample A -PCR amplification of pOW1 sample B side by side with CC     -RF: followed previous PCR protocol, used arg-ars F1 and R1 primers     -CC:  Followed NEB Phusion protocol found on their website. Is as follows:         -10µL of 5x Phusion HF Buffer         -1µL 10mM dNTPs         -2.5µL of Forward Primer         -2.5µL of Reverse Primer         -1µL of Template DNA (Diluted 50X from original stock)         -0.5µL of Phusion Polymerase     -Annealing temperature was decreased to 57ºC -Gel electrophoresis of pOW1 B with CC     -Lane 1: 1 Kb ladder     -Lane 2: RF PCR protocol with primers F1 and R1     -Lane 4: CC pOW1 B sample         -This was unnecessary, it is the entire plasmid, not a PCR product     -Lane 6: CC         -This is Cian’s new PCR ‘recipe’ with F2 and R2             -The new primers Simone gave us -Gel imaging showed that the PCR was unsuccessful     -We might not have the pOW1 plasmid -Ran restriction digest of pOW1 samples A-D and last year’s stock with Xbal1     -10µL reaction         -1µL cutsmart buffer         -1µL of each DNA sample including last year’s stock (which we know to be pOW1)         -0.1µL of Xbal1 GQ         -7.9µL of milliQ water     -Place reaction in 37ºC incubator for one hour -Ran gel electrophoresis of the five 10µL reactions     -Lane 1: “Go Green” 1 Kb ladder from Simone     -Lane 3: control pOW1 sample (from 2017)     -Lane 4: pOW1 sample A     -Lane 6: pOW1 sample B     -Lane 7: pOW1 sample C     -Lane 8: pOW1 sample D     -Lane 10: 1 Kb ladder from our -20ºC freezer (below benchtop) 6/27/18 -Ran PCR of pOW1 protocol     -Annealing temperature of 62ºC     -Reaction 1: F1 R1     -Reaction 2: F1 R2     -Reaction 3: F2 R1     -Reaction 4: F2 R2     -Protocol: 50µL PCR product         -32.5 µL milliQ water         -10µL of 5x Phusion HF Buffer         -1µL 10 µM dNTPs         -2.5µL of Forward Primer         -2.5µL of Reverse Primer         -1µL of Template DNA (Diluted 50X from original stock)             -Used pOW1 sample B         -0.5µL of Phusion Polymerase     -PCR product 4 opened in the machine; no product remaining -Ran gel of pOW1 sample B PCR products     -Lane 1: “Go Green” 1 Kb ladder     -Lane 3: 1 Kb ladder found in -20ºC freezer below benchtop     -Lane 5: pOW1 PCR product 1 (F1 + R1)     -Lane 7: pOW1 PCR product 2 (F1 + R2)     -Lane 9: pOW1 PCR product 3 (F2 + R2)     -110V for 30 minutes -Results of gel     -Our 1 Kb is accurate and had bands comparable to the “Go Green” 1 Kb ladder provided by Simone (grad student) -Sterilized new eppendorf tubes in autoclave     -Dry cycle, 15 minutes -Sent 4.5µL of mini prepped pOW1 plasmid sample, 3µL of seq.F.3 primer, and 3 µL of seq.R.3 primer for sequencing         -Made a 1:10 dilution of the stock primers         -Made a 3:10 dilution of the stock pOW1 plasmid sample C     -To result in a concentration of roughly 300 ng/µL -PCR amplified reaction 4 again     -F2 and R2 6/28/18 -Gel electrophoresis of reaction 4     -PCR product from 6/27/18 with annealing temperature of 62ºC     -Ran at 110V for 30 minutes     -Lane 1: “Go Green” 1 Kb ladder     -Lane 3: reaction 4     -Gel imaging showed only one DNA band which was smaller than the 2.55 Kb region we are trying to amplify -Ran PCR protocol with an annealing temperature of 56ºC     -Reaction 1: F1 R1     -Reaction 2: F1 R2     -Reaction 3: F2 R1     -Reaction 4: F2 R2     -50X dilution of pOW1 sample B         -49 µL of milliQ water, 1µL sample -Gel electrophoresis and imaging of reactions 1-3 PCR products     -Ran at 110V for 30 minutes     -PCR tube containing reaction 4 opened so no product remained     -Lane 1 & lane 8: “Go Green” 1 Kb ladder         -Suspected contents from lane 2 might have bled over into the lane 1     -Lane 2: reaction 1 PCR product     -Lane 4: reaction 2 PCR product     -Lane 6: reaction 3 PCR product     -Gel showed only the shorter band, not the 2.55 Kb band we are looking for -Remade master mix for reaction 4 and re-ran PCR protocol     -Reaction 4: F2 and R2 -Gel electrophoresis of reactions 1-4     -Ran at 110V 0.8% Agarose     -All bands came back negative         -Only had the 600bp banding.
July
7/2/18 -PCR of Samples A-F of pOW1 using 2-step method from last year (outlined below in pictures)     -Annealing temperature 68°C     -Ran out of Phusion DNA Polymerase-working on getting more         -As a result, only Samples A-D could be PCRd properly     -Potential issue with PCR-Step 1 of  2 step not entered correctly; may need to redo PCR but after checking results -Gel Electrophoresis     -100 V ran for 20 min     -1% 50 mL Gel created as per Jason’s introductory guide 7/3/18 -PCR Samples A-F or pOW1 using 2-step method BUT WITH Taq Polymerase/Buffer instead of Phusion done correctly as per photo     -Annealing Temperature 68°C -Gel electrophoresis     -5ul ladder, 2 ul loading dye added to each sample     -Sample A not electrophoresed     -1 % 50 mL Gel created as per Jason’s introductory guide     -110 V 7/5/18 -Learned to use nanodrop machine -Attempted to digest pBSC13mut with Hpa1 and measure concentration, however concentration was quite low     -5ul 10x Cutsmart Buffer used     -.5ul Hpa1     -Template DNA added to 1 microgram     -MilliQ water added to 50ul -Given ArgArs sequence by Simone, however there was contamination so PCR clean up performed     -Used Wizard SV Gel and PCR Clean-Up System protocol     -Still no clear usable results and thus must PCR ArgArs sequence from given DNA on our own 7/6/18 -Discovered concentration of pBSC13mut is lower than expected (17.6ng/ul); may explain why digestion with Hpa1 doesn’t seem to yield results     -Began transformation of more pBSC13mut to be completed over weekend     -Protocol is as follows:         -1. Get DH5alpha cells from the bottom shelf of the -80C and put on Ice.         -2. Once defrosted, add 1µL of the transformation product into the cells. Mix by flicking the bottom. DO NOT PIPETTE TO MIX!!         -3. Incubate on ice for 30 min.         -4. Heat Shock the bacteria for 45sec in the 42C water bath. Let recover on ice for 2 min         -5. Add 300µL of SOC medium and place in shaking 37C incubator for 1hr         -6. Plate 50µL on Plate. Place place in the 37C incubator overnight (Place after 4:00pm) -Ran PCR w/ POW1 using protocol sent by Simone     -Protocol is as follows:         -12.5ul Q5 Master Mix         -1.25ul F and R primers each         -1ul of template added         -Water added to 25ul     -5ul ladder, 2 ul loading dye added to each sample     -Q5 Polymerase used and Q5 2x Master-Mix polymerase protocol applied from online     -1% 50mL gel Created and ran at 110V     -No results for band 7/7/18 -Set up overnight cultures as per Cian’s instructions so that they could be miniprepped     -6 colonies chosen and labelled A-F     -Cian’s Overnight cultures protocol used (insert link) 7/8/18 -Mini Prepped Varun’s overnight cultures     -Concentrations ranged from 80ng/µL to 115ng/µL 7/9/18 -Prepared 100µM stocks of the Ben and Gib primers for alternates to the ArgArs     -Ran PCR on pOW1 using these Primers     -Used Q5 polymerase as per online protocol         -12.5ul Q5 Master Mix         -1.25ul F and R primers each         -1ul of template added         -Water added to 25ul -Ran diagnostic digest of pBSC13mut w/ Hpa1     -To 10ul rather than to 50ul         -1µg of DNA added to each sample         -.1ul Hpa1 added         -1ul Cutsmart 10x Buffer         -Water filled to 10ul         -If greater than 10µl, no water added -Ran PCR of ArgArs sequence again     -PCR protocol used: Igem ArgArs     -5ul ladder GoGreen, 2 ul loading dye (Ran out of Go Green, new 1Kb ladder used     -1% 50 mL gel Created & ran at 120V     -This time, 2.55kb band did form         -However, I accidentally discarded the gel itself and thus must repeat the PCR -Excised the Well 2 and Well 3 Bands from GE     -Ran Gel Purification     -Ended up with Well 2 concentration of 35.5ul and Well 3 concentration of 49.9ul 7/10/18 -25µL PCR product of arg-ars sequence from pOW1 plasmid - touch-down PCR     -15.6µL milliQ water     -5µL 5X buffer     -1µL template DNA (pOW1 plasmid)     -1.3µL of F2     -1.3 µL of R1     -0.5µL dNTP     -0.3µL Phusion     -iGEM arg-ars PCR cycle -Diagnostic digest of pBSC13mut with Hpa; 10µL product     -8.8 µL milliQ water     -0.1µL cutsmart buffer     -1 µL of pBSC13mut samples A-F     -0.1µL Hpa1     -Incubated for 1 hour at 37ºC -Gel electrophoresis of diagnostic digests run at 110V for 30 minutes     -0.8% agarose gel     -Well 1: Quickload 1 Kb+ ladder     -Well 2: pBSC13mut A     -Well 3: pBSC13mut B     -Well 4: pBSC13mut C     -Well 5: pBSC13mut D     -Well 6: pBSC13mut E     -Well 7:  pBSC13mut F -Gel electrophoresis of pOW1 arg-ars PCR product run at 110V for 12:10     -Well 1: Quickload 1 Kb+ ladder     -Well 2: pOW1 arg-ars PCR product with F2 and R1     -Well 3: pOW1 arg-ars PCR product with Ben Glick Primers         -Labelled S1 -25µL PCR product of Stage 2 and ArgArs from yesterday's products     -Recipe         -15.6µL milliQ water         -5µL 5X buffer         -1µL template DNA ArgArs product from yesterday (Stage 1 PCR Product)         -1.3µL of F2(FGib)         -1.3 µL of R1(RGib)         -0.5µL dNTP         -0.3µL Phusion     -Only 1 thermocycler was open, so created iGEM Combi Protocol to do both simultaneously         -First 5 cycles are a gradient from 56 to 65. Gib rxn run at the 56 end. ArgArs run at the 65 end         -Final 27 cycles run at 65 entire block (Reduce nonspecific Gib binding) -Diagnostic digest of all pSB1C3 mut stocks that I could find     -Found 9 stocks from 2017     -Did 10µL diagnostic digest with each as well as two controls         -Negative: pSB1C3 with HpaI (Expect no cut)         -Positive: pSB1C3 with EcoRI(Expect a cut) 7/11/18 -Ran Cian’s PCR’s on one large gel     -Large gel used, 0.8% agarose (120ml of SB Buffer)         -SB Buffer used to keep continuity with Glick Lab’s use of SB Buffer     -Well 1 Loaded with ladder 5ul     -Well 2 Loaded with Stock B 20ul     -Wells 3-4 Loaded with - and + Controls 20ul, respectively     -Wells 6-13 Loaded with Stocks 1-8 20ul     -Wells 14-17 Loaded with ArgArs amplified regular way 20ul     -Wells 18-21 Loaded with Stage 2 Alternate ArgArgs amplification     -Stocks 2,3,4, & 7 of pSB1C3mut resulted in bands of somewhat correct length         -These must be transformed     -Lots of streaking in wells 14-21, must be result of SB Buffer → Use TAE -Transformed PSB1C3 stocks 2,3,4,7 along with RFP- PSB1C3 from iGem parts and Competency cells test     -Used JM109 Bacteria instead of DH5A     -Process is same as listed above otherwise -Ran PCR on Simone’s Arg/Ars Sequence using Q5 Polymerase with exact same protocol as earlier trial which resulted in correct banding     -25ul reaction total, 5 reactions labeled 1-5     -12.5ul Q5 Master Mix     -1.25ul each of F1 and R1 Primers     -9ul template     -1ul H20     -0.8% 50ml Gel created to run DNA on (0.4g agarose)         -Once again, TAE buffer used 7/12/18 -Removed RFP-pSB1C3 stock + pSB1C3mut stocks 2, 3, 4, and 7 from 37ºC incubator at 9:15am -Made liquid overnight cultures of pSBC13mut 3     -Picked 6 colonies; labelled A-F     -Put in 37ºC incubator at 5:00pm -Began streak purification process of pSBC13mut sample RFP-pSB1C3 + pSB1C3mut stocks 2, 4, and 7     -Put in 37ºC incubator at 5:00pm -Stage 2 PCR (with Gib F and R primers); used GeneHackers cycle     -Annealing temperature of 57ºC     -Two 25µL reactions (labeled A and B) with Phusion         -12.6µL milliQ water         -5µL 5X buffer         -3µL template DNA ArgArs product from yesterday (Stage 1 PCR Product)         -1.3µL of F2 (FGib)         -1.3 µL of R1(RGib)         -0.5µL dNTP         -0.3µL Phusion     -2 reactions (labeled C and D) with Q5 (VP) -Ran gel of pOW1 arg-ars touchdown PCR product (56º – 70ºC) -Gel purification of pOW1 arg-ars touchdown PCR product     -Eppendorf tubes containing samples E and F are likely mixed together         -No gel fragment in eppendorf tube F after band excision     -1µL membrane binding solution per 0.001g of gel fragment excised -PCR of S1 Ben Alternate ArgArs Product     -Using Q5 Polymerase as follows         -12.5ul Q5 Master Mix         -1.25ul GibF and Gib R each         -4ul ArgArs PCR1 Product (Well 3)         -6ul H20         -Labelled C and D -PCR of ArgArs 2 protrocols     -ArgArs Programming (4rxm total)         -5µL Phusion HF buffer         -0.5 dNTP (New stock from NEB)         -1.3 µL F2         -1.3µL R1         -1µL pOW1 D (50X Dilution)         -15.6ul H20         -0.3µL Phusion Poly         -Assembled as part of a Master mix with Gradient     -Standard PCR Protocol with a gradient         -Above recipe         -Assembled as part of a Master Mix with above protocol         -Gradient from 56C to 70C     -Gel Results (Each PCR run on a seperate gel)         -All 12 reactions showed amplification of a band between 2Kb and 3Kb. Indicates correct product synthesized         -All gels showed banding below 500 bp. Likely a combination of mispriming and  Remaining primer as it is extremely faint.         -Overall impression: Desired product can be synthesized at a range of temperatures using standard protocols. Issues with cloning likely have been technique related or a problem with the dNTP stock as it was homemade. Using the new dNTP from NEB yielded correct results. -Gel Purification of all 12 fragments     -Purification of ArgArs         -Followed Promega protocol for Wizard PCR cleanup. Yields were less than 10 ng/µL and 260/230 was 0.03 avg. Discarded samples. Likely due to improper incubation with membrane binding solution (See reasoning below)     -Purification of Gradient products         -Done with RF and VP         -Followed directions according to promega protocol. Measured concentrations were 30-56 ng/µL with 260/230 of 0.46. Still not great, but might be able to improve on future runs. Suspect incubation with the membrane binding solution is too low and DNA does not have sufficient time to bind to column. Could also be a problem with adding the membrane binding solution as this was a problem in my lab with the Zymo Kit. May be able to be used for Gibson but will need to overshoot 7/13/18 -Removed overnight plates of streak purified RFP-pSB1C3 stock + pSB1C3mut stocks 2, 3, 4, and 7     -Only plates pSB1C3-2 and pSB1C3 4 had colonies         -Other plates had no observable growth -Mini-prepped overnight liquid bacterial cultures of pSB1C3mut-3     -Followed manufacturer's instructions for the PureYield Plasmid Miniprep Promega Kit     -Got less than ideal results when nano-dropped -Ran an ethanol precipitation protocol for the mini-prepped pSB1C3mut-3 DNA     -Didn’t work, no pellets seen in the eppendorf tubes -Made overnight cultures of pSBC13mut 2, 3, and 4 -Full digest of D and E     -32.2µL of nuclease free water     -5µL of buffer J     -12.3 µL of D and E     -0.5µL of Hpa1 -Placed overnight liquid culture tubes and plates in  37ºC incubator at 6:00pm -Made chloramphenicol plates     -380 µL of stock chloramphenicol     -JM109 bacteria used     -+ and - controls transformed set in incubator to be examined next day -Diagnostic digest of PSB1C3 D & E stocks     -10ul total     -Used 1ul of buffer J instead of Cutsmart this time     -Hpa1 1ul     -1ul template     -7.9ul DH20     -Ran on gel, came back with 1 band exactly as expected, means that full digest can be completed using these solutions 7/14/18 -Mini-prepped overnight cultures of pSBC13mut samples 2, 3, and 4     -2A, 2B, 2D, 3C, 3E, 3F, 4C, and 4D had appropriate nanodrop values -Ran a diagnostic digest of the above samples -Ran gel of diagnostic digest     -Well 1: 1 Kb ladder     -Well 2: 2A     -Well 3: 2B     -Well 4: 2D     -Well 5: 3C     -Well 6: 3F     -Well 7: 3E     -Well 8: 4C     -Well 9: 4D -Diagnostic digest of overnight mini-prepped overnight cultures     -One band that we wanted and several faint bands below -Performed Ethanol Precipitation on two samples of the ArgArs PCR product     -48µL of Impure DNA     -4.8µL of 3M KOAc     -105.9µL of 200 proof EtOH     -Stored at -20C for 20 min     -Spun at 13.3 rpm for 15 min     -No pellet formed. Discarded samples -Did a spin cleanup of two different samples of the ArgArs PCR product     -Concentration was upped to 100ng/µL, but the contamination still persisted -Examined + and - control plates from yesterday, determined to be usable for future plating 7/16/18 -PCR of ArgArs (4rxn)     -Recipe         -15.6µL of Water         -5µL of Phusion HF Buffer         -1.3µL of F2 and R1 (2.6 total)         -1µL pOW1-d (50X dilution)         -0.5µL dNTPs         -0.3µL Phusion     -Protocol         -Genehackers standard PCR             -Annealing 67C             -32 cycles -Gel imaging of diagnostic digest showed clear, bright bands around 2.55Kb (the desired size)     -A few bands further along in the gel were also observed, but were very faint -Full digest of 2A, 2B, 2D, 3C, 3E, 3F, 4C, and 4D     -37µL of nuclease free water     -5µL of buffer J     -7.5µL of template DNA         -Nanodrop measured the concentrations to all be around 200 ng/µL     -0.5µL of Hpa1     -Placed in the 37ºC incubator for three hours -Ran gel of full digest products     -Except for 4D, the tube was not in the incubator     -110V for 30 minutes     -Well 1: 1 Kb ladder     -Well 2: 2A     -Well 3: 2B     -Well 4: 2D     -Well 5: 3C     -Well 6: 3E     -Well 7: 3F     -Well 8: 4C -Forgot to dephosphorylate and run a spin-cleanup before running the full digest gel     -The desired band was observed when the gel was imaged, but there were fainter bands below as well -Ran another full digest (backup)     -37µL of nuclease free water     -5µL of buffer J     -7.5µL of template DNA         -Nanodrop measured the concentrations to all be around 200 ng/µL     -0.5µL of Hpa1     -Placed in the 37ºC incubator for three hours -Excised band from first full digest and gel purified         -The nanodrop measured the purities of the  DNA samples to be too low (around 0.30)             -Proceeding with the backup full digest which is incubating at 37ºC -Spin cleanup of full digests -Made .8% SB Buffer Agarose gel 50ml     -Ran Cian’s ArgArs PCR product on gel in order to purify     -Ran @ 110V for 35 minutes         -Lane 1 Buffer 5ul         -Lane 2-5 PCR Product 1-4 20ul         -Was taken out after 20 min, imaged, and placed back in b/c was not ready the first time         Lane 5 had 2 bright bands             -2nd bright band was the right size -Purified 2nd band from PCR using Cian’s gel purification instructions -Made .8% SB Buffer Agarose gel 50ml for Rachael 7/17/18 -PCR of ArgArs (5rxn)     -Recipe (Assembled as a Master Mix)         -15.6µL of Water         -5µL of Phusion HF Buffer         -1.3µL of F2 and R1 (2.6 total)         -1µL pOW1-d (50X dilution)         -0.5µL dNTPs         -0.3µL Phusion     -Protocol         -Genehackers standard PCR             -Annealing 68C             -32 cycles -Dephosphorylation of the first full digests from 7/16/18     -Placed in a PCR tube     -dH20 to 35µL (4.75µL)     -3.5µL Cutsmart     -25µL of the backbone     -1.75µL of the Quick CIP     -Incubated at 37ºC for 10 minutes     -Heat inactivated in thermocycler at 80ºC for 2 minutes -Ran gel of first full digests and arg-ars PCR products; 0.8% agarose gel     -Added 7µL loading dye to digest products     -Added 5µL loading dye to arg-ars PCR products     -Run at 110 V for 40 minutes -Excised arg-ars and full digest bands and gel purified     -Except for 2D (too faint)     -Followed Jason’s protocol and used the QIAquick gel extraction kit -Measured samples at nanodrop     -Low purities and yields -The full digests used were from the first set made on 7/16/18 and had purities which were too low     -Should’ve used the tubes containing  the digests from 7/16/18 which had undergone spin-cleanup -Dephosphorylation and gel purification of second set of full digests made on 7/16/18 -Made overnight cultures of RFP     -Placed in 37ºC shaker incubator at 4:00pm -PCR amplified arg-ars sequences using master mix (4 reactions)     -CC prepared -PCR of ArgArs (4rxn)     -Recipe (Assembled as a Master Mix)         -15.6µL of Water         -5µL of Phusion HF Buffer         -1.3µL of F2 and R1 (2.6 total)         -1µL pOW1-d (50X dilution)         -0.5µL dNTPs         -0.3µL Phusion     -Protocol         -Genehackers standard PCR             -Annealing 68C             -32 cycles     -Gel Purification Performed by VP         -Did not work -PCR of ArgArs (4rxn+ 1 mini rxn with leftovers) Afternoon     -Recipe (Assembled as a Master Mix)         -15.6µL of Water         -5µL of Phusion HF Buffer         -1.3µL of F2 and R1 (2.6 total)         -1µL pOW1-d (50X dilution)         -0.5µL dNTPs         -0.3µL Phusion     -Protocol         -Genehackers standard PCR             -Annealing 68C             -32 cycles     -Gel Purification Performed by CC using Qiagen Kit         -Did an addition 500µL wash because forgot to let first wash incubate for 2-5 min. Second wash incubated for 2 min         -Samples 2,3,4 were all within acceptable values (260/230 0.92,1.39, and 1.02 respectively. Concentrations 20-35ng/µL) 7/18/18 -Made full digests and placed in 37ºC incubator for three hours     -Used NEB reagents (not Promega), no need to spin-cleanup -Removed overnight cultures of pSB1C3mut-RFP from 37ºC incubator and mini-prepped     -Used Promega Pureyield kit with manufacturer's instructions         -Including alternative protocol for larger culture volumes     -Got good data; high concentrations (ranging from 95ng/µL to 344ng/µL and centered around 200 ng/µL) with good purities (260/230 values ranging from 1.3-2.1) -Made standard concentrations of pSB1C3mut-RFP stock using miniprep sample A (95.9ng/µL)     -100 pg/µL: 0.2 µL of pSB1C3mut-RFP A, 191.6 µL milliQ water     -50 pg/µL: 50 µL of 100 pg/µL standard concentration, 50 µL milliQ water     -25 pg/µL: 50 µL of 50 pg/µL standard concentration, 50 µL milliQ water     -Placed in – 20ºC freezer below benchtop -Autoclave Eppendorf tubes -Dephosphorylated full digest products and ran on a gel with arg-ars PCR product from 7/17/18 at 110V for 30 minutes     -Well 1: 1 Kb ladder     -Well 2: arg-ars PCR product 1     -Well 3: arg-ars PCR product 2     -Well 4: arg-ars PCR product 3     -Well 5: arg-ars PCR product 4     -Well 6: 1 Kb ladder     -Well 7: full digest 2B     -Well 8: full digest 2D     -Well 9: full digest 4C -Excised and gel purified arg-ars PCR product bands and pSB1C3mut full digest bands from gel     -Used QIAquick kit and manufacturer's protocol     -No bands for the full digest products     -Only half the arg-ars PCR product volume used         -Running another gel to gel purify the other half, will mix the two gel purification products together -Made overnight cultures of pSB1C3mut 2, 3, and  4     -Placed in 37ºC incubator at 5:00pm -Ran Gel of the 2nd round of PCR products     Well 1: 1 Kb ladder     -Well 2-5: arg-ars PCR products 1-4 -Excise and gel purified arg-ars PCR product bands     -Use QIAquick kit and manufacturer’s protocol     -All 4 samples produced bright bands c-After gel purification, however, did not return good results         -Concentrations were ~30ng/ul and 260/230s were around .05 7/19/18 -Made LB back in Glick lab     -Followed Jason’s Introductory Protocol     -Mixed 5.0g NaCl, 5.0g Tryptone, and 2.5g Yeast Extract with 250ml water and autoclaved on liquid setting for 15 minutes -Ran Restriction Digests of Hpa1 with Miniprepped pSB1C3mut 2,3, and 4     -50ul total reactions-Marked 2A, 3A-F, 4A-D     -32.5ul H20     -12ul Template     -5ul NEB Cutsmart Buffer     -.5ul NEB Hpa1 -Dephosphorylated Restriction Digest     -Added all 50ul of digest to 1ul of QuickCIP and incubated at 37 for 10 minutes followed by heat killing phosphatase at 80 for 2 minutes -Mini-prepped overnight cultures pSB1C3mut 2, 3, and 4     -Used promega manufacturer’s directions     -Nanodrop showed relatively low DNA concentrations (around 100ng/µL), but high purities (with 260/230 values around 1.5) -Autoclaved p1000 tips -Made full digests and incubated at 37ºC for three hours -Dephosphorylated full digests     -Transferred to eppendorf tubes and added 1µL of QuickCIP per manufacturer’s directions -Gel electrophoresis of full digests; large 0.8% agarose gel; run at 150V for 30 minutes     -Well 1: 1 Kb ladder     -Well 2: full digest 2A     -Well 3: full digest 3A     -Well 4: full digest 3B     -Well 5: full digest 3C     -Well 6: full digest 3D     -Well 7: full digest 3E     -Well 8: full digest 3F     -Well 9: full digest 4A     -Well 10: full digest 4B     -Well 11: full digest 4C     -Well 12: full digest 4D -Gel purified bands excised from gel loaded with full digests     -Only two of the bands, 3D and 3F, had purities and concentrations  suitable for Gibson assembly and transformation         -Attempted to salvage the other samples by eluting again, but this additional step did not produce acceptable nanodrop results -Gibson assembly of 3D and 3F     -Used 2X Gibson Assembly Master Mix located in –20ºC freezer below benchtop     -3F (10 µL reaction)         -3.6 µL of backbone full digest (from pSB1C3 – 3)         -1 µL of arg-ars PCR product         -5 µL of Gibson master mix         -0.4µL of dH2O     -3D (5 µL reaction)         -1.1 µL of backbone full digest (from pSB1C3 – 3)         -0.3 µL of arg-ars PCR product         -2.5 µL of Gibson master mix         -1.1µL of dH2O     -Incubated at 50ºC for 15 minutes in a heat block -Transformation of full digests and arg-ars sequence     -Added entire volume of both gibson assembly reactions to JM109 competent E. Coli cells         -Transformation reaction A used 3F (L1A)         -Transformation reaction B used 3D (L1B)     -Incubated on ice for 30 minutes     -Placed in 42ºC  water  bath for 45 seconds (heat shock)     -Recovery on ice for 2 minutes     -Added 300 µL SOC medium     -Placed in the 37ºC shaker for 1 hour     -Plated 50µL of each transformation reaction onto an LB + CAM 7/20/18 -After removing plates from 37ºC incubator we observed colonies     -Pending sequencing, we tentatively say we’ve created the plasmid pSD000 -Made liquid overnight cultures of potential pSD000 plasmid (L1A and L1B)     -Using LB (not LB + CAM) media that VP made 7/19/18     -Placed in 37ºC incubator at 5:00pm -Made liquid overnight culture of yeast     Used frozen glycerol stock in –80ºC freezer     -Dipped a sterile streaking pipette tip into glycerol stock and placed in 2 mL of YPAUD liquid media     -Placed in 37ºC incubator at 5:00pm -Diluted genomic pichia DNA stock 100x     -Concentration went from 13356 ng/ul to 77.5 ng/ul -Ran PCR of centromeric regions     -Used 3.CENF & R Primers     -32.5 µL milliQ water     -10µL of 5x Phusion HF Buffer     -1µL 10 µM dNTPs     -2.5µL of 3. CEN Forward Primer     -2.5µL of 3. CEN Reverse Primer     -1µL of Template DNA         -Genomic DNA     -0.5µL of Phusion Polymerase     -Annealing temperature set to 55ºC, that of CEN Reverse Primer -Miniprep of pSD000 overnights     -Used the Promega kit     -Eluted with 30µL to increase concentration     -All 12 samples were in acceptable purity tolerances -Diagnostic digests of pSD000 to determine which ones to be sequenced     -Both Prepared as master mixes with all 12 samples + pSB1C3 mut control     -Single Digest to Determine size         -14 Cut smart         -14 Temp         -7 XbaI         105 dH2O         -Results             -All Samples Except A5 returned control fragment size                 -Estimated to be 3-5Kb which is expected size             -A4 Has control fragment as brightest band but did have a band consistent with A5     -Double digest to determine correct Insert         -14 Cut         14 Temp         -1.4 AgeI (Cuts Only pSD000)         -1.4 EcoRI (Cuts control and pSD000)         -98 dH2O         -Results             -All fragments except A5 returned control fragment                     -A5 returned an 800 kb fragment and 3-4 kB fragment. Consistent with pSD000                 -Sample A4 had faint banding corresponding to A5, but had majority control banding     -Overall results         -Sample A5 is likely pSD000 due to banding patterns. Should be submitted for sequencing         -Sample A4 may have a small amount of pSD000. Could be purified with a yeast mini prepped if confirmed         -Overall, most samples returned parent plasmid. This indicates dephosphorylation failed either because of poor technique or poor enzyme             -Told Rachael and Varun to run some tests to figure out which -Submitted samples for sequencing     -Submitted A5 and A4 for 10 rxn sequencing. Gave the following primers         -Seq.F1         -Seq.F2         -Seq.F3         -Seq.F4         -Seq.F1         -Seq.F2         -Seq.F3         -Seq.F4         -VR (To confirm presence of XbaI site for RFP insert)         -VF2 (To confirm presence of XbaI site for RFP insert) 7/23/18 -Set up PCR of Centromeric Regions of PP gDNA     -Paired each F and R primer with each other         -1. cenF+R         -1.cenF2+R2         -2.cenF+R         -2.invF+R         -2.cenF2+R2         -3.cenF+R         -3.invF+R         -3.cenF2+R2         -Total of 8 reactions, 25ul each     -16.25ul H20     -5ul Phusion     -.5ul DNTP’s     -1.25ul of each primer F and R     -.5ul gDNA         -Diluted 77.5ng/ul stock made on Friday     -.25ul Phusion     -Ran at Gradient PCR from 52->62 -Ran PCR products on a Gel     -However, bands were much smaller than the 2-4kb expected     -Could have been issue with genomic DNA dilution or polymerase     -Must re-Do -Ran full digest of RFP pSB1C3     -Used Xba1 and Spe1 to 50ul, 4 samples     -38ul water     -5ul Cutsmart     -5ul template (211 ng/ul)     -1ul NEB Xba1     -1ul NEB Spe1-HF -Transformed pSD000 A4 and A5 into JM109 competent cells     -Added 1µL of A4 and A5 to JM109 competent cells     -Incubated competent cells on ice for 30 minutes         -Tapped gently every 10 min to mix     -Heat shocked cells by placing into 42ºC water bath for 45 seconds     -Placed cells on ice for 2 min recovery     -Added 300 µL SOC media to rescue cells         -Place in shaking 37ºC incubator for 1 hour     -Plated onto LB + CAM plates         -Pipetted and spread 50µL of the transformed JM109 onto the plate             -Placed the remaining cells in the 4ºC fridge     -Placed plates in the 37ºC incubator at 5:00pm -[VP] Ran full digest of pSB1C3-RFP with Xba1 and Sre1 (4 reactions) -Dephosphorylated 2 of the reactions     -Reaction 1 and reaction 2 (only phosphorylated half of the volume: 25µL)         -Stored reactions 3 and 4 in the fridge     -dH20 to 35µL (4.75µL)     -3.5µL Cutsmart     -25µL of the backbone     -1.75µL of the Quick CIP     -Incubated at 37ºC for 10 minutes     -Heat killed by placing in heat block at 80ºC for 2 minutes -Ran all reactions on a 0.8% agarose gel at 120V for 25 minutes     -Well 1: 1 Kb ladder     -Well 2: full digest 1 – not dephosphorylated     -Well 3: full digest 2 – not dephosphorylated     -Well 4: full digest 1 – dephosphorylated     -Well 5: full digest 2 – dephosphorylated     -Neither of the full digest 1 samples (regardless of whether or not it had been dephosphorylated) produced any bands         -Both full digest 2 samples (dephosphorylated and not) produced two bands             -The larger band is the backbone and the smaller band below it is RFP -Gel purified excised bands     -Excised the RFP bands from the lanes with full digests that had not been dephosphorylated     -Excised the backbone bands from the lanes with dephosphorylated full digests 7/24/18 -Removed plates from 37ºC incubator     -A4 plate grew a lawn         -Did not streak purify because we’re not entirely sure we need it yet     -A5 did grow colonies, but very few can be isolated         -Streak purified this plate, this is the sample most likely to be pSD000     -Placed streak purified LB+CAM plate with pSD000 A5 into 37ºC incubator at 6:00pm -PCR amplified centromeric region 1 using 1.cen.F and 1.cen.R (25µL reaction)     -15.75 µl H20     -0.5 µl DNTPs     -1.75µl of each primer F and R     -1µl gDNA         -Used 77.5ng/ul stock made on 7/20/18     -0.25µL Phusion     -Annealing temperature set at 60ºC -PCR amplified centromeric region 1 using 1.cen.F and 1.cen.R     -13.75ul H20     -5ul Phusion     -.5ul DNTPs     -1.25ul of each primer F and R     -1ul gDNA         -Diluted 250.0 ng/ul stock made     -.25ul Phusion -Ran full digest of RFP pSB1C3     -Used Xba1 and Spe1 to 50ul, 4 samples     -38ul water     -5ul Cutsmart     -5ul template (211 ng/ul)     -1ul NEB Xba1     -1ul NEB Spe1-HF -Dephosphorylated two of the samples along with 1 sample from yesterday     -Dephosphorylated 3 of the reactions         -Reaction 1 and reaction 2 (only phosphorylated half of the volume: 25µL)             -Stored reactions 3 and 4 in the fridge (extra sample used today)         -dH20 to 35µL (4.75µL)         -3.5µL Cutsmart         -25µL of the backbone         -1.75µL of the Quick CIP         -Incubated at 37ºC for 10 minutes         -Heat killed by placing in heat block at 80ºC for 2 minutes -Ran gel of Dephosphorylation products + PCR Products from earlier today     -.5% Large 120ml Agarose gel with 2 rows of wells         -Top Row:             -Well 1: Ladder 5ul             -Well 2,3,4,5,6: PCR Products V1-3 and C1,2 50ul             -Well 7: Ladder 5ul             -Well 8,9,10,11: PCR Products C3, R4-6 50ul             -Well 12: Ladder 5ul         -Bottom Row:             -Well 1: Ladder 5ul             -Well 2,3,4: Dephosphorylated Backbone+RFP 1-3 50ul             -Well 5,6: Non-Dephosphorylated Backbone+RFP 4,5 50ul             -Well 7: Ladder 5ul             -Well 8: Non-Dephosphorylated Backbone+RFP 6 50ul 7/25/18 -Made six liquid overnight cultures of pSD000-A5     -Placed in 37ºC incubator at 5:00pm -Full digest of pSB1C3-RFP with Xba1 and Spe1     -38µl water     -5µl Cutsmart buffer     -5µl pSB1C3-RFP C template (257.0 ng/ul)     -1µl NEB Xba1     -1µl NEB Spe1-HF     -Placed in 37ºC incubator for 3 hours -Made liquid LB media -Made LB + CAM plates     -Added 250µL of chloramphenicol stock     -Made a positive and negative control using RFP -Dephosphorylated full digest reactions 1 and 2     -Will extract only the backbone -Ran full digests on a gel at 120V for 30 minutes     -Well 1: 1 Kb ladder     -Well 2: full digest reaction 1 (dephosphorylated)     -Well 3: full digest reaction 2 (dephosphorylated)     -Well 4: full digest reaction 3     -Well 5: full digest reaction 4 -When imaging, the bands corresponding the dephosphorylated samples were too faint to excise -Made overnight cultures of yeast     -2 culture tubes with 12 mL of YPAUD media for each of the following:         -pOW1         -pSD000         -Blank control (just yeast, no foreign DNA)     -12 mL of YPAUD liquid media in each culture tube         -Added PPY12 yeast (A4 in Glick’s system)     -Placed in the 30ºC incubator at 5:00pm -PCR amplified centromeric region 3 using 3.cen.F and 3.cen.R     -13.75ul H20     -5ul Phusion     -.5ul DNTPs     -1.25ul of each primer F and R     -1ul gDNA         -Diluted 250.0 ng/ul stock made     -.25ul Phusion 7/26/18 -Positive and negative controls for LB + CAM plates made on 7/25/18 both produced colonies     -Not enough chloramphenicol added -Made chloramphenicol stock     -10 mL of ethanol, 0.50g of ethanol-soluble chloramphenicol     -Divided evenly into 10 eppendorf tubes labelled CAM and placed in antibiotic stock freezer box     -50 mg/mL -Made overnight cultures of yeast     -The cultures made on 7/25/18 did not have enough cells (solution was still clear)         -Possible that too much liquid was added to the culture tubes, the cells were not able to properly aerate     -25 mL of YPAUD liquid media in each 250 mL flask         -1 flask for each transformation             -pOW1             -pSD000             -Blank control (just yeast, no foreign DNA)     -Added PPY12 yeast (located in box A4 in Glick’s system)     -Placed in the 30ºC incubator at 5:00pm -Made a positive and negative controls for LB and  LB + CAM plates by transforming RFP into JM109 E. Coli competent cells     -Plated onto LB and LB + CAM plates     -Placed in 37ºC incubator at 7:30pm -Made LB plates     -5 g tryptone     -5 g NaCl     -2.5 g yeast extract     -10 g agar -Made LB + CAM plates     -5 g tryptone     -5 g NaCl     -2.5 g yeast extract     -10 g agar     -Added 380 µL of 50 mg/mL chloramphenicol stock -Set up PCR of all 15 possible primer combinations for centromeric sequences     -Made Master Mix:         -250ul DH20         -80ul Phusion Buffer         -8ul DNTP’s         -16 ul 250ng/ul diluted gDNA     -22.4ul of Master Mix in each tube 1-15     -1.3ul Primer F and R in each, total of 25 ul     -Ran a Gradient PCR in Thermocycler with 3:30 elongation time @ annealing temperatures from 57-60C -Ran a + and - Control of Plates Rachael Made 7/27/18 -Positive and negative control plates for LB + CAM plates made on 7/26/18     -Both grew on the LB plates as there were no antibiotics on those plates         -No need to test these plates in the future     -Both the positive and negative control plates grew colonies         -The colonies were very small however, and could only be seen when the plates were held to the light             -Since the colonies were so small and hard to see, we are going to re-transform JM109 competent cells with RFP                 -Re-testing the LB + CAM plates -Placed positive and negative control plates in 37ºC incubator at 6:30pm -Transformed pOW1, pSD000, and a blank control into pichia pastoris     -Removed yeast overnights at 9:30am and measured ODs using spectrometer         -All ODs were within 0.5-1, can proceed without diluting     -Added 500 µL of 1 M Na Hepes and 500 µL 1M DTT     -Placed yeast cultures back in 30ºC shaking incubator for 15 minutes     -Spun down cells in a 50 ml conical tube for 5 minutes at 3K rpm         -Discarded supernatant fluid     -Resuspended cells in 25 mL sterile cold water     -Spun down cells at 3K rpm for 5 minutes         -Discarded supernatant fluid     -Resuspended cells in 20 mL sterile cold 1M sorbitol     -Spun down cells at 3K rpm for 5 minutes         -Discarded supernatant fluid     -Resuspended cells in 100 µL of 1M cold sorbitol (pH = 7.5)     -Added 50 µL of the now competent cells to 100-250 ng of transforming DNA and transferred to an electroporation cuvette         -Pulsed at 1.5kV     -Added 1 mL of cold 1 M sorbitol immediately after pulsing     -Spun down cells for 1 minute at 5K rpm         -Removed the upper 800µL of sorbitol using a p1000 micropipette     -Plated transformed cells onto YPAUD-Arg plates         -Placed into 30ºC incubator at 6:30 pm -Made 1:10 dilutions of 30-mer primer stocks         -60-mer primers labeled in red         -30-mer primers labeled in blue -Ran 0.5% agarose gel of PCR products CC made 7/27/18     -Run at 140 V for 20 minutes     -Top row         -Well 1: 1 Kb ladder         -Well 2: reaction 1.1 (reaction 1)         -Well 3: reaction 1.2 (reaction 2)         -Well 4: reaction 1.3 (reaction 3)         -Well 5: reaction 1.4  (reaction 4)         -Well 6: reaction 1.5 (reaction 5)         -Well 7: 1 Kb ladder         -Well 8: reaction 2.1 (reaction 6)         -Well 9: reaction 2.2 (reaction 7)         -Well 10: reaction 2.3 (reaction 8)         -Well 11: reaction 2.4 (reaction 9)         -Well 12:  reaction 2.5 (reaction 10)     -Bottom row         -Well 1: 1 Kb ladder         -Well 2: reaction 3.1 (reaction 11)         -Well 3: 1 Kb ladder         -Well 4: reaction 3.2 (reaction 12)         -Well 5: reaction 3.3 (reaction 13)         -Well 6: reaction 3.4 (reaction 14)         -Well 7: reaction 3.5 (reaction 15) -Re-did + and - Control Transformations of CAM plates using JM109     -RFP Used in + Control, H20 Used in Negative Control -Set up PCR of all centromeric sequences >4kb long     -Diluted 30-mer primer stocks to 1:10 @100ul     -Ran primer combinations 2,3,4,6,7,8,9,10,13,14 (See picture below)     -Made Master Mix for 10 Reactions:         -60ul Phusion Buffer         -12ul 250ng/ul diluted gDNA         -6ul DNTP’s         -210ul DH20     -22.1ul MM in each tube     -1.3ul each F and R primers     -.3ul Phusion     -Ran gradient PCR based on lowest annealing temp for each rxn     -Used 3:00 elongation time (45sec/kb) -Gel Purified Cian’s PCR sample     -Wells 1.4,2.2,2.3,2.4,3.2, and 3.3 produced bands of correct size     -Followed Qiagen’s Purification Procedure with following changes:         -Did not add isopropanol         -Upon Addition of Buffer PE, let columns sit for 5 minutes         -After first PE centrifuge, add .5ml of PE and incubate for 2 minutes 7/30/18 -Checked control plates for LB + CAM plates made 7/26/18         -Checked plates on 7/28/18, but neither the positive nor negative plates had colonies         -When checking the plates today, both plates had colonies             -Likely contamination             -Perhaps the concentration of chloramphenicol was too high -Checked yeast transformation plates made on 7/27/18     -Checked plates on 7/28/18, but none of the three plates (transformed with pOW1, pSD000, and a blank control) had colonies     -When checking plates today, all of the plates had colonies         -The plates probably were not good since the control grew colonies as well -Made a master mix for the PCR reactions of all 15 centromeric sequences     -250µl dH2O     -80µl 5X Phusion buffer     -8µl DNTPs mix     -16 µl 250 ng/ul diluted gDNA     -22.1 µL of master mix in each PCR tube         -1.3 µL of each primer         -0.3µl Phusion polymerase     -Ran a Gradient PCR of centromeric sequences         -Annealing temperatures range from 57-60ºC         -Samples 1.5 and 3.3 opened in PCR and did not produce products -Made yeast overnight cultures for transformations     -25 mL YPAUD liquid media     -Pichia Pastoris yeast located in the –80ºC freezer         -A4 in Glick lab’s system     -Placed in 30ºC shaking incubator at 5:00pm -Made SD-Arg plates     -It’s not possible to make YPAUD-Arg plates         -Likely mislabelled previous plates that were actually SD-Arg plates     -Used CSM-Arg dropout to make media     -Made positive and negative control plates; placed in 37ºC incubator at 5:00pm -Made more LB+CAM plates     -Used Jason’s introductory protocol for 500ml of LB     -Added 250 µL of 50mg/mL chloramphenicol stock -Ran Gel of PCR Products from Friday     -Large gel 120ml .5% with 2 rows     -Well 1: 10ul 1kb+ ladder     -Well 2,3,4,5,6: 30ul Product 2,3,4,6,7     -Well 7: Excess Product 7 (Accident)     -Well 8,9,10,11: 30ul Product 8,9,10,13     -Well 12: 10ul 1kb+ ladder     -Bottom Row:         -Well 1: 10ul 1kb+ ladder         -Well 2: 30ul Product 14     -Ran at 125V for 25 minutes -Gel Purified PCR Products 1.4 and 2.4.     -However, when nanodropped, there was a very low purity 7/31/18 -Removed positive and negative controls for LB + CAM plates made 7/30/18 at 10:00am     -Growth was seen on both plates, but not the characteristic distinct, red colonies we expect from RFP     -Going to test again, but with new transformations         -Perhaps something was wrong with the pSB1C3-RFP sample we used -Transformed pOW1, pSD000, and a blank control into pichia pastoris     -Removed yeast overnights from 30ºC shaking incubator at 10:00am     -Added 500 µL of 1 M Na Hepes and 500 µL 1M DTT     -Placed yeast cultures back in 30ºC shaking incubator for 15 minutes     -Spun down cells in a 50 ml conical tube for 5 minutes at 3K rpm         -Discarded supernatant fluid     -Resuspended cells in 25 mL sterile cold water     -Spun down cells at 3K rpm for 5 minutes         -Discarded supernatant fluid     -Resuspended cells in 20 mL sterile cold 1M sorbitol     -Spun down cells at 3K rpm for 5 minutes         -Discarded supernatant fluid     -Resuspended cells in 100 µL of 1M cold sorbitol (pH = 7.5)     -Added 50 µL of the now competent cells to 100-250 ng of transforming DNA and transferred to an electroporation cuvette         -Pulsed at 1.5kV     -Added 1 mL of cold 1 M sorbitol immediately after pulsing     -Spun down cells for 1 minute at 5K rpm         -Removed the upper 800µL of sorbitol using a p1000 micropipette     -Plated transformed cells onto SD-Arg plates         -Placed into 30ºC incubator at 5:00 pm -Transformed and plates positive and negative controls for LB + CAM plates made 7/30/18     -Used JM109 competent cells     -Placed in the 37ºC incubator at 5:00pm -Gel purified PCR products     -Used 30-mer primers     -Followed QiaQuick Gel Extraction Kit and corresponding protocol         -Performed two washes with PE buffer             -1st wash: 750µL of PE buffer, let sit for 5 minutes             -2nd wash: 500µL of PE buffer, let sit for 2 minutes     -Used Glick lab’s Qiagen kit to see if our bad gel extraction results are a kit issue     -Purities were once again unsatisfactory         -Not a kit issue, most likely a technique issue -Added Phusion to 30-mer PCR and ran PCR at annealing temp of 60 w/ 3:00 elongation time -Plated Pichia transformations on SD-Arg plates     -3 plates made:         -1 Basic Control         -1 transformed w/ pOW1         -1 transformed w/ pSD000 -Made .8% Agarose small gel 50ml     -Ran PCR Products on gel:         -Well 1:1kb+ ladder 5ul         -Well 2,3,4: PCR Product 1.2,1.3,1.4 12.5ul         -Well 5,6,7,8,9: PCR Products 2.1-2.5 12.5ul         -Well 10: PCR Product 3.4 12.5ul         -Ran at 110V for 20 minutes -Set up PCR of all 15 centromeric sequences using 30-mer primers     -Made Master Mix:         -240ul H20         -80ul Phusion Buffer         -16 ul gDNA         -8ul DNTPs         -Added 22.1ul of MM to each well     -1.3ul of F and R each to all the wells     -Waited to add Phusion until tomorrow b/c not much Phusion left         -Left in 4C fridge
August
8/1/18 -Made LB + CAM plates     -Positive and negative controls for LB + CAM plates made 7/30/18 both grew colonies -Yeast transformations     -All plates grew colonies     -Pichia transformed with pSD000
    -Pichia transformed with pOW1
    -Pichia transformed with a blank control (water)
-Made LB + CAM plates     -Followed Glick lab’s protocol     -Added 250 µL of 50 mg/ml chloramphenicol stock 8/2/18 -Set up PCR of all 15 centromeric sequences using 60-mer primers     -Made Master Mix:         -240ul H20         -80ul Phusion Buffer         -16 ul gDNA         -8ul DNTPs         -Added 22.1ul of MM to each well         -0.3 Phusion/rxn     -1.3ul of F and R each to all the wells     -Gel results         -Fragment 2.4 was only fragment that worked. Attempted to gel purify             -Was unsuccessful -Set up PCR of all 15 centromeric sequences using 60-mer primers again     -Made Master Mix:         -240ul H20         -80ul Phusion Buffer         -16 ul gDNA         -8ul DNTPs         -Added 22.1ul of MM to each well         -0.3 Phusion/rxn     -1.3ul of F and R each to all the wells     -Gel results         -Fragments 1.4,2.2,3.1, and 3.2 were all successful. Attempted to gel purify             -Unsuccessful                 -DNA was still present in the QG solution when compared to blank gel dissolved in Agarose. Suspect it is an issue with the columns. Will try Glick columns tomorrow -Setup PCR of 2.2 in attempt to diagnose the problem with gel extraction     -Mix         -15.6µL dH20         -5 Phusion HF         -1 gDNA         -1.3 Cen.F, inv.R         -0.5 dNTP         -0.3Phusion     -Will run gel tomorrow -Pour LB+Cam plates with new aliquots of 50mg/mL CAM     -Note: When poured, the culture tube was still covered with white powder. Suspect plates might not work     -Controls         -Negative JM109         -Positive DH5alpha with RFP (Restreaked form previous plate) -Liquid LB was horribly contaminated. Made new liquid LB -Made overnights of pOW1-ymRFP with Lb+ 2.1µL of Amp 8/3/18 -Set up PCR of 1.4, 2.2     -Made Master Mix:         -15.6µL dH20         -5 Phusion HF         -1 gDNA         -1.3 F, R         -0.5 dNTP         -0.3Phusion     -1.3ul of F and R each to all the wells     -Gel results         -1.4 Appeared and is bright.         -2.2 was faint but did appear         -Attempts to gel purify both fragments were unsuccessful             -Used Jason’s Qiagen kit and columns to test if kit is not working                 -Unsure what the issue is at this point -Received sequencing back on pOW1     -SCARG4 mutation is present in parent plasmid         -pSD000 Marked as accepted -Miniprep of pOW1-RFP     -Outcome Successful 8/4/18 -Made new CAM Stocks     -10mL in green falcon tubes     -.500g CAM     -Vortexed -Made LB+CAM plates with new CAM stocks     -270µL CAM/500mL -Gel of 2.2 from last night     -Gel indicated nonspecific binding issue. Did not attempt to gel purify -Set up PCR of RFP fragment     -Recipe:         -15.6µL dH20         -5 Phusion HF         -1 pOW1-ymRFP (20X dilution)         -1.3 rfp.F, rfp.R         -0.5 dNTP         -0.3Phusion     -Gel results         -RFP Fragment was produced brightly             -Gel Purification was successful                 -Used Zymo Gel Extraction Kit from Lee Lab following manufacturer's instructions with following modifications                     -After second wash step, spin for 5 min instead of 1                     -Elution in 8µL instead of minimum 6µL -Miniprep of pOW1-ymRFP -Plated NEB5 (Dh5Alpha) on LB plate -PCR of RFP     -Made Master Mix:         -15.6µL dH20         -5 Phusion HF         -1 pOW1-ymRFP (20X dilution)         -1.3 rfp.F, rfp.R         -0.5 dNTP         -0.3Phusion     -Gel ran 8/5/18 8/5/18 -Plates did not work. Both positive and negative controls had growth. Negative control did have stunted growth though     -Troubleshooting         -Preparation of the CAM stocks yielded about 600µL extra liquid. Likely caused by falcon tube not being super accurate with its measurements         -Concluded CAM is at a lower concentration that originally thought -Made LB+CAM Plates     -Used 350µL CAM     -Positive Control was restreaking from RFP control plate     -Negative control JM109 w/o transformant -Gel results RFP     -RFP Fragment was produced brightly     -Did not proceed with gel purification because of already successful purification earlier         -Started this reaction before Purification was complete -PCR of All Centromeric fragments     -Made Master Mix (16 rxns):         -15.6µL dH20         -5 Phusion HF         -1 gDNA         -1.3 F, R         -0.5 dNTP         -0.3Phusion     -1.3ul of F and R each to all the wells     -Melting temperatures were determined by NEB calculation         -1.1 69C         -1.2 70C         -1.3 69C         -1.4 60C         -1.5 67C         -2.1 66C         -2.2 67C         -2.3 64C (Difference in Temps huge!!!)         -2.4 70C         -2.5 69C         -3.1 68C         -3.2 67C         -3.3 68C         -3.4 71C         -3.5 71C     -Gel results         -Fragments that worked             -1.4             -2.2 (Faint)             -3.1 (Faint)             -3.2 (Faint)             -3.3 (Faint)             -3.5 (Faint)         -Attempted to gel purify 1.4, 2.2, 3.1, 3.2             -All were unsuccessful                 -However, did a second re-elution and DNA did increase indicating we may need to just wash more of the salt. -Made 2 2mL overnight cultures of pSD000 A5 plate w/ 2.1µL of Simone’s CAM Stock -Made 5mL LB overnight of Dh5a Plate 8/6/18 -Plates did not work. Neither plate had growth. CAM too high concentration -Made new LB+CAM plates     -300µL/500mL CAM stock     -Positive control restreaked RFP control plate     -Negative control restreaked Dh5Alpha plate -Miniprep of pSD000 Overnights     -Combined into single tube     -Yield was 287.8ng/µL (measured 8/7/18) -Did not process Dh5Alpha overnight as needed someone else to help with their prep. No one else was here. Discarded 8/7/18 -LB+CAM Plate Results     -Negative control showed no growth. Passed     -Positive control showed growth but very limited     -Consulsion         -CAM plates are usable but are on the stronger side         -Recommendations for next plate batch will be 290µL of CAM stock -PCR of all fragments using 30-mers     -Made Master Mix (16 rxns):         -15.6µL dH20         -5 Phusion HF         -1 gDNA         -1.3 F, R         -0.5 dNTP         -0.3Phusion     -1.3ul of F and R each to all the wells     -Annealing temperature was 69C as this was the lowest calculated by NEB Tm Calc     -Gel Results         -Had many successful fragments (_______________)         -Attempted to Gel purify 1.2, 2.4, and 3.1             -All 3 were successful with the Zymo Kit -Restriction digest of pSD000     -Recipe         -5µL CutSmart Buffer         -5µL pSD000         -0.5µL XbaI         -9.5µL dH20     -Dephosphorylation Step was addition of 1µL of Quick CIP directly to mixture. 10 min@37C followed by 2min@80C     -Gel Results         -Fragments were fainted and showed One fragment at the proper size and one at >10kb.         -Gel Purification did not work. 260/230 =0.02 8/8/18 -Autoclaved eppendorf tubes and tips -Placed 55 eppendorf tubes on ice with lids closed -Ran a 0.5% agarose gel of the four centromeric 1.4 reactions using 30-mer primers     -Run at 135V on a large gel (started at 10:30am)     -Gel purified using promega kit, zymo kit, and Qiagen kit -Gel purification of the four 1.4 reactions using 30-mer primers     -Used the Qiagen kit, Promega kit, Zymo kit, and a homemade reagent kit brought by Simone (grad student)     -None of the kits yielded satisfactory concentrations (the largest  being 14 ng/µL) or purities (all 260/230 values were below 0.5)     -Followed each kit’s respective manufacturer's protocol -Incubated DH5ɑ cells (in the process of making competent cells) in 37ºC shaking incubator until an OD of 0.5 was reached     -At 600nm -Dephosphorylated restriction digest RD pSD000     -Placed in a PCR tube     -dH20 to 35µL (4.75µL)     -3.5µL Cutsmart     -25µL of the backbone     -1.75µL of the Quick CIP     -Incubated at 37ºC for 10 minutes     -Heat inactivated in thermocycler at 80ºC for 2 minutes     -Stored in the –20ºC -Measured OD of liquid 7002 culture     -Used A+ media as a blank     -Measured at 730 nm; OD was 0.001 8/9/19 -Made SD  – Arg plates     -10g glucose     -2.5g ammonium sulfate     -0.4g CSM - Arg     -0.85g YNB - amino acids     -10g agar     -500 mL milliQ water -Gel purified dephosphorylated digest of PSD000 from 8/8/18 and remaining (numbered) PCR products from 8/8/18     -0.5% agarose gel, run at 135 for 30 minutes     -Top row         -Lane 1: 1 Kb ladder         -Lane 2: 2.2         -Lane 3: 1 Kb ladder         -Lane 4: 2.5         -Lane 5: 3.1         -Lane 6: 3.2         -Lane 7: 3.3         -Lane 8: 3.4         -Lane 9: 3.5     -Bottom row         -Lane 1: 1 Kb ladder         -Lane 2: restriction digest of pSD000         -Lane 3: 1.2         -Lane 4: 2.4         -Lane 5: 3.1     -Used Qiagen kit for gel extraction and purification         -Excised and purified pSD000 digest and 2.4         -Nanodrop results: oth had concentrations of around 50 ng/µL             -pSD000 restriction digest had a 260/230 of 1.11             -Reaction 2.4 had a 260/230 value of 0.66 -Made another restriction digest of pSD000 using Xba1     -22.5µL pSD000     -0.5µL Xba1     -5µL CutSmart buffer     -23µL water     -Placed in the 37ºC incubator for 3 hours -Made liquid YPAUD media -Plated blank control, pOW1, and pSD000 yeast transformations made on 7/27/18     -Test SD -Arg plates     -Placed in 30ºC incubator at 5:00pm -Made yeast (pichia) overnight cultures to transform Gibson product into (RFP and pSD000) -Set up overnight cultures of pSB1C3 with RFP     -Used 2.1ul Chlor instead of CAM -Set up Restriction Digest of PSD000 w/ Xba1     -44.4ng/ul template required 22.5ul to have enough for digest:         -23ul H20         -5ul Cutsmart Buffer         -22.5ul template         -.5ul Xba1 from Glick stock     -Ultimately unnecessary because Rachel’s gel purification of PSD000 came back with good results -Set up Gibson Assembly of PSD000 with YM RFP (Step 3 of Workflow)     -10ul reaction total:         -5ul Gibson assembly MM         -3ul template PSD000         -1ul YM RFP         -1ul DH20     -Let sit in 50ºC water bath for 20 minutes     -Rest follows regular transformation procedure -Created .5% Agarose Gel 50 ml     -Ran PCR Products from earlier on Gel         -Well 1: 1kb+ ladder 8ul         -Well 2: PCR Product 1.2 30ul         -Well 3: PCR Product 2.4 30ul         -Well 4: PCR Product 3.1 30ul     -Ran at 110V for 30 min -Gel Purified Results from PCR using Qiagen Gel Extraction Protocol     -Used slightly altered protocol         -3 PE buffer washes, let incubate for 5 minutes in PE buffer before centrifuging 8/10/18 -Ran PCR of 30-mer centromeric region reactions -Mini-prepped overnight culture from 8/9/18     -Nanodrop reported good concentrations (around 200 ng/µL) and purity (260/230 value around 2.1)     -Mini-prepped pSB1C3-RFP         -Made two overnight cultures, but only one had the red color characteristic of RFP             -Discarded the yellow ‘RFP’ overnight culture and only mini-prepped the red one -Made an overnight yeast culture     -25 mL of YPAUD liquid media     -Used Pichia Pastoris strain PPY12 -Made a 0.5% agarose gel and ran 30-mer PCR products     -Samples 3.1 and 3.4 were not run as the tubes opened up in the PCR machine     -Top row         -Well 1: 1 Kb ladder         -Well 2: 1.1         -Well 3: 1.2         -Well 4: 1.3         -Well 5: 1.4         -Well 6: 1.5         -Well 7: 1 Kb ladder         -Well 8: 2.1         -Well 9: 2.2         -Well 10: 2.3         -Well 11: 2.4         -Well 12: 2.5     -Bottom row         -Well 1: 1 Kb ladder         -Well 2: 3.2         -Well 3: 3.3         -Well 4: 3.4     -Run at 135 volts for 30 minutes 8/11/18 -Mini Prepped overnight cultures     -None of the overnight cultures in the culture  tubes were cloudy indicating no  growth         -Added the overnight culture samples to an eppendorf tube and spun down in  microcentrifuge             -No pellet  seen, discarded samples     -Will have to make more overnight cultures and attempt  to mini-prep again 8/12/18 -Made 8 overnight cultures of pSD000+RFP     -Added 2.1µL of chloramphenicol stock to each culture tube     -Placed in the 37ºC incubator at 5:35pm -Streak purified pSD000+RFP plate from 8/9/18     -Previous overnight cultures did not grow         -Possible that the transformed pSD000+RFP was mistakenly plated on an LB plate     -If no growth is seen, pSD000+RFP was plated on an LB plate and will have to be re-transformed     -If growth is seen, the problem lies in the liquid media the overnight cultures are incubated in     -Placed in the 37ºC incubator at 5:35pm -All transformed yeast (with a blank control, pOW1, and pSD000) grew on SD -Arg plates     -Plated 8/9/18 and placed in the 30ºC incubator     -Suggests arginine contamination on the plates 8/13/18 -Overnight cultures of pSD000 + RFP failed     -No signs of bacterial growth (i.e. solutions were clear)     -It is likely that pSD000 + RFP was mistakingly plated on a LB plate instead of a LB + CAM plate -Made LB liquid media -Made LB + CAM plates     -290 µL of chloramphenicol stock     -Positive control: re-streaked from pSB1C3-RFP plate     -Negative control: transformed JM109 with H2O -Made SD-Arg plates -Transformed Pichia Pastoris to test SD - Arg plates     -Negative/blank control: transformed with H2O         -Plated on both our plate (from 8/9/18) and the Glick lab’s plate     -Positive control: transformed with pOW1         -Plated on both our plate (from 8/9/18) and the Glick lab’s plate     -Placed plates in the 30ºC incubator at 4:00pm -Ran 0.5% agarose gel of 30-mer PCR products from 8/10/18     -Top row         -Well 1: 1 Kb ladder         -Well 2: 1.1         -Well 3: 1.4         -Well 4: 1.5         -Well 5: 1 Kb ladder         -Well 6: 2.1         -Well 7: 2.2         -Well 8: 2.3         -Well 9: 2.5     -Bottom row         -Well 1: 1 Kb ladder         -Well 2: 3.2         -Well 3: 3.3         -Well 4: 3.4     -No bands of the appropriate size were seen when the gel was imaged -Re-labeled LB plates with the appropriate marking     -1 black line (1 purple line marks an LB + CAM plate) -Re-streaked pSD000 A5 plate onto a LB + CAM plate from 7/23/18     -Placed in 37ºC incubator at 5:00pm -Made overnight cultures of pSD000 A5 plate from 7/23/18     -Placed in 37ºC incubator at 5:00pm -Ran 30-mer centromeric PCR products from 8/13/18 on a 0.5% agarose gel     -135V for 30 minutes     -PCR product 2.5 wasn’t run since the tube opened in the thermal cycler     -Top row         -Well 1: 1 Kb ladder         -Well 2: 1.1         -Well 3: 1.3         -Well 4: 1.4         -Well 5: 1 Kb ladder         -Well 6: 2.1         -Well 7: 2.2         -Well 8: 2.3     -Bottom row         -Well 1: 1 Kb ladder         -Well 2: 3.1         -Well 4: 3.2         -Well 6: 3.3         -Well 8: 3.4         -Well 10: 3.5         -Well 12: 1.5 -Gel purified 30-mer PCR products from 8/13/18     -Used Zymo gel extraction/purification kit and followed manufacturer’s protocol Ran PCR of 13 centromeric sequences excluding 2.4 and 1.2 using 30-mer primers -Made Master Mix:         -70ul 5x Phusion Buffer         -14ul PP gDNA (100ng/ul dilution made today)         -7ul DNTP         -215ul H20         -1.3ul of each 30mer F and R primer         -0.3ul Phusion     -Ran at 69C under Genehackers Protocol -Made liquid YPAUD media using Jason’s Introductory Guide -Gel Purified Reactions using Zymo research kit         -Reactions 1.3,1.4, and 1.5 were able to be excised         -However, resulting purities were not high enough to be utilized -Performed Replica Plating of PSD001 plate         -Used Velvet from Glick lab along with glass bottle         -Set plate in incubator overnight 8/14/18 -Mini-prepped pSD000 overnight cultures     -Nanodrop results: satisfactory purities and concentrations, placed in freezer box in –20ºC -Replica plate of pSD001(pSD000 + RFP) produced colonies     -Had slightly fewer colonies than the parent plate indicating that the parent plate was mistakenly plated on a LB plate -Made overnight cultures of pSD001 and placed in 37ºC incubator at 5:00pm     -Added 1.8µL chloramphenicol stock -LB + CAM plates made 8/13/18     -Positive control: grew characteristically red RFP colonies     -Negative control: no colonies observed     -The plates are useable -Ran 30-mer PCR centromeric reactions     -Used a master mix     -Ran on a 0.5% agarose gel at 135V for 30 minutes -Checked SD-Arg control plates     -Positive control plated on our SD-Arg plate grew colonies     -Negative/blank control on our SD - Arg plate grew colonies     -Negative/blank control on Glick lab’s SD-Arg plate grew no colonies -Made yeast overnight cultures     -Yeast strain Pichia Pastoris (PPY12)     -Placed in 30ºC shaking incubator at 5:00pm -Set up 30-mer PCR reactions to be run in thermal cycler tomorrow morning     -Master mix         -220 µl DH20         -70µl 5X Phusion buffer         -14µl gDNA         -7µl DNTPs     -1.3ul of each forward and reverse primer     -Will add 0.3µL of phusion polymerase and run reactions in the morning         -Placed in the fridge next to the PCR machines -PCR of 1.4 (1.4b)     -15.6 dH20     -5 Phusion HF     -1.3 InvF and InvR     -1 gDNA     -0.5 dNTP     -0.3 Phusion -Results of plates from 8/13:     -PSD000 A5: streaks showed growth     -PSD001 Replica: Produced colonies in a ring due to velvet setting pattern     -JM109 Negative control: Produced 1 colony     -JM109 Positive control: Colonies seen in streaks -Created .5% agarose gel 150ml     -Ran all 13 remaining centromeric sequences w/ 30-mers on this gel -Ran gel of our 13 centromeric sequences     -Ran out of 6x Loading Dye so used Glick Lab stock from now on     -Top Row:         -Well 1,6,12: 1kb+ ladder 10ul         -Wells 2-5: PCR Products 2.1,2.2,2.3,2.5 25ul         -Wells 7-11: PCR Products 1.1,1.3,1.4,1.4b,1.5 25ul     -Bottom Row:         -Well 1,7: 1kb+ ladder 10ul         -Wells 2-6: PCR Products 3.1-3.5 25ul -Gel Purified Results using Zymo Kit     -This time, a maximum of 450ul ADB was added to reactions 1.3 and 2.2     -However, still yielded unsatisfactory results -Set up PCR of all 13 30-mer centromeric sequences     -Made Master Mix:         -220ul DH20         -70ul Phusion Buffer         -14ul gDNA         -7ul DNTP     -Added 1.3ul of each F and R primer     -Added 0.3µL of Phusion HF polymerase to each tube 8/15/18 -Ran 30-mer centromeric PCR reactions -Made more pSD001     -Digest pSD000 with Xba1         -23µl H20         -5µl Cutsmart Buffer         -22.5µl pSD000         -0.5µl Xba1 from Glick stock         -Placed in 37ºC incubator for 3 hours -Gel purified 30-mer PCR products from 8/14/18     -Ran on a 0.5% agarose gel at 135V for 30 minutes     -Top row         -Well 1: 1 Kb ladder         -Well 2: 1.1         -Well 3: 1.3         -Well 4: 1.4         -Well 5: 1.5         -Well 6: 1 Kb ladder         -Well 7: 2.1         -Well 8: 2.2         -Well 9: 2.3         -Well 10: 2.5     -Bottom row         -Well 1: 1 Kb ladder         -Well 2: 3.1         -Well 3: 3.2         -Well 4: 3.3     -Used Qiagen kit         -350µL of homemade GQ buffer from Simone’s lab         -3 washes with Qiagen PE buffer, let each wash incubate for 5 minutes         -Let elution buffer sit for 4 minutes before centrifuging         -Did not get satisfactory purity or concentration for 30-mer centromeric sequence 1.4 -Removed pSD001 overnight cultures from 37ºC incubator at 9:30am     -All culture tubes contained clear solution indicating no bacterial growth         -Colonies observed on pSD001 LB +  CAM plate might be from contamination -Removed yeast (PPY12) overnight culture from 30ºC incubator at 9:30am     -Placed in 4ºC cold room -Dephosphorylated pSD000 digested with Xba1 (35µL reaction)     -4.75µL dH2O     -3.5µL Cutsmart buffer     -25µL of digested pSD000     -1.75µL of the Quick CIP     -Incubated at 37ºC for 10 minutes     -Heat inactivated in heat block at 80ºC for 2 minutes -Gibson assembly reaction with YM-RFP and dephosphorylated pSD000     -10µl reaction         -5µl Gibson assembly master mix         -3µl template pSD000         -1µl YM-RFP         -1µl dH20     -Placed in 50ºC water bath for 20 minutes     -Transformed into JM109 competent cells         -Ran out of LB + CAM plates         -Spun down cells at 5K rpm for 2 minutes, removed the upper 200µL of supernatant fluid, resuspended in remaining 100µL of SOC media -Transformed dH2O into JM109 competent cells for another negative control test of LB + CAM plates     -Spun down cells at 5K rpm for 2 minutes, removed the upper 200µL of supernatant fluid, resuspended in remaining 100µL of SOC media     -Plated on LB + CAM plate and placed in 37ºC incubator at 5:00pm -Gel purified 30-mer PCR reactions from 8/15/18 morning     -Used Zymo gel DNA recovery kit according to manufacturer's directions -Ran PCR reaction of pOW1-RFP with rfp.F and rfp.R to get YM-RFP fragment -Set up 30-mer PCR reactions to be run in the morning     -Master mix         -440µl dH20         -140µl 5X Phusion buffer         -28µl gDNA         -14µl DNTPs     -22.1µL of master mix in each PCR tube -Set up 60-mer PCR reactions of 1.4 to be run in the morning     -14.6µL of dH2O     -5µL 5X Phusion buffer     -1.3 µL of each primer     -2 µL of 30-mer PCR product     -0.5µL DNTPs 8/16/18 -Made LB + CAM plates     -290 µL of chloramphenicol stock     -Negative control: transformed with sterile H2O     -Positive control: transformed with PSB1C3-RFP     -Positive and negative controls plated and placed in 37ºC incubator at 5:00pm -Gel purified YM-RFP fragment     -Ran on a 0.5% agarose gel at 110V for 30 minutes     -Used Zymo gel DNA recovery kit according to manufacturer’s directions -Ran of 30-mer PCR reactions in duplicate -Ran 60-mer PCR reaction of centromeric sequence 1.4 -Gel purified 30-mer PCR reactions and 60-mer centromeric sequence 1.4     -Ran on a 0.5% agarose gel at 135V for 30 minutes     -Used Zymo gel DNA recovery kit according to manufacturer’s directions     -Got the 60-mer product of centromeric sequence 1.4     -30-mer PCR reactions had unsatisfactory purities (260/230 values) -Ran 30-mer centromeric PCR reactions     -Master mix         -96.7µL dH2O         -31µL 5X Phusion buffer         -6.2µL gDNA         -3.1µL DNTPs         -22.1µL of master mix to each tube         -0.3 µL Phusion HF polymerase to each tube         -1.3 µL of each primer     -1.3     -2.2     -3.1     -3.2     -3.3 -Ran PCR reaction of pOW1-RFP with rfp.F and rfp.R to get YM-RFP fragment 8/17/18 -Gel purified 30-mer PCR products and YM-RFP PCR product     -Ran on a 0.5% agarose gel at 110V for 30 minutes     -Used Zymo kit according to manufacturer's directions -Negative control of LB + CAM plates grew colonies indicating that insufficient chloramphenicol was added     -These plates are unusable     -Will add 300µL of chloramphenicol next time -Made LB + CAM plates     -Added 300µL of chloramphenicol     -Negative control: transformed with sterile H2O     -Positive control: transformed with PSB1C3-RFP     -Positive and negative controls plated and placed in 37ºC incubator at 5:00pm -Ran a PCR of 30-mer reactions     -Centromeric reactions 2.2 (69ºC), 2.5 (69ºC), 3.3(68.4ºC or 68ºC) and 3.5 (70.9ºC or 71ºC) -Made overnight cultures of pSD001 (pSD000 + RFP) plate made 8/9/18     -2 mL of LB media + 1.25µL of CAM stock -Gel imaged PCR product of reaction 2.2     -Ran on a 0.5% agarose gel at 110V for 25 minutes     -No band of the appropriate size was seen when the gel was imaged 8/18/18 -Gel purified 30-mer PCR products from RF     -2.2 Popped open during reaction. Discarded     -2.5 Yielded a fragment of wrong size. Did not purify     -3.5 Yielded Very faint fragment of right size. Main band looks like primer dimer.         -Gel purification of faint band failed. Likely too faint (<50ng) 8/20/18 -Digested pSD000 with Xba1     -23µL water     -22.5µL pSD000     -5µL CutSmart buffer     -0.5µL Xba1     -Placed in the 37ºC incubator for 3 hours -Dephosphorylated pSD000 digested with Xba1 (35µL reaction)     -4.75µL dH2O     -3.5µL Cutsmart buffer     -25µL of digested pSD000     -1.75µL of the Quick CIP     -Incubated at 37ºC for 10 minutes     -Heat inactivated in thermal cycler at 80ºC for 2 minutes -Gel purified dephosphorylated pSD000 digested with Xba1     -Ran a 0.5% agarose gel at 110V for 30 minutes     -Used Zymo gel DNA recovery kit according to manufacturer's instructions     -Nanodrop reported unsatisfactory purity (260/230 value was 0.12) -Dephosphorylated pSD000 digested with Xba1 found in –20ºC freezer (digested 8/9, 35µL reaction)     -4.75µL dH2O     -3.5µL Cutsmart buffer     -25µL of digested pSD000     -1.75µL of the Quick CIP     -Incubated at 37ºC for 10 minutes     -Heat inactivated in thermal cycler at 80ºC for 2 minutes -Gel purified dephosphorylated pSD000 digested with Xba1     -Ran a 0.5% agarose gel at 110V for 30 minutes     -Used Zymo gel DNA recovery kit according to manufacturer's instructions -Made overnights of the Interlab Transformed Colonies     -2 colonies were picked from each of 8 plates for a total of 16 overnights     -5ul LB added to each culture tube     -3.13ul CAM stock added     -Will be set in incubator later today to be ready for Interlab Data Collection Tomorrow -Dephosphorylated restriction digest RD pSD000     -Placed in a PCR tube     -dH20 to 35µL (4.75µL)     -3.5µL Cutsmart     -25µL of the backbone     -1.75µL of the Quick CIP     -Incubated at 37ºC for 10 minutes     -Heat inactivated in thermocycler at 80ºC for 2 minutes     -Stored in the –20ºC -Made .5% 50 ml Agarose Gel to run dephosphorylated RD of pSD000 on -Gel Purified PCR Product using Zymo Kit but with Lianne’s Pipets     -Results were still too impure to be used -Placed overnight cultures for Interlab into Incubator at 5:15pm 8/21/18 -Set Up PCR of YM-RFP with Gibson Primers     -1 tube used:         -5ul Phusion Buffer         -1ul pOW1-RFP Template         -15.6ul H20         -1.3 ul each F and R RFP Primer         -.5ul DNTP         -.3ul Phusion         -Ran at annealing temperature of 69 C and time of 1:30         -Didn’t have time to gel purify today b/c of Interlab -Added 300ul CAM to each of Rachel’s plates and then poured them     -Made 33LB+CAM Plates -Gel Purified Dephosphorylated Digests     -Gel:         -Well 1: 1kb ladder 5ul         -Well 2-3: Dephosphorylated digest 25ul 1-2     -Gel Purified with Zymo Kit with minor changes made by Cian:         -For 2nd wash step, centrifuge for 5 min     -Only 1 result was somewhat successful w/ purity of .83 and minor spike in 260 Abs         -Other result displayed different kind of Guanadinium contamination -Digested pSD000 with Xba1     -23µL water     -22.5µL pSD000     -5µL CutSmart buffer     -0.5µL Xba1     -Placed in the 37ºC incubator for 3 hours -Made LB + CAM plates     -Added 300µL of chloramphenicol stock     -Control transformations         -Positive control: transformed with PSB1C3-RFP         -Negative control: transformed with sterile H2O 8/22/18 -Made .5% 50ml Agarose Gel to Run YM-RFP PCR Product on     -Gel:         -Well 1: 1kb Ladder 5ul         -Well 2:YM-RFP PCR Product 30ul         -Well 3-4: Leftover Dephosphorylated Digests 1 & 2 from 8/21 25ul         -Ran @ 110V for 30min -Gel-Purified all 3 test wells from Gel Today     -However, none of the 3 wells were pure enough to be used further. Results once again indicated regular Guanidinium contamination -Mini-Prepped Cian’s Overnight cultures     -All 5 samples (D,E,F,G,H) had suitable concentrations and purities to be used further -Set up diagnostic digests for 5 miniprepped samples     -Digest 1: MSCI (Cutsmart Buffer)     -Digest 2:MSCI+SBHI-HF (Buffer 3.1)     -Digest 3:MSCI+BGLI (Cutsmart Buffer)     -All 3 digests run for each of 5 samples plus 1 control RFP sample for 18 total samples     -Setup:         -4ul Template (D,E,F,G,H,control)         -1ul Buffer         -4.9ul H20         -.1ul Restriction Enzyme     -Set in 37 C incubator for 1 hr -Made .5% 50 ml gel for PCR samples from earlier this morning     -Gel:         -Well 1: 1kb+ Ladder 5ul         -Well 2: PCR Product 2.3 30ul         -Well 3: PCR Product 2.5 30ul     -Gel ran @110V for 30 min     -However, after gel was imaged, only sample 2.3 produced correct banding         -However, band was too faint to be excised properly -Made .5% 120ml gel for Diagnostic Digests     -Gel set up with different sized combs so Well designations are uneven     -Top Row:         -Clump 1:             -1kb+ Ladder 5ul             -Control Digest 3 10ul             -Control Digest 2 10ul             -Control Digest 1 10ul         -Clump 2:             -1kb+ Ladder 5ul             -Sample D Digest 3 10ul             -Sample D Digest 2 10ul             -Sample D Digest 1 10ul         -Clump 3:             -1kb+ Ladder 5ul             -Sample E Digest 3 10ul             -Sample E Digest 2 10ul             -Sample E Digest 1 10ul             -1kb+ Ladder 5ul     -Bottom Row:         -Clump 1:             -1kb+ Ladder 5ul             -Sample F Digest 3 10ul             -Sample F Digest 2 10ul             -Sample F Digest 1 10ul         -Clump 2:             -1kb+ Ladder 5ul             -Sample G Digest 3 10ul             -Sample G Digest 2 10ul             -Sample G Digest 1 10ul         -Clump 3:             -1kb+ Ladder 5ul             -Sample H Digest 3 10ul             -Sample H Digest 2 10ul             -Sample H Digest 1 10ul             -1kb+ Ladder 5ul -Made overnight cultures of yeast PPY12 in 25 ml YPAUD     -Left in 30ºC incubator overnight at 5:00pm 8/23/18 -Ran centromeric PCR reactions with new 30-mer primers -Ran PCR reaction of YM-RFP with new primers     -PCR tube popped open, no product left -Organized primer box -PCR amplified YM-RFP     -15.6µL water     -5µL 5X Phusion buffer     -1.3µL of each primer (RFP.F2 and RFP.R2)     -1µL pOW1 template     -0.5µL DNTPs mix     -0.3µL Phusion polymerase -Made liquid LB media -Made LB + CAM plates     -Added 300µL of chloramphenicol stock     -Control transformations         -Positive control: transformed with PSB1C3-RFP         -Negative control: transformed with sterile H2O     -Placed control plates in 37ºC incubator at 8:00pm -Gel purified 30-mer PCR products and YM-RFP PCR product     -Ran on a 0.5% agarose gel at 135V for 25 minutes     -Used Zymo gel recovery kit according to manufacturer's instructions with the following modifications         -Spun at 11K rpm         -Incubated the ABD for 1 minute in the column for 1 minute before spinning in microcentrifuge         -Spun the first wash step for 1 minute         -Spun the second wash step for 5 minutes         -Eluted all samples in a volume of 6µL     -The 30-mer centromeric sequences had sufficient concentrations and purities to proceed with their respective 60-mer PCR reactions     -The YM-RFP had sufficient concentration and purity for Gibson -Gibson assembly reaction of digested and dephosphorylated pSD000 + YM-RFP to make pSD001     -10ul reaction total:         -5µl Gibson assembly master mix         -2µl digested and dephosphorylated pSD000         -1µl YM-RFP         -2µl dH20     -Let sit in 50ºC water bath for 20 minutes     -Transformed 5µL of gibson product into JM109 competent cells         -Plated on LB + CAM plates from Simone’s lab and placed in 37ºC incubator at 7:00pm         -Stored the remaining product in ‘magic box’ located in bottom drawer of –20ºC freezer -Placed control plates for LB + CAM plates made 8/21/18 in the 37ºC incubator at 8:00pm -Ran 30-mer PCRs of centromertic reactions and 60-mer of reactions 3.2 and 3.4     -Master mix         -129.5µL dH2O         -41.5µL 5X Phusion buffer         -4.2µL DNTPs mix         -2.5µL Phusion polymerase         -1µL of gDNA and 21.4µL of master mix to each 30-mer reaction         -21.4µL of master mix to for 30-mer PCR reactions and 20.4µL of master mix for each 60-mer reaction -Made .5% 150ml Agarose gel for YM-RFP and Centromeric Sequences     -Gel Setup:         -Well 1, : 1kb + Ladder 5ul         -Well 2:PCR Product 2.1 30ul         -Well 3: PCR Product 2.3 30ul         -Well 4:PCR Product 2.5 30ul         -Well 5:PCR Product 3.1 30ul         -Well 6: PCR Product 3.2 30ul         -Well 7: PCR Product 3.4 30ul         -Well 8: PCR Prod         -Well 8-9: YMRFP PCR Products 1-2 30ul     -Gel Ran @130V for 30 min -Continued with yeast transformations following Jason’s Introductory Protocol Guide     -Used 50ml Falcon tubes to spin down supernatant     -Used Jason’s Sorbitol, DTT, and Hepes buffer 8/24/18 -Made overnight cultures of pSD001     -Added 1.25µL of chloramphenicol stock to each culture tube containing 2 mL LB liquid media     -Placed in 37ºC shaking incubator at 5:00pm -Positive and negative control plates for LB + CAM plates made on 8/21/18 and 8/23/18 both grew colonies     -Plates cannot be used -Ran on a 0.5% agarose gel of centromeric PCR products from 8/23/18 at 135V for 30 minutes     -No bands of the appropriate size, could not gel purify -Ran 30-mer centromeric PCR reactions     -Master mix         -129.5µL dH2O         -41.5µL 5X Phusion buffer         -4.2µL DNTPs mix         -8.3µL of gDNA         -2.5µL Phusion polymerase     -21.4µL master mix to each tube     -1.3µL of forward and reverse primers to each tube -Made LB + CAM plates     -Added 300µL of chloramphenicol stock     -Control transformations         -Positive control: transformed with PSB1C3-RFP         -Negative control: transformed with sterile H2O     -Placed control plates in 37ºC incubator at 5:00pm -Ran 30-mer PCR reactions     -Used a master mix -Re-made primer stocks -Gel purified PCR products from this morning     -Used Zymo gel recovery kit according to manufacturer's instructions with the following modifications         -Spun at 11K rpm         -Incubated the ABD for 1 minute in the column for 1 minute before spinning in microcentrifuge         -Spun the first wash step for 1 minute         -Spun the second wash step for 5 minutes -Gel purification resulted in a satisfactory sample for the 30-mer PCR product of reaction 2.3 8/25/18 -Control plates for LB + CAM plates made 8/24/18 removed at 9:30am     -Both the positive and negative control plates grew colonies -Miniprepped pSD001 overnight cultures     -Used Promega Pureyield system and manufacturer’s protocol     -Nanodrop reported sufficient 260/230 values, however the concentrations were quite low         -None of the samples had a concentration above 40ng/µL -Placed the pSD001 plate in the 37ºC incubator for an additional three hours -Ran diagnostic digests of pSD001     -1st test: tests for correct size (labeled in blue)         -3.9µL dH2O (control: 7.9µL dH2O)         -1µL Cutsmart buffer         -5µL of pSD001 template (control: 1 µL of pSD000)         -0.1µL of Sma1     -2nd test: tests that the correct insert is present (labeled in black)         -3.9µL dH2O (control: 7.9µL dH2O)         -1µL Cutsmart buffer         -5µL of pSD001 template (control: 1µL of pSD000)         -0.1µL of Btg1         -Produces 1 fragment in pSD000 and 2 fragments in pSD001     -Imaging the gel of the diagnostic digests showed none of the samples to be consistent with what we’d expect for pSD001 -Gel purified PCR products from 8/24/18     -Ran on a 0.5% agarose gel at 110V for 30 minutes     -Used Zymo gel recovery kit according to manufacturer's instructions with the following modifications         -Spun at 11K rpm         -Incubated the ABD for 1 minute in the column for 1 minute before spinning in microcentrifuge         -Spun the first wash step for 1 minute         -Spun the second wash step for 5 minutes     -Nanodrop did not report satisfactory concentrations or purities for any of the samples -Ran 30-mer centromeric PCR reactions     -Master mix         -163.8µL dH2O         -52.5µL 5X Phusion buffer         -10.5µL gDNA         -5.25µL DNTP mix         -3.15µL Phusion HF polymerase     -1.3µL of each forward and reverse primer to each tube     -21.4µL of master mix to each tube 8/26/18 -SD-Arg control plates transformed 8/23/18     -Both the positive and negative control plates grew distinct white colonies         -Positive control plate grew significantly fewer colonies         -Suggests arginine contamination is present on the plates -Miniprepped pSD001 overnight cultures     -Nanodrop reported satisfactory concentrations (ranging from 150 ng/µL to 200 ng/µL) and purities (260/230 values ranged from 1.5 to 2.3) -Ran diagnostic digests of pSD001     -1st test: tests for correct size (labeled in blue)         -7.9µL dH2O         -1µL Cutsmart buffer         -1µL of pSD001 template and pSD000 template         -0.1µL of Sma1     -2nd test: tests that the correct insert is present (labeled in black)         -7.9µL dH2O         -1µL Cutsmart buffer         -1µL of pSD001 template and pSD000 template         -0.1µL of Btg1             -Produces 1 fragment in pSD000 and 2 fragments in pSD001     -Incubated diagnostic digests in 37ºC incubator for one hour -Gel purified 30-mer PCR products from 8/25/18     -Ran on a 0.5% agarose gel at 135V for 30 minutes     -Only reaction 3.2 showed a band of the appropriate size when the gel was imaged     -Used Zymo gel recovery kit according to manufacturer's instructions with the following modifications         -Spun at 11K rpm         -Incubated the ABD for 1 minute in the column for 1 minute before spinning in microcentrifuge         -Spun the first wash step for 1 minute         -Spun the second wash step for 5 minutes     -Nanodrop did not report a satisfactory purity for sample 3.2 (260/230 value of 0.03) -Ran diagnostic digests on a 0.5% agarose gel at 135V for 30 minutes     -Imaging the gel showed none of the samples to be consistent with what we would expect for pSD001     -All of the bands were the same size indicating that pSD001 was not successfully made         -Gibson reaction was unsuccessful, YM-RFP was not inserted 8/27/18 -Ran diagnostic digests of pSD001 samples from 8/26/18     -1st test: tests for correct size (labeled in blue)         -7.9µL dH2O         -1µL Cutsmart buffer         -1µL of pSD001 template and pSD000 template         -0.1µL of Sma1     -2nd test: tests that the correct insert is present (labeled in black)         -7.9µL dH2O         -1µL Cutsmart buffer         -1µL of pSD001 template and pSD000 template         -0.1µL of Btg1             -Produces 1 fragment in pSD000 and 2 fragments in pSD001     -Incubated diagnostic digests in 37ºC incubator for one hour     -Imaging the gel showed none of the samples to be consistent with what we would expect for pSD001     -All of the bands were the same size indicating that pSD001 was not successfully made         -Gibson reaction was unsuccessful, YM-RFP was not inserted     Made overnight cultures of pSD000     -2 mL LB liquid media     -1.25µL of chloramphenicol stock -Dephosphorylated restriction digest RD pSD000     -Placed in an eppendorf tube     -9.5µL dH20     -7µL Cutsmart     -50µL of pSD000 digested with Sma1     -3.5µL of the Quick CIP     -Incubated at 37ºC for 10 minutes     -Heat inactivated in heat block at 80ºC for 2 minute -Gel purified dephosphorylated and digested pSD000 samples     -Ran on a 0.5% agarose gel at 110V for 30 minutes     -Used Zymo gel recovery kit according to manufacturer's instructions with the following modifications         -Spun at 11K rpm         -Incubated the ABD for 1 minute in the column for 1 minute before spinning in microcentrifuge         -Spun the first wash step for 1 minute         -Spun the second wash step for 5 minutes     -Nanodrop reported a concentration of 113 ng/µL and a 260/230 value of 0.70         -Will proceed with gibson assembly -Gibson assembly reaction with YM-RFP and dephosphorylated pSD000     -10µl reaction         -5µl Gibson assembly master mix         -2µl template pSD000         -1.17µl YM-RFP         -1.83µl dH20     -Placed in 50ºC water bath for 20 minutes     -Transformed entire Gibson product into JM109 competent cells         -Spun down cells at 5K rpm for 2 minutes, removed the upper 200µL of supernatant fluid, resuspended in remaining 100µL of SOC media         -Plated on a LB + CAM plate 8/28/18 -Made LB+CAM media for plates     -500ul, kept in 1 bottle     -Autoclaved on liquid setting for 20 min -Made SD-Arg media for plates     -Followed Jason’s Media Protocol -Set up Digests of pSD000 w/ Xba1     -22.5ul H20     -22.5ul Backbone pSD000     -5ul Cutsmart     -.5ul Xba1 -Took out Rachel’s Diagnostic Digests and loaded them     -2ul of loading dye in each     -Gel Setup:         -Top Row: Sma1             -Well 1: 1kb ladder 5ul             -Well 2-9: pSD001 samples A-H 10ul             -Well 10: control sample 10ul             -Well 11: 1kb ladder 5ul         -Bottom Row: Btg1             -Well 1: 1kb ladder 5ul             -Well 2: control sample 10ul             -Well 3-10: pSD001 samples A-H 10ul             -Well 11: 1kb ladder 5ul     -Run at 110V for 30 min -Positive and negative control plates for LB + CAM plates made 8/17/18 both grew colonies     -Plates are not usable -Miniprepped pSD000 overnight cultures and pSD001 overnight cultures     -Followed Promega PureYield Plasmid Miniprep System according to manufacturer's instructions     -Nanodrop reported satisfactory concentrations and purities for both sets of samples -Ran diagnostic digests of pSD001 samples 8/27p/18     -1st test: tests for correct size (labeled in blue)         -7.9µL dH2O         -1µL Cutsmart buffer         -1µL of pSD001 template and pSD000 template         -0.1µL of Sma1     -2nd test: tests that the correct insert is present (labeled in black)         -7.9µL dH2O         -1µL Cutsmart buffer         -1µL of pSD001 template and pSD000 template         -0.1µL of Btg1             -Produces 1 fragment in pSD000 and 2 fragments in pSD001     -Incubated diagnostic digests in 37ºC incubator for one hour -Removed Pichia Pastoris overnight culture from 30ºC incubator at 1:30pm -Made 50 mg/mL chloramphenicol stock     -10 mL of ethanol     -500mg of ethanol-soluble chloramphenicol     -Divided evenly into 10 eppendorf tubes labelled CAM and placed in antibiotic stock freezer box -Ran diagnostic digests of pSD001 samples A-J on a 0.5% agarose gel at 135V for 30 minutes     -1st test: tests for correct size (labeled in blue)         -7.9µL dH2O         -1µL Cutsmart buffer         -1µL of pSD001 template and pSD000 template         -0.1µL of Sma1     -2nd test: tests that the correct insert is present (labeled in black)         -7.9µL dH2O         -1µL Cutsmart buffer         -1µL of pSD001 template and pSD000 template         -0.1µL of Btg1             -Produces 1 fragment in pSD000 and 2 fragments in pSD001     -Incubated diagnostic digests in 37ºC incubator for one hour     -Imaging the gel showed none of the samples to be consistent with what we would expect for pSD001     -All of the bands were the same size indicating that pSD001 was not successfully made         -Gibson reaction was unsuccessful, YM-RFP was not inserted -Gel Purification failed, ran gel, no bands appeared     -Prepared new gel following -New lab meeting notes     -Run the gel for longer     -Something expired last July, we should get more of it     -PCRs have had some changes         -Primers modified to prevent primer dimers     -HS never responded     -Should we contact Northwestern??     -We PCRd the lux operon fragment 8/29/18 -Made .5% Agarose gel for Cian’s Colony PCRs     -Added 1ul Loading Dye     -Gel Setup:         -Top Row:             -Well 1: 1kb+ Ladder 5ul             -Well 2: + Control 6ul             -Well 3: -Control 6ul             -Well 4-24: Samples 1-21 6ul             -Well 25: 1kb+ Ladder 5ul         -Bottom Row:             -Well 1: 1kb+ Ladder 5ul             -Well 2-20: Samples 22-40 6ul             -Well 21: 1kb+ Ladder 5ul         -Ran @110V for 30 min     -Only Sample 9 produced any sort of banding but it was of wrong size. QuickchangePCR must be run     -Set up PCR of 9 Centromeric Regions (1.5, 2.1,2.2,2.3,2.5,3.1-3.4):         -Made Master Mix             -156ul DH20             -50ul Phusion Buffer             -10ul PPGDNA (Dil 8/29 135.5ng/ul)             -5ul DNTP’s             -1.3ul each of F and R Primer             -.3ul Phusion     -Set up Overnight Cultures of pSD001 from Gibson Plate made by VP 8/27         -10 colonies A-J placed in 2ml LB+2.1ulCAM from new stock         -Placed in incubator at 5pm     -Made .5% 150ml Agarose gel for centromeric PCR         -Gel Setup:             -Well 1: 1kb+ladder 5ul             -Well 2: PCR Sample 1.5 25ul             -Well 3,4,5,6, PCR Samples 2,1-2.3, 2.5 25ul             -Well 7,8,9,10 PCR Samples 3.1-3.14 25ul         -Ran @135V for 30min     -Only Samples 3.1 and 3.2 from Centromeric PCR amplified correctly         -Conducted Gel Purification with Zymo Kit with modified instructions         -However, nanodrop results were not successful 8/30/18 -Checked overnights, found that none grew- likely due to adding too much CAM from new stock     -Remade overnights with CAM from new stock but with 1ul CAM added -Set up PCR of Centromeric Regions 2.2,3.2,and 3.5     -5ul 5xPhusion Buffer     -15.6ul DH20     -.5ul DNTP     -1ul PPGDNA (DIluted 8/29)     -1.3ul Each F and R Primer     -.3ul Phusion     -Run @IGEM Grad with Temperatures between 67.5 & 70.5 C -Made .5% 50ml Agarose gel to run Centromeric Sequences     -Ran Gel @130V for 30 min     -However, correct banding pattern was not observed for any of the 3 samples 8/31/18-VP -Miniprepped Overnights from 8/30     -Used Promega PureYield MiniPrep Kit and followed manufacturer’s protocol     -Nanodrop results came back with good purities and concentrations for all 10 samples A-J -Set up PCR of Centromeric Regions 1.5, 2.1-2.3,2.5, and 3.1-3.4     -Made a Master Mix         -15.6ul DH20         -5ul Phusion HF         -.5ul DNTP         -1ul ppGDNA Dil. 8/29         -1.3ul each F and R Primer         -.3ul Phusion         -Run on GeneHackers protocol at 67C with elongation time of 4:30 -Made .5% Agarose Gel to run Centromeric PCR on     -Gel Setup:         -Samples 3.2 and 3.4 opened during PCR         -Well 1: 1kb+ Ladder 5ul         -Well 2: PCR Sample 1.5 25ul         -Well 3: PCR Sample 2.1 25ul         -Well 4: PCR Sample 2.2 25ul         -Well 5: PCR Sample 2.3 25ul         -Well 6: PCR Sample 2.5 25ul         -Well 7: PCR Sample 3.1 25ul         -Well 8: PCR Sample 3.3 25ul     -Ran @ 135V for 30 Min -Ran diagnostic digests of MiniPrepped pSD001 samples from today     -1st test: tests for correct size (labeled in blue)         -7.9µL dH2O         -1µL Cutsmart buffer         -1µL of pSD001 template         -0.1µL of Sma1     -2nd test: tests that the correct insert is present (labeled in black)         -7.9µL dH2O         -1µL Cutsmart buffer         -1µL of pSD001 template and pSD000 template         -0.1µL of Btg1             -Produces 1 fragment in pSD000 and 2 fragments in pSD001     -Incubated at 37C for 1 hr -Made .5% Agarose Gel to run Diagnostic Digest on     -Gel Setup:         -Top Row:             -Well 1: 1kb+ Ladder 5ul             -Well 2-11 Sma1 Digests A-H 10ul             -Well 12: 1kb+ Ladder 5ul         -Bottom Row:             -Well 1: 1kb+ Ladder 5ul             -Well 2-11 Btg1 Digests A-H 10ul             -Well 12: 1kb+ Ladder 5ul         -Ran @110V for 30 min     -Results were unclear and I forgot to include control; must run diagnostic digest again
September
9/4/18 -Ran diagnostic digests of MiniPrepped pSD001 samples again with control     -1st test: tests for correct insert (labeled in blue)         -7.9µL dH2O         -1µL Cutsmart buffer         -1µL of pSD001 template and pSD000 template         -0.1µL of Btg1             -Produces 1 fragment in pSD000 and 2 fragments in pSD001     -2nd test: tests that the correct size (labeled in black)         -7.9µL dH2O         -1µL Cutsmart buffer         -1µL of pSD001 template and pSD000 template         -0.1µL of Sma1             -Incubated at 37C for 1 hr -Made .5% 150ml Agarose gel to run diagnostic digest on     -Gel Setup:         -Top Row:             -Well 1: 1kb+ Ladder 5ul             -Well 2-11 Btg1 Digests A-H 10ul             -Well 12: pSD000 sample H 10ul             -Well 13: 1kb+ Ladder 5ul         -Bottom Row:             -Well 1: 1kb+ Ladder 5ul             -Well 2-11 Btg1 Digests A-H 10ul             -Well 12: pSD000 sample H 10ul             -Well 13: 1kb+ Ladder 5ul         -Ran @110V for 30 min -Gibson assembly reaction with YM-RFP and dephosphorylated pSD000     -10µl reaction         -5µl Gibson assembly master mix         -2µl template pSD000         -2µl YM-RFP         -1µl dH20     -Incubated @ 50C for 20 min and then transformed with JM109 -Ran PCR of 10 centromeric regions 1.5, 2.1-2.3,2.5,3.1-3.4     -Made master mix:         -156ulH20         -50ul Phusion HF Buffer         -10ul PPGDNA         -5ul DNTPs         -3ul Phusion     -22.4ul Buffer into each tube     -1.3ul each F and R Primer     -Ran at gradient PCR on right machine -Transformed Gibson Assembly Products with JM109 9/5/18 -Checked Gibson Transformation Plate for growth; found sufficient amount of colonies     -Made overnights of 10 Colonies A-J     -Used 2ml LB and 1ul 50ng/ul CAM Stock     -Placed overnights into incubator before leaving -Made .5% 120ml Agarose gel to run PCR Samples from yesterday     -Gel Setup:         -Well 1: 1kb ladder 5ul         -Well 2: PCR Sample 2.2 25ul         -Well 3: PCR Sample 2.3 25ul         -Well 3: PCR Sample 3.1 25ul         -Well 4:PCR Sample 3.2 25ul         -Well 5: PCR Sample 3.3 25ul         -Well 6-8: PCR Samples Unmarked         -Samples 2.5,3.4, and 2.1 had their markings rubbed off. Therefore, it was impossible to determine which was which     -Ran Gel @135V for 30 min     -However, correct banding pattern wasn’t observed -Transformed Cian’s NC64A QuickChange PCR onto JM109 Bacteria     -Placed Plate in incubator before leaving 9/6/2018 -Miniprepped Samples A-J from Overnight cultures made yesterday     -Followed Manufacturer’s Pureyield Miniprep Protocol found online     -Nanodropped Results         -Only Samples C and D had purities or concentrations not usable         -All other samples could be digested to check for proper insertion -Set up Diagnostic Digest of Miniprepped samples; -Ran diagnostic digests of MiniPrepped pSD001 samples again with control     -1st test: tests for correct insert (labeled in blue)         -7.9µL dH2O         -1µL Cutsmart buffer         -1µL of pSD001 template and pSD000 template         -0.1µL of Btg1             -Produces 1 fragment in pSD000 and 2 fragments in pSD001     -2nd test: tests that the correct size (labeled in black)         -7.9µL dH2O         -1µL Cutsmart buffer         -1µL of pSD001 template and pSD000 template         -0.1µL of Sma1             -Incubated at 37C for 1 hr -Made .5% 60ml Agarose gel to run diagnostic digest on     -Gel Setup:         -Top Row:             -Well 1: 1kb+ Ladder 5ul             -Well 2: pSD000 sample H 10ul             -Well 3-10: Sma1 Digests A,B,E,F,G,H,I,J 10ul         -Bottom Row:             -Well 1: 1kb+ Ladder 5ul             -Well 2: pSD000 sample H 10ul             -Well 3-10: Sma1 Digests A,B,E,F,G,H,I,J 10ul         -Ran @110V for 30 min     -pSD001 turned out no differently than the control, miniprepped results must not be any different -Made 2 .5% 60ml Agarose gels to run Rachel’s PCR on     -Gel 1 Setup:         -Well 1: 1kb+ Ladder 5ul         -Well 2-4: PCR Samples 1.2 25ul         -Wel 5-6: PCR Samples 2.4 25ul         -Rest of PCR Samples stored for tomorrow         -Wells 2,3,5,6 produced correct banding patterns     -Gel 2 Setup:         -Well 1: 1kb+ Ladder 5ul         -Well 2: PCR Sample 2.4 25ul         -Well 3-5: PCR Sample 3.4 25ul         -Only Well 2 Sample Produced correct banding pattern     -Gel Purified correct banding pattern wells         -Used Zymo Kit and followed Modified Zymo Protocol             -Only 1 sample, 2.4A, had usable purity -Ran 30-mer centromeric PCR reactions 1.2, 2.4, and 3.4 in triplicate 9/7/18 -MiniPrepped NC64A overnights from yesterday     -Followed Pureyield Promega Manufacturer’s Protocol found online     -Samples B,C,D,E,F produced usable results -Dephosphorylated Rd000 (35µL reaction)     -4.75µL dH2O     -3.5µL Cutsmart buffer     -25µL of digested pSD000     -1.75µL of the Quick CIP     -Incubated at 37ºC for 10 minutes     -Heat inactivated in thermal cycler at 80ºC for 2 minutes -Conducted Gibson Assembly of Dephosphorylated RD000 with YMRFP     -10µl reaction         -5µl Gibson assembly master mix         -2µl template Dephosphorylated Rd000         -2µl YM-RFP         -1µl dH20     -Incubated @ 50C for 20 min and then transformed with JM109 9/10/18 -Made 1LB Liquid     -IFollowed Jason’s introductory Protocol     -IAdded 250ul CAM stock to each 500ml jar of LB     -IMarked plates with a purple stripe -Made overnights of pSD001 transformation plate     -INot many colonies but found 6 usable ones for overnights     -IUsed 2ml LB+ 1ul CAM stock and put plates overnight -Made overnight of PPY12 with YPAUD     -IAdded PPY12 from -80C freezer to 25ul of YPAUD Media 9/11/18 -Made .5% Agarose gel to run Rachel’s PCR on     -Gel Setup:         -Well 1: 1kb+ Ladder 5ul         -Well 2-4: PCR Samples 1.2 25ul         -Well 5-6: PCR Samples 2.4 25ul     -Ran @ 135V for 30min     -None of PCR Samples amplified correctly -Miniprepped pSD001 samples from yesterday     -Only Sample A had enough growth to miniprep successfully     -Miniprep of A had good enough concentration and purity to be used -Set up restriction digest of Miniprepped DNA     -1st test: tests for correct insert (labeled in blue)         -7.8µL dH2O         -1µL Cutsmart buffer         -1µL of pSD001 template and pSD000 template         -0.2µL of Sma1     -2nd test: tests that the correct size (labeled in black)         -7.8µL dH2O         -1µL Cutsmart buffer         -1µL of pSD001 template and pSD000 template         -0.2µL of Btg1             -Produces 1 fragment in pSD000 and 2 fragments in pSD001             -Incubated at 37C for 1 hr 9/11/18 -Set up 1% Agarose gel for pSD001     -Gel Plate Set Up         -Well 1: 1kb+ Ladder 5ul         -Well 2: SMA1 Sample A (Control)         -Well 3: SMA1 Sample B         -Well 4: SMA1 Sample C         -Well 5: BTG1 Sample A (Control)         -Well 6: BTG1 Sample B         -Well 7: BTG1 Sample C     -Ran plate for 30 min at 110v Starting at 2:50pm -Set up PCR of Centromeric Fragments 1.1,1.3 & 3.3     -PCR Setup:         -15.6ul H20         -5ul 5x Phusion         -.5ul DNTPs         -1ul ppGDNA         -1.3ul each F and R Primer         -.3ul Phusion     -Set in PCR Machine with annealing temp of 69 C and elongation time of 3:30 9/12/18 -Set up .5% Agarose gel for PCR Products from yesterday     -Gel Setup:         -Well 1: 1kb+ Ladder 5ul         -Well 2: PCR Product 1.1 25ul         -Well 3: PCR Product 1.3 25ul         -Well 4: PCR Product 3.3 25ul     -Run @ 130 V for 30 min -Gibson assembly reaction with YM-RFP and dephosphorylated pSD000     -10µl reaction         -5µl Gibson assembly master mix         -2µl template pSD000         -2µl YM-RFP         -1µl dH20     -Incubated @ 50C for 20 min and then transformed with JM109 -Transformed + and - controls of JM109 for LB+CAM plates 9/13/18 -Set up pSD001 Colony PCR     -Plate 1: Red vials     -Plate 2: Green vials     -Plate 3: Orange vials     -Plate 4: Yellow vials     -Plate 5: Purple vials     -Plate 6: Blue vials     -Master mix composed of 240 ul total, 12 ul of PCR mix was placed into each vial, under the assumption .5 ul of genomic DNA would be provided via the selected colony -Each component of the mix was multiplied 10 times to determine the volumes of the master mix     -PCR included rfp F + rfp R 9/14/18 -Made .5% Agarose Gel for Colony PCR Reactions     -Gel Setup:         -Top Row:             -Well 1: 1kb+ Ladder 5ul             -Well 2-10: Samples A1-3, B1-3, C1-3 15ul             -Well 12: 1kb+ Ladder 5ul         -Bottom Row:             -Well 1: 1kb+ Ladder 5ul             -Well 2-10: Samples D1-3,E1-3,F1-3 15ul             -Well 12: 1kb+ Ladder 5ul         -Only 1 Sample, F3, produced correct 1.5kb amplification 9/14/18 -Mini Prep of pSD001 samples from yesterday     -Restriction digest prepared following successful DNA extraction -Sector Assay prepared for pichia pastoris POW1 control 9/17/18 -Familiarize self with lab equipment and techniques, checked on plates and sector assays 9/18/18 -Prepared LB media -Checked on plates and sector assays -Further familiarize self with lab techniques and lab locations 9/19/18 -Prepared overnights of pSD001 plate 6 -Prepared PCR of 1.1, 1.3, 2.1, and 3.3 9/20/18 -Ran Gel of PCRs     -Well 1 - Ladder     -Well 2 - 1.1     -Well 3 - 1.3     -Well 4 - 2.1     -Well 5 - 3.3 -Imaged Gel -Gel Failed 9/21/18 -Prepared PCR of 1.1, 1.3, 2.1, and 3.3 -Ran Gel of PCRs     -Well 1 - Ladder     -Well 2 - 1.1     -Well 3 - 1.3     -Well 4 - 2.1     -Well 5 - 3.3     -Imaged Gel     -Gel Once again failed, bands did not appear as they were supposed to
