<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UChicago_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UChicago/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar"><A href="https://2018.igem.org/Team:UChicago">Home</A><DIV class="dropdown"><BUTTON class="dropbtn">Team
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UChicago/Team">Team Members</A><A href="https://2018.igem.org/Team:UChicago/Collaborations">Collaborations</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Project
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UChicago/Description">Description</A><A href="https://2018.igem.org/Team:UChicago/Design">Design</A><A href="https://2018.igem.org/Team:UChicago/Experiments">Experiments</A><A href="https://2018.igem.org/Team:UChicago/Notebook">Notebook</A><A href="https://2018.igem.org/Team:UChicago/InterLab">InterLab</A><A href="https://2018.igem.org/Team:UChicago/Results">Results</A><A href="https://2018.igem.org/Team:UChicago/Improve">Improve</A><A href="https://2018.igem.org/Team:UChicago/Attributions">Attributions</A></DIV></DIV><A href="https://2018.igem.org/Team:UChicago/Parts">Parts</A><A href="https://2018.igem.org/Team:UChicago/Safety">Safety</A><A href="https://2018.igem.org/Team:UChicago/Human_Practices">Human Practices</A></DIV></DIV></DIV><DIV class="title"><H2>Experiments</H2></DIV><DIV class="section1"><P style="font-size:20px; font-family: 'Raleway', sans-serif;
font-weight: 100;">

In order to create a yeast shuttle vector containing an HpaI restriction site, we first mutagenized an existing iGem backbone (psB1C3) using QuickChange PCR. This step introduces a new restriction cut site (HpaI) that is utilized later in the experiment, thus creating our new plasmid psB1C3mut. The plasmid has chloramphenicol resistance, so it is grown in an LB+CAM (LB and chloramphenicol) media to ensure that bacteria/yeast contain the transformed plasmid. Bacteria and yeast without the plasmid die due to their lack of chloramphenicol resistance.

The lab we work in provided us with a plasmid, pOW1, which contains SCARG4 (a gene that produces arginine) and PARS2 (an autonomous replicating element). We PCRd the region containing these two genes. Using Gibson assembly, we added these two genes into psB1C3mut, resulting in psD000. The presence of SCARG4 and PARS2 in our construct means that psD000 is a shuttle vector (it can be transformed and expressed into both E. coli and yeast). It is required that this ArgArs sequence is included within the plasmid in order to gain proper functionality from the centromeric region.

After the insertion of the ArgArs sequence into the plasmid, we successfully amplied 12 of 15 centromeric regions using PCR and varying combinations of 30 bp and 60 bp primers. This segment of experimentation required numerous trials due to the dimerization of certain 30mer primers. Those primers which dimerized were modified into unique primers which then successfully amplified the centromeric region of a genomic DNA. 

The next step was to make identifiable yeast/bacteria cells by inserting an rfp sequence to the plasmid which would enable.

Distinctions from last year
-We successfully amplified 12 centromeric regions using PCR and varying 30mer and 60mer primers. 
-We also created separate 30mers that avoided the creation of primer dimers that occurred during the use of the original 30mers
-Attempted testing of insertion of rpf sequence to distinguish pichia pastoris cells.</P></DIV></DIV></DIV></DIV></BODY></HTML>