<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UGent_Belgium_LabNotebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UGent Belgium/LabNotebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Bootstrap 101 Template</TITLE></P><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI class="active"><A href="https://2016.igem.org/Team:UGent_Belgium">Home </A></LI><LI><A href=" https://2016.igem.org/Team:UGent_Belgium/Medals">Achievements</A></LI><LI><A href=" https://2016.igem.org/Team:UGent_Belgium/Team"> Team  </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Description">Description </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Shape"> Shape </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Filament"> Filament </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Biofunction"> Biofunction </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Proof"> Proof of concept </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Demonstrate"> Demonstration </A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Safety </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Safety">Lab measures </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Safety#project">Project measures </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Notebook </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Notebook">General notebook </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/LabNotebook">Lab notebook </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Primers">Primers </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/SOP">SOPs </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Parts">Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Parts#composite">Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Parts#fusion">Intermediate Parts </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Practices </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/HP/Silver">Silver </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Integrated_Practices">Integrated</A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Engagement">Engagement </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Awards </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Hardware">Hardware </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Software">Software </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Measurement">Measurement </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Design">Design </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Entrepreneurship">Entrepreneurship </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Collaborations">Collaborations </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Attributions">Attributions </A></LI></DIV></DIV><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:UGent_Belgium">HOME </A></LI><LI class="menu_item"> TEAM
			</LI><LI><A href=" https://2016.igem.org/Team:UGent_Belgium/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Collaborations">★  Collaborations </A></LI></UL><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Description"> ★  Description </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Design"> ★ Design </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Proof"> ★ Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Demonstrate"> ★ Demonstrate </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Notebook"> Notebook </A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Composite_Part"> ★ Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Part_Collection"> ★ Part Collection </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:UGent_Belgium/Safety"> SAFETY </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:UGent_Belgium/Attributions">★  ATTRIBUTIONS </A></LI><LI class="menu_item"> HUMAN PRACTICES 
			</LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Human_Practices"> Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/HP/Silver">★ Silver </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/HP/Gold">★ Gold </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Integrated_Practices"> ★ Integrated Practices </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Engagement">★ Engagement </A></LI><LI class="menu_item"> AWARDS 
			</LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Entrepreneurship"> ★ Entrepreneurship </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Hardware"> ★ Hardware </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Software">★ Software </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Measurement">★  Measurement </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Model">★ Model </A></LI></DIV><DIV class="content_wrapper"><DIV class="container"><H1>Lab Notebook</H1><DIV class="panel-group"><DIV class="panel"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#August">August</A></H4></DIV><DIV id="August" class="panel-collapse  collapse"><UL class="list-group"><LI class="list-group-item borderless"><B>August 29:</B></LI><LI>
We made a strong expression vector (pXS) by restriction/ligation of the linearized plasmid backbone pSB1C3 and a gblock made up of promoter-RBS-cloning site-terminator-terminator (cut with EcoRI and PstI). <A data-toggle="collapse" href="#29aug1">Details</A></LI><LI class="list-group-item borderless"><B>Details:</B><P>Strong expression vector: pSB1C3-P<SUB>BBaJ23119</SUB>-RBS<SUB>BBaB0034</SUB>-GGsite<SUB>BsaI</SUB>-TT<SUB>BBaB0010</SUB>-TT<SUB>BBaB0012</SUB></P><P>Restriction digests:</P></LI><LI>Resuspend gBlocks in Elution Buffer (=TE) to concentration of 50 ng/uL -&gt; 500 ng delivered =&gt; 10uL buffer </LI><LI>Linearised PSB1C3: 25ng/uL at 50uL (used 2013 iGEM tubes)</LI><LI>Prepare mixes (enzymes by NEB):
                     <DIV class="row center"><DIV class="center col-md-6"><P class="center"> Insert digest
<TABLE class="tg"><TBODY><TR><TD class="tg-rg0h">H2O</TD><TD class="tg-rg0h">30</TD><TD class="tg-rg0h" style="text-align: left">uL</TD></TR><TR><TD class="tg-rg0h">CutSmart buffer</TD><TD class="tg-rg0h">10</TD><TD class="tg-rg0h" style="text-align: left">uL</TD></TR><TR><TD class="tg-rg0h">EcoRI-HF</TD><TD class="tg-rg0h">2</TD><TD class="tg-rg0h" style="text-align: left">uL</TD></TR><TR><TD class="tg-rg0h">PstI</TD><TD class="tg-rg0h">3</TD><TD class="tg-rg0h" style="text-align: left">uL (tube 5 years past expiration date)</TD></TR><TR><TD class="tg-rg0h">gBlock 1</TD><TD class="tg-rg0h">5</TD><TD class="tg-rg0h" style="text-align: left">uL (final concentration 5 ng/uL)</TD></TR></TBODY></TABLE></P></DIV><DIV class="center col-md-6"><P class="center"> Backbone digest
<TABLE class="tg"><TBODY><TR><TD class="tg-yw4l">H2O</TD><TD class="tg-yw4l">5</TD><TD class="tg-yw4l" style="text-align: left">uL</TD></TR><TR><TD class="tg-yw4l">CutSmart buffer</TD><TD class="tg-yw4l">10</TD><TD class="tg-yw4l" style="text-align: left">uL</TD></TR><TR><TD class="tg-yw4l">EcoRI-HF</TD><TD class="tg-yw4l">2</TD><TD class="tg-yw4l" style="text-align: left">uL</TD></TR><TR><TD class="tg-yw4l">PstI</TD><TD class="tg-yw4l">3</TD><TD class="tg-yw4l" style="text-align: left">uL (tube 5 years past expiration date)</TD></TR><TR><TD class="tg-yw4l">linear pSB1C3</TD><TD class="tg-yw4l">30</TD><TD class="tg-yw4l" style="text-align: left">uL (final concentration 15 ng/uL)</TD></TR></TBODY></TABLE></P></DIV></DIV>
                    Incubate overnight in 37°C warm water bath (overdigestion because of enzyme expiration date and restriction sites close to fragment termini, especially for the gBlock).
                   </LI></UL></DIV><LI>
 Our first attempts to isolate the <I>inaZ</I> gene from <I>P. syringae</I> and the <I>inaX</I> gene from <I>X. campestris</I> failed. Try <I>inaZ</I> with more stringent conditions and Primestar polymerase, perform more thorough lysis of <I>X. campestris</I> to try <I>inaX</I> again. <A data-toggle="collapse" href="#29aug2">Details</A></LI><LI class="list-group-item borderless"><B>Details:</B><P>The <I>inaZ</I> gene was sourced from BBa_K584027 using primers oM7628 and oM7629* (length 3.6 kbp, anneal at 58°C). The plasmid was obtained from KU Leuven (concentration unknown): PCR (Q5) ran with 10x and 1000x template dilutions, transformed into <I>E. coli</I> DH5a by heatshock, and plated on chloramphenicol. </P><P>The inaX gene was sourced from <I>X. campestris</I> colonies using primers oM7626 and oM7627 (length 4.7 kbp, anneal at 70°C). Cloning overhangs were not included on these primers. The colony was suspended in 1000uL distilled water, vortexed, freezed 30 min. @ -80 deg. C, heated 20 min. @ 95 deg. C, vortexed, and debris spinned down.</P></LI></DIV><LI class="list-group-item borderless"><B>August 30:</B></LI><LI>
We transformed the strong expression vector (cf. 29/08) by electroshock into <I>E. coli</I> Top10, and plated it on chloramphenicol. <A data-toggle="collapse" href="#30aug1">Details</A></LI><LI class="list-group-item borderless"><B>Details:</B><P>Before transformation of the vector, we inactivated the restriction enzymes using heat: 20 min at 80°C, and did ligation without purification. We also purified the samples on a spin column, eluted in 10 uL. </P><DIV class="row"><P class="center"><TABLE><TBODY><TR><TD>length (bp)</TD><TD>conc. (ng/uL)</TD><TD>ratio</TD><TD>mass (ng)</TD><TD>volume (uL)</TD></TR><TR><TD>Backbone</TD><TD>2070</TD><TD>15</TD><TD>1</TD><TD>50</TD><TD>3.33</TD></TR><TR><TD>gBlock</TD><TD>263</TD><TD>5</TD><TD>3</TD><TD>19.06</TD><TD>3.81</TD></TR><TR><TD>H2O</TD><TD>9.86</TD><TD>uL</TD></TR><TR><TD>T4 ligase buffer</TD><TD>2.00</TD><TD>uL</TD></TR><TR><TD>Backbone</TD><TD>3.33</TD><TD>uL</TD></TR><TR><TD>Insert</TD><TD>3.81</TD><TD>uL</TD></TR><TR><TD>T4 ligase</TD><TD>1.00</TD><TD>uL</TD></TR></TBODY></TABLE></P></DIV></LI></DIV><LI>
As a backup strategy we also used CPEC with the backbone from the K584027 plasmid (100x dilution) to generate a strong expression vector. For this, we used Primestar HS PCR with primers oM7643 and oM7645. <A data-toggle="collapse" href="#30aug2">Details</A></LI><LI class="list-group-item borderless"><B>Details:</B><P>Annealing: 5&quot;@55C. Full program: 30x(10&quot;@98C;5&quot;@55C;2'@72C)</P></LI></DIV><LI class="list-group-item borderless"><B>August 31:</B></LI><LI>
Second attempt (cf. 29/08) to isolate the <I>inaZ</I> gene from <I>P. syringae</I> and the inaX gene from <I>X. campestris</I>. <I>InaX</I> failed, but <I>inaZ</I> was successful. Both Primestar HS and Primestar GXL PCR were used for this. On the gel, lane 4 and 8 are <I>inaX</I> attempts. <A data-toggle="collapse" href="#31aug1">Details</A></LI><LI class="list-group-item borderless"><B>Details:</B><P>For <I>inaZ</I>, we used BBa_K584027 as template (1/10, 1/100, and 1/1000 dilutions). Annealing temperature was 66.20°C for primer oM7628 and 61.93°C for primer oM7629* (length: 3.6 kbp). For <I>inaX</I>, we used <I>X. campestris</I> colonies as template. Annealing was done at 60.96°C for primer oM7626 and at 70.34°C for primer oM7627 (length 4.7 kbp). </P><P>Conditions for Primestar HS: 1'@98C;30x(10&quot;@98C;15&quot;@60C;5'@68C);10'@68C (not the standard protocol). Conditions for Primestar GXL: 1'@98C;30x(10&quot;@98C;15&quot;@60C;5'@72C);10'@72C (not the standard protocol).</P></LI></DIV><LI>
Transformation of the strong expression vector by restriction and ligation was not very successful, we only had a few colonies. The backbone amplification using CPEC was successful (see gel, cf. 30/08). <A data-toggle="collapse" href="#31aug2">Details</A></LI><LI class="list-group-item borderless"><B>Details:</B><P>CPEC calculations for strong expression vector:</P><DIV><P class="center"><TABLE><TBODY><TR><TD>length (bp)</TD><TD>conc. (ng/uL)</TD><TD>ratio</TD><TD>mass (ng)</TD><TD>volume (uL)</TD></TR><TR><TD>Backbone</TD><TD>2070</TD><TD>29</TD><TD>1</TD><TD>100.00</TD><TD>3.45</TD><TD>(diluted backbone 10x)</TD></TR><TR><TD>Insert</TD><TD>263</TD><TD>5</TD><TD>3</TD><TD>38.12</TD><TD>7.62</TD><TD>(diluted gblock 10x)</TD></TR><TR><TD>H2O</TD><TD>5.68</TD><TD>uL</TD></TR><TR><TD>Q5 buffer</TD><TD>2.00</TD><TD>uL</TD></TR><TR><TD>dNTPs (2mM)</TD><TD>5.00</TD><TD>uL</TD></TR><TR><TD>Backbone</TD><TD>3.45</TD><TD>uL</TD></TR><TR><TD>gBlock</TD><TD>7.62</TD><TD>uL</TD></TR><TR><TD>DMSO</TD><TD>0.75</TD><TD>uL</TD></TR><TR><TD>Q5 polymerase</TD><TD>0.50</TD><TD>uL</TD></TR></TBODY></TABLE></P></DIV><P>Incubation: 30&quot;@98C;15x(10&quot;@98C;30&quot;@55C;35&quot;@72C);10'@72C.</P></LI></DIV></DIV></DIV><DIV class="panel"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#September">September</A></H4></DIV><DIV id="September" class="panel-collapse  collapse"><UL class="list-group"><LI class="list-group-item borderless"><B>September 01:</B></LI><LI>
Heatshock transformation of the strong expression vector made with CPEC (cf. 31/08) yielded no colonies. 
             </LI><LI>
Colony PCR on the colonies with the strong expression vector made with restriction and ligation didn’t show any positive ones (cf. 31/08). As primers, we used BioBrick verification primers (length: 534 bp). 
             </LI></UL><LI class="list-group-item borderless"><B>September 02:</B><P>Electroshock transformation of the strong expression vector made with CPEC (cf. 31/08) into <I>E. coli</I> Top10 cells.</P></LI><LI class="list-group-item borderless"><B>September 04:</B></LI><LI>
             CPEC using Primestar GXL and an excess of insert. <A data-toggle="collapse" href="#4sept3">Details</A></LI><LI class="list-group-item borderless"><B>Details:</B><P class="center"><TABLE><TBODY><TR><TD>length (bp)</TD><TD>conc. (ng/uL)</TD><TD>ratio</TD><TD>mass (ng)</TD><TD>volume (uL)</TD></TR><TR><TD>Backbone</TD><TD>2070</TD><TD>58</TD><TD>1</TD><TD>100.00</TD><TD>1.72</TD><TD>(diluted backbone 5x)</TD></TR><TR><TD>Insert</TD><TD>263</TD><TD>50</TD><TD>10</TD><TD>127.05</TD><TD>2.54</TD><TD>(undiluted gBlock)</TD></TR><TR><TD>H2O</TD><TD>12.48</TD><TD>uL</TD></TR><TR><TD>GXL buffer</TD><TD>5.00</TD><TD>uL</TD></TR><TR><TD>dNTPs (GXL)</TD><TD>2.00</TD><TD>uL</TD></TR><TR><TD>Backbone</TD><TD>1.72</TD><TD>uL</TD></TR><TR><TD>gBlock</TD><TD>2.54</TD><TD>uL</TD></TR><TR><TD>DMSO</TD><TD>0.75</TD><TD>uL</TD></TR><TR><TD>GXL polymerase</TD><TD>0.50</TD><TD>uL</TD><TD colspan="4">(elongation 1min/kb@68C)</TD></TR></TBODY></TABLE></P></LI></DIV><LI>
Via colony PCR we saw that all 14 colonies with the strong expression vector made with CPEC and electroshock transformation were all positive (cf. 02/09). We used BioBrick verification primers for this. <A data-toggle="collapse" href="#4sept1">Details</A></LI><LI class="list-group-item borderless"><B>Details:</B><P>Conditions: 2'@95C; 30x(20&quot;@95C;30&quot;@55C;1'@68C)</P><P class="center"><TABLE><TBODY><TR><TD>colonies</TD><TD>15</TD><TD>rxns @ 15uL</TD></TR><TR><TD>H2O</TD><TD>181.2</TD><TD>uL</TD></TR><TR><TD>Taq buffer</TD><TD>24</TD><TD>uL</TD></TR><TR><TD>dNTPs (2mM)</TD><TD>24</TD><TD>uL</TD></TR><TR><TD>Fw primer</TD><TD>4.8</TD><TD>uL</TD></TR><TR><TD>Rv primer</TD><TD>4.8</TD><TD>uL</TD></TR><TR><TD>Taq polymerase</TD><TD>1.2</TD><TD>uL</TD></TR></TBODY></TABLE></P></LI></DIV><LI>
We assembled our weak expression vector (pXW), using a gBlock that came in today. Q5 CPEC was used for this with the linearized plasmid backbone pSB1C3, and as insert the gBlock consisting of promoter-RBS-cloning site-terminator-terminator. The promoter is weaker than the promoter of the strong expression vector (cf. 29/08). <I>E. coli</I> Top10 cells were transformed. <A data-toggle="collapse" href="#4sept2">Details</A></LI><LI class="list-group-item borderless"><B>Details:</B><P>Weak expression vector: P<SUB>BBaJ23105</SUB>-RBS<SUB>B0034</SUB>-MCS<SUB>BsaI</SUB>-TT<SUB>B0010</SUB>-TT<SUB>B0012</SUB></P><P>Incubation: 30&quot;@98C;15x(10&quot;@98C;30&quot;@55C;35&quot;@72C);10'@72C</P><P class="center"><TABLE><TBODY><TR><TD>length (bp)</TD><TD>conc. (ng/uL)</TD><TD>ratio</TD><TD>mass (ng)</TD><TD>volume (uL)</TD></TR><TR><TD>Backbone-weak</TD><TD>2070</TD><TD>34.3</TD><TD>1</TD><TD>100.00</TD><TD>2.92</TD><TD>(diluted backbone 10x)</TD></TR><TR><TD>Insert-weak</TD><TD>263</TD><TD>5</TD><TD>3</TD><TD>38.12</TD><TD>7.62</TD><TD>(diluted gblock 10x)</TD></TR><TR><TD>H2O</TD><TD>6.21</TD><TD>uL</TD></TR><TR><TD>Q5 buffer</TD><TD>2.00</TD><TD>uL</TD></TR><TR><TD>dNTPs (2mM)</TD><TD>5.00</TD><TD>uL</TD></TR><TR><TD>Backbone</TD><TD>2.92</TD><TD>uL</TD></TR><TR><TD>gBlock</TD><TD>7.62</TD><TD>uL</TD></TR><TR><TD>DMSO</TD><TD>0.75</TD><TD>uL</TD></TR><TR><TD>Q5 polymerase</TD><TD>0.05</TD><TD>uL</TD></TR></TBODY></TABLE></P></LI></DIV><LI class="list-group-item borderless"><B>September 06:</B><P>Colonies 1 and 2 of our strong expression vector made with CPEC (cf. 04/09) are cultured and miniprepped. These are used for following constructs: </P></LI><LI>pXS-INP_WT using the inaZ PCR fragment </LI><LI>pXS-INP_NC-mGFPuv </LI><LI>pXS-INP_NC-Strep (Strep: regular streptavidin)</LI><LI>pXS-INP_NC-mSA2 (mSA2: monomeric streptavidin)</LI><LI>pXS-Lpp-ompA-mGFPuv </LI><LI>pXS-Lpp-ompA-Strep</LI><LI>pXS-Lpp-ompA-mSA2 </LI>
All constructs are made using Golden Gate assembly. <A data-toggle="collapse" href="#6sept">Details</A><DIV id="6sept" class="panel-collapse  collapse"><UL class="list-group"><LI class="list-group-item borderless"><B>Details:</B><P>Incubation for pXS-INP_WT: 25x(3'@37C;4'@16C); 10'@80C 
                     Incubation for all others: 25x(3'@37C;4'@16C); 5'@50C; 5'@80C
                 </P><P class="center"><TABLE><TBODY><TR><TD>length (bp)</TD><TD>conc. (ng/uL)</TD><TD>ratio</TD><TD>mass (ng)</TD><TD>volume (uL)</TD></TR><TR><TD>pXS</TD><TD>2290</TD><TD>135.6</TD><TD>1</TD><TD>100.00</TD><TD>0.74</TD></TR><TR><TD>inaZ</TD><TD>3625</TD><TD>168</TD><TD>1</TD><TD>158.30</TD><TD>0.94</TD></TR><TR><TD>gBlock 4</TD><TD>1048</TD><TD>50</TD><TD>3</TD><TD>137.29</TD><TD>2.75</TD></TR><TR><TD>gBlock 5</TD><TD>829</TD><TD>50</TD><TD>3</TD><TD>108.60</TD><TD>2.17</TD></TR><TR><TD>gBlock 6_tetra</TD><TD>496</TD><TD>50</TD><TD>3</TD><TD>64.98</TD><TD>1.30</TD></TR><TR><TD>gBlock 6_mono</TD><TD>500</TD><TD>50</TD><TD>3</TD><TD>65.50</TD><TD>1.31</TD></TR><TR><TD>gBlock 7</TD><TD>553</TD><TD>50</TD><TD>3</TD><TD>72.45</TD><TD>1.45</TD></TR><TR><TD><B>Master Mix:</B></TD><TD>7</TD><TD colspan="5">reactions (10% pipetting loss will be taken into account)</TD></TR><TR><TD>H2O</TD><TD>23.97</TD><TD>uL</TD></TR><TR><TD>NEB T4 buffer</TD><TD>11.55</TD><TD>uL</TD></TR><TR><TD>BSA</TD><TD>1.16</TD><TD>uL</TD></TR><TR><TD>BsaI-HF</TD><TD>7.70</TD><TD>uL</TD></TR><TR><TD>NEB T4 ligase</TD><TD>7.70</TD><TD>uL</TD></TR><TR><TD>pXS vector</TD><TD>5.68</TD><TD>uL</TD></TR><TR><TD><B>pXS-INP_WT</B></TD></TR><TR><TD>Master Mix</TD><TD>7.50</TD><TD>uL</TD></TR><TR><TD>inaZ</TD><TD>0.94</TD><TD>uL</TD></TR><TR><TD>H2O</TD><TD>6.56</TD><TD>uL</TD></TR><TR><TD><B>pXS-INP_NC-mGFPuv</B></TD><TD><B>pXS-INP_NC-Strep</B></TD></TR><TR><TD>Master Mix</TD><TD>7.50</TD><TD>uL</TD><TD>Master Mix</TD><TD>7.50</TD><TD>uL</TD></TR><TR><TD>gBlock 4</TD><TD>2.75</TD><TD>uL</TD><TD>gBlock 4</TD><TD>2.75</TD><TD>uL</TD></TR><TR><TD>gBlock 5</TD><TD>2.17</TD><TD>uL</TD><TD>gBlock 6_tetra</TD><TD>1.30</TD><TD>uL</TD></TR><TR><TD>H2O</TD><TD>2.58</TD><TD>uL</TD><TD>H2O</TD><TD>3.45</TD><TD>uL</TD></TR><TR><TD><B>pXS-INP_NC-mSA2</B></TD><TD><B>pXS-Lpp-ompA-mGFPuv</B></TD></TR><TR><TD>Master Mix</TD><TD>7.50</TD><TD>uL</TD><TD>Master Mix</TD><TD>7.50</TD><TD>uL</TD></TR><TR><TD>gBlock 4</TD><TD>2.75</TD><TD>uL</TD><TD>gBlock 5</TD><TD>2.17</TD><TD>uL</TD></TR><TR><TD>gBlock 6_mono</TD><TD>1.31</TD><TD>uL</TD><TD>gBlock 7</TD><TD>1.45</TD><TD>uL</TD></TR><TR><TD>H2O</TD><TD>3.44</TD><TD>uL</TD><TD>H2O</TD><TD>3.88</TD><TD>uL</TD></TR><TR><TD><B>pXS-Lpp-ompA-Strep</B></TD><TD><B>pXS-Lpp-ompA-mSA2</B></TD></TR><TR><TD>Master Mix</TD><TD>7.50</TD><TD>uL</TD><TD>Master Mix</TD><TD>7.50</TD><TD>uL</TD></TR><TR><TD>gBlock 6_tetra</TD><TD>1.30</TD><TD>uL</TD><TD>gBlock 6_mono</TD><TD>1.31</TD><TD>uL</TD></TR><TR><TD>gBlock 7</TD><TD>1.45</TD><TD>uL</TD><TD>gBlock 7</TD><TD>1.45</TD><TD>uL</TD></TR><TR><TD>H2O</TD><TD>4.75</TD><TD>uL</TD><TD>H2O</TD><TD>4.74</TD><TD>uL</TD></TR></TBODY></TABLE></P></LI></UL></DIV><LI class="list-group-item borderless"><B>September 07:</B><P>We transformed our Golden Gate reactions (cf. 06/08) into <I>E. coli</I> Top10 cells.</P></LI><LI class="list-group-item borderless"><B>September 08:</B></LI><LI>
We did a colony PCR of our weak expression vector (cf. 04/09), and of our constructs made with the strong expression vector (cf. 06/09), both using BioBrick verification primers. <A data-toggle="collapse" href="#8sept1">Details</A></LI><LI class="list-group-item borderless"><B>Details:</B><P>
                 Expected length (bp) of the constructs made with the strong expression vector:
                   </P></LI><LI>pXS-inaZ: 4115</LI><LI>pXS-INP_NC-mGFPuv: 2166</LI><LI>pXS-INP_NC-Strep: 1857</LI><LI>pXS-INP_NC-mSA2: 1832</LI><LI>pXS-lpp-ompA-mGFPuv: 1670</LI><LI>pXS-lpp-ompA-Strep: 1362</LI><LI>pXS-lpp-ompA-mSA2: 1338</LI><P>Conditions weak expression vector: 2'@95C; 30x(20&quot;@95C;30&quot;@55C;1'@68C)</P><P class="center"><TABLE><TBODY><TR><TD>colonies</TD><TD>14</TD><TD>rxns @ 15uL</TD></TR><TR><TD>H2O</TD><TD>174.405</TD><TD>uL</TD></TR><TR><TD>Taq buffer</TD><TD>23.1</TD><TD>uL</TD></TR><TR><TD>dNTPs (2mM)</TD><TD>23.1</TD><TD>uL</TD></TR><TR><TD>Fw primer</TD><TD>4.62</TD><TD>uL</TD></TR><TR><TD>Rv primer</TD><TD>4.62</TD><TD>uL</TD></TR><TR><TD>Taq polymerase</TD><TD>1.155</TD><TD>uL</TD></TR></TBODY></TABLE></P></DIV><LI>
We tried to make inaZ Golden Gate safe by removing the internal BsaI sites. To achieve this, we used Primestar HS PCR and cut it into 3 different fragments. This was successful for the separate fragments, but we didn’t succeed to combine them. For the first fragment, this was simply from the original PCR fragment. The second fragment is a new PCR fragment, using oM7658 and oM7629 as primers and the inaZ plasmid as template (length:3039 bp). This band is smeared, probably due to excessive template concentration or annealing time. The third fragment is also a new PCR fragment, using oM7658 and oM7229 as primers and the inaZ plasmid as template (length: 293 bp). 
</LI><LI class="list-group-item borderless"><B>September 09:</B><P>The colony PCR of <I>E. coli</I> cells with the constructs with the strong expression vector showed no positive colonies (cf. 06/09). Not many bands are present, and all present bands show the original vector or self ligation. The colony PCR on our cells transformed with the weak expression vector (cf. 04/09) did show positive ones. We repeated the colony PCR under different conditions, but bands were still not of the correct lengths. 
We therefore tried cloning all our parts into the weak expression vector using Golden Gate. These reaction mixes were then transformed into <I>E. coli</I> Top10 cells. Incubation was done at 30°C instead of 37°C to repress the plasmid copy number. <A data-toggle="collapse" href="#9sept">Details</A></P></LI><LI class="list-group-item borderless"><B>Details:</B><P class="center"><TABLE><TBODY><TR><TD>length (bp)</TD><TD>conc. (ng/uL)</TD><TD>ratio</TD><TD>mass (ng)</TD><TD>volume (uL)</TD></TR><TR><TD>pXW</TD><TD>2290</TD><TD>59.8</TD><TD>1</TD><TD>100.00</TD><TD>1.67</TD></TR><TR><TD>gBlock 4</TD><TD>1048</TD><TD>50</TD><TD>3</TD><TD>137.29</TD><TD>2.75</TD></TR><TR><TD>gBlock 5</TD><TD>829</TD><TD>50</TD><TD>3</TD><TD>108.60</TD><TD>2.17</TD></TR><TR><TD>gBlock 6_tetra</TD><TD>496</TD><TD>50</TD><TD>3</TD><TD>64.98</TD><TD>1.30</TD></TR><TR><TD>gBlock 6_mono</TD><TD>500</TD><TD>50</TD><TD>3</TD><TD>65.50</TD><TD>1.31</TD></TR><TR><TD>gBlock 7</TD><TD>553</TD><TD>50</TD><TD>3</TD><TD>72.45</TD><TD>1.45</TD></TR><TR><TD><B>Master Mix:</B></TD><TD>7</TD><TD colspan="5">reactions (10% pipetting loss will be taken into account)</TD></TR><TR><TD>H2O</TD><TD>16.77</TD><TD>uL</TD></TR><TR><TD>NEB T4 buffer</TD><TD>11.55</TD><TD>uL</TD></TR><TR><TD>BSA</TD><TD>1.16</TD><TD>uL</TD></TR><TR><TD>BsaI-HF</TD><TD>7.70</TD><TD>uL</TD></TR><TR><TD>NEB T4 ligase</TD><TD>7.70</TD><TD>uL</TD></TR><TR><TD>pXW vector (k1)</TD><TD>12.88</TD><TD>uL</TD></TR><TR><TD><B>negative control</B></TD></TR><TR><TD>Master Mix</TD><TD>7.5</TD><TD>uL</TD></TR><TR><TD>H2O</TD><TD>7.5</TD><TD>uL</TD></TR><TR><TD><B>2) pXW-INP_NC-mGFPuv</B></TD><TD><B>3) pXW-INP_NC-Strep</B></TD></TR><TR><TD>Master Mix</TD><TD>7.50</TD><TD>uL</TD><TD>Master Mix</TD><TD>7.50</TD><TD>uL</TD></TR><TR><TD>gBlock 4</TD><TD>2.75</TD><TD>uL</TD><TD>gBlock 4</TD><TD>2.75</TD><TD>uL</TD></TR><TR><TD>gBlock 5</TD><TD>2.17</TD><TD>uL</TD><TD>gBlock 6_tetra</TD><TD>1.30</TD><TD>uL</TD></TR><TR><TD>H2O</TD><TD>2.58</TD><TD>uL</TD><TD>H2O</TD><TD>3.45</TD><TD>uL</TD></TR><TR><TD><B>4) pXW-INP_NC-mSA2</B></TD><TD><B>5) pXW-Lpp-ompA-mGFPuv</B></TD></TR><TR><TD>Master Mix</TD><TD>7.50</TD><TD>uL</TD><TD>Master Mix</TD><TD>7.50</TD><TD>uL</TD></TR><TR><TD>gBlock 4</TD><TD>2.75</TD><TD>uL</TD><TD>gBlock 5</TD><TD>2.17</TD><TD>uL</TD></TR><TR><TD>gBlock 6_mono</TD><TD>1.31</TD><TD>uL</TD><TD>gBlock 7</TD><TD>1.45</TD><TD>uL</TD></TR><TR><TD>H2O</TD><TD>3.44</TD><TD>uL</TD><TD>H2O</TD><TD>3.88</TD><TD>uL</TD></TR><TR><TD><B>6) pXW-Lpp-ompA-Strep</B></TD><TD><B>7) pXW-Lpp-ompA-mSA2</B></TD></TR><TR><TD>Master Mix</TD><TD>7.50</TD><TD>uL</TD><TD>Master Mix</TD><TD>7.50</TD><TD>uL</TD></TR><TR><TD>gBlock 6_tetra</TD><TD>1.30</TD><TD>uL</TD><TD>gBlock 6_mono</TD><TD>1.31</TD><TD>uL</TD></TR><TR><TD>gBlock 7</TD><TD>1.45</TD><TD>uL</TD><TD>gBlock 7</TD><TD>1.45</TD><TD>uL</TD></TR><TR><TD>H2O</TD><TD>4.75</TD><TD>uL</TD><TD>H2O</TD><TD>4.74</TD><TD>uL</TD></TR></TBODY></TABLE></P></LI><LI class="list-group-item borderless"><B>September 10:</B></LI><LI>
We got our sequencing results: the strong expression vector is good! 
             </LI><LI>
Colonies with the weak expression vector constructs are still too small to pick up with colony PCR (cf. 09/09).
             </LI><LI class="list-group-item borderless"><B>September 11:</B></LI><LI>
All our plates with the weak expression vector constructs show colonies! The control reaction has colonies as well, but much fewer, which was to be expected. Fluorescence is seen for the pXW-INP_NC-mGFPuv construct. All Lpp-ompA plates have a mixed phenotype, with 10-20 large colonies (same size as control reaction), and &gt;60 small colonies (see culture picture). Colony PCR was performed using OneTaq at 50°C annealing temperature to proof that the small colonies are the positives ones, the larger ones the negative ones. This theory was confirmed for all colonies (see gel picture). Keep incubating at 30°C to reduce copy number.</LI><LI>
We performed Primestar HS PCR to create following constructs:
               </LI><LI>pXW-Lpp-ompA-mGFPuv-(m)Strep using Lpp-ompA-mGFPuv fragment. As a template, we used the Golden Gate mix for pXW-Lpp-ompA-mGFPuv, and oM7667_Lpp_Fw and oM7663_GFP_Rv-ATCT as primers (expected length:1167). </LI><LI>pXW-mGFPuv-m(Strep) using mGFP. As a template, we used a 100x diluted gBlock, and oM7662_GFP_Fw-GATG and oM7663_GFP_Rv-ATCT as primers (expected length: 752 bp). </LI><LI>pXS-mGFPuv using mGFP. As a template, we used a 100x diluted gBlock, and oM7662_GFP_Fw-GATG and oM7664_GFP_Rv-TACT as primers (expected length: 752 bp).</LI>
We also repeated previous PCR (cf. 08/09) of the largest piece of <I>inaZ</I> to make it Golden Gate safe. As a template, we used the <I>inaZ</I> plasmid, and oM7658 and oM7629 as primers (expected length: 3039 bp). Because the band was smeared last time, we should either use lower template concentrations or a shorter anneal time.
             <LI class="list-group-item borderless"><B>September 12:</B></LI><LI>
Miniprep of a number of colony PCR positive colonies (cf. 09/09).
             </LI><LI>
Next picture is the gel of the PCR reactions of the previous day (cf. 11/09). inaZ was run 2 times (at 1/100 and 1/1000 dilution), these are the first 2 bands. The next 3 bands are the 3 constructs. PCR using the Golden Gate mix as a template didn’t work very well (two extra bands, lane 3). We therefore repeated this reaction using a pure plasmid.
</LI><LI>
Golden Gate assemblies of:
               </LI><LI>pXW-inaZ_GGsafe</LI><LI>pXW-mGFPuv-Strep</LI><LI>pXW-mGFPuv-mSA2</LI><LI>pXW-mGFPuv</LI><LI>pXS-mGFPuv </LI><A data-toggle="collapse" href="#12sept">Details</A><DIV id="12sept" class="panel-collapse  collapse"><UL class="list-group"><LI class="list-group-item borderless"><B>Details:</B><P class="center"><TABLE><TBODY><TR><TD>length (bp)</TD><TD>conc. (ng/uL)</TD><TD>ratio</TD><TD>mass (ng)</TD><TD>volume (uL)</TD></TR><TR><TD>pXS</TD><TD>2290</TD><TD>106.6</TD><TD>1</TD><TD>100.00</TD><TD>0.94</TD></TR><TR><TD>pXW</TD><TD>2290</TD><TD>59.8</TD><TD>1</TD><TD>100.00</TD><TD>1.67</TD></TR><TR><TD>INP full</TD><TD>3600</TD><TD>168.5</TD><TD>1</TD><TD>157.21</TD><TD>0.93</TD></TR><TR><TD>INP part 2</TD><TD>3039</TD><TD>206.6</TD><TD>1</TD><TD>132.71</TD><TD>0.64</TD></TR><TR><TD>INP part 3</TD><TD>293</TD><TD>58</TD><TD>3</TD><TD>38.38</TD><TD>0.66</TD></TR><TR><TD>mGFPuv-ATCT</TD><TD>752</TD><TD>210.1</TD><TD>3</TD><TD>98.52</TD><TD>0.47</TD></TR><TR><TD>mGFPuv-TACT</TD><TD>752</TD><TD>275.7</TD><TD>3</TD><TD>98.52</TD><TD>0.36</TD></TR><TR><TD>gBlock 6_tetra</TD><TD>496</TD><TD>50</TD><TD>3</TD><TD>64.98</TD><TD>1.30</TD></TR><TR><TD>gBlock 6_mono</TD><TD>500</TD><TD>50</TD><TD>3</TD><TD>65.50</TD><TD>1.31</TD></TR><TR><TD><B>Master Mix:</B></TD><TD>5</TD><TD colspan="5">reactions (10% pipetting loss will be taken into account)</TD></TR><TR><TD>H2O</TD><TD>34.93</TD><TD>uL</TD></TR><TR><TD>NEB T4 buffer</TD><TD>8.25</TD><TD>uL</TD></TR><TR><TD>BSA</TD><TD>0.83</TD><TD>uL</TD></TR><TR><TD>BsaI-HF</TD><TD>5.50</TD><TD>uL</TD></TR><TR><TD>NEB T4 ligase</TD><TD>5.50</TD><TD>uL</TD></TR><TR><TD><B>pXW-INP_GGsafe (8)</B></TD><TD><B>pXW-mGFPuv-Strep (9)</B></TD></TR><TR><TD>Master Mix</TD><TD>10.00</TD><TD>uL</TD><TD>Master Mix</TD><TD>10.00</TD><TD>uL</TD></TR><TR><TD>INP full</TD><TD>0.93</TD><TD>uL</TD><TD>mGFPuv-ATCT</TD><TD>0.47</TD><TD>uL</TD></TR><TR><TD>INP part 2</TD><TD>0.64</TD><TD>uL</TD><TD>gBlock 6_tetra</TD><TD>1.30</TD><TD>uL</TD></TR><TR><TD>INP part 3</TD><TD>0.66</TD><TD>uL</TD><TD>pXW</TD><TD>1.67</TD><TD>uL</TD></TR><TR><TD>pXW</TD><TD>1.67</TD><TD>uL</TD><TD>H2O</TD><TD>1.56</TD><TD>uL</TD></TR><TR><TD>H2O</TD><TD>1.09</TD><TD>uL</TD></TR><TR><TD><B>pXW-mGFPuv-mSA2 (10)</B></TD><TD><B>pXW-mGFPuv (11)</B></TD></TR><TR><TD>Master Mix</TD><TD>10.00</TD><TD>uL</TD><TD>Master Mix</TD><TD>10.00</TD><TD>uL</TD></TR><TR><TD>mGFPuv-ATCT</TD><TD>1.30</TD><TD>uL</TD><TD>mGFPuv-TACT</TD><TD>0.36</TD><TD>uL</TD></TR><TR><TD>gBlock 6_mono</TD><TD>1.31</TD><TD>uL</TD><TD>pXW</TD><TD>1.67</TD><TD>uL</TD></TR><TR><TD>pXW</TD><TD>1.67</TD><TD>uL</TD><TD>H2O</TD><TD>2.97</TD><TD>uL</TD></TR><TR><TD>H2O</TD><TD>0.72</TD><TD>uL</TD></TR><TR><TD colspan="3">note: mistakenly used 1.3uL of mGFPuv-TACT</TD></TR><TR><TD><B>pXS-mGFPuv (12)</B></TD></TR><TR><TD>Master Mix</TD><TD>10.00</TD><TD>uL</TD></TR><TR><TD>mGFPuv-TACT</TD><TD>0.36</TD><TD>uL</TD></TR><TR><TD>pXS</TD><TD>0.94</TD><TD>uL</TD></TR><TR><TD>H2O</TD><TD>3.70</TD><TD>uL</TD></TR></TBODY></TABLE></P></LI></UL></DIV><LI class="list-group-item borderless"><B>September 13:</B></LI><LI>
pXW-Lpp-ompA-mGFPuv miniprep tubes that did not grow yesterday (cf. 12/09) did show growth today, probably through mutations.
             </LI><LI>
Transformed Golden Gate assemblies (cf. 12/09) into <I>E. coli</I> Top10 cells.
             </LI><LI class="list-group-item borderless"><B>September 14:</B><P>Colony PCR of the transformed Golden Gate assemblies (cf. 13/09): fail for pXW-INP_GGsafe (first 7 lanes), okay for other assemblies. For pXW-INP_GGsafe, we will make a new Golden Gate design. </P></LI><LI class="list-group-item borderless"><B>September 15:</B><P>Colony PCR screening of 8 more colonies with pXW-inaZ_GGsafe showed no result (cf. 13/09)</P></LI><LI class="list-group-item borderless"><B>September 16:</B></LI><LI>
Minipreps of the following plasmids and sequencing with verification primers: 
               </LI><LI>pXW-mGFPuv-Strep</LI><LI>pXW-mGFPuv-mSA2</LI><LI>pXW-mGFPuv</LI><LI>pXS-mGFPuv</LI><LI>
We did a Primestar HS PCR to go from high copy to low copy (LC) backbone, with as backbone pSB3K3 and 4 inserts: INP_NC-Strep, INP_NC-mSA2, Lpp-ompA-Strep and Lpp-ompA-mSA2. The backbone and Lpp-ompA-mSA2 both failed. On the gel, all four inserts are seen. The backbone was run on a different gel (not pictured here). <A data-toggle="collapse" href="#16sept2">Details</A></LI><LI class="list-group-item borderless"><B>Details:</B><P>
                 Backbone: pSB3K3 with pSB3K3 as template (plate4, F6, iGem distribution 2016), and oM7669 &amp; oM 7670 as primers (length: 2790 bp). 
                 Insert1: INP_NC-Strep (156 ng/ul) with pXW-INP_NC-Strep as template, and oM7671 &amp; oM 7672 as primers (length: 1684 bp). 
                 Insert2:  INP_NC-mSA2 (290 ng/ul) with pXW-INP_NC-mSA2 as template and oM7671 &amp; oM 7672 as primers. 
                 Insert3: Lpp-ompA-Strep (130 ng/ul) with  pXW-Lpp-ompA-Strep as template, and oM7671 &amp; oM 7672 as primers. 
                 Insert4: Lpp-ompA-mSA2 with pXW-Lpp-ompA-mSA2 as template, and oM7671 &amp; oM 7672 as primers.
                </P></LI><LI class="list-group-item borderless"><B>September 19:</B></LI><LI>
We retried the Primestar HS PCR to go from high to low copy backbone. The backbone failed again,  Lpp-ompA-mSA2 was successful. We also made the ‘new parts’ of inaZ, was was also successful. <A data-toggle="collapse" href="#19sept2">Details</A></LI><LI class="list-group-item borderless"><B>Details:</B><P>
                  Retry backbone: used  pSB3K3 as template, and oM7675 &amp; oM7644 as new primers (length: 2790 bp). 
                  Retry insert4 (Lpp-ompA-mSA2): used Golden Gate mix as template (134 ng/ul), and oM7671 &amp; oM7672 as primers. 
                  New parts inaZ:  inaZ plasmid as template, and oM7628 &amp; oM7673 as primers for piece1 (150 ng/ul, 634 bp), oM7658 &amp; oM7674 as primers for piece2 (146 ng/ul, 2792 bp), and oM7659 &amp; oM7629 as primers for piece3 (122 ng/ul, 297 bp).
                  </P></LI><LI>
Golden gate assembly of pXW-inaZ_GGsafe. <A data-toggle="collapse" href="#19sept">Details</A></LI><LI class="list-group-item borderless"><B>Details:</B><P class="center"><TABLE><TBODY><TR><TD colspan="3"><B>pXW-INP_GGsafe   (NEW parts!)</B></TD></TR><TR><TD>Mastermix</TD><TD>4.50</TD><TD>uL</TD></TR><TR><TD>INP part 1</TD><TD>0.63</TD><TD>uL</TD></TR><TR><TD>INP part 2</TD><TD>1.42</TD><TD>uL</TD></TR><TR><TD>INP part 3</TD><TD>0.55</TD><TD>uL</TD></TR><TR><TD>pXW</TD><TD>1.67</TD><TD>uL</TD></TR><TR><TD>H2O</TD><TD>11.23</TD><TD>uL</TD></TR></TBODY></TABLE></P></LI><LI class="list-group-item borderless"><B>September 21:</B></LI><LI>
We did a colony PCR for the Golden Gate assembly of pXW-inaZ_GGsafe (cf. 19/09). Colony 2, 6, 8 are okay. 
</LI><LI>
We opted for a new backbone for the low copy vector (cf. 19/09): pSC101-Kan vector.
             </LI><LI class="list-group-item borderless"><B>September 22:</B></LI><LI>
We digested colony PCR samples 2, 6, 8, 13, and 32 (cf. 21/09), and a control with BsaI. If pXW-inaZ_GGsafe is indeed Golden Gate safe, it should not be cut. This was okay! Colony 2 and 6 are plasmid prepped for sequencing. Colony 13 is lane 5, control is lane 6.
</LI><LI>
We again retried the Primestar HS PCR to go from high to low copy backbone. For this, we have the low copy backbone pSC101 (right gel, lane 1). A new Lpp-ompA-mSA2 insert was also done to obtain an insert for the low copy vector (left gel, lane 1). A weak expression vector with an empty expression site was also added (left gel, lane 2), and also inaZ-RC. Lane 2 and 3 on the right gel are respectively a control, and backbone pSB1C3 (mSA2-REsafe gBlock assembly). <A data-toggle="collapse" href="#22sept">Details</A></LI><LI class="list-group-item borderless"><B>Details:</B><P>
                  New Backbone &quot;LC&quot; (Low Copy, pSC101 ORI):  pSC101-Kan vector as template, and oM7714 &amp; oM7715 as primers (length: 4034 bp). 
                  New Lpp-ompA-Strep insert: Golden Gate mix as template, and oM7671 &amp; oM 7672 as primers (1051 bp). 
                  XW empty expression site: pXW as template, and oM7671 &amp; oM7672 as primers (length: 224 bp).  
                  inaZ-RC: pXW-inaZ_GGsafe as template, and oM7665 &amp; oM7666 as primers (length: 3100 bp).
                 </P></LI><LI class="list-group-item borderless"><B>September 26:</B><P>CPEC assemblies to create:</P></LI><LI>pLC-XW</LI><LI>pLC-XW-INP_NC-Strep</LI><LI>pLC-XW-INP_NC-mSA2</LI><LI>pLC-XW-Lpp-ompA-Strep</LI><LI>pLC-XW-Lpp-ompA-mSA2</LI><LI>pSB1C3-mSA2</LI><A data-toggle="collapse" href="#26sept">Details</A><DIV id="26sept" class="panel-collapse  collapse"><UL class="list-group"><LI class="list-group-item borderless"><B>Details:</B><P class="center"><TABLE><TBODY><TR><TD>length (bp)</TD><TD>conc. (ng/uL)</TD><TD>ratio</TD><TD>mass (ng)</TD><TD>volume (uL)</TD></TR><TR><TD>pLC backbone</TD><TD>2790</TD><TD>322</TD><TD>1</TD><TD>100.00</TD><TD>0.31</TD></TR><TR><TD>XW insert</TD><TD>224</TD><TD>26.94</TD><TD>3</TD><TD>24.09</TD><TD>0.89</TD><TD>5x verdund</TD></TR><TR><TD>INP_NC-Strep</TD><TD>1546</TD><TD>156</TD><TD>3</TD><TD>166.24</TD><TD>1.07</TD></TR><TR><TD>INP_NC-mSA2</TD><TD>1522</TD><TD>290</TD><TD>3</TD><TD>163.66</TD><TD>0.56</TD></TR><TR><TD>Lpp-ompA-Strep</TD><TD>1051</TD><TD>249.6</TD><TD>3</TD><TD>113.01</TD><TD>0.45</TD></TR><TR><TD>Lpp-ompA-mSA2</TD><TD>1027</TD><TD>134</TD><TD>3</TD><TD>110.43</TD><TD>0.82</TD></TR><TR><TD><B>Master Mix:</B></TD><TD>5</TD><TD colspan="5">reactions (10% pipetting loss will be taken into account)</TD></TR><TR><TD>H2O</TD><TD>62.92</TD><TD>uL</TD></TR><TR><TD>Q5 buffer</TD><TD>11.00</TD><TD>uL</TD></TR><TR><TD>dNTPs (2mM)</TD><TD>27.50</TD><TD>uL</TD></TR><TR><TD>pLC backbone</TD><TD>1.71</TD><TD>uL</TD></TR><TR><TD>DMSO</TD><TD>4.13</TD><TD>uL</TD></TR><TR><TD>Q5 polymerase</TD><TD>2.75</TD><TD>uL</TD></TR><TR><TD><B>pLC-XW (13)</B></TD><TD><B>pLC-XW-INP_NC-Strep (14)</B></TD></TR><TR><TD>H2O</TD><TD>4.11</TD><TD>uL</TD><TD>H2O</TD><TD>3.93</TD><TD>uL</TD></TR><TR><TD>XW insert</TD><TD>0.89</TD><TD>uL</TD><TD>INP_NC-Strep</TD><TD>1.07</TD><TD>uL</TD></TR><TR><TD><B>pLC-XW-INP_NC-mSA2 (15)</B></TD><TD><B>pLC-XW-Lpp-ompA-Strep (16)</B></TD></TR><TR><TD>H2O</TD><TD>4.44</TD><TD>uL</TD><TD>H2O</TD><TD>4.55</TD><TD>uL</TD></TR><TR><TD>INP_NC-mSA2</TD><TD>0.56</TD><TD>uL</TD><TD>Lpp-ompA-Strep</TD><TD>0.45</TD><TD>uL</TD></TR><TR><TD><B>pLC-XW-Lpp-ompA-mSA2 (17)</B></TD><TD><B>incubation:</B></TD></TR><TR><TD>H2O</TD><TD>4.18</TD><TD>uL</TD><TD colspan="4">30&quot;@98C; 15x(10&quot;@98C;30&quot;@55C;1'15&quot;@72C); 10'@72C</TD></TR><TR><TD>Lpp-ompA-mSA2</TD><TD>0.82</TD><TD>uL</TD></TR></TBODY></TABLE></P><P class="center"><TABLE><TBODY><TR><TD>length (bp)</TD><TD>conc. (ng/uL)</TD><TD>ratio</TD><TD>mass (ng)</TD><TD>volume (uL)</TD></TR><TR><TD>pSB1C3 backbone</TD><TD>2044</TD><TD>375</TD><TD>1</TD><TD>100.00</TD><TD>0.27</TD></TR><TR><TD>gBlock mSA2</TD><TD>475</TD><TD>50</TD><TD>3</TD><TD>69.72</TD><TD>1.39</TD></TR><TR><TD><B>Mix: (18)</B></TD><TD><B>incubation:</B></TD></TR><TR><TD>H2O</TD><TD>15.09</TD><TD>uL</TD><TD colspan="4">30&quot;@98C; 15x(10&quot;@98C;30&quot;@55C;1'15&quot;@72C); 10'@72C</TD></TR><TR><TD>Q5 buffer</TD><TD>2.00</TD><TD>uL</TD></TR><TR><TD>dNTPs (2mM)</TD><TD>5.00</TD><TD>uL</TD></TR><TR><TD>pSB1C3 backbone</TD><TD>0.27</TD><TD>uL</TD></TR><TR><TD>gBlock mSA2</TD><TD>1.39</TD><TD>uL</TD></TR><TR><TD>DMSO</TD><TD>0.75</TD><TD>uL</TD></TR><TR><TD>Q5 polymerase</TD><TD>0.5</TD><TD>uL</TD></TR></TBODY></TABLE></P></LI></UL></DIV><LI class="list-group-item borderless"><B>September 27:</B><P>Colony PCR of all CPEC assemblies (cf. 26/09) with primers oM3441 + oM1760 for the first 5 assemblies and verification primers for the last assembly. The order of the gel is: pLC-XW-INP_NC-Strep, pLC-XW-INP_NC-mSA2, pLC-XW-Lpp-ompA-Strep, pLC-XW-Lpp-ompA-mSA2, pSB1C3-mSA2, and pLC-XW (only 3 colonies). </P></LI><LI class="list-group-item borderless"><B>September 28:</B></LI><LI>
Second attempt for CPEC assemblies (cf. 26/09), with increased backbone DNA (200 ng) and an altered backbone/insert ratio (1:5). <A data-toggle="collapse" href="#28sept1">Details</A></LI><LI class="list-group-item borderless"><B>Details:</B><P class="center"><TABLE><TBODY><TR><TD>length (bp)</TD><TD>conc. (ng/uL)</TD><TD>ratio</TD><TD>mass (ng)</TD><TD>volume (uL)</TD></TR><TR><TD>pLC backbone</TD><TD>2790</TD><TD>322</TD><TD>1</TD><TD>20.000</TD><TD>0.62</TD></TR><TR><TD>XW insert</TD><TD>224</TD><TD>134.7</TD><TD>5</TD><TD>80.29</TD><TD>0.60</TD></TR><TR><TD>INP_NC-Strep</TD><TD>1546</TD><TD>156</TD><TD>5</TD><TD>554.12</TD><TD>3.55</TD></TR><TR><TD>INP_NC-mSA2</TD><TD>1522</TD><TD>290</TD><TD>5</TD><TD>545.52</TD><TD>1.88</TD></TR><TR><TD>Lpp-ompA-Strep</TD><TD>1051</TD><TD>249.6</TD><TD>5</TD><TD>376.70</TD><TD>1.51</TD></TR><TR><TD>Lpp-ompA-mSA2</TD><TD>1027</TD><TD>134</TD><TD>5</TD><TD>368.10</TD><TD>2.75</TD></TR><TR><TD><B>Master Mix:</B></TD><TD>5</TD><TD colspan="5">reactions (10% pipetting loss will be taken into account)</TD></TR><TR><TD>H2O</TD><TD>61.21</TD><TD>uL</TD></TR><TR><TD>Q5 buffer</TD><TD>11.00</TD><TD>uL</TD></TR><TR><TD>dNTPs (2mM)</TD><TD>27.50</TD><TD>uL</TD></TR><TR><TD>pLC backbone</TD><TD>3.42</TD><TD>uL</TD></TR><TR><TD>DMSO</TD><TD>4.13</TD><TD>uL</TD></TR><TR><TD>Q5 polymerase</TD><TD>2.75</TD><TD>uL</TD></TR><TR><TD><B>pLC-XW (13*)</B></TD><TD><B>pLC-XW-INP_NC-Strep (14*)</B></TD></TR><TR><TD>H2O</TD><TD>4.40</TD><TD>uL</TD><TD>H2O</TD><TD>1.45</TD><TD>uL</TD></TR><TR><TD>XW insert</TD><TD>0.60</TD><TD>uL</TD><TD>INP_NC-Strep</TD><TD>3.55</TD><TD>uL</TD></TR><TR><TD><B>pLC-XW-INP_NC-mSA2 (15*)</B></TD><TD><B>pLC-XW-Lpp-ompA-Strep (16*)</B></TD></TR><TR><TD>H2O</TD><TD>3.12</TD><TD>uL</TD><TD>H2O</TD><TD>3.49</TD><TD>uL</TD></TR><TR><TD>INP_NC-mSA2</TD><TD>1.88</TD><TD>uL</TD><TD>Lpp-ompA-Strep</TD><TD>1.51</TD><TD>uL</TD></TR><TR><TD><B>pLC-XW-Lpp-ompA-mSA2 (17*)</B></TD><TD><B>incubation:</B></TD></TR><TR><TD>H2O</TD><TD>2.25</TD><TD>uL</TD><TD colspan="4">30&quot;@98C; 15x(10&quot;@98C;30&quot;@55C;1'15&quot;@72C); 10'@72C</TD></TR><TR><TD>Lpp-ompA-mSA2</TD><TD>2.75</TD><TD>uL</TD></TR></TBODY></TABLE></P></LI><LI>
Primestar HS PCR to create GATG-mGFPuv-ATCT, which is necessary for new attempts for the pXW-mGFPuv-Strep and pXW-mGFPuv-mSA2  constructs, due to mutations in previous versions. For this, we used primers oM7662 + oM7663, and pXS-mGFPuv as template.
</LI><LI>
Golden Gate assemblies to create:
     </LI><LI>pXW-inaZ-RC+Strep</LI><LI>pXW-inaZ-RC+mSA2</LI><LI>pXW-inaZ-RC+GFP</LI><LI>pXW-mGFPuv-Strep</LI><LI>pXS-mGFPuv-Strep</LI><LI>pXS-mGFPuv-mSA2</LI><A data-toggle="collapse" href="#28sept2">Details</A><DIV id="28sept2" class="panel-collapse  collapse"><UL class="list-group"><LI class="list-group-item borderless"><B>Details:</B><P class="center"><TABLE><TBODY><TR><TD><B>pXW-inaZ-RC+Strep (18)</B></TD><TD><B>pXW-inaZ-RC+mSA2 (19)</B></TD></TR><TR><TD>Mastermix</TD><TD>12.17</TD><TD>uL</TD><TD>Mastermix</TD><TD>12.17</TD><TD>uL</TD></TR><TR><TD>inaZ-RC</TD><TD>0.80</TD><TD>uL</TD><TD>inaZ-RC</TD><TD>0.80</TD><TD>uL</TD></TR><TR><TD>gBlock_6-tetra</TD><TD>1.30</TD><TD>uL</TD><TD>gBlock_6-mono</TD><TD>1.30</TD><TD>uL</TD></TR><TR><TD>H2O</TD><TD>5.73</TD><TD>uL</TD><TD>H2O</TD><TD>5.73</TD><TD>uL</TD></TR><TR><TD><B>pXW-inaZ-RC+GFP (20)</B></TD><TD><B>MasterMiX</B></TD></TR><TR><TD>Mastermix</TD><TD>12.17</TD><TD>uL</TD><TD>ligase buffer</TD><TD>6</TD></TR><TR><TD>inaZ-RC</TD><TD>0.80</TD><TD>uL</TD><TD>ligase</TD><TD>3</TD></TR><TR><TD>gBlock 5</TD><TD>2.17</TD><TD>uL</TD><TD>BsaI</TD><TD>3</TD></TR><TR><TD>H2O</TD><TD>4.86</TD><TD>uL</TD><TD>BSA</TD><TD>1.5</TD></TR><TR><TD>H20</TD><TD>18</TD></TR><TR><TD>pXW</TD><TD>5.01</TD></TR><TR><TD><B>36.51</B></TD></TR></TBODY></TABLE></P><P class="center"><TABLE><TBODY><TR><TD><B>MasterMix:</B></TD><TD>3</TD></TR><TR><TD>ligase buffer</TD><TD>6</TD></TR><TR><TD>ligase</TD><TD>3</TD></TR><TR><TD>BsaI</TD><TD>3</TD></TR><TR><TD>BSA</TD><TD>1.5</TD></TR><TR><TD>H20</TD><TD>18</TD></TR><TR><TD><B>31.5</B></TD></TR><TR><TD><B>pXW-mGFPuv-Strep (21 (old 9))</B></TD><TD><B>pXS-mGFPuv-Strep (22)</B></TD></TR><TR><TD>Mastermix</TD><TD>10.50</TD><TD>uL</TD><TD>Mastermix</TD><TD>10.50</TD><TD>uL</TD></TR><TR><TD>mGFPuv-ATCT</TD><TD>0.37</TD><TD>uL</TD><TD>mGFPuv-ATCT</TD><TD>0.37</TD><TD>uL</TD></TR><TR><TD>gBlock_6-tetra</TD><TD>1.30</TD><TD>uL</TD><TD>gBlock_6-tetra</TD><TD>1.30</TD><TD>uL</TD></TR><TR><TD>pXW</TD><TD>1.67</TD><TD>uL</TD><TD>pXS</TD><TD>0.94</TD><TD>uL</TD></TR><TR><TD>H2O</TD><TD>6.16</TD><TD>uL</TD><TD>H2O</TD><TD>6.89</TD><TD>uL</TD></TR><TR><TD><B>pXS-mGFPuv-mSA2 (23)</B></TD></TR><TR><TD>Mastermix</TD><TD>10.50</TD><TD>uL</TD></TR><TR><TD>mGFPuv-ATCT</TD><TD>0.37</TD><TD>uL</TD></TR><TR><TD>gBlock_6-mono</TD><TD>2.17</TD><TD>uL</TD></TR><TR><TD>pXS</TD><TD>0.94</TD><TD>uL</TD></TR><TR><TD>H2O</TD><TD>6.02</TD><TD>uL</TD></TR></TBODY></TABLE></P></LI></UL></DIV><LI>
We also started cultures for the cell binding assay to test the attachment to our filament, and for protein extraction. For the first, we used constructs pXW-INP_NC-Strep + pSC101-RFP, pXW-INP_NC-mSA2 + pSC101-RFP, and pSC101-RFP. For the latter, we used constructs pXW-mGFPuv-mSA2 and pXW-mGFPuv.
            </LI><LI class="list-group-item borderless"><B>September 29:</B></LI><LI>
Sequenced two new colonies of construct pXW-mGFPuv-Strep (due to a mutation in GFP).
            </LI><LI>
Created pSB1C3-mGFPuv2 for submission to the parts registry. For this we amplified the backbone from  pSB1C3 linear template using primers oM7722 and oM772 (length: 2093 bp), and mGFPuv2 from construct pXW-mGFPuv or pXS-mGFPuv using primers oM7662 and oM7664 (length: 744 bp). Golden Gate assembly was done using BsaI.
            </LI><LI class="list-group-item borderless"><B>September 30:</B><P>We did colony PCR on the CPEC assemblies with increased backbone DNA and altered backbone/insert ratio (cf. 28/09), and previous Golden Gate assemblies (cf. 28/09). We found positive colonies for following constructs: </P></LI><LI>pLC-XW</LI><LI>pLC-XW-INP_NC-Strep</LI><LI>pLC-XW-INP_NC-mSA2</LI><LI>pXW-inaZ-RC+mSA2</LI><LI>pXW-mGFPuv-Strep</LI><LI>pXS-mGFPuv-Strep </LI><LI>pXS-mGFPuv-mSA2</LI>
          
For constructs pXW-inaZ-RC+Strep and pXW-inaZ-RC+GFP, more colonies need testing. 
<DIV class="panel"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" href="#October">October</A></H4></DIV><DIV id="October" class="panel-collapse  collapse"><UL class="list-group"><LI class="list-group-item borderless"><B>October 03:</B><P>New colony PCR of 2x14 colonies of constructs pXW-inaZ-RC+Strep and pXW-inaZ-RC+GFP.</P></LI><LI class="list-group-item borderless"><B>October 04:</B><P>We found a positive colony for construct pXW-inaZ-RC+GFP (cf. 03/10).</P></LI><LI class="list-group-item borderless"><B>October 05:</B><P>Golden Gate assembly of pSB1C3-mGFPuv2, with a shorter protocol (cf. 29/09), and transformation into <I>E. coli</I> Top10 cells. <A data-toggle="collapse" href="#5oct1">Details</A></P></LI><LI class="list-group-item borderless"><B>Details:</B><P>Shorter protocol: 25x(2'@37C;2'@16C);5'@50C;10'@80C</P><P class="center"><TABLE><TBODY><TR><TD>length (bp)</TD><TD>conc. (ng/uL)</TD><TD>ratio</TD><TD>mass (ng)</TD><TD>volume (uL)</TD></TR><TR><TD>backbone</TD><TD>2093</TD><TD>172.1</TD><TD>1</TD><TD>200</TD><TD>1.16</TD></TR><TR><TD>mGFPuv2</TD><TD>744</TD><TD>156.2</TD><TD>3.</TD><TD>213.28</TD><TD>1.37</TD></TR><TR><TD><B>pSB1C3-mGFPuv2 (24)</B></TD></TR><TR><TD>H2O</TD><TD>8.67</TD><TD>uL</TD></TR><TR><TD>NEB T4 buffer</TD><TD>1.50</TD><TD>uL</TD></TR><TR><TD>BSA</TD><TD>0.30</TD><TD>uL</TD></TR><TR><TD>BsaI-HF</TD><TD>1.00</TD><TD>uL</TD></TR><TR><TD>NEB T4 ligase</TD><TD>1.00</TD><TD>uL</TD></TR><TR><TD>backbone</TD><TD>1.16</TD><TD>uL</TD></TR><TR><TD>mGFPuv2</TD><TD>1.37</TD><TD>uL</TD></TR></TBODY></TABLE></P></LI></UL></DIV><LI class="list-group-item borderless"><B>October 06:</B></LI><LI>
Colony PCR on  14 pSB1C3-mGFPuv2 colonies (cf. 05/10) (plates almost overgrown, assembly was apparently very efficient). For primers, we used verification primers (length: 1030 bp).
</LI><LI>
Combined pXW-INP_GGsafe  with pLC-XW-INP_NC-mSA2 by electroporation (plate on kan+chloramphenicol).
            </LI><LI class="list-group-item borderless"><B>October 12:</B><P>Sequencing results for constructs 13-24: sequencing okay for all constructs except those containing tetrameric streptavidin fusions. Point mutations were found in the coding sequence, multiple colonies showed mutations at different positions, suggesting a biological origin.</P></LI></DIV><DIV class="container"><FOOTER><DIV class="row "><H4 class="bottom text-right"> We like to thank following sponsors </H4></DIV><DIV class="row"><P class="center"> Your logo here? Contact us at 
     <A style="margin-right: 0px; padding-right: 0px;" href="mailto:igemugent@gmail.com" target="_top">igemugent@gmail.com</A></P></DIV><DIV class="row "><DIV class="col-md-6 "><P class="text-right"><A href="https://twitter.com/iGem_Ghent" class="twitter-follow-button" data-size="large" data-show-screen-name="false" data-show-count="false">Follow @iGem_Ghent</A></P></DIV></DIV></FOOTER></DIV></BODY></HTML>