Team:UGent Belgium/SOP
Bootstrap 101 Template
Toggle navigation
Home
Achievements
Team
Project
Description
Shape
Filament
Biofunction
Proof of concept
Demonstration
Safety
Lab measures
Project measures
Notebook
General notebook
Lab notebook
Primers
SOPs
Parts
Basic Parts
Composite Parts
Intermediate Parts
Practices
Silver
Integrated
Engagement
Awards
Hardware
Software
Measurement
Design
Entrepreneurship
Collaborations
Attributions
MENU ▤
HOME
TEAM
Team
★  Collaborations
PROJECT
★  Description
★ Design
Experiments
★ Proof of Concept
★ Demonstrate
Results
Notebook
PARTS
Parts
★ Basic Parts
★ Composite Parts
★ Part Collection
SAFETY
★  ATTRIBUTIONS
HUMAN PRACTICES
Human Practices
★ Silver
★ Gold
★ Integrated Practices
★ Engagement
AWARDS
★ Entrepreneurship
★ Hardware
★ Software
★  Measurement
★ Model
Protocols
PCR
Polymerase Chain Reation (PCR) is used extensively during both the assembly and verification of new constructs.      For assembly work, we use high-fidelity DNA polymerases: PrimeSTAR HS, PrimeSTAR GXL (TaKaRa Bio) or Q5 (New England Biolabs).     The manufacturer's instructions are followed unless mentioned otherwise. To create linear fragments suitable for     assembly, BsaI restriction sites or overlapping sequences can be added to the 5' end of the primers during the in silico design.     The length of each fragment is verified using agarose gel electrophoresis, reaction mixture are purified using a PCR purification kit (Analytik Jena)     and DNA concentration is determined using a NanoDrop device (Thermo Scientific).
After transformation, verification of construct assembly is mostly performed using Colony PCR with Taq DNA Polymerase or OneTaq Quick-Load Master Mix.     Colonies are randomly selected and picked by hand using sterile pipette tips, which are then briefly touched to a back-up plate and placed in a PCR tube.     We slightly modified the manufacturer's protocol by extending the initial denaturation time to 2 minutes, to ensure lysis of the colonies.     The length of each fragment is again determined by agarose gel electrophoresis.
Circular Polymerase Extension Cloning (CPEC)
Shorter constructs such as the plasmids carrying our basic parts, as well as the empty cloning vectors, were assembled from     linear backbones created by PCR and synthetic fragments (gBlocks) using Circular Polymerase Extension Cloning (CPEC)[1].     Our version of the protocol makes use of Q5 polymerase in the following reaction mixture:
Component
Amount
Vector backbone
100-200 ng
Insert
3:1 molar ratio to the vector
Q5 buffer
5.0 uL
dNTPs (2mM)
5.0 uL
DMSO
0.75 uL
Q5 DNA polymerase
0.5 uL
dH2O (add first)
to 15uL total volume
The tubes are incubated for 30 sec. at 98℃, 15 cycles of (10 sec. at 98℃; 30 sec. at 55℃; construct length(kb) x 15 sec. at 72℃) and 10 min. at 72 ℃.     1-2uL is then used directly for transformation, being careful not to add more as this increases the risk of arcing.
Golden Gate Cloning
Golden Gate cloning [2] was used to efficiently and seamlessly assemble and clone linear fragments into our expression vectors      ( BBa_K1896019 and BBa_K1896020 ).     15uL reactions are set up using NEB enzymes according to the table below. The tubes are then incubated for 25 cycles of      (3 min. at 37℃; 4 min. at 16℃), 10 min. at 50℃ and 10 min. at 80℃,     after which 2uL is used for transformation, without purification.
Component
Amount
Vector DNA
100 ng
Linear pieces
3:1 molar ratio to the vector
T4 buffer
1.5 uL
BSA (100x)
2.0 uL
BsaI-HF
1.0 uL
T4 DNA ligase
1.0 uL (400 units)
dH2O (add first)
to 15uL total volume
Transformation
Assembly mixes are transformed by electroporation in home-made electrocompetent cells of the E. coli Top10 genotype (Thermo Fisher)   using 0.1 mm cuvettes at 1.80 kV.   We create electrocompetent cells using the glycerol/mannitol density gradient wash method [3]. It is of vital importance to keep the   cells cold during the washing steps. One cryobox of competent cells is made as follows:
The day before:
Make SOB medium: to 500 mL of dH2O add 10 g of tryptone, 2.5 g of yeast extract, 0.25 g of NaCl, 5 mL of a 250 mM solution of KCl.         Set to pH 7.0 with 1 M NaOH and sterilise by autoclaving. Just before use add 2.5 mL of a sterile 2 M solution of MgCl2.
Make SOC medium: to 100 mL of the SOB medium add 2 mL of a sterile 1 M solution of glucose.
Make GM washing solution: 20 mL of glycerol, 1.5 g of mannitol, ultrapure water (Milli-Q) to total volume of 100 mL. Mix thoroughly, sterilise by filtration and refrigerate.
Sterilise 0.5 L of ultrapure water by autoclaving, refrigerate.
Autoclave a 2 L erlenmeyer flask as well as about 100 1.5 mL eppendorf tubes.
Start an E. coli Top10 preculture in a 5 mL LB culture tube.
In the morning:
Pour ~350 mL of SOB medium in the 2 L culture flask and inoculate with 3.5 mL of the overnight preculture.
Incubate at 37℃ with shaking until the OD measures 0.6-0.8 (roughly 2 h).
Distribute 300 mL of the OD 0.6-0.8 culture over 6x 50 mL falcons, incubate on ice for at least 30 min.
After lunch:
Centrifuge for 15 min. at 5000 g, 4℃, discard supernatant by decanting.
Resuspend each pellet in 20 mL cold ultrapure dH2O, vortex very briefly.
Carefully pipette a layer of 10 mL cold GM solution underneath cell suspension, forming 2 layers.
Centrifuge for 15 min. at 2000 g, 4℃, discard supernatant by pipetting from top to bottom.
Resuspend each pellet in 400 uL cold GM solution and distribute into pre-chilled sterile eppendorf tubes as 50 uL aliquots, immediately store at -80℃
Plasmid isolation
Colonies that tested positive by colony PCR are grown in 5 mL LB culture tubes, after which plasmid DNA is isolated using a Plasmid Miniprep kit (Analytik Jena).   To send a sample for sequencing by Macrogen: mix 7.5 uL plasmid DNA with 2.5 uL primer.
Protocol Lysate Preparation
All lysates were prepared as follows:
Grow 100 mL cultures in LB medium overnight for each strain.
Centrifuge in 50 mL tubes at 9000rpm for 10 minutes
Add 2.5 lysate buffer (PBS + 1mg/mL lysozyme + 0.1mM PMSF)
Incubate for 1 hour
Sonicate each sample for 10 minutes
Transfer to 2mL tubes and centrifuge @ 14000 rpm for 15 minutes
Use supernatant as lysate
Native PAGE
Polyacrylamide gel electrophoresis (PAGE) was required to show a difference in size between the original multimeric GFPuv and our monomeric version. Since the protein’s structure needs to be completely reserved to see this difference, proteins were prepared in a non-reducing, non-denaturating buffer (see Protein extraction protocol) and a Native PAGE was performed as followed:
A 4% acrylamide stacking gel (0.125M Tris pH 6.8) and a 10% acrylamide running gel (0.5M Tris pH 8.8) was prepared.
10 µL of the samples (~1 mg.ml-1) were mixed with 30 µL 4X loading buffer (3.5 ml 100% glycerol, 2.5 ml 1M Tris-HCl pH 6.8, 4 mg Bromophenol Blue and 4 ml water).
13 µL of the sample mixes was loaded to the gel.
The gel was run at 4°C in an electrode buffer containing 0.192M Glycine and 0.025M Tris.HCl (pH 8.3).
GFPuv and mGFPuv2 were visualized by placing the gel under a blacklight.
References
Quan, J., & Tian, J. (2009). Circular polymerase extension cloning of complex gene libraries and pathways. PloS one, 4(7), e6441.
Engler, C., Gruetzner, R., Kandzia, R., & Marillonnet, S. (2009). Golden gate shuffling: a one-pot DNA shuffling method based on type IIs restriction enzymes. PloS one, 4(5), e5553.
Warren, D. J. (2011). Preparation of highly efficient electrocompetent Escherichia coli using glycerol/mannitol density step centrifugation. Analytical biochemistry, 413(2), 206-207.
We like to thank following sponsors
Your logo here? Contact us at igemugent@gmail.com
Follow @iGem_Ghent
