Team:UGent Belgium/Biofunction
Bootstrap 101 Template
Toggle navigation
Home
Achievements
Team
Project
Description
Shape
Filament
Biofunction
Proof of concept
Demonstration
Safety
Lab measures
Project measures
Notebook
General notebook
Lab notebook
Primers
SOPs
Parts
Basic Parts
Composite Parts
Intermediate Parts
Practices
Silver
Integrated
Engagement
Awards
Hardware
Software
Measurement
Design
Entrepreneurship
Collaborations
Attributions
MENU ▤
HOME
TEAM
Team
★  Collaborations
PROJECT
★  Description
★ Design
Experiments
★ Proof of Concept
★ Demonstrate
Results
Notebook
PARTS
Parts
★ Basic Parts
★ Composite Parts
★ Part Collection
SAFETY
★  ATTRIBUTIONS
HUMAN PRACTICES
Human Practices
★ Silver
★ Gold
★ Integrated Practices
★ Engagement
AWARDS
★ Entrepreneurship
★ Hardware
★ Software
★  Measurement
★ Model
Biofunction
Overview
With the biofunction group we have made several constructs to test and validate our system. We tried to enhance the function of the 3D printed shape by using biological nucleation proteins. These proteins enhance the formation of ice crystals. Additionally, these proteins may help the condensation proces. This way, we can improve the condensation capacity of our water collector. To achieve this, we used the InaZ gene, an ice nucleating protein (INP) of Pseudomonas syringae. These INP’s are known to cause ice damage on plants and are also frequently used in snowmakers. Recently however, Pseudomonas syringae was also found in clouds, where they might help in rain formation [1-3].
The schematic structure of INP can be found underneath. It consists of a membrane binding N-terminal domain, some internal repeating domains responsible for the nucleation, and a C-terminal domain.
To attach our construct to the collector, biotin has been integrated in the filament. Hence, our constructs must contain streptavidin for effective binding to the 3D shape. To produce and attach these INP’s to our water collector, we investigated two options:
Production of an INP-streptavidin fusion protein in E. coli, followed by lysis of the cells and extraction of the fusion-protein
Separate membrane expression of both streptavidin and INP in E. coli
INP-streptavidin fusion protein
We designed a fusion protein of streptavidin with a modified INP lacking its membrane binding N-terminal domain, as depicted below.
To test the function of the different parts, and the fusion protein as a whole, we produced three control constructs:
A fusion protein of GFP and streptavidin to visualize the binding of streptavidin constructs to the biotin coated shape. The GFP part is a GFP mutant that gives the brightest signal after UV excitation and works as a monomer, in contrast to normal GFP which only works as a dimer [4,5].
A fusion protein of the InaZ lacking its terminal domain, GFP, and streptavidin. With this construct we can confirm that the InaZ doesn’t influence the streptavidin binding to biotin.
To ensure that the streptavidin is responsible for binding to the 3D printed structure, we will use a construct of the modified InaZ fused with GFP, but without streptavidin. If streptavidin is responsible for the binding to the 3D structure, the resulting protein of this construct should not be detected on the shape after applying it.
As the first two constructs both contain streptavidin, the resulting proteins from these constructs should, after lysis and extraction from E.coli, bind to the biotin on the 3D printed structure.
Membrane expression
As cell lysis and protein extraction is laborious and resource intensive we will also investigate the possibility to express both streptavidin and INP on the outer membrane surface of E. coli. For this purpose, two constructs will be made:
A custom made fusion protein of streptavidin with the membrane binding region of INP
As an alternative we could use an existing biobrick of streptavidin fused to the membrane binding domain Lpp-OmpA
The wild type (WT) InaZ
In this approach, E. coli will bind to the biotin on the surface of the 3D printed structure using the membrane-bound streptavidin. The WT InaZ (containing the membrane binding domain) can then perform its normal nucleation function.
Again, several control constructs were used to test the function of the different parts:
A custom made fusion protein of GFP with the membrane binding region of INP. This allows us to visually assess whether the membrane binding region of InaZ functions properly as display.
Alternatively, we will use a fusion of Lpp-OmpA to check its functionality as displayed below.
A fusion protein of the membrane binding domain of InaZ with GFP and streptavidin. With this construct we can confirm that the membrane binding domain doesn’t influence the streptavidin binding to biotin.
Similar to the previous construct, but with Lpp-OmpA
To make the constructs for these fusion proteins, we can use some existing biobricks to start from:
InaZ membrane binding region:
BBa_K811003
Lpp-OmpA - Streptavidin fusion protein:
BBa_J36846
WT InaZ:
BBa_K584024
Other necessary building blocks can be ordered as gblocks, and can be easily cloned to construct the necessary fusion proteins. Restriction sites will be included to ensure compatibility with golden gate, biobricks and bglbricks.
Results
To make sure the INP works we tested it in two different set ups. For both set ups we used the same protocol that can be found under the Measurement section.
In the movie on the left you can see a first experiment using whole E. coli cells. First we added a culture of E. coli containing a control construct expressing a RFP coding device (BBa_J04450). As you can see, nothing happens. When we added a culture of E. coli containing a  construct expressing WT INP (our own BBa_K1896011 based on BBa_KS84027) you see immediate ice formation.
In the movie on the right you can see a second experiment using protein lysates of E. coli. First we added a protein lysate of E. coli expressing mGFPuv2 in the cytosol (BBa_K1896010). As you can see, nothing happens. When we added a protein lysate of E. coli expressing a fusion protein of a truncated INP (without the membrane binding domain) and mSA2 (monomeric streptavidin) in the cytosol (BBa_K1896018) you see immediate ice formation.
References
DeLeon-Rodriguez, N. et al. Microbiome of the upper troposphere: Species composition and prevalence, effects of tropical storms, and atmospheric implications. Proc. Natl. Acad. Sci. 110, 2575–2580 (2013).
Pratt, K. A. et al. In situ detection of biological particles in cloud ice-crystals. Nat. Geosci. 2, 398–401 (2009).
Lundheim, R. Physiological and ecological significance of biological ice nucleators. Philos. Trans. R. Soc. B Biol. Sci. 357, 937–943 (2002).
von Stetten, D., Noirclerc-Savoye, M., Goedhart, J., Gadella, T. W. J. & Royant, A. Structure of a fluorescent protein from Aequorea victoria bearing the obligate-monomer mutation A206K. Acta Crystallogr. Sect. F. Struct. Biol. Cryst. Commun. 68, 878–82 (2012).
Ito, Y., Suzuki, M. & Husimi, Y. A novel mutant of green fluorescent protein with enhanced sensitivity for microanalysis at 488 nm excitation. Biochem. Biophys. Res. Commun. 264, 556–60 (1999).
We like to thank following sponsors
Your logo here? Contact us at igemugent@gmail.com
Follow @iGem_Ghent
