Team:UGent Belgium/LabNotebook
Bootstrap 101 Template
Toggle navigation
Home
Achievements
Team
Project
Description
Shape
Filament
Biofunction
Proof of concept
Demonstration
Safety
Lab measures
Project measures
Notebook
General notebook
Lab notebook
Primers
SOPs
Parts
Basic Parts
Composite Parts
Intermediate Parts
Practices
Silver
Integrated
Engagement
Awards
Hardware
Software
Measurement
Design
Entrepreneurship
Collaborations
Attributions
MENU ▤
HOME
TEAM
Team
★  Collaborations
PROJECT
★  Description
★ Design
Experiments
★ Proof of Concept
★ Demonstrate
Results
Notebook
PARTS
Parts
★ Basic Parts
★ Composite Parts
★ Part Collection
SAFETY
★  ATTRIBUTIONS
HUMAN PRACTICES
Human Practices
★ Silver
★ Gold
★ Integrated Practices
★ Engagement
AWARDS
★ Entrepreneurship
★ Hardware
★ Software
★  Measurement
★ Model
Lab Notebook
August
August 29:
We made a strong expression vector (pXS) by restriction/ligation of the linearized plasmid backbone pSB1C3 and a gblock made up of promoter-RBS-cloning site-terminator-terminator (cut with EcoRI and PstI). Details
Details:
Strong expression vector: pSB1C3-PBBaJ23119-RBSBBaB0034-GGsiteBsaI-TTBBaB0010-TTBBaB0012
Restriction digests:
Resuspend gBlocks in Elution Buffer (=TE) to concentration of 50 ng/uL -> 500 ng delivered => 10uL buffer
Linearised PSB1C3: 25ng/uL at 50uL (used 2013 iGEM tubes)
Prepare mixes (enzymes by NEB):
Insert digest
H2O
30
uL
CutSmart buffer
10
uL
EcoRI-HF
2
uL
PstI
3
uL (tube 5 years past expiration date)
gBlock 1
5
uL (final concentration 5 ng/uL)
Backbone digest
H2O
5
uL
CutSmart buffer
10
uL
EcoRI-HF
2
uL
PstI
3
uL (tube 5 years past expiration date)
linear pSB1C3
30
uL (final concentration 15 ng/uL)
Incubate overnight in 37°C warm water bath (overdigestion because of enzyme expiration date and restriction sites close to fragment termini, especially for the gBlock).
Our first attempts to isolate the inaZ gene from P. syringae and the inaX gene from X. campestris failed. Try inaZ with more stringent conditions and Primestar polymerase, perform more thorough lysis of X. campestris to try inaX again. Details
Details:
The inaZ gene was sourced from BBa_K584027 using primers oM7628 and oM7629* (length 3.6 kbp, anneal at 58°C). The plasmid was obtained from KU Leuven (concentration unknown): PCR (Q5) ran with 10x and 1000x template dilutions, transformed into E. coli DH5a by heatshock, and plated on chloramphenicol.
The inaX gene was sourced from X. campestris colonies using primers oM7626 and oM7627 (length 4.7 kbp, anneal at 70°C). Cloning overhangs were not included on these primers. The colony was suspended in 1000uL distilled water, vortexed, freezed 30 min. @ -80 deg. C, heated 20 min. @ 95 deg. C, vortexed, and debris spinned down.
August 30:
We transformed the strong expression vector (cf. 29/08) by electroshock into E. coli Top10, and plated it on chloramphenicol. Details
Details:
Before transformation of the vector, we inactivated the restriction enzymes using heat: 20 min at 80°C, and did ligation without purification. We also purified the samples on a spin column, eluted in 10 uL.
length (bp)
conc. (ng/uL)
ratio
mass (ng)
volume (uL)
Backbone
2070
15
1
50
3.33
gBlock
263
5
3
19.06
3.81
H2O
9.86
uL
T4 ligase buffer
2.00
uL
Backbone
3.33
uL
Insert
3.81
uL
T4 ligase
1.00
uL
As a backup strategy we also used CPEC with the backbone from the K584027 plasmid (100x dilution) to generate a strong expression vector. For this, we used Primestar HS PCR with primers oM7643 and oM7645. Details
Details:
Annealing: 5"@55C. Full program: 30x(10"@98C;5"@55C;2'@72C)
August 31:
Second attempt (cf. 29/08) to isolate the inaZ gene from P. syringae and the inaX gene from X. campestris. InaX failed, but inaZ was successful. Both Primestar HS and Primestar GXL PCR were used for this. On the gel, lane 4 and 8 are inaX attempts. Details
Details:
For inaZ, we used BBa_K584027 as template (1/10, 1/100, and 1/1000 dilutions). Annealing temperature was 66.20°C for primer oM7628 and 61.93°C for primer oM7629* (length: 3.6 kbp). For inaX, we used X. campestris colonies as template. Annealing was done at 60.96°C for primer oM7626 and at 70.34°C for primer oM7627 (length 4.7 kbp).
Conditions for Primestar HS: 1'@98C;30x(10"@98C;15"@60C;5'@68C);10'@68C (not the standard protocol).
Conditions for Primestar GXL: 1'@98C;30x(10"@98C;15"@60C;5'@72C);10'@72C (not the standard protocol).
Transformation of the strong expression vector by restriction and ligation was not very successful, we only had a few colonies. The backbone amplification using CPEC was successful (see gel, cf. 30/08). Details
Details:
CPEC calculations for strong expression vector:
length (bp)
conc. (ng/uL)
ratio
mass (ng)
volume (uL)
Backbone
2070
29
1
100.00
3.45
(diluted backbone 10x)
Insert
263
5
3
38.12
7.62
(diluted gblock 10x)
H2O
5.68
uL
Q5 buffer
2.00
uL
dNTPs (2mM)
5.00
uL
Backbone
3.45
uL
gBlock
7.62
uL
DMSO
0.75
uL
Q5 polymerase
0.50
uL
Incubation: 30"@98C;15x(10"@98C;30"@55C;35"@72C);10'@72C.
September
September 01:
Heatshock transformation of the strong expression vector made with CPEC (cf. 31/08) yielded no colonies.
Colony PCR on the colonies with the strong expression vector made with restriction and ligation didn’t show any positive ones (cf. 31/08). As primers, we used BioBrick verification primers (length: 534 bp).
September 02:
Electroshock transformation of the strong expression vector made with CPEC (cf. 31/08) into E. coli Top10 cells.
September 04:
CPEC using Primestar GXL and an excess of insert. Details
Details:
length (bp)
conc. (ng/uL)
ratio
mass (ng)
volume (uL)
Backbone
2070
58
1
100.00
1.72
(diluted backbone 5x)
Insert
263
50
10
127.05
2.54
(undiluted gBlock)
H2O
12.48
uL
GXL buffer
5.00
uL
dNTPs (GXL)
2.00
uL
Backbone
1.72
uL
gBlock
2.54
uL
DMSO
0.75
uL
GXL polymerase
0.50
uL
(elongation 1min/kb@68C)
Via colony PCR we saw that all 14 colonies with the strong expression vector made with CPEC and electroshock transformation were all positive (cf. 02/09). We used BioBrick verification primers for this. Details
Details:
Conditions: 2'@95C; 30x(20"@95C;30"@55C;1'@68C)
colonies
15
rxns @ 15uL
H2O
181.2
uL
Taq buffer
24
uL
dNTPs (2mM)
24
uL
Fw primer
4.8
uL
Rv primer
4.8
uL
Taq polymerase
1.2
uL
We assembled our weak expression vector (pXW), using a gBlock that came in today. Q5 CPEC was used for this with the linearized plasmid backbone pSB1C3, and as insert the gBlock consisting of promoter-RBS-cloning site-terminator-terminator. The promoter is weaker than the promoter of the strong expression vector (cf. 29/08). E. coli Top10 cells were transformed. Details
Details:
Weak expression vector: PBBaJ23105-RBSB0034-MCSBsaI-TTB0010-TTB0012
Incubation: 30"@98C;15x(10"@98C;30"@55C;35"@72C);10'@72C
length (bp)
conc. (ng/uL)
ratio
mass (ng)
volume (uL)
Backbone-weak
2070
34.3
1
100.00
2.92
(diluted backbone 10x)
Insert-weak
263
5
3
38.12
7.62
(diluted gblock 10x)
H2O
6.21
uL
Q5 buffer
2.00
uL
dNTPs (2mM)
5.00
uL
Backbone
2.92
uL
gBlock
7.62
uL
DMSO
0.75
uL
Q5 polymerase
0.05
uL
September 06:
Colonies 1 and 2 of our strong expression vector made with CPEC (cf. 04/09) are cultured and miniprepped. These are used for following constructs:
pXS-INP_WT using the inaZ PCR fragment
pXS-INP_NC-mGFPuv
pXS-INP_NC-Strep (Strep: regular streptavidin)
pXS-INP_NC-mSA2 (mSA2: monomeric streptavidin)
pXS-Lpp-ompA-mGFPuv
pXS-Lpp-ompA-Strep
pXS-Lpp-ompA-mSA2
All constructs are made using Golden Gate assembly. Details
Details:
Incubation for pXS-INP_WT: 25x(3'@37C;4'@16C); 10'@80C
Incubation for all others: 25x(3'@37C;4'@16C); 5'@50C; 5'@80C
length (bp)
conc. (ng/uL)
ratio
mass (ng)
volume (uL)
pXS
2290
135.6
1
100.00
0.74
inaZ
3625
168
1
158.30
0.94
gBlock 4
1048
50
3
137.29
2.75
gBlock 5
829
50
3
108.60
2.17
gBlock 6_tetra
496
50
3
64.98
1.30
gBlock 6_mono
500
50
3
65.50
1.31
gBlock 7
553
50
3
72.45
1.45
Master Mix:
7
reactions (10% pipetting loss will be taken into account)
H2O
23.97
uL
NEB T4 buffer
11.55
uL
BSA
1.16
uL
BsaI-HF
7.70
uL
NEB T4 ligase
7.70
uL
pXS vector
5.68
uL
pXS-INP_WT
Master Mix
7.50
uL
inaZ
0.94
uL
H2O
6.56
uL
pXS-INP_NC-mGFPuv
pXS-INP_NC-Strep
Master Mix
7.50
uL
Master Mix
7.50
uL
gBlock 4
2.75
uL
gBlock 4
2.75
uL
gBlock 5
2.17
uL
gBlock 6_tetra
1.30
uL
H2O
2.58
uL
H2O
3.45
uL
pXS-INP_NC-mSA2
pXS-Lpp-ompA-mGFPuv
Master Mix
7.50
uL
Master Mix
7.50
uL
gBlock 4
2.75
uL
gBlock 5
2.17
uL
gBlock 6_mono
1.31
uL
gBlock 7
1.45
uL
H2O
3.44
uL
H2O
3.88
uL
pXS-Lpp-ompA-Strep
pXS-Lpp-ompA-mSA2
Master Mix
7.50
uL
Master Mix
7.50
uL
gBlock 6_tetra
1.30
uL
gBlock 6_mono
1.31
uL
gBlock 7
1.45
uL
gBlock 7
1.45
uL
H2O
4.75
uL
H2O
4.74
uL
September 07:
We transformed our Golden Gate reactions (cf. 06/08) into E. coli Top10 cells.
September 08:
We did a colony PCR of our weak expression vector (cf. 04/09), and of our constructs made with the strong expression vector (cf. 06/09), both using BioBrick verification primers. Details
Details:
Expected length (bp) of the constructs made with the strong expression vector:
pXS-inaZ: 4115
pXS-INP_NC-mGFPuv: 2166
pXS-INP_NC-Strep: 1857
pXS-INP_NC-mSA2: 1832
pXS-lpp-ompA-mGFPuv: 1670
pXS-lpp-ompA-Strep: 1362
pXS-lpp-ompA-mSA2: 1338
Conditions weak expression vector: 2'@95C; 30x(20"@95C;30"@55C;1'@68C)
colonies
14
rxns @ 15uL
H2O
174.405
uL
Taq buffer
23.1
uL
dNTPs (2mM)
23.1
uL
Fw primer
4.62
uL
Rv primer
4.62
uL
Taq polymerase
1.155
uL
We tried to make inaZ Golden Gate safe by removing the internal BsaI sites. To achieve this, we used Primestar HS PCR and cut it into 3 different fragments. This was successful for the separate fragments, but we didn’t succeed to combine them. For the first fragment, this was simply from the original PCR fragment. The second fragment is a new PCR fragment, using oM7658 and oM7629 as primers and the inaZ plasmid as template (length:3039 bp). This band is smeared, probably due to excessive template concentration or annealing time. The third fragment is also a new PCR fragment, using oM7658 and oM7229 as primers and the inaZ plasmid as template (length: 293 bp).
September 09:
The colony PCR of E. coli cells with the constructs with the strong expression vector showed no positive colonies (cf. 06/09). Not many bands are present, and all present bands show the original vector or self ligation. The colony PCR on our cells transformed with the weak expression vector (cf. 04/09) did show positive ones. We repeated the colony PCR under different conditions, but bands were still not of the correct lengths.
We therefore tried cloning all our parts into the weak expression vector using Golden Gate. These reaction mixes were then transformed into E. coli Top10 cells. Incubation was done at 30°C instead of 37°C to repress the plasmid copy number. Details
Details:
length (bp)
conc. (ng/uL)
ratio
mass (ng)
volume (uL)
pXW
2290
59.8
1
100.00
1.67
gBlock 4
1048
50
3
137.29
2.75
gBlock 5
829
50
3
108.60
2.17
gBlock 6_tetra
496
50
3
64.98
1.30
gBlock 6_mono
500
50
3
65.50
1.31
gBlock 7
553
50
3
72.45
1.45
Master Mix:
7
reactions (10% pipetting loss will be taken into account)
H2O
16.77
uL
NEB T4 buffer
11.55
uL
BSA
1.16
uL
BsaI-HF
7.70
uL
NEB T4 ligase
7.70
uL
pXW vector (k1)
12.88
uL
negative control
Master Mix
7.5
uL
H2O
7.5
uL
2) pXW-INP_NC-mGFPuv
3) pXW-INP_NC-Strep
Master Mix
7.50
uL
Master Mix
7.50
uL
gBlock 4
2.75
uL
gBlock 4
2.75
uL
gBlock 5
2.17
uL
gBlock 6_tetra
1.30
uL
H2O
2.58
uL
H2O
3.45
uL
4) pXW-INP_NC-mSA2
5) pXW-Lpp-ompA-mGFPuv
Master Mix
7.50
uL
Master Mix
7.50
uL
gBlock 4
2.75
uL
gBlock 5
2.17
uL
gBlock 6_mono
1.31
uL
gBlock 7
1.45
uL
H2O
3.44
uL
H2O
3.88
uL
6) pXW-Lpp-ompA-Strep
7) pXW-Lpp-ompA-mSA2
Master Mix
7.50
uL
Master Mix
7.50
uL
gBlock 6_tetra
1.30
uL
gBlock 6_mono
1.31
uL
gBlock 7
1.45
uL
gBlock 7
1.45
uL
H2O
4.75
uL
H2O
4.74
uL
September 10:
We got our sequencing results: the strong expression vector is good!
Colonies with the weak expression vector constructs are still too small to pick up with colony PCR (cf. 09/09).
September 11:
All our plates with the weak expression vector constructs show colonies! The control reaction has colonies as well, but much fewer, which was to be expected. Fluorescence is seen for the pXW-INP_NC-mGFPuv construct. All Lpp-ompA plates have a mixed phenotype, with 10-20 large colonies (same size as control reaction), and >60 small colonies (see culture picture). Colony PCR was performed using OneTaq at 50°C annealing temperature to proof that the small colonies are the positives ones, the larger ones the negative ones. This theory was confirmed for all colonies (see gel picture). Keep incubating at 30°C to reduce copy number.
We performed Primestar HS PCR to create following constructs:
pXW-Lpp-ompA-mGFPuv-(m)Strep using Lpp-ompA-mGFPuv fragment. As a template, we used the Golden Gate mix for pXW-Lpp-ompA-mGFPuv, and oM7667_Lpp_Fw and oM7663_GFP_Rv-ATCT as primers (expected length:1167).
pXW-mGFPuv-m(Strep) using mGFP. As a template, we used a 100x diluted gBlock, and oM7662_GFP_Fw-GATG and oM7663_GFP_Rv-ATCT as primers (expected length: 752 bp).
pXS-mGFPuv using mGFP. As a template, we used a 100x diluted gBlock, and oM7662_GFP_Fw-GATG and oM7664_GFP_Rv-TACT as primers (expected length: 752 bp).
We also repeated previous PCR (cf. 08/09) of the largest piece of inaZ to make it Golden Gate safe. As a template, we used the inaZ plasmid, and oM7658 and oM7629 as primers (expected length: 3039 bp). Because the band was smeared last time, we should either use lower template concentrations or a shorter anneal time.
September 12:
Miniprep of a number of colony PCR positive colonies (cf. 09/09).
Next picture is the gel of the PCR reactions of the previous day (cf. 11/09). inaZ was run 2 times (at 1/100 and 1/1000 dilution), these are the first 2 bands. The next 3 bands are the 3 constructs. PCR using the Golden Gate mix as a template didn’t work very well (two extra bands, lane 3). We therefore repeated this reaction using a pure plasmid.
Golden Gate assemblies of:
pXW-inaZ_GGsafe
pXW-mGFPuv-Strep
pXW-mGFPuv-mSA2
pXW-mGFPuv
pXS-mGFPuv
Details
Details:
length (bp)
conc. (ng/uL)
ratio
mass (ng)
volume (uL)
pXS
2290
106.6
1
100.00
0.94
pXW
2290
59.8
1
100.00
1.67
INP full
3600
168.5
1
157.21
0.93
INP part 2
3039
206.6
1
132.71
0.64
INP part 3
293
58
3
38.38
0.66
mGFPuv-ATCT
752
210.1
3
98.52
0.47
mGFPuv-TACT
752
275.7
3
98.52
0.36
gBlock 6_tetra
496
50
3
64.98
1.30
gBlock 6_mono
500
50
3
65.50
1.31
Master Mix:
5
reactions (10% pipetting loss will be taken into account)
H2O
34.93
uL
NEB T4 buffer
8.25
uL
BSA
0.83
uL
BsaI-HF
5.50
uL
NEB T4 ligase
5.50
uL
pXW-INP_GGsafe (8)
pXW-mGFPuv-Strep (9)
Master Mix
10.00
uL
Master Mix
10.00
uL
INP full
0.93
uL
mGFPuv-ATCT
0.47
uL
INP part 2
0.64
uL
gBlock 6_tetra
1.30
uL
INP part 3
0.66
uL
pXW
1.67
uL
pXW
1.67
uL
H2O
1.56
uL
H2O
1.09
uL
pXW-mGFPuv-mSA2 (10)
pXW-mGFPuv (11)
Master Mix
10.00
uL
Master Mix
10.00
uL
mGFPuv-ATCT
1.30
uL
mGFPuv-TACT
0.36
uL
gBlock 6_mono
1.31
uL
pXW
1.67
uL
pXW
1.67
uL
H2O
2.97
uL
H2O
0.72
uL
note: mistakenly used 1.3uL of mGFPuv-TACT
pXS-mGFPuv (12)
Master Mix
10.00
uL
mGFPuv-TACT
0.36
uL
pXS
0.94
uL
H2O
3.70
uL
September 13:
pXW-Lpp-ompA-mGFPuv miniprep tubes that did not grow yesterday (cf. 12/09) did show growth today, probably through mutations.
Transformed Golden Gate assemblies (cf. 12/09) into E. coli Top10 cells.
September 14:
Colony PCR of the transformed Golden Gate assemblies (cf. 13/09): fail for pXW-INP_GGsafe (first 7 lanes), okay for other assemblies. For pXW-INP_GGsafe, we will make a new Golden Gate design.
September 15:
Colony PCR screening of 8 more colonies with pXW-inaZ_GGsafe showed no result (cf. 13/09)
September 16:
Minipreps of the following plasmids and sequencing with verification primers:
pXW-mGFPuv-Strep
pXW-mGFPuv-mSA2
pXW-mGFPuv
pXS-mGFPuv
We did a Primestar HS PCR to go from high copy to low copy (LC) backbone, with as backbone pSB3K3 and 4 inserts: INP_NC-Strep, INP_NC-mSA2, Lpp-ompA-Strep and Lpp-ompA-mSA2. The backbone and Lpp-ompA-mSA2 both failed. On the gel, all four inserts are seen. The backbone was run on a different gel (not pictured here). Details
Details:
Backbone: pSB3K3 with pSB3K3 as template (plate4, F6, iGem distribution 2016), and oM7669 & oM 7670 as primers (length: 2790 bp).
Insert1: INP_NC-Strep (156 ng/ul) with pXW-INP_NC-Strep as template, and oM7671 & oM 7672 as primers (length: 1684 bp).
Insert2:  INP_NC-mSA2 (290 ng/ul) with pXW-INP_NC-mSA2 as template and oM7671 & oM 7672 as primers.
Insert3: Lpp-ompA-Strep (130 ng/ul) with  pXW-Lpp-ompA-Strep as template, and oM7671 & oM 7672 as primers.
Insert4: Lpp-ompA-mSA2 with pXW-Lpp-ompA-mSA2 as template, and oM7671 & oM 7672 as primers.
September 19:
We retried the Primestar HS PCR to go from high to low copy backbone. The backbone failed again,  Lpp-ompA-mSA2 was successful. We also made the ‘new parts’ of inaZ, was was also successful. Details
Details:
Retry backbone: used  pSB3K3 as template, and oM7675 & oM7644 as new primers (length: 2790 bp).
Retry insert4 (Lpp-ompA-mSA2): used Golden Gate mix as template (134 ng/ul), and oM7671 & oM7672 as primers.
New parts inaZ:  inaZ plasmid as template, and oM7628 & oM7673 as primers for piece1 (150 ng/ul, 634 bp), oM7658 & oM7674 as primers for piece2 (146 ng/ul, 2792 bp), and oM7659 & oM7629 as primers for piece3 (122 ng/ul, 297 bp).
Golden gate assembly of pXW-inaZ_GGsafe. Details
Details:
pXW-INP_GGsafe   (NEW parts!)
Mastermix
4.50
uL
INP part 1
0.63
uL
INP part 2
1.42
uL
INP part 3
0.55
uL
pXW
1.67
uL
H2O
11.23
uL
September 21:
We did a colony PCR for the Golden Gate assembly of pXW-inaZ_GGsafe (cf. 19/09). Colony 2, 6, 8 are okay.
We opted for a new backbone for the low copy vector (cf. 19/09): pSC101-Kan vector.
September 22:
We digested colony PCR samples 2, 6, 8, 13, and 32 (cf. 21/09), and a control with BsaI. If pXW-inaZ_GGsafe is indeed Golden Gate safe, it should not be cut. This was okay! Colony 2 and 6 are plasmid prepped for sequencing. Colony 13 is lane 5, control is lane 6.
We again retried the Primestar HS PCR to go from high to low copy backbone. For this, we have the low copy backbone pSC101 (right gel, lane 1). A new Lpp-ompA-mSA2 insert was also done to obtain an insert for the low copy vector (left gel, lane 1). A weak expression vector with an empty expression site was also added (left gel, lane 2), and also inaZ-RC. Lane 2 and 3 on the right gel are respectively a control, and backbone pSB1C3 (mSA2-REsafe gBlock assembly). Details
Details:
New Backbone "LC" (Low Copy, pSC101 ORI):  pSC101-Kan vector as template, and oM7714 & oM7715 as primers (length: 4034 bp).
New Lpp-ompA-Strep insert: Golden Gate mix as template, and oM7671 & oM 7672 as primers (1051 bp).
XW empty expression site: pXW as template, and oM7671 & oM7672 as primers (length: 224 bp).
inaZ-RC: pXW-inaZ_GGsafe as template, and oM7665 & oM7666 as primers (length: 3100 bp).
September 26:
CPEC assemblies to create:
pLC-XW
pLC-XW-INP_NC-Strep
pLC-XW-INP_NC-mSA2
pLC-XW-Lpp-ompA-Strep
pLC-XW-Lpp-ompA-mSA2
pSB1C3-mSA2
Details
Details:
length (bp)
conc. (ng/uL)
ratio
mass (ng)
volume (uL)
pLC backbone
2790
322
1
100.00
0.31
XW insert
224
26.94
3
24.09
0.89
5x verdund
INP_NC-Strep
1546
156
3
166.24
1.07
INP_NC-mSA2
1522
290
3
163.66
0.56
Lpp-ompA-Strep
1051
249.6
3
113.01
0.45
Lpp-ompA-mSA2
1027
134
3
110.43
0.82
Master Mix:
5
reactions (10% pipetting loss will be taken into account)
H2O
62.92
uL
Q5 buffer
11.00
uL
dNTPs (2mM)
27.50
uL
pLC backbone
1.71
uL
DMSO
4.13
uL
Q5 polymerase
2.75
uL
pLC-XW (13)
pLC-XW-INP_NC-Strep (14)
H2O
4.11
uL
H2O
3.93
uL
XW insert
0.89
uL
INP_NC-Strep
1.07
uL
pLC-XW-INP_NC-mSA2 (15)
pLC-XW-Lpp-ompA-Strep (16)
H2O
4.44
uL
H2O
4.55
uL
INP_NC-mSA2
0.56
uL
Lpp-ompA-Strep
0.45
uL
pLC-XW-Lpp-ompA-mSA2 (17)
incubation:
H2O
4.18
uL
30"@98C; 15x(10"@98C;30"@55C;1'15"@72C); 10'@72C
Lpp-ompA-mSA2
0.82
uL
length (bp)
conc. (ng/uL)
ratio
mass (ng)
volume (uL)
pSB1C3 backbone
2044
375
1
100.00
0.27
gBlock mSA2
475
50
3
69.72
1.39
Mix: (18)
incubation:
H2O
15.09
uL
30"@98C; 15x(10"@98C;30"@55C;1'15"@72C); 10'@72C
Q5 buffer
2.00
uL
dNTPs (2mM)
5.00
uL
pSB1C3 backbone
0.27
uL
gBlock mSA2
1.39
uL
DMSO
0.75
uL
Q5 polymerase
0.5
uL
September 27:
Colony PCR of all CPEC assemblies (cf. 26/09) with primers oM3441 + oM1760 for the first 5 assemblies and verification primers for the last assembly. The order of the gel is: pLC-XW-INP_NC-Strep, pLC-XW-INP_NC-mSA2, pLC-XW-Lpp-ompA-Strep, pLC-XW-Lpp-ompA-mSA2, pSB1C3-mSA2, and pLC-XW (only 3 colonies).
September 28:
Second attempt for CPEC assemblies (cf. 26/09), with increased backbone DNA (200 ng) and an altered backbone/insert ratio (1:5). Details
Details:
length (bp)
conc. (ng/uL)
ratio
mass (ng)
volume (uL)
pLC backbone
2790
322
1
20.000
0.62
XW insert
224
134.7
5
80.29
0.60
INP_NC-Strep
1546
156
5
554.12
3.55
INP_NC-mSA2
1522
290
5
545.52
1.88
Lpp-ompA-Strep
1051
249.6
5
376.70
1.51
Lpp-ompA-mSA2
1027
134
5
368.10
2.75
Master Mix:
5
reactions (10% pipetting loss will be taken into account)
H2O
61.21
uL
Q5 buffer
11.00
uL
dNTPs (2mM)
27.50
uL
pLC backbone
3.42
uL
DMSO
4.13
uL
Q5 polymerase
2.75
uL
pLC-XW (13*)
pLC-XW-INP_NC-Strep (14*)
H2O
4.40
uL
H2O
1.45
uL
XW insert
0.60
uL
INP_NC-Strep
3.55
uL
pLC-XW-INP_NC-mSA2 (15*)
pLC-XW-Lpp-ompA-Strep (16*)
H2O
3.12
uL
H2O
3.49
uL
INP_NC-mSA2
1.88
uL
Lpp-ompA-Strep
1.51
uL
pLC-XW-Lpp-ompA-mSA2 (17*)
incubation:
H2O
2.25
uL
30"@98C; 15x(10"@98C;30"@55C;1'15"@72C); 10'@72C
Lpp-ompA-mSA2
2.75
uL
Primestar HS PCR to create GATG-mGFPuv-ATCT, which is necessary for new attempts for the pXW-mGFPuv-Strep and pXW-mGFPuv-mSA2  constructs, due to mutations in previous versions. For this, we used primers oM7662 + oM7663, and pXS-mGFPuv as template.
Golden Gate assemblies to create:
pXW-inaZ-RC+Strep
pXW-inaZ-RC+mSA2
pXW-inaZ-RC+GFP
pXW-mGFPuv-Strep
pXS-mGFPuv-Strep
pXS-mGFPuv-mSA2
Details
Details:
pXW-inaZ-RC+Strep (18)
pXW-inaZ-RC+mSA2 (19)
Mastermix
12.17
uL
Mastermix
12.17
uL
inaZ-RC
0.80
uL
inaZ-RC
0.80
uL
gBlock_6-tetra
1.30
uL
gBlock_6-mono
1.30
uL
H2O
5.73
uL
H2O
5.73
uL
pXW-inaZ-RC+GFP (20)
MasterMiX
Mastermix
12.17
uL
ligase buffer
6
inaZ-RC
0.80
uL
ligase
3
gBlock 5
2.17
uL
BsaI
3
H2O
4.86
uL
BSA
1.5
H20
18
pXW
5.01
36.51
MasterMix:
3
ligase buffer
6
ligase
3
BsaI
3
BSA
1.5
H20
18
31.5
pXW-mGFPuv-Strep (21 (old 9))
pXS-mGFPuv-Strep (22)
Mastermix
10.50
uL
Mastermix
10.50
uL
mGFPuv-ATCT
0.37
uL
mGFPuv-ATCT
0.37
uL
gBlock_6-tetra
1.30
uL
gBlock_6-tetra
1.30
uL
pXW
1.67
uL
pXS
0.94
uL
H2O
6.16
uL
H2O
6.89
uL
pXS-mGFPuv-mSA2 (23)
Mastermix
10.50
uL
mGFPuv-ATCT
0.37
uL
gBlock_6-mono
2.17
uL
pXS
0.94
uL
H2O
6.02
uL
We also started cultures for the cell binding assay to test the attachment to our filament, and for protein extraction. For the first, we used constructs pXW-INP_NC-Strep + pSC101-RFP, pXW-INP_NC-mSA2 + pSC101-RFP, and pSC101-RFP. For the latter, we used constructs pXW-mGFPuv-mSA2 and pXW-mGFPuv.
September 29:
Sequenced two new colonies of construct pXW-mGFPuv-Strep (due to a mutation in GFP).
Created pSB1C3-mGFPuv2 for submission to the parts registry. For this we amplified the backbone from  pSB1C3 linear template using primers oM7722 and oM772 (length: 2093 bp), and mGFPuv2 from construct pXW-mGFPuv or pXS-mGFPuv using primers oM7662 and oM7664 (length: 744 bp). Golden Gate assembly was done using BsaI.
September 30:
We did colony PCR on the CPEC assemblies with increased backbone DNA and altered backbone/insert ratio (cf. 28/09), and previous Golden Gate assemblies (cf. 28/09). We found positive colonies for following constructs:
pLC-XW
pLC-XW-INP_NC-Strep
pLC-XW-INP_NC-mSA2
pXW-inaZ-RC+mSA2
pXW-mGFPuv-Strep
pXS-mGFPuv-Strep
pXS-mGFPuv-mSA2
For constructs pXW-inaZ-RC+Strep and pXW-inaZ-RC+GFP, more colonies need testing.
October
October 03:
New colony PCR of 2x14 colonies of constructs pXW-inaZ-RC+Strep and pXW-inaZ-RC+GFP.
October 04:
We found a positive colony for construct pXW-inaZ-RC+GFP (cf. 03/10).
October 05:
Golden Gate assembly of pSB1C3-mGFPuv2, with a shorter protocol (cf. 29/09), and transformation into E. coli Top10 cells. Details
Details:
Shorter protocol: 25x(2'@37C;2'@16C);5'@50C;10'@80C
length (bp)
conc. (ng/uL)
ratio
mass (ng)
volume (uL)
backbone
2093
172.1
1
200
1.16
mGFPuv2
744
156.2
3.
213.28
1.37
pSB1C3-mGFPuv2 (24)
H2O
8.67
uL
NEB T4 buffer
1.50
uL
BSA
0.30
uL
BsaI-HF
1.00
uL
NEB T4 ligase
1.00
uL
backbone
1.16
uL
mGFPuv2
1.37
uL
October 06:
Colony PCR on  14 pSB1C3-mGFPuv2 colonies (cf. 05/10) (plates almost overgrown, assembly was apparently very efficient). For primers, we used verification primers (length: 1030 bp).
Combined pXW-INP_GGsafe  with pLC-XW-INP_NC-mSA2 by electroporation (plate on kan+chloramphenicol).
October 12:
Sequencing results for constructs 13-24: sequencing okay for all constructs except those containing tetrameric streptavidin fusions. Point mutations were found in the coding sequence, multiple colonies showed mutations at different positions, suggesting a biological origin.
We like to thank following sponsors
Your logo here? Contact us at igemugent@gmail.com
Follow @iGem_Ghent
