<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UGent_Belgium_SOP skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UGent Belgium/SOP</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Bootstrap 101 Template</TITLE></P><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI class="active"><A href="https://2016.igem.org/Team:UGent_Belgium">Home </A></LI><LI><A href=" https://2016.igem.org/Team:UGent_Belgium/Medals">Achievements</A></LI><LI><A href=" https://2016.igem.org/Team:UGent_Belgium/Team"> Team  </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Description">Description </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Shape"> Shape </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Filament"> Filament </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Biofunction"> Biofunction </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Proof"> Proof of concept </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Demonstrate"> Demonstration </A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Safety </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Safety">Lab measures </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Safety#project">Project measures </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Notebook </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Notebook">General notebook </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/LabNotebook">Lab notebook </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Primers">Primers </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/SOP">SOPs </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Parts </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Parts">Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Parts#composite">Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Parts#fusion">Intermediate Parts </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Practices </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/HP/Silver">Silver </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Integrated_Practices">Integrated</A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Engagement">Engagement </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Awards </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Hardware">Hardware </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Software">Software </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Measurement">Measurement </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Design">Design </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Entrepreneurship">Entrepreneurship </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Collaborations">Collaborations </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Attributions">Attributions </A></LI></DIV></DIV><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:UGent_Belgium">HOME </A></LI><LI class="menu_item"> TEAM
			</LI><LI><A href=" https://2016.igem.org/Team:UGent_Belgium/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Collaborations">★  Collaborations </A></LI></UL><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Description"> ★  Description </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Design"> ★ Design </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Proof"> ★ Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Demonstrate"> ★ Demonstrate </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Notebook"> Notebook </A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Composite_Part"> ★ Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Part_Collection"> ★ Part Collection </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:UGent_Belgium/Safety"> SAFETY </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:UGent_Belgium/Attributions">★  ATTRIBUTIONS </A></LI><LI class="menu_item"> HUMAN PRACTICES 
			</LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Human_Practices"> Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/HP/Silver">★ Silver </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/HP/Gold">★ Gold </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Integrated_Practices"> ★ Integrated Practices </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Engagement">★ Engagement </A></LI><LI class="menu_item"> AWARDS 
			</LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Entrepreneurship"> ★ Entrepreneurship </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Hardware"> ★ Hardware </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Software">★ Software </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Measurement">★  Measurement </A></LI><LI><A href="https://2016.igem.org/Team:UGent_Belgium/Model">★ Model </A></LI></DIV><DIV class="content_wrapper"><DIV class="container"><H1>Protocols</H1><H2>PCR</H2><P>Polymerase Chain Reation (PCR) is used extensively during both the assembly and verification of new constructs. 
    For assembly work, we use high-fidelity DNA polymerases: PrimeSTAR HS, PrimeSTAR GXL (TaKaRa Bio) or Q5 (New England Biolabs).
    The manufacturer's instructions are followed unless mentioned otherwise. To create linear fragments suitable for
    assembly, BsaI restriction sites or overlapping sequences can be added to the 5' end of the primers during the in silico design.
    The length of each fragment is verified using agarose gel electrophoresis, reaction mixture are purified using a PCR purification kit (Analytik Jena)
    and DNA concentration is determined using a NanoDrop device (Thermo Scientific).</P><P>After transformation, verification of construct assembly is mostly performed using Colony PCR with Taq DNA Polymerase or OneTaq Quick-Load Master Mix.
    Colonies are randomly selected and picked by hand using sterile pipette tips, which are then briefly touched to a back-up plate and placed in a PCR tube.
    We slightly modified the manufacturer's protocol by extending the initial denaturation time to 2 minutes, to ensure lysis of the colonies.
    The length of each fragment is again determined by agarose gel electrophoresis.</P><H2>Circular Polymerase Extension Cloning (CPEC)</H2><P>Shorter constructs such as the plasmids carrying our basic parts, as well as the empty cloning vectors, were assembled from
    linear backbones created by PCR and synthetic fragments (gBlocks) using Circular Polymerase Extension Cloning (CPEC)[1].
    Our version of the protocol makes use of Q5 polymerase in the following reaction mixture:</P><TABLE class="table table-condensed" style="width:auto; margin-left:auto; margin-right:auto; font-size:small"><THEAD><TR><TH>Component</TH><TH>Amount</TH></TR></THEAD><TBODY><TR><TD>Vector backbone</TD><TD>100-200 ng</TD></TR><TR><TD>Insert</TD><TD>3:1 molar ratio to the vector</TD></TR><TR><TD>Q5 buffer</TD><TD>5.0 uL</TD></TR><TR><TD>dNTPs (2mM)</TD><TD>5.0 uL</TD></TR><TR><TD>DMSO</TD><TD>0.75 uL</TD></TR><TR><TD>Q5 DNA polymerase</TD><TD>0.5 uL</TD></TR><TR><TD>dH2O (add first)</TD><TD>to 15uL total volume</TD></TR></TBODY></TABLE><P>The tubes are incubated for 30 sec. at 98℃, 15 cycles of (10 sec. at 98℃; 30 sec. at 55℃; construct length(kb) x 15 sec. at 72℃) and 10 min. at 72 ℃.
    1-2uL is then used directly for transformation, being careful not to add more as this increases the risk of arcing.</P><H2>Golden Gate Cloning</H2><P>Golden Gate cloning [2] was used to efficiently and seamlessly assemble and clone linear fragments into our expression vectors 
    (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1896019">BBa_K1896019</A> and <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1896020">BBa_K1896020</A>).
    15uL reactions are set up using NEB enzymes according to the table below. The tubes are then incubated for 25 cycles of 
    (3 min. at 37℃; 4 min. at 16℃), 10 min. at 50℃ and 10 min. at 80℃,
    after which 2uL is used for transformation, without purification.</P><TABLE class="table table-condensed" style="width:auto; margin-left:auto; margin-right:auto; font-size:small"><THEAD><TR><TH>Component</TH><TH>Amount</TH></TR></THEAD><TBODY><TR><TD>Vector DNA</TD><TD>100 ng</TD></TR><TR><TD>Linear pieces</TD><TD>3:1 molar ratio to the vector</TD></TR><TR><TD>T4 buffer</TD><TD>1.5 uL</TD></TR><TR><TD>BSA (100x)</TD><TD>2.0 uL</TD></TR><TR><TD>BsaI-HF</TD><TD>1.0 uL</TD></TR><TR><TD>T4 DNA ligase</TD><TD>1.0 uL (400 units)</TD></TR><TR><TD>dH2O (add first)</TD><TD>to 15uL total volume</TD></TR></TBODY></TABLE><H2>Transformation</H2><P>Assembly mixes are transformed by electroporation in home-made electrocompetent cells of the <I>E. coli</I> Top10 genotype (Thermo Fisher)
  using 0.1 mm cuvettes at 1.80 kV.
  We create electrocompetent cells using the glycerol/mannitol density gradient wash method [3]. It is of vital importance to keep the
  cells cold during the washing steps. One cryobox of competent cells is made as follows:</P><OL><I>The day before:</I><LI>Make SOB medium: to 500 mL of dH2O add 10 g of tryptone, 2.5 g of yeast extract, 0.25 g of NaCl, 5 mL of a 250 mM solution of KCl.
        Set to pH 7.0 with 1 M NaOH and sterilise by autoclaving. Just before use add 2.5 mL of a sterile 2 M solution of MgCl2. </LI><LI>Make SOC medium: to 100 mL of the SOB medium add 2 mL of a sterile 1 M solution of glucose.</LI><LI>Make GM washing solution: 20 mL of glycerol, 1.5 g of mannitol, ultrapure water (Milli-Q) to total volume of 100 mL. Mix thoroughly, sterilise by filtration and refrigerate.</LI><LI>Sterilise 0.5 L of ultrapure water by autoclaving, refrigerate.
    </LI><LI>Autoclave a 2 L erlenmeyer flask as well as about 100 1.5 mL eppendorf tubes.</LI><LI>Start an <I>E. coli</I> Top10 preculture in a 5 mL LB culture tube.</LI><I>In the morning:</I><LI>Pour ~350 mL of SOB medium in the 2 L culture flask and inoculate with 3.5 mL of the overnight preculture.</LI><LI>Incubate at 37℃ with shaking until the OD measures 0.6-0.8 (roughly 2 h).</LI><LI>Distribute 300 mL of the OD 0.6-0.8 culture over 6x 50 mL falcons, incubate on ice for at least 30 min.</LI><I>After lunch:</I><LI>Centrifuge for 15 min. at 5000 g, 4℃, discard supernatant by decanting.</LI><LI>Resuspend each pellet in 20 mL cold ultrapure dH2O, vortex very briefly.</LI><LI>Carefully pipette a layer of 10 mL cold GM solution underneath cell suspension, forming 2 layers.</LI><LI>Centrifuge for 15 min. at 2000 g, 4℃, discard supernatant by pipetting from top to bottom.</LI><LI>Resuspend each pellet in 400 uL cold GM solution and distribute into pre-chilled sterile eppendorf tubes as 50 uL aliquots, immediately store at -80℃</LI></OL><H2>Plasmid isolation</H2><P>Colonies that tested positive by colony PCR are grown in 5 mL LB culture tubes, after which plasmid DNA is isolated using a Plasmid Miniprep kit (Analytik Jena).
  To send a sample for sequencing by Macrogen: mix 7.5 uL plasmid DNA with 2.5 uL primer.</P><H2>Protocol Lysate Preparation</H2><P>All lysates were prepared as follows: </P><OL><LI>Grow 100 mL cultures in LB medium overnight for each strain. </LI><LI>Centrifuge in 50 mL tubes at 9000rpm for 10 minutes</LI><LI>Add 2.5 lysate buffer (PBS + 1mg/mL lysozyme + 0.1mM PMSF) </LI><LI>Incubate for 1 hour</LI><LI>Sonicate each sample for 10 minutes</LI><LI>Transfer to 2mL tubes and centrifuge @ 14000 rpm for 15 minutes</LI><LI>Use supernatant as lysate</LI></OL><H2>Native PAGE</H2><P>Polyacrylamide gel electrophoresis (PAGE) was required to show a difference in size between the original multimeric GFPuv and our monomeric version. Since the protein’s structure needs to be completely reserved to see this difference, proteins were prepared in a non-reducing, non-denaturating buffer (see Protein extraction protocol) and a <B>Native</B> PAGE was performed as followed:</P><OL><LI>A 4% acrylamide stacking gel (0.125M Tris pH 6.8) and a 10% acrylamide running gel (0.5M Tris pH 8.8) was prepared.</LI><LI>10 µL of the samples (~1 mg.ml-1) were mixed with 30 µL 4X loading buffer (3.5 ml 100% glycerol, 2.5 ml 1M Tris-HCl pH 6.8, 4 mg Bromophenol Blue and 4 ml water).</LI><LI>13 µL of the sample mixes was loaded to the gel. </LI><LI>The gel was run at 4°C in an electrode buffer containing 0.192M Glycine and 0.025M Tris.HCl (pH 8.3).</LI><LI>GFPuv and mGFPuv2 were visualized by placing the gel under a blacklight.</LI></OL><H3 class="bottom">References</H3><P><OL><LI>Quan, J., &amp; Tian, J. (2009). Circular polymerase extension cloning of complex gene libraries and pathways. <I>PloS one</I>, 4(7), e6441.</LI><LI>Engler, C., Gruetzner, R., Kandzia, R., &amp; Marillonnet, S. (2009). Golden gate shuffling: a one-pot DNA shuffling method based on type IIs restriction enzymes. <I>PloS one</I>, 4(5), e5553.</LI><LI>Warren, D. J. (2011). Preparation of highly efficient electrocompetent Escherichia coli using glycerol/mannitol density step centrifugation. <I>Analytical biochemistry</I>, 413(2), 206-207.</LI></OL></P></DIV><DIV class="container"><FOOTER><DIV class="row "><H4 class="bottom text-right"> We like to thank following sponsors </H4></DIV><DIV class="row"><P class="center"> Your logo here? Contact us at 
     <A style="margin-right: 0px; padding-right: 0px;" href="mailto:igemugent@gmail.com" target="_top">igemugent@gmail.com</A></P></DIV><DIV class="row "><DIV class="col-md-6 "><P class="text-right"><A href="https://twitter.com/iGem_Ghent" class="twitter-follow-button" data-size="large" data-show-screen-name="false" data-show-count="false">Follow @iGem_Ghent</A></P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>