<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_RDFZ-China_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:RDFZ-China/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="const" style="padding-left:-30px;padding-right:-30px;"><DIV id="nav"><UL><LI><A href="https://2018.igem.org/Team:RDFZ-China" title="Home">Home</A></LI><LI class="dropdownlist"><A href="javascript:void(0)" class="dropbtn">Xscape</A><DIV class="dropdown_content"><A href="https://2018.igem.org/Team:RDFZ-China/Project_Description" alt="Description">Description</A><A href="https://2018.igem.org/Team:RDFZ-China/Design" alt="Design">Design</A><A href="https://2018.igem.org/Team:RDFZ-China/Model" alt="Modelling">Modelling</A><A href="https://2018.igem.org/Team:RDFZ-China/InterLab" alt="InterLab">InterLab</A><A href="https://2018.igem.org/Team:RDFZ-China/Demonstrate" alt="Result">Result</A><A href="https://static.igem.org/mediawiki/2018/3/33/T--RDFZ-China--protocal.pdf" alt="Result" target="_blank">Protocol</A><A href="https://static.igem.org/mediawiki/2018/2/25/T--RDFZ-China--notebook.pdf" alt="Notebook" target="_blank">Notebook</A></DIV></LI><LI class="dropdownlist"><A href="javascript:void(0)" class="dropbtn">Human Practice</A><DIV class="dropdown_content"><A href="https://2018.igem.org/Team:RDFZ-China/Human_Practices" alt="Biosafety&amp;DIY Bio">Biosafety &amp; DIY Bio</A><A href="https://2018.igem.org/Team:RDFZ-China/Engagement" alt="Biosafety&amp;DIY Bio">Public Engagement</A></DIV></LI><LI class="dropdownlist"><A href="javascript:void(0)" class="dropbtn">Part</A><DIV class="dropdown_content"><A href="https://2018.igem.org/Team:RDFZ-China/Basic_Part" alt="Basic Parts">Basic</A><A href="https://2018.igem.org/Team:RDFZ-China/Composite_Parts" alt="Composite Parts">Composite</A><A href="https://2018.igem.org/Team:RDFZ-China/Improvement" alt="Improvement">Improvement</A><A href="https://2018.igem.org/Team:RDFZ-China/Measurement" alt="Measurement">Measurement</A></DIV></LI><LI><A href="https://2018.igem.org/Team:RDFZ-China/Safety" title="Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:RDFZ-China/Valhalla" title="Valhalla">Valhalla</A></LI><LI class="dropdownlist"><A href="javascript:void(0)" class="dropbtn">Team</A><DIV class="dropdown_content"><A href="https://2018.igem.org/Team:RDFZ-China/School" alt="School">School</A><A href="https://2018.igem.org/Team:RDFZ-China/Team" alt="Members">Members</A><A href="https://2018.igem.org/Team:RDFZ-China/Collaborations" alt="Collaboration">Collaboration</A><A href="https://2018.igem.org/Team:RDFZ-China/Attributions" alt="Attribution">Attribution &amp; Acknowledgement</A></DIV></LI><LI><A href="https://2018.igem.org/Team:RDFZ-China/Achievement" title="Achievement">Achievement</A></LI></UL></DIV></DIV><DIV id="body"><DIV id="sidebar" class="paper-sidenav"><UL><LI><A href="#section1">Overview</A></LI><LI><A href="#section2">Lethal Gene</A></LI><LI><A href="#section3">For Fermenter</A></LI><LI><A href="#section4">Therapeutic Bacteria</A></LI><LI><A href="#section5">References</A></LI></UL></DIV><DIV class="description"><H1>Design</H1><DIV class="topic-title" id="section1"><H3>Overview</H3><P>We want to build a device which can kill the engineered bacteria when it escapes from the fermenter and a device which can kill the therapeutic bacteria after it releases the drug. We used Thermal Sensitive Regulator and Quorum Sensing device to achieve the sensing and using ccdB and Colicin to achieve the killing. Furthermore, we used the capacity monitor to assess the resource occupied by our device, and optimized our device.</P></DIV><DIV class="topic-title" id="section2"><H3>Lethal Gene</H3><P>As we mentioned in the project description, the major problem of biosafety leakage is the spread of recombinant DNA to the environment, so this project we are mainly focused on Nuclease. From the parts registry, we found NucA/B, Colicin E9, and miniColicin E2. We found that miniColicin E2 is well documented by TU Darmstadt 2016, so we chose it as our lethal gene. Also, we found a nuclease from Bacteriophage which is reported as high efficiency at 65 degree Celsius. Moreover, we considered EndoG, derived from mammalian Caspase-independent apoptosis pathway, which we believe it is far in homology may lead to a reduced possibility of evolution of inhibitor.</P></DIV><DIV class="topic-title" id="section3"><H3>For Fermenter</H3><H4>1 Thermal Sensitive Regulator</H4><P>When constructing a “fail-safe” device for a known condition but unknown timing incident, any chemical inducer that needed to be added manually should not be considered. Then, as the temperature is usually being set to a certain level in a fermenter, as the engineered bacteria escaped from the fermenter, changes in temperature can be detected. So, the thermal regulator came to our site. From part registry, we found thermal sensitive transcriptional factors which can initiate the transcription when the temperature reaches above a certain level. Bacteriophage’s cl repressor are commonly used, and the tlpA system came out in 2017. ETH Zurich was using tlpA’s original version, and NUS Singapore was using the modified version, TlpA36, which ought to be initiated at 36 degree Celsius. We choose TlpA36 since the original version showed bit intensive leakage at the temperature it supposed not to be turned on. Besides, we add TlpA39 (TlpA repressor respond at 39 degree Celsius) and Tcl42 (cI repressor respond to 42 degree Celsius) to the registry, giving the user more selective temperature for future users’ project. </P><P>The first thing we want to verify is its function at a different temperature, we assumed that there would be leakage since the promoter we choose for TlpA protein was not the strongest promoter. Then we need it to sense the change from high to low temperature since, in the real-world scenario, the temperature is higher in the fermenter than the environment. According to the previously assumed mechanism by previous researches, we assumed that the dimerized TlpA protein would block the expression of TlpA promoter. Later, since lethal gene needs to turn on when the temperature drops, so the signal should be inverted, we decided to use repressor to invert this. dCas9 system was first mentioned, but it did not fit in to our project since cytotoxicity was too high for overexpression, and if the expression of this repression drops, lethal gene might be expressed ahead; Then we came up with the idea of tetR family repressor, (Stanton et al.), but the energy cost is quite high, so we chose sRNA(Storz et al.) as the repressor. sRNA will consume far less energy than protein repressor, also as a single strand RNA, it is unstable, so when the expression is blocked, the derepression will be quite fast. sRNA mainly consists of three regions: Seed Region, antisense to the target mRNA; the Hfq binding region, which binds to the Hfq protein which can stabilize the sRNA-mRNA complex, waiting for RNase to degrade it; and a termination region, it is a terminator. </P><P>We designed six set of sRNA, with different seed regions (one for 5’UTR and one for Coding Sequence) and different Hfq binding regions.</P><P>We designed sRNA based on the sequence of our reporter gene, mNeonGreen and our selected RBS, J61101. We designed nine sRNA, synthesized 6 of them, characterized 3, finally we chose 1 to use it on our final circuit.</P><P>sRNAs include a seed region, which is the region antisense to the target sequence; an Hfq protein binding region; a termination region, to stop the transcription of sRNA itself. Hfq protein was there to promote the sRNA-mRNA base pairing and increase the stability of sRNA-mRNA complex.</P><P>The seed region we choose was antisense to the 5’UTR region, which includes J61101 RBS and scar; also, we choose the protein coding sequence of mNeonGreen. Previous researches stated that the blocking could be performed by sRNA antisense to 70nt upstream or 15nt downstream. </P><H4>2 Density Regulator</H4><P>The other environmental signal we can detect is the cell density. As engineered bacteria accidentally felt out from the fermenter, it will be diluted by the environment; the density will decrease. </P><P>Quorum sensing device in iGEM is quite commonly used until now it has been used for 227 times, the entire system has been well characterized, so we used it directly.</P><P>The circuit was quite similar to F2420, but we changed the pTet promoter to pTac. We want the device to express lethal genes when cell density drops, as they get into and diluted by the environment.</P><P>The genetic circuit designed for density-regulated sensor contains an AHL acceptor protein and the quorum sensing pLux promoter. The acceptor protein is LuxR, which is constantly expressed (under promoter pTac) and forms AHL-LuxR complex with AHL molecules. The complex consequently binds to an operator lux box in the promoter region (pLux), which up-regulates the expression of the gene downstream. </P><P>Same as the thermal regulator, we have to invert the signal, so we choose a tetR family repressor. Yes, we did mention that protein transcriptional regulator is resource consuming, but the sRNAs were not able to block the expression entirely during our characterization. So, according to previous research and characterization, we choose PhlF repressor, since its repression is very tight. (Stanton et al.) PhlF will repress pPhlF promoter at PhlO region; the derepression is carried out by LVA degradation tag.</P><H4>3 Integrase</H4><P>One of the basic building blocks of our logic gate is temperature, through which we expect to control suicide switch. Under high cell temperature should the kill switch be turned OFF and contrarily under low cell density, which resembles the situation of strain leakage, should the bacteria express DNase or cytotoxic genes to kill themselves. So, we went through the registry, found that Peking iGEM2017 characterized several integrases, which can be used as an initiator in our project.</P><P>In order to eliminate leakage in the expression of the cytotoxic gene, integrase is employed in the genetic circuit of the regulator. The promoter upstream the executor ccdB colicin E2 is reversed so that no suicide gene is transcribed before the temperature reaches 36 degrees Celsius. Reaching temperature turns on the temperature-sensitive promoter upstream serine integrase that consequently flips over the promoter upstream suicide genes and initiates transcription and cell death.</P><P>So the final circuit will be  a NOR gate </P><H4>4 Cold-Lux-Repressible</H4><P>Since we target to modify bacteria used in the fermentation industry, which primarily are used for material synthesis and production, our sequence inserts should occupy as limited resources in bacteria as possible to avoid significant expression burden. In order to reduce this expression stress, we designed another device for fermentation which used a LuxR repressive promoter (Peking-S, 2011) and the “Cold Box” in 5’UTR region of CspA (Ionis Paris, 2017). With only one transcriptional regulator, less energy will be consumed, which is beneficial for cell resource distribution.</P></DIV><DIV class="topic-title" id="section4"><H3>Therapeutic Bacteria</H3><P>In iGEM, bacteria for human are popular, almost a thousand pages in all the wiki pages mentioned therapeutic bacteria therapeutic bacteria such as diagnostic bacteria and drug delivery bacteria. However, project for drug delivery bacteria, drugs were released by the lysis of bacteria, so bacteria are destroyed, but naked recombinant DNA will enter the human body, which may be taken up by the human microbiome. </P><P>We designed a set of devices, which can carry out real-time tracking of the bacteria, drug release and genome degradation, all in one host bacteria.</P><P>1. Tracking and locating of microorganisms in their host organism has always been a big challenge, previous methods like an optical reporter and radioactive reporter cannot be quickly or accurately track and locate it(Bourdeau et al.). From a set of research, we found that gas vesicle could act as a non-invasive reporter due to its unique acoustic characteristic.</P><P>Gas Vesicle is an organelle found in cyanobacteria, it acts as a buoyancy regulator by permitting air in and out of the vesicle, to send the bacteria to the optimum depth with sufficient oxygen needed. Several teams had tried to build a floating bacteria like OUC-China. </P><P>By using gas vesicle, we can carry out non-invasive imaging. We expect we can use ultrasound imaging to track the therapeutic bacteria to the nidus. </P><P>2. Then, using VioABDE designed by Cambridge 2009, utilized from SHSBNU 2017, we found it was a good candidate for the drug release since the product should be a small molecule which can diffuse out without cell lysis. So, we connect VioABDE to the pTlpA promoter, followed by TlpA39. In a real situation, we can use ultrasound tissue heating, to de-repress the pTlpA promoter, and activate the production of violacein, which is a precursor of antiphlogosis.</P><P>3. When the drug is released, we can keep heat the tissue. Using promoter pR followed by miniColicinE2 and Tcl42. So, when temperature reaches 42°C, miniColicin will be released, degrade the genome, and kill the bacteria. </P></DIV></DIV><DIV class="reference"><DIV class="topic-title" id="section5"><H3>References</H3><P>Bourdeau, Raymond W., et al. “Acoustic Reporter Genes for Noninvasive Imaging of Microorganisms in Mammalian Hosts.” Nature, vol. 553, no. 7686, Nature Publishing Group, 2018, pp. 86–90, doi:10.1038/nature25021.</P><P>Stanton, Brynne C., et al. “Genomic Mining of Prokaryotic Repressors for Orthogonal Logic Gates.” Nature Chemical Biology, vol. 10, no. 2, 2014, pp. 99–105, doi:10.1038/nchembio.1411.</P><P>Storz, Gisela, et al. “Regulation by Small RNAs in Bacteria: Expanding Frontiers.” Molecular Cell, vol. 43, no. 6, 2011, pp. 880–91, doi:10.1016/j.molcel.2011.08.022.</P><P>Piraner, Dan I., et al. &quot;Tunable thermal bioswitches for in vivo control of microbial therapeutics.&quot; Nature chemical biology 13.1 (2017): 75.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>