<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_RDFZ-China_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:RDFZ-China/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="const" style="padding-left:-30px;padding-right:-30px;"><DIV id="nav"><UL><LI><A href="https://2018.igem.org/Team:RDFZ-China" title="Home">Home</A></LI><LI class="dropdownlist"><A href="javascript:void(0)" class="dropbtn">Xscape</A><DIV class="dropdown_content"><A href="https://2018.igem.org/Team:RDFZ-China/Project_Description" alt="Description">Description</A><A href="https://2018.igem.org/Team:RDFZ-China/Design" alt="Design">Design</A><A href="https://2018.igem.org/Team:RDFZ-China/Model" alt="Modelling">Modelling</A><A href="https://2018.igem.org/Team:RDFZ-China/InterLab" alt="InterLab">InterLab</A><A href="https://2018.igem.org/Team:RDFZ-China/Demonstrate" alt="Result">Result</A><A href="https://static.igem.org/mediawiki/2018/3/33/T--RDFZ-China--protocal.pdf" alt="Result" target="_blank">Protocol</A><A href="https://static.igem.org/mediawiki/2018/2/25/T--RDFZ-China--notebook.pdf" alt="Notebook" target="_blank">Notebook</A></DIV></LI><LI class="dropdownlist"><A href="javascript:void(0)" class="dropbtn">Human Practice</A><DIV class="dropdown_content"><A href="https://2018.igem.org/Team:RDFZ-China/Human_Practices" alt="Biosafety&amp;DIY Bio">Biosafety &amp; DIY Bio</A><A href="https://2018.igem.org/Team:RDFZ-China/Engagement" alt="Biosafety&amp;DIY Bio">Public Engagement</A></DIV></LI><LI class="dropdownlist"><A href="javascript:void(0)" class="dropbtn">Part</A><DIV class="dropdown_content"><A href="https://2018.igem.org/Team:RDFZ-China/Basic_Part" alt="Basic Parts">Basic</A><A href="https://2018.igem.org/Team:RDFZ-China/Composite_Parts" alt="Composite Parts">Composite</A><A href="https://2018.igem.org/Team:RDFZ-China/Improvement" alt="Improvement">Improvement</A><A href="https://2018.igem.org/Team:RDFZ-China/Measurement" alt="Measurement">Measurement</A></DIV></LI><LI><A href="https://2018.igem.org/Team:RDFZ-China/Safety" title="Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:RDFZ-China/Valhalla" title="Valhalla">Valhalla</A></LI><LI class="dropdownlist"><A href="javascript:void(0)" class="dropbtn">Team</A><DIV class="dropdown_content"><A href="https://2018.igem.org/Team:RDFZ-China/School" alt="School">School</A><A href="https://2018.igem.org/Team:RDFZ-China/Team" alt="Members">Members</A><A href="https://2018.igem.org/Team:RDFZ-China/Collaborations" alt="Collaboration">Collaboration</A><A href="https://2018.igem.org/Team:RDFZ-China/Attributions" alt="Attribution">Attribution &amp; Acknowledgement</A></DIV></LI><LI><A href="https://2018.igem.org/Team:RDFZ-China/Achievement" title="Achievement">Achievement</A></LI></UL></DIV></DIV><DIV id="body"><DIV id="sidebar" class="paper-sidenav"><UL><LI><A href="#section1">Overall</A></LI><LI><A href="#section2">Lethal Gene</A></LI><LI><A href="#section3">Thermal Regulator</A></LI><LI><A href="#section4">Density Regulator</A></LI><LI><A href="#section5">Integrase</A></LI><LI><A href="#section6">Therapeutic Bacteria</A></LI><LI><A href="#section7">Capacity Monitor</A></LI></UL></DIV><DIV class="description"><DIV class="topic-title" id="section1"><H3>Overall</H3><P>✔Our lethal genes are functional, the DNase can degrade genome. </P><P>✔A serie of thermal regulators work as we expected, the switch can either work when we turn it from on to off and off to on, also it can convert the positive regulation to negative regulation through an additional repressor: sRNA</P><P>✔Density Regulator work as expected, they can convert the positive regulation to negative regulation through an additional repressor: PhlF</P><P>✔Integrase can be used as initiator for the device</P><P>✔Drug’s release can be thermal regulated</P></DIV><DIV class="topic-title" id="section2"><H3>Lethal Gene</H3><H4>-clearness</H4><P>ccdB colony was picked from the plate and transfer to the bacterium tube adding 10^3M and 10^2 M of IPTG to M9 medium including Cm antibiotic, tubes are cultured overnight, 37 degree Celsius and 200 rpm. </P><B class="text-comment">Fig.1 Differences between induced and uninduced ccdB broth.</B><P>Visible difference can be seen between non-induced and the induced ccdB, so we can qualitatively conclude the ccdB works as expected.</P><P>Since miniColicin was hard to construct, we changed our order several times and obtained a plasmid with missing RBS and an extra integration mutation behind the start codon, so we perform the characterization exam quite roughly.</P><B class="text-comment">Fig.2 Rough characterization of miniColicin’s lethality in DWP.</B><P>But still, the ccdB broth has left in the well for 2 days long, and miniColicin had stayed in for 1 day, the well with IPTG added showed visible clearness, which also indicates it is functional as a lethal gene.</P><H4>-cfu</H4><P>We carried out the cell forming unit exam for BBa_K2572019, by adding 100ul uninduced broth to the petri dish with or without IPTG. </P><B class="text-comment">Fig3 a) Plates with broth dilution factor of 5, plates do contain 1E-3M IPTG. With colony forming unit 546, 585, 622Fig3 b) Plates with broth dilution factor of 5, plates do not contain IPTG. With colony forming unit 1448, 2661, 1778.</B><P>There is about 70% reduction of induced colony formed compare to the non-induced ones, so we can say this device is functional properly. We assumed that if the induction started inside the tube, the cfu will drop dramatically.</P><H4>-DNA cleavage</H4><B class="text-comment">Fig.4 The DNA cleavage image from genscript.</B><P>The problem with Nucleases are that it might leak during synthesis or subclone construction, so our plasmid came very very late. The assay was carried out by genescript technicians during our subclone preparation. In the graph, DNA is quite completely degraded, which means our part worked as expected.</P></DIV><DIV class="topic-title" id="section3"><H3>Thermal Regulator</H3><H4>-can it work</H4><P>First, we characterize the Thermal Sensitive Regulators, to see if they work. We simply put them into 37 degree Celsius shaker and 42 degree Celsius shaker, and see if they glow as we expected.</P><P>K2572000 is TlpA39, which was selected out with the same method as TlpA36, we thought that 39 degree might be a good temperature for human, which was not too high to hurt people, also not too low to mixed up with others. K2572001 is Tcl42, quite similar with TlpA protein and its originated from bacteriophage, which with regulation system cl repressor. </P><P>However, we found that Tcl38 is not working as we expected, and after sequencing, we were quite sure that the reporter gene was lost. So, in the following characterization, Tcl38 was used as a negative control.</P><H4>-as we expected</H4><P>Then we performed the characterization under different temperature. Petri dishes were placed into incubators with different temperatures and grew for 24 hours.</P><P>We can see that at 35 degree Celsius, TlpA36 starts to derepress. At 37 degree Celsius, TlpA39 starts to derepress. At 39.5 degree Celsius, Tcl42 starts to express. Leakage was observed, transcription of pTlpA was initiated below the expected temperature. ETH Zurich iGEM2017 improved this by simply increase the expression of TlpA protein, since they assumed that the leakage was caused by lack of repression.</P><H4>We used plate reader to characterize this set of Thermal Regulators, they were incubicated under different temperature and transformed to plate reader after 18 hours.</H4><B class="text-comment">This fig. shows the expression of thermal regulators under 30 33 36 39 42 45 degree Celsius,we can see the leakage is low, unlike what we have seen on the petri dishes.</B><H4>We verified that Thermal Regulators can turn from on to off. Most of the user was using turn on, but seldom people try to turn it off. We cultivated TlpA36 from high 37 degree Celsius to 30 degree Celsius, and measure it on plate reader</H4><B class="text-comment">This fig indicated that the expression stop as soon as it is being transformed from 37 degree to 30 degree, we concluded that the remaining fluorescent was from the undegraded GFP.</B><H4>Invert the signal by sRNA</H4><P>We performed the thermal regulation characterization by co-transform our sensor plasmid (TlpA36-pTlpA-sRNA) and reporter plasmid(pTac-RiboJ-J61101-mNeonGreen). Then, they were incubated in 37°C and 30°C, for 20 hours. First, we performed a qualitative assay, by comparing the brightness of the broth.</P><B class=" text-comment">The result showed no significanlty reduce in eexpression, it may be casued by the shortness of our seed region which is for inhibition</B></DIV><DIV class="topic-title" id="section4"><H3>Density Regulator</H3><H4>-can it work</H4><P>In order to demonstrate the density-regulated sensor is well controlled by the concentration of signal molecules (acyl-homoserine lactone, or AHLs) produced, we characterized the quorum sensing system (Lux) with purchased AHL molecules. Different concentrations of AHL were used to induce the transcription of sfGFP, attempting to identify the threshold concentration that turns on the transcription downstream pLux.</P><P>AHL stocks in the concentration of 10<SUP>-3</SUP> M to 10<SUP>-14</SUP> M was prepared by serial dilutions. Then the AHL stocks were added into M9 media culture thousand-fold in volume to obtain M9 with AHL of concentrations ranging from 10<SUBP>-6</SUBP> M to 10<SUP>-14</SUP> M. IPTG was added into M9 to induce the expression of LuxR. </P><P>We first did a rough qualitative experiment to make sure this device is functional.</P><P>From this figure we can see the device is functional.</P><H4>-quantitatively</H4><P>The fluorescent intensity increased dramatically at the concentration of 10<SUBP>-8</SUBP> M, indicating the threshold concentration of AHL for Lux system is around 10<SUP>-8</SUP> M. At least 10<SUP>-7</SUP> M to 10<SUP>-8</SUP> M of AHL is required to activate the expression under the regulation of pLux.</P><H4>-and reverse the signal</H4><P>As mentioned before, we need to convert the signal to a negative response, which we accomplished by adding a PhlF repressor, combining to form part K2572016. PhlF binds to an operator in the pPhlF region to repress the expression of downstream suicide sequences (ccdB/GBSV1/colicin E2). When temperature and cell density are high, regulator PhlF (and sRNA) is produced to repress the expression of suicide genes. As temperature drops and community density decreases, simulating that engineered strains escape from the fermentation condition, PhlF is repressed and consequently turns on the expression of cytotoxic sequences.</P><P>Also a model was built to predict the function of this device. </P></DIV><DIV class="topic-title" id="section5"><H3>Integrase</H3><P>The second device we build is for therapeutic bacteria. The device can carry out noninvasive tracing through ultrasound imaging of the gas vesicle(Shapiro et al.), release the drug (from SHSBNU 2017) controlled by a thermosensitive regulator at nidus by ultrasound tissue heating, and heat to a higher temperature to release nuclease and kill the bacteria after it finishes its mission. </P><H4>-can it work</H4><P>Quantitative Characterization of integrase was not as expected . We gain access to the integrase Bxb1 from 2017 Peking University iGEM team, who constructed a time-sequential logic system with the use of integrase (Peking iGEM2017). The integrase Bxb1 is regulated by the upstream pBAD operon, which activates in the presence of arabinose. </P><P>Theoretically, presence of arabinose will turn on PBAD thus initiates the production of Bxb1. Bxb1 flips the promoter upstream of GFP and starts the transcription of GFP. Conversely, without arabinose induction, no green fluorescence should be observed in bacteria colonies. We assume this was due to the leakage of BAD operator. </P><B class="text-comment">From the fig, there are differences between the one with bxb1 plasmid and the ones without, they glows as expected with their combination. However, as the expression should increase with higher mole of arabinose, the experimental results showed that the expression was decreased.The plasmids co-transformed were extracted out for sequencing, so we can not assess it right away.</B></DIV><DIV class="topic-title" id="section6"><H3>Therapeutic Bacteria</H3><H4>-TlpA39-Vio</H4><P>We constructed the pTlpA-VioABDE-TlpA39, and cultured it under different temperature. </P><B class="text-comment">The fig shows that there was leakage of PVA, which was the molecule produced by VioABDE.</B></DIV><DIV class="topic-title" id="section7"><H3>Capacity Monitor</H3><P>We constructed bacteria with either GFP or VioABDE, and with both of them to monitor the capacity of gene expression. The three bacteria strains were cultured for 12 hours with green fluorescence and OD constantly monitored. </P><B class="text-comment">From the fig, by comparing the fluorescence of GFP-containing E. coli, the fluorescence intensity rose faster and higher for bacteria without VioABDE expression than E. coli with VioABDE inserted and expressed. It signifies that VioABDE expression consumed notable resources in the host cell and caused expression burden in the cell. The fluorescence of VioABDE shows that VioABDE does not emit green fluorescence thus not significantly influencing the results obtained.</B></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>