"
Page
Discussion
View source
History
teams
Log in
 
Team:Caltech/Week 2
From 2010.igem.org
 
iGEM 2010
 
Home
People
Project Details
Protocols
Completed Systems
Notebook
Biosafety
Human Impact
References
Support
Week 1 | Week 2 | Week 3 | Week 4 | Week 5 | Week 6 | Week 7 | Week 8 | Week 9 | Week 10 | Week 11 | Week 12 | Week 13 | Week 14 | Week 15
Monday 6/21
Transformed additional bricks from distribution with other antibiotic resistances onto plates & liquid culture.
I15008 , I15009 , I15010 , E0033 , M30109 , K112000 , J09250
Tuesday 6/22
Transformation success is ambiguous - the plates are covered in a lawn of bacteria, apparently indicating an issue with the antibiotic selection. This implies either a problem with the stock solution of antibiotic (Kan/Strep), or the procedure of plating the antibiotics on the surface of the LB-agar plates.
To find out what happened, we set up a simple experiment for both ampicillin (amp) and streptomyocin (strep):
Take 3 LB-agar plates with no antibiotic and plate 50uL of 1000x, 10x, or 2x antibiotic stock solution on each, spreading with sterile glass beads. Let the plates dry for 1 hour at RT. Additionally, prepare an LB-agar plate containing the desired antibiotic in the agar. Split all four plates in half and streak each plate with both a cell strain that should be resistant to your antibiotic, as well as a strain the should not be. Incubate the plates overnight at 37°C.
If the antibiotic was prepared correctly, and is present in high enough concentration on the plate, only half of the plate should have any colonies present: the side with the desired resistance. Plates with cells on both sides do not have enough antibiotic; those with no cells have too much.
Re-performed the transformations on two of the bricks & plated on LB-agar plates with the antibiotic incorporated.
Wednesday 6/23
Transformation plates are still covered in a film of bacteria.  The negative control for the kanamycin test grew bacteria, suggesting that there is an issue with the kanamycin stock, which we will have to remake.
The amp & strep antibiotic test plates indicate that plating 50uL of 1000x stock is sufficient for proper bacterial selection:
Thursday 6/24
Our NEB Biobrick assembly kit arrived today! We began digesting & ligating bricks to assemble our light induction/lysis construct.
Desired construct: R0082 (light-induced promoter) + B0034 (RBS) + K124017 (lysis gene cassette) + B0015 (terminator) in a non-amp/non-cm backbone (since our light transduction brick, M30109, is A/C resistant).
Began five digests according to the NEB Biobrick assembly kit protocol:
R0082 (us)
B0034 (ds)
K124017 (us)
B0015 (ds)
Cm linear backbone ( pSB1C3 )
Began two ligation reactions according to the Biobrick assembly kit protocol:
R0082 / B0034 / pSB1C3
K124017 / B0015 / pSB1C3
Transformed the ligation products into DH5alpha (Invitrogen) competent cells via heat shock and plated (incorrectly) on LB-amp plates.
Set up antibiotic test plates for kanamycin (kan) & tetracycline (tet), just as on 6/22 .
Friday 6/25
Re-transformed the ligation products from 6/24 into DH5alpha cells via heat shock and both: plated 100μL of cells on LB-cm plates and added remainder to create LB-cm liquid culture. Both were incubated overnight at 37°C.
Checked kan/tet antibiotic test plates: again, plating 50μL of 1000x stock antibiotic appears to be adequate for proper selection. Unfortunately, the XL1 Blue cells we used last week do not appear to actually be tet-resistant (as advertised).
Designed forward & reverse sequencing primers, using the sequences of G00100 & G00101 , which bind to the standard prefix & suffix of the provided plasmid backbone. Note: according to the IDT oligo analyzer , the forward primer creates a strong (~9kcal/mol) homodimer, which melts at about the primer melting temp of 55°C (according to NUPACK analysis). We decided to take a chance and order the G00100 & G00101 provided sequence since it is specified by the Registry (and therefore hopefully works), even though a better primer could conceivably have been designed (which would probably require different forward primers for each different plasmid). Placed the order for these primers with IDT today.
Protip from the future: We eventually figured out that these primers bind to the wrong location on the plasmids we were trying to sequence. We have since created new sequencing primers - see Week 6 .
Made LB-agar plates (blank, tet, kan)
Weekend 6/26-27
Saturday 6/26
Mini-prepped the transformation liquid cultures of the ligation products from 6/25 to extract the plasmid DNA. The pBAD18 positive control contained > 500 colonies (while the pBAD cells in the LB-cm liquid culture did not grow, as expected). Each of the LB-cm agar plates contained 3 colonies, indicating that we should transform > 1μL of ligation DNA in future transformations or change the ligation reaction concentrations.
Concentration (ng/μL)
Ligation 1: 75.5
Ligation 2: 86.0
Began 3 digestions on the miniprepped DNA:
Ligation prod 1 (promoter + RBS, us)
Ligation prod 2 (lysis gene cassette + terminator, ds)
Linear backbone, tet resistant ( pSB1T3 )
Began a ligation reaction using 4uL of digest DNA and 2uL backbone digest DNA (20μL total):
LigProd1/LigProd2/ pSB1T3
Transformed the ligation product into alpha cells via heat shock (2μL ligation product, 1μL pBAD18 positive control DNA in 20μL aliquots of competent cells).
Plated 100μL of ligation product cells on an LB-tet agar plate, 100μL of pBAD control cells on LB-amp agar.
Transferred remainder of cells of each into 5mL LB-tet liquid culture.
Incubated all overnight at 37°C.
Sunday 6/27
Transformation results: no colonies on the plate and no growth in the liquid culture. Either the ligation or the transformation failed entirely for some reason. We will verify the ligation product via restriction digest/gel electrophoresis on Monday.
Update: An alternative explanation is that because the construct was not transformed into ΔenvZ cells, the light promoter should be constitutively activated, possibly causing all transformants to express lysis proteins. To test, we will transform our ligation product into ΔenvZ cells ( V1012 ) later in the week, when they arrive from the Registry.
 
Error creating thumbnail: Unable to create destination directory
Retrieved from " http://2010.igem.org/Team:Caltech/Week_2 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
