Team:Nankai/Experiments
Medigo Blue, free responsive template
Home
Research & Lab
Project
Safety
Measurements
Parts
Member
Team
Attributions
Collaborations
Human Practices
Modeling
iShare
iShare
Modeling
Human Practices
Member
Team
Attributions
Collaborations
Research & Lab
Project
Safety
Measurements
Parts
Home
menu
Your place:  Home  >  Project  >  Experiment
Experiment & Protocol
1. Strains, plasmids and growth conditions
For routine strain construction and maintenance, the B. amyloliquefaciens and E. coli strains were grown at 37°C in Luria-Bertani (LB) medium. To produce γ-PGA, the B. amyloliquefaciens strains were cultured at 37°C and 180 rpm, for 48 h in γ-PGA fermentation medium (pH 7.2 ) , containing sucrose 50 g/L, (NH4)2SO4 2 g/L, MgSO4 0.6 g/L, KH2PO4 6 g/L, K2HPO4 14 g/L, 2 mL mineral elements including 1 mM FeSO4•4H2O, CaCl2•2H2O, MnSO4•4H2O and ZnCl2. And the concentrations of the antibiotics used in this work were as follow: 100 μg/mL ampicillin, 5 μg/mL chloramphenicol and 7.5 μg/mL erythromycin. The final concentration of 5-FU was 100 μg/mL.
2. DNA manipulation and plasmids construction
Six candidate promoters were digested by double endonucleases (listed in Table 1, Thermo Scientific, U.S.A.), and ligated to a verification vector pCB with a reporter gene bgaB that encodes β-galatosidase downstream of the multiple cloning sites (MCS).
Table 1. The specific endonucleases used for verification plasmid construction
To construct the XylR expression plasmid, the xylR gene from pWH1520 was ligated to the pHT01 plasmid by Bam HI and the generated plasmid was designated as was pHT01-xylR.
The construction of recombinant plasmid required a temperature-sensitive vector pKSU and fusion fragments carrying up-/downstream homologous arms of 600bp for each and specific promoter. The homologous arms were amplified from LL3 genome with Ex Taq polymerase (TaKaRa Bio, Japan) using specific primers N-SF/N-SR and N-XF/N-XR (N refers to the relevant name of the promoter, detailed in Table 2), respectively, while promoter sequence was amplified with primer N-F/N-R. Then the fusion fragments were spliced by means of overlapping PCR with LA Taq polymerase (TaKaRa Bio, Japan) using primer pairs PE-SF/PE-SR. After digested by Bam HI and SalI, all the fusion fragments were ligated to the similarly digested pKSU. They would be transformed into NK-1 once the DNA sequencing (Genewiz, China) was acceptable.
Table 2. Primers used in fusion fragments construction, and restriction sites used for the cloning of PCR ampilcons are indicated in italics and underline.
E. coli competent cells were purchased from Transgen Biotech (Beijing, China) and transformed according to the manufacturer’s instructions. Promoter replacement plasmids were transformed into B. amyloliquefaciens strains by electroporation (2,100 v).
3. Markerless gene replacement method
Promoters replacements in this study were carried out by a markerless gene replacement method reported previously and will be described briefly as below. B. amyloliquefaciens NK-1carrying an in-frame deletion of upp and resistant to 1.3 mM 5-fluorouracil (5-FU) was used as the parental strain for subsequent mutants construction. Introduction of the derivatives of plasmid pKSU would restore sensitivity to 5-FU for NK-1 and its derivatives.
Replacement of the promoter Pxyl will be used as an example to clarify the method. The upstream, downstream homologous arms and promoter Pxyl were obtained using primers odhS-F/odhS-R, odhX-F/odhX-R and TP-F/TP-R respectively. The three fragments were then combined via overlapping PCR using primers odhS-F/odhX-R. The resulting fragment was then ligated into pKSU to yield the plasmid pKSU-TP, which was then transformed in NK-1 in the presence of Cm at 30ºC. The recombinants were cultured at 42ºC in the presence of Cm to facilitate chromosomal integration. The obtained single-crossover recombinants were then grown in LB in the absence of Cm. Cultures were diluted and plated in LB agar supplemented with 5-FU. Target strains were identified by PCR using primers odh-SS/odh-XX and DNA sequencing. The Pxyl promoter replacement strain was designated NK-TP. The other promoter replacement mutants were similarly constructed.
4. Metabolic toggle switch construction
We transformed plasmid pHT01-xylR into NK-TP strain, and got a new strain named NK-TP-xylR, which contains all gene parts of metabolic toggle switch. We also transformed plasmids pHT01-xylR and pCB-Pxyl into NK-1 strain, to verify the activity of metabolic toggle switch.
5. Metabolic toggle switch verification
Fresh colonies of B. amyloliquefaciens strains (NK-1 strain containing plasmids pHT01-xylR and pCB-Pxyl and the control NK-1 strain containing plasmids pHT01 and pCB-Pxyl) were first cultured overnight in test tubes containing 5 mL LB liquid and then inoculated into 100 mL fresh fermentation medium. We added 1mM IPTG into the medium after 12h of cultivation. The β-galactosidase activity were measured at 12h, 18h, 24h, 30h, 36h, 42h to test the effect of metabolic toggle switch on the expression of bgaB.
6. β-galactosidase activity assay
Briefly, 0.15ml fermentation liquid, 0.375ml Z buffer (NaH2PO4·7H2O 60 mM, NaH2PO4 40 mM, KCl10mM, MgSO4·7H20 1 mM, β-mercaptoethanol 50 mM ) and 0.075ml lysozyme solution (4mg/ml) were mixed in 1.5ml centrifuge tube. The mixture was maintained at 37°C for 10 mins. Then 6μl 10% Triton X-100 was added to the solution. The solution was ice-bathed and then maintained at 55℃ for 3-5 mins.
o-Nitrophenyl-β-D-galactoside (ONPG) solution (4mg/ml) was added (0.15ml), and the mixture was maintained at 55℃ for 10 mins. Then 0.3ml Na2CO3 solution was added into the system to terminate the reaction. The resulting solution was centrifuged at 12000 rpm for 5 mins and absorbance of the supernatant at 420 nm was measured.
7. Metabolic toggle switch optimal experiments
To find out the most proper IPTG adding time, we added the same concentration (1mM) IPTG into fermentation flasks at different time points (0h, 3h, 6h, 9h, 12h, 15h, 18h, 21h, 24h).
We added different concentrations (1mM, 5mM, 10mM, 25mM) of IPTG at 15 h to verified their effects on γ-PGA production in NK-TP-xylR strain to obtain the optimized IPTG concentration.
8. γ-PGA production
For growth experiments, 1 mL cultures were withdrawn periodically for OD600 measurement. After 48 h of cultivation, 100 mL cultures were centrifuged at 8, 000 rpm (4ºC) for 20 min. The cell pellet was washed three times with dH2O then dried and weighed to determine DCW. The supernatant was used to extract γ-PGA using an ethanol precipitation method. Experiments were independently repeated at least three times and means and standard deviations were calculated.
9. Quantitative RT-PCR analysis of the odhA gene
The wild-type NK-1and its derivatives with different concentration of IPTG were grown to mid-log phase (36 h) and plateau (48h) in fermentation medium. The cells were collected at 4ºC and the RNA was isolated using TransZolTM Up (TransGen, Beijing, China) according to the manufacturer’s instructions. cDNA was reverse transcribed using a GoScriptTM Reverse Transription System (Promega, Wisconsin, USA). Real-time PCR analysis for the target genes was performed using the SYBR® Premix Ex Taq™ II (Takara, Dalian, China). Transcription levels of the target gene were normalized against the levels of rspU.
Description
Background
Experiments & Protocols
Results
Design - Pudding Health Kit
Inside laboratory animal room.
Incision enzyme kit.
Special table for IGEM.
Dialysis, purification of γ-PGA.
Dialysis, purification of γ-PGA.
Prepare for fermentation medium.
Medigo Blue, free responsive template
Copyright © Nankai University
Nankai IGEM Team 2015
