<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Nankai_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Nankai/Experiments</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Medigo Blue, free responsive template</TITLE></P><DIV class="responsive_menu"><UL class="main_menu"><LI><A href="https://2015.igem.org/Team:Nankai">Home</A></LI><LI><A href="#">Research &amp; Lab</A></LI><LI><A href="https://2015.igem.org/Team:Nankai/Description">Project</A></LI><LI><A href="https://2015.igem.org/Team:Nankai/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Nankai/Measurement">Measurements</A></LI><LI><A href="https://2015.igem.org/Team:Nankai/Parts">Parts</A></LI></UL><LI><A href="#">Member</A></LI><LI><A href="https://2015.igem.org/Team:Nankai/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Nankai/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Nankai/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Nankai/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Nankai/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Nankai/Practices/iShare" target="_blank">iShare</A></LI></DIV><HEADER class="site-header"><DIV><DIV><DIV><DIV class="main-navigation"><NAV class="main-nav visible-md visible-lg"><UL class="sf-menu"><LI id="sideli"><A href="https://2015.igem.org/Team:Nankai/Practices/iShare" target="_blank">iShare</A></LI><LI><A href="https://2015.igem.org/Team:Nankai/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Nankai/Practices">Human Practices</A></LI><LI><A href="#">Member</A></LI><LI><A href="https://2015.igem.org/Team:Nankai/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Nankai/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Nankai/Collaborations">Collaborations</A></LI></UL><LI><A href="#">Research &amp; Lab</A></LI><LI><A href="https://2015.igem.org/Team:Nankai/Description">Project</A></LI><LI><A href="https://2015.igem.org/Team:Nankai/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Nankai/Measurement">Measurements</A></LI><LI><A href="https://2015.igem.org/Team:Nankai/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Nankai">Home</A></LI></NAV><DIV class="hidemenu"><P><A href="#">menu</A></P></DIV></DIV></DIV></DIV></DIV></HEADER><DIV class="first-widget parallax" id="blog"><DIV class="parallax-overlay"><P>Your place: <A href="https://2015.igem.org/Team:Nankai">Home</A> &gt; <A href="https://2015.igem.org/Team:Nankai/Description">Project</A> &gt; <A href="https://2015.igem.org/Team:Nankai/Experiments">Experiment</A></P><DIV class="container pageTitle"><DIV class="row"><DIV class="col-md-6 col-sm-6"><H2 class="page-title">Experiment &amp; Protocol</H2></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-md-8 blog-posts"><H4>1. Strains, plasmids and growth conditions</H4><P>For routine strain construction and maintenance, the <EM>B. amyloliquefaciens</EM> and <EM>E. coli</EM> strains were grown at 37°C in Luria-Bertani (LB) medium. To produce γ-PGA, the <EM>B. amyloliquefaciens</EM> strains were cultured at 37°C and 180 rpm, for 48 h in γ-PGA fermentation medium (pH 7.2 ) , containing sucrose 50 g/L, (NH<SUB>4</SUB>)<SUB>2</SUB>SO<SUB>4</SUB> 2 g/L, MgSO<SUB>4</SUB> 0.6 g/L, KH<SUB>2</SUB>PO<SUB>4</SUB> 6 g/L, K<SUB>2</SUB>HPO<SUB>4</SUB> 14 g/L, 2 mL mineral elements including 1 mM FeSO<SUB>4</SUB>•4H<SUB>2</SUB>O, CaCl<SUB>2</SUB>•2H<SUB>2</SUB>O, MnSO<SUB>4</SUB>•4H<SUB>2</SUB>O and ZnCl<SUB>2</SUB>. And the concentrations of the antibiotics used in this work were as follow: 100 μg/mL ampicillin, 5 μg/mL chloramphenicol and 7.5 μg/mL erythromycin. The final concentration of 5-FU was 100 μg/mL.</P><H4>2. DNA manipulation and plasmids construction</H4><P>Six candidate promoters were digested by double endonucleases (listed in Table 1, Thermo Scientific, U.S.A.), and ligated to a verification vector pCB with a reporter gene <EM>bgaB</EM> that encodes β-galatosidase downstream of the multiple cloning sites (MCS). </P><DIV class="smallpic"><H6>Table 1. The specific endonucleases used for verification plasmid construction</H6></DIV><P>To construct the XylR expression plasmid, the <EM>xylR</EM> gene from pWH1520 was ligated to the pHT01 plasmid by <EM>Bam</EM> HI and the generated plasmid was designated as was pHT01-<EM>xylR</EM>.</P><P>The construction of recombinant plasmid required a temperature-sensitive vector pKSU and fusion fragments carrying up-/downstream homologous arms of 600bp for each and specific promoter. The homologous arms were amplified from LL3 genome with <EM>Ex Taq</EM> polymerase (TaKaRa Bio, Japan) using specific primers N-SF/N-SR and N-XF/N-XR (N refers to the relevant name of the promoter, detailed in Table 2), respectively, while promoter sequence was amplified with primer N-F/N-R. Then the fusion fragments were spliced by means of overlapping PCR with <EM>LA Taq</EM> polymerase (TaKaRa Bio, Japan) using primer pairs PE-SF/PE-SR. After digested by <EM>Bam</EM> HI and <EM>Sal</EM>I, all the fusion fragments were ligated to the similarly digested pKSU. They would be transformed into NK-1 once the DNA sequencing (Genewiz, China) was acceptable.</P><DIV class="smallpic"><H6>Table 2. Primers used in fusion fragments construction, and restriction sites used for the cloning of PCR ampilcons are indicated in italics and underline.</H6></DIV><P><EM>E. coli</EM> competent cells were purchased from Transgen Biotech (Beijing, China) and transformed according to the manufacturer’s instructions. Promoter replacement plasmids were transformed into <EM>B. amyloliquefaciens</EM> strains by electroporation (2,100 v).</P><H4>3. Markerless gene replacement method</H4><P>Promoters replacements in this study were carried out by a markerless gene replacement method reported previously and will be described briefly as below. <EM>B. amyloliquefaciens</EM> NK-1carrying an in-frame deletion of <EM>upp</EM> and resistant to 1.3 mM 5-fluorouracil (5-FU) was used as the parental strain for subsequent mutants construction. Introduction of the derivatives of plasmid pKSU would restore sensitivity to 5-FU for NK-1 and its derivatives. </P><P>Replacement of the promoter P<SUB>xyl</SUB> will be used as an example to clarify the method. The upstream, downstream homologous arms and promoter <SUB>Pxyl</SUB> were obtained using primers odhS-F/odhS-R, odhX-F/odhX-R and TP-F/TP-R respectively. The three fragments were then combined via overlapping PCR using primers odhS-F/odhX-R. The resulting fragment was then ligated into pKSU to yield the plasmid pKSU-TP, which was then transformed in NK-1 in the presence of Cm at 30ºC. The recombinants were cultured at 42ºC in the presence of Cm to facilitate chromosomal integration. The obtained single-crossover recombinants were then grown in LB in the absence of Cm. Cultures were diluted and plated in LB agar supplemented with 5-FU. Target strains were identified by PCR using primers odh-SS/odh-XX and DNA sequencing. The <SUB>Pxyl</SUB> promoter replacement strain was designated NK-TP. The other promoter replacement mutants were similarly constructed.</P><H4>4. Metabolic toggle switch construction</H4><P>We transformed plasmid pHT01-<EM>xylR</EM> into NK-TP strain, and got a new strain named NK-TP-<EM>xylR</EM>, which contains all gene parts of metabolic toggle switch. We also transformed plasmids pHT01-<EM>xylR</EM> and pCB-<SUB>Pxyl</SUB> into NK-1 strain, to verify the activity of metabolic toggle switch.</P><H4>5. Metabolic toggle switch verification</H4><P>Fresh colonies of <EM>B. amyloliquefaciens</EM> strains (NK-1 strain containing plasmids pHT01-<EM>xylR</EM> and pCB-<SUB>Pxyl</SUB> and the control NK-1 strain containing plasmids pHT01 and pCB-Pxyl) were first cultured overnight in test tubes containing 5 mL LB liquid and then inoculated into 100 mL fresh fermentation medium. We added 1mM IPTG into the medium after 12h of cultivation. The β-galactosidase activity were measured at 12h, 18h, 24h, 30h, 36h, 42h to test the effect of metabolic toggle switch on the expression of <EM>bgaB</EM>.</P><H4>6. β-galactosidase activity assay</H4><P>Briefly, 0.15ml fermentation liquid, 0.375ml Z buffer (NaH<SUB>2</SUB>PO<SUB>4</SUB>·7H<SUB>2</SUB>O 60 mM, NaH<SUB>2</SUB>PO<SUB>4</SUB> 40 mM, KCl10mM, MgSO<SUB>4</SUB>·7H<SUB>2</SUB>0 1 mM, β-mercaptoethanol 50 mM ) and 0.075ml lysozyme solution (4mg/ml) were mixed in 1.5ml centrifuge tube. The mixture was maintained at 37°C for 10 mins. Then 6μl 10% Triton X-100 was added to the solution. The solution was ice-bathed and then maintained at 55℃ for 3-5 mins.</P><P><EM>o</EM>-Nitrophenyl-<EM>β</EM>-D-galactoside (ONPG) solution (4mg/ml) was added (0.15ml), and the mixture was maintained at 55℃ for 10 mins. Then 0.3ml Na<SUB>2</SUB>CO<SUB>3</SUB> solution was added into the system to terminate the reaction. The resulting solution was centrifuged at 12000 rpm for 5 mins and absorbance of the supernatant at 420 nm was measured.</P><H4>7. Metabolic toggle switch optimal experiments</H4><P>To find out the most proper IPTG adding time, we added the same concentration (1mM) IPTG into fermentation flasks at different time points (0h, 3h, 6h, 9h, 12h, 15h, 18h, 21h, 24h).</P><P>We added different concentrations (1mM, 5mM, 10mM, 25mM) of IPTG at 15 h to verified their effects on γ-PGA production in NK-TP-<EM>xylR</EM> strain to obtain the optimized IPTG concentration.</P><H4>8. γ-PGA production</H4><P>For growth experiments, 1 mL cultures were withdrawn periodically for OD<SUB>600</SUB> measurement. After 48 h of cultivation, 100 mL cultures were centrifuged at 8, 000 rpm (4ºC) for 20 min. The cell pellet was washed three times with dH<SUB>2</SUB>O then dried and weighed to determine DCW. The supernatant was used to extract γ-PGA using an ethanol precipitation method. Experiments were independently repeated at least three times and means and standard deviations were calculated.</P><H4>9. Quantitative RT-PCR analysis of the <EM>odhA</EM> gene</H4><P>The wild-type NK-1and its derivatives with different concentration of IPTG were grown to mid-log phase (36 h) and plateau (48h) in fermentation medium. The cells were collected at 4ºC and the RNA was isolated using <EM>TransZolTM</EM> Up (TransGen, Beijing, China) according to the manufacturer’s instructions. cDNA was reverse transcribed using a GoScriptTM Reverse Transription System (Promega, Wisconsin, USA). Real-time PCR analysis for the target genes was performed using the SYBR® Premix Ex <EM>Taq™</EM> II (Takara, Dalian, China). Transcription levels of the target gene were normalized against the levels of <EM>rspU</EM>.</P></DIV><DIV class="col-md-4"><DIV class="sidebar"><DIV class="sidebar-widget"><H6><A href="https://2015.igem.org/Team:Nankai/Description">Description</A></H6><H6><A href="https://2015.igem.org/Team:Nankai/project_background">Background</A></H6><H6><A href="https://2015.igem.org/Team:Nankai/Experiments">Experiments &amp; Protocols</A></H6><H6><A href="https://2015.igem.org/Team:Nankai/Results">Results</A></H6><H6><A href="https://2015.igem.org/Team:Nankai/Design">Design - Pudding Health Kit</A></H6></DIV><DIV class="sidebar-widget"><P>Inside laboratory animal room.</P><P>Incision enzyme kit.</P><P>Special table for IGEM.</P><P>Dialysis, purification of γ-PGA.</P><P>Dialysis, purification of γ-PGA.</P><P>Prepare for fermentation medium.</P></DIV></DIV></DIV></DIV></DIV><TITLE>Medigo Blue, free responsive template</TITLE><FOOTER class="site-footer"><DIV class="copyright-content"><DIV class="copyright-text"><P>Copyright © Nankai University </P><P> Nankai IGEM Team 2015</P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>