Team:Melbourne/Experiments
Melbourne iGEM 2016
Toggle navigation
Home
Team
Team Members
Sponsors
Attributions
Collaboration
Project
Description
Design
Experiments
Proof of Concept
Demonstrate
Results
Notebook
Modelling
Safety
Parts
Human Practices
Achievements
Melbourne iGEM 2016
Experiments and Protocols
Experiment Protocols:
Part:Day:Titles: 1.1.Transformation buffer mixing, autoclaving &
cell plating
Date: 29/6/16
Experiment Planner: Rob
Experimenters: Rob and Jeff
Supervisor: Dr. Amber Willems-Jones
Lab book pages: Unclear
Materials: 
SOC:
20g tryptone, 5g yeast extract, 0.584g NaCl, 0.186g KCl, 1L ddH2O, 10mM MgCl2 (hexahydrate 2 g) AND 10mM MgSO4 (heptahydrate 2.467g), Glucose 3.6g.
CaCl2:
55.5 g
LB: Tryptone 18g, Yeast Extract 9g, NaCl 18g.
 
Time Breakdown: 
2 hours for mixing buffers, 8 hours for autoclaving (we can pick them up the next day I’d say)
1 hour for plating
3 hours total.
Method: Teaching lab protocol in experiment booklet
CaCl2: Make a 500L stock solution of 1M CaCl2, transfer into 10mL aliquots, freeze. Thaw and dilute when needed (Mol. Cloning pg 1.82)
Plating: LB plates in double door fridge, second shelf. Streak plating.
Output:
Sterile SOC, CaCl2 and LB
Relevant SWPs:
Streak plating SWP
Proper use of an autoclave SWP
Part:Day:Titles: Week 2 Day 1. Transform Cells and Test for Cell Competency
Start Date:4/7/16
Experiment Planner:
Ella
Eileen
Jeffrey
Experimenters:
Ella
Rob
Jeffrey
Supervisor:
Amber
Lab book start pages:
3
Materials:
Plasmids (pET-23a and pSB-1C3)
SOC media (200µl per transformation)
Competent Cells (50µl per transformation)
Petri plates w/ LB agar and antibiotic (2 plates per transformation)
Chloramphenicol (1uL per 1ml of agar)
Ampicillin (1uL per 1mL of agar)
Bunsen Burner
Sterile Petri Dishes
Sterile measuring flask
Foam tube rack
Ice
Water bath (42 degC)
Incubator (37 degC)
Sterile spreader
iGEM distributed Competent Cell Test Kit (contains pSB-1C3 DNA at 5 different concentrations)
Time Breakdown:
1.5 hour making LB plates and plasmid preparation
2 hours incubation
1 hour for plating
Overnight incubation of plates (14-18hrs incubation).
NOTE: this experiment needs to be done at the end of the day so that the plates do not incubate for longer than 18hrs. Someone must be in the lab the next morning to transfer the plates to the fridge.
Method:
Protocol:
Transformation protocol: http://parts.igem.org/Help:Protocols/Transformation
Competent cell test kit protocol
http://parts.igem.org/Help:Competent_Cell_Test_Kit
Plating protocol: Protocol A3 Making LB Agar Plate in the 2014 teams protocol book.
LB plates:
Make LB plates with antibiotics according to protocol A3 in lab protocol folder. Add the appropriate amount of antibiotics before pouring into petri dish (33 ug/mL Chloramphenicol for pSB-1C3 transformation and 100 ug/mL Ampicillin for pET-23a transformation) This can be done during the 37deg 2 hour incubation period during the transformation process. For each transformation you need two LB plates.
Preparation prior to transformation of cells:
There are 5 transformations of plasmid into DH5a cells: pSB1C3, pET23a, 0.5 pg/ul DNA, 0.5 pg/ul DNA, 5 pg/ul DNA and 10 pg/ul DNA from Competent Cell Test Kit
There are 3 transformations of plasmid into BL21 cells: 0.5 pg/ul DNA, 5 pg/ul DNA and 10 pg/ul DNA from Competent Cell Test Kit.
The transformation of DNA from Competent Cell Test Kit will test for the transformation efficiency of competent cells and also act as positive controls.
Preparation of pSB1C3 plasmid
A plasmid clone containing part BBa_B0030 will be used. It is located at Well 4G of 2016 Kit Plate 4 (column 4 row G)
Resuspend DNA Distribution Kit well with 10µl dH20. Pipet up and down several times, let sit for a few minutes. Resuspension will be red from cresol red dye.
[T--Melbourne--1_Protocols.jpg]
Preparation of pET23a plasmid
1uL of a 1/25 dilution of the standard pET23a solution and then make 20 ng/uL working solution in DNA/RNA work box
Preparation of plasmid DNA from Cell Competency Test Kit
Spin down the DNA tubes from the Competent Cell Test Kit/Transformation Efficiency Kit to collect all of the DNA into the bottom of each tube prior to use. A quick spin of 20-30 seconds at 8,000-10,000 rpm will be sufficient
Set water bath to 42°C. Use a thermometer for accurate measurement. Do not trust the setting on the water bath.
Transformation of cells:
Follow the transformation protocol found on the iGEM website: http://parts.igem.org/Help:Protocols/Transformation .
Thaw 5 DH5a competent cells tubes and 3 BL21 competent cells tubes on ice. Tubes should be labeled, pre-chilled, and in a floating tube rack for support. Keep all tubes on ice.
Also prepared 2 tubes of DH5a and 2 tubes BL21. These tubes will not be transformed but will undergo the same process as other tubes. They are then plated on the plates with appropriate antibiotic as negative control.
Pipette 1µl of prepared plasmid DNA into competent cell tubes: Pipette from well into appropriately labeled tube. Gently pipette up and down a few times. Keep all tubes on ice.
Close 2ml tubes, incubate on ice for 30min: Tubes may be gently agitated/flicked to mix solution, but return to ice immediately.
Heat shock tubes at 42°C in waterbath for 1 min: 2ml tubes should be in a floating foam tube rack. Place in water bath to ensure the bottoms of the tubes are submerged. Timing is critical.
Incubate on ice for 5min: Return transformation tubes to ice bucket.
Pipette 200µl SOC media to each transformation: SOC should be stored at 4°C, but can be warmed to room temperature before use. Check for contamination.
Incubate at 37°C in a shaking incubator for 2 hours
For pSB1C3 and pET21a transformation: Pipette each transformation on two petri plates for a 20µl and 200µl plating: Pipette 20µl and 200µl of the transformation onto appropriately labeled plates. Spread with sterilized spreader immediately. This helps ensure that you will be able to pick out a single colony.
For DNA from Cell Competency Kit: Pipet 20 µL from each tube onto the appropriate plate, and spread the mixture evenly across the plate. Do at least two plates. Triplicates (3 each) of each tube if possible, so you can calculate an average colony yield.
For negative control: Pipet 20 µL from each tube onto the appropriate plate, and spread the mixture evenly across the plate.
For lawn growth control: Pipet 20 µL of the transformed cell onto the plate without antibiotic.
Incubate transformations overnight (14-18hr) at 37°C: Incubate the plates upside down (agar side facing up, let the agar surface become semi-dry before putting it upside down). If incubated for too long, colonies may overgrow and the antibiotics may start to break down; un-transformed cells will begin to grow.
See the next experiment plan for steps after incubation
Output:
24 plates containing transformed bacteria
pSB1C3 into DH5a
pET23a into DH5a
0.5 pg/ul DNA from test kit into DH5a
5 pg/ul DNA from test kit into DH5a
10 pg/ul DNA from test kit into DH5a
0.5 pg/ul DNA from test kit into BL21
5 pg/ul DNA from test kit into BL21
10 pg/ul DNA from test kit into BL21
Relevant SWPs:
Agar Plating
Results/Comments:
5/7: After overnight incubation of plate, all controls were as expected. However, there was extremely little growth on any of the triplicates and only one colony on either of pSB1C3 transfected DH5a cell. There are many many transformation with pET23a transformation of DN5a cells. This probably means that the cells are not competent at pg level of DNA transformation but are competent at ng level of DNA transformation. Refer to p.6 of lab book for results.
Brent’s DH5a ultracompetent cells were transformed with pSB1C3, hoping that we will get colonies that can be used for inoculation and plasmid extractions. Chloramphenicol concentration was adjusted from 33 ug/ml to the standard 25 ug/mL with pure ethanol. Refer to p.7 of lab book for details.
6/7: 10-20 colonies were found on the plates with transfected Brent’s cells. It will be used for inoculation and plasmid extraction. Refer to p.7 of lab book for details.
Part:Day:Titles: 2.2 Check cell competency (transformation efficiency), Inoculate/Grow up the cell for downstream experiments (glycerol stock, miniprep, midiprep
Start Date: 5/7/16
Experiment Planner:
Eileen
Jeffrey
Experimenters:
Ella
Rob
Kimber
Jeffrey
Supervisor:
Amber
Leon
Lab book start pages:
8
Materials:
Calculator
LB media
Chloramphenicol and Ampicilin
Bunsen burner
Wire loop
Ethanol
Time Breakdown
15 min calculation
30 mins inoculation
Overnight growing
Method:
Protocol: http://parts.igem.org/Help:Competent_Cell_Test_Kit
Calculating cell competency:
Count the number of colonies on a light field or a dark background, such as a lab bench. Use the equation to calculate your competent cell efficiency (see http://parts.igem.org/File:CompCellsTest_Calculation.xls ). If you've done triplicates of each sample, use the average cell colony count in the calculation.
(colonies on plate) / ng of DNA plated x 1000ng/µg
Note: The measurement "ng of DNA plated" refers to how much DNA was plated onto each agar plate, not the total amount of DNA used per transformation. You can calculate this number using the following equation:
1 µL x concentration of DNA (refer to vial) x (volume plated / total reaction volume)
Competent cells should have an efficiency of 1.5x10^8 to 6x10^8 cfu/µg DNA
Here are some sample results:
DNA concentration
0.5pg/ul
5pg/ul
10pg/ul
20pg/ul
50pg/ul
# of colonies
10 - 20
120 - 170
280 - 360
480 - 802
500 - 1000+
Grow the cells overnight for glycerol stock and midiprep:
Prepare LB media containing antibody (25 uL/mL chloramphenicol for pSB1C3 and 100 uL/mL ampicillin for pET23a)
Pick a colony and inoculate the media following Bacterial Broth Innoculation SOP
Incubate bacterial culture at 37°C for 12-18hr in a shaking incubator
After incubation, check for growth, which is characterised by a cloudy haze in the media
Output:
Cell cultures
Relevant SWPs:
Bacterial Broth Innoculation SOP
Results/Comments:
5/7: Due to poor results of transformation, the cell competency test is delayed and will be performed at later time.
6 tubes of inoculation were made
pSB1C3 (1) colony in 25 mL LB-Cam
pET23a (1) colony in 25 mL LB-Amp
pET23a (2) colony in 25 mL LB-Cam
pET23a (3) colony in 25 mL LB-Cam
No colony in 25 mL LB-Cam
No colony in 25 mL LB-Amp
A large growing flask of LB-Amp was also inoculated for midiprep in the future. All tubes and the flask were incubated in shaking incubator overnight. Refer to p.8 of lab book.
6/7: The colonies of Brent’s DH5a cells transfected with pSB1C3 were used to inoculate LB media. The colony chosen from 200 uL transfection volume plate was used to inoculate 200 mL Lb in a conical flask for midiprep in the future. The colony chosen from 20 uL transfection volume plate was used to inoculate 25 uL LB in a falcon tube. Refer to p.12 of the lab book
Part:Day:Titles: 3.1 Make glycerol stock, miniprep (+restriction digest and gel electrophoresis)
Start Date: 6/7/16
Experiment Planner:
Kimber
Jeffrey
Experimenters:
Kimber
Ella
Rob
Jeffrey
Supervisor:
Amber
Leon
Lab book start pages:
9
Materials:
Cell cultures from the previous day
Glycerol
dH2O
FavorPrep Plamid Extraction Mini Kit
Agarose Gel
RedSafe DNA dye
Agarose
0.5x TBE buffer
6x Loading dye
DNA Ladder
10x NEB CutSmart buffer
EcoRI and Xhol restriction enzymes
Time Breakdown
30 min making glycerol stock
1-1.5 hours miniprep
1.5 hours restriction digest
1 hour gel electrophoresis and imaging
3.5 hours midiprep
Method:
Make Glycerol stock:
Make glycerol stocks for both DH5a cells transfected with pET23a and pSB1C3
After incubation, check for growth, which is characterised by a cloudy haze in the media
Pellet cells from larger volume of culture and remove supernatant. And then resuspend in smaller volume (1 mL) of LB without antibiotics.
Add glycerol (250 uL) to achieve 20 final glycerol concentration and mix by pipetting up and down.
Make replicates of glycerol stock if needed.
Freeze the glycerol stock tube at -80°C. Place the tubes in glycerol stock box located in the rack third from the right second row in -80 °C freezer. The stock is now stable for years, as long as it is kept at -80°C. Subsequent freeze and thaw cycles reduce shelf life.
Miniprep to confirm the presence of plasmid:
Purify plasmids (pET23a and pSB1C3) from DH5a culture following the manufacturer’s protocol.
Restriction digest and gel electrophoresis:
Follow NEB restriction digest protocol for restriction digest and lab protocol B2 for gel electrophoresis
Prepare 1.2% agarose gel following lab protocol B4
Place EcoRI-HF (EcoRI cannot be used) and Xhol RE on ice
Prepare the reaction
2 uL 10x NEB CutSmart buffer
x uL DNA (volume depends on concentration, 1µg in reaction recommended)
1 uL EcoRI-HF
1 uL XhI
16-x uL dH2O
 Incubate at optimum temperature for 1 hrs at 37 degC or adjust time according to amount of DNA and activity of enzyme
Heat inactivate at 65 degC (this step can be skipped)
Place gel in the gel electrophoresis apparatus and cover it with 0.5x TBE
Load reaction with dye and marker on gel and run it at 130-150V until the blue dye has migrated towards the end of the gel
Image the gel. It should look something similar to the following picture.
[T--Melbourne--2_Protocols.jpg]
pET23a fragment size: 34, 3632 bp
pSB1C3 fragment size: 124 bp, 892 bp, 1069 bp
Output:
Glycerol stock of DH5a cells transfected with pET23a and pSB1C3
Miniprep plasmid of the cell culture
Midiprep plasmid of the cell culture (if we do it)
Relevant SWPs:
Results/Comments:
6/7: pSB1C3(1), pET23a(1), pET23a(2), pET23a(3) in DH5a cells were “miniprepped” and the extracted plasmids were stored in the freezer.
pET23a (4) was recultured in a new LB medium in a conical flask
Four glycerol stocks were made:
pSB1C3(1) in DH5a cells
pET23a(1) in DH5a cells
pET23a(2) in DH5a cells
pET23a(3) in DH5a cells
7/7: pSB1C3(2), pSB1C3(3) in DH5a cells were “miniprepped” and the extracted plasmids were stored in the freezer.
pSB1C3(1), pET23a(1), pET23a(2), pET23a(3) plasmids from the miniprep were digested by EcoRI-HF and XhoI. Imaging shows very faint bands due to low amount of DNA being digested. However, we can confirm that the plasmids from DH5a cells transfected by pSB1C3(1) were actually pSB1C3. Plasmids from DH5a cells transfected by pSB1C3(2) seems to be pET23a but there was an extra band. Another digest with higher amount of DNA and running of the gel will be performed again on the next day to confirm the plasmids.
8/7: Three glycerol stocks were made:
pSB1C3(2) in DH5a cells
pSB1C3(3) in DH5a cells
pET23a(4) in DH5a cells
Part:Day:Titles: 3.2 Midiprep (+restriction digest and gel electrophoresis)
Start Date: 8/7/16
Experiment Planner:
Jeffrey
Experimenters:
Ella
Rob
Eileen
Jeffrey
Supervisor:
Amber
Leon
Lab book start pages:
16
Materials:
Cell cultures from the previous day
dH2O
FavorPrep Plamid Extraction Mini Kit
Agarose Gel
RedSafe DNA dye
Agarose
0.5x TBE buffer
6x Loading dye
DNA Ladder
10x NEB CutSmart buffer
EcoRI and Xhol restriction enzymes
Time Breakdown
1.5-2 hours midiprep
2.5 hours restriction digest, gel electrophoresis and imaging
Method:
For Midiprep protocol, see AxyPrep Midi Plasmid Kits.pdf
Midiprep:
Check OD value of the cell cultures.
Protocol suggest that 30 ml of overnight LB culture (high-copy plasmid), or 100 ml of overnight LB culture for the preparation of the (low-copy plasmid). Consider doubling (or more) the volume for preparation
Follow the protocol
Restriction digest and gel electrophoresis:
Perform digest and gel electrophoresis to all miniprep and midiprep samples as outlined in 3.1 experimental plan
Use 9 uL DNA for pSB1C3 (1), pET23a (1), pET23a (2) and pET23a (3)
Use 5 uL DNA for pSB1C3 (2) and pSB1C3 (2)
1.5 uL pSB1C3 (2) and 1 uL pET23a (4)
Single digest and uncut plasmid control
Output:
Midiprep product - pET23a and pSB1C3 plasmids
Relevant SWPs:
Proper centrifuge
Results/Comments:
11/7: Gel electrophoresis was performed. The banding pattern of digested pSB1C3 matched what we had expected. There are two extra bands in pET23a digest. Single digest of pET23a with XhoI seemed normal but that with EcoRI-HF was different from what we had predicted. We thought that the amount of EcoRI-HF might be too high and it caused star activity. Another restriction digest on pET23a will be carried out on the next day with less enzyme. Refer to p. 18 of the lab book.
12/7: We set up another sets of restriction digest (pET23a)
Slightly less EcoRI-HF and XhoI (0.8 uL each) and more DNA (2 uL rather than 1 uL) in a 50 uL reaction volume
Less EcoRI-HF and XhoI (0.5 uL each) and the same amount of DNA (1 uL) in a 20 uL reaction volume
EcoRI-HF and XhoI (1 uL each), 1 uL in a 20 uL reaction volume, single digest controls and uncut control
The result shows that using less enzyme decreases the amount of by-product. However, the two extra bands are still visible. The recommendation is that we should avoid performing EcoRI-HF on pET23a and use less enzymes. Refer to p. 21 of the lab book.
Documentations:
Part:Day:Titles: 3.3 PCR Optimisation of BioBrick Primers and Gel Electrophoresis
Start Date: 
13/07/16
Experiment Planner:
Jeffrey
Experimenters:
Jeffrey
Rob
Kimber
Supervisor:
Amber
Lab book start pages:
23
Materials:
TE buffer (EDTA and Tris-HCl)
DNA Kit 4 Well 4B
IDT Synthesised DNA
BioBrick primers
Vent DNA polymerase
dNTP solution mix
dH2O
ThermoPol reaction buffer (10x)
Time Breakdown
Method:
Resuspension of Primer (according to Genework Primer Reconstitution Ptotocol, https://www.geneworks.com.au/content.aspx?p=99)
Spin tube briefly in microcentrifuge to collect DNA in bottom of tube.
Resuspend the primers in TE buffer to make 100 mM stock
Allow to rehydrate for a few minutes with intermittent vortexing.
Create new 10 mM stocks for PCR use
Store in -20 C freezer
Resuspension of IDT Synthesised DNA (according to DNA spec sheet)
Centrifuge the tube for 3-5 secc at a minimum of 3000 x g to ensure the material is in the bottom of the tube.
Add TE buffer to reach a final concentration of 10 ng/uL.
Vortex briefly.
Incubate at 50 degC for 20 mins
Briefly vortex and centrifuge
PCR Optimisation:
Gel electrophoresis:
Output:
Relevant SWPs:
Results/Comments:
Part:Day:Titles: 3.3 Digest of Plasmid and DNA and Gel Purification
Start Date:
Experiment Planner:
Ella
Ray
Experimenters:
Supervisor:
Lab book pages:
Materials:
GeneJET Extraction Kit:
Binding Buffer
Wash Buffer (concentrated)
Elution Buffer (10 mM Tris-HCl, pH 8.5)
GeneJET Purification Columns (preassembled with collection tubes)
Isopropanol.
3 M sodium acetate, pH 5.2 (may be necessary).  
Microcentrifuge (>12000 x g)
1.5 or 2 mL microcentrifuge tubes.
Heating block or water bath (55degC and then 65decC)
Scalpel/razor blades
Time Breakdown:
The manual says 15 min. But more realistically:
1 hour LOL
Method:
Digesting Plasmid and Insert DNA:
Gel Purification:
Do not reuse electrophoresis buffer when extracted DNA fragment will be used directly for sequencing.
Close the bag with the purification columns tightly after each use!
Check the Binding Buffer for precipitates before each use. Re-dissolve any precipitate by warming the solution to 37 °C and cooling to 25 °C
Wear gloves when handling the Binding Buffer as this solution contains irritants
Each GeneJET purification column has a binding capacity of up to 25 μg of DNA and can process up to 1 g of agarose gel.
Prior to the initial use of the kit, dilute the 9mL of Wash Buffer (concentrated) with 45mL of ethanol (96-100%).
All centrifuge steps conducted at >12000 x g.
SELECTING THE BAND ON THE GEL:
Minimise the exposure of the gel to UV as this will damage the DNA.
Because the RedSafe stain might mess with the extracted DNA and other downstream experiments, Amber suggested that instead of putting the DNA stain in the Gel, we do a post electrophoresis stain. That way we can run the DNA sample in two different lanes. One with only a small amount of the sample and the other lane with the rest of it. We can cut the gel and stain only the half with the small amount of DNA, using this to locate the correct band in the lane with the rest of the DNA sample, which can then be extracted (without it being stained!)
Red safe can be used as a post electrophoresis stain, add 5uL of RedSafe to 100ml of buffer solution (may need to adjust this concentration).
[T--Melbourne--3_Protocols.jpg]
GEL PURIFICATION PROTOCOL:
Pre-heat the water bath to 55°C (also note that if DNA fragment is >10 kb, the Elution Buffer (step 10) should be pre-heated to 65 °C before applying to column.)
Cut out the gel slice containing the desired DNA fragment using a clean scalpel or razor blade. Cut as close to the DNA as possible to minimize the gel volume. Place the gel slice into a pre-weighed 1.5 mL tube and weigh. Record the weight of the gel slice.
Note. If the purified fragment will be used for cloning reactions, avoid damaging the DNA through UV light exposure. Minimize UV exposure to a few seconds or keep the gel slice on a glass or plastic plate during UV illumination.
Add 1:1 volume of Binding Buffer to the gel slice (volume: weight) (e.g., add 100 μL of Binding Buffer for every 100 mg of agarose gel).
Note. For gels with an agarose content greater than 2%, add 2:1 volumes of Binding Buffer to the gel slice.
Incubate the gel mixture at 50-60 °C for 10 min or until the gel slice is completely dissolved. Mix the tube by inversion every few minutes to facilitate the melting process and ensure that the gel is completely dissolved.
Check the color of the solution. A yellow color indicates an optimal pH for DNA binding. If the color of the solution is orange or violet, add 10 μL of 3 M sodium acetate, pH 5.2 solution and mix. The color of the mix will become yellow.
Optional: use this step only when DNA fragment is ≤500 bp or >10 kb long.
∙  If the DNA fragment is ≤500 bp, add 1 gel volume of 100% isopropanol to the solubilized gel solution (e.g. 100 μL of isopropanol should be added to 100 mg gel slice solubilized in 100 μL of Binding Buffer). Mix thoroughly.
∙  If the DNA fragment is >10 kb, add 1 gel volume of water to the solubilized gel solution (e.g. 100 μL of water should be added to 100 mg gel slice solubilized in 100 μL of Binding Buffer). Mix thoroughly.
Vortex the gel mixture briefly before loading on the column. Transfer up to 800 μL of the solubilized gel solution (from step 3 or 4) to the GeneJET purification column. Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube.
Note. If the total volume exceeds 800 μL, the solution can be added to the column in stages. After each application, centrifuge the column for 30-60 s and discard the flow-through after each spin. Repeat until the entire volume has been applied to the column membrane. Do not exceed 1 g of total agarose gel per column.
Optional: use this additional binding step only if the purified DNA will be used for sequencing.
∙  Add 100 μL of Binding Buffer to the GeneJET purification column. Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube.
Add 700 μL of Wash Buffer (diluted with ethanol) to the GeneJET purification column. Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube.
Centrifuge the empty GeneJET purification column for an additional 1 min to completely remove residual wash buffer.
Note. This step is essential to avoid residual ethanol in the purified DNA solution. The presence of ethanol in the DNA sample may inhibit downstream enzymatic reactions.
Transfer the GeneJET purification column into a clean 1.5 mL microcentrifuge tube (not included). Add 50 μL of Elution Buffer to the center of the purification column membrane. Centrifuge for 1 min.
Note.
∙  For low DNA amounts the elution volumes can be reduced to increase DNA concentration. An elution volume between 20-50 μL does not significantly reduce the DNA yield. However, elution volumes less than 10 μL are not recommended.
∙  If DNA fragment is >10 kb, prewarm Elution Buffer to 65 °C before applying to column.
∙  If the elution volume is 10 μL and DNA amount is ≤ 5 μg, incubate column for 1 min at room temperature before centrifugation.
Discard the GeneJET purification column and store the purified DNA at -20 °C.
Output:
Relevant SWPs:
Note: Wear gloves when handling the Binding Buffer as this solution contains irritants
Results/Comments:
The columns in the kit were not sealed in their ziplock bags so may not work as well as they should.
Part:Day:Titles: Week 4 Day 2. Transform ligated plasmid pSB1C3 (not the GoldenGate assembled gene) into DH5a cells,
PCR / Gel purify GoldenGate assembly
Start Date:4/7/16
Experiment Planner:
Ray
Experimenters:
Supervisor:
Lab book start pages:
Materials:
1 transformation+6 transformations if testing competency.
pSB1C3 Plasmid with insert (which one?)
SOC media (200µl per transformation)
Competent Cells (50µl per transformation)
Petri plates w/ LB agar and antibiotic (2 plates per transformation)
Chloramphenicol (1uL per 1ml of agar)
Bunsen Burner
Sterile Petri Dishes
Sterile measuring flask
Foam tube rack
Ice
Water bath (42 degC)
Incubator (37 degC)
Sterile spreader
iGEM distributed Competent Cell Test Kit (contains pSB-1C3 DNA at 5 different concentrations)
Time Breakdown:
1.5 hour making LB plates and plasmid preparation
2 hours incubation
1 hour for plating
Overnight incubation of plates (14-18hrs incubation).
NOTE: this experiment needs to be done at the end of the day so that the plates do not incubate for longer than 18hrs. Someone must be in the lab the next morning to transfer the plates to the fridge.
Method:
Protocol:
Transformation protocol:   http://parts.igem.org/Help:Protocols/Transformation
Competent cell test kit protocol
http://parts.igem.org/Help:Competent_Cell_Test_Kit
Plating protocol: Protocol A3 Making LB Agar Plate in the 2014 teams protocol book.
LB plates:
Make LB plates with antibiotics according to protocol A3 in lab protocol folder. Add the appropriate amount of antibiotics before pouring into petri dish (25 ug/mL Chloramphenicol for pSB-1C3 transformation) This can be done during the 37deg 2 hour incubation period during the transformation process. For each transformation you need two LB plates.
Preparation prior to transformation of cells:
If using a new batch of DH5a cells, conduct a cell competency test alongside transformation.
There are 4 transformations of plasmid into DH5a cells: pSB1C3, 0.5 pg/ul DNA, 5 pg/ul DNA and 10 pg/ul DNA from Competent Cell Test Kit
There are 3 transformations of plasmid into BL21 cells: 0.5 pg/ul DNA, 5 pg/ul DNA and 10 pg/ul DNA from Competent Cell Test Kit.
The transformation of DNA from Competent Cell Test Kit will test for the transformation efficiency of competent cells and also act as positive controls.
Preparation of pSB1C3 plasmid
A plasmid clone containing part BBa_B0030 will be used. It is located at Well 4G of 2016 Kit Plate 4 (column 4 row G)
Resuspend DNA Distribution Kit well with 10µl dH20. Pipet up and down several times, let sit for a few minutes. Resuspension will be red from cresol red dye.
[T--Melbourne--4_Protocols.jpg]
Preparation of pET23a plasmid
1uL of a 1/25 dilution of the standard pET23a solution and then make 20 ng/uL working solution in DNA/RNA work box
Preparation of plasmid DNA from Cell Competency Test Kit
Spin down the DNA tubes from the Competent Cell Test Kit/Transformation Efficiency Kit to collect all of the DNA into the bottom of each tube prior to use. A quick spin of 20-30 seconds at 8,000-10,000 rpm will be sufficient
Set water bath to 42°C. Use a thermometer for accurate measurement. Do not trust the setting on the water bath.
Transformation of cells:
Follow the transformation protocol found on the iGEM website:   http://parts.igem.org/Help:Protocols/Transformation .
Thaw 5 DH5a competent cells tubes and 3 BL21 competent cells tubes on ice. Tubes should be labeled, pre-chilled, and in a floating tube rack for support. Keep all tubes on ice.
Also prepared 2 tubes of DH5a and 2 tubes BL21. These tubes will not be transformed but will undergo the same process as other tubes. They are then plated on the plates with appropriate antibiotic as negative control.
Pipette 1µl of prepared plasmid DNA into competent cell tubes: Pipette from well into appropriately labeled tube. Gently pipette up and down a few times. Keep all tubes on ice.
Close 2ml tubes, incubate on ice for 30min: Tubes may be gently agitated/flicked to mix solution, but return to ice immediately.
Heat shock tubes at 42°C in waterbath for 1 min: 2ml tubes should be in a floating foam tube rack. Place in water bath to ensure the bottoms of the tubes are submerged. Timing is critical.
Incubate on ice for 5min: Return transformation tubes to ice bucket.
Pipette 200µl SOC media to each transformation: SOC should be stored at 4°C, but can be warmed to room temperature before use. Check for contamination.
Incubate at 37°C in a shaking incubator for 2 hours
For pSB1C3 transformation: Pipette each transformation on two petri plates for a 20µl and 200µl plating: Pipette 20µl and 200µl of the transformation onto appropriately labeled plates. Spread with sterilized spreader immediately. This helps ensure that you will be able to pick out a single colony.
For DNA from Cell Competency Kit: Pipet 20 µL from each tube onto the appropriate plate, and spread the mixture evenly across the plate. Do at least two plates. Triplicates (3 each) of each tube if possible, so you can calculate an average colony yield.
For negative control: Pipet 20 µL from each tube onto the appropriate plate, and spread the mixture evenly across the plate.
For lawn growth control: Pipet 20 µL of the transformed cell onto the plate without antibiotic.
Incubate transformations overnight (14-18hr) at 37°C: Incubate the plates upside down (agar side facing up, let the agar surface become semi-dry before putting it upside down). If incubated for too long, colonies may overgrow and the antibiotics may start to break down; un-transformed cells will begin to grow.
See the next experiment plan for steps after incubation
Output:
24 plates containing transformed bacteria
pSB1C3 into DH5a
pET23a into DH5a
0.5 pg/ul DNA from test kit into DH5a
5 pg/ul DNA from test kit into DH5a
10 pg/ul DNA from test kit into DH5a
0.5 pg/ul DNA from test kit into BL21
5 pg/ul DNA from test kit into BL21
10 pg/ul DNA from test kit into BL21
Relevant SWPs:
Agar Plating
