<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Melbourne_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Melbourne/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Melbourne iGEM 2016</TITLE></P><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="navbar"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2016.igem.org/Team:Melbourne">Home</A></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Melbourne/Team" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Team_Members">Team Members</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Collaborations">Collaboration</A></LI></UL><LI class="dropdown"><A href="https://2016.igem.org/Team:Melbourne/Project" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Model">Modelling</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Achievements">Achievements</A></LI></DIV></DIV><P><TITLE>Melbourne iGEM 2016</TITLE></P><DIV class="jumbotron feature"><DIV class="container"><H1 style="text-align:center;">Demonstration and Results</H1></DIV></DIV><DIV class="page_content"><P class="c1"><SPAN class="c3">Overview</SPAN></P><UL class="c4 lst-kix_hny06lybresu-0 start"><LI class="c2 c1"><SPAN>We designed the sequence for our novel StarScaffold protein, and chose a set of split inteins which could be used together to join the Scaffolds together (or to link enzymes to the Scaffold)</SPAN></LI><LI class="c2 c1"><SPAN>We successfully cloned our StarScaffold genes into E. coli using the pSB-1C3 and pET-23a vectors, and confirmed their size and sequences.</SPAN></LI><LI class="c2 c1"><SPAN>Our designed StarScaffold genes were sent to iGEM headquarters as part of the Biobrick registry.</SPAN></LI><LI class="c2 c1"><SPAN>We expressed our StarScaffold protein from BL21(DE3) E. coli, and confirmed the protein’s size via SDS-PAGE.</SPAN></LI></UL><P class="c1"><SPAN class="c3">Designing the StarScaffold</SPAN></P><P class="c1"><SPAN>Drawing our inspiration from the Melbourne 2014 Team’s Star Peptide project, we aimed to utilise the properties of star-shaped proteins. In the first stages of our project, we worked on designing the sequences for our StarScaffold protein.</SPAN></P><P class="c1"><SPAN>We designed sequences for several variants of the StarScaffold, of which four were successfully cloned and sent to the Biobrick registry: our two main Prototype parts, an alternative Double Helix design with lengthened helix domain that may lend itself to better in vivo disulphide formation, and an alternate Non-Sequential design which can be used to test enzyme co-localisation with previously tested inteins. More information about these parts can be found on our Parts page.</SPAN></P><P class="c1"><SPAN>[[T--Melbourne--1_stardesign.jpg]]</SPAN></P><P class="c1"><SPAN>We also selected a set of four split inteins which can be used together in a sequential fashion, as they have high reaction rates and mutually exclusive reaction conditions over a range of temperatures.</SPAN></P><P class="c1"><SPAN>[[T--Melbourne--2_intein_table.jpg]]</SPAN></P><P class="c1"><SPAN class="c3">Cloning the StarScaffold into E. coli</SPAN></P><P class="c1"><SPAN>Once we had received the synthesised genes we ordered, the first stage in our project was to clone the genes into the plasmid vectors pSB-1C3 and pET-23a for storage and expression, respectively. We successfully ligated our genes into the vectors, and transformed them into E. coli cells. These were then screened with colony PCR to check the size of our inserts, which were further confirmed via sequencing.</SPAN></P><P class="c1"><SPAN>Using colony PCR, we identified the pET-23a plasmids that had our inserts ligated in, as shown in the gel electrophoresis photos below.</SPAN></P><P class="c1"><SPAN>Colony PCR of pET inserts 160818 pg 89, gels from 160818 pg 93/95</SPAN></P><P class="c1"><SPAN>(proto 1 worked, the other gels failed or didn’t have right bands)</SPAN></P><P class="c1"><SPAN>Results: Correct bands for proto1 (1, 3-15)</SPAN></P><P class="c1"><SPAN class="c3">Expected Proto1 band size: 1270bp</SPAN></P><P class="c1"><SPAN>[[T--Melbourne--3_1270-bp_Proto1.jpg]]</SPAN></P><P class="c1"><SPAN>Colony pcr of pET inserts 160823 pg 101, gels from 160824 pg 105</SPAN></P><P class="c1"><SPAN>Results: Correct bands for proto2 (27, 28); nonseq1 (18, 25); double helix (18, 20, 24)</SPAN></P><P class="c1"><SPAN class="c3">Expected Proto2 band size: 1727bp</SPAN></P><P class="c1"><SPAN class="c3">Expected Non-Seq1 band size: 1580bp</SPAN></P><P class="c1"><SPAN>[[T--Melbourne--5_Proto2_NonSeq1.jpg]]</SPAN></P><P class="c1"><SPAN class="c3">Expected DoubleHelix band size: 1454bp</SPAN></P><P class="c1"><SPAN>[[T--Melbourne--6_DoubleHelix.jpg]]</SPAN></P><P class="c1"><SPAN>From the colonies identified on the gels above, one colony of each of the 4 variants was grown up and sequenced to confirm that the sequences were correct. After re-sequencing two more DoubleHelix colonies (as the first one we sequenced carried a mutation), we confirmed that we had correct sequences for these 4 StarScaffold proteins in pET-23a vectors.</SPAN></P><P class="c1"><SPAN>Sequencing of pET on 160830 pg 116</SPAN></P><UL class="c4 lst-kix_m7wngeok1za3-0 start"><LI class="c2 c1"><SPAN>Proto1 (1), proto2 (27), nonseq1 (18), dbhelix (18)</SPAN></LI><LI class="c2 c1"><SPAN>Results: all were fine except db helix, which had a mutation.</SPAN></LI></UL><P class="c1"><SPAN>Re-sequencing the other db helix pET colonies on 160905 pg 125</SPAN></P><UL class="c4 lst-kix_h6kiax6kbpkn-0 start"><LI class="c2 c1"><SPAN>Db helix (20) and (24)</SPAN></LI><LI class="c2 c1"><SPAN>Results: correct</SPAN></LI></UL><P class="c1"><SPAN class="c3">Expressing the StarScaffold proteins</SPAN></P><P class="c1"><SPAN>We continued onwards to expression with the 4 StarScaffolds above that were cloned into pET and had the right sequence. These were transformed into BL21(DE3) cells for expression.</SPAN></P><P class="c1"><SPAN>Using an IPTG-induced expression system, we expressed our StarScaffold Prototype 1 protein and visualised it on an SDS-PAGE gel. Whole cell samples were taken both before and after the induction period, and run with SDS-PAGE sample buffer; the gel was then visualised using Coomassie staining.</SPAN></P><P class="c1"><SPAN class="c3">Expected Prototype 1 size: 37kDa</SPAN></P><P class="c1"><SPAN class="c3">Note: we see leaky expression from the T7 promoter.</SPAN></P><P class="c1"><SPAN>[[T--Melbourne--7_prototype1.jpg]]</SPAN></P><P class="c1"><SPAN class="c3">Sequencing the pSB-1C3 inserts to send to the Biobrick Registry</SPAN></P><P class="c1"><SPAN>We sequenced the inserts we had cloned into pSB-1C3.</SPAN></P><P class="c1"><SPAN class="c3">Future plans</SPAN></P><UL class="c4 lst-kix_940prgd70zpf-0 start"><LI class="c1 c2"><SPAN>Continue characterising our expressed StarScaffold protein for solubility, disulphide formation etc.</SPAN></LI><LI class="c2 c1"><SPAN>Express the protein in SHuffle </SPAN><SPAN class="c6">E. coli</SPAN><SPAN> cells, which have a naturally reducing environment and thus will allow for</SPAN><SPAN class="c6"> in vivo </SPAN><SPAN>disulphide formation</SPAN></LI><LI class="c2 c1"><SPAN>Express the chosen split inteins together with our StarScaffold</SPAN></LI><LI class="c2 c1"><SPAN>Form a hydrogel.</SPAN></LI></UL><P class="c1"><SPAN class="c3">Characterising the 2014 Parts</SPAN></P><P class="c1"><SPAN>The Melbourne 2014 iGEM team, from whom we drew inspiration for our StarScaffold project, designed and submitted a number of Biobrick parts to the registry. As part of the iGEM medal criteria, we further characterised their parts by sequencing them.</SPAN></P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>