Team:Melbourne/Design
Melbourne iGEM 2016
Toggle navigation
Home
Team
Team Members
Sponsors
Attributions
Collaboration
Project
Description
Design
Experiments
Proof of Concept
Demonstrate
Results
Notebook
Modelling
Safety
Parts
Human Practices
Achievements
Melbourne iGEM 2016
Experiment Design
Project Development
        The StarScaffold came about through the mixing of ideas. It’s well known that one characteristic of natural enzymatic systems (such as the electron transport chain) is that the proteins are co-localised to increase their local concentrations. After hearing about the Heidelburg 2014 iGEM team’s ring of fire project we realised that inteins could be combined with star peptides to create exotic protein structures.
        Intein Choice:
        One of our first tasks was to pick which inteins to use. The aim was to choose a set of inteins which were shown to not cross react and also covered a range of activities and optimal temperatures. Eileen and Jeff created a table detailing the available properties of various inteins based on the literature and the work of previous iGEM teams.
Table 1: Intein Choice
Intein
Origin
Length N
Length C
Reactivity Variation
Optimal conditions
Reactivity (s^-1)
Yield
Cross
reactions
Number of Cysteines (N-term|||C term)
Reference
Npu DnaE
Natural
102
36
robust over most temps
Ssp DnaE
3|||0
http://www.sciencedirect.com/science/article/pii/S001457930900101X
Ssp DnaB
Artificial; mini-intein
reasonably robust for 25°-37°
25°
Npu DnaB
http://pubs.acs.org.ezp.lib.unimelb.edu.au/doi/pdf/10.1021/bi051697%2B
Sce VMA
good at 25°, but not higher than 30°
25°
http://pubs.acs.org.ezp.lib.unimelb.edu.au/doi/pdf/10.1021/bi051697%2B
Ssp GyrB
Long
Short
Rma DnaB
http://www.jbc.org/content/284/10/6194.full.pdf
gp41-1
Identified through bioinformatics
88
38
fastest known (faster than DnaE); quite robust over various temp/pH. The paper describes this intein as "superior on all accounts"
45°. pH 6-10 all ok
1.8 (+/- 0.5) x 10^-1
* * * *
none
3|||0
"Fractured genes: a novel genomic arrangement involving new split inteins and a new homing endonuclease family"
and " Unprecedented Rates and Efficiencies Revealed for New Natural Split Inteins from Metagenomic Sources"
gp41-8
-
91
38
37°
4.5 (+/- 0.6) x 10^-2
* * * *
none
2|||1
"Fractured genes - a novel genomic arrangement involving new split inteins and a new homing endonuclease family"
NrdJ-1
105
41
37°
9.8 (+/- 2.3) x 10^-2
* * * *
none
2|||0
"Fractured genes - a novel genomic arrangement involving new split inteins and a new homing endonuclease family"
IMPDH-1
?
101
41
37°
8.7 (+/- 3.2) x 10^-2
* * * *
none
2|||0
"Fractured genes - a novel genomic arrangement involving new split inteins and a new homing endonuclease family"
AceL-TerL
Identified through bioinformatics???
25
104
perform extremely well in 8C but not in high temperature; mutations, significantly improve ligation capability
Highest rate at 8°, still work at 25° and 37° (50 fold slower)
1.7 (+/- 0.2) X10^-3
* * * *
Not known
1|||1
"An Atypical Naturally Split Intein Engineered for Highly Efficient Protein Labeling"
Leucine Zipper Design:
        In forming a StarScaffold in vivo we need the disulphide bond to have reasonable kinetics of formation. One potential problem we identified with the 2014 team’s project was that there is significant separation between cysteine residues which would slow disulphide formation. Our advisor Associate Professor Paul Gooley suggested we increase localisation using a leucine zipper.
Leucine Zipper Research:
Leucine zippers are a coiled coil motif characterised by heptapeptide repeats that contain leucine or Isoleucine in their first and fourth positions (AbcDefg). Leucine zippers can come in a number of different varieties that may be composed of two or more alpha helixes. For our purposes we require a very specific interaction, that is; the formation of two antiparallel leucine zipper dimers that will not exchange subunits[1]. The specificity for a coiled coils composed of more than two coils is promoted by isoleucine in the place of a leucine in the heptapeptide repeat. By keeping the hydrophobic core composed of leucine residues we promote dimerisation specifically[1]. To promote the formation of an antiparallel coiled coil we place an Asn residue in place of a leucine on each coil such that in the parallel confirmation they will disrupt the formation of a hydrophobic core and when in the antiparallel conformation they will align and hydrogen bond each other, stabilising the macromolecular interaction[2]. Another way to promote preferred macromolecular interactions (green below) is the proper ionic interaction of residues flanking the hydrophobic core of the coiled coils to the exclusion of non-preferred macromolecular interactions.        
Zipper sequences were extrapolated from Oakley and Kim (1998).
Table 2: Unfavourable interactions forcing specificity in coiled coils:
Coil
Acid A
Base A
Acid B
Base B
Sequences
Acid A
Parallel: Ionic repulsion (I)
Antiparallel: (I),
Cystine Displacement (C) Asparagine Displacement
(A)
Parallel: (C)(N)
Antiparallel:
Parallel: (I)(N)
Antiparallel: (I)(C)
Parallel: (C)
Antiparallel: (N)
AQLEKELQALEKELAQLEWELQALEKENAQLEKELAQ
GGC
Base A
Parallel: (I)
Antiparallel: (I)(C)(N)
Parallel: (C)
Antiparallel: (N)
Parallel: (I)(N)
Antiparallel: (I)(C)
CGG
AQLKKKLQANKKKLAQLKWKLQALKKKLAQLKKKLAQ
Acid B
Parallel: (I)
Antiparallel: (I)(C)(N)
Parallel: (C)(N)
Antiparallel:
AQLEKELQANEKELAQLEWELQALEKELAQLEKELAQ
GGC
Base B
Parallel: (I)
Antiparallel: (I)(C)(N)
CGG
AQLKKKLQALKKKLAQLKWKLQALKKKNAQLKKKLAQ
References:                                        
1.         Woolfson, D. N. (2005). The design of coiled-coil structures and assemblies. Advances in protein chemistry, 70, 79-112.
                                                
2.         Oakley, M. G., & Kim, P. S. (1998). A buried polar interaction can direct the relative orientation of helices in a coiled coil. Biochemistry, 37(36), 12603-12610.
Disulphide Design:
Disulphide formation occurs between two cysteine anions, this generally requires an oxidant and a solution of pH above the pI of each cysteine residue. However the pI of a residue can be modified by it’s environment, positive charges surrounding the residue help to stabilise the anion form of the cysteine residue lowering the pI, allowing disulphide formation at a lower pH and at a higher rate than might usually occur[1][2].
References:
1.        Hansen, R. E., Østergaard, H., & Winther, J. R. (2005). Increasing the reactivity of an artificial dithiol-disulfide pair through modification of the electrostatic milieu. Biochemistry, 44(15), 5899-5906.
2.        Snyder, G. H., Cennerazzo, M. J., Karalis, A. J., & Locey, D. (1981). Electrostatic influence of local cysteine environments on disulfide exchange kinetics. Biochemistry, 20(23), 6509-6519.
StarScaffold variation design:
Prototype:  
        The prototype includes a single 5 repeat antiparallel leucine zipper for dimerisation and a set of four inteins, designed to cover a range of active temperatures. This allows us to progressively build up macromolecular structures in an effort to form proteinaceous monolayers, filaments and potentially more complicated structures. Each segment is bookended by flexible linkers to prevent steric hinderance of segment folding. This is particularly a concern for the natural split inteins found at each end of the protein as they form a folded structure before interacting. However the same is not true for the artificial split inteins which should be more amenable to internal protein segments as this is part of their native full intein structure.
dHelix:
        The original helix on which the dimerisation domain is based was altered by lengthening the coil and changing the positions of the Asn residue used to promote antiparallel structure. The single coiled coil ‘prototype’ version is preferred because of the complication to structure and disulphide bond formation caused by four helixes total in each protein. But the ‘dHelix’ version does have the advantage of avoiding the increased coiled coil stability caused by the lengthening of the helix in the ‘prototype’ version. It also may have superior in vivo disulphide formation thanks to the extra secondary structure around the cysteines acting to potentially exclude disulphide bond isomerases.
        Helix and linker structure:
Linker+cysteine         in yellow
Cysteine                 in purple
Linker                        in blue
Star_one_pair_one:
Base:                                         AQLKKKLQALKKKLAQLKWKNQALKKKLAQ
Acid:        GSGSGSGGKRCRKGG         AQLEKELQALEKELAQLEWENQALEKELAQ
Star_one_pair_two:
Base:         AQLKKKLQANKKKLAQLKWKLQALKKKLAQ         GGKRCRKGGSGSGS
Acid:        AQLEKELQANEKELAQLEWELQALEKELAQ
Star_two_pair_one:
Acid: GGKRCRKGG                AQLEKELQALEKELAQLEWENQALEKELAQ         GSGSGSG
Acid:                                AQLEKELQALEKELAQLEWENQALEKELAQ
Star_two_pair_two:
Base: AQLKKKLQANKKKLAQLKWKLQALKKKLAQ                GSGSGSG
Base: AQLKKKLQANKKKLAQLKWKLQALKKKLAQ          GGKRCRKGG
NonSequential:
        Sequential intein activation is not required for enzyme splicing, we have a set of four previously tested[11] inteins we can use for enzyme co-localisation which have been shown to have a high splicing rate, yield and orthologous splicing reactions. This construct allows us to check structural changes that occur due to splicing reactions occurring in the same mixture but at alternate temperatures.
NoHelix:
By removing the helixes altogether we may measure the effect of helix presence on structure.
Experiment Planning:
        Before beginning wet-lab work we drew up a plan to check with our supervisors what follows is a copy of the working document we used to prepare our experiments ahead of time:
Prepare stocks of competent BL21(DE3) and DH5α cells using CaCl2 method ([1] page 1.82)
Day 1.        Make FSB, SOC, SOB and SOB-agar and autoclave.
Plate BL21(DE3) and DH5α cells on LB plates and grow overnight.
Day 2.         Pick colonies and grow overnight in selective media.
Day 3.         Re-culture cells, resuspend and snap freeze.
Transform cells with empty pET-23a and pSB-1C3 and check cell competency:
Day 1.        Transform cells, plate, and grow overnight.
Day 2. Calculate transformation efficiency.
Grow the cells overnight for glycerol stock
Make LB-Agar
Prepare Plasmid and Insert DNA for ligation:
Day 1. Make glycerol stock
Miniprep, digest plasmids and gel electrophoresis.
Day 2. Midiprep, restriction digest and gel electrophoresis of pSB1C3 plasmid
Day 3. Run PCR Optimisation and gel electrophoresis. 
Day 4. Amplify insert DNA accordingly.
        Purify PCR products
Day 5. Digest and purify plasmid pSB1C3
                Digest and gel purify insert DNA and Gel purify.
Ligate Inserts into pSB-1C3 and pET-23α:
Day 1.         GoldenGate reaction using BsaI-HF and BsmBI
                Plasmid ligation reaction EcoRI/PstI and NdeI/NotI
Calculate transformation efficiency.
Day 2. Transform ligated plasmid pSB1C3 (not the GoldenGate assembled gene) into DH5a cells
PCR / Gel purify GoldenGate assembly
Assemble GoldenGate DNA into plasmids via  EcoRI/PstI and NdeI/NotI ligation
Transform constructs into BL21(DE3) and DH5α:
Day 1. Transform cells, plate, and grow overnight
Day 2. Colony PCR (+gel electrophoresis) and grow up cells.
Day 3. Make glycerol stocks.
        PCR
Day 4.         DNA purification (miniprep)
Day 5. Sequencing PCR
Express the protein:
Day 1. Grow them up overnight.
Day 2. Dilute and induce expression and leave overnight.
Day 3. Isolate protein in soluble and insoluble fraction.
Day 4. SDS-PAGE to check overexpression.
Day 5. Western Blot using Anti-His.
Purify our proteins:
Day 1. Grow them up overnight.
Day 2. Dilute, induce expression and leave overnight.
Day 3. Lyse cells, Isolate protein in soluble and insoluble fraction.
Day 4. Analyse protein concentration for splicing assays
        Day 5.        Purify protein using Nickel beads
Characterise inteins:        
Day 1.         Resuspend inteins in splicing buffer and perform time dependent assay of splicing activity.
        Day 2. SDS-PAGE reactions against controls to estimate activity.
Day 3. Prepare samples for Quantitative Mass-Spectrometry (or similar)
Disulphide formation:
Shuffle Cells:
Day 1.        Transform cells, plate, and grow overnight ([1] page 1.83).
        Day 2. Reculture and grow to correct OD600
Make glycerol stocks.
        Induce and leave cells overnight to express
Day 3. Isolate protein in soluble and insoluble fraction.
Day 4. SDS-PAGE to check overexpression.
Day 5. Western Blot using Anti-His
