<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Melbourne_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Melbourne/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Melbourne iGEM 2016</TITLE></P><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="navbar"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2016.igem.org/Team:Melbourne">Home</A></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Melbourne/Team" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Team_Members">Team Members</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Collaborations">Collaboration</A></LI></UL><LI class="dropdown"><A href="https://2016.igem.org/Team:Melbourne/Project" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Model">Modelling</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Melbourne/Achievements">Achievements</A></LI></DIV></DIV><P><TITLE>Melbourne iGEM 2016</TITLE></P><DIV class="jumbotron feature"><DIV class="container"><H1 style="text-align:center;">Experiment Design</H1></DIV></DIV><DIV class="page_content"><H1>Project Development</H1><P class="c1"><SPAN class="c4">        </SPAN><SPAN>The StarScaffold came about through the mixing of ideas. It’s well known that one characteristic of natural enzymatic systems (such as the electron transport chain) is that the proteins are co-localised to increase their local concentrations. After hearing about the Heidelburg 2014 iGEM team’s ring of fire project we realised that inteins could be combined with star peptides to create exotic protein structures.</SPAN></P><H2 class="c1 c26" id="h.fsqcp8c4tuir"><SPAN>        Intein Choice:</SPAN></H2><P class="c1"><SPAN>        One of our first tasks was to pick which inteins to use. The aim was to choose a set of inteins which were shown to not cross react and also covered a range of activities and optimal temperatures. Eileen and Jeff created a table detailing the available properties of various inteins based on the literature and the work of previous iGEM teams.</SPAN></P><TABLE class="c19"><TBODY><TR class="c62"><TD class="c75" colspan="11" rowspan="1"><P class="c2 c1"><SPAN class="c25 c4">Table 1:</SPAN><SPAN class="c0"> Intein Choice</SPAN></P></TD></TR><TR class="c27"><TD class="c11" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Intein</SPAN></P></TD><TD class="c24" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Origin</SPAN></P></TD><TD class="c59" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Length N</SPAN></P></TD><TD class="c23" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Length C</SPAN></P></TD><TD class="c46" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Reactivity Variation</SPAN></P></TD><TD class="c35" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Optimal conditions</SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Reactivity (s^-1)</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Yield</SPAN></P></TD><TD class="c43" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Cross</SPAN></P><P class="c2 c1"><SPAN class="c0">reactions</SPAN></P></TD><TD class="c16" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Number of Cysteines (N-term|||C term)</SPAN></P></TD><TD class="c33" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Reference</SPAN></P></TD></TR><TR class="c27"><TD class="c11" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c25 c4">Npu DnaE</SPAN></P></TD><TD class="c24" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Natural</SPAN></P></TD><TD class="c59" colspan="1" rowspan="1"><P class="c7 c1"><SPAN class="c0">102</SPAN></P></TD><TD class="c23" colspan="1" rowspan="1"><P class="c7 c1"><SPAN class="c0">36</SPAN></P></TD><TD class="c46" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">robust over most temps</SPAN></P></TD><TD class="c43" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Ssp DnaE</SPAN></P></TD><TD class="c16 c57" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c57 c25">3|||0</SPAN></P></TD><TD class="c33" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c58 c25 c42"><A class="c64" href="https://www.google.com/url?q=http://www.sciencedirect.com/science/article/pii/S001457930900101X&amp;sa=D&amp;ust=1476938022509000&amp;usg=AFQjCNG2z_9JNZ0FMt7IGTRsauEBL0oqkA">http://www.sciencedirect.com/science/article/pii/S001457930900101X</A></SPAN></P></TD></TR><TR class="c70"><TD class="c11" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c25 c4">Ssp DnaB</SPAN></P></TD><TD class="c24" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Artificial; mini-intein</SPAN></P></TD><TD class="c46" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">reasonably robust for 25°-37°</SPAN></P></TD><TD class="c35" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">25°</SPAN></P></TD><TD class="c43" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Npu DnaB</SPAN></P></TD><TD class="c33" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c58 c25 c42"><A class="c64" href="https://www.google.com/url?q=http://pubs.acs.org.ezp.lib.unimelb.edu.au/doi/pdf/10.1021/bi051697%252B&amp;sa=D&amp;ust=1476938022519000&amp;usg=AFQjCNEW5TZXncZ6_ClRLaalejg6tzXxVw">http://pubs.acs.org.ezp.lib.unimelb.edu.au/doi/pdf/10.1021/bi051697%2B</A></SPAN></P></TD></TR><TR class="c27"><TD class="c11" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Sce VMA</SPAN></P></TD><TD class="c46" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">good at 25°, but not higher than 30°</SPAN></P></TD><TD class="c35" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">25°</SPAN></P></TD><TD class="c33" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c25 c42 c58"><A class="c64" href="https://www.google.com/url?q=http://pubs.acs.org.ezp.lib.unimelb.edu.au/doi/pdf/10.1021/bi051697%252B&amp;sa=D&amp;ust=1476938022531000&amp;usg=AFQjCNGUfZBr2HbOPGojDkLpJxeyq8fJjg">http://pubs.acs.org.ezp.lib.unimelb.edu.au/doi/pdf/10.1021/bi051697%2B</A></SPAN></P></TD></TR><TR class="c27"><TD class="c11" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Ssp GyrB</SPAN></P></TD><TD class="c59" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Long</SPAN></P></TD><TD class="c23" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Short</SPAN></P></TD><TD class="c43" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Rma DnaB</SPAN></P></TD><TD class="c33" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c58 c25 c42"><A class="c64" href="https://www.google.com/url?q=http://www.jbc.org/content/284/10/6194.full.pdf&amp;sa=D&amp;ust=1476938022541000&amp;usg=AFQjCNHGfrPy6KW0NxRR6BDRoZFZlP_Q_A">http://www.jbc.org/content/284/10/6194.full.pdf</A></SPAN></P></TD></TR><TR class="c27"><TD class="c11" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c25 c4">gp41-1</SPAN></P></TD><TD class="c24" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Identified through bioinformatics</SPAN></P></TD><TD class="c59" colspan="1" rowspan="1"><P class="c7 c1"><SPAN class="c0">88</SPAN></P></TD><TD class="c23" colspan="1" rowspan="1"><P class="c7 c1"><SPAN class="c0">38</SPAN></P></TD><TD class="c46" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">fastest known (faster than DnaE); quite robust over various temp/pH. The paper describes this intein as &quot;superior on all accounts&quot;</SPAN></P></TD><TD class="c35" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">45°. pH 6-10 all ok</SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">1.8 (+/- 0.5) x 10^-1</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">* * * *</SPAN></P></TD><TD class="c43" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">none</SPAN></P></TD><TD class="c16" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">3|||0</SPAN></P></TD><TD class="c33" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">&quot;Fractured genes: a novel genomic arrangement involving new split inteins and a new homing endonuclease family&quot;</SPAN></P><P class="c2 c1"><SPAN class="c0">and &quot; Unprecedented Rates and Efficiencies Revealed for New Natural Split Inteins from Metagenomic Sources&quot;</SPAN></P></TD></TR><TR class="c27"><TD class="c11" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c25 c4">gp41-8</SPAN></P></TD><TD class="c24" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">-</SPAN></P></TD><TD class="c59" colspan="1" rowspan="1"><P class="c1 c7"><SPAN class="c0">91</SPAN></P></TD><TD class="c23" colspan="1" rowspan="1"><P class="c7 c1"><SPAN class="c0">38</SPAN></P></TD><TD class="c35" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">37°</SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">4.5 (+/- 0.6) x 10^-2</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">* * * *</SPAN></P></TD><TD class="c43" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">none</SPAN></P></TD><TD class="c16" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">2|||1</SPAN></P></TD><TD class="c33" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">&quot;Fractured genes - a novel genomic arrangement involving new split inteins and a new homing endonuclease family&quot;</SPAN></P></TD></TR><TR class="c27"><TD class="c11" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c4 c25">NrdJ-1</SPAN></P></TD><TD class="c59" colspan="1" rowspan="1"><P class="c7 c1"><SPAN class="c0">105</SPAN></P></TD><TD class="c23" colspan="1" rowspan="1"><P class="c7 c1"><SPAN class="c0">41</SPAN></P></TD><TD class="c35" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">37°</SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">9.8 (+/- 2.3) x 10^-2</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">* * * *</SPAN></P></TD><TD class="c43" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">none</SPAN></P></TD><TD class="c16" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">2|||0</SPAN></P></TD><TD class="c33" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">&quot;Fractured genes - a novel genomic arrangement involving new split inteins and a new homing endonuclease family&quot;</SPAN></P></TD></TR><TR class="c27"><TD class="c11" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c25 c4">IMPDH-1</SPAN></P></TD><TD class="c24" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">?</SPAN></P></TD><TD class="c59" colspan="1" rowspan="1"><P class="c7 c1"><SPAN class="c0">101</SPAN></P></TD><TD class="c23" colspan="1" rowspan="1"><P class="c7 c1"><SPAN class="c0">41</SPAN></P></TD><TD class="c35" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">37°</SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">8.7 (+/- 3.2) x 10^-2</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">* * * *</SPAN></P></TD><TD class="c43" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">none</SPAN></P></TD><TD class="c16" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">2|||0</SPAN></P></TD><TD class="c33" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">&quot;Fractured genes - a novel genomic arrangement involving new split inteins and a new homing endonuclease family&quot;</SPAN></P></TD></TR><TR class="c27"><TD class="c11" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c25 c4">AceL-TerL</SPAN></P></TD><TD class="c24" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Identified through bioinformatics???</SPAN></P></TD><TD class="c59" colspan="1" rowspan="1"><P class="c7 c1"><SPAN class="c0">25</SPAN></P></TD><TD class="c23" colspan="1" rowspan="1"><P class="c7 c1"><SPAN class="c0">104</SPAN></P></TD><TD class="c46" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">perform extremely well in 8C but not in high temperature; mutations, significantly improve ligation capability</SPAN></P></TD><TD class="c35" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Highest rate at 8°, still work at 25° and 37° (50 fold slower)</SPAN></P></TD><TD class="c30" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">1.7 (+/- 0.2) X10^-3</SPAN></P></TD><TD class="c21" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">* * * *</SPAN></P></TD><TD class="c43" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">Not known</SPAN></P></TD><TD class="c16" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">1|||1</SPAN></P></TD><TD class="c33" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c0">&quot;An Atypical Naturally Split Intein Engineered for Highly Efficient Protein Labeling&quot;</SPAN></P></TD></TR></TBODY></TABLE><H2 class="c1 c26" id="h.ke5tm63wfk"><SPAN>Leucine Zipper Design:</SPAN><SPAN class="c68">        In forming a StarScaffold </SPAN><SPAN class="c14 c68">in vivo</SPAN><SPAN class="c68"> we need the disulphide bond to have reasonable kinetics of formation. One potential problem we identified with the 2014 team’s project was that there is significant separation between cysteine residues which would slow disulphide formation. Our advisor Associate Professor Paul Gooley suggested we increase localisation using a leucine zipper.</SPAN></H2><P class="c1"><SPAN>Leucine Zipper Research:</SPAN></P><P class="c1"><SPAN>Leucine zippers are a coiled coil motif characterised by heptapeptide repeats that contain leucine or Isoleucine in their first and fourth positions (AbcDefg). Leucine zippers can come in a number of different varieties that may be composed of two or more alpha helixes. For our purposes we require a very specific interaction, that is; the formation of two antiparallel leucine zipper dimers that will not exchange subunits</SPAN><SPAN class="c52">[1]</SPAN><SPAN>. The specificity for a coiled coils composed of more than two coils is promoted by isoleucine in the place of a leucine in the heptapeptide repeat. By keeping the hydrophobic core composed of leucine residues we promote dimerisation specifically</SPAN><SPAN class="c52">[1]</SPAN><SPAN>. To promote the formation of an antiparallel coiled coil we place an Asn residue in place of a leucine on each coil such that in the parallel confirmation they will disrupt the formation of a hydrophobic core and when in the antiparallel conformation they will align and hydrogen bond each other, stabilising the macromolecular interaction</SPAN><SPAN class="c52">[2]</SPAN><SPAN>. Another way to promote preferred macromolecular interactions (green below) is the proper ionic interaction of residues flanking the hydrophobic core of the coiled coils to the exclusion of non-preferred macromolecular interactions.        </SPAN></P><P class="c1"><SPAN>Zipper sequences were extrapolated from Oakley and Kim (1998).</SPAN></P><TABLE class="c79"><TBODY><TR class="c27"><TD class="c54 c61" colspan="6" rowspan="1"><P class="c2 c1"><SPAN class="c22 c54 c4 c85">Table 2: Unfavourable interactions forcing specificity in coiled coils:</SPAN></P></TD></TR><TR class="c27"><TD class="c53 c54" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c22 c54 c12 c4">Coil</SPAN></P></TD><TD class="c63 c54" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c22 c54 c12 c4">Acid A</SPAN></P></TD><TD class="c28" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c22 c54 c12 c4">Base A</SPAN></P></TD><TD class="c32 c54" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c22 c54 c12 c4">Acid B</SPAN></P></TD><TD class="c28" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c22 c54 c12 c4">Base B</SPAN></P></TD><TD class="c56 c54" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c22 c54 c12 c4">Sequences</SPAN></P></TD></TR><TR class="c27"><TD class="c53" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c22 c12 c4">Acid A</SPAN></P></TD><TD class="c63" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c12 c4">Parallel</SPAN><SPAN class="c13 c12">: Ionic repulsion (I)</SPAN></P><P class="c2 c1"><SPAN class="c12 c4">Antiparallel</SPAN><SPAN class="c13 c12">: (I), </SPAN></P><P class="c2 c1"><SPAN class="c13 c12">Cystine Displacement (C) Asparagine Displacement</SPAN></P><P class="c2 c1"><SPAN class="c13 c12">(A) </SPAN></P></TD><TD class="c39" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c12 c4">Parallel</SPAN><SPAN class="c12 c13">: (C)(N)</SPAN></P><P class="c2 c1"><SPAN class="c12 c4 c42 c74">Antiparallel: </SPAN></P></TD><TD class="c32" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c12 c4">Parallel</SPAN><SPAN class="c13 c12">: (I)(N)</SPAN></P><P class="c2 c1"><SPAN class="c12 c4">Antiparallel</SPAN><SPAN class="c13 c12">: (I)(C)</SPAN></P></TD><TD class="c39" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c12 c4 c18">Parallel: (C)</SPAN></P><P class="c2 c1"><SPAN class="c12 c4">Antiparallel: (</SPAN><SPAN class="c13 c12">N)</SPAN></P></TD><TD class="c56" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c12">AQLEKE</SPAN><SPAN class="c12 c4">LQALEKE</SPAN><SPAN class="c12">LAQLEWE</SPAN><SPAN class="c12 c4">LQALEKE</SPAN><SPAN class="c9 c42">N</SPAN><SPAN class="c12">AQLEKE</SPAN><SPAN class="c22 c12 c4">LAQ</SPAN></P><P class="c2 c1"><SPAN class="c12">GG</SPAN><SPAN class="c13 c12 c44">C</SPAN></P></TD></TR><TR class="c27"><TD class="c53" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c22 c12 c4">Base A</SPAN></P></TD><TD class="c20 c39" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c12 c4 c20">Parallel</SPAN><SPAN class="c15 c31 c12">: (I)</SPAN></P><P class="c2 c1"><SPAN class="c12 c4 c20">Antiparallel</SPAN><SPAN class="c15 c31 c12">: (I)(C)(N)</SPAN></P></TD><TD class="c32 c20" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c18 c12 c4">Parallel: (C)</SPAN></P><P class="c2 c1"><SPAN class="c12 c4 c20">Antiparallel: (</SPAN><SPAN class="c15 c31 c12">N)</SPAN></P></TD><TD class="c39 c20" colspan="1" rowspan="1"><P class="c1 c2"><SPAN class="c12 c4 c20">Parallel</SPAN><SPAN class="c15 c31 c12">: (I)(N)</SPAN></P><P class="c2 c1"><SPAN class="c12 c4 c20">Antiparallel</SPAN><SPAN class="c15 c31 c12">: (I)(C)</SPAN></P></TD><TD class="c56 c20" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c44 c12">C</SPAN><SPAN class="c15 c31 c12">GG</SPAN></P><P class="c2 c1"><SPAN class="c12 c20">AQLKKK</SPAN><SPAN class="c12 c4 c20">LQA</SPAN><SPAN class="c9 c4">N</SPAN><SPAN class="c12 c4 c20">KKK</SPAN><SPAN class="c12 c20">LAQLKWK</SPAN><SPAN class="c12 c4 c20">LQALKKK</SPAN><SPAN class="c12 c20">LAQLKKK</SPAN><SPAN class="c15 c12 c4">LAQ</SPAN></P></TD></TR><TR class="c27"><TD class="c53" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c22 c12 c4">Acid B</SPAN></P></TD><TD class="c32" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c12 c4">Parallel</SPAN><SPAN class="c13 c12">: (I)</SPAN></P><P class="c2 c1"><SPAN class="c12 c4">Antiparallel</SPAN><SPAN class="c13 c12">: (I)(C)(N)</SPAN></P></TD><TD class="c39" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c4 c12">Parallel</SPAN><SPAN class="c13 c12">: (C)(N)</SPAN></P><P class="c2 c1"><SPAN class="c74 c12 c4 c42">Antiparallel: </SPAN></P></TD><TD class="c56" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c12">AQLEKE</SPAN><SPAN class="c12 c4">LQA</SPAN><SPAN class="c9 c4">N</SPAN><SPAN class="c12 c4">EKE</SPAN><SPAN class="c12">LAQLEWE</SPAN><SPAN class="c12 c4">LQALEKE</SPAN><SPAN class="c12">LAQLEKE</SPAN><SPAN class="c22 c12 c4">LAQ</SPAN></P><P class="c2 c1"><SPAN class="c12">GG</SPAN><SPAN class="c13 c44 c12">C</SPAN></P></TD></TR><TR class="c27"><TD class="c53" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c12 c4 c22">Base B</SPAN></P></TD><TD class="c39 c20" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c12 c4 c20">Parallel</SPAN><SPAN class="c15 c31 c12">: (I)</SPAN></P><P class="c2 c1"><SPAN class="c12 c4 c20">Antiparallel</SPAN><SPAN class="c15 c31 c12">: (I)(C)(N)</SPAN></P></TD><TD class="c56 c20" colspan="1" rowspan="1"><P class="c2 c1"><SPAN class="c12 c87">C</SPAN><SPAN class="c15 c31 c12">GG</SPAN></P><P class="c2 c1"><SPAN class="c12 c20">AQLKKK</SPAN><SPAN class="c12 c4 c20">LQALKKK</SPAN><SPAN class="c12 c20">LAQLKWK</SPAN><SPAN class="c12 c4 c20">LQALKKK</SPAN><SPAN class="c9">N</SPAN><SPAN class="c12 c20">AQLKKK</SPAN><SPAN class="c15 c12 c4">LAQ</SPAN></P></TD></TR></TBODY></TABLE><P class="c1 c26 title" id="h.a3mr0qko3tzt"><SPAN class="c68">References:</SPAN><SPAN>                                        </SPAN></P><P class="c1"><SPAN>1.         Woolfson, D. N. (2005). The design of coiled-coil structures and assemblies. Advances in protein chemistry, 70, 79-112.</SPAN></P><P class="c1"><SPAN>                                                </SPAN></P><P class="c1"><SPAN>2.         Oakley, M. G., &amp; Kim, P. S. (1998). A buried polar interaction can direct the relative orientation of helices in a coiled coil. Biochemistry, 37(36), 12603-12610. </SPAN></P><P class="c1"><SPAN class="c8">Disulphide Design:</SPAN></P><P class="c1"><SPAN>Disulphide formation occurs between two cysteine anions, this generally requires an oxidant and a solution of pH above the pI of each cysteine residue. However the pI of a residue can be modified by it’s environment, positive charges surrounding the residue help to stabilise the anion form of the cysteine residue lowering the pI, allowing disulphide formation at a lower pH and at a higher rate than might usually occur</SPAN><SPAN class="c52">[1][2]</SPAN><SPAN>.</SPAN></P><P class="c1"><SPAN>References:</SPAN></P><P class="c1"><SPAN>1.        Hansen, R. E., Østergaard, H., &amp; Winther, J. R. (2005). Increasing the reactivity of an artificial dithiol-disulfide pair through modification of the electrostatic milieu. </SPAN><SPAN class="c14">Biochemistry</SPAN><SPAN>, </SPAN><SPAN class="c14">44</SPAN><SPAN>(15), 5899-5906.</SPAN></P><P class="c1"><SPAN>2.        Snyder, G. H., Cennerazzo, M. J., Karalis, A. J., &amp; Locey, D. (1981). Electrostatic influence of local cysteine environments on disulfide exchange kinetics. </SPAN><SPAN class="c14">Biochemistry</SPAN><SPAN>, </SPAN><SPAN class="c14">20</SPAN><SPAN>(23), 6509-6519.</SPAN></P><P class="c1 c26 title" id="h.tz4v7eqadkci"><SPAN class="c8">StarScaffold variation design:</SPAN></P><P class="c1"><SPAN class="c4">Prototype:  </SPAN></P><P class="c1"><SPAN>        The prototype includes a single 5 repeat antiparallel leucine zipper for dimerisation and a set of four inteins, designed to cover a range of active temperatures. This allows us to progressively build up macromolecular structures in an effort to form proteinaceous monolayers, filaments and potentially more complicated structures. Each segment is bookended by flexible linkers to prevent steric hinderance of segment folding. This is particularly a concern for the natural split inteins found at each end of the protein as they form a folded structure before interacting. However the same is not true for the artificial split inteins which should be more amenable to internal protein segments as this is part of their native full intein structure.</SPAN></P><P class="c1"><SPAN class="c4">dHelix:</SPAN></P><P class="c1"><SPAN>        The original helix on which the dimerisation domain is based was altered by lengthening the coil and changing the positions of the Asn residue used to promote antiparallel structure. The single coiled coil ‘prototype’ version is preferred because of the complication to structure and disulphide bond formation caused by four helixes total in each protein. But the ‘dHelix’ version does have the advantage of avoiding the increased coiled coil stability caused by the lengthening of the helix in the ‘prototype’ version. It also may have superior in vivo disulphide formation thanks to the extra secondary structure around the cysteines acting to potentially exclude disulphide bond isomerases.</SPAN></P><P class="c1"><SPAN>        </SPAN><SPAN class="c4">Helix and linker structure:</SPAN></P><P class="c1"><SPAN class="c4">Linker+cysteine         in yellow</SPAN></P><P class="c1"><SPAN class="c4">Cysteine                 in purple</SPAN></P><P class="c1"><SPAN class="c4">Linker                        in blue</SPAN></P><P class="c1"><SPAN class="c4">Star_one_pair_one:</SPAN></P><P class="c1"><SPAN>Base:                                         AQLKKKLQALKKKLAQLKWK</SPAN><SPAN class="c4 c42">N</SPAN><SPAN>QALKKKLAQ</SPAN></P><P class="c1"><SPAN>Acid:        </SPAN><SPAN class="c37">GSGSGSGGKR</SPAN><SPAN class="c10">C</SPAN><SPAN class="c37">RKGG</SPAN><SPAN>         AQLEKELQALEKELAQLEWE</SPAN><SPAN class="c4 c42">N</SPAN><SPAN>QALEKELAQ</SPAN></P><P class="c1"><SPAN class="c4">Star_one_pair_two:</SPAN></P><P class="c1"><SPAN>Base:         AQLKKKLQA</SPAN><SPAN class="c4 c42">N</SPAN><SPAN>KKKLAQLKWKLQALKKKLAQ         </SPAN><SPAN class="c37">GGKR</SPAN><SPAN class="c10">C</SPAN><SPAN class="c37">RKGGSGSGS</SPAN></P><P class="c1"><SPAN>Acid:        AQLEKELQA</SPAN><SPAN class="c4 c42">N</SPAN><SPAN>EKELAQLEWELQALEKELAQ</SPAN></P><P class="c1"><SPAN class="c4">Star_two_pair_one:</SPAN></P><P class="c1"><SPAN>Acid: </SPAN><SPAN class="c37">GGKR</SPAN><SPAN class="c10">C</SPAN><SPAN class="c37">RKGG</SPAN><SPAN>                AQLEKELQALEKELAQLEWE</SPAN><SPAN class="c4 c42">N</SPAN><SPAN>QALEKELAQ         </SPAN><SPAN class="c77">GSGSGSG</SPAN></P><P class="c1"><SPAN>Acid:                                AQLEKELQALEKELAQLEWE</SPAN><SPAN class="c4 c42">N</SPAN><SPAN>QALEKELAQ</SPAN></P><P class="c1"><SPAN class="c4">Star_two_pair_two:</SPAN></P><P class="c1"><SPAN>Base: AQLKKKLQA</SPAN><SPAN class="c4 c42">N</SPAN><SPAN>KKKLAQLKWKLQALKKKLAQ                </SPAN><SPAN class="c77">GSGSGSG</SPAN></P><P class="c1"><SPAN>Base: AQLKKKLQA</SPAN><SPAN class="c4 c42">N</SPAN><SPAN>KKKLAQLKWKLQALKKKLAQ          </SPAN><SPAN class="c37">GGKR</SPAN><SPAN class="c37 c83">C</SPAN><SPAN class="c37">RKGG</SPAN></P><P class="c1"><SPAN class="c4">NonSequential:</SPAN></P><P class="c1"><SPAN>        Sequential intein activation is not required for enzyme splicing, we have a set of four previously tested</SPAN><SPAN class="c52">[11]</SPAN><SPAN> inteins we can use for enzyme co-localisation which have been shown to have a high splicing rate, yield and orthologous splicing reactions. This construct allows us to check structural changes that occur due to splicing reactions occurring in the same mixture but at alternate temperatures.</SPAN></P><P class="c1"><SPAN class="c4">NoHelix:</SPAN></P><P class="c1 c45"><SPAN>By removing the helixes altogether we may measure the effect of helix presence on structure.</SPAN></P><P class="c1 c26 title" id="h.guze9b3kztac"><SPAN>Experiment Planning:</SPAN></P><P class="c1"><SPAN>        Before beginning wet-lab work we drew up a plan to check with our supervisors what follows is a copy of the working document we used to prepare our experiments ahead of time:</SPAN></P><OL class="c5 lst-kix_d073a8e97bzq-0 start" start="1"><LI class="c1 c65 c41"><SPAN>Prepare stocks of competent BL21(DE3) and DH5α cells using CaCl</SPAN><SPAN class="c72">2</SPAN><SPAN> method ([1] page 1.82)</SPAN></LI></OL><P class="c1 c41"><SPAN>Day 1.        Make FSB, SOC, SOB and SOB-agar and autoclave.</SPAN></P><P class="c1 c34"><SPAN>Plate BL21(DE3) and DH5α cells on LB plates and grow overnight.</SPAN></P><P class="c1 c41"><SPAN>Day 2.         Pick colonies and grow overnight in selective media. </SPAN></P><P class="c1 c41"><SPAN>Day 3.         Re-culture cells, resuspend and snap freeze.</SPAN></P><OL class="c5 lst-kix_d073a8e97bzq-0" start="2"><LI class="c1 c65 c41"><SPAN>Transform cells with empty pET-23a and pSB-1C3 and check cell competency:Day 1.        Transform cells, plate, and grow overnight.Day 2. Calculate transformation efficiency. </SPAN></LI></OL><P class="c1 c48"><SPAN>Grow the cells overnight for glycerol stockMake LB-Agar</SPAN></P><OL class="c5 lst-kix_d073a8e97bzq-0" start="3"><LI class="c1 c41 c65"><SPAN>Prepare Plasmid and Insert DNA for ligation:Day 1. Make glycerol stock</SPAN></LI></OL><P class="c1 c48"><SPAN>Miniprep, digest plasmids and gel electrophoresis.</SPAN></P><P class="c1 c51 c48"><SPAN>Day 2. Midiprep, restriction digest and gel electrophoresis of pSB1C3 plasmid </SPAN></P><P class="c1 c51 c48"><SPAN>Day 3. Run PCR Optimisation and gel electrophoresis.</SPAN><SPAN class="c4"> </SPAN></P><P class="c1 c48 c51"><SPAN>Day 4. Amplify insert DNA accordingly.</SPAN></P><P class="c1 c51 c48"><SPAN class="c4">        </SPAN><SPAN>Purify PCR products </SPAN></P><P class="c1 c78"><SPAN>Day 5. Digest and purify plasmid pSB1C3                Digest and gel purify insert DNA and Gel purify. </SPAN></P><OL class="c5 lst-kix_d073a8e97bzq-0" start="4"><LI class="c1 c65 c41"><SPAN>Ligate Inserts into pSB-1C3 and pET-23α:</SPAN></LI></OL><P class="c1 c45"><SPAN>Day 1.         GoldenGate reaction using BsaI-HF and BsmBI                Plasmid ligation reaction EcoRI/PstI and NdeI/NotI </SPAN></P><P class="c34 c1"><SPAN>Calculate transformation efficiency. </SPAN></P><P class="c1 c51 c48"><SPAN>Day 2. Transform ligated plasmid pSB1C3 (not the GoldenGate assembled gene) into DH5a cells </SPAN></P><P class="c1 c51 c80"><SPAN>PCR / Gel purify GoldenGate assembly </SPAN></P><P class="c1 c48"><SPAN>Assemble GoldenGate DNA into plasmids via  EcoRI/PstI and NdeI/NotI ligation </SPAN></P><OL class="c5 lst-kix_d073a8e97bzq-0" start="5"><LI class="c1 c65 c41"><SPAN>Transform constructs into BL21(DE3) and DH5α:Day 1. Transform cells, plate, and grow overnightDay 2. Colony PCR (+gel electrophoresis) and grow up cells.Day 3. Make glycerol stocks.         PCRDay 4.         DNA purification (miniprep) </SPAN></LI></OL><P class="c1 c45"><SPAN>Day 5. Sequencing PCR </SPAN></P><OL class="c5 lst-kix_d073a8e97bzq-0" start="6"><LI class="c1 c65 c41"><SPAN>Express the protein:Day 1. Grow them up overnight. Day 2. Dilute and induce expression and leave overnight.Day 3. Isolate protein in soluble and insoluble fraction.Day 4. SDS-PAGE to check overexpression.Day 5. Western Blot using Anti-His. </SPAN></LI></OL><OL class="c5 lst-kix_d073a8e97bzq-0" start="7"><LI class="c1 c65 c41"><SPAN>Purify our proteins:Day 1. Grow them up overnight. Day 2. Dilute, induce expression and leave overnight. Day 3. Lyse cells, Isolate protein in soluble and insoluble fraction. </SPAN></LI></OL><P class="c1 c45"><SPAN>Day 4. Analyse protein concentration for splicing assays         Day 5.        Purify protein using Nickel beads </SPAN></P><OL class="c5 lst-kix_d073a8e97bzq-0" start="8"><LI class="c1 c65 c41"><SPAN>Characterise inteins:        </SPAN></LI></OL><P class="c1 c41"><SPAN>Day 1.         Resuspend inteins in splicing buffer and perform time dependent assay of splicing activity.</SPAN></P><P class="c1"><SPAN>        Day 2. SDS-PAGE reactions against controls to estimate activity.</SPAN></P><P class="c1 c45"><SPAN>Day 3. Prepare samples for Quantitative Mass-Spectrometry (or similar)</SPAN></P><OL class="c5 lst-kix_d073a8e97bzq-0" start="9"><LI class="c1 c65 c41"><SPAN>Disulphide formation: Shuffle Cells:</SPAN></LI></OL><P class="c1 c45"><SPAN>Day 1.        Transform cells, plate, and grow overnight ([1] page 1.83).        Day 2. Reculture and grow to correct OD</SPAN><SPAN class="c72">600</SPAN></P><P class="c34 c1"><SPAN>Make glycerol stocks.        Induce and leave cells overnight to expressDay 3. Isolate protein in soluble and insoluble fraction. Day 4. SDS-PAGE to check overexpression. Day 5. Western Blot using Anti-His </SPAN></P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>