<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Yale_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Yale/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="page-wrapper" class="homepage"><DIV id="header-wrapper"><DIV id="header" class="container"><NAV id="nav"><UL><LI><A class="buttonn" href="https://2016.igem.org/Team:Yale/Description">Project</A></LI><LI><A class="buttonn" href="https://2016.igem.org/Team:Yale/Results">Results</A></LI><LI><A class="buttonn" href="https://2016.igem.org/Team:Yale/Requirements">Achievements</A></LI><LI><A class="buttonn" href="https://2016.igem.org/Team:Yale/Notebook">Notebook</A></LI><LI><A class="buttonn" href="https://2016.igem.org/Team:Yale/Software">Software</A></LI><LI><A class="buttonn" href="https://2016.igem.org/Team:Yale/Parts">Parts</A></LI><LI><A class="buttonn" href="https://2016.igem.org/Team:Yale/Attributions">Team</A></LI><LI><A class="buttonn" href="https://2016.igem.org/Team:Yale/Description/HP/Silver">Outreach</A></LI></UL></NAV></DIV><SECTION id="hero" class="container"><HEADER><H2>Project Description</H2></HEADER><UL class="actions"><LI><A href="#overview" class="buttonn">Overview</A></LI><LI><A href="#MAGE" class="buttonn">Multiplex Genome Engineering</A></LI><LI><A href="#target" class="buttonn">Target Organisms</A></LI></UL><UL class="actions"><LI><A href="#recombinases" class="buttonn">Recombinases for Genome Engineering</A></LI><LI><A href="#portmage" class="buttonn">pORTMAGE for Genome Engineering</A></LI></UL><UL class="actions"><LI><A href="#synbio" class="buttonn">SynBio Toolkit for Rhizobia</A></LI><LI><A href="#nodulation" class="buttonn">The Nodulation of Plant Roots</A></LI></UL></SECTION></DIV><DIV class="wrapper"><DIV class="container"><ARTICLE id="overview"><HEADER><H1>To Mars and beyond: engineering today’s most important microbes</H1><P>What does the colonization of Mars, the energy crisis, and the food crisis have in common? </P></HEADER><P>Earlier this month, U.S. President Obama outlined his plans to send humans to Mars by the 2030s with the ultimate ambition to one day remain there for an extended time. He joins Elon Musk, SpaceX and a collection of people and organizations interested in pursuing this goal. However, to make this vision a reality, we need a way to grow food in the harsh environment of Mars. Here on Earth, we also face challenges in meeting our future demand for food and energy which forecasted to increase by 50% and 45% respectively by 2030 (Gopalakrishnan et al. 2015).</P><P> There is one organism that has the potential to solve all of these problems – rhizobia. Rhizobia are nitrogen-fixing bacteria that form symbiotic root nodules with legumes. By increasing their hardiness and nitrogen fixation efficiency, we can expand the arable land that can be used to grow crops and reduce our dependence on ecologically-damaging nitrogen-containing fertilizers. We chose to focus on two strains of rhizobia, <EM>Rhizobium tropici CIAT 899</EM> and <EM>Sinorhizobium meliloti 1021</EM>.</P><P> To engineer them, we selected a technique called multiplex automated genome engineering (MAGE). MAGE is a form of genetic engineering that generates increased genetic diversity at targeted genomic loci through the introduction of synthetic oligonucleotides, allowing for the directed evolution of cell populations (Wang et al. 2009). By creating a framework to implement MAGE in rhizobia, we open up numerous potential applications; for example the optimization of nitrogen fixation mechanisms. Furthermore, our work with rhizobia can be used to guide the implementation of MAGE in other non-model organisms.</P><ARTICLE id="MAGE"><HEADER id="MAGE"><H1>Multiplex engineering of rhizobial genomes</H1><P>Our 2016 Yale iGEM team is focusing on optimizing multiplex automated genome engineering (MAGE) in two strains of rhizobia, <EM>Rhizobium tropici (R. tropici)</EM> CIAT 899 and <EM>Sinorhizobium meliloti (S. meliloti) 1021</EM>, for applications in agriculture and nitrogen fixation.</P></HEADER><P>MAGE is a useful form of genetic engineering because it generates increased genetic diversity at targeted genomic loci through the introduction of synthetic oligonucleotides, and allows for directed evolution of cell populations (Wang, et al.). MAGE has a variety of applications in genome modification, including optimization of a protein or small molecule production pathway, tuning of gene expression, modification of ribosomal binding sites, and knocking out of undesired genes. Rhizobium was chosen not only for its significance in food production and nitrogen applications, but also because it is a non-model microorganism that can improve characterization of such organisms, ultimately providing a framework for implementing MAGE in non-model organisms.</P><P>MAGE increases genetic diversity by using the Red recombineering cassette to induce mutations along the genome of an organism, including insertions, deletions, and point mutations. However, the efficiency of MAGE is decreased by the cell’s natural mismatch repair (MMR) system, which corrects any mismatches caused by MAGE. To reduce this problem in <EM>E. coli</EM>, researchers knock out the mutS gene, a key component of the MMR system. By knocking out mutS, the efficiency of MAGE increases, but this process has not been shown to be easily done in non-model microorganisms such as rhizobia.</P></ARTICLE><HEADER id="target"><H1>Target Organisms: <EM>Rhizobium tropici CIAT 899</EM> and <EM>Sinorhizobium meliloti 1021</EM>, broadly symbiotic nitrogen-fixing bacteria</H1></HEADER><P>We selected two strains of rhizobia for our project based on the literature available, their ability to nodulate a variety of agriculturally relevant legumes and other potential applications.<EM>R. tropici</EM> was selected for its ability to form nitrogen-fixing symbiotic relationships with a broad range of legume hosts, including those of the <EM>Phaseolus</EM> and <EM>Leucaena</EM> genera (Martinez-Romero et al. 1991). Compared to other rhizobia species, it is more tolerable of stressful conditions such as acidic soils and high temperatures (Graham et al. 1994). It is ideal for our use because it has a reasonably fast doubling time of less than 6 hours (Morón et al. 2005) and because its genome is fully sequenced (NCBI Taxonomy ID: 698761). <EM>S. meliloti</EM> is similarly able to nodulate a wide range of host plants, primarily those from the <EM>Medicago, Melilotus,</EM> and <EM>Trigonella</EM> genera (Roumiantseva et al. 2002). It is particularly unique in its denitrification abilities. These rhizobia are able to reduce nitrate and nitrite into N2 as well as fix atmospheric nitrogen because they have all four nitrogen oxide reductases (House 2003). Similar to <EM>R. tropici, S. meliloti</EM> is a fast growing species whose genome is fully sequenced (NCBI Taxonomy ID: 266834).</P><HEADER id="recombinases"><H1>Constructing recombinases for Multiplex Automated Genome Engineering (MAGE)</H1></HEADER><P>MAGE was selected for our purposes because it efficiently generates increased genetic diversity at targeted genomic loci through the introduction of synthetic oligonucleotides, and allows for directed evolution of cell populations (Wang, et al. 2009). MAGE has a variety of applications in genome modification, including optimization of a protein or small molecule production pathway, tuning of gene expression, modification of ribosomal binding sites, and knocking out of undesired genes.</P><P>It relies on the introduction of degenerate oligonucleotides with 30-45 bp homology arms that are complimentary to the target sequence. The Lambda-Red recombineering cassette is used to induce mutations along the genome of an organism, including insertions, deletions, and point mutations. However, the efficiency of MAGE is decreased by the cell’s natural mismatch repair (MMR) system, which corrects any mismatches caused by MAGE. To reduce this problem in <EM>E. coli</EM>, researchers knock out the mutS gene, a key component of the MMR system. By knocking out mutS, the efficiency of MAGE increases, but this process has not been shown to be easily done in non-model microorganisms such as rhizobia.</P><P>Currently, the most effective recombination protein is Beta recombinase from Lambda, a phage that infects <EM>E. coli</EM>. This means that it is not optimized for our rhizobia species. Thus, in order to increase the efficiency of MAGE in rhizobia, we looked for homologues in the rhizobia genome and in phages that infect rhizobia. To facilitate this process, we developed a software tool that employs the hidden Markov model-based search algorithm hmmer to search for homologues on the Viral Genome Database and on the ENSEMBL Bacterial Genomes Database and display them in a phylogenetic tree.  Ligation independent cloning was then used to clone the recombinases into vectors for testing.</P><H1>pORTMAGE-3: An independent system for MAGE on a plasmid </H1></ARTICLE><P>To streamline the implementation of MAGE, scientists have developed pORTMAGE-3, a broad-host vector that contains a beta recombinase, a dominant negative mutL mutant, and a temperature-activated promoter. However, these components come from <EM>E. coli</EM> and are not validated in rhizobia. Thus, in parallel with our work with recombinases, we transformed and tested rhizobia with pORTMAGE-3 and hope to optimize it.</P><HEADER id="synbio"><H1>Constructing a synthetic biology toolkit</H1></HEADER><P>In order to make rhizobia easier to engineer, we worked on establishing a library of inducible and constitutive promoters compatible with rhizobia. To do this, we identified potential promoters through extensive literature searches on the organisms in question. Using ligation independent cloning, these promoters were cloned into vectors with citrine fluorescent protein and tested in <EM>E. coli</EM> and the rhizobia strains.</P><HEADER id="nodulation&quot;"><H1>Nodulation Assay: Ensuring that rhizobia do not lose the <EM>nod</EM> plasmid after MAGE heat-shock </H1></HEADER><P>During MAGE, a brief heat shock period (42°C for 30 seconds) is performed to promote the uptake of mutagenic oligonucleotides into the cell. However, it has been shown that upon heat shock, rhizobia are prone to losing peripheral plasmids. One plasmid of importance is the <EM>nod</EM> plasmid, which contains the genes necessary for producing proteins essential in nodule formation. The purpose of performing a nodulation assay is to ensure that the nod plasmid is not lost upon the MAGE heat-shock step. Such loss of the nod plasmid would undermine the significance of rhizobia in the legume-rhizobia symbiosis that allows for nitrogen fixation.</P><HEADER><H1>References</H1></HEADER><P>Gopalakrishnan, S., et al., Plant growth promoting rhizobia: challenges and opportunities. 3 Biotech, 2015. 5: p. 355-377.</P><P>Graham PH, Draeger KJ, Ferrey ML, Conroy MJ, Hammer BE, Martinez E, Aarons SR, Quinto C. Acid pH tolerance in strains of Rhizobium and Bradyrhizobium, and initial studies on the basis for acid tolerance of Rhizobium tropici UMR1899. Can J Microbiol/Rev Can Microbiol.1994;40:198-207. doi: 10.1139/m94-033</P><P>House, B. Novel Methods of Manipulating the Sinorhizobium meliloti Genome and Studying the Effects of Denitrification on Symbiotic Nitrogen Fixation. Washington State University, School of Molecular Biology, pg. 101-125, 2003.</P><P>Martinez-Romero E, Segovia L et al. &quot;Rhizobium tropici, a Novel Species NodulatingPhaseolus vulgaris, L. Beans and Leucaena sp. Trees.&quot; Int. J. of System. Biotech.1991 41(3), 417-426.</P><P>Morón B, Soria-Diaz ME, Ault J, Verroios G, Noreen S, Rodriguez-Navarro DN, Gil-Serrano A, Thomas-Oates J, Megias M, Sousa C. Low pH changes the profile of nodulation factors produced by Rhizobium tropici CIAT899. Chem Biol. 2005;12:1029-1040. doi: 10.1016/j.chembiol.2005.06.014.</P><P>Roumiantseva, M. L. et al., Diversity of Sinorhizobium meliloti from the Central Asian Alfalfa Gene Center. Applied and Environmental Microbiology, 68: 4694-4697.
Wang, H. H., et al., Programming cells by multiplex genome engineering and accelerated evolution. Nature, 2015. 460: p. 894-898.</P></DIV></DIV></DIV><DIV id="footer-wrapper"><DIV id="footer" class="container" style="padding:0px; line-height:0em; text-align:center;"><H2>Yale iGEM 2016</H2><P>Main Campus:</P><P>Yale Department of Molecular, Cellular &amp; Developmental Biology</P><P>Attn: Farren Isaacs/iGEM</P><P>219 Prospect St</P><P>PO Box 208103</P><P>New Haven, CT 06520</P><P>Tel. +1(203) 432-3783</P><P>E-mail: igem.yale@gmail.com</P></DIV><DIV id="copyright" class="container"><UL class="menu"><LI>©All rights reserved Yale iGEM 2016.</LI><LI>Design: <A href="http://html5up.net">HTML5 UP</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>