Team:Yale/Results
Project
Results
Achievements
Notebook
Software
Parts
Team
Outreach
Results
Directed Evolution
Synthetic Biology Toolkit
Recombinases
pORTMAGE
Nodulation
Directed Evolution of Rhizobia via Untargetted Mutagenesis to acheive tolerance to Extreme Conditions (UV)
First, we attemped to see if we could use classical untargeted mutagenesis to evolve tolerance to extreme conditions in Rhizobium. To this end, Rhizobium tropici CIAT was exposed to 3 successive rounds of 1000uJ/ch^2 254nm UV light to Evolve UV tolerance. After the final recovery, we exposed the Evolved cells and the ancestral Wild Type cells to a final round UV. Cells were immediatly plated on LB agar to observe phenotypic differences
The Evolved cells recovered more quickly than Wild Type, as seen by the substantially larger colonies formed the Evolved cells.
This data confirms that classical untargeted mutagenesis can be used to evolve extreme condition-tolerant rhizobia.
Establishing a Synthetic Biology Toolkit for Rhizobia
As mentioned previously , we attemped to characterize a synthetic biology toolkit for Rhizobia in order to make the strain for tractable for engineering.
First, we attempted characterize natural inducible promoters from Sinorhizobium meliloti. We chose to test Arabinose, Rhamnose, Taurine, and Melibiose - inducible promoters. Initially, we cloned these promoters upstream of a Citrine reporter gene into a the pZP vector compatible in both E. coli and Sinorhizobium meliloti. We cloned these promoters into E coli for initial propogation.
The resulting data worked as expected in E. coli. Becuase the Rhamnose promoter is regulated by a repressor, it was constitutively on in E. coli. Because the arabinose, taurine, and melibiose promoters are regulated by activators native to Sinorhizobium, they were all off in E. coli. We are in the process of moving these promoters into Sinorhizobium for characterization.
Next we attempted to characterize Constitutive Promoters based on the Anderson Collection (Bba_J23119). Our initial attempt to clone these promoters upstream of the Citrine reporter gene had gone poorly. When we inspected the Ribosome Binding Site used with the promoters, we realized that it would need to be improved. We used the Salis Lab Ribosome Binding Site Calculator to design a stronger Ribosome Binding Site.
Below, we demonstrate that the newly designed Ribosome Binding Site is appropriately compatible with the Anderson Promoter collection.
Cloning Recombinases to Promote MAGE Genome Engineering in Rhizobium
As introduced previously ,
MAGE is a useful form of genetic engineering because it generates increased genetic diversity at targeted genomic loci through the introduction of synthetic oligonucleotides, and allows for directed evolution of cell populations (Wang, et al.). MAGE has a variety of applications in genome modification, including optimization of a protein or small molecule production pathway, tuning of gene expression, modification of ribosomal binding sites, and knocking out of undesired genes. Rhizobium was chosen not only for its significance in food production and nitrogen applications, but also because it is a non-model microorganism that can improve characterization of such organisms, ultimately providing a framework for implementing MAGE in non-model organisms.
MAGE increases genetic diversity by using the Red recombineering cassette to induce mutations along the genome of an organism, including insertions, deletions, and point mutations. However, the efficiency of MAGE is decreased by the cell’s natural mismatch repair (MMR) system, which corrects any mismatches caused by MAGE. To reduce this problem in E. coli, researchers knock out the mutS gene, a key component of the MMR system. By knocking out mutS, the efficiency of MAGE increases, but this process has not been shown to be easily done in non-model microorganisms such as rhizobia.
We identified several recombinases (homologs of recombinase Beta of coliphage lambda) and cloned them into the aforementioned vectors and promoter systems.
Characterizing pORTMAGE for MAGE Genome Engineering
As introduced previously , an alternative strategy for MAGE Genome Engineering is the use the pORTMAGE system deceloped to be compatible with a wide array of Gram-Negative Bacteria. We transformed this plasmid into E coli to confirm its activity. For this assay, we activated pORTMAGE in E. coli MG1655 with a 15 minute 42C induction, prepared electrocompetent cells, and transformed a MAGE 90mer oligonucleotide designed to make a point mutation in the rpoB gene, which confers resistance to rifampicin. Cells are recovered for 3 hours and serial dilutions are plated on LB and LB+rifampicin to calculate the frequency of MAGE allele conversions
Encouragingly pORTMAGE was able to mutate nearly 5% of cells.
Given this encouraging result, we then transformed pORTMAGE into Sinorhizobium meliloti and Rhizobium ciat. Transformations were successful, as confirmed by colony PCR. We are now in the process of attempting a similar rifampicin assay to measure MAGE efficiencies.
Nodulating Rhizobium into Plant Roots
As introduced previously , ultimately, our engineered Rhizobia will be used to nodulate the roots of plants in harsh conditions here on Earth and potentially for the terraformation of Mars. We set up a nodulation apparatus to simulate real-life nodulation of plants with our engineered microbes. Indeed we were able to successfully nodule bean plants with rhizobia transformed with the pORTMAGE plasmid.
Protocol:
1. Sterilize cowpea (black-eyed pea) seeds by gently shaking for 15 minutes in a solution of 5% NaClO4 and 0.5-mL of Tween-20 detergent.
2. Using the CYG high-contrast nodulation apparatus from Mega International of Minneapolis, place sterilized cowpea seeds onto the apparatus.
3. Saturate the apparatus with distilled water, and allow root growth to occur (3-5 days), replenishing water as needed.
4. Inoculate the top of each apparatus with liquid culture of rhizobia (both strains), using roughly 1-mL of culture (grown to mid-log phase) per apparatus pouch.
5. Allow inoculation to proceed for 2-4 days, observing the presence or absence of nodule formation on the roots.
Yale iGEM 2016
Main Campus:
Yale Department of Molecular, Cellular & Developmental Biology
Attn: Farren Isaacs/iGEM
219 Prospect St
PO Box 208103
New Haven, CT 06520
Tel. +1(203) 432-3783
E-mail: igem.yale@gmail.com
©All rights reserved Yale iGEM 2016.
Design: HTML5 UP
