<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UC_San_Diego_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UC San Diego/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="header" id="top"><NAV class="navbar navbar-inverse" role="navigation"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" id="nav-toggle" class="navbar-toggle" data-toggle="collapse" data-target="#main-nav"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A href="https://2015.igem.org/Team:UC_San_Diego" class="navbar-brand">ucsd  igem</A></DIV><DIV id="main-nav" class="collapse navbar-collapse"><UL class="nav navbar-nav navbar-right"><LI><A>Project </A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Description">Overview</A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Background">Background</A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Module">Module</A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Interlab">Interlab</A></LI></UL><LI><A>Modeling </A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Modeling/Kinetics">Enzyme Kinetics</A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Modeling/Constraint">Constraint-Based</A></LI><LI><A>Notebook </A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Notebook">Timeline</A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Safety">Safety</A></LI><LI><A>Team </A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Team">Members</A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Collaborations">Collaborations</A></LI></DIV></DIV></NAV></DIV></DIV><DIV id="heading-pages"><DIV class="row"><DIV class="col-md-6 col-md-offset-3 col-xs-8 col-xs-offset-2"><H2>TIMELINE</H2></DIV></DIV></DIV><SECTION id="cd-timeline" class="cd-container"><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>September 24-28</H2><P>Giant Jamboree!</P><SPAN class="cd-date">Giant Jamboree!</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>September 16th</H2><P>Library presentation at La Jolla Riford Library.</P><A href="/Team:UC_San_Diego/Practices#library" class="cd-read-more">Read more!</A><SPAN class="cd-date">WEEK 14</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Modeling</H2><P>&gt;Finished documentation for the enzymatic kinetics model
&gt;Finished written documentation for constraint-based modeling
&gt;Finished documenting COBRA scripts
&gt;Set up Github account for uploading scripts</P><SPAN class="cd-date">WEEK 14</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Wet Lab</H2><P>+ Final preparations before the wiki freeze.</P><SPAN class="cd-date">WEEK 14</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Modeling</H2><P>&gt;Worked on documentation for the enzymatic kinetics model 
&gt;Began the development of a presentation in preparation for both the competition and upcoming talk at the library
&gt;Finalized all figures and scripts. 
Finished tracing pathways for upregulation of light and growth in COBRA model
&gt;Editing documentation, finding references
Making figures and tables for the presentation</P><SPAN class="cd-date">WEEK 13</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Wet Lab</H2><P>+ Mapped out errors in the parts that we assembled
                                + Designed mutagenesis primers to fix recurring errors in C sequence
</P><SPAN class="cd-date">WEEK 13</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>September 4th</H2><P>High school meet up.</P><A href="/Team:UC_San_Diego/Collaborations#hsmeetup" class="cd-read-more">Read more!</A><SPAN class="cd-date">WEEK 12</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Modeling</H2><P>&gt;Successfully managed to acquire results for the individual constructs under certain conditions. 
&gt;Began documentation, finished searching for our references 
Began documentation for constraint-based modeling
Started tracing pathways on reactions coupled with light and growth
	&gt;searched enzyme names on KEGG to identify upregulated subsystems
</P><SPAN class="cd-date">WEEK 12</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Wet Lab</H2><P>+ Added CDE fragments to AB plasmids via Gibson assembly
                                + Designed Sanger sequencing primers
                                + Miniprepped final plasmid clones
</P><SPAN class="cd-date">WEEK 12</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Modeling</H2><P>&gt;Reduced model as suggested by the work of D. Sekel and M. Iqbal in attempt to simulate <I>in vivo</I> behavior at steady state
&gt;Began analysis of the different genetic constructs
Defined COBRA model in YPD media
	&gt;listed exchange reactions (reaction ids, reaction names) of both models in an excel document
	&gt;set bounds for inactive reactants to ‘0’
	&gt;less risk of noise and variation in fba.</P><SPAN class="cd-date">WEEK 11</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Wet Lab</H2><P>+ Attempted Gibson assembly of AB fragments
                                + Miniprepped CDE fragments 
</P><SPAN class="cd-date">WEEK 11</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Modeling</H2><P>&gt;Utilized model reduction strategies (i.e. quasi-steady state assumption, rapid equilibrium assumption, total quasi-steady state assumption etc.) in attempt to reduce our current model to one time-scale and cut the elapsed time for our simulations. 
Decided to focus on identifying pathways essential to light production
	&gt;started writing script to capture the fluxes through every reaction while varying growth and light
	&gt;Troubleshooting: only one fba solution for growth and light (linear relationship)</P><SPAN class="cd-date">WEEK 10</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Wet Lab</H2><P>+ Analyzed sequencing data to identify error-free clones
         + Designed primers for PacBio Sequencing
                                + Amplified error-free clones for all fragments
</P><SPAN class="cd-date">WEEK 10</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Modeling</H2><P>&gt;Finished code for the induction curve simulations and began generating preliminary results
&gt;Explored light emission trace
Generated Robustness Analysis plots
	&gt;light vs. growth
	&gt;light vs. glucose
Generated Phenotype Phase Plane plots
	&gt;possible error in code, the graphs do not look as predicted
Updated both COBRA models with finalized reaction pathways</P><SPAN class="cd-date">WEEK 9</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Wet Lab</H2><P>+ Began subcloning of fragments into pUCGA vector to find error-free clones
</P><SPAN class="cd-date">WEEK 9</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>August 6th</H2><P>Southern California iGEM Meetup</P><A href="/Team:UC_San_Diego/Collaborations#socalmeetup" class="cd-read-more">Read more!</A><SPAN class="cd-date">Week 8</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Modeling</H2><P>&gt;Converted light production pathway and aldehyde synthesis pathway to a set of ODE equations 
&gt;Began simulations on MATLAB. In addition, most rate constants and fixed concentrations were found
&gt;Checked the functionality of our model.
Completed the isolated COBRA model using the BiGG database
	&gt;unable to resolve layout of isolated model in CyFluxViz
	&gt;started working on the isolated model in the version 7 database</P><SPAN class="cd-date">WEEK 8</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Wet Lab</H2><P>+ Received our DNA fragments from SGI thanks to the <A href="https://sgidna.com/bxp3200.html">BioXP!</A>+ Attempted amplify fragments via PCR for assembly and transformation.</P><SPAN class="cd-date">WEEK 8</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Modeling</H2><P>&gt;Finished algorithm for parameter scan
&gt;Finalized our core kinetic diagram
&gt;Started search for rate constants</P><SPAN class="cd-date">WEEK 7</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Wet Lab</H2><P>+ Assembled Interlab Devices. Resuspended and measured them successfully.
                                + Prepared YPD plates.</P><SPAN class="cd-date">WEEK 7</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Modeling</H2><P>&gt;Updated core model to include intermediate species, such as the primary light emitter, and other key reactions, such as aldehyde inhibition, that are key for continuous light emission
Researching literature for useful COBRA model constraints
	&gt;made a table of uptake and secretion fluxes in units of mmol/(g*hr)
	&gt;found a paper, Biomass composition: the ‘‘elephant in the room’’ of metabolic modelling
	&gt;contacted professor for supplemental information</P><SPAN class="cd-date">WEEK 6</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Wet Lab</H2><P>+ Continued interlab study with miniprep, restriction digests, ligation, gel electrophoresis and gel purification.</P><SPAN class="cd-date">WEEK 6</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Modeling</H2><P>&gt;Literature research 
&gt;Looking for techniques to combine the deterministic model and genome-scale model
&gt;Attended the Q-bio Summer School: Computational Synthetic Biology 
Learned to code SBML
	&gt;added reactions and metabolites manually to version 7 model since the built-in function addReactions was unusable
</P><SPAN class="cd-date">WEEK 5</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Wet Lab</H2><P>+ Started interlab study by transforming parts from the 2015 distribution.</P><SPAN class="cd-date">WEEK 5</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Modeling</H2><P>&gt;Searched for genome-scale model parameters 
&gt;Developed core network diagram and found some nominal parameters for ODE model 
&gt;Constructed core model for our bioluminescent system
&gt;Compile various literature sources necessary to understand bioluminescent network
&gt;Developed kinetic equations corresponding to network
Thinking of making an isolated COBRA model to visualize flux distribution
	&gt;installed Cytoscape and CyFluxViz
	&gt;investigation on exporting model and fluxes from Matlab structure format
</P><SPAN class="cd-date">WEEK 4</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Wet Lab</H2><P>+ Sent our genes to SGI for synthesis on the <A href="https://sgidna.com/bxp3200.html">BioXP</A>and ordered lab supplies. Almost ready to go!</P><SPAN class="cd-date">WEEK 4</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Modeling</H2><P>&gt;Developed lecture on enzymatic kinetics and gene expression modeling, along with corresponding MATLAB simulations 
Continued Investigation of literature on lux system reactions in-detail
	&gt;attempted to link pathways to rate-constants found in literature
	&gt;made first digitalized diagram of the entire bioluminescent system
Installed MATLAB student copy sponsored by iGEM
&gt; Made first script to add igem reactions
Installed APE
	&gt; thinking about incorporating protein synthesis into the COBRA model
Investigation about the CrabTree Effect in s288c
	&gt;could affect how much light is made (acetate is a substrate)</P><SPAN class="cd-date">WEEK 3</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Wet Lab</H2><P>+ Added frp gene sequence from Vibrio Harveyi to our plasmid to stabilize luminescent output.
+ Codon optimized our plasmid sequences for expression in yeast.
</P><SPAN class="cd-date">WEEK 3</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Modeling</H2><P>
Became familiarized with genome-scale modeling and metabolic control analysis
Literature research: Understanding the different components of the bioluminescent system
	&gt;aldehyde synthesis 
	&gt;light production 
Researched Literature on COBRA analysis and methods
&gt;used e.coli as a toy model 
&gt;ran stimulations of FBA, robustness analysis, and phenotype phase plane


</P><SPAN class="cd-date">WEEK 2</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Wet Lab</H2><P>+ Created a preliminary design for the plasmids using ApE.
+ Improved them over the week by adding tags, removed illegal restriction sites, and changing repetitive sequences.

</P><SPAN class="cd-date">WEEK 2</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Modeling</H2><P>
Developed and finalized Primer to MATLAB Programming
Researched potential modeling avenues
	&gt;genome scale analysis of synthetic construct
	&gt;analysis of the tradeoffs between different genetic circuit designs and gene dynamics
	&gt;analysis of  metabolic activity in an  isolated network via Metabolic Control Analysis
Browsed recent literature for lux system
	&gt;identified its advantages and disadvantages as a reporter system
	&gt;differences in luciferase systems of beetles and bacteria
Familiarized ourselves with basics of enzyme kinetics 
	&gt;Michaelis-Menten equations
Downloaded COBRA Toolbox plugin for MATLAB
</P><SPAN class="cd-date">WEEK 1</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Wet Lab</H2><P>
+ Found genes coding for fatty acid reductase complex that have been validated in an <I>in vitro</I> synthesis paper.
+ Planned how to assemble our plasmids and determined nucleotide sequences for <I>lux</I>CDABE of <I>Photobacterium Phosphoreum</I>. + Compared the amino acid sequences to those of other organisms on BLAST to check for significant discrepancies in our sequence.
</P><SPAN class="cd-date">WEEK 1</SPAN></DIV></DIV></SECTION><DIV class="logorow"><DIV class="row"><DIV class="copyright-text"><P><A href="mailto:ucsd.igem.2015@gmail.com"><FONT color="#1b988b">Contact Us: ucsd.igem.2015@gmail.com</FONT></A></P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>