<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UC_San_Diego_Modeling_Constraint skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UC San Diego/Modeling/Constraint</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="header" id="top"><NAV class="navbar navbar-inverse" role="navigation"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" id="nav-toggle" class="navbar-toggle" data-toggle="collapse" data-target="#main-nav"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A href="https://2015.igem.org/Team:UC_San_Diego" class="navbar-brand">ucsd  igem</A></DIV><DIV id="main-nav" class="collapse navbar-collapse"><UL class="nav navbar-nav navbar-right"><LI><A>Project </A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Description">Overview</A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Background">Background</A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Module">Module</A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Interlab">Interlab</A></LI></UL><LI><A>Modeling </A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Modeling/Kinetics">Enzyme Kinetics</A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Modeling/Constraint">Constraint-Based</A></LI><LI><A>Notebook </A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Notebook">Timeline</A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Safety">Safety</A></LI><LI><A>Team </A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Team">Members</A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:UC_San_Diego/Collaborations">Collaborations</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-md-12"><DIV class="main-post"><DIV class="post-content"><H3>Genome Scale Modeling</H3><P>One of our main objectives for this project is to build a comprehensive, genome-scale model that details the function of the Lux system and its interaction with the host organism, <I>Saccharomyces Cerevisiae</I> (s288c).
</P><P>Genome-scale models (M-Models) represent an effort to bridge the gap between isolated systems (e.g. signaling pathways, glycolysis) to modeling whole systems of organisms<SUP>1</SUP>. Metabolic network reconstructions input the full range of known biochemical reactions in an organism to give comprehensive insight into how the components of the genome result in a particular phenotype. These elaborate models are usually encoded in the Systems Biology Markup Language (SBML). 
</P><P>There are currently many M-Models available in literature, mostly detailing the networks of different strains of bacteria and the simpler eukaryotes<SUP>2</SUP>. The yeast strain of <I>Saccharomyces Cerevisiae</I> (s288c) was one of the first to be converted into a network model. For this and for reasons of experimental simplicity, we chose this strain as our base model.
</P><P>The UC system-wide biochemically, genetically, and genomically (BiGG) database features two different networks for the genome of <I>Saccharomyces Cerevisiae</I>. The latest model reconstruction, iND750, consists of 1059 metabolites, 1266 reactions, and 750 genes<SUP>2</SUP>. In early 2007, a community of researchers came together to standardize the metabolic map of <I>Saccharomyces Cerevisiae</I>. They collaborated to periodically update the model and make it available in the public domain (yeast.sf.net) for the years that followed. The latest version to date, version 7, includes reactions published by Aung et al. in 2013<SUP>3</SUP>.
</P><P>Ultimately, we decided to model the Lux system in both available models – BiGG and v7. After evaluating the strengths and weaknesses of both models, we were not convinced that either model provided a complete and accurate representation of the yeast metabolism. The v7 model would seem to be the logical choice, because it is frequently used and updated by the yeast community; however, we were concerned when we discovered many reactions and metabolites missing from the native fatty acid biosynthesis pathway. This pathway is crucial to the Lux system, because <I>lux</I>D interacts with myristic-ACP, also known as tetradecanoyl-ACP<SUP>4,5</SUP>. With the v7 model terminating fatty acid biosynthesis at octanoyl-ACP, we were forced to add the necessary reactions with the assistance of the KEGG database<SUP>4,5</SUP>.
</P><H3>Metabolic Control Analysis and Flux Balance Analysis</H3><P>The term metabolic flux is used to describe the rate of turnover of metabolites in these large-scale networks. Flux for an individual reaction is the difference between the rates of the forward reaction and the reverse reaction<SUP>6</SUP>.
</P><P>The main objective of Flux Balance Analysis (FBA) is to find a solution for a system of linear equations, <B>(S*v = b)</B>. Using the coefficients -1,0, and 1, we can identify which metabolites actively participate in each reaction and form the stoichiometric matrix, <B>S. b</B> is assumed to be a zero matrix, because at steady-state, the rate of consumption is equal to the rate of production. Hence, flux balance analysis returns <B>v</B>, the column vector of the flux of each reaction solved at steady state<SUP>7</SUP>. </P><DIV class="caption"><CENTER><I><B>Figure 1.</B> Converting a set of reactions to a mathematical model.</I></CENTER></DIV><P>However, FBA has many limitations. FBA cannot predict metabolic concentrations, because it does not utilize kinetic parameters. Also, it cannot account for regulatory agents such as those involved in transcription or translation of the genome<SUP>7,8</SUP>.</P><H3>Constraint-Based Modeling</H3><P>We used the COBRA Toolbox plugin in MATLAB to edit and analyze our SBML model<SUP>9</SUP>. The steady-state assumption that is employed in FBA is one example of a constraint. Constraints define the allowable solution space in FBA and accounts for many reaction properties that cannot be encoded by simple stoichiometry. Metabolic constraints, such as flux coefficients, define the physico-chemical boundaries of a reaction. The lower bound of the flux coefficient also tells us if the reaction is reversible or irreversible<SUP>10</SUP>.
</P><P>
Space constraints, such as the use of compartments, define the physical boundaries on which metabolites can react with one another. There are many other types of constraints that can be embedded into a genome-scale model such as regulatory constraints, thermodynamic constraints, and temporal constraints. All parameters seek to eliminate infeasible solutions and define the solution space as closely as possible<SUP>1,10</SUP>.
</P><P>The most important constraint is the objective function. The COBRA Toolbox allows us to maximize or minimize the flux through certain reactions of interest<SUP>1</SUP>. For instance, setting the objective to biomass production allows us to predict the growth rate of a yeast cell once it reaches steady-state. For our purposes, maximizing the flux through the light reaction was set as the main objective.
</P><H3>Results</H3><P>In both models, we first defined a minimum media to mimic experimental conditions. The components of Yeast Extract Peptone Dextrose (YPD) were listed and linked to exchange reactions in both the v7 and the BIGG model. Exchange reactions are defined to describe the metabolite composition in the immediate surroundings of the cell. The upper and lower bounds for all other exchange reactions were then set to ‘0’. 
</P><P>Once the Lux reactions were inserted into both models, we analyzed the metabolic burden on the cell using a two-step process. First, we ran the built-in COBRA function <B>robustnessAnalysis</B>. This function plots the flux through a pathway against the flux through the optimized pathway<SUP>1</SUP>. In this case, the flux through the light pathway was negatively correlated with the flux through the growth pathway. We concluded that this result could only have been obtained if the light pathway was redirecting a metabolite that would normally have lent itself to growth. 
</P><DIV class="caption"><CENTER><I><B>Figure 2.</B> Flux through light on the y-axis and flux through growth on the x-axis.</I></CENTER></DIV><P>To confirm this hypothesis, we needed to identify which reactions in the native genome were coupled to light. We developed a script that set light as the objective function, and then constrained growth in increments until it reached its max value. At each growth value, we took the vector of all the fluxes through every reaction. In the end, each reaction fell into one of four categories: <B>incr</B>, <B>decr</B>, <B>zero</B>, and <B>neither</B>. In <B>incr</B>, we had a list of reactions strongly coupled to growth. The flux through these reactions increased at every point that growth increased. Likewise, in <B>decr</B>, we held a list of reactions strongly coupled to light. In <B>zero</B>, we stored a list of reactions that were uncoupled to growth and light. In <B>neither</B>, we stored reactions, whose flux varied with no discernible pattern.

</P><DIV class="caption"><CENTER><I><B>Figure 3.</B> Reactions coupled with light and their respective subsystems.</I></CENTER></DIV><P>The BiGG model returned 34 reactions that were strongly coupled to light and the v7 model returned 56 reactions. The transport reactions in this category were then removed to better ascertain which subsystems (Glycolysis, Citric Acid cycle etc.) these reactions belonged to<SUP>4,5</SUP>. The diagram below traces the light pathway to its origin and shows its progression into the fatty acid biosynthesis pathway. This diagram allowed us to understand how the light pathway redirected metabolites from the growth pathway.
</P><DIV class="caption"><CENTER><I><B>Figure 4.</B> Flow Diagram detailing the upregulation of flux leading to the Lux pathway.</I></CENTER></DIV><P>Finally, to confirm our results, we ran a <B>robustnessAnalysis</B> on flux through the light pathway versus that through transketolase, an enzyme in the pentose phosphate pathway. We were expecting to see the flux of light monotonically increase, as predicted by the previous sorting algorithm. However, we observed a peak in the graph of the flux through light and a subsequent decline in light with increasing transketolase activity. This effect can only be explained by the lack of constraints on the flux through the growth pathway. From this, we conclude that past a threshold value, the metabolite produced through this pathway is redirected into the growth reaction. </P><DIV class="caption"><CENTER><I><B>Figure 5.</B> The relationship between light and transketolase activity shows a peak when the growth is unconstrained.</I></CENTER></DIV><H3>Future Directions</H3><P>We can use experimental data to further validate and refine predictive capabilities of our model. A measured growth curve, as well as values for nutrient uptake and secretion can be used as constraints on the flux of their respective reactions. We can also compare the flux value for light units given by the FBA function to that of experimental results. 
</P><P>Dynamic FBA is a technique that solves for the mathematical model at certain snapshots in time instead of at steady-state. It can also factor in concentrations of reactants and kinetic rate constants, which can make the predictive results of the model closer to experimental values<SUP>8</SUP>.
</P><H3>Scripts</H3><P><A href="https://github.com/ucsd-igem-2015/Constraint-Based-Modeling"><FONT color="#828A95&quot;">Link to Constraint-Based Modeling Scripts</FONT></A></P><H3>References</H3><P>
[1] Schellenberger J, Que R, Fleming RMT, Thiele I, Orth JD, Feist AM, Zielinski DC, Bordbar A, Lewis NE, Rahmanian S, Kang J, Hyduke DR, Palsson BØ.

[2] Duarte, Natalie C., Markus J. Herrgård, and Bernhard Ø. Palsson. “Reconstruction and Validation of Saccharomyces Cerevisiae iND750, a Fully Compartmentalized Genome-Scale Metabolic Model.” Genome Research 14.7 (2004): 1298–1309. PMC. Web. 18 Sept. 2015.

[3] Herrgard, M et al. “A consensus yeast metabolic network reconstruction obtained from a community approach to systems biology.” Nat Biotechnol 26 (2008) : 1155-1160.

[4] Kanehisa, M., Goto, S., Sato, Y., Kawashima, M., Furumichi, M., and Tanabe, M.; Data, information, knowledge and principle: back to metabolism in KEGG. Nucleic Acids Res. 42, D199–D205 (2014).

[5] Kanehisa, M. and Goto, S.; KEGG: Kyoto Encyclopedia of Genes and Genomes. Nucleic Acids Res. 28, 27-30 (2000).

[6] Voet, Donald, and Judith G. Voet. Biochemistry. New York: J. Wiley &amp; Sons, 1995. 439. Print.

[7] Orth, Jeffrey D., Ines Thiele, and Bernhard Ø. Palsson. &quot;What is flux balance analysis?.&quot; Nature biotechnology 28.3 (2010): 245-248.

[8] Vincent J. Henry; Anita E. Bandrowski; Anne-Sophie Pepin; Bruno J. Gonzalez; Arnaud Desfeux
Database 2014; doi: 10.1093/database/bau069

[9] Becker, S. et al., &quot;Quantitative prediction of cellular metabolism with constraint-based models: The COBRA Toolbox&quot;, Nat. Protoc 2, 727-738 (2007).

[10] Lewis NE, Nagarajan H, Palsson BO. Constraining the metabolic genotype-phenotype relationship using a phylogeny of in silico methods. Nature Reviews Microbiology. (2012)
</P></DIV></DIV></DIV></DIV></DIV><DIV class="logorow"><DIV class="row"><DIV class="copyright-text"><P><A href="mailto:ucsd.igem.2015@gmail.com"><FONT color="#1b988b">Contact Us: ucsd.igem.2015@gmail.com</FONT></A></P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>