<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NCTU_Formosa_Dry_Lab_Microbiota_Prediciton skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NCTU Formosa/Dry Lab/Microbiota Prediciton</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_menu_wrapper"><DIV class="menu_button" id="display_menu_control">
		MENU
	</DIV><DIV id="menu_content"><A href="https://2017.igem.org/Team:NCTU_Formosa"><DIV class="menu_button direct_to_page">
				HOME
			</DIV></A><DIV class="menu_button"> TEAM
		</DIV><DIV class="submenu_wrapper" id="team_submenu"><A href="https://2017.igem.org/Team:NCTU_Formosa/Team"><DIV class="submenu_button" id="Team_page">
					Team
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Collaborations"><DIV class="submenu_button" id="Collaborations_page">
					Collaborations
				</DIV></A></DIV><DIV class="menu_button"> PROJECT
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:NCTU_Formosa/Description"><DIV class="submenu_button" id="Description_page">
					Description
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Design"><DIV class="submenu_button" id="Design_page">
					Design
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Experiments"><DIV class="submenu_button" id="Experiments_page">
					Experiments
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Notebook"><DIV class="submenu_button" id="Notebook_page">
					Notebook
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/InterLab"><DIV class="submenu_button" id="InterLab_page">
					InterLab
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Contribution"><DIV class="submenu_button" id="Contribution_page">
					Contribution
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Results"><DIV class="submenu_button" id="Results_page">
					Results
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Demonstrate"><DIV class="submenu_button" id="Demonstrate_page">
					Demonstrate
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Improve"><DIV class="submenu_button" id="Improve_page">
					Improve
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Attributions"><DIV class="submenu_button" id="Attributions_page">
					Attributions
				</DIV></A></DIV><DIV class="menu_button"> PARTS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:NCTU_Formosa/Parts"><DIV class="submenu_button" id="Parts_page">
					Parts
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Basic_Part"><DIV class="submenu_button" id="Basic_Part_page">
					Basic Parts
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Composite_Part"><DIV class="submenu_button" id="Composite_Part_page">
					Composite Parts
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Part_Collection"><DIV class="submenu_button" id="Part_Collection_page">
					Part Collection
				</DIV></A></DIV><A href="https://2017.igem.org/Team:NCTU_Formosa/Safety"><DIV class="menu_button direct_to_page">
				SAFETY
			</DIV></A><DIV class="menu_button"> HUMAN PRACTICES
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:NCTU_Formosa/HP/Silver"><DIV class="submenu_button" id="Silver_page">
					Silver HP
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/HP/Gold_Integrated"><DIV class="submenu_button" id="Gold_Integrated_page">
					Integrated and Gold
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Engagement"><DIV class="submenu_button" id="Engagement_page">
					Public Engagement
				</DIV></A></DIV><DIV class="menu_button"> AWARDS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:NCTU_Formosa/Applied_Design"><DIV class="submenu_button" id="Applied_Design_page">
					Applied Design
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Entrepreneurship"><DIV class="submenu_button" id="Entrepreneurship_page">
					Entrepreneurship
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Hardware"><DIV class="submenu_button" id="Hardware_page">
					Hardware
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Measurement"><DIV class="submenu_button" id="Measurement_page">
					Measurement
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Plant"><DIV class="submenu_button" id="Plant_page">
					Plant
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Software"><DIV class="submenu_button" id="Software_page">
					Software
				</DIV></A></DIV><A href="https://igem.org/2017_Judging_Form?team=NCTU_Formosa"><DIV class="menu_button direct_to_page">
				JUDGING FORM
			</DIV></A></DIV></DIV><TITLE>Navigation Bar</TITLE><DIV class="main-container" onscroll="shadow()" id="main-container"><UL class="main-menu" style="list-style-image: none;"><LI class="dropdown"><A href="https://2018.igem.org/Team:NCTU_Formosa" class="dropbtn" id="navigation">Home</A></LI><LI class="dropdown"><A href="#" class="dropbtn"><SPAN class="menu_sign">Project</SPAN></A><DIV class="dropdown-content" style="display: none;"><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Project/Description" id="first">Description</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Applied_Design">Design</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Hardware">IoT System</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Demonstrate">Demonstration</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Entrepreneurship">Entrepreneurship</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/InterLab" id="last">InterLab</A></DIV></DIV></LI><LI class="dropdown"><A href="#" class="dropbtn"><SPAN class="menu_sign">Dry Lab</SPAN></A><DIV class="dropdown-content" style="display: none;"><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Model" id="first">Overview</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Dry_Lab/Microbiota_Prediciton">Microbiota Prediction</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Dry_Lab/Peptide_Prediction">Peptide Prediction</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Dry_Lab/Growth_Model">Growth Model</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Dry_Lab/Productivity_Model">Productivity Model</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Dry_Lab/NGS_Data_Analysis" id="last">NGS Data Analysis</A></DIV></DIV></LI><LI class="dropdown"><A href="#" class="dropbtn"><SPAN class="menu_sign">Wet Lab</SPAN></A><DIV class="dropdown-content" style="display: none;"><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Wet_Lab" id="first">Experiment Design</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Wet_Lab/Expression">Protein Expression</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Wet_Lab/Functional_Analysis">Functional Analysis</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Wet_Lab/Curcumin_Biosensor">Curcumin Bio-sensor</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Wet_Lab/Safety" id="last">Safety</A></DIV></DIV></LI><LI class="dropdown"><A href="#" class="dropbtn"><SPAN class="menu_sign">Parts</SPAN></A><DIV class="dropdown-content" style="display: none;"><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Parts" id="first">Parts</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Basic_Part">Basic Parts</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Composite_Part">Composite Parts</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Part_Collection">Part Collection</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Improve" id="last">Improvement</A></DIV></DIV></LI><LI class="dropdown"><A href="#" class="dropbtn"><SPAN class="menu_sign">Human Practice</SPAN></A><DIV class="dropdown-content" style="display: none;"><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Human_Practices" id="first">Overview</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Public_Engagement">Education &amp;Public Engagement</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Human_Practices/Integrated">Integrated HP</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Collaborations" id="last">Collaboration</A></DIV></DIV></LI><LI class="dropdown"><A href="#" class="dropbtn"><SPAN class="menu_sign">Team</SPAN></A><DIV class="dropdown-content" style="display: none;"><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Team_Member" id="first">Team Introduction</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Attributions" id="last">Attributions</A></DIV></DIV></LI><LI class="dropdown"><A href="#" class="dropbtn"><SPAN class="menu_sign">Notebook</SPAN></A><DIV class="dropdown-content" style="display: none;"><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Notebook" id="first">Lab Note</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Protocol" id="last">Protocol</A></DIV></DIV></LI><LI class="dropdown"><A href="https://2018.igem.org/Team:NCTU_Formosa/Achievements" class="dropbtn_final">Achievements</A></LI></UL></DIV><DIV id="Navigation2"><DIV id="three_bar"><P style="color: white; font-weight: bold; z-index: 100;">☰</P></DIV><DIV class="igem_2018_menu2_wrapper"><DIV class="menu2_content"><A href="https://2018.igem.org/Team:NCTU_Formosa"><DIV class="menu2_button direct_to_page">
                    HOME
                </DIV></A><DIV class="menu2_button"> Project
            </DIV><DIV class="submenu2_wrapper" id="Project_submenu"><A href="https://2018.igem.org/Team:NCTU_Formosa/Project/Description"><DIV class="submenu2_button" id="Description_page">
                        Description
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Applied_Design"><DIV class="submenu2_button" id="Design_page">
                        Design
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Hardware"><DIV class="submenu2_button" id="Peptide_Prediction_page">
                        IoT System
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Demonstrate"><DIV class="submenu2_button" id="Spore_Germinaition_Prediction_page">
                        Demostration
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Entrepreneurship"><DIV class="submenu2_button">
                        Entrepreneurship
                  </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/InterLab"><DIV class="submenu2_button" id="Safety_page">
                        InterLab
                    </DIV></A></DIV><DIV class="menu2_button"> Dry Lab
            </DIV><DIV class="submenu2_wrapper"><A href="https://2018.igem.org/Team:NCTU_Formosa/Model"><DIV class="submenu2_button" id="SCM_Model_page">
                        Overview
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Dry_Lab/Microbiota_Prediciton"><DIV class="submenu2_button" id="Spore_Germination_Model_page">
                        Microbiota Prediction
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Dry_Lab/Peptide_Prediction"><DIV class="submenu2_button" id="Spore_Germination_Model_page">
                        Peptide Prediction
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Dry_Lab/Growth_Model"><DIV class="submenu2_button" id="Spore_Germination_Model_page">
                        Growth Model
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Dry_Lab/Productivity_Model"><DIV class="submenu2_button" id="Spore_Germination_Model_page">
                        Productivity Model
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Dry_Lab/NGS_Data_Analysis"><DIV class="submenu2_button" id="Spore_Germination_Model_page">
                        NGS Data Analysis
                    </DIV></A></DIV><DIV class="menu2_button"> Wet Lab
            </DIV><DIV class="submenu2_wrapper"><A href="https://2018.igem.org/Team:NCTU_Formosa/Wet_Lab"><DIV class="submenu2_button" id="Fungal_Experiment_page">
                        Experiment Design
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Wet_Lab/Expression"><DIV class="submenu2_button" id="Fungal_Experiment_page">
                        Protein Expression
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Wet_Lab/Functional_Analysis"><DIV class="submenu2_button" id="Plant_Experiment_page">
                        Functional Analysis
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Wet_Lab/Curcumin_Biosensor"><DIV class="submenu2_button" id="Protein_Expression_page">                  Curcumin Bio-sensor
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Wet_Lab/Safety"><DIV class="submenu2_button" id="Safety_page">
                        Safety
                    </DIV></A></DIV><DIV class="menu2_button"> Parts
            </DIV><DIV class="submenu2_wrapper"><A href="https://2018.igem.org/Team:NCTU_Formosa/Parts"><DIV class="submenu2_button" id="Basic_Part_page">
                        Parts
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Basic_Part"><DIV class="submenu2_button" id="Basic_Part_page">
                        Basic Parts
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Composite_Part"><DIV class="submenu2_button" id="Composite_Part_page">
                        Composite Parts
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Part_Collection"><DIV class="submenu2_button" id="Composite_Part_page">
                        Part Collection
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Improve"><DIV class="submenu2_button" id="Improvement_page">
                        Improvement
                    </DIV></A></DIV><DIV class="menu2_button"> Human Practice
            </DIV><DIV class="submenu2_wrapper"><A href="https://2018.igem.org/Team:NCTU_Formosa/Human_Practices"><DIV class="submenu2_button" id="Gold_Integrated_page">
                        Overview
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Public_Engagement"><DIV class="submenu2_button" id="Gold_Integrated_page">
                      Education and Public Engagement
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Human_Practices/Integrated"><DIV class="submenu2_button" id="Gold_Integrated_page">
                        Integrated HP
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Collaborations"><DIV class="submenu2_button" id="Collaboration_page">
                        Collaboration
                    </DIV></A></DIV><DIV class="menu2_button"> Team
            </DIV><DIV class="submenu2_wrapper"><A href="https://2018.igem.org/Team:NCTU_Formosa/Team_Member"><DIV class="submenu2_button" id="Team_page">
                        Team Introduction
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Attributions"><DIV class="submenu2_button" id="Appreciation_page">
                        Attributions
                    </DIV></A></DIV><DIV class="menu2_button"> Notebook
            </DIV><DIV class="submenu2_wrapper"><A href="https://2018.igem.org/Team:NCTU_Formosa/Notebook"><DIV class="submenu2_button" id="Lab_Notes_page">
                        Lab Note
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Protocol"><DIV class="submenu2_button" id="Protocol_page">
                        Protocol
                    </DIV></A></DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Achievements"><DIV class="menu2_button direct_to_page">
                    Achievements
                </DIV></A></DIV></DIV></DIV><TITLE>Microbiota Prediction</TITLE><DIV class="wrapper"><DIV class="sec1"><DIV class="text"><P>     Artificial intelligence and machine learning allow us to realize the seemingly insurmountable goal of predicting the fluctuations of entire microbiotas due to the specific effects of bio-stimulators. While traditional ecologists may find it too difficult to consider every unique microbial relationship in an ecosystem, machine learning programs use numerical analysis to not only quickly determine these associations but also use them to predict overall population shifts caused by stimuli. For our modelling purposes we choose Weka, a software with strong classification capabilities, to establish accurate connections between every genera of bacteria in our soil.
        </P></DIV><DIV class="title_1">Considering General Factors</DIV><DIV class="text"><P>     To begin modelling the relationship between bio-stimulators and microbiota, we first determine the most important factors that affect bacterial growth in soil. Three conditions immediately came to mind: temperature, pH and salinity, whose effects are modelled through the respective equations below:
        </P></DIV><DIV class="equation">
        $$Ratkowsky\ Equation:\ R_{temp}(T)=a\cdot[(T-T_{min})\cdot(1-e^{(b\cdot(T-T_{max}))})]^2$$
        $$Cardinal\ pH\ Equation:\ R_{pH}(pH)=\frac{c\cdot(pH-pH_{min})\cdot(pH-pH_{max})}{d\cdot((pH-pH_{min})\cdot(pH-pH_{max})-e\cdot(pH-pH_{opt})^2)}$$
        $$Salinity\ Equation:\ R_{sal}(sal)=(f\cdot sal^2)+(g\cdot sal)+h$$
      </DIV><DIV class="text"><P>     These factors heavily influence bacterial fluctuation in any environment and are especially important in determining soil microbiota, according to soil expert, Professor Young from National Chung Hsing University. Professor Young also suggested that we should consider the relationship between nitrogen, phosphorus and potassium and soil bacteria, because farm soil is regularly applied with fertilizers containing these vital macronutrients. To take these elements into account, we collected literature discussing their impact on bacterial levels and found the following functions:
        </P></DIV><DIV class="equations"><DIV class="equation"><P>$$N=A*(Arthrobacter citreus)+B*(Arthrobacter globiformis)+C*(Arthrobacter terregens)+D*(Bacillus brevis)+E*(Bacillus cereus)+F*(Bacillus cereusvar. Mycoides)+G*(Bacillus circulans)+H*(Bacillus firmus)+I*(Bacillus globisporus)+.....$$</P></DIV><DIV class="equation"><P>$$P=Gammaproteobacteria (56 isolates)+Firmicutes (28 isolates)+Actinobacteria(8isolates)+
Alphaproteobacteria (2 isolates).$$
</P></DIV><DIV class="equation"><P>$$K=A*(Bacillus sp)+ B*(Pseudomonas sp)+ C*(Sinorhizobium metallidans)$$</P></DIV></DIV><DIV class="text"><P>     These equations model the direct relationship between levels of the elements and levels of bacteria in soil – specifically, the levels of bacteria that metabolize said elements.
        </P></DIV><DIV class="text"><P>     Combining these general equations together gives a method of obtaining a rough estimation of how our microbiota will change, based on fluctuation of these factors; our universal factors temperature, pH and salinity assist in modeling general fluctuations in the microbiota, while the more specific factors nitrogen, phosphorus and potassium are quite helpful in predicting how amount of nutrient metabolizing bacteria oscillates when dealing with the effects of fertilizers. But how do we deduce the change in level of bacteria that are unaffected by said nutrients?  We turn to our NGS analysis to find missing link.
        </P></DIV><DIV class="text"><P>     From our NGS report we calculate the Spearman correlation value of each pair of genera in our soil. This coefficient, assigned a value between -1 and +1, describes the degree of correlation between each pair, with values closer to -1 representing stronger negative correlation and values closer to +1 representing stronger positive correlation. Correlation values between the 20 most abundant bacterial genera in our soil samples are shown in the following heat map.
        </P></DIV><DIV class="explanation">
        Figure 1: Correlation heat map of top-20 bacteria in June
      </DIV><DIV class="title_1">Weka</DIV><DIV class="text"><P>     Once we have our 6 general equations and our correlation values we’re ready to begin using Weka to construct a prediction model. Weka is split into two parts: regression analysis to filter out the non-correlated pairs of bacteria, and cross validation to determine the weighting each bacterial relationship has under different conditions.
      </P></DIV><DIV class="title_1">Regression Analysis</DIV><DIV class="text"><P>     We first take advantage of the machine learning software’s classification ability, using the built-in regression analysis module to determine which pairs of bacteria are heavily affected by correlation. To do this we define coefficient values below -0.7 to be truly negatively correlated and coefficient values above +0.7 to be truly positively correlated; pairs assigned a value in between are ignored. Weka then separates truly correlated pairs from the rest; these are the bacteria that will change as an indirect effect of bio-stimulator application. We start with one genus of bacteria and assess the correlation coefficient it has with each other genus in soil. Any pairs with significant correlation are collected into a fold belonging to that bacteria. Once all pairs are assessed, the resulting fold should contain all the bacteria that are correlated with our starting genus.
        </P></DIV><DIV class="explanation">
        Figure 2: Weka determines correlation between bacteria in soil.
      </DIV><DIV class="text"><P>     For every pair in any particular fold, Weka plots that pair’s data on a graph to find a curve of regression to describe their relationship. For example:
        </P></DIV><DIV class="explanation">
        Figure 3: Spearman's correlation example between <I>Mesorhizobium</I> and <I>Xylanimicrobium</I>.
      </DIV><DIV class="title_1">Cross Validation</DIV><DIV class="text"><P>     The resulting curve is the theoretical relationship between the two bacteria; however, the wide range of soil conditions that vary between different samples may alter the relationship. To account for this, Weka assigns weights to each correlation regression curve by performing cross validation, in this case with three folds. The steps are as follows:</P></DIV><DIV class="explanation">
        Figure 4: Three fold cross validation.
      </DIV><DIV class="text" style="margin-top: 1vw; margin-bottom: 1vw;"><P>(1) Three folds of three different genera of bacteria are compared in pairs to determine the accuracy of each pair’s correlational relationship.</P></DIV><DIV class="text" style="margin-top: 1vw; margin-bottom: 1vw;"><P>(2) If they exhibit a relationship in line with Weka’s initial assessment, nothing changes and the pair keeps its assigned weight.</P></DIV><DIV class="text" style="margin-top: 1vw; margin-bottom: 1vw;"><P>(3) If they show unexpected associations, they are said to exhibit paradox. Paradox alerts Weka to the discrepancy between prediction and reality, causing it to adjust waiting accordingly.</P></DIV><DIV class="text" style="margin-top: 1vw; margin-bottom: 1vw;"><P>(4) Through this cross validation Weka calibrates weighting of each pair and can predict how an entire microbiota is related after analyzing all folds. The result can be expressed in a pie chart describing predicted microbial ratios</P></DIV><DIV class="explanation">
        Figure 5: Resulting correlational values used to predict microbiota ratios.
      </DIV><DIV class="title_1">Artificial Intelligence</DIV><DIV class="text"><P>     Once our initial model is complete we can begin to make rough predictions about microbiota changes based on a volume of bio-stimulator. The basic rules we established regarding different soil conditions point us in the right direction in terms of bacteria shifts, but to achieve true precise control over soil we must improve our prediction accuracy through artificial intelligence. Artificial intelligence feeds actual data back into our system; more data allows for more calibration and more cross validations, adapting our predictions to the specific nature of our soil sample and improving the accuracy of subsequent predictions.</P></DIV><DIV class="title_1">Model Learning</DIV><DIV class="text"><P>     We began by generating our model using NGS data from April through June. We entered a volume of bio-stimulator as well NGS data from before and after application, thus generating an initial prediction model. The accuracy of our model with only one month of data was approximately 21%, while inclusion of a second month’s data increased accuracy to 51% - at this point if we were to predict results for June, we would get about 51% of the total microbiota correct. Again, we applied bio-stimulator to our soil and waited for our data. Using June data to calibrate our model increased the prediction accuracy of our model by over another 25%, resulting in a microbiota prediction model with 78% accuracy. </P></DIV><DIV class="explanation">
        Figure 6: Pie charts of microbiota.<P>This picture shows the microbiota of July(reality), 1<SUP>st</SUP> prediction, and 2<SUP>nd</SUP> prediction.
          Obviously, the predictive results become more accurate with more training data.</P></DIV><DIV class="explanation">
        Figure 7: The learing curve of accuracy.<P>The accuracy significantly clime up with the increasing amount of dataset fed to our model.</P></DIV><DIV class="title_1">Result</DIV><DIV class="table"><TABLE><CAPTION><P class="explanation">
            Table 1: The comparison of top-20 bacteria between real July data and predictive data.
          </P></CAPTION><THEAD><TR><TH><P>Taxon (Genus)</P></TH><TH style="text-align: center;"><P>July</P></TH><TH style="text-align: center;"><P>Prediction</P></TH><TH style="text-align: center;"><P>Error</P></TH></TR></THEAD><TBODY><TR><TD style="white-space: nowrap;"><P><I>Nitrospira</I></P></TD><TD style="text-align: center;"><P>18.67%</P></TD><TD style="text-align: center;"><P>16.93%</P></TD><TD style="text-align: center;"><P>-1.74%</P></TD></TR><TR><TD style="white-space: nowrap;"><P><I>Flavobacterium</I></P></TD><TD style="text-align: center;"><P>11.94%</P></TD><TD style="text-align: center;"><P>10.18%</P></TD><TD style="text-align: center;"><P>-1.76%</P></TD></TR><TR><TD style="white-space: nowrap;"><P><I>Kaistobacter</I></P></TD><TD style="text-align: center;"><P>11.82%</P></TD><TD style="text-align: center;"><P>14.70%</P></TD><TD style="text-align: center;"><P>2.88%</P></TD></TR><TR><TD style="white-space: nowrap;"><P><I>Pseudomonas</I></P></TD><TD style="text-align: center;"><P>5.18%</P></TD><TD style="text-align: center;"><P>0.76%</P></TD><TD style="text-align: center;"><P>-4.42%</P></TD></TR><TR><TD style="white-space: nowrap;"><P><I>Candidatus Koribacter</I></P></TD><TD style="text-align: center;"><P>4.55%</P></TD><TD style="text-align: center;"><P>11.64%</P></TD><TD style="text-align: center;"><P>7.09%</P></TD></TR><TR><TD style="white-space: nowrap;"><P><I>Rhodoplanes</I></P></TD><TD style="text-align: center;"><P>4.40%</P></TD><TD style="text-align: center;"><P>9.84%</P></TD><TD style="text-align: center;"><P>5.44%</P></TD></TR><TR><TD style="white-space: nowrap;"><P><I>Candidatus Solibacter</I></P></TD><TD style="text-align: center;"><P>3.77%</P></TD><TD style="text-align: center;"><P>7.64%</P></TD><TD style="text-align: center;"><P>3.87%</P></TD></TR><TR><TD style="white-space: nowrap;"><P><I>Steroidobacter</I></P></TD><TD style="text-align: center;"><P>3.17%</P></TD><TD style="text-align: center;"><P>3.16%</P></TD><TD style="text-align: center;"><P>-0.01%</P></TD></TR><TR><TD style="white-space: nowrap;"><P><I>Flavisolibacter</I></P></TD><TD style="text-align: center;"><P>2.82%</P></TD><TD style="text-align: center;"><P>3.53%</P></TD><TD style="text-align: center;"><P>0.71%</P></TD></TR><TR><TD style="white-space: nowrap;"><P><I>Bdellovibrio</I></P></TD><TD style="text-align: center;"><P>2.15%</P></TD><TD style="text-align: center;"><P>1.77%</P></TD><TD style="text-align: center;"><P>-0.38%</P></TD></TR><TR><TD style="white-space: nowrap;"><P><I>Cellvibrio</I></P></TD><TD style="text-align: center;"><P>1.92%</P></TD><TD style="text-align: center;"><P>5.17%</P></TD><TD style="text-align: center;"><P>3.25%</P></TD></TR><TR><TD style="white-space: nowrap;"><P><I>Opitutus</I></P></TD><TD style="text-align: center;"><P>1.01%</P></TD><TD style="text-align: center;"><P>0.50%</P></TD><TD style="text-align: center;"><P>-0.51%</P></TD></TR><TR><TD style="white-space: nowrap;"><P><I>Hyphomicrobium</I></P></TD><TD style="text-align: center;"><P>0.99%</P></TD><TD style="text-align: center;"><P>0.96%</P></TD><TD style="text-align: center;"><P>-0.03%</P></TD></TR><TR><TD style="white-space: nowrap;"><P><I>Perlucidibaca</I></P></TD><TD style="text-align: center;"><P>0.95%</P></TD><TD style="text-align: center;"><P>5.18%</P></TD><TD style="text-align: center;"><P>4.23%</P></TD></TR><TR><TD style="white-space: nowrap;"><P><I>Phenylobacterium</I></P></TD><TD style="text-align: center;"><P>0.87%</P></TD><TD style="text-align: center;"><P>0.08%</P></TD><TD style="text-align: center;"><P>-0.79%</P></TD></TR><TR><TD style="white-space: nowrap;"><P><I>Bacillus</I></P></TD><TD style="text-align: center;"><P>0.85%</P></TD><TD style="text-align: center;"><P>0.92%</P></TD><TD style="text-align: center;"><P>0.07%</P></TD></TR><TR><TD style="white-space: nowrap;"><P><I>Fluviicola</I></P></TD><TD style="text-align: center;"><P>0.80%</P></TD><TD style="text-align: center;"><P>0.00%</P></TD><TD style="text-align: center;"><P>-0.80</P></TD></TR><TR><TD style="white-space: nowrap;"><P><I>Plesiocystis</I></P></TD><TD style="text-align: center;"><P>0.79%</P></TD><TD style="text-align: center;"><P>0.24%</P></TD><TD style="text-align: center;"><P>-0.55%</P></TD></TR><TR><TD style="white-space: nowrap;"><P><I>Lutibacterium</I></P></TD><TD style="text-align: center;"><P>0.75%</P></TD><TD style="text-align: center;"><P>0.00%</P></TD><TD style="text-align: center;"><P>-0.75%</P></TD></TR><TR><TD style="white-space: nowrap;"><P><I>Dechloromonas</I></P></TD><TD style="text-align: center;"><P>0.74%</P></TD><TD style="text-align: center;"><P>0.41%</P></TD><TD style="text-align: center;"><P>-0.33%</P></TD></TR><TR><TD style="white-space: nowrap;"><P><I>Devosia</I></P></TD><TD style="text-align: center;"><P>0.72%</P></TD><TD style="text-align: center;"><P>0.56%</P></TD><TD style="text-align: center;"><P>-0.16%</P></TD></TR><TR><TD style="white-space: nowrap;"><P><I>Sphingomonas</I></P></TD><TD style="text-align: center;"><P>0.64%</P></TD><TD style="text-align: center;"><P>0.75%</P></TD><TD style="text-align: center;"><P>0.11%</P></TD></TR></TBODY></TABLE></DIV><DIV class="text"><P>     According to our previous assumption: bacteria with a bacteria ratio of less than 1% will not affect soil conditions. We can infer that the bacteria with less than 1% change in actual and predicted data are still within acceptable limits. Then we predicted that the results were mainly different from the actual results of <I>Candidatus Koribacter, Rhodoplanes, Perlucidibaca, Candidatus Solibacter, Cellvibrio, Kaistobacter, Nitrospira, Flavobacterium</I>, and <I>Pseudomonas</I>. We analyzed these nine bacteria and soil functions and found that the soil nitrate and pollutant content are higher than the actual value, which may be related to the part of the soil loss that we did not consider. Of course, the error of this part will decrease as the number of data increases. After the microanalysis, we will then evaluate the overall composition and the fungus according to the functional classification.</P></DIV><DIV class="explanation"><P>
        Figure 8: The ratio amount of bacteria with different functions within different data.(PGPB stands for plant growth-promoting bacteria. NB stands for nitrifying bacteria.)</P></DIV><P class="explanation" style="margin-top: 2vw;">
        Table 2: Eveness of different samples.
      </P><DIV class="text"><P>     After sampling the NGS data of these three months, we classified the top 20 bacteria in those data to get the function table in Fig 8. We can see that the ratio of plant growth-promoting bacteria (PGPB) is rising, and the ratio of plant pathogenic bacteria is fluctuating in a small amount, which means that we can achieve the purpose of raising good bacteria in the case of controlling the proportion of plant pathogenic bacteria by using the biostimulator. Nitrifying bacteria and phosphate solubilizing bacteria rise. These bacteria produce nutrients plants need, so their increament can reduce the amount of fertilizer applied. The decline in pollution indicator bacteria indicates that the proportion of pollutants in the soil components has decreased. That is, we have not applied pollutants such as pesticides, herbicides, etc., and the soil is slowly repairing by itself over time. Finally, the data shows that the denitrifying bacteria decline. Denitrifying bacteria can represent the nitrate content, and the high nitrate content will cause environmental pollution. All information shows us that every month after spreading the biostimulator, the soil gets healthier. It is worth that the evenness of this three months is improving. Therefore, we confirmed that biostimulator can control bacterial changes and make the soil healthier, and evenness can be used as a criterion for segmental soil health.
        </P></DIV><DIV class="title_1">Conclusion</DIV><DIV class="text"><P>     Increasingly accurate prediction of microbial shifts due to bio-stimulators is a vital element of our smart farming system. Our goal is to regulate soil microbiota precisely, and we need accurate models to do so. Luckily, machine learning and artificial intelligence can provide just that. A general model formed using established relationships between key environmental factors and bacterial growth is supported by correlation values calculated from NGS data to allow for rough initial predictions of microbial shifts. Raw data obtained after subsequent applications of bio-stimulators is reintroduced into our models through a feedback system, calibrating the weightings of each bacteria correlational relationship to improve accuracy with each cycle. With increasingly precise regulation we can manipulate soil microbiota to produce any desired effect. Visit our real farm <A href="https://2018.igem.org/Team:NCTU_Formosa/Demonstrate">demonstration</A> to find out how we use artificial intelligence to increase curcumin concentration in turmeric while maintaining soil health.     If you are interesting in our model, please click the icon to find out how them work in GitHub!
        </P></DIV><DIV class="title_1">Reference</DIV><DIV class="text"><P>
        1. Bouckaert, R. R., et al. (2013). &quot;WEKA Manual for Version 3-7-8, 2013.&quot;  21.
        2. WI, H., et al. (2011). &quot;Practical machine learning tools and techniques.&quot;
        3. Barabasz, W. and J. J. P. J. o. E. S. Lipiec (2002). &quot;Biological effects of mineral nitrogen fertilization on soil microorganisms.&quot;  11(3): 193-198.
        4. KUMAR, A. and L. C. J. P. RAI (2017). &quot;Soil Organic Carbon and Availability of Soil Phosphorus Regulate Abundance of Culturable Phosphate Solubilizing Bacteria in Paddy Fields of the Indo-Gangetic Plain.&quot;
        5. Lambert, R. J. J. J. o. a. m. (2011). &quot;A new model for the effect of pH on microbial growth: An extension of the Gamma hypothesis.&quot;  110(1): 61-68.
        6. Nihala Jabin, P. (2017). Screening of potash solubilizing bacteria for plant growth promotional activity and nutrient uptake of brinjal, Vasantrao Naik Marathwada Krishi Vidyapeeth, Parbhani.
        7. Ratkowsky, D. A., et al. (1983). &quot;Model for bacterial culture growth rate throughout the entire biokinetic temperature range.&quot; J Bacteriol 154(3): 1222-1226.
        8. Rousk, J., et al. (2011). &quot;Bacterial salt tolerance is unrelated to soil salinity across an arid agroecosystem salinity gradient.&quot;  43(9): 1881-1887.
        9. Wikipedia contributors. (2018, October 12). Bacillus. In Wikipedia, The Free Encyclopedia. Retrieved 18:17, October 16, 2018, from https://en.wikipedia.org/w/index.php?title=Bacillus&amp;oldid=863696818
        10. Wikipedia contributors. (2018, March 23). Geobacter. In Wikipedia, The Free Encyclopedia. Retrieved 18:34, October 16, 2018, from https://en.wikipedia.org/w/index.php?title=Geobacter&amp;oldid=831990733
        11. Espenberg, M., et al. (2018). &quot;Differences in microbial community structure and nitrogen cycling in natural and drained tropical peatland soils.&quot; Scientific Reports 8(1): 4742.
        12. Hou, J., et al. (2015). &quot;PGPR enhanced phytoremediation of petroleum contaminated soil and rhizosphere microbial community response.&quot; Chemosphere 138: 592-598.
        13. Hruska, K., Vyzkumny Ustav Veterinarniho Lekarstvi, Brno (Czech Republic) and M. Kaevska, Vyzkumny Ustav Veterinarniho Lekarstvi, Brno (Czech Republic) (dec2012). &quot;Mycobacteria in water, soil, plants and air: a review.&quot;  v. 57.
        14. Jiao, S., et al. (2016). &quot;Microbial succession in response to pollutants in batch-enrichment culture.&quot; Scientific Reports 6: 21791.
        15. Leys, N. M. E. J., et al. (2004). &quot;Occurrence and Phylogenetic Diversity of Sphingomonas Strains in Soils Contaminated with Polycyclic Aromatic Hydrocarbons.&quot; Applied and Environmental Microbiology 70(4): 1944-1955.
        16. Ma, M., et al. (2018). &quot;Effect of long-term fertilization strategies on bacterial community composition in a 35-year field experiment of Chinese Mollisols.&quot; AMB Express 8(1): 20.
        17. Martineau, C., et al. (2015). &quot;Comparative analysis of denitrifying activity in Hyphomicrobium nitrativorans, Hyphomicrobium denitrificans and Hyphomicrobium zavarzinii.&quot; AEM. 00848-00815.
        18. Rodgers-Vieira, E. A., et al. (2015). &quot;Identification of Anthraquinone-Degrading Bacteria in Soil Contaminated with Polycyclic Aromatic Hydrocarbons.&quot; Applied and Environmental Microbiology.
        19. Sangwan, P., et al. (2005). &quot;Detection and cultivation of soil verrucomicrobia.&quot; Appl Environ Microbiol 71(12): 8402-8410.
        20. Sorensen, J. and O. Nybroe (2004). Pseudomonas in the Soil Environment. Pseudomonas: Volume 1 Genomics, Life Style and Molecular Architecture. J.-L. Ramos. Boston, MA, Springer US: 369-401.
        21. Umadevi, P., et al. (2018). &quot;Trichoderma harzianum MTCC 5179 impacts the population and functional dynamics of microbial community in the rhizosphere of black pepper (Piper nigrum L.).&quot; Brazilian Journal of Microbiology 49(3): 463-470.
        22. van Dijl, J. M. and M. Hecker (2013). &quot;Bacillus subtilis: from soil bacterium to super-secreting cell factory.&quot; Microb Cell Fact 12: 3.
        23. Wang, R., et al. (2017). &quot;Microbial community composition is related to soil biological and chemical properties and bacterial wilt outbreak.&quot; Scientific Reports 7(1): 343.
        24. Winston, M. E., et al. (2014). &quot;Understanding Cultivar-Specificity and Soil Determinants of the Cannabis Microbiome.&quot; PLOS ONE 9(6): e99641.
        25. Yan, G., et al. (2017). &quot;Effects of different nitrogen additions on soil microbial communities in different seasons in a boreal forest.&quot;  8(7): e01879.</P></DIV></DIV></DIV><TITLE>Template</TITLE></DIV></DIV></DIV></DIV></DIV></BODY></HTML>