<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NCTU_Formosa_Dry_Lab_NGS_Data_Analysis skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NCTU Formosa/Dry Lab/NGS Data Analysis</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_menu_wrapper"><DIV class="menu_button" id="display_menu_control">
		MENU
	</DIV><DIV id="menu_content"><A href="https://2017.igem.org/Team:NCTU_Formosa"><DIV class="menu_button direct_to_page">
				HOME
			</DIV></A><DIV class="menu_button"> TEAM
		</DIV><DIV class="submenu_wrapper" id="team_submenu"><A href="https://2017.igem.org/Team:NCTU_Formosa/Team"><DIV class="submenu_button" id="Team_page">
					Team
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Collaborations"><DIV class="submenu_button" id="Collaborations_page">
					Collaborations
				</DIV></A></DIV><DIV class="menu_button"> PROJECT
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:NCTU_Formosa/Description"><DIV class="submenu_button" id="Description_page">
					Description
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Design"><DIV class="submenu_button" id="Design_page">
					Design
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Experiments"><DIV class="submenu_button" id="Experiments_page">
					Experiments
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Notebook"><DIV class="submenu_button" id="Notebook_page">
					Notebook
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/InterLab"><DIV class="submenu_button" id="InterLab_page">
					InterLab
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Contribution"><DIV class="submenu_button" id="Contribution_page">
					Contribution
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Results"><DIV class="submenu_button" id="Results_page">
					Results
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Demonstrate"><DIV class="submenu_button" id="Demonstrate_page">
					Demonstrate
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Improve"><DIV class="submenu_button" id="Improve_page">
					Improve
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Attributions"><DIV class="submenu_button" id="Attributions_page">
					Attributions
				</DIV></A></DIV><DIV class="menu_button"> PARTS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:NCTU_Formosa/Parts"><DIV class="submenu_button" id="Parts_page">
					Parts
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Basic_Part"><DIV class="submenu_button" id="Basic_Part_page">
					Basic Parts
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Composite_Part"><DIV class="submenu_button" id="Composite_Part_page">
					Composite Parts
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Part_Collection"><DIV class="submenu_button" id="Part_Collection_page">
					Part Collection
				</DIV></A></DIV><A href="https://2017.igem.org/Team:NCTU_Formosa/Safety"><DIV class="menu_button direct_to_page">
				SAFETY
			</DIV></A><DIV class="menu_button"> HUMAN PRACTICES
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:NCTU_Formosa/HP/Silver"><DIV class="submenu_button" id="Silver_page">
					Silver HP
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/HP/Gold_Integrated"><DIV class="submenu_button" id="Gold_Integrated_page">
					Integrated and Gold
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Engagement"><DIV class="submenu_button" id="Engagement_page">
					Public Engagement
				</DIV></A></DIV><DIV class="menu_button"> AWARDS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:NCTU_Formosa/Applied_Design"><DIV class="submenu_button" id="Applied_Design_page">
					Applied Design
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Entrepreneurship"><DIV class="submenu_button" id="Entrepreneurship_page">
					Entrepreneurship
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Hardware"><DIV class="submenu_button" id="Hardware_page">
					Hardware
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Measurement"><DIV class="submenu_button" id="Measurement_page">
					Measurement
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Plant"><DIV class="submenu_button" id="Plant_page">
					Plant
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Software"><DIV class="submenu_button" id="Software_page">
					Software
				</DIV></A></DIV><A href="https://igem.org/2017_Judging_Form?team=NCTU_Formosa"><DIV class="menu_button direct_to_page">
				JUDGING FORM
			</DIV></A></DIV></DIV><TITLE>Navigation Bar</TITLE><DIV class="main-container" onscroll="shadow()" id="main-container"><UL class="main-menu" style="list-style-image: none;"><LI class="dropdown"><A href="https://2018.igem.org/Team:NCTU_Formosa" class="dropbtn" id="navigation">Home</A></LI><LI class="dropdown"><A href="#" class="dropbtn"><SPAN class="menu_sign">Project</SPAN></A><DIV class="dropdown-content" style="display: none;"><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Project/Description" id="first">Description</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Applied_Design">Design</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Hardware">IoT System</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Demonstrate">Demonstration</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Entrepreneurship">Entrepreneurship</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/InterLab" id="last">InterLab</A></DIV></DIV></LI><LI class="dropdown"><A href="#" class="dropbtn"><SPAN class="menu_sign">Dry Lab</SPAN></A><DIV class="dropdown-content" style="display: none;"><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Model" id="first">Overview</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Dry_Lab/Microbiota_Prediciton">Microbiota Prediction</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Dry_Lab/Peptide_Prediction">Peptide Prediction</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Dry_Lab/Growth_Model">Growth Model</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Dry_Lab/Productivity_Model">Productivity Model</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Dry_Lab/NGS_Data_Analysis" id="last">NGS Data Analysis</A></DIV></DIV></LI><LI class="dropdown"><A href="#" class="dropbtn"><SPAN class="menu_sign">Wet Lab</SPAN></A><DIV class="dropdown-content" style="display: none;"><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Wet_Lab" id="first">Experiment Design</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Wet_Lab/Expression">Protein Expression</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Wet_Lab/Functional_Analysis">Functional Analysis</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Wet_Lab/Curcumin_Biosensor">Curcumin Bio-sensor</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Wet_Lab/Safety" id="last">Safety</A></DIV></DIV></LI><LI class="dropdown"><A href="#" class="dropbtn"><SPAN class="menu_sign">Parts</SPAN></A><DIV class="dropdown-content" style="display: none;"><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Parts" id="first">Parts</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Basic_Part">Basic Parts</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Composite_Part">Composite Parts</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Part_Collection">Part Collection</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Improve" id="last">Improvement</A></DIV></DIV></LI><LI class="dropdown"><A href="#" class="dropbtn"><SPAN class="menu_sign">Human Practice</SPAN></A><DIV class="dropdown-content" style="display: none;"><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Human_Practices" id="first">Overview</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Public_Engagement">Education &amp;Public Engagement</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Human_Practices/Integrated">Integrated HP</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Collaborations" id="last">Collaboration</A></DIV></DIV></LI><LI class="dropdown"><A href="#" class="dropbtn"><SPAN class="menu_sign">Team</SPAN></A><DIV class="dropdown-content" style="display: none;"><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Team_Member" id="first">Team Introduction</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Attributions" id="last">Attributions</A></DIV></DIV></LI><LI class="dropdown"><A href="#" class="dropbtn"><SPAN class="menu_sign">Notebook</SPAN></A><DIV class="dropdown-content" style="display: none;"><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Notebook" id="first">Lab Note</A></DIV><DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Protocol" id="last">Protocol</A></DIV></DIV></LI><LI class="dropdown"><A href="https://2018.igem.org/Team:NCTU_Formosa/Achievements" class="dropbtn_final">Achievements</A></LI></UL></DIV><DIV id="Navigation2"><DIV id="three_bar"><P style="color: white; font-weight: bold; z-index: 100;">â˜°</P></DIV><DIV class="igem_2018_menu2_wrapper"><DIV class="menu2_content"><A href="https://2018.igem.org/Team:NCTU_Formosa"><DIV class="menu2_button direct_to_page">
                    HOME
                </DIV></A><DIV class="menu2_button"> Project
            </DIV><DIV class="submenu2_wrapper" id="Project_submenu"><A href="https://2018.igem.org/Team:NCTU_Formosa/Project/Description"><DIV class="submenu2_button" id="Description_page">
                        Description
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Applied_Design"><DIV class="submenu2_button" id="Design_page">
                        Design
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Hardware"><DIV class="submenu2_button" id="Peptide_Prediction_page">
                        IoT System
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Demonstrate"><DIV class="submenu2_button" id="Spore_Germinaition_Prediction_page">
                        Demostration
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Entrepreneurship"><DIV class="submenu2_button">
                        Entrepreneurship
                  </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/InterLab"><DIV class="submenu2_button" id="Safety_page">
                        InterLab
                    </DIV></A></DIV><DIV class="menu2_button"> Dry Lab
            </DIV><DIV class="submenu2_wrapper"><A href="https://2018.igem.org/Team:NCTU_Formosa/Model"><DIV class="submenu2_button" id="SCM_Model_page">
                        Overview
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Dry_Lab/Microbiota_Prediciton"><DIV class="submenu2_button" id="Spore_Germination_Model_page">
                        Microbiota Prediction
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Dry_Lab/Peptide_Prediction"><DIV class="submenu2_button" id="Spore_Germination_Model_page">
                        Peptide Prediction
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Dry_Lab/Growth_Model"><DIV class="submenu2_button" id="Spore_Germination_Model_page">
                        Growth Model
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Dry_Lab/Productivity_Model"><DIV class="submenu2_button" id="Spore_Germination_Model_page">
                        Productivity Model
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Dry_Lab/NGS_Data_Analysis"><DIV class="submenu2_button" id="Spore_Germination_Model_page">
                        NGS Data Analysis
                    </DIV></A></DIV><DIV class="menu2_button"> Wet Lab
            </DIV><DIV class="submenu2_wrapper"><A href="https://2018.igem.org/Team:NCTU_Formosa/Wet_Lab"><DIV class="submenu2_button" id="Fungal_Experiment_page">
                        Experiment Design
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Wet_Lab/Expression"><DIV class="submenu2_button" id="Fungal_Experiment_page">
                        Protein Expression
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Wet_Lab/Functional_Analysis"><DIV class="submenu2_button" id="Plant_Experiment_page">
                        Functional Analysis
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Wet_Lab/Curcumin_Biosensor"><DIV class="submenu2_button" id="Protein_Expression_page">
                        Curcumin Bio-sensor
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Wet_Lab/Safety"><DIV class="submenu2_button" id="Safety_page">
                        Safety
                    </DIV></A></DIV><DIV class="menu2_button"> Parts
            </DIV><DIV class="submenu2_wrapper"><A href="https://2018.igem.org/Team:NCTU_Formosa/Parts"><DIV class="submenu2_button" id="Basic_Part_page">
                        Parts
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Basic_Part"><DIV class="submenu2_button" id="Basic_Part_page">
                        Basic Parts
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Composite_Part"><DIV class="submenu2_button" id="Composite_Part_page">
                        Composite Parts
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Part_Collection"><DIV class="submenu2_button" id="Composite_Part_page">
                        Part Collection
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Improve"><DIV class="submenu2_button" id="Improvement_page">
                        Improvement
                    </DIV></A></DIV><DIV class="menu2_button"> Human Practice
            </DIV><DIV class="submenu2_wrapper"><A href="https://2018.igem.org/Team:NCTU_Formosa/Human_Practices"><DIV class="submenu2_button" id="Gold_Integrated_page">
                        Overview
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Public_Engagement"><DIV class="submenu2_button" id="Gold_Integrated_page">
                      Education and Public Engagement
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Human_Practices/Integrated"><DIV class="submenu2_button" id="Gold_Integrated_page">
                        Integrated HP
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Collaborations"><DIV class="submenu2_button" id="Collaboration_page">
                        Collaboration
                    </DIV></A></DIV><DIV class="menu2_button"> Team
            </DIV><DIV class="submenu2_wrapper"><A href="https://2018.igem.org/Team:NCTU_Formosa/Team_Member"><DIV class="submenu2_button" id="Team_page">
                        Team Introduction
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Attributions"><DIV class="submenu2_button" id="Appreciation_page">
                        Attributions
                    </DIV></A></DIV><DIV class="menu2_button"> Notebook
            </DIV><DIV class="submenu2_wrapper"><A href="https://2018.igem.org/Team:NCTU_Formosa/Notebook"><DIV class="submenu2_button" id="Lab_Notes_page">
                        Lab Note
                    </DIV></A><A href="https://2018.igem.org/Team:NCTU_Formosa/Protocol"><DIV class="submenu2_button" id="Protocol_page">
                        Protocol
                    </DIV></A></DIV><A href="https://2018.igem.org/Team:NCTU_Formosa/Achievements"><DIV class="menu2_button direct_to_page">
                    Achievements
                </DIV></A></DIV></DIV></DIV><TITLE>Correlation Model</TITLE><DIV class="wrapper"><DIV class="sec1"><DIV class="text" style="width: 75%;"><P>Â Â Â Â Â To train our model more effectively, we use NGS(Next Generation Sequencing) 16S to analyze the microbiota in the soil. We spray biostimulators into the soil to affect the entire microbiota and use NGS 16S regularly. By analyzing multiple NGS data, we can determine the nature of the relationships about bacteria and take advantage of these characteristics to add biostimulators again. Through this process training our model, we adjust to our system again and again, allowing our system predicts the changes of microbiota in soil accurately.
        </P></DIV><DIV class="title_1"><P>What is NGS 16S?</P></DIV><DIV class="text"><P>Â Â Â Â Â Next, we will interpret NGS and 16S rRNA separately.
        </P></DIV><DIV class="title_3" style="margin-left: 13%; margin-bottom: 1.5vw;"><P>Next Generation Sequencing</P></DIV><DIV class="text"><P>Â Â Â NGS(Next Generation Sequencing) is a kind of technique to sequence a number of genomes in very short time. There are three main platforms on the market  currently: Solexa from Illumina, SOLiD from ABI, and 454 from Roche. The procedures of these three sequencers are different, but they are all based on chain termination.
          Take Solixa for example, following these steps below:
          (1)	Use ultrasound to break original DNA sequences into fragments of about 200-500 base pairs, and then attach the adapters to both ends of the fragments.
          (2)	Place the DNA fragments on a flowcell with complementary adapter sequences on the surface. The adapters will adhere to each other to allow the DNA fragments stay at the flow cell.
          (3)	Amplify DNA fragments by bridge amplification.
          (4)	The sequencing uses the method like the Sanger sequencing, adding different bases (dNTPs) and synthetic reagents that have been calibrated for specific removable fluorescent molecules. Repeatedly the process of removing and detecting fluorescence. Last, the computer software will analysis large numbers of DNA sequences quickly.
        </P></DIV><DIV class="title_3" style="margin-left: 13%; margin-bottom: 1.5vw;"><P>FASTQC</P></DIV><DIV class="text" style="margin-top: 1.5vw;"><P>Â Â Â Â Â FASTQC is mainly used to filter the NGS data. It is very important to check the data quality before analyzing the data. Only when the data quality is high enough, the next step can be continued.Â Â Â Â After we input NGS sequence data into FASTQC, the program will analyze automatically and score each sequence to ensure that the quality of the gene sequences is suitable for computer calculation.
        </P></DIV><DIV class="title_3" style="margin-left: 13%; margin-bottom: 1.5vw;"><P>16S rRNA</P></DIV><DIV class="text" style="margin-top: 1.5vw;"><P>Â Â Â 16S rRNA is an important component of the ribosomal small subunit of prokaryote. The sequence contains several conserved regions and 9 hypervariable regions (V1 to V9). The hypervariable regions have genus or species specificity, considered to be the most suitable indicator for phylogeny of bacteria and identification of classification.  NGS 16S uses the sequence of V4 and V5 in the hypervariable regions to detect the bacterial clusters.
        </P></DIV><DIV class="title_3" style="margin-left: 13%; margin-bottom: 1.5vw;"><P>Sampling</P></DIV><DIV class="text" style="margin-top: 1.5vw;"><P>Â Â Â Â Â We divide the agricultural land into four large blocks of A, B, C, and D. In each block, there are three strips of 1, 2, and 3, each of which is divided into T, M, and D. Thus, we have thirty six sample in total( A1T, A2M, ....). We get 50 micro liters per sample from 10 to 15 centimeters depth near the root of each testing plant. Then, the samples are sent to the company for NGS analysis.Â Â Â Â Â The result will present the each bacteria ratio in each samples and report it in an OTU table.(Fig. 1)</P></DIV><DIV class="title_1"><P>Marker Gene Amplicon Analysis</P></DIV><DIV class="text" style="margin-top: 1.5vw;"><P>Â Â Â Â Â Microbiome data are generated from 16S ribosomal RNA(rRNA) gene. The PCR primers were designed to amplify the V4 region of the bacterial 16S ribosomal DNA. After profiling 16S rRNA sequencing, we used QIIME to generate operational taxonomic units (OTUs) table. Then we used bioinformatics tools and statistics methods to analyze microbial diversity in soil samples. We also used machine learning to predict how soil microbiota changes with addition of bio-stimulators.
        </P></DIV><DIV class="title_1"><P>Operational Taxonomic Units Table (OTUs Table)</P></DIV><DIV class="text"><P>Â Â Â Â Â Figure 1 is an example of OTUs table. Each column represents the type and amount of bacteria (OTU1, OTU2, â€¦, OTU7) in each soil sample (A1, A2, A3, B1, B2, and B3). We generate seven tables for each level: Phylum, Class, Order, Family, Genus, and Species.
        </P></DIV><DIV class="explanation">
        Figure 1: Schematic Diagram of Operation Taxnomy Unit table
      </DIV><DIV class="title_1"><P>Data Analysis Process</P></DIV><DIV class="explanation">
        Figure 2: The process of correlation analysis
      </DIV><DIV class="text"><P>Â Â Â Â Â The OTUs tables will consist of unclassified names using the open source pipeline of QIIME. Thus, we have to rearrange the data to facilitate analysis according to the following steps:
          (1) Delete unclassified genomic segments.
          (2) Calculate the ratios of the remaining entries.
          (3) Select the most abundant bacteria within soil samples to observe their distribution using the following bar charts (Fig. 3, Fig. 4, Fig. 5).
        </P></DIV><DIV class="text"><P>Â Â Â Â Â After making the Stacked bar, we organized the data to facilitate the analysis (Table 2). We observed growths and declines of bacteria in the soil, and then summarized the bacteria according to their functions. Moreover, we will explain what we do in our farm affect microbiota. For example: Sphingomonas, Alcanivorax, Devosia which are polluting indicator bacteria, are decreased continuously from May to July. We speculate that the reason of their decline is that we have not applied herbicides, pesticides or other pollutants in these three months. When the soil repaired by itself, the pollutants are falling, and the polluting indicator bacteria are also falling. After the series of analysis, we hope to prove the basic hypothesis of our project--we can precisely regulate microbiota in the soil by using bio-stimulator. We will put our details of analysis in our demonstration.
        </P></DIV><DIV class="explanation">
        Figure 3: Stacked bar chart of top-20 bacteria ratio in different samples (May)
      </DIV><DIV class="explanation">
        Figure 4: Stacked bar chart of top-20 bacteria ratio in different samples (June)
      </DIV><DIV class="explanation">
        Figure 5: Stacked bar chart of top-20 bacteria ratio in different samples (July)
      </DIV><DIV class="title_1"><P>Spearman's Rank Correlation</P></DIV><DIV class="text"><P>Â Â Â Â Â The strength of co-occurrence of bacteria within soil samples was evaluated by the Spearmanâ€™s rank correlation coefficients. It ranges from -1 to 1. The formula of Spearman correlation coefficient is as follows:
        </P></DIV><DIV class="equation">$$\rho_s=1-\frac{6\sum d_{i^2}}{n(n^2-1)}$$</DIV><DIV class="table"><TABLE><CAPTION><P class="explanation">
            Table 1: Variable and Parameter in Spearman's correlation equation.
          </P></CAPTION><THEAD><TR><TH><P>Symbol</P></TH><TH><P>Unit</P></TH><TH><P>Explanation</P></TH></TR></THEAD><TBODY><TR><TD>$\rho_s$</TD><TD>-</TD><TD><P>Spearman's correlation value</P></TD></TR><TR><TD>$d_i$</TD><TD>-</TD><TD><P>The difference in the ranked observations from each group</P></TD></TR><TR><TD>$n$</TD><TD>-</TD><TD><P>The sample size </P></TD></TR></TBODY></TABLE></DIV><DIV class="text"><P>Â Â Â Â Â We used heat maps of correlation to visualize the correlation strength. Figure 3, 4, and 5 show the top 20 abundant bacteria within soil sample in different months. The reason we selected the top 20 abundant bacteria is that we found out the top 20 abundant bacteria accounted for over 95 percent of the amouts of known bacteria. It is true when we analyze frome phylum level to class level. However, analyzing the proportion of those bacteria at genus level, we found that although the proportion of the original top 20 abundant bacteria decrease to about 75% of known bacteria in some months, the proportion of each other bacterium was still lower than 1%. According to the analysis, it seemed that the effect of the bacteria ranked below 20 could be ignored. A computer program can then visualize the results in a heat map. A map of the 20 most abundant bacteria of our soil is shown below:
        </P></DIV><DIV class="explanation">
        Figure 6: Correlation heat map of top-20 bacteria in June
      </DIV><DIV class="text"><P>Â Â Â Â Â For example, the figure above shows the heat map of correlation of June. We could select candidate bacteria by the heat map of June to do prediction of microbiota of July because Weka utilized the correlation formula of the bacteria in June to simulate the microbiota of July. To make it more easily to understand spearman, we could observe the heat map above: while the proportions of two kinds of bacteria increased simultaneously, the block in the table showing the correlation between the two bacteria is red. Conversely, the block would turn to blue if one bacterium increased while the other decreased. Since only when dataâ€™s correlation coefficient larger than 0.7, the data is meaningful in statistics. We then selected the combination of every two bacteria whose correlation coefficient is greater than 0.7 or less than -0.7 as correlative samples. Absolutely there are exceptions to the prediction of our system since the spearman coefficient could only show the ratio of the bacteria in the soil sample we collected of the month while show every single phenomenon in the nature and then lead to cause difference. However, the difference with low correlation coefficient could not increase error tour Weka training.
        </P></DIV><DIV class="table" style="width: 40%; margin-left: 30%;"><TABLE style="width: 100%;"><CAPTION><P>
            Table 2: The ratio variation of top-20 bacteria within June and July.
          </P></CAPTION><THEAD><TR><TH><P>Taxon (Genus)</P></TH><TH><P>June</P></TH><TH><P>July</P></TH></TR></THEAD><TBODY><TR><TD><P><I>Kaistobacter</I></P></TD><TD><P>10.87%</P></TD><TD><P>11.57%</P></TD></TR><TR><TD><P><I>Nitrospira</I></P></TD><TD><P>8.84%</P></TD><TD><P>18.03%</P></TD></TR><TR><TD><P><I>Steroidobacter</I></P></TD><TD><P>7.05%</P></TD><TD><P>3.20%</P></TD></TR><TR><TD><P><I>Rhodoplanes</I></P></TD><TD><P>6.50%</P></TD><TD><P>10.36%</P></TD></TR><TR><TD><P><I>Candidatus Solibacter</I></P></TD><TD><P>5.63%</P></TD><TD><P>3.81%</P></TD></TR><TR><TD><P><I>Candidatus Koribacter</I></P></TD><TD><P>4.36%</P></TD><TD><P>4.53%</P></TD></TR><TR><TD><P><I>Pseudomonas</I></P></TD><TD><P>3.56%</P></TD><TD><P>5.14%</P></TD></TR><TR><TD><P><I>Perlucidibaca</I></P></TD><TD><P>3.03%</P></TD><TD><P>1.06%</P></TD></TR><TR><TD><P><I>Flavisolibacter</I></P></TD><TD><P>2.71%</P></TD><TD><P>3.01%</P></TD></TR><TR><TD><P><I>Hyphomicrobium</I></P></TD><TD><P>2.37%</P></TD><TD><P>1.00%</P></TD></TR><TR><TD><P><I>Bdellovibrio</I></P></TD><TD><P>2.03%</P></TD><TD><P>2.11%</P></TD></TR><TR><TD><P><I>Bacillus</I></P></TD><TD><P>1.96%</P></TD><TD><P>0.84%</P></TD></TR><TR><TD><P><I>Streptomyces</I></P></TD><TD><P>1.87%</P></TD><TD><P>0.41%</P></TD></TR><TR><TD><P><I>Rhodanobacter</I></P></TD><TD><P>1.84%</P></TD><TD><P>0.11%</P></TD></TR><TR><TD><P><I>Flavobacterium</I></P></TD><TD><P>1.55%</P></TD><TD><P>12.12%</P></TD></TR><TR><TD><P><I>Opitutus</I></P></TD><TD><P>1.47%</P></TD><TD><P>1.02%</P></TD></TR><TR><TD><P><I>Devosia</I></P></TD><TD><P>1.46%</P></TD><TD><P>0.72%</P></TD></TR><TR><TD><P><I>Mycobacterium</I></P></TD><TD><P>1.44%</P></TD><TD><P>0.18%</P></TD></TR><TR><TD><P><I>Phenylobacterium</I></P></TD><TD><P>1.37%</P></TD><TD><P>0.87%</P></TD></TR><TR><TD><P><I>Bradyrhizobium</I></P></TD><TD><P>1.06%</P></TD><TD><P>0.42%</P></TD></TR></TBODY></TABLE></DIV><DIV class="title_1"><P>Alpha-Diversity Analysis </P></DIV><DIV class="text"><P>Â Â Â Â Â Use of bio-stimulators to manipulate soil factors requires careful consideration of the microbiota. Certain stimulators may cause specific genera of bacteria to become overly dominant, damaging soil integrity. As a method of monitoring the balance of the microbial ecosystem, we investigate the evenness of the soil.
        </P></DIV><DIV class="title_2"><P>Eveness--Shannon Index</P></DIV><DIV class="text"><P>Â Â Â Â Â Microbial diversity is measured by alpha-diversity (Î±-diversity). In our study, Î±-diversity refers richness and the Shannon diversity index. Richness means the number of OTUs, and evenness of bacterial community is measured by the Shannon diversity index, as shown below:</P></DIV><DIV class="equation">$$H'=-\sum_{i=1}^{S}p_{i}lnp_i$$</DIV><DIV class="table"><TABLE><CAPTION><P class="explanation">
            Table 3: Variable and Parameter in Shannon index equation.
          </P></CAPTION><THEAD><TR><TH><P>Symbol</P></TH><TH><P>Unit</P></TH><TH><P>Explanation</P></TH></TR></THEAD><TBODY><TR><TD>$H'$</TD><TD>-</TD><TD><P>Shannon index</P></TD></TR><TR><TD>$S$</TD><TD>-</TD><TD><P>The total number of genuses in samples</P></TD></TR><TR><TD>$p_i$</TD><TD>-</TD><TD><P>The ratio of bacteria amount of the i<SUP>th</SUP> genus in the sample </P></TD></TR></TBODY></TABLE></DIV><DIV class="text"><P>Â Â Â Â Â A higher Shannon index indicates greater evenness. The estimated degree of evenness can be derived from the exponential of the value. For example, a soil sample with Shannon index 2.85 and $$e^{2.85}=17$$ It means that the sample approximately consists of 17bacteria that are equal in numbers. Thus, the Shannon index can be used as an observational tool to determine whether bio-stimulators decrease the overall evenness or not, and thus health and stability, of the soil.</P></DIV><DIV class="title_3"><P>Triplicate Analysis</P></DIV><DIV class="explanation">
        Figure 7: The box plot of shannon index triplicate analysis
      </DIV><DIV class="title_1"><P>References</P></DIV><DIV class="text"><P>
        1. Kumar, A. and L. C. Rai (2017). &quot;Soil Organic Carbon and Availability of Soil Phosphorus Regulate Abundance of Culturable Phosphate Solubilizing Bacteria in Paddy Fields of the Indo-Gangetic Plain.&quot; Pedosphere.
        2. Wang, P., et al. (2015). &quot;Long-term rice cultivation stabilizes soil organic carbon and promotes soil microbial activity in a salt marsh derived soil chronosequence.&quot; Scientific Reports 5: 15704.</P></DIV></DIV></DIV><TITLE>Template</TITLE></DIV></DIV></DIV></DIV></DIV></BODY></HTML>