<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Arizona_State_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Arizona State/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_menu_wrapper"><DIV class="menu_button" id="display_menu_control">
		MENU  
	</DIV><DIV id="menu_content"><A href="https://2017.igem.org/Team:Arizona_State"><DIV class="menu_button direct_to_page">
				HOME
			</DIV></A><DIV class="menu_button"> TEAM 
		</DIV><DIV class="submenu_wrapper" id="team_submenu"><A href="https://2017.igem.org/Team:Arizona_State/Team"><DIV class="submenu_button" id="Team_page">
					Team 
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Collaborations"><DIV class="submenu_button" id="Collaborations_page">
					Collaborations
				</DIV></A></DIV><DIV class="menu_button"> PROJECT
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Arizona_State/Description"><DIV class="submenu_button" id="Description_page">
					Description
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Design"><DIV class="submenu_button" id="Design_page">
					Design
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Experiments"><DIV class="submenu_button" id="Experiments_page">
					Experiments
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Notebook"><DIV class="submenu_button" id="Notebook_page">
					Notebook
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Results"><DIV class="submenu_button" id="Results_page">
					Results
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Demonstrate"><DIV class="submenu_button" id="Demonstrate_page">
					Demonstrate
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Improve"><DIV class="submenu_button" id="Improve_page">
					Improve
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Attributions"><DIV class="submenu_button" id="Attributions_page">
					Attributions 
				</DIV></A></DIV><DIV class="menu_button"> PARTS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Arizona_State/Parts"><DIV class="submenu_button" id="Parts_page">
					Parts
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Basic_Part"><DIV class="submenu_button" id="Basic_Part_page">
					Basic Parts
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Part_Collection"><DIV class="submenu_button" id="Part_Collection_page">
					Part Collection
				</DIV></A></DIV><A href="https://2017.igem.org/Team:Arizona_State/Safety"><DIV class="menu_button direct_to_page">
				SAFETY
			</DIV></A><DIV class="menu_button"> HUMAN PRACTICES 
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Arizona_State/HP/Silver"><DIV class="submenu_button" id="Silver_page">
					Silver HP
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/HP/Gold_Integrated"><DIV class="submenu_button" id="Gold_Integrated_page">
					Integrated and Gold
				</DIV></A></DIV><DIV class="menu_button"> AWARDS
		</DIV></DIV><A href="https://2017.igem.org/Team:Arizona_State/Measurement"><DIV class="submenu_button" id="Measurement_page">
					Measurement
				</DIV></A></DIV></DIV></DIV></DIV><A href="https://igem.org/2017_Judging_Form?team=Arizona_State"><DIV class="menu_button direct_to_page">
				JUDGING FORM
			</DIV></A></DIV></DIV><DIV class="igem_2017_content_wrapper"><H1>Arizona_State</H1><DIV class="container"><H2>The Problem</H2><UL><LI> Quorum sensing (QS) is a type of bacterial communication, a system of communication signals synthesized and received by different bacterial species. Little is known about how this and other microbial sensing systems operate in nature (Decho 2010). </LI><LI>AHL molecules are not completely system-specific, as crosstalk can occur between systems-this causes problems with developing functional genetic circuits</LI><LI>AHL quorum sensing is used to regulate factors such as virulence and biofilm formation, potentially activating pathogenic bacteria</LI></UL><H2>Brief Summary</H2><P> Quorum sensing (QS) is at the foundation of a wide range of high-impact bioengineering efforts such as creating new biosensors and health monitoring devices while providing a toolbox of reusable genetic components that can be plugged into circuits at will (Kwok, 2010). QS involves systems of bacteria that use sender and receiver molecules to communicate gene expression when the bacteria reach optimal densities. In doing so, quorum sensing allows bacteria to express specific genes at a high population density that results in beneficial phenotype expression.There are many different types of QS sensing molecules and in this research we will be working with a type of chemical signal called acyl-homoserine lactones (HSLs). When these signals are received by the surrounding bacteria, the signal is transduced. Creating genetic circuits is done by characterizing genetic sequences that perform needed functions and combining them into devices that are inserted into cells (Kwok, 2010). </P><P> A major problem in engineered systems is crosstalk. This is when different species of senders can activate a noncanonical receiver's promoter. When QS pathways operate without communication between unwanted cells, the pathways are orthogonal and potentially viable options for bioengineering new synthetic circuits. A sender is defined as cell that expresses AHL synthase, while a receiver is a cell that includes an inducible promoter that initiates transcription of a gene and regulator controls the expression of one or more genes. Researchers want to find systems that are completely orthogonal, in efforts to enhance efficacy while maintaining specificity [7]. The application of defining cross-talk might lead to more sophisticated intracellular communication [7].  In addition, these circuits may then be engineered to detect specific combinations of input signals that could be used to engineer multi-strain, self monitoring microbial populations that perform energetically costly metabolic processes in a single culture (Davis et al., 2015).
 </P><P> The objective of our project is to design and test a variety of quorum sensing networks. This includes creating new receivers for our system, as well as researching various concentrations of AHL signals. We have developed a flexible testing platform in which the QS system is separated into two components designated the “Sender” and the “Receiver”. The AHL synthase is expressed in the Sender cell, while the inducible promoter and regulator are carried by a Receiver cell. When the Sender produces a signal, the HSL, it diffuses across cell membranes and activates the Receiver. In our current system, Receivers will express green fluorescent protein (GFP) in response to induction by Senders from different bacterial species. Ideally, the designed systems would have low amounts of interference and form a functional genetic circuit. Our team has built 3 new receivers.  </P><P> Our iGEM team is investigating the diverse applications that fit with our quorum sensing quest. Some of the side quests include: new receivers with hybrid promoters, concentration of N-acyl homoserine lactone (AHLs), combinations of different senders, induction and diffusion rates, mathematical models and the case of the disappearing mCherry. </P><H2>MOTIVATION</H2><H3> Suiting Up for Battle </H3><P>Promoters are a region of DNA that initiates transcription of a specific gene. In bacteria, promoters contain 2 short sequence elements about 10 and 35 nucleotides upstream from a transcription start site. For our project, we utilize inducible promoters to initiate the transcription of regulator proteins and inducible GFP signals. Inducible promoters are a power tool in genetic engineering because the expression of genes operably linked to them can be turned on or off at certain stages of development of an organism or in a particular tissue (Brautaset,2009). This allows us to track and analyze data to find orthogonality between senders and receivers. This is done when senders produce acyl-homoserine signals (AHLs) and attach to a regulator promoter in the corresponding receiver system. This allows for DNA binding and transcription initiation. This in turn allows for proteins to be made which then bind to the inducible promoter to allow GFP to be turned on. If there is no transcription of our regulator gene, aka no AHL attachment to the promoter of the regulator protein,  our system won’t turn on. This will allow our team to run various experiments and see if it is working. This particular production of HSLs is just one one type of quorum sensing system out there and the type of system that is utilized in this project. We can test our system of senders and receivers for orthogonality by setting up induction experiments with different sender signals. A positive result means the sender signal will only turn on the GFP signal of one regulator gene, meaning the GFP is expressed in the system. </P><P>During the design conception of this project, we noticed that last year’s 2016 ASU iGEM team’s nonfunctioning receivers were not constructed properly on the DNA level.  Since Last year’s team only had one receiver, it  was difficult to find orthogonal pairs of sender-receivers. For this reason,we decided to design and synthesize new receivers for orthogonality testing. First, we researched into designing new promoters for AHL quorum sensing systems. In doing so, we found that the some of the receivers in the system researched last year did not have a proper inducible promoter. For example, some receiver systems did not even include an inducible promoter within their system; or they used a wrong binding site within the inducible promoter. For this reason, designing new inducible promoters for receivers was a top priority this year for our team. </P><P>Our team came across a paper by Spencer R Scott and Jeff Hasty from UC San Diego. They designed new inducible promoters that lead to better expression and easier cloning in their specific AHL related QS systems (Scott, 2016). Due to this, our iGEM team utilized their thought and design process into our systems. The goal of our project is to successfully incorporate inducible promoters into our system to have them respond to HSLs and induce expression of GFP in E.coli. In addition, we hope to find undiscovered orthogonality between our senders and receivers. Hybrid promoters Ptra* and Prpa* were created by replacing the lux-box in the commonly used PluxI promoter with the tra-box and the rpa-box, respectively (Scott, 2016). Using this idea, we created new receivers for our system with tra, rpa, and las genes to test in our experiments. In addition to this, new receivers of Bja (Lindemann,2011), Aub (Nasuno,2012), and Rhl (Pearson, 1997) were created for testing. This was done by having a promoter from Lux and combining it with the specific regulator gene binding domain (Scott,2016). In addition to the inducible promoter, we also rearranged to order of our two part receiver system. The regulator and GFP were originally in that respective order. However, in our new receivers that orientation is switched. This was due to finding a leaky expression due to transcriptional read through of the receiver. By swapping the order, we can optimize the sequence to avoid transcriptional read through in our reporter gene. </P><P> Some broad descriptions of experiment ran are testing concentration of N-acyl homoserine lactone, testing combination of different senders, and testing induction and diffusion rates with senders and receivers. </P><DIV class="container"><H2> 2017 New Receivers </H2><TABLE style="width:100%" align="center"><TBODY><TR><TH>Part Name</TH><TH>Part Number</TH><TH>Part Type</TH></TR><TR><TD> TraR </TD><TD><A href="http://parts.igem.org/Part:BBa_K2357028">BBa_B2357028</A></TD><TD>Reciever</TD></TR><TR><TD> LasR </TD><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2357000">BBa_K2357000</A></TD><TD>Receiver</TD></TR></TBODY></TABLE></DIV><DIV class="container"><H2> iGEM F2620 Improvement  </H2><TABLE style="width:100%" align="center"><TBODY><TR><TH>Part Name</TH><TH>Part Number</TH><TH>Improvement</TH></TR><TR><TD> F2620 </TD><TD><A href=" http://parts.igem.org/Part:BBa_F2620:Experience">BBa_F2620</A></TD><TD>Reciever that our team further characterized and improved by running induction plates and various sender AHL experiments. In addition, safety for degrading and disposing. </TD></TR></TBODY></TABLE></DIV><H3> Lost in Translation  </H3><P> This exploration in our missing mCherry gives other QS related experiments an explanation for possible mCherry disappearance and allows an opportunity for a possible solution to non-inducing senders. This process is vital due to the cultures being used are neighboring colonies from the same transformed agar plates. There were no prior issues that could lead to this lack of expression , thus giving us an increases a need for a possible revelation to this mystery. This experiment can lead to possible reason why there is no mCherry in random cultures, what this lack of expression means senders, and possible solution to fix this lack of expression for future ventures. </P><P> The motivation for this side project is to determine whether or not AHL synthase is being produced in correspondence to mCherry. mCherry was used within the sender systems to indicate if the AHL synthases are being produced. The senders are used within our quorum system to produce AHL synthases, which produces AHL signals that then go over to the receiver system described up above to bind to the regulatory proteins to attach to the receptor binding site in the inducible promoter to initiate transcription of the GFP. By mini-prepping various sender systems from ASU's 2016 iGEM team senders, and then sequencing, we can determine whether or not mCherry is a good indicator of AHL synthase production</P><H3> Battle of the AHLs </H3><P>Using past research and experimental data from many sources, including the experiments performed in the paper written by Spencer and Hasty, 2016, we have chosen to study the issue of “crosstalk” in hopes of solving the issue so that more complex circuits can be built down the road. Crosstalk occurs when cells that receive AHL signals are also able to receive AHLs from different QS systems. When QS pathways operate without communication between unwanted cells, the pathways are orthogonal and potentially viable options for bioengineering new synthetic circuits. Reducing crosstalk will allow us to increase the complexity of the circuits in the cells allowing them to carry out more complex tasks while working independently of each other (Purnick et al, 2009). </P><P>On this side quest, multiple sender AHLs are preparing for battle against the three evil receivers: LuxR, LasR and TraR. With the available data on how single senders interact with a receiver, this quest aims to fight the evil receivers with TWO senders at a time and gather the results. This battle royale should give some new results on how the receivers act when attacked from two sides with different concentrations of AHLs. Understanding the enemy will give the upper hand to synthetic biology by adding new combinations of senders to the known registries of synthetic circuits. In the great battle, the winning combinations will express either a higher or lower GFP than a single sender would alone. This data, along with any new orthogonal pathways found in the process, will make the attack on the receivers all worthwhile. This is because adding new data on how to manipulate circuits beyond the ‘all or nothing’ responses of single sender tests will provide a higher level of control regarding gene expression. This level of control is important when a circuit needs to be made to express at a specific level versus just expressing the gene.  </P><P>This experiment explores BL21 E. coli bacteria that has been transformed with our sender plasmids, in the modular sender vector (MSV). MSV is used as a vehicle to artificially carry foreign genetic material into another cell, where it can be replicated and/or expressed. The sequence that includes the antibiotic resistance marker and the replication origin is held within the backbone of the plasmid. The image below depicts an example of a sender gene for LuxI, one of the senders we will be testing. </P><P>Previous research used just one AHL at a time with the Lux receiver to catalogue the functionality of potential circuits. The Lux F2620 is a composite gene constructed by standard assembly from five BioBrick standard biological parts (Canton et al., 2008). In this research, we will be combining and testing two different HSLs in the form of freshly grown cultured sender cell (filtered) supernatants. Different concentrations of the two senders will be tested with one receiver at a time to catalogue our ten senders for crosstalk, orthogonality, signal disruption and possible circuit enhancing combinations. Results will be measured using Green fluorescent protein (GFP) expression in the receiver bacteria. The GFP expression levels will depend on how well the AHLs bind to regulatory proteins inducing transcription of the GFP. This can change based on the AHL concentration and how much crosstalk there may be between the different senders. The differences in GFP expression levels will be used to evaluate how well the HSLs signals are being received compared to the other combination samples. Results can then be analyzed based on the varying percentages of one supernatant to the other combinations of senders that were used. Signal strength (induction) over time provides information about the strength of the overall induction (GFP/OD). The rate of induction will be analyzed using the hill equation to better understand how the concentration of AHLs affects induction. The addition of these details to the known catalogue of QS pathways will make for easier development of specific synthetic circuits in the future designed to operate either faster or slower depending on the specific need. </P><P>Some of the specific questions this research aims to answer are: how do combinations of senders affect gene output? Can we find any combinations of senders that increase the overall GFP expression? Can we find any combinations that do not affect the GFP expression? </P><P>The specific senders chosen for the induction tests were selected because previous research showed that they have either a very low or very high rate of GFP induction when used in a single sender/ receiver circuit. In other words, the chosen senders tend to either work very well or not very well at all and we need more data on how well these senders express the gene when used in combination with another. By combining two senders at a time, sometimes with senders that have shown to induce a high GFP expression and sometimes with senders that have shown a weak induction, our team wanted to see if it could increase or decrease the GFP expression on demand as needed. The controls used for the experiment were testing single sender inductions on the same plate as the combinations, the use of blank wells (LB AMP 100%), a positive GFP control, and a negative control with negative receiver cells and negative sender supernatant. </P><H3> Alternate Synthetic Dimension: Robot Quest for GFP Domination </H3><P>Last year’s ASU 2016 iGEM team only used the synthetic AHLs of Rhl, Lux, Tra, Las, and Rpa. These AHLs were tested on the receiver F2620, that last year’s team characterized using 2 concentrations of 1E-6M and 1E-7M (Davis et al). This year, the 2017 iGEM team decided to expand this type of characterization on the improved receivers of Lux, Las, and Tra. This was done by increasing the range of AHL concentrations to 1E-14M to 1E-4M (Scott et al). The number of synthetic AHLs was also expanded to 6 and included the Rhl, Rpa, Tra, Lux, Las, and Sin. These tests are important to characterize the newly improved receivers and to determine how varying the AHL concentration affects GFP expression. In summary, testing of these multiple AHLs is being taken a step further by looking at how the concentration of these AHL chemicals in the cells affect the gene expression of the receiver. </P><H3> AHL Disposal and Human Practices </H3><P>Our motivation for modeling LasR, TraR, and LuxR through quantifying induction rates over time was to inform our various “quests” by crafting an understanding of the dynamics between each Sender and Receiver combo. The imaging process allowed for a 3D representation of induction rates and strength of GFP production giving a visual representation of the AHL's (sender/receiver combos) and if they were inducible or did not respond at all. This information simulated expected behavior in our media induction tests which allowed for more precise system design when evaluating sender/receiver behavior.</P><P>The 3D analysis methods allowed for highly precise measurement of GFP induction rate and overall strength. This method has not been used with agar induction plates before and with the use of negative and positive controls it allowed for our modeling portion of our project to be highly effective at informing our project design. Previously with 2D induction modeling only coarse grain measurements and rough/inaccurate induction distance could be obtained. Use of the 3D analysis allowed for X,Y &amp; Z coordinates to be obtained which provided quantitative data to be obtained for the actual strength of induction. This was analyzed against a known positive GFP+ control. These measurements allowed for further contextualization of the various side quests which reﬁned our ability to analyze the differences between different sender/receiver combos. </P><DIV class="container"><H2>N-Acyl Homoserine Lactones</H2><P>AHL quorum sensing has a myriad of different systems. A total of 10 systems were investigated in this project </P><TABLE style="width:100%"><TBODY><TR><TH>AHL System</TH><TH>Bacteria of Origin</TH><TH>AHL Name</TH><TH>3D-Model</TH></TR><TR><TD>Aub</TD><TD>Unknown</TD><TD>N-(2-oxooxolan-3-yl)dodecanamide</TD></TR><TR><TD>Bja</TD><TD>Bradyrhizobium japonicum</TD><TD>3-methyl-N-[(3S)-2-oxooxolan-3-yl]butanamide</TD></TR><TR><TD>Bra</TD><TD>Paraburkholderia kururiensis</TD><TD>(3S)-3-[(2-oxo-3-phenylpropyl)amino]oxolan-2-one</TD></TR><TR><TD>Cer</TD><TD>Rhodobacter sphaeroides</TD><TD>(Z)-3-hydroxy-N-[(3S)-2-oxooxolan-3-yl]tetradec-7-enamide</TD></TR><TR><TD>Esa</TD><TD>Erwinia stewartii</TD><TD>3-oxo-N-[(3S)-2-oxooxolan-3-yl]hexanamide</TD></TR><TR><TD>Las</TD><TD>Pseudomonas aeruginosa</TD><TD>3-oxo-N-(2-oxooxolan-3-yl)dodecanamide</TD></TR><TR><TD>Lux</TD><TD>Vibrio fischeri</TD><TD>3-oxo-N-(2-oxooxolan-3-yl)hexanamide</TD></TR><TR><TD>Rhl</TD><TD>Rhizobium leguminosarum</TD><TD>N-(2-oxooxolan-3-yl)butanamide</TD></TR><TR><TD>Rpa</TD><TD>Rhodopseudomonas palustris</TD><TD>(S)-α-amino-γ-butyrolactone</TD></TR><TR><TD>Sin</TD><TD>Sinorhizobium meliloti</TD><TD>N-[(3S)-2-oxooxolan-3-yl]octanamide*</TD></TR></TBODY></TABLE><P>*Sin system produces 6 different variants of AHL. The 3D structures of all the Sin compounds can be found <A href="https://www.dropbox.com/sh/cfo744xq81t6iu4/AAAAG8yiFkHfnUojBZE7bWKla?dl=0">here</A>.</P><P>AHLs share the same basic structure, with a lactone ring, an N-acyl and ketone group. The defining R group lies in the acyl tail, which is the primary determinant in its transcription factor binding affinity. The graphic below demonstrates the categorization of the AHLs produced by the 10 studied systems</P><H2>REFERENCES</H2><LI>Brautaset, Trygve, Rahmi Lale, and Svein Valla. “Positively Regulated Bacterial Expression Systems.” Microbial biotechnology 2.1 (2009): 15–30. PMC. Web. 8 Sept. 2017. </LI><LI>Scott, S. R., and J. Hasty. &quot;Quorum Sensing Communication Modules for Microbial Consortia.&quot; ACS Synth Biol 5.9 (2016): 969-77. Web. 8 Sept 2017. </LI><LI>Nasuno, E., et al. &quot;Phylogenetically Novel Luxi/Luxr-Type Quorum Sensing Systems Isolated Using a Metagenomic Approach.&quot; Appl Environ Microbiol 78.22 (2012): 8067-74. Web. 9 Oct. 2017.</LI><LI>Lindemann, Andrea et al. “Isovaleryl-Homoserine Lactone, an Unusual Branched-Chain Quorum-Sensing Signal from the Soybean Symbiont Bradyrhizobium Japonicum.” Proceedings of the National Academy of Sciences of the United States of America 108.40 (2011): 16765–16770. PMC. Web. 9 Oct. 2017. </LI><LI>Pearson, J. P., E. C. Pesci, and B. H. Iglewski. &quot;Roles of Pseudomonas Aeruginosa Las and Rhl Quorum-Sensing Systems in Control of Elastase and Rhamnolipid Biosynthesis Genes.&quot; J Bacteriol 179.18 (1997): 5756-67. Web. 9 Oct. 2017.</LI><LI> Davis, Rene Michele, et al. “Corrigendum: Can the Natural Diversity of Quorum-Sensing Advance Synthetic Biology?” Frontiers in Bioengineering and Biotechnology, vol. 3, July 2015, doi:10.3389/fbioe.2015.00099 </LI><LI>Decho, Alan W., R. Sean Norman, and Pieter T. Visscher. 2010. “Quorum Sensing in Natural Environments: Emerging Views from Microbial Mats.” Trends in Microbiology 18 (2): 73–80.</LI><LI>Fuqua, C, Winans, SC, Greenberg, EP, et al. Census and consensus in bacterial ecosystems: the LuxR-LuxI family of quorum-sensing transcriptional regulators. Annu. Rev. Microbiol. 1996; 50: 727–751. doi:10.1146/annurev.micro.50.1.727</LI><LI>Miller, MB, and Bassler, BL. Quorum sensing in bacteria. Annu. Rev. Microbiol. 2001;55: 165–199. doi:10.1146/annurev.micro.55.1.165</LI><LI>Davis, RM, Muller, RY, Haynes, KA. Can the natural diversity of quorum sensing advance synthetic biology? Front. Bioeng. Biotechnol. 2015; 3.30.1-9. doi: 10.3389/fbioe.2015.00030</LI><LI>Lindemann, A, Pessi, G, Schaefer, A, Mattmann, M, Christensen, Q, Kessler, A, et al. Isovaleryl-homoserine lactone, an unusual branched-chain quorum-sensing signal from the soybean symbiont Bradyrhizobium japonicum. Proc. Natl. Acad. Sci. U.S.A. 2011;108: 16765–16770. doi:10.1073/pnas.1114125108</LI><LI>Nasuno, E, Kimura, N, Fujita, MJ, Nakatsu, CH, Kamagata, Y, Hanada, S, et al. Phylogenetically novel LuxI/LuxR-type quorum sensing systems iso- lated using a metagenomic approach. Appl. Environ. Microbiol. 2012;78: 8067–8074. doi:10.1128/AEM.01442- 12</LI><LI>Eberhard, A, Burlingame, AL, Eberhard, C, Kenyon, GL, Nealson, KH, Oppenheimer, NJ, et al. Structural Identification of Autoinducer of Photobacterium fischeri. Biochemistry. 1981;20: 2444–2449.</LI><LI>Marketon, MM, Gronquist, MR, Eberhard, A, González, JE et al. Characterization of the Sinorhizobium meliloti sinR / sinI Locus and the Production of Novel N-Acyl Homoserine Lactones Characterization of the Sinorhizobium meliloti sinR / sinI Locus and the Production of Novel N -Acyl Homoserine Lactones. J. Bacteriol. 2002;184: 5686–95. doi:10.1128/JB.184.20.5686.</LI><LI>Scott, Spencer R., and Jeff Hasty. 2016. “Quorum Sensing Communication Modules for Microbial Consortia.” ACS Synthetic Biology 5 (9): 969–77.</LI><LI>Davis, René Michele, Ryan Yue Muller, and Karmella Ann Haynes. 2015. “Can the Natural Diversity of Quorum-Sensing Advance Synthetic Biology?” Frontiers in Bioengineering and Biotechnology 3(March): 30.</LI><LI>Marchand, Nicholas, and Cynthia H. Collins. 2016. “Synthetic Quorum Sensing and Cell-Cell Communication in Gram-Positive Bacillus Megaterium.” ACS Synthetic Biology 5 (7): 597–606.</LI><LI>Choudhary, Swati, and Claudia Schmidt-Dannert. 2010. “Applications of Quorum Sensing in Biotechnology.” Applied Microbiology and Biotechnology 86 (5): 1267–79.</LI><LI>Jayaraman, Arul, and Thomas K. Wood. 2008. “Bacterial Quorum Sensing: Signals, Circuits, and Implications for Biofilms and Disease.” Annual Review of Biomedical Engineering 10: 145–67.</LI><LI>Purnick, Priscilla E. M., and Ron Weiss. 2009. “The Second Wave of Synthetic Biology: From Modules to Systems.” Nature Reviews. Molecular Cell Biology 10 (6): 410–22.</LI><LI>Kwok, Roberta. 2010. “Five Hard Truths for Synthetic Biology: Can Engineering Approaches Tame the Complexity of Living Systems? Roberta Kwok Explores Five Challenges for the Field and How They Might Be Resolved.” Nature 463 (7279). Nature Publishing Group: 288–91.</LI><LI>Wang, Baojun, Mauricio Barahona, and Martin Buck. 2013. “A Modular Cell-Based Biosensor Using Engineered Genetic Logic Circuits to Detect and Integrate Multiple Environmental Signals.” Biosensors &amp; Bioelectronics 40 (1): 368–76.</LI><LI>Canton, Barry, Anna Labno, and Drew Endy. 2008. “Refinement and Standardization of Synthetic Biological Parts and Devices.” Nature Biotechnology 26 (7): 787–93.</LI><LI>LaSarre, Breah, and Michael J. Federle. 2013. “Exploiting Quorum Sensing to Confuse Bacterial Pathogens.” Microbiology and Molecular Biology Reviews: MMBR 77 (1): 73–111. </LI></DIV></DIV></DIV><A href="/wiki/index.php?title=Template:Arizona_State_Footer&amp;action=edit&amp;redlink=1" class="new" title="Template:Arizona State Footer (page does not exist)">Template:Arizona State Footer</A></BODY></HTML>