<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Arizona_State_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Arizona State/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_menu_wrapper"><DIV class="menu_button" id="display_menu_control">
		MENU  
	</DIV><DIV id="menu_content"><A href="https://2017.igem.org/Team:Arizona_State"><DIV class="menu_button direct_to_page">
				HOME
			</DIV></A><DIV class="menu_button"> TEAM 
		</DIV><DIV class="submenu_wrapper" id="team_submenu"><A href="https://2017.igem.org/Team:Arizona_State/Team"><DIV class="submenu_button" id="Team_page">
					Team 
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Collaborations"><DIV class="submenu_button" id="Collaborations_page">
					Collaborations
				</DIV></A></DIV><DIV class="menu_button"> PROJECT
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Arizona_State/Description"><DIV class="submenu_button" id="Description_page">
					Description
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Design"><DIV class="submenu_button" id="Design_page">
					Design
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Experiments"><DIV class="submenu_button" id="Experiments_page">
					Experiments
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Notebook"><DIV class="submenu_button" id="Notebook_page">
					Notebook
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Results"><DIV class="submenu_button" id="Results_page">
					Results
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Demonstrate"><DIV class="submenu_button" id="Demonstrate_page">
					Demonstrate
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Improve"><DIV class="submenu_button" id="Improve_page">
					Improve
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Attributions"><DIV class="submenu_button" id="Attributions_page">
					Attributions 
				</DIV></A></DIV><DIV class="menu_button"> PARTS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Arizona_State/Parts"><DIV class="submenu_button" id="Parts_page">
					Parts
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Basic_Part"><DIV class="submenu_button" id="Basic_Part_page">
					Basic Parts
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Part_Collection"><DIV class="submenu_button" id="Part_Collection_page">
					Part Collection
				</DIV></A></DIV><A href="https://2017.igem.org/Team:Arizona_State/Safety"><DIV class="menu_button direct_to_page">
				SAFETY
			</DIV></A><DIV class="menu_button"> HUMAN PRACTICES 
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Arizona_State/HP/Silver"><DIV class="submenu_button" id="Silver_page">
					Silver HP
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/HP/Gold_Integrated"><DIV class="submenu_button" id="Gold_Integrated_page">
					Integrated and Gold
				</DIV></A></DIV><DIV class="menu_button"> AWARDS
		</DIV></DIV><A href="https://2017.igem.org/Team:Arizona_State/Measurement"><DIV class="submenu_button" id="Measurement_page">
					Measurement
				</DIV></A></DIV></DIV></DIV></DIV><A href="https://igem.org/2017_Judging_Form?team=Arizona_State"><DIV class="menu_button direct_to_page">
				JUDGING FORM
			</DIV></A></DIV></DIV><DIV class="igem_2017_content_wrapper"><H1>Arizona_State</H1><H1><INS>Notebook:Amber Mani </INS></H1><H2>Friday 6/23/17 - Running colony PCR on the AUBr and the BTA3</H2><P>From earlier this week the AUBr results showed that number 9 may be a match for the vector we needed but the sequencing was returned and it was an error. Redoing the AUBr today and running the PCR for the BTA3 to try for a result on at least one per receiver. </P><P>Since it is the weekend, instead of placing the new agar plate with the 24 samples of plated AUBr and BTA3 into the 37c incubator, leaving the sample out in room temperature as to not accelerate the growth too much over the weekend. See picture below:</P><UL><LI>Defrost template, 2x Master Mix and primers</LI><LI>Determine desired total working volume (~20uL per tube of 24 total/ each AUBr and BTA3 have 24 samples=520ul. The additional 40ul over the 480ul is to give 40ul (2 additional samples) over as precautionary buffer)</LI><LI>Label reaction tubes</LI><LI>Create below reaction mix</LI><LI>Place mixture in thermal cycler at settings shown below</LI></UL><P>* Used 24 PCR tubes and labeled 1-24 for each receiver</P><P>* Thawed two AMP plate and labeled 24 spots on tray / AUB/AMBER/6/23/17 and BTA3/BRIANNA/6/23/2017</P><P>* Used 260 μl of Go Taq Green - added to each 2 DNA tubes</P><P>* Added 249.6 μl of H20, to each of the 2 DNA tubes</P><P>* Added FP and RP 5.2 ul of each to DNA tubes,(1 μ molar as final concentration)</P><P>* Pipetted 20 μl of newly mixed green solution into each PCR tube</P><H3><INS>Colony Prep: See images at the beginning of todays entry for the new plated 24 samples of AUBr and BTA3</INS></H3><P>* Next scraped one colony at a time (up to the 24 used) from the existing grown AUB plate and gently placed the colonly onto the new AUB plate with the labeled 24 spots. Disposed of each tip used into the cooresponding PCR tube labeled to match the placement number on the new AUB plate. Same protocol for the BTA3.</P><P>* Used empty pipette to attach to each tip in the PCR tubes and pump up and down to mix and remove as much of the leftover bacteria from the tip as possible so it mixed into the PCR tube. Then disposed of the tips in dry waste.</P><P>* Took 2 full sets of 24 PCR tubes to the thermocycler and had Stephan help with the settings since each setting is different for each type of bacteria or sample being run for PCR.</P><P>* PCR set to take 2.5 hours/ in process now.</P><P>PCR finished up and running gels for both the AUBr and BTA3.</P><P>Running a gel to test the DNA vector concentration that we are looking for - AUBr has 24 samples with 20 μl in each PCR tube and BTA3 has 24 samples of 20 μl each PCR tube. Will run 2 gels, one for AUB and one for BTA3, each with with 5 μl of each sample in the gel. To fit everything we ran with 2 rows of the 14 wells.</P><P>* The first gel to run is the AUBr: The first well is the KB+ ladder, the 2nd well is blank. Across the top row, in numberical order, starting in the 3rd well, are samples number 1-12. In the second row the first well is the KB+ ladder, the second well is blank and starting in the 3rd well are numbers 13-24. Ran the gel at 110V for 45 minutes.</P><P>Image of the AUBr gel setup:</P><P>* The second gel to run is the BTA3: The first well is the KB+ ladder with the second well blank. Across the top row, in numerical order, starting in the 3rd well, are 5μl samples number 1-12. In the second row the first well is the KB+ ladder, the second well is blank and starting in the 3rd well are 5μl samples number 13-24. Ran the gel at 110V for 45 minutes.</P><P>Image of the BTA3 gel setup:</P><P>In the gel results for the AUBr there were ZERO/ 0 samples that were successful:</P><P>The gel results for the BTA3 below showed that there was ZERO / 0 successful BTA3 samples.</P><P>After closer examination there were some good samples but the placement of the DNA in the loading wells was mixed up. Sequencing confirmed the fragments were what we were looking for and the error was corrected resulting in a successful experiment.</P><H2>Transformation of DH5alpha with PSB1C3 and PSB1A3 backbones</H2><H3> Wednesday, 7/19 </H3><P>In preparation Alyssa and I rehydrated 2 backbones from iGEM, the PSB1C3 and PSB1A3. The C3 was from plate 7 location O,23 and the A3 was from plate 4 location H,2. Will let the plates grow overnight in the incubator and make MP samples, in triplicates of each, tomorrow.</P><P>DNA extraction from kit and rehydration protocol: </P><P>To use the DNA in the Distribution Kit, follow these instructions: </P><P><I>Note: There is an estimated 2-3ng of DNA in each well, following this protocol, assume that you are transforming with 200-300pg/µL</I></P><OL type="1"><LI>With a pipette tip, punch a hole through the foil cover into the corresponding well of the part that you want. Make sure you have properly oriented the plate. Do not remove the foil cover, as it could lead to cross contamination between the wells.</LI><LI>Pipette 10µL of dH2O (distilled water) into the well. Pipette up and down a few times and let sit for 5 minutes to make sure the dried DNA is fully resuspended. The resuspension will be red, as the dried DNA has cresol red dye. We recommend that you do not use TE to resuspend the dried DNA.</LI><LI>Transform 1µL of the resuspended DNA into your desired competent cells, plate your transformation with the appropriate antibiotic* and grow overnight.</LI><LI>Pick a single colony and inoculate broth (again, with the correct antibiotic) and grow for 16 hours.</LI><LI>Use the resulting culture to miniprep the DNA AND make your own glycerol stock (for further instruction on making a glycerol see this page). We recommend using the miniprepped DNA to run QC tests, such as restriction digests and sequencing.</LI></OL><P><I>* To know which antibiotics to use, look at the plasmid that the part is in. The naming scheme for plasmids is specifically designed to indicate antibiotic resistance.</I></P><P><B>7/20/2017 - something went wrong with the process, the plates to not show sufficient growth. Tests and plating will be redone today.</B></P><P>Gathered the available colonies and put in LB with (AMP or CHLOR) to grow overnight and replate for more usable growth to be MP. samples in the shaking incu overnight, was able to grab 3 colonies from the A3 and two from the C3. Put everything in the shaking at 345pm. </P><P>Doing the MP on the A3 and C3 growth tubes. Concentrations:</P><UL><LI>psb1c3: 80ng/ul</LI><LI>psb1c3: 40ng/ul</LI><LI>psb1a3: 50ng/ul</LI><LI>psb1a3: 67ng/ul</LI><LI>psb1a3: 75ng/ul</LI></UL><H1><INS> Notebook: Chris Connot </INS></H1><H2>AHL Degredation: Ethanol/ISO TraR</H2><H3>Wednesday, 9/20</H3><P>Notes: This is the second run of testing the degradation of AHLs via the treatment of 2-Propanol and ETOH. The first was with LasR and the next will be with 2620 (Lux) if the synthetic AHLs can be found. The first protocol is for preparing necessary parts for the plate reader which will have a protocol added below the Prep Protocol</P><P>Notes: This is the second run of testing the degradation of AHLs via the treatment of 2-Propanol and ETOH. The first was with LasR and the next will be with 2620 (Lux) if the synthetic AHLs can be found. The first protocol is for preparing necessary parts for the plate reader which will have a protocol added below the Prep Protocol. </P><H3>Prep Protocol: </H3><OL type="1"><LI>Grabbed 6 15mL culture tubes</LI><LI>Labeled 2 for Neg Sender, 2 for TraR #2, and 2 for Neg Receiver</LI><LI>Using a serological tube, 25mL of LB/AMP was pipetted out, and approximately 4 mL was pipetted into each culture tube (more than 4 mL will cause for decreased growth capacity). </LI><LI>A seperate 250mL flask was used for additional supernatant of neg sender, and 50mL of LB/AMP was also pippetted into the flask.</LI><LI>Using the bacteria plates for neg sender/receiver (9/14/17) and TraR #2 (8/16/17) transformation (s). The cells were scraped off from each plate using a different tip each time (single colony) and ejected into the corresponding culture tubes. </LI><LI>The tubes were then properly closed and a foil was placed over the flask</LI><LI>The tubes and flask were then placed into the 37C shaking incubator for overnight growth.</LI><LI>After incubation, the OD600 is taken for the GFP+, -REC and the Sender</LI><LI>Calculated volumes of GFP, -REC and the Sender to give a concentration of .70 at a volume of 500 ul</LI><LI>The appropriate volume of supernatant is taken from the overnight cultures (after being spun down in the centrifuge at max speed for 1 minute) and placed in a total volume (including supernatant) of 500 ul of fresh LB/Amp</LI><LI>Where calculated volumes utilize the equation C1V1=C2V2</LI><LI>Sender is spun down till clear in 50 ml conical vial </LI><LI>a filter syringe is used to transport the supernatant into a new 50 ml conical vial </LI><LI>The receiver (TraR) is now spun down in a 50 ml conical vial (4 ml) and supernatant is discarded and the cells are resuspended in 20 mL of LB/AMP. </LI><LI>After all cells have been spun down, resuspended and transferred into new media it is time to prepare the treatment tubes of AHL (treated with chosen alcohol) </LI><LI>1.5 ml tubes are used to create serial dilutions of the alcohol for concentrations 10^-1 to 10^-12 (choose range based on need)</LI><LI>Powdered synthetic AHL's are used to create the diluted 1.5ml concentrations </LI><LI>To each 1.5 ml tube (10^-1 to 10^-12) the chosen alcohol (i.e ETOH or 2-propanol) is added and the tubes are closed to allow treatment to take place over 15 minutes </LI><LI>The ETOH or 2-Propanol is added at an amount that maintains a total volume concentration of 70% ETOH or 2-propanol</LI><LI>Once the 15 minutes is over the tubes are opened and placed in a well ventilated incubator in order to assist in the evaporation of the alcohol and ethyl-acetate that the AHL's were suspended in </LI><LI>Once all components have evaporated the solid AHL's are resuspended in the original volume of ethyl-acetate that the dilutions were made in and placed on the vortex to ensure that the AHL's re-dissolve</LI><LI>Then the overnight induction plate is setup with the following layout</LI><LI>In the table below Well A4-A12 each have equal parts 150ul of the labeled product (i.e GFP+/-Sender supernatant) The wells were filled from 500ml stocks made for each different component. </LI><LI>For the wells from B1-F6 there was the following stocks made 500ul Receiver, 490 -Sender and 10ul AHL) </LI><LI>The wells themselves received 300 ul out of the stocks made from each concentration </LI><LI>For wells B1-C3 the 'U' stands for Untreated (AHLs were not treated with the alcohol), 
Wells C4-D12 the 'I' stands for Isopropanol and cells D13-F-6 the 'E' stands for ethyl alcohol (EtOH). </LI><LI>The well plate is then placed in the appropriate imaging machine and OD600 and GFP expression is measured over a 8-16 hr period depending on the length of data collection needed. </LI><H2>ESA Induction Plates Liquid/Plated</H2><H3>Wednesday, 7/19</H3><P>will be starting with 3 ul of solution for the liquid culture plates for each of the ESA/-REC/GFP+/2620</P><P>finale amount used was 9 ul </P><UL><LI>Liquid culture made @ 9:42 am </LI><LI>ESA Plated Culture: Plated at 10:08 am </LI><LI>-Sender Plate (was improvised) and plated at 10:30 am</LI><LI>Liquid culture checked @ 10:51 am</LI><LI>ESA Plated Culture: Checked at 11:08 am</LI><LI>-Sender Plate (was improvised) and Checked at 11:32 am</LI><LI>Liquid culture checked @ 11:51 am</LI><LI>ESA Plated Culture: Checked at 12:08 pm</LI><LI>-Sender Plate (was improvised) and Checked at 12:32 pm</LI><LI>Liquid culture checked @ 12:58 pm</LI><LI>ESA Plated Culture: Checked at 12:59 pm</LI><LI>-Sender Plate (was improvised) and Checked at 1:02 pm</LI><LI>Liquid culture checked @ 1:58 pm</LI><LI>ESA Plated Culture: Checked at 1:59 pm</LI><LI>-Sender Plate (was improvised) and Checked at 2:02 pm</LI><LI>Liquid culture checked @ 2:58 pm</LI><LI>ESA Plated Culture: Checked at 2:59 pm - Induction has been started (minimum 4-5 hrs for ESAI) </LI><LI>-Sender Plate (was improvised) and Checked at 3:08 pm</LI><LI>Liquid culture checked @ 3:58 pm</LI><LI>ESA Plated Culture: Checked at 3:59 pm</LI><LI>-Sender Plate (was improvised) and Checked at 4:08 pm</LI></UL><P>(initial plates need to be analyzed from an 18hr incubation time and distance migrated)</P><H3> Thursday, 7/20 </H3><P>Plates were checked and expression was seen in all components and all of the cells that can produce GFP. Though the GFP+ is still lacking expression compared to the GFP+ only plate. The Plated Culture and the negative Sender plates had the greatest amount of GFP+ though this is not of much concern as it is a cell that is simply there to verify that fluorescence is able to be seen. Since the 2620 is well characterized and expresses in each run the lack of GFP+ expression hasn't been much of an issue. Another phenomena that was noticed was the variation in the machine's exposure. I was using set exposure settings in the manual capture mode along with the UV 6 and Epi UV setting. it seems that the camera is still auto exposing to the brightest emitted light. This means that the relative exposure of all samples is dependent on the brightest object being photographed. So If I only have the GFP+ in the frame then the machine captures more of its detail and shows a (possible) different picture than when the GFP+ is imaged next to the 2620 which is marginally brighter than the GFP+</P><P>Today I will be investigating over time lapse the progression of GFP expression in the 2620. I will be plating ESAI as the sender at 1.5 inches away from the 2620 receiver as well as at .25 inches away from 2620 and placing them side by side under the imager if possible in order to time lapse both of their progressions. It is my aim to have a more uniform spread for this aspect of the experiment and I will be looking for time points of when expression begins and progresses towards its maximum. </P><UL><LI>Plated ESAI at 1:55 pm</LI><LI>Plated LuxI at 2:00 pm </LI><LI>Putting in Incubator at 2:20 pm till I leave the lab which should be around 5 pm. </LI><LI>There may be an issue with the plates as the 2620 is auto-inducing </LI><LI>Because 2620 was auto-inducing (and LUX was already induced.. possible auto-inducing also) I took another image at 5:22 pm and left the plates in the imaging device in order to gather 50 images over the course of 12.5 hours to see if the LUX/ESA/2620 will have a trend of induction at room temperature. While this will not show us the change from non-induced to induced and beyond... It will show us a possible trend of induction from where they started at 5:22pm on 7-20-17</LI></UL><H2>Induction Plates TraR, AubI, BjaI, CerI, LasI, LuxI, RpaI, EsaI</H2><H3> Sunday, 10, 1</H3>
These induction plates will have all senders with TraR (AubI, BjaI, CerI, LasI, LuxI, RpaI, TraI, EsaI) and one sender with LuxR (Bja).
 
These plates will be setup the same as previous induction plate (GFP+, -Rec, Rec) 

<H3> Saturday, 10/7 </H3><P>The induction plates were supposed to be setup on 10/1/17 but the TraR receiver, after transformation did not have enough viable colonies in order to make induction plates with 7 other receivers. Plates were setup on 10/6 and were left overnight to incubate at 37 degrees celsius since TraR is notorious for being slow to induce. </P><P>Unfortunately after 24 hours of imaging (1st image was at 12 hours after incubation and last image was at 24 hrs) there was little to no evidence of induction on the plates. This may be due to either the the Receiver being 5 days old or that the senders were 5 days old. It seems that upon refrigeration and after extended periods of time on Amp agar, many of the cells lose the ability to send or possibly receive signals via AHL's. This might be due to the fact that when the cells multiply there is somatic recombination and the genes that control the AHL's production just get folded out. There is also the possibility that during the initial transformation of the TraR and related senders the intended insert was not taken up by the new cells. </P><H1>Notebook: <INS>Brianna Lopez</INS></H1><H2>Transformation of Negative Sender and LuxR</H2><UL><LI>These colonies will be used for overnight liquid culture growth, which will be used in 96-well plating for overnight growth curves.</LI><LI>Transformation of a negative sender, and 2620 vector. </LI><LI>Concentration of 2620 &amp; Neg Sender:</LI></UL><P>-2620 = 135 ng/uL</P><P>-Neg Sender = 461 ng/uL (will have to do 1:10 dilution)</P><UL><LI>Calculation Calculation to get concentration of 60 ng/uL each:</LI><LI>Negative Sender: </LI></UL><P>- 461 ng/ul (1uL into 9uL H20) = 46.1 ng/uL </P><P>- 46.1 ng/uL * x uL = 60 ng </P><P><INS>- x = 1.3 uL </INS></P><UL><LI>2620: </LI></UL><P>- 135 ng/uL * x uL = 60 ng </P><P><INS> - x = .44 uL </INS></P><H3>Protocol:</H3><P>Materials:</P><UL><LI>Ice bucket </LI><LI>2620 </LI><LI>Negative Sender </LI><LI>DI H20 </LI></UL><H3>Dilution Procedure:</H3><OL type="1"><LI>One 1.5mL tube was used to perform a dilution, and labeled dilution tube.</LI><LI>9uL of DI H20 was pipetted into the dilution tube.</LI><LI>1uL of negative sender was taken out of stock and pipetted into the dilution tube.</LI><LI>The volume was then pipetted up and down to mix and dilute the solution. </LI></OL><H3>Transformation Procedure:</H3><OL type="1"><LI>One tube was labeled 2620, the other negative sender</LI><LI>1.3uL of negative sender was pipetted from the dilution tube into the negative sender tube</LI><LI>0.44uL of 2620 was pipetted from 2620 stock tube into the 2620 tube. </LI><LI>Tubes were then placed on ice for 2-5 minutes</LI><LI>The tubes were then placed on the hot plate of 42C for exactly 45 seconds then immediately placed on ice for 5 minutes. </LI><LI>300uL of SOC was added to each tube </LI><LI>It was then placed in the 37C shaking incubator for 20-30 minutes </LI><LI>The tubes were then spun down at 3000 rpm for 1 minute </LI><LI>The liquid in the tubes were gently tapped out into liquid waste</LI><LI>100uL of SOC was added to the tubes and gently pipetted up and down to resuspend the cells</LI><LI>100uL of each tube were then plated onto LB/AMP plates</LI><LI>Then placed in 37̊C still incubator over night </LI><H2>Serial Dilutions</H2><UL><LI>This is where the set range of dilution concentrations comes in handy. </LI><LI>The dilutions for these sets of ranges is a 1:10 dilution. So, a 1-part AHL and 9-part ethyl acetate solution. </LI><LI>No matter how large the volume of your dilutions, it will always be 1:10. 
</LI><LI>For this project, it is a 100uL volume, therefore 10uL will be AHL solution and 90uL ethyl acetate.</LI><LI>In some cases, the dilution mixture will be a 50uL volume with 5uL being AHL solution and 45uL will ethyl acetate. This is to conserve the longevity of the AHL when doing a single receiver at a time and plating with 2 AHLs. </LI><LI>For this particular example, however, a 100uL volume and its corresponding 1:10 dilution will be performed. </LI></UL><OL type="1"><LI>100uL of the 1*10^(-2)M stock solution is pipetted into a tube (for easy access and to prevent contamination of the stock solution)</LI><LI>10uL of the stock solution is pipetted into 90uL of Ethyl Acetate to bring the concentration down to 1*10^(-3)M </LI><LI>Mixing Each Tube:</LI></OL><OL type="A"><LI>Pipette up/down</LI><LI>Vortex</LI><LI>Spin down (using tabletop centrifuge)</LI><LI>Continue onto the next dilution</LI></OL><OL type="1"><LI>Steps 2- 3d will be repeated until the final lowest concentration is reached. </LI><LI>The dilutions will be set up as follows:</LI></OL><UL><LI>For half dilutions, i.e. 5*10^(-2), 5*10^(-3) it is the same concept except they are 1:1 dilutions instead of 1:10. </LI><LI>Two ways of doing this includes doing one 1:1 dilution with the stock solution, then doing a 1:10 dilution using the result from the 1:1 dilution. Or it can be done doing a 1:1 dilution for each individual concentration, i.e. 1*10^(-2), 1*10^(-3)</LI></UL><P>Note: When these solutions are plated, the concentration will be further diluted by 10^-2. For example, 1*10^-2M will be pipetted to its corresponding well on the well-plate, and the AHL final concentration on the well will be 1*10^-4M. This is where the 1*10^-4M to 1*10^-14M concentration range of AHL comes into effect. </P><H2>Dissolving AHL in Ethyl Acetate</H2><UL><LI>The first thing that must be done is to determine the molecular weight of the AHL, in this case the Lux AHL. </LI><LI>Usually the synthetic AHLs are purchased from Sigma-Aldrich and depending on the volume of what is bought from the company, the milligrams it comes in is important when calculating how much must be dissolved. </LI><LI>In the case of this project, the final stock concentration that was desired was 1*10^(-2)M. In this project, the highest AHL concentration desired was 1*10^(-4)M, but when plated the concentration is diluted down by 10^(-2). Therefore a starting concentrating of 1*10^(-2)M is beneficial. </LI><LI>Then Stoichiometry must be performed to determine the volume in which the AHL in milligrams will be dissolved in, then how much ethyl acetate must be added to get to the desired final stock concentration. </LI></UL><OL type="1"><LI>First is to determine the molecular weight and container size the AHL comes in. </LI><OL type="a"><LI>213.2 g/mol </LI><LI>10 milligrams of AHL in the container</LI></OL><LI>Then is to determine the desired stock concentration.</LI><OL type="a"><LI>This depends upon your range of concentrations, and how high the range is.</LI><LI>Range is: 1*10^(-4)M- 1*10^(-14)M </LI><LI>Therefore stock solution concentration: 1*10^(-2)M </LI></OL><LI>Then determine how large of a stock solution is needed</LI><OL type="a"><LI>Since it is difficult to measure out exact milligrams of the AHL and back into a container without losing any grams, a large enough stock solution would be the most beneficial and use the entire AHL in the container.</LI><LI>Desired Volume of Stock Solution: 500uL</LI></OL><LI>Determine Concentration of AHL contained in the stock volume</LI><OL type="a"><LI>Here is where stoichiometry comes into play. The milligrams per stock volume must be converted into standard SI molarity. </LI><LI>10mg/500μL * 1g/1*103mg * 1*106L/1μL * 1mol/213.2g = 0.094M </LI><LI>The concentration found is the stock solution concentration needed for dilutions. </LI><LI>Once done with the dilution to 1*10^-2 M, immediately put inside a -80C freezer for long term storage.</LI></OL><LI>With the concentration found above, it will be used in the C1V1=C2V2 equation. </LI><OL type="a"><LI>C1 will be the concentration found above. </LI><LI>V1 is the amount of AHL from stock solution into ethyl acetate solution</LI><LI>C2 is the concentration of the stock solution </LI><LI>V2 is the volume of the solution that is going to be further diluted and used in plating </LI><LI>(0.094M)(V1)=(50uL)(1e-2M)</LI><LI>V1=5.32uL AHL</LI></OL><LI>Subtract AHL volume from total volume</LI><OL type="a"><LI>This volume of AHL will be used to find the volume of ethyl acetate to dilute the AHL into a 1*10^-2M solution. </LI><LI>So, 500 – 5.32uL = 44.68uL of ethyl acetate</LI></OL><LI>Add these two amounts together in a 1.5mL centrifuge tube </LI><OL type="a"><LI>Mixing the Tube:</LI></OL><OL type="i"><LI>Pipette up/down</LI><LI>Vortex</LI><LI>Spin down (using tabletop centrifuge) </LI></OL><LI>Immediately place dilution in the -20C freezer for immediate use of plating or for further dilutions.</LI></OL><H1><INS>Notebook:Christina Smith </INS></H1><H2>Friday 9/22/17 - Running Gradient PCR on the BjaR</H2><P> Materials Used for this experiment </P><UL><LI> BjaR New Receiver G-Block </LI><LI> BioBrick prefix Forward Primer </LI><LI> BioBrick Suffix Reverse primer </LI><LI> Phusion Polymerase</LI><LI> dNTP 5 mM</LI><LI> HF Buffer (5x)</LI><LI> Distilled Water</LI></UL><P> There is no particular order to place these materials into an epindorph tube. However after these components are in the tube, needed to mix them thoroughly and spin them down. After, this mixture was divided into 8 PCR tubes with ~20 uL in each. A gradient PCR protocol was setup on the computer to run. The gradient annealing temperature would run from 70 ºC to 50 ºC. </P><P> Below is a chart to describe the particular quantities of the materials used: </P><TABLE style="width:100%" align="center"><TBODY><TR><TH>Materials</TH><TH>Quantity used in Lab’s Protocols (µL) </TH><TH>Quantity in Correct Ratio (µL) </TH></TR><TR><TD>BjaR New Receiver </TD><TD> 0.2</TD><TD> 1.6 </TD></TR><TR><TD>Forward primer </TD><TD> 0.2</TD><TD> 1.6 </TD></TR><TR><TD>Reverse Primer </TD><TD> 0.2</TD><TD> 1.6 </TD></TR><TR><TD>Distilled Water </TD><TD> 14.4</TD><TD> 115.2 </TD></TR><TR><TD>Phusion Polymerase </TD><TD> 0.2</TD><TD> 1.6 </TD></TR><TR><TD>dNTPs (5mM) </TD><TD> 0.8</TD><TD> 6.4 </TD></TR><TR><TD>HF Buffer (5x) </TD><TD> 4</TD><TD> 32 </TD></TR></TBODY></TABLE><P> This particular PCR ran for 1 hour and 16 minutes. After this, I needed to run a agarose gel to verify if the DNA amplified or not. Once the gel was made, I inputted a kb+1 DNA ladder into the first slot of the gel. I skipped the second slot and proceeded to input 2 µL 6X Loading Dye + 4 µL of PCR into the remaining 8 wells. The gel was ran at 110 V for 45 minutes. The following are the wells that correspond to the annealing temperature</P><P><UL><LI>A: 70 ºC</LI><LI>B: 66.8 ºC </LI><LI>C: 64 ºC </LI><LI>D: 61.2 ºC</LI><LI>E: 58.4 ºC</LI><LI>F: 55.6 ºC</LI><LI>G: 52.8 ºC</LI><LI>H: 50ºC </LI></UL></P><P> This is the process to make an agarose gel: </P><UL><LI> Weigh 0.6 grams of Agarose powder and place it into a 100 mL volumetric flask. </LI><LI> Next, add 60 mL of TAE to the powder and place into microwave. </LI><LI> Microwave for 1 minute. Take out the flask using heat protective gloves and swirl the flask to dissolve the particles </LI><LI> Place the flask back into the microwave and run it for another 8 seconds </LI><LI> Using heat protective gloves, take the flask out of microwave, swirl and add 6 µL of Syber Safe DNA Stain </LI><LI> Pour liquid into gel mold with small dividers and let it sit for 15 minutes or until opaque </LI></UL><H2>Tuesday 9/26/17 - PCR Cleanup of BjaR</H2><H3> PCR Cleanup </H3><P> The volume of this PCR is 128 uL. The following is the protocol for PCR cleanup in lab. </P><UL><H5> Binding</H5><LI> Add 5x Volume of PB </LI><LI> Add 1/10 X Volume of PCR results of NaAc pH 5.3 </LI><LI> Spin through PCR Cleanup tube at maximum speed for 1 minute and dump the flow through</LI><H5> Wash</H5><LI> Add 750 uL of PE wash solution </LI><LI> Spin through column at maximum speed for 1 minute </LI><LI> Dump the flow through liquid into waster container </LI><H5> Spin Again </H5><LI> Spin at maximum speed for 1 minute </LI><LI> Dump flow through liquid </LI><H5> Transfer PCR Column to a new epindorph tube </H5><LI> Add 35 uL of distilled water and let it sit for 1 minutes </LI><LI> Spin to elute at maximum speed for 1 minute </LI><H5> Test the Concentration in (ng/uL) </H5><P> The concentration of BjaR was 123.12 ng/uL </P></UL><H2> Wednesday 9/27/17 - Double Digest of BjaR with EcoRI and PtsI </H2><P> The following quantities were added to a edpindorph tube, mixed and spun down. They were then placed in a 37 degree incubator for 45 minutes. The order I placed them into the tube are as followed: 
<UL><LI>  Distilled Water </LI><LI>  10x Fast Digest Buffer + Green Dye </LI><LI>  BajR New Receiver from PCR Cleanup  </LI><LI>  EcoRI Restrictive Enzyme </LI><LI>  PtsI Restrictive Enzyme </LI><TABLE style="width:100%" align="center"><TBODY><TR><TH>Materials</TH><TH>Quantity (µL) </TH></TR><TR><TD>BjaR New Receiver </TD><TD> 7.3 μL</TD></TR><TR><TD>10x Fast Digest Buffer + Green Dye</TD><TD>  3 μL</TD></TR><TR><TD>Distilled Water </TD><TD>  17.7 μL</TD></TR><TR><TD>EcoRI Restrictive Enzyme</TD><TD> 1 μL</TD></TR><TR><TD>PtsI Restrictive Enzyme</TD><TD> 1 μL</TD></TR></TBODY></TABLE></UL></P><P> After the digest was completed, I ran an agarose gel to confirm the digestion. The results are displayed below. I realized after running the gel that it would be very hard to confirm whether or not the digest was successful. This is because the size that I digested out is less than 100 nucleotide difference between the original PCR cleanup and the digested version. The next step I need to take is to gel extract the top band, ligate it and then transform it in DH5(alpha) turbo cells. </P><H1><INS> Notebook: Xylaan Livingstone </INS></H1><H3>July 31st , 2017</H3><H3><INS>Mini Prep for backbone:</INS></H3><UL><LI>3 15 mL tubes of the LB Cor C</LI><LI>3 15 mL tubes of LB AMP A3</LI></UL><OL type="1"><LI>Centrifuge down for pellet at highest speeds</LI><LI>Each tube, I used 200 uL of resuspension fluid and transfer each to 2 mL tubes</LI><LI>Each tube added lysis (200 uL), invert the tubes 5 times and immediately added 350 ul of neutralization buffer</LI><LI>Cap each tube and centrifuge for 10 minutes at max speeds </LI><LI>While centrifuging, I got 6 microfuge tubes for column prep, Put 6 columns in each pipet 500 uL of column prep into each</LI><LI>Transfer the lysate from the 10-minute centrifuge to the labeled columns, then centrifuge the flow through </LI><LI>Dump the flow through after </LI></OL><P>~Proceeded with the same measurements from sigma mini prep</P><P>~ Error with the elution step, after water flow through on the last step. 2 PJ1A3 tubes DNA elution was poured out on accident, so a lower concentration for two b was expected </P><H3>Mini Preps Concentration</H3><H4>1st Concentration results</H4><TABLE style="width:100%"><TBODY><TR><TH>PSB1A3 </TH><TH>PSB1C3</TH></TR><TR><TD>16.068  ng/uL</TD><TD>6.024  ng/uL</TD></TR><TR><TD>41.607 ng/uL</TD><TD>40.571  ng/uL</TD></TR><TR><TD>12.99 ng/uL</TD><TD>7.578 ng/uL</TD></TR></TBODY></TABLE><H2>August 1, 2017</H2><P>Yesterday’s concentration created low DNA concentration. Thus, we are doing a second batch.</P><P>2mL of PSB1C3 </P><P>2mL of PS1A3 A3 </P><UL><LI>All have fairly the same expression/ Approximately</LI><LI>Bl21 bacteria for both batches </LI><LI>Ran same mini prep procedure as yesterday, as well found in the Sigma mini prep procedure </LI><P>~Side note: it’s possible I am getting a lot of expression but the bacteria is not taking the plasmid </P><H3>2nd Concentrations Results:</H3><TABLE style="width:100%"><TBODY><TR><TH>PSB1A3</TH><TH>PSB1C3</TH></TR><TR><TD>16.158 ng/uL</TD><TD>31.964 ng/uL</TD></TR><TR><TD>37.435 ng/uL</TD><TD>14.273  ng/uL</TD></TR></TBODY></TABLE><P>Will talk to graduate/ higher up to see a different result or possible error in steps of mini prep.</P><H2>August 1st, 2017</H2><P>Liquid Culture of </P><UL><LI>3ml Cor C3</LI><LI>3mL Amp A3</LI></UL><P>New liquid cultures overnight, two different tubes.</P><UL><LI>PSB1A3 in AMP</LI><LI>PSB1C3 in Cor </LI><LI>Left in the incubator at 3:21 p.m.</LI><LI>Coming in at 10:00 a.m. to finish </LI></UL><H2>August 2nd, 2017</H2><UL><LI>Took liquid culture out at 12:45 p.m. </LI><LI>Ran mini-prep procedure. Instead of using neutralization in fridge, I am using a bottle at room temperature </LI><LI>Lysis/ neutralization </LI><LI>Accidently put 550 uL of lysis solution within tubes to counter balance mistake, I used 700 uL of neutralization solution to make up amount. </LI></UL><H3> Concentration results: </H3><TABLE style="width:100%"><TBODY><TR><TH>PSB1A3</TH><TH>PSB1C3</TH></TR><TR><TD>122.576 ng/uL</TD><TD>98.408 ng/uL</TD></TR></TBODY></TABLE><P>Placed in freezer 2:20 P.M.</P></UL></OL></OL></DIV></BODY></HTML>