<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Arizona_State_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Arizona State/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_menu_wrapper"><DIV class="menu_button" id="display_menu_control">
		MENU  
	</DIV><DIV id="menu_content"><A href="https://2017.igem.org/Team:Arizona_State"><DIV class="menu_button direct_to_page">
				HOME
			</DIV></A><DIV class="menu_button"> TEAM 
		</DIV><DIV class="submenu_wrapper" id="team_submenu"><A href="https://2017.igem.org/Team:Arizona_State/Team"><DIV class="submenu_button" id="Team_page">
					Team 
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Collaborations"><DIV class="submenu_button" id="Collaborations_page">
					Collaborations
				</DIV></A></DIV><DIV class="menu_button"> PROJECT
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Arizona_State/Description"><DIV class="submenu_button" id="Description_page">
					Description
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Design"><DIV class="submenu_button" id="Design_page">
					Design
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Experiments"><DIV class="submenu_button" id="Experiments_page">
					Experiments
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Notebook"><DIV class="submenu_button" id="Notebook_page">
					Notebook
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Results"><DIV class="submenu_button" id="Results_page">
					Results
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Demonstrate"><DIV class="submenu_button" id="Demonstrate_page">
					Demonstrate
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Improve"><DIV class="submenu_button" id="Improve_page">
					Improve
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Attributions"><DIV class="submenu_button" id="Attributions_page">
					Attributions 
				</DIV></A></DIV><DIV class="menu_button"> PARTS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Arizona_State/Parts"><DIV class="submenu_button" id="Parts_page">
					Parts
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Basic_Part"><DIV class="submenu_button" id="Basic_Part_page">
					Basic Parts
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/Part_Collection"><DIV class="submenu_button" id="Part_Collection_page">
					Part Collection
				</DIV></A></DIV><A href="https://2017.igem.org/Team:Arizona_State/Safety"><DIV class="menu_button direct_to_page">
				SAFETY
			</DIV></A><DIV class="menu_button"> HUMAN PRACTICES 
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:Arizona_State/HP/Silver"><DIV class="submenu_button" id="Silver_page">
					Silver HP
				</DIV></A><A href="https://2017.igem.org/Team:Arizona_State/HP/Gold_Integrated"><DIV class="submenu_button" id="Gold_Integrated_page">
					Integrated and Gold
				</DIV></A></DIV><DIV class="menu_button"> AWARDS
		</DIV></DIV><A href="https://2017.igem.org/Team:Arizona_State/Measurement"><DIV class="submenu_button" id="Measurement_page">
					Measurement
				</DIV></A></DIV></DIV></DIV></DIV><A href="https://igem.org/2017_Judging_Form?team=Arizona_State"><DIV class="menu_button direct_to_page">
				JUDGING FORM
			</DIV></A></DIV></DIV><DIV class="igem_2017_content_wrapper"><H1>Arizona_State</H1><DIV class="column full_size"><H1>Protocols</H1><H2><INS>Christina Smith </INS></H2><H2>Ligation Protocol with T4 DNA Ligase</H2><H3><INS>Materials:</INS></H3><UL><LI>10X T4 DNA Ligase Reaction Buffer</LI><LI>T4 DNA Ligase</LI><LI>MRV Vector DNA</LI><LI>Receiver Insert DNA </LI><LI>Nuclease-free water </LI><H3>Procedure </H3><P><I>*Note:</I> T4 DNA Ligase should be added last. The table shows a ligation using a molar ratio of 1:3 vector to insert for the indicated DNA sizes. Use NEB calculator to calculate molar ratios.</P><OL type="1"><LI>Thaw the T4 DNA Ligase Buffer and resuspend at room temperature.</LI><LI> Set up the following reaction in a microcentrifuge tube on ice: </LI><TABLE style="width:100%"><TBODY><TR><TH> Component </TH><TH> Volume (uL)</TH></TR><TR><TD>10X T4 DNA Ligase Buffer </TD><TD>2</TD></TR><TR><TD>Vector DNA: 50 ng</TD><TD>X</TD></TR><TR><TD> Insert DNA : 37.5 ng </TD><TD>X</TD></TR><TR><TD> Nuclease Free Water </TD><TD>17</TD></TR><TR><TD> T4 DNA Ligase</TD><TD>1</TD></TR><TR><TD> Total</TD><TD>20</TD></TR></TBODY></TABLE><LI>Gently mix the reaction by pipetting up and down and microfuge briefly </LI><LI>For cohesive sticky ends, incubate at 16 C overnight or room temperature for 10 minutes. For blunt ends or single base overhangs, incubate at 1 C overnight or room temperature for 2 hours </LI><LI>Heat inactivate at 65 C for 10 minutes </LI><LI>Chill on ice and transform 1-5 uL of the reaction into 50 uL of competent cells</LI></OL><H2>Vector Digest Protocol for Modular Receiver Vector</H2><H3>Materials</H3><UL><LI>Vector DNA </LI><LI>10x Fast Digest Buffer + Green Dye </LI><LI>Restriction Enzymes </LI><LI>BbsI</LI><LI>SpeI</LI><LI>Deionized Water </LI></UL><H3>Procedure</H3><OL type="1"><LI>Set up the following reaction using the restriction endonuclease that has the lowest salt concentration in its recommended buffer (total reaction volume 50 µl). </LI><LI>Set up the following digest reaction on ice: </LI></OL>
table, th, td {
    border: 1px solid black;
    border-collapse: collapse;
}
<TABLE style="width:100%"><TBODY><TR><TH>Components </TH><TH>Volume (µL)</TH></TR><TR><TD> dH2O </TD><TD>25</TD></TR><TR><TD>10X Fast Digest Buffer + Green Dye </TD><TD>3</TD></TR><TR><TD> MRV Vector </TD><TD>10</TD></TR><TR><TD> BbsI </TD><TD>1</TD></TR><TR><TD> SpeI </TD><TD>1</TD></TR></TBODY></TABLE><OL type="1"><LI>Mix components by pipetting the reaction mixture up and down, or by &quot;flicking&quot; the reaction tube. </LI><LI>Quick (&quot;touch&quot;) spin-down in a microcentrifuge. Do not vortex the reaction. </LI><LI>Incubate </LI></OL><H2><INS>Brianna Lopez</INS></H2><H2>Plate Reader Protocol for Synthetic AHL Liquid Culture Inductions</H2><P>Purpose: To set up and run a plate of synthetic AHL liquid culture inductions for overnight growth curves in a plate reader. </P><P><B>Materials: </B></P><UL><LI>Use of 1 or 2 separate synthetic AHLs, in this example Las AHL and Lux AHL. </LI><LI>Use of 1 receiver that will interact with the synthetic AHLs, in this example LuxR.</LI><LI>Controls that will need to be transformed into colonies on LB/AMP plates:</LI>
o	Negative Sender (negative control for synthetic AHL)
o	Receiver
o	GFP (positive control)
o	Negative Receiver (negative control for the receiver)
o	Sender Supernatant (in this example, LuxI supernatant from the natural AHL that a sender cell produces, is a positive control for the receiver to show it works)
</UL><P><B>Procedure:</B></P><P>For plating purposes in this example, Negative Receiver, GFP Postive, and Reciver liquid cultures must have a final OD600 of 0.4 once it enters the plate reader for overnight growth curve readings. This specific OD600 value can be set depending on what range on the curve is needing analysis. </P><P>Day 1: </P><OL type="1"><LI>Make liquid cultures of LuxR, Negative Sender, Negative Receiver, Positive GFP, and Receiver cells. </LI><LI>In 15mL culture tubes for each individual sample, put 3-4 mL of LB/AMP broth into the tubes. </LI><LI>Then picking colonies off of the transformed plates of LuxR, Negative Sender, Negative Receiver, Positive GFP, and Receiver cells into their corresponding culture tube containing LB/AMP broth.</LI><LI>Make sure there is enough liquid culture for the following day of plating, taking into account the evaporation of LB/AMP broth. </LI><LI>Place culture tubes into a shaking incubator at 37̊C for overnight growth.</LI></OL><P>Day 2:</P><OL type="1"><LI>Take samples out of the shaking incubator as they have grown overnight.</LI><LI>Take another culture tube and using the culture tube of receiver grown the previous night, reseed in fresh LB/AMP inside the culture tube. </LI><LI>This reseeding culture contains 10% overnight culture and the remaining volume containing LB/AMP. </LI><LI>Leave this to regrow for 5 hours before plating. </LI><LI>Check OD600 for the LuxR (receiver), Negative Receiver, and Positive GFP samples with an LB/AMP well as a control comparison. </LI><LI>These OD600 readings should be between 0.4 and 0.6 once put into the plate reader for curve readings at the end of day. </LI><LI>Then filter senders from cells in the culture tubes into supernatant. These senders include Negative Sender, and LuxI or the positive sender control for the receiver. </LI></OL><OL type="a"><LI>Begin by centrifuging the culture tubes until cells are at the bottom, and the remaining supernatant is of similar color to LB.</LI><LI>Filter this remaining supernatant through a syringe with a filter at the bottom into a new tube and label the type of sender. </LI><LI>Do this for both senders. </LI></OL><OL type="1"><LI>Begin plating all samples into 96-well plate. </LI><LI>Plate Reader Protocol: </LI></OL><OL type="a"><LI>Go to Protocol &gt; Procedure &gt; Check Temperature is set to 37̊C, Check A(600) and F(485,515) are set &gt; Click Flourescence &gt; Gain &gt; Options &gt; Make sure the automatic Gain Adjustment is set properly &gt; scale to low wells &gt; scale wells (this is where the three Negative Sender and Receiver wells are on the plate) &gt; scale value 200 </LI></OL><OL type="1"><LI>Save this plate reader data to place of choice on the computer, and begin to run the plate for overnight growth curves. </LI></OL><H2> Mini-Prep Protocol </H2><H4> Harvest Cell : </H4><P> •	Pellet 1-5 mL of an overnight recombinant E. coli culture by centrifugation. 


-The optimal volume of culture to use depends upon the plasmid and culture density.
 <UL><LI>For best yields for E. coli grown in LB: </LI><LI>use 1-3 mL of culture for high copy plasmids</LI><LI>use 1-5 mL of culture for low copy plasmids </LI><LI>	for best yields for E. coli grown in TB or 2X YT: </LI><LI>	use only 1 mL of culture </LI></UL>
1.	Resuspend Cells (verify that appropriate volume RNase A solution was added to the Resuspension Solution
<OL type="a"><LI>	Completely resuspend the bacterial pellet with 200 µL of the Resuspension Solution. </LI><LI>	Vortex or pipette up and down to thoroughly resuspend the cells until homogenous </LI></OL>
2.	Lyse Cells
<OL type="a"><LI>	Lyse the resuspended cells by adding 200 µL of the Lysis Solution </LI><LI>	Mix the contents by gentle inversion (6-8 times) until the mixture becomes clear and viscous (Don’t vortex) </LI><LI>	Do not allow the lysis reaction to exceed 5 minutes </LI></OL>
3.	Neutralize
<OL type="a"><LI>	Precipitate the cell debris by adding 350 µL of the Neutralization/Binding Solution </LI><LI>	Gently invert the tube 4-6 times </LI><LI>	Pellet the cell debris by centrifuging at ≥ 12,000 x g or maximum speed for 10 minutes </LI><LI>	Cell debris, proteins, lipids, SDS, and chromosomal DNA should fall out as a cloudy, viscous precipitate. </LI><LI>	If the supernatant contains a large amount of floating particulates after centrifugation, re-centrifuge the supernatant </LI></OL>
4.	Prepare Column
<OL type="a"><LI>	Insert a GenElute Miniprep Binding Column into a provided micro-centrifuge tube, if not already assembled </LI><LI>	Add 500 µL of the Column Preparation Solution to each miniprep column and centrifuge at ≥ 12,000 x g for 30 seconds to 1 minute </LI><LI>	Discard the flow-through liquid </LI></OL>
5.	Load cleared lysate
<OL type="a"><LI>	Transfer the cleared lysate from step 3 to the column prepared in step 4 and centrifuge at ≥ 12,000 x g for 30 seconds to 1 minute. </LI><LI>	Discard the flow-through liquid </LI></OL>
6.	Wash Column (verify that ethanol has been added to the bottle of Wash Solution 2)
<OL type="a"><LI>	Add 750 µL of the diluted Wash Solution to the column </LI><LI>	Centrifuge at ≥ 12,000 x g for 30 seconds to 1 minute </LI><LI>	Discard the flow-through liquid and centrifuge again at maximum speed for 1-2 minutes without any additional Wash Solution to remove excess ethanol </LI></OL>
7.	Elute DNA
<OL type="a"><LI>	Transfer the column to a fresh collection tube. Add 100 µL of Elution Solution or molecular biology reagent water to the column </LI><LI>	For DNA sequencing and other enzymatic application, use water or 5 Mm Tris-HCL, Ph 8.0, as an eluant </LI><LI>	Centrifuge at ≥ 12,000 x g for 1 minute </LI><LI>	Use immediately or store at -20˚C </LI></OL></P><H2><INS> Amber Mani </INS></H2><H2>Plate reader protocol with Data Analysis for Liquid Culture Inductions </H2><P>How to set up and run the plate reader for overnight growth curves. </P><H3>Materials:</H3><UL><LI> (Your unique set of senders, receivers etc. that you want for testing) In this example set up, Lux I, Esa I, and RpaI were all used in BL21 and are shown to produce AHL </LI><LI>(Controls) Negative sender- missing the gene for synthase that produces AHL, used for
control purposes </LI><LI>Receiver</LI><LI>GFP (+)</LI><LI>(-) Receiver </LI></UL><H4>Procedure:</H4><H3>Done over 2 days and plate reader protocol at the end: </H3><P>You will want to know the concentrations of the samples you use so that you can see, based on the growth curves, which concentrations will yield the best results. In one example of the growth curve tests, we used concentrations of .05, .1, .2, .3, .5. after determining the OD600 concentrations. the actual growth reader will be on total volumes. We want our final OD600 to be 0.4.</P><H3>Day 1</H3><P>Make liquid cultures of Lux I, EsaI, RpaI, (-)sender in LB/Amp
3- 5 mL of LB with plated senders
<INS>want bacteria in the stationary phase of bacterial growth to maximize AHL</INS>
Make liquid culture of 3 receivers: F2620, GFP(+), and (-) receiver
3-5 mL of LB with plated senders </P><H3>Day 2</H3><P>Make another culture using the starter cultures and reseed them
get 5 mL of fresh LB/Amp media and make a 10% seed culture
5mL of LB with 600 uL of grown start
<INS>grow for 5 hours
wanting receivers at exponential phase of growth
can check this by checking their OD reading </INS>
between 0.4 and 0.6
GEN5&gt;protocol of OD600&gt; choose H8-11, H8 being LB well
Filter senders: (18, 19, 20 &amp; 21:)
put in Big centrifuge
start 1 min at 1 RPM, then 2nd minute at 2 RPM, then 8 minutes at 3.6 RPM
extract the supernatant and put in syringe with a filter on the bottom, filer it into a tube with a
screw tab on it and label it
AHL stable at room temperature or in the freezer in the back room with all the plates
</P><H3><INS>Plate Reader Protocol - check settings as these are examples from a previous experiment</INS></H3><H3><P>measures relative GFP fluorescence in unit of RFU
ReneRynaJiagi, overnight, OD GFP
save experiment to your folder
fix layout to say what rows you want to work with
go to side tab
protocol&gt;procedure&gt; make sure at 37C, A(600), F( 485,515), click on fluroescence &gt; Gain &gt;
options &gt; make sure setup at Automatic gain adjustment &gt; scale to low wells &gt; scale wells A1A3
&gt; scale value 200 </P></H3><H3>To analyze the data collected from the plate reader</H3><OL type="1"><LI> Export the data to Excel from the plate reader, graph the data to view the OD and GFP expression curves and pick out the data that stands out. </LI><LI> Take the triplicates and use excel to get averages for each triplicate of data. </LI><LI>Use the averaged OD and GFP figures to graph GFP over OD to get the normalization
curves. </LI><LI>Compare the graphs to see what data stands out, searching for orthogonality in
combinations of senders (low GFP expression) meaning low interference and possible
orthogonality. </LI><LI>Once you select the 'interesting' data, add error bars (standard deviations) to see the
success of the samples. </LI><LI>Use the Hill equation to re-graph the data in a more professional display. </LI></OL><H2>Plasmid transformation</H2><H3>Introduction</H3><P>How to insert a plasmid into bacteria to transform it.</P><H3>Materials</H3><UL><LI>Get how ever many 50ul tubes of the cells you need to to transform (in our case it was BL21 e. coli)</LI><LI>Get the plasmid tubes you need, for insertion into the cells. </LI><LI> Gather PCR tubes to be used for Plasmid DNA dilution, same number of tubes as cells and plasmids.</LI><LI>Keep everything on ice during preparation. </LI>
›
<H3>Procedure</H3><P>Calculate the concentrations and transform the cells</P><OL type="1"><LI> check ng/ul concentrations of each plasmid DNA - want to add about 60ng of DNA to each of the cell tubes. </LI><LI>Dilute each DNA sample in PCR tube with water and do the math to know how many ul to add of the new dilution to get appx 60ng of DNA for each of the 50ul cell tubes. </LI><LI>Add the appropriate amount of the plasmid needed into each cell tube. </LI><LI> Place cell tubes (with plasmids added) on ice for about 2-5 minutes </LI><LI>Then place tubes on hot plate (42C) for appx 45 seconds </LI><LI> Finally place cell tubes back on ice for 5 minutes. Suspend and grow newly transformed cells </LI><LI>Add SOC (300ul) to each cell tube</LI><LI>Place in shaking incubator (37C) for 30 minutes </LI><LI> Spin down tubes, max speed for one minute</LI><LI>Gently tap out liquid and resuspend cells in 100ul SOC </LI><LI>Plate all 100ul onto petri plate (in our case, AMP plate) and place in regular incubator overnight</LI><LI>The next day you have access to the freshly transformed, plated and grown cells</LI></OL><H2>DNA Gel Extraction from Agarose Gel</H2><H3>Introduction: </H3><P>How to extract specific DNA fragments from an agarose gel.</P><H3>Materials:</H3><UL><LI>Sigma GenElute Gel Extraction Kit with included materials:</LI><LI>15ml culture tubes</LI><LI>Scalpel</LI><LI>KB+ Ladder </LI><LI>GenElute Binding Column G tubes &amp; 2ml collection tubes</LI><H3>Procedure:</H3><P>*************1st do restriction digest protocol and run the agarose gel**************</P><OL type="1"><LI>Use the sample of cells you wish to extract DNA from AFTER completing the protocol for restriction digest on them. </LI><LI> Run the agarose gel with the digested DNA for 90 minutes at 90 V. </LI><LI>Use the appropriate sequencing map to 'map' out the length of the strand you want to
extract by finding the 1st restriction enzyme used and measuring the length of the pairs
to the 2nd restriction enzyme. Compare to the KB+ Ladder (#SM1331) and the plasmid
guide. </LI><LI>Once gel has run use a gel imager to view the DNA strand lengths and identify the
proper section of DNA you need to extract.
Removing DNA band </LI><LI> While gel is on an imager (so you can use UV light to see the bands of DNA) excise the
DNA fragment of interest from the agarose gel with a clean, sharp scalpel or razor blade.
Trim excess gel away to minimize the amount of agarose. </LI><LI>Place gel into a culture tube and weigh the gel - remember to 1st weigh an empty culture
tube and 'zero' the scale so the weight of the tube is not included. </LI><LI>Solubilize gel is needed for this step - included in the GenElute kit. Add 3 gel volumes of the gel solubilization solution to the gel tube. (for every 100mg of agarose gel add 300ul
of the gel solubilization solution). </LI><LI> Incubate the gel mixture at 50-60C for 10 minutes. As needed for total gel dissolution,
vortex briefly every 2-3 minutes to aid in gel dissolution. </LI><LI>Prepare the binding column: place GenElute binding column G into one of the provided 2ml collection tubes. Add 500ul of the column preparation solution to each of the binding columns, centrifuge for one minute and discard the flow through liquid. </LI><LI>Next go back to the dissolved gel and check the color, should be a yellowish color. </LI><LI>Add isopropanol: add 1 gel volume of 100% isopropanol and mix completely. </LI><LI> Bind the DNA: load the solubilized gel solution mixture from the culture tube into the
binding column (may turn red at this point, this is normal). Centrifuge for one minute and
discard the flow through liquid. </LI><LI>Wash column: add 700ul of wash solution (diluted from wash solution concentrate G as
described under preparation instructions in manual). Centrifuge for one minute. Remove
the binding column and discard the flow through liquid. replace binding column and
centrifuge for one more minute and discard the flow through liquid again. </LI><H3>Elute DNA: </H3><LI> Transfer the binding column to a fresh collection tube and add 50ul of the (preheated to 65C) Elution Solution to the center of the membrane and incubate for one minute. Then
centrifuge for one minute. What remains in the collection tube can be tested for DNA
concentration and kept as the collection tube now contains the extracted DNA segments. </LI></OL><H2>Colony PCR and Colony Prep</H2><P>Materials and protocol for running the colony PCR. PCR is used to reproduce (amplify) selected sections of DNA or RNA for analysis. </P><H3>Materials:</H3><UL><LI> 2x Master Mix </LI><LI>Forward and reverse primers </LI><LI> DNA template that will be used</LI><LI>Water </LI><LI>Agar plate(s) </LI></UL><P>Procedure:</P><H3>Follow these steps: </H3><OL type="1"><LI> Defrost template, 2x Master Mix and primers </LI><LI>Determine desired total working volume (~20-25uL per PCR tube)</LI><LI> Label reaction tubes</LI><LI>Create below reaction mix:</LI><LI>Use this template for the amounts and concentrations to use for making the
master mix</LI></OL><H3>Next steps after you have the newly made Master Mix: </H3><OL type="1"><LI> Pipette 20 μl of newly mixed green solution into each PCR tube </LI><LI>Colony Prep: to grow more of the sample that you took a colony from for
the colony PCR. Can be used to have backup plates when you get a
positive result from PCR gel. </LI><LI>Scraped one colony at a time from the existing plate and gently placed the
colonly onto the new plate with the labeled spots. Disposed of each tip used into
the cooresponding PCR tube labeled to match the placement number on the new
plate. </LI><LI>Used empty pipette to attach to each tip in the PCR tubes and pump up and
down to mix and remove as much of the leftover bacteria from the tip as possible
so it mixed into the PCR tube. Then disposed of the tips in dry waste.</LI><LI> Place the new plates in the incubator and remove the next day to freeze.</LI><LI>Place PCR tubes with the new DNA samples in thermal cycler using the
appropriate settings </LI></OL><H2> Growing samples in culture tubes for growth curves </H2><H3>This is used for growing up colony samples overnight for use in the plate reader. </H3><P>
Materials:
<UL><LI>15ml culture tubes </LI><LI>LB (AMP or CHLOR) </LI><LI>Grown colonies ready to use on petri dishes </LI><LI>going to use micropipette to grab one colony at a time and add to LB in culture tubes </LI></UL></P><H3> Procedure: </H3><H3>Collecting colonies and prepping the culture tubes </H3><H4>**Best to do this in the evening so the cultures don't grow too long**</H4><UL><LI>Label each culture tube with the contents, date and your name </LI><LI>Add LB (AMP or CHLOR) to the culture tube. (general rule for how much LB - use about 1/4 the container volume. If tube is 15ml use about 3-4ml LB) </LI><LI>Take pipette and use a clean tip to collect ONE colony from the plate. Discard the tip INTO the culture tube with the colony on the tip. </LI><LI>Place the lid on the culture tube (not too tight so there is room for gas exchange) </LI><LI>Place the culture tubes into the shaking (37F) incubator for overnight growth. </LI><LI>In the morning you can use the Synergy plate reader to check the OD of the sample and dilute or add fresh LB as needed for the test you are running. </LI></UL><H2> Making and running a gel </H2><H4> What you need and how much of it to make and run an agarose gel </H4><H3> Materials: </H3><UL><LI> * 60 ml of TAE liquid into the flask for gel making. </LI><LI> * 0.6g of the agarose powder to the flask and swirled </LI><LI>* 6.0 μl of the Syber Safe DNA gel stain to the flask and swirled </LI></UL><H3> Procedure: </H3><H3> How to make gel using ingredients </H3><H4> Gel making/running protocol used: </H4><OL type="1"><LI> Started with adding 60 ml of TAE liquid into the flask for gel making. </LI><LI> Add 0.6g of the agarose powder to the flask and swirled </LI><LI> Use the microwave to heat the mixture (1 min, swirl and repeat) till all pieces
dissolved and was a pure clear liquid </LI><LI> Add 6 μl of the Syber Safe DNA gel stain to the flask and swirled </LI><LI> Carefully poured the mixture into a blank gel tray with spacers in place. </LI><LI> Once the gel is cooled, add the KB+ DNA ladder to the first slot, 2nd slot is left
blank and each following slot is to be loaded with 5 μl of each sample. </LI><LI> Once the slots are filled place the gel (still inside the plastic gel holder) into the
Galileo/ BioRad electric current machine. Run for approximately 45 minutes at
110V. </LI><LI> To image the gel once finished, use the SynGene gel imager machine. </LI></OL><H2><INS>Chris Connot </INS></H2><H2>Plate Induction &amp; Imaging Protocol</H2><H3>Introduction</H3><P>This protocol will lead you through the steps of setting up induction plates for Sender/Receiver combos as well as the imaging of the agar plate. </P><H3>Materials</H3><UL><LI>Agar plates</LI><LI>Sender/receiver bacteria on plates (suggested to be freshly transformed) </LI><LI>Positive/negative controls (GFP+/-Receiver) </LI><LI>MicroPipetter (05 ul to 200ul) </LI><LI>Pipette Tips </LI><LI>GeneSys Machine </LI></UL>
o	
<H3>Procedure</H3><B>Induction Plate Setup</B><OL type="1"><LI>Procure agar plates and warm to 37℃. </LI><LI>Procure plates with desired bacteria (senders,receivers,positive/negative control)</LI><LI>design/print out template (fig. 1) for induction test</LI><B>Fig 1: Induction Plate Layout</B><LI>Using a micropipettor and a sterile tip, bend the tip on the back of the lid of the agar plate </LI><B>Fig 2: Bent Tip Method</B><LI>Use the bent tip to scrape bacteria from any of the required plates (sender/receiver, pos/neg controls) </LI><LI>Paint the bacteria (liberally) on the plate in the designated sections. </LI><LI>Be sure to add enough bacteria so that there is a semi-thick layer that is continuous throughout the designated area. </LI><H3>Imaging the Induction plates </H3><LI>Put magnetic cover on top of UV light ( will have to adjust placement after imaging) </LI><LI>Take off Lid of Culture plate </LI><B>Fig 3: Prepping Culture Plate for imaging </B><B>Fig 4: SynGene Settings </B><LI>Place culture plate on top of magnetic cover and close the tray. (Fig. 3)</LI><LI>Machine Settings: (Fig. 4) </LI><UL><LI>Blots: Fluorescent Blot</LI><LI>Alexa 488</LI><LI>Epi Mid Wave UV</LI><LI>UV06</LI></UL><LI>Press Green arrow once machine is done calibrating</LI><LI>Press Capture </LI><LI>Save as .sgd in order to have the original data still available to be able to save as .tif </LI><LI>If 3D imaging is needed: </LI><UL><LI>click edit on the lower right side of the screen and click 3D on following screen (upper left)</LI></UL><LI>If annotations are needed</LI><UL><LI>proceed to the edit screen </LI><LI>click on annotate option in upper left hand corner</LI><LI>then place text over parts of the image you want to be annotated </LI><LI>save image in preferred format &quot;as displayed&quot; to save annotations. </LI></UL></OL></UL></UL></UL></DIV></DIV></BODY></HTML>