<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Evry_Software_Predictions skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Evry/Software/Predictions</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="row"><DIV class="side-menu"><NAV class="navbar navbar-default" role="navigation"><DIV class="navbar-header"><DIV class="brand-wrapper"><BUTTON type="button" class="navbar-toggle" style="background-color:  #3f51b5;"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV></DIV><DIV class="side-menu-container" id="side-navigation"><UL class="nav navbar-nav"><LI class="panel panel-default dropdown" id="project-menu"><A data-toggle="collapse" href="#dropdown-project"> Project </A></LI><LI id="project_description"><A href="https://2015.igem.org/Team:Evry/Description">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Project/Chassis">Chassis choice</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Project/Encapsulation">Environment modulation</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Project/Biosensor">Hypoxia Bio-sensor</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Design">Proof of concept</A></LI></UL></DIV></NAV></DIV><LI class="panel panel-default dropdown" id="software-menu"><A data-toggle="collapse" href="#dropdown-software"> Antigen prediction </A></LI><LI><A href="https://2015.igem.org/Team:Evry/Software"><SPAN style="font-size:5%; color: transparent;">o</SPAN>Overview</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Software/Pipeline">Pipeline</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Software/Predictions">Prediction</A></LI></DIV></DIV><LI class="panel panel-default dropdown" id="practices-menu"><A data-toggle="collapse" href="#dropdown-practices"> Practices </A></LI><LI><A href="https://2015.igem.org/Team:Evry/Practices">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Practices/State">Funding state of art</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Practices/Innovative">Innovative funding</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Practices/Society">Society perception</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Practices/Conclusion">Conclusion</A></LI></DIV></DIV><LI class="panel panel-default dropdown" id="achievements-menu"><A data-toggle="collapse" href="#dropdown-achievements"> Achievements </A></LI><LI><A href="https://2015.igem.org/Team:Evry/Medals">Medal fulfillment</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Parts">Parts</A></LI></DIV><LI class="panel panel-default dropdown" id="notebook-menu"><A data-toggle="collapse" href="#dropdown-nb"> Notebook </A></LI><LI><A href="https://2015.igem.org/Team:Evry/Notebook">Day notes</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Protocols">Protocols</A></LI><LI class="panel panel-default dropdown" id="team-menu"><A data-toggle="collapse" href="#team-dropdown"> Team </A></LI><LI><A href="https://2015.igem.org/Team:Evry/Team">Members</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Acknowledgements">Acknowledgements</A></LI><DIV class="container"><DIV class="side-body" id="content-body"><DIV class="page-header"><H1>Prediction</H1></DIV><SECTION class="page-section"><H2>Protein sequences retrieval</H2><P class="text-justify text-muted"><STRONG>What we produced: </STRONG>shell script to retrieve all proteins sequences associated to a list of gene identifiers, FASTA file containing all the proteins encoded by a list of genes (<EM>i. e.</EM> genes that are both overexpressed and mutated in tumor samples).</P><P class="text-justify">So far, our pipeline allowed us to select genes that are over-expressed (differential expression analysis) and mutated (variant discovery) in tumor samples. We retrieved the proteins encoded by these genes with a shell script that
uses the <A href="http://rest.ensembl.org/" target="_blank">Ensembl REST API</A>, that produces a FASTA files containing all the sequences.</P><P class="text-justify">Two machine learning tools were used on this dataset to perform predictions related to proteasome cleavage of the protein and MHC-I affinity.</P></SECTION><SECTION class="page-section"><H2>Immune system processing</H2><P class="text-justify text-muted"><STRONG>What we produced: </STRONG>scripts to launch and parse results of NetChop and NetMHCpan, final result table containing candidate antigens and their final scores.</P><P class="text-justify">A good candidate antigen must be tumor-specific, sufficiently expressed in tumor cells, but also <STRONG>able to be processed efficiently</STRONG> by the immune system.</P><H3>Proteasome cleavage prediction</H3><P class="text-justify">The major histocompatibility complex class I (MHC-I) recognizes peptides of short length (8 to 10 aminoacids). These peptides are products of proteasomal degradation, a process that does not cut proteins randomly. Candidate antigens cannot contain proteasome cleavage sites as they would not be able to be presented to the immune system.</P><P class="text-justify">We performed proteasome cleavage sites prediction using <A href="http://www.cbs.dtu.dk/services/NetChop/" target="_blank">NetChop</A>, an open-source machine learning tool. We obtain a list of short peptides (with their associated NetChop score) that can be presented to the immune system. This list is then filtered by predicting if these antigens will be able to bind to the MHC-I. </P><H3>MHC-I affinity prediction</H3><P class="text-justify">Not all antigens are able to bind efficiently to the MHC-I. We used <A href="http://www.cbs.dtu.dk/services/NetMHCpan/" target="_blank">NetMHCpan</A>, an open-source machine learning tool, to predict the binding affinity of all the antigens in our list. The predicted affinity is given in units of IC50nM, therefore a lower number indicates higher affinity. It is generally assumed that peptides with IC50 values &lt; 50 nM are considered high affinity, &lt; 500 nM intermediate affinity. However, the binding affinity is not correlated to the immune response, meaning that some antigens might bind very effectively without triggering an intense immune response.</P><P class="text-justify">We finally sorted the candidate antigens using a scoring function that combines the two types of predictions (linear combination of normalized score).</P></SECTION><SECTION class="page-section"><H2>The YETIpredict web application</H2><P class="text-justify text-muted"><STRONG>What we produced: </STRONG>a web interface to explore the results of the differential expression analysis, the variant analysis and the immune predictions </P><P class="text-justify">In order to make our pipeline as easy-to-use as possible, we created a web application that allows users to run all the steps of our pipeline that do not require a lot of computing resources. You can reach it <A href="http://igem2015.issb.genopole.fr" target="_blank">here</A> (beta version).</P><P class="text-justify">Visualisations of differential expression and variant analysis are available, as well as interactive tables that allow users to browse, sort and search easily the differentially expressed gene list or the candidate antigens list. </P><P class="text-center"><STRONG>Figure 1:</STRONG> Input page of YETIpredict</P></SECTION></DIV></DIV><DIV id="toTop"><SPAN class="scrolltext">To top</SPAN></DIV></BODY></HTML>