Team:Evry/Notebook
Toggle navigation
Project
Overview
Chassis choice
Environment modulation
Hypoxia Bio-sensor
Proof of concept
Antigen prediction
oOverview
Pipeline
Prediction
Practices
Introduction
Funding state of art
Innovative funding
Society perception
Conclusion
Achievements
Medal fulfillment
Collaborations
Parts
Notebook
Day notes
Protocols
Team
Members
Attributions
Acknowledgements
Notebook
Here is our lab notebook. Follow all the wet lab experiments we did, day by day.
Use the navigation menu below to jump to notebook entries of a particular month.
Jump to:
June
July
August
September
Friday, 12th June 2015
Extraction of AgA1P from yeast genome with BSAI
AGA1P was extracted with BSAI overhangs for subsequent cloning from W303 and BY4000, according to Looke et al., PMC 2011.
Protocol
1) Resuspend one yeast colony in 100 µl of 200 mM LiAc, 1% SDS solution and incubate at 70°C
2) Add 300 µl of 36% ethanol and vortex
3) Spin down DNA at 15 000 g for 3 minute
4) Wash pellet with 70% ethanol
5) Disolve pellet in 100 µl of water and spin down debris at 15 000 g for 15 seconds
PCR of AGA1P with primers AGA1P R1/F1 with Q5 polymerase using a gradient (57/60/63°C)
Q5 PCR Program
step 1 : 98°C – 30 seconds
step 2 : 98°C – 10 seconds
step 3 : 57, 60 or 63°C
step 4 : 72°C – 1 min (repeat steps 2-4 for 40 cycles)
step 5 : 16°C - Hold
Monday, 14th June 2015
PCR on yeast genome for site directed mutagenesis :
ADH1 amplification with 2 sets of primers:
1) primers ADH1 sub F1 + ADH1 sub R1 with 3 anealing temperatures : 57°C (B1), 60°C (B2), 63°C (B3)
2) primers ADH1 sub F2 + ADH1 sub R2 with 3 anealing temperatures : 57°C (B4), 60°C (B5), 63°C (B6)
Malpha IFN gamma amplification with 2 sets of primers:
1) primers Mfalpha IFNgamma F1/R1 with 3 anealing temperatures : 57°C (B7), 60°C (B8), 63°C (B9)
2) primers Mfalpha IFNgamma F1 + Mfalpha GMCSF R1 with 3 anealing temperatures :57°C (B10), 60°C (B11), 63°C (B12)
Friday, 19th June 2015
Golden gates
Golden gates mix (20µl) and inserts:
2 µl T4 ligase buffer 10x
0.5 µl BSAI
0.5 µl T4 ligase
water (qsp 20µl)
Golden gate 1 (A1) inserts:
- New insert1 m1 Cter (AGA2P)
- New insert1 m2 Cter (-OVA1 DEC205)
- Insert1 m2 (GAL10 GAL7 AGA1P)
- Insert1 m3 extracted from yeast genome (AGA1P)
- pYGG1
Golden gate 2 (A2) inserts:
- New insert1 m1 Cter (AGA2P)
- Insert1 m2 (GAL10 GAL7 AGA1P)
- Insert1 m3 extracted from yeast genome (AGA1P)
- pYGG1
Golden gate 3 (A3) inserts :
- insert3 (OVA2)
- pYGG2
Saturday, 20thJune 2015
PCR colony using primers URA F1 and URA R5 2.0 on A1, A2 and A3
PCR colony mix
1 µl of resuspended colony in 20 µl LB
1 µl URA F1
1 µl URA R5 2.0
7 µl water
10 µl PCR mix (Dreamtaq)
PCR colony Program
Step 1 : 95°C - 5 min
Step 2 : 95°C – 30 s
Step 3 : 50°C – 30s
Step 4 : 72°C – 2 min (repeat step 2-4, 45 times)
Step 5 : 72°C – 10 min
Step 6 :  4°C – Pause
PCR colony products gel electrophoresis (1% agarose)
Results: Clones from golden 3 (A3, see 19/06) present the right size. Clones from golden 1 and 2 (A1 and A2, see 19/06) are negative. It seems there was no amplification.
Miniculture (40)
The remaining resusepended E.coli was put into 4 mL of Luria Bertoni media containing with 4 µl ampicilin.
Sunday, 21thJune 2015
Repeat of PCR colony on A1, A2 and A3
PCR colony mix
1 µl of resuspended colony in 20 µl LB
1 µl URA F1
1 µl URA R5 2.0
7 µl water
10 µl PCR mix (Dreamtaq)
PCR colony Program
Step 1 : 95°C - 5 min
Step 2 : 95°C – 30 s
Step 3 : 50°C – 30s
Step 4 : 72°C – 2 min (repeat step 2-4, 45 times)
Step 5 : 72°C – 10 min
Step 6 :  4°C – Pause
PCR colony products gel electrophoresis (1% agarose)
Results: We got clones with expected sizes.
Monday, 22ndJune 2015
PCR clean up and nanodrop :
A1 = 209.1 ng/µl (insert1 m3)
A2 = 56.1 ng/µl (AGA2P)
A6 = 114.3 ng/µl (IFN gamma)
A7 = 93.7 ng/µl (GMCSF)
B8 = 114 ng/µl (Malpha IFNgamma)
B10 = 134.2 ng/µl (Malpha GMCSF)
B11 = 163.9 ng/µl (Malpha GMCSF)
C1 = 180.7 ng/µl (ADH1)
C2 = 302.9 ng/µl (ADH1)
G2/G2/G3/G4 Golden gates
Golden gate mix (20 µl) and inserts:
2 µl T4 ligase buffer 10x
0.5 µl BSAI + 0.5 T4 ligase
water (qsp 20µl)
Inserts used for ADH1 Malpha IFNgamma construction (G1):
- C1
- B8
- A6
- pYGG1
Inserts used for ADH1 Malpha GMCSF construction (G2):
- C1
- B10
- A7
- pYGG1
Inserts used for ADH2 Malpha IFNgamma (G3):
- C2
- B8
- A6
- pYGG1
Inserts used for ADH2 … GMCSF construction (G4):
- C2
- B11
- A7
- pYGG1
E. coli transformation with golden gates G1, G2, G3, G4 products
Protocol :
1) Add 10 µl plasmids  to 50 µl E. coli competent cells on ice and let the mix rest on ice for 15 min
2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min
3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour
4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin
5) Put plates in growth incubators at 37°C for 24 hours
Tuesday, 23rd June 2015
G2/G2/G3/G4 colony PCR
PCR colony mix
1 µl of resuspended colony in 20 µl LB
1 µl URA F1
1 µl URA R5 2.0
7 µl water
10 µl PCR mix (Dreamtaq)
PCR colony Program
Step 1 : 95°C - 5 min
Step 2 : 95°C – 30 s
Step 3 : 50°C – 30s
Step 4 : 72°C – 2 min (repeat step 2-4, 45 times)
Step 5 : 72°C – 10 min
Step 6 :  4°C – Pause
G2/G2/G3/G4 minicultures
The remaining resusepended E.coli was put into 4 mL of Luria Bertoni media containing with 4 µl ampicilin.
Wednesday, 24th June 2015
G2/G2/G3/G4 library from minicultures (see 23/06) :
750 µl of G1 to G8 were mixed with 750 µl of glycerol 50% and put in the freezer at -80°C.
G2/G2/G3/G4 Minipreped using NucleoSpin Plasmid (LOT 1306/003 Macherey-Nagel) :
Samples corresponding to golden gate G1:
G1 mini = 270.5 ng/µl (260/280 = 1.81 ; 260/230 = 1.72)
G2 mini = 287.4 ng/µl (260/280 = 1.84 ; 260/230 = 1.63)
Samples corresponding to golden gate G2:
G3 mini = 146.4 ng/µl (260/280 = 1.81 ; 260/230 = 1.42)
G4 mini = 264.3 ng/µl (260/280 = 1.79 ; 260/230 = 1.80)
Samples corresponding to golden gate G3 (ADH2 Malpha IFNgamma):
G5 mini = 164.9 ng/µl (260/280 = 1.82 ; 260/230 = 1.49)
G6 mini = 128.1 ng/µl (260/280 = 1.84 ; 260/230 = 1.56)
Samples corresponding to golden gate G4:
G7 mini = 587.4 ng/µl (260/280 = 1.67 ; 260/230 = 0.63)
G8 mini = 160.2 ng/µl (260/280 = 1.81 ; 260/230 = 1.91)
Thursday, 25th June 2015
G samples were sent to sequencing using 3 primers sets :
- URA F1/SR1 lig IFN gamma
- S2 lig IFN gamma/SR2 lig IFN gamma
- URA F1/SR1 lig IFN gamma
Sequencing mixes (10µl):
G4 : 4 µl DNA + 1.25 µl/primer + qsq water
G6 : 6 µl DNA + 1.25 µl/primer + qsq water
Friday, 3rd July 2015
AGA2P -OVA1 DEC205 (pYYG1) construction : Nanodrop results
New insert1 m1 Cter (AGA2P) = 10 ng/µl
New insert1 m2 Cter PCR1 (-OVA1 DEC205) = 77 ng/µl
pYGG1 (P11) = 107.6 ng/µl
AGA2P OVA1 DEC205 (pYGG1) new golden gate
Golden gate mix (20µl):
15.5 µl H20
2 µl T4 ligase buffer 10x
0.425 µl New insert1 m1 Cter (AGA2P)
0.194 µl New insert1 m2 Cter PCR1 (-OVA1 DEC205)
1 µl pYGG1
0.5 µl BSA I
0.5 µl T4 ligase
AGA2P OVA1 DEC205 (pYGG1) E. coli transformation
Protocol:
1) Add 10 µl plasmids  to 50 µl E. coli competent cells on ice and let the mix rest on ice for 15 min
2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min
3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour
4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin
5) Put plates in growth incubators at 37°C for 24 hours
Sunday, 5th July 2015
AGA2P OVA1 DEC205 (pYGG1) construction: PCR colony
PCR colony mix (20 µl):
1 µl of resuspended colony in 20 µl LB
1 µl URA F1 (primers)
1 µl URA R5 2.0 (primers)
7 µl water
10 µl PCR mix (Dreamtaq)
PCR colony Program
Step 1 : 95°C - 5 min
Step 2 : 95°C – 30 s
Step 3 : 50°C – 30s
Step 4 : 72°C – 2 min (repeat step 2-4, 45 times)
Step 5 : 72°C – 10 min
Step 6 :  4°C – Pause
Tuesday, 7th July 2015
AGA2P OVA1 DEC205 (pYGG1) construction: Minicultures
The remaining resusepended E.coli was put into 4 mL of Luria Bertoni media containing with 4 µl ampicilin.
Wednesday, 8th July 2015
AGA2P OVA1 DEC205 (pYGG1) construction: PCR colony
PCR colony mix:
1 µl of resuspended colony in 20 µl LB
1 µl URA F1
1 µl URA R5 2.0
7 µl water
10 µl PCR mix (Dreamtaq)
PCR colony program :
Step 1 : 95°C -  5 min
Step 2 : 95°C – 30 seconds
Step 3 : 50°C – 30 seconds
Step 4 : 72°C –  2 minutes (repeat step 2-4, 45 times)
Step 5 : 72°C – 10 minutes
Step 6 :  4°C – Pause
AGA2P OVA1 DEC205 (pYGG1) construction: PCR colony gel electrophoresis (1% agarose)
Thursday, 9th July 2015
Miniculture of transformed AGA2P OVA1 DEC205 (pYGG1) E.coli
19 µl colony resuspended Luria Bertoni media in 4 mL of Luria Bertoni media complemented with 4 µl of ampicilin (100 mg/µl) and put to incubate at 37°C overnight.
Samples names : D13/ D23/ D33 / D43
Friday, 10th July 2015
Samples of AGA2P OVA1 DEC205 (pYGG1) construction were minipreped using NucleoSpin Plasmid (LOT 1306/003 Macherey-Nagel) and nanodroped :
Mini P1 = 449.9 µl (260/230= 1.82 ; 260/280= 1.89) (from D13 and D23, see 09/07)
Mini P2 = 414.2 µl (260/230= 1.83 ; 260/280= 1.94) (from D33 and D43, see 09/07)
Monday, 13th July 2015
ADH1 Mat alpha IFN gamma (pYGG1) construction: Golden gate
Golden gate mix :
11.35 µl water
2 µl T4 ligase buffer 10x
0.177 µl ADH1
4.473 µl (Mat alpha IFN gamma) = Gblock
1 µl pYGG1
0.5 µl BSA I
0.5 µl T4 ligase
Golden gate program :
Mat alpha IFN gamma ADH1 (pYGG1) E. coli transformation
Protocol :
1) Add 10 µl plasmids  to 50 µl E. coli competent cells on ice and let the mix rest on ice for 15 min
2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min
3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour
4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin
5) Put plates in growth incubators at 37°C for 24 hours
Monday, 13th July 2015
Samples of AGA2P OVA1 DEC205 (pYGG1) construction were minipreped using NucleoSpin Plasmid (LOT 1306/003 Macherey-Nagel) and nanodroped :
- AGA2P OVA1 DEC205 1 (SD1) = 449.9 ng/µl
- AGA2P OVA1 DEC205 2 (SD2) = 414.2 ng/µl
Samples of AGA2P OVA1 DEC205 (pYGG1) construction, Mini P1 and Mini P2 were sent to sequencing
Wednesday, 15th July 2015
Mat alpha IFN gamma ADH1 (pYGG1) construction : PCR colony
PCR colony mix (20 µl):
10 µl PCR mix (DreamTaq)
1 µl URA F1
1 µl URA R5 2.0
1 µl colony resuspended in Luria Bertoni media
7 µl water
PCR colony program:
Step 1 : 95°C - 5 minutes
Step 2 : 95°C – 30 seconds
Step 3 : 50°C – 30 seconds
Step 4 : 72°C – 2 minutes (Repeat step 2-4, 35 times)
Step 5 : 72°C – 10 minutes
Step 6 :  4°C – Pause
Thursday, 16th July 2015
Mat alpha IFN gamma ADH1 (pYGG1) PCR colony products gel electrophoresis (1% agarose)
Golden gate program :
Only clones A4, C6 and D7 corresponded to the size expected for (Mat alpha IFN gamma) ADH1 : 2200 pb.
Miniculture
19 µl colony resuspended Luria Bertoni media in 4 mL of Luria Bertoni media complemented with 4 µl of ampicilin (100 mg/µl) and put to incubate at 37°C overnight.
Friday, 17th July 2015
OVA2 (pYGG2): golden gate
Golden gate mix (20 µl):
2 µl T4 ligase
0.5 µl BSA I
0.5 T4 µl DNA ligase
0.801 µl OVA2 (12,6 ng/µl) (Gblock)
0.714 µl pYGG2 (107.6 ng/µl)
15.4 µl water
Golden gate program :
Monday, 20th July 2015
OVA2 (pYGG2) E. coli transformation
Protocol :
1) Add 10 µl plasmids  to 50 µl E. coli competent cells on ice and let the mix rest on ice for 15 min
2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min
3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour
4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin
5) Put plates in growth incubators at 37°C for 24 hours
Tuesday, 21st July 2015
OVA2 (pYGG2) E. coli transformation results
Previously (see 2015-07-21) plates were put overnight into incubators at 37°C.
There is no colonies on the plates, either because the E. coli mix died before being plated or because the golden gate failed in some ways. The latter hypothesis is unliky as we woµld have red negative colonies onto the plate.
=> It turns out we did not use the right plasmid. We shoµld have used pYGG1
Thursday, 23rd July 2015
OVA2 pYGG1 golden gate
Golden gate mix (20 µl):
2 µl T4 ligase
0.5 µl BSA I
0.5 T4 µl DNA ligase
1 µl OVA2 (12,6 ng/µl) (Gblock)
1 µl pYGG1 (107.6 ng/µl)
15 µl water
Golden gate program:
OVA2 pYGG1 E.Coli transformation
Protocol:
1) Add 10 µl plasmids  to 50 µl E. coli competent cells on ice and let the mix rest on ice for 15 min
2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min
3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour
4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin
5) Put plates in growth incubators at 37°C for 24 hours
AGA2P OVA1 DEC205 minipreped samples were sent to sequencing using two sets of primers :
Seq.ID 26EB08 => SD1.1 (primers : URA F1 / URA F1 reverse seq) (The sequence length is 0 nt)
Seq.ID 26EB09 => SD1.2 (primers : URA R5 2.0 forward seq / URA R5 2.0) (The sequence length is 0 nt)
Seq.ID 26EB10 => SD1.1’ (replicate of SD1.1) (results : The sequence length is 23 nt)
Seq.ID 26EB11 => SD1.2’ (replicate of SD1.2) (results : The sequence length is 41 nt)
Seq.ID 26EB12 => SD2.1 (same as SD1.1) (results : The sequence length is 22 nt)
Seq.ID 26EB13 => SD2.2 (same as SD1.2) (results : The sequence length is 737 nt)
Seq.ID 26EB14 => SD2.1’ (replicate of SD2.1) (results : The sequence length is 0 nt)
Seq.ID 26EB15 => SD2.2’ (Replicate of SD2.2) (Results : The sequence length is 20 nt)
Friday, 23rd July 2015
Mat alpha IFN gamma ADH1 (pYGG1) : Sample was minipreped using NucleoSpin Plasmid (LOT 1306/003 Macherey-Nagel) from the miniculture (see 15-07-15) and nanodroped :
Mata IFN gamma pYGG1 mini (A4 mini) = 256 ng/µl (260/230= 1.23 ; 260/280= 1.73)
Mata IFN gamma pYGG1 mini (C6 mini) = 66.8 ng/µl (260/230= 1.39 ; 260/280= 1.85)
Mata IFN gamma pYGG1 mini (D7 mini) = 71.4 ng/µl (260/230= 1.67 ; 260/280= 1.88)
Mat alpha IFN gamma ADH1 (pYGG1) sequencing using only one primer set (URA F1/ URA R5 2.0):
Seq.ID 26EB02 => A4 mini .1 (results : The sequence length is 1110 nt)
Seq.ID 26EB03 => A4 mini .2 (replicate) (results : The sequence length is 1076 nt)
Seq.ID 26EB04 => C6 mini .1 (results : The sequence length is 986 nt)
Seq.ID 26EB05 => C6 mini .2 (replicate) (results : The sequence length is 978 nt)
Seq.ID 26EB06 => D7 mini .1 (results : The sequence length is 23 nt)
Seq.ID 26EB07 => D7 mini .2 (replicate) (results : The sequence length is 38 nt)
Friday, 24th July 2015
OVA2 (pYGG1) E.coli transformation results
The 2 plates put into the incubators showed 4 colonies at most.
We decided to wait for the plates to develop a bit more and to put back the miniculture and colony PCR to Monday 27th 2015. Meanwhile, after a few hours into the incubator, plates were wrapped with parafilm and put into the fridge  at 4°C.
Monday, 27th July 2015
OVA2 (pYGG1): PCR colony and gel electrophoresis
Previously (see 2015-07-23), we transformed E. coli with our golden gate products and plated the transformed E. coli onto 2 plates.
We took 8 positive colonies from the plates and resuspend them separately into 20 µl of Luria Bertoni media.
PCR colony mix (20 µl):
10 µl PCR mix (DreamTaq)
1 µl URA F1
1 µl URA R5 2.0
1 µl colony resuspended in Luria Bertoni media
7 µl water
PCR colony program:
Step 1 : 95°C - 5 minutes
Step 2 : 95°C – 30 seconds
Step 3 : 50°C – 30 seconds
Step 4 : 72°C – 2 minutes (Repeat step 2-4, 35 times)
Step 5 : 72°C – 10 minutes
Step 6 :  4°C – Pause
PCR from New insert1 m1 Cter (PCR001): amplification of AGA2P with primers containing DEC205 overhang
Q5 PCR mix (50µl)
2,5 µl FWD AGA2P
2,5 µl  RV AGA2P
1 µl New insert1 m1 Cter
19 µl water
25 µl PCR mix (Q5)
Q5 PCR Program
step 1 : 95°C – 30 seconds
step 2 : 95°C – 10 seconds
step 3 : 60°C
step 4 : 72°C – 1 min (repeat steps 2-4 for 31 cycles)
step 5 : 12°C - Hold
PCR from New insert1 m2 Cter (PCR002): amplification of DEC205 with  primers containing AGA2P overhang
Q5 PCR mix (50µl)
2,5 µl FWD DEC205
2,5 µl  RV DEC205
1 µl New insert1 m2 Cter
19 µl water
25 µl PCR mix (Q5)
Q5 PCR Program
step 1 : 95°C – 30 seconds
step 2 : 95°C – 10 seconds
step 3 : 60°C
step 4 : 72°C – 1 min (repeat steps 2-4 for 31 cycles)
step 5 : 12°C - Hold
PCR001 and PCR002 gel electrophoresis (1% agarose):
PCR clean up and nanodrop of AGA2P and DEC205 products :
AGA2P = 72 .3 ng/µl (260/280 = 1.79 ; 260/230 = 0.76)
DEC205 = 104.2 ng/µl (260/280 = 1.79 ; 260/230 = 0.42)
Golden gates of :
- (pYGG1) + AGA2P + DEC205
- (pYGG1) + AGA2P + OVA1 (Gblock)
- (pYGG1) + ADH1 (miniprep « ADH1 ») + Malpha GMCSF (B10, see 22/06/15)
Golden gate program:
E. coli transformation with golden gate products
Protocol:
1) Add 10 µl plasmids  to 50 µl E. coli competent cells on ice and let the mix rest on ice for 15 min
2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min
3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour
4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin
5) Put plates in growth incubators at 37°C for 24 hours
Tuesday, 28th July 2015
We sent to sequencing Mat alpha IFN gamma ADH1 (pYGG1) with new primers since previous sequencing results were unconclusive.
Furthermore, in order to check the presence of our desired fragments, we also performed a multiple PCR on A4, C6 and D7 samples.
A4, C6 and D7 samples sequencing :
Seq.ID 26EB16 - A41 (URA F1/SR1 lig IFNg)
Seq.ID 26EB17 - A41.2 (replicate of A41)
Seq.ID 26EB18 - A42 (S2 lig IFNg/SR2 lig IFNg)
Seq.ID 26EB19 - A42.2 (replicate of A42)
Seq.ID 26EB20 - C61 (URA F1/SR1 lig IFNg)
Seq.ID 26EB21 - C61.2
Seq.ID 26EB22 - C62 (S2 lig IFNg/SR2 lig IFNg)
Seq.ID 26EB23 - C62.2
Seq.ID 26EB24 - D71 (URA F1/SR1 lig IFNg)
Seq.ID 26EB25 - D71.2
Seq.ID 26EB26 - D72 (S2 lig IFNg/SR2 lig IFNg)
Seq.ID 26EB27 - D72.2
Multiple PCR using URA F1/SR1 lig IFNg and S2 lig IFNg/SR2 lig IFNg primers :
PCR mix (20 µl):
1 µl of DNA
1 µl forward primer
1 µl reverse primer
7 µl water
10 µl PCR mix (Dreamtaq)
Multiple PCR gel electrophoresis (1% agarose)
Golden gate of ADH1 (miniprep « ADH1 ») Mat alpha GMCSF (B10, see 22/06/15) (pYGG1)
E. coli transformation with golden gate products
Protocol :
1) Add 10 µl plasmids  to 50 µl E. coli competent cells on ice and let the mix rest on ice for 15 min
2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min
3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour
4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin
5) Put plates in growth incubators at 37°C for 24 hours
Tuesday, 28th July 2015
We sent to sequencing AGA2P OVA1 DEC205 (pYGG2) with new primers since previous sequencing results were unconclusive.
Furthermore, in order to check the presence of our desired fragments, we also performed a multiple PCR on SD1 and SD2 samples.
SD1 and SD2 samples sequencing:
Sequencing ID and primers used for sample "SD2":
Seq.ID 26EB28 (URA F1/URA F1 Rev Seq V2)
Seq.ID 26EB29 (URA R5 2.0 Fw seq V2/URA R5 2.0)
Seq.ID 26EB30 (URA F1/ URA R1)
Seq.ID 26EB31 (URA F2/URA R5 2.0)
Seq.ID 26EB32 (URA F1/URA F1 Rev Seq V2)
Seq.ID 26EB33 (URA R5 2.0 Fw seq V2/URA R5 2.0)
Seq.ID 26EB34 (URA F1/ URA R1)
Seq.ID 26EB35 (URA F2/URA R5 2.0)
Sequencing ID and primers used for sample "SD1":
Seq.ID 26EB36 (URA F1/URA F1 Rev Seq V2)
Seq.ID 26EB37 (URA R5 2.0 Fw seq V2/URA R5 2.0)
Seq.ID 26EB38 (URA F1/ URA R1)
Seq.ID 26EB39 (URA F2/URA R5 2.0)
Seq.ID 26EB40 (URA F1/URA F1 Rev Seq V2)
Seq.ID 26EB41 (URA R5 2.0 Fw seq V2/URA R5 2.0)
Seq.ID 26EB42 (URA F1/ URA R1)
Seq.ID 26EB43 (URA F2/URA R5 2.0)
Multiple PCR
PCR mix (20 µl):
1 µl of DNA
1 µl forward primer
1 µl reverse primer
7 µl water
10 µl PCR mix (Dreamtaq)
Multiple PCR gel electrophoresis (1% agarose)
Golden gates of :
- Golden 1 : pYGG1 + AGA2P + DEC205
- Golden 2 : PYGG1 + AGA2P + OVA1 (Gblock) (dec 205 v2)
E. coli transformation with golden gate products
Protocol:
1) Add 10 µl plasmids  to 50 µl E. coli competent cells on ice and let the mix rest on ice for 15 min
2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min
3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour
4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin
5) Put plates in growth incubators at 37°C for 24 hours
Wednesday, 29th July 2015
Results of E.coli transformed with AGA2P OVA1 (Gblock "dEC205 - v2") (pYGG1) and AGA2P DEC205 (pYGG1)
We got a few colonies for AGA2P OVA1 but none for AGA2P DEC2065 so we performed a PCR colony for AFGA2P OVA1 and a second transformation for AGA2P DEC205 with remaining golden gate (see 28/07).
AGA2P OVA1: PCR colony
PCR colony mix (20 µl):
10 µl PCR mix (DreamTaq)
1 µl URA F1
1 µl URA R5 2.0
1 µl colony resuspended in Luria Bertoni media
7 µl water
PCR colony program :
Step 1 : 95°C - 5 minutes
Step 2 : 95°C – 30 seconds
Step 3 : 50°C – 30 seconds
Step 4 : 72°C – 2 minutes (Repeat step 2-4, 30 times)
Step 5 : 72°C – 10 minutes
Step 6 :  4°C – Pause
AGA2P OVA1: PCR colony gel electrophoresis (1% agarose)
Yeast transformations with following golden gate constructions :
- AGA2P OVA1 DEC205 (SD1)
- AGA1
- AGA2P OVA1 DEC205 (SD2)
Protocol :
1) From the W303 yeast culture at DO=1, harvest in sterile tube at 5000 rpm for 5 min
2) Pour off the medium, resuspend the cells in 25 ml of sterile water and centrifuge again
3) Pour off the water, resuspend the cells in 1 ml of 0.1 M LiAc and transfer the suspension to a 1.5 ml microfuge tube
4) Pellet the cells at top speed for 15 sec and remove the LiAc with a micropipette
5) Resuspend the cells with 0.1 LiAc to a final volume of 500 µl
6) Vortex the cell suspension and pipette 50 µl samples into new 1.5 ml tubes. Pellet the cells and remove the LiAc with a micropipette
7) Add the following to the samples in order:
- 240 µl PEG 50%
- 36 µl 1 M LiAc
- 25 µl Salmon sperm DNA (2 mg/ml)
- 50 µl water and plasmid (10 ug)
8) Vortex each tube vigorously until the cell pellet has been completely mixed. Usually takes about 1 min
9) Incubate at 30°C for 30 min
10) Heat shock in a water bath at 42°C for 15 minute
11) Ice for 2 minutes
12) Centrifuge at 5000 rpm for 15 sec and remove the transformation mix with a micropipette
13) Pipette 1 ml of sterile water into each tube and resuspend the pellet by pipetting it up and down gently
14) Plate 50 µl and 150 µl of the transformation mix onto plates with corresponding media
15) Incubate at 30°C for 3 days
Wednesday, 29th July 2015
Results of E.coli transformed with ADH1 (miniprep « ADH1 ») + Malpha GMCSF (B10, see 22/06/15) (pYGG1)
We got no colonies, so we performed a second E.coli transformation with remaining golden gate (see 28/07).
E. coli transformation with golden gate products
Protocol :
1) Add 10 µl plasmids  to 50 µl E. coli competent cells on ice and let the mix rest on ice for 15 min
2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min
3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour
4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin
5) Put plates in growth incubators at 37°C for 24 hours
AGA1P sequencing
Friday, 31st July 2015
Since on the 07/30, we could not perform all yeast transformations (transformation 5 or T5), due to lack of samples for AGA1P, we transformed all the constructions into yeast again.
The yeasts that were transformed the day before are already growing except for the plate CoT2 that did not contain enough substrate for the yeast to feed off.
Yeast transformation:
T4 : ADH1 Ma IFNg (pYGG1)
Protocol :
1) From the W303 yeast culture at DO=1, harvest in sterile tube at 5000 rpm for 5 min
2) Pour off the medium, resuspend the cells in 25 ml of sterile water and centrifuge again
3) Pour off the water, resuspend the cells in 1 ml of 0.1 M LiAc and transfer the suspension to a 1.5 ml microfuge tube
4) Pellet the cells at top speed for 15 sec and remove the LiAc with a micropipette
5) Resuspend the cells with 0.1 LiAc to a final volume of 500 µl
6) Vortex the cell suspension and pipette 50 µl samples into new 1.5 ml tubes. Pellet the cells and remove the LiAc with a micropipette
7) Add the following to the samples in order:
- 240 µl PEG 50%
- 36 µl 1 M LiAc
- 25 µl Salmon sperm DNA (2 mg/ml)
- 50 µl water and plasmid (10 ug)
8) Vortex each tube vigorously until the cell pellet has been completely mixed. Usually takes about 1 min
9) Incubate at 30°C for 30 min
10) Heat shock in a water bath at 42°C for 15 minute
11) Ice for 2 minutes
12) Centrifuge at 5000 rpm for 15 sec and remove the transformation mix with a micropipette
13) Pipette 1 ml of sterile water into each tube and resuspend the pellet by pipetting it up and down gently
14) Plate 50 µl and 150 µl of the transformation mix onto plates with corresponding media:
pYGG1 => URA-
pYGG2 => TRP-
pYGG1 + pYGG2 => TRP- URA-
15) Incubate at 30°C for 3 days
Friday, 31st July 2015
Since on the 07/30, we could not perform all yeast transformations (transformation 5 or T5), due to lack of samples for AGA1P, we transformed all the constructions into yeast again.
The yeasts that were transformed the day before are already growing except for the plate CoT2 that did not contain enough substrate for the yeast to feed off.
AGA1P minipreped using NucleoSpin Plasmid (LOT 1306/003 Macherey-Nagel) and nanodroped :
AGA1P.1 = 44.2 ng/µl
Yeast transformations:
T1: OVA2 (pYGG2)
T2: AGA2P OVA1 (pYGG1) + AGA1P (pYGG2)
T3: AGA2P OVA1 DEC205 (pYGG1) + AGA1P (pYGG2)
T5: AGA2P/GFP (pYGG1) + AGA1P (pYGG2)
T6: AGA2P GFP (pYGG1)
T7: AGA2P OVA DEC205 (pYGG1)
T8: AGA2P OVA1 (pYGG1)
Protocol :
1) From the W303 yeast culture at DO=1, harvest in sterile tube at 5000 rpm for 5 min
2) Pour off the medium, resuspend the cells in 25 ml of sterile water and centrifuge again
3) Pour off the water, resuspend the cells in 1 ml of 0.1 M LiAc and transfer the suspension to a 1.5 ml microfuge tube
4) Pellet the cells at top speed for 15 sec and remove the LiAc with a micropipette
5) Resuspend the cells with 0.1 LiAc to a final volume of 500 µl
6) Vortex the cell suspension and pipette 50 µl samples into new 1.5 ml tubes. Pellet the cells and remove the LiAc with a micropipette
7) Add the following to the samples in order:
- 240 µl PEG 50%
- 36 µl 1 M LiAc
- 25 µl Salmon sperm DNA (2 mg/ml)
- 50 µl water and plasmid (10 ug)
8) Vortex each tube vigorously until the cell pellet has been completely mixed. Usually takes about 1 min
9) Incubate at 30°C for 30 min
10) Heat shock in a water bath at 42°C for 15 minute
11) Ice for 2 minutes
12) Centrifuge at 5000 rpm for 15 sec and remove the transformation mix with a micropipette
13) Pipette 1 ml of sterile water into each tube and resuspend the pellet by pipetting it up and down gently
14) Plate 50 µl and 150 µl of the transformation mix onto plates with corresponding media:
pYGG1 => URA-
pYGG2 => TRP-
pYGG1 + pYGG2 => TRP- URA-
15) Incubate at 30°C for 3 days
Monday, 3rd August 2015
Colony PCR of yeast transformants with following primers :
AGA2P/DEC205
P1-4 => URA F1/ URA R1
P5-8 => URA R2/ URA R5 2.0
SD1
R1-4 => URA F1/URA R5 2.0
R5-6 => URA F1/URA R1
R7-8 => URA F2/URA R5 2.0
SD2
T1-4 => URA F1/ URA R5 2.0
T5-6 => URA F1/URA R1
T7-8 => URA F2/URA R5 2.0<:li>
Library
- "130"   P2 (AGA2P/DEC205)
- "131"   P4 (AGA2P/DEC205)
- "132"   R4 (SD1)
- "133"   R5 (SD1)
- "134"   R6 (SD1)
- "135"   T1 (SD2)
- "136"   T3 (SD2)
- "137"   T4 (SD2)
- "138"   T5 (SD2)
PCR Colony mix:
1 µl of resuspended colony in 20 µl LB
1 µl URA F1
1 µl URA R5 2.0
7 µl water
10 µl PCR mix (Dreamtaq)
PCR colony Program:
Step 1 : 95°C - 5 min
Step 2 : 95°C – 30 s
Step 3 : 50°C – 30s
Step 4 : 72°C – 2 min (repeat step 2-4, 45 times)
Step 5 : 72°C – 10 min
Step 6 :  4°C – Pause
Monday, 3rd August 2015
Colony PCR of yeast transformants with following primers :
GMCSF
Q1-8 => URA F1/URA R5 2.0
IFNgamma
S1-4 => URA F1/SR1 lig IFN gamma
S5-8 => S2 lig IFN gamma/SR2 lig IFN gamma
Tuesday, 4th August 2015
Colony PCR of transformed E. coli with ADH1 Mata GMCSF and ADH1 Mata IFNg
PCR colony mix (20 µl):
10 µl PCR mix (DreamTaq)
1 µl URA F1
1 µl URA R5 2.0
1 µl colony resuspended in Luria Bertoni media
7 µl water
PCR colony program:
Step 1 : 95°C - 5 minutes
Step 2 : 95°C – 30 seconds
Step 3 : 50°C – 30 seconds
Step 4 : 72°C – 2 minutes (Repeat step 2-4, 35 times)
Step 5 : 72°C – 10 minutes
Step 6 :  4°C – Pause
Colony PCR gel electrophoresis (1% agarose)
Colony PCR yeast gel electrophoresis (1% agarose)
Wednesday, 5th August 2015
Gel electrophoresis of ADH1 Mata GMCSF and ADH1 Mata IFNg
Colony PCR2 of ADH1 Mata GMCSF and ADH1 Mata IFNg :
PCR colony mix (10 µl):
5 µl PCR mix (DreamTaq)
0.5 µl URA F1
0.5 µl URA R5 2.0
0.5 µl colony resuspended in Luria Bertoni media
3.5 µl water
PCR colony program:
Step 1 : 95°C - 5 minutes
Step 2 : 95°C – 30 seconds
Step 3 : 72°C – 30 seconds
Step 4 : 60°C/53°C– 2 minutes (Repeat step 2-4, 35 times)
Step 5 : 72°C – 10 minutes
Step 6 : 16°C – Pause
Miniculture of ADH1 Mata GMCSF and ADH1 Mata IFNg
The remaining resusepended E.coli was put into 4 mL of Luria Bertoni media containing with 4 µl ampicilin.
PCR colony of T4 (see 31/07)
PCR Colony mix (20 µl):
1 µl of resuspended colony in 20 µl Luria Bertoni media
1 µl URA F1
1 µl URA R5 2.0
7 µl water
10 µl PCR mix (Dreamtaq)
PCR colony program:
Step 1 : 95°C - 5 minutes
Step 2 : 95°C – 30 seconds
Step 3 : 60°C – 30 seconds
Step 4 : 72°C – 2 minutes (repeat step 2-4, 45 times)
Step 5 : 72°C – 10 minutes
Step 6 :  4°C – Pause
Golden gate  of ADH1 mat alpha GMCSF (pYGG1) construction :
Golden gate mix (20 µl):
0.849 µl ADH1
3.243 µl Mat alpha GMCSF
3.1 µl PYGG1
2 µl T4 ligase buffer
0.5 µl T4 ligase
0.5 µl BSA I
Golden gate program :
E.coli transformation with golden gate ADH1 mat alpha GMCSF (pYGG1)
Protocol :
1) Add 10 µl plasmids  to 50 µl E.coli competent cells on ice and let the mix rest on ice for 15 min
2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min
3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour
4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin
5) Put plates in growth incubators at 37°C for 24 hours
Wednesday, 5th August 2015
Colony PCR of T1/2/3/5/6/7/8 (see 31/07)
PCR Colony mix (20 µl):
1 µl of resuspended yeast colony in 10 µl water
1 µl URA F1
1 µl URA R5 2.0
7 µl water
10 µl PCR mix (Dreamtaq)
PCR colony program :
Step 1 : 95°C - 5 minutes
Step 2 : 95°C – 30 seconds
Step 3 : 60°C – 30 seconds
Step 4 : 72°C – 4 minutes (repeat step 2-4, 45 times)
Step 5 :  4°C – Pause
Thursday, 6th August 2015
Yeast transformation without OVA1
Protocol :
1) From the W303 yeast culture at DO=1, harvest in sterile tube at 5000 rpm for 5 min
2) Pour off the medium, resuspend the cells in 25 ml of sterile water and centrifuge again
3) Pour off the water, resuspend the cells in 1 ml of 0.1 M LiAc and transfer the suspension to a 1.5 ml microfuge tube
4) Pellet the cells at top speed for 15 sec and remove the LiAc with a micropipette
5) Resuspend the cells with 0.1 LiAc to a final volume of 500 µl
6) Vortex the cell suspension and pipette 50 µl samples into new 1.5 ml tubes. Pellet the cells and remove the LiAc with a micropipette
7) Add the following to the samples in order:
- 240 µl PEG 50%
- 36 µl 1 M LiAc
- 25 µl Salmon sperm DNA (2 mg/ml)
- 50 µl water and plasmid (10 ug)
8) Vortex each tube vigorously until the cell pellet has been completely mixed. Usually takes about 1 min
9) Incubate at 30°C for 30 min
10) Heat shock in a water bath at 42°C for 15 minute
11) Ice for 2 minutes
12) Centrifuge at 5000 rpm for 15 sec and remove the transformation mix with a micropipette
13) Pipette 1 ml of sterile water into each tube and resuspend the pellet by pipetting it up and down gently
14) Plate 50 µl and 150 µl of the transformation mix onto plates with corresponding media:
pYGG1 => URA-
pYGG2 => TRP-
pYGG1 + pYGG2 => TRP- URA-
15) Incubate at 30°C for 3 days
Friday, 7th August 2015
PCR colony yeast and E.coli
PCR colony gel electrophoresis (1% agarose) :
Saturday, 8th August 2015
E.coli PCR colony of GMCSF and IFNg
PCR colony gel electrophoresis (1% agarose) :
Monday, 10th August 2015
Golden gate ADH1 matalpha GMCSF (pYGG1)
Golden gate mix (20 µl):
0.849 µl ADH1
3.243 µl Mat alpha GMCSF
3.1 µl PYGG1
2 µl T4 ligase buffer
0.5 µl T4 ligase
0.5 µl BSA I
Golden gate program:
Tuesday, 11th August 2015
E.coli transformation of ADH1 Mat alpha GMCSF (pYGG1) (again)
Protocol:
1) Add 10 µl plasmids  to 50 µl E.coli competent cells on ice and let the mix rest on ice for 15 min
2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min
3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour
4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin
5) Put plates in growth incubators at 37°C for 24 hours
Golden gate of ADH1 mat alpha IFNgamma (pYGG1)
Golden gate mix (20 µl):
0.033 (x 3) µl Sample « B8 » (M :atalpha IFNgamma) (114 ng/µl)
0.058 (x 3) µl Sample « A6 » (IFNgamma) (114 ng/µl)
0.274 µl Sample « ADH1 » (75 ng/µl)
0.929 µl pYGG1
2 µl T4 ligase buffer
0.5 µl T4 ligase
0.5 µl BSA I
Golden gate program :
Tuesday, 11th August 2015
Culture of T1, T2 T3 and WT yeast from plates in glucose (without induction)
Wednesday, 12th August 2015
Culture induction of T1/T2/T3 and WT yeast
1) Discard media after centrifugation at 3000 rpm for 4 minutes
2) Resuspend yeast in 10 mL induction media :
- galactose 1X without tryptophane for T1
- galactose 1X without tryptophane and uracile for T2 and T3
3) Put into incubator and agitation at 25 °C for 48 hours
Wednesday, 12th August 2015
E.coli transformation of ADH1 Mat alpha IFNgamma (pYGG1)
Protocol :
1) Add 10 µl plasmids  to 50 µl E.coli competent cells on ice and let the mix rest on ice for 15 min
2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min
3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour
4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin
5) Put plates in growth incubators at 37°C for 24 hours
Colony PCR of pYGG1 ADH1 matalpha GMCSF with 3 sets of primers :
- URA F1 / URA R5 2.0
- URA F1 / SR1 lig IFNgamma
- S2 lig IFNgamma / SR2 lig GMCSF
Thursday, 13th August 2015
Colony PCR of ADH1 Mat alpha IFN gamma (pYGG1)  with 3 sets of primers :
- URA F1 / URA R5 2.0
- URA F1 / SR1 lig IFNgamma
- S2 lig IFNgamma / SR2 lig IFNgamma
PCR colony mix (10 µl):
5 µl dreamtaq master mix
0.5 µl colony resuspended in 20 µl Luria Bertoni
0.5 µl  reverse primer
0.5 µl forward primer
3.5 µl water
Miniprep of ADH1 Mat alpha GMCSF (pYGG1) using NucleoSpin Plasmid (LOT 1306/003 Macherey-Nagel) :
AGA2P = 72 .3 ng/µl (260/280 = 1.79 ; 260/230 = 0.76)
GMCSF (1) = 304.2 ng/µl (280/260 = 1.82 ; 260/230 = 1.71)
Monday, 17th August 2015
Yeast Colony PCR on T1, T2 and T3 (induced/non induced)
Resuspend colony in 10 µl water
Microwave 8 minutes 5 µl of resuspended colony (900W)
Add 10 µl of mix dreamtaq
Add 2.5 µl primer (URA R5 2.0/ URA F1)
PCR yeast colony program:
Step 1 : 95°C - 5 minutes
Step 2 : 95°C – 30 seconds
Step 3 : 57/60/63°C – 30 seconds
Step 4 : 72°C – 2 minutes (repeat step 2-4, 41 cycles)
Step 4 : 72°C – 10 minutes
Step 5 :  4°C – Pause
T1/2/3 Colony PCR gel electrophoresis (1% agarose) :
Expected sizes :
T1 => OVA2 (290 bp)
T2 => AGA1P (2225 bp) & AGA2P OVA1 (461 bp)
T3 => AGA1P (2285 bp) & AGA2P OVA1 DEC205 (1381 bp)
Miniprep of ADH1 Mat alpha IFN gamma using NucleoSpin Plasmid (LOT 1306/003 Macherey-Nagel) :
IFN 1 = 343.5 ng/µl
IFN 4 = 290 ng/µl
IFN 6 = 265 ng/µl
IFN 7 = 377 ng/µl
Tuesday, 18th August 2015
We decided to remake the yeast colony PCR from 17/08 using same protocol to check our results a second time.
We tested plates of the same construction, T1, T2 and T3, made by different experimentators. For T1, we tested 3 different plates, each at 2 different temperatures : 53 and 60°C.
We also tested T3 with 3 different sets of primers (see gel picture below). Yeast Colony PCR on T1, T2 and T3 (induced/non induced)
Resuspend colony in 10 µl water
Microwave 8 minutes 5 µl of resuspended colony (900W)
Add 10 µl of mix dreamtaq
Add 2.5 µl primer (URA R5 2.0/ URA F1)
PCR yeast colony program :
Step 1 : 95°C - 5 minutes
Step 2 : 95°C – 30 seconds
Step 3 : 57/60/63°C – 30 seconds
Step 4 : 72°C – 2 minutes (repeat step 2-4, 41 cycles)
Step 4 : 72°C – 10 minutes
Step 5 :  4°C – Pause
T1/2/3 Colony PCR gel electrophoresis (1% agarose):
Primers used :
T1/T2/WT : URA F1/R5 2.0
T3.1 : URA R5 2.0/F1
T3.2 : URA F1/R1
T3.3 : URA F2/ R5.2.0
Wednesday, 19th August 2015
We started taking care of our biobricks ! We planned to deposite four biobricks : DEC205, OVA2, IFNgamma and Mat alpha IFN gamma. For OVA2 and DEC205, we just amplified the fragments with primers designed to make the fragment fit into the pSB1C3 plasmid. For IFN gamma and Mat alpha IFN gamma, as IFN gamma presented a restriction site, we proceeded to do a site directed mutagenenis to remove it.  Mutagenesis of IFNgamma and Mat alpha IFN gamma: We performed two consecutive PCR, the first one (PCR1) to introduce a mutation at the desired site and the second one (PCR2) to amplify the entire fragment (see 21/08). In between, PCR clean-ups were done using Nucleospin Gel and PCR Clean-up (LOT :1504/001).
IFNgamma PCR1
PCR1 primers:
- P005 FN IFNgamma/RV Mat IFNgamma
- FW Mat IFNgamma/P006 RV IFNgamma
Q5 PCR mix (50µl):
2,5 µl forward primers
2,5 µl reverse primers
1 µl New insert1 m2 Cter
19 µl water
25 µl PCR mix (Q5)
Q5 PCR Program
step 1 : 98°C – 30 seconds
step 2 : 98°C – 10 seconds
step 3 : 57, 60 or 63°C
step 4 : 72°C – 1 min (repeat steps 2-4 for 40 cycles)
step 5 : 16°C - Hold
Mat alpha IFN gamma PCR1
Q5 PCR mix (50µl):
2,5 µl P007 FW Mat alpha GMCSF
2,5 µl P007 RV Mat IFN gamma
1 µl New insert1 m2 Cter
19 µl water
25 µl PCR mix (Q5)
Q5 PCR Program
step 1 : 98°C – 30 seconds
step 2 : 98°C – 10 seconds
step 3 : 57, 60 or 63°C
step 4 : 72°C – 1 min (repeat steps 2-4 for 40 cycles)
step 5 : 16°C - Hold
Thursday, 20th August 2015
Mutagenesis of IFNgamma and Mat alpha IFN gamma: Gel electrophoresis (2% agarose) for the PCR1 products
Friday, 21th August 2015
We checked by digestion the following samples :
- ADH1 Mat alpha IFN gamma (pYGG1) (sample « IFNgamma)
- pYGG1
- water
Digestion mix :
2 µl NEB 2.1 Buffer
1 µl Hind III
2 µl DNA
15 µl water
We let the mix, 1 hour at 37°C and 500 rpm.
Digestion gel electrophoresis (agarose 1%) :
Friday, 21th August 2015
We checked by digestion the following samples :
- AGA2P OVA1 DEC205 pYGG1 (sample « SD »)
- AGA2P OVA1 pYGG1 (sample « DEC -»)
- OVA2 pYGG2 (sample « OVA2 »)
- pYGG2
- water
Digestion mix :
2 µl NEB 2.1 Buffer
1 µl Hind III
2 µl DNA
15 µl water
We let the mix, 1 hour at 37°C and 500 rpm.
Digestion gel electrophoresis (agarose 1%) :
Monday, 24th August 2015
PCR of the fragment RFP of pYGG1 for biosensor design using primers 3B FW 2.0 and  3B reverse :
Q5 PCR mix (50µl)
2,5 µl 3B FW 2.0
2,5 µl 3B RV
1 µl New insert1 m2 Cter
19 µl water
25 µl PCR mix (Q5)
Q5 PCR Program
step 1 : 95°C – 30 seconds
step 2 : 95°C – 10 seconds
step 3 : 57, 60 or 63°C
step 4 : 72°C – 1 min (repeat steps 2-4 for 40 cycles)
step 5 : 12°C - Hold
Mutagenesis of IFNgamma and Mat alpha IFN gamma: PCR2
IFN gamma PCR2 primers :
P015 FW IFN gamma/P016 RV IFNgamma
Mat alpha IFN gamma PCR2 primers :
P015 FW IFN gamma / P016 RV IFN gamma
Q5 PCR mix (50µl) :
2,5 µl forward primers
2,5 µl reverse primers
1 µl New insert1 m2 Cter
19 µl water
25 µl PCR mix (Q5)
Q5 PCR program :
step 1 : 95°C – 30 seconds
step 2 : 95°C – 10 seconds
step 3 : 57, 60 or 63°C
step 4 : 72°C – 1 min (repeat steps 2-4 for 40 cycles)
step 5 : 12°C - Hold
Wednesday, 26th August 2015
Miniprep and nanodrop of pYGG1, pYGG2 and AGA1P-pYGG2 using NucleoSpin Plasmid (LOT 1306/003 Macherey-Nagel):
pYGG1 = 68 ng/µL
pYGG2 = 127 ng/µL
AGA1P (pYGG2) = 32 ng/µL
Wednesday, 26th August 2015
PCR Clean up of biobrick Mata-IFNg and biobrick IFNg
- BB Mata-IFNg = 50 ng/µL
- BB IFNg = 35 ng/µL
Gel electrophoresis (1% agarose) of biobricks Mata-IFNg (BB Mata-IFNg) and IFNg (BB IFNg) :
Wednesday, 26th August 2015
PCR Clean up of Biosensor3
- Biosensor 3 = 46 ng/µL
Gel electrophoresis (1% agarose) of biosensor 3
Thursday, 27th August 2015
E.coli transformation of biosensor 2 and 3
Protocol :
1) Add 10 µl plasmids  to 50 µl E. coli competent cells on ice and let the mix rest on ice for 15 min
2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min
3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour
4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin
5) Put plates in growth incubators at 37°C for 24 hours
Thursday, 27th August 2015
PCR of biobrick OVA1
Q5 PCR mix (50µl) :
2,5 µl FWD
2,5 µl  RV
1 µl New insert1 m2 Cter
19 µl water
25 µl PCR mix (Q5)
Q5 PCR program :
step 1 : 95°C – 30 seconds
step 2 : 95°C – 10 seconds
step 3 : 57, 60 or 63°C
step 4 : 72°C – 1 min (repeat steps 2-4 for 40 cycles)
step 5 : 12°C - Hold
Gel electrophoresis (2% agarose) :
Digestion of the following biobricks using NEB kit :
- BB IFNg
- BB Mata-IFNg
- BB DEC 205
- BB OVA1
- BB OVA2
- pSB1C3
Digestion mix :
2µL NEB Buffer 2.1
0.5µL Eco RI
0.5 µL PstI
500 ng DNA
water qsp 20µL
The mix was left at 1h 37°C under agitation 500rpm and 20 minutes at 80°C.
Friday, 28th August 2015
PCR clean-up and nanodrop of digested products:
- BB IFNg D = 13 ng/µL
- BB Mata-IFNg D = 10 ng/µL
- BB DEC 205 D = 15.4 ng/µL
- BB OVA1 D = 9 ng/µL
- BB OVA2 D = 14 ng/µL
- pSB1C3 D = 7.4 ng/µL
Ligation of digested products into pSB1C3 with T4 Ligase (ratio 3:1):
2µL T4 DNA Ligase Buffer
1µL T4 DNA Ligase
50ng pSB1C3
3:1 insert
water qsp 20µL
We let the mix at room temperature for 30 minutes and at 65°C for 10 minutes.
E.coli transformation with ligated products:
Protocol:
1) Add 10 µl plasmids  to 50 µl E. coli competent cells on ice and let the mix rest on ice for 15 min
2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min
3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour
4) Plate 50 µl and 150 µl to plates containing LB agar media with chloramphenicol
5) Put plates in growth incubators at 37°C for 24 hours
Friday, 28th August 2015
Colony PCR of transformed E.coli with biosensor 2 and 3
PCR colony mix :
1 µl of resuspended colony in 20 µl LB
1 µl URA F1
1 µl URA R5 2.0
7 µl water
10 µl PCR mix (Dreamtaq)
PCR colony program :
Step 1 : 95°C -  5 min
Step 2 : 95°C – 30 seconds
Step 3 : 50°C – 30 seconds
Step 4 : 72°C –  2 minutes (repeat step 2-4, 45 times)
Step 5 : 72°C – 10 minutes
Step 6 :  4°C – Pause
Gel electrophoresis (1% agarose)
Miniculture of biosensor 2 and 3
Monday, 31st August 2015
Colony PCR gel electrophoresis (1% agarose)
Tuesday, 1st September 2015
Glycerol library of Biosensor 2 and 3 :
7.5 μl 50% glycerol + 7.5 μl miniculture
samples names:
"145" : Biosensor 2
"146" : Biosensor 3
Miniprep and nanodrop of biosensor 2 and 3 using NucleoSpin Plasmid (LOT 1306/003 Macherey-Nagel) :
2.1 = 380 ng/µL
2.2 = 440 ng/µL
2.3 = 392.8 ng/µL
3.1 = 225 ng/µL
3.2 = 250 ng/µL
3.3 = 85 ng/µL
Tuesday, 1st September 2015
Gal1 deletion: Q5 PCR of IFNγ and Biosensor 3
Q5 PCR primers :
- IFNγ amplification : Gal1- Fwd IFNγ / Gal1- Rv IFNγ
- Biosensor 3 amplification : Fwd Gal1- sur ligate RFP / Rv Gal1- sur ligate RFP
Q5 PCR mix (50µl):
2,5 µl FWD
2,5 µl  RV
1 µl New insert1 m2 Cter
19 µl water
25 µl PCR mix (Q5)
Q5 PCR program :
step 1 : 95°C – 30 seconds
step 2 : 95°C – 10 seconds
step 3 : 60°C
step 4 : 72°C – 1 min (repeat steps 2-4 for 40 cycles)
step 5 : 12°C - Hold
Gal1 deletion: Digestion DpnI
Digestion mix :
50μL PCR reaction
5μL NEB 2.1 Buffer
1μL DpnI
We let the mix at 37°C for 1 hour and then at 80°C for 20 minutes
PCR clean-up
Gel electrophoresis (0.7% agarose) :
Wednesday, 2nd September 2015
Miniprep and nanodrop of biobricks using NucleoSpin Plasmid (LOT 1306/003 Macherey-Nagel):
- BB IFNγ-p 1 = 85ng/μL
- BB IFNγ-p 2 = 40ng/μL
- BB MATα-IFNγ-p 1 = 85ng/μL
- BB MATα-IFNγ-p 2 = 100ng/μL
- BB DEC 205-p 1 = 105ng/μL
- BB DEC 205-p 2 = 90ng/μL
- BB OVA1-p 1 = 86ng/μL
- BB OVA1-p 2 = 72ng/μL
- BB OVA2-p 1 = 50ng/μL
- BB OVA2-p 2 = 57ng/μL
The biobricks were sent to sequencing:
Sequencing mix:
2.5μL primer VF2
2.5μL primer VR
5μL DNA (Ci=80-100ng/μL)
Each samples were sent with duplicates, which makes 20 tubes in total.
Thursday, 3rd September 2015
Gal1 deletion: Phosphorylation
Mix (10µl) :
2μL DNA
1μL T4 DNA Ligase buffer (with 10Mm ATP)
1μL PNK
6μL H20
We let the mix at 37°C for 1 hour.
Gal1 deletion: Ligation
Mix (10µl) :
1μL T4 DNA Ligase buffer
1μL T4 DNA Ligase
8μL H20
We let the mix at room temperature for 1 hour.
Gal1 deletion E.coli transformation with ligated products:
Protocol:
1) Add 10 µl plasmids  to 50 µl E.coli competent cells on ice and let the mix rest on ice for 15 min
2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min
3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour
4) Plate 50 µl and 150 µl to plates containing LB agar media with chloramphenicol
5) Put plates in growth incubators at 37°C for 24 hours
To top
