<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Evry_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Evry/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="row"><DIV class="side-menu"><NAV class="navbar navbar-default" role="navigation"><DIV class="navbar-header"><DIV class="brand-wrapper"><BUTTON type="button" class="navbar-toggle" style="background-color:  #3f51b5;"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV></DIV><DIV class="side-menu-container" id="side-navigation"><UL class="nav navbar-nav"><LI class="panel panel-default dropdown" id="project-menu"><A data-toggle="collapse" href="#dropdown-project"> Project </A></LI><LI id="project_description"><A href="https://2015.igem.org/Team:Evry/Description">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Project/Chassis">Chassis choice</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Project/Encapsulation">Environment modulation</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Project/Biosensor">Hypoxia Bio-sensor</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Design">Proof of concept</A></LI></UL></DIV></NAV></DIV><LI class="panel panel-default dropdown" id="software-menu"><A data-toggle="collapse" href="#dropdown-software"> Antigen prediction </A></LI><LI><A href="https://2015.igem.org/Team:Evry/Software"><SPAN style="font-size:5%; color: transparent;">o</SPAN>Overview</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Software/Pipeline">Pipeline</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Software/Predictions">Prediction</A></LI></DIV></DIV><LI class="panel panel-default dropdown" id="practices-menu"><A data-toggle="collapse" href="#dropdown-practices"> Practices </A></LI><LI><A href="https://2015.igem.org/Team:Evry/Practices">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Practices/State">Funding state of art</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Practices/Innovative">Innovative funding</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Practices/Society">Society perception</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Practices/Conclusion">Conclusion</A></LI></DIV></DIV><LI class="panel panel-default dropdown" id="achievements-menu"><A data-toggle="collapse" href="#dropdown-achievements"> Achievements </A></LI><LI><A href="https://2015.igem.org/Team:Evry/Medals">Medal fulfillment</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Parts">Parts</A></LI></DIV><LI class="panel panel-default dropdown" id="notebook-menu"><A data-toggle="collapse" href="#dropdown-nb"> Notebook </A></LI><LI><A href="https://2015.igem.org/Team:Evry/Notebook">Day notes</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Protocols">Protocols</A></LI><LI class="panel panel-default dropdown" id="team-menu"><A data-toggle="collapse" href="#team-dropdown"> Team </A></LI><LI><A href="https://2015.igem.org/Team:Evry/Team">Members</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Acknowledgements">Acknowledgements</A></LI><DIV class="container"><DIV class="side-body" id="content-body"><DIV class="page-header"><H1>Notebook</H1></DIV><SECTION class="page-section"><P class="lead text-justify"><SPAN class="text-muted">Here is our lab notebook.</SPAN> Follow all the wet lab experiments we did, day by day. </P><P class="text-justify">Use the navigation menu below to jump to notebook entries of a particular month. </P></SECTION><DIV id="top-menu"><UL><LI style="height: 34px; padding-top: 8px;" class="text-muted"><STRONG>Jump to:</STRONG></LI><LI><A href="#jun-section">June </A></LI><LI><A href="#jul-section">July</A></LI><LI><A href="#aug-section">August</A></LI><LI><A href="#sep-section">September</A></LI></UL></DIV><SECTION class="page-section" id="jun-section"><DIV id="secretion_06_12" class="notebook-entry"><H4>Friday, 12<SUP>th</SUP> June 2015</H4><P class="text-justify"><STRONG>Extraction of AgA1P from yeast genome with BSAI</STRONG>
AGA1P was extracted with BSAI overhangs for subsequent cloning from W303 and BY4000, according to <EM>Looke et al., PMC 2011</EM>.
</P><P class="text-justify"><SPAN class="text-primary">Protocol</SPAN><OL><LI> 1) Resuspend one yeast colony in 100 µl of 200 mM LiAc, 1% SDS solution and incubate at 70°C</LI><LI> 2) Add 300 µl of 36% ethanol and vortex</LI><LI> 3) Spin down DNA at 15 000 g for 3 minute</LI><LI> 4) Wash pellet with 70% ethanol</LI><LI> 5) Disolve pellet in 100 µl of water and spin down debris at 15 000 g for 15 seconds</LI></OL></P><P class="text-justify"><STRONG>PCR of AGA1P with primers AGA1P R1/F1 with Q5 polymerase using a gradient (57/60/63°C)</STRONG></P><P class="text-justify"><SPAN class="text-primary">Q5 PCR Program</SPAN><OL><LI>step 1 : 98°C – 30 seconds</LI><LI>step 2 : 98°C – 10 seconds</LI><LI>step 3 : 57, 60 or 63°C</LI><LI>step 4 : 72°C – 1 min (repeat steps 2-4 for 40 cycles)</LI><LI>step 5 : 16°C - Hold</LI></OL></P></DIV><DIV id="secretion_06_14" class="notebook-entry"><H4>Monday, 14<SUP>th</SUP> June 2015</H4><P class="text-justify"><STRONG>PCR on yeast genome for site directed mutagenesis :</STRONG>
ADH1 amplification with 2 sets of primers:
</P><P class="text-justify"><OL><LI>1) primers ADH1 sub F1 + ADH1 sub R1 with 3 anealing temperatures : 57°C (B1), 60°C (B2), 63°C (B3)</LI><LI>2) primers ADH1 sub F2 + ADH1 sub R2 with 3 anealing temperatures : 57°C (B4), 60°C (B5), 63°C (B6)</LI></OL></P><P class="text-justify">
Malpha IFN gamma amplification with 2 sets of primers:
</P><P class="text-justify"><OL><LI>1) primers Mfalpha IFNgamma F1/R1 with 3 anealing temperatures : 57°C (B7), 60°C (B8), 63°C (B9)</LI><LI>2) primers Mfalpha IFNgamma F1 + Mfalpha GMCSF R1 with 3 anealing temperatures :57°C (B10), 60°C (B11), 63°C (B12)</LI></OL></P></DIV><DIV id="sd_06_19" class="notebook-entry"><H4>Friday, 19<SUP>th</SUP> June 2015</H4><P class="text-justify"><STRONG>Golden gates</STRONG></P><P class="text-justify"><SPAN class="text-primary">Golden gates mix (20µl) and inserts:</SPAN><OL><LI>2 µl T4 ligase buffer 10x</LI><LI>0.5 µl BSAI</LI><LI>0.5 µl T4 ligase</LI><LI>water (qsp 20µl)</LI></OL></P><P class="text-justify">
Golden gate 1 (A1) inserts:
<OL><LI>- New insert1 m1 Cter (AGA2P)</LI><LI>- New insert1 m2 Cter (-OVA1 DEC205)</LI><LI>- Insert1 m2 (GAL10 GAL7 AGA1P)</LI><LI>- Insert1 m3 extracted from yeast genome (AGA1P)</LI><LI>- pYGG1</LI></OL></P><P class="text-justify">
Golden gate 2 (A2) inserts:
<OL><LI>- New insert1 m1 Cter (AGA2P)</LI><LI>- Insert1 m2 (GAL10 GAL7 AGA1P)</LI><LI>- Insert1 m3 extracted from yeast genome (AGA1P)</LI><LI>- pYGG1</LI></OL></P><P class="text-justify">
Golden gate 3 (A3) inserts :
<OL><LI>- insert3 (OVA2)</LI><LI>- pYGG2</LI></OL></P></DIV><DIV id="sd_06_20" class="notebook-entry"><H4>Saturday, 20<SUP>th</SUP>June 2015</H4><P class="text-justify"><STRONG>PCR colony using primers URA F1 and URA R5 2.0 on A1, A2 and A3</STRONG></P><P class="text-justify"><SPAN class="text-primary"> PCR colony mix</SPAN><OL><LI>1 µl of resuspended colony in 20 µl LB</LI><LI>1 µl URA F1</LI><LI>1 µl URA R5 2.0</LI><LI> 7 µl water</LI><LI>10 µl PCR mix (Dreamtaq)</LI></OL></P><P class="text-justify"><SPAN class="text-primary"> PCR colony Program</SPAN><OL><LI>Step 1 : 95°C - 5 min</LI><LI>Step 2 : 95°C – 30 s</LI><LI>Step 3 : 50°C – 30s</LI><LI>Step 4 : 72°C – 2 min (repeat step 2-4, 45 times)</LI><LI>Step 5 : 72°C – 10 min</LI><LI>Step 6 :  4°C – Pause</LI></OL></P><P class="text-justify"><STRONG>PCR colony products gel electrophoresis (1% agarose)</STRONG></P><P class="text-justify">
Results: Clones from golden 3 (A3, see 19/06) present the right size. Clones from golden 1 and 2 (A1 and A2, see 19/06) are negative. It seems there was no amplification.
</P><P class="text-justify"><STRONG>Miniculture (40)</STRONG></P><P class="text-justify">
The remaining resusepended <I>E.coli</I> was put into 4 mL of Luria Bertoni media containing with 4 µl ampicilin. 
</P></DIV><DIV id="sd_06_21" class="notebook-entry"><H4>Sunday, 21<SUP>th</SUP>June 2015</H4><P class="text-justify"><STRONG>Repeat of PCR colony on A1, A2 and A3</STRONG></P><P class="text-justify"><SPAN class="text-primary"> PCR colony mix</SPAN><OL><LI>1 µl of resuspended colony in 20 µl LB</LI><LI>1 µl URA F1</LI><LI>1 µl URA R5 2.0</LI><LI> 7 µl water</LI><LI>10 µl PCR mix (Dreamtaq)</LI></OL></P><P class="text-justify"><SPAN class="text-primary"> PCR colony Program</SPAN><OL><LI>Step 1 : 95°C - 5 min</LI><LI>Step 2 : 95°C – 30 s</LI><LI>Step 3 : 50°C – 30s</LI><LI>Step 4 : 72°C – 2 min (repeat step 2-4, 45 times)</LI><LI>Step 5 : 72°C – 10 min</LI><LI>Step 6 :  4°C – Pause</LI></OL></P><P class="text-justify"><STRONG>PCR colony products gel electrophoresis (1% agarose)</STRONG></P><P class="text-justify">
Results: We got clones with expected sizes.
</P></DIV><DIV id="secretion_06_22" class="notebook-entry"><H4>Monday, 22<SUP>nd</SUP>June 2015</H4><P class="text-justify"><STRONG>PCR clean up and nanodrop :</STRONG><OL><LI>A1 = 209.1 ng/µl (insert1 m3)</LI><LI>A2 = 56.1 ng/µl (AGA2P)</LI><LI>A6 = 114.3 ng/µl (IFN gamma)</LI><LI>A7 = 93.7 ng/µl (GMCSF)</LI><LI>B8 = 114 ng/µl (Malpha IFNgamma)</LI><LI>B10 = 134.2 ng/µl (Malpha GMCSF)</LI><LI>B11 = 163.9 ng/µl (Malpha GMCSF)</LI><LI>C1 = 180.7 ng/µl (ADH1)</LI><LI>C2 = 302.9 ng/µl (ADH1)</LI></OL></P><P class="text-justify"><STRONG>G2/G2/G3/G4 Golden gates</STRONG></P><P class="text-justify"><SPAN class="text-primary">Golden gate mix (20 µl) and inserts:</SPAN><OL><LI>2 µl T4 ligase buffer 10x</LI><LI>0.5 µl BSAI + 0.5 T4 ligase</LI><LI>water (qsp 20µl)</LI></OL></P><P class="text-justify">
Inserts used for ADH1 Malpha IFNgamma construction (G1):
<OL><LI>- C1</LI><LI>- B8</LI><LI>- A6</LI><LI>- pYGG1</LI></OL></P><P class="text-justify">
Inserts used for ADH1 Malpha GMCSF construction (G2):
<OL><LI>- C1</LI><LI>- B10</LI><LI>- A7</LI><LI>- pYGG1</LI></OL></P><P class="text-justify">
Inserts used for ADH2 Malpha IFNgamma (G3):
<OL><LI>- C2</LI><LI>- B8</LI><LI>- A6</LI><LI>- pYGG1</LI></OL></P><P class="text-justify">
Inserts used for ADH2 … GMCSF construction (G4):
<OL><LI>- C2</LI><LI>- B11</LI><LI>- A7</LI><LI>- pYGG1</LI></OL></P><P class="text-justify"><STRONG>E. coli transformation with golden gates G1, G2, G3, G4 products</STRONG></P><P class="text-justify"><SPAN class="text-primary">Protocol :</SPAN><OL><LI>1) Add 10 µl plasmids  to 50 µl E. coli competent cells on ice and let the mix rest on ice for 15 min</LI><LI>2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min</LI><LI>3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour</LI><LI>4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin</LI><LI>5) Put plates in growth incubators at 37°C for 24 hours</LI></OL></P></DIV><DIV id="secretion_06_23" class="notebook-entry"><H4>Tuesday, 23<SUP>rd</SUP> June 2015</H4><P class="text-justify"><STRONG>G2/G2/G3/G4 colony PCR</STRONG></P><P class="text-justify"><SPAN class="text-primary"> PCR colony mix</SPAN><OL><LI>1 µl of resuspended colony in 20 µl LB</LI><LI>1 µl URA F1</LI><LI>1 µl URA R5 2.0</LI><LI> 7 µl water</LI><LI>10 µl PCR mix (Dreamtaq)</LI></OL></P><P class="text-justify"><SPAN class="text-primary"> PCR colony Program</SPAN><OL><LI>Step 1 : 95°C - 5 min</LI><LI>Step 2 : 95°C – 30 s</LI><LI>Step 3 : 50°C – 30s</LI><LI>Step 4 : 72°C – 2 min (repeat step 2-4, 45 times)</LI><LI>Step 5 : 72°C – 10 min</LI><LI>Step 6 :  4°C – Pause</LI></OL></P><P class="text-justify"><STRONG>G2/G2/G3/G4 minicultures</STRONG></P><P class="text-justify">
The remaining resusepended <I>E.coli</I> was put into 4 mL of Luria Bertoni media containing with 4 µl ampicilin. 
</P></DIV><DIV id="secretion_06_24 class=" notebook-entry=""><H4>Wednesday, 24<SUP>th</SUP> June 2015</H4><P class="text-justify"><STRONG>G2/G2/G3/G4 library from minicultures (see 23/06) :</STRONG></P><P class="text-justify">
750 µl of G1 to G8 were mixed with 750 µl of glycerol 50% and put in the freezer at -80°C.
</P><P class="text-justify"><STRONG>G2/G2/G3/G4 Minipreped using NucleoSpin Plasmid (LOT 1306/003 Macherey-Nagel) :</STRONG></P><P class="text-justify">
Samples corresponding to golden gate G1:
<OL><LI>G1 mini = 270.5 ng/µl (260/280 = 1.81 ; 260/230 = 1.72)</LI><LI>G2 mini = 287.4 ng/µl (260/280 = 1.84 ; 260/230 = 1.63)</LI></OL></P><P class="text-justify">
Samples corresponding to golden gate G2:
<OL><LI>G3 mini = 146.4 ng/µl (260/280 = 1.81 ; 260/230 = 1.42)</LI><LI>G4 mini = 264.3 ng/µl (260/280 = 1.79 ; 260/230 = 1.80)</LI></OL></P><P class="text-justify">
Samples corresponding to golden gate G3 (ADH2 Malpha IFNgamma):
<OL><LI>G5 mini = 164.9 ng/µl (260/280 = 1.82 ; 260/230 = 1.49)</LI><LI>G6 mini = 128.1 ng/µl (260/280 = 1.84 ; 260/230 = 1.56)</LI></OL></P><P class="text-justify">
Samples corresponding to golden gate G4:
<OL><LI>G7 mini = 587.4 ng/µl (260/280 = 1.67 ; 260/230 = 0.63)</LI><LI>G8 mini = 160.2 ng/µl (260/280 = 1.81 ; 260/230 = 1.91)</LI></OL></P></DIV><DIV id="secretion_06_25" class="notebook-entry"><H4>Thursday, 25<SUP>th</SUP> June 2015</H4><P class="text-justify"><STRONG>G samples were sent to sequencing using 3 primers sets :</STRONG></P><P class="text-justify"><OL><LI>- URA F1/SR1 lig IFN gamma</LI><LI>- S2 lig IFN gamma/SR2 lig IFN gamma</LI><LI>- URA F1/SR1 lig IFN gamma</LI></OL></P><P class="text-justify"><SPAN class="text-primary">Sequencing mixes (10µl):</SPAN><OL><LI>G4 : 4 µl DNA + 1.25 µl/primer + qsq water</LI><LI>G6 : 6 µl DNA + 1.25 µl/primer + qsq water</LI></OL></P></DIV></SECTION><SECTION class="page-section" id="jul-section"><DIV id="sd_07_03" class="notebook-entry"><H4>Friday, 3<SUP>rd</SUP> July 2015</H4><P class="text-justify"><STRONG>AGA2P -OVA1 DEC205 (pYYG1) construction : Nanodrop results</STRONG><OL><LI>New insert1 m1 Cter (AGA2P) = 10 ng/µl</LI><LI>New insert1 m2 Cter PCR1 (-OVA1 DEC205) = 77 ng/µl</LI><LI>pYGG1 (P11) = 107.6 ng/µl</LI></OL></P><P class="text-justify"><STRONG>AGA2P OVA1 DEC205 (pYGG1) new golden gate</STRONG></P><P class="text-justify"><SPAN class="text-primary">Golden gate mix (20µl):</SPAN><OL><LI>15.5 µl H20</LI><LI>2 µl T4 ligase buffer 10x</LI><LI>0.425 µl New insert1 m1 Cter (AGA2P)</LI><LI>0.194 µl New insert1 m2 Cter PCR1 (-OVA1 DEC205)</LI><LI>1 µl pYGG1</LI><LI>0.5 µl BSA I</LI><LI>0.5 µl T4 ligase</LI></OL></P><P class="text-justify"><STRONG>AGA2P OVA1 DEC205 (pYGG1) E. coli transformation</STRONG></P><P class="text-justify"><SPAN class="text-primary">Protocol:</SPAN><OL><LI>1) Add 10 µl plasmids  to 50 µl E. coli competent cells on ice and let the mix rest on ice for 15 min</LI><LI>2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min</LI><LI>3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour</LI><LI>4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin</LI><LI>5) Put plates in growth incubators at 37°C for 24 hours</LI></OL></P></DIV><DIV id="sd_07_05" class="notebook-entry"><H4>Sunday, 5<SUP>th</SUP> July 2015</H4><P class="text-justify"><STRONG>AGA2P OVA1 DEC205 (pYGG1) construction: PCR colony</STRONG></P><P class="text-justify"><SPAN class="text-primary"> PCR colony mix (20 µl):</SPAN><OL><LI>1 µl of resuspended colony in 20 µl LB</LI><LI>1 µl URA F1 (primers)</LI><LI>1 µl URA R5 2.0 (primers)</LI><LI>7 µl water</LI><LI>10 µl PCR mix (Dreamtaq)</LI></OL></P><P class="text-justify"><SPAN class="text-primary"> PCR colony Program</SPAN><OL><LI>Step 1 : 95°C - 5 min</LI><LI>Step 2 : 95°C – 30 s</LI><LI>Step 3 : 50°C – 30s</LI><LI>Step 4 : 72°C – 2 min (repeat step 2-4, 45 times)</LI><LI>Step 5 : 72°C – 10 min</LI><LI>Step 6 :  4°C – Pause</LI></OL></P></DIV><DIV id="sd_07_07" class="notebook-entry"><H4>Tuesday, 7<SUP>th</SUP> July 2015</H4><P class="text-justify"><STRONG>AGA2P OVA1 DEC205 (pYGG1) construction: Minicultures</STRONG></P><P class="text-justify">
The remaining resusepended <I>E.coli</I> was put into 4 mL of Luria Bertoni media containing with 4 µl ampicilin. 
</P></DIV><DIV id="sd_07_07" class="notebook-entry"><H4>Wednesday, 8<SUP>th</SUP> July 2015</H4><P class="text-justify"><STRONG>AGA2P OVA1 DEC205 (pYGG1) construction: PCR colony</STRONG></P><P class="text-justify"><SPAN class="text-primary"> PCR colony mix:</SPAN><OL><LI>1 µl of resuspended colony in 20 µl LB</LI><LI>1 µl URA F1</LI><LI>1 µl URA R5 2.0</LI><LI> 7 µl water</LI><LI>10 µl PCR mix (Dreamtaq)</LI></OL></P><P class="text-justify"><SPAN class="text-primary">PCR colony program :</SPAN><OL><LI>Step 1 : 95°C -  5 min</LI><LI>Step 2 : 95°C – 30 seconds</LI><LI>Step 3 : 50°C – 30 seconds</LI><LI>Step 4 : 72°C –  2 minutes (repeat step 2-4, 45 times)</LI><LI>Step 5 : 72°C – 10 minutes</LI><LI>Step 6 :  4°C – Pause</LI></OL></P><P class="text-justify"><STRONG>AGA2P OVA1 DEC205 (pYGG1) construction: PCR colony gel electrophoresis (1% agarose)</STRONG></P></DIV><DIV id="sd_07_09" class="notebook-entry"><H4>Thursday, 9<SUP>th</SUP> July 2015</H4><P class="text-justify"><STRONG>Miniculture of transformed AGA2P OVA1 DEC205 (pYGG1) <I>E.coli</I></STRONG>19 µl colony resuspended Luria Bertoni media in 4 mL of Luria Bertoni media complemented with 4 µl of ampicilin (100 mg/µl) and put to incubate at 37°C overnight.
Samples names : D13/ D23/ D33 / D43
</P><DIV id="sd_07_10" class="notebook-entry"><H4>Friday, 10<SUP>th</SUP> July 2015</H4><P clas="text-justify"><STRONG>Samples of AGA2P OVA1 DEC205 (pYGG1) construction were minipreped using NucleoSpin Plasmid (LOT 1306/003 Macherey-Nagel) and nanodroped : </STRONG></P><P clas="text-justify"><OL><LI>Mini P1 = 449.9 µl (260/230= 1.82 ; 260/280= 1.89) (from D13 and D23, see 09/07)</LI><LI>Mini P2 = 414.2 µl (260/230= 1.83 ; 260/280= 1.94) (from D33 and D43, see 09/07)</LI></OL></P></DIV><DIV id="secretion_07_13" class="notebook-entry"><H4>Monday, 13<SUP>th</SUP> July 2015</H4><P class="text-justify"><STRONG>ADH1 Mat alpha IFN gamma (pYGG1) construction: Golden gate</STRONG></P><P class="text-justify"><SPAN class="text-primary">Golden gate mix :</SPAN><OL><LI>11.35 µl water</LI><LI>2 µl T4 ligase buffer 10x</LI><LI>0.177 µl ADH1</LI><LI>4.473 µl (Mat alpha IFN gamma) = Gblock</LI><LI>1 µl pYGG1</LI><LI>0.5 µl BSA I</LI><LI>0.5 µl T4 ligase</LI></OL></P><P class="text-justify"><SPAN class="text-primary">Golden gate program :</SPAN></P><P class="text-justify"><STRONG>Mat alpha IFN gamma ADH1 (pYGG1) E. coli transformation</STRONG></P><P class="text-justify"><SPAN class="text-primary">Protocol :</SPAN><OL><LI>1) Add 10 µl plasmids  to 50 µl E. coli competent cells on ice and let the mix rest on ice for 15 min</LI><LI>2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min</LI><LI>3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour</LI><LI>4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin</LI><LI>5) Put plates in growth incubators at 37°C for 24 hours</LI></OL></P></DIV><DIV id="sd_07_13" class="notebook-entry"><H4>Monday, 13<SUP>th</SUP> July 2015</H4><P class="text-justify"><STRONG>Samples of AGA2P OVA1 DEC205 (pYGG1) construction were minipreped using NucleoSpin Plasmid (LOT 1306/003 Macherey-Nagel) and nanodroped :</STRONG><OL><LI>- AGA2P OVA1 DEC205 1 (SD1) = 449.9 ng/µl</LI><LI>- AGA2P OVA1 DEC205 2 (SD2) = 414.2 ng/µl</LI></OL></P><P class="text-justify"><STRONG>Samples of AGA2P OVA1 DEC205 (pYGG1) construction, Mini P1 and Mini P2 were sent to sequencing</STRONG></P></DIV><DIV id="secretion_07_15" class="notebook-entry"><H4>Wednesday, 15<SUP>th</SUP> July 2015</H4><P class="text-justify"><STRONG>Mat alpha IFN gamma ADH1 (pYGG1) construction : PCR colony</STRONG></P><P class="text-justify"><SPAN class="text-primary">PCR colony mix (20 µl):</SPAN><OL><LI>10 µl PCR mix (DreamTaq)</LI><LI>1 µl URA F1</LI><LI>1 µl URA R5 2.0</LI><LI>1 µl colony resuspended in Luria Bertoni media</LI><LI>7 µl water</LI></OL></P><P class="text-justify"><SPAN class="text-primary">PCR colony program:</SPAN><OL><LI>Step 1 : 95°C - 5 minutes</LI><LI>Step 2 : 95°C – 30 seconds</LI><LI>Step 3 : 50°C – 30 seconds</LI><LI>Step 4 : 72°C – 2 minutes (Repeat step 2-4, 35 times)</LI><LI>Step 5 : 72°C – 10 minutes</LI><LI>Step 6 :  4°C – Pause</LI></OL></P></DIV><DIV id="secretion_07_16" class="notebook-entry"><H4>Thursday, 16<SUP>th</SUP> July 2015</H4><P class="text-justify"><STRONG>Mat alpha IFN gamma ADH1 (pYGG1) PCR colony products gel electrophoresis (1% agarose)</STRONG></P><P class="text-justify"><SPAN class="text-primary">Golden gate program :</SPAN></P><P class="text-justify">
Only clones A4, C6 and D7 corresponded to the size expected for (Mat alpha IFN gamma) ADH1 : 2200 pb.
</P><P class="text-justify"><STRONG>Miniculture</STRONG>19 µl colony resuspended Luria Bertoni media in 4 mL of Luria Bertoni media complemented with 4 µl of ampicilin (100 mg/µl) and put to incubate at 37°C overnight.
</P></DIV><DIV id="sd_07_17" class="notebook-entry"><H4>Friday, 17<SUP>th</SUP> July 2015</H4><P class="text-justify&quot;"><STRONG> OVA2 (pYGG2): golden gate</STRONG></P><P class="text-justify"><SPAN class="text-primary">Golden gate mix (20 µl):</SPAN><OL><LI>2 µl T4 ligase</LI><LI>0.5 µl BSA I</LI><LI>0.5 T4 µl DNA ligase</LI><LI>0.801 µl OVA2 (12,6 ng/µl) (Gblock)</LI><LI>0.714 µl pYGG2 (107.6 ng/µl)</LI><LI>15.4 µl water</LI></OL></P><P class="text-justify"><SPAN class="text-primary">Golden gate program :</SPAN></P></DIV><DIV id="sd_07_20" class="notebook-entry"><H4>Monday, 20<SUP>th</SUP> July 2015</H4><P class="text-justify"><STRONG> OVA2 (pYGG2) E. coli transformation </STRONG></P><P class="text-justify"><SPAN class="text-primary">Protocol :</SPAN><OL><LI>1) Add 10 µl plasmids  to 50 µl E. coli competent cells on ice and let the mix rest on ice for 15 min</LI><LI>2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min</LI><LI>3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour</LI><LI>4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin</LI><LI>5) Put plates in growth incubators at 37°C for 24 hours</LI></OL></P></DIV><DIV id="sd_07_21" class="notebook-entry"><H4>Tuesday, 21<SUP>st</SUP> July 2015</H4><P class="text-justify&quot;"><STRONG>OVA2 (pYGG2) E. coli transformation results</STRONG>Previously (see 2015-07-21) plates were put overnight into incubators at 37°C. 
There is no colonies on the plates, either because the E. coli mix died before being plated or because the golden gate failed in some ways. The latter hypothesis is unliky as we woµld have red negative colonies onto the plate. 
 =&gt; It turns out we did not use the right plasmid. We shoµld have used pYGG1
</P></DIV><DIV id="sd_07_23" class="notebook-entry"><H4>Thursday, 23<SUP>rd</SUP> July 2015</H4><P class="text-justify&quot;"><STRONG>OVA2 pYGG1 golden gate</STRONG></P><P class="text-justify"><SPAN class="text-primary">Golden gate mix (20 µl):</SPAN><OL><LI>2 µl T4 ligase</LI><LI>0.5 µl BSA I</LI><LI>0.5 T4 µl DNA ligase</LI><LI>1 µl OVA2 (12,6 ng/µl) (Gblock)</LI><LI>1 µl pYGG1 (107.6 ng/µl)</LI><LI>15 µl water</LI></OL></P><P class="text-justify"><SPAN class="text-primary">Golden gate program:</SPAN></P><P class="text-justify&quot;"><STRONG>OVA2 pYGG1 <I>E.Coli</I> transformation</STRONG></P><P class="text-justify"><SPAN class="text-primary">Protocol:</SPAN><OL><LI>1) Add 10 µl plasmids  to 50 µl E. coli competent cells on ice and let the mix rest on ice for 15 min</LI><LI>2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min</LI><LI>3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour</LI><LI>4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin</LI><LI>5) Put plates in growth incubators at 37°C for 24 hours</LI></OL></P><P class="text-justify"><STRONG>AGA2P OVA1 DEC205 minipreped samples were sent to sequencing using two sets of primers : </STRONG></P><P class="text-justify"><OL><LI>Seq.ID 26EB08 =&gt; SD1.1 (primers : URA F1 / URA F1 reverse seq) (The sequence length is 0 nt)</LI><LI>Seq.ID 26EB09 =&gt; SD1.2 (primers : URA R5 2.0 forward seq / URA R5 2.0) (The sequence length is 0 nt)</LI><LI>Seq.ID 26EB10 =&gt; SD1.1’ (replicate of SD1.1) (results : The sequence length is 23 nt)</LI><LI>Seq.ID 26EB11 =&gt; SD1.2’ (replicate of SD1.2) (results : The sequence length is 41 nt)</LI><LI>Seq.ID 26EB12 =&gt; SD2.1 (same as SD1.1) (results : The sequence length is 22 nt)</LI><LI>Seq.ID 26EB13 =&gt; SD2.2 (same as SD1.2) (results : The sequence length is 737 nt)</LI><LI>Seq.ID 26EB14 =&gt; SD2.1’ (replicate of SD2.1) (results : The sequence length is 0 nt)</LI><LI>Seq.ID 26EB15 =&gt; SD2.2’ (Replicate of SD2.2) (Results : The sequence length is 20 nt)</LI></OL></P></DIV><DIV id="secretion_07_23" class="notebook-entry"><H4>Friday, 23<SUP>rd</SUP> July 2015</H4><P class="text-justify&quot;"><STRONG>Mat alpha IFN gamma ADH1 (pYGG1) : Sample was minipreped using NucleoSpin Plasmid (LOT 1306/003 Macherey-Nagel) from the miniculture (see 15-07-15) and nanodroped : </STRONG><OL><LI>Mata IFN gamma pYGG1 mini (A4 mini) = 256 ng/µl (260/230= 1.23 ; 260/280= 1.73)</LI><LI>Mata IFN gamma pYGG1 mini (C6 mini) = 66.8 ng/µl (260/230= 1.39 ; 260/280= 1.85)</LI><LI>Mata IFN gamma pYGG1 mini (D7 mini) = 71.4 ng/µl (260/230= 1.67 ; 260/280= 1.88)</LI></OL></P><P class="text-justify"><STRONG>Mat alpha IFN gamma ADH1 (pYGG1) sequencing using only one primer set (URA F1/ URA R5 2.0):</STRONG></P><P class="text-justify"><OL><LI>Seq.ID 26EB02 =&gt; A4 mini .1 (results : The sequence length is 1110 nt)</LI><LI>Seq.ID 26EB03 =&gt; A4 mini .2 (replicate) (results : The sequence length is 1076 nt)</LI><LI>Seq.ID 26EB04 =&gt; C6 mini .1 (results : The sequence length is 986 nt)</LI><LI>Seq.ID 26EB05 =&gt; C6 mini .2 (replicate) (results : The sequence length is 978 nt)</LI><LI>Seq.ID 26EB06 =&gt; D7 mini .1 (results : The sequence length is 23 nt)</LI><LI>Seq.ID 26EB07 =&gt; D7 mini .2 (replicate) (results : The sequence length is 38 nt)</LI></OL></P></DIV><DIV id="sd_07_24" class="notebook-entry"><H4>Friday, 24<SUP>th</SUP> July 2015</H4><P class="text-justify&quot;"><STRONG>OVA2 (pYGG1) <I>E.coli</I> transformation results</STRONG></P><P class="text-justify&quot;">
The 2 plates put into the incubators showed 4 colonies at most. 
We decided to wait for the plates to develop a bit more and to put back the miniculture and colony PCR to Monday 27th 2015. Meanwhile, after a few hours into the incubator, plates were wrapped with parafilm and put into the fridge  at 4°C.
</P></DIV><DIV id="sd_07_27" class="notebook-entry"><H4>Monday, 27<SUP>th</SUP> July 2015</H4><P class="text-justify"><STRONG>OVA2 (pYGG1): PCR colony and gel electrophoresis</STRONG></P><P class="text-justify&quot;">
Previously (see 2015-07-23), we transformed E. coli with our golden gate products and plated the transformed E. coli onto 2 plates.
We took 8 positive colonies from the plates and resuspend them separately into 20 µl of Luria Bertoni media.
</P><P class="text-justify"><SPAN class="text-primary">PCR colony mix (20 µl):</SPAN><OL><LI>10 µl PCR mix (DreamTaq)</LI><LI>1 µl URA F1</LI><LI>1 µl URA R5 2.0</LI><LI>1 µl colony resuspended in Luria Bertoni media</LI><LI>7 µl water</LI></OL></P><P class="text-justify"><SPAN class="text-primary">PCR colony program:</SPAN><OL><LI>Step 1 : 95°C - 5 minutes</LI><LI>Step 2 : 95°C – 30 seconds</LI><LI>Step 3 : 50°C – 30 seconds</LI><LI>Step 4 : 72°C – 2 minutes (Repeat step 2-4, 35 times)</LI><LI>Step 5 : 72°C – 10 minutes</LI><LI>Step 6 :  4°C – Pause</LI></OL></P><P class="text justify"><STRONG>PCR from New insert1 m1 Cter (PCR001): amplification of AGA2P with primers containing DEC205 overhang</STRONG></P><P class="text-justify"><SPAN class="text-primary">Q5 PCR mix (50µl)</SPAN><OL><LI>2,5 µl FWD AGA2P</LI><LI>2,5 µl  RV AGA2P</LI><LI>1 µl New insert1 m1 Cter</LI><LI>19 µl water</LI><LI>25 µl PCR mix (Q5)</LI></OL></P><P class="text-justify"><SPAN class="text-primary">Q5 PCR Program</SPAN><OL><LI>step 1 : 95°C – 30 seconds</LI><LI>step 2 : 95°C – 10 seconds</LI><LI>step 3 : 60°C</LI><LI>step 4 : 72°C – 1 min (repeat steps 2-4 for 31 cycles)</LI><LI>step 5 : 12°C - Hold</LI></OL></P><P class="text-justify"><STRONG>PCR from New insert1 m2 Cter (PCR002): amplification of DEC205 with  primers containing AGA2P overhang</STRONG></P><P class="text-justify"><SPAN class="text-primary">Q5 PCR mix (50µl)</SPAN><OL><LI>2,5 µl FWD DEC205</LI><LI>2,5 µl  RV DEC205</LI><LI>1 µl New insert1 m2 Cter</LI><LI>19 µl water</LI><LI>25 µl PCR mix (Q5)</LI></OL></P><P class="text-justify"><SPAN class="text-primary">Q5 PCR Program</SPAN><OL><LI>step 1 : 95°C – 30 seconds</LI><LI>step 2 : 95°C – 10 seconds</LI><LI>step 3 : 60°C</LI><LI>step 4 : 72°C – 1 min (repeat steps 2-4 for 31 cycles)</LI><LI>step 5 : 12°C - Hold</LI></OL></P><P class="text-justify"><STRONG>PCR001 and PCR002 gel electrophoresis (1% agarose):</STRONG></P><P class="text-justify"><STRONG>PCR clean up and nanodrop of AGA2P and DEC205 products :</STRONG><OL><LI>AGA2P = 72 .3 ng/µl (260/280 = 1.79 ; 260/230 = 0.76)</LI><LI>DEC205 = 104.2 ng/µl (260/280 = 1.79 ; 260/230 = 0.42)</LI></OL></P><P class="text-justify"><STRONG>Golden gates of :
<OL><LI>- (pYGG1) + AGA2P + DEC205</LI><LI>- (pYGG1) + AGA2P + OVA1 (Gblock)</LI><LI>- (pYGG1) + ADH1 (miniprep « ADH1 ») + Malpha GMCSF (B10, see 22/06/15)</LI></OL></STRONG></P><P class="text-justify"><SPAN class="text-primary">Golden gate program:</SPAN></P><P class="text-justify"><STRONG>E. coli transformation with golden gate products</STRONG></P><P class="text-justify"><SPAN class="text-primary">Protocol:</SPAN><OL><LI>1) Add 10 µl plasmids  to 50 µl E. coli competent cells on ice and let the mix rest on ice for 15 min</LI><LI>2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min</LI><LI>3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour</LI><LI>4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin</LI><LI>5) Put plates in growth incubators at 37°C for 24 hours</LI></OL></P></DIV><DIV id="secretion_07_28" class="notebook-entry"><H4>Tuesday, 28<SUP>th</SUP> July 2015</H4><P class="text-justify">
We sent to sequencing Mat alpha IFN gamma ADH1 (pYGG1) with new primers since previous sequencing results were unconclusive.
Furthermore, in order to check the presence of our desired fragments, we also performed a multiple PCR on A4, C6 and D7 samples.
</P><P class="text-justify"><STRONG> A4, C6 and D7 samples sequencing :</STRONG><OL><LI>Seq.ID 26EB16 - A41 (URA F1/SR1 lig IFNg)</LI><LI>Seq.ID 26EB17 - A41.2 (replicate of A41)</LI><LI>Seq.ID 26EB18 - A42 (S2 lig IFNg/SR2 lig IFNg)</LI><LI>Seq.ID 26EB19 - A42.2 (replicate of A42)</LI><LI>Seq.ID 26EB20 - C61 (URA F1/SR1 lig IFNg)</LI><LI>Seq.ID 26EB21 - C61.2</LI><LI>Seq.ID 26EB22 - C62 (S2 lig IFNg/SR2 lig IFNg)</LI><LI>Seq.ID 26EB23 - C62.2</LI><LI>Seq.ID 26EB24 - D71 (URA F1/SR1 lig IFNg)</LI><LI>Seq.ID 26EB25 - D71.2</LI><LI>Seq.ID 26EB26 - D72 (S2 lig IFNg/SR2 lig IFNg)</LI><LI>Seq.ID 26EB27 - D72.2</LI></OL></P><P class="text-justify"><STRONG>Multiple PCR using URA F1/SR1 lig IFNg and S2 lig IFNg/SR2 lig IFNg primers :</STRONG></P><P class="text-justify"><SPAN class="text-primary">PCR mix (20 µl):</SPAN><OL><LI>1 µl of DNA</LI><LI>1 µl forward primer </LI><LI>1 µl reverse primer</LI><LI>7 µl water</LI><LI>10 µl PCR mix (Dreamtaq)</LI></OL></P><P class="text-justify"><STRONG>Multiple PCR gel electrophoresis (1% agarose)</STRONG></P><P class="text-justify"><STRONG>Golden gate of ADH1 (miniprep « ADH1 ») Mat alpha GMCSF (B10, see 22/06/15) (pYGG1)</STRONG></P><P class="text-justify"><STRONG><I>E. coli</I> transformation with golden gate products</STRONG></P><P class="text-justify"><SPAN class="text-primary">Protocol :</SPAN><OL><LI>1) Add 10 µl plasmids  to 50 µl E. coli competent cells on ice and let the mix rest on ice for 15 min</LI><LI>2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min</LI><LI>3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour</LI><LI>4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin</LI><LI>5) Put plates in growth incubators at 37°C for 24 hours</LI></OL></P></DIV><DIV id="sd_07_28" class="notebook-entry"><H4>Tuesday, 28<SUP>th</SUP> July 2015</H4><P class="text-justify">
We sent to sequencing AGA2P OVA1 DEC205 (pYGG2) with new primers since previous sequencing results were unconclusive.
Furthermore, in order to check the presence of our desired fragments, we also performed a multiple PCR on SD1 and SD2 samples.
</P><P class="text-justify"><STRONG>SD1 and SD2 samples sequencing:</STRONG></P><P class="text-justify">
Sequencing ID and primers used for sample &quot;SD2&quot;:
<OL><LI>Seq.ID 26EB28 (URA F1/URA F1 Rev Seq V2) </LI><LI>Seq.ID 26EB29 (URA R5 2.0 Fw seq V2/URA R5 2.0)</LI><LI>Seq.ID 26EB30 (URA F1/ URA R1) </LI><LI>Seq.ID 26EB31 (URA F2/URA R5 2.0) </LI><LI>Seq.ID 26EB32 (URA F1/URA F1 Rev Seq V2) </LI><LI>Seq.ID 26EB33 (URA R5 2.0 Fw seq V2/URA R5 2.0)  </LI><LI>Seq.ID 26EB34 (URA F1/ URA R1) </LI><LI>Seq.ID 26EB35 (URA F2/URA R5 2.0) </LI></OL></P><P class="text-justify">
Sequencing ID and primers used for sample &quot;SD1&quot;:
<OL><LI>Seq.ID 26EB36 (URA F1/URA F1 Rev Seq V2) </LI><LI>Seq.ID 26EB37 (URA R5 2.0 Fw seq V2/URA R5 2.0) </LI><LI>Seq.ID 26EB38 (URA F1/ URA R1) </LI><LI>Seq.ID 26EB39 (URA F2/URA R5 2.0) </LI><LI>Seq.ID 26EB40 (URA F1/URA F1 Rev Seq V2) </LI><LI>Seq.ID 26EB41 (URA R5 2.0 Fw seq V2/URA R5 2.0) </LI><LI>Seq.ID 26EB42 (URA F1/ URA R1) </LI><LI>Seq.ID 26EB43 (URA F2/URA R5 2.0)</LI></OL></P><P class="text-justify"><STRONG>Multiple PCR</STRONG></P><P class="text-justify"><SPAN class="text-primary">PCR mix (20 µl):</SPAN><OL><LI>1 µl of DNA 
    </LI><LI>1 µl forward primer
    </LI><LI>1 µl reverse primer
    </LI><LI>7 µl water
    </LI><LI>10 µl PCR mix (Dreamtaq)
</LI></OL></P><STRONG>Multiple PCR gel electrophoresis (1% agarose)</STRONG><P class="text-justify"><STRONG>Golden gates of :
<OL><LI>- Golden 1 : pYGG1 + AGA2P + DEC205</LI><LI>- Golden 2 : PYGG1 + AGA2P + OVA1 (Gblock) (dec 205 v2)</LI></OL></STRONG></P><P class="text-justify"><STRONG>E. coli transformation with golden gate products</STRONG></P><P class="text-justify"><SPAN class="text-primary">Protocol:</SPAN><OL><LI>1) Add 10 µl plasmids  to 50 µl E. coli competent cells on ice and let the mix rest on ice for 15 min</LI><LI>2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min</LI><LI>3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour</LI><LI>4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin</LI><LI>5) Put plates in growth incubators at 37°C for 24 hours</LI></OL></P></DIV><DIV id="sd_07_29" class="notebook-entry"><H4>Wednesday, 29<SUP>th</SUP> July 2015</H4><P class="text-justify"><STRONG>Results of <I>E.coli</I> transformed with AGA2P OVA1 (Gblock &quot;dEC205 - v2&quot;) (pYGG1) and AGA2P DEC205 (pYGG1)</STRONG></P><P class="text-justify">
We got a few colonies for AGA2P OVA1 but none for AGA2P DEC2065 so we performed a PCR colony for AFGA2P OVA1 and a second transformation for AGA2P DEC205 with remaining golden gate (see 28/07).
</P><P class="text-justify"><STRONG>AGA2P OVA1: PCR colony </STRONG></P><P class="text-justify"><SPAN class="text-primary">PCR colony mix (20 µl):</SPAN><OL><LI>10 µl PCR mix (DreamTaq)</LI><LI>1 µl URA F1</LI><LI>1 µl URA R5 2.0</LI><LI>1 µl colony resuspended in Luria Bertoni media</LI><LI>7 µl water</LI></OL></P><P class="text-justify"><SPAN class="text-primary">PCR colony program :</SPAN><OL><LI>Step 1 : 95°C - 5 minutes</LI><LI>Step 2 : 95°C – 30 seconds</LI><LI>Step 3 : 50°C – 30 seconds</LI><LI>Step 4 : 72°C – 2 minutes (Repeat step 2-4, 30 times)</LI><LI>Step 5 : 72°C – 10 minutes</LI><LI>Step 6 :  4°C – Pause</LI></OL></P><P class="text-justify"><STRONG>AGA2P OVA1: PCR colony gel electrophoresis (1% agarose)</STRONG></P><P class="text-justify"><STRONG>Yeast transformations with following golden gate constructions : 
<OL><LI>- AGA2P OVA1 DEC205 (SD1)</LI><LI>- AGA1</LI><LI>- AGA2P OVA1 DEC205 (SD2)</LI></OL></STRONG></P><P class="text-justify"><SPAN class="text-primary">Protocol :</SPAN><OL><LI>1) From the W303 yeast culture at DO=1, harvest in sterile tube at 5000 rpm for 5 min</LI><LI>2) Pour off the medium, resuspend the cells in 25 ml of sterile water and centrifuge again	</LI><LI>3) Pour off the water, resuspend the cells in 1 ml of 0.1 M LiAc and transfer the suspension to a 1.5 ml microfuge tube</LI><LI>4) Pellet the cells at top speed for 15 sec and remove the LiAc with a micropipette</LI><LI>5) Resuspend the cells with 0.1 LiAc to a final volume of 500 µl </LI><LI>6) Vortex the cell suspension and pipette 50 µl samples into new 1.5 ml tubes. Pellet the cells and remove the LiAc with a micropipette</LI><LI>7) Add the following to the samples in order: </LI><UL><LI>- 240 µl PEG 50%</LI><LI>- 36 µl 1 M LiAc</LI><LI>- 25 µl Salmon sperm DNA (2 mg/ml)</LI><LI>- 50 µl water and plasmid (10 ug)</LI></UL><LI>8) Vortex each tube vigorously until the cell pellet has been completely mixed. Usually takes about 1 min</LI><LI>9) Incubate at 30°C for 30 min</LI><LI>10) Heat shock in a water bath at 42°C for 15 minute</LI><LI>11) Ice for 2 minutes</LI><LI>12) Centrifuge at 5000 rpm for 15 sec and remove the transformation mix with a micropipette</LI><LI>13) Pipette 1 ml of sterile water into each tube and resuspend the pellet by pipetting it up and down gently</LI><LI>14) Plate 50 µl and 150 µl of the transformation mix onto plates with corresponding media</LI><LI>15) Incubate at 30°C for 3 days</LI></OL></P></DIV><DIV id="secretion_07_29" class="notebook-entry"><H4>Wednesday, 29<SUP>th</SUP> July 2015</H4><P class="text-justify"><STRONG>Results of <I>E.coli</I> transformed with ADH1 (miniprep « ADH1 ») + Malpha GMCSF (B10, see 22/06/15) (pYGG1)</STRONG></P><P class="text-justify">
 We got no colonies, so we performed a second <I>E.coli</I> transformation with remaining golden gate (see 28/07).
</P><P class="text-justify"><STRONG>E. coli transformation with golden gate products</STRONG></P><P class="text-justify"><SPAN class="text-primary">Protocol :</SPAN><OL><LI>1) Add 10 µl plasmids  to 50 µl <I>E. coli</I> competent cells on ice and let the mix rest on ice for 15 min</LI><LI>2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min</LI><LI>3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour</LI><LI>4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin</LI><LI>5) Put plates in growth incubators at 37°C for 24 hours</LI></OL></P><P class="text-justify"><STRONG>AGA1P sequencing </STRONG></P></DIV><DIV id="secretion_07_31" class="notebook-entry"><H4>Friday, 31<SUP>st</SUP> July 2015</H4><P class="text-justify">
Since on the 07/30, we could not perform all yeast transformations (transformation 5 or T5), due to lack of samples for AGA1P, we transformed all the constructions into yeast again. 
The yeasts that were transformed the day before are already growing except for the plate CoT2 that did not contain enough substrate for the yeast to feed off. 
</P><P class="text-justify"><STRONG>Yeast transformation:
<OL><LI>T4 : ADH1 Ma IFNg (pYGG1)</LI></OL></STRONG></P><P class="text-justify"><SPAN class="text-primary">Protocol :</SPAN><OL><LI>1) From the W303 yeast culture at DO=1, harvest in sterile tube at 5000 rpm for 5 min</LI><LI>2) Pour off the medium, resuspend the cells in 25 ml of sterile water and centrifuge again	</LI><LI>3) Pour off the water, resuspend the cells in 1 ml of 0.1 M LiAc and transfer the suspension to a 1.5 ml microfuge tube</LI><LI>4) Pellet the cells at top speed for 15 sec and remove the LiAc with a micropipette</LI><LI>5) Resuspend the cells with 0.1 LiAc to a final volume of 500 µl </LI><LI>6) Vortex the cell suspension and pipette 50 µl samples into new 1.5 ml tubes. Pellet the cells and remove the LiAc with a micropipette</LI><LI>7) Add the following to the samples in order: </LI><LI>- 240 µl PEG 50%</LI><LI>- 36 µl 1 M LiAc</LI><LI>- 25 µl Salmon sperm DNA (2 mg/ml)</LI><LI>- 50 µl water and plasmid (10 ug)</LI><LI>8) Vortex each tube vigorously until the cell pellet has been completely mixed. Usually takes about 1 min</LI><LI>9) Incubate at 30°C for 30 min</LI><LI>10) Heat shock in a water bath at 42°C for 15 minute</LI><LI>11) Ice for 2 minutes</LI><LI>12) Centrifuge at 5000 rpm for 15 sec and remove the transformation mix with a micropipette</LI><LI>13) Pipette 1 ml of sterile water into each tube and resuspend the pellet by pipetting it up and down gently</LI><LI>14) Plate 50 µl and 150 µl of the transformation mix onto plates with corresponding media:
	</LI><UL><LI>pYGG1 =&gt; URA-</LI><LI>pYGG2 =&gt; TRP-</LI><LI>pYGG1 + pYGG2 =&gt; TRP- URA-</LI></UL><LI>15) Incubate at 30°C for 3 days</LI></OL></P></DIV><DIV id="sd_07_31" class="notebook-entry"><H4>Friday, 31<SUP>st</SUP> July 2015</H4><P class="text-justify">
Since on the 07/30, we could not perform all yeast transformations (transformation 5 or T5), due to lack of samples for AGA1P, we transformed all the constructions into yeast again. 
The yeasts that were transformed the day before are already growing except for the plate CoT2 that did not contain enough substrate for the yeast to feed off. 
</P><P class="text-justify"><STRONG>AGA1P minipreped using NucleoSpin Plasmid (LOT 1306/003 Macherey-Nagel) and nanodroped :</STRONG></P><P class="text-justify"><OL><LI>AGA1P.1 = 44.2 ng/µl</LI></OL></P><P class="text-justify"><STRONG>Yeast transformations:
<OL><LI>T1: OVA2 (pYGG2)</LI><LI>T2: AGA2P OVA1 (pYGG1) + AGA1P (pYGG2)</LI><LI>T3: AGA2P OVA1 DEC205 (pYGG1) + AGA1P (pYGG2)</LI><LI>T5: AGA2P/GFP (pYGG1) + AGA1P (pYGG2)</LI><LI>T6: AGA2P GFP (pYGG1)</LI><LI>T7: AGA2P OVA DEC205 (pYGG1)</LI><LI>T8: AGA2P OVA1 (pYGG1)</LI></OL></STRONG></P><P class="text-justify"><SPAN class="text-primary">Protocol :</SPAN><OL><LI>1) From the W303 yeast culture at DO=1, harvest in sterile tube at 5000 rpm for 5 min</LI><LI>2) Pour off the medium, resuspend the cells in 25 ml of sterile water and centrifuge again	</LI><LI>3) Pour off the water, resuspend the cells in 1 ml of 0.1 M LiAc and transfer the suspension to a 1.5 ml microfuge tube</LI><LI>4) Pellet the cells at top speed for 15 sec and remove the LiAc with a micropipette</LI><LI>5) Resuspend the cells with 0.1 LiAc to a final volume of 500 µl </LI><LI>6) Vortex the cell suspension and pipette 50 µl samples into new 1.5 ml tubes. Pellet the cells and remove the LiAc with a micropipette</LI><LI>7) Add the following to the samples in order: </LI><UL><LI>- 240 µl PEG 50%</LI><LI>- 36 µl 1 M LiAc</LI><LI>- 25 µl Salmon sperm DNA (2 mg/ml)</LI><LI>- 50 µl water and plasmid (10 ug)</LI></UL><LI>8) Vortex each tube vigorously until the cell pellet has been completely mixed. Usually takes about 1 min</LI><LI>9) Incubate at 30°C for 30 min</LI><LI>10) Heat shock in a water bath at 42°C for 15 minute</LI><LI>11) Ice for 2 minutes</LI><LI>12) Centrifuge at 5000 rpm for 15 sec and remove the transformation mix with a micropipette</LI><LI>13) Pipette 1 ml of sterile water into each tube and resuspend the pellet by pipetting it up and down gently</LI><LI>14) Plate 50 µl and 150 µl of the transformation mix onto plates with corresponding media:
	</LI><UL><LI>pYGG1 =&gt; URA-</LI><LI>pYGG2 =&gt; TRP-</LI><LI>pYGG1 + pYGG2 =&gt; TRP- URA-</LI></UL><LI>15) Incubate at 30°C for 3 days</LI></OL></P></DIV></DIV></SECTION><SECTION class="page-section" id="aug-section"><DIV id="sd_08_03" class="notebook-entry"><H4>Monday, 3<SUP>rd</SUP> August 2015</H4><P class="text-justify"><STRONG>Colony PCR of yeast transformants with following primers :</STRONG></P><P class="text-justify">
AGA2P/DEC205
<OL><LI>P1-4 =&gt; URA F1/ URA R1</LI><LI>P5-8 =&gt; URA R2/ URA R5 2.0</LI></OL></P><P class="text-justify">
SD1
<OL><LI>R1-4 =&gt; URA F1/URA R5 2.0</LI><LI>R5-6 =&gt; URA F1/URA R1</LI><LI>R7-8 =&gt; URA F2/URA R5 2.0</LI></OL></P><P class="text-justify">
SD2
<OL><LI>T1-4 =&gt; URA F1/ URA R5 2.0</LI><LI>T5-6 =&gt; URA F1/URA R1</LI><LI>T7-8 =&gt; URA F2/URA R5 2.0&lt;:li&gt;
</LI></OL></P><P class="text-justify">
Library
<OL><LI>- &quot;130&quot;   P2 (AGA2P/DEC205)</LI><LI>- &quot;131&quot;   P4 (AGA2P/DEC205)</LI><LI>- &quot;132&quot;   R4 (SD1)</LI><LI>- &quot;133&quot;   R5 (SD1)</LI><LI>- &quot;134&quot;   R6 (SD1)</LI><LI>- &quot;135&quot;   T1 (SD2)</LI><LI>- &quot;136&quot;   T3 (SD2)</LI><LI>- &quot;137&quot;   T4 (SD2)</LI><LI>- &quot;138&quot;   T5 (SD2)</LI></OL></P><P class="text-justify"><SPAN class="text-primary"> PCR Colony mix:</SPAN><OL><LI>1 µl of resuspended colony in 20 µl LB</LI><LI>1 µl URA F1</LI><LI>1 µl URA R5 2.0</LI><LI> 7 µl water</LI><LI>10 µl PCR mix (Dreamtaq)</LI></OL></P><P class="text-justify"><SPAN class="text-primary"> PCR colony Program:</SPAN><OL><LI>Step 1 : 95°C - 5 min</LI><LI>Step 2 : 95°C – 30 s</LI><LI>Step 3 : 50°C – 30s</LI><LI>Step 4 : 72°C – 2 min (repeat step 2-4, 45 times)</LI><LI>Step 5 : 72°C – 10 min</LI><LI>Step 6 :  4°C – Pause</LI></OL></P><DIV id="secretion_08_03" class="notebook-entry"><H4>Monday, 3<SUP>rd</SUP> August 2015</H4><P class="text-justify"><STRONG>Colony PCR of yeast transformants with following primers :</STRONG></P><P class="text-justify">
GMCSF
<OL><LI>Q1-8 =&gt; URA F1/URA R5 2.0</LI></OL></P><P class="text-justify">
IFNgamma
<OL><LI>S1-4 =&gt; URA F1/SR1 lig IFN gamma
     </LI><LI>S5-8 =&gt; S2 lig IFN gamma/SR2 lig IFN gamma
</LI></OL></P></DIV><DIV id="secretion_08_04" class="notebook-entry"><H4>Tuesday, 4<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG>Colony PCR of transformed E. coli with ADH1 Mata GMCSF and ADH1 Mata IFNg</STRONG></P><P class="text-justify"><SPAN class="text-primary">PCR colony mix (20 µl):</SPAN><OL><LI>10 µl PCR mix (DreamTaq)</LI><LI>1 µl URA F1</LI><LI>1 µl URA R5 2.0</LI><LI>1 µl colony resuspended in Luria Bertoni media</LI><LI>7 µl water</LI></OL></P><P class="text-justify"><SPAN class="text-primary">PCR colony program:</SPAN><OL><LI>Step 1 : 95°C - 5 minutes</LI><LI>Step 2 : 95°C – 30 seconds</LI><LI>Step 3 : 50°C – 30 seconds</LI><LI>Step 4 : 72°C – 2 minutes (Repeat step 2-4, 35 times)</LI><LI>Step 5 : 72°C – 10 minutes</LI><LI>Step 6 :  4°C – Pause</LI></OL></P><P class="text-justify"><STRONG>Colony PCR gel electrophoresis (1% agarose)</STRONG></P><P class="text-justify"><STRONG>Colony PCR yeast gel electrophoresis (1% agarose)</STRONG></P></DIV><DIV id="secretion_08_05" class="notebook-entry"><H4>Wednesday, 5<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG>Gel electrophoresis of ADH1 Mata GMCSF and ADH1 Mata IFNg</STRONG></P><P class="text-justify"><STRONG>Colony PCR2 of ADH1 Mata GMCSF and ADH1 Mata IFNg :</STRONG></P><P class="text-justify"><SPAN class="text-primary">PCR colony mix (10 µl):</SPAN><OL><LI>5 µl PCR mix (DreamTaq)</LI><LI>0.5 µl URA F1</LI><LI>0.5 µl URA R5 2.0</LI><LI>0.5 µl colony resuspended in Luria Bertoni media</LI><LI>3.5 µl water</LI></OL></P><P class="text-justify"><SPAN class="text-primary">PCR colony program:</SPAN><OL><LI>Step 1 : 95°C - 5 minutes</LI><LI>Step 2 : 95°C – 30 seconds</LI><LI>Step 3 : 72°C – 30 seconds</LI><LI>Step 4 : 60°C/53°C– 2 minutes (Repeat step 2-4, 35 times)</LI><LI>Step 5 : 72°C – 10 minutes</LI><LI>Step 6 : 16°C – Pause</LI></OL></P><P class="text-justify"><STRONG>Miniculture of ADH1 Mata GMCSF and ADH1 Mata IFNg</STRONG></P><P class="text-justify">
The remaining resusepended <I>E.coli</I> was put into 4 mL of Luria Bertoni media containing with 4 µl ampicilin. 
</P><P class="text-justify"><STRONG>PCR colony of T4 (see 31/07)</STRONG></P><P class="text-justify"><SPAN class="text-primary"> PCR Colony mix (20 µl):</SPAN><OL><LI>1 µl of resuspended colony in 20 µl Luria Bertoni media</LI><LI>1 µl URA F1</LI><LI>1 µl URA R5 2.0</LI><LI> 7 µl water</LI><LI>10 µl PCR mix (Dreamtaq)</LI></OL></P><P class="text-justify"><SPAN class="text-primary"> PCR colony program:</SPAN><OL><LI>Step 1 : 95°C - 5 minutes</LI><LI>Step 2 : 95°C – 30 seconds</LI><LI>Step 3 : 60°C – 30 seconds</LI><LI>Step 4 : 72°C – 2 minutes (repeat step 2-4, 45 times)</LI><LI>Step 5 : 72°C – 10 minutes</LI><LI>Step 6 :  4°C – Pause</LI></OL></P><P class="text-justify"><STRONG>Golden gate  of ADH1 mat alpha GMCSF (pYGG1) construction :</STRONG></P><P class="text-justify"><SPAN class="text-primary"> Golden gate mix (20 µl):</SPAN><OL><LI>0.849 µl ADH1</LI><LI>3.243 µl Mat alpha GMCSF</LI><LI>3.1 µl PYGG1</LI><LI>2 µl T4 ligase buffer</LI><LI>0.5 µl T4 ligase</LI><LI>0.5 µl BSA I</LI></OL></P><P class="text-justify"><SPAN class="text-primary"> Golden gate program :</SPAN></P><P class="text-justify"><STRONG><I>E.coli</I> transformation with golden gate ADH1 mat alpha GMCSF (pYGG1)</STRONG></P><P class="text-justify"><SPAN class="text-primary"> Protocol :</SPAN><OL><LI>1) Add 10 µl plasmids  to 50 µl <I>E.coli</I> competent cells on ice and let the mix rest on ice for 15 min</LI><LI>2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min</LI><LI>3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour</LI><LI>4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin</LI><LI>5) Put plates in growth incubators at 37°C for 24 hours</LI></OL></P></DIV><DIV id="sd_08_05" class="notebook-entry"><H4>Wednesday, 5<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG>Colony PCR of T1/2/3/5/6/7/8 (see 31/07)</STRONG></P><P class="text-justify"><SPAN class="text-primary"> PCR Colony mix (20 µl):</SPAN><OL><LI>1 µl of resuspended yeast colony in 10 µl water</LI><LI>1 µl URA F1</LI><LI>1 µl URA R5 2.0</LI><LI> 7 µl water</LI><LI>10 µl PCR mix (Dreamtaq)</LI></OL></P><P class="text-justify"><SPAN class="text-primary"> PCR colony program :</SPAN><OL><LI>Step 1 : 95°C - 5 minutes</LI><LI>Step 2 : 95°C – 30 seconds</LI><LI>Step 3 : 60°C – 30 seconds</LI><LI>Step 4 : 72°C – 4 minutes (repeat step 2-4, 45 times)</LI><LI>Step 5 :  4°C – Pause</LI></OL></P></DIV><DIV id="sd_08_06" class="notebook-entry"><H4>Thursday, 6<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG>Yeast transformation without OVA1</STRONG></P><P class="text-justify"><SPAN class="text-primary">Protocol :</SPAN><OL><LI>1) From the W303 yeast culture at DO=1, harvest in sterile tube at 5000 rpm for 5 min</LI><LI>2) Pour off the medium, resuspend the cells in 25 ml of sterile water and centrifuge again	</LI><LI>3) Pour off the water, resuspend the cells in 1 ml of 0.1 M LiAc and transfer the suspension to a 1.5 ml microfuge tube</LI><LI>4) Pellet the cells at top speed for 15 sec and remove the LiAc with a micropipette</LI><LI>5) Resuspend the cells with 0.1 LiAc to a final volume of 500 µl </LI><LI>6) Vortex the cell suspension and pipette 50 µl samples into new 1.5 ml tubes. Pellet the cells and remove the LiAc with a micropipette</LI><LI>7) Add the following to the samples in order: </LI><UL><LI>- 240 µl PEG 50%</LI><LI>- 36 µl 1 M LiAc</LI><LI>- 25 µl Salmon sperm DNA (2 mg/ml)</LI><LI>- 50 µl water and plasmid (10 ug)</LI></UL><LI>8) Vortex each tube vigorously until the cell pellet has been completely mixed. Usually takes about 1 min</LI><LI>9) Incubate at 30°C for 30 min</LI><LI>10) Heat shock in a water bath at 42°C for 15 minute</LI><LI>11) Ice for 2 minutes</LI><LI>12) Centrifuge at 5000 rpm for 15 sec and remove the transformation mix with a micropipette</LI><LI>13) Pipette 1 ml of sterile water into each tube and resuspend the pellet by pipetting it up and down gently</LI><LI>14) Plate 50 µl and 150 µl of the transformation mix onto plates with corresponding media:</LI><UL><LI>pYGG1 =&gt; URA-</LI><LI>pYGG2 =&gt; TRP-</LI><LI>pYGG1 + pYGG2 =&gt; TRP- URA-</LI></UL><LI>15) Incubate at 30°C for 3 days</LI></OL></P></DIV><DIV id="secretion_08_07" class="notebook-entry"><H4>Friday, 7<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG>PCR colony yeast and <I><I>E.coli</I></I></STRONG></P><P class="text-justify"><STRONG>PCR colony gel electrophoresis (1% agarose) :</STRONG></P><DIV id="secretion_08_08" class="notebook-entry"><H4>Saturday, 8<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG><I>E.coli</I> PCR colony of GMCSF and IFNg </STRONG></P><P class="text-justify"><STRONG>PCR colony gel electrophoresis (1% agarose) :</STRONG></P></DIV><DIV id="secretion_08_10" class="notebook-entry"><H4>Monday, 10<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG>Golden gate ADH1 matalpha GMCSF (pYGG1)</STRONG></P><P class="text-justify"><SPAN class="text-primary"> Golden gate mix (20 µl):</SPAN><OL><LI>0.849 µl ADH1</LI><LI>3.243 µl Mat alpha GMCSF</LI><LI>3.1 µl PYGG1</LI><LI>2 µl T4 ligase buffer</LI><LI>0.5 µl T4 ligase</LI><LI>0.5 µl BSA I</LI></OL></P><P class="text-justify"><SPAN class="text-primary"> Golden gate program:</SPAN></P></DIV><DIV id="secretion_08_11" class="notebook-entry"><H4>Tuesday, 11<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG><I>E.coli</I> transformation of ADH1 Mat alpha GMCSF (pYGG1) (again)</STRONG></P><P class="text-justify"><SPAN class="text-primary"> Protocol:</SPAN><OL><LI>1) Add 10 µl plasmids  to 50 µl <I>E.coli</I> competent cells on ice and let the mix rest on ice for 15 min</LI><LI>2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min</LI><LI>3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour</LI><LI>4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin</LI><LI>5) Put plates in growth incubators at 37°C for 24 hours</LI></OL></P><P class="text-justify"><STRONG>Golden gate of ADH1 mat alpha IFNgamma (pYGG1)</STRONG></P><P class="text-justify"><SPAN class="text-primary"> Golden gate mix (20 µl):</SPAN><OL><LI>0.033 (x 3) µl Sample « B8 » (M :atalpha IFNgamma) (114 ng/µl) </LI><LI>0.058 (x 3) µl Sample « A6 » (IFNgamma) (114 ng/µl)</LI><LI>0.274 µl Sample « ADH1 » (75 ng/µl)</LI><LI>0.929 µl pYGG1</LI><LI>2 µl T4 ligase buffer</LI><LI>0.5 µl T4 ligase</LI><LI>0.5 µl BSA I</LI></OL></P><P class="text-justify"><SPAN class="text-primary"> Golden gate program :</SPAN></P></DIV><DIV id="sd_08_11" class="notebook-entry"><H4>Tuesday, 11<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG>Culture of T1, T2 T3 and WT yeast from plates in glucose (without induction)</STRONG></P></DIV><DIV id="sd_08_12" class="notebook-entry"><H4>Wednesday, 12<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG>Culture induction of T1/T2/T3 and WT yeast</STRONG></P><P class="text-justify"><OL><LI>1) Discard media after centrifugation at 3000 rpm for 4 minutes</LI><LI>2) Resuspend yeast in 10 mL induction media :
	 </LI><LI>- galactose 1X without tryptophane for T1</LI><LI>- galactose 1X without tryptophane and uracile for T2 and T3</LI><LI>3) Put into incubator and agitation at 25 °C for 48 hours</LI></OL></P></DIV><DIV id="secretion_08_12" class="notebook-entry"><H4>Wednesday, 12<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG><I>E.coli</I> transformation of ADH1 Mat alpha IFNgamma (pYGG1)</STRONG></P><P class="text-justify"><SPAN class="text-primary"> Protocol :</SPAN></P><OL><LI>1) Add 10 µl plasmids  to 50 µl <I>E.coli</I> competent cells on ice and let the mix rest on ice for 15 min</LI><LI>2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min</LI><LI>3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour</LI><LI>4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin</LI><LI>5) Put plates in growth incubators at 37°C for 24 hours</LI></OL><P class="text-justify"><STRONG>Colony PCR of pYGG1 ADH1 matalpha GMCSF with 3 sets of primers :</STRONG></P><P class="text-justify"><OL><LI>- URA F1 / URA R5 2.0</LI><LI>- URA F1 / SR1 lig IFNgamma</LI><LI>- S2 lig IFNgamma / SR2 lig GMCSF</LI></OL></P></DIV><DIV id="secretion_08_13" class="notebook-entry"><H4>Thursday, 13<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG>Colony PCR of ADH1 Mat alpha IFN gamma (pYGG1)  with 3 sets of primers :</STRONG></P><P class="text-justify"><OL><LI>- URA F1 / URA R5 2.0
     </LI><LI>- URA F1 / SR1 lig IFNgamma
     </LI><LI>- S2 lig IFNgamma / SR2 lig IFNgamma
</LI></OL></P><P class="text-justify"><SPAN class="text-primary"> PCR colony mix (10 µl):</SPAN><OL><LI>5 µl dreamtaq master mix</LI><LI>0.5 µl colony resuspended in 20 µl Luria Bertoni</LI><LI>0.5 µl  reverse primer</LI><LI>0.5 µl forward primer</LI><LI>3.5 µl water</LI></OL></P><P class="text-justify"><STRONG>Miniprep of ADH1 Mat alpha GMCSF (pYGG1) using NucleoSpin Plasmid (LOT 1306/003 Macherey-Nagel) :</STRONG><UL><LI>AGA2P = 72 .3 ng/µl (260/280 = 1.79 ; 260/230 = 0.76)</LI><LI>GMCSF (1) = 304.2 ng/µl (280/260 = 1.82 ; 260/230 = 1.71)</LI></UL></P></DIV><DIV id="sd_08_17" class="notebook-entry"><H4>Monday, 17<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG>Yeast Colony PCR on T1, T2 and T3 (induced/non induced)</STRONG><OL><LI>Resuspend colony in 10 µl water</LI><LI>Microwave 8 minutes 5 µl of resuspended colony (900W)</LI><LI>Add 10 µl of mix dreamtaq</LI><LI>Add 2.5 µl primer (URA R5 2.0/ URA F1)</LI></OL></P><P class="text-justify"><SPAN class="text-primary"> PCR yeast colony program:</SPAN><OL><LI>Step 1 : 95°C - 5 minutes</LI><LI>Step 2 : 95°C – 30 seconds</LI><LI>Step 3 : 57/60/63°C – 30 seconds</LI><LI>Step 4 : 72°C – 2 minutes (repeat step 2-4, 41 cycles)</LI><LI>Step 4 : 72°C – 10 minutes</LI><LI>Step 5 :  4°C – Pause</LI></OL></P><P class="text-justify"><STRONG>T1/2/3 Colony PCR gel electrophoresis (1% agarose) :</STRONG></P><P class="text-justify"><U>Expected sizes :</U><OL><LI>T1 =&gt; OVA2 (290 bp)</LI><LI>T2 =&gt; AGA1P (2225 bp) &amp; AGA2P OVA1 (461 bp) </LI><LI>T3 =&gt; AGA1P (2285 bp) &amp; AGA2P OVA1 DEC205 (1381 bp)</LI></OL></P><P class="text-justify"><STRONG>Miniprep of ADH1 Mat alpha IFN gamma using NucleoSpin Plasmid (LOT 1306/003 Macherey-Nagel) :</STRONG></P><P class="text-justify"><OL><LI>IFN 1 = 343.5 ng/µl</LI><LI>IFN 4 = 290 ng/µl</LI><LI>IFN 6 = 265 ng/µl</LI><LI>IFN 7 = 377 ng/µl</LI></OL></P></DIV><DIV id="sd_08_18" class="notebook-entry"><H4>Tuesday, 18<SUP>th</SUP> August 2015</H4><P class="text-justify">
We decided to remake the yeast colony PCR from 17/08 using same protocol to check our results a second time. 
We tested plates of the same construction, T1, T2 and T3, made by different experimentators. For T1, we tested 3 different plates, each at 2 different temperatures : 53 and 60°C. 
We also tested T3 with 3 different sets of primers (see gel picture below).
<STRONG>Yeast Colony PCR on T1, T2 and T3 (induced/non induced)
<OL><LI>Resuspend colony in 10 µl water</LI><LI>Microwave 8 minutes 5 µl of resuspended colony (900W)</LI><LI>Add 10 µl of mix dreamtaq</LI><LI>Add 2.5 µl primer (URA R5 2.0/ URA F1)</LI></OL></STRONG></P><P class="text-justify"><SPAN class="text-primary"> PCR yeast colony program :</SPAN><OL><LI>Step 1 : 95°C - 5 minutes</LI><LI>Step 2 : 95°C – 30 seconds</LI><LI>Step 3 : 57/60/63°C – 30 seconds</LI><LI>Step 4 : 72°C – 2 minutes (repeat step 2-4, 41 cycles)</LI><LI>Step 4 : 72°C – 10 minutes</LI><LI>Step 5 :  4°C – Pause</LI></OL></P><P class="text-justify"><STRONG>T1/2/3 Colony PCR gel electrophoresis (1% agarose):</STRONG></P><P class="text-justify">
Primers used :
<OL><LI>T1/T2/WT : URA F1/R5 2.0</LI><LI>T3.1 : URA R5 2.0/F1</LI><LI>T3.2 : URA F1/R1</LI><LI>T3.3 : URA F2/ R5.2.0</LI></OL></P></DIV><DIV id="secretion_08_19" class="notebook-entry"><H4>Wednesday, 19<SUP>th</SUP> August 2015</H4><P class="text-justify">
We started taking care of our biobricks ! We planned to deposite four biobricks : DEC205, OVA2, IFNgamma and Mat alpha IFN gamma. For OVA2 and DEC205, we just amplified the fragments with primers designed to make the fragment fit into the pSB1C3 plasmid. For IFN gamma and Mat alpha IFN gamma, as IFN gamma presented a restriction site, we proceeded to do a site directed mutagenenis to remove it. 
Mutagenesis of IFNgamma and Mat alpha IFN gamma:
We performed two consecutive PCR, the first one (PCR1) to introduce a mutation at the desired site and the second one (PCR2) to amplify the entire fragment (see 21/08). In between, PCR clean-ups were done using Nucleospin Gel and PCR Clean-up (LOT :1504/001).
</P><P class="text-justify"><STRONG>IFNgamma PCR1</STRONG></P><P class="text-justify">PCR1 primers: 
<UL><LI>- P005 FN IFNgamma/RV Mat IFNgamma</LI><LI>- FW Mat IFNgamma/P006 RV IFNgamma</LI></UL></P><P class="text-justify"><SPAN class="text-primary">Q5 PCR mix (50µl):</SPAN><OL><LI>2,5 µl forward primers </LI><LI>2,5 µl reverse primers </LI><LI>1 µl New insert1 m2 Cter</LI><LI>19 µl water</LI><LI>25 µl PCR mix (Q5)</LI></OL></P><P class="text-justify"><SPAN class="text-primary">Q5 PCR Program</SPAN><OL><LI>step 1 : 98°C – 30 seconds</LI><LI>step 2 : 98°C – 10 seconds</LI><LI>step 3 : 57, 60 or 63°C</LI><LI>step 4 : 72°C – 1 min (repeat steps 2-4 for 40 cycles)</LI><LI>step 5 : 16°C - Hold</LI></OL></P><P class="text-justify"><S>Mat alpha IFN gamma PCR1</S></P><P class="text-justify"><SPAN class="text-primary">Q5 PCR mix (50µl):</SPAN><OL><LI>2,5 µl P007 FW Mat alpha GMCSF </LI><LI>2,5 µl P007 RV Mat IFN gamma </LI><LI>1 µl New insert1 m2 Cter</LI><LI>19 µl water</LI><LI>25 µl PCR mix (Q5)</LI></OL></P><P class="text-justify"><SPAN class="text-primary">Q5 PCR Program</SPAN><OL><LI>step 1 : 98°C – 30 seconds</LI><LI>step 2 : 98°C – 10 seconds</LI><LI>step 3 : 57, 60 or 63°C</LI><LI>step 4 : 72°C – 1 min (repeat steps 2-4 for 40 cycles)</LI><LI>step 5 : 16°C - Hold</LI></OL></P></DIV><DIV id="secretion_08_20" class="notebook-entry"><H4>Thursday, 20<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG>Mutagenesis of IFNgamma and Mat alpha IFN gamma: Gel electrophoresis (2% agarose) for the PCR1 products</STRONG></P></DIV><DIV id="secretion_08_21" class="notebook-entry"><H4>Friday, 21<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG>We checked by digestion the following samples :
<UL><LI>- ADH1 Mat alpha IFN gamma (pYGG1) (sample « IFNgamma)</LI><LI>- pYGG1</LI><LI>- water</LI></UL></STRONG></P><P class="text-justify"><SPAN class="text-primary">Digestion mix :</SPAN><OL><LI>2 µl NEB 2.1 Buffer</LI><LI>1 µl Hind III</LI><LI>2 µl DNA</LI><LI>15 µl water</LI></OL></P><P class="text-justify">
We let the mix, 1 hour at 37°C and 500 rpm.
</P><P class="text-justify"><STRONG>Digestion gel electrophoresis (agarose 1%) :</STRONG></P></DIV><DIV id="sd_08_21" class="notebook-entry"><H4>Friday, 21<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG>We checked by digestion the following samples :
<UL><LI>- AGA2P OVA1 DEC205 pYGG1 (sample « SD »)</LI><LI>- AGA2P OVA1 pYGG1 (sample « DEC -»)</LI><LI>- OVA2 pYGG2 (sample « OVA2 »)</LI><LI>- pYGG2</LI><LI>- water</LI></UL></STRONG></P><P class="text-justify"><SPAN class="text-primary">Digestion mix :</SPAN><OL><LI>2 µl NEB 2.1 Buffer</LI><LI>1 µl Hind III</LI><LI>2 µl DNA</LI><LI>15 µl water</LI></OL></P><P class="text-justify">
We let the mix, 1 hour at 37°C and 500 rpm.
</P><P class="text-justify"><STRONG>Digestion gel electrophoresis (agarose 1%) :</STRONG></P></DIV><DIV id="biosensor_08_24" class="notebook-entry"><H4>Monday, 24<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG>PCR of the fragment RFP of pYGG1 for biosensor design using primers 3B FW 2.0 and  3B reverse : </STRONG></P><P class="text-justify"><SPAN class="text-primary">Q5 PCR mix (50µl)</SPAN><OL><LI>2,5 µl 3B FW 2.0</LI><LI>2,5 µl 3B RV</LI><LI>1 µl New insert1 m2 Cter</LI><LI>19 µl water</LI><LI>25 µl PCR mix (Q5)</LI></OL></P><P class="text-justify"><SPAN class="text-primary">Q5 PCR Program</SPAN><OL><LI>step 1 : 95°C – 30 seconds</LI><LI>step 2 : 95°C – 10 seconds</LI><LI>step 3 : 57, 60 or 63°C</LI><LI>step 4 : 72°C – 1 min (repeat steps 2-4 for 40 cycles)</LI><LI>step 5 : 12°C - Hold</LI></OL></P><P class="text-justify"><STRONG>Mutagenesis of IFNgamma and Mat alpha IFN gamma: PCR2</STRONG></P><P class="text-justify">
IFN gamma PCR2 primers :
<OL><LI>P015 FW IFN gamma/P016 RV IFNgamma</LI></OL></P><P class="text-justify">
Mat alpha IFN gamma PCR2 primers :
<OL><LI>P015 FW IFN gamma / P016 RV IFN gamma</LI></OL></P><P class="text-justify"><SPAN class="text-primary">Q5 PCR mix (50µl) :</SPAN><OL><LI>2,5 µl forward primers</LI><LI>2,5 µl reverse primers</LI><LI>1 µl New insert1 m2 Cter</LI><LI>19 µl water</LI><LI>25 µl PCR mix (Q5)</LI></OL></P><P class="text-justify"><SPAN class="text-primary">Q5 PCR program :</SPAN><OL><LI>step 1 : 95°C – 30 seconds</LI><LI>step 2 : 95°C – 10 seconds</LI><LI>step 3 : 57, 60 or 63°C</LI><LI>step 4 : 72°C – 1 min (repeat steps 2-4 for 40 cycles)</LI><LI>step 5 : 12°C - Hold</LI></OL></P></DIV><DIV id="secretion_08_26" class="notebook-entry"><H4>Wednesday, 26<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG>Miniprep and nanodrop of pYGG1, pYGG2 and AGA1P-pYGG2 using NucleoSpin Plasmid (LOT 1306/003 Macherey-Nagel):</STRONG></P><P class="text-justify"><OL><LI>pYGG1 = 68 ng/µL</LI><LI>pYGG2 = 127 ng/µL</LI><LI>AGA1P (pYGG2) = 32 ng/µL</LI></OL></P></DIV><DIV id="bb_08_26" class="notebook-entry"><H4>Wednesday, 26<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG>PCR Clean up of biobrick Mata-IFNg and biobrick IFNg </STRONG></P><P class="text-justify"><OL><LI>- BB Mata-IFNg = 50 ng/µL</LI><LI>- BB IFNg = 35 ng/µL</LI></OL></P><P class="text-justify"><STRONG>Gel electrophoresis (1% agarose) of biobricks Mata-IFNg (BB Mata-IFNg) and IFNg (BB IFNg) :</STRONG></P></DIV><DIV id="biosensor_08_26" class="notebook-entry"><H4>Wednesday, 26<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG>PCR Clean up of Biosensor3</STRONG></P><P class="text-justify"><OL><LI>- Biosensor 3 = 46 ng/µL</LI></OL></P><P class="text-justify"><STRONG>Gel electrophoresis (1% agarose) of biosensor 3</STRONG></P></DIV><DIV id="biosensor_08_27" class="notebook-entry"><H4>Thursday, 27<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG><I>E.coli</I> transformation of biosensor 2 and 3</STRONG></P><P class="text-justify"><SPAN class="text-primary"> Protocol :</SPAN><OL><LI>1) Add 10 µl plasmids  to 50 µl <I>E. coli</I> competent cells on ice and let the mix rest on ice for 15 min</LI><LI>2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min</LI><LI>3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour</LI><LI>4) Plate 50 µl and 150 µl to plates containing LB agar media with ampificilin</LI><LI>5) Put plates in growth incubators at 37°C for 24 hours</LI></OL></P></DIV><DIV id="bb_08_27" class="notebook-entry"><H4>Thursday, 27<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG>PCR of biobrick OVA1</STRONG></P><P class="text-justify"><SPAN class="text-primary">Q5 PCR mix (50µl) :</SPAN><OL><LI>2,5 µl FWD </LI><LI>2,5 µl  RV </LI><LI>1 µl New insert1 m2 Cter</LI><LI>19 µl water</LI><LI>25 µl PCR mix (Q5)</LI></OL></P><P class="text-justify"><SPAN class="text-primary">Q5 PCR program :</SPAN><OL><LI>step 1 : 95°C – 30 seconds</LI><LI>step 2 : 95°C – 10 seconds</LI><LI>step 3 : 57, 60 or 63°C</LI><LI>step 4 : 72°C – 1 min (repeat steps 2-4 for 40 cycles)</LI><LI>step 5 : 12°C - Hold</LI></OL></P><P class="text-justify"><STRONG>Gel electrophoresis (2% agarose) :</STRONG></P><P class="text-justify"><STRONG>Digestion of the following biobricks using NEB kit :</STRONG></P><P class="text-justify"><OL><LI>- BB IFNg</LI><LI>- BB Mata-IFNg</LI><LI>- BB DEC 205</LI><LI>- BB OVA1</LI><LI>- BB OVA2</LI><LI>- pSB1C3</LI></OL></P><P class="text-justify"><SPAN class="text-primary"> Digestion mix :</SPAN><OL><LI>2µL NEB Buffer 2.1
     </LI><LI>0.5µL Eco RI
     </LI><LI>0.5 µL PstI
     </LI><LI>500 ng DNA
     </LI><LI>water qsp 20µL
</LI></OL></P><P class="text-justify">
The mix was left at 1h 37°C under agitation 500rpm and 20 minutes at 80°C.
</P></DIV><DIV id="bb_08_28" class="notebook-entry"><H4>Friday, 28<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG>PCR clean-up and nanodrop of digested products:</STRONG><OL><LI>- BB IFNg D = 13 ng/µL</LI><LI>- BB Mata-IFNg D = 10 ng/µL</LI><LI>- BB DEC 205 D = 15.4 ng/µL</LI><LI>- BB OVA1 D = 9 ng/µL</LI><LI>- BB OVA2 D = 14 ng/µL</LI><LI>- pSB1C3 D = 7.4 ng/µL</LI></OL></P><P class="text-justify"><STRONG>Ligation of digested products into pSB1C3 with T4 Ligase (ratio 3:1):</STRONG><OL><LI>2µL T4 DNA Ligase Buffer
    </LI><LI>1µL T4 DNA Ligase
    </LI><LI>50ng pSB1C3
    </LI><LI>3:1 insert
    </LI><LI>water qsp 20µL
</LI></OL></P><P class="text-justify">
We let the mix at room temperature for 30 minutes and at 65°C for 10 minutes.
</P><P class="text-justify"><STRONG><I>E.coli</I> transformation with ligated products:</STRONG></P><P class="text-justify"><SPAN class="text-primary"> Protocol:</SPAN><OL><LI>1) Add 10 µl plasmids  to 50 µl <I>E. coli</I> competent cells on ice and let the mix rest on ice for 15 min</LI><LI>2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min</LI><LI>3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour</LI><LI>4) Plate 50 µl and 150 µl to plates containing LB agar media with chloramphenicol</LI><LI>5) Put plates in growth incubators at 37°C for 24 hours</LI></OL></P></DIV><DIV id="biosensor_08_28" class="notebook-entry"><H4>Friday, 28<SUP>th</SUP> August 2015</H4><P class="text-justify"><STRONG>Colony PCR of transformed <I>E.coli</I> with biosensor 2 and 3</STRONG></P><P class="text-justify"><SPAN class="text-primary"> PCR colony mix :</SPAN><OL><LI>1 µl of resuspended colony in 20 µl LB</LI><LI>1 µl URA F1</LI><LI>1 µl URA R5 2.0</LI><LI>7 µl water</LI><LI>10 µl PCR mix (Dreamtaq)</LI></OL></P><P class="text-justify"><SPAN class="text-primary">PCR colony program :</SPAN><OL><LI>Step 1 : 95°C -  5 min</LI><LI>Step 2 : 95°C – 30 seconds</LI><LI>Step 3 : 50°C – 30 seconds</LI><LI>Step 4 : 72°C –  2 minutes (repeat step 2-4, 45 times)</LI><LI>Step 5 : 72°C – 10 minutes</LI><LI>Step 6 :  4°C – Pause</LI></OL></P><P class="text-justify"><STRONG>Gel electrophoresis (1% agarose)</STRONG></P><P class="text-justify"><STRONG>Miniculture of biosensor 2 and 3</STRONG></P></DIV><DIV id="biosensor_08_31" class="notebook-entry"><H4>Monday, 31<SUP>st</SUP> August 2015</H4><P class="text-justify"><STRONG>Colony PCR gel electrophoresis (1% agarose) </STRONG></P></DIV></DIV></DIV></SECTION><SECTION class="page-section" id="sep-section"><DIV id="biosensor_09_01" class="notebook-entry"><H4>Tuesday, 1<SUP>st</SUP> September 2015</H4><P class="text-justify"><STRONG>Glycerol library of Biosensor 2 and 3 : </STRONG></P><P class="text-justify">
7.5 μl 50% glycerol + 7.5 μl miniculture
</P><P class="text-justify">
samples names:
<OL><LI>&quot;145&quot; : Biosensor 2</LI><LI>&quot;146&quot; : Biosensor 3</LI></OL></P><P class="text-justify"><STRONG>Miniprep and nanodrop of biosensor 2 and 3 using NucleoSpin Plasmid (LOT 1306/003 Macherey-Nagel) :</STRONG><OL><LI>2.1 = 380 ng/µL</LI><LI>2.2 = 440 ng/µL</LI><LI>2.3 = 392.8 ng/µL</LI><LI>3.1 = 225 ng/µL</LI><LI>3.2 = 250 ng/µL</LI><LI>3.3 = 85 ng/µL</LI></OL></P></DIV><DIV id="secretion_09_01" class="notebook-entry"><H4>Tuesday, 1<SUP>st</SUP> September 2015</H4><P class="text-justify"><STRONG>Gal1 deletion: Q5 PCR of IFNγ and Biosensor 3</STRONG></P><P class="text-justify"><SPAN class="text-primary">Q5 PCR primers :</SPAN><OL><LI>- IFNγ amplification : Gal1- Fwd IFNγ / Gal1- Rv IFNγ</LI><LI>- Biosensor 3 amplification : Fwd Gal1- sur ligate RFP / Rv Gal1- sur ligate RFP</LI></OL></P><P class="text-justify"><SPAN class="text-primary">Q5 PCR mix (50µl):</SPAN><OL><LI>2,5 µl FWD </LI><LI>2,5 µl  RV </LI><LI>1 µl New insert1 m2 Cter</LI><LI>19 µl water</LI><LI>25 µl PCR mix (Q5)</LI></OL></P><P class="text-justify"><SPAN class="text-primary">Q5 PCR program :</SPAN><OL><LI>step 1 : 95°C – 30 seconds</LI><LI>step 2 : 95°C – 10 seconds</LI><LI>step 3 : 60°C</LI><LI>step 4 : 72°C – 1 min (repeat steps 2-4 for 40 cycles)</LI><LI>step 5 : 12°C - Hold</LI></OL></P><P class="text-justify"><STRONG>Gal1 deletion: Digestion DpnI </STRONG></P><P class="text-justify"><SPAN class="text-primary">Digestion mix :</SPAN></P><P class="text-justify"><OL><LI>50μL PCR reaction</LI><LI>5μL NEB 2.1 Buffer</LI><LI>1μL DpnI</LI></OL></P><P class="text-justify">
We let the mix at 37°C for 1 hour and then at 80°C for 20 minutes
</P><P class="text-justify"><SPAN class="text-primary">PCR clean-up</SPAN></P><P class="text-justify"><SPAN class="text-primary">Gel electrophoresis (0.7% agarose) :</SPAN></P></DIV><DIV id="bb_09_02" class="notebook-entry"><H4>Wednesday, 2<SUP>nd</SUP> September 2015</H4><P class="text-justify"><STRONG>Miniprep and nanodrop of biobricks using NucleoSpin Plasmid (LOT 1306/003 Macherey-Nagel):</STRONG><OL><LI>- BB IFNγ-p 1 = 85ng/μL</LI><LI>- BB IFNγ-p 2 = 40ng/μL</LI><LI>- BB MATα-IFNγ-p 1 = 85ng/μL</LI><LI>- BB MATα-IFNγ-p 2 = 100ng/μL</LI><LI>- BB DEC 205-p 1 = 105ng/μL</LI><LI>- BB DEC 205-p 2 = 90ng/μL</LI><LI>- BB OVA1-p 1 = 86ng/μL</LI><LI>- BB OVA1-p 2 = 72ng/μL</LI><LI>- BB OVA2-p 1 = 50ng/μL</LI><LI>- BB OVA2-p 2 = 57ng/μL</LI></OL></P><P class="text-justify"><STRONG>The biobricks were sent to sequencing:</STRONG></P><P class="text-justify"><SPAN class="text-primary"> Sequencing mix:</SPAN><OL><LI>2.5μL primer VF2
     </LI><LI>2.5μL primer VR
	 </LI><LI>5μL DNA (Ci=80-100ng/μL)
</LI></OL></P><P class="text-justify">
Each samples were sent with duplicates, which makes 20 tubes in total.
</P><DIV id="secretion_09_03" class="notebook-entry"><H4>Thursday, 3<SUP>rd</SUP> September 2015</H4><P class="text-justify"><STRONG>Gal1 deletion: Phosphorylation</STRONG></P><P class="text-justify"><SPAN class="text-primary">Mix (10µl) :</SPAN><OL><LI>2μL DNA</LI><LI>1μL T4 DNA Ligase buffer (with 10Mm ATP)</LI><LI>1μL PNK</LI><LI>6μL H20</LI></OL></P><P class="text-justify">
We let the mix at 37°C for 1 hour.
</P><P class="text-justify"><STRONG>Gal1 deletion: Ligation</STRONG></P><P class="text-justify"><SPAN class="text-primary">Mix (10µl) :</SPAN><OL><LI>1μL T4 DNA Ligase buffer</LI><LI>1μL T4 DNA Ligase</LI><LI>8μL H20</LI></OL></P><P class="text-justify">
We let the mix at room temperature for 1 hour.
</P><P class="text-justify"><STRONG>Gal1 deletion <I>E.coli</I> transformation with ligated products:</STRONG></P><P class="text-justify"><SPAN class="text-primary"> Protocol:</SPAN><OL><LI>1) Add 10 µl plasmids  to 50 µl <I>E.coli</I> competent cells on ice and let the mix rest on ice for 15 min</LI><LI>2) Shock the mix at 42°C for 45 seconds, transfer the mix on ice for 10 min</LI><LI>3) Add 800 µl of Luria Bertoni (LB) media to the mix and let it recover at 37°C and 750 rpm on shakers for 1 hour</LI><LI>4) Plate 50 µl and 150 µl to plates containing LB agar media with chloramphenicol</LI><LI>5) Put plates in growth incubators at 37°C for 24 hours</LI></OL></P></DIV></DIV></SECTION></DIV></DIV><DIV id="toTop"><SPAN class="scrolltext">To top</SPAN></DIV></BODY></HTML>