<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Evry_Software skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Evry/Software</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="row"><DIV class="side-menu"><NAV class="navbar navbar-default" role="navigation"><DIV class="navbar-header"><DIV class="brand-wrapper"><BUTTON type="button" class="navbar-toggle" style="background-color:  #3f51b5;"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV></DIV><DIV class="side-menu-container" id="side-navigation"><UL class="nav navbar-nav"><LI class="panel panel-default dropdown" id="project-menu"><A data-toggle="collapse" href="#dropdown-project"> Project </A></LI><LI id="project_description"><A href="https://2015.igem.org/Team:Evry/Description">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Project/Chassis">Chassis choice</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Project/Encapsulation">Environment modulation</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Project/Biosensor">Hypoxia Bio-sensor</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Design">Proof of concept</A></LI></UL></DIV></NAV></DIV><LI class="panel panel-default dropdown" id="software-menu"><A data-toggle="collapse" href="#dropdown-software"> Antigen prediction </A></LI><LI><A href="https://2015.igem.org/Team:Evry/Software"><SPAN style="font-size:5%; color: transparent;">o</SPAN>Overview</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Software/Pipeline">Pipeline</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Software/Predictions">Prediction</A></LI></DIV></DIV><LI class="panel panel-default dropdown" id="practices-menu"><A data-toggle="collapse" href="#dropdown-practices"> Practices </A></LI><LI><A href="https://2015.igem.org/Team:Evry/Practices">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Practices/State">Funding state of art</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Practices/Innovative">Innovative funding</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Practices/Society">Society perception</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Practices/Conclusion">Conclusion</A></LI></DIV></DIV><LI class="panel panel-default dropdown" id="achievements-menu"><A data-toggle="collapse" href="#dropdown-achievements"> Achievements </A></LI><LI><A href="https://2015.igem.org/Team:Evry/Medals">Medal fulfillment</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Parts">Parts</A></LI></DIV><LI class="panel panel-default dropdown" id="notebook-menu"><A data-toggle="collapse" href="#dropdown-nb"> Notebook </A></LI><LI><A href="https://2015.igem.org/Team:Evry/Notebook">Day notes</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Protocols">Protocols</A></LI><LI class="panel panel-default dropdown" id="team-menu"><A data-toggle="collapse" href="#team-dropdown"> Team </A></LI><LI><A href="https://2015.igem.org/Team:Evry/Team">Members</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Acknowledgements">Acknowledgements</A></LI><DIV class="container"><DIV class="side-body" id="content-body"><DIV class="page-header"><H1>Antigen prediction</H1></DIV><P class="lead">We developed a software tool to select the best tumoral antigen candidates that could be used in the development of new personalized cancer immunotherapies, like the one our iGEM group is developing.</P><SECTION class="page-section"><H2>General concept</H2><DIV class="row"><DIV class="col-md-6"><STRONG>Figure 1:</STRONG> general concept of our antigen prediction pipeline. </DIV><DIV class="col-md-6"><P class="text-justify">Our idea was to develop a pipeline allowing us to select the best candidates tumoral antigens to use for a vaccine (immunotherapy). A good candidate must be tumor-specific and sufficiently expressed in tumor cells to be presented to the immune system. Furthermore, it must be able to be processed efficiently by the immune system. </P><P class="text-justify">We intended to identify relevant targets by:</P><UL><LI> - Looking for differentially expressed genes (specifically upregulated genes) in
tumoral tissue (vs. normal tissue), by a transcriptomic analysis (such as RNASeq)</LI><LI>- Looking for genetic mutations only found in tumoral tissue (identifying genetic
variants)</LI></UL><P class="text-justify">Once the targets are identified, the goal is not to express the whole corresponding genes but to express short fragments, corresponding to putative cleavage sites by
the proteasome to link to the MHC-I in order to potentially trigger an tumor-specific immune response. 
				</P></DIV></DIV></SECTION><SECTION class="page-section"><H2>RNAseq data</H2><P class="text-justify">We created the whole pipeline using data corresponding to RNAseq reads of <STRONG>melanocytes cell lines</STRONG>. All the figures and data presented in the software section were obtained using this dataset.</P><P class="text-justify">Experimental procedures for data generation are described in:
Vardabasso, C. et al. (2015). <STRONG>Histone Variant H2A.Z.2 Mediates Proliferation and Drug
Sensitivity of Malignant Melanoma.</STRONG><EM>Molecular Cell</EM>, 59:75-88</P><P class="text-justify">Data was accessed from the Sequence Read Archive website (NCBI). Study can be accessed <A href="http://ncbi.nlm.nih.gov/Traces/sra/?study=SRP057616" target="_blank">here</A> and sequencing data can be found <A target="_blank" href="http://ncbi.nlm.nih.gov/Traces/study/?acc=SRP057616">here</A>.
	</P></SECTION><SECTION class="page-section"><H2>Code availability</H2><P class="text-justify">Please see the <A href="http://github.com/fbucchini/igem-evry-pipeline" target="_blank">GitHub repository</A> of our antigen prediction pipeline for code and documentation. </P></SECTION><P class="lead">Start <A href="https://2015.igem.org/Team:Evry/Software/Pipeline">discovering our pipeline</A>.</P><SECTION class="page-section"><H2>References</H2><UL style="font-size: 80%;"><LI>Anders, S. and Huber, W. (2010). Differential expression analysis for sequence count data.
Genome Biology, 11:R106.</LI><LI>Bao, R. et al. (2014). Review of Current Methods, Applications, and Data Management for the
Bioinformatics Analysis of Whole Exome Sequencing. Cancer Informatics, 13(s2):67–82</LI><LI>Danecek, P. et al. (2011). The Variant Call Format and VCFtools. Bioinformatics, 27:2156-2158.</LI><LI>DePristo, M. et al. (2011). A framework for variation discovery and genotyping using nextgeneration
DNA sequencing data. Nature, 43:491-498.</LI><LI>McKenna, A. et al. (2010). The Genome Analysis Toolkit: a MapReduce framework for
analyzing next-generation DNA sequencing data. Genome Research, 20:1297-1303.</LI><LI>Piskol, R. et al. (2013). Reliable identification of genomic variants from RNA-Seq data. The
American Journal of Human Genetics, 93:641–651.</LI><LI>Quinlan, A. and Hall, I. (2010). BEDTools: a flexible suite of utilities for comparing genomic
features. Bioinformatics, 26:841-842.</LI><LI>R Development Core Team. (2008). R: A language and environment for statistical computing.
R Foundation for Statistical Computing, Vienna, Austria. URL: http://www.R-project.org.
San Lucas, F.A. et al. (2012). Integrated annotation and analysis of genetic variants from
next-generation sequencing studies with variant tools. Bioinformatics, 28:421-422.</LI><LI>Sherry, S.T. et al. (2001). dbSNP: the NCBI database of genetic variation. Nucleic Acids
Research, 29:308-11.</LI><LI>Van der Auwera, G.A. et al. (2013). From fastQ data to high-confidence variant calls: the
Genome Analysis Toolkit Best Practices Pipeline. Bioinformatics, 43:11.10.1-11.10.33.</LI><LI>Vardabasso, C. et al. (2015). Histone variant H2A.Z.2 mediates proliferation and drug
sensitivity of malignant melanoma. Molecular Cell, 59:75-88.</LI><LI>Wang, G., Peng, B. and Leal, S. (2014). Variant Association Tools for quality control and
analysis of large-scale sequence and genotyping array data. The American Journal of Human
Genetics, 94:770-783.</LI><LI>Wang, J., Duncan, D., Shi, Z. and Zhang, B. (2013). WEB-based GEne SeT AnaLysis Toolkit
(WebGestalt): update 2013. Nucleic Acids Research, 41(Web Server issue):W77-83.
Wang, K. et al. (2010). ANNOVAR: functional annotation of genetic variants from highthroughput
sequencing data. Nucleic Acids Research, 38:e164.</LI><LI>Kesmir C, Nussbaum AK, Schild H, Detours V, Brunak S.Prediction of proteasome cleavage motifs by neural networks.
Protein Eng., 15(4):287-96, 2002</LI><LI>Nielsen M, Lundegaard C, Blicher T, Lamberth K, Harndahl M, Justesen S, Roeder G, Peters B, Sette A, Lund O, Buus S.NetMHCpan, a method for quantitative predictions of peptide binding to any HLA-A and -B locus protein of known sequence.
</LI><LI>Ilka Hoof, Bjoern Peters, John Sidney, Lasse Eggers Pedersen, Ole Lund, Soren Buus, and Morten Nielsen. NetMHCpan - MHC class I binding prediction beyond humans 
</LI></UL></SECTION></DIV></DIV><DIV id="toTop"><SPAN class="scrolltext">To top</SPAN></DIV></BODY></HTML>