<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Evry_Software_Pipeline skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Evry/Software/Pipeline</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="row"><DIV class="side-menu"><NAV class="navbar navbar-default" role="navigation"><DIV class="navbar-header"><DIV class="brand-wrapper"><BUTTON type="button" class="navbar-toggle" style="background-color:  #3f51b5;"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV></DIV><DIV class="side-menu-container" id="side-navigation"><UL class="nav navbar-nav"><LI class="panel panel-default dropdown" id="project-menu"><A data-toggle="collapse" href="#dropdown-project"> Project </A></LI><LI id="project_description"><A href="https://2015.igem.org/Team:Evry/Description">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Project/Chassis">Chassis choice</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Project/Encapsulation">Environment modulation</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Project/Biosensor">Hypoxia Bio-sensor</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Design">Proof of concept</A></LI></UL></DIV></NAV></DIV><LI class="panel panel-default dropdown" id="software-menu"><A data-toggle="collapse" href="#dropdown-software"> Antigen prediction </A></LI><LI><A href="https://2015.igem.org/Team:Evry/Software"><SPAN style="font-size:5%; color: transparent;">o</SPAN>Overview</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Software/Pipeline">Pipeline</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Software/Predictions">Prediction</A></LI></DIV></DIV><LI class="panel panel-default dropdown" id="practices-menu"><A data-toggle="collapse" href="#dropdown-practices"> Practices </A></LI><LI><A href="https://2015.igem.org/Team:Evry/Practices">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Practices/State">Funding state of art</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Practices/Innovative">Innovative funding</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Practices/Society">Society perception</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Practices/Conclusion">Conclusion</A></LI></DIV></DIV><LI class="panel panel-default dropdown" id="achievements-menu"><A data-toggle="collapse" href="#dropdown-achievements"> Achievements </A></LI><LI><A href="https://2015.igem.org/Team:Evry/Medals">Medal fulfillment</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Parts">Parts</A></LI></DIV><LI class="panel panel-default dropdown" id="notebook-menu"><A data-toggle="collapse" href="#dropdown-nb"> Notebook </A></LI><LI><A href="https://2015.igem.org/Team:Evry/Notebook">Day notes</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Protocols">Protocols</A></LI><LI class="panel panel-default dropdown" id="team-menu"><A data-toggle="collapse" href="#team-dropdown"> Team </A></LI><LI><A href="https://2015.igem.org/Team:Evry/Team">Members</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Evry/Acknowledgements">Acknowledgements</A></LI><DIV class="container"><DIV class="side-body" id="content-body"><DIV class="page-header"><H1>Pipeline</H1></DIV><P class="text-justify">All the information presented on this page (quality-control, differential expression analysis, data visualisation, variant discovery) is also available as a <A href="https://static.igem.org/mediawiki/2015/c/c4/Igem_pipeline_description.pdf" _target="_blank">PDF file</A>.</P><DIV id="top-menu"><UL><LI style="height: 34px; padding-top: 8px;" class="text-muted"><STRONG>Jump to:</STRONG></LI><LI><A href="#qc-step">Data processing &amp; quality control</A></LI><LI><A href="#deg-step">Differential expression analysis</A></LI><LI><A href="#var-step">Variant discovery</A></LI></UL></DIV><SECTION class="page-section" id="qc-step"><H2>Data processing and quality control</H2><P class="text-justify text-muted"><STRONG>What we produced: </STRONG>FASTQ files (if starting with a different file format), FASTQC reports, BAM and SAM files. </P><P class="text-center"><STRONG>Figure 1:</STRONG> schematic overview of the pipeline for RNA-seq data analysis.</P></SECTION><SECTION class="page-section" id="deg-step"><H2>Differential expression analysis</H2><P class="text-justify text-muted"><STRONG>What we produced: </STRONG>script for differential expression analysis, table with read counts (tab separated format, 7 columns, ENSG ids).</P><P class="text-justify">RNA-seq  data  can  be  difficult  to  interpret  (especially  in  terms  of  differential  expression quantitation). Thus, we decided to adopt a simple method for the analysis, based on counting, for 
each  gene  and  for  each  sample,  the  number  of  available  reads  and  then  testing  for  significant 
differences between two experimental conditions or groups.</P><P class="text-justify">We  wrote  an  R  script  that  automatically  creates  a  PDF  file  (in  the  current  directory)  with  all  the 
figures  necessary  for  visual  inspection  and  result  interpretation. The  input  is  a  tab  separated  file 
with reads counts.</P><PRE>ensembl_id	melanocyte_1	melanocyte_2	melanoma_1	melanoma_2
ENSG00000000003	1964	2409	2328	2451
ENSG00000000005	0	2	10	12
ENSG00000000419	15122	19592	38225	36654
ENSG00000000457	12129	14893	7483	7812
ENSG00000000460	21930	25575	13123	13840
ENSG00000000938	48	58	26	42
ENSG00000000971	125	229	124	236
ENSG00000001036	11611	14125	14067	13518
ENSG00000001084	11429	13795	3549	3279</PRE><P class="text-center"><STRONG>Figure 2:</STRONG> Example input format for DE analysis.</P><P class="text-justify">We tested two designs, as illustrated in the tables below: normal cells vs cancerous cells (4 samples), cancerous cells vs cancerous drug treated (4 samples).</P><DIV class="row"><DIV class="col-md-4 col-md-offset-1"><TABLE class="table table-hover table-striped"><THEAD><TR><TH>Sample name</TH><TH>Condition</TH></TR></THEAD><TBODY><TR><TD>melanocyte_1</TD><TD>M</TD></TR><TR><TD>melanocyte_2</TD><TD>M</TD></TR><TR><TD>melanoma_1</TD><TD>C</TD></TR><TR><TD>melanoma_2</TD><TD>C</TD></TR></TBODY></TABLE></DIV><DIV class="col-md-4 col-md-offset-2"><TABLE class="table table-hover table-striped"><THEAD><TR><TH>Sample name</TH><TH>Condition</TH></TR></THEAD><TBODY><TR><TD>melanoma_1</TD><TD>C</TD></TR><TR><TD>melanoma_2</TD><TD>C</TD></TR><TR><TD>melanoma_drug_1</TD><TD>D</TD></TR><TR><TD>melanoma_drug_2</TD><TD>D</TD></TR></TBODY></TABLE></DIV></DIV><P class="text-center"><STRONG>Table 1 and 2: </STRONG>tested designs.</P><H3>Visual exploration of the samples</H3><P class="text-justify">Prior  to  checking  distances  between  our  samples,  we  applied  a  regularized-logarithm
transformation (rlog) to stabilise the variance across the mean. The effects of the transformation are shown in the figure below.</P><P class="text-center"><STRONG>Figure 3:</STRONG> Effect of the regularized-logarithm
transformation on 'melanocyte_1' and 'melanocyte_2' samples.</P><P class="text-justify">We noticed that this step was particularly important for genes with low read counts.</P><P class="text-justify">We  then  checked  the  distances  between  our  samples  by  performing  Principal  Components Analysis of the count data. </P><P class="text-center"><STRONG>Figure 4:</STRONG> Principal Components Analysis (PCA) plot, normal vs cancerous cells.</P><P class="text-justify">We observed that differences between groups (normal vs cancerous cells represented in the PCA plot  above)  were  greater  than  intra-groups  differences,  which  is  expected  in  this  kind  of  design. 
However,  as  the  inter-group  differences  were  so  pronounced,  we  figured  that  a  great  amount  of genes  would  appear  as  differentially  expressed. This  is  why  we  decided  to  apply  really  stringent 
thresholds for the detection:<UL><LI>- log2  fold  change  (logFC)  &gt;  5  for  upregulated  genes  or  log2  fold  change  (logFC)  &lt;  -5  for 
downregulated  genes.</LI><LI>- AND adjusted-p-value &lt; 0.01</LI></UL></P><H3>Differential expression analysis</H3><P class="text-justify">Firstly,  we  took  a  look  at  the  raw  data  (prior  to  any  kind  of  normalization).  We  calculated  mean counts for each gene and by condition and then the log2 fold change.</P><P class="text-justify">Prior  to  normalization,  we  filtered  the  data  set  to  remove  rows  with  very  little  or  no  information (remove  genes  with  no  counts  or  with  just  a  single  count).  This  allows  to  eliminate  17,386 transcripts already.</P><P class="text-justify">Using  the  DESeq  R  package  (from <A href="http://bioconductor.org/packages/release/bioc/html/DESeq.html" target="_blank">Bioconductor</A>), we were able to perform normalization of our data after calculation of size factors and we then were able to calculate mean counts for each gene and by condition and finally the logFC.</P><DIV class="row"><DIV class="col-md-6"><P class="text-center"><STRONG>Figure 5:</STRONG> Distribution of logFC(cancerous/normal) values - raw data.</P></DIV><DIV class="col-md-6"><P class="text-center"><STRONG>Figure 6:</STRONG> Distribution of logFC(cancerous/normal) values - normalized data.</P></DIV></DIV><P class="text-justify">Finally, we applied the nbinomWaldTest() function from the DESeq package to test for significance of coefficients in a negative binomial GLM, the model we used to assess differences in expression. As  previously  stated,  selection  of  significantly  up-  or  downregulated  genes  was  based  on  the 
establishment of two selection thresholds: logFC and adjusted p-value (Wald test M vs C).</P><P class="text-center"><STRONG>Figure 7:</STRONG> Differential expression as a function of mean expression. Left panel: threshold set at logFC &gt; 2 or &lt; -2. Right panel: threshold set at logFC &gt; 5 or &lt; -5. <EM>The red dots indicate genes for which the logFC was significantly higher than 5 or lower than -5. The circled point indicates the gene with the lowest adj-p-value.</EM></P><P class="text-justify">We  obtained  a  list  of  1,649  differentially  expressed  genes:  931  upregulated  genes  and  718 downregulated genes. </P><H3>Enrichment analysis</H3><P class="text-justify">We retrieved the list of the 931 unregulated genes and the list of the 718 downregulated genes and looked for significantly enriched GO (Gene Ontology) terms in these lists (independently).</P><P class="text-center"><STRONG>Figure 8:</STRONG>  Enrichment in GO terms, downregulated genes.</P><P class="text-center"><STRONG>Figure 9:</STRONG> Enrichment in GO terms, upregulated genes.</P></SECTION><SECTION class="page-section" id="var-step"><H2>Variant discovery</H2><P class="text-justify text-muted"><STRONG>What we produced: </STRONG><UL><LI>- Bash scripts for variant calling, quality control (filtering steps), and variant association analysis</LI><LI>- VCF files (before and after QC)</LI><LI>- Table: identified variants (exonic, non-synonymous)</LI></UL></P><P class="text-center"><STRONG>Figure 10:</STRONG> schematic overview of the pipeline for variant discovery and evaluation.</P><H3>Variant calling</H3><P class="text-justify">Genetic variants were called with samtools and bcftools, using sorted .bam files as input. This simple variant calling is followed by a stringent filtering step.</P><H3>Quality control (filtering steps)</H3><P class="text-justify">As recommended in the GATK Best Practices Guideline for variant discovery using RNA-Seq data, we applied hard filters to the raw variants obtained after variant calling, in an attempt to optimise both high sensitivity and specificity.</P><P class="text-justify">Furthermore,  as  we  only  have  4  samples,  we  decided  to  use  quite  stringent  parameters  / thresholds to filter the data, hoping to retain “true” and of as high a quality as possible variants. Filtering was performed using scripts from GATK and VCFtools.</P><P class="text-justify"><STRONG>Filters:</STRONG><UL><LI>(1) <STRONG>Diallelic</STRONG> variants only.</LI><LI>(2) <STRONG>Hardy-Weinberg equilibrium (HWE) deviation test</STRONG>. It is a common practice to remove sites that deviate from HWE because the deviation can be caused by genotyping errors. Normally, for case-control data, only controls should be tested for deviation from HWE (because for cases, sites associated with disease status can deviate from HWE). In our case, as all tests were performed in a bidirectional manner, deviation from HWE was tested in all the samples and we excluded sites with a HWE p-value &lt; 1.10<SUP>-7</SUP>.</LI><LI>(3) <STRONG>Call  rate  (percentage  of  samples  with  a  non-missing  genotype,  CR)</STRONG>.  The  proportion  of missing genotypes is an useful indicator of poor genotype quality. We decided to keep variants with a  CR  &gt;  98%,  which allows  to  keep  good  quality  variants  only. As  mean  CR  in  raw  data  was  of about 64%, we discarded over 60% of variants using this filter.</LI><LI>(4) Filtering based on <STRONG>Fisher Strand values</STRONG> (FS &gt; 30.0) and <STRONG>Quality by Depth</STRONG> (QD &lt; 2.0), as well as filtering out clusters of at least 3 SNPs in a window of 35 bases between them. </LI></UL></P><P class="text-justify">In order to assess the quality gain at each QC step, we estimated the ratio of transitions (Ti, purine to  purine  or  pyrimidine  to  pyrimidine  mutation)  to  transversions  (Tv,  purine  to  pyrimidine  or  vice 
versa) in the identified single nucleotide variants (SNVs). Particularly in coding regions, a higher number of transitions is expected, as transversions are more likely to change the underlying amino acid and lead to a deleterious mutation. Ti/Tv ratios are an approximate measure of quality: higher Ti/Tv ratios are associated with lower false positives. </P><P class="text-center"><STRONG>Figure 11:</STRONG>  Number of variants retained and Ti/Tv ratio for every QC step.</P><TABLE class="table table-hover table-striped"><THEAD><TR><TH>QC_stage</TH><TH>NVAR</TH><TH>Call rate</TH><TH>Ti/Tv</TH><TH>meanQUAL</TH></TR></THEAD><TBODY><TR><TD><STRONG>Raw_data</STRONG></TD><TD>868330</TD><TD>0.68</TD><TD>2280</TD><TD>98</TD></TR><TR><TD><STRONG>Diallelic_only</STRONG></TD><TD>868037</TD><TD>0.69</TD><TD>2280</TD><TD>98</TD></TR><TR><TD><STRONG>HWE_pvalue</STRONG></TD><TD>868037</TD><TD>0.69</TD><TD>2280</TD><TD>98</TD></TR><TR><TD><STRONG>CR_98</STRONG></TD><TD>294034</TD><TD>1</TD><TD>2423</TD><TD>223</TD></TR></TBODY></TABLE><P class="text-center"><STRONG>Table 3: </STRONG>Number of variants retained and Ti/Tv ratio for every QC step.</P><H3>Annotation</H3><P class="text-justify">Annotation  attributes  such  as  genomic  region,  gene  name,  variant  type  and  consequence  are attached to the variants list according to the reference hg19 using ANNOVAR (AnnotateVariation perl script). The primary genomic effects that are annotated include splice sites, nonsense, nonsynonymous and synonymous variants.</P><H3>Association testing between individual variants and phenotypic traits</H3><P class="text-justify">Here, common variants were defined as being those that are present in more than one sample. Of the 294 034 variants retained after quality control, 233 294 were identified as common. We identified 24 347 exonic variants only, over 19 000 of these were common. Thus, we decided to work only on common variants.
We performed standard single variant test to assess association: logistic regression and fisher’s exact test. </P><P class="text-justify">We found 531 exonic non-synonymous variants having a Fisher’s p-value &lt; 0.05 (p = 0.02, being the lowest value we could get with 4 samples). 315 of these variants were only present in the melanoma cell lines (all were homozygous variants).</P></SECTION><SECTION class="page-section"><P class="text-justify">After idenfication of genes that are both overexpressed and mutated in tumor samples, we want to know if good candidate antigens can be predicted. Read more about the <A href="https://2015.igem.org/Team:Evry/Software/Predictions">prediction step</A>. </P></SECTION></DIV></DIV><DIV id="toTop"><SPAN class="scrolltext">To top</SPAN></DIV></BODY></HTML>