<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Purdue_ProteinProfiles skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Purdue/ProteinProfiles</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Purdue">HOME </A></LI><LI class="menu_item"><A href=" https://2016.igem.org/Team:Purdue/Team">TEAM</A></LI><LI class="menu_item"> BACKGROUND  
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Problem">Problem </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Phosphorous">Phosphorous </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Wastewater">Wastewater </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Organisms">Organisms </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/ProteinProfiles">Protein Profiles </A></LI></UL><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Description"> ★ Description </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Design"> ★ Design </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Model"> ★ Model </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Hardware"> Hardware/ Prototypes</A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Experiments"> ★ Experiments </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Proof"> ★ Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Results"> ★ Results </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/MDC/MFC">Side Project </A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Parts">★ Parts </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Composite_Part"> ★ Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Part_Collection"> ★ Part Collection </A></LI><LI class="menu_item"> HUMAN PRACTICES 
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Human_Practices"> ★ Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/HP/Silver">★ Silver </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/HP/Gold">★ Gold </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Integrated_Practices"> ★ Integrated Practices </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Engagement"> ★ Engagement </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Collaborations"> ★ Collaborations </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/SYNENERGENE">SYNENERGENE </A></LI><LI class="menu_item"> NOTES FROM THE LAB
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Notes_from_the_Lab"> ★ Notes from the Lab </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Notebook"> ★ Notebook </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Protocols"> ★ Protocols </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/InterLabStudy"> InterLab Study </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Safety"> Safety </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Purdue/Attributions">  ATTRIBUTIONS </A></LI></DIV><DIV class="content_wrapper"><DIV style="right:0px;top:60px;position:fixed;"><P>Purdue Biomakers</P><P style="padding-left: 30px;"><A title="Purdue Biomakers Twitter" href="https://twitter.com/purduebiomakers">Twitter</A></P><P style="padding-left: 30px;"><A title="Purdue Biomakers Facebook" href="https://www.facebook.com/PurdueBiomakers/">Facebook</A></P><P style="padding-left: 30px;"><A title="Purdue Biomakers Website" href="http://purduebiomakers.com/">Website</A></P></DIV><DIV class="column full_size"><H1>Meet the Proteins</H1><P>We have selected ten genes from <I>Microlunatus phosphovorus</I>  that encode for proteins relevant to our project goals:  phosphate remediation and recovery. Only one of these proteins has already been characterized prior to our project.</P><P><CENTER><B>Figure 1:</B> Phosphate enters the cell through Pit proteins, it is then converted from nucleoside phosphates to polyphosphate via kinases. Cells use glucokinases to phosphorylate glucose using the terminal phosphate of a polyphosphate chain. When phosphates are needed exopolyphosphatase breaks the last phosphate on the chain free. Stars depict uncharacterized proteins, and their putative function was predicted using phylogenetic analysis.</CENTER></P><P>These ten proteins can be broken up into four different categories:</P><H3>Transport</H3><P>Inorganic phosphate transporters (Pit) transport metal-chelated phosphate across the cell membrane using the proton motive force [1]. We have selected three Pit gene homologs from <I>M. phosphovorus</I> for expression in <I>E. coli</I>.</P><H3>Phosphate transfer (kinase)</H3><P> Polyphosphate kinases (PPK) catalyze the reversible transfer of phosphate between nucleoside phosphates and polyphosphates. [1] There are three PPK subtypes: PPK1 favors polyphosphate synthesis, PAP favors polyphosphate hydrolysis, and PPK2’s affinity for hydrolysis or synthesis varies by homolog. [2] We have selected four PPK homologs from <I>M. phosphovorus</I> to express in <I>E. coli</I>:  one PPK1, two PPK2s, and one similar to both a PPK2 and PAP.</P><H3>Phosphate transfer (glucokinase)</H3><P>Polyphosphate-dependent glucokinases (PPGK) catalyze the first step of glycolysis, the phosphorylation of glucose using a phosphate group from polyphosphate or ATP. We have selected two PPGK homologs from <I>M. phosphovorus</I> to express in <I>E. coli</I>, one of which is the only protein of the ten chosen that have been characterized. The characterized PPGK cannot use ATP, and is referred to as PPGK ATP-independent (ATPI) [1]. </P><H3>Preparation for Export</H3><P>Exopolyphosphatase (PPX) hydrolyzes of the terminal phosphate of a polyphosphate chain [1] The presence of PPX decreases polyphosphate accumulation, which allows PPX to maintain polyphosphate balance. [3] There are two variants of PPX proteins, PPX1 and PPX2, and we have  selected one PPX2 homolog from <I>M. phosphovorus</I> to express in <I>E. coli</I>.</P><H2>A Closer Look</H2><H3>Inorganic Phosphate Transporters</H3><P><CENTER><B>Figure 2:</B> Potential three-dimensional structures of <I>M. phosphovorus</I> Pit homolog A (left), Pit homolog B (middle), and Pit homolog C (right), as predicted by I-TASSER software [4]. The C-score measures confidence, in which 2 represents the highest level of confidence, and -5 represents the lowest level of confidence. Data for protein size was found using the Universal Protein Resource (Uniprot) [5].</CENTER></P><P>The primary structure of 

<I>E. coli</I>’s PitA was found on Uniprot [5]. Primary structures (amino acid sequence) were compared using BLAST [7]. Since the amino acid sequence of all <I>M. phosphovorus</I> Pits have relatively low identity similarities to one another and the Pit native to E. coli, they likely function in different capacities. Because of this low similarity, we believe that expressing and characterizing all three Pit genes would be worthwhile and that expressing <I>M. phosphovorus</I> Pits could be beneficial in increasing E. coli’s ability to uptake phosphate.</P><H3>Polyphosphate Kinases</H3><P><CENTER><B>Figure 3:</B> Potential three-dimensional structures of PPK1 (top left), PPK2 homolog A (top right), PPK2 homolog B (bottom left), and PPK2 homolog C (bottom right), as predicted by I-TASSER software [4]. The C-score measures confidence, in which 2 represents the highest level of confidence, and -5 represents the lowest level of confidence. Data for protein size was found using the Universal Protein Resource (Uniprot) [5].</CENTER></P><P><I>Important note: </I>The NCBI Reference Sequence, and not the GenBank sequence, was used for PPK2 homolog C. This is the only case in which the two sequences vary from one another. </P><P>While none of the four PPKs in <I>M. phosphovorus</I> have been previously characterized, the affinity for polyphosphate hydrolysis or synthesis could be predicted by phylogenetic analysis for PPK1, PPK2 homolog A, and PPK2 homolog B. **PPK2 homolog C is phylogenetically similar to other uncharacterized PPKs [1]. </P><P>The primary structure of <I>E. coli</I>’s PitA was found on Uniprot. [5]. Primary structures (amino acid sequence) were compared using BLAST [6]. * No significant similarity was found for PPK1 and PPK2 homolog B. Since the amino acid sequence of all <I>M. phosphovorus</I> PPKs have relatively low identity similarities to one another and the PPK native to <I>E. coli</I>, they likely function in different capacities. Because of this low similarity, we believe that expressing and characterizing all four PPK genes would be worthwhile and that expressing <I>M. phosphovorus</I> PPKs could be beneficial in modifying <I>E. coli</I>’s ability to store polyphosphate.</P><H3>Polyphosphate-dependent Glucokinases</H3><P><CENTER><B>Figure 4:</B> Potential three-dimensional structures of PPGK ATP-independent (ATPI) (left) and PPGK homolog (right), as predicted by I-TASSER software [4]. The C-score measures confidence, in which 2 represents the highest level of confidence, and -5 represents the lowest level of confidence. Data for protein size was found using the Universal Protein Resource (Uniprot) [5].</CENTER></P><P>PPGK ATP-independent cannot use ATP to phosphorylate glucose, and it is also the only protein of the ten used that has been previously characterized [7]. The second PPGK homolog’s ability to utilize ATP is unknown.</P><P>Primary structures were compared using BLAST [6]. Because PPGKs are not native to <I>E. coli</I>, we believe that expressing PPGK could be beneficial in allowing <I>E. coli</I> to utilize its polyphosphate reserve. As <I>M. phosphovorus</I>’s PPGKs have only a 50% similarity to one another, we believe that characterizing both would be useful, as their functioning capacity may differ.</P><H3>Exopolyphosphatase</H3><P><CENTER><B>Figure 5</B>: Potential three-dimensional structures of PPX2, as predicted by I-TASSER software [4]. The C-score measures confidence, in which 2 represents the highest level of confidence, and -5 represents the lowest level of confidence. Data for protein size was found using the Universal Protein Resource (Uniprot) [5].</CENTER></P><P>Identity similarity of PPX2 from <I>M. phosphovorus</I> and PPX2 from <I>E. coli</I>:  31%
Primary structures were compared using BLAST [6]. As the amino acid sequences have a relatively low level of similarity, it is probable that they function at different capacities, which is why we believe expressing <I>M. phosphovorus</I>’s PPX2 could be beneficial in increasing <I>E. coli</I>’s ability to export phosphate.</P><H2>References</H2><P>
[1]     A. Kawakoshi, H. Nakazawa, J. Fukada, M. Sasagawa, Y. Katano, S. Nakamura, A. Hosoyama, H. Sasaki, N. Ichikawa, S. Hanada, Y. Kamagata, K. Nakamura, S. Yamazaki and N. Fujita, &quot;Deciphering the Genome of Polyphosphate Accumulating Actinobacterium <I>Microlunatus phosphovorus</I>&quot;, <I>DNA Research</I>, vol. 19, no. 5, pp. 383-394, 2012.
[2]     L. Batten, A. Parnell, N. Wells, A. Murch, P. Oyston and P. Roach, &quot;Biochemical and structural characterization of polyphosphate kinase 2 from the intracellular pathogen <I>Francisella tularensis</I>&quot;, <I>Bioscience Reports</I>, vol. 36, no. 1, pp. e00294-e00294, 2015.
[3]     S. Thayil, N. Morrison, N. Schechter, H. Rubin and P. Karakousis, &quot;The Role of the Novel Exopolyphosphatase MT0516 in <I>Mycobacterium tuberculosis</I> Drug Tolerance and Persistence&quot;,<I> PLoS ONE </I>, vol. 6, no. 11, p. e28076, 2011.
[4]     Yang, R. Yan, A. Roy, D. Xu, J. Poisson and Y. Zhang, &quot;The I-TASSER Suite: protein structure and function prediction&quot;, <I>Nature Methods</I>, vol. 12, no. 1, pp. 7-8, 2014. 
[5]     &quot;Universal Protein Resource&quot;, <I>Uniprot</I>, 2016. [Online]. Available: http://www.uniprot.org/. [Accessed: 16- Oct- 2016].
[6]     BLAST Scoring Parameters, E. Michael Gertz, 16 March 2005.
[8]     S. Tanaka, S. Lee, K. Hamaoka, J. Kato, N. Takiguchi, K. Nakamura, H. Ohtake and A. Kuroda, &quot;Strictly Polyphosphate-Dependent Glucokinase in a Polyphosphate-Accumulating Bacterium, <I>Microlunatus phosphovorus</I>&quot;, <I>Journal of Bacteriology</I>, vol. 185, no. 18, pp. 5654-5656, 2003.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>