<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Purdue_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Purdue/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Purdue">HOME </A></LI><LI class="menu_item"><A href=" https://2016.igem.org/Team:Purdue/Team">TEAM</A></LI><LI class="menu_item"> BACKGROUND  
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Problem">Problem </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Phosphorous">Phosphorous </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Wastewater">Wastewater </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Organisms">Organisms </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/ProteinProfiles">Protein Profiles </A></LI></UL><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Description"> ★ Description </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Design"> ★ Design </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Model"> ★ Model </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Hardware"> Hardware/ Prototypes</A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Experiments"> ★ Experiments </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Proof"> ★ Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Results"> ★ Results </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/MDC/MFC">Side Project </A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Parts">★ Parts </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Composite_Part"> ★ Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Part_Collection"> ★ Part Collection </A></LI><LI class="menu_item"> HUMAN PRACTICES 
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Human_Practices"> ★ Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/HP/Silver">★ Silver </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/HP/Gold">★ Gold </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Integrated_Practices"> ★ Integrated Practices </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Engagement"> ★ Engagement </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Collaborations"> ★ Collaborations </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/SYNENERGENE">SYNENERGENE </A></LI><LI class="menu_item"> NOTES FROM THE LAB
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Notes_from_the_Lab"> ★ Notes from the Lab </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Notebook"> ★ Notebook </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Protocols"> ★ Protocols </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/InterLabStudy"> InterLab Study </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Safety"> Safety </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Purdue/Attributions">  ATTRIBUTIONS </A></LI></DIV><DIV class="content_wrapper"><DIV style="right:0px;top:60px;position:fixed;"><P>Purdue Biomakers</P><P style="padding-left: 30px;"><A title="Purdue Biomakers Twitter" href="https://twitter.com/purduebiomakers">Twitter</A></P><P style="padding-left: 30px;"><A title="Purdue Biomakers Facebook" href="https://www.facebook.com/PurdueBiomakers/">Facebook</A></P><P style="padding-left: 30px;"><A title="Purdue Biomakers Website" href="http://purduebiomakers.com/">Website</A></P></DIV><H2>Plate Reader Calibration Protocols</H2><P>
1. OD600 Reference point

You will use LUDOX-S30 as a single point reference to obtain a ratiometric conversion factor to
transform your absorbance data into a standard OD600 measurement. This has two key objectives.With standard 1 cm pathlength spectrophotometers, the reading is still instrument dependent (see above). With plate readers the path length is less than 1 cm and is volume dependent. In this instance the ratiometric conversion can both transform Abs600 measurements (i.e. the basic output of the instrument and not standardised optical density with 1 cm pathlength) into OD600 measurements, whilst simultaneously accounting for instrument differences. 

[IMPORTANT NOTE: many plate readers have an automatic path length correction, this is based on volume adjustment using a ratio of absorbance measurements at 900 and 950 nm. Because scattering increases with longer wavelengths, this adjustment is confounded by scattering solutions, such as dense cells. YOU MUST THEREFORE TURN OFF PATHLENGTH CORRECTION.] 

To measure your standard LUDOX Abs600 you must use the same cuvettes, plates and volumes (suggestion: use 100 μl for plate reader measurement and 1 mL for spectrophotometer measurement) that you will use in your cell based assays. The LUDOX solution is only weakly scattering and so will give a low absorbance value. 

If using plates prepare a column of 4 wells with 100 μl 100% LUDOX and 4 wells containing 100 μl H2O. Repeat the measurement in all relevant modes used in your experiments (e.g. settings for orbital averaging). 

If using a cuvette, you will only have enough material for a single measurement, but repeat the reading multiple times. Use the same cuvette to measure the reference with H2O (this value will be subtracted by the instrument to give the OD600 reading).

Materials:

1ml LUDOX (provided in kit)
H20 (provided by team)
96 well plate or cuvettes (provided by team)

Method

Add 100 μl LUDOX into wells A1, B1, C1, D1 (or 1 mL LUDOX into cuvette)
Add 100 μl of H2O into wells A2, B2, C2, D2 (or 1 mL H2O into cuvette)
Measure absorbance 600 nm of all samples in all standard measurement modes in instrument 
Record the data in the table below or in your notebook
Import data into Excel (OD600 reference point tab) Sheet_1 provided 

2. Protocol FITC fluorescence standard curve

You will prepare a dilution series of FITC in 4 replicates and measure the fluorescence in a 96 well 
plate in your plate reader or individually in cuvettes in a fluorimeter. By measuring these in all standard modes in your plate reader or fluorimeter, you will generate a standard curve of fluorescence for FITC concentration. You will be able to use this to correct your cell based readings to an equivalent fluorescein concentration. You will then be able to convert this into a concentration of GFP.

Before beginning this protocol ensure that you are familiar with the GFP settings and measurement modes of your instrument.

Materials:

187 μg FITC (provided in kit)
10ml 1xPBS (phosphate buffered saline; provided by team)
96 well plate or cuvettes (provided by team)

Method

Prepare the FITC stock solution:

Spin down FITC stock tube to make sure pellet is at the bottom of tube.
Prepare 10x FITC stock solution by resuspending FITC in 1 mL of 1xPBS
Incubate the solution at 42°C for 4 hours
Dilute the 10x FITC stock solution in half with 1xPBS to make a 5x FITC solution and resulting concentration of FITC stock solution 2.5 μM.

[Note: it is important that the FITC is properly dissolved. To check this after the incubation period pipetted up and down – if any particulates are visible in the pipette tip continue to incubate overnight.]

Prepare the serial dilutions of FITC:

Accurate pipetting is essential. Serial dilutions will be performed across columns 1-11. COLUMN 12 MUST CONTAIN PBS BUFFER ONLY. Initially you will setup the plate with the FITC stock in column 1 and an equal volume of 1xPBS in columns 2 to 12. You will perform a serial dilution by consecutively transferring 100 μl from column to column with good mixing.

Add 100 μl of PBS into wells A2, B2, C2, D2....A12, B12, C12, D12
Add 200 μl of FITC 5x stock solution into A1, B1, C1, D1
Transfer 100 μl of FITC stock solution from A1 into A2.
Mix A2 by pipetting up and down 3x and transfer 100 μl into A3...
Mix A3 by pipetting up and down 3x and transfer 100 μl into A4...
Mix A4 by pipetting up and down 3x and transfer 100 μl into A5...
Mix A5 by pipetting up and down 3x and transfer 100 μl into A6...
Mix A6 by pipetting up and down 3x and transfer 100 μl into A7...
Mix A7 by pipetting up and down 3x and transfer 100 μl into A8...
Mix A8 by pipetting up and down 3x and transfer 100 μl into A9...
Mix A9 by pipetting up and down 3x and transfer 100 μl into A10...
Mix A10 by pipetting up and down 3x and transfer 100 μl into A11...
Mix A11 by pipetting up and down 3x and transfer 100 μl into liquid waste

TAKE CARE NOT TO CONTINUE SERIAL DILUTION INTO COLUMN 12.

Repeat dilution series for rows B, C, D
Measure fluorescence of all samples in all standard measurement modes in instrument
Record the data in your notebook
Import data into Excel (FITC standard curve tab) Sheet_1 provide
</P><H2>Plate Reader Measurement protocol</H2><P>
Prior to performing the measurement on the cells you should perform the calibration measurements. This will ensure that you understand the measurement process and that you can take the cell measurements under the same conditions.

Materials:

Competent cells (Escherichia coli strain DH5α)
LB (Luria Bertani) media as an alternative
Chloramphenicol (stock concentration 25 mg/mL dissolved in EtOH)50 ml Falcon tube
Incubator at 37°C
1.5 ml eppendorf tubes for sample storage
2x 96-well plates:
	1 Completely translucent - clear
	1 Completely opaque - black 
Ice bucket with ice
Pipettes

Devices (from InterLab Measurement Kit):
Positive control
Negative control
Device 1: J23101+I13504
Device 2: J23106+I13504
Device 3: J23117+I13504
</P><P>
Method

Day 1: transform Escherichia coli DH5α with these following plasmids:
Positive control
Negative control
Device 1: J23101+I13504
Device 2: J23106+I13504
Device 3: J23117+I13504

Day 2: Pick 2 colonies from each of plate and inoculate it on 5-10 mL LB medium + Chloramphenicol.

Grow the cells overnight (16-18 hours) at 37°C and 220 rpm.

Day 3: Cell growth, sampling, and assay

Using 96-well plates and a plate reader:

Set your instrument to read OD600 (as OD calibration setting)
Measure OD600 of the overnight cultures
Record data in your notebook
Import data into Excel (normalisation tab) Sheet_1 provided
Dilute the cultures to a target OD600 of 0.02 (see the volume of preloading culture and media in Excel (normalisation tab) Sheet_1) in 20 ml LB medium + Chloramphenicol in 50 mL falcon tube.
Incubate the cultures at 37°C and 220 rpm.
Take 200 μL (1% of total volume) samples of the cultures at 0, 1, 2, 3, 4, 5, and 6 hours of incubation
Place samples on ice.
At the end of sampling point you need to measure your samples (OD and Fl measurement), see the below for details.
Record data in your notebook
Import data into Excel (cell measurement tab) Sheet_1 provided 

Measurement

It is important that you use the same instrument settings that you used when measuring the FITC standard curve. This includes using the sample volume 100 ul. Samples should be laid out according to the figure below. Pipette 100 μl of each sample into each well of the clear plate and then 100 ul of sample into each well of the black plate. Set the instrument settings as those that gave the best results in your calibration curves (no measurements off scale). If necessary you can test more than one of the previously calibrated settings to get the best data (no measurements off scale).

Hint:
No measurement off scale means the data you get does not out of range of your calibration curve.
</P><H2>Transformation Protocol</H2><P>
Materials:

Resuspended DNA
10pg/µl Control DNA
Competent Cells
2ml Microtubes
Floating Foam Tube Rack
Ice &amp; ice bucket
Lab Timer
42°C water bath
SOC Media
37°C incubator
LB broth
Chloramphenicol 
Petri plates w/ LB agar and chloramphenicol antibiotic
Sterile spreader or glass beads
Pipettes and Tips
Protocol
<OL><LI>Thaw competent cells on ice</LI><LI>Pipette 25µl of competent cells into 2ml tube</LI><LI>Pipette 1µl of resuspended DNA into 2ml tube</LI><LI>Pipette 1µl of control DNA into 2ml tube</LI><LI>Close 2ml tubes, incubate on ice for 30min</LI><LI>Heat shock tubes at 42°C for 1 min</LI><LI>Incubate on ice for 5min</LI><LI>Pipette 200µl SOC media to each transformation</LI><LI>Incubate at 37°C for 2 hours, shaker or rotor recommended:</LI><LI>Pipette each transformation on two chlor petri plates for a 20µl and 200µl plating</LI><LI>Incubate transformations overnight (14-18hr) at 37°C</LI><LI>Pick single colonies and inoculate in 5ml of LB with chlor antibiotic to grow up cell cultures overnight (14-18hr) at 37°C</LI></OL></P><H2>Continuous culture protocol from Execter iGEM</H2><P>
This is the protocol we are using for the sequential batch culture, we are also using a ministat to continuous culture our kill switch. This is the main part of our project and is quite a lot of work so adapt this as you like to fit it in.

Make LB broth and No Salt LB broth.

LB broth recipe for 1 litre is 10g Tryptone, 5g Yeast extract, 10g NaCl

No salt LB is as above but No NACl (we are doing this as we can’t find a reason for having salt in LB and are interested to see the growth characteristics)

Transform your killswitch into whichever strain is best for your killswitch and overnight a colony in 5 ml of LB broth.

Take the OD in triplicate of the overnight and get an average. Use whatever settings you have for your plate reader. Calculate the amount of overnight culture you need to add to get a starting OD of 0.05 in 50 ml of media

Use this equation (0.05*50)/starting average OD

Put 50 ml of the media (one of LB and one of No salt LB) into 250ml erlenmeyer flasks and add antibiotics for the control you are using to the recommended iGEM concentration. Amp- 100 micrograms/ml, Cm- 35 micrograms /ml, Kan- 50micrograms/ml, Tet- 5 microgram/ml

Innoculate with the required amount and incubate the flasks at 37 degrees and 220 rpm.

Each morning and evening, take the OD of the culture and add to a fresh flask as before to reach a starting OD of 0.05 again. We are not allowed in the lab before nine or after five thirty so these are the times we will be doing it but whatever suits you is fine. This will keep the culture going.

Take a sample from the culture and make a glycerol stock using 0.5ml of sample and 0.5ml of 50% glycerol every day in week 1 and every two days in week 2 (this is what we are doing if it’s too much then however is fine). Then test all of these in whichever way is appropriate for your killswitch. 

If you could miniprep and send for sequencing DNA from each of your samples this would also be really interesting as we would like to know how much mutation happens, and how much needs to happen in order for a kill switch no longer be functional.
</P><H2>3A Assembly Protocol</H2><P>This protocol is used to combine two parts into a plasmid backbone with higher efficiency than standard assembly

Materials</P><UL><LI>RFC 10 compatible parts</LI><LI>Linearlized plasmid backbone</LI><LI>EcoRI</LI><LI>XbaI</LI><LI>SpeI</LI><LI>PstI</LI><LI>NEB Buffer 2</LI><LI>BSA</LI><LI>dH2O</LI><LI>DNA Purification kit</LI><LI>Transformation materials (with positive/negative agar plate controls)</LI></UL>
Procedure
<OL type="”1”"><LI>Digest the first Part with EcoRI and SpeI (may use EcoRI-HF instead of EcoRI)</LI><OL type="A"><LI>Fill a PCR tube with the following reaction (see Discussion for example):
</LI><LI>500 ng plasmid DNA (see Discussion for example calculation)</LI><LI>Fill to 50μL with 1X NEB Buffer 2 (comes as 10X so you have to dilute)</LI><LI>.5 μl of 100X BSA (see Discussion for calculation)</LI><LI>1 μl each enzyme (always add enzyme last!)</LI></OL><LI>Incubate at 37°C for 60 min then at 80°C for 20 min to heat inactivate enzymes (return to 4°C forever at the end)</LI><LI>Place BSA solution in freezer after use</LI></OL><LI>Digest the second Part with XbaI and PstI</LI><LI>Digest plasmid backbone with EcoRI and PstI</LI><LI>Purify restriction digests with DNA purification kit (optional)</LI><LI>Ligate the two constructs and the linearized plasmid together
</LI><LI>For a total reaction volume of 20 μl, add 2 μl from each digest, 1 μl T4 DNA ligase, and 1X T4 DNA ligase reaction buffer</LI><LI>Incubate ligation at 16 degrees celsius overnight</LI><LI>Heat shock at 65 for 20 minutes</LI><LI>Store products at -20°C if not used immediately for transformation</LI><LI>Transform the ligation product</LI><DIV>In Paragraph Form:</DIV><P>First, digest the first Part with EcoRI and SpeI (may use EcoRI-HF instead of EcoRI). Next, Fill a PCR tube with the following reaction: 500 ng plasmid DNA, fill to 50 μL with 1X NEB Buffer 2 (comes as 10X so you have to dilute), 0.5 μL of 100X BSA, and 1 μL of each enzyme. Incubate at 37°C for 30 min then at 80°C for 20 min to heat inactivate enzymes (return to 4°C forever at the end). Digest the second Part with XbaI and PstI and digest plasmid backbone with EcoRI and PstI. Purify restriction digests with DNA purification kit. Then, ligate the two constructs and the linearized plasmid together. For a total reaction volume of 20 μl, add 2 μl from each digest, 1 μl T4 DNA ligase, and 1X T4 DNA ligase reaction buffer. Incubate ligation at room temperature for 10 min. Store products at -20°C if not used immediately for transformation. Finally, transform the ligation product.</P><P>
Discussion
	DNA Calculation Example (with 25ng/μL DNA)
		500ng DNA * (1μL/25ng) = 20 μl DNA solution
	Making 100X BSA Solution
<OL><LI>Cover a 100mL bottle with foil</LI><LI>Add 100mL of ddH20</LI><LI>Measure and add 1g of BSA</LI><LI>Swirl to dissolve</LI></OL>
	Total Digest Reaction Example
		Say you have 74.7ng/μL DNA
<OL><LI>Add 6.69μl DNA</LI><LI>Add .5 μl 100X BSA</LI><LI>Add 1 μl of first enzyme</LI><LI>Add 1 μl of second enzyme</LI><LI>Add 4.08 μl of 10X NEB Buffer 2</LI><LI>Add 36.73 μl of ddH20</LI></OL>
Source:
http://dspace.mit.edu/bitstream/handle/1721.1/65066/BioBrickAssemblyFinalAuthorsVersion.pdf?sequence=2
</P><H2>Acid Digestion for Phosphates</H2><P>
Purpose
This is a protocol for converting both polyphosphates and organic phosphates to the orthophosphate form.

Materials
Phenolphthalein
 Sulfuric acid solution
Carefully add 300 mL concentrated Sulfuric acid to approximately 600 mL distilled water and dilute to 1 L with distilled water.
Ammonium persulfate, crystal
Sodium hydroxide, 1M

Equipment
Hot plate
125 mL Erlenmeyer flask (acid washed)
50 mL graduated cylinders (acid washed)

Protocol
Measure 50 mL or an appropriate amount of sample diluted to 50 mL with distilled water. Add to the Erlenmeyer flask.
Add 1 drop phenolphthalein indicator.  If a red color develops, add sulfuric acid solution until color just disappears.
Add 1 mL of sulfuric acid solution and 0.4 g of ammonium persulfate.  
Boil gently for 30 to 40 minutes or until the total volume is 10 mL.
Cool, add 1 drop of phenolphthalein and neutralize to a faint pink color with 1 N sodium hydroxide.
Make up to 50 mL with distilled water.  The digested sample is then tested for total phosphate as outlined in Section 12.

Discussion
Provide any commentary or advice you or another person has for running this protocol

Source
Pennsylvania Department of Environmental Protection Document “Chapter 8:  TOTAL PHOSPHORUS”

Paragraph form
Obtain a 50 mL sample of the phosphate solution and place in the Erlenmeyer flask. If necessary, obtain a smaller volume of sample and dilute to 50 mL. Add a drop of phenolphthalein to the sample and, if the sample turns red, add drops sulfuric acid until the red color disappears. Then, add one mL of the prepared sulfuric acid solution and 0.4 grams of ammonium persulfate to the solution. Using the hot plate, boil the solution gently for 30 to 40 minutes, when the total volume is 10 mL. Let the solution cool, and then add another drop of phenolphthalein. Add drops of the 1 M sodium hydroxide solution until the solution is a faint pink color. Add distilled water to bring the total volume back to 50 mL. 
</P><H2>Bioreactor Flow Rate Protocol</H2><P>
Purpose 
To consistently measure flow rate from the bioreactor system

Materials
Bioreactor
Dry 1L bottles
Graduated Cylinder (100ml)
Two other people 
Water

Equipment
Stopwatch
Thermometer 

Protocol
Prepare bioreactor by ensuring tubing is tightly sealed, filter canisters are sitting upright on the table, and the pump is on and refilling the bioreactor. Record the water temperature. Ensure the water level in the bioreactor is staying at the constant level around the overflow port before opening any outlet streams. Adjust the inlet valve accordingly. If measuring only one outlet stream, open desired outlet stream and adjust bioreactor inlet valve to maintain constant water level in bioreactor. Let the effluent run into the large beaker while adjusting valves. Prepare for sampling by obtaining dry bottles and someone to time. Place the bottle in the already running effluent stream and immediately begin timing for 10 seconds. Remove the bottle from the stream at 10 seconds, DO NOT close the valve at 10 seconds. If measuring all outlet streams, follow the above procedure identically at each outlet valve. Make sure to adjust the inlet stream to maintain a constant water level. Three people are recommended, one per outlet stream and one additional person to run the stopwatch. Measure the effluent liquid volume in a large graduated cylinder. Repeat the process to get an average volume for 10 seconds. Divide the average output volume by 10 seconds to get flow rate per second. 
</P><H2>E. coli Transformation Protocol</H2><P>Purpose 
To transform parts from the registry into competent cells

Materials
<OL><LI>Parts from Kit Plates</LI><LI>Plates with Antibiotic Resistance (Must be warmed up in 37℃ room)
</LI><LI>2 plates per part (specific antibiotic type depends on part resistance)</LI><LI>1 plate for negative control (for each type of antibiotic used)</LI></OL><LI>1 LB Plate (for positive control)</LI><LI>1 LB Plate w/ amp for blank plate</LI><LI>Eppendorf Tubes</LI><OL type="a"><LI>2 tubes for controls</LI><LI>1 tube per part</LI></OL><LI>PCR Tubes (1 tube per part)</LI><LI>5 alpha Competent cells
</LI><LI>50µL for controls</LI><LI>25µL per part</LI><LI>SOC Media
</LI><LI>400µL for controls</LI><LI>200µL per part</LI><LI>Ice</LI><LI>Glass Beads</LI><LI>Pipettes and Tips (Filtered Tips if possible)</LI><LI>Deionized Water</LI>
Equipment
<OL><LI>42℃ Water Bath (BIND 134 balance room)</LI><LI>37℃ Incubator (with shaker)</LI></OL>
Expected Data
<UL><LI>Cell growth observed on the Positive Control (Ensures that cells are growing properly)</LI><LI>No cell growth observed on the Negative Control (Ensures that the antibiotic is functioning)</LI><LI>Observed growth and fluorescence on the RFP control plate</LI></UL>

Paragraph Form
The purpose of this protocol is to transform parts from the registry into competent cells. Prepare the lab space by wiping the counter with ethanol and lighting a flame. Next, add 10µL of DI water to the kit plates and pipette up and down. Transferred this to a labeled PCR tube and repeat for each part. Warm the water bath to 42℃. Transfer competent cells to ice from the -80℃ freezer. Next, add 25µL of competent cells to each Eppendorf tube. The top of the cell containers should be marked to show use, and each Eppendorf tube should be labeled with the part name of control. Return the cells to the -80℃ freezer as quickly as possible. Next, add 4 µL of DNA were to the experimental Eppendorf tubes. Mix the DNA well and store leftover DNA was in PCR tubes in the -20℃ freezer. Incubate tubes on ice for 45 minutes. Next, heat shock cells at 42℃ for 60 seconds to open the cell walls for DNA insertion. Place Eppendorf tubes into Styrofoam holders and place into the water bath. Time very precisely and immediately place tubes back into ice after 60 seconds. After about a minute, move tubes around in ice because the nearby ice has probably melted. Put cells back on ice for 5 minutes. Add 200µL of SOC media to each tube. Incubate cells for 2 hours at 37℃ at 250 rpm. Next, plate 200µL of the cells on antibiotic plates for the experimental and negative control. Plate 200µL of cells on LB plates labeled positive control. Scatter the cell suspension in drops on the plate. Pour about five glass beads onto the plate. Swirl the plate to move the glass beads around and evenly coat the media with the liquid suspension of cells. Ensure that all plates are properly labeled. Incubate plates overnight from 12-18 hours at 37℃. There should be cell growth on the positive control and no cell growth on the negative control.</P><H2>Extraction and Quantification of Intracellular Poly-P</H2><P>Reagents
<UL><LI>Sodium phosphate glass type 45 (Sigma-Aldrich, S4379-100MG-089K5011)</LI><LI>Toluidine blue dye (Loba Chemie, Cat. No. 52040-25GM)</LI><LI>Glacial acetic acid (Rankem, Product Code A0030)</LI><LI>Chloroform (HiMedia, Cat. No. AS039-2.5LT)</LI><LI>Isoamyl alcohol (HiMedia, Cat. No. MB091-500ML)</LI><LI>De-ionized water</LI></UL>
Equipment
<UL><LI>Boiling water bath (OVFU)</LI><LI>Sonicator (Sartorius Stedim Labsonic R M)</LI><LI>Centrifuge (Remi C-24 Plus)</LI><LI>Spectrophotometer (Intech Microprocessor Uv-Vis Spectrophotometer Single Beam 290)</LI></UL>

Procedure
Preparation of standard curve of sodium phosphate glass type 45
<OL><LI>Weigh 0.3 mg of sodium phosphate glass type 45 and dissolve it in 150 µl de-ionized water to make a 2 µg/ µl standard stock solution.</LI><LI>Make 30 mg/ L toluidine blue stock solution with double distilled water.</LI><LI>Make 0.2 N acetic acid stock solution with glacial acetic acid and double distilled water.</LI><LI>Use 1, 2, 3, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14 and 15 µl of the standard stock solution to obtain 2, 4, 6, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28 and 30 µg polyphosphate in the experimental set up.</LI><LI>Make three replicates of each set up.</LI><DIV><LI>Set up the experiment in test tubes following Table 1. (Suggested: change the amount of water to make volume constant) </LI><DIV><TABLE><TBODY><TR><TD>Serial Number</TD><TD>Amount of standard polyphosphate (ug)</TD><TD>Volume of standard polyphosphate solution added (uL)</TD><TD>Volume of de-ionized water added (uL)</TD><TD>Volume of toluidine blue solution added (ml)</TD><TD>Volume of acetic acid solution added (ml)</TD></TR><TR><TD>1</TD><TD>0</TD><TD>0</TD><TD>300</TD><TD>3</TD><TD>3</TD></TR><TR><TD>2</TD><TD>2</TD><TD>1</TD><TD>300</TD><TD>3</TD><TD>3</TD></TR><TR><TD>3</TD><TD>4</TD><TD>2</TD><TD>300</TD><TD>3</TD><TD>3</TD></TR><TR><TD>4</TD><TD>6</TD><TD>3</TD><TD>300</TD><TD>3</TD><TD>3</TD></TR><TR><TD>5</TD><TD>10</TD><TD>5</TD><TD>300</TD><TD>3</TD><TD>3</TD></TR><TR><TD>6</TD><TD>12</TD><TD>6</TD><TD>300</TD><TD>3</TD><TD>3</TD></TR><TR><TD>7</TD><TD>14</TD><TD>7</TD><TD>300</TD><TD>3</TD><TD>3</TD></TR><TR><TD>8</TD><TD>16</TD><TD>8</TD><TD>300</TD><TD>3</TD><TD>3</TD></TR><TR><TD>9</TD><TD>18</TD><TD>9</TD><TD>300</TD><TD>3</TD><TD>3</TD></TR><TR><TD>10</TD><TD>20</TD><TD>10</TD><TD>300</TD><TD>3</TD><TD>3</TD></TR><TR><TD>11</TD><TD>22</TD><TD>11</TD><TD>300</TD><TD>3</TD><TD>3</TD></TR><TR><TD>12</TD><TD>24</TD><TD>12</TD><TD>300</TD><TD>3</TD><TD>3</TD></TR><TR><TD>13</TD><TD>26</TD><TD>13</TD><TD>300</TD><TD>3</TD><TD>3</TD></TR><TR><TD>14</TD><TD>28</TD><TD>14</TD><TD>300</TD><TD>3</TD><TD>3</TD></TR><TR><TD>15</TD><TD>30</TD><TD>15</TD><TD>300</TD><TD>3</TD><TD>3</TD></TR></TBODY></TABLE></DIV><LI>Vortex the contents of each test tube and incubate for 15 minutes at 25°C.</LI><LI>Record the absorbance at 630 nm, by setting de-ionized water as blank.</LI><LI>Prepare a standard curve of sodium polyphosphate glass type 45 in Microsoft Excel by plotting the amount of polyphosphate in the X-axis and the A630 in the Y-axis (Figure 1).</LI></DIV></OL></P><P>Microbial cell lysis for extracting polyphosphate granule</P><LI>Take 5 ml of bacterial culture grown for 72 hours at 37°C in a medium with phosphorus source.</LI><LI>Make 2 mM with K2PO4, as https://www.ncbi.nlm.nih.gov/pmc/articles/PMC124021/ says that is a sufficient </LI>
		Stock - add 45 ul of 0.22 M stock to 5 ml of culture<LI>Take 10 ml of microalgal and cyanobacterial culture grown for 25 days at 28°C in BG-11 (non-N2 fixer medium) and BG-0 (N2 fixer medium).</LI><LI>Centrifuge the samples at 2,350 g for 5 minutes. Discard the supernatant.</LI><LI>Dissolve the cell pellets in 500 µl autoclaved de-ionized water and centrifuge at 2350 g for 5 minutes. Discard the supernatant.</LI><LI>Take the fresh weight of the samples.</LI><LI>Add 600 µl of de-ionized water to the samples and mix by flickering.</LI><LI>Sonicate the samples for 5 minutes at 30 Hz (Cycle 0.5, Amplitude 65%).</LI><LI>Place the tubes containing the samples in boiling water bath at 100°C and boil for 2 hours.</LI><P>Recovery of polyphosphate granules and quantification by spectrophotometer</P><LI>After boiling, cool down the tubes at room temperature.</LI><LI>Add 600 µl of 24:1 (v/v) chloroform: isoamyl alcohol solution to all the tubes. Mix by vigorous shaking.</LI><LI>Centrifuge at 13,520 g for 15 minutes at room temperature.</LI><LI>Collect the upper aqueous phase in separate tubes.
CRITICAL STEP: Care should be taken to pipette the aqueous phase without disturbing the organic phase. The presence of even small amount of organic phase in the next steps may give ambiguous results. </LI><LI>Take 300 µl of the aqueous phase in a fresh test tube. Add 3 ml each of toluidine blue solution (Stock conc. of 30 mg/ L) and 0.2 N acetic acid solution. Mix by gentle vortexing and incubate for 15 minutes at 25°C till the colour of the solution changes from blue to purple.
CRITICAL STEP: The change in colour implies that polyphosphate is successfully extracted and is present in the aqueous phase. No colour change indicates that extraction is unsuccessful and has to be done again.</LI><LI>Make a control in the same way with 300 µl de-ionized water.</LI><LI>Record the absorbance at 630 nm, by setting de-ionized water as blank.</LI>
Calculation of the amount of polyphosphate in the microbial cells
<LI>Calculate the amount of polyphosphate present in the samples by trend analysis of its A630 on the standard curve (Figure 1).</LI><LI>Calculate the amount of polyphosphate in µg present per gm of sample fresh weight by the following formula: (amount of polyphosphate in ug/ gm of sample fresh weight) = 1,000 x (ug of polyphosphate from standard curve) / (fresh weight of sample in mg)</LI>
Source: http://www.nature.com/protocolexchange/protocols/4073#/anticipated_results <H2>DNA Electrophoresis</H2><P>DNA electrophoresis is a method used to separate and visualize DNA.  An electric field can be used to pull DNA through an agarose gel matrix because the phosphate backbone in DNA is negatively charged.  Smaller pieces of DNA will move faster through the gel than larger pieces.  A molecular weight marker (DNA ladder) containing DNA fragments of known length can be used to determine the size (in base pairs) of the DNA samples run on the gel. </P><P>The migration rate of the DNA fragments will depend on the density of agarose in your gel.  However, the migration rate is not linearly related to the fragment size so different gel densities yield different resolutions.   Thick gels (&gt;1.5 % (w/v) agarose) give low resolution of large fragments, but high resolution of smaller fragments (&lt;500 bp).   The opposite is true for thinner gels – smaller fragments move quickly, stick together and never resolve while thicker bands separate and resolve.  Typical gels run in our lab contain 0.7 or 0.8% (w/v) agarose.  The chart below, from Biorad, can be used to select the best gel concentration for your application.</P><TABLE><TBODY><TR><TH>% (w/v) Agarose</TH><TH>DNA fragment size (kb)</TH></TR><TR><TD>.5</TD><TD>1-30</TD></TR><TR><TD>.75</TD><TD>.8-12</TD></TR><TR><TD>1</TD><TD>.5-10</TD></TR><TR><TD>1.25</TD><TD>.4-7</TD></TR><TR><TD>1.5</TD><TD>.2-3</TD></TR><TR><TD>2-5</TD><TD>.01-.5</TD></TR></TBODY></TABLE><UL><LI>For a 0.7% (w/v) gel, dissolve approximately 0.21 grams of agarose powder in 30 mL of 1X TAE in an Erlenmeyer flask or other microwaveable vessel.  1X TAE can be prepared from the 10X TAE concentrate (a common lab resource).</LI><LI>Add the 30 mL of 1X TAE into the flask containing the 0.21g of agarose powder.  Gently swirl the mixture a few times to disperse the agarose powder throughout the TAE.  The agarose should not dissolve in the TAE at this point.  Loosely plug the mouth of the flask with a couple of Kimwipes to prevent excessive water loss during microwaving.</LI><LI>Microwave the flask in the lab microwave for 70-80 seconds.  Remove the flask with a hot glove.  If you still see undissolved agarose in the flask, microwave it for an additional 20 seconds.  The final heated solution should be clear and colorless (no agarose powder should be visible).</LI><LI>Add 3 μL SYBR Safe in the 30 mL molten gel with 1:10,000 dilution. Gently swirl the mixture and make sure the SYBR Safe is evenly distributed. Let the flask cool briefly on the benchtop. </LI><LI>While your molten agarose is cooling, set up an electrophoresis tray and place a well comb in it.  The well comb creates the holes in the gel where you will add your DNA samples.  If you are planning on running samples larger than 15 μL on the gel (about what one large-toothed comb well will hold), you can carefully tape together multiple teeth on the well comb together to create a larger well.  Alternatively, you can run a larger gel with wider teeth.  For smaller volumes of liquid, the smaller-toothed comb wells can hold about 7 μL.</LI><LI>When the agarose cools such that you can hold the flask easily in the palm of your hand, pour the solution into an electrophoresis tray.  Be careful not to generate any bubbles in the molten agarose.  Bubbles can be removed by sucking them up using a pipet.</LI><LI><B>Immediately</B> rinse out the flask that contained the hot agarose with water.  Residual agarose will solidify quickly in the flask.</LI><LI>Let the solution cool and solidify into a gel.  This takes roughly 20 minutes.  Once the gel has solidified, gently remove the well comb from the gel.</LI><LI>Transfer the gel tray into an electrophoresis chamber.  <B>The side of the gel with the wells should be facing towards the black side (anode) of the chamber.</B>  Fill the chamber with 1X TAE until TAE just covers the gel.</LI><LI>Prepare each of your DNA samples in the following way:
x μL DNA Sample
10 – x μL 1X TAE
2 μL 6X Loading Dye (found in the 4oC refrigerator)
12 μL Total

For larger samples (x &gt; 10 μL), simply scale up the recipe to accommodate all of your sample.  If you plan on loading your samples into wells generated by the small-toothed comb, scale down this recipe by half (to make a total of 6 μL of sample).
</LI><LI>Prepare a 12 μL (or 6 μL if using the smaller wells) DNA ladder standard at a concentration of 0.25 μg/lane.   For the Thermo Fisher 1kb Plus DNA ladder (1 µg⁄µL), this comes out to 0.25 μL ladder, 9.75 μL 1X TAE, and 2 μL of 6X loading dye.</LI><LI>Carefully pipet your prepared DNA samples and the DNA ladder standard into the wells on your gel.</LI><LI><B>Gently</B> seal off the electrophoresis chamber with the chamber cap and plug the cables extending from the chamber cap into a Biorad power supply.</LI><LI>Turn on the power supply and set it to deliver 90 volts for 60 minutes.  Leave the current setting on the power unit alone.  Hit the Run button to start running the gel.
<I>Alternative:</I>  For more throughput or shorter run times, 2 half gels (2 sets of combs per gel) can be run at a setting of 110V, 30 min.   Note, the gels run only half as far so there is a hit in resolution.  However, with the optics on the imager, the effects of this are negligible.  As a general rule of thumb, the time affects the migration distance – shorter times equal shorter migration distances and lower resolutions.  Conversely, the voltage controls the migration speed.   Higher voltage increases the migration speed and increases the resolution.  However, higher voltage also heats the system potentially leading to smearing of bands (increased diffusion) or DNA damage (if it’s to be extracted later) so play with these settings carefully.
</LI><LI>Visualize your DNA on the UVP imager.</LI></UL><H2>Gel Extraction</H2><P>Written up by Melissa Robins</P><H4>Purpose</H4><P>Allows a DNA fragment to be isolated from an agarose gel.</P><H4>Materials</H4><UL><LI>QIAquick Gel Extraction Kit</LI><LI>100% Ethanol</LI><LI>100% Isopropanol</LI><LI>In some cases, 3M sodium acetate</LI></UL><H4>Equipment</H4><UL><LI>Scalpel or razor blade</LI><LI>Gel imager for band isolation</LI><LI>Water bath or thermocycler for incubation</LI><LI>Centrifuge (BIND 134)</LI></UL><H4>Protocol</H4><UL><LI>Cut the desired band from the agarose gel with a clean, sharp scalpel.</LI><LI>Weigh the gel slice in a colorless tube.</LI><LI>Add 3 volumes Buffer QG to 1 volume gel (100 mg gel ~ 100 uL). </LI><LI>Incubate at 50℃ for 10 min and vortex the tube every 2-3 min to dissolve the gel.</LI><LI>Check the color of the mixture. If it is orange or purple, add 10 uL 3M sodium acetate until the mixture turns yellow.</LI><LI>Add 1 gel volume isopropanol to the sample and mix.</LI><LI>Transfer mixture to a QIAquick spin column and centrifuge for 1 min at 13,000rpm. Discard flow-through.</LI><LI>If the DNA will be used for sequencing, add 500 uL Buffer QG to the QIAquick column and centrifuge for 1 min. Discard flow-through and place the spin column back into the same tube.</LI><LI>Add 750 uL Buffer PE and centrifuge for 1 min. Discard flow through.</LI><LI>Centrifuge the column for 1 min again to remove residual wash buffer.</LI><LI>Place QIAquick column into a clean 1.5 mL microcentrifuge tube.</LI><LI>To elute DNA, add 50 uL Buffer EB or water to the center of the QIAquick membrane and centrifuge the column for 1 min. OR, for increased DNA concentration, add 30 uL Buffer EB to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge for one min. Incubate for 4 min.</LI></UL><H4>Discussion</H4><P>The maximum amount of gel per spin column is 400mg. For &gt;2% agarose gels, add 6 volumes Buffer QG.</P><H4>Source</H4><P>QIAquick Gel Extraction Kit Instructions</P><H4>Paragraph Form</H4><P>By Jill Osterhus</P><P>Cut the desired band from the agarose gel with a clean, sharp scalpel. Weigh the gel slice in a colorless tube. Add 3 volumes Buffer QG to 1 volume gel (100 mg gel ~ 100 uL). Incubate at 50℃ for 10 min and vortex the tube every 2-3 min to dissolve the gel. Check the color of the mixture. If it is orange or purple, add 10 uL 3M sodium acetate until the mixture turns yellow. Add 1 volume isopropanol to the sample and mix. Transfer mixture to a QIAquick spin column and centrifuge for 1 min at 13,000rpm. Discard flow-through. If the DNA will be used for sequencing, add 500 uL Buffer QG to the QIAquick column and centrifuge for 1 min. Discard flow-through and place the spin column back into the same tube. Centrifuge the column for 1 min again to remove residual wash buffer. Place QIAquick column into a clean 1.5 mL microcentrifuge tube. To elute DNA, add 50 uL Buffer EB or water to the center of the QIAquick membrane and centrifuge the column for 1 min. For increased DNA concentration, add 30 uL Buffer EB to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge for one min. Incubate for 4 min.</P><H2>His tag  Protein Purification</H2><P>Provided by Thermo Scientific</P><P><B>Additional Materials Required</B>Note: The buffers listed below are recommended. To decrease nonspecific binding and increase yield, adjustments to the imidazole concentration might be required for specific proteins. <B>For native conditions prepare the following buffers:</B></P><UL><LI>Equilibration Buffer: 20mM sodium phosphate, 300mM sodium chloride (PBS) with 10mM imidazole; pH 7.4</LI><LI>Wash Buffer: PBS with 25mM imidazole; pH 7.4 </LI><LI>Elution Buffer: PBS with 250mM imidazole; pH 7.4 </LI></UL><B>For denaturing conditions prepare the following buffers: </B><UL><LI>Equilibration Buffer: PBS with 6M guanidine</LI><LI>HCl and 10mM imidazole; pH 7.4 </LI><LI>Wash Buffer: PBS with 6M guanidine</LI><LI>HCl and 25mM imidazole; pH 7.4 </LI><LI>Elution Buffer: PBS with 6M guanidine</LI><LI>HCl and 250mM imidazole; pH 7.4 </LI></UL><B>For resin regeneration prepare the following buffer:</B><UL><LI>MES Buffer: 20mM 2-(N-morpholine)-ethanesulfonic acid, 0.1M sodium chloride; pH 5.0 </LI></UL><H4>Procedure for Spin Purification of His-Tagged Proteins </H4><P><B>Note:</B> The total volume of the 0.2, 1 and 3mL columns are 1, 8 and 22mL, respectively. If a sample volume is greater than the column, perform multiple applications and centrifugations until the entire sample has been processed. Be careful not to exceed the resin’s binding capacity. The HisPur Ni-NTA Spin Columns also may be used for gravity-flow purifications. </P><OL><LI>Equilibrate column(s) to working temperature. Perform purifications at room temperature or at 4°C. </LI><LI>Prepare sample by mixing the protein extract with an equal volume of Equilibration Buffer. Use the Equilibration Buffer to adjust the total volume to be ≥ 2 resin-bed volumes. </LI><LI>Remove the bottom tab from the HisPur Ni-NTA Spin Column by gently twisting. Place column into a centrifuge tube. 
<B>Note:</B> Use 2.0, 15 or 50mL centrifuge tubes for the 0.2, 1 and 3mL spin columns, respectively. </LI><LI>Centrifuge column at 700 × g for 2 minutes to remove storage buffer. </LI><LI>Equilibrate column with two resin-bed volumes of Equilibration Buffer. Allow buffer to enter the resin bed. </LI><LI>Centrifuge column at 700 × g for 2 minutes to remove buffer. </LI><LI>Place the bottom plug in the column and add the prepared protein extract. Mix on an orbital shaker or end-over-end mixer for 30 minutes at room temperature or 4°C. </LI><LI>Remove the bottom plug. Centrifuge the column at 700 × g for 2 minutes and collect the flow-through in a centrifuge tube.</LI><LI>Wash resin with two resin-bed volumes of Wash Buffer. Centrifuge at 700 × g for 2 minutes and collect fraction in a centrifuge tube. Repeat this step two more times collecting each fraction in a separate centrifuge tube. </LI><LI>Elute His-tagged proteins from the resin by adding one resin-bed volume of Elution Buffer. </LI><LI>Centrifuge at 700 × g for 2 minutes. Repeat this step two more times, collecting each fraction in a separate tube. </LI><LI>Monitor protein elution by measuring the absorbance of the fractions at 280nm or by Coomassie Plus (Bradford) Assay Reagent (Product No. 23238). The eluted protein can be directly analyzed by SDS-PAGE. </LI></OL><P><B>Note:</B> To remove imidazole for downstream applications, use gel filtration (e.g., Thermo Scientific Zeba Spin Desalting Columns) or dialysis (e.g., Thermo Scientific™ Slide-A-Lyzer™ Dialysis Cassettes). Samples containing 6M guanidine•HCl must be dialyzed against a buffer containing 8 M urea before SDS-PAGE analysis. The Thermo Scientific™ Pierce™ SDS-PAGE Sample Prep Kit (Product No. 89888) may also be used to remove guanidine.</P><H4>Procedure for Ni-NTA Resin Regeneration</H4><P>The Ni-NTA resin may be used at least five times without affecting protein yield or purity. Between each use, perform the procedure as described below to remove residual imidazole and any nonspecifically adsorbed protein. To prevent cross-contamination of samples, designate a given column to one specific fusion protein. </P><OL><LI>Wash resin with 10 resin-bed volumes of MES Buffer. </LI><LI>Wash resin with 10 resin-bed volumes of ultrapure water.</LI><LI>Store resin as a 50% slurry in 20% ethanol.</LI></OL><H2>LB Agar</H2><P>Written up by Lexi Petrucciani</P><H4>Purpose</H4><P>This protocol can be used to make more growth media for E. Coli, whether selective or not.</P><H4>Materials(1 Liter ~30 plates)</H4><UL><LI>With LB agar powder: 30.5 g LB agar powder</LI><LI>Else: 10 g NaCl;1 0 g Tryptone; 5 g Yeast Extract; 15 g Agar; 1 L ddH20</LI><LI>Optional antibiotic: Chlor: 25 ug/mL(25 mg/L) or Amp</LI><LI>sterile petri dishes</LI></UL><H4>Equipment</H4><UL><LI>List all the equipment needed (using bullet points) to complete this protocol, including what lab # in Bindley or another building it can be found in.</LI></UL><H4>Protocol</H4><OL><LI>Mix LB agar powder(or replacement mixture) with ddH2O in flask
</LI><LI>Flask should have a capacity of at least two time the total volume as not to boil over in the autoclave.</LI></OL><LI>Swirl mixture until no clumps of powder remain.Be sure to check the bottom!
</LI><LI>Can microwave the mixture to help dissolve powder </LI><LI>Autoclave mixture for liquid30 cycle</LI><LI>Add optional antibiotic</LI><LI>Pour Plates</LI><LI>Store inverted at 4 degrees C</LI><H4>Discussion</H4><P>Don’t let the agar cool too long after the autoclaving process. The agar will start to solidify, it will be harder to pour plates, and the plates will have more bubbles.</P><H4>Source</H4><H4><P>https://www.addgene.org/plasmid-protocols/bacterial-plates/
 and side of LB agar powder</P></H4><H4>Paragraph Form</H4><P>Mix 30.5 g LB agar powder and 1 L double distilled water in a flask with a capacity at least 2 times that of the total volume(2 L in this case) such that it will not boil over in the autoclave. Swirl mixture until thoroughly mixed. Autoclave mixture for the liquid30 cycle. When the flask is cool enough to hold add antibiotic, pour into plates, and let cool. Store inverted at 4 degrees C. Note that the agar powder can be replaced with a mixture of 10 g NaCl, 10 g Tryptone, 5 g Yeast Extract, and 15 g Agar.</P><H2>Ligation Protocol</H2><P>written by Bowman Clark</P><H4>Purpose</H4><P>Protocol for assembling biological parts.</P><H4>Materials</H4><UL><LI>25 ng of digested plasmid backbone </LI><LI>equimolar amount of EcoRI-HF SpeI digested fragment (4 ul) </LI><LI>equimolar amount of XbaI PstI digested fragment (4 ul)</LI><LI>2 ul T4 DNA ligase buffer. Note: Do NOT use quick ligase</LI><LI>1 ul T4 DNA ligase</LI><LI>water, 20 ul</LI><LI>1 tube per ligation</LI></UL>
Equipment
<UL><LI>Thermocycler (BIND 222)</LI><LI>Mini centrifuge (BIND 134, B025)</LI></UL>
Protocol
<OL><LI>Determined desired ligation ratio (usually 3:1)</LI><LI>Calculate necessary volumes using the NEB ligation calculator</LI><LI>Place all tubes in ice. </LI><LI>Add 25 ng of digested plasmid backbone </LI><LI>Add equimolar amount of EcoRI-HF SpeI digested fragment (&lt; 3 ul). Volume determined by NEB calculator and desired ratio.</LI><LI>Add equimolar amount of XbaI PstI digested fragment (&lt; 3 ul). Volume determined by NEB calculator and desired ratio.</LI><LI>Add 2 ul T4 DNA ligase buffer. Note: Do NOT use quick ligase</LI><LI>Add 1 ul T4 DNA ligase</LI><LI>Add water to 20 ul </LI><LI>Flick tube to mix. Vortex gently if necessary. </LI><LI>Place in centrifuge to collect reaction at bottom of tube (usually at 3000 rpm for 30 sec)</LI><LI>Place in thermocycler: 
</LI><LI>Set cycle for temp at 16C for 16 hours to ligate </LI><LI>Temp to 65C for 20 min to heat kill ligation enzymes</LI><LI>Chill on ice</LI></OL><LI>Transform with 1-5 ul of product or store at -4 for short term storage/-20C for long term storage</LI><H2>Tris Buffer with P (Na2HPO4) Protocol</H2><P>Written up by Bowman Clark and Paige Rudin</P><P>Estimated Time: 90 minutes</P><H4>Purpose</H4><P>To prepare 250 mL minimal media necessary for suspending cells while they uptake phosphorus from solution (phosphorus is in the form of orthophosphate [PO43-] from Na2HPO4)</P><H4>Materials</H4><H4><UL><LI>19 mL concentrated HCl (approx. 12.1 M)</LI><LI>250 mL ddH2O</LI><LI>pH strips</LI><LI>0.0378 mg sodium phosphate dibasic, ACS reagent (Na2HPO4)</LI><LI>30.275 g TRIS base powder</LI><LI>7.5 mg KCl</LI><LI>7.5 mg MgCl2</LI><LI>7.5 mg NaCl</LI><LI>7.5 mg NH4Cl</LI><LI>7.5 mg FeCl3</LI><LI>7.5 mg CaCl2 </LI></UL></H4><H4>Equipment</H4><UL><LI>Automated pH meter (if grad student-approved)</LI><LI>1 L flask</LI><LI>Autoclave (1st floor)</LI></UL><H4>Protocol</H4><OL><LI>Add 30.275 g TRIS base and 7.5 mg of KCl, MgCl2, NaCl, NH4Cl, FeCl3, and CaCl2 to 200 mL ddH2O; mix until dissolved</LI><LI>For a pH of 7.6, add 16-19 mL (?) HCl
<OL type="a">pH of 7.6 desired because when Na2HPO4 is added, pH will lower; we want in the range of 7.0-7.4</OL></LI><LI>Use pH strips to verify correct pH OR ask a grad student to use pH meter</LI></OL><LI>Add 0.0378 mg Na2HPO4 to mixture; mix until dissolved</LI><LI>Double-check pH to make sure in range of 7.0-7.4; record</LI><LI>Add more ddH2O, bringing final volume of solution to 250 mL</LI><LI>Seal bottle with cap and autoclave tape and run on liquid30 cycle</LI><H4>Paragraph Form</H4><P>Add 121.1 g TRIS base to 800 mL ddH2O, and mix until TRIS is dissolved. Add 65 mL HCl to achieve a desired mixture pH of 7.6. Either use pH test strips or the automated pH meter to verify success. Adjust as necessary, but beware that pH will drop with the next step. Add 0.151 mg Na2HPO4 and mix until dissolved. Re-check the pH to ensure that it is in the range of 7.0-7.4. Add more HCl if necessary. Add more ddH2O, bringing the final volume to 1 L. Seal the bottle with its cap and autoclave tape and put it in the autoclave for a liquid30 cycle (approx. 70 minutes).</P><H4>Source</H4><P>https://www.neb.com/protocols/1/01/01/protocol-ii-1-m-tris-hcl-buffer-stock-solution-1-liter </P><H2>Miniprep Protocol</H2><P>Erich Leazer</P><H4>Purpose:</H4><P>Isolate and extract genomic DNA from transformed colonies.</P><H4>Materials</H4><UL><LI>LyseBlue</LI><LI>Buffer P1</LI><LI>Buffer P2</LI><LI>Buffer EB</LI><LI>Buffer NE3</LI></UL><H4>Equipment:</H4><UL><LI>Centrifuge</LI><LI>Micropipette and Tips</LI><LI>Eppendorf Tubes</LI><LI>Qiagen Spin Columns</LI></UL><H4>Protocol:</H4><OL><LI>Add LyseBlue to Buffer P1 at a ratio of 1:1000</LI><LI>Warm up EB in water bath to 50 C</LI><OL type="a"><LI>place EB Buffer in Styrofoam holder to float in water bath</LI></OL><LI>Pellet 1-5mL of bacterial overnight culture by centrifugation at &gt;8000 rpm for 3 minutes at room temperature.
</LI><LI>Add 1mL of each liquid cell culture to each eppendorf tube
</LI><LI>Place eppendorf tubes in centrifuge (balance)</LI><LI>Centrifuge for 3 minutes at 10,000 rpm</LI><LI>Pour out liquid into sink (pellet should remain at the bottom)</LI><LI>Continue process, add liquid cell culture to eppendorf tube with pellet until all of the cells have been pelleted</LI></OL><LI>Resuspend pelleted cells in 250uL Buffer P1 and transfer to a micro-centrifuge tube (Buffer P1 lyses the cells)
</LI><LI>Can vortex or tap tube on counter to resuspend the cells (must not be stuck to the bottom)</LI><LI>Put Buffer P1 back into 4 C fridge.</LI><LI>Add 250uL Buffer P2 and mix thoroughly by interverting the tube for 3 minutes (Buffer P2 also lyses the cells)</LI><LI><B>Promptly</B> add  350 uL Buffer NE3 and mix by inverting the tube for one minute (this neutralizes the lysing buffers so it doesn’t damage the DNA)
</LI><LI>Should not be ‘gloopy’ (?)</LI><LI>Centrifuge for 10 minutes at 13,000 rpm. Supernatant will contain the DNA, and the cells will collect at the bottom of the tube.</LI><LI>Apply supernatant from step 7 to the Qiagen SPin Column. Centrifuge for 60 seconds and discard the flow-through (separates DNA from liquid in the supernatant)</LI><LI>Add 750uL Buffer PE and centrifuge for 60 seconds and discard flow-through (Buffer PE prevents DNA from dissociating from the column while washing away contaminants)</LI><LI>Repeat step 9</LI><LI>Centrifuge for 2 minutes to remove residual wash buffer</LI><LI>Transfer to a new microcentrifuge tube</LI><LI>Add 25uL Buffer EB, let it stand for 5 minutes, centrifuge for 60 seconds. (Buffer EB solubilizes the DNA so that it can flow through the spin column).</LI><LI>Repeat Step 13.</LI><LI>Measure DNA concentrations with the Nanodrop (bring a blank of the EB Buffer)</LI><LI>When finished, store DNA in the -20 C freezer.</LI><H4>Paragraph Form:</H4><P>In order to isolate and extract genomic DNA from transformed colonies, use a miniprep kit. First, add LyseBlue to Buffer P1 at a ratio of 1:1000. Then warm EB buffer up in a water bath to 50 C. Pellet the bacterial culture by centrifugation at &gt;8000 rpm for 3 min at room temperature. Next, extract 1 mL of the supernatant, add it to each Eppendorf tube, and centrifuge again for 3 minutes at 10,000 rpm. The liquid should be discarded and then liquid cell culture should be added to the Eppendorf tube with the remaining pellet. Resuspend the pelleted cells in 250µL Buffer P1 and transfer to a micro-centrifuge tube to lyse the cells. Next, add 250 µL Buffer P2 and mix thoroughly to complete the lysing process. To neutralize the lysing buffers and prevent DNA damage, promptly add 350 µL Buffer NE3. Then, centrifuge the mixture for 10 minutes at 13,000 rpm. This step results in a cell pellet at the bottom of the tube and a supernatant containing the desired DNA. Add the supernatant to the Qiagen Spin Column and centrifuge for one minute.</P><P>Discard the flow-through. Next, add 750 µL Buffer PE, centrifuge for 1 minute, and then discard the flow-through. Buffer PE prevents DNA from dissociating from the column while washing away contaminants. Repeat the previous step and centrifuge for 2 minutes to remove residual wash buffer. Transfer the resulting mixture to a new micro-centrifuge tube and add 25 µL Buffer EB. Allow it to sit for 5 minutes, then centrifuge for 60 seconds. Buffer EB solubilizes the DNA so that it can flow through the spin column. Repeat this step again then measure final DNA concentrations with a Nanodrop machine, using EB Buffer as a blank. Store the purified plasmids in the -20 C freezer.</P><H2>Neisser Stain</H2><P>Written up by Emma Foster and Paige Rudin</P><H4>Purpose</H4><P>Neisser staining is useful for determining the presence of polyphosphate in bacteria, indicating successful accumulation. Those bacteria that accumulate polyphosphate are Neisser (+) and stain bluish purple. Neisser (-) bacteria stain brown. This protocol was retrieved from a document by the Iowa Rural Water Association entitled &quot;Filamentious Bacteria Identification &amp; Process Control: A Simple Approach&quot; [http://www.iowaruralwater.org/tools_tips/toni_glymp/Filaments.pdf]. </P><H4>Materials</H4><UL><LI>Solution 1
</LI><LI>Part A
</LI><LI>Methlylene Blue: 0.1 g</LI><LI>Ethanol, 95%: 5 mL</LI><LI>Acetic acid, glacial: 5 mL</LI><LI>Distilled water: 100 mL</LI></UL><LI>Part B
</LI><LI>Crystal Violet (10% w/v in 95% ethanol): 3.3 mL (0.333 g in 3.3 mL)</LI><LI>Ethanol, 95%: 6.7 mL</LI><LI>Distilled water: 100 mL</LI><LI>Mix 2 parts by volume of A with 1 part by volume of B; prepare fresh monthly.</LI><LI>Solution 2</LI><UL><LI>Bismark Brown (1% w/v aqueous): 33.3 mL (0.333 g in 33.3 mL)</LI><LI>Distilled water: 66.7 mL</LI></UL><LI>Microscope slide</LI><H4>Equipment</H4><UL><LI>Microscope (EVOS)</LI></UL><H4>Protocol</H4><OL>Prepare a smear and dry; heat-fix bacteria to slide
To heat-fix (http://www.microbehunter.com/fixing-specimens-for-making-permanent-slides/):
<OL type="a"><LI>Drop small amount of culture on slide (10 uL)</LI><LI>Let it air dry in sanitary conditions (i.e. in a hood or near a flame) do NOT use heat to dry the smear because cells might pop</LI><LI>Once dry, pass slide through a flame 1-2 times to fix cells to the glass slide</LI><LI>Ensure the cells do not come into contact with the flame, only the underside of the slide (slide should be just hot enough that you can hold it in your hands without causing burns)</LI></OL><LI>Completely cover the smear with Solution 1 for 30 seconds and rinse for 1 sec with ddH2O</LI><LI>Completely cover the smear with Solution 2 for 1 minute, rinse well with ddH2O and blot dry.</LI><LI>Examine under oil immersion using the 100X objective with brightfield illumination (do not use phase contrast)</LI><OL type="a"><LI>If using EVOS, do not use any fluorescent illumination--natural light will work</LI>Capture images for comparison</OL></OL><H4>Paragraph Form</H4><P>Prepare a smear and dry </P><H4>Source</H4><P>http://www.iowaruralwater.org/tools_tips/toni_glymp/Filaments.pdf </P><H2>Phosphorus Uptake Measurement Protocol</H2><P>Written up by Barrett Davis, Emma Foster, and Paige Rudin</P><P>Estimated Time: 18 hours </P><H4>Purpose</H4><P>To prepare E. coli cultures for phosphorus uptake characterization</P><H4>Materials</H4><UL><LI>Flasks (1 x Number of Genes) </LI><LI>LB Broth (1L per Gene) </LI><LI>Phosphate Rich Media (1L per Gene)</LI><LI>96-well plate </LI><LI>Falcon tubes (2 x Number of Genes) </LI></UL><H4>Equipment</H4><UL><LI>Lachat</LI><LI>ICP</LI><LI>Plate Reader</LI><LI>Spectrophotometer</LI></UL><H4>Protocol</H4><OL><LI>Grow up cells in LB Broth with shaking at 37°C for 12h (25 mL)</LI><LI>At 9h, measure OD600 and dilute culture to 0.02
</LI><LI>Use the plate reader to do this</LI><LI>Pipette 100 uL into a well; repeat for a total of 2 wells (wells will be averaged)</LI><LI>Take plate to reader upstairs; use plate reader
</LI><LI>Data returned is contained within an Excel file</LI><LI>Manipulate data using formulas below--spreadsheet “P uptake dilution”
Use the following equation to make 500 mL  (0.5 L) of diluted culture: 
 	Ac = (0.02 * 0.5L)/(x-c)
	APm = 0.5L -Ac
Ac + APm =	0.5L total culture
Variables: Ac= amount of overnight culture, APm= Amount of phosphate media, x= measured OD600 of overnight culture, c= OD600 of pure LB broth </LI></OL><LI>Remove 2x 5 mL of culture at the 0h mark and transfer to two 15 mL falcon tubes and remove 500 uL to an eppendorf tube 
</LI><LI>Store epp tube directly at -20°C</LI><LI>Centrifuge to separate liquid and cells in falcon tubes</LI><LI>Transfer liquid into separate 15 mL falcon tube (2)</LI><LI>Store both the falcon tubes with the liquid and with the centrifuged cells at -20°C until testing</LI><LI>Do this as quickly as possible to minimize growth and/or uptake of phosphorus </LI><LI>The samples for the ICP need little prepping, leave them on ice until ready for measuring
</LI><LI>Re-suspend one pellet in 5 mL ddH2O or appropriate volume for ICP</LI><LI>The samples going to the Lachat need to be prepped in the following manner:
</LI><LI>Keep everything cold </LI><LI>Taking the remaining pellet, dissolve cells in 500 uL ddH2O, transfer to an epp tube (if convenient), and centrifuge at 2350 g for 5 minutes; discard supernatant</LI><LI>Add 600 uL ddH2O and mix</LI><LI>Sonicate samples for 5 mins at 30 Hz (cycle 0.5, amplitude 65%)</LI><LI>Place tubes containing samples in boiling water bath at 100 deg. C; boil for 2 h</LI><LI>After boiling, cool tubes at room temperature</LI><LI>Add 600 uL of 24:1 (v/v) chloroform: isoamyl alcohol solution; mix by vigorous shaking</LI><LI>Centrifuge at 13,520 g for 15 minutes at room temperature</LI><LI>Collect the upper aqueous phase; put in 15 mL Falcon tube with 4.5 mL H2O</LI><LI>Taking the 500 uL sample, use the plate reader to find the optical cell density</LI><LI>Repeat step 3-5 every hour for 6 total hours (Cultures for: 0h, 1h, 2h, 3h, 4h, 5h, 6h) </LI><LI>Check how many cells are uptaking phosphorus with Neisser stain at the 6h</LI><H4>Paragraph Form</H4><P>Grow up cells of choice in LB broth with shaking at 37°C for 9h. After 9h incubation period, measure OD600 and dilute as necessary to a measurement of 0.02 relative units. Then </P><H4>Source</H4><P>http://aem.asm.org/content/60/10/3485.full.pdf</P><H2>PCR Preparation</H2><P>Written up by Mark Aronson</P><H4>Purpose</H4><P>PCR amplification makes more copies of a segment of DNA.</P><H4>Materials</H4><UL><LI>5X Phusion HF Buffer</LI><LI>10 mM dNTPs</LI><LI>10 μM Forward primer</LI><LI>10 μM Reverse primer</LI><LI>50 mM MgCl2</LI><LI>Phusion DNA Polymerase</LI><LI>1 pg - 10 ng template DNA</LI><LI>nuclease free water</LI></UL><H4>Equipment</H4><UL><LI>Thermocycler (BIND 222)</LI><LI>PCR Tubes (1 per reaction)</LI><LI>Nanopure Water Machine (BIND 134 balance room or BIND 222)</LI><LI>Sterile technique equipment </LI></UL><H4>Protocol</H4><OL><LI>Calculate amount of all liquids to be added to the tube to make a 50μL reaction</LI><LI>Add the calculated amount of nuclease free water to a PCR tube (for 1μL of 1ng/μL DNA, this is 32μL of water)</LI><LI>Add 10μL of 5X Phusion HF (or GC) buffer</LI><LI>Add 1μL of 10mM dNTPs</LI><LI>Add 0.5μL of 50mM MgCl2</LI><LI>Add 2.5μL of 10μM forward primer</LI><LI>Add 2.5μL of 10μM reverse primer</LI><LI>Add the given volume of DNA (which contains 1pg-10ng of the template DNA)</LI><LI>Add 0.5μL of Phusion DNA Polymerase</LI><LI>Place the PCR tubes in a microcentrifuge tube and microcentrifuge to collect everything in the bottom of the PCR tubes</LI><LI>Set the thermocycler (make sure you are using the correct block!) to the following setting for a 50μL reaction
<TABLE><TBODY><TR><TD>98°C</TD><TD>30 s</TD></TR><TR><TD>25 cycles</TD></TR><TR><TD>98°C</TD><TD> 10 s</TD></TR><TR><TD>Tm°C </TD><TD> 30 s</TD></TR><TR><TD>72°C</TD><TD>30 s</TD></TR><TR><TD>72°C </TD><TD> 5 min</TD></TR><TR><TD>4°C</TD><TD>forever</TD></TR></TBODY></TABLE></LI><LI>Retrieve tubes from thermocycler and place in -20°C freezer</LI></OL><H4>Discussion</H4><UL><LI>Resuspend your primer oligos to 100 uM then make a 10 uM working stock in a microfuge tube for PCR</LI><LI>After your first PCR, run some of the sample on an agarose gel to check your products. If you have just your desired band, you can clean up the rest of your sample with the Qiagen PCR cleanup kit. If you have multiple bands, you might have to optimize the PCR protocol or gel extract the correct band or both.</LI><LI>After you figure out the optimal PCR conditions, you can do more than one 50 uL reaction to keep as stock</LI><LI>Don't use all of the original G Block stock. If sequences don't match up when you check your sequencing, it may have been because of an error in PCR because even the high fidelity polymerase isn't always 100% </LI></UL><H4>Source</H4><P>Sam Lee and IDT (https://www.idtdna.com/pages/docs/default-source/user-guides-and-protocols/gblocks-amplification.pdf?sfvrsn=12)</P><H4>Paragraph Form</H4><P>For a 50μL reaction, calculate the volume of DNA that will be added (recommend using 1μL of 1ng/μL DNA solution). Using this and the other given volumes, calculate amount of nuclease free water to be added (for 1μL of DNA solution this is 32μL of nuclease free water). Add this amount of water to a PCR tube. Add 10μL of 5X Phusion HF (or GC) buffer. Add 1μL of 10mM dNTPs. Add 2.5μL of 10μM forward primer. Add 2.5μL of 10μM reverse primer. Add 0.5μL of 50mM MgCl2. Add the given volume of DNA (which contains 1pg-10ng of the template DNA). Finally, add 0.5μL of Phusion DNA Polymerase. Set the thermocycler to the following settings: 98°C for 30 seconds, then 25 cycles of 98°C for 10s, TM- 5°C for 30s, 72°C for 30s, and end with 72°C for 5 min and then 4°C forever. Place PCR tubes in thermocycler and start cycle. When completed, retrieve PCR tubes and place in -20°C for storage. </P><H2>PCR Purification</H2><P>Written up by Mark Aronson</P><H4>Purpose</H4><P>Used to purify target DNA after PCR amplification</P><H4>Materials</H4><UL><LI>Buffer PB</LI><LI>2mL collection tubes</LI><LI>Spin columns (1 per reaction)</LI><LI>Buffer PE</LI><LI>1.5mL microcentrifuge tubes (1 per reaction)</LI><LI>Elution buffer (or Buffer EB or water)</LI></UL>

Equipment
<UL><LI>Centrifuge (BIND 134)</LI></UL>

Protocol
<OL><LI>Add Buffer PB to PCR reaction (5:1 volume ratio) and mix
</LI><LI>if color is orange or violet, add 10μL 3M sodium acetate (pH 5.0) and mix</LI><LI>the color of the mix should be yellow</LI></OL><LI>Place a QIAquick column in a 2mL collection tube</LI><LI>Apply sample to column and centrifuge for 30-60s (room temp, 13000rpm)</LI><LI>Discard flow-through and place spin column back in tube</LI><LI>Add 750μL of Buffer PE to column</LI><LI>Centrifuge for 30-60s (room temp, 13000rpm)</LI><LI>Discard flow-through and place spin column back in tube</LI><LI>Centrifuge again for 1min (room temp, 13000rpm) to remove residual wash buffer</LI><LI>Place spin column in a clean 1.5mL microcentrifuge tube</LI><LI>Add 30μL elution buffer to center of spin column membrane, let stand for 1min, then centrifuge for 1min (room temp, 13000rpm)
</LI><LI>Alternatively, use 50μL Buffer EB (10 mM Tris-Cl, pH 8.5)</LI><LI>Alternatively, use 50μL water (pH 7.0-8.5)</LI><H4>Discussion</H4><UL><LI>This protocol is for the purification of up to 10μg PCR products (100bp-10,000bp)</LI><LI>Add ethanol (96-100%) to Buffer PE before use</LI><LI>Add 1:250 volume pH indicator I to Buffer PB</LI></UL><H4>Source</H4><P>https://www.qiagen.com/us/resources/resourcedetail?id=390a728a-e6fc-43f7-bf59-b12091cc4380&amp;lang=en</P><H4>Paragraph Form</H4><P>Add Buffer PB to PCR reaction in a 5:1 volume ratio. Mixture should turn yellow. If mixture is orange or violet, add 10μL 3M sodium acetate (pH 5.0) and mix. Place spin column in a 2mL collection tube. Apply sample to column and centrifuge for 30-60s (room temp, 13000rpm). Discard flow-through and place spin column back in collection tube. Add 750μL of Buffer PE to column and centrifuge for 30-60s (room temp, 13000rpm). Discard flow-through and place column back in tube. Centrifuge dry column again for 1min (room temp, 13000rpm) to remove residual wash buffer. Place spin column in a clean 1.5mL microcentrifuge tube. Add 30μL elution buffer to center of spin column membrane, let stand for 1min, then centrifuge for 1min (room temp, 13000rpm). It is also possible to use 50μL Buffer EB (10 mM Tris-Cl, pH 8.5) or Alternatively, use 50μL water (pH 7.0-8.5) at this step. Collected solution at the bottom of the tube is the purified DNA. </P><H2>Sol Gel Beads</H2><P>Creator: Jenna Rickus</P><P>Date Originated: 10-13- 2003</P><H4>Description:</H4><P>The following is a procedure for making bio sol-gel from TMOS. Typically used for protein encapsulation.</P><H4>Materials</H4><UL><LI>Tetramethoxysilane (aka tetraorthosilicate, TMOS)</LI><LI>0.04 N HCl</LI><LI>Water</LI><LI>Buffer (0.02M Sodium Phosphate, 0.02M NaCl, pH 7.0)</LI></UL><H4>Procedure</H4><OL><LI>Combine in a 15 mL corning tube:
</LI><LI>3.8 mL TMOS</LI><LI>0.85 mL (850 μL) Water</LI><LI>55 μL 0.04N HCl</LI>
You should see 2 clear immiscible liquids<LI>Sonicate Mixture for 15 minutes in a water bath sonicator.</LI><LI>If the water in the bath is too warm, add ice.</LI><LI>The solution should be homogenous after sonication.</LI><LI>If cloudy, filter sonicated mixture using a 0.2 μm Whatman filter &amp; a 5 mL syringe.</LI><LI>Combine filtered TMOS Sol with Buffer in a 20% : 80 % ratio by volume and dispense in 0.5 ml increments, e.g. 100 uL TMOS Sol + 400uL Buffer. The solution will gel in ~ 30 seconds.</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>