Team:Purdue/Organisms
HOME
TEAM
BACKGROUND
Problem
Phosphorous
Wastewater
Organisms
Protein Profiles
PROJECT
★ Description
★ Design
★ Model
Hardware/ Prototypes
★ Experiments
★ Proof of Concept
★ Results
Side Project
PARTS
★ Parts
★ Basic Parts
★ Composite Parts
★ Part Collection
HUMAN PRACTICES
★ Human Practices
★ Silver
★ Gold
★ Integrated Practices
★ Engagement
★ Collaborations
SYNENERGENE
NOTES FROM THE LAB
★ Notes from the Lab
★ Notebook
★ Protocols
InterLab Study
Safety
ATTRIBUTIONS
Purdue Biomakers
Twitter
Facebook
Website
Previous Page
Next Page
Why we chose E. coli
Escherichia coli (E. coli) is a gram-negative bacteria of the genus Escherichia. It is classified as a facultative anaerobe due to its ability to switch over to ethanol fermentation process when little to no oxygen is present in the environment. While most strains of E.coli reside within mammalian intestines (and as such have developed a controversial reputation for the diseases they can cause), laboratory strains (such as K-12 derivative MG1655) neither survive in intestinal settings nor form problematic biofilms [1]. In fact, these strains are favored by most researchers because their rapid growth rates, low purchase and maintenance costs, relationship to most by bacterial species by approximately 20%, and versatility in plasmid based protein production or genetic recombination techniques. Since the complete sequencing of the genome in 1997 [4], E. coli has played a key role in the research of synthetic metabolic pathways, gene regulation mechanisms of prokaryotes, and production of high value chemicals and proteins [2].
Figure 1: Colorized scanning electron micrograph of Escherichia coli, grown in culture and adhered to a cover slip. [5].
We decided E.coli would work best as our chassis because we thought that the large body of research on its metabolism would makes it preferable to the relatively unknown microbes currently used for phosphorus accumulation in biological wastewater treatment.
Why we used Microlunatus phosphovorus
Microlunatus phosphovorus (M. phosphovorus) is a gram-positive, aerobic, coccus-shaped, actinobacteria of the relatively new genus Microlunatus. First isolated in 1995 from activated sludge, M. phosphovorus showed extraordinarily high phosphorus removal from wastewater (up to 48% of dry weight) when compared to other phosphorus accumulating organisms (PAOs) reported in literature and significant promise for application in enhanced biological phosphorus removal (EBPR). After its genome was sequenced in 2012, Kawakoshi et. al confirmed the presence of “four polyP kinases (ppks), two polyP-dependent glucokinases (ppgks), and three phosphate transporters (pits)” responsible for polyP metabolism in M. phosphovorus -- several more genes than reported in any other known actinobacterial PAOs. Interestingly, Kawakoshi et. al also discovered that as a PAO, M. phosphovorus only expresses one exopolyphosphatase gene rather than the two that are typically observed. In expressing only a single copy of the exopolyphosphatase gene (responsible for the hydrolysis of polyphosphate) it is believed that M. phosphovorus is better able to prevent the export of phosphorus in the form of orthophosphates from the cell.
Figure 2: Dividing M. Phosphovorus NM-1 cell as viewed under SEM
M. phosphovorus appears to accumulate phosphorus in a manner similar to proteobacterial PAOs in aerobic conditions. However, during anaerobic conditions, rather than synthesizing polyhydroxyalkanoate carbon reserves to later be oxidized to produce energy for growth and polyphosphate accumulation in aerobic conditions, M. phosphovorus instead ferments glucose to to provide instantaneous energy phosphorus accumulation without the need for a reoxygenation of its growth media.
References
[1] C. A. Fux, M. Shirtliff, P. Stoodley, and J. W. Costerton, "Can laboratory reference strains mirror ‘real-world’ pathogenesis?," Trends in Microbiology, vol. 13, no. 2, pp. 58–63, Feb. 2005. [Online]. Available: http://www.sciencedirect.com/science/article/pii/S0966842X04002550. Accessed: Jun. 2, 2016.
[2] D. Barr, "Research guides: Lab rats and more: Model organisms in research today: E. Coli," inHarvard Research Guides, 2011. [Online]. Available: http://guides.library.harvard.edu/c.php?g=310307&p=2071641#s-lg-box-6324388. Accessed: Jun. 2, 2016.
[3] Healthline,. Escherichia Coli. 2015. Web. 2 June 2016.
[4] F. R. Blattner et al., "The complete genome sequence of Escherichia coli K-12," Articles, vol. 277, no. 5331, pp. 1453–1462, Sep. 1997. [Online]. Available: http://science.sciencemag.org/content/277/5331/1453. Accessed: Jun. 2, 2016.
[5] https://www.flickr.com/photos/niaid/16578744517/
