Team:Purdue/Experiments
HOME
TEAM
BACKGROUND
Problem
Phosphorous
Wastewater
Organisms
Protein Profiles
PROJECT
★ Description
★ Design
★ Model
Hardware/ Prototypes
★ Experiments
★ Proof of Concept
★ Results
Side Project
PARTS
★ Parts
★ Basic Parts
★ Composite Parts
★ Part Collection
HUMAN PRACTICES
★ Human Practices
★ Silver
★ Gold
★ Integrated Practices
★ Engagement
★ Collaborations
SYNENERGENE
NOTES FROM THE LAB
★ Notes from the Lab
★ Notebook
★ Protocols
InterLab Study
Safety
ATTRIBUTIONS
Purdue Biomakers
Twitter
Facebook
Website
Experimentation
Gene Design
When designing our IDT gBlocks, we added the standard iGEM prefix between the relatively weak constitutive Anderson promoter BBa_J23105 and the start of our coding region to allow easy exchange of promoters, which allows us to use promoters of different strengths for protein characterization and our final construct. We fused a 10x polyhistidine tag (BBa_K844000) to the c-terminus of all proteins except PPK2 homolog A, which was fused with a 6x polyhistidine tag to the n-terminus due to synthesis complexities, in order to allow protein detection and purification. The standard iGEM suffix was added between the coding region and the terminator BBa_B0010 to allow for easy exchange of terminators or use of the plasmid’s terminator. Hind III restriction sites could be used to insert the entire promoter to terminator construct, as synthesized by IDT, into a plasmid.
As PPK1 is a relatively long gene, the genetic construct for PPK1 was divided into two gBlocks, and Gibson assembly will be used to join the two pieces together.
Our first goal this summer was to characterize ten genes that would enable the uptake, accumulation, and release of phosphorous in our engineered E. coli.
two inorganic phosphate transport (Pit) genes
four polyphosphate kinase (PPK) genes, which catalyze the transfer of phosphate groups from polyphosphate chains to nucleoside phosphates
two polyphosphate-dependent glucokinase (PPGK) genes, which catalyze the phosphorylation of glucose using the terminal phosphate of a polyphosphate chain
one exopolyphosphate (PPX) gene, which catalyzes the hydrolysis of the terminal phosphate of a polyphosphate chain
Only one of of these genes (a PPGK) has been characterized before. To characterize, we are using a medium strength constitutive Anderson promoter (BBa_J23106) and the terminator present on the plasmid PSB1C3.
Transformations of E. coli
After receiving our phosphorus-related genes from IDT, we amplified the amount of DNA using polymerase chain reactions, and then used gel purification to obtain the amplified product. Then, we performed 3A assembly in order to ligate the constitutive Anderson promoter BBa_J23106 and each of our ten individual phosphorus-related genes into the backbone PSB1C3. The ligated product was then transformed into DH5-alpha E. coli.
The resulting colonies were then inoculated into LB broth and mini-prepped in order to obtain the plasmid DNA. The plasmid was digested to separate the insert and the backbone. Gel electrophoresis was performed on the digested product in order to determine the length of any inserts. The plasmids were then taken to the Purdue Genomics Core Facility for low throughput Sanger sequencing.
Verification of Protein Expression
To use Immoilized Metal Affinity Chromotagraphy (IMAC) for protein purification, sequences for polyhistidine tags were added to an end of coding regions for each of our ten phosphorus-related genes. In IMAC [1], the desired metal, in this case nickel, is immobilized in a nitrilotriacetic acid and agarose resin. The protein sample is then applied to the resin in a neutral buffer, which can be used in spin columns, to promote binding of the his-tagged proteins to the nickel. The protein can be eluted by using high concentrations of imidazole.
Once purified proteins samples for our modified and unmodified E. coli are collected, the concentration of protein samples will be be analyzed using a Nanodrop under the A280 setting. If concentrations are high enough, the samples will be run through a sodium dodecyl sulfate polyacrylamide gel (SDS PAGE) in order to confirm the presence of his-tagged proteins of the correct molecular weight. We have not achieved high enough concentrations to justify performing the SDS PAGE.
Analysis of Phosphorus Uptake, Accumulation, and Exportation
Phosphorus Uptake and Export Assay
Cultures of all samples were grown up overnight in LB-broth T 37°C. The optical density at 600 nm was obtained for all samples, and then all samples were diluted into minimal media that contains a phosphate source to produce 250 ml of minimal media with an optical density of 0.01 at 600 nm. At each hour, starting at hour 0, for 6 hours, two 5 ml samples and one 500 µl were collected and then frozen at -20 °C. After sample collection, all 5 ml samples were centrifuged to separate the cellular component and supernatant. The supernatant was transferred to a second tube.
Our 5 ml samples were then to be tested using equipment from the United States Department of Agriculture National Soil Erosion Research Lab.
One sample (both the supernatant and cellular component) from each hour was destined for a machine known as an Inductively Coupled Plasma (ICP) optical emission spectrometer, which detects total phosphorus content. [2] To test the total phosphorus component of the cellular component, the cells were resuspended in 5 ml of double distilled water. The second sample (both supernatant and cellular component) from each hour was tested in a Lachat machine to measure orthophosphate. The orthophosphate is measured by reacting it with ammonium molybdate and antimony potassium tartrate, and then reducing the complex to form a blue complex, which is quantified spectrophotometrically. [3] To test the cellular component’s orthophosphate concentration, polyphosphate granules were extracted from the second sample’s cellular component by suspending the cells in 0.6 ml of double distilled water, sonicating and boiling the samples, and adding 0.6 ml of a 24:1 chloroform: isoamyl alcohol solution. We then collected the aqueous phase and added 4.5 ml of double distilled water.
Due to difficulties in using or generating a standardized matrix and the machine’s ability to measure only orthophosphate, and not polyphosphate, we later decided that just using the ICP spectrometer would be sufficient.
The phosphate concentration of the supernatant indicates whether the E. coli is uptaking phosphate (removing phosphate from supernatant) or releasing phosphate (adding phosphate to supernatant).
The optical density at 600 nm of every sample for all hours was obtained in order to verify that differences in phosphorus concentrations in both the supernatant and cellular component are likely not due to differences in cell count.
Polyphosphate Accumulation Assay
We also wanted to quantify how much polyphosphate was present in the polyphosphate granules in the cell. To do this, we used a colorimetric assay that utilizes toluidine blue and acetic acid, in which the absorbance at 630 nm decreases with increases in polyphosphate concentration.
Before we could quantify how much polyphosphate was present in our cells, we constructed a standard curve using the chemically-synthesized polyphosphate sodium phosphate glass type 45. Fifteen samples were produced in triplicate, and all samples had concentrations varying from 0 µg phosphate per µl solution to 0.05 µg phosphate per µl of solution. Three ml each of toluidine blue solution and 0.2 N acetic acid was added to each sample. After an incubation period, the absorbance at 630 nm was read using a plate reader, and three readings were averaged, and a standard curve was constructed.
Samples of all cells were grown up in either LB-broth or LB-broth with the appropriate antibiotic, both of which were supplement with potassium phosphate to obtain a 2 mM  phosphate concentration, which is deemed a sufficient amount of phosphate for E. coli growth [4]. The polyphosphate granules were extracted by suspending the cells in 0.6 ml of double distilled water, sonicating and boiling the samples, and adding 0.6 ml of a 24:1 chloroform: isoamyl alcohol solution. Three ml each of toluidine blue solution and 0.2 N acetic acid was added to 300 µl of the aqueous phase of the phosphorus extract, and the colorimetric reaction was allowed to proceed for 25 minutes before the absorbance was read at 630 nm and compared against the standard curve [5].
Phosphorus Reclaim Module Design and Testing
After we modify our E. coli for phosphate uptake, accumulation, and release, we plan on implementing them in Phosphorus Reclaim Modules (PRMs), a bioreactor that can be applied to specific applications in order to reduce the amount of phosphorus in water.
Before we can implement our E. coli in PRMs, we need to develop a method for containing the E. coli within the PRM. As suggested by our faculty advisor Dr. Jenna Rickus, we decided to use the sol-gel process to immobilize the E. coli in sol-gel beads. This process combining a neutral solution of tetramethyl orthosilicate (TMOS) with E. coli suspended in acidic phosphate-buffered saline (PBS). When the TMOS sol-precursor is added to the buffer, the pH decrease favors rapid polymerization of the sol-gel. The TMOS sol-precursor and PBS mixture is then dripped into a vat of mineral oil using a syringe. The hydrophobic mineral oil causes the sol-gel to form a bead as it polymerizes.
The sol-gel beads, as shown in the two-dimensional diagram on the right, contain pores large enough to allow small molecules, such as water and phosphate, to pass through, but too small for E. coli to pass through. To test whether or not E. coli could be immobilized within these beads, we compared sol-gel beads produced with PBS without E. coli to sol-gel beads produced with PBS with E. coli that produce either a red or green fluorescent protein. If the beads with the E. coli fluoresce either red or green, respectively, and the beads produced without E. coli do not fluoresce, that provides evidence that E. coli can be contained within the sol-gel beads.
With the help of the United States Department of Agriculture’s National Soil Erosion Research Lab, we designed a prototype for a PRM, which they then constructed for us. The prototype contains two reservoirs, one for overflow, and one for delivery. Water is pumped from the overflow to the delivery reservoir, from which it enters one of three filter canisters. These filter canisters will be packed with sol-gel beads. As the sol-gel beads contain the E. coli, the filter canister contains the sol-gel beads, thus containing the E. coli within the PRM.
Since we are packing the filter canister with sol-gel beads, the PRM reduces flow rates, which could be an issue in high-flow rate applications, such as tile drains. To test the effects of packed-bed canister on flow rates, we first obtained flow rates for the bioreactor without beads. As we could not produce enough sol-gel beads this summer, we bought 2 mm beads, which will be used to pack the canister. While pore size and water flow through the sol-gel beads and the store-bought crafting beads differ, the differences in flow rates give us an estimate on the effects of a packed-bed PRM on flow rates.
Operon Construction
As the majority of our genes are uncharacterized, we wanted to get more characterization data before designing operons. Characterization data on the affinity of the Pit homologs for phosphate uptake could have allowed us determine which, if not all would best serve in our polyphosphate accumulating E. coli. Data indicating whether the PPKs favored polyphosphate synthesis or hydrolysis would have also been useful in designing operons as well. While characterization data was only obtained for PPX2, theoretical operons were constructed  based on protein function, as predicted by phylogenetic analysis.
One operon was designed for polyphosphate accumulation: a weak constitutive Anderson promoter BBa_J23105 was used with the coding regions for the PPK1 homolog, PPK2 homolog A, and the three Pit homologs. The PPK homologs were selected due to their predicted affinity for polyphosphate synthesis. The Pit genes were placed at the end of the operon, as the genes at the end of the operon are likely to be expressed less than the genes at the beginning of the operon. As Pits are membrane proteins, we do not want to overexpress the membrane proteins and risk destabilizing the membrane. A weak constitutive promoter was chosen because, in addition to concerns of overexpressing membrane proteins, there are concerns of taxing the cell’s metabolism, as the PPKs associated with polyphosphate synthesis utilize ATP. As our operons are designed with a ribosome binding site (RBS) between all coding regions and all gBlocks were synthesized with the RBS, Gibson assembly would need to be used to switch out the RBS in order to enhance operon stability.
The second operon was designed for depleting the cell’s polyphosphate reserve. The weak Anderson promoter BBa_J23105 was used with the coding regions for both PPGK homologs, PPK2 homolog B, PPX2, and PPK2 homolog C. While only PPX2 hydrolyzes polyphosphate into orthophosphate, allowing the phosphate to be exported, the PPGK and PPK homologs were selected for their ability to hydrolyze phosphate to transfer the phosphate group to molecules beneficial to the cell. PPGK uses the terminal phosphate of a polyphosphate chain to phosphorylate glucose, the first step of glycolysis, and PPK uses the terminal phosphate of a polyphosphate chain to produce nucleoside triphosphates. [6].
While using a constitutive promoter allows us to characterize the effects of our operons on polyphosphate uptake, accumulation, and release, the next step would be to choose inducible and repressible promoters. Therefore, when the E. coli needs to accumulate phosphorus, the first operon is expressed and the second operon is repressed, and so the polyphosphate is not being hydrolyzed or released. When the E. coli needs to release phosphorus, the first operon is repressed and the second operon is expressed, which prevents the cell from wasting ATP on polyphosphate synthesis and uptaking the released phosphorus.
Future Experimentation
Leaching Assay
In order to quantify the efficacy of the sol-gel beads on E. coli containment, we will perform an assay to detect the amount of bacteria that leaches from the beads.
This can be done using a plate assay, in which the sol-gel beads containing E. coli are placed in a flask with water. Over time, several 100 samples of the water will be plated to determine the concentration of colony forming units per ml of water. This concentration can be used to estimate the total amount of colony forming units in the flask of water, which can then be used to estimate the amount of colony forming units released per gram of sol-gel beads.
The leaching assay can also be done by immobilizing red fluorescent protein-expressing E. coli in sol-gel beads, and placing the beads in a flask with water. Then, over time, samples of the water will be analyzed spectrophotometrically for absorbance at 580 nm. In this case, the specific concentration of colony forming units leached per gram of sol-gel beads cannot be quantified, but the absorbance values can be used to estimate how many colony forming units are leached.
Phosphorus Reclaim Module (PRM) Application Testing
After successfully transforming our final phosphorus uptake, accumulation, and release construct into E. coli, we plan on immobilizing it within sol-gel beads and packing the canister of our PRM in order to test the ability of our prototype PRM to remove phosphate from water, as it would in a real-life application.
References
[1] "His-tagged Proteins–Production and Purification", Thermo Fisher Scientific, 2016. [Online]. Available: http://www.thermofisher.com/us/en/home/life-science/protein-biology/protein-biology-learning-center/protein-biology-resource-library/pierce-protein-methods/his-tagged-proteins-production-purification.html. [Accessed: 17- Oct- 2016].
[2] O. Adesanwo, D. Ige, L. Thibault, D. Flaten and W. Akinremi, "Comparison of Colorimetric and ICP Methods of Phosphorus Determination in Soil Extracts", Communications in Soil Science and Plant Analysis, vol. 44, no. 21, pp. 3061-3075, 2013.
[3] Data Pack: Lachat applications in standard methods, 21st ed. Lachat Instruments, 2008, pp. 48-51.
[4] T. Morohoshi, T. Maruo, Y. Shirai, J. Kato, T. Ikeda, N. Takiguchi, H. Ohtake and A. Kuroda, "Accumulation of Inorganic Polyphosphate in phoU Mutants of Escherichia coli and Synechocystis sp. Strain PCC6803", Applied and Environmental Microbiology, vol. 68, no. 8, pp. 4107-4110, 2002.
[5] C. Mukherjee, C. Mukherjee and K. Ray, "An improved method for extraction and quantification of polyphosphate granules from microbial cells", Nature Protocol Exchange, 2015.
[6] A. Kawakoshi, H. Nakazawa, J. Fukada, M. Sasagawa, Y. Katano, S. Nakamura, A. Hosoyama, H. Sasaki, N. Ichikawa, S. Hanada, Y. Kamagata, K. Nakamura, S. Yamazaki and N. Fujita, "Deciphering the Genome of Polyphosphate Accumulating Actinobacterium Microlunatus phosphovorus", DNA Research, vol. 19, no. 5, pp. 383-394, 2012.
