<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Purdue_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Purdue/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Purdue">HOME </A></LI><LI class="menu_item"><A href=" https://2016.igem.org/Team:Purdue/Team">TEAM</A></LI><LI class="menu_item"> BACKGROUND  
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Problem">Problem </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Phosphorous">Phosphorous </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Wastewater">Wastewater </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Organisms">Organisms </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/ProteinProfiles">Protein Profiles </A></LI></UL><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Description"> ★ Description </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Design"> ★ Design </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Model"> ★ Model </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Hardware"> Hardware/ Prototypes</A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Experiments"> ★ Experiments </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Proof"> ★ Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Results"> ★ Results </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/MDC/MFC">Side Project </A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Parts">★ Parts </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Composite_Part"> ★ Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Part_Collection"> ★ Part Collection </A></LI><LI class="menu_item"> HUMAN PRACTICES 
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Human_Practices"> ★ Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/HP/Silver">★ Silver </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/HP/Gold">★ Gold </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Integrated_Practices"> ★ Integrated Practices </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Engagement"> ★ Engagement </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Collaborations"> ★ Collaborations </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/SYNENERGENE">SYNENERGENE </A></LI><LI class="menu_item"> NOTES FROM THE LAB
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Notes_from_the_Lab"> ★ Notes from the Lab </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Notebook"> ★ Notebook </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Protocols"> ★ Protocols </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/InterLabStudy"> InterLab Study </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Safety"> Safety </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Purdue/Attributions">  ATTRIBUTIONS </A></LI></DIV><DIV class="content_wrapper"><DIV style="right:0px;top:60px;position:fixed;"><P>Purdue Biomakers</P><P style="padding-left: 30px;"><A title="Purdue Biomakers Twitter" href="https://twitter.com/purduebiomakers">Twitter</A></P><P style="padding-left: 30px;"><A title="Purdue Biomakers Facebook" href="https://www.facebook.com/PurdueBiomakers/">Facebook</A></P><P style="padding-left: 30px;"><A title="Purdue Biomakers Website" href="http://purduebiomakers.com/">Website</A></P></DIV><DIV class="column full_size"><H1>Results</H1><H2>Transformations into <I>E. coli</I></H2><P><CENTER><B>Figure 1</B>: Transformation of PPX2 homolog on PSB1A3 plasmid into <I>E. coli</I> grown on an LB agar plate with ampicillin.</CENTER></P><P><CENTER><B>Figure 2</B>: Transformation of PPK2 homolog C on PSB1A3 plasmid into <I>E. coli</I> grown on an LB agar plate with ampicillin.</CENTER></P><P>
Figure 1 and Figure 2 demonstrate the growth of modified <I>E. coli</I> on LB agar plates containing ampicillin. This antibiotic was shown to be effective in controlling the growth of bacteria that did not take up the modified plasmid with antibiotic resistance, as no growth was found on the negative control. The success of these transformations, as confirmed through plasmid sequencing, is discussed below.
</P><H2>Plasmid Sequencing</H2><P>Plasmid DNA was then obtained for colonies resulting from transformations and brought to the Purdue Genomics Core Facility for low throughput Sanger sequencing. </P><P>The sequencing for plasmids in the PSB1A3 backbone were done with a reverse primer that targets the his operon terminator on PSB1A3, and so the reaction should proceed, regardless of whether or not an insert was present in the plasmid. No forward primer was used, as we did not have a forward primer that would work for PSB1A3 at the time of sequencing.</P><P><CENTER><B>Figure 3.</B>: The results of Sanger sequencing for the transformation of PPK2 homolog C in the plasmid PSB1A3 indicate that PPK2 homolog C plasmid has the correct sequence.</CENTER></P><P>The sequencing for plasmids in the PSB1C3 backbone were done with the standard BioBrick primers VR and VF2. Therefore, a reaction will occur regardless of whether or not an insert is present in the backbone. </P><P><CENTER><B>Figure 3.b.</B>: The results of Sanger sequencing for the transformations of PPX2 homolog in the plasmid PSB1C3 indicate that the PPX2 homolog contains the correct sequence.</CENTER></P><P><CENTER><B>Figure 3.c.</B>: The results of Sanger sequencing for the transformations of PPX2 homolog in the plasmid PSB1A3 indicate that the PPX2 homolog contains the correct sequence.</CENTER></P><H2> Analysis of Phosphorus Uptake, Accumulation, and Exportation</H2><H3>Phosphorus Uptake and Export Assay</H3><P><CENTER><B>Figure 4</B>: The amount of extracellular total phosphorus in Tris-HCl buffered minimal media after extended 72-hour phosphorus transportation assay with <I>E. coli</I>.</CENTER></P><P>Inductively coupled plasma optical emission spectrometry (ICP-OES) was used to quantify the importation and exportation of phosphorus by modified and unmodified <I>E. coli</I> with results detailed in Figure 4. Unmodified <I>E. coli</I> served as a negative control to which PPX2 in PSB1A3 backbone was compared. The results, reflective of 72 hours of <I>E. coli</I> suspension in Tris-HCl buffered minimal media, indicate that PPX2 effectively exported phosphorus from the cell compared to the exportation of phosphorus from unmodified <I>E. coli</I> because PPX2 culture demonstrated a higher extracellular phosphorus concentration.</P><H3>Polyphosphate Accumulation Assay</H3><P><CENTER><B>Figure 5</B>A standard curve for amount of phosphate per 300 µl of solution was generated using sodium phosphate glass type 45. The coefficient of determination (R<SUP>2</SUP>), at 0.975, is relatively close to 1, which indicates that the linear regression is a suitable equation for estimating the relationship between concentration of polyphosphate and absorbance at 630 nm. The suitability of the linear regression is also supported by the root mean square error, which, at 0.02, is relatively close to 0.</CENTER></P><P><CENTER><B>Figure 6</B>: the results indicate that the samples for <I>E. coli</I> containing PPK2 C (both PSB1C3 and PSB1A3), PPX2 (both PSB1C3 and PSB1A3), and PPGK accumulate less polyphosphate than unmodified <I>E. coli</I>, and samples for <I>E. coli</I> containing Pit homolog A and PPGK ATPI accumulate more polyphosphate than the unmodified <I>E. coli</I>. </CENTER></P><P>These results indicate that PPX2 and PPGK works as expected, as PPX2 hydrolyzes phosphate to produce orthophosphate, and PPGK transfers a phosphate group from polyphosphate to glucose. While PPK2 homolog C’s preference for polyphosphate hydrolysis or synthesis is unknown, the low concentration of polyphosphate in cells containing PPK2 homolog C suggests that PPK2 C favors polyphosphate hydrolysis to phosphorylate nucleoside diphosphates (or nucleoside monophosphates). Meanwhile, these results also suggest that Pit A is working as expected, as Pit A is predicted to transport phosphate into the cell. PPGK ATPI is not working as expected, as PPGK ATPI, like PPGK, transfers a phosphate group from polyphosphate to glucose.</P><P>In the future, we should extract polyphosphate granules from a large sample of unmodified <I>E. coli</I> in order to obtain a normal distribution curve to which our modified samples could be compared. </P><H2>Immobilization in Sol-gel beads</H2><P>From the outset of this project, our team recognized the absolute importance of maintaining biosafety both in the lab and in the real-world application of our problem. Although 5-a strains of <I>E. coli</I> like the ones used in our project are non-pathogenic, tests for water contamination are nondiscriminatory as to strain or viability of any bacteria present in water. As such, kill switches alone would not be enough to address concerns around the safe application of our phosphate reclaim modules (bioreactors, buoys, etc.).</P><P>
Instead we needed a means of isolating our constructs from their treated effluent without sealing our constructs away from treating influent in the first place. To do so we settled upon a technique known as sol-gel immobilization. This technique, in which cells are isolated in highly porous silica matrices (xerogels) through an epoxy-like process allows for the easy diffusion of nutrients into and out of cells without the diffusion of cells from the matrices. 
</P><P>
After attempting to isolate unmodified and RFP-expressing E. coli as negative and positive controls respectively, we imaged the two xerogels through fluorescence microscopy and affirmed that yes, in fact, we were able to immobilize <I>E. coli</I>.
</P><H2>References</H2><P>
[1] JMP®, Version 12. SAS Institute Inc., Cary, NC, 1989-2007
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>