<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Purdue_Organisms skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Purdue/Organisms</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Purdue">HOME </A></LI><LI class="menu_item"><A href=" https://2016.igem.org/Team:Purdue/Team">TEAM</A></LI><LI class="menu_item"> BACKGROUND  
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Problem">Problem </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Phosphorous">Phosphorous </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Wastewater">Wastewater </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Organisms">Organisms </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/ProteinProfiles">Protein Profiles </A></LI></UL><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Description"> ★ Description </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Design"> ★ Design </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Model"> ★ Model </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Hardware"> Hardware/ Prototypes</A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Experiments"> ★ Experiments </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Proof"> ★ Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Results"> ★ Results </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/MDC/MFC">Side Project </A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Parts">★ Parts </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Composite_Part"> ★ Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Part_Collection"> ★ Part Collection </A></LI><LI class="menu_item"> HUMAN PRACTICES 
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Human_Practices"> ★ Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/HP/Silver">★ Silver </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/HP/Gold">★ Gold </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Integrated_Practices"> ★ Integrated Practices </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Engagement"> ★ Engagement </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Collaborations"> ★ Collaborations </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/SYNENERGENE">SYNENERGENE </A></LI><LI class="menu_item"> NOTES FROM THE LAB
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Notes_from_the_Lab"> ★ Notes from the Lab </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Notebook"> ★ Notebook </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Protocols"> ★ Protocols </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/InterLabStudy"> InterLab Study </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Safety"> Safety </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Purdue/Attributions">  ATTRIBUTIONS </A></LI></DIV><DIV class="content_wrapper"><DIV style="right:0px;top:60px;position:fixed;"><P>Purdue Biomakers</P><P style="padding-left: 30px;"><A title="Purdue Biomakers Twitter" href="https://twitter.com/purduebiomakers">Twitter</A></P><P style="padding-left: 30px;"><A title="Purdue Biomakers Facebook" href="https://www.facebook.com/PurdueBiomakers/">Facebook</A></P><P style="padding-left: 30px;"><A title="Purdue Biomakers Website" href="http://purduebiomakers.com/">Website</A></P></DIV><DIV class="button_click" onclick=" parent.location= 'https://2016.igem.org/Team:Purdue/Phosphorous '" style="bottom:0;left:0; position:fixed;">  
Previous Page
</DIV><DIV class="button_click" onclick=" parent.location= 'https://2016.igem.org/Team:Purdue/Description '" style="bottom:0;right:0; position:fixed;">  
Next Page
</DIV><DIV class="column full_page"><H2>Why we chose <I>E. coli</I></H2><P><I>Escherichia coli </I>(<I>E. coli</I>) is a gram-negative bacteria of the genus <I>Escherichia</I>. It is classified as a facultative anaerobe due to its ability to switch over to ethanol fermentation process when little to no oxygen is present in the environment. While most strains of <I>E.coli </I> reside within mammalian intestines (and as such have developed a controversial reputation for the diseases they can cause), laboratory strains (such as K-12 derivative MG1655) neither survive in intestinal settings nor form problematic biofilms [1]. In fact, these strains are favored by most researchers because their rapid growth rates, low purchase and maintenance costs, relationship to most by bacterial species by approximately 20%, and versatility in plasmid based protein production or genetic recombination techniques. Since the complete sequencing of the genome in 1997 [4], <I>E. coli</I> has played a key role in the research of synthetic metabolic pathways, gene regulation mechanisms of prokaryotes, and production of high value chemicals and proteins [2].</P><P><CENTER><B>Figure 1</B>: Colorized scanning electron micrograph of <I>Escherichia coli</I>, grown in culture and adhered to a cover slip. [5].</CENTER></P><P>We decided <I>E.coli</I> would work best as our chassis because we thought that the large body of research on its metabolism would makes it preferable to the relatively unknown microbes currently used for phosphorus accumulation in biological wastewater treatment. </P><H2>Why we used <I>Microlunatus phosphovorus</I></H2><P><I>Microlunatus phosphovorus</I> (<I>M. phosphovorus</I>) is a gram-positive, aerobic, coccus-shaped, actinobacteria of the relatively new genus <I>Microlunatus</I>. First isolated in 1995 from activated sludge, <I>M. phosphovorus</I> showed extraordinarily high phosphorus removal from wastewater (up to 48% of dry weight) when compared to other phosphorus accumulating organisms (PAOs) reported in literature and significant promise for application in enhanced biological phosphorus removal (EBPR). After its genome was sequenced in 2012, Kawakoshi et. al confirmed the presence of “four polyP kinases (ppks), two polyP-dependent glucokinases (ppgks), and three phosphate transporters (pits)” responsible for polyP metabolism in <I>M. phosphovorus</I> -- several more genes than reported in any other known actinobacterial PAOs. Interestingly, Kawakoshi et. al also discovered that as a PAO, <I>M. phosphovorus</I> only expresses one exopolyphosphatase gene rather than the two that are typically observed. In expressing only a single copy of the exopolyphosphatase gene (responsible for the hydrolysis of polyphosphate) it is believed that <I>M. phosphovorus</I> is better able to prevent the export of phosphorus in the form of orthophosphates from the cell. </P><P><CENTER><B>Figure 2</B>: Dividing <I>M. Phosphovorus</I> NM-1 cell as viewed under SEM</CENTER></P><P><I>M. phosphovorus</I> appears to accumulate phosphorus in a manner similar to proteobacterial PAOs in aerobic conditions. However, during anaerobic conditions, rather than synthesizing polyhydroxyalkanoate carbon reserves to later be oxidized to produce energy for growth and polyphosphate accumulation in aerobic conditions, <I>M. phosphovorus</I> instead ferments glucose to to provide instantaneous energy phosphorus accumulation without the need for a reoxygenation of its growth media. 
</P><H2>References</H2>
[1] C. A. Fux, M. Shirtliff, P. Stoodley, and J. W. Costerton, &quot;Can laboratory reference strains mirror ‘real-world’ pathogenesis?,&quot; Trends in Microbiology, vol. 13, no. 2, pp. 58–63, Feb. 2005. [Online]. Available: http://www.sciencedirect.com/science/article/pii/S0966842X04002550. Accessed: Jun. 2, 2016. 
[2] D. Barr, &quot;Research guides: Lab rats and more: Model organisms in research today: E. Coli,&quot; inHarvard Research Guides, 2011. [Online]. Available: http://guides.library.harvard.edu/c.php?g=310307&amp;p=2071641#s-lg-box-6324388. Accessed: Jun. 2, 2016. 
[3] Healthline,. Escherichia Coli. 2015. Web. 2 June 2016. 
[4] F. R. Blattner et al., &quot;The complete genome sequence of Escherichia coli K-12,&quot; Articles, vol. 277, no. 5331, pp. 1453–1462, Sep. 1997. [Online]. Available: http://science.sciencemag.org/content/277/5331/1453. Accessed: Jun. 2, 2016.
[5] https://www.flickr.com/photos/niaid/16578744517/ </DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>