<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Purdue_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Purdue/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Purdue">HOME </A></LI><LI class="menu_item"><A href=" https://2016.igem.org/Team:Purdue/Team">TEAM</A></LI><LI class="menu_item"> BACKGROUND  
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Problem">Problem </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Phosphorous">Phosphorous </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Wastewater">Wastewater </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Organisms">Organisms </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/ProteinProfiles">Protein Profiles </A></LI></UL><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Description"> ★ Description </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Design"> ★ Design </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Model"> ★ Model </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Hardware"> Hardware/ Prototypes</A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Experiments"> ★ Experiments </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Proof"> ★ Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Results"> ★ Results </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/MDC/MFC">Side Project </A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Parts">★ Parts </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Composite_Part"> ★ Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Part_Collection"> ★ Part Collection </A></LI><LI class="menu_item"> HUMAN PRACTICES 
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Human_Practices"> ★ Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/HP/Silver">★ Silver </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/HP/Gold">★ Gold </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Integrated_Practices"> ★ Integrated Practices </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Engagement"> ★ Engagement </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Collaborations"> ★ Collaborations </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/SYNENERGENE">SYNENERGENE </A></LI><LI class="menu_item"> NOTES FROM THE LAB
			</LI><LI><A href="https://2016.igem.org/Team:Purdue/Notes_from_the_Lab"> ★ Notes from the Lab </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Notebook"> ★ Notebook </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Protocols"> ★ Protocols </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/InterLabStudy"> InterLab Study </A></LI><LI><A href="https://2016.igem.org/Team:Purdue/Safety"> Safety </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Purdue/Attributions">  ATTRIBUTIONS </A></LI></DIV><DIV class="content_wrapper"><DIV style="right:0px;top:60px;position:fixed;"><P>Purdue Biomakers</P><P style="padding-left: 30px;"><A title="Purdue Biomakers Twitter" href="https://twitter.com/purduebiomakers">Twitter</A></P><P style="padding-left: 30px;"><A title="Purdue Biomakers Facebook" href="https://www.facebook.com/PurdueBiomakers/">Facebook</A></P><P style="padding-left: 30px;"><A title="Purdue Biomakers Website" href="http://purduebiomakers.com/">Website</A></P></DIV><DIV class="column full_size"><P>Click on each week for more information</P><DIV class="lab"><H2>Week 1</H2><H3>MONDAY, MAY 23</H3><P>Day 1 on the job–the morning began with a tour of Bindley Bioscience Center for the new interns (Barrett, Caleigh, Emma, and Paige), during which Susan (the world’s best receptionist) kindly showed the fledgling researchers around and gave them the run-down on BBC do’s and don’t’s. The remainder of the day was spent working on gene design via Benchling, researching potential assays, and meeting with Drs. Rickus and Solomon. Important topics they raised included the importance of an experimental design plan, the need to begin with the basics (i.e. how do we know a protein is being expressed?) before delving into more complex issues of function, and gene design specifics.</P><H3>TUESDAY, MAY 24</H3><P>Nine out of ten genes needed for luxury phosphorus uptake were ordered at the conclusion of Day 2! Throughout the day, genes were restructured for use with 3A Gibson Assembly protocol, and new constituent promoters were added (both sigma70 and sigmaS to arrest growth in P E. coli). Primer design began. Additionally, the hunt for assays continued for both phosphorus and nanowire constructs. Interns learned how to optimize codons using IDT both automatically and manually, checking for segments such as EcoRI, SpeI, XbaI, PstI, and HINDIII that should not be present in coding regions of constructs, as these sequences code for restriction sites.</P><H3>WEDNESDAY, MAY 25</H3><P> A review of sterile technique and iGEM kit DNA extractions in the lab comprised a majority of the morning. Some golden rules of sterile technique: Always wipe down the lab bench with 70% ethanol before beginning; work as close to the flame as possible, as it creates a sterile “bubble”; and get into and out of pipette tip boxes as quickly as possible to keep them sterile. The autoclaving of new pipette tips and water was also used for demonstration purposes. Afternoon work consisted of continued definition of desired assays and design of nanowire genes.</P><H3>THURSDAY, MAY 26</H3><P>The creation of ready-to-use plates was the focus of lab work on Thursday. Interns followed an existing protocol to make plain LB agar, LB agar with ampicillin, and LB broth, culminating in the pouring and storing of too many plates to count. Caleigh, Sean, Suraj, and Paige took a field trip to the West Lafayette Waste Management facility to speak with Sara Peel of the Wabash River Enhancement Corporation (WREC) and Angela Andrews of the City of Lafayette Water Pollution Control. It was discovered that the facility is newly permitted for phosphorus removal; however, a chemical method as opposed to a biological one will be implemented August 1 for both political and financial reasons. The up-front cost for chemical treatment was less than for biological treatment, which made it the optimal choice for the time being, but the facility plans to switch to biological treatment as soon as it has the funds to do so, as this will be more cost-effective longterm. The team plans to return to the plant sometime in July to receive a full tour and collect wastewater samples to be used for testing.</P><H3>FRIDAY, MAY 27</H3><P>Brainstorming for both background content for the official iGEM project wiki and a project timeline were completed in addition to an establishment of a protocol for completing lab notebooks. A Work Breakdown Structure (WBS) and Gantt Chart were constructed to record timeline information. Practice transformations of GFP and RFP kit plate parts were completed by the interns throughout the day. The conclusion of a successful first week!</P></DIV><DIV class="lab"><H2>Week 2</H2><H3>MONDAY, MAY 30</H3><P>Memorial Day was an optional day on the job, but the dedicated iGEM team put in an afternoon of work in Bindley. Practice transformations were successful–both GFP and RFP were apparently expressed, and growth on control plates was as expected. Practice minipreps were next on the agenda, which the interns also completed successfully in addition to some supply inventory.</P><H3>TUESDAY, MAY 31</H3><P>Lists of supplies needed for various assays were assembled, and an assessment of necessary ordering was made, thus finalizing the functional assays to run for both nanowire and phosphorus genes. Additionally, brainstorming for an Experiment (crowd-funding for scientific research) campaign was completed.</P><H3>WEDNESDAY, JUNE 1</H3><P> Background research continued, and some ideas were tossed around concerning the official project theme for this year (more to come on that later). A meeting with Drs. Rickus and Solomon led to recommendations to practice with both the transmission electron microscope (TEM) and the process of electroporation for genome integration to prepare for performing these procedures later. Project write-ups for the Experiment campaign were completed, as was an application for a grant from SYNENERGENE for advanced human practices.</P><H3>THURSDAY, JUNE 2</H3><P>The Experiment online crowdfunding platform was submitted for approval in the morning, and an order for both competent cells and primers was placed. Background research was again the main focus, and most team members concentrated their efforts on learning the ins and outs of both Shewe (Shewanella oneidensis) and tiny moon phosphorus-eaters (Microlunatus phosphovorus).</P><H3>FRIDAY, JUNE 3</H3><P>This week’s conclusion was easily one of its most exciting moments, as the team took a field trip to visit Drs. Chi-hua Huang and Ashley Hammac of the USDA National Soil Erosion Research Lab, who generously donated both their time and expertise to answer questions and allow the team to tour the facility. Analytical labs, rain-producing machines for the purpose of studying erosion patterns, bioreactors, and anaerobic growth chambers were among the cutting-edge technologies the team observed. Thank you, Dr. Huang and Dr. Hammac!

Word was received later in the afternoon that the Experiment proposal had been approved; the campaign launches June 9!</P></DIV><DIV class="lab"><H2>Week 3</H2><H3>MONDAY, JUNE 6</H3><P>Monday morning got rolling with the attendance of ABE PhD student Dr. Andi Hodaj’s thesis defense regarding two-stage ditches as a tool to reduce nutrient loads. A two-stage ditch is a variation of a traditional agricultural ditch in which the banks are dredged to create floodplains, dissipating the energy of high-flow water and increasing interaction time between vegetation and water. Purdue’s two-stage ditch utilized in Dr. Hodaj’s was established in 2012 and drains 660 acres; it may be a possible implementation point for the project’s biological method.  Additionally, primers arrived, and the Experiment project write-ups were finalized and proofed.</P><H3>TUESDAY, JUNE 7</H3><P>The Experiment campaign went live! Thank you to both Drs. Rickus and Solomon for endorsing the project, and to those who are project backers–your donations are greatly appreciated. See last week’s previous post for more information; the Experiment page can be found here. Work continued on both the spring publication and protein profiles for the wiki.</P><H3>WEDNESDAY, JUNE 8</H3><P> Formal lab meeting #1 was a success with the most highly prioritized goal for the upcoming week to find or generate relevant numbers for the project including necessary flow rates for a microbial fuel cell, an appropriate density of E. coli within it, etc. The rest of the afternoon was spent working on action items from the meeting and printing the spring publication, which is FINISHED.</P><H3>THURSDAY, JUNE 9</H3><P>Team members Barrett Davis, Paige Rudin, and Suraj Mohan attended the Regional Clean Energy Innovation Forum held at Ross-Ade Stadium. Leading scientists from across the country gathered to discuss topics like energy storage, biomass and synthetic biology applications (a conversation that piqued the students’ interest), materials manufacturing, and more. The theme the team took away from the day is best summarized with the words of Purdue President Mitch Daniels during his opening remarks: “It’s only an innovation when it’s useful to someone.” How can new technology of nanowires and microbial fuel cells be integrated efficiently into established systems? Is it a cost-effective solution? Only time will tell, but the team is excited to find out.</P><H3>FRIDAY, JUNE 10</H3><P>Happy birthday wishes to Barrett, as he celebrated his 20th birthday in the lab on Friday. In the best interest of keeping a sterile work environment, the team refrained from strewing confetti across the bench, although it would’ve been both fun and festive.
Dow AgroSciences generously donated various lab supplies to the iGEM workspace that the team was thrilled to pick up in Indianapolis on Friday morning. Thank you to Dr. Steve Evans, Mr. Mike Doody, and DAS administration for their support in realizing the dream of a self-sufficient, independent lab for the sole use of undergraduate iGEM students. With equipment, it is possible to operate without reliance upon the lab space of other Bindley Bioscience Center scientists, lending additional flexibility to the projects iGEMmers are able to complete.
Phosphorus genes arrived, and the team got to work on PCR amplification in the afternoon. This included resuspending IDT DNA gBlocks and using PCR to increase the amount of DNA present for transformation.</P><H3>THE WEEKEND</H3><P>So diligent is this year’s team that several members completed additional PCR amplification and iGEM kit plate promoter transformations on Saturday and Sunday. After digestion and ligation, they did some gel electrophoresis, as well, running several different samples.</P></DIV><DIV class="lab"><H2>Week 4</H2><H3>MONDAY, JUNE 13</H3><P>Most significantly this week, team members hosted Molecular Agriculture Summer Institutes (MASI) high school students in the lab as they worked with Emma on completing InterLab protocols [InterLab is an iGEM-wide study working to establish a consistent unit of measurement for fluorescence; experimentation and measurement is completed using GFP.] The transformations completed over the weekend proved successful–the positive RFP control exhibited growth, while there was no growth on the negative control. Transformed cells were inoculated and plates parafilmed for short-term storage. PCR was completed for 3 phosphorus genes whose gel electrophoresis analyses were unclear when attempted over the weekend. Wiki design was also discussed to get this critical project element up and running ASAP.</P><H3>TUESDAY, JUNE 14</H3><P>Lab work for Tuesday included the mixing of gels for later electrophoresis, the miniprep of promoters that were PCR’ed previously, and continued InterLab transformations.</P><H3>WEDNESDAY, JUNE 15</H3><P>Meeting day! Highlights of this week’s meeting include a discussion of the utilization of mass spectroscopy to complete proteomic profiles, the wastewater analytics tools possibly available to us through the generosity of Dr. Ashley Hammac and the USDA National Soil Erosion Research Lab, and the idea to replicate a study upregulating E. coli genes related to phosphorus uptake and storage as a basis of comparison to engineered strains. Via Skype, the team established a collaboration agreement with Rose-Hulman Institute of Technology’s iGEM team, aiding them with fluorescent microscopy in exchange for help characterizing some proteins.

In the afternoon, several team members volunteered with the Wabash River Enhancement Corporation (WREC) by aiding in the upkeep of a rain garden in Lafayette designed to use plants to stop pollutants from infiltrating the Wabash. Water flows off the roof of site buildings (Oakland Elementary), and nutrients (such as phosphorus) are captured by plant roots. This garden needed some weeding and re-planting to be in tip-top shape again.</P><H3>THURSDAY, JUNE 16</H3><P>Team members were busy in the lab completing transformations of phosphorus genes, more PCR, autoclaving, and inoculation of transformed InterLab genes. Excitement of the day was a discussion of hosting a conference for other iGEM teams–stay tuned for more to come.</P><H3>FRIDAY, JUNE 17</H3><P>Gel electrophoresis, inoculations of transformations from yesterday, and the production of more LB agar plates with ampicillin closed out the week in Bindley Bioscience Center. Additionally, here’s an update on the Experiment crowdfunding campaign in the form of an advertisement to be featured on news screens across campus. Thank you to everyone who has aided our cause!</P></DIV><DIV class="lab"><H2>Week 5</H2><H3>MONDAY, JUNE 20</H3><P>The week began with a strong focus on human practices as the team fleshed out 3 ideas. The first, iNAC, the iGEM North American Conference, was sparked by the observation that many more European than North American teams medal, and of those that medal, the proportion of golds is substantially higher for Europe. The concept is a “Going for Gold” event to facilitate better understanding of iGEM requirements, lab practices, and the constitution of a winning project. This event would take place in summer 2017, co-hosted in Chicago by Purdue and several other Midwest teams (in theory, anyway). In conjunction with iNAC, the team is hoping to host iNIC (iGEM Northern Indiana Conference) to discuss the possibility of coordinating iNAC. If this event is pursued, it would be a day-long conference at the end of July.

The third idea is in regards to the Great North American Skype Series (G-NASS), a “Skype roulette” to foster inter-team communication. There would be a goal of 20 teams participating in a couple of 30-minute calls. More to come soon.</P><H3>TUESDAY, JUNE 21</H3><P>Tuesday morning kicked off with the digestion of phosphorus genes for ligation later that afternoon. Researchers met with Dr. Tommy Sors, Bindley Bioscience Center’s Chief Liaison, regarding the use of equipment for Western blot protocols and an overview of the project. He commented on its ambitious scope, quoting Wes Jackson: “If your life’s work can be accomplished in your lifetime, you’re not thinking big enough.”</P><H3>WEDNESDAY, JUNE 22</H3><P> Modeling commenced with Mark Aronson, former iGEM president and current advisor who Skyped the team from New York. A working bioreactor model is top priority, with layers of complexity to be added as experimental data is collected. Phosphorus genes were again transformed into competent cells, and newly-received plasmids resuspended after transport.</P><H3>THURSDAY, JUNE 23</H3><P>Transformed phosphorus cells were inoculated, several colonies taken from LB agar plates and suspended in LB broth with ampicillin to incubate overnight. All phosphorus genes also underwent PCR amplification. An afternoon Skype call with the University of Exeter in England topped off the day as teams exchanged greetings, project synopses, and ideas for future collaboration.</P><H3>FRIDAY, JUNE 24</H3><P>A Shewanella oneidensis culture, generously donated by Argonne National Laboratory, arrived amidst great celebration on Friday morning. This Shewie will serve as a basis of comparison for E. coli expressing organic nanowires. The team was busy in the lab making and running gels for phosphorus and re-transforming phosphorus genes after Thursday’s inoculations proved less than satisfactory. The University of Nebraska–Lincoln Skyped in the afternoon, which was especially relevant as they are tackling a similar wastewater treatment dealing with nitrogen and nitrate remediation. A rewarding end to a productive week.</P></DIV><DIV class="lab"><H2>Week 6</H2><H3>JUNE 27 – JULY 1</H3><P>This week, things are going to be shaken up a little bit. Here’s a quick summary of the goings-ons in short paragraph form:

The team visited Imagination Station in downtown Lafayette regarding the possibility of volunteering both this summer and continuing forward. Imagination Station is a museum with interactive exhibits; iGEM would assist by educating about synthetic biology. This would be a great opportunity for club community outreach, and we’re excited to launch this collaboration together.

Lab work continued with PCR amplification, gel electrophoresis, extractions, and transformations of phosphorus and nanowire genes.

Collaboration Skype calls with Exeter, the University of Chicago, and SVCE Chennai were successful. Exeter and UChicago are onboard to help with database entries and protein characterization, and SVCE will distribute a survey about wastewater management practices.

Through a meeting with Dr. Ron Semmel, a Monsanto scientist, the team acquired useful contacts and information regarding local phosphorus and nitrogen soil concentrations. Dr. Semmel concisely described his job: “They send me things. I try to kill them. If I can’t kill them, they go to market.”</P></DIV><DIV class="lab"><H2>Week 7</H2><H3>TUESDAY, JULY 5</H3><P>Returning from holiday, the team worked into the week with a couple of collaborative Skype calls. The first, Northwestern University, was in regard to co-hosting the iGEM North American Conference next summer. They are excited about the project and are looking into Chicago-area venues as the Purdue team investigates Indianapolis as a potential host city. Denny Luan, a founder of Experiment, the crowdfunding website utilized to raise money for lab supplies, spoke with the team on the day’s second call. An indication of Denny’s personality is given by the title of his groundbreaking study, “Qualitative survey of burritos in San Francisco.” He offered to assist with the iGEM wiki database, which is hugely exciting. Thank you for your enthusiasm, Denny!</P><H3>WEDNESDAY, JULY 6</H3><P> Mid-week lab work consisted of transformations (ppgk was successful), PCR amplification, and gels. A reporter from local WLFI 18 TV news visited the lab, as well, and the story, related to Florida’s algal blooms, aired on the nightly news. Checked out the previous post for the link.</P><H3>THURSDAY, JULY 7</H3><P>The team attended a presentation by Dr. Kevin King entitled “Production Agriculture and Environmental Targets: Can They Coexist?” He highlighted edge-of-field research aimed at quantifying the impacts of agricultural production practices and discussed potential management practices that might be used to reduce offsite nutrient transport, meeting established water quality targets. Additionally, the team is excited to announce that their submission for a SYNENERGENE Grant-Funded Collaboration proposal was selected! Collaboration with the Rathenau Institute will begin shortly.</P><H3>FRIDAY, JULY 8</H3><P>After completing a survey designed to measure public perception of wastewater treatment methods, the team gathered supplies for outreach at the Wabash River Fest the following day. Existing gels were annotated and more were run as loose ends were tied up in the wet lab.</P><H3>SATURDAY, JULY 9</H3><P>Purdue iGEM hosted a booth at the Wabash River Enhancement Corporation’s (WREC) annual Wabash River Fest. It was great fun for all parties involved as passersby learned about DNA, built (and ate) their own licorice and marshmallow double helix models, and asked questions about synthetic biology and the phosphorus cycle. Young scientists tried their hands at pipetting, mixing colored water in eppendorf tubes.</P></DIV><DIV class="lab"><H2>Week 8</H2><H3>MONDAY, JULY 11</H3><P>After a successful weekend participating in the Wabash River Fest, the team jumped back into lab work, plating a new culture of Shewie, miniprepping a promoter for use with digestion the next day, and preparing glycerol stock cultures of successfully transformed parts. A bioreactor brainstorming session with USDA-NSERL staff Dr. Ashley Hammac, Stan Livingston, and Scott McAfee left team members enthused about the prototype to come later in the week. Additionally, the team met with representatives of the Purdue Foundry regarding the possibility of developing a business model for the phosphorus removal system. If lab work continues to go well, this may be something to pursue in the fall.</P><H3>TUESDAY, JULY 12</H3><P>Wet lab work consisted of the digestion and ligation of all three remaining phosphorus genes in preparation for transformations into E. coli. Mental effort for the day was directed toward understanding E. coli‘s Pho regulon, the system of genes that control phosphorus within the organism, in an effort to determine which specific sequences of DNA may be helpful to up-regulate in the synthetic microbes to greater improve their efficiency.</P><H3>WEDNESDAY, JULY 13</H3><P> The bioreactor prototype was complete! Many thanks to the staff of the NSERL (National Soil Erosion Research Laboratory) for making it possible.

The prototype consists of two 5-gallon buckets, an aquarium pump and outflow tube to maintain constant water level and flow, and three separate ports around the base to channel water from the main reservoir into water filters that will eventually contain modified E. coli cartridges. Phosphorus is to be collected in these chambers, and effluent is controlled by the valves at the point where water flows from the filters. Further testing of the prototype will include the determination of the proper substrate to contain E. coli, optimum temperature and pH, and best water flow rate.</P><H3>THURSDAY, JULY 14</H3><P>Thursday morning began with celebration as all of the previous day’s transformations worked with multiple colonies appearing on every plate. Go, team! With all phosphorus genes successfully transformed, the team can now move forward with characterizing proteins. Along with this, some strains will be sent to other collaborating teams for characterization to confirm results.</P><H3>FRIDAY, JULY 15</H3><P>Are synthetic biology terms sometimes confusing? Is scientific jargon difficult to understand? Look for explanation no further than the team’s newly-released glossary of synthetic biology and iGEM terms hosted on the Experiment page.

With the new bioreactor prototype, several protocols needed to be written including a standard for measuring water flow and a process for system maintenance. The plan for the upcoming week is to characterize individual proteins and begin conducting tests to determine the optimal substrate for E. coli to inhabit in the bioreactor’s external chambers. Any thoughts on what to name the bioreactor? Tweet @PurdueBiomakers to let the team know what you think.</P></DIV><DIV class="lab"><H2>Week 9</H2><H3>MONDAY, JULY 18</H3><P>In collaboration with Exeter, the team will be growing up last summer’s killswitch genetic construct under both aerobic and anaerobic conditions. To begin the week, they set up Dr. Rickus’s anaerobic chamber, which is a large, clear box with thick black gloves attached to reach into it–very mad science-y (example below). This week, culture growth will begin after air tanks to sustain the anaerobic culture arrive.</P><H3>TUESDAY, JULY 19</H3><P>Mid-week discussion centered around using the sol-gel method to suspend E. coli in silica beads with which to fill containers external to the main containment area of the bioreactor. The bacteria will be immobilized within the matrix of silica fibers while phosphorus is still able to pass through, making this solution ideal for easy input and retrieval of E. coli from the bioreactor system. See the diagram below for an explanation of the method.</P><H3>WEDNESDAY, JULY 20</H3><P> Wednesday’s focus was the development of a protocol for the quantification of phosphorus both intracellularly and extracellularly. It involved the creation of a guide for phosphorus on the Experiment page, so if you want to become familiar with the differences between total phosphorus, orthophosphate, and polyphosphate, feel free to check it out.</P><H3>THURSDAY, JULY 21</H3><P>The lab hosted Peter Oladipupo, a Mandela Washington Fellow, answering questions regarding the project as it relates to the generation of energy. It was enjoyable to hear his perspective and show him the progress being made. The day also brought continued research into the development of a model to both inform experimental decisions and make future predictions.</P><H3>FRIDAY, JULY 22</H3><P>During the week, the team found out that IDT was unable to synthesize another nanowire gene, cancelling the order and decisively tabling the wet lab work for the nanowire side of the project for this competition cycle. So, full steam ahead with phosphorus. Nanowires will be revisited during the school year–strong side project potential!</P></DIV><DIV class="lab"><H2>Week 10</H2><H3>MONDAY, JULY 25</H3><P>Preliminary trials of testing for phosphorus using the lachat and the ICP-OES at the USDA National Soil Erosion Research Laboratory began. Both machines operate via spectrophotometry, comparing measured values to standard curves. The lachat measures orthophosphate, or soluble reactive phosphorus, while the ICP measures total phosphorus, the entirety of phosphorus contained in a sample. Corresponding with this endeavor, many bottles of Tris-HCl buffer, the media in which phosphorus-eating E. coli will be suspended, were created and autoclaved for future use.</P><H3>TUESDAY, JULY 26</H3><P>The week continued by learning to make methanol-free bioencapsulation beads using the sol-gel method. A graduate student in the lab worked with team members to describe and demonstrate the protocol. After practice forming the beads, the team switched to a different method that could be more easily controlled and manipulated. E. coli were added to the beads to determine whether or not they would be able to survive within the structure.</P><H3>WEDNESDAY, JULY 27</H3><P>Based on results from the lachat and ICP and the discovery that E. coli would not grow in the originally proposed Tris-HCl buffered solution, the measurement of phosphorus uptake protocol was modified after a productive meeting with advising scientists. Modified microbes will now be suspended in a minimal media solution whose components can be processed by both the sensitive lachat and ICP instruments. Prototype modeling progressed, with team member Barrett composing a 3D model to better-depict uptake system hardware. In the afternoon, a few team members also visited the West Lafayette Wastewater Treatment facility, where they learned about the local area’s methods for phosphorus removal featuring their newly-implemented chemical system.</P><H3>THURSDAY, JULY 29</H3><P>Additional trials of media and various water samples were run on both the lachat and ICP as team members were trained in their use. Colony PCR and sequencing to confirm the transformation of phosphorus genes into E. coli were also begun, with completion planned to coincide with the conclusion of next week.</P><H3>FRIDAY, JULY 30</H3><P>Focus was placed on brainstorming possible development strategies of silica beads. Two components combine to make these beads: a sol phase and a buffer (Tris-HCl). The team has been trying to find the proper ratio of these two ingredients so beads solidify at an appropriate rate–too slowly, and an amorphous silica blob forms in the bottom of the mineral oil into which the beads drop; too quickly, and the mixture hardens inside the tubing through which it is dispensed. Cool temperatures slow the reaction, so the team has also tried completing the process in the cold room with mineral oil warmed on a hot plate, but to no avail. The current method involves the use of syringes to inject the two components into a small Y-shaped tube where they mix and then pass through a needle at the bottom to drop into mineral oil. According to calculations, 99,000 beads will be needed to fill a standard water filter canister; that’s a lot of drips!</P></DIV><H1>VIRTUAL LAB NOTEBOOK</H1><H2>June 10, 2016</H2><H4>Caleigh, Barrett, and Paige</H4><P> Re-suspended IDT gBlocks 
Performed PCR amplification on DNA for PPK1 homolog part 1 and part 2, the ATP-independent PPGK, the PPGK homolog, Pit homolog A, Pit homolog B, Pit homolog C, PPK2 homolog A, PPK2 homolog B, PPK2 homolog C, and PPX2 homolog. </P><H2>June 11, 2016</H2><H4>Caleigh</H4><P>
Performed gel electrophoresis of PCR products from June 10 on a 1% agarose gel. The gel was run at room temperature and at 90 V. Bands representing the expected DNA length were present for PPK1 homolog part 2, PPX2 homolog, and PPK2 homolog C. </P><H2>June 12, 2016</H2><H4>Caleigh</H4><P>
Performed gel electrophoresis of PCR products on 1% agarose gel. The gel was run at room temperature and at 85 V. Bands representing the expected DNA length were present for the ATP-independent PPGK, Pit homolog A, Pit homolog B, Pit homolog C,  PPK2 homolog B, PPK2 homolog C, and PPX2 homolog.</P><H2>June 14, 2016</H2><H4>Caleigh</H4><P>
Mini-prepped five different cultures of E. coli to extract the plasmid with Anderson promoter J23106. Reinoculated the Anderson promoter in LB broth due to low concentrations below 85 ng/µL. </P><H2>June 15, 2016</H2><H4>Caleigh</H4><P>
Mini-prepped two different cultures of E. coli to extract the plasmid with Anderson promoter J23106. </P><H2>June 16, 2016</H2><H4>Caleigh</H4><P>
Performed transformations for both the 3:1 and 6:1 (insert:vector) ligations of Pit homolog A, Pit homolog B, Pit homolog C, PPK2 homolog B, PPK2 homolog C, ATP-independent PPGK, PPGK homolog, and PPX2 homolog. All results came back negative.
Performed gel electrophoresis of PCR products on 0.67% agarose gel. PCR products included Pit homolog A, Pit homolog B, Pit homolog C, PPK1 homolog part 1, PPK1 homolog part 2, PPK2 homolog B, PPK2 homolog C, ATP-independent PPGK, PPGK homolog, and PPX2 homolog. Gel was run at room temperature and 85 V. All results came back negative.</P><H2>June 18, 2016</H2><H4>Caleigh</H4>
Performed gel electrophoresis of PCR products on 1.5% agarose gel. PCR products included Pit homolog A, Pit homolog B, Pit homolog C, PPK1 homolog part 1, PPK1 homolog part 2, PPK2 homolog B, PPK2 homolog C, ATP-independent PPGK, PPGK homolog, and PPX2 homolog. Gel was run at room temperature and 85 V. Bands indicating product were found for PPK1 homolog part 2, PPK2 homolog C, PPGK homolog, and PPX2 homolog. <H2>June 20, 2016</H2><H4>Paige and Caleigh</H4><P>Performed gel extraction of DNA fragments from gel electrophoresis run on June 18, including ATP-independent PPGK, Pit homolog A, PPX2 homolog, and PPK2 homolog C. Concentrations of DNA obtained were 4.5 ng/µL, 3.9 ng/µL, 9.3 ng/µL, and 6.4 ng/µL, respectively, which were below a viable level of 10 ng/µL.</P><H2>June 21, 2016</H2><H4>Caleigh, Paige, and Emma</H4><P>Digested Anderson promoter J23106, Pit homolog A, Pit homolog B, Pit homolog C, PPK2 homolog B, PPK2 homolog C, PPGK homolog, PPGK homolog, PPX2 homolog, and PSB1A3 for 3A assembly.
<LI>Ligated the Anderson promoter and each of the eight individual genes listed above into PSB1A3 in the ratios 6:1, 3:1, and 1:3. (part:vector).
</LI></P><H2>June 22, 2016</H2><H4>Caleigh</H4><LI>Transformed the 6:1, 3:1, and 1:3 ligations from yesterday into E. coli. A bacterial lawn was present on both the negative control and all transformation plates except for the 3:1 ligation of Pit homolog A and the ATP-independent PPGK and the 6:1 ligation of Pit homolog A. We determined that a bad stock of ampicillin was used in our LB-ampicillin agar plates. 
<H2>June 24, 2016</H2><H4>Caleigh</H4></LI><LI>Transformed the 6:1 and 1:3 ligations from June 21 into E. coli. Bacterial growth was only present on the positive control, and all the transformation results came back negative.
<H2>June 26, 2016</H2><H4>Caleigh</H4></LI><LI>Digested Anderson promoter J23106, the ATP-independent PPGk, and PSB1A3 for 3A assembly.
</LI><LI>Ligated J23106, ATP-independent PPGK, and PSB1A3 together.
<H2>June 28, 2016</H2><H4>Caleigh</H4></LI><LI>Inoculated two samples of E. coli with the plasmid PBR322 and one sample of E. coli with the plasmid PACYC4 into LB.
<H2>June 29, 2016</H2><H4>Caleigh</H4></LI><LI>Performed minipreps on two cultures of  E. coli containing the plasmid PBR322 and one sample of PACYC4. Due to low concentrations below 45 ng/µL, the colonies containing these plasmids were reinoculated.
<H2>June 30, 2016</H2><H4>Caleigh</H4></LI><LI>Performed minipreps on two cultures of E. coli containing the plasmid PBR322 and one sample of PACYC4. Concentrations were satisfactory at above 200 ng//µL.
<H2>July 1, 2016</H2><H4>Caleigh</H4></LI><LI>Performed gel electrophoresis of PCR products for Pit homolog A, Pit homolog C, PPK1 homolog part 1, PPK1 homolog part 2, PPK2 homolog C, and PPGK homolog on a 1% agarose gel. The gel was run at room temperature at 90 V. Bands indicating product were present for Pit homolog A, Pit homolog C, and PPK2 homolog C. Gel was then stored at 4°C.
<H2>July 3, 2016</H2><H4>Caleigh</H4>
Performed gel extraction for Pit homolog A, Pit homolog C, and PPK2 homolog C from gel from July 1. 
Performed PCR on g-blocks for Pit homolog A, Pit homolog B, Pit homolog C, PPK1 homolog part 1, PPK1 homolog part 2, PPK2 homolog A, PPK2 homolog B, PPK2 homolog C, PPGK homolog, PPX2 homolog, DmsE, and FccA part 2. 
Performed gel electrophoresis of PCR products listed above. Bands indicating product were present for Pit homolog B, PPK2 homolog A, PPK2 homolog B, PPK2 homolog C, PPGK homolog, and PPX2 homolog on a 1% agarose gel. The gel was run at room temperature and at 90 V. 
Performed gel extraction of the six PCR products.
<H2>July 4, 2016</H2><H4>Barrett</H4></LI><LI>Performed digestion and ligation (both 3:1 and 5:1) on the Anderson promoter J23106, six of our phosphorus-related genes, and PSB1A3.<H2>July 5, 2016</H2><H4>Caleigh</H4></LI><LI>Transformed the six phosphorus-gene constructs ligated yesterday into <I>E. coli</I>, . The 3:1 ligation was used to transform three samples, and the 5:1 ligation was used to transform the other three.<H2>July 6, 2016</H2><H4>Emma</H4></LI><LI>Transformed the three remaining 3:1 ligations for the phosphorus-gene constructs into <I>E. coli</I></LI><LI>Performed gel extraction on PPK1 homologue part 1 and Pit A. 
</LI><LI>Performed restriction enzyme digestion and ligation on Pit A for 3A assembly
</LI><LI>Inoculated PPK2 homologs B and C into Lb broth.<H2>July 7, 2016</H2><H4>Caleigh</H4></LI><LI>Prepared glycerol stock cultures for PPK2 homolog A, Pit B, PPX2, and PPGK ATPI.</LI><LI>Transformed previous ligation of Pit A into <I>E. coli</I><H2>July 7, 2016</H2><H4>Caleigh</H4></LI><LI>Prepared glycerol stock cultures for PPK2 homolog A, Pit B, PPX2, and PPGK ATPI.</LI><LI>Transformed previous ligation of Pit A into <I>E. coli</I><H2>July 10, 2016</H2><H4>Barrett</H4></LI><LI>Performed gel electrophoresis on Gibson assemblies for PPK1.
</LI><LI>Extracted DNA fragments from gels and obtained 8.7 ng/µL and 88.0 ng/µL DNA concentrations for new and old Gibson assemblies 
</LI><LI>Performed PCR amplification on several phosphorus genes (again), and a 1ng/µL dilution of 8.7ng/µL Gibson assembly product. 0.5µL MgCl2 was not added to this PCR reaction, and it ran for 20 cycles.
</LI><LI>Began performing gel electrophoresis on PCR products.
</LI><LI>Extracted DNA concentrations and obtained results of usable concentration, greater than 20 ng/µL, averaging 31 ng/µL.
<H4>Paige and Bowman</H4></LI><LI>
Transformed Anderson promoter (J23106) into E. coli and stored on middle shelf of incubator for overnight growth on ampicillin plates.<H2>July 11, 2016</H2><H4>Barrett</H4></LI><LI>
Successfully finished gel extractions from night before<H4>Paige and Bowman</H4></LI><LI>
Generated glycerol stock of PPK2 homolog B (stored in -80)
</LI><LI>Replated PPK2 homolog B and grew overnight
</LI><LI>Inoculated Anderson promoter
</LI><LI>Performed miniprep of Anderson promoter<H2>July 12, 2016</H2><H4>Barrett and Caleigh</H4><P>Ligated Pit homolog A, Pit homolog C, and PPK1 homolog</P><H4>Emma</H4><P>Measured OD of InterLab bacteria to determine growth rate curves</P><H4>Paige and Barrett</H4><P>Digested Pit homolog A, Pit homolog C, and PPK1 homologue (Gibsoned part)</P><H2>July 13, 2016</H2><H4>Emma</H4><P>Completed transformations of Pit homolog A, Pit homolog C, and PPK1 homologue.</P><H2>July 14, 2016</H2><H4>Paige</H4><P>Inoculated Pit homolog A, Pit homolog C, and PPK1 homolog (1:1) cultures</P><H2>July 15, 2016</H2><H4>Caleigh</H4><P>Made stock cultures for colonies from the Pit A, Pit C, and PPK1 homolog transformations</P><H2>July 26, 2016 </H2><H4>Paige</H4><P>Practiced a phosphorus extraction assay run for Lachat and ICP machines for the quantification of extracellular and intracellular phosphorus.
Practiced sol-gel method of creating silica beads for E. coli bioencapsulation.</P><H2>July 27, 2016</H2><H4>Paige</H4><P>Lysed E. coli for the extraction of intracellular polyphosphate in samples to quantify phosphorus using the Lachat and ICP.
Analyzed phosphorus content of samples, establishing a baseline for comparison of future experimental results.</P><H2>August 1, 2016</H2><H4>Paige</H4><P>Completion of Lachat and ICP protocol for the determination of phosphorus concentration in newly made Tris-HCl buffered solution</P><H2>August 2, 2016</H2><H4>Paige</H4><P>Performed a Neisser stain with Bismarck Brown and Crystal Violet dyes on Pit homologue B, PPGK, PPK2 homolog A, PPK2 homolog B, PPK2 homolog C, and PPX2 to determine intracellular poly-phosphorus.</P><H2>August 4, 2016</H2><H4>Paige</H4><P>Added 30 mg KCl, MgCl2, NaCl, NH4Cl, FeCl3, and CaCl2 to Tris-HCl buffered solution to create minimal media to ensure E. coli survival</P><H2>September 24, 2016</H2><H4>Bowman</H4><P>Performed miniprep of successfully transformed PPK2 homolog C</P><H2>September 25, 2016</H2><H4>Paige and Caleigh</H4><P>Ran 6-hour phosphorus uptake assay with PPGK ATP-independent, Pit homolog B, PPX2, PPK2 homolog A, PPK2 homolog C, and unmodified E. coli, collecting samples every hour from 0 to 6 hours. 
Completed Lachat and ICP analysis of samples for phosphorus content.</P><H4>Bowman</H4><P>Completed Gibson assembly of PPK1 homolog parts 1 and 2.
Digested all phosphorus parts.
Ligated in ratios of 1:1 and 1:3 using combined PPK1 homolog with phosphorus parts using miniprepped PPK2 homolog C for promoter digest and recently assembled PPK1 homolog with a chloramphenicol backbone.
Transformed ligated parts into E. coli</P><H2>October 9, 2016</H2><H4>Bowman</H4><P>Transformed the 3A ligation of Anderson promoter J23106 with each of the following genes:  Pit homolog A, Pit homolog B, Pit homolog C, PPK2 homolog A, PPK2 homolog B, PPK2 homolog C, ATP-independent PPGK, PPGK homolog, and PPX2 homolog. All backbones were PSB1C3. Pit homolog A, Pit homolog B, PPK2 homolog C, ATP-independent PPGK, PPGK homolog, and PPX2 homolog were all successful. Transformations were done into E. coli.
Inoculated transformations of the temperature sensitive promoter K608351 and tsPurple reporter (RBS+coding+terminator) K1357008 into LB-chloramphenicol.
Prepare minimal media. </P><H2>October 10, 2016</H2><H4>Caleigh</H4><P>Inoculated successful transformations from October 9 into LB- chloramphenicol. 
Performed minipreps on plasmids for the temperature sensitive promoter K608351 and tsPurple reporter (RBS+coding+terminator) K1357008. Both concentrations were satisfactory. 
Ran the six-hour phosphorus uptake assay for two samples each of unmodified E. coli, E. coli containing the PPK2 homolog C in PSB1A3, and E. coli containing the PPX2 homolog in PSB1A3.
Digested temperature sensitive promoter K608351, tsPurple reporter (RBS+coding+terminator) K1357008, and PSB1K3 for 3A assembly.
Ligated temperature sensitive promoter K608351 and tsPurple reporter (RBS+coding+terminator) K1357008 into the backbone PSB1K3.
Ran the six-hour phosphorus uptake assay for the successful transformations of E. coli for the genes Pit homolog A, Pit homolog B, PPK2 homolog C, PPGK homolog, ATP-independent PPGK, and PPX2 homolog. </P><H2>October 11, 2016</H2><H4>Caleigh</H4><P>Inoculated unmodified E. coli into LB broth, all six successful transformations in PSB1C3 into LB-chloramphenicol, and the two sequence-verified transformations into PSB1A3 (PPK2 homolog C and PPX2 homolog) into LB-ampicillin. K2PO4 was added to make all broths 2 mM K2PO4.
Transformed  the temperature sensitive promoter K608351 with the tsPurple reporter (RBS+coding+terminator) K1357008 into E. coli.
Performed two minipreps for the successful transformations for Pit homolog A, Pit homolog B, PPK2 homolog C, PPGK homolog, ATP-independent PPGK, and PPX2 homolog. 
Ran the six-hour phosphorus uptake assay for the successful transformations of E. coli for the genes Pit homolog A, Pit homolog B, PPK2 homolog C, PPGK homolog, ATP-independent PPGK, and PPX2 homolog for a second trial. </P><H2>October 12, 2016</H2><H4>Caleigh</H4><P>Centrifuged samples from phosphorus assays. Separated supernatant and cellular component and resuspended cellular component in double distilled water.
Submit plasmids from successful transformations into PSB1C3 for sequencing.</P><H2>October 12, 2016</H2><H4>Caleigh</H4><P>Prepared standard curve for quantification of intracellular polyphosphate using sodium phosphate glass 45 and toluidine blue.
Extracted and quantified intracellular polyphosphate for unmodified E. coli, all six successful transformations in PSB1C3, and the two sequence-verified transformations into PSB1A3 (PPK2 homolog C and PPX2 homolog).
Inoculated numerous colonies from 6 transformations into PSB1C3 in LB-chloramphenicol </P><H2>October 17, 2016</H2><H4>Paige</H4><P>
Lysed inoculated cells in LB broth for the purpose of protein extraction. Cells lysed include negative control E. coli, PPX2 in ampicillin backbone, PPX2, PPGK ATPI, and PPK2 C in ampicillin backbone.</P><H4>Caleigh</H4><P>Purified proteins extracted by Paige using nickel columns</P></LI></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>