<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Oxford_Results_Protein skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Oxford/Results Protein</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav navbar-right"><LI id="home-link"><A href="https://2017.igem.org/Team:Oxford">HOME</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PROJECT
            </A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Chagas_Disease">Chagas Disease</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Chagas_Public_Policy">Chagas Public Policy</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Applied_Design_Developing">--Developing our Design</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Applied_Design_Solution">--Our Solution</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Applied_Design_Implementation">--Implementation</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Cell_Free_Report">Cell Free Report</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/InitialIdeas">Initial Ideas</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">WET LAB
            </A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Overview_Wet_Lab">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Cloning">Results (Cloning)</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Results_DNA">Results (DNA-Based system)</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Results_Protein">Results (Protein-Based system)</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Future_experiments">Future Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/InterLab">Interlab</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Measurement">Measurement</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">DRY LAB
            </A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/DNA_Based_Model">DNA-Based Model</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Protein_Based_Model">Protein-Based Model</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Disease_Model">Disease Model</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Software">Software</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">HUMAN PRACTICES
            </A></LI><LI><A href="https://2017.igem.org/Team:Oxford/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/HP/Gold_Integrated">Integrated and Gold</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Engagement">Public Engagement</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PEOPLE
            </A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Team">Meet the Team</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Sponsors">Sponsors</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Awards">Our Awards</A></LI></DIV></DIV><DIV class="container"><H1>Results (Protein-Based System)</H1><H2> Introduction </H2><P>In order to demonstrate the feasibility of our protein-based system for detecting cruzipain, we designed and cloned  three parts into the pSB1C3 vector:<OL><LI>SpyTag-OmpA-sfGFP-His tag <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2450401"> BBa_K2450401</A></LI><LI>OmpA-SpyTag-sfGFP-His tag <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2450451"> BBa_K2450451 </A></LI><LI>TorA Leader-SpyCatcher-sfGFP-TEV cleavable linker-Reach quencher-His tag <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2450501"> BBa_K2450501 </A></LI></OL></P><P id="OMV_modelling">From our modelling of this system we knew that the final concentration of our proteins was important, and therefore packing efficiency into the OMVs was vital. Therefore, we attached sfGFP to both OmpA parts (BBa_K2450401 and  BBa_K2450451)  to measure the efficiency of their transport to the outer membrane and OMVs. We also designed the BBa_K2450501 part with the dark quencher near the end of the part. This would allow the quencher to easily be removed by PCR and so we would be able to assay the colocalization of this part to the outer membrane and OMVs. This value would be important to determine protein concentration. It would also be able to inform on the ratio of bound to free protein in the OMVs by performing an assay on the fluorescence in OMVs with and without a targeting OmpA part, which was another unknown variable in the modelling. 
      </P><H2> Shipping Vector Cloning </H2><P>A more in-depth description of this stage of our project can be found on the Shipping Vector Cloning <A href="https://2017.igem.org/Team:Oxford/Cloning">results</A> page. We successfully cloned the three parts above into the shipping vector. We had designed two additional parts, two halves of a split TEV protease, however these contained illegal restriction sites. We attempted quick-change PCR and overlap extension PCR for site-directed mutagenesis, and the two protocols are on our page, however these were unsuccessful. We therefore could not submit these parts to the registry.
    </P><H2>Expression Vector Cloning</H2><P>For full characterisation of both parts we needed to express both of them in the same system, so we designed an experiment where OmpA-SpyTag and SpyCatcher-GFP-quencher would be present on different plasmids in the same cell. We would clone the OmpA-SpyTag part into the pQE-60 vector, and the SpyCatcher-GFP-Quencher part into pBAD-33. These have compatible origins of replication, differing antibiotic resistances, and are induced by two different and easily-obtainable inducers, IPTG and arabinose respectively.</P><P>We designed primers for three purposes:</P><P>To amplify the parts from the shipping vector constructs with restriction sites that allowed for cloning into the expression vectors. These were <I>NcoI</I>I and <I>Bam</I>HI for pQE-60 and <I>Xba</I>I and <I>Pst</I>I for pBAD-33. These enzyme combinations ensured that the start codon of the part was the optimal distance from the RBS in the plasmid for efficient expression. </P><LI><P>To amplify a ‘noF’ version of the OmpA-SpyTag part which would remove the sfGFP fluorophore and therefore fluorescence would only be detected in the OMV fraction if the SpyCatcher-sfGFP-Quencher part was localised. In hindsight a better design may have been to have used a different fluorophore to assay both at once. This could even have been a CFP in order to use FRET to see if the two parts were close to one another.</P></LI><LI><P>To amplify a ‘noQuencher’ version of the SpyCatcher-sfGFP-Quencher part, in order to determine the effectiveness of quenching in our system.</P></LI><H3>Planned Plasmid Constructs</H3><DIV class="row"><DIV class="col-sm-4"><H6> Plasmid Map of BBa_K2450401 in pQE60 </H6></DIV><DIV class="col-sm-4"><H6> Plasmid Map of BBa_K2450401 without sfGFP in pQE60 </H6></DIV><DIV class="col-sm-4"><H6> Plasmid Map of BBa_K2450451 in pQE60 </H6></DIV></DIV><DIV class="row"><DIV class="col-sm-4"><H6> Plasmid Map of BBa_K2450451 without sfGFP in pQE60 </H6></DIV><DIV class="col-sm-4"><H6> Plasmid Map of BBa_K2450501 in pBAD33 </H6></DIV><DIV class="col-sm-4"><H6> Plasmid Map of BBa_K2450501 in pBAD33 without the quenching peptide </H6></DIV></DIV><P>We successfully cloned the SpyCatcher-GFP-Quencher part into pBAD-33, however were unable to clone the OmpA-SpyTag part into pQE-60. This limited the experiments we were able to do with OMVs. However we were able to characterise the quencher, ensuring that if future teams can express the two parts together they will be able to use the finished system as an assay for protease presence. This can either be for the presence of protease which has been localised to OMVs or for the lysis of OMVs, by adding protease to the media they are in. </P><H2 id="SC_sfGFP_Q">SpyCatcher-sfGFP-Quencher Characterisation</H2><DIV class="row"><DIV class="col-sm-6"><H3>Inital Microscopy to Show Fluorescence</H3><P>The first experiments we ran aimed to show that sfGFP was produced, and that the quencher worked to reduce fluorescence. Initial microscopy showed that when the quencher was removed from the part the cells did exhibit fluorescence. We were unfortunately unable to see whether the TorA leader sequence had worked as planned with the part. We were expecting to see the fluorescence transfer to the periplasm, which would have manifested as a ‘halo’ around the cell. </P></DIV><DIV class="col-sm-6"><H6>Figure 1: Microscopy to qualitatively detect the expression of sfGFP, the protocol for this can be found on our protocols page.  </H6></DIV></DIV><DIV class="row"><DIV class="col-sm-6"><H6>Figure 2: Graph to show efficacy of quencher. Cells were grown overnight in minimal media (M9) in a plate reader and sfGFP was measured. Fluorescence was corrected against a blank of M9 and then the starting fluorescence was subtracted from end fluorescence to give absolute response. </H6></DIV><DIV class="col-sm-6"><H3>Plate Reader Assay of Quencher</H3><P>Next, we ran a plate reader assay comparing the fluorescence of the quenched and unquenched part. Induction with arabinose to produce the part lead to an almost 10-fold increase in fluorescence in the noQuencher version of the part, showing a quenching efficiency of 90% under these conditions.</P></DIV></DIV><DIV class="row"><DIV class="col-sm-6"><H3><I>In vitro</I> Cleavage Assay </H3><P>To characterise relief of the quenching by cleavage with TEV protease we aimed to combine purified sfGFP+Quencher and TEV protease in wells and let it run in the plate reader, to see relief of quenching. Originally we attempted to do this <I>in vitro</I>, by purifying the part and then adding purified TEV protease. We were helped greatly in this regard by Associate Prof. Maike Bublitz, from the Biochemistry department, who provided us with purified TEV protease and guided us on how to purify the protein using a nickel column, which was kindly lent to us by Prof. Matt Higgins, also from the department.</P></DIV><DIV class="col-sm-6"><H6>Figure 3: SDS-PAGE gel from purification, the ladder used was Benchmark Protein Standard from Novex. Lanes 2 and 4 show wash fractions, 3 supernatant, and 5-8 elution fractions. The part was expected at a Mw of 74kDA</H6></DIV></DIV><DIV class="row"><P>It appeared from our SDS-PAGE gel, however, that we may have used too strong a wash for our purification, and washed our protein from the column before we got to our elution fractions. We tried some of the fractions in the plate reader to be sure, however there was no increase in sfGFP over time.</P></DIV><DIV class="row"><H3><I>In vivo</I> Cleavage Assay </H3></DIV><DIV class="row"><P>Following this setback, we instead decided to characterise the cleavage <I>in vivo</I>. A member of the Biochemistry department, Matteo Ferla, kindly gave us a plasmid with TEV protease which was compatible with our part, and was IPTG-inducible. We co-transformed it with our plasmid and induced subcultures from an overnight growth with varying IPTG levels. We used three biological and three technical repeats and used a BMG CLARIOSTAR platereader to obtain sfGFP readings over the course of 10 hours in order to obtain end-point fluorescence and dynamics of the reaction. </P></DIV><DIV class="row"><DIV class="col-sm-6"><H6>Figure 4: End-point analysis of induction and cleavage. Cells were grown overnight in M9 media and subcultured down before being induced with arabinose at 0.1% w/v and IPTG. They were then allowed to run in a BMG CLARIOSTAR platereader for 10 hours. Three biological and technical repeats were averaged and the SEM used to make the error bars.</H6></DIV><DIV class="col-sm-6"><P>After removing anomalous results and calculating the average and standard error, we were able to plot a graph comparing the non-quenched strain, quenched strain, and quenched-strain with the TEV-protease plasmid co-transformed. The end-point analysis showed that the induction of the TEV protease did indeed relieve quenching of the fluorescence. The levels of unquenched and Quenched+TEV protease did not differ significantly for IPTG levels of over 5µM, and even at 0µM the level of fluorescence was significantly higher in the strain with the TEV plasmid. These observations can be explained by analysing the dynamics using the time-course graph.</P></DIV></DIV><DIV class="row"><DIV class="col-sm-6"><P>We used the statistical F-test (Figure 5) to show that there was a significant difference between Flu/Abs of the sfGFP+Quencher and the sfGFP+Quencher+TEV protease; by an F value of 26.3 which is larger than the critical value of 0.000124. This suggests that the TEV protease is able to relieve the quenching by cleavage of the linker between the sfGFP and the quencher.</P><P>This test has also shown a significant difference in Flu/Abs when the quencher is present compared to the no Quencher version of the part, as the F value is 58.7 which is larger than the critical value of 1.47x10-6.</P></DIV><DIV class="col-sm-6"><H6>Figure 5: ANOVA statistical tests to show the significance of our data.</H6></DIV></DIV><DIV class="row"><P>As can be seen in Figure 6 below, until 200 minutes the strains appear to be acclimatising, and then induction begins to happen, this section of the graph is expanded below. From this graph it is clear that the level of induction does have an effect. The steady-state level of TEV protease produced is presumably different, and this leads to different rates of increase of sfGFP. Induction with only 5µM IPTG leads to a higher difference in fluorescence at 400 than at 600 minutes. The uninduced strain does not increase above the quenched strain until after 400 minutes, when it begins to climb. One interesting thing to consider is if the level of the sfGFP-quenching peptide reaches steady state in the cell, as can possibly be seen from the fluorescence of the non-quenched strain. This would mean that there is a maximum possible fluorescence level, and that over time the endpoints of all three induction levels will be equal.</P></DIV><DIV class="row"><H6>Figure 6: Graphs showing the time-course of induction and cleavage. Conditions were the same as for Figure 4, above. The second graph shows error bars in dotted lines above and below each line, in the same colour.</H6></DIV><H2> Summary and Conclusions </H2><H3>Our Results</H3><UL><LI> We cloned three parts into the pSB1C3 shipping vector </LI><LI><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2450401"> BBa_K2450401</A> - OmpA-SpyTag-sfGFP(SpyTag N-terminus of OmpA) </LI><LI><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2450451"> BBa_K2450451 </A> - OmpA-SpyTag-sfGFP (SpyTag C-terminus of OmpA) </LI><LI><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2450501"> BBa_K2450501 </A> SpyCatcher-sfGFP-Quencher </LI></UL><LI> We cloned BBa_K2450401 with and without the quenching peptide into pBAD-33, our expression vector of choice </LI><LI>We showed that the sfGFP was produced though microscopy and plate reader experiments </LI><LI> We determined that the quenching peptide was effective at quenching fluorescence by comparing the two parts in pBAD33 </LI><LI>We showed that this quenching could be relieved by cleavage by the TEV protease by co-transforming another plasmid into the same strain and inducing both BBa_K2450401 and the TEV protease </LI><P>It was unfortunate that we could not characterise the TorA leader, OmpA, or SpyTag-SpyCatcher system, however due to the design of the system we needed both parts to be induced simultaneously to get this characterisation. Our difficulties in cloning the OmpA-SpyCatcher part into an expression vector prevented this.</P><H3> Improving Characterisation of an Existing Part </H3><P>The <A href="https://2014.igem.org/Team:Aachen"> Aachen 2014 iGEM team </A>  from which we took the GFP-TEV cleavage-Reach2 section of our part showed in their system that the quencher worked and that it could be relieved by cleavage by the TEV protease. We have <A href="http://parts.igem.org/Part:BBa_K1319002#Additional_Characterisation">added to this characterisation</A> by comprehensively showing that the same effects occur even with: </P><OL><LI> The addition of a His tag onto the quenching peptide </LI><LI> The addition of upstream leader sequences (TorA and SpyTag) </LI><LI> The substitution of GFP for sfGFP </LI></OL><P>We have therefore taken another step towards the development of an assay for protease cleavage in OMVs. This could either be an assay to check for the localisation of a protease to the OMV or to see if the vesicles have been lysed by adding the protease to the media.</P></DIV><DIV class="container-fluid" style="margin-top: 80px"><DIV id="footer-row" class="row"><DIV id="footer-sponsor" class="col-sm-8"><P>Sponsors</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>