<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Oxford_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Oxford/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav navbar-right"><LI id="home-link"><A href="https://2017.igem.org/Team:Oxford">HOME</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PROJECT
            </A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Chagas_Disease">Chagas Disease</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Chagas_Public_Policy">Chagas Public Policy</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Applied_Design_Developing">--Developing our Design</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Applied_Design_Solution">--Our Solution</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Applied_Design_Implementation">--Implementation</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Cell_Free_Report">Cell Free Report</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/InitialIdeas">Initial Ideas</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">WET LAB
            </A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Overview_Wet_Lab">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Cloning">Results (Cloning)</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Results_DNA">Results (DNA-Based system)</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Results_Protein">Results (Protein-Based system)</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Future_experiments">Future Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/InterLab">Interlab</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Measurement">Measurement</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">DRY LAB
            </A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/DNA_Based_Model">DNA-Based Model</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Protein_Based_Model">Protein-Based Model</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Disease_Model">Disease Model</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Software">Software</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">HUMAN PRACTICES
            </A></LI><LI><A href="https://2017.igem.org/Team:Oxford/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/HP/Gold_Integrated">Integrated and Gold</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Engagement">Public Engagement</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PEOPLE
            </A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Team">Meet the Team</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Sponsors">Sponsors</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Awards">Our Awards</A></LI></DIV></DIV><DIV class="container"><H1>Project Design</H1><H2>Introduction</H2><P>It was important to us that our design was as simple and elegant as possible. To effectively diagnose Chagas disease, we needed a system that could be performed in a field setting. This eventually pushed us in the direction of a cell-free system, inspired partly by the work of Keith Pardee in diagnosing the Zika virus cell-free.</P><P>Synthetic biology presents many possible paths, and we chose to investigate two of them: one DNA-based, one protein-based. The DNA-based system uses the TetR repressor; the protein-based system uses OmpA and SpyTag/SpyCatcher to tether proteins to the outer membrane. 
    </P><H2>Background</H2><P>There are many issues associated with developing a diagnostic for a neglected tropical disease. The method has to be (1) effective, (2) affordable, and (3) viable to perform and store within existing infrastructure. Additionally it needs to be accepted by the local community as a reliable and safe method for them to use. Currently, diagnostics for congenital Chagas disease do not sufficiently meet these requirements. For more information visit our <A href="https://2017.igem.org/Team:Oxford/Chagas_Disease">Chagas disease page</A>. </P><P>Synthetic biology is a currently underutilised as an option for diagnostics. Many tests use enzyme linked immunoassays (ELISAs), which are expensive and difficult to conduct in the field. They also rely on having common molecular epitopes, which may not be present for some diseases. Synthetic biology allows us to detect the function of biomolecules, rather than the structure which is detected by antibodies, making it more consistent and harder to evolve against. It can also give scope for smaller, faster, and cheaper detection. We conducted a survey of the general public and found that a majority were in favour of the development of a diagnostic using synthetic biology. For more information visit our <A href="https://2017.igem.org/Team:Oxford/HP/Silver">Silver Human Practices page</A>.
    </P><H2>Cruzipain as a biomarker</H2><P>Most current tests for Chagas disease focus on detecting the whole parasite, <I> T. cruzi </I>, its DNA, or antibodies produced in response to its biomarkers. We looked at a traditionally difficult biomarker, cruzipain, a specific cysteine protease. Most tests have failed to use it because it has many isoforms and no stable epitope, demonstrating the flaws in relying on detecting the structure of biomolecules in disease diagnostics. However, given the conserved cleavage sequence of its substrates and frequency of other pathogen associated proteases we decided cruzipain was ideal to use in our synthetic biology system. Cruzipain is secreted by the parasite, and as the parasite is present in relatively high concentrations during the acute phase of the disease, it follows that there are likely to be relatively high concentrations of cruzipain present in the blood making detection feasible. It is also highly specific to <I> T. cruzi </I>, reducing likelihood of false positives. By looking at the protease directly we also overcome a traditional problem which is diagnosis of newborn babies, who do not have fully developed immune systems and can therefore not use immunological tests. This developed our focus on congenital diagnosis.</P><P>Ultimately, we designed and investigated two systems to detect it - one DNA-based; one protein-based.</P><DIV class="row"><DIV class="col-sm-7"><H2>Design-Build-Test cycle</H2><P><A href="https://2011.igem.org/Team:Imperial_College_London/Tour">We applied the engineering cycle from Imperial 2011 to our project</A>. The cycle shows how interactions with modellers, consumers, and the public, contribute to the development and wet lab integration of the project.  The constant dialogue enabled by the cycle is key to creating a product that matches the needs of the consumer as well as being scientifically sound. Crucially, it  allows for iterative improvement of the device.</P></DIV><DIV class="col-sm-5"><H6>Figure 1: Imperial 2011's Engineering Cycle</H6></DIV></DIV><H2>First DBT cycle</H2><P>Originally we envisioned producing the anticoagulant peptide directly from our system. However, results from our <A href="https://2017.igem.org/Team:Oxford/DNA_Based_Model#Production_of_hirudin">modelling data</A> suggested that this would not be fast enough to prevent blood clotting. Thus, we needed an amplification system to increase the speed of anticoagulant production and we settled on doing this by releasing it from a sterically hindered system by cleavage by TEV protease. From there, we considered the options of how we could control the levels of TEV protease in the system.</P><H2>Click on the buttons below to jump to DNA-system design, protein-system design, and output design.</H2><DIV class="row"><DIV class="col-sm-5"><CENTER><A href="#C1"><SPAN>DNA-based System</SPAN></A></CENTER></DIV><DIV class="col-sm-3"><CENTER><A href="#C2"><SPAN>Protein-based System</SPAN></A></CENTER></DIV><DIV class="col-sm-4"><CENTER><A href="#C3"><SPAN>Output</SPAN></A></CENTER></DIV></DIV><H2 id="C1">DNA-Based System</H2><P>To detect cruzipain, its proteolytic function should lead to a detectable output which could be either positive or negative. For our DNA-based system, in which mRNA transcription is being controlled  this would require either using an activating or repressing transcription factor. Cleavage by the protease to allow an activator to work would be a positive output; cleavage to disable a repressor would be a negative output. Inability to repress an activating transcription factor would cause some basal expression that would reduce sensitivity of our output, therefore we chose to investigate repressor proteins which we could edit to be inactivated by cleavage with cruzipain.</P><P>After researching several options, we decided on the TetR repressor, as it was shown to be a very strong repressor, and had an easily accessible linker region between its two domains which we could modify to have a cruzipain recognition sequence.</P><H6>Figure 2: TetR monomer, showing domains</H6><P>TetR acts as a dimer, and each monomer has a DNA binding domain and a dimerisation domain, consequently the Tet promoter has two TetR binding sites, both needed for full repression of transcription. We took the DNA sequence of TetR Class B from BBa_K106669 and located the linker region using PyMOL and PDB entry 3zqi. The linker region was long enough to modify into a protease recognition sequence.</P><P>Due to the <A href="https://2017.igem.org/Team:Oxford/Safety">dangers of working</A> with a protein from a pathogen, we decided to use a common laboratory protease instead to model our system and provide a proof-of-concept. Therefore, the TetR part we have designed (<A href="http://parts.igem.org/Part:BBa_K2450201">BBa_K2450201</A>) has a Tobacco Etch Virus (TEV) protease recognition site instead of a cruzipain recognition site. The TEV protease is well-characterised, very specific, and can be expressed in simple bacterial systems, making it ideal for testing our sensor.</P><H6>Figure 3: PyMOL image showing the mutated bases in the linker region</H6><P>We also believe that this version of TetR provides more versatility for relieving repression of a gene or other DNA sequence, as an alternative to adding tetracycline. 

</P><P>As a control, we also created a part with an unmodified TetR repressor (<A href="http://parts.igem.org/Part:BBa_K2450251">BBa_K2450251</A>), which would give no output when the TEV protease is added. This would enable us to understand the background rate of transcription and consequently the rate of false positive diagnoses from the repressor being ‘leaky’. Both TetR parts were created with CFP as fusion proteins so we could visualise them in a fluorescence plate-reader.</P><P>In addition to the TetR parts, we also created auxiliary parts to test them. We have designed a non-self-cleavable TEV protease with an mCherry tag (<A href="http://parts.igem.org/Part:BBa_K2450101">BBa_K2450101</A>), and a DNA part which contains a medium-strength tet operator and eYFP (<A href="http://parts.igem.org/Part:BBa_K2450101">BBa_K2450301</A>).</P><P>The design of the pTet-RBS-eYFP is essential to having an appropriate balance between false negative and false positives for our diagnostic device. Our <A href="https://2017.igem.org/Team:Oxford/Cell_Free_Model">modelling</A> showed that we wanted to use a strong RBS so that we could produce our output at a rate which was fast enough to prevent blood from clotting, and in a time frame that was appropriate for a point-of-use diagnostic. Thus we chose an RBS, <A href="http://parts.igem.org/Part:BBa_B0030">BBa_B0030</A>, that was reported to have the kinetics to fit these parameters of our model, as seen in the registry and confirmed by additional teams.</P><P>The combination of CFP, mCherry, and YFP was chosen because they have very limited overlap in their absorption and emission frequencies, meaning it was easier to confidently quantify them relative to one another. </P><H3>Overall, our proof-of-concept system consists of three parts:
	</H3><UL><LI>BBa_K2450101 - TEV protease, with mCherry, expressed in pBAD33</LI><LI>BBa_K2450201 - TetR with a TEV protease cleavage site, with CFP, expressed in pQE-60(BBa_K2450251 - TetR without a TEV protease cleavage site, with CFP, expressed in pQE-60)</LI><LI>BBa_K2450301 - Tet promoter, followed by an RBS and the sequence for YFP, expressed in pQE-60</LI></UL><H6>Figure 4: Overall DNA system design</H6><P>Please see our <A href="https://2017.igem.org/Team:Oxford/Results_DNA">DNA-based system wet lab page</A> to follow the progression of this project.</P><H2 id="C2">Protein-Based System</H2><P>Having identified some of the problems associated with the cell-free DNA-based system, we innovated a system that overcame these. In our DNA-based system, TEV protease is activated in the presence of cruzipain; this mechanism relied, however, on DNA expression which may act too slowly for our desired output (see section on output). We decided to generate an entirely protein-based circuit that still produced the TEV protease.
</P><P>
We initially considered sterically inhibiting TEV protease in the absence of cruzipain. However, due to the crystal structure of the protease, specifically the location of the N- and C-termini oriented away from the active site, we decided this would not be a reliable method. We then discovered split TEV protease Wehr et al. 2006, in which two haves of the protein are present individually in an unconnected, and therefore inactive form. The halves are attached to dimerisation partners and on dimerisation TEV becomes active. We combined the split protease design with the ideas put forward in a paper using split fluorescent proteins in a protease detector Shekhawat et al. 2009. The result of this was two TEV protease fragments, each of which are bound to leucine zipper coils A, B, and B*. A is bound to inhibitory coil B* by a cruzipain-cleavable linker to prevent spontaneous activation. Inhibition is relieved on cleavage by cruzipain. When cruzipain cleaves, the A and B leucine zippers are free to dimerise, activating the TEV protease. This goes on to amplify the signal, producing a detectable output. This is shown schematically below.

	</P><P>Before cleavage, it is more entropically favorable for the B* coil to bind to the A coil as it is joined by a linker; dimerisation of the A and B coils is therefore inhibited. Once the linker (A–B*) is cleaved, A binds to B as this is the enthalpically favored reaction. Two of the Leu residues in the B* leucine zipper have been mutated to Ala residues, decreasing the enthalpic benefit of binding to A (another Leu zipper) by approximately 0.5 - 2kcal/mol per substitution, giving the B coil the advantage.</P><P>We hypothesised that introducing a feedback loop to amplify the signal would make the signal more sensitive to the levels of cruzipain and decrease the time taken to produce enough output to detect. In this system, both our input (cruzipain) and our intermediate output (TEV protease) are proteases. We designed the system so that TEV, as well as cruzipain, would cleave the A-B* linker. As a result, our system would require only a small initial signal to produce a clear, sustained response. See the <A href="https://2017.igem.org/Team:Oxford/Protein_Based_Model">protein-based model</A> for a quantitative comparison between these two systems.</P><H6>Figure 5: Formation of a TEV protease from two fragments</H6><P>As this was a protein based circuit, we recognised that freeze-drying the system from cell free extract, as we did with the DNA based system, would be unlikely to be possible. We found an alternative method in outer membrane vesicles (OMVs). OMVs are lipid vesicles, made from the outer membrane, containing periplasmic solution and proteins. OMVs are constitutively made by all Gram-negative bacteria. A recent paper by Alves et al. 2016, demonstrated that enzymatic function could be protected during freeze-drying cycles by containing the enzyme in an OMV. We aimed to target the proteins in our circuit to the bacterial outer membrane, extract OMVs containing them, freeze-dry the OMVs for storage and transport, and finally re-solubilise and lyse the OMVs for diagnosis.</P><P>The bacterial outer membrane transport system is still not yet fully understood. In order to target our proteins to OMVs, we adopted the technique described in Alves et al. 2016: (1) OmpA, a membrane protein in E. coli which is known to be transported into OMVs at a high rate, is fused to SpyTag and (2) the functional circuitry component is fused to SpyCatcher and a TorA leader sequence. The TorA leader sequence will transport our protein into the periplasmic space (via the Tat translocase system), where SpyTag and SpyCatcher can form an isopeptide bond (between Asp117 and Lys31, respectively) and fuse the two parts. The circuit proteins are then taken up into the OMVs with OmpA.
	</P><P>As part of a collaboration, the McMaster II iGEM Team ran a 300 ns all-atom molecular dynamics (MD) simulation of the SpyTag/SpyCatcher interaction. The last few nanoseconds of the simulation and the RMSF plot of residues (RMSF reflects the stability of the residue in water) are shown below. This data gave us confidence in the functionality of the SpyTag/SpyCatcher system, which is essential to our project.</P><P>The OMVs containing the protein circuits can be freeze-dried and then resolubilised when the diagnostic is needed - the latter step occurs upon the addition of blood. The OMVs then need to be lysed to expose the protein system within. We propose to accomplish the lysis by adding powdered detergent to the freeze-dried OMV  system, which is solubilized and therefore activated when blood is added to the kit. In order to test OMV lysis without denaturing protein we designed the two parts shown below:</P><DIV class="row"><DIV class="col-sm-6"><H6 class="text-center">Biobrick: BBa_K2450401</H6></DIV><DIV class="col-sm-6"><H6 class="text-center">Biobrick: BBa_K2450501</H6></DIV></DIV><P>The OmpA protein is able to be inserted into the outer membrane as the SpyTag motif is very small. The quenched sfGFP can then bind to SpyTag and be taken up into OMVs as explained above. This results in the protein complex shown below:</P><P>As seen in the diagram above, the sfGFP in an intact OMV is quenched by the quenching peptide. When the OMV is lysed, the TEV protease can access its cleavage site and relieve the quenching of fluorescence. This assay can be used to identify the best detergent to lyse the OMVs while keeping proteins stable. This is vital to both this project, and future projects looking to use OMVs as a chassis for freeze-drying and reactivating protein-based systems. If the BBa_K2450501 sfGFP-dark quencher part can be standardised against a known concentration of purified TEV protein, it can be used to quantify the concentration of TEV added to the lysed OMVs. To see how this could be useful look at the <A href="https://2017.igem.org/Team:Oxford/Future_experiments">future protein-based experiments page</A>.</P></DIV><DIV class="container"><H2 id="C3">Output</H2><P>Our system detects the <I> T. cruzi</I> parasite in the blood. Thus, we decided that an anticoagulant (which prevents blood clotting) would be the most appropriate output. When our system is activated, the blood of the Chagas-infected patient will be prevented from clotting. To see how this would work in practice, go to our <A href="https://2017.igem.org/Team:Oxford/Applied_Design">applied design page</A>.</P><P>As our system is designed to be cell-free, simplicity is key. We attempted to find the simplest way in which we could inhibit the clotting of blood and discovered a short peptide anticoagulant called hirudin, which inhibits thrombin, the enzyme directly responsible for causing clotting in blood. Both of our systems either produce or interact with proteins making this an ideal solution. On further research, we came across an engineered analogue of hirudin called bivalirudin. Bivalirudin is a bivalent, high-affinity, reversible inhibitor of thrombin;it binds to both the active site and exosite 1. It is 20 amino acids long, as opposed to hirudin which is 65 amino acids long, and has similar kinetic properties. Approximately 0.5 nM of bivalirudin is needed to prevent clotting in blood. For a more detailed discussion on how blood clotting was accounted for in modelling, see the <A href="https://2017.igem.org/Team:Oxford/DNA_Based_Model">DNA Based Model page</A>.</P><P>We initially considered encoding the anticoagulant in our DNA-based system and expressing it in response to the protease. Our modelling, however, informed us that this method would not provide sufficent anticoagulant fast enough to inhibit clotting. For more details, see the <A href="https://2017.igem.org/Team:Oxford/Cell_Free_Model">modelling page</A>. We therefore decided to use a sterically inhibited version of bivalirudin in both our systems (DNA- and protein-based). Due to the small size of bivalirudin and the extensive contacts between its termini and hirudin, we should be able to engineer a suitable peptide.</P><H5 class="text-center">Interaction between Bivalirudin (red) and Thrombin (green) (pdb: 3VXE)</H5><P>Our solution is to sterically inhibit the anticoagulant by attaching a TEV protease cleavage site and, if necessary, additional large domains domains onto the termini. We believe bivalirudin will provide a more chemically and economically feasible solution due to its smaller size: hirudin with attached TEV cleavage site/sterically hindering domains exceeds  the size limit of peptides which can be synthesised chemically. Please visit the <A href="https://2017.igem.org/Team:Oxford/Applied_Design">applied design page</A> for more details on our cell-free systems, including how our output would work in practice and the physical device.</P><H2>References</H2><H3>Background References</H3><P>Duschak, V.G. and Couto, A.S., 2009. Cruzipain, the major cysteine protease of Trypanosoma cruzi: a sulfated glycoprotein antigen as relevant candidate for vaccine development and drug target. A review. Current medicinal chemistry, 16(24), pp.3174-3202.</P><H3>DNA-Based System</H3><P>Pardee, K., Green, A. A., Takahashi, M. K., Braff, D., Lambert, G., Lee, J. W., … Collins, J. J. (2016). Rapid, Low-Cost Detection of Zika Virus Using Programmable Biomolecular Components. Cell, 165(5), 1255–1266. http://doi.org/10.1016/j.cell.2016.04.059</P><P>Ramos, J.L., Martínez-Bueno, M., Molina-Henares, A.J., Terán, W., Watanabe, K., Zhang, X., Gallegos, M.T., Brennan, R. and Tobes, R., 2005. The TetR family of transcriptional repressors. Microbiology and Molecular Biology Reviews, 69(2), pp.326-356.</P><H3>Protein-Based</H3><P>Phan, J., Zdanov, A., Evdokimov, A. G., Tropea, J. E., Peters, H. K., Kapust, R. B., … Waugh, D. S. (2002). Structural basis for the substrate specificity of tobacco etch virus protease. Journal of Biological Chemistry, 277(52), 50564–50572. http://doi.org/10.1074/jbc.M207224200</P><P>Wehr, M. C., Laage, R., Bolz, U., Fischer, T. M., Grünewald, S., Scheek, S., … Rossner, M. J. (2006). Monitoring regulated protein-protein interactions using split TEV. Nature Methods, 3(12), 985–993. http://doi.org/10.1038/nmeth967</P><P>Walker, M., Kublin, J. G., &amp; Zunt, J. R. (2009). NIH Public Access, 42(1), 115–125. http://doi.org/10.1086/498510.Parasitic</P><P>Alves, N. J., Turner, K. B., Medintz, I. L., &amp; Walper, S. A. (2016). Protecting enzymatic function through directed packaging into bacterial outer membrane vesicles. Scientific Reports, 6(1), 24866. http://doi.org/10.1038/srep24866</P><H3>Output</H3><P>Johnson, P. H. (1994). Hirudin : clinical potential of a thrombin inhibitor. Annu. Rev. Med., 45, 165–177. http://doi.org/10.1146/annurev.med.45.1.165</P></DIV><DIV class="container-fluid" style="margin-top: 80px"><DIV id="footer-row" class="row"><DIV id="footer-sponsor" class="col-sm-8"><P>Sponsors</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>