<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Oxford_Engagement skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Oxford/Engagement</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav navbar-right"><LI id="home-link"><A href="https://2017.igem.org/Team:Oxford">HOME</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PROJECT
            </A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Chagas_Disease">Chagas Disease</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Chagas_Public_Policy">Chagas Public Policy</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Applied_Design_Developing">--Developing our Design</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Applied_Design_Solution">--Our Solution</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Applied_Design_Implementation">--Implementation</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Cell_Free_Report">Cell Free Report</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/InitialIdeas">Initial Ideas</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">WET LAB
            </A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Overview_Wet_Lab">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Cloning">Results (Cloning)</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Results_DNA">Results (DNA-Based system)</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Results_Protein">Results (Protein-Based system)</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Future_experiments">Future Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/InterLab">Interlab</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Measurement">Measurement</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">DRY LAB
            </A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/DNA_Based_Model">DNA-Based Model</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Protein_Based_Model">Protein-Based Model</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Disease_Model">Disease Model</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Software">Software</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">HUMAN PRACTICES
            </A></LI><LI><A href="https://2017.igem.org/Team:Oxford/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/HP/Gold_Integrated">Integrated and Gold</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Engagement">Public Engagement</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PEOPLE
            </A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Team">Meet the Team</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Sponsors">Sponsors</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Awards">Our Awards</A></LI></DIV></DIV><DIV class="container"><H1 class="text-center">Education and Public Engagement</H1><H2>Introduction</H2><P>Designing educational activities around synthetic biology is difficult because the field is new and interdisciplinary (Kuldell, 2007). Consequently there are a lack of standard principles and definitions, creating an enlarging division of knowledge between the general public and synthetic biology community which fosters generalisation and misconceptions. Bidirectional public engagement is necessary to help bridge this gap, and was therefore a key focus of our public engagement.</P><H2>Summer School Engagement</H2><P>A major reason for the gap is a lack of synthetic biology training of the young. To accurately tackle this problem we base our activities on learning objectives for MIT undergraduates training in synthetic biology (Kuldell, 2007) (Figure 1). Our approach means our educational activities are not generalised, true to core synthetic biology concepts, and encourage scientific thinking.
</P><P><OL><LI>Not generalised</LI><LI>True to core synthetic biology</LI><LI>Encouraging scientific thinking</LI></OL></P><P>Although there are some excellent secondary education tools available (e.g. Biobuilder), we found that many of these require extensive access to lab equipment and expensive resources, creating a barrier to use in teaching. Therefore, our designed activities are <B>low cost</B>, require <B>minimal materials</B> and <B>do not assume lab access</B>.
    </P><H6>Figure 1: Main outcomes of synthetic biology education and activities we designed to fulfil these</H6><H2>Our Activities</H2><P>We carried out workshops at 3 different summer schools in Oxford: Northwest Science (students aged 17-18 yrs interested a broad range of sciences), Oriel College summer school (students aged 13-14 yrs interested in medicine) and UNIQ (academically selected state school students aged 17-18 yrs from state schools interested in biochemistry).</P><P>To meet these outcomes we designed a variety of activities including: a genetic memory computer lab, a bacterial photography system worksheet, an ethical washing line activity, and a 'Design your own biosensor' activity. These activities have been formatted into lesson plans and printable resources, made publicly available for use by future iGEM teams. Each activity has a set of aims to ensure a directed learning approach.</P><H3> Lesson Plans </H3><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse1">
        Genetic Memory Computer Lab (30 minutes)</A></H4></DIV><DIV id="collapse1" class="panel-collapse collapse"><DIV class="panel-body"><CENTER><H3>Aims</H3></CENTER><UL><LI>Understand how the genetic toggle switch works as an example of a synthetic biology system</LI><LI>Interpret a model of the circuit for the toggle switch built in iBioSim</LI><LI>Understand the importance of modelling in the context of the design test build cycle</LI></UL><H3>Resources</H3><P>Computer with the free software <A href="http://www.async.ece.utah.edu/ibiosim">iBioSim</A> and <A href="https://static.igem.org/mediawiki/2017/9/99/Genetic_Toggle.zip">toggle switch model</A></P><H3>Activities</H3><P>Before starting the simulation explain the toggle switch mechanism (Figure​ ​1​) to students e.g. A toggle switch is a circuit that allows switching of 2 stable states, much like turning on and off a computer. The switch is on when cells produce green fluorescent protein output, and is flipped between states by adding different on and off chemicals. The green fluorescence output is maintained even after the on and off chemicals are removed creating genetic memory.</P><CENTER><H6>Figure​ ​1:​ DNA circuit for genetic toggle switch</H6></CENTER><P>Run the model in stochastic and deterministic modes, and vary the GFP degradation rate and promoter strength to alter GFP output (Figures 2a – d).</P><CENTER><H6>Figure​ ​2:​ Expected graphs a) Deterministic simulation b) Stochastic simulation c) Decreased GFP degradation rate d) Increased promoter strength</H6></CENTER><P><B>Ask students the following questions:</B></P><P>Look at the graph a, can you see identify when the system is on and off? What happens
when you add more on chemical when the system is on?</P><P><I>Peaks. No change.</I></P><P>Compare the deterministic and stochastic models in graphs a and b? Which is better?</P><P><I>The deterministic model is a simplification to ignore output fluctuations due to random
collisions of molecules in the cell. The deterministic model is easier to identify trends but
the stochastic is more realistic of the noise in a cell.</I></P><P>How could you modify the circuit to get greater output (greater fluorescence)?</P><P><I>Increase promoter strength or decrease degradation rate of GFP modelled by graphs c
and d.</I></P><P>Which of these introduces the most time lag?</P><P><I>Decreasing degradation rate.</I></P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse2">
       Bacterial Photography System (30 minutes)</A></H4></DIV><DIV id="collapse2" class="panel-collapse collapse"><DIV class="panel-body"><CENTER><H3>Aims</H3></CENTER><UL><LI>Explain synthetic biology concepts of biobricks and modularity</LI></UL><H3>Resources</H3><P><UL><LI><A href="https://static.igem.org/mediawiki/2017/f/f3/Bacterial-photography-circuit-cards.pdf">Protein and DNA paper parts</A></LI><LI>A3 paper or whiteboard to draw cell outline</LI></UL></P><CENTER><H6>Figure​ ​1:​ Key words</H6></CENTER><H3>Activity</H3><P>Before carrying out the activity give a brief overview of the system. The circuit (Figure​ ​3​) works by taking an input of light to a light detector, the output of which controls the activity of a colour generator to produce a colour output, therefore allowing the bacterial population to reproduce an image taped to the petri dish by changing colour in the dark regions masked by the image.</P><CENTER><H6>Figure​ ​2:​ DNA circuit for genetic toggle switch</H6></CENTER><P>Figure​ ​3​: Bacterial photography circuit (a) inputs and outputs and (b) complete circuitry</P><P>Students will be given a set of proteins and DNA and challenged piece them together to
construct the circuit. This exercise can be used to distinguish key biochemical processes
and DNA modules (defined above).</P><P><B>After building the basic circuit extend the exercise with
the following discussion points:</B></P><P>The natural function of the receptor in <EM>E coli</EM> is to respond to extracellular salt concentration
(osmoregulation). How could this receptor be modified to be light responsive?</P><P><I>The receptor sequence is modified to contain the sequence from a light receptor from
the photosynthetic single celled organism cyanobacterium to encode a recombinant
receptor protein with the extracellular region from the cyanobacterium of the protein
is light responsive, but the intracellular region from the <EM>E. coli</EM> receptor conserved to
activate its cognate transcription factor.</I></P><P>In this model the proteins are already present in the cell. How is this simplification from the real
cell system and why is this simplification valid?</P><P><I>The proteins need to be expressed by the cell so the genes would be present. These
would be on plasmids expressed all the time (constitutively). Only the Beta
galactosidase gene is regulated in this system therefore it is only relevant to include
its DNA to interpret the circuit.</I></P><P>The transcription factor is phosphorylated. What do you think is the function of this modification
and how does it occur?</P><P><I>Phosphorylation is the covalent addition of a phosphate group to an -OH containing
amino acid on the protein. This changes the binding surface and/or structure of the
transcription factor allowing it to make new protein-protein or protein-DNA
interactions that can change the outcome of a cell signalling pathway, allowing it to
act as a switch in cell signalling. Phosphorylation is by enzymes (kinases) using the
phosphate from ATP. Dephosphorylation is by separate enzymes (phosphatases).
The relative activity of these enzymes can in turn be regulated to fine tune activity of
a cell signalling pathway.
</I></P><P>How could you modify the circuit?</P><P><I>Change the promoter strength and RBS strength to tune the output intensity, and
change the degradation rate of the proteins to change the timing of colour output
production.</I></P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse3">
        Ethical washing line (15 minutes)</A></H4></DIV><DIV id="collapse3" class="panel-collapse collapse"><DIV class="panel-body"><CENTER><H3>Aims</H3></CENTER><UL><LI>Stimulate an ethical discussion around use and ownership of synthetic biology.
</LI></UL><H3>Resources</H3><P><UL><LI>Strongly agree and strongly disagree cards</LI><LI>Case study cards: x10</LI><LI>Ownership cards: x3</LI><LI>String and clips or board and blue tac</LI></UL><A href="https://static.igem.org/mediawiki/2017/d/d9/Ethical-washing-line-case-studies.pdf">Link to cards</A></P><H3>Activity</H3><P>Students are given a set of cards of case studies of current and potential applications of synthetic biology applications and asked to rank them on a scale of agreement to disagreement on the ethics of each application on a piece of string. Similarly, students will rank who they think should own the synthetic biology devices.</P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse4">
       Biosensor​ ​design​ ​(15​ ​minutes)</A></H4></DIV><DIV id="collapse4" class="panel-collapse collapse"><DIV class="panel-body"><CENTER><H3>Aims</H3></CENTER><UL><LI>Explain key features of expression vectors and their function in molecular cloning.</LI><LI>Explain concepts of modularity and biobrick parts in synthetic biology.</LI><LI>Highlight the design aspect of synthetic biology.
</LI></UL><H3>Resources</H3><P><UL><LI><A href="https://static.igem.org/mediawiki/2017/b/b4/Design_your_own_biosensor_worksheet.pdf
">Worksheet</A> and pens</LI><LI>Scrap paper</LI></UL></P><H3>Activity</H3><P>Work with student groups to identify and explain the key features of an <EM>E. coli</EM>
expression vector (Figure​ ​4​) before asking them to design their own simple biosensor part
including promoter, ribosome binding site and up to 2 protein coding genes. Promoter
regulation and sensitivity, ribosome binding site strength and proteins can be varied. The
worksheet can be used as a template for the activity.</P><CENTER><H6>Figure​ ​4: Annotated key features of expression vector</H6></CENTER></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-sm-6"><H6>Image 1: Zoë C and John giving a presentation at the Northwest Science school at Corpus Christi College</H6></DIV><DIV class="col-sm-6"><H6>Image 2: Chun, Jei, Alissa, Helen and Sumaera at the UNIQ summer school</H6></DIV></DIV></DIV><DIV class="container"><H2>Improvement through Feedback</H2><P>Student feedback allowed us to refine our activities after each summer school session <B>(Table 1)</B>. For example, we converted the TinkerCell computer based bacterial photography system activity into a paper based game so computer access was no longer needed, and we also developed a formalised ethical washing line workshop after finding students disengaged  by an unstructured ethical discussion.  In addition to workshops, Zoe F gave a university style <A href="https://static.igem.org/mediawiki/2017/1/1a/T--oxford--public_engagement--zoe_pres.pptx">lecture</A>about synthetic biology and our project to students at the Northwest Science school.</P><TABLE class="table table-hover"><THEAD><TR><TH>School</TH><TH>Positives</TH><TH>Problems</TH><TH>Modifications</TH></TR></THEAD><TBODY><TR><TH>Northwest School</TH><TD><UL><LI>One  group had high engagement in the toggle switch lab,  despite not choosing A level biology </LI><LI>One student chose to use the TinkerCell computer aided design for their school extended project</LI><LI>The biosensor design was presented as an extension discussion and was very well received with some interesting designs including a bacterial Geiger counter</LI></UL></TD><TD><UL><LI>The logistics of using the TinkerCell program was difficult as laptops were limited, therefore students could not work independently</LI></UL></TD><TD><UL><LI>Due to lack of computer resources we had to create a paper based bacterial tinkercell circuit, which was just as well received in the other sessions</LI><LI>Originally the ethical discussion was left as an open discussion at the end of the session, however this was not well received and discussion were very general. As a consequence we decided to create a structured ethical washing line using real synthetic biology applications as case studies</LI><LI>Developed the biosensor design into a structured worksheet after the success of the extension activity </LI></UL></TD></TR><TR><TH>Oriel</TH><TD><UL><LI>The students  creatively engaged with the biosensor design worksheet</LI></UL></TD><TD><UL><LI>Groups worked at different rates</LI><LI>Students struggled to conceptualise the toggle switch</LI></UL></TD><TD><UL><LI>Suggested discussion points on the worksheet so demonstrators could extend the activities if they had a particularly quick group</LI></UL></TD></TR><TR><TH>UNIQ</TH><TD><UL><LI>The students were highly engaged and discussed beyond the activities </LI><LI>Many students described the workshop as the highlight of the week</LI><LI>In the general question and answer session students had many general questions about synthetic biology</LI></UL></TD><TD><UL><LI>Timing was an issue with groups not completing all tasks within the set 2  hour time frame</LI></UL></TD><TD><UL><LI>Added timings of activities to the lesson plans</LI></UL></TD></TR></TBODY></TABLE><H2>Curiosity Carnival</H2><P>The Curiosity Carnival is an Oxford-wide festival for researchers to share their research in innovative way as a part of the European Researchers' Night. The Oxford team were thrilled to be accepted to participate by having an activities stand at this event. Our stand was in the centre of the city and attracted a diverse audience of more than 4500 members of the public, from young children to other researchers.</P></DIV><DIV class="container"><DIV class="row"><CENTER><DIV class="col-sm-11"><H6>Image 3: Kushal, Angela, Sumaera and Zoe C at the Curiosity Carnival</H6></DIV></CENTER></DIV></DIV><DIV class="container"><H3>Our stand involved:</H3><H4>Poster display of our diagnostic solution</H4><P>Our poster explained the need for a novel congenital chagas diagnostic and gave molecular details of our parts composing our solution. The poster provided an aid to describe the technical details of our project to those who were interested, as we aimed to appeal to a wide range of audiences</P><H4>Where's <I>cruzi</I>? Game</H4><P>This game  challenged stall goers to identify the <I>Trypanosoma cruzi</I> parasite on a blood smear whilst wearing glasses that obscured vision. The game highlighted the difficulty and high false positive rate associated with microscopy diagnosis of Chagas due to confusion with similar parasites such as <I>Trypanosoma brucei</I>, <I>Trypanosoma rangeli</I>, <I>Toxoplasma gondii</I>, <I>Leishmania</I> species, and <I>Plasmodium falciparum</I>. The game, outlined in Figure 2, was popular with adults and children alike and provided an interactive way of showing the problems with existing diagnostics.</P><CENTER><H6>Figure 3: Our 'Where's <I>cruzi</I>?' game from the Curiosity Carnival</H6></CENTER><H4><EM>E. coli</EM> pong</H4><P><EM>E. coli</EM> pong, an improved version of beer pong, challenged festival-goers to throw ping pong balls (representing  plasmids) into plastic cups (representing <EM>E. coli</EM> cells). The aim of the game was to illustrate the technical difficulty of transformation that increases with multiple plasmids. The game acted as a vehicle for us to explain the basic principle of molecular cloning, particularly engaging children.</P><CENTER><H6>Image 4: Zoe F and Kushal demonstrating our ‘<EM>E. coli</EM> pong’ activity</H6></CENTER><H4>Voting on ethical scenarios</H4><P>
We introduced topical synthetic biology ethical issues in the form of for or against statements for the public to vote on.</P><P>These included:</P><OL><LI>Synthetic biology should be used in medicine even when existing treatment already exists
</LI><LI>There should not be a distinct line between artificial intelligence and life
</LI><LI>It is never justifiable to release synthetic genetically modified organisms be released into the environment
</LI><LI>Everyone should have access to the tools to genetically modify organisms
</LI><LI>Synthetic biology companies are justified in creating patents for their discoveries
</LI><LI>Cell free systems should be classified as living systems
</LI><LI>All synthetic biology systems should be designed with a real world problem in mind
</LI><LI>All products involving or containing genetically modified organisms should be extensively labelled
</LI><LI>Synthetic biology is accurately portrayed by the media
</LI></OL><P>Some people were initially reluctant to vote, or undecided on a vote, facilitating debate around the issues. Our most successful question garnered over 200 responses. This strengthened our opinion that education is necessary for developing opinions on synthetic biology, and having reasoned discussions around the subject.</P></DIV><DIV class="container"><H2>iGEM UK Meet-ups</H2><P>Engaging with other teams to share ideas and techniques is a vital component of iGEM: we were grateful to be able to attend the UK meet up jointly organised by the Universities of Westminster, Warwick, and UCL during which we gave Jamboree style poster and powerpoint presentation to other teams, providing invaluable preparation for Boston. The event was lots of fun and we appreciated the opportunity to meet, network and form collaborations with other UK teams.</P></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-sm-5"><H6>Image 5: Our excellent speakers (Arthur and John) pose with Tom cruzi (complete with his own little tie) before they go in to give our presentation at the Shard</H6></DIV><DIV class="col-sm-6"><H6>Image 6: UCL held a fantastic set of debates about synthetic biology, and our team members were very successful - here's Zoe F with their winning team, who argued in favour of using genetically modified organs</H6></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-sm-9"><H6>Image 7: At Westminster, Arthur talks through our poster with members of the UCL team</H6></DIV></DIV><H2>Social Media</H2><P>We have worked hard to develop a solid and consistent social media presence across three platforms during our project. Our Instagram account has included regular updates on what the team was getting up to inside and outside of the lab, helping to promote a more well-rounded image of scientists, and also participating in tags such as #femalesinstem to display the gender diversity in our team. We have a high level of engagement for our Instagram posts, regularly reaching a variety of accounts including Oxford colleges, scientific institutions, and the general public. Our Twitter account keeps abreast of the important instagram and facebook posts, and is also key to engaging with other iGEM teams. The Oxford iGEM Facebook page has the biggest following of all our accounts, with over 1000 followers, and displays a range of posts from ‘Meet the Team’ to information about our outreach events and human practices. We have made sure to advertise any outreach we have been involved in across all platforms, such as the Curiosity Carnival and the UNIQ summer schools. We are also careful to display best practice in all laboratory-based photos.</P></DIV><DIV class="container"><H2>References</H2><P>Kuldell, N., 2007. Authentic teaching and learning through synthetic biology. Journal of biological engineering, 1(1), p.8.</P></DIV><DIV class="container-fluid" style="margin-top: 80px"><DIV id="footer-row" class="row"><DIV id="footer-sponsor" class="col-sm-8"><P>Sponsors</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>