<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Oxford_Results_DNA skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Oxford/Results DNA</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav navbar-right"><LI id="home-link"><A href="https://2017.igem.org/Team:Oxford">HOME</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PROJECT
            </A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Chagas_Disease">Chagas Disease</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Chagas_Public_Policy">Chagas Public Policy</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Applied_Design_Developing">--Developing our Design</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Applied_Design_Solution">--Our Solution</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Applied_Design_Implementation">--Implementation</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Cell_Free_Report">Cell Free Report</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/InitialIdeas">Initial Ideas</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">WET LAB
            </A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Overview_Wet_Lab">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Cloning">Results (Cloning)</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Results_DNA">Results (DNA-Based system)</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Results_Protein">Results (Protein-Based system)</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Future_experiments">Future Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/InterLab">Interlab</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Measurement">Measurement</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">DRY LAB
            </A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/DNA_Based_Model">DNA-Based Model</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Protein_Based_Model">Protein-Based Model</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Disease_Model">Disease Model</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Software">Software</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">HUMAN PRACTICES
            </A></LI><LI><A href="https://2017.igem.org/Team:Oxford/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/HP/Gold_Integrated">Integrated and Gold</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Engagement">Public Engagement</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PEOPLE
            </A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Team">Meet the Team</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Sponsors">Sponsors</A></LI><LI><A href="https://2017.igem.org/Team:Oxford/Awards">Our Awards</A></LI></DIV></DIV><DIV class="container"><H1>Results(DNA-Based System)</H1><H2>Introduction</H2><P>In order to demonstrate the feasibility of our DNA-based system for detecting cruzipain, we designed and cloned four parts into the pSB1C3 vector:</P><OL><LI>mCherry-TEV Protease-His tag <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2450101"> BBa_K2450101</A></LI><LI>TetR(With an engineered TEV protease cleavage site)-CFP-His tag <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2450201"> BBa_K2450201</A></LI><LI>TetR(WT)-CFP-His tag <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2450251"> BBa_K2450251</A></LI><LI>pTet-RBS-eYFP <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2450301"> BBa_K2450301</A></LI></OL><P>From our modelling of this system and our research into cruzipain levels in the blood, we estimated the level of cruzipain that would be present. We then calculated the optimum levels of TetR and our pTet construct that would limit false positives and negatives in such a system. We added fluorescent reporters (which did not have significant overlap in their spectra) to our three parts in order to visualise their levels, and hence determine the levels of inducers we needed to accurately replicate our in silico model <EM>in vivo</EM>.</P><H2>Shipping Vector Cloning</H2><P>A more in-depth description of this stage of our project can be found on the <A href="https://2017.igem.org/Team:Oxford/Results_Wet_Lab#shipping_vector">Shipping Vector Cloning</A> results page. For the DNA-based part of our project, we cloned the four parts into the shipping vector.</P><P>We then used quick-change PCR to perform site-directed mutagenesis on our TetR parts and the pTet part. In the TetR parts we had mistakenly included a stop codon at the end of the CFP coding sequence before the His-tag, so although this would not have affected the workings of the part it would have meant we wouldn’t be able to purify it. The pTet-eYFP had an illegal restriction site so we used Quick-Change PCR  to make it compatible with registry standards.</P><P>This, as the name suggests, was a quick and easy procedure and we recommend it to any team that is in a similar situation. The protocol can be found on our <A href="https://2017.igem.org/Team:Oxford/Protocols">protocols</A> page.</P><H2>Expression Vector Cloning</H2><P>In order to perform full characterisation of our parts we needed TEV Protease, TetR, and pTet-eYFP in the same cell. As the likelihood of a triple transformation was very low, we decided to clone both the TetR and the pTet-eYFP into the same plasmid. We then would only need to do a double transformation of this and the TEV protease plasmid. We selected pQE-60 and pBAD-33 as they had compatible origins, different antibiotic resistances, and were induced by two different and easily-obtainable inducers, IPTG and arabinose respectively.</P><P>We designed primers for two purposes:</P><OL><LI><P>To amplify the parts from the shipping vector constructs with restriction sites that allowed for cloning into the expression vectors. These were NcoI and BamHI for pBAD-33 and XbaI and PstI for pQE-60. These enzyme combinations ensured that the start codon of the part was the optimal distance from the RBS in the plasmid for efficient expression.</P></LI><LI><P>To amplify a version of the mCherry-TEV Protease-His tag, TetR(With an engineered TEV protease cleavage site)-CFP-His tag and TetR(WT)-CFP-His tag parts without fluorophores. This was ensure the fluorophore did not affect the activity.</P></LI></OL><H3>Planned Plasmid Constructs</H3><DIV class="row"><DIV class="col-sm-6"><H6> Plasmid Map of BBa_K2450101 in pBAD-33 </H6></DIV><DIV class="col-sm-6"><H6> Plasmid Map of BBa_K2450101 without mCherry in pBAD-33 </H6></DIV></DIV><DIV class="col-sm-6"><H6> Plasmid Map of BBa_K2450201 and BBa_K2450301 in pQE60 </H6></DIV><DIV class="row"><DIV class="col-sm-6"><H6> Plasmid Map of BBa_K2450251 and BBa_K2450301 in pQE60 </H6></DIV></DIV><P>We were successful in doing this for mCherry-TEV protease-His tag, as well as the no fluorophore version, and pTet-eYFP into pBAD33 and pQE-60 respectively. We further characterised the mCherry-TEV protease-His tag and pTet-eYFP.</P><P>For the other parts in our system, we unfortunately did not obtain a correctly sequenced version in the expression vector, despite obtaining the correctly sized bands in the test digest agarose gels of our minipreped DNA.</P><P>To try and resolve this problem we did a number of things:</P><UL><LI><P>We repeated the cloning many times with no luck.</P></LI><LI><P>We thought that it could be a problem with the primers, so we tried longer primers, which still yielded no positive results.</P></LI><LI><P>We wondered if the vector stock has been mixed up, and we had managed to clone the part into a different vector. Therefore, we created some primers that would sequence from inside the part to out - again to no avail.</P></LI><LI><P>We requested the sequencing company to optimise the reaction conditions and help troubleshoot, and though this was attempted it did not give us decent sequencing data.</P></LI></UL><P>Due to the integrated nature of our system, it was therefore difficult to characterise the whole circuit without having every part correctly sequenced. As we couldn’t get our novel TetR engineered with a TEV protease cleavage sequence (which was central to this particular system) into an expression vector, it was particularly difficult to get data on the auxiliary parts.</P><H2>TetR(with an engineered TEV protease cleavage site)-CFP-His tag</H2><P>Whilst waiting for sequencing optimisation, we decided to continue with the characterisation of this part for two main reasons:</P><UL><LI><P>It was the final piece to complete the DNA-based system</P></LI><LI><P>Test digests suggested that the DNA we had miniprepped was of the right size.</P></LI></UL><H3><EM>In vivo</EM> - Fluorescence microscopy</H3><DIV class="imagel">Figure 1: Microscopy images showing that when eYFP fluoroscence is observed in cells, there is no CFP fluorescence, and that when there is CFP observed there is no eYFP signal.
        </DIV><P>We co-transformed the TetR-CFP with the correctly sequenced pTet-YFP into the same cell. The hope was to see just YFP when the cells were uninduced with IPTG and then a reciprocal relationship between CFP and YFP and induction levels were increased. </P><P>Figure 1 is a promising microscopy image which showed a binary, yet reciprocal expression of YFP and CFP, suggesting that our TetR could possibly be functioning as designed to repress pTet eYFP. However, this wasn’t consistently reproducible.</P><H3><EM>In vitro</EM> - Protein purification and TEV protease assay</H3><DIV class="imagel">Figure 2: Protein Purification of engineered TetR showing the stages of protein purification, including a band at 50kDa which is potentially the protein of interest.
	</DIV><P>In parallel to the fluorescence microscopy, we attempted to purify the engineered TetR based on a protocol provided in a paper by Mortlock et.al.</P><P>We were helped greatly in this regard by  <A href="https://2017.igem.org/Team:Oxford/HP/Gold_Integrated#Maike">Associate Prof. Maike Bublitz</A>, who guided us on how to purify the protein using a nickel column, which was kindly lent to us by <A href="https://2017.igem.org/Team:Oxford/HP/Gold_Integrated#Higgins">Prof. Matt Higgins</A>, also from the Oxford Department of Biochemistry.</P><P>An SDS-PAGE gel, Figure 2, showed that we had eluted an array of non-specific proteins. However, there was a thick band near 50kDa which is the size of TetR+CFP, suggesting that we had been partially successful. Due to the impure nature of the gel, it meant it could be a different protein.</P><P>If the band was our engineered TetR, it would be sensitive to TEV protease action after concentrating the protein through a 5kDa filter, which gave us a 1.56 mg/ml solution. We tested the concentrated engineered TetR with TEV protease provided by Prof. Bublitz.</P><DIV class="imager">Figure 3: TEV protease assay showing the time the reaction was quenched after the addition of protease, analysed on a gel to look at the fragments. Unfortunately there was no change in the previously identified 50kDa band.
	</DIV><P>We tested two concentrations of TEV protease, a 1:100 molar ratio (relative concentration of TEV protease to concentration of ‘Engineered TetR’) and a 1:10 ratio. We quenched the reaction at different time points (0mins, 60mins, 120mins, 180mins, O/N) by adding it to SDS Loading Buffer. Another SDS PAGE gel was run using these time points.</P><P>The gel in Figure 3 shows us that there doesn’t seem to be any change in the band near the 50kDa mark.</P><P>However, interestingly at the 150kDa mark, there is a sharp band that disappears over time in 1:10 TEV protease. Seeing as there was no real result for our TetR, we decided to focus our efforts on the characterisation of other parts.</P><DIV id="mCherry_TEV"><H2>mCherry-TEV Protease-His tag</H2><DIV class="imagel">Figure 4: Microscopy images mCherry labelled TEV protease, comparing fluorescence with the background of empty pBAD33 vector.
	</DIV><P>We obtained some fluorescent microscopy images of the expression of mCherry-TEV protease in E.colicells. However, since we could not characterise our engineered TetR, we had no method for testing if this TEV protease has the ability to cleave in vivowithin the DNA-based system. However, you can see a preliminary image of these cells in Figure 4. This could be used in the future to compare to cells expressing our engineered TetR.</P></DIV><H2 id="pTet">pTet with eYFP reporter</H2><P>As we managed to successfully clone this part into an expression vector (pQE-60), we were able to characterise this part.</P><P>This part has a carefully picked ribosome binding site and promoter strength to optimise our system for minimal false positives and negatives. In our final kit, eYFP will be replaced with TEV protease to amplify the input signal. Hence it was vitally important to detect how repression was relieved with the introduction of anhydrotetracyline (ATC) which mimics the relief of expression by cruzipain cleavage.</P><P>For more information see our <A href="https://2017.igem.org/Team:Oxford/Design">Design Page</A>.</P><H3>Aims</H3><P>This part required a two-step characterisation:</P><UL><LI><P> Check TetR can bind to pTet and repress eYFP production</P></LI><LI><P>Check ATC can relieve the repression by TetR (link to TetR design page) and hence prove that TetR is the component that is causing the repression</P></LI></UL><H3>Experimental Design</H3><H4>Cloning</H4><P>Two strains of <EM>E. coli</EM> were used to test our part:</P><OL><LI><P>DH5a containing pTet-eYFP plasmid</P></LI><LI><P>JBEI-2492 - a strain that contains TetR in its bacterial genome transformed with pTet-eYFP plasmid</P></LI></OL><H4>Initial Microscopy to Show Fluorescence</H4><P>We did an initial fluorescence microscopy test with the pTet-eYFP to see if YFP was being produced constitutively.</P><DIV class="row" id="figure_4_all"><DIV class="col-sm-8"><DIV class="imagel">Figure 5: (A and B) Microscopy images showing the expression of eYFP vs an empty pQE60 control after being grown to mid-log with do inducer. (C) Separate image of pTet-eYFP grown to mid-log in pSB1C3. Although the direct levels are not comparable, the expression of eYFP appears to be more consistent when expressed in pSB1C3, this could be due to variations in copy number of pQE60.
	</DIV></DIV><DIV class="col-sm-4"><P>We did an initial fluorescence microscopy test with the pTet-eYFP to see if YFP was being produced constitutively. Figure 5 shows some of the images from this test, where it can be seen that eYFP is produced in the cells even when it is in a vector which does not contain an additional promoter. This shows that our pTet is functional.</P></DIV></DIV><H4>Assay procedure</H4><P>We ran steady state analyses of the two strains as we were interested in the qualitative ability of TetR to repress pTet.</P><P>To achieve this, we performed the following steps:</P><UL><LI><P>Three colonies of each strain were picked and grown overnight in minimal media (M9-clear liquid, which does not contain amino acids hence has less background fluorescence than LB)</P></LI><LI><P>The OD600 of the cells was measured to ensure growth that reached stationary phase.</P></LI><LI><P>The cells were diluted and fluorescence for YFP was recorded in a BMG Labtech Clariostar Plate reader at 495 +/– 15nm, as well as the absorbance at 600nm</P></LI><LI><P>Three aliquots of the overnight culture of each colony of the strains were taken and diluted down.</P></LI><LI><P>The aliquots were introduced to various levels of ATC and allowed to grow until stationary phase was reached.</P></LI><LI><P>The readings for three technical repeats of the three biological repeats were taken on the plate reader.</P></LI></UL><H4>Results</H4><P>In Figure 6 we could see the significant (nearly 4-fold) difference in Fluorescence/Absorbance upon the introduction of TetR, which suggests that our pTet is sensitive to repression by TetR.</P><P>In the future, we would like to extract quantitative data from such an assay to find the amount of TetR we would need to add to our system to completely repress production of our output factor.</P><DIV class="col-sm-6"><DIV class="imagel">Figure 6: Steady State Analysis of pTet sensitivity to TetR
	</DIV></DIV><DIV class="col-sm-6"><DIV class="imager">Figure 7: Steady State Analysis of TetR sensitivity to ATC in pTet+eYFP, showing that addition of &gt;1nM ATC leads to an increase in the fluorescence levels in the cells.
	</DIV></DIV><P>In Figure 7, ATC has been used to determine that repression is indeed being caused by TetR and to mimic cruzipain.</P><H3>Statistical Analysis</H3><DIV class="row"><DIV class="col-sm-6"><H6>Figure 8: Student t-test statistical analysis on our data, showing that the effect of adding ATC is significant</H6></DIV><DIV class="col-sm-6"><P>The t-test in Figure 8 showed that there was a significant difference in the Flu/Abs when TetR was introduced to the pTet-eYFP system. This suggests that TetR significantly represses the pTet reducing eYFP production. </P><P>The significant release of repression when ATC is added clearly identifies TetR as the component that is causing repression.
</P></DIV></DIV><H2>Summary and Conclusions</H2><UL><LI><P>We cloned four parts into the pSB1C3 shipping vector</P></LI><LI><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2450101">BBa_K2450101</A> - mCherry-TEV protease </LI><LI><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2450201">BBa_K2450201 </A>- Engineered TetR-CFP</LI><LI><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2450251">BBa_K2450251 </A> - TetR-CFP </LI><LI><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2450301">BBa_K2450301 </A>- pTet-RBS-eYFP </LI></UL><LI><P>We cloned two parts into pBAD-33 and pQE-60, our expression vectors of choice</P></LI><LI><P>We showed that the mCherry-TEV protease was produced.</P></LI><LI><P>We determined that the pTet-eYFP is statistically sensitive to repression by TetR and is optimised for low false positives and false negatives.</P></LI><H2>References:</H2><P>Mortlock, A., Low, W. and Crisanti, A., 2003. Suppression of gene expression by a cell‐permeable Tet repressor. Nucleic acids research, 31(23), pp.e152-e152.</P></DIV><DIV class="container-fluid" style="margin-top: 80px"><DIV id="footer-row" class="row"><DIV id="footer-sponsor" class="col-sm-8"><P>Sponsors</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>