"
Page
Discussion
View source
History
teams
Log in
 
Team:BostonU/Data
From 2012.igem.org
BostonU iGEM Team: Welcome
Our Team
Team
Summer Fun
Project
Project Overview and Abstract
Introduction to MoClo
Introduction to Characterization
Introduction to Data Sheets
Methodology Overview
Results Summary
Achievements
Data Collected
Parts Submitted
MoClo Kit
Medal Fulfillment
Notebook
Protocols
Clotho and Eugene
Weekly Notebook
Considerations
New England iGEM Regional Meeting
Human Practices
Safety
Acknowledgements
Collaborations
Acknowledgements
JUMP TO...
Top
Sequencing
Characterization
BioBrick Characterization
New BioBrick Characterization
Data Collected
Sequencing Data for Converted MoClo Parts
We sequenced all of our new MoClo parts to ensure the proper fusion sites were flanking the part sequence. Below is a list of all of the confirmed, sequenced parts, including our two BioBrick parts for mmoR and pMmoR.
After the wiki freezed we kept working on converting Biobricks to Level 0 MoClo parts and creating Level 2 Destination vectors. Below is a list of all of the confirmed, sequenced  new parts.
Characterizing BioBrick Based Devices
We began to work on our characterization workflow in September because we had originally planned to have MoClo Level 1 and Level 2 parts to test by then. Unfortunately, due to many technical challenges, we do not have a Level 1 or Level 2 part to test yet. In order to move forward with our characterization thrust, we decided to test devices that had been made in our lab using BioBricks so we could begin to learn how to characterize devices using flow cytometry.
One of those devices was an inverter under pBad (BBa_I13458-BBa_I13453) control. This inverter was built using the two-way BioBrick protocol and was submitted with the rest of our parts this year ( BBa_K783067 ). As you can see below, this inverter functions as expected, where GFP increases with the addition of arabinose and thus the induction of the pBad promoter. The amount of RFP decreases as the amount of tetR increases, thus repressing the pTetR promoter.
This inverter uses the RBS BBa_B0032 to drive the tetR gene, while the other two ribosomal binding sites are BBa_B0034. Previous work in our lab showed the use of BBa_B0034 to drive the tetR gene resulted in poor function for the inverter. By changing the RBS, the function of this inverter was improved.
We also tried to test the function of a cI based inverter, where cI and pCI replace tetR and pTetR. Unfortunately, those cells grew poorly and we were unable to complete our experiment. We believe cI may have some level of toxicity to E. coli, since our cultures always grow poorly when cI is expressed.
We also wanted to study the function of various inducible and repressible promoters. One such promoter was the pLacI promoter. However, as you can see below, the RFP showed no change over the different concentrations of IPTG, suggesting something within the genetic circuit is broken. We are in the process of sequencing this circuit to see if we can determine if one of the BioBrick parts is incorrect.
Finally, we wanted to test some more basic BioBrick composite parts with the hope that we would be comparing this data directly with MoClo Level 1 parts. Despite lacking these Level 1 parts, we still tested the BioBrick composite parts. These more basic parts consisted of J231XX promoters controlling GFP (BBa_E0040) or RFP (BBa_E1010). They all had the BBa_B0034 RBS and BBa_B0015 terminator. We noticed that one of the promoters showed a large difference in expression of RFP and GFP (shown in bottom graph) despite the circuits having the same RBS. This suggests that the Anderson library promoters may not function the same when controlling different genes.
Characterizing New BioBrick Parts
A New Repressible System: Copper and mmoR
We are also introducing a new promoter and gene pair to iGEM that forms a copper-dependent repressible system. Since the library of metal sensitive promoters was relatively small on the Registry of Standard Biological Parts, we decided to investigate a metal-dependent system. We chose mmoR and a σ54 promoter associated with it from a bacterium called Methylosinus trichosporium OB3b. mmoR is a σ54-dependent transcriptional activator.
It is unclear if the σ54 promoter will function in E. coli in the absence of mmoR. nBLAST results comparing the mmoR to the E. coli genomes available yielded very short hits, with the top hit showing matches for less than 200bp out of the possible 2070bp length of mmoR. However, the pBLAST hits showed stronger hits with other σ54-dependent transcriptional activators found in E. coli. The top nBLAST and pBLAST hits are shown below.
To determine if the σ54 promoter will function without mmoR present in E. coli, we PCR amplified 395bp of the σ54 promoter and cloned the product into the pSB1C3 BioBrick backbone. We then placed the promoter in front of GFP, RFP, and YFP using standard BioBrick cloning. Based on colony color from our transformations, it does not appear that the σ54 promoter will function alone in E. coli. We will be running a flow cytometry experiment on these clones for the Jamboree and working towards building the larger circuit shown at the bottom of this section.
We also amplified the same region of the σ54 promoter using MoClo primers and cloned it into one of our Level 0 MoClo destination vector.
We've also cloned mmoR into pSB1C3 and are in the process of building the circuit shown below. We will have an update on our progress for the Jamboree.
Our mmoR and σ54 promoter work was inspired by Scanlan et. al, 2009 .
References
[1] J. Scanlan et al. "Involvement of MmoR and MmoG..." FEMS Microbiology 301(2). 2009.
Retrieved from " http://2012.igem.org/Team:BostonU/Data "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
