"
Page
Discussion
View source
History
teams
Log in
 
Team:BostonU/Methodology
From 2012.igem.org
BostonU iGEM Team: Welcome
Our Team
Team
Summer Fun
Project
Project Overview and Abstract
Introduction to MoClo
Introduction to Characterization
Introduction to Data Sheets
Methodology Overview
Results Summary
Achievements
Data Collected
Parts Submitted
MoClo Kit
Medal Fulfillment
Notebook
Protocols
Clotho and Eugene
Weekly Notebook
Considerations
New England iGEM Regional Meeting
Human Practices
Safety
Acknowledgements
Collaborations
Acknowledgements
JUMP TO...
Top
Destination Vectors
MoClo Level 0 Parts
PCR Troubleshooting
Methodology
Making Destination Vectors and MoClo Parts
Building Destination Vectors
In order to generate MoClo Destination Vectors (DV)s, we had to add the alpha fragment of lacZ to BioBrick backbones where DNA parts are normally inserted. First, we PCR amplified the alpha lacZ fragment with primers designed to add both the MoClo fusion sites and type IIs restriction sites to it.
The amplified products and the backbones were digested with SpeI and ligated together to generate the Destination Vectors (DV)s. We performed blue-white screening to select the correct DVs and selected the blue colonies for mini preps. These were sent for sequencing to verify the correct orientation of the MoClo sites and type IIS restriction sites.
Level 0 DVs contain Chloramphenicol resistance on the backbone (BioBrick backbone used: pSB1C3) and a BsaI site followed by a Bpil site. Level 1 DVs contain Kanamycin resistance on the backbone (BioBrick backbone used: pSB1K3)  and a Bpil site followed by a BsaI site. Level 2 DVs contain Ampicillin resistance on the backbone (BioBrick backbone used: pSB1A3) and a BsaI site followed by a Bpil site.
Building MoClo Parts
Like the destination vectors, we performed PCR reactions to add Moclo fusions sites and type IIs restriction sites (BsaI and Bpil) to commonly used BioBrick parts: promoters, RBSs, coding sequences (CDS) and terminators.  For further information on this odyssey, check the section PCR Strategies and Troubleshooting .
Then, we performed MoClo digestion/ligation reactions with the modified parts and the L0 DVs.  We performed blue-white screening, selecting white colonies and sending DNA for sequencing to check the L0 MoClo parts.
After converting the BioBricks parts to L0 MoClo parts, we are able to combine them to make L1 MoClo parts using standard MoClo digestion/ligation reactions. We submitted our L0 parts library to Eugene and generated a list of all the possible L1 parts.  We selected some of them and proceeded to the MoClo reactions. Each MoClo reaction included a L0 promoter, L0 RBS, L0 CDS, L0 terminator and L1 DV, in other words, 5 restriction digest and 4 ligations concluded in one single step. We performed blue-white screening, selecting white colonies and sending DNA for sequencing to verify the L1 MoClo parts.
We encoded our L1 parts library in Eugene (a computer language for synthetic biology) and generated a list of all the possible L2 parts, which are more complex genetic circuits formed by multiple transcriptional units. We selected some of them according to rules created in Eugene and proceeded to ligate an L2 DV and the chosen L1 MoClo parts in one single reaction. For further information in Eugene, refer to the Eugene section. Then, we performed blue-white screening, selecting white colonies and verifying sequences of the L2 MoClo parts.
PCR Strategies and Troubleshooting
One of the crucial steps in  converting BioBricks to MoClo is adding the fusion sites to all the parts:  promoters, RBSs, genes and terminators. In our project, we utilized PCR  amplification to insert the fusion sites into the parts sequence. However, this  is not as simple as it may sounds. PCR is a sensitive technique that requires  optimized conditions for each set of primers and templates used. Figuring out  what were the conditions for each reaction we ran was a learning experience  that we intend to share in this section.
The first aspect to be considered  is what kind of PCR is going to be utilized. For parts bigger than 50 bp:  genes, terminators and some of the promoters we utilized amplification PCR and  designed the forward and reverse primers to have the fusion tags in their 5’ end.
For smaller parts: RBSs and J series promoters, our first approach was to use inverse PCR, which has the primers  oriented in the reverse direction of the usual orientation. The primers tags contain the fusion sites and the parts sequence that will anneal after the  extension. The mechanism can be better understood by the diagram below:
Unfortunately, we were not  successful utilizing this method. The second approach we implemented was  ligation PCR1 that consists in two primary PCRs that are then  ligated together in a third PCR. For each of the primary PCRs we utilize one  primer whose tag that contains the fusion sites and the part (B, C) and another  regular primer to amplify the complementary strand (A, D). The third PCR is run  with the outermost primer pair (A,D) as clarified by the diagram below:
Besides  determining the PCR strategy there was troubleshooting involved in the  determination of the optimized conditions to run each PCR.
We realized  that reagents such as:  MgCl and DMSO can play a big role in optimizing the reaction and we often  ran gradient PCR varying the concentrations of these reagents to achieve the  best conditions.
The amount of template added is very  important as well. Too much template might lead to unspecific amplification  while too little template may not be enough to obtain the desired PCR product.
Also,  determining the primers ideal annealing  temperature required running temperature gradients PCRs with the melting temperature (Tm) ranging from from 2°C through 10°C lower than the lower Tm for the two primers. In our work, we found most PCRs worked with a Tm of 2°C lower than the lowest Tm for the forward and reverse primers.
Another aspect to take into account is the extension time that depends on the  enzyme efficiency and the size of the PCR product. Extension time longer than  necessary may lead to amplification of unspecific products, while short  extension time may not be enough to amplify the product entire sequence.
References
[1] http://www.biot echniques.com/multimedia/archive/00036/BTN_A_04363BM04_O_36287a.pdf
Retrieved from " http://2012.igem.org/Team:BostonU/Methodology "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
