<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_BostonU_Methodology"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:BostonU/Methodology">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:BostonU/Methodology&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:BostonU/Methodology&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:BostonU/Methodology&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:BostonU/Methodology" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:BostonU/Methodology</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>BostonU iGEM Team: Welcome</TITLE></P><DIV id="bu-wellesley_wiki_content"><UL id="nav"><LI><A href="#">Our Team</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Team">Team</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Social">Summer Fun</A></LI></UL><LI><A href="#">Project</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Project_Overview">Project Overview and Abstract</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/MoClo2">Introduction to MoClo</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Characterization">Introduction to Characterization</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/DataSheet">Introduction to Data Sheets</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Methodology ">Methodology Overview</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Results ">Results Summary</A></LI><LI><A href="#">Achievements</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Data">Data Collected</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Parts">Parts Submitted</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/MoClo">MoClo Kit</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Gold">Medal Fulfillment</A></LI><LI><A href="#">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Clotho">Clotho and Eugene</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Notebook">Weekly Notebook</A></LI><LI><A href="#">Considerations</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/NEGEM">New England iGEM Regional Meeting</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Human Practices">Human Practices</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Safety">Safety</A></LI><LI><A href="#">Acknowledgements</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Collaborations">Collaborations</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Acknowledgements">Acknowledgements</A></LI><DIV id="tracking_nav">
JUMP TO...<A href="#bu-wellesley_wiki_content">Top</A><A href="#BuildDV">Destination Vectors</A><A href="#BuildParts">MoClo Level 0 Parts</A><A href="#PCR">PCR Troubleshooting</A></DIV><H4>Methodology</H4><H9>Making Destination Vectors and MoClo Parts</H9><DIV id="BuildDV"><UL><H8>Building Destination Vectors</H8><UL><H7><P dir="ltr">
In order to generate MoClo Destination Vectors (DV)s, we had to add the alpha fragment of <I>lacZ</I> to BioBrick backbones where DNA parts are normally inserted. First, we PCR amplified the alpha <I>lacZ</I> fragment with primers designed to add both the MoClo fusion sites and type IIs restriction sites to it. 

The amplified products and the backbones were digested with SpeI and ligated together to generate the Destination Vectors (DV)s. We performed blue-white screening to select the correct DVs and selected the blue colonies for mini preps. These were sent for sequencing to verify the correct orientation of the MoClo sites and type IIS restriction sites.
</P><P dir="ltr">
Level 0 DVs contain Chloramphenicol resistance on the backbone (BioBrick backbone used: pSB1C3) and a BsaI site followed by a Bpil site. Level 1 DVs contain Kanamycin resistance on the backbone (BioBrick backbone used: pSB1K3)  and a Bpil site followed by a BsaI site. Level 2 DVs contain Ampicillin resistance on the backbone (BioBrick backbone used: pSB1A3) and a BsaI site followed by a Bpil site.

</P></H7></UL></UL><DIV id="BuildParts"><UL><H8>Building MoClo Parts</H8><UL><H7><P dir="ltr">
Like the destination vectors, we performed PCR reactions to add Moclo fusions sites and type IIs restriction sites (BsaI and Bpil) to commonly used BioBrick parts: promoters, RBSs, coding sequences (CDS) and terminators.  For further information on this odyssey, check the section  <A href="https://2012.igem.org/Team:BostonU/Methodology#PCR"> PCR Strategies and Troubleshooting</A>.

Then, we performed MoClo digestion/ligation reactions with the modified parts and the L0 DVs.  We performed blue-white screening, selecting white colonies and sending DNA for sequencing to check the L0 MoClo parts.
</P><P dir="ltr">
After converting the BioBricks parts to L0 MoClo parts, we are able to combine them to make L1 MoClo parts using standard MoClo digestion/ligation reactions. We submitted our L0 parts library to Eugene and generated a list of all the possible L1 parts.  We selected some of them and proceeded to the MoClo reactions. Each MoClo reaction included a L0 promoter, L0 RBS, L0 CDS, L0 terminator and L1 DV, in other words, 5 restriction digest and 4 ligations concluded in one single step. We performed blue-white screening, selecting white colonies and sending DNA for sequencing to verify the L1 MoClo parts.
</P><P dir="ltr">
We encoded our L1 parts library in Eugene (a computer language for synthetic biology) and generated a list of all the possible L2 parts, which are more complex genetic circuits formed by multiple transcriptional units. We selected some of them according to rules created in Eugene and proceeded to ligate an L2 DV and the chosen L1 MoClo parts in one single reaction. For further information in Eugene, refer to the <A href="https://2012.igem.org/Team:BostonU/Clotho#eugntro"> Eugene</A> section. Then, we performed blue-white screening, selecting white colonies and verifying sequences of the L2 MoClo parts.
</P></H7></UL></UL><DIV id="PCR"><H9>PCR Strategies and Troubleshooting</H9><H7><UL><P dir="ltr">
One of the crucial steps in  converting BioBricks to MoClo is adding the fusion sites to all the parts:  promoters, RBSs, genes and terminators. In our project, we utilized PCR  amplification to insert the fusion sites into the parts sequence. However, this  is not as simple as it may sounds. PCR is a sensitive technique that requires  optimized conditions for each set of primers and templates used. Figuring out  what were the conditions for each reaction we ran was a learning experience  that we intend to share in this section.

  The first aspect to be considered  is what kind of PCR is going to be utilized. For parts bigger than 50 bp:  genes, terminators and some of the promoters we utilized amplification PCR and  designed the forward and reverse primers to have the fusion tags in their 5’  end. 
</P><P dir="ltr">For smaller parts: RBSs and J series promoters, our first approach was to use inverse PCR, which has the primers  oriented in the reverse direction of the usual orientation. The primers tags contain the fusion sites and the parts sequence that will anneal after the  extension. The mechanism can be better understood by the diagram below: 
</P><P dir="ltr">
Unfortunately, we were not  successful utilizing this method. The second approach we implemented was  ligation PCR1 that consists in two primary PCRs that are then  ligated together in a third PCR. For each of the primary PCRs we utilize one  primer whose tag that contains the fusion sites and the part (B, C) and another  regular primer to amplify the complementary strand (A, D). The third PCR is run  with the outermost primer pair (A,D) as clarified by the diagram below:</P><P dir="ltr">

Besides  determining the PCR strategy there was troubleshooting involved in the  determination of the optimized conditions to run each PCR.

  We realized  that reagents such as:  <STRONG>MgCl</STRONG> and<STRONG> DMSO</STRONG> can play a big role in optimizing the reaction and we often  ran gradient PCR varying the concentrations of these reagents to achieve the  best conditions.

  The <STRONG>amount of template</STRONG> added is very  important as well. Too much template might lead to unspecific amplification  while too little template may not be enough to obtain the desired PCR product. 

  Also,  determining the <STRONG>primers ideal annealing  temperature</STRONG> required running temperature gradients PCRs with the melting temperature (Tm) ranging from from 2°C through 10°C lower than the lower Tm for the two primers. In our work, we found most PCRs worked with a Tm of 2°C lower than the lowest Tm for the forward and reverse primers.

  Another aspect to take into account is the <STRONG>extension time</STRONG> that depends on the  enzyme efficiency and the size of the PCR product. Extension time longer than  necessary may lead to amplification of unspecific products, while short  extension time may not be enough to amplify the product entire sequence. 
<H8>References</H8>
  [1] <A href="http://www.biotechniques.com/multimedia/archive/00036/BTN_A_04363BM04_O_36287a.pdf">http://www.biot echniques.com/multimedia/archive/00036/BTN_A_04363BM04_O_36287a.pdf</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:BostonU/Methodology">http://2012.igem.org/Team:BostonU/Methodology</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:BostonU/Methodology" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:BostonU/Methodology" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:BostonU/Methodology&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:BostonU/Methodology&amp;oldid=287134" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></UL></H7></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>