<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_BostonU_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:BostonU/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:BostonU/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:BostonU/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:BostonU/Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:BostonU/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:BostonU/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>BostonU iGEM Team: Welcome</TITLE></P><DIV id="bu-wellesley_wiki_content"><UL id="nav"><LI><A href="#">Our Team</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Team">Team</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Social">Summer Fun</A></LI></UL><LI><A href="#">Project</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Project_Overview">Project Overview and Abstract</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/MoClo2">Introduction to MoClo</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Characterization">Introduction to Characterization</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/DataSheet">Introduction to Data Sheets</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Methodology ">Methodology Overview</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Results ">Results Summary</A></LI><LI><A href="#">Achievements</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Data">Data Collected</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Parts">Parts Submitted</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/MoClo">MoClo Kit</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Gold">Medal Fulfillment</A></LI><LI><A href="#">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Clotho">Clotho and Eugene</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Notebook">Weekly Notebook</A></LI><LI><A href="#">Considerations</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/NEGEM">New England iGEM Regional Meeting</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Human Practices">Human Practices</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Safety">Safety</A></LI><LI><A href="#">Acknowledgements</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Collaborations">Collaborations</A></LI><LI><A href="https://2012.igem.org/Team:BostonU/Acknowledgements">Acknowledgements</A></LI><H4>Weekly Notebook</H4><DIV id="tracking_nav">
JUMP TO...<A href="#bu-wellesley_wiki_content">Top</A><A href="#May5">May Week 5</A><A href="#June1">June Week 1</A><A href="#June2">June Week 2</A><A href="#June3">June Week 3</A><A href="#June4">June Week 4</A><A href="#July1">July Week 1</A><A href="#July2">July Week 2</A><A href="#July3">July Week 3</A><A href="#July4">July Week 4</A><A href="#July5">July Week 5</A><A href="#Aug1">August Week 1</A><A href="#Aug2">August Week 2</A><A href="#Aug3">August Week 3</A><A href="#Aug4">August Week 4</A><A href="#Sept1">September Week 1</A><A href="#Sept2">September Week 2</A><A href="#Sept3">September Week 3</A><A href="#Sept4">September Week 4</A><A href="#Oct1">October Week 1</A><A href="#Oct2">October Week 2</A><A href="#Oct3">October Week 3</A><A href="#Oct4">October Week 4</A></DIV><DIV id="May5"><H2>May Week 5: Warming up!</H2><H7><UL><LI> We met with our mentors and got an overview of the basics of synthetic biology, namely characterization, which will be the focus of our works during the summer. We also went through the “MoClo” technique, which we intend to introduce to iGEM.</LI></UL><UL><LI>We did a survey of previous iGEM teams that got the Best Experimental Measurements award, then presented the findings to our mentors.</LI></UL></H7><DIV id="June1"><H2>JuneWeek 1</H2><H7><UL type="disc"><LI>We performed PCR to amplify pMJSAF(LacZ) using the       following MoClo forward and reverse primers: DVL0_AF with DVL0_BR (MF1),       DVL0_BF with DVL0_CR (MF2),  DVL0_CF with DVL0_DR (SJ1) and DVL0_DF       with DVL0_ER (SJ2). We performed Gel Electrophoresis to confirm that       pMJSAF was amplified in PCR:        The gel ran for too long thus       the bands ran off of the gel. which can be seen towards the bottom of the       picture. We expect for the band to be around 500 bp.</LI></UL><UL type="disc"><LI>We reperformed the PCR amplification of pMJSAF with the       same pairs of primers: DVL0_AF with DVL0_BR (MF3), DVL0_BF       with DVL0_CR (MF4), DVL0_CF with DVL0_DR (SJ3), DVL0_DF with DVL0_ER       (SJ4). We performed Gel Electrophoresis on this set of DNA.</LI></UL><UL type="disc"><LI>The amplification of pMJSAF displayed strong bands at       500 bp for the primers : DVL0_BF with DVL0_CR and DVL0_DF with DVL0_ER.       However for the   DVL0_AF with DVL0_BR showed a much weaker       band, and the DVL0_CF with DVL0_DR did not show at all. We gel extracted       MF3,4 and SJ4. Using the gel extraction protocol. We then quantified the       amount of DNA we had using NanoDrop DNA quantification protocol andset up       a restriction digest (SpeI, BSA, Buffer 4)for MF3,4 and SJ4. following the       Restriction digest protocol </LI></UL><UL type="disc"><LI>We did PCR clean up of MF3,4 and SJ4 and We quantified       the product of MF3,4 and SJ4 PCR clean up using NanoDrop</LI></UL><UL type="disc"><LI>For the amplification that didnìt work, we        performed gradient PCR  of SJ3 and MF3 in order to set the best       parameters for the amplification (62-69 with interval of 1 degree C). We       later found out, that our step 1 was set at 62C instead of 95C, which       could have affected denaturation of the DNA template; although the SJ5-12       was amplified while the MF5-12 did not. We are unsure to the cause. The       results were inconclusive compared to the Gel made previously.We ran       another PCR to verify results.</LI><LI>We were given new primers to make stock and  working solutions from. In the process we realized that we used DVL1_ER instead  of DVL0_ER in SJ4, thus all SJ4 products were discarded and data obsolete. We  remade the DVL1_ER stock solution by adding additional EB buffer till the right  concentration has been reached. We then proceeded to make stock and working  solutions for: DVL0_ER, DVL1_AF, DVL1_ER, DVL1_FF, DVL1_FR, DVL1_GF, DVL1_GR  and  DVL1_HR.  Then we paired AF-ER, FF-GR, EF-FR, GF-HR to amplify  the template DNA using PCR. We made the gel from the previously mentioned PCR  of AF-ER (MS4), FF-GR (MS5), EF-FR(MS6), GF-HR (MS7) and also 3 L0 (MF3=MS1,  SJ3=MS2,  SJ4=MS3) </LI></UL><UL type="disc"><LI>Alll the L0 failed and the L1 showed bands</LI><LI>We performed gel extraction on the 4 L1 primer  based amplification. We performed    nanodrop quantification of  the products of the gel extraction. </LI></UL><UL type="disc"><LI>We performed restriction digest (SpeI, BSA, Buffer 4)of the products of the gel extraction.From the restriction digest of MS4-7 we performed PCR clean up.  We nano dropped the PCR clean up product. </LI></UL><UL type="disc"><LI>We transformed iGEM plasmids with competent bacteriafrom stock, and plated them.  We grew them overnight in 37C incubator: pSB1A3 plate 1 well 1G  (1A3), pSB1AT3 plate 1 well 13A (1AT3), pSB1C3 plate 1 well 3A (1C3), pSB1K3 plate 1 well 5A (1K3). We picked colonies from the plates that were stored in the incubator and allowed the picked colonies to be incubated in a 37C shaker.   We performed miniprep on 3 of the 4 overnight cultures(1k3 was not used due to minimal growth while 1A3, 1AT3 and 1C3 were able to grow sufficiently). We performed nanodrop quantification on the products of the miniprep and then performed restriction digest (SpeI, BSA, Buffer 4)of the products of the miniprep.</LI></UL><UL type="disc"><LI>  We made a second generation of culture of the plasmid carrying bacteria by injecting 20 uL of the original culture into 5 mL of fresh LB broth with the appropriate antibiotics. Using the generation 2 culture of the bacteria with the 1K3 plasmid, we mini prepped it. We nanodrop quantified the mini prepped material. We then ran restriction digest(SpeI, BSA, Buffer 4) of the mini prepped sample.</LI></UL><UL type="disc"><LI>We streaked plates from the overnight cultures of transformed bacteria (psB1A3, pSB1AT3, pSB1C3, pSB 1k3) and placed them       into the 37C incubator.</LI><LI>We prepared a presentation of our project for Wellesley       college, whom we will be collaborating with during the summer :)</LI><LI>We had a meeting with professor Doug and other undergrads from CIDAR lab.</LI></UL><DIV id="June2"><H2>JuneWeek 2</H2><H7><UL type="disc"><LI>
 Attended meeting with Wellesley team. Collaboration has started!</LI><LI><P dir="ltr">We performed PCR amplification (20 μL reactions) with new sets of L0 primers: DVL0_EF with DVL0_BR (MS8), DVL0_FF with DVL0_BR (MS9), DVL0_GF with DVL0_BR (MS10), DVL0_DF with DVL0_FR (MS11), DVL0_DF with DVL0_GR (MS12) and DVL0_DF with DVL0_HR (MS13).   We then performed gel electrophoresis of MS8-13, all were successful.</P></LI><LI>We performed Gel extraction and nanodropped the samples. </LI><LI>Ran 50 μL PCR reactions of MS8.1-13.1, the same samples as MS8-3, in order to have enough DNA for restriction digest. We performed Gel Electrophoresis of the PCR samples and we performed Gel Extraction. Then, we nanodropped the samples. </LI>    <P><LI>We performed Restriction digest with the samples MS9.1 and MS 12.1 that had enough DNA concentration.  We PCR clean up-ed MS9.1 and 12.1</LI></P></UL><UL><LI><P dir="ltr">We performed a new PCR in triplicate for the samples MS8.1, 10.1, 11.1 and 13.1 to achive a higher concentration of DNA. We ran gel of MS8.1, 10.1, 13.1,11.1 to amplify more DNA template to achieve sufficient DNA concentration upon gel extraction</P></LI><P dir="ltr"> </P><P>
    </P><P><LI>We performed ligation of the successful amplified samples of MS4-7 and MF4: 	MS4-7 with backbone of 1AT3, MS4-7 with backbone of 1A3 and MF4 with backbone of 1K3 </LI></P></UL><UL><LI> Ran MgCl2 gradient of MF3 PCR to troubleshoot, along with DMSO separately.     </LI></UL><UL><LI>We transformed the ligations:MS4-7 with backbone of 1AT3, MS4-7 with backbone of 1A3, MF4 with backbone of 1K3 with competent bacterial cells. We spread plated them with 20 μL of a 40 mg/mL X-Gal solution and 8 μL of a 0.5 IPTG solution for blue-white screening. </LI></UL><DIV id="June3"><H2>JuneWeek 3</H2><H7><UL type="disc"><LI>
We found out that MS8-13 are all wrong due to primer dimer.  We redid the PCR of all level 0 primers that failed: L0AB (FJ1), L0EB (FJ2), L0FB (FJ3), L0GB (FJ4), L0DF (FJ5), L0DG (FJ6),L0DH (FJ7), L0DE (FJ8), L0CD (FJ9). Instead of running a gel after the PCR, PCR clean up was performed. The product of the clean up was then nanodropped.</LI><P dir="ltr"> </P></UL><UL><LI><P dir="ltr">Then the PCR clean up samples of FJ1-9 underwent restriction digest. We ran gel of digest, bands came out too weak, but all present.</P></LI><P dir="ltr"> </P></UL><UL><LI><P dir="ltr">Redid digest with higher amount of DNA. New gel is much more   clear</P></LI><P dir="ltr"> </P></UL><UL><LI><P dir="ltr">We perfomed gel extraction and nanodropped gel extraction product.</P></LI><P dir="ltr"> </P></UL><UL><LI><P dir="ltr"> Performed PCR clean up on RD of 1A3, 1AT3, 1C3, 1K3 transformed at week1 and nanodropped</P>
   <P dir="ltr"> </P></LI></UL><UL><LI><P dir="ltr"> Ligated 1K3 with pSMJSAF from samples of FJ1-9. Transformed the ligations on to plates.  The plates did not yield positives for the transformation.</P></LI></UL><UL><LI><P dir="ltr"> RD the backbones in two sets, one set with just SpeI other with both SpeI and EcoRI.  Gel of the RD showed that there were unexpected bands when cut with both enzymes, something not shown on the registry experience pages for the backbones.</P></LI><P dir="ltr"> </P></UL><UL><LI><P dir="ltr">Plated bacteria transformed with the backbones. Picked colonies from the plates. Miniprepped and nanodropped the cultures with the the backbone</P></LI><P dir="ltr"> </P></UL><UL><LI><P dir="ltr">Made glycerol stocks of 1AT3, 1A3, 1C3, 1K3 both for -20 and -80 celsius</P></LI></UL><UL><LI><P dir="ltr">We prepared plate media of Kan, Amp and CAM</P></LI></UL><DIV id="June4"><H2>JuneWeek 4</H2><H7><UL type="disc"><P dir="ltr"> We designed primers and gene synthesis template.</P><LI><P dir="ltr">We worked on Safety Answers for our Wiki.</P></LI><LI><P dir="ltr">We attended an Eugene Tutorial given by Ernst, the Pos-Doc responsible for Eugene in CIDAR Lab.</P></LI><LI><P dir="ltr">We worked on Human Practices project.</P></LI><LI><P dir="ltr">We met the Wellesley and MIT iGEM teams for a Physics lecture in MIT given by Walter Levine, had lunch together and visited the MIT iGEM team lab :) </P></LI></UL><DIV id="July1"><H2>JulyWeek 1 - Happy July 4th!</H2><H7><UL type="disc"><LI><P dir="ltr">We transformed plasmids containing the genes, promoters and terminator to be used as templates for the primers that we had designed. The plasmids come from the iGEM kit plates.</P></LI></UL><UL><UL><LI>we used bioline cells but didn’t follow the protocols because the cells were more competent so didn’t need to go through heat shock.</LI><LI>cells sat on bench over the weekend. Colonies will be picked next week!</LI></UL><LI>We also autoclaved Broth, Media and DI water to have enough plates for our next round of transformations.</LI><LI>We used to Clotho to input the primers that we had designed into a database.</LI><UL><LI>in doing so, we learned how to use Spreadit Features, Spreadit Oligos, Registry Importer, Collection Viewer and Sequence Viewer.</LI></UL><LI>We are still waiting for our primers to continue wet lab work.</LI></UL><DIV id="July2"><H2>JulyWeek 2</H2><H7><UL type="disc"><UL><LI>We transformed the following parts from the iGEM plates into bioline cells : E1010, C0051, C0040, B0015, I13458, E0030, E0040, C0012, C0071, R0010, R0063, R0079, I13453, C0079, C0080, R0040, R0051, R0062, R0071, C0062.  Cultures from transformation were made and then a miniprep was performed on all of them. We ran a gel to confirm the plasmid presence and performed nanodrop to measure DNA concentration. After that, glycerol stocks were made for all of them.</LI><P dir="ltr"> </P><P dir="ltr"> </P></UL><UL><LI>We created working and stock solutions for the new primers that arrived to use at PCR. We performed PCR using the transformed parts as templates and the primers we designed in order to add moclo fusion sites to them. The PCR conditions were set according to the table below.</LI><LI><P dir="ltr"> </P></LI></UL><UL><LI>We confirmed the PCR reactions by running 5μL agarose gels. For samples bigger than 300bp we ran 1% gel, for samples smaller than that we ran 2% gels. </LI></UL></UL><UL><LI>We performed PCR Clean up in the samples: C0051, C0040, C0062, I13458, R0063AB, R0063EB, R0063FB, R0063GB, R0010AB, E1010, E3030, C0012, C0071, C0080, B0015DE, B0015DF, B0015DG, B0015DH, E0040, C0079, pSB1C3, pSB1k3. Then, we nanodropped them. </LI></UL><DIV id="July3"><H2>JulyWeek 3</H2><H7><UL type="disc"><LI>

We performed restriction digest of the samples with SpeI and NEB buffer 4 for 1 hour. Then, we performed PCR clean up of the RD samples and measured the DNA concentration by nanodrop.</LI></UL><UL><LI>We did ligation of pSB1C3 and pSB1k3, in order to circularize the plasmid that we amplified in order to remove RFP.Then, we transformed the ligation product using bioline cells and spread plate them.  The 1C3 yielded colonies but the 1K3 didn’t. We picked colonies of 1C3 for overnight culture. </LI></UL><UL><LI>We performed PCR on pSB1A3, R0010(EB, FB, GB) and R0079 (AB, EB, FB, GB) with phusion from NEB to trouble shoot. For PCR we extended 1A3 extension time, while lowering annealing temperature for all samples. We run a gel to check the PCR product. For 1A3, we ran a 1% gel and for the other samples we ran a 2% gel. Again the amplifications  did not work and we decided to double check the primers we designed. We realized that the primers R0010_For_E, R0010_For_F, R0010_For_G,  R0079_For_A, R0079_For_E, R0079_For_F an R0079_For_G had problems on their design and couldn`t anneal properly to the template. We are ordering new correct primers and troubleshooting the amplification of pSB1A3.</LI></UL><UL><LI>We performed Miniprep in sample pSB1C3 and digested it with SpeI and buffer N4. For the sample pSB1k3 we performed a new ligation, transformed the ligation product using bioline cell and spread plate it.  We ran a 1% gel to check the digestion product of pSB1C3 and proceeded to PCR clean up of the sample and DNA quantification.</LI></UL><UL><LI>We started building our level 0 destination vectors :) We ligated our backbone pSB1C3 with the following pcr amplified products: DVL0_GF_BR, DVL0_DF_FR, DVL0_DF_GR, DVL0_CF_DR and DVL0_BF_CR. We transformed them using Bioline competent cells, and plated them with X-gal and IPTG for blue-white screening.</LI><LI>We updated our <A href="https://2012.igem.org/Team:BostonU/Clotho">Clotho experience section</A>, check it out!

</LI></UL><DIV id="July4"><H2>JulyWeek 4</H2><H7><UL type="disc"><LI>
We made cultures of E1010 and pSB1C3, pSB1k3 did not work.</LI>
 We transformed ligations of DVL0_GF_BR, DVL0_DF_FR, DVL0_DF_GR, DVL0_CF_DR and DVL0_BF_CR to the pSB1C3 backbone following the bioline cell protocol for ligations because the last set of transformations didn`t work without heat shock. The cells grew but they were not blue. We thought that it might be related to the fact that we did not leave enough time for IPTG and X-Gal be absorbed into the plate. We made a new set of ligations and repeated the transformations, this time placing the plates with IPTG and X-Gal into the incubator for 30 min before plating the cells. And it finally worked! All of them had blue colonies, except for DVL0_FF_BR. <LI>In order to perform new ligations with the Level 0 inserts, we repeated the protocol from last week to obtain more backbone of pSB1C3.</LI><LI>Also, we worked on the pSB1k3 backbone. We ran the PCR using the conditions we previously set. We performed PCR clean up AND DNA quantification and ran a gel to confirm the amplification. </LI></UL><UL><LI>We performed a single restriction digest (SpeI) of the sample, DNA quantified it and ligated to itself in order to get circularized backbone.</LI></UL><UL><LI>We transformed the ligation product following the Alpha Select cells protocol. The colonies grew! We picked colonies and incubated them overnight. We performed Miniprep and DNA quantification.</LI></UL><UL><LI>We ran a single restriction digest (SpeI) and a double restriction digest (SpeI and EcoRI) of the pSB1k3 sample. We ran gel of the double cut and the uncut to confirm the circularized backbone.</LI></UL><UL><LI>Because the transformations of pSB1C3 yielded low count of blue colonies we decided to dephosphorylate the pSB1K3 backbone before ligation in order to decrease the likelihood of the backbone re-ligating to itself.  We, then performed PCR clean up and Dna quantification.</LI></UL><UL><LI>We, then, ligated the pSB1K3 backbone to the DVL1_ AF_ER, DVL1_ FF_GR, DVL1_ EF_FR and DVL1_ GF_HR inserts we had amplified before. We transformed using the same method as with the Level 0 transformations.
</LI></UL><DIV id="July5"><H2>JulyWeek 5</H2><H7><UL type="disc"><LI>We ran a restriction digest with BsaI on miniprep samples from DVL0s transformations of last week to check sizes and send them for sequencing. We ran a ran a gel to confirm it. </LI></UL><UL><LI>Unfortunately, the sequences we got back showed multiple plasmids contamination. Two of them worked and had a clean sequencing file but lacZ had the wrong orientation. We had to work on the DVL0s again :(</LI></UL><UL><LI>To resequence the DVL0s, we learned in depth on how to streak plates and re streaked plates of the DVL0 cells in order to get single blue colonies. Then, we picked the colonies and ran minipreps on them the other day and made glycerol stocks.</LI></UL><UL><LI>We kept working in pSB1K3 backbone, in order to perform new ligations of DVL1. We ran a restriction digest of the psb1k3 backbone with SpeI and ran gel of the RD product with the uncut psB1K3</LI></UL><UL><LI>We gel extracted the RD product of pSB1K3, it`s concentration was 5 ng/uL. We, then, dephosphorylated pSB1K3 and ran a PCR clean up with concentration of 5 ng/uL. We finally ligated all the L1s amplified lacZ with psb1K3, doing two transformations for each set, one using dephosphorylated backbone and the other, regular backbone. And they finally grew!!!! :) :)</LI></UL><UL><LI>We picked blue colonies and let them grow overnight. On the other day, we ran minipreps and made glycerol stocks for them.</LI></UL><UL><LI>Besides the struggle with DVs, we proceeded with the PCR of the parts to add MoClo and Restriction sites to them. We ran PCR of J23100 to J23109 primers with circular pSB1C3 template. Ran gel to confirm, but none showed correct bands.</LI></UL><UL><LI>We ran PCR of J23105-J3108 primers with cut 1C3 backbone hoping that would improve the result and we ran a gel to confirm. Again, no success</LI></UL><UL><LI>Then, we performed PCR with J23112 primer on uncut 1C3 backbone. We extended the annealing time from 15 seconds to 30 seconds and also decreased the annealing temperature 4 degrees celsius below the indicated temperature on the IDT sheet as an attempt to improve the primers performance. Again, no success. </LI></UL><DIV id="Aug1"><H2>AugustWeek 1</H2><H7><UL type="disc"><LI>We ran a restriction digest of all the DVL1 and DVL0 miniprep samples (42 in total!!) cut with BsaI, and ran them  in gel with their uncut counterparts to check sizes before sending them for sequencing.</LI><LI>We proceeded our troubleshooting PCR of the J and B series using sewing/ligation PCR, a 10 cycle PCR reaction amplifying pSB1C3  and transforming it directly. For more details, check the Methodology section for PCR strategies and troubleshooting. In the beginning, the ligation PCR didn`t work, so we tried a gradient temperature. It showed faint bands but enough to move on and improve it.    /li&gt;
</LI><LI>We realized that non specific bands may be due to excess of DNA template. We ran the PCR again with the decreased amount of template and they worked. Finally :) :) Currently we have J23100-J23112 and J23110-J23118BA-BE,  except for J23113 BA. The gel pictures for the amplification can be seen below. </LI></UL><DIV id="Aug2"><H2>AugustWeek 2</H2><H7><UL type="disc"><LI>For the DVs we submitted for sequencing last week, we checked the sequences by blasting them against the expected fragment. All of them worked, except for the DVL0s: CFDR, DFER, FFBR and GFBR that showed the wrong MoClo sites. We had to begin again the process of making these DVL0s :we redid the PCR for them, performed a restriction digest with SpeI, ran a gel,  gel extracted them and quantified them.</LI></UL><UL><LI> We ligated those parts to 1C3 backbone and transformed them. Unfortunately, the cells we used the first time were not so competent anymore and the transformations didn't work. So, we repeated the transformations with Bioline cells and let them growing on the bench over the weekend.</LI></UL><UL><LI>  After using pSB1C3 for the ligations, we were running short of the backbone to use as template for ligation PCR. So we streaked plates from glycerol stocks, pick colonies and ran miniprep on the other day. Then, we ran a RD, pcr clean up and quantified it. </LI></UL><UL><LI>  We did our first set of MoClo reactions in order to make L0 MoClo parts.</LI></UL><UL><LI>  We transformed them and let them growing on the bench over the weekend to perform blue-white screening next week. </LI></UL><UL><LI>  Also, we started working on our DVL2s. We are using amp as our resistance backbone, so we decided to get it from one of miniprep samples we had from C0040 that has an amp backbone. We ran RD of C0040 with XbaI and SpeI to get C0040 out, we ran a gel to confirm it and gel extracted it.</LI></UL><UL><LI>  Then, we performed PCR using pSB1A2 as template and BBpls_For and BBpls_Rev as primers.  We ran a gel to confirm it and gel extracted it.</LI></UL><UL><LI>  Then, we ran Restriction digest with SpeI, PCR clean up and its concentration was 3.2 ng/uL. DVL2 backbone is ready to be ligated with amplified LacZ having the MoClo sites :)</LI></UL><DIV id="aug3"><H2>AugustWeek 3</H2><H7><UL type="disc"><LI>We kept working on L2 DVs. We performed a ligation of backbone pSB1A2 to itself  and transformed the circularized backbone. We picked colonies the following and set overnight cultures of the plasmid. Then, we performed minipreps and got 28 ng/uL DNA concentration. Also, we amplified LacZ with the following primers: DVL0_For_A and DVL0_Rev_F, DVL0_For_A and DVL0_Rev_G, DVL0_For_A and DVL0_Rev_H, DVL0_For_E and DVL0_Rev_G,  DVL0_For_E and DVL0_Rev, H and DVL0_For_F and DVL0_Rev_H.   We digested both the LacZ amplified fragments and the pSB1A2 backbone with SpeI cut. We ran the cut sample son a gel to confirm sizes.</LI></UL><UL><LI>We performed gel extraction and then, quantified the samples. </LI></UL><UL><LI>We performed ligation of the DVL2_AF_BR, DVL2_AF_GR, DVL2_AF_HR, DVL2_EF_GR and DVL2_EF_HR with pSB1A2 and transformed them with bioline gold cells for blue-white screening. Unfortunately, colonies grew but none of them turned blue. </LI><LI>We dephosphorylated the backbone pSB1A2 to prevent recircularization when ligating it to amplified LacZ. We performed new ligation and transformed them. Again, colonies grew but none of them turned blue. We are troubleshooting the experiment now. </LI><LI>The MoClo L0 parts transformations worked nicely, plates showed more white colonies then blue, indicating that LacZ was cut out of the backbone and the Moclo parts took place. We picked white colonies and let them grow overnight. We performed miniprep in all of them and Dna quantified them.</LI></UL><UL><LI>In order to check the MoClo parts before send it for sequencing, we ran BsaI digests and ran the digest product on a gel. </LI></UL><DIV id="Aug4"><H2>AugustWeek 4</H2><H7><UL type="disc"><LI>We continued the ligation PCRs to add MoClo sites to J series promoters, RBSs and genes. We tried to scale down the first set of reactions, because we were only using 2 uL of the reaction product  as template for third set of pcr reactions. It didn`t work, however. The gel showed really faint bands and it was impossible to gel extract them.</LI></UL><UL><LI>We performed new 50 ul PCRs for J23103-J23106 BA-BE.</LI></UL><UL><LI>Also, we ran regular PCRs to amplify the genes with correct primers that arrived recently: R0010EB, R0010FB, R0010GB, R0079AB, R0079EB, R0079FB, R0079GB and I1453FB. </LI></UL><UL><LI>We, then, gel extracted the bands and quantified the DNA from both gels.</LI></UL><UL><LI>Also, we performed ligation PCR for B0030-34 BC.</LI></UL><UL><LI>Then, we gel extracted and quantified it.</LI></UL><UL><LI>After using so much pSB1C3 as template for PCR we ran out of it and had to streak plates from the Glycerol stocks, run minipreps, digest, clean up and finally continue with our PCRs  :) We ran ligation PCR of J23107-09 AB-EB, J23119AB-EB and J23113AB, J23103AB.</LI></UL><DIV id="Sept1"><H2>September Week 1</H2><UL><LI>We ran a gel to check the ligation PCRs from last week: 23107-09 AB-EB, J23119AB-EB and J23113AB. Then, we performed gel extraction.</LI></UL><UL><LI>We quantified the samples</LI></UL>.<UL><LI>We also repeated the PCR of lacZ with to add the fusion sites and build our DVL2s. We ran a PCR clean up and quantified the samples.</LI></UL><UL><LI>We digested the samples with SpeI and ran a gel of the digested samples to check sizes. Then, we ran a gel extraction.</LI></UL><UL><LI>And quantified the samples</LI></UL><UL><DIV id="Sept2"><H2>September 
Week 2</H2><UL><LI>We continued to work with DVL2s. We ran ligation of lacZ  amplified with level 2 fusion sites and pSB1A2 as backbone and transformed them. However, they did not grow blue, we could only see white colonies. We tried once more but they are still not growing. We are working on troubleshooting them.</LI><LI>We kept troubleshooting the DVL0s that didn`t work. We picked colonies of DVL0_BFCR plate and miniprepped 3 samples to check the correct sequence. We submmited them for sequencing along with previous MoClo reactions 1 and 14-22. We continued with the MoClo reactions and we ran reactions 23-49.</LI></UL><UL><LI>We transformed all MoClo reactions (23-49) with Zymo cells. Unfortunately, the cells were not very competent and none of plates grew sufficiently.  We tried transforming once more with Bioline Gold Cells, this time we included Moclo 50-54 .</LI></UL>
table moclo 50-54.<UL><LI>We started our first Flow Cytometry experiments. In the first day, we streaked plates for getting fresh colonies. The samples were:</LI><UL><LI>Amp: GFP+, RFP+, lacI-&gt;pLacI(IPTG curv), TetR-&gt;pTetR(aTe Curve), TetR inverter (Ara curve), LuxR-&gt;pLuxR(AHL curve)</LI><LI>Negative control</LI></UL><LI>In the second day, we picked colonies for flow cytometry in triplicates and let them growing in 2 mL of culture for 6 hours. After that, we added: 199ul of LB with proper antibiotic and inducer/ repressor molecules to tubes and innoculated 1 uL of the culture and let them growing overnight. </LI><LI>In the third day, we added 450ul of PBS and 50 uL of overnight cultures to the Flow Cytometry tubes and ran the experiments. The graphs and results can be seen in Results page. </LI></UL><DIV id="Sept3"><H2>September
Week 3</H2><UL>
li&gt;We continued troubleshooting the DVL0s that didn`t work and picked colonies from a DVL0_AF_BR plate and send them for sequencing. We also sent MoClo 2, 4, 5, 7, 8, 9, 10, 11, 12, 13 and 21 for sequencing.<LI>We transformed another set of Moclo reactions using Bioline Gold cells: 23-29 and 50-54, picked colonies and ran miniprep on them. We picked two colonies for each gene and four colonies for each RBS. We quantified all the 34 minipreps.</LI></UL><LI>We ran more Flow Cytometry experiments. In the first day, we streaked plates for getting fresh colonies. The samples were:pConst + rbs-lacI-GFPc-pLacI-RFPc , pConst + rbs-tetR-GFPc-pTet-RFPc,  I13458-I13453+B0032+C0040+B0034+E0040+B0015+R0040+B0034+E1010+B0015, R0040-B0034-C0062-B0015-R0065-J23101, GFP positive control, RFP positive control and negative control. We followed the same protocol from last week and ran the Flow Cytometry experiment.</LI><UL><LI>We ran more MoClo reactions (55-66) and transformed them using Zymo cells. </LI></UL><UL><LI>We  picked colonies from  MoClo plates (30-39) ran minipreps in the samples  and quantified them.</LI></UL><DIV id="Sept4"><H2>September
Week 4</H2><UL><H7><LI>We sent  53 MoClo reactions for sequencing: MoClo23- MoClo 54, with duplicates for genes and quadruples for RBSs.</LI></H7></UL><UL><LI>We transformed them using Zymo cells and overloaded the reaction using 50 uL of cells and 4 uL of DNA. We picked 4 colonies for each transformation the next day and ran a small diges with X,P to check the sizes. However, they were incorrect, and we are in the process of troubleshooting.</LI></UL><UL><LI>We quantified the ones that were correct and streaked plates for running Flow Cytometry experiments. </LI></UL><UL><LI>For the Level 0 MoClo parts that didn`t show the correct sequences we picked two new colonies from each plate we had.  MoClo 36, 37,  38, 39, 40, 10, 32, 27, 26, 6, 7, 8, 2, 3, 29, 23, 24, 30, 11, 43, 44, 16, 45, 4, 47, 33, 48 and 49. We ran small digest with x and p and ran them on a gel to check the sizes.</LI></UL><UL><LI>We submitted the samples that seemed to show the correct band for sequencing. </LI></UL><UL><LI> Also, we worked on streaking plates for the samples we are submitting to the registry.</LI></UL><DIV id="Oct1"><H2>October
Week 1</H2><UL><LI>We miniprepped the samples we are sending to the registry and they are ready for submission :) The list of parts can be seen in the section Parts Submitted. </LI><LI>We picked colonies for the Flow Cytometry Experiment: All the Moclo L1 parts that worked (L1.2, L1.4, L1.5, L1.6, L1.7, L1.8 L1.10) along with their Biobrick counterparts. We ran the Flow Cytometry Experiments and the results can be checked at the section Results Summary. </LI><LI>We went over MIT to drop off the samples for the Registry.</LI><LI>Wiki is freezed! See you at the Jamboree :)</LI></UL><DIV id="Oct2"><H2>October Week 2</H2><UL><H7><LI>One problem that we found out was keeping us from creating Level1 parts was because we mislabeled tubes, and were using incorrect Level 1 destination vectors to ligate with the parts. </LI><LI>We transformed with bioline cells level 2 destination vectors, but there were not many blue colonies upon blue white screening.</LI><LI>We continued to convert biobrick promoters of the J series into level 0 moclo parts and also set up moclo level 1 reactions. They were transformed and picked using blue white screening. The next day, we prepared cultures and then ran prep. We checked the constructs by running  digest with X,P for MoClo level 0 and level 1,  then we ran gel to confirm.</LI><LI>The correct bands were then sequenced and they yielded the correct orientation and nucleotide order in the forward direction.</LI><LI>In terms of troubleshooting DVL2s, we decided to remake the pSB1A2 backbone from a glycerol stock of the biobrick of C0080  and digested it with X,P, in order to cut out the part. </LI><LI>And we are going to the iGEM Regionals at Pittsburgh! </LI></H7></UL><DIV id="Oct3"><H2>October Week 3</H2><UL><LI>We are very excited to be back in the lab and work for the Worlds Jamboree!! :)</LI><LI>First, we are taking care of transferring the parts we submitted from pSB1A2 to pSB1C3.  We did this by digesting all the parts with E, S and pSB1C3 also. We ligated the digested parts and pSB1C3 and transformed.  However, none of them grew. We believe this is due to the low yield of Gel Extraction.</LI><LI>We also added fusion sites to R0040 using ligation PCR, we verified them on the gel and gel extracted it. </LI><DIV id="Oct4"><H2>October Week 4</H2><UL><H7><LI>We transformed a new set of Moclo Level 0 reactions , We got white colonies for the blue-white screening and we made cultures and ran a prep. However, when we digested them to confirm the sizes none of the parts showed the bands we were expecting.  We think this might be due to the amount of colonies we screened for each part, usually we have to screen four colonies per part and we just screened two.</LI><LI>Regarding the MoClo Level 1, we set up new reactions using the Promega buffer. Also, we doubled the volume of BsaI after we realized it was two times lower than predicted. We transformed them but we didn`t get any colonies.  We tried new reaction with a higher volume of Ligase and we are waiting for transformation to check them. Fingers crossed :)</LI><LI>Because the initial effort to convert the biobrick parts into the pSB1C3 backbone didn’t work, we tried the entire process again. However, as of Friday 10/26, we haven`t get to the point of transforming the ligations. </LI><LI>In the process of creating level 2 destination vectors, we already isolated the backbone from a biobrick part, we then digested with SpeI and gel extracted it in preparation for ligation with lacZ. </LI><LI>Afterwards we ligated Lacz with different sites
</LI><LI> New results will  included as part of our presentation. See you all at MIT!</LI></H7></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:BostonU/Notebook">http://2012.igem.org/Team:BostonU/Notebook</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:BostonU/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:BostonU/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:BostonU/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:BostonU/Notebook&amp;oldid=287304" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></UL></DIV></H7></DIV></H7></DIV></H7></DIV></H7></DIV></H7></DIV></H7></DIV></H7></DIV></H7></DIV></H7></DIV></H7></DIV></H7></DIV></H7></DIV></H7></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>