<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Minnesota_Project_Modelling skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Minnesota/Project/Modelling</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html PUBLIC &quot;-//W3C//DTD XHTML 1.0 Transitional//EN&quot; &quot;<A rel="nofollow" class="external free" href="http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd</A>&quot;&gt;<TITLE>Team:Minnesota/Project/Pichia Expression System - 2015.igem.org</TITLE></P><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Minnesota/Project/Pichia_Expression_System">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Minnesota/Project/Pichia_Expression_System&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Minnesota/Project/Pichia_Expression_System&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Minnesota/Project/Pichia_Expression_System&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Minnesota/Project/Pichia_Expression_System" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Minnesota/Project/Pichia Expression System</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2015.igem.org</H3><P><TITLE>Team:Minnesota - Main Style Template</TITLE><TITLE>Team:Minnesota - Template</TITLE></P><DIV id="ContentAll"><DIV id="ContentAll"><DIV id="SideBarAll"><DIV class="sideNavBar2"><UL><LI><A href="C:\Documents and Settings\aj\Desktop\IGEM webpage/TeamMinnesota.html">HOME</A></LI><LI><A href="C:\Documents and Settings\aj\Desktop\IGEM webpage/TeamMinnesota/Team.html">TEAM</A></LI><LI><A class="current" href="C:\Documents and Settings\aj\Desktop\IGEM webpage/Project.html">PROJECT</A></LI><LI><A href="C:\Documents and Settings\aj\Desktop\IGEM webpage/Notebook.html">NOTEBOOK</A></LI><LI><A href="C:\Documents and Settings\aj\Desktop\IGEM webpage/Attributions.html">ATTRIBUTIONS</A></LI><LI><A href="C:\Documents and Settings\aj\Desktop\IGEM webpage/Safety.html">SAFETY</A></LI><LI><A href="C:\Documents and Settings\aj\Desktop\IGEM webpage/Sponsors.html">SPONSORS</A></LI></UL></DIV><DIV id="MainBoxAll"><DIV id="MainBoxContent" style="background-color:white;"><DIV style="position:absolute;top:0px;margin-left:50px; width:600px; height:450px; overflow-y:scroll; text-align:left; overflow-x:hidden;"><H1>Modelling Title</H1><B><FONT size="4">Modelling Background Info </FONT></B><P><I><U><FONT size="3">• </FONT>What's the idea behind this expression system?</U></I>    Laboratory and industrial projects that involve the high volume production of a protein often select E. coli as an expression system due to its rapid growth. However, bacterial expression systems are not always a viable option. In the case where proper folding of the protein of interest requires post-translational modification (such as the addition of disulfide bonds or glycosylation,) a eukaryote must be used. Although several well-defined eukaryotic options exist, yeast is often selected for its ease of use in the laboratory. One yeast species in particular, Pichia pastoris, has gained popularity as an expression system for recombinant human proteins. 
<I><U><FONT size="4">• </FONT>Why did we choose P. Pastoris?</U></I>    P. pastoris has a glycosylation pattern that is more compatible with the human immune system, when compared to the glycosylation pattern of Saccharomyces cerevisiae. P. pastoris is also known for its ability to grow in high densities using methanol as its only food source. Despite the usefulness of yeast species such as P. pastoris there are currently few items in the parts registry that are designed for use within yeast, and none that are specifically designed to be used with P. pastoris. Our team intends on producing pBB3G1 and pBB1Z1, two BioBrick compatible P. pastoris-E. coli shuttle vectors.
<I><U><FONT size="4">• </FONT>What are some benefits of this vector system?</U></I>    The pBB3G1 and pBB1Z1 vectors include features that increase its ease-of-use and versatility, such as: optional inducibility, optional product secretion, trans-kingdom conjugation (TKC), and episomal maintenance of the vector in the host organism. The vectors vary in their expression level. Constitutive expression is achieved in pBB1Z1 by the pGAP promoter. Methanol-induced expression is available in pBB3G1 by means of the pAOX1 promoter.
<I><U><FONT size="4">• </FONT> Why use Trans Kingdom Conjugation instead of traditional transformation methods?</U></I>    TKC involves the transfer of DNA from a bacterial cell to a eukaryotic cell, by means of conjugation. Harnessing the ability to shuttle plasmids between E. coli and P. pastoris through TKC would simplify transformation protocols. Currently, transformation methods (using shuttle plasmids cloned in E. coli) are a time consuming process, requiring isolation of the cloned plasmid and transformation into yeast. Transformation using TKC shortens the process by transferring the plasmid directly into the yeast cell. Utilizing TKC as a transformation protocol would translate to faster results in the laboratory and reduced costs in an industrial setting. Currently there are no BioBrick vectors in the parts registry that enable TKC between E. coli and P. pastoris.
<I><U><FONT size="4">• </FONT>How will TKC be achieved?</U></I>    TKC functionality is provided by the OriT<SUP>P</SUP> sequence which hosts the nick site that will be cleaved during the initiation of conjugation in order to linearize the plasmid, and ligated once transferred to the recipient. Importantly, the OriT<SUP>P</SUP> sequence -compared to other variations of OriT- does not require the presence of a helper plasmid within the recipient to complete the final ligation step of conjugal transfer.  
<I><U><FONT size="4">• </FONT>Are there limitations to this system?</U></I>    One limitation to using P. pastoris as an expression system is that most shuttle plasmids must be integrated into the yeast chromosome. This results in lower expression of desired protein products, as well as lower transformation efficiency. We hope to improve the functionality of the pBB3G1/pBB1Z1 system by allowing the plasmid -once transferred to the yeast cell- to remain as an episomal plasmid. This is made possible by the inclusion of the PARS1 yeast autonomous replication sequence. This sequence ensures that the plasmid is maintained through several (~200) generations.
<I><U><FONT size="4">• </FONT>How will we screen the genes?</U></I>    Transformant selection is simplified by the inclusion of antibiotic resistance genes that are effective in both E. coli and P. pastoris. The backbones that will be submitted to the parts registry will include either Zeocin or Geneticin, however the resistance genes have been cloned into a KpnI site, and may be swapped for any selective marker specific to the user’s needs.


</P><P><B><FONT size="3">Methods</FONT></B><B><FONT size="4">• </FONT>Vector Design</B>   Sequences for AOX1, GAP, G418, and Zeocin were obtained from pPICZ, pPICZ?, pGAP from Invitrogen. The Wintergreen odor generator (BBa_J45700) was selected as a reporter gene, as well as yeast green fluorescent protein (GFP). In order to improve upon an existing part in the Registry of Standard Biological Parts, we codon-optimized the Wintergreen odor generator for expression in P. pastoris using the Codon Optimization tool provided by IDT. Kozak sequences were added for each open reading frame to be expressed in P. pastoris. The basal vectors pMNBB-ICI and pMNBB-CCI (please refer to Vector Nomenclature, and Figure 1) were assembled from the above sequences in Clone Manager 9.

<B><I>Figure 1. Basal vectors of the pMNBB P. pastoris expression system. Note that the basal vectors do not contain the oriT sequence that enables conjugation, nor the PARS1 sequence that confers episomal maintenance. Restriction enzymes presented here are entirely unique to ensure compatibility with the BioBrick™ platform.</I></B>   Our approach for assembling these backbones was based on PCR amplification and Gibson assembly rather than a restriction enzyme-mediated cloning strategy. The sequences for pMNBB-ICI and pMNBB-CCI were divided into four and five pieces, respectively. The first 20 bp of the 5’ end of each partition contains the final 20 bp of the 3’ end of the preceding piece. For example, the first 20 bp of the fourth piece of pMNBB-ICI are identical to the final 20 bp of the 3’ end of the third piece of pMNBB-ICI. These complementary sequences allow pieces to be joined together via overlap extension PCR. Once joined, the vector backbones can be circularized using Gibson assembly (Gibson et al. 2009).
Once the basal vectors are created, elements may be swapped using restriction enzymes. For example, the basal, methanol-inducible vector pMNBB-ICI contains the pAOX1 promoter without an internal OriT element. Swapping pAOX1 for pAOX1-OriT would lead to the generation of pMNBB-ITI, a version of the pMNBB vector, which contains a methanol-inducible promoter (pAOX) and can be transferred to P. pastoris using TKC (Figure 2).
<I><B>Figure 2. Creating the conjugal variant of pMNBB-IXX. (A.) As seen in pMNBB-ICI, the pAOX1 promoter does not contain the OriT sequence. (B.) Addition of the OriT sequence within the AOX1 promoter enables conjugal transfer.
</B></I>   The project was designed to yield two basal P. pastoris expression vectors, as well as two variations upon the basal vectors, as listed in Table 1.
<B><FONT size="4">• </FONT>Vector Assembly</B>   We began assembling the vectors by amplifying the individual fragments by PCR using Phusion HF polymerase (New England Biolabs). Next, we attempted several transformations of E. coli C2566 with Gibson assemblies of the vectors. Unfortunately the transformations resulted in either no growth, or growth of unidentifiable colonies, as determined by Go-Green Taq (Promega) colony screens. We believe that the lack of growth may be due to vector not circularizing, or in the case of the unidentifiable colonies that appeared on the pMNBB-CCI plates, limited selectivity imparted by our Zeocin media (25ug/mL).
<B><FONT size="4">• </FONT>Results</B>   First we investigated the possibility that the vectors were not circularizing. In order to simplify the Gibson assembly we began overlapping fragments, reducing the pieces that would need to join in a successful Gibson assembly. The initial two piece overlaps were successful, however overlaps beyond two pieces were problematic. Gibson assemblies in which the fragment number had been reduced by using two piece overlaps produced the same results as the single piece Gibson assemblies, either no growth, or growth of unidentified colonies.
We decided to try an alternate PCR protocol (Shevchuk et al. 2004). Finally we were able to string together the five fragments of pMNBB-ICI. 
<B><FONT size="4">• </FONT>Conclusions</B>   Unfortunately we have not yet been able to assemble either vector in its entirety. The basal inducible vector, pMNBB-ICI, has been visualized in its linear state on a gel. We will continue to work on assembling both vectors so that they may be available for future teams to improve upon and characterize. If the pMNBB system functions as expected, it will certainly be a very useful tool not only for iGEM teams, but for academic research, and industrial processes.



<A href="C:\Documents and Settings\aj\Desktop\IGEM webpage/Project.html">Click here to return to Projects page.</A></P></DIV></DIV><DIV id="MainBoxOtherLinks"><P class="otherLinks"><A href="https://2015.igem.org/Main_Page">iGEM Home</A> | <A href="https://igem.org/Team.cgi?id=1187">Team Minnesota Info</A> | 
<A href="http://www1.umn.edu/twincities/index.html">University of Minnesota Home</A> |  <A href="C:\Documents" and="" settings="">Contact Us!</A></P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="C:\Documents and Settings\aj\Desktop\IGEM webpage/Modelling.html">C:\Documents and Settings\aj\Desktop\IGEM webpage/Modelling.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Minnesota/Project/Pichia_Expression_System" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Minnesota/Project/Pichia_Expression_System" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Minnesota/Project/Pichia_Expression_System&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Minnesota/Project/Pichia_Expression_System&amp;oldid=235719" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2015.igem.org:Privacy_policy" title="2015.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2015.igem.org:General_disclaimer" title="2015.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>