Team:Minnesota/Parts
Add a banner to your wiki!
You can make the image 980px  by  200px
Remember to call the file: "Team_Minnesota_banner.jpg"
HOME
TEAM
PROJECT
Description
Experiments & Protocols
Results
Design
PARTS
Team Parts
Basic Parts
Composite Parts
Part Collection
NOTEBOOK
ATTRIBUTIONS
COLLABORATIONS
HUMAN PRACTICES
SAFETY
MODELING
MEASUREMENT
SOFTWARE
ENTREPRENEURSHIP
Part Documentation
Each team will make new parts during iGEM and will submit them to the Registry of Standard Biological Parts. The iGEM software provides an easy way to present the parts your team has created. The <groupparts> tag (see below) will generate a table with all of the parts that your team adds to your team sandbox.
Remember that the goal of proper part documentation is to describe and define a part, so that it can be used without needing to refer to the primary literature. Registry users in future years should be able to read your documentation and be able to use the part successfully. Also, you should provide proper references to acknowledge previous authors and to provide for users who wish to know more.
Note
Note that parts must be documented on the Registry . This page serves to showcase the parts you have made. Future teams and other users and are much more likely to find parts by looking in the Registry than by looking at your team wiki.
CrtE
Protein data table for BioBrick BBa_ automatically created by the BioBrick-AutoAnnotator version 1.0
Nucleotide sequence in RFC 10: (underlined part encodes the protein)
 ATGGTCACA ... AGGGATCGTTAA
 ORF from nucleotide position 1 to 900 (excluding stop-codon)
Amino acid sequence: (RFC 25 scars in shown in bold, other sequence features underlined; both given below)
1 
101 
201 
301 
MVTAVRTEFDFAAYLRAARESVELALDGSLGTERPTSLSEAMRYSLLAGGKRLRPILCLAACELAGGNVAAAMPTAVALEMIHTMSLIHDDLPAMDNDDL
RRGRPTNHKVYGEATAILAGDALLTRAFEIVALRSPGVSPECILKVIGELSLAAGAPGLVGGQMVDLESEGKEVNLDTLEYIHLHKTGALLSACVICGGL
IGDASDELLQALRNYARGIGLAFQIIDDILDVTASSDVLGKTAGKDLVADKTTYPKLLGLKASRERAEQLVAEAKQGLDPWRDRAAPLLALADYITGRDR
*
Sequence features: (with their position in the amino acid sequence, see the list of supported features )
None of the supported features appeared in the sequence
Amino acid composition:
Ala (A)
43 (14.3%)
Arg (R)
20 (6.7%)
Asn (N)
5 (1.7%)
Asp (D)
22 (7.3%)
Cys (C)
5 (1.7%)
Gln (Q)
5 (1.7%)
Glu (E)
19 (6.3%)
Gly (G)
27 (9.0%)
His (H)
5 (1.7%)
Ile (I)
15 (5.0%)
Leu (L)
45 (15.0%)
Lys (K)
11 (3.7%)
Met (M)
7 (2.3%)
Phe (F)
4 (1.3%)
Pro (P)
11 (3.7%)
Ser (S)
15 (5.0%)
Thr (T)
16 (5.3%)
Trp (W)
1 (0.3%)
Tyr (Y)
7 (2.3%)
Val (V)
17 (5.7%)
Amino acid counting
Total number:
300
Positively charged (Arg+Lys):
31 (10.3%)
Negatively charged (Asp+Glu):
41 (13.7%)
Aromatic (Phe+His+Try+Tyr):
17 (5.7%)
Biochemical parameters
Atomic composition:
C1396H2292N392O431S12
Molecular mass [Da]:
31848.6
Theoretical pI:
5.08
Extinction coefficient at 280 nm [M-1 cm-1]:
15930 / 16243 (all Cys red/ox)
Plot for hydrophobicity, charge, predicted secondary structure, solvent accessability, transmembrane helices and disulfid bridges 
Codon usage
Organism:
E. coli
B. subtilis
S. cerevisiae
A. thaliana
P. patens
Mammals
Codon quality ( CAI ):
good (0.70)
good (0.76)
good (0.74)
good (0.74)
good (0.76)
good (0.62)
Alignments (obtained from PredictProtein.org )
   There were no alignments for this protein in the data base. The BLAST search was initialized and should be ready in a few hours.Predictions (obtained from PredictProtein.org )   There were no predictions for this protein in the data base. The prediction was initialized and should be ready in a few hours. The BioBrick-AutoAnnotator was created by TU-Munich 2013 iGEM team. For more information please see the documentation .
If you have any questions, comments or suggestions, please leave us a comment .
CrtB
Protein data table for BioBrick BBa_ automatically created by the BioBrick-AutoAnnotator version 1.0
Nucleotide sequence in RFC 10: (underlined part encodes the protein)
 ATGCTGAAC ... GGTTCCGACTGA
 ORF from nucleotide position 1 to 1017 (excluding stop-codon)
Amino acid sequence: (RFC 25 scars in shown in bold, other sequence features underlined; both given below)
1 
101 
201 
301 
MLNESQVEAGKEIQQRTGKTFYFATRLLPERVRHATYVLYAFFRVADEVVDAEETAPPEQQRERLERLRAEALGDVETGDPVLSAFAELREQYDIDPDDV
NVFIDAMESDIHKDRHATYEELETYMDGSASAVGRMMTAVMRPDDPKQALPHATALGQAFQLSNFLRDVGEDVVERDRVYLPETTLERHGVDVAQILDLE
FDDRVAEVMRDELRRAESLYRDGVAGIEYLPHDCQFPVLLSAVLYADHHRSIRKLGYDTLSTTPDLSFGRKVFLLGKTAAYWAVWRDPVAVFKAVSVVPY
PEDEEFDADFGPGSTPVGKVRSRGRVSGWIRALSRWGSD*
Sequence features: (with their position in the amino acid sequence, see the list of supported features )
None of the supported features appeared in the sequence
Amino acid composition:
Ala (A)
33 (9.7%)
Arg (R)
31 (9.1%)
Asn (N)
3 (0.9%)
Asp (D)
32 (9.4%)
Cys (C)
1 (0.3%)
Gln (Q)
11 (3.2%)
Glu (E)
30 (8.8%)
Gly (G)
20 (5.9%)
His (H)
8 (2.4%)
Ile (I)
8 (2.4%)
Leu (L)
30 (8.8%)
Lys (K)
9 (2.7%)
Met (M)
7 (2.1%)
Phe (F)
15 (4.4%)
Pro (P)
17 (5.0%)
Ser (S)
17 (5.0%)
Thr (T)
17 (5.0%)
Trp (W)
4 (1.2%)
Tyr (Y)
13 (3.8%)
Val (V)
33 (9.7%)
Amino acid counting
Total number:
339
Positively charged (Arg+Lys):
40 (11.8%)
Negatively charged (Asp+Glu):
62 (18.3%)
Aromatic (Phe+His+Try+Tyr):
40 (11.8%)
Biochemical parameters
Atomic composition:
C1703H2637N475O525S8
Molecular mass [Da]:
38422.0
Theoretical pI:
4.79
Extinction coefficient at 280 nm [M-1 cm-1]:
41370 / 41433 (all Cys red/ox)
Plot for hydrophobicity, charge, predicted secondary structure, solvent accessability, transmembrane helices and disulfid bridges 
Codon usage
Organism:
E. coli
B. subtilis
S. cerevisiae
A. thaliana
P. patens
Mammals
Codon quality ( CAI ):
good (0.70)
good (0.76)
good (0.73)
good (0.74)
good (0.75)
good (0.66)
Alignments (obtained from PredictProtein.org )
   There were no alignments for this protein in the data base. The BLAST search was initialized and should be ready in a few hours.Predictions (obtained from PredictProtein.org )Subcellular Localization (reliability in brackets)
Archaea:
arc (%)
Bacteria:
bac (%)
Eukarya:
euk (%)
Gene Ontology (reliability in brackets)
Molecular Function Ontology:
0005515 0. (NaN%), 0004310 0. (NaN%)
Biological Process Ontology:
0016117 0. (NaN%), 0015995 0. (NaN%)
 
Predicted features:
Disulfid bridges:
 -
Transmembrane helices:
 -
The BioBrick-AutoAnnotator was created by TU-Munich 2013 iGEM team. For more information please see the documentation .
If you have any questions, comments or suggestions, please leave us a comment .
CrtI
Protein data table for BioBrick BBa_ automatically created by the BioBrick-AutoAnnotator version 1.0
Nucleotide sequence in RFC 10: (underlined part encodes the protein)
 ATGGATGAC ... TTGTCCACTTAA
 ORF from nucleotide position 1 to 1278 (excluding stop-codon)
Amino acid sequence: (RFC 25 scars in shown in bold, other sequence features underlined; both given below)
1 
101 
201 
301 
401 
MDDSHIVVVGAGLAGLVAARHLAVDGADVTVIERRDDVGGRVRTRTKDGFTLDRGFQVLLTGYPAVRRELDFDALDLRPLAPGATICRPSRRSVLSDPLR
DINGLFDSLRNPEVTTSDKLRTLALRQDVSTRDEGEIFDSPDQSIRAYLRDWGFSEDFIEHFVAPFYGGITLDRSLSTSKRVFEYTFKMLATGRAAVPAD
GMQAIPEQLAATARDEGATIRLGERVESVQSDADGAVVATDDESVEADAVVVATDPKEARRLTGVGSIPTDARGCVTQYYTLPTGSGLDAGKRIMLNAPN
PDPNTVVPLSTVAPEYAPPGKELLNATFLGAAAQDESEEELFEKTRRTLEAWYPERYFDDLELLHTDYISFAQFAQPPEIHDSLPDHRDAPGRTYLAGDY
TAWSSIQGAMRSGKEAADAIRDDLST*
Sequence features: (with their position in the amino acid sequence, see the list of supported features )
None of the supported features appeared in the sequence
Amino acid composition:
Ala (A)
48 (11.3%)
Arg (R)
37 (8.7%)
Asn (N)
6 (1.4%)
Asp (D)
44 (10.3%)
Cys (C)
2 (0.5%)
Gln (Q)
11 (2.6%)
Glu (E)
27 (6.3%)
Gly (G)
32 (7.5%)
His (H)
6 (1.4%)
Ile (I)
16 (3.8%)
Leu (L)
39 (9.2%)
Lys (K)
9 (2.1%)
Met (M)
5 (1.2%)
Phe (F)
16 (3.8%)
Pro (P)
25 (5.9%)
Ser (S)
26 (6.1%)
Thr (T)
32 (7.5%)
Trp (W)
3 (0.7%)
Tyr (Y)
12 (2.8%)
Val (V)
30 (7.0%)
Amino acid counting
Total number:
426
Positively charged (Arg+Lys):
46 (10.8%)
Negatively charged (Asp+Glu):
71 (16.7%)
Aromatic (Phe+His+Try+Tyr):
37 (8.7%)
Biochemical parameters
Atomic composition:
C2037H3206N578O656S7
Molecular mass [Da]:
46513.8
Theoretical pI:
4.65
Extinction coefficient at 280 nm [M-1 cm-1]:
34380 / 34505 (all Cys red/ox)
Plot for hydrophobicity, charge, predicted secondary structure, solvent accessability, transmembrane helices and disulfid bridges 
Codon usage
Organism:
E. coli
B. subtilis
S. cerevisiae
A. thaliana
P. patens
Mammals
Codon quality ( CAI ):
good (0.69)
good (0.73)
good (0.74)
good (0.76)
good (0.79)
good (0.67)
Alignments (obtained from PredictProtein.org )
   There were no alignments for this protein in the data base. The BLAST search was initialized and should be ready in a few hours.Predictions (obtained from PredictProtein.org )Subcellular Localization (reliability in brackets)
Archaea:
arc (%)
Bacteria:
bac (%)
Eukarya:
euk (%)
Gene Ontology (reliability in brackets)
Molecular Function Ontology:
0050660 0. (NaN%), 0080103 0. (NaN%)
Biological Process Ontology:
0015995 0. (NaN%), 0015979 0. (NaN%)
 
Predicted features:
Disulfid bridges:
 -
Transmembrane helices:
 -
The BioBrick-AutoAnnotator was created by TU-Munich 2013 iGEM team. For more information please see the documentation .
If you have any questions, comments or suggestions, please leave us a comment .
Contents
1 Adding parts to the registry
2 What information do I need to start putting my parts on the Registry?
3 Inspiration
4 Part Table
Adding parts to the registry
You can add parts to the Registry at our <a href=" http://parts.igem.org/Add_a_Part_to_the_Registry ">Add a Part to the Registry</a> link.
We encourage teams to start completing documentation for their parts on the Registry as soon as you have it available. The sooner you put up your parts, the better you will remember all the details about your parts. Remember, you don't need to send us the DNA sample before you create an entry for a part on the Registry. (However, you do need to send us the DNA sample before the Jamboree. If you don't send us a DNA sample of a part, that part will not be eligible for awards and medal criteria.)
What information do I need to start putting my parts on the Registry?
The information needed to initially create a part on the Registry is:
Part Name
Part type
Creator
Sequence
Short Description (60 characters on what the DNA does)
Long Description (Longer description of what the DNA does)
Design considerations
We encourage you to put up much more information as you gather it over the summer. If you have images, plots, characterization data and other information, please also put it up on the part page.
Inspiration
We have a created  a <a href=" http://parts.igem.org/Well_Documented_Parts ">collection of well documented parts</a> that can help you get started.
You can also take a look at how other teams have documented their parts in their wiki:
<a href=" https://2014.igem.org/Team:MIT/Parts "> 2014 MIT </a>
<a href=" https://2014.igem.org/Team:Heidelberg/Parts "> 2014 Heidelberg</a>
<a href=" https://2014.igem.org/Team:Tokyo_Tech/Parts ">2014 Tokyo Tech</a>
Part Table
</html> <groupparts>iGEM015 Example</groupparts>
