Team:Minnesota/Project/Modelling
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" " http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd ">
Team:Minnesota/Project/Pichia Expression System - 2015.igem.org
"
Page
Discussion
View source
History
teams
Log in
 
Team:Minnesota/Project/Pichia Expression System
From 2015.igem.org
Team:Minnesota - Main Style Template
Team:Minnesota - Template
HOME
TEAM
PROJECT
NOTEBOOK
ATTRIBUTIONS
SAFETY
SPONSORS
Modelling Title
Modelling Background Info
• What's the idea behind this expression system?
      Laboratory and industrial projects that involve the high volume production of a protein often select E. coli as an expression system due to its rapid growth. However, bacterial expression systems are not always a viable option. In the case where proper folding of the protein of interest requires post-translational modification (such as the addition of disulfide bonds or glycosylation,) a eukaryote must be used. Although several well-defined eukaryotic options exist, yeast is often selected for its ease of use in the laboratory. One yeast species in particular, Pichia pastoris, has gained popularity as an expression system for recombinant human proteins.
• Why did we choose P. Pastoris?
      P. pastoris has a glycosylation pattern that is more compatible with the human immune system, when compared to the glycosylation pattern of Saccharomyces cerevisiae. P. pastoris is also known for its ability to grow in high densities using methanol as its only food source. Despite the usefulness of yeast species such as P. pastoris there are currently few items in the parts registry that are designed for use within yeast, and none that are specifically designed to be used with P. pastoris. Our team intends on producing pBB3G1 and pBB1Z1, two BioBrick compatible P. pastoris-E. coli shuttle vectors.
• What are some benefits of this vector system?
      The pBB3G1 and pBB1Z1 vectors include features that increase its ease-of-use and versatility, such as: optional inducibility, optional product secretion, trans-kingdom conjugation (TKC), and episomal maintenance of the vector in the host organism. The vectors vary in their expression level. Constitutive expression is achieved in pBB1Z1 by the pGAP promoter. Methanol-induced expression is available in pBB3G1 by means of the pAOX1 promoter.
• Why use Trans Kingdom Conjugation instead of traditional transformation methods?
      TKC involves the transfer of DNA from a bacterial cell to a eukaryotic cell, by means of conjugation. Harnessing the ability to shuttle plasmids between E. coli and P. pastoris through TKC would simplify transformation protocols. Currently, transformation methods (using shuttle plasmids cloned in E. coli) are a time consuming process, requiring isolation of the cloned plasmid and transformation into yeast. Transformation using TKC shortens the process by transferring the plasmid directly into the yeast cell. Utilizing TKC as a transformation protocol would translate to faster results in the laboratory and reduced costs in an industrial setting. Currently there are no BioBrick vectors in the parts registry that enable TKC between E. coli and P. pastoris.
• How will TKC be achieved?
      TKC functionality is provided by the OriTP sequence which hosts the nick site that will be cleaved during the initiation of conjugation in order to linearize the plasmid, and ligated once transferred to the recipient. Importantly, the OriTP sequence -compared to other variations of OriT- does not require the presence of a helper plasmid within the recipient to complete the final ligation step of conjugal transfer.
• Are there limitations to this system?
      One limitation to using P. pastoris as an expression system is that most shuttle plasmids must be integrated into the yeast chromosome. This results in lower expression of desired protein products, as well as lower transformation efficiency. We hope to improve the functionality of the pBB3G1/pBB1Z1 system by allowing the plasmid -once transferred to the yeast cell- to remain as an episomal plasmid. This is made possible by the inclusion of the PARS1 yeast autonomous replication sequence. This sequence ensures that the plasmid is maintained through several (~200) generations.
• How will we screen the genes?
      Transformant selection is simplified by the inclusion of antibiotic resistance genes that are effective in both E. coli and P. pastoris. The backbones that will be submitted to the parts registry will include either Zeocin or Geneticin, however the resistance genes have been cloned into a KpnI site, and may be swapped for any selective marker specific to the user’s needs.
Methods
• Vector Design
   Sequences for AOX1, GAP, G418, and Zeocin were obtained from pPICZ, pPICZ?, pGAP from Invitrogen. The Wintergreen odor generator (BBa_J45700) was selected as a reporter gene, as well as yeast green fluorescent protein (GFP). In order to improve upon an existing part in the Registry of Standard Biological Parts, we codon-optimized the Wintergreen odor generator for expression in P. pastoris using the Codon Optimization tool provided by IDT. Kozak sequences were added for each open reading frame to be expressed in P. pastoris. The basal vectors pMNBB-ICI and pMNBB-CCI (please refer to Vector Nomenclature, and Figure 1) were assembled from the above sequences in Clone Manager 9.
Figure 1. Basal vectors of the pMNBB P. pastoris expression system. Note that the basal vectors do not contain the oriT sequence that enables conjugation, nor the PARS1 sequence that confers episomal maintenance. Restriction enzymes presented here are entirely unique to ensure compatibility with the BioBrick™ platform.
   Our approach for assembling these backbones was based on PCR amplification and Gibson assembly rather than a restriction enzyme-mediated cloning strategy. The sequences for pMNBB-ICI and pMNBB-CCI were divided into four and five pieces, respectively. The first 20 bp of the 5’ end of each partition contains the final 20 bp of the 3’ end of the preceding piece. For example, the first 20 bp of the fourth piece of pMNBB-ICI are identical to the final 20 bp of the 3’ end of the third piece of pMNBB-ICI. These complementary sequences allow pieces to be joined together via overlap extension PCR. Once joined, the vector backbones can be circularized using Gibson assembly (Gibson et al. 2009). Once the basal vectors are created, elements may be swapped using restriction enzymes. For example, the basal, methanol-inducible vector pMNBB-ICI contains the pAOX1 promoter without an internal OriT element. Swapping pAOX1 for pAOX1-OriT would lead to the generation of pMNBB-ITI, a version of the pMNBB vector, which contains a methanol-inducible promoter (pAOX) and can be transferred to P. pastoris using TKC (Figure 2).
Figure 2. Creating the conjugal variant of pMNBB-IXX. (A.) As seen in pMNBB-ICI, the pAOX1 promoter does not contain the OriT sequence. (B.) Addition of the OriT sequence within the AOX1 promoter enables conjugal transfer.
   The project was designed to yield two basal P. pastoris expression vectors, as well as two variations upon the basal vectors, as listed in Table 1.
• Vector Assembly
   We began assembling the vectors by amplifying the individual fragments by PCR using Phusion HF polymerase (New England Biolabs). Next, we attempted several transformations of E. coli C2566 with Gibson assemblies of the vectors. Unfortunately the transformations resulted in either no growth, or growth of unidentifiable colonies, as determined by Go-Green Taq (Promega) colony screens. We believe that the lack of growth may be due to vector not circularizing, or in the case of the unidentifiable colonies that appeared on the pMNBB-CCI plates, limited selectivity imparted by our Zeocin media (25ug/mL).
• Results
   First we investigated the possibility that the vectors were not circularizing. In order to simplify the Gibson assembly we began overlapping fragments, reducing the pieces that would need to join in a successful Gibson assembly. The initial two piece overlaps were successful, however overlaps beyond two pieces were problematic. Gibson assemblies in which the fragment number had been reduced by using two piece overlaps produced the same results as the single piece Gibson assemblies, either no growth, or growth of unidentified colonies. We decided to try an alternate PCR protocol (Shevchuk et al. 2004). Finally we were able to string together the five fragments of pMNBB-ICI.
• Conclusions
   Unfortunately we have not yet been able to assemble either vector in its entirety. The basal inducible vector, pMNBB-ICI, has been visualized in its linear state on a gel. We will continue to work on assembling both vectors so that they may be available for future teams to improve upon and characterize. If the pMNBB system functions as expected, it will certainly be a very useful tool not only for iGEM teams, but for academic research, and industrial processes.
Click here to return to Projects page.
iGEM Home | Team Minnesota Info | University of Minnesota Home | Contact Us!
Retrieved from " C:\Documents and Settings\aj\Desktop\IGEM webpage/Modelling.html "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
