Team:Minnesota/Software/PyMOL360
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" " http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd ">
Team:Minnesota/Project/Insulin - 2015.igem.org
"
Page
Discussion
View source
History
teams
Log in
 
Team:Minnesota/Project/Insulin
From 2015.igem.org
Team:Minnesota - Main Style Template
Team:Minnesota - Template
HOME
TEAM
PROJECT
NOTEBOOK
ATTRIBUTIONS
SAFETY
SPONSORS
Attributions
Insulin: Basic Background
• What is the role of Insulin in the Human body?
      Insulin is a hormone produced by the pancreas that removes glucose from the blood. In healthy individuals, excess glucose is readily removed from the blood stream by a proportional production of insulin. In persons with diabetes mellitus however; the body is either resistant to insulin, or it has a reduced capacity to produce insulin. Those individuals require an external source of insulin.
• Why are we expressing human Insulin?
      The ability to produce recombinant human Insulin cheaply has long been a lucrative goal. There are millions of people worldwide who are dependent on Insulin derived from production methods that make the product expensive -and further yet- potentially dangerous.Our team thinks that the current production methods for human Insulin are inefficient and can be optimized by being expressed in Pichia pastoris.
• What is the current method for Human Insulin production?
      Saccharomyces cerevisiae and Escherichia coli have historically been the preferred host cells to produce recombinant human insulin; however, each one of these organisms has great disadvantages.  E. coli lack secretory mechanisms, thus the cells must be lysed and processed to isolate the insulin. Another concern with E.coli is the lactose operon acting as an inducible system to control gene expression. S. cerevisiae, on the other hand, have the necessary secretory machinery to secrete insulin, but they produce it at very low yields compared to E. coli.
• Why are we using Pichia pastoris to express Human Insulin?
      P. pastoris is a eukaryotic organism in the yeast family that overcomes both obstacles encountered by the aforementioned organisms.  P. pastoris has been shown to excrete proteins at a rate of approximately five times that of S. cerevisiae with higher qualitative value as well. Recombinant human insulin has been produced in P. pastoris, but the harvesting and processing procedures are long, complex, and would be greatly simplified through the application of the BioBrick system. P. pastoris is already utilized by the Chinese for various industrial tasks, lending more promise to it as our choice, as industrial application would require very little change to our system.
• What are some risks associated with this design?
      Our system poses a hopeful future for diabetes sufferers, but, as with any medical breakthrough there are some risks to be aware of.  Although the risk of allergy or rejection will be reduced, there may still be adverse side effects that we cannot foresee despite careful testing in the laboratory, until the synthetically secreted protein is actually trial tested in humans.  Although great caution can be taken to solidify the safety of our synthetic system, unpredictable mutations may still occur that, again, cannot be foreseen until human trials are conducted.  The proposal of human trials alone poses a dilemma for ethical reasons.
PCSK1
• What is PCSK1?
      PCSK1 encodes for preprotein convertase type I, which is regarded as the most important enzyme in the first step of insulin processing in humans. Since our model organism P. pastoris lacks this enzyme it is hypothesized that its addition will increase efficiency of Insulin over the lineage.
Methods
   Insulin and PCSK1 Open Reading Frame Design Sequences for insulin and its chaperone PCSK1 were obtained through NCBI. Both insulin and PCSK1 were codon optimized for expression in P. pastoris using the online Codon Optimization tool from IDT. The BioBrick consensus sequence was added to both insulin and PCSK1 in preparation for cloning into the shipping vector. Restriction sites that were incompatible with the BioBrick standard, as well as our Pichia pastoris expression system pMNBB were eliminated by altering single nucleotides while maintaining the proper amino acid sequence. Both genes were synthesized by IDT. Insulin was obtained as one 333 bp fragment, while the longer PCSK1 was split into three, roughly 750 bp fragments.
   Fragments for insulin and PCSK1 were amplified by polymerase chain reaction (PCR). Insulin was cloned into the shipping vector. The three PCSK1 fragments were joined into the the full 2300 bp product using overlap extension PCR. The full PCSK1 product was cloned into the shipping vector. Cloning primers were not designed for PCSK1 with flanking BioBrick consensus sequences. PCSK1 was cloned into the pCR®Blunt II-TOPO® vector, provided by Invitrogen.
E. coli C2566 cultures were transformed with PCSK1- pCR®Blunt II-TOPO®. PCSK1- pCR®Blunt II-TOPO® was isolated  from transformation cultures and used as a template for PCR amplification of PCSK1 with our insulin primers that contained the BioBrick consensus sequence.
Two transformations were performed using Escherichia coli C2566, one with pSB1C3-Insulin and another with pSB1C3-PCSK1. Both constructs were submitted for sequencing.
Results
The products that were cloned into the shipping vector were verified by gel electrophoresis (Figure 1).
Several colonies were identified in colony screens for PCSK1 and insulin in transformants containing the shipping vector. Seen below are verification gels for PCSK1 and insulin.
Figure 1. PCR colony screen of PCSK1-pSB1C3 transformants. Note the banding in each lane at roughly 750 bp, consistent with the second fragment of PCSK1.
Figure 2. PCR colony screen of Ins-pSB1C3 transformants. Banding can be seen at roughly 333 bp, consistent with the full insulin ORF.  Sequencing data received for both PCSK1 and insulin was inconclusive.
Parts List
BBa_K1187001   Human insulin, codon optimized for expression in P. pastoris
Click here to return to Projects page.
iGEM Home | Team Minnesota Info | University of Minnesota Home | Contact Us!
Retrieved from " C:\Documents and Settings\aj\Desktop\IGEM webpage/Policy And Practices.html "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
