<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Minnesota_Parts skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Minnesota/Parts</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="bannerContainer"><H2> Add a banner to your wiki! </H2><P>You can make the image 980px  by  200px</P><P> Remember to call the file: &quot;<I>Team_Minnesota_banner.jpg</I>&quot; </P></DIV><DIV id="menuContainer"><UL><LI><A href="https://2015.igem.org/Team:Minnesota">HOME</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Team">TEAM</A></LI><LI><A href="#">PROJECT
            					</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Experiments">Experiments &amp; Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Design">Design</A></LI></UL><LI><A href="#">PARTS
            					</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Basic_Part">Basic Parts</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Composite_Part">Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Part_Collection">Part Collection</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Safety">SAFETY</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Modeling">MODELING</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Measurement">MEASUREMENT</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Software">SOFTWARE</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Entrepreneurship">ENTREPRENEURSHIP</A></LI></DIV><DIV id="contentContainer"><H2> Part Documentation</H2><P>Each team will make new parts during iGEM and will submit them to the Registry of Standard Biological Parts. The iGEM software provides an easy way to present the parts your team has created. The <CODE>&lt;groupparts&gt;</CODE> tag (see below) will generate a table with all of the parts that your team adds to your team sandbox.</P><P>Remember that the goal of proper part documentation is to describe and define a part, so that it can be used without needing to refer to the primary literature. Registry users in future years should be able to read your documentation and be able to use the part successfully. Also, you should provide proper references to acknowledge previous authors and to provide for users who wish to know more.</P><DIV class="highlightBox"><H4>Note</H4><P>Note that parts must be documented on the <A href="http://parts.igem.org/Main_Page"> Registry</A>. This page serves to <I>showcase</I> the parts you have made. Future teams and other users and are much more likely to find parts by looking in the Registry than by looking at your team wiki.</P></DIV><P>CrtE</P><DIV id="AutoAnnotator_container_1441068999102"><TABLE id="AutoAnnotator"><TBODY><TR><TH id="AutoAnnotatorHeader" colspan="2">Protein data table for BioBrick <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_&lt;!------------------------Enter BioBrick number here------------------------&gt;">BBa_</A> automatically created by the <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator">BioBrick-AutoAnnotator</A> version 1.0</TH></TR><TR><TD class="AutoAnnotator1col" colspan="2"><STRONG>Nucleotide sequence</STRONG> in <STRONG>RFC 10</STRONG>: (underlined part encodes the protein)<SPAN class="AutoAnnotatorSequence"> <U>ATGGTCACA ... AGGGATCGT</U>TAA</SPAN> <STRONG>ORF</STRONG> from nucleotide position 1 to 900 (excluding stop-codon)</TD></TR><TR><TD class="AutoAnnotator1col" colspan="2"><STRONG>Amino acid sequence:</STRONG> (RFC 25 scars in shown in bold, other sequence features underlined; both given below)</TD></TR><TR><TD class="AutoAnnotatorSeqNum">1 101 201 301 </TD><TD class="AutoAnnotatorSeqSeq">MVTAVRTEFDFAAYLRAARESVELALDGSLGTERPTSLSEAMRYSLLAGGKRLRPILCLAACELAGGNVAAAMPTAVALEMIHTMSLIHDDLPAMDNDDLRRGRPTNHKVYGEATAILAGDALLTRAFEIVALRSPGVSPECILKVIGELSLAAGAPGLVGGQMVDLESEGKEVNLDTLEYIHLHKTGALLSACVICGGLIGDASDELLQALRNYARGIGLAFQIIDDILDVTASSDVLGKTAGKDLVADKTTYPKLLGLKASRERAEQLVAEAKQGLDPWRDRAAPLLALADYITGRDR*</TD></TR></TBODY></TABLE><STRONG>Sequence features:</STRONG> (with their position in the amino acid sequence, see the <A href="https://2013.igem.org/Team:TU-Munich/Results/Software/FeatureList">list of supported features</A>)<TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorSeqFeat2b">None of the supported features appeared in the sequence</TD></TR></TBODY></TABLE><STRONG>Amino acid composition:</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Ala (A)</TD><TD class="AutoAnnotatorInnerAmino">43 (14.3%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Arg (R)</TD><TD class="AutoAnnotatorInnerAmino">20 (6.7%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Asn (N)</TD><TD class="AutoAnnotatorInnerAmino">5 (1.7%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Asp (D)</TD><TD class="AutoAnnotatorInnerAmino">22 (7.3%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Cys (C)</TD><TD class="AutoAnnotatorInnerAmino">5 (1.7%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Gln (Q)</TD><TD class="AutoAnnotatorInnerAmino">5 (1.7%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Glu (E)</TD><TD class="AutoAnnotatorInnerAmino">19 (6.3%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Gly (G)</TD><TD class="AutoAnnotatorInnerAmino">27 (9.0%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">His (H)</TD><TD class="AutoAnnotatorInnerAmino">5 (1.7%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Ile (I)</TD><TD class="AutoAnnotatorInnerAmino">15 (5.0%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Leu (L)</TD><TD class="AutoAnnotatorInnerAmino">45 (15.0%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Lys (K)</TD><TD class="AutoAnnotatorInnerAmino">11 (3.7%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Met (M)</TD><TD class="AutoAnnotatorInnerAmino">7 (2.3%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Phe (F)</TD><TD class="AutoAnnotatorInnerAmino">4 (1.3%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Pro (P)</TD><TD class="AutoAnnotatorInnerAmino">11 (3.7%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Ser (S)</TD><TD class="AutoAnnotatorInnerAmino">15 (5.0%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Thr (T)</TD><TD class="AutoAnnotatorInnerAmino">16 (5.3%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Trp (W)</TD><TD class="AutoAnnotatorInnerAmino">1 (0.3%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Tyr (Y)</TD><TD class="AutoAnnotatorInnerAmino">7 (2.3%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Val (V)</TD><TD class="AutoAnnotatorInnerAmino">17 (5.7%)</TD></TR></TBODY></TABLE><STRONG>Amino acid counting</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorAminoCountingInner2">Total number:</TD><TD class="AutoAnnotatorAminoCountingInner3">300</TD></TR><TR><TD class="AutoAnnotatorAminoCountingInner2">Positively charged (Arg+Lys):</TD><TD class="AutoAnnotatorAminoCountingInner3">31 (10.3%)</TD></TR><TR><TD class="AutoAnnotatorAminoCountingInner2">Negatively charged (Asp+Glu):</TD><TD class="AutoAnnotatorAminoCountingInner3">41 (13.7%)</TD></TR><TR><TD class="AutoAnnotatorAminoCountingInner2">Aromatic (Phe+His+Try+Tyr):</TD><TD class="AutoAnnotatorAminoCountingInner3">17 (5.7%)</TD></TR></TBODY></TABLE><STRONG>Biochemical parameters</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorBiochemParInner2">Atomic composition:</TD><TD class="AutoAnnotatorBiochemParInner3">C<SUB>1396</SUB>H<SUB>2292</SUB>N<SUB>392</SUB>O<SUB>431</SUB>S<SUB>12</SUB></TD></TR><TR><TD class="AutoAnnotatorBiochemParInner2">Molecular mass [Da]:</TD><TD class="AutoAnnotatorBiochemParInner3">31848.6</TD></TR><TR><TD class="AutoAnnotatorBiochemParInner2">Theoretical pI:</TD><TD class="AutoAnnotatorBiochemParInner3">5.08</TD></TR><TR><TD class="AutoAnnotatorBiochemParInner2">Extinction coefficient at 280 nm [M<SUP>-1</SUP> cm<SUP>-1</SUP>]:</TD><TD class="AutoAnnotatorBiochemParInner3">15930 / 16243 (all Cys red/ox)</TD></TR></TBODY></TABLE><STRONG>Plot for hydrophobicity, charge, predicted secondary structure, solvent accessability, transmembrane helices and disulfid bridges</STRONG> <STRONG>Codon usage</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorCodonUsage2">Organism:</TD><TD class="AutoAnnotatorCodonUsage3"><I>E. coli</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>B. subtilis</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>S. cerevisiae</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>A. thaliana</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>P. patens</I></TD><TD class="AutoAnnotatorCodonUsage3">Mammals</TD></TR><TR><TD class="AutoAnnotatorCodonUsage2">Codon quality (<A href="http://en.wikipedia.org/wiki/Codon_Adaptation_Index">CAI</A>):</TD><TD class="AutoAnnotatorCodonUsage3">good (0.70)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.76)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.74)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.74)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.76)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.62)</TD></TR></TBODY></TABLE><STRONG>Alignments</STRONG> (obtained from <A href="http://predictprotein.org">PredictProtein.org</A>)   There were no alignments for this protein in the data base. The BLAST search was initialized and should be ready in a few hours.<STRONG>Predictions</STRONG> (obtained from <A href="http://predictprotein.org">PredictProtein.org</A>)   There were no predictions for this protein in the data base. The prediction was initialized and should be ready in a few hours. The BioBrick-AutoAnnotator was created by <A href="https://2013.igem.org/Team:TU-Munich">TU-Munich 2013</A> iGEM team. For more information please see the <A href="https://2013.igem.org/Team:TU-Munich/Results/Software">documentation</A>.If you have any questions, comments or suggestions, please leave us a <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator">comment</A>.</DIV><P>CrtB</P><DIV id="AutoAnnotator_container_1441069277246"><TABLE id="AutoAnnotator"><TBODY><TR><TH id="AutoAnnotatorHeader" colspan="2">Protein data table for BioBrick <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_&lt;!------------------------Enter BioBrick number here------------------------&gt;">BBa_</A> automatically created by the <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator">BioBrick-AutoAnnotator</A> version 1.0</TH></TR><TR><TD class="AutoAnnotator1col" colspan="2"><STRONG>Nucleotide sequence</STRONG> in <STRONG>RFC 10</STRONG>: (underlined part encodes the protein)<SPAN class="AutoAnnotatorSequence"> <U>ATGCTGAAC ... GGTTCCGAC</U>TGA</SPAN> <STRONG>ORF</STRONG> from nucleotide position 1 to 1017 (excluding stop-codon)</TD></TR><TR><TD class="AutoAnnotator1col" colspan="2"><STRONG>Amino acid sequence:</STRONG> (RFC 25 scars in shown in bold, other sequence features underlined; both given below)</TD></TR><TR><TD class="AutoAnnotatorSeqNum">1 101 201 301 </TD><TD class="AutoAnnotatorSeqSeq">MLNESQVEAGKEIQQRTGKTFYFATRLLPERVRHATYVLYAFFRVADEVVDAEETAPPEQQRERLERLRAEALGDVETGDPVLSAFAELREQYDIDPDDVNVFIDAMESDIHKDRHATYEELETYMDGSASAVGRMMTAVMRPDDPKQALPHATALGQAFQLSNFLRDVGEDVVERDRVYLPETTLERHGVDVAQILDLEFDDRVAEVMRDELRRAESLYRDGVAGIEYLPHDCQFPVLLSAVLYADHHRSIRKLGYDTLSTTPDLSFGRKVFLLGKTAAYWAVWRDPVAVFKAVSVVPYPEDEEFDADFGPGSTPVGKVRSRGRVSGWIRALSRWGSD*</TD></TR></TBODY></TABLE><STRONG>Sequence features:</STRONG> (with their position in the amino acid sequence, see the <A href="https://2013.igem.org/Team:TU-Munich/Results/Software/FeatureList">list of supported features</A>)<TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorSeqFeat2b">None of the supported features appeared in the sequence</TD></TR></TBODY></TABLE><STRONG>Amino acid composition:</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Ala (A)</TD><TD class="AutoAnnotatorInnerAmino">33 (9.7%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Arg (R)</TD><TD class="AutoAnnotatorInnerAmino">31 (9.1%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Asn (N)</TD><TD class="AutoAnnotatorInnerAmino">3 (0.9%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Asp (D)</TD><TD class="AutoAnnotatorInnerAmino">32 (9.4%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Cys (C)</TD><TD class="AutoAnnotatorInnerAmino">1 (0.3%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Gln (Q)</TD><TD class="AutoAnnotatorInnerAmino">11 (3.2%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Glu (E)</TD><TD class="AutoAnnotatorInnerAmino">30 (8.8%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Gly (G)</TD><TD class="AutoAnnotatorInnerAmino">20 (5.9%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">His (H)</TD><TD class="AutoAnnotatorInnerAmino">8 (2.4%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Ile (I)</TD><TD class="AutoAnnotatorInnerAmino">8 (2.4%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Leu (L)</TD><TD class="AutoAnnotatorInnerAmino">30 (8.8%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Lys (K)</TD><TD class="AutoAnnotatorInnerAmino">9 (2.7%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Met (M)</TD><TD class="AutoAnnotatorInnerAmino">7 (2.1%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Phe (F)</TD><TD class="AutoAnnotatorInnerAmino">15 (4.4%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Pro (P)</TD><TD class="AutoAnnotatorInnerAmino">17 (5.0%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Ser (S)</TD><TD class="AutoAnnotatorInnerAmino">17 (5.0%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Thr (T)</TD><TD class="AutoAnnotatorInnerAmino">17 (5.0%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Trp (W)</TD><TD class="AutoAnnotatorInnerAmino">4 (1.2%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Tyr (Y)</TD><TD class="AutoAnnotatorInnerAmino">13 (3.8%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Val (V)</TD><TD class="AutoAnnotatorInnerAmino">33 (9.7%)</TD></TR></TBODY></TABLE><STRONG>Amino acid counting</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorAminoCountingInner2">Total number:</TD><TD class="AutoAnnotatorAminoCountingInner3">339</TD></TR><TR><TD class="AutoAnnotatorAminoCountingInner2">Positively charged (Arg+Lys):</TD><TD class="AutoAnnotatorAminoCountingInner3">40 (11.8%)</TD></TR><TR><TD class="AutoAnnotatorAminoCountingInner2">Negatively charged (Asp+Glu):</TD><TD class="AutoAnnotatorAminoCountingInner3">62 (18.3%)</TD></TR><TR><TD class="AutoAnnotatorAminoCountingInner2">Aromatic (Phe+His+Try+Tyr):</TD><TD class="AutoAnnotatorAminoCountingInner3">40 (11.8%)</TD></TR></TBODY></TABLE><STRONG>Biochemical parameters</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorBiochemParInner2">Atomic composition:</TD><TD class="AutoAnnotatorBiochemParInner3">C<SUB>1703</SUB>H<SUB>2637</SUB>N<SUB>475</SUB>O<SUB>525</SUB>S<SUB>8</SUB></TD></TR><TR><TD class="AutoAnnotatorBiochemParInner2">Molecular mass [Da]:</TD><TD class="AutoAnnotatorBiochemParInner3">38422.0</TD></TR><TR><TD class="AutoAnnotatorBiochemParInner2">Theoretical pI:</TD><TD class="AutoAnnotatorBiochemParInner3">4.79</TD></TR><TR><TD class="AutoAnnotatorBiochemParInner2">Extinction coefficient at 280 nm [M<SUP>-1</SUP> cm<SUP>-1</SUP>]:</TD><TD class="AutoAnnotatorBiochemParInner3">41370 / 41433 (all Cys red/ox)</TD></TR></TBODY></TABLE><STRONG>Plot for hydrophobicity, charge, predicted secondary structure, solvent accessability, transmembrane helices and disulfid bridges</STRONG> <STRONG>Codon usage</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorCodonUsage2">Organism:</TD><TD class="AutoAnnotatorCodonUsage3"><I>E. coli</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>B. subtilis</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>S. cerevisiae</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>A. thaliana</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>P. patens</I></TD><TD class="AutoAnnotatorCodonUsage3">Mammals</TD></TR><TR><TD class="AutoAnnotatorCodonUsage2">Codon quality (<A href="http://en.wikipedia.org/wiki/Codon_Adaptation_Index">CAI</A>):</TD><TD class="AutoAnnotatorCodonUsage3">good (0.70)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.76)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.73)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.74)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.75)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.66)</TD></TR></TBODY></TABLE><STRONG>Alignments</STRONG> (obtained from <A href="http://predictprotein.org">PredictProtein.org</A>)   There were no alignments for this protein in the data base. The BLAST search was initialized and should be ready in a few hours.<STRONG>Predictions</STRONG> (obtained from <A href="http://predictprotein.org">PredictProtein.org</A>)<STRONG>Subcellular Localization</STRONG> (reliability in brackets)<TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorLocalization2">Archaea:</TD><TD class="AutoAnnotatorLocalization3">arc (%)</TD></TR><TR><TD class="AutoAnnotatorLocalization2">Bacteria:</TD><TD class="AutoAnnotatorLocalization3">bac (%)</TD></TR><TR><TD class="AutoAnnotatorLocalization2">Eukarya:</TD><TD class="AutoAnnotatorLocalization3">euk (%)</TD></TR></TBODY></TABLE><STRONG>Gene Ontology</STRONG> (reliability in brackets)<TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorGO2">Molecular Function Ontology:</TD><TD class="AutoAnnotatorGO3"><A href="http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=0005515 0.">0005515 0.</A> (NaN%), <A href="http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=0004310 0.">0004310 0.</A> (NaN%)</TD></TR><TR><TD class="AutoAnnotatorGO2">Biological Process Ontology:</TD><TD class="AutoAnnotatorGO3"><A href="http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=0016117 0.">0016117 0.</A> (NaN%), <A href="http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=0015995 0.">0015995 0.</A> (NaN%)</TD></TR><TR><TD class="AutoAnnotatorGO3"> </TD></TR></TBODY></TABLE><STRONG>Predicted features:</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorPredFeat2a">Disulfid bridges:</TD><TD class="AutoAnnotatorPredFeat3"> - </TD></TR><TR><TD class="AutoAnnotatorPredFeat2a">Transmembrane helices:</TD><TD class="AutoAnnotatorPredFeat3"> - </TD></TR></TBODY></TABLE> The BioBrick-AutoAnnotator was created by <A href="https://2013.igem.org/Team:TU-Munich">TU-Munich 2013</A> iGEM team. For more information please see the <A href="https://2013.igem.org/Team:TU-Munich/Results/Software">documentation</A>.If you have any questions, comments or suggestions, please leave us a <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator">comment</A>.</DIV><P>CrtI</P><DIV id="AutoAnnotator_container_1441069421964"><TABLE id="AutoAnnotator"><TBODY><TR><TH id="AutoAnnotatorHeader" colspan="2">Protein data table for BioBrick <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_&lt;!------------------------Enter BioBrick number here------------------------&gt;">BBa_</A> automatically created by the <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator">BioBrick-AutoAnnotator</A> version 1.0</TH></TR><TR><TD class="AutoAnnotator1col" colspan="2"><STRONG>Nucleotide sequence</STRONG> in <STRONG>RFC 10</STRONG>: (underlined part encodes the protein)<SPAN class="AutoAnnotatorSequence"> <U>ATGGATGAC ... TTGTCCACT</U>TAA</SPAN> <STRONG>ORF</STRONG> from nucleotide position 1 to 1278 (excluding stop-codon)</TD></TR><TR><TD class="AutoAnnotator1col" colspan="2"><STRONG>Amino acid sequence:</STRONG> (RFC 25 scars in shown in bold, other sequence features underlined; both given below)</TD></TR><TR><TD class="AutoAnnotatorSeqNum">1 101 201 301 401 </TD><TD class="AutoAnnotatorSeqSeq">MDDSHIVVVGAGLAGLVAARHLAVDGADVTVIERRDDVGGRVRTRTKDGFTLDRGFQVLLTGYPAVRRELDFDALDLRPLAPGATICRPSRRSVLSDPLRDINGLFDSLRNPEVTTSDKLRTLALRQDVSTRDEGEIFDSPDQSIRAYLRDWGFSEDFIEHFVAPFYGGITLDRSLSTSKRVFEYTFKMLATGRAAVPADGMQAIPEQLAATARDEGATIRLGERVESVQSDADGAVVATDDESVEADAVVVATDPKEARRLTGVGSIPTDARGCVTQYYTLPTGSGLDAGKRIMLNAPNPDPNTVVPLSTVAPEYAPPGKELLNATFLGAAAQDESEEELFEKTRRTLEAWYPERYFDDLELLHTDYISFAQFAQPPEIHDSLPDHRDAPGRTYLAGDYTAWSSIQGAMRSGKEAADAIRDDLST*</TD></TR></TBODY></TABLE><STRONG>Sequence features:</STRONG> (with their position in the amino acid sequence, see the <A href="https://2013.igem.org/Team:TU-Munich/Results/Software/FeatureList">list of supported features</A>)<TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorSeqFeat2b">None of the supported features appeared in the sequence</TD></TR></TBODY></TABLE><STRONG>Amino acid composition:</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Ala (A)</TD><TD class="AutoAnnotatorInnerAmino">48 (11.3%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Arg (R)</TD><TD class="AutoAnnotatorInnerAmino">37 (8.7%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Asn (N)</TD><TD class="AutoAnnotatorInnerAmino">6 (1.4%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Asp (D)</TD><TD class="AutoAnnotatorInnerAmino">44 (10.3%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Cys (C)</TD><TD class="AutoAnnotatorInnerAmino">2 (0.5%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Gln (Q)</TD><TD class="AutoAnnotatorInnerAmino">11 (2.6%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Glu (E)</TD><TD class="AutoAnnotatorInnerAmino">27 (6.3%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Gly (G)</TD><TD class="AutoAnnotatorInnerAmino">32 (7.5%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">His (H)</TD><TD class="AutoAnnotatorInnerAmino">6 (1.4%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Ile (I)</TD><TD class="AutoAnnotatorInnerAmino">16 (3.8%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Leu (L)</TD><TD class="AutoAnnotatorInnerAmino">39 (9.2%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Lys (K)</TD><TD class="AutoAnnotatorInnerAmino">9 (2.1%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Met (M)</TD><TD class="AutoAnnotatorInnerAmino">5 (1.2%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Phe (F)</TD><TD class="AutoAnnotatorInnerAmino">16 (3.8%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Pro (P)</TD><TD class="AutoAnnotatorInnerAmino">25 (5.9%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Ser (S)</TD><TD class="AutoAnnotatorInnerAmino">26 (6.1%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Thr (T)</TD><TD class="AutoAnnotatorInnerAmino">32 (7.5%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Trp (W)</TD><TD class="AutoAnnotatorInnerAmino">3 (0.7%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Tyr (Y)</TD><TD class="AutoAnnotatorInnerAmino">12 (2.8%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Val (V)</TD><TD class="AutoAnnotatorInnerAmino">30 (7.0%)</TD></TR></TBODY></TABLE><STRONG>Amino acid counting</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorAminoCountingInner2">Total number:</TD><TD class="AutoAnnotatorAminoCountingInner3">426</TD></TR><TR><TD class="AutoAnnotatorAminoCountingInner2">Positively charged (Arg+Lys):</TD><TD class="AutoAnnotatorAminoCountingInner3">46 (10.8%)</TD></TR><TR><TD class="AutoAnnotatorAminoCountingInner2">Negatively charged (Asp+Glu):</TD><TD class="AutoAnnotatorAminoCountingInner3">71 (16.7%)</TD></TR><TR><TD class="AutoAnnotatorAminoCountingInner2">Aromatic (Phe+His+Try+Tyr):</TD><TD class="AutoAnnotatorAminoCountingInner3">37 (8.7%)</TD></TR></TBODY></TABLE><STRONG>Biochemical parameters</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorBiochemParInner2">Atomic composition:</TD><TD class="AutoAnnotatorBiochemParInner3">C<SUB>2037</SUB>H<SUB>3206</SUB>N<SUB>578</SUB>O<SUB>656</SUB>S<SUB>7</SUB></TD></TR><TR><TD class="AutoAnnotatorBiochemParInner2">Molecular mass [Da]:</TD><TD class="AutoAnnotatorBiochemParInner3">46513.8</TD></TR><TR><TD class="AutoAnnotatorBiochemParInner2">Theoretical pI:</TD><TD class="AutoAnnotatorBiochemParInner3">4.65</TD></TR><TR><TD class="AutoAnnotatorBiochemParInner2">Extinction coefficient at 280 nm [M<SUP>-1</SUP> cm<SUP>-1</SUP>]:</TD><TD class="AutoAnnotatorBiochemParInner3">34380 / 34505 (all Cys red/ox)</TD></TR></TBODY></TABLE><STRONG>Plot for hydrophobicity, charge, predicted secondary structure, solvent accessability, transmembrane helices and disulfid bridges</STRONG> <STRONG>Codon usage</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorCodonUsage2">Organism:</TD><TD class="AutoAnnotatorCodonUsage3"><I>E. coli</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>B. subtilis</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>S. cerevisiae</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>A. thaliana</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>P. patens</I></TD><TD class="AutoAnnotatorCodonUsage3">Mammals</TD></TR><TR><TD class="AutoAnnotatorCodonUsage2">Codon quality (<A href="http://en.wikipedia.org/wiki/Codon_Adaptation_Index">CAI</A>):</TD><TD class="AutoAnnotatorCodonUsage3">good (0.69)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.73)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.74)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.76)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.79)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.67)</TD></TR></TBODY></TABLE><STRONG>Alignments</STRONG> (obtained from <A href="http://predictprotein.org">PredictProtein.org</A>)   There were no alignments for this protein in the data base. The BLAST search was initialized and should be ready in a few hours.<STRONG>Predictions</STRONG> (obtained from <A href="http://predictprotein.org">PredictProtein.org</A>)<STRONG>Subcellular Localization</STRONG> (reliability in brackets)<TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorLocalization2">Archaea:</TD><TD class="AutoAnnotatorLocalization3">arc (%)</TD></TR><TR><TD class="AutoAnnotatorLocalization2">Bacteria:</TD><TD class="AutoAnnotatorLocalization3">bac (%)</TD></TR><TR><TD class="AutoAnnotatorLocalization2">Eukarya:</TD><TD class="AutoAnnotatorLocalization3">euk (%)</TD></TR></TBODY></TABLE><STRONG>Gene Ontology</STRONG> (reliability in brackets)<TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorGO2">Molecular Function Ontology:</TD><TD class="AutoAnnotatorGO3"><A href="http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=0050660 0.">0050660 0.</A> (NaN%), <A href="http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=0080103 0.">0080103 0.</A> (NaN%)</TD></TR><TR><TD class="AutoAnnotatorGO2">Biological Process Ontology:</TD><TD class="AutoAnnotatorGO3"><A href="http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=0015995 0.">0015995 0.</A> (NaN%), <A href="http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=0015979 0.">0015979 0.</A> (NaN%)</TD></TR><TR><TD class="AutoAnnotatorGO3"> </TD></TR></TBODY></TABLE><STRONG>Predicted features:</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorPredFeat2a">Disulfid bridges:</TD><TD class="AutoAnnotatorPredFeat3"> - </TD></TR><TR><TD class="AutoAnnotatorPredFeat2a">Transmembrane helices:</TD><TD class="AutoAnnotatorPredFeat3"> - </TD></TR></TBODY></TABLE> The BioBrick-AutoAnnotator was created by <A href="https://2013.igem.org/Team:TU-Munich">TU-Munich 2013</A> iGEM team. For more information please see the <A href="https://2013.igem.org/Team:TU-Munich/Results/Software">documentation</A>.If you have any questions, comments or suggestions, please leave us a <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator">comment</A>.</DIV><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Adding_parts_to_the_registry"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Adding parts to the registry</SPAN></A></LI><LI class="toclevel-1"><A href="#What_information_do_I_need_to_start_putting_my_parts_on_the_Registry.3F"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">What information do I need to start putting my parts on the Registry?</SPAN></A></LI><LI class="toclevel-1"><A href="#Inspiration"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Inspiration</SPAN></A></LI><LI class="toclevel-1"><A href="#Part_Table"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Part Table</SPAN></A></LI></UL></DIV><H4><SPAN class="mw-headline" id="Adding_parts_to_the_registry">Adding parts to the registry</SPAN></H4><P>You can add parts to the Registry at our &lt;a href=&quot;<A rel="nofollow" class="external free" href="http://parts.igem.org/Add_a_Part_to_the_Registry">http://parts.igem.org/Add_a_Part_to_the_Registry</A>&quot;&gt;Add a Part to the Registry&lt;/a&gt; link.</P><P>We encourage teams to start completing documentation for their parts on the Registry as soon as you have it available. The sooner you put up your parts, the better you will remember all the details about your parts. Remember, you don't need to send us the DNA sample before you create an entry for a part on the Registry. (However, you <B>do</B> need to send us the DNA sample before the Jamboree. If you don't send us a DNA sample of a part, that part will not be eligible for awards and medal criteria.)</P><H4><SPAN class="mw-headline" id="What_information_do_I_need_to_start_putting_my_parts_on_the_Registry.3F">What information do I need to start putting my parts on the Registry?</SPAN></H4><P>The information needed to initially create a part on the Registry is:</P><UL><LI>Part Name</LI><LI>Part type</LI><LI>Creator</LI><LI>Sequence</LI><LI>Short Description (60 characters on what the DNA does)</LI><LI>Long Description (Longer description of what the DNA does)</LI><LI>Design considerations</LI></UL><P>
We encourage you to put up <EM>much more</EM> information as you gather it over the summer. If you have images, plots, characterization data and other information, please also put it up on the part page. </P><H4><SPAN class="mw-headline" id="Inspiration">Inspiration</SPAN></H4><P>We have a created  a &lt;a href=&quot;<A rel="nofollow" class="external free" href="http://parts.igem.org/Well_Documented_Parts">http://parts.igem.org/Well_Documented_Parts</A>&quot;&gt;collection of well documented parts&lt;/a&gt; that can help you get started.</P><P> You can also take a look at how other teams have documented their parts in their wiki:</P><UL><LI>&lt;a href=&quot;<A rel="nofollow" class="external free" href="https://2014.igem.org/Team:MIT/Parts">https://2014.igem.org/Team:MIT/Parts</A>&quot;&gt; 2014 MIT &lt;/a&gt;</LI><LI>&lt;a href=&quot;<A rel="nofollow" class="external free" href="https://2014.igem.org/Team:Heidelberg/Parts">https://2014.igem.org/Team:Heidelberg/Parts</A>&quot;&gt; 2014 Heidelberg&lt;/a&gt;</LI><LI>&lt;a href=&quot;<A rel="nofollow" class="external free" href="https://2014.igem.org/Team:Tokyo_Tech/Parts">https://2014.igem.org/Team:Tokyo_Tech/Parts</A>&quot;&gt;2014 Tokyo Tech&lt;/a&gt;</LI></UL><H4><SPAN class="mw-headline" id="Part_Table">Part Table</SPAN></H4><P>&lt;/html&gt;&lt;groupparts&gt;iGEM015 Example&lt;/groupparts&gt;</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>