<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Minnesota_2A_Tags skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Minnesota/2A Tags</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:Minnesota/Project/Insulin - 2015.igem.org</TITLE></P><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Minnesota/Project/Insulin">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Minnesota/Project/Insulin&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Minnesota/Project/Insulin&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Minnesota/Project/Insulin&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Minnesota/Project/Insulin" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Minnesota/Project/Insulin</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2015.igem.org</H3><P><TITLE>Team:Minnesota - Main Style Template</TITLE><TITLE>Team:Minnesota - Template</TITLE></P><DIV id="ContentAll"><DIV id="ContentAll"><DIV id="SideBarAll"><DIV class="sideNavBar2"><UL><LI><A href="https://2015.igem.org/Team:Minnesota">HOME</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Team">TEAM</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Project">PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Software">SOFTWARE</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Practices">PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Safety">SAFETY</A></LI><LI><A href="https://2015.igem.org/Team:Minnesota/Sponsor">SPONSOR</A></LI></UL></DIV></DIV><DIV id="MainBoxAll"><DIV id="MainBoxContent" style="background-color:white;"><DIV style="position:absolute;top:0px;margin-left:50px; width:600px; height:450px; overflow-y:scroll; text-align:left; overflow-x:hidden;"><H1>Polycistronic Eukaryotic Gene Expression</H1>

Over the last several decades, the metabolic engineering of microbes to produce medicines, biofuels, food additives, and enzymes has grown to a multi-billion dollar industry. The use of engineered biosynthetic pathways has many advantages over more conventional organic methods including cost efficiency, relative environmental friendliness, chiral specificity, and the ability to produce products that would otherwise be difficult to synthesize with conventional organic chemistry techniques (6). While much of current metabolic engineering is being done in bacteria such as Escherichia coli due to its high growth rate and ease to work with, eukaryotic systems, such as <I>Saccharomyces cerevisiae</I> the industrially relevant yeast, <I>Pichia pastoris</I>, are also attractive  and have received much attention due to their ability to glycosylate proteins and form disulfide bonds (1) within proteins which may otherwise be misfolded and inactive if expressed in a prokaryotic system. However, expressing novel and complex metabolic pathways in a eukaryotic system such as <I>S. cerevisiae</I> has its own unique problems. Unlike in <I>E. coli</I>, where it is possible to organize a large number of coding sequences into a single operon with conventional methods, each gene in a eukaryotic system requires its own promoter, transcription factors, and ribosome binding site. Coordinating multiple unique promoters and the absence of sufficient antibiotic selective markers provides a significant challenge, and the use of several identical promoters poses the risk of interference, recombination, and suppression between the promoters (2, 4, 5, 8). To circumnavigate this problem, polycistronic sequences can be recreated in eukaryotes using viral 2A sequences (7).

<B><FONT size="4"><P align="center"> Viral 2A Tags: Basic Background </P></FONT></B><P><FONT size="3">• </FONT>    The 2A peptide sequence, also known as a cis­acting hydrolase element, is ~20 amino acids in length and can easily be placed between genes of interest. When translated, the sequence causes the ribosome to skip over a peptide bond and allow the translation of multiple discrete polypeptides from a single mRNA molecule, leaving an 18 amino acid sequence on the C-terminus of the upstream protein and a proline residue on the N-terminus of the downstream protein (5). 2A viral sequences are particularly promising due to their small size (~60-70 nucleotides) and high cleavage rate which has been found to form 1:1 molar ratios of gene product in biscistronic sequences (3).  These features are especially impressive when compared to internal ribosome entry sites (IRES), another popular method for creating polycistronic sequences in eukaryotes. IRES require large sequences (~500 nucleotides), which can be problematic when using size restricted vectors and can experience up to a 10-fold decrease in expression levels for downstream gene products (5). The 2A tag sequence can be used to achieve ribosomal “skip” at the transcript C ­terminus of the upstream gene, allowing a discrete protein to be produced from that sequence and polycistronic expression in a eukaryotic organism. However, despite the attractiveness of using 2A sequences to create large multi-enzyme polycistronic sequences, little work has been done beyond simple bicistronic insertions and preliminary investigations suggest that the gene order within larger polycistronic sequences can affect the overall efficiency of larger pathways such that genes further downstream of the translational start site have lower levels of translation in 2A polycistronic sequences, and that the differences in relative molar amounts in enzyme result in the changes of product production (5). Understanding how the order of genes in longer polycistronic sequences affects translation rates is important for optimizing engineered metabolic pathways and  limiting the buildup of potentially toxic intermediates.

<I><U><FONT size="3">• </FONT>Experimental Approach</U></I>    Our team developed a <A href="https://2015.igem.org/Team:Minnesota/Modeling">Mathematical Model </A> to estimate gene order for optimal biosynthetic production using 2A sequences and gladly present it as an open-source community tool to streamline future applications. We also attempted to demonstrate the utility of this technology in yeast by expressing genes to produce compounds in the beta carotenoid pathway. 

The genes of interest illustrated above <I>(crtEBI)</I> were synthesized via IDT and cloned into a <I> pESC-URA</I> plasmid as shown via homologous recombination in <I>S. cerevisiae</I>, and screened by induction of the galactose promoter. Since <I>S.cerevisiae</I> is capable of producing the precursor farnesyl diphosphate, successful transformants expressing all 3 genes would turn red in the presence of galactose due to the lycopene produced while the unsuccessful ones would remain white. 



We could then simultaneously inoculate <I>S. cerevisiae</I> in order to harvest the plasmid and for transformation into <I>E. coli </I>. Once a successful transformation had occurred, we could again harvest the plasmid, digest it for our insert, and ligate it into a linearized shipping vector <I>(pSB1C3)</I>. Testing is currently in progress.

The efficiency of the 2A sequences is also to be tested by inserting a yeast enhanced GFP gene between each of the crt genes. We would then grow the yeast, read the GFP fluorescence using a plate reader, and normalize the readings based on OD600. 
Ideally, we plan to test the model by reorganizing the genes and quantifying the amount of lycopene that was produced using high-purity liquid chromatography, and use the data to enforce model validity. 
<A href="https://2015.igem.org/Team:Minnesota/Practices">Click here to learn more about our software development, education, and public engagement work! </A><FONT size="1.5">References


(1)   Cregg, J. M.; Tolstorukov, I.; Kusari, A.; Sunga, J.; Madden, K.; Chappell, T., Expression in the yeast Pichia pastoris. Meth. Enzymol. 2009, 463:169-89

(2)   Cullen, B. R.; Lomedico, P. T.; Ju, G., Transcriptional interference in avian retroviruses - implications for the promoter insertion model of leukaemogenesis. Nature. 1984, 307(5948): 241-245

(3)   Donnelly, M. L. L.; Hughes, L. E.; Luke, G. A.; Mendoza, H.; ten Dam, E.; Gani, D.; Ryan, M. D., The “cleavage” activities of foot-and-mouth disease virus 2A site-directed mutants and naturally occuring “2A-like” sequences. Journal General Virology, 2001, 82: 1027-1041

(4)   Emmerman, M.; Temin, H. M., Comparison of promoter suppression in avian and murine retrovirus vectors. Nucleic Acids Res. 1986, 14(23): 9381-9396

(5)   Geier, M.; Fauland, P.; Vogl, T.; Glieder, A., Compact multi-enzyme pathways in P. pastoris. Chem. Commun. 2014, 51, 1643-
</FONT></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>