"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/Project/BsDetector/SamplePreparation
From 2014.igem.org
Project
B.s. Detector
Target Diseases
Sample Preparation
B.s. Chassis
Genetic Circuit
Modelling & Optimization
Device Prototype
Platform
Collaboration
Achievements
Final System
Our Team
Team Members
Official Profile
Outreach
Telus Spark
Beakernight
About the University
Contact Us
Attributions
Sponsors
Notebook
Journal
Detectives
Transformers
Modelling & Prototype
Policies & Practices
Protocol Manual
General
DNA
B. subtilis
Sample Preparation
Parts
References
Policies
Diagnostic Landscape
Implementation
Informed Design
Safety
Biological Weapons Convention
FIND
iGEM
Calgary 2013
Calgary 2012
Calgary 2011
Target Sequence Amplification
A major advantage of our diagnostic system is an internal sample preparation stage. All that is required by the end-user is inputting the whole blood sample into the device.
Two technical challenges were identified regarding sample preparation, and amplification of our target DNA sequence. First, we must be capable of amplifying a target DNA sequence in a resource poor environment in addition to a potentially hostile climate. Conventional PCR methods are not feasible due to the inhibitory cost of thermocyclers, the necessary refrigeration of PCR components such as enzymes and buffers, and the training required to carry out and analyze PCR experiments. Secondly, we must be capable of amplifying the target DNA sequence directly from whole blood, a complex environment containing inhibitors of Taq polymerase.
Isothermal PCR (isoPCR) serves as an alternative to conventional PCR. isoPCR refers to the many molecular strategies that allow for amplification of a DNA sequence at a constant, relatively low temperature. Although there are several approaches to isoPCR, all strategies employ a molecular mechanism to circumvent the need for thermal denaturation of double-stranded DNA. In our system we utilized Recombinase Polymerase Amplification (RPA). RPA utilizes a strand-displacing polymerase to synthesize a complimentary strand of DNA using a double-stranded substrate; the recombinase and a single-stranded DNA-binding protein allow for oligonucleotide primers to stably bind to the template DNA. Because RPA is not dependent on a thermocylcer, it is capable of cycling through the amplification process continuously until the exhaustion of resources such as ATP and primers. Continuous amplification in RPA results in an amplified signal that is comparative to conventional PCR in a fraction of the time. For our purposes, in developing the sample preparation stage of our system, we used the commercially-available RPA kit purchased from TwistDx.
To eliminate the need for refrigeration, and further improve the shelf-life of our system, the components of isoPCR are freeze-dried, allowing activation by rehydration immediately prior to use. This was demonstrated to be effective as the TwistDx isoPCR kit provided us with freeze-dried pellets and rehydration buffer (Fig. 1).
Figure 1: Freeze-dried pellets provided by the RPA Basic Kit from TwistDxisoPCR, reaction is activated upon addition of hydration buffer and magnesium-acetate also provided in the kit.
Whole-blood compatibility in our amplification protocol was achieved using a Taq polymerase tolerant of inhibitors typically found in whole blood. Our sponsor KAPA provided us with KAPA Blood PCR Kits containing an engineered, second-generation Taq polymerase. By supplementing the RPA mixture with the blood-compatible Taq we achieved a greater signal output of isoPCR in the presence of blood. This demonstrated the efficacy of our system to amplify a target DNA sequence in the presence of blood, without the use of a thermocycler (Fig. 2).
Figure 2: Results of isoPCR using a 132bp positive control sequence provided with the RPA Basic Kit from TwistDx. Amplification was performed with blood concentrations of 0%, 1%, 5% or 10% v/v sheeps blood treated with the anti-coagulant citrate. Amplification was performed with either RPA exclusively, or with a 25% v/v substitution using the KAPA Blood PCR Kit mix containing the blood- compatible Taq polymerase. At 10% v/v blood, RPA failed to produce the expected amplicon; the addition of blood-compatible Taq produced the expected 132bp amplicon in the presence of 10% v/v blood.
Isolation of genomic DNA was not necessary for successful amplification of the target DNA sequence. Typically, conventional PCR requires a prolonged initial denaturation step (e.g. 10 minutes at 95C) to lyse the bacterial cells and release DNA. However, our experiments demonstrate that isoPCR is effective on whole E. coli cells without prior lysis (Fig. 3).
Figure 3: Results of isoPCR under standard isothermal conditions using BioBrick primers. Two different sequences were targeted for amplification; C1-λ is a promoter sequence with an expected amplicon size of 406bp; RFP is a coding sequence with an expected amplicon size of 1100bp. Templates used in this experiment were either whole colonies labelled ‘C’, or purified plasmid extracts labelled ‘P’.
Target Sequence Specificity
To selectively detect the presence of N. meningititis, we identified a unique yet conserved sequence present in all known N. meningititis genomes. The selected sequence was approximately 400bp, the maximum length of DNA our isoPCR was capable of amplifying. The specificity of our chosen sequence was confirmed by conventional PCR; a ~2.8kb amplicon was produced from the genomic DNA of six different N. meningititis strains, no amplicons were produced from the genomic DNA of five other pathogens including a commensal strain of Neisseria (Fig. 4). These results were replicated using isoPCR, amplifying a shorter target sequence of ~450bp. Although other smaller amplicons are produced by isoPCR these are the result of ‘primer noise’ inherent to the isoPCR method of RPA and may be reduced in the future by optimizing conditions such as primer concentration and screening primer variants for reduced primer noise (Fig. 5).
Figure 4: Results of PCR on genomic DNA samples from eleven pathogenic parasites. All six N. meningititis strains produced the expected amplicon at ~2.8kb as indicated by white asterisks. The five other pathogens did not produce any products of the appropriate size.
Figure 5: Results of isoPCR using RPA on the eleven genomic DNA samples as described in Fig. 3. Bands below 500bp are indicative of primer noise inherent to the RPA system.
Retrieved from " http://2014.igem.org/Team:Calgary/Project/BsDetector/SamplePreparation "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
