"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/Project/BsDetector
From 2014.igem.org
\
Project
B.s. Detector
Target Diseases
Sample Preparation
B.s. Chassis
Genetic Circuit
Modelling & Optimization
Device Prototype
Platform
Collaboration
Achievements
Final System
Our Team
Team Members
Official Profile
Outreach
Telus Spark
Beakernight
About the University
Contact Us
Attributions
Sponsors
Notebook
Journal
Detectives
Transformers
Modelling & Prototype
Policies & Practices
Protocol Manual
General
DNA
B. subtilis
Sample Preparation
Parts
References
Policies
Diagnostic Landscape
Implementation
Informed Design
Safety
Biological Weapons Convention
FIND
iGEM
Calgary 2013
Calgary 2012
Calgary 2011
B.s. Detector
Using Bacillus Subtilis to diagnose disease
The B.s. detector is a diagnostic tool designed to simultaneously identify the presence of several pathogens within a blood sample through the detection of specific sequences within genomic DNA. At the core of our device lies the gram positive and eopnymous bacteria B. subtilis, which has been genetically modified to harbour two specific operons (a cluster of genes intended to be activated together) of our own design working in tandem. The first operon (i.e., the reporter operon) consists of the lacZ gene along with an operator gene which is necessary for the function of lacZ. lacZ, when operational, is designed to produce an indigo pigment in the presence of the organic compound X-gal. lacZ accomplishes this by cleaving X-gal with its corresponding protein (enzyme), β-galactosidase. The second operon (i.e., the repressor operon) consists of a repressor gene which encodes for repressor proteins designed to inhibit the function of the first operon and consequently prevent the indigo pigment from being synthesized. The repressor gene is surrounded by two flanking sequences on the 5' and 3' ends which will play a crucial role in the diagnostic mechanism of our device.
The flanking regions are designed to share a high level of homology with target sequences found within the genome of our pathogens of interest. These sequences will be identified beforehand using public genome databases. Upon contact with a pathogen, the flanking regions within our repressor operon will recognize the target sequence and undergo homologous recombination , a process by which sequences of a certain size are exchanged between two similar (homologous) DNA molecules. Homologous recombination results in the flanking regions - and the repressor gene sandwiched in between - "switching places" with the target sequence. With the repressor gene removed, the repressor operon will no longer be able to inhibit the lacZ gene within the reporter operon and indigo pigment will subsequently be produced. The presence of pathogens will be indicated by an easily identifiable colorimetric output.
Figure 1: B. s Detector Schematic. Our genetically engineered B. subtilis will by default start in a negative state and lack indigo pigment due to the repression of the reporter operon by the repressor gene. However, upon contact with a target sequence, the repressor gene will be removed from the repressor operon through homologous recombination and the reporter operon will be free to produce the pigment. This simple negative (no colour) to positive (indigo colour) transition is the basis of presenting the diagnostic results.
The genetically engineered B. subtilis and its two interacting reporter and repressor operons constitute the in vivo pathogen detection component of our device. The physical device will consist of a single, self-contained unit made from a combination of lightweight and durable polymers. The blood or fluid sample given by the patient will be placed into a small outlet located in the body of the device and flow into the main chamber through gravitational force. However, preparation of the sample must be conducted prior to the diagnostic assay, as the B. subtilis will be unable to identify the target sequence if it remains integrated in the pathogen's genome. First, the target sequence will be amplified using a recombinase-based amplification method in order to create multiple DNA replicates and isolate the sequence from the rest of the genome. This will enhance the sensitivity of the device as the B. subtilis will have a much easier time recognizing and absorbing the target sequence if a large number of replicates are present within the sample. Isothermal PCR is a crucial requirement for our device because our objective is to implement it in regions where resources such as electricity and a thermocycler may not always be available. Amplification will be conducted using a recombinase enzyme and a unique Taq DNA Polyermase capable of functioning within human blood (“Blood-Taq"). The presence of Blood-Taq” is essential to our isothermal amplification method because human blood is heavily concentrated with various enzyme inhibitors which interfere with PCR. The entire sample preparation process will take approximately one hour and occur at a constant temperature between 30°C to 42°C degrees. 37°C is the optimum temperature and will yield the greatest concentration of DNA replicates, but isothermal amplification can function at an adequate level anywhere within the aforementioned temperature range. Upon completion, the processed sample will be forced into the several adjacent chambers by pressing a plunger located at the top of the device.
The sample will divide and enter each secondary chamber through small tubes lined with a inner Teflon coating. The Teflon coating ensures that no volume of sample adheres to the inside of the tube during transport. Mathematical models and fluid flow analysis were conducted to align the tubes in such a way that each chamber receives an equal volume of sample at an identical rate and pressure. Each chamber will contain a sample of our genetically modified B. subtilis and will be representative of a single pathogen. For example, if we wish to simultaneously test for malaria, typhoid fever, and meningitis, our device would have three secondary chambers corresponding to the three separate diseases. Depending on which pathogen is to be identified, the repressor gene will consists of different flanking regions specific to the pathogen's target sequence. Once the blood sample comes into contact with the B. subtilis contained within the chambers, a colourimetric output will be acquired through the homologous recombination mechanism mentioned previously. We decided on colourimetric output as the basis for diagnostic results due to its easily understood binary format. In keeping with the theme of affordability and convenience, our device was designed to be utilized by a broad spectrum of end-users, including the untrained layman if necessary.
Figure 2: A 2D sketch of our device.
Three secondary chambers are attached to the central chamber or body.
A major feature of our diagnostic tool is its ability to detect a wide variety of pathogens simply by modifying the flanking regions surrounding the repressor gene. As for the target sequences, our experiments have shown that a sequence only needs to be roughly 150bp in length in order to recombine with the flanking regions. In theory, our genetically engineered B. subtilis could be customized to detect any pathogen whose genome has been sequenced to the extent where choosing a 150bp target sequence is possible. With the presence of ever-expanding public genome databases, the number of diseases our device can diagnose will continue to grow. Beyond the field of diagnostics, our device could be used as part of molecular biology techniques in order to detect various DNA or RNA sequences, similar to DNA microarrays. If used properly, the applications of our device in science and medicine are expansive.
Video: A 3D animation of our device from start to finish made using Autodesk Maya ( Click here if video isn't working )
Retrieved from " http://2014.igem.org/Team:Calgary/Project/BsDetector "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
