"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Calgary/Project/BsDetector/invivoDetection
From 2014.igem.org
Project
B.s. Detector
Target Diseases
Sample Preparation
B.s. Chassis
Genetic Circuit
Modelling & Optimization
Device Prototype
Platform
Collaboration
Achievements
Final System
Our Team
Team Members
Official Profile
Outreach
Telus Spark
Beakernight
About the University
Contact Us
Attributions
Sponsors
Notebook
Journal
Detectives
Transformers
Modelling & Prototype
Policies & Practices
Protocol Manual
General
DNA
B. subtilis
Sample Preparation
Parts
References
Policies
Diagnostic Landscape
Implementation
Informed Design
Safety
Biological Weapons Convention
FIND
iGEM
Calgary 2013
Calgary 2012
Calgary 2011
in vivo Detection
Our pathogen detection system consists of two collections of interconnected genes (operons) placed inside specific regions (loci) of the chromosome. The first operon is designated the "reporter" and consists of a constitutive promoter (Pveg), a ribosomal binding site, a repressible operator (C2P22), and (lacZ), a gene capable of producing a blue pigment (5,5'-dibromo-4,4'-dichloro-indigo) in the presence of X-gal. The blue pigment is made when the enzyme beta-galactosidase, encoded by lacZ, cleaves the X-gal molecule. Our "reporter" operon works in tandem with our "repressor" operon, which consists of the same promoter, ribosomal binding site, and a repressor gene designed to negate the function of the aforementioned operator gene (C2P22). Repression of the operator within the reporter operon is made possible by the unique repressor proteins produced by the repressor gene. The repressor gene is flanked by sequences homologous to a conserved target sequence of our intended pathogen of interest, such that we can identify any strain present through homologous recombination.
By default, when the two operons are placed inside B. subtilis, the repressor gene will act upon the operator of our reporter and prevent the transcription of lacZ). With the chromophore not being produced, the B. subtilis will refrain from exhibiting a colorimetric output and will remain in the "negative state". However, when a particular pathogen is introduced to our engineered B. subtilis, it will uptake the target sequence contained within the pathogen through homologous recombination. Essentially, the repressor gene, which is flanked by DNA regions homologous to the target sequence, will be replaced by the target sequence and "knocked off" the chromosome. With the repressor gene absent, the reporter operon will be at liberty to produce the chromophore and cause B. subtilis to yield a blue colorimetric output.
Figure 1: DNA detection via homologous recombination. A) Reporter operon consisting of a constitutive promoter (grey arrow), ribosome-binding site (RBS; semicircle), operator (orange), and coding sequence of a reporter (lacZ). Expression of the reporter is repressed in the default state. B) Repressor system for regulation of reporter expression. i) Repressor operon consisting of a repressor driven by a constitutive promoter. Expression of the repressor inhibits the expression of the reporter - the repressor specifically targets the operator shown in a). Blue regions indicate flanking sequences that are homologous to the amplified target DNA as shown in ii). Upon homologous recombination (dotted lines), the disruption of the repressor operon leads to active reporter expression. ii) Target DNA from pathogen, amplified via isothermal-PCR (iPCR). Blue regions indicate sequences homologous to the flanking regions of the repressor operon as shown in i).
A major strength of our project design lies in its high level of customization and modularity. Using our reporter and repressor operons, we can in theory facilitate the detection of any pathogen that might be present within a blood sample simply by modifying the homologous flanking regions of our repressor gene with the proper sequences. To make this possible, we would only need to identify beforehand a target sequence for our pathogen of interest by using a public genome database . Our experiments have shown that a mere 150bp target sequence is sufficient for homologous recombination. In theory, our device should be able to detect any pathogen whose genome has been sequenced to an adequate degree and has been made publicly available.
In keeping with the consistent record of safety present in iGEM, we have also taken cautious measures to ensure that our engineered B. subtilis cells are virtually harmless in the unlikely event that they escape the casing of our device. The reporter and repressor genes have been placed into the thrC locus of B. subtilis and have effectively replaced the genes that were originally present in this location. Because the thrC gene is needed for the synthesis of threonine - an essential amino acid - replacing it with our operons makes B. subtilis incapable of producing endogenous threonine and thus renders it auxotrophic. Any amount of B. subtilis which may escape our device would be unable to replicate and die promptly.
Retrieved from " http://2014.igem.org/Team:Calgary/Project/BsDetector/invivoDetection "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
