<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Calgary_Notebook_Journal_Transformers"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Calgary/Notebook/Journal/Transformers">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Calgary/Notebook/Journal/Transformers&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Notebook/Journal/Transformers&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Notebook/Journal/Transformers&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Calgary/Notebook/Journal/Transformers" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Calgary/Notebook/Journal/Transformers</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><SECTION id="Page"><HEADER id="TopBar"><UL id="TopLvlNav"><LI><A href="https://2014.igem.org/Team:Calgary/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Project/BsDetector"><I>B.s.</I> Detector</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Project/BsDetector/TargetDiseases">Target Diseases</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Project/BsDetector/SamplePreparation">Sample Preparation</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Project/BsDetector/BsChassis"><I>B.s.</I> Chassis</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Project/BsDetector/GeneticCircuit">Genetic Circuit</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Project/BsDetector/ModellingAndOptimization">Modelling &amp; Optimization</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Project/BsDetector/DevicePrototype">Device Prototype</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Project/BsDetector/Platform">Platform</A></LI></UL><LI><A href="https://2014.igem.org/Team:Calgary/Project/Collaboration">Collaboration</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Project/Achievements">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Project/FinalSystem">Final System</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/OurTeam">Our Team</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/OurTeam/TeamMembers">Team Members</A></LI><LI><A href="https://igem.org/Team.cgi?id=1444">Official Profile</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/OurTeam/Outreach">Outreach</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/OurTeam/Outreach/TelusSpark">Telus Spark</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/OurTeam/Outreach/Beakernight">Beakernight</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/OurTeam/TheUniversity">About the University</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/OurTeam/ContactUs">Contact Us</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/OurTeam/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/OurTeam/Sponsors">Sponsors</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/Journal">Journal</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/Journal/Detectives">Detectives</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/Journal/Transformers">Transformers</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/Journal/ModellingAndPrototype">Modelling &amp; Prototype</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/Journal/PoliciesAndPractices">Policies &amp; Practices</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/ProtocolManual">Protocol Manual</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/ProtocolManual/General">General</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/ProtocolManual/DNA">DNA</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/ProtocolManual/Bsubtilis"><I>B. subtilis</I></A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/ProtocolManual/SamplePreparation">Sample Preparation</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/References">References</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/PoliciesAndPractices">Policies</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/PoliciesAndPractices/DiagnosticLandscape">Diagnostic Landscape</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/PoliciesAndPractices/DiagnosticLandscape/Implementation">Implementation</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/PoliciesAndPractices/DiagnosticLandscape/InformedDesign">Informed Design</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/PoliciesAndPractices/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/PoliciesAndPractices/BiologicalWeaponsConvention">Biological Weapons Convention</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/PoliciesAndPractices/FIND">FIND</A></LI><LI><A href="https://2014.igem.org/Main_Page">iGEM</A></LI><LI><A href="https://2013.igem.org/Team:Calgary" target="_blank">Calgary 2013</A></LI><LI><A href="https://2012.igem.org/Team:Calgary" target="_blank">Calgary 2012</A></LI><LI><A href="https://2011.igem.org/Team:Calgary" target="_blank">Calgary 2011</A></LI></HEADER></SECTION></P><H1>Transformers' Journal</H1><H2>Week 1: May 1-2</H2><P>
    Our first week consisted mainly of formal introductions and various brainstorming sessions regarding the direction of the 2014 project. The options were narrowed down to a biofilm based water filtration system and a rapid, multi-diagnostic test. Experts in the field of bioremediation, molecular biology, and medicine were consulted in order to gain more insight on the prospects of both project ideas. Ultimately, the rapid multi-diagnostic was voted over the water filtration system and we began preliminary planning.</P><H2>Week 2: May 5-9</H2><P>
    We attended a series of workshops tailored towards those who are researching various aspects of molecular biology. We learned the basics of genetic engineering and lab protocol in preparation for future research. These workshops were especially helpful for our team members who had no prior experience working in a lab.</P><H2>Week 3: May 12-16</H2><P>
    The techniques we learned during the workshops were applied within our own lab; we made stocks of competent <I>E. coli</I>., poured anti-bacterial agar, performed plasmid mini-prep, ran gels, etc.. The team decided upon an in-vivo diagnostic system that could detect pathogens through homologous recombination. Literature research was conducted on the subject of how to enhance the natural transformation of <I>E. coli</I>.</P><P>
    In regards to <I>E. coli</I>., it was discovered that <I>E. coli</I>. was closely related to <I>H. influenzae</I>, a species of bacteria well known for it's high frequencies of natural transformation (competency). <I>H. influenzae</I> uses a Type-IV pilus structure to uptake DNA, so the team researched whether or not such a system could be implemented into <I>E. coli</I>. to yield similar transformation rates. Studies conducted by Dr. Rosemary Redfield - an <I>E. coli</I>. expert - showed that all gram-negative bacteria share two common regulator genes for competency; <I>sxy</I> and <I>CRP</I><A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">(Sinha &amp; Redfield, 2012)</A>. A deeper analysis of Dr. Redfield's papers also revealed that <I>E. coli</I>. lacks a pilF2 homologue. Curious, we contacted Dr. Redfield to gain insight into the subject. Dr. Redfield stated that during her experiments, all the proteins involved in DNA uptake were able to be expressed, but the rates of competency in <I>E. coli</I>. were quite low to begin with. It was decided that <I>E. coli</I>. would not be used for our project as there were many unknown variables which dictated the expression of competence gene homologues in <I>E. coli</I>.</P><P>
    Additional studies on <I>E. coli</I>. conducted by Sun et. al. were looked at. The findings of these studies stated that OmpA (an outer membrane protein) plays a role in uptaking DNA from conjugative plasmids and bacteriophages <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">(Sun, Wang, Chen, &amp; Zhan, 2013)</A>. The researchers also hypothesized that OmpA plays a stimulatory role in DNA uptake in liquid Ca2+ cultures, along with an inhibitory role on agar plates <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">(Sun et al., 2013)</A>. OmpA negative strains (JW0940) were found to have a higher transformation frequency than wild-type strains (BW25113) when grown on agar <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">(Sun et al., 2013)</A>. Surprisingly, transformation frequencies did not change with varying agar thickness, so it remains uncertain how agar plays a role in <I>E. coli</I>.'s natural transformation <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">(Sun et al., 2013)</A>. Conversely, the OmpA negative strains showed a lower frequency of transformation than wild-type strains when grown in liquid Ca2+ cultures <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">(Sun et al., 2013)</A>. We concluded that it may be possible to make <I>E. coli</I>. naturally competent on agar by replacing the <I>OmpA</I> gene with a gene that codes for kanamycin resistance.</P><P>
    In additional to <I>E. coli</I>., <I>B. subtilis</I>. was researched as a potential platform for DNA uptake. Current trends and methods in <I>B. subtilis</I> transformation were analyzed, with competency rates ranging from 104 to 107 transformants per microgram <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">(Zhang &amp; Zhang, 2013)</A>. One method utilized xylose in conjunction with the pAX01-comK plasmid to overexpress the competency regulator gene, <I>ComK</I>, in <I>B. subtilis</I> and create &quot;supercompetent cells&quot; <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">(Zhang &amp; Zhang, 2013)</A>. This yielded the highest transformation rate among the methods studied <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">(Zhang &amp; Zhang, 2013)</A>.</P><P>
    Other members of our team developed the basis for a &quot;genetic circuit&quot; which could detect the DNA of pathogens upon uptake by a competent bacterial cell.</P><H2>Week 4: May 20-23</H2><P>
    Literature searches were conducted on the regulation of competence in <I>B. subtilis</I>. We contacted Dr. Zhang and requested the pAX01-comK plasmid, also making sure to ask if only certain strains of <I>B. subtilis</I> can be made supercompetent. Dr. Zhang stated that any 168 derivative strain could be made supercompetent and told us to request the necessary plasmid from the Bacillus Genetic Stock Centre. Further literature searches were conducted to find the best plasmids to induce competency in <I>B. subtilis</I>. A highly experienced <I>B. subtilis</I> researcher at the University of Calgary, Dr. Wong, was contacted in order to gain more information on <I>B. subtilis</I> and hopefully acquire a supercompetent <I>B. subtilis</I> strain.</P><P>
    Following the decision to use <I>B. subtilis</I> as the platform for our in-vivo diagnostic tool, we officially divided our team into two groups: the Transformers and the Detectives. The goal of the former is to facilitate high transformation rates in <I>B. subtilis</I>, while the goal of the latter is to develop and tweak the pathogen detection system.</P><H2>Week 5: May 25-30</H2><P>
    We attended the Alberta Innovates Technology Futures workshops hosted by geekStarter in order to learn more about policy &amp; practices, presentation skills, current trends in synthetic biology, and 3D digital modelling. Speakers included Kelly Drinkwater of iGEM, Cesar Rodriguez of Autodesk, and David Lloyd of FredSense. Following the presentations, the speakers provided useful feedback on our project from both a scientific and human practices perspective. We also met the Lethbridge iGEM team and was introduced to their project.</P><P>
    Later in the week, we prepared overnight cultures and plasmid mini-preps for the parts required to assemble the genetic circuit. We also created antibiotic solutions for spectinomycin, erythromycin, and lincomycin in preparation for the supercompetent <I>B. subtilis</I> colonies we were going to receive, along with future use. The <I>B. subtilis</I> strains were generously donated to us by Dr. Wong and Dr. Wu of the University of Calgary. Given the importance of these strains, we immediately prepared backup stocks of the received colonies and incubated them. Meanwhile, colony PCR was performed on <I>E. coli</I>. colonies containing tetR+RBS and the 5`integration, which we prepared beforehand. Dr. Wong recommended we do additional background research on the life cycle and biology of <I>B. subtilis</I>.</P><P>
    We learned that <I>B. subtilis</I>, like most bacteria, are dependent on quorum sensing when trying to induce competence. When <I>B. subtilis</I> cells cluster together and their cell density increases, a collection of small peptides (called competence hormones) are excreted from each cell and recognized by adjacent cells. Research shows that the cells will only become competent if the concentration of these peptides is quite high <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">(Graumann, 2012)</A>. Research was also done on sporulation, a process by which bacteria enter a dormant state in order to survive adverse conditions such as starvation, irradiation, and heat <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">(Eichenberger, 2012)</A>. As <I>B. subtilis</I> reach the stationary phase of their life cycle, some cells acquire competence while others sporulate <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">(Maier, 2012)</A>. We determined that <I>B. subtilis</I>' tendency to sporulate under extreme conditions may be beneficial for our purposes, as the durability of a <I>B. subtilis</I> spore would facilitate an easy transportation of our diagnostic tool around the world. In theory, the spores would not require refrigeration en-route to their destination and could be stored for significantly long periods of time.</P><P>
    In regards to sporulation, the mother cell and the foreshore are genetically identical, but certain proteins must be made specifically in the developing spore. Thus, certain set of genes transcribed from mother cell DNA must differ from the set transcribed from foreshore DNA. The sporulation sigma factors replace the principal negative cell sigma factor A in RNA polymerase holoenzyme - possibly by out competing sigmaA for RNA polymerase <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">(Eichenberger, 2012)</A>.</P><H2>Week 6: June 2 - 6</H2><P>This week we requested parts specific to <I>B. subtilis</I> through the iGEM registry, as they were absent on the 2013 and 2014 plates we already had in our possession. Meanwhile, we performed plasmid mini-preps and confirmation digests, also designing primers for <I>lacA</I>,  <I>thrC</I>, and <I>amyE</I>. Doing this was necessary because the necessary <I>B. subtilis</I> integration vectors were unavailable on current distribution plates.We then decided to submit these sequences to the iGEM registry for the benefit of future teams who may want to use them.</P><P>We dedicated an entire day to primer design and uploaded our newly developed primer sequences to Benchling, an online, cloud-based sequencing tool. Benchling was especially useful for our purposes as it allowed the easy sharing of numerous sequences amongst the whole team sans the hassle of word processors and e-mails. Due to Benchling’s numerous built-in features (e.g.the  displaying of enzyme cut sites, base pair tracking, automatic melting point calculations, illustrated sequence maps, etc.) productivity increased dramatically and a significant amount of time was saved during the designing of the primers. Once the necessary primers were designed, a request was sent out for them.</P><P>Colony PCRs were performed on the plasmids the Detective team had constructed thus far, but the colonies harbouring GFP and LacI ultimately failed. However, the failure of these colonies led us to the discovery that the plasmid containing LacI with a Chloramphenicol resistance grew better on Ampicillin agar plates. This strange phenomenon was explained by the hypothesis that ampicillin may have been more easily diffused by the bacteria.</P><P>Towards the latter half of the week , our Transformer team determined the components of the BioBrick we wished to construct. We decided to construct this BioBrick in order to gauge the transformation efficiency of <I>B. subtilis</I> and prepare ourselves for the insertion of the actual pathogen detection system in the coming weeks. It was crucial that we determine beforehand the loci into which we would insert our BioBrick. Simply put, our BioBrick was designed to “knock out” and replace the original gene within the receiving locus. This would mean that upon the successful insertion of our BioBrick, the bacteria would lose a certain inherent function, such as the metabolization of starch or the synthesis of threonine, at the cost of integrating our BioBrick. This “loss of function” strategy would allow us to determine which cells were successfully transformed.  After much research, we narrowed down the potential receiving loci to <I>amyE</I>, SacA, <I>thrC</I>, and <I>lacA</I>.   Meanwhile, we prepared more agar plates containing Chloramphenicol and Ampicillin as our initial supply was running low. Much to our relief, we later received an email from iGEM headquarters stating that our requested parts were in transit.</P><H2>Week 7: June 9 - 13</H2><P>This week consisted of ligations and the confirmation thereof. First, colony PCR was conducted on existing RFP transformed colonies to confirm their viability for the genetic circuit which is to be assembled the following week. Fortunately, all of the six colonies which were screened turned out to be successful. Following the confirmation of the RFP colonies, Pveg + RBS ligations were created and transformed into the competent Top 10 <I>E. coli</I>. stocks at hand. One out of several colonies was successful. A confirmation digest was performed on the successful colony following a plasmid mini-prep, and for reasons unknown, the results were negative.</P><P>A single successful Pveg + RBS colony was deemed insufficient for our purposes, thus a second, identical ligation was conducted on new <I>E. coli</I>. stock. Meanwhile, another colony PCR was performed on the first ligation, and all tested colonies were ultimately negatives. Fortunately, two of the newly ligated were determined to be successful, ensuring ample Pveg + RBS stock for future use.</P><P>We began to prepare our <I>B. subtilis</I> 1A976 (comK) cells for transformation, following the protocol outlined by Zhang et al. (ref). We made 1% (w/v) xylose solution in LB, as xylose induces the comK promoter to generate supercompentency in <I>B. subtilis</I>. We inoculated LB (containing erythromycin and lincomycin) with the <I>B. subtilis</I> cells and shook them overnight (37°C). The protocol suggested an incubation period of around 12 hours, but we found that optimal cell density required a slightly longer period in the incubator/shaker. We adapted the protocol to fit our needs, and decided upon a shaking period of 18 hours.</P><P>The protocol required the dilution of our liquid cell cultures with 1% xylose solution until an absorbance reading of 1.0 at A600 was achieved. We made several replicates of these cultures and generated a reference curve involving change in absorbance reading as a function of 1% xylose added. This way, future cultures can be diluted more efficiently.</P><P>Finally, our team was happy to receive the <I>B. subtilis</I> parts we ordered from iGEM HQ. These were in the form of a liquid culture, so we incubated them shortly after. We also received the custom primers we requested from the University of Calgary synthesis lab the week prior. These were diluted immediately and stored safely for future use. Now that we have the <I>B. subtilis</I> parts, the custom primers, and ample <I>B. subtilis</I> glycerol stocks, we will transform our <I>B. subtilis</I> with the Detective team’s genetic circuit as soon as possible.</P><H2>Week 8: June 16 - 20</H2><P>By the beginning of this week, we had in our possession a sufficient supply of <I>B. subtilis</I> integration vectors (SacA, thrC, <I>amyE</I>) and began tinkering with them. However, the iGEM registry stated that the integration vectors needed to be linearized with the ScaI restriction enzyme before any transformations could take place. Upon a closer look at the ScaI enzyme, we soon discovered that it had two cut sites, although we required only one. We carried out restriction digests on our vectors of interest to confirm that ScaI cuts them only once and refrains from cutting in unwanted places. The restriction digest of our SacA vector met our expectations, but we decided to conduct a second set of digests for the other vectors.</P><P>Our team also continued with the ligations from last week, attempting to combine GFP with the Pveg+RBS ligation product we had created earlier. Both the first and second attempts failed, but we decided to try a third time towards the end of the week. Samples of DNA from previous weeks were also sent to the sequencing centre and the results came in quickly.</P><P>We prepared for the transformation of <I>B. subtilis</I> - a major milestone of our project - by preparing M9 minimal media, inducing supercompetency in our <I>B. subtilis</I>, and linearizing the integration vectors. With our <I>B. subtilis</I> colonies almost ready to be transformed, we worked together to gather as quickly as possible the parts necessary to build our reporter system (i.e., the genetic circuit). Although we were busy in the lab this week, we also kept the community outreach portion of our project in mind and began preparations for our next event; painting with fluorescent bacteria.</P><H2>Week 9: June 23-27</H2><P>We continued with the restriction digests of our integration vectors from last week, digesting the <I>thrC</I>, <I>amyE</I>, and sacA vectors with the ScaI enzyme. After running a gel, we luckily saw the proper bands in the right places. However, we also noticed a single band around the 800 bp range, and after contacting iGEM HQ, we concluded that it must have been the RFP generator. We decided to contact the LMU team to discuss our findings. Following the restriction digests, we proceeded to cut two of the vectors with BsaI since the ScaI enzyme cut out the <I>bla</I> gene.</P><P class="Center"><B>Figure 1: ScaI was used to linearize the thrC, sac A and Amy E vectors. An unexpected 1000 bp product was present in all of the samples. Amresco 1000Kb ladder was used inorder to determine the size of the bands.</B></P><P>The team met one of it’s largest project goals this week, having finally transformed supercompetent <I>B. subtilis</I> with a plasmid we designed.. Unfortunately, our first set of transformations failed and we witnessed a “lawn” of bacteria on our plates. After reviewing various papers, we realized that our antibiotic plates only had a chloramphenicol concentration of 5mg/L, when the optimal concentration was actually 100mg/L (of?) and 5mg/L of chloramphenicol. We also suspected our antibiotics to be defective, but nonetheless decided to attempt a second set of transformations on 30mg/L chloramphenicol plates.</P><P>We successfully transformed <I>B. subtilis</I> with select vectors (we observed red colonies, which means that the vectors integrated since it includes RFP in its backbone).</P><P class="Center"><B>Figure 2: <I>B. subtilis</I> was transformed with the thrC vector, the sac A vector and the <I>amyE</I> vector. All three of these vectors carry the RFP cassette.</B></P><H2>Week 10: June 30 - July 4</H2><P>This week we attempted to optimize our <I>B. subtilis</I> transformations by using fresh <I>B. subtilis</I> cells containing the <I>comK</I> gene in lieu of frozen samples. Unfortunately, working with fresh <I>B. subtilis</I> cells resulted in no viable transformants. Last week we were able to successfully transform frozen <I>B. subtilis</I> cells with all three of our integration vectors, however each plate yielded only a handful of colonies and many of these colonies were transformed. After some discussion, we reached the conclusion that our <I>B. subtilis</I> cells either had a very high transformation efficiency, or a very low cell viability. We also assumed there was an issue with our antibiotic plates (spectinomycin) since untransformed colonies (white) were able to grow.</P><P>To test the viability of our <I>B. subtilis</I>, we will be manipulating the amount of glycerol added to our stocks before storing them at -80°C. We plan on completing this as soon as possible next week. We will also modify certain aspects of our current transformation protocol. First, we will use freshly streaked cells for our transformations, as we are under the impression that stale cells may be more fragile. Next, we will grow the cells for a shorter period of time in order to minimize any dilutions caused by adding the 1% LB-Xylose solution. Alternatively, we will use a 2% xylose (w/v) solution to ensure that all of the <I>B. subtilis</I> cells receive enough xylose to achieve supercompetency. Finally, we plan on including both a 10 minute ice and room temperature incubation step prior to the 90 minute shaker step. We believe that this will reduce the trauma experienced by the cells due to the rapid temperature fluctuations.</P><P>While scrutinizing our transformation protocol, we also tested whether our vectors integrated into the desired locus by using colony PCR. We did not see any bands for the sacA vectors, however we also saw abnormal  samples that were not even run on the gel. We believe that our initial lysis step was not long enough, thus the Taq enzyme could not get access to the DNA and the PCR failed.. We will attempt a second colony PCR with a longer lysis step early next week.. For the lacZ vector we saw a band that was approximately 2700 bp long; this was consistent among our three replicates. However, we were expecting to see a band that was actually ~6000 bp long and hypothesized that the <I>B. subtilis</I> may have forcefully ejected the <I>lacZ</I> genes.Dr. Wong had warned us earlier that <I>B. subtilis</I> has a tendency to eject genes when multiple copies are present. After taking into account the RFP generator we were still missing ~2000 bp. We decided to get this product sequenced in order to determine exactly what parts of the vector were being amplified by our vectors. For the <I>thrC</I> vector we saw a band around 4000 bp, which was exactly what we were expecting after taking the RFP generator into consideration!  Only one of the three replicates produced a band, and as mentioned previously, many unknown samples were found in our gel. We plan on modifying the protocol in order to receive more favourable results.Next week, we will try our PCR again using confirmation methods suggested by the 2012 LMU iGem team (threonine test for the thrC vectors, starch test for the <I>amyE</I> vector and colony PCR with their primers).</P><H2>Week 11: July 7 - 11</H2><P>During our weekly meeting with the faculty advisors, we further discussed the variables we will be manipulating to optimize our <I>B. subtilis</I> transformation protocol. Our supervisor Dr. Tony Schryvers suggested we not shake the cells in the incubator after exposing them to our plasmids. Based on his experience he warned us that vigorous shaking may sheer the pilus; a vital structure bacteria used for the uptake of DNA.
-&gt; (reference literature here)</P><P>In the lab, we continued with the task of optimizing our transformation protocol by testing the effects of higher xylose concentrations, using fresh comK cells, and omitting the shaker step following the addition of plasmids to our <I>B. subtilis</I> cells. In addition, we prepared MNGE-agar and starch plates in order to determine whether our integration vectors are being inserted into the proper loci.

-&gt; (Nice pictures)</P>We also heard back from the LMU iGEM team regarding the unusual bands we saw in last week’s colony PCR of the integration vectors. They suggested that it may be an RNA contamination, thus we tried using RNase to remove the RNA. Unfortunately, no bands were seen on the gel created thereafter.

-&gt; (Maybe a picture of last week’s gel beside the gel with no bands?)<H2>Week 12: July 14 - 18</H2><P>We did not get accurate results from last week’s optimization due to plating the bacteria on the wrong resistance plates. We redid the experiments but this time just focusing on transforming the ComK strain with the thrC vector and compared the effects of the following on transformation: 1% and 2% xylose, shaking and not shaking following xylose induction, and ComK cells that were stored in 10% and 25% glycerol. Initial trials of manipulating these variables showed 2% xylose, no shaking, and 10% glycerol to lead to greater growth of colonies and therefore better transformation. More trials need to be conducted before we can make definite conclusions.</P><P>Although there was colony growth for all of the variables tested, none of the colonies turned red despite the vector containing an RFP cassette. We conducted colony PCR and ran the products on a gel, on which bands of the right size appeared. Literature search is still being done as to the cause behind the colonies not turning red.</P><P>We further tested for the insertion of the thrC vector with the threonine test, in which transformed colonies with the thrC insertion would only grow on MNGE media with the amino acid threonine but we did not see any growth on this plate.</P><H2>Week 13: July 21 - 25</H2><P>The team decided on having each subgroup present their findings in the form of a work-in-progress presentation at the weekly meetings and the Transformers were the first group to present. We received valuable input from our advisors, such as having quantitative results for our transformations instead of qualitative ones that consisted of pictures of plates.</P><P>We planned on making serial dilutions of each transformation next week and repeating conditions we have already tested in order to obtain quantitative results. We also researched on protocols available to make spores. We found one protocol, but we are missing one reagent required to make a media. We are still searching for this reagent and once we obtain it, we will immediately start working on making <I>B. subtilis</I> spores.</P><H2>Week 14: July 26 - August 1</H2><P>In order to quantify transformation, we did 1/10 serial dilutions of <I>B. subtilis</I> transformed with thrC and tested 1% xylose vs. 2% xylose, along with shake vs. no shake. All of the colonies were red, suggesting successful transformation. The amount of colonies consistently decreased with each dilution, except for the 2% xylose + shake condition, for which colonies only grew on the the 100% concentrated plate. Qualitatively, the 2% xylose + no shake condition resulted in the most number of colonies on the 1/10 000 dilution plate, however there were too many colonies to count and this suggested that we should dilute the colonies further in the future.</P><P>
    When discussing the results with Dr. Schryvers, he recommended some changes to our protocol. One major change we considered was using a stock solution of 10% xylose to dilute our cells since previously we were unaware of the final concentration of xylose that was added to the cells. We will apply these changes to our future transformations.</P><H2>Week 15: August 5 - August 8</H2><P>We instituted the changes suggested by Dr. Schryvers to our existing protocol. However, cells did not seem to transform, as plenty of growth was observed on the plain agar plates, but nothing grew on the spectinomycin plates, suggesting that DNA was not uptaken by the cells. Similarly, none of the colonies expressed RFP. Thus, transformation continues to be a work in progress.</P><P>We borrowed a small amount of calcium nitrate from Dr. Wong, as it was the only reagent we were missing for making 2x SG media for sporulation of <I>B. subtilis</I>. We obtained the sporulation protocol and media recipe from <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">OpenNetWire</A>. We put the spores into the shaker for incubation over the weekend for the sporulation process to take place.</P><H2>Week 16: August 11 - 15</H2><P>We harvested the spores that grew over the weekend by spinning down small volumes in the benchtop centrifuge. They were then washed with cold nanopure water. Once we obtained these spores in 2mL tubes, half were designated for storage in the fridge (short-term, require weekly changes of water), and the other half were designated for long-term storage in the -20°C freezer.</P><P>We then researched different ways we could verify the presence of spores. These primarily compose of:

Researched ways to verify their presence
Gram stain! But no reagents</P><H3> Transforming <I>B. subtilis</I> with PCR product</H3><P>Naturally competent bacteria take up foreign DNA when exposed to conditions of energy starvation <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">(Hamoen, Venema, Kuipers &amp; 2003)</A>. Once taken up, foreign DNA is generally degraded into nucleotide constituents and used as an energy source <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">(Hamoen, Venema, Kuipers &amp; 2003)</A>.  Because <I>B. subtilis</I> has an aggressive mechanism for doing so, we are interested in determining how much “buffering” DNA is required to protect the sequence we wish to recombine into the <I>B. subtilis</I> genome. <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">(Khasanov et al)</A>. determined that a minimal of 70 bp is required for homologous recombination to take place. Furthermore, it was determined that a linear relationship exists between target sequence length and recombination frequency <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">(Khasanov et al)</A>.  The target sequence length and transformation efficiencies we obtain using the supercompetent <I>B. subtilis</I> strain will play a significant role in our sample preparation. We can use our results from this experiment to determine how long our primers need to be for the target sequence within the pathogen(s) during isothermal amplification.</P><P>Currently, we have been optimizing transformation efficiency using the <I>thrC</I> vector digested with ScaI. When we digest the <I>thrC</I> vector with ScaI we have approximately 700 bp of “buffering DNA” on one side and 1800 on the other.  We have designed several primers for different sites within the <I>thrC</I> and <I>thrC</I>’ regions for the <I>thrC</I> vector (ID K823022).  These primers will give us products containing 500 bp of homologous sequence down to 50 bp. By using these primers we hope to determine how much buffering DNA and homology is necessary.</P><H2>Week 17: August 18 - 22</H2><P>We made the first batch of spores of the supercompetent <I>B. subtilis</I> strain this week using a <A href="https://2014.igem.org/Team:Calgary/Notebook/ProtocolManual/Bsubtilis">standard sporulation protocol</A> and stored them at 4 C for short-term storage and -20 C for long-term storage. In order to visualize the spores under the microscope, we performed Gram stains. Theoretically, spores should stain clear, vegetative <I>B. subtilis</I> cells should stain purple, while <I>E. coli</I> cells should stain red. This is because the Gram’s iodine reagent fixes the purple dye (crystal violet) to Gram positive species, while the purple is unable to adhere to Gram-negative species. Thus, <I>B. subtilis</I>, which is Gram-positive, will stain purple, while <I>E. coli</I> stains red (from Safranin). Spores lack the membranes that bind the colour, and so will not stain at all, and we see them as round, clear entities under the microscope.</P><P>A majority of the cells from the sporulated tube were in the spore form, being round-shaped and clear. Although <I>E. coli</I> stained red, vegetative <I>B. subtilis</I> cells also stained red instead of purple. However, the <I>B. subtilis</I> cells were rod-shaped so we assumed that the reagents we used for Gram staining were contaminated or out-of-date. We are planning on obtaining new reagents to try the Gram stain again.</P><P>We attempted transformation by incorporating the suggestions made by Dr. Schryvers. Instead of diluting the overnight culture directly with xylose, we diluted it with LB media that was also shaken overnight. This was followed by adding xylose to make a final xylose concentration of 1%. Unfortunately, we did not get transformation despite repeated trials. We returned to our old method of diluting the overnight culture with a stock solution of 2% xylose. However, this method no longer worked. We are guessing that our cells might have been contaminated so we will try to get new cells.</P><H3> Transforming <I>B. subtilis</I> with PCR product</H3><P>This week we received the primers designed for the <I>thrC</I> sites. We tried using miniprepped <I>thrC</I> vector to obtain our desired PCR products which we could then use to transform ''B. subtilis''. Our first few tries were unsuccessful. Our team advisor suggested diluting the miniprepped sample since too much template can interfere with PCR. We tried a 1 in 100 dilution and it worked!</P><H2>Week 18: August 25 - August 29</H2><P>Dr. Wong was kind enough to provide us with a fresh batch of supercompetent <I>B. subtilis</I> cells. Initially, we were still having trouble transforming the cells since the starting absorbance was not high enough, but we figured out that this was due to the LB being contaminated. For the first set of transformations, although colonies that grew on the plain agar plates were clear while colonies that grew on spectinomycin plates were red suggesting successful transformation, this was not the case; the untransformed colonies grew on the spectinomycin plates, which should not have occurred. Also, these cells were red so they were somehow exposed to the thrC DNA.</P><P>The second set of transformations were also unsuccessful because not only were the colonies that grew on the plain agar plates red, but so were the untransformed colonies that grew on the plain plates. We are guessing that our pipettes might be contaminated with spores, which are highly resistant to heat and ethanol among other things <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">(Eichenberger, 2012)</A>.</P><H3> Transforming <I>B. subtilis</I> with PCR product</H3><P>This week we transformed <I>B. subtilis</I> using PCR product. Unfortunately, each trial gave us different results. For the first trial we used frozen <I>B. subtilis</I> cells (1% xylose, 10% glycerol). The plates did not contain any colonies on the selective pates. Therefore, we concluded that there may have been a problem with our stock. We tried again using a different batch of comK cells (1%xylose). The results are shown below.  We successfully transformed <I>B. subtilis</I> using DNA amplified from the thrC vector using primer set 0, 1, 2, 3, and 4 (500-125 bp of homologous sequence). We were unable to successful transform bacteria with primer product containing 50 bp of homologous sequence, this was expected based on the literature. A few of our selective plates showing  transformed with PCR product and digested thrC vector (plate A) are shown below. We got a single colony for <I>B. subtilis</I> transformed with PCR product using primer set 2 and 4 (215 bp and 125 bp of homology, respectively). It was surprising to see higher transformation efficiency using primer product 3 (190 bp of homologous sequence) compared to primer product 2 (215 bp of homologous sequence) since transformation efficiency is found to decrease with a reduction in homologous sequence.  We did not control the DNA concentration which could explain these results.  Furthermore, we used B. subtilis stock cells that were made competent using 1% xylose and stored in 10% glycerol. We are currently working on optimizing the Zhang protocol, we expect to see greater efficiency once we confirm  the best conditions for inducing competence.</P><P class="Center"><B>Figure 3:</B><I>B. subtilis</I> was transformed with DNA containing various lengths of homologous sequence and was plated on selective spectinomycin plates. Plate A shows <I>B. subtilis</I> colonies transformed with digested <I>thrC</I> vector  (700, 1300 bp of homology). Plate B, C and D show <I>B. subtilis</I> transformed with PCR product 2,3 and 4 respectively (215 bp, 190 bp and 125 bp of homology).</P><H2>Week 19: September 2 - 5</H2><P>We continued to work on transformations but we were consistently unsuccessful.  After trying to transform cells that were made competent the day we transformed them, we turned to transform with glycerol stocks of competent cells that we had made earlier. Unfortunately, neither the fresh nor glycerol stock cells led to successful transformation.</P><P>We also made a new batch of spores with the more recent supercompetent <I>B. subtilis</I> colonies we obtained from Dr. Wong and stored them at 4°C and -20°C.</P><H3> Transforming <I>B. subtilis</I> with PCR product</H3><P>We were surprised to see white <I>B. subtilis</I> colonies growing on spec plates. The transformed colonies should have been red since the thrC integration vector contains both the RFP and spec cassettes. Untransformed cells (comK strain) do not carry the spec cassette therefore they should not grow on spec plates; this was confirmed by plating untransformed cells on plain agar and spec plates.</P><P>In order to ensure <I>B. subtili</I>s was actually being transformed with our PCR product, we did a colony PCR (Fig. 19.1) using thrC specific primers. We expected to see a 2300 bp product which is the size of DNA within the homologous regions in the thrC vector, this region includes the spec and RFP cassette. We either saw bands around 3000 bp or around 700 bp for <I>B. subtilis</I> transformed with PCR product. A <I>B. subtilis</I> colony that was transformed earlier in the year and expressed RFP also produced a 3000 bp band. Untransformed cells gave us a band around 600 bp). These results suggest that RFP expression was turned off in <I>B. subtilis</I> colonies in which a 3000 bp band was amplified. A mystery band around 700 bp was visible for a few of the samples. A different set of primers which amplified 150 bp more were used for a cPCR (Fig. 19.2). cPCR on untransformed colonies (comK) produced a product around 650 bp while the mystery bands appear around 850. This gel suggests that <I>B. subtilis</I> was being transformed. We speculated the transformed colonies containing the smaller bands kept the spec cassette (675 bp) and removed the RFP from its genome. Dr. Wong had warned us about <I>B. subtilis</I> constantly remodeling its genome under stress.</P><P class="Center"><B>Figure 4:</B> A cPCR using thrC specific primers was done on <I>B. subtilis</I> colonies transformed with PCR product. P2, P3 and P4 colonies were obtained using Primers containing varying amounts of homologous sequence (293 bp, 190 bp and 125 bp of homologous sequence respectively). A band around 3000 bp or 700 bp is visible for colonies transformed with PCR product amplified using primer 3. <I>B. subtilis</I> colonies which were transformed earlier in the summer (late June) and expressed RFP were used as a positive control, cPCR amplified a 3000 bp region. Invitrogen 1Kb plus ladder was used.</P><P class="Center"><B>Figure 5:</B> A cPCR using thrC specific primers was done on <I>B. subtilis</I> colonies transformed with PCR product. Untransformed colonies (UT) were compared with colonies transformed with PCR product containing 293 bp and 190 bp of homology. Invitrogen 1kb plus ladder was used to determine the size of the products.</P><P>Over the weekend we had our aGem workshop. We were fortunate enough to talk to Dr. Karmella Hyne, an experienced iGEM judge.  We told her that our transformed cells were not expressing RFP despite the fact that we had cells growing on selective plates and had results from the colony PCR which suggested that we had successful transformation. She suggested that RFP might be toxic to the cell or there was a transposon inserting into the gene inhibiting expression. As a team we decided to try this experiment using lacZ, an enzymatic reporter which we believed would be easier on the cell.</P><H2>Week 20: September 8- September 14</H2><P>After encountering many trials of unsuccessful transformations, we thought the issue with our transformations might have been due to the addition of inconsistent volumes of xylose. We were adding enough xylose to make the final absorbance 1.0 at 600 nm, but the starting absorbance was often not consistent. This meant that even though we were keeping the final absorbance and thus cell density consistent, the volume of xylose added was not consistent. There is the possibility that for some trials, not enough xylose might have been added for competency, which could be a potential explanation for our failed transformations. We thought that there might be a threshold for the addition of xylose required for competency, and that adding volumes beyond this threshold will still result in the competence of all cells. We reasoned that flooding the cells with a high concentration of xylose might be better to ensure that all cells are competent, instead of adding small amounts. We attempted transformation with final xylose concentrations of 3% and 5%, but we obtained the same results as previous trials with no growth on spectinomycin plates and thus no transformation.</P><H3>Staining</H3><P>
    Gram and spore stains of <I>Bacillus subtilis</I> was desired to ensure that after transformation had occured the cells still gave a result that was expected.  <I>B. subtilis</I> is a Gram-positive bacteria, and when a Gram-stain is performed on it a purple to dark blue colour is expected, where as <I>Escherichia coli</I> is Gram-negative and would stain red to a pink colour.  During a spore stain of <I>B. subtilis</I> the spores stain green and the cells stain pink.  <I>E. coli</I> was used as a positive control to ensure that the stain worked, providing a colour that was expected, and that the bacteria was growing successfully in our lab.  Two Gram stains for each of <I>E. coli</I>, <I>B. subtilis</I> were performed, as well as two spore stains for <I>B. subtilis</I> were performed.  Before the stains were performed new streak plates of the bacteria were madeto incubate at 37 ° C overnight, ensuring that the bacteria came from fresh colonies.  The bacteriawere plated on regular LB agar plates for incubation.  The spores for <I>B. subtilis</I> were not streaked, as Israt and Anna Fei were taking care of the spores by changing the water in which they were stored to keep them maintained properly.</P><OL>Plates:
	<LI><I>E. coli</I> GFP E0040 combined transform. July 21, 2014 LB-amp</LI><LI><I>E. coli</I> GFP E0040 combined transform. July 21, 2014 LB-amp</LI><LI><I>B. subtilis</I> comK strain. September 5, 2014 Erythromycin + Lincomycin</LI><LI><I>B. subtilis</I> comK strain. September 5, 2014 Erythromycin + Lincomycin</LI></OL><H3>Bacterial smear:</H3><P>
    These newly streaked plates provided the colonies for the bacterial smear slides necessary for the staining procedure.  Clean microscope slides were labeled according to the corresponding plates from which the colonies were taken, and a circle on the back of the slide was drawn to show the area in which the smear would be done.  2 μL of distilled water was dropped in the circled area on the slides, this water was then used to spread the bacteria around in.  The bacteria was spread using a sterilized, using a Bunsen burner, inoculation loop, before the bacteria was fixed by running the slide through the Bunsen burner flame a four times.  Once the smear was dried the staining was able to be performed.</P><H3>Gram stain:</H3><P>
    The bacterial smear was flooded with Crystal Violet for one minute before it was rinsed with distilled water.  
It was then flooded with Lugol's Iodine for one minute before being rinsed again.  The smear was then decolourized
using alternating applications of 95% ethanol for ten second and then distilled water until the water running off
of the slide runs clear.  The smear was then counter-stained with Safrinin for one minute before one last rinse 
with distilled water, and being blotted with Kim-wipes to dry the slide.
</P><P>
    These stains were performed for each plate, after which it was evident from the naked eye that the smears with 
<I>E. coli</I> were pink, and the smears with <I>B. subtilis</I> were dark purple.  In order to analyze the smears 
further they were analyzed under a high-tech photo-microscope, for this cover slips were placed on the smears to
protect both the bacterial smears and the microscope.</P><P class="Center"><B>Figure 6:</B><I>B. subtilis</I> Gram stain of regenerated spores.
</P><P class="Center"><B>Figure 7:</B><I>E. coli</I> Gram stain of regenerated spores.
</P><H3>Spore stain:</H3><P>
    Two bacterial smears were performed for the <I>B. subtilis</I> spores, before the stains were done.  To collect 
the spores for the smear the suspended spores were centrifuged for two minutes at 14,000 rpm, so that the spores 
could be collected from the pellet.  After the bacteria were fixed on to the slide and the smear was dried, the 
slides were placed onto a hot plate, at 150 °C, for the Malachite Green staining.  A piece of paper 
towel was placed over the smear so that it could be completely flooded and steamed with Malachite Green for five 
minutes.  The smear was continuously damped with the stain for the entire five minutes.  After the five minutes 
and the paper towel was removed the smear was rinsed with distilled water.  The smear was then counter-stained with
Safrinin for one minute before the final rinse and drying.  The stain was visualized in the photo-microscope and a
photo was taken using a mobile device as the microscope itself could not take coloured photos which were necessary
to show the spores as green and the cells as not green.
</P><P class="Center"><B>Figure 8:</B><I>B. subtilis</I> spore stain of regenerated spores.
</P><H3> Transforming <I>B. subtilis</I> with PCR product</H3><P>We are currently working on biobricking lacZ. We are planning on submitting this part to the iGEM registry since the lacZ available in the registry has bad sequencing.</P><H2>Week 21: September 14 - September 20</H2><P>Now that we had sporulated <I>B. subtilis</I> and confirmed the presence of spores using Gram stains and spore stains, we were ready to activate the spores back to the vegetative state. All that is required for this switch to occur is the presence of nutrients, which describes a regular LB-agar plate. We attempted activation by spreading 50 uL of thawed spore solution (from 4 °C and -20 °C) on plain agar plates. Colonies successfully grew on the plates and based on a qualitative analysis, more colonies grew for the long-term storage spore solution than short-term storage spore solutions. Although we had colonies growing, we needed to confirm whether spores actually converted to vegetative cells. 


</P><P><B>Figure 9:</B><I>B. subtilis</I> spore regeneration on plain LB-agar plates. The plate on the left was spread with spore solution stored at 4°C for short-term storage. The plate on the right was spread with spore solution stored at -20°C for long-term storage. Based on colony growth, more cells were viable under long-term storage conditions.</P><H3>Post sporulation and regeneration staining</H3><P>
    Israt had plated regenerated <I>B. subtilis</I> on LB agar plates, which were incubated overnight at 37 ° 
C so that staining could be done on them to check to make sure the bacteria looked the same when analyzed as 
before they were sporulated.  Six slides were made using Israt's plates, four for Gram staining and two for spore 
staining.</P><OL>Plates:
	<LI><I>B. subtilis</I> Spore. September 18, 2014</LI><LI><I>B. subtilis</I> Spore+xylose. September 16, 2014</LI></OL><P>
    From these plates two slides for each was made for Gram staining, and then two slides from Plate 1 for spore 
staining.  The same Gram and spore staining procedures, as above, was used for the slides, the only modification 
made to the staining procedure was that the counter-stain Safrinin was left on for one and half minutes instead 
of just one minute as it was noticed that the Safrinin was too light in colour in the previous staining slides.
</P><P>
    From the naked eye the slides appeared to be successful, the Gram stained slides were all purple and the spore 
stain slides were completely pink, but they will be analyzed at a later date under a microscope.
</P><H3> Transforming <I>B. subtilis</I> with PCR product</H3><P>Unfortunately, we are having troubles biobricking lacZ. In the meantime we transformed <I>B. subtilis</I> again using primer product and got similar results.</P><H2>Week 22: September 21 - September 27</H2><H3>Post sporulation and regeneration imaging</H3><P>
    Michael Wilton, University of Calgary Health Sciences Campus, allowed us to use the microscope in his lab to 
image the slide stains for the regenerated <I>B. subtilis</I> spores.  All of the slides gave desired results:
</P><H3>Gram stain:</H3><P>
    All slides gave a purple colour, showing that the bacteria still behaves as a Gram-positive bacteria after 
sporulation and regeneration.
</P><P class="Center"><B>Figure 10:</B><I>B. subtilis</I> post-sporulation Gram-stain.
</P><H3>Spore stain:</H3><P>
    All slides showed vastly more pink cells compared to green spores.  This is good because it shows that the 
regeneration of spores has a high efficiency even when the spores are only grown on LB at 37 °C overnight.
</P><P class="Center"><B>Figure 11:</B><I>B. subtilis</I> post-sporulation spore stain.
</P><H3>Treatment of spores</H3><P>
    We wanted to test if the spores could still be regenerated and generate viable bacteria after the spores were
subject to harsh conditions.  To do this, a collection of short term storage of spores (kept in distilled water
at 4 ° C), and a collection of long term storage (kept in distilled water and frozen) were separated into
five 0.5 mL tubes, one for each condition.  Each of these tubes was then centrifuged, for five minutes at 14 000 rpm, 
so that the distilled water that the spores were suspended in could be removed before the spores were subjected to 
each condition.  The chosen conditions were designed to subject the spores to a wide range of conditions, some of 
which were chosen to simulate some of the conditions that the diagnostic tool might be subject to, such as excessive
heat, while others were chosen to see what exactly the pores were able to survive through, such as chemical conditions.  
The five chosen conditions were; 1% bleach, 70% ethanol, complete drying, heating at 60 ° C, and heating
at 90 ° C.</P><UL><LI>1% bleach: Centrifuged spores re-suspended in 50 μL 1% bleach.</LI><LI>70% ethanol: Centrifuged spores re-suspended in 50 μL 70% ethanol.</LI><LI>Drying: Centrifuged spores were completely dried in a Vacufuge at 30 °C for 10 minutes.</LI><LI>Heating at 60 °C: Centrifuged spores were placed in a hot water bath at 60 °C.</LI><LI>Heating at 90 °C: Centrifuged spores were placed on a hot plate at 90 °C.</LI></UL><P>
    The spores were subjected to each of these conditions, at 5:35 pm on September 25, 2014, and left for 24 hours before 
they were re-suspended in 50 μL of distilled water.  From this solution the spores were centrifuged to separate spores
from the supernatant so that the spores could be spread on LB agar overnight to see if the spores were able to regenerate
and cultures were able to grow.  This allowed for quantitative data to be taken of the cultures to see which conditions
proved to be the hardest on the spores.
</P><H3> Transforming <I>B. subtilis</I> spores</H3><P>The final design of our device would contain <I>B. subtilis</I> in the spore form due to its robust nature. Before the blood sample reaches the compartments containing <I>B. subtilis</I>, the cells need to be activated back to the vegetative state. Due to this, it was essential to test activation and transformation directly from spores. We made our first attempt by pelleting cells from long-term and short-term stored spore solutions and added LB, xylose, and linear <I>thrC</I> DNA (~200 ng) at the same time. We incubated these tubes for a day, and spread them on plain and spectinomycin plates. There was growth on plain plates but not spectinomycin plates, suggesting unsuccessful transformation. One reason for failed transformation might be the addition of insufficient DNA.</P><H3> Transforming <I>B. subtilis</I> with PCR product</H3><P>We continued trying to biobrick lacZ. This week the team focused on getting parts ready for submission. We are having issues plasmid switching out parts.</P><H3>Sporulation with xylose</H3><P>
    It was decided that sporulation of <I>B. subtilis</I> should be done but with using xylose during different times
during the sporulation process to test if and when spores are be able to retain competence after activation. The initial culture and addition of xylose (to end up with 1% xylose in culture) to <I>B. subtilis</I> was done by Israt and I added xylose when the culture was added to the SGx2 media before being shaken for 2-3 days.  Unfortunately, I miss interpreted Israt's instructions and put xylose in both cultures, and not just in the one that did not already have xylose in it.  Because of this, is was determined that we would re-do the experiment after making a new batch of xylose (Israt) and SGx2 media (Shelby).</P><H2>Week 23: September 29 - October 5</H2><H3>Xylose sporulation and SGx2 media</H3><P>
    SGx2 media was made for the sporulation process. The recipe was originally obtained from Anna Fei, and was modified 
for a smaller volume, as the media tends to get cloudy quickly and then can no longer be used.  The original recipe was
for 500 mL total volume and was reduced to 250 mL for our purposes.</P><H4>Original recipe:</H4><OL><LI>500 mL of distilled water</LI><LI>8 g Difo nutrient broth</LI><LI>1 g KCl</LI><LI>0.5 g MgSO4 • 7H2O</LI><LI>Autoclave above mixture before adding the remaining ingredients</LI><LI>0.5 mL 1 M Ca(NO3)2</LI><LI>0.5 mL 0.1 M MnCl2 • 4H2O</LI><LI>0.5 mL 1 mM FeSO4</LI><LI>1 mL 50% Glucose (filtered)</LI></OL><P>
    This recipe was cut in half and was finished being made on September 30, 2014, the same day that culture and xylose was
added for the 2-3 day shaking process.</P><P>
    A new batch of <I>B. subtilis</I> was cultured and a new batch of 10% xylose were made by Israt in preparation for the 
xylose sporulation of <I>B. subtilis</I>.  She also added xylose (final concentration of 1%) to one of the cultures before shaking for 6-8 hours, resulting in a 1% xylose concentration in the culture, and left the other with out xylose so that it could be added when SGx2 media was added.  After being shaken, 0.125mL of culture was added to a volume of 25 mL of SGx2 media, and 2.5 mL of 10% xylose was added to the tube not containing xylose already. This meant that both tubes had a concentration of 1%
xylose in the end.  These were then shaken for 3 days and placed in the fridge, after which the spores were washed and resuspended in distilled water <A href="https://2014.igem.org/Team:Calgary/Notebook/ProtocolManual/Bsubtilis">(refer to B. subtilis protocols)</A>.</P><H3>Spore treatment results:</H3><P>
    There was growth seen for each condition, with exception to the spores being lit on fire in the Bunsen burner which was
expected.  Overall, the long term storage spores, frozen, seemed to better survive the chemical conditions, and the short term, 
fridge, spores seemed to better survive the extreme temperature conditions.  For long term storage, there was 3 large colonies
seen on 1% Bleach plate, 5 large colonies on 70% ethanol, and 11 large colonies on drying.  Whereas, for short term storage,
there was 2 large and 1 small colonies on 70% ethanol, 1 medium colony on 60 ° C, and 1 medium colony on 90 °
C plate.  It was later decided that the long term storage spores should be plated by themselves, without them being
subject to any other conditions.  This resulted in a full lawn of large colonies after being incubated at 37 ° C
for 72 hours.  From this it appears that long term storage is a better method for preserving the <I>B. subtilis</I> spores
than short term storage for the purposes of our diagnostic device.</P><P class="Center"><B>Figure 12:</B><I>B. subtilis</I> post-sporulation colonies 
grown on LB media after being subjected to being refrigerated and 
then submersed in 70% ethanol for 24 hours.
</P><P class="Center"><B>Figure 13:</B><I>B. subtilis</I> post-sporulation colonies 
grown on LB media after being subjected to being refrigerated and 
then submersed in 60 ° C for 24 hours.
</P><P class="Center"><B>Figure 14:</B><I>B. subtilis</I> post-sporulation colonies 
grown on LB media after being subjected to being refrigerated and 
then submersed in 100 ° C for 24 hours.
</P><P class="Center"><B>Figure 15:</B><I>B. subtilis</I> post-sporulation colonies 
grown on LB media after being subjected to being frozen and 
then submersed in 70% ethanol for 24 hours.
</P><P class="Center"><B>Figure 16:</B><I>B. subtilis</I> post-sporulation colonies 
grown on LB media after being subjected to being frozen and 
then being completely dried for 24 hours.
</P><P class="Center"><B>Figure 17:</B><I>B. subtilis</I> post-sporulation colonies 
grown on LB media after being subjected to being frozen and 
then submersed in 1% bleach for 24 hours.
</P><H2>Week 24: October 6 - October 12</H2><H3>Optimization of transformation in small liquid volume</H3><P>
    We manipulated various conditions in the original transformation protocol, but it was also useful to streamline transformation in small liquid volumes, as that may more closely approximate the transformation apparatus in our final device. Competent <I>B. subtilis</I> are prepped the same way, but transformation took place in aliquots of 100 μL liquid medium. The results were quantified by either plating and counting the number of colonies that grew on selective media, or by taking absorbance measurements against a standard.</P><P class="Center"><B>Figure 18: Colony growth of transformed <I>B. subtilis</I> at 30°C and 37°C for different xylose concentrations.</B><I>B. subtilis</I> SCK6 pAX01-comK cells transformed with linearized plasmid DNA and incubated at 30°C and 37°C with different amounts of xylose, followed by growth on selective media. Colony counts were higher for all concentrations of xylose when grown at 37°C. The conditions resulting in highest growth were an incubation temperature of 37°C and a xylose concentration of 2%.</P><P class="Center"><B>Figure 19: Absorbance readings of small volume <I>B. subtilus</I> in liquid volume for different xylose concentrations.</B><I>B. subtilis</I> SCK6 pAX01-comK cells in small liquid volume transformed with linearized plasmid DNA and incubated at 37°C with different amounts of xylose, followed by growth and addition of selection marker (antibiotic). All transformation occurred in liquid. The negative control comprised of untransformed cells (killed by antibiotic), while for positive control, antibiotic was not added to untransformed cells. There was a slight increase as xylose concentration increased.</P><H3>&quot;Biobricking&quot; of the <I>comK</I> gene:</H3><P>
    In preparation for the parts submission deadline, the <I>comK</I> gene contained in the genome of <I>B. subtilis</I> was PCRed and placed into a BioBrick vector. First, primers specially designed to anneal to the PxylA+RBS+comK region within <I>B. subtilis</I> were used to amplify and isolate the desired gene. This took multiple tries, but was eventually completed. It was discovered that the amplicon required an extension time of 3:00 minutes at 72 ° C in order to be successfully amplified, an unusually long period of time for a sequence of only ~840 base pairs. The PCR products were run on a 1% agarose gel and the gene was extracted using the EZNA Gel Extraction Kit by Promega. As mentioned previously, the objective was to place <I>comK</I>, including its xylose inducible promoter and RBS, into a Biobrick vector. This was made possible by excising the RFP gene from an RFP Generator Biobrick plasmid and replacing it with the <I>comK</I> gene. Both the vector and insert were digested with XbaI and PstI, and the product was cloned within Top10 <I>E. coli</I>. Our secondary objective was to remove the EcoI and SpeI cut sites contained within the open reading frame of the <I>comK</I> gene. Unfortunately, this could not be completed before the deadline.

</P><P>
    However, we have successfully biobricked and sequenced the <I>lacZ</I> gene. It is on its way to Boston!</P><H3>Desiccation and heat conditioning of <I>B. subtilis</I> spores</H3><P>
    It was decided that the long term, freezing, and short term, refrigerating, storage 
<I>B. subtilis</I> spores should be subjected to complete desiccation, drying, and then 
subjected to heating.  These heating conditions were designed to imitate conditions that 
the spores might be subjected to while in the dignostic device.  Two temperatures were 
chosen, 40 °C and 60 °C.  Because the device would be in very hot developing
countries for what could be years hot, but still environmentally plausible, temperatures
were tested.  As well, long and short term storage spores of spores that were made in
the presence of xylose, either with xylose added when bacteria was added or xylose added 
when SG x2 was added, to test if the desiccation and temperature treatments had any adverse 
effects on spore regeneration.</P><P>
    The same method of desiccation as above was used and then the spore were
either placed on a hot plate, for 40 °C, or in a water bath, for 60 °C, for 24 
hours.  After the 24 hours the desiccated spores were resuspended in distilled water, 
100 uL, before 50 uL was pipetted onto LB media petri dishes to allow for growth for 24 
hours.  These plates were placed in a 37 °C incubator and allowed to grow for the 
full 24 hours.  Lastly, these plates were removed and analysed for <I>B. subtilis</I>
growth.</P><P>
    There was either an excess of colonies seen on the plates, or no growth at all, as seen 
in only two plates, long term storage spores at both temperature treatments.  The excess growth
seen was likely due to the highly concentrated resuspended mixture of distilled water and spores
that was used for plating, or it could be due to the presence of xylose during sporulation.  
The most growth was seen under long term storage where xylose was added when bacteria was added 
during sporulation.  The remaining treatments had similar growth seen.  This may mean that 
providing xylose to the bacteria before sporulation and storage may enable the bacteria to be 
able to survive the storage treatment and regenerate better.</P><P class="Center"><B>Figure 20:</B> Colonies on LB media for short term storage <I>B. subtilis</I> spores which were then desiccated before being 
subjected to a water bath of 40 °C for 24 hours.
</P><P class="Center"><B>Figure 21:</B> Colonies on LB media for short term storage <I>B. subtilis</I> spores which were then desiccated before being 
subjected to a water bath of 60 °C for 24 hours.
</P><P class="Center"><B>Figure 22:</B> Colonies on LB media for short term storage <I>B. subtilis</I> spores, sporulated with xylose, which was added 
when 2xSG media was added.  These spores were then desiccated before being subjected to a water bath of 40 °C for 24 hours.
</P><P class="Center"><B>Figure 23:</B> Colonies on LB media for short term storage <I>B. subtilis</I> spores, sporulated with xylose, which was added 
when 2xSG media was added.  These spores were then desiccated before being subjected to a water bath of 60 °C for 24 hours.
</P><P class="Center"><B>Figure 24:</B> No colonies on LB media for long term storage <I>B. subtilis</I> spores which were then desiccated before being 
subjected to a water bath of 40 °C for 24 hours.
</P><P class="Center"><B>Figure 25:</B> No colonies on LB media for long term storage <I>B. subtilis</I> spores which were then desiccated before being 
subjected to a water bath of 60 °C for 24 hours.
</P><P class="Center"><B>Figure 26:</B> Colonies on LB media for long term storage <I>B. subtilis</I> spores, sporulated with xylose, which was added 
when 2xSG media was added.  These spores were then desiccated before being subjected to a water bath of 40 °C for 24 hours.
</P><P class="Center"><B>Figure 27:</B> Colonies on LB media for long term storage <I>B. subtilis</I> spores, sporulated with xylose, which was added 
when 2xSG media was added.  These spores were then desiccated before being subjected to a water bath of 60 °C for 24 hours.
</P><P class="Center"><B>Figure 28:</B> Colonies on LB media for long term storage <I>B. subtilis</I> spores, sporulated with xylose, which was added 
when the bacteria was added.  These spores were then desiccated before being subjected to a water bath of 40 °C for 24 hours.
</P><P class="Center"><B>Figure 29:</B> Colonies on LB media for long term storage <I>B. subtilis</I> spores, sporulated with xylose, which was added 
when the bacteria was added.  These spores were then desiccated before being subjected to a water bath of 60 °C for 24 hours.
</P><H3>Transforming with <I>B. subtilis</I> spores</H3><P>Now that we had stored spores with xylose added in the first inoculation step and xylose added with the SG media (different time points during the sporulation protocol), we were ready to activate and transform them mimicking device conditions. We pelleted the spores and put them in a vacufuge to ensure they were dry since the spores in the device will be in the dried form. For the final device, we were planning on activating spores with water so we added water to the experimental spores after they were out of the vacufuge. Then we added 500 ng of linear <I>thrC</I> DNA (since 200 ng was insufficient) and left the tubes out at room temperature for a day and then spread them on spectinomycin plates. Colonies grew for both conditions (xylose added to bacteria in initial inoculation, xylose added to SG media) and this suggested successful transformation. However, the next day the colonies appeared fuzzy and resembled fungal growth, so we were doubtful whether we were able to transform spores. 

</P><P class="Center"><B>Figure 30:</B>Transformation of spores that had xylose added to the sporulation SG media. It appears that transformation with spores was successful, but we are doubtful if these are <I>B. subtilis</I> cells or fungal growth since the cells were fuzzy the next day.</P><P class="Center"><B>Figure 31:</B>Transformation of spores that had xylose added to the bacteria during the initial inoculation. It appears that transformation with spores was successful, but we are doubtful if these are <I>B. subtilis</I> cells or fungal growth since the cells were fuzzy the next day.</P>


We also left eight more tubes of dried spores (without the addition of water and DNA) and left these at room temperature to ensure they were as dry as possible.


<H2>Week 25: October 13 - October 17</H2><H3>Transforming with <I>B. subtilis</I> spores</H3><P>Since the previous trial of spore transformation was doubtful due to the likely presence of fungal growth, we attempted transformation again. This time, we tried transforming with spores that were left to dry for several days and spread them on plain agar and spectinomycin plates. We left the plates at room temperature and saw no growth the next day. We then placed them in the 37°C incubator and the next day we saw growth on the plain plates and limited growth on the spectinomycin plates. Since the plates for the previous trial were checked after several days, we think more time might be required for the colonies to grow on the spectinomycin plates. As of now though, transformation with spores was unsuccessful.</P><P class="Center"><B>Figure 32:</B>Transformation of dried spores that had xylose added to the bacteria during the initial inoculation. Colonies grew on the plain agar plate (left), while there was no growth on the spectinomycin plate (right). More time might be required for colonies to grow on the spectinomycin plate but as of now, transformation of spores was unsuccessful.</P><P class="Center"><B>Figure 33:</B>Transformation of dried spores that had xylose added to the sporulation SG media during the initial inoculation. Colonies grew on the plain agar plate (left), while there was no growth on the spectinomycin plate (right). More time might be required for colonies to grow on the spectinomycin plate but as of now, transformation of spores was unsuccessful.</P><P>We also tested if vegetative <I>B. subtilis</I> cells survive the long-term storage conditions (-20°C) for spores and if they can be transformed. They grew on plain agar plates but not spectinomycin plates, which was expected since the cells were not induced with xylose and thus incompetent. In addition, we tested if incompetent <I>B. subtilis</I> spores could be transformed and no colonies grew on the spectinomycin plates. This was expected since the cells were not induced with xylose.</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Calgary/Notebook/Journal/Transformers">http://2014.igem.org/Team:Calgary/Notebook/Journal/Transformers</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Calgary/Notebook/Journal/Transformers" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Calgary/Notebook/Journal/Transformers" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Calgary/Notebook/Journal/Transformers&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Calgary/Notebook/Journal/Transformers&amp;oldid=397549" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>