<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Calgary_Project_BsDetector_BsChassis"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Calgary/Project/BsDetector/BsChassis">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Calgary/Project/BsDetector/BsChassis&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Project/BsDetector/BsChassis&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Project/BsDetector/BsChassis&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Calgary/Project/BsDetector/BsChassis" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Calgary/Project/BsDetector/BsChassis</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="contentSub">(Redirected from <A href="/wiki/index.php?title=Team:Calgary/Sandbox/Project/BsDetector/BsChassis&amp;redirect=no" title="Team:Calgary/Sandbox/Project/BsDetector/BsChassis">Team:Calgary/Sandbox/Project/BsDetector/BsChassis</A>)</DIV><P><SECTION id="Page"><HEADER id="TopBar"><UL id="TopLvlNav"><LI><A href="https://2014.igem.org/Team:Calgary/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Project/BsDetector"><I>B.s.</I> Detector</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Project/BsDetector/TargetDiseases">Target Diseases</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Project/BsDetector/SamplePreparation">Sample Preparation</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Project/BsDetector/BsChassis"><I>B.s.</I> Chassis</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Project/BsDetector/GeneticCircuit">Genetic Circuit</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Project/BsDetector/ModellingAndOptimization">Modelling &amp; Optimization</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Project/BsDetector/DevicePrototype">Device Prototype</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Project/BsDetector/Platform">Platform</A></LI></UL><LI><A href="https://2014.igem.org/Team:Calgary/Project/Collaboration">Collaboration</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Project/Achievements">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Project/FinalSystem">Final System</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/OurTeam">Our Team</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/OurTeam/TeamMembers">Team Members</A></LI><LI><A href="https://igem.org/Team.cgi?id=1444">Official Profile</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/OurTeam/Outreach">Outreach</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/OurTeam/Outreach/TelusSpark">Telus Spark</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/OurTeam/Outreach/Beakernight">Beakernight</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/OurTeam/TheUniversity">About the University</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/OurTeam/ContactUs">Contact Us</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/OurTeam/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/OurTeam/Sponsors">Sponsors</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/Journal">Journal</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/Journal/Detectives">Detectives</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/Journal/Transformers">Transformers</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/Journal/ModellingAndPrototype">Modelling &amp; Prototype</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/Journal/PoliciesAndPractices">Policies &amp; Practices</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/ProtocolManual">Protocol Manual</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/ProtocolManual/General">General</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/ProtocolManual/DNA">DNA</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/ProtocolManual/Bsubtilis"><I>B. subtilis</I></A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/ProtocolManual/SamplePreparation">Sample Preparation</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/Notebook/References">References</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/PoliciesAndPractices">Policies</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/PoliciesAndPractices/DiagnosticLandscape">Diagnostic Landscape</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/PoliciesAndPractices/DiagnosticLandscape/Implementation">Implementation</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/PoliciesAndPractices/DiagnosticLandscape/InformedDesign">Informed Design</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/PoliciesAndPractices/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/PoliciesAndPractices/BiologicalWeaponsConvention">Biological Weapons Convention</A></LI><LI><A href="https://2014.igem.org/Team:Calgary/PoliciesAndPractices/FIND">FIND</A></LI><LI><A href="https://2014.igem.org/Main_Page">iGEM</A></LI><LI><A href="https://2013.igem.org/Team:Calgary" target="_blank">Calgary 2013</A></LI><LI><A href="https://2012.igem.org/Team:Calgary" target="_blank">Calgary 2012</A></LI><LI><A href="https://2011.igem.org/Team:Calgary" target="_blank">Calgary 2011</A></LI></HEADER></SECTION></P><H1><I>B.s.</I> Chassis</H1><H2>Why <I>Bacillus subtilis</I>?</H2><P><I>Bacillus subtilis</I> is capable of <B>natural transformation</B>. Under conditions of nutrient deprivation (energy starvation), <I>B. subtilis</I> efficiently uptakes foreign DNA. We can also bypass the need for energy starvation to make transformation even easier! By using cells derived from a specific strain of <I>B. subtilis</I> (SCK6) with the pAX01-comK plasmid (Zhang &amp; Zhang, 2010), we can use xylose to induce competence and transform <I>B. subtilis</I> with our DNA constructs. Read more about this transformation process in our <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/ProtocolManual/Bsubtilis">protocol manual for <I>B. subtilis</I></A>.</P><P>In addition, it is easy to induce <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/ProtocolManual/Bsubtilis">sporulation</A> in <I>B. subtilis</I>. This leads to the generation of robust spores that can be activated back to their normal state when needed.</P><H2>Transformation</H2><P>Although <I>B. subtilis</I> is naturally transformable, the traditional transformation process via starvation is difficult to control, time-consuming and labour-intensive. Fortunately, we can bypass the need for energy starvation by using cells derived from a specific strain of <I>B. subtilis</I> (SCK6) with the pAX01-comK plasmid constructed by Zhang &amp; Zhang (2010).</P><P>The key feature conferred by this plasmid is the overexpression of <A href="https://2014.igem.org/Team:Calgary/Notebook/Parts"><I>comK</I></A>, which is the master regulator of competence in <I>B. subtilis</I>. It is known as such because it encodes a transcription factor which upregulates the expression of these competence genes (<I>comC, comE, comF, comG, nucA</I>). All these genes contribute to the DNA uptake mechanism in <I>B. subtilis</I>.</P><P><CENTER><B>Figure 1:</B> Diagram of DNA uptake within <I>B. subtilis</I> (Maier <I>et al.</I>, 2004). Five genes are involved in the mechanism of competency seen in <I>B. subtilis</I>; <I>comC</I>, <I>comE</I>, <I>comF</I>, <I>comG</I> and <I>nucA</I><A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">(Hamoen, Venema, Kuipers &amp; 2003)</A>. <I>comG</I> encodes several proteins which assemble to form a pilus structure. <I>comC</I> facilitates the correct assembly of this structure. <I>comE</I> encodes for a polytopic transmembrane protein (ComEC), which forms a pore for DNA entry. dsDNA is unwound and converted into ssDNA upon entry by a protein (resembling DNA-helicase) encoded by <I>comF</I>. ssDNA is eventually integrated into the genome through the function of the general DNA-recombination factor RecA and the DNA-helicase AddAB.</CENTER></P><P>In our “supercompetent” plasmid constructed by Zhang &amp; Zhang (2010), the control of the master regulator <I>comK</I> is placed under the xylose-inducible promoter <I>P<SUB>xylA</SUB></I>. This means that if we add xylose to the cells, they will activate <I>P<SUB>xylA</SUB></I> and subsequently <I>comK</I>, resulting in the “turning on” of competence genes. The cells can then uptake foreign DNA without the need for energy starvation. The figure below depicts the pAX01-comK plasmid (Zhang &amp; Zhang, 2010).</P><P><CENTER><B>Figure 2:</B> pAX01-comK vector map (Zhang &amp; Zhang, 2010). Constructed plasmid for <I>B. subtilis</I> wherein the master regulator <I>comK</I> competence gene is placed under the control of <I>P<SUB>xylA</SUB></I>, a xylose-inducible promoter.</CENTER></P><P>The <A href="”https://2014.igem.org/Team:Calgary/Sandbox/Notebook/ProtocolManual/Bsubtilis”">transformation protocol utilized by Zhang &amp; Zhang (2010)</A> with the pAX01-comK plasmid in SCK6 included the addition of 1% xylose in LB to an overnight culture of SCK6 cells. The amount of xylose added was dictated by OD values. After taking an initial reading at OD<SUB>600</SUB>, the 1% xylose in LB solution was added until the reading was 1.0; the culture was then grown for 2 hours to develop competence. This means that the final xylose concentration of cultures was not fixed, as it depended on the cell growth that occurred after the overnight incubation. Thus, we were curious as to the optimal incubation conditions (temperature, degree of shaking), and xylose concentration. To investigate the latter, we <A href="”https://2014.igem.org/Team:Calgary/Sandbox/Notebook/ProtocolManual/Bsubtilis”">modified the protocol</A> by diluting with only LB to obtain the desired OD reading. We then added the appropriate volume of a stock 20% xylose solution to obtain desired final xylose concentrations. The results of manipulating xylose concentration at different temperatures are shown below in Figure 2.</P><P><CENTER><B>Figure 3:</B> Colony growth of transformed <I>B. subtilis</I> at 30°C and 37°C for different xylose concentrations. <I>Bacillus subtilis</I> SCK6 pAX01-comK cells transformed with linearized plasmid DNA and incubated at 30°C and 37°C with different concentrations of xylose, followed by growth on selective media. Colony counts were higher for all concentrations of xylose when grown at 37°C. The conditions resulting in highest growth were an incubation temperature of 37°C and a xylose concentration of 2%.</CENTER></P><P>We further optimized incubation conditions by investigating whether or not shaking had an adverse effect on colony growth. As previously described above in Figure 1, DNA uptake in <I>B. subtilis</I> involves a set of machinery working together with the pilus to allow foreign DNA to cross the membrane. We postulated that excessive shaking can lead to disturbance of this uptake process. Our findings are presented below.

</P><P><CENTER><B>Figure 4:</B> The effect of shaking on colony growth. Following addition of DNA to competent <I>B. subtilis</I> cells, the tubes were incubated at 37°C either without shaking (left) or shaken at 200 rpm (right). The lack of shaking resulted in greater colony growth on selective plates. Given that no shaking results in more transformed colonies, we implemented this change in protocol.</CENTER></P><P>Besides the optimization of incubation conditions outlined above, we also spent time investigating how the entire process carries over in small liquid volume form. This is because the “small, liquid volume” model will most closely approximate the transformation process occurring within our physical <I>B. s.</I> detection device. We quantified results of optimization via OD readings on an ELISA plate reader. These results are quantified below in Figure 7.</P><P><CENTER><B>Figure 5:</B> OD readings of small volume <I>B. subtilus</I> in liquid volume for different xylose concentrations. <I>B. subtilis</I> SCK6 pAX01-comK cells in small liquid volume transformed with linearized plasmid DNA and incubated at 37°C with different amounts of xylose, followed by growth and addition of selection marker (antibiotic). All transformation occurred in liquid. The negative control comprised of untransformed cells (killed by antibiotic), while for positive control, antibiotic was not added to untransformed cells. There was a slight increase as xylose concentration increased.</CENTER></P><P>In addition to manipulating incubation conditions for the optimization of the transformation protocol, we generated a “standard curve” of changes in OD readings corresponding to different volumes of LB added to a given volume of <I>B. subtilis</I> overnight culture. The graph is shown below in Figure 5.</P><P><CENTER><B>Figure 6:</B> Decrease in OD reading at 600 nm for different volumes of LB added to overnight <I>B. subtilis</I> cultures. We added different volumes of LB to 500 μL of <I>B. subtilis</I> overnight culture and graphed the subsequent decrease in OD<SUB>600</SUB>.</CENTER></P><P>Using Figure 6 means that there is no need to take an OD reading at every addition of a minute volume of LB. Instead, we can simply take an initial OD reading, note the difference to the desired reading of 1.0, and find the corresponding volume of LB that needs to be added to obtain this desired reading.</P><P>We thus optimized the transformation protocol developed by Zhang &amp; Zhang (2010) by changing incubation conditions. We also developed tools for making transformation more convenient to carry out. Our <A href="”https://2014.igem.org/Team:Calgary/Sandbox/Notebook/ProtocolManual/Bsubtilis”">modified protocol</A> was used in order to transform <I>B. subtilis</I> more easily and obtain greater numbers of transformed colonies.</P><P>In addition to optimization of the <I>B. subtilis</I> transformation process, we also investigated the length of DNA required. We began by studying existing literature, and found that 70 bp is the minimum size for successful uptake. There is a linear relationship between substrate length and transformation efficiency <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">(Khasanov et al, 1992)</A>. We were interested in finding out the required DNA length because it needed to be compatible with our isothermal PCR process.</P><P>We designed specific primers spanning the <I>thrC</I> region. These primers amplified the spec and RFP casettes with varying lengths of homologous sequence. The smallest length resulting in successful transformation with the desired product was 125 bp.</P><P><CENTER><B>Figure 7:</B><I>B. subtilis</I> was transformed with DNA containing various lengths of homologous sequence and was plated on selective spectinomycin plates. Plate A shows <I>B. subtilis</I> colonies transformed with digested <I>thrC</I> vector (700, 1300 bp of homology). Plate B, C and D show <I>B. subtilis</I> transformed with PCR product 2,3 and 4 respectively (215 bp, 190 bp and 125 bp of homology).</CENTER></P><P>These experiments were necessary to carry out, because the results enabled us to optimize isoPCR as well. That is, determining the proper length of uptake DNA informs the timing and duration of the amplification step.</P><H2>Sporulation</H2><P>One compelling reason for choosing this organism was its tendency to sporulate, a process by which bacteria enter a dormant state in order to survive adverse conditions such as starvation, irradiation, and heat <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">(Eichenberger, 2012)</A>. <I>B. subtilis</I> has two types of states: vegetative and spore. In the vegetative state, bacteria are able to use nutrients and can actively divide. For the spore state, bacteria are metabolically inactive with the genetic material stored and protected in the spore coat <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">(Eichenberger, 2012)</A>. When bacteria sense insufficient nutrients, they convert from the metabolically active vegetative state to the metabolically inactive spore state <A href="https://2014.igem.org/Team:Calgary/Sandbox/Notebook/References">(Eichenberger, 2012)</A>. Once sufficient nutrients are available, spores switch back to the vegetative state.</P><P>The ability of <I>B. subtilis</I> to form spores under extreme conditions is beneficial for our purposes because the durability of the bacteria facilitates transportation of our diagnostic tool around the world.</P><P>
    Based on the growth of <I>B. subtilis</I> seen on the LB plates after the spores were 
subjected to various extreme conditions showed that we were indeed successful at sporulating 
the bacteria.  This is evident because there would be no bacteria growth seen on LB plates 
after the spores were subjected to the harsh treatments, such as 70% ethanol and 1% bleach, 
if they were not in fact in spore form. 
</P><P class="Center"><B>Figure 8:</B><I>B. subtilis</I> post-sporulation colonies 
grown on LB media after being subjected to being refrigerated and 
then submersed in 70% ethanol for 24 hours.
</P><P class="Center"><B>Figure 9:</B><I>B. subtilis</I> post-sporulation colonies 
grown on LB media after being subjected to being refrigerated and 
then submersed in 60 ° C for 24 hours.
</P><P class="Center"><B>Figure 10:</B><I>B. subtilis</I> post-sporulation colonies 
grown on LB media after being subjected to being refrigerated and 
then submersed in 100 ° C for 24 hours.
</P><P class="Center"><B>Figure 11:</B><I>B. subtilis</I> post-sporulation colonies 
grown on LB media after being subjected to being frozen and 
then being completely dried for 24 hours.
</P><P class="Center"><B>Figure 12:</B><I>B. subtilis</I> post-sporulation colonies 
grown on LB media after being subjected to being frozen and 
then submersed in 1% bleach for 24 hours.
</P><P>
    It was also evident that complete desiccation, as would be seen in the diagnostic device, 
and high temperatures, 40 °C and 60 °C, did not completely kill the <I>B. subtilis</I> spores.  
This means that we were successful in making robust and applicable spores that would be able 
to survive in the device for the desired two year shelf life, and possibly longer.</P><P class="Center"><B>Figure 13:</B><I>B. subtilis</I> post-sporulation colonies 
grown on LB media after being subjected to being refrigerated and 
then submersed in 70% ethanol for 24 hours.
</P><P class="Center"><B>Figure 14:</B><I>B. subtilis</I> post-sporulation colonies 
grown on LB media after being subjected to being refrigerated and 
then submersed in 60 ° C for 24 hours.
</P><P>
    As well as proving that the spores we generated were robust we also showed that adding 
xylose when the bacteria is added during sporulation may allow a higher efficiency of 
regeration of the spores into vegetative cells.</P><P class="Center"><B>Figure 15:</B><I>B. subtilis</I> post-sporulation colonies 
grown on LB media after being subjected to being frozen and 
then being completely dried for 24 hours.
</P><P class="Center"><B>Figure 16:</B><I>B. subtilis</I> post-sporulation colonies 
grown on LB media after being subjected to being frozen and 
then submersed in 1% bleach for 24 hours.
</P><P>
    Based on the stains performed on the <I>B. subtilis</I> spores after regeration it was 
evident that we were able to regenerate the spore that were made in the lab.  It was also 
evident that the spores were able to regenerate at a rate that is greater than the amount 
of spores that were not regenerated.</P><P class="Center"><B>Figure 17:</B><I>B. subtilis</I> post-sporulation Gram-stain.
</P><P class="Center"><B>Figure 18:</B><I>B. subtilis</I> post-sporulation spore stain.
</P><H3>Transforming with <I>B. subtilis</I> spores</H3><P>The final design of our device would contain <I>B. subtilis</I> in the spore form due to its robust nature. Before the blood sample reaches the compartments containing <I>B. subtilis</I>, the cells need to be activated back to the vegetative state. Due to this, it was essential to test activation and transformation directly from spores. We made spores and added xylose, which makes the supercompetent <I>B. subtilis</I> cells competent, at different time points during the sporulation process. We added xylose in the first inoculation step, and we also added it with the SG media. After we stored these at 4°C and -20°C, which used for short-term and long-term storage respectively, we pelleted this and added water and 500 ng of linear <I>thrC</I> DNA to the tube. After leaving the tube at room temperature for a day, we spread these solutions on spectinomyin plates since the <I>thrC</I> vector has spectinomycin resistance. Colonies grew for both conditions and this suggested successful transformation. However, the next day the colonies appeared fuzzy and resembled fungal growth, so we were doubtful whether we were able to transform spores.</P><P class="Center"><B>Figure 19:</B>Transformation of spores that had xylose added to the sporulation SG media. It appears that transformation with spores was successful, but we are doubtful if these are <I>B. subtilis</I> cells or fungal growth since the cells were fuzzy the next day.</P><P class="Center"><B>Figure 20:</B>Transformation of spores that had xylose added to the bacteria during the initial inoculation. It appears that transformation with spores was successful, but we are doubtful if these are <I>B. subtilis</I> cells or fungal growth since the cells were fuzzy the next day.</P><P>Since the previous trial of spore transformation was doubtful due to the likely presence of fungal growth, we attempted transformation again. This time, we tried transforming with spores that were left to dry for several days and spread them on plain agar and spectinomycin plates. We left the plates at room temperature and saw no growth the next day. We then placed them in the 37°C incubator and the next day we saw growth on the plain plates and limited growth on the spectinomycin plates. Since the plates for the previous trial were checked after several days, we think more time might be required for the colonies to grow on the spectinomycin plates. As of now though, transformation with spores was unsuccessful.</P><P class="Center"><B>Figure 21:</B>Transformation of dried spores that had xylose added to the bacteria during the initial inoculation. Colonies grew on the plain agar plate (left), while there was no growth on the spectinomycin plate (right). More time might be required for colonies to grow on the spectinomycin plate but as of now, transformation of spores was unsuccessful.</P><P class="Center"><B>Figure 22:</B>Transformation of dried spores that had xylose added to the sporulation SG media during the initial inoculation. Colonies grew on the plain agar plate (left), while there was no growth on the spectinomycin plate (right). More time might be required for colonies to grow on the spectinomycin plate but as of now, transformation of spores was unsuccessful.</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Calgary/Project/BsDetector/BsChassis">http://2014.igem.org/Team:Calgary/Project/BsDetector/BsChassis</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Calgary/Project/BsDetector/BsChassis" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Calgary/Project/BsDetector/BsChassis" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Calgary/Project/BsDetector/BsChassis&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Calgary/Project/BsDetector/BsChassis&amp;oldid=393271" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>