"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Calgary/Project/BsDetector/GeneticCircuit
From 2014.igem.org
(Redirected from Team:Calgary/Sandbox/Project/BsDetector/GeneticCircuit )
Project
B.s. Detector
Target Diseases
Sample Preparation
B.s. Chassis
Genetic Circuit
Modelling & Optimization
Device Prototype
Platform
Collaboration
Achievements
Final System
Our Team
Team Members
Official Profile
Outreach
Telus Spark
Beakernight
About the University
Contact Us
Attributions
Sponsors
Notebook
Journal
Detectives
Transformers
Modelling & Prototype
Policies & Practices
Protocol Manual
General
DNA
B. subtilis
Sample Preparation
Parts
References
Policies
Diagnostic Landscape
Implementation
Informed Design
Safety
Biological Weapons Convention
FIND
iGEM
Calgary 2013
Calgary 2012
Calgary 2011
Genetic Circuit
Our pathogen detection system consists of two interacting genes located within the genome of B. subtilis. The reporter gene consists of a constitutive promoter (Pveg), a repressible promoter, a ribosomal binding site (RBS), and a reporter coding sequence. Our preferred reporter of choice is lacZ . The lacZ gene corresponds to beta-galactosidase, an enzyme catalyzing the breakdown of X-gal to produce indigo (5,5'-dibromo-4,4'-dichloro-indigo), a dark blue pigment. We decided upon an enzymatic reporter instead of a chromophoric protein reporter. The signal of produced by a chromophoric protein is dependent upon, and proportional to the concentration of protein produced; an enzymatic reporter is able to continuously produce a signal that is relatively independent of enzyme concentration allowing a greater signal intensity.
Our genetic circuit consists of a reporter gene regulated by the repressor gene. In the default state, expression of the reporter is inhibited by the repressor bound to the repressible promoter upstream of the reporter. Activation occurs by homologous recombination between the repressor gene and the target DNA sequence. By flanking the repressor gene with sequences homologous to the target DNA, we are able to knock-out the repressor gene and thus allowing expression of our reporter.
Figure 1: DNA detection via homologous recombination. A) Reporter gene consisting of a constitutive promoter (grey arrow), ribosome-binding site (RBS; semicircle), operator (orange), and coding sequence of a reporter (lacZ). Expression of the reporter is repressed in the default state. B) Repressor system for regulation of reporter expression. i) Repressor gene consisting of a repressor driven by a constitutive promoter. Expression of the repressor inhibits the expression of the reporter - the repressor specifically targets the operator shown in a). Blue regions indicate flanking sequences that are homologous to the amplified target DNA as shown in ii). Upon homologous recombination (dotted lines), the disruption of the repressor gene leads to active reporter expression. ii) Target DNA from pathogen, amplified via isothermal-PCR (iPCR). Blue regions indicate sequences homologous to the flanking regions of the repressor gene as shown in i).
A major strength of our project design lies in its high level of customization and modularity. Using our reporter and repressor genes, we can in theory facilitate the detection of any pathogen that might be present within a blood sample simply by modifying the homologous flanking regions of our repressor gene with the proper sequences. To make this possible, we would only need to identify beforehand a target sequence for our pathogen of interest by using a public genome database . Our experiments have shown that a mere 150bp target sequence is sufficient for homologous recombination. In theory, our device should be able to detect any pathogen whose genome has been sequenced to an adequate degree and has been made publicly available.
In keeping with the consistent record of safety present in iGEM, we have also taken cautious measures to ensure that our engineered B. subtilis cells are virtually harmless in the unlikely event that they escape the casing of our device. The reporter and repressor genes have been placed into the thrC locus of B. subtilis and have therefore replaced the genes that were originally present at that locus. Because the thrC gene is needed for the synthesis of threonine - an essential amino acid - replacing it with our genes makes B. subtilis incapable of producing endogenous threonine and thus renders it auxotrophic. Any amount of B. subtilis which may escape our device would be unable to replicate and die promptly.
Retrieved from " http://2014.igem.org/Team:Calgary/Project/BsDetector/GeneticCircuit "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
