<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Production_modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Paris/Production_modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris/Production_modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris/Production_modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris/Production_modeling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris/Production_modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris/Production modeling</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><SPAN id="bottom"><A href="https://2009.igem.org/" class="external text" rel="nofollow">iGEM </A>&gt;<A href="/Team:Paris#top" title="Team:Paris"> Paris</A>&gt;<A href="/Team:Paris/DryLab#bottom" title="Team:Paris/DryLab"> DryLab</A>&gt;<A href="/Team:Paris/Production_modeling#bottom" title="Team:Paris/Production modeling"> Vesicle production Model (delay model)</A></SPAN></P><DIV id="header"><DIV id="header_left"><SPAN class="plainlinks"><A href="https://2009.igem.org/Team:Paris/iLife#top" class="external text" rel="nofollow"> </A></SPAN></DIV><DIV id="header_right"><SPAN class="plainlinks"><A href="https://2009.igem.org/Team:Paris#top" class="external text" rel="nofollow"> </A></SPAN></DIV><DIV id="header_bar_lab"><P><A href="/Team:Paris/Protocols#top" title="Team:Paris/Protocols"> Protocols</A><A href="/Team:Paris/Freezer#top" title="Team:Paris/Freezer"> Freezer</A><A href="/Team:Paris/NoteBook#top" title="Team:Paris/NoteBook"> NoteBook</A><A href="/Team:Paris/Papers#top" title="Team:Paris/Papers"> Bibliography</A><A href="/Team:Paris/Parts#top" title="Team:Paris/Parts"> Parts </A><A href="/Team:Paris/WetLab#top" title="Team:Paris/WetLab"> WetLab </A><A href="/Team:Paris/DryLab#top" title="Team:Paris/DryLab"> DryLab </A></P></DIV><DIV id="header_bar_link"><P><A href="/Team:Paris/Team#top" title="Team:Paris/Team"> Team</A><A href="/Team:Paris/Collaborations#top" title="Team:Paris/Collaborations"> Collaborations</A><A href="/Team:Paris/Contact#top" title="Team:Paris/Contact"> Contacts</A><A href="/Team:Paris/Acknowledgements#top" title="Team:Paris/Acknowledgements"> Acknowledgements</A><A href="/Team:Paris/Links#top" title="Team:Paris/Links"> Links</A></P></DIV></DIV><DIV id="menu2"><P><TITLE>Menu accordéon avec jQuery</TITLE><TABLE cellspacing="0"><TBODY><TR><TD><UL id="navigation"><LI class="menu_header"> </LI><LI class="toggleSubMenu"><SPAN> Home </SPAN></LI><LI><A href="https://2009.igem.org/Team:Paris#top">Synopsis</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Project#top">Project overview</A></LI><LI><A href="https://2009.igem.org/Team:Paris/DryLab#top">Modeling overview</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Conclusion#top">Conclusion &amp; Results</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Brainstorm#top">Brainstorm</A></LI><LI><A href="https://2009.igem.org/Team:Paris/safety#top">Safety</A></LI></UL><LI class="menu_footer"> </LI><LI class="menu_header"> </LI><LI class="toggleSubMenu"><SPAN> Addressing the message </SPAN></LI><LI><A href="https://2009.igem.org/Team:Paris/Addressing_overview#top">To the periplasm</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Addressing_overview2#top">In the outer membrane</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Addressing_testing#top">WetLab</A></LI><LI class="toggleSubMenu2"> Producing the messenger
            </LI><LI><A href="https://2009.igem.org/Team:Paris/Production_overview#top">Vesicle production system</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Production_modeling#top">Vesicle production model</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Production_modeling2#top">Vesicle biophysics model</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Production_testing#top">WetLab</A></LI><LI class="toggleSubMenu"><SPAN> Receiving the message</SPAN></LI><LI><A href="https://2009.igem.org/Team:Paris/Transduction_overview#top">Membrane fusion</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Transduction_overview2#top">Signal Transduction</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Transduction_modeling#top">Fec operon simulation</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Transduction_testing#top">WetLab</A></LI><LI class="menu_footer"> </LI><LI class="menu_header"> </LI><LI class="toggleSubMenu"><SPAN> Ethics </SPAN></LI><LI><A href="https://2009.igem.org/Team:Paris/Ethics_overview#top">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Ethics_ethicallabbook#top">Lab book</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Ethics_ethicalreport#top">Report</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Ethics_Materials#top">Materials</A></LI><LI class="menu_footer"> </LI><LI class="menu_header"> </LI><LI class="toggleSubMenu"><SPAN> Tool, iPhone </SPAN></LI><LI><A href="https://2009.igem.org/Team:Paris/Tool_introduction#top">Introduction</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Tool_DataBase#top ">Databases</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Tool_OSXSoft#top">Mac Software</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Tool_iPhone#top">iPhone</A></LI><LI class="menu_footer"> </LI></TD></TR></TBODY></TABLE></P></DIV><DIV id="paris_content_wrapper"><DIV id="paris_content"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#DryLab_-_Delay_model:_Improving_message_quality"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">DryLab - Delay model: Improving message quality</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Implementing_the_delay_by_a_transcriptional_cascade"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Implementing the delay by a transcriptional cascade</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Implementing_the_delay_by_feed-forward_motifs"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Implementing the delay by feed-forward motifs</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#The_Feed_Forward_Motif"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">The Feed Forward Motif</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Our_design_proposition"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Our design proposition</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#A_more_flexible_design"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">A more flexible design</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#References"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="DryLab_-_Delay_model:_Improving_message_quality"> DryLab - Delay model: Improving message quality</SPAN></H2><DIV id="middle-side"><CENTER><A class="menu_sub" href="https://2009.igem.org/Team:Paris/DryLab#bottom"> Main </A>|
<A class="menu_sub_active" href="https://2009.igem.org/Team:Paris/Production_modeling#bottom"> Delay model</A>|
<A class="menu_sub" href="https://2009.igem.org/Team:Paris/Production_modeling2#bottom"> Vesicle model</A>|
<A class="menu_sub" href="https://2009.igem.org/Team:Paris/Transduction_modeling#bottom"> Fec simulation</A></CENTER></DIV><DIV id="middle-side2"><CENTER><A class="menu_sub_active" href="https://2009.igem.org/Team:Paris/Production_modeling#bottom"> Introduction </A>|
<A class="menu_sub" href="https://2009.igem.org/Team:Paris/Production_modeling#Implementing_the_delay_by_a_transcriptional_cascade"> Transcriptional Cascade</A>|
<A class="menu_sub" href="https://2009.igem.org/Team:Paris/Production_modeling#Implementing_the_delay_by_feed-forward_motifs"> The feed-forward motif</A></CENTER></DIV><P>This part aimed at solving the first problem linked to the creation of messengers :
</P><CENTER><B>To improve the quality of signal sent, how can we get a good synchronization between the maximal vesicles production rate and the maximal concentration of proteins to encapsulate</B></CENTER><P><SPAN id="1"><SPAN id="2">
Obviously, it is crucial to send vesicles only when the concentration of proteins to encapsulate is at its maximum level in the periplasm. To optimize the quality of our message, we tried to make this peak of vesicles creation corresponding to maximum levels of protein concentration. We assumed that the number of vesicles produced is at its maximum when the concentration of TolRII has reached its peak. 
</SPAN></SPAN></P><P>
In what follows, we consider two different designs for implementing the delay: using a transcriptional cascade and using feed-forward motifs.
We found that the <B>design based on transcriptional cascade is sufficient for obtaining the desired delay</B><SUP>[<A href="https://2009.igem.org/Team:Paris/Production_modeling#References" class="external text" rel="nofollow">1</A>]</SUP>, whereas the <B>design based on feed-forward motifs allows for more flexibility, at the cost of an increased complexity</B><SUP>[<A href="https://2009.igem.org/Team:Paris/Production_modeling#References" class="external text" rel="nofollow">2</A>]</SUP>.
These studies are described below.
</P><H3><SPAN class="mw-headline" id="Implementing_the_delay_by_a_transcriptional_cascade">Implementing the delay by a transcriptional cascade</SPAN></H3><P>We have decided to construct the following system. 
The addition of arabinose to the medium leads to the activation of the pBad promoters: two proteins are produced: the protein to encapsulate and LacI-LVA<SUP>[<A href="https://2009.igem.org/Team:Paris/Production_modeling#References" class="external text" rel="nofollow">3</A>]</SUP><SUP>[<A href="https://2009.igem.org/Team:Paris/Production_modeling#References" class="external text" rel="nofollow">4</A>]</SUP>. The latter inhibits the expression of TetR and thus releases the inhibition of TolRII, responsible for vesicle formation. 
</P><P><SPAN id="5">
To study this system, we simply used differential equations. We used the method described by U. Alon in his book <I>An Introduction to Systems Biology</I>, with notably Hill functions to describe the regulation of the promoters<SUP>[<A href="https://2009.igem.org/Team:Paris/Production_modeling#References" class="external text" rel="nofollow">5</A>]</SUP>.
</SPAN></P><P>
The network behavior can be described by a system of differential equations:
</P><CENTER><TABLE><TBODY><TR style="background: #0d3e99; text-align: center; color:white;"><TD width="100%"> Differential System
</TD></TR></TBODY></TABLE></CENTER><DIV style="width: 400px;
text-align: justify;
margin-left:200px;">
These equations describe the evolution of the concentrations of the messenger protein P, LacI repressor, TetR repressor, and the TolRII protein responsible for vesiculation. The gammas are the degradation rates of the respective compounds, while the sigmoidal functions describe the non-linear production rates. The parameters are estimated according to typical values as discussed below</DIV><P>
As a first approximation, we assumed that :
</P><UL><LI>When the pBad promoter is induced, the concentration of arabinose in the medium is very high and constant during he whole study; as a consequence, we will consider that the creation rate of Protein P and LacI* is constant during the experiment:
</LI></UL><UL><LI>We considered that all the binding constants are identical<SUP>[<A href="https://2009.igem.org/Team:Paris/Production_modeling#References" class="external text" rel="nofollow">6</A>]</SUP> and of an average of 40nM which correspond to approximately 40 monomers per cell; we can write that:
</LI></UL><UL><LI>We chose identical intrinsic promoter activities, β, all equal to 4000 proteins/cell cycle:
</LI></UL><UL><LI>All the time units were expressed in units of cell cycle (approximately half an hour). As a consequence we chose a dilution rate γ of 1 for protein without special tags. For the LacI protein with a degradation tag (LacI-LVA), the dilution rate is 3 times higher<SUP>[<A href="https://2009.igem.org/Team:Paris/Production_modeling#References" class="external text" rel="nofollow">3</A>]</SUP>:
</LI></UL><P>
For simulation, we used the following initial quantities:
</P><UL><LI>4000 units of TetR (steady state level at inactivation)
</LI><LI>no protein to be encapsulated
</LI><LI>no LacI
</LI><LI>no TolRII 
</LI></UL><P>
We got the following dynamics using Matlab software, showing that <B>this simple system is capable of generating the desired behavior</B>.
</P><CENTER><P>Plot showing that vesiculation happens only once the protein encapsulated has reached its maximal concentration.
</P></CENTER><H3><SPAN class="mw-headline" id="Implementing_the_delay_by_feed-forward_motifs">Implementing the delay by feed-forward motifs</SPAN></H3><P>Using the previous design, we were able to find a way to get the maximal vesicle formation (ie maximal TolRII quantity) when the concentration of proteins to encapsulate is at its maximum. This system satisfies our requirements. 
However, it is not very flexible. In particular, the concentration of the protein to encapsulate remains always high after induction. This could be detrimental for the cell if this protein is toxic. Consequently, we <B>considered an alternative design allowing for more flexibility</B>, in which the expression of the protein to encapsulate can be <B>transient</B>.
</P><H4><SPAN class="mw-headline" id="The_Feed_Forward_Motif">The Feed Forward Motif</SPAN></H4><P>The feed forward motif was studied and described by U. Alon in his book &quot;An Introduction to Synthetic Biology&quot; where he describes the different forms of existing feed forward; they can be either coherent or incoherent<SUP>[<A href="https://2009.igem.org/Team:Paris/Production_modeling#References" class="external text" rel="nofollow">7</A>]</SUP>.
</P><P>In each case, there exist a different network organization to create a feed-forward loop as shown on the picture
</P><P>
The X and Y inputs can be combined either using a logical AND or OR gate thus giving different dynamics properties in each case.
</P><P>
The coherent feed forward loop allow the creation of a delay either on activation or on deactivation depending on the logical function between the two input X and Y.
When the output logical gate is an AND function, the delay appears on activation, while when the output gate is an OR function, the delay appears on deactivation.
An example of characterized feed-forward loop is the fliA system which was used by the <A href="https://2008.igem.org/Team:Paris|" class="external text" rel="nofollow">2008 Paris team</A> in order to create a oscillator.
</P><P>
In an incoherent feed forward loop, the two inputs of the last gate function have two different logical levels; one molecule (molecule X for example) is an activator while the other molecule is a repressor of the promoter controlling the expression of the output.
This way, when the input signal is on, the expression of molecule X starts activating both the promoters controlling the expression of Y and Z ; transcription starts and the amount of Y and Z molecules increases. Nevertheless, when the concentration of Y has reached its repression threshold, repression starts and the concentration of Z molecule decrease to its steady state. 
</P><P>Two elementary incoherent feed forward motifs can be combined in order to create two time delayed bursts as described by U.Alon in its description of the development of <I>Bacillus subtilis</I> spore. Anyway, it seems quite difficult to find a molecule which would be both an activator and a repressor; we decided to try and think about another combination of these two feed forwards so as to get the dynamics we wanted as shown on the following part.
</P><H4><SPAN class="mw-headline" id="Our_design_proposition">Our design proposition</SPAN></H4><P>The two feed-forward loops used are incoherent and organized in a way that each molecule is either an activator or a repressor (and not both). The following scheme gives the functioning of the system.
</P><TABLE><TBODY><TR style="background: #0d3e99; text-align: center; color:white;"><TD width="10%"> Steps
</TD><TD width="70%"> Description
</TD><TD width="20%"> Changes in concentration
</TD></TR><TR style="background: #bebebe; text-align: center; color:black;"><TD>1
</TD><TD>When the input signal is ON, molecule1 is created and activates the promoter upstream the coding sequence for molecules Z2 and Z3 as well as Z4 since there is no molecule Z3 in the medium (see the AND gate function) at the beginning  ; there is an initial amount of Z5 molecule but no Z2 so the no Z6 molecule is produced.
</TD><TD>Z1 +++
<P>Z2+  
Z3+  
Z4++  
Z5~  
Z6~  
</P></TD></TR><TR style="background: #d8d8d8; text-align: center; color:black;"><TD>2
</TD><TD>After a while, there is enough Z3 to start the repression of Z4 promoter and there is also enough Z2 to start producing Z6 signficantly.
</TD><TD>Z1+++
<P>Z2+++  
Z3+++  
Z4-  
Z5~  
Z6+  
</P></TD></TR><TR style="background: #bebebe; text-align: center; color:black;"><TD>3
</TD><TD>Then, because of the repression of Z3, the Z4 molecule reaches its steady level ; in the same time, Z3 starts repressing the production of Z5 and the amount of Z6 starts decreasing.
</TD><TD>Z1+
<P>Z2+  
Z3+  
Z4~  
Z5-  
Z6+  
</P></TD></TR><TR style="background: #d8d8d8; text-align: center; color:black;"><TD>4
</TD><TD>Finally, Z3 represses the promoter allowing the creation of Z5 and every molecules reaches its steady level concentration.
</TD><TD>Z1~
<P>Z2~  
Z3~  
Z4~  
Z5-  
Z6~  
</P></TD></TR></TBODY></TABLE><P>To study the dynamics of our model, we decided to use a classical approach using Michaelis-Menten equations to model the activation and repression of the various promoters, including a cooperativity of order 2 for the repressor which often the case for 'classical' repressors such as LacI. We obtained the following system of equations:
</P><P>
It is possible to obtain a broad spectrum of different dynamical behaviors by playing with the value of the maximum production rate of Z3, called here β . This maximal production rate impacts deeply both the evolution of the exported protein concentration and the evolution of TolRII concentration. This rate is initially chosen equal to 40 and 400 molecules by cell cycle and we have plotted the different responses for multiples of these initial values.
</P><P>In each case, the initial conditions are the same :
</P><UL><LI>no proteins encapsulated
</LI><LI>no TolRII
</LI><LI>no Z1 molecules
</LI><LI>no Z2 molecules
</LI><LI>no Z3 molecules
</LI><LI>a weak amount of Z4 molecules (around a hundred)
</LI></UL><CENTER>Weak maximal Z3 production rate</CENTER><CENTER><I>An example of dynamics for a range of weak β values; for each plot, the maximal Z3 production rate is given by: β=k*40</I></CENTER><CENTER>Strong maximal Z3 production rate</CENTER><CENTER><I>An example of dynamics for a range of strong β values; for each plot, the maximal Z3 production rate is given by: β=k*400</I></CENTER><H4><SPAN class="mw-headline" id="A_more_flexible_design">A more flexible design</SPAN></H4><P>Regardless of the steady state concentrations of the molecules, the use of two incoherent feed-forward loops allows for <B>more flexibility in the behaviors of our system</B>. With a simple delay system, we just get one possible dynamic behavior whereas using a variable range of production rates, we can obtain two behaviors:
</P><UL><LI>When β is weak (β = 40), the number of vesicles (ie of TolRII) produced slowly increase before returning to zero, whereas the protein to be exported inside the vesicles is created very quickly and remains at a high concentration compared to the TolRII protein. We can therefore send vesicles containing an important amount of interesting proteins and, after a while, stop expressing dangerous proteins for the cell. All of this can be performed with a single input signal.
</LI></UL><UL><LI>When β is stronger, the behavior is different and we notice a change of concavity of the TolRII curve when the amount of protein exported is at its maximum. During such a change of concavity, the derivative reaches its maximum - in other words, the amount of TolRII produced by time units is at its highest level. Considering the hypothesis according to which the production rate of vesicles is proportional to TolRII production rate, we get a peak in vesicle production when the concentration of the proteins to exported is at its maximum.
</LI></UL><P>Unfortunately, we did not have the time to try to construct this system in the lab and only the first design was implemented. Nevertheless, we thought that <B>this analysis was very useful to get a good insight of the diversity of the possible behaviors of the sender cells</B>.
</P><CENTER><P><A href="/Team:Paris/Production_overview" title="Team:Paris/Production overview">← Previous</A> - <A href="/Team:Paris/Production_modeling2" title="Team:Paris/Production modeling2">Next →</A></P></CENTER></DIV><DIV id="paris_content"><H4><SPAN class="mw-headline" id="References">References</SPAN></H4><OL class="References"><LI><A href="/Team:Paris/Production_modeling#1" title="Team:Paris/Production modeling"> ^</A>Ultrasensitivity and noise propagation in a synthetic transcriptional cascade S.Hooshangi &amp; R.Weiss
 2005  - <A href="http://www.ncbi.nlm.nih.gov/pubmed/15738412" class="external text" rel="nofollow">15738412</A></LI><LI><A href="/Team:Paris/Production_modeling#1" title="Team:Paris/Production modeling"> ^</A>Network motifs : theory and experimental approaches U.Alon
 2007  - <A href="http://www.ncbi.nlm.nih.gov/pubmed/17510665" class="external text" rel="nofollow">117510665</A></LI><LI><A href="/Team:Paris/Production_modeling#3" title="Team:Paris/Production modeling"> ^</A>New Unstable Variants of Green Fluorescent Protein for Studies of Transient Gene Expression in Bacteria Jens Bo Andersen &amp; Søren Molin 1998  - <A href="http://www.ncbi.nlm.nih.gov:80/pmc/articles/PMC106306/" class="external text" rel="nofollow">106306</A></LI><LI><A href="/Team:Paris/Production_modeling#4" title="Team:Paris/Production modeling"> ^</A>Spatiotemporal control of gene expression with pulse-generating networks S.Basu &amp; R.Weiss
 2003  - <A href="http://www.ncbi.nlm.nih.gov/pubmed/15096621" class="external text" rel="nofollow">15096621</A></LI><LI><A href="/Team:Paris/Production_modeling#5" title="Team:Paris/Production modeling"> ^</A>Negative Autoregulation Speeds The Response Times of Transcription Network N.Rosenfold &amp; U.Alon
 2002  - <A href="http://www.ncbi.nlm.nih.gov/pubmed/12417193" class="external text" rel="nofollow">2417193</A></LI><LI><A href="/Team:Paris/Production_modeling#6" title="Team:Paris/Production modeling"> ^</A>A Synthetic oscillatory network of transcriptionnal regulators  M.Ellowitz &amp; S.Leibler 1999  - <A href="http://www.ncbi.nlm.nih.gov/pubmed/10659856" class="external text" rel="nofollow">10659856</A></LI><LI><A href="/Team:Paris/Production_modeling#7" title="Team:Paris/Production modeling"> ^</A>Structure and function ot the feed-forward Loop Network Motif S.Mangan &amp; U.Alon
 2003  - <A href="http://www.ncbi.nlm.nih.gov/pubmed/14530388" class="external text" rel="nofollow">14530388</A></LI></OL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Paris/Production_modeling">http://2009.igem.org/Team:Paris/Production_modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris/Production_modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris/Production_modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris/Production_modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris/Production_modeling&amp;oldid=160475" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>