<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Transduction_overview2_transduction"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Paris/Transduction_overview2_transduction">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris/Transduction_overview2_transduction&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris/Transduction_overview2_transduction&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris/Transduction_overview2_transduction&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris/Transduction_overview2_transduction" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris/Transduction overview2 transduction</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><SPAN id="bottom"><A href="https://2009.igem.org/" class="external text" rel="nofollow">iGEM </A>&gt;<A href="/Team:Paris#top" title="Team:Paris"> Paris</A>&gt;<A href="/Team:Paris/Transduction_overview#top" title="Team:Paris/Transduction overview"> Receiving the message</A>&gt;<A href="/Team:Paris/Transduction_overview2#bottom" title="Team:Paris/Transduction overview2"> Transduction</A></SPAN></P><DIV id="header"><DIV id="header_left"><SPAN class="plainlinks"><A href="https://2009.igem.org/Team:Paris/iLife#top" class="external text" rel="nofollow"> </A></SPAN></DIV><DIV id="header_right"><SPAN class="plainlinks"><A href="https://2009.igem.org/Team:Paris#top" class="external text" rel="nofollow"> </A></SPAN></DIV><DIV id="header_bar_lab"><P><A href="/Team:Paris/Protocols#top" title="Team:Paris/Protocols"> Protocols</A><A href="/Team:Paris/Freezer#top" title="Team:Paris/Freezer"> Freezer</A><A href="/Team:Paris/NoteBook#top" title="Team:Paris/NoteBook"> NoteBook</A><A href="/Team:Paris/Papers#top" title="Team:Paris/Papers"> Bibliography</A><A href="/Team:Paris/Parts#top" title="Team:Paris/Parts"> Parts </A><A href="/Team:Paris/WetLab#top" title="Team:Paris/WetLab"> WetLab </A><A href="/Team:Paris/DryLab#top" title="Team:Paris/DryLab"> DryLab </A></P></DIV><DIV id="header_bar_link"><P><A href="/Team:Paris/Team#top" title="Team:Paris/Team"> Team</A><A href="/Team:Paris/Collaborations#top" title="Team:Paris/Collaborations"> Collaborations</A><A href="/Team:Paris/Contact#top" title="Team:Paris/Contact"> Contacts</A><A href="/Team:Paris/Acknowledgements#top" title="Team:Paris/Acknowledgements"> Acknowledgements</A><A href="/Team:Paris/Links#top" title="Team:Paris/Links"> Links</A></P></DIV></DIV><DIV id="menu2"><P><TITLE>Menu accordéon avec jQuery</TITLE><TABLE cellspacing="0"><TBODY><TR><TD><UL id="navigation"><LI class="menu_header"> </LI><LI class="toggleSubMenu"><SPAN> Home </SPAN></LI><LI><A href="https://2009.igem.org/Team:Paris#top">Synopsis</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Project#top">Project overview</A></LI><LI><A href="https://2009.igem.org/Team:Paris/DryLab#top">Modeling overview</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Conclusion#top">Conclusion &amp; Results</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Brainstorm#top">Brainstorm</A></LI><LI><A href="https://2009.igem.org/Team:Paris/safety#top">Safety</A></LI></UL><LI class="menu_footer"> </LI><LI class="menu_header"> </LI><LI class="toggleSubMenu"><SPAN> Addressing the message </SPAN></LI><LI><A href="https://2009.igem.org/Team:Paris/Addressing_overview#top">To the periplasm</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Addressing_overview2#top">In the outer membrane</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Addressing_testing#top">WetLab</A></LI><LI class="toggleSubMenu"><SPAN> Producing the messenger </SPAN></LI><LI><A href="https://2009.igem.org/Team:Paris/Production_overview#top">Vesicle production system</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Production_modeling#top">Vesicle production model</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Production_modeling2#top">Vesicle biophysics model</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Production_testing#top">WetLab</A></LI><LI class="toggleSubMenu2"> Receiving the message
            </LI><LI><A href="https://2009.igem.org/Team:Paris/Transduction_overview#top">Membrane fusion</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Transduction_overview2#top">Signal Transduction</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Transduction_modeling#top">Fec operon simulation</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Transduction_testing#top">WetLab</A></LI><LI class="menu_footer"> </LI><LI class="menu_header"> </LI><LI class="toggleSubMenu"><SPAN> Ethics </SPAN></LI><LI><A href="https://2009.igem.org/Team:Paris/Ethics_overview#top">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Ethics_ethicallabbook#top">Lab book</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Ethics_ethicalreport#top">Report</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Ethics_Materials#top">Materials</A></LI><LI class="menu_footer"> </LI><LI class="menu_header"> </LI><LI class="toggleSubMenu"><SPAN> Tool, iPhone </SPAN></LI><LI><A href="https://2009.igem.org/Team:Paris/Tool_introduction#top">Introduction</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Tool_DataBase#top ">Databases</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Tool_OSXSoft#top">Mac Software</A></LI><LI><A href="https://2009.igem.org/Team:Paris/Tool_iPhone#top">iPhone</A></LI><LI class="menu_footer"> </LI></TD></TR></TBODY></TABLE></P></DIV><DIV id="paris_content_wrapper"><DIV id="paris_content"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Transduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Transduction</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#ABC_transporters"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">ABC transporters</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Uses"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Uses</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Advantages.2Fdrawbacks"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Advantages/drawbacks</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#Two-component_system"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Two-component system</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Uses_2"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Uses</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Advantages.2Fdrawbacks_2"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Advantages/drawbacks</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#References"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Transduction">Transduction</SPAN></H2><DIV id="middle-side"><CENTER><A class="menu_sub" href="https://2009.igem.org/Team:Paris/Transduction_overview2#bottom"> Main </A>|
<A class="menu_sub_active" href="https://2009.igem.org/Team:Paris/Transduction_overview2_transduction#bottom"> Transduction</A>|
<A class="menu_sub" href="https://2009.igem.org/Team:Paris/Transduction_overview2_strategy#bottom"> Our strategy</A>|
<A class="menu_sub" href="https://2009.igem.org/Team:Paris/Transduction_overview2_construction#bottom"> Construction</A></CENTER></DIV><DIV id="middle-side2"><CENTER><A class="menu_sub_active" href="https://2009.igem.org/Team:Paris/Transduction_overview2_transduction#ABC_transporters"> ABC transporters</A>|
<A class="menu_sub" href="https://2009.igem.org/Team:Paris/Transduction_overview2_transduction#Two-component_system"> Two-component system</A></CENTER></DIV><P>In biology, signal transduction refers to any process by which a cell converts one kind of signal or stimulus into another. Most processes of signal transduction involve ordered sequences of biochemical reactions inside the cell, which are carried out by enzymes and sometimes activated by second messengers, resulting in a signal transduction pathway. 
</P><P>
Such processes are usually rapid, lasting on the order of milliseconds in the case of ion flux, or minutes for the activation of proteic cascades, but some can take hours, and even days (as is the case with gene expression), to complete. 
</P><P>
We must also refer to the amplification of the signal , in which a relative small stimulus can elicit a large response. It is often du to the increasing numbers of protein ativation
implied in the transduction pathways. 
</P></DIV><DIV id="paris_content"><SPAN id="1"><H2><SPAN class="mw-headline" id="ABC_transporters">ABC transporters</SPAN></H2><P>The ABC transporter is a major class of cellular translocation machinery encoded in the largest set of paralogous genes <SUP><A href="/Team:Paris/Transduction_overview2_transduction#References" title="Team:Paris/Transduction overview2 transduction">[1]</A></SUP>.
</P><P>
ABC (ATP binding cassette) transporters are active transport systems of the cell, which is widespread in archaea, eubacteria, and eukaryotes. They require energy in the form of adenosine triphosphate (ATP) to translocate substrates across cell membranes. These proteins harness the energy of ATP binding and/or hydrolysis to drive conformational changes in the transmembrane domain (TMD) and consequently transports molecules <SUP><A href="/Team:Paris/Transduction_overview2_transduction#References" title="Team:Paris/Transduction overview2 transduction">[2]</A></SUP>.
ABC transporters are also known as the periplasmic binding protein-dependent transport system in Gram-negative bacteria and the binding-lipoprotein-dependent transport system in Gram positive bacteria. 
The transporter shows a common global organization with three types of molecular components. Typically, it consists of two integral membrane proteins (permeases) each having six transmembrane segments, two peripheral membrane proteins that bind and hydrolyze ATP, and a periplasmic (or lipoprotein) substrate-binding protein. The ATP-binding protein component is the most conserved, the TMD is somewhat less conserved, and the substrate-binding protein component is most divergent in terms of the sequence similarity. The ABC transporters form the largest group of paralogous genes in bacterial and archaeal genomes , and the genes for the three components frequently form an operon. 
</P><H4><SPAN class="mw-headline" id="Uses">Uses</SPAN></H4><P>Bacterial ABC transporters are essential in cell viability, virulence, and pathogenicity <SUP><A href="/Team:Paris/Transduction_overview2_transduction#References" title="Team:Paris/Transduction overview2 transduction">[3]</A></SUP> Iron ABC uptake systems, for example, are important effectors of virulence
<SUP><A href="/Team:Paris/Transduction_overview2_transduction#References" title="Team:Paris/Transduction overview2 transduction">[4]</A></SUP>.
Prokaryotic ABC transporters utilize the energy of ATP hydrolysis to transport lot of different substrates across cellular membranes. They are divided into 2 functionnal categories :
</P><P>- <B>importers</B> : they mediate the uptake of amino-acids, peptids, sugar ... into the cell, so they allow the transport of protein directly into the cytolpasm to activate the transcription <SUP><A href="/Team:Paris/Transduction_overview2_transduction#References" title="Team:Paris/Transduction overview2 transduction">[5]</A></SUP>. 
</P><P>- <B>exporters</B> : they transport lipids and some polysaccharides from the cytoplasm to the periplasm.
</P><H4><SPAN class="mw-headline" id="Advantages.2Fdrawbacks">Advantages/drawbacks</SPAN></H4><P>The import system of these transporters is really interesting and attractive four our project to transmit a &quot;message&quot; from vesicles to target cells.
The main advantage is that the protein of interest is directly translocated in the cytoplasm and if it is a transcription factor it could activated immediatly the response. The communication is simple,fast and efficient.
But this system is not perfect... One major drawback is that ABC transporter system is a nutriment uptake system. So, basically only (very) small molecules are able to pass thought the membranes.
</P></SPAN></DIV><DIV id="paris_content"><H2><SPAN class="mw-headline" id="Two-component_system">Two-component system</SPAN></H2><P>The Two-Component System (TCS) can be considered as a widely spread class of biosensor knowing that adaptive signal transduction within microbial cells involving a multi-faceted regulated phosphotransfer mechanism that comprises structural rearrangements of sensor histidine kinases upon ligand-binding and phosphorylation-induced conformational changes in response regulators of versatile TCS, arisen early in bacterial evolution <SUP><A href="/Team:Paris/Transduction_overview2_transduction#References" title="Team:Paris/Transduction overview2 transduction">[6]</A></SUP>, <SUP><A href="/Team:Paris/Transduction_overview2_transduction#References" title="Team:Paris/Transduction overview2 transduction">[7]</A></SUP></P><P>
In most eubacteria, two-component proteins typically constitute 1% of encoded peptides. In pathogenic bacteria they control the expression of important pathogenetic factors, in addition to regulating basic housekeeping functions. The widespread distribution of two-component signal transduction systems in Bacteria and Achaea reﬂects their biological value as major sensing and response elements to a wide range of environmental insults that are tuned to respond from within milliseconds to hours <SUP><A href="/Team:Paris/Transduction_overview2_transduction#References" title="Team:Paris/Transduction overview2 transduction">[8]</A></SUP>. Although TCSs are probably the most eﬁcient means of adaptation to conventional stressful stimuli encountered by bacteria during their lifespan, the plasticity of some of these sophisticated systems may contribute to strain-speciﬁc cellular processes and to the acquisition of distinct features and phenotypes, particularly in pathogens <SUP><A href="/Team:Paris/Transduction_overview2_transduction#References" title="Team:Paris/Transduction overview2 transduction">[9]</A></SUP>. 
</P><P>
To put the structure in a nutshell: A typical TCS consists of a transmembrane dimeric sensor histidine kinase (HK) and a cytoplasmic cognate response regulator (RR). In gram negative bacteria there is often a Periplasmic Binding Protein which optimize the detection of the molecule localized in the periplasm by a high affinity for the HK after binding the specific molecule. The following scheme shows a typical ABC mechanism : 
</P><H4><SPAN class="mw-headline" id="Uses_2">Uses</SPAN></H4><P>Two-component systems serve as a basic stimulus-response coupling mechanism to allow organisms to sense and respond to changes in many different environmental conditions <SUP><A href="/Team:Paris/Transduction_overview2_transduction#References" title="Team:Paris/Transduction overview2 transduction">[6]</A></SUP>.
</P><P>
By transferring a packet of molecule synthesize by the donor but not present in the medium, 
the arrival could activate the transcription of gene of interest. This will allow us to check for the &quot;bacterial communication&quot;
</P><H4><SPAN class="mw-headline" id="Advantages.2Fdrawbacks_2">Advantages/drawbacks</SPAN></H4><P>Most gene regulation proteins that were known were single proteins, often homodimers or homotetramers, which bound to two ligands, a metabolic intermediate or a cis-acting gene regulation element. The functional state of the regulatory protein was thus modified by  binding to the metabolic intermediate. The consequence of ligand binding, an altered state of the regulatory protein, was directed to the appropriate gene(s) by the protein's DNA binding activity. 
</P><P>
The main advantage of TCS is that the mechanism is dedicated to the transcription of gene under a specific promoter. Four our project, the difficulty is to find an easily exportable and detectable signal. 
In contrats, most of the biosensor are generally sensible to high diffusible molecule for which the vesicles transport is useless, so using vesicules to transmit a message doesn't seems the best way to explore.
</P><CENTER><P><A href="/Team:Paris/Transduction_overview2#top.23top" title="Team:Paris/Transduction overview2">← Previous</A> - <A href="/Team:Paris/Transduction_overview2_strategy#bottom" title="Team:Paris/Transduction overview2 strategy">Next →</A></P></CENTER></DIV><DIV id="paris_content"><H4><SPAN class="mw-headline" id="References">References</SPAN></H4><OL class="references"><LI><A href="/Team:Paris/Transduction_overview2_transduction#1" title="Team:Paris/Transduction overview2 transduction"> ^</A>Kentaro Tomii and Minoru Kanehisa, A Comparative Analysis of ABC Transporters in Complete Microbial Genomes,Genome Res. 1998 Oct;8(10):1048-59. <A href="http://www.ncbi.nlm.nih.gov/pubmed/9799792" class="external text" rel="nofollow">9799792</A></LI><LI><A href="/Team:Paris/Transduction_overview2_transduction#1" title="Team:Paris/Transduction overview2 transduction"> ^</A>Hollenstein, K.; R.J. Dawson; K.P. Locher. 2007. Structure and mechanism of ABC transporter proteins. Curr. Opin. Struct. Biol. 17, 412-418. <A href="http://www.ncbi.nlm.nih.gov/pubmed/17723295" class="external text" rel="nofollow">17723295</A></LI><LI><A href="/Team:Paris/Transduction_overview2_transduction#1" title="Team:Paris/Transduction overview2 transduction"> ^</A>Davidson, A.L.; E. Dassa; C. Orelle; and J. Chen. 2008. Structure, function, and evolution of bacterial ATP-binding cassette systems. Microbiol. Mol. Biol. Rev. 72(2), 317-364.<A href="http://www.ncbi.nlm.nih.gov/pubmed/18535149" class="external text" rel="nofollow">18535149</A></LI><LI><A href="/Team:Paris/Transduction_overview2_transduction#1" title="Team:Paris/Transduction overview2 transduction"> ^</A>Henderson, D.P. and S.M. Payne. 1994. Vibrio cholerae iron transport system: roles of heme and siderophore iron transport in virulence and identification of a gene associated with multiple iron transport systems. Infect. Immun. 62, 5120-5125. <A href="http://www.ncbi.nlm.nih.gov/pubmed/7927795" class="external text" rel="nofollow">7927795</A></LI><LI><A href="/Team:Paris/Transduction_overview2_transduction#1" title="Team:Paris/Transduction overview2 transduction"> ^</A>
Davidson, A.L.; E. Dassa; C. Orelle; and J. Chen. 2008. Structure, function, and evolution of bacterial ATP-binding cassette systems. Microbiol. Mol. Biol. Rev. 72(2), 317-364.<A href="http://www.ncbi.nlm.nih.gov/pubmed/18535149" class="external text" rel="nofollow">18535149</A></LI><LI><A href="/Team:Paris/Transduction_overview2_transduction#1" title="Team:Paris/Transduction overview2 transduction"> ^</A>Stock AM, Robinson VL, Goudreau PN. Two-component signal transduction.2000, Annu. Rev. Biochem. 69: 183–215.<A href="http://www.ncbi.nlm.nih.gov/pubmed/10966457" class="external text" rel="nofollow">10966457</A></LI><LI><A href="/Team:Paris/Transduction_overview2_transduction#1" title="Team:Paris/Transduction overview2 transduction"> ^</A>Mascher T, Helmann JD, Unden G. Stimulus perception in bacterial signal-transducing histidine kinases. 2006. Microbiol. Mol. Biol. Rev. 70 (4): 910–38.<A href="http://www.ncbi.nlm.nih.gov/pubmed/17158704" class="external text" rel="nofollow">17158704</A></LI><LI><A href="/Team:Paris/Transduction_overview2_transduction#1" title="Team:Paris/Transduction overview2 transduction"> ^</A>
Attwood PV, Piggott MJ, Zu XL, Besant PG. Focus on phosphohistidine, 2007, Amino Acids 32 (1): 145–56. <A href="http://www.ncbi.nlm.nih.gov/pubmed/17103118" class="external text" rel="nofollow">17103118</A></LI><LI><A href="/Team:Paris/Transduction_overview2_transduction#1" title="Team:Paris/Transduction overview2 transduction"> ^</A>Kyriakidis DA, Tiligada E. Signal transduction and adaptive regulation through bacterial two-component systems: the Escherichia coli AtoSC paradigm, 2009,  Amino Acids.37(3):443-58. <A href="http://www.ncbi.nlm.nih.gov/pubmed/19198978" class="external text" rel="nofollow">19198978</A></LI></OL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Paris/Transduction_overview2_transduction">http://2009.igem.org/Team:Paris/Transduction_overview2_transduction</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris/Transduction_overview2_transduction" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris/Transduction_overview2_transduction" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris/Transduction_overview2_transduction&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris/Transduction_overview2_transduction&amp;oldid=167645" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>