"
Page
Discussion
View source
History
teams
Log in
 
Team:Paris/Production design
From 2009.igem.org
iGEM > Paris > Production > Parts Design
 
 
Protocols Freezer NoteBook Bibliography Parts WetLab DryLab
Team Collaborations Contacts Acknowledgements Links
Menu accordéon avec jQuery
 
Home
Synopsis
Project overview
Modeling overview
Conclusion & Results
Brainstorm
Safety
 
 
Addressing the message
To the periplasm
In the outer membrane
WetLab
Producing the messenger
Vesicle production system
Vesicle production model
Vesicle biophysics model
WetLab
Receiving the message
Membrane fusion
Signal Transduction
Fec operon simulation
WetLab
 
 
Ethics
Overview
Lab book
Report
Materials
 
 
Tool, iPhone
Introduction
Databases
Mac Software
iPhone
 
Contents
1Parts used for the production of vesicles
1.1Our contributions
1.2Registered parts we used
1.2.1Backbones
1.2.2Biobricks
Parts used for the production of vesicles
Main | Production | Addressing | Reception | RBS
Our contributions
TolRII
Colicin E3
etc...
NB: All coding sequences intended to be expressed are associated with a standard RBS incorporated in the forward primer used to amplify the sequence by PCR. The standard RBS used was carefully designed to ensure maximal functionality (see RBS )
The RBS were designed by Guillaume Cambray and integrated directly inside the forward primer to yield expressible units.
The RBS sequence used is the following: 5'-AAGGAGGTATATA-3', immediately followed by the ATG initiation codon.
The AAGGAGG motif is complementary to the 16S RNA of the ribosome and ensure full recognition by the translational machinery. The first A overlap the XbaI site to reduce primer length.  The distance between the first GG and the initiation codon is set to 9 bp, which appear to be the optimal spacing according to Anatomy of Escherichia coli ribosome binding sites - Shultzaberger et al, 2001.  The (TA) repeats spacer is used because the GC content of the RBS should be low to favor the separation of the DNA strand. Alternating T and A are meant to avoid errors during primer synthesis. At last, this spacer is designed so that the -3 bp before ATG is an adenine, which may be important according to old reports.
Overall the forward primer used for amplifying CDS has the following template sequence:
5'-TTGAATTCGCGGCCGCTTCTAGAAGGAGGTATATAATG-3'    -- 2 bases to ensure efficient EcoRI cutting while minimizing primer length      --------------------- Biobrick standard RFC10                          ------- RBS                                 ------ Low GC spacer                                    - potentially important A                                       --- Start codon
Registered parts we used
Backbones
The function "increased production of OMV" is borne on two plasmid backbones pSB1A3 and pSB3T5 .
Biobricks
Expression of the proteins targeted to OMVs as well as lacI is controled by the Pbad promoter BBa_K11309 .
We used the Ptet promoter BBa_ROO40 and the coding sequence for the TetR repressor protein BBa_COO40 to tightly repress the expression of protein domains that are expected to destabilize the outer membrane, thereby leading to increased production of OMVs. We hope that TetR mediated repression is strong enough to avoid leakage and weak expression of these deleterious polypeptides.
The Plac promoter BBa_I14032 associated to lacI [BBa_C0012 http://partsregistry.org/Part:BBa_C0012 ] to control the expression of tetR.
The double terminator BBa_B0014 was used to isolates the different expression units from each other.
Retrieved from " http://2009.igem.org/Team:Paris/Production_design "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
