<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NYMU-Taipei_Modeling_Ethanol"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NYMU-Taipei/Modeling/Ethanol">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NYMU-Taipei/Modeling/Ethanol&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/Modeling/Ethanol&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/Modeling/Ethanol&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NYMU-Taipei/Modeling/Ethanol" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NYMU-Taipei/Modeling/Ethanol</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="head"><DIV class="nymu"><DIV id="nymu-taipei">
			National Yang Ming University
		</DIV></DIV><DIV class="nav"><DIV id="nav-bar"><UL id="sdt_menu" class="sdt_menu"><LI><A href="https://2013.igem.org/Team:NYMU-Taipei"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Home</SPAN></SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Team"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Team</SPAN></SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Overview"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Project</SPAN></SPAN></A><DIV class="sdt_box" style="height: 270px; width: 190px;"><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Overview">Overview</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Enter">Entering of Bee.coli</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Inhibition/Prohibition">Prohibiting Sprouting</A><A href="https://2013.igem.org/Team:NYMU-
Taipei/Project/Inhibition">Inhibiting Infection</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Inhibition/Sensor" class="sdt_aWsub">Sensing Nosema</A><A href="https://2013.igem.org/Team:NYMU-
Taipei/Project/Inhibition/Killing" class="sdt_aWsub">Killing Nosema</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Inhibition/Inhibition circuit" class="sdt_aWsub">Circuit</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Kill">Suicide Solution</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/safe">eco-friendly e.coli</A><A href="https://2013.igem.org/Team:NYMU-
Taipei/Project/Inhibition/Id-Nosema">Identifying Nosema</A></DIV></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Experiments"><SPAN class="sdt_wrap"><SPAN class="sdt_link">experiment</SPAN></SPAN></A><DIV class="sdt_box" style="width: 155px;"><A href="https://2013.igem.org/Team:NYMU-Taipei/Experiments/Protocols">Protocols</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Experiment/Wet_Lab">Data</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Experiments/Functional_Assays">Notes</A></DIV></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/Overview"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Modeling</SPAN></SPAN></A><DIV class="sdt_box" style="width: 135px; height: 160px;"><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/Overview">Overview</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/ModSensors">Sensors</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/MainParts">Main Parts</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/Ethanol">Ethanol Model</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/Linear_epidemic_model">Epidemic Model</A></DIV></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Human_Practice"><SPAN class="sdt_wrap"><SPAN class="sdt_link" style="font-size: 25px; padding-top: 5px; line-height: 1em;">Human Practice</SPAN></SPAN></A><DIV class="sdt_box" style="width: 185px; height: 160px;"><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/Overview" style="margin-left: 13px;">Overview</A><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/HumOut" style="margin-left: 13px;">Outreach</A><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/HumCollab" style="margin-left: 13px;">Collaborations</A><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/HumFlows" style="margin-left: 13px;">Flows Built-up</A><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/HumPracticalApp">Practical Applications</A></DIV></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Safety"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Safety</SPAN></SPAN></A></LI></UL></DIV></DIV></DIV></DIV><DIV class="main"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Ethanol_model"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Ethanol model</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Background"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Objectives"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Objectives</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#System"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">System</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Results"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Discussion"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Circuit_Design_and_Improvements"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Circuit Design and Improvements</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Parameters"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Parameters</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Ethanol_model">Ethanol model</SPAN></H1><H2><SPAN class="mw-headline" id="Background">Background</SPAN></H2><P>AhpCp promoter is an oxyR-activated promoter. Behind AhpCp promoter are several terminals to inhibit T7 polymerase-producing gene from being opened easily ; namely, it will not open until the concentration of oxyR is high enough, which means the bees are facing disastrous infection. Once oxyR concentration overcomes the threshold and conquers the terminal obstacles, T7 polymerase is produced and will bind to T7 promoter, which is a specific promoter binding only to T7 polymerase. Behind the T7promoter are enzyme PDC and ADH-producing genes, which will convert pyruvate to ethanol, and thus kill spores of <I>Nosema Ceranae</I>.
<B>Pathway of ethanol</B></P><P>PDC= pyruvate decarboxylase; ADH= Acetaldehyde
</P><H2><SPAN class="mw-headline" id="Objectives">Objectives</SPAN></H2><OL><LI>To simulate how many terminal do we need as a threshold to have the T7 polymerase-producing gene open at a proper 　　time.</LI><LI>To simulate the threshold concentration of oxyR to conquer the terminal.</LI><LI>To determine the time of opening to see if the circuit could open in time (useful).</LI></OL><H2><SPAN class="mw-headline" id="System">System</SPAN></H2><P>It is assumed that the dose of pyruvate is sufficient in bees’ body through enough proliferation/copy number of Beecoli; the possibility of RNApolymerase skipping terminators is assumed to be proportional to time span; equilibrium between pyruvate and acetaldehyde is dominantly rightward, while equilibrium between acetaldehyde and ethanol is bidirectional; concentration of ethanol will not easily decrease and will sustain for a period of time.</P><P>As for how we measure promoter strengths, we choose PoPS, which is the rate of RNA polymerase binding to the DNA and trigger the transcription of the certain gene.</P><P><B>Equation1</B></P><DIV lang="latex" class="equation">
\frac{d[mRNA_T_7]}{dt}=\frac{ [ROSoxyR]^{nROSoxyR} }{ KdROSoxyR^{nROSoxyR}+[ROSoxyR]^{nROSoxyR} }\times{PoPSAhpCp}\times\frac{N}{V}\times{a}+\frac{ [T7]^{nT7} }{ KdT7^{nT7}+[T7]^{nY7} }\times{PoPST7}\times\frac{N}{V}-KdegmRNA\times[mRNAPDC]
</DIV><OL><LI>KdROSoxyR = dissociation constant of ROSoxyR</LI><LI>n ROSoxyR = Hill coefficient of ROSoxyR</LI><LI>PoPSAhpCp = promoter strength of AhpCp</LI><LI>KdT7 = dissociation constant of T7</LI><LI>NT7 = Hill coefficient of T7</LI><LI>PoPST7 = promoter strength of T7 promoter</LI><LI>KdegmRNA = degrading constant of sensor promoter mRNA</LI><LI>N = number of plasmid in a single cell</LI><LI>V = volume of a cell</LI></OL><P><A href="/More1" title="More1">more1</A></P><P><B>Equation2</B></P><DIV lang="latex" class="equation">
\frac{d[protein of T7 polymerase ]}{dt}=RBS\times{PoPSAhpCp}\times\frac{N}{V}\times{a}\times{\frac{ [ROSoxyR]^{n1} }{ [Kd]^{n1}+[ROSoxyR]^{n1} }}}+PoPST7\times{T7 effect}\times\frac{N}{V}-{KdegT7}\times{T7}
</DIV><OL><LI>PoPSAhpCp = promotor strength of AhpCp promoter</LI><LI>PoPST7 = promoter strength of T7 promoter</LI><LI>RBS = binding site strength</LI><LI>kdegT7 = degrading constant of T7 polymerase</LI><LI>a = posibility of conquering the threshold concentration</LI><LI>n1 = Hill coefficient of ROSoxyR (complex of ROS+oxyR)</LI><LI>N = number of plasmid in a single cell </LI><LI>V = volume of a cell</LI></OL><P><B>The aim of the equation is to know the threshold concentration of oxyR to conquer the terminal and when T7 polymerase can reach the required concentration to activate T7 promoter.</B></P><P><A href="/More2" title="More2">more2</A></P><P><B>Equation3</B></P><DIV lang="latex" class="equation">
\frac{d[mRNA_P_D_C]}{dt}=\frac{ [T7]^{nT7} }{ KdT7^{nT7}+[T7]^{nY7} }\times{PoPST7}\times\frac{N}{V}-KdegmRNA\times[mRNAPDC]
</DIV><OL><LI>KdT7 = dissociation constant of T7</LI><LI>NT7 = Hill coefficient of T7</LI><LI>PoPST7 = promoter strength of T7 promoter</LI><LI>KdegmRNA = degrading constant of sensor promoter mRNA</LI><LI>N = number of plasmid in a single cell</LI><LI>V = volume of a cell</LI></OL><P><B>The aim of the equation is to know mRNA of enzyme PDC production rate and when it can reach the level to translate enough PDC.</B></P><P><A href="/More3" title="More3">more3</A></P><P><B>Equation4</B></P><DIV lang="latex" class="equation">
\frac{d[PDC]}{dt}=RBS\times{PoPST7effect}\times\frac{N}{V}-{kdegPDC}\times[PDC}]
</DIV><OL><LI>RBS = binding site strength</LI><LI>PoPST7 = promoter strength of T7 promoter</LI><LI>N = number of plasmid in a single cell</LI><LI>V = volume of a cell</LI><LI>KdegPDC = degrading constant of PDC (pyruvate decarboxylase)</LI></OL><P><B>The aim of the equation is to know PDC production rate and when it can reach the concentration of ethanol pathway equilibrium.</B></P><P><A href="/More4" title="More4">more4</A></P><P><B>Equation5</B></P><DIV lang="latex" class="equation">
\frac{d[mRNA_A_D_H]}{dt}=\frac{ [T7]^{nT7} }{ KdT7^{nT7}+[T7]^{nY7} }\times{PoPST7}\times\frac{N}{V}-KdegmRNA\times[mRNAADH]
</DIV><OL><LI>KdT7 = dissociation constant of T7</LI><LI>NT7 = Hill coefficient of T7</LI><LI>PoPST7 = promoter strength of T7 promoter</LI><LI>KdegmRNA = degrading constant of sensor promoter mRNA</LI><LI>N = number of plasmid in a single cell</LI><LI>V = volume of a cell</LI></OL><P><A href="/More5" title="More5">more5</A></P><P><B>Equation6</B></P><DIV lang="latex" class="equation">
\frac{d[ADH]}{dt}=RBS\times{PoPST7effect}\times\frac{N}{V}-{KdegADH}\times[ADH]}
</DIV><OL><LI>RBS = binding site strength</LI><LI>PoPST7 = promoter strength of T7 promoter</LI><LI>N = number of plasmid in a single cell</LI><LI>V = volume of a cell</LI><LI>KdegADH = degrading constant of ADH (Acetaldehyde)</LI></OL><P><B>The aim of the equation is to know ADH production rate and when it can reach the concentration of ethanol pathway equilibrium.</B></P><P><A href="/More6" title="More6">more6</A></P><P><B>Equation7</B></P><DIV lang="latex" class="equation">
\frac{d[ethanol]}{dt}=Kpyruvateacetaldehyde\times{Kacetaldehydeethanol}\times\frac{[PDC]^{nPDC} }{ KdPDC^{nPDC}+[PDC]^{nPDC}}\times\frac{[ADH]^{nADH} }{ KdADH^{nADH}+[ADH]^{nADH}}-Km\times\frac{[ADH]^{nADH}}{ KdADH^{nADH}+[ADH]^{nADH}}\times[ethonal]
</DIV><OL><LI>Kpyruvateacetaldehyde = pyruvate→acetaldehyde reaction rate constant</LI><LI>Kacetaldehydeethanol = acetaldehyde→ethanol reaction rate constant</LI><LI>Km = ethanol→acetaldehyde reaction rate constant</LI><LI>KdADH = dissociation constant of ADH</LI></OL><P><B>The aim of the equation is to know ethanol production rate and when it can reach the concentration to kill the spores of <I>Nosema</I> Ceranae.</B></P><P><A href="/More7" title="More7">more7</A></P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><P>Figure1:this figure shows that after adding several terminators and T7 polymerase mechanism, we successfully build a time delay as well as a boom of PDC and ADH enzymes to attain our goal-ethanol production can reach its effective concentration(0.0005M) after 80 hours of <I>Nosema</I> infection.
</P><P>Figure2:This picture is a control group model. It shows that without <I>Nosema</I> infection (ROS concentration=0), ethanol production is low enough to be ignored. As a result, the bee will not be harmed by ethanol. 
</P><P>Figure3: ethanol production turning off after <I>Nosema</I> is killed  (step 1)
</P><P>Figure4: ethanol production turning off after <I>Nosema</I> is killed  (step 2)
</P><P>Figure3, Figure4: The two pictures above show that if <I>Nosema</I> is killed by kill protein effectively, the ethanol production will soon be turned off (will not reach the effective concentration and decrease to a neglectable low level).
</P><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><P>Figures show that after sensor AhpCp is activated, ethanol reaches its effective concentration 72-80 hours after the infection, which is the time in <I>Nosema</I> lifecycle when bees carrying <I>Nosema</I> spores become incurable and infectious to other bees), In other words, this means if killing protein targeting at <I>Nosema</I> wasn't produced or functioning properly (click here to see the previous model for killing protein), Beecoli turns against the sick bee to kill it within 3-4days.
</P><P>To make ethanol production on time, we adjusted the expression level of OxyR <A href="/(click_here_for_more_information)" title="(click here for more information)">(click here for more information)</A>, the terminators in front of the desired gene, and the promoter strength for independent production of PDC and ADH<A href="/(click_here_for_detailed_information_about_circuit_design)" title="(click here for detailed information about circuit design)">(click here for detailed information about circuit design)</A>.
</P><H2><SPAN class="mw-headline" id="Circuit_Design_and_Improvements">Circuit Design and Improvements</SPAN></H2><P>To postpone ethanol production until it is needed, we adjusted the terminators’ effect in terms of possibility (click here to see the ODE). Results show that there’s a short lag (8-10 hours) between invasion sensing and ethanol production rather than a long one expected. This means that possibility isn’t a sensitive parameter and therefore adding terminator is not an efficient way to manipulate timing when ethanol is produced. Nevertheless, these terminators still has the function of preventing leakage of T7 polymerase production <A href="/(click_here_for_detailed_information_about_circuit_design)" title="(click here for detailed information about circuit design)">(click here for detailed information about circuit design)</A>.
By adjusting the parameters in the ODEs given, we found that the nature of T7 transcription is key to adjusting ethanol production time. If postponing ethanol production is needed, a T7 promoter with lower promoter strength and T7 polymerase with higher degrading rate should be chosen
</P><H2><SPAN class="mw-headline" id="Parameters">Parameters</SPAN></H2><TABLE class="wikitable"><TBODY><TR><TH>Model</TH><TH>Parameter</TH><TH>Description</TH><TH>Value</TH><TH>Unit</TH><TH>Reference
</TH></TR><TR><TD rowspan="3" style="text-align: center;"> Ethanol
</TD><TD colspan="1" style="text-align: center;">pyruvate
</TD><TD colspan="1" style="text-align: center;"> Initial concentration of pyruvate in MG1655
</TD><TD colspan="1" style="text-align: center;"> 1.18 x 2
</TD><TD colspan="1" style="text-align: center;"> g/L
</TD><TD colspan="1" style="text-align: center;">Expression of pyruvate carboxylase enhances succinate production in Escherichia coli without affecting glucose uptake
</TD></TR><TR><TD style="text-align: center;"> nPDC
</TD><TD> Hill coefficient of PDC
</TD><TD style="text-align: center;">2.1
</TD><TD style="text-align: center;"> -
</TD><TD>Purification, characterization and cDNA sequencing of pyruvate decarboxylase Zygosaccharomyces biporus
</TD></TR><TR><TD style="text-align: center;">nADH</TD><TD>Hill coefficient of ADH</TD><TD style="text-align: center;">at high pH values, 30◦C, n=1; at low temperature, n=3
</TD><TD style="text-align: center;">  -
</TD><TD>Evidence for co-operativity in coenzyme binding to tetrameric Sulfolobus
<P>solfataricus alcohol dehydrogenase and its structural basis: ﬂuorescence,
kinetic and structural studies of the wild-type enzyme and non-co-operative
N249Y mutant
</P></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:NYMU-Taipei/Modeling/Ethanol">http://2013.igem.org/Team:NYMU-Taipei/Modeling/Ethanol</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NYMU-Taipei/Modeling/Ethanol" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NYMU-Taipei/Modeling/Ethanol" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NYMU-Taipei/Modeling/Ethanol&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NYMU-Taipei/Modeling/Ethanol&amp;oldid=359150" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>