<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NYMU-Taipei_Modeling_MainParts"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NYMU-Taipei/Modeling/MainParts">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NYMU-Taipei/Modeling/MainParts&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/Modeling/MainParts&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/Modeling/MainParts&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NYMU-Taipei/Modeling/MainParts" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NYMU-Taipei/Modeling/MainParts</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="head"><DIV class="nymu"><DIV id="nymu-taipei">
			National Yang Ming University
		</DIV></DIV><DIV class="nav"><DIV id="nav-bar"><UL id="sdt_menu" class="sdt_menu"><LI><A href="https://2013.igem.org/Team:NYMU-Taipei"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Home</SPAN></SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Team"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Team</SPAN></SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Overview"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Project</SPAN></SPAN></A><DIV class="sdt_box" style="height: 270px; width: 190px;"><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Overview">Overview</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Enter">Entering of Bee.coli</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Inhibition/Prohibition">Prohibiting Sprouting</A><A href="https://2013.igem.org/Team:NYMU-
Taipei/Project/Inhibition">Inhibiting Infection</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Inhibition/Sensor" class="sdt_aWsub">Sensing Nosema</A><A href="https://2013.igem.org/Team:NYMU-
Taipei/Project/Inhibition/Killing" class="sdt_aWsub">Killing Nosema</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Inhibition/Inhibition circuit" class="sdt_aWsub">Circuit</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Kill">Suicide Solution</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/safe">eco-friendly e.coli</A><A href="https://2013.igem.org/Team:NYMU-
Taipei/Project/Inhibition/Id-Nosema">Identifying Nosema</A></DIV></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Experiments"><SPAN class="sdt_wrap"><SPAN class="sdt_link">experiment</SPAN></SPAN></A><DIV class="sdt_box" style="width: 155px;"><A href="https://2013.igem.org/Team:NYMU-Taipei/Experiments/Protocols">Protocols</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Experiment/Wet_Lab">Data</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Experiments/Functional_Assays">Notes</A></DIV></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/Overview"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Modeling</SPAN></SPAN></A><DIV class="sdt_box" style="width: 135px; height: 160px;"><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/Overview">Overview</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/ModSensors">Sensors</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/MainParts">Main Parts</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/Ethanol">Ethanol Model</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/Linear_epidemic_model">Epidemic Model</A></DIV></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Human_Practice"><SPAN class="sdt_wrap"><SPAN class="sdt_link" style="font-size: 25px; padding-top: 5px; line-height: 1em;">Human Practice</SPAN></SPAN></A><DIV class="sdt_box" style="width: 185px; height: 160px;"><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/Overview" style="margin-left: 13px;">Overview</A><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/HumOut" style="margin-left: 13px;">Outreach</A><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/HumCollab" style="margin-left: 13px;">Collaborations</A><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/HumFlows" style="margin-left: 13px;">Flows Built-up</A><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/HumPracticalApp">Practical Applications</A></DIV></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Safety"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Safety</SPAN></SPAN></A></LI></UL></DIV></DIV></DIV></DIV><DIV class="main"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Main_parts"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Main parts</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Backgrounds"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Backgrounds</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Objectives"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Objectives</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#System"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">System</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Results"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Discussion"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Circuit_design_and_improvement"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Circuit design and improvement</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Parameters:"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Parameters:</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Main_parts">Main parts</SPAN></H1><H2><SPAN class="mw-headline" id="Backgrounds">Backgrounds</SPAN></H2><P><B>Circuit regulators</B></P><P>• ptet regulated promoter (ptet) : when ptet exists, it will bind to ptet regulated promoter (ptet) and represses the promoter.
</P><P>• pLux/cI hybrid promoter (plux/cI) : when luxR/AHL exists, it will open pLux/cI hybrid promoter (plux/cI), while cI will repress the promoter. What’s more, plux/cI is cI-dominant, which means when cI exists, the hybrid promoter will be repressed whether luxR/AHL exists or not.
</P><P><B>Circuit regulation:</B></P><P>• Circuit condition without <I>Nosema</I>:
</P><P>Without <I>Nosema ceranae</I>, the first circuit will not open, and thus, no ptet will be produced. Since there is no ptet binding to ptet, ptet will not be repressed, which means cI will be produced. After that, cI will bind to pLux/cI hybrid promoter, leading the third circuit to be off.
</P><P>• Circuit condition with<I> Nosema</I>:
</P><P>When <I>Nosema ceranae</I> infects the bee, bee’s immune system will be activated, leading to the concentration of ROS (reactive oxygen species) to be high. In response,  Bee. coli will enhance the production of oxyR, which forms a complex with ROS and binds to transcription binding site ahead of AhpCp (an oxyR-activated promoter; namely, the sensor), leading to the first circuit to be on.
</P><P>Since the first circuit is on, the downstream ptet gene will be produced and bind to ptet, leading to the second circuit to be off.
</P><P>After the second circuit is closed, no cI is produced, leading to the third circuit to be on (no repressor at all). And then, kill protein will be produced and kill <I>Nosema Ceranae</I>.
</P><P>• Positive feedback:
</P><P>Besides the LuxI and LuxR(which then form a complex called luxR/AHL) produced by first circuit, the third circuit will also produce LuxI and LuxR, which acts as a positive feedback and strongly enhances the production of killer protein.
</P><P>The picture shows how LuxI will transfer to AHL, which then bind to LuxR to form a dimer:
</P><P>• Circuit condition when <I>Nosema</I> is killed:
</P><P>After <I>Nosema</I> is killed, the sensor will be off and no ptet will be produced.
</P><P>Without ptet, the ptet will not be repressed, and the second circuit will be on, leading to the production of cI. After cI binds to pLux/cI hybrid promoter, the third circuit to be off. After that, the overall circuit will switch to the beginning stage when there is no<I> Nosema</I>.
</P><H2><SPAN class="mw-headline" id="Objectives">Objectives</SPAN></H2><P>1. To know how much time it needs from sensing to killing the <I>Nosema</I> after infection
</P><P>2. To know whether the pathway is effective or not
</P><P>3. To know the range of kill protein concentration (from the minimal concentration which Is effective to kill <I>Nosema</I> to the maximal concentration which will not do harm to the bees)
</P><H2><SPAN class="mw-headline" id="System">System</SPAN></H2><P>It is assumed that AHL is abundant and thus the formation of AHL2LuxR complex is determined only by the concentration of LuxR; CI’s mechanism of binding to the promoter is assumed to act as hill effect form; kill protein will sustain a period of time before it is degraded naturally without any other types of decomposition.
</P><P><B>Equation1</B></P><DIV lang="latex" class="equation">
\frac{d[mRNA_C_I]}{dt}= \frac{1-[ptet]^nptet}{ Kdptet^nptet+ [ptet]^nptet}

\times PoPSconstitutive\times\frac{N}{V}

-KdegmRNA\times [mRNA_C_I]
</DIV><P>Kdptet = dissociation constant of CI
</P><P>nptet = Hill coefficient of ptet
</P><P>PoPSptet = promoter strength of ptet
</P><P>kdegmRNA = degrading constant of sensor promoter mRNA
</P><P>N = number of plasmid in a single cell 
</P><P>V = volume of a cell
</P><P><B> The aim of the equation is to know mRNA_C_I production rate and when it can reach the level to translate the desired concentration.</B></P><P><A href="/Read_more1" title="Read more1">read more1</A></P><P><B>Equation2</B></P><DIV lang="latex" class="equation">
\frac{d[mRNA_L_a_c_I]}{dt}= \frac{1-[ROSoxyR]^nROSoxyR }{ KdROSoxyR ^nROSoxyR + [ROSoxyR]^nROSoxyR} PoPSAhpCp\times\frac{N}{V}-KdegmRNA[mRNA_L_a_c_I]
</DIV><P>KdROSoxyR = dissociation constant ofROSoxyR
</P><P>nROSoxyR = Hill coefficient of ROSoxyR
</P><P>PoPSAhpCp = promoter strength of AhpCp
</P><P>kdegmRNA = degrading constant of sensor promoter mRNA
</P><P>N = number of plasmid in a single cell 
</P><P>V = volume of a cell
</P><P><B> The aim of the equation is to know how AhpCp promoter strength (in PoPS) influences the production of ptet.</B></P><P><A href="/Read_more2" title="Read more2">read more2</A></P><P><B>Equation3:</B></P><DIV lang="latex" class="equation">
\frac{d[mRNA_O_x_y_R]}{dt}=PoPSconstitutive\times\frac{N}{V}-KdegmRNA\times [mRNAOxyR]</DIV><P>PoPSconstitutive= promoter strength of constitutive promoter (J23102)
</P><P>N = number of plasmid in a single cell 
</P><P>V = volume of a cell
</P><P>kdegmRNA = degrading constant of sensor promoter mRNA
</P><P><B> The aim of the equation is to know the production rate ofmRNAoxyR, and choose the proper constitutive promoter for boosting OxyR's concentration.</B></P><P><A href="/Read_more3" title="Read more3">read more3</A></P><P><B>Equation4:</B></P><DIV lang="latex" class="equation">
\frac{d[mRNA_L_u_x_I]}{dt}= \frac{1-[ROSoxyR]^n ROSoxyR }{ Kd ROSoxyR ^n ROSoxyR + [ROSoxyR]^n ROSoxyR I} PoPSAhpCp\times\frac{N}{V}+\frac{1-[ LuxRAHL]^nLuxRAHL}{ KdLuxRAHL ^nLuxRAHL + [LuxRAHL]^nLuxRAHL}\times\frac{1-{1-[CI leak]^nCI}}{ KdCI^nCI+[CI]^nCI}\times PoPSLuxCI\times\frac{N}{V}-KdegmRNA[mRNALuxI]
</DIV><P>KdROSoxyR = dissociation constant ofROSoxyR
</P><P>nROSoxyR = Hill coefficient of ROSoxyR
</P><P>PoPSAhpCp = promoter strength of AhpCp
</P><P>KdLuxRAHL = dissociation constant ofLuxRAHL
</P><P>nLuxRAHL = Hill coefficient of LuxRAHL
</P><P>KdcI = dissociation constant of CI
</P><P>ncI = Hill coefficient of CI
</P><P>PoPSLuxcI = promoter strength of LuxcI hybrid promoter
</P><P>kdegmRNA = degrading constant of sensor promoter Mrna
</P><P>N = number of plasmid in a single cell 
</P><P>V = volume of a cell
</P><P><B> The aim of the equation is to know how AhpCp promoter strength (in PoPS) influences the production of LuxI. </B></P><P><A href="/Read_more4" title="Read more4">read more4</A></P><P><B>Equation5:</B></P><DIV lang="latex" class="equation">
\frac{d[mRNA_L_u_x_R]}{dt}= \frac{1-[ROSoxyR]^n ROSoxyR }{ Kd ROSoxyR ^n ROSoxyR + [ROSoxyR]^n ROSoxyR I} \times PoPSAhpCp\times\frac{N}{V}+\frac{1-[ LuxRAHL]^nLuxRAHL}{ KdLuxRAHL ^nLuxRAHL + [LuxRAHL]^nLuxRAHL}\times\frac{1-{1-[CI leak]^nCI}}{ KdCI^nCI+[CI]^nCI}\times PoPSLuxCI\times\frac{N}{V}-KdegmRNA[mRNALuxR]
</DIV><P>KdROSoxyR = dissociation constant ofROSoxyR
</P><P>nROSoxyR = Hill coefficient of ROSoxyR
</P><P>PoPSAhpCp = promoter strength of AhpCp
</P><P>KdLuxRAHL = dissociation constant ofLuxRAHL
</P><P>nLuxRAHL = Hill coefficient of LuxRAHL
</P><P>KdcI = dissociation constant of CI
</P><P>ncI = Hill coefficient of CI
</P><P>PoPSLuxcI = promoter strength of LuxcI hybrid promoter
</P><P>kdegmRNA = degrading constant of sensor promoter Mrna
</P><P>N = number of plasmid in a single cell 
</P><P>V = volume of a cell
</P><P><B> The aim of the equation is to know how AhpCp promoter strength (in PoPS) influences the production of LuxR. </B></P><P><A href="/Read_more5" title="Read more5">read more5</A></P><P><B>Equation6</B></P><DIV lang="latex" class="equation">
\frac{d[mRNA_k_i_l_l]}{dt}= \frac{1-[ LuxRAHL]^nLuxRAHL}{ KdLuxRAHL ^nLuxRAHL + [LuxRAHL]^nLuxRAHL}\times\frac{1-{1-[CI leak]^nCI}}{ KdCI^nCI+[CI]^nCI}\times PoPSLuxCI\times\frac{N}{V}-KdegmRNA[mRNA_k_i_l_l]
</DIV><P>KdLuxRAHL = dissociation constant ofLuxRAHL
</P><P>nLuxRAHL = Hill coefficient of LuxRAHL
</P><P>KdcI = dissociation constant of CI
</P><P>ncI = Hill coefficient of CI
</P><P>PoPSLuxcI = promoter strength of LuxcI hybrid promoter
</P><P>kdegmRNA = degrading constant of sensor promoter mRNA
</P><P>N = number of plasmid in a single cell 
</P><P>V = volume of a cell
</P><P><B>The aim of the equation is to know mRNA of kill protein production rate and when it can reach the level of the desired concentration. </B></P><P><A href="/Read_more6" title="Read more6">read more6</A></P><P><A href="/To_learn_more_about_how_we_characterize_pLuxCI,_click_here" title="To learn more about how we characterize pLuxCI, click here">to learn more about how we characterize pLuxCI, click here</A></P><P><B>Equation7</B></P><DIV lang="latex" class="equation">
\frac{d[mRNA_t_7]}{dt}= \frac{1-[ROSoxyR]^n ROSoxyR }{ Kd ROSoxyR ^n ROSoxyR + [ROSoxyR]^nROSoxyR }\timesPoPSAhpCp\times\frac{N}{V}\times {a}+ \frac{1-[T7]^nT7}{ Kd T7^nT7 + [T7]^nT7}\times{PoPST7}\times\ frac{N}{V}-KdegmRNA\times [mRNA_T_7]
</DIV><P>KdROSoxyR = dissociation constant ofROSoxyR
</P><P>nROSoxyR = Hill coefficient of ROSoxyR
</P><P>PoPSAhpCp = promoter strength of AhpCp
</P><P>KdT7 = dissociation constant of T7
</P><P>NT7 = Hill coefficient of T7
</P><P>PoPST7 = promoter strength of T7 promoter
</P><P>kdegmRNA = degrading constant of sensor promoter mRNA
</P><P>N = number of plasmid in a single cell 
</P><P>V = volume of a cell
</P><P><B>The aim of the equation is to know mRNA of T7 polymerase production rate and when it can reach the level to translate enough T7 polymerase. </B></P><P><A href="/Read_more7" title="Read more7">read more7</A></P><P><B>Equation8:</B></P><DIV lang="latex" class="equation">
\frac{d[mRNA_P_D_C]}{dt}=\frac{ [T7]^{nT7} }{ KdT7^{nT7}+[T7]^{nY7} }\times{PoPST7}\times\frac{N}{V}-KdegmRNA\times[mRNAPDC]
</DIV>
KdT7 = dissociation constant of T7
<P>NT7 = Hill coefficient of T7
</P><P>PoPST7 = promoter strength of T7 promoter
</P><P>KdegmRNA = degrading constant of sensor promoter mRNA
</P><P>N = number of plasmid in a single cell
</P><P>V = volume of a cell
</P><P><B>The aim of the equation is to knowmRNA of enzyme PDC production rate and when it can reach the level to translate enough PDC. </B></P><P><A href="/Read_more8" title="Read more8">read more8</A></P><P><B>Equation9:</B></P><DIV lang="latex" class="equation">
\frac{d[mRNA_A_D_H]}{dt}=\frac{ [T7]^{nT7} }{ KdT7^{nT7}+[T7]^{nY7} }\times{PoPST7}\times\frac{N}{V}-KdegmRNA\times[mRNAADH]
</DIV><P>KdT7 = dissociation constant of T7
</P><P>NT7 = Hill coefficient of T7
</P><P>PoPST7 = promoter strength of T7 promoter
</P><P>kdegmRNA = degrading constant of sensor promoter mRNA
</P><P>N = number of plasmid in a single cell 
</P><P>V = volume of a cell
</P><P><B>The aim of the equation is to knowmRNA of enzyme ADH production rate and when it can reach the level to translate enough ADH. </B></P><P><A href="/Read_more9" title="Read more9">read more9</A></P><P><B>Equation10 </B></P><DIV lang="latex" class="equation">
\frac{d[CI]}{dt}={RBS}\times{[mRNA_C_I]} -KdegCI\times{[CI]}
</DIV><P>RBS = binding site strength
</P><P>KdegCI = degrading constant of CI
</P><P><B>The aim of the equation is to know the production rate of CI and when it can reach the concentration to repress LuxCI hybrid promoter. </B></P><P><A href="/Read_more10" title="Read more10">read more10</A></P><P><B>Equation11</B></P><DIV lang="latex" class="equation">
\frac{d[ptet]}{dt}={RBS}\times{[mRNA_L_a_c_I]} - {Kdeg ptet} \times{[ptet]}
</DIV><P>RBS = binding site strength
</P><P>Kdegptet = degrading constant of ptet
</P><P><B>The aim of the equation is to known ptet production rate and when it can reach the concentration to repress promoter ptet.</B></P><P><A href="/Read_more11" title="Read more11">read more11</A></P><P>'<I>Equation12</I></P><DIV lang="latex" class="equation">
\frac{d[LuxI]}{dt}={RBS}\times{[mRNA_L_u_x_I]} -KAHL\times{[LuxI]}
 -Kdeg {[LuxI]} \times{[LuxI]}
</DIV><P>RBS = binding site strength
</P><P>KAHL = LuxI to AHL rate constant
</P><P>KdegLuxI = degrading constant of LuxI
</P><P><B>The aim of the equation is to know the production rate of LuxI concerning the LuxIAHL reaction and LuxI degrading. </B></P><P><A href="/Read_more12" title="Read more12">read more12</A></P><P>'<I>Equation13</I></P><DIV lang="latex" class="equation">
\frac{d[LuxR]}{dt}={RBS}\times{[mRNA_L_u_x_R]} – {KmAHL}\times{[AHL]}
-	{Kdeg LuxR} \times{[LuxR]} \times {KmAHL}\{times[AHL]^{2}} \times{[LuxR]}- KdegAHL\times{[AHL]}
</DIV><P>RBS = binding site strength
</P><P>KmAHL =AHL to LuxI rate constant
</P><P>KdegLuxR= degrading constant of LuxR
</P><P>KonAHL = 2AHL + LuxR to (AHL)2/LuxR complex rate constant
</P><P>KdegAHL = degrading constant of AHL
</P><P><B>The aim of the equation is to know the production rate of LuxR concerning the 2AHL + LuxR to (AHL)2/LuxR reaction , AHL to LuxI reaction, and LuxR, AHL degrading. </B></P><P><A href="/Read_more13" title="Read more13">read more13</A></P><P>'<I>Equation14:</I></P><DIV lang="latex" class="equation">
\frac{d[AHL]}{dt}={KAHL}\times{[LuxI]}+2\times KoffAHL \times [AHLLuxR] - 2\times KonAHL \times {[AHL]}^{2}\times {[LuxR]} - KdegAHL\times{[AHL]}
</DIV><P>KAHL = LuxI to AHL rate constant
</P><P>KonAHL = 2AHL + LuxR to (AHL)^2/LuxR complex rate constant
</P><P>KoffAHL = (AHL)2/LuxR complex to 2AHL + LuxR rate constant
</P><P>KdegAHL = degrading constant of AHL
</P><P><B>The aim of the equation is to know the production rate of AHL concerning the LuxIAHL reaction, 2AHL + LuxR to (AHL)2/LuxR reaction, (AHL)2/LuxR complex to 2AHL + LuxR reaction and AHL degrading. </B></P><P><A href="/Read_more14" title="Read more14">read more14</A></P><P><B>Equation17</B></P><DIV lang="latex" class="equation">
\frac{d[ADH]}{dt}=RBS\times{PoPST7effect}\times\frac{N}{V}-{KdegADH}\times{[ADH]}
</DIV><P>
RBS = binding site strength
</P><P>PoPST7 = promoter strength of T7 promoter
</P><P>N = number of plasmid in a single cell
</P><P>V = volume of a cell
</P><P>KdegADH = degrading constant of ADH (Acetaldehyde)
</P><P><B>The aim of the equation is to know ADH production rate and when it can reach the concentration of ethanol pathway equilibrium.</B></P><P><A href="/Read_more17" title="Read more17">read more17</A></P><P><B>Equation18</B></P><DIV lang="latex" class="equation">
\frac{d[AHL]}{dt}=\times KonAHL \times [AHL]^{2}\times{[LuxR]}- {KoffAHL} \times {[AHLLuxR]} - KdegAHL\times{[AHL]}
</DIV><P>KAHL = LuxI to AHL rate constant
</P><P>KonAHL = 2AHL + LuxR to (AHL)2/LuxR complex rate constant
</P><P>KoffAHL = (AHL)2/LuxR complex to 2AHL + LuxR rate constant
</P><P>KdegAHL = degrading constant of AHL
</P><P><B>The aim of the equation is to know the production rate of AHL concerning the LuxIAHL reaction, 2AHL + LuxR to (AHL)2/LuxR reaction, (AHL)2/LuxR complex to 2AHL + LuxR reaction and AHL degrading. </B></P><P><A href="/Read_more18" title="Read more18">read more18</A></P><P><B>Equation19</B></P><DIV lang="latex" class="equation">
\frac{d[kill]}{dt}={RBS}\times{[mRNA_k_i_l_l]} –Kdegkill\times{[kill]}
</DIV><P>RBS = binding site strength
</P><P>Kdegkill = degrading constant of kill
</P><P><B>The aim of the equation is to knowkiller protein production rate and when it can reach the effective concentration to kill <I>Nosema</I>. </B></P><P><A href="/Read_more19" title="Read more19">read more19</A></P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><P>Figure1: This picture shows kill protein concentration to time under pLux/CI hybrid promoter’s control and LuxR, LuxI’s positive feedback. The result indicates that kill protein concentration will reach its effective level and kill <I>Nosema</I> in time (36 hours after <I>Nosema</I> infection)
</P><P>Figure2: This picture is a control group model. It shows that without <I>Nosema</I> infection (ROS concentration=0), ethanol production is low enough to be ignored.
</P><P>Figure3: This is the picture of kill protein concentration to time under promoter CI’s control only(control group)without pLux/CI's regulation.The result shows that kill protein's full expression is too low to kill <I>Nosema</I>.
</P><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><P>Comparing to the ROS-level control group where ROS remains low when there’s no <I>Nosema</I> infection, kill protein production is highly boasted upon sensing <I>Nosema</I>.
</P><P>The result indicates that kill protein concentration will reach its effective level and kill <I>Nosema</I> in time (36 hours after sensing <I>Nosema</I> infection, within the 5-day deadline when<I> Nosema</I> spores proliferates and the bee become incurable and infectious to other bees), at the given ROS changes and the enhanced OxyR expression <A href="/(to_learn_more_about_our_sensor_model,_click_here)" title="(to learn more about our sensor model, click here)">(to learn more about our sensor model, click here)</A>.
</P><H2><SPAN class="mw-headline" id="Circuit_design_and_improvement">Circuit design and improvement</SPAN></H2><P>Upon using pCI to negatively regulate kill protein production, results show that kill protein's full expression while no CI present is too low to kill <I>Nosema</I>. To solve this problem we subsidized CI promoter into pLux/CI promoter for additional positive feedback to boast kill protein’s production. As the modeling results regarding the modified circuit shown, the consequences are positive. <A href="/(click_here_to_learn_more_about_our_circuit)" title="(click here to learn more about our circuit)">(click here to learn more about our circuit)</A></P><H2><SPAN class="mw-headline" id="Parameters:">Parameters:</SPAN></H2><TABLE class="wikitable"><TBODY><TR><TH style="text-align: center;">Model
</TH><TH style="text-align: center;">Parameter
</TH><TH style="text-align: center;">Description
</TH><TH style="text-align: center;">Value
</TH><TH style="text-align: center;">Unit
</TH><TH style="text-align: center;">Reference
</TH></TR><TR><TD rowspan="5" style="text-align: center;"><B>Sensor</B></TD><TD style="text-align: center;">KdegOxyR
</TD><TD>OxyR degrading rate
</TD><TD style="text-align: center;">10<SUP>7</SUP></TD><TD style="text-align: center;">M<SUP>-1</SUP> x min<SUP>-1</SUP></TD><TD rowspan="3">Regulation of the OxyR transcription factor by hydrogen peroxide and the cellular thiol—disulfide status
</TD></TR><TR><TD style="text-align: center;">KdOxyR*
</TD><TD>Dissociation constant of OxyR*
</TD><TD style="text-align: center;">10<SUP>-7.33</SUP></TD><TD style="text-align: center;">M
</TD></TR><TR><TD style="text-align: center;">KOxyR
</TD><TD>OxyR producing rate constant
</TD><TD style="text-align: center;">5.012 x 10<SUP>14</SUP></TD><TD style="text-align: center;">X
</TD></TR><TR><TD style="text-align: center;">nOxyR*
</TD><TD>Hill coefficient of OxyR*
</TD><TD>Ranging from 0.75~3.5,depends on what kind of ROS it react with
</TD><TD style="text-align: center;">X
</TD><TD>OxyR: A Molecular Code for Redox-Related Signaling
</TD></TR><TR><TD style="text-align: center;">N
</TD><TD>Copy number
</TD><TD>887(cells containing 25％ plasmid bearing cells)
</TD><TD>Single piece
</TD><TD><OL><LI>Escherichia coli Plasmid Copy Number Assay
</LI><LI>Improved determination of plasmid copy number using quantitative real-time PCR for monitoring fermentation processes
</LI></OL></TD></TR><TR><TD rowspan="3" style="text-align: center;"> Ethanol
</TD><TD colspan="1" style="text-align: center;">pyruvate
</TD><TD colspan="1" style="text-align: center;"> Initial concentration of pyruvate in MG1655
</TD><TD colspan="1" style="text-align: center;"> 1.18 x 2
</TD><TD colspan="1" style="text-align: center;"> g/L
</TD><TD colspan="1" style="text-align: center;">Expression of pyruvate carboxylase enhances succinate production in Escherichia coli without affecting glucose uptake
</TD></TR><TR><TD style="text-align: center;"> nPDC
</TD><TD> Hill coefficient of PDC
</TD><TD style="text-align: center;">2.1
</TD><TD style="text-align: center;"> -
</TD><TD>Purification, characterization and cDNA sequencing of pyruvate decarboxylase Zygosaccharomyces biporus
</TD></TR><TR><TD style="text-align: center;">nADH</TD><TD>Hill coefficient of ADH</TD><TD style="text-align: center;">at high pH values, 30◦C, n=1; at low temperature, n=3
</TD><TD style="text-align: center;">  -
</TD><TD>Evidence for co-operativity in coenzyme binding to tetrameric Sulfolobus
<P>solfataricus alcohol dehydrogenase and its structural basis: ﬂuorescence,
kinetic and structural studies of the wild-type enzyme and non-co-operative
N249Y mutant
</P></TD></TR><TR><TD rowspan="6"> SEIR(exponential)
</TD><TD colspan="1" style="text-align: center;"> b
</TD><TD colspan="1" style="text-align: center;"> Infection rate constant of <I>Nosema</I> ceranae to the suspected
</TD><TD colspan="1" style="text-align: center;"> 24/75
</TD><TD colspan="1" style="text-align: center;"> Period(days)-1
</TD></TR><TR><TD style="text-align: center;"> r1
</TD><TD> Infection rate constant of K12 to the suspected
</TD><TD style="text-align: center;">3/20
</TD><TD> Period(days)-1
</TD><TD> 1. Environment protection administration executive yuan of R.O.C Medical bacteriology of J.A.T
</TD></TR><TR><TD style="text-align: center;">r2</TD><TD>Infection rate constant of K12 to the latent</TD><TD style="text-align: center;">3/20  </TD><TD>Period(days)-1</TD><TD rowspan="2"> 2. Environment protection administration executive yuan of R.O.C. 3. Medical bacteriology of J.A.T
</TD></TR><TR><TD style="text-align: center;"> e </TD><TD>rate of the latent turns infectious</TD><TD style="text-align: center;">1/4</TD><TD>Period(days)-1
</TD></TR><TR><TD style="text-align: center;"> u </TD><TD> Death rate of the infected </TD><TD style="text-align: center;">1/8</TD><TD>Period(days)-1</TD></TR><TR><TD style="text-align: center;"> k </TD><TD> Rate of intaking capsule </TD><TD style="text-align: center;">24/11</TD><TD>Period(days)-1</TD></TR><TR><TD rowspan="4">SEIR(exponential&amp;linear)
</TD><TD colspan="1" style="text-align: center;"> S
</TD><TD colspan="1" style="text-align: center;"> x(1)
</TD><TD colspan="1" style="text-align: center;"> Amount of total population
</TD></TR><TR><TD style="text-align: center;"> E
</TD><TD style="text-align: center;">x(2)
</TD><TD style="text-align: center;"> Amount of suspected individuals
</TD></TR><TR><TD style="text-align: center;">I</TD><TD style="text-align: center;">x(3)</TD><TD style="text-align: center;">Amount of individuals in the latent period </TD></TR><TR><TD style="text-align: center;"> R </TD><TD style="text-align: center;">x(4)</TD><TD style="text-align: center;">Amount of infected individuals</TD></TR></TBODY></TABLE></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:NYMU-Taipei/Modeling/MainParts">http://2013.igem.org/Team:NYMU-Taipei/Modeling/MainParts</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NYMU-Taipei/Modeling/MainParts" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NYMU-Taipei/Modeling/MainParts" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NYMU-Taipei/Modeling/MainParts&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NYMU-Taipei/Modeling/MainParts&amp;oldid=359199" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>