<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NYMU-Taipei_Experiments_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NYMU-Taipei/Experiments/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NYMU-Taipei/Experiments/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/Experiments/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/Experiments/Protocols&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NYMU-Taipei/Experiments/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NYMU-Taipei/Experiments/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="head"><DIV class="nymu"><DIV id="nymu-taipei">
			National Yang Ming University
		</DIV></DIV><DIV class="nav"><DIV id="nav-bar"><UL id="sdt_menu" class="sdt_menu"><LI><A href="https://2013.igem.org/Team:NYMU-Taipei"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Home</SPAN></SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Team"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Team</SPAN></SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Overview"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Project</SPAN></SPAN></A><DIV class="sdt_box" style="height: 270px; width: 190px;"><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Overview">Overview</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Enter">Entering of Bee.coli</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Inhibition/Prohibition">Prohibiting Sprouting</A><A href="https://2013.igem.org/Team:NYMU-
Taipei/Project/Inhibition">Inhibiting Infection</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Inhibition/Sensor" class="sdt_aWsub">Sensing Nosema</A><A href="https://2013.igem.org/Team:NYMU-
Taipei/Project/Inhibition/Killing" class="sdt_aWsub">Killing Nosema</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Inhibition/Inhibition circuit" class="sdt_aWsub">Circuit</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Kill">Suicide Solution</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/safe">eco-friendly e.coli</A><A href="https://2013.igem.org/Team:NYMU-
Taipei/Project/Inhibition/Id-Nosema">Identifying Nosema</A></DIV></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Experiments"><SPAN class="sdt_wrap"><SPAN class="sdt_link">experiment</SPAN></SPAN></A><DIV class="sdt_box" style="width: 155px;"><A href="https://2013.igem.org/Team:NYMU-Taipei/Experiments/Protocols">Protocols</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Experiment/Wet_Lab">Data</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Experiments/Functional_Assays">Notes</A></DIV></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/Overview"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Modeling</SPAN></SPAN></A><DIV class="sdt_box" style="width: 135px; height: 160px;"><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/Overview">Overview</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/ModSensors">Sensors</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/MainParts">Main Parts</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/Ethanol">Ethanol Model</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/Linear_epidemic_model">Epidemic Model</A></DIV></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Human_Practice"><SPAN class="sdt_wrap"><SPAN class="sdt_link" style="font-size: 25px; padding-top: 5px; line-height: 1em;">Human Practice</SPAN></SPAN></A><DIV class="sdt_box" style="width: 185px; height: 160px;"><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/Overview" style="margin-left: 13px;">Overview</A><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/HumOut" style="margin-left: 13px;">Outreach</A><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/HumCollab" style="margin-left: 13px;">Collaborations</A><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/HumFlows" style="margin-left: 13px;">Flows Built-up</A><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/HumPracticalApp">Practical Applications</A></DIV></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Safety"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Safety</SPAN></SPAN></A></LI></UL></DIV></DIV></DIV></DIV><DIV class="main"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Protocols_for_wet-lab_experiments"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Protocols for wet-lab experiments</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Consistent_tube_labelling"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Consistent tube labelling</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#1.5ml_microcentrifuge_tubes"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">1.5ml microcentrifuge tubes</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#200uL_microcentrifuge_tubes"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">200uL microcentrifuge tubes</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Agar_plates"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">Agar plates</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#PCR"><SPAN class="tocnumber">1.1.4</SPAN><SPAN class="toctext">PCR</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-7"><A href="#Two-day_efficient_cloning_cycle"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Two-day efficient cloning cycle</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Light_Day"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Light Day</SPAN></A></LI><LI class="toclevel-4 tocsection-9"><A href="#The_3-in-1"><SPAN class="tocnumber">1.2.1.1</SPAN><SPAN class="toctext">The 3-in-1</SPAN></A></LI><LI class="toclevel-4 tocsection-10"><A href="#Making_Gel_for_gel_electrophoresis"><SPAN class="tocnumber">1.2.1.2</SPAN><SPAN class="toctext">Making Gel for gel electrophoresis</SPAN></A></LI><LI class="toclevel-4 tocsection-11"><A href="#Gel_electrophoresis"><SPAN class="tocnumber">1.2.1.3</SPAN><SPAN class="toctext">Gel electrophoresis</SPAN></A></LI><LI class="toclevel-4 tocsection-12"><A href="#Liquid_culture"><SPAN class="tocnumber">1.2.1.4</SPAN><SPAN class="toctext">Liquid culture</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Heavy_day"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Heavy day</SPAN></A></LI><LI class="toclevel-4 tocsection-14"><A href="#Plasmid_extraction"><SPAN class="tocnumber">1.2.2.1</SPAN><SPAN class="toctext">Plasmid extraction</SPAN></A></LI><LI class="toclevel-4 tocsection-15"><A href="#Digestion"><SPAN class="tocnumber">1.2.2.2</SPAN><SPAN class="toctext">Digestion</SPAN></A></LI><LI class="toclevel-4 tocsection-16"><A href="#Gel_extraction_and.2For_purification"><SPAN class="tocnumber">1.2.2.3</SPAN><SPAN class="toctext">Gel extraction and/or purification</SPAN></A></LI><LI class="toclevel-4 tocsection-17"><A href="#Ligation"><SPAN class="tocnumber">1.2.2.4</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-4 tocsection-18"><A href="#Transformation"><SPAN class="tocnumber">1.2.2.5</SPAN><SPAN class="toctext">Transformation</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Promoter_Testing"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Promoter Testing</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#General_Gene_Reporter_Assay_protocol"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">General Gene Reporter Assay protocol</SPAN></A></LI><LI class="toclevel-4 tocsection-21"><A href="#Previous-night_Preparation_.28.7E10m-60min.29"><SPAN class="tocnumber">1.3.1.1</SPAN><SPAN class="toctext">Previous-night Preparation (~10m-60min)</SPAN></A></LI><LI class="toclevel-4 tocsection-22"><A href="#Pre-start_.28.7E30mins.29"><SPAN class="tocnumber">1.3.1.2</SPAN><SPAN class="toctext">Pre-start (~30mins)</SPAN></A></LI><LI class="toclevel-4 tocsection-23"><A href="#Culturing_.282.7E3mins.29"><SPAN class="tocnumber">1.3.1.3</SPAN><SPAN class="toctext">Culturing (2~3mins)</SPAN></A></LI><LI class="toclevel-4 tocsection-24"><A href="#Measurement_point_retrieval"><SPAN class="tocnumber">1.3.1.4</SPAN><SPAN class="toctext">Measurement point retrieval</SPAN></A></LI><LI class="toclevel-4 tocsection-25"><A href="#Measuring"><SPAN class="tocnumber">1.3.1.5</SPAN><SPAN class="toctext">Measuring</SPAN></A></LI><LI class="toclevel-3 tocsection-26"><A href="#H202_Promoter_Gene_Reporter_Assay"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">H<SUB>2</SUB>0<SUB>2</SUB> Promoter Gene Reporter Assay</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#Inhibition_of_Spore_Germination_Testing---Mannosidase_Function_Assay"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Inhibition of Spore Germination Testing---Mannosidase Function Assay</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#Purification_of_Nosema_ceranae"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Purification of Nosema ceranae</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#Extraction_of_Nosema_ceranae_Genomic_DNA"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Extraction of Nosema ceranae  Genomic DNA</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#Germination_of_Nosema_ceranae_Spores"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Germination of Nosema ceranae  Spores</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#Points_Calculation_under_Microscope"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">Points Calculation under Microscope</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#keeping_the_bees"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">keeping the bees</SPAN></A></LI><LI class="toclevel-3 tocsection-33"><A href="#the_device"><SPAN class="tocnumber">1.9.1</SPAN><SPAN class="toctext">the device</SPAN></A></LI><LI class="toclevel-3 tocsection-34"><A href="#the_feeder"><SPAN class="tocnumber">1.9.2</SPAN><SPAN class="toctext">the feeder</SPAN></A></LI><LI class="toclevel-2 tocsection-35"><A href="#Encapsulation"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">Encapsulation</SPAN></A></LI><LI class="toclevel-3 tocsection-36"><A href="#Encapsulation_steps"><SPAN class="tocnumber">1.10.1</SPAN><SPAN class="toctext">Encapsulation steps</SPAN></A></LI><LI class="toclevel-3 tocsection-37"><A href="#Encapsulation_machine"><SPAN class="tocnumber">1.10.2</SPAN><SPAN class="toctext">Encapsulation machine</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Protocols_for_wet-lab_experiments"> Protocols for wet-lab experiments </SPAN></H1><H2><SPAN class="mw-headline" id="Consistent_tube_labelling"> Consistent tube labelling </SPAN></H2><P>Labelling tubes and plates in a consistent manner expedites the experimental process, and decreases the time taken to understand what others have done.
</P><H3><SPAN class="mw-headline" id="1.5ml_microcentrifuge_tubes"> 1.5ml microcentrifuge tubes </SPAN></H3><P>These 1.5ml microcentrifuge tubes are used to store extracted plasmids, cleaned up PCR or digested products, and frozen liquid culture.
</P><P>Labelling:
</P><UL><LI> Black pen = Circular DNA, Blue pen = Linear DNA
</LI><LI> Folding part: Plasmid Backbone (&quot;A&quot; for pSB1A2 or &quot;C&quot; for pSB1C3 or &quot;X&quot; for other)
</LI><LI> On top (in this order):
</LI><LI> Item name
</LI><LI> Concentration (in ng/uL written in red)
</LI><LI> &quot;(PCR)&quot; for PCR products; &quot;(XP)&quot;,&quot;(SP)&quot;,&quot;(ES)&quot;,&quot;(EX)&quot; for digestions
</LI></UL><LI> On the side:
</LI><LI> Date (YYYY-MM-DD),
</LI><LI> Person who made it,
</LI><LI> In red pen: the A260/280 and A260/230 ratios.
</LI><LI> For Digestions, write the number of hours (or &quot;overnight&quot;) it was digested.
</LI><H3><SPAN class="mw-headline" id="200uL_microcentrifuge_tubes"> 200uL microcentrifuge tubes </SPAN></H3><P>200uL microcentrifuge tubes (or PCR tubes) are used as temporary storage of various reactions such as digestions, ligations, colony PCR, and temporary liquid cultures.
</P><P>
Labelling:
</P><UL><LI> Folding part: Write the day of the month.
</LI><LI> On top: Write something that can identify what it is (at least one english letter).
</LI><LI> Additionally:
</LI><LI> Colony PCR tubes: Write everything in Black pen
</LI><LI> Colony PCR LB tubes: Write everything in Blue pen
</LI><LI> Digestion: Write the cutting site (e.g. SP,XP)
</LI><LI> Ligation: Make sure you put it in those small bags for 4C overnight ligation after 1uL is taken out for transformation.
</LI></UL><H3><SPAN class="mw-headline" id="Agar_plates"> Agar plates </SPAN></H3><P>Agar plates are for transformations and obtaining single colonies.
</P><P>
Labelling:
</P><UL><LI> On top (agar side):
</LI><LI> Full item names (not just numbers!), date (YYYY-MM-DD), time of incubation start.
</LI><LI> Additionally for transformation plates: Name of person who did the plating.
</LI></UL><LI> On side (still on agar side): type of antibiotic (A/Amp or C/CHL/CM).
</LI><LI> On bottom (non-agar side): Don't write anything here.
</LI><H3><SPAN class="mw-headline" id="PCR"> PCR </SPAN></H3><P>For any kind of PCR, make sure to list:
</P><UL><LI> the things that are PCR'd, and their expected PCR lengths.
</LI><LI> the PCR mix protocol: write the 50uL first, then to multiply to get the new mix total version.
</LI><LI> the PCR time protocol (the 94,[94,55,72]x30-35,72).
</LI><LI> how the gel was loaded. Which wells was loaded with which items.
</LI><LI> Which items were correct and are wrong after the gel run, in as much detail as possible (e.g. was the output the length of a self-ligated vector?)
</LI></UL><H2><SPAN class="mw-headline" id="Two-day_efficient_cloning_cycle"> Two-day efficient cloning cycle </SPAN></H2><P>We used an efficient two day cloning cycle split into a &quot;Heavy&quot; day and a &quot;Light&quot; day.
</P><P>Approximate Light day:
</P><UL><LI> 10am (5min): Plate-&gt;4<SUP>o</SUP>C
</LI><LI> 10:05am (~1hr): 3-in-1, put liquid and plate in fridge.
</LI><LI> Lunch
</LI><LI> After lunch (~1hr): Run Gel
</LI><LI> 5pm: Incubate 
</LI></UL><P>Approximate Heavy day:
</P><UL><LI> 10am (2-3hrs): Plasmid extraction &amp; Digestion
</LI><LI> Lunch (while digesting)
</LI><LI> After lunch (2-3hrs): Gel Extraction/Purification + Ligation + Transformation
</LI><LI> 5pm: Incubate plate 
</LI></UL><H3><SPAN class="mw-headline" id="Light_Day"> Light Day </SPAN></H3><P>The light day consists of Colony PCR and liquid culture of colonies transformed from a previous day.
</P><H4><SPAN class="mw-headline" id="The_3-in-1"> The 3-in-1 </SPAN></H4><P>The 3-in-1 protocol consists of a combination of three things:
</P><UL><LI> Colony PCR
</LI><LI> Liquid culture
</LI><LI> 2nd time plate 
</LI></UL><P>Time needed: [10min + 1min/colony]
</P><P><B>Preparation</B>
0. First, count the number of colonies you want to check.
</P><P>1a. Make the Colony PCR mix (we use <A href="http://www.thermoscientificbio.com/pcr-enzymes-master-mixes-and-reagents/dreamtaq-dna-polymerase/" class="external text" rel="nofollow">Thermo' DreamTaq</A>) with the mix amount slightly modified:
</P><TABLE class="wikitable"><TBODY><TR><TH> Item </TH><TH> uL
</TH></TR><TR><TD> Template </TD><TD> 1
</TD></TR><TR><TD> FP (VR) + RP (VF2) </TD><TD> 2
</TD></TR><TR><TD> dNTP (2mM) </TD><TD> 2
</TD></TR><TR><TD> 10x DreamTaq buffer </TD><TD> 5
</TD></TR><TR><TD> Taq Polymerase </TD><TD> 0.2
</TD></TR><TR><TD> ddH<SUB>2</SUB>0 </TD><TD> 39.8
</TD></TR><TR><TD> Total </TD><TD> 50
</TD></TR></TBODY></TABLE><P>We aliquoted 15uL of PCR mix for each colony.
</P><P>1b. Aliquot 20uL of LB medium in a PCR tube for each colony. <B>Put on the LB rack</B></P><P>1c. Take out new plates from the fridge and divide them into smaller sections (upto 24). This is the second-time plate. <B>Make sure to write the date on it. Cross out in red pen any sections you know is wrong after the Colony PCR check</B></P><P><B>Steps</B>
In this order:
</P><OL><LI> Select a colony using a tip or toothpick.
</LI><LI> Dip it in a PCR tube and swirl it around.
</LI><LI> Use the same tip and dip it in the LB culture and swirl it around.
</LI><LI> Use the same tip and draw on the 2nd time plate.
</LI></OL><P>PCR run protocol:
</P><TABLE class="wikitable"><TBODY><TR><TH> Temperature </TH><TH> Time
</TH></TR><TR><TD> 94<SUP>o</SUP>C </TD><TD> 60s
</TD></TR><TR><TD> 94<SUP>o</SUP>C </TD><TD> 15s </TD><TD rowspan="3"> 30-35 cycles
</TD></TR><TR><TD> 55<SUP>o</SUP>C </TD><TD> 20s
</TD></TR><TR><TD> 72<SUP>o</SUP>C </TD><TD> 1kb/min + 5-10s
</TD></TR><TR><TD> 72<SUP>o</SUP>C </TD><TD> 300s
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Making_Gel_for_gel_electrophoresis"> Making Gel for gel electrophoresis </SPAN></H4><OL><LI> Make either 100ml or 200ml of gel at at time and select a relevant percentage (e.g. 1%, 1.5%, or 2%)
</LI><LI> Measure out the relevant percentage in agarose (e.g. 2g of agarose for 100ml of 2% gel).
</LI><LI> Fill it up with 1xTAE buffer.
</LI><LI> Microwave on low, constantly taking it out every so often to swirl and mix it. Keep microwaving until clear. <B>Make sure it is completely clear</B></LI><LI> Cool to a temperature that is still hot but still can be held in your hand. If the gel gets too cold and starts to harden, <I>start microwaving again until clear</I>.
</LI><LI> Add 5uL of Safe-seeing dye for every 100ml of gel after cooled to the correct tempearature. Mix it well by swirling
</LI><LI> Pour it onto the molds quickly. Put a cover on it to block out the light.
</LI><LI> Wait at least 1/2hr until using it.
</LI><LI> Store in 4<SUP>o</SUP> and away from light.
</LI></OL><H4><SPAN class="mw-headline" id="Gel_electrophoresis"> Gel electrophoresis </SPAN></H4><OL><LI> Select a relevantÂ % agarose gel based on your own experience (general guide: all &lt;1000: 2%, all &gt;2000: 1% <A href="http://www.lifetechnologies.com/tw/zt/home/references/ambion-tech-support/rna-electrophoresis-markers/general-articles/gel-electrophoresis-tables.html" class="external text" rel="nofollow">guide</A>)
</LI><LI> Load 5uL from each tube of Colony PCR, mix it with 1uL of 6x DNA Dye and put it in a well.
</LI><LI> Load 3uL of marker into a well.
</LI><LI> Run in the 13x13cm box at 90V or 100V and 400mA for the desired amount of time.
</LI><LI> View in the gel viewer machine.
</LI></OL><H4><SPAN class="mw-headline" id="Liquid_culture"> Liquid culture </SPAN></H4><P>After confirming which colonies are correct from running the colony PCR on gel and deciding which ones to culture in liquid:
</P><OL><LI> Aliquot 6-15ml of LB in a 15ml centrifuge tube or 10ml of LB in a 50ml centrifuge tube for each colony.
</LI><LI> Add the 20uL of LB containing the correct colony from the 3-in-1 into the liquid.
</LI><LI> Add the relevant amount of antibiotics into the liquid
</LI><LI> 0.5uL of Ampicillin (100mg/ml) per 1ml of LB
</LI><LI> 0.5uL of Chloramphenicol (50mg/ml) per 1ml of LB
</LI><LI> Split the LB in the 15ml centrifuge tubes into multiple 15ml centrifuge tubes containing a maximum of 3ml each.
</LI><LI> Grow overnight at 37<SUP>o</SUP>C at 180rpm in the incubator.
</LI></OL><H3><SPAN class="mw-headline" id="Heavy_day"> Heavy day </SPAN></H3><P>The heavy day consists of extracting a previously grown plasmid, digesting it, and ligating digestions together, then transforming.
</P><H4><SPAN class="mw-headline" id="Plasmid_extraction"> Plasmid extraction </SPAN></H4><P>We use GeneArt's plasmid extraction kit. We followed the protocol with the addition of warming the relevant amount of EB to 60-70<SUP>o</SUP>C before elution. Elute with 50uL.
</P><H4><SPAN class="mw-headline" id="Digestion"> Digestion </SPAN></H4><P>Thermo's fast-digest enzymes:
</P><TABLE class="wikitable" style="text-align:center;"><TBODY><TR><TH> Item </TH><TH> XP from plasmid </TH><TH> XP from PCR
</TH></TR><TR><TD> DNA </TD><TD> 1ug or 10uL </TD><TD> 10uL
</TD></TR><TR><TD> 10x fast-digest (FD) buffer </TD><TD> 2 (green) </TD><TD> 2 (white)
</TD></TR><TR><TD> Thermo XbaI </TD><TD colspan="2"> 0.5
</TD></TR><TR><TD> Thermo PstI </TD><TD colspan="2"> 0.5
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> to 20uL </TD><TD> 6
</TD></TR><TR><TD> Total </TD><TD colspan="2"> 20
</TD></TR></TBODY></TABLE><P>Theoretically 10mins should be enough for this digestion. Over 1hr is better.
</P><P>NEB enzymes:
</P><TABLE class="wikitable" style="text-align:center;"><TBODY><TR><TH> Item </TH><TH> SP </TH><TH> EX </TH><TH> ES
</TH></TR><TR><TD> DNA </TD><TD colspan="3"> 1ug or 10uL
</TD></TR><TR><TD> 10x Buffer </TD><TD> 2 (Cutsmart) </TD><TD colspan="2"> 2 (EcoRI)
</TD></TR><TR><TD> NEB Enzyme1 </TD><TD colspan="3"> 0.5
</TD></TR><TR><TD> NEB Enzyme2 </TD><TD colspan="3"> 0.5
</TD></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD colspan="3"> to 20uL
</TD></TR><TR><TD> Total </TD><TD colspan="3"> 20
</TD></TR></TBODY></TABLE><P>Digest for at least 1hr. Over 2hr is better. EcoRI doesn't have star activity when cut this way (even overnight)
</P><H4><SPAN class="mw-headline" id="Gel_extraction_and.2For_purification"> Gel extraction and/or purification </SPAN></H4><P>We run Insert digests (ES, XP) on a gel and then use GeneArt's gel extraction kit. We directly use GeneArt's PCR purification kit for Vector digests (EX,SP). The modifications in the protocol include:
</P><UL><LI> Warm EB to 60-70<SUP>o</SUP>C before elution.
</LI><LI> Always use the Gel (sequencing) protocol for gel extraction. It's just essentially Wash buffer twice instead of W1 then Wash.
</LI></UL><H4><SPAN class="mw-headline" id="Ligation"> Ligation </SPAN></H4><P>We use Thermo's and NEB's T4 Ligase:
</P><TABLE class="wikitable"><TBODY><TR><TH> Item </TH><TH> amount
</TH></TR><TR><TD> Vector </TD><TD rowspan="3"> 8.5uL, total of approximately 100ng of DNA.
</TD></TR><TR><TD> Insert
</TD></TR><TR><TD> ddH<SUB>2</SUB>O
</TD></TR><TR><TD> 10x Ligase Buffer </TD><TD> 1
</TD></TR><TR><TD> T4 Ligase </TD><TD> 0.5
</TD></TR><TR><TD> Total </TD><TD> 10
</TD></TR></TBODY></TABLE><P>Incubate at room temperature for 17-30 minutes then transform 1uL THEN <B>put the remaining amount in a small bag and put in 4<SUP>o</SUP>C overnight</B> incase the transformation fails and retransformation is required.
</P><H4><SPAN class="mw-headline" id="Transformation"> Transformation </SPAN></H4><P>We use self-made <I>E. coli</I> K-12 MG1655 competent cells most of the time (it requires the long protocol [upto 3hrs], but only requires 11hrs to see colonies), or GeneArt's DG5Î± competent cells if there's only time for the short protocol [~15min]. MG1655 competent cells should not be refrozen.
</P><TABLE class="wikitable"><TBODY><TR><TH> Step </TH><TH> Short </TH><TH> Long
</TH></TR><TR><TD colspan="3"> Add 1uL of plasmid or ligation mix to competent cells (20-100uL of MG1655, 15-25uL of DH5Î±)
</TD></TR><TR><TD> Put mixture on ice for </TD><TD> 5mins </TD><TD> 30-60mins
</TD></TR><TR><TD colspan="3"><B>Heat shock at 42<SUP>o</SUP>C for 45 seconds.</B> (very important step)
</TD></TR><TR><TD> Add prewarmed LB and incubate at 37<SUP>o</SUP>C, 180rpm for 1-1.5hr </TD><TD> - </TD><TD> 200uL LB
</TD></TR><TR><TD> Plate it on a relevant antibiotic plate </TD><TD> plate it all </TD><TD> 200uL on one plate, 20uL on another
</TD></TR><TR><TD colspan="3"> Incubate plate at 37<SUP>o</SUP>C overnight
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Promoter_Testing"> Promoter Testing </SPAN></H2><P>These are the protocols we used for promoter testing
</P><H3><SPAN class="mw-headline" id="General_Gene_Reporter_Assay_protocol"> General Gene Reporter Assay protocol </SPAN></H3><P>This is our general setup for gene reporting assays.
</P><H4><SPAN class="mw-headline" id="Previous-night_Preparation_.28.7E10m-60min.29"> Previous-night Preparation (~10m-60min) </SPAN></H4><P>Previous night:
</P><UL><LI> Culture overnight (~16hrs) at 37C and shaking at 180rpm 2-3ml of single colonies of things to be measured.
</LI></UL><P>Preparation (can be done the previous night too):
</P><UL><LI> Determine the number of measurement points = number of items x number of replicates
</LI><LI> Prepare that amount of 15ml round-bottom tubes and 1.5uL microcentrifuge tubes.
</LI><LI> Add 2ml of LB + the relevant antibiotics in each round-bottom tube.
</LI></UL><H4><SPAN class="mw-headline" id="Pre-start_.28.7E30mins.29"> Pre-start (~30mins) </SPAN></H4><P>Preparation:
</P><UL><LI> Prewarm the LB+antibioics to the temperature to test at (e.g. 20mins prewarming in the incubator takes it up to 37C).
</LI><LI> Get some ice buckets and pre-cool down the 1xPBS
</LI></UL><H4><SPAN class="mw-headline" id="Culturing_.282.7E3mins.29"> Culturing (2~3mins) </SPAN></H4><OL><LI> Add any relevant chemicals to the prewarmed round-bottom tubes containing 2ml LB.
</LI><LI> Dilute the overnight culture to an OD600 of about 0.0325
</LI><LI> Incubate at the required conditions (usually 37C and 180rpm).
</LI><LI> Repeat for every replicate.
</LI></OL><H4><SPAN class="mw-headline" id="Measurement_point_retrieval"> Measurement point retrieval </SPAN></H4><P>For each culture tube:
</P><OL><LI> Take out the relevant culture and put it on ice
</LI><LI> Extract 1.3ml from the culture tube and put it in an 1.5uL microcentrifuge tube
</LI><LI> Centrifuge for 1min at 16.2g (13.2rpm on tabletop centrifuge)
</LI><LI> WASH: Tip out the supernatant and add 650uL of 1xPBS, resuspend, then centrifuge again.
</LI><LI> WASH again.
</LI><LI> Carefully remove and discard all the supernatant, resuspend in 1.3ml of 1xPBS.
</LI><LI> Store at 4C until measuring
</LI></OL><H4><SPAN class="mw-headline" id="Measuring"> Measuring </SPAN></H4><P>Requires black plates for measuring fluorescence and transparent plates for measuring OD600. For each measurement point, load 200uL each into 3 black wells and 3 transparent wells (i.e. 3 technical replicates each for fluorescence and OD600). Measure the fluorescence on the black plate with the settings depending on your machine. We use excitation/emission wavelength of 485/525 (100ms per measurement). Measure the OD600 on the transparent plate.
</P><H3><SPAN class="mw-headline" id="H202_Promoter_Gene_Reporter_Assay"> H<SUB>2</SUB>0<SUB>2</SUB> Promoter Gene Reporter Assay </SPAN></H3><P>For measuring promoters under different H202 stress concentrations: (e.g.: 0.01mM, 0.05mM, 0.1mM, 0.5mM, 1mM, 2mM, 2.5mM, 5mM).
</P><UL><LI> Add the H202 just before adding the bacteria.
</LI></UL><P>For reporting assays using 2ml round-bottom tubes:
</P><UL><LI> First serially dilute the relevant concentrations from the stock (35%) everytime the experiments start.
</LI><LI> Add 13.4uL of the relevant concentration of H202 into the tube.
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> 35% </TH><TH> 3.5% </TH><TH> 1.75% </TH><TH> 0.875% </TH><TH> 0.7% </TH><TH> 0.35% </TH><TH> 0.175% </TH><TH> 0.035% </TH><TH> 0.0035% </TH><TH> 0.00175%
</TH></TR><TR><TD> 5mM    </TD><TD> 13.4
</TD></TR><TR><TD> 2.5mM  </TD><TD> 13.4
</TD></TR><TR><TD> 2mM    </TD><TD> 13.4
</TD></TR><TR><TD> 1mM    </TD><TD> 13.4
</TD></TR><TR><TD> 0.5mM  </TD><TD> 13.4
</TD></TR><TR><TD> 0.1mM  </TD><TD> 13.4
</TD></TR><TR><TD> 0.05mM </TD><TD> 13.4
</TD></TR><TR><TD> 0.01mM </TD><TD> 13.4
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Inhibition_of_Spore_Germination_Testing---Mannosidase_Function_Assay">Inhibition of Spore Germination Testing---Mannosidase Function Assay</SPAN></H2><P>We define the germination degree as follows:
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B> 1 point:A spore begins its budding reproduction.</B></DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B> 2 points:A new spore is ccompletely germinanated from the old one.</B></DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B> 3 points:A cluster of duplicated spores is formed.</B></DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B> 4 points:A Spore extrude a polar filament.</B></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>And so far we have accomplished experiments of the first two groups, discovering that spores underwent the germination procesure got much more points than the control group. Next we will compare the points of group 3 and group 4 to see if Bee. coli can inhibit the germination of N. ceranae.
</P><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B>The midguts of healthy bees were yellow.</B></DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B>The midguts of bees that infected with N. ceranae were white.</B></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Purification_of_Nosema_ceranae">Purification of <I>Nosema ceranae</I></SPAN></H2><H2><SPAN class="mw-headline" id="Extraction_of_Nosema_ceranae_Genomic_DNA">Extraction of <I>Nosema ceranae</I>  Genomic DNA </SPAN></H2><H2><SPAN class="mw-headline" id="Germination_of_Nosema_ceranae_Spores">Germination of <I>Nosema ceranae</I>  Spores</SPAN></H2><H2><SPAN class="mw-headline" id="Points_Calculation_under_Microscope">Points Calculation under Microscope</SPAN></H2><H2><SPAN class="mw-headline" id="keeping_the_bees">keeping the bees</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Bees' keepers keep bees in hives.</DIV></DIV></DIV><P>Beesâ keepers are used to keep bees in the beesâ hives, which are cells made in propolis. Living in the hives, bees used to feed themselves by collecting the nectar.
Our project required bees to live outside the hives for controlling the intake of their food. 
So we had to build up a device to keep bees alive. Our device is kept at 32 Celcius, and 70% humidity
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">We use our device to keep bees in the lab and control their food.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="the_device">the device</SPAN></H3><P>The device is composed of an eppendorf and a bottle. The bottle is the space bees fool around. There are many tiny pores on top of the bottle for exchanging the air. 
</P><H3><SPAN class="mw-headline" id="the_feeder">the feeder</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">This is the feeder we used.</DIV></DIV></DIV><P>We use eppendorf as the feeder. We insert an special straw( a straw stuffed with cotton in order to make the fluid inside flows out slowly by capillarity) to let the bees drink the fluid inside. The fluid is 50% sucrose sugar water, which is the food of bees. We place the engineered eppendorf inside the bottle for bees to eat the food inside it. Sometimes we replaced the eppendorf with a sugar cube. The high concentration of sugar may stimulates beesâ appetite.
</P><H2><SPAN class="mw-headline" id="Encapsulation">Encapsulation</SPAN></H2><P>We used encapsulation to protect our <I>Bee. coli</I> from the attack of immune system and digest system of bees.
</P><H3><SPAN class="mw-headline" id="Encapsulation_steps">Encapsulation steps</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">This is alginate beads.</DIV></DIV></DIV><OL><LI>Mix our <I>Bee. coli</I> with the same volume of 1.5% alginate solution
</LI><LI>Use the encapsulation machine to spray the solution into the 0.2M calcium chloride. And wait for the alginate beads to precipitate. 
</LI><LI>Wash by PBS buffer
</LI></OL><H3><SPAN class="mw-headline" id="Encapsulation_machine">Encapsulation machine</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">This is the encapsulation machine we used.</DIV></DIV></DIV><P>To control the size of alginate beads, we use the encapsulation machine to help us spray the solution into the calcium chloride solution. The encapsulation machine is pneumatic, that is, we can control the beadsâ size by adjusting the flow rate of the gas.
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:NYMU-Taipei/Experiments/Protocols">http://2013.igem.org/Team:NYMU-Taipei/Experiments/Protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NYMU-Taipei/Experiments/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NYMU-Taipei/Experiments/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NYMU-Taipei/Experiments/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NYMU-Taipei/Experiments/Protocols&amp;oldid=360787" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>