<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NYMU-Taipei_Modeling_ModSensors"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NYMU-Taipei/Modeling/ModSensors">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NYMU-Taipei/Modeling/ModSensors&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/Modeling/ModSensors&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/Modeling/ModSensors&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NYMU-Taipei/Modeling/ModSensors" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NYMU-Taipei/Modeling/ModSensors</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="head"><DIV class="nymu"><DIV id="nymu-taipei">
			National Yang Ming University
		</DIV></DIV><DIV class="nav"><DIV id="nav-bar"><UL id="sdt_menu" class="sdt_menu"><LI><A href="https://2013.igem.org/Team:NYMU-Taipei"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Home</SPAN></SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Team"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Team</SPAN></SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Overview"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Project</SPAN></SPAN></A><DIV class="sdt_box" style="height: 270px; width: 190px;"><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Overview">Overview</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Enter">Entering of Bee.coli</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Inhibition/Prohibition">Prohibiting Sprouting</A><A href="https://2013.igem.org/Team:NYMU-
Taipei/Project/Inhibition">Inhibiting Infection</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Inhibition/Sensor" class="sdt_aWsub">Sensing Nosema</A><A href="https://2013.igem.org/Team:NYMU-
Taipei/Project/Inhibition/Killing" class="sdt_aWsub">Killing Nosema</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Inhibition/Inhibition circuit" class="sdt_aWsub">Circuit</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Kill">Suicide Solution</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/safe">eco-friendly e.coli</A><A href="https://2013.igem.org/Team:NYMU-
Taipei/Project/Inhibition/Id-Nosema">Identifying Nosema</A></DIV></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Experiments"><SPAN class="sdt_wrap"><SPAN class="sdt_link">experiment</SPAN></SPAN></A><DIV class="sdt_box" style="width: 155px;"><A href="https://2013.igem.org/Team:NYMU-Taipei/Experiments/Protocols">Protocols</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Experiment/Wet_Lab">Data</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Experiments/Functional_Assays">Notes</A></DIV></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/Overview"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Modeling</SPAN></SPAN></A><DIV class="sdt_box" style="width: 135px; height: 160px;"><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/Overview">Overview</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/ModSensors">Sensors</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/MainParts">Main Parts</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/Ethanol">Ethanol Model</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/Linear_epidemic_model">Epidemic Model</A></DIV></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Human_Practice"><SPAN class="sdt_wrap"><SPAN class="sdt_link" style="font-size: 25px; padding-top: 5px; line-height: 1em;">Human Practice</SPAN></SPAN></A><DIV class="sdt_box" style="width: 185px; height: 160px;"><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/Overview" style="margin-left: 13px;">Overview</A><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/HumOut" style="margin-left: 13px;">Outreach</A><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/HumCollab" style="margin-left: 13px;">Collaborations</A><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/HumFlows" style="margin-left: 13px;">Flows Built-up</A><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/HumPracticalApp">Practical Applications</A></DIV></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Safety"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Safety</SPAN></SPAN></A></LI></UL></DIV></DIV></DIV></DIV><DIV class="main"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Sensors"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Sensors</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Backgrounds"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Backgrounds</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Objectives"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Objectives</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#System"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">System</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Results"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Discussion"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Parameters"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Parameters</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Sensors">Sensors</SPAN></H1><H2><SPAN class="mw-headline" id="Backgrounds">Backgrounds</SPAN></H2><P>When <I>Nosema</I> enters the bees, it will trigger bees’ ROS (Reactive Oxygen Species) production, which in turn launches E. coli’s OxyR production. After that ROS and OxyR will form a complex and binds to AhpCp promoter, which function as a sensor/ switch to have the whole circuit opened when <I>Nosema</I> exists, and closed when <I>Nosema</I> is killed..
To increase the sensitivity of this switch, we add an additional circuit expressing OxyR regulated by a constitutive promoter to enhance the basal OxyR expression in <I>E. coli</I>.
</P><P>The picture shows how oxyR and ROS form a complex:
</P><H2><SPAN class="mw-headline" id="Objectives">Objectives</SPAN></H2><OL><LI>To know the lag time between sensing the invasion and the production of the killing protein.
</LI><LI>The minimal Oxidative stress that can activate the switch by choosing the proper constitutive promoter for boosting OxyR's concentration
</LI><LI>To know the relation between ROS input and the AhpCp promoter open strength.
</LI><LI>To see to which degree will AhpCp promoter influence the production of LuxI, LuxR, and TetR, which will influence the second and the third circuit.
</LI></OL><H2><SPAN class="mw-headline" id="System">System</SPAN></H2><P>It is assumed that the concentration of ROS in bees is relative to the severity of <I>Nosema</I> infection, and that OxyR is so abundant that once ROS appears, it will soon bind to ROS to form OxyR* and reach equilibrium. 
</P><P><B>Equation1:</B></P><DIV lang="latex" class="equation">
\frac{d[mRNA_O_x_y_R]}{dt}=PoPSconstitutive\times\frac{N}{V}-KdegmRNA[mRNAOxyR]
</DIV><P>PoPSconstitutive = promoter strength of constitutive promoter (J23102)
</P><P>N = number of plasmid in a single cell 
</P><P>V = volume of a cell
</P><P><B>The aim of the equation is to know the production rate of mRNAOxyR, and choose the proper constitutive promoter for boosting OxyR's concentration.</B></P><P><A href="/Know_more1" title="Know more1">know more1</A></P><P><B>Equation2:</B></P><DIV lang="latex" class="equation">
\frac{d[OxyR]}{dt}=RBS\times[mRNAOxyR]-KdegOxyR[OxyR]
</DIV><P>RBS = binding site strength
</P><P>KdegOxyR = degrading constant of OxyR
</P><P><B>The aim of the equation is to know the production rate of OxyR and when it can reach the concentration activating AhpCp promoter, which can in turn be deduced to get the lag time between sensing the invasion and the production of the killing protein.</B></P><P><A href="/Know_more2" title="Know more2">know more2</A></P><P><B>Equation3:</B></P><DIV lang="latex" class="equation">
[OxyR*]=KOxyR*\times[ROS]\times[OxyR]
</DIV><P>KOxyR* = constant rate of ROS+OxyR→OxyR*
</P><P><B>The aim of the equation is to know the concentration of OxyR* given the concentration of ROS and OxyR, which will know the relation between ROS input and the AhpCp promoter open time.</B></P><P><A href="/Know_more3" title="Know more3">know more3</A></P><P><B>Equation4:</B></P><DIV lang="latex" class="equation">
\frac{d[mRNA_L_a_c_I]}{dt}=\frac{ [mRNALuxI] }{dt}=\frac{ [mRNALuxR] }{dt}
=\frac{ {[OxyR*]} ^{{n}^{OxyR*}}}{{{KdOxyR*}^{{n}^{OxyR*}}}+{[OxyR*]}^{{n}^{OxyR*}}}\times{PoPsAhpCp}\times\frac{N}{V}
</DIV><P>PoPSconstitutive = promoter strength of constitutive promoter
</P><P>N = number of plasmid in a single cell 
</P><P>V = volume of a cell
</P><P>The aim of the equation is to know how AhpCp promoter strength (in PoPS) influences the production of TetR, LuxI, and LuxR.
</P><P><A href="/Know_more4" title="Know more4">know more4</A></P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><P>This picture predicts the buildup of concentration of mRNAOxyR, mRNACI, OxyR, CI will reach in a newborn Bee. coli (without <I>Nosema</I> invasion). 
</P><P>Since naturally produced oxyR in Bee.coli (our E.coli) is not sufficient, we add a constitutive promoter before the oxyR-producing gene to increase oxyR concentration to reach the effective level (0.005M). The picture shows that after adding constitutive promoter, oxyR concentration can reach the effective level to activate ROS/oxyR complex-induced promoter 50 hours after <I>Nosema</I> infection.
</P><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><P>The results shown in killing protein and ethanol production means that the adjusted OxyR concentration is high enough to swiftly form a complex after ROS is heighten, therefore turning on the switch. 
</P><P>In reality, the final centration of OxyR can reach much more higher than we had expected after a longer period of time. The inaccuracy will be discussed below.
</P><P>The reason why the actual result of OxyR concentration differing from the expected one may be that we did not take the negative feedback of OxyR production.
</P><P>This picture shows OxyR cycle in E. coli:
</P><P>The top molecule is reduced form of OxyR, whose Cys199 to sulfoxide will be oxidized by hydrogen peroxide (the oxidative reagent as well as a kind of ROS).
</P><P>After the water exclusion and disulfide bond formation, the reduced form of oxy turns into the oxidized one. Oxidized OxyR is an activator of target promoter, which stimulates downstream gene expression.
However, Oxidized form of OxyR protein is transformed into reduced OxyR protein (inactivated) by thioredoxin reductase (GSH). Reduced OxyR protein (the top molecule)provides the autorepression of OxyR gene (a negative feedback).
</P><H2><SPAN class="mw-headline" id="Parameters">Parameters</SPAN></H2><TABLE class="wikitable"><TBODY><TR><TH style="text-align: center;">Model
</TH><TH style="text-align: center;">Parameter
</TH><TH style="text-align: center;">Description
</TH><TH style="text-align: center;">Value
</TH><TH style="text-align: center;">Unit
</TH><TH style="text-align: center;">Reference
</TH></TR><TR><TD rowspan="5" style="text-align: center;"><B>Sensor</B></TD><TD style="text-align: center;">KdegOxyR
</TD><TD>OxyR degrading rate
</TD><TD style="text-align: center;">10<SUP>7</SUP></TD><TD style="text-align: center;">M<SUP>-1</SUP> x min<SUP>-1</SUP></TD><TD rowspan="3">Regulation of the OxyR transcription factor by hydrogen peroxide and the cellular thiol—disulfide status
</TD></TR><TR><TD style="text-align: center;">KdOxyR*
</TD><TD>Dissociation constant of OxyR*
</TD><TD style="text-align: center;">10<SUP>-7.33</SUP></TD><TD style="text-align: center;">M
</TD></TR><TR><TD style="text-align: center;">KOxyR
</TD><TD>OxyR producing rate constant
</TD><TD style="text-align: center;">5.012 x 10<SUP>14</SUP></TD><TD style="text-align: center;">X
</TD></TR><TR><TD style="text-align: center;">nOxyR*
</TD><TD>Hill coefficient of OxyR*
</TD><TD>Ranging from 0.75~3.5,depends on what kind of ROS it react with
</TD><TD style="text-align: center;">X
</TD><TD>OxyR: A Molecular Code for Redox-Related Signaling
</TD></TR><TR><TD style="text-align: center;">N
</TD><TD>Copy number
</TD><TD>887(cells containing 25％ plasmid bearing cells)
</TD><TD>Single piece
</TD><TD><OL><LI>Escherichia coli Plasmid Copy Number Assay
</LI><LI>Improved determination of plasmid copy number using quantitative real-time PCR for monitoring fermentation processes
</LI></OL></TD></TR></TBODY></TABLE></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:NYMU-Taipei/Modeling/ModSensors">http://2013.igem.org/Team:NYMU-Taipei/Modeling/ModSensors</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NYMU-Taipei/Modeling/ModSensors" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NYMU-Taipei/Modeling/ModSensors" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NYMU-Taipei/Modeling/ModSensors&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NYMU-Taipei/Modeling/ModSensors&amp;oldid=358612" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>