<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NYMU-Taipei_Project_Inhibition_Id-Nosema"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NYMU-Taipei/Project/Inhibition/Id-Nosema">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NYMU-Taipei/Project/Inhibition/Id-Nosema&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/Project/Inhibition/Id-Nosema&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/Project/Inhibition/Id-Nosema&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NYMU-Taipei/Project/Inhibition/Id-Nosema" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NYMU-Taipei/Project/Inhibition/Id-Nosema</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="head"><DIV class="nymu"><DIV id="nymu-taipei">
			National Yang Ming University
		</DIV></DIV><DIV class="nav"><DIV id="nav-bar"><UL id="sdt_menu" class="sdt_menu"><LI><A href="https://2013.igem.org/Team:NYMU-Taipei"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Home</SPAN></SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Team"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Team</SPAN></SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Overview"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Project</SPAN></SPAN></A><DIV class="sdt_box" style="height: 270px; width: 190px;"><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Overview">Overview</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Enter">Entering of Bee.coli</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Inhibition/Prohibition">Prohibiting Sprouting</A><A href="https://2013.igem.org/Team:NYMU-
Taipei/Project/Inhibition">Inhibiting Infection</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Inhibition/Sensor" class="sdt_aWsub">Sensing Nosema</A><A href="https://2013.igem.org/Team:NYMU-
Taipei/Project/Inhibition/Killing" class="sdt_aWsub">Killing Nosema</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Inhibition/Inhibition circuit" class="sdt_aWsub">Circuit</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/Kill">Suicide Solution</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Project/safe">eco-friendly e.coli</A><A href="https://2013.igem.org/Team:NYMU-
Taipei/Project/Inhibition/Id-Nosema">Identifying Nosema</A></DIV></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Experiments"><SPAN class="sdt_wrap"><SPAN class="sdt_link">experiment</SPAN></SPAN></A><DIV class="sdt_box" style="width: 155px;"><A href="https://2013.igem.org/Team:NYMU-Taipei/Experiments/Protocols">Protocols</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Experiment/Wet_Lab">Data</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Experiments/Functional_Assays">Notes</A></DIV></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/Overview"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Modeling</SPAN></SPAN></A><DIV class="sdt_box" style="width: 135px; height: 160px;"><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/Overview">Overview</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/ModSensors">Sensors</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/MainParts">Main Parts</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/Ethanol">Ethanol Model</A><A href="https://2013.igem.org/Team:NYMU-Taipei/Modeling/Linear_epidemic_model">Epidemic Model</A></DIV></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Human_Practice"><SPAN class="sdt_wrap"><SPAN class="sdt_link" style="font-size: 25px; padding-top: 5px; line-height: 1em;">Human Practice</SPAN></SPAN></A><DIV class="sdt_box" style="width: 185px; height: 160px;"><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/Overview" style="margin-left: 13px;">Overview</A><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/HumOut" style="margin-left: 13px;">Outreach</A><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/HumCollab" style="margin-left: 13px;">Collaborations</A><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/HumFlows" style="margin-left: 13px;">Flows Built-up</A><A href="https://2013.igem.org/Team:NYMU-Taipei/HumanPractice/HumPracticalApp">Practical Applications</A></DIV></LI><LI><A href="https://2013.igem.org/Team:NYMU-Taipei/Safety"><SPAN class="sdt_wrap"><SPAN class="sdt_link">Safety</SPAN></SPAN></A></LI></UL></DIV></DIV></DIV></DIV><DIV class="main"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Identifying_Nosema_ceranae"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Identifying Nosema ceranae</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Introduction"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Background"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Nosema_spp._Differentiation_Methods"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Nosema spp. Differentiation Methods</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-5"><A href="#Experiment"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Experiment</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#The_Protocol_of_OldPrimers"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">The Protocol of OldPrimers</SPAN></A></LI><LI class="toclevel-4 tocsection-7"><A href="#Nosema_primers"><SPAN class="tocnumber">1.3.1.1</SPAN><SPAN class="toctext">Nosema primers</SPAN></A></LI><LI class="toclevel-4 tocsection-8"><A href="#N._ceranae_primers"><SPAN class="tocnumber">1.3.1.2</SPAN><SPAN class="toctext">N. ceranae primers</SPAN></A></LI><LI class="toclevel-4 tocsection-9"><A href="#N._apis_primers"><SPAN class="tocnumber">1.3.1.3</SPAN><SPAN class="toctext">N. apis primers</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Results_of_Old_Primers"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Results of Old Primers</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#New_Primers"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">New Primers</SPAN></A></LI><LI class="toclevel-4 tocsection-12"><A href="#1._Common_gene_of_N._spp.---_NcORF_00182_hypothetical_protein_.26_NAPIS_ORF01775_chitin_synthase_d"><SPAN class="tocnumber">1.3.3.1</SPAN><SPAN class="toctext">1. Common gene of N. spp.--- NcORF_00182 hypothetical protein &amp; NAPIS_ORF01775 chitin synthase d</SPAN></A></LI><LI class="toclevel-4 tocsection-13"><A href="#2._Specific_gene_of_N._ceranae---NcORF_00714_hypothetical_protein"><SPAN class="tocnumber">1.3.3.2</SPAN><SPAN class="toctext">2. Specific gene of N. ceranae---NcORF_00714 hypothetical protein</SPAN></A></LI><LI class="toclevel-4 tocsection-14"><A href="#3._Specific_gene_of_N._apis---NAPIS_ORF02138_class_iv_chitinase"><SPAN class="tocnumber">1.3.3.3</SPAN><SPAN class="toctext">3. Specific gene of N. apis---NAPIS_ORF02138 class iv chitinase</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#The_Protocol_of_New_Primers"><SPAN class="tocnumber">1.3.4</SPAN><SPAN class="toctext">The Protocol of New Primers</SPAN></A></LI><LI class="toclevel-4 tocsection-16"><A href="#Nosema_primers_2"><SPAN class="tocnumber">1.3.4.1</SPAN><SPAN class="toctext">Nosema primers</SPAN></A></LI><LI class="toclevel-4 tocsection-17"><A href="#N._ceranae_primers_2"><SPAN class="tocnumber">1.3.4.2</SPAN><SPAN class="toctext">N. ceranae primers</SPAN></A></LI><LI class="toclevel-4 tocsection-18"><A href="#N._apis_primers_2"><SPAN class="tocnumber">1.3.4.3</SPAN><SPAN class="toctext">N. apis primers</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Results_of_New_Primers"><SPAN class="tocnumber">1.3.5</SPAN><SPAN class="toctext">Results of New Primers</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Reference"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Reference</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Identifying_Nosema_ceranae">Identifying <I>Nosema ceranae</I></SPAN></H1><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><P>The identification of <I>Nosema apis</I> and <I>Nosema ceranae</I> is important and still needs further research. So far the best differentiation method is the use of species–specific primers; however, the sequence differences are limited to the ribosomal RNA loci, leading to overestimation of Nosema infection rate. Thanks to recent genome study of <I>Nosema apis</I> and <I>Nosema ceranae</I>, we proposed new primers for differentiation of <I>Nosema spp.</I> based on highly conserved, species–specific protein coding genes rather than species–specific sequences in rRNA genes, which is likely to improve the accuracy of species identification and can be used to quantify <I>Nosema ceranae</I>. We confirmed the new primers by Blast, PCR with Nosema genomic DNA and the latter was sent for DNA Sequencing. After the DNA Sequencing, the result showed that there was only N. ceranae and no N. apis in the sample, while old primers produced a <I>fake</I> N apis band and new primers did not make the mistake. 
</P><H2><SPAN class="mw-headline" id="Background">Background</SPAN></H2><H3><SPAN class="mw-headline" id="Nosema_spp._Differentiation_Methods"><I>Nosema spp.</I> Differentiation Methods</SPAN></H3><UL><LI><B>Light microscopy method</B>: It is based on morphology difference of <I>N. apis</I> and <I>N. ceranae</I>, revealing that N. ceranae are oval or rod shaped like a rice, and also slightly thinner than N. apis. Light microscopy method is convenient but not scientific enough since <I>N. apis</I> bears a striking resemblance to <I>N. ceranae</I>.
</LI></UL><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B><I>N. ceranae</I> and <I>N. apis</I> are hard to distingush under light microscope.</B><A href="http://scientificbeekeeping.com/nosema-ceranae-additional-reports-and-ramblings/" class="external free" rel="nofollow">http://scientificbeekeeping.com/nosema-ceranae-additional-reports-and-ramblings/</A></DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Transmission electron micrographs of (A)<I>N. apis</I> and (B)<I>N. ceranae</I>.<B><A href="http://www.zoologie.uni-halle.de/allgemeine_zoologie/research/host-parasite/" class="external free" rel="nofollow">http://www.zoologie.uni-halle.de/allgemeine_zoologie/research/host-parasite/</A></B></DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B>Some <I>N. ceranae</I> spores shown in 400x manification rate.</B></DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B> Many <I>N. ceranae</I> spores shown in 400x manification rate.</B></DIV></DIV></DIV></TD></TR></TBODY></TABLE><UL><LI><B>Molecular methods</B>: Species-specific primers are more accurate than light microscopy method. The problem is that so far nearly all protocols based on species–specific sequence differences are limited to the ribosomal RNA loci, which may lead to confusing results due to polymorphisms in and recombination between the multi-copy 16S rRNA genes, and hence overestimate the rate of Nosema infections.
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:752px;"><DIV class="thumbcaption"><B>Source: Molecular differentiation of Nosema apis and Nosema ceranae based on species–specific sequence differences in a protein coding gene </B><A href="http://www.ncbi.nlm.nih.gov/pubmed/23352902" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/23352902</A></DIV></DIV></DIV></DIV><UL><LI><B>Our new primers</B>: This year, we created new primers based on recently published, highly conserved and absolutely species–specific sequence differences in protein coding genes. To be more specific, the primers are based on species–specific genes instead of species–specific sequences. As more and more genes of <I>Nosema spp.</I> are revealed, we are able to choose the genes that are not homologs in <I>N. apis</I> and <I>N. ceranae</I> for new primers. Our new primers are confirmed by BLAST and PCR results and then sent for DNA Sequencing. We propose that our new primers should become the preferred molecular tool for differentiation of <I>Nosema spp.</I> and thus assist in apiculture.
</LI></UL><H2><SPAN class="mw-headline" id="Experiment">Experiment</SPAN></H2><H3><SPAN class="mw-headline" id="The_Protocol_of_OldPrimers">The Protocol of OldPrimers</SPAN></H3><P>link to our <A href="https://2013.igem.org/Team:NYMU-Taipei/Experiments/Protocols" class="external text" rel="nofollow">protocol</A></P><H4><SPAN class="mw-headline" id="Nosema_primers"><I>Nosema primers</I></SPAN></H4><P>forward primer:ggcagttatgggaagtaaca
</P><P>reverse primer:ggtcgtcacatttcatctct
</P><P>Product=208bp for Nosema
</P><H4><SPAN class="mw-headline" id="N._ceranae_primers"><I>N. ceranae primers</I></SPAN></H4><P>forward primer:cggataaaagagtccgttacc
</P><P>reverse primer:tgagcagggttctagggat
</P><P>Product=250bp for <I>N. ceranae</I></P><H4><SPAN class="mw-headline" id="N._apis_primers"><I>N. apis</I> primers</SPAN></H4><P>forward primer:ccattgccggataagagagt
</P><P>reverse primer:cacgcattgctgcatcattgac
Product=401bp for <I>N. apis</I></P><TABLE class="wikitable"><TBODY><TR><TH> Temperature </TH><TH> Time
</TH></TR><TR><TD> 94<SUP>o</SUP>C </TD><TD> 120s
</TD></TR><TR><TD> 94<SUP>o</SUP>C </TD><TD> 30s </TD><TD rowspan="3"> 30 cycles
</TD></TR><TR><TD> 53<SUP>o</SUP>C </TD><TD> 30s
</TD></TR><TR><TD> 72<SUP>o</SUP>C </TD><TD> 30s
</TD></TR><TR><TD> 72<SUP>o</SUP>C </TD><TD> 600s
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Results_of_Old_Primers">Results of Old Primers</SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:524px;"><DIV class="thumbcaption"><B>Results of Old Primers</B> Electrophoresis photo 1:From the left to the right shows the Nosema genomic DNA with the new primers of <I>Nosema apis</I> ,<I>Nosema ceanae</I> ,<I>Nosema apis</I> and <I>Nosema ceanae</I> respectively</DIV></DIV></DIV></DIV><P>The <I>N. apis</I> primers produced several bands rather than one, which was not coherent with the result of the previous paper; amd thus we sent the PCR mixtures for DNA Sequencing and found out that there was actually only <I>N. ceranae</I> and no <I>N. apis</I> existing in the sample. The old <I>N. apis</I> primers produced a fake <I>N. apis</I> band while our new <I>N. apis</I> primers did not make the mistake.     
</P><H3><SPAN class="mw-headline" id="New_Primers">New Primers</SPAN></H3><H4><SPAN class="mw-headline" id="1._Common_gene_of_N._spp.---_NcORF_00182_hypothetical_protein_.26_NAPIS_ORF01775_chitin_synthase_d">1. Common gene of <I>N. spp.</I>--- NcORF_00182 hypothetical protein &amp; NAPIS_ORF01775 chitin synthase d</SPAN></H4><P>NcORF_00182 for <I>N. apis</I></P><P>Product=1089bp  
</P><P>NAPIS_ORF01775 <I>N. ceranae</I></P><P>Product=1103bp 
</P><H4><SPAN class="mw-headline" id="2._Specific_gene_of_N._ceranae---NcORF_00714_hypothetical_protein">2. Specific gene of <I>N. ceranae</I>---NcORF_00714 hypothetical protein</SPAN></H4><P>NcORF_00714 
</P><P>Product=861bp for <I>N. ceranae</I></P><H4><SPAN class="mw-headline" id="3._Specific_gene_of_N._apis---NAPIS_ORF02138_class_iv_chitinase">3. Specific gene of <I>N. apis</I>---NAPIS_ORF02138 class iv chitinase</SPAN></H4><P>NAPIS_ORF02138 
Product=1320bp for <I>N. apis</I></P><P>Besides testing our new primers, we also chose traditional primers to double-check whether the bees were infected with <I>N. app.</I> The first step is to do real PCR with primers and spores extracted from infected bees, and then run the electrophoresis to check the length.
</P><H3><SPAN class="mw-headline" id="The_Protocol_of_New_Primers">The Protocol of New Primers</SPAN></H3><H4><SPAN class="mw-headline" id="Nosema_primers_2">Nosema primers</SPAN></H4><P>forward primer:aatttatactggaacttttgaaaataaaagt
</P><P>reverse primer:catttatttaaaatcattgtttctccacaaat
</P><P>Product=1103bp for for <I>N. apis</I>and 1089bp for <I>N. ceranae</I></P><H4><SPAN class="mw-headline" id="N._ceranae_primers_2"><I>N. ceranae</I> primers</SPAN></H4><P>forward primer:tcatttaattaaactttcaacgtaattttctaaattct
</P><P>reverse primer:atgttatcacgtaatattgagaagaaacttt
Product=861bp for <I>N. ceranae</I></P><H4><SPAN class="mw-headline" id="N._apis_primers_2"><I>N. apis</I> primers</SPAN></H4><P>forward primer:ttagtaacatccttcttcagaagctaaattttc
</P><P>reverse primer:atgcaacttcaaaatccaataactactgtaac
</P><P>Product=1320bp for <I>N. apis</I></P><TABLE class="wikitable"><TBODY><TR><TH> Temperature </TH><TH> Time
</TH></TR><TR><TD> 94<SUP>o</SUP>C </TD><TD> 120s
</TD></TR><TR><TD> 94<SUP>o</SUP>C </TD><TD> 30s </TD><TD rowspan="3"> 45 cycles
</TD></TR><TR><TD> 55<SUP>o</SUP>C </TD><TD> 30s
</TD></TR><TR><TD> 72<SUP>o</SUP>C </TD><TD> 30s
</TD></TR><TR><TD> 72<SUP>o</SUP>C </TD><TD> 600s
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Results_of_New_Primers">Results of New Primers</SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:343px;"><DIV class="thumbcaption">Electrophoresis photo 2:From the left to the right shows the Nosema genomic DNA with the new primers of <I>Nosema apis</I> ,<I>Nosema ceanae</I> ,<I>Nosema apis</I> and <I>Nosema ceanae</I> respectively.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Reference">Reference</SPAN></H2><OL><LI>Gisder, S., &amp; Genersch, E. (May 01, 2013). Molecular differentiation of Nosema apis and Nosema ceranae based on species–specific sequence differences in a protein coding gene. Journal of Invertebrate Pathology, 113, 1, 1-6.<A href="http://www.sciencedirect.com/science/article/pii/S0022201113000165" class="external autonumber" rel="nofollow">[1]</A></LI><LI>Nabian, S., Ahmadi, K., Nazem, S. M., &amp; Gerami, S. A. (January 01, 2011). First Detection of Nosema ceranae, a Microsporidian Protozoa of European Honeybees (Apis mellifera) In Iran. Iranian Journal of Parasitology, 6, 3, 89-95.<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3279895/" class="external autonumber" rel="nofollow">[2]</A></LI><LI>Chen, Y., Pettis, J. S., Zhao, Y., Liu, X., Tallon, L. J., Sadzewicz, L. D., Li, R., ... Evans, J. D. (January 01, 2013). Genome sequencing and comparative genomics of honey bee microsporidia, Nosema apis reveal novel insights into host-parasite interactions. Bmc Genomics, 14.<A href="http://www.biomedcentral.com/1471-2164/14/451/" class="external autonumber" rel="nofollow">[3]</A></LI><LI>Sagastume, S., del, A. C., Martín-Hernández, R., Higes, M., &amp; Henriques-Gil, N. (January 01, 2011). Polymorphism and recombination for rDNA in the putatively asexual microsporidian Nosema ceranae, a pathogen of honeybees. #Environmental Microbiology, 13, 1, 84-95.<A href="http://www.ncbi.nlm.nih.gov/pubmed/21199250" class="external autonumber" rel="nofollow">[4]</A></LI><LI>Chen, Y. P., Evans, J. D., Murphy, C., Gutell, R., Zuker, M., Gundensen-Rindal, D., &amp; Pettis, J. S. (January 01, 2009). Morphological, molecular, and phylogenetic characterization of Nosema ceranae, a microsporidian parasite isolated from the European honey bee, Apis mellifera. The Journal of Eukaryotic Microbiology, 56, 2.)<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2779023/" class="external autonumber" rel="nofollow">[5]</A></LI><LI>Huang, W. F., Bocquet, M., Lee, K. C., Sung, I. H., Jiang, J. H., Chen, Y. W., &amp; Wang, C. H. (January 01, 2008). The comparison of rDNA spacer regions of Nosema ceranae isolates from different hosts and locations. Journal of Invertebrate Pathology, 97, 1, 9-13. <A href="http://ansc.niu.edu.tw/files/writing_journal/20/27_2fcb1cab.pdf" class="external autonumber" rel="nofollow">[6]</A></LI><LI>Bollan, K. A., Hothersall, J. D., Moffat, C., Durkacz, J., Saranzewa, N., Wright, G. A., Raine, N. E., ... Connolly, C. N. (January 01, 2013). The microsporidian parasites Nosema ceranae and Nosema apis are widespread in honeybee (Apis mellifera) colonies across Scotland. Parasitology Research, 112, 2, 751-9. <A href="http://www.ncbi.nlm.nih.gov/pubmed/23180128" class="external autonumber" rel="nofollow">[7]</A></LI><LI>Huang, W. F., Solter, L. F., Yau, P. M., &amp; Imai, B. S. (January 01, 2013). Nosema ceranae escapes fumagillin control in honey bees. Plos Pathogens, 9, 3.) <A href="http://www.plospathogens.org/article/info%3Adoi%2F10.1371%2Fjournal.ppat.1003185" class="external autonumber" rel="nofollow">[8]</A></LI></OL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:NYMU-Taipei/Project/Inhibition/Id-Nosema">http://2013.igem.org/Team:NYMU-Taipei/Project/Inhibition/Id-Nosema</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NYMU-Taipei/Project/Inhibition/Id-Nosema" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NYMU-Taipei/Project/Inhibition/Id-Nosema" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NYMU-Taipei/Project/Inhibition/Id-Nosema&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NYMU-Taipei/Project/Inhibition/Id-Nosema&amp;oldid=360432" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>