Team:USTC/Model/Parameters/m
HOME
PROJECT
Description
Design
Results
MODEL
Overview
Single-cell Model
System Analysis
Effectiveness
Parameters
INTERLAB
PARTS
Basic Parts
Composite Parts
Contributions
Improve
NOTEBOOK
Monthly Diary
Protocols
Materials
HUMAN PRACTICE
Overview
Survey
Analyze
Practice
Teamwork
COLLABORATION
TEAM
Nav
Page
Page
Page
Page
Parameters
Description
a. The average transcription rate is about 50 nucleotide per second in E.coli. The transcription rate of each transcript is calculated according to their lengths.
b. The average translation rate is about 15 amino acids per second in E.coli. The translation rate of each protein is calculated according to their amino acid numbers.
c. The average half-life of mRNAs is set to 10 min. The degradation rate is calculated according to the half-life value.
d. The average half-life of proteins is set to 2 h. The degradation rate is calculated according to the half-life value.
e. There are no data about the degradation rate constant, kinetic parameters of hdnoR. Because hdnoR is homologous to TetR protein, so the parameters related to hdnoR is from the experiment data of TetR
f. We add SrrA tag to the end of GFP, so the degradation rate is 10 times faster than the normal condition.
Reference
[1] Milo, R., Phillips, R. and Orme, N. (2016). Cell biology by the numbers. New York, NY: Garland Science.
[2] Stamatakis, Michail, and Nikos V. Mantzaris. " Comparison of deterministic and stochastic models of the lac operon genetic network. " Biophysical journal 96.3 (2009): 887-906.
[3] 2011 iGEM Tsinghua-a team wiki
[4] Goryachev, A. B., D. J. Toh, and T. Lee. " Systems analysis of a quorum sensing network: design constraints imposed by the functional requirements, network topology and kinetic constants. " Biosystems 83.2-3 (2006): 178-187.
[5] Cabantous, St√©phanie, et al. " A new protein-protein interaction sensor based on tripartite split-GFP association. " Scientific reports 3 (2013): 2854.
[6] McGinness K E, Baker T A, Sauer R T. Engineering controllable protein degradation[J]. Molecular cell, 2006, 22(5): 701-707.
[7] Basu, Subhayu, et al. " A synthetic multicellular system for programmed pattern formation. "" Nature 434.7037 (2005): 1130.
[8] Nevozhay, Dmitry, et al. " Negative autoregulation linearizes the dose?response and suppresses the heterogeneity of gene expression. " Proceedings of the National Academy of Sciences 106.13 (2009): 5123-5128.
[9] Garcia-Ojalvo, Jordi, Michael B. Elowitz, and Steven H. Strogatz. " Modeling a synthetic multicellular clock: repressilators coupled by quorum sensing. " Proceedings of the National Academy of Sciences 101.30 (2004): 10955-10960.
[10] Hochstein, L. I., and B. P. Dalton. " The purification and properties of nicotine oxidase. " Biochimica et Biophysica Acta (BBA)-Enzymology 139.1 (1967): 56-68.
[11] Williams, David E., Xinxin Ding, and Minor J. Coon. " Rabbit nasal cytochrome P-450 NMa has high activity as a nicotine oxidase. " (1990).
[12] Qiu, Jiguo, et al. " Molecular cloning, characterization, and function analysis of a novel (S)-6-hydroxy-nicotine oxidase gene from Shinella sp. HZN7 strain. " Applied and environmental microbiology (2014): AEM-01312.
[13] Fitzpatrick, Paul F., et al. " Mechanism of Flavoprotein l-6-Hydroxynicotine Oxidase: pH and Solvent Isotope Effects and Identification of Key Active Site Residues. " Biochemistry 56.6 (2017): 869-875.
[14] Hu, Haiyang, et al. " Characterization of pseudooxynicotine amine oxidase of Pseudomonas putida S16 that is crucial for nicotine degradation. " Scientific reports 5 (2015): 17770.
[15] Weber, Marc, and Javier Buceta. " Dynamics of the quorum sensing switch: stochastic and non-stationary effects. " BMC systems biology 7.1 (2013): 6.
