<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_USTC_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:USTC/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="wrapper"><NAV id="sidebar"><UL class="list-unstyled components"><LI><A href="https://2018.igem.org/Team:USTC">HOME</A></LI><LI><A href="#projectSubmenu" data-toggle="collapse" aria-expanded="false">PROJECT </A></LI><LI><A href="https://2018.igem.org/Team:USTC/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:USTC/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:USTC/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:USTC/Hardware">Hardware</A></LI><LI><A href="https://2018.igem.org/Team:USTC/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:USTC/Parts">Parts</A></LI></UL><LI><A href="#modelSubmenu" data-toggle="collapse" aria-expanded="false">MODEL </A></LI><LI><A href="https://2018.igem.org/Team:USTC/Model">Overview</A></LI><LI><A href="https://2018.igem.org/Team:USTC/Model/Single_Cell_Model">Single-cell Model</A></LI><LI><A href="https://2018.igem.org/Team:USTC/Model/System_Analysis">System Analysis</A></LI><LI><A href="https://2018.igem.org/Team:USTC/Model/Performance_evaluation">Performance Evaluation</A></LI><LI><A href="https://2018.igem.org/Team:USTC/Model/Parameters">Parameters</A></LI><LI><A href="https://2018.igem.org/Team:USTC/InterLab">INTERLAB</A></LI><LI class="active"><A href="https://2018.igem.org/Team:USTC/Notebook">NOTEBOOK</A></LI><LI><A href="#HPSubmenu" data-toggle="collapse" aria-expanded="false">HUMAN PRACTICES </A></LI><LI><A href="https://2018.igem.org/Team:USTC/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:USTC/Human_Practices/General_HP">General HP</A></LI><LI><A href="https://2018.igem.org/Team:USTC/Human_Practices/Integrated_HP">Integrated HP</A></LI><LI><A href="https://2018.igem.org/Team:USTC/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:USTC/Achievements">ACHIEVEMENTS</A></LI><LI><A href="#Teamsubmenu" data-toggle="collapse" aria-expanded="false">TEAM </A></LI><LI><A href="https://2018.igem.org/Team:USTC/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:USTC/Attributions">Attributions</A></LI></NAV><DIV id="content"><NAV class="navbar navbar-expand-lg navbar-light bg-light shadow-lg sticky-top" id="navId"><DIV class="container-fluid"><BUTTON type="button" id="sidebarCollapse" class="btn btn-info"><SPAN>Nav</SPAN></BUTTON><DIV class="collapse navbar-collapse" id="navbarSupportedContent"><UL class="nav navbar-nav ml-auto"><LI class="nav-item"><A class="nav-link" href="#p1">monthly diary</A></LI><LI class="nav-item"><A class="nav-link" href="#p2">protocols</A></LI><LI class="nav-item"><A class="nav-link" href="#p3">materials</A></LI></UL></DIV></DIV></NAV><DIV class="container"><DIV class="alert alert-success" role="alert"><H3 class="alert-heading text-center">

            notebook
          </H3></DIV><DIV class="card mb-3" id="p1"><DIV class="card-body"><H4 class="card-title">
              monthly diary
            </H4><DIV id="accordion1"><DIV class="card"><DIV class="card-header"><A class="card-link" data-toggle="collapse" href="#collapse11" aria-expanded="false">January</A></DIV><DIV id="collapse11" class="collapse show" data-parent="#accordion1"><DIV class="card-body"><P class="card-text">
                      We found a new team—USTC 2018! At the end of the term, the new team members gathered together in
                      the 363 lab, where we will start and finish our project this year. The old members shows fresh
                      men how to do molecular experiments and edit gene. Wow, we are just ready to make some amazing
                      things.</P></DIV></DIV></DIV><DIV class="card"><DIV class="card-header"><A class="card-link" data-toggle="collapse" href="#collapse12" aria-expanded="false">February</A></DIV><DIV id="collapse12" class="collapse show" data-parent="#accordion1"><DIV class="card-body"><P class="card-text">All the members are divided into different groups before winter holiday
                      begins. The Group leaders pick some excellent project done by other team for new members, which
                      make them know the project in the iGEM competition. Then, it’s time for us to think and put
                      forward a project ourselves.</P></DIV>
                          
                </DIV></DIV><DIV class="card"><DIV class="card-header"><A class="card-link" data-toggle="collapse" href="#collapse13" aria-expanded="false">March</A></DIV><DIV id="collapse13" class="collapse show" data-parent="#accordion1"><DIV class="card-body"><P class="card-text">It’s surprised that we have came up with so much novel ideas during the
                      brainstorm. All the ideas are so interesting that it’s hard to choose one. So We do research in
                      project groups to get further information about each projects. </P><P class="card-text">And here are some ideas that hopefully may become candidates for our project.

                      brainstorm
                      Nicotine-degrading bacteria
                      The treatment of Lime concretion black soils
                      Using Vibrio anguillarum to treat biofilm
                      Eliminate mercury pollution in nature
                      A bacterium capable of extracting a plasmid by itself
                      The self-evolution of bacteria
                      Helps remove scars from bacteria
                      JS: Project—From putting forward to giving up
                    </P></DIV></DIV></DIV><DIV class="card"><DIV class="card-header"><A class="card-link" data-toggle="collapse" href="#collapse14" aria-expanded="false">April</A></DIV><DIV id="collapse14" class="collapse show" data-parent="#accordion1"><DIV class="card-body"><P class="card-text">We meet once a week and arguing fiercely for the different project. At the end
                      of this month, We have selected three candidate projects.</P>1.Using the QSI system to manage the problems bringing by biofilms
                    2.Treatment of tobacco waste
                    3.Governance heavy metals

                    <P class="card-text">So we divide team members into three groups again to determine our final
                      project.
                    </P></DIV>
                          
                </DIV></DIV><DIV class="card"><DIV class="card-header"><A class="card-link" data-toggle="collapse" href="#collapse15" aria-expanded="false">May</A></DIV><DIV id="collapse15" class="collapse show" data-parent="#accordion1"><DIV class="card-body"><P class="card-text">We registered the team on the 2018.igem.org and everyone join in the team
                      USTC! We really make more research about the three projects and we are confident that we will get
                      good results this year for all the projects are amazing. But we need to make a decision as soon
                      as possible because of time constraints.</P><P class="card-text">And we begin to make pre-experiment to make sure that we can finish our
                      project on time.</P><P class="card-text">Non-biological students are very interested in the experiments of extracting
                      plasmid, and we are very happy for the team members improving their experimental skills.</P></DIV></DIV></DIV><DIV class="card"><DIV class="card-header"><A class="card-link" data-toggle="collapse" href="#collapse16" aria-expanded="false">June</A></DIV><DIV id="collapse16" class="collapse show" data-parent="#accordion1"><DIV class="card-body"><P class="card-text">Congratulations! We have decided our project “TW”, and now we can concentrate
                      on completing our project. But it’s not a good idea for us that our gene circuit is a little bit
                      complicated. We have to hurry to do experiment to make sure that we can finish our project on
                      time.</P><P class="card-text">As we order the genes of the sensing system from IDT, we plan to construct our
                      regulation system firstly.</P><P class="card-text">We meet a few trouble that the gene from the iGEM Distribution Kit is NOT
                      right? It cost us some time to choose another gene in the kit. And finally we successfully get
                      the correct gene we need at the end of June.</P><DIV class="card border-light"><DIV class="card-body"><H5 class="card-title text-center">Double digestion of luxR</H5></DIV></DIV></DIV></DIV></DIV><DIV class="card"><DIV class="card-header"><A class="card-link" data-toggle="collapse" href="#collapse17" aria-expanded="false">July</A></DIV><DIV id="collapse17" class="collapse show" data-parent="#accordion1"><DIV class="card-body"><P class="card-text">The summer holiday has came and we can do experiment in 363 from now on. How
                      Wonderful！I believe that we can finish our project on time now.</P><P class="card-text">At the end of July, we almost finish the Regulation System, in the way of
                      assembling the Bio-brick from the Kit. I can't wait to see our system work now.</P><DIV class="card-deck mb-3"><DIV class="card"><DIV class="card-body"><H4 class="card-title text-center">double digestion of JT</H4></DIV></DIV><DIV class="card"><DIV class="card-body"><H4 class="card-title text-center">double digestion of PI</H4></DIV></DIV></DIV><P class="card-text">At the same time, we also start to construct the key system—degradation
                      system, which includes three enzymes to convert nicotine to useful drugs. </P><DIV class="card border-light"><DIV class="card-body"><H5 class="card-title text-center">purifaction of nicA2</H5></DIV><P class="card-text">In order to conduct on-the-spot investigations and understand the treatment
                        of Tobacco fertilizer in our lives, we used the holiday to visit the Hefei Cigarette Factory
                        and do further research. We have learned a lot from it. Details can be found in our Human
                        Practice.</P><DIV class="card-deck mb-3"><DIV class="card"><DIV class="card-body"><H4 class="card-title text-center">p1</H4></DIV></DIV></DIV><P class="card-text">And we take part in the Fifth International InterLab Measurement Study. We
                        are very delighted that we have completed the work of interlab and got accepted. Hoping that
                        our work can make some contributions to the future work of Interlab measurement study.</P><DIV class="card-deck mb-3"><DIV class="card"><DIV class="card-body"><H4 class="card-title text-center">p3</H4></DIV></DIV><DIV class="card"><DIV class="card-body"><H4 class="card-title text-center">p4</H4></DIV></DIV></DIV><P class="card-text">Of course, sometimes we also encounter some frustrating failures…Like this…</P><DIV class="card border-light"><DIV class="card-body"><H5 class="card-title text-center">where is my dna?</H5></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="card"><DIV class="card-header"><A class="card-link" data-toggle="collapse" href="#collapse18" aria-expanded="false">Augest</A></DIV><DIV id="collapse18" class="collapse show" data-parent="#accordion1"><DIV class="card-body"><P class="card-text">We have received the gene synthesized by IDT. Now we can start to construct
                      our sensing system. But we get nothing when we make PCR using these genes as template. And we
                      heard the same unfortunate news from other iGEM teams from China. Oh, it’s too terrible and we
                      have to commission another company to synthesis our genes, and the time for us to finish our
                      experiment is little.</P><P class="card-text">As we are not very sure which are the most suitable promoter for our sensing
                      system, we ask our modeling group for help. But their ideas do not make us satisfied, because it
                      will spend too much time. Now JS is nearly mad for he have to try thirty promoters one by one.</P><P class="card-text">At the end of August we go to the ShanghaiTech University to take part in the
                      fifth ccic (Conference of China iGEMer Community). We communicated with other iGEM teams, showing
                      our project and sharing our experience.</P><DIV class="card-deck mb-3"><DIV class="card"><DIV class="card-body"><H4 class="card-title text-center">p5</H4></DIV></DIV><DIV class="card"><DIV class="card-body"><H4 class="card-title text-center">p6</H4></DIV></DIV></DIV><P class="card-text">Very lucky, we meet Miss Meagan in ShanghaiTech University. We eat dinner
                      together and have a pleasant conversation, talking about our team, our experience and improvement
                      in the competition, the development of Synthetic biology in China and the After iGEM for the
                      iGEMers.</P></DIV></DIV></DIV><DIV class="card"><DIV class="card-header"><A class="card-link" data-toggle="collapse" href="#collapse19" aria-expanded="false">September</A></DIV><DIV id="collapse19" class="collapse show" data-parent="#accordion1"><DIV class="card-body"><P class="card-text">It’s not very good news for us that the summer holiday is over and we haven’t
                      finished our project, which means we have less time to stay in the lab than holiday. Fortunately,
                      we get some suggestions and help from our seniors whom were iGEMers before. They help us to
                      improve our project, especially in our modeling. </P><P class="card-text">Although we are busy in study, we also seen ind time the lab and try to make
                      the project better. Thanks to all people’s help, we finally finish our regulation system and
                      degradation system, and our sensor is going to be finished.</P><P class="card-text">In this month we share our ideas with our classmates and teachers, talking
                      with them the harm of nicotine and what our project can help. They show interest in our systems
                      and give some advice about making more people understand our work.</P></DIV></DIV></DIV><DIV class="card"><DIV class="card-header"><A class="card-link" data-toggle="collapse" href="#collapse10" aria-expanded="false">October</A></DIV><DIV id="collapse10" class="collapse show" data-parent="#accordion1"><DIV class="card-body"><P class="card-text">As the day of wiki freeze is closer and closer, we are busy in editing our
                      wiki, organizing data and figure which analyzed from our experiment. </P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="card mb-3" id="p2"><DIV class="card-body"><H4 class="card-title">Protocols</H4><DIV class="card"><H4 class="card-header">Plasmid Extraction</H4><DIV class="card-body"><P class="card-text">1.Centifuge 1.5 mL bacterium solution at 11000 rpm for 2 minutes. Remove the
                  supernatant. Repeat twice.

                  2.Add 250 μL Buffer P1, resuspend cells.

                  3.Add 250 μL Buffer P2, mix well, 3 min's standing.

                  4.Add 350 μL Buffer P3, mix well.

                  5.13400 rpm centifuge 10 min, move all supernatant to adsorption column, 11000 rpm centifuge 30
                  s, discard filtrate.

                  6.Add 500 μL Buffer DW1, 12000 rpm centifuge 30 s, discard filtrate.

                  7.Add 500 μL Wash Solution, 12000 rpm centifuge 30 s, discard filtrate. Repeat once.

                  8.12000 rpm centifuge 1 min.

                  9.Lying for 10 min.

                  10.Put the adsorption column in a new EP tube. Add 50 μL 50°C ddH2O, 10 min's standing, 12000 rpm
                  centifuge 1 min. Preserve in the EP tube with ddH2O at 4 degree Celsius.

                  After extraction, we measured the OD A260/A280, OD A260/A230 and the density of plasmids. </P></DIV></DIV><DIV class="card"><H4 class="card-header">Extraction of bacterial genome</H4><DIV class="card-body"><P class="card-text">
                  1.Centifuge 1.5 mL bacterium solution at 8000 rpm for 1 minute. Remove the supernatant. Repeat twice.

                  2.Add 400 μL Buffer Digestion, resuspend cells.

                  3.Put the tube in 65 ° C water bath 1 h until the cells are completely lysed.

                  4.Add 200 μL Buffer PB, mix well.

                  5.Standing at -20℃ for 5 min.

                  6.10000 rpm centifuge 5 min, move all supernatant to a new clean EP tube.

                  7.Add an equal volume of isopropanol, invert it 5~8 times and mix well. Standing for 2-3 min.

                  8. 11000 rpm centifuge 30 s, discard filtrate.

                  9.Add 1 mL 75% Ethanol, Rinse upside down for 1~3 min, 10000 rpm centifuge 2 min, discard
                  filtrate. Repeat once.

                  10.Lying for 10 min.

                  11.Put the adsorption column in a new EP tube. Add 50 μL TE Buffer Preserve in the EP tube with
                  ddH2O at 4 degree Celsius. </P></DIV><DIV class="card"><H4 class="card-header">Restriction digests</H4><DIV class="card-body"><P class="card-text">1.Set up the following reaction on ice: </P><TABLE class="table table-hover table-secondary"><THEAD><TR><TH>component</TH><TH>Volume(µl)</TH></TR></THEAD><TBODY><TR><TD>DNA</TD><TD>X(200ng~1µg)</TD></TR><TR><TD>Restriction enzyme</TD><TD>0.5~2 µl</TD></TR><TR><TD>Green Buffer </TD><TD>2</TD></TR><TR><TD>nuclease-free water</TD><TD>Up to 20 µl</TD></TR></TBODY></TABLE><P class="card-text">2.Vortex briefly.</P><P class="card-text">3.Incubate at 37°C for at least 60 minutes.</P></DIV></DIV><DIV class="card"><H4 class="card-header">Making a gel</H4><DIV class="card-body"><P class="card-text">
                    To make a 1% agarose gel:

                    1.Weigh out 0.4g of agarose powder

                    2.Transfer to a microwave bottle

                    3.Pour 40mL of 1x TAE buffer into the bottle

                    4.Microwave until clear

                    5.Pour the gel and let it polymerize until it is solid.</P></DIV></DIV><DIV class="card"><H4 class="card-header">Agarose Gel-electrophoresis</H4><DIV class="card-body"><P class="card-text">
                    1.Add the appropriate 6x DNA loading Buffer to the PCR or digestion reaction mixture.

                    2.Place the gel into the electrophoresis tank and pour 1x TAE buffer

                    3.Pipet the DNA into the gel pockets.

                    4.Run the gel at 120V for 30 min.

                    5.Image the gel using ultraviolet light. If necessary, cut out bands. </P></DIV></DIV><DIV class="card"><H4 class="card-header">Gel Extraction using Omega Kit</H4><DIV class="card-body"><P class="card-text">
                    1.Perform agarose gel electrophoresis to fractionate DNA fragments.

                    2.Minimize the size of the gel slice by removing extra agarose.

                    3.Add 1 volume Binding Buffer (XP2).

                    4.Incubate at 50°C-60°C for 7 minutes or until the gel has completely melted. Vortex or shake
                    the tube every 2-3 minutes.

                    5.Insert a HiBind DNA Mini Column in a 2 mL Collection Tube.

                    6.Add no more than 700 μL DNA/agarose solution from Step 4 to the HiBind DNA Mini Column.

                    7.Centrifuge at 10,000g for 1 minute at room temperature.

                    8.Discard the filtrate and reuse collection tube.

                    9. Repeat Steps 6-8 until all of the sample has been transferred to the column.
                    Add 300 μL Binding Buffer (XP2).

                    10. Centrifuge at maximum speed for 1 minute.

                    11. Discard the filtrate and reuse collection tube.

                    12.Add 700 μL SPW Wash Buffer.Centrifuge at maximum speed for 1 minute at room
                    temperature.Discard the filtrate and reuse collection tube.

                    13.Repeat Step 12 once.

                    14.Centrifuge the empty HiBind DNA Mini Column for 2 minutes at maximum speed to dry the column
                    matrix.

                    15.Lying for 10 min.

                    16.Put the HiBind DNA Mini Column in a new EP tube. Add 30 μL 50°C ddH2O, 10 min's standing,
                    maximum speed centifuge 1 min. Preserve in the EP tube with ddH2O at 4 degree Celsius. </P></DIV></DIV><DIV class="card"><H4 class="card-header">Transformation of competent cells</H4><DIV class="card-body"><P class="card-text">
                    1.Take the competent bacteria from refrigerator and incubate them into ice about 5 mins until it is
                    dissolved

                    2.Absorb 100pg to 10 ng plasmid and mix it with bacteria solution thoroughly.
                    ATTENTION: Please operate this step tenderly!!!

                    3.Put the tubes on the ice about 30 mins.(Time SHOULD BE ACCURATE)

                    4.Make a heat shock at 42 degree centigrade about 90 sec (TIME SHOULD BE ACCURATE)

                    5.Put the tubes on the ice about 5 mins again.

                    6.Add 900 ul LB medium into EP tubes and cultivate the bacteria at 37 degree centrigrade about
                    1 hours .

                    7.Centrifuge them at 4000 rpm about 2 mins and we will see sediment in the tubes.

                    8.Discard the supernatant liquid and leave about 200 ul medium.

                    9.Coat plate: add 200 ul solution in a large plate.
                    10.Cultivate these bacteria overnight for further use. </P></DIV></DIV><DIV class="card"><H4 class="card-header">Preparation of CaCl2 competent cells</H4><DIV class="card-body"><P class="card-text">
                    1.Pick up some bacterial solution with the inoculating loop and inoculate it onto the anti-LB plate
                    by plate scribing.

                    2.Incubate at 37⁰C overnight.

                    3.Pick colonies in 5 mL EP tubes without anti-LB liquid mediumIncubate at 37⁰C, shaking at
                    250rpm for 6 hours.

                    4.Transfer 2 mL of bacterial solution to 200 mL of LB liquid medium at 37 ° C shaker until the
                    OD value reaches 0.4.

                    5.Transfer the medium to a 50 mL pre-cooled centrifuge tube and place on ice for 30 min.

                    6.Spin down for 15 minutes at 4000rpm at 4⁰C.

                    7.Discard supernatant, resuspend pellet in 30ml of 80mM CaCl2-20mM MgCl2, keep on ice.

                    8.Spin down for 15 minutes at 2700rpm at 4⁰C.

                    9.Discard supernatant, resuspend pellet in 2mL 100mM CaCl2-18% glycerin, keep on ice，then
                    dispense to EP tube.

                    10.Store at -80℃ for further use. </P></DIV></DIV><DIV class="card"><H4 class="card-header">Ligation</H4><DIV class="card-body"><P class="card-text">1.Set up the following reaction on ice: </P><TABLE class="table table-hover table-secondary"><THEAD><TR><TH>component</TH><TH>Volume(µl)</TH></TR></THEAD><TBODY><TR><TD>Insert DNA</TD><TD>3:1 molar excess over vector</TD></TR><TR><TD>Linearized vector DNA</TD><TD>X(100ng)</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>0.4</TD></TR><TR><TD>10x T4 DNA Ligase Buffer</TD><TD>2</TD></TR><TR><TD>Sterilized distilled water </TD><TD>up to 20 μl</TD></TR></TBODY></TABLE><P class="card-text">2.Vortex thoroughly and spin briefly.</P><P class="card-text">3.Incubate the mixture at 16℃ overnight.</P></DIV></DIV><DIV class="card"><H4 class="card-header">Polymerase Chain Reactions (PCRs)</H4><DIV class="card-body"><P class="card-text">1.Set up the following reaction on ice: </P><TABLE class="table table-hover table-secondary"><THEAD><TR><TH>component</TH><TH>Volume(µl)</TH></TR></THEAD><TBODY><TR><TD>PrimeSTAR Max Premix (2X)</TD><TD>25</TD></TR><TR><TD>Primer 1</TD><TD>1</TD></TR><TR><TD>Primer 2</TD><TD>1</TD></TR><TR><TD>Template</TD><TD>1 &amp;#60 200 ng ＊</TD></TR><TR><TD>Sterilized distilled water </TD><TD>up to 50 μl</TD></TR></TBODY></TABLE><P class="card-text">2.PCR Conditions</P><TABLE class="table table-hover table-secondary"><THEAD><TR><TH>step</TH><TH>temperature</TH><TH>time</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>95℃</TD><TD>10min</TD></TR><TR><TD>2</TD><TD>98℃</TD><TD>10 sec</TD></TR><TR><TD>3</TD><TD>55℃</TD><TD>5 sec. or 15 sec.</TD></TR><TR><TD>4</TD><TD>72℃</TD><TD>5 sec./kb</TD></TR><TR><TD>30 cycles(step 2 ~ step 4)</TD></TR><TR><TD>5</TD><TD>72℃</TD><TD>10 min</TD></TR></TBODY></TABLE><P class="card-text">3.Store at 4℃ for further use. </P></DIV></DIV><DIV class="card"><H4 class="card-header">Purification of PCR product </H4><DIV class="card-body"><P class="card-text">
                    1.Transfer the PCR reaction or enzymatic reaction to a clean 1.5 ml centrifuge tube and add 5
                    volumes Buffer B3, mix well.

                    2.Transfer all the mixture to the adsorption column and centrifuged at 8,000 × g for 30 sec,
                    discard filtrate.

                    3.Add 500 μL Wash Solution, 12000 rpm centifuge 30 s, discard filtrate. Repeat once.

                    4.12000 rpm centifuge 1 min.

                    5.Lying for 10 min.

                    6.Put the adsorption column in a new EP tube. Add 20~30 μL 50°C ddH2O, 10 min's standing,
                    12000 rpm centifuge 1 min. Preserve in the EP tube with ddH2O at 4 degree Celsius.</P></DIV></DIV><DIV class="card"><H4 class="card-header">Performing the Seamless Cloning Reaction </H4><DIV class="card-body"><TABLE class="table table-hover table-secondary"><THEAD><TR><TH>component</TH><TH>Volume(µl)</TH></TR></THEAD><TBODY><TR><TD>Fragments</TD><TD>X</TD></TR><TR><TD>Linearized vector</TD><TD>X(50ng)</TD></TR><TR><TD>Sterilized distilled water </TD><TD>up to 20 μl</TD></TR></TBODY></TABLE><P class="card-text"> *Optimized cloning efficiency is 50 ng of vectors with 3 fold of excess
                    inserts. </P><P class="card-text">2.Incubate at 50°C for 30 minutes to 1 hour, depending on number of fragments
                    being assembled. </P><P class="card-text">3.Transform 4 µl of the reaction mixture into competent E. coli, or use the
                    mixture directly in other applications.</P></DIV></DIV><DIV class="card"><H4 class="card-header">Prepare of cell lysis</H4><DIV class="card-body"><P class="card-text">
                    1.Add bacteria in 500mL LB medium, shake in 37℃ to OD600 reach 0.8

                    2.Add 500μL 1M IPTG, cultivate in (temperature) for (time)

                    3.Centrifuge at 8000 rpm, 4℃ for 10 min, then remove the supernatant

                    4.Repeat step 3 untill all the bacteria have been collected

                    5.Resuspend the bacteria with 50 mL TBS

                    6.Centrifuge at 8000 rpm, 4°C for 10 min, then remove the supernatant

                    7.Resuspend the bacteria with 15 mL lysis buffer

                    8.The cells were disrupted via ultrasonication (Power 35%, 30min, total duty in
                    cycles of 1s on, 2s off)

                    9.Centrifuge at 14000 rpm, 4°C for 20 min

                    10.Retrieve the supernatant for future experiments.</P></DIV></DIV><DIV class="card"><H4 class="card-header">Purification of protein with 6XHis-tag</H4><DIV class="card-body"><P class="card-text">
                    1.Pure out 20% ethanol inside of the tube

                    2.Wash the tube with 10mL 1M imidazole 3 times

                    3.Wash the tube with 10mL H2O

                    4.Add 10mL 100mM NiCl into the tube, then release the solution after a few minutes

                    5.Wash the tube with 10mL H2O

                    6.Balance the tube with 10mL lysis buffer 3 times

                    7.Add the cell lysis in the tube (flow rate: 1~1.5mL/min)

                    8.Wash the tube with 10mL lysis buffer

                    9.Wash the tube with 10mL buffer contained 20、50、80mM imidazole

                    10.Wash the tube with 10mL elution buffer and collect

                    11.Wash the tube with 10mL 1M imidazole 3 times

                    12.Wash the tube with 10mL H2O

                    13.Add 5mL 20% ethanol to storage</P></DIV></DIV></DIV></DIV></DIV><DIV class="card mb-3" id="p3"><DIV class="card-body"><H4 class="card-title">Materials</H4><DIV class="card"><H4 class="card-header">Antibiotics</H4><DIV class="card-body"><TABLE class="table table-hover table-secondary"><THEAD><TR><TH>Name</TH><TH>Supplier</TH></TR></THEAD><TBODY><TR><TD>Chloramphenicol</TD><TD>BBI Life Sciences</TD></TR><TR><TD>Kanamycin sulfate</TD><TD>BBI Life Sciences</TD></TR><TR><TD>Ampicillin</TD><TD>BBI Life Sciences</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="card"><H4 class="card-header">Enzymes</H4><DIV class="card-body"><TABLE class="table table-hover table-secondary"><THEAD><TR><TH> Name </TH><TH> Supplier</TH></TR></THEAD><TBODY><TR><TD>EcoRI</TD><TD>ThermoFisher Scientific</TD></TR><TR><TD>XbaI</TD><TD>ThermoFisher Scientific</TD></TR><TR><TD>SpeI(BcuI)</TD><TD>ThermoFisher Scientific</TD></TR><TR><TD>PstI</TD><TD>ThermoFisher Scientific</TD></TR><TR><TD>XhoI</TD><TD>ThermoFisher Scientific</TD></TR><TR><TD>BglII</TD><TD>ThermoFisher Scientific</TD></TR><TR><TD>SamI</TD><TD>ThermoFisher Scientific</TD></TR><TR><TD>DpnI</TD><TD>ThermoFisher Scientific</TD></TR><TR><TD>T4 DNALigase</TD><TD>TakaRa</TD></TR><TR><TD>SeamlessCloning Master Mix</TD><TD>SangonBiotech</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="card"><H4 class="card-header">Polymerases</H4><DIV class="card-body"><TABLE class="table table-hover table-secondary"><THEAD><TR><TH>Name</TH><TH>Supplier</TH></TR></THEAD><TBODY><TR><TD>Taq PCRMaster Mix</TD><TD> Sangon Biotech </TD></TR><TR><TD>PrimeSTAR®Max DNA Polymerase</TD><TD>TaKaRa</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="card"><H4 class="card-header">Media</H4><DIV class="card-body"><TABLE class="table table-hover table-secondary"><THEAD><TR><TH>Media</TH><TH>Composition</TH></TR></THEAD><TBODY><TR><TD>LBmedium (liquid)</TD><TD>10 gNaCl, 10 g tryptone, 5 g yeast extract in 1 L dH2O</TD></TR><TR><TD>LBmedium (solid)</TD><TD>10 gNaCl, 10 g tryptone, 5 g yeast extract, 15 g Agar A in 1 L dH2O</TD></TR><TR><TD>Hestrinmedium</TD><TD>5 gbacto-peptone, 5 g yeast extract, 20 gglucose, 1 g potassium momohydrogen phosphate</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="card"><H4 class="card-header">Kits</H4><DIV class="card-body"><TABLE class="table table-hover table-secondary"><THEAD><TR><TH>Name</TH><TH>Supplier</TH></TR></THEAD><TBODY><TR><TD>SanPrepColumn DNA Gel Extraction Kit</TD><TD>SangonBiotech</TD></TR><TR><TD>SanPrepColumn Plasmid Mini-Preps Kit</TD><TD>SangonBiotech</TD></TR><TR><TD>SanPrepColumn PCR Product Purification Kit</TD><TD>SangonBiotech</TD></TR><TR><TD>E.Z.N.A.®Gel Extraction Kit</TD><TD>OmegaBio-tek</TD></TR><TR><TD>Ready-to-UseSeamless Cloning Kit</TD><TD>SangonBiotech</TD></TR><TR><TD>Seamlesscloning Master Mix</TD><TD>SangonBiotech</TD></TR><TR><TD>RapidBacterial Genomic DNA Isolation Kit</TD><TD>SangonBiotech</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="card"><H4 class="card-header">Markers</H4><DIV class="card-body"><TABLE class="table table-hover table-secondary"><THEAD><TR><TH>Name</TH><TH>Supplier</TH></TR></THEAD><TBODY><TR><TD>1 kb DNALadder, DNA Marker-Q, Ready-to-use</TD><TD>SangonBiotech</TD></TR><TR><TD>GeneRuler1 kb DNA Ladder </TD><TD>SangonBiotech</TD></TR><TR><TD>DNAMarker S Plus (100~5000 bp)</TD><TD>SangonBiotech</TD></TR><TR><TD>MarkerJS (600bp, 2000bp, 2600bp)</TD><TD>DIY byour team member JS</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="card"><H4 class="card-header">Plasmids</H4><DIV class="card-body"><TABLE class="table table-hover table-secondary"><THEAD><TR><TH>Name</TH><TH>plasmid backbone</TH><TH>gene</TH><TH>assistant</TH><TH>source</TH></TR></THEAD><TBODY><TR><TD>j23100</TD><TD>J61002</TD><TD>PromoterJ23100</TD><TD>Ampicillin</TD><TD>iGEMDistribution Kit</TD></TR><TR><TD>RBS</TD><TD>pSB1A2</TD><TD>RBS B0034</TD><TD>Ampicillin</TD><TD>iGEMDistribution Kit</TD></TR><TR><TD>luxR</TD><TD>pSB1A2</TD><TD>luxR</TD><TD>Ampicillin</TD><TD>iGEMDistribution Kit</TD></TR><TR><TD>pluxR</TD><TD>pSB1C3</TD><TD>luxpR</TD><TD>Chloramphenicol</TD><TD>iGEMDistribution Kit</TD></TR><TR><TD>luxI</TD><TD>pSB1C3</TD><TD>luxI</TD><TD>Chloramphenicol</TD><TD>iGEMDistribution Kit</TD></TR><TR><TD>Tem</TD><TD>pSB1C3</TD><TD>terminatorB0015</TD><TD>Chloramphenicol</TD><TD>iGEMDistribution Kit</TD></TR><TR><TD>RBuI</TD><TD>pSB1A2</TD><TD>luxI</TD><TD>Ampicillin</TD><TD>RBS+luxI</TD></TR><TR><TD>RBuR</TD><TD>pSB1A2</TD><TD>luxR</TD><TD>Ampicillin</TD><TD>RBS+luxR</TD></TR><TR><TD>JR</TD><TD>pSB1A2</TD><TD>luxR</TD><TD>Ampicillin</TD><TD>j23100+RBuI</TD></TR><TR><TD>JT</TD><TD>pSB1C3</TD><TD>luxR</TD><TD>Chloramphenicol</TD><TD>JR+Tem</TD></TR><TR><TD>PI</TD><TD>pSB1C3</TD><TD>luxI</TD><TD>Chloramphenicol</TD><TD>pluxR+RBuI</TD></TR><TR><TD>JTPI</TD><TD>pSB1C3</TD><TD>luxR, luxI</TD><TD>Chloramphenicol</TD><TD>JT+PI</TD></TR><TR><TD>GFP</TD><TD>pSB1A2</TD><TD>GFP</TD><TD>Ampicillin</TD><TD>iGEMDistribution Kit</TD></TR><TR><TD>RBFP</TD><TD>pSB1A2</TD><TD>GFP</TD><TD>Ampicillin</TD><TD>RBS+GFP</TD></TR><TR><TD>JTPig</TD><TD>pSB1C3</TD><TD>luxR,luxI, GFP</TD><TD>Chloramphenicol</TD><TD>JTPI+GFP</TD></TR><TR><TD>JTP</TD><TD>pSB1C3</TD><TD>luxR</TD><TD>Chloramphenicol</TD><TD>JT+GFP</TD></TR><TR><TD>JTPG</TD><TD>pSB1C3</TD><TD>luxR, GFP</TD><TD>Chloramphenicol</TD><TD>JTP+RBFP</TD></TR><TR><TD>nicA2</TD><TD>pet28</TD><TD>nicA2</TD><TD>Kanamycinsulfate</TD><TD>synthesizedby GenScript</TD></TR><TR><TD>pnao</TD><TD>pet28</TD><TD>pnao</TD><TD>Kanamycinsulfate</TD><TD>synthesizedby GenScript</TD></TR><TR><TD>sapd</TD><TD>pet28</TD><TD>sapd</TD><TD>Kanamycinsulfate</TD><TD>synthesizedby GenScript</TD></TR><TR><TD>pnao-part</TD><TD>pSB1C3</TD><TD>pnao</TD><TD>Chloramphenicol</TD><TD>pnao+pSB1C3</TD></TR><TR><TD>sapd-part</TD><TD>pSB1C3</TD><TD>sapd</TD><TD>Chloramphenicol</TD><TD>sapd+pSB1C3</TD></TR><TR><TD>nps</TD><TD>pet28</TD><TD>nicA2,pnao, sapd</TD><TD>Kanamycinsulfate</TD><TD>nicA2+pnao+sapd</TD></TR><TR><TD>GFRA</TD><TD>pSB1A2</TD><TD>GFP withssra-tag</TD><TD>Ampicillin</TD><TD>PCR</TD></TR><TR><TD>RFRA</TD><TD>pSB1A2</TD><TD>GFP withssra-tag</TD><TD>Ampicillin</TD><TD>RBS+GFRA</TD></TR><TR><TD>VppA</TD><TD>pSB1C3</TD><TD>VppA</TD><TD>Chloramphenicol</TD><TD>synthesizedby GenScript</TD></TR><TR><TD>RBVA</TD><TD>pSB1A2</TD><TD>VppA </TD><TD>Ampicillin</TD><TD>RBS+VppA</TD></TR><TR><TD>hdnoR</TD><TD>pSB1C3</TD><TD>hdnoR</TD><TD>Chloramphenicol</TD><TD>synthesizedby IDT</TD></TR><TR><TD>RdoR</TD><TD>pSB1A2</TD><TD>hdnoR</TD><TD>Ampicillin</TD><TD>RBS+hdnoR</TD></TR><TR><TD>RdRt</TD><TD>pSB1C3</TD><TD>hdnoR</TD><TD>Chloramphenicol</TD><TD>RdoR+Tem</TD></TR><TR><TD>hdno</TD><TD>pSB1C3</TD><TD>hdno withits promoter</TD><TD>Chloramphenicol</TD><TD>synthesizedby IDT</TD></TR><TR><TD>hlno</TD><TD>pSB1C3</TD><TD>hlno</TD><TD>Chloramphenicol</TD><TD>synthesizedby GenScript</TD></TR><TR><TD>Rlno</TD><TD>pSB1A2</TD><TD>hlno</TD><TD>Ampicillin</TD><TD>RBS+hlno</TD></TR><TR><TD>hlux</TD><TD>pSB1A2</TD><TD>hlno, luxI</TD><TD>Ampicillin</TD><TD>Rlno+RBuI</TD></TR><TR><TD>CBM</TD><TD>PCG</TD><TD>cBM, GFP</TD><TD>Ampicillin</TD><TD>get fromour tutor</TD></TR><TR><TD>CBni</TD><TD>pet28</TD><TD>cBM, nicA2</TD><TD>Kanamycinsulfate</TD><TD>cBM+nicA2</TD></TR><TR><TD>PJ4</TD><TD>J61002</TD><TD>Promoter </TD><TD>Ampicillin</TD><TD>iGEMDistribution Kit</TD></TR><TR><TD>PJ5</TD><TD>J61002</TD><TD>Promoter </TD><TD>Ampicillin</TD><TD>iGEMDistribution Kit</TD></TR><TR><TD>PJ6</TD><TD>J61002</TD><TD>Promoter </TD><TD>Ampicillin</TD><TD>iGEMDistribution Kit</TD></TR><TR><TD>PJ7</TD><TD>J61002</TD><TD>Promoter </TD><TD>Ampicillin</TD><TD>iGEMDistribution Kit</TD></TR><TR><TD>PJ8</TD><TD>J61002</TD><TD>Promoter </TD><TD>Ampicillin</TD><TD>iGEMDistribution Kit</TD></TR><TR><TD>PJ9</TD><TD>J61002</TD><TD>Promoter </TD><TD>Ampicillin</TD><TD>iGEMDistribution Kit</TD></TR><TR><TD>moc</TD><TD>pSB1C3</TD><TD>moc</TD><TD>Chloramphenicol</TD><TD>PCR fromthe E.coli genome</TD></TR><TR><TD>mog</TD><TD>pSB1C3</TD><TD>mog</TD><TD>Chloramphenicol</TD><TD>PCR fromthe E.coli genome</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="card"><H4 class="card-header">Equipment</H4><DIV class="card-body"><TABLE class="table table-hover table-secondary"><THEAD><TR><TH>Equipment</TH><TH>Name</TH><TH>Manufacturer</TH></TR></THEAD><TBODY><TR><TD>Palmcentrifuge</TD><TD>centrifugeLX-100</TD><TD>HaimenQilin Bei'er</TD></TR><TR><TD>Minicentrifuge</TD><TD>Velocity13u Minifuge</TD><TD>Dynamic</TD></TR><TR><TD>Minicentrifuge</TD><TD>CentrifugeMiniSpin®</TD><TD>eppendorf</TD></TR><TR><TD>Tablecentrifuge</TD><TD>CentrifugeZ 326 K</TD><TD>HERMLE</TD></TR><TR><TD>Centrifuge</TD><TD>CR21GⅢHigh-Speed Centrifuges</TD><TD>HITACHI</TD></TR><TR><TD>Cryogenicbath</TD><TD>Cryogenicbath SDC-6 </TD><TD>NingboScientz Biotechnology</TD></TR><TR><TD>Waterbath</TD><TD>TW8Water Bath</TD><TD>Julabo</TD></TR><TR><TD>Pipette</TD><TD>pipetlite xls</TD><TD>Rainin</TD></TR><TR><TD>Ultrasonicbreaker</TD><TD>JY92-IIN Noise isolating chamber</TD><TD>NingboScientz Biotechnology</TD></TR><TR><TD>PCR-Machine</TD><TD>96well thermal cycler</TD><TD>AppliedBiosystems</TD></TR><TR><TD>PCR-Machine</TD><TD>2720thermal cycler</TD><TD>AppliedBiosystems</TD></TR><TR><TD>Microwaveoven</TD><TD>Microwaveoven</TD><TD>Galanz</TD></TR><TR><TD>Electrophoresisbath</TD><TD>EPS600 Electrophoresis</TD><TD>Tanon</TD></TR><TR><TD>GelImage System</TD><TD>1600Gel Image System</TD><TD>Tanon</TD></TR><TR><TD>Icemachine</TD><TD>Grant XB-70</TD><TD>Grant</TD></TR><TR><TD>Shaker</TD><TD>ZQZY-10BF</TD><TD>ShanghaiZhichu Instrument</TD></TR><TR><TD>Scales</TD><TD>TE212-L</TD><TD>sartorius</TD></TR><TR><TD>Scales</TD><TD>BSA1245</TD><TD>sartorius</TD></TR><TR><TD>VerticalFlow Clean Bench</TD><TD>zhjh-c1112b</TD><TD>ZHICHENG</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="card"><H4 class="card-header">Primer</H4><DIV class="card-body"><TABLE class="table table-hover table-secondary"><THEAD><TR><TH>name</TH><TH>template</TH><TH>product</TH><TH>sequence</TH><TH>length</TH></TR></THEAD><TBODY><TR><TD>nps-f</TD><TD>nps-plasmid</TD><TD>nicA2+pnao+sapd+pet28</TD><TD>gtttaactttaagaaggagatataccatgggc </TD><TD>32</TD></TR><TR><TD>nps-r</TD><TD>nps-plasmid</TD><TD>nicA2+pnao+sapd+pet28</TD><TD>tcaacctactactgggctgcttcc</TD><TD>24</TD></TR><TR><TD>JT-f</TD><TD>JT-plasmid</TD><TD>j23100+luxR</TD><TD>ggccgcatctagagttgacg</TD><TD>20</TD></TR><TR><TD>JT-r </TD><TD>JT-plasmid</TD><TD>j23100+luxR</TD><TD>ggtctctagtataaacgcagaaaggc </TD><TD>26</TD></TR><TR><TD>PI-f</TD><TD>PI-plasmid</TD><TD>luxpR+luxI</TD><TD>Gcgtttatactagagacctgtaggatcg</TD><TD>28</TD></TR><TR><TD>gfra-r</TD><TD>GFP-plasmid</TD><TD>gfra</TD><TD>ctagtattattaAGCAGCCAGTGCATAGTTCTCG</TD><TD>34</TD></TR><TR><TD>jtpig-f</TD><TD>JTPI-plasmid</TD><TD>j23100+luxR+luxpR+luxI</TD><TD>ccgctgcagcggctgctaacaaagc </TD><TD>25</TD></TR><TR><TD>jtpig-r</TD><TD>JTPI-plasmid</TD><TD>j23100+luxR+luxpR+luxI</TD><TD>cgcgaattcatctcgatcctctacgc</TD><TD>26</TD></TR><TR><TD>jtp-f</TD><TD>JTP-plasmid</TD><TD>j23100+luxR+luxpR</TD><TD>ccagtagtaggttgagcatctagagttgacg</TD><TD>31</TD></TR><TR><TD>jtp-r</TD><TD>JTP-plasmid</TD><TD>j23100+luxR+luxpR</TD><TD>cttcttaaagttaaacccgctactagtatttattcg</TD><TD>36</TD></TR><TR><TD>psb-new-f</TD><TD>pSB1C3-plamid</TD><TD>pSB1C3-plamidbackbone</TD><TD>TGGCTGCTtaataatactagtagcggccgctgc</TD><TD>33</TD></TR><TR><TD>psb-new-r</TD><TD>pSB1C3-plamid</TD><TD>pSB1C3-plamidbackbone</TD><TD>gcatctagaagcggccgcgaattcc</TD><TD>25</TD></TR><TR><TD>gfp-new-f</TD><TD>GFP-plasmid</TD><TD>GFP+ssrA-tag</TD><TD>tctggaattcgcggccgcttctagatgc </TD><TD>28</TD></TR><TR><TD>gfp-snew1-r</TD><TD>GFP-plasmid</TD><TD>GFP+ssrA-tag</TD><TD>TAGTTCTCGTCGTTAGCTGCtttgtatagttcatccatgcc</TD><TD>41</TD></TR><TR><TD>gfp-snew2-r </TD><TD>GFP-plasmid</TD><TD>GFP+ssrA-tag</TD><TD>tagtattattaAGCAGCCAGTGCATAGTTCTCGTCGTTAGC</TD><TD>41</TD></TR><TR><TD>nic-pet-f</TD><TD>nicA2-plasmid</TD><TD>nicA2</TD><TD>ccatcatcatcatcatcacATGAGCGATAAGACC</TD><TD>34</TD></TR><TR><TD>nic-pnao-r</TD><TD>nicA2-plasmid</TD><TD>nicA2</TD><TD>AGTCCATgttgtggtttctcctctttcttaaTTAGC</TD><TD>36</TD></TR><TR><TD>pnao-nic-f</TD><TD>pnao-plasmid</TD><TD>pnao</TD><TD>aggagaaaccacaacATGGACTACAAGGACG</TD><TD>31</TD></TR><TR><TD>pnao-sapd1-r</TD><TD>pnao-plasmid</TD><TD>pnao</TD><TD>TAGTATTCTCCTCTTTAATCACGGCTTTACGCG</TD><TD>33</TD></TR><TR><TD>pnao-sapd2-r</TD><TD>pnao-plasmid</TD><TD>pnao</TD><TD>GGTAGTCACGCATCTAGTATTCTCCTCTTTAATCACG</TD><TD>37</TD></TR><TR><TD>sapd-pnao1-f</TD><TD>sapd-plasmid</TD><TD>sapd</TD><TD>GATTAAAGAGGAGAATACTAGATGCGTGACTACCG</TD><TD>35</TD></TR><TR><TD>sapd-pnao2-f</TD><TD>sapd-plasmid</TD><TD>sapd</TD><TD>ATTAAAGAGGAGAATACTAGATGCGTGACTACCGTAAG</TD><TD>38</TD></TR><TR><TD>sapd-pnao3-f</TD><TD>sapd-plasmid</TD><TD>sapd</TD><TD>TACTAGATGCGTGACTACCGTAAGTTCTATATCAACG</TD><TD>37</TD></TR><TR><TD>sapd-pet1-r</TD><TD>sapd-plasmid</TD><TD>sapd</TD><TD>gggctttgttagcagccgTTACG</TD><TD>23</TD></TR><TR><TD>sapd-pet2-r</TD><TD>sapd-plasmid</TD><TD>sapd</TD><TD>tttcgggctttgttagcagccg</TD><TD>22</TD></TR><TR><TD>T7</TD><TD>genes onpet28-plamid</TD><TD>TAATACGACTCACTATAGGG</TD><TD>20</TD></TR><TR><TD>TER</TD><TD>genes onpet28-plamid</TD><TD>GCTAGTTATTGCTCAGCGG</TD><TD>19</TD></TR><TR><TD>nic-part-f</TD><TD>nicA2-plasmid</TD><TD>nicA2</TD><TD>ggccgcttctagagATGAGCGATAAGACC </TD><TD>29</TD></TR><TR><TD>nic-part-r</TD><TD>nicA2-plasmid</TD><TD>nicA2</TD><TD>gccgctactagtaTTAGCTCAGCAGTTGC</TD><TD>29</TD></TR><TR><TD>psb-only-f </TD><TD>pSB1C3-plamid</TD><TD>pSB1C3-plamidbackbone</TD><TD>tactagtagcggccgctgc </TD><TD>19</TD></TR><TR><TD>psb-only-r</TD><TD>pSB1C3-plamid</TD><TD>pSB1C3-plamidbackbone</TD><TD>ctctagaagcggccgcg</TD><TD>17</TD></TR><TR><TD>pnao-part-f</TD><TD>pnao-plasmid</TD><TD>pnao</TD><TD>ccgcttctagagATGACCAAAGACGGCgaTGAGG</TD><TD>34</TD></TR><TR><TD>pnao-part-r</TD><TD>pnao-plasmid</TD><TD>pnao</TD><TD>gccgctactagtaTTACGCGTTGTCGTTCTCACG</TD><TD>34</TD></TR><TR><TD>sapd-part-f</TD><TD>sapd-plasmid</TD><TD>sapd</TD><TD>ggccgcttctagagATGCGTGACTACCG</TD><TD>28</TD></TR><TR><TD>sapd-part-r</TD><TD>sapd-plasmid</TD><TD>sapd</TD><TD>cgctactagtaTTAGCAACCCATAATGC</TD><TD>28</TD></TR><TR><TD>nic-XhoI-f</TD><TD>nicA2-plasmid</TD><TD>nicA2</TD><TD>CCGCTCGAGATGAGCGATAAGACC</TD><TD>20</TD></TR><TR><TD>nic-PstI-r</TD><TD>nicA2-plasmid</TD><TD>nicA2</TD><TD>TGCACTGCAGTTAGCTCAGCAGTTGC</TD><TD>22</TD></TR><TR><TD>nic-petn-f</TD><TD>nps-plasmid</TD><TD>nicA2+pnao+sapd+pet28</TD><TD>gtttaactttaagaaggagatataccatgggc</TD><TD>32</TD></TR><TR><TD>pet-new-r</TD><TD>nps-plasmid</TD><TD>nicA2+pnao+sapd+pet28</TD><TD>tgcccatggtatatctcc</TD><TD>18</TD></TR><TR><TD>psb-Eco-f</TD><TD>genessynthesized by the IDT</TD><TD>ndh,hdnoR, hdno, hlno</TD><TD>gatttctggaattcgcggcc</TD><TD>20</TD></TR><TR><TD>psb-Pst-r</TD><TD>genessynthesized by the IDT</TD><TD>ndh,hdnoR, hdno, hlno</TD><TD>gccggactgcagcgg </TD><TD>15</TD></TR><TR><TD>gfp-fk1-f</TD><TD>GFP-plasmid</TD><TD>GFP+ssrA-tag</TD><TD>gatgcgtaaaggagaagaacttttcactgg</TD><TD>30</TD></TR><TR><TD>gfp-fk1-r</TD><TD>GFP-plasmid</TD><TD>GFP+ssrA-tag</TD><TD>GTCGTTAGCTGCtttgtatagttcatccatgccatg</TD><TD>36</TD></TR><TR><TD>gfp-fk2-f</TD><TD>GFP-plasmid</TD><TD>GFP+ssrA-tag</TD><TD>gccgcttctagatgcgtaaaggagaag</TD><TD>27</TD></TR><TR><TD>gfp-fk2-r</TD><TD>GFP-plasmid</TD><TD>GFP+ssrA-tag</TD><TD>CCAGTGCATAGTTCTCGTCGTTAGCTGCtttg</TD><TD>32</TD></TR><TR><TD>gfp-fk3-f</TD><TD>GFP-plasmid</TD><TD>GFP+ssrA-tag</TD><TD>tcgcggccgcttctagatgc</TD><TD>20</TD></TR><TR><TD>gfp-fk3-r</TD><TD>GFP-plasmid</TD><TD>GFP+ssrA-tag</TD><TD>agtattattaAGCAGCCAGTGCATAGTTCTCG </TD><TD>32</TD></TR><TR><TD>psb-Spe-r</TD><TD>genessynthesized by the IDT</TD><TD>ndh,hdnoR, hdno, hlno</TD><TD>tgcagcggccgctactagtaTCAGG</TD><TD>25</TD></TR><TR><TD>6hdno-f</TD><TD>hdnosynthesized by the IDT</TD><TD>hdno </TD><TD>gccgcttctagagTGACAAGG</TD><TD>21</TD></TR><TR><TD>6hdno-r</TD><TD>hdnosynthesized by the IDT</TD><TD>hdno </TD><TD>ccgctactagtaTCAGGATCCTTCC</TD><TD>25</TD></TR><TR><TD>6hlno-r</TD><TD>hlnosynthesized by the IDT</TD><TD>hlno </TD><TD>ccgctactagtaCTAACTATGAAGAATAGC</TD><TD>30</TD></TR><TR><TD>moa-psb-f</TD><TD>E. coligenome </TD><TD>Coenzymemoa</TD><TD>gccgcttctagagATGCTCGTAAAGATTGCC</TD><TD>31</TD></TR><TR><TD>moa-psb-r</TD><TD>E. coligenome </TD><TD>Coenzymemoa</TD><TD>cgctactagtaCTACCAGCGTTTTGCCG</TD><TD>28</TD></TR><TR><TD>moc-psb-f</TD><TD>E. coligenome </TD><TD>Coenzymemoc</TD><TD>gccgcttctagagATGTCAGCCATCG</TD><TD>26</TD></TR><TR><TD>moc-psb-r</TD><TD>E. coligenome </TD><TD>Coenzymemoc</TD><TD>cggccgctactagtaCTAAATTTCAGTATACCTTTCC</TD><TD>37</TD></TR><TR><TD>mog-psb-f</TD><TD>E. coligenome </TD><TD>Coenzymemog</TD><TD>ggccgcttctagagATGAATACTTTACGTATTGGC</TD><TD>35</TD></TR><TR><TD>mog-psb-r</TD><TD>E. coligenome </TD><TD>Coenzymemog</TD><TD>gccgctactagtaTTATTCGCTAACGTCGC </TD><TD>30</TD></TR><TR><TD>JTPI-SphI-f</TD><TD>JTPI-plasmid</TD><TD>j23100+luxR+luxpR+luxI</TD><TD>ggtgcatgcagttgacggctagc</TD><TD>23</TD></TR><TR><TD>JTPI-XbaI-r</TD><TD>JTPI-plasmid</TD><TD>j23100+luxR+luxpR+luxI</TD><TD>tatttctagattgccggactgcagc</TD><TD>25</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>