Team:USTC/Project/Result
HOME
PROJECT
Description
Design
Results
MODEL
Overview
Single-cell Model
System Analysis
Effectiveness
Parameters
INTERLAB
PARTS
Basic Parts
Composite Parts
Contributions
Improve
NOTEBOOK
Monthly Diary
Protocols
Materials
HUMAN PRACTICE
Overview
Survey
Analyze
Practice
Teamwork
COLLABORATION
TEAM
Nav
Page
Page
Page
Page
Sensing system
We obtained the sequence of genes we needed, then made the codon optimization for these genes and added two Restriction sites for EcoRI and PstI.. The VppA and hlno are synthesized by GenScript, and the ndh hdnoR, hdno are synthesized by IDT. Then we inserted the genes to plasmid pSB1C3, and transformed these plasmids into E.coli (top 10). We selected some colonies for culture and confirmed the transformation results by double digestion after plasmid extraction (shown in Figure 1). From the result of electrophoresis, we confirmed the transformation of VppA, hdnoR and hlno.
Figure 1:Electrophoresis result of double digestion of VppA, hdnoR, hlno plasmid.
We tried to select the most suitable RBS and promotor for our sensing system under the guidance of modeling groups. We got the Promoter J23107 for VppA, the Promotor J23105 for hdnoR, and the RBS B0034 for the whole sensing system, which are all from the iGEM Distribution Kit.
In order to improve the accuracy of our sensing system, we added SsrA tag on C terminus of the Green Fluorescent Protein. We amplified the GFP genes with the primers designed by ourselves, then inserted the gene into the pSB1A2 and transformed this plasmid into E.coli(DH5α). The sequence of the gene was validated with DNA sequencing by Sangon Biotech, which indicated that we successfully constructed the plasmid.
Since the maturation of VppA in our sensing system requires the presence of a molybdenum cofactor, we need to construct a series of moX plasmids. We amplified genes moX from the genome of E.coli (BL21) by PCR. Then we separately inserted these genes to plasmid pSB1C3. The sequence of gene moc and mog was validated with DNA sequencing by Sangon Biotech. We transformed these plasmids into E.coli (BL21), respectively.We picked some colonies for cultivation and confirmed the transformation result by double digestion after plasmid extraction (shown in Figure 2 and Figure 3). From the result of electrophoresis, we confirmed the transformation of moc and mog.
Figure 2. Electrophoresis result of double digestion of moc plasmid.
Figure 3. Electrophoresis result of double digestion of mog plasmid.
Regulation system
1. Plasmid construction
We got the genes: Promotor j23100, RBS B0034, luxR C0062, luxI C0061, lux pR R0062 from the iGEM Distribution Kit. Then we assembled the standard Bio-brick using the four restriction enzymes: EcoRI, XbaI, SpeI, PstI. We performed double digestion on JTPI plasmid using EcoRI and PstI, which were only on the plasmid backbone, and the result shown in Figure 1. From the result of electrophoresis, we confirmed the of JTPI.
Considering that the positive feedback may make our system unstable, we constructed another plasmid without  LuxI gene, and the result of double digestion on JTP shown in Figure 1.
Figure 1. Electrophoresis result of double digestion of JTPI and JTP plasmid.
We confirmed our plasmids by genetic sequencing in Sangon Biotech afterwards.
Since the data from modeling simulation indicated that the copy number of the plasmid also made an impact on the feasibility of our system, we constructed the same system on the pET28b. We amplified the Regulation System from the JTPI plasmid and the pET28b plasmid backbone, then we assembled two gene fragments using the Seamless Cloning Master Mix. We transformed the plasmid into E.coli(BL21).
In order to verify the function of our system, we constructed the plasmid combined the regulation system and GFP. From the result of electrophoresis shown in Figure 2, we confirmed that JTPIG and JTPG were successfully constructed.
Figure 2. Electrophoresis result of double digestion of JTPIG and JTPG plasmid.
2. RFI curve and growth curve of E.coli with JTPI/JTP
To figure out the impact of gene LuxI and different plasmids on the function of our system in E.coli. We measured relative fluorescent intensity (RFI) and OD600 at various time respectively. Then we plotted the scatter graph and drew the RFI curve and growth curve in B-spline (shown in Figure 4 and Figure 5).
Figure 4.  RFI curve of E.coli with JTPI/JTP
Figure 5. growth curve of E.coli with  JTPI/JTP
According to the graph of RFI curve and growth curve of E.coli with JTPI/JTP, we can reach these conclusions:
(a) JTPI on pSB1C3 is a plasmid constructed by inserting JTPI into pSB1C3 with GFP downstream. The RFI curve of JTPI on pSB1C3 indicated that the leaky expression of LuxI in JTPI on pSB1C3 was too severe, resulting in high expression of GFP. At the same time, the growth curve showed that the bacterial density was lower compared with others, indicating too much growth pressure of E. coli because of the leaky expression of LuxI. For these reasons, inserting JTPI into pSB1C3 was not a good solution for controlling expression leakage.
(b) JTP on pSB1C3 is a plasmid constructed by inserting JTP into pSB1C3 with GFP downstream. The RFI curve of JTP on pSB1C3 indicated that the low expression of green fluorescent protein was due to the absence of  LuxI gene. Meanwhile, the growth curve indicated that the growth pressure of E. coli was not so severe, so bacterial density was higher than JTPI on pSB1C3.
(c)  JTPI on pET28 is a plasmid in which we inserted JTPI into pET28 with GFP downstream. As far as we know, the copy number of pET28 is less than the copy number of pSB1C3. The comparison of the RFI curve of JTPI on pET28 with JTP on pSB1C3 indicated that it can meet the needs of our regulation system. At the same time, the growth curve of JTPI on pET28 was not much different compared with the growth curve of JTP on pSB1C3, so the growth pressure of E. coli was not so severe. For these reasons, we can decrease expression leakage via inserting JTPI into pET28.
In summary, we decided to insert JTPI into pET28 and we finally successfully constructed this plasmid.
Degradation system
1. Plasmid Construction
We obtained the sequence of genes of three enzymes (nicA2, pnao and sapd) and made the codon optimization for these genes. Then we ordered these gene synthesis from GenScript. We separately inserted these genes into plasmid pET28b(+) with promoter  T7 upstream successfully. The sequence of genes was validated with DNA sequencing by Sangon Biotech. We separately transformed these plasmids into E.coli (BL21). We picked some colonies for cultivation and confirmed the transformation result by PCR (shown in Figure 1). From the result of electrophoresis, we confirmed that the transformation was successful.
Figure 1. Electrophoresis result of PCR of nicA2, pnao and sapd.
To combine the NPS (nicA2 & pnao & sapd) genes together, we inserted the genes of three enzymes to one plasmid pET28b(+) and added promoter J23100 upstream which was from distribution kit. The sequence of gene NPS was validated with DNA sequencing by Sangon Biotech. We transformed this NPS plasmid into E.coli (BL21). Then we picked some colonies for cultivation and confirmed the transformation result by double digestion after plasmid extraction (shown in Figure 2). From the result of electrophoresis, we confirmed the transformation of NPS plasmid.
Figure 2. Electrophoresis result of double digestion of NPS plasmid.
2. Expression and purification
NicA2
We used 200 mL LB to cultivate our bacteria in 37℃, 150 rpm till its OD600 reached 0.8. We added IPTG (final concentration=0.3mM) to induce nicA2 expression in 25℃ for 6 hours. After that we prepared the cell lysis and used Ni-resin for purification.
Figure 3.SDS-PAGE for strain expressing nicA2.
As figure3 showed, after purification by Ni-resin, there was only one band, whose location was about 60kDa according to the protein marker. It proved that we had successfully expressed nicA2 and collected pure nicA2 with 6XHis-tag.
Pnao
We used 200 mL LB to cultivate our bacteria in 37℃, 150 rpm till its OD600 reached 0.8. We added IPTG (final concentration=0.5mM) to induce pnao expression in 20℃ for 20 hours. After that we prepared the cell lysis and used Ni-resin to purify it.
Figure 4.SDS-PAGE for strain expressing pnao.
As the figure showed, we had successfully induced the expression of pnao with 6XHis-tag, and it was purified well.
Sapd
We used 200 mL LB to cultivate our bacteria in 37℃, 150 rpm till its OD600 reached 0.8. We added IPTG (final concentration=1mM) to induce pnao expression in 16℃ for 20 hours. After that we prepared the cell lysis and purified it with Ni-resin.
Figure 5.SDS-PAGE for strain expressing sapd.
As we can see, we had successfully induced the expression of sapd. Also, we had purified sapd with 6XHis-tag.
NPS (nicA2 & pnao & sapd)
We cultivated the bacteria in 200mL LB medium in 37℃, 150 rpm till OD600 reached 0.8. Then we added IPTG (final concentration=1mM) to induce their expression in 37℃ for 3 hours.
Figure 6.SDS-PAGE for strain expressing nicA2, pnao and sapd.
As we can see, we had expressed nicA2, pnao, and sapd together successfully.
