<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cambridge_Experiments_Squid_Dissection_and_Tissue_Sample"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cambridge/Experiments/Squid_Dissection_and_Tissue_Sample">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cambridge/Experiments/Squid_Dissection_and_Tissue_Sample&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Experiments/Squid_Dissection_and_Tissue_Sample&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Experiments/Squid_Dissection_and_Tissue_Sample&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cambridge/Experiments/Squid_Dissection_and_Tissue_Sample" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cambridge/Experiments/Squid Dissection and Tissue Sample</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="cam_page"><DIV id="cam_loading"><DIV>Loading...</DIV></DIV><DIV id="cam_topbar" class="cam-box"><DIV style="margin: 0px 12px; width:160px;float: left;"><A href="https://2011.igem.org/Main_Page">
			iGEM HQ
		</A></DIV><DIV style="margin: 0px 12px;float:left;"><A id="edit-link" class="cam-external" href="#">
			Page Source
		</A><A id="edit-intro-link" class="cam-external" href="#">
			Intro Source
		</A></DIV><DIV id="cam_login_status" style="margin: 0px 12px;float:right;"><SPAN id="logged_out"><A id="login-link" class="cam-external" href="#">
			Login
		</A></SPAN><SPAN id="logged_in" style="display:none;"><A id="account-link" class="cam-external" href="https://igem.org/User_Information">
			My Account
		</A><A id="logout-link" class="cam-external" href="/wiki/index.php?title=Special:UserLogout">
			Logout
		</A></SPAN></DIV></DIV><DIV id="cam_left"><DIV id="cam_nav" class="cam-box"><DIV id="cam_desc"><DIV id="cam_desc_top"><DIV id="cam_desc_title"><DIV id="cam_desc_area">OVERVIEW</DIV><DIV id="cam_desc_subarea">home</DIV></DIV></DIV></DIV></DIV></DIV><DIV id="cam_right"><DIV id="cam_menubar"><DIV id="cam_overview" class="cam-menu-item cam-box cam-overview"><A href="/Team:Cambridge"><DIV class="cam-menu-top"><DIV class="cam-menu-text">OVERVIEW</DIV></DIV></A><DIV class="cam-menu-all cam-subitem cam-overview"><UL class="cam-menu-drop"><LI class="cam-overview-home"><A href="/Team:Cambridge">HOME</A></LI><LI class="cam-overview-parts"><A href="/Team:Cambridge/Parts">PARTS</A></LI><LI class="cam-overview-media"><A href="/Team:Cambridge/Media">MEDIA</A></LI><LI class="cam-overview-blog"><A href="/Team:Cambridge/Blog">BLOG</A></LI></UL></DIV></DIV><DIV id="cam_project" class="cam-menu-item cam-box cam-project"><A href="/Team:Cambridge/Project"><DIV class="cam-menu-top"><DIV class="cam-menu-text">PROJECT</DIV></DIV></A><DIV class="cam-menu-all cam-subitem cam-project"><UL class="cam-menu-drop"><LI class="cam-project-overview"><A href="/Team:Cambridge/Project">OVERVIEW</A></LI><LI class="cam-project-background"><A href="/Team:Cambridge/Project/Background">BACKGROUND</A></LI><LI class="cam-project-microscopy"><A href="/Team:Cambridge/Project/Microscopy">MICROSCOPY</A></LI><LI class="cam-project-invivo"><A href="/Team:Cambridge/Project/In_Vivo">IN VIVO</A></LI><LI class="cam-project-invitro"><A href="/Team:Cambridge/Project/In_Vitro">IN VITRO</A></LI><LI class="cam-project-gibthon"><A href="/Team:Cambridge/Project/Gibthon">GIBTHON</A></LI><LI class="cam-project-future"><A href="/Team:Cambridge/Project/Future">FUTURE</A></LI></UL></DIV></DIV><DIV id="cam_labwork" class="cam-menu-item cam-box cam-labwork"><A href="/Team:Cambridge/Labwork"><DIV class="cam-menu-top"><DIV class="cam-menu-text">LABWORK</DIV></DIV></A><DIV class="cam-menu-all cam-subitem cam-labwork"><UL class="cam-menu-drop"><LI class="cam-labwork-overview"><A href="/Team:Cambridge/Labwork">OVERVIEW</A></LI><LI class="cam-labwork-experiments"><A href="/Team:Cambridge/Experiments">EXPERIMENTS</A></LI><LI class="cam-labwork-protocols"><A href="/Team:Cambridge/Protocols">PROTOCOLS</A></LI><LI class="cam-labwork-constructs"><A href="/Team:Cambridge/Experiments/Plasmid_Constructs">CONSTRUCTS</A></LI><LI class="cam-labwork-safety"><A href="/Team:Cambridge/Safety">SAFETY</A></LI></UL></DIV></DIV><DIV id="cam_society" class="cam-menu-item cam-box cam-society"><A href="/Team:Cambridge/Society"><DIV class="cam-menu-top"><DIV class="cam-menu-text">SOCIETY</DIV></DIV></A><DIV class="cam-menu-all cam-subitem cam-society"><UL class="cam-menu-drop"><LI class="cam-society-overview"><A href="/Team:Cambridge/Society">OVERVIEW</A></LI><LI class="cam-society-interviews"><A href="/Team:Cambridge/Society/interviews">INTERVIEWS</A></LI><LI class="cam-society-report"><A href="/Team:Cambridge/Society/report">IMPACT OF iGEM</A></LI></UL></DIV></DIV><DIV id="cam_team" class="cam-menu-item cam-box cam-team"><A href="/Team:Cambridge/Team"><DIV class="cam-menu-top"><DIV class="cam-menu-text">TEAM</DIV></DIV></A><DIV class="cam-menu-all cam-subitem cam-team"><UL class="cam-menu-drop"><LI class="cam-team-overview"><A href="/Team:Cambridge/Team">OVERVIEW</A></LI><LI class="cam-team-students"><A href="/Team:Cambridge/Team/Students">STUDENTS</A></LI><LI class="cam-team-academics"><A href="/Team:Cambridge/Team/Academics">ACADEMICS</A></LI><LI class="cam-team-sponsors"><A href="/Team:Cambridge/Team/Sponsors">SPONSORS</A></LI></UL></DIV></DIV></DIV><DIV class="cam-box"><DIV id="cam_pagecontent" class="cam-box-content"><DIV id="wiki_content">

Back to <A href="/Team:Cambridge/Experiments">Experiments</A><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Amplification_of_Reflectin_Genes_from_Squid_Genomic_DNA"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Amplification of Reflectin Genes from Squid Genomic DNA</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Summary"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Summary</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Detail"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Detail</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Squid_Dissection"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Squid Dissection</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#DNA_Extraction"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">DNA Extraction</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Design_of_Primers"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Design of Primers</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#PCR_Reaction"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">PCR Reaction</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Gel_Electrophoresis"><SPAN class="tocnumber">3.5</SPAN><SPAN class="toctext">Gel Electrophoresis</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#References"><SPAN class="tocnumber">3.6</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Amplification_of_Reflectin_Genes_from_Squid_Genomic_DNA">Amplification of Reflectin Genes from Squid Genomic DNA</SPAN></H2><P>Before we could begin with the rest of our project, we needed the reflectin coding region. <A href="http://en.wikipedia.org/wiki/Loligo" class="external text" rel="nofollow"><I>Loligo</I></A> tissue was sourced from fishing bait suppliers and culinary wholesalers in order to attempt genomic DNA extraction. Our literature search indicated that the reflectin gene from <I>E. scolopes</I> contains no introns, so genomic DNA should be suitable for expression in <I>E. coli</I>.
</P><H2><SPAN class="mw-headline" id="Summary">Summary</SPAN></H2><P>We aimed to <A href="/Team:Cambridge/Protocols/Extraction_of_genomic_DNA_from_squid" title="Team:Cambridge/Protocols/Extraction of genomic DNA from squid"> extract DNA</A> from dissected squid tissue, amplify the relevant gene by <A href="/Team:Cambridge/Protocols/PCR" title="Team:Cambridge/Protocols/PCR"> PCR</A> and perform <A href="/Team:Cambridge/Protocols/Gel_Electrophoresis" title="Team:Cambridge/Protocols/Gel Electrophoresis"> gel electrophoresis</A> to verify that we had successfully extracted reflectin.
</P><P>Our literature search suggested that (for reasons unknown) cephalopod DNA is difficult to isolate and clone from. Since we did not have any other way of getting our reflectin gene, we tried two different <A href="/Team:Cambridge/Protocols/Extraction_of_genomic_DNA_from_squid" title="Team:Cambridge/Protocols/Extraction of genomic DNA from squid"> DNA extraction protocols</A>, in the hope that one would work successfully. One extraction protocol appeared to work, while the other left us with DNA which was pink – a sure sign that it was heavily contaminated.
</P><P>We proceeded with PCR on the successful extraction. By designing multiple sets of primers for each different DNA sample, it was hoped that we would see some successful amplification. However, by running <A href="/Team:Cambridge/Protocols/Gel_Electrophoresis" title="Team:Cambridge/Protocols/Gel Electrophoresis"> gel electrophoresis</A> we found that none of our samples had been successfully amplified.
</P><P>Whether this was due to our DNA extraction or due to a PCR problem soon became a moot point, as we were kindly sent synthetic reflectin genes by Wendy Crookes-Goodson, author of many of the papers on reflectin.
</P><H2><SPAN class="mw-headline" id="Detail">Detail</SPAN></H2><H3><SPAN class="mw-headline" id="Squid_Dissection">Squid Dissection</SPAN></H3><P>Specimens of what we identified as <A href="http://www.marlin.ac.uk/speciesinformation.php?speciesID=3718" class="external text" rel="nofollow"><I>Loligo vulgaris</I></A> and <A href="http://en.wikipedia.org/wiki/Opalescent_Inshore_Squid" class="external text" rel="nofollow"><I>Loligo opalescens</I></A> were dissected for examination.  Samples of skin tissue and eye cups were taken from <I>L. vulgaris</I> for further imaging.  No obvious iridescence was seen in the skin sample under a dissection microscope or by confocal microscopy, (though this is known to require the neurotransmitter acetyl choline in order to be activated). In accordance with <A href="http://rsif.royalsocietypublishing.org/content/early/2011/02/14/rsif.2010.0702.full" class="external text" rel="nofollow">Holt <I>et al</I></A>, we found clearly visible static iridescence in the tissue surrounding the eye lens in the form of the silvery tissue exhibiting a sheen of colours from across the visible spectrum.
</P><CENTER><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 155px;"><DIV class="gallerytext"><P>Preparing squid samples
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 155px;"><DIV class="gallerytext"><P><I>Loligo opalescens</I></P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 155px;"><DIV class="gallerytext"><P><I>L. vulgaris</I> eye cup, showing Bragg reflectance
</P></DIV></DIV></TD></TR></TBODY></TABLE></CENTER><H3><SPAN class="mw-headline" id="DNA_Extraction">DNA Extraction</SPAN></H3><P>To isolate squid genomic DNA as a template for PCR, samples of squid tissue were cut from representative specimens.  Internal organs and tissue were dissected with the aim of minimizing external contamination. <I>L. vulgaris</I> specimens were obtained from two different sources, so samples were taken from both.  
</P><P>A <A href="/Team:Cambridge/Protocols/Extraction_of_genomic_DNA_from_squid" title="Team:Cambridge/Protocols/Extraction of genomic DNA from squid"> DNA extraction protocol</A> was followed to isolate genomic DNA from tissue.
</P><H3><SPAN class="mw-headline" id="Design_of_Primers">Design of Primers</SPAN></H3><P>We decided to perform PCR in order to isolate and amplify reflectin coding sequences from genomic DNA of <I>Loligo vulgaris</I> and <I>Loligo opalescens</I>.
As no reflectin gene sequences from the two species that we worked with are known, we designed primers relying on the published sequences of A1, A2 and B1 reflectin mRNAs from <I>Loligo pealei</I>. 
</P><P>The assumptions that we made and the way of our reasoning are the following: 
</P><P><B>1. Because of close evolutionary relationship and an important role of reflectin proteins in the animal physiology and survival, we expect other squids from the Loligo genus to express the same, or very similar, set of reflectin proteins.</B></P><DL><DD>Although the phylogeny of the loliginid squids has not been fully revealed yet, a fairly consistent picture of evolutionary relationships within the taxon has emerged from several studies, which included:
<UL><LI>comparison of sequences of two mitochondrial genes (the 16S rRNA and the cytochrome c oxidase subunit I genes) between members of 19 loliginid species and several outgroups (Anderson, 2000 [1]);
</LI><LI>comparison of multiple data sets, such as morphology, allozymes and DNA seqence data for the two mitochondrial genes (Anderson, 2000 [2]);
</LI></UL></DD><DD>The fairly reliable phylogenetic tree of the loliginid squids is presented below:
</DD></DL><DL><DD>According to the phylogenetic tree, <I>L. pealei</I> is fairly closely related to <I>L. opalescens</I>, and <I>L. vulgaris</I> shows close evolutionary relationship with <I>L. forbesi</I>, whose sequence of reflectin-like protein mRNA is also published in the GenBank database. However, the exact evolutionary history of the two taxons is not resolved yet.  
</DD></DL><P><B>2. As the amino acid sequence of proteins is generally more conserved than nucleotide sequence, we considered designing degenerate primers, using <A href="http://www.kazusa.or.jp/codon/cgi-bin/spsearch.cgi?species=loligo&amp;c=i" class="external text" rel="nofollow">codon usage tables</A> to maximize the likelihood of amplifying <I>L. opalescens</I> and <I>L. vulgaris</I> reflectin genes.</B></P><DL><DD>However, comparison of reflectin mRNA sequences of <I>Loligo pealei</I> and reflectin-like methionine-rich repeat protein 1 mRNA sequence from <I>Loligo forbesi</I> showed quite high conservation of the nucleotide sequence.
<UL><LI>We performed BLAST analysis using the methionine-rich repeat protein as a query and comparing it to A1 and A2 reflectin sequences from <I>Loligo pealei</I>. 
</LI><LI>After that, we analysed conservation of nucleotide sequences of the highly conserved protein regions.
</LI></UL></DD></DL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption"><I>Loligo pealei</I> reflectin-like A1 protein</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption"><I>Loligo pealei</I> reflectin-like A2 protein</DIV></DIV></DIV><P>Consequently, we decided not to take codon bias into consideration and design primers complementary to terminal sequences of reflectin mRNAs from <I>L. pealei</I>, so that they would allow us to amplify the entire coding sequence, from the START to the STOP codon. 
</P><TABLE border="1px" align="center" style="text-align:center;"><TBODY><TR><TH scope="col" width="80"> Primer
</TH><TH scope="col" width="220"> Reflectin A1
</TH><TH scope="col" width="220"> Reflectin A2
</TH><TH scope="col" width="220"> Reflectin B1
</TH></TR><TR><TD><B>Forward</B></TD><TD>  5’ ATGAATCGAT ATCTGAATCG 3’
</TD><TD>  5’ ATGAATCGCT ACATGATGAG 3’
</TD><TD>  5’ ATGTCTTCTT TTATGGATCC 3’
</TD></TR><TR><TD><B>Reverse</B></TD><TD>  5’ TTAATACATG TGATAGTCGT 3’
</TD><TD>  5’ CTAATACCAA GAATTGTAAT 3’
</TD><TD>  5’ TTAGGCTGAA TCTGTGAGCT 3’
</TD></TR></TBODY></TABLE><P>It is worth emphasizing that the published sequences are mRNA sequences, giving us no information about endogenous promoters or 5'UTR and 3'UTR regions. Additionally, they do not provide any information about the presence and position of introns either. 
</P><UL><LI>However, according to the literature (Crookes, 2004), the bobtail squid reflectin genes do not contain introns, and thus, assuming the common evolutionary origin, reflectin genes from other squid species are likely not to include introns as well. 
</LI><LI>Thus, high deviation of PCR products from the expected length will give us a clue about the presence of intro sequences, although this might be also caused by other factors, such as:
</LI><LI>interspecific variation
</LI><LI>misannealing and amplification of other sequences than reflectin genes
</LI></UL><H3><SPAN class="mw-headline" id="PCR_Reaction">PCR Reaction</SPAN></H3><P>The next step involved PCR reaction with Phusion Hot Start II DNA Polymerase, according to the following <A href="/Team:Cambridge/Protocols/PCR" title="Team:Cambridge/Protocols/PCR"> protocol</A>. We chose a modified version of PCR - gradient PCR, which allowed us to test different annealing temperatures. As we did not expect the primers to show complete fidelity to the coding sequences of the analyzed species, different conditions of annealing would ensure that we would amplify the most matching sequences from the genomic DNA template.
</P><P>We prepared 72 PCR reactions:
</P><TABLE align="center" style="text-align:center;"><TBODY><TR><TD scope="col" width="300"><B>genomic DNA template</B> from three different squids: two <I>L. vulgaris</I> individuals and one <I>L. opalescens</I> individual
</TD><TD scope="col" width="20">×
</TD><TD scope="col" width="300"><B>three sets of primers: </B> A1, A2 and B1
</TD><TD scope="col" width="20">×
</TD><TD scope="col" width="300"><B>eight different annealing temperatures</B> ranging from 72°C to 50°C
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Gel_Electrophoresis">Gel Electrophoresis</SPAN></H3><P>We performed <A href="/Team:Cambridge/Protocols/Gel_Electrophoresis" title="Team:Cambridge/Protocols/Gel Electrophoresis"> gel electrophoresis</A> of the products of PCR reaction in order to check:
</P><UL><LI>how effective the reaction of amplification was,
</LI><LI>if the size of PCR products roughly matched the length of reflectin mRNAs from Loligo pealei, and
</LI><LI>how successful the DNA extraction protocol was.
</LI></UL><P>Samples run on the first gel included:
</P><TABLE border="1" style="text-align:center;"><TBODY><TR><TH scope="col" width="120"> Well
</TH><TH scope="col" width="120"> 1
</TH><TH scope="col" width="120"> 2
</TH><TH scope="col" width="120"> 3
</TH><TH scope="col" width="120"> 4
</TH><TH scope="col" width="120"> 5
</TH><TH scope="col" width="120"> 6
</TH></TR><TR align="center"><TD><B>Sample</B></TD><TD>HyperLadder I
</TD><TD><I>L. opalescens</I> A1 primers  72.0°C
</TD><TD><I>L. opalescens</I> A1 primers  68.9°C
</TD><TD><I>L. opalescens</I> A1 primers  65.7°C
</TD><TD><I>L. opalescens</I> A1 primers  62.6°C
</TD><TD><I>L. opalescens</I> A1 primers  59.4°C
</TD></TR><TR align="center"><TH><B>7</B></TH><TH><B>8</B></TH><TH><B>9</B></TH><TH><B>10</B></TH><TH><B>11</B></TH><TH><B>12</B></TH><TH><B>13</B></TH></TR><TR align="center"><TD><I>L. opalescens</I> A1 primers  56.3°C
</TD><TD><I>L. opalescens</I> A1 primers  53.1°C
</TD><TD><I>L. opalescens</I> A1 primers  50.0°C
</TD><TD>HyperLadder I
</TD><TD><I>L. opalescens</I> A2 primers  72.0°C
</TD><TD><I>L. opalescens</I> A2 primers  68.9°C
</TD><TD><I>L. opalescens</I> A2 primers  65.7°C
</TD></TR><TR align="center"><TH><B>14</B></TH><TH><B>15</B></TH><TH><B>16</B></TH><TH><B>17</B></TH><TH><B>18</B></TH><TH><B>19</B></TH><TH><B>20</B></TH></TR><TR align="center"><TD><I>L. opalescens</I> A2 primers  62.6°C
</TD><TD><I>L. opalsecens</I> A2 primers  59.4°C
</TD><TD><I>L. opalescens</I> A2 primers  56.3°C
</TD><TD><I>L. opalescens</I> A2 primers  53.1°C
</TD><TD><I>L. opalescens</I> A2 primers  50.0°C
</TD><TD>HyperLadder I
</TD><TD><I>L. opalescens</I> genomic DNA
</TD></TR></TBODY></TABLE><P>Note: HyperLadder I is a set of molecular weight markers that allow for quantification and size determination within the 200bp - 10,000bp range.
</P><P>As we can see from the picture of the gel:
</P><UL><LI>No bands are visible in the lanes loaded with the PCR products, and surprisingly neither primers nor template DNA can be detected either.
</LI><LI>There is no trace of DNA in the last lane (numer 20) which was loaded with the supernatant obtained during the squid DNA extraction process  
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">A picture of the first gel with wells numbered from the left to the right. Only three lanes with HyperLadder are visible.</DIV></DIV></DIV></DIV><P>
These observations suggest low efficiency of the extraction method. To check if this problem was unique to DNA from <I>Loligo opalescens</I>, we ran a second gel which included genomic DNA from the two <I>Loligo vulgaris</I> individuals as well as an array of PCR reactions conducted at different temperatures (65.7°C and 56.3°C) and with different sets of primers.
</P><TABLE border="1" style="text-align:center;"><TBODY><TR><TH scope="col" width="120"> Well
</TH><TH scope="col" width="120"> 1
</TH><TH scope="col" width="120"> 2
</TH><TH scope="col" width="120"> 3
</TH><TH scope="col" width="120"> 4
</TH><TH scope="col" width="120"> 5
</TH><TH scope="col" width="120"> 6
</TH></TR><TR align="center"><TD><B>Sample</B></TD><TD>HyperLadder I
</TD><TD><I>L. vulgaris lf</I> genomic DNA </TD><TD><I>L. vulgaris ba</I> genomic DNA </TD><TD><I>L. opalescens</I> B1 primers  65.7°C
</TD><TD><I>L. vulgaris lf</I> A1 primers  65.7°C
</TD><TD><I>L. vulgaris lf</I> A2 primers  65.7°C
</TD></TR><TR align="center"><TH><B>7</B></TH><TH><B>8</B></TH><TH><B>9</B></TH><TH><B>10</B></TH><TH><B>11</B></TH><TH><B>12</B></TH><TH><B>13</B></TH></TR><TR align="center"><TD><I>L. vulgaris lf</I> B1 primers  65.7°C
</TD><TD><I>L. vulgaris ba</I> A1 primers  65.7°C
</TD><TD><I>L. vulgaris ba</I> A2 primers  65.7°C
</TD><TD><I>L. vulgaris ba</I> B1 primers  65.7°C
</TD><TD>HyperLadder I
</TD><TD><I>L. opalescens</I> B1 primers  56.3°C
</TD><TD><I>L. vulgaris lf</I> A1 primers  56.3°C
</TD></TR><TR align="center"><TH><B>14</B></TH><TH><B>15</B></TH><TH><B>16</B></TH><TH><B>17</B></TH><TH><B>18</B></TH><TH><B>19</B></TH><TH><B>20</B></TH></TR><TR align="center"><TD><I>L. vulgaris lf</I> A2 primers  56.3°C
</TD><TD><I>L. vulgaris lf</I> B1 primers  56.3°C
</TD><TD><I>L. vulgaris ba</I> A1 primers  56.3°C
</TD><TD><I>L. vulgaris ba</I> A2 primers  56.3°C
</TD><TD><I>L. vulgaris ba</I> B1 primers  56.3°C
</TD><TD>HyperLadder I
</TD><TD><I>L.opalescens</I> genomic DNA
</TD></TR></TBODY></TABLE><P>Note: <I>L. vulgaris ba</I> stands for the <I>Loligo</I> squid obtained from fishing bait suppliers, while <I>L. vulgaris lf</I>  stands for the <I>Loligo</I> squid bought from the local culinary wholesaler. 
</P><P>We could see a similar pattern in the second gel:
</P><UL><LI>no products of PCR reactions, as well as primers and template DNA were visible,
</LI><LI>no smear of genomic DNA from <I>L. opalescens</I> and <I>L. vulgaris ba</I> was detected,
</LI><LI>however, we noticed a thin faint band of genomic DNA from <I>L. vulgaris lf</I>, but the detectable amount of the template DNA did not improve the efficiency of corresponding PCR reactions.
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">A close-up of the second gel with the HyperLadder I on the leftmost lane and a faint band of <I>L. vulgaris</I> genomic DNA in the adjacent lane.</DIV></DIV></DIV></DIV><P>We concluded that the DNA extraction protocol applied in this experiment was not very efficient and that this was the main reason why we failed to obtain any amplified reflectin or reflectin-related genes.
</P><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><OL><LI><B>Anderson F.E. (2000)</B> Phylogeny and historical biogeography of the loliginid squids (Mollusca: Cephalopoda) based on mitochondrial DNA sequence data. <I>Molecular Phylogenetics and Evolution</I>, 15: 191-214.
</LI><LI><B>Anderson F.E. (2000)</B> Phylogenetic relationships among loliginid squids (Cephalopoda: Myopsida) based on analyses of multiple data sets. <I>Zoological Journal of the Linnean Society</I>, 130: 603–633.
</LI><LI><B>Crookes W.J., Ding L., Huang Q.L., Kimbell J.R., Horwitz J., McFall-Ngai M.J. (2004)</B> Reflectins: the unusual proteins of squid reflective tissues. <I>Science</I>, 303: 235–238.
</LI></OL><P>

Back to <A href="/Team:Cambridge/Experiments">Experiments</A></P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Cambridge/Experiments/Squid_Dissection_and_Tissue_Sample">http://2011.igem.org/Team:Cambridge/Experiments/Squid_Dissection_and_Tissue_Sample</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cambridge/Experiments/Squid_Dissection_and_Tissue_Sample" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cambridge/Experiments/Squid_Dissection_and_Tissue_Sample" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cambridge/Experiments/Squid_Dissection_and_Tissue_Sample&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cambridge/Experiments/Squid_Dissection_and_Tissue_Sample&amp;oldid=166035" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>