<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cambridge_Experiments_Synthetic_Reflectin_PCR_and_Construction_of_GA1_to_6"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cambridge/Experiments/Synthetic_Reflectin_PCR_and_Construction_of_GA1_to_6">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cambridge/Experiments/Synthetic_Reflectin_PCR_and_Construction_of_GA1_to_6&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Experiments/Synthetic_Reflectin_PCR_and_Construction_of_GA1_to_6&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Experiments/Synthetic_Reflectin_PCR_and_Construction_of_GA1_to_6&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cambridge/Experiments/Synthetic_Reflectin_PCR_and_Construction_of_GA1_to_6" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cambridge/Experiments/Synthetic Reflectin PCR and Construction of GA1 to 6</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="contentSub">(Redirected from <A href="/wiki/index.php?title=Team:Cambridge/Experiments/Transformation_of_E.coli_with_Plasmids_from_Wendy&amp;redirect=no" title="Team:Cambridge/Experiments/Transformation of E.coli with Plasmids from Wendy">Team:Cambridge/Experiments/Transformation of E.coli with Plasmids from Wendy</A>)</DIV><DIV id="cam_page"><DIV id="cam_loading"><DIV>Loading...</DIV></DIV><DIV id="cam_topbar" class="cam-box"><DIV style="margin: 0px 12px; width:160px;float: left;"><A href="https://2011.igem.org/Main_Page">
			iGEM HQ
		</A></DIV><DIV style="margin: 0px 12px;float:left;"><A id="edit-link" class="cam-external" href="#">
			Page Source
		</A><A id="edit-intro-link" class="cam-external" href="#">
			Intro Source
		</A></DIV><DIV id="cam_login_status" style="margin: 0px 12px;float:right;"><SPAN id="logged_out"><A id="login-link" class="cam-external" href="#">
			Login
		</A></SPAN><SPAN id="logged_in" style="display:none;"><A id="account-link" class="cam-external" href="https://igem.org/User_Information">
			My Account
		</A><A id="logout-link" class="cam-external" href="/wiki/index.php?title=Special:UserLogout">
			Logout
		</A></SPAN></DIV></DIV><DIV id="cam_left"><DIV id="cam_nav" class="cam-box"><DIV id="cam_desc"><DIV id="cam_desc_top"><DIV id="cam_desc_title"><DIV id="cam_desc_area">OVERVIEW</DIV><DIV id="cam_desc_subarea">home</DIV></DIV></DIV></DIV></DIV></DIV><DIV id="cam_right"><DIV id="cam_menubar"><DIV id="cam_overview" class="cam-menu-item cam-box cam-overview"><A href="/Team:Cambridge"><DIV class="cam-menu-top"><DIV class="cam-menu-text">OVERVIEW</DIV></DIV></A><DIV class="cam-menu-all cam-subitem cam-overview"><UL class="cam-menu-drop"><LI class="cam-overview-home"><A href="/Team:Cambridge">HOME</A></LI><LI class="cam-overview-parts"><A href="/Team:Cambridge/Parts">PARTS</A></LI><LI class="cam-overview-media"><A href="/Team:Cambridge/Media">MEDIA</A></LI><LI class="cam-overview-blog"><A href="/Team:Cambridge/Blog">BLOG</A></LI></UL></DIV></DIV><DIV id="cam_project" class="cam-menu-item cam-box cam-project"><A href="/Team:Cambridge/Project"><DIV class="cam-menu-top"><DIV class="cam-menu-text">PROJECT</DIV></DIV></A><DIV class="cam-menu-all cam-subitem cam-project"><UL class="cam-menu-drop"><LI class="cam-project-overview"><A href="/Team:Cambridge/Project">OVERVIEW</A></LI><LI class="cam-project-background"><A href="/Team:Cambridge/Project/Background">BACKGROUND</A></LI><LI class="cam-project-microscopy"><A href="/Team:Cambridge/Project/Microscopy">MICROSCOPY</A></LI><LI class="cam-project-invivo"><A href="/Team:Cambridge/Project/In_Vivo">IN VIVO</A></LI><LI class="cam-project-invitro"><A href="/Team:Cambridge/Project/In_Vitro">IN VITRO</A></LI><LI class="cam-project-gibthon"><A href="/Team:Cambridge/Project/Gibthon">GIBTHON</A></LI><LI class="cam-project-future"><A href="/Team:Cambridge/Project/Future">FUTURE</A></LI></UL></DIV></DIV><DIV id="cam_labwork" class="cam-menu-item cam-box cam-labwork"><A href="/Team:Cambridge/Labwork"><DIV class="cam-menu-top"><DIV class="cam-menu-text">LABWORK</DIV></DIV></A><DIV class="cam-menu-all cam-subitem cam-labwork"><UL class="cam-menu-drop"><LI class="cam-labwork-overview"><A href="/Team:Cambridge/Labwork">OVERVIEW</A></LI><LI class="cam-labwork-experiments"><A href="/Team:Cambridge/Experiments">EXPERIMENTS</A></LI><LI class="cam-labwork-protocols"><A href="/Team:Cambridge/Protocols">PROTOCOLS</A></LI><LI class="cam-labwork-constructs"><A href="/Team:Cambridge/Experiments/Plasmid_Constructs">CONSTRUCTS</A></LI><LI class="cam-labwork-safety"><A href="/Team:Cambridge/Safety">SAFETY</A></LI></UL></DIV></DIV><DIV id="cam_society" class="cam-menu-item cam-box cam-society"><A href="/Team:Cambridge/Society"><DIV class="cam-menu-top"><DIV class="cam-menu-text">SOCIETY</DIV></DIV></A><DIV class="cam-menu-all cam-subitem cam-society"><UL class="cam-menu-drop"><LI class="cam-society-overview"><A href="/Team:Cambridge/Society">OVERVIEW</A></LI><LI class="cam-society-interviews"><A href="/Team:Cambridge/Society/interviews">INTERVIEWS</A></LI><LI class="cam-society-report"><A href="/Team:Cambridge/Society/report">IMPACT OF iGEM</A></LI></UL></DIV></DIV><DIV id="cam_team" class="cam-menu-item cam-box cam-team"><A href="/Team:Cambridge/Team"><DIV class="cam-menu-top"><DIV class="cam-menu-text">TEAM</DIV></DIV></A><DIV class="cam-menu-all cam-subitem cam-team"><UL class="cam-menu-drop"><LI class="cam-team-overview"><A href="/Team:Cambridge/Team">OVERVIEW</A></LI><LI class="cam-team-students"><A href="/Team:Cambridge/Team/Students">STUDENTS</A></LI><LI class="cam-team-academics"><A href="/Team:Cambridge/Team/Academics">ACADEMICS</A></LI><LI class="cam-team-sponsors"><A href="/Team:Cambridge/Team/Sponsors">SPONSORS</A></LI></UL></DIV></DIV></DIV><DIV class="cam-box"><DIV id="cam_pagecontent" class="cam-box-content"><DIV id="wiki_content">

Back to <A href="/Team:Cambridge/Experiments">Experiments</A><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Synthetic_Gene_Amplification_.26_Plasmid_Construction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Synthetic Gene Amplification &amp; Plasmid Construction</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Synopsis"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Synopsis</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-3"><A href="#Extraction_.26_Amplification"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Extraction &amp; Amplification</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Construct_Design"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Construct Design</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Assembly_of_GA1.2C_GA2.2C_GA3_and_GA4_-_First_Attempt"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Assembly of GA1, GA2, GA3 and GA4 - First Attempt</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#PCR"><SPAN class="tocnumber">3.1.1</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Gibson_Assembly"><SPAN class="tocnumber">3.1.2</SPAN><SPAN class="toctext">Gibson Assembly</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Transformation"><SPAN class="tocnumber">3.1.3</SPAN><SPAN class="toctext">Transformation</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Colony_PCR"><SPAN class="tocnumber">3.1.4</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#pSB1AK3_or_pSB1A3_backbone.3F"><SPAN class="tocnumber">3.1.5</SPAN><SPAN class="toctext">pSB1AK3 or pSB1A3 backbone?</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Diagnostics"><SPAN class="tocnumber">3.1.6</SPAN><SPAN class="toctext">Diagnostics</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Assembly_of_GA1.2C_GA2.2C_GA3.2C_GA4.2C_GA5_and_GA6_-_Second_Attempt"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Assembly of GA1, GA2, GA3, GA4, GA5 and GA6 - Second Attempt</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#PCR_2"><SPAN class="tocnumber">3.2.1</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Gibson_Assembly_2"><SPAN class="tocnumber">3.2.2</SPAN><SPAN class="toctext">Gibson Assembly</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Transformation_2"><SPAN class="tocnumber">3.2.3</SPAN><SPAN class="toctext">Transformation</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Colony_PCR_2"><SPAN class="tocnumber">3.2.4</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Assembly_of_GA13_and_GA14"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Assembly of GA13 and GA14</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#PCR_3"><SPAN class="tocnumber">3.3.1</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Gibson_Assembly_3"><SPAN class="tocnumber">3.3.2</SPAN><SPAN class="toctext">Gibson Assembly</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#Transformation_3"><SPAN class="tocnumber">3.3.3</SPAN><SPAN class="toctext">Transformation</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#Colony_PCR_3"><SPAN class="tocnumber">3.3.4</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Synthetic_Gene_Amplification_.26_Plasmid_Construction">Synthetic Gene Amplification &amp; Plasmid Construction</SPAN></H2><H3><SPAN class="mw-headline" id="Synopsis">Synopsis</SPAN></H3><P>The synthetic genes were extracted from the filter paper on which they were sent, and amplified by PCR. 
</P><P><A href="/Team:Cambridge/Experiments/Plasmid_Constructs#.2FExperiments.2FPlasmid_Constructs" title="Team:Cambridge/Experiments/Plasmid Constructs"> Eight plasmids</A> were then constructed, named GA1-6, GA13 and GA14 as shown below. Each puts reflectin expression under the control of pBAD promoter, an arabinose inducible promoter. 
HIS tags were included on some of the constructs in order to purify the protein for the in vitro work. 
We also put both reflectin A2 and 1B in a low expression plasmid. 
In  addition, we created Reflectin A1 translational fusions with GFP at the C-terminus, which allowed for tracing the localization of reflectin in a bacterial cell at high and low level of expression.
</P><CENTER><TABLE border="1" style="text-align:center;"><TBODY><TR><TD rowspan="2"><I><B>Name</B></I></TD><TD colspan="3"><I><B>Description</B></I></TD></TR><TR><TD><I>Gene</I></TD><TD><I>Plasmid</I></TD><TD><I>Promoter</I></TD></TR><TR><TD scope="col" width="50"><B>GA1</B></TD><TD scope="col" width="180">Reflectin A1
</TD><TD scope="col" width="200">on PSB1A2 (high copy)
</TD><TD scope="col" width="150">pBAD promoter
</TD></TR><TR><TD><B>GA2</B></TD><TD>Reflectin A1
</TD><TD>on PSB3K3 (low  copy)
</TD><TD>pBAD promoter
</TD></TR><TR><TD><B>GA3</B></TD><TD>Reflectin A1 + HIS tag
</TD><TD>on PSB1A2 (high copy)
</TD><TD>pBAD promoter
</TD></TR><TR><TD><B>GA4</B></TD><TD>Reflectin A1 + HIS tag
</TD><TD>on PSB3K3 (low  copy)
</TD><TD>pBAD promoter
</TD></TR><TR><TD><B>GA5</B></TD><TD>Reflectin A2
</TD><TD>on PSB3K3 (low  copy)
</TD><TD>pBAD promoter
</TD></TR><TR><TD><B>GA6</B></TD><TD>Reflectin 1B
</TD><TD>on PSB3K3 (low  copy)
</TD><TD>pBAD promoter
</TD></TR><TR><TD><B>GA13</B></TD><TD>Reflectin A1 : GFP
</TD><TD>on PSB3K3 (low  copy)
</TD><TD>pBAD promoter
</TD></TR><TR><TD><B>GA14</B></TD><TD>Reflectin A1 : GFP
</TD><TD>on PSB3K3 (low  copy)
</TD><TD>pBAD promoter
</TD></TR></TBODY></TABLE></CENTER><P>These constructs were grown up in E.coli and verified by <A href="/Team:Cambridge/Protocols/Colony_PCR" title="Team:Cambridge/Protocols/Colony PCR"> colony PCR</A>. The plasmids were then <A href="/Team:Cambridge/Protocols/Mini_Prep" title="Team:Cambridge/Protocols/Mini Prep"> miniprepped </A> and stored at -20°C.
</P><H2><SPAN class="mw-headline" id="Extraction_.26_Amplification">Extraction &amp; Amplification</SPAN></H2><P>Wendy Crookes-Goodson kindly supported us with three <A href="http://www.bio-protech.com.tw/databank/DataSheet/CustomSC/pUC57map.pdf" class="external text" rel="nofollow">pUC57</A> plasmids containg reflectin genes: <I>Euprymna scolopes</I><A href="http://www.ncbi.nlm.nih.gov/nuccore/33521697" class="external text" rel="nofollow">Reflectin 1B</A>, and Loligo pealei <A href="http://www.ncbi.nlm.nih.gov/nuccore/269996957" class="external text" rel="nofollow">Reflectin A1</A> and <A href="http://www.ncbi.nlm.nih.gov/nuccore/FJ824805.1" class="external text" rel="nofollow">Reflectin A2</A>. These are synthetic genes, codon optimized for expression in E.coli, and with all restriction sites removed from the coding region.   
</P><P>The plasmids with the reflectin gene were first <A href="/Team:Cambridge/Protocols/Filter_Paper" title="Team:Cambridge/Protocols/Filter Paper"> extracted from filter paper</A>, and then used to transform competent E. coli cells. A standard <A href="/Team:Cambridge/Experiments#.2FProtocols.2FMini_Prep" title="Team:Cambridge/Experiments"> miniprep</A> was used to recover the plasmid DNA from the overnight culture of the transformed bacteria in LB. The purpose of this procedure was to amplify and store the plasmid DNA.
</P><H2><SPAN class="mw-headline" id="Construct_Design">Construct Design</SPAN></H2><P>For the assembly of the required plasmids, we amplified desired components by PCR and then joined them using Gibson Assembly procedure. 
</P><P>We used three different DNA templates:
</P><UL><LI><A href="http://partsregistry.org/Part:BBa_I13520" class="external text" rel="nofollow">I13520</A>:<A href="http://partsregistry.org/Part:pSB1A3" class="external text" rel="nofollow">pSB1A3</A>, which contains mRFP gene under the control of pBAD promoter. It provided high copy number plasmid backbone and elements required for the expression of the gene of interest, such as pBAD/araC promoter <A href="http://partsregistry.org/Part:BBa_I0500" class="external text" rel="nofollow">I0500</A>, ribosome binding site <A href="http://partsregistry.org/Part:BBa_B0034" class="external text" rel="nofollow">B0034</A> and terminator <A href="http://partsregistry.org/Part:BBa_B0015" class="external text" rel="nofollow">B0015</A>.
</LI><LI><A href="http://partsregistry.org/Part:BBa_J69511" class="external text" rel="nofollow">J69511</A>:<A href="http://partsregistry.org/Part:pSB3K3" class="external text" rel="nofollow">pSB3K3</A>, which contains GFP under the control of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_R0011" class="external text" rel="nofollow">R0011</A> promoter that allows for strong constitutive expression unless it is repressed by LacI. It provided low copy number plasmid backbone and terminator <A href="http://partsregistry.org/Part:BBa_B0015" class="external text" rel="nofollow">B0015</A>.
</LI><LI>pUC57 plasmid with reflectin A1/A2/1B coding regions.
</LI></UL><P>The his tag present in GA3 and GA4 constructs was included in primers.
</P><P>We created high copy number plasmids <B>GA1</B> and <B>GA3</B> by replacing mRFP coding region of pSB1A2:I13520 with reflectin A1 coding region. 
</P><P>In the case of low copy number plasmids, <B>GA2</B> and <B>GA4-6</B>, the respective reflectin coding region was joined to a part of pSB1A2 upstream (carrying pBAD/araC and RBS) and was attached to a part of the pSB3K3 downstream (contributing a terminator and a backbone). 
We managed to join pBAD/araC promoter and B0015 ribosome binding site with the low copy number plasmid backbone using a pair of primers complementary to the region around the standard annealing site, included in every standard biobrick backbone. 
</P><P>For <B>GA13</B> and <B>GA14</B> constructs, the template DNA we used were GA1 and GA2 plasmids, respectively. We inserted GFP gene <A href="http://partsregistry.org/Part:BBa_E0040" class="external text" rel="nofollow">E0040</A>, contributed by J69511:pSB3K3 plasmid, between the reflectin coding region and the B0015 terminator. 
</P><P>Sequences of the primers used in the experiment and their predicted melting temperatures are the following:
</P><CENTER><TABLE border="1" style="text-align:center;"><TBODY><TR><TD><I><B>Name</B></I></TD><TD><I><B>Sequence</B></I></TD><TD><I><B>Tm</B></I></TD></TR><TR><TD><B>1-a1</B></TD><TD>AGAAAGAGGAGAAATACTAGATGGGATCCATGAACCGTTACC
</TD><TD>67.9°C
</TD></TR><TR><TD><B>1-his</B></TD><TD>ATACTAGATGCATCATCACCATCACCACATGGGATCCATGAACCGTTAC
</TD><TD>64.9°C
</TD></TR><TR><TD><B>1-a2</B></TD><TD>GAAAGAGGAGAAATACTAGATGAATCGTTATATGATGCGC
</TD><TD>61.7°C
</TD></TR><TR><TD><B>1-1b</B></TD><TD>AGAAAGAGGAGAAATACTAGATGGGATCCATGAATCGC
</TD><TD>63.9°C
</TD></TR><TR><TD><B>2-a1</B></TD><TD>CTGGCTCTAGTATTAGTACATGTGGTAATCGTAATAATTAC
</TD><TD>59.9°C
</TD></TR><TR><TD><B>2-a2</B></TD><TD>CTGGCTCTAGTATTAATACCAACTATTATAGTTATCATACATATTTG
</TD><TD>59.7°C
</TD></TR><TR><TD><B>2-1b</B></TD><TD>ATGCCTGGCTCTAGTATTAATAACAATAATCGTTCATGCC
</TD><TD>60.9°C
</TD></TR><TR><TD><B>3-a1</B></TD><TD>ATTACGATTACCACATGTACTAATACTAGAGCCAGGCATCAAATAAAAC
</TD><TD>64.4°C
</TD></TR><TR><TD><B>3-a2</B></TD><TD>ATAACTATAATAGTTGGTATTAATACTAGAGCCAGGCATCAAATAAAAC
</TD><TD>64.4°C
</TD></TR><TR><TD><B>3-1b</B></TD><TD>GCATGAACGATTATTGTTATTAATACTAGAGCCAGGCATCAAATAAAAC
</TD><TD>64.4°C
</TD></TR><TR><TD><B>4-a1</B></TD><TD>GGTAACGGTTCATGGATCCCATCTAGTATTTCTCCTCTTTCTCTAGTAGCTAG
</TD><TD>65.0°C
</TD></TR><TR><TD><B>4-his</B></TD><TD>CATGGATCCCATGTGGTGATGGTGATGATGCATCTAGTATTTCTCCTCTTTCTCTAGTAG
</TD><TD>60.0°C
</TD></TR><TR><TD><B>4-a2</B></TD><TD>GCGCATCATATAACGATTCATCTAGTATTTCTCCTCTTTCTCTAGTAGCTAG
</TD><TD>60.5°C
</TD></TR><TR><TD><B>4-1b</B></TD><TD>GCATGAACGATTATTGTTATTAATACTAGAGCCAGGCATCAAATAAAAC
</TD><TD>64.4°C
</TD></TR><TR><TD><B>5</B></TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_G00100" class="external text" rel="nofollow">VF2</A></TD><TD>55°C
</TD></TR><TR><TD><B>6</B></TD><TD>primer donated by our supervisor James
</TD><TD>–</TD></TR><TR><TD><B>7</B></TD><TD>TATTACGATTACCACATGTACATGCGTAAAGGCGAAGAGC
</TD><TD>64.66°</TD></TR><TR><TD><B>8</B></TD><TD>GTTTTATTTGATGCCTGGCTCTAGTATCATTTGTACAGTTCATCCATACC
</TD><TD>61.84°</TD></TR><TR><TD><B>9</B></TD><TD>GGTATGGATGAACTGTACAAATGATACTAGAGCCAGGCATCAAATAAAAC
</TD><TD>64.41°</TD></TR><TR><TD><B>10</B></TD><TD>AGCTCTTCGCCTTTACGCATGTACATGTGGTAATCGTAATAATTACG
</TD><TD>60.62°</TD></TR></TBODY></TABLE></CENTER><P>The pictures below give an overview of the assembly of the plasmids:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">GA1 construct</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">GA2 construct</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">GA3 construct</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">GA4 construct</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">GA5 construct</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">GA6 construct</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">GA13 construct</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">GA14 construct</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Assembly_of_GA1.2C_GA2.2C_GA3_and_GA4_-_First_Attempt">Assembly of GA1, GA2, GA3 and GA4 - First Attempt</SPAN></H3><H4><SPAN class="mw-headline" id="PCR">PCR</SPAN></H4><P>In the first round of PCR, we amplified fragments required for the assembly of GA1, GA2, GA3 and GA4 constructs.
</P><UL><LI>We performed PCR using <A href="/Team:Cambridge/Protocols/PCR#Reagents_used_in_PCR_reaction" title="Team:Cambridge/Protocols/PCR"> Phusion Hot Start DNA Polymerase</A> in 20 μl reaction volume.
</LI><LI>Time profile used in the PCR machine was the following:
</LI></UL><CENTER><TABLE border="1" align="center" style="text-align:center;"><TBODY><TR><TD scope="col" width="50"><B>Hold</B></TD><TD>95°C
</TD><TD>2 min
</TD></TR><TR><TD scope="col" width="120" rowspan="3"><B>Cycling ×30</B></TD><TD scope="col" width="120"><I>Denaturing</I></TD><TD scope="col" width="80"> 95°C
</TD><TD scope="col" width="80"> 10 s
</TD></TR><TR><TD scope="col" width="80"><I>Annealing</I></TD><TD scope="col" width="50"> 55°C
</TD><TD scope="col" width="50"> 20 s
</TD></TR><TR><TD scope="col" width="80"><I>Elongation</I></TD><TD scope="col" width="50"> 72°C
</TD><TD scope="col" width="50"> 150 s
</TD></TR></TBODY></TABLE></CENTER><P>We decided to use the 55°C annealing temperature, although the predicted temperature for most primers is 5-10°C higher, because of a low annealing temperature of the VF2 primer.
</P><UL><LI>Primers and template DNA provided by our supervisor Paul served as a positive control, but eventually we did not detect any products on the gels.
</LI></UL><P>
The pictures below present results of the gel electrophoresis of PCR products. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The order of samples: HyperLadder I, GA1-1, GA1-2, positive control, HyperLadder I, GA2-1, GA2-2 and GA2-3.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The order of samples: HyperLadder I, GA3-1 and GA3-2.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The order of samples: HyperLadder I, GA4-1, GA4-2 and GA4-3.</DIV></DIV></DIV></DIV><UL><LI>In most cases position of a band matches the expected length of DNA fragment. The only exception are GA1-2 and GA3-2 products. According to the position on the gel the length of these DNA fragments is 4-5kb, whereas the predicted length is 3.5kb. Our hypothesis is that we were provided <A href="http://partsregistry.org/Part:pSB1AK3" class="external text" rel="nofollow">pSB1AK3</A> backbone instead of <A href="http://partsregistry.org/Part:pSB1A3" class="external text" rel="nofollow">pSB1A3</A> backbone. 
</LI><LI>For GA1-1, GA2-1, GA3-1 and GA4-1 we obtained two bands: 1000bp and 400bp, with the latter resulting from non-specific priming most probably. We extracted the two bands for GA1-1, GA2-1 and GA4-1 products, labelling the 1000kb and 400bp fragments GAX-1a and GAX1-b respectively. 
</LI><LI>The molecular weight marker that we used in all gels is HyperLadder I, which produces regularly spaced bands ranging from 200 to 10,000bp.
</LI></UL><P>For the gel extraction of DNA we followed the <A href="/Team:Cambridge/Protocols/Gel_Extraction_of_DNA" title="Team:Cambridge/Protocols/Gel Extraction of DNA"> protocol</A>, assuming that one slice of gel is 100μl.
</P><H4><SPAN class="mw-headline" id="Gibson_Assembly">Gibson Assembly</SPAN></H4><P>We conducted Gibson Assembly of GA1, GA2, GA3 and GA4 constructs according to <A href="/Team:Cambridge/Protocols/Gibson_Assembly#Practice" title="Team:Cambridge/Protocols/Gibson Assembly"> the protocol</A>. The volumes of Master Mix and solutions of amplified DNA were the following:
</P><CENTER><TABLE border="1" align="center" style="text-align:center;"><TBODY><TR><TH scope="col" width="200"> GA1
</TH><TH scope="col" width="200"> GA2
</TH><TH scope="col" width="200"> GA3
</TH><TH scope="col" width="200"> GA4
</TH></TR><TR><TD>15µl Master Mix
</TD><TD>15µl Master Mix
</TD><TD>15µl Master Mix
</TD><TD>15µl Master Mix
</TD></TR><TR><TD>2.5µl GA1-1a
</TD><TD>1.67µl GA2-1a
</TD><TD>2.5µl GA3-1
</TD><TD>1.67µl GA4-1a
</TD></TR><TR><TD>2.5µl GA1-2
</TD><TD>1.67µl GA2-2
</TD><TD>2.5µl GA3-2
</TD><TD>1.67µl GA4-2
</TD></TR><TR><TD>1.67µl GA2-3
</TD><TD>1.67µl GA4-3
</TD></TR></TBODY></TABLE></CENTER><H4><SPAN class="mw-headline" id="Transformation">Transformation</SPAN></H4><P>We transformed competent E.coli cells according to the following <A href="/Team:Cambridge/Protocols/Transformation_of_E.Coli" title="Team:Cambridge/Protocols/Transformation of E.Coli">  protocol</A>. 
We cultured each class of transformants on four different plates.
</P><CENTER><TABLE><TBODY><TR><TD scope="col" width="180"><B>GA1 and GA3</B></TD><TD scope="col" width="180"><B>GA2 and GA4</B></TD></TR><TR><TD><I><B>Plate 1 non-induced</B></I></TD><TD>(10μl of cell suspension)
</TD><TD>LB + ampicillin
</TD><TD>LB + kanamycin
</TD></TR><TR><TD><I><B>Plate 2 non-induced</B></I></TD><TD>(100μl of cell suspension)
</TD><TD>LB + ampicillin
</TD><TD>LB + kanamycin
</TD></TR><TR><TD><I><B>Plate 3 induced</B></I></TD><TD>(10μl of cell suspension)
</TD><TD>LB + ampicillin + arabinose
</TD><TD>LB + kanamycin + arabinose
</TD></TR><TR><TD><I><B>Plate 4 induced</B></I></TD><TD>(100μl of cell suspension)
</TD><TD>LB + ampicillin + arabinose
</TD><TD>LB + kanamycin + arabinose
</TD></TR></TBODY></TABLE></CENTER><P>After overnight incubation at 37°C of transformed E.coli cells, we could see colonies on only some of our plates.
We examined successful plates under a fluorescence microscope to check if the cells had been transformed with a carry-through of the template DNA used in the initial PCR reactions. The risk of contaminating Gibson Assembly reactions with template DNA was fairly high because circular DNA of around 4kb was likely to co-localize with the linear PCR products. 
</P><TABLE border="1" align="center" style="text-align:center;"><TBODY><TR><TD rowspan="2" scope="col" width="250"><B>Type of plasmid</B></TD><TD colspan="2"><B>Expected phenotype of transformed colonies</B></TD></TR><TR><TD scope="col" width="250"><I><B>LB + Antibiotic</B></I></TD><TD scope="col" width="250"><I><B>LB + Antibiotic + Arabinose</B></I></TD></TR><TR><TD><I><B>Reflectin A1 expression plasmid</B></I><I>desired product</I></TD><TD>No fluorescence
</TD><TD>No fluorescence
</TD></TR><TR><TD><I><B>I13520:pSB1A3 plasmid</B></I><I>template DNA</I></TD><TD>No fluorescence
</TD><TD>RFP red fluorescence
</TD></TR><TR><TD><I><B>J69511:pSB3K3 plasmid</B></I><I>template DNA</I></TD><TD>GFP green fluorescence
</TD><TD>GFP green fluorescence
</TD></TR></TBODY></TABLE><P>
These are the results of the examination of transformants:
</P><TABLE><TBODY><TR><TD scope="col" width="60"><B>GA1</B></TD><TD scope="col" width="350"> one culture on LB + ampicillin 10μl plate
</TD><TD scope="col" width="350"> no fluorescence detected
</TD></TR><TR><TD><B>GA2</B></TD><TD>no colonies observable
</TD><TD>-</TD></TR><TR><TD><B>GA3</B></TD><TD>colonies on all plates
</TD><TD>no fluorescence detected
</TD></TR><TR><TD><B>GA4</B></TD><TD>colonies on all plates
</TD><TD>green fluorescence on induced and non-induced plates
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Colony_PCR">Colony PCR</SPAN></H4><P>We decided to conduct colony PCR of E.coli transformed with GA1 and GA3 constructs in order to confirm successful assembly of the plasmids.
</P><UL><LI>We prepared overnight liquid cultures of GA3 and GA1 transformants, both in LB + amplicillin and LB + amplicillin + arabinose medium. We picked 6 colonies, labelled a-f, from GA3 LB + ampicillin 100μl plate, and one colony, labelled x, from  GA1 LB + ampicillin 10μl plate. 
</LI><LI>We performed colony PCR using Taq polymerase according to the following <A href="/Team:Cambridge/Protocols/Colony_PCR" title="Team:Cambridge/Protocols/Colony PCR"> protocol</A>, using I13520:pSB1A3 plasmid as a positive control. We used standard <A href="http://partsregistry.org/Part:BBa_G00101" class="external text" rel="nofollow">VR</A> and <A href="http://partsregistry.org/Part:BBa_G00100" class="external text" rel="nofollow">VF2</A> primers, as indicated on the diagrams below: 
</LI></UL><TABLE align="center"><TBODY><TR><TD colspan="2"><I>Expected lengths of products:</I></TD></TR><TR><TD scope="col" width="120"> Positive control
</TD><TD scope="col" width="70"> ~2400 bp
</TD></TR><TR><TD>GA1 insert
</TD><TD>~2800 bp
</TD></TR><TR><TD>GA3 insert
</TD><TD>~2800 bp
</TD></TR></TBODY></TABLE><UL><LI>Time profile used in the PCR machine:
</LI></UL><CENTER><TABLE border="1" align="center" style="text-align:center;"><TBODY><TR><TD scope="col" width="50"><B>Hold 1</B></TD><TD>95°C
</TD><TD>6 min
</TD></TR><TR><TD scope="col" width="100" rowspan="3"><B>Cycling ×30</B></TD><TD scope="col" width="100"><I>Denaturing</I></TD><TD scope="col" width="80"> 98°C
</TD><TD scope="col" width="80"> 10 s
</TD></TR><TR><TD scope="col" width="80"><I>Annealing</I></TD><TD scope="col" width="50"> 55°C
</TD><TD scope="col" width="50"> 30 s
</TD></TR><TR><TD scope="col" width="80"><I>Elongation</I></TD><TD scope="col" width="50"> 72°C
</TD><TD scope="col" width="50"> 180 s
</TD></TR><TR><TD><B>Hold 2</B></TD><TD>72°C
</TD><TD>5 min
</TD></TR></TBODY></TABLE></CENTER><UL><LI>As we can see from the picture of the agarose gel, c-f and x colonies carried correctly assembled plasmids. We decided to <A href="/Team:Cambridge/Protocols/Mini_Prep" title="Team:Cambridge/Protocols/Mini Prep"> extract plasmids</A> from and prepare <A href="/Team:Cambridge/Protocols/Glycerol_Stocks" title="Team:Cambridge/Protocols/Glycerol Stocks"> glycerol stocks</A> of these successful transformants.
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The order of samples: HyperLadder I, GA3a, GA3b, GA3c, GA3d, HyperLadder I, GA3e, GA3f, positive control, GA1x.</DIV></DIV></DIV></DIV><UL><LI>We also ran a gel with products of <A href="/Team:Cambridge/Protocols/Mini_Prep" title="Team:Cambridge/Protocols/Mini Prep"> MiniPrep procedure</A> to measure its effectiveness and also check if there were any differences in yield between induced and non-induced liquid cultures. The order of samples is the following and we can clearly see that induced cultures gave smaller yield than those non-induced. This proves that replication of multi-copy plasmids acts as a burden for proliferating cells, giving lower densities of liquid cultures.
</LI></UL><CENTER><TABLE border="1" align="center" style="text-align:center;"><TBODY><TR><TD scope="col" width="70"><B>Well</B></TD><TD scope="col" width="100"><B>1</B></TD><TD scope="col" width="100"><B>2</B></TD><TD scope="col" width="100"><B>3</B></TD><TD scope="col" width="100"><B>4</B></TD><TD scope="col" width="100"><B>5</B></TD><TD scope="col" width="100"><B>6</B></TD><TD scope="col" width="100"><B>7</B></TD></TR><TR><TD><B>Sample</B></TD><TD>Hyperladder I
</TD><TD>GA3-c
</TD><TD>GA3-d
</TD><TD>GA3-e
</TD><TD>GA3-f
</TD><TD>GA1-x
</TD><TD>Hyperladder I
</TD></TR><TR><TD><B>Culture of origin</B></TD><TD>induced
</TD><TD>induced mixed with non-induced
</TD><TD>induced
</TD><TD>induced
</TD><TD>non-induced
</TD></TR></TBODY></TABLE></CENTER><H4><SPAN class="mw-headline" id="pSB1AK3_or_pSB1A3_backbone.3F">pSB1AK3 or pSB1A3 backbone?</SPAN></H4><P>As a side experiment, we decided to determine whether constructs with pSB1A3 backbone also carry a gene for kanamycin resistance. When preparing glycerol stocks of successful transformants, we cultured GA1x and GA3c bacteria in LB + kanamycin liquid medium in addition to LB + ampicillin medium. After 4 hour incubation at 37° the turbidity of both cultures was the same. This suggests that GA1 and GA3 contructs carry genes for both ampicillin and kanamycin resistance, which means that we were working with <A href="http://partsregistry.org/Part:pSB1AK3" class="external text" rel="nofollow">pSB1AK3</A> instead of <A href="http://partsregistry.org/Part:pSB1A3" class="external text" rel="nofollow">pSB1A3</A> backbone.
</P><H4><SPAN class="mw-headline" id="Diagnostics">Diagnostics</SPAN></H4><P>In addition, we performed a series of tests to identify the cause of the low efficiency of transformation. The proposed sources of error included:
</P><UL><LI>Low efficiency of DNA gel extraction
</LI><LI>Unsuccessful Gibson Assembly
</LI><LI>Low viability of competent E.coli cells
</LI></UL><P>In order to check if gel extraction of DNA was successful, we ran a 1% agarose gel with samples of purified products of the initial PCR reactions. 
Although we could see distinct, fairly thick bands on the first gels, from which PCR products were purified, bands on the diagnostic gel were very faint, even missing in some lanes.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The order of samples: HyperLadder I, GA1-1a, GA1-2, GA2-1a, GA2-2, GA2-3, HyperLadder I, GA3-1, GA3-2, GA4-1a, GA4-2 and GA4-3.</DIV></DIV></DIV></DIV><P>This indicates that the yield of our extraction procedure was very low and probably this was the main reason why the performed transformation was fairly unsuccessful. Therefore, we decided to repeat PCR reactions, this time at higher 50μl volume, and try different conditions of the gel extraction procedure.
</P><H3><SPAN class="mw-headline" id="Assembly_of_GA1.2C_GA2.2C_GA3.2C_GA4.2C_GA5_and_GA6_-_Second_Attempt">Assembly of GA1, GA2, GA3, GA4, GA5 and GA6 - Second Attempt</SPAN></H3><H4><SPAN class="mw-headline" id="PCR_2">PCR</SPAN></H4><P>In the second round of PCR, we amplified fragments required for the assembly of GA1, GA2, GA3, GA4, as well as GA5 and GA6 constructs.
</P><UL><LI>We performed PCR using <A href="/Team:Cambridge/Protocols/PCR#Reagents_used_in_PCR_reaction" title="Team:Cambridge/Protocols/PCR"> Phusion Hot Start DNA Polymerase</A> in 50 μl reaction volume.
</LI><LI>The time profile used in the PCR machine was the following, the same as the one used before:
</LI></UL><CENTER><TABLE border="1" align="center" style="text-align:center;"><TBODY><TR><TD scope="col" width="50"><B>Hold</B></TD><TD>95°C
</TD><TD>2 min
</TD></TR><TR><TD scope="col" width="120" rowspan="3"><B>Cycling ×30</B></TD><TD scope="col" width="120"><I>Denaturing</I></TD><TD scope="col" width="80"> 95°C
</TD><TD scope="col" width="80"> 10 s
</TD></TR><TR><TD scope="col" width="80"><I>Annealing</I></TD><TD scope="col" width="50"> 55°C
</TD><TD scope="col" width="50"> 20 s
</TD></TR><TR><TD scope="col" width="80"><I>Elongation</I></TD><TD scope="col" width="50"> 72°C
</TD><TD scope="col" width="50"> 150 s
</TD></TR></TBODY></TABLE></CENTER><P>
We ran PCR products on a 1% agarose gel to separate amplified products from template DNA and primers, as well as to check how efficient and specific the amplification process was. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The order of samples: HyperLadder I, GA1-1, GA1-2, HyperLadder I, GA2-1, GA2-2, GA2-3 and GA2-2.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The order of samples: HyperLadder I, GA3-1, GA3-2, HyperLadder I, GA4-1, GA4-2, GA4-3 and GA4-3.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The order of samples: HyperLadder I, GA5-1, GA5-2, GA5-3, HyperLadder I, GA6-1, GA6-2 and GA6-3.</DIV></DIV></DIV></DIV><UL><LI>In most cases position of a band matches the expected length of DNA fragment. The only exception are again 4-5kb GA1-2 and GA3-2 products, although the predicted length is 3.5kb. 
</LI><LI>This time we obtained single bands on lanes with amplified reflectin, whereas we observed some additional bands on almost all lanes with amplified backbones of plasmids to be constructed. Most probably these were products of non-specific annealing, which we did not detect on the first gel due to low concentration of these additional products in 25μl reaction. Moreover, these differences emphasize the fact that the amount of non-specific products of DNA amplification can vary greatly between reactions depending on the timing of random mis-annealing events. 
</LI><LI>All components of GA5 and GA6 constructs produced clear fairly thick bands with positions matching expected lengths. No non-specific bands on GA5-1 (Reflectin A2) and GA6-1 (Reflectin 1B) lanes were observed. 
</LI><LI>The molecular weight marker that we used in all gels is HyperLadder I, which produces regularly spaced bands ranging from 200 to 10,000bp.
</LI></UL><P>We also ran a gel with products of extraction to check how efficient the procedure was. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The order of samples: HyperLadder I, GA5-1, GA5-2, GA5-3, HyperLadder I, GA6-1, GA6-2 and GA6-3.</DIV></DIV></DIV></DIV><UL><LI>We obtained clear distinct bands on each lane loaded with components of GA5 and GA6 Gibson Assembly constructs.
</LI></UL><H4><SPAN class="mw-headline" id="Gibson_Assembly_2">Gibson Assembly</SPAN></H4><P>We conducted Gibson Assembly of GA1, GA2, GA3, GA4, GA5 and GA6 constructs according to <A href="/Team:Cambridge/Protocols/Gibson_Assembly" title="Team:Cambridge/Protocols/Gibson Assembly"> the protocol</A>. The total reaction volume was 20μl.
</P><H4><SPAN class="mw-headline" id="Transformation_2">Transformation</SPAN></H4><P>We transformed competent E.coli cells according to the following <A href="/Team:Cambridge/Protocols/Transformation_of_E.Coli" title="Team:Cambridge/Protocols/Transformation of E.Coli">  protocol</A>. 
We cultured each class of transformants on four different plates.
</P><CENTER><TABLE><TBODY><TR><TD scope="col" width="180"><B>GA1, GA3 and positive control</B></TD><TD scope="col" width="180"><B>GA2, GA4, GA5 and GA6</B></TD></TR><TR><TD><I><B>Plate 1 non-induced</B></I></TD><TD>(10μl of cell suspension)
</TD><TD>LB + ampicillin
</TD><TD>LB + kanamycin
</TD></TR><TR><TD><I><B>Plate 2 non-induced</B></I></TD><TD>(100μl of cell suspension)
</TD><TD>LB + ampicillin
</TD><TD>LB + kanamycin
</TD></TR><TR><TD><I><B>Plate 3 induced</B></I></TD><TD>(10μl of cell suspension)
</TD><TD>LB + ampicillin + arabinose
</TD><TD>LB + kanamycin + arabinose
</TD></TR><TR><TD><I><B>Plate 4 induced</B></I></TD><TD>(100μl of cell suspension)
</TD><TD>LB + ampicillin + arabinose
</TD><TD>LB + kanamycin + arabinose
</TD></TR></TBODY></TABLE></CENTER><P>As a positive control, we transformed cells with I13520:pSB1A3 plasmid, expecting to detect RFP fluorescence on induced plates. As a negative control, we plated non-transformed heat-shocked competent cells on LB + kanamycin and LB + ampicillin plates.
</P><P>After overnight incubation at 37°C we examined plates under a fluorescence microscope to check if the cells had been also transformed with the template DNA used in the initial PCR reactions.  
The table presents expected phenotype of cells transformed with the correct Gibson Assembly construct or two DNA templates.
</P><TABLE border="1" align="center" style="text-align:center;"><TBODY><TR><TD rowspan="2" scope="col" width="250"><B>Type of plasmid</B></TD><TD colspan="2"><B>Expected phenotype of transformed colonies</B></TD></TR><TR><TD scope="col" width="250"><I><B>LB + Antibiotic</B></I></TD><TD scope="col" width="250"><I><B>LB + Antibiotic + Arabinose</B></I></TD></TR><TR><TD><I><B>Reflectin A1 expression plasmid</B></I><I>desired product</I></TD><TD>No fluorescence
</TD><TD>No fluorescence
</TD></TR><TR><TD><I><B>I13520:pSB1A3 plasmid</B></I><I>template DNA</I></TD><TD>No fluorescence
</TD><TD>RFP red fluorescence
</TD></TR><TR><TD><I><B>J69511:pSB3K3 plasmid</B></I><I>template DNA</I></TD><TD>GFP green fluorescence
</TD><TD>GFP green fluorescence
</TD></TR></TBODY></TABLE><P>
We observed colonies on all plates except for the negative control:
</P><TABLE><TBODY><TR><TD scope="col" width="150"><B>GA1</B></TD><TD scope="col" width="800">no fluorescence detected
</TD></TR><TR><TD><B>GA2</B></TD><TD>green fluorescence of some colonies on LB + kanamycin + arabinose plates (both 100μl and 10μl)
</TD></TR><TR><TD><B>GA3</B></TD><TD>no fluorescence detected
</TD></TR><TR><TD><B>GA4</B></TD><TD>green fluorescence of some colonies on both induced and non-induced plates
</TD></TR><TR><TD><B>GA5</B></TD><TD>green fluorescence of some colonies on both induced and non-induced plates
</TD></TR><TR><TD><B>GA6</B></TD><TD>green fluorescence of some colonies on both induced and non-induced plates
</TD></TR><TR><TD><B>+ve control</B></TD><TD>red fluorescence on LB + ampicillin + arabinose plate (as expected)
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Colony_PCR_2">Colony PCR</SPAN></H4><P>We conducted colony PCR of E.coli transformed with GA1 - GA6 constructs in order to confirm successful assembly of these plasmids.
</P><UL><LI>We prepared overnight liquid cultures of transformants - GA1 and GA3 in LB + ampicillin medium, and GA2, GA4, GA5 and GA6 in LB + kanamycin medium. For each class of E.coli we picked four colonies that did not show fluorescence on induced LB + antibiotic + arabinose plates.
</LI><LI>We performed colony PCR using Taq polymerase according to the following <A href="/Team:Cambridge/Protocols/Colony_PCR" title="Team:Cambridge/Protocols/Colony PCR"> protocol</A>, but we did not include any positive control. We used standard <A href="http://partsregistry.org/Part:BBa_G00101" class="external text" rel="nofollow">VR</A> and <A href="http://partsregistry.org/Part:BBa_G00100" class="external text" rel="nofollow">VF2</A> primers, as indicated on the diagrams below: 
</LI></UL><TABLE align="center"><TBODY><TR><TD colspan="2"><I>Expected lengths of products:</I></TD></TR><TR><TD scope="col" width="120"> GA1 insert
</TD><TD scope="col" width="70"> ~2800 bp
</TD></TR><TR><TD>GA2 insert
</TD><TD>~2800 bp
</TD></TR><TR><TD>GA3 insert
</TD><TD>~2800 bp
</TD></TR><TR><TD>GA4 insert
</TD><TD>~2800 bp
</TD></TR><TR><TD>GA5 insert
</TD><TD>~2450 bp
</TD></TR><TR><TD>GA6 insert
</TD><TD>~2600 bp
</TD></TR></TBODY></TABLE><UL><LI>Time profile used in the PCR machine was the following:
</LI></UL><CENTER><TABLE border="1" align="center" style="text-align:center;"><TBODY><TR><TD scope="col" width="50"><B>Hold 1</B></TD><TD>95°C
</TD><TD>6 min
</TD></TR><TR><TD scope="col" width="100" rowspan="3"><B>Cycling ×30</B></TD><TD scope="col" width="100"><I>Denaturing</I></TD><TD scope="col" width="80"> 98°C
</TD><TD scope="col" width="80"> 10 s
</TD></TR><TR><TD scope="col" width="80"><I>Annealing</I></TD><TD scope="col" width="50"> 55°C
</TD><TD scope="col" width="50"> 30 s
</TD></TR><TR><TD scope="col" width="80"><I>Elongation</I></TD><TD scope="col" width="50"> 72°C
</TD><TD scope="col" width="50"> 180 s
</TD></TR><TR><TD><B>Hold 2</B></TD><TD>72°C
</TD><TD>5 min
</TD></TR></TBODY></TABLE></CENTER><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The order of samples: HyperLadder I, GA1-A1, GA1-A2, GA1-A3, GA1-A4, GA2-K1, GA2-K2, GA2-K3 GA2-K4, GA3-A1, GA3-A2, GA3-A3 and GA3-A4, and HyperLadder I.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The order of samples: HyperLadder I, GA4-K1, GA4-K2, GA4-K3, GA4-K4, GA5-K1, GA5-K2, GA5-K3, GA5-K4, GA6-K1, GA6-K2, GA6-K3 and GA6-K4.</DIV></DIV></DIV></DIV><UL><LI>We obtained distinct bright bands on a majority of lanes loaded with products of colony PCR, and their position on the gel corresponds to the predicted sizes of these DNA fragments. This suggests that most colonies, except for GA3-A3, GA4-K3, GA6-K1 and GA6-K4 carried correctly assembled plasmids. Therefore, we decided to <A href="/Team:Cambridge/Protocols/Mini_Prep" title="Team:Cambridge/Protocols/Mini Prep"> extract plasmids</A> from and prepare <A href="/Team:Cambridge/Protocols/Glycerol_Stocks" title="Team:Cambridge/Protocols/Glycerol Stocks"> glycerol stocks</A> of successful transformants.
</LI></UL><H3><SPAN class="mw-headline" id="Assembly_of_GA13_and_GA14">Assembly of GA13 and GA14</SPAN></H3><H4><SPAN class="mw-headline" id="PCR_3">PCR</SPAN></H4><P>In the third round of PCR, we amplified fragments required for the assembly of GA13 and GA14 constructs.
</P><UL><LI>We performed PCR using <A href="/Team:Cambridge/Protocols/PCR#Reagents_used_in_PCR_reaction" title="Team:Cambridge/Protocols/PCR"> Phusion Hot Start DNA Polymerase</A> in 50 μl reaction volume.
</LI><LI>The time profile used in the PCR machine was the following:
</LI></UL><CENTER><TABLE border="1" align="center" style="text-align:center;"><TBODY><TR><TD scope="col" width="50"><B>Hold</B></TD><TD>95°C
</TD><TD>2 min
</TD></TR><TR><TD scope="col" width="100" rowspan="3"><B>Cycling ×30</B></TD><TD scope="col" width="100"><I>Denaturing</I></TD><TD scope="col" width="80"> 95°C
</TD><TD scope="col" width="80"> 10 s
</TD></TR><TR><TD scope="col" width="80"><I>Annealing</I></TD><TD scope="col" width="50"> 55°C
</TD><TD scope="col" width="50"> 20 s
</TD></TR><TR><TD scope="col" width="80"><I>Elongation</I></TD><TD scope="col" width="50"> 72°C
</TD><TD scope="col" width="50"> 190 s
</TD></TR></TBODY></TABLE></CENTER><UL><LI>As we can see from photographs of agarose gels, we obtained bands of expected lengths on each lane. However, we also observed a second, about 1000bp, band on lanes with amplified DNA coding for sfGFP. Probably, this was an effect of non-specific priming at 55° which is a fairly low annealing temperature. We extracted DNA from two bands, but DNA from the lower band (GA13/14-1b) was used in the Gibson Assembly reactions.
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The order of samples: HyperLadder I, GA13-1, GA13-2, GA14-1 and GA14-2.</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Gibson_Assembly_3">Gibson Assembly</SPAN></H4><P>We performed <A href="/Team:Cambridge/Protocols/Gibson_Assembly" title="Team:Cambridge/Protocols/Gibson Assembly"> Gibson Assembly</A> of GA13 and GA14 construct in 20μ reactions.
</P><H4><SPAN class="mw-headline" id="Transformation_3">Transformation</SPAN></H4><P>We transformed competent E.coli cells according to the following <A href="/Team:Cambridge/Protocols/Transformation_of_E.Coli" title="Team:Cambridge/Protocols/Transformation of E.Coli">  protocol</A>. 
We cultured each class of transformants on four different plates, and we transformed cells with I13520:pSB1A3 as a positive control. Non-transformed heat-shocked competent E.coli cells plated on LB + ampicillin and LB + kanamycin served as a negative control. 
</P><TABLE><TBODY><TR><TD scope="col" width="180"><B>GA13 and positive control</B></TD><TD scope="col" width="180"><B>GA14</B></TD></TR><TR><TD><I><B>Plate 1 non-induced</B></I></TD><TD>(10μl of cell suspension)
</TD><TD>LB + ampicillin
</TD><TD>LB + kanamycin
</TD></TR><TR><TD><I><B>Plate 2 non-induced</B></I></TD><TD>(100μl of cell suspension)
</TD><TD>LB + ampicillin
</TD><TD>LB + kanamycin
</TD></TR><TR><TD><I><B>Plate 3 induced</B></I></TD><TD>(10μl of cell suspension)
</TD><TD>LB + ampicillin + arabinose
</TD><TD>LB + kanamycin + arabinose
</TD></TR><TR><TD><I><B>Plate 4 induced</B></I></TD><TD>(100μl of cell suspension)
</TD><TD>LB + ampicillin + arabinose
</TD><TD>LB + kanamycin + arabinose
</TD></TR></TBODY></TABLE><P>After overnight incubation at 37° we screened colonies under a fluorescence microscope in order to check if they had been transformed with template DNA. 
</P><CENTER><TABLE border="1" align="center" style="text-align:center;"><TBODY><TR><TD rowspan="2" scope="col" width="250"><B>Type of plasmid</B></TD><TD colspan="2"><B>Expected phenotype of transformed colonies</B></TD></TR><TR><TD scope="col" width="250"><I><B>LB + Antibiotic</B></I></TD><TD scope="col" width="250"><I><B>LB + Antibiotic + Arabinose</B></I></TD></TR><TR><TD><I><B>Reflectin A1:sfGFP expression plasmid</B></I><I>desired product</I></TD><TD>No fluorescence
</TD><TD>Green fluorescence
</TD></TR><TR><TD><I><B>Reflectin expression plasmid GA1/GA2</B></I><I>template DNA</I></TD><TD>No fluorescence
</TD><TD>No fluorescence
</TD></TR><TR><TD><I><B>J69511:pSB3K3 plasmid</B></I> template DNA
</TD><TD>GFP green fluorescence
</TD><TD>GFP green fluorescence
</TD></TR></TBODY></TABLE></CENTER><P>For both classes of transformants, colonies on non-induced plates showed no fluorescence, whereas they showed green fluorescence on induced plates, which served as a confirmation of correct expression of reflectin:GFP fusion. 
</P><P>In order to prepare liquid cultures for colony PCR, we picked 4 colonies for each construct - two colonies from LB + antibiotic 100μ plate, one colony from LB + antibiotic 10μ plate and one colony from LB + antibiotic + arabinose 100μ plate.
</P><H4><SPAN class="mw-headline" id="Colony_PCR_3">Colony PCR</SPAN></H4><P>We conducted colony PCR of E.coli transformed with GA13 and GA14 constructs in order to confirm successful assembly of these plasmids.
</P><UL><LI>We prepared overnight liquid cultures of transformants - GA13 in LB + ampicillin medium, and GA14 in LB + kanamycin medium. 
</LI><LI>We performed colony PCR using Taq polymerase according to the following <A href="/Team:Cambridge/Protocols/Colony_PCR" title="Team:Cambridge/Protocols/Colony PCR"> protocol</A>, but we did not include any positive control. We used standard <A href="http://partsregistry.org/Part:BBa_G00101" class="external text" rel="nofollow">VR</A> and <A href="http://partsregistry.org/Part:BBa_G00100" class="external text" rel="nofollow">VF2</A> primers, as indicated on the diagrams below: 
</LI></UL><TABLE align="center"><TBODY><TR><TD colspan="2"><I>Expected lengths of products:</I></TD></TR><TR><TD scope="col" width="120"> GA13 insert
</TD><TD scope="col" width="70"> ~3520 bp
</TD></TR><TR><TD>GA14 insert
</TD><TD>~3520 bp
</TD></TR></TBODY></TABLE><UL><LI>Time profile used in the PCR machine was the following:
</LI></UL><CENTER><TABLE border="1" align="center" style="text-align:center;"><TBODY><TR><TD scope="col" width="50"><B>Hold 1</B></TD><TD>95°C
</TD><TD>6 min
</TD></TR><TR><TD scope="col" width="100" rowspan="3"><B>Cycling ×30</B></TD><TD scope="col" width="100"><I>Denaturing</I></TD><TD scope="col" width="80"> 98°C
</TD><TD scope="col" width="80"> 10 s
</TD></TR><TR><TD scope="col" width="80"><I>Annealing</I></TD><TD scope="col" width="50"> 55°C
</TD><TD scope="col" width="50"> 30 s
</TD></TR><TR><TD scope="col" width="80"><I>Elongation</I></TD><TD scope="col" width="50"> 72°C
</TD><TD scope="col" width="50"> 180 s
</TD></TR><TR><TD><B>Hold 2</B></TD><TD>72°C
</TD><TD>5 min
</TD></TR></TBODY></TABLE></CENTER><UL><LI>We obtained distinct bright bands on all lanes loaded with products of colony PCR, and their position on the gel corresponds to the predicted sizes of these DNA fragments. This suggests that these colonies carried correctly assembled plasmids. Therefore, we decided to <A href="/Team:Cambridge/Protocols/Mini_Prep" title="Team:Cambridge/Protocols/Mini Prep"> extract plasmids</A> from and prepare <A href="/Team:Cambridge/Protocols/Glycerol_Stocks" title="Team:Cambridge/Protocols/Glycerol Stocks"> glycerol stocks</A> of successful transformants.
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:398px;"><DIV class="thumbcaption">The order of samples: Hyperladder I, GA13d-1, GA13d-1, HyperLadder I, GA13d-2, GA13d-3, GA13d-4, GA14d-1, GA14d-2, GA14d-3 and GA14d-4.</DIV></DIV></DIV></DIV><P>

Back to <A href="/Team:Cambridge/Experiments">Experiments</A></P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Cambridge/Experiments/Synthetic_Reflectin_PCR_and_Construction_of_GA1_to_6">http://2011.igem.org/Team:Cambridge/Experiments/Synthetic_Reflectin_PCR_and_Construction_of_GA1_to_6</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cambridge/Experiments/Synthetic_Reflectin_PCR_and_Construction_of_GA1_to_6" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cambridge/Experiments/Synthetic_Reflectin_PCR_and_Construction_of_GA1_to_6" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cambridge/Experiments/Synthetic_Reflectin_PCR_and_Construction_of_GA1_to_6&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cambridge/Experiments/Synthetic_Reflectin_PCR_and_Construction_of_GA1_to_6&amp;oldid=159024" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>