"
Page
Discussion
View source
History
teams
Log in
 
Team:Cambridge/Experiments/Initial Exercise Group One
From 2011.igem.org
Loading...
iGEM HQ
Page Source
Intro Source
Login
My Account
Logout
OVERVIEW
home
OVERVIEW
HOME
PARTS
MEDIA
BLOG
PROJECT
OVERVIEW
BACKGROUND
MICROSCOPY
IN VIVO
IN VITRO
GIBTHON
FUTURE
LABWORK
OVERVIEW
EXPERIMENTS
PROTOCOLS
CONSTRUCTS
SAFETY
SOCIETY
OVERVIEW
INTERVIEWS
IMPACT OF iGEM
TEAM
OVERVIEW
STUDENTS
ACADEMICS
SPONSORS
Contents
1 Initial Exercise: Katy, Heather, Marta and Veronica
1.1 Role of DivIVA
1.2 Construct Design
1.3 Primer Design
1.3.1 Left side of the Div IVA coding sequence (site 1)
1.3.2 Right side of the Div IVA coding sequence (site 2)
1.4 Predicted results
1.5 Method
1.5.1 PCR Taq
1.5.2 PCR Phusion
1.6 Technical Data
1.7 PCR Results using Taq
1.8 PCR Results using Phusion
1.9 Electrophoresis
1.10 Gel Extraction of DNA
1.11 Gibson Assembly
1.12 Transformation of E.coli
Initial Exercise: Katy, Heather, Marta and Veronica
The aim of the project was to produce interesting patterns of fluorescence in a colony of Bacillus Subtilis bacteria by fusion of a single gene of interest in the bacillus genome to GFP using Gibson Assembly. These patterns would need to be clearly observable using a confocal microscope. After some research, our group decided to fuse GFP to the DivIVA gene.
Role of DivIVA
DivIVA is believed to control septum positioning in Bacillus Subtilis. This was investigated (in this study by Edwards and Errington) by considering the phenotype of mutants produced when certain bases in the gene were altered.
Mutants showed:
inhibition of cell division initiation*
abnormally placed septum*
some small anucleate cells*
We also hoped that this fusion would have a high chance of success, since a successful DivIVA-GFP fusion had already been documented, indicating that fluorescence was constitutively expressed at all sites of cell division, new and old, in the bacteria. We therefore know what characteristics to expect and so we should be able to determine whether or not the experiment was successful almost immediately under the microscope.
Construct Design
We wish to:
keep the original promotor for DivIVA*
remove the GFP promotor sequence*
remove the DivivA stop codon so that GFP is translated together with the DivIVA protein*
Remove the ComGA stabiliser sequence pre-existing on the plasmid vector*
‘Linker’ code was also used by Edwards and Errington - however, we suspect that this is related to the restriction enzyme assembly process they used and hence we didn’t include it in our design.
Primer Design
We used the 'Gibthon' Construct Designer developed by the 2010 iGEM Cambridge team to aid us when designing our primers; we did however alter one of the primers slightly in order to achieve similar melting temperatures for primers that would operate on the same DNA fragment.
Left side of the Div IVA coding sequence (site 1)
Right side of the Div IVA coding sequence (site 2)
Predicted results
Bacillus bacteria constitutively express fluorescence
Fluorescence seen at both old and new sites of cell division
Method
PCR Taq
20bp primers with 20bp tails were designed to perform PCR to amplify our chosen DNA sequences and Gibson Assembly to fuse the SrfA promoter with a plasmid containing GFP.
100 μM stock solutions of the primers were made and subsequently diluted four-fold to make 25μM 'working solutions'
Particular care was taken in preparing the inital stock as these affect all future dilutions and were 'contingency' solutions in case of mistake.
The genomic DNA of B. subtilis and the vector plasmid containing GFP were provided.
The vector plasmid is to be cloned in two fragments (otherwise the sequence would be too long for effective PCR), so two additional primers were required. These were provided for us in the appropriate concentration.
Our three PCR reactions consisted of:
Reaction A
1μl primer MKX2
1μl primer MKX3
1μl B. subtilis genomic DNA
Primer datasheet
Reaction B
1μl primer MKX1
1μl primer A Forward (provided)
1μl vector template
Reaction C
1μl primer MKX4
1μl primer B reverse (provided)
1μl vector template
To each tube we also added:
9.5μl of water
12.5μl Master mix (SyBR Green and Rox, Hotstart Taq polymerase, dNTPs and dyes)
for a total volume 25μl
The complete tubes were run in a real-time PCR machine (so that we would be able to observe the progress of our very first PCR reaction in real time) for 30 cycles with a 2 minute extension time and a primer annealing temperature of 50ºc.
Unfortunately, the PCR reaction that took place using the above solutions was not successful (see below), so afterwards we repeated the experiment using the same primers and templates as above, but with a polymerase--PCR Phusion (see section 1.5.2)
PCR Phusion
PCR reaction is repeated using the same primers and templates with Phusion, a much more processive polymerase. The total volume of master mix was 22ul per tube, which was comprised of:
5ul buffer 0.5ul dNTP 0.25ul phusion polymerase 16.25ul H2O
Technical Data
We used Finnzymes melting temperature calculator to work out the melting temperature of the primer part (not the tail) of our Gibson Assembly oligos (this was the 3' 20bp of each oligo) these values in ºC are shown below Needs to be changed MKX1 - 63.49 MKX2 - 56.92 MKX3 - 56.49 MKX4 - 62.24
PCR Results using Taq
Results from PCR reaction
Reaction 1 and 3 were amplified in the PCR however reaction 2 was not
It is suspected this was a result of a compromise run of the PCR where:
Annealing temperature set at an arbitrary 50ºc to produce annealing for every group
Primer extension time was arbitrarily set at 3 mins
Results from all groups indicated the PCR failed for the longest fwdA fragment. We ascribe this to the use of Taq polymerase instead of Phusion polymerase - Phusion is faster with higher processivity.
It is intended to carry out the PCR again using Phusion polymerase.
PCR Results using Phusion
We conducted PCR reaction using Phusion polymerase instead of Taq polymerase. As we did not monitor the progress of amplification in real time, we performed gel electrophoresis afterwards to check the size and amount of products obtained.
Electrophoresis
We conducted gel electrophoresis of our samples according to the following protocol .
the bands were not of equal thickness, suggesting that we obtained different amounts of products in the amplification reactions
the sizes of DNA products, as estimated from the ladder of molecular markers, agreed with predictions based on the sequence of the plasmid
Gel Extraction of DNA
We extracted DNA from the agarose gel and purified it according to the following protocol .
Gibson Assembly
We performed Gibson Assembly of the three amplified DNA fragments, as described in the following protocol . The expected product of the reaction was a modified plasmid, in which the promoter and GFP-coding region were replaced by DivIVA:GFP with an original Div IVA promoter region.
Transformation of E.coli
We used products of the Gibson Assembly to transform competent E.coli cells (details of the procedure described in the protocol ) and after overnight incubation at 37°C we examined colonies under the fluorescent microscope. Unfortunately, no colonies were present on the 100 agar plate (100µl of suspension of bacterial cells plated), and two transparent spots found on 10 agar plate (10µl of suspension of bacterial cells plated) turned out not to be bacterial colonies.  We decided not to repeat the experiment, but the following steps of the procedure would include:
growth of cultures in liquid medium
isolation of DNA (QIAGEN MiniPrep Kit )
digestion of DNA with restriction enzymes and gel electrophoresis of products - a method that allows to test for the correct assembly of the vector. The restriction map of the plasmid together with predicted sizes of digestion product is presented below:
Restriction map of the original plasmid
Restriction map of the plasmid with DivIVA:GFP fusion
transformation of competent Bacillus subtilis cells with isolated plasmids
examination of colonies and single cells under fluorescent miscroscope
Retrieved from " http://2011.igem.org/Team:Cambridge/Experiments/Initial_Exercise_Group_One "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
