"
Page
Discussion
View source
History
teams
Log in
 
Team:Cambridge/Protocols/PCR
From 2011.igem.org
Loading...
iGEM HQ
Page Source
Intro Source
Login
My Account
Logout
OVERVIEW
home
OVERVIEW
HOME
PARTS
MEDIA
BLOG
PROJECT
OVERVIEW
BACKGROUND
MICROSCOPY
IN VIVO
IN VITRO
GIBTHON
FUTURE
LABWORK
OVERVIEW
EXPERIMENTS
PROTOCOLS
CONSTRUCTS
SAFETY
SOCIETY
OVERVIEW
INTERVIEWS
IMPACT OF iGEM
TEAM
OVERVIEW
STUDENTS
ACADEMICS
SPONSORS
Back to Protocols
Contents
1 Polymerase Chain Reaction (PCR)
1.1 Theory
1.2 Practice
1.2.1 Reagents used in PCR reaction
1.2.2 Conditions of PCR reaction
1.3 PCR Modifications
1.3.1 Gradient PCR
1.4 Safety
Polymerase Chain Reaction (PCR)
The goal of PCR is to amplify a section of DNA of interest for DNA analysis (e.g. gene insertion, sequencing, etc). The amplification rate is exponential.
Theory
Schematic drawing of the PCR cycle.
(1) Denaturing at 94–96 °C           (2) Annealing at ~65 °C           (3) Elongation at 72 °C
Four cycles are shown here. The blue lines represent the DNA template to which primers ( red arrows ) anneal that are extended by the DNA polymerase ( light green circles ), to give shorter DNA products ( green lines ), which themselves are used as templates as PCR progresses. Credit, WikiMedia
The reaction works by the annealing of primers to single stranded DNA, which are extended by DNA polymerase. By repeating the process, the section of DNA between the primers is amplified.
During our project, we are using Phusion Hot Start II DNA Polymerase that is characterised by approximately 50-fold lower error rate than Thermus aquaticus Taq polymerase (the first thermostable polymerase used in PCR). The polymerase, in addition to having 5'→ 3' DNA polymerase activity, also possesses 3'→ 5' exonuclease activity.
The Phusion Polymerase allows us to perform Hot Start PCR - a modified version of PCR that avoids non-specific amplification of DNA. The Phusion Polymerase is reversibly bound to specific Affibody protein, which inhibits the DNA polymerase activity at ambient temperatures. Additionally, the Affibody ligand inhibits the 3'→ 5' exonuclease activity of the polymerase, preventing degradation of primers and template DNA during reaction setup. At polymerisation temperatures, the Affibody molecule is released, rendering the polymerase fully active.
Practice
Reagents used in PCR reaction
The PCR reaction with Phusion Hot Start II DNA Polymerase contains the following reagents:
Name
50 μl reaction
20 μl reaction
Final concentration
Water (reverse osmosis)
add to make 50 μl
add to make 20 μl
5x Phusion Buffer
10 μl
4 μl
1x
10mM dNTPs
1 μl
0.4 μl
200 μM each
Forward Primer 10 μM
x μl
x μl
0.5 μM
Reverse Primer 10 μM
x μl
x μl
0.5 μM
Template DNA
x μl
x μl
Phusion Polymerase
0.5 μl
0.2 μl
0.02 U/μl
Tip: The recommended buffer is 5x Phusion HF Buffer, used as a default buffer for high fidelity amplification.
Tip: The DNA polymerase should be pipetted carefully and gently as the high glycerol content (50%) in the storage buffer may otherwise lead to pipetting errors.
When you receive primers synthesized by a biotechnological company
centrifuge tubes prior to opening to prevent loss of pelleted oligonucleotides (at 3000 rpm for 1 minute)
add the amount of deionized water specified in the table provided with primers to obtain 100 μM solution
prepare a 4-fold dilution of a portion of the primer solution
Only the primers and the DNA template are specific to the reaction. The remaining reagents can be made as a 'Master Mix' in order to reduce the need to repeatedly pipette small volumes which amplify experimental error.
The amount of template DNA added to a PCR reaction varies with its complexity:
for plasmid, lambda or BAC DNA, 1 pg - 10 ng per 50 μl reaction volume is recommended
for high complexity genomic DNA, the amount of DNA template should be 50 - 250 ng per 50 μl reaction volume.
As with all enzyme-containing reagents, the master mix should be kept in the freezer and small aliquots thawed when required.
The remainder of the reaction is handled automatically by a PCR machine. It is common practice to perform a gel electrophoresis to extract the correct DNA after PCR.
Conditions of PCR reaction
An example of conditions we used for a PCR reaction using Phusion polymerase are as follows:
Step
Temp.
Time
Hold 1
95°C
6 min
Cycling ×30
Denaturing
98°C
10 s
Annealing
55°C
30 s
Elongation
72°C
180 s
Tip: Use 15-30 s/kb for extension. Do not exceed 1 min/kb.
PCR Modifications
Gradient PCR
Gradient PCR is a modification to a standard PCR procedure that allows to optimize an annealing temperature in order to increase the specificity of the amplification process. It is usually conducted in a thermal cycler in which different annealing temperatures can be set in different parts of the block, maintaining constant denaturing and elongation temperatures. Thus, after running a gel with PCR products and comparing the pattern of bands, it is possible to empirically determine the optimal annealing temperature for a given set of primers.
Safety
No bacteria are used during the reaction there is therefore little or no biological hazard. Nevertheless, it is important to observe correct laboratory procedure and wear appropriate clothing and gloves. PCR occurs at high temperature, and this may present a risk, depending on the PCR machine employed.
Back to Protocols
Retrieved from " http://2011.igem.org/Team:Cambridge/Protocols/PCR "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
