<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_IIT_Madras_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:IIT_Madras/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:IIT_Madras/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:IIT_Madras/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:IIT_Madras/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:IIT_Madras/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:IIT Madras/Project</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><TITLE>Indian Institute of Technology,Madras</TITLE></P><DIV class="header"><DIV class="title">
IIT Madras
</DIV></DIV><DIV class="menu"><DIV class="outer"><UL id="nav"><LI><P><A href="https://2009.igem.org/Team:IIT_Madras">  Home</A></P></LI><LI><P><A href="#">  Team</A></P></LI><LI><A href="https://2009.igem.org/Team:IIT_Madras/Team1">Team</A></LI><LI><A href="https://2009.igem.org/Team:IIT_Madras/Team_under">Undergraduates</A></LI><LI><A href="https://2009.igem.org/Team:IIT_Madras/Team_advis">Advisors</A></LI><LI><A href="https://2009.igem.org/Team:IIT_Madras/Team_uni">Our University</A></LI></UL><LI><P><A href="#">  Project</A></P></LI><LI><A href="https://2009.igem.org/Team:IIT_Madras/Summary">Summary</A></LI><LI><A href="https://2009.igem.org/Team:IIT_Madras/Theory">Theory</A></LI><LI><A href="https://2009.igem.org/Team:IIT_Madras/Circuit">Circuit</A></LI><LI><A href="https://2009.igem.org/Team:IIT_Madras/exp_beh">Expected Behaviour</A></LI><LI><A href="https://2009.igem.org/Team:IIT_Madras/Workplan">Work Plan</A></LI><LI><A href="https://2009.igem.org/Team:IIT_Madras/Experiments">Experiments</A></LI><LI><A href="https://2009.igem.org/Team:IIT_Madras/Results">Results</A></LI><LI><P><A href="#">  Notebook</A></P></LI><LI><A href="https://2009.igem.org/Team:IIT_Madras/Notebook/Calendar">Dynamic Calendar</A></LI><LI><A href="http://openwetware.org/wiki/IGEM:Indian_Institue_of_Technology_Madras/2009/Notebook/PLASMID_-_Plasmid_Locking_Assembly_for_Sustaining_Multiple_Insert_DNA">OpenWetWare</A></LI><LI><A href="https://2009.igem.org/Team:IIT_Madras/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2009.igem.org/Team:IIT_Madras/Notebook/Parts Submitted">Parts Submitted</A></LI><LI><A href="https://2009.igem.org/Team:IIT_Madras/Notebook/Sandbox">Sandbox</A></LI><LI><P><A href="#">  Miscellaneous</A></P></LI><LI><A href="https://2009.igem.org/Team:IIT_Madras/Miscellaneous/Safety">Safety</A></LI><LI><A href="https://2008.igem.org/Team:IIT_Madras">IIT iGEM 2008</A></LI></DIV></DIV><DIV id="aContent"><P class="justifyalign">Our project PLASMID - Plasmid Locking Assembly for Sustaining Multiple Inserted DNA - introduces a new paradigm in gene regulation. The study is based on the concept of plasmid loss. Any episome (extrachromosomal genetic element) introduced into the cell shows a segregational asymmetry accompanied with differential growth rates in the absence and presence of episome leading to an overall loss of the episomal unit in the absence of any maintaining selective pressure. It is hypothesized that by appropriately controlling the external selective pressures, one can control the direction of plasmid loss in the cell, modifying the existing gene regulation system in a pre determined manner. It is also hypothesized that introducing negative selective pressures against certain other directions of plasmid loss, in the form of constitutively repressed endotoxins will help streamline the regulatory system even further. 

</P><P class="justifyalign">If successful, this study allows for exquisitely delicate and precise multifactorial regulation of gene control in the future. This model can also be used to hide genes of commercial interest to protect it from unauthorized use (under some conditions).</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Summary"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Summary</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Theory"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Theory</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Circuit"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Circuit</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Expected_behavior_of_the_system"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Expected behavior of the system</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Work_Plan"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Work Plan</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Building_the_constructs"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Building the constructs</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Working_with_the_construct"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Working with the construct</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-8"><A href="#Experiments"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Experiments</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Growth_curves"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Growth curves</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Modeling"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Modeling</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Fluorescent_Imaging"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">Fluorescent Imaging</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Results"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Growth_curves_2"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Growth curves</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Modeling_2"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">Modeling</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Fluorescent_Imaging_2"><SPAN class="tocnumber">7.3</SPAN><SPAN class="toctext">Fluorescent Imaging</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Summary">Summary</SPAN></H2><P class="justifyalign">Our project is based on the fundamental concept of plasmid instability in a novel way to conceal information or ‘lock’ a gene’s function in a cell until the correct combination of inputs is fed into the cell. We call this a ‘combinatorial lock’ or PLASMID. It involves the positive regulation of the gene of interest only on receiving the correct inputs from the user. We use plasmids which can confer resistance to certain antibiotics in the medium and link them up in a certain way (i.e, essentially designing a genetic circuit) so that they repress the expression of the gene of our interest. As the selection pressure is lifted from the media, the plasmids which have the repressors for the gene of interest are lost, hence revealing the gene on using the correct series of antibiotic washes. In essence, the process of unlocking would simply be the correct sequence of antibiotic media in which the cells should be washed.</P><P class="justifyalign">We would be working with a 2 plasmid system and it is easy to see that this principle, theoretically, could be extended to N plasmids. In general the code length required to &quot;unlock&quot; is N-1 if the number of plasmids introduced are N. In our case, since the number of plasmids being introduced are 2, the code would essentially be just 1 unit long. Particularly in this case, the 1 unit of code corresponds to growing the cells in one correct antibiotic medium.
</P><P class="justifyalign"><B><FONT color="#000">Fig 1:</FONT></B><I>The gene of interest is &quot;locked&quot; or repressed by the inhibitor in the plasmid 2. The plasmids are linked up in a certain way that the plasmids need to be lost only in a very specific order, else the cells die due to the release of a toxin. Thus the &quot;unlocking&quot; of the gene of interest requires a predetermined order of growth conditions which allows for a directional loss of the plasmids, and hence the repressor for the promoter that expresses the gene of interest.</I></P><P class="justifyalign">
However, in the experiments we have not incorporated any particular gene to be repressed. Instead we study how can we achieve a directed loss of plasmids which is the idea central to the working of the system. In place of a gene of interest, we have placed fluorescent reporters in each plasmid to monitor the presence or absence of any particular plasmid.</P><H2><SPAN class="mw-headline" id="Theory">Theory</SPAN></H2><P class="justifyalign">The most important idea behind the working of the lock is plasmid loss due to lack of selection. Any extra-chromosomal genetic material introduced into the cell tends to disappear over the generations, unless it confers a selective survival advantage over the cells that do not possess the plasmid. During the growth of bacteria, plasmid-free variants arise in the initially homogeneous plasmid-bearing cell population basically in two ways. First, each plasmid-bearing cell has a certain probability to give rise to a plasmid-free cell at cell division (this depends on the mechanisms of plasmid distribution between daughter cells, plasmid copy number at the cell division, the presence of multimer resolution loci, etc.).</P><P class="justifyalign"><B><FONT color="#000">Fig 2.1:</FONT></B><I>During cell division, rarely all the plasmids segregate into only one of the daughter cells, thus giving rise to plasmid free cells in the population. This event happens independent of what kind of plasmid the cell contains.</I></P><P class="justifyalign"><B><FONT color="#000">Fig 2.2:</FONT></B><I>Cells containing plasmids have a higher metabolic strain of synthesizing the proteins that are encoded by the plasmid, thus they tend to grow slower than the cells without any plasmids. The growth difference is pronounced especially during the exponential phase.</I></P><P class="justifyalign">Usually, such probability is very low for natural plasmids (about 10-7) whereas recombinant plasmids (i.e. genetically modified) may segregate with a higher probability (10-3—10-5). Several hypotheses have been put forward to explain this tremendous difference: impaired copy number control, the absence/impairment of the multimer resolution genes and random distribution of plasmid among daughter cells for low copy number plasmids. 
Second, it was experimentally found that plasmid-bearing cells usually have a lower maximum specific growth rate than their plasmid-free counterparts, and once a plasmid-free cell arises, it competes with its plasmid-bearing counterparts rapidly and ultimately phases it out. Since most recombinant plasmids are not conjugative (not capable of self-transfer to other plasmid-free cells), if a cell has lost a plasmid, there is no way for the cell to acquire it again. Thus, a segregation of plasmids at cell division and the difference in the growth rates of plasmid-free and plasmid bearing subpopulations determine the rate at which plasmids are lost during prolonged cultivation. Here, we use plasmids which can confer resistance to certain antibiotics in the medium and link them up in a certain way so that they repress the expression of the gene of our interest. As the selection pressure is removed, the plasmids which have the repressors for the gene of interest are lost, hence revealing the gene on using the correct series of antibiotic washes.</P><P class="justifyalign"><B><FONT color="#000">Fig 2.3:</FONT></B><I>In the initial state, all the cells bear a network of plasmids which can sustain each other under the influence of a particular selection pressure. In this state, the plasmids other than the one containing the gene of interest have repressors for the promoter that expresses the gene of interest in plasmid 1. As the cells are grown in a correct order of varying selection pressures, the cells lose plasmids directionally, thus leading to the expression or &quot;unlocking&quot; of the gene of interest.</I></P><H2><SPAN class="mw-headline" id="Circuit">Circuit</SPAN></H2><P class="justifyalign">One of the plasmids will have the gene of interest which will be expressed only when the plasmids apart from this particular one are lost in a particular order. This particular order or “code” will consist of a sequence of antibiotic treatments given to the transformed cells. The correct code triggers the loss of plasmids in the cells in a particular order. Each plasmid will be linked to the plasmids which are supposed to be lost before and after it in a highly regulated fashion. The plasmid loss can be regulated very tightly using a “suicide gene” (gene coding for the bacterial gyrase poison). These genes will be triggered when the culture is subjected to the wrong antibiotic (Out of sequence). To ensure that the suicide genes don’t fire randomly, they are under the control of repressors which are on the plasmids that are supposed to be lost after it and are expressed constitutively. Thus, they fire only when the plasmid containing the repressor is lost. When all the plasmids other than the plasmid containing the gene of interest are lost, the repressors that have been blocking the required gene are lost, thus allowing its expression, which can manifest as a phenotype or function in the cell. This will be the “unlocking” of the lock.</P><P>The grand idea of a 3 plasmid locking system revolved around constructing something like the below:
</P><P class="justifyalign"><B><FONT color="#000">Fig 3.1:</FONT></B><I>This is a general circuit for a 3 plasmid system which can show locking property with a code length of 2 units - Ab2-&gt;Ab1. This means that the cells need to be grown first in antibiotic medium 2 and then transferred to a medium containing antibiotic 1. It is easy to see that to maintain the whole system of 3 plasmids, the cells need to be grown in medium containing antibiotic 3.</I></P><P>To which we thought we could fit in the following parts (the gene of interest is not included. It could be anything):
</P><P><B><FONT color="#000">Fig 3.2:</FONT></B><I>The 3 plasmid system.</I></P><P>
Then we scaled it down and began to build constructs for a 2 plasmid system which works like this:
</P><P class="justifyalign"><B><FONT color="#000">Fig 3.3:</FONT></B><I>This is a general circuit for a 2 plasmid system that can show locking property with a code length of 1 unit - which means the system is unlocked when it is grown in a medium containing antibiotic 1.</I></P><P>To which we fit in parts from the registry to make it look like this:
</P><P class="justifyalign"><B><FONT color="#000">Fig 3.4:</FONT></B><I>The 2 plasmid system.</I></P><P>For the proof of concept experiments, we built the following constructs:
</P><P><A href="http://partsregistry.org/Part:BBa_K272001" class="external text" rel="nofollow">K272001-constitutive RFP expressor</A></P><P><B><FONT color="#000">Fig 3.5:</FONT></B><I>This plasmid confers constitutive expression of RFP in the transformed cells.</I></P><P><A href="http://partsregistry.org/Part:BBa_K272002" class="external text" rel="nofollow">K272002-constitutive CFP expressor</A></P><P><B><FONT color="#000">Fig 3.5:</FONT></B><I>This plasmid confers constitutive expression of CFP in the transformed cells.</I></P><H2><SPAN class="mw-headline" id="Expected_behavior_of_the_system">Expected behavior of the system</SPAN></H2><B><FONT color="#000">Fig 4.1: The cultures are grown in a medium containing Ampicillin to direct the loss of the plasmid 2 so that the gene of interest is expressed as the repressor for the gene of interest lies in the plasmid 2.  </FONT></B><B><FONT color="#000">Fig 4.2: The cultures are grown first in medium containing Chloramphenicol and then transferred to a medium containing Ampicillin. This means that the medium containing Chloramphenicol is requried to maintain both the plasmdis while growth in Ampicillin is required for unlocking the gene of interest </FONT></B><H2><SPAN class="mw-headline" id="Work_Plan">Work Plan</SPAN></H2><H3><SPAN class="mw-headline" id="Building_the_constructs">Building the constructs</SPAN></H3><P>We have been following 3 different strategies to build our constructs.
</P><P>1. The 3A assembly
</P><P>2. The 2A or Standard Assembly
</P><P>3. PCR based approach: We plan to proceed with the experiment using a PCR based amplification system, followed by a specific restriction digest and subsequent ligation.
</P><P>This strategy saves a lot of time in cloning the parts together. All it takes is a PCR, a purification step, digestion and ligation. This strategy takes much less time in comparision with the traditional cloning steps using bacteria. Moreover, this process can be automated to ligate a huge number of parts together in a short period of time. It saves the labor of dealing with bacteria for amplifying the required insert after each ligation. 
</P><H3><SPAN class="mw-headline" id="Working_with_the_construct">Working with the construct</SPAN></H3><P class="justifyalign">We will be performing a proof-of-concept experiment using the 2 plasmids - K272001 and K272002. The cells will be co-transformed with the plasmids. Each of these plasmids have a different fluorescent reporter and their backbones have different antibiotic resistances. However, the origins of replication in both the plasmids are the same, thus making the plasmids compete with each other in order to remain in the cell based on the selection pressures. The bacteria &quot;titrates&quot; the number of plasmids with a certain origin of replication before each replication. Thus if there are a couple of plasmids with the same origin of replication, and one of them happens to be non essential and does not confer any advantage to the cell for a given growth condition it will eventually be phased out from the population. This is essentially mimicking natural selection. The fluorescent reporters will help in tracing which plasmid is phasing out for a given set of growth conditions.</P><P class="justifyalign">The cotransformed cells will be grown in different antibiotic media (in our case - Ampicillin, Chloramphenicol and both Ampicillin and Chloramphenicol) and the loss of each plasmid will be studied using fluorescent markers under a microscope. Presently we are working with the K272001 (constitutive RFP) in pSB1C3 and K272002 (constitutive CFP) in pSB1A2. We expect to see that the co-transformed cells when grown in a medium with both chloramphenicol and ampicillin should have more number of cells with  both the fluorescent reporters than with those which are grown in medium containing either or none of the antibiotics. Specifically, if the cells are grown in medium containing only ampicillin, then the number of cells without CFP should increase and when grown in the absence of chloramphenicol, the number of cells without RFP should be on the rise.</P><P class="justifyalign"><B><FONT color="#000">Fig 6:</FONT></B><I>The cotransformed cells when grown in different antibiotic media, show different fates. For instance, when the cells are grown in medium containing Ampicillin but not Chloramphenicol, the lack of selection pressure due to chloramphenicol on the cells leads to a gradual phasing out of those cells which contain both the plasmids. Only those with the Ampicillin selection marker eventually dominate the population.</I></P><P>The constructs required to demonstrate the locking property are being built and will be tested once the construction is done.
</P><H2><SPAN class="mw-headline" id="Experiments">Experiments</SPAN></H2><H3><SPAN class="mw-headline" id="Growth_curves">Growth curves</SPAN></H3><H3><SPAN class="mw-headline" id="Modeling">Modeling</SPAN></H3><H3><SPAN class="mw-headline" id="Fluorescent_Imaging">Fluorescent Imaging</SPAN></H3><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><H3><SPAN class="mw-headline" id="Growth_curves_2">Growth curves</SPAN></H3><H3><SPAN class="mw-headline" id="Modeling_2">Modeling</SPAN></H3><H3><SPAN class="mw-headline" id="Fluorescent_Imaging_2">Fluorescent Imaging</SPAN></H3></DIV><P> </P><DIV id="wikifooter"><P><A href="#top">Back to Top of Page</A></P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:IIT_Madras/Project">http://2009.igem.org/Team:IIT_Madras/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:IIT_Madras/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:IIT_Madras/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:IIT_Madras/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:IIT_Madras/Project&amp;oldid=145705" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>