"
Page
Discussion
View source
History
teams
Log in
 
Team:SCAU-China/Omega-PCR
From 2014.igem.org
(Redirected from Team:SCAU-China/Parts )
Home
Team +_
- Members - Committee Members
Project +_
- Overview - arcA knock out - nadE overexpression - ppsA promoter - MFC/MDC device
Omega-PCR
Results +_
- Parts - Modeling
Human practices +_
- Expert Consultation - Questionnaire - Safety - Outreach
Notebook +_
- Methods & Materials - Photo Gallery
Attributions
Omega-PCR
Introduction
Ω-PCR is a simple, flexible and low-cost molecular manipulation strategy developed by Letian Chen in 2012 [1].
It is a sequence modification strategy based on an overlap extension site-directed mutagenesis technique, which enables multiple types of sequence modification including precise insertion, deletion and substitution in any positions of a circular plasmid. In our project, we utilized this method to insert certain gene, promoter sequences into existing parts and substitute subpart of an existing constructs. It can be performed in two steps. The product of the first-step PCR, which amplifies the insertional or substitutional sequences, plays the role of omega-primers of the second-step PCR and forms a Ω-like secondary structure in the mega-primers, which achieve insertion or substitution of target sequence in a new construct. Thus, we have adopted this efficient and seamless method to construct our parts.
Advantages
1. Robust--enables multiple types of precise sequence manipulation of existed standard BioBricks in purification-free manner.
2. Seamless--beneficial for wide applications for protein engineering, gene function analysis and in vitro gene splicing.
3. Low cost--each reaction costs less than $0.1.
4. Timesaving-–all processes perform based on PCR reaction, normally 1 biobrick 1 day.
Disadvantages
1. Capacity varies on the quality of DNA polymerase.
2. Necessary treatments for avoiding false positive clones.
How does Ω-PCR work?
Ω-PCR can be applied in sequence insertion, deletion and substitution of an existed plasmid, and the principles of different applications are similar. Here we describe the insertion type.
1st Ω-PCR—— Create insertion sequence with two ends of overlap terminals.
If we want to insert the target gene (nadE gene from E.coli MG1655）sequence at the insertion site（between BBa_K608002 and the standard BioBrick suffix) shown above, we need to design the pair of primers which are able to amplify the open reading frame of nadE gene with an overlapping sequence at their 5’terminals from the genomic DNA of E.coli MG1655. In this way, the amplification product can extent and contains two short overlap sequences on the edge of the insertion site.   After treatment by exonuclease I, the single strand primers are digested, minimizing the side effects in 2nd Ω-PCR.
2nd Ω-PCR——Amplify the full length vector with‘omega primers’
In the second PCR, the insertion sequence, amplified from the 1st Ω-PCR, serve as‘omega-primers', with the overlap sequence specifically binding flank of the insertion site on template plasmid pSB1C3-BBa_K608002. The target gene sequence between two overlap sequences, however, is shaped into an Ω-letter-liked secondary structure. Under the catalysis of high-fidelity DNA polymerase, the whole plasmid will then be amplified with the target gene sequence inserted. Although there are still nicks in the 2nd Ω-PCR product, they will be repaired after transformed into E.coli competent cells.
Protocol
Step1: Design primers
A.For insertion Omega PCR
If you want to insert sequence A at site 1, the primers can be designed as the graph shown above, the length of the overlap sequence is more than 20 bps.
B.For substitution Omega PCR
If you want to substitute sequence B with the gene of interest sequence A between site 1 and site 2, design the pair of primers as the graph shown above.
C.For deletion Omega PCR
If you want to delete sequence B between site 1 and site 2, design primers as the graph shown above.
Step2: 1st Ω-PCR
As the amplification reaction require high fidelity and elongation capability, we use KOD FX ( TOYOBO CO., LTD. Life Science Department OSAKA JAPAN ) as the PCR enzyme.
1.Reaction mixture
 
For 1 reaction
Final concentration
2x PCR buffer for KOD FX
10 μl
1 ×
2mM dNTPs
4 μl
0.4 mM each
10μM Primer #1
0.4
0.2 μM
10μM Primer #2
0.4
0.2 μM
Template DNA
≧ 1 μl
Genomic DNA ~200 ng / 50μl
Plasmid DNA ~50 ng / 50μl
cDNA (from ~200 ng RNA) / 50μl
KOD FX (1.0U/μl)
0.4 μl
1.0 U / 50 μl
Autoclaved, distilled water
up to 20 μl
 
2.PCR cycle conditions
Step
Temp
Time
1:Predenature
94 °C
4 min
2:Denature
98 °C
15 s
3:Annealing
（Tm-5）°C
30 s
4:Extension
68 °C
1min/kb DNA
5:Go To step 2
/
35 cycles
6:Final Extension
68 °C
10 min
7:Storage
4 °C
∞
Step3: Exonuclease I digestion
To eliminate the side effect of primers in 1st Ω-PCR to the next step, we use exonuclease I to digest the single strand primers.
1.Reaction mixture
10X Exonuclease I Reaction Buffer
1 μl
Exonuclease I
5-10 U
1st Ω-PCR product
8 μl
Distilled water
up to 10 μl
2.Reaction condition
Incubate at 37 ℃ for 30 min
Step4: 2nd Ω-PCR
1.Reaction mixture
 
For 1 reaction
Final concentration
2 x PCR buffer for KOD FX
10 μl
1×
2mM dNTPs
4 μl
0.4 mM each
1st Ω-PCR product
≥1 μl
50~400 ng
Template plasmid
≥ 1 μl
5~100 ng
KOD FX (1.0 U/μl)
0.4 μl
1.0 U / 50 μl
Autoclaved, distilled water
up to 20 μl
 
2.PCR cycle conditions
Step5: Transformation
The 2nd Ω-PCR products are digested by DpnI to remove template plasmid before transformation, avoiding the excess of negative clones.
Reference
[1] Chen L, Wang F, Wang X, Liu YG. Robust one-tube Ω-PCR strategy accelerates precise sequence modification of plasmids for functional genomics. Plant Cell Physiology. 2013, 54: 634-642
Retrieved from " http://2014.igem.org/Team:SCAU-China/Omega-PCR "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
