<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SDU-Denmark_labnotes2"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SDU-Denmark/labnotes2">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SDU-Denmark/labnotes2&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SDU-Denmark/labnotes2&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SDU-Denmark/labnotes2&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SDU-Denmark/labnotes2" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SDU-Denmark/labnotes2</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="header"><DIV id="return"><A href="https://2010.igem.org/">Back to iGEM 2010</A></DIV></DIV><DIV id="navigation"><UL id="topnav"><LI><A href="/Team:SDU-Denmark">               Home</A></LI><LI><A href="#">Team</A><SPAN><A href="/Team:SDU-Denmark/team">Students</A>|
            <A href="/Team:SDU-Denmark/team2">Advisors</A>|<A href="/Team:SDU-Denmark/team3">Sponsors</A>|
            <A href="/Team:SDU-Denmark/team4">Collaborators</A></SPAN></LI><LI><A href="#">Project</A><SPAN><A href="/Team:SDU-Denmark/project-i">Idea</A>|
            <A href="/Team:SDU-Denmark/project-t">Theory</A>|
            <A href="/Team:SDU-Denmark/project-activities">Activities</A>|
            <A href="/Team:SDU-Denmark/parts">Parts</A>|
            <A href="/Team:SDU-Denmark/project-r">Result summary</A>|
            <A href="/Team:SDU-Denmark/project-p">Characterization</A></SPAN></LI><LI><A href="#">Modelling</A><SPAN><A href="/Team:SDU-Denmark/project-m">Physical</A>|
            <A href="/Team:SDU-Denmark/project-bc">Biochemical</A></SPAN></LI><LI><A href="#">Safety</A><SPAN><A href="/Team:SDU-Denmark/safety-a">Overview</A>|
            <A href="/Team:SDU-Denmark/safety-b">Project and Researcher Safety</A>|
            <A href="/Team:SDU-Denmark/safety-c">Laws and Guidelines</A>|
            <A href="/Team:SDU-Denmark/safety-d">Watermark</A></SPAN></LI><LI><A href="#">Notebook</A><SPAN><A href="/Team:SDU-Denmark/notebook">Progress</A>|
            <A href="/Team:SDU-Denmark/labnotes0">Lab notes</A>|
            <A href="/Team:SDU-Denmark/protocols">Protocols</A></SPAN></LI><LI><A target="blank" href="http://igem.sdu.dk">Blog</A></LI></UL></DIV><DIV id="subnavi"><DIV id="leftcolumn"><H1><SPAN class="mw-headline" id="Lab_notes_.287.2F19_-_7.2F25.29"> Lab notes (7/19 - 7/25) </SPAN></H1><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Lab_notes_.287.2F19_-_7.2F25.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Lab notes (7/19 - 7/25)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Group:_Flagella"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Group: Flagella</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Annealing_of_the_two_mutated_strands_of_FlhDC_.28FlhDCmut.29"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Annealing of the two mutated strands of FlhDC (FlhDCmut)</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Amplification_of_FlhDCmut"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Amplification of FlhDCmut</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Gel_extraction_of_FlhDCmut"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">Gel extraction of FlhDCmut</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Restriction_digest_of_FlhDCmut_and_pSB1C3_X2"><SPAN class="tocnumber">1.1.4</SPAN><SPAN class="toctext">Restriction digest of FlhDCmut and pSB1C3 X2</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Gel_extraction_of_FlhDCmut_and_pSB1C3_X2"><SPAN class="tocnumber">1.1.5</SPAN><SPAN class="toctext">Gel extraction of FlhDCmut and pSB1C3 X2</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Amplification_of_FlhDCmut_2"><SPAN class="tocnumber">1.1.6</SPAN><SPAN class="toctext">Amplification of FlhDCmut</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Gel_extraction_of_FlhDCmut_2"><SPAN class="tocnumber">1.1.7</SPAN><SPAN class="toctext">Gel extraction of FlhDCmut</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Gel_extraction_of_flhD.2FC_.28native_coding_sequence.29"><SPAN class="tocnumber">1.1.8</SPAN><SPAN class="toctext">Gel extraction of flhD/C (native coding sequence)</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Digestion_of_flhD.2FC_.28native_coding_sequence.29_and_pSB1A2_with_EcoRI_SpeI"><SPAN class="tocnumber">1.1.9</SPAN><SPAN class="toctext">Digestion of flhD/C (native coding sequence) and pSB1A2 with EcoRI SpeI</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Gel_extraction_of_digested_flhD.2FC_.28native_coding_sequence.29_and_pSB1C3"><SPAN class="tocnumber">1.1.10</SPAN><SPAN class="toctext">Gel extraction of digested flhD/C (native coding sequence) and pSB1C3</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Ligation_of_flhD.2FC_.28native_coding_sequence.29_and_pSB1C3"><SPAN class="tocnumber">1.1.11</SPAN><SPAN class="toctext">Ligation of flhD/C (native coding sequence) and pSB1C3</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Coloni_PCR_of_flhD.2FC_.28native_coding_sequence.29_in_pSB1C3"><SPAN class="tocnumber">1.1.12</SPAN><SPAN class="toctext">Coloni PCR of flhD/C (native coding sequence) in pSB1C3</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Coloni_PCR_of_flhD.2FC_.28native_coding_sequence.29_in_pSB1C3_2"><SPAN class="tocnumber">1.1.13</SPAN><SPAN class="toctext">Coloni PCR of flhD/C (native coding sequence) in pSB1C3</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Digestion_of_flhD.2FC_PCR_product_with_EcoRI_and_SpeI"><SPAN class="tocnumber">1.1.14</SPAN><SPAN class="toctext">Digestion of flhD/C PCR product with EcoRI and SpeI</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Miniprep_of_ligation_coloni_.234_and_.239"><SPAN class="tocnumber">1.1.15</SPAN><SPAN class="toctext">Miniprep of ligation coloni #4 and #9</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#Digestion_of_flhD.2FC_mut_with_PstI"><SPAN class="tocnumber">1.1.16</SPAN><SPAN class="toctext">Digestion of flhD/C mut with PstI</SPAN></A></LI><LI class="toclevel-4 tocsection-19"><A href="#Experiment_no.1"><SPAN class="tocnumber">1.1.16.1</SPAN><SPAN class="toctext">Experiment no.1</SPAN></A></LI><LI class="toclevel-4 tocsection-20"><A href="#experiment_no.2"><SPAN class="tocnumber">1.1.16.2</SPAN><SPAN class="toctext">experiment no.2</SPAN></A></LI></UL><LI class="toclevel-3 tocsection-21"><A href="#Purification_of_pSB1A2_containing_J13002"><SPAN class="tocnumber">1.1.17</SPAN><SPAN class="toctext">Purification of pSB1A2 containing J13002</SPAN></A></LI><LI class="toclevel-3 tocsection-22"><A href="#Amplification_of_pSB1A2_w._J13002.2C_pSB1A2_w._B0015_and_pSB1A2_w._B0034"><SPAN class="tocnumber">1.1.18</SPAN><SPAN class="toctext">Amplification of pSB1A2 w. J13002, pSB1A2 w. B0015 and pSB1A2 w. B0034</SPAN></A></LI><LI class="toclevel-3 tocsection-23"><A href="#Amplification_and_purification_of_pSB1A2_w._J13002"><SPAN class="tocnumber">1.1.19</SPAN><SPAN class="toctext">Amplification and purification of pSB1A2 w. J13002</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Group:_Photosensor"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Group: Photosensor</SPAN></A></LI><LI class="toclevel-3 tocsection-25"><A href="#Miniprep_of_K081005_in_pSB1A2_and_R0011_in_pSB1A2_from_transformation_20.2F7"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Miniprep of K081005 in pSB1A2 and R0011 in pSB1A2 from transformation 20/7</SPAN></A></LI><LI class="toclevel-3 tocsection-26"><A href="#Gel_extraction_test_.28GFX_vs._Fermentas.29"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Gel extraction test (GFX vs. Fermentas)</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#Group:_Retinal"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Group: Retinal</SPAN></A></LI><LI class="toclevel-3 tocsection-28"><A href="#Transformation_of_K081005_in_pSB1A2_.28constitutive_promoter_and_RBS_combined.29.2CR0011_in_pSB1A2.2C_pSB3C5_w._J04450_and_pSB3T5_w._J04450_in_Top_10_E.Coli"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Transformation of K081005 in pSB1A2 (constitutive promoter and RBS combined),R0011 in pSB1A2, pSB3C5 w. J04450 and pSB3T5 w. J04450 in Top 10 E.Coli</SPAN></A></LI><LI class="toclevel-3 tocsection-29"><A href="#Transformation_of_flhD.2FC_in_pSB1C3_and_test_plasmid_in_Top_10_E.Coli"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Transformation of flhD/C in pSB1C3 and test plasmid in Top 10 E.Coli</SPAN></A></LI><LI class="toclevel-3 tocsection-30"><A href="#Coloni_PCR_of_R0011_in_pSB1A2_and_K081005_in_pSB1A2"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">Coloni PCR of R0011 in pSB1A2 and K081005 in pSB1A2</SPAN></A></LI><LI class="toclevel-3 tocsection-31"><A href="#Miniprep_of_pSB3T5_w._J04450_and_pSB3C5_w._J04450_from_transformation_20.2F7"><SPAN class="tocnumber">1.3.4</SPAN><SPAN class="toctext">Miniprep of pSB3T5 w. J04450 and pSB3C5 w. J04450 from transformation 20/7</SPAN></A></LI><LI class="toclevel-3 tocsection-32"><A href="#Transformation_of_TOP_10_e._coli_with_ninaB_gene_in_pOT2:chlor_vector"><SPAN class="tocnumber">1.3.5</SPAN><SPAN class="toctext">Transformation of TOP 10 e. coli with ninaB gene in pOT2:chlor vector</SPAN></A></LI><LI class="toclevel-4 tocsection-33"><A href="#Transformation_of_cells_with_gene_in_plasmid"><SPAN class="tocnumber">1.3.5.1</SPAN><SPAN class="toctext">Transformation of cells with gene in plasmid</SPAN></A></LI><LI class="toclevel-3 tocsection-34"><A href="#Coloni_PCR_of_flhD.2FC_in_pSB1C3"><SPAN class="tocnumber">1.3.6</SPAN><SPAN class="toctext">Coloni PCR of flhD/C in pSB1C3</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Group:_Flagella"> Group: Flagella </SPAN></H2><H3><SPAN class="mw-headline" id="Annealing_of_the_two_mutated_strands_of_FlhDC_.28FlhDCmut.29"> Annealing of the two mutated strands of FlhDC (FlhDCmut) </SPAN></H3><P><I>Experiment done by:</I> Sheila <I>Date:</I> July 19th <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#CP1.1" class="external text" rel="nofollow">CP1.1</A><I>Method:</I> PCR of the two mutated strands of the FlhDC operon<I>Notes:</I> To samples were run at two different temperatures: 56,1˚C and 64,5˚C respectively. Polymerase used: Pfu Primers used: None, as the two strands are supposed to anneal to each other<TABLE style="text-align: left; width: 258px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top; width: 180px;">Pfu buffer</TD><TD style="vertical-align: top; width: 60px;">  5,0uL</TD></TR><TR><TD style="vertical-align: top; width: 180px;">dNTP</TD><TD style="vertical-align: top; width: 60px;">  1,5uL</TD></TR><TR><TD style="vertical-align: top; width: 180px;">FlhDC mutated long
strand</TD><TD style="vertical-align: top; width: 60px;">  1,5uL</TD></TR><TR><TD style="vertical-align: top; width: 180px;">FlhDC mutated
short strand</TD><TD style="vertical-align: top; width: 60px;">  1,5uL</TD></TR><TR><TD style="vertical-align: top; width: 180px;">H2O</TD><TD style="vertical-align: top; width: 60px;">41,0uL</TD></TR><TR><TD style="vertical-align: top; width: 180px;">Pfu</TD><TD style="vertical-align: top; width: 60px;">  1,0uL</TD></TR></TBODY></TABLE>

The samples were run in the following PCR program:<TABLE style="text-align: left; width: 389px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top; width: 108px;">Program</TD><TD style="vertical-align: top; width: 91px;">1st sample</TD><TD style="vertical-align: top; width: 89px;">2nd sample</TD><TD style="vertical-align: top; width: 67px;">Time</TD></TR><TR><TD style="vertical-align: top; width: 108px;">1: Start</TD><TD style="vertical-align: top; width: 91px;">   94˚C</TD><TD style="vertical-align: top; width: 89px;">   94˚C</TD><TD style="vertical-align: top; width: 67px;">    
3min</TD></TR><TR><TD style="vertical-align: top; width: 108px;">2: Denaturing</TD><TD style="vertical-align: top; width: 91px;">   94˚C</TD><TD style="vertical-align: top; width: 89px;">   94˚C</TD><TD style="vertical-align: top; width: 67px;">    
2min</TD></TR><TR><TD style="vertical-align: top; width: 108px;">3: Annealing</TD><TD style="vertical-align: top; width: 91px;">   56,1˚C</TD><TD style="vertical-align: top; width: 89px;">   64,5˚C</TD><TD style="vertical-align: top; width: 67px;">   30sec</TD></TR><TR><TD style="vertical-align: top; width: 108px;">4: Elongation</TD><TD style="vertical-align: top; width: 91px;">   72˚C</TD><TD style="vertical-align: top; width: 89px;">   72˚C</TD><TD style="vertical-align: top; width: 67px;">    
2min</TD></TR><TR><TD style="vertical-align: top; width: 108px;">5:</TD><TD style="vertical-align: top; width: 91px;">  GOTO</TD><TD style="vertical-align: top; width: 89px;">  Repeat</TD><TD style="vertical-align: top; width: 67px;">    
X29</TD></TR><TR><TD style="vertical-align: top; width: 108px;">6: End</TD><TD style="vertical-align: top; width: 91px;">   72˚C</TD><TD style="vertical-align: top; width: 89px;">   72˚C</TD><TD style="vertical-align: top; width: 67px;">    
2min</TD></TR><TR><TD style="vertical-align: top; width: 108px;">7: Hold</TD><TD style="vertical-align: top; width: 91px;">    
4˚C</TD><TD style="vertical-align: top; width: 89px;">    
4˚C</TD></TR></TBODY></TABLE></P><P><I>Results:</I> A gel, containing the PCR product was run, using <A href="http://www.fermentas.de/product_info.php?info=p1110" class="external text" rel="nofollow">GeneRuler(TM) DNA Ladder Mix, ready-to-use (#SM0333)</A> loaded in lane one, lane 2 contains sample run at 56,1 degrees celcius and in lane 3 sample run at 64,5 degrees celcius is found.
We can se very faint bands in lane 2 and 3, at around 1000bp, which is the right size. The band in lane 3 is a little more visible than lane 2. Because we haven't used primers in this PCR, the pieces will only have been extended once and we need to ad flhDC fw and rv in order to obtain more product. 
--<A href="/wiki/index.php?title=User:Sheila&amp;action=edit&amp;redlink=1" class="new" title="User:Sheila (page does not exist)">Sheila</A> 09:10, 22 July 2010 (UTC)
</P><H3><SPAN class="mw-headline" id="Amplification_of_FlhDCmut"> Amplification of FlhDCmut </SPAN></H3><P><I>Experiment done by:</I> Sheila <I>Date:</I> July 19th <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#CP1.1" class="external text" rel="nofollow">CP1.1</A><I>Method:</I> The experiment was done in two rounds. First, the annealing product, made by Pernille on July 19th was amplified. Then we amplified Pernilles amplification simultaneously as we amplified <A href="https://2010.igem.org/Team:SDU-Denmark/labnotes2#Annealing_of_the_two_mutated_strands_of_FlhDC_.28FlhDCmut.29" class="external text" rel="nofollow">Sheilas previous annealing</A>. Primers used: FlhDC fw and FlhDC rev  Polymerase used: Pfu<I>Notes:</I><I>Results:</I></P><H3><SPAN class="mw-headline" id="Gel_extraction_of_FlhDCmut"> Gel extraction of FlhDCmut </SPAN></H3><P><I>Experiment done by:</I> Sheila <I>Date:</I> July 20th <I>Protocol:</I><I>Method:</I><I>Notes:</I><I>Results:</I></P><H3><SPAN class="mw-headline" id="Restriction_digest_of_FlhDCmut_and_pSB1C3_X2"> Restriction digest of FlhDCmut and pSB1C3 X2 </SPAN></H3><P><I>Experiment done by:</I> Sheila <I>Date:</I> July 20th <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#RD1.1" class="external text" rel="nofollow">RD1.1</A><I>Method:</I> 'Premixes:'<TABLE style="text-align: left; width: 256px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="width: 116px;"><SPAN style="font-weight: bold;">FlhDCmut</SPAN></TD><TD style="text-align: center; font-weight: bold; width: 57px;">X1</TD><TD style="text-align: center; font-weight: bold; width: 57px;">X3</TD></TR><TR><TD style="width: 116px;">H<SMALL><SMALL>2<BIG><BIG>O</BIG></BIG></SMALL></SMALL></TD><TD style="text-align: right; width: 57px;">12<SPAN style="font-size: 11pt; line-height: 115%; font-family: &quot;Calibri&quot;,&quot;sans-serif&quot;;" lang="EN-US">µL</SPAN></TD><TD style="text-align: right; width: 57px;">3<SPAN style="font-size: 11pt; line-height: 115%; font-family: &quot;Calibri&quot;,&quot;sans-serif&quot;;" lang="EN-US">6µL</SPAN></TD></TR><TR><TD style="width: 116px;">FD Green Buffer</TD><TD style="text-align: right; width: 57px;">2<SPAN style="font-size: 11pt; line-height: 115%; font-family: &quot;Calibri&quot;,&quot;sans-serif&quot;;" lang="EN-US">µL</SPAN></TD><TD style="text-align: right; width: 57px;">6<SPAN style="font-size: 11pt; line-height: 115%; font-family: &quot;Calibri&quot;,&quot;sans-serif&quot;;" lang="EN-US">µL</SPAN></TD></TR><TR><TD style="width: 116px;">Enzyme A: EcoRI</TD><TD style="text-align: right; width: 57px;">1<SPAN style="font-size: 11pt; line-height: 115%; font-family: &quot;Calibri&quot;,&quot;sans-serif&quot;;" lang="EN-US">µL</SPAN></TD><TD style="text-align: right; width: 57px;">3<SPAN style="font-size: 11pt; line-height: 115%; font-family: &quot;Calibri&quot;,&quot;sans-serif&quot;;" lang="EN-US">µL</SPAN></TD></TR><TR><TD style="width: 116px;">Enzyme B: S</TD><TD style="text-align: right; width: 57px;">1<SPAN style="font-size: 11pt; line-height: 115%; font-family: &quot;Calibri&quot;,&quot;sans-serif&quot;;" lang="EN-US">µL</SPAN></TD><TD style="text-align: right; width: 57px;">3<SPAN style="font-size: 11pt; line-height: 115%; font-family: &quot;Calibri&quot;,&quot;sans-serif&quot;;" lang="EN-US">µL</SPAN></TD></TR></TBODY></TABLE><TABLE style="text-align: left; width: 254px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="font-weight: bold; width: 116px;"><SPAN style="font-weight: bold;">pSB1C3</SPAN></TD><TD style="font-weight: bold; text-align: center; width: 56px;"><SPAN style="font-weight: bold;">X1</SPAN></TD><TD style="font-weight: bold; text-align: center; width: 56px;">X3</TD></TR><TR><TD style="width: 116px;">H<SMALL><SMALL>2<BIG><BIG>O</BIG></BIG></SMALL></SMALL></TD><TD style="width: 56px; text-align: right;">12<SPAN style="font-size: 11pt; line-height: 115%; font-family: &quot;Calibri&quot;,&quot;sans-serif&quot;;" lang="EN-US">µL</SPAN></TD><TD style="width: 56px; text-align: right;">36<SPAN style="font-size: 11pt; line-height: 115%; font-family: &quot;Calibri&quot;,&quot;sans-serif&quot;;" lang="EN-US">µL</SPAN></TD></TR><TR><TD style="width: 116px;">FD Green Buffer</TD><TD style="width: 56px; text-align: right;">2<SPAN style="font-size: 11pt; line-height: 115%; font-family: &quot;Calibri&quot;,&quot;sans-serif&quot;;" lang="EN-US">µL</SPAN></TD><TD style="width: 56px; text-align: right;">6<SPAN style="font-size: 11pt; line-height: 115%; font-family: &quot;Calibri&quot;,&quot;sans-serif&quot;;" lang="EN-US">µL</SPAN></TD></TR><TR><TD style="width: 116px;">Enzyme A: EcoRI</TD><TD style="width: 56px; text-align: right;">1<SPAN style="font-size: 11pt; line-height: 115%; font-family: &quot;Calibri&quot;,&quot;sans-serif&quot;;" lang="EN-US">µL</SPAN></TD><TD style="width: 56px; text-align: right;">3<SPAN style="font-size: 11pt; line-height: 115%; font-family: &quot;Calibri&quot;,&quot;sans-serif&quot;;" lang="EN-US">µL</SPAN></TD></TR><TR><TD style="width: 116px;">Enzyme B: X</TD><TD style="width: 56px; text-align: right;">1<SPAN style="font-size: 11pt; line-height: 115%; font-family: &quot;Calibri&quot;,&quot;sans-serif&quot;;" lang="EN-US">µL</SPAN></TD><TD style="width: 56px; text-align: right;">3<SPAN style="font-size: 11pt; line-height: 115%; font-family: &quot;Calibri&quot;,&quot;sans-serif&quot;;" lang="EN-US">µL</SPAN></TD></TR></TBODY></TABLE>

The digested samples were then run on a 1,5% agarose gel for approximately 20 minutes<I>Results:</I> Because it failed the first time, this experiment was repeated, with the same results (see notes section)<B>The first gel</B><B>The second gel</B>
The gels do not show any sign of a successful digest. Because the restiction enzymes only cut in the prefix and suffix of the FlhDCmut operon, it is not surprising that we do not see any bands of approximately 10bp length. However, the digestion should be very clearly visible in the plasmid backbone, as we should be cutting out RFP, which is approximately 1000bp long. The plasmid do not appear to have been digested by the restriction enzymes. 2 bands are visible, but they are both much longer than expected, in fact they appear around the length of the complete plasmid. So this experiment appears to have been unsuccessful.<I>Notes:</I> First troubleshooting session led to a repeat of the experiment, as we found that human error must have been the reason for the failed results, because it was the first time i performed the experiment. However, when the repeat of the restriction digest yielded the same results, we took a closer look and found that the combination of restriction enzymes was wrong. They were the combination we will use when we begin the assembly... In this case we want to insert the biobrick (FlhDCmut) into the plasmid backbone (pSB1C3, in which case we need to use the same restriction enzymes on both the plasmid and the biobrick. 
--<A href="/wiki/index.php?title=User:Sheila&amp;action=edit&amp;redlink=1" class="new" title="User:Sheila (page does not exist)">Sheila</A> 22:25, 1 August 2010 (UTC)
</P><H3><SPAN class="mw-headline" id="Gel_extraction_of_FlhDCmut_and_pSB1C3_X2"> Gel extraction of FlhDCmut and pSB1C3 X2 </SPAN></H3><P><I>Experiment done by:</I> Sheila <I>Date:</I> July 20th and July 21st<I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#DE1.3" class="external text" rel="nofollow">DE1.3</A><I>Method:</I> The samples were run on a 1,5% agarose gel.<I>Notes:</I> No restriction were seen in the pSB1C3 lane, which lead to the conclusion that the restriction digest failed (<A href="https://2010.igem.org/Team:SDU-Denmark/labnotes2#Restriction_digest_of_FlhDCmut_and_pSB1C3_X2" class="external text" rel="nofollow">see above</A>. The FlhDC was extracted from the gel in order to amplify it with PCR<I>Results:</I><B>The first gel</B><B>The second gel</B>
--<A href="/wiki/index.php?title=User:Sheila&amp;action=edit&amp;redlink=1" class="new" title="User:Sheila (page does not exist)">Sheila</A> 14:42, 2 August 2010 (UTC)
</P><H3><SPAN class="mw-headline" id="Amplification_of_FlhDCmut_2"> Amplification of FlhDCmut </SPAN></H3><P><I>Experiment done by:</I> Sheila <I>Date:</I> July 20th-21st <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#CP1.1" class="external text" rel="nofollow">CP1.1</A><I>Method:</I><I>Notes:</I><I>Results:</I></P><H3><SPAN class="mw-headline" id="Gel_extraction_of_FlhDCmut_2"> Gel extraction of FlhDCmut </SPAN></H3><P><I>Experiment done by:</I> Sheila <I>Date:</I> July 21st <I>Protocol:</I><I>Method:</I><I>Notes:</I><I>Results:</I></P><H3><SPAN class="mw-headline" id="Gel_extraction_of_flhD.2FC_.28native_coding_sequence.29"> Gel extraction of flhD/C (native coding sequence) </SPAN></H3><P><I>Start date:</I> 19/07    <I>End date:</I> 19/07<I>Methods:</I>  Gel extraction, nanodrop <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#DE1.1" class="external text" rel="nofollow">DE1.1</A><I>Experiment done by:</I>Maria <I>Notes:</I>
70uL of flhD/C PCR product from <A href="https://2010.igem.org/Team:SDU-Denmark/labnotes#Amplification_of_FlhDC_with_Pfu_and_mutation_of_stopcodon.2C_located_in_FlhD" class="external text" rel="nofollow">Amplification of flhD/C</A> was loaded onto a 1.5 agarose gel and extracted according to protocol. 
DNA was eluted in 20uL elution buffer.<I>Results:</I>
Nanodrop:
</P><TABLE style="text-align: left; width: 100px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top;">Sample ID</TD><TD style="vertical-align: top;">ng/uL</TD><TD style="vertical-align: top;">260/280</TD><TD style="vertical-align: top;">260/230</TD></TR><TR><TD style="vertical-align: top;">flhDC 1</TD><TD style="vertical-align: top;">19.82</TD><TD style="vertical-align: top;">2.43</TD><TD style="vertical-align: top;">0.08</TD></TR><TR><TD style="vertical-align: top;">flhDC 2</TD><TD style="vertical-align: top;">25.24</TD><TD style="vertical-align: top;">2.22</TD><TD style="vertical-align: top;">0.12</TD></TR></TBODY></TABLE><P><I>Analysis:</I>
Nanodrop measurements indicated a possible contamination. However the DNA was pooled and used for <A href="https://2010.igem.org/Team:SDU-Denmark/labnotes2#Digestion_of_flhD.2FC_and_pSB1A2_with_EcoRI_Spe" class="external text" rel="nofollow">Digestion</A>
--<A href="/wiki/index.php?title=User:Tipi&amp;action=edit&amp;redlink=1" class="new" title="User:Tipi (page does not exist)">Tipi</A> 17:22, 19 July 2010 (UTC)
</P><H3><SPAN class="mw-headline" id="Digestion_of_flhD.2FC_.28native_coding_sequence.29_and_pSB1A2_with_EcoRI_SpeI"> Digestion of flhD/C (native coding sequence) and pSB1A2 with EcoRI SpeI </SPAN></H3><P><I>Start date:</I> 19/07    <I>End date:</I> 19/07<I>Methods:</I>  Digestion, Gel electrophoresis <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#Restriction_digest" class="external text" rel="nofollow">RD1.1</A><I>Experiment done by:</I>Maria <I>Notes:</I>
purified pSB1C3 (tube 18 blue) and flhD/C from <A href="https://2010.igem.org/Team:SDU-Denmark/labnotes2#Gel_extraction_of_flhD.2FC" class="external text" rel="nofollow">Gel extraction</A> was digested with EcoRI and SpeI.
Restriction mixture: </P><TABLE style="text-align: left; width: 100px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top;">flhD/C</TD><TD style="vertical-align: top;">pSB1C3</TD></TR><TR><TD style="vertical-align: top;">H2O</TD><TD style="vertical-align: top;">30uL</TD><TD style="vertical-align: top;">24uL</TD></TR><TR><TD style="vertical-align: top;">EcoRI</TD><TD style="vertical-align: top;">4uL</TD><TD style="vertical-align: top;">2uL</TD></TR><TR><TD style="vertical-align: top;">SpeI</TD><TD style="vertical-align: top;">4uL</TD><TD style="vertical-align: top;">2uL</TD></TR><TR><TD style="vertical-align: top;">FD green buffer</TD><TD style="vertical-align: top;">8uL</TD><TD style="vertical-align: top;">4uL</TD></TR><TR><TD style="vertical-align: top;">DNA</TD><TD style="vertical-align: top;">38uL</TD><TD style="vertical-align: top;">10uL</TD></TR><TR><TD style="vertical-align: top;">total vol.</TD><TD style="vertical-align: top;">84</TD><TD style="vertical-align: top;">42</TD></TR></TBODY></TABLE><P>
samples were loaded onto a 1.5 agarose gel. Gene ruler red were used as marker.<I>Results:</I>
Gel electrophoresis:<I>Analysis:</I>
In lane 2 containing pSB1C3 2 bands are detected indicating a succesful digestion of the plasmid (the band at 1000 bp corresponds to J04450). A succesful digestion of the flhD/C cannot be concluded from the gel. However both bands was excised and extracted from gel (<A href="https://2010.igem.org/Team:SDU-Denmark/labnotes2#Gel_extraction_of_digested_flhD.2FC_and_pSB1C3" class="external text" rel="nofollow">Gel extraction</A>) 
--<A href="/wiki/index.php?title=User:Tipi&amp;action=edit&amp;redlink=1" class="new" title="User:Tipi (page does not exist)">Tipi</A> 17:42, 19 July 2010 (UTC)</P><H3><SPAN class="mw-headline" id="Gel_extraction_of_digested_flhD.2FC_.28native_coding_sequence.29_and_pSB1C3"> Gel extraction of digested flhD/C (native coding sequence) and pSB1C3  </SPAN></H3><P><I>Start date:</I> 19/07    <I>End date:</I> 19/07<I>Methods:</I>  Gel extraction, nanodrop <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#DE1.2" class="external text" rel="nofollow">DE1.2</A><I>Experiment done by:</I>Maria <I>Notes:</I>
Digested flhD/C and pSB1C3 from <A href="https://2010.igem.org/Team:SDU-Denmark/labnotes2#Digestion_of_flhD.2FC_and_pSB1A2_with_EcoRI_Spe" class="external text" rel="nofollow">Digestion</A> were extracted from gel according to protocol. 
DNA was eluted in 20uL H20. <I>Results:</I>
Nanodrop:</P><TABLE style="text-align: left; width: 100px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top;">Sample ID</TD><TD style="vertical-align: top;">ng/uL</TD><TD style="vertical-align: top;">260/280</TD><TD style="vertical-align: top;">260/230</TD></TR><TR><TD style="vertical-align: top;">flhD/C </TD><TD style="vertical-align: top;">18.18</TD><TD style="vertical-align: top;">4.59</TD><TD style="vertical-align: top;">0.18</TD></TR><TR><TD style="vertical-align: top;">pSB1C3</TD><TD style="vertical-align: top;">14.30</TD><TD style="vertical-align: top;">1.89</TD><TD style="vertical-align: top;">0.04</TD></TR></TBODY></TABLE><P><I>Analysis:</I>
nanodrop measurements indicated contamination. However both samples were used for <A href="https://2010.igem.org/Team:SDU-Denmark/labnotes2#Ligation_of_flhD.2FC_and_pSB1C3" class="external text" rel="nofollow">Ligation</A>.
--<A href="/wiki/index.php?title=User:Tipi&amp;action=edit&amp;redlink=1" class="new" title="User:Tipi (page does not exist)">Tipi</A> 17:57, 19 July 2010 (UTC)</P><H3><SPAN class="mw-headline" id="Ligation_of_flhD.2FC_.28native_coding_sequence.29_and_pSB1C3"> Ligation of flhD/C (native coding sequence) and pSB1C3 </SPAN></H3><P><I>Start date:</I> 19/07    <I>End date:</I> 20/07<I>Methods:</I>  Ligation <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#LG1.2" class="external text" rel="nofollow">LG1.2</A><I>Experiment done by:</I>Maria <I>Notes:</I>
3 ligation reactions was prepared. </P><TABLE style="text-align: left; width: 100px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top;">LG1</TD><TD style="vertical-align: top;">LG2</TD><TD style="vertical-align: top;">LG3</TD></TR><TR><TD style="vertical-align: top;">10x T4 ligase buffer</TD><TD style="vertical-align: top;">2uL</TD><TD style="vertical-align: top;">2uL</TD><TD style="vertical-align: top;">2uL</TD></TR><TR><TD style="vertical-align: top;">flhD/C</TD><TD style="vertical-align: top;">5uL</TD><TD style="vertical-align: top;">5uL</TD><TD style="vertical-align: top;">5uL</TD></TR><TR><TD style="vertical-align: top;">pSB1C3</TD><TD style="vertical-align: top;">1uL</TD><TD style="vertical-align: top;">2.8uL</TD><TD style="vertical-align: top;">5uL</TD></TR><TR><TD style="vertical-align: top;">H20</TD><TD style="vertical-align: top;">11uL</TD><TD style="vertical-align: top;">9.2uL</TD><TD style="vertical-align: top;">7uL</TD></TR><TR><TD style="vertical-align: top;">T4 ligase</TD><TD style="vertical-align: top;">1uL</TD><TD style="vertical-align: top;">1uL</TD><TD style="vertical-align: top;">1uL</TD></TR></TBODY></TABLE><P>Ligation mixtures were not run on gel but were directly used for <A href="https://2010.igem.org/Team:SDU-Denmark/labnotes2#Transformation_of_flhD.2FC_in_pSB1C3_and_test_plasmid_in_Top_10_E.Coli" class="external text" rel="nofollow">transformation</A>
surplus ligation mixture was stored at -20 degrees as L1, L2 and L3. 
--<A href="/wiki/index.php?title=User:Tipi&amp;action=edit&amp;redlink=1" class="new" title="User:Tipi (page does not exist)">Tipi</A> 18:15, 19 July 2010 (UTC)</P><H3><SPAN class="mw-headline" id="Coloni_PCR_of_flhD.2FC_.28native_coding_sequence.29_in_pSB1C3"> Coloni PCR of flhD/C (native coding sequence) in pSB1C3 </SPAN></H3><P>Start date: 21/07    End date: 21/07<I>Methods:</I>  Coloni PCR and gel electrophoresis <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#CP1.2" class="external text" rel="nofollow">CP1.2</A><I>Experiment done by:</I> Maria and LC<I>Notes:</I>
Coloni PCR was made on flhD/C in pSB1C3 from <A href="https://2010.igem.org/Team:SDU-Denmark/labnotes2#Transformation_of_flhD.2FC_in_pSB1C3_and_test_plasmid_in_Top_10_E.Coli" class="external text" rel="nofollow">Transformation</A>.
10 colonies from plates plated with cells transformed with L2 and L3 ligation mixtures (see <A href="https://2010.igem.org/Team:SDU-Denmark/labnotes2#Ligation_of_flhD.2FC_.28native_coding_sequence.29_and_pSB1C3" class="external text" rel="nofollow">Ligation</A>). 
Coloni 1-5 is taken from L2 plates
coloni 6-10 is taken from L3 plates 
PCR was made with only half the amount of premix.
Premix for 12 PCR reactions: </P><TABLE style="text-align: left; width: 100px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top;">10xtaq buffer</TD><TD style="vertical-align: top;">30uL</TD></TR><TR><TD style="vertical-align: top;">MgCl2</TD><TD style="vertical-align: top;">12uL</TD></TR><TR><TD style="vertical-align: top;">VF2</TD><TD style="vertical-align: top;">12uL</TD></TR><TR><TD style="vertical-align: top;">VR</TD><TD style="vertical-align: top;">12uL</TD></TR><TR><TD style="vertical-align: top;">dNTP</TD><TD style="vertical-align: top;">6uL</TD></TR><TR><TD style="vertical-align: top;">H2O</TD><TD style="vertical-align: top;">42uL</TD></TR><TR><TD style="vertical-align: top;">Total vol.</TD><TD style="vertical-align: top;">114uL</TD></TR></TBODY></TABLE><P>0.5uL Taq polymerase from ampliqon was used.
Taq PCR program:</P><TABLE style="text-align: left; height: 260px; width: 225px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top; width: 102px;">1:Start</TD><TD style="vertical-align: top; width: 52px;">94C</TD><TD style="vertical-align: top; width: 45px;">2 min</TD></TR><TR><TD style="vertical-align: top; width: 102px;">2: Denaturing</TD><TD style="vertical-align: top; width: 52px;">94C</TD><TD style="vertical-align: top; width: 45px;">1 min</TD></TR><TR><TD style="vertical-align: top; width: 102px;">3: Annealing</TD><TD style="vertical-align: top; width: 52px;">55C</TD><TD style="vertical-align: top; width: 45px;">1 min</TD></TR><TR><TD style="vertical-align: top; width: 102px;">4: Elongation</TD><TD style="vertical-align: top; width: 52px;">72C</TD><TD style="vertical-align: top; width: 45px;">1 min 30 s</TD></TR><TR><TD style="vertical-align: top; width: 102px;">5:</TD><TD style="vertical-align: top; width: 52px;">GO TO</TD><TD style="vertical-align: top; width: 45px;">rep. 29x</TD></TR><TR><TD style="vertical-align: top; width: 102px;">6: End </TD><TD style="vertical-align: top; width: 52px;">72C</TD><TD style="vertical-align: top; width: 45px;">3 min</TD></TR><TR><TD style="vertical-align: top; width: 102px;">7: Hold</TD><TD style="vertical-align: top; width: 52px;">4C</TD></TR></TBODY></TABLE><P>PCR product was loaded onto a 2% agarose gel. Gene ruler DNA ladder mix (red) was used as marker.
VF2 - coding sequence (in pSB1C3) : 140bp 
Coding sequence - VR (in pSB1C3)  : 176bp 
flhD/C                            : 932bp 
VF2 - VR for pSB1C3 w. flhD/C     :1248bp 
VF2 - VR for pSB1C3 without insertion : 316bp <I>Results:</I>
Gel electrophoresis: </P><P><I>Analysis:</I>
All lanes except 4, 9 and 10 showed a band at app. 1200bp. which might indicate plasmids where flhD/C has been inserted.
Lane 4 and 9 has a band at app. 1500bp which could indicate undigested pSB1C3 (VF2 - VR for pSB1C3 w. J04450 is 1385bp).
Streak plates of coloni 1, 2, 3, 5, 6, 7, 8 and 10 showed red colonies, indicating that these contained undigested pSB1C3. Streak plates of coloni 4 and 9 had only white colonies, suggesting an insertion has occured. To verify if flhD/C has been inserted coloni PCR using VF2 and flhD/C rev. as primers was carried out for colonies from streak plates of coloni 4 and 9 and of all white colonies from the transformation plates. (see coloni PCR)
--<A href="/wiki/index.php?title=User:Tipi&amp;action=edit&amp;redlink=1" class="new" title="User:Tipi (page does not exist)">Tipi</A> 11:46, 24 July 2010 (UTC) </P><H3><SPAN class="mw-headline" id="Coloni_PCR_of_flhD.2FC_.28native_coding_sequence.29_in_pSB1C3_2"> Coloni PCR of flhD/C (native coding sequence) in pSB1C3 </SPAN></H3><P>Start date: 22/07    End date: 22/07<I>Methods:</I>  Coloni PCR and gel electrophoresis <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#CP1.3" class="external text" rel="nofollow">CP1.3</A><I>Experiment done by:</I> Maria<I>Notes:</I>
Coloni PCR was made on flhD/C in pSB1C3 from <A href="https://2010.igem.org/Team:SDU-Denmark/labnotes2#Transformation_of_flhD.2FC_in_pSB1C3_and_test_plasmid_in_Top_10_E.Coli" class="external text" rel="nofollow">Transformation</A>.
All remaining white colonies from plates plated with cells transformed with L2 and L3 ligation mixtures (see <A href="https://2010.igem.org/Team:SDU-Denmark/labnotes2#Ligation_of_flhD.2FC_.28native_coding_sequence.29_and_pSB1C3" class="external text" rel="nofollow">Ligation</A>) was selected. 
coloni 1-6 is taken from L2.3 plates
coloni 7-15 is taken from L3.2 plates 
coloni 16-27 is taken from L3.3 plates
coloni 28 is taken from streak plate of PCR no. 4
coloni 29 is taken from streak plate of PCR no. 9 (see <A href="https://2010.igem.org/Team:SDU-Denmark/labnotes2#Coloni_PCR_of_flhD.2FC_in_pSB1C3_2" class="external text" rel="nofollow">Coloni PCR</A>) 
Premix for 31 PCR reactions: </P><TABLE style="text-align: left; width: 100px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top;">10xtaq buffer</TD><TD style="vertical-align: top;">77.5uL</TD></TR><TR><TD style="vertical-align: top;">MgCl2</TD><TD style="vertical-align: top;">31uL</TD></TR><TR><TD style="vertical-align: top;">VF2</TD><TD style="vertical-align: top;">31uL</TD></TR><TR><TD style="vertical-align: top;">flhD/C rev.</TD><TD style="vertical-align: top;">31uL</TD></TR><TR><TD style="vertical-align: top;">dNTP</TD><TD style="vertical-align: top;">15.5uL</TD></TR><TR><TD style="vertical-align: top;">H2O</TD><TD style="vertical-align: top;">108.5uL</TD></TR><TR><TD style="vertical-align: top;">Total vol.</TD><TD style="vertical-align: top;">294.5uL</TD></TR></TBODY></TABLE><P>0.5uL Taq polymerase from fermentas was used.
Taq PCR program:</P><TABLE style="text-align: left; height: 260px; width: 225px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top; width: 102px;">1:Start</TD><TD style="vertical-align: top; width: 52px;">94C</TD><TD style="vertical-align: top; width: 45px;">2 min</TD></TR><TR><TD style="vertical-align: top; width: 102px;">2: Denaturing</TD><TD style="vertical-align: top; width: 52px;">94C</TD><TD style="vertical-align: top; width: 45px;">1 min</TD></TR><TR><TD style="vertical-align: top; width: 102px;">3: Annealing</TD><TD style="vertical-align: top; width: 52px;">55C</TD><TD style="vertical-align: top; width: 45px;">1 min</TD></TR><TR><TD style="vertical-align: top; width: 102px;">4: Elongation</TD><TD style="vertical-align: top; width: 52px;">72C</TD><TD style="vertical-align: top; width: 45px;">1 min 30 s</TD></TR><TR><TD style="vertical-align: top; width: 102px;">5:</TD><TD style="vertical-align: top; width: 52px;">GO TO</TD><TD style="vertical-align: top; width: 45px;">rep. 29x</TD></TR><TR><TD style="vertical-align: top; width: 102px;">6: End </TD><TD style="vertical-align: top; width: 52px;">72C</TD><TD style="vertical-align: top; width: 45px;">3 min</TD></TR><TR><TD style="vertical-align: top; width: 102px;">7: Hold</TD><TD style="vertical-align: top; width: 52px;">4C</TD></TR></TBODY></TABLE><P>PCR product was loaded onto a 2% agarose gel. Gene ruler DNA ladder mix (red) was used as marker.</P><P><I>Results:</I>
Gel electrophoresis: </P><P><I>Analysis:</I>
None of the samples contained amplified DNA.
This indicates either that the flhD/C gene has not been inserted to the pSB1C3 backbone or an insertion has been made but the flhD/C primer has not been able to anneal to the template.
To verify that a ligation has occurred, PCR product 4 and 9 from <A href="https://2010.igem.org/Team:SDU-Denmark/labnotes2#Coloni_PCR_of_flhD.2FC_in_pSB1C3" class="external text" rel="nofollow">Coloni PCR</A> was digested with EcoRI and SpeI.
--<A href="/wiki/index.php?title=User:Tipi&amp;action=edit&amp;redlink=1" class="new" title="User:Tipi (page does not exist)">Tipi</A> 13:53, 24 July 2010 (UTC)</P><H3><SPAN class="mw-headline" id="Digestion_of_flhD.2FC_PCR_product_with_EcoRI_and_SpeI"> Digestion of flhD/C PCR product with EcoRI and SpeI </SPAN></H3><P>Start date: 24/07    End date: 24/07<I>Methods:</I>  Restriction digest and gel electrophoresis <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#RD1.1" class="external text" rel="nofollow">RD1.1</A><I>Experiment done by:</I> Maria<I>Notes:</I>
To verify if a ligation has occurred Taq PCR product of coloni 4 and 9 (see <A href="https://2010.igem.org/Team:SDU-Denmark/labnotes2#Coloni_PCR_of_flhD.2FC_in_pSB1C3" class="external text" rel="nofollow">Coloni PCR</A>) was digested with the same restiction enzymes used for the digestion of flhD/C DNA and pSB1C3 prior to ligation. If an ligation has occurred, then the restriction sites of EcoRI and SpeI should be intact, and two fragments of 122bp and 170bp should be observed on the gel, indicating VF2-E site and S site-VR respectively.
RFP (PCR poduct of coloni no.8) was used as controle.
2.5uL of undigested PCR product of coloni 4, 8 and 9 were loaded onto the gel as well.
Samples were loaded onto a 2% agarose gel. Hyper Ladder II was used as marker.
Loading scheme:</P><TABLE style="text-align: left; width: 100%;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top;">Lane</TD><TD style="vertical-align: top;">1</TD><TD style="vertical-align: top;">2</TD><TD style="vertical-align: top;">3</TD><TD style="vertical-align: top;">4</TD><TD style="vertical-align: top;">5</TD><TD style="vertical-align: top;">6</TD></TR><TR><TD style="vertical-align: top;">digested no. 4</TD><TD style="vertical-align: top;">undigested no. 4</TD><TD style="vertical-align: top;">digested no. 9</TD><TD style="vertical-align: top;">undigested no. 9</TD><TD style="vertical-align: top;">digested no. 8</TD><TD style="vertical-align: top;">undigested no. 8</TD></TR></TBODY></TABLE><P><I>Results:</I>
Gel electrophoresis:<I>Analysis:</I>
The 2 bands at app. 100bp and 200bp respectively indicates that the E and S restiction sites have been preserved, suggesting that a ligation has occured.
An ON culture of coloni 4 and 9 is made to use for mini-prep.
--<A href="/wiki/index.php?title=User:Tipi&amp;action=edit&amp;redlink=1" class="new" title="User:Tipi (page does not exist)">Tipi</A> 14:37, 24 July 2010 (UTC)</P><H3><SPAN class="mw-headline" id="Miniprep_of_ligation_coloni_.234_and_.239"> Miniprep of ligation coloni #4 and #9 </SPAN></H3><P>Start date: 25/07    End date: 25/07<I>Methods:</I>  Mini prep and nanodrop <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#MP1.1" class="external text" rel="nofollow">MP1.1</A><I>Experiment done by:</I> Maria<I>Notes:</I>
The samples are loaded onto a 1.5% agarose gel. Gene ruler DNA ladder mix (red) is used as marker.
Loading sheme:</P><TABLE style="text-align: left; width: 300px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top;">Lanes</TD><TD style="vertical-align: top;">1</TD><TD style="vertical-align: top;">2</TD><TD style="vertical-align: top;">3</TD><TD style="vertical-align: top;">4</TD></TR><TR><TD style="vertical-align: top;"># 4.1</TD><TD style="vertical-align: top;"># 4.2</TD><TD style="vertical-align: top;"># 9.1</TD><TD style="vertical-align: top;"># 9.2</TD></TR></TBODY></TABLE><P><I>Results:</I>
Gel electrophoresis:
Nanodrop:</P><TABLE style="text-align: left; width: 100px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top;">Sample ID</TD><TD style="vertical-align: top;">ng/uL</TD><TD style="vertical-align: top;">260/280</TD><TD style="vertical-align: top;">260/230</TD></TR><TR><TD style="vertical-align: top;">miniprep coloni #4.1</TD><TD style="vertical-align: top;">115.68</TD><TD style="vertical-align: top;">1.91</TD><TD style="vertical-align: top;">2.17</TD></TR><TR><TD style="vertical-align: top;">miniprep coloni #4.2</TD><TD style="vertical-align: top;">115.74</TD><TD style="vertical-align: top;">1.91</TD><TD style="vertical-align: top;">2.13</TD></TR><TR><TD style="vertical-align: top;">miniprep coloni #9.1</TD><TD style="vertical-align: top;">131.32</TD><TD style="vertical-align: top;">1.92</TD><TD style="vertical-align: top;">2.23</TD></TR><TR><TD style="vertical-align: top;">miniprep coloni #9.2</TD><TD style="vertical-align: top;">119</TD><TD style="vertical-align: top;">1.93</TD><TD style="vertical-align: top;">2.16</TD></TR><TR><TD style="vertical-align: top;">miniprep coloni #4 pooled</TD><TD style="vertical-align: top;">116.51</TD><TD style="vertical-align: top;">1.92</TD><TD style="vertical-align: top;">2.15</TD></TR><TR><TD style="vertical-align: top;">miniprep coloni #9 pooled</TD><TD style="vertical-align: top;">123.94</TD><TD style="vertical-align: top;">1.93</TD><TD style="vertical-align: top;">2.17</TD></TR></TBODY></TABLE><P><I>Analysis:</I>
We have purified the plasmids. The concentrations however are too low for sequencing, so the samples are dried down to reach a conc. of 200ng/uL 
--<A href="/wiki/index.php?title=User:Tipi&amp;action=edit&amp;redlink=1" class="new" title="User:Tipi (page does not exist)">Tipi</A> 08:44, 28 July 2010 (UTC)</P><H3><SPAN class="mw-headline" id="Digestion_of_flhD.2FC_mut_with_PstI"> Digestion of flhD/C mut with PstI </SPAN></H3><H4><SPAN class="mw-headline" id="Experiment_no.1">Experiment no.1</SPAN></H4><P>Start date: 21/07    End date: 21/07<I>Methods:</I>  Restriction digest and gel electrophoresis <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#RD1.1" class="external text" rel="nofollow">RD1.1</A><I>Experiment done by:</I> Maria<I>Notes:</I>
To verify if the silent mutation has been made PCR product of flhD/C mut (see [<A href="https://2010.igem.org/Team:SDU-Denmark/labnotes2#Amplification_of_FlhDCmut" class="external text" rel="nofollow">flhD/C mut PCR</A>] was digested with restiction enzyme PstI.
If the mutation has been made the flhD/C fragment should not be digested with PstI.
PCR product of flhD/C (native coding sequence) was used as controle.
Undigested PCR product of flhD/C (native coding sequence) was loaded onto the gel as well.Samples were loaded onto a 2% agarose gel. Hyper Ladder II was used as marker.
Loading scheme:</P><TABLE style="text-align: left; width: 100%;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top;">Lane</TD><TD style="vertical-align: top;">1</TD><TD style="vertical-align: top;">2</TD><TD style="vertical-align: top;">3</TD></TR><TR><TD style="vertical-align: top;">Digested flhD/C (native coding sequence) </TD><TD style="vertical-align: top;">Digested flhD/C mut</TD><TD style="vertical-align: top;">Undigested flhD/C (native coding sequence)</TD></TR></TBODY></TABLE><P><I>Results:</I>
Gel electrophoresis:<I>Analysis:</I>
The intensity of the bands are too low to conclude anything. The experiment was remade using a gel with smaller wells.
--<A href="/wiki/index.php?title=User:Tipi&amp;action=edit&amp;redlink=1" class="new" title="User:Tipi (page does not exist)">Tipi</A> 14:37, 24 July 2010 (UTC)</P><H4><SPAN class="mw-headline" id="experiment_no.2">experiment no.2</SPAN></H4><P>Start date: 22/07    End date: 22/07<I>Methods:</I>  Restriction digest and gel electrophoresis <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#RD1.1" class="external text" rel="nofollow">RD1.1</A><I>Experiment done by:</I> Maria<I>Notes:</I>
Taq PCR product of flhD/C (native coding sequence) (no.5 green) was used as controle.
Undigested PCR product of flhD/C (native coding sequence) was loaded onto the gel as well.Samples were loaded onto a 2% agarose gel. Hyper Ladder II was used as marker.
Loading scheme:</P><TABLE style="text-align: left; width: 100%;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top;">Lane</TD><TD style="vertical-align: top;">1</TD><TD style="vertical-align: top;">2</TD><TD style="vertical-align: top;">3</TD></TR><TR><TD style="vertical-align: top;">Digested flhD/C (native coding sequence) </TD><TD style="vertical-align: top;">Digested flhD/C mut</TD><TD style="vertical-align: top;">Undigested flhD/C (native coding sequence)</TD></TR></TBODY></TABLE><P><I>Results:</I>
Gel electrophoresis:<I>Analysis:</I>
A small fragment is be observed for both the flhD/C mut and flhD/C (native coding sequence), so it cannot be fully concluded that the mutation has been made. Another digestion reaction must be carried out.
--<A href="/wiki/index.php?title=User:Tipi&amp;action=edit&amp;redlink=1" class="new" title="User:Tipi (page does not exist)">Tipi</A> 15:46, 24 July 2010 (UTC)
</P><H3><SPAN class="mw-headline" id="Purification_of_pSB1A2_containing_J13002">Purification of pSB1A2 containing J13002</SPAN></H3><P><I>Experiment done by:</I> Pernille <I>Date:</I> the 19 of July <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#Plasmid_miniprep_kit_.28Fermentas.29" class="external text" rel="nofollow">MP1.1</A> and <A href="https://2010.igem.org/Team:SDU-Denmark/protocols#CP1.1" class="external text" rel="nofollow">CP1.1</A><I>Method:</I>  Miniprep, Nano drop, gel electrophoresis and PCR <I>Notes:</I> The pSB1A2 plasmid are a high-copy-plasmid and therefore 5mL ON culture are used for purification of the plasmid. for the PCR the following PRC program are used:
1: 94°C 3min
2: 94°C 2min
3: 55°C 30 sec
4: 72°C 1min
5: x 29 rep. 
6: 72°C 5min
7: hold 4°C 
<I>Results:</I>After the purification the concentration of nucleic acid concentration was measured on the Nano Drop. There was made four purification tubes. Only tube 1 and 3 was used for further experiments because the other contained a very low concentration of nucleic acid. The concentration in tube 1 and 3 was 25,72ng/uL and 25,76ng/uL.
The purified DNA was loaded in a 1,5% agarose gel together with a 10.000bp long ladder (red).
PCR was run on the DNA using the VR and VF2 primers wich ensures the only the brick part are amplificated. The pfu polymerase was used but unfortunately i did not get any results since i did not add any MgSO4 to the premix.
--<A href="/wiki/index.php?title=User:Pernm07&amp;action=edit&amp;redlink=1" class="new" title="User:Pernm07 (page does not exist)">Pernm07</A> 08:48, 22 July 2010 (UTC)
</P><H3><SPAN class="mw-headline" id="Amplification_of_pSB1A2_w._J13002.2C_pSB1A2_w._B0015_and_pSB1A2_w._B0034">Amplification of pSB1A2 w. <A href="http://partsregistry.org/Part:BBa_J13002" class="external text" rel="nofollow">J13002</A>, pSB1A2 w. <A href="http://partsregistry.org/Part:BBa_B0015" class="external text" rel="nofollow">B0015</A> and pSB1A2 w. <A href="http://partsregistry.org/Part:BBa_B0034" class="external text" rel="nofollow">B0034</A></SPAN></H3><P><I>Experiment done by:</I> Pernille <I>Date:</I> The 20th of July <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#CP1.1" class="external text" rel="nofollow">CP1.1</A>and <A href="https://2010.igem.org/Team:SDU-Denmark/protocols#DNA_extraction_from_gel_.28fermentas.29" class="external text" rel="nofollow">DNA extraction</A><I>Method:</I> PCR and DNA extraction <I>Notes:</I> The purified DNA was amiplificated with PRC using the following program:
<TABLE style="text-align: left; width: 334px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top;">Temperature (Celcius)</TD><TD style="vertical-align: top; width: 99px;">Time (min)</TD><TD style="vertical-align: top; width: 56px;">Description</TD></TR><TR><TD style="vertical-align: top;">94</TD><TD style="vertical-align: top; width: 99px;">3</TD><TD style="vertical-align: top; width: 56px;">Initialization</TD></TR><TR><TD style="vertical-align: top;">94</TD><TD style="vertical-align: top; width: 99px;">2</TD><TD style="vertical-align: top; width: 56px;">Denaturation</TD></TR><TR><TD style="vertical-align: top;">55</TD><TD style="vertical-align: top; width: 99px;">½</TD><TD style="vertical-align: top; width: 56px;">Annealing</TD></TR><TR><TD style="vertical-align: top;">72</TD><TD style="vertical-align: top; width: 99px;">1</TD><TD style="vertical-align: top; width: 56px;">Elongation</TD></TR><TR><TD style="vertical-align: top;">-</TD><TD style="vertical-align: top; width: 99px;">x 29</TD><TD style="vertical-align: top; width: 56px;">GOTO step 2</TD></TR><TR><TD style="vertical-align: top;">72</TD><TD style="vertical-align: top; width: 99px;">5</TD><TD style="vertical-align: top; width: 56px;">Final Elongation</TD></TR><TR><TD style="vertical-align: top;">4</TD><TD style="vertical-align: top; width: 56px;">Hold</TD></TR></TBODY></TABLE></P><P>the experiment for each biobrick was do in duplicats.<I>Results:</I>After amplification the nucleic acid concentration was measured on the Nano Drop. The concentrations was measued to:

tube 1: pSB1A2 w. J13002 was 825.2ng/uL
tube 2: pSB1A2 w. B0015 was 547.76ng/uL
tube 3: pSB1A2 w. B0034 was 777.84ng/uL

The PCR product was loaded on a 2% agarose gel together with a 1000 bp long marker.
During the DNA extraction 50uL DNA anf 7uL loading buffer was loaded in a lane. The samples was loaded from left to right in continuous numbers. 
The concentration of the extracted DNA from tube 1,2 and  was found to 13.60ng/uL, 16,58ng/uL and 12,6313.60ng/uL respectively. while we have to use the dobbelt terminator and the constitutive active promotor in a lot of experiment the next step is to do another round of PRC and DNA extraction on thise biobricks.  
--<A href="/wiki/index.php?title=User:Pernm07&amp;action=edit&amp;redlink=1" class="new" title="User:Pernm07 (page does not exist)">Pernm07</A> 10:06, 22 July 2010 (UTC)
</P><H3><SPAN class="mw-headline" id="Amplification_and_purification_of_pSB1A2_w._J13002">Amplification and purification of pSB1A2 w. J13002</SPAN></H3><P><I>Experiment done by:</I> Pernille <I>Date:</I> The 22 of July <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#CP1.1" class="external text" rel="nofollow">CP1.1</A>and <A href="https://2010.igem.org/Team:SDU-Denmark/protocols#DNA_extraction_from_gel_.28fermentas.29" class="external text" rel="nofollow">DNA extraction</A><I>Method:</I> PCR and DNA extraction <I>Notes:</I> The DNA purified in the last experiment was amplified with PRC using VR og VF2 primers and pfu. the normal pfu program was used for running PCR. The PRC product was loaded on a 2% agarose gel. the brick J13002 (promotor+RBS) are 312bp long wich correspond to the visible band on the gel.
The DNA was extracted and afterwards the DNA concentration was measured on Nanodrop. the concentration was found to be 13,02ng/uL. The DNA was saved and it was cut with restriction enzymes.
</P><H2><SPAN class="mw-headline" id="Group:_Photosensor"> Group: Photosensor </SPAN></H2><H3><SPAN class="mw-headline" id="Miniprep_of_K081005_in_pSB1A2_and_R0011_in_pSB1A2_from_transformation_20.2F7"> Miniprep of <A href="http://partsregistry.org/Part:BBa_K081005" class="external text" rel="nofollow">K081005</A> in pSB1A2 and <A href="http://partsregistry.org/Part:BBa_R0011" class="external text" rel="nofollow">R0011</A> in pSB1A2 from transformation 20/7 </SPAN></H3><P>Start date: July 22nd<I>Methods:</I>  Miniprep, gel electrophoresis and nano drop <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#MP1.2" class="external text" rel="nofollow">MP1.2</A><I>Experiment done by:</I>  LC<I>Notes:</I> / <I>Results:</I>
Gel electrophoresis: 
Nanodrop: 
The concentration was around 23 ng/ul for all samples. Not overwhelming, but usable.</P><P><I>Analysis:</I>
Sample 1-4 of K081005 were pooled and frozen and sample 1-4 of R0011 were too.</P><H3><SPAN class="mw-headline" id="Gel_extraction_test_.28GFX_vs._Fermentas.29"> Gel extraction test (GFX vs. Fermentas) </SPAN></H3><P>Start date: July 25th <I>Methods:</I> Gel electrophoresis, gel extraction and nano drop <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#DE1.2" class="external text" rel="nofollow">DE1.2</A><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#DE1.3" class="external text" rel="nofollow">DE1.3</A><I>Experiment done by:</I>  Maria<I>Notes:</I>
FlhD/C taq PCR product (5 green) is used in the test.
PCR product is diluted in H2O to reach a sample concentration below 100ng/uL.
Nanodrop:</P><TABLE style="text-align: left; width: 100px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top; width: 98px;">Sample ID</TD><TD style="vertical-align: top; width: 43px;">ng/uL</TD><TD style="vertical-align: top;">260/280</TD><TD style="vertical-align: top;">260/230</TD></TR><TR><TD style="vertical-align: top; width: 49px;">PCR product prior
<P>to dilution</P></TD><TD style="vertical-align: top; width: 43px;">149.93</TD><TD style="vertical-align: top;">0.99</TD><TD style="vertical-align: top;">0.38</TD></TR><TR><TD style="vertical-align: top; width: 49px;">PCR product after
<P>dilution</P></TD><TD style="vertical-align: top; width: 43px;">89.76</TD><TD style="vertical-align: top;">0.99</TD><TD style="vertical-align: top;">0.35</TD></TR></TBODY></TABLE><P>A total of 70 microlitres is loaded onto a 2% agarose gel.
For the GFX gel extraction 350mg of gel was used.
For the Fermentas extraction 460mg of gel was used.
Gel electrophoresis:<I>Results:</I></P><TABLE style="text-align: left; width: 100px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top;">Sample ID</TD><TD style="vertical-align: top;">ng/uL</TD><TD style="vertical-align: top;">260/280</TD><TD style="vertical-align: top;">260/230</TD></TR><TR><TD style="vertical-align: top;">GFX 1</TD><TD style="vertical-align: top;">13.72</TD><TD style="vertical-align: top;">2.17</TD><TD style="vertical-align: top;">0.08</TD></TR><TR><TD style="vertical-align: top;">GFX2</TD><TD style="vertical-align: top;">16.8</TD><TD style="vertical-align: top;">2.02</TD><TD style="vertical-align: top;">0.05</TD></TR><TR><TD style="vertical-align: top;">GFX pooled</TD><TD style="vertical-align: top;">15.51</TD><TD style="vertical-align: top;">2.09</TD><TD style="vertical-align: top;">0.06</TD></TR><TR><TD style="vertical-align: top;">Fermentas 1</TD><TD style="vertical-align: top;">20.72</TD><TD style="vertical-align: top;">1.9</TD><TD style="vertical-align: top;">0.61</TD></TR><TR><TD style="vertical-align: top;">Fermentas 2</TD><TD style="vertical-align: top;">20.42</TD><TD style="vertical-align: top;">1.86</TD><TD style="vertical-align: top;">0.45</TD></TR><TR><TD style="vertical-align: top;">Fermentas pooled</TD><TD style="vertical-align: top;">21.38</TD><TD style="vertical-align: top;">1.87</TD><TD style="vertical-align: top;">0.51</TD></TR></TBODY></TABLE><P><I>Analysis:</I>
More DNA is extracted with the fermentas kit. This may be due to the short centrifugation time in the GFX protocol. Furthermore the GFX kit that was used was old and the buffers may have been contaminated.</P><H2><SPAN class="mw-headline" id="Group:_Retinal"> Group: Retinal </SPAN></H2><H3><SPAN class="mw-headline" id="Transformation_of_K081005_in_pSB1A2_.28constitutive_promoter_and_RBS_combined.29.2CR0011_in_pSB1A2.2C_pSB3C5_w._J04450_and_pSB3T5_w._J04450_in_Top_10_E.Coli"> Transformation of <A href="http://partsregistry.org/Part:BBa_K081005" class="external text" rel="nofollow">K081005</A> in pSB1A2 (constitutive promoter and RBS combined),<A href="http://partsregistry.org/Part:BBa_R0011" class="external text" rel="nofollow">R0011</A> in pSB1A2, pSB3C5 w. <A href="http://partsregistry.org/Part:BBa_J04450" class="external text" rel="nofollow">J04450</A> and pSB3T5 w. J04450 in Top 10 E.Coli </SPAN></H3><P>Start date: 19/07    End date: 20/07<I>Methods:</I>  ON culture, making competent cells, transformation <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#CC1.1" class="external text" rel="nofollow">CC1.1</A><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#TR1.1" class="external text" rel="nofollow">TR1.1</A><I>Experiment done by:</I> Maria <I>Notes:</I>ON colony was made of 110 ml lb medium inoculated with a top10 coloni.

<TABLE style="text-align: left; width: 100px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top;">Time</TD><TD style="vertical-align: top;">Optical  density</TD></TR><TR><TD style="vertical-align: top;">8:12</TD><TD style="vertical-align: top;">2.9</TD></TR><TR><TD style="vertical-align: top;">8:17</TD><TD style="vertical-align: top;">0.02</TD></TR><TR><TD style="vertical-align: top;">9:17</TD><TD style="vertical-align: top;">0.035</TD></TR><TR><TD style="vertical-align: top;">10:17</TD><TD style="vertical-align: top;">0.204</TD></TR><TR><TD style="vertical-align: top;">10:40</TD><TD style="vertical-align: top;">0.380</TD></TR><TR><TD style="vertical-align: top;">10:50</TD><TD style="vertical-align: top;">0.49</TD></TR></TBODY></TABLE>
pSB1A2 w. R0011 and pSB1A2 w. K081005 was plated with 150uL on plates containing LA, LA + Amp, LA + Tetracycline, LA + Chloramphenicol and LA + Kanamycine.Upconcentration of these samples was not made. pSB3T5 w. J04450 and pSB3C5 w. J04450 was plated according to protocol
<I>Results:</I><I>Analysis:</I>
Both pSB3T5 and pSB3C5 was succesfully transformed, and ON cultures with appropiate antibiotics were made for mini-prep.
For pSB1A2 w. R0011 only 6 colonies was observed on the LA+amp plate.
For pSB1A2 w. K081005 only 4 colonies was observed on the LA+amp plate.
All 10 colonies were used for <A href="https://2010.igem.org/Team:SDU-Denmark/labnotes2#Coloni_PCR_of_R0011_in_pSB1A2_and_K081005_in_pSB1A2" class="external text" rel="nofollow">Coloni PCR</A>.
--<A href="/wiki/index.php?title=User:Tipi&amp;action=edit&amp;redlink=1" class="new" title="User:Tipi (page does not exist)">Tipi</A> 16:33, 19 July 2010 (UTC)</P><H3><SPAN class="mw-headline" id="Transformation_of_flhD.2FC_in_pSB1C3_and_test_plasmid_in_Top_10_E.Coli"> Transformation of flhD/C in pSB1C3 and test plasmid in Top 10 E.Coli </SPAN></H3><P>Start date: 20/07    End date: 19/07<I>Methods:</I>  ON culture, making competent cells, transformation <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#CC1.1" class="external text" rel="nofollow">CC1.1</A><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#TR1.1" class="external text" rel="nofollow">TR1.1</A><I>Experiment done by:</I> Maria <I>Notes:</I>
Ligated flhD/C from <A href="https://2010.igem.org/Team:SDU-Denmark/labnotes2#Ligation_of_flhD.2FC_and_pSB1C3" class="external text" rel="nofollow">Ligation</A> and test plasmid from Whatman was transformed. 
ON colony was made of 25 ml lb medium inoculated with a top10 coloni.<TABLE style="text-align: left; width: 100px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top;">Time</TD><TD style="vertical-align: top;">Optical  density</TD></TR><TR><TD style="vertical-align: top;">8:17</TD><TD style="vertical-align: top;">3.5</TD></TR><TR><TD style="vertical-align: top;">8:20</TD><TD style="vertical-align: top;">0.02</TD></TR><TR><TD style="vertical-align: top;">9:15</TD><TD style="vertical-align: top;">0.035</TD></TR><TR><TD style="vertical-align: top;">10:30</TD><TD style="vertical-align: top;">0.222</TD></TR><TR><TD style="vertical-align: top;">10:52</TD><TD style="vertical-align: top;">0.350</TD></TR><TR><TD style="vertical-align: top;">11:05</TD><TD style="vertical-align: top;">0.52</TD></TR></TBODY></TABLE>
Compotent cells were washed using 10mL 50mM CaCl2.
3 parallel transformations were carried out for L1, L2 and L3 respectively (see <A href="https://2010.igem.org/Team:SDU-Denmark/labnotes2#Ligation_of_flhD.2FC_and_pSB1C3" class="external text" rel="nofollow">Ligation</A>). L1.1, L1.2, L2.1, L2.2 and L3.1 were transformed using compotent cells from 19/7 (<A href="https://2010.igem.org/Team:SDU-Denmark/labnotes2#Transformation_of_K081005_in_pSB1A2_.28constitutive_promoter_and_RBS_combined.29.2CR0011_in_pSB1A2.2C_pSB3C5_w._J04450_and_pSB3T5_w._J04450_in_Top_10_E.Coli" class="external text" rel="nofollow">Compotent cells</A>). 
Prior to transformation test plasmid was washed with 10xTE pH 8.0. <I>Results:</I><I>Analysis:</I>
Test plasmid was succelsfully transformed and we can carry on transforming the plasmid containing the NinaB gene.
Colonies was seen on plates transformed with L2 and L3. 10 colonies were selected and used for coloni PCR.
--<A href="/wiki/index.php?title=User:Tipi&amp;action=edit&amp;redlink=1" class="new" title="User:Tipi (page does not exist)">Tipi</A> 08:17, 21 July 2010 (UTC)</P><H3><SPAN class="mw-headline" id="Coloni_PCR_of_R0011_in_pSB1A2_and_K081005_in_pSB1A2"> Coloni PCR of <A href="http://partsregistry.org/Part:BBa_R0011" class="external text" rel="nofollow">R0011</A> in pSB1A2 and <A href="http://partsregistry.org/Part:BBa_K081005" class="external text" rel="nofollow">K081005</A> in pSB1A2 </SPAN></H3><P>Start date: 20/07    End date: 20/07<I>Methods:</I>  Coloni PCR and gel electrophoresis <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#CP1.2" class="external text" rel="nofollow">CP1.2</A><I>Experiment done by:</I> Maria and LC<I>Notes:</I>
6 colonies transformed with pSB1A2 w. R0011 and 4 colonies transformed with pSB1A2 w. K081005 (see <A href="https://2010.igem.org/Team:SDU-Denmark/labnotes2#Transformation_of_K081005_in_pSB1A2_.28constitutive_promoter_and_RBS_combined.29.2CR0011_in_pSB1A2.2C_pSB3C5_w._J04450_and_pSB3T5_w._J04450_in_Top_10_E.Coli" class="external text" rel="nofollow">Transformation</A>) was used for coloni PCR. 
Colonies with K081005 was denoted A1, A2 A3 and A4 respectively.
Colonies with R0011 was denotes B1, B2, B3, B4, B5 and B6 respectively
PCR was made with only half the amount of premix.
Premix for 12 PCR reactions: </P><TABLE style="text-align: left; width: 100px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top;">10xtaq buffer</TD><TD style="vertical-align: top;">30uL</TD></TR><TR><TD style="vertical-align: top;">MgCl2</TD><TD style="vertical-align: top;">12uL</TD></TR><TR><TD style="vertical-align: top;">VF2</TD><TD style="vertical-align: top;">12uL</TD></TR><TR><TD style="vertical-align: top;">VR</TD><TD style="vertical-align: top;">12uL</TD></TR><TR><TD style="vertical-align: top;">dNTP</TD><TD style="vertical-align: top;">6uL</TD></TR><TR><TD style="vertical-align: top;">H2O</TD><TD style="vertical-align: top;">42uL</TD></TR><TR><TD style="vertical-align: top;">Total vol.</TD><TD style="vertical-align: top;">114uL</TD></TR></TBODY></TABLE><P>0.5uL Taq polymerase from ampliqon was used.
Taq PCR program:</P><TABLE style="text-align: left; height: 260px; width: 225px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top; width: 102px;">1:Start</TD><TD style="vertical-align: top; width: 52px;">94C</TD><TD style="vertical-align: top; width: 45px;">2 min</TD></TR><TR><TD style="vertical-align: top; width: 102px;">2: Denaturing</TD><TD style="vertical-align: top; width: 52px;">94C</TD><TD style="vertical-align: top; width: 45px;">1 min</TD></TR><TR><TD style="vertical-align: top; width: 102px;">3: Annealing</TD><TD style="vertical-align: top; width: 52px;">55C</TD><TD style="vertical-align: top; width: 45px;">1 min</TD></TR><TR><TD style="vertical-align: top; width: 102px;">4: Elongation</TD><TD style="vertical-align: top; width: 52px;">72C</TD><TD style="vertical-align: top; width: 45px;">30 s</TD></TR><TR><TD style="vertical-align: top; width: 102px;">5:</TD><TD style="vertical-align: top; width: 52px;">GO TO</TD><TD style="vertical-align: top; width: 45px;">rep. 29x</TD></TR><TR><TD style="vertical-align: top; width: 102px;">6: End </TD><TD style="vertical-align: top; width: 52px;">72C</TD><TD style="vertical-align: top; width: 45px;">3 min</TD></TR><TR><TD style="vertical-align: top; width: 102px;">7: Hold</TD><TD style="vertical-align: top; width: 52px;">4C</TD></TR></TBODY></TABLE><P>PCR product was loaded onto a 2% agarose gel. Gene ruler 100bp DNA ladder was used as marker.<I>Results:</I>
Gel electrophoresis: <I>Analysis:</I>
A band at app. 300bp was observed in all lanes, indicating that all 10 colonies contained the correct plasmids.
ON cultures were made for mini-prep.
--<A href="/wiki/index.php?title=User:Tipi&amp;action=edit&amp;redlink=1" class="new" title="User:Tipi (page does not exist)">Tipi</A> 06:35, 21 July 2010 (UTC)</P><H3><SPAN class="mw-headline" id="Miniprep_of_pSB3T5_w._J04450_and_pSB3C5_w._J04450_from_transformation_20.2F7"> Miniprep of pSB3T5 w. J04450 and pSB3C5 w. J04450 from transformation 20/7 </SPAN></H3><P>Start date: 21/07    End date: 21/07<I>Methods:</I>  Miniprep, gel electrophoresis and nano drop <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#MP1.2" class="external text" rel="nofollow">MP1.2</A><I>Experiment done by:</I> Maria and LC<I>Notes:</I>
For pSB3T5 no.3 one half of the material was lost during the neutralization step. 
pSB3T5 no.1 was eluted in 200uL elution buffer.
Samples were loaded onto a 1.5 agarose gel. Generuler DNA ladder-mix (red) was used as marker<I>Results:</I>
Gel electrophoresis: 
Nanodrop: </P><TABLE style="text-align: left; width: 300px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top;">Sample ID</TD><TD style="vertical-align: top;">ng/uL</TD><TD style="vertical-align: top;">260/280</TD><TD style="vertical-align: top;">260/230</TD></TR><TR><TD style="vertical-align: top;">pSB3C5 1</TD><TD style="vertical-align: top;">208.2</TD><TD style="vertical-align: top;">1.93</TD><TD style="vertical-align: top;">2.31</TD></TR><TR><TD style="vertical-align: top;">pSB3C5 2</TD><TD style="vertical-align: top;">225.09</TD><TD style="vertical-align: top;">1.93</TD><TD style="vertical-align: top;">2.3</TD></TR><TR><TD style="vertical-align: top;">pSB3C5 3</TD><TD style="vertical-align: top;">215.2</TD><TD style="vertical-align: top;">1.93</TD><TD style="vertical-align: top;">2.3</TD></TR><TR><TD style="vertical-align: top;">pSB3C5 4</TD><TD style="vertical-align: top;">260.68</TD><TD style="vertical-align: top;">1.93</TD><TD style="vertical-align: top;">2.33</TD></TR><TR><TD style="vertical-align: top;">pSB3T5 1</TD><TD style="vertical-align: top;">14.16</TD><TD style="vertical-align: top;">2.06</TD><TD style="vertical-align: top;">2.0</TD></TR><TR><TD style="vertical-align: top;">pSB3T5 2</TD><TD style="vertical-align: top;">52.50</TD><TD style="vertical-align: top;">1.96</TD><TD style="vertical-align: top;">2.15</TD></TR><TR><TD style="vertical-align: top;">pSB3T5 3</TD><TD style="vertical-align: top;">26.45</TD><TD style="vertical-align: top;">1.99</TD><TD style="vertical-align: top;">2.3</TD></TR><TR><TD style="vertical-align: top;">pSB3T5 4</TD><TD style="vertical-align: top;">53.43</TD><TD style="vertical-align: top;">2.0</TD><TD style="vertical-align: top;">2.19</TD></TR></TBODY></TABLE><P><I>Analysis:</I>
Both pSB3T5 and pSB3C5 was succesfully purified.
pSB3C5 1-4 was pooled and stored as no.28 (white) at -20 degrees.
pSB3T5 1+3 was pooled and stored as no.30 (white) at -20 degrees.
pSB3T5 2+4 was pooled and stored as no.29 (white) at -20 degrees.</P><P>Nanodrop of pooled samples:</P><TABLE style="text-align: left; width: 300px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top;">Sample ID</TD><TD style="vertical-align: top;">ng/uL</TD><TD style="vertical-align: top;">260/280</TD><TD style="vertical-align: top;">260/230</TD></TR><TR><TD style="vertical-align: top;">pSB3C5 1-4</TD><TD style="vertical-align: top;">256.71</TD><TD style="vertical-align: top;">1.92</TD><TD style="vertical-align: top;">2.3</TD></TR><TR><TD style="vertical-align: top;">pSB3T5 1+3</TD><TD style="vertical-align: top;">54.01</TD><TD style="vertical-align: top;">1.95</TD><TD style="vertical-align: top;">2.19</TD></TR><TR><TD style="vertical-align: top;">pSB3T5 2+4</TD><TD style="vertical-align: top;">16.0</TD><TD style="vertical-align: top;">1.88</TD><TD style="vertical-align: top;">2.11</TD></TR></TBODY></TABLE><P>--<A href="/wiki/index.php?title=User:Tipi&amp;action=edit&amp;redlink=1" class="new" title="User:Tipi (page does not exist)">Tipi</A> 09:41, 21 July 2010 (UTC)
</P><H3><SPAN class="mw-headline" id="Transformation_of_TOP_10_e._coli_with_ninaB_gene_in_pOT2:chlor_vector"> Transformation of TOP 10 e. coli with ninaB gene in pOT2:chlor vector </SPAN></H3><P>
Date: 21/7

Done by: Christian
</P><H4><SPAN class="mw-headline" id="Transformation_of_cells_with_gene_in_plasmid"> Transformation of cells with gene in plasmid </SPAN></H4><P>
Date: 21/7
Methods: Transformation
Protocols: TR1.1 and paperblotter protocol from plasmid cDNA package.

Notes: Filterpaper was washed with 50ul TE buffer for 2 seconds. Paper punch was then transfered to a sterile eppendorff tube, and transformation protocol TR1.1 was followed from here. The usual positive control was used.
Plates were dried for ca. 12 minutes, showed no sign of cracking.
The gene is inserted into a pOT2 plasmid. A plasmid map can be found <A href="http://www.fruitfly.org/about/methods/pOT2vector.html" class="external text" rel="nofollow">here</A>:
Two of the chlor plates were damaged (one very slightly) and one ampicilin plate was torn up, again. (good thing two were dried). Plates were incubated ON at 37°.
Results: Colonies showed on all plates.
Transformations have been succesfull. If our gene is in there will be tested next week.
</P><H3><SPAN class="mw-headline" id="Coloni_PCR_of_flhD.2FC_in_pSB1C3"> Coloni PCR of flhD/C in pSB1C3 </SPAN></H3><P>Start date: 21/07    End date: 21/07<I>Methods:</I>  Coloni PCR and gel electrophoresis <I>Protocol:</I><A href="https://2010.igem.org/Team:SDU-Denmark/protocols#CP1.2" class="external text" rel="nofollow">CP1.2</A><I>Experiment done by:</I> Maria and LC<I>Notes:</I>
Coloni PCR was made on flhD/C in pSB1C3 from <A href="https://2010.igem.org/Team:SDU-Denmark/labnotes2#Transformation_of_flhD.2FC_in_pSB1C3_and_test_plasmid_in_Top_10_E.Coli" class="external text" rel="nofollow">Transformation</A>.
10 colonies from plates plated with cells transformed with L2 and L3 ligation mixtures (see <A href="https://2010.igem.org/Team:SDU-Denmark/labnotes2#Ligation_of_flhD.2FC_.28native_coding_sequence.29_and_pSB1C3" class="external text" rel="nofollow">Ligation</A>). 
Coloni 1-5 is taken from L2 plates
coloni 6-10 is taken from L3 plates 
PCR was made with only half the amount of premix.
Premix for 12 PCR reactions: </P><TABLE style="text-align: left; width: 100px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top;">10xtaq buffer</TD><TD style="vertical-align: top;">30uL</TD></TR><TR><TD style="vertical-align: top;">MgCl2</TD><TD style="vertical-align: top;">12uL</TD></TR><TR><TD style="vertical-align: top;">VF2</TD><TD style="vertical-align: top;">12uL</TD></TR><TR><TD style="vertical-align: top;">VR</TD><TD style="vertical-align: top;">12uL</TD></TR><TR><TD style="vertical-align: top;">dNTP</TD><TD style="vertical-align: top;">6uL</TD></TR><TR><TD style="vertical-align: top;">H2O</TD><TD style="vertical-align: top;">42uL</TD></TR><TR><TD style="vertical-align: top;">Total vol.</TD><TD style="vertical-align: top;">114uL</TD></TR></TBODY></TABLE><P>0.5uL Taq polymerase from ampliqon was used.
Taq PCR program:</P><TABLE style="text-align: left; height: 260px; width: 225px;" border="1" cellpadding="2" cellspacing="2"><TBODY><TR><TD style="vertical-align: top; width: 102px;">1:Start</TD><TD style="vertical-align: top; width: 52px;">94C</TD><TD style="vertical-align: top; width: 45px;">2 min</TD></TR><TR><TD style="vertical-align: top; width: 102px;">2: Denaturing</TD><TD style="vertical-align: top; width: 52px;">94C</TD><TD style="vertical-align: top; width: 45px;">1 min</TD></TR><TR><TD style="vertical-align: top; width: 102px;">3: Annealing</TD><TD style="vertical-align: top; width: 52px;">55C</TD><TD style="vertical-align: top; width: 45px;">1 min</TD></TR><TR><TD style="vertical-align: top; width: 102px;">4: Elongation</TD><TD style="vertical-align: top; width: 52px;">72C</TD><TD style="vertical-align: top; width: 45px;">1 min 30 s</TD></TR><TR><TD style="vertical-align: top; width: 102px;">5:</TD><TD style="vertical-align: top; width: 52px;">GO TO</TD><TD style="vertical-align: top; width: 45px;">rep. 29x</TD></TR><TR><TD style="vertical-align: top; width: 102px;">6: End </TD><TD style="vertical-align: top; width: 52px;">72C</TD><TD style="vertical-align: top; width: 45px;">3 min</TD></TR><TR><TD style="vertical-align: top; width: 102px;">7: Hold</TD><TD style="vertical-align: top; width: 52px;">4C</TD></TR></TBODY></TABLE><P>PCR product was loaded onto a 2% agarose gel. Gene ruler DNA ladder mix (red) was used as marker.
VF2 - coding sequence (in pSB1C3) : 140bp 
Coding sequence - VR (in pSB1C3)  : 176bp 
flhD/C               : 932bp 
VF2 - VR for pSB1C3 w. flhD/C     :1248bp 
VF2 - VR for pSB1C3 without insertion : 316bp <I>Results:</I>
Gel electrophoresis: </P><P><I>Analysis:</I>
All lanes except 4, 9 and 10 showed a band at app. 1200bp. which might indicate plasmids where flhD/C has been inserted.
Lane 4 and 9 has a band at app. 1500bp which could indicate undigested pSB1C3 (VF2 - VR for pSB1C3 w. J04450 is 1385bp).</P></DIV><DIV id="rightcolumn"><P>Include column content here.
</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:SDU-Denmark/labnotes2">http://2010.igem.org/Team:SDU-Denmark/labnotes2</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SDU-Denmark/labnotes2" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SDU-Denmark/labnotes2" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SDU-Denmark/labnotes2&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SDU-Denmark/labnotes2&amp;oldid=136491" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>