<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SDU-Denmark_project-t"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SDU-Denmark/project-t">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SDU-Denmark/project-t&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SDU-Denmark/project-t&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SDU-Denmark/project-t&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SDU-Denmark/project-t" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SDU-Denmark/project-t</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="header"><DIV id="return"><A href="https://2010.igem.org/">Back to iGEM 2010</A></DIV></DIV><DIV id="navigation"><UL id="topnav"><LI><A href="/Team:SDU-Denmark">               Home</A></LI><LI><A href="#">Team</A><SPAN><A href="/Team:SDU-Denmark/team">Students</A>|
            <A href="/Team:SDU-Denmark/team2">Advisors</A>|<A href="/Team:SDU-Denmark/team3">Sponsors</A>|
            <A href="/Team:SDU-Denmark/team4">Collaborators</A></SPAN></LI><LI><A href="#">Project</A><SPAN><A href="/Team:SDU-Denmark/project-i">Idea</A>|
            <A href="/Team:SDU-Denmark/project-t">Theory</A>|
            <A href="/Team:SDU-Denmark/project-activities">Activities</A>|
            <A href="/Team:SDU-Denmark/parts">Parts</A>|
            <A href="/Team:SDU-Denmark/project-r">Result summary</A>|
            <A href="/Team:SDU-Denmark/project-p">Characterization</A></SPAN></LI><LI><A href="#">Modelling</A><SPAN><A href="/Team:SDU-Denmark/project-m">Physical</A>|
            <A href="/Team:SDU-Denmark/project-bc">Biochemical</A></SPAN></LI><LI><A href="#">Safety</A><SPAN><A href="/Team:SDU-Denmark/safety-a">Overview</A>|
            <A href="/Team:SDU-Denmark/safety-b">Project and Researcher Safety</A>|
            <A href="/Team:SDU-Denmark/safety-c">Laws and Guidelines</A>|
            <A href="/Team:SDU-Denmark/safety-d">Watermark</A></SPAN></LI><LI><A href="#">Notebook</A><SPAN><A href="/Team:SDU-Denmark/notebook">Progress</A>|
            <A href="/Team:SDU-Denmark/labnotes0">Lab notes</A>|
            <A href="/Team:SDU-Denmark/protocols">Protocols</A></SPAN></LI><LI><A target="blank" href="http://igem.sdu.dk">Blog</A></LI></UL></DIV><DIV id="subnavi"><DIV id="leftcolumn"><H1><SPAN class="mw-headline" id="Theory"> Theory </SPAN></H1><P>In this section we will review the theory behind our biological work. 
</P><H2><SPAN class="mw-headline" id="Phototaxis"> Phototaxis </SPAN></H2><H4><SPAN class="mw-headline" id="Bacterial_flagellar_motility">Bacterial flagellar motility</SPAN></H4><P>
Bacteria have evolved many modes of propulsion for the microscale environments they inhabit. At these scales materials behave very different than at macroscale, and of particular interest to us is the way liquids seem more viscous [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">1</A>]. One of the many ways bacteria move around in liquids, is by means of flagella. A single flagellum is a thin filament around 100-150 Å thick, that extends many cell lengths out from the cell [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">2</A>]. It consists mainly of flagellin subunits that assemble into a helical structure forming a long hollow cylindrical filament [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">3</A>]. In <I>E. coli</I> the mean number of flagella per cell is 4 [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">4</A>], but there is a wide variance between strains, and even between individual cells of each strain. The environment around the cell also has a large influence on how many flagellae are present, or if they are present at all [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">4</A>]. Flagella rotate to generate force that allows bacterial cells to swim through fluids in characteristic patterns, more on which later, in search of better conditions for proliferation or survival [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">5</A>]. Normally flagellated strains of <I>E. coli</I> can achieve speeds up to 20µm/sec [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">4</A>], and considering a cell length of only 1-2µm, this is an impressive feat indeed.</P><P>A flagellum  is anchored to the cell body by a large, wheel-like protein complex spanning both inner and outer membranes [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">4</A>]. Through this complex subunits are secreted to the tip of the flagellar tube [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">3</A>], thus elongating the filament (This mechanism is slightly different in archaea, where the filament is assembled from the base of the flagellum). The membrane anchor also functions as a rotary engine, driven by the proton motive force [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">4</A>]. In <I>E. coli</I> the flagellar motor rotates at up to 300hz and can turn either clockwise or counter-clockwise [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">4</A>], both resulting in a distinct movement pattern for the cell.</P><P>Spinning in the counter-clockwise direction, the flagella will twist into a bundle in the shape of a corkscrew, and create a linear driving force [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">4</A>], propelling the cell in a straight line through the liquid. This form of movement is termed run. Spun in the clockwise direction one might then expect the cell to reverse, but this is not the case. Instead the flagellar bundle will unwind, thereby creating chaotic movement [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">4</A>]. This movement reorients the cell randomly and is termed tumbling. A cell will typically run for some time, then change it’s orientation by tumbling, and then run again [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">5</A>]. The direction of flagella rotation is controlled by the binding of a cytosolic protein CheY, more on which later [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">4</A>].</P><P>Different taxis pathways that steer cells towards favorable conditions and away from danger work by regulating the frequency of tumbling events [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">5</A>]. An example is when a bacterium gets close to the source of a lethal toxin, then intracellular pathways will increase the frequency of tumbling events, in effect preventing the cell from dying. Since the frequency of tumbling events will decrease if the cell is going in a direction away from the toxin, it will “encourage” the cell to continue in that direction. In the case of an attractant such as an increase in nutrient concentration, the pattern will be opposite, so that the cell is encouraged to continue towards the source of the attractant. This form of movement, combining tumbling and running, with regulation of the tumbling frequency is termed a biased random walk [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">5</A>]. </P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 1:</B> A biased random walk motion pattern.</DIV></DIV></DIV><P>To understand how this can work we need a simplified understanding of the chemotaxis pathway at a molecular level. Chemotactic receptors can both increase and decrease tumbling frequencies to generate a biased random walk behavior [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">6</A>]. Increased tumbling is achieved through a phosphorylation cascade beginning with the binding of a repellant to a transmembrane receptor [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">4</A>]. The receptor is linked to two proteins, CheW and CheA [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">4</A>]. CheA is a histidine-kinase that will autophosphorylate when the repellant binds [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">4</A>]. The phosphoryl group is then transferred to CheY [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">4</A>]. The flagellar motor complex has high affinity for phosphorylated CheY (CheY-p), and binding reverses the mode of movement from run to tumble [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">7</A>]. CheY-p is continuously dephosphorylated back to CheY by CheZ [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">8</A>]. A receptor sensing an attractant might instead switch from the default active CheA state to an inactive state when it’s ligand is bound, thus decreasing CheY phosphorylation [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">6</A>].</P><H4><SPAN class="mw-headline" id="Photosensor">Photosensor</SPAN></H4><P>In our system we want to be able to control the amount of flow in the channel, through a remote signal. The signal we have chosen is light since we want to avoid altering the chemical composition of the fluid running through the channel. Having looked at previous iGEM work on light sensitive systems which have all been focused on transcriptional regulation (<A href="http://partsregistry.org/wiki/index.php/Part:BBa_I15010" class="external text" rel="nofollow">e.g. I15010</A>), we realized that we would need a different approach for the fast response times our system requires. We have therefore focused our work on proteorhodopsins that integrate into the chemotaxis pathway, giving us very fast responses to light stimulation. </P><P>Our construct centers around a synthetic protein created by Spudich <I>et al.</I> [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">9</A>]. It is a fusion chimeric protein that consists of an archaeal proteorhodopsin SopII (Sensory Rhodopsin II) and its transducer protein HtrII, both from <I>Natronomonas pharaonis</I>. These are coupled to a Tar domain from a transmembrane receptor from <I>Salmonella enterica</I>. The Tar domain is the part of the receptor that couples with CheA and CheW, and although it is taken from a different species, it has been shown to work in <I>E. coli</I> as well [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">10</A>]. In the construct we are working with light acting as an attractant, reducing the tumbling rate upon illumination (see picture). This might help us to control our pumping power, by decreasing the fraction of bacteria tumbling in the channel by increasing light stimulus, thus promoting linear drive. The photosensor should be most active in light with a wavelength of about 500nm, according to the original article. We have used DNA sent to us from the original authors to isolate the coding sequence for the protein generator.
Note that although the bacteria will be stationary in our system, since they are glued to the inner surface of the flowchannel, our construct in reality confers a phototactic ability to <I>E. coli</I>.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 2:</B><B>A.</B> The SopII proteorhodopsin has not been activated. Note that CheA is active by default, continuosly autophosphorylating itself, and cycling back to it's unphosphorylated state by transfering the phosphoryl group to CheY. High levels of CheY-p will induce tumbling motion in the flagella. Also, note that CheZ continuously dephosphorylates CheY. <B>B.</B> SopII is hit with a photon, causing conformational change of the entire complex. The inactivation of CheA halts production of CheY-p, and CheZ rapidly dephosphorylates the remaining CheY-p, resulting in a reduced frequency of tumbling.</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="BioBrick_design">BioBrick design</SPAN></H4><P>
Photosensor coding biobrick: <A href="http://partsregistry.org/Part:BBa_K343003" class="external text" rel="nofollow">K343003</A>
Photosensor composite part: <A href="http://partsregistry.org/Part:BBa_K343007" class="external text" rel="nofollow">K343007</A></P><H2><SPAN class="mw-headline" id="Retinal_Generator"> Retinal Generator </SPAN></H2><H4><SPAN class="mw-headline" id="Retinal_requirements_of_light-sensing_proteins">Retinal requirements of light-sensing proteins</SPAN></H4><P>Proteorhodopsins belong to a larger group of proteins called retinylidene proteins consisting of proteins called an opsins with a retinoid cromophore such as retinal attached as a prosthetic group[<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">11</A>]. In our system retinal is responsible for the initial light-activation, as it undergoes photoizomerization when it is struck by a photon[<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">12</A>]. It is this change in conformation of the retinal molecule that is relayed through the entire protein complex to regulate chemotaxis [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">12</A>]. Thus either an external supply of retinal or an internal supply, generated by means of genes coding for enzymes in the retinal biosynthesis pathway are required, if we wish to see phototactic behaviour in our cells [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">10</A>]. </P><P>Light-sensitive pigments can be found in a large variety of organisms from archaea and bacteria to both uni- and multicellular eukaryotes [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">11</A>], [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">13</A>], [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">14</A>]. Many plants and microbes have complete retinal biosynthesis pathways in their genomes [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">15</A>]. In these organisms rhodopsins play an essential role, not only for photosensing but also directly in energy production [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">11</A>],[<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">13</A>],[<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">14</A>]. In fact in some organisms rhodopsins are used to create proton motive force directly by pumping protons out into the extracellular space using light energy to drive the process [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">14</A>]. Humans and other animals on the other hand often only have enzymes coding for the final steps of the pathway, more on which later. They rely on a supply of retinal precursors or vitamin A (a group of molecules consisting of retinal and it's metabolites) in their diet [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">16</A>]. This is why vitamin A deficiency causes night-blindness as an early symptom in humans [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">17</A>].
</P><H4><SPAN class="mw-headline" id="Retinal_biosynthesis"> Retinal biosynthesis </SPAN></H4><P>Retinal can be synthesized from the enzymatic cleavage of some carotenes. In our system we focus on cleavage of beta-carotene, partly because it yields 2 <I>all-trans</I> retinal molecules which are the molecules we desire, and partly because the beta-carotene biosynthesis pathway has already been introduced to <I>E. coli</I> by the <A href="http://partsregistry.org/Part:BBa_K274210" class="external text" rel="nofollow">Cambridge 2009 </A> iGEM team.
</P><P>The Cambridge construct uses genes from the plant pathogen <I>Pantoea ananatis</I> and our construct completes the pathway to retinal with a gene from the common fruit fly, <I>Drosophila melanogaster</I>.  </P><P>The Cambridge 2009 construct consists of four genes <I>crtE</I>, <I>crtB</I>, <I>crtI</I> and <I>crtY</I> from <I>P. ananatis</I> that together make up the pathway that converts farnesyl pyrophosphate to beta-carotene, which is a precursor for retinal. Farnesyl pyrophosphate is naturally present in <I>E. coli</I>. 
•	<I>crtE</I> encodes the protein geranyl-geranyl pyrophosphate synthase that converts farnesyl pyrophosphate to geranyl-geranyl pyrophosphate by elongating it by one unit of isopentenyl. 
•	<I>crtB</I> encodes the protein phytoene synthase and synthesizes phytoene by putting together two molecules of geranyl-geranyl pyrophosphate whilst cutting off 2 molecules of pyrophosphate. 
•	<I>crtI</I> encodes the protein phytoene dehydrogenase and converts phytoene to lycopene by converting the trans bond to a cis bond and adding more cis double bonds. 
•	<I>crtY</I> encodes the protein lycopene B-cyclase and converts lycopene to beta-carotene. </P><P>The pathway (including the step that generates retinal) is summed up below: </P><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Figure 3: The retinal Biosynthesis pathway.</DIV></DIV></DIV><P>To introduce the final step from beta-carotene to retinal, we use the gene ninaB from D. melanogaster. This gene encodes the protein beta-carotene 15,15’-monooxygenase, which cleaves beta-carotene to produce two molecules of trans-retinal under the consumption of oxygen [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">18</A>]. We have inserted the part K343006 into a different plasmid from the K274210 part since both parts are very long, so a plasmid containing both wouldn't have been viable for transformations.
</P><H3><SPAN class="mw-headline" id="BioBrick_design_2"> BioBrick design </SPAN></H3><P>
Retinal coding biobrick: <A href="http://partsregistry.org/Part:BBa_K343001" class="external text" rel="nofollow">K343001 </A>
Retinal composite part: <A href="http://partsregistry.org/Part:BBa_K343006" class="external text" rel="nofollow">K343006</A></P><H3><SPAN class="mw-headline" id="Further_use_of_the_retinal_BioBrick"> Further use of the retinal BioBrick </SPAN></H3><H4><SPAN class="mw-headline" id="Role_in_light-based_signal_transduction"> Role in light-based signal transduction </SPAN></H4><P>Since retinal plays such an essential role in photosensing in both eukaryotes as well as bacteria and archaea, all work with rhodopsins and proteorhodopsins will need a retinal supply to function. This supply might come from the external environment, but it is an appealing thought that we might be able to supply the retinal internally. Our project centers around phototaxis, but constructs combining photorhodopsins with other membrane associated tyrosine kinases may also be imagined, opening vast posibilities for regulation of phopsphorylation cascades using light as input. In such systems, retinal biosynthesis could play a very valuable role.</P><H2><SPAN class="mw-headline" id="Hyperflagellation"> Hyperflagellation </SPAN></H2><H3><SPAN class="mw-headline" id="Background"> Background </SPAN></H3><P>
Motility can be a very beneficial quality for a microorganism. Evolution has therefore provided bacteria with two general means of transportation; the flagella and the pili. These qualities allow the bacteria to move away from a hostile environment and towards more favorable conditions. Flagella and pili are however viewed as a virulence factor as they also serve as an advantage in colonizing a host organism and yet they can cause a strong immune response [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">19</A>]. </P><P>Many organisms are able to synthesize a flagellum, if the external environment calls for it. The synthesis of a flagellum is a huge and energy consuming process and is therefore tightly regulated by the bacteria’s external environment. One of the most well characterized flagellation systems is the one found in <I>E. coli</I>. Here at least 50 genes are involved in the hierarchical synthesis and operation of the flagella. These genes are sorted into 15 operons which are expressed in a transcriptional cascade separated into three classes. Class I consists of the master operon <I>flhDC</I>. The active FlhDC protein is a hexamer organized into an FlhD<SUB>4</SUB>C<SUB>2</SUB> complex with a computed value of 96,4kDa [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">20</A>]. The homodimeric FlhC protein is able to bind DNA, while the FlhD homodimers are not. The formation of the FlhDC complex however, stabilizes and increases the DNA binding ability [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">21</A>]. The transcription of <I>flhDC</I> is heavily regulated by nutritional and environmental conditions. Flagellum synthesis is inhibited at high temperatures, at high salt concentrations, at extreme pH or in the presence of carbohydrates, low molecular alcohols or DNA gyrase inhibitors, as these conditions stimulate growth as opposed to motility [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">22</A>]. Because the flagellum synthesis is so energy consuming, the process is not started unless the environment calls for motility rather than growth. In fact, in situations where nutrition is plenty over a long period, the bacteria will focus on growth and over time lose the ability to synthesize the flagellum, as seen with the <I>E. coli</I> strain MG1655 localized in mouse intestines [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">23</A>].
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Figure 4: Overviews cascade of the flagellum synthesis.</DIV></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:212px;"><DIV class="thumbcaption"><B>Figure 5:</B>3D structure of the FlhD<SUB>4</SUB>C<SUB>2</SUB> hexamer.[<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">23</A>][<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">24</A>]</DIV></DIV></DIV><P>The FlhD<SUB>4</SUB>C<SUB>2</SUB> hexamer acts as a transcription factor for the Class II genes, which encodes the basal body, that is embedded in the cell membrane as well as hook proteins, which are transported to the cell exterior through the basal body. Another Class II gene is the σ<SUP>28</SUP> transcription factor, which is responsible for the transcription of the Class III genes. This includes <I>fliC</I>, which encodes the flagellin subunit that composes the flagella “tail”. To ensure that the Class III genes are not transcribed before the assembly of the basal body and the hook is complete another Class II protein FliM acts as an anti-sigma factor and bind σ<SUP>28</SUP>, thereby preventing the transcription of <I>fliC</I>.
Several studies regarding the motility of <I>E. coli</I> has shown the expression of the <I>flhDC</I> operon to be crucial [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">23</A>][<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">25</A>]. These focused on insertion sequence (IS) elements upstream of the <I>flhDC</I> regulon. IS are sequences that can be inserted randomly within the DNA and therefore serve as an important factor in the plasticity of the <I>E. coli</I> genome as well as in many other organisms. Generally they do not encode any genes apart from those responsible for their movement within the genome, however, they can also serve as activators of neighboring genes, by disrupting repression or by the formation of hybrid promoters [<A href="https://2010.igem.org/Team:SDU-Denmark/project-t#References" class="external text" rel="nofollow">25</A>]. In the beforementioned studies, bacteria containing an activating IS upstrem of the <I>flhDC</I> operon showed an increased motility compared to bacteria without this IS. It is therefore resonable to asume that by placing a constitutive active promoter in front of the <I>flhDC</I> operon, hyperflagellation will be induced.
</P><H3><SPAN class="mw-headline" id="Our_system"> Our system </SPAN></H3><P>
As we are trying to create a microflow in our system using flagella, we want to increase the flow created by a single bacteria by increasing the number of flagella on the bacteria. However, inducing flagellation in a wildtype <I>E. coli</I> strain requires too much of the bacterial environment. In order to avoid the tight control of the <I>flhDC</I> operon, we insert it into a plasmid backbone containing a constitutive active promoter. As we bypass the original regulations and continuously express the master regulon of the flagellum cascade we hope to see a significant difference in motility of the cells containing our composite part.
As <I>E. coli</I> already express the <I>flhDC</I> operon, we isolate this coding sequence by purifying genomic DNA from the <I>E. coli</I> MG1655 strain and then amplifying the operon using PCR with specially designed <A href="https://2010.igem.org/Team:SDU-Denmark/primers" class="external text" rel="nofollow">primers</A>, before assembly and the insertion into the plasmid backbone  
Furthermore, it will be interesting to see whether the overexpression of the flhDC operon will have an effect on the bacterial growth.
</P><H3><SPAN class="mw-headline" id="Biobrick_design_3"> Biobrick design </SPAN></H3><P>
FlhD,C mutated coding sequence: <A href="http://partsregistry.org/Part:BBa_K343000" class="external text" rel="nofollow">K343000 </A>
FlhDC composite part: <A href="http://partsregistry.org/Part:BBa_K343004" class="external text" rel="nofollow">K343004</A></P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><OL><LI> Li T-D, Gao J, Szoszkiewicz R, Landman U, Riedo E, <A href="http://prb.aps.org/abstract/PRB/v75/i11/e115415" class="external text" rel="nofollow">Structured and viscous water in subnanometer gaps</A>,Phys. Rev. B 75, 115415 (2007)</LI><LI> Samatey FA, et. al.,<A href="http://www.nature.com/nature/journal/v410/n6826/abs/410331a0.html" class="external text" rel="nofollow">Structure of the bacterial flagellar protofilament and implications for a switch for supercoiling</A>Nature 410, 331-337 (15 March 2001)</LI><LI> Macnab RM, <A href="http://www.annualreviews.org/doi/full/10.1146/annurev.micro.57.030502.090832?select23=Choose" class="external text" rel="nofollow">How bacteria assemble flagella</A> Annual Review of Microbiology Vol. 57: 77-100 (October 2003)</LI><LI> Berg HC, <A href="http://www.annualreviews.org/eprint/cDJrS190m62mDRwHrlp9/full/10.1146/annurev.biochem.72.121801.161737" class="external text" rel="nofollow">The rotary motor of bacterial flagella</A> Annual Review of Biochemistry Vol. 72: 19-54 (July 2003)</LI><LI> Berg HC, <A href="http://www.ncbi.nlm.nih.gov/pubmed/1098551" class="external text" rel="nofollow">Chemotaxis in bacteria</A> Annu Rev Biophys Bioeng. 1975;4(00):119-36.</LI><LI> Zhao J, Parkinson JS, <A href="http://jb.asm.org/cgi/content/abstract/188/9/3299" class="external text" rel="nofollow">Mutational Analysis of the Chemoreceptor-Coupling Domain of the Escherichia coli Chemotaxis Signaling Kinase CheA </A>, Journal of Bacteriology, May 2006, p. 3299-3307, Vol. 188, No. 9</LI><LI> Sarkar MK, Paul K, Blair D,<A href="http://www.pnas.org/content/early/2010/04/26/1000935107.short" class="external text" rel="nofollow">Chemotaxis signaling protein CheY binds to the rotor protein FliN to control the direction of flagellar rotation in <I>Escherichia coli</I></A>, Published online before print May 3, 2010, doi: 10.1073/pnas.1000935107</LI><LI> Hess JF, Oosawa K, Kaplan N, Simon MI, <A href="http://www.ncbi.nlm.nih.gov/pubmed/3280143" class="external text" rel="nofollow">Phosphorylation of three proteins in the signaling pathway of bacterial chemotaxis.</A>, Cell. 1988 Apr 8;53(1):79-87.</LI><LI> Trivedi VD, Spudich JL, <A href="http://www.ncbi.nlm.nih.gov/pubmed/14636056" class="external text" rel="nofollow">Photostimulation of a sensory rhodopsin II/HtrII/Tsr fusion chimera activates CheA-autophosphorylation and CheY-phosphotransfer in vitro.</A>, Biochemistry. 2003 Dec 2;42(47):13887-92.</LI><LI> Jung K-H, Spudich EN, Trivedi VD, Spudich JL, <A href="http://jb.asm.org/cgi/content/short/183/21/6365" class="external text" rel="nofollow">An Archaeal Photosignal-Transducing Module Mediates Phototaxis in Escherichia coli</A>, Journal of Bacteriology, November 2001, p. 6365-6371, Vol. 183, No. 21</LI><LI> Spudich JL, Yang CS, Jung KH, Spudich EN <A href="http://www.ncbi.nlm.nih.gov/pubmed/11031241" class="external text" rel="nofollow">Retinylidene proteins: structures and functions from archaea to humans.</A>, Annu Rev Cell Dev Biol. 2000;16:365-92.</LI><LI> Hoff WD, Jung KH, Spudich JL. <A href="http://www.ncbi.nlm.nih.gov/pubmed/9241419?dopt=Abstract" class="external text" rel="nofollow">Molecular mechanism of photosignaling by archaeal sensory rhodopsins.</A>, Annu Rev Biophys Biomol Struct. 1997;26:223-58</LI><LI> Stoeckenius W, Bogomolni, RA <A href="http://www.annualreviews.org/doi/abs/10.1146/annurev.bi.51.070182.003103" class="external text" rel="nofollow">Bacteriorhodopsin and Related Pigments of Halobacteria</A>, Annual Review of Biochemistry Vol. 51: 587-616 (July 1982)</LI><LI> Möglich A, Yang X, Ayers RA, Moffat K, <A href="http://www.annualreviews.org/doi/abs/10.1146/annurev-arplant-042809-112259?journalCode=arplant" class="external text" rel="nofollow">Structure and Function of Plant Photoreceptors</A>, Annual Review of Plant Biolog Vol. 61: 21-47 (June 2010)</LI><LI> Klassen JL, <A href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0011257" class="external text" rel="nofollow">Phylogenetic and Evolutionary Patterns in Microbial Carotenoid Biosynthesis Are Revealed by Comparative Genomics.</A> PLoS ONE 5(6): e11257. doi:10.1371/journal.pone.0011257 (2010)</LI><LI> Moore T, <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1254511/" class="external text" rel="nofollow">Vitamin A and carotene</A>, Biochem J. 1930; 24(3): 692–702.</LI><LI> Wald G, Dowling JE, [Vitamin A deficiency and nigth blindness] , Proc Natl Acad Sci U S A. 1958 July 15; 44(7): 648–661</LI><LI> von Lintig J, Vogt K, <A href="http://www.jbc.org/content/275/16/11915" class="external text" rel="nofollow">Filling the Gap in Vitamin A Research: Molecular Identification of An Enzyme Cleaving Beta-carotene to Retinal</A> Journal of Biological Chemistry (ASBMB) 275 (16): 11915–11920 (2000)</LI><LI> Josenhans, C. and Suerbaum, S. (2002) <A href="http://www.ncbi.nlm.nih.gov/pubmed/12008914" class="external text" rel="nofollow">The role of motility as a virulence factor in bacteria.</A> Int. J. Med. Microbiol. 291, 605-614 </LI><LI> Wang, S. <I>et al.</I> (2006) <A href="http://www.sciencedirect.com/science?_ob=MImg&amp;_imagekey=B6WK7-4HMGKJ0-D-H&amp;_cdi=6899&amp;_user=644074&amp;_pii=S0022283605014063&amp;_origin=search&amp;_coverDate=01%2F27%2F2006&amp;_sk=996449995&amp;view=c&amp;wchp=dGLzVzz-zSkzS&amp;md5=6454221c64ea21917221df6a2bcfaaaa&amp;ie=/sdarticle.pdf" class="external text" rel="nofollow">Structure of the Escherichia coli FlhDC Complex, a Prokaryotic Heteromeric Regulator of Transcription.</A> Journ. of mol. Biol. 355, 4, 798-808 </LI><LI> Claret, L. and Hughes, C. (2000) <A href="http://www.sciencedirect.com/science?_ob=MImg&amp;_imagekey=B6WK7-45M7T8W-51-1&amp;_cdi=6899&amp;_user=644074&amp;_pii=S0022283600941494&amp;_origin=search&amp;_coverDate=11%2F03%2F2000&amp;_sk=996969995&amp;view=c&amp;wchp=dGLbVzW-zSkzV&amp;md5=25cecb82828b382819b79c207eaaf63b&amp;ie=/sdarticle.pdf" class="external text" rel="nofollow">Functions of the Subunits in the FlhD<SUB>2</SUB>C<SUB>2</SUB> Transcriptional Master Regulator of Bacterial Flagellum Biogenesis and Swarming.</A> J. Mol. Biol. 303, 467-478. </LI><LI> Li, C., Louise, C.J., Shi, W. and Adler, J. (1993) <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC204508/pdf/jbacter00050-0069.pdf" class="external text" rel="nofollow">Adverse Conditions Which Cause Lack of Flagella in <I>Escherichia coli</I>.</A> J. Bacteriol. 175,2229-2235. </LI><LI> Gauger, E.J. <I>et al</I> (2007) <A href="http://iai.asm.org/cgi/reprint/75/7/3315" class="external text" rel="nofollow">Role of Motility and the <I>flhDC Operon</I> in <I>Escherichia coli</I> MG1655 Colonization of the Mouse Intestine.</A> Infection and Immunity. vol. 75, No. 7. p3315–3324</LI><LI> Wang S, Fleming RT, Westbrook EM, Matsumura P, McKay DB <A href="http://www.ncbi.nlm.nih.gov/pubmed/16337229" class="external text" rel="nofollow">Structure of the Escherichia coli FlhDC complex, a prokaryotic heteromeric regulator of transcription.</A> J Mol Biol. 2006 Jan 27;355(4):798-808. Epub 2005 Nov 22.
</LI><LI> Baker, C. S. <I>et al</I> (2004) <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC524886/pdf/0575-04.pdf" class="external text" rel="nofollow">Increased Motility of Escherichia coli by Insertion Sequence Element Integration into the Regulatory Region of the flhD Operon</A> Journal of Bacteriology, Vol 186. No22. p7529–7537</LI></OL></DIV><DIV id="rightcolumn"><P>And now for all of the truly brainy stuff!
</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Theory"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Theory</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Phototaxis"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Phototaxis</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Bacterial_flagellar_motility"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Bacterial flagellar motility</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Photosensor"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Photosensor</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#BioBrick_design"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">BioBrick design</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-6"><A href="#Retinal_Generator"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Retinal Generator</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Retinal_requirements_of_light-sensing_proteins"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Retinal requirements of light-sensing proteins</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Retinal_biosynthesis"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Retinal biosynthesis</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#BioBrick_design_2"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">BioBrick design</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Further_use_of_the_retinal_BioBrick"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">Further use of the retinal BioBrick</SPAN></A></LI><LI class="toclevel-4 tocsection-11"><A href="#Role_in_light-based_signal_transduction"><SPAN class="tocnumber">1.2.4.1</SPAN><SPAN class="toctext">Role in light-based signal transduction</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Hyperflagellation"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Hyperflagellation</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Background"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Our_system"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Our system</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Biobrick_design_3"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">Biobrick design</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#References"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:SDU-Denmark/project-t">http://2010.igem.org/Team:SDU-Denmark/project-t</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SDU-Denmark/project-t" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SDU-Denmark/project-t" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SDU-Denmark/project-t&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SDU-Denmark/project-t&amp;oldid=201607" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>