<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SDU-Denmark_safety-d"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SDU-Denmark/safety-d">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SDU-Denmark/safety-d&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SDU-Denmark/safety-d&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SDU-Denmark/safety-d&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SDU-Denmark/safety-d" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SDU-Denmark/safety-d</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="header"><DIV id="return"><A href="https://2010.igem.org/">Back to iGEM 2010</A></DIV></DIV><DIV id="navigation"><UL id="topnav"><LI><A href="/Team:SDU-Denmark">               Home</A></LI><LI><A href="#">Team</A><SPAN><A href="/Team:SDU-Denmark/team">Students</A>|
            <A href="/Team:SDU-Denmark/team2">Advisors</A>|
          <A href="/Team:SDU-Denmark/team3">Sponsors</A>|
            <A href="/Team:SDU-Denmark/team4">Collaborators</A></SPAN></LI><LI><A href="#">Project</A><SPAN><A href="/Team:SDU-Denmark/project-i">Idea</A>|
            <A href="/Team:SDU-Denmark/project-t">Theory</A>|
            <A href="/Team:SDU-Denmark/project-activities">Activities</A>|
            <A href="/Team:SDU-Denmark/parts">Parts</A>|
            <A href="/Team:SDU-Denmark/project-r">Result summary</A>|
            <A href="/Team:SDU-Denmark/project-p">Characterization</A></SPAN></LI><LI><A href="#">Modelling</A><SPAN><A href="/Team:SDU-Denmark/project-m">Physical</A>|
            <A href="/Team:SDU-Denmark/project-bc">Biochemical</A></SPAN></LI><LI><A href="#">Safety</A><SPAN><A href="/Team:SDU-Denmark/safety-a">Overview</A>|
            <A href="/Team:SDU-Denmark/safety-b">Project and Researcher Safety</A>|
            <A href="/Team:SDU-Denmark/safety-c">Laws and Guidelines</A>|
            <A href="/Team:SDU-Denmark/safety-d">Watermark</A></SPAN></LI><LI><A href="#">Notebook</A><SPAN><A href="/Team:SDU-Denmark/notebook">Progress</A>|
            <A href="/Team:SDU-Denmark/labnotes0">Lab notes</A>|
            <A href="/Team:SDU-Denmark/protocols">Protocols</A></SPAN></LI><LI><A target="blank" href="http://igem.sdu.dk">Blog</A></LI></UL></DIV><DIV id="subnavi"><DIV id="leftcolumn"><H1><SPAN class="mw-headline" id="Useful_ideas_for_Improving_Safety">Useful ideas for Improving Safety</SPAN></H1><H2><SPAN class="mw-headline" id="Watermarking">Watermarking</SPAN></H2><P>To increase public safety we propose to introduce a water-marking standard. 
Following the example of J. Craig Venter, who, in may 2010, created the first watermark in a bacteria, containing several readable messages, we propose to create a watermarking standard to increase the safety of the environment, as well as the safety of the community at large. &lt;vr&gt;
But why should we consider creating and using a watermarking standard for work in synthetic biology? Let us consider the following example: 
A company has created a synthetic organism capable of absorbing harmful substances from the ground. The government, eager to help clean polluted ground, releases the bacteria into the wild, confident that the bacteria will not pose any threat to the environment. These bacteria do not, however, react as planned. Instead of absorbing only the harmful substances from the ground, they transfer the substances to other organisms, or simply run amok spreading themselves in an uncontrolled manner, causing harm instead of good. Let us imagine that this happens close to the nation's border. The government in the neighboring state finds itself with a unexplained biological phenomenon, possibly causing great harm. 
Should the rogue bacteria contain engineered, watermarked parts, it would be a small matter to have the parts sequenced, thus the government would be able to easily access all the relevant information on the rogue bacteria, contact the manufacturers and would know how to stop it.
</P><H2><SPAN class="mw-headline" id="The_watermark">The watermark</SPAN></H2><P><B>We believe that a watermark should contain the following:</B></P><P>
I.        A 12 nucleotide ‘license’
</P><P><B>We have set the following criteria for a good watermark:</B></P><P>    
I.        It should contain the creating team’s ‘license’
</P><P>II.        It should not interfere with the other functions of the part
</P><P>III.        It should be persistent in the plasmid, i.e. not be removed from the plasmid due to natural evolution
</P><P>IV.        It should be easy to find, easy to read and easy to insert by the developing team
</P><P><B>This 12 nucleotide license would then refer to a database where the following information should be available:</B></P><P>
I.       It should contain information on the creating team
</P><P>II.        The name of the part
</P><P>III.        A description of the part
</P><P>IV.        A risk-assessment of the part by the creating team
</P><P>V.       Information on how to neutralize organism, and, if available, kill-code
</P><H2><SPAN class="mw-headline" id="Placing_the_watermark">Placing the watermark</SPAN></H2><P>The initial thought was to insert the watermark into the genome of the bacteria, so as to increase the stability of the watermark in the bacteria. However, as all parts inserted into the bacteria are placed in plasmids, it would make no sense to insert the watermark into the genome. The bacteria could transfer their plasmids to other bacteria, and retain their watermark, and the watermark would have become useless, persisting in bacteria that have no modified material at all. 
Also, the thought was to mark the bacteria, and not the parts. Thus we could have a single watermark to cover the entire modified organism. However, this presents us with many of the above problems. Should anyone encounter  a rogue bacteria which has lost some of its plasmids, the watermark would be useless. It would only cover the entire modified system, and should some, or all bacteria have discarded the plasmids, the watermark would be useless. We therefore propose to mark every plasmid, before and after the coding sequence. The watermark will therefore be split into two separate components. The team creates a watermark for every part, and inserts it into the plasmid, before and after the coding sequence. 
We chose to divide the code into two separate parts for two reasons. Firstly, being split into two equally long parts at each end of the coding sequence ensures symmetry , which again should help make insertion of a watermark easy, as this will reduce the complexity of the primers we need to design to insert them. Secondly this ensures that identification is done easily, since the combination of nucleotides of the sequence from E to X and from S to P is known. Thus we effectively increase the sequence we search for from 6 to 26 nucleotides. Also this means that the risk of a naturally occurring combination of nucleotides, identical to the synthetically created one decreases. 
</P><H3><SPAN class="mw-headline" id="Alternative_placement">Alternative placement</SPAN></H3><P>We also thought about the possibility of placing the watermark between the cutting sites of E and X, and S and P respectively. Since this area is a spacing area, it might be a good spot to insert the nucleotide combination we want. Difficulties in this might cause disturbance. Another idea was to expand the restriction sites E-X and S-P, but we are uncertain whether this might disturb the functionality of the part.
</P><H2><SPAN class="mw-headline" id="Size_and_design">Size and design</SPAN></H2><P>Placing the watermark after the restriction sites, it should be relatively small, as to not interfere with the functionality of the part into which it is inserted. We propose that the watermark should be 12 nucleotides, divided in two groups of six which will allow for 16.777.216 combinations of A, C, G and T in total. 
The watermark must not contain any restriction-enzymes or stop codons. If the watermark accidentally contained a restriction-enzyme or a stop codon, the watermark would interfere with the function of the part into which it is inserted, and would likely render the part useless. This severely restricts the number of combinations we can use.
Ideally we would have liked to use more nucleotides, as we would have been able to generate more combinations. But the watermark should be as small as possible as not to interfere with the functions (i.e. cause a frame shift) of the part and not make the design of primers unduly complicated. We could make relatively long watermarks to satisfy our need for a very large number of possible combinations, but it would make the design of the necessary primers extremely complicated, and would go against our goal of making the insertion of watermarks as small and easy a procedure as possible. We believe that the best compromise between the amount of combinations and the ease of insertion would be at around 12 nucleotides.
</P><H2><SPAN class="mw-headline" id="License">License</SPAN></H2><P>Every team and lab/company which works with synthetic biology will be assigned a specific “license plate”, unique to them and being their ‘ID’ on parts-registry. An extension to parts registry’s search function should then be added. If its database contains information on each “license”, one could easily find information on a foreign organism, in case they found it in the wild (that is, anywhere else than the lab), and had it sequenced afterwards. Upon entering the parts-registry, one should then be able to enter the license and gain access to information on the parts that the team / lab / company has created. Anyone should be able to enter the license code into the parts-registry and gain access to all information on the creating team, the parts created by the team, and of course some contact information, in order to seek advice.
</P><H2><SPAN class="mw-headline" id="Procedure">Procedure</SPAN></H2><P>Retrieval of the watermark should be easy, in the event of a rogue bacteria spreading havoc in the environment. Standardized placement of the watermark within the part should make it possible to easily retrieving the watermark through sequencing. Thus, if a rogue bacteria is discovered, it should be a simple matter to sequence the part, and obtain the license. Then it is just a matter of consulting the parts-registry to access all available information on the part and on how to contact the creating team. Getting a bacteria sequenced is a technology which is easy to access, and a lot of companies worldwide offers sequencing of bacteria, in addition it is a relative speedy process, which further makes the identification of a malign bacteria, and its properties relatively easy.
</P><H2><SPAN class="mw-headline" id="Questions_for_assessing_parts">Questions for assessing parts</SPAN></H2><P>Here is a list of questions that we believe the team should ask itself and answer, to its best ability when considering what information is relevant for the safety and security of their project. 
</P><H3><SPAN class="mw-headline" id="General_use">General use</SPAN></H3><P>What type of lab should work with this BioBrick be done in? 
Is any special care needed when working with this BioBrick in addition to ordinary laboratory work-safety protocols? 
Potential pathogenicity  
Does this BioBrick produce any product that is toxic to plants or animals? 
Could a host with this BioBrick produce sufficient amounts of an otherwise non-toxic product to toxic levels? 
Is the gene that the BioBrick codes for associated with disease processes or infection in humans? 
Does the gene that the BioBrick codes for have homologs in known pathogenic bacteria? What are these genes’ functions? 
Does this BioBrick produce any product that can regulate the immune system in animals or humans? </P><H3><SPAN class="mw-headline" id="Environmental_impact">Environmental impact</SPAN></H3><P>Does this BioBrick produce anything that plays an significant role in environmental processes? 
Does this BioBricks increase it’s hosts ability to outmatch naturally occurring bacteria? Does it disrupt natural  occurring symbiosis? 
Does this BioBrick increase it’s host’s ability to replicate? 
Does this BioBrick increase it’s host’s ability to spread? 
Does this BioBrick increase it’s host’s ability to survive outside laboratory conditions? 
Does this BioBrick create something that might be toxic to it’s environment? </P><H3><SPAN class="mw-headline" id="Possible_malign_use">Possible malign use</SPAN></H3><P>Can host’s with this BioBrick survive storage conditions (in pressured conditions, under alternative temperatures and in large containers?) 
Can your chosen host be arosoled? 
Does this BioBrick increase the host’s ability to vaporize?   
Does this BioBrick increase the host’s ability to create spores? 
Does this BioBrick produce any product that can regulate the immune system in animals or humans? 
Is this BioBrick in anyway pathogenic in animals or humans or towards plants? </P><H3><SPAN class="mw-headline" id="Construct_notes">Construct notes</SPAN></H3><P>What is the origin of the genetic material used? What does the the genetic materiale do in this origin? Is there uncertainty about the genetical materials function? 
What modifications were done on the genetic materiale before insertion? If anything was modified, what function do you hope to achieve?
What vector did you use? Which antibiotic resistance were involved? Which protocol was used to insert the vector? 
What is the stability of the insert with respect to genetic traits? 
How easily can the insert transfer to other bacteria or lifeforms? 
Where there safer alternatives to achieve this function? Where there safer alternatives to the host organism and vector used? 
Is your construct watermarked? </P><P>
The above list of question is more or less derived from a more throughout list of questions relevant to assessing the safety and securty of the parts, see appendix II.
</P><H2><SPAN class="mw-headline" id="Further_Information">Further Information</SPAN></H2><H3><SPAN class="mw-headline" id="Risk-assessment_for_organism">Risk-assessment for organism</SPAN></H3><P>Should the part, or a number of parts, be inserted into an organism the team should perform a risk-assessment and make it available on the parts-registry. In some countries, it is mandatory to submit a risk-assessment prior to engaging in a project involving synthetic biology, so we believe that any risk-assessments should be made public through parts-registry.
</P><H3><SPAN class="mw-headline" id="Inclusion_of_copyright_information.3F">Inclusion of copyright information?</SPAN></H3><P>We do not believe in any form of copyright prohibition. We believe in an open-source approach to the field of synthetic biology, as in iGEM. Any copy-right prohibitions would only stall the progress in this most vital field of science. We believe that any and all information on created parts, and experience with these parts in particular organisms, should be shared freely.
</P><H3><SPAN class="mw-headline" id="Information_on_how_to_neutralize_bacteria">Information on how to neutralize bacteria</SPAN></H3><P>This clause is intended as security measure. Should the bacteria be released into the environment, the parts-registry site should contain information on how to neutralize the bacteria. If the bacteria has an kill-code inserted, the site should describe how to enact the self-destruct mechanism.
</P><H2><SPAN class="mw-headline" id="Anticipated_problems">Anticipated problems</SPAN></H2><H3><SPAN class="mw-headline" id="Code_deterioration">Code deterioration</SPAN></H3><P>The code will deteriorate over time due to mutation. This could prove to be a serious problem should watermarking become an integrated part of synthetic biology. Should the genetic watermark deteriorate to the point where one is no longer able to read it, it would not constitute any kind of safety measure, being able to tell that the part was likely to have been made artificially being the only thing we would be able to tell.
Knowing that data will deteriorate, it may be impossible to determine whether the watermark found in rogue bacteria is authentic or a degenerate. The deterioration is however slow and arbitrary. Our code is so small, that the change that any nucleotide associated with the watermark is going to mutate is very limited. The chances of finding the authentic code intact should be very good.
</P><H3><SPAN class="mw-headline" id="The_open_source_approach">The open source approach</SPAN></H3><P>Sharing all this information on creating new synthetic parts that can be inserted into living organisms, also means that people with harmful agendas have access to this knowledge. This naturally means that wrongful actions are possible by use of synthetic biology, but we do not think that this should stand in the way of all the possibilities synthetic biology holds. Also, the more we know about how to create synthetic DNA strands, the better we are equipped if any harmful incident should occur. As of now, it might also just be easier to drop a bomb or send out letters of anthrax.
Should a bacteria be used for a malign purpose it would be quite easy to insert a false watermark to blame others. So we need to keep in mind that a plot could be made against someone. However, if anyone was interested in harming as many as possible, this person probably wouldn’t care about watermarking at all. This also means that we cannot expect watermarking to play a part in any legal case.
</P></DIV><DIV id="rightcolumn"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Useful_ideas_for_Improving_Safety"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Useful ideas for Improving Safety</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Watermarking"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Watermarking</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#The_watermark"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">The watermark</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Placing_the_watermark"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Placing the watermark</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Alternative_placement"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Alternative placement</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-6"><A href="#Size_and_design"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Size and design</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#License"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">License</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Procedure"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Procedure</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Questions_for_assessing_parts"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Questions for assessing parts</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#General_use"><SPAN class="tocnumber">1.7.1</SPAN><SPAN class="toctext">General use</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Environmental_impact"><SPAN class="tocnumber">1.7.2</SPAN><SPAN class="toctext">Environmental impact</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Possible_malign_use"><SPAN class="tocnumber">1.7.3</SPAN><SPAN class="toctext">Possible malign use</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Construct_notes"><SPAN class="tocnumber">1.7.4</SPAN><SPAN class="toctext">Construct notes</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Further_Information"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">Further Information</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Risk-assessment_for_organism"><SPAN class="tocnumber">1.8.1</SPAN><SPAN class="toctext">Risk-assessment for organism</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Inclusion_of_copyright_information.3F"><SPAN class="tocnumber">1.8.2</SPAN><SPAN class="toctext">Inclusion of copyright information?</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Information_on_how_to_neutralize_bacteria"><SPAN class="tocnumber">1.8.3</SPAN><SPAN class="toctext">Information on how to neutralize bacteria</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Anticipated_problems"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">Anticipated problems</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Code_deterioration"><SPAN class="tocnumber">1.9.1</SPAN><SPAN class="toctext">Code deterioration</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#The_open_source_approach"><SPAN class="tocnumber">1.9.2</SPAN><SPAN class="toctext">The open source approach</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:SDU-Denmark/safety-d">http://2010.igem.org/Team:SDU-Denmark/safety-d</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SDU-Denmark/safety-d" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SDU-Denmark/safety-d" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SDU-Denmark/safety-d&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SDU-Denmark/safety-d&amp;oldid=167149" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>