<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UNSW_Australia_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UNSW Australia/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="navbar"><DIV id="menu"><DIV class="main-menu"><LABEL for="navToggle" id="navigation">Navigation</LABEL></DIV><DIV class="main-menu first-menu"><BUTTON id="home" class="dropbutton"><A href="https://2018.igem.org/Team:UNSW_Australia">Home</A></BUTTON></DIV><DIV class="main-menu"><BUTTON id="project" class="dropbutton">Project</BUTTON><DIV id="project-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Description">Description</A><A href="https://2018.igem.org/Team:UNSW_Australia/Design">Design</A><A href="https://2018.igem.org/Team:UNSW_Australia/Experiments">Experimental Protocols</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab" class="submenu-header">Lab Overview</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/Cloning" class="submenu-sub submenu-sub-first">Cloning</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/Protein" class="submenu-sub">Protein Production</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/Assembly" class="submenu-sub">Assembly</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/FRET" class="submenu-sub">FRET</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/Assays" class="submenu-sub">Enzyme Assays</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/Plants" class="submenu-sub submenu-sub-last">Plants</A><A href="https://2018.igem.org/Team:UNSW_Australia/Results">Results</A><A href="https://2018.igem.org/Team:UNSW_Australia/InterLab">Interlab</A></DIV></DIV><DIV class="main-menu"><BUTTON id="modelling" class="dropbutton">Modelling</BUTTON><DIV id="modelling-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Model">Overview</A><A href="https://2018.igem.org/Team:UNSW_Australia/Model/EKD">Enzyme Kinetics &amp; Diffusion</A><A href="https://2018.igem.org/Team:UNSW_Australia/Model/MD">Molecular Dynamics</A></DIV></DIV><DIV class="main-menu"><BUTTON id="parts" class="dropbutton">Parts</BUTTON><DIV id="parts-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Parts">Overview</A><A href="https://2018.igem.org/Team:UNSW_Australia/Basic_Part">Basic Part</A><A href="https://2018.igem.org/Team:UNSW_Australia/Improve">Improved Part</A><A href="https://2018.igem.org/Team:UNSW_Australia/Software">Software</A></DIV></DIV><DIV class="main-menu"><BUTTON id="hp" class="dropbutton">Human Practices</BUTTON><DIV id="hp-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Human_Practices">Overview</A><A href="https://2018.igem.org/Team:UNSW_Australia/Human_Practices/Commercialisation">Commercialisation</A><A href="https://2018.igem.org/Team:UNSW_Australia/Public_Engagement">Education &amp; Engagement</A><A href="https://2018.igem.org/Team:UNSW_Australia/Human_Practices/Law">Law &amp; Regulation</A><A href="https://2018.igem.org/Team:UNSW_Australia/Human_Practices/Integrate">Integration</A></DIV></DIV><DIV class="main-menu"><BUTTON id="people" class="dropbutton">People</BUTTON><DIV id="people-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Team">Team</A><A href="https://2018.igem.org/Team:UNSW_Australia/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:UNSW_Australia/Attributions">Attributions</A><A href="https://2018.igem.org/Team:UNSW_Australia/Journal">Experts</A></DIV></DIV><DIV class="main-menu"><BUTTON id="doc" class="dropbutton">Documentation</BUTTON><DIV id="doc-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Notebook">Notebook</A><A href="https://2018.igem.org/Team:UNSW_Australia/Safety">Safety</A></DIV></DIV><DIV class="main-menu"><BUTTON id="judging" class="dropbutton">Judging</BUTTON><DIV id="judging-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Medals">Medals</A><A target="_blank" href="https://igem.org/2018_Judging_Form?id=2710">Judging Form</A></DIV></DIV></DIV></DIV><DIV id="footer"><P>© 2018 UNSW Australia iGEM</P></DIV><DIV class="header" id="design-header"><DIV class="header-img-txt"><H1 class="shadow-text main-heading">Design</H1></DIV></DIV><DIV id="design-content" class="to-load"><DIV class="box"><H2>Overview</H2><P>When developing our Assemblase system, we strived to follow an engineering design procedure. As design is the first step of the ‘Design-Build-Test’ engineering paradigm, we felt it was important to consider all aspects of the Assemblase system to minimise the occurrence of downstream problems during the building and testing phases. Thus, we started our design process by researching and defining the need for our scaffold tool in the market, which then allowed us to develop a list of requirements to which our system should adhere. Following this, we went through design iterations, learning from modelling data and experimental results to ultimately arrive upon our finalised Assemblase scaffold design.</P></DIV><H2>Defining a Need</H2><P>In industry and research there has been a growing interest in the construction of novel metabolic pathways for the production of cheap and renewable products, as an alternative to synthetic chemistry<SUP>1</SUP>. However, as these biocatalytic pathways become more complex their productivity is diminished due to undesirable side reactions and low reaction rates.</P><P>One way that this can be addressed is through the development of scaffolds that co-localise enzymes from a multi-step pathway. Spatially organising enzymes to generate high local concentrations of substrate enhances metabolic flux and reduces the diversion of pathway intermediates into competing pathways<SUP>2</SUP>. This ultimately can lead to an increase in product titre, which has significant implications for complex novel metabolic pathways.</P><P>This approach mimics enzyme clustering that has frequently been observed in nature<SUP>3</SUP>. However, previous attempts to replicate this natural enzyme organisation have fallen short, with no commercially-viable modular tool available to facilitate enzyme clustering for the production of valuable metabolites. As such, the UNSW iGEM team decided to reflect on these past attempts and compile a list of requirements that would enable us to develop a versatile, commercially viable tool that could be used in laboratories and industry worldwide.</P><H2>Functional Requirements</H2><P>Having decided to develop an enzyme scaffold for use in biocatalytic systems, the team worked to compile a list of functional requirements (FRs) that our system would have to meet:</P><P>Given our research, it was evident that the main goal of our scaffold was to increase the product titre from a given synthetic enzyme pathway (FR<SUB>1</SUB>: Increase product titre). With previous studies showing that a considerable increase in titre could be achieved through the utilisation of synthetically scaffolded proteins<SUP>4,5</SUP>, this seemed like an achievable goal. However, this main requirement of increasing yield was relatively broad, so breaking down this overall function into sub-requirements (FR<SUB>1.1</SUB>, FR<SUB>1.2</SUB> and FR<SUB>1.3</SUB>) was necessary<SUP>6</SUP>:</P><TABLE id="FR1" class="design-table"><TBODY><TR><TH>Current Challenge</TH><TH>Criteria to Satisfy this problem</TH><TH>Naming Convention (relating to the flowchart)</TH></TR><TR><TD rowspan="2">Engineered metabolic pathways often suffer from flux imbalances<SUP>5</SUP>.</TD><TD>Metabolic flux rates of the system could be increased by increasing the concentration of intermediates<SUP>8</SUP>.</TD><TD>FR<SUB>1.2.1</SUB></TD></TR><TR><TD>Decreasing the enzyme concentration required to produce the same product titre would have significant cost saving outcomes in an industrial context.</TD><TD>FR<SUB>1.2.2</SUB></TD></TR><TR><TD>Unintended interactions between synthetic pathways and the cellular environment of the host can have significant implications for the productivity of novel pathways<SUP>9</SUP>.</TD><TD>Considering the benefits observed by the natural channelling of intermediates <I>in vivo</I><SUP>10</SUP>, further lower level requirements can be identified such as:
            <UL><LI>Limiting the diffusion of intermediates into the surroundings</LI><LI>Facilitating the fast turnover of labile or toxic intermediates</LI></UL></TD><TD>FR<SUB>1.1.1</SUB> and FR<SUB>1.1.2</SUB></TD></TR><TR><TD>The critical determinant of scaffold effectiveness has been suggested to be the ability to provide control over reaction stoichiometry<SUP>9</SUP>.</TD><TD>As such the system must provide the capability of optimising enzyme stoichiometry.</TD><TD>FR<SUB>1.3</SUB></TD></TR></TBODY></TABLE><P>On top of these key requirements, the system was also required to have commercial value and, as such, would have to be able to be used in a variety of applications (FR<SUB>2</SUB>: Adapt to different applications):</P><TABLE id="FR2" class="design-table"><TBODY><TR><TH>Current Challenge</TH><TH>Criteria to Satisfy this problem</TH><TH>Naming Convention (relating to the flowchart)</TH></TR><TR><TD rowspan="2">Previous enzyme agglomerates have been observed to only form under certain conditions, such as purinosomes discovered in HeLa cells that were found to cluster only in response to higher purine levels<SUP>11</SUP>. Yet, for the envisioned use of our system, the team determined that the scaffold would have to assemble independent of environmental conditions.</TD><TD>Therefore, it is necessary for the system to be able to function both in vivo and ex vivo environments.
        </TD><TD>FR<SUB>2.2</SUB></TD></TR><TR><TD>Furthermore, the system would have to adapt to utilising a range of different enzymes. This would require enzyme exchange without extensive protein modification (as was done by the <A href="https://2010.igem.org/Team:Slovenia"> 2010 Slovenian iGEM team</A>). This dictates some form of modularity in the system.</TD><TD>FR<SUB>2.1</SUB></TD></TR></TBODY></TABLE><P>Finally, our scaffold system would have to be easy to produce and be utilised by laboratories and other interested parties in the future (FR<SUB>3</SUB>: Facilitate accessible use):</P><TABLE id="FR3" class="design-table"><TBODY><TR><TH>Current Challenge</TH><TH>Criteria to Satisfy this problem</TH><TH>Naming Convention (relating to the flowchart)</TH></TR><TR><TD>The need for a synthetic biology tool to be as simple as possible and be cost effective to assemble.</TD><TD>This can be achieved by having only a minimal number of subunits in the system.</TD><TD>FR<SUB>3.1</SUB></TD></TR><TR><TD rowspan="2">In order to be replicated, a synthetic biology tool would need have clear characterisation.</TD><TD>The system should provide through characterisation of DNA parts.</TD><TD>FR<SUB>3.2</SUB></TD></TR><TR><TD>The system should provide standardised experimental protocols that can be replicated seamlessly.</TD><TD>FR<SUB>3.3</SUB></TD></TR></TBODY></TABLE><H2>Concept Generation</H2><P>Having developed a list of requirements our Assemblase system would have to meet, we moved onto brainstorming different components that would provide the required functions. This included going through various design iterations as a result of consultation with industry and feedback from our experimental and modelling work. Please click on the tabs below to find out about the design process for each of our Assemblase components.</P><DIV onclick="Scaffold()" class="concept-group"><H2 class="shadow-text">Scaffold</H2></DIV><DIV class="scaffold subheading"><P>With the functional requirements of our system in mind, the team assessed how we could construct a scaffolding platform that could add modularity and stability to past enzyme clustering methods. We steered away from the concept of substrate channelling or using physical tunnels which have been shown to be unnecessary<SUP>10</SUP>, and instead focused on the idea of a synthetic molecular scaffold that would co-localise proteins. With research on the stable, modular, self-assembling chaperone proteins called prefoldin having already been conducted by our supervisor, Dominic Glover, we decided to investigate its use as a potential candidate for our scaffold.</P><H3>Iteration 1 – Gamma Prefoldin</H3><P>Prefoldins are generally hexameric molecular chaperone complexes that bind to non-native target proteins in a cell. They have a general role in de novo protein folding in archaea and the biogenesis of cytoskeleton proteins in eukaryotes.</P><P>A distinct variant of archaeal prefoldin (PFD), gamma prefoldin (gPFD), has a unique filamentous structure composed of hundreds of monomeric subunits<SUP>11</SUP>. This protein complex, which originates from hyperthermophilic <I>Methanocaldococcus jannaschii</I>, is involved in binding non-native proteins via distal regions of its highly flexible coiled coils. Our interest in this protein complex as a scaffold, stemmed from the fact that it has been well characterised by the lab our supervisor. Furthermore, Glover’s lab has shown that enzymes can be attached to gPFD sparking an interest in our team to replicate such attachment mechanisms.</P><P>However, the use of gPFD in a scaffold has its limitations, including an inability to control the order of the subunits without using a heterogenous mix of filaments. This contradicts FR<SUB>3.1</SUB> which dictates that a minimal amount of subunits are desired and, as such, this protein complex was deemed inappropriate for our desired scaffold system.</P><H3>Iteration 2 – Alpha and Beta Prefoldin</H3><P>An elegant solution to this was found in another prefoldin variant, derived from archaea, which has just two subunits. This archaeal prefoldin complex is constructed from two alpha and four beta subunits which self-assemble into a jellyfish-like formation, creating a heterohexamer. Like gPFD, this complex originates from a thermophilic archaeon, <I>Methanobacterium thermoautotrophicum</I>, which affords the archaeal prefoldin complex with a relatively high thermostability, a function that falls under the required adaptability of our system (FR<SUB>2.2</SUB>). Furthermore, the six components of this structure allow for the modification and expression of each subunit individually as desired, enabling extensive adaptability of our scaffold to various scenarios.</P><H3>Future developments</H3><P>Our research also led us to consider how we could develop our scaffold into the future, in particular with the use of eukaryotic prefoldin. This is also a heteroheaxamer, but unlike its archaeal partner, this variant is made up of six distinctly unique proteins. This would be beneficial as it would allow for us to easily incorporate six different enzymes onto the one scaffold, rather than just two. In accordance with FR<SUB>2</SUB>, this would enhance the adaptability of our system to work with multiple biocatalytic pathways.</P></DIV><DIV onclick="Attachment()" class="concept-group"><H2 class="shadow-text">Attachment System</H2></DIV><DIV class="attachment subheading"><P>Having decided on the base of the Assemblase system, the team then began to look at the ways in which we could attach the desired enzymes onto this prefoldin scaffold. Most fusion proteins are connected by non-covalent interactions such as dimerising linkers or streptavidin-biotin<SUP>12</SUP>. However, we wanted our system to be highly stable with the enzymes covalently bonded to the scaffold to allow for permanent localisation. This could be achieved with covalent linkers such as inteins or sortases, however these limit molecular topologies, with sortase linkers only occurring between the N and C termini of target proteins.</P><P>Research instead led us to an alternative rapid covalent ligation strategy involving a short polypeptide tag (Spy- or Snoop-Tag) and its reactive domain partner (Spy- or Snoop-Catcher)<SUP>13</SUP>. This attachment mechanism involves the spontaneous formation of an isopeptide bond between the respective Tag and Catcher components, with Snoop only reacting with Snoop, and Spy only reacting with Spy. This bond has been shown to be robust under a whole range of reaction conditions<SUP>13</SUP>, fulfilling the requirement of functional adaptability (FR<SUB>2</SUB>) as the enzymes would remain attached to the scaffold irrespective of the metabolic pathway.</P><H3>Future developments</H3><P>If we moved to using the six-subunit eukaryotic prefoldin for our scaffold we would need to employ a further range of attachment systems that do not cross react. Although we could resort to using the afore mentioned systems, such as dimerizing linkers or sortase, sticking with similar Tag-Catcher arrangement would add value to the assembly and function of the system. This is feasible through the use of a third covalent ligation strategy, SdyCatcher<SUP>14</SUP>. The team will look out for other developments in the Tag-Catcher space.</P></DIV><DIV onclick="Enzymes()" class="concept-group"><H2 class="shadow-text">Enzymes</H2></DIV><DIV class="enzymes subheading"><P>Having designed the scaffold and conjugation strategy, we then considered how to demonstrate the functionality of our approach. <A href="https://2018.igem.org/Team:UNSW_Australia/Model/EKD">Mathematical modelling</A> indicated that localising enzymes would result in an increase in product titre as required by our function specifications (FR<SUB>1</SUB>). Yet, we needed to develop protocols that would allow us to demonstrate this in the <A href="https://2018.igem.org/Team:UNSW_Australia/Lab">lab</A>. We used the KEGG database to search through a variety of pathways<SUP>15</SUP>, looking at secondary metabolites and xenobiotic biodegradation to identify a set of enzymes that would produce a useful product. We also reviewed classic exemplars of synthetic pathways that are used in metabolic engineering, such as the production of terpanoids<SUP>16</SUP>. However, through the requirement of reducing complexity (FR<SUB>1.3.1</SUB>), our current design had two subunit types meaning that a most appropriate reaction pathway would be one that only required two enzymes.</P><H4>Iteration 1 – Plastic Degrading Enzymes</H4><P>Initially, it was proposed that we work with novel plastic degrading enzymes, PETase and MHETase. These enzymes were isolated in 2016 from a polyethylene terephthalate (PET) degrading bacterium, <I>Ideonella sakaiensis</I> 201-F6<SUP>17</SUP>. They were found to be involved extracellular enzymatic PET degradation, and their use in metabolic engineering could have widespread impacts for our plastic dependent society. However, quantifying the enzymatic activity of PETase and MHETase was considered to be relatively difficult. A previous iGEM team, <A href="https://2016.igem.org/Team:Tianjin">Tianjin 2016</A>, attempted this with cell-free protein expression which does not directly or clearly show the alteration in product titre. An alternative suggestion was to use electron microscopy to physically measure plastic degradation, however this would have presented challenges for our team regarding training and funding. So, despite the benefits that could come from using such a metabolic pathway in our scaffold, it was evident that using this plastic degrading pathway would not be the best method of achieving proof of concept for our scaffolding system. Instead, the team decided to look for a more feasible pathway that would involve the synthesis of a product that was easier to measure and quantify.</P><H3>Iteration 2 – The IAA Pathway</H3><P>This led us to consider using the indole-3-acetic acid (IAA) biosynthesis pathway:</P><P>This metabolic pathway involves the conversion of tryptophan, a simple amino acid, to IAA using enzymes tryptophan 2-monooxygenase (IaaM) and indole-3-acetamide hydrolase (IaaH)<SUP>18</SUP>. In practice, IAA has a role in promoting plant root growth and has been shown to be chemically labile and easily metabolised by plants, by the <A href="https://2011.igem.org/Team:Imperial_College_London">2011 iGEM Imperial College London team</A>. The biosynthesis of IAA can also easily be assessed in the lab. Previous iGEM teams who have worked on enzyme scaffolding projects, such as <A href="https://2012.igem.org/Team:ZJU-China/project.htm"> ZJU-China 2012 </A> and <A href="https://2015.igem.org/Team:NUDT_CHINA">NUDT-China 2015</A>, used this two-step enzyme reaction and developed protocols, standard curves to quantify IAA biosynthesis. </P><P>On evaluation of this pathway, it was clear to us why previous teams had chosen to work with it, providing the ideal features to satisfy some of our functional requirements:</P><TABLE id="IAA" class="design-table"><TBODY><TR><TH>Feature</TH><TH>Benefit</TH><TH>Function(s) Satisfied</TH></TR><TR><TD>Two-step reaction pathway</TD><TD>This is ideal to work with our two subunit-type archaeal protein, allowing for the attachment of each enzyme to a different subunit.</TD><TD>FR<SUB>3.1</SUB>: Express a minimal number of subunits</TD></TR><TR><TD>Well characterised in literature and iGEM</TD><TD>The depth of research on this pathway and its use by other teams enabled us to reliably highlight the function and benefits of our scaffold.</TD><TD>FR<SUB>3.3</SUB>: Provide standardised experimental procedures. </TD></TR><TR><TD>Easy to measure and quantify products</TD><TD>IAA is relatively easy to measure with assays such as Salkowski and HPLC. Its effect on plant root growth can also be exploited.</TD><TD>FR<SUB>3</SUB>: Facilitate accessible use.</TD></TR></TBODY></TABLE><H3>Iteration 3 – Taxol Side-Chain Synthesis</H3><P>Although we decided to use the IAA biosynthesis pathway as a proof of concept for our experimental work, <A href="https://2018.igem.org/Team:UNSW_Australia/Human_Practices/Commercialisation">consultation with industry</A> led us to realise that IAA itself was not widely used. Therefore, the team wanted to find an enzyme pathway we could utilise in the future that would have commercial value while still demonstrating the prime functionality of our scaffold system. This led us to the Taxol side-chain synthesis pathway, which involves the synthesis of the side-chain of the anticancer drug Taxol, by the enzymes phenylalanine aminomutase (PAM) and tyrocidine synthase I (S563A) (TycA-S563A). These enzymes have a slow turnover rate which would be improved through co-localisation, making the pathway a great candidate for our Assemblase scaffold system. It was therefore decided that these enzymes would be used in the place of IaaM and IaaH in our <A href="https://2018.igem.org/Team:UNSW_Australia/Model/EKD">Enzyme Kinetics and Diffusion model</A> when investigating the yield rates afforded by our scaffold system.</P><H3>Future Developments</H3><P>Although most of our design iterations in this section were focused on identifying enzymes that would be attached to the Assemblase scaffold, it should be noted that the system has the potential to accommodate a range of different proteins. Our focus on enzyme attachment stemmed from the identified need to improve product yield efficiencies for metabolic pathways. However, it would be just as simple to fuse desired proteins to the Snoop or Spy Tags opening up a range of <A href="https://2018.igem.org/Team:UNSW_Australia/Description">future applications</A>
for our scaffold. One way we aimed to show this was through the attachment of fluorescent proteins for <A href="https://2018.igem.org/Team:UNSW_Australia/Lab/FRET">the FRET experiments</A></P></DIV><H2>Experimental Design</H2><P>Having determined the protein components that make up the scaffold, the iGEM UNSW team then moved on to planning out how to experimentally characterise the Assemblase system. It was necessary to first determine the experimental tests we would perform before moving on to building the system, since experimental design informed the design of our <A href="https://2018.igem.org/Team:UNSW_Australia/Lab/Cloning">DNA sequences</A> for each part.</P><H3>1. Assembly</H3><P>Firstly, the team wanted to develop a <A href="https://2018.igem.org/Team:UNSW_Australia/Lab/Assembly">series of practical experiments</A> that would be able to show how well each of the assembled.</P><P>This included SDS PAGE, an analytical method for separating charged molecules by their molecular masses, and size exclusion chromatography (SEC), a method which allows for the separation of large protein complexes based on size. Using the known sizes of alpha prefoldin and beta prefoldin subunits (15kDa and 13.8kDa, respectively) and the assembled prefoldin hexamer (87kDa)<SUP>19</SUP>, performing SDS PAGE and SEC with our individual subunits versus assembled hexamer would allow us to investigate whether our subunits are correctly assembling to form the scaffold hexamer. This could also be done using our alpha/beta prefoldin-Spy/Snoop catcher fusion proteins, to assess whether the fusion of catcher components to the prefoldin subunits would affect their ability to self-assemble.</P><H3>2. FRET</H3><P>Forster resonance energy transfer (FRET) is a mechanism that describes the energy transfer between two light sensitive molecules. It is widely used to study the dynamics of proteins, particularly by quantifying the distance between two subsequent proteins. In deciding that we wanted to use FRET, we prepared the fluorescent proteins mVenus and mCerulean to be attached to the Assemblase scaffold in order to determine their distance apart once assembled on our structure. This would enable us to get a good understanding of how enzymes in our scaffold would be spatially organised.</P><H3>3. Enzyme Assays</H3><P>On top of being able to characterise the spatial organisation and assembly of our Assemblase scaffold, it was imperative that we also experimentally showed that our system satisfied our main functional requirement (FR<SUB>1</SUB>: Increase product titre). We therefore planned to do assays that would measure the reaction kinetics of our IAA proof of concept pathway.</P><P>A simple yet effective way to achieve this was by performing a Salkowski assay. This colorimetric assay is standard for experiments involving the IAA pathway and is an easy way to quantify the biosynthesis of the end product, IAA<SUP>20</SUP>. It was also a method the <A href="http://" https:=""> ZJU-China 2012 iGEM team</A>  used to show the effectiveness of their riboscaffold design, and we therefore had their experimental protocols as a reference. By creating a standard curve for the unbound enzymatic reaction of the IAA pathway and comparing this to our scaffolded reaction, we could analyse how co-localising IaaM and IaaH onto our Assemblase scaffold impacts product yield.</P><P>This could be further built upon through the use of high-performance liquid chromatography (HPLC) or liquid chromatography mass spectrometry (LC-MS). These techniques would allow us to separate, identify and quantify each component the reaction mixture of the IAA biosynthesis pathway. This would not only give us an understanding of how the product yield would change when the enzymes were attached to our scaffold but also allow us to understand the changes in the concentration of the reaction intermediate substrate. This is important as it will enable us to consider the probability of undesired reactions occurring in our system, another key functional requirement (FR<SUB>1.1</SUB>: Reduce the probability of encountering undesired reactions). Due to the accessibility of the HPLC protocol and its ability to produce accurate results, we decided to set up this enzyme assay first.</P><H3>4. Plants</H3><P>The team also wanted to demonstrate the functionality of our biosynthesised IAA. After revising  the <A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Design">2011 iGEM Imperial College London</A> team’s project, it was decided that a similar plant based root growth assay could be performed. Using the model plant, <I>Arabidopsis thaliana</I>, we would be able to investigate how varying concentrations of IAA affect the growth and development of the plants. As well as using this to connect the use of our scaffold to a commercial application, we wanted to develop a protocol for testing IAA which would enable us to observe whether the IAA produced from reaction by enzymes fused to our scaffold would have the same phenotypical effects as the IAA which is already commercially available.</P><DIV id="references"><H2>References</H2><OL><LI>Galanie S. et al. Complete biosynthesis of opioids in yeast. <I>Science</I><B>349</B> 1095-1100 (2016).</LI><LI>Lee, H., DeLoache, WC. and Dueber, JE. Spatial organization of enzymes for metabolic engineering. <I>Metabolic Engineering</I><B>14</B> 242-251, (2012).</LI><LI>Bassard, JE. and Halkier, BA. How to prove the existence of metabolons? <I>Phytochemistry Reviews</I><B>17</B> 211-227, (2018).</LI><LI>Dueber, J. E. et al. Synthetic protein scaffolds provide modular control over metabolic flux. <I>Nature Biotechnology</I><B>27</B> 753, (2009).</LI><LI>Moon, TS. et al. Production of glucaric acid from a synthetic pathway in recombinant <I>Escherichia coli</I>. <I>Appl. Environ. Microbiol</I>. <B>75</B> 589–595 (2009).</LI><LI>Liu, A. et al. Function recommender system for product planning and design. <I>CIRP Annals - Manufacturing Technology</I><B>66</B> 181–184, (2017).</LI><LI>Singleton, C., Howard, TP. and Smirnoff, N. Synthetic metabolons for metabolic engineering. <I>Journal of Experimental Botany</I><B>65</B> 1947-1954, (2014). </LI><LI>Moller, BL. Dynamic metabolons. <I>Science</I><B>330</B>, 1328–1329 (2010)</LI><LI>Castellana, M. et al. Enzyme clustering accelerates processing of intermediates through metabolic channeling. <I>Nature Biotechnology</I><B>32</B> 1011, (2014).</LI><LI>Agapakis, CM. Designing synthetic biology. <I>ACS Synth. Biol.</I><B>3</B>(3) 121-8, (2014).</LI><LI>Whitehead, TA., et al. A filamentous molecular chaperone of the prefoldin family from the deep-sea hyperthermophile <I>Methanocaldococcus jannaschii</I>. <I>Protein Science</I><B>16</B> 626-634, (2007).</LI><LI>Tan, LL., Hoon, SS. and Wong, FT. Kinetic controlled Tag-Catcher interactions for directed covalent protein assembly. <I>PLoS ONE</I><B>11</B>(10) e0165074, (2016).</LI><LI>Zakeri, B. et al. Peptide tag forming a rapid covalent bond to a protein, through engineering a bacterial adhesin. <I>Proc Nat Acad Sci.</I><B>109</B>:</LI><LI>E690–E697, (2012).</LI><LI>Veggiani, G. et al. Programmable polyproteams built using twin peptide superglues. <I>Proceedings of the National Academy of Sciences</I><B>113</B> 1202-1207, (2016).</LI><LI>KEGG. Available at: https://www.genome.jp/kegg/pathway.html </LI><LI>Martin, VJJ. et al. Engineering a mevalonate pathway in <I>Escherichia coli</I> for production of terpenoids. <I>Nature Biotechnology</I> <B>21</B> 796–802 (2003).</LI><LI>Yoshida, S. et al. A bacterium that degrades and assimilates poly(ethylene terephthalate). <I>Science</I><B>351</B> 1196-1199, (2016).</LI><LI>Gaweska, H.M. et al. Structure of the Flavoprotein Tryptophan 2-Monooxygenase, a Key Enzyme in the Formation of Galls in Plants. <I>Biochemistry</I>, <B>52</B>(15) 2620–2626, (2013).</LI><LI>Siegert, R., et al. &quot;Structure of the Molecular Chaperone Prefoldin.&quot; <I>Cell</I><B>103</B>(4): 621-632. (2000).</LI><LI>Szkop, M., Sikora, P. and Orzechowski, S. A novel, simple, and sensitive colorimetric method to determine aromatic amino acid aminotransferase activity using the Salkowski reagent. <I>Folia Microbiologica</I><B>57</B> 1-4, (2012).</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>