<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UNSW_Australia_Lab_Cloning skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UNSW Australia/Lab/Cloning</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="navbar"><DIV id="menu"><DIV class="main-menu"><LABEL for="navToggle" id="navigation">Navigation</LABEL></DIV><DIV class="main-menu first-menu"><BUTTON id="home" class="dropbutton"><A href="https://2018.igem.org/Team:UNSW_Australia">Home</A></BUTTON></DIV><DIV class="main-menu"><BUTTON id="project" class="dropbutton">Project</BUTTON><DIV id="project-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Description">Description</A><A href="https://2018.igem.org/Team:UNSW_Australia/Design">Design</A><A href="https://2018.igem.org/Team:UNSW_Australia/Experiments">Experimental Protocols</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab" class="submenu-header">Lab Overview</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/Cloning" class="submenu-sub submenu-sub-first">Cloning</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/Protein" class="submenu-sub">Protein Production</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/Assembly" class="submenu-sub">Assembly</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/FRET" class="submenu-sub">FRET</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/Assays" class="submenu-sub">Enzyme Assays</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/Plants" class="submenu-sub submenu-sub-last">Plants</A><A href="https://2018.igem.org/Team:UNSW_Australia/Results">Results</A><A href="https://2018.igem.org/Team:UNSW_Australia/InterLab">Interlab</A></DIV></DIV><DIV class="main-menu"><BUTTON id="modelling" class="dropbutton">Modelling</BUTTON><DIV id="modelling-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Model">Overview</A><A href="https://2018.igem.org/Team:UNSW_Australia/Model/EKD">Enzyme Kinetics &amp; Diffusion</A><A href="https://2018.igem.org/Team:UNSW_Australia/Model/MD">Molecular Dynamics</A></DIV></DIV><DIV class="main-menu"><BUTTON id="parts" class="dropbutton">Parts</BUTTON><DIV id="parts-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Parts">Overview</A><A href="https://2018.igem.org/Team:UNSW_Australia/Basic_Part">Basic Part</A><A href="https://2018.igem.org/Team:UNSW_Australia/Improve">Improved Part</A><A href="https://2018.igem.org/Team:UNSW_Australia/Software">Software</A></DIV></DIV><DIV class="main-menu"><BUTTON id="hp" class="dropbutton">Human Practices</BUTTON><DIV id="hp-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Human_Practices">Overview</A><A href="https://2018.igem.org/Team:UNSW_Australia/Human_Practices/Commercialisation">Commercialisation</A><A href="https://2018.igem.org/Team:UNSW_Australia/Public_Engagement">Education &amp; Engagement</A><A href="https://2018.igem.org/Team:UNSW_Australia/Human_Practices/Law">Law &amp; Regulation</A><A href="https://2018.igem.org/Team:UNSW_Australia/Human_Practices/Integrate">Integration</A></DIV></DIV><DIV class="main-menu"><BUTTON id="people" class="dropbutton">People</BUTTON><DIV id="people-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Team">Team</A><A href="https://2018.igem.org/Team:UNSW_Australia/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:UNSW_Australia/Attributions">Attributions</A><A href="https://2018.igem.org/Team:UNSW_Australia/Journal">Experts</A></DIV></DIV><DIV class="main-menu"><BUTTON id="doc" class="dropbutton">Documentation</BUTTON><DIV id="doc-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Notebook">Notebook</A><A href="https://2018.igem.org/Team:UNSW_Australia/Safety">Safety</A></DIV></DIV><DIV class="main-menu"><BUTTON id="judging" class="dropbutton">Judging</BUTTON><DIV id="judging-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Medals">Medals</A><A target="_blank" href="https://igem.org/2018_Judging_Form?id=2710">Judging Form</A></DIV></DIV></DIV></DIV><DIV id="footer"><P>© 2018 UNSW Australia iGEM</P></DIV><DIV class="header" id="cloning-header"><DIV class="header-img-txt"><H1 class="shadow-text main-heading">DNA Cloning</H1></DIV></DIV><DIV id="cloning-content" class="to-load"><DIV class="box"><H2>Overview</H2><P>To express the various components of our scaffold for protein conjugation and assembly experiments, our DNA constructs were cloned into appropriate plasmid vectors. We used Gibson assembly cloning to assemble the DNA inserts with the vectors in one simple isothermal reaction<SUP>1</SUP>. The products of the reaction were transformed into competent DH5-alpha <I>Escherichia coli</I> cells and colonies were screened for recombinant plasmids. Plasmids containing the desired DNA inserts were transformed into T7 cells for protein expression and purification experiments. In total, eight constructs were successfully cloned into corresponding pET-Duet1 and pRSF-Duet1 plasmids, while 6 constructs were cloned into pET-19b plasmids.</P></DIV><H2>Introduction</H2><P>The components of the prefoldin scaffold and the associated attached enzymes were to be expressed for enzymatic and self-assembly experiments. To produce the proteins required for these analytical experiments, our “genes of interest” were to be cloned into an appropriate plasmid vector for expression and purification in <I>E. coli</I> cells. This page describes the methods undertaken to produce recombinant plasmids containing our DNA constructs for protein expression. Eight codon optimised DNA constructs were designed and synthesised in the form of g-Blocks from Integrated DNA Technologies (IDT).</P><P>The hetero-hexameric structure of our scaffold is composed of two alpha prefoldin (aPFD) subunits and four beta prefoldin (bPFD) subunits covalently attached to enzymes<SUP>2</SUP>. To produce a functional and complete scaffold, genes encoding the proteins which form the scaffold must first be cloned into an appropriate vector. Plasmids containing aPFD and bPFD were constructed. In addition, aPFD-SpyCatcher and bPFD-SnoopCatcher fusion constructs were created for comparative enzyme activity experiments.</P><P>SpyTags<SUP>3</SUP>and SnoopTags<SUP>3</SUP> were fused to indole-3-acetamide hydrolase (IaaH-SpyTag) and tryptophan 2-monoxygenase (IaaM-SnoopTag) and cloned into appropriate plasmid vectors. Expression of these plasmids would enable protein conjugation experiments with the prefoldin-catcher protein constructs. Fluorescent mCerulean3-SnoopTag and mVenus-SpyTag gBlocks were also cloned into plasmids for Försters Resonance Energy Transfer (FRET) experiments. 6x His-Tags were also attached to the start of each DNA construct via a GSG linker, which would allow the recovery of purified protein.</P><P>All original gBlock designs contain the BioBrick prefix and suffix sequences which were flanked by 20-25 bp long 5’ and 3’ Gibson overhangs. Cloning the BioBrick restriction sites into the DNA construct allowed excision of the insert out from the plasmid vector for diagnostic purposes and for transfer of the inserts into the pSB1C3 BioBrick backbone. Gibson Assembly cloning techniques were used to clone the DNA constructs into the plasmids. 5’ exonuclease activity generates complementary overhang sequences on the insert and vector, and polymerase fills in the gaps of the single strand regions. DNA ligase seals the nicks of the gaps, allowing the two fragments to covalently link together <B>(Figure 1)</B><SUP>1</SUP>.</P><P class="figure-legend"><B>Figure 1:</B> Diagram illustrating the process of Gibson assembly sequence insertion into the plasmid vector<SUP>1</SUP>.</P><P>Our DNA constructs were cloned into pETDuet-1 and pRSFDuet-1 plasmid vectors, as well as pET-19b in our later experiments. The Duet vectors carry two expression units that are controlled by a T7-lac promoter and terminator for protein expression. The Duet plasmids, pETDuet-1 and pPRSFDuet-1, both possess an ampicillin and kanamycin resistance gene, respectively. Meanwhile pET-19 confers ampicillin resistance <B>(Figure 2)</B>. These specific vectors were chosen so that the prefoldin-catcher and enzyme-tag DNA constructs could be cloned into the same cell, allowing the entire scaffold to be expressed simultaneously. Furthermore, pETDuet-1 and pRSFDuet-1 plasmids possess different origins of replication,  which enables in vivo production of the scaffold-enzyme complex.</P><P class="figure-legend"><B>Figure 2:</B> Plasmid maps depicting pETDuet-1, pRSFDuet-1 and pET19-b. Resistance genes are shown in red. Images were generated by Benchling.</P><P>We successfully cloned 8 DNA constructs were successfully cloned into pET-Duet1 and pRSF-Duet1 plasmids. However, we experienced difficulties expressing protein from these plasmids. The constructs were then redesigned to omit the iGEM prefix and suffix sequences and the Gibson overhangs were modified. This enabled the cloning of 6 modified constructs into the pET-19b vector for protein expression and purification <B>(Table 1)</B>. We opted to switch to this vector because it was currently being used successfully by a collaborator for protein expression.</P><P>Circular pETDuet-1 and pRSFDuet-1 and linearised pET-19b were kindly supplied by Dr Dominic Glover.</P><P class="table-legend"><B>Table 1:</B> The constructs to be cloned into each vector.</P><DIV class="flex-center"><TABLE class="lab-table-header"><TBODY><TR><TH>pETDuet-1</TH><TH>pRSFDuet-1</TH><TH>pET-19b</TH></TR><TR><TD><UL><LI>aPFD</LI><LI>bPFD</LI><LI>aPFD-SpyCatcher</LI><LI>bPFD-SnoopCatcher</LI></UL></TD><TD><UL><LI>IaaH-SpyTag</LI><LI>IaaM-SnoopTag</LI><LI>mCerulean3-SnoopTag</LI><LI>mVenus-SpyTag</LI></UL></TD><TD><UL><LI>aPFD</LI><LI>bPFD</LI><LI>IaaH-SpyTag</LI><LI>IaaM-SnoopTag</LI><LI>IaaH</LI><LI>IaaM</LI></UL></TD></TR></TBODY></TABLE></DIV><H2>Aim</H2><P>To clone genes encoding the parts required to form our scaffold into appropriate plasmid vectors by Gibson assembly. In particular:</P><OL><LI>Cloning original DNA constructs into pETDuet-1 and pRSFDuet-1</LI><LI>Cloning modified DNA constructs into pET-19b</LI></OL><H2>DNA design</H2><P>We designed 8 gBlocks for cloning by Gibson assembly into the first multiple cloning sites of pETDuet-1 and pRSFDuet-1 <B>(Figure 3)</B>. All sequences included an N-terminal 6xHis-Tag immediately after the start codon to enable purification using Nickel affinity, followed by a Glycine-Serine-Glycine (GSG) linker. The GSG linker provides flexibility as the side chains of glycine and serine are small, and can allow the 6xHis-Tag to move freely in solution. The amino acid sequences for each protein were obtained. For prefoldin-catcher fusion proteins, a GSGSGSGSG linker and SpyCatcher or SnoopCatcher followed the alpha or beta prefoldin sequence, yielding aPFD-SpyCatcher and bPFD-SnoopCatcher. For enzyme-tag fusion proteins, a GSG linker and SpyTag or SnoopTag followed the enzyme sequence, yielding IaaH-SpyTag and IaaM-SnoopTag. The longer 9 amino acid linker was used for SpyCatcher and SnoopCatcher fusion proteins as the catcher domains are large and may sterically interfere with protein folding if the C-terminus of the original protein is not solvent accessible. An increased linker length on the aPFD-SpyCatcher and bPFD-SnoopCatcher fusions may also enable the scaffold to accommodate the attachment of large enzymes. The DNA sequences were codon optimised for <I>E. coli</I> with manual removal of EcoRI, XbaI, SpeI and PstI restriction sites for RFC10 compatibility. The BioBrick prefix and suffix were then placed on the 5’ and 3’ ends of each sequence. Finally, the DNA sequences were flanked with 25 bp Gibson overhangs identical to the 25 bp immediately upstream and downstream of the insertion site into pETDuet-1 and pRSFDuet-1.</P><P class="figure-legend"><B>Figure 3:</B> DNA constructs designed by Brian Ee. Images were generated by Benchling.</P><H2>Method</H2><P>pETDuet-1 and pRSFDuet-1 plasmids were linearised with PCR (NEB), removing the first multiple cloning site. Enzymes were removed by PCR clean up (Sigma Aldrich). A DpnI digest was performed to remove template circular plasmid and the product was then cleaned up again. Linearity was confirmed by agarose gel electrophoresis. Gibson assembly was used to construct plasmids by combining linearised plasmids with gBlocks ordered from IDT. The Gibson assembly product was transformed via heat shock into <I>E. coli</I> DH5-alpha (NEB) cells before plating onto antibiotic-selective Luria broth agar plates. A colony PCR was performed and analysed by gel electrophoresis in order to identify single colonies that had been successfully transformed with Gibson assembly products. Successful transformants were grown in 10 mL of LB with appropriate antibiotic and plasmids were prepared using QIAprep Spin Miniprep Kit (Qiagen). To confirm the insertion of our DNA construct in the miniprepped plasmids, we performed a diagnostic digest. Plasmids were digested with restriction enzymes that excised the insert and analysed with agarose gel electrophoresis. Sanger sequencing was also performed (Ramaciotti Centre for Genomics) to verify gene sequences.</P><P>For the transfer of inserts into pET-19b, primers were designed to PCR amplify the inserts whilst also removing the BioBrick prefix and suffix, and adding Gibson overhangs appropriate for insertion into the multiple cloning site of pET-19b. Gibson assembly was then performed as previously described with PCR linearised pET-19b <B>(Figure 4)</B>.</P><P>For more detail on our cloning protocols, visit our <A href="https://2018.igem.org/Team:UNSW_Australia/Experiments">experiment page</A></P><P class="figure-legend"><B>Figure 4:</B> Flowchart depicting the overall cloning process undertaken.</P><DIV id="results"><H2>Results</H2><H3>Cloning original DNA constructs into pETDuet-1 and pRSFDuet-1</H3><P>All 8 original DNA constructs were successfully cloned into pET-Duet1 and pRSF-Duet1 plasmid vectors <B>(Figure 3)</B>. The presence of each inserted gene within its plasmid was confirmed by performing a diagnostic digest , and the recombinant plasmids were also each sequence verified <B>(Figure 5)</B>. These plasmids were used for our initial attempts at protein expression and purification.</P><P class="figure-legend"><B>Figure 5:</B> Diagnostic digest of recombinant pETDuet-1 and pRSFDuet-1 plasmids. Gel demonstrates the construction of plasmids containing the desired DNA inserts. Plasmids were restriction enzyme digested with EcoRI and PstI and analysed by agarose gel electrophoresis. The red boxes indicate the presence of the insert at the expected size in comparison to the 2-Log DNA marker.</P><H3>Cloning modified DNA constructs into pET-19b</H3><P>The eight recombinant pETDuet-1 and pRSFDuet-1 recombinant plasmids were unable to be expressed, so six DNA constructs were modified and cloned into the pET-19b vector instead. The presence of aPFD, bPFD and IaaH-SpyTag in pET-19b was confirmed by a diagnostic digest <B>(Figure 6)</B>. Moreover, Sanger sequencing verified successful insertion of these genes as well as aPFD-SpyCatcher, bPFD-SnoopCatcher and IaaM-SnoopTag. Therefore, the following modified constructs were successfully cloned into pET-19b:</P><UL><LI>aPFD</LI><LI>bPFD</LI><LI>aPFD-SpyCatcher</LI><LI>bPFD-SnoopCatcher</LI><LI>IaaH-SpyTag</LI><LI>IaaM-SnoopTag</LI></UL><P>In addition, the following BioBricks were obtained from the iGEM distribution plates and cloned into pET-19b for creation of our <A href="https://2018.igem.org/Team:UNSW_Australia/Improve" target="_blank">Improved Part</A>.</P><P><UL><LI>IaaM <A target="_blank" href="http://parts.igem.org/Part:BBa_K1789000">(BBa_K1789000)</A></LI><LI>IaaH <A target="_blank" href="http://parts.igem.org/Part:BBa_K1789001">(BBa_K1789001)</A></LI></UL></P></DIV><P class="figure-legend"><B>Figure 6:</B> Diagnostic digest of recombinant pET-19b plasmids. Gel demonstrates the construction of recombinant pET-19b plasmids containing aPFD, bPFD and IaaH-SpyT. Plasmids were restriction enzyme digested with EcoRI and XbaI and analysed by agarose gel electrophoresis. The red boxes indicate the presence of the insert at the expected size in comparison to the 2-Log DNA marker. </P><H2>Discussion</H2><P>The initial aim of cloning all 8 DNA constructs into pETDuet-1 and pRSFDuet-1 plasmid vectors was successful. We originally opted for these two Duet vectors as it would enable us to co-transform and express two target genes into MCS1 and MCS2 if time permitted. This could have streamlined the process for hetero-hexameric complex assembly, increasing the efficiency of future cloning experiments.
However, we were unable to express our target proteins when the recombinant plasmids were transformed into expression strains for protein purification experiments. We hypothesised that this was due to the 20 bp long BioBrick prefix situated between the ribosomal binding site (RBS) and the start codon of our construct. This displaces the RBS away from the start of transcription, which is the likely cause for the difficulties experienced with protein expression. Translation studies in E. coli have demonstrated that the optimal spacing between the RBS and the start codon in E. coli ranges from 7-9 nucleotides<SUP>4</SUP>.</P><P>We decided to clone the DNA constructs into the pET-19b vector as it was used successfully in previous cloning experiments for our collaborators. Dr Dominic Glover kindly supplied us with linearised pET-19b plasmids for our experiments, and we were successful in cloning 6 of our 8 DNA constructs into these vectors. Unfortunately due to time restrictions, we were unable to clone all 8 DNA constructs in pET-19b. In the future, we hope to clone more parts into pET-19b which would enable us to perform more assembly tests and ultimately piece together and characterise a complete and functional scaffold. The parts we hope to clone include those for enzymes in the Taxol side-chain biosynthesis pathway.</P><DIV id="references"><H2>References</H2><OL><LI>Gibson, D. et al. Enzymatic assembly of DNA molecules up to several hundred kilobases. <I>Nature Methods</I>. <B>6</B> 343-345 (2009).</LI><LI>Siegert, R. et al. Structure of the Molecular Chaperone Prefoldin. <I>Cell</I>. <B>103</B> 621-632 (2000).</LI><LI>Zakeri, B. et al. Peptide tag forming a rapid covalent bond to a protein, through engineering a bacterial adhesin. <I>Proceedings of the National Academy of Sciences</I>. <B>109</B> E690-E697 (2012).</LI><LI>Vellanoweth, R. and Rabinowitz, J. The influence of ribosome-binding-site elements on translational efficiency in Bacillus subtilis and <I>Escherichia coli</I> in vivo. <I>Molecular Microbiology</I>. <B>6</B> 1105-1114 (1992).</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>