<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UNSW_Australia_Lab_Protein skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UNSW Australia/Lab/Protein</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="navbar"><DIV id="menu"><DIV class="main-menu"><LABEL for="navToggle" id="navigation">Navigation</LABEL></DIV><DIV class="main-menu first-menu"><BUTTON id="home" class="dropbutton"><A href="https://2018.igem.org/Team:UNSW_Australia">Home</A></BUTTON></DIV><DIV class="main-menu"><BUTTON id="project" class="dropbutton">Project</BUTTON><DIV id="project-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Description">Description</A><A href="https://2018.igem.org/Team:UNSW_Australia/Design">Design</A><A href="https://2018.igem.org/Team:UNSW_Australia/Experiments">Experimental Protocols</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab" class="submenu-header">Lab Overview</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/Cloning" class="submenu-sub submenu-sub-first">Cloning</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/Protein" class="submenu-sub">Protein Production</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/Assembly" class="submenu-sub">Assembly</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/FRET" class="submenu-sub">FRET</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/Assays" class="submenu-sub">Enzyme Assays</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/Plants" class="submenu-sub submenu-sub-last">Plants</A><A href="https://2018.igem.org/Team:UNSW_Australia/Results">Results</A><A href="https://2018.igem.org/Team:UNSW_Australia/InterLab">Interlab</A></DIV></DIV><DIV class="main-menu"><BUTTON id="modelling" class="dropbutton">Modelling</BUTTON><DIV id="modelling-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Model">Overview</A><A href="https://2018.igem.org/Team:UNSW_Australia/Model/EKD">Enzyme Kinetics &amp; Diffusion</A><A href="https://2018.igem.org/Team:UNSW_Australia/Model/MD">Molecular Dynamics</A></DIV></DIV><DIV class="main-menu"><BUTTON id="parts" class="dropbutton">Parts</BUTTON><DIV id="parts-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Parts">Overview</A><A href="https://2018.igem.org/Team:UNSW_Australia/Basic_Part">Basic Part</A><A href="https://2018.igem.org/Team:UNSW_Australia/Improve">Improved Part</A><A href="https://2018.igem.org/Team:UNSW_Australia/Software">Software</A></DIV></DIV><DIV class="main-menu"><BUTTON id="hp" class="dropbutton">Human Practices</BUTTON><DIV id="hp-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Human_Practices">Overview</A><A href="https://2018.igem.org/Team:UNSW_Australia/Human_Practices/Commercialisation">Commercialisation</A><A href="https://2018.igem.org/Team:UNSW_Australia/Public_Engagement">Education &amp; Engagement</A><A href="https://2018.igem.org/Team:UNSW_Australia/Human_Practices/Law">Law &amp; Regulation</A><A href="https://2018.igem.org/Team:UNSW_Australia/Human_Practices/Integrate">Integration</A></DIV></DIV><DIV class="main-menu"><BUTTON id="people" class="dropbutton">People</BUTTON><DIV id="people-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Team">Team</A><A href="https://2018.igem.org/Team:UNSW_Australia/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:UNSW_Australia/Attributions">Attributions</A><A href="https://2018.igem.org/Team:UNSW_Australia/Journal">Experts</A></DIV></DIV><DIV class="main-menu"><BUTTON id="doc" class="dropbutton">Documentation</BUTTON><DIV id="doc-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Notebook">Notebook</A><A href="https://2018.igem.org/Team:UNSW_Australia/Safety">Safety</A></DIV></DIV><DIV class="main-menu"><BUTTON id="judging" class="dropbutton">Judging</BUTTON><DIV id="judging-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Medals">Medals</A><A target="_blank" href="https://igem.org/2018_Judging_Form?id=2710">Judging Form</A></DIV></DIV></DIV></DIV><DIV id="footer"><P>© 2018 UNSW Australia iGEM</P></DIV><DIV class="header" id="protein-header"><DIV class="header-img-txt"><H1 class="shadow-text main-heading">Protein Production</H1></DIV></DIV><DIV id="protein-content" class="to-load"><DIV class="box"><H2>Overview</H2><P>Protein scaffold components and proteins that attach to the scaffold must be expressed and purified for self-assembly and enzyme activity experiments. Sequence-verified plasmids were heat shock transformed into <EM>Escherichia coli</EM> cells and expressed for recombinant protein production. The proteins were then purified from cell lysates with Immobilised Metal Affinity Chromatography (IMAC). Nine proteins have been successfully expressed and purified, enabling the construction of our scaffold-enzyme complex.</P></DIV><H2>Introduction</H2><P>A large range of proteins are required for the construction and characterisation of our scaffold-enzyme complex. The scaffold consists of the molecular chaperones alpha prefoldin (aPFD) and beta prefoldin (bPFD), derived from <EM>Methanobacterium thermoautotrophicum</EM><SUP><A href="#references">1</A></SUP>, fused with SpyCatcher<SUP>2</SUP> (SpyC) and SnoopCatcher<SUP><A href="#references">3</A></SUP> (SnoopC) respectively on their C-termini. These self-assemble to form a hexameric complex that is able to covalently bind SpyTags and SnoopTags. Two enzymes were designed for attachment to the scaffold: the enzymes tryptophan 2-monooxygenase (IaaM) originating from <EM>Pseudomonas savastanoi</EM><SUP><HREF>4</HREF></SUP> fused with a SnoopTag (SnoopT) and indole acetamide hydrolase (IaaH) originating from <EM>Alcaligenes sp.</EM> Strain HPC1271<SUP>5</SUP> fused with a SpyTag (SpyT). In addition, the fluorescent proteins mVenus and mCerulean3<SUP>6</SUP> were fused with a SpyTag and SnoopTag respectively. All proteins have been expressed with a 6xHis-Tag, utilising the affinity of the HisTag for nickel ions for IMAC purification<SUP>7</SUP>.</P><P>Initial attempts to express these proteins in <EM>E. coli</EM> using the pETDuet-1 and pRSF-Duet-1 vectors were unsuccessful, likely due to our design of the plasmids. Following the subcloning of the inserts into a new vector, pET-19b, 5 of these proteins were successfully expressed and purified. In addition, strains of <EM>Escherichia coli</EM> (<EM>E. coli</EM>) containing plasmids for His-tagged mVenus, mCerulean3, gamma prefoldin with a C-terminal SpyC (gPFD-SpyC) and gamma prefoldin with an N- and a C-terminal SpyC (SpyC-gPFD-SpyC) were used for protein expression and purification.</P><P>Purification of these proteins enables characterisation of the self-assembly of the scaffold-enzyme complex with Size Exclusion Chromatography (SEC), SDS-PAGE and Transmission Electron Microscopy (TEM) and characterisation of assembly conditions, measurement of enzyme activity with the Salkowski assay and HPLC and characterisation of the distance between attachment sites with Förster resonance energy transfer (FRET).</P><P>We have developed a robust method for recombinant expression and purification of our novel protein scaffold. The purity of the proteins was investigated with SDS-PAGE, which is essential to ensure the quality and accuracy of all experimental characterisation of our scaffold-enzyme complex.</P><H2>Aims</H2><P>To produce, purify and characterise prefoldin scaffold proteins and proteins that attach to our scaffold for further experiments on the stability, assembly and efficacy of our enzyme scaffold.</P><H2>Methods</H2><P><EM>Escherichia coli</EM> T7 Express cells (NEB) were heat shock transformed with a plasmid containing the gene of interest.  The bacteria were grown in Luria broth (LB) media with ampicillin at 37<SUP>o</SUP>C at 200 rpm, induced with isopropyl β-D-1-thiogalactopyranoside (IPTG) at 1 mM when the OD<SUB>600</SUB> of the media reached 0.6 and then grown overnight at room temperature. The cell pellet was collected by centrifugation and lysed by sonication. The cell lysate was then centrifuged to remove cell debris, and only the soluble fraction was collected. The soluble fraction was loaded onto a HisTrap HP 1 mL column (GE Healthcare) and purified using immobilised metal affinity chromatography (IMAC). Elutions were analysed with SDS-PAGE and buffer exchanged into PBS pH 8 using Pierce Protein Concentrators PES, 10K MWCO, 2-6 mL (Thermo Scientific) or by dialysis. The concentration of buffer exchanged proteins were then quantified by the bicinchoninic acid (BCA) assay <B>(Figure 1)</B>.</P><P class="figure-legend"><B>Figure 1:</B> Summary of methods for protein expression and purification.</P><P>Detailed protocols can be found <A target="_blank" href="https://2018.igem.org/Team:UNSW_Australia/Experiments">on our experiments page</A>.</P><H2>Results</H2><P>Following the successful subcloning of inserts from pETDuet-1, pRSFDuet-1 or pSB1C3 into pET-19b, 9 proteins were successfully purified and analysed by SDS-PAGE <B>(Figure 2)</B>. In addition, 3 parts from the Registry of Standard Biological Parts (BBa_K1789000, BBa_K1789001 and BBa_K515100) were expressed, but were not purified. These parts contained the enzymes IaaM and IaaH without His-tags.</P><P class="figure-legend"><B>Figure 2:</B> SDS-PAGE analysis of IMAC purifications of His-tagged proteins. <B>A:</B> mVenus (MW: 27 kDa). <B>B:</B> mCerulean3 (MW: 27 kDa). <B>C:</B> gPFD-SpyC (MW: 31 kDa). <B>D:</B> SpyC-gPFD-SpyC (MW: 46 kDa). <B>E:</B> aPFD (MW: 17 kDa) (left) and bPFD (MW: 15 kDa) (right). <B>F:</B> bPFD-SnoopC (MW: 28 kDa). <B>G:</B> IaaH (without His-tag, unsuccessful purification) (MW: 49 kDa) (left) and IaaH-SpyT (MW: 53 kDa) (right). <B>H:</B> aPFD-SpyC (MW: 30 kDa). SeeBlue Plus 2 Pre-stained Protein Standard (Invitrogen) was used as the molecular weight standard for all SDS-PAGE analysis. Lanes are labelled as cell lysate (L), flow through (FT), wash (W) and elutions (E1, E2, E3, E4, E5).</P><H2>Discussion</H2><P>Initial attempts at protein expression were unsuccessful using pETDuet-1 and pRSFDuet-1 plasmids. After cloning our desired inserts into these plasmids, we attempted to express these proteins, but no expression could be detected by SDS-PAGE or Western Blot. Both <EM>E. coli</EM> T7 Express and Lemo21(DE3) cell lines were used for expression, and tested with 0.1 mM, 0.4 mM and 1 mM IPTG inductions. We hypothesised that the design of our plasmids inhibited expression, as the BioBrick prefix was placed between the ribosome binding site and the start codon of our coding sequence. We decided to subclone our inserts into pET-19b and remove the BioBrick prefix and suffix before retrying protein expression and purification.</P><P>Despite these difficulties, 9 proteins were successfully purified, and 3 protein constructs from the iGEM registry were expressed <B>(Figure 2)</B>. The following constructs were successfully purified:</P><UL><LI>His-aPFD &amp; His-bPFD – for assembly of the aPFD/bPFD hexamer, and as a negative control for the effect of scaffolding on enzyme activity.</LI><LI>His-aPFD-SpyCatcher &amp; His-bPFD-SnoopCatcher – the scaffold components of our complex that can covalently attach Spy-Tagged and Snoop-Tagged enzymes.</LI><LI>His-mVenus &amp; His-mCerulean3 – for FRET experiments to investigate the distance between proteins attached to the scaffold.</LI><LI>His-gPFD-SpyCatcher &amp; His-SpyCatcher-gPFD-SpyCatcher – filamentous variants of prefoldin fused with SpyCatchers, to test SpyTag/SpyCatcher reactions and to determine if gPFD can form filaments with enzymes attached to its N- and/or C- terminus.</LI><LI>His-IaaH-SpyTag – the second enzyme of our reaction pathway, indole-3-acetamide hydrolase, fused with a SpyTag for attachment to His-aPFD-SpyCatcher.</LI></UL><P>The following BioBricks were expressed for comparison with the tagged versions of the enzymes:</P><UL><LI>BBa_K1789000 – IaaM</LI><LI>BBa_K1789001 – IaaH</LI><LI>BBa_K515100 – IaaM and IaaH under a Pveg2 promoter</LI></UL><P>Following protein purification, we encountered issues with solubility for some proteins. This was likely due to non-optimal buffer conditions or high concentrations of protein. Further experimentation and optimisation is required to identify the range of conditions in which these proteins are stable and able to assemble. In the future, we would like to express and purify all of our successfully cloned constructs, including the first enzyme of our reaction pathway, tryptophan-2-monooxygenase (IaaM), fused to the SnoopTag, fluorescent proteins with appropriate tags for FRET experiments and enzymes from other pathways fused with SpyTag and SnoopTag. In addition, we would like to increase the purity of our purifications and attempt larger scale protein expressions. These provide materials that are fundamental for the characterisation of the assembly of our scaffold, the distance between the attachment site, the rate of indole acetic acid production and the modularity of our system.</P><DIV id="references"><H2>References</H2><OL><LI>Siegert, R. et al. Structure of the molecular chaperone prefoldin: unique interaction of multiple coiled coil tentacles with unfolded proteins. <EM>Cell</EM>. <B>103</B> 621–32 (2000).</LI><LI>Zakeri, B. et al. Peptide tag forming a rapid covalent bond to a protein, through engineering a bacterial adhesin. <EM>Proc. Natl. Acad. Sci.</EM><B>109</B> e690–e697 (2012).</LI><LI>Veggiani, G. et al. Programmable polyproteams built using twin peptide superglues. <EM>Proc. Natl. Acad. Sci.</EM><B>113</B> 1202–1207 (2016).</LI><LI>Gaweska, H. M. et al. Structure of the flavoprotein tryptophan 2-monooxygenase, a key enzyme in the formation of galls in plants. <EM>Biochemistry</EM>. <B>52</B> 2620–6 (2013).</LI><LI>Mishra, P. et al. Characterization of an Indole-3-Acetamide Hydrolase from Alcaligenes faecalis subsp. parafaecalis and Its Application in Efficient Preparation of Both Enantiomers of Chiral Building Block 2,3-Dihydro-1,4-Benzodioxin-2-Carboxylic Acid. <EM>PLoS One</EM>. <B>11</B> e0159009 (2016).</LI><LI>Markwardt, M. L. et al. An Improved Cerulean Fluorescent Protein with Enhanced Brightness and Reduced Reversible Photoswitching. <EM>PLoS One</EM>. <B>6</B> e17896 (2011).</LI><LI>Hochuli, E. et al. Genetic Approach to Facilitate Purification of Recombinant Proteins with a Novel Metal Chelate Adsorbent. <EM>Nat. Biotechnol.</EM><B>6</B> 1321–1325 (1988).</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>