<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UNSW_Australia_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UNSW Australia/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="navbar"><DIV id="menu"><DIV class="main-menu"><LABEL for="navToggle" id="navigation">Navigation</LABEL></DIV><DIV class="main-menu first-menu"><BUTTON id="home" class="dropbutton"><A href="https://2018.igem.org/Team:UNSW_Australia">Home</A></BUTTON></DIV><DIV class="main-menu"><BUTTON id="project" class="dropbutton">Project</BUTTON><DIV id="project-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Description">Description</A><A href="https://2018.igem.org/Team:UNSW_Australia/Design">Design</A><A href="https://2018.igem.org/Team:UNSW_Australia/Experiments">Experimental Protocols</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab" class="submenu-header">Lab Overview</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/Cloning" class="submenu-sub submenu-sub-first">Cloning</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/Protein" class="submenu-sub">Protein Production</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/Assembly" class="submenu-sub">Assembly</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/FRET" class="submenu-sub">FRET</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/Assays" class="submenu-sub">Enzyme Assays</A><A href="https://2018.igem.org/Team:UNSW_Australia/Lab/Plants" class="submenu-sub submenu-sub-last">Plants</A><A href="https://2018.igem.org/Team:UNSW_Australia/Results">Results</A><A href="https://2018.igem.org/Team:UNSW_Australia/InterLab">Interlab</A></DIV></DIV><DIV class="main-menu"><BUTTON id="modelling" class="dropbutton">Modelling</BUTTON><DIV id="modelling-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Model">Overview</A><A href="https://2018.igem.org/Team:UNSW_Australia/Model/EKD">Enzyme Kinetics &amp; Diffusion</A><A href="https://2018.igem.org/Team:UNSW_Australia/Model/MD">Molecular Dynamics</A></DIV></DIV><DIV class="main-menu"><BUTTON id="parts" class="dropbutton">Parts</BUTTON><DIV id="parts-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Parts">Overview</A><A href="https://2018.igem.org/Team:UNSW_Australia/Basic_Part">Basic Part</A><A href="https://2018.igem.org/Team:UNSW_Australia/Improve">Improved Part</A><A href="https://2018.igem.org/Team:UNSW_Australia/Software">Software</A></DIV></DIV><DIV class="main-menu"><BUTTON id="hp" class="dropbutton">Human Practices</BUTTON><DIV id="hp-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Human_Practices">Overview</A><A href="https://2018.igem.org/Team:UNSW_Australia/Human_Practices/Commercialisation">Commercialisation</A><A href="https://2018.igem.org/Team:UNSW_Australia/Public_Engagement">Education &amp; Engagement</A><A href="https://2018.igem.org/Team:UNSW_Australia/Human_Practices/Law">Law &amp; Regulation</A><A href="https://2018.igem.org/Team:UNSW_Australia/Human_Practices/Integrate">Integration</A></DIV></DIV><DIV class="main-menu"><BUTTON id="people" class="dropbutton">People</BUTTON><DIV id="people-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Team">Team</A><A href="https://2018.igem.org/Team:UNSW_Australia/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:UNSW_Australia/Attributions">Attributions</A><A href="https://2018.igem.org/Team:UNSW_Australia/Journal">Experts</A></DIV></DIV><DIV class="main-menu"><BUTTON id="doc" class="dropbutton">Documentation</BUTTON><DIV id="doc-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Notebook">Notebook</A><A href="https://2018.igem.org/Team:UNSW_Australia/Safety">Safety</A></DIV></DIV><DIV class="main-menu"><BUTTON id="judging" class="dropbutton">Judging</BUTTON><DIV id="judging-sub" class="sub-menu"><A href="https://2018.igem.org/Team:UNSW_Australia/Medals">Medals</A><A target="_blank" href="https://igem.org/2018_Judging_Form?id=2710">Judging Form</A></DIV></DIV></DIV></DIV><DIV id="footer"><P>© 2018 UNSW Australia iGEM</P></DIV><DIV class="header" id="experiments-header"><DIV class="header-img-txt"><H1 class="shadow-text main-heading">Experiments</H1></DIV></DIV><DIV id="experiments-content" class="to-load"><P>This page includes all the experimental protocols used by the UNSW iGEM team. Please click on each heading to view the protocols or view our weekly progress on the <A href="https://2018.igem.org/Team:UNSW_Australia/Notebook">notebook</A> page. Alternatively, to see the detailed experimental introduction, results and discussion, please visit the <A href="https://2018.igem.org/Team:UNSW_Australia/Lab">lab overview</A> page.</P><DIV onclick="Cloning()" class="experiment-group"><H2 class="shadow-text">DNA Cloning</H2></DIV><DIV class="cloning subheading"><DIV id="cloning" class="toggle-content"><DIV id="linearization" class="protocol" cloning=""><H2>Linearisation pETDuet1 and pRSFDuet1 Plasmid Backbones Using PCR</H2><OL><LI>Forward and reverse primers created for the plasmids.</LI><LI>2 µL of plasmid + 198 µL of water.</LI><LI>PCR amplification, according to the table below:</LI></OL><TABLE class="lab-table"><THEAD><TR><TH>Component</TH><TH>50 µL Reaction</TH><TH>Final Concentration</TH></TR></THEAD><TBODY><TR><TD>Q5 High-Fidelity 2X Master Mix</TD><TD>25 µL</TD><TD>1X</TD></TR><TR><TD>10 µM Forward Primer</TD><TD><P>2.5 µL</P></TD><TD><P>0.5 µM</P></TD></TR><TR><TD><P>10 µM Reverse Primer</P></TD><TD><P>2.5 µL</P></TD><TD><P>0.5 µM</P></TD></TR><TR><TD><P>Template DNA</P></TD><TD><P>2 µL diluted</P></TD><TD><P>&lt; 1,000 ng</P></TD></TR><TR><TD><P>Nuclease-Free Water</P></TD><TD><P>18 µL</P></TD><TD> </TD></TR></TBODY><THEAD><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR></THEAD><TBODY><TR><TD><P>Initial Denaturation</P></TD><TD><P>98°C</P></TD><TD><P>30 seconds</P></TD></TR><TR><TD rowspan="3"><P>25–35 Cycles</P></TD><TD><P>98°C</P></TD><TD><P>5–10 seconds</P></TD></TR><TR><TD><P>62°C</P></TD><TD><P>10–30 seconds</P></TD></TR><TR><TD><P>72°C</P></TD><TD><P>2 minutes</P></TD></TR><TR><TD><P>Final Extension</P></TD><TD><P>72°C</P></TD><TD><P>2 minutes</P></TD></TR><TR><TD><P>Hold</P></TD><TD><P>4–10°C</P></TD><TD> </TD></TR></TBODY></TABLE></DIV><DIV id="dpn1" class="protocol" cloning=""><H2>Plasmid Digest (Dpn1 Digest)</H2><OL><LI>Set-up the reaction mixture:
<TABLE class="lab-table"><TBODY><TR><TD><P>Restriction Enzyme</P></TD><TD><P>1 µL</P></TD></TR><TR><TD><P>DNA</P></TD><TD><P>1 µg</P></TD></TR><TR><TD><P>10X Cutsmart</P></TD><TD><P>5 µL (1X)</P></TD></TR><TR><TD><P>Total Reaction Volume</P></TD><TD><P>50 µL</P></TD></TR></TBODY></TABLE></LI><LI>Incubate for 1 hr at 37°C.</LI><LI>Heat inactivate at 80°C for 20 minutes.</LI></OL></DIV><DIV id="gel" class="cloning" protocol=""><H2>Agarose Gel Electrophoresis</H2><OL><LI>Combine 1 g of agarose powder with 100 mL of 1x TAE buffer in a microwavable flask.</LI><LI>Microwave for 1-2 minutes until the agarose is completely dissolved. Avoid over-boiling, and stop to swirl the flask every 20-30 seconds until the solution is clear.</LI><LI>Allow the solution to cool down until you can comfortably hold the flask with your hand, then add 1 µL of RedSafe.</LI><LI>Seal the ends of a gel tray and pour the solution into the tray with a well comb in place. Let the solution sit at room temperature for 20-30 minutes until it solidifies into a gel.</LI><LI>Place the gel into the gel tank, and fill the tank with 1X TAE buffer until the gel is covered. Remove the well comb.</LI><LI>Mix 2 µL of the DNA sample with 3 µL of H<SUB>2</SUB>O and 1 µL of 6X loading dye.</LI><LI>Load DNA ladder into the first well of the gel, followed by your DNA samples.</LI><LI>Connect the gel tank to a power pack and run the gel at 100 V for 1 hr.</LI><LI>Carefully take the gel tray to a Gel Doc for imaging.</LI></OL></DIV><DIV id="gibson" class="cloning" protocol=""><H2>Gibson Assembly</H2><H3>Materials:</H3><P>5X Isothermal Reaction Mix (6 mL total, Store at -20°C):</P><TABLE class="lab-table"><TBODY><TR><TD>1 M Tris-HCl (pH 7.5)</TD><TD>3 mL</TD></TR><TR><TD>1 M MgCl<SUB>2</SUB></TD><TD>300 µL</TD></TR><TR><TD>100 mM dGTP</TD><TD>60 µL</TD></TR><TR><TD>100 mM dATP</TD><TD>60 µL</TD></TR><TR><TD>100 mM dTTP</TD><TD>60 µL</TD></TR><TR><TD>100 mM dCTP</TD><TD>60 µL</TD></TR><TR><TD>1 M DTT</TD><TD>300 µL</TD></TR><TR><TD>PEG-8000</TD><TD>1.5 g</TD></TR><TR><TD>100 mM NAD</TD><TD>300 µL</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>360 µL</TD></TR></TBODY></TABLE><P>Assembly Master Mix (1.2 mL total, store in 15 µL aliquots at -20°C.):</P><TABLE class="lab-table"><TBODY><TR><TD>5X Isothermal Master Mix</TD><TD>320 µL</TD></TR><TR><TD>510 U/µL T5 exonuclease</TD><TD>0.64 µL</TD></TR><TR><TD>2 U/µL Phusion DNA Polymerase</TD><TD>20 µL</TD></TR><TR><TD>40 U/µL T4 DNA Ligase</TD><TD>0.16 µL</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>860 µL</TD></TR></TBODY></TABLE><H3>Method:</H3><OL><LI>PCR amplify or digest your fragments of choice and gel purify.</LI><LI>If PCR from a methylated DNA template (e.g. linearised plasmid), a DpnI digest can be used to remove the unwanted template plasmid. Clean up afterwards.</LI><LI>Thaw a 15 µL assembly mixture aliquot and keep on ice until ready to be used.</LI><LI>Nanodrop your DNA fragments. Prepare 50-100 ng of vector with 2-3 fold excess of insert in nuclease-free water to a total volume of 5 µL.</LI><LI>Add 15 µL assembly mixture to the DNA mixture, for a total reaction volume of 20 µL. </LI><LI>Incubate at 50°C for 15 to 60 minutes (60 minutes is optimal).</LI></OL></DIV><DIV id="heat-shock" class="protocol" cloning=""><H2>Heat Shock Transformation</H2><OL><LI>Incubate 50 ng of plasmid construct  with 25 µL of chemically competent <I>E. coli</I> T7 express or DH5α on ice for 30 minutes.</LI><LI>Heat shock the cells for 45 seconds at 42°C and place back onto ice for 2 minutes.</LI><LI>Allow cells to grow for 45 minutes in 200 µL of SOC outgrowth media (NEB) at 37°C and 200 rpm.</LI><LI>Spread plate onto Luria Broth (LB) agar plates containing appropriate antibiotics at working concentration and grow at 37°C overnight.</LI></OL></DIV><DIV id="colonyPCR" class="protocol" cloning=""><H2>Colony PCR</H2><OL><LI>Pick up individual bacterial colonies from an agar plate that was grown overnight using a pipette tip, and dilute each colony into 50 µL of water.</LI><LI>Prepare the PCR master mix by multiplying the volume for each component shown below (for 1 reaction) by the number of colonies to sample +1 (i.e. number of reactions plus 1 excess).
</LI><LI>18 µL nuclease free water.</LI><LI>5 µL 5X Taq master mix.</LI><LI>0.5 µL 10 µM T7 promotor primer</LI><LI>0.5 µL 10 µM T7 terminator primer</LI><LI>Add 1 µL of each diluted colony and 24 µL of PCR master mix to a PCR tube.</LI><LI>Run PCR with the following steps (lid at 105°C and volume = 20 µL):
</LI><LI>Step 1: 95°C for 5 minutes</LI><LI>Step 2: 95°C for 30 seconds</LI><LI>Step 3: 55°C for 30 seconds</LI><LI>Step 4: 68°C for 2 minutess</LI><LI>Repeat 25-35 cycles of steps 2-4</LI><LI>Step 5: 68°C 5 minutes</LI><LI>Step 6: Hold at 4°C</LI><LI>Run PCR products on a 1 % agarose gel and image for analysis.</LI></OL></DIV><DIV id="sequencing" class="protocol" cloning=""><H2>Sequencing</H2><OL><LI>Transfer 10 µL of purified plasmid sample (50-100 ng/µL) to a PCR tube. Add 5 µL of one primer (forward or reverse).</LI><LI>Request sequencing at the UNSW Centre for Genomics via the on-line portal.</LI><LI>Label PCR tubes carefully with numbers as per the on-line order.</LI><LI>Take PCR tubes to the Ramaciotti Centre for Genomics Lvl 2 (at UNSW, Sydney, Australia), and store the samples in the fridge provided.</LI><LI>Sanger sequencing is carried out by the Centre staff following <A target="_blank" href="http://www.ramaciotti.unsw.edu.au/wp-content/uploads/2016/09/RAMAC_LIMS_User_Guide-2016.pdf">the provided protocol</A>.</LI></OL></DIV><DIV id="restriction" class="cloning" protocol=""><H2>Restriction Cloning</H2><OL><LI>Set-up the reaction mixture:
<TABLE class="lab-table"><TBODY><TR><TD><P>Restriction Enzyme</P></TD><TD><P>1 µL of each enzyme</P></TD></TR><TR><TD><P>DNA</P></TD><TD><P>1 µg</P></TD></TR><TR><TD><P>10X Cutsmart</P></TD><TD><P>5 µL (1X)</P></TD></TR><TR><TD><P>Total Reaction Volume</P></TD><TD><P>50 µL</P></TD></TR></TBODY></TABLE></LI><LI>Incubate for 1 hr at 37°C.</LI><LI>Heat inactivate at 80°C for 20 minutes.</LI></OL><H3>Ligation</H3><OL><LI>Set up the following reaction in a microcentrifuge tube on ice:
<TABLE class="lab-table"><TBODY><TR><TH>Component</TH><TH>20 µL Reaction</TH></TR><TR><TD><P>T4 DNA Ligase Buffer (10X)*</P></TD><TD><P>2 µL</P></TD></TR><TR><TD><P>Vector DNA</P></TD><TD><P>50 ng</P></TD></TR><TR><TD><P>Insert DNA</P></TD><TD><P>A molar ratio of 1:3 vector to insert should be used</P></TD></TR><TR><TD><P>Nuclease-free water</P></TD><TD><P>to 20 µL</P></TD></TR><TR><TD><P>T4 DNA Ligase</P></TD><TD><P>1 µL</P></TD></TR></TBODY></TABLE></LI><LI>Gently mix the reaction by pipetting up and down and microfuge briefly.</LI><LI>For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.</LI><LI>Heat inactivate at 65°C for 10 minutes.</LI><LI>Chill on ice and transform 1-5 µL of the reaction into 25 µL competent cells.</LI></OL></DIV><DIV id="miniprep" class="protocol" cloning=""><H2>Miniprep</H2><P>Protocols were followed from the Qiagen QIAprep Spin Miniprep Kit Cat No./ID: 27104. No changes were made.</P></DIV></DIV></DIV><DIV onclick="Protein()" class="experiment-group"><H2 class="shadow-text">Protein Production</H2></DIV><DIV class="protein subheading"><DIV id="protein" class="toggle-content"><DIV id="starter-culture" class="protocol" protein=""><H2>Starter Culture</H2><P>One colony was selected from the plate grown overnight and grown in 2 mL of LB containing appropriate antibiotic at 37°C and 200 rpm overnight.</P></DIV><DIV id="grow-up" class="protocol" protein=""><H2>Large-scale grow-up</H2><OL><LI>Baffled shake flasks containing 500 mL of LB with 50uL of the appropriate antibiotic at 37°C are inoculated with the starter culture.</LI><LI>The cells are grown at 37°C and 200 rpm and OD<SUB>600</SUB> is periodically measured. </LI><LI>Once OD<SUB>600</SUB> reaches an absorbance reading of 0.6, add 1mM of IPTG to induce expression of the protein.</LI><LI>After induction, grow the cells overnight at 24°C, 200 rpm.</LI></OL></DIV><DIV id="cell-collection" class="protocol" protein=""><H2>Collection of Cells by Centrifugation</H2><OL><LI>Centrifuge the bacterial culture at 4600 x g for 20 minutes.</LI><LI>Collect cell pellet and resuspended in binding buffer (20 mM NaH<SUB>2</SUB>PO<SUB>4</SUB>, 500 mM NaCl, 10 mM Imidazole).</LI></OL></DIV><DIV id="lysis" class="protocol" protein=""><H2>Cell Lysis by Sonication</H2><OL><LI>Lyse the cell pellet by sonication (Branson) for 10 minutes at 50% amplitude, alternating 2 second intervals, kept on ice.</LI><LI>Centrifuge the cell lysate at 15000 rpm for 45 minutes.</LI><LI>Collect the supernatant (soluble fraction).</LI></OL></DIV><DIV id="IMAC" class="protocol" protein=""><H2>IMAC</H2><P>Immobilised Metal ion Affinity Chromatography (IMAC) was performed to purify the expressed proteins.</P><OL><LI>His-tagged protein is bound to a 1 mL Ni-NTA Superflow Cartridge (Qiagen) by loading the soluble fraction of the cell lysate onto the column.</LI><LI>Wash with 10 mL of binding buffer (20 mM NaH<SUB>2</SUB>PO<SUB>4</SUB>, 500 mM NaCl, 10 mM Imidazole).</LI><LI>Elute with 2 mL of elution buffer (20 mM NaH<SUB>2</SUB>PO<SUB>4</SUB>, 500 mM NaCl, 500 mM Imidazole).</LI><LI>Analyse fractions with SDS-PAGE.</LI></OL></DIV><DIV id="buffer-exchange" class="protocol" protein=""><H2>Buffer Exchange</H2><H3>Column</H3><OL><LI>Elutions were analysed with SDS-PAGE and buffer exchanged into PBS (pH 8) using Pierce Protein Concentrators PES, 10K MWCO, 2-6 mL (Thermo Scientific).</LI><LI>Add protein to the column.</LI><LI>Top up column to 6 mL with PBS buffer.</LI><LI>Centrifuge column at 4600 x g for 20 minutes.</LI><LI>Repeatedly centrifuge, discard flow through, and top up with PBS buffer (pH 8) until dilution factor of 0.01 is reached. That is, until there is 1% of the old buffer left in the solution.</LI></OL><H3>Dialysis</H3><OL><LI>Add 1 mL of protein and 1 mL of PBS buffer (pH 8) to a 15 mL Falcon tube.</LI><LI>Add 2 mL of the solution to a SnakeSkin™ Dialysis Tubing, 10K MWCO, 22 mm.</LI><LI>Use dialysis tubing clamps (one-piece polypropylene clamp) to further secure the solution inside the SnakeSkin dialysis tubing.</LI><LI>Add 500 mL of PBS buffer, pH 8, (this is the buffer we want to exchange into) into a 500 mL glass beaker.</LI><LI>Place the dialysis tubing with the solution into the beaker.</LI><LI>Place the beaker on top of a magnetic stirrer, 75 rpm, and leave overnight.</LI></OL></DIV><DIV id="western" class="protocol" protein=""><H2>Western Blot</H2><H3>Materials</H3><UL><LI>NuPAGE Bis-Tris gel</LI><LI>NuPAGE MES running buffer</LI><LI>Mini iBlotTM stack</LI><LI>TBS-T: </LI><LI>1X TBS with 0.1 % Tween20</LI><LI>Blocking Solution: </LI><LI>5 % skim milk in TBS-T</LI><LI>Antibody Solution: </LI><LI>1:2000 dilution of HRP conjugated anti-His-tag antibody in TBS-T + 1 % BSA</LI><LI>Chemiluminescent HRP substrate</LI></UL><H3>Sample Preparation</H3><OL><LI>Add reducing buffer to the bacterial lysates.</LI><LI>Heat at 95°C for 5 minutes.</LI></OL><H3>SDS-Page Gel</H3><OL><LI>Remove the NuPAGE gel from its packaging and peel off the plastic strip from its base.</LI><LI>Place the gel inside the tank, and fill with NuPAGE MES running buffer.</LI><LI>Load 5 µL of the protein standards ladder into the first well.</LI><LI>Load up to 20 µL of each lysate sample into the wells.</LI><LI>Connect the gel tank to a power pack, and run at 160 V for 40 minutes.</LI></OL><H3>Protein Transfer</H3><OL><LI>Remove and rinse the gel in water.</LI><LI>Inside an iBlot Transfer Device, assemble the mini stack with the gel inside.</LI><LI>Run at 20 V for 7 minutes.</LI></OL><H3>Blocking</H3><OL><LI>Incubate the membrane for 1-2 hours in blocking solution at room temperature, shaking.</LI></OL><H3>Antibody Staining</H3><OL><LI>Incubate the membrane in antibody solution either at 4°C overnight, or at room temperature for 2 hours.</LI><LI>Wash the membrane in TBS-T three times for 10 minutes per wash at room temperature, shaking.</LI></OL><H3>Detection</H3><OL><LI>Remove membrane from the last wash and place in chemiluminescent image analyser.</LI><LI>Prepare HRP substrate according to manufacturer's instructions and add to the membrane.</LI><LI>Image.</LI></OL></DIV></DIV></DIV><DIV onclick="Assembly()" class="experiment-group"><H2 class="shadow-text">Assembly</H2></DIV><DIV class="assembly subheading"><DIV id="assembly" class="toggle-content"><DIV id="catcher-tag" class="protocol" assembly=""><H2>Catcher and Tag Assembly</H2><OL><LI>Purified IaaH fused to SpyTag (IaaH-SpyTag) and prefoldin proteins fused to SpyCatcher (aPFD-SpyCatcher, gPFD-SpyCatcher and SpyCatcher-gPFD-SpyCatcher) were obtained.</LI><LI>IaaH-SpyTag was mixed with either aPFD-SpyCatcher, gPFD-SpyCatcher and SpyCatcher-gPFD-SpyCatcher at a concentration of 3 µM and 15 µM respectively in a total volume of 250 µL in PBS pH 8, and incubated at room temperature.</LI><LI>After 0, 10, 20 and 30 minutes of incubation, a 10 µL sample was obtained.</LI><LI>5 µL of 4X Bolt LDS sample buffer was added to each sample, and then boiled for 10 minutes at 95°C to cease SpyCatcher reactivity whilst preserving any covalent interactions.</LI><LI>The samples were then examined via SDS-PAGE.</LI></OL></DIV><DIV id="SEC" class="protocol" assembly=""><H2>Size Exclusion Chromotography (SEC)<P>We would like to thank Hélène Lebhar of the UNSW Recombinant Proteins facility for conducting SEC experiments for us.</P></H2><H2>TEM</H2><H3>Materials</H3><UL><LI>Protein sample (ideally in water or low salt buffer)</LI><LI>Milli-Q water</LI><LI>2 % uranyl acetate</LI><LI>Parafilm</LI><LI>Self-closing forceps</LI></UL><H3>Staining</H3><OL><LI>Glow discharge the carbon-coated grids (carbon-coated side up) to render them hydrophilic.</LI><LI>Place the grids (carbon-coated side up) on parafilm.</LI><LI>Place 7 µL of sample on a grid.</LI><LI>Let the grid stain for 10 minutes.</LI><LI>Using a pasteur pipette, transfer 5 droplets of Milli-Q water on a piece parafilm (for each grid).</LI><LI>After the 10 minutes of staining, pick up grid with the forceps and wash residue salts by passing the grid through the five droplets of water.</LI><LI>In a fume hood, place one droplet of 2 % uranyl acetate on a piece of parafilm (for each grid).</LI><LI>Wear appropriate PPE to handle uranium; do not work with uranium outside the fume hood.</LI><LI>Place the grid on the uranyl acetate drop (carbon-coated side down) and stain for 5-7 minutes.</LI><LI>Pick the grid up with the forceps.</LI><LI>Remove excess uranyl acetate by touching one side of the grid with filter paper for a few seconds, then briefly touch the other side of the grid with the filter paper.</LI><LI>Let the grid dry for a few minutes before transferring back it to its box.</LI><LI>The grid can be fully dried in a oven if imaging the sample the next day, or just left in the box until imaging.</LI></OL></DIV></DIV></DIV><DIV onclick="Fret()" class="experiment-group"><H2 class="shadow-text">FRET</H2></DIV><DIV class="fret subheading"><DIV id="FRET" class="toggle-content"><H2>FRET Protocol for Negative Controls</H2><H3>Part 1</H3><OL><LI>Dilute mCerulean3 (Cerulean) and mVenus (Venus) protein samples to 1 mg/mL (≈ 34 µM) with PBS pH 7.4</LI><LI>Create a serial dilution of both Cerulean and Venus from 1 mg/mL down to 1 µg/mL in an (ideally) black sided 96-well plate (200 µL of undiluted protein, transfer 20 µL to 180 µL PBS, and so on).</LI><LI>Scan the excitation and emission spectra of these wells to determine the ideal values of excitation and emission for each value, using the values from Markwardt <I>et al.</I> and Jonáš<I>et al.</I> as shown in the table as a starting point.</LI><LI>Blank appropriately with 180 µL PBS.</LI></OL><TABLE class="lab-table"><TBODY><TR><TH>Excitation</TH><TH>Emission</TH></TR><TR><TD>mCerulean3</TD><TD>433 (400-465)</TD><TD>475 (465-525)</TD></TR><TR><TD>mVenus</TD><TD>515 (500-525)</TD><TD>528 (520-550)</TD></TR></TBODY></TABLE><P>The values in these tables were taken from, Markwardt <I>et al.</I> (2011)<SUP>1</SUP> and Jonáš<I>et al.</I> (2014)<SUP>2</SUP>.</P><UL><LI>Dilute samples equally if concentration is too high for the fluorescence reader of the machine.</LI></UL><H3>Part 2</H3><OL><LI>Add 90 µL of 1 mg/mL protein to 90 µL PBS for each protein in triplicate.</LI><LI>Add 90 µL of Cerulean to 90 µL Venus in triplicate.</LI><LI>Blank with 180 µL PBS.</LI><LI>Excite all wells at the ideal excitation value determined for Cerulean and scan wavelengths longer than this.</LI><LI>Please see <A target="_blank" href="https://static.igem.org/mediawiki/2018/3/35/T--UNSW_Australia--FRET2018.zip">supplementary data</A> for specific parameters used.</LI></OL></DIV></DIV><DIV onclick="Assays()" class="experiment-group"><H2 class="shadow-text">Enzyme Assays</H2></DIV><DIV class="assays subheading"><DIV id="enzyme-assays" class="toggle-content"><DIV id="BCA" class="protocol" assays=""><H2>BCA Protein Assay</H2><P>Protocol followed as per Pierce BCA Protein Assay Kit 23225 to determine protein concentration in solution:</P><UL><LI>Sample to working reagent ratio =  1:8</LI><LI>96-well plate reader: SPECTROstar Nano BMG LABTECH</LI><LI>Software used: SPECTROstar Nano</LI></UL></DIV><DIV id="salkowski" class="protocol" assays=""><H2>Salkowski Assay</H2><P>The Salkowski assay is a measurement-based protocol used to assess the quantity of indole-3-acetic acid within a sample. This particular protocol has been adapted from the methods used by <A href="https://2011.igem.org/Team:Imperial_College_London/Project_Auxin_Design">2011 iGEM Imperial College London</A>, Tang and Bonner’s 1947 paper<SUP>3</SUP>, and Glickmann and Dessaux's 1995 paper<SUP>4</SUP>.</P><H3>Cellular Protocol</H3><OL><LI>Pick a single colony of untransformed T7 and transformed T7 and inoculate a 2-5ml overnight culture in LB.</LI><LI>Add antibiotics to the transformed culture.</LI><LI>The next day, add 50 mL of LB + 2.5 mg/mL tryptophan to a 250 mL shake flask labelled “Control”. </LI><LI>Add 50 mL of LB + 2.5 mg/mL tryptophan + antibiotic to another baffled shake flask labelled “IAA”. </LI><LI>Add 1 mL of the respective overnight cultures to each flask. Incubate with shaking (200 rpm) at 37°C. Sample at 1h, 4h, 7h and 20-24h. </LI><LI>For each sample: a. Measure the OD at 600 nm.</LI><LI>Spin down 1 mL. Retain pellet and freeze. Retain supernatant and store at 4°C. Label these 4.</LI><LI>The next day:  Add Salkowski reagent to supernatants at the ratio of 1 mL per 0.5 mL of supernatant. Incubate in the dark for 30 minutes.</LI><LI>Add 1 mL of Salkowski reagent to each cell pellet.</LI><LI>Vortex to mix.</LI><LI>Incubate in the dark for 30 minutes.</LI><LI>Centrifuge at full speed for 5 minutes.</LI><LI>Measure the absorbance at 530 nm Measure appropriate controls as well.</LI><LI>Normalise readings at each time point against OD<SUB>600</SUB>.</LI></OL><P>The assay is performed by adding the Salkowski reagent to a prepared sample:</P><OL><LI>To prepare the Salkowski assay reagent, slowly add 95 % H<SUB>2</SUB>SO<SUB>4</SUB> to Milli-Q water in a ratio of 3:2, allowing time for heat dissipation between additions.</LI><LI>Dissolve 4.5 g/L of anhydrous FeCl<SUB>3</SUB> into the solution. Store in the dark, wrapped in foil.</LI><LI>Prepare the standard by using a 3 mM stock, then diluted 1 in 10 for usage – with 300 µM to 0 µM concentrations. </LI><LI>Prepare a standard curve.</LI><LI>Samples are run by adding the sample (or standard) to the Salkowski reagent in a ratio of 1:2 in a time sensitive manner within a 96-well microtitre plate. </LI><LI>Run the plate on a plate reader for absorbance at 530 nm, after being left for 30 minutes in the dark.</LI></OL></DIV><DIV id="HPLC" class="protocol" assays=""><H2>High Performance Liquid Chromotography (HPLC)</H2><OL><LI>Acidify supernatant with 5 M HCl to pH 2.5.</LI><LI>Extract supernatant with an equal volume of ethyl acetate.</LI><LI>Evaporate off the solvent by either drying it through a vacuum or dissolving in methanol &amp; acetic acid at pH 4.5 (25 % methanol, 1 % acetic acid, made up to 1 mL with H<SUB>2</SUB>O).</LI><LI>250 µL methanol, 10 µL acetic acid, 740 µL H<SUB>2</SUB>O.</LI><LI>Run system with solution 72 % of 1 % acetic acid and 28 % of 100 % methanol for 0.8 mL/minute with injection of 30.</LI></OL></DIV></DIV></DIV><DIV onclick="Plants()" class="experiment-group"><H2 class="shadow-text">Plants</H2></DIV><DIV class="plants subheading"><DIV id="plants-content" class="toggle-content"><H2>Seed Surface Sterilisation</H2><OL><LI>Wild-type <I>Arabidopsis thaliana</I> seeds were surface sterilised by mixing in a solution of 2.5 % sodium hypochlorite with a drop of Tween for 10 minutes.</LI><LI>Seeds were subsequently washed eight times in sterile water.</LI></OL><H2>Indole-3-acetic Acid Filter Sterilisation</H2><OL><LI>10 mM stock indole-3-acetic acid (IAA) was prepared by dissolving indole-3-acetic acid sodium salt purchased from Sigma Aldrich in ethanol.</LI><LI>In a laminar flow hood, the solution was filter sterilised through a 0.22 µm Millex® syringe driven filter and stored at 4°C in a foil covered container to protect from light.</LI></OL><H2>MS Agar Plate Preparation</H2><OL><LI>Murashige and Skoog (MS) media was prepared using M5519 MS basal powdered medium purchased from Sigma Aldrich.</LI><LI>In four separate Schott bottles, 250 mL of MS media was prepared using 1.1 g of MS basal salts (per manufacturer instructions) and Milli-Q purified water.</LI><LI>Each mixture was supplemented with sucrose (1 % w/v) and agar powder (0.8 % w/v) and adjusted to pH 5.7 using KOH.</LI><LI>Mixtures were autoclaved and allowed to cool to approximately 50°C (or until able to be held comfortably).</LI><LI>Cooled bottles were transferred to a dark laminar flow hood where indole-3-aecetic acid solution was added to prepare four separate MS agar mixtures at 0 µM, 1 µM, 10 µM and 100 µM concentrations IAA respectively.</LI><LI>In a dark laminar flow hood, 50 mL of media was pipetted into a square plate using a serological pipette. Four 50 mL plates were prepared for each concentration of IAA.</LI><LI>Plates were labelled, sealed with parafilm and stored wrapped in foil at 4°C until further use.</LI></OL><H2>Liquid Culture Media Preparation</H2><OL><LI>MS media was prepared using M5519 MS basal powdered medium purchased from Sigma Aldrich.</LI><LI>In an autoclavable Schott bottle, 500 mL 1/2 MS media was prepared using 1.1 g of MS basal salts and 500 ml of Milli-Q purified water (4.4 g salts equivalent to 1 L MS media per manufacturer’s instructions).</LI><LI>Media was supplemented with sucrose (1 % w/v) and adjusted to pH 5.7 using KOH.</LI><LI>Media was autoclaved and stored at room temperature until further use.</LI></OL><H2><I>Arabidopsis thaliana</I> Growth on Agar Plates</H2><OL><LI>Surface sterilised seeds were added to a flask containing an autoclaved solution of Milli-Q water with 1 % w/v sucrose in a laminar flow hood.</LI><LI>This flask was covered with foil and seeds were stratified at 4°C for two days.</LI><LI>Stratified seeds were then allowed to germinate under light at room temperature for 2 days.</LI><LI>In a dark laminar flow hood, seedlings were transplanted to MS media plates with varying concentrations of IAA. Approximately five seedlings were transplanted per plate using sterile tweezers and loops. Seedlings were arranged in a line, 2 cm from the top of each plate. Three plates with seedlings were prepared for each concentration.</LI><LI>Plates were sealed with parafilm and covered with foil leaving only the top 2 cm exposed in order to limit IAA photodegradation and simulate the natural growth conditions of higher plants whereby only shoots are exposed to illumination<SUP>5</SUP>. Plates were stored upright at an approximately 90° angle. The plated seedlings were grown under light at room temperature for 20 days.</LI><LI>Primary root length and lateral root numbers were measured at 4, 6, 11 and 20 days following the initial seedling transplant.</LI></OL><H2><I>Arabidopsis thaliana</I> Liquid Culture</H2><OL><LI>In a laminar flow hood, 25 mL of 1/2 MS media with 1 % sucrose was aliquoted into twelve 50 mL foil wrapped falcon tubes.</LI><LI>Filter sterilised IAA was added to prepare duplicate tubes at 0 nM, 1nM, 10 nM, 100 nM, 1 µM and 10 µM concentrations of IAA.</LI><LI>100 µL of surface sterilised <I>A. thaliana</I> seeds were added to each tube.</LI><LI>Liquid cultures were stratified at 4°C for two days, then grown under light at room temperature for 9 days.</LI><LI>Three seedlings from each tube were selected at random and measured for primary root length and shoot length.</LI></OL></DIV></DIV><DIV id="references"><H2>References</H2><OL><LI>Markwardt, M. et al. An improved cerulean fluorescent protein with enhanced brightness and reduced reversible photoswitching. <I>PLOS ONE</I><B>6</B> e17896 (2011).</LI><LI>Jonáš, A. et al. In vitro and in vivo biolasing of fluorescent proteins suspended in liquid microdroplet cavities. <I>Lab Chip</I><B>14</B> 3093-3100 (2014).</LI><LI>Tang, YW. and Bonner, J. The enzymatic inactivation of indoleacetic acid; some characteristics of the enzyme contained in pea seedlings. <I>Arch. Biochem.</I><B>13</B> 11–25 (1947).</LI><LI>Glickmann, E. and Dessaux, Y. A critical examination of the specificity of the salkowski reagent for indolic compounds produced by phytopathogenic bacteria. Applied and Environmental Microbiology <B>61</B> 793–796 (1995).</LI><LI>Xu, W. et al. An improved agar-plate method for studying root growth and response of <I>Arabidopsis thaliana</I>. <I>Sci Rep</I><B>3</B> 1273 (2013).</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>