<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Project_Indigoidine"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Project/Indigoidine">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Project/Indigoidine&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/Indigoidine&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/Indigoidine&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Project/Indigoidine" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Project/Indigoidine</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="row"><DIV id="nav" class="navigation"><DIV class="item team" data-offset-l="-130"><H2>Team</H2><UL><LI><A href="/Team:Heidelberg/Team">Team Members</A></LI><LI><A href="/Team:Heidelberg/AttributionsII">Attributions</A></LI><LI><A href="/Team:Heidelberg/Team/Sponsors">Sponsors</A></LI><LI><A href="/Team:Heidelberg/Team/Collaboration">Collaborations</A></LI><LI><A href="/Team:Heidelberg/Team/Gallery">Gallery</A></LI></UL></DIV><DIV class="item project" data-offset-l="-144"><H2>Project</H2><UL><LI><A href="/Team:Heidelberg/NRPS">Background</A></LI><LI><A href="/Team:Heidelberg/Project">Overview</A></LI><LI><A href="/Team:Heidelberg/Project/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Project/Indigoidine-Tag">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Project/Tag-Optimization">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Project/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Project/Achievements">Achievements</A></LI></UL></DIV><DIV class="item notebook" data-offset-l="-139"><H2>Notebook</H2><UL><LI><A href="/Team:Heidelberg/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Tyrocidine">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Indigoidine">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/NRPSDesigner">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/Notebook/Methods">Methods</A></LI><LI><A href="/Team:Heidelberg/Materials">Materials</A></LI></UL></DIV><DIV class="item parts" data-offset-l="-134"><H2>Parts</H2><UL><LI><A href="/Team:Heidelberg/Parts">Submitted Parts</A></LI><LI><A href="/Team:Heidelberg/Favorite_Parts">Favorite Parts</A></LI><LI><A href="/Team:Heidelberg/RFCs">Our RFCs</A></LI></UL></DIV><DIV class="item software" data-offset-l="-134"><H2>Software</H2><UL><LI><A href="/Team:Heidelberg/Project_Software">Project</A></LI><LI><A href="http://igem2013.bioquant.uni-heidelberg.de/NRPSDesigner/">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/iGEM42">iGEM42</A></LI><LI><A href="/Team:Heidelberg/OpenSource">Open Source</A></LI><LI><A href="/Team:Heidelberg/Project/AchievementsSoftware">Achievements</A></LI></UL></DIV><DIV class="item human" data-offset-l="-118"><H2>Human Practice</H2><UL><LI><A href="/Team:Heidelberg/HumanPractice">Overview</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BroadPublic">Broad Public</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Survey">Survey</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/YoungGeneration">Young Generation</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Artists">Arts and Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BeyondScience">Beyond Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Experts">Experts</A></LI></UL></DIV><DIV class="item safety" data-offset-l="-115"><H2>Safety</H2><UL><LI><A href="/Team:Heidelberg/Safety">Safety</A></LI></UL></DIV><DIV class="item model" data-offset-l="-144"><H2>Modeling</H2><UL><LI><A href="/Team:Heidelberg/Modelling/Gold_Recovery">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Modelling/Ind_Production">ind-Production</A></LI></UL></DIV><DIV class="item"><SPAN style="margin-left:6%">HOME</SPAN></DIV></DIV></DIV><DIV class="container"><DIV style="margin-top:5%"><H1><SPAN style="font-size:170%;color:#0B2161;">Indigoidine.</SPAN><SPAN class="text-muted" style="font-family:Arial, sans-serif; font-size:100%"> Proving Modularity of NRPS by Shuffling Domains.</SPAN></H1></DIV><DIV class="row"><DIV class="col-sm-6"><DIV class="jumbotron"><H2>Highlights</H2><P style="font-size:16px"><UL style="font-size:14px"><LI> Endogenous PPTase of <EM>E. coli</EM> was proven sufficient for activation of the <EM>P. lumninescens</EM> derived indigoidine synthetase indC
</LI><LI> Production of Indigoidine is improved by co-transformation of host strain with supplementary PPTases
</LI><LI> Synthetic T-Domains generated by consensus and guided random design yield functional indigoidine synthetases
</LI><LI> Domain shuffling works across modules derived from different pathways and host organisms
</LI><LI> Strong influence on the yield of indigoidine production was proven for the interaction of PPTase and T-domains 
                            </LI></UL></P></DIV></DIV><DIV class="col-sm-6"><DIV class="jumbotron abstract"><H2>Abstract</H2><P style="font-size:14px; text-align:justify">
                            An integral characteristic of synthetic biology yet often undermined is the ability to learn fundamental knowledge by systematically perturbing a biological system. Non-ribosomal peptide synthetases (NRPS) are predestinated for such a trial and error approach. Their hierarchical organization into modules and domains offer a unique opportunity to spin around their inherent logical assembly and observe if their functionality is preserved.
                            Following this idea, we prove the interchangeability of NRPS domains at the example of <EM>indC</EM> from <EM>Photorhabdus luminescens laumondii</EM> TT01 (DSM15139). The native NRPS domains have been replaced with domains from other bacterial organisms and fully synthetic domains. To quantify the NRPS efficiency we established an indigoidine assay based on OD measurement of the blue-colored pigment. Interestingly, we find that our data points out the dependence on the T-domain and the 4'-Phosphopanthetheinyl-transferases (PPTases), resulting in different levels of indigoidine synthesis. Furthermore, we introduce HiCT - High throughput protocols for circular polymerase extension Cloning and Transformation - a new standard for the assembly of combinatorial gene libraries (<A href="http://hdl.handle.net/1721.1/81332">RFC 99</A>).
                        </P></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row" style="float:left"><DIV id="tyrocidineText" class="col-sm-12"><H2 id="introduction">Introduction</H2><P>
                  Most modules of non-ribosomal peptide synthetase (NRPS) pathways consist of three domain types: condensation, adenylation and thiolation domain (see <A class="fancybox fancyFigure" title="NRPS module and domain structure and activation of T-domains." href="https://static.igem.org/mediawiki/2013/a/a9/Heidelberg_IndPD_Fig1.png" rel="gallery1">Figure 1a</A>), also called peptidyl-carrier-protein domain (PCP)-domain (reviewed in )). Remarkably, precisely this order of domains is highly conserved among different NRPS pathways except for the very first and last module of each NRPS. In the initial module, the A-domain is always followed by a T-domain. The last module of an NRPS usually ends with a TE-domain.  During the process of non-ribisomal peptide (NRP) synthesis, a new amino acid is first adenylated by the A-domain and then bound to the T-domain via a thioester bond. The C-domain catalyzes the condensation of the substrate - which is bound to the T-domain of the previous module - and the amino acid of the next module. The T-domain itself shows no substrate specificity but acts as a carrier domain, which keeps the peptide attached to the NRPS module complex. The core element of every T-domain is a conserved 4’-phosphopanthetheinylated (4’-PPT) serine. The 4’-PPT residue is added by a 4’-Phosphopanthetheinyl-transferase (PPTase), which converts the NRPS apo-enzyme to its active holo-form (see <A class="fancybox fancyFigure" title="NRPS module and domain structure and activation of T-domains." href="https://static.igem.org/mediawiki/2013/a/a9/Heidelberg_IndPD_Fig1.png" rel="gallery1">Figure 1b</A>).
                  </P><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/a/a9/Heidelberg_IndPD_Fig1.png"><FIGCAPTION><B>Figure 1</B>: NRPS module and domain structure and activation of T-domains. 
     
a) Basic structure of NRPS pathways
Typically, a NRPS is composed of 1 to about 10 single modules, of which each consists of a 
C-domain (condensation domain), an A-domain (adenylation domain) and a T-domain (thiolation 
domain). Moreover, the initial module lacks the C-domain and the last module of a pathway has 
an additional TE-domain (thioesterase domain), which cleaves the synthesized nonribosomal 
peptide from the last T-domain.

b) Activation of NRPS modules by 4'-Phosphopanthetheinylation of the T-domain.
Every NRPS module has to be activated by a 4'-Phosphopanthetheinyl-transferase (PPTase). 
which transfers the 4'-Phosphopanthetheinyl moiety of Coenzyme A to a conserved serine 
residue in the T-domain.
     </FIGCAPTION></A><P>
Besides these fundamental domains (C-domain, A-domain and T-domain), some NRPS modules incorporate additional domains enlarging the amount of potential catalytic reactions, such as cyclization, epimerization or oxidation of the amino acid [Reference]. 
For example, a single module of <EM>P. luminescens laumondii</EM> TT01 (DSM15139) contains an internal oxidation domain (Ox-domain) in its A-domain and a special TE-domain (<A class="fancybox fancyFigure" title="Exchange of the indC T-domain on a pSB1C3 derived expression plasmid" href="https://static.igem.org/mediawiki/2013/c/ca/Heidelberg_IndPD_Fig2.png" rel="gallery1">Figure 2a</A>). This enzyme first adenylates L-glutamine (A-domain), which is then attached to the T-domain. The TE-domain cleaves and catalyzes the cyclization of the substrate, which is further oxidized by the Ox-domain. The oxidation of two cyclic glutamines results in the formation of an insoluble small molecule (<A class="fancybox fancyFigure" title="Exchange of the indC T-domain on a pSB1C3 derived expression plasmid" href="https://static.igem.org/mediawiki/2013/c/ca/Heidelberg_IndPD_Fig2.png" rel="gallery1">Figure 2b</A>)[Reference].
</P><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/c/ca/Heidelberg_IndPD_Fig2.png"><FIGCAPTION><B>Figure 2</B>: Exchange of the indC T-domain on a pSB1C3 derived expression plasmid 
     The indigoidine synthetase indC from <EM>P. luminescens</EM> consists of an adenylation domain with an internal oxidation domain, a thiolation domain and a thioesterase domain. We replaced the indC T-domain with T-domains of other NRPS modules (one of which is the indigoidine synthetase bpsA from <EM>S. lavendulae</EM>) and seven synthetic T-domains, which were customized to be introduced to indC, respectively.
     </FIGCAPTION></A>
The small molecule produced by the pathway described above is a blue-colored pigment called <EM>indigoidine</EM>. Accordingly, the catalytic NRPS is referred to as <EM>indigoidine synthetase</EM> or <EM>blue pigment synthetase</EM> encoded by various bacterial strains such as <EM>S. lavendulae subsp. lavendulae</EM> (ATCC11924) or <EM>P. luminescens</EM> (). Previous publications showed that replacing the T-domain of the blue pigment synthetase bpsA from <EM>S. lavendulae</EM> with T-domains of other NRPS modules results in a loss of function, i.e. the engineered indigoidine synthetase does not produce the blue pigment (). So far the only T-domain exchange that resulted in a functional bpsA was achieved using the T-domain of entF - a gene involved in the enterobactin biosynthesis of <EM>E. coli</EM>. For this approach, random mutagenesis was performed to yield various mutated entF T-domains, some of which were capable to preserve the enzyme function when being introduced into bpsA (). Other studies revealed that it is possible to exchange the A-domains of NRPS modules in <EM>B. subtilis</EM> resulting in modified non-ribosomal peptide products ( ). Additionally, the selectivity of an NRPS module for specific amino acids could be modified by altering the conserved motif in the active site of the A-domain (). Furthermore, since the endogenous 4'-Phoshopanthetheinyl-transferase (PPTase) entD from <EM>E. coli</EM> has been reported to exhibit low efficiency in activating heterologous NRPS pathways, most research in the field of NRPS involves co-expression of another PPTase (), ).

<H2 id="results">Results </H2><H2><SPAN class="mw-headline" id="Expression_of_Functional_Indigoidine_Synthetase_indC_derived_from_P._lumninescens_in_five_substrains_of_E._coli">Expression of Functional Indigoidine Synthetase indC derived from <I>P. lumninescens</I> in five substrains of <I>E. coli</I></SPAN></H2><H3><SPAN class="mw-headline" id="Endogenous_PPTAse_of_E._coli_Is_Sufficient_for_Activation_of_the_P._lumninescens_Derived_Indigoidine_Synthetase_IndC">Endogenous PPTAse of <I>E. coli</I> Is Sufficient for Activation of the <I>P. lumninescens</I> Derived Indigoidine Synthetase IndC</SPAN></H3><P>The open reading frame of the native indigoidine synthetase indC was amplified from genomic DNA of <I>P. lumninescens</I> and cloned into a plasmid under the control of an lac-inducible promoter. This indC expression cassette was transformed into different substrains of <I>E. coli</I>, namely DH5alpha, MG1655, BAP1, TOP10 and NEB Turbo. All of these host strains express the endogenous PPTAse entD which is responsible for the transfer of the 4'-phosphopantetheine residue from coenzyme A to the apo-domain of EntF,  a T-domain in the enterobactin pathway(&lt;bib id=&quot;pmid:9214294&quot;/&gt;). As depicted in Figure 6a, entD does not exhibit strict substrate speceficity in being restricted to activating domains of the enterobactin pathway, but is able to activate the T-domain of indC as determined by the blue phenotype of the transformed cells. Except for NEB Turbo cells, all transformed host strains displayed a decelerated growth and significantly smaller colonies on plate when compared to the negative control. The blue phenotype developed late after transformation ranging from first blue colonies after 24 h and taking up to three days for visible poduction of the blue pigment. NEB Turbo showed regular colony growth and developed a strong blue phenotype upon induction with IPTG. As all host strains were able to express the functional indigoidine sythetase derived from a different species, further experiments were only conducted with one <I>E. coli</I> strain. Due to its simplicity in handling and sufficient expression of the constructs, the substrain TOP10 was chosen.  
</P><H3><SPAN class="mw-headline" id="Improved_Production_of_Indigoidine_by_Co-transformation_of_Host_Strain_with_Supplementary_PPTases">Improved Production of Indigoidine by Co-transformation of Host Strain with Supplementary PPTases</SPAN></H3><P>The expression of indC under activation by endC was sufficient for easy detection of indigoidine production on plates harboring indC-carrying cells. In order to determine, whether the amount of indigoidine production in the <I>E. coli</I>TOP10 cells is dependent on the quality of the interacting of indC with the PPTase, four PPTase dervied from varying origins were selected and amplified from the genome of the hosts of origin. <I>E. coli</I>TOP10 cells were co-transformed with plasmids coding for the different PPTases and the plasmid containing the expression cassette for indC. As reference for the endogenous PPTase activity served cells only transformed with the indC plasmid. Irrespective of the PPTase, growth of colonies was retarded. Remarkably however, colonies co-transformed with the PPTase plasmid remained of smaller size than the ones only carrying the indC construct. On the other side, indigoidine production was more diffuse in the latter cells with secretion of the blue pigment into the agar (Figure 6b, indC) and only slight blue-greenish coloring of the colonies. The four PPTases additionally introduced into the TOP10 cells were all shown to be functional (blue phenotype of the transformants, Figure 6b), but lead to the retention of most of the indigoidine within the cells. Colonies of cells transformed with thess constructs, were of convex shape and of distinct, dark blue color. Overall, cells carrying an additional PPTase showed increased indigoidine production compared to the cells relying on the endogenous entD. 
</P><H2><SPAN class="mw-headline" id="Synthetic_T-Domains_Generated_by_Consensus_and_Guided_Random_Design_Method_are_Functional">Synthetic T-Domains Generated by Consensus and Guided Random Design Method are Functional</SPAN></H2><P>The main structurel characteristic of NRPSs is their modular composition on different levels. The indogoidine synthetase indC is a one-module NRPS comprised of the three domains, namely AoxA, T and TE. Since the functionality of this NRPS is detectable by the bare eye, it offers a perfect and simple experimental set-up for proof of principle experiments regarding the interchangeability of domains from different NRPS. Out of the three domains in indC, the T-domain is suppossed to exhibit the least substrate specificity and was thus chosen for first domain shuffeling approaches. For the initial definition of T-domain boundaries of indC, we used PFAM, a web-tool which allows -amongst other functions- for the prediction of NRPS's module and domain boundaries. Following the boundary prediction, we choose a two-pronged domain shuffling approach: First, we transferred native T-domains derived from either different host species and/or NRPSs of entirely different function into the indC indigoidine synthetase. Second, we deviced three methods for the generation of synthetic T-domains based on different NRPS libraries generated by BLAST search against either specific subranges of host organisms or restricting the query sequence to be BLASTed.(link)
As depicted in Figure 6, both approaches lead principally to fully functional indCs. The synthetic T-domains 1, 3 and 4 showed the same diecreased growth and indigoidin production on plates as did the native T-domain derived from <I>P. lumninescens</I>. The colonies obtained after co-transformation with supplementary PPTase plasmid were small in size and of dark blue color. Compared to synthetic T-domain 5, indigoidin production started earlier (approximately after 24-30 hours). In contrast to the synthetic domains 1,3 and 4 which were designed by the consensus method and showed medium to high similarity to the sequence of origin, synthetic domain 5 was generated by the guided random method. Remarkably, even though 39 out of the 62 amino acids of the original T-domain were exchange, the indigoidine synthetase with this T-domain was still functional. Closer analysis of the sequence compared to the original indC T-domain sequence  showed, that the characteristics of the amino acid sequence, i. e. for instance polar or charged amino acids, were retained in 72% of the sequence. Also, the GGXS core sequence of the T-domain at which the activation by the PPTas occurs was conserved.
</P><H2><SPAN class="mw-headline" id="Domain_Shuffling_Works_across_Modules_Derived_from_Different_Pathways_and_Host_Organisms">Domain Shuffling Works across Modules Derived from Different Pathways and Host Organisms</SPAN></H2><P>Multiple web-tools exist which offer the prediction of NRPS module and domain boundaries. One of the most common used prediction tools is PFAM which we used as a starting point to determine the best method for defining domain boundaries. PFAM predicted large linker structures between the end of the A and the beginning of the T-domain (compare Figure 8, T-boundaries from B to two). Using these domain boundaries for the native T-domains did only yield one functional native T-domain(results not depicted). We tried to improve this yield by defining new T-domain boundaries based on the predictions of PFAM and multiple sequence alingments with the respective homology libraris at the predicted linker regions. Boundaries were set closer to the preceeding A-domain, at regions were less sequence conservation was observed. Figure 8 shows the indigoidine production after insertion of native T-domains with revised boundaries. T-domain boundary combinations A1, A2 and C1 yielded functional T-domains. As the indigoidine production and cell growth was best for the T-domains created with boundary combination A2, this boundary design was used for all subsequent cloning strategies. Figure 9 depicts the success of this boundary desgin as two additional native T-domains derived from delH4 and bpsA (indigoidine synthetase) led to functional indCs and the production of indigoidine. In addition, the native T-domain from plu2642 which was already shown to be functional(compare Figure 7) showed faster and increased indigoidine production (deep blue agar plate, lower right panel on Figure 9). The results obtained from this experiments proofed two concepts. First, domain shuffling is possible across different species as the T-domains of delH4 and bpsA were derived from <I>D. acidovorans</I> and <I>S. lavendulae lavendulae</I>, respectively and were functional in <I>E. coli</I>. Also, shuffling of domains from modules of different substrate specificity has been proofen herein. Second,  manually adjusting the boundaries predicted by PFAM based on MSA is a functional method to predict functional T-domain boundaries.
</P><H2><SPAN class="mw-headline" id="PPTase_and_T-domain_Interaction_Strongly_Influence_the_Yield_of_Indigoidine_Production">PPTase and T-domain Interaction Strongly Influence the Yield of Indigoidine Production</SPAN></H2><P>As the previous experiments of shuffled T-domains and different combinations of PPTAses showed, there are substantial differences in cell growth and indigoidine production when observed on plates. However, this observations were always of qualitative nature and did not give any insight into quantitative differences. We approached the quantification of indigoidine production in a time-resolved and highly-combinatorial manner: plasmids coding for indC containing all synthetic (4) and native T-domains (3) proven functional by the previous assays were co-transformed with the four functional PPTases. The indigoidine production over time (30 hours) was measured at its absorption maximum of 590 nm and corrected for the contribution of the cellular components in the medium as described in the methods. As Figure 10 shows, synthetic T-domains in combination with different PPTases lead to distinct differneces in indogoidin production. As a comparisons between the left and right panel of Figure 10 shows, PPTases working best with one T-domain might not lead to any indigoidine production when used with a indigoidine synthetase containing a different T-domain (Figure 10, pink line, delC). In addition, as distinctly visible in Figure 11, indigoidine production over time is not a strictly monoton function (blue line). After an indigoidin production peak at 16 hours, indigoidine production caused by indC containing the synthetic domain 4 decreases again. The indigoidin production in cells transformed with indC/synthetic T-domain 3 is still increasing.  



                  <A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/c/cf/Heidelberg_IndPD_Fig6.png"><FIGCAPTION><B>Figure 6</B>: Comparison between different <EM>E. coli</EM> strains and PPTases:
     a) Comparison of different <EM>E. coli</EM> strains examining growth and indigoidine production
The figure shows five different strains of <EM>E. coli</EM> that have been co-transformed with an indC expression plasmid and a sfp expression plasmid. The negative control is <EM>E. coli</EM> TOP10 without a plasmid. All transformants have been grown on LB agar for 48 hours at room temperature, cells were not induced. One can see that even without induction all strains express the indigoidine synthetase and produce the blue pigment indigoidine. However, the strains BAP1 and NEB Turbo grow faster in the first day, exhibiting a white phenotype (data not shown). Colonies on the plate of <EM>E. coli</EM> TOP10  are very small and dark blue/ black. Assuming that indigoidine production inhibits cell growth due to its toxicity, we concluded that TOP10 produced the most indigoidine among the strains we tested. We used <EM>E. coli</EM> TOP10 for the following experiments.

b) Comparison between different PPTases concerning overall indigoidine production
The Figure shows ''E. coli'' TOP10 cells co-transformed with indC and four different PPTases (sfp, svp, entD and delC), respectively. The image bottom left shows ''E. coli'' TOP10 cells without additional PPTase and the negative control is TOP10 without a plasmid.

     </FIGCAPTION></A><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/f/f1/Heidelberg_IndPD_Fig7.png"><FIGCAPTION><B>Figure 7: Modified variants of indC with replaced T-domains</B>
     We replaced the indC T-domain with both the T-domains of other native NRPS modules (entF, delH, tycC, tycA, bpsA, plu2642 and plu2670) and synthetic T-domains. The figure shows the five modified versions of indC that remain the enzyme function, thus resulting in a blue phenotype of transformed ''E. coli'' TOP10. The cells have been co-transformed with a plasmid containing the respective engineered variant of indC and a second plasmid coding for the PPTase sfp, svp, entD and delC. The figure shows representative results; the total 85 transformation results can be found in the indigoidine notebook, week 17.
     </FIGCAPTION></A><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/e/eb/Heidelberg_IndPD_Fig8.png"><FIGCAPTION><B>Figure 8: Determination of required domain borders for T-domain exchange</B>
     a) Definition of different domain border combinations for T-domain exchanges
The figure shows a sequence alignment of the indC and bpsA amino acid sequences.
The alignment was created using clustalO (http://www.ebi.ac.uk/Tools/msa/clustalo/) with
standard parameters. The lines marked A, B and C reflect the borders we used between the A- 
and the T-domain, whereas those marked, 1, 2, 3 and 4 reflect the borders between the T- and 
the TE-domain. In total we tried all twelve combinations of a domain border {A, B, C} and a 
domain border {1, 2, 3, 4}, replacing the sequence inbetween with the respective part of bpsA.

b) E. coli TOP10 co-transformed with modified versions of indC and the PPTase sfp
The co-tranformation of the modified indC-(bpsA-T) plasmids described above with a second 
plasmid coding for the PPTase Sfp shows that only three domain border combinations can be 
used for exchanging the indC T-domain with the T-domain of bpsA. These are the 
combinations A1, A2 and C1. We applied combination A2 for further T-domain exchanges. 
     </FIGCAPTION></A><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/6/6e/Heidelberg_IndPD_Fig9.png"><FIGCAPTION><B>Figure 9: Applying optimized domain border combinations by T-domain exchange of native NRPS T-domains</B>
     As described above, we determined an optimized domain border combination for the exchange of the native indC T-domain with the T-domain of the indigoidine synthetase bpsA. We applied this border combination (previously referred to as ''A2'') to the T-domains of the NRPS modules entF, delH4, delH5, tycA1, tycC6, plu2642 and plu2670, replacing the indC T-domain with the respective fragments of those modules. This figure shows three transformants with a blue phenotype in which the indC T-domain was exchanged by the T-domain of the respective NRPS module. The pictures were taken after 60 hours of incubation at room temperature. Once more one can see the differences in growth kinetics due to the production of indigoidine: Cells expressing the indC variant with the T-domain of bpsA  grow very slow and form small and dark blue colonies, whereas cells expressing other variants grow faster. Comparing the images on the very right, we suggest that cells expressing indC with the T-domain of plu2642 produce the most indigoidine in the given timeframe, compared to both the delH4- and the bpsA-variant. The combination of the plu2642 T-domain and the indC indigoidine synthetase seems to be ideal, concerning both indigoidine production and overall growth.
     </FIGCAPTION></A><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/4/4d/Heidelberg_IndPD_Fig10.png"><FIGCAPTION><B>Figure 10</B>:  Synthetic T-domains in combination with different PPTases lead to distinct differneces in indogoidin production.
     </FIGCAPTION></A><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/3/38/Heidelberg_IndPD_Fig11.png"><FIGCAPTION><B>Figure 11</B>: Indigoidine production over time is not a strictly monoton function
     </FIGCAPTION></A></P><H2 id="discussion">Discussion</H2>
In this subproject, we wanted to set the basis for engineering entirely synthetic NRPS modules composed of user-defined domains.

As model system, we used the unimodular indigoidine synthetase NRPS from <EM>P. luminescens subsp. Laumondii</EM> TT01. We predicted the modular composition and domain borders of IndC using our own NRPS-Designer software.

We then started by replacing the native IndC T-domain with T-domains derived from different NRPS pathways from different bacterial strains, among those the T-domain from the BpsA indigoidine synthetase from <EM>S. lavendulae</EM> ATCC11924. Constructs were transformed into E. coli alongside with an PPTase expression cassette in order to screen for functional IndC variants. As hoped, a subset of the natural T-domains were functioning in context of the IndC scaffold module, leading to indigoidine production and thereby blue coloring of colonies and corresponding liquid cultures. We then engineered a variety of synthetic T-domains derived from consensus sequences of different natural T-domains. Again, a subset of these T-domains successfully maintained indigoidine production (<A class="fancybox fancyFigure" title="Figure 9" href="https://static.igem.org/mediawiki/2013/6/6e/Heidelberg_IndPD_Fig9.png" rel="gallery1">Fig. 9</A>).

Notably, one of our engineered IndC construcats showed an indigoidine production even higher compared to the wild-type IndC (T-domain Plu2642; <A class="fancybox fancyFigure" title="Figure 3" href="https://static.igem.org/mediawiki/2013/6/6c/Heidelberg_IndPD_Fig3.png" rel="gallery1">Figure 3</A>).

This is particularly remarkably as our results contradict to previous studies of NRPS domains that reported the native T-domain of the indigoidine synthetase BpsA to be absolutely essential for protein function (and therefore not replaceable by other T-domains).  ([Owen2012]).

However, to our surprise, the BpsA T-domain-containing IndC construct did not yield any detectable indigoidine production, although BpsA shares strong sequence homology with IndC. We hypothesized, that the selection of the exact border could be critical for maintaining domain functionality when introduced into a novel NRPS module scaffold. Therefore, we amplified different BpsA T-domain variants differing in their domain border and introduced them into the IndC scaffold. Remarkably, a subset of the resulting IndC variants showed successful indigoidine production.

We thus revaluated all native and synthetic T-domains in light of this finding and performed a second screening round in which we were able to rescue even more functional IndC variants, proofing our abovementioned hypothesis.

We also co-transformed all engineered IndC construct bearing different natural and synthetic T-domains with four different PPTase expression constructs. To our surprise, the T-domains used not only determined general efficiency of indigoidine production, but also the efficiency of NRPS activation by the different activating PPTases.

In conclusion we were able to demonstrate, that it is indeed possible to replace single Domains from NRPS modules, while preserving or even enhancing its functionality. In addition, we established an approach for the design of synthetic T-domains and proved their functionality by introducing them into the indigoidine synthetase indC scaffold. Moreover, we established a high throughput protocol for circular polymerase extension cloning and transformation (Hi-CT) (<A href="http://hdl.handle.net/1721.1/81332">BBF RFC 99</A>), which we applied for our domain shuffling approach. In summary, we created a library of 58 engineered indC variants. In addition we perforemd measurement of blue pigement production over time, which gave us novel insights in how NRPS domains should be designed, where the domain borders between different domains in a single NRPS module have to be set and which domains from respective NRPS pathways and bacterial strains can be used, when creating novel engineered NRPS pathways. We implemented our findings into the &quot;NRPS-Designer&quot; Software, so that the underlying algorithm for NRPS design takes into consideration the abovementioned findings (e.g. domain border setting) which are certainly crucial for successful in silico prediction of functional NRPSs.

Thereby, our project pioneers the research on high-throughput methods for creation of synthetic NRPS modules composed of user-defined domains. We believe that our findings will highly contribute to future development of custom NRPSs.


<H2>Methods</H2><STRONG>Table 1: Bacterial strains and genes of interest derived thereof.</STRONG> The indigoidine synthetase bpsA was kindly supplied by the Fussenegger lab at ETH Zurich.
<TABLE class="wikitable"><TBODY><TR><TH> Strain </TH><TH> Gene </TH><TH> Function
</TH></TR><TR><TD><EM>Photorhabdus luminescens laumondii</EM> TT01 DSM15139 </TD><TD> indC </TD><TD> Indigoidine synthetase
</TD></TR><TR><TD><EM>Streptomyces lavendulae lavendulae</EM></TD><TD> bpsA </TD><TD> Indigoidine synthetase
</TD></TR><TR><TD><EM>Photorhabdus luminescens laumondii</EM> TT01 DSM15139  </TD><TD>ngrA </TD><TD> PPTase
</TD></TR><TR><TD><EM>Escherischia coli</EM> BAP1 </TD><TD>sfp </TD><TD> PPTase
</TD></TR><TR><TD><EM>Streptomyces verticillus</EM> ATCC15003 </TD><TD>svp </TD><TD> PPTase
</TD></TR><TR><TD><EM>Escherischia coli</EM> MG1655 </TD><TD>entD </TD><TD> PPTase
</TD></TR><TR><TD><EM> Delftia acidovorans</EM> SPH-1 </TD><TD>delC </TD><TD> PPTase
</TD></TR></TBODY></TABLE><H3>Cloning Strategy</H3>
We assembled the different indC variants on a chloramphenicol resistance backbone (pSB1C3) with an IPTG-inducable lac-promoter, the ribosome binding site BBa_B0034 and the coding sequence of the respective indC variant. The indC plasmids should be co-transformed with a PPTase construct to get a significant and fast indigoidine production. Therefore, we used a second plasmid backbone carrying a kanamycin resistance (pSB3K3). We assembled five pSB3K3 derived plasmids, each carrying an expression cassette with an IPTG induceable lac-promotor, the BBa_B0029 ribosome binding site and the coding sequence of the respective PPTase (sfp, svp, entD, delC and ngrA; see Table 1).
We used <EM>E. coli</EM> TOP10 for co-transformations of the possible combination of the indC variants (2) and all PPTase plasmids (5). 

<H3>Circular Polymerase Extension Cloning</H3>
Circular Polymerase Extension Cloning (CPEC) is a sequence-independent cloning method based on homologous recombination of double-strand DNA overlaps of vector and insert(s) (). It is suitable for the generation of combinatorial, synthetic construct libraries as it allows for multi-fragment assembly in an accurate, efficient and economical manner.  
CPEC relies on a simple polymerase extension of the DNA fragments to be assembled. Crucial to this concept is the design of vector and insert fragments which MUST share overlapping regions at the ends (<A class="fancybox fancyFigure" title="CPEC method" href="https://static.igem.org/mediawiki/2013/6/6c/Heidelberg_IndPD_Fig3.png" rel="gallery1">Figure 1.1</A>). In a single reaction set-up, insert DNA fragments and linear vector are heat denaturized and allowed to anneal at elevated temperature, resulting in specific hybridized insert-vector constructs (<A class="fancybox fancyFigure" title="CPEC method" href="https://static.igem.org/mediawiki/2013/6/6c/Heidelberg_IndPD_Fig3.png" rel="gallery1">Figure 1.2</A>). Subsequently, the single-strand hybrid constructs are extended under PCR-elongation conditions (72 °C for 20 s/kbp of longest fragment) which yield completely assembled, double-stranded circular constructs (<A class="fancybox fancyFigure" title="CPEC method" href="https://static.igem.org/mediawiki/2013/6/6c/Heidelberg_IndPD_Fig3.png" rel="gallery1">Figure 1.3</A>) ready for transformation into competent cells. The single strands nicks introduced on each strand due to the unidirectional nature of the polymerase chain reaction will be removed by endogenous ligases upon transformation into <EM>Escherichia coli</EM>.

         <A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/6/6c/Heidelberg_IndPD_Fig3.png"><FIGCAPTION><B>Figure 3: Circular polymerase extension cloning: a sequence-independent, homologous recombination based cloning approach</B>Insert and backbone fragments sharing overlapping regions at their ends are transferred into a single reaction set-up in molecular ratios determined by equation 1 (compare to 5.1.4). 2)  The insert/backbone reaction mixture is heat-denaturized and subsequently cooled down to 53°C to allow for annealing of the complementary overlaps.  3)  By polymerase chain reaction, the single strand hybrid-regions are filled up to double strands yielding circular, double-stranded molecules with nicks at overlapping regions. 4) Plasmids resulting from CPEC can be used directly for transformation. Figure adapted from [Quan &amp; Tian, 2009 ()]
     </FIGCAPTION></A>

We provide instructions (<A href="http://hdl.handle.net/1721.1/81332">RFC 99</A>) for a rapid and cost efficient cloning and transformation method based on CPEC which allows for the manufacturing of multi-fragment plasmid constructs in a parallelized manner: High Throughput Circular Extension Cloning and Transformation (HiCT)

CPEC was performed according to the following protocol:
The total mass of DNA used per CPEC reaction varied between 50 to 200 ng. The insert to backbone molar ratio was 3:1 for insert-backbone and 1:1 for insert-insert molar ratio. Conversion from mass concentration of fragments to molar concentration was done using the formula: cM = c*10^6/(n*660), where c is the measured oligonucleotide concentration [ng/µl], n is the number of dinucleotides of the fragment and cM is the resulting concentration [nM]. 

The final reaction volume was adjusted to 6 µl with polymerase master mix (Phusion® High-Fidelity PCR Master Mix with HF Buffer, NEB #M0531S/L). The CPEC reaction was carried out under the following conditions:


<UL><LI> initial denaturation at 98°C for 30 s
</LI><LI> 5 cycles with:
</LI><LI> denaturation step at 98°C for 5 s.
</LI><LI> annealing step at 53°C for 15 s
</LI><LI> elongation/filling up step at 72°C for 20 s/kbp of longest fragment.
</LI></UL><LI> final extension at 72°C for three  times the calculated elongation time.
</LI><LI> (Optional: Hold at 12°C )
</LI><P>

After CPEC, 5 µl of of the reaction mixture were used for transformation. The remaining volume was used for quality check on a gel with small pockets (10 to 20 µl in volume).

The following primers were used for all CPEC experiments into standard BioBrick backbones: BBa_J04450_stem_loop_fw, Bba_J04450_B0034-RBS_ATG_rv, Bba_J04450_B0029-RBS_ATG_rv. The reverse primers (rv) differ in the ribosomal binding sites they introduce: Bba_J04450_B0034-RBS_ATG_rv contains the ribosomal binding site used in J04450, Bba_J04450_B0029-RBS_ATG_rv introduces the ribosomal binding site B0029 which is of weaker than B0034.


</P><H3>Generation of the ccdB-Ind construct</H3>
To minimize the background colonies when exchanging the T-domain of the indigoidine synthetase we generated the ccdB-Ind plasmid  where we replaced the indC T-domain with the ccdB gene (Modul structure: AoxA-ccdb-TE) which kills <EM>E. coli</EM> TOP10 cells but not <EM>E. coli</EM> OneShot ccdB survival cells. Test-transformation in both <EM>E. coli</EM> TOP10 and the <EM>E. coli</EM> OneShot ccdB survival cells showed that background colonies could be eliminated by this strategy (Plattenbild top10 vs survival cells).
We used the ccdB-Ind for all further CPEC experiments aiming to swap T-domains. Primers for the backbone CPEC fragments were designed to facilitate the amplification of the entire ccdB-Ind plasmid while omitting the ccdB sequence (compare to Figure??). Assembly of the finale indigoidine synthase products with exchanged T-domain was achieved by CPEC as described or above or HiCT (<A href="http://hdl.handle.net/1721.1/81332">RFC 99</A>).

<H3>Examination of T-domain borders</H3>
We exchanged the T-domain of indC with the T-domain of bpsA and varied the size of the exchanged DNA sequence, thus examining several domain borders (compare to Figure ??). We used the CPEC assembly method and the indC-ccdB plasmid for this approach.
<H3>Test of various T-domains from different NRPS modules</H3>
For the investigation of additional T-domains from less related NRPS modules, we selected the border combination b31??? which was positive in the test with bpsA. We used the T-domains of the following genes:


<STRONG>Table 2: Genes of which T-domains have been extracted and introduced to indC</STRONG><TABLE class="wikitable"><TBODY><TR><TH>Gene </TH><TH>Organism </TH><TH>	Original function
</TH></TR><TR><TD> entF	</TD><TD><EM>Escherichia coli</EM> K-12</TD><TD>	NRPS module of enterobactin synthesis pathway
</TD></TR><TR><TD>tycA1</TD><TD><EM>Brevibacillus parabrevis</EM></TD><TD>		1st module in tyrocidine synthesis cluster
</TD></TR><TR><TD>tycC6</TD><TD><EM>Brevibacillus parabrevis</EM></TD><TD>		Last module in tyrocidine synthesis cluster
</TD></TR><TR><TD>delH4</TD><TD><EM>Delftia acidovorans</EM> SPH-1</TD><TD>		2nd but last module in delftibactin synthesis cluster
</TD></TR><TR><TD>delH5</TD><TD><EM>Delftia acidovorans</EM> SPH-1</TD><TD>		Last module in delftibaction synthesis cluster
</TD></TR><TR><TD>plu2642</TD><TD><EM>P. luminescens</EM> DSM15139</TD><TD>		NRPS of unknown function (one module: A-T-TE)
</TD></TR><TR><TD>plu2670</TD><TD><EM>P. luminescens</EM> DSM15139</TD><TD>		module of NRPS pathway of unknown function
</TD></TR></TBODY></TABLE><P>

All T-domains from the respective genomes were amplified using CPEC primers with a uniform 5’-end and a 3’-end specific for the respective gene. For the assembly of the hybrid-indigoidine synthetases by CPEC, the indC-ccdB construct was used.

</P><H3>Creation of synthetic T-domains</H3>
All R scripts used in the following sections are based on R version R-3.0.1. 

Different assumptions about the evolutionary conservation of T-domains were examined: i) conservation of a specific module across different species, ii) conservation of T-domains across different modules for the same species, iii) conservation of T-domains across different species,  iv) conservation of similar modules across different species. According to these three assumptions, different libraries of homologous protein sequences were generated using ncbi protein BLAST (blast.ncbi.nlm.nih.gov) with standard parameters:

<OL><LI> query sequence: indC; Search set: non-redundant protein sequences without organism restriction
</LI><LI> query sequence: indC T-domain; Search set: non-redundant protein sequences within <EM>P. luminescens</EM></LI><LI> query sequence: indC T-domain; Search set: non-redundant protein sequences without organism restriction
</LI><LI> query sequences: indC, bpsA, entF, delH5 and tycC6; Search set: non-redundant protein sequences without organism restriction;
</LI></OL><P>
The 50 closest related protein sequences contained in each the library were subjected to a multiple sequence alignment (MSA) using clustalO (<A href="http://www.ebi.ac.uk/Tools/msa/clustalo/">http://www.ebi.ac.uk/Tools/msa/clustalo/</A>). with standard parameters for protein alignments. For library generation iv), each query sequence was BLASTed separately and the 50 best results of each query were combined i.e. a total of 250 sequences for the MSA. 

After library generation, the following three methods were employed to design different synthetic T-domains.

</P><H4>Consensus method</H4><H4>
Based on the  .clustal file obtained from the MSA of the homology libraries, a consensus sequence using the UGENE software (<A href="http://ugene.unipro.ru/">http://ugene.unipro.ru/</A>) with a threshold of 50% was created (i.e. if an amino acid appears in 50% or more of all sequences at a specific position it is considered as a consensus amino acid). For the creation of the synthetic T-domains, this consensus sequence was used to fill the gaps where there was no consensus amino acid with the original amino acid from the indC T-domain. By this approach, T-domains were generated which might deviate from the original sequence at positions with at least average conservation but coreespond to the original one if there is less conservation.  


</H4><H4>Guided random method</H4>
In this approach, the multiple sequence alignments (MSA) generated by the consensus method was used. Implemented in R [Referenz], a position-specific profile was generated which has the same length as the MSA and contains the rate at which amino acids occur at any given position of the sequence alignment. The synthetic T-domain is created by position-wise generation of the sequence where the probability of choosing an amino acid at a given position is determined by the rate in the profile.


<H4>Randomized generation method</H4>
For generation of synthetic sequences by the randomized generation method, every amino acid was assigned a score of 1 or 0, i.e. occuring at least ones or not at all at a given position in the MSA. In the subsequent generation of the synthetic T-domain sequence of the synthetic domain, any amino acid assigned 1 had the same likelihood of being chosen at this position. 
<P>
Seven synthetic T-domains were designed based on differnt combinations of the homology libraries and sequence generation methods.
</P><STRONG>Table 3: Overview of the homology libraries and sequence generation methods employed for the generation of seven synthetic T-domains</STRONG><TABLE class="wikitable"><TBODY><TR><TH>Domain ID </TH><TH>Homology library </TH><TH>Sequence generation method
</TH></TR><TR><TD>synT1</TD><TD>	library i</TD><TD>	consensus
</TD></TR><TR><TD>synT2</TD><TD>	library ii</TD><TD>		consensus
</TD></TR><TR><TD>synT3</TD><TD>	library iii</TD><TD>		consensus
</TD></TR><TR><TD>synT4</TD><TD>	library iv</TD><TD>		consensus
</TD></TR><TR><TD>synT5</TD><TD>	library i</TD><TD>		guided random
</TD></TR><TR><TD>synT6</TD><TD>	library iv</TD><TD>		guided random
</TD></TR><TR><TD>synT7</TD><TD>	library i</TD><TD>		randomized generation
</TD></TR></TBODY></TABLE><P>
Figure ??shows the multiple sequence alignment of the seven synthetic T-domains and the native indC T-domain.

After the generation of the T-domain amino acid sequences, the OPTIMIZER web-tool(<A href="http://genomes.urv.es/OPTIMIZER/">http://genomes.urv.es/OPTIMIZER/</A>) was used to obtain the corresponding DNA sequence. <EM>E. coli</EM> K-12 was set as strain for codon optimization and <EM>most frequent</EM> was chosen as codon option. The generated DNA sequence was cured from internal RFC10 cutting sites and CPEC cloning overhang required for the T-domain swapping into the ccdb construct were introduced. The synthetic T-domains were ordered at IDT (Integrated DNA Technologies, Coralville, Iowa). In order to obtain sufficient amounts of DNA, the synthetic T-domains were amplified via PCR.

IndC-hybrid constructs of the native IndC with exchange of the native T-domain by the synthetic variants were assembled using CPEC and the indC-ccdB construct as backbone. The synthetic T-domains were amplified for CPEC using the same primers as for the native indC T-domain.
</P><STRONG>Table 4: <EM>Overview of primers that MAY be utilized for backbone linearization:</EM> The reverse primers (rv) differ in
the ribosomal binding sites they introduce: BBa_J04450_B0034-RBS_ATG_rv contains the ribosomal binding site
B0034, BBa_J04450_B0029-RBS_ATG_rv introduces the ribosomal binding site B0029 which is weaker than B0034.
Note that the 5' overhangs (underlined) of the reverse primers (rv) already include the start codon (depicted in bold) of
the coding sequence to be introduced as insert into the corresponding backbone. The resulting expression cassette will
be driven by the Plac promoter (R0010).
</STRONG><TABLE class="wikitable"><TBODY><TR><TH>Primer </TH><TH>Primer sequence(5’ --&gt; 3’)</TH><TH>Cutting site
</TH></TR><TR><TD>BBa_J04450_stem_loop_fw</TD><TD><U>TAATGA <I>GCTAGC</I></U> TAATAACGCTGATAGTGCTAGTG</TD><TD><I>NheI</I></TD></TR><TR><TD>BBa_J04450_B0034-RBS_ATG_rv</TD><TD><U>CAT <I>GGTACC</I></U> TTTCTCCTCTTT CTCTAGTATGTGTG</TD><TD><I>KpnI</I></TD></TR><TR><TD>BBa_J04450_B0029-RBS_ATG_rv</TD><TD><U>CAT <I>GGATCC</I> GGTTTCCTGTGTGAA</U> CTCTAGTATGTGTGAAATTGTTATCC</TD><TD><I>NheI</I></TD></TR></TBODY></TABLE><H3>Quantitative indigoidine production assay</H3><H4>1. OD MEASUREMENT by TECAN plate reader</H4>
96-well plates are prepared with 100 μl LB-medium/well containing appropriate antibiotics (chloramphenicol and kanamycin for the indigoidine and PPTase contrcuts, respectively) and each well is inoculated with single colonies (in duplicates) from plates positive for the co-tansformation experiments i.e. from plates with blue colonies. Two sets of negative controls are also inoculated on the plate: First, pure medium serving as the baseline for background correction for the OD measurements. Second, transformation controls accounting for potential differences in cell growth due to expression of proteins contained on the plasmids, i.e. the antibitotic resistance gene and IndC. In this set of controls, the plasmid used in co-transformation with the PPTase plasmid contains IndC-constructs carrying a randomly generated sequence instead of the T-domain.  A second 96 well plate was prepared with 180 µl LB-medium/well for the measurement itself. The 96-well plate containing the pre-cultures of the co-transformed colonies was inoculated for 24 hours at 37°C. Subsequently, 20 µl of the pre-culture was transferred to the measurement plate. The absorbance of the bacterial cultures was measured at wavelengths ranging from 400 nm to 800 nm in intervals of 10 nm for each well every 30 min for 30 hours at 30°C in a Tecan infinite M200 plate reader. For the measurement plate, Greiner 96-well flat black plates with a clear lid were used.

<H4>2. Data analysis</H4><P>
Detecting the amount of the NRP expressed by the bacterial host strain is desirable. By tagging the NRP with indigoidine, the amount of the fusion peptide can be determined by quantifying the amount of blue pigment present in the cells. As the amount of blue pigment is proportional to the amount of the NRP of interest, a method for the quantification of the blue pigment will yield information about the expression of the NRP. Quantification of the pure indigoidine pigment can be easily achieved by optical density (OD) measurements at its maximum wavelength of about 590 nm. 
In cellular culture, indigoidine quantification by OD measurements is impaired. Cellular density of liquid cultures is standardly measured as the optical density (OD) at a wave length of 600 nm, i. e.  the absorption peak of indigoidine interferes with the measurement of cell density at the preferred wave length (compare to Figure 3, grey dashed line).   Thus, for measurement of NRP expression without time consuming a priori purification of the tagged-protein, a method to separate the cellular and pigment-derived contributions to the OD is required (compare to Figure 3, brown and blue lines, respectively).  The method of choice, as described by Myers et al.[2013], requires the OD measurement of  cell culture at two distinct wavelengths: the robust wave length ODR and the sensitive wave length ODS.  The concentration of indigoidine will have to be deducted from measurements at ODS = 590 nm:
&lt;math&gt;
OD_{S,+P}
&lt;/math&gt;
[Indigoidine]= 〖〖OD〗_(S,+P)-OD〗_(S,-P)
with 〖OD〗_(S,+P) being the overall OD measurement and 〖OD〗_(S,-P) being the scattering contribution of the cellular components at the sensitive OD. 
The scattering contribution of the cellular compenents at ODS  (ODS,-P ) can be calculated from the scattering contribution measured at the robust wave length according to the following formula: 
[[File: 
The correction factor δ is be determined by measuring the OD of pure cellular culture without indigoidine at both the wavelength  〖OD〗_(S,-P) and 〖OD〗_R and calculating their ratio. 
Finally, the indigoidine production can be determined as
[[File:Heidelberg_5.png]]

For the calculation of the cellular component when measuring indigoidine producing liquid cell cultures, OD measurement at 800 nm as robust wavelength is recommended. By the approach described above, quantitative observation of the indigoidine production in a liquid culture over time as well as the indigoidine production in relation to the cell growth can be conducted. 

Background correction i. e. the contribution of the culture medium to the OD measurement is achieved by subtracting the mean of pure culture medium replicates from all OD values measured.
</P><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/4/4b/Heidelberg_IndPD_Fig4.png"><FIGCAPTION><B>Figure 4</B>: 
     </FIGCAPTION></A><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/6/64/Heidelberg_IndPD_Fig5.png"><FIGCAPTION><B>Figure 5</B>: Quantification of dye in cellular culture by OD measurements at robust and sensitive wavelengths. The contribution of the scattering by the cellular components at the sensitive wavelength, i.e. 590 nm for indigoidine has to be subtracted from the overall OD at this wavelength. For a detailed description of the calculation refer to text below.
Figure adopted from [Myers, 2013] 
     </FIGCAPTION></A><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/7/75/Heidelberg_IndPD_Fig_multiplot.png"><FIGCAPTION><B>Figure 12</B>: 
     </FIGCAPTION></A></DIV><DIV class="references jumbotron" style="margin-top:5%"><P>1. Fischbach MA, Walsh CT (2006) Assembly-line enzymology for
polyketide and nonribosomal Peptide antibiotics: logic, machinery, and
mechanisms. Chem Rev 106: 3468–3496.</P><P>2. Takahashi H, Kumagai T, Kitani K, Mori M, Matoba Y, et al.
(2007) Cloning and characterization of a Streptomyces single module
type non-ribosomal peptide synthetase catalyzing a blue pigment
synthesis. In:. Vol. 282. pp. 9073–9081.</P><P>3. Brachmann AO, Kirchner F, Kegler C, Kinski SC, Schmitt I, et al.
(2012) Triggering the production of the cryptic blue pigment
indigoidine from Photorhabdus luminescens. In:. Vol. 157. pp.
96–99.</P><P>4. Owen JG, Robins KJ, Parachin NS, Ackerley DF (2012) A functional
screen for recovery of 4’-phosphopantetheinyl transferase and
associated natural product biosynthesis genes from metagenome
libraries. In:. Vol. 14. pp. 1198–1209.</P><P>5. Doekel S, Marahiel MA (2000) Dipeptide formation on engineered
hybrid peptide synthetases. In:. Vol. 7. pp. 373–384.</P><P>6. Thirlway J, Lewis R, Nunns L, Al Nakeeb M, Styles M, et al.
(2012) Introduction of a non-natural amino acid into a nonribosomal
peptide antibiotic by modification of adenylation domain specificity.
In:. Vol. 51. pp. 7181–7184.</P><P>7. Pfeifer BA, Admiraal SJ, Gramajo H, Cane DE, Khosla C (2001)
Biosynthesis of complex polyketides in a metabolically engineered
strain of E. coli. In:. Vol. 291. pp. 1790–1792.</P></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12"><DIV class="imageMap" aria-haspopup="true"><CENTER>Thanks to </CENTER></DIV></DIV></DIV><DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Heidelberg/Project/Indigoidine">http://2013.igem.org/Team:Heidelberg/Project/Indigoidine</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Project/Indigoidine" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Project/Indigoidine" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Project/Indigoidine&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Project/Indigoidine&amp;oldid=320725" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>