"
Page
Discussion
View source
History
teams
Log in
 
Team:Heidelberg/Templates/MM week18p
From 2013.igem.org
Contents
1 2013-08-26
2 2013-08-27
3 2013-08-28
4 2013-08-29
5 2013-08-30
6 2013-08-31
7 2013-09-01
2013-08-26
Lane 1: NEB 2-Log, Lane 2: pIK6.1 digested with HindIII+BamHI; lane 3: pIK6.2 digested with HindIII+BamHI
make miniPreps of colonies 1 and 2: 71.4 ng/µl and 75.1 ng/µl in 38 µl
digest with HindIII+BamHI (3 µl DNA, 0.5 µl of each enzyme, 20 µl total); expected: 4.1 kb, 2.1 kb, 0.9 kb
no 0.9 kb fragment => no BamHI site in KanR gene, 2.1 fragment visible corresponds to acca-sfp => insert completely within backbone
co-transform TOP10 with pIK6.1+pRB21
sequences of pIK1.3 arrived: PIK1.3-2013-08-26.zip , PIK1.3-2013-08-26 VF2.clustal.txt , PIK1.3-2013-08-26 VR.clustal.txt
point mutation in permeability device, interestingly, both the frameshift in pIK1.2 and this point mutation create a stop codon at the beginning of the permeability device
send pIK1.4 to sequencing with primer VF2
2013-08-27
Lane 1: NEB 2-log, lane 2: pSB3K3-BBa_J04450 digested with EcoRI+SpeI
TOP10-pIK6.1-pRB21 are not blue, but the indigoidine group is having problems with pRB21 -> pRB21 might not work
sequence of pIK1.4 arrived: PIK1.4-2013-08-27.zip , PIK1.4-2013-08-27 VF2.clustal.txt
again frame-shift in permeability device -> send pIK1.6, pIK1.7, pIK1.10 to sequencing with primer VF2
make miniPrep of pSB3K3: 54.7 ng/µl in 38 µl
digest pSB3K3 with EcoRI+SpeI (20 µl total volume, 5 µl DNA, 0.5 µl of each enzyme)
gel-purify, ligate with pIK2.6 fragment from 2013-08-24 at RT for 1h -> pIK7:
what
µl
pSB3K3
9
pIK2.6
8
T4 ligase
1 µl
T4 ligase buffer
2 µl
heat-inactivate: 75°C for 5 min
transform 10 µl of ligation into TOP10, plate on Kan, grow at 37°C
2013-08-28
Colony-PCR of TOP10 transformed with pSB3K3 ligated with pIK2 (-> pIK7) and primers VF2+IK25. Lane 1: NEB 2-log; lanes 2-6: pIK7
pick colonies, run colony-PCR with primers VF2+IK25 (expected: 1kb, iTaq, 20 µl total volume):
Cycles
temperature [°C]
Time [s]
1
95
300
35
95
30
54
30
72
90
1
72
600
1
10
inf
grow in 2xYT+Kan at 37°C
2013-08-29
Lane 1: NEB 2-Log, Lane 2: pIK7.1 digested with HindIII+BamHI; lane 3: pIK7.2 digested with HindIII+BamHI
sequences of pIK1 arrived: PIK1.6-pIK1.7-pIK1.10-2013-08-29.zip , PIK1.6-2013-08-29 VF2.clustal.txt , PIK1.7-2013-08-29 VF2.clustal.txt , PIK1.10-2013-08-29 VF2.clustal.txt
frame-shift in pIK1.6, point mutations in pIK1.7, pIK1.10
perform miniPreps of pIK7 -> ca. 50 ng/µl in 38 µl
digest 5 µl of pIK7.1, pIK7.2 with BamHI+HindIII (0.5 µl of each enzyme, 20 µl total volume)
bands at right positions (expected: 4.1, 2.1, 0.8 kb)
prepare glycerol stock of TOP10-pIK7.1
2013-08-30
electroporate electrocompetent DH10ß with 1 µl, 14 µl of 1:4 Gibson mix for pIK1 from 2013-08-07
plate 10 µl, rest of each electroporation on Cm, grow at 37°C
2013-08-31
very large and small colonies present, after heat-schock transformation last time only large colonies present
pick 5 small colonies from each plate, grow in 2xYT+Cm (no 2xYT left => last 3 colonies from 14 µl / rest plate grown in LB+Cm) at 37°C
2013-09-01
pIK1.12 did not grow
make miniPreps of all cultures -> concentrations about 200-300 ng/µl in 37.5 µl (except pIK1.14: 50 ng/µl)
digest with BamHI+HindIII (20 µl total volume, 0.5 µl enzyme, 1 µl DNA)
expected fragments: 2151 bp + 7265 bp
Lane 1: NEB 2-log; lanes 2-13: pIK1 miniPreps digested with BamHI+HindIII; lanes 2-5: 1 µl / 10 µl plate; lanes 6-10: 1 µl / rest plate; lanes 11-13: 14 µl / 10 µl plate
Lane 1: NEB 2-log; lanes 2-8: pIK1 miniPreps digested with BamHI+HindIII; lanes 2-3: 14 µl / 10 µl plate; lanes 4-8: 14 µl / rest plate
Retrieved from " http://2013.igem.org/Team:Heidelberg/Templates/MM_week18p "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
