<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Modelling_Gold_Recovery"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Modelling/Gold_Recovery">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Modelling/Gold_Recovery&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Modelling/Gold_Recovery&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Modelling/Gold_Recovery&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Modelling/Gold_Recovery" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Modelling/Gold Recovery</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="row"><DIV id="nav" class="navigation"><DIV class="item team" data-offset-l="-130"><H2>Team</H2><UL><LI><A href="/Team:Heidelberg/Team">Team Members</A></LI><LI><A href="/Team:Heidelberg/AttributionsII">Attributions</A></LI><LI><A href="/Team:Heidelberg/Team/Sponsors">Sponsors</A></LI><LI><A href="/Team:Heidelberg/Team/Collaboration">Collaborations</A></LI><LI><A href="/Team:Heidelberg/Team/Gallery">Gallery</A></LI></UL></DIV><DIV class="item project" data-offset-l="-144"><H2>Project</H2><UL><LI><A href="/Team:Heidelberg/NRPS">Background</A></LI><LI><A href="/Team:Heidelberg/Project">Overview</A></LI><LI><A href="/Team:Heidelberg/Project/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Project/Indigoidine-Tag">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Project/Tag-Optimization">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Project/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Project/Achievements">Achievements</A></LI></UL></DIV><DIV class="item notebook" data-offset-l="-139"><H2>Notebook</H2><UL><LI><A href="/Team:Heidelberg/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Tyrocidine">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Indigoidine">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/NRPSDesigner">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/Notebook/Methods">Methods</A></LI><LI><A href="/Team:Heidelberg/Materials">Materials</A></LI></UL></DIV><DIV class="item parts" data-offset-l="-134"><H2>Parts</H2><UL><LI><A href="/Team:Heidelberg/Parts">Submitted Parts</A></LI><LI><A href="/Team:Heidelberg/Favorite_Parts">Favorite Parts</A></LI><LI><A href="/Team:Heidelberg/RFCs">Our RFCs</A></LI></UL></DIV><DIV class="item software" data-offset-l="-134"><H2>Software</H2><UL><LI><A href="/Team:Heidelberg/Project_Software">Project</A></LI><LI><A href="http://igem2013.bioquant.uni-heidelberg.de/NRPSDesigner/">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/iGEM42">iGEM42</A></LI><LI><A href="/Team:Heidelberg/OpenSource">Open Source</A></LI><LI><A href="/Team:Heidelberg/Project/AchievementsSoftware">Achievements</A></LI></UL></DIV><DIV class="item human" data-offset-l="-118"><H2>Human Practice</H2><UL><LI><A href="/Team:Heidelberg/HumanPractice">Overview</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BroadPublic">Broad Public</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Survey">Survey</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/YoungGeneration">Young Generation</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Artists">Arts and Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BeyondScience">Beyond Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Experts">Experts</A></LI></UL></DIV><DIV class="item safety" data-offset-l="-115"><H2>Safety</H2><UL><LI><A href="/Team:Heidelberg/Safety">Safety</A></LI></UL></DIV><DIV class="item model" data-offset-l="-144"><H2>Modeling</H2><UL><LI><A href="/Team:Heidelberg/Modelling/Gold_Recovery">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Modelling/Ind_Production">ind-Production</A></LI></UL></DIV><DIV class="item"><SPAN style="margin-left:6%">HOME</SPAN></DIV></DIV></DIV><DIV class="container"><H1><SPAN style="font-size:130%; color:#ffcc00">Modeling Gold Recycling with Delftibactin.</SPAN><SPAN style="color:#999999; font-size:70%"> Would It be Feasible?</SPAN></H1><DIV class="row"><DIV class="jumbotron col-md-4" style="margin-left:2%; height:585px"><H2>Highlights:</H2><UL><LI style="font-size:14px">Constraint based modeling allows for yield estimation of delftibactin production</LI><LI style="font-size:14px">Cost estimation of delftibactin production using a fed batch process</LI><LI style="font-size:14px">Cost equivalency to more traditional chemical methods</LI><LI style="font-size:14px">Gold price exceeds expenses for delftibactin approach by one order of magnitude</LI></UL></DIV></DIV><H2 id="introduction">Introduction</H2><P>Non-ribosomal peptide synthetases expressed in natural organisms help to develop evolutionary advantages over competitors. This ability has been recognized at the industrial level for example, by pharmaceutical companies like Cubist that produce antibiotics (e.g. Daptomycin <SPAN class="citation">[1]</SPAN>) based on non-ribosomal peptide synthesis. Of course, we were also fascinated by the idea to elevate our system to a larger scale and to test its industrial feasibility. Accompanying our experimental results confirming the ability of delftibactin  to precipitate gold we attempt to use theoretical considerations and metabolic modeling to show the realistic potential of our idea.</P><P>In particular, the feasibility of utilizing delftibactin for recycling at industrial scale was assessed in the following steps: A genome scale metabolic model of recombinant E.coli cells capable of producing Delftibactin was formulated and then simulated using constraint-based modeling (Flux Balance Analysis). Subsequently, the optimal production envelope was used for simulating the bacterial growth and delftibactin production in a bioreactor, which was used to estimate the cost for the produced delftibactin. Finally, we suggest a workflow for the isolation of gold from printed circuit boards using delftibactin and compare its potential financial impact to a state of the art method  for gold recovery, which also utilizes the same leaching agent (aqua regia).</P><H2 id="suggested-workflow">Suggested Workflow</H2><CENTER><A class="fancybox" style="margin: 10px; width:80%;" href="https://static.igem.org/mediawiki/2013/3/35/Heidelberg_Modeling_Delftibactin_1.png" title="&lt;b&gt;Figure 1: Comparison of the two processes for recovery of gold from
electronic trash. In both cases aqua regia is used in order to
dissolve the metals and then in the one case delftibactin can be
immediately applied, while in the other one several chemical steps
have to be followed.&lt;/b&gt;"><FIGCAPTION style="width:80%"><B>Figure 1: Comparison of the two processes for recovery of gold from
electronic trash. In both cases aqua regia is used in order to
dissolve the metals and then in the one case delftibactin can be
immediately applied, while in the other one several chemical steps
have to be followed.</B></FIGCAPTION></A></CENTER><H3 id="general-overview">General overview</H3><P>Our proposed method is compared to a state of the art publication, which builds upon several patents <SPAN class="citation">[2]</SPAN>. This method was chosen, because it starts with exactly the same steps as our proposed workflow, namely dissolution in aqua regia and neutralization with NaOH and also aims at gold recovery, thus making the comparison possible. On the other hand, in order to selectively extract gold from the aqua regia solutions many more chemical reactions, extraction using dibutyl carbitol from now on abbreviated as DBC, chromatography and reduction of the gold ions are necessary. This is contrasted to our method, in which delftibactin binds selectively to gold ions and then also reduces them with no need for previous dispose other metal ions potentially contaminating the final product.</P><P>The two compared processes are summarized in Fig.1</P><P>So as to benchmark the efficiency of gold recycling using recombinantly produced delfibactin to chemical methods, the costs of the latter had to be estimated..</P><P>Those are calculated based on stoichiometric equations which we derived from the chemical process description published by Byoung et al.</P><P>$$ HNO_3 \: + \: 3 \: HCl \: \to \: NOCl \: + \: 2 \: Cl_{nasc.} \: + 2 \: H_2O $$</P><P>$$ 2 \: Au \: + \: 2 \: NOCl \: + \: 3 \: Cl_2 \: + \: 2 \: HNO_3 \: \to \: 2 \: HAuCl_4 \: + \: 4 \: NO_2 $$</P><P>$$ Fe \: + \: 4 \: HNO_3 \: \to \: Fe(NO_3)_3 \: + \: 2 \: H_2O \: + \: NO $$</P><P>$$ Fe(NO_3)_3 \cdot 9H_2O \: + \: 3 \: NaOH \: \to \: FeOOH \downarrow \: + \: 3 \: NaNO_3 \: + \: 10 \: H_2O $$</P><P>$$ DBC^+/OH^- \: + \: H^+/AuCl_4^- \: \to \: DBC^+/AuCl_4^- \: + \: H_2O $$ $$ DBC^+/AuCl_4^- \: + \: NH_4OH \: \to \: DBC^+ \: + \: NH_4^+AuCl_4^- $$</P><P>$$ 2 \: AuCl_4^-NH_4^+ \: + \: 9 \: NH_4OH \: \to \: Au_2O_3 \cdot 3NH_3 \downarrow \: + \: 6 \: H_2O \: + \: 8 \: NH_4^+Cl^- $$</P><P>$$ 4 \: Au_2O_3 \cdot 3NH_3 \: + \: 6 \: N_2H_4 \: \to \: 8 \: Au \downarrow \: + \: 12 \: NH_4^+OH^- \: + \: 6 \: N_2 \uparrow $$</P><P>Furthermore we took the published experimental findings into account. In particular, we recalculated the amount of DBC<SUP>+</SUP> needed based on the reference stating that up to 190 g Au can be loaded into 1 L of DBC<SUP>+</SUP>.</P><P>The cost calculations, in which operational costs and ion-exchange chromatography cost have been neglected, show that the final cost per mol of Gold is 176.79. They are summarized in the following table:</P><TABLE class="table table-hover"><THEAD><TR><TH><P>Substance</P></TH><TH><P>Molecules/2 Au</P></TH><TH><P>Mol of Substance/Mol of Au</P></TH><TH><P>Amount of substance/Mol of Au</P></TH><TH><P>Price [€]/Mol of Au</P></TH><TH><P>CAS (Supplier)</P></TH></TR></THEAD><TBODY><TR><TD><P>HNO<SUB>3</SUB></P></TD><TD><P>9</P></TD><TD><P>4.5</P></TD><TD><P>0.188</P></TD><TD><P>3.46</P></TD><TD><P>7697-37-2 (Carl Roth)</P></TD></TR><TR><TD><P>HCl</P></TD><TD><P>92.86</P></TD><TD><P>13.5</P></TD><TD><P>1.41</P></TD><TD><P>30.879</P></TD><TD><P>7647-01-0 (Carl Roth)</P></TD></TR><TR><TD><P>NaOH</P></TD><TD><P>6</P></TD><TD><P>30</P></TD><TD><P>3</P></TD><TD><P>0.12</P></TD><TD><P>1310-73-2 (Carl Roth)</P></TD></TR><TR><TD><P>DBC<SUP>+</SUP></P></TD><TD><P>8</P></TD><TD><P>4.2</P></TD><TD><P>1.05</P></TD><TD><P>111.58</P></TD><TD><P>112-73-2 (Sigma Aldrich)</P></TD></TR><TR><TD><P>NH<SUB>4</SUB>OH</P></TD><TD><P>9</P></TD><TD><P>4.5</P></TD><TD><P>0.085</P></TD><TD><P>2.79</P></TD><TD><P>1336-21-6 (Carl Roth)</P></TD></TR><TR><TD><P>N<SUB>2</SUB>H<SUB>4</SUB></P></TD><TD><P>6</P></TD><TD><P>3</P></TD><TD><P>0.104</P></TD><TD><P>27.66</P></TD><TD><P>10217-52-4 (Sigma Aldrich)</P></TD></TR><TR><TD colspan="4"><P>Sum</P></TD><TD><P>176.79</P></TD></TR></TBODY></TABLE><H2 id="delftibactin-production">Delftibactin Production</H2><H3 id="explanation-of-constraints-based-modeling">Explanation of constraints based modeling</H3><P>Constraint based modeling <SPAN class="citation">[3]</SPAN> is a computational method to mechanistically simulate complex metabolic networks <SPAN class="citation">[4]</SPAN>, that operates based one one key assumption: The system is assumed to be in steady-state, which means that the concentration of all of the cell’s inner metabolites remains constant throughout the process. Based on these assumptions, constraint based modeling, allows to make quantitative predictions about the cellular behavior, by utilizing a minimal set of information. The essential prerequisite of any type of constraint based modeling is the existence of a reconstructed metabolic network, where all reactions of the network have been characterized stoichiometrically. Once this is done, one constructs the stoichiometric matrix S of the network, which includes the stoichiometric coefficients of all metabolites in each of the reactions. In particular, each column of the matrix corresponds to a reaction and each row to a metabolite (Fig. 2). This represents a mass balance of the network. Beyond the stoichiometric coefficients, another essential part of the model are the boundary constraints: These place upper and lower bounds on the flux (turnover rate) of some of these reactions, based on physicochemical considerations and experimental data. For example, lower or upper bounds are set to 0 if a given reaction is considered as irreversible.</P><CENTER><A class="fancybox" style="margin: 10px; width:80% " href="https://static.igem.org/mediawiki/2013/f/fa/Heidelberg_stoichmatrix.png" title="&lt;b&gt;Figure 2: Visualization of the concept of a stoichiometric matrix: In
(a) a biochemical network is shown, whose reactions can be represented
mathematically by the coefficients of the reacting substrates and
products (b). Finally, in the flux space representation, each reaction
is constrained by a lower and an upper bound. (from [3])&lt;/b&gt;"><FIGCAPTION style="width:80%"><B>Figure 2: Visualization of the concept of a stoichiometric matrix: In
(a) a biochemical network is shown, whose reactions can be represented
mathematically by the coefficients of the reacting substrates and
products (b). Finally, in the flux space representation, each reaction
is constrained by a lower and an upper bound. (from [3])</B></FIGCAPTION></A></CENTER><CENTER><A class="fancybox" style="margin: 10px; width: 80%;" href="https://static.igem.org/mediawiki/2013/3/3a/Heidelberg_FBA.png" title="&lt;b&gt;Figure 3: The constraints imposed by the steady state condition and the
bounds to the reaction fluxes restricts n-dimensional space to a
polytope, the so called flux space. Maximization of a linear objective
function using linear programming returns a vertex of this polytope.
(from [3])
&lt;/b&gt;"><FIGCAPTION style="width:80%"><B>Figure 3: The constraints imposed by the steady state condition and the
bounds to the reaction fluxes restricts n-dimensional space to a
polytope, the so called flux space. Maximization of a linear objective
function using linear programming returns a vertex of this polytope.
(from [3])
</B></FIGCAPTION></A></CENTER><P>Interestingly, the constraints define polytopes in a high-dimensional space, which is usually called the flux space. One can easily prove that polytopes are convex sets, which is a property that makes them amenable to several manipulations, such as random sampling based on artificially centered hit and run algorithms <SPAN class="citation">[5]</SPAN>. These polytopes are usually represented as follows (Fig.3):</P><P>$$ S\cdot v = 0 $$ $$ v_{min} \leq v \leq v_{max} $$</P><P>where $S$ is the stoichiometric matrix, $v_{min},v_{max}$ describe lower and upper bounds of the metabolic fluxes.</P><P>As this polytope is high dimensional, methods have to be applied in order to determine probable flux distributions or to compare flux spaces corresponding to different cells or states. The usual procedure consists of the maximization of a linear objective function $c^T\ v$ subjected to the constraints defining the polytope. This concept is based on the assumption that biological systems have evolved in order to maximize a certain objective for example the growth rate of the organism. To convert this principle into linear programming the following algorithm is formulated: Maximize $c^T\ v$ in respect to the above constraints (Fig. 3). This method is usually called Flux Balance Analysis (FBA) and the linear objective maximized is in most cases the so-called biomass reaction, though it can also take other forms, such as ethanol production or ATP maximization, depending on the context.</P><P>Although linear programs can be solved quickly with an optimal objective function value returned, there can exist many alternate optimal solutions, of which available solvers return only one. In order to capture this alternate solution space, flux variability analysis (FVA) can be employed <SPAN class="citation">[6]</SPAN>. In FVA, for each reaction $i$, the flux $v_i$ is maximized, then minimized, subject to the previous constraints and $c^Tv \geq s \cdot \max(c^Tv)$ with $s \in [0,1]$ and usually equal to 1.</P><P>Most of the constraint based modeling approaches have also been implemented in diverse software packages. Here, the very popular COBRA toolbox for Matlab <SPAN class="citation">[7]</SPAN> was used.</P><H3 id="metabolic-model-of-delftibactin-producing-e.coli">Metabolic model of Delftibactin producing <EM>E.coli</EM></H3><P>As explained in the previous section, to simulate the metabolism of a cell using flux balance analysis, the stoichiometric representation of the underlying metabolic network has to be available. As the biochemistry of <EM>E.coli</EM> has been extensively studied, there also exist comprehensive metabolic reconstructions. Here we used the iAF1260 model <SPAN class="citation">[8]</SPAN>, which captures 2077 reactions of K-12 MG1655 <EM>E.coli</EM> corresponding to 1260 genes. Of course, wild type <EM>E.coli</EM> is not able to produce Delftibactin. Thus the reactions corresponding to the gene products we are trying to introduce into our TOP10 cells were appended to the aforementioned model.</P><CENTER><A class="fancybox" style="margin: 10px; width: 80%;" href="https://static.igem.org/mediawiki/2013/f/f9/Heidelberg_ProdEnvelope.png" title="&lt;b&gt;Figure 4: The production envelope: The line in dark gray represents the
Pareto Frontier for the delftibactin production rate. It shows the
maximal possible flux as a function of the growth rate, as simulated
by FBA using our modified iAF1260 model. The light gray consists of
all other growth rate and delftibactin combinations that are possible
based on the above model (recall that the flux space is convex). The
10 circled points represent some of the 150 points that were used for
simulation of the fed-batch process.
&lt;/b&gt;"><FIGCAPTION style="width:80%"><B>Figure 4: The production envelope: The line in dark gray represents the
Pareto Frontier for the delftibactin production rate. It shows the
maximal possible flux as a function of the growth rate, as simulated
by FBA using our modified iAF1260 model. The light gray consists of
all other growth rate and delftibactin combinations that are possible
based on the above model (recall that the flux space is convex). The
10 circled points represent some of the 150 points that were used for
simulation of the fed-batch process.
</B></FIGCAPTION></A></CENTER><P>In particular, wild type E.coli can produce all the monomers necessary for Delftibactin production, with the exception of methylmalonyl-CoA, which is required for the PKS part of the synthetase. Thus, a reaction converting propanoyl-CoA to methylmalonyl-CoA with the following stoichiometry was added:</P><P>$$ 1 \: ppcoa[c] \: + \: 1 atp[c] \: + \: 1 \: hco3[c] \: \to \: 1 \: mmcoa-S[c] \: + \: 1 \: pi[c] \: + \: 1 adp[c] \: + \: 1 \: h[c] $$</P><P>Subsequently, the main delftibactin production reaction was added. The stoichiometry is governed by the amino acids monomers which are combined under usage of 1 ATP each and the Claisen condensation of methyl-malonyl CoA. This core reaction of the synthetase was accompanied with the subsequent modification enzymes <SPAN class="citation">[9]</SPAN> which are also necessary for the functional activity of Delftibactin. These comprise the aspartic acid dioxygenase (encoded by the gene DelD), the N5-hydroxyornithine formyltransferase (DelP) and finally the lysine/ornithine N-monooxygenase (DelL). The stoichiometry of this lumped reaction is the following:</P><P>1 ala-L[c] + 2 ser-L[c] + 1 asp-L[c] + 2 thr-L[c] + 1 gly[c] + 2 orn[c] + 1 arg-L[c] + 10 atp[c] + 1 akg[c] + 2 o2[c] + 1 10fthf[c] + 1 nadph[c] + 1 h[c] + 1 mmcoa-S[c] -&gt; 10 amp[c] + 20 pi[c] + 1 co2[c] + 1 succ[c] + 1 thf[c] + 1 nadp[c] + 1 h2o[c] + 1 coa[c] + 1 co2[c] + 1 delftibactin</P><P>Considering constraints of the metabolic model, it was assumed that <EM>E.coli</EM> grows on minimal glucose medium under aerobic conditions. The maximal glucose uptake rate was set to 10.5 $ \frac{mmol}{g_{dw}h} $ and the oxygen uptake rate to 15 $ \frac{mmol}{g_{dw}h} $ and the ATP maintenance flux (which represents the non-growth associated energy required for maintaining the biological processes of the cells) was set to 7.6 $ \frac{mmol}{g_{dw}h} $. Those fluxes have been previously measured for aerobically growing <EM>E.coli</EM><SPAN class="citation">[10]</SPAN>.</P><P>As we were interested in the optimal case, that is the maximal possible delftibactin production based on the stoichiometrically imposed constraints, we initially used FBA with delftibactin production as the objective function. The resulting flux was 1.2105 $ \frac{mmol}{g_{dw}h} $. However, the corresponding specific growth rate was 0. Thus, the maximally possible specific growth rate in dependence of the delftibactin synthesis rate was determined. The polytope was split into 150 intervals. For each of these intervals, the growth rate was fixed and FBA was simulated again. Fig. 4 shows the resulting production envelope, which in turn shows the optimal delftibactin production rate as a function of the specific growth rate. As could have been expected, the delftibactin synthesis drains many resources which are also necessary for bacterial growth (ATP, amino acids) and thus these two objectives represent a natural trade-off. The maximal growth rate, when the bacteria are not producing delftibactin at all is 0.8093 $ \frac{1}{h} $.</P><H3 id="bioreactor-simulations">Bioreactor Simulations</H3><P>Next, we estimated the cost of delftibactin production using a fed batch process. A flux balance analysis model only gives an estimate for steady-state fluxes and does not capture dynamic behaviors, such as fermentation processes. This is solved by dynamic FBA (dFBA) frameworks <SPAN class="citation">[11]</SPAN>, which essentially discretize time into small steps. Flux and growth rates are calculated by FBA and subsequently included into appropriate differential equations. The results are used to constrain the metabolic network in next time step, etc.</P><CENTER><A class="fancybox" style="margin: 10px; width: 80%;" href="https://static.igem.org/mediawiki/2013/4/4c/Heidelberg_Model_del_double.png" title="&lt;b&gt;Figure 5: Trajectories of biomass (a) and delftibactin concentration (b)
for 3 of the simulated points corresponding to 3 representative growth
rates.&lt;/b&gt;"><FIGCAPTION style="width:80%"><B>Figure 5: Trajectories of biomass (a) and delftibactin concentration (b)
for 3 of the simulated points corresponding to 3 representative growth
rates.</B></FIGCAPTION></A></CENTER><P>A fed-batch process with an exponential feeding strategy, in which the glucose concentration, remains constant and delftibactin is produced, was modelled by dFBA using the DyMMM (Dynamic Multispecies Metabolic Modeling framework ) MATLAB framework <SPAN class="citation">[12]</SPAN> with the following equations adapted from Zhuang et al. <SPAN class="citation">[13]</SPAN>.</P><P>$$ \frac{dV}{dt}=\left\{\begin{matrix}\frac{v_{glc}XV}{S_{glc}-S_{glc}^{feed}}, if \ V &lt; V_{max} \\ 0, if \ V \geq V_{max} \end{matrix}\right.$$ $$ \frac{dX}{dt}=\mu X $$ $$ \frac{dS_{glc}}{dt} = v_{glc}X + \frac{dV}{dt}\frac{S_{glc}^{feed}-S_{glc}}{V} $$ $$ \frac{dS_{delftibactin}}{dt} = v_{delftibactin}X $$ $$ |v_{glc}| \leq |v_{glc}^{max}| \frac{S_{glc}}{S_{glc} + K_m} $$</P><P>where X [g/L] is the <EM>E.coli</EM> biomass, V, Vmax [L] are the currently and maximally filled volume of the bioreactor, $S_{glc}$, $S_{glc}^{feed}$, $S_{delftibactin}$ [mmol/L] are the concentrations of glucose and delftibactin in the reactor or the feed, $K_m$ [mmol/L] the Michaelis-Menten constant for glucose uptake, $ \mu $ [1/h] the growth rate of the bacteria and finally $ v_{glc},v_{delftibactin} $ $ \frac{mmol}{g_{dw}h} $ the flux rates of the corresponding reactions with $ v_{glc}^{max} $ being the maximal possible flux.</P><P>The starting conditions were set to a volume of 1 L, a biomass of 0.01 g/L and a glucose concentration of 20 mmol/L, as in <SPAN class="citation">[13]</SPAN>. The other bioreactor parameters were set as follows: The concentration of glucose in the feed was equal to 1 mol/L, $K_m$ equal to 1 mmol/L <SPAN class="citation">[13]</SPAN> and finally, the maximal glucose uptake rate was set to 10.5 $ \frac{mmol}{g_{dw}h} $ <SPAN class="citation">[10]</SPAN>. The time step was set to 0.1 h.</P><P>Simulations with those conditions were performed for 150 equally spaced growth rate values on the production envelope, as shown previously in Fig.4. Fig. 5 shows the resulting trajectory for the biomass concentration and delftibactin concentration for 3 representative points from the envelope. As expected, higher growth rates result faster convergence towards the steady-state. Reaching the steady state implies the end of the fermentation process. This is expected, as in the model, the glucose feed is stopped as soon as the 10 L of the bioreactor have been filled. Also, the final concentration of delftibactin ranges from approximately 40 to 80 $ \frac{mmol}{L} $, with higher titers achieved for lower growth rates, while dry biomass (10 to 50 $ \frac{g}{L}) $ titer is higher in the case of high growth rates.</P><P>The above observations actually illustrate another aspect of the growth rate - production rate tradeoff, namely the economic aspect. Lower growth rates actually lead to higher final delftibactin concentration in the bioreactor, which is of course wanted, but on the other hand the process takes a lot longer to finish, thus requiring more operational costs. Exactly this trade-off will be explored in the next session.</P><H3 id="delftibactin-production-cost">Delftibactin Production Cost</H3><P>The ultimate goal of the aforementioned simulations was the optimistic calculation of the cost required for production of delftibactin. Thus, for each of the simulated points, we calculated the costs by starting with the following assumptions:</P><UL><LI style="font-size:14px">The cost of the growth medium of the <EM>E.coli</EM> is equal to the cost of glucose spent.</LI><LI style="font-size:14px">The operation of the 10 L bioreactor requires 1 full time technician with a wage of 2000 euros per month.</LI><LI style="font-size:14px">After finishing the operation of the bioreactor, the next cycle can start after a time offset of 3 hours.</LI><LI style="font-size:14px">The electricity cost of the bioreactor is calculated based on its power rate and assuming constant operation.</LI><LI style="font-size:14px">Delftibactin has been secreted to the supernatant and it does not have to be purified in order to selectively precipitate gold, as shown in our experiments. Instead hydrogen peroxide has to be used to remove reducing agents except delftibactin from the medium.</LI></UL><CENTER><A class="fancybox" style="margin: 10px; height:400px;" href="https://static.igem.org/mediawiki/2013/1/18/Heidelberg_Parabola.png" title="&lt;b&gt;Figure 6: Delftibactin cost as a function of the growth rate. The
trade-off between high growth rate and delftibactin production rate,
which is also seen in Fig. 4, results in this parabola-shaped curve.
The global minimum is attained at 0.37 1/h and has a value of 0.1755
Euro per mmol of Delftibactin.&lt;/b&gt;"><FIGCAPTION style="width:80%"><B>Figure 6: Delftibactin cost as a function of the growth rate. The
trade-off between high growth rate and delftibactin production rate,
which is also seen in Fig. 4, results in this parabola-shaped curve.
The global minimum is attained at 0.37 1/h and has a value of 0.1755
Euro per mmol of Delftibactin.</B></FIGCAPTION></A></CENTER><P>For these calculations, the following prices were looked up and used:</P><TABLE class="table table-hover"><TBODY><TR><TD><P>Glucose</P></TD><TD><P>0.13 Dollar per mol</P></TD><TD><P><A href="http://ers.usda.gov/data-products/sugar-and-sweet
eners-yearbook-tables.aspx"> US Dept. of Agriculture’s Economic Research </A> and [13]</P></TD></TR><TR><TD><P>Technician</P></TD><TD><P>2000 € per month</P></TD><TD><P>rough estimation based on German wages</P></TD></TR><TR><TD><P>Reactor Power</P></TD><TD><P>2.5 kW</P></TD><TD><P>Value for 30 L bioreactor  <SPAN class="citation">[14]</SPAN></P></TD></TR><TR><TD><P>Hydrogen peroxide</P></TD><TD><P>6.54 € per Liter</P></TD><TD><P>Carl Roth</P></TD></TR><TR><TD><P>Hydrogen peroxide per Medium</P></TD><TD><P>1/500</P></TD><TD><P>estimated in our experiments</P></TD></TR><TR><TD><P>Electricity Cost</P></TD><TD><P>0.1879 € per kWh</P></TD><TD><P><A href="http://appsso.eurostat.ec.europa.eu/nui/show.do?dataset=nrg_pc_205&amp;lang=en">Eurostat Germany 2013, incl. taxes</A></P></TD></TR><TR><TD><P>USD-EUR Exchange Rate </P></TD><TD><P>0.7253 € per Dollar</P></TD><TD><P><A href="http://www.bloomberg.com/quote/EURUSD:CUR"> as of 09:06:59, 28/10/2013 Bloomberg</A></P></TD></TR></TBODY></TABLE><P>In particular, let $n_{delftibactin}$ [mol] denote the number of delftibactin mol produced, $n_{glc}$ [mol] the glucose consumed and $t_f$ [h] the time at which the fermentation ends. All of these values can be can be easily calculated based on the results of the previous simulations.</P><P>$$ n_{delftibactin} = S_{delftibactin}V_f $$ $$ n_{glc} = S_{glc}^{start}V_0 + S_{glc}^{feed}(V_f-V_0) $$</P><P>and $t_f$ is just the time after which $\frac{dS_{delftibactin}}{dt} = 0 $.</P><P>Now also let</P><P>$t_{offset}$ [h] be the time until the next fermentation cycle can start, $P_{glc}$ [euros/mol glucose] be the price of glucose and $ p_{reactor} $, the cost of operating the reactor per time (equal to the wage for 1 technician per time and the electricity cost). Then the final cost $P_{delftibactin}$ per mol of delftibactin is equal to:</P><P>$$ P_{delftibactin} = \frac{n_{glc}P_{glc} + (t_f+t_{offset})p_{reactor}}{n_{delfti}} $$</P><P>Based on the above equation equation, for each simulated point of the production envelope, the cost per mmol of delftibactin was calculated (Fig.6). In the figure the trade-off between growth rate and delftibactin production immediately becomes obvious. In fact, the relationship of price to growth rate appears to be parabolic and actually has a global minimum. This global minimum of 0.1755 Euro per mmol of Delftibactin achieved to a growth rate $ \mu = 0.37 h^{-1}$ (total process time of 29.7 hours, final delftibactin titer of 61 mmol/L) is the cost we will assume in the next section.</P><H2 id="cost-estimation-for-whole-procedure">Cost estimation for whole procedure</H2><P>Having estimated the cost for produced delftibactin, it is now possible to also calculate the cost for the whole gold recovery process. Similarly, to the cost calculation for the gold procedure and using the aforementioned costs for aqua regia and NaOH, we calculated that for recovery of 1 mol Gold approximately 5 Euros are required in addition to the necessary delftibactin. Under the assumptions that 1 molecule of delftibactin binds and reduces exactly one gold ion, one can then quickly calculate the cost for the whole process (175.5 Euros for 1 mol Deftibactin, hence 180.5 Euros total per mol Gold). The current gold price, as well as the price for recovery from electronic trash with the different methods has been summarized in the table below.</P><TABLE class="table table-hover"><TBODY><TR><TD><P>Gold price</P></TD><TD><P>6828.9  € per mol</P></TD></TR><TR><TD><P>Chemical gold recovery cost</P></TD><TD><P>176.8 € per mol</P></TD></TR><TR><TD><P>Delftibactin gold recovery cost</P></TD><TD><P>180.5 € per mol</P></TD></TR></TBODY></TABLE><P>In summary, our engineered bacteria, assuming optimality of delftibactin production and a laboratory scale bioreactor can recover gold from trash at similar costs with state of the arts methods and significantly cheaper compared to gold price.</P><H2 id="discussion">Discussion</H2><H3 id="proposed-workflow">Proposed workflow</H3><P>In order to recover metals from electronic waste, several different methods have been developed and are industrially utilized. In some of the common processes, the trash is incinerated, thus releasing toxic dioxins <SPAN class="citation">[15]</SPAN>. Pyrometallurgical processing, based on smelting, is a very common process, but also one that is often considered to be non-selective, requiring huge energy inputs and releasing harmful products in an uncontrolled manner <SPAN class="citation">[16]</SPAN>.</P><P>Another commonly used method is hydrometallurgical processing, in which diverse chemicals are used for the leaching of the trash in order to dissolve the metals in aqueous solutions. Traditionally, cyanide has been used , but due to environmental accidents, it is increasingly avoided <SPAN class="citation">[17]</SPAN>. Still, the hydrometallurgical method is often considered to be highly selective and have to have a good recovery of gold.</P><P>Aqua regia used in our proposed pipeline also falls into the category of hydrometallurgy associated chemicals. Note that aqua regia is not necessarily the best choice of leaching agent to be used at industrial scale, because of the high cost of the necessary equipment (special stainless steel) <SPAN class="citation">[18]</SPAN>. There are also toxic by-products, such as chlorine gas, especially if plastic is not separated from the metals in an initial step. Thus, the environmental friendliness could be further improved for example by using bacteria to recycle the fantastic plastic, as has been done in different <A href="https://2013.igem.org/Team:Imperial_College">iGEM projects </A>.</P><P>It is important to note again that the method proposed here only requires the existence of Gold ions in a fairly neutral solution. Thus, new developments in ecologically friendly leaching agents or even bioleaching agents <SPAN class="citation">[17]</SPAN> could be immediately coupled to the delftibactin process. In fact, it could easily replace the gold extraction step in most industrially used processes in a plug-and-play fashion, whereas the chemical isolation steps would have to be extensively readjusted to the new conditions.</P><P>Finally, it has to be mentioned that efficient industrial processes consist of closed-loop systems <SPAN class="citation">[19]</SPAN> and pipelines, which try to isolate as many metals as possible, e.g. by iterative steps of electrolysis at different voltages to precipitate and reduce the various metals. Such processes of course appear optimal from an economic perspective, but are difficult to compare to our suggestion in a quantitative manner. Of course, further developments could allow the creation of similar pipelines based almost exclusively on bioengineering and chelating NRP molecules in particular.</P><H3 id="constraint-based-modeling">Constraint-based modeling</H3><P>One of the main assumptions of FBA based modeling is that <EM>E.coli</EM> bacteria have been engineered to maximize the delftibactin production rate. In fact, the simulated points for the maximal delftibactin production lie on the pareto frontier (product envelope) and they place an upper bound to what is theoretically possible, simply due to stoichiometric considerations.</P><P>In order to get an insight into the resulting quantities, a comparison with another recent approach that modelled NRPS/PKS systems in a similar way appears useful. Huang et al. <SPAN class="citation">[20]</SPAN> used metabolic flux analysis in order to simulate the production of the antibiotic Daptomycin by <EM>Streptomyces roseosporus</EM>. Here the maximum daptomycin production was approximately equal to 0.06 $\frac{mmol}{g_{dw}h}$ for a glucose uptake rate of 0.7 $\frac{mmol}{g_{dw}h}$. Thus, the yield of daptomycin per glucose is appropriately equal to 0.086. This is in a comperable range to the result of our simulations, in which the yield was $ \frac{v_{delftibactin}}{v_{glucose}} = \frac{0.68}{10.5} = 0.065 $. As daptomycin and delftibactin have similar molecular mass, this shows that the metabolic capabilities for the production of NRPS/PKS products of these two organisms, according to current flux balance models, are fairly similar.</P><P>In the above publication, the experimentally measured titer of daptomycin in fed-batch culture was 581.5 mg/L. Our modelled fed-batch process titer of 63 g/L (61 mmol/L and delftibactin has a molecular mass of 1033 dalton). This factor of 2 shows two things: For one, natural systems will have to be significantly engineered in order to achieve NRP production rates close to the Pareto frontier. Yet, this was one of the main reasons why we started the experiments for the introduction of the <EM>D.acidovorans</EM> genes into <EM>E.coli</EM>: The latter can be manipulated and used for metabolic engineering in a standardized framework. On the other hand, this factor also shows that our calculation might be very optimistic for the time being and that significant further research will have to be done until such as a system can applied at industrial scale.</P><H3 id="bioreactor-simulation">Bioreactor simulation</H3><P>In general, there exist three main methods for the cultivation of bacteria in bioreactors: The most classical one is the batch method, in which all of the initial medium is inoculated at the beginning point. In contrast, for the continuous fermentation, new medium keeps flowing into the reactor while also being withdrawn at the same rate. Continuous fermentation of course requires a lot less labour costs, provides higher yields and is more controllable than the batch process. On the other hand, in real processes, the microorganisms often mutate to non producing variants, thus making continuous processes problematic <SPAN class="citation">[21]</SPAN>. In between these methods lies the fed-batch, which was chosen here, in which medium keeps being added to the reactor, but is not withdrawn.</P><P>This method, was simulated using dynamic FBA (dFBA), which quickly proved to be a very valuable method for combining dynamic processes with stoichiometric metabolic models. However, one disadvantage is the rather long simulation time, but recent methods have been developed in order to efficiently simulate differential equations numerically of which the right hand side actually requires the optimization of a linear program <SPAN class="citation">[22]</SPAN>. Applying such methods allows a quicker coverage of the parameter space.</P><P>In particular, one of the problems that appear in aerobic <EM>E.coli</EM> fermentations is the fact that at high growth rates (almost surely for growth rates above 0.35 1/h) a lot of acetate is produced which actually suppresses growth <SPAN class="citation">[23]</SPAN>. This is something that was not considered in the simulations here, but could be easily added by an appropriate inhibition term to the differential equations and also by tracking the acetate concentration in the dFBA model.</P><H2 id="final-judgment">Final judgment</H2><P>In total, under optimal conditions, our method achieves similar cost to more traditional chemical methods. Of course, optimal production is only theoretically possible and compared to actual measurements in <EM>Streptomyces</EM> (see above), the final titer is 2 orders of magnitude higher. Still, it has to be also considered that here the fermentation process was assumed to take place in a small scale 10 L bioreactor with high operation costs. Production at industrial scale with 10 $ m^3  $ bioreactors would drastically reduce those costs.</P><P>Finally, it also has to be considered that there is still 1 order of magnitude difference in respect to the profitability (cost of Gold compared to price for gold recovery with our method). With the gold price steadily rising and with improved NRPS and metabolic engineering, our approach could eventually turn out to be lucrative.</P><H2 id="outlook">Outlook</H2><P>
In our project, we have demonstrated the feasibility of <A class="fancybox fancyFigure" title="&lt;i&gt;D. acidovorans&lt;/i&gt; can be used for gold recovery from electronic waste. a) Pins removed from an old CPU. b) Green solution of dissolved pins in aqua regia. c) Gold chloride solution obtained after the dilution of an old CPU. d) " dissolved="" electronic="" waste="">small scale recovery of gold from electronic waste</A> by <I>D. acidovorans</I>. The simulations of the cost for gold recovery in industrial scale have been described and discussed above. Ideally, recovery of valuable metals from electronic waste should not be limited to gold alone but could be extended to for instance palladium and platine. As we also provide a tool for the prediction and design of novel peptides, the <A href="http://igem2013.bioquant.uni-heidelberg.de/NRPSDesigner/">NRPSDesigner</A>, it is conceivable to create NRPs which exhibit specifity for different precious metals. Furthermore, other innovative recycling techniques could be combined with our gold recyling approach, such as the plastic recycling proposed by <A href="https://2013.igem.org/Team:Imperial_College">Imperial College's iGEM team 2013</A>. Thus, a complete pipeline for the recycling of a mixture of different wastes based on synthetic biology approaches can be envisaged for the future.
</P><A class="btn btn-default btn-lg" href="https://static.igem.org/mediawiki/2013/e/e4/DelftibactinModeling.zip" style="margin-bottom:20px">Access our script here!</A><DIV class="jumbotron references"><P>1. Eisenstein BI, Oleson FB, Baltz RH (2010) Daptomycin: from the mountain to the clinic, with essential help from Francis Tally, MD. Clinical Infectious Diseases 50: S10–S15.</P><P>2. Jung BH, Park YY, An JW, Kim SJ, Tran T, et al. (2009) Processing of high purity gold from scraps using diethylene glycol di-&lt; i&gt; N&lt;/i&gt;-butyl ether (dibutyl carbitol). Hydrometallurgy 95: 262–266.</P><P>3. Becker SA, Feist AM, Mo ML, Hannum G, Palsson BØ, et al. (2007) Quantitative prediction of cellular metabolism with constraint-based models: the COBRA Toolbox. Nature protocols 2: 727–738.</P><P>4. Orth JD, Thiele I, Palsson BØ (2010) What is flux balance analysis? Nature biotechnology 28: 245–248.</P><P>5. Schellenberger J, Palsson BØ (2009) Use of randomized sampling for analysis of metabolic networks. The Journal of biological chemistry 284: 5457–5461.</P><P>6. Mahadevan R, Schilling CH (2003) The effects of alternate optimal solutions in constraint-based genome-scale metabolic models. Metabolic Engineering 5: 264–276.</P><P>7. Schellenberger J, Que R, Fleming RMT, Thiele I, Orth JD, et al. (2011) Quantitative prediction of cellular metabolism with constraint-based models: the COBRA Toolbox v2.0. Nature protocols 6: 1290–1307.</P><P>8. Feist AM, Henry CS, Reed JL, Krummenacker M, Joyce AR, et al. (2007) A genome-scale metabolic reconstruction for Escherichia coli K-12 MG1655 that accounts for 1260 ORFs and thermodynamic information. Molecular systems biology 3.</P><P>9. Johnston CW, Wyatt MA, Li X, Ibrahim A, Shuster J, et al. (2013) Gold biomineralization by a metallophore from a gold-associated microbe. Nature chemical biology 9: 241–243.</P><P>10. Varma A, Palsson BO (1994) Stoichiometric flux balance models quantitatively predict growth and metabolic by-product secretion in wild-type Escherichia coli W3110. Applied and environmental microbiology 60: 3724–3731.</P><P>11. Mahadevan R, Edwards JS, Doyle III FJ (2002) Dynamic Flux Balance Analysis of Diauxic Growth in&lt; i&gt; Escherichia coli&lt;/i&gt;. Biophysical journal 83: 1331–1340.</P><P>12. Zhuang K, Ma E, Lovley DR, Mahadevan R (2012) The design of long-term effective uranium bioremediation strategy using a community metabolic model. Biotechnology and Bioengineering 109: 2475–2483.</P><P>13. Zhuang K, Yang L, Cluett WR, Mahadevan R (2013) Dynamic strain scanning optimization: an efficient strain design strategy for balanced yield, titer, and productivity. DySScO strategy for strain design. BMC biotechnology 13: 8.</P><P>14. Chmiel H (2011) Bioprozesstechnik. Spektrum Akademischer Verlag.</P><P>15. Cui J, Zhang L (2008) Metallurgical recovery of metals from electronic waste: a review. Journal of hazardous materials 158: 228–256.</P><P>16. Chehade Y, Siddique A, Alayan H, Sadasivam N, Nusri S, et al. (2012) Recovery of Gold, Silver, Palladium, and Copper from Waste Printed Circuit Boards. International Conference on Chemical, Civil and Environment engineering.</P><P>17. Syed S (2012) Recovery of gold from secondary sources—A review. Hydrometallurgy 115-116: 30–51.</P><P>18. Williams E (2009) Review and prospects of recycling methods for waste printed circuit boards. 2009 IEEE International Symposium on Sustainable Systems and Technology: 1–5.</P><P>19. Owens B (2013) Mining: Extreme prospects. Nature 495: S4–S6.</P><P>20. Huang D, Wen J, Wang G, Yu G, Jia X, et al. (2012) In silico aided metabolic engineering of Streptomyces roseosporus for daptomycin yield improvement. Applied microbiology and biotechnology 94: 637–649.</P><P>21. Villadsen J, Nielsen J, Lidén G (2011) Bioreaction Engineering Principles. Springer.</P><P>22. Höffner K, Harwood SM, Barton PI (2013) A reliable simulator for dynamic flux balance analysis. Biotechnology and bioengineering 110: 792–802.</P><P>23. Lee J, Lee SY, Park S, Middelberg AP (1999) Control of fed-batch fermentations. Biotechnology advances 17: 29–48.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><DIV class="imageMap" aria-haspopup="true"><CENTER>Thanks to </CENTER></DIV></DIV></DIV><DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Heidelberg/Modelling/Gold_Recovery">http://2013.igem.org/Team:Heidelberg/Modelling/Gold_Recovery</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Modelling/Gold_Recovery" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Modelling/Gold_Recovery" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Modelling/Gold_Recovery&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Modelling/Gold_Recovery&amp;oldid=362791" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>