<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Templates_Indigoidine_week15"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Templates/Indigoidine_week15">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Templates/Indigoidine_week15&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Templates/Indigoidine_week15&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Templates/Indigoidine_week15&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Templates/Indigoidine_week15" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Templates/Indigoidine week15</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#CPEC_Conditions_Assay_.28Ralf.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">CPEC Conditions Assay (Ralf)</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#PPTase_test_.28Ralf.29"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">PPTase test (Ralf)</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#colony_PCR_screening_.231"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">colony PCR screening #1</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#MiniPrep_of_pRB4-10"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">MiniPrep of pRB4-10</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#MiniPrep_PCR_screening_.232"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">MiniPrep PCR screening #2</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Sequencing_of_pRB3-10"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Sequencing of pRB3-10</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Glycerols_stocks_pRB3-10"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Glycerols stocks pRB3-10</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="CPEC_Conditions_Assay_.28Ralf.29">CPEC Conditions Assay (Ralf)</SPAN></H3><P>We performed CPEC assembly of pRB13 with eight different CPEC conditions to see whether we get difeerent results.
</P><P><B>Table 11.1 CPEC mix for assembly of pRB13</B></P><TABLE class="wikitable"><TBODY><TR><TH>Annealing Temp [°C]</TH><TH colspan="3">Cycles
</TH></TR><TR><TH>5</TH><TH>10</TH><TH>15
</TH></TR><TR><TD>53</TD><TD><B>2: </B>10 ul; 4 ul backbone; 1 ul CCDB</TD></TR><TR><TD>55</TD><TD><B>3: </B>10 ul; 4 ul backbone; 1 ul CCDB <B>7: </B>20 ul; 8.5 ul backbone; 1.5 ul CCDB</TD><TD><B>4: </B>10 ul; 4 ul backbone; 1 ul CCDB<B>6: </B>20 ul; 8.5 ul backbone; 1.5 ul CCDB<B>8: </B>10 ul; 4.5 ul backbone; 0.5 ul CCDB </TD><TD><B>5: </B>10 ul; 4 ul backbone; 1 ul CCDB
</TD></TR><TR><TD>57</TD><TD><B>1: </B>10 ul; 4 ul backbone; 1 ul CCDB</TD></TR></TBODY></TABLE><P><B>Table 11.2 CPEC Assembly of pRB13</B></P><TABLE class="wikitable"><TBODY><TR><TH colspan="3">BioRad T100
</TH></TR><TR><TH>cycles</TH><TH>temp [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">5/10/15</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>53/55/57</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>120
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>100 ul of competent TOP10 were transformed with 5 ul of reaction product and incubated for 30 hours at 37 °C. 
</P><P>
Apparently there is no big difference using various CPEC conditions.
We will continue with the standard protocol.
</P><H3><SPAN class="mw-headline" id="PPTase_test_.28Ralf.29">PPTase test (Ralf)</SPAN></H3><P>pRB3-10 are plasmids containing indC (pRB-3-6) or bpsA(pMM64) (pRB7-10) and a PPTase (sfp, svp, svp(pMM65), entD). 
</P><P>This week the blue colonies are screened and sequenced for validation.
</P><H4><SPAN class="mw-headline" id="colony_PCR_screening_.231">colony PCR screening #1</SPAN></H4><P>Two blue colonies are picked from each plate and screened with the primers of the PPTase they should have and the 
</P><P>ones which they could have if something went wrong. PPTase PCR products werde used as a positive control and TOP10 
</P><P>without any plasmid as a negative control. So numbers 5 to 18 are subdivided into alpha and beta.
</P><TABLE class="wikitable"><TBODY><TR><TH colspan="4">PPTase to be screened
</TH></TR><TR><TH rowspan="14">template</TH><TH>sfp (RB35/36)</TH><TH>svp (RB29/30)</TH><TH>svp(pMM65) (RB25/26)</TH><TH>entD (RB33/34)
</TH></TR><TR><TH> 1</TH></TR><TR><TH> 2</TH></TR><TR><TH> 3</TH></TR><TR><TH> 4
</TH></TR><TR><TH> 5</TH></TR><TR><TH> 6</TH></TR><TR><TH> 7</TH></TR><TR><TH>13 </TH><TH>14 </TH><TH> 15</TH><TH> 8
</TH></TR><TR><TH> 9</TH></TR><TR><TH>10 </TH></TR><TR><TH>11 </TH></TR><TR><TH> 16</TH><TH>17 </TH><TH>18 </TH><TH> 12
</TH></TR><TR><TH> 19</TH><TH>20 </TH><TH>21 </TH><TH> 22
</TH></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH colspan="3">BioRad T100
</TH></TR><TR><TH>cycles</TH><TH>temp [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>120
</TD></TR><TR><TD rowspan="3">30</TD><TD>95</TD><TD>60
</TD></TR><TR><TD>65</TD><TD>30
</TD></TR><TR><TD>72</TD><TD>60
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH colspan="13">Loading Scheme
</TH></TR><TR><TD>5a</TD><TD>5b</TD><TD>9a</TD><TD>9b</TD><TD>1</TD><TD>19</TD><TD><B>2-log</B></TD><TD>6a</TD><TD>6b</TD><TD>10a</TD><TD>10b</TD><TD>2</TD><TD>20
</TD></TR><TR><TD>7a</TD><TD>7b</TD><TD>11a</TD><TD>11b</TD><TD>3</TD><TD>21</TD><TD><B>2-log</B></TD><TD>8a</TD><TD>8b</TD><TD>12a</TD><TD>12b</TD><TD>4</TD><TD>22
</TD></TR></TBODY></TABLE><P>The gel is not conclusive. Therefore I will repeat the screening but screen every colony for all PPTases.
</P><H4><SPAN class="mw-headline" id="MiniPrep_of_pRB4-10">MiniPrep of pRB4-10</SPAN></H4><P>5 ml liquid cultures of TOP10+pRB4-10 have been miniprepped using QIAquick miniprep kit. Only blue cultures have 
</P><P>been prepped (except pRB9 because there was no blue culture). pRB4-6 and pRB10 were prepped in duplicates (liquid 
</P><P>cultures of two different clones) whereas there are only single preps of pRB7-9. This is because pRB3-6 contain 
</P><P>indc ahich we will use for further studies. pRB6 and 10 contain entD, which is the E. coli endogenous PPTase and 
</P><P>was previously declared to be &quot;inefficient&quot; in activating bpsA [Takahashi 2007].
</P><P>Concentrations of the prepped plasmids have been measured using ??ThermoScientific NanoDrop
</P><TABLE class="wikitable"><TBODY><TR><TH>plasmid</TH><TH>version</TH><TH>concentration [ng/ ul]
</TH></TR><TR><TD rowspan="2">pRB4</TD><TD>alpha</TD><TD>289.7
</TD></TR><TR><TD>beta</TD><TD>278.7
</TD></TR><TR><TD rowspan="2">pRB5</TD><TD>alpha</TD><TD>228.3
</TD></TR><TR><TD>beta</TD><TD>348.9
</TD></TR><TR><TD rowspan="2">pRB6</TD><TD>alpha</TD><TD>428.0
</TD></TR><TR><TD>beta</TD><TD>432.2
</TD></TR><TR><TD>pRB7</TD><TD>alpha</TD><TD>187.3
</TD></TR><TR><TD>pRB8</TD><TD>alpha</TD><TD>768.2
</TD></TR><TR><TD>pRB9</TD><TD>alpha</TD><TD>142.5
</TD></TR><TR><TD rowspan="2">pRB10</TD><TD>alpha</TD><TD>112.0
</TD></TR><TR><TD>beta</TD><TD>212.0
</TD></TR><TR><TD colspan="2">water after measurement</TD><TD>0.7
</TD></TR></TBODY></TABLE><P>Minipreps were reduced to a concentration of about 50 ng/ ul for sequencing and 1 ng/ ul for further studies.
</P><H4><SPAN class="mw-headline" id="MiniPrep_PCR_screening_.232">MiniPrep PCR screening #2</SPAN></H4><TABLE class="wikitable"><TBODY><TR><TH colspan="4">PPTase to be screened
</TH></TR><TR><TH>template</TH><TH>sfp (RB35/36)</TH><TH>svp (RB29/30)</TH><TH>svp(pMM65) (RB25/26)</TH><TH>entD (RB33/34)
</TH></TR><TR><TD> TOP10+pRB4</TD><TD>5 </TD><TD> 6</TD><TD>7 </TD><TD> 8
</TD></TR><TR><TD> TOP10+pRB5</TD><TD> 9</TD><TD> 10</TD><TD>11</TD><TD> 12
</TD></TR><TR><TD> TOP10+pRB6</TD><TD>13 </TD><TD>14 </TD><TD> 15</TD><TD> 16
</TD></TR><TR><TD> TOP10+pRB7</TD><TD> 17</TD><TD> 18</TD><TD> 19</TD><TD> 20
</TD></TR><TR><TD> TOP10+pRB8</TD><TD>21 </TD><TD>22 </TD><TD> 23</TD><TD> 24
</TD></TR><TR><TD> TOP10+pRB9</TD><TD> 25</TD><TD> 26</TD><TD>27 </TD><TD> 28
</TD></TR><TR><TD> TOP10+pRB10</TD><TD> 29</TD><TD>30 </TD><TD>31 </TD><TD> 32
</TD></TR><TR><TD> sfp</TD><TD> 33</TD></TR><TR><TD> svp</TD><TD> 34</TD></TR><TR><TD>svp(pMM65)</TD><TD> 35</TD></TR><TR><TD>entD</TD><TD> 36
</TD></TR><TR><TD> TOP10</TD><TD>37</TD><TD>38 </TD><TD>39 </TD><TD> 40
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH colspan="3">BioRad T100
</TH></TR><TR><TH>cycles</TH><TH>temp [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>95</TD><TD>120
</TD></TR><TR><TD rowspan="3">30</TD><TD>95</TD><TD>60
</TD></TR><TR><TD>60</TD><TD>30
</TD></TR><TR><TD>72</TD><TD>60
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH colspan="25">Loading Scheme
</TH></TR><TR><TD>5a</TD><TD>5b</TD><TD>9a </TD><TD> 9b</TD><TD>13a</TD><TD>13b</TD><TD>17</TD><TD>21</TD><TD>25</TD><TD>29a</TD><TD>29b</TD><TD>sfp</TD><TD><B>2-log</B></TD><TD>6a</TD><TD>6b</TD><TD>10a</TD><TD>10b</TD><TD>14a</TD><TD>14b</TD><TD>18</TD><TD>22</TD><TD>26</TD><TD>30a</TD><TD>30b</TD><TD>svp
</TD></TR><TR><TD>7a</TD><TD>7b</TD><TD>11a</TD><TD>11b</TD><TD>15a</TD><TD>15b</TD><TD>19</TD><TD>23</TD><TD>27</TD><TD>31a</TD><TD>31b</TD><TD>svp(pMM65)</TD><TD><B>2-log</B></TD><TD>8a</TD><TD>8b</TD><TD>12a</TD><TD>12b</TD><TD>16a</TD><TD>16b</TD><TD>20</TD><TD>24</TD><TD>28</TD><TD>32a</TD><TD>32b</TD><TD>entD
</TD></TR></TBODY></TABLE><P>pRB9 is negative. All screens are positive in the svp(pMM65) and entD section. In entD this could be due to genomic 
</P><P>DNA in the miniprep. svp(pMM65) is also positive in all plasmids, which is strange. Maybe the PCR conditions have 
</P><P>been too tolerant. Nevertheless the bands of the plasmids which should carry the respective PPTase appear to be 
</P><P>stronger. We will sequence the plasmids anyways.
</P><H4><SPAN class="mw-headline" id="Sequencing_of_pRB3-10">Sequencing of pRB3-10</SPAN></H4><P>plasmid minipreps have been sequenced by GATC [reference] using the sequencing primers VF2 and VR. The sequencing 
</P><P>results show that all the inserts are correct except for pRB9, which isn't a pity because we have previously shown 
</P><P>with pKH1-2 that the combination of bpsA(pMM64) and svp(pMM65) successful produces indigoidine.
</P><TABLE class="wikitable"><TBODY><TR><TH>plasmid</TH><TH>Primer</TH><TH>ind synthetase/ PPTase</TH><TH>sequencing result/ reference
</TH></TR><TR><TD rowspan="4">pRB3</TD><TD rowspan="2">VF2</TD><TD rowspan="2">indC</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="2">VR</TD><TD rowspan="2">sfp-rev</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="4">pRB4</TD><TD rowspan="2">VF2</TD><TD rowspan="2">indC</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="2">VR</TD><TD rowspan="2">svp(pMM65)-rev</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="4">pRB5</TD><TD rowspan="2">VF2</TD><TD rowspan="2">indC</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="2">VR</TD><TD rowspan="2">svp-rev</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="4">pRB6</TD><TD rowspan="2">VF2</TD><TD rowspan="2">indC</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="2">VR</TD><TD rowspan="2">entD-rev</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="4">pRB7</TD><TD rowspan="2">VF2</TD><TD rowspan="2">bpsA</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="2">VR</TD><TD rowspan="2">sfp-rev</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="4">pRB8</TD><TD rowspan="2">VF2</TD><TD rowspan="2">bpsA</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="2">VR</TD><TD rowspan="2">svp(pMM65)-rev</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="4">pRB9</TD><TD rowspan="2">VF2</TD><TD rowspan="2">bpsA</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="2">VR</TD><TD rowspan="2">svp-rev</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="4">pRB10</TD><TD rowspan="2">VF2</TD><TD rowspan="2">bpsA</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="2">VR</TD><TD rowspan="2">entD-rev</TD></TR><TR><TD>seq2
</TD></TR></TBODY></TABLE><P>
In the following pRB4-beta, pRB5-alpha, pRB5-beta, pRB6-alpha and pRB10-alpha have been selected so that there are 
</P><P>single versions of pRB3-10 for furhter experiments.
</P><H4><SPAN class="mw-headline" id="Glycerols_stocks_pRB3-10">Glycerols stocks pRB3-10</SPAN></H4><P>3 stocks each were made using 2 mL liquid cultures (48 h at 37 °C 200 rpm) of TOP10+pRB3-10, respectively.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Heidelberg/Templates/Indigoidine_week15">http://2013.igem.org/Team:Heidelberg/Templates/Indigoidine_week15</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Templates/Indigoidine_week15" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Templates/Indigoidine_week15" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Templates/Indigoidine_week15&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Templates/Indigoidine_week15&amp;oldid=361820" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>