<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Project_Indigoidine-Tag"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Project/Indigoidine-Tag">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Project/Indigoidine-Tag&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/Indigoidine-Tag&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/Indigoidine-Tag&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Project/Indigoidine-Tag" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Project/Indigoidine-Tag</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="row"><DIV id="nav" class="navigation"><DIV class="item team" data-offset-l="-130"><H2>Team</H2><UL><LI><A href="/Team:Heidelberg/Team">Team Members</A></LI><LI><A href="/Team:Heidelberg/AttributionsII">Attributions</A></LI><LI><A href="/Team:Heidelberg/Team/Sponsors">Sponsors</A></LI><LI><A href="/Team:Heidelberg/Team/Collaboration">Collaborations</A></LI><LI><A href="/Team:Heidelberg/Team/Gallery">Gallery</A></LI></UL></DIV><DIV class="item project" data-offset-l="-144"><H2>Project</H2><UL><LI><A href="/Team:Heidelberg/NRPS">Background</A></LI><LI><A href="/Team:Heidelberg/Project">Overview</A></LI><LI><A href="/Team:Heidelberg/Project/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Project/Indigoidine-Tag">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Project/Tag-Optimization">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Project/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Project/Achievements">Achievements</A></LI></UL></DIV><DIV class="item notebook" data-offset-l="-139"><H2>Notebook</H2><UL><LI><A href="/Team:Heidelberg/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Tyrocidine">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Indigoidine">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/NRPSDesigner">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/Notebook/Methods">Methods</A></LI><LI><A href="/Team:Heidelberg/Materials">Materials</A></LI></UL></DIV><DIV class="item parts" data-offset-l="-134"><H2>Parts</H2><UL><LI><A href="/Team:Heidelberg/Parts">Submitted Parts</A></LI><LI><A href="/Team:Heidelberg/Favorite_Parts">Favorite Parts</A></LI><LI><A href="/Team:Heidelberg/RFCs">Our RFCs</A></LI></UL></DIV><DIV class="item software" data-offset-l="-134"><H2>Software</H2><UL><LI><A href="/Team:Heidelberg/Project_Software">Project</A></LI><LI><A href="http://igem2013.bioquant.uni-heidelberg.de/NRPSDesigner/">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/iGEM42">iGEM42</A></LI><LI><A href="/Team:Heidelberg/OpenSource">Open Source</A></LI><LI><A href="/Team:Heidelberg/Project/AchievementsSoftware">Achievements</A></LI></UL></DIV><DIV class="item human" data-offset-l="-118"><H2>Human Practice</H2><UL><LI><A href="/Team:Heidelberg/HumanPractice">Overview</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BroadPublic">Broad Public</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Survey">Survey</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/YoungGeneration">Young Generation</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Artists">Arts and Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BeyondScience">Beyond Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Experts">Experts</A></LI></UL></DIV><DIV class="item safety" data-offset-l="-115"><H2>Safety</H2><UL><LI><A href="/Team:Heidelberg/Safety">Safety</A></LI></UL></DIV><DIV class="item model" data-offset-l="-144"><H2>Modeling</H2><UL><LI><A href="/Team:Heidelberg/Modelling/Gold_Recovery">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Modelling/Ind_Production">ind-Production</A></LI></UL></DIV><DIV class="item"><SPAN style="margin-left:6%">HOME</SPAN></DIV></DIV></DIV><DIV class="container"><DIV><H1><SPAN style="font-size:170%;color:#000080;">Indigoidine-Tag.</SPAN><SPAN class="text-muted" style="font-family:Arial, sans-serif; font-size:100%"> Introducing the GFP for NRPs.</SPAN></H1></DIV><DIV class="row"><DIV class="col-sm-6"><DIV class="jumbotron" style="padding:63px"><H2>Highlights</H2><P style="font-size:18px"><UL style="font-size:16px"><LI><B> Creation of an easily detectable, inert and universal NRPS-Tag using the indigoidine Synthetase
</B></LI><LI><B> Simple Verification of Non-Ribosomal Peptides via Thin Layer Chromatography
</B></LI><LI><B> Proof of principle module shuffling and domain-import among species
</B></LI><LI><B> Establishment of a standardized protocol for High-Throughput NRPS-assays: <A href="https://2013.igem.org/Team:Heidelberg/RFCs">RFC 100</A> and <A href="https://2013.igem.org/Team:Heidelberg/RFCs#rfc99">RFC 99</A></B></LI></UL></P></DIV></DIV><DIV class="col-sm-6"><DIV class="jumbotron abstract"><H2>Abstract</H2><P style="font-size:14px; text-align:justify">
                            Synthesizing peptides with various functions using Non-Ribosomal Peptide Syntheatses (NRPS) provides access to more than 500 building blocks, and hence opens up a diverse spectrum of possible products. However, detection of the Non-Ribosomal Peptide (NRP), as well as high-throughput screening of peptide functionality remained complicated or even impossible. Thus, an <B>easily detectable, inert and universal tag</B> that allows simplified screening and detection, similar to the <B>GFP</B>-tag for proteins would be required.
</P><P style="font-size:14px; text-align:justify"><B>Indigoidine</B>, a blue pigment which is synthesized by the NRPS IndC from <EM>Photorhabdus luminescens laumondii</EM> TT01 (DSM15139), was <B>established as a novel and application-oriented tag functionally fused to synthesized peptides</B>. Streamlining this process demonstrates a substantial reinforcement of the <A href="http://igem2013.bioquant.uni-heidelberg.de/NRPSDesigner/">NRPS-Designer</A>, our software tool, and offers the possibility to evaluate and optimize synthesis of NRPs in a <B>high-throughput</B> manner (<A href="https://2013.igem.org/Team:Heidelberg/RFCs#rfc99">RFC 99</A>&amp;<A href="https://2013.igem.org/Team:Heidelberg/RFCs">RFC 100</A>).
                        </P></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV id="tyrocidineText" class="col-sm-12"><H2 id="introduction">Introduction</H2><P>
Non-Ribosomal Peptide Synthetases (NRPS) are assembly lines which consist of several modules, each module incorporating one specific amino acid into the growing peptide chain (for a more detailled introduction into NRPS, please refer to our <A href="https://2013.igem.org/Team:Heidelberg/NRPS">Background Page</A>). As previously shown by both other research groups [1] and our work (Project <A href="https://2013.igem.org/Team:Heidelberg/Project/Tyrocidine">Peptide-Synthesis</A>), NRPS modules can be rearranged to form a novel assembly line which produces a custom, non-ribosomal peptide. However, the detection of those synhetic peptides using mass spectrometry is still challenging. In order to simplify the detection of peptides created by custom NRPSs, we developed a blue pigment tag for non-ribosomal peptides - the Indigoidine-Tag.
</P><P>
The indigoidine synthetase indC from <I>Photorhabdus luminescens laumondii</I> TT01 consists of an <B>A</B>denylation domain with an internal <B>Ox</B>idation domain, a <B>T</B>hiolation domain and a <B>T</B>hio<B>E</B>sterase domain. The A-domain adenylates L-glutamine which is then attached to the T-domain via a thioester bond. The TE-domain catalyzes the cyclization of the glutamine and cleaves it from the T-domain. Two cyclic glutamines are oxidized by the Ox-domain, resulting in the blue pigment dimer indigoidine (<A class="fancybox fancyFigure" title="Figure 1: The indigoidine synthetase IndC catalyzes the formation of the blue pigment indigoidine. a) The indigoidine synthetase indC from &lt;i&gt;P. luminescens&lt;/i&gt; is a single module NRPS catalyzing the formation of the blue pigment indigoidine by cyclization and oxidation of two L-glutamines. b) Expression of a functional indigoidine synthetase in &lt;i&gt;E. coli&lt;/i&gt; BAP1 cells leads to a blue phenotype." href="https://static.igem.org/mediawiki/2013/a/a2/Heidelberg_indtag_fig1.png" rel="gallery1">Fig. 1a</A>)[2]. This leads to a blue phenotype of <I>E. coli</I> cells expressing the indC gene when grown on plates or in liquid cultures (<A class="fancybox fancyFigure" title="Figure 1: The indigoidine synthetase IndC catalyzes the formation of the blue pigment indigoidine. a) The indigoidine synthetase indC from &lt;i&gt;P. luminescens&lt;/i&gt; is a single module NRPS catalyzing the formation of the blue pigment indigoidine by cyclization and oxidation of two L-glutamines. b) Expression of a functional indigoidine synthetase in &lt;i&gt;E. coli&lt;/i&gt; BAP1 cells leads to a blue phenotype." href="https://static.igem.org/mediawiki/2013/a/a2/Heidelberg_indtag_fig1.png" rel="gallery1">Fig. 1b</A>). The blue pigment can be purified and dissolved in DMSO using a simple protocol.
</P><CENTER><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/a/a2/Heidelberg_indtag_fig1.png" title="Figure 1: The indigoidine synthetase IndC catalyzes the formation of the blue pigment indigoidine. a) The indigoidine synthetase indC from &lt;i&gt;P. luminescens&lt;/i&gt; is a single module NRPS catalyzing the formation of the blue pigment indigoidine by cyclization and oxidation of two L-glutamines. b) Expression of a functional indigoidine synthetase in &lt;i&gt;E. coli&lt;/i&gt; BAP1 cells leads to a blue phenotype."><FIGCAPTION style="width:60%;"><B>Figure 1: The indigoidine synthetase IndC catalyzes the formation of the blue pigment indigoidine.</B> a) The indigoidine synthetase indC from <I>P. luminescens</I> is a single module NRPS catalyzing the formation of the blue pigment indigoidine by cyclization and oxidation of two L-glutamines. b) Expression of a functional indigoidine synthetase in <I>E. coli</I> BAP1 cells leads to a blue phenotype.</FIGCAPTION></A></CENTER><P>
We fused the indC gene to NRPS modules of the tyrocidine synthesis cluster from <I>Brevibacillus parabrevis</I> to create novel NRPS assembly lines which attach the blue pigment indigoidine to the last amino acid of the synthesized peptide (<A class="fancybox fancyFigure" title="Figure 2: The IndC module is fused to other NRPS modules to establish the indigoidine-Tag. When combining the coding sequences of NRPS modules from diverse Non-Ribosomal Peptide Synthetases, such as antibiotic biosynthesis clusters, with the indC indigoidine synthetase module, the resulting assembly line will eventually produce a indigoidine-tagged peptide." href="https://static.igem.org/mediawiki/2013/e/e5/Heidelberg_indtag_fig2.png" rel="gallery1">Fig. 2</A>). After purification, the tagged peptide can be validated using comparative Thin Layer Chromatography (TLC) or Mass-Spectrometry (MassSpec) after purification by High Pressure Liquid Chromatography (HPLC). 
</P><CENTER><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/e/e5/Heidelberg_indtag_fig2.png" title="Figure 2: The IndC module is fused to other NRPS modules to establish the Indigoidine-Tag.
When combining the coding sequences of NRPS modules from diverse Non-Ribosomal Peptide Synthetases, such as antibiotic biosynthesis clusters, with the indC indigoidine synthetase module, the resulting assembly line will eventually produce an indigoidine-tagged peptide."><FIGCAPTION style="width:60%;"><B>Figure 2: The IndC module is fused to other NRPS modules to establish the Indigoidine-Tag.</B> When combining the coding sequences of NRPS modules from diverse Non-Ribosomal Peptide Synthetases, such as antibiotic biosynthesis clusters, with the indC indigoidine synthetase module, the resulting assembly line will eventually produce a indigoidine-tagged peptide. 
</FIGCAPTION></A></CENTER><P>
The possibility of tagging non-ribosomal peptides makes high-throughput protocols possible. Therefore, we created a standardized procedure for the production of NRPs in our <A href="https://2013.igem.org/Team:Heidelberg/RFCs">RFC100</A>: i) design of novel NRPSs with our NRPS-Designer software, ii) high-throughput construction of NRPS-libraries, iii) the detection and validation of the synthetic peptides and iv) functional assays with possible upscaling of the peptide production to industrial level.
</P><H2 id="results">Results </H2><H3><SPAN class="mw-headline" id="Showing_Inter-Species_Module_Compatibility_by_Fusion_of_Tyrocidine_Modules_to_the_Indigoidine_Synthetase">Showing Inter-Species Module Compatibility by Fusion of Tyrocidine Modules to the Indigoidine Synthetase</SPAN></H3><P>
Our module-shuffling approach within the tyrocidine cluster was confirmed by MassSpec. Access to such special technical devices and the referring expertise is demanded for detection of small peptides but often limited. That is why we were thinking of a potential alternative to test for the synthesis of NRPs. We wanted to establish an assay accessible to the majority of the community for validation of the presence of custom peptides. Since the synthetase for indigoidine consists of only a single module, it could serve as a paradigm for the fusion to modules of other NRPSs. The pigment indigoidine could potentially ease detection of peptides when they are fused to the dye that is visible by eye.
</P><P>
Thus we wanted to investigate whether it is possible to fuse indigoidine as a tag to NRPs of other pathways even originating from other species. In the following, we will showcase the extent of NRPSsâ modular compatibility by creating inter-species module-fusions between the indigoidine synthetase from <EM>P. luminescens</EM> and modules of the tyrocidine synthetase from <EM>B. parabrevis</EM>. In those fusion NRPs, indigoidine serves as a tag that eases identification of <EM>E. coli</EM> clones synthesizing the customized peptide.
</P><H3><SPAN class="mw-headline" id="Fusing_Single_Amino_Acids_to_Indigoidine">Fusing Single Amino Acids to Indigoidine</SPAN></H3><P>
First, we combined single modules of the tyrocidine synthetase with the indC synthetase resulting in three distinct NRPSs producing asparagine, valine or phenylalanine respectively, all tagged with indigoidine (see <A class="fancybox fancyFigure" title="Figure 3: Composition of three fusion constructs originating from tyrocidine (tyc) synthetase modules and the indigoidine synthetase (indC). Labeling of modules in the first row describes the modules of the tyrocidine synthetase. Coloured modules in the three rows below were fused together to create plasmids encoding for novel NRPSs (depicted in gray rectangle)." href="https://static.igem.org/mediawiki/2013/f/fc/Heidelberg_Ind-Tag_Figure4.png" rel="gallery1">Fig. 3</A>).
</P><CENTER><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/f/fc/Heidelberg_Ind-Tag_Figure4.png" title="Figure 3: Composition of three fusion constructs originating from tyrocidine (tyc) synthetase modules and the indigoidine synthetase (indC). Labeling of modules in the first row describes the modules of the tyrocidine synthetase. Coloured modules in the three rows below were fused together to create plasmids encoding for novel NRPSs (depicted in gray rectangle)."><FIGCAPTION style="width:60%;"><B>Figure 3: Composition of three fusion constructs originating from tyrocidine (tyc) synthetase modules and the indigoidine synthetase (indC)</B>. Figure 3: Composition of three fusion constructs originating from tyrocidine (tyc) synthetase modules and the indigoidine synthetase (indC). Labeling of modules in the first row describes the modules of the tyrocidine synthetase. Coloured modules in the three rows below were fused together to create plasmids encoding for novel NRPSs (depicted in gray rectangle).</FIGCAPTION></A></CENTER><P>
To assure compatibility, the indigoidine module was always preceded by the C2 module, as the  <A class="fancybox fancyFigure" title="Figure 1: Overview of the tyrocidine Cluster. Ten Modules are formed from three gene clusters resulting in ten amino acid long peptides. In an assembly line manner each amino acid is added consecutively to the nascent, before the finale product is cleaved and released. (Adapted from [10])" href="https://static.igem.org/mediawiki/2013/4/44/Heidelberg_TycCluster_Scheme.png" rel="gallery1">tyc-C2</A> is specific for glutamine which is required for indigoidine production. SDS-PAGE showed the expected bands for the expression of the NRP synthetases in the transformed BAP1. The <EM>E. coli</EM> strain BAP1 was used for expression of the NRPS fusions because it carries the required PPTase sfp under the control of a T7 promoter. All three of the fusion variants turned the colonies blue even before expression induction with IPTG. The blue pigment thus served a first indicator that peptide synthesis was successful. To further verify the existence of the fusion peptide, we ran comparative TLC. The native, purified indigoidine ran further than our purified dipeptides suggesting that the amino acids were indeed fused to the pigment (<A class="fancybox fancyFigure" title="Figure 4: Comparative TLC of our tagged NRP Val-Ind with native indigoidine. Three different biological replicates of the produced NRP, a valin-indigoidine fusion peptide (Val-Ind), are compared to an indigoidine control (Ind-ctrl). Clearly visibly, the tagged NRP shows an altered migration behavior on TLC with Dichloromethane as running solvent." href="https://static.igem.org/mediawiki/2013/9/9e/Heidelberg_TLC_5.png" rel="gallery1">Fig. 4</A>). The peptides were detected under visible and UV light due to indigoidineâs properties as a dye.
</P><CENTER><A class="fancybox fancyGraphical" href="http://igem2013.bioquant.uni-heidelberg.de/wiki/images/7/73/TLC_5.png" title="Figure 4: Comparative TLC of our tagged NRP Val-Ind with native indigoidine. Three different biological replicates of the produced NRP, a valin-indigoidine fusion peptide (Val-Ind), are compared to an indigoidine control (Ind-ctrl). Clearly visibly, the tagged NRP shows an altered migration behavior on TLC with Dichloromethane as running solvent."><FIGCAPTION style="width:60%;"><B>Figure 4: Comparative TLC of our tagged NRP Val-Ind with native indigoidine.</B> Three different biological replicates of the produced NRP, a valin-indigoidine fusion peptide (Val-Ind), are compared to an indigoidine control (Ind-ctrl). Clearly visibly, the tagged NRP shows an altered migration behavior on TLC with Dichloromethane as running solvent.</FIGCAPTION></A></CENTER><H3><SPAN class="mw-headline" id="Using_Indigoidine_as_a_Tag_for_Non-Ribosomal_Peptides">Using Indigoidine as a Tag for Non-Ribosomal Peptides</SPAN></H3><P>
To gather additional evidence for our functional indigoidine tag, we assembled seven variants with up to three modules in front of the indigoidine synthetase (<A class="fancybox fancyFigure" title="Figure 5: Composition of seven fusion constructs originating from tyrocidine (tyc) synthetase modules and the indigoidine synthetase (indC). The contructs depicted above serve as a proof of principle for the tagging of Non-Ribosomal Peptides with indigoidine. Several constructs were created using a valin-spacer in order to assess the influence of sterically hindrance by larger or polar amino acids. To be able to show the applicability of the tag most clearly, the di- and tripeptide constructs described on the Peptide Synthesis page, were tagged with indigoidine as well." href="https://static.igem.org/mediawiki/2013/d/db/Heidelberg_Ind-Tag_Figure6.png" rel="gallery1">Fig. 5</A>) following the same approach as described above.
</P><CENTER><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/d/db/Heidelberg_Ind-Tag_Figure6.png" title="Figure 5: Composition of seven fusion constructs originating from tyrocidine (tyc) synthetase modules and the indigoidine synthetase (indC). The contructs depicted above serve as a proof of principle for the tagging of Non-Ribosomal Peptides with indigoidine. Several constructs were created using a valin-spacer in order to assess the influence of sterically hindrance by larger or polar amino acids. To be able to show the applicability of the tag most clearly, the di- and tripeptide constructs described on the Peptide Synthesis page, were tagged with indigoidine as well."><FIGCAPTION style="width:60%;"><B>Figure 5: Composition of seven fusion constructs originating from tyrocidine (tyc) synthetase modules and the indigoidine synthetase (indC).</B> The contructs depicted above serve as a proof of principle for the tagging of Non-Ribosomal Peptides with indigoidine. Several constructs were created using a valin-spacer in order to assess the influence of sterically hindrance of bigger or polar amino acids. To be able to show the applicability of the tag most clearly, the di- and tripeptide constructs described on the Peptide Synthesis page, were tagged with indigoidine as well.</FIGCAPTION></A></CENTER><P>
Again the constructs pPW06, pPW09, pPW10, pPW11 and pPW12 (<A class="fancybox fancyFigure" title="Figure 5: Composition of seven fusion constructs originating from tyrocidine (tyc) synthetase modules and the indigoidine synthetase (indC). The contructs depicted above serve as a proof of principle for the tagging of Non-Ribosomal Peptides with indigoidine. Several constructs were created using a valin-spacer in order to assess the influence of sterically hindrance by larger or polar amino acids. To be able to show the applicability of the tag most clearly, the di- and tripeptide constructs described on the Peptide Synthesis page, were tagged with indigoidine as well." href="https://static.igem.org/mediawiki/2013/d/db/Heidelberg_Ind-Tag_Figure6.png" rel="gallery1">Fig. 5</A>) turned <EM>E. coli</EM> BAP1 colonies blue upon transformation. Even with increasing peptide length, synthesis did not seem to be affected and the dye-properties of the indigoidine were still preserved (<A class="fancybox fancyFigure" title="Figure 6: Comparison of liquid cultures of &lt;em&gt;E. coli&lt;/em&gt; BAP1 transformed with different indigoidine-Tag constructs. The image shows a comparison of different constructs, varying in length. Shown are two tagged amino acids (Val-Ind and Asn-Ind), one tagged dipeptide (Orn-Val-Ind) and one tagged tripeptide (Phe-Orn-Leu-Ind). The constructs are hence two dimers, one trimer and one tetramer." href="https://static.igem.org/mediawiki/2013/6/60/Heidelberg_Ind-Tag_Figure7.png" rel="gallery1">Fig. 6</A>).
</P><CENTER><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/6/60/Heidelberg_Ind-Tag_Figure7.png" title="Figure 6: Comparison of liquid cultures of &lt;em&gt;E. coli&lt;/em&gt; BAP1 transformed with different Indigoidine-Tag constructs. The image shows a comparison of different constructs, varying in length. Shown are two tagged amino acids (Val-Ind and Asn-Ind), one tagged dipeptide (Orn-Val-Ind) and one tagged tripeptide (Phe-Orn-Leu-Ind). The constructs are hence two dimers, one trimer and one tetramer."><FIGCAPTION style="width:60%;"><B>Figure 6: Comparison of liquid cultures of <EM>E. coli</EM> BAP1 transformed with different Indigoidine-Tag constructs.</B> The image shows a comparison of different constructs, varying in length. Shown are two tagged amino acids (Val-Ind and Asn-Ind), one tagged dipeptide (Orn-Val-Ind) and one tagged tripeptide (Phe-Orn-Leu-Ind). The constructs are hence two dimers, one trimer and one tetramer.</FIGCAPTION></A></CENTER><P>
From this, we deduced that indigoidine possesses characteristics required for a proper peptide tag that we would like to propose for the use within the community (<A href="https://2013.igem.org/Team:Heidelberg/RFCs">RFC 100</A>). For this purpose we have created a <A href="https://2013.igem.org/Team:Heidelberg/Favorite_Parts">ccdB-dependent vector</A> to ease the tagging of NRPs, accessible through the parts registry. Design of such constructs is enabled with our software, the <A href="http://igem2013.bioquant.uni-heidelberg.de/NRPSDesigner/" title="wikilink">NRPS Designer</A>. The ccdB-helper-construct (<A href="https://2013.igem.org/Team:Heidelberg/Favorite_Parts">BBa_K1152007</A>) is designed to lower the background of negative transformants: If the insert is cloned in the vecor correctly, there is no active ccdB present and the cell will survive. If the backbone religates or template-backbone is still present in the Gibson-Mix, cells will die due to ccdB expression. The impact of ccdB expression is depicted in <A class="fancybox fancyFigure" href="https://static.igem.org/mediawiki/2013/4/40/Heidelberg_ccdB_comparison_2.png" title="Figure 7: Effect of ccdB on non-resistant cells. Regular &lt;em&gt;E. coli&lt;/em&gt; TOP10 cells die in presence of active ccdB (left side), while ccdB-resistant cells survive (right side). Hence this comparison shows the effectiveness of ccdB and the minimization of background to about 0%. Using the ccdB-helper plasmid for a Gibson-driven tagging-approach enhances effectiveness significantly." rel="gallery1">Fig. 7</A></P><CENTER><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/9/9a/Heidelberg_ccdB_comparison.png" title="Figure 7: Effect of ccdB on non-resistant cells. Regular &lt;em&gt;E. coli&lt;/em&gt; TOP10 cells die in presence of active ccdB (left side), while ccdB-resistant cells survive (right side). Hence this comparison shows the effectiveness of ccdB and the minimization of background to about 0%. Using the ccdB-helper plasmid for a Gibson-driven tagging-approach enhances effectiveness significantly."><FIGCAPTION style="width:60%;"><B>Figure 7: Effect of ccdB on non-resistant cells.</B> Regular <EM>E. coli</EM> TOP10 cells die in presence of active ccdB (left side), while ccdB-resistant cells survive (right side). Hence this comparison shows the effectiveness of ccdB and the minimization of background to about 0%. Using the ccdB-helper plasmid for a Gibson-driven tagging-approach enhances effectiveness significantly.</FIGCAPTION></A></CENTER><H3><SPAN class="mw-headline" id="Experimental_Validation_of_Software_Predictions">Experimental Validation of Software Predictions</SPAN></H3><P>
We propose the Indigoidine-Tag as a part of our standards <A href="https://2013.igem.org/Team:Heidelberg/RFCs#rfc99">RFC 99</A> and <A href="https://2013.igem.org/Team:Heidelberg/RFCs">RFC 100</A>. The crucial tool in this standardized process is the <A href="http://igem2013.bioquant.uni-heidelberg.de/NRPSDesigner/" title="wikilink">NRPS Designer</A>. The <A href="http://igem2013.bioquant.uni-heidelberg.de/NRPSDesigner/" title="wikilink">NRPS Designer</A> predicts module boundaries and linker regions based on <A class="fancybox fancyFigure" href="https://static.igem.org/mediawiki/2013/2/2e/Heidelberg2013_TycCAclustalComparison.png" title=" Clustal Omega MSA of different C-A domain borders of Tyrocidine cluster with annotation (prediction) of the start of the A domain according to different tools and experimental results." rel="gallery1">Hidden Markov Models</A>. We used these predictions for our module shuffling experiments, which all worked successfully in our constructs. However, other tools predict <A class="fancybox fancyFigure" href="https://static.igem.org/mediawiki/2013/2/2e/Heidelberg2013_TycCAclustalComparison.png" title=" Clustal Omega MSA of different C-A domain borders of Tyrocidine cluster with annotation (prediction) of the start of the A domain according to different tools and experimental results." rel="gallery1">different boarders</A>, hence, we wanted to evaluate the predictions to contribute more data to the NRPS Designer and to provide experimental feedback to our software. Therefore, we systematically varied the module boundaries of the A domain and C domain of C2 within the Val-Ind-construct in relation to the <A href="http://pfam.sanger.ac.uk/">Pfam</A> prediction. Based on this altered domain positioning, eight constructs were designed combining narrow, broad and original <A href="http://pfam.sanger.ac.uk/">Pfam</A> module regions (<A class="fancybox fancyFigure" title="Figure 8: Overview of our constructs for investigation of the different domain borders in relation to Pfam. Dark colors are variations in the start-region of the A domain borders and light colors are variations in the end-region of the C domain. Setting of proper domain boarders is crucial for module shuffling, both, within and in between pathways and organisms. We hence assessed adequate definitions of where and how to build the linker regions by vaying them and checking for functionality. We therefore created the eight constructs shown above. Those are all valin-indigoidine synthetases with different settings of domain boarders." href="https://static.igem.org/mediawiki/2013/e/ed/Heidelberg_Results_LV_overview2.png" rel="gallery1">Fig. 8</A>). Their functionality was always compared to the original construct based on the boundaries obtained from <A href="http://pfam.sanger.ac.uk/">Pfam</A>. We could induce the production of Val-Ind in two of our constructs: pLV03 (narrow and Pfam border) and pLV08 (broad and narrow border)
</P><CENTER><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/e/ed/Heidelberg_Results_LV_overview2.png" title="Figure 8: Overview of our constructs for investigation of the different domain borders in relation to Pfam. Setting of proper domain boarders is crucial for module shuffling, both, within and in between pathways and organisms. We hence assessed adequate definitions of where and how to build the linker regions by vaying them and checking for functionality. We therefore created the eight constructs shown above. Those are all valin-indigoidine synthetases with different settings of domain boarders."><FIGCAPTION style="width:60%"><B>Figure 8: Overview of our constructs for investigation of the different domain borders in relation to Pfam.</B> Dark colors are variations in the start-region of the A domain borders and light colors are variations in the end-region of the C domain. Setting of proper domain boarders is crucial for module shuffling, both, within and in between pathways and organisms. We hence assessed adequate definitions of where and how to build the linker regions by vaying them and checking for functionality. We therefore created the eight constructs shown above. Those are all valin-indigoidine synthetases with different settings of domain boarders.</FIGCAPTION></A></CENTER><P>
To summarize, we successfully validated the concept of modularity both for intra- and interspecies shuffeling. We have integrated our experimental data and fuelled it into our software to improve its prediction accuracy. To make NRPSs and their custom design more accessible to the community, we have submitted standardized versions of the modules we used for our shuffling experiments, to the parts registry (see the <A href="http://parts.igem.org" title="wikilink">parts registry</A> entries).
</P><H2 id="discussion">Discussion</H2><P>
The bottleneck to test libraries of combinatorial NRPSs with an array of different modules is in fact the screening for functional enzymes <SPAN class="citation">[3]</SPAN><SPAN class="citation">[4]</SPAN>. How to identify novel NRPSs consisting of compatible modules? Our experimental results (<A class="fancybox fancyFigure" title="Figure 4: Comparative TLC of our tagged NRP Val-Ind with native indigoidine. Three different biological replicates of the produced NRP, a valin-indigoidine fusion peptide (Val-Ind), are compared to an indigoidine control (Ind-ctrl). Clearly visibly, the produced NRP shows a significantly altered migration behavior on Thin Layer Chromatography with Dichloromethane as running solvent." href="https://static.igem.org/mediawiki/2013/9/9e/Heidelberg_TLC_5.png" rel="gallery2">Fig. 4</A>) strongly support the hypothesis that the synthesis of short peptides can be easily monitored when fused to indigoidine. This novel technique that represents an equivalence of the GFP-tag for proteins in NRPs outrules classical detection by Mass-Spec due to the increased efficiency and high-throughput character. Our findings that NRPSs seem to offer a framework going beyond species borders confirm the results shown by Marahiel <SPAN class="citation">[1]</SPAN>. Using indigoidine as pigment-module for the fusion results in a fusion NRP which is even detectable by eye. 
</P><P>
With this, we offer a novel and very efficient way of tagging NRPs with indigoidine. The dye can be easily measured, quantified and even <A href="https://2013.igem.org/Team:Heidelberg/Project/Tag-Optimization">optimized</A>. This aspect furthermore appends a valuable feature of the NRPS Designer. Users now have the opportunity to easily add an Indigoidine-Tag when they design their constructs, which is also described in our proposed RFC (<A href="https://2013.igem.org/Team:Heidelberg/RFCs">RFC 100</A>). Noteworthy, our experiments we could show that the indigoidine tag works fine when put behind modules for different amino acids, but the tagging clearly works best, when an apolar, small spacer amino-acid such as Glycine, Alanine or, as in our experiments, valin is used, due to less steric hindrance.
</P><CENTER><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/e/ef/Heidelberg_Val-Ind_NRPS_Protein.png" title="Title 9: Predicted tertiary structure of the valin-indigoidine-synthetase. Crucial domains are clearly visible and distinguishable in prediction. Prediction was carried out using PHYRE2 secondary structure prediction with multiple subsets of the protein sequence."><FIGCAPTION style="width:60%"><B>Figure 9: Predicted tertiary structure of the valin-indigoidine-Synthetase.</B> Crucial domains are clearly visible and distinguishable in prediction. Prediction was carried out using PHYRE2 secondary structure prediction with multiple subsets of the protein sequence.</FIGCAPTION></A></CENTER><P>
Compared to other potential methods for <EM>in-vivo</EM> tagging of NRPs, as it has been described before <SPAN class="citation">[5]</SPAN> and <SPAN class="citation">[6]</SPAN>, the Indigoidine tag has the apparent advantage that it is relatively small compared to e.g. a His tag, for which four to nine Histidines are required. Synthesizing a short peptide with several modules for Histidine is imaginable, but would double or triple the size of the required NRPS, and hence of the vector, which is highly ineffective. Fluorescent proteins (FPs) have the advantage of easy detection but are simply too big to use them. Imagining e.g. GFP synthesized by an NRPS is practically not feasible. In summary, the Indigoidine-Tag, in contrast to all other imaginable tagging methods for NRPs fulfills the required characteristics of being small, inert, universal <EM>and</EM> easily detectable.
</P><P>
As far as <EM>in-vitro</EM> approaches are concerned, there are, in principle, two ways. 

I) Either one could add a tagging agent to the cells or the medium before purification â which would be the case for e.g. Click-Chemistry.

II) Or one could add a certain tag such as a His tag to the NRPS and perform the entire synthesis of the NRP <EM>in vitro</EM>.

The latter approach has been widely used <SPAN class="citation">[7]</SPAN><SPAN class="citation">[8]</SPAN>, is, however in vitro and hence less effective for a high-throughput advance as a functioning in-vivo-method. Besides, the upscaling of such an approach would hardly be feasible due to increasing expenses that would turn down financability. We ourselves thought of using methods such as Click-Chemistry for the purification of the short, synthetic peptides. Increased masses and easy cleavage enable high purities and can be determined via HPLC with UV and ESI-MS detection <SPAN class="citation">[9]</SPAN>. This approach, however, does not offer any opportunity to evaluate expression <EM>in vivo</EM><SPAN class="citation">[10]</SPAN>.
 
</P><P>
Hence, using and Indigoidine-Tag, which can be added to the nascent NRP in the process of its formation by adding a 4 kbp indigoidine-module to the NRPS is a novel and effective approach for labeling NRPs for quantitative expression analysis. Its properties and effects for peptide synthesis via NRPS compare to the ones of <B>GFP</B> for proteins: it is relatively <B>inert, easily detectable and universal.</B></P><H2 id="meth">Methods</H2><H3><SPAN class="mw-headline" id="Purification_of_Indigoidine_and_Tagged_Constructs">Purification of Indigoidine and Tagged Constructs</SPAN></H3><P>
1 ml of IPTG-induced, blue culture was spun down at full speed (14,000 rpm) for 20 minutes, washed in 1 ml of methanol and centrifuged once more for 5 minutes at 14,000 rpm. Methanol was discarded and samples were dissolved in 200-400 Âµl DMSO.
</P><CENTER><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/5/55/Heidelberg_Methods_Purification.png" title="Figure 10: Workflow followed during purification of the tagged peptides. A: Blue overnight culture with the secreted tagged peptide to purify. B: Blue pellet after spinning down cells for 20 minutes at 14,000 rpm. C: The pellet is washed in methanol and spun down again. D: The tagged peptide dissolves in DMSO and can be applied to TLC or further purified if wished."><FIGCAPTION style="width:60%;"><B>Figure 10: Workflow followed during purification of the tagged peptides.</B> A: Blue overnight culture with the secreted tagged peptide to purify. B: Blue pellet after spinning down cells for 20 minutes at 14,000 rpm. C: The pellet is washed in methanol and spun down again. D: The tagged peptide dissolves in DMSO and can be applied to TLC or further purified if wished.</FIGCAPTION></A></CENTER><H3><SPAN class="mw-headline" id="Thin_Layer_Chromatography">Thin Layer Chromatography</SPAN></H3><P>
TLC was carried out on silica-gel as immobile phase and Dichloromethane as mobile phase. For the procedure, a 50 ml beaker was filled with ~15 ml Dichloromethane and let stand for about 10 minutes (in order to let the Dichloromethane-vapor fill the beaker). The TLC plate, coated with silica-gel was spotted with sample 0.5 to 1 cm above the lower edge and placed in the beaker. TLC was run until the solvent front was at two thirds of the TLC plate. As indigoidine is light-sensitive, the beaker was covered with aluminium foil in order to prevent direct light irradiation.
</P><H3><SPAN class="mw-headline" id="Column_Chromatography">Column Chromatography</SPAN></H3><P>
In order to show that further purification (beyond the regular purification procedure) is feasible and easy with the tagged peptides, we provide a protocol for analysis and purification of the sample via column-chromatography. For this purpose, the purified sample that is dissolved in DMSO is applied to a Pre-Packed column G25, as running buffer, DMSO was used. DMSO is constantly applied to the column to prevent it from drying. All uncolored fractions are discarded, while the blue fraction is kept and collected in a 2 ml Eppendorf tube. This purified extract (with the sample dissolved in DMSO) can then be used for TLC or Mass-Spectrometry if desired.
</P><CENTER><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/b/b2/Heidelberg_Methods_Column.png" title="Figure 11: Column Chromatography of tagged peptides. A: The blue fraction can be easily detected and collected. B: The purified product is dissolved and DMSO and can then be further analyzed."><FIGCAPTION style="width:60%;"><B>Figure 11: Column Chromatography of tagged peptides.</B> A: The blue fraction can be easily detected and collected. B: The purified product is dissolved and DMSO and can then be further analyzed.</FIGCAPTION></A></CENTER><H3><A href="https://2013.igem.org/Team:Heidelberg/RFCs">RFC 100</A></H3><B>For the entire practical procedure of the lab-work that is involved in our standardized protocol for high-throughput cloning and fast purification and validation of the product, feel welcome to visit the <A href="https://2013.igem.org/Team:Heidelberg/RFCs">RFC 100</A>-page.
</B></DIV><DIV class="col-sm-12 jumbotron"><DIV class="references"><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Heidelberg/Project/Indigoidine-Tag">http://2013.igem.org/Team:Heidelberg/Project/Indigoidine-Tag</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Project/Indigoidine-Tag" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Project/Indigoidine-Tag" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Project/Indigoidine-Tag&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Project/Indigoidine-Tag&amp;oldid=361494" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>