<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Project_Methods"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Project/Methods">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Project/Methods&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/Methods&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/Methods&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Project/Methods" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Project/Methods</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="row"><DIV id="nav" class="navigation"><DIV class="item team" data-offset-l="-130"><H2>Team</H2><UL><LI><A href="/Team:Heidelberg/Team">Team Members</A></LI><LI><A href="/Team:Heidelberg/AttributionsII">Attributions</A></LI><LI><A href="/Team:Heidelberg/Team/Sponsors">Sponsors</A></LI><LI><A href="/Team:Heidelberg/Team/Collaboration">Collaborations</A></LI><LI><A href="/Team:Heidelberg/Team/Gallery">Gallery</A></LI></UL></DIV><DIV class="item project" data-offset-l="-144"><H2>Project</H2><UL><LI><A href="/Team:Heidelberg/NRPS">Background</A></LI><LI><A href="/Team:Heidelberg/Project">Overview</A></LI><LI><A href="/Team:Heidelberg/Project/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Project/Indigoidine-Tag">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Project/Tag-Optimization">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Project/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Project/Achievements">Achievements</A></LI></UL></DIV><DIV class="item notebook" data-offset-l="-139"><H2>Notebook</H2><UL><LI><A href="/Team:Heidelberg/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Tyrocidine">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Indigoidine">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/NRPSDesigner">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/Notebook/Methods">Methods</A></LI><LI><A href="/Team:Heidelberg/Materials">Materials</A></LI></UL></DIV><DIV class="item parts" data-offset-l="-134"><H2>Parts</H2><UL><LI><A href="/Team:Heidelberg/Parts">Submitted Parts</A></LI><LI><A href="/Team:Heidelberg/Favorite_Parts">Favorite Parts</A></LI><LI><A href="/Team:Heidelberg/RFCs">Our RFCs</A></LI></UL></DIV><DIV class="item software" data-offset-l="-134"><H2>Software</H2><UL><LI><A href="/Team:Heidelberg/Project_Software">Project</A></LI><LI><A href="http://igem2013.bioquant.uni-heidelberg.de/NRPSDesigner/">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/iGEM42">iGEM42</A></LI><LI><A href="/Team:Heidelberg/OpenSource">Open Source</A></LI><LI><A href="/Team:Heidelberg/Project/AchievementsSoftware">Achievements</A></LI></UL></DIV><DIV class="item human" data-offset-l="-118"><H2>Human Practice</H2><UL><LI><A href="/Team:Heidelberg/HumanPractice">Overview</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BroadPublic">Broad Public</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Survey">Survey</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/YoungGeneration">Young Generation</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Artists">Arts and Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BeyondScience">Beyond Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Experts">Experts</A></LI></UL></DIV><DIV class="item safety" data-offset-l="-115"><H2>Safety</H2><UL><LI><A href="/Team:Heidelberg/Safety">Safety</A></LI></UL></DIV><DIV class="item model" data-offset-l="-144"><H2>Modeling</H2><UL><LI><A href="/Team:Heidelberg/Modelling/Gold_Recovery">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Modelling/Ind_Production">ind-Production</A></LI></UL></DIV><DIV class="item"><SPAN style="margin-left:6%">HOME</SPAN></DIV></DIV></DIV><DIV class="container" style="min-width:1000px;"><DIV style="margin-top:80px"><H1><SPAN style="font-size:150%;color:#FFCC00;">Methods.</SPAN><SPAN class="text-muted" style="font-size:90%"> How to obtain cutting-edge results.</SPAN></H1><P>In the following, we provide a comprehensive list of the methods we applied during our experiments and for our software.</P></DIV><DIV class="row"><DIV class="col-md-6"><DIV class="jumbotron"><DIV class="container"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Cloning"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Cloning</SPAN></A></LI><LI class="toclevel-2"><A href="#Colony_PCR"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI><LI class="toclevel-2"><A href="#Agarose_gel_electrophoresis"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Agarose gel electrophoresis</SPAN></A></LI><LI class="toclevel-2"><A href="#Gibson_assembly"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Gibson assembly</SPAN></A></LI><LI class="toclevel-2"><A href="#Biobrick_assembly"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Biobrick assembly</SPAN></A></LI><LI class="toclevel-2"><A href="#Restriction_digests"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Restriction digests</SPAN></A></LI><LI class="toclevel-2"><A href="#Extraction_of_DNA_fragments_from_an_agarose_gel"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Extraction of DNA fragments from an agarose gel</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-2"><A href="#Nucleotide_Removal"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">Nucleotide Removal</SPAN></A></LI><LI class="toclevel-2"><A href="#Purification_of_PCR_products"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">Purification of PCR products</SPAN></A></LI><LI class="toclevel-2"><A href="#Plasmid-DNA_isolation"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">Plasmid-DNA isolation</SPAN></A></LI><LI class="toclevel-3"><A href="#Miniprep"><SPAN class="tocnumber">1.10.1</SPAN><SPAN class="toctext">Miniprep</SPAN></A></LI><LI class="toclevel-3"><A href="#Midiprep"><SPAN class="tocnumber">1.10.2</SPAN><SPAN class="toctext">Midiprep</SPAN></A></LI><LI class="toclevel-3"><A href="#Preparation_of_large_plasmids"><SPAN class="tocnumber">1.10.3</SPAN><SPAN class="toctext">Preparation of large plasmids</SPAN></A></LI></UL><LI class="toclevel-2"><A href="#Sequencing"><SPAN class="tocnumber">1.11</SPAN><SPAN class="toctext">Sequencing</SPAN></A></LI><LI class="toclevel-1"><A href="#Protein_detection"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Protein detection</SPAN></A></LI><LI class="toclevel-2"><A href="#Thin_Layer_Chromatography"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Thin Layer Chromatography</SPAN></A></LI><LI class="toclevel-2"><A href="#Purification_of_Indigoidine_and_tagged_constructs"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Purification of Indigoidine and tagged constructs</SPAN></A></LI><LI class="toclevel-2"><A href="#Purification_of_Delftibactin"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Purification of Delftibactin</SPAN></A></LI><LI class="toclevel-2"><A href="#Mass_spectrometry"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Mass spectrometry</SPAN></A></LI><LI class="toclevel-3"><A href="#Cell_preparation"><SPAN class="tocnumber">2.4.1</SPAN><SPAN class="toctext">Cell preparation</SPAN></A></LI><LI class="toclevel-3"><A href="#Data_Acquisition"><SPAN class="tocnumber">2.4.2</SPAN><SPAN class="toctext">Data Acquisition</SPAN></A></LI><LI class="toclevel-2"><A href="#SDS-PAGE"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">SDS-PAGE</SPAN></A></LI><LI class="toclevel-1"><A href="#1st_Pre-Experiment"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">1st Pre-Experiment</SPAN></A></LI><LI class="toclevel-2"><A href="#Making_ACM_Medium"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Making ACM Medium</SPAN></A></LI><LI class="toclevel-3"><A href="#for_1_L"><SPAN class="tocnumber">3.1.1</SPAN><SPAN class="toctext">for 1 L</SPAN></A></LI><LI class="toclevel-3"><A href="#Making_Plates"><SPAN class="tocnumber">3.1.2</SPAN><SPAN class="toctext">Making Plates</SPAN></A></LI><LI class="toclevel-2"><A href="#Reactivation_of_freeze-dried_bacteria"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Reactivation of freeze-dried bacteria</SPAN></A></LI><LI class="toclevel-2"><A href="#Inoculation_of_agar_plates"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Inoculation of agar plates</SPAN></A></LI><LI class="toclevel-2"><A href="#Gold-precipitation_on_agar_plates"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Gold-precipitation on agar plates</SPAN></A></LI><LI class="toclevel-2"><A href="#Delftibactin_purification"><SPAN class="tocnumber">3.5</SPAN><SPAN class="toctext">Delftibactin purification</SPAN></A></LI><LI class="toclevel-1"><A href="#DNA_Storage"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">DNA Storage</SPAN></A></LI><LI class="toclevel-2"><A href="#Plasmid_Shipping"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Plasmid Shipping</SPAN></A></LI><LI class="toclevel-3"><A href="#Sending"><SPAN class="tocnumber">4.1.1</SPAN><SPAN class="toctext">Sending</SPAN></A></LI><LI class="toclevel-3"><A href="#Receiving"><SPAN class="tocnumber">4.1.2</SPAN><SPAN class="toctext">Receiving</SPAN></A></LI><LI class="toclevel-1"><A href="#Electrocompetent_Bacteria"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Electrocompetent Bacteria</SPAN></A></LI><LI class="toclevel-2"><A href="#Materials"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Materials</SPAN></A></LI><LI class="toclevel-2"><A href="#Day_1"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Day 1</SPAN></A></LI><LI class="toclevel-2"><A href="#Day_2"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Day 2</SPAN></A></LI><LI class="toclevel-1"><A href="#Electroporation_of_E.coli_DH10.CE.B2"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Electroporation of E.coli DH10β</SPAN></A></LI><LI class="toclevel-2"><A href="#Preparation:"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Preparation:</SPAN></A></LI><LI class="toclevel-2"><A href="#Procedure"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Procedure</SPAN></A></LI><LI class="toclevel-2"><A href="#plate"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">plate</SPAN></A></LI><LI class="toclevel-2"><A href="#Additionally"><SPAN class="tocnumber">6.4</SPAN><SPAN class="toctext">Additionally</SPAN></A></LI><LI class="toclevel-1"><A href="#Heat_shock_competent_bacteria"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Heat shock competent bacteria</SPAN></A></LI><LI class="toclevel-2"><A href="#Materials_2"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Materials</SPAN></A></LI><LI class="toclevel-2"><A href="#Day_1_2"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">Day 1</SPAN></A></LI><LI class="toclevel-2"><A href="#Day_2_2"><SPAN class="tocnumber">7.3</SPAN><SPAN class="toctext">Day 2</SPAN></A></LI><LI class="toclevel-1"><A href="#Isopropanol_PCR_Purification"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Isopropanol PCR Purification</SPAN></A></LI><LI class="toclevel-1"><A href="#PCR"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-2"><A href="#validation_PCR"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">validation PCR</SPAN></A></LI><LI class="toclevel-1"><A href="#Preparing_Glycerol_Stock"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Preparing Glycerol Stock</SPAN></A></LI><LI class="toclevel-1"><A href="#Software"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Software</SPAN></A></LI><LI class="toclevel-2"><A href="#Main_packages"><SPAN class="tocnumber">11.1</SPAN><SPAN class="toctext">Main packages</SPAN></A></LI><LI class="toclevel-3"><A href="#Packages.2C_plugins_or_libraries_loaded_by_some_of_our_python_functions"><SPAN class="tocnumber">11.1.1</SPAN><SPAN class="toctext">Packages, plugins or libraries loaded by some of our python functions</SPAN></A></LI><LI class="toclevel-3"><A href="#Django_packages"><SPAN class="tocnumber">11.1.2</SPAN><SPAN class="toctext">Django packages</SPAN></A></LI><LI class="toclevel-2"><A href="#C.2B.2B"><SPAN class="tocnumber">11.2</SPAN><SPAN class="toctext">C++</SPAN></A></LI><LI class="toclevel-2"><A href="#Javascript"><SPAN class="tocnumber">11.3</SPAN><SPAN class="toctext">Javascript</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="col-md-6"><DIV class="jumbotron"><DIV class="container"><H2><SPAN class="mw-headline" id="Cloning"> Cloning </SPAN></H2><H3><SPAN class="mw-headline" id="Colony_PCR">Colony PCR</SPAN></H3><P>Colonies were picked with pipet tips and dipped into the PCR mixture.
</P><TABLE class="wikitable"><TBODY><TR><TH> Ingredient </TH><TH> amount [µl]
</TH></TR><TR><TD> Overnight Culture </TD><TD> Top of a pipet tip
</TD></TR><TR><TD> 10x PCR buffer </TD><TD> 2.0
</TD></TR><TR><TD> MgCl2 (25 mM) </TD><TD> 2.0
</TD></TR><TR><TD> Forward primer (10µM) </TD><TD> 0.2
</TD></TR><TR><TD> Reverse primer (10µM) </TD><TD> 0.2
</TD></TR><TR><TD> dNTPs (10 mM) </TD><TD> 1.0
</TD></TR><TR><TD> Polymerase </TD><TD> 0.5
</TD></TR><TR><TD> Sterile water </TD><TD> ad 10.0
</TD></TR><TR><TD><B>Total</B></TD><TD><B>10.0</B></TD></TR></TBODY></TABLE><P>The following PCR program was adjusted according to annealing conditions.
</P><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [min:s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 2:00
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 98 </TD><TD> 0:05
</TD></TR><TR><TD> 68 (annealing) </TD><TD> 0:15
</TD></TR><TR><TD> 72 </TD><TD> 1:00 per kbp
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 10:00
</TD></TR><TR><TD> 1  </TD><TD> 12 </TD><TD> inf
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Agarose_gel_electrophoresis">Agarose gel electrophoresis</SPAN></H3><DIV style="font-size: 14px; text-align: justify;">
To separate DNA fragments of different sizes, agarose flat-bed gels were casted in various concentrations (0.6% to 2% agarose in 0.5x TAE buffer). By applying an electrical field (10 to 20 V/cm), these gels were used for analytical or preparative separation. Therefore, the desired amount of agarose was boiled up in 0.5x TAE buffer until it was completely dissolved. After the solution cooled down to 50 to 60°C, it was poured into a flat-bed tray and ethidium bromide (EtBr) solution (0.5 µg/ml final concentration) was added. Finally, a comb was inserted to create pockets while solidification. The gel was put into the running buffer (0.5x TAE buffer) before the DNA mixture with loading buffer was loaded into the pockets and separated by electrophoresis. As ethidium bromide intercalates with the DNA (GC nucleotides) the resulting DNA-EtBr-complexes are visible under UV irradiation at 265 nm. As standard ladder for fragment sizes, 1kb+ ruler, 2-log ladder or 50 bp ladder were used.</DIV><H3><SPAN class="mw-headline" id="Gibson_assembly"> Gibson assembly </SPAN></H3><UL><LI><A href="/wiki/images/3/3a/Gibson_for_Heidelberg.xls" class="internal" title="Gibson for Heidelberg.xls">Excel template</A> kindly provided by the <A href="https://2013.igem.org/Team:Freiburg/Project/Coop" class="external text" rel="nofollow">iGEM Team Freiburg 2013</A></LI></UL><P>
Gibson assembly was prepared according to calculation (see file above). The mixture was incubated at 50°C for 60 minutes and purified by isopropanol precipitation. The Gibson constructs were then transformed into competent bacteria.
</P><H3><SPAN class="mw-headline" id="Biobrick_assembly"> Biobrick assembly</SPAN></H3><UL><LI><A href="http://ginkgobioworks.com/support/BioBrick_Assembly_Manual.pdf" class="external text" rel="nofollow">BioBrick_Assembly_Manual.pdf</A></LI></UL><H3><SPAN class="mw-headline" id="Restriction_digests"> Restriction digests </SPAN></H3><P>For enzymatic digests, the following formula was used if bovine serum albumin (BSA) was not part of the buffer, which was the case for NEB buffer 3.0:
</P><UL><LI> ... µl DNA (~500 ng)
</LI><LI> 5 µl BSA 
</LI><LI> 5 µl buffer (for example NEBuffer 2)
</LI><LI> 2 x 1.5 µl enzymes (for example EcoRI-HF &amp; PstI)
</LI><LI> ... µl water 
</LI></UL><P>=&gt; 50 µl in total
</P><P>The mixture was incubated at 37°C for 1h or up to 1:30h and analyzed by gel electrophoresis.

In case BSA was already part of the buffer, the following recipe was used:
</P><UL><LI> ... µl DNA (~500 ng)
</LI><LI> 2 µl buffer (e.g. NEB CutSmart buffer)
</LI><LI> 1 µl restriction enzymes
</LI><LI> water ad 20 µl
</LI></UL><P>The mixture was incubated at 37°C for 1h or up to 1:30h and analyzed by gel electrophoresis.
</P><H3><SPAN class="mw-headline" id="Extraction_of_DNA_fragments_from_an_agarose_gel">Extraction of DNA fragments from an agarose gel</SPAN></H3><DIV style="font-size: 14px; text-align: justify;">After gel electrophoresis, the gel parts containing the desired DNA fragement size were excised from the agarose gel with a clean and sharp scalpel. For DNA extraction the QIAquick® Gel Extraction Kit was used. Following the instrucion manual all necessary solutions have been prepared. The slices were weighted and accordingly Buffer QG added to the gel (for 1 slice 600 μl; for 2 slices 900 μl). The tubes with the mixture were incubated at 55°C until the gel slices had completely dissolved. In between samples were vortexed, if fragement sizes were not too large. Since the color of the solution never changed, no pH adjustments had to be done. In the next step isopropanol was added (for 1 slice 200 μl; for 2 slices 300 μl) to the sample and mixed. For each sample a QIAquick spin column was placed into the vacuum manifold and the sample decanted to it to bind DNA. Vacuum was applied until the whole sample has been drawn through the column. For further purification samples were washed with 500 μl buffer QG and vacuum was applied. Afterwards 750 μl Buffer PE were added to the samples, let stand for 2-5 minutes and vacuum applied. To remove residual wash buffer, the columns were placed into collection tubes and centrifuged for 1 min at 17,900 x g (13,000 rpm). Finally, the columns were put into clean 1.5 ml microcentrifuge tubes and 20-30 μl water (55°C) were added to the center of the column. After letting the column stand for 1 minute it was centrifuged for 1 minute to collect the elute.</DIV><DIV style="font-size: 14px; text-align: justify;">For larger DNA fragments, the QIAEX II® Gel Extraction Kit was used (suitable for 40 bp to 50 kbp). Solutions were prepared as advised in the instruction protocol. DNA bands were excised from the agarose gel with a clean and sharp scalpel and transfered to a microcentrifuge tube. After weighing the slices, buffer QX1 was added according to the DNA fragment size (6 volumes for &lt;100 bp; 3 volumes for 100 bp – 4 kb; 3 volumes with 2 volumes of water for &gt;4 kb). QIAEX II was vortexed for 30 seconds to resuspend the agent and added to the samples (10 μl QIAEXII for ≤2 μg DNA; 30 μl for 2–10 μg DNA; and an additional 30 μl for each additional 10 μg DNA). Subsequently the solution was mixed and incubated at 50°C for 10 min to solubilize the agarose and bind the DNA. Since we mainly used this kit for larger fragments, the resuspended pellet was only inverted and flicked to avoid shearing forces. We did not observe changes in color, that is why no pH adjustment were done. Afterwards, samples were centrifuged for 30 seconds and supernatant carefully removed with a pipet. The pellet was washed with 500 μl buffer QX1 to remove residual agarose contaminants and resuspended by inverting and flicking. Then samples were centrifuged for 30 seconds and all supernatant residues removed with a pipet. The pellets were washed twice with 500 μl buffer PE to remove salt contaminants and resuspended by inverting and flicking. Then samples were centrifuged for 30 seconds and supernatant removed with a pipet. Pellets were air-dried for 10-15 minutes until they became white. To elute the DNA, 20 μl of water were added and pellet resuspended by flicking. After incubation according to the DNA fragment size (5 minutes at room temperature (15–25°C) for ≤4 kb; 5 minutes at 50°C for 4–10 kb; 10 minutes at 50°C for &gt;10 kb), samples were centrifuged for 30 seconds. Supernatant, which contained the purified DNA, was pipetted into a clean tube.</DIV><H3><SPAN class="mw-headline" id="Ligation"> Ligation </SPAN></H3><P>For ligation of DNA fragments, the mixture was prepared as follows:
</P><UL><LI> ... µl DNA (=&lt;200ng in total-&gt; 20-70ng Backbone)
</LI><LI> 2 µl buffer 
</LI><LI> 1 µl ligase (T4)
</LI><LI> ... µl water 
</LI></UL><PRE>=&gt; 20 µl in total
</PRE><P>After an incubation for 2 minutes at room temperature, or for 1h or over night at 4°C, the product was loaded on an gel for electrophoretic separation of the DNA fragments.
</P><H3><SPAN class="mw-headline" id="Nucleotide_Removal">Nucleotide Removal</SPAN></H3><DIV style="font-size: 14px; text-align: justify;">
For removal of excess nucleotides after enzymatic reactions, the QIAquick® Nucleotide Removal Kit (QIAGEN) was used. All solutions were prepared as recommended by the manufacturer. Ten volumes of Buffer PNI were added to 1 volume of the sample and mixed. The QIAquick spin column was placed in a provided 2 ml collection tube and sample decanted to it. The tube was centrifuged at 6000 rpm for 1 minute. Flow-through was discarded and the column placed back into the collection tube. Afterwards, 750 μl Buffer PE were added, incubated for 2-3 minutes and centrifuged for 1 min at 3800 x g (6000 rpm). Flow-through was discarded and the column placed back into the collection tube. The column was centrifuged for an additional 1 min at 17,900 x g (13,000 rpm) to remove residual buffer. Finally, the column was put into a clean 1.5 ml microcentrifuge tube and 20-30 μl water (55°C) was added to the center. After 1 minute incubation the sample was centrifuged for 1 minute at 17,900 x g (13,000 rpm). The 1.5 ml tube now contained the eluted and purified DNA.</DIV><H3><SPAN class="mw-headline" id="Purification_of_PCR_products">Purification of PCR products</SPAN></H3><DIV style="font-size: 14px; text-align: justify;">To purify PCR products the QIAquick® PCR Purification Kit (QIAGEN) was used. Therefore, all solutions were prepared as recommended by the manufacturer. Five volumes of buffer PB were added to 1 volume of the PCR reaction before mixing. Since no changes in color were observed the pH was not adjusted. A QIAquick column was put into a vacuum manifold and the sample was decanted to it for binding the DNA. Vacuum was applied until all liquid residues were drawn through the column. Afterwards 750 μl buffer PE were added to wash the column and vacuum applied. Te remove residual wash buffer the QIAquick column was put into the provided 2 ml collection tube and centrifuged for 1 min at 17,900 x g (13,000 rpm). Finally, the column was put into a clean 1.5 ml microcentrifuge tube and 20 ml water (55°C) were added to the center to elute the DNA. After 1 minute incubation the tube containing the column was centrifuged for 1 minute. The microcentrifuge tube now contained the eluted DNA.</DIV><DIV style="font-size: 14px; text-align: justify;">For small amounts of DNA PCR products (cleanup of up to 5 μg PCR product (70 bp to 4 kb)) the MinElute® PCR Purification Kit (QIAGEN) was used. All steps were conducted and solutions prepared as specified in the protocol.
Five volumes of buffer PB were added to 1 volume of the PCR reaction and mixed. Since no changes in color were observed the pH was not adjusted. A MinElute column was put into a vacuum manifold and the sample was decanted to it for binding the DNA. Vacuum was applied until all liquid residues were drawn through the column. Afterwards 750 μl buffer PE were added to wash the column and vacuum applied. Te remove residual wash buffer the QIAquick column was put into the provided 2 ml collection tube and centrifuged for 1 min at 17,900 x g (13,000 rpm). The MinElute column was placed into a clean 1.5 ml microcentrifuge tube and 20 ml water (55°C) were added toe the center to elute the DNA. After 1 minute incubation the tube containing the column was centrifuged for 1 minute. The microcentrifuge tube now contained the eluted DNA.</DIV><H3><SPAN class="mw-headline" id="Plasmid-DNA_isolation">Plasmid-DNA isolation</SPAN></H3><H4><SPAN class="mw-headline" id="Miniprep">Miniprep</SPAN></H4><DIV style="font-size: 14px; text-align: justify;">
Plasmids were isolated by the use of the QIAprep® Spin Miniprep Kit. According to the instructions included, all necessary solutions were prepared and steps conducted. First, overnight cultures (1-2 ml) have been pelleted by centrifugation at &gt;8000 rpm</DIV><DIV style="font-size: 14px; text-align: justify;">
(6800 x g) for 3 min at room temperature (15–25°C). Supernatant was discarded and pellet resuspended in 250 μl (procedure 2: 200 μl) buffer P1 in a microcentrifuge tube. Afterwards 250 μl (procedure 2: 400 μl) Buffer P2 were added and the tube inverted 4-6 times to mix the solution until it became clear. The sample was lysed for 2-3 minutes before adding 350 μl (procedure 2: 300 μl) buffer N3 and inverting the whole solution 4-6 times. Following the neutralization, the sample was centrifuged for 10 minutes at 3,000 rpm (~17,900 x g) in a table-top microcentrifuge. Supernatant was decanted to a QIAprep spin column and vacuum applied to the manifold until the solution was drawn through the column. To wash the column 500 μl Buffer PB were added and again vacuum applied. In addition the QIAprep spin column was washed by adding 750 μl Buffer PE and applying vacuum. To remove residual wash Buffer the column was transfered to a collection tube and centrifuged for 1 min. Finally, the column was put into a clean 1.5 ml microcentrifuge tube and 20-30 μl water (heated up to 55°C) added to the center of the column. After 1 minute the tube containing the column was centrifuged for 1 minute to collect eluted DNA. </DIV><H4><SPAN class="mw-headline" id="Midiprep">Midiprep</SPAN></H4><DIV style="font-size: 14px; text-align: justify;">
For larger amounts of bacterial cultures the QIAGEN® Plasmid Plus Midi Kit was used to isolate plasmids. According to the instructions included, all necessary solutions were prepared and 25 ml bacterial culture harvested after 12-16 hours incubation. The sample was centrifuged at 6000 x g for 15 min at 4°C and pelleted cells were completely resuspended in 2 ml Buffer P1. Then, 2 ml buffer P2 were added and mixed thoroughly by inverting until the lysate appeared viscous. The solution was incubated at room temperature (15–25°C) for 3 min. The QIAfilter Cartridge was placed into a new tube. Afterwards, 2 ml buffer S3 were added to the lysate and mixed by inverting 4-6 times. The lysate was transfered to the cartridge and incubated at room temperature for 10 minutes. During incubation the QIAGEN Plasmid Plus spin column was put into the vacuum manifold and tube extenders inserted into each column. The plunger was inserted gently into the QIAfilter Cartridge and the lysate filtered into the tube. Thereafter, 2 ml buffer BB were added to the cleared lysate and mixed by inverting 4-6 times. Lysate was then transfered to a QIAGEN Plasmid Plus spin column on the vacuum manifold. Vacuum was applied until all liquid remains were drawn through the column. To wash the DNA 700 μl buffer ETR was added and vacuum applied. Residual wash buffer was removed by centrifugation of the column at 10,000 x g (9,700 rpm) for 1 min in a tabletop microcentrifuge. The column was put into a clean 1.5 ml tube. Finally, 200 μl water (55°C) was added to the center of the spin column. The column was let stand for 1 minute and centrifuged for 1 minute to collect the eluted DNA.</DIV><H4><SPAN class="mw-headline" id="Preparation_of_large_plasmids">Preparation of large plasmids</SPAN></H4><DIV style="font-size: 14px; text-align: justify;">
For isolation of plasmids larger than 15 kb in size, 7 ml bacterial culture were harvested after 12-16 hours of growth in appropriate medium. Samples were centrifuged at 6000 xg for 20 min at 4°C. Supernatant was discarded and cell pellet resuspended in 200 µl of Buffer P1 (QIAGEN). Afterwards, 400 µl of Buffer P2 were added, mixed thoroughly by inverting until the lysate appeared viscous. Solution was incubated at room temperature (15-25°C) for 2 min. Afterwars 300 µl of Buffer P3 were added to stop lysis. Cell debris and Proteins were pelleted by centtrifugation at ~ 17,900 x g for 30 min at 4°C. Supernatant was carefully transferred into a fresh 2ml eppendorf centrifuge tube and isopropanol purification of DNA was conducted.</DIV><H3><SPAN class="mw-headline" id="Sequencing">Sequencing</SPAN></H3><P>Samples were sent to <A href="http://www.gatc-biotech.com/en/index.html" class="external text" rel="nofollow">GATC Biotech</A> for sequencing.
</P><H2><SPAN class="mw-headline" id="Protein_detection">Protein detection</SPAN></H2><H3><SPAN class="mw-headline" id="Thin_Layer_Chromatography">Thin Layer Chromatography</SPAN></H3><DIV style="font-size: 14px; text-align: justify;">TLC was carried out on silica-gel as immobile phase and Dichloromethane as mobile phase. For the procedure a 50 ml beaker was filled with ~15 ml Dichloromethane and let stand for about 10 minutes (in order to let the Dichloromethane-vapor fill the beaker). The TLC plate, coated with silica-gel was spotted with sample 0.5 to 1 cm above the lower edge and placed in the beaker. TLC was run until the solvent front was at two thirds of the TLC plate. As Indigoidine is light-sensitive, the beaker was covered with aluminium foil in order to prevent direct light irradiation.</DIV><H3><SPAN class="mw-headline" id="Purification_of_Indigoidine_and_tagged_constructs">Purification of Indigoidine and tagged constructs</SPAN></H3><DIV style="font-size: 14px; text-align: justify;">1 ml of IPTG-induced, blue culture was spun down at full speed (14,000 rpm) for 20 minutes, washed in 1 ml of methanol and centrifuged once more for 5 minutes at 14,000 rpm. Methanol was discarded and samples were dissolved in 200-400 µl DMSO.</DIV><H3><SPAN class="mw-headline" id="Purification_of_Delftibactin">Purification of Delftibactin</SPAN></H3><UL><LI> Centrifugation of cultures at 3750 rpm for 30 min
</LI><LI> Retrieve supernatant
</LI><LI> Add 20 g/L HP-20
</LI><LI> Stirr at RT for 2 h
</LI><LI> Filtration with Buchner funnel to retain HP-20
</LI><LI> Wash with 400 ml ddH<SUB>2</SUB>O 
</LI><LI> Elution with 400 ml Methanol
</LI><LI> Evaporated methanol with rotary evaporator
</LI><LI> Resuspended in 2 ml 50 methanol : 50 ddH<SUB>2</SUB>O
</LI></UL><H3><SPAN class="mw-headline" id="Mass_spectrometry">Mass spectrometry</SPAN></H3><H4><SPAN class="mw-headline" id="Cell_preparation">Cell preparation</SPAN></H4><DIV style="font-size: 14px; text-align: justify;">For analysis by mass spectrometry, 3 ml of LB media containing chloramphenicol (1 micromolar) was inoculated with transformed bacterial cells (expression of the NRPS tested before by SDS-PAGE). Cultures were incubated at 37°C overnight until OD<SUB>600</SUB> ~ 0.6. After centrifugation for 30 minutes at 3750 rpm and 4°C, supernatant was discarded and pellet washed with M9 minimal media. Again samples were centrifuged and resuspended in 50 ml M9 minimal media. Following an incubation time of 12 hours (OD600=0.6) cells were induced with IPTG (1 mM).</DIV><DIV style="font-size: 14px; text-align: justify;">To assess product levels over time, 7.5 ml samples were taken at several time points (8.5 hrs, 12 hrs, 24 hrs and 48 hrs). Each sample was spun down for 45 minutes at 3750 rpm and 4°C, and supernatant was separated from pellet.</DIV><UL><LI><DIV style="font-size: 14px; text-align: justify;">Pellet was resolubilzed in 500 µl 1x PBS and lysed by ultra-sonification (3x20 seconds; on ice). Finally the sample has been centrifuged and supernatant decanted to 1.5 ml tubes. Frozen with nitrogen the sample was handed over for mass spectrometry.</DIV></LI></UL><UL><LI><DIV style="font-size: 14px; text-align: justify;">Supernatant was frozen with nitrogen and lyophilized for 12 hours ( at ~-45°C and ~0,1 mbar) to remove liquid residues. The lyophilisate was resuspended in 500 microliters methanol and vortexed. Supernatant was transferred to a 1.5 ml tube, which was frozen in liquid nitrogen. The sample was then handed over to mass spectrometry for further analysis.</DIV></LI></UL><H4><SPAN class="mw-headline" id="Data_Acquisition">Data Acquisition</SPAN></H4><P><B>Sampling</B></P><P>Two empty 3.2 mm die-cut sheets were prepared from Whatman 903 filter paper. Afterwards, 5 µl of the sample were pipetted on the sheets and dried in the fridge overnight.
</P><P><B>Sample preparation</B></P><DIV style="font-size: 14px; text-align: justify;">The dried sheets were extracted for ca. 20 min with 100 μl solution 1. Then the polypropylen microtiter plates were centrifuged for ca. 2 minutes at 4000 rpm. Supernatant was vaporized at 55°C for 30 minutes at the blowing station until all liquid residues were removed. Subsequently the plate was warmed up for 1 minute at a cabinet dryer (65°C).
<P>After drying 60 μl of solution 2 were added and the mircotiter plate tightly closed with micromates. The plate was shaken gently and heated up again for ca. 15 minutes at the cabinet dryer (65°C).
Micromates were drawn off and supernatant vaporated on heating plates (55°C surface temperature) with a constant air stream. Again no liquid residues should be remaining. Samples were heated up for 1 minute in the cabinet dryer (65°C).
</P>
Finally 100 μl solution 3 were added (it is possible to dilute the sample up to factor 3 with solution 3). To avoid evaporation losses, the microtiter plate was covered with aluminium foil.</DIV><H3><SPAN class="mw-headline" id="SDS-PAGE">SDS-PAGE</SPAN></H3><DIV style="font-size: 14px; text-align: justify;">1 ml culture was pipetted into a 1.5 ml centrifuge tube. This was centrifuged at 13,000 rpm for 10 minutes and supernatant was discarded. The pellet was resuspended in 100 µl loading buffer. The samples were boiled with closed lids at 98°C for at least 10 minutes under the hood, after which they were let cool down and centrifuged at 13,000 rpm for another 10 minutes. 25µl were loaded in a 3-8% Tris-Acetate Gradient Gel from Invitrogen, 10 µl NOVEX Pre-Stained Standard was used and PAGE was let run for 1 hour at 180 V with Tris-Acetate running buffer from Invitrogen.
The aforementioned loading buffer is composed of 800 µl ddH20, 250 µl 0.5M Tris-HCl (pH = ~7.3), 400 µl Glycerol (50%), 400 µl SDS (10%), 100µl β-Mercaptoethanol (99%) and bromophenol-blue to obtain 2ml of loading buffer.</DIV><H2><SPAN class="mw-headline" id="1st_Pre-Experiment">1st Pre-Experiment</SPAN></H2><DIV style="font-size: 14px; text-align: justify;">The first step is the establishment of the reaction of delftibactin with soluble gold. Therefore we plan an experiment (one to two weeks), where we reproduce the results of the paper [&lt;bib id=&quot;pmid23377039&quot;/&gt;]. After mixing the Acidovorax complex medium (ACM) and adding 4g Chelex-100 resin / 1L ACM  we prepare 25 agarplates (0,5 L ) and two media for liquid culture (1L +0,5L). The second day is about the resuspending of the strain D. acidovorans in 5 ml [recepy in methods] and letting it grow over night. The next day 5 agarplates are inoculated with D. Acidovorans by fractional streak and 5 agarplates with streak over half of the plate. In addition to these 10 agarplates, also the two flasks with the liquid media are inoculated with the strain. The rest of the resuspended cells is stored in a glycerol stock. After 3 days of culture growth (shaking at 190rpm, at a temperature of 30°C) the cells are big enough to start with the delftibactin purification (liquid culture) and the fast test on the agarplates. For the fast test on the agarplates we prepare an soft agarose solution (0,5% agarose and 10mM AuCl3) and incubate every plate with 10 ml solution for two hours at rooms temperature. If delftibactin is present there can be observed a ring structure around the cells. </DIV><DIV style="font-size: 14px; text-align: justify;">For the purification of delftibactin, the liquid culture will be centrifugated at  7000 rpm for 15 min. The supernatant will be treated with HP20 resin (20 g/L) and then shaken 2h at 220rpm. With the Buchner funnel filtration the resin will be collected and washed with 400 ml dH2O. Afterwards the resin is again washed with 400 ml methanol which will be evaporate under rotary vacuum. The leftover is resuspended in 2 ml of ddH2O:MeOH (1:1). The solution is added to a HPLC column and the HPLC (high performance liquid chromatography) is analysed in a mass spectroscopy facility. The mobile phase consists in 2%-14% acetonitrile and  98% water +5 mM (NH4)2CO3. 
<P>After the purification we plan to mix the purified delftibactin with different gold solution (AuCl3) concentrations (between 5mM and 20mM). After a 2h incubation a precipitation would be observed. 
</P>
In case the pre-experiment works, we have proven the reliability of the paper and the expected read out of the assay.</DIV><H3><SPAN class="mw-headline" id="Making_ACM_Medium"> Making ACM Medium </SPAN></H3><H4><SPAN class="mw-headline" id="for_1_L"> for 1 L </SPAN></H4><UL><LI> 0.5 g yeast extract (Difco-BectonDickinson) 
</LI><LI> 1.0 g Cas amino acids (Difco)
</LI><LI> 2.0 g pyruvic acid
</LI><LI> 2.0 g L-glutamine
</LI><LI> 0.3 g KH<SUB>2</SUB>PO<SUB>4</SUB></LI><LI> 0.3 g MgSO<SUB>4</SUB></LI><LI> 2.0 g MOPS 
</LI><LI> 4.0 g Chelex 100-resin (Sigma)
</LI><LI> pH adjusted to 7.2–7.3 with 5 M KOH
</LI></UL><P>Important Notes:
</P><UL><LI> fill up to 900mL before adding pyruvic acid and L-glutamine
</LI><LI> adjust pH
</LI><LI> fill up to 1L
</LI><LI> treat with Chelex for 1h
</LI><LI> remove Chelex by filtration
</LI></UL><H4><SPAN class="mw-headline" id="Making_Plates">Making Plates</SPAN></H4><UL><LI> prepare 2 l medium
</LI><LI> make 25 agar plates
</LI><LI> autoclave
</LI></UL><H3><SPAN class="mw-headline" id="Reactivation_of_freeze-dried_bacteria"> Reactivation of freeze-dried bacteria </SPAN></H3><UL><LI> prepare 10 ml of liquid-activation-media with:
</LI><LI> 50 mg peptone
</LI><LI> 30 mg meat extract
</LI><LI> 150 mg agar
</LI><LI> 10 ml destilled water
</LI><LI> adjust pH to 7.0
</LI></UL><LI> rehydrate and grow cells in 5 ml of liquid broth
</LI><LI> grow at 30° in incubator over night
</LI><H3><SPAN class="mw-headline" id="Inoculation_of_agar_plates"> Inoculation of agar plates </SPAN></H3><UL><LI> inoculate 5 agar plates with inoculation loop
</LI><LI> grow at 30° in incubator for 3 days
</LI></UL><H3><SPAN class="mw-headline" id="Gold-precipitation_on_agar_plates"> Gold-precipitation on agar plates </SPAN></H3><UL><LI> prepare 50 ml of 0.5 % Agarose solution which contains 10 mM AuCl3
</LI><LI> pour 10 ml of the prepared agarose solution onto each of 3 agar plates
</LI><LI> recepy for the gold solution <A href="/wiki/images/c/c9/Recepy_for_0%2C5%25_agarose_solution_with_10mM_AuCl3.pdf" class="internal" title="Recepy for 0,5% agarose solution with 10mM AuCl3.pdf">media:Recepy_for_0,5%_agarose_solution_with_10mM_AuCl3.pdf</A></LI></UL><H3><SPAN class="mw-headline" id="Delftibactin_purification"> Delftibactin purification </SPAN></H3><UL><LI> centrifuge liquid culture at 7000 r.p.m. 
</LI><LI> keep supernatant, remove cell mass
</LI><LI> treat supernatant with with 20 g/l HP20
</LI><LI> shake HP20 with supernatant for 1 h at 220 r.p.m.
</LI><LI> remove aqueous phase by buchner vacuum filtration
</LI><LI> elute dehydrated material with 400 ml methanol
</LI><LI> remove HP20 by buchner vacuum filtration
</LI><LI> dry solution with rotary vacuum evaporator
</LI><LI> resuspend material in 2 ml 50 % water and methanol
</LI><LI> liquid chromatography
</LI></UL><H2><SPAN class="mw-headline" id="DNA_Storage">DNA Storage</SPAN></H2><H3><SPAN class="mw-headline" id="Plasmid_Shipping"> Plasmid Shipping </SPAN></H3><UL><LI> uses Whatman Filter Paper
</LI><LI> used the protocol of: <A href="https://labs.mcdb.ucsb.edu/weimbs/thomas/links/methods/plasmid_shipping.html" class="external free" rel="nofollow">https://labs.mcdb.ucsb.edu/weimbs/thomas/links/methods/plasmid_shipping.html</A></LI></UL><H4><SPAN class="mw-headline" id="Sending"> Sending </SPAN></H4><OL><LI> Mark a circled area with a pencil (not a marker pen) on a clean Whatman #1 filter paper (or equivalent).
</LI><LI> Spot about 2 µg of plasmid DNA into the circle. Allow the filter paper dry at room temperature.
</LI><LI> Insert spotted filter paper inside a plastic bag and heat-seal it.
</LI><LI> Send by regular (air) mail.
</LI></OL><H4><SPAN class="mw-headline" id="Receiving"> Receiving </SPAN></H4><OL><LI> To recover the DNA, use clean gloves and cut the marked circle area that contains the dried plasmid DNA.
</LI><LI> Using clean forceps, insert the filter paper into a 1.5 ml micro centrifuge tube. Add 100 µl of TE buffer, vortex briefly and incubate at room temperature for 5 minutes. Vortex again and centrifuge the tube for a few seconds.
</LI><LI> Remove about 10 µl of supernatant for use in transfecting E. coli by electroporation or chemical means. Please do not try to use the DNA directly for any application other than to transform bacteria and prepare a plasmid stock.
</LI><LI> Store the remainder of the filter paper/TE mix at -20 or -80 C as a permanent archive in case that your plasmid stock ever gets lost or if something turns out to be wrong with it.
</LI></OL><H2><SPAN class="mw-headline" id="Electrocompetent_Bacteria">Electrocompetent Bacteria</SPAN></H2><H3><SPAN class="mw-headline" id="Materials"> Materials </SPAN></H3><UL><LI> 10% glycerol
</LI><LI> ddH<SUB>2</SUB>O
</LI></UL><H3><SPAN class="mw-headline" id="Day_1"> Day 1 </SPAN></H3><UL><LI> Inocculate 50 ml of appropriate medium
</LI><LI> Grow ON at appropriate temperature, shaking (e.g. 37°C, 180 rpm)
</LI></UL><H3><SPAN class="mw-headline" id="Day_2"> Day 2 </SPAN></H3><OL><LI> Transfer 10 - 30 ml of the ON culture into 400 ml appropriate medium (measure OD<SUB>600</SUB>, should not exceed 0.1)
</LI><LI> Grow at appropriate temperature, shaking (e.g. 37°C, 180 rpm) until an OD of 0.4-0.6 is reached
</LI><LI> Meanwhile, put H<SUB>2</SUB>O, 10% glycerol on ice, prechill tubes and centrifuge
</LI><LI> Chill cells on ice for 10 min
</LI><LI> Centrifuge for 30 min at 3750 rpm, 4°C<B>work on ice from here on</B></LI><LI> Discard supernatant, resuspend bacteria in 2 ml ice cold H<SUB>2</SUB>O / 50 ml falcon, fill up to 20 ml ice cold H<SUB>2</SUB>O
</LI><LI> Centrifuge for 30 min at 3750 rpm, 4°C
</LI><LI> Discard supernatant, resuspend in 1 ml glycerol / falcon, add up to 5 ml glycerol (falcons may be pooled at this point)
</LI><LI> Centrifuge for 20 min at 3750 rpm, 4°C
</LI><LI> Resuspend in same volume glycerol as pellet size (ca. 100 µl / original 50 ml falcon)
</LI><LI> Aliquot 50 µl per tube
</LI><LI> Optional: shock freeze with liquid nitrogen
</LI><LI> Store aliquots at -80°C
</LI></OL><H2><SPAN class="mw-headline" id="Electroporation_of_E.coli_DH10.CE.B2"> Electroporation of E.coli DH10β </SPAN></H2><H3><SPAN class="mw-headline" id="Preparation:"> Preparation: </SPAN></H3><UL><LI> Source material: 1ml LB media + DH10β overnight culture (12-16 h) at 32°C (1 day before) 
</LI><LI> Preperate 4 eppis  with 1 ml LB media and a) 25µl,  b) 35µł,  c) 45µł,  d)  55µl of source material and incubate it at 32°C (2h)
</LI><LI> Put bacteria on ice for 10 – 15 min (since this step everything is at 0°C – the centrifuge, the water!!!!!!!!)
</LI><LI> Centrifuge 30 s at 13 000 rpm (0°C) and remove via vacuum the medium, DON'T touch the pellet!!!
</LI><LI> Resuspend first pellet in 1 ml H2O bidest comercial (0°C),transfer to next eppi an dos on to collect all four cell-pellets resuspended in water in '<B>ONE eppi</B>.
</LI><LI> Centrifuge and wash again (2x)
</LI></UL><H3><SPAN class="mw-headline" id="Procedure"> Procedure </SPAN></H3><UL><LI> add 50 µl H2O bidest comercial (0°C) and resuspend pellet
</LI><LI> add 2-3 µl of Ligation Reaction and incubate it on ice for 5-10 min
</LI><LI> transfer all of it to the special cuvette (without any air bubbles) – the cuvette has to be perfectly dry on the outside
</LI><LI> press button – electroshock (and from now on QUICK handling)
</LI><LI> add 400 µl SOC-medium and transfer all of it into an eppi
</LI><LI> incubation for 1 h at 37 °C shaking at 240 rpm
</LI></UL><H3><SPAN class="mw-headline" id="plate"> plate </SPAN></H3><UL><LI> plate on one agarplate with the corresponding antibiotic 100 µl of the bacteria
</LI><LI> plate on another agarplate with the corresponding antibiotic 10 µl of the bacteria
</LI></UL><H3><SPAN class="mw-headline" id="Additionally"> Additionally </SPAN></H3><UL><LI> Bio-Rad MicroPulser
</LI><LI> Program 1 for brown cuvettes
</LI><LI> Program 2 for blue cuvettes
</LI></UL><P>=&gt; these two programs are for bacteria
</P><H2><SPAN class="mw-headline" id="Heat_shock_competent_bacteria"> Heat shock competent bacteria </SPAN></H2><H3><SPAN class="mw-headline" id="Materials_2"> Materials </SPAN></H3><UL><LI> 100 mM CaCl<SUB>2</SUB> (aq)
</LI><LI> Glycerol
</LI></UL><H3><SPAN class="mw-headline" id="Day_1_2"> Day 1 </SPAN></H3><P>grow bacteria over night in 50 ml medium at 37°C
</P><H3><SPAN class="mw-headline" id="Day_2_2"> Day 2 </SPAN></H3><OL><LI> transfer 20-30 ml of the ON culture into 400 ml LB
</LI><LI> grow at 37°C until an OD of 0.5-0.6 is reached (3-5 hours), put CaCl<SUB>2</SUB> on ice, prepare aliquoting buffer (20 ml CaCl<SUB>2</SUB> with 10% glycerol)
</LI><LI> centrifuge bacteria for 30 min at 3750 rpm, 4°C<B>work on ice from here on</B>==&gt; you will need 8 falcons (50 ml) in order to do so
</LI><LI> discard supernatant, resuspend bacteria in 5 ml CaCl<SUB>2</SUB> / falcon, pool 2 falcons, add up to 50 ml CaCl<SUB>2</SUB>==&gt; you should have 4 falcons from here on
</LI><LI> leave on ice for 30 min
</LI><LI> centrifuge for 20 min at 3750 rpm, 4°C
</LI><LI> discard supernatant, resuspend in 10 ml aliquoting buffer for all 4 falcons, add another 10 ml aliquoting buffer<B>Do not vortex</B></LI><LI> pipet 100 µl per eppendorf tube (pre-cooled on ice), put aliquots in -80°C freezer
</LI></OL><H2><SPAN class="mw-headline" id="Isopropanol_PCR_Purification"> Isopropanol PCR Purification </SPAN></H2><UL><LI> pool PCR reactions if there are several
</LI><LI> add &gt;= 4 volumes isopropanol, invert tube 10 times
</LI><LI> centrifuge 20 min at full speed (use cooling centrifuge at 20°C)
</LI><LI> decant supernatant
</LI><LI> add 4 volumes 70% ethanol
</LI><LI> centrifuge 5 min at full speed
</LI><LI> decant supernatant, tap eppendorf tube on paper tissue
</LI><LI> let it dry for ca. 5 min
</LI><LI> dry off inside of tube with a piece of paper tissue. <B>Do not touch pellet!</B></LI><LI> resuspend in 130 µl H<SUB>2</SUB>O, add 15 µl CutSmart buffer and 5 µl DpnI (digests methylated DNA -&gt; template)
</LI><LI> incubate 4h at 37°C
</LI><LI> repeat purification, resuspend in 30-40 µl H<SUB>2</SUB>O
</LI></UL><H2><SPAN class="mw-headline" id="PCR"> PCR </SPAN></H2><UL><LI> ... µl DNA (1-10ng)
</LI><LI> 2x 0,5 µl Primer 
</LI><LI> 25 µl Phusion Flash Ready Mix(2x)
</LI><LI> ... µl water 
</LI></UL><P>=&gt; 50 µl in total
</P><UL><LI> fill in PCR tubes 
</LI><LI> PCR amplification takes ca. 2h
</LI></UL><H3><SPAN class="mw-headline" id="validation_PCR"> validation PCR</SPAN></H3><P>( using 2x Taq MasterMix from NEB )
</P><UL><LI> ~ 5 ng template DNA (picked colony, miniPrep...)
</LI><LI> 2x 0.2 µl primer (_fw, _rv)
</LI><LI> 10 µl 2x Taq MM
</LI><LI> ... H_2O (add up to 20 µl)
</LI></UL><P>=&gt; 20 µl in total
</P><UL><LI> fill in PCR tubes 
</LI><LI> PCR amplification takes ca. 2h
</LI></UL><H2><SPAN class="mw-headline" id="Preparing_Glycerol_Stock"> Preparing Glycerol Stock </SPAN></H2><UL><LI> grow liquid culture of the desired bacteria
</LI><LI> pipet 1 ml of liquid culture into cryotube, add 300 µl 50% glycerol
</LI><LI> RT for 1 h
</LI><LI> freeze at -80°C
</LI></UL><H2><SPAN class="mw-headline" id="Software">Software</SPAN></H2><H3><SPAN class="mw-headline" id="Main_packages">Main packages</SPAN></H3><UL><LI><B>Python(2.7)</B>: The high-level programming language <A href="http://www.python.org/" class="external text" rel="nofollow">python</A> was used in combination with Django to develop the web-based interface and client-side applications. 
</LI></UL><H4><SPAN class="mw-headline" id="Packages.2C_plugins_or_libraries_loaded_by_some_of_our_python_functions">Packages, plugins or libraries loaded by some of our python functions</SPAN></H4><UL><LI><B>Requests</B>: The purpose of this library is to do simple http requests.
</LI></UL><UL><LI><B>Biopython (&gt;= 1.61)</B>: Just like bioperl this package contains tools for computational molecular biology.
</LI></UL><UL><LI><B>xhtml2pdf</B>: The converter package xhtml2pdf was applied to generate pdf from HTML/CSS templates.
</LI></UL><UL><LI><B>BeautifulSoup</B>: The BeautifulSoup HTML/XML parser was used for quick-turnaround applications.
</LI></UL><UL><LI><B>celery</B>: Asynchronous scheduling framework.
</LI></UL><UL><LI><B>kombu</B>: Database-based message passing framework.
</LI></UL><UL><LI><B>python-openbabel</B>(&gt;= 2.3.2) : Python bindings to openbabel for 2D (chemical) structure generation.
</LI></UL><UL><LI><B>UNAFold and MFold utils</B>: UNAFold was required by Gibthon for DNA/RNA folding predictions.
</LI></UL><UL><LI><B>MySQL</B>: The relational database management system <A href="http://www.mysql.com/" class="external text" rel="nofollow">MySQL</A> was utilized for database creation and maintenance.
</LI></UL><UL><LI><B>MySQL - Python connector</B>: A conncetor between the programming language python and the database management system.
</LI></UL><UL><LI><B>Django(&gt;= 1.5)</B>: The web application framework <A href="https://www.djangoproject.com/" class="external text" rel="nofollow">Django</A> was used to achieve an easier database connection and is based on python. Since we improved and added basic functions of Gibthon (<A href="https://2010.igem.org/Team:Cambridge" class="external text" rel="nofollow">iGEM Team Cambridge 2010</A>), the integration was easier to handle using the same frame work.
</LI></UL><H4><SPAN class="mw-headline" id="Django_packages">Django packages</SPAN></H4><UL><LI><B>django-registration</B>: Handles user registrations and logins.
</LI></UL><UL><LI><B>django-annoying</B>: Gibthon uses this package. It is thought to eliminate annoying things in Django.
</LI></UL><UL><LI><B>south</B>: The south package handles database migration (database independent).
</LI></UL><UL><LI><B>django-celery</B>: The package was used for the integration of Celery (asynchronous task queue/job queue) into Django.
</LI></UL><H3><SPAN class="mw-headline" id="C.2B.2B"> C++ </SPAN></H3><UL><LI><B>CMake</B>: Cross platform build system used by the NRPSDesigner.
</LI></UL><UL><LI><B>MySQL C++ connector</B>: C++ library for MySQL access.
</LI></UL><UL><LI><B>libXML</B>: C library for XML handling.
</LI></UL><UL><LI><B>libcurl</B>: C library for client-side URL transfers.
</LI></UL><UL><LI><B>Boost.program_options</B>: C++ library for command-line option parsing.
</LI></UL><UL><LI><B>GCC (&gt;= 4.8)</B> or a comparable compiler with C++11 support.
</LI></UL><H3><SPAN class="mw-headline" id="Javascript">Javascript</SPAN></H3><UL><LI><B>Intro.js</B>: This Javascript has been adopted for the guided step-by-step tour on the NRPS Designer homepage.
</LI></UL><UL><LI><B>select2.js</B>: The Select2 library is a jQuery based library for the replacements of select boxes, which support different connections to other functions.

                                    </LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Heidelberg/Project/Methods">http://2013.igem.org/Team:Heidelberg/Project/Methods</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Project/Methods" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Project/Methods" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Project/Methods&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Project/Methods&amp;oldid=313903" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>