<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Templates_Delftibactin_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Templates/Delftibactin_Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Templates/Delftibactin_Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Templates/Delftibactin_Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Templates/Delftibactin_Results&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Templates/Delftibactin_Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Templates/Delftibactin Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Efficient_Recycling_of_Gold_from_Electronic_Waste_using_Delftibactin"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Efficient Recycling of Gold from Electronic Waste using Delftibactin</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Amplification_of_the_Del_Cluster_Genes"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Amplification of the Del Cluster Genes</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Gibson_Assembly_.26_Transformation"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Gibson Assembly &amp; Transformation</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Expression_of_the_Delftibactin_NRPSs_.26_Associated_Genes"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Expression of the Delftibactin NRPSs &amp; Associated Genes</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Efficient_Recycling_of_Gold_from_Electronic_Waste_using_Delftibactin">Efficient Recycling of Gold from Electronic Waste using Delftibactin</SPAN></H3><P>As first, important step in direction of an environmental friendly procedure for recycling gold from gold-containing waste, we wanted to show that the non-ribosomal peptide delftibactin can be used to precipitate gold from gold ion-containing solutions.
</P><P>We obtained <I>D. acidovorans</I> DSM-39 from the DSMZ and successfully reproduced the paper by Johnsson <I>et al.</I> [5]. In our experiments, precipitation on agar plates worked even better than described in the paper as shown in Figure 1. <I>D. acidovorans</I> is capable to precipitate solid gold from gold chloride solution as purple-black nanoparticles. Already at low concentrations of gold chloride, gold nonaparticles are precipitated increasing with concentration of gold chloride in solution (Fig. 2).
</P><P><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/6/66/Heidelberg_IMAG0449.png" style="float:right; margin-left: 10px;" title="ACM agar plate with D. acidovorans (left) overlaid with 0.2% HAuCl4 in 0.5% agarose."><FIGCAPTION style="width:200px;"><B>Fig. 4</B> ACM agar plate with <I>D. acidovorans</I> (left) overlaid with 0.2% HAuCl<SUB>4</SUB> in 0.5% agarose.</FIGCAPTION></A><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/1/10/Heidelberg_Goldsequenzen.png" style="float:right; margin-left: 10px;" title="Sequences of movie showing gold precipitation in D. acidovorans supernatant using gold concentrations ranging from 0 to 2.55 µg/ml."><FIGCAPTION style="width:200px;"><B>Fig. 5</B> Sequences of movie showing gold precipitation in <I>D. acidovorans</I> supernatant using gold concentrations ranging from 0 to 2.55 µg/ml.</FIGCAPTION></A></P><P>Using supernatants from the new <I>Delftia acidovorans</I> strain SPH-1, we showed precipitation of gold chloride solution to gold nanoparticles. Furthermore, we melted the purple-black nanoparticles to shiny solid gold as shown in figures 6 to 8.
</P><P><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/c/c5/Heidelberg_IMG_0624.JPG" style="float:right; margin-left: 10px;" title="Sparkling gold appearing in the melting pot."><FIGCAPTION style="width:150px;"><B>Fig. 6</B> Sparkling gold appearing in the melting pot.</FIGCAPTION></A></P><P><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/1/10/Heidelberg_IMG_4059.JPG" style="float:right; margin-left: 10px;" title="Final recovered solid gold collected in tube."><FIGCAPTION style="width:150px;"><B>Fig. 7</B> Final recovered solid gold collected in tube.</FIGCAPTION></A></P><P><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/8/8c/Heidelberg_P1010724.JPG" style="float:right; margin-left: 10px;" title="Solid gold recovered from nanoparticles in 2 ml tube."><FIGCAPTION style="width:150px;"><B>Fig. 8</B> Solid gold recovered from nanoparticles in 2 ml tube.</FIGCAPTION></A></P><P>
Next, we established purification of Delftibactin using HP20 resins. Additionally, we proved precipitation of gold by the purified Delftibactin (figure 6) and detected it by Micro-TOF. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Fig.8: Micro-TOF result for <I>D. acidovaorans</I></DIV></DIV></DIV><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 235px;"><DIV class="gallerytext"><P>Figure 6: Test if purified Delftibactin (diluted 1:10 in H<SUB>2</SUB>O) is able to precipitate gold. From left to right: water, 1:10 ACM media, 1:10 supernatant <I>D. acidovorans</I>, 1:10 filtered supernatant <I>D. acidovorans</I>, 1:10 purified Delftibactin
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>An important prerequisite for efficient gold recycling from electronic waste is, undoubtedly, that gold precipitation should occur even if the concentration of the gold ions in solution would be rather low, as this would likely be the case for most gold solutions derived from electronic waste.
</P><P>Therefore, we used an old, broken CPU and established a protocol for dissolving gold from gold-containing metal waste. We incubated golden CPU pins in aqua regia resulting in a gold-ion containing solution (figures 8 to 10).
</P><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 235px;"><DIV class="gallerytext"><P>Figure 8:  Pins removed from an old CPU.
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 235px;"><DIV class="gallerytext"><P>Figure 9: Green solution of dissolved pins.
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 235px;"><DIV class="gallerytext"><P>Figure 10: Gold chloride solution obtained from an old CPU.
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>Moreover, we showed precipitation of dissolved gold recovered from electronic waste by <I>D. acidovorans</I>.
</P><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 235px;"><DIV class="gallerytext"><P>Figure 11: &quot;Dissolved electronic waste&quot; and <I>D. acidovorans</I>.
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 235px;"><DIV class="gallerytext"><P>Figure 12: &quot;Dissolved electronic waste&quot; on <I>D. acidovorans</I></P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 235px;"><DIV class="gallerytext"><P>Figure 13: Precipitated &quot;dissolved electronic waste&quot; and <I>D. acidovorans</I></P></DIV></DIV></TD></TR></TBODY></TABLE><P>Adding this solution to <I>D. acidovorans</I> agar plates resulted in the formation of solid gold nanoparticles. 
</P><P>Taken together, we have successfully established a method enabling the recycling of pure gold from electronic waste using delftibactin produced by <I>D. acidovorans</I>. 

Although the recycling was been working efficiently in our hands, the approach of using the natural <I>D. acidovorans</I> bacterial strain for delftibactin production on a larger scale has several disadvantages:
</P><DL><DD>a)	<I>D. acidovorans</I> are relatively slow in growth (colony formation on plates occurs after 2-3 days)
</DD><DD>b)	Efficient production of delftibactin requires the strain to be grown in ACM media, which is rather expensive compared to typical <I>E. coli</I> growth media, making the procedure less economic
</DD></DL><P>Therefore, we wanted to engineer an <I>E. coli</I> strain producing delftibactin in high yields, thereby circumventing the abovementioned limitations. To this end, we first had to develop a thorough cloning strategy which would allow us to clone all necessary genes encoding the delftibactin-producing non-ribosomal peptide synthetases and polyketide synthetases from the del cluster (about 50 kb in total) and express them in <I>E. coli</I> alongside with the MethylMalonyl-CoA pathway providing one of the basic substrates in this pathway not naturally present in <I>E. coli</I>.
</P><H3><SPAN class="mw-headline" id="Amplification_of_the_Del_Cluster_Genes">Amplification of the Del Cluster Genes</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Fig.15: Overview on cloning strategy for the introduction of the delftibactin production pathway in <I>E. coli</I>.</DIV></DIV></DIV><P>The first step towards introducing the Delftibactin expression pathway into <I>E. coli</I> was the amplification of the Del-cluster encoded on the genome of <I>Delftia acidovorans</I>. To this end, we designed Gibson primers and amplified the genes of the non-ribosomal peptide synthetase and the polyketide synthase (PKS) pathway as well as of additional proteins, which were predicted to be necessary for the production of Delftibactin [5]. In the first weeks, PCRs were successfully established and optimized. At the same time, a separate plasmid was created encoding the PPTase from <I>Bacillus subtilis</I>, the MethylMalonyl-CoA pathway and a permeability device [BBa_I746200] <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_I746200" class="external autonumber" rel="nofollow">[1]</A> for the export of the synthesized delftibactin NRP. Additionally, low-copy plasmids from the partsregistry <A href="http://parts.igem.org/Main_Page" class="external autonumber" rel="nofollow">[2]</A> were successfully transformed and amplified using corresponding Gibson primers in order to generate the destination backbone fragments for the assembly.
</P><P>Details of our cloning strategy are shown in figure 14. Notably, the three plasmids we wanted to assemble are huge (23, 32 and 10 kbp in size) and was further complicated due to the high GC content and presence of repetitive elements in Del genes.
</P><H3><SPAN class="mw-headline" id="Gibson_Assembly_.26_Transformation">Gibson Assembly &amp; Transformation</SPAN></H3><P>Assembly of plasmids above 20 kb in size and composed of multiple different fragments is challenging when using conventional restriction-enzyme based cloning. Thus, we have used the Gibson Assembly method <A href="https://2010.igem.org/Team:Cambridge/Gibson/Protocol" class="external autonumber" rel="nofollow">[3]</A>, which was introduced to the iGEM community by Cambridge in iGEM 2010 as powerful alternative to such common cloning procedures. The assembled constructs of up to 32 kbp in size were transformed into E. coli via electroporation. Correct assemblies of the fragments was tested by analytical restriction digests. The exemplary restriction digest shown above (Fig. 16) confirmed the correct assembly of the three desired constructs as it shows the expected band pattern expected from in silico digestion. Clones (6 and 9) contained the plasmid that encodes for the Methylmalonyl-CoA pathway (Fig. 16a). The obtained DNA sequences were sent in for sequencing over the Gibson-assembled regions for confirmation. <A href="http://www.mwg-biotech.com/" class="external autonumber" rel="nofollow">[4]</A>,<A href="http://www.gatc-biotech.com/de/index/.html" class="external autonumber" rel="nofollow">[5]</A> The sequencing confirmed the accuracy of the sequence. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Fig.16: Restriction digest of three different plasmids needed for the NRPS/PKS pathway which to generated Delftibactin. a) Four digested colonies clones of the pIK8-plasmid, where clones 6 and 9 show the expected pattern. b) shows one clone of <I>E. coli</I> clone with the 32 kpb DelRest plasmid and was digested with three different enzymes and every lane shows the specific pattern for the according enzyme. c) Rshows the restriction digfest of the DelH plasmid with PvuI. Clone 5 shows the expected pattern and is probably positive.</DIV></DIV></DIV><P>The successful generation of DelRest plasmid was proven by different enzymatic restriction digests (Fig. 16b) and also attested by the sequencing. The sequence was compared with the available <I>D. acidovorans</I> SPH1 reference sequence of the Del cluster obtained from NCBI <A href="http://tools.neb.com/" class="external autonumber" rel="nofollow">[6]</A> (For further information please visit our labjournal). This shows, that Gibson assembly is a suitable cloning approach for rapid assembly of large NRPS and PKS expression constructs.
</P><P>Analytical restriction digest of DelH (Fig.16c) also gave rise to a number of positive clones. However, in contrary to the two successfully assembled and sequenced plasmids discussed above, the sequencing results derived from all DelH clones showed various mutations, which were mostly located within the region of the first DelH forward primer. Most of these mutations were deletions present at the very beginning of the DelH coding region. Interestingly, one specific deletion in the DNA sequence was found consistently in several independent clones. As we did not have any clone without mutations, we proceeded with the a DelH clone (termed C5), as this clone did not have any bp deletion but only harbored a minor base-pair substitution (leading to conversion of the DelH Alanine at position 10 to Threonine). Some exemplary sequences are listed below (Tab. 1) of <I>Delftia acidovorans</I> and two observed mutations in different DelH clones. 
</P><P><I>Tab.1 DelH 5’ sequence, in which most mutations were observed. The ATG start codon is depicted in bold. The table shows the sequence comparison between the DelH reference strand of </I>D. acidovorans<I> and two different exemplary </I>E. coli<I> clones transformed with the plasmid pHM04 (assembled DelH expression vector). The second line shows the accumulated deletion and the third line shows the clone containing 'only' single base pair substitution. Deletions appeared quite frequently while a substitution was only found in a single clone C5. The substitution changes the corresponding Alanine codon to Threonine.</I></P><TABLE class="wikitable"><TBODY><TR><TH> Organism </TH><TH> Plasmid containing </TH><TH> DNA -Sequence </TH><TH>Conclusion
</TH></TR><TR><TD><I>D. acidovorans</I></TD><TD> none </TD><TD>  ATG GACCGTGGC  CGCCTGCGC  CAAATCGC </TD><TD> correct
</TD></TR><TR><TD><I>E. coli</I> DH10ß</TD><TD> pHM04 </TD><TD> ATG GACCGTG-C   CGCCTGCGC   CAAATCGC </TD><TD> deletion
</TD></TR><TR><TD><I>E. coli</I> DH10ß C5 </TD><TD> pHM04 </TD><TD> ATG GACCGTGGC  CGCCTGCGC  CAAATCAC </TD><TD> substitution
</TD></TR></TBODY></TABLE><P>Due to the fact that <I>E. coli</I> seemed to somehow selected for mutated DelH clones, we hypothesize that expression of DelH in absence of the other proteins might be toxic for the cells. This would explain why <I>E. coli</I> selects for mutated, none functional, truncated DelH proteins. The same phenomenon of frequent mutations in presumably positive clones was also observed when we started cloning of the permeability device used in the pIK8 construct. The sequenced plasmids showed an unusually high accumulation of mutations compared to other constructs. In case of the methylmalonyl-CoA plasmid (pIK8), the problem was solved by the usage of a weak promoter and a weak ribosome binding site from the partsregistry for driving the expression of the permeability device. Based on this knowledge, DelH is currently being re-assembled into a new backbone (pSB6A1) <A href="http://parts.igem.org/Part:.pSB6A1?title=Part:pSB6A1" class="external autonumber" rel="nofollow">[7]</A> containing a weak promoter BBa_J23114 <A href="http://parts.igem.org/Part:BBa_J23114" class="external autonumber" rel="nofollow">[8]</A> and the ribosome binding site BBa_B0032<A href="http://parts.igem.org/Part:BBa_B0032" class="external autonumber" rel="nofollow">[9]</A>. While the new plasmid is constructed, the following experiments were performed with the C5 clone as we hypothesized, that pHM04 construct #C5 bearing the single nucleotide exchange at position 28 might still show expression of functional DelH when transformed into <I>E. coli</I> (the corresponding  amino acid exchange is located at the N-terminus).
</P><P>Therefore, we transformed all three plasmids, namely the pHM04 #C5 (encoding DelH), the DelRest plasmid (encoding all other del genes despite delH) and the pIK8 construct (bearing the MethylMalonyl-CoA pathway, the Sfp PPTase and the permeability device expression cassette) into <I>E. coli</I> DH10ß and <I>E. coli</I>BL21 DE3 in parallel. This should lead to <I>E. coli</I> cells producing delftibactin and secreting it into the media.
</P><H3><SPAN class="mw-headline" id="Expression_of_the_Delftibactin_NRPSs_.26_Associated_Genes">Expression of the Delftibactin NRPSs &amp; Associated Genes </SPAN></H3><P>We carried out experiments to test whether our constructs enable expression of the delftibactin NRPS/PKS pathway in <I>E. coli</I>.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Fig.17: SDS-Page of a) <I>E. coli</I> DH10ß DelRest D8w and b) <I>E. coli</I> BL21 DE3 and <I>E. coli</I> BL21 DE3 containing DelH C5</DIV></DIV></DIV><P>For expression of DelH and DelRest, we conducted SDS-PAGE followed by Coomassie stainig. As negative controls we used untransformed cells and cells transformed only with the original plasmid backbones. The proteins DelE, DelG and DelH are significantly larger than any protein that is expressed by our host <I>E. coli</I>. Therefore, the expression of the introduced genes was clearly visible on the SDS-PAGE (Fig. 17). Even though the expression was weak, as we have expected for such a large proteins, clear distinct bands at the expected size of DelE and DelG were detected for the clone transformed with the DelRest plasmid and a band at the size of DelH for the clone transformed with the DelH plasmid. As the promoter in front of DelE and DelG controls the expression of DelA, DelB, DelC, DelD and DelF, too, one can assume simultaneous expression of these Del proteins. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 19: Left: TOP10-pIK8.6, right: TOP10-pIK8.1 (negative control)counterclockwise, starting top right: 8 µl, 4 µl, 2 µl, 1 µl bacitracin</DIV></DIV></DIV> Furthermore, the expression of the PPTase was verified by performing an IndC activity assay. The indigoidine synthetase IndC activity is dependent on the presence of a functional PPTase which is needed for synthesis of the blue pigment indigoidine. Co-transformation of the corresponding plasmid pIK8 (enabling Sfp PPTase expression) with an IndC indigoidine synthetase expression construct lacking a PPTase expression cassette. The transformed <I>E. coli</I> grew very slowly and developed the expected blue phenotype. From these results, we can clearly conclude that the PPTase on pIK8 is functionally expressed (note: decelerated growth kinetics of <I>E. coli</I> results from the metabolic burden that is caused by the synthesis of the indigoidine).
<P>For proving that <I>E. coli</I> also produces the permeability device, which is needed for export ofDelftibactin out of the cells, a Hemmhof agar diffusion test with bactracin was performed. Bacitracin is a very large antibiotic which is usually not able to diffuse across the cell membrane passively. Absent growth upon application of bacitracin of bacteria containing the plasmid while in the control cells without the device were not affected by the antibiotic (Fig. 18) confirms expression of the transporter. 
</P><P>In conclusion, we successfully expressed the recombinant Delftibactin NRPS/ PKS pathway as well as the required Methylmalonyl-CoA pathway, the PPTase and permeability device in <I>E. coli</I>. 
</P><P>We furthermore showed, that it is not only possible to assemble large plasmids (in sum these were about 64 kpb in total size in our case) and transform them into <I>E. coli</I>, but showed successful expression of large NRPS/PKS modules in our host strain.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Heidelberg/Templates/Delftibactin_Results">http://2013.igem.org/Team:Heidelberg/Templates/Delftibactin_Results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Templates/Delftibactin_Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Templates/Delftibactin_Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Templates/Delftibactin_Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Templates/Delftibactin_Results&amp;oldid=329404" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>