<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Delftibactin_MMCoA"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Delftibactin/MMCoA">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Delftibactin/MMCoA&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Delftibactin/MMCoA&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Delftibactin/MMCoA&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Delftibactin/MMCoA" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Delftibactin/MMCoA</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="row"><DIV id="nav" class="navigation"><DIV class="item team" data-offset-l="-130"><H2>Team</H2><UL><LI><A href="/Team:Heidelberg/Team">Team Members</A></LI><LI><A href="/Team:Heidelberg/AttributionsII">Attributions</A></LI><LI><A href="/Team:Heidelberg/Team/Sponsors">Sponsors</A></LI><LI><A href="/Team:Heidelberg/Team/Collaboration">Collaborations</A></LI><LI><A href="/Team:Heidelberg/Team/Gallery">Gallery</A></LI></UL></DIV><DIV class="item project" data-offset-l="-144"><H2>Project</H2><UL><LI><A href="/Team:Heidelberg/NRPS">Background</A></LI><LI><A href="/Team:Heidelberg/Project">Overview</A></LI><LI><A href="/Team:Heidelberg/Project/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Project/Indigoidine-Tag">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Project/Tag-Optimization">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Project/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Project/Achievements">Achievements</A></LI></UL></DIV><DIV class="item notebook" data-offset-l="-139"><H2>Notebook</H2><UL><LI><A href="/Team:Heidelberg/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Tyrocidine">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Indigoidine">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/NRPSDesigner">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/Notebook/Methods">Methods</A></LI><LI><A href="/Team:Heidelberg/Materials">Materials</A></LI></UL></DIV><DIV class="item parts" data-offset-l="-134"><H2>Parts</H2><UL><LI><A href="/Team:Heidelberg/Parts">Submitted Parts</A></LI><LI><A href="/Team:Heidelberg/Favorite_Parts">Favorite Parts</A></LI><LI><A href="/Team:Heidelberg/RFCs">Our RFCs</A></LI></UL></DIV><DIV class="item software" data-offset-l="-134"><H2>Software</H2><UL><LI><A href="/Team:Heidelberg/Project_Software">Project</A></LI><LI><A href="http://igem2013.bioquant.uni-heidelberg.de/NRPSDesigner/">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/iGEM42">iGEM42</A></LI><LI><A href="/Team:Heidelberg/OpenSource">Open Source</A></LI><LI><A href="/Team:Heidelberg/Project/AchievementsSoftware">Achievements</A></LI></UL></DIV><DIV class="item human" data-offset-l="-118"><H2>Human Practice</H2><UL><LI><A href="/Team:Heidelberg/HumanPractice">Overview</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BroadPublic">Broad Public</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Survey">Survey</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/YoungGeneration">Young Generation</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Artists">Arts and Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BeyondScience">Beyond Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Experts">Experts</A></LI></UL></DIV><DIV class="item safety" data-offset-l="-115"><H2>Safety</H2><UL><LI><A href="/Team:Heidelberg/Safety">Safety</A></LI></UL></DIV><DIV class="item model" data-offset-l="-144"><H2>Modeling</H2><UL><LI><A href="/Team:Heidelberg/Modelling/Gold_Recovery">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Modelling/Ind_Production">ind-Production</A></LI></UL></DIV><DIV class="item"><SPAN style="margin-left:6%">HOME</SPAN></DIV></DIV></DIV><DIV class="container"><DIV><H1><SPAN style="font-size:140%;color:#FFCC00;">Methylmalonyl-CoA Pathway.</SPAN><SPAN class="text-muted" style="font-family:Arial, sans-serif; font-size:90%"> Making the whole thing work.</SPAN></H1></DIV><DIV class="row"><DIV class="col-sm-12 col-md-6"><UL class="pagination" style="margin-bottom:2%; margin-left:17%;"><LI><A href="#" id="backwards">«</A></LI><LI class="active month_tab" id="june"><A href="#" style="width:90px; text-align:center">June</A></LI><LI class="month_tab" id="july"><A href="#" style="width:90px; text-align:center">July</A></LI><LI class="month_tab" id="august"><A href="#" style="width:90px; text-align:center">August</A></LI><LI class="month_tab" id="september"><A href="#" style="width:90px; text-align:center">September</A></LI><LI><A href="#" id="forwards">»</A></LI></UL><DIV style="margin-bottom:8%;"><DIV id="myCarousel" class="carousel slide"><DIV class="carousel-inner" style="margin-top:15%"><DIV class="item active june first"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0" id="week1"><H1>Week 6</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">In this week the first experiments for the genomic integration of the methyl-malonyl-CoA synthetase were set up. First of all the plasmids were transformed into TOP10 grown and purified. The first attempts on the amplification of the fragment didn't work at all and we ended up with a specific product of the wrong size.</P></DIV></DIV></DIV><DIV class="item june"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0" id="week1"><H1>Week 7</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">After a validation of the template plasmid via restriction digest the PCR reaction was optimised. Unfortunately none of the cells transformed with the PCR product using electroporation could be positively screened, no matter how the PCR fragment was amplified.</P></DIV></DIV></DIV><DIV class="item june"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0" id="week1"><H1>Week 8</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">As the repetition of the colony PCR under different conditions was still negative, new PCR products were prepared and pooled. To eliminate possible errors fresh competent cells were prepared.</P></DIV></DIV></DIV><DIV class="item june last"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0" id="week1"><H1>Week 9</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">After doing colony PCR again and getting inconclusive results, it was decided that the genomic integration failed using this experimental setup. Thus we started from scratch using a new primer pair and finally it turned out that the polymerase was causing the problems. By the end of the week BAP1-pKD46 could be transformed.</P></DIV></DIV></DIV><DIV class="item july first"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0" id="week1"><H1>Week 10</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">Even after running several colony PCRs there were no positive clones to be found. Thus the pKD46 was digested for validation. On the gel a miniprep from the PCR-transformed cells was included. The identity of the pKD46 was fine and the miniprep didn't contain any DNA, as expected.</P></DIV></DIV></DIV><DIV class="item july"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 11</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">In this week the whole procedure was repeated again from fragment amplification to extensive colony PCR. This week several different colony types were encountered and one that grew only after 3 days actually gave a band at 1.2 kb, but it turned out to be unspecific.</P></DIV></DIV></DIV><DIV class="item july"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 12</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">Several possible error sources such as unselective plates were investigated, but all turned out to be fine. The colony PCR was then repeated using yet another primer pair, but it still only gave unspecific or no bands.</P></DIV></DIV></DIV><DIV class="item july"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 13</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">In order to check whether the cell line used was actually the right one primers against sfp were used and cell from other subgroups were included. It turned out that something really weird was going onwith the cells, since sfp could not be detected. Thus the entirely new strategy of generating a plasmid instead of doing genomic integration was initialised.</P></DIV></DIV></DIV><DIV class="item july last"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 14</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">To be really sure about the sfp a new plasmid preparation was made and cells were tranformed again. In parallel the pLF03 was also prepared and sent for sequencing. It turned out, that there is a deletion in the sequence and thus the primers can't bind properly.</P></DIV></DIV></DIV><DIV class="item august first"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 15</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">A final try on integrating the methyl-malonyl-CoA pathway was made. At the same time new plasmids carrying the pathway arrived and had to be validated first via PCR and restriction digestion. Several PCR reaction were run and also the first Gibson assembly and transformation was done for both plasmids. The positive colonies were sent for sequencing by the end of the week.</P></DIV></DIV></DIV><DIV class="item august"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 16</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">After finally giving up on the genomic integration we focused on the plasmid strategy, where the sequencing results of the two plasmids arrived. As they were already positive glycerol stocks were prepared.</P></DIV></DIV></DIV><DIV class="item august"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 17</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">In order to check for sfp expression pMM64, carrying an indigoidine synthetase was cotransformed, but didn't give any blue colonies. Thus the plasmid backbone had to be changed to allow for cotransformation with our own indigoidine construct. One of the new plasmids was sent for sequencing.</P></DIV></DIV></DIV><DIV class="item august"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 18</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">The sequencing results of pIK1.3 revealed a point mutation in the permeability device. All the other plasmids sent for sequencing also carried some kind of mutation that introduced a stop codon within the permeability device, which is thus non-functional. Moreover there were still no blue colonies even though we now used our own construct.</P></DIV></DIV></DIV><DIV class="item august last"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 19</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">After doing some research in 2007's Cambridge team wiki it was found, that the permeability device is probably toxic to the cells and can thus only be succesfully cloned behind a very weak promotor.</P></DIV></DIV></DIV><DIV class="item september first"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 20</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">This week we grew the first blue colonies with an updated version of the indigoidine construct. We also put the permeability device behind a very weak promotor and prepared new plasmids for sequencing.</P></DIV></DIV></DIV><DIV class="item september"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 21</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">When the sequencing results were analysed there were no mutations in the permeability device. Now the cloning of the plasmids for the parts submission started and by the end of the week we sent them for sequening.</P></DIV></DIV></DIV><DIV class="item september last"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 22</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">Unfortunately we noticed that we accidentally introduced an additional restriction site and could thus not submit the part to the registry. We already removed the restriction site and validated it by restriction digestion. Anyway we are going to try different restriction enzymes and send it for sequencing after wiki freeze.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="col-sm-12 col-md-6"><DIV class="jumbotron methods" data-spy="scroll" data-target="#navbarExample" data-offset="0" style="margin-top:10%"><H3>Graphical Abstract</H3></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-sm-12"><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#a6" data-toggle="tab">Overview:genomic integration</A></LI><LI><A href="#b6" data-toggle="tab">Lab book: genomic integration</A></LI></UL></DIV><DIV class="jumbotron"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a6"><H2><SPAN class="mw-headline" id="Goal"> Goal </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Vector map of the <A href="/wiki/index.php?title=File:Heidelberg_pKD46.gb&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg pKD46.gb (page does not exist)">pKD46 plasmid</A>. pKD20 was used in the original publication, but pKD46, constructed by the same authors, shows higher recombination efficiency and is the one used in all recombination experiments.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Vector map of the <A href="/wiki/index.php?title=File:Heidelberg_PET21c.gb&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PET21c.gb (page does not exist)">pET-21c plasmid</A>.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">MCS of pET-21c with the binding sites of the IK07, IK08, and IK02 primers highlighted.</DIV></DIV></DIV><P>Integration of the methylmalonyl-CoA sythesis pathway into the BAP1 genome. This is a reproduction of a part of &lt;bib id=&quot;pmid17959404&quot;/&gt;. For the integration, the λ-red protocol established by Datsenko and Wanner&lt;bib id=&quot;pmid10829079&quot;/&gt; is used. pKD20/pKD46 plasmids contain the λ-red genes begind an arabinose inducible promoter, a temperature-sensitive origin, and an ampicillin selection marker. The part to be integrated is on the pLF03 plasmid, which is identical to pYW201&lt;bib id=&quot;pmid17959404&quot;/&gt; in every way except for the selection marker in the insert (pYW201 has kanamycin resistance, pLF03 chloramphenicol). pLF03 contains an ampicillin marker that is expressed constitutively, chloramphenicol is only used to test for successful integration, as it is behind an IPTG-inducible T7 promoter (as are the other two genes, pccB and accA1). pLF03 is derived from pET-21c. The expected amplificate size for PCR of pLF03 is 4kb. pccB-accA will replace the ygfG gene in the BAP1 genome.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption"><A href="/wiki/index.php?title=File:Heidelberg_E._coli_BL21-DE3_ygfG.gb&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg E. coli BL21-DE3 ygfG.gb (page does not exist)">BL21-DE3 genomic region around ygfG, which will be replaced by pccB-accA in BAP1. Sites homologous to ygfG21C1 and ygfG21C2 as well as the IK01 primer binding site are shown.</A></DIV></DIV></DIV></DIV><DIV class="tab-pane" id="b6"><H2><SPAN class="mw-headline" id="2013-06-03"> 2013-06-03 </SPAN></H2><UL><LI> inoculate 50 ml LB with BAP1, grow at 37°C until OD=0.57
</LI><LI><A href="/wiki/index.php?title=Preparing_glycerol_stocks&amp;action=edit&amp;redlink=1" class="new" title="Preparing glycerol stocks (page does not exist)">prepare BAP1 glycerol stocks</A></LI><LI><A href="/wiki/index.php?title=Heat-shock_competent_bacteria&amp;action=edit&amp;redlink=1" class="new" title="Heat-shock competent bacteria (page does not exist)">prepare competent BAP1</A></LI><LI><A href="/wiki/index.php?title=DNA-Storage&amp;action=edit&amp;redlink=1" class="new" title="DNA-Storage (page does not exist)">recover pKD46, pCP20, pLF03</A></LI><LI> transform TOP10 with pKD46, plate on Amp, grow at 30°C
</LI><LI> transform TOP10 with pLF03, plate on Amp, grow at 37°C
</LI><LI> transform TOP10 with pCP20, plate on Amp, grow at 30°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-06-04"> 2013-06-04 </SPAN></H2><UL><LI> inoculate 4 ml LB+Amp with TOP10-pLF03, grow at 37°C
</LI><LI> make miniPrep of pLF03 =&gt; 97 ng/µl in 27 µl
</LI><LI> transform <A href="/wiki/index.php?title=E._coli_BAP1,_competent&amp;action=edit&amp;redlink=1" class="new" title="E. coli BAP1, competent (page does not exist)">BAP1</A> with pKD46, plate on Amp, grow at 30°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-06-05"> 2013-06-05 </SPAN></H2><UL><LI> prepare 50 ml LB + Amp + Arabinose(0.1%), inoculate with BAP1-pKD46
</LI><LI> prepare 4 ml LB + Amp, inoculate with TOP10-pKD46 (for miniPrep)
</LI><LI> no colonies with pCP20 =&gt; repeat transformation with three times the amount of plasmid (30 µl)
</LI><LI> BAP1-pKD46 reached OD=0.68 at 22:30 =&gt; put in fridge over night
</LI></UL><H2><SPAN class="mw-headline" id="2013-06-06"> 2013-06-06 </SPAN></H2><UL><LI> inoculate 50 ml LB + Amp + Arabinose(0.1%) with 500 µl BAP1-pKD46 from ON culture (fridge)
</LI><LI> at OD=0.49: <A href="/wiki/index.php?title=Preparing_glycerol_stocks&amp;action=edit&amp;redlink=1" class="new" title="Preparing glycerol stocks (page does not exist)">make glycerol stocks</A>, <A href="/wiki/index.php?title=Electrocompetent_bacteria&amp;action=edit&amp;redlink=1" class="new" title="Electrocompetent bacteria (page does not exist)">prepare electrocompetent</A> BAP1-pKD46
</LI><LI> make miniPrep of TOP10-pKD46 ON culture -&gt; 31 ng/µl
</LI><LI> low yield of TOP10 miniPrep: make 2 miniPreps of BAP1-pKD46 ON culture -&gt; 45.5 ng/µl and 46.0 ng/µl
</LI><LI> evening: no colonies with pCP20 -&gt; write Lei Fang (Pfeifer lab) -&gt; their stock is bad, we should get pCP20 somewhere else
</LI></UL><H2><SPAN class="mw-headline" id="2013-06-07"> 2013-06-07 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Gel electrophoresis of the PCR amplificate. Left: NEB 2 log DNA ladder; middle: PCR without annealing step; right: full PCR</DIV></DIV></DIV><UL><LI> run PCR of 5 ng pLF03 (1 µl 1:20 dilution of miniPrep from 2013-06-04) of with primers ygfG21C1 and ygfG21C2 using Phusion polymerase:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 30
</TD></TR><TR><TD rowspan="3"> 30 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD> 66 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 90
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 450
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> using hot start at 98°C
</LI><LI> two PCR  samples were run, in one the annealing step at 66°C was left out
</LI><LI> =&gt; no amplificate
</LI></UL><H2><SPAN class="mw-headline" id="2013-06-08"> 2013-06-08 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Gel electrophoresis of the PCR amplificate. Left: NEB 2 log DNA ladder; middle: full PCR; right: PCR without 58°C annealing step</DIV></DIV></DIV><UL><LI> Looking in more detail at the primers, the parts binding to the template have melting temperatures of 40 and 45°C (the rest is homologous to <I>E. coli</I> genome for recombination)
</LI><LI> run PCR of 5 ng pLF03 (1 µl 1:20 dilution of miniPrep from 2013-06-04) with primers ygfG21C1 and ygfG21C2 using Phusion polymerase:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 30
</TD></TR><TR><TD rowspan="3"> 5 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD> 39 </TD><TD> 45
</TD></TR><TR><TD> 72 </TD><TD> 90
</TD></TR><TR><TD rowspan="3"> 5 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD> 45 </TD><TD> 45
</TD></TR><TR><TD> 72 </TD><TD> 90
</TD></TR><TR><TD rowspan="3"> 20 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD> 58 </TD><TD> 45
</TD></TR><TR><TD> 72 </TD><TD> 90
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> using hot start at 98°C
</LI><LI> two PCR  samples were run, in one the annealing step at 58°C was left out
</LI><LI> =&gt; no amplificate
</LI></UL><H2><SPAN class="mw-headline" id="2013-06-09"> 2013-06-09 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Gel electrophoresis of the PCR amplificate. Left: NEB 2 log DNA ladder; right: PCR product</DIV></DIV></DIV><UL><LI> run PCR of 5 ng pLF03 (1 µl 1:20 dilution of miniPrep from 2013-06-04) with primers ygfG21C1 and ygfG21C2 using Phusion polymerase:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 30
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD> 40 </TD><TD> 60
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> using hot start at 98°C
</LI><LI> amplificate has 1.3kb -&gt; too short, but specific (one distinct band)
                                </LI><LI class="active"><A href="#a7" data-toggle="tab">Lab book: genomic integration</A></LI></UL></DIV><DIV class="jumbotron"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a7"><H2><SPAN class="mw-headline" id="2013-06-10"> 2013-06-10 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Gel electrophoresis of the pLF03 digestion. Left: NEB 2 log DNA ladder; middle: digestion with EcoRI, right: digestion with XhoI+NdeI</DIV></DIV></DIV><UL><LI> need to verify plasmid identity: digest pLF03 with EcoRI (for linearization) and XhoI+NdeI (verify insert size); 388 ng (4 µl of miniPrep from 2013-06-04) each
</LI><LI> insert only 3.3 kb in length, expected &gt;4 kb (<A href="http://www.uniprot.org/uniprot/Q9RGQ6" class="external text" rel="nofollow">AccA1</A>: 590AS * 3 = 1770 bp, <A href="http://www.uniprot.org/uniprot/Q9X4K7" class="external text" rel="nofollow">PccB</A>: 530AS * 3 = 1590 bp, <A href="http://www.uniprot.org/uniprot/P62577" class="external text" rel="nofollow">CatR</A>: 219 AS = 657 bp; adds up to 4017 bp, not counting RBS and spacers between the individual genes)
</LI><LI> contacted Lei Fang (Pfeifer Lab), there are NdeI and EcoRI sites in catR (they were not indicated in his plasmid map)
</LI><LI> to verify presence of chloramphenicol acetyl transferase in insert: transform <A href="/wiki/index.php?title=E._coli_BAP1,_competent&amp;action=edit&amp;redlink=1" class="new" title="E. coli BAP1, competent (page does not exist)">BAP1</A> with 97 ng pLF03 (1 µl of miniPrep from 2013-06-04), plate on Amp + IPTG
</LI><LI> evening: pick BAP1-pLF03 colonies, transfer to Cm + IPTG
</LI><LI> inoculate 4 and 5 ml of liquid culture (Amp) with BAP1-pLF03
</LI></UL><H2><SPAN class="mw-headline" id="2013-06-11"> 2013-06-11 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Gel electrophoresis of the PCR amplificate. Lane 1: NEB 2-log ladder; lane 2: PCR with Phusion Flash; lane 3: PCR with Taq at 43°C annealing temperature; lane 4: PCR with Taq at 41.5°C annealing temperature</DIV></DIV></DIV><UL><LI> BAP1-pLF03 grew on Cm + IPTG
</LI><LI> try to optimize PCR:
</LI><LI> run PCR of 5 ng pLF03 (1 µl 1:20 dilution of miniPrep from 2013-06-04) with primers ygfG21C1 and ygfG21C2 using Phusion Flash F548 polymerase (performed by Dominik):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD rowspan="3"> 5 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 45 </TD><TD> 5
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD rowspan="3"> 25 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 55 </TD><TD> 5
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 120
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> run PCR of 5 ng pLF03 (1 µl 1:20 dilution of miniPrep from 2013-06-04) with primers ygfG21C1 and ygfG21C2 using Taq polymerase (cheaper):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 94 </TD><TD> 180
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 94 </TD><TD> 20
</TD></TR><TR><TD> 41.5/43 </TD><TD> 60
</TD></TR><TR><TD> 72 </TD><TD> 480
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 300
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI><A href="/wiki/index.php?title=Preparing_glycerol_stocks&amp;action=edit&amp;redlink=1" class="new" title="Preparing glycerol stocks (page does not exist)">prepare glycerol stock</A> of BAP1-pLF03
</LI><LI> make miniPreps of BAP1-pLF03 -&gt; 21 ng / µl and 27 ng / µl in 27.5 µl
</LI></UL><UL><LI> received just plated DH5α-pCP20 from Sourjik Lab, grow at 30°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-06-12"> 2013-06-12 </SPAN></H2><UL><LI> purify Phusion Flash PCR product with Qiagen PCR purification kit -&gt; 120 ng / µl in 27.5 µl
</LI><LI> inoculate 2 x 5 ml LB + Amp with DH5α-pCP20, grow at 30°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-06-13"> 2013-06-13 </SPAN></H2><UL><LI> miniPreps of DH5α-pCP20 -&gt; 7.8 ng / µl and 14.6 ng / µl in 27.5 µl, respectively
</LI><LI> low yield of miniPreps: inoculate 2 x 5 ml LB + Amp with DH5α-pCP20, grow at 30°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-06-14"> 2013-06-14 </SPAN></H2><UL><LI> miniPreps of DH5α-pCP20 -&gt; 25 ng / µl and 29 ng / µl in 27.5 µl, respectively
</LI><LI> prepare glycerol stocks of DH5α-pCP20
</LI><LI> to avoid electroporation with intact pLF03: add 10 µl of PCR amplificate (1.2 mg) on gel, perform gel extraction (QiaGen gel extraction kit) -&gt; 3.2 ng / µl
</LI><LI> add 240 ng (2 µl) raw PCR product to 100 µl electrocompetent BAP1-pKD46
</LI><LI> add 16 ng (5 µl) gel-extracted PCR product to 100 µl electrocompetent BAP1-pKD46
</LI><LI> electroporate 50 µl each, add to 1 ml SOC with 1 mM IPTG
</LI><LI> shake at 300 rpm, 37°C for 1 h
</LI><LI> spin cells down (3 min 10 000 rpm = 8000 g), decant supernatant (leave about 200 µl supernatant in eppendorf tube)
</LI><LI> resuspend pellet, plate 100 µl on Cm-IPTG, leave rest at RT
</LI><LI> grow at 37°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-06-15"> 2013-06-15 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG. Lane 1: NEB 2-log; lane 2: negative control (BAP1-pLF03); rest: colony-PCRs from PCR-purified plate.</DIV></DIV></DIV><UL><LI> one colony on plate with gel-extracted PCR product
</LI><LI> many colonies on plate with PCR-purified PCR product
</LI><LI> pick colonies from plate with PCR-purified PCR product, run colony-PCR (Taq, 25 µl total volume) with primers IK01 and IK02 (expected product: 465 bp):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 180
</TD></TR><TR><TD rowspan="3"> 30 </TD><TD> 95 </TD><TD> 30
</TD></TR><TR><TD> 61 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 30
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG. Lane 1: NEB 2-log; lane 2: negative control (BAP1-pLF03); lane 3: colony-PCR of colony electroporated with gel-extracted PCR product; rest: colony-PCRs of colonies electroporated with PCR-purified PCR product.</DIV></DIV></DIV><UL><LI> no product -&gt; pick colony from gel-extracted plate, pick colonies from PCR-purified plate, run colony-PCR (Taq, 25 µl total volume) with primers IK01 and IK02:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 240
</TD></TR><TR><TD rowspan="3"> 30 </TD><TD> 95 </TD><TD> 30
</TD></TR><TR><TD> 61 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 30
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> no product -&gt; 2 possibilities:
</LI><LI> all colonies bear complete pLF03 plasmid (PCR was run with 5 ng template, DNA concentration in PCR product: 120 ng / µl, electroporated with 2 µl -&gt; 1/6 ng pL03 vs. 240 ng amplificate)
</LI><LI> PCR did not work
</LI><LI> for colony from gel-extracted plate + 3 colonies from PCR-purified plate: add up to 3 ml medium, add IPTG (1 mM) + Cm, grow at 37°C
</LI><LI> plate rest of bacteria electroporated with gel-extracted PCR product on Cm + IPTG
</LI></UL><H2><SPAN class="mw-headline" id="2013-06-16"> 2013-06-16 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">miniPrep of BAP1-pKD46 electroporated with PCR product from pLF03. Lane 1: NEB 2-log; lane 2: miniPrep of colony electroporated with gel-extracted product; rest: miniPreps of colonies electroporated with PCR-purified product.</DIV></DIV></DIV><UL><LI> no colonies on gel-extracted plate
</LI><LI> make miniPreps of ON cultures, run gel
</LI><LI> pLF03: 9.2 kb, bands above 10 kb, bands smear, low intensity (applied all 30 µl of miniPrep to gel) -&gt; fragments of genomic DNA?
</LI><LI> =&gt; PCR went wrong?
                                </LI><LI class="active"><A href="#a8" data-toggle="tab">Lab book: genomic integration</A></LI></UL></DIV><DIV class="jumbotron"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a8"><H2><SPAN class="mw-headline" id="2013-06-17"> 2013-06-17 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG. Lanes 1-6: annealing at 66°C; lanes 8-11: touch-down 66-&gt;60°C; lane 6: NEB 2-log; lane 1,7: negative control (BAP1-pLF03); lane 2,8: colony-PCR of colony electroporated with gel-extracted PCR product; rest: colony-PCRs of colonies electroporated with PCR-purified PCR product.</DIV></DIV></DIV><UL><LI> repeat colony PCR, use 1 µl of liquid culture from gel-extracted plate, pick colonies from PCR-purified plate (Taq, 25 µl total volume)
</LI><LI> 2 conditions:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 600
</TD></TR><TR><TD rowspan="3"> 30 </TD><TD> 95 </TD><TD> 240
</TD></TR><TR><TD> 64 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 60
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 600
</TD></TR><TR><TD rowspan="3"> 12 </TD><TD> 95 </TD><TD> 240
</TD></TR><TR><TD> 66°C ↓0.5°C </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 60
</TD></TR><TR><TD rowspan="3"> 18 </TD><TD> 95 </TD><TD> 240
</TD></TR><TR><TD> 60°C </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 60
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> no positives
</LI><LI> plate samples of liquid cultures from 2013-06-15 on Amp, grow at 37°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-06-18"> 2013-06-18 </SPAN></H2><UL><LI> colonies grew on Amp
</LI></UL><H2><SPAN class="mw-headline" id="2013-06-19"> 2013-06-19 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Gel electrophoresis of the PCR amplificate. Lane 1: NEB 2-log ladder; lanes 2-5: PCR with Phusion Flash (2 µl of product were put on gel)</DIV></DIV></DIV><UL><LI> prepare for repetition of experiment: run 4 PCRs of 1 ng pLF03 (miniPrep from 2013-06-04) with primers ygfG21C1 and ygfG21C2 using Phusion Flash (performed by Dominik), one reaction = 50 µl total volume
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD rowspan="3"> 5 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 45 </TD><TD> 5
</TD></TR><TR><TD> 72 </TD><TD> 130
</TD></TR><TR><TD rowspan="3"> 25 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 55 </TD><TD> 5
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 120
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Gel electrophoresis of the purified PCR amplificate. Lane 1: NEB 2-log ladder; lane 2: purified PCR with Phusion Flash (1 µl of product were put on gel)</DIV></DIV></DIV><UL><LI> pool products, <A href="/wiki/index.php?title=Isopropanol_PCR_purification&amp;action=edit&amp;redlink=1" class="new" title="Isopropanol PCR purification (page does not exist)">purify</A> (digestion only 3 h)
</LI><LI> nanoDrop: 2088 ng/µl -&gt; remeasure: 532 ng/µl
</LI><LI> load 1 µl purified PCR product on gel -&gt; very weak band -&gt; nanoDrop wrong
</LI><LI> to eliminate error: make fresh electrocompetent cells:
</LI><LI> pick colony from BAP1-pKD46 plate from 2013-06-04
</LI><LI> inoculate 1 ml LB + Amp
</LI><LI> grow at 30°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-06-20"> 2013-06-20 </SPAN></H2><UL><LI> too little cells: use aliquot from 2013-06-06
</LI><LI> use all available DNA (14 µl) and 100 µl cells (complete aliquot)
</LI><LI> grow in SOC + IPTG(1mM) at 300 rpm for 3h
</LI><LI> spin cells down, decant supernatant, resuspend in remainder of medium
</LI><LI> plate 10 µl, rest on Cm + IPTG
                                </LI><LI class="active"><A href="#a9" data-toggle="tab">Lab book: genomic integration</A></LI></UL></DIV><DIV class="jumbotron"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a9"><H2><SPAN class="mw-headline" id="2013-06-24"> 2013-06-24 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG. Lane 1: NEB 2-log; lanes 2-4: negative control (BAP1-pLF03); lanes 5-10: colony-PCR of colony electroporated with purified PCR product; lanes 2,5,8: primers IK01+IK02; lanes 3,6,9: primers IK01+IK03 (positive control); lanes 4,7,10: primers IK05+IK06 (negative control)</DIV></DIV></DIV><UL><LI> no colonies on 10 µl plate, 2 colonies on rest plate
</LI><LI> colony-PCR using primer pairs IK01+IK02, IK01+IK03 (positive control), IK05+IK06 (negative control) (Taq, 20 µl total volume)
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 30 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 62 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 60
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> inconclusive results: no band for IK05+IK06 in BAP1-pLF03 (negative control)
</LI><LI> grow liquid cultures at 37°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-06-25"> 2013-06-25 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG and then in LB ON. Lane 1: NEB 2-log; lanes 2-5: negative control (BAP1-pLF03); lanes 6-13: colony-PCR of colonies electroporated with purified PCR product; lanes 2,6,10: primers IK01+IK02; lanes 3,7,11: primers IK01+IK03 (positive control); lanes 4,8,12: primers IK05+IK06 (negative control); lanes 5,9,13: primers IK01+IK06 (negative control)</DIV></DIV></DIV><UL><LI> repeat PCR with 1 µl of ON cultures:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 12 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 68 ↓0.5°C </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 120 (IK01+IK02; IK01+IK03;IK05+IK06) / 240 (IK01+IK06)
</TD></TR><TR><TD rowspan="3"> 18 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 62 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 120 (IK01+IK02; IK01+IK03;IK05+IK06) / 240 (IK01+IK06)
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> genomic integration failed
</LI><LI> pick colony from BAP1-pKD46 plate from 2013-06-04, grow in 3 ml LB+Amp at 30°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-06-26"> 2013-06-26 </SPAN></H2><UL><LI> inoculate 50 ml LB + Amp + Ara(0.5%) with 1.1 ml of ON culture
</LI><LI> grow at 30°C to OD=0.73, <A href="/wiki/index.php?title=Electrocompetent_bacteria&amp;action=edit&amp;redlink=1" class="new" title="Electrocompetent bacteria (page does not exist)">prepare electrocompetent</A> BAP1-pKD46
</LI></UL><H2><SPAN class="mw-headline" id="2013-06-27"> 2013-06-27 </SPAN></H2><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 135px;"><DIV class="gallerytext"><P>Gel electrophoresis of the PCR amplificate. Lane 1: NEB 2-log ladder; lane 2: 1 µl of PCR amplificate (first PCR)
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 135px;"><DIV class="gallerytext"><P>Gel electrophoresis of the PCR amplificate. Lane 1: NEB 2-log ladder; lane 2: 1 µl of PCR amplificate (second PCR)
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 135px;"><DIV class="gallerytext"><P>Gel electrophoresis of the PCR amplificate. Lane 1: 19 µl of amplificate from first PCR; lane 2: 19 µl of amplificate from second PCR; lane 3: NEB 2-log ladder; lane 4: 1 µl of PCR amplificate (third PCR)
</P></DIV></DIV></TD></TR></TBODY></TABLE><UL><LI> run PCR of 5 ng pLF03 (1 µl 1:20 dilution of miniPrep from 2013-06-04) with primers IK07 and IK08 using Phusion polymerase (20 µl total volume) on cycler 1 (does not cool down to 4°C) (using hot start):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 30
</TD></TR><TR><TD rowspan="3"> 30 </TD><TD> 98 </TD><TD> 5
</TD></TR><TR><TD> 66 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 150
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> no product -&gt; repeat PCR with cycler 2 (fully functional)
</LI><LI> no product -&gt; run 2-step PCR with 4 ng pLF03 (0.2 µl of 21 ng/µl miniPrep from 2013-06-11) with primers IK07 and IK08 using Phusion polymerase (20 µl total volume, using hot start) on cycler 2:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 30
</TD></TR><TR><TD rowspan="3"> 30 </TD><TD> 98 </TD><TD> 5
</TD></TR><TR><TD> 72 </TD><TD> 150
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI>no product
</LI></UL><H2><SPAN class="mw-headline" id="2013-06-28"> 2013-06-28 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Gel electrophoresis of the PCR amplificate. Lane 1: NEB 2-log ladder; lane 2: 1 µl of PCR amplificate</DIV></DIV></DIV><UL><LI> run PCR of 4 ng pLF03 (0.2 µl of 21 ng/µl miniPrep from 2013-06-11) with primers IK07 and IK08 using Q5 polymerase (20 µl total volume, using hot start) on cycler 2:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 30
</TD></TR><TR><TD rowspan="3"> 30 </TD><TD> 98 </TD><TD> 5
</TD></TR><TR><TD> 66 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 150
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 1800 (30 min)
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> perfect PCR =&gt; Phusion is crappy
</LI><LI> run 4 PCRs of 4 ng pLF03 (0.2 µl of 21 ng/µl miniPrep from 2013-06-11) with primers IK07 and IK08 using Q5 polymerase (50 µl total volume, using hot start) on cycler 2:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 30
</TD></TR><TR><TD rowspan="3"> 45 </TD><TD> 98 </TD><TD> 5
</TD></TR><TR><TD> 66 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 150
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 1800 (30 min)
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="2013-06-29"> 2013-06-29 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Gel electrophoresis of the PCR amplificate. Lane 1: NEB 2-log ladder; lanes 2-5: 1 µl of PCR amplificate</DIV></DIV></DIV><UL><LI> load 1 µl of PCR product on gel -&gt; specific amplificate
</LI><LI> pool products (also include amplificate from 2013-06-28), <A href="/wiki/index.php?title=Isopropanol_PCR_purification&amp;action=edit&amp;redlink=1" class="new" title="Isopropanol PCR purification (page does not exist)">purify</A> (only first step, as no DpnI available)
</LI></UL><H2><SPAN class="mw-headline" id="2013-06-30"> 2013-06-30 </SPAN></H2><UL><LI> inoculate 1 ml LB+Amp with 10 µl of ON culture from 2013-06-25, grow at 30°C
                                </LI><LI class="active"><A href="#a10" data-toggle="tab">Lab book: genomic integration</A></LI></UL></DIV><DIV class="jumbotron"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a10"><H2><SPAN class="mw-headline" id="2013-07-01"> 2013-07-01 </SPAN></H2><UL><LI> finish <A href="/wiki/index.php?title=Isopropanol_PCR_purification&amp;action=edit&amp;redlink=1" class="new" title="Isopropanol PCR purification (page does not exist)">purification</A> (digest for 5h, dissolve in 50 µl H<SUB>2</SUB>O) -&gt; 1162 ng/µl (performed by Fanny)
</LI><LI><A href="/wiki/index.php?title=Electroporation_of_E._coli_DH10%CE%B2&amp;action=edit&amp;redlink=1" class="new" title="Electroporation of E. coli DH10β (page does not exist)">prepare electrocompetent cells</A> from ON culture (add 0.5% arabinose at dilution) (performed by Fanny)
</LI><LI> electroporate fresh electrocompetent cells, 1 aliquot from 2013-06-26 (23 µl DNA each) (performed by Fanny)
</LI><LI> grow in SOC at 30°C for 30 min, transfer to falcons, add 1 ml LB, add IPTG (1 mM), grow at 37°C for 4h
</LI><LI> spin cell down (8000 rcf for 3 min), decant supernatant, resuspend pellet in remaining medium
</LI><LI> plate on Cm+IPTG (2 plates for each sample: 10µl + rest, plates marked F: from -80°C, marked H: newly prepared cells), grow at 37°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-07-02"> 2013-07-02 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG. Lane 1: NEB 2-log; lanes 2, 3: negative control (BAP1-pLF03); lanes 4-9: colony-PCR of F-rest (lanes 8,9: from lawn); lanes 10-13: colony-PCR of H-10µl; lanes 2,4,6,8,10,12: primers IK01+IK02; lanes 3,5,7,9,11,13: primers IK01+IK03 (positive control)</DIV></DIV></DIV><UL><LI> bacteria lawn on rest-plate from -80°C, possible colonies on other plates (barely visible, may be bubbles)
</LI><LI> pick 2 colonies from F-rest, sample of lawn, 2 colonies from H-10µl, colony-PCR with primers IK01+IK02, IK01+IK03 (positive control) (20µl total volume):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 12 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 68 ↓0.5°C </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD rowspan="3"> 18 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 62 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> very evening: colonies on F-10µl, H-rest, H-10µl plates appear -&gt; leave at 37°C ON
</LI></UL><H2><SPAN class="mw-headline" id="2013-07-03"> 2013-07-03 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG. Lane 1: NEB 2-log Top: lanes 2-4: negative control (BAP1-pLF03); lanes 5-13: colony-PCR F-10µl bottom:lanes 2-4: colony-PCR of F-10µl; lanes 5-13: colony-PCR of H-10µl; lanes 2,5,11: primers IK01+IK02; lanes 3,6,12: primers IK01+IK03 (positive control); lanes 4,7,13: primers IK05+IK06 (negative control)</DIV></DIV></DIV><UL><LI> colonies on F-rest plate appeared (plate left ON at RT)
</LI><LI> pick 4 colonies from F-10µl, 3 colonies from H-10µl, colony-PCR with primers IK01+IK02, IK01+IK03 (positive control), IK05+IK06 (negative control):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 12 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 68 ↓0.5°C </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD rowspan="3"> 18 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 62 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> F-10µl 1,2,4 show no negative control -&gt; grow at 37°C
</LI><LI> continue growing plates at 37°C until evening, leave at RT ON
</LI></UL><H2><SPAN class="mw-headline" id="2013-07-04"> 2013-07-04 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG. Lane 1: NEB 2-log Top: lanes 2-10: cultures from 2013-07-03; lanes 2,5,8: primers IK01+IK02; lanes 3,6,9: primers IK01+IK03; lanes 4,7,10: primers IK05+IK06; lane 11: BAP1-pLF03 (negative control); lanes 12-13: colonies from F-10µl plate; lanes 11-13: primers IK05+IK06 bottom: lanes 2-13: primers IK05+IK06; lanes 2,3: colonies from F-10µl plate; lanes 4-6: colonies from F-rest plate; lanes 7-10: colonies from H-10µl plate; lanes 11-13: colonies from H-rest plate</DIV></DIV></DIV><UL><LI> repeat colony-PCR for 3 ambiguous cultures form 2013-07-03 (use 1 µl of culture), pick 14 new colonies (only primers IK05+IK06); 20 µl total volume:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 12 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 68 ↓0.5°C </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD rowspan="3"> 18 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 62 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> no positives
</LI></UL><H2><SPAN class="mw-headline" id="2013-07-05"> 2013-07-05 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Restriction digest of pKD46. Lane 1: NEB 2-log; lane 2: EcoRI; lane 3: BamHI+NotI; lane 4: PstI+NotI; lane 5: undigested (225 ng (5 ml) loaded)</DIV></DIV></DIV><UL><LI> digest 450 ng pKD46 (10 µl of 45.5 and 46 ng/µl miniPreps from 2013-06-06) with EcoRI (expected: 4816+1505 bp), BamHI+NotI (expected: 3675+2646 bp), PstI+NotI (expected: 3711+2363+243 bp); 30 µl total volume
</LI><LI> right plasmid
</LI><LI> inoculate 2 x 7 ml LB+Cm+IPTG(1mM) with colonies from F-10µl plate, grow at 30°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-07-06"> 2013-07-06 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Restriction digest of BAP1-pKD46 electroporated with pLF03-PCR grown in Cm+IPTG(1mM), miniprepped. Lane 1: NEB 2-log; lanes 2-5: Konrad's PCR; lane 6: digested with EcoRI; lane 8: undigested</DIV></DIV></DIV><UL><LI> add Cm (1:3000), IPTG (1:1000) to liquid cultures, grow for 8h at 30°C
</LI><LI> make miniPreps -&gt; 25 ng/µl and 14.4 ng/µl in 27.5 µl
</LI><LI> digest 25 ng/µl miniPrep with EcoRI (use all of miniPrep)
</LI><LI> load digest completely, 10 µl of 14.4 ng/µl miniPrep on gel
</LI><LI> no bands -&gt; nanoDrop crappy, no plasmid
</LI><LI> wrong strain? Streak BAP1 on Cm, grow at 37°C
                                </LI><LI class="active"><A href="#a11" data-toggle="tab">Lab book: genomic integration</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a11"><H2><SPAN class="mw-headline" id="2013-07-08"> 2013-07-08 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">gel electrophoresis of PCR of pLF03 with primers IK07 and IK08 using Q5 polymerase. Product from one PCR tube was split to two neighboring lanes.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">1 µl of gel-extracted PCR amplificate was loaded. Lane 1: NEB 2-log; lane 2: amplificate extracted by Dominik, lane 3: amplificate extracted by Ilia</DIV></DIV></DIV><UL><LI> no colonies of BAP1 on Cm
</LI><LI> repeat PCR of pLF03:
</LI><LI> run 4 PCRs of 4 ng pLF03 (0.2 µl of 21 ng/µl miniPrep from 2013-06-11) with primers IK07 and IK08 using Q5 polymerase (50 µl total volume):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 30
</TD></TR><TR><TD rowspan="3"> 45 </TD><TD> 98 </TD><TD> 5
</TD></TR><TR><TD> 66 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 150
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 1800 (30 min)
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> extract amplificate from gel (4 lanes pooled, extracted by Dominik -&gt; 119 ng/µl; 4 lanes pooled, extracted by Ilia -&gt; 111.5 ng/µl)
</LI><LI> load 1 µl of each extract on gel
</LI></UL><H2><SPAN class="mw-headline" id="2013-07-09"> 2013-07-09 </SPAN></H2><UL><LI> electroporate electrocompetent BAP1-pKD46 with 1µl, 5µl, 25µl DNA
</LI><LI> resuspend in 1 ml SOC + IPTG (1mM) + Ara (0.1%)
</LI><LI> grow for 90 min at 37°C, 400 rpm
</LI><LI> spin cells down, plate 10µl, rest of each sample on Cm+IPTG
</LI></UL><H2><SPAN class="mw-headline" id="2013-07-10"> 2013-07-10 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG with primers IK05+IK06. Lane 1: NEB 2-log; lanes 2, 13: BAP1-pLF03 (control); lanes 3-12: colony-PCRs of large colonies from 25µl/10µl plate. Lanes 2-12: OneTaq; lane 13: iTaq</DIV></DIV></DIV><UL><LI> on all plates: large + small colonies
</LI><LI> small colonies too small to pick =&gt; pick 10 large colonies from 25µl DNA / 10 µl bacteria plate, run colony-PCR with primers IK05+IK06 (Taq, 20 µl total volume):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 12 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 68 ↓0.5°C </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD rowspan="3"> 18 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 62 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> no integration
</LI><LI> continue growing plates at 37°C
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG. Top: lane 1: NEB 2-log; lanes 2,4,6,8,10,12: primers IK05+IK06; lanes 3,5,7,9,11,13: primers IK01+IK03 (positive control); lanes 2,3: BAP1-pLF03 (control); lanes 4-7: colonies from 1µl/10µl plate; lanes 8-11: 1µl/rest; lanes 12-13: 5µl/10µl Bottom: lane 11: NEB 2-log; lanes 1,3,5,7,9: primers IK05+IK06; lanes 2,4,6,8,10: primers IK01+IK03 (positive control); lanes 1-2: 5µl/10µl; lanes 3-6: 5µl/rest; lanes 7-8: 25µl/10µl; lanes 9-10: 25µl/rest</DIV></DIV></DIV><UL><LI> when small colonies large enough: pick, run colony-PCR (OneTaq, 20µl total volume):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 12 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 68 ↓0.5°C </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD rowspan="3"> 23 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 62 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> 1st colony from 5µl/rest plate (marked 7) seems promising =&gt; grow at 37°C in 1 ml LB
</LI></UL><H2><SPAN class="mw-headline" id="2013-07-11"> 2013-07-11 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG and then in LB at 37°C. Lanes 1,4: primers IK01+IK03 (positive control); lanes 2,5: primers IK01+IK02; lanes 3,6: primers IK05+IK06; lanes 1-3: BAP1-pLF03 (control); lanes 4-6: colony 7; lane 7: NEB 2-log</DIV></DIV></DIV><UL><LI> run full colony-PCR of liquid culture (OneTaq, 20 µl total volume, use 1 µl of culture):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 12 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 68 ↓0.5°C </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD rowspan="3"> 18 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 62 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> no integration
</LI><LI> 3rd population of colonies appeared on plates (stored at RT): very small, probably satellites
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG and then in LB at 37°C with primers IK07+IK08. Lane 1: NEB 2-log; lane 2: BAP1-pLF03 (control); lane 3: large colony picked from 25µl/10µl plate; lane 4: colony marked 1 from 1µl/10µl plate; lane 5: colony marked 6 from 5µl/10µl plate; lane 6: colony marked 7 from 5µl/rest plate.</DIV></DIV></DIV><UL><LI> run colony-PCR of liquid with primers IK07+IK08 (test for integration in wrong place) (OneTaq, 20µl total volume, use 1 µl liquid culture):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 30 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 66 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 12 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> no integration at all
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG with primers IK05+IK06. Top: lane 1: NEB 2-log; lane 2: BAP1-pLF03(control); lanes 3-9: colonies from 1µl/10µl plate; lanes 10-13: colonies from 1µl/rest plate Bottom: lane 1: NEB 2-log; lanes 2-10: colonies from 1µl/rest plate (lane 5: swiped 3rd population of extremely small colonies)</DIV></DIV></DIV><UL><LI> pick 20 new colonies, run colony-PCR with primers IK05+IK06 (iTaq, 20 µl total volume):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 12 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 68 ↓0.5°C </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD rowspan="3"> 23 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 62 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> unspecific bands (also present in colony-PCR from 2013-07-10)
</LI><LI> unspecific bands not present in BAP1-pLF03 PCR (control) and bottom lane 5 (satellite colonies) =&gt; not due to iTaq
</LI><LI> assume: some amount of satellites was picked together with colonies from plates -&gt; give band at 500 bp; partially amplified ygfG fragments prime integrated insert
</LI><LI> do <A href="/wiki/index.php?title=File:Heidelberg_YgfG-pccB-accA-catR.txt&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg YgfG-pccB-accA-catR.txt (page does not exist)">multiple sequence alignment</A> of ygfG, <A href="http://www.ncbi.nlm.nih.gov/nuccore/AF113605" class="external text" rel="nofollow">pccB</A>, <A href="http://www.ncbi.nlm.nih.gov/nuccore/AF113603" class="external text" rel="nofollow">accA1</A>, and <A href="http://www.ncbi.nlm.nih.gov/nuccore/AY048742" class="external text" rel="nofollow">catR</A> using ClustalO =&gt; long enough stretches of partially identical sequences present
</LI><LI> add 1mM IPTG to cultures with colonies from 1µl/10µl plate picked today (grew in 500 µl LB at 37°C for several hours), after 2h at 37°C: streak on Cm+IPTG
</LI><LI> pick 10 colonies each from 1µl/10µl and 1µl/rest plates, transfer to new Cm+IPTG plate, grow at 37°C
</LI><LI> pick 1 colony from 1µl/10µl plate, inoculate 1 ml LB + Cm + IPTG (1mM)
</LI></UL><H2><SPAN class="mw-headline" id="2013-07-12"> 2013-07-12 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG with primers IK01+IK02. Top: lane 1: NEB 2-log; lane 2: BAP1-pLF03(control); lanes 3-12: colonies liquid culture that were plated on Cm+IPTG; lane 13: colony from 1µl/10µl plate that was transferred to new Cm+IPTG plate Bottom: lane 1: NEB 2-log; lanes 2-6: colonies from 1µl/10µl plate that were transferred to new Cm+IPTG plate; lanes 7-12: colonies from 1µl/rest plate that were transferred to new Cm+IPTG plate; lane 13: liquid culture (Cm+IPTG(1mM)) from 1µl/10µl plate</DIV></DIV></DIV><UL><LI> colonies grew slowly
</LI><LI> run colony-PCR with primers IK01+IK02 (iTaq, 20 µl total volume):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 12 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 68 ↓0.5°C </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD rowspan="3"> 23 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 62 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> no bands
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG, grown in liquid culture and then transferred to Cm+IPTG with primers IK05+IK06. Lane 1: NEB 2-log; lane 2: BAP1-pLF03 (control); lanes 3-4: colony-PCRs (from colonies marked 1 and 2)</DIV></DIV></DIV><UL><LI> run colony-PCR with primers IK05+IK06 of 2 colonies from liquid culture that were transferred to Cm + IPTG plate (iTaq, 20µl total volume):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 12 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 68 ↓0.5°C </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD rowspan="3"> 23 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 62 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> band at 1.2 kb is now specific -&gt; WTF?!?
</LI></UL><H2><SPAN class="mw-headline" id="2013-07-13"> 2013-07-13 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG, grown in liquid culture and then transferred to Cm+IPTG (marked 2) with primers IK05+IK06. Lane 1: NEB 2-log; lanes 2,4,6,8,10,12: BAP1-pLF03 (control); lanes 3,5,7,9,11,13: colony-PCR; lanes 2,3: annealing at 63°C; lanes 4,5: 64°C; lanes 6,7: 65°C; lanes 8,9: 66°C; lanes 10,11: 67°C; lanes 12,13: 68°C.</DIV></DIV></DIV><UL><LI> need to determine whether 1.2 kb band is specific: run gradient PCR of colony 2 (iTaq, 20 µl total volume) with primers IK05+IK06:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 63-68 (ΔT = 1) </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> intensity of 1.2kb band decreases, intensity of 0.5 kb band (control) does not -&gt; 1.2 kb band is unspecific product
</LI><LI> check for presence of genomically integrated insert: run colony-PCR of colony 2 with primers IK07+IK08 (iTaq, 20 µl total volume):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 66 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 12 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> transfer some of colony 2 on Amp (check for presence of pLF03 as a whole), grow at 37°C
</LI><LI> transfer some of colony 2 to liquid culture: LB + Cm + IPTG (1mM), grow at 37°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-07-14"> 2013-07-14 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG, grown in liquid culture and then transferred to Cm+IPTG (marked 2) with primers IK07+IK08. Lane 1: NEB 2-log; lane 2: BAP1-pLF03 (control); lane 3: colony-PCR</DIV></DIV></DIV><UL><LI> bacteria grew on Amp =&gt; 3 possibilities:
</LI><LI> Amp plates not selective
</LI><LI> bacteria have complete pLF03 plasmid
</LI><LI> bacteria still have pKD46, although grown at 37°C
</LI></UL><LI> plate BAP1, BAP1-pKD46 on Amp, grow at 37°C
</LI><LI> colony-PCR did not work for the control -&gt; repeat with OneTaq (use 1 µl of liquid culture)
</LI><LI> no amplificate for colony: possibly, Taq has problems amplifying 4.8 kb from genomic DNA
</LI><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG, grown in liquid culture and then transferred to Cm+IPTG (marked 2) and grown in liquid culture with Cm+IPTG(1mM). Lane 3: NEB 2-log; lanes 1,4: BAP1-pLF03 (control); lanes 2,5: 1 µl of liquid culture; lanes 1,2: primers IK07+IK08; lanes 4,5: primers IK01+IK06</DIV></DIV></DIV><UL><LI> run colony-PCR with primers IK01+IK06 (OneTaq, 20 µl total volume, use 1 µl of liquid culture):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 12 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 68 ↓0.5°C </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 240
</TD></TR><TR><TD rowspan="3"> 23 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 62 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 240
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> control shows bright band where expected, colony does not =&gt; something is integrated, unknown what
                                </LI><LI class="active"><A href="#a12" data-toggle="tab">Lab book genomic integration</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a12"><H2><SPAN class="mw-headline" id="2013-07-15"> 2013-07-15 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG, grown in liquid culture and then transferred to Cm+IPTG (marked 2) and grown in liquid culture with Cm+IPTG(1mM) with primers IK07+IK08. Lane 1: NEB 2-log; lane 2: BAP1-pLF03 (control); lane 3: 1 µl of liquid culture.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG, grown in liquid culture and then transferred to Cm+IPTG (marked 2) with primers IK07+IK08. Lane 1: NEB 2-log; lane 2: BAP1-pLF03 (control); lane 3: 1 µl of liquid culture.</DIV></DIV></DIV><UL><LI> BAP1 did not grow on Amp, BAP1-pKD46 did grow on Amp at 37°C =&gt; no discrimination between pLF03 and pKD46 possible
</LI><LI> Taq might have problems amplifying 4.8 kb from genomic DNA =&gt; use Phusion Flash (primers IK07+IK08, 20 µl total volume, use 1 µl of liquid culture, old BioRad cycler):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 66 </TD><TD> 5
</TD></TR><TR><TD> 72 </TD><TD> 100
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 12 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> control did not work -&gt; repeat in new BioRad cycler (pick from plate)
</LI><LI> same result
</LI></UL><H2><SPAN class="mw-headline" id="2013-07-18"> 2013-07-18 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG, grown in liquid culture and then transferred to Cm+IPTG (marked 2). Lane 1: NEB 2-log; lanes 2,4,6: BAP1-pLF03 (control); lanes 3,5,7: colony-PCR. Lanes 2,3: primers IK24+IK25; lanes 4,5: primers IK07+IK25; lanes 6,7: primers IK01+IK25</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG, grown in liquid culture and then transferred to Cm+IPTG (marked 2). Lane 1: NEB 2-log; lanes 2,4,6: BAP1-pLF03 (control); lanes 3,5,7: colony-PCR. Lanes 2,3: primers IK24+IK25; lanes 4,5: primers IK07+IK25; lanes 6,7: primers IK01+IK25</DIV></DIV></DIV><UL><LI> run colony-PCR with primers IK24, IK25 (OneTaq, 20 µl total volume):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 12 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 68 ↓0.5°C </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 180
</TD></TR><TR><TD rowspan="3"> 23 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 62 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 180
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 12 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> 1.2 kb band with primers IK01+IK25 in control =&gt; makes no sense, repeat
</LI><LI> same result
</LI><LI> transfer BAP1-pLF03 to new Cm+IPTG plate, colony 2 to LB w/o antibiotics, grow at 37°c#
                                </LI><LI class="active"><A href="#a13" data-toggle="tab">Lab book: genomic integration</A></LI><LI><A href="#b13" data-toggle="tab">Overview: plasmid construction</A></LI><LI><A href="#c13" data-toggle="tab">Lab book: plasmid construction</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a13"><H2><SPAN class="mw-headline" id="2013-07-22"> 2013-07-22 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG, grown in liquid culture and then transferred to Cm+IPTG (marked 2). Lane 1: NEB 2-log; lanes 2,4: BAP1-pLF03 (control); lanes 3,5: colony-PCR. Lanes 2,3: primers IK01+IK03; lanes 4,5: primers RB43+RB44</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG, grown in liquid culture and then transferred to Cm+IPTG (marked 2) with primers RB43+RB44. Lane 1: NEB 2-log; lane 2: BAP1-pLF03 (control); lane 3: colony-PCR; lane 4: streaked BAP1 by Konrad</DIV></DIV></DIV><UL><LI> no positive PCR of colony yet: run colony-PCR with primers IK01+IK03 (iTaq, 20 ml total volume):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 12 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 68 ↓0.5°C </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 180
</TD></TR><TR><TD rowspan="3"> 23 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 62 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 180
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 12 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> primers RB43+RB44 (against sfp) iTaq, 20 µl total volume:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 55 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 180
</TD></TR><TR><TD> 1  </TD><TD> 12 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> unspecific bands in colony, no sfp bands at all
</LI><LI> re-run sfp PCR (primers RB43+RB44), include streaked BAP1 by Konrad
</LI><LI> sfp present in BAP1, not present in BAP1-pLF03, colony
</LI><LI> makes no sense, Konrad streaked his cells from the same batch of competent cells
</LI><LI> transfer BAP1, BAP1-pLF03, colony to liquid culture LB, grow at 37°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-07-23"> 2013-07-23 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 electroporated with PCR amplificate of pLF03 grown on Cm+IPTG, grown in liquid culture and then transferred to Cm+IPTG (marked 2) and grown in liquid culture with primers RB43+RB44. Lane 1: NEB 2-log; lane 2: BAP1-pLF03 (control); lane 3: colony-PCR; lane 4: streaked BAP1 by Konrad, lane 5: BAP1-pMM64-pMM65 from 2013-06-11</DIV></DIV></DIV><UL><LI> run colony-PCR with primers RB43+RB44 (iTaq, 20 µl total volume, use 1 µl of liquid culture)
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 55 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 180
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 12 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> no sfp present
                                <DIV class="tab-pane active" id="b13"><P style="font-size:12pt; text-align:justify;">
                                == Goal ==
</P></DIV></LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Vector map of the <A href="/wiki/index.php?title=File:Heidelberg_pIK1.gb&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg pIK1.gb (page does not exist)">pIK1 plasmid</A>.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Vector map of the <A href="/wiki/index.php?title=File:Heidelberg_pIK2.gb&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg pIK2.gb (page does not exist)">pIK2 plasmid</A>.</DIV></DIV></DIV><P>Back-up plan in case the genomic integration does not work. Two plasmids are to be constructed, pIK1 (containing BBa_I746200, pccB-accA, and sfp) and pIK2 (containing pccB-accA and sfp) in the pSB3C5 backbone. 6 fragments are to be amplified:
</P><TABLE class="wikitable"><TBODY><TR><TH> fragment ID </TH><TH> source </TH><TH> primers </TH><TH> size [kb]
</TH></TR><TR><TD> f1 </TD><TD> BBa_J04450 in pSB3C5 </TD><TD> IK32+IK33 </TD><TD> 3
</TD></TR><TR><TD> f2 </TD><TD> BBa_I746200 </TD><TD> IK26+IK27 </TD><TD> 2.3
</TD></TR><TR><TD> f3 </TD><TD> pccB-accA in pLF03 </TD><TD> IK28+IK29 </TD><TD> 3.4
</TD></TR><TR><TD>f4 </TD><TD> sfp in BAP1 </TD><TD> IK30+IK31 </TD><TD> 0.7
</TD></TR><TR><TD> f5 </TD><TD> BBa_J04450 in pSB3C5 </TD><TD> IK32+IK35 </TD><TD> 3
</TD></TR><TR><TD> f6 </TD><TD> pccB-accA in pLF03 </TD><TD> IK34+IK29 </TD><TD> 3.4
</TD></TR></TBODY></TABLE></DIV><DIV class="tab-pane active" id="c13"><H2><SPAN class="mw-headline" id="2013-07-22_2"> 2013-07-22 </SPAN></H2><UL><LI> transform BBa_I746200 in pSB1AK3 (Spring 2012 Distribution Plate 2 Well 15E) in TOP10, plate on Amp
</LI><LI> transform BBa_J04450 in pSB3C5 (Spring 2012 Distribution Plate 1 Well 3C) in TOP10, plate on Cm
</LI></UL><H2><SPAN class="mw-headline" id="2013-07-23_2"> 2013-07-23 </SPAN></H2><UL><LI> pick TOP10-BBa_I746200 colony, inoculate LB+Kan, grow at 37°C
</LI><LI> pick TOP10-BBa_J04450 colony, inoculate LB+Cm, grow at 37°C
</LI><LI> after 12 hours: add new antibiotics to medium, continue growing at 37°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-07-24"> 2013-07-24 </SPAN></H2><UL><LI><A href="/wiki/index.php?title=Preparing_glycerol_stocks&amp;action=edit&amp;redlink=1" class="new" title="Preparing glycerol stocks (page does not exist)">Prepare glycerol stocks</A></LI><LI> make miniPreps: 11.1 ng/µl (BBa_J04450), 18 ng/µl (BBa_I746200)
</LI></UL><H2><SPAN class="mw-headline" id="2013-07-25"> 2013-07-25 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><A href="/File:Methylmalonyl-PCR2013-07-25_1.png" title="File:Methylmalonyl-PCR2013-07-25 1.png">File:Methylmalonyl-PCR2013-07-25 1.png</A><DIV class="thumbcaption">Lane 1: NEB 2-log; lane 3: f1; lane 4: f2; lane 5: f3; lane 6: f4; lane 7: f5; lane 8: f6</DIV></DIV></DIV><UL><LI> run PCRs (using Q5, 20 µl total volume) using 4 ng of each template (pLF03 miniPrep from 2013-06-11, 21 ng/µl), sfp from Konrad's BAP1 plate:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 30 (f1,f2,f3,f5,f6) / 300 (f4)
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 98 </TD><TD> 5
</TD></TR><TR><TD> 66 (f1,f2) / 60 (f4) / 61 (f3,f5,f6) </TD><TD> 10
</TD></TR><TR><TD> 72 </TD><TD> 120 (f1,f2,f3,f5,f6) / 30(f4)
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 12 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> did not manage to get f3, f4, f6
</LI><LI> repeat: add 1 µl DMSO to reactions:
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><A href="/File:Methylmalonyl-PCR2013-07-25_2.png" title="File:Methylmalonyl-PCR2013-07-25 2.png">File:Methylmalonyl-PCR2013-07-25 2.png</A><DIV class="thumbcaption">Lane 1: NEB 2-log; lane 2: f3; lane 3: f4 with lots of bacteria from new plate; lane 4: f4 with lots of bacteria from old plate; lane 5: f4 with little bacteria from new plate; lane 6: f6</DIV></DIV></DIV><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 30
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 98 </TD><TD> 5
</TD></TR><TR><TD> 56 (f4) / 57 (f3,f6) </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 150 (f3,f6) / 45(f4)
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 10 </TD><TD> inf
</TD></TR></TBODY></TABLE><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><A href="/File:Methylmalonyl-PCR2013-07-25_3.png" title="File:Methylmalonyl-PCR2013-07-25 3.png">File:Methylmalonyl-PCR2013-07-25 3.png</A><DIV class="thumbcaption">Lane 1: NEB 2-log; lane 2: f3; lane 3: f6</DIV></DIV></DIV><UL><LI> use pLF03 miniPrep from 2013-06-11, 27 ng/µl, repeat for f3, f6 (with 1 µl DMSO):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 30
</TD></TR><TR><TD rowspan="3"> 12 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD> 62 ↓0.5°C </TD><TD> 10
</TD></TR><TR><TD> 72 </TD><TD> 150
</TD></TR><TR><TD rowspan="3"> 23 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD> 58 </TD><TD> 10
</TD></TR><TR><TD> 72 </TD><TD> 150
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 10 </TD><TD> inf
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="2013-07-26"> 2013-07-26 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><A href="/File:Methylmalonyl-PCR2013-07-26.png" title="File:Methylmalonyl-PCR2013-07-26.png">File:Methylmalonyl-PCR2013-07-26.png</A><DIV class="thumbcaption">Lane 1: NEB 2-log; lane 2: f3 with annealing; lane 3: f6 with annealing; lane 4: f6 2-step; lane 5: f3 2-step</DIV></DIV></DIV><UL><LI> run PCR of f3, f6 with Phusion Flash (using pLF03 miniPrep from 2013-06-11, 21 ng/µl), without DMSO:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 65 </TD><TD> 5
</TD></TR><TR><TD> 72 </TD><TD> 75
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 10 </TD><TD> inf
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD rowspan="3"> 5 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 65 </TD><TD> 5
</TD></TR><TR><TD> 72 </TD><TD> 75
</TD></TR><TR><TD rowspan="2"> 30 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 72 </TD><TD> 75
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 10 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> inoculate 10 ml LB+Amp with TOP10-pLF03, grow for 36h at 37°C, add Amp every 12 hours
</LI></UL><H2><SPAN class="mw-headline" id="2013-07-28"> 2013-07-28 </SPAN></H2><UL><LI> make miniPrep -&gt; very low yield (ca. 1-2 ng/µl)
</LI><LI> inoculate 6 ml LB+Amp with TOP10-pLF03, grow at 37°C
                                </LI><LI class="active"><A href="#a14" data-toggle="tab">Lab book: genomic integration</A></LI><LI><A href="#b14" data-toggle="tab">Lab book: plasmid construction</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a14"><H2><SPAN class="mw-headline" id="2013-07-29"> 2013-07-29 </SPAN></H2><UL><LI> inoculate 10 ml 2xYT+Amp with TOP10-pKD46, grow at 30°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-07-31"> 2013-07-31 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">miniPrep of pKD46. Lane 1: NEB 2-log (5 µl), lane 2: 1 µl of miniPrep</DIV></DIV></DIV><UL><LI> miniPrep of pKD46 -&gt; ca. 10 ng/µl
</LI></UL><H2><SPAN class="mw-headline" id="2013-08-03"> 2013-08-03 </SPAN></H2><UL><LI> transform BAP1 with 5µl of pKD46 miniPrep form 2013-07-31, plate on Amp, grow at 30°C
                                </LI></UL><H2><SPAN class="mw-headline" id="2013-07-29_2"> 2013-07-29 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">1 µl of pLF03 miniPrep from 2013-06-04</DIV></DIV></DIV><UL><LI> make miniPrep -&gt; very low yield (1-2 ng/µl)
</LI><LI> load 1µl of 97ng/µl pLF03 miniPrep from 2013-06-04 on gel -&gt; ca. 20 ng/µl
</LI><LI> inoculate 50 ml LB+Amp with TOP10-pLF03, grow at 37°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-07-30"> 2013-07-30 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">1 µl of pLF03 midiPrep from 2013-07-30</DIV></DIV></DIV><UL><LI> make midiPrep -&gt; 1352 ng/µl in 66.5 µl
</LI><LI> medium was contaminated (also check with Fanny) -&gt; low yields of previous miniPreps
</LI></UL><H2><SPAN class="mw-headline" id="2013-07-31_2"> 2013-07-31 </SPAN></H2><UL><LI> send 20 µl of 1:15 dilution of midiPrep to sequencing with primers IK07, IK36, IK37: <A href="/File:Heidelberg_PLF03_seq_2013-07-31.zip" title="File:Heidelberg PLF03 seq 2013-07-31.zip">File:Heidelberg PLF03 seq 2013-07-31.zip</A></LI><LI> perform multiple sequence alignment against assumed pLF03 sequence: <A href="/wiki/index.php?title=File:Heidelberg_PLF03-2013-07-31_IK07.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PLF03-2013-07-31 IK07.clustal.txt (page does not exist)">IK07</A>, <A href="/wiki/index.php?title=File:Heidelberg_PLF03-2013-07-31_IK36.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PLF03-2013-07-31 IK36.clustal.txt (page does not exist)">IK36</A>, <A href="/wiki/index.php?title=File:Heidelberg_PLF03-2013-07-31_IK37.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PLF03-2013-07-31 IK37.clustal.txt (page does not exist)">IK37</A></LI><LI> deletion at the end of accA2 -&gt; primer can not bind
                                </LI><LI class="active"><A href="#a15" data-toggle="tab">Lab book: genomic integration</A></LI><LI><A href="#b15" data-toggle="tab">Lab book: plasmid construction</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a15"><H2><SPAN class="mw-headline" id="2013-08-05"> 2013-08-05 </SPAN></H2><UL><LI> run colony-PCR against sfp on BAP1-pKD46 with primers RB43+RB44 (iTaq, 20 µl total volume):
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Colony-PCR of BAP1-pKD46 with primers RB43+RB44. Lane 1: sfp, lane 2: NEB 2-log</DIV></DIV></DIV><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 55 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 180
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 10 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> inoculate 4 ml 2xYT+Amp with BAP1-pKD46, grow at 30°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-08-06"> 2013-08-06 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Lane 1: NEB 2-log; lane 3: colony-PCR with primers RB43+RB44</DIV></DIV></DIV><UL><LI> inoculate 50 ml 2xYT+Amp+Ara(0.5%) with BAP1-pKD46, grow at 30°C until OD=0.71
</LI><LI> run colony-PCR with primers RB43+RB44 from 50 ml culture:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 55 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 180
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 10 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI><A href="/wiki/index.php?title=Electrocompetent_bacteria&amp;action=edit&amp;redlink=1" class="new" title="Electrocompetent bacteria (page does not exist)">prepare electrocompetent</A> BAP1-pKD46
</LI></UL><H2><SPAN class="mw-headline" id="2013-08-08"> 2013-08-08 </SPAN></H2><UL><LI> electroporate BAP1-pKD46 with 500 ng DNA (5 µl of PCR amplificate from 2013-07-08)
</LI><LI> grow in SOC + IPTG(1mM) + Arabinose (0.5%) at 37°C, 400 rpm for 3h
</LI><LI> spin cells down (3 min at 8000 rcf)
</LI><LI> plate 10 µl, rest on Cm, grow at 37°C
                                </LI></UL><H2><SPAN class="mw-headline" id="2013-08-05_2"> 2013-08-05 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Lane 1: NEB 2-log; lane 2: f3; lane 3: f6</DIV></DIV></DIV><UL><LI> received constructs from Matthew Mattozzi, Boston: pET21c-pcc-acca-mcce in NEB Turbo (Amp<SUP>r</SUP>), pWH8: mcc-pcs-pcc-acc-mmce in NEB Turbo (Kan<SUP>r</SUP>), E. coli MM17: MG1655 lambda(DE3) d(lacY) d(E14) mcr-pcs-pcc-acc::KanR integrated at phage site HK022 (Kan<SUP>r</SUP>)
</LI><LI> plates contaminated with <I>Drosophila</I> larvae
</LI><LI> re-streak on fresh plates, run colony-PCR for f3, f6 from pET21c-pcc-acca-mcce (Q5, 20 µl total volume):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 120
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 98 </TD><TD> 5
</TD></TR><TR><TD> 61 </TD><TD> 10
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 10 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> no bands
</LI></UL><H2><SPAN class="mw-headline" id="2013-08-06_2"> 2013-08-06 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Lane 1: NEB 2-log; lane 2: f3; lane 3: f6</DIV></DIV></DIV><UL><LI> run colony-PCR from pET21c-pcc-acca-mcce grown ON (Q5, 20 µl total volume) for fragments f3, f6:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 60
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 98 </TD><TD> 5
</TD></TR><TR><TD> 61 </TD><TD> 10
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 10 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> inoculate 5 ml 2xYT+Amp with colony from which colony-PCR was run, grow at 37°C
</LI><LI> no fragment for f3, re-run PCR (1 µl of liquid culture, 20 µl total, Phusion Flash):
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Lane 1: NEB 2-log; lane 2: f3; lane 3: from genomic integration; lane 4: 2 µl of pET21c-pcc-acc-mmce miniPrep.</DIV></DIV></DIV><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 60
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 65 </TD><TD> 5
</TD></TR><TR><TD> 72 </TD><TD> 60
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 10 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> make miniPrep of liquid culture
</LI><LI> no PCR product, miniPrep runs above 10 kb supercoiled (whole plasmid is expected to have 9 kb) (miniPrep has ca. 10 ng/µl)
</LI><LI> re-run PCR for f3 from miniPrep (Q5, 0.2 µl DNA, 20 µl total volume):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 60
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 98 </TD><TD> 5
</TD></TR><TR><TD> 61 </TD><TD> 10
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 10 </TD><TD> inf
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="2013-08-07"> 2013-08-07 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Lane 1: NEB 2-log; lane 2: f3; lane 3: BamHI digest of pET21c-pcc-acc-mcce; lane 4: 1 kb plus ladder.</DIV></DIV></DIV><UL><LI> need to verify plasmid identity: linearize with BamHI (unique restriction site in both pET21c... and pWH8, use 5 µl of miniPrep, 30 µl total volume)
</LI><LI> linearized pET21c... plasmid runs at 9 kb -&gt; right
</LI><LI> PCR of f3 worked -&gt; hypothesis: Gibson overhang against permeability device bound to <I>E. coli</I> genome in colony-PCRs, as permeability device is mutated version of FepA, which is present in <I>E. coli</I></LI><LI> gel-purify all fragments
</LI><LI> f1: 54.2 ng/µl, f2: 64.4 ng/µl, f3: 24.4 ng/µl, f4: 14 ng/µl, f5: 47.6 ng/µl, f6: 25.2 ng/µl; all in 30 µl
</LI><LI> perform Gibson assembly:
</LI><LI> pIK1: 0.8 µl f1, 1.6 µl f2, 5 µl f3, 2 µl f4, 0.4 µl H<SUB>2</SUB>O, 10 µl NEB Gibson MM
</LI><LI> pIK2: 1 µl f5, 6 µl f6, 3 µl f4
</LI><LI> incubate at 50°C for 1h
</LI></UL><LI> transform TOP10: 2 transformations per plasmid: 5 µl of Gibson assembly, 5 µl of 1:4 diluted Gibson assembly
</LI><LI> grow in 2xYT at 37°C, plate on Cm, grow at 37°C
</LI><H2><SPAN class="mw-headline" id="2013-08-08_2"> 2013-08-08 </SPAN></H2><UL><LI> pIK1: several colonies, some of them pink -&gt; pick 10 white, grow in 2xYT+Cm at 37°C
</LI><LI> pIK2: 2 populations of colonies: large and small, several large colonies are pink -&gt; pick 5 white large, 5 white small colonies, grow in 3 ml 2xYT+Cm at 37°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-08-09"> 2013-08-09 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Lane 1: NEB 2-log; lanes 2-10: pIK1 miniPreps digested with BamHI+HindIII; lanes 11-13: large colonies of pIK2 miniPreps digested with BamHI+HindIII</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Lane 1: NEB 2-log; lanes 2-3: large colonies of pIK2 miniPreps digested with BamHI+HindIII; lanes 4-8: small colonies of pIK2 miniPreps digested with BamHI+HindIII; lanes 9-13: Flo's PCR</DIV></DIV></DIV><UL><LI> colony 8 of pIK1 did not grow
</LI><LI> make miniPreps of 2 ml everything else (2 ml each) -&gt; ca. 62 - 600 ng/µl
</LI><LI> miniPrep yields quite high, but consistent: pACYC with p15A origin (4 kb) <A href="http://www.abgent.com.cn/downloads/BG0003-Plasmid-mini-kit.pdf" class="external text" rel="nofollow">yields 0.6 µg DNA / ml culture</A> in LB, culture in 2xYT/TB yields 2-5 times more cells (according to QiaGen) =&gt; ca. 10-15 µg expected for 9 kb plasmid
</LI><LI> digest with BamHI+HindIII (20 µl total volume, 0.5 µl enzyme, 1 µl DNA [except for sample with 62 ng/µl, there 2 µl DNA was used])
</LI><LI> expected fragments: 2151 bp + 7265 bp for pIK1, 2151 bp + 5029 bp for pIK2
</LI><LI> send pIK1.2 and pIK2.2 to sequencing with primers VF2 and VR
                                </LI><LI class="active"><A href="#a16" data-toggle="tab">Lab book: genomic integration</A></LI><LI><A href="#b16" data-toggle="tab">Lab book: plasmid construction</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a16"><H2><SPAN class="mw-headline" id="2013-08-12"> 2013-08-12 </SPAN></H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Lane 1: NEB 2-log; lanes 2-4: Colony-PCRs with primers IK01+IK02</DIV></DIV></DIV><UL><LI> no colonies on 10 µl plate
</LI><LI> 2 large colonies, 1 small colony on rest plate
</LI><LI> run colony-PCR with primers IK01+IK02:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 12 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 68 ↓0.5°C </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD rowspan="3"> 23 </TD><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 62 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 12 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> genomic integration failed
                                <DIV class="tab-pane" id="b16"><P style="font-size:12pt; text-align:justify;">
                                     == 2013-08-12 ==
</P></DIV></LI><LI> sequences arrived: <A href="/File:PIK1.2-pIK2.2_seq_2013-08-12.zip" title="File:PIK1.2-pIK2.2 seq 2013-08-12.zip">File:PIK1.2-pIK2.2 seq 2013-08-12.zip</A></LI><LI> perform multiple sequence alignment against assumed sequences: <A href="/wiki/index.php?title=File:PIK1.2-2013-08-12_VF2.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:PIK1.2-2013-08-12 VF2.clustal.txt (page does not exist)">pIK1.2 with VF2</A>, <A href="/wiki/index.php?title=File:PIK1.2-2013-08-12_VR.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:PIK1.2-2013-08-12 VR.clustal.txt (page does not exist)">pIK1.2 with VR</A>, <A href="/wiki/index.php?title=File:PIK2.2-2013-08-12_VF2.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:PIK2.2-2013-08-12 VF2.clustal.txt (page does not exist)">pIK2.2 with VF2</A>, <A href="/wiki/index.php?title=File:PIK2.2-2013-08-12_VR.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:PIK2.2-2013-08-12 VR.clustal.txt (page does not exist)">pIK2.2 with VR</A></LI><LI> pIK1.2 looks good
</LI><LI> part of RBS missing in pIK2.2 -&gt; makes no sense, as part of binding part of IK35, with which the backbone was amplified
</LI><LI> send pIK2.6 to sequencing with primers VF2 and VR, also sequence pIK1.2 with primer IK25 to find out whether RBS is also missing there
</LI></UL><H2><SPAN class="mw-headline" id="2013-08-13"> 2013-08-13 </SPAN></H2><UL><LI> sequences arrived: <A href="/File:PIK1.2-pIK2.6_seq_2013-08-13.zip" title="File:PIK1.2-pIK2.6 seq 2013-08-13.zip">File:PIK1.2-pIK2.6 seq 2013-08-13.zip</A></LI><LI> perform multiple sequence alignment against assumed sequences: <A href="/wiki/index.php?title=File:PIK1.2-2013-08-13_IK25.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:PIK1.2-2013-08-13 IK25.clustal.txt (page does not exist)">pIK1.2 with IK25</A>, <A href="/wiki/index.php?title=File:PIK2.6-2013-08-13_VF2.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:PIK2.6-2013-08-13 VF2.clustal.txt (page does not exist)">pIK2.6 with VF2</A>, <A href="/wiki/index.php?title=File:PIK2.6-2013-08-13_VR.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:PIK2.6-2013-08-13 VR.clustal.txt (page does not exist)">pIK2.6 with VR</A></LI><LI> sequences look good
</LI><LI><A href="/wiki/index.php?title=Preparing_glycerol_stocks&amp;action=edit&amp;redlink=1" class="new" title="Preparing glycerol stocks (page does not exist)">prepare glycerol stock</A> of TOP10-pIK1.2
</LI></UL><H2><SPAN class="mw-headline" id="2013-08-14"> 2013-08-14 </SPAN></H2><UL><LI><A href="/wiki/index.php?title=Preparing_glycerol_stocks&amp;action=edit&amp;redlink=1" class="new" title="Preparing glycerol stocks (page does not exist)">prepare glycerol stock</A> of TOP10-pIK2.6
                                </LI><LI class="active"><A href="#a17" data-toggle="tab">Lab book: plasmid construction</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a17"><H2><SPAN class="mw-headline" id="2013-08-19"> 2013-08-19 </SPAN></H2><UL><LI> received BL21(DE3) from Gruss Lab, ZMBH
</LI><LI> grow in LB to OD=0.53, <A href="/wiki/index.php?title=Heat-shock_competent_bacteria&amp;action=edit&amp;redlink=1" class="new" title="Heat-shock competent bacteria (page does not exist)">prepare competent cells</A></LI><LI> co-transform with pIK1.2(0.2 µl = 120 ng)+pMM64(1 µl = 15 ng), pIK2.6(0.2 µl = 60 ng)+pMM64(1 µl = 15 ng), plate on Cm+Amp+IPTG, grow at 37°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-08-20"> 2013-08-20 </SPAN></H2><UL><LI> all colonies white
</LI><LI> pick 1 colony each, inoculate LB+Cm+Amp, grow at 37°C until cultures are opaque, add IPTG to 1mM, grow at 30°C
</LI><LI> no visible indigoidine production
</LI><LI> need to clone into a different backbone in order to be able to cotransform with pRB21
</LI><LI> inoculate LB+Kan with TOP10-pSB3K3-BBa_J04450, grow at 37°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-08-21"> 2013-08-21 </SPAN></H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Lane 1: NEB 2-log; lanes 2-3: pSB3K3-BBa_J04450 digested with EcoRI+SpeI; lane 4: pIK1.2 digested with EcoRI+SpeI; lane 5: pIK2.6 digested with EcoRI+SpeI</DIV></DIV></DIV><UL><LI><A href="/wiki/index.php?title=Preparing_glycerol_stocks&amp;action=edit&amp;redlink=1" class="new" title="Preparing glycerol stocks (page does not exist)">prepare glycerol stock</A> of TOP10-pSB3K3-BBa_J04450
</LI><LI> perform miniPrep -&gt; 10.1 ng/µl in 38 µl
</LI><LI> digest with EcoRI+SpeI: 380 ng of pSB3K3-BBa_J04450: use everything (50 µl total volume, 2x0.5 µl enzyme, expect 2.7kb &amp; 1kb), 1200 ng of pIK1.2 (2 µl of miniPrep from 2013-08-09, 20 µl total volume, 2x0.5 µl enzyme, expect 6.7kb &amp; 2.7kb), 900 ng of pIK2.6 (3 µl of miniPrep from 2013-08-09, 20 µl total volume, 2x0.5 µl enzyme, expect 4.4kb &amp; 2.7kb)
</LI><LI> pSB3K3-BBa_J04450 looks as though it was not cut: inoculate 50 ml TB+Kan with TOP10-pSB3K3-BBa_J04450, grow at 37°C
</LI><LI> as control for indigoidine production from pMM64: co-transform BL21(DE3) with pMM64(1 µl = 15 ng)+pMM65(0.2 µl = 13 ng), plate on Amp+Kan+IPTG
</LI></UL><H2><SPAN class="mw-headline" id="2013-08-22"> 2013-08-22 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Lane 1: pSB3K3-BBa_J04450 digested with EcoRI+SpeI, lane 2: NEB 2-log</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Lane 1: NEB 2-log; lanes 2-9: pRB* plasmids; lane 10: 1 µl of pSB3K3-BBa_J04450 midiPrep</DIV></DIV></DIV><UL><LI> no colonies of BL21(DE3)-pMM64-pMM65: co-transform BL21(DE3) with pMM64(1 µl = 15 ng)+pMM65(1 µl = 75 ng), plate on Amp+Kan+IPTG
</LI><LI> prepare midiPrep of pSB3K3-BBa_J04450: 2952.1 ng/µl
</LI><LI> digest 0.4 µl of midiPrep with EcoRI+SpeI (20 µl total volume, 2x0.5 µl enzyme)
</LI><LI> band too weak: 2 possibilities:
</LI><LI> concentration measurement wrong
</LI><LI> mispipeted
</LI><LI> =&gt; load 1 µl of midiPrep on gel -&gt; concentration measurement about right
</LI><LI> large pSB3K3 fragment over 3 kb, consistent with miniPrep from 2013-08-21 -&gt; 2.7 kb expected, sequence wrong on parts.igem.org
</LI><LI> gel-purify pSB3K3, 6 kb fragment of pIK1, 4 kb fragment of pIK2
</LI><LI> 18 µl of pSB3K3 (not measured), 12.3 ng/µl pIK1 in 21 µl; 7.9 ng/µl pIK2 in 21 µl
</LI><LI> ligate at RT for 1h:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> what </TH><TH> µl
</TH></TR><TR><TD> pSB3K3 </TD><TD> 9
</TD></TR><TR><TD> insert </TD><TD> 8
</TD></TR><TR><TD> T4 ligase </TD><TD> 1 µl
</TD></TR><TR><TD> T4 ligase buffer </TD><TD> 2 µl
</TD></TR></TBODY></TABLE><UL><LI> heat-inactivate: 75°C for 5 min
</LI><LI> transform 10 µl of each ligation into TOP10, plate on Kan, grow at 37°C (pSB3K3+pIK2 = pIK6)
</LI></UL><H2><SPAN class="mw-headline" id="2013-08-23"> 2013-08-23 </SPAN></H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Colony-PCR of TOP10 transformed with pSB3K3 ligated with pIK1/pIK2 and primers RB43+VR. Lane 1: NEB 2-log; lanes 2-6: pIK1; lanes 7-11: pIK2</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Colony-PCR of TOP10 transformed with pSB3K3 ligated with pIK2 and primers RB43+VR. Lane 1: NEB 2-log; lanes 2-11: pIK2</DIV></DIV></DIV><UL><LI> pick 5 colonies each, run colony-PCR with primers RB43+VR (expected: 1kb, using iTaq, 20 µl total volume):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 95 </TD><TD> 30
</TD></TR><TR><TD> 54 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 90
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 10 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> no amplificate
</LI><LI> have another look at sequences (motivated by DelRest results): 1 bp missing in pIK1.2: frame shift in permeability device =&gt; send pIK1.3 to sequencing with primers VF2, VR, IK25
</LI><LI> pick 30 colonies of pSB3K3-pIK2, run colony-PCR with primers RB43+VR (pool 3 colonies in 1 PCR reaction, iTaq, 20 µl total volume)
</LI><LI> no positives =&gt; ligation did not work, most likely due to low DNA concentrations
</LI><LI> need to repeat: inoculate 2x5 ml 2xYT+Cm with TOP10-pIK2.6
</LI></UL><H2><SPAN class="mw-headline" id="2013-08-24"> 2013-08-24 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Lane 1: NEB 2-log; lane 3: pSB3K3 digested with EcoRI+SpeI; lane 5: pIK2.6 digested with EcoRI+SpeI</DIV></DIV></DIV><UL><LI> make miniPrep of pIK2.6: 97 ng/µl in 37.5 µl
</LI><LI> digest 3 µg pSB3K3 (1 µl of midiPrep from 2013-08-22), 1.7 µg pIK2.6 (17 µl of miniPrep) with EcoRI+SpeI (20 µl total volume, 0.5 µl of each enzyme)
</LI><LI> treat pSB3K3 digest with antarctic phosphatase (37°C, 60 min)
</LI><LI> run gel, gel-purify -&gt; 18 µl pSB3K3 (not measured), 15.0 ng/µl pIK2.6 in 20.5 µl
</LI><LI> ligate at RT for 1h:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> what </TH><TH> µl
</TH></TR><TR><TD> pSB3K3 </TD><TD> 9
</TD></TR><TR><TD> pIK2.6 </TD><TD> 8
</TD></TR><TR><TD> T4 ligase </TD><TD> 1 µl
</TD></TR><TR><TD> T4 ligase buffer </TD><TD> 2 µl
</TD></TR></TBODY></TABLE><UL><LI> heat-inactivate: 75°C for 5 min
</LI><LI> transform 10 µl of ligation into TOP10, plate on Kan, grow at 37°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-08-25"> 2013-08-25 </SPAN></H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Colony-PCR of TOP10 transformed with pSB3K3 ligated with pIK2 and primers RB43+VR. Lane 1: NEB 2-log; lanes 2-11: pIK2</DIV></DIV></DIV><UL><LI> pick 10 colonies, run colony-PCR with primers RB43+VR (expected: 1kb, using iTaq, 20 µl total volume):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 95 </TD><TD> 30
</TD></TR><TR><TD> 54 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 90
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 10 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> no amplificate
</LI><LI> VR primer might not be binding to pSB3K3: <A href="http://www.ccbi.cam.ac.uk/iGEM2006/index.php/Primer_List#BioBrick_verification_primers" class="external free" rel="nofollow">http://www.ccbi.cam.ac.uk/iGEM2006/index.php/Primer_List#BioBrick_verification_primers</A> , <A href="https://2009.igem.org/Team:Groningen/Notebook/22_July_2009" class="external free" rel="nofollow">https://2009.igem.org/Team:Groningen/Notebook/22_July_2009</A></LI><LI> pick another 10 colonies, run colony-PCR with primers VF2+IK25 (expected: 1kb, iTaq, 20 µl total volume):
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Colony-PCR of TOP10 transformed with pSB3K3 ligated with pIK2 and primers VF2+IK25. Lane 1: NEB 2-log; lanes 2-11: pIK2</DIV></DIV></DIV><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 95 </TD><TD> 30
</TD></TR><TR><TD> 54 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 90
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 10 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> definite product, but at 1.3 kb =&gt; pSB3K3 was longer than expected in digest, might be a result from that
</LI><LI> unspecific bands probably due to melting temperature of IK25 being at 67°C
</LI><LI> inoculate 2xYT+Kan, grow at 37°C
</LI><LI> pSB3K3 used was from plate 5 =&gt; transform TOP10 with pSB3K3 from plate 2, well 6 F
                                </LI><LI class="active"><A href="#a18" data-toggle="tab">Lab book: plasmid construction</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a18"><H2><SPAN class="mw-headline" id="2013-08-26"> 2013-08-26 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Lane 1: NEB 2-Log, Lane 2: pIK6.1 digested with HindIII+BamHI; lane 3: pIK6.2 digested with HindIII+BamHI</DIV></DIV></DIV><UL><LI> make miniPreps of colonies 1 and 2: 71.4 ng/µl and 75.1 ng/µl in 38 µl
</LI><LI> digest with HindIII+BamHI (3 µl DNA, 0.5 µl of each enzyme, 20 µl total); expected: 4.1 kb, 2.1 kb, 0.9 kb
</LI><LI> no 0.9 kb fragment =&gt; no BamHI site in KanR gene, 2.1 fragment visible corresponds to acca-sfp =&gt; insert completely within backbone
</LI><LI> co-transform TOP10 with pIK6.1+pRB21
</LI><LI> sequences of pIK1.3 arrived: <A href="/wiki/index.php?title=File:Heidelberg_PIK1.3-2013-08-26.zip&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK1.3-2013-08-26.zip (page does not exist)">PIK1.3-2013-08-26.zip</A>, <A href="/wiki/index.php?title=File:Heidelberg_PIK1.3-2013-08-26_VF2.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK1.3-2013-08-26 VF2.clustal.txt (page does not exist)">PIK1.3-2013-08-26 VF2.clustal.txt</A>, <A href="/wiki/index.php?title=File:Heidelberg_PIK1.3-2013-08-26_VR.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK1.3-2013-08-26 VR.clustal.txt (page does not exist)">PIK1.3-2013-08-26 VR.clustal.txt</A></LI><LI> point mutation in permeability device, interestingly, both the frameshift in pIK1.2 and this point mutation create a stop codon at the beginning of the permeability device
</LI><LI> send pIK1.4 to sequencing with primer VF2
</LI></UL><H2><SPAN class="mw-headline" id="2013-08-27"> 2013-08-27 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Lane 1: NEB 2-log, lane 2: pSB3K3-BBa_J04450 digested with EcoRI+SpeI</DIV></DIV></DIV><UL><LI> TOP10-pIK6.1-pRB21 are not blue, but the indigoidine group is having problems with pRB21 -&gt; pRB21 might not work
</LI><LI> sequence of pIK1.4 arrived: <A href="/wiki/index.php?title=File:Heidelberg_PIK1.4-2013-08-27.zip&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK1.4-2013-08-27.zip (page does not exist)">PIK1.4-2013-08-27.zip</A>, <A href="/wiki/index.php?title=File:Heidelberg_PIK1.4-2013-08-27_VF2.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK1.4-2013-08-27 VF2.clustal.txt (page does not exist)">PIK1.4-2013-08-27 VF2.clustal.txt</A></LI><LI> again frame-shift in permeability device -&gt; send pIK1.6, pIK1.7, pIK1.10 to sequencing with primer VF2
</LI><LI> make miniPrep of pSB3K3: 54.7 ng/µl in 38 µl
</LI><LI> digest pSB3K3 with EcoRI+SpeI (20 µl total volume, 5 µl DNA, 0.5 µl of each enzyme)
</LI><LI> gel-purify, ligate with pIK2.6 fragment from 2013-08-24 at RT for 1h -&gt; pIK7:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> what </TH><TH> µl
</TH></TR><TR><TD> pSB3K3 </TD><TD> 9
</TD></TR><TR><TD> pIK2.6 </TD><TD> 8
</TD></TR><TR><TD> T4 ligase </TD><TD> 1 µl
</TD></TR><TR><TD> T4 ligase buffer </TD><TD> 2 µl
</TD></TR></TBODY></TABLE><UL><LI> heat-inactivate: 75°C for 5 min
</LI><LI> transform 10 µl of ligation into TOP10, plate on Kan, grow at 37°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-08-28"> 2013-08-28 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Colony-PCR of TOP10 transformed with pSB3K3 ligated with pIK2 (-&gt; pIK7) and primers VF2+IK25. Lane 1: NEB 2-log; lanes 2-6: pIK7</DIV></DIV></DIV><UL><LI> pick colonies, run colony-PCR with primers VF2+IK25 (expected: 1kb, iTaq, 20 µl total volume):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 95 </TD><TD> 30
</TD></TR><TR><TD> 54 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 90
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 10 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> grow in 2xYT+Kan at 37°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-08-29"> 2013-08-29 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Lane 1: NEB 2-Log, Lane 2: pIK7.1 digested with HindIII+BamHI; lane 3: pIK7.2 digested with HindIII+BamHI</DIV></DIV></DIV><UL><LI> sequences of pIK1 arrived: <A href="/wiki/index.php?title=File:Heidelberg_PIK1.6-pIK1.7-pIK1.10-2013-08-29.zip&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK1.6-pIK1.7-pIK1.10-2013-08-29.zip (page does not exist)">PIK1.6-pIK1.7-pIK1.10-2013-08-29.zip</A>, <A href="/wiki/index.php?title=File:Heidelberg_PIK1.6-2013-08-29_VF2.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK1.6-2013-08-29 VF2.clustal.txt (page does not exist)">PIK1.6-2013-08-29 VF2.clustal.txt</A>, <A href="/wiki/index.php?title=File:Heidelberg_PIK1.7-2013-08-29_VF2.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK1.7-2013-08-29 VF2.clustal.txt (page does not exist)">PIK1.7-2013-08-29 VF2.clustal.txt</A>, <A href="/wiki/index.php?title=File:Heidelberg_PIK1.10-2013-08-29_VF2.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK1.10-2013-08-29 VF2.clustal.txt (page does not exist)">PIK1.10-2013-08-29 VF2.clustal.txt</A></LI><LI> frame-shift in pIK1.6, point mutations in pIK1.7, pIK1.10
</LI><LI> perform miniPreps of pIK7 -&gt; ca. 50 ng/µl in 38 µl
</LI><LI> digest 5 µl of pIK7.1, pIK7.2 with BamHI+HindIII (0.5 µl of each enzyme, 20 µl total volume)
</LI><LI> bands at right positions (expected: 4.1, 2.1, 0.8 kb)
</LI><LI><A href="/wiki/index.php?title=Preparing_glycerol_stocks&amp;action=edit&amp;redlink=1" class="new" title="Preparing glycerol stocks (page does not exist)">prepare glycerol stock</A> of TOP10-pIK7.1
</LI></UL><H2><SPAN class="mw-headline" id="2013-08-30"> 2013-08-30 </SPAN></H2><UL><LI> electroporate electrocompetent DH10ß with 1 µl, 14 µl of 1:4 Gibson mix for pIK1 from 2013-08-07
</LI><LI> plate 10 µl, rest of each electroporation on Cm, grow at 37°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-08-31"> 2013-08-31 </SPAN></H2><UL><LI> very large and small colonies present, after heat-schock transformation last time only large colonies present
</LI><LI> pick 5 small colonies from each plate, grow in 2xYT+Cm (no 2xYT left =&gt; last 3 colonies from 14 µl / rest plate grown in LB+Cm) at 37°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-09-01"> 2013-09-01 </SPAN></H2><UL><LI> pIK1.12 did not grow
</LI><LI> make miniPreps of all cultures -&gt; concentrations about 200-300 ng/µl in 37.5 µl (except pIK1.14: 50 ng/µl)
</LI><LI> digest with BamHI+HindIII (20 µl total volume, 0.5 µl enzyme, 1 µl DNA)
</LI><LI> expected fragments: 2151 bp + 7265 bp
</LI></UL><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 185px;"><DIV class="gallerytext"><P>Lane 1: NEB 2-log; lanes 2-13: pIK1 miniPreps digested with BamHI+HindIII; lanes 2-5: 1 µl / 10 µl plate; lanes 6-10: 1 µl / rest plate; lanes 11-13: 14 µl / 10 µl plate
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 185px;"><DIV class="gallerytext"><P>Lane 1: NEB 2-log; lanes 2-8: pIK1 miniPreps digested with BamHI+HindIII; lanes 2-3: 14 µl / 10 µl plate; lanes 4-8: 14 µl / rest plate
</P></DIV></DIV></TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#a19" data-toggle="tab">Lab book: plasmid construction</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a19"><H2><SPAN class="mw-headline" id="2013-09-02"> 2013-09-02 </SPAN></H2><UL><LI> send pIK1.11, pIK1.13, pIK1.14 to sequencing with primer VF2
</LI></UL><H2><SPAN class="mw-headline" id="2013-09-03"> 2013-09-03 </SPAN></H2><UL><LI> sequences arrived: <A href="/wiki/index.php?title=File:Heidelberg_PIK1.11-pIK1.13-pIK1.14-2013-09-03.zip&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK1.11-pIK1.13-pIK1.14-2013-09-03.zip (page does not exist)">PIK1.11-pIK1.13-pIK1.14-2013-09-03.zip</A>, <A href="/wiki/index.php?title=File:Heidelberg_PIK1.11-2013-09-03_VF2.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK1.11-2013-09-03 VF2.clustal.txt (page does not exist)">PIK1.11-2013-09-03 VF2.clustal.txt</A>, <A href="/wiki/index.php?title=File:Heidelberg_PIK1.13-2013-09-03_VF2.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK1.13-2013-09-03 VF2.clustal.txt (page does not exist)">PIK1.13-2013-09-03 VF2.clustal.txt</A>, <A href="/wiki/index.php?title=File:Heidelberg_PIK1.14-2013-09-03_VF2.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK1.14-2013-09-03 VF2.clustal.txt (page does not exist)">PIK1.14-2013-09-03 VF2.clustal.txt</A></LI><LI> insertions leading to stop-codon in pIK1.11, pIK1.13; two bases of RBS missing in pIK1.14
</LI><LI> check wiki of Cambridge 2007 team, who constructed the permeability device: <A href="http://www.ccbi.cam.ac.uk/iGEM2007/index.php/Detailed_background_information_-_signalling#Plan" class="external text" rel="nofollow">they tried different promoter strengths</A>, <A href="http://www.ccbi.cam.ac.uk/iGEM2007/index.php/Signalling_project_description#Standard_Construction_of_Permeability_Device" class="external text" rel="nofollow">only the weakest promoter gave colonies</A> =&gt; bacteria might be selecting against the permeability device
</LI><LI> put pIK1.12 in incubator again, maybe it has the right sequence and grows slowly
</LI><LI> pIK1.12 did grow, make miniPrep -&gt; 67 ng/µl in 38 µl
</LI><LI> send pIK1.14 to sequencing with primers IK25, VR
</LI></UL><H2><SPAN class="mw-headline" id="2013-09-04"> 2013-09-04 </SPAN></H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Lane 1: NEB 2-log; lanes 2: pIK1.12 digested with BamHI+HindIII</DIV></DIV></DIV><UL><LI> sequences arrived: <A href="/wiki/index.php?title=File:Heidelberg_PIK1.14-2013-09-04.zip&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK1.14-2013-09-04.zip (page does not exist)">PIK1.14-2013-09-04.zip</A>, <A href="/wiki/index.php?title=File:Heidelberg_PIK1.14-2013-09-04_IK25.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK1.14-2013-09-04 IK25.clustal.txt (page does not exist)">PIK1.14-2013-09-04 IK25.clustal.txt</A>, <A href="/wiki/index.php?title=File:Heidelberg_PIK1.14-2013-09-04_VR.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK1.14-2013-09-04 VR.clustal.txt (page does not exist)">PIK1.14-2013-09-04 VR.clustal.txt</A></LI><LI> sequences match
</LI><LI><A href="/wiki/index.php?title=Preparing_glycerol_stocks&amp;action=edit&amp;redlink=1" class="new" title="Preparing glycerol stocks (page does not exist)">prepare glycerol stock</A> of DH10ß-pIK1.14
</LI><LI> digest pIK1.12 with BamHI+HindIII (20 µl total volume, 134 ng DNA [2 µl of miniPrep from 2013-09-03], 0.5 µl enzyme)
</LI><LI> digest looks about right, send to sequencing with primer VF2
</LI></UL><H2><SPAN class="mw-headline" id="2013-09-05"> 2013-09-05 </SPAN></H2><UL><LI> sequence arrived: <A href="/wiki/index.php?title=File:Heidelberg_PIK1.12-2013-09-05.zip&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK1.12-2013-09-05.zip (page does not exist)">PIK1.12-2013-09-05.zip</A>, <A href="/wiki/index.php?title=File:Heidelberg_PIK1.12-2013-09-05_VF2.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK1.12-2013-09-05 VF2.clustal.txt (page does not exist)">PIK1.12-2013-09-05 VF2.clustal.txt</A></LI><LI> ca. 1 kb missing from end of permeability device
                                </LI><LI class="active"><A href="#a20" data-toggle="tab">Lab book: plasmid construction</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a20"><H2><SPAN class="mw-headline" id="2013-09-09"> 2013-09-09 </SPAN></H2><UL><LI> co-transform TOP10 with pRB22.3 and pIK7.1, plate on Cm+Kan, grow at 37°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-09-10"> 2013-09-10 </SPAN></H2><UL><LI> after ca. 18h at 37°C: lots of tiny colonies, not blue
</LI><LI> leave at RT in drawer (no light)
</LI></UL><H2><SPAN class="mw-headline" id="2013-09-11"> 2013-09-11 </SPAN></H2><UL><LI> colonies became blue
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">TOP10 cotransformed with pRB22.3 and pIK7.1</DIV></DIV></DIV><UL><LI> start constructing pIK8: promoter even weaker than the one used by Cambridge 2007, weak RBS
</LI><LI> run PCRs of (2 ng DNA, 20 µl total volume, using Q5):
</LI><LI> pIK2.6 with primers IK40+IK43 -&gt; f7
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 30
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD> 58 </TD><TD> 10
</TD></TR><TR><TD> 72 </TD><TD> 300
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 10 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> BBa_I746200 with primers IK41+IK42 -&gt; f8
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 30
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD> 60 </TD><TD> 10
</TD></TR><TR><TD> 72 </TD><TD> 90
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 10 </TD><TD> inf
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="2013-09-12"> 2013-09-12 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Lane 1: NEB 2-log; lane 2: f7; lane 3: f8</DIV></DIV></DIV><UL><LI> run gel, gel-purify fragments -&gt; 6.3 ng/µl in 20 µl (f7); 36.1 ng/µl in 20 µl (f8)
</LI><LI> perform Gibson assembly: 8 µl f7, 2 µl f8, 10 µl MM, incubate at 50°C for 1h
</LI><LI> dilute Gibson mix 1:4, electroporate electrocompetent DH10ß with 1 µl, 14 µl of dilution
</LI><LI> plate 10 µl, rest on Cm, grow at 37°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-09-13"> 2013-09-13 </SPAN></H2><UL><LI> very large and very small colonies present, pick starting with small colonies, run colony-PCR with primers IK26+VR (iTaq, 20 µl total volume) using 2 ng of BBa_I746200 in pSB1AK3 as positive control:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 95 </TD><TD> 30
</TD></TR><TR><TD> 54 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 240
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 10 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> clones grown in eppendorf tubes in 2xYT+Cm at 37°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-09-14"> 2013-09-14 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Colony-PCR of DH10ß transformed with pIK8 and primers IK26+VR. Lane 1: NEB 2-log; lanes 2-11: colonies; lane 12: positive control (BBa_I746200 in pSB1AK3)</DIV></DIV></DIV><UL><LI> run gel
</LI><LI> colonies designated pIK8.1, pIK8.2, pIK8.6, pIK8.9 positive
</LI><LI> transfer to baci falcons, add 2xYT+Cm, grow at 37°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-09-15"> 2013-09-15 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Restriction digest of pIK8 with BamHI+HindIII. Lane 1: NEB 2-log; lane 2: pIK8.1; lane 3: pIK8.2; lane 4: pIK8.6; lane 5: pIK8.9</DIV></DIV></DIV><UL><LI> make miniPreps: 73.5 ng/µl (pIK8.1), 12.9 ng/µl (pIK8.2), 289.0 ng/µl (pIK8.6), 169.0 ng/µl (pIK8.9) in 37.5 µl each
</LI><LI> digest with BamHI+HindIII using 2 µl (pIK8.1), 10 µl (pIK8.2), 1 µl (pIK8.6), 1 µl (pIK8.9) DNA (20 µl total volume)
</LI><LI> expected: 2.1 + 7.3 kb
                                </LI><LI class="active"><A href="#a21" data-toggle="tab">Lab book: plasmid construction</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a21"><H2><SPAN class="mw-headline" id="2013-09-16"> 2013-09-16 </SPAN></H2><UL><LI> send pIK8.6, pIK8.9 to sequencing with primer RB43
</LI></UL><H2><SPAN class="mw-headline" id="2013-09-17"> 2013-09-17 </SPAN></H2><UL><LI> sequences arrived: <A href="/wiki/index.php?title=File:Heidelberg_PIK8.6-pIK8.9-2013-09-16.zip&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK8.6-pIK8.9-2013-09-16.zip (page does not exist)">PIK8.6-pIK8.9-2013-09-16.zip</A>, <A href="/wiki/index.php?title=File:Heidelberg_PIK8.6-2013-09-16_RB43.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK8.6-2013-09-16 RB43.clustal.txt (page does not exist)">PIK8.6-2013-09-16 RB43.clustal.txt</A>, <A href="/wiki/index.php?title=File:Heidelberg_PIK8.9-2013-09-16_RB43.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK8.9-2013-09-16 RB43.clustal.txt (page does not exist)">File:Heidelberg_PIK8.9-2013-09-16 RB43.clustal.txt</A></LI><LI> no mutations at primer site
</LI><LI> sequence with VR primer
</LI><LI> inoculate 2x 6 ml 2xYT+Cm with DH10ß-pIK8.6
</LI></UL><H2><SPAN class="mw-headline" id="2013-09-18"> 2013-09-18 </SPAN></H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Lane 1: NEB 2-log; lane 2: pIK8.6 digested with EcoRI+SpeI, lane 3: pSB1C3 digested with EcoRI+SpeI</DIV></DIV></DIV><UL><LI> sequences arrived: <A href="/wiki/index.php?title=File:Heidelberg_PIK8.6-pIK8.9-2013-09-18.zip&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK8.6-pIK8.9-2013-09-18.zip (page does not exist)">PIK8.6-pIK8.9-2013-09-18.zip</A>, <A href="/wiki/index.php?title=File:Heidelberg_PIK8.6-2013-09-18_VR.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK8.6-2013-09-18 VR.clustal.txt (page does not exist)">PIK8.6-2013-09-18 VR.clustal.txt</A>, <A href="/wiki/index.php?title=File:Heidelberg_PIK8.9-2013-09-18_VR.clustal.txt&amp;action=edit&amp;redlink=1" class="new" title="File:Heidelberg PIK8.9-2013-09-18 VR.clustal.txt (page does not exist)">File:Heidelberg_PIK8.9-2013-09-18 VR.clustal.txt</A></LI><LI> sequences look good
</LI><LI> make miniPrep of pIK8.6 -&gt; 269 ng/µl in 38 µl
</LI><LI> to verify functionality of permeability device: need to co-transform with indigoidine synthetase =&gt; need to clone into pSB3K3 -&gt; pIK9
</LI><LI> need to clone into pSB1C3 for submission (pIK10, pIK11): digest 465 ng pSB1C3 (1 µl of Ralf's midiPrep) with EcoRI+SpeI, treat with antarctic phosphatase (37°C, 60 min)
</LI><LI> digest pIK8.6 with EcoRI+SpeI (3 µl DNA, 0.5 µl enzyme, 20 µl total volume)
</LI><LI> gel-purify pIK8.6 insert (20 ng / µl in 20 µl), pSB1C3 (18 µl, concentration not measured)
</LI><LI> ligate for 1 h at RT:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> pIK9 </TH><TH> pIK10 </TH><TH> pIK11
</TH></TR><TR><TD> backbone </TD><TD> 9 µl pSB3K3 ES from 2013-09-27 </TD><TD> 9 µl pSB1C3 ES </TD><TD> 9 µl pSB1C3 ES
</TD></TR><TR><TD> insert </TD><TD> 8 µl pIK8.6 ES </TD><TD> 8 µl pIK8.6 ES </TD><TD> 4.5 µl pIK2.6 ES (rest from 2013-08-24)
</TD></TR><TR><TD> T4 ligase </TD><TD> 1 µl </TD><TD> 1 µl </TD><TD> 1 µl
</TD></TR><TR><TD> T4 ligase buffer </TD><TD> 2 µl </TD><TD> 2 µl </TD><TD> 2 µl
</TD></TR><TR><TD> H<SUB>2</SUB>O </TD><TD> 0 µl </TD><TD> 0 µl </TD><TD> 3.5 µl
</TD></TR></TBODY></TABLE><UL><LI> transform TOP10 with 10 µl of each ligation, plate on LB+Kan(pIK9), LB+Cm(pIK10, pIK11)
</LI></UL><H2><SPAN class="mw-headline" id="2013-09-19"> 2013-09-19 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Lane 1: NEB 2-log; lanes 2-5: pIK9; lanes 6-9: pIK10; lanes 10-13: pIK11</DIV></DIV></DIV><UL><LI> colonies present for all ligations
</LI><LI> colonies for pIK10 extremely small -&gt; permeability device toxic
</LI><LI> run colony-PCR using primers VF2+IK25 (expected: 1 kb, OneTaq, 20 µl total volume):
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 95 </TD><TD> 30
</TD></TR><TR><TD> 54 </TD><TD> 30
</TD></TR><TR><TD> 72 </TD><TD> 90
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 10 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> grow in TB+Kan (pIK9) / TB+Cm (pIK10, pIK11)
</LI><LI> evening: make miniPreps of pIK9, pIK11 (pIK10 grew very slowly) -&gt; ca. 50-60 ng/µl (pIK9), 100-300 ng / µl (pIK11) in 38 µl
</LI><LI> inoculate 2xYT+Cm with pIK8.1, pIK8.6
</LI></UL><H2><SPAN class="mw-headline" id="2013-09-20"> 2013-09-20 </SPAN></H2><UL><LI> make miniPreps of pIK10 -&gt; 400-600 ng/µl in 38 µl
</LI><LI> digest everything with BamHI+HindIII, expected: 2.1 kb + 3.1 kb + 4.1 kb (pIK9), 2.1 kb + 6.6 kb (pIK10), 2.1 kb + 4.3 kb (pIK11) (20 µl total volume, 0.5 µl enzyme, using 0.5 µl (pIK10), 1 µl (pIK11), 2 µl (pIK9) DNA)
</LI><LI> to distinguish between pSB3C5 and pSB1C3: digest pIK10, pIK11 with XhoI (expected: 1 band for pSB3C5, 3 bands for pSB1C3)
</LI></UL><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 335px;"><DIV class="gallerytext"><P>All digested with BamHI+HindIII. Lane 1: NEB 2-log; lanes 2-5: pIK9; lanes 6-9: pIK10; lanes 10-11: pIK11
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 335px;"><DIV class="gallerytext"><P>All digested with XhoI. Lane 1: NEB 2-log; lanes 2-3: pIK11; lanes 4-7: pIK10.
</P></DIV></DIV></TD></TR></TBODY></TABLE><UL><LI> send pIK10.1, pIK10.3, pIK10.3, pIK11.1, pIK11.2 to sequencing with primers VF2, VR
</LI><LI> test permeability device:
</LI><LI> inoculate 3 ml LB+Cm soft agar with 100 µl of pIK8.1, pIK8.6 ON culture, pour on LB+Cm plates
</LI><LI> add sterile filter paper, pipet 8 µl, 4 µl, 2 µl, 1 µl of <A href="/wiki/index.php?title=Bacitracin_stock_solution&amp;action=edit&amp;redlink=1" class="new" title="Bacitracin stock solution (page does not exist)">Bacitracin</A> on filter paper discs
</LI><LI> grow at 37°C
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Left: TOP10-pIK8.6, right: TOP10-pIK8.1 (negative control)counterclockwise, starting top right: 8 µl, 4 µl, 2 µl, 1 µl bacitracin</DIV></DIV></DIV></DIV><UL><LI> Co-transform TOP10 with 25 ng pIK9.1, 10 ng pRB22.4, plate on Cm+Kan, grow at 37°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-09-21"> 2013-09-21 </SPAN></H2><UL><LI> co-transformats: lots of extremely small, blue-ish colonies, several big white colonies -&gt; possibly mutated indigoidine synthetase, several big black colonies -&gt; possibly mutated permeability device
</LI><LI> pick one little blue-ish colony, grow in 2xYT+Cm+Kan at 37°C
                                </LI><LI class="active"><A href="#a22" data-toggle="tab">Lab book: plasmid construction</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a22"><P style="font-size:12pt; text-align:justify;">
                                == 2013-09-23 ==
<LI> sequences arrived: <A href="/File:Heidelberg_PIK10.1-pIK10.2-pIK10.3-pIK11.1-pIK11.2-2013-09-23.zip" title="File:Heidelberg PIK10.1-pIK10.2-pIK10.3-pIK11.1-pIK11.2-2013-09-23.zip">File:Heidelberg PIK10.1-pIK10.2-pIK10.3-pIK11.1-pIK11.2-2013-09-23.zip</A>, <A href="/File:Heidelberg_PIK10.1-2013-09-23_VF2.clustal.txt" title="File:Heidelberg PIK10.1-2013-09-23 VF2.clustal.txt">File:Heidelberg PIK10.1-2013-09-23 VF2.clustal.txt</A>, <A href="/File:Heidelberg_PIK10.1-2013-09-23_VR.clustal.txt" title="File:Heidelberg PIK10.1-2013-09-23 VR.clustal.txt">File:Heidelberg PIK10.1-2013-09-23 VR.clustal.txt</A>, <A href="/File:Heidelberg_PIK10.2-2013-09-23_VF2.clustal.txt" title="File:Heidelberg PIK10.2-2013-09-23 VF2.clustal.txt">File:Heidelberg PIK10.2-2013-09-23 VF2.clustal.txt</A>, <A href="/File:Heidelberg_PIK10.2-2013-09-23_VR.clustal.txt" title="File:Heidelberg PIK10.2-2013-09-23 VR.clustal.txt">File:Heidelberg PIK10.2-2013-09-23 VR.clustal.txt</A>, <A href="/File:Heidelberg_PIK10.3-2013-09-23_VF2.clustal.txt" title="File:Heidelberg PIK10.3-2013-09-23 VF2.clustal.txt">File:Heidelberg PIK10.3-2013-09-23 VF2.clustal.txt</A>, <A href="/File:Heidelberg_PIK10.3-2013-09-23_VR.clustal.txt" title="File:Heidelberg PIK10.3-2013-09-23 VR.clustal.txt">File:Heidelberg PIK10.3-2013-09-23 VR.clustal.txt</A>, <A href="/File:Heidelberg_PIK11.1-2013-09-23_VF2.clustal.txt" title="File:Heidelberg PIK11.1-2013-09-23 VF2.clustal.txt">File:Heidelberg PIK11.1-2013-09-23 VF2.clustal.txt</A>, <A href="/File:Heidelberg_PIK11.1-2013-09-23_VR.clustal.txt" title="File:Heidelberg PIK11.1-2013-09-23 VR.clustal.txt">File:Heidelberg PIK11.1-2013-09-23 VR.clustal.txt</A>, <A href="/File:Heidelberg_PIK11.2-2013-09-23_VF2.clustal.txt" title="File:Heidelberg PIK11.2-2013-09-23 VF2.clustal.txt">File:Heidelberg PIK11.2-2013-09-23 VF2.clustal.txt</A>, <A href="/File:Heidelberg_PIK11.2-2013-09-23_VR.clustal.txt" title="File:Heidelberg PIK11.2-2013-09-23 VR.clustal.txt">File:Heidelberg PIK11.2-2013-09-23 VR.clustal.txt</A></LI><LI> sequences are right
</LI><LI> when pasting sequence of pIK10 into parts registry: realized that introduced XbaI site before BBa_B0030, can not submit part
</LI></P><H2><SPAN class="mw-headline" id="2013-09-27"> 2013-09-27 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Lane 1: NEB 2-log; lane 3: pIK8.6 digested with EcoRI+XbaI; lane 5: PCR of pIK8.6 with primers VF2+IK44</DIV></DIV></DIV><UL><LI> need to remove XbaI site for submission (for Boston) -&gt; pIK12: amplify pccB-accA2 with primer introducing SpeI, ligate SpeI+Xba+ sites
</LI><LI> run PCR of pIK8.6 with primers VF2+IK44 (3 ng DNA, 20 µl total volume, using Q5) -&gt; f9:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 30
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD> 55 </TD><TD> 10
</TD></TR><TR><TD> 72 </TD><TD> 180
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 10 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> digest 867 ng pIK8.6 (3 µl of miniPrep from 2013-09-15) with EcoRI+XbaI (20 µl total volume, using 0.5 µl enzyme), treat with antarctic phosphatase (37°C, 60 min)
</LI><LI> gel-purify large pIK8.6 fragment, f9 (elute f9 in 17 µl)
</LI></UL><H2><SPAN class="mw-headline" id="2013-09-28"> 2013-09-28 </SPAN></H2><UL><LI> digest f9 with EcoRI+SpeI (0.5 µl enzyme, 20 µl total volume), purify with Nucleotide Removal Kit
</LI><LI> ligate at RT for 1h:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> what </TH><TH> µl
</TH></TR><TR><TD> pIK8.6 </TD><TD> 8
</TD></TR><TR><TD> f9 </TD><TD> 9
</TD></TR><TR><TD> T4 ligase </TD><TD> 1 µl
</TD></TR><TR><TD> T4 ligase buffer </TD><TD> 2 µl
</TD></TR></TBODY></TABLE><UL><LI> heat-inactivate at 75°C for 5 min
</LI><LI> transform 10 µl into TOP10, plate 10 µl, rest on Cm, grow at 37°C
</LI></UL><H2><SPAN class="mw-headline" id="2013-09-28_2"> 2013-09-28 </SPAN></H2><UL><LI> lots of small colonies on both plates, several large colonies on both plates
</LI><LI> pick 4 small, 1 large colonies from 10 µl plate, grow in 2xYT+Cm at 37°C
                                </LI></UL><DIV class="row"><DIV class="col-md-12"><DIV class="imageMap" aria-haspopup="true"><CENTER>Thanks to </CENTER></DIV></DIV></DIV><DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Heidelberg/Delftibactin/MMCoA">http://2013.igem.org/Team:Heidelberg/Delftibactin/MMCoA</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Delftibactin/MMCoA" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Delftibactin/MMCoA" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Delftibactin/MMCoA&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Delftibactin/MMCoA&amp;oldid=320963" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>