"
Page
Discussion
View source
History
teams
Log in
 
Team:Heidelberg/Templates/Methods
From 2013.igem.org
Contents
1 Cloning
1.1 Colony PCR
1.2 Agarose gel electrophoresis
1.3 Gibson assembly
1.4 Biobrick assembly
1.5 Restriction digests
1.6 Extraction of DNA fragments from an agarose gel
1.7 Ligation
1.8 Nucleotide Removal
1.9 Purification of PCR products
1.10 Plasmid-DNA isolation
1.10.1 Miniprep
1.10.2 Midiprep
1.10.3 Preparation of large plasmids
1.11 Sequencing
2 Protein detection
2.1 Thin Layer Chromatography
2.2 Purification of Indigoidine and tagged constructs
2.3 Purification of Delftibactin
2.4 Mass spectrometry
2.4.1 Cell preparation
2.4.2 Data Acquisition
2.5 SDS-PAGE
3 1st Pre-Experiment
3.1 Making ACM Medium
3.1.1 for 1 L
3.1.2 Making Plates
3.2 Reactivation of freeze-dried bacteria
3.3 Inoculation of agar plates
3.4 Gold-precipitation on agar plates
3.5 Delftibactin purification
4 DNA Storage
4.1 Plasmid Shipping
4.1.1 Sending
4.1.2 Receiving
5 Electrocompetent Bacteria
5.1 Materials
5.2 Day 1
5.3 Day 2
6 Electroporation of E.coli DH10β
6.1 Preparation:
6.2 Procedure
6.3 plate
6.4 Additionally
7 Heat shock competent bacteria
7.1 Materials
7.2 Day 1
7.3 Day 2
8 Isopropanol PCR Purification
9 PCR
9.1 validation PCR
10 Preparing Glycerol Stock
11 Software
11.1 Main packages
11.1.1 Packages, plugins or libraries loaded by some of our python functions
11.1.2 Django packages
11.2 C++
11.3 Javascript
Cloning
Colony PCR
Colonies were picked with pipet tips and dipped into the PCR mixture.
Ingredient
amount [µl]
Overnight Culture
Top of a pipet tip
10x PCR buffer
2.0
MgCl2 (25 mM)
2.0
Forward primer (10µM)
0.2
Reverse primer (10µM)
0.2
dNTPs (10 mM)
1.0
Polymerase
0.5
Sterile water
ad 10.0
Total
10.0
The following PCR program was adjusted according to annealing conditions.
Cycles
temperature [°C]
Time [min:s]
1
98
2:00
35
98
0:05
68 (annealing)
0:15
72
1:00 per kbp
1
72
10:00
1
12
inf
Agarose gel electrophoresis
To separate DNA fragments of different sizes, agarose flat-bed gels were casted in various concentrations (0.6% to 2% agarose in 0.5x TAE buffer). By applying an electrical field (10 to 20 V/cm), these gels were used for analytical or preparative separation. Therefore, the desired amount of agarose was boiled up in 0.5x TAE buffer until it was completely dissolved. After the solution cooled down to 50 to 60°C, it was poured into a flat-bed tray and ethidium bromide (EtBr) solution (0.5 µg/ml final concentration) was added. Finally, a comb was inserted to create pockets while solidification. The gel was put into the running buffer (0.5x TAE buffer) before the DNA mixture with loading buffer was loaded into the pockets and separated by electrophoresis. As ethidium bromide intercalates with the DNA (GC nucleotides) the resulting DNA-EtBr-complexes are visible under UV irradiation at 265 nm. As standard ladder for fragment sizes, 1kb+ ruler, 2-log ladder or 50 bp ladder were used.
Gibson assembly
Excel template kindly provided by the iGEM Team Freiburg 2013
Gibson assembly was prepared according to calculation (see file above). The mixture was incubated at 50°C for 60 minutes and purified by isopropanol precipitation. The Gibson constructs were then transformed into competent bacteria.
Biobrick assembly
BioBrick_Assembly_Manual.pdf
Restriction digests
For enzymatic digests, the following formula was used if bovine serum albumin (BSA) was not part of the buffer, which was the case for NEB buffer 3.0:
... µl DNA (~500 ng)
5 µl BSA
5 µl buffer (for example NEBuffer 2)
2 x 1.5 µl enzymes (for example EcoRI-HF & PstI)
... µl water
=> 50 µl in total
The mixture was incubated at 37°C for 1h or up to 1:30h and analyzed by gel electrophoresis.
In case BSA was already part of the buffer, the following recipe was used:
... µl DNA (~500 ng)
2 µl buffer (e.g. NEB CutSmart buffer)
1 µl restriction enzymes
water ad 20 µl
The mixture was incubated at 37°C for 1h or up to 1:30h and analyzed by gel electrophoresis.
Extraction of DNA fragments from an agarose gel
After gel electrophoresis, the gel parts containing the desired DNA fragement size were excised from the agarose gel with a clean and sharp scalpel. For DNA extraction the QIAquick® Gel Extraction Kit was used. Following the instrucion manual all necessary solutions have been prepared. The slices were weighted and accordingly Buffer QG added to the gel (for 1 slice 600 μl; for 2 slices 900 μl). The tubes with the mixture were incubated at 55°C until the gel slices had completely dissolved. In between samples were vortexed, if fragement sizes were not too large. Since the color of the solution never changed, no pH adjustments had to be done. In the next step isopropanol was added (for 1 slice 200 μl; for 2 slices 300 μl) to the sample and mixed. For each sample a QIAquick spin column was placed into the vacuum manifold and the sample decanted to it to bind DNA. Vacuum was applied until the whole sample has been drawn through the column. For further purification samples were washed with 500 μl buffer QG and vacuum was applied. Afterwards 750 μl Buffer PE were added to the samples, let stand for 2-5 minutes and vacuum applied. To remove residual wash buffer, the columns were placed into collection tubes and centrifuged for 1 min at 17,900 x g (13,000 rpm). Finally, the columns were put into clean 1.5 ml microcentrifuge tubes and 20-30 μl water (55°C) were added to the center of the column. After letting the column stand for 1 minute it was centrifuged for 1 minute to collect the elute.
For larger DNA fragments, the QIAEX II® Gel Extraction Kit was used (suitable for 40 bp to 50 kbp). Solutions were prepared as advised in the instruction protocol. DNA bands were excised from the agarose gel with a clean and sharp scalpel and transfered to a microcentrifuge tube. After weighing the slices, buffer QX1 was added according to the DNA fragment size (6 volumes for <100 bp; 3 volumes for 100 bp – 4 kb; 3 volumes with 2 volumes of water for >4 kb). QIAEX II was vortexed for 30 seconds to resuspend the agent and added to the samples (10 μl QIAEXII for ≤2 μg DNA; 30 μl for 2–10 μg DNA; and an additional 30 μl for each additional 10 μg DNA). Subsequently the solution was mixed and incubated at 50°C for 10 min to solubilize the agarose and bind the DNA. Since we mainly used this kit for larger fragments, the resuspended pellet was only inverted and flicked to avoid shearing forces. We did not observe changes in color, that is why no pH adjustment were done. Afterwards, samples were centrifuged for 30 seconds and supernatant carefully removed with a pipet. The pellet was washed with 500 μl buffer QX1 to remove residual agarose contaminants and resuspended by inverting and flicking. Then samples were centrifuged for 30 seconds and all supernatant residues removed with a pipet. The pellets were washed twice with 500 μl buffer PE to remove salt contaminants and resuspended by inverting and flicking. Then samples were centrifuged for 30 seconds and supernatant removed with a pipet. Pellets were air-dried for 10-15 minutes until they became white. To elute the DNA, 20 μl of water were added and pellet resuspended by flicking. After incubation according to the DNA fragment size (5 minutes at room temperature (15–25°C) for ≤4 kb; 5 minutes at 50°C for 4–10 kb; 10 minutes at 50°C for >10 kb), samples were centrifuged for 30 seconds. Supernatant, which contained the purified DNA, was pipetted into a clean tube.
Ligation
For ligation of DNA fragments, the mixture was prepared as follows:
... µl DNA (=<200ng in total-> 20-70ng Backbone)
2 µl buffer
1 µl ligase (T4)
... µl water
=> 20 µl in total
After an incubation for 2 minutes at room temperature, or for 1h or over night at 4°C, the product was loaded on an gel for electrophoretic separation of the DNA fragments.
Nucleotide Removal
For removal of excess nucleotides after enzymatic reactions, the QIAquick® Nucleotide Removal Kit (QIAGEN) was used. All solutions were prepared as recommended by the manufacturer. Ten volumes of Buffer PNI were added to 1 volume of the sample and mixed. The QIAquick spin column was placed in a provided 2 ml collection tube and sample decanted to it. The tube was centrifuged at 6000 rpm for 1 minute. Flow-through was discarded and the column placed back into the collection tube. Afterwards, 750 μl Buffer PE were added, incubated for 2-3 minutes and centrifuged for 1 min at 3800 x g (6000 rpm). Flow-through was discarded and the column placed back into the collection tube. The column was centrifuged for an additional 1 min at 17,900 x g (13,000 rpm) to remove residual buffer. Finally, the column was put into a clean 1.5 ml microcentrifuge tube and 20-30 μl water (55°C) was added to the center. After 1 minute incubation the sample was centrifuged for 1 minute at 17,900 x g (13,000 rpm). The 1.5 ml tube now contained the eluted and purified DNA.
Purification of PCR products
To purify PCR products the QIAquick® PCR Purification Kit (QIAGEN) was used. Therefore, all solutions were prepared as recommended by the manufacturer. Five volumes of buffer PB were added to 1 volume of the PCR reaction before mixing. Since no changes in color were observed the pH was not adjusted. A QIAquick column was put into a vacuum manifold and the sample was decanted to it for binding the DNA. Vacuum was applied until all liquid residues were drawn through the column. Afterwards 750 μl buffer PE were added to wash the column and vacuum applied. Te remove residual wash buffer the QIAquick column was put into the provided 2 ml collection tube and centrifuged for 1 min at 17,900 x g (13,000 rpm). Finally, the column was put into a clean 1.5 ml microcentrifuge tube and 20 ml water (55°C) were added to the center to elute the DNA. After 1 minute incubation the tube containing the column was centrifuged for 1 minute. The microcentrifuge tube now contained the eluted DNA.
For small amounts of DNA PCR products (cleanup of up to 5 μg PCR product (70 bp to 4 kb)) the MinElute® PCR Purification Kit (QIAGEN) was used. All steps were conducted and solutions prepared as specified in the protocol. Five volumes of buffer PB were added to 1 volume of the PCR reaction and mixed. Since no changes in color were observed the pH was not adjusted. A MinElute column was put into a vacuum manifold and the sample was decanted to it for binding the DNA. Vacuum was applied until all liquid residues were drawn through the column. Afterwards 750 μl buffer PE were added to wash the column and vacuum applied. Te remove residual wash buffer the QIAquick column was put into the provided 2 ml collection tube and centrifuged for 1 min at 17,900 x g (13,000 rpm). The MinElute column was placed into a clean 1.5 ml microcentrifuge tube and 20 ml water (55°C) were added toe the center to elute the DNA. After 1 minute incubation the tube containing the column was centrifuged for 1 minute. The microcentrifuge tube now contained the eluted DNA.
Plasmid-DNA isolation
Miniprep
Plasmids were isolated by the use of the QIAprep® Spin Miniprep Kit. According to the instructions included, all necessary solutions were prepared and steps conducted. First, overnight cultures (1-2 ml) have been pelleted by centrifugation at >8000 rpm
(6800 x g) for 3 min at room temperature (15–25°C). Supernatant was discarded and pellet resuspended in 250 μl (procedure 2: 200 μl) buffer P1 in a microcentrifuge tube. Afterwards 250 μl (procedure 2: 400 μl) Buffer P2 were added and the tube inverted 4-6 times to mix the solution until it became clear. The sample was lysed for 2-3 minutes before adding 350 μl (procedure 2: 300 μl) buffer N3 and inverting the whole solution 4-6 times. Following the neutralization, the sample was centrifuged for 10 minutes at 3,000 rpm (~17,900 x g) in a table-top microcentrifuge. Supernatant was decanted to a QIAprep spin column and vacuum applied to the manifold until the solution was drawn through the column. To wash the column 500 μl Buffer PB were added and again vacuum applied. In addition the QIAprep spin column was washed by adding 750 μl Buffer PE and applying vacuum. To remove residual wash Buffer the column was transfered to a collection tube and centrifuged for 1 min. Finally, the column was put into a clean 1.5 ml microcentrifuge tube and 20-30 μl water (heated up to 55°C) added to the center of the column. After 1 minute the tube containing the column was centrifuged for 1 minute to collect eluted DNA.
Midiprep
For larger amounts of bacterial cultures the QIAGEN® Plasmid Plus Midi Kit was used to isolate plasmids. According to the instructions included, all necessary solutions were prepared and 25 ml bacterial culture harvested after 12-16 hours incubation. The sample was centrifuged at 6000 x g for 15 min at 4°C and pelleted cells were completely resuspended in 2 ml Buffer P1. Then, 2 ml buffer P2 were added and mixed thoroughly by inverting until the lysate appeared viscous. The solution was incubated at room temperature (15–25°C) for 3 min. The QIAfilter Cartridge was placed into a new tube. Afterwards, 2 ml buffer S3 were added to the lysate and mixed by inverting 4-6 times. The lysate was transfered to the cartridge and incubated at room temperature for 10 minutes. During incubation the QIAGEN Plasmid Plus spin column was put into the vacuum manifold and tube extenders inserted into each column. The plunger was inserted gently into the QIAfilter Cartridge and the lysate filtered into the tube. Thereafter, 2 ml buffer BB were added to the cleared lysate and mixed by inverting 4-6 times. Lysate was then transfered to a QIAGEN Plasmid Plus spin column on the vacuum manifold. Vacuum was applied until all liquid remains were drawn through the column. To wash the DNA 700 μl buffer ETR was added and vacuum applied. Residual wash buffer was removed by centrifugation of the column at 10,000 x g (9,700 rpm) for 1 min in a tabletop microcentrifuge. The column was put into a clean 1.5 ml tube. Finally, 200 μl water (55°C) was added to the center of the spin column. The column was let stand for 1 minute and centrifuged for 1 minute to collect the eluted DNA.
Preparation of large plasmids
For isolation of plasmids larger than 15 kb in size, 7 ml bacterial culture were harvested after 12-16 hours of growth in appropriate medium. Samples were centrifuged at 6000 xg for 20 min at 4°C. Supernatant was discarded and cell pellet resuspended in 200 µl of Buffer P1 (QIAGEN). Afterwards, 400 µl of Buffer P2 were added, mixed thoroughly by inverting until the lysate appeared viscous. Solution was incubated at room temperature (15-25°C) for 2 min. Afterwars 300 µl of Buffer P3 were added to stop lysis. Cell debris and Proteins were pelleted by centtrifugation at ~ 17,900 x g for 30 min at 4°C. Supernatant was carefully transferred into a fresh 2ml eppendorf centrifuge tube and isopropanol purification of DNA was conducted.
Sequencing
Samples were sent to GATC Biotech for sequencing.
Protein detection
Thin Layer Chromatography
TLC was carried out on silica-gel as immobile phase and Dichloromethane as mobile phase. For the procedure a 50 ml beaker was filled with ~15 ml Dichloromethane and let stand for about 10 minutes (in order to let the Dichloromethane-vapor fill the beaker). The TLC plate, coated with silica-gel was spotted with sample 0.5 to 1 cm above the lower edge and placed in the beaker. TLC was run until the solvent front was at two thirds of the TLC plate. As Indigoidine is light-sensitive, the beaker was covered with aluminium foil in order to prevent direct light irradiation.
Purification of Indigoidine and tagged constructs
1 ml of IPTG-induced, blue culture was spun down at full speed (14,000 rpm) for 20 minutes, washed in 1 ml of methanol and centrifuged once more for 5 minutes at 14,000 rpm. Methanol was discarded and samples were dissolved in 200-400 µl DMSO.
Purification of Delftibactin
Centrifugation of cultures at 3750 rpm for 30 min
Retrieve supernatant
Add 20 g/L HP-20
Stirr at RT for 2 h
Filtration with Buchner funnel to retain HP-20
Wash with 400 ml ddH2O
Elution with 400 ml Methanol
Evaporated methanol with rotary evaporator
Resuspended in 2 ml 50 methanol : 50 ddH2O
Mass spectrometry
Cell preparation
For analysis by mass spectrometry, 3 ml of LB media containing chloramphenicol (1 micromolar) was inoculated with transformed bacterial cells (expression of the NRPS tested before by SDS-PAGE). Cultures were incubated at 37°C overnight until OD600 ~ 0.6. After centrifugation for 30 minutes at 3750 rpm and 4°C, supernatant was discarded and pellet washed with M9 minimal media. Again samples were centrifuged and resuspended in 50 ml M9 minimal media. Following an incubation time of 12 hours (OD600=0.6) cells were induced with IPTG (1 mM).
To assess product levels over time, 7.5 ml samples were taken at several time points (8.5 hrs, 12 hrs, 24 hrs and 48 hrs). Each sample was spun down for 45 minutes at 3750 rpm and 4°C, and supernatant was separated from pellet.
Pellet was resolubilzed in 500 µl 1x PBS and lysed by ultra-sonification (3x20 seconds; on ice). Finally the sample has been centrifuged and supernatant decanted to 1.5 ml tubes. Frozen with nitrogen the sample was handed over for mass spectrometry.
Supernatant was frozen with nitrogen and lyophilized for 12 hours ( at ~-45°C and ~0,1 mbar) to remove liquid residues. The lyophilisate was resuspended in 500 microliters methanol and vortexed. Supernatant was transferred to a 1.5 ml tube, which was frozen in liquid nitrogen. The sample was then handed over to mass spectrometry for further analysis.
Data Acquisition
Sampling
Two empty 3.2 mm die-cut sheets were prepared from Whatman 903 filter paper. Afterwards, 5 µl of the sample were pipetted on the sheets and dried in the fridge overnight.
Sample preparation
The dried sheets were extracted for ca. 20 min with 100 μl solution 1. Then the polypropylen microtiter plates were centrifuged for ca. 2 minutes at 4000 rpm. Supernatant was vaporized at 55°C for 30 minutes at the blowing station until all liquid residues were removed. Subsequently the plate was warmed up for 1 minute at a cabinet dryer (65°C).
After drying 60 μl of solution 2 were added and the mircotiter plate tightly closed with micromates. The plate was shaken gently and heated up again for ca. 15 minutes at the cabinet dryer (65°C). Micromates were drawn off and supernatant vaporated on heating plates (55°C surface temperature) with a constant air stream. Again no liquid residues should be remaining. Samples were heated up for 1 minute in the cabinet dryer (65°C).
Finally 100 μl solution 3 were added (it is possible to dilute the sample up to factor 3 with solution 3). To avoid evaporation losses, the microtiter plate was covered with aluminium foil.
SDS-PAGE
1 ml culture was pipetted into a 1.5 ml centrifuge tube. This was centrifuged at 13,000 rpm for 10 minutes and supernatant was discarded. The pellet was resuspended in 100 µl loading buffer. The samples were boiled with closed lids at 98°C for at least 10 minutes under the hood, after which they were let cool down and centrifuged at 13,000 rpm for another 10 minutes. 25µl were loaded in a 3-8% Tris-Acetate Gradient Gel from Invitrogen, 10 µl NOVEX Pre-Stained Standard was used and PAGE was let run for 1 hour at 180 V with Tris-Acetate running buffer from Invitrogen. The aforementioned loading buffer is composed of 800 µl ddH20, 250 µl 0.5M Tris-HCl (pH = ~7.3), 400 µl Glycerol (50%), 400 µl SDS (10%), 100µl β-Mercaptoethanol (99%) and bromophenol-blue to obtain 2ml of loading buffer.
1st Pre-Experiment
The first step is the establishment of the reaction of delftibactin with soluble gold. Therefore we plan an experiment (one to two weeks), where we reproduce the results of the paper [<bib id="pmid23377039"/>]. After mixing the Acidovorax complex medium (ACM) and adding 4g Chelex-100 resin / 1L ACM  we prepare 25 agarplates (0,5 L ) and two media for liquid culture (1L +0,5L). The second day is about the resuspending of the strain D. acidovorans in 5 ml [recepy in methods] and letting it grow over night. The next day 5 agarplates are inoculated with D. Acidovorans by fractional streak and 5 agarplates with streak over half of the plate. In addition to these 10 agarplates, also the two flasks with the liquid media are inoculated with the strain. The rest of the resuspended cells is stored in a glycerol stock. After 3 days of culture growth (shaking at 190rpm, at a temperature of 30°C) the cells are big enough to start with the delftibactin purification (liquid culture) and the fast test on the agarplates. For the fast test on the agarplates we prepare an soft agarose solution (0,5% agarose and 10mM AuCl3) and incubate every plate with 10 ml solution for two hours at rooms temperature. If delftibactin is present there can be observed a ring structure around the cells.
For the purification of delftibactin, the liquid culture will be centrifugated at  7000 rpm for 15 min. The supernatant will be treated with HP20 resin (20 g/L) and then shaken 2h at 220rpm. With the Buchner funnel filtration the resin will be collected and washed with 400 ml dH2O. Afterwards the resin is again washed with 400 ml methanol which will be evaporate under rotary vacuum. The leftover is resuspended in 2 ml of ddH2O:MeOH (1:1). The solution is added to a HPLC column and the HPLC (high performance liquid chromatography) is analysed in a mass spectroscopy facility. The mobile phase consists in 2%-14% acetonitrile and  98% water +5 mM (NH4)2CO3.
After the purification we plan to mix the purified delftibactin with different gold solution (AuCl3) concentrations (between 5mM and 20mM). After a 2h incubation a precipitation would be observed.
In case the pre-experiment works, we have proven the reliability of the paper and the expected read out of the assay.
Making ACM Medium
for 1 L
0.5 g yeast extract (Difco-BectonDickinson)
1.0 g Cas amino acids (Difco)
2.0 g pyruvic acid
2.0 g L-glutamine
0.3 g KH2PO4
0.3 g MgSO4
2.0 g MOPS
4.0 g Chelex 100-resin (Sigma)
pH adjusted to 7.2–7.3 with 5 M KOH
Important Notes:
fill up to 900mL before adding pyruvic acid and L-glutamine
adjust pH
fill up to 1L
treat with Chelex for 1h
remove Chelex by filtration
Making Plates
prepare 2 l medium
make 25 agar plates
autoclave
Reactivation of freeze-dried bacteria
prepare 10 ml of liquid-activation-media with:
50 mg peptone
30 mg meat extract
150 mg agar
10 ml destilled water
adjust pH to 7.0
rehydrate and grow cells in 5 ml of liquid broth
grow at 30° in incubator over night
Inoculation of agar plates
inoculate 5 agar plates with inoculation loop
grow at 30° in incubator for 3 days
Gold-precipitation on agar plates
prepare 50 ml of 0.5 % Agarose solution which contains 10 mM AuCl3
pour 10 ml of the prepared agarose solution onto each of 3 agar plates
recepy for the gold solution media:Recepy_for_0,5%_agarose_solution_with_10mM_AuCl3.pdf
Delftibactin purification
centrifuge liquid culture at 7000 r.p.m.
keep supernatant, remove cell mass
treat supernatant with with 20 g/l HP20
shake HP20 with supernatant for 1 h at 220 r.p.m.
remove aqueous phase by buchner vacuum filtration
elute dehydrated material with 400 ml methanol
remove HP20 by buchner vacuum filtration
dry solution with rotary vacuum evaporator
resuspend material in 2 ml 50 % water and methanol
liquid chromatography
DNA Storage
Plasmid Shipping
uses Whatman Filter Paper
used the protocol of: https://labs.mcdb.ucsb.edu/weimbs/thomas/links/methods/plasmid_shipping.html
Sending
Mark a circled area with a pencil (not a marker pen) on a clean Whatman #1 filter paper (or equivalent).
Spot about 2 µg of plasmid DNA into the circle. Allow the filter paper dry at room temperature.
Insert spotted filter paper inside a plastic bag and heat-seal it.
Send by regular (air) mail.
Receiving
To recover the DNA, use clean gloves and cut the marked circle area that contains the dried plasmid DNA.
Using clean forceps, insert the filter paper into a 1.5 ml micro centrifuge tube. Add 100 µl of TE buffer, vortex briefly and incubate at room temperature for 5 minutes. Vortex again and centrifuge the tube for a few seconds.
Remove about 10 µl of supernatant for use in transfecting E. coli by electroporation or chemical means. Please do not try to use the DNA directly for any application other than to transform bacteria and prepare a plasmid stock.
Store the remainder of the filter paper/TE mix at -20 or -80 C as a permanent archive in case that your plasmid stock ever gets lost or if something turns out to be wrong with it.
Electrocompetent Bacteria
Materials
10% glycerol
ddH2O
Day 1
Inocculate 50 ml of appropriate medium
Grow ON at appropriate temperature, shaking (e.g. 37°C, 180 rpm)
Day 2
Transfer 10 - 30 ml of the ON culture into 400 ml appropriate medium (measure OD600, should not exceed 0.1)
Grow at appropriate temperature, shaking (e.g. 37°C, 180 rpm) until an OD of 0.4-0.6 is reached
Meanwhile, put H2O, 10% glycerol on ice, prechill tubes and centrifuge
Chill cells on ice for 10 min
Centrifuge for 30 min at 3750 rpm, 4°C
work on ice from here on
Discard supernatant, resuspend bacteria in 2 ml ice cold H2O / 50 ml falcon, fill up to 20 ml ice cold H2O
Centrifuge for 30 min at 3750 rpm, 4°C
Discard supernatant, resuspend in 1 ml glycerol / falcon, add up to 5 ml glycerol (falcons may be pooled at this point)
Centrifuge for 20 min at 3750 rpm, 4°C
Resuspend in same volume glycerol as pellet size (ca. 100 µl / original 50 ml falcon)
Aliquot 50 µl per tube
Optional: shock freeze with liquid nitrogen
Store aliquots at -80°C
Electroporation of E.coli DH10β
Preparation:
Source material: 1ml LB media + DH10β overnight culture (12-16 h) at 32°C (1 day before)
Preperate 4 eppis  with 1 ml LB media and a) 25µl,  b) 35µł,  c) 45µł,  d)  55µl of source material and incubate it at 32°C (2h)
Put bacteria on ice for 10 – 15 min (since this step everything is at 0°C – the centrifuge, the water!!!!!!!!)
Centrifuge 30 s at 13 000 rpm (0°C) and remove via vacuum the medium, DON'T touch the pellet!!!
Resuspend first pellet in 1 ml H2O bidest comercial (0°C),transfer to next eppi an dos on to collect all four cell-pellets resuspended in water in 'ONE eppi.
Centrifuge and wash again (2x)
Procedure
add 50 µl H2O bidest comercial (0°C) and resuspend pellet
add 2-3 µl of Ligation Reaction and incubate it on ice for 5-10 min
transfer all of it to the special cuvette (without any air bubbles) – the cuvette has to be perfectly dry on the outside
press button – electroshock (and from now on QUICK handling)
add 400 µl SOC-medium and transfer all of it into an eppi
incubation for 1 h at 37 °C shaking at 240 rpm
plate
plate on one agarplate with the corresponding antibiotic 100 µl of the bacteria
plate on another agarplate with the corresponding antibiotic 10 µl of the bacteria
Additionally
Bio-Rad MicroPulser
Program 1 for brown cuvettes
Program 2 for blue cuvettes
=> these two programs are for bacteria
Heat shock competent bacteria
Materials
100 mM CaCl2 (aq)
Glycerol
Day 1
grow bacteria over night in 50 ml medium at 37°C
Day 2
transfer 20-30 ml of the ON culture into 400 ml LB
grow at 37°C until an OD of 0.5-0.6 is reached (3-5 hours), put CaCl2 on ice, prepare aliquoting buffer (20 ml CaCl2 with 10% glycerol)
centrifuge bacteria for 30 min at 3750 rpm, 4°C
work on ice from here on
==> you will need 8 falcons (50 ml) in order to do so
discard supernatant, resuspend bacteria in 5 ml CaCl2 / falcon, pool 2 falcons, add up to 50 ml CaCl2
==> you should have 4 falcons from here on
leave on ice for 30 min
centrifuge for 20 min at 3750 rpm, 4°C
discard supernatant, resuspend in 10 ml aliquoting buffer for all 4 falcons, add another 10 ml aliquoting buffer
Do not vortex
pipet 100 µl per eppendorf tube (pre-cooled on ice), put aliquots in -80°C freezer
Isopropanol PCR Purification
pool PCR reactions if there are several
add >= 4 volumes isopropanol, invert tube 10 times
centrifuge 20 min at full speed (use cooling centrifuge at 20°C)
decant supernatant
add 4 volumes 70% ethanol
centrifuge 5 min at full speed
decant supernatant, tap eppendorf tube on paper tissue
let it dry for ca. 5 min
dry off inside of tube with a piece of paper tissue. Do not touch pellet!
resuspend in 130 µl H2O, add 15 µl CutSmart buffer and 5 µl DpnI (digests methylated DNA -> template)
incubate 4h at 37°C
repeat purification, resuspend in 30-40 µl H2O
PCR
... µl DNA (1-10ng)
2x 0,5 µl Primer
25 µl Phusion Flash Ready Mix(2x)
... µl water
=> 50 µl in total
fill in PCR tubes
PCR amplification takes ca. 2h
validation PCR
( using 2x Taq MasterMix from NEB )
~ 5 ng template DNA (picked colony, miniPrep...)
2x 0.2 µl primer (_fw, _rv)
10 µl 2x Taq MM
... H_2O (add up to 20 µl)
=> 20 µl in total
fill in PCR tubes
PCR amplification takes ca. 2h
Preparing Glycerol Stock
grow liquid culture of the desired bacteria
pipet 1 ml of liquid culture into cryotube, add 300 µl 50% glycerol
RT for 1 h
freeze at -80°C
Software
Main packages
Python(2.7): The high-level programming language python was used in combination with Django to develop the web-based interface and client-side applications.
Packages, plugins or libraries loaded by some of our python functions
Requests: The purpose of this library is to do simple http requests.
Biopython (>= 1.61): Just like bioperl this package contains tools for computational molecular biology.
xhtml2pdf: The converter package xhtml2pdf was applied to generate pdf from HTML/CSS templates.
BeautifulSoup: The BeautifulSoup HTML/XML parser was used for quick-turnaround applications.
celery: Asynchronous scheduling framework.
kombu: Database-based message passing framework.
python-openbabel(>= 2.3.2) : Python bindings to openbabel for 2D (chemical) structure generation.
UNAFold and MFold utils: UNAFold was required by Gibthon for DNA/RNA folding predictions.
MySQL: The relational database management system MySQL was utilized for database creation and maintenance.
MySQL - Python connector: A conncetor between the programming language python and the database management system.
Django(>= 1.5): The web application framework Django was used to achieve an easier database connection and is based on python. Since we improved and added basic functions of Gibthon ( iGEM Team Cambridge 2010 ), the integration was easier to handle using the same frame work.
Django packages
django-registration: Handles user registrations and logins.
django-annoying: Gibthon uses this package. It is thought to eliminate annoying things in Django.
south: The south package handles database migration (database independent).
django-celery: The package was used for the integration of Celery (asynchronous task queue/job queue) into Django.
C++
CMake: Cross platform build system used by the NRPSDesigner.
MySQL C++ connector: C++ library for MySQL access.
libXML: C library for XML handling.
libcurl: C library for client-side URL transfers.
Boost.program_options: C++ library for command-line option parsing.
GCC (>= 4.8) or a comparable compiler with C++11 support.
Javascript
Intro.js: This Javascript has been adopted for the guided step-by-step tour on the NRPS Designer homepage.
select2.js: The Select2 library is a jQuery based library for the replacements of select boxes, which support different connections to other functions.
Retrieved from " http://2013.igem.org/Team:Heidelberg/Templates/Methods "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
