<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_NRPSDesigner"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/NRPSDesigner">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/NRPSDesigner&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/NRPSDesigner&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/NRPSDesigner&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/NRPSDesigner" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/NRPSDesigner</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="row"><DIV id="nav" class="navigation"><DIV class="item team" data-offset-l="-130"><H2>Team</H2><UL><LI><A href="/Team:Heidelberg/Team">Team Members</A></LI><LI><A href="/Team:Heidelberg/AttributionsII">Attributions</A></LI><LI><A href="/Team:Heidelberg/Team/Sponsors">Sponsors</A></LI><LI><A href="/Team:Heidelberg/Team/Collaboration">Collaborations</A></LI><LI><A href="/Team:Heidelberg/Team/Gallery">Gallery</A></LI></UL></DIV><DIV class="item project" data-offset-l="-144"><H2>Project</H2><UL><LI><A href="/Team:Heidelberg/NRPS">Background</A></LI><LI><A href="/Team:Heidelberg/Project">Overview</A></LI><LI><A href="/Team:Heidelberg/Project/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Project/Indigoidine-Tag">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Project/Tag-Optimization">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Project/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Project/Achievements">Achievements</A></LI></UL></DIV><DIV class="item notebook" data-offset-l="-139"><H2>Notebook</H2><UL><LI><A href="/Team:Heidelberg/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Tyrocidine">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Indigoidine">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/NRPSDesigner">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/Notebook/Methods">Methods</A></LI><LI><A href="/Team:Heidelberg/Materials">Materials</A></LI></UL></DIV><DIV class="item parts" data-offset-l="-134"><H2>Parts</H2><UL><LI><A href="/Team:Heidelberg/Parts">Submitted Parts</A></LI><LI><A href="/Team:Heidelberg/Favorite_Parts">Favorite Parts</A></LI><LI><A href="/Team:Heidelberg/RFCs">Our RFCs</A></LI></UL></DIV><DIV class="item software" data-offset-l="-134"><H2>Software</H2><UL><LI><A href="/Team:Heidelberg/Project_Software">Project</A></LI><LI><A href="http://igem2013.bioquant.uni-heidelberg.de/NRPSDesigner/">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/iGEM42">iGEM42</A></LI><LI><A href="/Team:Heidelberg/OpenSource">Open Source</A></LI><LI><A href="/Team:Heidelberg/Project/AchievementsSoftware">Achievements</A></LI></UL></DIV><DIV class="item human" data-offset-l="-118"><H2>Human Practice</H2><UL><LI><A href="/Team:Heidelberg/HumanPractice">Overview</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BroadPublic">Broad Public</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Survey">Survey</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/YoungGeneration">Young Generation</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Artists">Arts and Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BeyondScience">Beyond Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Experts">Experts</A></LI></UL></DIV><DIV class="item safety" data-offset-l="-115"><H2>Safety</H2><UL><LI><A href="/Team:Heidelberg/Safety">Safety</A></LI></UL></DIV><DIV class="item model" data-offset-l="-144"><H2>Modeling</H2><UL><LI><A href="/Team:Heidelberg/Modelling/Gold_Recovery">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Modelling/Ind_Production">ind-Production</A></LI></UL></DIV><DIV class="item"><SPAN style="margin-left:6%">HOME</SPAN></DIV></DIV></DIV><DIV class="container"><DIV><H1><SPAN style="font-size:180%;color:#074E0B;">NRPS-Designer.</SPAN><SPAN class="text-muted" style="font-family:Arial, sans-serif; font-size:100%"> Designing custom peptides.</SPAN></H1><P>This tool allows you to design any customized peptide you can build using the most common NRPS domains. This gives you the oppurtunity to include non-proteinogenic amino acids and secondary modifications without going through chemical synthesis or tRNA reprogramming. On top of providing you with a target pathway sequence the tool is also integrated with the parts registry to include any further BioBricks and with Gibthon to directly go from NRP design to Gibson cloning strategy within 10 minutes.</P><P>You should also read our <A href="https://2013.igem.org/Team:Heidelberg/Project_Software">documentation</A> and our <A href="https://2013.igem.org/Team:Heidelberg/RFCs"><B>RFC100</B></A>!
            </P></DIV><DIV class="row"><DIV class="col-sm-4"><UL class="pagination" style="margin-bottom:2%; margin-left:4%;"><LI><A href="#" id="backwards">«</A></LI><LI class="active month_tab" id="july"><A href="#" style="width:90px; text-align:center">July</A></LI><LI class="month_tab" id="august"><A href="#" style="width:90px; text-align:center">August</A></LI><LI class="month_tab" id="september"><A href="#" style="width:90px; text-align:center">September</A></LI><LI><A href="#" id="forwards">»</A></LI></UL><DIV style="margin-bottom:8%;"><DIV id="myCarousel" class="carousel slide"><DIV class="carousel-inner" style="margin-top:26%"><DIV class="item active july first"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 12</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">In this week we implemented a first draft of the NRPS-Designer algorithm in C++ and designed a databse structure, as well as some scripts for entry making</P></DIV></DIV></DIV><DIV class="item july"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 13</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">While revising the database structure and the input interface we integrated pfam for semi-automated domain prediction.</P></DIV></DIV></DIV><DIV class="item july last"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 14</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">Since we noticed, that writing the whole interface ourselves will be very nasty and time-consuming we put our whole concept into the framework Django. Meanwhile our database design evolved further an the pfam integration was improved.</P></DIV></DIV></DIV><DIV class="item august first"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 15</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">Besides the regular changes to the database design, we integrated the graphical user interface in Django and also included OpenBable for displaying chemical structures. Furthermore we visualised our concept this week.</P></DIV></DIV></DIV><DIV class="item august"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 16</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">This week we mainly made changes to the NRP-Design interface and integrated ajax.</P></DIV></DIV></DIV><DIV class="item august"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 17</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">Besides fixing the relations in our database and improving th interface, we integrated Gibthon - the tool for Gibson primer design developed by the 2011 Cambridge team.</P></DIV></DIV></DIV><DIV class="item august last"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 18</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">This week we started filling our database, using the NRPS-PKS database. Besides that we introduced session data and updated the designer algorithm.</P></DIV></DIV></DIV><DIV class="item september first"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 19</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">As we noticed, that different prediction tools give different domain borders, we further evaluated them and finally integrated antiSMASH</P></DIV></DIV></DIV><DIV class="item september"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 20</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">This week we started the manual sequence annotation and database curation and integrated the multiple sequence alignment in the database input interface.</P></DIV></DIV></DIV><DIV class="item september"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 21</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">During this week we integrated our tool with the parts regisrty, did more manual database curation, added more hidden Markov-Models to antiSMASH and improved Gibthon.</P></DIV></DIV></DIV><DIV class="item september"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 22</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">In order to do request management we introduced celery to our system. Furthermore we optimised the multiple sequence alignment and did lots of manual database curation.</P></DIV></DIV></DIV><DIV class="item september last"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 23</H1><P style="font-size:10pt; text-align:justify; position:relative; margin-left:6%;">In the final week we added several linkouts, including SBOL and GenBank files and fixed lots of bugs. After the Regional Jamboree in Lyon, we fixed several bugs and most important we introduced a library function. This allows for construction of many similar NRPs at a time, which could for example be useful for medical compund screening.</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="jumbotron" data-spy="scroll" data-target="#navbarExample" data-offset="0" style="display:none;"><H2>Methods:</H2><P style="font-size:10pt; text-align:justify">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. .
                </P></DIV></DIV><DIV class="col-sm-8"><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#a12" data-toggle="tab">Initial setup</A></LI></UL></DIV><DIV class="jumbotron" style="height:820px"><DIV class="scrollContent nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a12"><H2><SPAN class="mw-headline" id="Initial_database_setup"> Initial database setup </SPAN></H2><P>First of all we began to design a database - which we modified, edited and changed all-over again numerous times - which ought to be capable of saving the data concerning NRPS origins, pathways, modules, specificity and lots of more information on them we needed for a later prediction of domains capable of producing a certain NRP. We focused on saving the data in a non-redundantly and easy to rerieve manner. Therefore we first set up a GitHub repository which contained the SQL-file of our first database-design. For it's implementation we used phpmyadmin, which spared us writing the whole thing in SQL. Creating forms to &quot;easily&quot; fill our newly implemented database we started with - what later turned out to be a quite naive idea - writing php scripts which connected to our MySQL-database using nasty index-files containing root-username and passwort of different localhosts.
</P><H2><SPAN class="mw-headline" id="Draft_of_command-line_application"> Draft of command-line application </SPAN></H2><P>We thought of a common XML format for NRPs. Additionally a first command line application was written in C++ for extracting data about an NRP from those XML files.
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#a13" data-toggle="tab">Design algorithm</A></LI><LI><A href="#b13" data-toggle="tab">Miscellaneous</A></LI></UL></DIV><DIV class="jumbotron" style="height:820px"><DIV class="scrollContent nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a13"><H2><SPAN class="mw-headline" id="Pathway_finding_implemented_in_C.2B.2B"> Pathway finding implemented in C++ </SPAN></H2><P>The basic C++ code needed for MySQL requests depending on a defined input was written. It fetches all the domains, that fit a certain substrate and calculates a pathway, which would theoretically produce the designed NRP.
                                </P></DIV><DIV class="tab-pane" id="b13"><H2><SPAN class="mw-headline" id="Database_redesign"> Database redesign </SPAN></H2><P>The database was redesigned in order to better represent the natural composition of NRPS.
</P><H2><SPAN class="mw-headline" id="Extensions_of_user_interface"> Extensions of user interface </SPAN></H2><P>The php-scripts for filling our database were adapted to the new database-structure and customized so that different tables in the database could be filled with them.
</P><H2><SPAN class="mw-headline" id="Nils_finally_uses_a_real_operating_system"> Nils finally uses a real operating system </SPAN></H2><P>After weeks of agony and trouble with his MacBook, Nils finally decided to follow the advice of master Ilia and made him install <A href="http://aptosid.com/" class="external text" rel="nofollow">aptosid</A> on his computer. Of course, because aptosid (based on Debian Sid) is not as user-friendly as Ubuntu, Nils and Ilia spent quite some time in order to change the background image from Fred the crab (the mascot of aptosid) to something else. Anyway, at least Nils has now a Linux distribution and maybe we will be able to convince our advisor Tim to also get rid of his Mac operating system.
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#a14" data-toggle="tab">Miscellaneous</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="scrollContent nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a14"><H2><SPAN class="mw-headline" id="Database_redesign_2"> Database redesign </SPAN></H2><P>The domains are now saved all in one table and they each have a domain type specified. Due to this setup the database can now also cover modification domains.
</P><H2><SPAN class="mw-headline" id="GUI_extension"> GUI extension </SPAN></H2><P>The php scripts were extended to allow for entries to the new database. Additionally an XML export funciton was added to the design interface.
</P><H2><SPAN class="mw-headline" id="Draft_of_PFAM_querying"> Draft of PFAM querying </SPAN></H2><P>In the experimental parts, we determined a first rough estimate of NRPS domain boundaries based on Pfam. Thus we also wanted our software to use the Pfam predictions to make it easier to semi-automatically add new domains into our database. A first test Python script was written which queried the Pfam API using the Requests library according to the online documentation and then parsed the XML output. The Biopython library also proved to be of great help.
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#a15" data-toggle="tab">Django</A></LI><LI><A href="#b15" data-toggle="tab">Miscellaneous</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="scrollContent nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a15"><H2><SPAN class="mw-headline" id="Concept_transferred_to_django_framework"> Concept transferred to django framework </SPAN></H2><P>Having distributed diverse tasks e.g. to write php scripts, we figured out that we would be constantly be reinventing the wheel and doing web-development like it was done a decade ago. Thus, we started looking at different web frameworks, which would make a lot of standardized tasks such as user login easy. We then decided to use Django, a Python framework, mainly because of the language. Some of us already knew Python, it is in general a language that is very easy to pick up and the Biopython project is pretty comprehensive. Additionally, a previously very successful iGEM Project (Gibthon) also used Django. Compared to most other choices (except Ruby on Rails, which has the disadvantage of being written in Ruby) it also seemed to be the most mature choice. We also thought of using a Perl or a Haskell framework, such as Yesod, but some team members disliked those languages (Haskellers disliked Perl and vice versa; the fact that the first Perl6 compiler was written in Haskell could not convince the Perl people either).
</P><P>We then started reading the Django tutorials and the (great!) documentation and started porting our previous MySQL database designs to Django models. This was made a lot easier by an automated Django utility which converts a database into a draft models file, which we could then adapt to our needs. The rest of the team was impressed after seeing the admin site, which again was very easy to get up and running with Django. This week our official lab-youtube-song was this <A href="http://www.youtube.com/watch?v=jMq5joFVxpQ" class="external text" rel="nofollow">one.</A></P></DIV><DIV class="tab-pane" id="b15"><H2><SPAN class="mw-headline" id="Database_redesign_3"> Database redesign </SPAN></H2><P>Working with Django models rather than MySQL tables made it a lot easier to get a better grasp of the structure of our database and possible shortcomings. In particular, we noticed that our original idea of an &quot;Origin&quot; representing a large DNA sequence was problematic, as it would life a lot harder in regards to automated prediction: For example, coding sequences would have to be extracted out of the DNA sequence using a tool like Glimmer or by running through all 6 reading frames as Pfam does. Also it was not immediately obvious how big a DNA sequence in the database could or should be (for example, a biosynthetic cluster for NRPS might be split into two or more &quot;Origins&quot; depending on who added the entries).
</P><P>Thus, we decided to change the models (which was very easy with Django) as follows: The &quot;Origin&quot; table now represented an actual source of DNA, out of which a domain could be amplified by PCR, such as a particular species or a biobrick. The DNA sequence were now saved in the new &quot;Cds&quot; (Coding sequence) table. The foreign key of the domain table entries also pointed at &quot;Cds&quot; entries rather than &quot;Origins&quot;.
</P><H2><SPAN class="mw-headline" id="PFAM_integration"> PFAM integration </SPAN></H2><P>Having learned a lot about one of the core concepts of Django, namely the models, we also started experimenting with the views and template systems, which basically are responsible for the rendering of the actual web pages. Thus, we tried to integrate the Pfam scripts we had previously written into Django views. Pfam actually includes a very nice help page (with gems such as: &quot; IE, being IE, needs extra help [..] &quot;)for its <A href="http://pfam.sanger.ac.uk/help#tabview=tab9" class="external text" rel="nofollow">domain graphics</A>, which basically beautifully render JSON documents using Prototype.js and other JS libraries. The JSON representation of the different domains is also calculated by the Pfam server after the domain prediction. Therefore, we adapted the original Pfam script in order to actually extract this JSON file and then it was passed on to the Pfam domain graphics javascript. This had to be done asynchronously (user enters DNA sequence, presses button, then the sequence is passed to the server, the Django function which queries Pfam API is executed and finally the Pfam JSON File is returned to the user's browser). Here, the extensibility of Django via reusable apps came in handy, because the <A href="http://www.dajaxproject.com/dajaxice/" class="external text" rel="nofollow">Dajaxice</A> app made the AJAX requests a matter of a few lines of code.
</P><H2><SPAN class="mw-headline" id="Database_entries"> Database entries </SPAN></H2><P>In order to fill the database with the standard monomers, the L- and D-conformation of the 20 proteinogenic amino acids, as well as those of ornithine were taken from pubchem and entered into the database via the standard admin page. To allow for displaying the structures in the user interface the 3D-SDF formatted text files were also copied to the database.
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#a16" data-toggle="tab">Overview</A></LI><LI><A href="#b16" data-toggle="tab">Miscellaneous</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="scrollContent nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a16"><H2><SPAN class="mw-headline" id="Concept_visualisation"> Concept visualisation </SPAN></H2><P>For presentation purposes and in order to have a visual representation of all the different aspects of our project we drew a huge graphic on the blackboard. After two days of revision we redrew the whole thing using inkscape. The result is displayed in figure 16.1.
</P></DIV><DIV class="tab-pane" id="b16"><H2><SPAN class="mw-headline" id="Graphical_user_interface_and_Django-registration"> Graphical user interface and Django-registration </SPAN></H2><P>This week we started trying to get the basics necessary for a decent user experience up and running. The pages for the Pfam input and the selection of NRP monomers were created. For this we used <A href="http://jqueryui.com/" class="external text" rel="nofollow">jQuery UI</A> and we very especially fond of its tabs. After playing around with some html and css, we soonly decided to integrate <A href="http://getbootstrap.com/2.3.2/" class="external text" rel="nofollow">bootstrap 2</A>for styling our interface. As our Django models and forms would deal a lot with many-to-many relationships and as the default multichoice html box is extremely user-unfriendly, we searched for JS libraries capable  of improving this experience. We initially decided to use <A href="http://harvesthq.github.io/chosen/" class="external text" rel="nofollow">chosen.js</A>, but then we quickly switched to <A href="http://ivaynberg.github.io/select2/" class="external text" rel="nofollow">select2</A>, which provides many more functionalities.
</P><P>Also in order to enable the smooth handling of user registration, login etc., the <A href="https://django-registration.readthedocs.org" class="external text" rel="nofollow">django-registration</A> was roughly integrated to the rest of the software.
</P><H2><SPAN class="mw-headline" id="Use_of_OpenBable_for_structure_display"> Use of OpenBable for structure display </SPAN></H2><P>OpenBable was included in the tool in order to display the currently designed NRP. The peptide bonds are formed automatically based on the SDF representations saved for each substrate in the database. One of the challenges faced was the correct conformation and the aesthetic display of the peptide chain.
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#a17" data-toggle="tab">Miscellaneous</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="scrollContent nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a17"><H2><SPAN class="mw-headline" id="Monomer_selection_page_aesthetic_improvements"> Monomer selection page aesthetic improvements </SPAN></H2><P>This week we focused on improving the aesthetics of the NRP selection page. Most of the page layout was shuffled around, so that everything important (Monomer selection, buttons, peptide graphic) fit onto a laptop screen and no scrolling around is necessary. For this we also removed the help text, which added a lot of bloat to the page. We'll try to find a fancier way to document things and help the user later on. We also used a nice feature of s<A href="http://ivaynberg.github.io/select2/" class="external text" rel="nofollow">select2</A>, which basically allows you to determine how the dropdown selection options are styled. More specifically, each monomer is displayed together with its 2d structure as generated by the OpenBabel script we had already written.  
</P><H2><SPAN class="mw-headline" id="AJAX_functionality"> AJAX functionality </SPAN></H2><P>In order to facilitate the communication of the web interface with the C++ code, we started testing the AJAX functionality. One thing we noticed was that the (otherwise great) Dajaxice library, can actually cause problems due to relative URLs, as it does not correctly utilize the wsgi settings. This was especially problematic, as the development and production environments differed, so that a very ugly hack would have to be written for this to work. Instead, we decided to use a more barebones/standard AJAX implementation by actually using jQuery. A test was written with the monomer selection being passed via AJAX and then a popup alert (yay!) which produced an XML out of this selection. This XML could be passed into the C++ function.
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#a18" data-toggle="tab">Interface</A></LI><LI><A href="#c18" data-toggle="tab">Gibthon</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="scrollContent nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a18"><H2><SPAN class="mw-headline" id="Designer_interface_-_modifications"> Designer interface - modifications </SPAN></H2><P>A selection box for the modifications a substrate can have was added to the designer interface. This will also be adopted for the chemical structure and the domains will be included in the NRPS design.
                                </P></DIV><DIV class="tab-pane" id="c18"><H2><SPAN class="mw-headline" id="Integration_of_Gibthon"> Integration of Gibthon </SPAN></H2><P>The Gibthon software by Cambridge 2010 was integrated in our software to provide a cloning strategy for the user. Some features (such as the primer folding analysis) were excluded at first, in order to prevent possible bugs and to runtime when testing the funtionality. During this process of thesting, we also ran into bugs in Gibthon and fixed them in order to improve both tools.
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#b19" data-toggle="tab">Database</A></LI><LI><A href="#c19" data-toggle="tab">Frontend</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="scrollContent nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane" id="b19"><H2><SPAN class="mw-headline" id="Database_updates"> Database updates </SPAN></H2><P>As many organisms have several NRPS pathways and some NRPS pathways are present in different organisms (e.g. different bacterial strains) the database was changed in order to separate origin and product. The coding sequences now function as many to many connection between origin and products.
</P><H2><SPAN class="mw-headline" id="Integrating_NRPS-PKS"> Integrating NRPS-PKS </SPAN></H2><P>We manually put all the organisms and pathways annotated in NRPS-PKS in our database. This basis was then used to write simple scraping script.
                                </P></DIV><DIV class="tab-pane active" id="c19"><H2><SPAN class="mw-headline" id="Session_data"> Session data </SPAN></H2><P>The user can now have his own personalised data kept in the database. This contains peptides designed and the results of the domains and primer predictions. An additional interface for the organisation of the NRPs was created.
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#a20" data-toggle="tab">antiSMASH</A></LI><LI><A href="#b20" data-toggle="tab">Miscellaneous</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="scrollContent nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a20"><H2><SPAN class="mw-headline" id="Tool_selection_for_domain_prediciton"> Tool selection for domain prediciton </SPAN></H2><P>We tried to determine, which tool would be most appropriate for automated determination of domains. Thus we used the first CDS of the teicoplanin NRPS, as curated in the NRPS-PKS (SBSPKS) database. We then compared the automated prediction with the tool of Maryland, as well as antiSMASH2.
</P><TABLE class="wikitable" style="font-size:9pt;"><TBODY><TR><TH colspan="2">A domain </TH><TH colspan="2">T domain </TH><TH colspan="2">C domain </TH><TH colspan="2">A domain </TH><TH colspan="2">T domain </TH><TH colspan="2">E domain
</TH></TR><TR><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End
</TH></TR><TR><TD> NRPS-PKS</TD><TD> 1 </TD><TD> 491 </TD><TD> 505 </TD><TD> 564 </TD><TD> 604 </TD><TD> 1021 </TD><TD> 1015 </TD><TD> 1511 </TD><TD> 1526 </TD><TD> 1584 </TD><TD> 1608 </TD><TD> 2053
</TD></TR><TR><TD> Maryland </TD><TD> 18 </TD><TD> 503 </TD><TD> 504 </TD><TD> 567 </TD><TD> 599 </TD><TD> 1012 </TD><TD> 1008 </TD><TD> 1524 </TD><TD> 1524 </TD><TD> 1587 </TD><TD> 1604 </TD><TD> 2063
</TD></TR><TR><TD> antiSMASH </TD><TD> 35 </TD><TD> 426 </TD><TD> 502 </TD><TD> 567 </TD><TD> 598 </TD><TD> 891 </TD><TD> 1056 </TD><TD> 1447 </TD><TD> 1522 </TD><TD> 1589 </TD><TD> 1602 </TD><TD> 1896
</TD></TR><TR><TD> pfam </TD><TD> 15 </TD><TD> 478 </TD><TD> 504 </TD><TD> 566 </TD><TD> 598 </TD><TD> 891 </TD><TD> 1036 </TD><TD> 1498 </TD><TD> 1524 </TD><TD> 1586  </TD><TD> 1620  (as C)</TD><TD> 1896 (as C)
</TD></TR></TBODY></TABLE><P>Note that Pfam can't differentiate between C and E domains. Pfam also does not return any predictions in regard to A-domain specificity. antiSMASH is able to predict the A-domain specificities as annotated in NRPS-PKS, while Maryland's tool can only predict the second A-domain (Tyrosine), while the first is incorrectly predicted to be specific for Leucine (curated: HpG, 4-hydroxyphenyl glycine).
</P><P>The same analysis was also repeated for thaxtomin.
</P><TABLE class="wikitable" style="font-size:9pt;"><TBODY><TR><TH colspan="2">A domain </TH><TH colspan="2">NM domain </TH><TH colspan="2">T domain </TH><TH colspan="2">C domain
</TH></TR><TR><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End
</TH></TR><TR><TD> NRPS-PKS</TD><TD> 1 </TD><TD> 506 </TD><TD> 509 </TD><TD> 712 </TD><TD> 944 </TD><TD> 1005 </TD><TD> 1029 </TD><TD> 1458
</TD></TR><TR><TD> Maryland </TD><TD> 5 </TD><TD> 521 </TD><TD> 468 </TD><TD> 879 </TD><TD> 942 </TD><TD> 1006 </TD><TD> 1028 </TD><TD> 1455
</TD></TR><TR><TD> antiSMASH </TD><TD> 42 </TD><TD> 445 </TD><TD> 511 </TD><TD> 732 </TD><TD> 940 </TD><TD> 1007 </TD><TD> 1025 </TD><TD> 1325
</TD></TR><TR><TD> pfam </TD><TD> 22 </TD><TD> 486 </TD><TD> 538 </TD><TD> 638 </TD><TD> 942 </TD><TD> 1006 </TD><TD> 1023 </TD><TD> 1326
</TD></TR></TBODY></TABLE><P>In this case, both tools correctly predict the L-Phenylalanine specificity of the A-domain.
</P><P>Analysis of tycC from the tyrocidine-cluster.
</P><TABLE class="wikitable" style="font-size:9pt;"><TBODY><TR><TH colspan="2">C domain </TH><TH colspan="2">A domain </TH><TH colspan="2">T domain </TH><TH colspan="2">C domain </TH><TH colspan="2">A domain </TH><TH colspan="2">T domain </TH><TH colspan="2">C domain </TH><TH colspan="2">A domain </TH><TH colspan="2">T domain </TH><TH colspan="2">C domain </TH><TH colspan="2">A domain </TH><TH colspan="2">T domain </TH><TH colspan="2">C domain </TH><TH colspan="2">A domain </TH><TH colspan="2">T domain </TH><TH colspan="2">C domain </TH><TH colspan="2">A domain </TH><TH colspan="2">T domain </TH><TH colspan="2">TE domain
</TH></TR><TR><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End
</TH></TR><TR><TD> Marahiel (C-T), Ilia (A-T), Philipp (C-A)</TD><TD> 953 </TD><TD> 963 </TD><TD> 1033 </TD><TD> 1050 </TD><TD> 1345 </TD><TD> 1990 </TD><TD> 2000 </TD><TD> 2070 </TD><TD> 2087 </TD><TD> 3028</TD><TD> 3038</TD><TD> 3108 </TD><TD> 3125 </TD><TD> 3571 </TD><TD>  4063 </TD><TD> 4073 </TD><TD> 4143 </TD><TD> 4160 </TD><TD> 5107</TD><TD> 5117 </TD><TD> 5187 </TD><TD> 5204 </TD><TD> 6150 </TD><TD> 6160 </TD><TD> 6230 </TD><TD> 6247 </TD></TR><TR><TD> NRPS-PKS</TD><TD> 13 </TD><TD> 447 </TD><TD> 442 </TD><TD> 957 </TD><TD> 972 </TD><TD> 1036 </TD><TD> 1058 </TD><TD> 1483 </TD><TD> 1478 </TD><TD> 1994 </TD><TD> 2009 </TD><TD> 2073 </TD><TD> 2095 </TD><TD> 2520 </TD><TD> 2515 </TD><TD> 3032 </TD><TD> 3048 </TD><TD> 3111 </TD><TD> 3133 </TD><TD> 3558 </TD><TD> 3553 </TD><TD> 4066 </TD><TD> 4082 </TD><TD> 4146 </TD><TD> 4168 </TD><TD> 4593 </TD><TD> 4588 </TD><TD> 5111 </TD><TD> 5126 </TD><TD> 5190 </TD><TD> 5212 </TD><TD> 5639 </TD><TD> 5636 </TD><TD> 6151 </TD><TD> 6169 </TD><TD> 6233 </TD><TD> 6256 </TD><TD> 6466 </TD></TR><TR><TD> Maryland </TD><TD> 8 </TD><TD> 438 </TD><TD> 441 </TD><TD> 964 </TD><TD>  972 </TD><TD>1036 </TD><TD>1053 </TD><TD> 1474 </TD><TD> 1477 </TD><TD> 2001 </TD><TD> 2009 </TD><TD> 2073 </TD><TD> 2090 </TD><TD> 2511 </TD><TD> 2516 </TD><TD> 3039 </TD><TD> 3047 </TD><TD> 3111 </TD><TD> 3128 </TD><TD> 3549</TD><TD> 3552 </TD><TD> 4074 </TD><TD> 4082 </TD><TD> 4146 </TD><TD> 4163 </TD><TD> 4584 </TD><TD> 4587 </TD><TD> 5118 </TD><TD> 5126 </TD><TD> 5190 </TD><TD> 5207 </TD><TD> 5630 </TD><TD> 5637 </TD><TD> 6160 </TD><TD> 6169 </TD><TD> 6233 </TD><TD> 6254 </TD><TD> 6482
</TD></TR><TR><TD> antiSMASH </TD><TD> 8 </TD><TD> 305 </TD><TD> 490 </TD><TD> 887 </TD><TD> 970  </TD><TD> 1039 </TD><TD> 1054 </TD><TD> 1339 </TD><TD> 1526 </TD><TD> 1924 </TD><TD> 2007 </TD><TD> 2075 </TD><TD> 2090 </TD><TD> 2376 </TD><TD> 2563 </TD><TD> 2962 </TD><TD> 3045 </TD><TD> 3113 </TD><TD> 3128 </TD><TD> 3413 </TD><TD> 3601 </TD><TD> 3997 </TD><TD> 4080 </TD><TD> 4149 </TD><TD> 4165 </TD><TD> 4449 </TD><TD> 4636 </TD><TD> 5041 </TD><TD> 5124 </TD><TD> 5193 </TD><TD> 5208 </TD><TD> 5494 </TD><TD> 5682 </TD><TD> 6083 </TD><TD> 6167 </TD><TD> 6236 </TD><TD> 6253 </TD><TD> 6482
</TD></TR><TR><TD> pfam </TD><TD>  7 </TD><TD>  306 </TD><TD>  470 </TD><TD>  944 </TD><TD> 972 </TD><TD> 1036 </TD><TD> 1052 </TD><TD> 1341 </TD><TD> 1506 </TD><TD> 1981 </TD><TD> 2009 </TD><TD> 2073 </TD><TD> 2089 </TD><TD> 2378 </TD><TD> 2543 </TD><TD> 3019 </TD><TD> 3047 </TD><TD> 3111 </TD><TD> 3127 </TD><TD> 3416 </TD><TD> 3581 </TD><TD> 4054 </TD><TD> 4082 </TD><TD> 4146 </TD><TD> 4162 </TD><TD> 4450 </TD><TD> 4616 </TD><TD> 5098 </TD><TD> 5126 </TD><TD> 5190 </TD><TD> 5206 </TD><TD> 5495 </TD><TD> 5662 </TD><TD> 6141 </TD><TD> 6169 </TD><TD> 6233 </TD><TD> 6254 </TD><TD> 6482
</TD></TR></TBODY></TABLE><P>In regards to prediction of A-domain specificity, antiSMASH predictions and the curated NRPS-PKS amino acids were the same. On the other hand, Maryland predicted did not get any hit for A6 (Leu) and for A1/A3 respectively it predicted to possible amino acids (Asn+Asp compared to Asn / Tyr + Trp compared to Tyr in antiSMASH and NRPS-PKS).
</P><P>TycB3 C-A domain 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Clustal Omega MSA of different C-A domain borders of Tyrocidine cluster with annotation (prediction) of the start of the A domain according to different tools.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Clustal Omega MSA of different C-A domain borders of Tyrocidine cluster with annotation (prediction) of the end of the C domain according to different tools.</DIV></DIV></DIV><TABLE class="wikitable" style="font-size:9pt;"><TBODY><TR><TH colspan="2">C domain </TH><TH colspan="2">A domain
</TH></TR><TR><TH> Start </TH><TH> End </TH><TH> Start </TH><TH> End
</TH></TR><TR><TD> paper primer position </TD><TD colspan="4"> 2520 (MLTAA..)
</TD></TR><TR><TD> NRPS-PKS</TD><TD> 2100 </TD><TD> 2527 </TD><TD> 2540 </TD><TD> 3029
</TD></TR><TR><TD> Maryland </TD><TD>  2095 </TD><TD> 2518 </TD><TD> 2521 </TD><TD> 3039
</TD></TR><TR><TD> antiSMASH </TD><TD> 2096</TD><TD> 2381 </TD><TD> 2570 </TD><TD> 2961
</TD></TR><TR><TD> pfam </TD><TD> 2094 </TD><TD> 2383 </TD><TD> 2550 </TD><TD> 3019
</TD></TR></TBODY></TABLE><P>== Inegration of antiSMASH ==
                                </P></DIV><DIV class="tab-pane" id="b20"><H2><SPAN class="mw-headline" id="Domain_visualisation_using_pfam"> Domain visualisation using pfam </SPAN></H2><P>We added the domain visualisation as it is in pfam to the domain types in the database in order to display the resulting NRPS structure.
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#a21" data-toggle="tab">Misc</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="scrollContent nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a21"><H2><SPAN class="mw-headline" id="NRPS-PKS_sequence_search_and_manual_curation"> NRPS-PKS sequence search and manual curation </SPAN></H2><P>To find the right DNA sequences for the pathways we blasted the protein sequences and looked for those DNA-Clusters exactly matching all protien sequences annotade on NRPS-PKS
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#a22" data-toggle="tab">Misc</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="scrollContent nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a22"><P style="font-size:12pt; text-align:justify;">
                              Please see our documentation for our final results.
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#a23" data-toggle="tab">Misc</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="scrollContent nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a23"><H2><SPAN class="mw-headline" id="SBOL_and_GenBank"> SBOL and GenBank </SPAN></H2><P>Proper output of the designed NRPS as well as the primers, was created both for GenBank and for SBOL. Moreover the tool internal communication was shifted to SBOL.
</P><H2><SPAN class="mw-headline" id="Database_Curation"> Database Curation </SPAN></H2><P>The final state of the database after a major portion of SBSPKS was curated is displayed in the table below.
</P><TABLE class="wikitable" style="font-size:9pt;"><TBODY><TR><TD> Pathway</TD><TD>Gene</TD><TD>Module</TD><TD>SBSPKS domains</TD><TD>HMM domains</TD><TD>Specificity (Chirality C, TE)</TD><TD>Specificity (CA Substrate)</TD><TD>SBSPKS</TD><TD>SVM (0)</TD><TD>Stachelhaus (0)</TD><TD>Minowa (1)</TD><TD>Literature</TD><TD>Modifications</TD><TD>E-domain contained</TD><TD>Literature references</TD><TD>Comment
</TD></TR><TR><TD> Actinomycin</TD><TD>acmA</TD><TD>1</TD><TD>A</TD><TD>A</TD><TD>-</TD><TD>4-MHA</TD><TD>4-MHA</TD><TD>hydrophobic/alphatic</TD><TD>PIP</TD><TD>4-MHA</TD><TD>4-MHA</TD><TD>-</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pubmed/9573200" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/9573200</A></TD><TD>&quot;T-domain after A is definitely missing C-starter can go with or without 4-MHA&quot;
</TD></TR><TR><TD>acmB</TD><TD>2</TD><TD>C-A-T</TD><TD>CSt-A-T</TD><TD>-/4-MHA</TD><TD>Thr</TD><TD>Thr</TD><TD>Thr</TD><TD>Thr</TD><TD>Thr</TD><TD>Thr</TD><TD>-</TD></TR><TR><TD>3</TD><TD>C-A-T-E</TD><TD>C-A-T-E-com</TD><TD>L</TD><TD>Val</TD><TD>D-Val</TD><TD>Val</TD><TD>Val</TD><TD>Val</TD><TD>Val</TD><TD>-</TD><TD>X</TD></TR><TR><TD>acmC</TD><TD>4</TD><TD>C-A-T</TD><TD>com-C-A-T</TD><TD>D</TD><TD>Pro</TD><TD>Pro</TD><TD>Pro</TD><TD>Pro</TD><TD>Pro</TD><TD>Pro</TD><TD>-</TD></TR><TR><TD>5</TD><TD>C-A-M-T</TD><TD>C-A-NM-T</TD><TD>L</TD><TD>Gly</TD><TD>Gly</TD><TD>Gly</TD><TD>Gly</TD><TD>Gly</TD><TD>Gly</TD><TD>N-Methylierung</TD></TR><TR><TD>6</TD><TD>C-A-M-T-TE</TD><TD>C-A-NM-T-TE</TD><TD>L, L</TD><TD>Val</TD><TD>Val</TD><TD>Val</TD><TD>Val</TD><TD>Val</TD><TD>Val</TD><TD>N-Methylierung</TD></TR><TR><TD> ACV</TD><TD>acvA</TD><TD>1</TD><TD>A-T</TD><TD>A-T</TD><TD>-</TD><TD>Aad</TD><TD>Aad</TD><TD>Aad</TD><TD>Aad</TD><TD>Aad</TD><TD>Aad</TD><TD>-</TD><TD>&quot;A: <A href="http://www.ncbi.nlm.nih.gov/pubmed/9266851" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/9266851</A>E: <A href="http://www.ncbi.nlm.nih.gov/pubmed/21889568" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/21889568</A>TE: <A href="http://www.ncbi.nlm.nih.gov/pubmed/10715209" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/10715209</A> &quot;</TD></TR><TR><TD>2</TD><TD>C-A-T</TD><TD>C-A-T</TD><TD>D (pred) / L (sbspks, lit)</TD><TD>Cys</TD><TD>Cys</TD><TD>Cys</TD><TD>Cys</TD><TD>Cys</TD><TD>Cys</TD><TD>-</TD></TR><TR><TD>3</TD><TD>C-A-T-E-TE</TD><TD>C-A-T-E-TE</TD><TD>L, D</TD><TD>Val</TD><TD>Val-D</TD><TD>Val</TD><TD>Val</TD><TD>Val</TD><TD>Val</TD><TD>-</TD><TD>X</TD></TR><TR><TD> A47934</TD><TD>staA</TD><TD>1</TD><TD>A-T</TD><TD>A-T</TD><TD>HpG</TD><TD>DHpG</TD><TD>HpG</TD><TD>HpG</TD><TD>HpG</TD><TD>HpG</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pubmed/12060705" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/12060705</A></TD><TD>&quot;- staB E-domain is nonfunctional because of His-Pro mutation - CX in the end is either non-functional or L-selective&quot;
</TD></TR><TR><TD>2</TD><TD>C-A-T-E</TD><TD>Cglyc-A-T-E</TD><TD>D</TD><TD>Tyr</TD><TD>Tyr</TD><TD>Tyr</TD><TD>Tyr</TD><TD>Tyr</TD><TD>Tyr</TD><TD>C-glyc linking hydroxy</TD><TD>X</TD></TR><TR><TD>staB</TD><TD>3</TD><TD>C-A-T-E</TD><TD>Cglyc-A-T-E*-com</TD><TD>D</TD><TD>DHpG</TD><TD>HpG</TD><TD>DHpG</TD><TD>DHpG</TD><TD>DHpG</TD><TD>DHpG</TD><TD>C-glyc linking hydroxy</TD><TD>X</TD></TR><TR><TD>staC</TD><TD>4</TD><TD>C-A-T-E</TD><TD>com-Cglyc-A-T-E</TD><TD>L</TD><TD>HpG</TD><TD>HpG2Cl</TD><TD>HpG</TD><TD>HpG</TD><TD>HpG</TD><TD>HpG</TD><TD>C-glyc linking hydroxy</TD><TD>X</TD></TR><TR><TD>5</TD><TD>C-A-T-E</TD><TD>Cglyc-A-T-E</TD><TD>D</TD><TD>HpG</TD><TD>HpG</TD><TD>HpG</TD><TD>HpG</TD><TD>HpG</TD><TD>HpG</TD><TD>C-glyc linking hydroxy</TD><TD>X</TD></TR><TR><TD>6</TD><TD>C-A-T</TD><TD>Cglyc-A-T</TD><TD>D</TD><TD>bht</TD><TD>Tyrb-O</TD><TD>bht</TD><TD>bht</TD><TD>bht</TD><TD>bht</TD><TD>C-glyc linking hydroxy</TD></TR><TR><TD>staD</TD><TD>7</TD><TD>C-A-T-E-TE</TD><TD>com-Cglyc-A-T-CX-TE</TD><TD>L, L</TD><TD>DHpG</TD><TD>DHpG</TD><TD>DHpG</TD><TD>DHpG</TD><TD>DHpG</TD><TD>DHpG</TD><TD>C-glyc linking hydroxy</TD></TR><TR><TD> Arthrofactin</TD><TD>arfA</TD><TD>1</TD><TD>C-A-T</TD><TD>C-A-T</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pubmed/14522057" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/14522057</A></TD><TD>Paper claims that there are epimerisation domains, but the dual domains sound more reasonable.
</TD></TR><TR><TD>2</TD><TD>C-A-T</TD><TD>CDual-A-T</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD><TD>X</TD></TR><TR><TD>arfB</TD><TD>3</TD><TD>C-A-T</TD><TD>CDual-A-T</TD><TD>Thr</TD><TD>Thr</TD><TD>Thr</TD><TD>Thr</TD><TD>Thr</TD><TD>Thr</TD><TD>X</TD></TR><TR><TD>4</TD><TD>C-A-T</TD><TD>CDual-A-T</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>X</TD></TR><TR><TD>5</TD><TD>C-A-T</TD><TD>CDual-A-T</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>X</TD></TR><TR><TD>6</TD><TD>C-A-T</TD><TD>CDual-A-T</TD><TD>Ser</TD><TD>Ser</TD><TD>Ser</TD><TD>Ser</TD><TD>Ser</TD><TD>Ser</TD><TD>X</TD></TR><TR><TD>arfC</TD><TD>7</TD><TD>C-A-T</TD><TD>CDual-A-T</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>X</TD></TR><TR><TD>8</TD><TD>C-A-T</TD><TD>C-A-T</TD><TD>L</TD><TD>Ser</TD><TD>Ser</TD><TD>Ser</TD><TD>Ser</TD><TD>Ser</TD><TD>Ser</TD></TR><TR><TD>9</TD><TD>C-A-T</TD><TD>CDual-A-T</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile</TD><TD>X</TD></TR><TR><TD>10</TD><TD>C-A-T</TD><TD>C-A-T</TD><TD>L</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile</TD></TR><TR><TD>11</TD><TD>C-A-T-TE-TE</TD><TD>C-A-T-TE-TE</TD><TD>L,L</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD></TR><TR><TD> Bacitracin</TD><TD>bacA</TD><TD>1</TD><TD>A-T</TD><TD>A-T</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pubmed/9427658" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/9427658</A></TD></TR><TR><TD>2</TD><TD>C-A-T</TD><TD>Cy-A-T</TD><TD>L</TD><TD>Cys</TD><TD>Cys</TD><TD>Cys</TD><TD>Cys</TD><TD>Cys</TD><TD>Cys</TD></TR><TR><TD>3</TD><TD>C-A-T</TD><TD>C-A-T</TD><TD>L</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD></TR><TR><TD>4</TD><TD>C-A-T-E</TD><TD>C-A-T-E</TD><TD>L</TD><TD>Glu</TD><TD>Glu</TD><TD>Glu</TD><TD>Glu</TD><TD>Glu</TD><TD>D-Glu</TD></TR><TR><TD>5</TD><TD>C-A-T</TD><TD>C-A-T</TD><TD>D</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile</TD></TR><TR><TD>bacB</TD><TD>6</TD><TD>C-A-T</TD><TD>C-A-T</TD><TD>L</TD><TD>Lys</TD><TD>Lys</TD><TD>Lys</TD><TD>Lys</TD><TD>Lys</TD><TD>Lys</TD></TR><TR><TD>7</TD><TD>C-A-T-E</TD><TD>C-A-T-E</TD><TD>L</TD><TD>Orn</TD><TD>Orn</TD><TD>Orn</TD><TD>Orn</TD><TD>Orn</TD><TD>D-Orn</TD></TR><TR><TD>bacC</TD><TD>8</TD><TD>C-A-T</TD><TD>com-C-A-T</TD><TD>D</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile</TD></TR><TR><TD>9</TD><TD>C-A-T-E</TD><TD>C-A-T-E</TD><TD>L</TD><TD>Phe</TD><TD>Phe</TD><TD>Phe</TD><TD>Phe</TD><TD>Phe</TD><TD>D-Phe</TD></TR><TR><TD>10</TD><TD>C-A-T</TD><TD>C-A-T</TD><TD>D</TD><TD>His</TD><TD>His</TD><TD>Tyr</TD><TD>His</TD><TD>His</TD><TD>His</TD></TR><TR><TD>11</TD><TD>C-A-T-E</TD><TD>C-A-T-E</TD><TD>L</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD><TD>D-Asp</TD></TR><TR><TD>12</TD><TD>C-A-T-TE</TD><TD>C-A-T-TE</TD><TD>D, L</TD><TD>Asn</TD><TD>Asn</TD><TD>Asn</TD><TD>Asn</TD><TD>Asn</TD><TD>Asn</TD></TR><TR><TD> CDA</TD><TD>cdaPSI</TD><TD>1</TD><TD>C-A-T</TD><TD>CSt-A-T</TD><TD>Ser</TD><TD>Ser</TD><TD>Ser</TD><TD>Ser</TD><TD>Ser</TD><TD>Ser</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pubmed/12445768" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/12445768</A></TD><TD>Not sure why Cglyc instead of C since there is no modification here.
</TD></TR><TR><TD>2</TD><TD>C-A-T</TD><TD>C-A-T</TD><TD>L</TD><TD>Thr</TD><TD>Thr</TD><TD>Thr</TD><TD>Thr</TD><TD>Thr</TD><TD>Thr</TD></TR><TR><TD>3</TD><TD>C-A-E</TD><TD>C-A-T-E</TD><TD>L</TD><TD>Trp</TD><TD>Trp</TD><TD>Trp</TD><TD>Trp</TD><TD>Trp</TD><TD>Trp</TD><TD>X</TD></TR><TR><TD>4</TD><TD>C-A-T</TD><TD>C-A-T</TD><TD>D</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD></TR><TR><TD>5</TD><TD>C-A-T</TD><TD>C-A-T</TD><TD>L</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD></TR><TR><TD>6</TD><TD>C-A-E</TD><TD>C-A-T-E</TD><TD>L</TD><TD>HpG</TD><TD>HpG</TD><TD>HpG</TD><TD>HpG</TD><TD>HpG</TD><TD>HpG</TD><TD>X</TD></TR><TR><TD>cdaPS2</TD><TD>7</TD><TD>C-A-T</TD><TD>Cglyc-A-T</TD><TD>D</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD><TD>no glycosylation here</TD></TR><TR><TD>8</TD><TD>C-A-T</TD><TD>C-A-T</TD><TD>L</TD><TD>Gly</TD><TD>Gly</TD><TD>Gly</TD><TD>Gly</TD><TD>Gly</TD><TD>Gly</TD></TR><TR><TD>9</TD><TD>C-A-E</TD><TD>C-A-T-E</TD><TD>L</TD><TD>Asn</TD><TD>Asn</TD><TD>Asn</TD><TD>Asn</TD><TD>Asn</TD><TD>Asn</TD><TD>X</TD></TR><TR><TD>cdaPS3</TD><TD>10</TD><TD>C-A-T</TD><TD>com-C-A-T</TD><TD>D</TD><TD>Glu/Glu3Me</TD><TD>GluMe3</TD><TD>Asp/Asn</TD><TD>Glu</TD><TD>Asn</TD><TD>GluMe3</TD></TR><TR><TD>11</TD><TD>C-A-T-TE</TD><TD>C-A-T-TE</TD><TD>L, L</TD><TD>Trp</TD><TD>Trp</TD><TD>Trp</TD><TD>Trp</TD><TD>Trp</TD><TD>Trp</TD></TR><TR><TD> Cyclosporin</TD><TD>simA</TD><TD>1</TD><TD>C-A-T</TD><TD>C-A-T</TD><TD>???</TD><TD>D-Ala</TD><TD>D-Ala</TD><TD>Pro</TD><TD>D-Ala</TD><TD>Ala</TD><TD>D-Ala</TD><TD>&quot;<A href="http://www.ncbi.nlm.nih.gov/pubmed/8376400" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/8376400</A>domain order: <A href="http://www.ncbi.nlm.nih.gov/pubmed/16895337" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/16895337</A>D-Ala:<A href="http://www.ncbi.nlm.nih.gov/pubmed/8175682" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/8175682</A>&quot;</TD><TD>final C-domain is never metioned anywhere, but since the product is cyclic it might have similar function as a thioesterase. Thus it is uncertain, how the stereochemistry of the first alanine is achieved.
</TD></TR><TR><TD>2</TD><TD>C-A-T</TD><TD>C-A-NM-T</TD><TD>L</TD><TD>Leu</TD><TD>Leu</TD><TD>val,leu,ile,abu,iva</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>N-Methylierung</TD></TR><TR><TD>3</TD><TD>C-A-T</TD><TD>C-A-NM-T</TD><TD>L</TD><TD>Leu</TD><TD>Leu</TD><TD>val,leu,ile,abu,iva</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>N-Methylierung</TD></TR><TR><TD>4</TD><TD>C-A-T</TD><TD>C-A-NM-T</TD><TD>L</TD><TD>Val</TD><TD>Val</TD><TD>val,leu,ile,abu,iva</TD><TD>Val</TD><TD>Val</TD><TD>Val</TD><TD>N-Methylierung</TD></TR><TR><TD>5</TD><TD>C-A-T</TD><TD>C-A-NM-T</TD><TD>L</TD><TD>Bmt</TD><TD>Bmt</TD><TD>phe,trp,phg,tyr,bht</TD><TD>Bmt</TD><TD>Bmt</TD><TD>Bmt</TD><TD>N-Methylierung</TD></TR><TR><TD>6</TD><TD>C-A-T</TD><TD>C-A-T</TD><TD>L</TD><TD>Abu</TD><TD>Abu</TD><TD>val,leu,ile,abu,iva</TD><TD>Abu</TD><TD>Abu</TD></TR><TR><TD>7</TD><TD>C-A-T</TD><TD>C-A-NM-T</TD><TD>L</TD><TD>Gly (NM &gt; Sar)</TD><TD>Glu</TD><TD>Pro</TD><TD>Gly</TD><TD>Sar</TD><TD>sarcosine</TD><TD>N-Methylierung</TD></TR><TR><TD>8</TD><TD>C-A-T</TD><TD>C-A-NM-T</TD><TD>L</TD><TD>Leu</TD><TD>Leu</TD><TD>val,leu,ile,abu,iva</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>N-Methylierung</TD></TR><TR><TD>9</TD><TD>C-A-T</TD><TD>C-A-T</TD><TD>L</TD><TD>Val</TD><TD>Val</TD><TD>Pro</TD><TD>Val</TD><TD>Val</TD><TD>Val</TD></TR><TR><TD>10</TD><TD>C-A-T</TD><TD>C-A-NM-T</TD><TD>L</TD><TD>Leu</TD><TD>Leu</TD><TD>val,leu,ile,abu,iva</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>N-Methylierung</TD></TR><TR><TD>11</TD><TD>C-A-T</TD><TD>C-A-T</TD><TD>L</TD><TD>Ala</TD><TD>Ala</TD><TD>Pro</TD><TD>Ala</TD><TD>Ala</TD><TD>Ala</TD></TR><TR><TD>12</TD><TD>C</TD><TD>C</TD><TD>???</TD></TR><TR><TD> Gramicidin</TD><TD>gramicidin synthetase 1</TD><TD>1</TD><TD>A-T-E</TD><TD>A-T-E</TD><TD>D</TD><TD>Phe</TD><TD>D-Phe</TD><TD>Phe</TD><TD>Phe</TD><TD>Phe</TD><TD>Phe</TD><TD>X</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pubmed/1560782" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/1560782</A></TD><TD>&quot;On the genomic sequence for first synthetase are an additional TE and an additional Com-C domain on separate gene products. Two primary peptides can be fused to one ring.&quot;
</TD></TR><TR><TD>grsB</TD><TD>2</TD><TD>C-A-T</TD><TD>com-C-A-T</TD><TD>L</TD><TD>Pro</TD><TD>Pro</TD><TD>Pro</TD><TD>Pro</TD><TD>Pro</TD><TD>Pro</TD></TR><TR><TD>3</TD><TD>C-A-T</TD><TD>C-A-T</TD><TD>L</TD><TD>Val</TD><TD>Val</TD><TD>Val</TD><TD>Val</TD><TD>Val</TD><TD>Val</TD></TR><TR><TD>4</TD><TD>C-A-T</TD><TD>C-A-T</TD><TD>L</TD><TD>Orn</TD><TD>Orn</TD><TD>Orn</TD><TD>Orn</TD><TD>Orn</TD><TD>Orn</TD></TR><TR><TD>5</TD><TD>C-A-T-TE</TD><TD>C-A-T-TE</TD><TD>L, L</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD></TR><TR><TD> Lichenycin</TD><TD>licA</TD><TD>1</TD><TD>C-A-T</TD><TD>CSt-A-T</TD><TD>Gln</TD><TD>Gln</TD><TD>Gln</TD><TD>Gln</TD><TD>Gln</TD><TD>Gln</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pubmed/9864322" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/9864322</A></TD><TD>Separate TE domain has unknown function and can't be predicted. The first C-Domain and the Thioesterase put a beta-hydroxy fatty acid in the circular peptide
</TD></TR><TR><TD>2</TD><TD>C-A-T</TD><TD>C-A-T</TD><TD>L</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD></TR><TR><TD>3</TD><TD>C-A-T-E</TD><TD>C-A-T-E</TD><TD>L</TD><TD>Leu</TD><TD>D-Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>X</TD></TR><TR><TD>licB</TD><TD>4</TD><TD>C-A-T</TD><TD>com-C-A-T</TD><TD>D</TD><TD>Val</TD><TD>Val</TD><TD>Val</TD><TD>Val</TD><TD>Val</TD><TD>Val</TD></TR><TR><TD>5</TD><TD>C-A-T</TD><TD>C-A-T</TD><TD>L</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD><TD>Asp</TD></TR><TR><TD>6</TD><TD>C-A-T-E</TD><TD>C-A-T-E</TD><TD>L</TD><TD>Leu</TD><TD>D-Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>Leu</TD><TD>X</TD></TR><TR><TD>licC</TD><TD>7</TD><TD>C-A-T-TE</TD><TD>com-C-A-T-TE</TD><TD>D, L</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile</TD><TD>Ile/Val/Leu</TD></TR><TR><TD>licTE (deleted)</TD><TD>TE</TD><TD>TE</TD></TR><TR><TD> Syringomycin</TD><TD>syrE</TD><TD>1</TD><TD>C-A-T</TD><TD>Cst-A-T</TD><TD>Ser</TD><TD>Ser</TD><TD>Ser</TD><TD>Ser</TD><TD>Ser</TD><TD>Ser</TD><TD><A href="http://www.ncbi.nlm.nih.gov/pubmed/9830033" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/9830033</A></TD><TD>syrB1 is 9th module somehow attaching to C-TE after 8th module.
</TD></TR><TR><TD>2</TD><TD>C-A-T</TD><TD>Cdual-A-T</TD><TD>L</TD><TD>Ser</TD><TD>D-Ser</TD><TD>Ser</TD><TD>Ser</TD><TD>Ser</TD><TD>Ser</TD><TD>X</TD></TR><TR><TD>3</TD><TD>C-A-T</TD><TD>Cdual-A-T</TD><TD>D</TD><TD>Dab</TD><TD>D-Dab</TD><TD>gly,ala,val,leu,ile,abu,iva</TD><TD>Dab</TD><TD>Dab</TD><TD>Dab</TD><TD>X</TD></TR><TR><TD>4</TD><TD>C-A-T</TD><TD>Cdual-A-T</TD><TD>D</TD><TD>Dab</TD><TD>Dab</TD><TD>gly,ala,val,leu,ile,abu,iva</TD><TD>Dab</TD><TD>Dab</TD><TD>Dab</TD><TD>X</TD></TR><TR><TD>5</TD><TD>C-A-T</TD><TD>C-A-T</TD><TD>L</TD><TD>Arg</TD><TD>Arg</TD><TD>Arg</TD><TD>Arg</TD><TD>Arg</TD><TD>Arg</TD></TR><TR><TD>6</TD><TD>C-A-T</TD><TD>C-A-T</TD><TD>L</TD><TD>Phe</TD><TD>Phe</TD><TD>Phe</TD><TD>Phe</TD><TD>Phe</TD><TD>Phe</TD></TR><TR><TD>7 (not curated)</TD><TD>C-A-T</TD><TD>C-A-T</TD><TD>L</TD><TD>DhBu-3OH</TD><TD>Thr</TD><TD>Thr</TD><TD>Thr</TD><TD>maybe Thr is recognised and Cdual is dehydrating instead of racemising</TD></TR><TR><TD>8</TD><TD>C-A-T</TD><TD>Cdual-A-T</TD><TD>X</TD><TD>Asp</TD><TD>Asp-3OH</TD><TD>Asp</TD><TD>Asp</TD><TD>Ala</TD><TD>Asp</TD><TD>X</TD></TR><TR><TD>C-T-TE</TD><TD>C-T-TE</TD><TD>L, L</TD><TD>only TE kept</TD></TR><TR><TD>syrB1 (del)</TD><TD>9</TD><TD>A-T</TD><TD>A-T</TD><TD>Thr</TD><TD>Thr-4Cl</TD><TD>Thr</TD></TR></TBODY></TABLE>
After the Regional Jamboree in Lyon, we fixed several bugs and most important we introduced a library function. This allows for construction of many similar NRPs at a time, which could for example be useful for medical compund screening.
                                </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12"><DIV class="imageMap" aria-haspopup="true"><CENTER>Thanks to </CENTER></DIV></DIV></DIV><DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Heidelberg/NRPSDesigner">http://2013.igem.org/Team:Heidelberg/NRPSDesigner</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/NRPSDesigner" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/NRPSDesigner" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/NRPSDesigner&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/NRPSDesigner&amp;oldid=361708" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>