<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Meetings"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Meetings">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Meetings&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Meetings&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Meetings&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Meetings" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Meetings</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="row"><DIV id="nav" class="navigation"><DIV class="item team" data-offset-l="-130"><H2>Team</H2><UL><LI><A href="/Team:Heidelberg/Team">Team Members</A></LI><LI><A href="/Team:Heidelberg/AttributionsII">Attributions</A></LI><LI><A href="/Team:Heidelberg/Team/Sponsors">Sponsors</A></LI><LI><A href="/Team:Heidelberg/Team/Collaboration">Collaborations</A></LI><LI><A href="/Team:Heidelberg/Team/Gallery">Gallery</A></LI></UL></DIV><DIV class="item project" data-offset-l="-144"><H2>Project</H2><UL><LI><A href="/Team:Heidelberg/NRPS">Background</A></LI><LI><A href="/Team:Heidelberg/Project">Overview</A></LI><LI><A href="/Team:Heidelberg/Project/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Project/Indigoidine-Tag">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Project/Tag-Optimization">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Project/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Project/Achievements">Achievements</A></LI></UL></DIV><DIV class="item notebook" data-offset-l="-139"><H2>Notebook</H2><UL><LI><A href="/Team:Heidelberg/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Tyrocidine">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Indigoidine">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/NRPSDesigner">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/Notebook/Methods">Methods</A></LI><LI><A href="/Team:Heidelberg/Materials">Materials</A></LI></UL></DIV><DIV class="item parts" data-offset-l="-134"><H2>Parts</H2><UL><LI><A href="/Team:Heidelberg/Parts">Submitted Parts</A></LI><LI><A href="/Team:Heidelberg/Favorite_Parts">Favorite Parts</A></LI><LI><A href="/Team:Heidelberg/RFCs">Our RFCs</A></LI></UL></DIV><DIV class="item software" data-offset-l="-134"><H2>Software</H2><UL><LI><A href="/Team:Heidelberg/Project_Software">Project</A></LI><LI><A href="http://igem2013.bioquant.uni-heidelberg.de/NRPSDesigner/">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/iGEM42">iGEM42</A></LI><LI><A href="/Team:Heidelberg/OpenSource">Open Source</A></LI><LI><A href="/Team:Heidelberg/Project/AchievementsSoftware">Achievements</A></LI></UL></DIV><DIV class="item human" data-offset-l="-118"><H2>Human Practice</H2><UL><LI><A href="/Team:Heidelberg/HumanPractice">Overview</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BroadPublic">Broad Public</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Survey">Survey</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/YoungGeneration">Young Generation</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Artists">Arts and Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BeyondScience">Beyond Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Experts">Experts</A></LI></UL></DIV><DIV class="item safety" data-offset-l="-115"><H2>Safety</H2><UL><LI><A href="/Team:Heidelberg/Safety">Safety</A></LI></UL></DIV><DIV class="item model" data-offset-l="-144"><H2>Modeling</H2><UL><LI><A href="/Team:Heidelberg/Modelling/Gold_Recovery">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Modelling/Ind_Production">ind-Production</A></LI></UL></DIV><DIV class="item"><SPAN style="margin-left:6%">HOME</SPAN></DIV></DIV></DIV><DIV class="container"><DIV style="margin-top:40px"><H1><SPAN class="text-muted" style="font-family:Arial, sans-serif; font-size:100%; color:#333333;">Meetings.</SPAN></H1><P>We met regularly together with all our advisors and instructors to discuss the project strategy, to distribute tasks and to initialize additional social events.</P></DIV><DIV class="row"><DIV class="col-sm-12"><UL class="pagination" style="margin-bottom:2%; margin-left:4%;"><LI><A href="#" id="backwards">«</A></LI><LI class="month_tab" id="april"><A href="#" style="width:90px; text-align:center">April</A></LI><LI class="month_tab" id="may"><A href="#" style="width:90px; text-align:center">May</A></LI><LI class="month_tab" id="june"><A href="#" style="width:90px; text-align:center">June</A></LI><LI class="month_tab" id="july"><A href="#" style="width:90px; text-align:center">July</A></LI><LI class="month_tab" id="august"><A href="#" style="width:90px; text-align:center">August</A></LI><LI class="month_tab" id="september"><A href="#" style="width:90px; text-align:center">September</A></LI><LI><A href="#" id="forwards">»</A></LI></UL><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#aapr" data-toggle="tab">2013-04-08</A></LI><LI><A href="#bapr" data-toggle="tab">2013-04-15</A></LI><LI><A href="#capr" data-toggle="tab">2013-04-17</A></LI><LI><A href="#dapr" data-toggle="tab">2013-04-26</A></LI><LI><A href="#eapr" data-toggle="tab">2013-04-29</A></LI></UL></DIV><DIV class=" weekly"><DIV class="scrollContent nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="aapr"><H2><SPAN class="mw-headline" id="NRPS_.E2.80.93_Delfibactin_project"> NRPS – Delfibactin project </SPAN></H2><P>by Ilia
</P><P>1. delfibactin:
</P><UL><LI> export from biocompartiments: unspecific outer membrane transporter (so delfibactin gets into medium)
</LI><LI> synthesis: pathway known, e.coli strain for production (maybe not clone DelC)
</LI></UL><P>2. NRPS: introduction
</P><UL><LI> modular enzymes: 3 modular domains (together 110 kDa)
</LI><LI> steps: adenylation, acylation, condensation
</LI></UL><P>3. software: input 
</P><UL><LI> match amino acid to known A-domain substrates
</LI><LI> recognize easy modification (in amino acids) for which substrate you need it 
</LI><LI> method: substructure based algorithm for searching and predicting substrate specifity
</LI><LI> it works in a R-package
</LI><LI> 10 amino acids are important for the substrate specifity
</LI></UL><P>4. biobrick library
</P><UL><LI> there is no standard for the Biobricks of NRPS
</LI></UL><H2><SPAN class="mw-headline" id="GHB"> GHB </SPAN></H2><P>by Flo and Nils
</P><P>1. Jugend forscht: done a test (we do not understand how it works)
</P><UL><LI> not reached the supervisor yet
</LI><LI> she didn't win anything
</LI><LI> detect it via Spectrometry
</LI></UL><P>2. 3 ideas for integrating the receptor
</P><UL><LI> artifiial membranes are not an option (you can't predict how the protein is located)
</LI><LI> in vitro expression of GPCRs (takes too long)
</LI><LI>  Luciferase activity under directt ligand-dependent control of muscarinic acetylcholine receptor
</LI></UL><P>=&gt; measurments possible (very sensitive: represses glowing a little bit, a bit more... but never completely)
(integrate it is fast, but the mutagenesis takes a long time)
</P><H2><SPAN class="mw-headline" id="Artificial_yeast-cells_for_sensing_Insulin-level_and_producing.2Fdistribute_the_Insulin"> Artificial yeast-cells for sensing Insulin-level and producing/distribute the Insulin </SPAN></H2><P>by Flo
</P><UL><LI> presentation of the idea and past igem projects regarding this theme
</LI><LI> semipermeable biocapsules (with cells inside)
</LI><LI> the cells (yeast , bacillus subtilis, e.coli... ) for producing insulin if it is necessary
</LI><LI> there exists a famous lab for synthetic biology which works on insulin production
</LI></UL><H2><SPAN class="mw-headline" id="Software_Project"> Software Project </SPAN></H2><P>by Julia, Konrad and Hanna
</P><UL><LI> parameters for statistical analysis and how they correlate (like teammembers, advisors, organsims, score regarding a scoring funtion)
</LI><LI> graphics: basically histograms
</LI></UL><H2><SPAN class="mw-headline" id="work_for_the_next_weeks"> work for the next weeks </SPAN></H2><P>1. software project
</P><UL><LI> look on the focus of the team!
</LI><LI> better definition of the parameters
</LI><LI> estimate the cost of an IGEM project (there exists something for the nature paper... just looking at the methods etc...)
</LI><LI> already made something with these parameters
</LI><LI> clustering for example, not only directed
</LI><LI> software project should be finished before the lab phase: end of June/ July
</LI></UL><P>2. Gold Project
</P><UL><LI> working on this project: Nils, Ilia, Julia, Ralf, Hanna
</LI><LI> find stuff we have to order and send Dominik an email =&gt; done!
</LI><LI> talk to companies (interested, is it done already and how, are there established assays)
</LI><LI> find succesful examples
</LI></UL><P>3. general information
</P><UL><LI> we have to register and then apply for the team. LogIn
</LI><LI> send an email for getting the contact details (parameters which here already used)
</LI></UL><H2><SPAN class="mw-headline" id="Agenda_for_next_meeting_.2829.04.2013_at_5_pm.29"> Agenda for next meeting (29.04.2013 at 5 pm) </SPAN></H2><OL><LI> results of pilot project (delfibactin in the lab)
</LI><LI> present a project plan (goals, time etc.)
</LI><LI> update on the software project. How to proceed etc.
                                </LI></OL><H2><SPAN class="mw-headline" id="Structure_and_Project_Proposal:"> Structure and Project Proposal: </SPAN></H2><UL><LI> will be written in the Wiki
</LI><LI> catchy pictures have to be placed into the part Motivation / State of the Art
</LI><LI> Abstract has to be added
</LI></UL><DL><DD> - hanger gold or NRPS? --&gt; probably start with NRPS and outline gold-project as application
</DD><DD> - depends on how many other teams choose gold-project
</DD><DD> - gold into the project-title?
</DD></DL><UL><LI> outlook/final statement has to be added
</LI><LI> postpone costs, work on timeschedule
</LI><LI> methods have to be described more precisely --&gt; look for experimental protocols
</LI><LI> what has to be formulated in the PP was written down in note form 
</LI><LI><B>deadline for motivation part:</B> Wednesday, April 17th evening
</LI><LI><B>general deadline:</B>  Monday, April 22nd evening
</LI><LI><B>deadline schedule, material + costs, final part + abstract:</B> Wednesday, April 24th evening
</LI></UL><H2><SPAN class="mw-headline" id="Cloningmethods"> Cloningmethods </SPAN></H2><UL><LI> ask advisors for additional introduction to cloningmethods --&gt; Nikos (Wednesday)
                                </LI></UL><H2><SPAN class="mw-headline" id="Laborplan_Pre-project"> Laborplan Pre-project </SPAN></H2><UL><LI> prepare ACM (1,5L + 0,5L (mit Agar)) Phytogel(NOT with Agar)
</LI><LI> reactivation solution (without agar)
</LI><LI> 0,5% agarose solution with ACM (maybe water) with 10mM AuCl3
</LI><LI> purification
</LI><LI> HP 20 resin-beads: the Delftibactin attaches to the resin 
</LI><LI> Methanol dissolves the Delftibactin, afterwards dry (rotary vacuum) 
</LI><LI> resuspend it (in 2ml)
</LI></UL><H2><SPAN class="mw-headline" id="Motivation_for_Proposal_.28discussed_with_Tim_.26_Tinka.29"> Motivation for Proposal (discussed with Tim &amp; Tinka) </SPAN></H2><UL><LI> central idea: NRPS (gold is the application)
</LI><LI> Weighing not clear
</LI><LI> start with gold
</LI><LI> Structure: Application = Framework 
</LI><LI> Main part: relevance of NRPS 
</LI><LI> generally more elaborate (not too much details)
</LI><LI> more numbers (tons of gld recycled) 
</LI><LI> more structure
</LI><LI> why this pathway for gold precipitation and not another one
</LI><LI> don't use might or may
</LI><LI> softwareproject: how important is it for the gold?? why is it important??? judgement
</LI><LI> Trends of IGEM: no NRPS so far, thus new project for IGEM -&gt; helps students and scientist in finding trends, etc...
</LI></UL><H1><SPAN class="mw-headline" id="3._Podiums_discussion_27.6.13"> 3. Podiums discussion 27.6.13 </SPAN></H1><UL><LI> 18.00 talk about synthetic biology
</LI><LI> 18.30 discussion 
</LI><LI> church, politics, scientists, NGOs(?) - 5 people 
</LI><LI> Prof. Tanner 
</LI><LI> first of all: make a concept with our goal
</LI><LI> catering afterwards (sponsoring)
</LI></UL><H2><SPAN class="mw-headline" id="Cloning"> Cloning </SPAN></H2><UL><LI> BACs
</LI><LI> amplify genes via PCR (Problem, same Codons for E.coli and D.acidovorans)
</LI><LI> maybe not optimal codons
</LI><LI> Synthesis is not possible
</LI><LI> Biobrickstandard: separation in 1 Kb sequenzes
</LI><LI> first plasmids with one protein each
</LI><LI> then plasmids with multiple proteins
</LI><LI> and another try with whole pathway (BAC)
</LI><LI> now we need to find experts and ask them for help
                                </LI></UL><H2><SPAN class="mw-headline" id="Cloning_Techniques.2FStrategies_.28Dominik.29"> Cloning Techniques/Strategies (Dominik) </SPAN></H2><UL><LI> BBa (BioBrick Standard)
</LI><LI> Golden Gate
</LI><LI> Gibson Assambly
</LI></UL><H2><SPAN class="mw-headline" id="Discussion_on_Experimental_Procedures"> Discussion on Experimental Procedures </SPAN></H2><UL><LI> Transformations
</LI><LI> Cosmids
</LI><LI> BACs
</LI></UL><H2><SPAN class="mw-headline" id="NRPS_Library_-_with_Discussion"> NRPS Library - with Discussion </SPAN></H2><UL><LI> main ideas
</LI><LI> swaping of domains done already
</LI><LI> Q: Biology behind the System? (short presentation by Ilia on monday?)
</LI></UL><H2><SPAN class="mw-headline" id="Agenda_for_next_meeting_.2829.04.2013.29"> Agenda for next meeting (29.04.2013) </SPAN></H2><UL><LI> presentation of NRPS (library) (Ilia)
</LI><LI> presentation about methodology (BAC, PAC, Cosmide, Gibson Assambly (Fanny, Flo, Hanna, Joshua, Nicos)
</LI><LI> schedule (Hanna, Julia) ?
</LI></UL><H2><SPAN class="mw-headline" id="Ressource_Collection"> Ressource Collection </SPAN></H2><UL><LI> Vector Design
</LI><LI><A href="http://de-de.invitrogen.com/site/de/de/home/Products-and-Services/Applications/Cloning/vector-nti-software.html" class="external text" rel="nofollow">Vector NTI</A></LI><LI><A href="http://www.geneious.com/" class="external text" rel="nofollow">Geneious</A></LI></UL><UL><LI> Gibson Assembly
</LI><LI><A href="https://www.neb.com/applications/cloning-and-mapping/gibson-assembly" class="external text" rel="nofollow">NEB</A> Discripton &amp;<A href="https://www.neb.com/products/e5510-gibson-assembly-cloning-kit" class="external text" rel="nofollow">Kit</A></LI><LI> Reference:
</LI></UL><OL><LI> Gibson, D.G. et al. (2009) Nature Methods, 343–345.
</LI><LI> Gibson, D.G. et al. (2010) Nature Methods, 901–903.
</LI></OL><UL><LI> Reserach on NRPS
</LI><LI><A href="http://www.uni-marburg.de/fb15/ag-marahiel/research" class="external text" rel="nofollow">AG Marahiel, Marburg</A> --&gt; Niels
</LI><LI><A href="http://www.mpimf-heidelberg.mpg.de/gruppen/cytochrome" class="external text" rel="nofollow">AG Cryle, MPI Heidelberg</A>  --&gt; Konrad
</LI></UL><H2><SPAN class="mw-headline" id="further_planning"> further planning </SPAN></H2><UL><LI> Sat. 4. May: primer Design meeting (&quot;Giant Inserts in E. coli&quot;)
                                </LI></UL><H2><SPAN class="mw-headline" id="Gold"> Gold </SPAN></H2><UL><LI> we were able to reproduce the paper --&gt; images with gold
</LI><LI> delftibactin purification didn't work by now --&gt; try again
</LI></UL><H5><SPAN class="mw-headline" id="recapitulation_of_NRPS_domains_and_modularity"> recapitulation of NRPS domains and modularity </SPAN></H5><UL><LI> domains that are essential: C: Condesantion, A: Adenylation, T: Thiolation
</LI></UL><H5><SPAN class="mw-headline" id="what_has_been_done"> what has been done </SPAN></H5><UL><LI> simplification of synthases (smaller linkers) -&gt; no negative effects
</LI><LI> replacement of alanin to serene by changing domain
</LI><LI> module extention
</LI><LI> short de-novo NRPs
</LI></UL><H5><SPAN class="mw-headline" id="NRPS_biobrick_library"> NRPS biobrick library </SPAN></H5><UL><LI> produce a library with different combinable NRPS-subunits
</LI><LI> start with producing staphcillin (has already been done --&gt; realistic to achieve)
</LI></UL><H2><SPAN class="mw-headline" id="Coloured_NRP:_indigoidine_.28Ralf.29"> Coloured NRP: indigoidine (Ralf) </SPAN></H2><UL><LI> we could prove the modularity of our NRPS-library by producing indigoidine
</LI><LI> is a blue dye
</LI><LI> synthesized from 2 gylcin molecules with a certain enzyme
</LI><LI> there are only a few cutting sites in the required genes
</LI></UL><H2><SPAN class="mw-headline" id="Cosmide_.28Hanna.29"> Cosmide (Hanna) </SPAN></H2><UL><LI> insert size 50kbp
</LI><LI> has got cos site --&gt; in vitro packaging
</LI><LI> has got only very few cutting sites (BamH1)
</LI><LI> selection maker: ampicillin
</LI><LI> check of digestion with 0.8% agarose gel
</LI><LI> preparation with phenol/chloroform extraction
</LI></UL><H2><SPAN class="mw-headline" id="Phagemids_.26_PACs_.28Flo.29"> Phagemids &amp; PACs (Flo) </SPAN></H2><H5><SPAN class="mw-headline" id="Phagemids"> Phagemids </SPAN></H5><UL><LI> you can infect cells with phages or by electroporation
</LI><LI> 10-20 kbp
</LI><LI> high yield of DNA
</LI><LI> E. coli have too be competent for F-Phages
</LI><LI> DNA extraction via phages
</LI></UL><H5><SPAN class="mw-headline" id="PACs"> PACs </SPAN></H5><UL><LI> p1 artificial chromosomes
</LI><LI> hybrid system of plasmid and p1 phage replication
</LI><LI> insert size 70-150 kbp
</LI><LI> transformation using phages
</LI><LI> isolation with kit possible (QIAGEN)
</LI><LI> good transformation-rate but complex system
</LI><LI> you have to be very careful working with phages --&gt; easy contamination
</LI></UL><H2><SPAN class="mw-headline" id="BACs_.28Fanny.29"> BACs (Fanny) </SPAN></H2><H5><SPAN class="mw-headline" id="Recombineering"> Recombineering </SPAN></H5><UL><LI> well established system
</LI><LI> 150-350 kbp
</LI><LI> low copy numbers
</LI><LI> &quot;lamda red recombination&quot; using 3 viral genes: 5'-3' exonuclease, overhang binding protein, inhibitor of bacterial exonuclease
</LI><LI> use of cassette with pos and neg selection marker
</LI><LI> electroporation of gene of interest
</LI><LI> cloning: cassette is brought into the BAC by homologous recombination --&gt; also several times possible
</LI><LI> there is a library of E. coli strains which already have the cassette
</LI></UL><H2><SPAN class="mw-headline" id="Gibson_cloning_.28Nikos.29"> Gibson cloning (Nikos) </SPAN></H2><UL><LI> iGEM biobrick standard will be hard for us to realize --&gt; delH
</LI><LI> we will promote modularity with the NRPS library
</LI><LI> for the gold project we will likely use gibson cloning
</LI><LI> we will talk about primer design on saturday (4th May)
</LI><LI> start PCR as soon as possible
                                </LI><LI class="active"><A href="#a12" data-toggle="tab">2013-05-04</A></LI><LI><A href="#b12" data-toggle="tab">2013-05-09</A></LI><LI><A href="#c12" data-toggle="tab">2013-05-13</A></LI><LI><A href="#d12" data-toggle="tab">2013-05-15</A></LI><LI><A href="#e12" data-toggle="tab">2013-05-22</A></LI><LI><A href="#f12" data-toggle="tab">2013-05-27</A></LI></UL></DIV><DIV class=" weekly"><DIV class="scrollContent nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a12"><H2><SPAN class="mw-headline" id="Cloning_2"> Cloning </SPAN></H2><UL><LI><B>DelH (review of the work done so far)</B></LI><LI>primers were designed and ordered
</LI><LI><I>DelH gene</I> first try with plamsid backbone: pSB6A1
</LI><LI>AraC Promoter, RBs, lacZ, low ori pMB1, amp-R
</LI><LI>PacI and KpnI are not in DelH gene, thus used for ligation
</LI></UL><LI><I>DelA to P</I> into single plasmid with about 26kbp: pSB4K5
</LI><LI>lacI Promoter, RBs, mRFP1, low ori pSC101, Kan-R
</LI><LI> Hanna's work so far: <I>Top 10 E.coli</I></LI><LI>pSB4K5 with Insert
</LI><LI>pSB6A1
</LI><LI>AraC
</LI><LI>lacZ
</LI><LI>pSB1C3 with insert
</LI><PRE><I>Are supposed to be joined by cloning to complete backbones by Monday</I></PRE><UL><LI><B>Indigoidine</B></LI><LI>similar to DelH
</LI><LI>pSB1C3
</LI><LI>high ori 0034, lacI promoter, mRFP1, Cm-R
</LI></UL><LI>cloning with +TypeII RE (recognize sequence and then cut at site one nucleotide next to recognition site)
</LI><LI>division into three fragments
</LI><P><B>ask Dominik</B>: 
</P><UL><LI>how exactly does this strategy work?
</LI><LI>when is this an appropriate strategy to use?
</LI><LI>Is &quot;PCR Ligation&quot; also a good Method (seperate PCRs for fragments; primers have &quot;overlapping tail&quot; takend from next fragment; ligation of fragments in one single PCR&quot;)?
</LI></UL><P><B>Important! What about Freiburg? are they willing to help us with Gibson Cloning?</B></P><H2><SPAN class="mw-headline" id="Approach_of_the_Next_Few_Weeks"> Approach of the Next Few Weeks </SPAN></H2><P>What should be bought:
---&gt; Konrad's list
</P><UL><LI> divison into smaller groups dealing with the main topics of our projects:
</LI></UL><OL><LI> Indigoidine (closed): Rald, Konrad, Nikos
</LI><LI> Del H lab protocol/methods (closed) : Hanna, Fanny, Sophie
</LI><LI> Del Rest (A to P) (closed) : Flo, Ilia, Nikos, Nils
</LI><LI> NRPS : Tania, Joschua, Ilia
</LI><LI> Software : Nils, Ilia, Konrad, Joschua, Hanna
</LI></OL><H2><SPAN class="mw-headline" id="Decisions_Made"> Decisions Made </SPAN></H2><UL><LI><I>serial cloner</I> should be our main software used for primer design etc.
</LI><LI> Should use tool for oligo 2ndary structre prediction
</LI><LI> http/mfold.rna.albany.edu
</LI></UL><LI> Should create common primer list; structure as in the following:
</LI><LI> continous number, primer name, abbreviation (initials), sequence, details (if possible with a link to protocol used), creator, date, evaluation (e.g. worked, low yield ... )
</LI><H2><SPAN class="mw-headline" id="Agenda_for_next_meeting_.2813.05.2013_6_pm.29"> Agenda for next meeting (13.05.2013 6 pm) </SPAN></H2><UL><LI>all groups meet up seperately and continue working
</LI><LI> brief presentation on next meeting
</LI></UL><LI> next meeting (with Dominik) 
</LI><LI> progress report of individual groups and lab
</LI><LI> prepping for upcoming big meeting
</LI><LI> Next big meeting will be 15.5.2013
                                </LI></DIV><DIV class="tab-pane" id="b12"><H2><SPAN class="mw-headline" id="Breakfast"> Breakfast </SPAN></H2><UL><LI> socializing =)
</LI></UL><H2><SPAN class="mw-headline" id="Catching_Up_on_Lab_Work_.28Hanna.29"> Catching Up on Lab Work (Hanna) </SPAN></H2><UL><LI> news on the lab work 
</LI><LI> present protocol 
</LI></UL><H2><SPAN class="mw-headline" id="Gibson_Cloning_Primer_Design_.28Nils.29"> Gibson Cloning Primer Design (Nils) </SPAN></H2><P><B>presentation:</B></P><UL><LI> Backbone pSB1C3, lacI, RBS, mRFP1, Cm-R, pMB! ori
</LI><LI>finding data for introduction of RBS-DelE (PacI and KpnI)
</LI><LI>design of four primers
</LI><LI>2 for backbone 
</LI><LI>2 for insert
</LI></UL><LI> go to parts registry, search pSB1C3
</LI><LI>shows sequence and features --&gt; get selected sequence
</LI><LI>look at sequence in <I>Serial Cloner</I></LI><LI>graphic map should show sequence and particular RE sites (bold names are unique sites, italic names are not existing sites)
</LI><LI>look for cutting site that is not present in backbone nor insert/cassette
</LI><LI>find cassette BBa_503495 sequence in parts registry (not standardized, not best example)
</LI><LI>add cassette sequence to backbone sequence
</LI><LI>show in graphic map
</LI><LI>get DelE sequence (has a lot of PstI cutting sites, but EcoRI only a single site --&gt; can be mutated)
</LI><LI>Primer for adding RBS and cuttingssites between backbone and insert 
</LI><LI>If primer gets too large one can make two PCR steps
</LI><LI>methyl specific cutting enzymes can digest backbone to separate the PCR amplified
</LI><LI>Primer nomenclatur: Kürzel_Nummer:Protein_CuttingSite_fw/rev
</LI><LI>Buffer of Restriction enzymes should be compatible
</LI><LI>Forward primer protein: Overhang - cutting site e.g. PacI - RBS - 6bp (can be used to enhance specifity when taken from gDNA) - atg-Start of Protein - additional 22-27 bp for specificity (or smaller for big primers) - last one should be G/C, no repetitive G/C if possible
</LI><LI>Reverse primer protein: Overhang - cutting site e.g. KpnI (reverse complementary in case of non palindromic enzymes) - reverse complementary end of protein
</LI><LI>Primers should be same melting temperature / length
</LI><LI>Watch out for first and second melting temperature and other reactions run in parallel!
</LI><LI>Forward primer backbone: Overhang - cutting site e.g. KpnI - backbone specific sequence starting with RBS
</LI><LI>Reverse primer backbone: Overhang - cutting site e.g. PacI - reverse complementary sequence of backbone until RBS
</LI><LI>Fold DNA on mfold:
</LI><LI>3' end should be accessible
</LI><LI>5' end can be varied (overhang)
</LI><LI>wobble base pairs can be replaced
</LI><LI>change length of primer to reduce folding
</LI><LI>target would be a folding free energy &gt; -2
</LI><LI>Run in silico PCR in serial cloner
</LI><LI>target melting temperature between 60 and 70°C (if possible - not a critical parameter)
</LI><LI>if you make mutagenesis the sequence is not complementary anymore!
</LI><LI>Evaluate PCR chacks for complementary parts and gives product length
</LI><LI>Annotation
</LI><LI>Features of sequence in serial cloner
</LI><LI>in gene bank files
</LI><H2><SPAN class="mw-headline" id="Ribosomal_binding_sites_.28Konrad.29"> Ribosomal binding sites (Konrad) </SPAN></H2><UL><LI>Current Plan:
</LI><LI>add medium RBS for every part
</LI></UL><LI>RBS in Del Cluster
</LI><LI>RBS from Data base in file (Mail Konrad)
</LI><LI>We can see the direction of transcription according to the primers
</LI><H2><SPAN class="mw-headline" id="Use_of_J5_for_Gibson_assembly_.28Nikos.29"> Use of J5 for Gibson assembly (Nikos) </SPAN></H2><UL><LI>Parts in device editor all at once
</LI><LI>Plasmid backbone
</LI><LI>RBS
</LI><LI>Several parts for Insert - every mutation is a single bp part
</LI></UL><LI>Define how tot obtain the part
</LI><LI>most of the PCR amplified
</LI><LI>other option would be digestion
</LI><LI>RBS would be embedded in a primer
</LI><LI>Choose none for very small parts since it will choose lowest cost strategy
</LI><LI>Controls
</LI><LI>Choose parameters
</LI><LI>Choose Assembly meths
</LI><LI>Result is Vector Editor
</LI><LI>Can show restriction sites
</LI><LI>Primer sequences in csv file
</LI><LI>Gives you cloning strategy
</LI><H2><SPAN class="mw-headline" id="Agenda_for_next_meeting_.2813.05.2013_18:00.29"> Agenda for next meeting (13.05.2013 18:00) </SPAN></H2><UL><LI> One point per sub group
</LI><LI>NRPS
</LI><LI>Software
</LI><LI>Del
</LI><LI>Indi
</LI></UL><LI> Define agenda for 15.5.
</LI><H1><SPAN class="mw-headline" id="Group_picture"> Group picture </SPAN></H1><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><A href="/File:Protocol_09-05-2013.jpg" title="File:Protocol 09-05-2013.jpg">File:Protocol 09-05-2013.jpg</A><DIV class="thumbcaption">Breakfast!</DIV></DIV></DIV></DIV></DIV><DIV class="tab-pane" id="c12"><H2><SPAN class="mw-headline" id="NRPS_subgroup_.28Julia.2C_Philipp.29"> NRPS subgroup (Julia, Philipp) </SPAN></H2><UL><LI> Possible cloning standards
</LI><LI> Handing in biobricks only in pSB1C3 (RFC10 standard)
</LI><LI> RFC25 (scar 6bp) standard because RFC10 scar (7 bp) causes frameshift
</LI></UL><LI> Parts to be cloned from Tyc
</LI><LI> T domains either in front of E or in front of C --&gt; cloning 1 per group
</LI><LI> A and C as single domains and as couple in parallel
</LI><LI> E domains and Thioesterase
</LI><LI> Cutting sites
</LI><LI> not in Tyc A/B/C SpeI, EcoRI, XbaI, NgoMIV
</LI><LI> Tyc B NgOMIV in C domain, in T-C linker
</LI><LI> Tyc C PStI in CIII domain, NotI in AIII domain
</LI><H2><SPAN class="mw-headline" id="Indigoidine_subgroup"> Indigoidine subgroup </SPAN></H2><UL><LI> Plasmid optimised for eucariotic cells
</LI><LI> Impossible to clone it as RFC10
</LI><LI> Use RFC21 instead -&gt; compatible with Tyc?
</LI><LI> first as whole part
</LI><LI> Can be transformed wihtin in this week
</LI><LI> primer design within this week
</LI></UL><H2><SPAN class="mw-headline" id="Cloning_standard_discussion"> Cloning standard discussion </SPAN></H2><UL><LI> Gibson!!!
</LI><LI> find good reason, concept for Gibson cloning
</LI></UL><LI> standard cloning techniques unfeasible
</LI><LI> Further plan
</LI><LI> Order all he plasmids and strains for permeable strain (BAP1  strain, red plasmid, FLP plasmid, template plasmid)
</LI><LI> Indigoidine (Ralf) <A href="http://tools.ietf.org/html/rfc21" class="external mw-magiclink-rfc">RFC 21</A> + check for other NRPS (Ilia) pathways
</LI><LI> Modularity of Indigoidine <A href="http://tools.ietf.org/html/rfc21" class="external mw-magiclink-rfc">RFC 21</A></LI><LI> in parallel check on Gibson assembly in Freiburg (Dominik)
</LI><LI> get Backbones from Washington (Fanny)
</LI><H2><SPAN class="mw-headline" id="Software_subgroup_.28Konrad.29"> Software subgroup (Konrad) </SPAN></H2><UL><LI> subgroups for data mining, data management, statistical analysis, front end
</LI><LI> right now mainly building data management + concept for frontend and analysis
</LI><LI> starting with NRPS software only makes sense as soon as NRPS gives any positive results
</LI><LI> next one and a half weeks for being productive and then we decide for or against the project
</LI></UL><H2><SPAN class="mw-headline" id="Delftibactin_subgroup"> Delftibactin subgroup </SPAN></H2><UL><LI> 8 and 10 kb successfully amplified
</LI><LI> digest tomorrow
</LI><LI> Ligating AraC promotor this week
</LI><LI> If cloning in E.coli doesn't work we skip Del or search for different organism (next week earliest)
</LI><LI> definitely no restrictiton based cloning -&gt; depending on Gibson cloning technique
</LI></UL><H2><SPAN class="mw-headline" id="Miscellaneous"> Miscellaneous </SPAN></H2><UL><LI> Send EMail to registry: How to submit parts? , Promote system open for new standards. (Fanny)
</LI><LI> We should find a plan B for Delftibactin!
</LI><LI> in vitro parts expression, ex vivo delftibactin synthesis?
</LI></UL><H2><SPAN class="mw-headline" id="Agenda_for_next_meeting_.2815.05.2013.29"> Agenda for next meeting (15.05.2013) </SPAN></H2><UL><LI> State of the lab (Hanna)
</LI><LI> Further plan (Fanny)
</LI><LI> Gibson approach (Flo, Nikos)
</LI><LI> Enzymatic cloning and dependencies of NRPS and indigoidine (Nikos)
</LI><LI> Software project deadlinev + plan (Julia, Philipp)
</LI><LI> Budgetplanung (Prof. Eils)
                                </LI></UL><H2><SPAN class="mw-headline" id="State_of_the_lab">State of the lab</SPAN></H2><H3><SPAN class="mw-headline" id="backbones">backbones</SPAN></H3><UL><LI> psB4K5 for Del A-P is ready
</LI><LI> psB6A1 for DelH not
</LI><LI> psB1C3 for Indigoidine is ready
</LI></UL><H3><SPAN class="mw-headline" id="DelH">DelH</SPAN></H3><UL><LI> two parts via SalI (8 kb and 10 kb) successfully amplified
</LI><LI> will be integrated into plasmid for electroporation in <I>E. coli</I></LI></UL><H2><SPAN class="mw-headline" id="Further_plan">Further plan</SPAN></H2><H3><SPAN class="mw-headline" id="methylmalonyl-CoA-strain">methylmalonyl-CoA-strain</SPAN></H3><UL><LI> K207-3 is not available
</LI><LI> we get BAP1
</LI><LI> Lambda-red plasmid
</LI><LI> FLP plasmid
</LI><LI> Template plasmid
</LI></UL><H3><SPAN class="mw-headline" id="DelH_2">DelH</SPAN></H3><UL><LI> Expression in <I>E. coli</I> end of may
</LI></UL><H3><SPAN class="mw-headline" id="Indigoidine_and_NRPS_library">Indigoidine and NRPS library</SPAN></H3><UL><LI> problems concerning RFC10
</LI><LI> Gibson Info Freiburg required
</LI><LI> Washington standard: contact Washington team! (Fanny?)
</LI></UL><H3><SPAN class="mw-headline" id="Gibson_assembly">Gibson assembly</SPAN></H3><UL><LI> maybe method of choice
</LI><LI> are there protocols or do we have to delelop standard protocols
</LI><LI> small fragments could be a problem (PCP-domain)
</LI><LI> develop Gibson RFC standard for NRPS library?
</LI></UL><H2><SPAN class="mw-headline" id="Gibson_approach.2C_Indigoidine_and_NRPS_library">Gibson approach, Indigoidine and NRPS library</SPAN></H2><UL><LI> problems with RFC10 restriction sites and gene size
</LI></UL><H3><SPAN class="mw-headline" id="DelA-P_Cloning_strategy">DelA-P Cloning strategy</SPAN></H3><UL><LI> leave out not necessary genes like DelC
</LI><LI> five cluster fragments
</LI><LI> use Gibson cloning
</LI><LI> Dominik: mid of next week we will get further information
</LI><LI> avoid too big PCR-constructs; 10 kb has been very difficult to amplify
</LI></UL><H3><SPAN class="mw-headline" id="Indigoidine">Indigoidine</SPAN></H3><UL><LI> try plasmid, reproduction of paper
</LI><LI> seperate domains (A, T, TE) and fuse again to proove modularity
</LI><LI> maybe exchange domains with domains from other pathways
</LI><LI> Müller Plasmid -&gt; new problem of 11 restriction sites
</LI><LI> How do we want to assemble NRPS modules in our project?
</LI><LI> maybe Berkeley RFC21 standard because it works very good for our gene clusters
</LI><LI> compatible with linkers
</LI><LI> allows fusion protein
</LI><LI> not accepted by iGEM registry
</LI><LI> BglIII cannot be heat-inactivated
</LI><LI> benign scar (GS)
</LI></UL><H3><SPAN class="mw-headline" id="Washington_pGA">Washington pGA</SPAN></H3><UL><LI> Gibson Assembly Toolkit
</LI><LI> based on RFC21 backbone
</LI></UL><H3><SPAN class="mw-headline" id="standards...">standards...</SPAN></H3><UL><LI> mix and match domains not using biobrick standard but another strategy
</LI><LI> it is too much work to get our parts conform to RFC10 and the NRPS-pathways are not realy compatible with RFC10 at all
</LI><LI> after first experiments consider codon optimized synthesis for part submission
</LI><LI> other NRPS not standardised, but propose a new standard
</LI><LI> few characterized parts are of more value than a hundret useless ones
</LI></UL><H3><SPAN class="mw-headline" id="conclusion">conclusion</SPAN></H3><UL><LI> why not omit RFC and start with Gibson at the first point
</LI><LI> get expertise in Gibson cloning (read papers)
</LI><LI> Hanna and Julia (TA) should try Gibson assembly together
</LI></UL><H2><SPAN class="mw-headline" id="Software">Software</SPAN></H2><H3><SPAN class="mw-headline" id="iGEM_pulse">iGEM pulse</SPAN></H3><UL><LI> use scrapy (python) to get data from last years wikis -&gt; MySQL (first subgroup)
</LI><LI> develop parameters for iGEM oulse webpage (second subgroup)
</LI><LI> First Deadline: May 24
</LI><LI> detailed plan on data mining and parameter design
</LI></UL><LI> Second Deadline: End of June
</LI><LI> valuable product
</LI><H3><SPAN class="mw-headline" id="NRPS_software">NRPS software</SPAN></H3><UL><LI> NRPS library software starting after proof of cencept in Indigoidine
</LI><LI> predict required domains for input structure and propose cloning protocol
</LI></UL><H2><SPAN class="mw-headline" id="Budget">Budget</SPAN></H2><UL><LI> possible sponsors
</LI><LI> Klaus Tschira as main sponsor
</LI><LI> talk to other sponsors for small fundings (connections...)
</LI><LI> university funding, helmholtz, biotech, 
</LI></UL><LI> sponsorship benefits?
</LI><LI> logos on Shirt, Website, presentation
</LI><H2><SPAN class="mw-headline" id="Miscellaneous_2"> Miscellaneous </SPAN></H2><UL><LI> reservation of lufthansa flights for Boston!
</LI><LI> group tickets are cheaper
</LI><LI> greater flexibility
</LI></UL><LI> book hotels/ guest houses in Lyon!
</LI><LI> marketing will be done by professionals
</LI><LI> They will need the project description with loads of random pictures (domains, gold, electronic waste, cash, ...)
</LI><LI> will be there at the next meeting
</LI><H2><SPAN class="mw-headline" id="Conclusion_2">Conclusion</SPAN></H2><UL><LI> start Indigoidine subproject with Gibson cloning (faster!)
</LI><LI> take care about hotels, flight (Sophie)
</LI><LI> project proposal, sponsor list (coordinated by Fanny)
</LI><LI> creative input for marketing people
</LI></UL><H2><SPAN class="mw-headline" id="Agenda_for_next_meeting_with_Advisors_.2806.06.2013.29"> Agenda for next meeting with Advisors (06.06.2013) </SPAN></H2><UL><LI> talk to marketing people
</LI><LI> progress in labwork
</LI><LI> DelA-P
</LI><LI> Indigoidine
</LI><LI> DelH
</LI></UL><LI> travel agency
</LI><LI> project proposal, sponsor list
                                </LI></DIV><DIV class="tab-pane" id="e12"><P style="font-size:12pt; text-align:justify;">
                                 No english protocol available.
                                </P></DIV><DIV class="tab-pane" id="f12"><H2><SPAN class="mw-headline" id="Power_Cut">Power Cut</SPAN></H2><P>Power was down for 10 min in Labs --&gt; Lab Materials should still be fine.
</P><H2><SPAN class="mw-headline" id="Labupdate">Labupdate</SPAN></H2><H3><SPAN class="mw-headline" id="DelH_3">DelH</SPAN></H3><UL><LI> DelH fragments are ready for ligation
</LI><LI> LacZ, AraC --&gt; Cut again and purified
</LI><LI> Backbone --&gt; New Backbone is being amplified in bacteria taken from stock
</LI></UL><P>Plan: Purify, Digest and PCR of backbone (could then be ligated with DelH on wednesday)
</P><H3><SPAN class="mw-headline" id="Indigoidine_2">Indigoidine</SPAN></H3><UL><LI> Transformation of SVP and Bps in individual E.coli succeeded 
</LI></UL><P>Plan: Get BL23C3 and cotransform both Plasmids
</P><H3><SPAN class="mw-headline" id="General">General</SPAN></H3><UL><LI> Partsregistry kit shipping: At the moment some problems with customs
</LI><LI> Lab materials will be ordered tomorrow
</LI></UL><P>To do: 
</P><UL><LI> Document glycerol stocks etc.
</LI><LI> Nikos: Ask Silvan and Jan if we can have script for getting PCR-protocols of PCR machine
</LI><LI> Hanna: Doodle for date for arranging everything in the lab
</LI></UL><H2><SPAN class="mw-headline" id="Symposium_BMBF">Symposium BMBF</SPAN></H2><UL><LI> 27th of June
</LI><LI> Human practice?
</LI><LI> Poster has to be made --&gt; don't put too much effort in it
</LI><LI> Nils starts creating a poster
</LI></UL><P>Who wants to go?: Hanna, Philipp, Nils, Fanny, Tania, maybe Konrad?
</P><H2><SPAN class="mw-headline" id="Software_2">Software</SPAN></H2><H3><SPAN class="mw-headline" id="Statistics">Statistics</SPAN></H3><UL><LI> Script which gets data from databank to R is ready (uploaded on Github)
</LI><LI> Next step: designing apps
</LI></UL><H3><SPAN class="mw-headline" id="Webdesign">Webdesign</SPAN></H3><UL><LI> rApache (Ilia) vs. Shiny (Nikos)
</LI></UL><H3><SPAN class="mw-headline" id="Cost_analysis">Cost analysis</SPAN></H3><UL><LI> Complete one is not feasible
</LI><LI> Instead: Only analyse single methods (can be chosen by user, website returns protocols, costs, teams that have used the method)
</LI><LI> Next step: Look for methods, which could be analysed
</LI></UL><H3><SPAN class="mw-headline" id="General_2">General</SPAN></H3><UL><LI> Which is our target group??
</LI><LI> Connect iGEM pulse with our project (for example evaluate our team with it)
</LI></UL><H2><SPAN class="mw-headline" id="Sponsoring">Sponsoring</SPAN></H2><UL><LI> Template mail for sponsoring is nearly ready
</LI><LI> Template project proposal of Eils didn't arrive yet
                                </LI><LI class="active"><A href="#a13" data-toggle="tab">2013-06-03</A></LI><LI><A href="#b13" data-toggle="tab">2013-06-06</A></LI><LI><A href="#d13" data-toggle="tab">2013-06-17</A></LI><LI><A href="#e13" data-toggle="tab">2013-06-19</A></LI><LI><A href="#f13" data-toggle="tab">2013-06-25</A></LI><LI><A href="#g13" data-toggle="tab">2013-06-30</A></LI></UL></DIV><DIV class=" weekly"><DIV class="scrollContent nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a13"><P style="font-size:12pt; text-align:justify;">
                                 No english protocol available.
                                </P></DIV><DIV class="tab-pane" id="b13"><H2><SPAN class="mw-headline" id="Introduction_Marketing">Introduction Marketing</SPAN></H2><UL><LI> Gold versus NRPS-library
</LI><LI> entirely new design but link to the last ones
</LI></UL><H2><SPAN class="mw-headline" id="Update_labwork">Update labwork</SPAN></H2><UL><LI> backbones ready
</LI></UL><H2><SPAN class="mw-headline" id="software_3">software</SPAN></H2><UL><LI> plans for apps and analysis
</LI><LI> cost analysis tool
</LI><LI> front end
</LI></UL><H2><SPAN class="mw-headline" id="indigoidine_3">indigoidine</SPAN></H2><UL><LI> shuffle T-domains as proof of modularity
</LI></UL><H2><SPAN class="mw-headline" id="Sponsoring_2">Sponsoring</SPAN></H2><UL><LI> brochure for sponsors
</LI><LI> we need a group photo
</LI></UL><H2><SPAN class="mw-headline" id="next_meetings">next meetings</SPAN></H2><UL><LI> barbecue: june 10th
</LI><LI> with advisors: june 12th
</LI><LI> with supervisors: june 19th
                                </LI></UL><H2><SPAN class="mw-headline" id="sub-groups"> sub-groups </SPAN></H2><P>how do we organize sub-group-meetings?</P><DL><DD> one group 100 % or two groups (1st group 70% and 2nd group 30%)
</DD></DL><TABLE class="wikitable"><TBODY><TR><TH> groups </TH><TH> how many persons do we need? </TH><TH> main person </TH><TH> who works here? </TH></TR><TR><TD> IGEM42-software </TD><TD> 1 </TD><TD> Julia </TD><TD> Julia, Philipp, Ilia, Anja, Nikos, Konrad, Hanna
</TD></TR><TR><TD> NRPS-library-software </TD><TD> 3 </TD><TD> Ilia </TD><TD> Ilia, Nils, Joshua, Nikos, Konrad, Julia
</TD></TR><TR><TD> Indigoidine/Tyrocidine </TD><TD> 4 </TD><TD> Ralf </TD><TD> Ralf, Ilia, Joshua, Flo, Nikos, Konrad, Sophie
</TD></TR><TR><TD> Delftibactin </TD><TD> 2 </TD><TD> Hanna </TD><TD> Flo, Sophie, Anja, Nils, Philipp, Hanna
</TD></TR><TR><TD> Human Practice </TD><TD> 1 </TD><TD> Philipp </TD><TD> Philipp, Fanny, Hanna
</TD></TR><TR><TD> Wiki </TD><TD> 1 </TD><TD> Ralf &amp; Flo </TD><TD> Ralf, Flo
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="NRPS-library"> NRPS-library </SPAN></H2><OL><LI> how can you synthetisize your wanted protein?
</LI><LI> which modules do you have to shuffle for this protein? 
</LI><LI> evaluate the modules? which modules are better than the others? =&gt; pathway should be with modules of ONE organism. 
</LI></OL><DL><DD> =&gt; prepare for wednesday a presentation of concept (Nils &amp; Ilia)= Topic 1 (name) =
</DD></DL><UL><LI> result1
</LI><LI> result2
</LI></UL><H2><SPAN class="mw-headline" id="Agenda_for_next_meeting_.2817.06.2013.29">Agenda for next meeting (17.06.2013)</SPAN></H2><UL><LI> subgroup presentations
</LI><LI> NRPS-library - present a concept
</LI><LI> hotel and flights to Boston
                                </LI></UL><H2><SPAN class="mw-headline" id="Logo_Presentation_Lange_und_Pflanz">Logo Presentation Lange und Pflanz</SPAN></H2><UL><LI> couple of proposals, which partially stick to the James Bond style and partially don't
</LI><LI> the Team chose logo 2
</LI><LI> there were additional comments to the logos 
</LI><LI> after a second vote logo 3 was chosen
</LI></UL><H2><SPAN class="mw-headline" id="Sponsoring_3">Sponsoring</SPAN></H2><UL><LI> we have to write into the application what we want from the companies!
</LI></UL><H2><SPAN class="mw-headline" id="Slovenian_Company">Slovenian Company</SPAN></H2><UL><LI> Prof. Eils met people of a Slovenian company, which focuses on transferring very big amounts of DNA (up to 200kb) from one kind of bacteria to another one
                                </LI></UL><H2><SPAN class="mw-headline" id="DelH_Cloning"> DelH Cloning </SPAN></H2><UL><LI> DelH colonies grown after 24 h (two big and one small colony)
</LI><LI> No blue colonies
</LI><LI> Anja will replace Fanny and Hannah in the lab until friday
</LI><LI> Suggestion: check on blue colonies only with easy backbone ligation
</LI></UL><H2><SPAN class="mw-headline" id="Indigoidine_4"> Indigoidine </SPAN></H2><UL><LI> Probably received the right plasmids
</LI><LI> results not as good as in paper (not as blue)
</LI><LI> BAP1 + plasmid is stronger
</LI><LI> Gibson cloning of activating gene and NRPS on one plasmid
</LI><LI> Tomorrow miniprep on picked colonies
</LI><LI> Suggestion: Check expression of ribosome binding site with mCherry-lac system
</LI></UL><H2><SPAN class="mw-headline" id="Genomic_integration"> Genomic integration </SPAN></H2><UL><LI> Probably not enough DNA -&gt; PCR with new primers
</LI><LI> Colonies definitely negative
</LI><LI> making fresh competent cells
</LI></UL><H2><SPAN class="mw-headline" id="Software_4"> Software </SPAN></H2><UL><LI> Several deadlines for different components
</LI><LI> Deadline 8.7.
</LI><LI> text mining should work
</LI><LI> currently working on components and putting them together
</LI><LI> next meeting on 1.7. 19h at Julia
</LI></UL><H2><SPAN class="mw-headline" id="Miscellaneous_3"> Miscellaneous </SPAN></H2><UL><LI> NRPS estimation of computational feasibility
</LI><LI> Project proposal - lots of feedback -&gt; will be finished tomorrow evening
</LI><LI> NRPS library should maybe get some more space in proposal -&gt; Philipp until tomorrow
</LI><LI> If we lack time in the lab we have to pe prepared properly including all primers.
</LI><LI> We should have an internal meeting before tuesday to discuss about motivation, time we want to spend.
</LI></UL><H2><SPAN class="mw-headline" id="Agenda_for_next_meeting_.282.7.2013_19h.29"> Agenda for next meeting (2.7.2013 19h) </SPAN></H2><UL><LI> Report Berlin (Philipp)
</LI><LI> Lab update (all lab people)
</LI><LI> iGEM42 (Julia)
</LI><LI> Motivation &amp; time schedule
</LI><LI> Sponsoring update
</LI></UL><H2><SPAN class="mw-headline" id="Agenda_for_next_meeting_.283.7.2013_18h.29"> Agenda for next meeting (3.7.2013 18h) </SPAN></H2><UL><LI> Lab update (all lab people)
                                </LI></UL><H2><SPAN class="mw-headline" id="iGem42_-_input_on_Design:">iGem42 - input on Design: </SPAN></H2><OL><LI>no greek 
</LI><LI>round edges
</LI><LI>one uniform lettering
</LI><LI>design: where am I? 
</LI><LI>dark blue, rest modern
</LI></OL><P>=&gt; 3 different designs until tuesday 
</P><H2><SPAN class="mw-headline" id="lab_update">lab update</SPAN></H2><OL><LI>general
</LI><LI>polymerase dosn't work as expected (Q5 in contrary works)
</LI><LI>Phusion Flash from advisors works also
</LI><LI>CyclerProblems: Gradienten-PCR doesn't work
</LI><LI>indigoidine
</LI><LI>streptomyces strain arrived =&gt; monday culture the streptomyces
</LI><LI>plasmid containing sc-indc is ordered
</LI><LI>primer monday to Dominik and then order
</LI><LI>DelH
</LI><LI>colonies after electroporesis had no integrated DelH
</LI><LI>amplification of DelH fragments again didn't work - only with Phusion Flash (1b &amp; 2)
</LI><LI>Backbone is not amplified (pSB6A1-AraC-lacZ)
</LI><LI>Del-Rest
</LI><LI>Primers are ordered 
</LI><LI>since wednesday: Del-rest amplify from the genome (Flo &amp; Nils)
</LI></OL><H2><SPAN class="mw-headline" id="organization">organization</SPAN></H2><OL><LI>Google-Kalendar for all
</LI><LI>responsibility of ordering stuff: Sophie 
</LI><LI>create Milestones &amp; lab managment on the wiki: Sophie
</LI><LI> tyrocidine has to be started = parts for sending in (check-liste)
</LI><LI> wiki team (Ralf, Flo und Sophie) check the list for gold medal! 
</LI></OL><H2><SPAN class="mw-headline" id="Agenda_for_next_meeting.2802.07.2013.29"> Agenda for next meeting(02.07.2013) </SPAN></H2><OL><LI>Milestones - Groups make the Milestones for their group erstellen diese mit Gruppe
</LI><LI>timeschedule ?who, when?
</LI><LI>iGEM42
</LI><LI>Sponsoring update
                                </LI><LI class="active"><A href="#a14" data-toggle="tab">2013-07-02</A></LI><LI><A href="#b14" data-toggle="tab">2013-07-12</A></LI><LI><A href="#c14" data-toggle="tab">2013-07-18</A></LI><LI><A href="#d14" data-toggle="tab">2013-07-19</A></LI><LI><A href="#e14" data-toggle="tab">2013-07-24</A></LI><LI><A href="#f14" data-toggle="tab">2013-07-29</A></LI><LI><A href="#g14" data-toggle="tab">2013-07-30</A></LI><DIV class=" weekly"><DIV class="scrollContent nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a14"><H2><SPAN class="mw-headline" id="Logo">Logo</SPAN></H2><UL><LI> new logos were presented to us by Kai
</LI><LI> the majority and Prof. Eils of the team supported the 3. logo (which is more complex, represents modularity)
</LI><LI> Kai continues to work on the logo (colors, animation)
</LI></UL><H2><SPAN class="mw-headline" id="iGEM42_.26_Website">iGEM42 &amp; Website</SPAN></H2><UL><LI> Julia presented 3 proposals of possible websites
</LI><LI> we should use one layout for all websites/wikis
</LI><LI> Kai will send us his background from the logo-presentationns
</LI></UL><H2><SPAN class="mw-headline" id="Milestones">Milestones</SPAN></H2><UL><LI> we went through all milestones with Prof. Eils
</LI></UL><H2><SPAN class="mw-headline" id="Thermocycler-problem">Thermocycler-problem</SPAN></H2><UL><LI> talk to the IPMB!
</LI><LI> find a solution for next two weeks --&gt; Tinka's lab (plan in advance and let her know!)
</LI><LI> we will propably buy 1-2 new cyclers
</LI></UL><H2><SPAN class="mw-headline" id="Slovenian_Company_2">Slovenian Company</SPAN></H2><UL><LI> talk to them for Del-transformation
</LI></UL><H2><SPAN class="mw-headline" id="Sponsoring_4">Sponsoring</SPAN></H2><UL><LI> Steimbuch Stiftung (Software founding up to 10.000€) might be interesting
</LI><LI> Prof. Eils: the timeline is pretty long
</LI></UL><LI> Claus Tschirra is most important
</LI><LI> Fanny will send a corrected version of our project proposal to Prof. Eils
</LI><H2><SPAN class="mw-headline" id="Human_Practise">Human Practise</SPAN></H2><UL><LI> Stage discussion: on synthetic biology...
</LI><LI> after European Jamboree
</LI><LI> Prof. Eils: human practise is important! but we shouldn't overdo it
</LI></UL><LI> go into schools
</LI><LI> science &amp; arts --&gt; bioquant is already in some cooperation
                                </LI></DIV><DIV class="tab-pane" id="b14"><H2><SPAN class="mw-headline" id="Cooperation_with_Freiburg.28Hanna.29"> Cooperation with Freiburg(Hanna) </SPAN></H2><UL><LI> arrive on Monday(14.7.) want to show us Gibbson Assembly
</LI><LI> accomodation: Philipp, Julia, Hanna/Ralf
</LI><LI> Would go out for Dinner at Schmidt's, Hanna is going to send an Email
</LI><LI> we should take away all the things in the lab indicating our actual project with Delftibactin 
</LI></UL><H2><SPAN class="mw-headline" id="Lyon_.28Hanna.2C_Philipp.2C_Tania.29"> Lyon (Hanna, Philipp, Tania)</SPAN></H2><UL><LI> hotel: 7x 3 people in one room
</LI><LI> train to Lyon is booked, need to book the ride back on 15.7. (Monday)
</LI></UL><H2><SPAN class="mw-headline" id="Human_Practice"> Human Practice </SPAN></H2><UL><LI> Konrad offered to take care of the soil samples to send to the UK (?)
</LI><LI> want to get some info on Streptomycces from them. 
</LI></UL><LI> Fanny and bts take care of the discussion round
</LI><H2><SPAN class="mw-headline" id="Financing.2C_Karl_Steinbruch_Stiftung.2C_Software_Project"> Financing, Karl Steinbruch Stiftung, Software Project </SPAN></H2><UL><LI> Anja: we need three people's CVs that have worked in Bioinfomatics and have more than 120 ECTS
</LI><LI> they want to know about where we spent the money
</LI><LI> max ammount is app. 830 €/month
</LI><LI> dead line is 31.7.
</LI></UL><H2><SPAN class="mw-headline" id="Lab_Update_2">Lab Update </SPAN></H2><H3><SPAN class="mw-headline" id="Delftibactin">Delftibactin</SPAN></H3><UL><LI> Del H 
</LI><LI> problems with 1B bands
</LI><LI> new amplification worked
</LI><LI> were digested and purified today
</LI><LI> are going to be ligated tomorrow
</LI></UL><P>Dominiks suggestion:
</P><UL><LI> divide ligation after 1 hour into two, one is kept at -20°C the other is let continue over night at 4°C
</LI><LI> pool both together and electroporate this batch 
</LI></UL><LI> gibbson primers arrived today; this will be the new approach, if ligation is unsuccessful
</LI><LI> parallel gibbson does not make sense, to expensive
</LI><P>---&gt; gel is running right now and will influence decision on further proceeding
</P><UL><LI> Del Rest
</LI><LI>7-8 fragments of 32 are still missing
</LI><LI>backbone has been amplified
</LI><LI> Flo designed new primers
</LI><LI> for DelOP
</LI></UL><H3><SPAN class="mw-headline" id="Indigoidine_5">Indigoidine</SPAN></H3><UL><LI>pptase could be amplified, but not indigoidine synthetase 
</LI><LI>brainstorming
</LI><LI>maybe the gene is not in this sub-species?
</LI></UL><LI>plamsid from UTAH arrived
</LI><LI> want to write to other labs if they can send us the plasmid
</LI><LI>there is a different Streptomyces (Stamm) that has this enzyme but need to finde the correct name
</LI><LI>burocracy discussion.. MTA, Hauspost, Eils signature
</LI><LI>primer strategy with BPSA
</LI><H3><SPAN class="mw-headline" id="Tyrocidine">Tyrocidine</SPAN></H3><UL><LI>Stamm come from Marahije (?)
</LI><LI>Glycerol stocks,colony PCRs
</LI><LI>new strategy has be developed
</LI><LI>ordering of primers by end of the week
</LI></UL><H3><SPAN class="mw-headline" id="Methyl-malonyl_CoA">Methyl-malonyl CoA</SPAN></H3><UL><LI> want genomic integration of substrate synthesis complex for DelF
</LI><LI> got the insert from the USA
</LI><LI>just need to amplify the insert with appropritate primers with some extra DNA
</LI><LI> first tries didn't work
</LI><LI>has several primers for accurate testing/screening
</LI><LI>if integration worked only one resistance gene copy 
</LI><LI> gets fragment PLUS other bands of something else...
</LI><LI> did electroporation and purification this week
</LI><LI> some altered conditions
</LI></UL><LI> some good colonies
</LI><LI> screening
</LI><H2><SPAN class="mw-headline" id="NRPS">NRPS</SPAN></H2><UL><LI>primary goals of the NRPS etc
</LI><LI>NRPS are modulary
</LI><LI>three domains, CA-T-optional
</LI><LI>flexible linkers between domains
</LI><LI>can be put into different orders
</LI><LI>wetlab:
</LI><LI>goal: library of physical DNA of single modules to be able to put into different orders
</LI><LI>start with tyrocidine pathway
</LI></UL><UL><LI>software:
</LI><LI>tell what you want
</LI><LI>has info on different pathways and domains
</LI><LI>tells me how to ligate everything together etc
</LI><LI>specifity is dependent on CA domain
</LI></UL><UL><LI>look at what is most probable to work later
</LI></UL><P>--&gt; calculate evolutionary distance according to the taxon ids in the database
</P><UL><LI>4 nodes:
</LI><LI>b.brevis
</LI><LI>e.coli.
</LI><LI>p. aeruginosa
</LI><LI>b.subtilis
</LI></UL><UL><LI>Discussion about Linkers:
</LI><LI>should they be standardized?
</LI><LI>how should they look like?
</LI><LI>domains are highly conserved among species and linkers are NOT conserved at all
                                </LI></UL><H2><SPAN class="mw-headline" id="Wiki-Design"> Wiki-Design </SPAN></H2><DL><DD><UL><LI> responsible persons?? Anja, Tania
</LI><LI> A brainstorming session would be good
</LI><LI>we are sure that wwe want it simple but the motto philosopher stone 
</LI><LI> explain logo on the wiki
</LI></UL></DD></DL><DL><DD><B>ideas</B><UL><LI> every project has ONE object from the logo
</LI><LI> Höhlenmalerei was an idea
</LI><LI> maybe two possible ways to get to the wiki
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="software_NRPS"> software NRPS </SPAN></H2><UL><LI> visualize an overview of project (structure)
</LI><LI> what is already done? 
</LI><LI> what is to do? 
</LI><LI> linker discussion 
</LI><LI> there are more than 500 monomer
</LI><LI> one module consists in different domains 
</LI></UL><H2><SPAN class="mw-headline" id="Agenda_for_the_next_meeting_.2819.07.2013.29"> Agenda for the next meeting (19.07.2013) </SPAN></H2><OL><LI> Organisation (Hanna)
</LI><LI> cooperation with Freiburg (Sophie)
</LI><LI> sponsoring
</LI><LI> software-NRPS (Konrad &amp; Software)
</LI><LI> Lab-work 
</LI><LI> human practices (slight)
</LI></OL><H2><SPAN class="mw-headline" id="to_do"> to do </SPAN></H2><DL><DD><UL><LI> wiki design
</LI><LI> human practices
</LI><LI> organisation of team: 
</LI><LI> list of protocol writers, 
</LI><LI> for every detail there has to be defined a responsible person 
</LI></UL><LI> doodle for concept day
                                </LI></DD></DL><H2><SPAN class="mw-headline" id="Organisation">Organisation</SPAN></H2><P>Hanna is new in the team as an advisor and will support us during the project. We have to book hotel and train tickets for her.
</P><H2><SPAN class="mw-headline" id="Freiburg-Cooperation">Freiburg-Cooperation</SPAN></H2><P>Three Freiburg team mmbers have been to Heidelberg monday/tuesday for cooperation.
We did a Gibson Assembly with their constructs and transformed e. coli via electroporation and chemical transformation.
After having dinnner they left the next day.
</P><H2><SPAN class="mw-headline" id="Sponsoring_5">Sponsoring</SPAN></H2><UL><LI> Klaus-Tschira-Stiftung supports us
</LI><LI> eppendorf supports us with consumables and hardware to lend
</LI></UL><H2><SPAN class="mw-headline" id="NRPS-Software">NRPS-Software</SPAN></H2><P>plan for web-interface and software/database-structure
</P><UL><LI>webinterface is ready to feed the database with details to single NRPSs; we will provide the software with about 10-20 NRPS pathways.
</LI><LI>getting the single domains is time consuming. automatization?
</LI><LI>-&gt; tool to connect Pfam, clustalO and NCBI to automatically get domain borders for the databas!
</LI></UL><H2><SPAN class="mw-headline" id="Lab">Lab</SPAN></H2><UL><LI>delftibacin
</LI><LI>DelH is amplified; cloning using gibson approach after reassemlby of backbone
</LI><LI>Del-rest: single parts are missing
</LI></UL><LI>indigoidine
</LI><LI>first Streptomyces strain didn't carry our gene of interest; new strain arrived today
</LI><LI>construct with exchangeable PPTases (gibson or restriction)
</LI><LI>tyrocidine
</LI><LI>module shuffling to test concept of modularity
</LI><LI>bacterial strain is there, primer will be there next week
</LI><LI>almost no restriction sites for RFC25
</LI><H2><SPAN class="mw-headline" id="milestones_2">milestones</SPAN></H2><UL><LI>DelH is few days in delay, backbone causes trouble
</LI><LI>DelRest is somewhat in plan
</LI><LI>Indigoidine is two weeks in delay but no other project part is influenced since we decided to start the tyrocidine part in parallel
</LI><LI>Tyrocidine; few days in delay because of primer shipment
</LI><LI>iGEM42; part time work is sufficient as expected
</LI><LI>NRPS-library; milestones need to be set; until now in time
</LI><LI>genomic integration; back-up strategy to clone the methylmalonyl-CoA-pathway on a plasmid to validate our delftibactin-e.coli-strain
</LI></UL><H2><SPAN class="mw-headline" id="human_practise_2">human practise</SPAN></H2><UL><LI>panel discussion
</LI><LI>life science lab, schools, essay on gold recovery
</LI><LI><I>discuss with NGOs about gold recovery, ecology and economy</I></LI><LI>regional small jewelries, maybe for sponsoring
</LI><LI>interaction with lab-art project in summer 
</LI><LI>publications in systembiologie.de, RNZ, Uni-Zeitung, ...
</LI><LI>don't put too much time in human practise
</LI></UL><H2><SPAN class="mw-headline" id="next_meeting">next meeting</SPAN></H2><P>with supervisors: tuesday July 30th; 5.30 pm
                                </P></DIV><DIV class="tab-pane" id="e14"><H2><SPAN class="mw-headline" id="General_organisation_stuff"> General organisation stuff </SPAN></H2><UL><LI> Fridge should be defrosted -&gt; does not close to well right now
</LI><LI> Keep an eye on backup bottles of agarose, buffers, etc. and discard gel remains....
</LI><LI> Handle ethidiumbromide with care and do use new gloves for touching other stuff as buffers for instance; close gel chambers; better communication and more chambers needed
</LI><LI> Sign in to the cycler calendars
</LI><LI> Mini prep: more efficient with 2 min lysis
</LI><LI> Write on autolcavation stuff: who, content, date
</LI><LI> Store autoclaved PCR tubes in the cupboard over the sink
</LI><LI> General duties lab service: tips, autoclavation waste, generel order, plates of all resistances stored, 	consumables -&gt; Hanna will prepare a short to do list
</LI></UL><LI> Keep door to preparation room shut (noise)
</LI><LI> Friday at 10 am weekly team meeting for general issues (advisor meeting on tuesday and thursday 8.30 am)
</LI><LI> Further team building events (e.g. barbecues) should be planned
</LI><LI> Karl-Steinbruch scholarship won't be pursued further; Additional discussion with Roland Eils at the next meeting
</LI><LI> Dispute disagreements with other persons directly 
</LI><LI> Google calendars: keep lab times updated!
</LI><LI> Plasmid cards: we need them of each subproject; Nomenclature: as primers only with 'p' prefix; ask others if you do not know how to prepare them
</LI><LI> Do not forget the wiki documentation
</LI><LI> Abstract, tracks etc. have to be submitted until beginning of august -&gt; has to be communicated with Roland, too. Project titles, abstracts etc. can be changed until later deadline. =&gt; Hanna, Fanny and Tania will prepare some drafts.
</LI><LI> Biosafety fonds -&gt;  Philipp
</LI><H2><SPAN class="mw-headline" id="Design"> Design </SPAN></H2><UL><LI> Invite Lang &amp; Pflanz again? Ask Roland.
</LI><LI> T-shirts -&gt; polo, embroidery, color: <B>dark blue</B>, black or <B>light green</B>, light blue -&gt; proposals will be prepared by Tania for next meeting
</LI><LI> wiki design:
</LI><LI> different languages: german (all), english(all), italian (Barbara), portugese(Tania), russian(Ilia), 	france (Philipp), spanish (Hanna), greek (Nikos), ukrain (Fanny), chinese (Philipp), japanese(Ralf), 	vietnamese(Fanny), hebrew (Tania), klingon(Ilia), korean (Jin)
</LI><LI> chart of colour range?
</LI><LI> Tania and Anja present their Wiki layout proposals -&gt; Web-implementation by Joshua and Nils when layout graphics design terminated
</LI></UL><H2><SPAN class="mw-headline" id="Human_practices"> Human practices </SPAN></H2><P>- Art, panel discussion (synthetic foods), schools (are interested, Tania will talk to the teachers) (-&gt; as backup)
                                </P></DIV><DIV class="tab-pane" id="f14"><H2><SPAN class="mw-headline" id="Judging_criteria"> Judging criteria </SPAN></H2><UL><LI> no dates online yet
</LI><LI> we have to select three tracks -&gt; which ones?
</LI><LI> our favorites (until now):
</LI></UL><P>- Manufacturing
- foundational Advance
- environment
</P><UL><LI> project description can be adjusted until wiki freeze (but is no judging criteria): make it more lurid (abstract as well) and integrate synthetic biology better
</LI></UL><H2><SPAN class="mw-headline" id="Freiburg_constructs"> Freiburg constructs </SPAN></H2><UL><LI> characterize constructs? =&gt;<I>Help another Team</I> criteria
</LI><LI> but these are eucaryotic constructs
</LI><LI> check on different cell lines, media and digests
</LI><LI> critical because incubator could be contaminated by us -&gt; disastrous for Di Ventura group
</LI><LI> we have no cell culture -&gt; send cell lines to Freiburg instead
</LI></UL><UL><LI> we have to fulfill one of three gold medal requirements (Parts debugging, characterize other parts, essay on safety/environment/effects of proeject)
</LI><LI> characterize SFP? Has not been submitted -&gt; cannot do it -&gt; re-submit pptases  
</LI><LI> essay backup solution
</LI></UL><H2><SPAN class="mw-headline" id="Plan_for_Tomorrow_.28Meeting_with_Roland_Eils.29"> Plan for Tomorrow (Meeting with Roland Eils)</SPAN></H2><UL><LI> Milestones
</LI><LI> Short lab update
</LI><LI> Indigoidine [as a new reporter system; try all suggested T-domains; show benefits of software for Indigoidine project]
</LI><LI> Software [set up criteria, which describe similarity of modules, e.g. by literature research, Nikos will ask a member of König group, who worked with t-domains, for help]
</LI><LI> Tyrocidine [detection: Wombacher group; in-vitro/in-vivo in parallel; maybe his-tag?; HPLC before? -&gt; enrichment of fragments; CPEC as alternative for Gibson (worked for Ralf); <I>e. coli</I> lysate (e.g. in-vitro translation); do some literature research on how other groups detected NRPs; non-proteinogenic amino acids for labeling; concentration gradients native &lt;-&gt; transformed; contact Mr. Lehmann]
</LI><LI> Del H [wait for results (tomorrow)]
</LI><LI> Del Rest [waiting for digestion results; Have all fragments, but F-G is too yet]
</LI><LI> Software [meeting tomorrow before Eils meeting? Present our scheme of the last meeting + slides of subteams]
</LI><LI> send all slides to Philipp until 3.30 p.m. tomorrow
                                </LI></UL><H2><SPAN class="mw-headline" id="Sponsoring_6"> Sponsoring </SPAN></H2><UL><LI> Böhringer Ingelheim sponsors us with 5.000 €
</LI><LI> a Gold Smith from Heidelberg donates us 50 €
</LI></UL><H2><SPAN class="mw-headline" id="Wiki-Design_.26_T-shirts"> Wiki-Design &amp; T-shirts </SPAN></H2><UL><LI> we decided to stich our logo on the shirts
</LI><LI> color: navy
</LI><LI> golden border around the logo with blue background
</LI></UL><H2><SPAN class="mw-headline" id="Lab_update_3"> Lab update </SPAN></H2><DL><DD><B>Indigiodine</B></DD></DL><UL><LI> we have 3 constructs which work
</LI><LI> now we want to find out which T-domains are the best to use for the NRPS library
</LI><LI> we are going to clone indigoidine into igem-standard and characterize it
</LI></UL><DL><DD><B>Tyrocidine</B></DD></DL><UL><LI> a few fragments don't have sufficient concentration --&gt; optimization
</LI><LI> amplification of some fragments didn't work
</LI><LI> first gibson assembly will be done this week
</LI><LI> talked to different people because of using their mass-spectrometry 
</LI></UL><DL><DD><B>DelH</B></DD></DL><UL><LI> changed the backbone
</LI><LI> managed to create constructs, did the electroporation
</LI><LI> there were no colonies growing on the plates
</LI></UL><DL><DD><B>DelRest</B></DD></DL><UL><LI> were able to amplify all parts
</LI><LI> started to validate fragments
</LI><LI> start gibson as soon as we checked fragments
</LI></UL><H2><SPAN class="mw-headline" id="NRPS-Software_2"> NRPS-Software </SPAN></H2><UL><LI> Database is more or less implemented
</LI><LI> main programm is able to most of what it is supposed to
</LI><LI> Webinterface has been started but doesn't comprise all features
</LI><LI> connection Pfam works
</LI></UL><H2><SPAN class="mw-headline" id="Art_Cooperation"> Art Cooperation </SPAN></H2><UL><LI> maybe involving artists into our meeting
</LI><LI> talk to them about our projects --&gt; maybe they like some parts of our projects and we can collaborate
</LI><LI> we could give them a insight into synthetic biology
</LI><LI> we can think about offering some of the artists to stay with us during the period of the art project
                                </LI><LI class="active"><A href="#a15" data-toggle="tab">2013-08-06</A></LI><LI><A href="#b15" data-toggle="tab">2013-08-07</A></LI><LI><A href="#c15" data-toggle="tab">2013-08-14</A></LI><LI><A href="#d15" data-toggle="tab">2013-08-28</A></LI></UL></DIV><DIV class=" weekly"><DIV class="scrollContent nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="a15"><H2><SPAN class="mw-headline" id="Organisation_2"> Organisation </SPAN></H2><UL><LI> Slides until Wednesday, 11 am to Philipp
</LI><LI> Slides will also go to advisors
</LI><LI> bring two laptops with working Skype
</LI><LI> will not talk about safety to Eils
</LI><LI> BioBricks have to arrive at the registry until September 18
</LI></UL><H2><SPAN class="mw-headline" id="Software_5"> Software </SPAN></H2><UL><LI> C++ part builds almost fully statically
</LI><LI> database finished, first entries
</LI><LI> need to verify iGEM requirements for software submission
</LI><LI> use same design and color palette for all web sites
</LI><LI> iGEM42, NRPSDesigner will be linked to in the wiki
</LI></UL><H2><SPAN class="mw-headline" id="Del"> Del </SPAN></H2><UL><LI> Prof. Eils knows about strains, company's progress
</LI></UL><H2><SPAN class="mw-headline" id="Tyrocidine_2"> Tyrocidine </SPAN></H2><UL><LI> assembled constructs
</LI><LI> picked colonies, miniPreps
</LI><LI> planned: quantification gel -&gt; restriction digest
</LI><LI> short peptide NRPS in RFC10
</LI><LI> individual modules for submission planned in RFC10
</LI></UL><H2><SPAN class="mw-headline" id="Indigoidine_6"> Indigoidine </SPAN></H2><UL><LI> assembly of parts for submission in progress
</LI><LI> T-domains shuffling in progress
</LI><LI> blue colonies with different PPTases (constructs not verified yet)
</LI><LI> use RFC10 for submission
                                <DIV class="tab-pane" id="b15"><P style="font-size:12pt; text-align:justify;"><A href="/wiki/index.php?title=Team:Heidelberg/Templates/M-07-08-13&amp;action=edit&amp;redlink=1" class="new" title="Team:Heidelberg/Templates/M-07-08-13 (page does not exist)">Team:Heidelberg/Templates/M-07-08-13</A></P></DIV><DIV class="tab-pane" id="c15"><P style="font-size:12pt; text-align:justify;"><A href="/wiki/index.php?title=Team:Heidelberg/Templates/M-14-08-13&amp;action=edit&amp;redlink=1" class="new" title="Team:Heidelberg/Templates/M-14-08-13 (page does not exist)">Team:Heidelberg/Templates/M-14-08-13</A></P></DIV><DIV class="tab-pane" id="d15"><P style="font-size:12pt; text-align:justify;"><A href="/wiki/index.php?title=Team:Heidelberg/Templates/M-28-08-13&amp;action=edit&amp;redlink=1" class="new" title="Team:Heidelberg/Templates/M-28-08-13 (page does not exist)">Team:Heidelberg/Templates/M-28-08-13</A></P></DIV></LI><LI class="active"><A href="#a16" data-toggle="tab">Overview</A></LI><LI><A href="#b16" data-toggle="tab">Date1</A></LI></UL></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#a17" data-toggle="tab">Overview</A></LI><LI><A href="#b17" data-toggle="tab">Date1</A></LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12"><DIV class="imageMap" aria-haspopup="true"><CENTER>Thanks to </CENTER></DIV></DIV></DIV><DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Heidelberg/Meetings">http://2013.igem.org/Team:Heidelberg/Meetings</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Meetings" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Meetings" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Meetings&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Meetings&amp;oldid=316322" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>