<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Project_Software"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Project_Software">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Project_Software&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project_Software&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project_Software&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Project_Software" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Project Software</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="row"><DIV id="nav" class="navigation"><DIV class="item team" data-offset-l="-130"><H2>Team</H2><UL><LI><A href="/Team:Heidelberg/Team">Team Members</A></LI><LI><A href="/Team:Heidelberg/AttributionsII">Attributions</A></LI><LI><A href="/Team:Heidelberg/Team/Sponsors">Sponsors</A></LI><LI><A href="/Team:Heidelberg/Team/Collaboration">Collaborations</A></LI><LI><A href="/Team:Heidelberg/Team/Gallery">Gallery</A></LI></UL></DIV><DIV class="item project" data-offset-l="-144"><H2>Project</H2><UL><LI><A href="/Team:Heidelberg/NRPS">Background</A></LI><LI><A href="/Team:Heidelberg/Project">Overview</A></LI><LI><A href="/Team:Heidelberg/Project/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Project/Indigoidine-Tag">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Project/Tag-Optimization">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Project/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Project/Achievements">Achievements</A></LI></UL></DIV><DIV class="item notebook" data-offset-l="-139"><H2>Notebook</H2><UL><LI><A href="/Team:Heidelberg/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Tyrocidine">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Indigoidine">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/NRPSDesigner">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/Notebook/Methods">Methods</A></LI><LI><A href="/Team:Heidelberg/Materials">Materials</A></LI></UL></DIV><DIV class="item parts" data-offset-l="-134"><H2>Parts</H2><UL><LI><A href="/Team:Heidelberg/Parts">Submitted Parts</A></LI><LI><A href="/Team:Heidelberg/Favorite_Parts">Favorite Parts</A></LI><LI><A href="/Team:Heidelberg/RFCs">Our RFCs</A></LI></UL></DIV><DIV class="item software" data-offset-l="-134"><H2>Software</H2><UL><LI><A href="/Team:Heidelberg/Project_Software">Project</A></LI><LI><A href="http://igem2013.bioquant.uni-heidelberg.de/NRPSDesigner/">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/iGEM42">iGEM42</A></LI><LI><A href="/Team:Heidelberg/OpenSource">Open Source</A></LI><LI><A href="/Team:Heidelberg/Project/AchievementsSoftware">Achievements</A></LI></UL></DIV><DIV class="item human" data-offset-l="-118"><H2>Human Practice</H2><UL><LI><A href="/Team:Heidelberg/HumanPractice">Overview</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BroadPublic">Broad Public</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Survey">Survey</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/YoungGeneration">Young Generation</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Artists">Arts and Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BeyondScience">Beyond Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Experts">Experts</A></LI></UL></DIV><DIV class="item safety" data-offset-l="-115"><H2>Safety</H2><UL><LI><A href="/Team:Heidelberg/Safety">Safety</A></LI></UL></DIV><DIV class="item model" data-offset-l="-144"><H2>Modeling</H2><UL><LI><A href="/Team:Heidelberg/Modelling/Gold_Recovery">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Modelling/Ind_Production">ind-Production</A></LI></UL></DIV><DIV class="item"><SPAN style="margin-left:6%">HOME</SPAN></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-sm-8"><H1><SPAN style="font-size:150%;color:#074E0B;">NRPSDesigner.</SPAN><SPAN class="text-muted" style="font-size:90%"> Design your own NRP.</SPAN></H1></DIV></DIV><DIV class="row"><DIV class="col-sm-6"><DIV class="jumbotron" style="padding:38px"><H2>Highlights</H2><P style="font-size:16px"><UL style="font-size:14px"><LI><B> Computer aided design of fully synthetic NRPS determined to produce a user-defined short peptide</B></LI><LI> Optimal domain assembly based on evolutionary distance</LI><LI> The curated database stores information of 658 Domains encoded on 99 DNA sequences</LI><LI> Automated domain recognition for newly entered NRPS sequences</LI><LI> Integration of Gibthon to facilitate implementation of cloning strategy</LI><LI> Parts registry interface and SBOL output format</LI><LI><A href="http://hdl.handle.net/1721.1/81333"><B>RFC100</B></A> for non-ribosomal peptide synthesis design</LI><LI> Have a look at our tutorial video: <A href="http://igem2013.bioquant.uni-heidelberg.de/NRPSDesigner/">here</A></LI></UL></P></DIV></DIV><DIV class="col-sm-6"><DIV class="jumbotron abstract"><H2>Abstract</H2><P style="font-size:14px; text-align:justify"><B>Non-ribosomal peptide (NRP) synthesis</B> is a biochemical process of remarkable hierarchical organization. Vertically it can be described stepwise starting from the coding DNA sequence that is translated into a giant enzyme catalyzing in turn the actual NRP assembly. Horizontally its complexity is established by a modular order of functional proteinogenic units. Due to this systematic composition a bioinformatic approach appears most suitable, if we aim for the automated design of <B>fully synthetic NRPs</B>. 
Here, we introduce a comprehensive software tool, the <A href="http://igem2013.bioquant.uni-heidelberg.de/NRPSDesigner/">NRPSDesigner</A>, which facilitates the prediction and synthesis of <B>non-ribosomal synthetases (NRPS)</B> that catalyze customized NRP-assembly. The predictive power of the NRPSDesigner is based on a curated database storing information of about <B>200 NRPS modules</B>, their DNA coding sequences and substrate specificities. It is used to calculate the optimal domain sequence according to the <B>weighted phylogenetic distance</B> between domain origins. Additionally an integrated <B>domain recognition</B> algorithm allows for curated expansion of the database. To accelerate the process from in silco NRPS design towards experimental validation we embedded the <B>Gibthon iGEM software</B> tool of Cambridge 2010 for Gibson primer construction. With this framework we want to suggest a new standard for the fast and accurate computer aided design of customized short peptides.
                        </P></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row" style="float:left"><DIV id="softwareText" class="col-sm-12"><H2 id="introduction">Introduction</H2><P>Even though biological processes can be characterized by their physicochemical properties, they can also be translated into an abstract model of interconnected functional entities. This is exemplified by the central dogma of Biology describing the information flow from DNA, to RNA and finally proteins. Synthetic biology has always tried to interfere with these levels of organization with the goal of systematically controlling the projected outcome <SPAN class="citation">[1]</SPAN>.</P><P>NRPS carry this principle to the extreme by adding yet another hierarchical level:</P><P>The modular complex sequentially synthesizes its own short non-ribosomal peptide (NRP). These peptides in turn are not limited by the standard set of proteinogenic amino acids; instead D-isoforms and diverse modifications can be utilized <SPAN class="citation">[2]</SPAN>. Nature has made great use of this system by creating versatile natural products such as antibiotics, metallophores or dyes. <SPAN class="citation">[2]</SPAN><SPAN class="citation">[3]</SPAN></P><P>Although, little is known about the actual dynamic properties of the synthesis process most of its logical rules are indeed understood: A NRP-synthetase consists of a number of modules, each of which is responsible for adding one amino-acid to the nascent peptide. But even a module can be further sub-divided into domains, each with a distinct functionality <SPAN class="citation">[4]</SPAN>.</P><P>This hierarchical organization demonstrates the large potential for the synthetic biology community: The exchange or combination of modules and domains from different organisms or different proteins has been repeatedly shown to produce fully functional NRPS <SPAN class="citation">[5]</SPAN><SPAN class="citation">[6]</SPAN>. Not surprisingly, several bioinformatics approaches have put great effort to meticulous categorize NRPS and their functionality. For example, databases such as NRPS-PKS <SPAN class="citation">[7]</SPAN> or Clustermine360  describe the domain organization of diverse NRPS, while the Norine <SPAN class="citation">[8]</SPAN> database includes information about non-ribosomal peptides and their sequence of monomers. Also, many tools are capable of predicting domain sequences, substrate specificity and hence the putative product of a particular NRPS. Exemplary, the NRPS-PKS <SPAN class="citation">[7]</SPAN> and PKS-NRPS <SPAN class="citation">[9]</SPAN>analysis tool (hereafter referred to as Maryland tool) elaborate in this direction. While antiSMASH <SPAN class="citation">[10]</SPAN><SPAN class="citation">[11]</SPAN> also provides similar prediction capabilities its scope is broader and covers many different secondary metabolite pathways.</P><P>At the same time, as the understanding of the underlying biological processes and methods for assembly of diverse DNA constructs has improved, many novel software tools aim at the computer aided design (CAD) of DNA sequences (e.g. J5, ). Such tools have been particularly valuable to the iGEM community, as they stimulate the design of more complicated, yet less error-prone biological devices. Two examples originating from the iGEM community are: <A href="http://www.clothocad.org/">Clotho</A>, a framework enabling the automated and computer-assisted design of synthetic biology constructs introduced by the <A href="https://2008.igem.org/Team:UC_Berkeley_Tools">Berkeley iGEM team of 2008</A> and <A href="http://django.gibthon.org/">Gibthon</A> created as a web app by the <A href="https://2010.igem.org/Team:Cambridge/Tools/Gibson">Cambridge iGEM Team of 2010</A>, which suggests primers in order to assemble a set of predefined DNA fragments using Gibson cloning.</P><P>Influenced by this development, we introduce here the <A href="http://igem2013.bioquant.uni-heidelberg.de/NRPSDesigner/">NRPSDesigner</A>, an integrated CAD software, implemented to facilitate the design of customized synthetic NRPs. In particular, the NRPSDesigner includes the following features: Based on the NRPS-PKS database we built a manually curated database capturing the biological complexity of NRPS while storing information of about 200 NRPS modules their coding sequences and substrate specificity. The database can be easily extended with curated content using automated domain prediction based on Hidden Markov Models. By applying this information, the NRPSDesigner can calculate an optimal sequence of domains based on simple evolutionary assumptions. To accelerate the process of testing this synthetic construct and eventually produce a customized peptide we included additional assisting software to the framework. We offer to incorporate the necessary domains for combining the nascent peptide with an <A href="https://2013.igem.org/Team:Heidelberg/Project/Indigoidine"> Indigoidine tag </A>. Furthermore, embedding of the Gibthon software automates the suggestion of primers necessary for the assembly of the predicted domains by Gibson cloning.</P><H2 id="results">Results</H2><H3 id="nrpsdesigner-database-structure">NRPSDesigner Database Structure</H3><P>The NRPSDesigner is a knowledge-based software tool using stored information about NRPS pathways to predict the optimal domain sequence that is able to produce a user-defined NRP. The storage organization is of great importance for the functionality of the designer because of its dependence on a comprehensive description of the biological and biochemical properties of NRPSs. For this purpose we built a hierarchical database 
that comprises three layers of complexity: i) the DNA level represented by all DNA coding sequences. They are directly linked to ii) their encoded NRPS domain, respectively. Finally iii) our database stores detailed information about the substrate and its potential modification of the corresponding domain. (see <A class="fancybox fancyFigure" href="https://static.igem.org/mediawiki/2013/6/65/Heidelberg2013_DesignerDb.png" title="Schematic overview of the tabular structure of the NRPSDesigner database. The database layout comprises three layers of complexity: The coding sequences comprising all information on the DNA level; the NRPS domain level describing all building blocks that are available for the NRPS Designer and the substrate level harbouring all monomers that have domain specificity. ">Fig. 1</A>)</P><CENTER><A class="fancybox fancyFigure" href="https://static.igem.org/mediawiki/2013/6/65/Heidelberg2013_DesignerDb.png" title="Schematic overview of the tabular structure of the NRPSDesigner database. The database layout comprises three layers of complexity: The coding sequences comprising all information on the DNA level; the NRPS domain level describing all building blocks that are available for the NRPS Designer and the substrate level harbouring all monomers that have domain specificity. " rel="gallery1"><FIGCAPTION style="width:60%;"><B>Figure 1: Schematic overview of the tabular structure of the NRPSDesigner database. </B> The database layout comprises three layers of complexity: The coding sequences comprising all information on the DNA level; the NRPS domain level describing all building blocks that are available for the NRPS Designer and the substrate level harbouring all monomers that have domain specificity. </FIGCAPTION></A></CENTER><P>Next to the tight links between these layers all of them also point at additional database entries that complete the needed information for the design algorithm. For example, a DNA coding sequence is linked not only to its product, the translated domain, but also to its origin (organism, plasmid etc.) Additionally, a coding sequence can also be connected to another coding sequence. This âparentâ sequence is a predecessor of the stored sequence that already underwent biosynthetic modification. On the domain level there is an upstream link to the coding sequence but also to the specific type of domain (e.g. thioesterase or the condensation domains). Some Domains, such as adenylation domains, also point at monomers, based on their substrate specificity. Subsequently, for these substrates we store their chirality, modification and if they are proteinogenic or not, e.g. histidine or ornithine (see <A class="fancybox fancyFigure" href="https://static.igem.org/mediawiki/2013/7/76/Heidelberg2013_ModifiedPeptide.png" title="Exemplary substrate composition retrieved from the NRPSDesigner database. Proteinogenic (e.g L-histidine) and non-poteinogenic amino acids (D-ornithine) can be selected. L/D describe the chirality of the molecule. Modifications are depicted in cyan."> Fig. 2 </A>).

To enable the NRPSDesigner to use information from outside of the database it is equipped with global identifiers. For organisms we saved the NCBI taxon id, while for BioBricks the unique identifier in the Parts registry. To integrate the content with other databases, we created for every layer a linkout entry that consists of a type and specific identifier. The linkout type includes a description of the corresponding resource, as well as a URL, which in combination with the specific identifier enables the cross-linking of each database entry to other resources. The most common linkout types are Norine and Pubchem IDs for the substrates, PFAM IDs for the domain types and GenBank identifiers for the coding sequences.</P><P>For visualization of the NRPS domains and the chemical structures of the substrates by Open Babel <SPAN class="citation">[12]</SPAN> we added a <A href="http://www.json.org/">JSON</A> representation of each domain, based on the <A href="http://pfam.sanger.ac.uk/help/#tabview=tab9">Pfam Graphics library</A> and the SDF (structure data file) format, respectively.</P><CENTER><A class="fancybox fancyFigure" href="https://static.igem.org/mediawiki/2013/7/76/Heidelberg2013_ModifiedPeptide.png" title="Exemplary substrate composition retrieved from the NRPSDesigner database. Proteinogenic (e.g L-histidine) and non-poteinogenic amino acids (D-ornithine) can be selected. L/D describe the chirality of the molecule. Modifications are depicted in cyan." rel="gallery1"><FIGCAPTION style="width:60%;"><B>Figure 2: Exemplary substrate composition retrieved from the NRPSDesigner database. </B> Proteinogenic (e.g L-histidine) and non-poteinogenic amino acids (D-ornithine) can be selected. L/D describe the chirality of the molecule. Modifications are depicted in cyan.</FIGCAPTION></A></CENTER><H3 id="nrpsdesigner-database-content">NRPSDesigner Database Content</H3><P>The information currently stored in the NRPS database was mainly retrieved from already published data <SPAN class="citation">[7]</SPAN> and extended or changed according to our own experimental results. Because of the already present domain organization and the related substrate specificities we mainly filled our database with information from the NRPS-PKS  library, which saves the protein sequence of diverse NRPS, and added the missing coding sequences. Accordingly, the positions of domain boundaries and linkers had to be converted from their protein specific coordinates to DNA coordinates. In some cases manual curation was necessary, because the substrate specificity stored in the NRPS-PKS database erroneously did not agree with the published experimental results. Furthermore, several pathways seem to contain special domain types with unclear functionality. In such a case we used antiSMASH combined with in-depth literature research to clarify the correct domain functionality. The current progress of the database curation is shown in <A href="https://2013.igem.org/Team:Heidelberg/NRPSDesignerDB_Curation_Table"> table 1</A>. In total our database contains curated information about 658 Domains encoded on 99 DNA sequences.</P><H3 id="extension-and-validation-of-the-database">Extension and Validation of the Database</H3><P>One of the core requirements for our software is the ability to detect NRPS domains. This feature enables the automatic and standardized definition of domain boundaries and also facilitates the addition of new entries into the NRPSDesigner database by the community.</P><P>Based on the antiSMASH <SPAN class="citation">[10]</SPAN><SPAN class="citation">[11]</SPAN> program we established our own pipeline to maximize domain recognition specificity. antiSMASH is implemented in <A href="http://biopython.org/wiki/Main_Page">(BIO)Python</A> and uses HMMER3 <SPAN class="citation">[13]</SPAN> and thus, could be easily integrated in our framework. To improve domain recognition especially for the Adenylation/Oxidation/Adenylation (AOxA) domain of IndC, a HMM profile of the AOxA domain appearing in diverse indigoidine synthetases was constructed and added to the pipeline. Also, the Adenylation domain HMM (from Pfam, id PF00501.21) of antiSMASH was replaced by the HMM of the Maryland tool (special thanks to Prof. Jacques Ravel for providing us with the seed alignment). Note, that the start boundary for the A domain varied a lot depending on the prediction tool used. We chose the Maryland HMM, because it agreed most with our Tyrocidine experiments, as well as previously published results <SPAN class="citation">[14]</SPAN>. Furthermore, Thiolation domains have been split into two categories, depending on whether an epimerisation domain follows or not, as experimental evidence has shown that their functionality differs <SPAN class="citation">[15]</SPAN> (see <A class="fancybox fancyFigure" href="https://static.igem.org/mediawiki/2013/2/2e/Heidelberg2013_TycCAclustalComparison.png" title=" Domain border prediction. Upper panel: Clustal Omega MSA of different C-A domain borders of Tyrocidine cluster with annotation (prediction) of the start of the A domain according to different tools and experimental results. In accordance with our experimental results this domain border is predicted with the Maryland HMM. Lower panel: Consensus sequence of the MSA.">Fig. 3</A>).</P><CENTER><A class="fancybox fancyFigure" href="https://static.igem.org/mediawiki/2013/2/2e/Heidelberg2013_TycCAclustalComparison.png" title=" Upper panel: Clustal Omega MSA of different C-A domain borders of Tyrocidine cluster with annotation (prediction) of the start of the A domain according to different tools and experimental results. In accordance with our experimental results this domain border is predicted with the Maryland HMM. Lower panel: Consensus sequence of the MSA." rel="gallery1"><FIGCAPTION style="width:60%;"><B>Figure 3: Domain border prediction.</B> Upper panel: Clustal Omega MSA of different C-A domain borders of Tyrocidine cluster with annotation (prediction) of the start of the A domain according to different tools and experimental results. In accordance with our experimental results this domain border is predicted with the Maryland HMM. Lower panel: Consensus sequence of the MSA.</FIGCAPTION></A></CENTER><P>To easily extend the NRPSDesigner database this domain recognition pipeline was integrated into a user-friendly interface. The user can add an additional description for each domain, possibly change the specificity of the adenylation domains if it has not been predicted correctly and also define his own domain boundaries, with the help of an integrated multiple sequence alignment against the domains of the same type already present in the database.</P><H3 id="guidance-for-cloning-of-nrps-constructs-with-gibthon">Guidance for Cloning of NRPS Constructs with Gibthon</H3><P>Going further than offering in-silico predicted sequences of NRPS domains producing a particular NRP would be the implementation of a cloning procedure based on Gibson assembly for the NRPSDesigner. This should make NRPS more accessible to the synthetic biology community. One of the most popular tools for computer-aided primer design is Gibthon. Created as a web app by Bill Collins of the Cambridge iGEM Team of 2010 Gibthon suggests primers for the assembly of predefined DNA fragments (entered by the user or imported from the Parts Registry) using Gibson cloning. Since Gibthon was such a successful iGEM software project and also written in Django, we decided to use it as our tool of choice for automated primer design.</P><P>Gibthon was integrated into the core NRPSDesigner GUI using a modular interface. Particular care was taken to keep Gibthon and the rest of the NRPSDesigner clearly separated, to enable the use of other primer design software such as J5 <SPAN class="citation">[16]</SPAN> in the future. The strategy applied for the integration is the following: For each of the domains returned by the in-silico prediction, the DNA sequence is extracted from the NRPSDesigner database. The resulting sequences are returned in a plasmid structure, while ensuring that a minimal number of Gibson fragments has to be assembled. These sequences, together with metadata, such as references and descriptions, are appropriately converted to the Gibthon database format and then copied into the Gibthon gene fragment table. In return the user has access to the standard Gibthon interface to get an overview over the suggested primers. Similar to Gibthon the NRPSDesigner is tightly linked with the registry of standard biological parts. The user can add his parts of choice, using the automated parts registy import tool. However, some additional restrictions have been placed in order to ensure the integrity of the designed NRPS sequence: The user cannot enter a new fragment/part in between one of the NRPS domains; instead he can only place it after the Thioesterase and before the initiation adenylation domain.</P><H3 id="in-silico-design-of-the-optimal-nrps-sequence"><I>In silico</I> Design of the Optimal NRPS Sequence</H3><P>The quickly growing number of known NRPS pathways increases the pool of modules and domains catalysing the incorporation of a specific peptide monomer. Especially on the domain level there is often more than one NRPS protein specific for the same substrate or catalytic reaction. To be able to determine the optimal selection we assume that a combination of domains exists which will provide the best possible performance when combined to a NRPS, and that this performance can be objectively quantified relative to other domain combinations. To find the optimal domain combination the shortest path from a start to an end point on a graph-based representation of the problem needs to be determined. The NRPSDesigner constructs a directed acyclic graph (DAG) representing all possible domain combinations capable of synthesizing the desired peptide (see <A class="fancybox fancyFigure" href="https://static.igem.org/mediawiki/2013/e/eb/Heidelberg2013_Dijkstra.png" title="Dijkstra Algorithm illustration as applied in the NRPSDesigner. In this trivial example, Adenylation domains are connected with thiolation domain and the minimal evolutionary distance is returned when both domains come from the same species (&lt;i&gt;Photorhabdus luminescens &lt;/i&gt;)" rel="gallery1">Fig. 4</A>). The DAG is organized in layers, corresponding to positions in the final NRPS. Every layer comprises a discrete number of nodes representing NRPS domains and every node of a layer is connected to every node of the next layer. However, no edges within a layer or to previous layers exist. The first and last nodes, being the start and end points of the shortest path problem, are dummy nodes without an associated domain, edges pointing to or from them having a weight of zero.</P><CENTER><A class="fancybox fancyFigure" href="https://static.igem.org/mediawiki/2013/e/eb/Heidelberg2013_Dijkstra.png" title="Dijkstra Algorithm illustration as applied in the NRPSDesigner. In this trivial example, Adenylation domains are connected with thiolation domain and the minimal evolutionary distance is returned when both domains come from the same species (&lt;i&gt;Photorhabdus luminescens &lt;/i&gt;)"><FIGCAPTION style="width:60%;"><B>Figure 4: Dijkstra Algorithm illustration as applied in the NRPSDesigner. </B> In this trivial example, Adenylation domains are connected with thiolation domain and the minimal evolutionary distance is returned when both domains come from the same species (<I>Photorhabdus luminescens </I>).</FIGCAPTION></A></CENTER><P>So far, very little is known about the potential functionality of synthetic NRPS including possible assembly rules for modules and domains. Therefore we assume a simple scoring function for the edge weights between the user-selected domains, where combinations of domains from closely related organisms have a higher probability to form a functional NRPS than domain combinations involving organisms which are far apart on the taxonomic tree. The taxonomic distance is currently determined from lineage information available in the NCBI taxonomy database <SPAN class="citation">[17]</SPAN>, but could be easily extended to include sequence-based information <SPAN class="citation">[18]</SPAN> or other, possibly better fitting, criteria as soon as more experimental evidence becomes available. In addition, domains which are directly consecutive in their native organism are preferred over those from different modules, if possible. The well known shortest path algorithm by E.W. Dijkstra <SPAN class="citation">[19]</SPAN> is then employed to find the optimal domain combination, using the dummy nodes as start and stop nodes.</P><H3 id="implementation">Implementation</H3><P>Following the example of major bioinformatics applications like BLAST <SPAN class="citation">[20]</SPAN>, the NRPSDesigner was split into a command-line executable comprising the core algorithm, and a web interface providing easy access to the functionality enriched with some extra features. This design enables potential users to set up their own NRPSDesigner instance according to their needs using the compiled executable and a database dump.</P><H3 id="the-command-line-executable">The Command-line Executable</H3><P>We chose C++ as implementation language for the command-line executable to avoid dependencies on run-time interpreters and to maximise speed. Additionally the usage of C++ features like smart pointers, lambda functions, or hash tables from the new C++11 standard allows for terser, faster, less error-prone code. To facilitate re-use of the code in other contexts, all of the functionality is contained within a (currently static) library, with the executable being a thin wrapper around it. In the future, even tighter integration with the web interface can be imagined. In this case a python wrapper would be implemented around the library using a software package like SIP or SWIG.</P><P>In case the PartsRegistry or some other publically accessible NRPS database offers the necessary functionality at some point in the future we decoupled the algorithm from the database and allowed multiple database backends. We designed an abstract base class defining the interface required by other parts of our code, and a static factory function responsible for constructing an object of the appropriate backend class employing the singleton pattern, which allows only one instance of the database backend to exist within the entire program. Currently, only one database backend exists, capable of retrieving data from our <A href="http://www.mysql.com/">MySQL</A> database by using the MySQL C++ connector library. Custom exception classes complete the abstraction, as all SQL exceptions are caught within the MySQL backend and a DatabaseError is thrown containing information about the specific error.</P><P>Taxonomy information is retrieved on-demand from the NCBI server via the NCBI XML API <SPAN class="citation">[21]</SPAN> using the CURL network library and further parsed using the LibXML library. In case of network failure, the NRPSDesigner can fall back to a local copy of the taxonomy database, which can be retrieved from the NCBI FTP server in advance. In order to reduce network traffic and memory consumption when the final graph is built, as little data as possible is loaded in advance. The missing taxonomy information is retrieved lazily in parallel to the shortest path computation and cached afterwards. DNA sequences and metadata not required for the pathway construction are fetched after the pathway has been constructed only for the domains the pathway is comprised of. The final pathway is returned in form of an XML file, built with the LibXML library, containing information on the domains and their origin.</P><P>For command-line option parsing, the <A href="http://www.boost.org/doc/libs/1_54_0/doc/html/program_options.html">Boost.program_options</A> library was chosen, as it promotes modularity by allowing every component to specify its own set of options. This enables switching between different database backends or changing the existing one without the rest of the program knowing about it. Although this approach introduces an extra dependency to the NRPSDesigner library itself and not just the executable, Boost.program_options allows querying of options, their associated data type and other metadata. Therefore e.g. a graphical user interface written in C++ could extract this information and dynamically build the required input fields, without knowing anything about the underlying algorithm or backends.</P><H3 id="the-web-interface">The Web Interface</H3><P>In order to allow tight integration with the Gibthon software package of iGEM 2010 we chose the <A href="https://www.djangoproject.com/">Django framework</A>, written in Python, as the back-end of the web interface (see <A class="fancybox fancyFigure" href="https://static.igem.org/mediawiki/2013/6/66/Heidelberg_Figure5NRPS.png" title="The NRPS Designer GUI. A) Peptide monomers can be selected from the NRPSDesigner database. Optional an Indigoidine-Tag can be added to the NRP layout. B) Gibthon interface allows for Gibson primer prediction and download to facilitate cloning strategy development." rel="gallery1">
Fig. 5</A>). The front-end was implemented using the Bootstrap 2, jQuery and jQueryUI frameworks. The interface allows to select the desired peptide monomers and specify their configuration/modification. Subsequently, a 2D structure preview of the resulting molecule is generated on-the-fly using the OpenBabel library <SPAN class="citation">[12]</SPAN> via its Python bindings. The structures of the individual amino acids are captured from the database, performing a SMARTS pattern search for the NH2-CR-COOH group and constructing peptide bonds between consecutive monomers. The resulting peptide structure is projected to 2D coordinates, exported as SVG and then directly sent to the browser. To reduce server load, structures are cached using Djangoâs view caching functionality.Heavy computations like pathway construction or domain prediction are performed asynchronously outside of the web server to counter long computation times and request timeouts. The Celery Python package combined with RabbitMQ is used for managing and scheduling the tasks. Additionally log messages are collected and retrieved by the browser via AJAX polling. After the pathway has been constructed, a visualizaton of the resulting NRPS is generated using the PFAM graphics JavaScript library, as well as a Gibson construct. To this end, the Gibthon software package was integrated and slightly modified, such that no fragments can be inserted between consecutive NRPS domains, automatically generated fragments are hidden from the user, and user-defined fragments are sorted by their type. Additionally, a bug was fixed preventing import of BioBricks from the Parts Registry.</P><CENTER><A class="fancybox fancyFigure" href="https://static.igem.org/mediawiki/2013/6/66/Heidelberg_Figure5NRPS.png" title="The NRPS Designer GUI. A) Peptide monomers can be selected from the NRPSDesigner database. Optional an Indigoidine-Tag can be added to the NRP layout. B) Gibthon interface allows for Gibson primer prediction and download to facilitate cloning strategy development."><FIGCAPTION style="width:100%;"><B>Figure 5: The NRPS Designer GUI. </B> A) Peptide monomers can be selected from the NRPSDesigner database. Optional an Indigoidine-Tag can be added to the NRP layout. B) Gibthon interface allows for Gibson primer prediction and download to facilitate cloning strategy development.</FIGCAPTION></A></CENTER><P>Users are able to enter new domains and pathways into the database by providing the coding sequence of their NRPS. After simple validity checks using the BioPython package, a modified version of the antiSMASH pipeline <SPAN class="citation">[10]</SPAN><SPAN class="citation">[11]</SPAN>, optimized for NRPS domain prediction, is run on the sequence to determine domain types, their positions, and substrate specificity if there is any. Results of the prediction are presented to the user, who is able to manually verify their accuracy and correct possible mistakes. To further improve domain linker definition, a multiple sequence alignment of a domain against all other domains of the same type in the database can be performed using the Clustal Omega software package <SPAN class="citation">[22]</SPAN>. The alignment is presented to the user in an interactive dialog via JavaScript extracted from the Kalignvu <SPAN class="citation">[23]</SPAN> software package, where he can inspect the conserved residues and set the domain linker positions appropriately.</P><H2 id="discussion">Discussion</H2><H3 id="nrpsdesigner-database-compared-to-other-nrps-databases">NRPSDesigner Database Compared to other NRPS Databases</H3><P>While exploring the most commonly used databases for NRPS (Norine <SPAN class="citation">[8]</SPAN> , NRPS-PKS <SPAN class="citation">[7]</SPAN>, Clustermine360 ), we realized that most tools were not designed for the needs of the biosynthetic community. The organization of a gene cluster, the relative arrangement of individual genes, possible native promoters or even other genomic elements as available in Clustermine360 can be neglected for the purpose of NRPS design. In contrast, the approach of NRPS-PKS, to include only protein sequences would not be sufficient as synthetic modification starts at the DNA level. Consequently we decided to establish our own database focusing mainly on DNA coding sequences, domain types and substrate specificities. Additional information needed for the customization of the final construct (regulative promoters, restriction sites etc.) can be included directly via an interface with the registry of standard biological parts.</P><P>To maintain a high curation standard we implemented a semi-automated domain recognition algorithm for the NRPSDesigner framework based on Hidden Markov Models (HMM). Again, we chose to adapt already existing approaches in agreement with our own experimental results (see section). Accordingly, we added a HMM specific for Adenylation/Oxidation/Adenylation domains making our tool the only available that can detect this combined domain commonly found in Indigoidine synthetases. Another HMM exclusively distinguishing between epimerisation domains that follow on thiolation domains further extends the predictive power of the NRPSDesigner.</P><P>Despite the advantages of automated domain recognition we strongly emphasize the importance of manual curation with the help of peer-reviewed sources. In this way the user is allowed to change database entries based on experimental evidence. </P><H3 id="scoring-function">Scoring Function</H3><P>Initially the internal algorithm calculating the optimal sequence of NRPS domains was based on the simple assumption that phylogenetically close modules also work best if assembled together. This principal was translated into a scoring function that assigned linear weights to the edges between neighboring domains proportional to their phylogenetic distance taken from taxonomic data. The shortest path over all domains is calculated by a Dijkstraâs algorithm. As it turns out the underlying scoring function can be further improved incorporating specific experimental evidence. For example, <SPAN class="citation">[24]</SPAN> has shown that the success of domain shuffling depends tremendously on the linker between the new neighboring domains. In particular it was suggested that the C-A linker should be kept intact. Likewise domains that are already originating from the same organism get an additional preference in the scoring function, thus, reducing the number of fragments that have to be amplified by PCR and assembled subsequently. Although already showing convenient results this scoring function will be improved in the next update of the NRPSDesigner. For example, the distance between two domains can be further specified using homology information or the differences in the tertiary structure of both domains.</P><H3 id="interaction-with-the-wet-lab">Interaction with the Wet Lab</H3><P>One of the core aspects of synthetic biology is its interaction with systems biology <SPAN class="citation">[25]</SPAN> and computational modeling in general, which in turn intensively depend on the thorough knowledge exchange with experimentalists. Consequently this was also one of the main goals throughout the development of the NRPSDesigner.</P><P>Experimental evidence had a variety of implication in the design of our software. For one, experiments showed at an early stage that the assumptions of modularity that are essential for the NRPSDesignerâs do indeed hold. Also, they allowed us to further refine the scoring function in regards to the importance of linkers between different domains. The Indigoidine project showed that the linker between T-TE domains is more important than the one between A-T, a result not previously published (to the best of our knowledge). In addition, the exact definition of domain boundaries and the use of appropriate HMMs was guided by the actual experimental results (e.g. C-A domain boundary based on Tyrocidine results). Finally, the Indigoidine tag was included in the software only, once its functionality had been proven in the wet lab.</P><P>But also in opposite direction, the NRPSDesigner could easily influence experimentalists: The most obvious way is the cloning and expression of novel NRPS constructs <EM>in vivo</EM>, as suggested by the Designer. But even the prediction pipeline which is part of the Designer can have valuable implications for the experiments: Domain shuffling, could be facilitated using the domain boundaries currently predicted by the pipeline. Especially, for the recognition of totally unknown NRPS pathways our tool offers the optimal framework.</P><H3 id="connecting-with-the-nrpsdesigner">Connecting with the NRPSDesigner</H3><P>One of they key issues of the NRPSDesigner is its ability to combine scientific rigorousness with a user-friendly interface making both NRPS and synthetic biology more approachable to the broader community. This principal is implemented with the help of several features and interfaces facilitating the process of NRP in silico design and experimental validation. For example, in order to help developing a cloning strategy of the desired construct we added a tool for automated primer design. Although J5 <SPAN class="citation">[16]</SPAN> is a very rigorous software tool with complex primer design algorithms using Primer3  we decided on Gibthon, well known to the iGEM community. It has a very user-friendly interface with direct connection to the Parts Registry. In contrast to J5, Gibthon uses mFold <SPAN class="citation">[26]</SPAN> and throws warnings in case of misprimings or self-primings. But even if J5 proves to be more long lasting or yet an entirely different tool, due to its modular implementation within the Django framework another primer design software could be incorporated. However, so far we focused on the improvement of the Gibthon software solely by including a GeneBank output, which saves primer positions and the description of the assembled fragments. With this version the Gibthon construct can be exported in SBOL, which is also used internally in the NRPSDesigner when C++ communicates with the Python code.</P><P>Beyond providing a user-friendly way of designing new NRPs, the NRPSDesigner also incorporates our novel experimental findings showing that NRPs can be tagged by Indigoidine. The results of the Gibson assembly can be easily screened using a very simple blue-colored readout. This indeed fits very well with the general goal of the NRPSDesigner, which tries to make NRPS cloning accessible to a wider scientific audience: In fact, the Indigoidine tag is a great and cheaper alternative to expensive screening procedures such as mass-spectrometry.</P><H2 id="outlook">Outlook</H2><P>With the NRPSDesigner, we have managed to provide a powerful tool that introduces NRPS to the synthetic biology community, making it easier to create novel NRPs or improve existing antibiotics, pigments, etc. As the community gets involved, the NRPSDesigner will also continuously improve, as more domains will be entered into the database and thus more exotic amino acids will become available for inclusion into the final NRP. Also, as more teams follow our paradigm of submitting NRPS domains to the Parts Registry, it will become possible to clone NRPS constructs for diverse peptides by just using parts available in the Registry, rather than trying to access the original strains. For this foreseeable future we have several important changes planned; some of which will be implemented within the next month, while others are long term goals:</P><H3 id="nrpsdesigner-1.1">NRPSDesigner 1.1:</H3><UL><LI><STRONG>Database access</STRONG>: Currently the wealth of information available in the NRPSDesigner database can only be accessed by loading the provided MySQL dump. We intend to make access a lot easier for everyone, by introducing a user-friendly interface for database exploration.</LI><LI><STRONG>Improved Multiple Sequence Alignment view</STRONG>: The multiple sequence alignment (MSA) is supposed to simplify the setting of domain boundaries following the automated boundary predictions, giving extra information for the improvement of the HMMs. This view will get more user-friendly, as the predicted boundaries will be visualized on the MSA and the selection of new boundaries will become the matter of a few clicks.</LI><LI><STRONG>Modifications</STRONG>: You will be able to add N-methylation modifications to a wide-range of amino acids.</LI><LI><STRONG>User Interface based on Bootstrap 3</STRONG>: To simplify the coding architecture for the GUI and stay up to date at the same time we plan to change the coding environment from Boostrap2/jQuery UI into a Bootstrap 3 based application.</LI></UL><H3 id="nrpsdesigner-2.0">NRPSDesigner 2.0:</H3><UL><LI><STRONG>REST API</STRONG>: Programatically access the wealth of knowledge available in the NRPSDesigner database, design domains for new peptides or get suggestions for DNA assembly of the former.</LI><LI><STRONG>Even more modifications</STRONG>: Add cyclizations, heterocyclizations, reductions and oxidations to your final NRP.</LI><LI><STRONG>More safety</STRONG>: Categorize species according to their safety level and issue appropriate warnings for non-S1 organisms. Automatically disable design of NRPS for known or predicted toxic peptides.</LI><LI><STRONG>Structure pattern matching</STRONG>: Rather than having to enter the sequence of monomers for the final NRPs, the user will be able to enter the structure of a molecule of choice and the NRPSDesigner will use structure matching algorithms in order to check whether this molecule is a NRP and suggest a cloning strategy if this is the case.</LI><LI><STRONG>Multiplasmid cloning strategies</STRONG>: For very large NRPS, nature splits the domains on multiple coding sequences and the translated proteins are then combined by N-terminal and C-terminal communication domains. Thus, rather than having to assemble huge constructs on single plasmids, the NRPSDesigner will be able to suggest constructs split across multiple plasmids.</LI><LI><STRONG>Improved Gibthon</STRONG>: More state of the art algorithms for automated primer suggestions allow synthesis of short DNA sequences, such as RBS, in the 5â end of the primers and possibly include other cloning techniques, such as Golden Gate assembly.</LI></UL></DIV><DIV class="col-sm-12 jumbotron"><DIV class="references"><P>1. Endy D (2005) Foundations for engineering biology. Nature 438: 449â453.</P><P>2. Hur GH, Vickery CR, Burkart MD (2012) Explorations of catalytic domains in non-ribosomal peptide synthetase enzymology. Nat Prod Rep 29: 1074â1098.</P><P>3. Reverchon S, Rouanet C, Expert D, Nasser W (2002) Characterization of indigoidine biosynthetic genes in Erwinia chrysanthemi and role of this blue pigment in pathogenicity. J Bacteriol 184: 654â665.</P><P>4. Finking R, Marahiel M a (2004) Biosynthesis of nonribosomal peptides1. Annual review of microbiology 58: 453â488.</P><P>5. Duerfahrt T, Doekel S, Sonke T, Quaedflieg PJLM, Marahiel MA (2003) Construction of hybrid peptide synthetases for the production of alpha-l-aspartyl-l-phenylalanine, a precursor for the high-intensity sweetener aspartame. Eur J Biochem 270: 4555â4563.</P><P>6. Duerfahrt T, Eppelmann K, MÃ¼ller R, Marahiel MA (2004) Rational design of a bimodular model system for the investigation of heterocyclization in nonribosomal peptide biosynthesis. Chem Biol 11: 261â271.</P><P>7. Ansari MZ, Yadav G, Gokhale RS, Mohanty D (2004) NRPS-PKS: a knowledge-based resource for analysis of NRPS/PKS megasynthases. Nucleic acids research 32: W405â13.</P><P>8. Caboche S, Pupin M, LeclÃ¨re V, Fontaine A, Jacques P, et al. (2008) NORINE: a database of nonribosomal peptides. Nucleic acids research 36: D326â31.</P><P>9. Bachmann BO, Ravel J (2009) Chapter 8. Methods for in silico prediction of microbial polyketide and nonribosomal peptide biosynthetic pathways from DNA sequence data. 1st ed. Elsevier Inc.</P><P>10. Medema MH, Blin K, Cimermancic P, de Jager V, Zakrzewski P, et al. (2011) antiSMASH: rapid identification, annotation and analysis of secondary metabolite biosynthesis gene clusters in bacterial and fungal genome sequences. Nucleic acids research 39: W339âW346.</P><P>11. Blin K, Medema MH, Kazempour D, Fischbach M a, Breitling R, et al. (2013) antiSMASH 2.0âa versatile platform for genome mining of secondary metabolite producers. Nucleic acids research 41: W204â12.</P><P>12. OâBoyle NM, Banck M, James CA, Morley C, Vandermeersch T, et al. (2011) Open Babel: An open chemical toolbox. J Cheminform 3: 33.</P><P>13. Finn RD, Clements J, Eddy SR (2011) HMMER web server: interactive sequence similarity searching. Nucleic acids research 39: W29âW37.</P><P>14. Linne U, Marahiel M a (2000) Control of directionality in nonribosomal peptide synthesis: role of the condensation domain in preventing misinitiation and timing of epimerization. Biochemistry 39: 10439â10447.</P><P>15. Linne U, Doekel S, Marahiel M a (2001) Portability of epimerization domain and role of peptidyl carrier protein on epimerization activity in nonribosomal peptide synthetases. Biochemistry 40: 15824â15834.</P><P>16. Hillson NJ, Rosengarten RD, Keasling JD (2012) j5 DNA assembly design automation software. ACS synthetic biology 1: 14â21.</P><P>17. Federhen S (2012) The NCBI Taxonomy database. Nucleic Acids Res 40: D136âD143.</P><P>18. Fang H, Oates ME, Pethica RB, Greenwood JM, Sardar AJ, et al. (2013) A daily-updated tree of (sequenced) life as a reference for genome research. Sci Rep 3: 2015.</P><P>19. Dijkstra EW (1959) A note on two problems in connexion with graphs. Numerische Mathematik 1: 269.</P><P>20. Boratyn GM, Camacho C, Cooper PS, Coulouris G, Fong A, et al. (2013) BLAST: a more efficient report with usability improvements. Nucleic Acids Res.</P><P>21. Sayers E (2008) E-utilities Quick Start. National Center for Biotechnology Information.</P><P>22. Sievers F, Wilm A, Dineen D, Gibson TJ, Karplus K, et al. (2011) Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega. Mol Syst Biol 7: 539.</P><P>23. Lassmann T, Sonnhammer EL (2006) Kalign, Kalignvu and Mumsa: web servers for multiple sequence alignment. Nucleic acids research 34: W596âW599.</P><P>24. Marahiel M a (2009) Working outside the protein-synthesis rules: insights into non-ribosomal peptide synthesis. Journal of peptide science 15: 799â807.</P><P>25. Smolke CD, Silver PA (2011) Informing biological design by integration of systems and synthetic biology. Cell 144: 855â859.</P><P>26. Zuker M (2003) Mfold web server for nucleic acid folding and hybridization prediction. Nucleic acids research 31: 3406â3415.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12"><DIV class="imageMap" aria-haspopup="true"><CENTER>Thanks to </CENTER></DIV></DIV></DIV><DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Heidelberg/Project_Software">http://2013.igem.org/Team:Heidelberg/Project_Software</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Project_Software" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Project_Software" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Project_Software&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Project_Software&amp;oldid=362446" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>