<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_AttributionsII"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/AttributionsII">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/AttributionsII&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/AttributionsII&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/AttributionsII&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/AttributionsII" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/AttributionsII</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="row"><DIV id="nav" class="navigation"><DIV class="item team" data-offset-l="-130"><H2>Team</H2><UL><LI><A href="/Team:Heidelberg/Team">Team Members</A></LI><LI><A href="/Team:Heidelberg/AttributionsII">Attributions</A></LI><LI><A href="/Team:Heidelberg/Team/Sponsors">Sponsors</A></LI><LI><A href="/Team:Heidelberg/Team/Collaboration">Collaborations</A></LI><LI><A href="/Team:Heidelberg/Team/Gallery">Gallery</A></LI></UL></DIV><DIV class="item project" data-offset-l="-144"><H2>Project</H2><UL><LI><A href="/Team:Heidelberg/NRPS">Background</A></LI><LI><A href="/Team:Heidelberg/Project">Overview</A></LI><LI><A href="/Team:Heidelberg/Project/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Project/Indigoidine-Tag">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Project/Tag-Optimization">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Project/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Project/Achievements">Achievements</A></LI></UL></DIV><DIV class="item notebook" data-offset-l="-139"><H2>Notebook</H2><UL><LI><A href="/Team:Heidelberg/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Tyrocidine">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Indigoidine">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/NRPSDesigner">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/Notebook/Methods">Methods</A></LI><LI><A href="/Team:Heidelberg/Materials">Materials</A></LI></UL></DIV><DIV class="item parts" data-offset-l="-134"><H2>Parts</H2><UL><LI><A href="/Team:Heidelberg/Parts">Submitted Parts</A></LI><LI><A href="/Team:Heidelberg/Favorite_Parts">Favorite Parts</A></LI><LI><A href="/Team:Heidelberg/RFCs">Our RFCs</A></LI></UL></DIV><DIV class="item software" data-offset-l="-134"><H2>Software</H2><UL><LI><A href="/Team:Heidelberg/Project_Software">Project</A></LI><LI><A href="http://igem2013.bioquant.uni-heidelberg.de/NRPSDesigner/">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/iGEM42">iGEM42</A></LI><LI><A href="/Team:Heidelberg/OpenSource">Open Source</A></LI><LI><A href="/Team:Heidelberg/Project/AchievementsSoftware">Achievements</A></LI></UL></DIV><DIV class="item human" data-offset-l="-118"><H2>Human Practice</H2><UL><LI><A href="/Team:Heidelberg/HumanPractice">Overview</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BroadPublic">Broad Public</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Survey">Survey</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/YoungGeneration">Young Generation</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Artists">Arts and Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BeyondScience">Beyond Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Experts">Experts</A></LI></UL></DIV><DIV class="item safety" data-offset-l="-115"><H2>Safety</H2><UL><LI><A href="/Team:Heidelberg/Safety">Safety</A></LI></UL></DIV><DIV class="item model" data-offset-l="-144"><H2>Modeling</H2><UL><LI><A href="/Team:Heidelberg/Modelling/Gold_Recovery">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Modelling/Ind_Production">ind-Production</A></LI></UL></DIV><DIV class="item"><SPAN style="margin-left:6%">HOME</SPAN></DIV></DIV></DIV><DIV class="container"><DIV><DIV><H1><SPAN style="font-size:180%;color:#7137c8;">Attributions. </SPAN><SPAN class="text-muted" style="font-size:100%"> Thank You very much!</SPAN></H1></DIV></DIV><DIV><H2 style="color:#7137c8">iGEM Team Heidelberg 2013 </H2><H3 style="" text-align:middle=""> Ilia Kats </H3><P style="text-align:justify">
                One of our team members, the secret nerd in chief of our group, Ilia Kats, had the initial idea for our project and inspired all of us with his dream of easy and efficient recycling of gold. He also came up with the concept to exploit the NRPS modularity for setting up the NRPSDesigner.
                Besides all of his brainwork, Ilia was indispensable for all of the wetlab work for the different subprojects. He was single-handedly responsible for the cloning the methylmalonyl-CoA patwhay for the delftibactin project. In the synthetic peptides, the indigoidine-tag and tag-optimization subproject, he was involved in the pilot experiments. And last but not least, his programming skills were essential for the successfull implementation of our NRPSDesigner software where he used his supernatural force to eliminate even the toughest bugs. 
                 A big THANK U from the whole team for the inspiration and great humor.
                </P></DIV><DIV><H3 style="text-align:middle"> Synthetic Peptides</H3><P style="text-align:justify">
               The main focus of this project was to demonstrate the feasibility of NRPS module shuffling, performed by Joshua Sachs, Philipp Walch, Tania Christiansen and Anja Riedel. They established the experimental strategy and designed all corresponding primers needed for the amplification of the different tyrocidine modules. Tania Christiansen was especially responsible for the BioBrick submission and the cloning of the fragments into the needed RFCs.
                
                 Another essential part in the synthetic peptides subproject was the mass spectrometry, which Joshua Sachs was in charge of. Everyone in this group contributed to the great success of this part of the project! Here, we want to thank our advisor Lorenz Adlung, who helped us with his great experience he shared in frequent lab-meeting discussions.
                </P></DIV><DIV><H3 style="text-align:middle"> Indigoidine-Tag </H3><P style="text-align:justify">
               The indigoidine-tag subproject was about developing an easily detectable, inert and universal tag for NRPs. The work on the module-chains comprising fusion peptides from the tyrocidine cluster were performed by Joshua Sachs, Philipp Walch, Tania Christiansen and Anja Riedel. Philipp Walch and Joshua Sachs performed the valine-indigoidine as well as the di- and tripeptide-indigoidine tagging; Anja Riedel took care of the linker variation. Joshua Sachs and Philipp Walch generated the ccdB-construct. Without the devotion of all of the people mentioned above, the project would not have been feasible. Philipp Walch performed the biobricking of the submitted parts.
              
                We want to thank Dominik Niopek and Hannah Meyer for their expertise when writing the RFC100.
                </P></DIV><DIV><H3 style="text-align:middle"> Tag Optimization </H3><P style="text-align:justify">
                Ralf Beer, Konrad Herbst and Nikolaos Ignatiadis designed strategies for the optimization of the indigoidine-Tag, which includes the creation of 58 engineered indigoidine synthetases and the development of a quantitative time-resolved indigoidine assay. These hard-working fellows did not mind spending 48 hours in a row at the bench, generating and testing large plasmid libraries and performing hundreds of co-transformations, thereby establishing the RFC99. Meanwhile, Konrad proved that one can write a script for almost any purpose. He contributed most to the analysis of the indigoidine assay, the progress of the tag optimization strategy and the RFC99. Nikos was involved in the design of the first constructs and was a big help in the creation of the synthetic T-domains. Ralf focussed on plasmid and primer design as well as the labwork and was actually considered to be integrated into the NRPSDesigner as an alternative sequence-based domain prediction tool. But not only these three guys were infected by the blue fever - thanks to Hannah Meyer they always had an advisor at their side to engage in more and sometimes less scientific discussions.
                </P></DIV><DIV><H3> Gold Recycling </H3><P style="text-align:justify">
                The cloning strategies and amplification of the Del cluster were performed and developed by Florian Schmidt, Sophie Rabe, Nils Kurzawa and Johanna Meichsner. The experiments started with the amplification of DelH executed by Johanna Meichsner, which turned out to be one of the most challenging parts of our project. Johanna proved to be extremely resilient to recurrent setbacks and her power of endurance gave a great example to the whole team. Florian Schmidt, Sophie Rabe and Nils Kurzawa took care of the plasmid assembly of the remaining genes from the Del cluster. After DelRest was successfully amplified, Florian Schmidt and Sophie Rabe worked as well on the assembly of the DelH plasmid and the gold precipitation by delftibactin. Sophie Rabe was furthermore responsible of proteins assays, delftibactin purification and MICRO-TOF, Florian Schmidt established the protocol to dissolve electronic waste. All of them were extremely dedicated and cloned day and night.
                
                Ilia Kats cloned the pIK8 plasmid and all prior plasmids necessary for the establishment of the methylmalonyl-CoA pathway. He performed all steps of the cloning procedure of these plasmids from primer design up to electroporation and screening. 
                
                Here we want to give a special thank you to our advisors Dominik Niopek, Katharina Genreith and Fanny Georgi for their great help in planning of the cloning strategies and introducing us to the world of primer design! Thanks to Fanny for helping with electroporations any time of the day (or night).
                </P></DIV><DIV><H3 style="text-align:middle"> NRPSDesigner </H3><P style="text-align:justify">
                The software-core-team consisted of Konrad Herbst, Ilia Kats, Nikolaos Ignatiadis, Julia Schessner and Nils Kurzawa. They worked together on the implementation of the NRPSDesigner software and the establishment of the respective database. Additionally, Joshua Sachs, Anja Riedel, Julia Schessner and Philipp Walch contributed to the maintenance of the database. Furthermore, administrator Ilia Kats was in charge of our internal wiki and server. All of the software geeks mentioned above worked on our sub-software project iGEM42. The advisors Hannah Meyer and Tim Heinemann helped not only with the NRPSDesigner, but also with motivating the entire team. A big thanks to Hannah for bringing all these yummy cookies and to Tim for helping to eat them!
                </P></DIV><DIV><H3> Modeling </H3><P style="text-align:justify">
                Modeling of the recombinant delftibactin production and the feasibility of the industrial scale gold recycling from electronic waste using delftibactin was mainly performed by Nikolaos Ignatiadis. On his journey through metabolic modeling of delftibactin producing <I>E. coli</I>, bioreactor simulations and cost estimations, he was supported by Johanna Meichsner and Joshua Sachs, who did research on financial data and Florian Schmidt on chemical equations. They received helpful advice by Tim Heinemann.
                
                The modeling of the production kinetics of indigoidine to disentangle the underlying mechanisms of bacterial growth and peptide synthesis was realized by the task force consisting of Konrad Herbst, Ilia Kats, supported by Lorenz Adlung. Konrad was responsible of processing of the time-resolved experimental data, which were generated by him and Ralf Beer. Ilia then generated a model to elucidate the influence of indigoidine production on bacterial growth of bacteria and to provide a more quantitative understanding of the synthesis efficiency of the different T domains and PPTases.
                </P></DIV><DIV><H3 style="text-align:middle"> Wiki Design  </H3><P style="text-align:justify">
                Our wiki is mainly build on stylesheets from Bootstrap 3.0 and was implemented by Nils Kurzawa. He wrote the main javascripts and css stylesheets used on this wiki. Nils and Tim Heinemann came up with the idea of the main page and subsequently designed the templates for all other pages. Tania Christiansen was responsible several pictures such as the symbols used in the wiki navigation. All team members developed quite some talent when generating graphics and figures in Inkskape (yes Ilia, it is a great piece of software). Thanks also to our creative advisors Fanny Georgi (who should think of an alternative career as photographer) and Katharina Genreit who came up with many ideas. After the basic implementation was finished, Nils received help in the wiki design from Nikolaos Ignatiadis, Ilia Kats, Konrad Herbst and Johanna Meichsner. Important to mention is Tim Heinemann for his niggling but always helpful comments.
                </P></DIV><DIV><H3 style="text-align:middle"> Videos</H3><P style="text-align:justify">
               We produced two videos to spread our project to a wider audience.
</P><H4 style="text-align:middle"> Introduction to NRPS Whiteboard Animation </H4>
This short clip explains the basics on NRPS and gives a very short outline of our project.
Tania Christiansen is responsible for the script, illustration and audio/video production. Ralf Beer helped with the audio recording and processing. <H4 style="text-align: middle"> RFC 99 &amp; 100: Peptide Production Pipeline Video </H4><P>
This video demonstrates the implementation of our two RFCs: We show short sequences of the process and explain the single steps to encourage the community to use this new peptide production approach. 
Hanna Meichsner is responsible for the script and audio/video production. Ralph Beer, Tania Christiansen, Nikolaos Ignatidis, Sophie Rabe and Philipp Walch were recorded for several different scenes. 
                </P></DIV><DIV><H3 style="text-align:middle"> Human Practice</H3><P style="text-align:justify">
                The entire team worked on our human practice projects. Philipp Walch is a member of the education initiative &quot;Life-Science Lab&quot; and organized the essay competition for the high school students. Philipp Walch, Sophie Rabe and Tania Christiansen presented our project at various events to the interested public together with members of the high school iGEM Team from 2012. Joshua Sachs who volunteers as a fire fighter established the contact with the army, which gave us insight in the bio-weapons defense program (ABC). Ralf Beer and Konrad Herbst received valuable input from Peptide Specialties Laboratories GmbH. Philipp Walch and Nikos Ignatiadis visited TBM Edelmetall Recycling. Johanna Meichsner introduced us to Dorothea van Aaken, who organizes environmental workshops for primary students. Fanny Georgi organized the contact to Dr. Zawatzky from German Cancer Research Center (DKFZ). Dominik Niopek introduced us to the science author Olaf Fritsche and Lorenz Adlung, who also writes, got us in contact with Isabel Marleen PÃ¶tzsch. Both Dominik Niopek and Lorenz Adlung also helped with the &quot;not invented by nature&quot; collaboration.
</P></DIV><DIV><H2 style="color:#7137c8">BioQuant, University of Heidelberg </H2><H3 style="" text-align:middle=""> Prof. Dr. Thomas HÃ¶fer </H3><P style="text-align:justify">
                Prof. Dr. Thomas HÃ¶fer's group provided us a sample of <I>E. coli</I> BL21 (DE3) (pLys). 
                </P><H3 style="" text-align:middle=""> Thi Bach Nga Ly-Hartig </H3><P style="text-align:justify">
                Nga Ly-Hartig got us in touch with several research groups working with mass spectrometry and supported us with great cupcakes. 
                </P><H3 style="" text-align:middle=""> Prof. Dr. Robert B. Russel </H3><P style="text-align:justify">
                Prof. Russel supplied us with a vial of <I>E. coli</I> BL21 (DE3).
                </P><H3 style="" text-align:middle=""> Dr. Oliver Wichmann </H3><P style="text-align:justify">
                Dr. Wichmann supported us with his equipment especially by allowing us to use his lyophilizer.
</P></DIV><DIV><H2 style="color:#7137c8">Center for Molecular Biology Heidelberg (ZMBH), University of Heidelberg</H2><H3 style="" text-align:middle=""> Core Facility for Mass Spectronomy and Proteomics </H3><P style="text-align:justify">
                Dr. Thomas Ruppert and Dr. Bernd Hessling helped us with Mass Spectrometry measurements for the detection of our non-ribosomal peptides.
                </P><H3 style="" text-align:middle=""> Dr. Oliver Gruss </H3><P style="text-align:justify">
               The Gruss Lab kindly provided us the <I>E. coli</I> BL21 (DE3) (pLys) strain.
</P></DIV><DIV><H2 style="color:#7137c8"> Department of Biochemistry, Philipps-University Marburg</H2><H3 style="" text-align:middle=""> Prof.âDr. Mohamed Marahiel</H3><P style="text-align:justify">
                Prof. Dr. Marahiel kindly provided us with the <I>Brevibacillus parabrevis</I> ATCC 8185 strain and contributed helpful feedback on NRPS and our NRPSDesigner.
</P></DIV><DIV><H2 style="color:#7137c8"> Department of Biosystems Science and Engineering, ETH Zurich</H2><H3 style="" text-align:middle=""> Prof. Dr. Martin Fussenegger</H3><P style="text-align:justify">
                Prof. Fussenegger send us the expression plasmids for bpsA and svp, codon optimized for mammalian cells.
</P></DIV><DIV><H2 style="color:#7137c8">Department of Chemical and Biological Engineering, University of Buffalo</H2><H3 style="" text-align:middle=""> Prof. Blaine Pfeifer</H3><P style="text-align:justify">
                Lei Fang from the Pfeifer Lab kindly provided us with the <I>E. coli</I> BAP1 strain and the pLF03 and pKD46 plasmids.
</P></DIV><DIV><H2 style="color:#7137c8"> ESRC Centre for Economic and Social Aspects of Genomics (Cesagen), Lancaster University, Institute of Advanced Studies (UNU-IAS), United Nations University</H2><H3 style="" text-align:middle="">Dr. Paul Oldham</H3><P style="text-align:justify">
                For kindly supporting us with his expertise in information processing, interpretation of the social aspects of synthetic biology and for sharing his collection of synthetic biology terms with <A href="/Team:Heidelberg/HumanPractice/iGEM42">us</A>.
</P></DIV><DIV><H2 style="color:#7137c8">German Cancer Research Center (DKFZ), Heidelberg</H2><H3 style="" text-align:middle=""> Prof. Dr. Stephan Herzig </H3><P style="text-align:justify">
                Prof. Herzig and his group kindly offered the possibility to use their electroporator.
                </P><H3 style="" text-align:middle=""> Dr. Dirk Nettelbeck </H3><P style="text-align:justify">
               Dr. Nettelbeck, Ph.D.contributed with valuable input on homologous recombination.
                </P><H3 style="" text-align:middle=""> Prof. Dr. Rainer Zawatzky </H3><P style="text-align:justify">
               Prof. Zawatzky gave us lots of feedback on safety considerations and ethical concerns.
</P></DIV><DIV><H2 style="color:#7137c8">Harvard Wyss Institute for Biologically Inspired Engineering, Center for Life Sciences, Boston</H2><H3 style="" text-align:middle=""> Dr. Matthew Mattozzi</H3><P style="text-align:justify">
                Dr. Mattozzi supported us with valuable advice on genomic homologous recombination. Dr. Mattozzi also kindly provided the pET21c-pcc-acc-mmce plasmid. His PhD student Marika Ziesack, M.sc. kindly introduced us to Gibson Cloning.
</P></DIV><DIV><H2 style="color:#7137c8">Heidelberg Technology Park UniTT Start-Up Centre, University of Heidelberg</H2><H3 style="" text-align:middle=""> Dr. Raoul Haschke</H3><P style="text-align:justify">
                Dr. Haschke opened up new economical perspectives on the possible industrial scale NRP production and theidea to start our own company.
</P></DIV><DIV><H2 style="color:#7137c8">Institute of Chemical Engineering, University of Vienna</H2><H3 style="" text-align:middle=""> Dipl.-Ing.Paul Kroll</H3><P style="text-align:justify">
                Mr. Kroll contributed with great scientific input on modelling the industrial scale Delftibactin production in <I>E. coli</I>.
</P></DIV><DIV><H2 style="color:#7137c8">Institute of Organic Chemistry, University of Heidelberg</H2><H3 style="" text-align:middle=""> Mass Spectrometry Facility</H3><P style="text-align:justify">
                Prof. GroÃ kindly provided us with the possibility to use his HR-ESI MS and helped with data analysis.
</P></DIV><DIV><H2 style="color:#7137c8">Institute of Pharmacology and Molecular Biotechnology (IPMB), University of Heidelberg</H2><H3 style="" text-align:middle=""> Prof. Dr. Christian Klein and Lena Weigel </H3><P style="text-align:justify">
                Prof. Klein kindly allowed us to use the MICRO-TOF of his group. Special thanks to Lena Weigel who helped us with the operation.
                </P><H3 style="" text-align:middle=""> Dr. Holger SchÃ¤fer </H3><P style="text-align:justify">
                Dr. SchÃ¤fer supported us with scientific and moral support during the cloning of the Del constructs.
                </P><H3 style="" text-align:middle=""> Dr. Richard Wombacher </H3><P style="text-align:justify">
               Marcel Best from Dr. Wombacher's group saved us with the last minute supply of a rotary evaporator!
</P></DIV><DIV><H2 style="color:#7137c8">Peptide Specialties Laboratories GmbH, Heidelberg</H2><H3 style="" text-align:middle=""> Dr. Hans-Richard Rackwitz</H3><P style="text-align:justify">
                Dr. Rackwitz gave us fascinating insights in chemical peptide synthesis and valuable scientific input on our project.
</P></DIV><DIV><H2 style="color:#7137c8">School of Medicine, University of Maryland</H2><H3 style="" text-align:middle=""> Prof. Jacques Ravel, Ph.D.</H3><P style="text-align:justify">
               Professor Jacques Ravel continuously helped us with the automated domain prediction pipeline of the NRPSDesigner software, for example by providing us with seed alignments for the Hidden Markov Models of his own <A href="http://nrps.igs.umaryland.edu/nrps/"> PKS/NRPS analysis tool </A>. He also tested the NRPSDesigner software and gave us valuable feedback.
</P></DIV><DIV><H2 style="color:#7137c8">The Novo Nordisk Foundation Center for Biosustainability, Technical University of Denmark</H2><H3 style="" text-align:middle=""> Dr. Kai Zhuang </H3><P style="text-align:justify">
                Dr. Kai Zhuang gave us very helpful advice on integrating constraint based models of metabolism with bioreactor process descriptions.
</P></DIV><DIV><H2 style="color:#7137c8">University Medical Center Heidelberg, University of Heidelberg</H2><H3 style="" text-align:middle=""> Neonate Screening Facility</H3><P style="text-align:justify">
                Dr. Dirk KohlmÃ¼ller and Dr. Kathrin Schmidt supported us in amino acid level detection by mass spectrometry.
</P></DIV><DIV class="row"><DIV class="col-md-12"><DIV class="imageMap" aria-haspopup="true"><CENTER>Thanks to </CENTER></DIV></DIV></DIV><DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Heidelberg/AttributionsII">http://2013.igem.org/Team:Heidelberg/AttributionsII</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/AttributionsII" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/AttributionsII" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/AttributionsII&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/AttributionsII&amp;oldid=353616" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>