<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="row"><DIV id="nav" class="navigation"><DIV class="item team" data-offset-l="-130"><H2>Team</H2><UL><LI><A href="/Team:Heidelberg/Team">Team Members</A></LI><LI><A href="/Team:Heidelberg/AttributionsII">Attributions</A></LI><LI><A href="/Team:Heidelberg/Team/Sponsors">Sponsors</A></LI><LI><A href="/Team:Heidelberg/Team/Collaboration">Collaborations</A></LI><LI><A href="/Team:Heidelberg/Team/Gallery">Gallery</A></LI></UL></DIV><DIV class="item project" data-offset-l="-144"><H2>Project</H2><UL><LI><A href="/Team:Heidelberg/NRPS">Background</A></LI><LI><A href="/Team:Heidelberg/Project">Overview</A></LI><LI><A href="/Team:Heidelberg/Project/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Project/Indigoidine-Tag">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Project/Tag-Optimization">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Project/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Project/Achievements">Achievements</A></LI></UL></DIV><DIV class="item notebook" data-offset-l="-139"><H2>Notebook</H2><UL><LI><A href="/Team:Heidelberg/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Tyrocidine">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Indigoidine">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/NRPSDesigner">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/Notebook/Methods">Methods</A></LI><LI><A href="/Team:Heidelberg/Materials">Materials</A></LI></UL></DIV><DIV class="item parts" data-offset-l="-134"><H2>Parts</H2><UL><LI><A href="/Team:Heidelberg/Parts">Submitted Parts</A></LI><LI><A href="/Team:Heidelberg/Favorite_Parts">Favorite Parts</A></LI><LI><A href="/Team:Heidelberg/RFCs">Our RFCs</A></LI></UL></DIV><DIV class="item software" data-offset-l="-134"><H2>Software</H2><UL><LI><A href="/Team:Heidelberg/Project_Software">Project</A></LI><LI><A href="http://igem2013.bioquant.uni-heidelberg.de/NRPSDesigner/">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/iGEM42">iGEM42</A></LI><LI><A href="/Team:Heidelberg/OpenSource">Open Source</A></LI><LI><A href="/Team:Heidelberg/Project/AchievementsSoftware">Achievements</A></LI></UL></DIV><DIV class="item human" data-offset-l="-118"><H2>Human Practice</H2><UL><LI><A href="/Team:Heidelberg/HumanPractice">Overview</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BroadPublic">Broad Public</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Survey">Survey</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/YoungGeneration">Young Generation</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Artists">Arts and Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BeyondScience">Beyond Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Experts">Experts</A></LI></UL></DIV><DIV class="item safety" data-offset-l="-115"><H2>Safety</H2><UL><LI><A href="/Team:Heidelberg/Safety">Safety</A></LI></UL></DIV><DIV class="item model" data-offset-l="-144"><H2>Modeling</H2><UL><LI><A href="/Team:Heidelberg/Modelling/Gold_Recovery">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Modelling/Ind_Production">ind-Production</A></LI></UL></DIV><DIV class="item"><SPAN style="margin-left:6%">HOME</SPAN></DIV></DIV></DIV><DIV class="container"><DIV><H1><SPAN style="font-size:200%;color:#333333;">Our Project.</SPAN><SPAN class="text-muted" style="font-family:Arial, sans-serif; font-size:110%"> Foundational Advance in Peptide Synthesis.</SPAN></H1></DIV><DIV class="row"><DIV class="col-sm-6"><DIV class="jumbotron" style="padding:20px"><H2>Highlights</H2><P style="font-size:16px"><UL style="font-size:14px"><LI><B>Novel approach for creating customized peptides
</B></LI><LI><B>Blue pigment tag for <I>in-vivo</I> labeling of synthetic peptides
</B></LI><LI><B>Demonstration of NRPS modularity
</B></LI><LI><B>Engineering of entirely synthetic NRPS domains
</B></LI><LI><B>Software for <I>in-silico</I> design of custom NRPSs
</B></LI><LI><B>Sustainable and efficient gold recycling from electronic waste using Delftibactin
</B></LI></UL></P></DIV></DIV><DIV class="col-sm-6"><DIV class="jumbotron abstract" style="height:590px; padding:45px"><H2>Abstract</H2><P style="font-size:14px; text-align:justify">
Several secondary metabolites, such as commonly used antibiotics, pigments and detoxifying enzymes, are synthesized by <B>non-ribosomal peptide synthetases (NRPSs)</B>. These enzymes beautifully reflect one of the fundamental principles of synthetic biology, as they are <B>remarkably modular</B>. We will <B>assemble new NRPSs by combining individual domains and modules of different origin</B>, thus setting the basis for novel and <B>customized synthesis of non-ribosomal peptides</B>To make the use of NRPSs amenable to a wider community, we will devise a new software-tool, called <B>NRPS Designer</B>, which predicts the optimal modular composition of synthetic NRPSs for production of any desired peptide and outputs a cloning strategy based on Gibson assembly. As an application relevant to society, we will engineer <I>Escherichia coli</I> to <B>recycle gold from electronic waste in a cost-efficient and environmentally friendly</B> way through the heterologous expression of the NRPS pathway of <I>Delftia acidovorans</I> that naturally enables precipitation of gold ions from solution.
                        </P></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV id="ProjectText" class="col-sm-12"><DIV class="bronzebox"><DIV class="row"><DIV style="text-align:justify; margin:2%"> 
Peptides represent an astonishingly diverse class of molecules comprising highly versatile functions ranging from signaling, detoxifying and antibiotic function to enzymatic activity in nature. The invention of solid-phase synthesis of small peptides pioneered by Robert Bruce Merrifield in 1963 <SPAN class="citation">[1]</SPAN> sparked the hope to exploit and use those functions for our own purposes. This set the basis for an entirely novel research field termed “synthetic peptide chemistry” associated with big hopes to now being able to engineer novel antibiotics and develop peptides for solving environmental issues. 
                            Although solid-phase peptide synthesis could be successfully standardized and automated, an important limitation was never overcome: the method is too expensive to be applied for industry-scale production of synthetic peptides <SPAN class="citation">[2]</SPAN>. <B>Recombinant peptide synthesis</B>invented in the 1980s was advertised as an alternative to chemical peptide synthesis <SPAN class="citation">[3,4]</SPAN>, as it is easily scalable once the production is up and running (reviewed in  <SPAN class="citation">[5]</SPAN>). However, this approach is mostly restricted to peptides composed of <B>proteinogenic amino acids</B><SPAN class="citation">[6]</SPAN>, thus limiting the number of available amino acid building blocks and thereby narrowing the applicability of this approach.

                                </DIV></DIV></DIV><DIV class="bronzebox"><H2>Non-Ribosomal Peptide Synthesis</H2><DIV class="row"><DIV class="col-md-8" style="text-align:justify">
                            Our team developed a novel approach for creating customized peptides that overcomes the above mentioned limitations by engineering synthetic <B>Non-Ribosomal Peptide Synthetases (NRPSs)</B>. NRPSs are organized in modules recognizing one specific amino acid substrate and catalyzing the formation of a peptide bond between the amino acid substrate and the nascent peptide chain. Notably, non-ribosomal peptide synthesis does not require mRNA to direct the sequence of amino acid monomers incorporated into the growing peptide and is therefore <B>not limited</B> to proteinogenic amino acids (reviewed in <SPAN class="citation">[7]</SPAN>).
                                </DIV></DIV></DIV><DIV class="bronzebox"><H2>Functionality and Labeling of NRPSs</H2><DIV class="row"><DIV class="col-md-8" style="text-align:justify">
                    In order to show the applicability of custom NRPSs for synthetic peptide production, we created a set of synthetic NRPSs composed of varying modules derived from the <I>Bacillus brevis</I><A href="https://2013.igem.org/Team:Heidelberg/Project/Tyrocidine">Tyrocidine NRPS</A> pathway. We successfully produced a number of di- and tri-peptides composed of proteinogenic and non-proteinogenic amino acids in <I>E. coli</I>. In order to simplify the detection, purification and quality control of those peptides, <B>we invented an <I>in-vivo</I> labeling procedure</B> for non-ribosomal peptides by fusion of the corresponding, synthetic NRPSs to an <A href="https://2013.igem.org/Team:Heidelberg/Project/Indigoidine">engineered <B>blue pigment</B> module (IndC)</A>.
                    
                    Furthermore, we developed a procedure for improving the functionality of NRPS modules by shuffling single domains derived from different species and by engineering entirely synthetic domains. As proof of concept, we shuffled domains within the unimodular blue pigment NRPS IndC or introduced synthetic domains derived from consensus sequences across different species. To this end, we devised a novel cloning strategy termed HiCT (<A href="/Team:Heidelberg/RFCs#rfc99">BBF RFC 99</A>), which enables a cost-efficient and rapid assembly of synthetic NRPS module libraries. 
                    We were able to engineer a library of IndC modules exhibiting varying blue pigment production efficacies in combination with different NRPS activating PPTases. Remarkebly, a subset of these synthetic IndC variants showed a broader PPTase specificity compared to their natural counterpart. 
                    Furthermore, we identified the location of domain borders and optimized linker regions used for introduction of synthetic domains. By <A href="https://2013.igem.org/Team:Heidelberg/Modelling/Ind_Production">quantitative dynamic modelling</A>, we obtained a more detailed understanding on the synthesis efficiency of the different IndC variants in different contexts and investigated the influence of indigoidine production on bacterial growth.
                    
                     </DIV></DIV></DIV><DIV class="bronzebox"><H2>Standardized NRPS-Assembly</H2><DIV class="row"><DIV class="col-md-8" style="text-align:justify">
                        In order to enable the synthetic biology community to easily engineer custom NRPSs producing user-defined peptides, we developed a standardized framework for the production of non-ribosomal peptides. The framework consists of (1) the <A href="http://igem2013.bioquant.uni-heidelberg.de/NRPSDesigner/">NRPS Designer</A>, a software tool for the <I>in-silico</I> design of user-defined NRPSs, (2) a platform for standardized cloning and expression of NRPSs in different bacterial hosts and (3) a quality control procedure for the validation of NRP production. We documented detailed instructions for applying our framework in <A href="/Team:Heidelberg/RFCs">BBF RFC 100</A>.
                        </DIV></DIV></DIV><DIV class="bronzebox"><H2>Environmental Applications</H2><DIV class="row"><DIV class="col-md-8" style="text-align:justify">
To exemplify the infinite application possibilities of NRPSs, we developed an sustainable procedure for <A href="https://2013.igem.org/Team:Heidelberg/Project/Delftibactin">gold recycling from electronic waste</A> using the non-ribosomal peptide Delftibactin produced by <I>Delftia acidovorans</I>, which is known to precipitate elemental gold from gold-ion solutions <SPAN class="citation">[8]</SPAN>. Again we applied mathematical modelling in order to <A href="https://2013.igem.org/Team:Heidelberg/Modelling/Gold_Recovery">quantitatively investigate the feasibility</A> of this approach for an industrial-scale recycling of gold. Currently, we are working on transferring the complete delftibacin production pathway into <I>E. coli</I> in order to increase yield and lower costs of delftibactin production.

We believe, that our project demonstrates the power of non-ribosomal peptide synthesis and that our standardized framework will enable the synthetic biology community to use this power to address many of the challenges of our century.
</DIV></DIV></DIV></DIV><DIV class="col-sm-12 jumbotron references" style="margin-top:5%"><P>1. Merrifield RB (1963) Solid Phase Peptide Synthesis. I. The Synthesis of a Tetrapeptide. J. Am. Chem. Soc. 85 (14): 2149–2154</P><P>2. Marahiel MA, Stachelhaus T, Mootz HD (1997)Modular Peptide Synthetases Involved in Nonribosomal Peptide Synthesis. Chem Rev. 97(7):2651-2674</P><P>3. Löwenadler B, Jansson B, Paleus S, Holmgren E, Nilsson B, Moks T, Palm G, Josephson S, Philipson L, Uhlén M (1987) A gene fusion system for generating antibodies against short peptides. Gene 58(1):87-97</P><P>4. Nilsson B, Moks T, Jansson B, Abrahmsén L, Elmblad A, Holmgren E, Henrichson C, Jones TA, Uhlén M. (1987) A synthetic IgG-binding domain based on staphylococcal protein A. Protein Eng. 1(2):107-13</P><P>5. Bommarius B, Jenssen H, Elliott M, Kindrachuk J, Pasupuleti M, Gieren H, Jaeger KE, Hancock RE, Kalman D. (2010) Cost-effective expression and purification of antimicrobial and host defense peptides in Escherichia coli. 
Peptides 31(11):1957-65</P><P>6. Lu Y, Freeland S. (2006) On the evolution of the standard amino-acid alphabet. Genome Biol.7(1):102
</P><P>7. Andersson L, Blomberg L, Flegel M, Lepsa L, Nilsson B, Verlander M (2000) Large-scale synthesis of peptides. Peptide Science 55(3):227–250</P><P>8. Johnston CW, Wyatt MA, Li X, Ibrahim A, Shuster J, et al. (2013) Gold biomineralization by a metallophore from a gold-associated microbe. Nature chemical biology 9: 241–243.</P></DIV><DIV class="row"><DIV class="col-md-12"><DIV class="imageMap" aria-haspopup="true"><CENTER>Thanks to </CENTER></DIV></DIV></DIV><DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Heidelberg/Project">http://2013.igem.org/Team:Heidelberg/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Project&amp;oldid=361595" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>