<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Templates_Materials_list"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Templates/Materials_list">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Templates/Materials_list&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Templates/Materials_list&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Templates/Materials_list&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Templates/Materials_list" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Templates/Materials list</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Kits"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Kits</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Marker"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Marker</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Enzymes"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Enzymes</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Restriction_Enzymes"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Restriction Enzymes</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Bacterial_Strains"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Bacterial Strains</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Antibiotics_and_Media_Supplements"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Antibiotics and Media Supplements</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Media"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Media</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Buffers"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Buffers</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Other_Chemicals"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Other Chemicals</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Electrophoresis"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Electrophoresis</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Miscellaneous_Primers"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Miscellaneous Primers</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Primers_and_Oligos"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Primers and Oligos</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Delftibactin"><SPAN class="tocnumber">12.1</SPAN><SPAN class="toctext">Delftibactin</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Module_Shuffling"><SPAN class="tocnumber">12.2</SPAN><SPAN class="toctext">Module Shuffling</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Linker_Variation"><SPAN class="tocnumber">12.3</SPAN><SPAN class="toctext">Linker Variation</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Domain_Shuffling_and_PPTases"><SPAN class="tocnumber">12.4</SPAN><SPAN class="toctext">Domain Shuffling and PPTases</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-17"><A href="#Plasmids"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">Plasmids</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Delftibactin_2"><SPAN class="tocnumber">13.1</SPAN><SPAN class="toctext">Delftibactin</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Module_Shuffling_2"><SPAN class="tocnumber">13.2</SPAN><SPAN class="toctext">Module Shuffling</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Linker_variation_2"><SPAN class="tocnumber">13.3</SPAN><SPAN class="toctext">Linker variation</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Dinner_plasmids"><SPAN class="tocnumber">13.4</SPAN><SPAN class="toctext">Dinner plasmids</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#Domain_Shuffling_and_PPTases_2"><SPAN class="tocnumber">13.5</SPAN><SPAN class="toctext">Domain Shuffling and PPTases</SPAN></A></LI><LI class="toclevel-1 tocsection-23"><A href="#Used_T_and_TE_Domains"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">Used T and TE Domains</SPAN></A></LI><LI class="toclevel-1 tocsection-24"><A href="#Instruments"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">Instruments</SPAN></A></LI><LI class="toclevel-1 tocsection-25"><A href="#Mass_Spectrometry_and_Neonate_Screening"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">Mass Spectrometry and Neonate Screening</SPAN></A></LI><LI class="toclevel-1 tocsection-26"><A href="#Lab_Materials"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">Lab Materials</SPAN></A></LI><LI class="toclevel-1 tocsection-27"><A href="#Further_Recipies_and_Stocks"><SPAN class="tocnumber">18</SPAN><SPAN class="toctext">Further Recipies and Stocks</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#Acidovorans_Complex_Medium"><SPAN class="tocnumber">18.1</SPAN><SPAN class="toctext">Acidovorans Complex Medium</SPAN></A></LI><LI class="toclevel-3 tocsection-29"><A href="#for_1_L"><SPAN class="tocnumber">18.1.1</SPAN><SPAN class="toctext">for 1 L</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#Ampicillin_Stock_Solution"><SPAN class="tocnumber">18.2</SPAN><SPAN class="toctext">Ampicillin Stock Solution</SPAN></A></LI><LI class="toclevel-3 tocsection-31"><A href="#Efficiency_Test"><SPAN class="tocnumber">18.2.1</SPAN><SPAN class="toctext">Efficiency Test</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#Bacitracin_Stock_Solution"><SPAN class="tocnumber">18.3</SPAN><SPAN class="toctext">Bacitracin Stock Solution</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#Brevibacillus_Parabrevis_Glycerol_Stock"><SPAN class="tocnumber">18.4</SPAN><SPAN class="toctext">Brevibacillus Parabrevis Glycerol Stock</SPAN></A></LI><LI class="toclevel-2 tocsection-34"><A href="#Chloramphenicol_Stock_Solution"><SPAN class="tocnumber">18.5</SPAN><SPAN class="toctext">Chloramphenicol Stock Solution</SPAN></A></LI><LI class="toclevel-2 tocsection-35"><A href="#D._Acidovorans_Glycerol_Stock"><SPAN class="tocnumber">18.6</SPAN><SPAN class="toctext">D. Acidovorans Glycerol Stock</SPAN></A></LI><LI class="toclevel-2 tocsection-36"><A href="#E.Coli_BAP_1_Glycerol_Stock"><SPAN class="tocnumber">18.7</SPAN><SPAN class="toctext">E.Coli BAP 1 Glycerol Stock</SPAN></A></LI><LI class="toclevel-2 tocsection-37"><A href="#E.Coli_BAP1.2C_Competent"><SPAN class="tocnumber">18.8</SPAN><SPAN class="toctext">E.Coli BAP1, Competent</SPAN></A></LI><LI class="toclevel-2 tocsection-38"><A href="#E._coli_BAP1-pKD46_Glycerol_Stock"><SPAN class="tocnumber">18.9</SPAN><SPAN class="toctext">E. coli  BAP1-pKD46 Glycerol Stock</SPAN></A></LI><LI class="toclevel-2 tocsection-39"><A href="#E.Coli_BAP1-pLF03_Glycerol_Stock"><SPAN class="tocnumber">18.10</SPAN><SPAN class="toctext">E.Coli BAP1-pLF03 Glycerol Stock</SPAN></A></LI><LI class="toclevel-2 tocsection-40"><A href="#E._Coli_DH5.CE.B1-pCP20_Glycerol_Stock"><SPAN class="tocnumber">18.11</SPAN><SPAN class="toctext">E. Coli DH5α-pCP20 Glycerol Stock</SPAN></A></LI><LI class="toclevel-2 tocsection-41"><A href="#Heat-Shock_Competent_E._Coli_TOP10"><SPAN class="tocnumber">18.12</SPAN><SPAN class="toctext">Heat-Shock Competent E. Coli TOP10</SPAN></A></LI><LI class="toclevel-2 tocsection-42"><A href="#E._Coli_TOP10-BBa_I746200.2FpSB1AK3_Glycerol_Stock"><SPAN class="tocnumber">18.13</SPAN><SPAN class="toctext">E. Coli TOP10-BBa I746200/pSB1AK3 Glycerol Stock</SPAN></A></LI><LI class="toclevel-2 tocsection-43"><A href="#E._Coli_TOP10-BBa_J04450.2FpSB3C5_Glycerol_Stock"><SPAN class="tocnumber">18.14</SPAN><SPAN class="toctext">E. Coli TOP10-BBa J04450/pSB3C5 Glycerol Stock</SPAN></A></LI><LI class="toclevel-2 tocsection-44"><A href="#E._Coli_TOP10-pIK1_Glycerol_Stock"><SPAN class="tocnumber">18.15</SPAN><SPAN class="toctext">E. Coli TOP10-pIK1 Glycerol Stock</SPAN></A></LI><LI class="toclevel-2 tocsection-45"><A href="#E._Coli_TOP10-pKD46_Glycerol_Stock"><SPAN class="tocnumber">18.16</SPAN><SPAN class="toctext">E. Coli TOP10-pKD46 Glycerol Stock</SPAN></A></LI><LI class="toclevel-2 tocsection-46"><A href="#E._Coli_TOP10-pMM65_Glycerol_Stock"><SPAN class="tocnumber">18.17</SPAN><SPAN class="toctext">E. Coli TOP10-pMM65 Glycerol Stock</SPAN></A></LI><LI class="toclevel-2 tocsection-47"><A href="#IPTG_Stock_Solution"><SPAN class="tocnumber">18.18</SPAN><SPAN class="toctext">IPTG Stock Solution</SPAN></A></LI><LI class="toclevel-2 tocsection-48"><A href="#Kanamycin_Stock_Solution"><SPAN class="tocnumber">18.19</SPAN><SPAN class="toctext">Kanamycin Stock Solution</SPAN></A></LI><LI class="toclevel-2 tocsection-49"><A href="#M9_Medium"><SPAN class="tocnumber">18.20</SPAN><SPAN class="toctext">M9 Medium</SPAN></A></LI><LI class="toclevel-2 tocsection-50"><A href="#Reactivation_Medium"><SPAN class="tocnumber">18.21</SPAN><SPAN class="toctext">Reactivation Medium</SPAN></A></LI><LI class="toclevel-3 tocsection-51"><A href="#for_1L"><SPAN class="tocnumber">18.21.1</SPAN><SPAN class="toctext">for 1L</SPAN></A></LI><LI class="toclevel-2 tocsection-52"><A href="#Solution_for_MS_Preparation"><SPAN class="tocnumber">18.22</SPAN><SPAN class="toctext">Solution for MS Preparation</SPAN></A></LI><LI class="toclevel-3 tocsection-53"><A href="#Solution_1_.28for_MS_Sample_Preparation_Neo_Gen_Stable_Isotope_Standard_Kit_A_and_B_.28NSK-AB.2C_EurIsotop.29"><SPAN class="tocnumber">18.22.1</SPAN><SPAN class="toctext">Solution 1 (for MS Sample Preparation Neo Gen Stable Isotope Standard Kit A and B (NSK-AB, EurIsotop)</SPAN></A></LI><LI class="toclevel-3 tocsection-54"><A href="#Solution_2_.28for_MS_sample_preparation.29:_butanolic_hydrochloric_acid_.283_N.29.2C_water-free"><SPAN class="tocnumber">18.22.2</SPAN><SPAN class="toctext">Solution 2 (for MS sample preparation): butanolic hydrochloric acid (3 N), water-free</SPAN></A></LI><LI class="toclevel-3 tocsection-55"><A href="#Solution_3_.28for_MS_sample_preparation.29"><SPAN class="tocnumber">18.22.3</SPAN><SPAN class="toctext">Solution 3 (for MS sample preparation)</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Kits">Kits</SPAN></H1><TABLE class="wikitable sortable"><TBODY><TR><TH> Kit </TH><TH> Supplier </TH><TH> Catalog Number
</TH></TR><TR><TD> MinElute® PCR Purification Kit (250) </TD><TD> QIAGEN </TD><TD> 28006
</TD></TR><TR><TD> Plasmid Plus Maxi Kit (25) </TD><TD> QIAGEN </TD><TD> 12963
</TD></TR><TR><TD> Plasmid Plus Midi Kit (25) </TD><TD> QIAGEN </TD><TD> 12943
</TD></TR><TR><TD> QIAEX II® Gel Extraction Kit (500) </TD><TD> QIAGEN </TD><TD> 20051
</TD></TR><TR><TD> QIAGEN® Plasmid Plus Midi Kit (100) </TD><TD> QIAGEN </TD><TD> 12945
</TD></TR><TR><TD> QIAquick® Gel Extraction Kit (250) </TD><TD> QIAGEN </TD><TD> 28706
</TD></TR><TR><TD> QIAquick® Nucleotid Removal Kit (250) </TD><TD> QIAGEN </TD><TD> 28306
</TD></TR><TR><TD> QIAquick® PCR Purification Kit (250) </TD><TD> QIAGEN </TD><TD> 28106
</TD></TR><TR><TD> QIAprep® Spin Miniprep Kit (250) </TD><TD> QIAGEN </TD><TD> 27106
</TD></TR><TR><TD> QIAprep® Spin Miniprep Columns </TD><TD> QIAGEN </TD><TD> 27115
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Marker">Marker</SPAN></H1><TABLE class="wikitable sortable"><TBODY><TR><TH> Marker </TH><TH> Supplier </TH><TH> Catalog Number
</TH></TR><TR><TD> Quick-Load® 2-Log DNA Ladder (0.1-10.0 kb)  </TD><TD> New England BioLabs </TD><TD> N3200S
</TD></TR><TR><TD> Quick-Load® 1 kb DNA Ladder  </TD><TD> New England BioLabs </TD><TD> N0468S
</TD></TR><TR><TD> 50 bp DNA Ladder </TD><TD> New England BioLabs </TD><TD> N3236S
</TD></TR><TR><TD> Gel loading solution </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> G2526-5ML
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Enzymes">Enzymes</SPAN></H1><TABLE class="wikitable sortable"><TBODY><TR><TH> Enzyme </TH><TH> Supplier </TH><TH> Catalog Number
</TH></TR><TR><TD> DreamTaq Green PCR Master Mix (2X) </TD><TD> Thermo Fisher Scientific Biosciences GmbH </TD><TD> K1081
</TD></TR><TR><TD> DreamTaq PCR MM </TD><TD> Fermentas Life Sciences </TD><TD> K1071
</TD></TR><TR><TD> Gibson Assembly® Master Mix </TD><TD> New England Biolabs </TD><TD> E2611 S
</TD></TR><TR><TD> Lysozyme from Chicken Egg White </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> L4919-500MG
</TD></TR><TR><TD> Phusion® Flash High-Fidelity PCR Master Mix </TD><TD> Biozym Scientific GmbH </TD><TD> F-548L
</TD></TR><TR><TD> Phusion® High-Fidelity PCR Master Mix </TD><TD> New England Biolabs </TD><TD> M0531 L
</TD></TR><TR><TD> T4 DNA Ligase </TD><TD> New England Biolabs GmbH </TD><TD> M0202 S
</TD></TR><TR><TD> 2x PCR Master mix Solution (iTaq) </TD><TD> HISS DIAGNOSTICS GmbH </TD><TD> 25028
</TD></TR><TR><TD> Pronase from <I>Streptomyces griseus</I></TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> P6911-100MG
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Restriction_Enzymes">Restriction Enzymes</SPAN></H1><TABLE class="wikitable sortable"><TBODY><TR><TH> Enzyme </TH><TH> Supplier </TH><TH> Catalog Number
</TH></TR><TR><TD> BamI </TD><TD> New England Biolabs </TD><TD> R3136 S
</TD></TR><TR><TD> BgIII </TD><TD> New England Biolabs </TD><TD> R0144 S
</TD></TR><TR><TD> DpnI </TD><TD> New England Biolabs </TD><TD> R0176 S
</TD></TR><TR><TD> EcoRI  </TD><TD> New England BioLabs </TD><TD> R0101S
</TD></TR><TR><TD> EcoRI-HF </TD><TD> New England Biolabs </TD><TD> R3101
</TD></TR><TR><TD> HindIII-HF </TD><TD> New England Biolabs </TD><TD> R3104 S
</TD></TR><TR><TD> KpnI-HF </TD><TD> New England Biolabs </TD><TD> R3142 S
</TD></TR><TR><TD> MfeI-HF </TD><TD> New England Biolabs </TD><TD> R3589 S
</TD></TR><TR><TD> NheI-HF </TD><TD> New England BioLabs </TD><TD> R3131 S
</TD></TR><TR><TD> NotI-HF  </TD><TD> New England BioLabs </TD><TD> R3189 S
</TD></TR><TR><TD> PacI </TD><TD> New England Biolabs </TD><TD> R0547 S
</TD></TR><TR><TD> PstI-HF </TD><TD> New England Biolabs </TD><TD> R3140 S
</TD></TR><TR><TD> PvuI-HF </TD><TD> New England BioLabs </TD><TD> R3150S
</TD></TR><TR><TD> SalI-HF </TD><TD> New England Biolabs </TD><TD> R3138 S
</TD></TR><TR><TD> SpeI-HF </TD><TD> New England BioLabs </TD><TD> R3133 S
</TD></TR><TR><TD> XbaI  </TD><TD> New England BioLabs </TD><TD> R0145 S
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Bacterial_Strains">Bacterial Strains</SPAN></H1><TABLE class="wikitable sortable"><TBODY><TR><TH> Strain </TH><TH> Supplier </TH><TH> Catalog Number
</TH></TR><TR><TD><I>Delftia acidovorans</I> DSM-39 </TD><TD> DSMZ </TD><TD> DSM 50251
</TD></TR><TR><TD><I>Delftia acidovorans</I> SPH-1 </TD><TD> DSMZ </TD><TD> DSM 14801
</TD></TR><TR><TD><I>E. coli</I> DH10ß </TD><TD> New England Biolabs </TD><TD> C3019
</TD></TR><TR><TD><I>E. coli</I> Top10 </TD><TD> invitrogen </TD><TD> C404010
</TD></TR><TR><TD><I>Photorhabdus laumondii luminescens</I></TD><TD> DSMZ </TD><TD> DSM 15139
</TD></TR><TR><TD><I>Streptomyces lavendulae lavendulae</I></TD><TD> DSMZ </TD><TD> DSM 40708
</TD></TR><TR><TD><I>Streptomyces mobaraensis</I></TD><TD> DSMZ </TD><TD> DSM 40903
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Antibiotics_and_Media_Supplements">Antibiotics and Media Supplements</SPAN></H1><TABLE class="wikitable sortable"><TBODY><TR><TH> Antibiotic </TH><TH> Supplier </TH><TH> Catalog Number </TH><TH> Concentration stock solution </TH><TH> Dilution </TH><TH> Solvent
</TH></TR><TR><TD> Ampicillin Anhydrous Crystalline </TD><TD> Sigma-Aldrich Chemie GmbH  </TD><TD> A9393-5G </TD><TD> 100 mg/ml </TD><TD> 1:1000 </TD><TD> H<SUB>2</SUB>O
</TD></TR><TR><TD> Ampicillin Sodium Crystalline </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> A9518-5G </TD><TD> 100 mg/ml </TD><TD>1:1000 </TD><TD> H<SUB>2</SUB>O
</TD></TR><TR><TD> Chloramphenicol Crystalline </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> C0378-5G </TD><TD> 30 mg/ml </TD><TD> 1:3000 </TD><TD> Ethanol
</TD></TR><TR><TD> Kanamycinsulfat <I>Mixture of Componenta</I></TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> 60615-5G </TD><TD> 50 mg/ml </TD><TD> 1:1000 </TD><TD> H<SUB>2</SUB>O
</TD></TR><TR><TD> Tetracycline </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> T7660 </TD><TD> 10 mg/ml </TD><TD> 1:1000 </TD><TD> Ethanol
</TD></TR><TR><TD> Propionic Acid Sodium Insect Cell*Culture </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> P5436-100G </TD><TD>  100mM </TD><TD> 10mM </TD><TD> H<SUB>2</SUB>O
</TD></TR><TR><TD> Bacitracin </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> B0125-50KU </TD><TD> - </TD><TD> -
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Media">Media</SPAN></H1><TABLE class="wikitable sortable"><TBODY><TR><TH> Medium </TH><TH> Supplier </TH><TH> Catalog Number
</TH></TR><TR><TD> SOC Outgrowth Medium </TD><TD> New England Biolabs GmbH </TD><TD> B9020 S
</TD></TR><TR><TD> LB BROTH BASE </TD><TD> Th. Geyer GmbH &amp; Co KG </TD><TD> SA/L3022/001000
</TD></TR><TR><TD> LB Broth Powder </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> L3022-1KG
</TD></TR><TR><TD> M9 Minimal Salts </TD><TD> SERVA </TD><TD> 48505.01
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Buffers">Buffers</SPAN></H1><TABLE class="wikitable sortable"><TBODY><TR><TH>Buffer </TH><TH> Supplier </TH><TH> Catalog Number
</TH></TR><TR><TD> NEBuffer Pack #4 (green) </TD><TD> New England Biolabs GmbH </TD><TD> B7004 S
</TD></TR><TR><TD> NEBuffer Pack #1 (yellow) </TD><TD> New England Biolabs GmbH </TD><TD> B7001 S
</TD></TR><TR><TD> NEBuffer Pack for T4 DNA Ligase </TD><TD> New England Biolabs GmbH </TD><TD> B0202 S
</TD></TR><TR><TD> NEBuffer Pack #2 (blue) </TD><TD> New England Biolabs GmbH </TD><TD> B7002 S
</TD></TR><TR><TD> NEBuffer Pack #3 (red) </TD><TD> New England Biolabs GmbH </TD><TD> B7003 S
</TD></TR><TR><TD> TAE - Buffer (50X) for Molecular Biology </TD><TD> VWR International GmbH </TD><TD> A4686.1000
</TD></TR><TR><TD> Gel Loading Buffer </TD><TD> Sigma-Aldrich </TD><TD> G2526-5ML
</TD></TR><TR><TD> Tris Acetate-EDTA Buffer </TD><TD> Sigma-Aldrich </TD><TD> T9650-1L
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Other_Chemicals">Other Chemicals</SPAN></H1><TABLE class="wikitable sortable"><TBODY><TR><TH> Chemical </TH><TH> Supplier </TH><TH> Catalog Number
</TH></TR><TR><TD> Isopropyl B-D-Thiogalactopyranoside 1 piece </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> I5502-1G
</TD></TR><TR><TD> Dimethyl Sulfoxide PCR Reagent </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> D9170-1VL
</TD></TR><TR><TD> Glycerol Sigma Grade </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> G9012-100ML
</TD></TR><TR><TD> 5-Bromo-4-Chloro-3-Indolyl B-D-*Galactop</TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> B4252-100MG
</TD></TR><TR><TD> Bacteriological Agar </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> A5306-250G
</TD></TR><TR><TD> L-Plus-Arabinose Crystalline </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> A3256-25G
</TD></TR><TR><TD> Calciumchlorid Dihydrat </TD><TD> Th. Geyer GmbH &amp; Co KG </TD><TD> SA/00223506/000500
</TD></TR><TR><TD> Malt Extract from Starch Digestion </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> M0383-100G
</TD></TR><TR><TD> D(+)-Saccharose, ACS, for Micro-Biology </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> 84100-1KG
</TD></TR><TR><TD> Dimethyl Sulfoxide Plant Cell Culture*TE </TD><TD> D4540-100ML
</TD></TR><TR><TD> Sodium Hydroxide Anhydrous Pellets </TD><TD> Th. Geyer GmbH &amp; Co.KG </TD><TD> SA/S5881/000500
</TD></TR><TR><TD> TRIZMA(R) Hydrochloride PH 3.5-5.0 </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> T6666-50G
</TD></TR><TR><TD> L-Glutamine 200 MM Sterile </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> G7513-20ML
</TD></TR><TR><TD> Ethanol 96% Denatured </TD><TD> Carl Roth GmbH &amp; Co.KG </TD><TD> T171.3
</TD></TR><TR><TD> Propanol-2 </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD>  59309-1L
</TD></TR><TR><TD> Natriumdodecylsulfat,SDS,99%, Ultra Pure </TD><TD> 13904
</TD></TR><TR><TD> Gold(III)-Chloride </TD><TD> Carl Roth GmbH &amp; Co.KG </TD><TD> 5624.1
</TD></TR><TR><TD> Pro-Leu </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> P1130-1G
</TD></TR><TR><TD> Nitric Acid 65% p.a. Iso </TD><TD> Carl Roth GmbH &amp; Co.KG </TD><TD> X943.1
</TD></TR><TR><TD> Mops, Sodium </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> M9024-25G
</TD></TR><TR><TD> L-Glutamine </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> G7513-100ML
</TD></TR><TR><TD> Chelating Resin </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> C7901-50G
</TD></TR><TR><TD> Potassium Hydroxide in Platellets</TD><TD> 6751.3
</TD></TR><TR><TD> Hydrochloric Acid 37% </TD><TD>  4625.1
</TD></TR><TR><TD> Pyruvic Acid </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> 107360-25G
</TD></TR><TR><TD> Fmoc-Orn(BOC)-OH 96.0 % </TD><TD> 47560-5G-F
</TD></TR><TR><TD> Glycerol &gt;99.5% </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> G9012-1L
</TD></TR><TR><TD> Water Molecular Biology Reagent </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> W4502-1L
</TD></TR><TR><TD> Acetonitrile </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> 34967-1L
</TD></TR><TR><TD> Ascorbic Acid 99% </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> A92902-100G
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Electrophoresis">Electrophoresis</SPAN></H1><TABLE class="wikitable sortable"><TBODY><TR><TH> Reagent </TH><TH> Supplier </TH><TH> Catalog Number </TH><TH> Concentration </TH><TH> Solvent
</TH></TR><TR><TD> Agarose Molecular Biology Reagent </TD><TD> Th. Geyer GmbH &amp; Co KG </TD><TD> SA/A9539/000050 </TD><TD> 0.5% </TD><TD> H<SUB>2</SUB>O
</TD></TR><TR><TD> Agarose for Routine Use </TD><TD> Sigma-Aldrich Chemie GmbH </TD><TD> A9539-100G </TD><TD> - </TD><TD> -
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Miscellaneous_Primers">Miscellaneous Primers</SPAN></H1><TABLE class="wikitable sortable"><TBODY><TR><TH> Identifier </TH><TH> Order date </TH><TH> Note </TH><TH> Sequence
</TH></TR><TR><TD> VF2 </TD><TD> - </TD><TD> for screening in standard BB backbone- binds on the Backbone before Insert  </TD><TD> TGCCACCTGACGTCTAAGAA
</TD></TR><TR><TD> VR </TD><TD> - </TD><TD> for screening in standard BB backbone- binds on Backbone behind Insert </TD><TD> ATTACCGCCTTTGAGTGAGC
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Primers_and_Oligos">Primers and Oligos</SPAN></H1><H2><SPAN class="mw-headline" id="Delftibactin">Delftibactin</SPAN></H2><TABLE class="wikitable sortable" style="width:50%;"><TBODY><TR><TH> Identifier </TH><TH> Order date </TH><TH>Note </TH><TH> Sequence
</TH></TR><TR><TD> DN01:delH_f1_PacI_fw </TD><TD> - </TD><TD> fw Primer for DelH-fragment1 with RBS and PacI-restriction site </TD><TD> TTTT TTAATTAA    TCACACAGGAAAGTACTAGATGGACCGTGGCCGCCTGC    GCCAAATCG
</TD></TR><TR><TD> DN02:delH_f1_SalI_rev </TD><TD> - </TD><TD> - </TD><TD> TTTT GTCGACCAACACCTGTGCCTGC
</TD></TR><TR><TD> DN03:delH_f2_SalI_fw </TD><TD> - </TD><TD> - </TD><TD> TTTT GTCGACTGGATGGAGCCTGGTGAAAG
</TD></TR><TR><TD> DN04:delH_f2_KpnI_rev </TD><TD> - </TD><TD> - </TD><TD> TTTT GGTACC   TCAGTCCAGCGCGTACTCCAG
</TD></TR><TR><TD> DN05:AraCbb_KpnI_fw </TD><TD> - </TD><TD> amplifying the Backbone for DelH (pSB6A1-AraC-lacZ) </TD><TD> TTTT  GGTACC  AAAGAGGAGAAATACTAGATGACCATG
</TD></TR><TR><TD> DN06:AraCbb_PacI_rev2 </TD><TD>  15-05-2013  </TD><TD> amplifying the Backbone for DelH (pSB6A1-AraC-lacZ) </TD><TD> TTTT   TTAATTAA   GCTAGCCCAAAAAAACGGTATG
</TD></TR><TR><TD> DN07:Screen_delH_rev</TD><TD> 15-05-2013 </TD><TD> for screening if DelH is present - binds on the very beginning of DelH </TD><TD> CTTTCCTCGAACACCGTGCGCAG
</TD></TR><TR><TD> DN08:DelH_EcoRI_rev </TD><TD> - </TD><TD> rev_Primer for DelH Fragment f1a </TD><TD> CTCGTCGCCATGGACCAGGCAG
</TD></TR><TR><TD> DN09:DelH_f1_fw_long </TD><TD> 2013-06-11 </TD><TD> for amplifying DelH-1a from the genome: doesn't work</TD><TD> ATGGACCGTGGCCGCCTGCGCCAAATCG
</TD></TR><TR><TD> DN10:DelH_f1_fw_short </TD><TD> 2013-06-11 </TD><TD> for amplifying DelH-1a from the genome: doesn't work </TD><TD> ATGGACCGTGGCCGCCTGC
</TD></TR><TR><TD> DN11:DelH_f1_fw_short2</TD><TD> 2013-06-11 </TD><TD> for amplifying DelH-1a from the genome: works!!!! </TD><TD> GCCGCCTGCGCCAAATCG
</TD></TR><TR><TD> DN12:DelH_f1_PacI_fw_short</TD><TD> 2013-06-11 </TD><TD> for amplifying DelH-1a from the genome: doesn't work </TD><TD> TTTTTTAATTAATCACACAGGAAAGTACTAGATGGACCGTGGCCGCCTGC
</TD></TR><TR><TD> DN13:Screen_DelH_fw</TD><TD> 15-05-2013 </TD><TD> PCR Screening for presence of DelH insert </TD><TD> GTAAACCCACTGGTGATACCATTC
</TD></TR><TR><TD> FS_01: pSB4K5_DelA_rv</TD><TD>20-13-06-28</TD><TD>Amplification of pSB4K5 from the iGEM Distribution Gibson Primer with overhang to DelA introducing the RBS BBa_B0035</TD><TD> TCGCGGCGATCCGGTACTGCGCCTCTGTTGAACATCTGATATTCTCCTCTTTAATCG ACAGATTGTGTGAAATTGTTATCCGCTCAC
</TD></TR><TR><TD>FS_02: DelAG_1_fw</TD><TD>2013-06-28 </TD><TD> Amplification of DelAG from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> TTCAACAGAGGCGCAGTACCGGATC
</TD></TR><TR><TD>FS_03: DelAG_1_rv</TD><TD>2013-06-28 </TD><TD>Amplification of DelAG from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> GTCGGAGACGATGTGGTGCATCAC
</TD></TR><TR><TD>FS_04: DelAG_2_fw</TD><TD>2013-06-28 </TD><TD>Amplification of DelAG from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> CTGCAGGCCAATGAGCACATCCTG
</TD></TR><TR><TD>FS_05: DelAG_2_rv</TD><TD>2013-06-28 </TD><TD>Amplification of DelAG from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> CACAGGTGGTAGATGGCGTC
</TD></TR><TR><TD>FS_06: DelAG_3_fwG</TD><TD>2013-06-28 </TD><TD>Amplification of DelAG from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> ATTGCGAGGACTTGCTCGATG
</TD></TR><TR><TD>FS_07: DelAG_3_rv</TD><TD>2013-06-28 </TD><TD>Amplification of DelAG from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> TTTGCTGCAGCGCCAGCACATCGAG
</TD></TR><TR><TD>FS_08: DelAG_4_fw</TD><TD>2013-06-28 </TD><TD>Amplification of DelAG from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> GTACGGCCTATCACATCAGCG
</TD></TR><TR><TD>FS_09: DelAG_4_rv</TD><TD>2013-06-28 </TD><TD>Amplification of DelAG from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> GAAGCTCAGCAGGTTGGGCGAGACG
</TD></TR><TR><TD>FS_10: DelAG_5_fw</TD><TD>2013-06-28 </TD><TD>Amplification of DelAG from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> GAATTTTGTTCCACCACCTGCTG
</TD></TR><TR><TD>FS_11: DelAG_5_rv</TD><TD>2013-06-28 </TD><TD>Amplification of DelAG from <I>Delftia acidovorans</I> genome Gibson Primer with overhang to DelOP</TD><TD> CTTGAGCAGGCGCAGTACCTCGGAGGG CGGTCGGCTGGCGTTTTCCATGATTCAGGTTTCCTGTGTGAAGCTCATCTCAGATA TCTCCCAGAGTTTCGAGAAAG
</TD></TR><TR><TD>FS_11: DelAG_5_short_rv</TD><TD>2013-05-07 </TD><TD>Amplification of DelAG from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> TCAGATATCTCCCAGAGTTTCGAGAAAG
</TD></TR><TR><TD>FS_12: DelOP_fw</TD><TD>2013-06-28 </TD><TD>Amplification of DelOP from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> GAATCATGGAAAACGCCAGCCGAC
</TD></TR><TR><TD>FS_13: DelOP_rv</TD><TD>2013-06-28 </TD><TD>Amplification of DelOP from <I>Delftia acidovorans</I> genome Gibson Primer with overhang to DelL</TD><TD> CAATGTTGGAGGGGCCGAAGCCGATGCCGATCAGCGGGTGGGTTTGCATGGAAGGTC CTTTCATTGGGTCGATGCGTCCAGTGT CACACCGTGGTGTCTGCAGGCG
</TD></TR><TR><TD>FS_13: DelOP_short_rv</TD><TD>2013-05-07 </TD><TD>Amplification of DelOP from <I>Delftia acidovorans</I> genome Gibson Primer with overhang to DelL</TD><TD>TCACACCGTGGTGTCTGCAGGCG
</TD></TR><TR><TD>FS_14: DelL_fw</TD><TD>2013-06-28 </TD><TD>Amplification of DelL from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> CAAACCCACCCGCTGATCGGCATC
</TD></TR><TR><TD>FS_15: DelL_mRFP_pSB4K5_rv</TD><TD>2013-06-28 </TD><TD>Amplification of DelL Gibson Primer with overhang to BBa_J04450</TD><TD> GAAACGCATGAACTCTTTGATAACGTCT TCGGAGGAAGCCATCTAGTATTTCTCCT CTTTCTCTAGTATCAGTCCTGCAGCG CCAGCTGTTCTGTG
</TD></TR><TR><TD>FS_15: DelL_mRFP_pSB4K5__short_rv</TD><TD>2013-05-07 </TD><TD>Amplification of DelL Gibson Primer with overhang to BBa_J04450</TD><TD>TCAGTCCTGCAGCGCCAGCTGTTCTGTG
</TD></TR><TR><TD>FS_16: mRFP_pSB4K5_fw (Del)</TD><TD>2013-06-28 </TD><TD>Amplification of pSB4K5 from iGEM Distribution Gibson Primer</TD><TD> GCTTCCTCCGAAGACGTTATC
</TD></TR><TR><TD>FS_20: DelF_fw</TD><TD>2013-07-13 </TD><TD>Amplification of DelF from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> GACTTGCTCGATGCGGTGCAG
</TD></TR><TR><TD>FS_21: DelF_fw</TD><TD>2013-07-13 </TD><TD>Amplification of DelF from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> GACGCCATCTACCACCTGTG
</TD></TR><TR><TD>FS_22: DelOP_short_fw</TD><TD>2013-08-02 </TD><TD>Amplification of DelOP from <I>Delftia acidovorans</I> genome inlcuding the recently predicted endogenous Promotor for DelOP Gibson Primer</TD><TD> GATGACGCAGGGCGGCGGAATTTGTTCATC
</TD></TR><TR><TD>FS_23: DelG_long_rv</TD><TD>2013-07-13 </TD><TD>Amplification of DelG from <I>Delftia acidovorans</I> genome Gibson primer with overhang to DelOP element including the recently predicted endogenous Promotor</TD><TD> GATGAACAAATTCCGCCGCCCTGCGTCATCTCAGATATCTCCCAGAGTTTCGAGAAAG
</TD></TR><TR><TD>FS_24: DelAE_rv</TD><TD>2013-07-13 </TD><TD>Amplification of DelAE from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> CAGAAGAATTCCCAGAAGGAGATGTCGAAG
</TD></TR><TR><TD>FS_25: DelEF_fw</TD><TD>2013-07-13 </TD><TD>Amplification of DelEF from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> ACACGGTGCTGCAGAAAACGCCCTTC
</TD></TR><TR><TD>FS_26: DelFG_rv</TD><TD>2013-07-13 </TD><TD>Amplification of DelFG from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> GAATTCATCCACGATGATCTGCATG
</TD></TR><TR><TD>FS_27: DelOP_rv</TD><TD>2013-08-02 </TD><TD>Amplification of DelOP from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> CTTTGGGCCGTGCCGGTTTTTGAGATAC
</TD></TR><TR><TD>FS_28: DelOP_rv</TD><TD>2013-08-02 </TD><TD>Amplification of DelOP from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> GTTTTTGAGATACGCGCGTTGTCAC
</TD></TR><TR><TD>FS_29: DelOP_rv</TD><TD>2013-08-02 </TD><TD>Amplification of DelOP from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> TTCCCCCTCTCTTTCTCGCTTC
</TD></TR><TR><TD>FS_30: DelOP_rv</TD><TD>2013-08-02 </TD><TD>Amplification of DelOP from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> CCGCTTCCCCCTCTCTTTCTCGCTTC
</TD></TR><TR><TD>FS_31: DelOP_fw</TD><TD>2013-08-02 </TD><TD>Amplification of DelOP from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> GTTGGCGAGTTCAAGAAATG
</TD></TR><TR><TD>FS_32: DelOP_fw</TD><TD>2013-08-02 </TD><TD>Amplification of DelOP from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> TCCTTCAGGTGTGCGGCAGACAAG
</TD></TR><TR><TD>FS_33: DelOP_fw</TD><TD>2013-08-02 </TD><TD>Amplification of DelOP from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> TTCTTCGTGATGACGCAGGGCGGCGGAATTTGTTC
</TD></TR><TR><TD>FS_35: DelG_fw</TD><TD>2013-08-05 </TD><TD>Amplification of DelG from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> CATGCAGATCATCGTGGATGAATTC
</TD></TR><TR><TD>FS_45: pSB4K5_fw</TD><TD>2013-08-02 </TD><TD>Amplification of pSB4K5 from iGEM Distribution Gibson Primer without mRFP</TD><TD> CCAGGCATCAAATAAAACGAAAG
</TD></TR><TR><TD>FS_46: DelL_rv </TD><TD>2013-08-02 </TD><TD>Amplification of DelL from <I>Delftia acidovorans</I> genome Gibson Primer creating overlap to pSB4K5 without mRFP</TD><TD> TCAGTCCTGCAGCGCCAGCTGTTCTG TGCTTTCGTTTTATTTGATGCCT
</TD></TR><TR><TD>FS_47_screening_BB_AF_fw</TD><TD>2013-08-02 </TD><TD>Primer for screening/sequencing of pFSN construct</TD><TD> GTTGGCCGATTCATTAATGC
</TD></TR><TR><TD>FS_48_screening_BB_AF_rv</TD><TD>2013-08-02 </TD><TD>Primer for screening/sequencing of pFSN construct</TD><TD> TAACGGTATCGGTATCGCTTTG
</TD></TR><TR><TD>FS_49_screening_AFI_AFII_fw</TD><TD>2013-08-02 </TD><TD>Primer for screening/sequencing of pFSN construct</TD><TD> GTTTCTCTGGAAGATGGATAC
</TD></TR><TR><TD>FS_50_screening_AFI_AFII_rv</TD><TD>2013-08-02 </TD><TD>Primer for screening/sequencing of pFSN construct</TD><TD> GTTGACGAAAAAGCCGACCAC
</TD></TR><TR><TD>FS_51_screening_AF_FG(21-26)_fw</TD><TD>2013-08-02 </TD><TD>Primer for screening/sequencing of pFSN construct</TD><TD> TGGATATCGACTGGACTGCCTG
</TD></TR><TR><TD>FS_52_screening_AF_FG(21-26)_rv</TD><TD>2013-08-02 </TD><TD>Primer for screening/sequencing of pFSN construct</TD><TD> TGCACCACATCGACGAAACGG
</TD></TR><TR><TD>FS_53_screening_FG(21-26)_G_fw</TD><TD>2013-08-02 </TD><TD>Primer for screening/sequencing of pFSN construct</TD><TD> GTACGGCCTATCACATCAGCG
</TD></TR><TR><TD>FS_54_screening_FG(21-26)_G_rv</TD><TD>2013-08-02 </TD><TD>Primer for screening/sequencing of pFSN construct</TD><TD> GAACCTGGGTGTTCACGAAAAAGCC
</TD></TR><TR><TD>FS_55_screening_G_OP_fw</TD><TD>2013-08-02 </TD><TD>Primer for screening/sequencing of pFSN construct</TD><TD> GTATCTCTACATGCATCGCTAC
</TD></TR><TR><TD>FS_56_screening_G_OP_fw</TD><TD>2013-08-02 </TD><TD>Primer for screening/sequencing of pFSN construct</TD><TD> AGGACATTTTCCGCACCCCG
</TD></TR><TR><TD>FS_57_screening_G_OP_rv</TD><TD>2013-08-02 </TD><TD>Primer for screening/sequencing of pFSN construct</TD><TD> GCTGGCGTTTTCCATAAG
</TD></TR><TR><TD>FS_58_screening_OP_L_fw</TD><TD>2013-08-02 </TD><TD>Primer for screening/sequencing of pFSN construct</TD><TD> GAACAACTTCCAGCACAGCCTGTTC
</TD></TR><TR><TD>FS_59_screening_OP_L_rv</TD><TD>2013-08-02 </TD><TD>Primer for screening/sequencing of pFSN construct</TD><TD> CGTTGAAGATTTCGTTGACG
</TD></TR><TR><TD>FS_60_screening_L_BB_fw</TD><TD>2013-08-02 </TD><TD>Primer for screening/sequencing of pFSN construct</TD><TD> CATCTTCAAGGTGTTCTATGAAC
</TD></TR><TR><TD>FS_61_screening_L_BB(with_mRFP)_rv</TD><TD>2013-08-02 </TD><TD>Primer for screening/sequencing of pFSN construct</TD><TD> CAGTTTAACTTTGTAGATGAAC
</TD></TR><TR><TD>FS_66: DelH_rv</TD><TD>2013-08-26 </TD><TD> Amplification of DelH from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> TGGGCATTCACCGCATCGATC
</TD></TR><TR><TD>FS_67: DelH_fw</TD><TD>2013-08-26 </TD><TD> Amplification of DelH from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> CTTCACGTTGATTGCGCATG
</TD></TR><TR><TD>FS_68: DelH_rv</TD><TD>2013-08-26 </TD><TD> Amplification of DelH from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> CAGAAGAACTCCCAGACCGAC
</TD></TR><TR><TD>FS_69: DelH_fw</TD><TD>2013-08-26 </TD><TD> Amplification of DelH from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> GACACCGTTCAGCTTCGATG
</TD></TR><TR><TD>FS_70: DelH_rv</TD><TD>2013-08-26 </TD><TD> Amplification of DelH from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> GAAGCTGCTCCGCTGATAGAT
</TD></TR><TR><TD>FS_71: DelH_fw</TD><TD>2013-08-26 </TD><TD> Amplification of DelH from <I>Delftia acidovorans</I> genome Gibson Primer</TD><TD> ATGTGCTGTCGCTCAAGATG
</TD></TR><TR><TD>FS_72_SR_02_fw</TD><TD>2013-08-30 </TD><TD> Screening of pFHFSN</TD><TD> ATGTGCTGTCGCTCAAGATG
</TD></TR><TR><TD>FS_73_SR_03_fw</TD><TD>2013-08-30 </TD><TD> Screening of pFHFSN</TD><TD> GTGCTGTTTGGCCGTATG
</TD></TR><TR><TD>FS_74_SR_04_fw</TD><TD>2013-08-30 </TD><TD> Screening of pFHFSN</TD><TD> ATCAGGTGCTGAGCTACGAC
</TD></TR><TR><TD>FS_75_SR_05_fw</TD><TD>2013-08-30 </TD><TD> Screening of pFHFSN</TD><TD> CTGTTCATCAACACCTTGCC
</TD></TR><TR><TD>FS_76_SR_06_rv</TD><TD>2013-08-30 </TD><TD> Screening of pFHFSN</TD><TD> GAAGACAGTCATAAGTGCGGC
</TD></TR><TR><TD>FS_77_rv</TD><TD> 2013-09-11 </TD><TD>Gibson-Primer rev, Amplficiation of the Backbone pSB6A1 with overlap to the RBS BBa_B0034 and the lacI-promotor, it creates an overlap to the beginning of DelH</TD><TD> GCGATTTGGCGCAGGCGGCCACGGTC CATCTAGTATTTCTCCTCTTTC
</TD></TR><TR><TD>FS_78_rv</TD><TD> 2013-09-26 </TD><TD>Gibson-Primer rev, Amplficiation of the Backbone pSB6A1 introducing the RBS BBa_B0032 and the promotor BBa_J23114  and creating an overlap to the first fragment of DelH amplified with primer DN_11</TD><TD> GATTTGGCGCAGGCGGCCACGGTCCATCTAGTACTTTCCTGTGTGACTCTAG AGCTAGCATTGTACCTAGGACTGAGCTAGCCATAAACTCTAGAAGCGGCCGCGAATTC
</TD></TR><TR><TD>FS_84_fw</TD><TD> 2013-09-26 </TD><TD>Gibson-Primer fw, Amplficiation of the first fragment of DelH introducing the RBS BBa_B0032 and creating an overlap to primer FS_85 thereby partially introducing the promotor BBa_J23114</TD><TD> GCTCAGTCCTAGGTACAATGCTAGCTCTAGAGTCACACAGGAAAGTACTAGATGGACCGTGGCCGCCTGCG
</TD></TR><TR><TD>FS_85_rv</TD><TD> 2013-09-26 </TD><TD>Gibson-Primer rev, Amplficiation of the Backbone pSB6A1, partially introducing the promotor BBa_B0032 with overlap to primer FS_84 and therefore the promotor BBa_J23114, it creates an overlap to the beginning of DelH</TD><TD> GCGATTTGGCGCAGGCGGCCACGGTCCATCTAGTATTTCTCCTCTTTC
</TD></TR><TR><TD>FS_86_rv</TD><TD> 2013-09-27 </TD><TD>Gibson-Primer rev, Amplficiation of the Backbone pSB4K5 without any promotor introducing a KpnI cutting site for restriction cloning, creates an overlap to DelH and will be used for the ccdB strategy</TD><TD> GGCGATTTGGCGCAGGCGGCCACGGTCCATGTACTTCGAGTCACTAAGGGCTAAC
</TD></TR><TR><TD>FS_87_fw</TD><TD> 2013-09-27 </TD><TD>Gibson-Primer fw, Amplficiation of the Backbone pSB6A1 introducing a BamHI cutting site for restriction cloning and creating an overlap to the last fragment of DelH</TD><TD> CGCTGGAGTACGCGCTGGACTGAGATCCCAGGCATCAAATAAAACG
</TD></TR><TR><TD>FS_90_fw</TD><TD> 2013-09-27 </TD><TD>Gibson-Primer fw, Amplficiation of the ccdB cassette from the template pDonorPlasmid introducing a KpnI cutting site for restriction cloning, creates an overlap to the promotor BBa_J23114 and will be used for the ccdB strategy</TD><TD> CTCAGTCCTAGGTACAATGCTAGCTCTAGAGTCACACAGGAAAGCAGTACACTGGCTGTGTATAAGGGAG
</TD></TR><TR><TD>FS_93_rv</TD><TD> 2013-09-27 </TD><TD>Gibson-Primer rev, Amplficiation of the ccdB cassette from the template pDonorPlasmid introducing a BamHI cutting site for restriction cloning, creates an overlap to the backbone pSB6A1 and will be used for the ccdB strategy</TD><TD> GTTCACCGACAAACAACAGATGA TCCGCGTGGATCCGGCTTAC
</TD></TR><TR><TD>FS_94_fw</TD><TD> 2013-09-27 </TD><TD>Primer fw, Amplficiation of the backbone pSB6A1, will be used for the ccdB strategy</TD><TD> ATCTGTTGTTTGTCGGTGAACGC
</TD></TR><TR><TD> HM01:DelH_EcoRI_fw </TD><TD>  -  </TD><TD> fw_Primer for DelH Fragment f1b</TD><TD> GCATTGGAGCCTCAATGGCAAGTC
</TD></TR><TR><TD>HM02:DelH_Gib1.1_rev</TD><TD>2013-07-09 </TD><TD>Gibson-Primer DelH</TD><TD>  TGCTGCGCCTGCATACGGCCAAACA
</TD></TR><TR><TD>HM03:DelH_Gib1.2_fw</TD><TD>2013-07-09 </TD><TD>Gibson-Primer DelH</TD><TD> AGCGGCAGGGACGACGTGGT
</TD></TR><TR><TD>HM04:DelH_Gib1.2_rev</TD><TD>2013-07-09 </TD><TD>Gibson-Primer DelH</TD><TD> CATAGAGGTTGTAGAGA
</TD></TR><TR><TD>HM05:DelH_Gib2.1_fw</TD><TD>2013-07-09 </TD><TD> Gibson-Primer DelH </TD><TD> AGAACGCCGTCTTCAGGCTCCTG
</TD></TR><TR><TD>HM06:DelH_Gib2.1_rev</TD><TD>2013-07-09 </TD><TD>Gibson-Primer DelH</TD><TD>  CAATGCTTTG CCGCTCGAA
</TD></TR><TR><TD>HM07:DelH_Gib2.2_fw</TD><TD>2013-07-09 </TD><TD>Gibson-Primer DelH</TD><TD> TCGCCACGGCAGCTGTTCGA
</TD></TR><TR><TD>HM08:DelH_Gib2_end_rev</TD><TD>2013-07-09 </TD><TD>Gibson-Primer DelH</TD><TD> TCAGTCCAGCGCGTACTCCAG
</TD></TR><TR><TD>HM09:AraC_RBS_Delh_rev</TD><TD>2013-07-09</TD><TD>Gibson-Primer rev, introduces a new RBS and has the AraC-promotor and the beginning of DelH</TD><TD> TTGCAAAGCGCTCGGCGATTTGGCGCAGGCGGCCACGGTCCATTTAACTTTCTCCTCTTTAATACTTTGAGCTAGCCCAAAAAAACGGTATGGAGAAACAGTAGAGAGTT
</TD></TR><TR><TD>HM10:RBS_lacZ</TD><TD>2013-07-09 </TD><TD>Gibson-Primer fw for the pSB6A1 Backbone with the end of DelH, RBS(1) and the beginning of lacZ </TD><TD> TGGAGTACGCGCTGGACTGA  TCTAGAGAAAGAGGAGAAA TACTAG ATGACCATGATTA
</TD></TR><TR><TD>HM11:lacI_RBS(1)_DelH_rev</TD><TD> 2013-07-24 </TD><TD>Gibson-Primer rev, amplify the Backbone with overlap with the RBS and the lacI-promotor and it creates and overlap to the start of DelH</TD><TD> TCGGCGATTTGGCGCAGGCGGCCACGGTCC ATCTAGTATTTCTCCTCTTTCTCTAGTATGTGTG
</TD></TR><TR><TD>HM12:DelH_RBS(1.2)_mRFP_fw</TD><TD> 2013-07-24 </TD><TD> Gibson-Primer fw for the pSB6A1 Backbone with the end of DelH, introducing a new RBS(new) and the beginning of mRFP</TD><TD> ATTGGCGCTGGAGTACGCGCTGGACTG ATCAAAGTATTAAAGAGGAGAAAGT TAAATGGCTTCCTCCGAAGACGTTATCAAAGAG
</TD></TR><TR><TD>HM13:Screen_DelH_end_fw</TD><TD>2013-08-16 </TD><TD>New screening primer for the end of DelH together with the VR2 primer from the registry</TD><TD>  TTTCTGACGACCCTGCACCTGAAG
</TD></TR><TR><TD>HM14:DelH_tetR_fw</TD><TD>2013-08-16 </TD><TD>Gibson-Primer DelH-tetR: amplifies the tetracycline resistance from the pSB1T3 Backbone and creates an overlap to the end of DelH</TD><TD> ATTGGCGCTGGAGTACGCGCTGGACTGAATGAAGTTTTAAATCAATCTAAAG
</TD></TR><TR><TD>HM15:tetR_stop_BB_rev</TD><TD>2013-08-16 </TD><TD>Gibson-Primer tetR-pSB6A1: amplifies the tetracycline resistance and creates an overlap with the Terminator of the Backbone pSB6A1</TD><TD> CGACTGAGCCTTTCGTTTTATTTGATGCCTGGCCTCGTGATACGCCTATTTTTATAGG
</TD></TR><TR><TD>HM16:tetR_pSB6A1_fw</TD><TD>2013-08-16</TD><TD>Gibson-Primer DelH, amplifies the Backbone pSB6A1 creating an overlap with the tetracycline resistance </TD><TD> AAAAATAGGCGTATCACGAG  GCCAGGCATCAAATAAAACGAAAGGCTCAG
</TD></TR><TR><TD>HM17:DelH_Terminator_BB_fw</TD><TD> 2013-08-16 </TD><TD> Gibson-Primer fw for the pSB6A1 Backbone (binding the terminator) and creating an overlap with the end of DelH</TD><TD> ATTGGCGCTGGAGTACGCGCTGGACTGAAGGCATCAAATAAAACGAAAGGCTCAG
</TD></TR><TR><TD> HM20:BB_HPLC_rev </TD><TD> 11-09-2013 </TD><TD> HPLC version of HM11  Gibson-Primer rev, amplify the Backbone with overlap with the RBS and the lacI-promotor and it creates and overlap to the start of DelH  </TD><TD> GATTTGGCGCAGGCGGCCAC GGTCCATCTAGTATTTCTCCTCTTTC
</TD></TR><TR><TD> HM21:fw_lacI_BbsI_Xba </TD><TD> 2013-09-15 </TD><TD> Forward primer for cutting out mutated fragment for mutagenesis </TD><TD> TTTTGAAGACAA CTAGGCAATACGCAA
</TD></TR><TR><TD> HM22:rev_RBS </TD><TD> 2013-09-15 </TD><TD> Reverse Primer in RBS for mutagenesis </TD><TD> TTTTGAAGACAA CTCTTTCTCTAGTATGTGTGAAATTG
</TD></TR><TR><TD> HM23:fw_RBS </TD><TD> 2013-09-15 </TD><TD> Forward Primer in RBS for mutagenesis </TD><TD> TTTTGAAGACAA AGAGGAGAAATACTAGATGGACCGTGGC
</TD></TR><TR><TD> HM24:rev_BbsI_MfeI </TD><TD> 2013-09-15 </TD><TD> Reverse primer for cutting out mutated fragment for mutagenesis </TD><TD> TTTTGAAGACAA AATTGGACAGCGCGGCATGCCGGTTG
</TD></TR><TR><TD> IK01:pLF03_integr_argK_fw </TD><TD> 2013-06-12 </TD><TD> For verification of correct genomic integration of pLF03 into <I>E.coli</I> BAP1 via colony-PCR. Primer against <I>E.coli</I> BAP1 genome. </TD><TD> GCTGATGGAAGTGGCTGATCTGATC
</TD></TR><TR><TD> IK02:pLF03_integr_pET21c_rev </TD><TD> 2013-06-12 </TD><TD> For verification of correct genomic integration of pLF03 into <I>E.coli</I> BAP1 via colony-PCR. Primer against pLF03 backbone (pET-21c). </TD><TD> TCCGCTCACAATTCCCCTATAGTG
</TD></TR><TR><TD> IK03:pLF03_integr_argK_rev </TD><TD> 2013-06-18 </TD><TD> For verification of correct genomic integration of pLF03 into <I>E.coli</I> BAP1 via colony-PCR. Primer against <I>E.coli</I> BAP1 genome for positive control (to be used with IK01 or IK04). </TD><TD> GATAAATTCACTGAGCTGCCGCAG
</TD></TR><TR><TD> IK04:pLF03_integr_argK_fw </TD><TD> 2013-06-18 </TD><TD> For verification of correct genomic integration of pLF03 into <I>E.coli</I> BAP1 via colony-PCR. Primer against <I>E.coli</I> BAP1 genome. Alternative forward primer in case IK01 does not work. </TD><TD> GCGGGAATTAATGCTGTTATGCGAAG
</TD></TR><TR><TD> IK05:pLF03_integr_ygfG_fw </TD><TD> 2013-06-18 </TD><TD> For verification of correct genomic integration of pLF03 into <I>E.coli</I> BAP1 via colony-PCR. Primer against <I>E.coli</I> BAP1 genome, namely ygfG, which should be replaced by the construct. </TD><TD> GCGGTCATTGAGTTTAACTATGGCC
</TD></TR><TR><TD> IK06:pLF03_integr_ygfG_rev </TD><TD> 2013-06-18 </TD><TD> For verification of correct genomic integration of pLF03 into <I>E.coli</I> BAP1 via colony-PCR. Primer against <I>E.coli</I> BAP1 genome, namely ygfG, which should be replaced by the construct. </TD><TD> AACATGGTTGAGGATGCCGACAGC
</TD></TR><TR><TD> IK07:ygfG21C1 </TD><TD> 2013-06-25 </TD><TD> For insertion of methylmalonyl-CoA synthesis pathway into <I>E. coli</I>. Extended ygfG21C1 with higher melting temperature. </TD><TD> TCACCGCGCACCGGCCTGCGGCAGCTCAGTGAATTTATCCAGATCTCGATCCCGCGAAATTAATAC
</TD></TR><TR><TD> IK08:ygfG21C2 </TD><TD> 2013-06-25 </TD><TD> For insertion of methylmalonyl-CoA synthesis pathway into <I>E. coli</I>. Extended ygfG21C2 with higher melting temperature. </TD><TD> GTTATGCTGGATAATTTCTGCCGCTTCATTGGCGGTCATCCAAAAAACCCCTCAAGACCCGTTTAG
</TD></TR><TR><TD> IK24:pccB_fw </TD><TD> 2013-07-16 </TD><TD> Colony-PCR of methylmalonyl-CoA synthesis pathway </TD><TD> CGTATCGAGGAAGCGACGCAC
</TD></TR><TR><TD> IK25:pccB_rev </TD><TD> 2013-07-16 </TD><TD> Colony-PCR of methylmalonyl-CoA synthesis pathway </TD><TD> GGTGATGAACATGTGGCTGGTCTG
</TD></TR><TR><TD> IK26:BBa_I746200_fw </TD><TD> 2013-07-19 </TD><TD> Forward primer for permeability device </TD><TD> ATGAACAAGAAGATTCATTCCCTGGCCTTG
</TD></TR><TR><TD> IK27:BBa_I746200-BBa_B0029_rev </TD><TD> 2013-07-19 </TD><TD> Reverse primer for permeability device with partial Gibson overhang for BBa_B0029 RBS </TD><TD> GAATACCAGT TCAGAAGTGGGTGTTTACGCTCATATAC
</TD></TR><TR><TD> IK28:pccB-BBa_B0029-BBa_I746200_fw </TD><TD> 2013-07-19 </TD><TD> Forward primer for pccB with BBa_B0029 RBS and Gibson overhang for permeability device </TD><TD> ATATGAGCGTAAACACCCACTTCTGAACTGGTATTCACACAGGAAACCTACTAG ATGTCCGAGCCGGAAGAGC
</TD></TR><TR><TD> IK29:accA2-BBa_B0030_rev </TD><TD> 2013-07-19 </TD><TD> Reverse primer for accA2 with partial Gibson overhang for BBa_B0030 RBS </TD><TD> TAATGAAGTTG CTAGTGATTCTCGCAGATGGC
</TD></TR><TR><TD> IK30:sfp-BBa_B0030-accA2_fw </TD><TD> 2013-07-19 </TD><TD> Forward primer for sfp with BBa_B0030 RBS and Gibson overhang for accA2 </TD><TD> TCCGGCGCCGCCATCTGCGAGAAT CACTAGCAACTTCATTAAAGAGGAGAAATACTAG ATGAAGATTTACGGAATTTATATGGAC
</TD></TR><TR><TD> IK31:sfp_rev </TD><TD> 2013-07-19 </TD><TD> Reverse primer for sfp </TD><TD> TTATAAAAGCTCTTCGTACGAGAC
</TD></TR><TR><TD> IK32:BBa_J04450-sfp_fw </TD><TD> 2013-07-19 </TD><TD> Forward primer for mRFP-containing backbones with Gibson overhang for sfp </TD><TD> ATCACAATGGTCTCGTACGAAGAGCTTTTATAA TACTAGAGCCAGGCATCAAATAAAACG
</TD></TR><TR><TD> IK33:BBa_J04450-BBa_I746200_rev </TD><TD> 2013-07-19 </TD><TD> Reverse primer for mRFP-containing backbones with Gibson overhang for permeability device </TD><TD> ACAAGGCCAGGGAATGAATCTTCTTGTTCAT CTAGTATTTCTCCTCTTTCTCTAGTATG
</TD></TR><TR><TD> IK34:pccB_fw </TD><TD> 2013-07-19 </TD><TD> Forward primer for pccB </TD><TD> ATGTCCGAGCCGGAAGAGC
</TD></TR><TR><TD> IK35:BBa_J04450-pccB_rev </TD><TD> 2013-07-19 </TD><TD> Reverse primer for mRFP-containing backbones with Gibson overhang for pccB </TD><TD> ATGTCGGGCTGCTGCTCTTCCGGCTCGGACAT CTAGTATTTCTCCTCTTTCTCTAG
</TD></TR><TR><TD> IK36:pLF03_seq_rev </TD><TD> 2013-07-26 </TD><TD> reverse primer for pLF03 for sequencing </TD><TD> CCGGTATCAACAGGGACACCAG
</TD></TR><TR><TD> IK37:pLF03-catR_seq_rev </TD><TD> 2013-07-26 </TD><TD> reverse primer for pLF03 sequencing </TD><TD> CATTGAGCAACTGACTGAAATGCCTC
</TD></TR><TR><TD> IK38:pIK1_fw </TD><TD> 2013-08-30 </TD><TD> Forward mutagenic primer for pIK1 </TD><TD> ATTCCCTGGCCTTGT T GGTCAATCTGGGGATTTATG
</TD></TR><TR><TD> IK39:pIK_rev </TD><TD> 2013-08-30 </TD><TD> Reverse mutagenic primer for pIK1 </TD><TD> CCCAGATTGACC A ACAAGGCCAGGGAATGAATCTTC
</TD></TR><TR><TD> IK40:pIK2-BBa_J23114-BBa_B0030_rev </TD><TD> 2013-09-06 </TD><TD> reverse primer for pIK2 with Gibson overhang for BBa_J23114-BBa_B0030 </TD><TD> TTTAATCTCTAGAGCTAGCATTGTACCTAGGACTGAGCTAGCCATAAA CTCTAGTAGAGAGCGTTCAC
</TD></TR><TR><TD> IK41:BBa_I746200-BBa_B0030-BBa_J23114_fw </TD><TD> 2013-09-06 </TD><TD> forward primer for BBa_I746200 with Gibson overhang for BBa_B0030-BBa_J23114 </TD><TD> ATGCTAGCTCTAGAGATTAAAGAGGAGAAATACTAG ATGAACAAGAAGATTCATTCCCTG
</TD></TR><TR><TD> IK42:BBa_I746200_rev </TD><TD> 2013-09-06 </TD><TD> reverse primer for BBa_I746200 </TD><TD> TCAGAAGTGGGTGTTTACGCTC
</TD></TR><TR><TD> IK43:pIK2-BBa_I746200_fw </TD><TD> 2013-09-06 </TD><TD> forward primer for pIK2 with Gibson overhang for BBa_I746200 </TD><TD> TGAGCGTAAACACCCACTTCTGA TACTAGAGTCACACTGGCTC
</TD></TR><TR><TD>NK_01_FS_62_screening_L_BB(without_mRFP)_rv</TD><TD>2013-08-02 </TD><TD>Primer for screening/sequencing of pFSN construct</TD><TD>GTTCACCGACAAACAACAGATAAAACG
</TD></TR><TR><TD> ygfG21C1 </TD><TD> 2013-06-03 </TD><TD> For insertion of methylmalonyl-CoA synthesis pathway into <I>E. coli</I>. Primer from &lt;bib id=&quot;pmid17959404&quot;/&gt;. </TD><TD> TCACCGCGCACCGGCCTGCGGCAGCTCAGTGAATTTATCC AGATCTCGATCCC
</TD></TR><TR><TD> ygfG21C2 </TD><TD> 2013-06-03 </TD><TD> For insertion of methylmalonyl-CoA synthesis pathway into <I>E. coli</I>. Primer from &lt;bib id=&quot;pmid17959404&quot;/&gt;. </TD><TD> GTTATGCTGGATAATTTCTGCCGCTTCATTGGCGGTCATCC CAAAAAACCCCTCAAG
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Module_Shuffling">Module Shuffling</SPAN></H2><TABLE class="wikitable sortable"><TBODY><TR><TH> Identifier </TH><TH> Order date </TH><TH> Note </TH><TH> Sequence
</TH></TR><TR><TD> AT01:RFC10prefix_TycA_fw</TD><TD> 2013-08-12 </TD><TD> Fw primer for amplification of TycAdCom; introduction of RFC10 prefix </TD><TD> TTTT GAATTC GCGGCCGC T TCTAG ATG TTA GCA AAT CAa GCC AAT C
</TD></TR><TR><TD> AT02:RFC10suffix_TycA_rv
</TD><TD> 2013-08-12 </TD><TD> Rv primer for amplification of TycAdCom; introduction of RFC10 suffix</TD><TD> TTTT CTGCAG CGGCCGC T ACTAGT A aGT TCG tTC TAC TTC TTT TTT C
</TD></TR><TR><TD> AT03:RFC10prefix-TycB1_fw
</TD><TD> 2013-08-12 </TD><TD> Fw primer for amplification of TycB1dCom; introduction of RFC10 prefix</TD><TD> TTTT GAATTC GCGGCCGC T TCTAG ATG AGT GTA TTT AGC AAA GAA CAA G
</TD></TR><TR><TD> AT04:RFC10suffix_TycB1_rv
</TD><TD> 2013-08-12 </TD><TD> Rv primer for amplification of TycB1dCom; introduction of RFC10 suffix </TD><TD> TTTT CTGCAG CGGCCGC T ACTAGT A TTC CTC CCC aCC TTC
</TD></TR><TR><TD> AT05:RFC10prefix-TycC5_fw
</TD><TD> 2013-08-12 </TD><TD> Fw primer for amplification of TycC5; introduction of RFC10 prefix </TD><TD> TTTT GAATTC GCGGCCGC T TCTAG AG GCG CAT ATT GCa GAG AG
</TD></TR><TR><TD> AT06:RFC10suffix_TycC5_rv
</TD><TD> 2013-08-14 </TD><TD> Rv primer for amplification of TycC5; introduction of RFC10 suffix </TD><TD> TTTT CTGCAG CGGCCGC T ACTAGT A TTT GGC TGT CTC TTC GAT GAA C
</TD></TR><TR><TD> AT07:RFC10prefix-TycC6_fw
</TD><TD> 2013-08-12 </TD><TD> Fw primer for amplification of TycC6; introduction of RFC10 prefix  </TD><TD> TTTT GAATTC GCGGCCGC T TCTAG AG GGG AAT GTC TTC TCG ATC
</TD></TR><TR><TD> AT08:RFC10suffix_TycC6_rv
</TD><TD> 2013-08-14 </TD><TD> Rv primer for amplification of TycC6; introduction of RFC10 suffix </TD><TD> TTTT CTGCAG CGGCCGC T ACTAGT A TTA TTT CAG GAT aAA CAG TTC TTG
</TD></TR><TR><TD> AT09:R10_B1_fw_longer
</TD><TD> 2013-08-18 </TD><TD> Fw primer (longer) for amplification of TycB1dCom; introduction of RFC10 prefix  </TD><TD> TTTT GAA TTC GCG GCC GCT TCT AG     ATG AGT GTA TTT AGC AAA GAA CAA GTT C
</TD></TR><TR><TD> AT10:R10_B1_rv_longer
</TD><TD> 2013-08-18 </TD><TD> Rv primer (longer for amplification of TycB1dCom; introduction of RFC10 suffix </TD><TD> TTTT CTG CAG CGG CCG CTA CTA GTA ATA CGC aCT TTC CTC CCC GCC
</TD></TR><TR><TD> AT11:R10_C5_fw_rpos
</TD><TD> 2013-08-18 </TD><TD> Fw primer (repositioned) for amplification of TycC6; introduction of RFC10 prefix</TD><TD> TTTT GAATTC GCGGCCGC T TCTAG AG GAGCAGTTCGAGACGATCCAGCC
</TD></TR><TR><TD> AT101</TD><TD> 2013-09-05</TD><TD> forward screening primer TycAdCom </TD><TD> GGACATCATCGAACAGGC
</TD></TR><TR><TD> AT102</TD><TD> 2013-09-05</TD><TD> reverse screening primer TycAdCom </TD><TD> GACATAGGCAAGATCCGTAG
</TD></TR><TR><TD> AT103</TD><TD> 2013-09-05</TD><TD> forward screening primer TycAdCom </TD><TD> GCAAGGACAAGTAGATGGC
</TD></TR><TR><TD> AT104 </TD><TD> 2013-09-05</TD><TD> reverse screening primer TycAdCom </TD><TD> GTACAGCCATACAGTCGC
</TD></TR><TR><TD> AT105 </TD><TD> 2013-09-05</TD><TD> forward screening primer TycC5 </TD><TD> GTCACCATGCTGAGAACG
</TD></TR><TR><TD> AT106 </TD><TD> 2013-09-05</TD><TD> reverse screening primer TycC5 </TD><TD> CAGCAAAGCGGTCATAATC
</TD></TR><TR><TD> AT107 </TD><TD> 2013-09-05</TD><TD> forward screening primer TycC6 </TD><TD> AAGCTACGCTGTTGATTGC
</TD></TR><TR><TD> AT108 </TD><TD>  2013-09-05</TD><TD> reverse screening primer TycC6 </TD><TD> AGCACGTAACGATCCTC
</TD></TR><TR><TD> IK09:TycA_A1_fw </TD><TD> 2013-07-08 </TD><TD> Colony-PCR of <I>Brevibacillus parabrevis</I>: TycA A domain </TD><TD> ATGTTAGCAAATCAAGCCAATCTC
</TD></TR><TR><TD> IK10:TycA_A1_rev </TD><TD> 2013-07-08 </TD><TD> Colony-PCR of <I>Brevibacillus parabrevis</I>: TycA A domain </TD><TD> TTGGTTTGCTGTAAGATCAGGCTC
</TD></TR><TR><TD> IK11:TycB_A1_fw </TD><TD> 2013-07-08 </TD><TD> Colony-PCR of <I>Brevibacillus parabrevis</I>: TycB A1 domain </TD><TD> AATTCGGGAGTCGAGCTTTGTCAG
</TD></TR><TR><TD> IK12:TycC6_rev </TD><TD> 2013-07-19 </TD><TD> reverse primer for tyrocidine TycC6 module </TD><TD> TTATTTCAGGATGAACAGTTCTTGCAGG
</TD></TR><TR><TD> IK13:IK13:TycB1-dCom-dC_fw </TD><TD> 2013-07-19 </TD><TD> forward primer for tyrocidine TycB1 module without Com and C domain </TD><TD> GATTGCGTGGCAAACAATTCGGGAGTC
</TD></TR><TR><TD> IK14:TycB1-TycC6_rev </TD><TD> 2013-07-19 </TD><TD> reverse primer for tyrocidine TycB1 module with Gibson overhang for TycC6 </TD><TD> CAGGCTCGATCGAGAAGACATTCCC TTCCTCCCCGCCTTCCACATACGC
</TD></TR><TR><TD> IK15:TycC6-TycB1_fw </TD><TD> 2013-07-19 </TD><TD> forward primer for tyrocidine TycC6 module with Gibson overhang for TycB1 </TD><TD> GCGTATGTGGAAGGAGGGGAGGAA GGGAATGTCTTCTCGATCGAGCCTG</TD></TR><TR><TD> IK16:TycA_fw </TD><TD> 2013-07-19 </TD><TD> forward primer for tyrocidine TycA module </TD><TD> ATGTTAGCAAATCAGGCCAATCTCATC
</TD></TR><TR><TD> IK17:TycA-dCom-TycC5_rev </TD><TD> 2013-07-19 </TD><TD> reverse primer for tyrocidine TycA module without Com domain with Gibson overhang for TycC5 </TD><TD> GAATGCGCTCTCGGCAATATGGGC TGTTCGCTCTACTTCTTTTTTCTCGG
</TD></TR><TR><TD> IK18:TycC5-TycA-dCom_fw </TD><TD> 2013-07-19 </TD><TD> forward primer for tyrocidine TycC5 module with Gibson overhang for TycAdCom </TD><TD> CCGAGAAAAAAGAAGTAGAGCGAACA GCCCATATTGCCGAGAGCGCATTC
</TD></TR><TR><TD> IK19:TycB1-TycC5_rev </TD><TD> 2013-07-19 </TD><TD> reverse primer for tyrocidine TycB1 module with Gibson overhang for TycC5 </TD><TD> GAATGCGCTCTCGGCAATATGGGC TTCCTCCCCGCCTTCCACATACGC
</TD></TR><TR><TD> IK20:TycC5-TycB1_fw </TD><TD> 2013-07-19 </TD><TD> forward primer for tyrocidine TycC5 module with Gibson overhang for TycB1 </TD><TD> GCGTATGTGGAAGGAGGGGAGGAA GCCCATATTGCCGAGAGCGCATTC
</TD></TR><TR><TD> IK21:pSB4K5-TycB1dComdC_rev </TD><TD> 2013-07-19 </TD><TD> reverse primer for mRFP-carrying backbones with Gibson overhang for tyrocidine TycA module (nomenclature wrong) </TD><TD> GTCGATGAGATTGGCCTGATTTGCTAACAT CTAGTATTTCTCCTCTTTCTCTAGTATGTG
</TD></TR><TR><TD> IK22::pSB4K5-TycC6_fw </TD><TD> 2013-07-19 </TD><TD> forward primer for mRFP-carrying backbones with Gibson overhang for tyrocidine TycC6 module </TD><TD> AACATCCTGCAAGAACTGTTCATCCTGAAA TAATAACGCTGATAGTGCTAGTGTAGATC
</TD></TR><TR><TD> IK23:pSB4K5-TycA_rev </TD><TD> 2013-07-19 </TD><TD> reverse primer for mRFP-carrying backbones with Gibson overhang for tyrocidine TycB1 module without Com and C domains + start codon (nomenklature wrong) </TD><TD> GACTCCCGAATTGTTTGCCACGCAATCCAT CTAGTATTTCTCCTCTTTCTCTAGTATGTG
</TD></TR><TR><TD> PW01:TycB_A1_rev </TD><TD> 2013-07-08 </TD><TD> Colony-PCR of <I>Brevibacillus parabrevis</I>: TycB A1 domain </TD><TD> CTTGGCACTTCCTTCAGGCTTC
</TD></TR><TR><TD> PW02:TycB_E1_fw </TD><TD> 2013-07-08 </TD><TD> Colony-PCR of <I>Brevibacillus parabrevis</I>: TycB E domain </TD><TD> CGAGAGAGCGAGCAAGGTG
</TD></TR><TR><TD> PW03:TycB_E1_rev </TD><TD> 2013-07-08 </TD><TD> Colony-PCR of <I>Brevibacillus parabrevis</I>: TycB E domain </TD><TD> GTACTCGCCTTCTTCTTTTGC
</TD></TR><TR><TD> PW04:pSB1C3-TycC5ΔC_rev </TD><TD> 2013-07-19 </TD><TD> Integration of Tetrapeptide NRPS from <I>Brevibacillus parabrevis</I> in backbone; Gibson primer for Tetrapeptide I &amp; II </TD><TD> TGTTTTGGTTGCGAGGAAGCTGTGCAGCAT CTAGTATTTCTCCTCTTTCTCTAGTATGTG
</TD></TR><TR><TD> PW05:TycC5ΔC_fwd </TD><TD> 2013-07-19 </TD><TD> Amplification of TycC5-Module from <I>Brevibacillus parabrevis</I>; Gibson primer for Tetrapeptide I &amp; II </TD><TD> ATGCTGCACAGCTTCCTCGCAACCAAAACAGCC
</TD></TR><TR><TD> PW06:TycC5ΔC-TycB1ΔCom_rev </TD><TD> 2013-07-19 </TD><TD> Amplification of TycC5-Module from <I>Brevibacillus parabrevis</I>; Gibson primer for Tetrapeptide I &amp; II </TD><TD> GTGAAACAGCATCCCCTCTTGCATCGG AGGCTCGATCGAGAAGACATTCCCTTTG
</TD></TR><TR><TD> PW07:TycC5ΔC-TycB1ΔCom_fwd </TD><TD> 2013-07-19 </TD><TD> Amplification of TycB1+C(TycB2)-Module from <I>Brevibacillus parabrevis</I>; Gibson primer for Tetrapeptide I &amp; II </TD><TD> CAAAGGGAATGTCTTCTCGATCGAGCCT CCGATGCAAGAGGGGATGCTGTTTCAC
</TD></TR><TR><TD> PW08:C(TycB2)-TycAΔCom_rev </TD><TD> 2013-07-19 </TD><TD> Amplification of TycB1+C(TycB2)-Module from <I>Brevibacillus parabrevis</I>; Gibson primer for Tetrapeptide I &amp; II </TD><TD> GTTGTCGATGAGATTGGCCTGATTTGCTAACAT GATTTGCGCCAGCTCCTGCTCCGTGTT
</TD></TR><TR><TD> PW09:C(TycB2)-TycAΔCom_fwd </TD><TD> 2013-07-19 </TD><TD> Amplification of TycA-Module from <I>Brevibacillus parabrevis</I>; Gibson primer for Tetrapeptide I &amp; II </TD><TD> AACACGGAGCAGGAGCTGGCGCAAATC ATGTTAGCAAATCAGGCCAATCTCATCGACAAC
</TD></TR><TR><TD> PW10:TycAΔCom-TycC6_rev </TD><TD> 2013-07-19 </TD><TD> Amplification of TycAΔCom-Module from <I>Brevibacillus parabrevis</I>; Gibson primer for Tetrapeptide I </TD><TD> CTTTGGCTGTCTCTTCGATGAACGC TCGCTCTACTTCTTTTTTCTCGGTGCAATG
</TD></TR><TR><TD> PW11:TycAΔCom-TycC6_fwd </TD><TD> 2013-07-19 </TD><TD> Amplification of TycC6-Module from <I>Brevibacillus parabrevis</I>; Gibson primer for Tetrapeptide I </TD><TD> CATTGCACCGAGAAAAAAGAAGTAGAGCGA GCGTTCATCGAAGAGACAGCCAAAG
</TD></TR><TR><TD> PW12:TycAΔE-TycC6_rev </TD><TD> 2013-07-19 </TD><TD> Amplification of TycAΔE-Module from <I>Brevibacillus parabrevis</I>; Gibson primer for Tetrapeptide II; one mismatch C-&gt;t </TD><TD> CTTTTGCACAGGCTCGATCGAGAAGAC GCTtTTGACAAAAAGAGCAACCTG
</TD></TR><TR><TD> PW13:TycAΔE-TycC6_fwd </TD><TD> 2013-07-19 </TD><TD> Amplification of TycC6-Module from <I>Brevibacillus parabrevis</I>; Gibson primer for Tetrapeptide II; one mismatch G-&gt;a </TD><TD> CAGGTTGCTCTTTTTGTCAAaAGC GTCTTCTCGATCGAGCCTGTGCAAAAG
</TD></TR><TR><TD> PW14:C(TycC2)-indC_rev </TD><TD> 2013-08-16 </TD><TD> Amplification of C-domain from TycC2 from <I>Brevibacillus parabrevis</I>; Gibson overhang to IndC; for construct 1, 2 &amp; 3 </TD><TD> ACATTGTGTAATATTATTTTCTAACAT CGTTTTGCTGCTGGCAGGCTG
</TD></TR><TR><TD> PW15:C(TycC2)-indC_fwd </TD><TD> 2013-08-16 </TD><TD> Amplification of indC from <I>Photorhabdus luminescens</I>; Gibson overhang to C-domain from TycC2; for construct 1, 2 &amp; 3 </TD><TD> CAGCCTGCCAGCAGCAAAACG ATGTTAGAAAATAATATTACACAATGT
</TD></TR><TR><TD> PW16:indC_rev </TD><TD> 2013-08-16 </TD><TD> Amplification of indC from <I>Photorhabdus luminescens</I>; no Gibson overhang; for construct 1, 2 &amp; 3 </TD><TD> TTAGATTATTTTCTCAATCTCAGCAACACCTTC
</TD></TR><TR><TD> PW17:TycAdE-C(TycC2)_rev </TD><TD> 2013-08-16 </TD><TD> Amplification of TycAdE from <I>Brevibacillus parabrevis</I>; Gibson overhang to C-domain from TycC2; for construct 1 </TD><TD> CGAAAGGAAGCGGGCCAGCTC AGCAACCTGCTCGATCGTCGGGTA
</TD></TR><TR><TD> PW18:TycAdE-C(TycC2)_fwd </TD><TD> 2013-08-16 </TD><TD> Amplification of C-domain from TycC2-module from <I>Brevibacillus parabrevis</I>; Gibson overhang to TycAdE; for construct 1 </TD><TD> TACCCGACGATCGAGCAGGTTGCT GAGCTGGCCCGCTTCCTTTCG
</TD></TR><TR><TD> PW19:TycC4dC_fwd </TD><TD> 2013-08-16 </TD><TD> Amplification of TycC4-module from <I>Brevibacillus parabrevis</I> without the C-domain; no Gibson overhang, ATG added; for construct 3 </TD><TD> ATGTATCCGCGCGATCTGACGATTC
</TD></TR><TR><TD> PW20:TycC4dC-C(TycC2)_rev </TD><TD> 2013-08-16 </TD><TD> Amplification of TycC4-module from <I>Brevibacillus parabrevis</I> without the C-domain; Gibson overhang to C-domain form TycC2; for construct 3 </TD><TD> GGTGTACTCGGTTTTTTCCGA AATATGCGCAGCCAACTCATG
</TD></TR><TR><TD> PW21:TycC4dC-C(TycC2)_fwd </TD><TD> 2013-08-16 </TD><TD> Amplification of C-domain from TycC2-module from <I>Brevibacillus parabrevis</I>; Gibson overhang to TycC4dC; for construct 3 </TD><TD> CATGAGTTGGCTGCGCATATT TCGGAAAAAACCGAGTACACC
</TD></TR><TR><TD> PW22:pSB1C3-TycC4dC_rev </TD><TD> 2013-08-16 </TD><TD> Insertion of construct 3 into pSB1C3-backbone, amplification of pSB1C3; Gibson overhang to TycC4dC; for construct 3 </TD><TD> GAATCGTCAGATCGCGCGGATACAT CTAGTATTTCTCCTCTTTCTCTAGTATGTG
</TD></TR><TR><TD> PW23:indC-pSB1C3_fwd </TD><TD> 2013-08-16 </TD><TD> Insertion of either fragments into pSB1C3-backbone, amplification of pSB1C3; Gibson overhang to indC from <I>Photorhabdus luminescens</I>; for constructs 1, 2 &amp; 3 </TD><TD> GGTGTTGCTGAGATTGAGAAAATAATCTAA TAATAACGCTGATAGTGCTAGTGTAGATC
</TD></TR><TR><TD> PW24:TycC1dC_fwd </TD><TD> 2013-08-16 </TD><TD> Amplification of the TycC1-module from <I>Brevibacillus parabrevis</I> without the C-domain; no Gibson overhang, ATG added; for construct 2 </TD><TD> ATGCAGACGAACAAACAACAGACG
</TD></TR><TR><TD> PW25:pSB1C3-TycC1dC </TD><TD> 2013-08-16 </TD><TD> Insertion of construct 2 in pSB1C3-backbone, amplification of pSB1C3; Gibson overhang to TycC1-module without C-domain; for construct 2 </TD><TD> CGTCTGTTGTTTGTTCGTCTGCAT CTAGTATTTCTCCTCTTTCTCTAGTATGTG
</TD></TR><TR><TD> PW26:TycC5dC_fwd </TD><TD> 2013-09-04 </TD><TD> Amplification of TycC5-module without C-domain from <I>Brevibacillus parabrevis</I>; no Gibson overhang, ATG added; for constructs A &amp; B </TD><TD> ATGCTGCACAGCTTCCTCGCAACC
</TD></TR><TR><TD> PW27:TycC5dC-C(TycC4)_rev </TD><TD> 2013-09-04 </TD><TD> Amplification of TycC5-module without C-domain from <I>Brevibacillus parabrevis</I>; Gibson overhang to C-domain from TycC4; for constructs A &amp; B </TD><TD> CACATACGTCTCTTTTCCGCTCGT TTCGATGAACGCCGCCAGTTC
</TD></TR><TR><TD> PW28:TycC5dC-C(TycC4)_fwd </TD><TD> 2013-09-04 </TD><TD> Amplification of C-domain from TycC4-module from <I>Brevibacillus parabrevis</I>; Gibson overhang to TycC5-module without C-domain; for constructs A &amp; B </TD><TD> GAACTGGCGGCGTTCATCGAA ACGAGCGGAAAAGAGACGTATGTG
</TD></TR><TR><TD> PW29:C(TycC4)-TycC4dC_rev </TD><TD> 2013-09-04 </TD><TD> Amplification of C-domain from TycC4-module from <I>Brevibacillus parabrevis</I>; Gibson overhang to TycC4-module without C-domain; for constructs A, B, C, E &amp; G </TD><TD> GAATCGTCAGATCGCGCGGATA GGCAAACGTGTTGTTGAAATC
</TD></TR><TR><TD> PW30:C(TycC4)-TycC4dC_fwd </TD><TD> 2013-09-04 </TD><TD> Amplification of TycC4-module without C-domain from <I>Brevibacillus parabrevis</I>; Gibson overhang to C-domain from TycC4-module; for constructs A, B, C, E &amp; G </TD><TD> GATTTCAACAACACGTTTGCC TATCCGCGCGATCTGACGATTC
</TD></TR><TR><TD> PW31:TycC4-C(TycC4)_rev </TD><TD> 2013-09-04 </TD><TD> Amplification of TycC4-module from <I>Brevibacillus parabrevis</I>; Gibson overhang to C-domain from TycC4-module; for constructs B &amp; C </TD><TD> CACATACGTCTCTTTTCCGCTCGT GGCAATATGCGCAGCCAACTCATG
</TD></TR><TR><TD> PW32:TycC4-C(TycC4)_fwd </TD><TD> 2013-09-04 </TD><TD> Amplification of C-domain from TycC4-module from <I>Brevibacillus parabrevis</I>; Gibson overhang to TycC4-module; for constructs B &amp; C </TD><TD> CATGAGTTGGCTGCGCATATTGCC ACGAGCGGAAAAGAGACGTATGTG
</TD></TR><TR><TD> PW33:TycC6dTE-C(TycC2)_rev </TD><TD> 2013-09-04 </TD><TD> Amplification of TycC6-module without the TE-domain from <I>Brevibacillus parabrevis</I>; Gibson overhang to C-domain from TycC2-module; for constructs D &amp; F </TD><TD> GGTGTACTCGGTTTTTTCCGA CGTGATGAAATCGGCCACCTTTTC
</TD></TR><TR><TD> PW34:TycC6dTE-C(TycC2)_fwd </TD><TD> 2013-09-04 </TD><TD> Amplification of C-domain from TycC2-module from <I>Brevibacillus parabrevis</I>; Gibson overhang to TycC6-module without TE-domain; for constructs D &amp; F </TD><TD> GAAAAGGTGGCCGATTTCATCACG TCGGAAAAAACCGAGTACACC
</TD></TR><TR><TD> PW35:TycC6dTE-C(TycC4)_rev </TD><TD> 2013-09-04 </TD><TD> Amplification of TycC6-module without the TE-domain from <I>Brevibacillus parabrevis</I>; Gibson overhang to C-domain from TycC4-module; for constructs E &amp; G </TD><TD> CACATACGTCTCTTTTCCGCTCGT CGTGATGAAATCGGCCACCTTTTC
</TD></TR><TR><TD> PW36:TycC6dTE-C(TycC4)_fwd </TD><TD> 2013-09-04 </TD><TD> Amplification of C-domain from TycC4-module from <I>Brevibacillus parabrevis</I>; Gibson overhang to TycC6-module without TE-domain; for constructs E &amp; G </TD><TD> GAAAAGGTGGCCGATTTCATCACG ACGAGCGGAAAAGAGACGTATGTG
</TD></TR><TR><TD> PW37:pSB1C3-TycC5dC_rev </TD><TD> 2013-09-04 </TD><TD> Insertion of constructs A &amp; B in pSB1C3-backbone, amplification of pSB1C3; Gibson overhang to TycC5-module without C-domain; for constructs A &amp; B </TD><TD> GGTTGCGAGGAAGCTGTGCAGCAT CTAGTATTTCTCCTCTTTCTCTAGTATGTG
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Linker_Variation">Linker Variation</SPAN></H2><TABLE class="wikitable sortable"><TBODY><TR><TH> Identifier </TH><TH> Order date </TH><TH> Note </TH><TH> Sequence
</TH></TR><TR><TD>AR01</TD><TD> 2013-09-17 </TD><TD> sTycC4dC_fw </TD><TD> ATG AAGCACTTGCTGCCGCTCGTC
</TD></TR><TR><TD>AR02</TD><TD> 2013-09-17 </TD><TD>c(TycC2)s_rev</TD><TD> TTGTGTAATATTATTTTCTAACAT CGCTACCTGTTGCAACCGCTC
</TD></TR><TR><TD>AR03</TD><TD> 2013-09-17 </TD><TD>s-Ind_fw</TD><TD> GAGCGGTTGCAACAGGTAGCG	ATGTTAGAAAATAATATTACAATGT
</TD></TR><TR><TD>AR04</TD><TD> 2013-09-17 </TD><TD>c(TycC2)l_rev_l</TD><TD>TTGTGTAATATTATTTTCTAACAT AAGCGACTGCTGGTTCGCG
</TD></TR><TR><TD>AR05</TD><TD> 2013-09-17 </TD><TD>l-Ind_fw</TD><TD>AACGCGAACCAGCAGTCGCTT	ATGTTAGAAAATAATATTACACAATGT
</TD></TR><TR><TD>AR06</TD><TD> 2013-09-17 </TD><TD>lTycC4dC_fw</TD><TD>ATG CGCAACTATCCGGTCGAGACG
</TD></TR><TR><TD>AR07</TD><TD> 2013-09-17 </TD><TD>TycC4dCom_short-BB_rev</TD><TD>GACGAGCGGCAGCAAGTGCTTCAT CTAGTATTTCTCCTCTTTCTCTAGTATGTG
</TD></TR><TR><TD>AR08</TD><TD> 2013-09-17 </TD><TD>TycC4dCom_long-BB_rev</TD><TD>CGTCTCGACCGGATAGTTGCGCAT CTAGTATTTCTCCTCTTTCTCTAGTATGTG
</TD></TR><TR><TD> PW19:TycC4dC_fwd </TD><TD> 2013-08-16 </TD><TD> Amplification of TycC4-module from <I>Brevibacillus parabrevis</I> without the C-domain; no Gibson overhang, ATG added; for construct 3 </TD><TD> ATGTATCCGCGCGATCTGACGATTC
</TD></TR><TR><TD> PW20:TycC4dC-C(TycC2)_rev </TD><TD> 2013-08-16 </TD><TD> Amplification of TycC4-module from <I>Brevibacillus parabrevis</I> without the C-domain; Gibson overhang to C-domain form TycC2; for construct 3 </TD><TD> GGTGTACTCGGTTTTTTCCGA AATATGCGCAGCCAACTCATG
</TD></TR><TR><TD> PW21:TycC4dC-C(TycC2)_fwd </TD><TD> 2013-08-16 </TD><TD> Amplification of C-domain from TycC2-module from <I>Brevibacillus parabrevis</I>; Gibson overhang to TycC4dC; for construct 3 </TD><TD> CATGAGTTGGCTGCGCATATT TCGGAAAAAACCGAGTACACC
</TD></TR><TR><TD> PW14:C(TycC2)-indC_rev </TD><TD> 2013-08-16 </TD><TD> Amplification of C-domain from TycC2 from <I>Brevibacillus parabrevis</I>; Gibson overhang to IndC; for construct 1, 2 &amp; 3 </TD><TD> ACATTGTGTAATATTATTTTCTAACAT CGTTTTGCTGCTGGCAGGCTG
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Domain_Shuffling_and_PPTases">Domain Shuffling and PPTases</SPAN></H2><TABLE class="wikitable sortable"><TBODY><TR><TH> Identifier </TH><TH> Order date </TH><TH> Note </TH><TH> Sequence
</TH></TR><TR><TD>	KH1_ccdb_fw	</TD><TD>	2013-07-23	</TD><TD>	amplifing ccdb with RBS from pDONR	</TD><TD>	TGGTGCCATTACATACAGATACT GAGAACAGGGGCTGGTGAAATGC
</TD></TR><TR><TD>	KH2_ccdb_rv	</TD><TD>	2013-07-23	</TD><TD>	amplifing ccdb with RBS from pDONR	</TD><TD>	AGAGTCTGTCTGTTCAATCCACTT TTATATTCCCCAGAACATCAGGTTAATGGCG
</TD></TR><TR><TD>	KH3_indC_backbone-withoutT_fw	</TD><TD>	2013-07-23	</TD><TD>	linearization of vector with partial indigoidine synthase	</TD><TD>	AAGTGGATTGAACAGACAGACTCTAAAAC
</TD></TR><TR><TD>	KH4_indC_backbone-withoutT_rv	</TD><TD>	2013-07-23	</TD><TD>	linearization of vector with partial indigoidine synthase	</TD><TD>	AGTATCTGTATGTAATGGCACCAATAGACGC
</TD></TR><TR><TD>	KH5_indC_T_fw	</TD><TD>	2013-07-23	</TD><TD>	introducing T domain of indC	</TD><TD>	TGGTGCCATTACATACAGATACT GAAATAAGGCTTGGAAAAATTTGGATGGAAGT
</TD></TR><TR><TD>	KH6_indC_T_rv	</TD><TD>	2013-07-23	</TD><TD>	introducing T domain of indC	</TD><TD>	AGAGTCTGTCTGTTCAATCCACTT AGCCAATTCTGCTATATTAGGAGATTGA
</TD></TR><TR><TD>	KH7_bpsA_T_fw	</TD><TD>	2013-07-23	</TD><TD>	introducing T domain of bpsA	</TD><TD>	TGGTGCCATTACATACAGATACT GAGAAGGAAATCGCAGCCGTGTGGG
</TD></TR><TR><TD>	KH8_bpsA_T_rv	</TD><TD>	2013-07-23	</TD><TD>	introducing T domain of bpsA	</TD><TD>	AGAGTCTGTCTGTTCAATCCACTT GGCCAGCTTTTCAATTGTTGGTGACTCC
</TD></TR><TR><TD>	KH9_ccdB-Big_fw	</TD><TD>	2013-08-06	</TD><TD>	ccdB_Big amplified from pDONOR (Dominik) for indC(ccdB) construct	</TD><TD>	TGCCATTACATACAGATACT ACTGGCTGTGTATAAGGGAGCCTGAC
</TD></TR><TR><TD>	KH10_ccdB-Big_rv	</TD><TD>	2013-08-06	</TD><TD>	ccdB_Big amplified from pDONOR (Dominik) for indC(ccdB) construct	</TD><TD>	AGAGTCTGTCTGTTCAATCCACTT CGCGTGGATCCGGCTTAC
</TD></TR><TR><TD>	KH11_plu2642-T_fw	</TD><TD>	2013-08-14	</TD><TD>	T domain exchange plu2642	</TD><TD>	TGGTGCCATTACATACAGATACT GAAACACAGATCGTAAAGATATGG
</TD></TR><TR><TD>	KH12_plu2642-T_rv	</TD><TD>	2013-08-14	</TD><TD>	T domain exchange plu2642	</TD><TD>	AGAGTCTGTCTGTTCAATCCACTT TGCCAATTGTTTAACCGTTG
</TD></TR><TR><TD>	KH13_BBa-entD_fw	</TD><TD>	2013-08-09	</TD><TD>	submitting endD to registry	</TD><TD>	GCAT GAATTCGCGGCCGCTTCTAG ATGAAAACTACGCATACCTCCCTC
</TD></TR><TR><TD>	KH14_BBa-entD_rv	</TD><TD>	2013-08-09	</TD><TD>	submitting endD to registry	</TD><TD>	GCAT CTGCAGCGGCCGCTACTAGTA TCATTAATCGTGTTGGCACAGCG
</TD></TR><TR><TD>	KH15_BBa-sfp_Bsub_fw	</TD><TD>	2013-08-09	</TD><TD>	submitting sfp_Bsub to registry	</TD><TD>	GCAT GAATTCGCGGCCGCTTCTAG ATGAAGATTTACGGAATTTATATGGAC
</TD></TR><TR><TD>	KH16_BBa-sfp_Bsub_rv	</TD><TD>	2013-08-09	</TD><TD>	submitting sfp_Bsub to registry	</TD><TD>	GCAT CTGCAGCGGCCGCTACTAGTA TCATTATAAAAGCTCTTCGTACGAGAC
</TD></TR><TR><TD>	KH17_BBa-svp_Svert_fw	</TD><TD>	2013-08-09	</TD><TD>	submitting svp_Svert to registry 	</TD><TD>	GCAT GAATTCGCGGCCGCTTCTAG A TGATCGCCGCCCTCCTGCCCTCC
</TD></TR><TR><TD>	KH18_BBa-svp_Svert_rv	</TD><TD>	2013-08-09	</TD><TD>	submitting svp_Svert to registry 	</TD><TD>	GCAT CTGCAGCGGCCGCTACTAGTA AGCTAGCTCATTACGGGAC
</TD></TR><TR><TD>	KH19_BBa-delC_fw	</TD><TD>	2013-08-09	</TD><TD>	submitting delC to registry	</TD><TD>	GCAT GAATTCGCGGCCGCTTCTAG ATGCAGCTCGTGTCCGTGCG
</TD></TR><TR><TD>	KH20_BBa-delC_rv	</TD><TD>	2013-08-09	</TD><TD>	submitting delC to registry	</TD><TD>	GCAT CTGCAGCGGCCGCTACTAGTA TTA TCATGTCGATTCCTTGGTGC
</TD></TR><TR><TD>	KH21_BBa-indC_fw	</TD><TD>	2013-08-09	</TD><TD>	submitting indC to registry	</TD><TD>	GCAT GAATTCGCGGCCGCTTCTAG ATGTTAGAAAATAATATTACACAATG
</TD></TR><TR><TD>	KH22_BBa-indC_rv	</TD><TD>	2013-08-09	</TD><TD>	submitting indC to registry	</TD><TD>	GCAT CTGCAGCGGCCGCTACTAGTA TTAGATTATTTTCTCAATCTCAG
</TD></TR><TR><TD> NI01:bpsA_AOxA_PstI_fw</TD><TD> 2013-06-04 </TD><TD> for amplifying bpsA (AOxA domain) </TD><TD> GAGGAGAAATACTAGATGACACTGCAGGAAACAAGCGTGC
</TD></TR><TR><TD> NI02:bpsA_AOxA_rv</TD><TD> 2013-06-04 </TD><TD> for amplifying bpsA (AOxA domain)  </TD><TD> GAAGGGCCGTTCCACCAGCTCAGCGTTGACCTGGTCAGAAGCG
</TD></TR><TR><TD> NI03:bpsA_T_fw</TD><TD> 2013-06-04 </TD><TD> for amplifying bpsA (T domain) </TD><TD> GACCAGGTCAACGCTGAGCTGGTGGAACGGCCCTTCGTCG
</TD></TR><TR><TD> NI04:bpsA_T_rv</TD><TD> 2013-06-04 </TD><TD> for amplifying bpsA (T domain) </TD><TD> GACGAAGCGACTAGACTCCTGAGCCACTTCTCTCTCCAGCCG
</TD></TR><TR><TD> NI05:bpsA_TE_fw</TD><TD> 2013-06-04 </TD><TD> for amplifying bpsA (TE domain) with RBS 1 </TD><TD> GAGAGAGAAGTGGCTCAGGAGTCTAGTCGCTTCGTCCGA
</TD></TR><TR><TD> NI06:bpsA_TE_Stop_XbaI_rbs_rv</TD><TD> 2013-06-04 </TD><TD> for amplifying bpsA (TE domain) with RBS 1 </TD><TD> TGGCAGCAGAGCAGCGATCATCTAGTATTT CTCCTCTTTCCTCTAGATCATCATTCCC CCAGCAGGTATCTAAT
</TD></TR><TR><TD> NI07:svp_fw</TD><TD> 2013-06-04 </TD><TD> for amplifying svp </TD><TD> GAGGAGAAATACTAGATGATCGCTGCTCTGCTGCCAAGTTGG
</TD></TR><TR><TD> NI08:svp_rv</TD><TD> 2013-06-04 </TD><TD> for amplifying svp </TD><TD> GCACTATCAGCGTTATTATGGCACGGCAGTCCTATCGTCG
</TD></TR><TR><TD> NI09:pSB1C3_fw</TD><TD> 2013-06-04 </TD><TD> for linearizing, amplifying pSB1C3 </TD><TD> GATAGGACTGCCGTGCCATAATAA CGCTGATAGTGCTAGTGTAGATCGC
</TD></TR><TR><TD> NI10:pSB1C3_PstI_rv</TD><TD> 2013-06-04 </TD><TD> for linearizing, amplifying pSB1C3 </TD><TD> GCTTGTTTCCTGCAGTGTCATCTAG TATTTCTCCTCTTTCTCTAGTATGTG
</TD></TR><TR><TD> NI11:bpsA_rvN</TD><TD> 2013-07-15 </TD><TD> for amplifying bpsA (TE domain) with RBS 2 </TD><TD> TGGCAGCAGAGCAGCGATCATTATTTAGGTTTCCTGTGTGAATCATCATTCCCCCAGCAGGTATCTAAT
</TD></TR><TR><TD> NI12:svp_fwN</TD><TD> 2013-07-15 </TD><TD> for amplifying svp with RBS 2  </TD><TD> CAGGAAACCTAAATAATGATC GCTGCTCTGCTGCCAAGTTGG
</TD></TR><TR><TD>	NI13_ccdb_fw	</TD><TD>	2013-07-23	</TD><TD>	amplifing ccdb with RBS from pDONR	</TD><TD>	CGTCGCACCTAGGACCGAAACA GAGAACAGGGGCTGGTGAAATGC
</TD></TR><TR><TD>	NI14_ccdb_rv	</TD><TD>	2013-07-23	</TD><TD>	amplifing ccdb with RBS from pDONR	</TD><TD>	GCCACTTCTCTCTCCAGCCGTCG TTATATTCCCCAGAACATCAGGTTAATGGCG
</TD></TR><TR><TD>	NI15_bpsA_backbone-withoutT_fw	</TD><TD>	2013-07-23	</TD><TD>	linearization of vector with partial indigoidine synthase	</TD><TD>	CGACGGCTGGAGAGAGAAGTGGC
</TD></TR><TR><TD>	NI16_bpsA_backbone-withoutT_rv	</TD><TD>	2013-07-23	</TD><TD>	linearization of vector with partial indigoidine synthase	</TD><TD>	TC TGTTTCGGTCCTAGGTGCGACG
</TD></TR><TR><TD>	NI17_indC_T_fw	</TD><TD>	2013-07-23	</TD><TD>	introducing T domain of indC	</TD><TD>	CGTCGCACCTAGGACCGAAACA GAAATAAGGCTTGGAAAAATTTGGATGGAAGT
</TD></TR><TR><TD>	NI18_indC_T_rv	</TD><TD>	2013-07-23	</TD><TD>	introducing T domain of indC	</TD><TD>	GCCACTTCTCTCTCCAGCCGTCG AGCCAATTCTGCTATATTAGGAGATTGA
</TD></TR><TR><TD>	NI19_bpsA_T_fw	</TD><TD>	2013-07-23	</TD><TD>	introducing T domain of bpsA	</TD><TD>	CGTCGCACCTAGGACCGAAACA GAGAAGGAAATCGCAGCCGTGTGGG
</TD></TR><TR><TD>	NI20_bpsA_T_rv	</TD><TD>	2013-07-23	</TD><TD>	introducing T domain of bpsA	</TD><TD>	GCCACTTCTCTCTCCAGCCGTCG GGCCAGCTTTTCAATTGTTGGTGACTCC
</TD></TR><TR><TD>	NI19:ngrA_fw	</TD><TD>	GAATTCGCGGCCGCTTCTAG ATGGAACAGACAGTTATACACACC
</TD></TR><TR><TD>	NI20:ngrA_rv	</TD><TD>	CTGCAGCGGCCGCTACTAGTA TTATTGAATAATTAGCGTTAATATTTCCTG
</TD></TR><TR><TD>RB05:RBS1-BpsA_fw</TD><TD>2013-07-01</TD><TD>Gibson assembly of pSB1C3 bpsA svp</TD><TD>AAAGAGGAGAAA TACTAG ATGACACTTCAGGAAACCAGCG
</TD></TR><TR><TD>RB06:BpsA-RBS2_rv</TD><TD>2013-07-01</TD><TD>Gibson assembly of pSB1C3 bpsA svp</TD><TD>CTTTCCTGTGTGA AAGCTT TCA TCACTCCCCGAGCAGATATCG
</TD></TR><TR><TD>RB07:RBS2-svp_fw</TD><TD>2013-07-01</TD><TD>Gibson assembly of pSB1C3 bpsA svp</TD><TD>CTTTCACACAGGAAAG TAAATA ATGGCTGCTCTTCTTCCTAGTTGGGC
</TD></TR><TR><TD>RB08:svp-pSB1C3_rv</TD><TD>2013-07-01</TD><TD>Gibson assembly of pSB1C3 bpsA svp</TD><TD>CTATCAGCGTTATTA AAGCTT TCATCA TGGCACGGCAGTCCTATCG
</TD></TR><TR><TD>RB09:pSB1C3_fw</TD><TD>2013-07-01</TD><TD>Gibson assembly of pSB1C3 bpsA svp</TD><TD>AAGCTT TAATAACGCTGATAGTGCTAGTGTAGATCGC
</TD></TR><TR><TD>RB10:pSB1C3-RBS1_rv</TD><TD>2013-07-01</TD><TD>Gibson assembly of pSB1C3 bpsA svp</TD><TD>CTAGTA TTTCTCCTCTTT CTCTAGTATGTGTG
</TD></TR><TR><TD>RB11:BpsA_Taka_fw</TD><TD>2013-07-08</TD><TD>colony PCR bpsA Takahashi</TD><TD>CAT ATGACTCTTCAGGAGACCAGCGTGCTC
</TD></TR><TR><TD>RB12:BpsA_Taka_rv</TD><TD>2013-07-08</TD><TD>colony PCR bpsA Takahashi</TD><TD>AAG CTTCTCGCCGAGCAGGTAGCGGATGTG
</TD></TR><TR><TD>RB13:svp_Taka_fw</TD><TD>2013-07-08</TD><TD>colony PCR svp Takahashi</TD><TD>CAT ATGATCGCCGCCCTCCTGCCCTCCTG
</TD></TR><TR><TD>RB14:svp_Taka_rv</TD><TD>2013-07-08</TD><TD>colony PCR svp Takahashi</TD><TD>CTCGAGCGGGACGGCGGTCCGGTCGTCCGC
</TD></TR><TR><TD>RB15:svp_Sanch_fw</TD><TD>2013-07-08</TD><TD>PCR svp Sanchez</TD><TD>TATA ATGCATGCTCGCCGCCCTCCCC
</TD></TR><TR><TD>RB16:svp_Sanch_rv</TD><TD>2013-07-08</TD><TD>PCR svp Sanchez</TD><TD>TTAAGATCTCGGGACGGCGGTCCGGTC
</TD></TR><TR><TD>RB17:sfp_fw</TD><TD>2013-07-08</TD><TD>sfp extraction from <I>E.coli</I> BAP1</TD><TD>ATGAAGATTTACGGAATTTATATGG
</TD></TR><TR><TD>RB18:sfp_rv</TD><TD>2013-07-08</TD><TD>sfp extraction from <I>E.coli</I> BAP1</TD><TD>TTATAAAAGCTCTTCGTACGAGACC
</TD></TR><TR><TD>RB19:entD_fw</TD><TD>2013-07-08</TD><TD>entD extraction from <I>E.coli</I> Lambalot</TD><TD>TAAATA ATGGTCGATATGAAAACTACGC
</TD></TR><TR><TD>RB20:entD_rv</TD><TD>2013-07-08</TD><TD>entD extraction from <I>E.coli</I> Lambalot</TD><TD>AAGCTT ATTAATCGTGTTGGCACAGCG
</TD></TR><TR><TD>	RB21:pSB1C3_fw	</TD><TD>	2013-07-15	</TD><TD>	indigoidine exchangeable construct	</TD><TD>	TAATGA GCTAGC TAATAACGCTGATAGTGCTAGTG
</TD></TR><TR><TD>	RB22:pSB1C3_rv	</TD><TD>	2013-07-15	</TD><TD>	indigoidine exchangeable construct	</TD><TD>	CAT GGTACC TTTCTCCTCTTT CTCTAGTATGTGTG
</TD></TR><TR><TD>	RB23:bpsA-fus_fw	</TD><TD>	2013-07-15	</TD><TD>	indigoidine exchangeable construct	</TD><TD>	CACACATACTAGAG AAAGAGGAGAAA GGTACC ATGACTAGTACACTGCAGG
</TD></TR><TR><TD>	RB24:bpsA-fus_rv	</TD><TD>	2013-07-15	</TD><TD>	indigoidine exchangeable construct	</TD><TD>	CAT GGATCC GGTTTCCTGTGTGAA TCATTA TTCCCCCAGCAGGTATCTAATATG
</TD></TR><TR><TD>	RB25:svp-fus_fw	</TD><TD>	2013-07-15	</TD><TD>	indigoidine exchangeable construct	</TD><TD>	CACAGGAAACC GGATCC ATGACTAGTATCGCTGCTCTGCTG
</TD></TR><TR><TD>	RB26:svp-fus_rv	</TD><TD>	2013-07-15	</TD><TD>	indigoidine exchangeable construct	</TD><TD>	CAGCGTTATTA GCTAGC TCATTA TGGCACGGCAGTCCTATC
</TD></TR><TR><TD>	RB27:indC-Plum_fw	</TD><TD>	2013-07-15	</TD><TD>	indigoidine exchangeable construct	</TD><TD>	CATACTAGAG AAAGAGGAGAAA GGTACC ATGTTAGAAAATAATATTACACAATG
</TD></TR><TR><TD>	RB28:indC-Plum_rv	</TD><TD>	2013-07-15	</TD><TD>	indigoidine exchangeable construct	</TD><TD>	CAT GGATCC GGTTTCCTGTGTGAA TTA TTAGATTATTTTCTCAATCTCAG
</TD></TR><TR><TD>	RB29:svp-Svert_fw	</TD><TD>	2013-07-15	</TD><TD>	indigoidine exchangeable construct	</TD><TD>	CACAGGAAACC GGATCC A TGATCGCCGCCCTC
</TD></TR><TR><TD>	RB30:svp-Svert_rv	</TD><TD>	2013-07-15	</TD><TD>	indigoidine exchangeable construct	</TD><TD>	CAGCGTTATTA GCTAGC TCA TTACGGGACGGCGGTC
</TD></TR><TR><TD>	RB31:Sc-indC-Schr_fw	</TD><TD>	2013-07-15	</TD><TD>	indigoidine exchangeable construct	</TD><TD>	GAG AAAGAGGAGAAA GGTACC ATGAGCGTAGAGACCATC
</TD></TR><TR><TD>	RB32:Sc-indC-Schr_rv	</TD><TD>	2013-07-15	</TD><TD>	indigoidine exchangeable construct	</TD><TD>	CAT AGATCT GGTTTCCTGTGTGAA TTA TCAGTAGTTGGGCGTCTTG
</TD></TR><TR><TD>	RB33:entD-MG1655_fw	</TD><TD>	2013-07-15	</TD><TD>	indigoidine exchangeable construct	</TD><TD>	CACAGGAAACC GGATCC ATGAAAACTACGCATACCTC
</TD></TR><TR><TD>	RB34:entD-MG1655_rv	</TD><TD>	2013-07-15	</TD><TD>	indigoidine exchangeable construct	</TD><TD>	CAGCGTTATTA GCTAGC TCA TTAATCGTGTTGGCACAGC
</TD></TR><TR><TD>	RB35:sfp-Naka_fw	</TD><TD>	2013-07-15	</TD><TD>	indigoidine exchangeable construct	</TD><TD>	CACAGGAAACC GGATCC ATGAAGATTTACGGAATTTATATGGAC
</TD></TR><TR><TD>	RB36:sfp-Naka_rv	</TD><TD>	2013-07-15	</TD><TD>	indigoidine exchangeable construct	</TD><TD>	CAGCGTTATTA GCTAGC TCA TTATAAAAGCTCTTCGTACGAGAC
</TD></TR><TR><TD>	RB37:Plum-extr_fw	</TD><TD>	2013-07-15	</TD><TD>	genomic extraction	</TD><TD>	ATGTTAGAAAATAATATTACACAATG
</TD></TR><TR><TD>	RB38:Plum-extr_rv	</TD><TD>	2013-07-15	</TD><TD>	genomic extraction	</TD><TD>	TTAGATTATTTTCTCAATCTCAG
</TD></TR><TR><TD>	RB39:Svert-extr_fw	</TD><TD>	2013-07-15	</TD><TD>	genomic extraction	</TD><TD>	GTGATCGCCGCCCTC
</TD></TR><TR><TD>	RB40:Svert-extr_rv	</TD><TD>	2013-07-15	</TD><TD>	genomic extraction	</TD><TD>	TTACGGGACGGCGGTC
</TD></TR><TR><TD>	RB41:MG1655-extr_fw	</TD><TD>	2013-07-15	</TD><TD>	genomic extraction	</TD><TD>	ATGAAAACTACGCATACCTCCCTC
</TD></TR><TR><TD>	RB42:MG1655-extr_rv	</TD><TD>	2013-07-15	</TD><TD>	genomic extraction	</TD><TD>	TTAATCGTGTTGGCACAGCGTTATG
</TD></TR><TR><TD>	RB43:Bsub-extr_fw	</TD><TD>	2013-07-15	</TD><TD>	genomic extraction	</TD><TD>	ATGAAGATTTACGGAATTTATATGGAC
</TD></TR><TR><TD>	RB44:Bsub-extr_rv	</TD><TD>	2013-07-15	</TD><TD>	genomic extraction	</TD><TD>	TTATAAAAGCTCTTCGTACGAGAC
</TD></TR><TR><TD>	RB45:Sc-indC-Schr_rvG	</TD><TD>	2013-07-15	</TD><TD>	indigoidine exchangeable construct	</TD><TD>	CAT GGATCC GGTTTCCTGTGTGAA TTA TCAGTAGTTGGGCGTCTTG
</TD></TR><TR><TD>	RB46:indC-woPPT_rv	</TD><TD>	2013-08-05	</TD><TD>	indC-pSB1C3 w/o PPTase; RB21 overlap	</TD><TD>	GCACTATCAGCGTTATTA GCTAGCTCATTA TTA TTAGATTATTTTCTCAATCTCAG
</TD></TR><TR><TD>	RB47:indC-SpeI_fw	</TD><TD>	2013-08-05	</TD><TD>	remove cutting sites from indC-Plum	</TD><TD>	CAAGTTCCTAAACCC ACTAGT CTGGCTTAT ATTATTTATACCTCTGGTAGCAC
</TD></TR><TR><TD>	RB48:indC-EcoRI_rv	</TD><TD>	2013-08-05	</TD><TD>	remove cutting sites from indC-Plum	</TD><TD>	CAATACCCACC GAATTC TTTGAGCT AATTTCTGACAGACAATACC
</TD></TR><TR><TD>	RB49:indC-EcoRI_fw	</TD><TD>	2013-08-05	</TD><TD>	remove cutting sites from indC-Plum	</TD><TD>	AGCTCAAA GAATTC GGTGGGTATTG GGCTTTTTTGTGATC
</TD></TR><TR><TD>	RB50:indC-SpeI_rv	</TD><TD>	2013-08-05	</TD><TD>	remove cutting sites from indC-Plum	</TD><TD>	ATAAGCCAG ACTAGT GGGTTTAGGAACTTG GAACTTGAACTGTG
</TD></TR><TR><TD>	RB51:pSB(2/3)K3_fw	</TD><TD>	2013-08-05	</TD><TD>	PPTase plasmid	</TD><TD>	TAATGA GCTAGC tactagtagcggccgctgcagtc
</TD></TR><TR><TD>	RB52:pSB(2/3)K3_rv	</TD><TD>	2013-08-05	</TD><TD>	PPTase plasmid	</TD><TD>	CAT GGATCC GGTTTCCTGTGTGAA ctctagaagcggccgcgaattcc
</TD></TR><TR><TD>	RB53:entF-T_fw	</TD><TD>	2013-08-07	</TD><TD>	bring T-Domain in indC	</TD><TD>	TGGTGCCATTACATACAGATACT GAAACGATTATCGCCGCGGCATTC
</TD></TR><TR><TD>	RB54:entF-T_rv	</TD><TD>	2013-08-07	</TD><TD>	bring T-Domain in indC	</TD><TD>	AGAGTCTGTCTGTTCAATCCACTT TGCCAGTTTGGCGACAGTTGACG
</TD></TR><TR><TD>	RB55:tycA1-T_fw	</TD><TD>	2013-08-07	</TD><TD>	bring T-Domain in indC	</TD><TD>	TGGTGCCATTACATACAGATACT GAATCGATTCTCGTCTCCATCTGG
</TD></TR><TR><TD>	RB56:tycA1-T_rv	</TD><TD>	2013-08-07	</TD><TD>	bring T-Domain in indC	</TD><TD>	AGAGTCTGTCTGTTCAATCCACTT AGCAACCTGCTCGATCGTCGGGTAATTC
</TD></TR><TR><TD>	RB57:tycC6-T_fw	</TD><TD>	2013-08-07	</TD><TD>	bring T-Domain in indC	</TD><TD>	TGGTGCCATTACATACAGATACT GAACAGCAACTGGCAGCCATCTGGCAAG
</TD></TR><TR><TD>	RB58:tycC6-T_rv	</TD><TD>	2013-08-07	</TD><TD>	bring T-Domain in indC	</TD><TD>	AGAGTCTGTCTGTTCAATCCACTT GGCCACCTTTTCGATCGTCGGATACTG
</TD></TR><TR><TD>	RB59:delH4-T_fw	</TD><TD>	2013-08-07	</TD><TD>	bring T-Domain in indC	</TD><TD>	TGGTGCCATTACATACAGATACT GAGACGCTGCTGGCCCGTATCTG
</TD></TR><TR><TD>	RB60:delH4-T_rv	</TD><TD>	2013-08-07	</TD><TD>	bring T-Domain in indC	</TD><TD>	AGAGTCTGTCTGTTCAATCCACTT CGCCAGCTCGGCAATGCTTTGC
</TD></TR><TR><TD>	RB61:delH5-T_fw	</TD><TD>	2013-08-07	</TD><TD>	bring T-Domain in indC	</TD><TD>	TGGTGCCATTACATACAGATACT GCCATGGCGCTGGCCCGCATC
</TD></TR><TR><TD>	RB62:delH5-T_rv	</TD><TD>	2013-08-07	</TD><TD>	bring T-Domain in indC	</TD><TD>	AGAGTCTGTCTGTTCAATCCACTT CAGCAGGCCGGCAATGGTCG
</TD></TR><TR><TD>	RB63:pSB(2/3)K3_rv_korr	</TD><TD>	2013-08-07	</TD><TD>	PPTase plasmid	</TD><TD>	CAT GGATCC GGTTTCCTGTGTGAA CTCTAGTATGTGTGAAATTGTTATCC
</TD></TR><TR><TD>	RB64:DelC-extr_fw	</TD><TD>	2013-08-07	</TD><TD>	get Delftia PPTase	</TD><TD>	ATGCAGCTCGTGTCCGTGCGTGAG
</TD></TR><TR><TD>	RB65:DelC-extr_rv	</TD><TD>	2013-08-07	</TD><TD>	get Delftia PPTase	</TD><TD>	TCATGTCGATTCCTTGGTGCGCCAC
</TD></TR><TR><TD>	RB66:DelC-pSB2K3_fw	</TD><TD>	2013-08-07	</TD><TD>	DelC in PPTase-construct	</TD><TD>	CACAGGAAACC GGATCC ATGCAGCTCGTGTCCGTGCGTGAG
</TD></TR><TR><TD>	RB67:DelC-pSB2K3_rv	</TD><TD>	2013-08-07	</TD><TD>	DelC in PPTase-construct	</TD><TD>	CAGCGTTATTA GCTAGC TCA TCATGTCGATTCCTTGGTGCGCCAC
</TD></TR><TR><TD>	RB68:indC-SpeI_fw_corr	</TD><TD>	2013-08-09	</TD><TD>	indC w/o cutting sites corrected	</TD><TD>	CAAGTTCCTAAACCC ACgAGT CTGGCTTAT ATTATTTATACCTCTGGTAGCAC
</TD></TR><TR><TD>	RB69:indC-EcoRI_rv_corr	</TD><TD>	2013-08-09	</TD><TD>	indC w/o cutting sites corrected	</TD><TD>	CAATACCCACC GAAcTC TTTGAGCT AATTTCTGACAGACAATACC
</TD></TR><TR><TD>	RB70:indC-EcoRI_fw_corr	</TD><TD>	2013-08-09	</TD><TD>	indC w/o cutting sites corrected	</TD><TD>	AGCTCAAA GAgTTC GGTGGGTATTG GGCTTTTTTGTGATC
</TD></TR><TR><TD>	RB71:indC-SpeI_rv_corr	</TD><TD>	2013-08-09	</TD><TD>	indC w/o cutting sites corrected	</TD><TD>	ATAAGCCAG ACTcGT GGGTTTAGGAACTTG GAACTTGAACTGTG
</TD></TR><TR><TD>	RB72:bpsA-TTE_rv	</TD><TD>	2013-08-09	</TD><TD>	exchange T and TE-domain	</TD><TD>	GCACTATCAGCGTTATTA GCTAGC TCATTA TTCCCCCAGCAGGTATCTAATATG
</TD></TR><TR><TD>	RB73:entF-TTE_rv	</TD><TD>	2013-08-09	</TD><TD>	exchange T and TE-domain	</TD><TD>	GCACTATCAGCGTTATTA GCTAGC TCA TTACCTGTTTAGCGTTGCGCGAATAATCG
</TD></TR><TR><TD>	RB74:tycC6-TTE_rv	</TD><TD>	2013-08-09	</TD><TD>	exchange T and TE-domain	</TD><TD>	GCACTATCAGCGTTATTA GCTAGC TCA TTATTTCAGGATGAACAGTTCTTGC
</TD></TR><TR><TD>	RB75:delH5-TTE_rv	</TD><TD>	2013-08-09	</TD><TD>	exchange T and TE-domain	</TD><TD>	GCACTATCAGCGTTATTA GCTAGC TCATTA TCAGTCCAGCGCGTACTCCAG
</TD></TR><TR><TD>	RB76:plu2670-T_fw	</TD><TD>	2013-08-15	</TD><TD>	bring T-Domain in indC	</TD><TD>	TGGTGCCATTACATACAGATACT GAAACCACACTGGCTGCTATC
</TD></TR><TR><TD>	RB77:plu2670-T_rv	</TD><TD>	2013-08-15	</TD><TD>	bring T-Domain in indC	</TD><TD>	AGAGTCTGTCTGTTCAATCCACTT CGCAAATGCGGATAACACC
</TD></TR><TR><TD>	RB78:indC-SpeI_rv_corr2	</TD><TD>	2013-08-26	</TD><TD>	indC w/o cutting sites w/o frameshift	</TD><TD>	ATAAGCCAG ACTcGT GGGTTTAGGAACTTG AACTGTG
</TD></TR><TR><TD>	RB79:indC_woPPTase_rv2	</TD><TD>	2013-08-29	</TD><TD>	indC-pSB1C3 w/o PPTase; RB35 5' part	</TD><TD>	CAGCGTTATTA GCTAGC TCA TTAGATTATTTTCTCAATCTCAG
</TD></TR><TR><TD>	RB80:indCdT_3_fw	</TD><TD>	2013-09-06	</TD><TD>	GGCGGATATCCTATGAGTTTGAGATTGC
</TD></TR><TR><TD>	RB81:indCdT_3_rv	</TD><TD>	2013-09-06	</TD><TD>	TCCATTGGCCGTCAAAGGTAATTTATCG
</TD></TR><TR><TD>	RB82:bpsA-T3_fw	</TD><TD>	2013-09-06	</TD><TD>	CGATAAATTACCTTTGACGGCCAATGGA AAGATCGATGTGAAAGCACTGGCCGCTTCTGACC
</TD></TR><TR><TD>	RB83:bpsA-T3_rv	</TD><TD>	2013-09-06	</TD><TD>	GCAATCTCAAACTCATAGGATATCCGCC CAGACCTGGCCAGCAGATCACAGG
</TD></TR><TR><TD>	RB84:indCdT_4_fw	</TD><TD>	2013-09-06	</TD><TD>	TTTGAAGTTGCATACCAGCTTGAACAAGC
</TD></TR><TR><TD>	RB85:bpsA-T4_rv	</TD><TD>	2013-09-06	</TD><TD>	GCTTGTTCAAGCTGGTATGCAACTTCAAA TGCCACGCGAGCTCCGAAGCTATATCC
</TD></TR><TR><TD>	RB86:indC-T2_fw	</TD><TD>	2013-09-06	</TD><TD>	TTACAGATACTTTTTCAATCTCCTAATATAGCAG
</TD></TR><TR><TD>	RB87:indC-T2_rv	</TD><TD>	2013-09-06	</TD><TD>	TACTTCCATCCAAATTTTTCCAAGC
</TD></TR><TR><TD>	RB88:bpsA-T2_fw	</TD><TD>	2013-09-06	</TD><TD>	GCTTGGAAAAATTTGGATGGAAGTA CTGAGACGCGAAAATGCTAGTGTCC
</TD></TR><TR><TD>	RB89:bpsA-T2_rv	</TD><TD>	2013-09-06	</TD><TD>	TTAGGAGATTGAAAAAGTATCTGTAA AGGCAGGGACACTCCCAGTCTAGCATTCAG
</TD></TR><TR><TD>	RB90:indC-T-screen_fw	</TD><TD>	2013-09-13	</TD><TD>	GATCAATGCGGCCTTTAATATTCG
</TD></TR><TR><TD>	RB91:bpsA-T-screen_fw	</TD><TD>	2013-09-13	</TD><TD>	GGAACTGAATGCTAGACTGGGAG
</TD></TR><TR><TD>	RB92:entF-T3_fw	</TD><TD>	2013-09-16	</TD><TD>	CGATAAATTACCTTTGACGGCCAATGGA AAGCTGGATCGCAAAGCCTTACCGTTG
</TD></TR><TR><TD>	RB93:tycA1-T3_fw	</TD><TD>	2013-09-16	</TD><TD>	CGATAAATTACCTTTGACGGCCAATGGA AAGATCGACCGCAAAGCGTTGC
</TD></TR><TR><TD>	RB94:tycC6-T3_fw	</TD><TD>	2013-09-16	</TD><TD>	CGATAAATTACCTTTGACGGCCAATGGA AAAGTGGATCGCAAAGCTTTG
</TD></TR><TR><TD>	RB95:delH4-T3_fw	</TD><TD>	2013-09-16	</TD><TD>	CGATAAATTACCTTTGACGGCCAATGGA AAGCTGGACCGGCAGGCCCTG
</TD></TR><TR><TD>	RB96:delH5-T3_fw	</TD><TD>	2013-09-16	</TD><TD>	CGATAAATTACCTTTGACGGCCAATGGA AAGCTTGACCGTGGTGCGCTG
</TD></TR><TR><TD>	RB97:plu2642-T3_fw	</TD><TD>	2013-09-16	</TD><TD>	CGATAAATTACCTTTGACGGCCAATGGA AAAATCGATTTCGACACATTACAAG
</TD></TR><TR><TD>	RB98:plu2670-T3_fw	</TD><TD>	2013-09-16	</TD><TD>	CGATAAATTACCTTTGACGGCCAATGGA AAGCTGGACCGTCGGGCGTTACCGGCAC
</TD></TR><TR><TD>	RB99:plu2642_pRB23-plu_fw	</TD><TD>	2013-09-24	</TD><TD>	plu2642 on pSB1C3	</TD><TD>	CATACTAGAG AAAGAGGAGAAA GGTACC ATGCAATCAACTCTCCCAATAATAAAATGG
</TD></TR><TR><TD>	RB100:plu2642_pRB23-plu_rv	</TD><TD>	2013-09-24	</TD><TD>	plu2642 on pSB1C3	</TD><TD>	CAGCGTTATTA GCTAGC TCA TCAACTCAAGAATCGAGCTAATTCGTTAACAC
</TD></TR><TR><TD>	RB101:plu2642_A_rv	</TD><TD>	2013-09-24	</TD><TD>	GTTCTGGCATCCTTCTTAATTAACAT TGCAGCGTGTTTCACTTGATCGATAAC
</TD></TR><TR><TD>	RB102:indC_Ox_fw	</TD><TD>	2013-09-24	</TD><TD>	ATGTTAATTAAGAAGGATGCCAGAAC
</TD></TR><TR><TD>	RB103:plu2642_valInd_fw	</TD><TD>	2013-09-24	</TD><TD>	CAGCCTGCCAGCAGCAAAACG ATGCAATCAACTCTCCCAATAATAAAATGG
</TD></TR><TR><TD>	RB104:plu2642_valInd_rv	</TD><TD>	2013-09-24	</TD><TD>	CAT CGTTTTGCTGCTGGCAGGCTG
</TD></TR><TR><TD>	RB105:tycC2_T3_fw	</TD><TD>	2013-09-24	</TD><TD>	CGATAAATTACCTTTGACGGCCAATGGA AAAGTGGATCGCAAGGCATTG
</TD></TR><TR><TD>	RB106:tycC2_T1_rv	</TD><TD>	2013-09-24	</TD><TD>	AGAGTCTGTCTGTTCAATCCACTT GGCCAGGCCCGCGATCGTTGG
</TD></TR><TR><TD>	RB107:tycC2_A_fw	</TD><TD>	2013-09-24	</TD><TD>	CATACTAGAG AAAGAGGAGAAA GGTACC ATGACGGAAGCGGAAAAACGCACACTCCTTC
</TD></TR><TR><TD>	RB108:tycC2_A_rv	</TD><TD>	2013-09-24	</TD><TD>	GTTCTGGCATCCTTCTTAATTAACAT GACGACCGCTTTGACCGCTTCATG
</TD></TR><TR><TD>	RB109:tycC2_valInd_fw	</TD><TD>	2013-09-24	</TD><TD>	CAGCCTGCCAGCAGCAAAACG ATGACGGAAGCGGAAAAACGCACACTCCTTC
</TD></TR><TR><TD>	RB110:plu2642_TE_fw	</TD><TD>	AGTGGATTGAACAGACAGACTCT AAAATCGTTGAAGGTGAAGTAACCAG
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Plasmids">Plasmids</SPAN></H1><H2><SPAN class="mw-headline" id="Delftibactin_2">Delftibactin</SPAN></H2><TABLE class="wikitable sortable"><TBODY><TR><TH>Name</TH><TH>Date</TH><TH>Brief Description</TH><TH>Genotype</TH><TH>Plasmid Map</TH><TH>GenBank-File
</TH></TR><TR><TD>pIK1</TD><TD>2013-08-12</TD><TD>Methylmalonyl-CoA pathway with permeability device and sfp</TD><TD>pSB3C5-lacP-BBa_B0034-BBa_I746200-BBa_B0029-pcc-acca2-BBa_B0030-sfp</TD><TD><A href="/wiki/images/1/19/Heidelberg_PIK1.gb.txt" class="internal" title="Heidelberg PIK1.gb.txt">pIK1</A></TD></TR><TR><TD>pIK2</TD><TD>2013-08-12</TD><TD>Methylmalonyl-CoA pathway with sfp</TD><TD>pSB3C5-lacP-BBa_B0034-pcc-acca2-BBa_B0030-sfp</TD><TD><A href="/wiki/images/a/aa/Heidelberg_PIK2.gb.txt" class="internal" title="Heidelberg PIK2.gb.txt">pIK2</A></TD></TR><TR><TD>pIK7</TD><TD>2013-08-28</TD><TD>Methylmalonyl-CoA pathway with sfp</TD><TD>pSB3K3-lacP-BBa_B0034-pcc-acca2-BBa_B0030-sfp</TD><TD><A href="/wiki/images/3/35/Heidelberg_PIK7.gb.txt" class="internal" title="Heidelberg PIK7.gb.txt">pIK7</A></TD></TR><TR><TD>pIK8</TD><TD>2013-09-12</TD><TD>Methylmalonyl-CoA pathway with sfp and the permeability device behind a weak promoter</TD><TD>pSB3C5-lacP-BBa_B0034-pcc-acca2-BBa_B0030-sfp-BBa_B0010-BBa_B0030-BBa_I746200-BBa_B0012</TD><TD><A href="/wiki/images/5/50/Heidelberg_PIK8.gb.txt" class="internal" title="Heidelberg PIK8.gb.txt">pIK8</A></TD></TR><TR><TD>pIK9</TD><TD>2013-09-18</TD><TD>Methylmalonyl-CoA pathway with sfp and the permeability device behind a weak promoter</TD><TD>pSB3K3-lacP-BBa_B0034-pcc-acca2-BBa_B0030-sfp-BBa_B0010-BBa_B0030-BBa_I746200-BBa_B0012</TD><TD><A href="/wiki/images/b/bb/Heidelberg_PIK9.gb.txt" class="internal" title="Heidelberg PIK9.gb.txt">pIK9</A></TD></TR><TR><TD>pIK10</TD><TD>2013-09-18</TD><TD>Methylmalonyl-CoA pathway with sfp and the permeability device behind a weak promoter, for submission</TD><TD>pSB1C3-lacP-BBa_B0034-pcc-acca2-BBa_B0030-sfp-BBa_B0010-BBa_B0030-BBa_I746200-BBa_B0012</TD><TD><A href="/wiki/images/d/d4/Heidelberg_PIK10.gb.txt" class="internal" title="Heidelberg PIK10.gb.txt">pIK10</A></TD></TR><TR><TD>pIK11</TD><TD>2013-09-18</TD><TD>Methylmalonyl-CoA pathway with sfp, for submission</TD><TD>pSB1C3-lacP-BBa_B0034-pcc-acca2-BBa_B0030-sfp</TD><TD><A href="/wiki/images/c/ce/Heidelberg_PIK11.gb.txt" class="internal" title="Heidelberg PIK11.gb.txt">pIK11</A></TD></TR><TR><TD>pHM01</TD><TD>2013-04-29</TD><TD> DelH construct assembled by Restriction Ligation Strategy </TD><TD>pSB6A1-BBa_K206000(AraC)-DelH-BBa_I732019(lacZ)-B0015(stop-codon) </TD><TD><A href="/wiki/images/a/aa/Heidelberg_DelH-pSB6A1-AraC-lacZ.gb.txt" class="internal" title="Heidelberg DelH-pSB6A1-AraC-lacZ.gb.txt">pHM01</A></TD></TR><TR><TD>pHM02</TD><TD>2013-07-08</TD><TD> DelH construct assembled by Gibson Assemby </TD><TD>pSB6A1-R0010(lacI)-B0035(RBS)-DelH-B0034(RBS)-E1010(mRFP)-B0015(stop) </TD><TD><A href="/wiki/images/5/57/Heidelberg_DelH-BB%28Gibson-pSB6A1-AraC-lacZ%29.gb.txt" class="internal" title="Heidelberg DelH-BB(Gibson-pSB6A1-AraC-lacZ).gb.txt">pHM02</A></TD></TR><TR><TD>pHM03</TD><TD>2013-04-29</TD><TD> DelH construct assembled by Gibson Assemby for existing backbone </TD><TD>pSB6A1-R0010(lacI)-B0034(RBS)-DelH-B0035(RBS)-E1010(mRFP)-B0015(stop) </TD><TD><A href="/wiki/images/e/eb/Heidelberg_PSB6A1%2BDelH.gb.txt" class="internal" title="Heidelberg PSB6A1+DelH.gb.txt">pHM03</A></TD></TR><TR><TD>pHM04</TD><TD>2013-08-12</TD><TD> DelH construct assembled by Gibson Assemby without mRFP </TD><TD>pSB6A1-R0010(lacI)-B0034(RBS)-DelH-B0015(stop) </TD><TD><A href="/wiki/images/5/50/Heidelberg_PHM04-DelH-pSB6A1%28without_mRFP%29.gb.txt" class="internal" title="Heidelberg PHM04-DelH-pSB6A1(without mRFP).gb.txt">pHM04</A></TD></TR><TR><TD>pHM05</TD><TD>2013-08-12</TD><TD> DelH construct assembled by Gibson Assemby with tetracycline resistance, without mRFP</TD><TD> pSB6A1-R0010(lacI)-DelH-B0015(stop)-pSB1T3(tetR) </TD><TD><A href="/wiki/images/1/15/Heidelberg_PHM05-DelH-tetR-pSB6A1.gb.txt" class="internal" title="Heidelberg PHM05-DelH-tetR-pSB6A1.gb.txt">pHM05</A></TD></TR><TR><TD>pFSN</TD><TD>2013-06-28</TD><TD>DelRest, Delftibactin cluster from <I>D. Acidovorans</I></TD><TD>pSB4K5-lacZ-BBa_B0035-DelA-DelB-DelC-DelE-DelF-DelG-DelO-DelP-DelL-BBa_B0034</TD><TD><A href="/wiki/images/a/a0/Heidelberg_Psb4k5%2BBBa_J04450_DelRest.gb.txt" class="internal" title="Heidelberg Psb4k5+BBa J04450 DelRest.gb.txt">pFSN</A></TD></TR><TR><TD>pFS_02</TD><TD>2013-09-26</TD><TD>DelH construct assembled by Gibson Assemby with weak Promotor, weak RBS </TD><TD>pSB6A1-BBa_J23114-BBa_B0032-DelH</TD><TD><A href="/wiki/images/5/55/Heidelberg_PSB6A1_BBa_J23114_BBa_B0032_DelH.gb.txt" class="internal" title="Heidelberg PSB6A1 BBa J23114 BBa B0032 DelH.gb.txt">pFS_02</A></TD></TR><TR><TD>pFS_03</TD><TD>2013-09-26</TD><TD> Helper plasmid for DelH restriction cloning, DelH, flanked by KpnI and BamHI site, without promotor,</TD><TD>pSB4K5-DelH</TD><TD><A href="/wiki/images/8/84/Heidelberg_PSB4K5min_KpnI_DelH_BamHI.gb.txt" class="internal" title="Heidelberg PSB4K5min KpnI DelH BamHI.gb.txt">pFS_03</A></TD></TR><TR><TD>pFS_04</TD><TD>2013-09-26</TD><TD>Target plasmid for DelH restriction cloning, ccdB cassette, flanked by KpnI and BamHI site </TD><TD>pSB6A1-BBa_J23114-BBa_B0032-ccdBcassette</TD><TD><A href="/wiki/images/5/5c/Heidelberg_PSB6A1_BBa_J23114_BBa_B0032_ccdB_cassette.gb.txt" class="internal" title="Heidelberg PSB6A1 BBa J23114 BBa B0032 ccdB cassette.gb.txt">pFS_04</A></TD></TR><TR><TD>pFS_05</TD><TD>2013-09-26</TD><TD> DelH construct assembled by restriction cloning using pFS_03 and pFS-04 DelH, flanked by KpnI and BamHI site, under weak promotor,</TD><TD>pSB6A1-BBa_J23114-BBa_B0032-DelH</TD><TD><A href="/wiki/images/c/c2/Heidelberg_DELH_final_pSB6A1_BBa_J23114_BBa_B0032_KpnI_DelH_BamHI.gb.txt" class="internal" title="Heidelberg DELH final pSB6A1 BBa J23114 BBa B0032 KpnI DelH BamHI.gb.txt">pFS_05</A></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Module_Shuffling_2">Module Shuffling</SPAN></H2><TABLE class="wikitable sortable"><TBODY><TR><TH>Name</TH><TH>Date</TH><TH>Brief Description</TH><TH>Genotype</TH><TH>Plasmid Map</TH><TH>GenBank-File
</TH></TR><TR><TD>pIK03</TD><TD>2013-mm-dd</TD><TD>NRPS for Pro-Leu</TD><TD>pSB1C3+TycB1dComdC+TycC6</TD><TD><A href="/wiki/images/1/16/Heidelberg_PSB1C3%2BTycB1dComdC%2BTycC6.gb.txt" class="internal" title="Heidelberg PSB1C3+TycB1dComdC+TycC6.gb.txt">pIK03</A></TD></TR><TR><TD>pIK04</TD><TD>2013-mm-dd</TD><TD>NRPS for Phe-Orn-Leu</TD><TD>pSB1C3+TycAdCom+TycC5-TycC6</TD><TD><A href="/wiki/images/c/c6/Heidelberg_PSB1C3%2BTycAdCom%2BTycC5-TycC6.gb.txt" class="internal" title="Heidelberg PSB1C3+TycAdCom+TycC5-TycC6.gb.txt">pIK04</A></TD></TR><TR><TD>pIK05</TD><TD>2013-mm-dd</TD><TD>NRPS for Pro-Orn-Leu</TD><TD>pSB1C3+TycB1dComdC+TycC5-TycC6</TD><TD><A href="/wiki/images/9/9b/Heidelberg_PSB1C3%2BTycBdComdC%2BTycC5-TycC6.gb.txt" class="internal" title="Heidelberg PSB1C3+TycBdComdC+TycC5-TycC6.gb.txt">pIK05</A></TD></TR><TR><TD>pPW01</TD><TD>2013-mm-dd</TD><TD>NRPS for Orn-Pro-Phe-Leu</TD><TD>pSB1C3+TycC5+TycB1dCom-C(TycB2)+TycAdCom+TycC6</TD><TD><A href="/wiki/images/c/cd/Heidelberg_Bla.gb.txt" class="internal" title="Heidelberg Bla.gb.txt">pPW01</A></TD></TR><TR><TD>pPW02</TD><TD>2013-mm-dd</TD><TD>NRPS for Orn-Pro-Phe-Leu</TD><TD>pSB1C3+TycC5+TycB1dCom-C(TycB2)+TycAdE+TycC6</TD><TD><A href="/wiki/images/8/8e/Heidelberg_Blaa.gb.txt" class="internal" title="Heidelberg Blaa.gb.txt">pPW02</A></TD></TR><TR><TD>pPW03</TD><TD>2013-08-18</TD><TD>NRPS for Phe-Indigoidine</TD><TD>pSB1C3+TycA+C(TycC2)+IndC</TD><TD><A href="/wiki/images/5/54/Heidelberg_PSB1C3%2BTycAdE%2BC%28TycC2%29%2BindC.gb.txt" class="internal" title="Heidelberg PSB1C3+TycAdE+C(TycC2)+indC.gb.txt">pPW03</A></TD></TR><TR><TD>pPW04</TD><TD>2013-08-18</TD><TD>NRPS for Asn-Indigoidine</TD><TD>pSB1C3+TycC1-C(TycC2)+IndC</TD><TD><A href="/wiki/images/f/fd/Heidelberg_PSB1C3%2BTycC1-C%28C2%29%2BindC.gb.txt" class="internal" title="Heidelberg PSB1C3+TycC1-C(C2)+indC.gb.txt">pPW04</A></TD></TR><TR><TD>pPW05</TD><TD>2013-08-18</TD><TD>NRPS for Val-Indigoidine</TD><TD>pSB1C3+TycC4+C(TycC2)+IndC</TD><TD><A href="/wiki/images/9/94/Heidelberg_PSB1C3%2BTycC4%2BC%28TycC2%29%2BindC.gb.txt" class="internal" title="Heidelberg PSB1C3+TycC4+C(TycC2)+indC.gb.txt">pPW05</A></TD></TR><TR><TD>pPW06</TD><TD>2013-09-07</TD><TD>NRPS for Orn-Val-Indigoidine</TD><TD>pSB1C3+TycC5dC+C(TycC4)+TycC4dC+C(TycC2)+indC</TD><TD><A href="/wiki/images/4/4b/Heidelberg_A_pSB1C3%2BTycC5dC%2BC%28TycC4%29%2BTycC4dC%2BC%28TycC2%29%2BindC.gb.txt" class="internal" title="Heidelberg A pSB1C3+TycC5dC+C(TycC4)+TycC4dC+C(TycC2)+indC.gb.txt">pPW06</A></TD></TR><TR><TD>pPW07</TD><TD>2013-09-07</TD><TD>NRPS for Orn-Val-Val-Indigoidine</TD><TD>pSB1C3+TycC5dC+TycC4+C(TycC4)+TycC4dC+C(TycC2)+indC</TD><TD><A href="/wiki/images/c/cc/Heidelberg_B_pSB1C3%2BTycC5dC%2BTycC4%2BC%28TycC4%29%2BTycC4dC%2BC%28TycC2%29%2BindC.gb.txt" class="internal" title="Heidelberg B pSB1C3+TycC5dC+TycC4+C(TycC4)+TycC4dC+C(TycC2)+indC.gb.txt">pPW07</A></TD></TR><TR><TD>pPW08</TD><TD>2013-09-07</TD><TD>NRPS for Val-Val-Indigoidine</TD><TD>pSB1C3+TycC4dC+C(TycC4)+TycC4dC+C(TycC2)+indC</TD><TD><A href="/wiki/images/2/2f/Heidelberg_C_pSB1C3%2BTycC4dC%2BC%28TycC4%29%2BTycC4dC%2BC%28TycC2%29%2BindC.gb.txt" class="internal" title="Heidelberg C pSB1C3+TycC4dC+C(TycC4)+TycC4dC+C(TycC2)+indC.gb.txt">pPW08</A></TD></TR><TR><TD>pPW09</TD><TD>2013-09-07</TD><TD>NRPS for Pro-Leu-Indigoidine</TD><TD>pSB1C3+TycB1dComdC+TycC6dTE+C(TycC2)+indC</TD><TD><A href="/wiki/images/6/61/Heidelberg_D_pSB1C3%2BTycB1dComdC%2BTycC6dTE%2BC%28TycC2%29%2BindC.gb.txt" class="internal" title="Heidelberg D pSB1C3+TycB1dComdC+TycC6dTE+C(TycC2)+indC.gb.txt">pPW09</A></TD></TR><TR><TD>pPW10</TD><TD>2013-09-07</TD><TD>NRPS for Pro-Leu-Val-Indigoidine</TD><TD>pSB1C3+TycB1dComdC+TycC6dTE+C(TycC4)+TycC4dC+C(TycC2)+indC</TD><TD><A href="/wiki/images/0/0c/Heidelberg_E_pSB1C3%2BTycB1dComdC%2BTycC6dTE%2BC%28TycC4%29%2BTycC4dC%2BC%28TycC2%29%2BindC.gb.txt" class="internal" title="Heidelberg E pSB1C3+TycB1dComdC+TycC6dTE+C(TycC4)+TycC4dC+C(TycC2)+indC.gb.txt">pPW10</A></TD></TR><TR><TD>pPW11</TD><TD>2013-09-07</TD><TD>NRPS for Phe-Orn-Leu-Indigoidine</TD><TD>pSB1C3+TycAdCom+TycC5-TycC6dTE+C(TycC2)+indC</TD><TD><A href="/wiki/images/8/81/Heidelberg_F_pSB1C3%2BTycAdCom%2BTycC5-TycC6dTE%2BC%28TycC2%29%2BindC.gb.txt" class="internal" title="Heidelberg F pSB1C3+TycAdCom+TycC5-TycC6dTE+C(TycC2)+indC.gb.txt">pPW11</A></TD></TR><TR><TD>pPW12</TD><TD>2013-09-07</TD><TD>NRPS for Phe-Orn-Leu-Val-Indigoidine</TD><TD>pSB1C3+TycAdCom+TycC5-TycC6dTE+C(TycC4)+TycC4dC+C(TycC2)+indC</TD><TD><A href="/wiki/images/0/05/Heidelberg_G_pSB1C3%2BTycAdCom%2BTycC5-TycC6dTE%2BC%28TycC4%29%2BTycC4dC%2BC%28TycC2%29%2BindC.gb.txt" class="internal" title="Heidelberg G pSB1C3+TycAdCom+TycC5-TycC6dTE+C(TycC4)+TycC4dC+C(TycC2)+indC.gb.txt">pPW12</A></TD></TR><TR><TD>pJS01</TD><TD>2013-09-18</TD><TD>Helper plasmid for the tagging of NRPs with Indigoidine</TD><TD>pSB1C3+ccdB+C(TycC2)+indC</TD><TD><A href="/wiki/images/9/99/Heidelberg_PSB1C3%2BccdB%2Bc%28TycC2%29%2BindC.gb.txt" class="internal" title="Heidelberg PSB1C3+ccdB+c(TycC2)+indC.gb.txt">pJS01</A></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Linker_variation_2">Linker variation</SPAN></H2><TABLE class="wikitable sortable"><TBODY><TR><TH>Name</TH><TH>Date</TH><TH>Brief Description</TH><TH>Genotype</TH><TH>Plasmid Map</TH><TH>GenBank-File
</TH></TR><TR><TD>pAR01</TD><TD>2013-16-09</TD><TD> NRPS for Valin-Asparagine-Indigoidine-Expression with double short linkers </TD><TD> pSB1C3-TycC4dCom_s+TycC2_s+indC </TD><TD><A href="/wiki/images/7/73/Heidelberg_PSB1C3%2BTycC4_s%2BC%28TycC2%29_s%2BindC.gb" class="internal" title="Heidelberg PSB1C3+TycC4 s+C(TycC2) s+indC.gb">pAR01</A></TD></TR><TR><TD>pAR02</TD><TD>2013-16-09</TD><TD>  NRPS for Valin-Asparagine-Indigoidine-Expression with C4 short linker </TD><TD> pSB1C3-TycC4dCom_s+TycC2_m+indC </TD><TD><A href="/wiki/images/4/44/Heidelberg_PSB1C3%2BTycC4_s%2BC%28TycC2%29_m%2BindC.gb" class="internal" title="Heidelberg PSB1C3+TycC4 s+C(TycC2) m+indC.gb">pAR02</A></TD></TR><TR><TD>pAR03</TD><TD>2013-16-09</TD><TD>  NRPS for Valin-Asparagine-Indigoidine-Expression with C2 short linker </TD><TD> pSB1C3-TycC4dCom_m+TycC2_s+indC </TD><TD><A href="/wiki/images/8/8a/Heidelberg_PSB1C3%2BTycC4_m%2BC%28TycC2%29_s%2BindC.gb" class="internal" title="Heidelberg PSB1C3+TycC4 m+C(TycC2) s+indC.gb">pAR03</A></TD></TR><TR><TD>pAR04</TD><TD>2013-16-09</TD><TD>  NRPS for Valin-Asparagine-Indigoidine-Expression with medium linkers </TD><TD> pSB1C3-TycC4dCom+TycC2+indC </TD><TD><A href="/wiki/images/8/81/Heidelberg_PSB1C3%2BTycC4_m%2BC%28TycC2%29_m%2BindC.gb" class="internal" title="Heidelberg PSB1C3+TycC4 m+C(TycC2) m+indC.gb">pAR04</A></TD></TR><TR><TD>pAR05</TD><TD>2013-16-09</TD><TD>  NRPS for Valin-Asparagine-Indigoidine-Expression with C2 long linker </TD><TD> pSB1C3-TycC4dCom_m+TycC2_l+indC </TD><TD><A href="/wiki/images/2/2b/Heidelberg_PSB1C3%2BTycC4_m%2BC%28TycC2%29_l%2BindC.gb" class="internal" title="Heidelberg PSB1C3+TycC4 m+C(TycC2) l+indC.gb">pAR05</A></TD></TR><TR><TD>pAR06</TD><TD>2013-16-09</TD><TD>  NRPS for Valin-Asparagine-Indigoidine-Expression with C4 long linker </TD><TD> pSB1C3-TycC4dCom_l+TycC2_m+indC </TD><TD><A href="/wiki/images/d/de/Heidelberg_PSB1C3%2BTycC4_l%2BC%28TycC2%29_m%2BindC.gb" class="internal" title="Heidelberg PSB1C3+TycC4 l+C(TycC2) m+indC.gb">pAR06</A></TD></TR><TR><TD>pAR07</TD><TD>2013-16-09</TD><TD>  NRPS for Valin-Asparagine-Indigoidine-Expression with long linkers </TD><TD> pSB1C3-TycC4dCom_l+TycC2_l+indC </TD><TD><A href="/wiki/images/2/24/Heidelberg_PSB1C3%2BTycC4_l%2BC%28TycC2%29_l%2BindC.gb" class="internal" title="Heidelberg PSB1C3+TycC4 l+C(TycC2) l+indC.gb">pAR07</A></TD></TR><TR><TD>pAR08</TD><TD>2013-16-09</TD><TD>  NRPS for Valin-Asparagine-Indigoidine-Expression with C4 short and C2 long linker </TD><TD> pSB1C3-TycC4dCom_s+TycC2_l+indC </TD><TD><A href="/wiki/images/d/d7/Heidelberg_PSB1C3%2BTycC4_s%2BC%28TycC2%29_l%2BindC.gb" class="internal" title="Heidelberg PSB1C3+TycC4 s+C(TycC2) l+indC.gb">pAR08</A></TD></TR><TR><TD>pAR09</TD><TD>2013-16-09</TD><TD>  NRPS for Valin-Asparagine-Indigoidine-Expression with C4 long and C2 short linker </TD><TD> pSB1C3-TycC4dCom_l+TycC2_s+indC </TD><TD><A href="/wiki/images/7/78/Heidelberg_PSB1C3%2BTycC4_l%2BC%28TycC2%29_s%2BindC.gb" class="internal" title="Heidelberg PSB1C3+TycC4 l+C(TycC2) s+indC.gb">pAR09</A></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Dinner_plasmids">Dinner plasmids</SPAN></H2><TABLE class="wikitable sortable"><TBODY><TR><TH>Name</TH><TH>Date</TH><TH>Brief Description</TH><TH>Genotype</TH><TH>Plasmid Map</TH><TH>GenBank-File
</TH></TR><TR><TD><DIV id="pIK6">pIK6</DIV></TD><TD>2013-08-27</TD><TD>pIK6: special Ilia plasmid. This plasmid does not fullfill iGEM's RFC10 but is nevertheless special. It is for expression of tasty Ham driven by a strong lacCheese inducible promoter, a RBSalami for increased fleshiness, which can only terminated by a Champignon stop codon. The pIZa backbone with Pepper resistance and high copy Onion ori will improve the users taste stimulation. Bon Appétit! </TD><TD>pIZa-lacCheese P-RBSalami-ham CDS-Champ Term-Pepper res-Onion Ori</TD><TD><A href="/wiki/images/a/a2/Heidelberg_PIK6.txt" class="internal" title="Heidelberg PIK6.txt">pIK6</A></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Domain_Shuffling_and_PPTases_2">Domain Shuffling and PPTases</SPAN></H2><TABLE class="wikitable sortable"><TBODY><TR><TH>Name</TH><TH>Date</TH><TH>Brief Description</TH><TH>Genotype</TH><TH>Plasmid Map</TH><TH>GenBank-File
</TH></TR><TR><TD><DIV id="pMM64">pMM64</DIV></TD><TD>2013-06-01</TD><TD>bpsA(fus)</TD><TD>pETDuet-1-T7lac-bpsA(fus)</TD><TD><A href="/wiki/images/f/f4/Heidelberg_PMM064.gb.txt" class="internal" title="Heidelberg PMM064.gb.txt">pMM64</A></TD></TR><TR><TD><DIV id="pMM65">pMM65</DIV></TD><TD>2013-06-01</TD><TD>svp(fus)</TD><TD>pETDuet-1-T7lac-svp(fus)</TD><TD><A href="/wiki/images/9/92/Heidelberg_PMM065.gb.txt" class="internal" title="Heidelberg PMM065.gb.txt">pMM65</A></TD></TR><TR><TD><DIV id="pKH4">pKH4</DIV></TD><TD>2013-09-01</TD><TD>pRB3/pMM64 derived plasmid without sfp</TD><TD>pSB1C3-lacP-BBa_B0034-KpnI-indC-BBa_B0029-nonsense</TD><TD><A href="/wiki/images/2/2e/Heidelberg_PKH4.gb" class="internal" title="Heidelberg PKH4.gb">pKH4</A></TD></TR><TR><TD><DIV id="pKH6">pKH6</DIV></TD><TD>2013-09-05</TD><TD>BBa CDS sfp only for part submission</TD><TD>pSB1C3-sfp</TD><TD><A href="/wiki/images/9/98/Heidelberg_PKH6.gb" class="internal" title="Heidelberg PKH6.gb">pKH6</A></TD></TR><TR><TD><DIV id="pKH7">pKH7</DIV></TD><TD>2013-09-05</TD><TD>BBa CDS svp only for part submission</TD><TD>pSB1C3-svp</TD><TD><A href="/wiki/images/0/0d/Heidelberg_PKH7.gb" class="internal" title="Heidelberg PKH7.gb">pKH7</A></TD></TR><TR><TD><DIV id="pKH8">pKH8</DIV></TD><TD>2013-09-05</TD><TD>BBa CDS entD only for part submission</TD><TD>pSB1C3-entD</TD><TD><A href="/wiki/images/1/1e/Heidelberg_PKH8.gb" class="internal" title="Heidelberg PKH8.gb">pKH8</A></TD></TR><TR><TD><DIV id="pKH9">pKH9</DIV></TD><TD>2013-09-05</TD><TD>BBa CDS delC only for part submission</TD><TD>pSB1C3-delC</TD><TD><A href="/wiki/images/0/00/Heidelberg_PKH9.gb" class="internal" title="Heidelberg PKH9.gb">pKH9</A></TD></TR><TR><TD><DIV id="pRB1">pRB1</DIV></TD><TD>2013-07-01</TD><TD>bpsA svp(pMM65)</TD><TD>pSB1C3-lacP-BBa_B0034-bpsA-BBa_B0029-svp(pMM65)</TD><TD>N.A</TD><TD>N.A
</TD></TR><TR><TD><DIV id="pRB2">pRB2</DIV></TD><TD>2013-07-08</TD><TD>bpsA svp</TD><TD>HindIII-pSB1C3-lacP-BBa_B0034-bpsA-BBa_B0029-HindIII-svp</TD><TD>N.A</TD><TD>N.A
</TD></TR><TR><TD><DIV id="pRB3">pRB3</DIV></TD><TD>2013-07-15</TD><TD>indC sfp</TD><TD>NheI-pSB1C3-lacP-BBa_B0034-KpnI-indC-BBa_B0029-BamHI-sfp</TD><TD><A href="/wiki/images/7/71/Heidelberg_PRB03.gb" class="internal" title="Heidelberg PRB03.gb">pRB03</A></TD></TR><TR><TD><DIV id="pRB4">pRB4</DIV></TD><TD>2013-07-15</TD><TD>indC svp</TD><TD>NheI-pSB1C3-lacP-BBa_B0034-KpnI-indC-BBa_B0029-BamHI-svp</TD><TD><A href="/wiki/images/2/2d/Heidelberg_PRB04.gb" class="internal" title="Heidelberg PRB04.gb">pRB04</A></TD></TR><TR><TD><DIV id="pRB5">pRB5</DIV></TD><TD>2013-07-15</TD><TD>indC svpF</TD><TD>NheI-pSB1C3-lacP-BBa_B0034-KpnI-indC-BBa_B0029-BamHI-svp(pMM65)</TD><TD><A href="/wiki/images/7/7a/Heidelberg_PRB05.gb" class="internal" title="Heidelberg PRB05.gb">pRB05</A></TD></TR><TR><TD><DIV id="pRB6">pRB6</DIV></TD><TD>2013-07-15</TD><TD>indC entD</TD><TD>NheI-pSB1C3-lacP-BBa_B0034-KpnI-indC-BBa_B0029-BamHI-entD</TD><TD><A href="/wiki/images/c/c9/Heidelberg_PRB06.gb" class="internal" title="Heidelberg PRB06.gb">pRB06</A></TD></TR><TR><TD><DIV id="pRB7">pRB7</DIV></TD><TD>2013-07-15</TD><TD>bpsA(pMM64) sfp</TD><TD>NheI-pSB1C3-lacP-BBa_B0034-KpnI-bpsA(pMM64)-BBa_B0029-BamHI-sfp</TD><TD><A href="/wiki/images/a/a0/Heidelberg_PRB07.gb" class="internal" title="Heidelberg PRB07.gb">pRB07</A></TD></TR><TR><TD><DIV id="pRB8">pRB8</DIV></TD><TD>2013-07-15</TD><TD>bpsA(pMM64) svpF</TD><TD>NheI-pSB1C3-lacP-BBa_B0034-KpnI-bpsA(pMM64)-BBa_B0029-BamHI-svp(pMM65)</TD><TD><A href="/wiki/images/f/ff/Heidelberg_PRB08.gb" class="internal" title="Heidelberg PRB08.gb">pRB08</A></TD></TR><TR><TD><DIV id="pRB9">pRB9</DIV></TD><TD>2013-07-15</TD><TD>bpsA(pMM64) svp</TD><TD>NheI-pSB1C3-lacP-BBa_B0034-KpnI-bpsA(pMM64)-BBa_B0029-BamHI-svp</TD><TD><A href="/wiki/images/4/47/Heidelberg_PRB09.gb" class="internal" title="Heidelberg PRB09.gb">pRB09</A></TD></TR><TR><TD><DIV id="pRB10">pRB10</DIV></TD><TD>2013-07-15</TD><TD>bpsA(pMM64) entD</TD><TD>NheI-pSB1C3-lacP-BBa_B0034-KpnI-bpsA(pMM64)-BBa_B0029-BamHI-entD</TD><TD><A href="/wiki/images/0/0f/Heidelberg_PRB10.gb" class="internal" title="Heidelberg PRB10.gb">pRB10</A></TD></TR><TR><TD><DIV id="pRB11">pRB11</DIV></TD><TD>2013-07-29</TD><TD>pKH1-der bpsA(pMM64)-ccdb svpF</TD><TD>pSB1C3-lacP-BBa_B0034-bpsA(pMM64)(ccdb)-BBa_B0029-svp(pMM65)</TD><TD>N.A</TD><TD>N.A
</TD></TR><TR><TD><DIV id="pRB12">pRB12</DIV></TD><TD>2013-07-29</TD><TD>pKH2-der bpsA(pMM64)-ccdb svpF</TD><TD>pSB1C3-lacP-BBa_B0034-bpsA(pMM64)(ccdb)-BBa_B0029-svp(pMM65)</TD><TD>N.A</TD><TD>N.A
</TD></TR><TR><TD><DIV id="pRB13">pRB13</DIV></TD><TD>2013-07-29</TD><TD>pRB3-der indC-ccdb sfp</TD><TD>NheI-pSB1C3-lacP-BBa_B0034-KpnI-indC(ccdb)-BBa_B0029-BamHI-sfp</TD><TD>N.A</TD><TD>N.A
</TD></TR><TR><TD><DIV id="pRB14">pRB14</DIV></TD><TD>2013-08-12</TD><TD>pRB3-der indC-ccdB sfp</TD><TD>NheI-pSB1C3-lacP-BBa_B0034-KpnI-indC(ccdB)-BBa_B0029-BamHI-sfp</TD><TD>N.A</TD><TD>N.A
</TD></TR><TR><TD><DIV id="pRB15">pRB15</DIV></TD><TD>2013-08-12</TD><TD>pSB3K3 sfp</TD><TD>pSB3K3-lacP-BBa_B0029-BamHI-sfp-NheI</TD><TD><A href="/wiki/images/6/60/Heidelberg_PRB15.gb" class="internal" title="Heidelberg PRB15.gb">pRB15</A></TD></TR><TR><TD><DIV id="pRB16">pRB16</DIV></TD><TD>2013-08-12</TD><TD>pSB3K3 svp</TD><TD>pSB3K3-lacP-BBa_B0029-BamHI-svp-NheI</TD><TD><A href="/wiki/images/a/a0/Heidelberg_PRB16.gb" class="internal" title="Heidelberg PRB16.gb">pRB16</A></TD></TR><TR><TD><DIV id="pRB17">pRB17</DIV></TD><TD>2013-08-12</TD><TD>pSB3K3 entD</TD><TD>pSB3K3-lacP-BBa_B0029-BamHI-entD-NheI</TD><TD><A href="/wiki/images/2/21/Heidelberg_PRB17.gb" class="internal" title="Heidelberg PRB17.gb">pRB17</A></TD></TR><TR><TD><DIV id="pRB18">pRB18</DIV></TD><TD>2013-08-12</TD><TD>pSB3K3 delC</TD><TD>pSB3K3-lacP-BBa_B0029-BamHI-delC-NheI</TD><TD><A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=Heidelberg_pRB18.gb" class="new" title="Heidelberg pRB18.gb">pRB18</A></TD></TR><TR><TD><DIV id="pRB19">pRB19</DIV></TD><TD>2013-08-19</TD><TD>pRB14-der indC-ccdB</TD><TD>NheI-pSB1C3-lacP-BBa_B0034-KpnI-indC(ccdB)</TD><TD>N.A</TD><TD>N.A
</TD></TR><TR><TD><DIV id="pRB20">pRB20</DIV></TD><TD>2013-08-19</TD><TD>pRB19-der indC-HD-ccdB</TD><TD>NheI-pSB1C3-lacP-BBa_B0034-KpnI-indC-HD(ccdB)</TD><TD>N.A</TD><TD>N.A
</TD></TR><TR><TD><DIV id="pRB21">pRB21</DIV></TD><TD>2013-08-19</TD><TD>pSB1C3 indC</TD><TD>NheI-pSB1C3-lacP-BBa_B0034-KpnI-indC</TD><TD><A href="/wiki/images/6/6b/Heidelberg_PRB22.gb" class="internal" title="Heidelberg PRB22.gb">pRB21</A></TD></TR><TR><TD><DIV id="pRB22">pRB22</DIV></TD><TD>2013-08-19</TD><TD>pRB21-der indC-HD</TD><TD>NheI-pSB1C3-lacP-BBa_B0034-KpnI-indC-HD</TD><TD><A href="/wiki/images/6/6b/Heidelberg_PRB22.gb" class="internal" title="Heidelberg PRB22.gb">pRB22</A></TD></TR><TR><TD><DIV id="pRB23">pRB23</DIV></TD><TD>2013-08-19</TD><TD>pRB22-der indC-HD</TD><TD>NheI-pSB1C3-lacP-BBa_B0034-KpnI-indC*dT(ccdB)</TD><TD>N.A</TD><TD>N.A
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Used_T_and_TE_Domains">Used T and TE Domains</SPAN></H1><TABLE class="wikitable sortable"><TBODY><TR><TH>ID</TH><TH>Name (derived from)</TH><TH>Description
</TH></TR><TR><TD><DIV id="T1">-T1</DIV></TD><TD>indC</TD><TD>native T-Domain of indC from <I>P. luminescens</I></TD></TR><TR><TD><DIV id="T2">-T2</DIV></TD><TD>bpsA</TD><TD> native T-Domain of bpsA from <I>S. lavendulae</I></TD></TR><TR><TD><DIV id="T3">-T3</DIV></TD><TD>entF</TD><TD>native T-Domain of entF from <I>E. coli</I></TD></TR><TR><TD><DIV id="T4">-T4</DIV></TD><TD>tycA1</TD><TD>native T-domain of tycA1 from <I>Brevibacillus parabrevis</I></TD></TR><TR><TD><DIV id="T5">-T5</DIV></TD><TD>tycC6</TD><TD>native T-domain of tycC6 from <I>B. parabrevis</I></TD></TR><TR><TD><DIV id="T6">-T6</DIV></TD><TD>delH4</TD><TD>native T-domain of delH4 from <I>Delftia acidovorans</I></TD></TR><TR><TD><DIV id="T7">-T7</DIV></TD><TD>delH5</TD><TD>native T-domain of delH5 from <I>D. acidovorans</I></TD></TR><TR><TD><DIV id="T8">-T8</DIV></TD><TD>plu2642</TD><TD>native T-domain of plu2642 from <I>Photorhabdus luminescens</I></TD></TR><TR><TD><DIV id="T9">-T9</DIV></TD><TD>plu2670</TD><TD>native T-domain of plu2670 from <I>P. luminescens</I></TD></TR><TR><TD><DIV id="T10">-T10</DIV></TD><TD>synT1</TD><TD>synthetic T-Domain from indC-BLAST
</TD></TR><TR><TD><DIV id="T11">-T11</DIV></TD><TD>synT2</TD><TD>synthetic T-Domain from indC_T-Plum-BLAST
</TD></TR><TR><TD><DIV id="T12">-T12</DIV></TD><TD>synT3</TD><TD>synthetic T-Domain from indC_T-BLAST
</TD></TR><TR><TD><DIV id="T13">-T13</DIV></TD><TD>synT4</TD><TD>synthetic T-Domain from project_5 BLAST
</TD></TR><TR><TD><DIV id="T14">-T14</DIV></TD><TD>synT5</TD><TD>synthetic T-Domain random 1 (indigoidine synthetases)
</TD></TR><TR><TD><DIV id="T15">-T15</DIV></TD><TD>synT6</TD><TD>synthetic T-Domain random 2 (project_5)
</TD></TR><TR style="border-top: double;"><TD><DIV id="TTE1">-TTE1</DIV></TD><TD>bpsA</TD><TD>native TTE-Domain of bpsA from <I>S. lavendulae</I></TD></TR><TR><TD><DIV id="TTE2">-TTE2</DIV></TD><TD>entF</TD><TD>native TTE-Domain of entF from <I>E. coli</I></TD></TR><TR><TD><DIV id="TTE3">-TTE3</DIV></TD><TD>tycC6</TD><TD>native TTE-domain of tycC6 from <I>B. parabrevis</I></TD></TR><TR><TD><DIV id="TTE4">-TTE4</DIV></TD><TD>delH5</TD><TD>native TTE-domain of delH5 from <I>D. acidovorans</I></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Instruments"> Instruments </SPAN></H1><TABLE class="wikitable sortable"><TBODY><TR><TH> Instrument </TH><TH> Type </TH><TH> Manufacturer
</TH></TR><TR><TD>Table Centrifuge </TD><TD> Microfuge® 18 Centrifuge </TD><TD> Beckman CoulterTM
</TD></TR><TR><TD>Table Centrifuge </TD><TD> Microfuge® 22R Centrifuge </TD><TD> Beckman CoulterTM
</TD></TR><TR><TD>Centrifuge </TD><TD> Allegra X-12R </TD><TD> Beckman CoulterTM
</TD></TR><TR><TD>Shaker </TD><TD> VORTEXGENIE 2 </TD><TD> Scientific Industries, SITM
</TD></TR><TR><TD>Heating plate (magnet stirrer)</TD><TD>MR Hei-Standard </TD><TD>Heidolph
</TD></TR><TR><TD>Heatblock </TD><TD> QBD4 </TD><TD>Grant
</TD></TR><TR><TD>Heatblock (shakeing function) </TD><TD> Thermomixer comfort </TD><TD>Eppendorf
</TD></TR><TR><TD>PCR-Machine</TD><TD>MyCyclerTM thermo cycler</TD><TD>BioRad
</TD></TR><TR><TD>PCR-Machine</TD><TD>T100 Therma Cycler</TD><TD>BioRad
</TD></TR><TR><TD>UV-Chamber </TD><TD> Transluminator </TD><TD>Vilber Lourmat
</TD></TR><TR><TD>Scale (fine)</TD><TD>PioneerTM PA114C </TD><TD>OHAUS
</TD></TR><TR><TD>Scale</TD><TD>PioneerTM PA4101C </TD><TD>OHAUS
</TD></TR><TR><TD>Fridge </TD><TD>KTP 1750 Premium </TD><TD>Liebherr
</TD></TR><TR><TD>Freezer </TD><TD>GP 1366 Premium </TD><TD>Liebherr
</TD></TR><TR><TD>Hood</TD><TD>Tischabzug</TD><TD>Wesemann® Laboreinrichtung
</TD></TR><TR><TD>Draw-Off Pump</TD><TD>Vacuhand control</TD><TD>Vacubrand
</TD></TR><TR><TD>Incubator </TD><TD>HT Multitron Version 2  </TD><TD>INFORS
</TD></TR><TR><TD>Plate Reader </TD></TR><TR><TD>Computer  </TD><TD> Sun Microsystems
</TD></TR><TR><TD> UV/VIS Spectrometer </TD><TD>Ultrospec 3300 pro  </TD><TD>Amersham Biosciences
</TD></TR><TR><TD>Photometer  </TD><TD> NanoDrop® ND-1000 Spectrophotometer</TD><TD> peQLab Biotechnologie GmbH
</TD></TR><TR><TD>Photometer </TD><TD>NanoVue </TD><TD> General Electric
</TD></TR><TR><TD>Ice Machine </TD><TD> MF22</TD><TD>SCOTSMAN®
</TD></TR><TR><TD>Gel Electrophoresis Chamber </TD><TD> Mupid®-One </TD><TD> 	Advance
</TD></TR><TR><TD>Cell Density Meter</TD><TD> Ultrospec10 </TD><TD> Amersham Biosciences
</TD></TR><TR><TD>PH-Meter</TD><TD> PH-Meter 765 Calimatic </TD><TD> Knick
</TD></TR><TR><TD>Incubator </TD><TD> Heraeus </TD><TD> Thermo Scientific
</TD></TR><TR><TD>Lyophilizer </TD><TD>Gefriertrocknungsanlage ALPHA 1-2 LD Plus Best. Nr. 101521 </TD><TD>Christ
</TD></TR><TR><TD>Ultra Sonification Stick </TD><TD>Sonoplus Gm 2070 (2002)</TD><TD>Bandelin
</TD></TR><TR><TD>Vacuum Manifold </TD><TD> Qiavac 24 Plus</TD><TD> QIAGEN
</TD></TR><TR><TD>Gas Cartridge Ventil CV470</TD><TD> Burner </TD><TD> 13883
</TD></TR><TR><TD>NuPAGE(R) Novex 3-8% Tris-Acetate Gel 1</TD><TD> SDS-Gel </TD><TD> Life Technologies GmbH
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Mass_Spectrometry_and_Neonate_Screening">Mass Spectrometry and Neonate Screening</SPAN></H1><TABLE class="wikitable sortable"><TBODY><TR><TH> Instrument </TH><TH> Type </TH><TH> Manufacturer </TH><TH> Order number
</TH></TR><TR><TD> Micro Filter Plates </TD><TD> Multiscreen 96 well </TD><TD> Merck Millipore </TD><TD> MSHVN4550
</TD></TR><TR><TD> Microtiter Plates</TD><TD> 96 well Mikrotiter plates made of Polypropylen (F-Form)  </TD><TD>Greiner</TD><TD> 655201
</TD></TR><TR><TD> Microtiter Plates Locks</TD><TD>Micromats </TD></TR><TR><TD> Aluminium Foil</TD></TR><TR><TD> Mass Spectrometer </TD><TD> Quattro Ultima (ESI-MS/MS)</TD><TD> Micromass (today: Waters)</TD></TR><TR><TD> Methanol </TD><TD> (20837.320 METHANOL HIPERSOLV HPLC ISOCRATIC GRADE) </TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Lab_Materials">Lab Materials</SPAN></H1><TABLE class="wikitable sortable"><TBODY><TR><TH> Material </TH><TH> Name </TH><TH> Manufacturer
</TH></TR><TR><TD> Bottle 50 ml, 100 ml, 200 ml, 500 ml, 1000 ml </TD></TR><TR><TD> Beaker 50 ml, 100 ml, 200 ml, 500 ml 1000 ml </TD></TR><TR><TD> Conical Flask 500 ml, 1000 ml, 100 ml </TD></TR><TR><TD> Conical Flask 300 ml </TD></TR><TR><TD> Single Channel Pipette </TD><TD> Pipetman® P2, P20, P200, P1000 </TD><TD>Gilson
</TD></TR><TR><TD> Multichannel Pipette </TD></TR><TR><TD> Multistep Pipette  </TD></TR><TR><TD> Pipette </TD></TR><TR><TD> 0.2 ml PCR Tube, Flat Cap, Natural </TD><TD> I1402-8200 </TD><TD> Starlab GmbH
</TD></TR><TR><TD> Microcentrifuge Tubes  </TD></TR><TR><TD> Centrifugetubes  </TD></TR><TR><TD> PCR 8-Strips </TD></TR><TR><TD> Gloves Nitril Gr. L </TD><TD>  Starlab powderfree </TD><TD> Starlab
</TD></TR><TR><TD> Gloves Nitril Gr. M </TD><TD>  Starlab powderfree </TD><TD> Starlab
</TD></TR><TR><TD> Gloves Nitril Gr. S</TD><TD>  Starlab powderfree </TD><TD> Starlab
</TD></TR><TR><TD> Filtertips </TD></TR><TR><TD> Inoculating Loop   </TD></TR><TR><TD> Disposal Bags   </TD></TR><TR><TD> Flasks   </TD></TR><TR><TD> Plate   </TD></TR><TR><TD> Petri Dishes </TD><TD> P5731-500EA </TD><TD> Sigma-Aldrich Chemie GmbH
</TD></TR><TR><TD> Petri Dishes </TD><TD> N221.2 </TD><TD> Carl Roth GmbH &amp; Co.KG
</TD></TR><TR><TD> TipOne® Pipette Tip </TD><TD> S1110-1700 </TD><TD> Starlab GmbH
</TD></TR><TR><TD> TipOne® Pipette Tip, 1000μl, Graduated </TD><TD> S1111-2721 </TD><TD> Starlab GmbH
</TD></TR><TR><TD> NeoBox-81 6er Set, je 1 x Transparent, g </TD><TD> 22916 </TD></TR><TR><TD> NeoLab-Marker for Reaction-Flaks </TD><TD> 19079 </TD></TR><TR><TD> Gene Pulser/MicroPulser Cuvettes, 0.1 cm </TD><TD> 165-2089 </TD></TR><TR><TD> NeoLab-Paper Scissors, 23 cm Long, Even </TD><TD> 23272 </TD></TR><TR><TD> TipOne® Pipette Tip, 200μl </TD><TD> S1110-1700 </TD><TD> Starlab GmbH
</TD></TR><TR><TD> TipOne® Pipette Tip, 10μl, Graduated, Re </TD><TD> S1111-3700 </TD><TD> Starlab GmbH
</TD></TR><TR><TD> Pipette, 5 ml </TD><TD> 606180 </TD><TD> Greiner bio-one GmbH
</TD></TR><TR><TD> Ring out of Plumbum with Vinyl Coating, 57 mm In </TD><TD> 310161013 </TD><TD> NEOLAB GmbH
</TD></TR><TR><TD> TipOne® Pipette Tip 10μl, Graduated, Rac </TD><TD> S1111-3800 </TD><TD>Starlab GmbH
</TD></TR><TR><TD> Reaction Tube,S.L.1.5 ml,Colorless. </TD><TD> 12682 </TD><TD> Eppendorf, Fisher Scientific GmbH
</TD></TR><TR><TD> Reaction Tube,S.L.,2 ml, Colorless </TD><TD> 12776 </TD><TD> Eppendorf
</TD></TR><TR><TD> NeoTape-Writing Tape, 13 mm, Gray </TD><TD> 280126114 </TD><TD> NEOLAB GMBH
</TD></TR><TR><TD> NeoTape-Writing Tape, 25 mm, Salmon-Colored </TD><TD> 280126229 </TD><TD> NEOLAB GMBH
</TD></TR><TR><TD> 10 ml Serological Pipette, Filter, Sterile </TD><TD> E4860-1011 </TD><TD> Starlab GmbH
</TD></TR><TR><TD> Gloves Latex + Alovera L </TD><TD> 14089 </TD></TR><TR><TD> Gloves Latex + Alovera M </TD><TD> 14088 </TD></TR><TR><TD> Gloves Latex + Alovera S </TD><TD> 14087 </TD></TR><TR><TD> PH-Stripes </TD><TD> WHAT10362000 </TD><TD> PANPEHA PLUS
</TD></TR><TR><TD> 100 Run24Barcode </TD><TD> 20110099-100 </TD><TD> GATC Biotech AG
</TD></TR><TR><TD> Tube Conical, Polypropylen, 50 ml </TD><TD> 352070 </TD><TD> NEOLAB GMBH
</TD></TR><TR><TD> Gene Pulser/MicroPulser Cuvettes, 0.1 cm </TD><TD> 165-2089 </TD><TD> Bio-Rad Laboratories GmbH
</TD></TR><TR><TD> 0.2 ml 8-Tube Strips Without Caps, Nature </TD><TD> TBS-0201 </TD><TD> Bio-Rad Laboratories GmbH
</TD></TR><TR><TD> Round-Bottom Tubule, Polypropylen, 14 ml </TD><TD> 352059 </TD><TD> NEOLAB GMBH
</TD></TR><TR><TD> Optical Flat 8-Cap Strips </TD><TD> TCS-0803 </TD><TD> Bio-Rad Laboratories GmbH
</TD></TR><TR><TD> Inoculation Loop 10 μl, Blue, Sterile </TD><TD> 2900254437 </TD></TR><TR><TD> Corning Serological Pipette 50 ml </TD><TD> 14303 </TD></TR><TR><TD> Weighing Dish 500 ST </TD><TD> 1884.1 </TD><TD> Carl Roth GmbH &amp; Co.KG
</TD></TR><TR><TD> Filter Paper </TD><TD> Z274836-1PAK </TD><TD> Sigma-Aldrich Chemie GmbH
</TD></TR><TR><TD> Inoculation Loop 10 µl</TD><TD> 2900254437 </TD><TD> NEOLAB GMBH
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Further_Recipies_and_Stocks"> Further Recipies and Stocks</SPAN></H1><H2><SPAN class="mw-headline" id="Acidovorans_Complex_Medium">Acidovorans Complex Medium</SPAN></H2><H3><SPAN class="mw-headline" id="for_1_L"> for 1 L </SPAN></H3><P><UL><LI style="font-size:14px"> 0.5 g Yeast Extract (Difco-BectonDickinson)</LI><LI style="font-size:14px"> 1.0 g Cas Amino Acids (Difco)</LI><LI style="font-size:14px"> 2.0 g Pyruvic Acid</LI><LI style="font-size:14px"> 2.0 g L-Glutamine</LI><LI style="font-size:14px"> 0.3 g KH<SUB>2</SUB>PO<SUB>4</SUB></LI><LI style="font-size:14px"> 0.3 g MgSO<SUB>4</SUB></LI><LI style="font-size:14px"> 2.0 g MOPS </LI><LI style="font-size:14px"> 4.0 g Chelex 100-resin (Sigma)</LI><LI style="font-size:14px"> PH adjusted to 7.2–7.3 with 5 M KOH</LI></UL>

Important Notes:
<UL style="font-size:14px"><LI style="font-size:14px"> Fill up to 900 ml before adding pyruvic acid and L-glutamine</LI><LI style="font-size:14px"> Adjust pH</LI><LI style="font-size:14px"> Fill up to 1L</LI><LI style="font-size:14px"> Treat with Chelex for 1h</LI><LI style="font-size:14px"> Remove Chelex by filtration</LI></UL></P><H2><SPAN class="mw-headline" id="Ampicillin_Stock_Solution"> Ampicillin Stock Solution </SPAN></H2><TABLE style="font-size:14px; width:300px"><TBODY><TR><TD>Stock</TD><TD> Ampicillin 100 mg / ml
</TD></TR><TR><TD>Amount </TD><TD> 50 ml
</TD></TR><TR><TD>Storage </TD><TD> -24°C freezer
</TD></TR><TR><TD>Notes </TD><TD> Use in 1:1000 dilution
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Efficiency_Test">Efficiency Test</SPAN></H3><DIV style="align:justify; font-size:14px;"><DIV>
Because we doubted the effiency of our ampicillin stock solution, we prepared an effiency test. TB or LB media were prepared with different ampicillin solutions in order to detect at which concentration bacteria cells carrying a mRFP expressing plasmid with ampicillin resistence loose it.</DIV><CENTER><A class="fancybox fancyFigure" href="https://static.igem.org/mediawiki/2013/2/26/AmpTest.png" title="Figure 1: Ampicilin efficiency test.(1A) Our ampicilin stock and a different ampicilin stock from the 3rd floor was diluted in 300 µl TB media, oculated with DH10beta + pSB1A6 (FannyTest plate, 2013-08-15) and grown at 37 °C for around 1.5 day. &lt;br/&gt;1B: Our ampicilin stock and a different ampicilin stock from the 3rd floor was diluted in 300 µl LB media, oculated with DH10beta + pSB1A6 (FannyTest plate, 2013-08-15) and grown at 37 °C for around 1.5 day. Additionally two &lt;b&gt;LB media from&lt;/b&gt; the fridge already prepared with resistence were tested."><FIGCAPTION style="width:80%;align:justify; font-size:14px;"><B>Figure 1A:</B> Ampicilin efficiency test: Our ampicilin stock and a different ampicilin stock from the 3rd floor was diluted in 300 µl <B>TB media</B>, oculated with DH10beta + pSB1A6 (FannyTest plate, 2013-08-15) and grown at 37 °C for around 1.5 day.<B>Figure 1B:</B> Ampicilin efficiency test: Our ampicilin stock and a different ampicilin stock from the 3rd floor was diluted in 300 µl LB media, oculated with DH10beta + pSB1A6 (FannyTest plate, 2013-08-15) and grown at 37 °C for around 1.5 day. Additionally two <B>LB media from</B> the fridge already prepared with resistence were tested.</FIGCAPTION></A></CENTER><DIV><DIV style="align:justify; clear:both;">We can conclude that our ampicilin should still be usable, since it is at least as efficient as the ampicilin stock solution optained from a different group.</DIV><H2><SPAN class="mw-headline" id="Bacitracin_Stock_Solution"> Bacitracin Stock Solution </SPAN></H2><TABLE style="font-size:14px; width:300px"><TBODY><TR><TD>Stock </TD><TD>Bacitracin 5000 U/ml
</TD></TR><TR><TD>Amount </TD><TD> 5 ml
</TD></TR><TR><TD>Storage </TD><TD> -24°C freezer
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Brevibacillus_Parabrevis_Glycerol_Stock"><I>Brevibacillus Parabrevis</I> Glycerol Stock</SPAN></H2><TABLE style="font-size:14px; width:500px; clear:both;"><TBODY><TR><TD>Stock </TD><TD><I>Brevibacillus parabrevis</I> ATCC8185 glycerol stock
</TD></TR><TR><TD>Amount </TD><TD> 16 x 1.3 ml
</TD></TR><TR><TD>Storage </TD><TD> -80°C freezer
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Chloramphenicol_Stock_Solution"> Chloramphenicol Stock Solution </SPAN></H2><TABLE style="font-size:14px; width:500px; clear:both;"><TBODY><TR><TD>Stock </TD><TD> Chloramphenicol 30 mg / ml
</TD></TR><TR><TD>Amount </TD><TD> 10 ml
</TD></TR><TR><TD>Storage </TD><TD> -24°C freezer
</TD></TR><TR><TD>Notes </TD><TD> Solved in 100% ethanol. Use in 1:3000 dilution
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="D._Acidovorans_Glycerol_Stock"><I>D. Acidovorans</I> Glycerol Stock</SPAN></H2><TABLE style="font-size:14px; width:300px; clear:both;"><TBODY><TR><TD>Stock </TD><TD><I>D. acidovorans</I> glycerol stock
</TD></TR><TR><TD>Amount </TD><TD> 10 x 1 ml
</TD></TR><TR><TD>Storage </TD><TD> -80°C freezern
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="E.Coli_BAP_1_Glycerol_Stock"><I>E.Coli</I> BAP 1 Glycerol Stock </SPAN></H2><TABLE style="font-size:14px; width:300px; clear:both;"><TBODY><TR><TD>Stock </TD><TD><I>E. coli</I> BAP1 glycerol stock
</TD></TR><TR><TD>Amount </TD><TD> 2 x 1.3 ml
</TD></TR><TR><TD>Storage </TD><TD> -80°C freezer
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="E.Coli_BAP1.2C_Competent"><I>E.Coli </I>BAP1, Competent </SPAN></H2><TABLE style="font-size:14px; width:500px; clear:both;"><TBODY><TR><TD>Stock </TD><TD><I>E. coli</I> BAP1
</TD></TR><TR><TD>Amount </TD><TD> 18 x 100 µl
</TD></TR><TR><TD>Storage </TD><TD> -80°C freezer
</TD></TR><TR><TD>Notes </TD><TD> Grows extremely fast. Be careful with miniPreps, at least in cultures with ampicillin it tends to degrade all available ampicillin and then lose the respective plasmid.
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="E._coli_BAP1-pKD46_Glycerol_Stock"><I> E. coli </I> BAP1-pKD46 Glycerol Stock </SPAN></H2><TABLE style="font-size:14px; width:500px; clear:both;"><TBODY><TR><TD>What </TD><TD><I>E. coli</I> BAP1-pKD46 glycerol stock (Amp<SUP>r</SUP>)
</TD></TR><TR><TD>Amount </TD><TD> 2 x 1.3 ml
</TD></TR><TR><TD>Storage </TD><TD> -80°C freezer
</TD></TR><TR><TD>Notes </TD><TD><B>Grow at 30°C only!</B> Growth at 37°C will lead to loss of pKD46 plasmid.
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="E.Coli_BAP1-pLF03_Glycerol_Stock"><I>E.Coli</I> BAP1-pLF03 Glycerol Stock </SPAN></H2><TABLE style="font-size:14px; width:500px; clear:both;"><TBODY><TR><TD>Stock </TD><TD><I>E. coli</I> BAP1-pLF03 glycerol stock (Amp<SUP>r</SUP>)
</TD></TR><TR><TD>Amount </TD><TD> 1 x 1.3 ml
</TD></TR><TR><TD>Storage </TD><TD> -80°C freezer
</TD></TR><TR><TD>Notes </TD><TD> Might have low amount of plasmid-carrying bacteria due to long culturing time (all Amp in medium cleaved)</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="E._Coli_DH5.CE.B1-pCP20_Glycerol_Stock"><I>E. Coli</I> DH5α-pCP20 Glycerol Stock </SPAN></H2><TABLE style="font-size:14px; width:500px; clear:both;"><TBODY><TR><TD>Stock </TD><TD><I>E. coli</I> DH5α-pCP20 glycerol stock (Amp<SUP>r</SUP>, Cm<SUP>r</SUP>)
</TD></TR><TR><TD>Amount </TD><TD> 2 x 1.3 ml
</TD></TR><TR><TD>Storage </TD><TD> -80°C freezer
</TD></TR><TR><TD>Notes </TD><TD><B>Grow at 30°C only!</B> Growth at 37°C will lead to loss of pCP20 plasmid.
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Heat-Shock_Competent_E._Coli_TOP10"> Heat-Shock Competent <I>E. Coli</I> TOP10 </SPAN></H2><TABLE style="font-size:14px; width:500px; clear:both;"><TBODY><TR><TD>Stock</TD><TD><I>E. coli</I> TOP10, Heat-shock competent
</TD></TR><TR><TD>Amount</TD><TD> 200 x 100 µl
</TD></TR><TR><TD>Storage </TD><TD> -80°C freezer
</TD></TR><TR><TD>Notes </TD><TD>Verified on 2013-06-07.
</TD></TR></TBODY></TABLE><P><CENTER><A class="fancybox fancyGraphical" href="https://static.igem.org/mediawiki/2013/2/20/TOP10_test_2013-06-07.jpg" title="Control transformation of competent TOP10 cells with 80 ng pSB4K5 with insert J04450 (IPTG-inducible mRFP production). Left: transformation with plasmid; right: transformation with water."><FIGCAPTION style="width:400px;align:justify; font-size:14px;"><B>Figure 3</B> Control transformation of competent TOP10 cells with 80 ng pSB4K5 with insert J04450 (IPTG-inducible mRFP production). Left: transformation with plasmid; right: transformation with water.</FIGCAPTION></A></CENTER></P><H2><SPAN class="mw-headline" id="E._Coli_TOP10-BBa_I746200.2FpSB1AK3_Glycerol_Stock"><I>E. Coli</I> TOP10-BBa I746200/pSB1AK3 Glycerol Stock </SPAN></H2><TABLE style="font-size:14px; width:500px; clear:both;"><TBODY><TR><TD>Stock </TD><TD><I>E. coli</I> TOP10-(BBa I746200 in pSB1AK3) (Amp<SUP>r</SUP>, Kan<SUP>r</SUP>)
</TD></TR><TR><TD>Amount </TD><TD> 1 x 1.3 ml
</TD></TR><TR><TD>Storage </TD><TD> -80°C freezer
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="E._Coli_TOP10-BBa_J04450.2FpSB3C5_Glycerol_Stock"><I>E. Coli TOP10</I>-BBa J04450/pSB3C5 Glycerol Stock </SPAN></H2><TABLE style="font-size:14px; width:500px; clear:both;"><TBODY><TR><TD>Stock </TD><TD><I>E. coli</I> TOP10-(BBa_J04450 in pSB3C5) glycerol stock (Cm<SUP>r</SUP>)
</TD></TR><TR><TD>Amount </TD><TD> 1 x 1.3 ml
</TD></TR><TR><TD>Storage </TD><TD> -80°C freezer
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="E._Coli_TOP10-pIK1_Glycerol_Stock"><I> E. Coli</I> TOP10-pIK1 Glycerol Stock </SPAN></H2><TABLE style="font-size:14px; width:500px; clear:both;"><TBODY><TR><TD>Stock </TD><TD><I>E. coli</I> TOP10-pIK1 glycerol stock (Cm<SUP>r</SUP>)
</TD></TR><TR><TD>Amount </TD><TD> 1 x 1.3 ml
</TD></TR><TR><TD>Storage </TD><TD> -80°C freezer
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="E._Coli_TOP10-pKD46_Glycerol_Stock"><I>E. Coli</I> TOP10-pKD46 Glycerol Stock </SPAN></H2><TABLE style="font-size:14px; width:500px; clear:both;"><TBODY><TR><TD>Stock </TD><TD><I>E. coli</I> TOP10-pKD46 glycerol stock (Amp<SUP>r</SUP>)
</TD></TR><TR><TD>Amount </TD><TD> 2 x 1.3 ml
</TD></TR><TR><TD>Storage </TD><TD> -80°C freezer
</TD></TR><TR><TD>Notes </TD><TD><B>Grow at 30°C only!</B> Growth at 37°C will lead to loss of pKD46 plasmid.
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="E._Coli_TOP10-pMM65_Glycerol_Stock"><I>E. Coli</I> TOP10-pMM65 Glycerol Stock </SPAN></H2><TABLE style="font-size:14px; width:500px; clear:both;"><TBODY><TR><TD>Stock </TD><TD><I>E. coli</I> TOP10-pMM65 glycerol stock (Kan<SUP>r</SUP>)
</TD></TR><TR><TD>Amount </TD><TD> 1 x 1.3 ml
</TD></TR><TR><TD>Storage </TD><TD> -80°C freezer
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="IPTG_Stock_Solution"> IPTG Stock Solution </SPAN></H2><TABLE style="font-size:14px; width:300px; clear:both;"><TBODY><TR><TD>Stock </TD><TD>IPTG 23.8 mg / ml
</TD></TR><TR><TD>Amount </TD><TD> 10 ml
</TD></TR><TR><TD>Storage </TD><TD> -20°C freezer
</TD></TR><TR><TD>Notes </TD><TD> Use  0.1 to 1 mM
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Kanamycin_Stock_Solution"> Kanamycin Stock Solution </SPAN></H2><TABLE style="font-size:14px; width:300px; clear:both;"><TBODY><TR><TD>Stock </TD><TD>Kanamycin 50 mg / ml
</TD></TR><TR><TD>Amount </TD><TD> 10 ml
</TD></TR><TR><TD>Storage </TD><TD> -24°C freezer
</TD></TR><TR><TD>Notes </TD><TD> Use in 1:1000 dilution
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="M9_Medium"> M9 Medium </SPAN></H2><TABLE style="font-size:14px; width:300px; clear:both;"><TBODY><TR><TD>Name </TD><TD>M9 Minimal Salts 5x, Powder
</TD></TR><TR><TD>Amount </TD><TD> 1 l
</TD></TR><TR><TD>Storage </TD><TD> room temperature
</TD></TR><TR><TD>Notes </TD><TD> close tightly, hygroscopic
</TD></TR></TBODY></TABLE><P><UL><LI style="font-size:14px">Add 200 ml of sterile M9 salt solution to 750 ml sterile, distilled H<SUB>2</SUB>O (45-50°C)</LI><LI style="font-size:14px">Add sterile 20 ml 20% Glucose-solution, 2 ml 1 M MgSO<SUB>4</SUB> and (optionally) 1 M CaCal</LI></UL></P><H2><SPAN class="mw-headline" id="Reactivation_Medium"> Reactivation Medium  </SPAN></H2><H3><SPAN class="mw-headline" id="for_1L"> for 1L </SPAN></H3><P><UL><LI style="font-size:14px">5.0 g Peptone</LI><LI style="font-size:14px">3.0 g Meat extract</LI><LI style="font-size:14px">1000.0 ml Distilled water</LI></UL>

Adjust pH to 7.0

</P><H2><SPAN class="mw-headline" id="Solution_for_MS_Preparation">Solution for MS Preparation</SPAN></H2><H3><SPAN class="mw-headline" id="Solution_1_.28for_MS_Sample_Preparation_Neo_Gen_Stable_Isotope_Standard_Kit_A_and_B_.28NSK-AB.2C_EurIsotop.29">Solution 1 (for MS Sample Preparation Neo Gen Stable Isotope Standard Kit A and B (NSK-AB, EurIsotop)</SPAN></H3><DIV style="align:justify; font-size:14px"><P>1. One vial of amino acid standard (NSK-A) is taken up in 1 ml methanol/water (1:1 v/v) and solubilized for ca. 15 minutes in an ultra-sonification bath. Caution: make sure that the lid is closed well and not wetted by water of the ultra-sonification bath.
2. One vial of acylcarnitine standard (NSK-B) is taken up in 1 ml of methanol and solubilized for ca. 15 minutes in the ultra-sonification bath.Caution: make sure that the lid is closed well and not wetted by water of the ultra-sonification bath.
Contents of both vials are transfered quantitatively with methanol in a 200 ml graduate flask.The graduate flask is filled up to the calibration mark with methanol.
Standard solutions get a consecutive number. When two 200 ml graduate flasks are prepared simultaneously, contents of both flasks can be combined, if levels of both standards (old and new) coincide. Afterwards the whole solution will be labeled with one common number.</P>
Stability of solution 1: 7 days</DIV><H3><SPAN class="mw-headline" id="Solution_2_.28for_MS_sample_preparation.29:_butanolic_hydrochloric_acid_.283_N.29.2C_water-free"> Solution 2 (for MS sample preparation): butanolic hydrochloric acid (3 N), water-free </SPAN></H3><DIV style="align:justify; font-size:14px"><P>1. 900 ml butanol (p.A., Merck 1.01990.1000) are cooled in an ice bath at 0°C (magnetic stirrer).
The vessel has to be closed to avoid ice condensation of water.
2. 100 ml acetylchloride (p.A., Merck 1.00031.0250) are added dropwiese from a dropping funnel over a a period of 5 minutes while stirring constantly (protection goggles and gloves!).
Caution: make sure that no water or ice gets into the solution.
3. The solution is stirred for additional 10 minutes with closed lid on ice.
</P>
Stability of solution 2: 1 year </DIV><H3><SPAN class="mw-headline" id="Solution_3_.28for_MS_sample_preparation.29"> Solution 3 (for MS sample preparation) </SPAN></H3><DIV style="align:justify; font-size:14px"><P>Acetonitrile p.A./water (1:1 v/v) (83639.320DE ACETONITRIL HIPERSOLV SUPER GR REAG.PE)</P>
Stability of solution 3: 1 year</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Heidelberg/Templates/Materials_list">http://2013.igem.org/Team:Heidelberg/Templates/Materials_list</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Templates/Materials_list" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Templates/Materials_list" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Templates/Materials_list&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Templates/Materials_list&amp;oldid=362785" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>