<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Indigoidine"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Indigoidine">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Indigoidine&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Indigoidine&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Indigoidine&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Indigoidine" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Indigoidine</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="row"><DIV id="nav" class="navigation"><DIV class="item team" data-offset-l="-130"><H2>Team</H2><UL><LI><A href="/Team:Heidelberg/Team">Team Members</A></LI><LI><A href="/Team:Heidelberg/AttributionsII">Attributions</A></LI><LI><A href="/Team:Heidelberg/Team/Sponsors">Sponsors</A></LI><LI><A href="/Team:Heidelberg/Team/Collaboration">Collaborations</A></LI><LI><A href="/Team:Heidelberg/Team/Gallery">Gallery</A></LI></UL></DIV><DIV class="item project" data-offset-l="-144"><H2>Project</H2><UL><LI><A href="/Team:Heidelberg/NRPS">Background</A></LI><LI><A href="/Team:Heidelberg/Project">Overview</A></LI><LI><A href="/Team:Heidelberg/Project/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Project/Indigoidine-Tag">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Project/Tag-Optimization">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Project/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Project/Achievements">Achievements</A></LI></UL></DIV><DIV class="item notebook" data-offset-l="-139"><H2>Notebook</H2><UL><LI><A href="/Team:Heidelberg/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Tyrocidine">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Indigoidine">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/NRPSDesigner">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/Notebook/Methods">Methods</A></LI><LI><A href="/Team:Heidelberg/Materials">Materials</A></LI></UL></DIV><DIV class="item parts" data-offset-l="-134"><H2>Parts</H2><UL><LI><A href="/Team:Heidelberg/Parts">Submitted Parts</A></LI><LI><A href="/Team:Heidelberg/Favorite_Parts">Favorite Parts</A></LI><LI><A href="/Team:Heidelberg/RFCs">Our RFCs</A></LI></UL></DIV><DIV class="item software" data-offset-l="-134"><H2>Software</H2><UL><LI><A href="/Team:Heidelberg/Project_Software">Project</A></LI><LI><A href="http://igem2013.bioquant.uni-heidelberg.de/NRPSDesigner/">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/iGEM42">iGEM42</A></LI><LI><A href="/Team:Heidelberg/OpenSource">Open Source</A></LI><LI><A href="/Team:Heidelberg/Project/AchievementsSoftware">Achievements</A></LI></UL></DIV><DIV class="item human" data-offset-l="-118"><H2>Human Practice</H2><UL><LI><A href="/Team:Heidelberg/HumanPractice">Overview</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BroadPublic">Broad Public</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Survey">Survey</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/YoungGeneration">Young Generation</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Artists">Arts and Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BeyondScience">Beyond Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Experts">Experts</A></LI></UL></DIV><DIV class="item safety" data-offset-l="-115"><H2>Safety</H2><UL><LI><A href="/Team:Heidelberg/Safety">Safety</A></LI></UL></DIV><DIV class="item model" data-offset-l="-144"><H2>Modeling</H2><UL><LI><A href="/Team:Heidelberg/Modelling/Gold_Recovery">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Modelling/Ind_Production">ind-Production</A></LI></UL></DIV><DIV class="item"><SPAN style="margin-left:6%">HOME</SPAN></DIV></DIV></DIV><DIV class="container"><DIV><H1><SPAN style="font-size:150%;color:#0B2161;">Tag-Optimization.</SPAN><SPAN class="text-muted" style="font-family:Arial, sans-serif; font-size:85%"> Exploring NRPS Efficiency by Exchanging Domains.</SPAN></H1></DIV><DIV class="row"><DIV class="col-sm-12 col-md-6"><UL class="pagination" style="margin-left:20%;"><LI><A href="#" id="backwards">«</A></LI><LI class="active month_tab" id="may"><A href="#" style="width:90px; text-align:center">May</A></LI><LI class="month_tab" id="june"><A href="#" style="width:90px; text-align:center">June</A></LI><LI class="month_tab" id="july"><A href="#" style="width:90px; text-align:center">July</A></LI><LI class="month_tab" id="august"><A href="#" style="width:90px; text-align:center">August</A></LI><LI class="month_tab" id="september"><A href="#" style="width:90px; text-align:center">September</A></LI><LI><A href="#" id="forwards">»</A></LI></UL><DIV style="margin-bottom:8%;"><DIV id="myCarousel" class="carousel slide"><DIV class="carousel-inner"><DIV class="item active may first last"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 5</H1><H3><SPAN class="mw-headline" id="Indigoidine_Production_-_bpsA">Indigoidine Production - bpsA</SPAN></H3><P>
At the beginning of our wetlab phase, we wanted to transform <I>E. coli</I> cells with plasmids containing an indigoidine synthetase and a 4'-Phosphopanthetheinyl-transferase (PPTase) to see whether we can observe blue colonies on our plates as this has been reported by groups working with indigoidine synthetases (Takahashi 2007, Brachmann2012). 
We focused on the work of Marius Müller et. al. in 2012 (Muller 2012). The group used the bpsA indigoidine synthetase (<I>blue pigment synthetase A</I>) from <I>S. lavendulae</I>  ATCC11924 (Takahashi 2007) and the PPTase svp from <I>S. verticillus</I> ATCC15003 (Sanchez 2001) to establish both a fluorescence and a chromophore based reporter assay for mammalian cells by expression of bpsA and svp which results in production of a blue pigment/ fluorophore. The group kindly supported us by sending two of their constructs, namely the pET derived expression vectors pMM64 carrying the bpsA indigoidine synthetase with an ampicillin resistance gene and pMM65 carrying svp and a kanamycin resistance gene. Both bpsA and svp rom the Fussenegger lab are codon-optimized versions for expression in eukaryotic cell lines. 
We transformed competent <I>E. coli</I> TOP10 with each plasmid to prepare plasmid DNA and perform a cotransformation of an <I>E. coli</I> Rosetta strain with both plasmids. Transformed cells which carry both plasmids should produce the blue pigment indigoidine.
</P><P>
Unfortunately, there were no blue colonies after the co-transformation. We repeated the experiment under various growth conditions; i.e. we used different incubation temperatures, light conditions, shaking and addition of ascorbic acid, which was reported to stabilize indigoidine in liquid cultures (Muller 2012).
</P></DIV></DIV></DIV><DIV class="item june first"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 6</H1><H3><SPAN class="mw-headline" id="Indigoidine_Production_-_bpsA_2">Indigoidine Production - bpsA</SPAN></H3><P>
This week we repeated the co-transformation of both the <I>E. coli</I> Rosetta and the BAP1 strain with the bpsA (pMM64) and the svp (pMM65) plasmids. The <I>E. coli</I> BAP1 strain is a variant of <I>E. coli</I> BL21(DE3) and contains the sfp PPTase from <I>B. subtilis</I> 168 (Nakano 1992, Quadri 1998) and a T7-promoter, brought in using a genomic integration approach (Pfeifer 2001). The Pfeifer lab kindly supported us by sending the BAP1 strain.
After induction, liquid cultures of positive BAP1 transformants should turn blue.
</P><P>
Both strains still didn't turn blue when grown and induced on plate and in liquid culture, even after some days with different growth conditions. We continued with the validation of the pMM-plasmids.
</P><P><A class="btn btn-large btn-primary" href="#">Learn more</A></P></DIV></DIV></DIV><DIV class="item june"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 7</H1><H3><SPAN class="mw-headline" id="Indigoidine_Production_-_bpsA_3">Indigoidine Production - bpsA</SPAN></H3><P>
Since we couldn't observe indigoidine production using pMM64 and pMM65 in the previous two weeks we validated the pMM-plasmids using restriction digests.
</P><P>
The restriction digest of pMM65 was correct but pMM64 didn't show the expected bands on an agarose gel. We continued trying to amplify the bpsA and svp gene in a PCR screening. </P><P><A class="btn btn-large btn-primary" href="#">Browse gallery</A></P></DIV></DIV></DIV><DIV class="item june"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 8</H1><H3><SPAN class="mw-headline" id="Indigoidine_Production_-_bpsA_4">Indigoidine Production - bpsA</SPAN></H3><P>
We performed PCR screenings of the pMM64 (bpsA) and pMM65 (svp) plasmid for validation. 
</P><P>
The PCR screenings showed that the pMM-plasmids are correct. We designed primers to assemble our first <A href="http://parts.igem.org/Part:pSB1C3" class="external text" rel="nofollow">pSB1C3</A> derived plasmid pKH1, which carries both the bpsA and the svp gene with the lacI-Promoter <A href="http://parts.igem.org/Part:BBa_R0010?title=Part:BBa_R0010" class="external text" rel="nofollow">BBa_R0010</A> and the ribosome binding site <A href="http://parts.igem.org/Part:BBa_B0034?title=Part:BBa_B0034" class="external text" rel="nofollow">BBa_B0034</A> before both the bpsA and the svp gene, respectively.. 
</P><P><A class="btn btn-large btn-primary" href="#">Browse gallery</A></P></DIV></DIV></DIV><DIV class="item june last"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0" id="week1"><H1>Week 9</H1><H3><SPAN class="mw-headline" id="Indigioidine_Production_.E2.80.93_bpsA">Indigioidine Production – bpsA</SPAN></H3><P>
This week we assembled our first <A href="http://parts.igem.org/Part:pSB1C3" class="external text" rel="nofollow">pSB1C3</A> derived plasmid pKH1, which carries both the bpsA and the svp gene with the lacI-Promoter <A href="http://parts.igem.org/Part:BBa_R0010?title=Part:BBa_R0010" class="external text" rel="nofollow">BBa_R0010</A> using a Gibson (Gibson 2009) and a CPEC (Quan 2009) cloning strategy. We replaced the <A href="http://parts.igem.org/Part:BBa_E1010" class="external text" rel="nofollow">mRFP1 coding sequence</A> with the bpsA and svp coding sequences with the <A href="http://parts.igem.org/Part:BBa_B0034" class="external text" rel="nofollow">RBS B0034</A>, respectively. Thus we assemble three fragments in total, which are pSB1C3ΔmRFP1 (Primer NI9/10), bpsA (Primer NI1/6) and svp (NI7/8). 
 With this plasmid it is possible to produce indigoidine in <I>E. coli</I> TOP10 without IPTG induction, since TOP10 cells lack the lac repressor gene.
</P><P>
As the PCR amplifications of all fragments for the assembly of pKH1 were positive, we could perform both the Gibson and CPEC assembly for pKH1. 
After IPTG induction, co-transformation of BAP1 with the originalpMM64 and pMM65 yielded blue liquid cultures. We compared the blue color using four samples induced by different amounts of IPTG. We couldn't detect significant correlation between indigoidine production and varying IPTG concentration. 
Since we have had problems with the plasmids received from Zurich and we wanted to use the original gene rather than a codon optimized version, we planned to extract the bpsA indigoidine synthetase gene directly from the organism it originates from, namely <I>Streptomyces lavendulae subsp. lavendulae</I> strain ATCC11924 (Takahashi 2007). As the DSMZ (<I>German Collection of Microorganisms and Cell Cultures</I>) doesn't provide this specific strain any longer, we used another <I>S. lavendulae subsp. lavendulae</I> strain (<A href="http://www.dsmz.de/catalogues/details/culture/DSM-40708.html?tx_dsmzresources_pi5%5BreturnPid%5D=304" class="external text" rel="nofollow">DSM40708</A>) expecting the bpsA gene to be present as well. We cultivated <I>S. lavendulae</I> in both YEME medium (Kieser 2000) and GYM medium (<A href="http://www.dsmz.de/microorganisms/medium/pdf/DSMZ_Medium65.pdf" class="external text" rel="nofollow">DSMZ medium No 65</A>) at 28 °C.
</P></DIV></DIV></DIV><DIV class="item july first"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0" id="week1"><H1>Week 10</H1><H3><SPAN class="mw-headline" id="Indigoidine_Production_-_bpsA_5">Indigoidine Production - bpsA</SPAN></H3><P>
For the PCR amplification of native bpsA, <I>Streptomyces lavendulae subsp. lavendulae</I> DSM40708 was grown on GYM agar plates. We wanted to use the native indigoidine synthetase bpsA and the PPTase svp from pMM65 and to assemble pRB1, which is similar to pKH1. In this new assembly approach, we amplify the bpsA gene from the <I>S. lavendulae</I> genome instead of using a codon optimized sequence. Furthermore the svp coding sequence was placed behind a weaker promoter (<A href="http://parts.igem.org/Part:BBa_B0029" class="external text" rel="nofollow">BBa_B0029</A>) since the PPTase has to activate every indigoidine synthetase only once and thus is not required in huge amounts (Lambalot 1996). 
In the subsequent weeks we wanted to exchange the Thiolation-domain (T-domain) of bpsA with T-Domains of other NRPS modules and show that it can be activated by various PPTases. We wanted to find out whether there are differences in the PPTases' efficiency concerning the activation of engineered indigoidine synthetases.
</P><P>
The PCR amplification of bpsA was unsuccessful. We will try again with a new set of primers. 
In addition we will try to amplify the native svp PPTase from <I>Streptomyces verticillus</I> ATCC15003, which has been described in previous studies (Sanchez 2001).
</P></DIV></DIV></DIV><DIV class="item july"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 11</H1><H3><SPAN class="mw-headline" id="Indigoidine_Production_-_bpsA_6">Indigoidine Production - bpsA</SPAN></H3><P><I>Streptomyces verticillus</I> ATCC15003 was cultivated and used for amplification of the svp PPTase. We tried to extract bpsA from <I>S. lavendulae</I> DSM40708 with a new set of primers (Primer RB11-20). 
</P><P>
PCR results with various primers and PCR conditions suggested that the bpsA indigoidine synthetase gene is not present or mutated in <I>S. lavendulae</I> DSM40708. The amplification of svp was successful.  <I>Photorhabdus luminescens laumondii</I> TT01 has been shown to carry an indioidine synthetase called indC (Brachmann 2012). We will use indC instead of bpsA for the following project parts.
</P></DIV></DIV></DIV><DIV class="item july"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 12</H1><H3><SPAN class="mw-headline" id="Indigoidine_Production_-_indC">Indigoidine Production - indC</SPAN></H3><P>
Since <I>S. lavendulae</I> DSM40708  does not carry the bpsA gene, we used indC from <I>Photorhabdus luminescens laumondii</I> TT01 instead (Brachmann 2012).  
The PPTase Sfp, originating from the <I>B. subtilis</I> strain 168, has been shown to exhibit a broad substrate specificity (Nakano 1988) and thus was used to activate indC on pRB3. pRB3 is similar to previous plasmids (indigoidine synthetase and PPTase on pSB1C3) but contains a KpnI cutting site as a spacer between the first ribosome binding site (RBS) and the indC coding sequence, a BamHI cutting site as a spacer between the second RBS and the sfp coding sequence as well as a NheI cutting site at the end of sfp. Therefore, this plasmid can be used with both Gibson or CPEC and a classical restriction cloning assembly strategies.  
If pRB3 is functional, we will continue testing various combinations of PPTases and indigoidine synthetases.
</P><P>
pRB3 is a functional pSB1C3 construct with the genotype <A href="http://parts.igem.org/Part:BBa_R0010" class="external text" rel="nofollow">lacI-Promoter</A>-<A href="http://parts.igem.org/Part:BBa_B0034" class="external text" rel="nofollow">RBS1</A>-KpnI-indC-<A href="http://parts.igem.org/Part:BBa_B0029" class="external text" rel="nofollow">RBS2</A>-BamHI-sfp-NheI-pSB1C3ΔmRFP1. The blue phenotype of pRB3-transformed <I>E.coli</I> TOP10 cells proofs production of the blue pigment indigoidine after 30 hours and incubation at 37 °C. Transformed cells expressing the indigoidine synthetase gene and producing indigoidine grow much slower than usual TOP10 cells. This effect was already reported by other groups (Owen 2011) and is supposed to be due to the slight toxicity of indigoidine. 
Next week we will amplify other PPTases and assemble plasmids with combinations of indC, bpsA(pMM64), sfp, svp, svp(pMM65) and entD. The ladder is the endogenous PPTase of <I>E. coli</I> and considered to be inefficient in activating indigoidine synthetases (Takahashi 2007). We want to test and compare entD with the other PPTases when overexpressed on a plasmid.
</P></DIV></DIV></DIV><DIV class="item july"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 13</H1><H3><SPAN class="mw-headline" id="Indigoidine_Production_.E2.80.93_Testing_PPTases">Indigoidine Production – Testing PPTases</SPAN></H3><P>
We wanted to test various PPTases on their functionality in activating the indigoidine synthetases indC and bpsA. The plasmids pRB3-10 represent all possible combinations of either indC or bpsA with the PPTases sfp, svp, svp (pMM65) or entD. We transformed <I>E. coli</I> TOP10 with all the plasmids.
</P><P>
All PPTases are capable of attaching the 4'-Phosphopanthetheinyl residue to the apo-form of the indigoidine synthetases as was observed by the blue phenotype of transformed <I>E. coli</I> TOP10 cells. The main focus was now on exchanging the T-Domain of the indigoidine synthetase indC to further proof the concept of modularity in the field of NRPS domains. We can easily determine which engineered versions of indC remain functionality by screening for blue colonies.
</P></DIV></DIV></DIV><DIV class="item july last"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 14</H1><H3><SPAN class="mw-headline" id="T-Domain_Shuffling_.E2.80.93_Assembly_of_ccdB_Plasmids">T-Domain Shuffling – Assembly of ccdB Plasmids</SPAN></H3><P>
In the subsequent weeks we wanted to exchange the indC T-domain with several T-domains of other NRPS modules. These are modules derived from the Tyrocidine and Delftibactin pathway, from the <I>E. coli</I> NRPS entF and two NRPS modules of unknown function from  <I>P. luminescens</I> itself. 
pRB11 (derived from pKH1), pRB12 (derived from pKH2) and pRB13 (derived from pRB3) carry the ccdB gene with a RBS instead of their native T-Domain, thus they express an unfunctional indC and the ccdb toxin which kills <I>E. coli</I> TOP10 cells. We extracted the ccdB gene from pDONR (Invitrogen, <A href="http://www.lifetechnologies.com/order/catalog/product/12536017" class="external text" rel="nofollow">pDONR™221</A>). <I>E. coli</I> OneShot ccdb survival cells produce an antidote to ccdB and are able to survive upon transformation with the ccdB carrying plasmid. After exchange of the ccdB gene with a novel T-domain only positive <I>E. coli</I> TOP10 transformants survive and the background colonies will be diminished. 
</P><P>
Assembly and/or transformations were inefficient, as blue colonies on each plate suggest that an excess of template was contained in the transformation mix. 
A ccdB PCR screening of pRB12 was positive but transformation in DH5alpha, TOP10 and OneShot ccdb survival cells showed that the ccdB gene is unfunctional. In the following, all the insert from pDONR was used including a promoter and the ccdA gene. 
Also we used a two plasmid strategy to test every possible combination of engineered indigoidine synthetases and the respective PPTases. We worked with indC on pSB1C3 and the PPTases on separate pSK3K3 derived plasmids.
</P></DIV></DIV></DIV><DIV class="item august first"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 15</H1><H3><SPAN class="mw-headline" id="Indigoidine_Production_.E2.80.93_Testing_PPTases_2"> Indigoidine Production – Testing PPTases </SPAN></H3><P>
pRB3 contains indC and sfp and results in a blue phenotype after tranformation of <I>E. coli</I> MG1655, BAP1 and TOP10. pRB4-10 plasmids code for different combinations of either indigoidine synthetase indC or bpsA and the PPTase sfp, svp, svp (pMM65) or entD. Transformation of <I>E. coli</I> TOP10 showed that all combinations result in indigoidine production. This week we validated these plasmids using colony PCR screenings and Sanger sequencing (GATC-biotech, Konstanz).
</P><P>
The PCR screening was inconclusive but sequencing results showed that all combinations of indC or bpsA with sfp, svp, svp (pMM65) or entD have been assembled correctly. 
These results are remarkable since entD was previously described as an inefficient activator of indigoidine synthetases (Takahashi 2007). In the following week we prepared separate PPTase plasmids for co-transformation with engineered versions of indC. Also, we included the delC PPTase from <I>D. acidovorans</I> SPH-1, which is the PPTase involved in delftibactin synthesis.
</P></DIV></DIV></DIV><DIV class="item august"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 16</H1><H3><SPAN class="mw-headline" id="T-Domain_Shuffling_.E2.80.93_PPTase_Plasmids">T-Domain Shuffling – PPTase Plasmids</SPAN></H3><P>
This week we wanted to prepare plasmids for the T-Domain exchange experiments. We assembled four plasmids pRB15-18, each containing one PPTase on the pSB3K3-backbone with a kanamycine resistance. These plasmids should be used for co-transformations with the engineered indigoidine synthetase indC on a pSB1C3 plasmid pRB19 with a chloramphenicole resistance. 
</P><P>
We successfully assembled pRB15-18.
</P><H3><SPAN class="mw-headline" id="BioBricks_for_the_Registry">BioBricks for the Registry</SPAN></H3><P>
As a first set of BioBricks we chose the coding sequences of the four PPTases sfp, svp, entD and delC as well as the indigoidine synthetase indC. Since indC contains two internal RFC[10] cutting sites (EcoRI and SpeI), the DNA sequence had to be mutated for the part submission. Therefore we first assembled pRB21, which is identical to pRB3 but doesn't contain a PPTase.
</P><P>
We successfully assembled pRB21 and amplified the PPTases with the BioBrick RFC[10] prefix and suffix (Primer KH13-20). Next week we cloned the PPTases into the pSB1C3 backbone using restriction cloning and removed the EcoRI and SpeI cutting sites contained in indC using a CPEC mutagenesis strategy.
</P></DIV></DIV></DIV><DIV class="item august"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 17</H1><H3><SPAN class="mw-headline" id="T-Domain_Shuffling">T-Domain Shuffling</SPAN></H3><P>
We started inserting new T-Domains into indC to check their functionality. In total we first assembled 19 variants of pRB19, which are named pRB19-Txx. The appended three-letter code identifies the respective T-domain or TTE-domain and is listed on our plasmid page in the materials section. 
</P><P>
Since PCR screenings of pRB19 were inconsistent, we started with replacing the T-Domain of pRB14, which contains sfp as a PPTase. Screenings and sequencing results showed that assembly and transformation were correct but all colonies except for that on the positive control retained a white phenotype. We tested all the other T-domains on the newly assembled pRB19 with every PPTase pRB15-18 next week.
</P><H3><SPAN class="mw-headline" id="T-Domain_Shuffling_.E2.80.93_PPTase_Plasmids_2">T-Domain Shuffling – PPTase Plasmids</SPAN></H3><P>
As the PPTase plasmids pRB15-18 showed red colonies after retransformation of a positively prepped colony, we picked new colonies and prepped them. We will do the co-transformation experiments with the new version of pRB15 but with the old ones of pRB16-18.
</P><H3><SPAN class="mw-headline" id="BioBricks_for_the_Registry_2">BioBricks for the Registry</SPAN></H3><P>
This week we removed the two internal RFC[10]-cutting sites of indC (EcoRI and SpeI) on pRB21 using a CPEC approach to yield pRB22]. The modified indC (<I>indC*</I>) was amplified introducing the RFC[10] prefix and suffix (Primer KH21/22). The four PPTases and indC* coding sequences were cloned into pSB1C3 using standard BioBrick Assembly. </P><P>
Due to faulty primer design of RB71, the resulting plasmid pRB22 was incorrect as well. We ordered a corrected version of RB78 and assembled the plasmids again in the following week.
</P></DIV></DIV></DIV><DIV class="item august"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 18</H1><H3><SPAN class="mw-headline" id="T-Domain_Shuffling_2">T-Domain Shuffling</SPAN></H3><P>
We again assembled pRB19 from pRB14 and did all T-domain-exchanges in this plasmid. After we picked and prepped positive colonies we performed co-transformations with pRB15-18 to see which PPTases are able to activate any of the T-domains inserted into indC. 
To obtain experimental validation that the strategy to use two separate plasmids works and to verify that the PPTase plasmids are free of template backbone we co-transformed TOP10 cells with pRB21 and pRB15-18.
In parallel we assembled pKH4, which is derived from pRB14. The sfp coding sequence was replaced by a nonsense part of pMM65 using the cutting sites BamHI and NheI. In this way, we were independent from the success of the assembly of pRB19.
</P><H3><SPAN class="mw-headline" id="BioBricks_for_the_Registry_3">BioBricks for the Registry</SPAN></H3><P>
After the corrected primer RB78 arrived we assembled pRB22 and cloned the BioBricks pKH6-10 for the part submission.
</P></DIV></DIV></DIV><DIV class="item august last"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 19</H1><H3><SPAN class="mw-headline" id="Domain_Shuffling_.E2.80.93_PPTase_Plasmids">Domain Shuffling – PPTase Plasmids</SPAN></H3><P>
The PPTase plasmids pRB15-18 have been reassembled with pSB3K3 from a different plate of the partsdistribution, using a CPEC approach. This week we validated those plasmids. Additionally, we assembled them by restriction cloning.
</P><H3><SPAN class="mw-headline" id="T-Domain_Shuffling_3">T-Domain Shuffling</SPAN></H3><P>
Last week we assembled pKH4, which is a pSB1C3 derived plasmid coding for indC but lacking a PPTase. We validated the plasmid and used it as the template to assemble pKH5 in which the T-domain of indC were replaced by ccdB. In parallel and as a backup strategy, we assembled pRB19 using restriction cloning. 
As a second backup strategy, we assembled pRB22 and pRB23 using CPEC with single fragments that were amplified from a <I>P. luminescens</I> cell pellet.
</P><P>
Both pKH5 and pRB22 were correct as seen by PCR screenings and sequencing. We used pRB22 to assemble pRB23, which is the ccdB-variant of pRB22, enabling to test every combination of T-domain in co-expression with the four PPTases.
</P></DIV></DIV></DIV><DIV class="item september first"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 20</H1><H3><SPAN class="mw-headline" id="T-Domain_Shuffling_4">T-Domain Shuffling</SPAN></H3><P>
We prepared pRB23-T1 to -TTE19 and performed transformations in <I>E. coli</I> TOP10 as well as co-transformations with the four PPTase plasmids pRB15-18 and a pSB3K3-plasmid with a nonsense insert. This shows whether there is a dependency of T-Domain activation on the PPTases and if there are differences in indigoidine yield using different combinations of T-Domains and PPTases. The co-transformation with a pSB3K3 plasmid with a nonsense insert shows whether the smaller size of the colonies in our co-transformation experiments is due to the co-transformation itself or to the PPTases.</P><P>
We sequenced every variant of pRB23-Txx and they were all correct. The cotransformation experiment showed, that some of the engineered indigoidine synthetases are still capable of producing the blue pigment. The indigoidine production strongly differs among the various T-Domains. All the combinations are shown in a big table in the detailled lab journal below.
</P><H3><SPAN class="mw-headline" id="T-Domain_Borders">T-Domain Borders</SPAN></H3><P>
This week we tried to assemble pRB23-b11 to -b34, which are 12 pRB22-derived plasmids, in which the indC-T-Domain was exchanged by the bpsA T-domain. In every variant, the domain borders were defined differently. Those constructs were co-transformed with the four PPTase plasmids pRB15-18 as well as with a pSB3K3 plasmid containing a nonsense insert (Transformation strain: <I>E. coli</I> TOP10). The appearance of blue colonies on the plates showed which domain borders work best for the  domain exchange between functionally related NRPS. </P><P>
The cotransformations of the pRB23-bxx CPEC assembly products with the PPTase plasmids was not successful. Therefore, we prepared plasmid DNA of the pRB23-bxx variants and repeated the co-transformation experiments.
</P><H3><SPAN class="mw-headline" id="T-Domain_Shuffling_.E2.80.93_PPTase_Plasmids_3">T-Domain Shuffling – PPTase Plasmids</SPAN></H3><P>
As there is always leaky expression of indC* - due to the fact that <I>E. coli</I> TOP10 lacks the lac repressor and we use a lacPromoter - we assembled variants of the PPTase plasmids which also carry the lacI gene with a lacPromoter. </P><H3><SPAN class="mw-headline" id="Engineered_indC_-_Quantitative_Assay">Engineered indC - Quantitative Assay</SPAN></H3><P>
Since we want to characterize the indC*-variants and PPTases in a quantitative manner, we establish an assay for measuring the correlations between T-domains, PPTases, growth rate and indigoidine production using OD measurements. We take 96 well plates and measure absorption spectra of various indigoidine producing cultures.
</P></DIV></DIV></DIV><DIV class="item september"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 21</H1><H3><SPAN class="mw-headline" id="BioBricks_for_the_Registry_4">BioBricks for the Registry</SPAN></H3><P>
This week we prepared our parts for submission to the registry. We submitted the coding sequences of indC, sfp, svp, entD and delC, as well as expression constructs coding for indC with the exchanged T-domains that resulted in a blue phenotype. These are pRB22 and pRB23-T8, -T10, -T12, -T13, -T14. We also submitted our construct in which the indC-T-Domain is exchanged by the ccdB gene. 
A table of all the parts we submitted can be found on the <I>Parts</I> site.
</P><H3><SPAN class="mw-headline" id="Engineered_indC_.E2.80.93_Quantitative_Assay">Engineered indC – Quantitative Assay</SPAN></H3><P>
We started with the detailled characterization of our engineered indC-variants using a Tecan infinite M200 plate reader to measure the absorbance spectra of 52 liquid cultures of different (co-)transformants over time. The table in the detailled lab journal shows the loading scheme of our 96 well plate and the transformants we used. 
Since a variant of indC, where the T-domain has been exchanged by the bpsA T-domain with the specific domain borders b31 resulted in a blue phenotype, we ordered primers to try other native T-Domains with these newly defined domain borders.

</P></DIV></DIV></DIV><DIV class="item september"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 22</H1><H3><SPAN class="mw-headline" id="T-domain_shuffling_5">T-domain shuffling</SPAN></H3><P>
Last week's co-transformation experiments showed that exchanging the indC-T-domain with the bpsA T-domain results in a functional indigoidine synthetase, provided specific domain borders were chosen. We now used this domain borders to also try the other native T-Domains T3-9 from <I>B. parabrevis</I>, <I>D. acidovorans</I>, <I>P. luminescens</I> and <I>E. coli</I>. We assembled the pRB23 variants T3 to T9 with the domain borders b31 and performed co-transformations with the PPtase plasmids pRB15-18.
</P><P>
We analyzed the data of the spectral absorption measurements versus time and compared the growth synamics and pigment production between different engineered indigoidine synthetases in interplay with different PPTases.
</P><H3><SPAN class="mw-headline" id="Controlling_Indigoidine_synthesis">Controlling Indigoidine synthesis</SPAN></H3><P>
Furthermore we co-transformed <I>E. coli</I> TOP10 with the lacI-Sfp-constructs and pRB22 to determine whether the indigoidine production is controllable.
</P><P>
The lacI-sfp-constructs were not functional because the transformed cells did not turn blue even if induced.  
Instead we tried the <I>E. coli</I> NEB Turbo strain which overexpresses the lac repressor. In this way, leaky expression and thereby growth retardation might be prevented.
</P></DIV></DIV></DIV><DIV class="item september last"><DIV class="container"><DIV class="carousel-caption scrollContent2" data-spy="scroll" data-target="#navbarExample" data-offset="0"><H1>Week 23</H1><H3><SPAN class="mw-headline" id="Domain_Shuffling_.E2.80.93_Indigoidine_Synthetase_Conversions">Domain Shuffling – Indigoidine Synthetase Conversions</SPAN></H3><P>
Besides the delH4 T-domain only the plu2642 T-domain resulted in a functional indigoidine synthetase when introduced to indC. We used our <I>NRPS-Designer</I> to predict the domain sequence of plu2642 and the selectivity of its A-domain and found that it is a single-module NRPS with the domain sequence A, T, TE. Its A-domain exhibits glutamine specificity, analogous to our indigoidine synthetase. Also the basic domain sequence is similar, whereas the internal oxidation domain of indC is missing in plu2642. 
Another NRPS module which shows glutamine specificity is tycC2 from the Tyrocidine pathway (further described in the &lt;a href=&quot;<A href="https://2013.igem.org/Team:Heidelberg/Project/Tyrocidine" class="external free" rel="nofollow">https://2013.igem.org/Team:Heidelberg/Project/Tyrocidine</A>&quot;&gt;Peptide Synthesis Project Page&lt;/a&gt;). 
To take the domain shuffling experiment even one step further, we assembled every possible domain combination involving the indC oxidation domain and an A-, T- and TE-domain from indC. plu2642 or tycC2 and transform a given, naturally occuring NRPS module into a functional indigoidine synthetase.
</P><H3><SPAN class="mw-headline" id="Engineered_indC_.E2.80.93_Quantitative_Assay_2">Engineered indC – Quantitative Assay</SPAN></H3><P>
This week we also repeated the spectral measurement of our engineered indC variants cotransformed with single PPTases to see which combination of PPTase and T-domain is most efficient in producing indigoidine.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-12"><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#b5" data-toggle="tab">Lab book</A></LI></UL></DIV><DIV class="jumbotron"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="b5"><P>We got two plasmids from the Fussenegger group at the ETH Zurich containing a codon optimized version of bpsA 
</P><P>(pMM64) and svp (pMM65), respectively, for expression in HEK 297-3 cells [Muller 2012]. The first goal is to 
</P><P>transform the plasmids into competent E. coli (Rosetta) and check functionality of the plasmids in our cells.
</P><H3><SPAN class="mw-headline" id="Indigoidine_production_with_pMM-plasmids_I_.28Ilia.29">Indigoidine production with pMM-plasmids I (Ilia)</SPAN></H3><P>After Transformation cells are grown first on Amp medium and thereafter on Kan medium.
</P><UL><LI> transform competent Rosetta with 225 ng pMM065 and 253.5 ng pMM064
</LI><LI> plate on Amp plate
</LI><LI> pick colonies from Amp plate, make liquid cultures in LB + Kan + IPTG(1 mM)
</LI><LI> Evening: no growth in liquid culture =&gt; prepare ON culture in LB+Amp
</LI><LI> Put ON culture in incubator
</LI><LI><A href="/wiki/index.php?title=Heat-shock_competent_bacteria&amp;action=edit&amp;redlink=1" class="new" title="Heat-shock competent bacteria (page does not exist)">prepare competent</A> Rosetta-pMM064 from ON culture
</LI><LI> transform with 225 ng pMM065
</LI><LI> plate on Kan+IPTG plate
</LI><LI> no growth on Kan plate
</LI></UL><H3><SPAN class="mw-headline" id="Results_and_Discussion">Results and Discussion</SPAN></H3><P>There were no blue colonies. Transformation will be repeated and medium will be provided with both antibiotics.
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#b6" data-toggle="tab">Lab book</A></LI></UL></DIV><DIV class="jumbotron"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="b6"><P style="font-size:12pt; text-align:justify;">
                                     
Transformed Rosetta with pMM64 and pMM65 didn't show a blue phenotype. Transformation will be repeated.
</P><H3><SPAN class="mw-headline" id="Indigoidine_production_with_pMM-plasmids_II_.28Ilia.29">Indigoidine production with pMM-plasmids II (Ilia)</SPAN></H3><UL><LI> one blue colony on plate
</LI><LI> make ON culture in LB + Amp + Kan + IPTG(1 mM)
</LI><LI> transform <A href="/wiki/index.php?title=E._coli_BAP1,_competent&amp;action=edit&amp;redlink=1" class="new" title="E. coli BAP1, competent (page does not exist)">BAP1</A> with 169 ng pMM064, plate on LB+Amp+IPTG
</LI><LI> ON culture opaque, but colorless
</LI><LI> plate contains BAP1-pMM064 colonies, but colorless
</LI><LI> prepare fresh IPTG
</LI><LI> inoculate 5 ml LB + Amp + Kan + IPTG(10 mM) with 1 ml from Rosetta-pMM064-pMM065 ON culture
</LI><LI> inoculate 5 ml LB + Amp + IPTG(10 mM) with BAP1-pMM064
</LI><LI> ON cultures opaque, but colorless
</LI><LI> according to &lt;bib id=&quot;pmid23053349&quot;/&gt;, indigoidine is extremely unstable, they did not observe any color above 
</LI></UL><P>30°C
</P><UL><LI> inoculate 5 ml LB + Amp + Kan with 1 ml from Rosetta-pMM064-pMM065 ON culture
</LI><LI> inoculate 5 ml LB + Amp with 1 ml from BAP1-pMM064 ON culture
</LI><LI> when cultures reach opaqueness: add 1 mM IPTG, shake at 25-28°C
</LI><LI> liquid cultures opaque, but colorless
</LI><LI> indigoidine extremely unstable, but incubator has transparent door, incubator room well lit
</LI><LI> prepare Amp+IPTG plate, plate BAP1-pMM064 from ON culture, leave at RT in drawer -&gt; dark
</LI><LI> inoculate 5 ml LB + Amp + Kan with 100 µl from Rosetta-pMM064-pMM065 ON culture
</LI><LI> inoculate 5 ml LB + Amp with 100 µl from BAP1-pMM064 ON culture
</LI><LI> when cultures reach opaqueness: add 1mM IPTG, wrap in tin foil (-&gt; dark), shake at 25-28°C
</LI><LI> no coloration of liquid cultures
</LI><LI> white colonies on plate
</LI></UL><H3><SPAN class="mw-headline" id="Results_and_Discussion_2">Results and Discussion</SPAN></H3><P>Cells don't show a blue phenotype. We received the E. coli BAP1-strain which is commonly used for heterologuos 
</P><P>expression of NRPS. It carries sfp, a 4'-Phosphopanthetheinyl-Transferase from Bacillus subtilis to be able to 
</P><P>activate NRPS PCP-domains (i.e. T-Domains)[Pfeifer 2001]. Transformation with pMM64 (bpsA) should result in 
</P><P>indigoidine production since sfp was previously shown to activate indigoidine synthetases [Yu 2013].
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#b7" data-toggle="tab">Lab book</A></LI></UL></DIV><DIV class="jumbotron"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="b7"><P style="font-size:12pt; text-align:justify;">
                                     
We received the E. coli BAP1-strain which is commonly used for heterologuos expression of NRPS. It carries sfp, a 
</P><P>4'-Phosphopanthetheinyl-Transferase from Bacillus subtilis to be able to activate NRPS PCP-domains (i.e. T-
</P><P>Domains)[Pfeifer 2001]. Transformation with pMM64 (bpsA) should result in indigoidine production since sfp was 
</P><P>previously shown to activate indigoidine synthetases [Yu 2013].
</P><H3><SPAN class="mw-headline" id="Indigoidine_production_with_pMM-plasmids_III_.28Ilia.29">Indigoidine production with pMM-plasmids III (Ilia)</SPAN></H3><UL><LI> co-transform <A href="/wiki/index.php?title=E._coli_BAP1,_competent&amp;action=edit&amp;redlink=1" class="new" title="E. coli BAP1, competent (page does not exist)">BAP1</A> with 84.5 ng pMM64 and 225 ng pMM65, plate on Amp + Kan + IPTG 
</LI></UL><P>plate
</P><UL><LI> inoculate 4 ml LB + Amp with BAP1-pMM64, grow at 37°C
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Gel electrophoresis of the pMM64-pMM65 digestion. Lane   1: NEB 2-log ladder; lane 2: Undigested pMM64; lane 3: Digestion of pMM64 with EcoRI+NotI; lane 4: Digestion of   pMM65 with Pst+XhoI</DIV></DIV></DIV><UL><LI> only white colonies on Amp + Kan + IPTG plate
</LI><LI> make miniPrep of BAP1-pMM64 ON culture -&gt; 19 ng / µl
</LI><LI> digest 190 ng pMM64 with EcoRI+NotI (10 µl of miniPrep, 30 µl total volume), digest 450 ng pMM65 with PstI+XhoI 
</LI></UL><P>(30 µl total volume) =&gt; expect: 2 bands at 3.5 and 4 kb for pMM64, bands at 3.3 and 0.8 kb for pMM65
</P><UL><LI> pMM64 shows fragments at 4 kb and 5 kb =&gt; wrong plasmid
</LI><LI> pMM64 shows one weak band at 4 kb =&gt; might be right, but NanoDrop concentration measurement wrong
</LI><LI> add 30 µl H<SUB>2</SUB>O to original Fussenegger filter paper of pMM64
</LI><LI> transform <A href="/wiki/index.php?title=E._coli_BAP1,_competent&amp;action=edit&amp;redlink=1" class="new" title="E. coli BAP1, competent (page does not exist)">BAP1</A> with 2 µl pMM64 from original filter paper, plate on Amp + IPTG, grow 
</LI></UL><P>at 37°C
</P><UL><LI> transform <A href="/wiki/index.php?title=E._coli_TOP10,_competent&amp;action=edit&amp;redlink=1" class="new" title="E. coli TOP10, competent (page does not exist)">TOP10</A> with 2 µl pMM64 from original filter paper, plate on Amp, grow at 
</LI></UL><P>37°C
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Gel electrophoresis of the pMM64 digestion. Lane 1: NEB 2-log   ladder; lane 2: Undigested pMM64; lane 3: Digestion of pMM64 with EcoRI+NotI</DIV></DIV></DIV><UL><LI> White BAP1-pMM64 colonies
</LI><LI> inoculate 4 ml LB + Amp with TOP10-pMM64 from plate
</LI><LI> miniPrep: 15.2 ng / µl in 27.5 µl
</LI><LI> digest 152 ng pMM64 (10 µl of miniPrep) with EcoRI+NotI (total volume 30 µl)
</LI><LI> 2 bands at 4.5 and 6 kb =&gt; wrong plasmid
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Gel electrophoresis of the pMM64-pMM65 digestion. Lane 1: NEB   2-log ladder; lane 2: Digestion of pMM64 with EcoRI+HindIII; lane 3: Digestion of pMM65 with EcoRI+HindIII</DIV></DIV></DIV><UL><LI> inoculate 4 ml LB + Amp with TOP10-pMM64 from plate
</LI><LI> miniPrep: 14.6 ng / µl in 27.5 µl
</LI><LI> digest 175.2 ng (12 µl from miniPrep) pMM64 with EcoRI+HindIII (30 µl total volume, NEB buffer 2 + BSA)
</LI><LI> digest 7 µl pMM65 with EcoRI+HindIII (30 µl total volume, NEB buffer 2 + BSA)
</LI><LI> pMM64: 2 bands at 4.5 and 6 kb =&gt; wrong plasmid
</LI><LI> pMM65: 1 band at 4 kb =&gt; might be right
</LI><LI> add 30 µl H<SUB>2</SUB>O to original Fussenegger filter paper of pMM65
</LI><LI> transform <A href="/wiki/index.php?title=E._coli_TOP10,_competent&amp;action=edit&amp;redlink=1" class="new" title="E. coli TOP10, competent (page does not exist)">TOP10</A> with 10 µl pMM65 from original filter paper, plate on Kan, grow at 
</LI></UL><P>37°C
</P><UL><LI> inoculate 5 ml LB + Kan with TOP10-pMM65
</LI><LI> miniPrep of pMM65 -&gt; 23 ng / µl
</LI><LI><A href="/wiki/index.php?title=Preparing_glycerol_stocks&amp;action=edit&amp;redlink=1" class="new" title="Preparing glycerol stocks (page does not exist)">prepare TOP10-pMM65 glycerol stock</A></LI><LI class="active"><A href="#b8" data-toggle="tab">Lab book</A></LI></UL></DIV><DIV class="jumbotron"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="b8"><H3><SPAN class="mw-headline" id="Indigoidine_production_with_pMM-plasmids_IV_.28Ilia.29">Indigoidine production with pMM-plasmids IV (Ilia)</SPAN></H3><UL><LI> inoculate 5 ml LB + Kan with TOP10-pMM65
</LI><LI> miniPrep of pMM65 -&gt; 23 ng / µl
</LI><LI><A href="/wiki/index.php?title=Preparing_glycerol_stocks&amp;action=edit&amp;redlink=1" class="new" title="Preparing glycerol stocks (page does not exist)">prepare TOP10-pMM65 glycerol stock</A></LI></UL><UL><LI> run PCR for T and TE domain of bpsA for pMM64 validation and of svp for pMM65 validation:
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">PCR for pMM64/65 validation (17-06-13); Ladder was missing, probably taken loading buffer by mistake; run at 100 V, 0.8 % gel (TAE); wanted amplicon size are: * bpsA_T: ~250 bp * bpsA_TE: ~850 bp * svp: ~800 bp</DIV></DIV></DIV><UL><LI> template each:
</LI><LI> 5 ng of miniPrep (from 2013-06-15)
</LI><LI> picked colony from transformed TOP10
</LI><LI> water as control
</LI></UL><LI> polymerase: Taq (<A href="/wiki/index.php?title=PCR&amp;action=edit&amp;redlink=1" class="new" title="PCR (page does not exist)">validation PCR</A>)
</LI><P>used cycler protocol for svp and T domain of bpsA:
</P><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 180
</TD></TR><TR><TD rowspan="3"> 12 </TD><TD> 95 </TD><TD> 30
</TD></TR><TR><TD><B>71</B> (incr. down with 0.5 °C) </TD><TD> 30
</TD></TR><TR><TD> 95 </TD><TD> 30
</TD></TR><TR><TD rowspan="3"> 18 </TD><TD> 95 </TD><TD> 30
</TD></TR><TR><TD><B>65</B></TD><TD> 30
</TD></TR><TR><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><P>used cycler protocol for TE domain of bpsA:
</P><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 180
</TD></TR><TR><TD rowspan="3"> 12 </TD><TD> 95 </TD><TD> 30
</TD></TR><TR><TD><B>63</B> (incr. down with 0.5 °C) </TD><TD> 30
</TD></TR><TR><TD> 95 </TD><TD> 30
</TD></TR><TR><TD rowspan="3"> 18 </TD><TD> 95 </TD><TD> 30
</TD></TR><TR><TD><B>57</B></TD><TD> 30
</TD></TR><TR><TD> 95 </TD><TD> 60
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> retransformation from filter paper provided bei Fussenegger group
</LI><LI><A href="/wiki/index.php?title=E._coli_TOP10,_competent&amp;action=edit&amp;redlink=1" class="new" title="E. coli TOP10, competent (page does not exist)">TOP10</A></LI><LI> with 10 µl pMM64 from original filter paper, plate on Amp, grow at 37°C
</LI><LI> with 10 µl pMM65 from original filter paper, plate on Kan, grow at 37°C
</LI></UL><LI> prepare <A href="/wiki/index.php?title=Terrific_broth_medium&amp;action=edit&amp;redlink=1" class="new" title="Terrific broth medium (page does not exist)">TB medium</A> for ON (for comparement with LB)
</LI><LI> colony PCR for plated pMM64 and pMM65 strains from 2013-06-16
</LI><LI> prepare combinatorial experiment with gibson cloning or CPEC (polysytronic expression of bpsA and svp in TOP10)
</LI><LI> prepare primer for bpsA only plasmid for BAP1 validation
</LI><P>===Results and Discussion===
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#b9" data-toggle="tab">Lab book</A></LI></UL></DIV><DIV class="jumbotron"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="b9"><P style="font-size:12pt; text-align:justify;">
                                     We want to assemble pKH1, which is a pSB1C3-derived plasmid containing pSB1C3-lacPromotor-BBa_B0034-bpsA(pMM64)-
</P><P>BBa_B0034-svp(pMM65). Single fragments are amplified and cloned using a Gibson approach.
We received Streptomyces lavendulae lavendulae since S. lavendulae ATCC11924 has been shown to carry bpsA 
</P><P>[Takahashi 2007].
</P><H3><SPAN class="mw-headline" id="Fragment_amplification_for_pKH1_.28Konrad.29">Fragment amplification for pKH1 (Konrad)</SPAN></H3><UL><LI> prepare <A href="/wiki/index.php?title=Terrific_broth_medium&amp;action=edit&amp;redlink=1" class="new" title="Terrific broth medium (page does not exist)">TB medium</A> for ON (for comparement with LB)
</LI><LI> colony PCR for plated pMM64 and pMM65 strains from 2013-06-16
</LI><LI> prepare combinatorial experiment with gibson cloning or CPEC (polysytronic expression of bpsA and svp in TOP10)
</LI><LI> prepare primer for bpsA only plasmid for BAP1 validation
</LI></UL><UL><LI> 0.4 µl template
</LI><LI> 2x 5 µl Primer (1:10 dilution)
</LI><LI> 25 µl Phusion MM
</LI><LI> 14.6 µl H2O
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> fragment </TH><TH> primer </TH><TH> template (DNA) </TH><TH> annealing temp (X1;X2) [°C] </TH><TH> elongation time (Y) [s]
</TH></TR><TR><TD> f1: bpsA (all) </TD><TD> (NI01,NI06) </TD><TD rowspan="5"> pMM64 </TD><TD> 66 (30x, no touch down) </TD><TD> 135
</TD></TR><TR><TD> f2: bpsA (AOxA) </TD><TD> (NI01,NI02) </TD><TD rowspan="5"> 68;65 </TD><TD rowspan="5"> 60
</TD></TR><TR><TD> f3: bpsA (T) </TD><TD> (NI03,NI04)
</TD></TR><TR><TD> f4: bpsA (TE) </TD><TD> (NI05,NI06)
</TD></TR><TR><TD> f5: bpsA (T-TE) </TD><TD> (NI03,NI06)
</TD></TR><TR><TD> f6: svp </TD><TD> (NI07,NI08) </TD><TD> pMM65
</TD></TR><TR><TD> f7: pSB1C3 (linear.) </TD><TD> (NI09,NI10) </TD><TD> pSB1C3 with J04450 (pJM03) </TD><TD> 66;61 </TD><TD> 90
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 30
</TD></TR><TR><TD rowspan="3"> 10 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD><B>X1</B> (incr. down with 0.5 °C) </TD><TD> 5
</TD></TR><TR><TD> 72 </TD><TD><B>Y</B></TD></TR><TR><TD rowspan="3"> 20 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD><B>X2</B></TD><TD> 5
</TD></TR><TR><TD> 72 </TD><TD><B>Y</B></TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 240
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> gel purification
</LI></UL><H3><SPAN class="mw-headline" id="Assembly_of_pKH1_and_Transformation_.28Konrad.29">Assembly of pKH1 and Transformation (Konrad)</SPAN></H3><H4><SPAN class="mw-headline" id="Gibson_approach">Gibson approach</SPAN></H4><UL><LI> molecular ratio = backbone : inserts = 1 : 3
</LI><LI> partslength: bpsA=3.8 kbp; svp=0.8 kbp; pSB1C3=2.4 kbp
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> fragment </TH><TH> volume [µl] </TH><TH> DNA amount [ng]
</TH></TR><TR><TD> f2: bpsA (AOxA) </TD><TD> 3 </TD><TD>  126
</TD></TR><TR><TD> f5: bpsA (T-TE) </TD><TD> 1.5 </TD><TD> 60
</TD></TR><TR><TD> f6: svp </TD><TD> 1.5 </TD><TD> 109
</TD></TR><TR><TD> f7: pSB1C3 (linear.) </TD><TD> 2 </TD><TD> 51
</TD></TR><TR><TD><B>total</B></TD><TD> 8 </TD></TR><TR><TD> 2 µl </TD><TD> H2O
</TD></TR><TR><TD> 10 µl </TD><TD> Gibson MM
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="CPEC_approach"> CPEC approach </SPAN></H4><P>Two different approaches were used. In the first approach 3 fragments (backbone (pSB1C3 based), bpsA and svp) were 
</P><P>used for assembly while for the second approach 4 different fragments (Backbone, AOxA, T-TE, svp) were used (like 
</P><P>for the Gibson mix). The amount of used template was calculated for backbone : insert ratio of 1:1 and for insert 
</P><P>fragment ratio at equimolar amount.
</P><UL><LI> 3 fragment mix
</LI><LI> 3 µl template (f7:backbone)
</LI><LI> 8 µl template (f1:bpsA)
</LI><LI> 0.3 µl template (f6:svp)
</LI><LI> 12.5 µl Phusion MM
</LI><LI> 1.2 µl H2O
</LI></UL><UL><LI> 4 fragment mix
</LI><LI> 3 µl template (f7:backbone)
</LI><LI> 1.7 µl template (f2:AOxA)
</LI><LI> 1.5 µl template (f5:T-TE)
</LI><LI> 0.3 µl template (f6:svp)
</LI><LI> 12.5 µl Phusion MM
</LI><LI> 6 µl H2O
</LI></UL><P>The used cycler program was carried out similar to CPEC paper for multipart assembly but I forgot to note down the 
</P><P>exact parameters. (Shame on me...)
</P><H4><SPAN class="mw-headline" id="Transformation"> Transformation </SPAN></H4><UL><LI> into TOP10
</LI><LI> 4 plates (1:1 and 1:4 of Gibson mix; 1x of each CPEC mix)
</LI></UL><H4><SPAN class="mw-headline" id="Colony_PCR"> Colony PCR </SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">colony PCR of construct assembeled by Gibson; wanted amplicon   size is around 1 kbp</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="analytical_digestion_of_selected_clones">analytical digestion of selected clones </SPAN></H4><H4><SPAN class="mw-headline" id="validation_of_transformed_BAP1_with_Fussenegger_plasmids">validation of transformed BAP1 with Fussenegger plasmids </SPAN></H4><H4><SPAN class="mw-headline" id="fusion_PCR"> fusion PCR </SPAN></H4><H4><SPAN class="mw-headline" id="Gibson_Assembly_of_bpsA_.2B_svp_insert"> Gibson Assembly of bpsA + svp insert </SPAN></H4><H4><SPAN class="mw-headline" id="PCR_for_colony_validation"> PCR for colony validation </SPAN></H4><H4><SPAN class="mw-headline" id="IPTG_induction.2C_titration"> IPTG induction, titration </SPAN></H4><H4><SPAN class="mw-headline" id="IPTG_induction.2C_titration_RESULT"> IPTG induction, titration RESULT </SPAN></H4><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Fussenegger plasmids in BAP1 under various conditions. BAP1   without any Plasmid, BAP1 transformed only with the bpsA carrying plasmid pMM64 and BAP1 transformed with the bpsA   carrying plasmid pMM64 and svp carrying pMM65 were induced with four different IPTG concentration ranging from 100   µM to 10 mM at two different temperatures (18 °C and 24°C) for around 24 h.</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="CPEC_and_transformation"> CPEC and transformation </SPAN></H4><H4><SPAN class="mw-headline" id="IPTG_induction.2C_titration_RESULT_2"> IPTG induction, titration RESULT </SPAN></H4><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">2 ml of samples from induction experiement from 2013-06-22 were   centrifugated down and then visually compared. Like it can be seen, indigoidine is exported into the supernatant.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Streptomyces_cultivation_.28Ralf.29">Streptomyces cultivation (Ralf)</SPAN></H3><P>media for streptomyces culture
</P><UL><LI> YEME medium (Kieser et al. 2000)
</LI><LI>10 g Glucose
</LI><LI>3 g Yeast extract
</LI><LI>5 g Bacteriological Peptone (I used Pankreatic Digest of Casein)
</LI><LI>3 g Malt Extract
</LI><LI>340 g Sucrose
</LI><LI>ad 1 L water
</LI></UL><UL><LI> DSMZ medium 65 GYM streptomyces medium
</LI><LI> 4 g Glucose
</LI><LI> 4 g Yeast extract
</LI><LI> 10 g Malt extract
</LI><LI> (2 g CaCO3 for plates)
</LI><LI> (12 g Agar for plates)
</LI><LI> ad 1 L water
</LI></UL><H3><SPAN class="mw-headline" id="Results_and_Discussion_3">Results and Discussion</SPAN></H3><P>BAP1+pMM64+pMM65 eventually produced indigoidine. We will now focus on assembling our own constructs starting with 
</P><P>pKH1. Streptomyces cultures look like yeast and have a strong smell as in the forest.
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#b10" data-toggle="tab">Lab book</A></LI></UL></DIV><DIV class="jumbotron"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="b10"><P>In parallel to the experiments with the pMM-plasmids (Konrad) we start to get the native bpsA from Streptomyces 
</P><P>lavendulae lavendulae DSMXXXX (Ralf).
</P><H3><SPAN class="mw-headline" id="Indigoidine_production_with_pKH1_.28Konrad.29">Indigoidine production with pKH1 (Konrad)</SPAN></H3><H4><SPAN class="mw-headline" id="colony_PCR_2"> colony PCR </SPAN></H4><H4><SPAN class="mw-headline" id="transformation_2"> transformation </SPAN></H4><H4><SPAN class="mw-headline" id="analytic_digestion"> analytic digestion </SPAN></H4><H4><SPAN class="mw-headline" id="fragment_amplification"> fragment amplification </SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">PCR for bpsA and backbone amplification and   Phusion polymerase validation (lanes to the right side of ladder; white stripe on the right is not a band); run at 135 V, 0.8 % gel (TAE); wanted amplicon size are: (bpsA: ~3.8 kbp; linearized pSB1C3: ~2.4 kbp)</DIV></DIV></DIV><P>==&gt; for two fragments with our Phusion MM and Phusion MM of SYNtheSYS (old, don't know if functional...)
</P><UL><LI> 0.4 µl template
</LI><LI> 2x 0.5 µl Primer
</LI><LI> 25 µl Phusion MM
</LI><LI> 14.6 µl H2O
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> fragment </TH><TH> primer </TH><TH> template (DNA) </TH><TH> annealing temp (X1;X2) [°C] </TH><TH> elongation time (Y) [s]
</TH></TR><TR><TD> f1: bpsA (all) </TD><TD> (NI01,NI06) </TD><TD> pMM64 </TD><TD> 65;66 </TD><TD> 120
</TD></TR><TR><TD> f7: pSB1C3 (linear.) </TD><TD> (NI09,NI10) </TD><TD> pSB1C3 with J04450 (pJM03) </TD><TD> 66;61 </TD><TD> 75
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 30
</TD></TR><TR><TD rowspan="3"> 10 </TD><TD> 98 </TD><TD> 5
</TD></TR><TR><TD><B>X1</B> (incr. down with 0.5 °C) </TD><TD> 15
</TD></TR><TR><TD> 72 </TD><TD><B>Y</B></TD></TR><TR><TD rowspan="3"> 20 </TD><TD> 98 </TD><TD> 5
</TD></TR><TR><TD><B>X2</B></TD><TD> 15
</TD></TR><TR><TD> 72 </TD><TD><B>Y</B></TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 360
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> RESULT: no PCR product for both amplicons, maybe due to wrong cycle conditions
</LI></UL><H3><SPAN class="mw-headline" id="construction_of_pSB1C3-bpsA-svp_pRB1">construction of pSB1C3-bpsA-svp pRB1</SPAN></H3><P>Streptomyces lavendulae lavendulae DSMZXXXX was cultivated in GYM medium from freeze dried cell pellet at 28 °C and 170 rpm. 
cultivation of S. lavendulae lavendulae
</P><UL><LI> freeze dried cell pellet in 5 ml medium 65
</LI><LI> 3x 200 ul in 3 mL medium 65 and YEME, respectively. 28 °C 200 rpm (3 p.m.)
</LI><LI> 3x agar plate medium 65, 50/75/100 ul inoculation; 28 °C (7 p.m.)
</LI><LI> prepare YEME agar for next day
</LI></UL><H4><SPAN class="mw-headline" id="PCR_with_Gibson_Primers_RB05-RB10">PCR with Gibson Primers RB05-RB10</SPAN></H4><UL><LI>pSB1C3 backbone from PCR product 50 uL Phusion Flash; 05 uM Primer, 25 uL Master mix, 19,4 uL water, 0,6 uL pSB1C3
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD rowspan="2"> 30 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 72  </TD><TD> 40
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 420
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf Gibson assembly and
</TD></TR></TBODY></TABLE><UL><LI>BpsA colony PCR 50 ul Phusion Flash; 0,5 uM Primer, 25 uL Master Mix, 20 uL colony with water
</LI><LI> colony was picked and held into liquid nitrogen/ warm water repeatedly
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 11
</TD></TR><TR><TD> 1 </TD><TD> 65 </TD><TD> 5
</TD></TR><TR><TD> 1 </TD><TD> 72  </TD><TD> 60
</TD></TR><TR><TD rowspan="3"> 30 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 72  </TD><TD> 60
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 300
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI>pMM65 PCR for svp 50 ul Phusion Flash; 0,5 uM Primer, 25 uL Master mix, 15 uL water, 5 uL 23 ng/ul pMM65
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 11
</TD></TR><TR><TD> 1 </TD><TD> 65 </TD><TD> 5
</TD></TR><TR><TD> 1 </TD><TD> 72  </TD><TD> 15
</TD></TR><TR><TD rowspan="2"> 30 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 72  </TD><TD> 15
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 300
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Agarose_Gel">Agarose Gel</SPAN></H4><UL><LI> pSB1C3 backbone
</LI><LI> weak band on gel &lt; 50 ng/ 17 ul
</LI><LI>Troubleshooting
</LI><LI> two step PCR -&gt; try annealing step with 66 °C annealing temp (NEB Tm Calculator Phusion)
</LI><LI> circular template
</LI><LI> template sequence?
</LI></UL><LI> bpsA
</LI><LI> cells not lysed? DNA damaged (nitrogen)?
</LI><LI> svp
</LI><H4><SPAN class="mw-headline" id=".232_PCR_pSB1C3_RB09.2F10">#2 PCR pSB1C3 RB09/10</SPAN></H4><UL><LI> 8 ul water
</LI><LI> 10 ul Phusion Flash HF MM
</LI><LI> je 0,5 ul Primer 100 uM
</LI><LI> 1 ul pSB1C3 1:5
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD rowspan="3"> 30 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 62  </TD><TD> 5
</TD></TR><TR><TD> 72  </TD><TD> 45
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 180
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><P>PCR svp RB07/08 from pMM65 23 ng/ ul circular
</P><UL><LI> 8 ul water
</LI><LI> 10 ul Phusion Flash HF MM
</LI><LI> je 0,5 ul Primer 100 uM
</LI><LI> 1 ul pMM65 1:10
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 15
</TD></TR><TR><TD rowspan="3"> 30 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 62  </TD><TD> 5
</TD></TR><TR><TD> 72  </TD><TD> 15
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 300
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><P>PCR bpsA RB05/06 from S. lavendulae
</P><UL><LI>Streptomyces was washed 3x in H20 dest; 2 &quot;colonies&quot; in 1 uL
</LI><LI> 8 ul water
</LI><LI> 10 ul Phusion Flash HF MM
</LI><LI> je 0,5 ul Primer 100 uM
</LI><LI> 1 ul Streptomyces culture
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 180
</TD></TR><TR><TD rowspan="3"> 30 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 62  </TD><TD> 5
</TD></TR><TR><TD> 72  </TD><TD> 60
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 300
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><P>0,8 % Agarose Gel
</P><UL><LI> pSB1C3 worked well with 20-50 ng/ ul
</LI><LI> BpsA
</LI><LI> annealing temp still too high?
</LI><LI> cell lysate destroys master mix? -&gt; nitrogen/ boil before PCR, use liquid phase
</LI><LI> cells not lysed? vortex with glas beads
</LI></UL><H4><SPAN class="mw-headline" id=".233_desperation_PCR">#3 desperation PCR</SPAN></H4><TABLE class="wikitable"><TBODY><TR><TH> PCR </TH><TH> water </TH><TH> Phusion Flash HF MM </TH><TH> Primer 100 uM </TH><TH> DMSO </TH><TH> template </TH><TH> template
</TH></TR><TR><TD>Ia</TD><TD>6</TD><TD>10</TD><TD>RB05, RB06 à 0,5 ul</TD><TD>1</TD><TD>2</TD><TD>S. lav, medium 65, nitrogen shock, glass beads vortex, 98 °C with water
<P>bidest; liquid phase
</P></TD></TR><TR><TD>Ib</TD><TD>6</TD><TD>10</TD><TD>RB05, RB06 à 0,5 ul</TD><TD>0</TD><TD>3</TD><TD>S. lav, medium 65, nitrogen shock, glass beads vortex, 98 °C with water
<P>bidest; liquid phase
</P></TD></TR><TR><TD>II</TD><TD>6</TD><TD>10</TD><TD>RB05, RB06 à 0,5 ul</TD><TD>0</TD><TD>3</TD><TD>S. lav, medium 65, glass beads vortex, 98 °C with water bidest; liquid phase
</TD></TR><TR><TD>III</TD><TD>6</TD><TD>10</TD><TD>RB05, RB06 à 0,5 ul</TD><TD>0</TD><TD>3</TD><TD>S. lav, medium 65, nitrogen shock, 98 °C with water bidest; liquid phase
</TD></TR><TR><TD>IVa</TD><TD>6</TD><TD>10</TD><TD>RB05, RB06 à 0,5 ul</TD><TD>1</TD><TD>2</TD><TD>S. lav, medium 65, 98 °C with water bidest; liquid phase
</TD></TR><TR><TD>IVb</TD><TD>6</TD><TD>10</TD><TD>RB05, RB06 à 0,5 ul</TD><TD>0</TD><TD>3</TD><TD>S. lav, medium 65, 98 °C with water bidest; liquid phase
</TD></TR><TR><TD>V</TD><TD>6</TD><TD>10</TD><TD>RB05, RB06 à 0,5 ul</TD><TD>0</TD><TD>3</TD><TD>S. lav, medium 65, nitrogen shock, glass beads vortex; pellet
</TD></TR><TR><TD>VI</TD><TD>6</TD><TD>10</TD><TD>RB05, RB06 à 0,5 ul</TD><TD>0</TD><TD>3</TD><TD>S. lav, YEME medium, nitrogen shock, glass beads vortex; pellet/ liquid
<P>phase
</P></TD></TR><TR><TD>VII</TD><TD>6</TD><TD>10</TD><TD>RB05, RB06 à 0,5 ul</TD><TD>0</TD><TD>3</TD><TD>S. lav, YEME medium, nitrogen shock, 98 °C with water bidest; liquid phase
</TD></TR><TR><TD>VIII</TD><TD>7</TD><TD>10</TD><TD>RB07, RB08 à 0,5 ul</TD><TD>1</TD><TD>1</TD><TD>pMM65 2,3 ng/ ul, trace RB05
</TD></TR><TR><TD>IX</TD><TD>6</TD><TD>10</TD><TD>RB07, RB08 à 0,5 ul</TD><TD>0</TD><TD>1</TD><TD>pMM65 2,3 ng/ ul, should be 8 ul water
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 120
</TD></TR><TR><TD rowspan="3"> 2 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 40  </TD><TD> 5
</TD></TR><TR><TD> 72  </TD><TD> 70
</TD></TR><TR><TD rowspan="3"> 2 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 45  </TD><TD> 5
</TD></TR><TR><TD> 72  </TD><TD> 70
</TD></TR><TR><TD rowspan="3"> 5 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 50  </TD><TD> 5
</TD></TR><TR><TD> 72  </TD><TD> 70
</TD></TR><TR><TD rowspan="3"> 5 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 55  </TD><TD> 5
</TD></TR><TR><TD> 72  </TD><TD> 70
</TD></TR><TR><TD rowspan="3"> 15 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 65  </TD><TD> 5
</TD></TR><TR><TD> 72  </TD><TD> 70
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 300
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><P>Desperation Gel
</P><DL><DD><A href="/File:Heidelberg_20130705_slav3.jpg" title="File:Heidelberg 20130705 slav3.jpg">0,8 % Agarose, 100 V, 60 min.</A></DD></DL><UL><LI> band around 200 bp? unspecific product due to PCR conditions, but then why no product?
</LI><LI> VI glowing pocket -&gt; sucrose in YEME medium? -&gt; PCR purification and new gel -&gt; still nothing
</LI></UL><P>Phusion Flash HF has proofreading activity; so mutations may stop elongation -&gt; try intron iTaq and design new 
</P><P>primers without mutations.
</P><UL><LI>PCR svp from NI7/8 PCR product and pMM65, respectively; svp with NI7/8 as control; intron iTaq 2x Master mix.
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>PCR</TH><TH>iTaq 2x Master Mix</TH><TH>template</TH><TH>Primer 10 uM</TH><TH>water
</TH></TR><TR><TD>A</TD><TD>10 ul</TD><TD> pMM65 2 ul 2,3 ng/ ul</TD><TD> 1 ul RB07 and RB08</TD><TD> 6 ul
</TD></TR><TR><TD>B</TD><TD>10 ul </TD><TD>pMM65 2 ul 2,3 ng/ ul</TD><TD> 2 ul RB07 and RB08</TD><TD>4 ul
</TD></TR><TR><TD>C</TD><TD>10 ul  </TD><TD>svp 2 ul 7,2 ng/ ul</TD><TD> 1 ul RB07 and RB08</TD><TD> 6 ul
</TD></TR><TR><TD>D</TD><TD>10 ul </TD><TD>svp 2 ul 7,2 ng/ ul</TD><TD> 2 ul RB07 and RB08</TD><TD>4 ul
</TD></TR><TR><TD>E</TD><TD>10 ul </TD><TD>svp 1 ul 7,2 ng/ ul</TD><TD> 1 ul NI07 and NI08</TD><TD>8 ul
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 94 </TD><TD> 120
</TD></TR><TR><TD rowspan="3"> 35 </TD><TD> 94 </TD><TD> 20
</TD></TR><TR><TD> 60  </TD><TD> 10
</TD></TR><TR><TD> 70  </TD><TD> 60
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 240
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><P>Gel 0,8 % 100 V 60 min
</P><UL><LI>nothing
</LI><LI> intron iTaq maybe doesn't work so well since control isn't working; or standard protocol is bad.
</LI><LI> run PCR with other Taq Master Mix
</LI></UL><LI> iTaq with Slav colony YEME
</LI><TABLE class="wikitable"><TBODY><TR><TH>Tube</TH><TH>Master Mix</TH><TH>MM 2x [ul]</TH><TH>Primer 100 mM [ul]</TH><TH>template</TH><TH>template [ul]</TH><TH>water [ul]
</TH></TR><TR><TD>A:svp RB7/8</TD><TD>Fermentas Taq PCR Master Mix 2x</TD><TD>25</TD><TD>7,5</TD><TD>svp 7,2 ng/ ul</TD><TD>10</TD><TD>0
</TD></TR><TR><TD>B:svp RB7/8</TD><TD>Fermentas Taq PCR Master Mix 2x</TD><TD>25</TD><TD>7,5</TD><TD>svp 7,2 ng/ ul</TD><TD>10</TD><TD>0
</TD></TR><TR><TD>C:pMM65 RB7/8</TD><TD>Fermentas Taq PCR Master Mix 2x</TD><TD>25</TD><TD>7,5</TD><TD>pMM65 2,3 ng/ ul</TD><TD>10</TD><TD>0
</TD></TR><TR><TD>D:pMM65 RB7/8</TD><TD>Fermentas Taq PCR Master Mix 2x</TD><TD>25</TD><TD>7,5</TD><TD>pMM65 2,3 ng/ ul</TD><TD>10</TD><TD>0
</TD></TR><TR><TD>E:pMM65 NI7/8</TD><TD>Fermentas Taq PCR Master Mix 2x</TD><TD>25</TD><TD>7,5</TD><TD>pMM65 2,3 ng/ ul</TD><TD>10</TD><TD>0
</TD></TR><TR><TD>F:svp RB7/8</TD><TD>Finnzymes Taq PCR Master Mix 2x</TD><TD>25</TD><TD>7,5</TD><TD>svp 7,2 ng/ ul</TD><TD>10</TD><TD>0
</TD></TR><TR><TD>G:svp RB7/8</TD><TD>Finnzymes Taq PCR Master Mix 2x</TD><TD>25</TD><TD>7,5</TD><TD>svp 7,2 ng/ ul</TD><TD>10</TD><TD>0
</TD></TR><TR><TD>H:pMM65 RB7/8</TD><TD>Finnzymes Taq PCR Master Mix 2x</TD><TD>25</TD><TD>7,5</TD><TD>pMM65 2,3 ng/ ul</TD><TD>10</TD><TD>0
</TD></TR><TR><TD>I:pMM65 RB7/8</TD><TD>Finnzymes Taq PCR Master Mix 2x</TD><TD>25</TD><TD>7,5</TD><TD>pMM65 2,3 ng/ ul</TD><TD>10</TD><TD>0
</TD></TR><TR><TD>J:pMM65 NI7/8</TD><TD>Finnzymes Taq PCR Master Mix 2x</TD><TD>25</TD><TD>7,5</TD><TD>pMM65 2,3 ng/ ul</TD><TD>10</TD><TD>0
</TD></TR><TR><TD>K:svp RB7/8</TD><TD>NEB OneTaq Quick Load PCR Master Mix 2x</TD><TD>25</TD><TD>7,5</TD><TD>svp 7,2 ng/ ul</TD><TD>10</TD><TD>0
</TD></TR><TR><TD>L:svp RB7/8</TD><TD>NEB OneTaq Quick Load PCR Master Mix 2x</TD><TD>25</TD><TD>7,5</TD><TD>svp 7,2 ng/ ul</TD><TD>10</TD><TD>0
</TD></TR><TR><TD>M:pMM65 RB7/8</TD><TD>NEB OneTaq Quick Load PCR Master Mix 2x</TD><TD>25</TD><TD>7,5</TD><TD>pMM65 2,3 ng/ ul</TD><TD>10</TD><TD>0
</TD></TR><TR><TD>N:pMM65 RB7/8</TD><TD>NEB OneTaq Quick Load PCR Master Mix 2x</TD><TD>25</TD><TD>7,5</TD><TD>pMM65 2,3 ng/ ul</TD><TD>10</TD><TD>0
</TD></TR><TR><TD>O:pMM65 NI7/8</TD><TD>NEB OneTaq Quick Load PCR Master Mix 2x</TD><TD>25</TD><TD>7,5</TD><TD>pMM65 2,3 ng/ ul</TD><TD>10</TD><TD>0
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 535px;"><DIV class="gallerytext"><P>500px|2-log; A-E; 2-log; F-J
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 535px;"><DIV class="gallerytext"><P>2-log; K-O; 2-log; Slav IVb; VI; V; VII
</P></DIV></DIV></TD></TR></TBODY></TABLE><UL><LI> gel analysis
</LI><LI> svp worked well as template for RB07/08, but lots of wrong product around 100-150 bp. Maybe primers bind to each other, folding results in other preferred sequence or gibson overlap has big affinity though alignments didn't show that
</LI><LI>pMM65 amplification only with NEB OneTaq
</LI><LI>no product with NI07/08 primers...because they are for TE-domain of BpsA. Ni09/10 would have been the right choice
</LI><LI> for further runs, use OneTaq and Fermentas Taq
</LI><LI> was svp amplified or is it just template?
</LI><LI> Again glowing pocket in Slav VI
</LI><LI> huge amount of primers was used in al reactions
</LI></UL><P>PCR with Phusion Flash HF from PCR products (yesterday with Taq-polymerases)
</P><UL><LI> 10 ul MM; Primer je 2 ul unv.; water 4 ul; template 2 ul
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD rowspan="3"> 30 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 55  </TD><TD> 5
</TD></TR><TR><TD> 72  </TD><TD> 30
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 300
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><P>gel 9 ul 1x loading buffer, 1 ul pcr product
</P><UL><LI>gel analysis
</LI><LI> no significant amplification of svp with Phusion Flash
</LI><LI> Taq PCR amplified template
</LI><LI> run different protocol
</LI></UL><LI> PCR Phusion flash taq-amplified svp 2
</LI><LI> 10 ul MM, 1 ul primer, 7 ul water, 1 ul water
</LI><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD rowspan="3"> 30 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 65  </TD><TD> 5
</TD></TR><TR><TD> 72  </TD><TD> 30
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 180
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD rowspan="2"> 30 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 72  </TD><TD> 35
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 180
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><P>gel with 9 ul 1x loading buffer and 1 ul pcr product
</P><UL><LI>gel analysis
</LI><LI> 3-step worked better than 2-step
</LI><LI> primers might prime each other or the fragment
</LI><LI> it's not just too much primer because lane 4/5 are a lot brighter than 6/7 -&gt; the 120 bp-band is an 
</LI></UL><P>amplification product.
-&gt; order S. verticillus and published primers from Takahashi and Sanchez
-&gt; order published Primers for sfp, entD (Lambalot), bpsA (Takahashi)
-&gt; order stuff for DNA isolation of Streptomyces
===Results and Discussion===
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#b11" data-toggle="tab">Lab book</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="b11"><H3><SPAN class="mw-headline" id="Indigoidine_production_with_pKH1_II_.28Konrad.29">Indigoidine production with pKH1 II (Konrad)</SPAN></H3><H4><SPAN class="mw-headline" id="fragment_amplification_2"> fragment amplification </SPAN></H4><P>==&gt; f1: bpsA (all) 
</P><UL><LI> 14.6 µl H2O
</LI><LI> 25 µl Phusion Flash
</LI><LI> 2x 0.5 µl Primer (NI01,NI06)
</LI><LI> 0.4 µl template pMM64
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD>10
</TD></TR><TR><TD rowspan="3"> 30 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 66 </TD><TD> 5
</TD></TR><TR><TD> 72 </TD><TD> 120
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 420
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><DL><DD>Gel purification (50µl): by accident did not take 3 but 1 Volume buffer QG
</DD><DD>Labelling fragements: 1 I and 1 II (from different lanes)
</DD></DL><H4><SPAN class="mw-headline" id="fusion_PCR_2"> fusion PCR </SPAN></H4><P>==&gt; for fusion of both fragments bpsA and svp using Phusion Flash HF
</P><UL><LI> 19.8 µl H2O
</LI><LI> 25 µl Phusion Flash MM
</LI><LI> 2x 1.0 µl Primer (NI01,NI08)
</LI><LI> 2.6 µl template (f1:bpsA)
</LI><LI> 0.6 µl template as 1:10 dilution (f6:svp)
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD rowspan="2"> 30 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 72 </TD><TD> 70
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 180
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><DL><DD>Gel purification (50µl): by accident did not take 3 but 1 Volume buffer QG
</DD><DD>Labelling fragments: 1 I and 1 II (from different lanes)
</DD></DL><H4><SPAN class="mw-headline" id="Amplification_pSB1C3_.28f7.29">Amplification pSB1C3 (f7)</SPAN></H4><P>==&gt; f7: pSB1C3 (linear.) with our Phusion MM and Phusion MM of SYNtheSYS (old, don't know if functional...)
</P><P>(add items in this order)
</P><UL><LI> 14.6 µl H2O
</LI><LI> 25 µl Phusion MM
</LI><LI> 2x 0.5 µl Primer (NI09,NI10)
</LI><LI> 0.4 µl template (pSB1C3 with J04450 (pJM03))
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 30
</TD></TR><TR><TD rowspan="2"> 30 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD> 72  </TD><TD> 90
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 420
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><P>--&gt;Hot start
</P><P>RESULT: no product neither with Synthesys fusion nor our fusion
</P><P>
==&gt; f7: pSB1C3 (linear.) with Fusion Flash
</P><P>(add items in this order)
</P><UL><LI> 14.6 µl H2O
</LI><LI> 25 µl Phusion Flash
</LI><LI> 2x 0.5 µl Primer (NI09,NI10)
</LI><LI> 0.4 µl template (pSB1C3 with J04450 (pJM03))
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD rowspan="2"> 30 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 72  </TD><TD> 40
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 420
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="fusion_PCR_3"> fusion PCR </SPAN></H4><P>==&gt; for fusion of both fragments bpsA and svp using Phusion Flash HF
</P><UL><LI> 19.8 µl H2O
</LI><LI> 25 µl Phusion Flash MM
</LI><LI> 2x 1.0 µl Primer (NI09,NI10)
</LI><LI> 5 µl template (f1:bpsA I,03.07.13)
</LI><LI> 0.6 µl template as 1:10 dilution (f6:svp)
</LI></UL><P>--&gt; wrong primers, should have taken NI01 and NI08 
--&gt; more than 50µl as not enough bpsA template was left
</P><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD rowspan="2"> 30 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 72 </TD><TD> 70
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 180
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Re-PCR_of_fusion_.28bpsA.2Bsvp.29"> Re-PCR of fusion (bpsA+svp) </SPAN></H4><P>==&gt; for amplification of fusion fusion product 03.07 (bpsA + svp) using Phusion Flash HF
</P><UL><LI> 18 µl H2O
</LI><LI> 25 µl Phusion Flash MM
</LI><LI> 2x 1.0 µl Primer (NI09,NI10)
</LI><LI> 5 µl template (fusion bpsA+svp,03.07.13)
</LI></UL><P>--&gt; wrong primers, should have taken NI01 and NI08
</P><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD rowspan="2"> 30 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 72 </TD><TD> 70
</TD></TR><TR><TD>1 </TD><TD> 72 </TD><TD> 180
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Re-PCR_of_fusion_.28bpsA.2Bsvp.29_2"> Re-PCR of fusion (bpsA+svp) </SPAN></H4><P>==&gt; for amplification of fusion fusion product 03.07 (bpsA + svp) using Phusion Flash HF
</P><UL><LI> 19 µl H2O
</LI><LI> 25 µl Phusion Flash MM
</LI><LI> 2x 0.5 µl Primer (NI01,NI08)
</LI><LI> 5 µl template (fusion bpsA+svp,03.07.13)
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 10
</TD></TR><TR><TD rowspan="2"> 30 </TD><TD> 98 </TD><TD> 1
</TD></TR><TR><TD> 72 </TD><TD> 70
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 180
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Preparations"> Preparations </SPAN></H4><UL><LI> prepare liquid culture of Rosetta + pMM64 + pMM65 in 2 ml LB with Amp + Kan for plate oculation
</LI><LI> prepare liquid cultures of TOP10 + pMM64/65 for minipreps: 3 ml TB + Amp/Kan
</LI><LI> prepare liquid cultures for induction test: 3 ml LB + appropiate AB
</LI><LI> BAP1
</LI><LI> BAP1 + pMM64 (Amp)
</LI><LI> BAP1 + pMM64 + pMM65 (Amp + Kan)
</LI><LI> TOP10 + pJH1 (Kan)
</LI></UL><H4><SPAN class="mw-headline" id="Re-PCR_of_fusion_.28bpsA.2Bsvp.29_3"> Re-PCR of fusion (bpsA+svp) </SPAN></H4><H3><SPAN class="mw-headline" id="Ralf">Ralf</SPAN></H3><P>PCR from glycerol stock S. lavendulae with Fermentas; Thermoscientific and NEB Taq 50 ul
</P><UL><LI> MM 25 ul, 7,5 ul primer RB7/8 100 mM, 5 ul water, 5 ul culture
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [s]
</TH></TR><TR><TD> 1 </TD><TD> 95 </TD><TD> 300
</TD></TR><TR><TD rowspan="3"> 30 </TD><TD> 95 </TD><TD> 30
</TD></TR><TR><TD> 50  </TD><TD> 40
</TD></TR><TR><TD> 70  </TD><TD> 300
</TD></TR><TR><TD> 1 </TD><TD> 70 </TD><TD> 300
</TD></TR><TR><TD> 1  </TD><TD> 4 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> S. verticillus and S lavendulae in 50 ml YEME medium for 46 h at 28 °C.
</LI><LI> filamentous growth and spherical colonies in S. lavendulae flask after 1 day -&gt; contamination?!
</LI><LI> no growth in S. verticillus flask after 18 h
</LI><LI> prepare plates
</LI></UL><P>colony PCRs
</P><TABLE class="wikitable"><TBODY><TR><TH>1 bpsA S. lav</TH><TH>2 svp S. vert</TH><TH>3 svp S. vert</TH><TH>4 entD MG1655</TH><TH>5 sfp BAP1
</TH></TR><TR><TD>water</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10
</TD></TR><TR><TD>Phusion Flash MM</TD><TD>25</TD><TD>25</TD><TD>25</TD><TD>25</TD><TD>25
</TD></TR><TR><TD>Primer</TD><TD>à 5 ul RB11/12</TD><TD>à 5 ul RB13/14</TD><TD>à 5 ul RB15/16</TD><TD>à 5 ul RB19/20</TD><TD>à 5 ul RB17/18
</TD></TR><TR><TD>template</TD><TD>colony pick from plate</TD><TD>cell pellet liquid culture</TD><TD>cell pellet liquid culture</TD><TD>cell pellet glycerol
<P>stock||colony from plate
</P></TD></TR><TR><TD>biometra</TD><TD colspan="4">BioRAD test machine
</TD></TR></TBODY></TABLE><P>colony PCRs 50 ul Phusion flash (25 ul + 5 ul Primer 10 mM + 10 ul water
</P><UL><LI> bpsA - S. lav (Takahashi) RB11/12
</LI><LI> svp - S. vert (Taka Primer) RB13/14
</LI><LI> svp - S. vert (Sanchez) RB15/16
</LI><LI> entD - MG1655 (Lambalot) RB19/20
</LI><LI> sfp - BAP1 RB17/18
</LI></UL><P>biometra
</P><TABLE class="wikitable"><TBODY><TR><TD>1</TD><TD>98</TD><TD>120
</TD></TR><TR><TD rowspan="3">98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>20/60
</TD></TR><TR><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>Gel
</P><P>Gradient PCR with both Streptomyces under same conditions to validate PCR parameters.
Phusion Flash HF 20 ul (10 ul MM; 2 ul Primer 10 mM; 2 uL template (cell pellet from GYM 48 h liquid culture; 6 ul 
</P><P>H2O)
</P><TABLE class="wikitable"><TBODY><TR><TD>1</TD><TD>98</TD><TD>120
</TD></TR><TR><TD rowspan="3">30</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>53-57-59-61-65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>60
</TD></TR><TR><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>Gel
</P><P>===Results and Discussion===
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#b12" data-toggle="tab">Lab book</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="b12"><H3><SPAN class="mw-headline" id="Indigoidine_production_pKH_and_pMM_.28Konrad.29">Indigoidine production pKH and pMM (Konrad)</SPAN></H3><H4><SPAN class="mw-headline" id="IPTG_Induction"> IPTG Induction </SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Induction of 2 days old cultures with/-out 1 mM IPTG. Rosetta   seems to produce indigoidine, even without induction (leaky exppression?).</DIV></DIV></DIV><P>Occulate 2 ml LB + appropiate AB with:
</P><OL><LI> Rosetta + pMM64 + pMM65
</LI><LI> BAP1
</LI><LI> BAP1 + pMM64
</LI><LI> BAP1 + pMM64 + pMM65
</LI></OL><P>Rosetta was oculated from plate (06.07.), the BAP1 culture with 10 µl of ON from 06.07. These DC were incubated for 
</P><P>around 7 hours at 37 °C.
</P><P>Old broth from 06.07 were induced with 1 mM IPTG at 24 °C for 2 hours. Rosetta strain becomes very blue, even 
</P><P>without IPTG induction.
</P><H4><SPAN class="mw-headline" id="fusion_PCR_.28bpsA.2Bsvp.29"> fusion PCR (bpsA+svp) </SPAN></H4><P>based on <A href="/wiki/index.php?title=PCA&amp;action=edit&amp;redlink=1" class="new" title="PCA (page does not exist)">PCA</A> protocol: Used two main steps, first elongation of fragments (f1;f6) without any primers, than 
</P><P>enrichment of desired fusion product with fw primer of fragment 1 and rv primer of fragment 2.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">PCA in order to gain bpsA and svp as one fragment fused together. The   wanted fragment size is ~ 4.6 kbp, so the blue indicated bands were cut out.</DIV></DIV></DIV><P>Step 1:
</P><UL><LI> 13 µl H2O
</LI><LI> 25 µl Phusion Flash MM
</LI><LI> 2 µl template (f6:svp)
</LI><LI> 10 µl template (f1:bpsA from 4.07.)
</LI></UL><P>(concentration of f1 from 4.07. unknown)
</P><P>Step 2:
</P><UL><LI> 23.5 µl H2O
</LI><LI> 25 µl Phusion Flash MM
</LI><LI> 0.5 µl template (reaction from step 1)
</LI><LI> 2x 0.5 µl primer (NI:01; NI:08)
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Step </TH><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [min]
</TH></TR><TR><TD rowspan="6"> 1 </TD><TD> 1 </TD><TD> 98 </TD><TD> 0:10
</TD></TR><TR><TD rowspan="3"> 5 </TD><TD> 98 </TD><TD> 0:01
</TD></TR><TR><TD> 52 </TD><TD> 0:30
</TD></TR><TR><TD> 72 </TD><TD> 0:20
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 5:00
</TD></TR><TR><TD> 1  </TD><TD> 12 </TD><TD> inf
</TD></TR><TR><TD colspan="4"> use only 0.5 µl of mixture for 2nd step
</TD></TR><TR><TD rowspan="5"> 2</TD><TD> 1 </TD><TD> 98 </TD><TD> 0:10
</TD></TR><TR><TD rowspan="2"> 30 </TD><TD> 98 </TD><TD> 0:01
</TD></TR><TR><TD> 72 </TD><TD> 1:10
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 5:00
</TD></TR><TR><TD> 1  </TD><TD> 12 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> Gel extraction of ~4.6 kbp bands cut out
</LI></UL><H4><SPAN class="mw-headline" id="Gibson_assembly"> Gibson assembly </SPAN></H4><UL><LI> molecular ratio = backbone : inserts = 1 : 3
</LI><LI> partslength: bpsA=3.8 kbp; svp=0.8 kbp; pSB1C3=2.4 kbp
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> fragment </TH><TH> volume [µl] </TH><TH> DNA amount [ng]
</TH></TR><TR><TD> f8: bpsA + svp </TD><TD> 7.5 </TD><TD>  214
</TD></TR><TR><TD> f7: pSB1C3 (linear.) </TD><TD> 2.5 </TD><TD> 55
</TD></TR><TR><TD><B>total</B></TD><TD> 10 </TD></TR><TR><TD> 0 µl </TD><TD> H2O
</TD></TR><TR><TD> 10 µl </TD><TD> Gibson MM
</TD></TR></TBODY></TABLE><UL><LI> transformation of TOP10 with 2 µl directly of GA reaction mix and 1:4 dilution respectively
</LI></UL><H4><SPAN class="mw-headline" id="additional"> additional </SPAN></H4><UL><LI> preparations of JM1 from 4.5 ml TOP10 ON culture (used a bit for transformation, gave rest to Johanna)
</LI><LI> transformation of BAP1 with 88 ng (5 µl) JM1 (probably way to much DNA)
</LI><LI> IPTG induction (1mM) experiment with Rosetta + pMM64 + pMM65, BAP1, BAP1 + pMM64, BAP1 + pMM64 + pMM65 (DC, was 
</LI></UL><P>in 4 °C fridge overnight)
</P><UL><LI> RESLUT: did not turn blue at all, may the IPTG be bad? (did not use the same as the day before)
</LI></UL><H4><SPAN class="mw-headline" id="CPEC_Assembly"> CPEC Assembly </SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">CPEC for pKH1 assembly: only insert seems visible, no backbone and CPEC   product; run at 135 V, 0.8 % gel (TAE); wanted amplicon size is: 2,4 kbp</DIV></DIV></DIV><TABLE class="wikitable"><TBODY><TR><TH> fragment </TH><TH> volume [µl] </TH><TH> DNA amount [ng]
</TH></TR><TR><TD> f8: bpsA + svp </TD><TD> 2.3 </TD><TD>  150
</TD></TR><TR><TD> f7: pSB1C3 (linear.) </TD><TD> 5.2 </TD><TD> 50
</TD></TR><TR><TD> total </TD><TD> 7.5 </TD></TR><TR><TD> 2.5 µl </TD><TD> H2O
</TD></TR><TR><TD> 10 µl </TD><TD> PhusionFlash MM
</TD></TR><TR><TD><B>total</B></TD><TD> 20 </TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [min]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 0:10
</TD></TR><TR><TD rowspan="3"> 3 </TD><TD> 98 </TD><TD> 0:01
</TD></TR><TR><TD> 55 </TD><TD> 0:30
</TD></TR><TR><TD> 72 </TD><TD> 2:00
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 5:00
</TD></TR><TR><TD> 1  </TD><TD> 12 </TD><TD> inf
</TD></TR></TBODY></TABLE><UL><LI> use 10 µl for gel picture, store 10 µl
</LI></UL><H4><SPAN class="mw-headline" id="PCR_amplification_f7_.28pSB1C3.29"> PCR amplification f7 (pSB1C3) </SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">PCR for amplification of pSB1C3: (<B>left</B>) lane 1 &amp; 2: todays   amplification, lane 3: Marker, lane 4: 2 µl of gel/nucleotide purified amplification of 4th of July, lane 5 &amp; 6:   delRest 11 - 12 with long and short primers (unknown order); (<B>right</B>) gel cut out run at 135 V, 0.8 % gel (TAE); wanted amplicon size is: 2,4 kbp</DIV></DIV></DIV><P>(add items in this order)
</P><UL><LI> 14.6 µl H2O
</LI><LI> 25 µl Phusion MM
</LI><LI> 2x 0.5 µl Primer (NI09,NI10)
</LI><LI> 0.4 µl template (pSB1C3 with J04450 (pJM03))
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [min]
</TH></TR><TR><TD> 1 </TD><TD> 98 </TD><TD> 0:10
</TD></TR><TR><TD rowspan="2"> 30 </TD><TD> 98 </TD><TD> 0:01
</TD></TR><TR><TD> 72  </TD><TD> 0:40
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 5:00
</TD></TR><TR><TD> 1  </TD><TD> 12 </TD><TD> inf
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Ralf_2">Ralf</SPAN></H3><P>new primers are there yippie
</P><P>pSB1C3
RB21/22
</P><UL><LI> 2,5 ul template pSB1C3 1:5
</LI><LI> 25 ul Phusion Flash MM
</LI><LI> 5 ul Primer 10 mM
</LI><LI> 12,5 ul water
</LI></UL><TABLE class="wikitable"><TBODY><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">1</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>61</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>60
</TD></TR><TR><TD rowspan="2">29</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>72</TD><TD>60
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>T1000 BIORAD
</P><P>indC-Plum
RB27/28
</P><UL><LI> 1 ul template cell pellet from DSMZ stem culture
</LI><LI> 25 ul Phusion Flash MM
</LI><LI> 5 ul Primer 10 mM
</LI><LI> 14 ul water
</LI></UL><TABLE class="wikitable"><TBODY><TR><TD>1</TD><TD>98</TD><TD>120
</TD></TR><TR><TD rowspan="3">1</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>57</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>90
</TD></TR><TR><TD rowspan="3">29</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>90
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>10</TD><TD>-
</TD></TR></TBODY></TABLE><P>biometra
</P><P>sfp-BAP1
RB35/36
</P><UL><LI> 1 ul template cell pellet from BAP1 liquid culture
</LI><LI> 25 ul Phusion Flash MM
</LI><LI> 5 ul Primer 10 mM
</LI><LI> 14 ul water
</LI></UL><TABLE class="wikitable"><TBODY><TR><TD>1</TD><TD>98</TD><TD>120
</TD></TR><TR><TD rowspan="3">1</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>63</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>30
</TD></TR><TR><TD rowspan="3">29</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>30
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>10</TD><TD>-
</TD></TR></TBODY></TABLE><P>BioRAD 2-block
</P><P>Electrophoresis 1 % Agarose 0,5 % TAE 100 V 60 min; PCR product 50 ul + 10 ul 6x loading dye
10 ul quickload 2-log; 2 ul PCR pSB1C3-indC-sfp; 29 ul 2x pSB1C3; 2x indC; 2x sfp
</P><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>400px|1st run with long primers
</P></DIV></DIV><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>400px|cut backbone for gel extraction
</P></DIV></DIV><P>indC-Plum #2
Q5 50 ul
1, 2: RB37/38; 
3:    RB27/28
</P><UL><LI> 5 ul template liquid culture - pellet
</LI><LI> 25 ul Q5 MM
</LI><LI> 5 ul Primer 10 mM
</LI><LI> 10 ul water
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>cycles</TH><TH>temperature °C</TH><TH>time seconds
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>180
</TD></TR><TR><TD rowspan="3">30</TD><TD>98</TD><TD>8
</TD></TR><TR><TD>56</TD><TD>10
</TD></TR><TR><TD>72</TD><TD>150
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>sfp BAP1 #2
Q5 50 ul
a: RB35/36 
b: RB43/44
</P><UL><LI> template colony pick
</LI><LI> 25 ul Q5 MM
</LI><LI> 5 ul Primer 10 mM
</LI><LI> 15 ul water
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>cycles</TH><TH>temperature °C</TH><TH>time seconds
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>180
</TD></TR><TR><TD rowspan="3">30</TD><TD>98</TD><TD>8
</TD></TR><TR><TD>62</TD><TD>10
</TD></TR><TR><TD>72</TD><TD>40
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>2-log (1 ug); (1 ul) 1 - 2 - 3 - a - b; (all) 1 - 3 - 3 - a - b - b
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>2-log (1 ug); (1 ul) 1 - 2 - 3 - a - b; (all) 1 - 3 - 3 - a - b - b
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>analysis
</P><UL><LI>indC fragmet with short primers worked well -&gt; gel extraction and re-PCR with long primers
</LI><LI>sfp and indC long almost nothing -&gt; improve conditions
</LI></UL><P>sfp BAP1 #3
Q5 50 ul
A: RB17/18; naka; 60 °C
B: RB17/18; naka; 56 °C
C: RB43/44; short; 60 °C
D: RB43/44; short; 56 °C
E: RB35/36; long; 60 °C
F: RB35/36; long; 65 °C
</P><UL><LI> template colony pick
</LI><LI> 25 ul Q5 MM
</LI><LI> 5 ul Primer 10 mM
</LI><LI> 15 ul water
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>cycles</TH><TH>temperature °C</TH><TH>time seconds
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>180
</TD></TR><TR><TD rowspan="3">30</TD><TD>98</TD><TD>8
</TD></TR><TR><TD>56-60-65</TD><TD>10
</TD></TR><TR><TD>72</TD><TD>30
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>2-log (1 ug); A-F
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>analysis
</P><UL><LI>B, C and E worked best -&gt; gel extraction
</LI><LI> re-pcr E
</LI></UL><H4><SPAN class="mw-headline" id="fragments_for_pRB3">fragments for pRB3</SPAN></H4><UL><LI>rePCR from indC-Plum #2 with long primers
</LI><LI>2x 50 ul Phusion Flash HF
</LI><LI> RB27/28 long primer for pRB3 5 ul 10 uM
</LI><LI>water 14.8 ul
</LI><LI>template 0.2 ul pcr gel extracted
</LI></UL><P>biorad T100
</P><TABLE class="wikitable"><TBODY><TR><TH>cycles</TH><TH>temperature °C</TH><TH>time seconds
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">12</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>td 57-53 -0.5</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>70
</TD></TR><TR><TD rowspan="3">18</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>70
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><UL><LI>rePCR from sfp-BAP1 #3 E with long primers
</LI><LI>2x 50 ul Phusion Flash HF
</LI><LI> RB35/36 long primer for pRB3 5 ul 10 uM
</LI><LI>water 14.8 ul
</LI><LI>template 0.2 ul pcr gel extracted
</LI></UL><P>biorad T100
</P><TABLE class="wikitable"><TBODY><TR><TH>cycles</TH><TH>temperature °C</TH><TH>time seconds
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">30</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>20
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><UL><LI>rePCR from pSB1C3 Hanna 1:5 with long primers
</LI><LI>2x 50 ul Phusion Flash HF
</LI><LI> RB21/22 long primer for pRB3 5 ul 10 uM
</LI><LI>water 14.8 ul
</LI><LI>template 0.2 ul pcr gel extracted
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>cycles</TH><TH>temperature °C</TH><TH>time seconds
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">12</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>td 64 -0.5</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>70
</TD></TR><TR><TD rowspan="3">23</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>70
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>2-log (1 ug); pSB1C3; indC; sfp
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>2-log (1 ug); pSB1C3; indC; sfp
</P></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="assembly_pRB3">assembly pRB3</SPAN></H4><UL><LI>digest 25 ul (2.5 CutSmart 10x; 0.5 enzyme each; 21.5 DNA; 1 h at 37 °C)
</LI><LI>indC KpnI-HF and BamHI-HF
</LI><LI>sfp BamHI-HF and NheI-HF
</LI><LI>pSB1C3 NheI-HF and KpnI-HF
</LI></UL><UL><LI>Ligation 1 20 ul (2.0 T4 buffer 10x; 1.0 T4; 10.0 indC; 4.0 sfp; 3.0 pSB1C3)
</LI><LI>Ligation 2 
</LI><LI> 2.0 T4 buffer 10x; 1.0 T4; 7.0 indC; 10.0
</LI><LI> 2.0 T4 buffer 10x; 1.0 T4; 10.0 product; 7.0 pSB1C3
</LI></UL><UL><LI>CPEG 20 ul Phusion Flash HF (10 MM; 7 indC; 2 pSB1C3; 1 sfp)
</LI></UL><P>biometra
</P><TABLE class="wikitable"><TBODY><TR><TH>cycles</TH><TH>temperature °C</TH><TH>time seconds
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">10</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>3grad. 50 +5</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>70
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>180
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><UL><LI>Transformation TOP10
</LI><LI>CPEG 50 5 ul
</LI><LI> CPEG 55 5 ul
</LI><LI> CPEG 55 plus 20 min T4 ligase 10 ul (5 ul + 5 ul ligation master mix)
</LI><LI> CPEG 60 5 ul
</LI><LI> Lig1 2 ul
</LI><LI> Lig1 4 ul
</LI><LI> Lig2 4 ul
</LI></UL><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>TOP10-pRB3-CPEG50-5ul
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>TOP10-pRB3-CPEG55-5ul
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>TOP10-pRB3-CPEG55lig-5ul
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>TOP10-pRB3-CPEG60-5ul
</P></DIV></DIV></TD></TR><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>TOP10-pRB3-lig1-2ul
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>TOP10-pRB3-lig1-4ul
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>TOP10-pRB3-lig2-4ul
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>-&gt; order primers for the other PPTases and indigoidine synthetases (set RB21-44 complete)
</P><P>===Results and Discussion===
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#b13" data-toggle="tab">Lab book</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="b13"><H3><SPAN class="mw-headline" id="Indigoidine_production_Konrad_.28Konrad.29">Indigoidine production Konrad (Konrad)</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">PCR for amplification of pSB1C3: (<B>left</B>) first analytical   gel; (<B>right</B>) second analytical gel. f7 was twice eluted after gel extraction, the second run was 30 min in 15   µl H<SUB>2</SUB>O at 50 °Ca before centrifugation; run at 100 V, 0.8 % gel (TAE);</DIV></DIV></DIV><UL><LI> gel extraction of yesterdays fragment f7:pSB1C3
</LI><LI> analytical gel (small, half size): (M: 3 µl NEB log2, 2 µl f7 + 2 µl 1x loading dye)==&gt; [f7] = 3 ng/µl
</LI><LI> 2nd analytical gel (small, half size): (M: 3 µl NEB log2, (2 µl f1:4.07., 2 µl f7:14.07., 2 µl f7:14.07.-2nd 
</LI></UL><P>eluation, 2 µl delRest-f6-9) + 8 µl 1x loading dye)==&gt; [f1:4.07.] = 18 ng/µl, [f7:14.07.] = 5 ng/µl, 
</P><P>[f7:14.07.-2nd] = 2 ng/µl,[delRest-f] = 5 ng/µl
</P><H4><SPAN class="mw-headline" id="fusion_PCR_.28bpsA.2Bsvp.29_2"> fusion PCR (bpsA+svp) </SPAN></H4><P>based on <A href="/wiki/index.php?title=PCA&amp;action=edit&amp;redlink=1" class="new" title="PCA (page does not exist)">PCA</A> protocol: Used two main steps, first elongation of fragments (f1;f7) without any primers, than 
</P><P>enrichment of desired fusion product with fw primer of fragment 1 and rv primer of fragment 2.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">PCA in order to gain bpsA and pSB1C3 as one fragment fused together. The   wanted fragment size is ~ 6.2 kbp.</DIV></DIV></DIV><P>Step 1:
</P><UL><LI> 10 µl Phusion Flash MM
</LI><LI> 6.3 µl template (f7:pSB1C3)
</LI><LI> 3.7 µl template (f1:bpsA from 4.07.)
</LI></UL><P>Step 2:
</P><UL><LI> 20.0 µl H<SUB>2</SUB>O
</LI><LI> 25 µl Phusion Flash MM
</LI><LI> 4.0 µl template (reaction from step 1)
</LI><LI> 2x 0.5 µl primer (NI:06; NI:09)
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Step </TH><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [min]
</TH></TR><TR><TD rowspan="6"> 1 </TD><TD> 1 </TD><TD> 98 </TD><TD> 0:10
</TD></TR><TR><TD rowspan="3"> 5 </TD><TD> 98 </TD><TD> 0:01
</TD></TR><TR><TD> 52 </TD><TD> 0:30
</TD></TR><TR><TD> 72 </TD><TD> 0:20
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 5:00
</TD></TR><TR><TD> 1  </TD><TD> 12 </TD><TD> inf
</TD></TR><TR><TD colspan="4"> use only 4.0 µl of mixture for 2nd step
</TD></TR><TR><TD rowspan="5"> 2</TD><TD> 1 </TD><TD> 98 </TD><TD> 0:10
</TD></TR><TR><TD rowspan="2"> 30 </TD><TD> 98 </TD><TD> 0:01
</TD></TR><TR><TD> 72 </TD><TD> 1:10
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 5:00
</TD></TR><TR><TD> 1  </TD><TD> 12 </TD><TD> inf
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Preparation_for_electroporation"> Preparation for electroporation </SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">analytical gel of purified Gibson assembly and delRest;run at 100 V, 0.8 %   gel (TAE);</DIV></DIV></DIV><UL><LI> Gibsonassembly purified with Nucleotide Removal Kit
</LI><LI> analytical gel (small, half size): (M: 2.5 µl NEB log2)==&gt; [delRest(short)] = 6-7 ng/µl, [delRest(long)] = 4 
</LI></UL><P>ng/µl, [GA] = 3 ng/µl
</P><H4><SPAN class="mw-headline" id="Validation_of_Gibson_assembly"> Validation of Gibson assembly </SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:271px;"><DIV class="thumbcaption">Gibson assembly. The wanted fragment size is ~ 7.0 kbp.;run at 100 V, 0.8   % gel (TAE);</DIV></DIV></DIV><UL><LI> put 10 µl on gel.
</LI><LI> electroporation with left 4 µl
</LI></UL><H4><SPAN class="mw-headline" id="fusion_PCR_.28bpsA.2Bsvp.29_3"> fusion PCR (bpsA+svp) </SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">PCA in order to gain bpsA and pSB1C3 as one fragment fused together. The   wanted fragment size is ~ 6.2 kbp.;run at 135 V, 0.8 % gel (TAE);</DIV></DIV></DIV><P>Redo 2nd step with 1st step result of the 13th of July
</P><P>Step 2:
</P><UL><LI> 23.5 µl H<SUB>2</SUB>O
</LI><LI> 25 µl Phusion Flash MM
</LI><LI> 0.5 µl template (reaction from step 1)
</LI><LI> 2x 0.5 µl primer (NI:06; NI:09)
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Step </TH><TH> Cycles </TH><TH> temperature [°C] </TH><TH> Time [min]
</TH></TR><TR><TD rowspan="5"> 2</TD><TD> 1 </TD><TD> 98 </TD><TD> 0:10
</TD></TR><TR><TD rowspan="2"> 30 </TD><TD> 98 </TD><TD> 0:01
</TD></TR><TR><TD> 72 </TD><TD> 1:10
</TD></TR><TR><TD> 1 </TD><TD> 72 </TD><TD> 5:00
</TD></TR><TR><TD> 1  </TD><TD> 12 </TD><TD> inf
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="soil_sample_preparation"> soil sample preparation </SPAN></H4><UL><LI> experimental CPEC assembly with left mix from day before
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Analytical gel for verification of CPE success.</DIV></DIV></DIV><P>File:Heidelberg_20130719-CPEC.png
</P><UL><LI> PCR of bpsA and svp with new Primers
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">PCR amplification of svp with new Primers.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">PCR amplification of bpsA with new Primers.</DIV></DIV></DIV><UL><LI> gel extraction of new bpsA and svp fragments
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Analytical gel bpsA and svp gel extraction.</DIV></DIV></DIV><UL><LI> Gibson Assembly
</LI><LI> CPEC
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Analytical gel for verification of Gibson assembly and CPEC   success.</DIV></DIV></DIV><UL><LI> Trafo
</LI></UL><UL><LI> Colony PCR of electroporation with GA
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Colony PCR for pKH1 validation. The T-TE domain of bpsA   was used as amplicon (size: 1.1 kbp) as well as svp-part of backbone (similar size of 1.1 kbp). Every lane   represents one PCR reaction with 1 picked colony each. Except for colony 1 only black colonies were picked.</DIV></DIV></DIV><UL><LI> Colony PCR of trafos of day before
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Colony PCR for pKH1 und pKH2 validation. The T domain   of bpsA was used as amplicon (size: 300 bp). Every lane represents one PCR reaction with 3 picked colony each.</DIV></DIV></DIV><UL><LI> Heat-shock competent Rosetta
</LI><LI> Minipreparation of 2 ml ON culture of one picked black colony of GA plate (electroporation)
</LI></UL><H3><SPAN class="mw-headline" id="Ralf_3">Ralf</SPAN></H3><P>PCR PPTases #1 Phusion Flash HF 50 ul (long) 20 ul (short); colony pick/0.2 ul pcr template; primer 2/5 ul 10 uM
</P><UL><LI> MG1655 pick RB33/34 and RB 41/42
</LI><LI> S. vert pick liquid RB29/30 and RB 39/40
</LI><LI> pMM65 pcr RB25/26
</LI></UL><P>biorad T100
</P><TABLE class="wikitable"><TBODY><TR><TH>cycles</TH><TH>temperature °C</TH><TH>time seconds
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>180/10
</TD></TR><TR><TD rowspan="3">12</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>td 68 -0.5</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>20
</TD></TR><TR><TD rowspan="3">18</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>20
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>180
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>PCR bpsA64 Phusion Flash HF 50 ul (long); 0.2 ul pcr template; primer 5 ul 10 uM
</P><UL><LI> pMM64 Fussenegger miniprep RB23/24
</LI></UL><P>biometra
</P><TABLE class="wikitable"><TBODY><TR><TH>cycles</TH><TH>temperature °C</TH><TH>time seconds
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">12</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>td 58 -0.5</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>70
</TD></TR><TR><TD rowspan="3">18</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>70
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>bpsA; entD-l; svpV-l; svpF-l; entD-s; svpV-s
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>bpsA; entD-l; svpV-l; svpF-l; entD-s; svpV-s
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>
PCR PPTases #3 Phusion Flash HF 50 ul (25 MM; 5/5 primer; 14.5 water; 0.5 template)
</P><UL><LI>1: entD from PPTases #1 gel ex w/ RB33/34 long
</LI><LI>2: svpV from ??? w/ RB29/30 long
</LI><LI>3: svpF from PPTases #1 gel ex w/ RB25/26 long
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>cycles</TH><TH>temperature °C</TH><TH>time seconds
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">30</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>20
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>180
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>1 ug 2-log; entDlong; svpVlong; svpFlong
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>1 ug 2-log; entDlong; svpVlong; svpFlong
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>analysis
</P><UL><LI> entD and svpF -&gt; gel extraction
</LI><LI> svpV is too small -&gt; wrong product. new colony PCR with short primers
</LI></UL><P>Konrad PCR svpV #3
</P><TABLE class="wikitable"><TBODY><TR><TH>cycles</TH><TH>temperature °C</TH><TH>time seconds
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>180
</TD></TR><TR><TD rowspan="3">30</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>3grad. 65-62-57</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>20
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>180
</TD></TR><TR><TD>1</TD><TD>10</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>2-log; RB13/14 65°C; RB39/40 62 °C; RB 39/40 57 °C
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>analysis
</P><UL><LI>hmm
</LI></UL><P>PCR svpV #4 20 ul Phusion Flash HF (10 MM; 9.4 water; 0.2 ul primer 100 uM; 0.2 cell pellet)
conditions as in Jul 12th; RB13/14
biorad T100
</P><TABLE class="wikitable"><TBODY><TR><TH>cycles</TH><TH>temperature °C</TH><TH>time seconds
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>120
</TD></TR><TR><TD rowspan="3">30</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>20
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>2-log; Hanna; svpV RB13/14
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>2-log; Hanna; svpV RB13/14
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>rePCR svpV #5 from #4 50 ul (25 MM; 14 water; 0.5 primer 100 uM; 0.2 template)
</P><UL><LI> 1: RB29/30 from short RB13/14
</LI><LI> 2: RB29/30 from short RB 39/40
</LI></UL><P>biorad T100
</P><TABLE class="wikitable"><TBODY><TR><TH>cycles</TH><TH>temperature °C</TH><TH>time seconds
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">10</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>td 65 -0.5</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>20
</TD></TR><TR><TD rowspan="3">25</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>20
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>2-log; from taka; from RB39/40
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>2-log; from taka; from RB39/40
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>-&gt; gel extraction; CPEG assembly
</P><TABLE class="wikitable"><TBODY><TR><TH>plasmid name</TH><TH>backbone</TH><TH>cutting site</TH><TH>indigoidine-Synthetase</TH><TH>cutting site</TH><TH>PPTase</TH><TH>cutting site</TH><TH>primers
</TH></TR><TR><TD>pRB3</TD><TD>pSB1C3-BBa_B0034</TD><TD>KpnI</TD><TD>P.lum indC</TD><TD>BamHI</TD><TD>Bsub sfp</TD><TD>NheI</TD><TD>RB21/22-27/28-35/36
</TD></TR><TR><TD>pRB4</TD><TD>pSB1C3-BBa_B0034</TD><TD>KpnI</TD><TD>P.lum indC</TD><TD>BamHI</TD><TD>Svert svp</TD><TD>NheI</TD><TD>RB21/22-27/28-29/30
</TD></TR><TR><TD>pRB5</TD><TD>pSB1C3-BBa_B0034</TD><TD>KpnI</TD><TD>P.lum indC</TD><TD>BamHI</TD><TD>Svert svp65</TD><TD>NheI</TD><TD>RB21/22-27/28-25/26
</TD></TR><TR><TD>pRB6</TD><TD>pSB1C3-BBa_B0034</TD><TD>KpnI</TD><TD>P.lum indC</TD><TD>BamHI</TD><TD>Eco entD</TD><TD>NheI</TD><TD>RB21/22-27/28-33/34
</TD></TR><TR><TD>pRB7</TD><TD>pSB1C3-BBa_B0034</TD><TD>KpnI</TD><TD>S.lav bpsA64</TD><TD>BamHI</TD><TD>Bsub sfp</TD><TD>NheI</TD><TD>RB21/22-23/24-35/36
</TD></TR><TR><TD>pRB8</TD><TD>pSB1C3-BBa_B0034</TD><TD>KpnI</TD><TD>S.lav bpsA64</TD><TD>BamHI</TD><TD>Svert svp</TD><TD>NheI</TD><TD>RB21/22-23/24-29/30
</TD></TR><TR><TD>pRB9</TD><TD>pSB1C3-BBa_B0034</TD><TD>KpnI</TD><TD>S.lav bpsA64</TD><TD>BamHI</TD><TD>Svert svp65</TD><TD>NheI</TD><TD>RB21/22-23/24-25/26
</TD></TR><TR><TD>pRB10</TD><TD>pSB1C3-BBa_B0034</TD><TD>KpnI</TD><TD>S.lav bpsA64</TD><TD>BamHI</TD><TD>Eco entD</TD><TD>NheI</TD><TD>RB21/22-23/24-33/34
</TD></TR></TBODY></TABLE><P>
CPEG assembly 20 ul Phusion Flash HF (10 MM; 2 pSB1C3; 1.5 PPTase; 6 ind synthetase)
biometra
</P><TABLE class="wikitable"><TBODY><TR><TH>cycles</TH><TH>temperature °C</TH><TH>time seconds
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">10</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>55</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>70
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>180
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>Transformation 10 pm; 80 ul +/- 20 ul; 5 ul CPEG pcr product
</P><UL><LI> medium LB: FG 2013Jul19
</LI><LI> plates LB+Cm: FG 2013Jul22
</LI><LI> TOP10 from stock -80 °C
</LI></UL><H3><SPAN class="mw-headline" id="Results_and_Discussion.3D.3D.3D.0A_.0A_.0A_.0A_.0A_.0A_.0A_.0A.0A_.0A_.0A_.0A_.0A_Lab_book.0A_.0A.0A_.0A_.0A_.0A_.0A.0A_.0A.0A_.0A.0A_.0A_.0A_.3D.3D.3DKonrad">Results and Discussion===
                                <DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#b14" data-toggle="tab">Lab book</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="b14"><P style="font-size:12pt; text-align:justify;">
                                ===Konrad</P></DIV></DIV></DIV></DIV></DIV></SPAN></H3><H4><SPAN class="mw-headline" id="validation_of_ccdB_constructs">validation of ccdB constructs</SPAN></H4><UL><LI> MP of pKH2(ccdB) constructs from independent colonies: 3rd and 4th were used according to colony PCR result
</LI><LI> (elution in 30 µl): col3=170 ng/µl ; col4=145 ng/µl
</LI></UL><LI> testtrafo in ccdB resistent Survival, TOP10 and DH5alpha cells (from Dominik) of constructs pKH2, pKH2(ccdB)-
</LI><P>col3, pKH2(ccdB)-col4
</P><UL><LI> transformation with 20 ng
</LI><LI> plated 30 µl of cell suspension on 3 partiated plates for each constructs
</LI></UL><H3><SPAN class="mw-headline" id="Construction_of_ccdb-constructs_pRB11-13">Construction of ccdb-constructs pRB11-13</SPAN></H3><P>pRB11, pRB12 (bpsA+svpF) and pRB13 (indC+sfp) carry the ccdb gene with RBS instead of their native T-Domain so that 
</P><P>they don't produce the blue pigment, survive in OneShot ccdb survival cells and die when transformed to TOP10 or 
</P><P>DH5alpha. If the new T-Domain is successfully inserted, transformed cells survive in TOP10 and hopefully produce 
</P><P>the blue pigment.
</P><H4><SPAN class="mw-headline" id="amplification_of_CPEC_fragments">amplification of CPEC fragments</SPAN></H4><P>NI primer in this case are for pRB11 and pRB12; KH primer for pRB13
templates are miniprepped plasmids
</P><UL><LI> ccdb (NI13/14; KH1/2; 350 bp)
</LI><LI> construct pKH1/2 and pRB3 without T-Domain (NI15/16; KH3/4; 7000 bp)
</LI><LI> indC T-Domain as a control (NI17/18; KH5/6; 200 bp)
</LI><LI> bpsA T-Domain as a control (NI19/20; KH7/8; 200 bp)
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>component</TH><TH>concentration</TH><TH>amount
</TH></TR><TR><TD>Phusion Flash HF MM</TD><TD>2x</TD><TD>25 ul
</TD></TR><TR><TD>Primer each</TD><TD>10 uM</TD><TD>5 ul
</TD></TR><TR><TD>water</TD><TD>14.7 ul
</TD></TR><TR><TD>template</TD><TD>&gt;20 ng/ul</TD><TD>0.3 ul
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH colspan="3">NI13/14 in T100
</TH></TR><TR><TH>cycles</TH><TH>temperature (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">8 </TD><TD>98</TD><TD>1
</TD></TR><TR><TD>64 ↓ 0.5</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>10
</TD></TR><TR><TD rowspan="3">22</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>10
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>60
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR><TR><TH colspan="3">KH1/2 in biometra
</TH></TR><TR><TH>cycles</TH><TH>temperature (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">8 </TD><TD>98</TD><TD>1
</TD></TR><TR><TD>56 ↓ 0.5</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>10
</TD></TR><TR><TD rowspan="3">22</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>10
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>60
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR><TR><TH colspan="3">KH3/4 in T100
</TH></TR><TR><TH>cycles</TH><TH>temperature (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">10 </TD><TD>98</TD><TD>1
</TD></TR><TR><TD>62 ↓ 0.5</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>120
</TD></TR><TR><TD rowspan="3">20</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>57</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>120
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR><TR><TH colspan="3">NI15/16 in T100
</TH></TR><TR><TH>cycles</TH><TH>temperature (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">30</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>120
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR><TR><TH colspan="3">KH5/6 in C1000
</TH></TR><TR><TH>cycles</TH><TH>temperature (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">10 </TD><TD>98</TD><TD>1
</TD></TR><TR><TD>63 ↓ 0.5</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>10
</TD></TR><TR><TD rowspan="3">30</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>59</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>10
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>30
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR><TR><TH colspan="3">NI17/18 in C1000
</TH></TR><TR><TH>cycles</TH><TH>temperature (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">10 </TD><TD>98</TD><TD>1
</TD></TR><TR><TD>63 ↓ 0.5</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>10
</TD></TR><TR><TD rowspan="3">30</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>10
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>30
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR><TR><TH colspan="3">KH7/8 and NI19/20 in BioRad old one
</TH></TR><TR><TH>cycles</TH><TH>temperature (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">30</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>10
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>30
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Agarose_Gel_and_Gel_Extraction">Agarose Gel and Gel Extraction</SPAN></H4><P>For NI15/16 wrong template (pMM64) was used in the first PCR. The second run (pKH1/2) yielded the right product
fragments have been gel extracted using QIAquick Gel extraction kit. NI15/16 and KH3/4 have been extracted using 
</P><P>QiaEXII gel extraction kit.
</P><H4><SPAN class="mw-headline" id="CPEC_assembly_and_Transformation">CPEC assembly and Transformation</SPAN></H4><P>CPEC assembly of CCDB fragment (KH1/2 and NI13/14) and indigoidine synthesis construct (pRB3-delta-T, pKH1-delta-T 
</P><P>and pKH2-delta-T to yield pRB11, pRB12 and pRB13. As a positive control, native T-Domains (KH5/6 and NI19/20) are 
</P><P>reinserted. Transformation in ccdb resistant strain (OneShot CCDB survival cells) using standard protocol and 5 ul 
</P><P>of CPEC reaction product.
</P><TABLE class="wikitable"><TBODY><TR><TH colspan="3">CPEC assembly Phusion Flash HF in biometra cycler
</TH></TR><TR><TH>cycles</TH><TH>temp (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">10</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>55</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>120
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>On plates with colonies are always blue colonies as well as white ones -&gt; no information on assembly success.
</P><H4><SPAN class="mw-headline" id="CCDB_screening_and_test">CCDB screening and test</SPAN></H4><P>Five white colonies from pRB12 were picked as well as a blue colony as a control. Conlony PCR using intron iTaq 
</P><P>polymerase was performed and 6x 0.5 mL LB+Cm were inoculated with the six colonies.
</P><TABLE class="wikitable"><TBODY><TR><TH colspan="3">ccdb screening pRB12 colonies with NI13/14 in
</TH></TR><TR><TH>cycles</TH><TH>temperature (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">35</TD><TD>98</TD><TD>60
</TD></TR><TR><TD>60</TD><TD>30
</TD></TR><TR><TD>72</TD><TD>80
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>Screening is positive -&gt; miniprep of colonies #3 and #4 from 2 ml liquid culture.
Purified plasmid pRB12 was transformed to TOP10, DH5alpha and OneShot on LB+Cm to check CCDB functionality.
OneShot should grow whereas plates of DH5alpha and TOP10 were expected to stay empty since ccdb causes cell death.
</P><P>All strains grow on LB+Cm which shows that ccdb doesn't work as expected. ccdb with RBS was amplified from pDONR 
</P><P>plasmid. 
</P><H3><SPAN class="mw-headline" id="Results_and_Discussion_4">Results and Discussion</SPAN></H3><P>Assembly and/or Transformation were inefficient, blue colonies on each plate suggest that a lot of template was in 
</P><P>the Transformation mix. The positive control doesn't provide information on whether the assembly was successful. 
</P><P>CCDB screenig of pRB12 was positive but transformation in DH5alpha, TOP10 and OneShot ccdb survival cells showed 
</P><P>that ccdb is unfunctional. In the following, all the insert from pDONR will be used including a promoter and the 
</P><P>ccda gene. We will work with indC on pSB1C3 with PPTases on separate plasmids to be able to easily test different 
</P><P>combinations of T-Domain and PPTase.
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#b15" data-toggle="tab">Lab book</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="b15"><P style="font-size:12pt; text-align:justify;">

                                ===CPEC Conditions Assay (Ralf)===
We performed CPEC assembly of pRB13 with eight different CPEC conditions to see whether we get difeerent results.
</P><P><B>Table 11.1 CPEC mix for assembly of pRB13</B></P><TABLE class="wikitable"><TBODY><TR><TH>Annealing Temp [°C]</TH><TH colspan="3">Cycles
</TH></TR><TR><TH>5</TH><TH>10</TH><TH>15
</TH></TR><TR><TD>53</TD><TD><B>2: </B>10 ul; 4 ul backbone; 1 ul CCDB</TD></TR><TR><TD>55</TD><TD><B>3: </B>10 ul; 4 ul backbone; 1 ul CCDB <B>7: </B>20 ul; 8.5 ul backbone; 1.5 ul CCDB</TD><TD><B>4: </B>10 ul; 4 ul backbone; 1 ul CCDB<B>6: </B>20 ul; 8.5 ul backbone; 1.5 ul CCDB<B>8: </B>10 ul; 4.5 ul backbone; 0.5 ul CCDB </TD><TD><B>5: </B>10 ul; 4 ul backbone; 1 ul CCDB
</TD></TR><TR><TD>57</TD><TD><B>1: </B>10 ul; 4 ul backbone; 1 ul CCDB</TD></TR></TBODY></TABLE><P><B>Table 11.2 CPEC Assembly of pRB13</B></P><TABLE class="wikitable"><TBODY><TR><TH colspan="3">BioRad T100
</TH></TR><TR><TH>cycles</TH><TH>temp [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">5/10/15</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>53/55/57</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>120
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>100 ul of competent TOP10 were transformed with 5 ul of reaction product and incubated for 30 hours at 37 °C. 
</P><P>
Apparently there is no big difference using various CPEC conditions.
We will continue with the standard protocol.
</P><H3><SPAN class="mw-headline" id="PPTase_test_.28Ralf.29">PPTase test (Ralf)</SPAN></H3><P>pRB3-10 are plasmids containing indC (pRB-3-6) or bpsA(pMM64) (pRB7-10) and a PPTase (sfp, svp, svp(pMM65), entD). 
</P><P>This week the blue colonies are screened and sequenced for validation.
</P><H4><SPAN class="mw-headline" id="colony_PCR_screening_.231">colony PCR screening #1</SPAN></H4><P>Two blue colonies are picked from each plate and screened with the primers of the PPTase they should have and the 
</P><P>ones which they could have if something went wrong. PPTase PCR products werde used as a positive control and TOP10 
</P><P>without any plasmid as a negative control. So numbers 5 to 18 are subdivided into alpha and beta.
</P><TABLE class="wikitable"><TBODY><TR><TH colspan="4">PPTase to be screened
</TH></TR><TR><TH rowspan="14">template</TH><TH>sfp (RB35/36)</TH><TH>svp (RB29/30)</TH><TH>svp(pMM65) (RB25/26)</TH><TH>entD (RB33/34)
</TH></TR><TR><TH> 1</TH></TR><TR><TH> 2</TH></TR><TR><TH> 3</TH></TR><TR><TH> 4
</TH></TR><TR><TH> 5</TH></TR><TR><TH> 6</TH></TR><TR><TH> 7</TH></TR><TR><TH>13 </TH><TH>14 </TH><TH> 15</TH><TH> 8
</TH></TR><TR><TH> 9</TH></TR><TR><TH>10 </TH></TR><TR><TH>11 </TH></TR><TR><TH> 16</TH><TH>17 </TH><TH>18 </TH><TH> 12
</TH></TR><TR><TH> 19</TH><TH>20 </TH><TH>21 </TH><TH> 22
</TH></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH colspan="3">BioRad T100
</TH></TR><TR><TH>cycles</TH><TH>temp [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>120
</TD></TR><TR><TD rowspan="3">30</TD><TD>95</TD><TD>60
</TD></TR><TR><TD>65</TD><TD>30
</TD></TR><TR><TD>72</TD><TD>60
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH colspan="13">Loading Scheme
</TH></TR><TR><TD>5a</TD><TD>5b</TD><TD>9a</TD><TD>9b</TD><TD>1</TD><TD>19</TD><TD><B>2-log</B></TD><TD>6a</TD><TD>6b</TD><TD>10a</TD><TD>10b</TD><TD>2</TD><TD>20
</TD></TR><TR><TD>7a</TD><TD>7b</TD><TD>11a</TD><TD>11b</TD><TD>3</TD><TD>21</TD><TD><B>2-log</B></TD><TD>8a</TD><TD>8b</TD><TD>12a</TD><TD>12b</TD><TD>4</TD><TD>22
</TD></TR></TBODY></TABLE><P>The gel is not conclusive. Therefore I will repeat the screening but screen every colony for all PPTases.
</P><H4><SPAN class="mw-headline" id="MiniPrep_of_pRB4-10">MiniPrep of pRB4-10</SPAN></H4><P>5 ml liquid cultures of TOP10+pRB4-10 have been miniprepped using QIAquick miniprep kit. Only blue cultures have 
</P><P>been prepped (except pRB9 because there was no blue culture). pRB4-6 and pRB10 were prepped in duplicates (liquid 
</P><P>cultures of two different clones) whereas there are only single preps of pRB7-9. This is because pRB3-6 contain 
</P><P>indc ahich we will use for further studies. pRB6 and 10 contain entD, which is the E. coli endogenous PPTase and 
</P><P>was previously declared to be &quot;inefficient&quot; in activating bpsA [Takahashi 2007].
</P><P>Concentrations of the prepped plasmids have been measured using ??ThermoScientific NanoDrop
</P><TABLE class="wikitable"><TBODY><TR><TH>plasmid</TH><TH>version</TH><TH>concentration [ng/ ul]
</TH></TR><TR><TD rowspan="2">pRB4</TD><TD>alpha</TD><TD>289.7
</TD></TR><TR><TD>beta</TD><TD>278.7
</TD></TR><TR><TD rowspan="2">pRB5</TD><TD>alpha</TD><TD>228.3
</TD></TR><TR><TD>beta</TD><TD>348.9
</TD></TR><TR><TD rowspan="2">pRB6</TD><TD>alpha</TD><TD>428.0
</TD></TR><TR><TD>beta</TD><TD>432.2
</TD></TR><TR><TD>pRB7</TD><TD>alpha</TD><TD>187.3
</TD></TR><TR><TD>pRB8</TD><TD>alpha</TD><TD>768.2
</TD></TR><TR><TD>pRB9</TD><TD>alpha</TD><TD>142.5
</TD></TR><TR><TD rowspan="2">pRB10</TD><TD>alpha</TD><TD>112.0
</TD></TR><TR><TD>beta</TD><TD>212.0
</TD></TR><TR><TD colspan="2">water after measurement</TD><TD>0.7
</TD></TR></TBODY></TABLE><P>Minipreps were reduced to a concentration of about 50 ng/ ul for sequencing and 1 ng/ ul for further studies.
</P><H4><SPAN class="mw-headline" id="MiniPrep_PCR_screening_.232">MiniPrep PCR screening #2</SPAN></H4><TABLE class="wikitable"><TBODY><TR><TH colspan="4">PPTase to be screened
</TH></TR><TR><TH>template</TH><TH>sfp (RB35/36)</TH><TH>svp (RB29/30)</TH><TH>svp(pMM65) (RB25/26)</TH><TH>entD (RB33/34)
</TH></TR><TR><TD> TOP10+pRB4</TD><TD>5 </TD><TD> 6</TD><TD>7 </TD><TD> 8
</TD></TR><TR><TD> TOP10+pRB5</TD><TD> 9</TD><TD> 10</TD><TD>11</TD><TD> 12
</TD></TR><TR><TD> TOP10+pRB6</TD><TD>13 </TD><TD>14 </TD><TD> 15</TD><TD> 16
</TD></TR><TR><TD> TOP10+pRB7</TD><TD> 17</TD><TD> 18</TD><TD> 19</TD><TD> 20
</TD></TR><TR><TD> TOP10+pRB8</TD><TD>21 </TD><TD>22 </TD><TD> 23</TD><TD> 24
</TD></TR><TR><TD> TOP10+pRB9</TD><TD> 25</TD><TD> 26</TD><TD>27 </TD><TD> 28
</TD></TR><TR><TD> TOP10+pRB10</TD><TD> 29</TD><TD>30 </TD><TD>31 </TD><TD> 32
</TD></TR><TR><TD> sfp</TD><TD> 33</TD></TR><TR><TD> svp</TD><TD> 34</TD></TR><TR><TD>svp(pMM65)</TD><TD> 35</TD></TR><TR><TD>entD</TD><TD> 36
</TD></TR><TR><TD> TOP10</TD><TD> 37</TD><TD>38 </TD><TD>39 </TD><TD> 40
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH colspan="3">BioRad T100
</TH></TR><TR><TH>cycles</TH><TH>temp [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>95</TD><TD>120
</TD></TR><TR><TD rowspan="3">30</TD><TD>95</TD><TD>60
</TD></TR><TR><TD>60</TD><TD>30
</TD></TR><TR><TD>72</TD><TD>60
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH colspan="25">Loading Scheme
</TH></TR><TR><TD>5a</TD><TD>5b</TD><TD>9a </TD><TD> 9b</TD><TD>13a</TD><TD>13b</TD><TD>17</TD><TD>21</TD><TD>25</TD><TD>29a</TD><TD>29b</TD><TD>sfp</TD><TD><B>2-log</B></TD><TD>6a</TD><TD>6b</TD><TD>10a</TD><TD>10b</TD><TD>14a</TD><TD>14b</TD><TD>18</TD><TD>22</TD><TD>26</TD><TD>30a</TD><TD>30b</TD><TD>svp
</TD></TR><TR><TD>7a</TD><TD>7b</TD><TD>11a</TD><TD>11b</TD><TD>15a</TD><TD>15b</TD><TD>19</TD><TD>23</TD><TD>27</TD><TD>31a</TD><TD>31b</TD><TD>svp(pMM65)</TD><TD><B>2-log</B></TD><TD>8a</TD><TD>8b</TD><TD>12a</TD><TD>12b</TD><TD>16a</TD><TD>16b</TD><TD>20</TD><TD>24</TD><TD>28</TD><TD>32a</TD><TD>32b</TD><TD>entD
</TD></TR></TBODY></TABLE><P>pRB9 is negative. All screens are positive in the svp(pMM65) and entD section. In entD this could be due to genomic 
</P><P>DNA in the miniprep. svp(pMM65) is also positive in all plasmids, which is strange. Maybe the PCR conditions have 
</P><P>been too tolerant. Nevertheless the bands of the plasmids which should carry the respective PPTase appear to be 
</P><P>stronger. We will sequence the plasmids anyways.
</P><H4><SPAN class="mw-headline" id="Sequencing_of_pRB3-10">Sequencing of pRB3-10</SPAN></H4><P>plasmid minipreps have been sequenced by GATC [reference] using the sequencing primers VF2 and VR. The sequencing 
</P><P>results show that all the inserts are correct except for pRB9, which isn't a pity because we have previously shown 
</P><P>with pKH1-2 that the combination of bpsA(pMM64) and svp(pMM65) successful produces indigoidine.
</P><TABLE class="wikitable"><TBODY><TR><TH>plasmid</TH><TH>Primer</TH><TH>ind synthetase/ PPTase</TH><TH>sequencing result/ reference
</TH></TR><TR><TD rowspan="4">pRB3</TD><TD rowspan="2">VF2</TD><TD rowspan="2">indC</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="2">VR</TD><TD rowspan="2">sfp-rev</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="4">pRB4</TD><TD rowspan="2">VF2</TD><TD rowspan="2">indC</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="2">VR</TD><TD rowspan="2">svp(pMM65)-rev</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="4">pRB5</TD><TD rowspan="2">VF2</TD><TD rowspan="2">indC</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="2">VR</TD><TD rowspan="2">svp-rev</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="4">pRB6</TD><TD rowspan="2">VF2</TD><TD rowspan="2">indC</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="2">VR</TD><TD rowspan="2">entD-rev</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="4">pRB7</TD><TD rowspan="2">VF2</TD><TD rowspan="2">bpsA</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="2">VR</TD><TD rowspan="2">sfp-rev</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="4">pRB8</TD><TD rowspan="2">VF2</TD><TD rowspan="2">bpsA</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="2">VR</TD><TD rowspan="2">svp(pMM65)-rev</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="4">pRB9</TD><TD rowspan="2">VF2</TD><TD rowspan="2">bpsA</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="2">VR</TD><TD rowspan="2">svp-rev</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="4">pRB10</TD><TD rowspan="2">VF2</TD><TD rowspan="2">bpsA</TD></TR><TR><TD>seq2
</TD></TR><TR><TD rowspan="2">VR</TD><TD rowspan="2">entD-rev</TD></TR><TR><TD>seq2
</TD></TR></TBODY></TABLE><P>
In the following pRB4-beta, pRB5-alpha, pRB5-beta, pRB6-alpha and pRB10-alpha have been selected so that there are 
</P><P>single versions of pRB3-10 for furhter experiments.
</P><H4><SPAN class="mw-headline" id="Glycerols_stocks_pRB3-10">Glycerols stocks pRB3-10</SPAN></H4><P>3 stocks each were made using 2 mL liquid cultures (48 h at 37 °C 200 rpm) of TOP10+pRB3-10, respectively.
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#b16" data-toggle="tab">Lab book</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="b16"><H3><SPAN class="mw-headline" id="Preparation_for_T-Domain_exchanges">Preparation for T-Domain exchanges</SPAN></H3><P>We assemble pRB14, which is a version of pRB3, in which the T-Domain is exchanged by the whole insert of the pDONR vector. Since pRB13 grew on DH5alpha and TOP10 we hope that pRB14 won't.
This vector will be used to exchange T-Domains and will be modified to get a version without the PPTase sfp and the cutting sites EcoRI and SpeI in indC.
To check whether a successfully exchanged T-Domain can be activated by a defined PPTase, we build pSB2K3-derived plasmids with a single PPTase (sfp, svp, entD or delC) under control of the lac-Promotor and the RBS BBa_B0029.
</P><H4><SPAN class="mw-headline" id="Fragment_Amplification_3">Fragment Amplification</SPAN></H4><H5><SPAN class="mw-headline" id="PCR_of_Fragments_for_PPTase_plasmids">PCR of Fragments for PPTase plasmids</SPAN></H5><P><B>Table 12.x PCR for pRB15-18: </B>25 ul Phusion Flash HF MM 2x; 5 ul Primer 10 uM each; template according to table; water ad 50 ul.
</P><TABLE class="wikitable"><TBODY><TR><TH colspan="3">pSB2K3 RB21/63 in BioRAD old old
</TH></TR><TR><TD colspan="3">template: 3 ul pSB2K3 250 ng/ ul (distribution)
</TD></TR><TR><TH>cycles</TH><TH>temperature (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">10 </TD><TD>98</TD><TD>1
</TD></TR><TR><TD>62 ? 0.5</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>40
</TD></TR><TR><TD rowspan="3">25</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>57</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>40
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>pSB2K3 contains the 3'-part of the reversed Primer in an internal lac-Operator-structure, so the PCR product is just half of the backbone (2500 bp instead of 4800 bp).
We will use the same primers but pSB3K3 of the 2013 spring distribution plate 5 well 5E; which is around 3000 bp.
</P><P><B>Table 12.x PCR for pRB15-18: </B>25 ul Phusion Flash HF MM 2x; 5 ul Primer 10 uM each; template according to table; water ad 50 ul.
</P><TABLE class="wikitable"><TBODY><TR><TH colspan="3">pSB3K3 RB21/63 in BioRAD old old
</TH></TR><TR><TD colspan="3">template: 2 ul pSB3K3 250 ng/ ul (distribution)
</TD></TR><TR><TH>cycles</TH><TH>temperature (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">3 </TD><TD>98</TD><TD>1
</TD></TR><TR><TD>58</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>50
</TD></TR><TR><TD rowspan="3">32</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>50
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>180
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>We get unspecific product -&gt; repetition with more stringent conditions
</P><TABLE class="wikitable"><TBODY><TR><TH colspan="3">pSB3K3 RB21/63 in BioRAD T100
</TH></TR><TR><TD colspan="3">template: 0.5/1.5 ul pSB3K3 250 ng/ ul (distribution)
</TD></TR><TR><TH>cycles</TH><TH>temperature (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">10 </TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65 -0.5</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>50
</TD></TR><TR><TD rowspan="3">25</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>69</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>50
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>180
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="PCR_of_Fragments_for_T-Domain_exchange">PCR of Fragments for T-Domain exchange</SPAN></H5><P><B>Table 12.x PCR for pRB13: </B>25 ul Phusion Flash HF MM 2x; 5 ul Primer 10 uM each; template according to table; water ad 50 ul.
</P><TABLE class="wikitable"><TBODY><TR><TH colspan="3">pRB3?T KH3/4 in T100
</TH></TR><TR><TD colspan="3">template: 1 ul pRB3 1 ng/ ul
</TD></TR><TR><TH>cycles</TH><TH>temperature (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">10 </TD><TD>98</TD><TD>1
</TD></TR><TR><TD>62 ? 0.5</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>120
</TD></TR><TR><TD rowspan="3">25</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>57</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>120
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR><TR><TH colspan="3">KH9/10 in T100
</TH></TR><TR><TD colspan="3">template: 2 ul pDONR 1 ng/ ul
</TD></TR><TR><TH>cycles</TH><TH>temperature (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">8 </TD><TD>98</TD><TD>1
</TD></TR><TR><TD>70 ? 0.5</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>15
</TD></TR><TR><TD rowspan="3">28</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>15
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>90
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH colspan="3">T-Domains in BioRAD old
</TH></TR><TR><TD colspan="3">tycC6-T RB57/58 1 ul B. para overnight culture
</TD></TR><TR><TD colspan="3">tycA1-T RB55/56 1 ul B. para overnight culture
</TD></TR><TR><TD colspan="3">entF-T RB53/54 colony MG1655
</TD></TR><TR><TD colspan="3">delH4-T RB59/60 colony D. aci SPH-1
</TD></TR><TR><TD colspan="3">delH5-T RB61/62 colony D. aci SPH-1
</TD></TR><TR><TH>cycles</TH><TH>temperature (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>120
</TD></TR><TR><TD rowspan="3">40</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>5
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>30
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="Gel_Extraction">Gel Extraction</SPAN></H5><P>Gel extraction was performed using QIAquick gel extraction kit.
DNA yield was measured using NanoDrop Spectrophotometer ND-1000
</P><P><B>Table 12.x DNA concentrations for assembly of pRB-T</B></P><TABLE class="wikitable"><TBODY><TR><TH>Fragment</TH><TH>Concentration [ng/ ul]</TH><TH>~ Fragment size [bp]</TH><TH>Molarity [nM]
</TH></TR><TR><TD>sfp</TD><TD>329.6</TD><TD>700</TD><TD>713.42
</TD></TR><TR><TD>svp</TD><TD>44.9</TD><TD>750</TD><TD>90.7
</TD></TR><TR><TD>entD</TD><TD>289.2</TD><TD>650</TD><TD>674.13
</TD></TR><TR><TD>delC</TD><TD>54.5</TD><TD>700</TD><TD>117.97
</TD></TR><TR><TD>pSB3K3</TD><TD> 26.8 </TD><TD> 3000 </TD><TD> 13.54
</TD></TR><TR><TD>pRB3dT</TD><TD>124.8</TD><TD>6800</TD><TD>27.81
</TD></TR><TR><TD>ccdB</TD><TD>121.2</TD><TD>700</TD><TD>262.33
</TD></TR><TR><TD>indC-T</TD><TD>192.4</TD><TD>200</TD><TD>1457.58
</TD></TR><TR><TD>bpsA-T</TD><TD>198.9</TD><TD>200</TD><TD>1506.18
</TD></TR><TR><TD>entF-T</TD><TD>184.8</TD><TD>200</TD><TD>1400.00
</TD></TR><TR><TD>tycA1-T</TD><TD>184.2</TD><TD>200</TD><TD>1395.45
</TD></TR><TR><TD>tycC6-T</TD><TD>198.6</TD><TD>200</TD><TD>1504.55
</TD></TR><TR><TD>delH4-T</TD><TD>175.0</TD><TD>200</TD><TD>1325.76
</TD></TR><TR><TD>delH5-T</TD><TD>162.4</TD><TD>200</TD><TD>1230.30
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="CPEC_Assembly_and_Transformation_2">CPEC Assembly and Transformation</SPAN></H4><H5><SPAN class="mw-headline" id="pRB14-18">pRB14-18</SPAN></H5><TABLE class="wikitable"><TBODY><TR><TH>Plasmid</TH><TH>Fragment 1</TH><TH>Molarity [nM]</TH><TH>Volume in MM</TH><TH>Fragment 2</TH><TH>Molarity [nM]</TH><TH>Volume in MM
</TH></TR><TR><TD>pRB14</TD><TD>pRB3dT</TD><TD>27.81</TD><TD>3.8</TD><TD>ccdB</TD><TD>262.33</TD><TD>1.2
</TD></TR><TR><TD>pRB15</TD><TD>pSB3K3</TD><TD>13.54</TD><TD>4.5</TD><TD>sfp</TD><TD>713.42</TD><TD>0.5
</TD></TR><TR><TD>pRB16</TD><TD>pSB3K3</TD><TD>13.54</TD><TD>3.5</TD><TD>svp</TD><TD>90.7</TD><TD>1.5
</TD></TR><TR><TD>pRB17</TD><TD>pSB3K3</TD><TD>13.54</TD><TD>4.5</TD><TD>entD</TD><TD>674.13</TD><TD>0.5
</TD></TR><TR><TD>pRB18</TD><TD>pSB3K3</TD><TD>13.54</TD><TD>3.8</TD><TD>delC</TD><TD>117.97</TD><TD>1.2
</TD></TR><TR><TD>pRB21</TD><TD>indC(RB27/46)</TD><TD>25.40</TD><TD>3.5</TD><TD>pSB1C3(RB21/22)</TD><TD>133.08</TD><TD>1.5
</TD></TR></TBODY></TABLE><P><B>Table 12.x </B></P><DL><DD>CPEC Assembly pRB14-18
</DD></DL><TABLE class="wikitable"><TBODY><TR><TD colspan="3">BioRAD T100
</TD></TR><TR><TH>Cycles</TH><TH>Temperature [°C]</TH><TH>Time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">5</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>53</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>120
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P><B>Transforation according to standard protocol for chemical transformation</B></P><UL><LI>TOP10 with pRB15-18 (-&gt; LB+Kan)
</LI><LI>TOP10 with pRB15-18 and pMM64, respectively (-&gt; LB+Kan+Amp)
</LI><LI>TOP10 with pMM64 (-&gt; LB+Amp)
</LI><LI>OneShot with pRB14 (-&gt; LB+Cm)
</LI></UL><UL><LI>OneShot+pRB14: small colonies after 20 hours
</LI><LI>TOP10+pRB15-18: colonies on all plates
</LI><LI>TOP10+pMM64+pRB15-18: (almost) no colonies after 20 hours; hopefully due to indigoidine growth retardation
</LI><LI>TOP10+pMM64 TraFo was efficient
</LI></UL><P>We will perform PCR screening in triplicates of TOP10+pRB15-19 and OneShot+pRB14 to get a first impression on whether the assembly was successful.
</P><H4><SPAN class="mw-headline" id="Plasmid_Validation">Plasmid Validation</SPAN></H4><H5><SPAN class="mw-headline" id="PCR_Screening">PCR Screening</SPAN></H5><P>We use forward primers of the insert and standard reversed primer VR to screen pRB14-18 in triplicates.
We use iTaq DNA-polymerase in 20 ul PCR mix: 
</P><UL><LI>10 ul iTaq 2x Master Mix
</LI><LI>2 ul Primer 10 uM each
</LI><LI>6 ul water
</LI><LI>colony pick from plate
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>Plasmid</TH><TH>Primer fw</TH><TH>Primer rv</TH><TH>Expected Fragment Size [bp]
</TH></TR><TR><TD>pRB14</TD><TD>VF2</TD><TD>KH10</TD><TD>1000
</TD></TR><TR><TD>pRB15</TD><TD>VF2</TD><TD>RB36</TD><TD>1050
</TD></TR><TR><TD>pRB16</TD><TD>VF2</TD><TD>RB30</TD><TD>1100
</TD></TR><TR><TD>pRB17</TD><TD>VF2</TD><TD>RB34</TD><TD>1000
</TD></TR><TR><TD>pRB18</TD><TD>VF2</TD><TD>RB67</TD><TD>1050
</TD></TR></TBODY></TABLE><P><B>Table 12.x </B></P><DL><DD>PCR screening pRB14-18
</DD></DL><TABLE class="wikitable"><TBODY><TR><TD colspan="3">BioRAD T100
</TD></TR><TR><TH>Cycles</TH><TH>Temperature [°C]</TH><TH>Time [s]
</TH></TR><TR><TD>1</TD><TD>95</TD><TD>120
</TD></TR><TR><TD rowspan="3">35</TD><TD>95</TD><TD>30
</TD></TR><TR><TD>53</TD><TD>30
</TD></TR><TR><TD>72</TD><TD>60
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>Annealing temperature according to NEB Tm calculator for Taq DNA polymerase and VF2
</P><TABLE class="wikitable"><CAPTION>Table 12.x concentrations of pRB15-18 (NanoDrop ND-1000)
</CAPTION><TBODY><TR><TH>Plasmid</TH><TH>Concentration [ng/ ul]</TH><TH>~ size [bp]</TH><TH>Molarity [nM]
</TH></TR><TR><TD>pRB15</TD><TD>98.7 </TD><TD>3500 </TD></TR><TR><TD>pRB16</TD><TD>106.6 </TD><TD>3500 </TD></TR><TR><TD>pRB17</TD><TD>188.3 </TD><TD>3500 </TD></TR><TR><TD>pRB18</TD><TD>129.9 </TD><TD>3500 </TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="Test_Transformation">Test Transformation</SPAN></H5><P>OneShot and TOP10-sells have been transformed with pRB14. We expect OneShot cells to grow as usual and no colonies on TOP10. As a control group we transformed OneShot as well as TOP10 with pRB3.
</P><H4><SPAN class="mw-headline" id="pRB19">pRB19</SPAN></H4><P>pRB19 is a pRB14-derived plasmid without the PPTase sfp.
The genotype is pSB1C3-lacPromotor-BBa_B0034-indCdT(ccdB)
</P><P><B>Table 12.x </B></P><DL><DD>PCR Amplification of pRB14 for pRB19
</DD></DL><TABLE class="wikitable"><TBODY><TR><TH colspan="3">BioRAD T100
</TH></TR><TR><TD colspan="3">RB21/46 from pRB14 miniprep 0.6 ng
</TD></TR><TR><TH>Cycles</TH><TH>Temperature [°C]</TH><TH>Time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">8</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>TD 61</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>100
</TD></TR><TR><TD rowspan="3">30</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>100
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>Smear -&gt; Annealing temperatures in the touchdown-part were too high, so RB21 3' end couldn't bind properly. 
Second run with milder conditions.
</P><P><B>Table 12.x </B></P><DL><DD>PCR Amplification of pRB14 for pRB19 #2
</DD></DL><TABLE class="wikitable"><TBODY><TR><TH colspan="3">BioRAD T100
</TH></TR><TR><TD colspan="3">RB21/46 from pRB14 miniprep 1 ng
</TD></TR><TR><TH>Cycles</TH><TH>Temperature [°C]</TH><TH>Time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">12</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>TD 60</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>110
</TD></TR><TR><TD rowspan="3">25</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>110
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>Smear -&gt; we will run another strategy, i.e. amplification of indC from pRB14 with RB27/46 to further assemble it with pSB1C3(RB21/22)
</P><P><B>Table 12.x </B></P><DL><DD>PCR Amplification of indC(RB27/46) for pRB19 #3
</DD></DL><TABLE class="wikitable"><TBODY><TR><TH colspan="3">BioRAD T100
</TH></TR><TR><TD colspan="3">RB27/46 from pRB14 miniprep 1 ng
</TD></TR><TR><TH>Cycles</TH><TH>Temperature [°C]</TH><TH>Time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">10</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>TD 57</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>60
</TD></TR><TR><TD rowspan="3">25</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>60
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>180
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>Gel extraction was performed using QIAquick gel extraction kit.
DNA yield was measured using NanoDrop Spectrophotometer ND-1000
</P><P><B>Table 12.x DNA concentrations for assembly of pRB19</B></P><TABLE class="wikitable"><TBODY><TR><TH>Fragment</TH><TH>Concentration [ng/ ul]</TH><TH>~ Fragment size [bp]</TH><TH>Molarity [nM]
</TH></TR><TR><TD>indC-pRB14(RB27/46)</TD><TD>133.0</TD><TD>3800</TD><TD>53.03
</TD></TR><TR><TD>pSB1C3(RB21/22)</TD><TD>210.8</TD><TD>2400</TD><TD>133.08
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH rowspan="2">Plasmid</TH><TH>Fragment 1</TH><TH>Molarity [nM]</TH><TH>Volume in MM
</TH></TR><TR><TH>!Fragment 2</TH><TH>Molarity [nM]</TH><TH>Volume in MM
</TH></TR><TR><TD rowspan="2">pRB19</TD><TD>indC-pRB14(RB27/46)</TD><TD>pRB14(RB21/46)</TD><TD>53.03</TD><TD>3.3
</TD></TR><TR><TD>pSB1C3(RB21/22)</TD><TD>133.08</TD><TD>1.7
</TD></TR></TBODY></TABLE><P><B>Table 12.x </B></P><DL><DD>CPEC Assembly pRB19
</DD></DL><TABLE class="wikitable"><TBODY><TR><TD colspan="3">BioRAD T100
</TD></TR><TR><TH>Cycles</TH><TH>Temperature [°C]</TH><TH>Time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">5</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>53</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>100
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>Transformation into OneShot Survival cells. Doing colony PCR for screening with primer KH9/VR which would give a fragment size of around 1.8 kbp:
(Note: Throw picked colony 4 away since unintendently pooled 2 colonies in PCR tube). Used iTaq.
</P><P><B>Table 12.x </B></P><DL><DD>PCR screening pRB19 and pRB21
</DD></DL><TABLE class="wikitable"><TBODY><TR><TD colspan="3">Cycler 2
</TD></TR><TR><TD colspan="3">KH9/VR from picked colonies.
</TD></TR><TR><TH>Cycles</TH><TH>Temperature [°C]</TH><TH>Time [s]
</TH></TR><TR><TD>1</TD><TD>95</TD><TD>120
</TD></TR><TR><TD rowspan="3">35</TD><TD>95</TD><TD>30
</TD></TR><TR><TD>53</TD><TD>30
</TD></TR><TR><TD>72</TD><TD>120
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>colony PCR for pRB19 and pRB21
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>Colony PCR for pRB19 gave a lot of unspecific bands, probably because of less stringent parameters. Oculutated 6 ml of TB+Cm with colony 2 for MP preparation.
</P><H3><SPAN class="mw-headline" id="Biobricks">Biobricks</SPAN></H3><P>The PPTases sfp, svp, entD and delC and the indigoidine synthetase indC will be prepared for submission to the registry. indC contains two RFC10-cutting site, that have been removed during the assembly of pRB22.
</P><H4><SPAN class="mw-headline" id="pRB21_series">pRB21 series</SPAN></H4><P>pRB21 is a pSB1C3-plasmid with lacPromotor, BBa_B0034 and indC. After the first assembly, EcoRI and SpeI cutting sites in indC will be removed to yield pRB22.
After cutting site removal, the native T-Domain will be exchanged with ccdB to yield pRB22. pRB23 has the same genotype as pRB20 but with two PCR steps less.
</P><H5><SPAN class="mw-headline" id="Fragment_Amplification_4">Fragment Amplification</SPAN></H5><P><B>Table 12.x </B></P><DL><DD>PCR Amplification of indC for pRB21 #1
</DD></DL><TABLE class="wikitable"><TBODY><TR><TH colspan="3">BioRAD T100
</TH></TR><TR><TD colspan="3">RB27/46 from P. luminescens pellet
</TD></TR><TR><TH>Cycles</TH><TH>Temperature [°C]</TH><TH>Time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">8</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>TD 61</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>120
</TD></TR><TR><TD rowspan="3">25</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>120
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>Amplification was not successful. Next try from PCR product with short primers as a rePCR.
</P><P><B>Table 12.x </B></P><DL><DD>PCR Amplification of indC for pRB21 #2
</DD></DL><TABLE class="wikitable"><TBODY><TR><TH colspan="3">BioRAD T100
</TH></TR><TR><TD colspan="3">RB27/46 from indC short gel extraction
</TD></TR><TR><TH>Cycles</TH><TH>Temperature [°C]</TH><TH>Time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">8</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>TD 56</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>60
</TD></TR><TR><TD rowspan="3">25</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>60
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>180
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV style="height: 377px;">Heidelberg 20130815 PPTBBa indCpRB21 cut2.PNG</DIV></DIV></TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="Gel_Extraction_2">Gel Extraction</SPAN></H5><P>Gel extraction was performed using QIAquick gel extraction kit.
DNA yield was measured using NanoDrop Spectrophotometer ND-1000
</P><P><B>Table 12.x DNA concentrations for assembly of pRB21</B></P><TABLE class="wikitable"><TBODY><TR><TH>Fragment</TH><TH>Concentration [ng/ ul]</TH><TH>~ Fragment size [bp]</TH><TH>Molarity [nM]
</TH></TR><TR><TD>indC(RB27/46)</TD><TD>63.7</TD><TD>3800</TD><TD>25.40
</TD></TR><TR><TD>pSB1C3(RB21/22)</TD><TD>210.8</TD><TD>2400</TD><TD>133.08
</TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="CPEC_Assembly_and_Transformation_3">CPEC Assembly and Transformation</SPAN></H5><TABLE class="wikitable"><TBODY><TR><TH>Plasmid</TH><TH>Fragment 1</TH><TH>Molarity [nM]</TH><TH>Volume in MM</TH><TH>Fragment 2</TH><TH>Molarity [nM]</TH><TH>Volume in MM
</TH></TR><TR><TD>pRB21</TD><TD>indC(RB27/46)</TD><TD>25.40</TD><TD>3.5</TD><TD>pSB1C3(RB21/22)</TD><TD>133.08</TD><TD>1.5
</TD></TR></TBODY></TABLE><P><B>Table 12.x </B></P><DL><DD>CPEC Assembly pRB21
</DD></DL><TABLE class="wikitable"><TBODY><TR><TD colspan="3">BioRAD T100
</TD></TR><TR><TH>Cycles</TH><TH>Temperature [°C]</TH><TH>Time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">5</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>53</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>60
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>180
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="Validation">Validation</SPAN></H5><P>Colony PCR for pRB21 was unsuccessful (see <A href="#Fig:20130817-colPCR">gel picture</A>), try again with less strigent parameters and positive control: KH5/VR on MP pRB3, which should give an amplicon size of around 2 kbp.
</P><P><B>Table 12.x </B></P><DL><DD>PCR screening pRB21
</DD></DL><TABLE class="wikitable"><TBODY><TR><TD colspan="3">T100 (right)
</TD></TR><TR><TD colspan="3">KH5/VR from picked colonies and 5 ng pRB3 MP
</TD></TR><TR><TH>Cycles</TH><TH>Temperature [°C]</TH><TH>Time [s]
</TH></TR><TR><TD>1</TD><TD>95</TD><TD>120
</TD></TR><TR><TD rowspan="3">35</TD><TD>95</TD><TD>30
</TD></TR><TR><TD>50</TD><TD>30
</TD></TR><TR><TD>72</TD><TD>90
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>150
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><DIV>colony PCR forpRB21</DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>Colony PCR for pRB21 worked now as well as the positive control. Oculutated 6 ml of TB+Cm with colony 8 for MP preparation.
</P><H4><SPAN class="mw-headline" id="PPTase_BioBricks">PPTase BioBricks</SPAN></H4><P><I>have to look up used primer/template combinations again</I><A href="/wiki/index.php?title=User:Konrad&amp;action=edit&amp;redlink=1" class="new" title="User:Konrad (page does not exist)">Konrad</A> (<A href="/wiki/index.php?title=User_talk:Konrad&amp;action=edit&amp;redlink=1" class="new" title="User talk:Konrad (page does not exist)">talk</A>)
</P><P><B>Table 12.x </B></P><DL><DD>PCR Amplification of PPTases for BioBrick submission
</DD></DL><TABLE class="wikitable"><TBODY><TR><TH colspan="3">BioRAD T100
</TH></TR><TR><TD colspan="3">sfp: KH17/18 from pRB15
</TD></TR><TR><TD colspan="3">svp: KH19/20 from pRB16
</TD></TR><TR><TD colspan="3">entD: KH13/14 from pRB17
</TD></TR><TR><TD colspan="3">delC: KH15/16 from pRB18
</TD></TR><TR><TH>Cycles</TH><TH>Temperature [°C]</TH><TH>Time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">12</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>TD 66</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>15
</TD></TR><TR><TD rowspan="3">25</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>15
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>60
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV style="height: 377px;">Heidelberg 20130815 PPTBBa indCpRB21 cut2.PNG</DIV></DIV></TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="Gel_Extraction_3">Gel Extraction</SPAN></H5><H4><SPAN class="mw-headline" id="Assembly_and_Transformation">Assembly and Transformation</SPAN></H4><H5><SPAN class="mw-headline" id="Digest">Digest</SPAN></H5><H5><SPAN class="mw-headline" id="Ligation">Ligation</SPAN></H5><P>=====Transformation=====
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#i17" data-toggle="tab">Lab book</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="i17"><H3><SPAN class="mw-headline" id="Preparation_for_T-Domain_exchange">Preparation for T-Domain exchange</SPAN></H3><H4><SPAN class="mw-headline" id="Fragment_Amplification_5">Fragment Amplification</SPAN></H4><TABLE class="wikitable"><CAPTION>Table 12.x PCR for T/TE-Domain exchange with CPEC. Phusion Flash HF
</CAPTION><TBODY><TR><TD colspan="3">template: 0.5 ul from <A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB19</A> miniprep, Primer <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KH3/4</A></TD></TR><TR><TD colspan="3">template: 0.5 ul from <A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB19</A> miniprep, Primer <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KHRB21</A>/<A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KH4</A></TD></TR><TR><TH>cycles</TH><TH>temperature (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">35</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>60</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>100
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>PCR #1 for fragment for T-Domain exchange
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>No band at 6400 bp. Second try with less stringent conditions
</P><TABLE class="wikitable"><CAPTION>Table 12.x PCR #2 <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KH3/4</A> from <A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB19</A>, Phusion Flash HF
</CAPTION><TBODY><TR><TD colspan="3">template: 0.5 ul from <A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB19</A> miniprep, Primer <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KH3/4</A></TD></TR><TR><TD colspan="3">template: 0.5 ul from <A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB19</A> miniprep, Primer <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KHRB21</A>/<A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KH4</A></TD></TR><TR><TH>cycles</TH><TH>temperature (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">10 </TD><TD>98</TD><TD>1
</TD></TR><TR><TD>62 ? 0.5</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>100
</TD></TR><TR><TD rowspan="3">25</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>57</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>100
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>PCR #2 for fragment for T-Domain exchange
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>Still nothing; maybe the MiniPrep didn't work so fine. We will screen pRB19-transformed cells on pRB19.
We want to get a first impression on whether the Domain-exchange works, so we'll use <A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB14</A> instead.
</P><TABLE class="wikitable"><CAPTION>Table 12.x PCR #2 <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KH3/4</A> from <A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB14</A>, Phusion Flash HF
</CAPTION><TBODY><TR><TD colspan="3">template: 0.8 ug from <A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB14</A> miniprep, Primer <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KH3/4</A></TD></TR><TR><TD colspan="3">template: 0.8 ug from <A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB14</A> miniprep, Primer <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KHRB21</A>/<A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KH4</A></TD></TR><TR><TH>cycles</TH><TH>temperature (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">10 </TD><TD>98</TD><TD>1
</TD></TR><TR><TD>62 ? 0.5</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>120
</TD></TR><TR><TD rowspan="3">25</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>57</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>120
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>PCR #3 for fragment for T-Domain exchange
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>PCR worked but with very low yield. We try to amplify from pRB19-transformed cells instead of the miniprep.
</P><TABLE class="wikitable"><CAPTION>Table 12.x PCR #2 <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KH3/4</A> from <A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB19</A>, Phusion Flash HF
</CAPTION><TBODY><TR><TD colspan="3">template: 1 ul from pRB19-transformed cells, Primer <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KH3/4</A></TD></TR><TR><TD colspan="3">template: 1 ul from pRB19-transformed cells, Primer <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KHRB21</A>/<A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KH4</A></TD></TR><TR><TH>cycles</TH><TH>temperature (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">30 </TD><TD>98</TD><TD>1
</TD></TR><TR><TD>57-60-68</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>120
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>
25 ul Phusion HF MM 2x; 5 ul Primer 10 uM each; template according to table; water ad 50 ul
</P><TABLE class="wikitable"><CAPTION>Table 12.x Table 12.x PCR TTE-domains
</CAPTION><TBODY><TR><TH colspan="3">TTE-Domains in BioRAD cycler 2
</TH></TR><TR><TD colspan="3">bpsA-TTE KH7/RB72 1 ng pMM64 miniprep
</TD></TR><TR><TD colspan="3">entF-TTE RB53/73 colony MG1655
</TD></TR><TR><TD colspan="3">tycC6-TTE RB57/74 1 ul B. para overnight culture
</TD></TR><TR><TD colspan="3">delH5-TTE RB61/75 colony D. aci SPH-1
</TD></TR><TR><TH>cycles</TH><TH>temperature (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>120
</TD></TR><TR><TD rowspan="3">40</TD><TD>98</TD><TD>10
</TD></TR><TR><TD>65</TD><TD>20
</TD></TR><TR><TD>72</TD><TD>30
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>120
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>PCR #1 for TTE-Domains
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>bpsA-TTE and tycC6-TTE worked, the other ones will be amplified using less stringent conditions
</P><P>25 ul Phusion Flash HF MM 2x; 5 ul Primer 10 uM each; template according to table; water ad 50 ul
</P><TABLE class="wikitable"><CAPTION>Table 12.x Table 12.x PCR TTE-domains
</CAPTION><TBODY><TR><TH colspan="3">TTE-Domains in BioRAD T100
</TH></TR><TR><TD colspan="3">entF-TTE RB53/73 colony MG1655
</TD></TR><TR><TD colspan="3">delH5-TTE RB61/75 colony D. aci SPH-1
</TD></TR><TR><TH>cycles</TH><TH>temperature (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>120
</TD></TR><TR><TD rowspan="3">10 </TD><TD>98</TD><TD>1
</TD></TR><TR><TD>62 ? 0.5</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>20
</TD></TR><TR><TD rowspan="3">25</TD><TD>98</TD><TD>10
</TD></TR><TR><TD>60</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>20
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>60
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>PCR #2 for TTE-Domains
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>delH5-TTE worked, though with low yield.
entF-TTE will be amplified in a third PCR.
</P><P>25 ul Phusion Flash HF MM 2x; 5 ul Primer 10 uM each; template according to table; water ad 50 ul
</P><TABLE class="wikitable"><CAPTION>Table 12.x Table 12.x PCR TTE-domains
</CAPTION><TBODY><TR><TH colspan="3">TTE-Domains in BioRAD T100
</TH></TR><TR><TD colspan="3">entF-TTE RB53/73 colony MG1655
</TD></TR><TR><TH>cycles</TH><TH>temperature (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>120
</TD></TR><TR><TD rowspan="3">10 </TD><TD>98</TD><TD>1
</TD></TR><TR><TD>60 ? 0.5</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>20
</TD></TR><TR><TD rowspan="3">25</TD><TD>98</TD><TD>10
</TD></TR><TR><TD>55</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>20
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>60
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>PCR #3 for TTE-Domains
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>entF-TTE didn't work. We will postpone this PCR.
</P><H3><SPAN class="mw-headline" id="T-Domain_exchange">T-Domain exchange</SPAN></H3><H4><SPAN class="mw-headline" id="CPEC_Assembly_and_Transformation_4">CPEC Assembly and Transformation</SPAN></H4><TABLE class="wikitable"><TBODY><TR><TH>Plasmid</TH><TH>Fragment 1</TH><TH>Molarity [nM]</TH><TH>Volume in ul</TH><TH>Fragment 2</TH><TH>Molarity [nM]</TH><TH>Volume in MM</TH><TH>CPEC Master Mix total [ul]
</TH></TR><TR><TD>pRB14-T1</TD><TD>pRB14d(ccdB)</TD><TD>3.1</TD><TD>2.7 </TD><TD>indC-T</TD><TD>1457.6</TD><TD>0.3</TD><TD>6
</TD></TR><TR><TD>pRB14-T2</TD><TD>pRB14d(ccdB)</TD><TD>3.1</TD><TD> 2.7 </TD><TD>bpsA-T</TD><TD>1506.1</TD><TD>0.3</TD><TD>6
</TD></TR><TR><TD>pRB14-T4</TD><TD>pRB14d(ccdB)</TD><TD>3.1</TD><TD> 2.7 </TD><TD>tycA1-T</TD><TD>1395.5</TD><TD>0.3</TD><TD>6
</TD></TR><TR><TD>pRB14-T5</TD><TD>pRB14d(ccdB)</TD><TD>3.1</TD><TD> 2.7 </TD><TD>tycC6-T</TD><TD>1504.5</TD><TD>0.3</TD><TD>6
</TD></TR><TR><TD>pRB14-T6</TD><TD>pRB14d(ccdB)</TD><TD>3.1</TD><TD> 2.7 </TD><TD>delH4-T</TD><TD>1325.8</TD><TD>0.3</TD><TD>6
</TD></TR><TR><TD>pRB14-T7</TD><TD>pRB14d(ccdB)</TD><TD>3.1</TD><TD> 2.7 </TD><TD>delH5-T</TD><TD>1230.3</TD><TD>0.3</TD><TD>6
</TD></TR><TR><TD>pRB14-TTE1</TD><TD>pRB14d(ccdB-TE)</TD><TD>33.5</TD><TD> 3</TD><TD>bpsA-TTE</TD><TD>140.8</TD><TD>2</TD><TD>10
</TD></TR><TR><TD>pRB14-TTE3</TD><TD>pRB14d(ccdB-TE)</TD><TD>33.5</TD><TD> 1.5</TD><TD>tycC6-TTE</TD><TD>31.8</TD><TD>3.5</TD><TD>10
</TD></TR><TR><TD>pRB14-TTE4</TD><TD>pRB14d(ccdB-TE)</TD><TD>33.5</TD><TD> 1.5</TD><TD>delH5-TTE</TD><TD>34.7</TD><TD>3.5</TD><TD>10
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><CAPTION>Table 13.x CPEC Assembly pRB14-Tx/TTEx
</CAPTION><TBODY><TR><TD colspan="3">BioRAD T100
</TD></TR><TR><TH>Cycles</TH><TH>Temperature [°C]</TH><TH>Time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">8</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>53</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>120
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>Transformation in TOP10 according to standard protocol. Incubation for 30 hours at 37 °C and 10 hours 
</P><P>on the bench.
</P><H3><SPAN class="mw-headline" id="BioBricks_2">BioBricks</SPAN></H3><H4><SPAN class="mw-headline" id="Fragment_Preparation">Fragment Preparation</SPAN></H4><P>25 ul Phusion Flash HF MM 2x; 5 ul Primer 10 uM each; template according to table; water ad 50 ul
</P><TABLE class="wikitable"><CAPTION>Table 12.x PCR for pRB22
</CAPTION><TBODY><TR><TD colspan="3">miniprep of <A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB22</A> with <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB68/69</A></TD></TR><TR><TH>cycles</TH><TH>temperature (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">35</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>60</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>100
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>PCR #1 for fragment for T-Domain exchange
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>PCR worked well. To make sure that we don't transform template plasmid pRB21, we digest the pRB22d(EcoRI-SpeI) fragment Gel Extraction together with pRB22d(SpeI-EcoRI) with DpnI for four hours at 37 °C.
</P><TABLE class="wikitable"><CAPTION>Table 13.x DpnI-digest for pRB22
</CAPTION><TBODY><TR><TD colspan="3">37 °C; 4 hours
</TD></TR><TR><TH>Component</TH><TH>Concentration</TH><TH>Volume [ul]
</TH></TR><TR><TD>pRB22d(EcoRI-SpeI)</TD><TD>32.3 nM</TD><TD>11.0
</TD></TR><TR><TD>pRB22d(SpeI-EcoRi)</TD><TD>839.4 nM</TD><TD>2.4
</TD></TR><TR><TD>CutSmart buffer</TD><TD>10x</TD><TD>1.6
</TD></TR><TR><TD>NEB DpnI</TD><TD>1
</TD></TR></TBODY></TABLE><P>The Digest was purified using QiaGen Nucleotide Removal Kit. Unfortunately I dropped 2/3 onto the table... The final DNA concentration was 47.5 ng/ ul according to NanoVue.
Standard CPEC assembly was performed in the old broken BioRAD cycler which didn't cool down to 53 °C but only to 58 °C for the annealing step. Hopefully it worked. TOP10 cells were transformed with 5 ul of CPEC master mix.
</P><P>
Transformation was successful. Ten colonies were screened using primers <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">VF2</A> and <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KH6</A> in a 20 ul iTaq Master Mix according to standard protocol (50 °C).
</P><P>
Probes number x to x were purified using Qiaquick PCR purification kit and digested with NEB DpnI, EcoRI-HF and SpeI-HF over night to see whether <A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB22</A> is really free of those restriction sites. The result was put on an AgaroseGel.
</P><P>Lanes 1 to 4 show a slight smear beneath 200 bp. Lane 5 (probe 10) doesn't show that smear and the 
</P><P>specific band runs maybe 300 bp higher. So maybe the first four lanes contained restriction sites and 
</P><P>the last one didn't. We put probes 1 and 10 in 40 ml culture for midiprep. The midiprep will then be 
</P><P>digested as well so that we get certainity about the restriction sites.
In parallel we did a colony PCR from probe 10 with KH3/4 using Phusion Flash HF to already have our 
</P><P>fragment for <A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB23</A> for the T-Domain-exchange.</P><TABLE class="wikitable"><CAPTION>Table 13.x PCR for pRB23
</CAPTION><TBODY><TR><TD colspan="3">miniprep of <A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB23</A> with <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB68/69</A></TD></TR><TR><TH>cycles</TH><TH>temperature (°C)</TH><TH>time (s)
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>120
</TD></TR><TR><TD rowspan="3">10</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>TD 62</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>140
</TD></TR><TR><TD rowspan="3">25</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>57</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>120
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>300
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#i18" data-toggle="tab">Lab book</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="i18"><H3><SPAN class="mw-headline" id="Preparation_for_T-Domain_exchange_2">Preparation for T-Domain exchange</SPAN></H3><H4><SPAN class="mw-headline" id="Assembly_of_pRB19">Assembly of pRB19</SPAN></H4><P>PCR indC(ccdB) RB27/28 and pSB1C3 RB21/22
25 ul Phusion Flash HF MM 2x; 5 ul Primer 10 uM each; template according to table; water ad 50 ul
</P><TABLE class="wikitable"><CAPTION>Table 14.x PCR indC(ccdB) for pRB19
</CAPTION><TBODY><TR><TH colspan="3">RB27/46 in BioRAD T100
</TH></TR><TR><TH>cycles</TH><TH>temperature [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">5</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>55</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>60
</TD></TR><TR><TD rowspan="2">30</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>72</TD><TD>60
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>180
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><CAPTION>Table 14.x PCR indC(ccdB) for pRB19
</CAPTION><TBODY><TR><TH colspan="3">RB27/46 in BioRAD T100
</TH></TR><TR><TH>cycles</TH><TH>temperature [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">10</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>td 55</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>70
</TD></TR><TR><TD rowspan="3">25</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>60</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>70
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>180
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>
Gel Extraction with QIAquick gel extraction kit.
</P><TABLE class="wikitable"><CAPTION>Table 13.x CPEC Assembly Master Mix pRB19 second try
</CAPTION><TBODY><TR><TH>Plasmid</TH><TH>Fragment 1</TH><TH>Molarity [nM]</TH><TH>Volume in ul</TH><TH>Fragment 2</TH><TH>Molarity [nM]</TH><TH>Volume in MM</TH><TH>DpnI
<P>Master Mix total [ul]
</P></TH></TR><TR><TD>pRB19</TD><TD>indC(ccdB) from pRB14</TD><TD>14.6</TD><TD>14</TD><TD>pSB1C3 RB21/22</TD><TD>105.3</TD><TD>3</TD><TD>20
</TD></TR></TBODY></TABLE><P>Digest ran for 1 hour at 37 °C, was put on a agarose gel, gel extracted and eluted in 15 ul water.
</P><P>
Concentration was measured using NanoVue: 30 ng/ ul. 10 ul were put together with 10 ul of Phusion 
</P><P>Flash HF Master Mix for standard CPEC assembly.
OneShot and TOP10 cellc were transformed using different amounts of CPEC product.
</P><P>
Ten colonies were screened using iTaq with KH9/VR and RB35/VR, respectively, to see whether the assembly worked and sfp was kicked out.
</P><P>The screening was positive, so probe 10 was kept for 10 ml liquid culture (TB+Cm).
Miniprep of 4 ml yielded 69.4 ng/ ul in 50 ul.
Miniprep was used for PCR with primers <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KH3/4</A> to get the fragment <A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">

pRB19</A> without ccdB for insertion of T-Domains via CPEC.
First we tried two different polymerases at two different conditions each. The first is Phusion Flash 
</P><P>HF 2x Master Mix (ThermoScientific) and the other is Phusion HF 2x Master Mix (NEB).
</P><TABLE class="wikitable"><CAPTION>Table 14.x Test PCR <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KH3/4</A> from <A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB19</A></CAPTION><TBODY><TR><TH colspan="3">KH3/4 in BioRAD T100 with Phusion HF (NEB)
</TH></TR><TR><TH>Cycles</TH><TH>Temperature [°C]</TH><TH>Time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>30
</TD></TR><TR><TD rowspan="3">30</TD><TD>98</TD><TD>15
</TD></TR><TR><TD>57-60</TD><TD>15
</TD></TR><TR><TD>72</TD><TD>3.30
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>10.00
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR><TR><TH colspan="3">KH3/4 in BioRAD T100 with Phusion Flash HF (ThermoScientific)
</TH></TR><TR><TH>Cycles</TH><TH>Temperature [°C]</TH><TH>Time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">30</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>57-60</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>2.00
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>6.00
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><CAPTION>Table 14.x synT-PCR
</CAPTION><TBODY><TR><TH colspan="3">KH5/6 in BioRAD T100
</TH></TR><TR><TH>cycles</TH><TH>temperature [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>30
</TD></TR><TR><TD rowspan="3">45</TD><TD>98</TD><TD>5
</TD></TR><TR><TD>60</TD><TD>15
</TD></TR><TR><TD>72</TD><TD>15
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>60
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>
Gel extraction and measurement using NanvoVue. The final table of T-domain concentration is as 
</P><P>follows:
</P><TABLE class="wikitable"><CAPTION>Table 13.x Concentrations of T- and TTE-domains to be shuffled
</CAPTION><TBODY><TR><TH>Fragment</TH><TH>Concentration [ng/ ul]</TH><TH>~ Fragment size [bp]</TH><TH>Molarity [nM]
</TH></TR><TR><TD>ccdB</TD><TD>121.2</TD><TD>700</TD><TD>262.33
</TD></TR><TR><TD>indC-T</TD><TD>192.4</TD><TD>200</TD><TD>1457.58
</TD></TR><TR><TD>bpsA-T</TD><TD>198.9</TD><TD>200</TD><TD>1506.18
</TD></TR><TR><TD>entF-T</TD><TD>64.5</TD><TD>200</TD><TD>488.6
</TD></TR><TR><TD>tycA1-T</TD><TD>184.2</TD><TD>200</TD><TD>1395.45
</TD></TR><TR><TD>tycC6-T</TD><TD>198.6</TD><TD>200</TD><TD>1504.55
</TD></TR><TR><TD>delH4-T</TD><TD>175.0</TD><TD>200</TD><TD>1325.76
</TD></TR><TR><TD>delH5-T</TD><TD>162.4</TD><TD>200</TD><TD>1230.30
</TD></TR><TR><TD>plu2642-T</TD><TD>79.0</TD><TD>200</TD><TD>598.5
</TD></TR><TR><TD>plu2670-T</TD><TD>31.5</TD><TD>200</TD><TD>238.6
</TD></TR><TR><TD>synT1</TD><TD>43</TD><TD>200</TD><TD>325.8
</TD></TR><TR><TD>synT2</TD><TD>37.5</TD><TD>200</TD><TD>284.1
</TD></TR><TR><TD>synT3</TD><TD>40.5</TD><TD>200</TD><TD>306.8
</TD></TR><TR><TD>synT4</TD><TD>36</TD><TD>200</TD><TD>272.7
</TD></TR><TR><TD>synT5</TD><TD>37.5</TD><TD>200</TD><TD>284.1
</TD></TR><TR><TD>synT6</TD><TD>37</TD><TD>200</TD><TD>280.3
</TD></TR><TR><TD>synT7</TD><TD>40.5</TD><TD>200</TD><TD>306.8
</TD></TR><TR><TD>bpsA-TTE</TD><TD>92.9</TD><TD>1000</TD><TD>140.8
</TD></TR><TR><TD>entF-TTE</TD><TD>-</TD><TD>1000</TD><TD>-
</TD></TR><TR><TD>tycC6-TTE</TD><TD>21</TD><TD>1000</TD><TD>31.8
</TD></TR><TR><TD>delH5-T</TD><TD>22.9</TD><TD>1000</TD><TD>34.7
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Assembly_of_pRB15-18">Assembly of pRB15-18</SPAN></H4><P><A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB15-18</A> were unable to be sequenced with various primers. We now use pSB3K3 backbone from plate 3 of the 2013 spring distribution and assemble those PPTase-plasmids de novo.
</P><TABLE class="wikitable"><CAPTION>Table 14.x PCR pSB3K3 with RB21/63 #1
</CAPTION><TBODY><TR><TH colspan="3">RB21/63 in BioRAD T100
</TH></TR><TR><TH>Cycles</TH><TH>Temperature [°C]</TH><TH>Time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">14</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>td 66</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>60
</TD></TR><TR><TD rowspan="3">16</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>68</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>60
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>180
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>PCR was run with with a fragment of the Tyrocidine group with slightly suboptimal conditions. It will be repeated using otimal conditions.
</P><TABLE class="wikitable"><CAPTION>Table 14.x PCR pSB3K3 with RB21/63 #2
</CAPTION><TBODY><TR><TH colspan="3">RB21/63 in BioRAD T100
</TH></TR><TR><TH>Cycles</TH><TH>Temperature [°C]</TH><TH>Time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">12</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>td 61</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>50
</TD></TR><TR><TD rowspan="3">20</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>50
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>180
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>Gel extraction yielded 144.2 ng/ ul (Nanodrop)
</P><P>CPEC was performed using pSB3K3 gel extraction and gel extractions made for assembly of pRB3-10.
TOP10 cells were transformed using standard protocol.
</P><TABLE class="wikitable"><CAPTION>Table 14.x CPEC Assembly <A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB15-18</A></CAPTION><TBODY><TR><TH>Plasmid</TH><TH>Fragment 1</TH><TH>Molarity [nM]</TH><TH>Volume [ul]</TH><TH>Fragment 2</TH><TH>Molarity [nM]</TH><TH>Volume [ul]</TH><TH>CPEC Master Mix total [ul]
</TH></TR><TR><TD>pRB15</TD><TD>pSB3K3(RB21/63)</TD><TD>87.4</TD><TD>2 </TD><TD>sfp</TD><TD>713.4</TD><TD>1</TD><TD>6
</TD></TR><TR><TD>pRB16</TD><TD>pSB3K3(RB21/63)</TD><TD>87.4</TD><TD> 2 </TD><TD>svp</TD><TD>90.7</TD><TD>4</TD><TD>12
</TD></TR><TR><TD>pRB17</TD><TD>pSB3K3(RB21/63)</TD><TD>87.4</TD><TD> 2</TD><TD>entD</TD><TD>674.1</TD><TD>1</TD><TD>6
</TD></TR><TR><TD>pRB18</TD><TD>pSB3K3(RB21/63)</TD><TD>87.4</TD><TD> 1 </TD><TD>delC</TD><TD>118.1</TD><TD>2</TD><TD>6
</TD></TR></TBODY></TABLE><P>Cells were plated on LB+Kanamycin and incubated at 37 °C for 24 hours.
</P><P>Five colonies of each plate were screened using iTaq polymerase and two primer combinations for each colony. This is VF2/PPTase_rv primer and PPTase_fw/VR primer.
</P><P><A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB16</A> is obviously wrong. We checked the PCR product we used for the assembly and recognized that it's wrong, so we have to amplify it again from S. verticillus. DelC screenings show another small band, we have too check whether the primers bind elsewhere on the plasmid and run a negative control with those primers.
</P><H4><SPAN class="mw-headline" id="Assembly_of_pKH4">Assembly of pKH4</SPAN></H4><P><A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pKH4</A> is a pRB3 derived version of an indigoidine synthetase expression plasmid w/o sfp as activating PPtases. So the two plasmid strategy can be conducted (at first) with this plasmid instead to rely on <A href="/wiki/index.php?title=Plasmid&amp;action=edit&amp;redlink=1" class="new" title="Plasmid (page does not exist)">pRB21</A>.
Basically the idea is to use restriction sites before and after sfp to discard it and to religate with another sequence to yield a functional backbone.
</P><P><A href="/wiki/index.php?title=Plasmid&amp;action=edit&amp;redlink=1" class="new" title="Plasmid (page does not exist)">pRB3</A> has a BamHI before and a NheI cutting site after sfp. <A href="/wiki/index.php?title=Plasmid&amp;action=edit&amp;redlink=1" class="new" title="Plasmid (page does not exist)">pMM64</A> and <A href="/wiki/index.php?title=Plasmid&amp;action=edit&amp;redlink=1" class="new" title="Plasmid (page does not exist)">pMM65</A> both can be cut with a combination of BglII and XbaI to yield for example a 164 bp long fragment with compatible ends to the linearized <A href="/wiki/index.php?title=Plasmid&amp;action=edit&amp;redlink=1" class="new" title="Plasmid (page does not exist)">pRB3</A>. This fragment unfortunately contains a T7lac hybrid promoter which could interfere with our used primers, but still is worth a shot.
</P><UL><LI> digest mix volume: 30 µl (3 µl NEBuffer 3.1, 6.5 µl pMM65 (~481 ng), 0.3 µl (BglII, XbaI) each, 19.9 µl H2O)
</LI></UL><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>Resulting gel of digestion of pMM65 with BglII and XbaI in order to gain sequence with cohesive ends to linerized pRB3 w/o sfp.
</P></DIV></DIV></TD></TR></TBODY></TABLE><UL><LI> digestion gave expected bands at 2 kbp and 2.4 kbp but band at ~160 bp is missing. Reaction was carried out with not enough DNA.
</LI><LI> prepared ON for pMM64/65 MP as well as pRB3 MP
</LI><LI> the next day ON were extended to 2x 4 ml LB cultures which were then
</LI></UL><P>prepped after several hours of incubation at 37 °C
</P><UL><LI> MP with cultures: DNA conc. measurement with NanoVue gave: 280
</LI></UL><P>ng/µl (pMM64), 225 ng/µl (pMM65), 250 ng/µl (pRB3); analysis gel shows
different concentration for pMM65 (~15 ng/µl ?)
</P><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>Analytical gel for verification of MP pMM64/65 and pRB3: concentrations measured with NanoVue seem to be ok for pMM64 and pRB3, but much less for pMM65. 2log ladder lane contain 420 ng each; 100 V, TAE
</P></DIV></DIV></TD></TR></TBODY></TABLE><UL><LI> prepare digestion (20 µl), 37 °C, 1.5 h:
</LI><LI> pMM64: 2 µl NEBuffer 3.1, 17 µl pMM64, 0.5 µl (BglII, XbaI)
</LI><LI> pRB3: 2 µl NEBuffer CutSmart, 10 µl pRB3, 0.5 µl (BamHI, NheI), 7
</LI></UL><P>µl H2O
</P><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>Digestion of pMM64 (BglII, XbaI) and pRB3 (BamHI, NheI) were desired bands for insert of ~170 bp and backbone-indC of ~6300 bp were cut out. 100 V, TAE
</P></DIV></DIV></TD></TR></TBODY></TABLE><UL><LI> after gelextraction (elution in 20 µl) ligation was conducted with 2
</LI></UL><P>µl 10x T4 ligase buffer, 1 µl T4 ligase, 2 µl pRB3 Dig_GE, 8 µl pMM64
Dig_GE and 7 µl H2O for 40 min at RT
</P><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>Analytical gel of 1 µl of ligation mix, compared to 0.5 µl of pRB3 digestion with (BamHI, NheI). Additionally 1 µl of MP pMM65 was put on gel to verify previous concentration measurement. 1.5 µl of log2 ladder, 100 V, TAE
</P></DIV></DIV></TD></TR></TBODY></TABLE><UL><LI> transformation in TOP10 with 1 µl and 5 µl of ligation mix
                                </LI><LI class="active"><A href="#i19" data-toggle="tab">Lab book</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="i19"><H3><SPAN class="mw-headline" id="pRB15-18">pRB15-18</SPAN></H3><P>The first mini
</P><H3><SPAN class="mw-headline" id="Screening_delC">Screening delC</SPAN></H3><TABLE class="wikitable"><CAPTION>Table 15.x Screening pRB18
</CAPTION><TBODY><TR><TH>Template</TH><TH>Primer</TH><TH>Comment
</TH></TR><TR><TD rowspan="2">TOP10+pSB3K3</TD><TD>RB64/VR</TD><TD rowspan="2">negative control
</TD></TR><TR><TD>VF2/65
</TD></TR><TR><TD rowspan="3">TOP10+pRB18</TD><TD>RB64/VR</TD><TD rowspan="3">screening
</TD></TR><TR><TD>VF2/65
</TD></TR><TR><TD>VF2/VR
</TD></TR><TR><TD>delC PCR product</TD><TD>RB64/RB65</TD><TD>positive control
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Screening_pKH4">Screening pKH4</SPAN></H3><TABLE class="wikitable"><CAPTION>Table 15.x Screening pKH4
</CAPTION><TBODY><TR><TH>Template</TH><TH>Primer
</TH></TR><TR><TD rowspan="2">TOP10+pKH4</TD><TD>KH52/VR
</TD></TR><TR><TD>VF2/RB71
</TD></TR></TBODY></TABLE><P>
A miniprep of probe 4 was sent to sequencing, which was positive.
</P><H3><SPAN class="mw-headline" id="Assembly_of_pKH4-derivates">Assembly of pKH4-derivates</SPAN></H3><P>pKH4 is used for assembly of pKH5 and variants of pKH4 with all the different T-Domains.
</P><TABLE class="wikitable"><CAPTION>Table 15.x Amplification KH3/4 from pKH4
</CAPTION><TBODY><TR><TH colspan="3">KH3/4 in BioRAD T100
</TH></TR><TR><TH>cycles</TH><TH>temperature [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">10</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>td 62</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>2.00
</TD></TR><TR><TD rowspan="3">25</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>57</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>2.00
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>6.00
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>
KH3/4-PCRs were unsuccessful, so we did a gradient PCR 
</P><TABLE class="wikitable"><CAPTION>Table 15.x Amplification KH3/4 from pKH4
</CAPTION><TBODY><TR><TH colspan="3">KH3/4 in biometra
</TH></TR><TR><TH>cycles</TH><TH>temperature [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">10</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>58-59-60-61-62-63</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>2.00
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>6.00
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>
We run the same protocol for 3x 50 ul.
</P><P>
nothing
</P><P>16 x 10 ul with Q5 and Tm calculator suggested parameters (63 °C annealing), pRB19 ligation as template
</P><P>
We tried NEB Phusion HF with 61 °C annealing temperature in a 10-, 20- and 50 ul reaction from pRB19 ligation and in a 20 ul reaction from pKH4 ind T100 (left).
As well we ran with Phusion HF (59 °C annealing) in 20 ul reactions with 0, 1 and 2 ul DMSO (Biorad Cycler 2)
</P><P>For further evaluation of the single plasmid strategy we admired a plasmid with bpsA instead of indC as indigoidine synthetase. For a quick shot we wanted to use pKH4 and pRB7 for a restriction approach with the enzymes KpnI and BamHI.
</P><UL><LI> prepare digestion (20 µl), 37 °C, ~0.5 h:
</LI><LI> pKH4: 2 µl NEBuffer CutSmart, 4 µl pKH4, 13 µl H2O,  0.5 µl (KpnI, BamHI)
</LI><LI> pRB7: 2 µl NEBuffer CutSmart, 2.3 µl pRB7, 14.7 µl H2O, 0.5 µl (KpnI, BamHI)
</LI></UL><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>Digestion of pKH4 (KpnI, BamHI) and pRB7 (KpnI, BamHI). Desired bands for insert of ~3.8 kbp and backbone+sfp of ~3.9 kbp were expected. 100 V, 0.8 % Agarose, TAE
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>The restriction for pKH4 did not work correctly due to the fact that with assembly of pKH4 the BamHI cutting site was removed and the plasmid was only linearized by the KpnI cutting site. For pRB7 two bands at almost equal site can be expected and thereby the digestion approach can not be used in this manner. We will try again with a similar approach like for the assembly of pKH4 as here a nonsense sequence instead of the PPtases will be introduced into pRB7.
</P><UL><LI> preparation of pRB7 MP by retransformation of TOP10 (as no glycerol stock is available).
</LI></UL><H3><SPAN class="mw-headline" id="Amplification_of_svp">Amplification of svp</SPAN></H3><P>The svp fragment we used so far was wrong, so we try to amplify it again from the genome, using various primers:
</P><UL><LI>RB13/14
</LI><LI>RB15/16
</LI><LI>RB29/30	
</LI><LI>RB39/40
</LI></UL><P>Gradient on biometra cycler (72 °C, 65 °C, 58 °C)
</P><P>
svp was always right; only after assembly it turned wrong. We will try to bring it in using restriction cloning and to sequence it; maybe the screening is just inconclusive.
</P><H3><SPAN class="mw-headline" id="Amplification_of_indC-fragments">Amplification of indC-fragments</SPAN></H3><P>indC fragments are amplified from Plum genome and indC short PCR product in a 50 ul Phusioin HF reaction. 
The fragments are:
</P><UL><LI> RB27/69
</LI><LI> RB70/71
</LI><LI> RB68/46
</LI><LI> RB68/KH4
</LI><LI> KH5/RB46
</LI><LI> RB27/46 from pRB14
</LI><LI> RB27/28
</LI></UL><TABLE class="wikitable"><CAPTION>Table 14.x PCR indC(ccdB) for pRB19
</CAPTION><TBODY><TR><TH colspan="3">RB27/69 in BioRAD T100 from indC short
</TH></TR><TR><TH colspan="3">RB27/69 in BioRAD T100 from P. luminescens pellet
</TH></TR><TR><TH colspan="3">RB70/71 in BioRAD T100 from indC short
</TH></TR><TR><TH colspan="3">RB70/71 in BioRAD T100 from P. luminescens pellet
</TH></TR><TR><TH colspan="3">KH5/RB46 in BioRAD T100 from indC short
</TH></TR><TR><TH colspan="3">KH5/RB46 in BioRAD T100 from P. luminescens pellet
</TH></TR><TR><TH>cycles</TH><TH>temperature [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>2.00
</TD></TR><TR><TD rowspan="3">35</TD><TD>98</TD><TD>5
</TD></TR><TR><TD>55</TD><TD>15
</TD></TR><TR><TD>72</TD><TD>30
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>2.00
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR><TR><TH colspan="3">RB68/46 in BioRAD T100 from indC short
</TH></TR><TR><TH colspan="3">RB68/46 in BioRAD T100 from P. luminescens pellet
</TH></TR><TR><TH colspan="3">RB68/KH4 in BioRAD T100 from indC short
</TH></TR><TR><TH colspan="3">RB68/KH4 in BioRAD T100 from P. luminescens pellet
</TH></TR><TR><TH>cycles</TH><TH>temperature [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>2.00
</TD></TR><TR><TD rowspan="3">30</TD><TD>98</TD><TD>15
</TD></TR><TR><TD>55</TD><TD>15
</TD></TR><TR><TD>72</TD><TD>2.00
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>5.00
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>Fragment RB70/71 is wrong because RB71 is wrong. We repeated the amplification with RB78 instead of RB71.
</P><P>Fragment KH3/RB46 is very weak, so we try to improve this reaction as well.
</P><TABLE class="wikitable"><CAPTION>Table 15.x Amplification indC RB21/46 from pRB14 and pSB1C3 RB21/22 from miniprep
</CAPTION><TBODY><TR><TH colspan="3">KH3/RB46 in BioRAD T100
</TH></TR><TR><TH>cycles</TH><TH>temperature [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">12</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>td 60</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>20
</TD></TR><TR><TD rowspan="3">23</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>63</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>20
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>60
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>And reaction RB68/RB46 with the same protocol but 60 s elongation time.
</P><P>We try KH3/RB79 with improved conditions.
</P><P>We tried to get this fragment with KH9/RB79 with Q5 in biometra cycler (60 °C annealing)
</P><P>We tried to get this fragment with KH9/RB79 with Phusion HF in biometra old cycler (61 °C annealing)
</P><P>Gel Extraction yielded xx ng/ ul
</P><P>CPEC assemblies were performed for following plasmids with the respective fragments:
</P><UL><LI> pRB19: RB27/79 from pRB14 and RB21/22 from pSB1C3 midiprep
</LI><LI> pRB22: RB27/69 RB70/78 RB68/79 RB21/22
</LI><LI> pRB23: RB27/69 RB70/78 RB68/KH4 KH9/RB79 (pRB14) RB21/22
</LI><LI> pRB24: RB27/28 RB66/67 RB21/22
</LI><LI> pKH5:  KH3/4 (pKH4) KH9/10
</LI></UL><P>Transformed cells looked as follows:
</P><P>
Colony screenings
</P><P>
pRB23 assembly was unsuccessful, pRB22 is correct (plates turn blue), pRB24 seems to be correct (small deep blue colonies), pRB19 didn't work, pKH5 seems correct. We prepped the marked colonies. Sequencing showed that pRB22 is correct and doesn't carry endogenous RFC10 cutting sites any longer. Test digestion with EcoRI and SpeI also enforced this result.
</P><P>We performed a PCR with Phusion HF and <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KH3/4</A> from pRB22, assembled it in a CPEC approach with ccdB KH9/10 and transformed OneShot cells.
</P><P>Colony screenings and test-transformation in TOP10 cells showed that the assembly was successful and that we're able to shuffle the T-domains.
</P><H3><SPAN class="mw-headline" id="Minipreps">Minipreps</SPAN></H3><P>Minipreps of pRB15, pRB17, pRB18, pKH4 and pRB14 yielded:
</P><UL><LI>pRB15
</LI><LI>pRB17
</LI><LI>pRB18
</LI><LI>pRB14: 284.8 ng/ ul in 20 ul
</LI><LI>pKH4: 316.8 ng/ ul in 20 ul
</LI></UL><P>5 ul of each pRB15, 17 and 18 were put on an agarose gel with small pockets.
</P><P>PPTase plasmid minipreps are empty. We will try to assemble them again using a restriction strategy besides prepping them again.
Repetitioin of the miniprep was successful and the plasmids were sent to sequencing.
</P><H3><SPAN class="mw-headline" id="Cotransformation">Cotransformation</SPAN></H3><P>We did cotransformation of pKH4 with pRB15-18, respectively, and single transformation of pSB3K3, pKH4 and pRB15-18.
</P><H3><SPAN class="mw-headline" id="Restriction_cloning_for_pRB15-19">Restriction cloning for pRB15-19</SPAN></H3><TABLE class="wikitable"><CAPTION>Table 15.x Amplification indC RB21/46 from pRB14 and pSB1C3 RB21/22 from miniprep
</CAPTION><TBODY><TR><TH colspan="3">RB21/22 in BioRAD T100
</TH></TR><TR><TH>cycles</TH><TH>temperature [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">12</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>td 60</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>50
</TD></TR><TR><TD rowspan="3">23</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>40
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>3.00
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR><TR><TH colspan="3">RB27/46 in BioRAD T100
</TH></TR><TR><TH>cycles</TH><TH>temperature [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">12</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>td 60</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>70
</TD></TR><TR><TD rowspan="3">18</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>60
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>2.00
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>
svp was reamplified from short product
</P><TABLE class="wikitable"><CAPTION>Table 15.x Amplification indC RB21/46 from pRB14 and pSB1C3 RB21/22 from miniprep
</CAPTION><TBODY><TR><TH colspan="3">RB21/22 in BioRAD T100
</TH></TR><TR><TH>cycles</TH><TH>temperature [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">35</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>20
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>60
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>
indC and pSB1C3 were digested with KpnI and NheI, sfp, svp, entD, delC and pSB3K3 with BamHI and NheI for two hours at 37 °C.
</P><P>
Unfortunately pSB3K3 contains an internal NheI cutting site. We repeated the amplification, purification, restriction digest and purification with BamHI and SpeI, which builds a scar with NheI.
</P><P>
Ligation of pSB1C3 with indC(ccdB) was performed over night in a 10 ul reaction with 4 ul fragments each.
</P><P>Transformation of ligation mixes were performed with 5 µl each in TOP10. Additionally 5 µl of ligation mix of pRB19 was transformed into OneShot cells.
</P><P>
Colony screenings showed following results:
</P><P>We prepped pRB15, 17 and 18 and repeated the assembly of pRB16 from scratch. Therefore we used <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB13/14</A> for amplification of svp from an S. verticilllus ATCC15003 liquid culture. Afterwards we reamplified this PCR gel extraction with <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB29/30</A> and performed a restriction digest with BamHI and NheI to ligate the fragment with pSB3K3 (BamHI; SpeI).
</P><H3><SPAN class="mw-headline" id="Reevaluation_of_old_MP_pRB15-18.2C_pRB21">Reevaluation of old MP pRB15-18, pRB21</SPAN></H3><P>Because we could observe small colonies with darkgray center on cotransformed pRB21 with all PPtases constructs (plates of 2013-08-27), we conducted a colony screen and a retransformation with 1 µl of previous MP of pRB15.2 (Konrad) and pRB16-18 (Ralf) and pRB21 (Ralf). Additionally a cotransformation with this old pRB21 MP with the old pRB15.2 as well with 2.5 µl of newly assembled pRB15 was performed.
</P><P>
The colony screening was performed using the primers <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">VR</A> and <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KH5</A>. If the gray colonies carry two plasmids with indigoidine synthetase and activating PPtase separately an amplicon of size ~1.3 kbp. Otherwise if for some reason the PPtase and indigoidine synthetase are still on one plasmid, we could expect an amplicon of 2.1 kbp.
</P><TABLE class="wikitable"><CAPTION>Table 15.x colony screening cotrafo pRB21 and pRB15-18, 2013-08-27
</CAPTION><TBODY><TR><TH colspan="3">KH5/VR in Biometra, 10 µl
</TH></TR><TR><TH>cycles</TH><TH>temperature [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>95</TD><TD>120
</TD></TR><TR><TD rowspan="3">30</TD><TD>95</TD><TD>30
</TD></TR><TR><TD>60</TD><TD>30
</TD></TR><TR><TD>72</TD><TD>120
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>240
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>Colony PCR of cotransformation pRB21 and pRB15-18. Desired bands would be expected at 1.3 kbp for positive clones. For positive controls amplicons were expected at ~2 kbp for pRB3 and at ~1.5 kbp for pKH4. 100 V, 0.8 % Agarose, TAE
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>The positive controls gave the expected bands. All picked clones also gave an expected band at 1.3 kbp which leads to the conclusion that they carry the indC-only plasmid pRB21. Because every third clone was a picked white colony we could also conclude, that not all transformants carry an intact PPtase plasmid.
</P><H3><SPAN class="mw-headline" id="T-domain_exchange_with_different_domain_borders">T-domain exchange with different domain borders</SPAN></H3><P>We thought of a different way to set the T-Domain borders. Starting from the Pfam-predicted domain borders we made pairwise alignments to find amino acid sequences up- and downstream the predicted borders, that are identical in both the native and the brought in sequence. In the case of the exchange of indC and bpsA those new borders would be around 10 amino acids more on each side. We designed primers for this strategy.
</P><H3><SPAN class="mw-headline" id="Screenings">Screenings</SPAN></H3><TABLE class="wikitable"><CAPTION>Table 15.x Screening of new constructs
</CAPTION><TBODY><TR><TH>Plasmid</TH><TH>Primer 1</TH><TH>Primer 2</TH><TH>Expected product [bp]</TH><TH>wrong product [bp]</TH><TH>additional test
</TH></TR><TR><TD>pKH6</TD><TD>VR</TD><TD>RB17</TD><TD>1070</TD></TR><TR><TD>pKH7</TD><TD>VR</TD><TD>RB13</TD><TD>1120</TD></TR><TR><TD>pKH8</TD><TD>VR</TD><TD>RB41</TD><TD>1000</TD></TR><TR><TD>pKH9</TD><TD>VR</TD><TD>RB64</TD><TD>1070</TD></TR><TR><TD>pRB15_lig</TD><TD>VR</TD><TD>RB17</TD><TD>1070</TD></TR><TR><TD>pRB16_lig</TD><TD>VR</TD><TD>RB13</TD><TD>1120</TD></TR><TR><TD>pRB17_lig</TD><TD>VR</TD><TD>RB41</TD><TD>1000</TD></TR><TR><TD>pRB18_lig</TD><TD>VR</TD><TD>RB64</TD><TD>1070</TD></TR><TR><TD rowspan="2">pRB19_lig</TD><TD>VR</TD><TD>KH9</TD><TD>1900</TD><TD>2600</TD></TR><TR><TD>VR</TD><TD>RB17</TD><TD>-</TD><TD>1070</TD></TR><TR><TD>pRB22_frag</TD><TD>digest
</TD></TR><TR><TD rowspan="2">pRB23_frag</TD><TD>VR</TD><TD>KH9</TD><TD>1900</TD><TD>digest
</TD></TR><TR><TD>RB27</TD><TD>KH4</TD><TD>2900</TD></TR><TR><TD>pKH5</TD><TD>VR</TD><TD>KH9</TD><TD>2100</TD></TR><TR><TD rowspan="2">pRB24</TD><TD>VR</TD><TD>KH5</TD><TD>2600</TD></TR><TR><TD>VR</TD><TD>RB64</TD><TD>1070</TD></TR><TR><TD>pRB7</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>digest
</TD></TR></TBODY></TABLE><P>We repeated the screening of pKH8 with 20 colonies and amplified svp with RB29/30 from pKH7.
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#i20" data-toggle="tab">Lab book</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="i20"><P style="font-size:12pt; text-align:justify;">
                               ===pRB23-T variants===
We amplified pRB23ΔccdB with KH3/4 in 2x 50 ul Phusion HF reactions and did a gel extraction.
</P><P>
We performed CPEC assembly using 2.7 ul of the KH3/4 fragment and 0.3 ul of each of the T-Domain PCR products, which have been amplified some weeks ago. <I>E. coli</I> TOP10 were transformed with 5 ul of the CPEC reaction mix. The transformants were incubated at 37 °C over night. We picked colonies to inoculate liquid cultures for the preparation of plasmid DNA.
The prepped plasmid were then used for co-transformed with each of the PPTase plasmids <A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB15-18</A> as well as without a PPTase. The following table shows the transformants after four days at room temperature in the drawer.



synT1, synT3 and synT4 result in blue colonies!!! We sequenced the prepped plasmids using the VR. They are all correct.
</P><H3><SPAN class="mw-headline" id="pRB15-19">pRB15-19</SPAN></H3><P>Transformation of TOP10 cells with the new assembled pRB16.3 variant.
</P><P>
Colony screening of six clones with VF2 and the svp reverse primer showed following result:
</P><P>
We prepped colony &quot;3&quot; and performed the first bulk of cotransformations with it.
Sequencing of the PPTase plasmids we used for the cotransformations were positive except that we found a point mutation in pRB16.3 which results in a stop codon right in the middle of svp. Therefore we will prepare the five other colonies except number 2 and screen 21 more colonies.
</P><P>
We prepped colonies 19 and 26 and cotransformed TOP10 cells together with pRB22. The shape of the colonies will show us whether the PPTase is correct or carries the stop codon.
</P><H3><SPAN class="mw-headline" id="Quantitative_Analysis_of_Indigoidine_production_-_Assay_developpement">Quantitative Analysis of Indigoidine production - Assay developpement</SPAN></H3><H4><SPAN class="mw-headline" id="Find_best_ODs_to_measure">Find best ODs to measure</SPAN></H4><P>We perform spectral analysis of an E. coli TOP10+<A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB22</A>+<A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB15</A> over night liquid culture in an Amersham Biosciences Ultrospec 3300 pro.
Pure TOP10 liquid culture is used as a reference and pure LB medium for blanking.
The absorption spectrum hsows a significant maximum at about 590 nm.
</P><H4><SPAN class="mw-headline" id="Upscaling">Upscaling</SPAN></H4><P>To be able to increase throughput we will use a Tecan infinite M200 plate reader for our growth kinetics and indigoidine production measurement. 
In a first test run we measure the OD at the wavelengths 400 nm, 530 nm, 590 nm and 800 nm every 5 minutes. The liquid cultures in this case are:
</P><UL><LI> TOP10 negative control
</LI><LI> TOP10 + pRB22
</LI><LI> TOP10 + pRB3
</LI><LI> MG1655 + pRB3
</LI></UL><P>By this we get an impression how we can detect and quantify both the total indigoidine production per well and the indigoidine production in relation to the cell density.
The measurement properties can be seen in the following figure.
</P><TABLE class="wikitable"><CAPTION>Table 16.x Bacterial cultures used for quantitative assay

</CAPTION></TABLE></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#i21" data-toggle="tab">Lab book</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="i21"><H3><SPAN class="mw-headline" id="Quantitative_Assay">Quantitative Assay</SPAN></H3><H4><SPAN class="mw-headline" id="Measurement">Measurement</SPAN></H4><P>This week we go for a first big measurement with our engineered indigoidine synthetases. The following figure is a scheme of our 96 well plate for the first measurement. We used one plate to grow positive (i.e. indigoidine producing) <I>E. coli TOP10</I> transformants from an agar plate in liquid culture for 30 hours at 37 °C. We prepared a second 96 well plate with 180 ul media and the respective antibiotics an inoculated with 20 ul of the liquid culture from the first plate.
The second plate was observed in our Tecan plate reader. This time we measured the absorbance from 400 nm to 800 nm in 2 nm steps and 25 reads for every well. This configuration unfortunately leads to the problem that the reader takes a long time for each measurement and we get huge time intervals between each measurement. Therefore we stopped the measurement after aboout 12 hours and changed the configuration. For the next 15 hours we used an interval of 10 nm and used 25 reads per well, so we are able to get a higher resolution on the time scale.
</P><P>The following table shows the loading scheme of our 96 well plate in this run.
</P><TABLE class="wikitable"><TBODY><TR><TH>	96-well	</TH><TH>	1	</TH><TH>	2	</TH><TH>	3	</TH><TH>	4	</TH><TH>	5	</TH><TH>	6	</TH><TH>	7	</TH><TH>	8	</TH><TH>	9	</TH><TH>	10	</TH><TH>	11	</TH><TH>	12
</TH></TR><TR><TD>	A	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-
</TD></TR><TR><TD>	B	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	Mücke/ Spinne	</TD><TD>	-	</TD><TD>	-	</TD><TD>	MG1655+pRB3	</TD><TD>	TOP10+pRB3	</TD><TD>	BAP1+pRB3	</TD><TD>	-
</TD></TR><TR><TD>	C	</TD><TD>	-	</TD><TD>	TOP10	</TD><TD>	-	</TD><TD>	pRB22	</TD><TD>	pRB23-T2	</TD><TD>	pRB23-T8	</TD><TD>	pRB23-T10	</TD><TD>	pRB23-T12	</TD><TD>	pRB23-T13	</TD><TD>	pRB23-T14	</TD><TD>	pRB23-T16	</TD><TD>	-
</TD></TR><TR><TD>	D	</TD><TD>	-	</TD><TD>	TOP10+pRB15	</TD><TD>	pRB3	</TD><TD>	pRB22+pRB15	</TD><TD>	pRB23-T2+pRB15	</TD><TD>	pRB23-T8+pRB15	</TD><TD>	pRB23-T10+pRB15	</TD><TD>	pRB23-T12+pRB15	</TD><TD>	pRB23-T13+pRB15	</TD><TD>	pRB23-T14+pRB15	</TD><TD>	pRB23-T16+pRB15	</TD><TD>	-
</TD></TR><TR><TD>	E	</TD><TD>	-	</TD><TD>	TOP10+pRB16	</TD><TD>	pRB4	</TD><TD>	pRB22+pRB16	</TD><TD>	pRB23-T2+pRB16	</TD><TD>	pRB23-T8+pRB16	</TD><TD>	pRB23-T10+pRB16	</TD><TD>	pRB23-T12+pRB16	</TD><TD>	pRB23-T13+pRB16	</TD><TD>	pRB23-T14+pRB16	</TD><TD>	pRB23-T16+pRB16	</TD><TD>	-
</TD></TR><TR><TD>	F	</TD><TD>	-	</TD><TD>	TOP10+pRB17	</TD><TD>	pRB6	</TD><TD>	pRB22+pRB17	</TD><TD>	pRB23-T2+pRB17	</TD><TD>	pRB23-T8+pRB17	</TD><TD>	pRB23-T10+pRB17	</TD><TD>	pRB23-T12+pRB17	</TD><TD>	pRB23-T13+pRB17	</TD><TD>	pRB23-T14+pRB17	</TD><TD>	pRB23-T16+pRB17	</TD><TD>	-
</TD></TR><TR><TD>	G	</TD><TD>	-	</TD><TD>	TOP10+pRB18	</TD><TD>	pRB24	</TD><TD>	pRB22+pRB18	</TD><TD>	pRB23-T2+pRB18	</TD><TD>	pRB23-T8+pRB18	</TD><TD>	pRB23-T10+pRB18	</TD><TD>	pRB23-T12+pRB18	</TD><TD>	pRB23-T13+pRB18	</TD><TD>	pRB23-T14+pRB18	</TD><TD>	pRB23-T16+pRB18	</TD><TD>	-
</TD></TR><TR><TD>	H	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Results">Results</SPAN></H4><P>We created plots showing the OD absorption versus wavelength for all timepoint from 0 to 30 hours (top). The plot down left is the OD590 versus OD800. This plot should be linear for every negative liquid culture which doesn't produce indigoidine. The last plot shows the OD of the indigoidine without the cells versus the OD800 from the liquid culture, thus showing how much indigoidine is produced ind a liquid culture at a specific OD.
</P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#i22" data-toggle="tab">Lab book</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="i22"><H3><SPAN class="mw-headline" id="BioBrick_Characterization">BioBrick Characterization</SPAN></H3><H4><SPAN class="mw-headline" id="Test_Digest">Test Digest</SPAN></H4><P>We digested 400 ng of each of our parts <A href="http://parts.igem.org/Part:BBa_K1152008" class="external text" rel="nofollow">BBa_K1152008</A> to BBa_K1152019 with EcoRI, XbaI, SpeI and PstI. We used 0.5 ul enzyme and incubated the reaction at 37 °C for 2 hours.
We submitted the plasmids pKH6-10, pRB22, pRB23, pRB23-T8, -T10, -T12, -T13 and -T14.
</P><P>The digest is positive and the fragment sizes are correct. Together with the sequencing results we can be sure that our submitted parts are alright.
                                </P></DIV></DIV></DIV></DIV></DIV><DIV class="labjournal-weekly"><DIV><UL class="nav nav-tabs"><LI class="active"><A href="#i23" data-toggle="tab">Lab book</A></LI></UL></DIV><DIV class="jumbotron weekly"><DIV class="nav navbar" data-spy="scroll" data-target="#navbarExample" data-offset="0"><DIV class="tab-content"><DIV class="tab-pane active" id="i23"><H3><SPAN class="mw-headline" id="Domain_Shuffling_-_Indigoidine_Conversions">Domain Shuffling - Indigoidine Conversions</SPAN></H3><P>This week we assemble 28 plasmids <A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB23-ds1112 to pRB23-ds3132</A>, which are variants of indC, in which single domains are exchanged by the corresponding domains of <A href="/wiki/index.php?title=Gene&amp;action=edit&amp;redlink=1" class="new" title="Gene (page does not exist)">plu2642</A>, a single module NRPS of unknown function from <I>P. luminescens</I> and <A href="/wiki/index.php?title=Gene&amp;action=edit&amp;redlink=1" class="new" title="Gene (page does not exist)">tycC2</A> from the tyrocidine cluster. The A-domains of both modules were predicted to be specific for glutamine binding and activation using the NRPS Designer. We will assemble all the domain combinations listed below. The plasmid code consissts of <I>pRB23-ds</I> and a code with four numbers. Each number defines the origin of the domain at the respective positions: indC is &quot;1&quot;, plu2642 is &quot;2&quot; and tycC2 is &quot;3&quot;. ds1122 for example means that in the general sequence of A-OxA-T-TE the A and OxA domain come from indC, whereas the T and TE domain are the ones of plu2642.
</P><P>The following table shows the fragments the pRB23-dsxxxx plasmids consist of:
</P><TABLE class="wikitable"><CAPTION> Table 19.1: Possible domain combinations using domains of indC (number<I>1</I> in the <I>dsxxxx</I> code), plu2642 (<I>2</I>) and tycC2 (<I>3</I>)
</CAPTION><TBODY><TR><TH>Fragment 1	</TH><TH>	Primer_fw	</TH><TH>	Primer_rv	</TH><TH>	Fragment 2	</TH><TH>	Primer_fw	</TH><TH>	Primer_rv	</TH><TH>	Fragment 3	</TH><TH>	Primer_fw	</TH><TH>	Primer_rv	</TH><TH>	Fragment 4	</TH><TH>	Primer_fw	</TH><TH>	Primer_rv	</TH><TH>	Fragment 5	</TH><TH>	Primer_fw	</TH><TH>	Primer_rv
</TH></TR><TR><TD>pRB23-ds1112	</TD><TD>	indC BB GE	</TD><TD>	RB27	</TD><TD>	KH6	</TD><TD>	Plum	</TD><TD>	RB110	</TD><TD>	RB100	</TD><TD>	pRB23	</TD><TD>	RB21	</TD><TD>	RB22	</TD></TR><TR><TD>pRB23-ds1121	</TD><TD>	pRB23	</TD><TD>	KH3	</TD><TD>	KH4	</TD><TD>	Plum	</TD><TD>	RB97	</TD><TD>	KH12	</TD></TR><TR><TD>pRB23-ds1122	</TD><TD>	pRB23	</TD><TD>	RB21	</TD><TD>	KH4	</TD><TD>	Plum	</TD><TD>	RB97	</TD><TD>	RB100	</TD></TR><TR><TD>pRB23-ds1131	</TD><TD>	pRB23	</TD><TD>	KH3	</TD><TD>	KH4	</TD><TD>	Bpara	</TD><TD>	RB105	</TD><TD>	RB106	</TD></TR><TR><TD>pRB23-ds1132	</TD><TD>	pRB23	</TD><TD>	RB21	</TD><TD>	KH4	</TD><TD>	Bpara	</TD><TD>	RB105	</TD><TD>	RB106	</TD><TD>	Plum	</TD><TD>	RB110	</TD><TD>	RB100	</TD></TR><TR><TD>pRB23-ds2022	</TD><TD>	pRB23	</TD><TD>	RB21	</TD><TD>	RB22	</TD><TD>	Plum	</TD><TD>	RB99	</TD><TD>	RB100	</TD></TR><TR><TD>pRB23-ds2111	</TD><TD>	Plum	</TD><TD>	RB99	</TD><TD>	RB101	</TD><TD>	indC BB GE	</TD><TD>	RB102	</TD><TD>	RB79	</TD><TD>	pRB23	</TD><TD>	RB21	</TD><TD>	RB22	</TD></TR><TR><TD>pRB23-ds2112	</TD><TD>	Plum	</TD><TD>	RB99	</TD><TD>	RB101	</TD><TD>	indC BB GE	</TD><TD>	RB102	</TD><TD>	KH6	</TD><TD>	Plum	</TD><TD>	RB110	</TD><TD>	RB100	</TD><TD>	pRB23	</TD><TD>	RB21	</TD><TD>	RB22	</TD></TR><TR><TD>pRB23-ds2121	</TD><TD>	Plum	</TD><TD>	RB99	</TD><TD>	RB101	</TD><TD>	pRB23	</TD><TD>	RB102	</TD><TD>	KH4	</TD><TD>	Plum	</TD><TD>	RB97	</TD><TD>	KH12	</TD><TD>	pRB23	</TD><TD>	KH3	</TD><TD>	RB22	</TD></TR><TR><TD>pRB23-ds2122	</TD><TD>	Plum	</TD><TD>	RB99	</TD><TD>	RB101	</TD><TD>	pRB23	</TD><TD>	RB102	</TD><TD>	KH4	</TD><TD>	Plum	</TD><TD>	RB97	</TD><TD>	RB100	</TD><TD>	pRB23	</TD><TD>	RB21	</TD><TD>	RB22	</TD></TR><TR><TD>pRB23-ds2131	</TD><TD>	Plum	</TD><TD>	RB99	</TD><TD>	RB101	</TD><TD>	pRB23	</TD><TD>	RB102	</TD><TD>	KH4	</TD><TD>	Bpara	</TD><TD>	RB105	</TD><TD>	RB106	</TD><TD>	pRB23	</TD><TD>	KH3	</TD><TD>	RB22	</TD></TR><TR><TD>pRB23-ds2132	</TD><TD>	Plum	</TD><TD>	RB99	</TD><TD>	RB101	</TD><TD>	pRB23	</TD><TD>	RB102	</TD><TD>	KH4	</TD><TD>	Bpara	</TD><TD>	RB105	</TD><TD>	RB106	</TD><TD>	Plum	</TD><TD>	RB110	</TD><TD>	RB100	</TD><TD>	pRB23	</TD><TD>	RB21	</TD><TD>	RB22
</TD></TR><TR><TD>pRB23-ds3111	</TD><TD>	Bpara	</TD><TD>	RB107	</TD><TD>	RB108	</TD><TD>	indC BB GE	</TD><TD>	RB102	</TD><TD>	RB79	</TD><TD>	pRB23	</TD><TD>	RB21	</TD><TD>	RB22	</TD></TR><TR><TD>pRB23-ds3112	</TD><TD>	Bpara	</TD><TD>	RB107	</TD><TD>	RB108	</TD><TD>	indC BB GE	</TD><TD>	RB102	</TD><TD>	KH6	</TD><TD>	Plum	</TD><TD>	RB110	</TD><TD>	RB100	</TD><TD>	pRB23	</TD><TD>	RB21	</TD><TD>	RB22	</TD></TR><TR><TD>pRB23-ds3121	</TD><TD>	Bpara	</TD><TD>	RB107	</TD><TD>	RB108	</TD><TD>	pRB23	</TD><TD>	RB102	</TD><TD>	KH4	</TD><TD>	Plum	</TD><TD>	RB97	</TD><TD>	KH12	</TD><TD>	pRB23	</TD><TD>	KH3	</TD><TD>	RB22	</TD></TR><TR><TD>pRB23-ds3122	</TD><TD>	Bpara	</TD><TD>	RB107	</TD><TD>	RB108	</TD><TD>	pRB23	</TD><TD>	RB102	</TD><TD>	KH4	</TD><TD>	Plum	</TD><TD>	RB97	</TD><TD>	RB100	</TD><TD>	pRB23	</TD><TD>	RB21	</TD><TD>	RB22	</TD></TR><TR><TD>pRB23-ds3131	</TD><TD>	Bpara	</TD><TD>	RB107	</TD><TD>	RB108	</TD><TD>	pRB23	</TD><TD>	RB102	</TD><TD>	KH4	</TD><TD>	Bpara	</TD><TD>	RB105	</TD><TD>	RB106	</TD><TD>	pRB23	</TD><TD>	KH3	</TD><TD>	RB22	</TD></TR><TR><TD>pRB23-ds3132	</TD><TD>	Bpara	</TD><TD>RB107	</TD><TD>	RB108	</TD><TD>	pRB23	</TD><TD>	RB102	</TD><TD>	KH4	</TD><TD>	Bpara	</TD><TD>	RB105	</TD><TD>	RB106	</TD><TD>	Plum	</TD><TD>	RB110	</TD><TD>	RB100	</TD><TD>	pRB23	</TD><TD>	RB21	</TD><TD>	RB22
</TD></TR></TBODY></TABLE><P>All fragments are amplified with Phusion HF (NEB) in 40 ul reactions in three thermocyclers and purified using agarose gel extraction. 
</P><TABLE class="wikitable"><CAPTION>Table 19.2: PCR fragments for assembly of pRB23-dsxxxx.
</CAPTION><TBODY><TR><TH>	Fragment</TH><TH>	Template	</TH><TH>	Primer_fw	</TH><TH>	Primer_rv	</TH><TH>	Usage	</TH><TH>	Fragment size [bp]	</TH><TH>	Annealing [°C]	</TH><TH>	Elongation [s]	</TH><TH>	x 40 ul MM
</TH></TR><TR><TD>	C_1	</TD><TD><A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB23</A></TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB21</A></TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB22</A></TD><TD>	8	</TD><TD>	2500	</TD><TD>	65	</TD><TD>	70	</TD><TD>	8
</TD></TR><TR><TD>	C_2	</TD><TD><A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB23</A></TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KH3</A></TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB22</A></TD><TD>	4	</TD><TD>	3300	</TD><TD>	65	</TD><TD>	70	</TD><TD>	4
</TD></TR><TR><TD>	C_3	</TD><TD><A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB23</A></TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB102</A></TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB22</A></TD><TD>	2	</TD><TD>	5000	</TD><TD>	65	</TD><TD>	120	</TD><TD>	2
</TD></TR><TR><TD>	C_4	</TD><TD><A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB23</A></TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB21</A></TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB81</A></TD><TD>	2	</TD><TD>	2900	</TD><TD>	65	</TD><TD>	70	</TD><TD>	2
</TD></TR><TR><TD>	C_5	</TD><TD><A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB23</A></TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KH3</A></TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB81</A></TD><TD>	2	</TD><TD>	6000	</TD><TD>	65	</TD><TD>	120	</TD><TD>	2
</TD></TR><TR><TD>	C_6	</TD><TD><A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB23</A></TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB102</A></TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB811</A></TD><TD>	8	</TD><TD>	1400	</TD><TD>	65	</TD><TD>	30	</TD><TD>	8
</TD></TR><TR><TD>	C_7	</TD><TD>	indC PCR product from <A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB22</A></TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB27</A></TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KH6</A></TD><TD>	1	</TD><TD>	3100	</TD><TD>	65	</TD><TD>	70	</TD><TD>	1
</TD></TR><TR><TD>	C_8	</TD><TD>	indC PCR product from <A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB22</A></TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB102</A></TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KH6</A></TD><TD>	2	</TD><TD>	1700	</TD><TD>	65	</TD><TD>	30	</TD><TD>	2
</TD></TR><TR><TD>	P_1	</TD><TD><A href="/wiki/index.php?title=Strain&amp;action=edit&amp;redlink=1" class="new" title="Strain (page does not exist)"><I>P. luminescens</I></A> cell pellet	</TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB99</A></TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB101</A></TD><TD>	6	</TD><TD>	1500	</TD><TD>	65	</TD><TD>	30	</TD><TD>	6
</TD></TR><TR><TD>	P_2	</TD><TD><A href="/wiki/index.php?title=Strain&amp;action=edit&amp;redlink=1" class="new" title="Strain (page does not exist)"><I>P. luminescens</I></A> cell pellet</TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB97</A></TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KH12</A></TD><TD>	3	</TD><TD>	300	</TD><TD>	65	</TD><TD>	30	</TD><TD>	2
</TD></TR><TR><TD>	P_3	</TD><TD><A href="/wiki/index.php?title=Strain&amp;action=edit&amp;redlink=1" class="new" title="Strain (page does not exist)"><I>P. luminescens</I></A> cell pellet</TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB110</A></TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB100</A></TD><TD>	6	</TD><TD>	850	</TD><TD>	65	</TD><TD>	30	</TD><TD>	4
</TD></TR><TR><TD>	P_4	</TD><TD><A href="/wiki/index.php?title=Strain&amp;action=edit&amp;redlink=1" class="new" title="Strain (page does not exist)"><I>P. luminescens</I></A> cell pellet</TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB97</A></TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB100</A></TD><TD>	3	</TD><TD>	1150	</TD><TD>	65	</TD><TD>	30	</TD><TD>	3
</TD></TR><TR><TD>	P_5	</TD><TD><A href="/wiki/index.php?title=Strain&amp;action=edit&amp;redlink=1" class="new" title="Strain (page does not exist)"><I>P. luminescens</I></A> cell pellet</TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB99</A></TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB100</A></TD><TD>	1	</TD><TD>	2800	</TD><TD>	65	</TD><TD>	70	</TD><TD>	1
</TD></TR><TR><TD>	T_1	</TD><TD><A href="/wiki/index.php?title=Strain&amp;action=edit&amp;redlink=1" class="new" title="Strain (page does not exist)"><I>B. parabrevis</I></A> cell pellet	</TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB107</A></TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB108</A></TD><TD>	6	</TD><TD>	1450	</TD><TD>	65	</TD><TD>	30	</TD><TD>	6
</TD></TR><TR><TD>	T_2	</TD><TD><A href="/wiki/index.php?title=Strain&amp;action=edit&amp;redlink=1" class="new" title="Strain (page does not exist)"><I>B. parabrevis</I></A> cell pellet	</TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB105</A></TD><TD><A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB106</A></TD><TD>	6	</TD><TD>	300	</TD><TD>	65	</TD><TD>	30	</TD><TD>	3
</TD></TR></TBODY></TABLE><P>The fragments then will be used for the <A href="/wiki/index.php?title=Methods&amp;action=edit&amp;redlink=1" class="new" title="Methods (page does not exist)">CPEC assembly</A> of all plasmids. CPEC is performed in 20 ul reactions for each plasmid so that it is sufficient for three transformations and an analytical agarose gel electrophoresis.
</P><TABLE class="wikitable"><CAPTION>Table 19.3: CPEC Assembly of pRB23-dsxxxx ind a 20 ul reaction
</CAPTION><TBODY><TR><TH><B>CPEC</B></TH><TH>	Fragment 1	</TH><TH>	~ conc. [nM]	</TH><TH>	Size [bp]	</TH><TH>	Volume in CPEC [ul]	</TH><TH>	Fragment 2	</TH><TH>	~ conc. [nM]	</TH><TH>	Size [bp]	</TH><TH>	Volume in CPEC [ul]	</TH><TH>	Fragment 3	</TH><TH>	~ conc. [nM]	</TH><TH>	Size [bp]	</TH><TH>	Volume in CPEC [ul]	</TH><TH>	Fragment 4	</TH><TH>	~ conc. [nM]	</TH><TH>	Size [bp]	</TH><TH>	Volume in CPEC [ul]	</TH><TH>	Fragment 5	</TH><TH>	~ conc. [nM]	</TH><TH>	Size [bp]	</TH><TH>	Volume in CPEC [ul]
</TH></TR><TR><TD>pRB23-ds1112	</TD><TD>	C_7	</TD><TD>	9,78	</TD><TD>	3100	</TD><TD>	7,0	</TD><TD>	P_2	</TD><TD>	404,04	</TD><TD>	300	</TD><TD>	1,0	</TD><TD>	C_1	</TD><TD>	90,91	</TD><TD>	2500	</TD><TD>	2,0	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-
</TD></TR><TR><TD>pRB23-ds1121	</TD><TD>	C_5	</TD><TD>	0,51	</TD><TD>	6000	</TD><TD>	9,0	</TD><TD>	P_2	</TD><TD>	404,04	</TD><TD>	300	</TD><TD>	1,0	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-
</TD></TR><TR><TD>pRB23-ds1122	</TD><TD>	C_4	</TD><TD>	0,00	</TD><TD>	2900	</TD><TD>	P_4	</TD><TD>	39,53	</TD><TD>	1150	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-
</TD></TR><TR><TD>pRB23-ds1131	</TD><TD>	C_5	</TD><TD>	0,51	</TD><TD>	6000	</TD><TD>	9,0	</TD><TD>	T_2	</TD><TD>	757,58	</TD><TD>	300	</TD><TD>	1,0	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-
</TD></TR><TR><TD>pRB23-ds1132	</TD><TD>	C_4	</TD><TD>	0,00	</TD><TD>	2900	</TD><TD>	T_2	</TD><TD>	757,58	</TD><TD>	300	</TD><TD>	P_3	</TD><TD>	35,65	</TD><TD>	850	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-
</TD></TR><TR><TD>pRB23-ds2022	</TD><TD>	P_5	</TD><TD>	2,16	</TD><TD>	2800	</TD><TD>	8,5	</TD><TD>	C_1	</TD><TD>	90,91	</TD><TD>	2500	</TD><TD>	1,5	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-
</TD></TR><TR><TD>pRB23-ds2111	</TD><TD>	P_1	</TD><TD>	4,04	</TD><TD>	1500	</TD><TD>	5,5	</TD><TD>	C_8	</TD><TD>	53,48	</TD><TD>	1700	</TD><TD>	1,5	</TD><TD>	C_2	</TD><TD>	18,37	</TD><TD>	3300	</TD><TD>	3,0	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-
</TD></TR><TR><TD>pRB23-ds2112	</TD><TD>	P_1	</TD><TD>	4,04	</TD><TD>	1500	</TD><TD>	5,0	</TD><TD>	C_8	</TD><TD>	53,48	</TD><TD>	1700	</TD><TD>	2,0	</TD><TD>	P_3	</TD><TD>	35,65	</TD><TD>	850	</TD><TD>	2,0	</TD><TD>	C_1	</TD><TD>	90,91	</TD><TD>	2500	</TD><TD>	1,0	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-
</TD></TR><TR><TD>pRB23-ds2121	</TD><TD>	P_1	</TD><TD>	4,04	</TD><TD>	1500	</TD><TD>	6,0	</TD><TD>	C_6	</TD><TD>	162,34	</TD><TD>	1400	</TD><TD>	1,0	</TD><TD>	P_2	</TD><TD>	404,04	</TD><TD>	300	</TD><TD>	1,0	</TD><TD>	C_2	</TD><TD>	18,37	</TD><TD>	3300	</TD><TD>	2,0	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-
</TD></TR><TR><TD>pRB23-ds2122	</TD><TD>	P_1	</TD><TD>	4,04	</TD><TD>	1500	</TD><TD>	6,0	</TD><TD>	C_6	</TD><TD>	162,34	</TD><TD>	1400	</TD><TD>	1,0	</TD><TD>	P_3	</TD><TD>	35,65	</TD><TD>	850	</TD><TD>	2,0	</TD><TD>	C_1	</TD><TD>	90,91	</TD><TD>	2500	</TD><TD>	1,0	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-
</TD></TR><TR><TD>pRB23-ds2131	</TD><TD>	P_1	</TD><TD>	4,04	</TD><TD>	1500	</TD><TD>	6,0	</TD><TD>	C_6	</TD><TD>	162,34	</TD><TD>	1400	</TD><TD>	1,0	</TD><TD>	T_2	</TD><TD>	757,58	</TD><TD>	300	</TD><TD>	1,0	</TD><TD>	C_2	</TD><TD>	18,37	</TD><TD>	3300	</TD><TD>	2,0	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-
</TD></TR><TR><TD>pRB23-ds2132	</TD><TD>	P_1	</TD><TD>	4,04	</TD><TD>	1500	</TD><TD>	6,0	</TD><TD>	C_6	</TD><TD>	162,34	</TD><TD>	1400	</TD><TD>	1,0	</TD><TD>	T_2	</TD><TD>	757,58	</TD><TD>	300	</TD><TD>	1,0	</TD><TD>	P_3	</TD><TD>	35,65	</TD><TD>	850	</TD><TD>	2,0	</TD><TD>	C_1	</TD><TD>	90,91	</TD><TD>	2500	</TD></TR><TR><TD>pRB23-ds3111	</TD><TD>	T_1	</TD><TD>	20,90	</TD><TD>	1450	</TD><TD>	4,0	</TD><TD>	C_8	</TD><TD>	53,48	</TD><TD>	1700	</TD><TD>	2,0	</TD><TD>	C_2	</TD><TD>	18,37	</TD><TD>	3300	</TD><TD>	3,0	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-
</TD></TR><TR><TD>pRB23-ds3112	</TD><TD>	T_1	</TD><TD>	20,90	</TD><TD>	1450	</TD><TD>	4,0	</TD><TD>	C_8	</TD><TD>	53,48	</TD><TD>	1700	</TD><TD>	2,0	</TD><TD>	P_3	</TD><TD>	35,65	</TD><TD>	850	</TD><TD>	3,0	</TD><TD>	C_1	</TD><TD>	90,91	</TD><TD>	2500	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-
</TD></TR><TR><TD>pRB23-ds3121	</TD><TD>	T_1	</TD><TD>	20,90	</TD><TD>	1450	</TD><TD>	4,0	</TD><TD>	C_6	</TD><TD>	162,34	</TD><TD>	1400	</TD><TD>	1,0	</TD><TD>	P_2	</TD><TD>	404,04	</TD><TD>	300	</TD><TD>	1,0	</TD><TD>	C_2	</TD><TD>	18,37	</TD><TD>	3300	</TD><TD>	4,0	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-
</TD></TR><TR><TD>pRB23-ds3122	</TD><TD>	T_1	</TD><TD>	20,90	</TD><TD>	1450	</TD><TD>	4,0	</TD><TD>	C_6	</TD><TD>	162,34	</TD><TD>	1400	</TD><TD>	1,0	</TD><TD>	P_3	</TD><TD>	35,65	</TD><TD>	850	</TD><TD>	3,5	</TD><TD>	C_1	</TD><TD>	90,91	</TD><TD>	2500	</TD><TD>	1,5	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-
</TD></TR><TR><TD>pRB23-ds3131	</TD><TD>	T_1	</TD><TD>	20,90	</TD><TD>	1450	</TD><TD>	4,0	</TD><TD>	C_6	</TD><TD>	162,34	</TD><TD>	1400	</TD><TD>	1,0	</TD><TD>	T_2	</TD><TD>	757,58	</TD><TD>	300	</TD><TD>	1,0	</TD><TD>	C_2	</TD><TD>	18,37	</TD><TD>	3300	</TD><TD>	4,0	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-	</TD><TD>	-
</TD></TR><TR><TD>pRB23-ds3132	</TD><TD>	T_1	</TD><TD>	20,90	</TD><TD>	1450	</TD><TD>	4,0	</TD><TD>	C_6	</TD><TD>	162,34	</TD><TD>	1400	</TD><TD>	1,0	</TD><TD>	T_2	</TD><TD>	757,58	</TD><TD>	300	</TD><TD>	1,0	</TD><TD>	P_3	</TD><TD>	35,65	</TD><TD>	850	</TD><TD>	4,0	</TD><TD>	C_1	</TD><TD>	90,91	</TD><TD>	2500	</TD></TR></TBODY></TABLE><P>We transform NEB Turbo cells with each plasmid (5 ul of the CPEC reaction product) and cotransform the cells with each plasmid and <A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB15</A> and <A href="/wiki/index.php?title=Plasmids&amp;action=edit&amp;redlink=1" class="new" title="Plasmids (page does not exist)">pRB17</A>, respectively.
                                </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12"><DIV class="imageMap" aria-haspopup="true"><CENTER>Thanks to </CENTER></DIV></DIV></DIV><DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Heidelberg/Indigoidine">http://2013.igem.org/Team:Heidelberg/Indigoidine</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Indigoidine" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Indigoidine" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Indigoidine&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Indigoidine&amp;oldid=360442" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>