<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Project_Tyrocidine"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Project/Tyrocidine">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Project/Tyrocidine&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/Tyrocidine&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/Tyrocidine&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Project/Tyrocidine" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Project/Tyrocidine</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="row"><DIV id="nav" class="navigation"><DIV class="item team" data-offset-l="-130"><H2>Team</H2><UL><LI><A href="/Team:Heidelberg/Team">Team Members</A></LI><LI><A href="/Team:Heidelberg/AttributionsII">Attributions</A></LI><LI><A href="/Team:Heidelberg/Team/Sponsors">Sponsors</A></LI><LI><A href="/Team:Heidelberg/Team/Collaboration">Collaborations</A></LI><LI><A href="/Team:Heidelberg/Team/Gallery">Gallery</A></LI></UL></DIV><DIV class="item project" data-offset-l="-144"><H2>Project</H2><UL><LI><A href="/Team:Heidelberg/NRPS">Background</A></LI><LI><A href="/Team:Heidelberg/Project">Overview</A></LI><LI><A href="/Team:Heidelberg/Project/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Project/Indigoidine-Tag">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Project/Tag-Optimization">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Project/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Project/Achievements">Achievements</A></LI></UL></DIV><DIV class="item notebook" data-offset-l="-139"><H2>Notebook</H2><UL><LI><A href="/Team:Heidelberg/Tyrocidine">Synthetic Peptides</A></LI><LI><A href="/Team:Heidelberg/Tyrocidine">Indigoidine-Tag</A></LI><LI><A href="/Team:Heidelberg/Indigoidine">Tag Optimization</A></LI><LI><A href="/Team:Heidelberg/Delftibactin">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/NRPSDesigner">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/Notebook/Methods">Methods</A></LI><LI><A href="/Team:Heidelberg/Materials">Materials</A></LI></UL></DIV><DIV class="item parts" data-offset-l="-134"><H2>Parts</H2><UL><LI><A href="/Team:Heidelberg/Parts">Submitted Parts</A></LI><LI><A href="/Team:Heidelberg/Favorite_Parts">Favorite Parts</A></LI><LI><A href="/Team:Heidelberg/RFCs">Our RFCs</A></LI></UL></DIV><DIV class="item software" data-offset-l="-134"><H2>Software</H2><UL><LI><A href="/Team:Heidelberg/Project_Software">Project</A></LI><LI><A href="http://igem2013.bioquant.uni-heidelberg.de/NRPSDesigner/">NRPSDesigner</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/iGEM42">iGEM42</A></LI><LI><A href="/Team:Heidelberg/OpenSource">Open Source</A></LI><LI><A href="/Team:Heidelberg/Project/AchievementsSoftware">Achievements</A></LI></UL></DIV><DIV class="item human" data-offset-l="-118"><H2>Human Practice</H2><UL><LI><A href="/Team:Heidelberg/HumanPractice">Overview</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BroadPublic">Broad Public</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Survey">Survey</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/YoungGeneration">Young Generation</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Artists">Arts and Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/BeyondScience">Beyond Science</A></LI><LI><A href="/Team:Heidelberg/HumanPractice/Experts">Experts</A></LI></UL></DIV><DIV class="item safety" data-offset-l="-115"><H2>Safety</H2><UL><LI><A href="/Team:Heidelberg/Safety">Safety</A></LI></UL></DIV><DIV class="item model" data-offset-l="-144"><H2>Modeling</H2><UL><LI><A href="/Team:Heidelberg/Modelling/Gold_Recovery">Gold Recycling</A></LI><LI><A href="/Team:Heidelberg/Modelling/Ind_Production">ind-Production</A></LI></UL></DIV><DIV class="item"><SPAN style="margin-left:6%">HOME</SPAN></DIV></DIV></DIV><DIV class="container"><DIV><H1><SPAN style="font-size:180%;color:#800000;">Synthetic Peptides.</SPAN><SPAN class="text-muted" style="font-family:Arial, sans-serif; font-size:100%"> Employing Modularity of NRPS.</SPAN></H1></DIV><DIV class="row"><DIV class="col-sm-6"><DIV class="jumbotron" style="height:480px"><H2>Highlights</H2><P style="font-size:16px"><UL style="font-size:14px"><LI><B> Interchanging of modules from the tyrocidine-cluster and thereby creating novel, synthetic peptides.</B></LI><LI><B> Successful detection of the synthetic peptides produced via Mass-Spectrometry.</B></LI><LI><B> Demonstration that custom NRPSs can be applied for efficient production of synthetic peptides.</B></LI></UL></P></DIV></DIV><DIV class="col-sm-6"><DIV class="jumbotron abstract" style="height:470px"><H2>Abstract</H2><P style="font-size:14px; text-align:justify">
                            Non-Ribosomal Peptide Synthetases (NRPSs) are specialized enzymes functioning as <B>peptide assembly lines</B>. NRPSs are composed of distinctive modules which determine the order of amino acids to be incorporated into a peptide and thereby the peptide sequence. Since NRPS have been shown to be <B>highly modular</B>, we performed a semi-rational shuffling approach of NRPS modules to investigate the feasibility of producing <B>artificial peptides</B> by creating custom NRPSs. As starting point we chose the NRPS encoded by the <B>tyrocidine cluster</B> of <EM>Brevibacillus Parabrevis</EM> and rearranged modules to form novel <B>synthetic peptides of different lengths</B>. Evaluation of recombinant NRPS expression was performed by SDS-PAGE and final peptide products were detected using Mass Spectrometry. We demonstrate the compatibility of different modules when placed in different orders and thereby show that engineered NRPSs can be used for efficient production of synthetic peptides..</P></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row" style="float:left"><DIV id="tyrocidineText" class="col-sm-12"><H2 id="introduction">Introduction</H2><P>
  Tyrocidine is a ten amino acid long non-ribosomal peptide (NRP) produced by the tyrocidine synthetase. This non-ribosomal peptide synthetase (NRPS) is composed of ten modules with specificity for different proteinogenic and non-proteinogenic amino acids <SPAN class="citation">[1]</SPAN>. The tyrocidine pathway can be found e.g. in <I>Brevibacillus parabrevis</I>. The gene cluster (

  <A class="fancybox fancyFigure" title="Figure 1: Schematic of the tyrocidine NRPS cluster. The tyrocidine-producing NRPS consists of ten Modules partitioned onto three proteins. In an assembly line manner each amino acid is added on after the other to the nascent peptide chain, before the final 10 amino acid product is cleaved from the last module and thereby released. (Adapted from [10])" href="https://static.igem.org/mediawiki/2013/4/44/Heidelberg_TycCluster_Scheme.png" rel="gallery1">


      Fig. 1

</A>


  ) encodes three proteins functioning together and constituting the NRPS: tycA, tycB and tycC. Those NRPSs contain different numbers of modules - tycA is a single module NRPS, tycB is composed of three modules and tycC of six modules. Only the complex composed of all three proteins is able to yield the specific NRP product, whereas the single modules are not functional or only produce fragments of the NRP <SPAN class="citation">[2]</SPAN>. Every module is again subdivided into domains. Among the most common domains are the <B>A</B>denylation domain, the <B>C</B>ondensation domain and the <B>T</B>hiolation domain. Beside, there are <B>E</B>pimerization domains, <B>T</B>hio-<B>E</B>sterase domains and <B>Com</B>munication domains <SPAN class="citation">[3]</SPAN><SPAN class="citation">[4]</SPAN><SPAN class="citation">[5]</SPAN>.
</P><CENTER><A class="fancybox fancyGraphical" title="Figure 1: Schematic of the tyrocidine NRPS cluster. The tyrocidine-producing NRPS consists of ten Modules partitioned onto three proteins. In an assembly line manner each amino acid is added on after the other to the nascent peptide chain, before the final 10 amino acid product is cleaved from the last module and thereby released. (Adapted from [10])" href="https://static.igem.org/mediawiki/2013/4/44/Heidelberg_TycCluster_Scheme.png" style="margin: 10px; width: 80%;"><FIGCAPTION style="width:80%"><B>Figure 1: Schematic of the tyrocidine NRPS cluster.</B> The tyrocidine-producing NRPS consists of ten Modules partitioned onto three proteins. In an assembly line manner each amino acid is added on after the other to the nascent peptide chain, before the final 10 amino acid product is cleaved from the last module and thereby released. Adapted from <SPAN class="citation">[10]</SPAN>. </FIGCAPTION></A></CENTER><P>During the synthesis of non-ribosomal peptides, the growing peptide-chain is transferred from one module to the next. The domains within the modules fulfill distinct functions and determine the modules' amino acid specificity. The C domain catalyzes the condensation of the nascent peptide chain (bound to the T domain) to the amino acid of the next module <SPAN class="citation">[6]</SPAN><SPAN class="citation">[7]</SPAN>. 

The remaining domains vary in their functions. For instance, Com domains are required for mediating protein-protein communication in NRPS composed of multiple proteins and are thus critical for their function <SPAN class="citation">[8]</SPAN>. This is, e.g. the case for the communication between the TycA and TycB1 modules present on seperate proteins <SPAN class="citation">[9]</SPAN>. Remarkably, all the six domain types described above are present in the tyrocidine synthetase of <I>B. parabrevis</I><SPAN class="citation">[10]</SPAN>.
As modules represent almost standalone units, each responsible for the incorporation of a specific amino acid monomer, they can be interchanged to form new non-ribosomal peptide synthetases synthesizing novel peptides <SPAN class="citation">[11]</SPAN>. Remarkably, the compatibility among moduoles is not restricted by the substrate specificty of the modules, but requires the general domain order to bew intact. Following this idea, we assembled new synthetic non-ribosomal peptide synthetases by shuffling modules of the tyrocidine NRPS and performed mass spectrometry in order to detect the corresponding synthetic peptide products.


</P><H2 id="results">Results </H2><H3 id="claims">Claims</H3><P>Here we show that modules of NRPSs can be interchanged resulting in engineered enzymes with novel functionalities. In the following, we have three claims to propose.</P><OL><LI>Modules of NRPSs can be interchanged.</LI><LI>The enzyme modularity allows the synthesis of custom peptides.</LI><LI>Customized NRPs can be purified and detected by Mass Spectrometry.</LI></OL><P>
  The establishment of a standard framework (<A href="https://2013.igem.org/Team:Heidelberg/RFCs#RFC100" title="wikilink"><B>RFC 100</B></A>) for <I>in vivo</I> synthesis of customized NRPs by non-ribosomal peptide synthetases required systematic investigation of NRPS modularity and compatibility. As proof of principle, we semi-rationally interchanged NRPS modules, a process we refer to as module shuffling   (<A class="fancybox fancyFigure" title="Figure 2: Assembly of synthetic peptides.&lt;/b&gt; Modules of native non-ribosomal peptide synthetases are interchanged to form artificial synthetases, which are capable of producing synthetic peptides." href="https://static.igem.org/mediawiki/2013/6/6c/Heidelberg_SynPep_Overview.png" rel="gallery1">


      Fig. 2
).
</A></P><CENTER><A class="fancybox fancyGraphical" title="&lt;b&gt;Figure 2: Assembly of synthetic Peptides.&lt;/b&gt; Modules of native non-ribosomal peptide synthetases are interchanged to form artificial synthetases, which are capable of producing synthetic peptides." href="https://static.igem.org/mediawiki/2013/6/6c/Heidelberg_SynPep_Overview.png" style="margin: 10px; width: 80%;"><FIGCAPTION style="width:80%"><B>Figure 2: Assembly of synthetic peptides.</B> Modules of native non-ribosomal peptide synthetases are interchanged to form artificial synthetases, which are capable of producing synthetic peptides.</FIGCAPTION></A></CENTER><H3 id="shuffling-modules-of-a-single-nrps">Shuffling Modules of a Single NRPS</H3>
We tried to interchange modules of the tyrocidine cluster of <EM>B. parabrevis</EM> in <EM>E. coli</EM>. As engineering of NRPS by shuffling modules underlies certain constraints, we had to develop certain design principles for NRPS engineering first. These included respecting the domain orders in modules as well as finding optimal linkers for fusing two different modules together.
The condensation domain catalyses the peptide bond formation between the nascent peptide and a new amino acid. We hence concluded, that the module TycA could be introduced at any position within a NRPS when putting it behind a suitable C domain. As the condensation domain takes up the amino acid attached to the preceding module it was removed for modules used as initiation modules. Furthermore, the peptide-peptide interaction is performed by communication domains. To investigate how crucial their functionality actually is the com-domains were replaced by suitable linkers when incorporating corresponding. When respecting these basic design principles for NRPS engineering, the engineering can be conducted in an efficient manner.

<H3>Assembling of a Synthetic NRPS </H3><P>To obtain a basis for shuffling and creation of artificial peptide synthetases, the coding sequences for each individual module were amplified from the tyrocidine cluster by the use of primers optimized for a Gibson assembly strategy. The primers were annealed to the unconserved linker regions between the NRPS domains that were predicted by <A href="http://pfam.sanger.ac.uk/">Pfam</A>. We successfully validated the correct amplification of 12 single DNA fragments comprising different modules as well as corresponding <A href="http://parts.igem.org/Part:pSB1C3">pSB1C3</A> backbone fragments by electrophoresis (
  <A class="fancybox fancyFigure" title="&lt;b&gt;Figure 3: Gel electrophoresis of our amplified fragments needed for module shuffling.&lt;/b&gt; Each fragment showed adequate bands on the gel.  All constructs were amplified with primers optimized for Gibson assembly, respectively. Numbers 1 to 15 refer to the different fragments needed: 1, 4, 9: pSB1C3; 2, 7: tycB1dCom; 3, 13, 15: tycC6; 5, 12: tycAdCom; 6, 8: tycC5-C6; 10: tycC5; 11: tycB1dCom-C(tycB2); 14: tycAdE." href="https://static.igem.org/mediawiki/2013/6/69/Heidelberg_Tyc_All_Fragments.png" rel="gallery1">
      Fig. 3
    </A>
    ). 
</P><CENTER><A class="fancybox fancyGraphical" title="&lt;b&gt;Figure 3: Gel electrophoresis of our amplified fragments needed for module shuffling.&lt;/b&gt; Each fragment showed adequate bands on the gel. All constructs were amplified with primers optimized for Gibson assembly, respectively. Numbers 1 to 15 refer to the different fragments needed: 1, 4, 9: pSB1C3; 2, 7: tycB1dCom; 3, 13, 15: tycC6; 5, 12: tycAdCom; 6, 8: tycC5-C6; 10: tycC5; 11: tycB1dCom-C(tycB2); 14: tycAdE." rel="gallery1" href="https://static.igem.org/mediawiki/2013/6/69/Heidelberg_Tyc_All_Fragments.png" style="  width:60%; margin: 10px;"><FIGCAPTION style=" width:60%; margin: 10px;"><B>Figure 3: Gel Electrophoresis of our amplified fragments needed for module shuffling.</B> Each fragment showed adequate bands on the gel. All constructs were amplified with primers optimized for Gibson assembly, respectively. Numbers 1 to 15 refer to the different fragments needed: 1, 4, 9: pSB1C3; 2, 7: tycB1dCom; 3, 13, 15: tycC6; 5, 12: tycAdCom; 6, 8: tycC5-C6; 10: tycC5; 11: tycB1dCom-C(tycB2); 14: tycAdE.</FIGCAPTION></A></CENTER><P>Since functionality of isolated modules cannot be shown, as they simply take up single amino acids without linking them to another monomer, we started to design small peptides two to four amino acid long. To show the compatibility of the tyrocidine modules with one another, combined different modules via <A href="https://2013.igem.org/Team:Heidelberg/Notebook/Methods" title="wikilink">Gibson Assembly</A>. The resulting constructs led to synthetic NRPSs and the production of five new peptides, i.e. one dipeptide, two tripeptides and two tetrapeptides.We were able to successfully assemble all of our plasmids and continued our work with our synthetic dipeptide (Proline-Leucine) NRPS and tripeptide-I-NRPS (Phenylalanine-Ornithine-Leucine)(

<A class="fancybox fancyFigure" title="&lt;b&gt;Figure 4: Creation of artificial non-ribosomal peptide synthetases from genes encoding for modules of the tyrocidine synthetase via Gibson assembly.&lt;/b&gt; Corresponding novel module sequences allow synthesis of a synthetic dipeptide (Pro-Leu) and a tripeptide (Phe-Orn-Leu)." href="https://static.igem.org/mediawiki/2013/d/d6/Heidelberg_Shuffling_scheme_di_tri_01.png" rel="gallery1">


      Fig. 4

    </A>





  ) by transforming them into the <EM>E. coli</EM> strain BAP1. 
</P><CENTER><A class="fancybox fancyGraphical" title="&lt;b&gt;Figure 4:Creation of artificial non-ribosomal peptide synthetases from genes encoding for modules of the tyrocidine synthetase via Gibson assembly.&lt;/b&gt; Corresponding novel module sequences allow synthesis of a synthetic dipeptide (Pro-Leu) and a tripeptide (Phe-Orn-Leu)." href="https://static.igem.org/mediawiki/2013/d/d6/Heidelberg_Shuffling_scheme_di_tri_01.png" style=" width:60%; margin: 10px;" rel="gallery1"><FIGCAPTION style="width:60%"><B>Figure 4: Creation of artificial non-ribosomal peptide synthetases from genes encoding for modules of the tyrocidine synthetase via Gibson assembly.</B> Corresponding novel module sequences allow synthesis of a synthetic dipeptide (Pro-Leu) and a tripeptide (Phe-Orn-Leu).</FIGCAPTION></A></CENTER><H3>Expression and Detection of the NRPS and its Products</H3><P>
Three hours after induction with IPTG samples expressing the putative synthetases were taken. The expression of the 212 kDa dipeptide synthetase and the 380 kDa tripeptide synthetase was shown by <A href="https://2013.igem.org/Team:Heidelberg/Notebook/Methods" title="wikilink">SDS-PAGE</A> (

  <A class="fancybox fancyFigure" title="Figure 5: SDS-PAGE confirming the expression of our engineered NRPSs. The 380 KDa tripeptide synthetase (Tri I) can be seen after 3 hours while 212 kDa dipeptide synthetase (in the samples Di A and Di B) can be seen 12 hours after induction. Respective bands are indicated by blue arrows." href="https://static.igem.org/mediawiki/2013/4/47/Heidelberg_SDS_2nd_20130830.png" rel="gallery1">


      Fig. 5

    </A>
    ). 
<CENTER><A class="fancybox fancyGraphical" title="Figure 5: SDS-PAGE confirming the expression of our engineered NRPSs. The 380 KDa tripeptide synthetase (Tri I) can be seen after 3 hours while 212 kDa dipeptide synthetase (in the samples Di A and Di B) can be seen 12 hours after induction. Respective bands are indicated by blue arrows." href="https://static.igem.org/mediawiki/2013/4/47/Heidelberg_SDS_2nd_20130830.png" style=" width:60%; margin: 10px;"><FIGCAPTION style="margin:5px; width:60%;"><B>Figure 5: SDS-PAGE confirming the expression of our engineered NRPSs.</B> The bands indicated by blue arrows show at 212 kDa the dipeptide synthetase and at 380 kDa the tripeptide synthetase. </FIGCAPTION></A></CENTER>

     Since SDS-PAGE is not sensitive enough to detect small peptides, we wanted to assess the presence of the newly synthesized peptides by the use of <A href="https://2013.igem.org/Team:Heidelberg/Notebook/Methods" title="wikilink">mass spectrometry</A>. As our tripeptide contains the non-proteinogenic amino acid ornithine, teh ornithine content in bacterial peptide samples should be increased in case the tripeptide would be produced in large amounts. We induced expression of the NRPSs with Isopropyl β-D-1-thiogalactopyranoside (IPTG). Samples were taken at different time points post induction. Since residual salts could potentially disturb the acquisition of small peptide abundances, we washed our LB-Cm culture in M9 minimal medium to minimize this effect and improve the detection of our short NRPs. Supernatant and the bacterial pellet were processed separately with distinct purification protocols to examine the distribution of the synthetic peptides and optimize purity of each sample. Final sample preparation for tandem mass-spectrometry was conducted at the neonate screening facility of the university medical center.
     Note that the peptides were hydrolyzed during the acidic butylation reaction. As ornithine is incorporated into the tripeptide but not the dipeptide, the ornithe content should be highly increased for the tripeptide samples when compared to the dipeptide samples.
</P><CENTER><A class="fancybox fancyGraphical" title="Figure 6: Retraced spectrum of the amino acid screening for ornithine for the tripeptide sample at 21 h.&lt;/b&gt; A peak at 189 Da indicates enrichment of ornithine, which can be translated to quantitative measures by a conversion factor deduced from a running standard." href="https://static.igem.org/mediawiki/2013/1/15/Heidelberg_Ornithine_tri_retraced.png" style=" width:60%; margin: 10px;"><FIGCAPTION style="margin:5px; width:60%;"><B>Figure 6: Retraced spectrum of the amino acid screening for ornithine for the tripeptide sample at 21 h.</B> A peak at 189 Da indicates enrichment of ornithine, which can be translated to quantitative measures by a conversion factor deduced from a running standard. </FIGCAPTION></A></CENTER><P>
     Finally the highly specific m/z profile allowed the identification of different amino acid abundances <SPAN class="citation">[12]</SPAN><SPAN class="citation">[14]</SPAN>. Since ornithine is a non-proteinogenic amino acid that is incorporated in our tripeptide (Phe-Orn-Leu) but not into the dipeptide (used as control for this assay), we no compared ornithe levels between those samples. (      
<A class="fancybox fancyFigure" title="Figure 6: Retraced spectrum of the amino acid screening for ornithine for the tripeptide sample at 21 h.&lt;/b&gt; A peak at 189 Da indicates enrichment of ornithine, which can be translated to quantitative measures by a conversion factor deduced from a running standard." href="https://static.igem.org/mediawiki/2013/1/15/Heidelberg_Ornithine_tri_retraced.png">Fig. 6</A>
     ).
Ornithine is a side-product of metabolic routes and has therefore not to be provided in the medium.
The abundance of ornithine in the tripeptide samples was highly increase compared to our dipeptide control sample (Pro-Leu), indicating that our tripeptide is successfully produced and the corresponding artificial NRPS is functional. 
     <A class="fancybox fancyFigure" title="&lt;b&gt;Figure 7: Levels of ornithine of samples containig the putative dipeptide or tripeptide.&lt;/b&gt; Pellets and supernatants were processes separately. Especially the supernatant of the tripeptide displays highly elevated ornithine levels. The plot shows ornithine levels at the peak 21 hours after induction. Error bars indicate the standard deviation from three independent experiments. Measurements were performed at neonate screening facility on campus via mass spectrometry." href="https://static.igem.org/mediawiki/2013/5/50/Heidelberg_barchart_ornithine.png" rel="gallery1">Fig. 7</A>
     ).
Due to variation of the general amino acid concentration in both used media and samples, we normalized the ornithine values by the amount of all amino acids present in the respective medium.
</P><CENTER><A class="fancybox fancyGraphical" title="&lt;b&gt;Figure 7: Levels of ornithine of samples containig the putative dipeptide or tripeptide.&lt;/b&gt; Pellets and supernatants were processes separately. Especially the supernatant of the tripeptide displays highly elevated ornithine levels. The plot shows ornithine levels at the peak 21 hours after induction. Error bars indicate the standard deviation from three independent experiments. Measurements were performed at neonate screening facility on campus via mass spectrometry." href="https://static.igem.org/mediawiki/2013/5/50/Heidelberg_barchart_ornithine.png" style=" width:60%; margin: 10px;"><FIGCAPTION style="margin:5px; width:60%;"><B>Figure 7: Levels of ornithine of samples containig the putative dipeptide or tripeptide.</B> Pellets and supernatants were processes separately. Especially the supernatant of the tripeptide displays highly elevated ornithine levels. The plot shows ornithine levels at the peak 21 hours after induction. Error bars indicate the standard deviation from three independent experiments. Measurements were performed at neonate screening facility on campus via mass spectrometry.</FIGCAPTION></A></CENTER><P>
The ornithine level in the supernatant of the tripeptide samples peaked 21 hours upon induction. Samples prepared from bacteria pellets showed neglectable increases in ornithine levels in comparison to the pure medium  and our negative control (untransformed <EM>E. coli</EM> strain BAP1) (data not shown). After having shown successful production of our tripeptide containing ornithine, we wanted to further characterize our (other) short peptides. Therefore, we sent two corresponding samples to analysis via high-resolution electrospray ionization (HR-ESI) mass spectrometry at the mass spectrometry facility of the Institute of Chemistry on campus. Remarkably, it turned out to be quite difficult to optimize the purification of the short peptides to the level needed for mass spec. Mass spec measurement background was too high for obtaining results that would allow final conclusions about the presence of the different peptides (MS Results: <A title="File:Heidelberg MS Pro Leu.pdf" href="https://static.igem.org/mediawiki/2013/c/cf/Heidelberg_MS_Pro_Leu.pdf">

    dipeptide

</A>
and

  <A class="fancybox fancyFigure" title="&lt;b&gt;Figure 8: Spectrum of the supernatant containing the tripeptide Phe-Orn-Leu.&lt;/b&gt; Only a small peak close to the expected size of 392,49 Da was observed. Due to the high background signal the presence of the peptide could not ultimately be confirmed by mass spec, calling for alternative methods of peptide characterization." href="https://static.igem.org/mediawiki/2013/5/5d/Heidelberg_MS_tri_360-540_OCI.png" rel="gallery1">


      Fig. 8

    </A>
    ).
</P><CENTER><A class="fancybox fancyGraphical" title="Figure 8: Spectrum of the supernatant containing the tripeptide Phe-Orn-Leu.&lt;/b&gt; Only a small peak close to the expected size of 392,49 Da was observed. Due to the high background signal the presence of the peptide could not ultimately be confirmed by mass spec, calling for alternative methods of peptide characterization." href="https://static.igem.org/mediawiki/2013/5/5d/Heidelberg_MS_tri_360-540_OCI.png" style=" width:60%; margin: 10px;"><FIGCAPTION style="margin:5px; width:60%;"><B>Figure 8: Spectrum of the supernatant containing the tripeptide Phe-Orn-Leu.</B> Only a small peak close to the expected size of 392,49 Da was observed. Due to the high background signal the presence of the peptide could not ultimately be confirmed by mass spec, calling for alternative methods of peptide characterization. </FIGCAPTION></A></CENTER><P>In summary, we were able to amplify single modules from the tyrocidine NRPS cluster, and we shuffled them via Gibson Assembly. Two constructs encoding two entirely new NRPSs were successfully transformed. The synthetases are both well expressed from the IPTG-inducible promoter present in pSB1C3 in <EM>E. coli</EM> strain BAP1. For the articficial NRPS producing the ornithine-containing tripeptide, NRPs production was succesffully confirmed by performing an ornithin quantification by mass spec. However, detecting successful peptide production turned out to be difficult for peptides not incorporating the non-proteinogenic amino acid ornithine. This led us to the conclusion that we need to develop alternative methods for detecting and characterizing synthetic NRPs, ideally via <A href="https://2013.igem.org/Team:Heidelberg/Project/Indigoidine-Tag" title="wikilink"><B>a simple NRP tagging method</B></A>. </P><H2 id="discussion">Discussion</H2><H3 id="discussion">Interchangeability of Modules</H3><P>
Module compatibility is the vital basis for any standardized work with NRPSs. Hence, the major objective of this project was to investigate flexible interchangeability of modules, which allows for customized synthesis of short peptides via NRPSs, as we propose in our standard (<A href="https://2013.igem.org/Team:Heidelberg/RFCs#RFC100" title="wikilink"><B>RFC 100</B></A>). Tyrocidine served as a paradigm for semi-rational rearrangements in the modular structure of NRPSs, as demonstrated by the creation of two different novel peptides through rearrangement of the respective modules that were amplified from <I>B. parabrevis</I>. Comparing the normalized levels of the non-proteinogenic amino acid ornithine in the different samples, we could conclude that the synthetic NRPS is in fact expressed enabling the creation of customized peptides that can be detected via mass spectrometry. 
</P><H3 id="discussion">Synthetic Peptides</H3><P>
Before our experiments, there was no evidence whether the synthetic peptides would be released to the medium or remained in the pellet. Showing that the resuspension of lyophilized supernatant in ethanol obtains a higher yield in ornithine content compared to the pellet samples, we can conclude that the small peptides are emitted into the medium. As salt concentrations did not seem to interfere with mass spectrometry measurements, the whole work up process has been successful. The final output of the tandem mass spectrometry demonstrated a highly elevated concentration of synthetic peptides in the medium compared to the cell interior, which leads to an improved protocol for standardized purification of short non-ribosomal peptides (<A href="https://2013.igem.org/Team:Heidelberg/RFCs#RFC100" title="wikilink"><B>RFC 100</B></A>). 
</P><P>
Furthermore, we found that there was no need to provide additional ornithine to the media since it was incorporated from <EM>E. coli</EM> as an intermediate product from L-glutamate. Moreover, this assay accounted for the functional incorporation of non-proteinogenic amino acids into artificial non-ribosomal peptides. A high variety of non-proteinogenic amino acids as constituents has already been described in literature <SPAN class="citation">[13]</SPAN>. The kinetics of ornithine as a proxy for the production of the tripeptide synthesis implied a more than fifty times higher ornithin content in the samples expressing the respective NRPS providing strong evidence for a functional synthetic NRPS.
</P><P>
In the future, synthesized non-ribosomal peptides could be elongated even further. Therefore, a stepwise-assembly strategy should be considered since the complexity of cloning will increase with the number and size of DNA fragments as well. Of course, additional modules for shuffling will increase the amount of opportunities to create comprehensive constructs. Such an approach is guided by the NRPS Designer, which frames our proposed standard (<A href="https://2013.igem.org/Team:Heidelberg/RFCs#RFC100" title="wikilink"><B>RFC 100</B></A>).
</P><H3 id="discussion">Detection Methods</H3><P>
Mass spectrometry often requires several tries and iterations of sample work up procedures to acquire good results. Since the synthetic peptides assembled by our artificial non-ribosomal peptide synthetases were rather small, their properties would not differ significantly from other salts in the solution. As we were able to cut salt concentrations significantly with our purification techniques, no problems of signal interference by the salty background occurred at the tandem mass spectrometry of the neonate screening facility. In combination with a butylation reaction a highly specific compound detection was possible making this method less prone to background noise. This technique could be the reason for the proper detection as it makes the product more distinctive. However, the procedure was not used for the mass spectrometry attempts to detect the whole peptides (as they should be kept intact) which could be the reason for the inconclusive results and also account for a bad or non-ionizability of the synthetic peptides. The generation of different unexpected adducts could be another reason for inconclusive spectra. Hence, for other mass spectrometry measurements an even more improved purification procedure could make the difference and lead to the spectra that were expected. The fragmentation of molecules at the thought target mass did not display significant decay products. Results of the differential approaches were aqcuired at two different facilities minimizing the risk of the mass spectrometre itself as an source of errors.
Increasing masses of the peptides and therefore facilitated purification procedures could be accomplished by means of click chemistry. The higher mass in combination with an easy cleavage would enable a better purification via HPLC with UV and ESI-MS detection <SPAN class="citation">[15]</SPAN>. This approach, however, does not offer any opportunity to evaluate expression <EM>in vivo</EM>’ <SPAN class="citation">[16]</SPAN>. 
</P><H3 id="discussion">Combinatorial Usage</H3><P>
Combinatorial chemistry gained significant recognition over the last decades. Screening of large libraries enables a target-oriented approach of compound research <SPAN class="citation">[17]</SPAN>. Exhibiting the manifoldness of non-ribosomal Peptides in combinatorial biology even short NRP sequences result in an impressive number of possible differentially composed products  (

  <A class="fancybox fancyFigure" title="&lt;b&gt;Figure 9: Number of different peptides molecules composed of 2, 3 or 4 amino acids that could be produced by ribosomal peptide synthesis in comparison to non-ribosomal peptide synthesis.&lt;/b&gt; Non-ribosomal peptides show a by far wider range of possible peptides. Note that the Y-axis is scaled logarithmically." href="https://static.igem.org/mediawiki/2013/8/85/Heidelberg_combinatorial_usage.png" rel="gallery1">


      Fig. 9

</A>


  ). 
As approximately 500 different monomers for NRPS exist even a dipeptide would result in 500<SUP>2</SUP> = 250 000 possible combinations of which, of course only a fraction would obtain a reasonable function. We managed a tripeptide, this is state of the art [Marahiel et al.] and gives us 125 million (500<SUP>3</SUP>) potential constructs to screen for, with our (<A href="https://2013.igem.org/Team:Heidelberg/RFCs#RFC100" title="wikilink"><B>RFC 100</B></A>).  This impressive numbers would increase even more with sequence length and the introduction of possible modifications, as branching or cyclization. In Comparison to cannonical peptide synthesis (22<SUP>3</SUP> = 10 648) this is far superior in terms of numbers and high throughput screening possibilities (Tab. 1). Although most of the products won´t inhabit specific features, for instance customized production of toxins, siderophores, pigments, antibiotics, cytostatics, and immunosuppressants would be facilitated. That even small peptides are functional can be seen from the success of peptide therapeutics <SPAN class="citation">[18]</SPAN> and from various natural products like the peptide enterobactin, for example. Databases like Norine <SPAN class="citation">[19]</SPAN> and Software as the NRPS Designer developed by us will assist these purposes.
</P><CENTER><TABLE border="1" bordercolor="#424242" style="background-color:#FFFFFF" width="70%" cellpadding="7" cellspacing="7"><CAPTION><B>Table 1:</B> Comparison of possible combinations for peptides of different sequence lengths for Ribosomal and Non-Ribosomal Peptides</CAPTION><TBODY><TR><TD align="center" valign="middle"><B>Sequence Length</B></TD><TD align="center" valign="middle"><B>Possible Non-Ribosomal Peptide Combinations*</B></TD><TD align="center" valign="middle"><B>Possible Ribosomal Peptide Combinations</B></TD></TR><TR><TD align="center" valign="middle">2</TD><TD align="center" valign="middle">250 000</TD><TD align="center" valign="middle">484</TD></TR><TR><TD align="center" valign="middle">3</TD><TD align="center" valign="middle">125 000 000</TD><TD align="center" valign="middle">10 648</TD></TR><TR><TD align="center" valign="middle">4</TD><TD align="center" valign="middle">62 500 000 000</TD><TD align="center" valign="middle">234 256</TD></TR></TBODY></TABLE>
*on basis of estimations of 500 possible monomers and exclusion of modifications as branching.
</CENTER><CENTER><A class="fancybox fancyGraphical" title="&lt;b&gt;Figure 9: Number of different peptides molecules composed of 2, 3 or 4 amino acids that could be produced by ribosomal peptide synthesis in comparison to non-ribosomal peptide synthesis.&lt;/b&gt; Non-ribosomal peptides show a by far wider range of possible peptides. Note that the Y-axis is scaled logarithmically." href="https://static.igem.org/mediawiki/2013/8/85/Heidelberg_combinatorial_usage.png" style="margin: 10px; width: 60%;"><FIGCAPTION style="width:80%"><B>Figure 9: Number of different peptides molecules composed of 2, 3 or 4 amino acids that could be produced by ribosomal peptide synthesis in comparison to non-ribosomal peptide synthesis.</B> Non-ribosomal peptides show a by far wider range of possible peptides. Note that the Y-axis is scaled logarithmically.</FIGCAPTION></A></CENTER><H2 id="conclusion">Conclusion</H2><P>In Conclusion we were able to prove interchangeability and recombinational potential of modules deriving from a single NRPS. The proper expression of custome dipeptide- and tripeptide-synthetase was shown by SDS-PAGES. Furthermore, we were able to establish an ornithine detection assay in order to prove the incorporation of non-proteinogenic amino acids (ornithine in our case) accounting for a production of the desired peptide. Using this assay we showed, that one of our engineered NRPSs producing a tripeptide composed of Phenylalanine-Ornithine-Leucine was successfully produced. Our findings are in line with research done by the group of Marahiel <EM>et al.</EM><SPAN class="citation">[11]</SPAN>. Our work opens up the engineering diversity of NRPS to the iGEM community. The achievements were enabled through and fed back to the <A href="http://igem2013.bioquant.uni-heidelberg.de/NRPSDesigner/" title="wikilink"><B>NRPS Designer</B></A>. This software tool incorporates a database utilizing our experimental on NRPS modules for guiding through user-oriented design of NRPSs. 
</P><H2 id="methodsMS">Methods</H2><H3 id="CloningStrategy">Cloning Strategy</H3><P>
The different synthetic non-ribosomal peptide synthetases were assembled on a chloramphenicol resistance backbone (pSB1C3) with an IPTG-inducable lac-promoter and the sequences coding for the desired target modules via Gibson assembly. The resulting constructs of this isothermal pot reaction were transformed into DH10β and tested by restriction digests and sequencing. Positive constructs were isolated and chemically transformed into <EM>E. coli</EM> BAP1 cells.
</P><H3 id="SDS-PAGE">SDS-PAGE</H3><P>
To test on the expression of the putative synthetic NRPS SDS-PAGES were performed. Therefore, cell cultures were centrifuged and supernatant discarded. The remaining pellet was resuspended in loading buffer and boiled to rupture the cells. After cooling the mixture down, it was centrifuged and loaded together with the NOVEX pre-stained standard on a 3-8% tris-acetate gradient gel for 1 hour at 180 V tris-acetate running buffer. Finally, the gels were stained with coomassie to visualize the running bands.
</P><H3 id="MassSpectrometrymethods">Mass Spectrometry</H3><H4 id="MSPreparationofSamples">Preparation of Samples</H4><P>
As high salt concentrations interfere with the m/z signal at mass spectrometry, we tried to cut salt concentrations from the very beginning.  Therefore, pre-cultures of transformed cells were prepared in LB media containing chloramphenicol and incubated overnight until OD600 ~ 0.6. To reduce the amounts of complex components, the cultures were centrifuged and remaining pellet washed with M9 minimal media. The washed cells were resuspended in M9 minimal media and again incubated until OD600 ~ 0.6 before inducing them with IPTG. After 21 hours pellets and supernatants were centrifuged and processed separated.
</P><H4 id="Sample Processing">Sample Processing</H4><P>Pellets were resuspended in PBS and lysed via ultra-sonification to release all components in to the buffer. After centrifugation the supernatant was extracted and frozen with liquid nitrogen to avoid peptide hydrolysis.
Supernatants of the samples after separation were frozen with liquid nitrogen and lyophilized to concentrate the solution. The lyophilisate was solubilized in a small amount of methanol, a standard solvent for HPLC and mass spectrometry, and vortexed to mix the phases. The supernatant was then frozen in liquid nitrogen. 
Samples of each purification procedure were analysed on a HR-ESI mass spectrometer (Bruker ApexQe hybrid 9.4 T FT-ICR).
</P><H4 id="SampleProcessingMSMS">Sample Processing for Tandem Mass Spectrometry at Neonate Screening</H4><P>
For evaluation of amino acid levels the frozen pellet and supernatant samples (Sample Processing) were hydrolysed and butylated to achieve a better ionization. Therefore, the samples were pipetted on filter paper sheets and dried chilled overnight. The sheets were extracted with Neo Gen stable isotope standard kit A und B (NSK-AB, EurIsotop) and centrifuged. The resulting supernatant was vaporized and the remaining dried compounds warmed up, before adding butanolic hydrochloric acid. The samples were shaken and heated up to improve the chemical reaction. Liquid residues were vaporated and samples heated up. Finally, an acetonitrile p.A. /water (1:1 vol/vol) solution was added as solvent for tandem mass spectrometry (Micromass Quattro Ultima (ESI-MS/MS)).
</P></DIV><DIV class="col-sm-12 jumbotron"><DIV class="references"><P>1. Mootz HD, Marahiel MA (1997) The tyrocidine biosynthesis operon of Bacillus brevis: complete nucleotide sequence and biochemical characterization of functional internal adenylation domains. J Bacteriol 179: 6843–6850.</P><P>2. Linne U, Marahiel MA (2000) Control of directionality in nonribosomal peptide synthesis: role of the condensation domain in preventing misinitiation and timing of epimerization. Biochemistry 39: 10439–10447.</P><P>3. Marahiel MA, Stachelhaus T, Mootz HD (1997) Modular Peptide Synthetases Involved in Nonribosomal Peptide Synthesis. Chem Rev 97: 2651–2674.</P><P>4. Finking R, Marahiel MA (2004) Biosynthesis of nonribosomal peptides. Annu Rev Microbiol 58: 453–488.</P><P>5. Marahiel MA (2009) Working outside the protein-synthesis rules: insights into non-ribosomal peptide synthesis. J Pept Sci 15: 799–807.</P><P>6. Weber T, Marahiel MA (2001) Exploring the domain structure of modular nonribosomal peptide synthetases. Structure 9: R3–R9.</P><P>7. Stachelhaus T, Mootz HD, Bergendahl V, Marahiel MA (1998) Peptide bond formation in nonribosomal peptide biosynthesis. Catalytic role of the condensation domain. J Biol Chem 273: 22773–22781.</P><P>8. Hur GH, Meier JL, Baskin J, Codelli JA, Bertozzi CR, <EM>et al.</EM> (2009) Crosslinking studies of protein-protein interactions in nonribosomal peptide biosynthesis. Chem Biol 16: 372–381.</P><P>9. Hahn M, Stachelhaus T (2004) Selective interaction between nonribosomal peptide synthetases is facilitated by short communication-mediating domains. Proc Natl Acad Sci USA 101: 15585–15590.</P><P>10. Linne U, Stein DB, Mootz HD, Marahiel MA (2003) Systematic and quantitative analysis of protein-protein recognition between nonribosomal peptide synthetases investigated in the tyrocidine biosynthetic template. Biochemistry 42: 5114–5124.</P><P>11. Doekel S, Marahiel MA (2000) Dipeptide formation on engineered hybrid peptide synthetases. Chem Biol 7: 373–384.</P><P>12.  Andreas Schulze, MD, Martin Lindner, MD, Dirk Kohlmüller, PhD, Katharina Olgemöller, Ertan 
  Mayatepek, MD, and Georg F. Hoffmann, MD. Expanded newborn screening for inborn errors of  
  metabolism by electrospray ionization-tandem mass spectrometry: results, outcome, and  
  implications. Pediatrics. (2003) Jun;111(6 Pt 1):1399-1406.</P><P>13. Banta-Wright SA, Steiner RD. (2004) Tandem mass spectrometry in newborn screening: a primer for neonatal and perinatal nurses. J Perinat Neonatal Nurs. 2004 Jan-Mar;18(1):41-58; quiz 59-60.</P><P>14. Symmank H, Franke P, Saenger W, Bernhard F (2002) Modification of biologically active peptides: production of a novel lipohexapeptide after engineering of Bacillus subtilis surfactin synthetase. Protein engineering 15: 913–921.</P><P>15. Franke R, Doll C, Eichler J (2005) Peptide ligation through click chemistry for the generation of assembled and scaffolded peptides. Tetrahedron letters 46: 4479–4482.</P><P>16. Kolb HC, Finn MG, Sharpless KB (2001) Click chemistry: diverse chemical function from a few good reactions. Angewandte Chemie International Edition 40: 2004–2021.</P><P>17. Jean-Marie Lehn, Alexey V. Eliseev (2001) Dynamic Combinatorial Chemistry. Science 23 March 2001: 291 (5512), 2331-2332.</P><P>18. David J. Craik, David P. Fairlie,Spiros Liras, David Price (2013) The Future of Peptide-based Drugs.Chem Biol Drug Des 2013; 81:136–147</P><P>19.Ségolène Caboche, Maude Pupin, Valérie Leclère, Arnaud Fontaine, Philippe Jacques, Gregory Kucherov
 (2008) NORINE: a database of nonribosomal peptides. Nucleic Acids Res. 2008 January; 36 D326–D331. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12"><DIV class="imageMap" aria-haspopup="true"><CENTER>Thanks to </CENTER></DIV></DIV></DIV><DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Heidelberg/Project/Tyrocidine">http://2013.igem.org/Team:Heidelberg/Project/Tyrocidine</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Project/Tyrocidine" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Project/Tyrocidine" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Project/Tyrocidine&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Project/Tyrocidine&amp;oldid=361790" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>