<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Templates_Indigoidine_week19"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Heidelberg/Templates/Indigoidine_week19">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Templates/Indigoidine_week19&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Templates/Indigoidine_week19&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Templates/Indigoidine_week19&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Templates/Indigoidine_week19" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Templates/Indigoidine week19</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#pRB15-18"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">pRB15-18</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Screening_delC"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Screening delC</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Screening_pKH4"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Screening pKH4</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Assembly_of_pKH4-derivates"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Assembly of pKH4-derivates</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Amplification_of_svp"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Amplification of svp</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Amplification_of_indC-fragments"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Amplification of indC-fragments</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Minipreps"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Minipreps</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Cotransformation"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Cotransformation</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Restriction_cloning_for_pRB15-19"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Restriction cloning for pRB15-19</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Reevaluation_of_old_MP_pRB15-18.2C_pRB21"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Reevaluation of old MP pRB15-18, pRB21</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#T-domain_exchange_with_different_domain_borders"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">T-domain exchange with different domain borders</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Screenings"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Screenings</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="pRB15-18">pRB15-18</SPAN></H3><P>The first mini
</P><H3><SPAN class="mw-headline" id="Screening_delC">Screening delC</SPAN></H3><TABLE class="wikitable"><CAPTION>Table 15.x Screening pRB18
</CAPTION><TBODY><TR><TH>Template</TH><TH>Primer</TH><TH>Comment
</TH></TR><TR><TD rowspan="2">TOP10+pSB3K3</TD><TD>RB64/VR</TD><TD rowspan="2">negative control
</TD></TR><TR><TD>VF2/65
</TD></TR><TR><TD rowspan="3">TOP10+pRB18</TD><TD>RB64/VR</TD><TD rowspan="3">screening
</TD></TR><TR><TD>VF2/65
</TD></TR><TR><TD>VF2/VR
</TD></TR><TR><TD>delC PCR product</TD><TD>RB64/RB65</TD><TD>positive control
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Screening_pKH4">Screening pKH4</SPAN></H3><TABLE class="wikitable"><CAPTION>Table 15.x Screening pKH4
</CAPTION><TBODY><TR><TH>Template</TH><TH>Primer
</TH></TR><TR><TD rowspan="2">TOP10+pKH4</TD><TD>KH52/VR
</TD></TR><TR><TD>VF2/RB71
</TD></TR></TBODY></TABLE><P>
A miniprep of probe 4 was sent to sequencing, which was positive.
</P><H3><SPAN class="mw-headline" id="Assembly_of_pKH4-derivates">Assembly of pKH4-derivates</SPAN></H3><P>pKH4 is used for assembly of pKH5 and variants of pKH4 with all the different T-Domains.
</P><TABLE class="wikitable"><CAPTION>Table 15.x Amplification KH3/4 from pKH4
</CAPTION><TBODY><TR><TH colspan="3">KH3/4 in BioRAD T100
</TH></TR><TR><TH>cycles</TH><TH>temperature [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">10</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>td 62</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>2.00
</TD></TR><TR><TD rowspan="3">25</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>57</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>2.00
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>6.00
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>
KH3/4-PCRs were unsuccessful, so we did a gradient PCR 
</P><TABLE class="wikitable"><CAPTION>Table 15.x Amplification KH3/4 from pKH4
</CAPTION><TBODY><TR><TH colspan="3">KH3/4 in biometra
</TH></TR><TR><TH>cycles</TH><TH>temperature [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">10</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>58-59-60-61-62-63</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>2.00
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>6.00
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>
We run the same protocol for 3x 50 ul.
</P><P>
nothing
</P><P>16 x 10 ul with Q5 and Tm calculator suggested parameters (63 °C annealing), pRB19 ligation as template
</P><P>
We tried NEB Phusion HF with 61 °C annealing temperature in a 10-, 20- and 50 ul reaction from pRB19 ligation and in a 20 ul reaction from pKH4 ind T100 (left).
As well we ran with Phusion HF (59 °C annealing) in 20 ul reactions with 0, 1 and 2 ul DMSO (Biorad Cycler 2)
</P><P>For further evaluation of the single plasmid strategy we admired a plasmid with bpsA instead of indC as indigoidine synthetase. For a quick shot we wanted to use pKH4 and pRB7 for a restriction approach with the enzymes KpnI and BamHI.
</P><UL><LI> prepare digestion (20 µl), 37 °C, ~0.5 h:
</LI><LI> pKH4: 2 µl NEBuffer CutSmart, 4 µl pKH4, 13 µl H2O,  0.5 µl (KpnI, BamHI)
</LI><LI> pRB7: 2 µl NEBuffer CutSmart, 2.3 µl pRB7, 14.7 µl H2O, 0.5 µl (KpnI, BamHI)
</LI></UL><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>Digestion of pKH4 (KpnI, BamHI) and pRB7 (KpnI, BamHI). Desired bands for insert of ~3.8 kbp and backbone+sfp of ~3.9 kbp were expected. 100 V, 0.8 % Agarose, TAE
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>The restriction for pKH4 did not work correctly due to the fact that with assembly of pKH4 the BamHI cutting site was removed and the plasmid was only linearized by the KpnI cutting site. For pRB7 two bands at almost equal site can be expected and thereby the digestion approach can not be used in this manner. We will try again with a similar approach like for the assembly of pKH4 as here a nonsense sequence instead of the PPtases will be introduced into pRB7.
</P><UL><LI> preparation of pRB7 MP by retransformation of TOP10 (as no glycerol stock is available).
</LI></UL><H3><SPAN class="mw-headline" id="Amplification_of_svp">Amplification of svp</SPAN></H3><P>The svp fragment we used so far was wrong, so we try to amplify it again from the genome, using various primers:
</P><UL><LI>RB13/14
</LI><LI>RB15/16
</LI><LI>RB29/30	
</LI><LI>RB39/40
</LI></UL><P>Gradient on biometra cycler (72 °C, 65 °C, 58 °C)
</P><P>
svp was always right; only after assembly it turned wrong. We will try to bring it in using restriction cloning and to sequence it; maybe the screening is just inconclusive.
</P><H3><SPAN class="mw-headline" id="Amplification_of_indC-fragments">Amplification of indC-fragments</SPAN></H3><P>indC fragments are amplified from Plum genome and indC short PCR product in a 50 ul Phusioin HF reaction. 
The fragments are:
</P><UL><LI> RB27/69
</LI><LI> RB70/71
</LI><LI> RB68/46
</LI><LI> RB68/KH4
</LI><LI> KH5/RB46
</LI><LI> RB27/46 from pRB14
</LI><LI> RB27/28
</LI></UL><TABLE class="wikitable"><CAPTION>Table 14.x PCR indC(ccdB) for pRB19
</CAPTION><TBODY><TR><TH colspan="3">RB27/69 in BioRAD T100 from indC short
</TH></TR><TR><TH colspan="3">RB27/69 in BioRAD T100 from P. luminescens pellet
</TH></TR><TR><TH colspan="3">RB70/71 in BioRAD T100 from indC short
</TH></TR><TR><TH colspan="3">RB70/71 in BioRAD T100 from P. luminescens pellet
</TH></TR><TR><TH colspan="3">KH5/RB46 in BioRAD T100 from indC short
</TH></TR><TR><TH colspan="3">KH5/RB46 in BioRAD T100 from P. luminescens pellet
</TH></TR><TR><TH>cycles</TH><TH>temperature [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>2.00
</TD></TR><TR><TD rowspan="3">35</TD><TD>98</TD><TD>5
</TD></TR><TR><TD>55</TD><TD>15
</TD></TR><TR><TD>72</TD><TD>30
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>2.00
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR><TR><TH colspan="3">RB68/46 in BioRAD T100 from indC short
</TH></TR><TR><TH colspan="3">RB68/46 in BioRAD T100 from P. luminescens pellet
</TH></TR><TR><TH colspan="3">RB68/KH4 in BioRAD T100 from indC short
</TH></TR><TR><TH colspan="3">RB68/KH4 in BioRAD T100 from P. luminescens pellet
</TH></TR><TR><TH>cycles</TH><TH>temperature [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>2.00
</TD></TR><TR><TD rowspan="3">30</TD><TD>98</TD><TD>15
</TD></TR><TR><TD>55</TD><TD>15
</TD></TR><TR><TD>72</TD><TD>2.00
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>5.00
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>Fragment RB70/71 is wrong because RB71 is wrong. We repeated the amplification with RB78 instead of RB71.
</P><P>Fragment KH3/RB46 is very weak, so we try to improve this reaction as well.
</P><TABLE class="wikitable"><CAPTION>Table 15.x Amplification indC RB21/46 from pRB14 and pSB1C3 RB21/22 from miniprep
</CAPTION><TBODY><TR><TH colspan="3">KH3/RB46 in BioRAD T100
</TH></TR><TR><TH>cycles</TH><TH>temperature [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">12</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>td 60</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>20
</TD></TR><TR><TD rowspan="3">23</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>63</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>20
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>60
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>And reaction RB68/RB46 with the same protocol but 60 s elongation time.
</P><P>We try KH3/RB79 with improved conditions.
</P><P>We tried to get this fragment with KH9/RB79 with Q5 in biometra cycler (60 °C annealing)
</P><P>We tried to get this fragment with KH9/RB79 with Phusion HF in biometra old cycler (61 °C annealing)
</P><P>Gel Extraction yielded xx ng/ ul
</P><P>CPEC assemblies were performed for following plasmids with the respective fragments:
</P><UL><LI> pRB19: RB27/79 from pRB14 and RB21/22 from pSB1C3 midiprep
</LI><LI> pRB22: RB27/69 RB70/78 RB68/79 RB21/22
</LI><LI> pRB23: RB27/69 RB70/78 RB68/KH4 KH9/RB79 (pRB14) RB21/22
</LI><LI> pRB24: RB27/28 RB66/67 RB21/22
</LI><LI> pKH5:  KH3/4 (pKH4) KH9/10
</LI></UL><P>Transformed cells looked as follows:
</P><P>
Colony screenings
</P><P>
pRB23 assembly was unsuccessful, pRB22 is correct (plates turn blue), pRB24 seems to be correct (small deep blue colonies), pRB19 didn't work, pKH5 seems correct. We prepped the marked colonies. Sequencing showed that pRB22 is correct and doesn't carry endogenous RFC10 cutting sites any longer. Test digestion with EcoRI and SpeI also enforced this result.
</P><P>We performed a PCR with Phusion HF and <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KH3/4</A> from pRB22, assembled it in a CPEC approach with ccdB KH9/10 and transformed OneShot cells.
</P><P>Colony screenings and test-transformation in TOP10 cells showed that the assembly was successful and that we're able to shuffle the T-domains.
</P><H3><SPAN class="mw-headline" id="Minipreps">Minipreps</SPAN></H3><P>Minipreps of pRB15, pRB17, pRB18, pKH4 and pRB14 yielded:
</P><UL><LI>pRB15
</LI><LI>pRB17
</LI><LI>pRB18
</LI><LI>pRB14: 284.8 ng/ ul in 20 ul
</LI><LI>pKH4: 316.8 ng/ ul in 20 ul
</LI></UL><P>5 ul of each pRB15, 17 and 18 were put on an agarose gel with small pockets.
</P><P>PPTase plasmid minipreps are empty. We will try to assemble them again using a restriction strategy besides prepping them again.
Repetitioin of the miniprep was successful and the plasmids were sent to sequencing.
</P><H3><SPAN class="mw-headline" id="Cotransformation">Cotransformation</SPAN></H3><P>We did cotransformation of pKH4 with pRB15-18, respectively, and single transformation of pSB3K3, pKH4 and pRB15-18.
</P><H3><SPAN class="mw-headline" id="Restriction_cloning_for_pRB15-19">Restriction cloning for pRB15-19</SPAN></H3><TABLE class="wikitable"><CAPTION>Table 15.x Amplification indC RB21/46 from pRB14 and pSB1C3 RB21/22 from miniprep
</CAPTION><TBODY><TR><TH colspan="3">RB21/22 in BioRAD T100
</TH></TR><TR><TH>cycles</TH><TH>temperature [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">12</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>td 60</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>50
</TD></TR><TR><TD rowspan="3">23</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>40
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>3.00
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR><TR><TH colspan="3">RB27/46 in BioRAD T100
</TH></TR><TR><TH>cycles</TH><TH>temperature [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">12</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>td 60</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>70
</TD></TR><TR><TD rowspan="3">18</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>60
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>2.00
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>
svp was reamplified from short product
</P><TABLE class="wikitable"><CAPTION>Table 15.x Amplification indC RB21/46 from pRB14 and pSB1C3 RB21/22 from miniprep
</CAPTION><TBODY><TR><TH colspan="3">RB21/22 in BioRAD T100
</TH></TR><TR><TH>cycles</TH><TH>temperature [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>98</TD><TD>10
</TD></TR><TR><TD rowspan="3">35</TD><TD>98</TD><TD>1
</TD></TR><TR><TD>65</TD><TD>5
</TD></TR><TR><TD>72</TD><TD>20
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>60
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><P>
indC and pSB1C3 were digested with KpnI and NheI, sfp, svp, entD, delC and pSB3K3 with BamHI and NheI for two hours at 37 °C.
</P><P>
Unfortunately pSB3K3 contains an internal NheI cutting site. We repeated the amplification, purification, restriction digest and purification with BamHI and SpeI, which builds a scar with NheI.
</P><P>
Ligation of pSB1C3 with indC(ccdB) was performed over night in a 10 ul reaction with 4 ul fragments each.
</P><P>Transformation of ligation mixes were performed with 5 µl each in TOP10. Additionally 5 µl of ligation mix of pRB19 was transformed into OneShot cells.
</P><P>
Colony screenings showed following results:
</P><P>We prepped pRB15, 17 and 18 and repeated the assembly of pRB16 from scratch. Therefore we used <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB13/14</A> for amplification of svp from an S. verticilllus ATCC15003 liquid culture. Afterwards we reamplified this PCR gel extraction with <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">RB29/30</A> and performed a restriction digest with BamHI and NheI to ligate the fragment with pSB3K3 (BamHI; SpeI).
</P><H3><SPAN class="mw-headline" id="Reevaluation_of_old_MP_pRB15-18.2C_pRB21">Reevaluation of old MP pRB15-18, pRB21</SPAN></H3><P>Because we could observe small colonies with darkgray center on cotransformed pRB21 with all PPtases constructs (plates of 2013-08-27), we conducted a colony screen and a retransformation with 1 µl of previous MP of pRB15.2 (Konrad) and pRB16-18 (Ralf) and pRB21 (Ralf). Additionally a cotransformation with this old pRB21 MP with the old pRB15.2 as well with 2.5 µl of newly assembled pRB15 was performed.
</P><P>
The colony screening was performed using the primers <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">VR</A> and <A href="/wiki/index.php?title=Primer&amp;action=edit&amp;redlink=1" class="new" title="Primer (page does not exist)">KH5</A>. If the gray colonies carry two plasmids with indigoidine synthetase and activating PPtase separately an amplicon of size ~1.3 kbp. Otherwise if for some reason the PPtase and indigoidine synthetase are still on one plasmid, we could expect an amplicon of 2.1 kbp.
</P><TABLE class="wikitable"><CAPTION>Table 15.x colony screening cotrafo pRB21 and pRB15-18, 2013-08-27
</CAPTION><TBODY><TR><TH colspan="3">KH5/VR in Biometra, 10 µl
</TH></TR><TR><TH>cycles</TH><TH>temperature [°C]</TH><TH>time [s]
</TH></TR><TR><TD>1</TD><TD>95</TD><TD>120
</TD></TR><TR><TD rowspan="3">30</TD><TD>95</TD><TD>30
</TD></TR><TR><TD>60</TD><TD>30
</TD></TR><TR><TD>72</TD><TD>120
</TD></TR><TR><TD>1</TD><TD>72</TD><TD>240
</TD></TR><TR><TD>1</TD><TD>12</TD><TD>-
</TD></TR></TBODY></TABLE><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 435px;"><DIV class="gallerytext"><P>Colony PCR of cotransformation pRB21 and pRB15-18. Desired bands would be expected at 1.3 kbp for positive clones. For positive controls amplicons were expected at ~2 kbp for pRB3 and at ~1.5 kbp for pKH4. 100 V, 0.8 % Agarose, TAE
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>The positive controls gave the expected bands. All picked clones also gave an expected band at 1.3 kbp which leads to the conclusion that they carry the indC-only plasmid pRB21. Because every third clone was a picked white colony we could also conclude, that not all transformants carry an intact PPtase plasmid.
</P><H3><SPAN class="mw-headline" id="T-domain_exchange_with_different_domain_borders">T-domain exchange with different domain borders</SPAN></H3><P>We thought of a different way to set the T-Domain borders. Starting from the Pfam-predicted domain borders we made pairwise alignments to find amino acid sequences up- and downstream the predicted borders, that are identical in both the native and the brought in sequence. In the case of the exchange of indC and bpsA those new borders would be around 10 amino acids more on each side. We designed primers for this strategy.
</P><H3><SPAN class="mw-headline" id="Screenings">Screenings</SPAN></H3><TABLE class="wikitable"><CAPTION>Table 15.x Screening of new constructs
</CAPTION><TBODY><TR><TH>Plasmid</TH><TH>Primer 1</TH><TH>Primer 2</TH><TH>Expected product [bp]</TH><TH>wrong product [bp]</TH><TH>additional test
</TH></TR><TR><TD>pKH6</TD><TD>VR</TD><TD>RB17</TD><TD>1070</TD></TR><TR><TD>pKH7</TD><TD>VR</TD><TD>RB13</TD><TD>1120</TD></TR><TR><TD>pKH8</TD><TD>VR</TD><TD>RB41</TD><TD>1000</TD></TR><TR><TD>pKH9</TD><TD>VR</TD><TD>RB64</TD><TD>1070</TD></TR><TR><TD>pRB15_lig</TD><TD>VR</TD><TD>RB17</TD><TD>1070</TD></TR><TR><TD>pRB16_lig</TD><TD>VR</TD><TD>RB13</TD><TD>1120</TD></TR><TR><TD>pRB17_lig</TD><TD>VR</TD><TD>RB41</TD><TD>1000</TD></TR><TR><TD>pRB18_lig</TD><TD>VR</TD><TD>RB64</TD><TD>1070</TD></TR><TR><TD rowspan="2">pRB19_lig</TD><TD>VR</TD><TD>KH9</TD><TD>1900</TD><TD>2600</TD></TR><TR><TD>VR</TD><TD>RB17</TD><TD>-</TD><TD>1070</TD></TR><TR><TD>pRB22_frag</TD><TD>digest
</TD></TR><TR><TD rowspan="2">pRB23_frag</TD><TD>VR</TD><TD>KH9</TD><TD>1900</TD><TD>digest
</TD></TR><TR><TD>RB27</TD><TD>KH4</TD><TD>2900</TD></TR><TR><TD>pKH5</TD><TD>VR</TD><TD>KH9</TD><TD>2100</TD></TR><TR><TD rowspan="2">pRB24</TD><TD>VR</TD><TD>KH5</TD><TD>2600</TD></TR><TR><TD>VR</TD><TD>RB64</TD><TD>1070</TD></TR><TR><TD>pRB7</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>digest
</TD></TR></TBODY></TABLE><P>We repeated the screening of pKH8 with 20 colonies and amplified svp with RB29/30 from pKH7.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Heidelberg/Templates/Indigoidine_week19">http://2013.igem.org/Team:Heidelberg/Templates/Indigoidine_week19</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Templates/Indigoidine_week19" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Templates/Indigoidine_week19" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Templates/Indigoidine_week19&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Templates/Indigoidine_week19&amp;oldid=323328" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>