"
Page
Discussion
View source
History
teams
Log in
 
Team:Melbourne/Project
From 2014.igem.org
Home
Team
Project
Human Practices
Achievements
Notebook
Protocols
Safety
Sponsors
Attributions
Project and results
Read about our experimental work below, or jump to our theoretical collaboration with the University of Oxford
Introduction and theory
Synthetic peptide chemists have long produced peptide-based materials in vitro. Star-shaped peptides are a promising type of biomaterial being explored in the field of nanomedicine (Sulistio et al., 2012). Star peptides can have several biomedical uses, such as acting as drug delivery vehicles (Sulistio et al., 2011) or linkers for other biomacromolecules. Star peptides generally take the form of several linear peptide arms linked together in a central core. One way of linking these linear peptide arms together is to used covalent bonds such as those in disulfides. Typically, disulfide bonds are formed synthetically by taking several linear arms and treating them with an oxidant in vitro. Here, we introduce a new approach to forming star peptides using E. coli and synthetic biology. Thus, we aimed to show how the peptides synthesis and disulfide bond forming machinery of E. coli can be used to form disulfide linked star peptides and key star peptide precursors.
 
Synthesis approach
E. coli naturally possesses the capacity to form disulfide bonds. In native strains, disulfide bonds are naturally formed by an array of enzymes which are part of the Dsb family (e.g. DsbA and DsbC) (Kadokura et al., 2003, Kadokura and Beckwith, 2009). Normally, these enzymes are found in the oxidizing periplasm of the cell. However, several new strains of E. coli have recently been engineered which contain an oxidizing cytoplasm conducive to disulfide bond formation. One example of this is the SHuffle cell line (Lobstein et al., 2012). The cell line contains mutations to key enzymes responsible for the reducing nature of the cytoplasm, namely thioredoxin reductase (trxB) and glutathione reductase (gor). Furthermore, the Shuffle cell line over expresses the disulfide bond isomerase DsbC to the cytoplasm. Together, these mutations allow SHuffle to fold disulfide-bonded proteins in the cytoplasm at a higher success rate compared to non-mutants. We aimed to take advantage of the disulfide bond forming capabilities of this strain of E. coli to synthesize star peptides in cells. As shown in the figure below, the synthesis steps may proceed as follows:
 
Express a short peptide containing two cysteine residues either to the E. coli periplasm or the cytoplasm of a trxB gor mutant.
E. coli disulfide bond forming enzymes fold the peptide into a hairpin loop structure.
Cut the loop at a protease recognition site engineered into the peptide. This may be done by extracting the folded peptide from the cell and treating it with the protease in vitro. Alternatively, the protease may be co-expressed in the cell to allow for in vivo cleavage.
 
 
This synthesis approach has several benefits over purely in vitro approaches. Firstly, the exact peptide sequence can be precisely programmed into E. coli using recombinant DNA synthesis. Secondly, by performing the disulfide bond formation in cells and optionally the proteolytic cleavage, several synthesis steps which would need to be performed in vitro are eliminated. From a scale up perspective, this would eliminate entire unit operations which would otherwise be required to produce this product. Given these benefits, in the current study, we aimed to express a star peptide precursor to the cytoplasm of SHuffle cells, which would later be extracted and externally digested with the star-forming protease. In order to achieve this, we first designed several star peptides which might be amenable to this synthetic strategy. These peptides are described below.
 
Rationally designed peptides
There are two approaches to functionalising star peptides. In the first, the identity of the arms can be chosen to be bioactive peptide molecules. This is the simplest approach to producing a biologically-relevant star. In the second, the arms can be functionalised by ligating molecules to them at any point. We used both of these approaches to design two separate peptides.
Magainin 1 star and linear peptides
Our first strategy was to make a star peptide using antimicrobial peptides(AMPs) as building blocks. AMPs are small, approximately 50 residue peptides secreted by some bacterial and eukaryotic cells which selectively kill microbial cells. It is thought that AMPs work by forming pores in the membrane of prokaryotic cells (Brogden, 2005). AMPs have been recombinantly expressed in a number of organisms, including E. coli (for a review, see Li, 2011) and B. Subtilis (Chen et al., 2009, Yu et al., 2013).
Our concept was to design a star peptide with antimicrobial peptide arms. Wiradharma et al. (2012) first showed that placing linear antimicrobial peptides in a star configuration could lead to enhanced antimicrobial activity and decreased hemolytic activity. Although it is unclear why this is the case, it may be due to the ability of neighbouring antimicrobial peptide arms to interact with each other to synergistically rupture the membrane.
While Wiradharma used a synthetic peptide sequence, we designed a peptide using the naturally occurring AMP, Magainin 1 (Zasloff, 1987). Magainin 1 peptides will be placed to the ends of each star arm.
Magainin 1 was chosen because the Magainins are one of the major classes of antimicrobial peptides, being well studied and characterised. In addition, we were concerned that tethering the antimicrobial peptide to the star peptide might interfere with its antimicrobial activity. Magainin 1, however, has previously been tethered to surfaces, where it has imparted the surfaces with microbicidal properties (Glinel et al., 2008; Humblot et al., 2009). We surmised that if Magainin 1 maintained its activity while anchored to surfaces, it may also maintain its activity while anchored to a star peptide.
The sequence for Magainin 1 is: GIGKFLHSAGKFGKAFVGEIMKS.
When attached to a star peptide it will have the following structure:
There are several design elements to note:
The antimicrobial peptide star will be expressed with a SUMO fusion protein. This is because without the fusion, it is likely that the peptide would be toxic to the host cell.
The peptide includes a Factor X cutting site between the two cysteines for eventual proteolytic cleavage and formation of the star peptide.
In addition to the star peptide Magainin 1, we synthesised a gene for a linear Magainin 1 peptide as well. This is identical to the construct above, except that there is only one Magainin 1 peptide attached to the SUMO fusion.
Unstructured Peptide (USP) Construct
In the second approach, we designed a peptide which can be functionalised using chemical approaches. This peptide was designed to have flexible, unstructured arms and was termed the USP construct. Unlike the Magainin 1 star, the arms of this peptide serve not as active peptides themselves, but as inert structural linkers.
The arms were designed with the following elements in mind:
Lack of structure. The arms were designed with a bioinspired approach, using the FxFG motif of nucleoporins (where x is a variable amino acid residue). Such segments naturally repeat in nucleoporins and are thought to lead to disorder/lack of stable secondary structure. Nucleoporins are found in mammalian cells, serving as flexible brushes around nuclear pores (Ader et al., 2010).
Water-soluble. The arms were designed with several charged amino acids to improve solubility.
Designed to form a disulfide bond. Although it is difficult to rationally ensure that the disulfide bond will form between two cysteines in our peptide, we incorporated a beta turn between the two cysteines which may encourage the peptide to fold at the apex of the hairpin loop. This may bring the cysteines into closer proximity, providing more probable bond formation.
The ultimate utility of this peptide lies in its ability to be functionalised with other biomacromolecules. For example, the technique of Native Chemical Ligation(NCL) can be used to join peptides, proteins, and other ligands to the arms (Dawson et al., 1994). The idea of attaching enzymes to the star peptide was explored by the University of Oxford iGEM 2014 team in a collaborative effort between our two teams (See Supplementary Project Work at the end of this page).
 
Expression system
In order to successfully express our constructs, we designed our protein expression vectors to include a fusion protein. The fusion protein was necessary for two reasons. First, some of our constructs are very small (e.g. the non-star Magainin 1), and expression levels of very small peptides can be difficult without a fusion partner. Second, two of our constructs code for antimicrobial peptides. Without a fusion partner, it is likely that these genes would be toxic to their hosts upon induction.
To find a suitable expression system, we looked towards the Registry of Standard Parts. We used the SUMO protein expression system designed by TU Delft 2014 (for example, see http://parts.igem.org/wiki/index.php?title=Part:BBa_K1022101 ). This system essentially consists of a N-terminal HIS-tag followed by the SUMO protein (also known as UlpI).
The strategy of expressing toxic AMPs using SUMO has been successfully reported in the literature (Bommarius et al., 2010). We surmise that the SUMO protein could inhibit the antimicrobial activity of single, linear peptides, and that it may also inhibit the activity of our star antimicrobial peptide.
SUMO as a fusion protein also has the benefit of leaving no residues at the C-terminal end of the cleavage site. This means that upon cleavage with the SUMO protease, the native protein can be recovered. In our case, this means that one of the arms of the star can be designed without the need to take into account the addition of any amino acid residues left behind by the protease.
We used the SUMO peptide sequence reported by TU Delft. However, our construct contained the following unique features:
Standardisation of the biobrick by substituting the T7 promoter and RBS (the origins of which are both not specified in the Delft documentation) with the standard T7 promoter and RBS BioBrick BBa_K525998 . In addition to supporting the principle of standardization, using the well-characterized promoter BioBrick should help assure expression levels.
Biobrick BBa_K1022101 lacks a terminator sequence (this was presumably because the part was meant to be integrated into a larger genetic construct with a terminator). A terminator from the registry of standard parts was added (specifically, the wild type terminator from T7 bacteriophage,  BBa_K731721 ).
The original biobrick BBa_K1022101 codes for three amino acid residues before the HIS-tag (ASM), which appeared to be redundant. Correspondence with the 2013 TU Delft team suggested that these residues were unnecessary and appear to be cleaved within the cell as part of the cells post-translational modifications. However, their presence complicates the addition of additional tags at the N-terminus of the protein (e.g. periplasmic export tags), and therefore were not included.
The SUMO sequence was codon optimised for E. coli during synthesis. As the Delft documentation did not specify whether the gene was codon optimal, codon optomisation was undertaken to potentially improve expression levels. The linear Magainin 1 construct, however, was not codon optimised in the SUMO region in order to provide a control condition.
To summarise, the protein expression devices used in our project to the following form:
The protein coding region consisted of a 6x-HIS tag followed by the SUMO protein and the relevant star or linear peptide.
 
Plasmid preparation: Cloning and acquisition of the genetic constructs
Magainin 1 Star Peptide
This construct was synthesised in the standard shipping plasmid from Life Technologies- plasmid pMK. We had originally planned to express our protein to the periplasm of E. coli, and therefore had included in the synthesis a periplasmic export tag, the TorTss signal sequence (Steiner et al., 2006).
Cytoplasmic and periplasmic expression may both allow for disulfide bond formation in the cell. We decided, however, to focus on cytoplasmic expression. There are distinct advantages to cytoplasmic expression (e.g. the absence of several periplasmic proteases and potentially higher expression levels (Baneyx, 1999)).
Another reason the cytoplasm was chosen was to allow us to test the effects of an oxidising versus reducing intracellular environment on disulfide bond formation. We planned to express the construct in both SHuffle T7 cells (oxidising cytoplasm) and BL21(DE3) (reducing cytoplasm) to probe whether there was a difference in disulfide bond formation. Therefore, we needed to remove the periplasmic export tag from the gene.
To do this, we use the following cloning strategy (noting that the top row represents the gene initially synthesised in plasmid pMK):
The gene was inserted into plasmid pSB1C3 containing the T7 promoter and ribosome binding site, BBa_K525998. It was inserted after the promoter and before the BioBrick suffix. This was accomplished by digesting the destination vector with SpeI and PstI. At the same time, PCR was used to amplify the segment of the gene containing the SUMO fusion and the Magainin 1 Star peptide, adding XbaI and keeping the PstI site existing in the gene.
Digestion of the PCR product then allowed for ligation of the insert and destination vector using the PstI sites and XbaI and SpeI (XbaI and SpeI have compatible sticky ends). Note that after the ligation, there will be a scar in the gene where the XbaI and the SpeI sites were ligated.
The ligation appeared to be successful. The ligation mixture was transformed to DH5α competent cells and plated onto chloramphenicol-containing agar plates.
Plasmid from 5 colonies (C1, C2, C3, C4, and C5) was cultured, extracted, and then digested with EcoRI and PstI.
As evident in the figure below, at least 4 of the colonies appeared to contain the insert. The empty, linearised pSB1C3 backbone ran at approximately the correct size (2070 bps), as did the insert (740 bps).
N.B. MW marker is the 100 bp Ladder from Axygen. * indicates the colony picked for sequencing and eventual transformation to the expression cell lines.
The DNA from Colony C2 was confirmed using Sanger sequencing at the Australian Genome Research Facility. This DNA appears in the registry of standard parts as BioBrick BBa_K1394000 .
For the USP peptide and the linear Magainin 1 peptide, the genes were synthesised by GenScript and delivered in pSB1C3. The expression vectors had identical gene regulatory elements to that used for the Magainin 1 Star peptide. They only differed in the codon optimisation used, and they also lacked the assembly scar described above.
Protein expression and characterisation
After acquiring our genes, the project moved to protein expression and purification. We expressed both star peptides (Magainin 1 Star Peptide and the USP peptide) in both the SHuffle T7 and BL21(DE3) cell lines, both sourced from New England Biolabs. As the Linear Magainin 1 peptide does not have any special disulfide bonding requirements, we expressed it in BL21(DE3).
The plasmids were transformed to the expression cells, and a single colony was cultured and induced overnight at 17 °C. A whole cell sample both before IPTG induction (-IPTG) and after the induction period (+IPTG) were boiled in SDS-PAGE sample buffer and loaded on a 15% tris-glycine gel. The whole cell Coomassie stained gel is shown below alongside the NEB P7712S molecular weight marker.
From theoretical prediction of the peptide masses, we would expect the following distribution of molecular weights:
Magainin 1 Star Peptide (Mag1 Star)
USP
Linear Magainin 1 (Linear Mag1)
22.15 kDa
29.3 kDa
14.32 kDa
We looked for bands corresponding to overexpression of the proteins. We observed a thick band, post IPTG around 32 kDa in the BL21(DE3) cells carrying the USP plasmid. However, we saw this same band appearing in the Linear Magainin 1 lane but the Linear Magainin 1 protein should have a much lower molecular weight than what was observed. Therefore, we did not assume that we had overexpression overexpression of the USP 1 protein.
We also noted bands in all post-IPTG lanes around 17 kDa. This would be roughly consistent with both the Linear Magainin 1 and Magainin 1 Star Peptide, noting that small proteins may not run at their expected molecular weight. Again, the analysis is complicated by the fact that the band also appears in the lane corresponding to the larger molecular weight protein, USP I. Given this ambiguity, we decided to purify all the protein in the sample using Ni-NTA purification.
Purification and further characterisation
A small-scale purification was carried out according to our protocol . The cells were lysed with iodoacetamide, an alkylating agent, added to the lysis buffer. Iodoacetamide blocks all free cysteines on proteins with a short alkyl group. This was added because we wanted to determine if a disulphide bond had formed inside the cell. If we did not block free cysteines, then any bond that had formed could be attributed to spontaneous/air oxidation of disulfides outside the cell.
After purification, we ran a concurrent Western Blot and Coomassie stain on both the pre-induction samples from above and the purified protein (namely, the first elution from the batch purification).
The Western Blot used mouse monoclonal antibodies against the N-terminal HIS-tag as primary antibodies and anti-mouse secondary antibodies:
The corresponding Coomassie stain is show below:
There are several interesting aspects of the Western Blot. Firstly, there was a strong band between 25 and 32 kDa in most lanes. However, it appears in all of the pre-induction controls, suggesting non-specific binding of the anti-HIS antibodies.
Secondly, we observed a band in most of the lanes around 17 kDa (with the exception being the USP protein in SHuffle cells). As these bands were not present in the pre-induction controls, we suspected they were a result of the induction.
Mass spectrometry
In order to assess the identity of the bands, we performed an in-gel tryptic digestion on select bands. We digested bands in the Coomassie stain which corresponded to the HIS-tagged bands in the Western Blot. This was followed by mass spec analysis (LC MS/MS). We focused our analysis on the Magainin 1 Star Peptide bands and the Linear Magainin 1 band. Our USP peptide was, by design, highly rich in basic residues, greatly reducing the likelihood that tryptic fragments would be detected by the mass spec.
We found that the Linear Magainin 1 peptide appeared to be present in the approximately 17 kDa band, with the following detected tryptic fragments in the sequence below (bold; basic residues underlined):
Note that spaces have been added in the sequence to emphasise distinct domains in the protein (in this case, the fusion protein versus the Magainin 1 peptide). Together with the fact that the protein runs close to the expected molecular weight, this seems to provide good evidence that the protein is being expressed.
We then examined the bands corresponding to the Magainin 1 Star Peptide.
Again, we digested bands in the Coomassie stained gel corresponding to the prominent HIS-tagged bands on the Western Blot (near 17 kDa).
The mass spec coverage for the Magainin 1 Star Peptide expressed in SHuffle cells was as follows:
The coverage for the same peptide expressed in BL21(DE3) was found to be:
Again, we can see that tryptic peptides from the expressed protein appear to present in the gel band under analysis.
There is a relatively lower sequence coverage. This could be a limitation of our procedure: for example, improper destaining during the in gel digestion procedure can inhibit tryptic digestion.
However, even if the digestion was complete, the mass spec will only detect fragments which ionize well. The Magainin 1 Star peptide consists of four repeats of the Magainin 1 sequence (corresponding to the arms of the star). If the two tryptic fragments within Magainin 1 do not ionize well, then indeed the entire peptide would not be read.
Finally, it is possible that the protein has been cleaved or degraded. This would account for the slightly lower-than-expected mass on the SDS page gel.
Nonetheless, it is a possibility that the peptide fragments are present but not detected. Replication of the in-gel mass spec would be useful in clarifying this point, or provided the purity of the sample is acceptable, it may be possible to run an intact mass spec. Digestion with alternative enzymes may also provide a more robust sequence coverage, particularly for the USP peptide.
Conclusions
We designed several novel star peptides based on several rational design criteria. Further, we began the optimisation process required to express these peptides in E. coli.
To this end, we constructed an expression system based on the SUMO protein and standard BioBrick parts. The function of this system was confirmed, as it appears it does produce HIS-tagged SUMO fusion protein in E. coli, as expected. Nonetheless, we hope to continue to characterise our parts, likely through further purification and measurement of the mass.
Given the success of SUMO in the protein expression community, we hope our BioBrick will encourage further use of the fusion domain within iGEM. In addition, we hope that our ideas and concepts will inspire future iGEM efforts to explore the applications of synthetic biology for material science. We continue to believe that bacteria have great promise for the in vivo construction of unique biomaterials, and we look forward to seeing how the synthetic biology community will develop in this area.
References
ADER, C., FREY, S., MAAS, W., SCHMIDT, H. B., GÖRLICH, D. & BALDUS, M. 2010. Amyloid-like interactions within nucleoporin FG hydrogels. Proceedings of the National Academy of Sciences, 107, 6281-6285.
BANEYX, F. 1999. Recombinant protein expression in Escherichia coli. Current Opinion in Biotechnology, 10, 411-421.
BOMMARIUS, B., JENSSEN, H., ELLIOTT, M., KINDRACHUK, J., PASUPULETI, M., GIEREN, H., JAEGER, K. E., HANCOCK, R. E. W. & KALMAN, D. 2010. Cost-effective expression and purification of antimicrobial and host defense peptides in Escherichia coli. Peptides, 31, 1957-1965.
BROGDEN, K. A. 2005. Antimicrobial peptides: Pore formers or metabolic inhibitors in bacteria? Nature Reviews Microbiology, 3, 238-250.
CHANG, C., CHHOR, G., CLANCY, S. & JOACHIMIAK, A. 2014. Crystal Structure of Glutathione S-transferase Domain Protein From Haliangium Ochraceum DSM 14365 [Online]. ''NCBI. Available: http://www.ncbi.nlm.nih.gov/Structure/mmdb/mmdbsrv.cgi?uid=4w66' [Accessed September 9 2014].
CHEN, X., ZHU, F., CAO, Y. & QIAO, S. 2009. Novel expression vector for secretion of cecropin AD in Bacillus subtilis with enhanced antimicrobial activity. Antimicrobial agents and chemotherapy, 53, 3683-3689.
GLINEL, K., JONAS, A. M., JOUENNE, T., LEPRINCE, J., GALAS, L. & HUCK, W. T. 2008. Antibacterial and antifouling polymer brushes incorporating antimicrobial peptide. Bioconjug Chem, 20, 71-77.
HUMBLOT, V., YALA, J.-F., THEBAULT, P., BOUKERMA, K., HÉQUET, A., BERJEAUD, J.-M. & PRADIER, C.-M. 2009. The antibacterial activity of Magainin I immobilized onto mixed thiols self-assembled monolayers. Biomaterials, 30, 3503-3512.
DAWSON, P. E., MUIR, T. W., CLARK-LEWIS, I. & KENT, S. 1994. Synthesis of proteins by native chemical ligation. Science, 266, 776-779.
KADOKURA, H. & BECKWITH, J. 2009. Detecting folding intermediates of a protein as it passes through the bacterial translocation channel. Cell, 138, 1164-1173.
KADOKURA, H., KATZEN, F. & BECKWITH, J. 2003. Protein disulfide bond formation in prokaryotes. Annual review of biochemistry, 72, 111-135.
LI, Y. 2011. Recombinant production of antimicrobial peptides in< i> Escherichia coli</i>: A review. Protein Expression and Purification, 80, 260-267.
LOBSTEIN, J., EMRICH, C. A., JEANS, C., FAULKNER, M., RIGGS, P. & BERKMEN, M. 2012. SHuffle, a novel Escherichia coli protein expression strain capable of correctly folding disulfide bonded proteins in its cytoplasm. Microb Cell Fact, 11, 56-56.
PROTEIN MODEL PORTAL. Query result, GST and DcmA [Online]. Available: http://www.proteinmodelportal.org/query/uniprot/P21161 [Accessed September 9 2014].
STEINER, D., FORRER, P., STUMPP, M. T. & PLUCKTHUN, A. 2006. Signal sequences directing cotranslational translocation expand the range of proteins amenable to phage display. Nat Biotech, 24, 823-831.
SULISTIO, A., GURR, P. A., BLENCOWE, A. & QIAO, G. G. 2012. Peptide-Based Star Polymers: The Rising Star in Functional Polymers. Australian Journal of Chemistry, 65, 978-984.
SULISTIO, A., LOWENTHAL, J., BLENCOWE, A., BONGIOVANNI, M. N., ONG, L., GRAS, S. L., ZHANG, X. & QIAO, G. G. 2011. Folic Acid Conjugated Amino Acid-Based Star Polymers for Active Targeting of Cancer Cells. Biomacromolecules, 12, 3469-3477.
TANAKA, N., KUSAKABE, Y., ITO, K., YOSHIMOTO, T. & NAKAMURA, K. T. 2002. Crystal Structure of Formaldehyde Dehydrogenase from< i> Pseudomonas putida: the Structural Origin of the Tightly Bound Cofactor in Nicotinoprotein Dehydrogenases. Journal of molecular biology, 324, 519-533.
WIRADHARMA, N., LIU, S. Q. & YANG, Y. Y. 2012. Branched and 4-Arm Starlike α-Helical Peptide Structures with Enhanced Antimicrobial Potency and Selectivity. Small, 8, 362-366.
YU, Z., WANG, Q., MA, Q. & ZHANG, R. 2013. Secretory expression of lacticin Q fused with SUMO in Bacillus subtilis. Protein Expression and Purification, 89, 51-55.
ZASLOFF, M. 1987. Magainins, a class of antimicrobial peptides from Xenopus skin: isolation, characterization of two active forms, and partial cDNA sequence of a precursor. Proceedings of the National Academy of Sciences, 84, 5449-5453.
 
Collaboration with Oxford
To strengthen the sense of iGEM community, we contacted University of Oxford iGEM team to discuss the possibility of collaboration between our teams. The Oxford project aims to develop a system that can dispose of the carcinogenic, hazardous solvent dichloromethane (DCM). To do this, Oxford team has proposed the use of the DcmA enzyme. This enzyme degrades DCM to form a toxic intermediate which in turn is converted by another enzyme, FdhA, into a neutral molecule. Schematically this can be presented in the following way:
Reaction 1: DCM + DcmA -> toxic intermediate
Reaction 2: toxic intermediate + FdhA -> neutral product.
The problem for their system lies in bringing the two enzymes together to ensure efficient reaction kinetics. An idea that we discussed with Oxford was to use the star peptide platform to link the two enzymes together, in a structure similar to the following:
We worked with Oxford to study this system from a theoretical standpoint. Our team studied the 3D structure of both enzymes involved to confirm that the enzyme could in theory be attached to a linker in this manner, while Oxford team did stochastic modelling to determine how reaction rate changes when the linker length, labeled D on the diagram, changes.
As a first check, it was necessary to determine whether anchoring these enzymes to our star would be sterically possible. We studied the structures of FdhA and DcmA determined by crystallography using data from the protein data bank. As no crystallographic data was available in the databank for DcmA, GST was examined as a proxy, as GST is structurally homologous to DcmA.
The crystalographically resolved structure of FdhA enzyme (Tanaka et al., 2002) from the protein data bank revealed the following image:
The crystalographically resolved structure of GST was as follows (Chang et al., 2014):
It can be seen that the amino- and carboxy-terminals are located away from the active site. This suggests that both enzymes, DcmA and FdhA, might be linked together via linkers that are used in our star peptide. That is, the active site will not be sterically hindered by attachment to the star peptide. That said, it is difficult to predict how anchoring the protein to the star will affect its tertiary structure. Further, it is difficult to predict how limiting the rotational degrees of freedom will affect enzyme activity.
The Oxford team modeled this system and found that, indeed, the star peptide as an enzyme linker was favourable to enzyme kinetics. Their work can be found here .
Retrieved from " http://2014.igem.org/Team:Melbourne/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
