<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Melbourne_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Melbourne/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Melbourne/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Melbourne/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Melbourne/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Melbourne/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Melbourne/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TABLE width="100%" style="text-transform: uppercase; font-family: 'Lato', sans-serif;"><TBODY><TR height="10px"><TD align="center" height="30px" onmouseover="this.bgColor='#d3d3d3'" onmouseout="this.bgColor='#e7e7e7'" bgcolor="#e7e7e7"><A href="https://2014.igem.org/Team:Melbourne" style="color:#000000">
Home</A></TD><TD align="center" height="30px" onmouseover="this.bgColor='#d3d3d3'" onmouseout="this.bgColor='#e7e7e7'" bgcolor="#e7e7e7"><A href="https://2014.igem.org/Team:Melbourne/Team" style="color:#000000">
Team</A></TD><TD align="center" height="30px" onmouseover="this.bgColor='#d3d3d3'" onmouseout="this.bgColor='#e7e7e7'" bgcolor="#e7e7e7"><A href="https://2014.igem.org/Team:Melbourne/Project" style="color:#000000">
Project</A></TD><TD align="center" height="30px" onmouseover="this.bgColor='#d3d3d3'" onmouseout="this.bgColor='#e7e7e7'" bgcolor="#e7e7e7"><A href="https://2014.igem.org/Team:Melbourne/Human_Practices" style="color:#000000"> 
Human Practices</A></TD><TD align="center" height="30px" onmouseover="this.bgColor='#d3d3d3'" onmouseout="this.bgColor='#e7e7e7'" bgcolor="#e7e7e7"><A href="https://2014.igem.org/Team:Melbourne/Achievements" style="color:#000000">
Achievements</A></TD><TD align="center" height="30px" onmouseover="this.bgColor='#d3d3d3'" onmouseout="this.bgColor='#e7e7e7'" bgcolor="#e7e7e7"><A href="https://2014.igem.org/Team:Melbourne/Notebook" style="color:#000000">
Notebook</A></TD><TD align="center" height="30px" onmouseover="this.bgColor='#d3d3d3'" onmouseout="this.bgColor='#e7e7e7'" bgcolor="#e7e7e7"><A href="https://2014.igem.org/Team:Melbourne/Protocols" style=" color:#000000"> 
Protocols</A></TD><TD align="center" height="30px" onmouseover="this.bgColor='#d3d3d3'" onmouseout="this.bgColor='#e7e7e7'" bgcolor="#e7e7e7"><A href="https://2014.igem.org/Team:Melbourne/Safety" style="color:#000000">
Safety</A></TD><TD align="center" height="30px" onmouseover="this.bgColor='#d3d3d3'" onmouseout="this.bgColor='#e7e7e7'" bgcolor="#e7e7e7"><A href="https://2014.igem.org/Team:Melbourne/Sponsors" style="color:#000000">
Sponsors</A></TD><TD align="center" height="30px" onmouseover="this.bgColor='#d3d3d3'" onmouseout="this.bgColor='#e7e7e7'" bgcolor="#e7e7e7"><A href="https://2014.igem.org/Team:Melbourne/Attributions" style="color:#000000">
Attributions</A></TD></TR></TBODY></TABLE></P><H1>Notebook</H1><H2>Semester 1</H2><H3>Preliminary  work – Lab setup, training, etc.</H3><P><EM>In Semester 1, we began the process of  recruiting the team and setting up the lab. Because our lab was new, we needed  to order all supplies, secure scientific sponsorships, and establish protocols.  The founding team was also able to do some exploratory lab work before the  changeover to the full team later in the semester.</EM><EM>Lab members were recruited from the wider  University of Melbourne student community through an advertising campaign,  followed by a selection process. The team underwent training during the Easter  break and the following weeks.</EM></P><P> </P><H3>Week beginning 6 January</H3><P>Grew stock of competent DH5α E. coli. Cells stored at -80°C.</P><P> </P><H3>Week beginning 13 January</H3><P>Note that the 2013 iGEM Founding Team originally came up with the concept for a star peptide similar to the USP peptide reported on the Project page. This peptide was meant to be unstructured. It contained an RGD motif to support potential binding of the peptide to integrin (hence why it was termed the RGD peptide). Unlike the peptides reported on the project page, the RGD contained a TEV protease recognition sequence. This would have allowed it to be co-expressed with the TEV protease, and therefore cleaved to form a star peptide inside the cell. 
</P><P>The RGD peptide sequence was: MRV LLF LLL SLF MLP AFS RGD RGD GGG AKQ RGG C GGR QKA GGG RGD RGD EQLYFQG RGD RGD GGG AKQ RGG C GGR QKA GGG RGD RGD HHHHHH

</P><P>The founding team ordered this peptide's gene to be synthesised and most of the early work in the lab was focused on using it to practice lab techniques. At this stage, we were waiting for the gene to arrive.</P><H3>Week beginning 20 January</H3><P><STRONG> (20-01-14)</STRONG>
  RGD construct DNA arrived.
  Completed transformation of DH5α competent cells with heat  shock and LB (see Protocols).
  OD of competent cell growth:
</P>
  Digestion was performed using EcoRI (see Protocols).
  Amount of plasmid was measured.
  RGD001 – 37.1 ug/uL
  RGD002 – 51.2 ug/uL
  RGD003 – 34.5 uG/uL
  Gel was run. No DNA was visible but the ladder was fine.<STRONG>(24-01-14)</STRONG><UL><LI>Redid miniprep. Quantified RGD plasmid again.</LI></UL><P>RGD001 – 37.1 ug/uL
  RGD002 – 51.2 ug/uL
  RGD003 – 34.5  uG/uL</P><UL><LI>Performed double digest of RGD plasmid with  EcoRI and PstI (see Protocols).</LI><LI>Performed gel electrophoresis of digested  plasmid and undigested plasmid. Plasmids were present at approx 3.5 kb. RGD001  seemed to be the better digested plasmid and best to use for transformation.</LI></UL><P> </P><H3>Week beginning 27 January</H3><UL><LI>Transformed BL21 with RGD. Placed in incubator  overnight at 37°C.</LI><LI>Ran third gel of transformed DH5α (with RGD).</LI><LI>Performed double digests before running another  gel to check success.</LI></UL><P> </P><H3>Week beginning 3 February</H3><P>Performed protein induction on cells containing RGD plasmid.
  NOTE: </P><UL><LI>Grew 800 mL culture and measured OD (absorbance  at 600nm):</LI></UL><P>0 hrs – 0.184
  1 hr – 0.322
  2 hrs – 1.265
  2.5 hrs –  1.471</P><UL><LI>Despite overshooting OD target, went ahead with  IPTG induction anyway.</LI></UL><H3> </H3><H3>Week beginning 10 February</H3><P><STRONG>Project  design/planning</STRONG>
  Did some project planning based on meeting with advisor Neil  O’Brien-Simpson:<U>Selenocysteine</U>
  First thought when he saw the project was the liability of  the disulfide bond (i.e. tends to fall apart under producing environments).
  He suggested trying selenocysteine, because selenocysteine  bonds are stronger and the amino acids should be sitting around in E. coli  ready to use.<U>Comments on AMP</U>
  Suggested a good idea to stick with the Wiradharma, 2012  structure rather than alter too much so as to make comparisons easier.
  We briefly discussed the mechanism of action of AMPs. As we  know, there are several models, including the barrel stave, toroidal poor, and  carpet models. In addition, peptide aggregation at the surface of the bacterial  cell can lead to membrane thinning, which facilitates pore formation.
  Neil wasn’t sure how the star polymers would work in  particular. They may work using the above mechanisms… The disulfide bond may  also be broken as the stars enter the cell cytoplasm.<U>Related work</U>
A researcher by the name of Tam has developed a Multiple  Antigenic Peptide (MAP) similar to ours. This peptide includes branched lysine  residues. However, the problem with this molecule is that it is very  challenging to synthetically produce.</P><H3> </H3><H3>Week beginning 17 February</H3><UL><LI>Began new protein expression method (see  Protocols). </LI><LI>Pelleted cells (see Protocols).</LI></UL><H3> </H3><H3>Week beginning 24 February</H3><UL><LI>Transformed DH5α Kan.R resistance (see  Protocols). Placed 4 plates on incubator overnight.</LI><LI>Transferred cells to 4°C. Colonies grew,  suggesting the transformation was successful.</LI></UL><H3> </H3><H3>Week beginning 3 March</H3><P><STRONG>(04-03-14)</STRONG></P><UL><LI>To check success of transformation, we ran an SDS-PAGE  gel at 150V, 400mA, 45 mins with positive controls and RGD protein (see  Protocols):</LI></UL><UL><LI>Transformed DH5α colonies with Magainin I.</LI><LI>DH5α (MagI) transformed cells were miniprepped  (see Protocols). Some of cell pellet was loaded into SDS-PAGE gel with two  positive controls for His-tags. Results appeared successful so we continued  onto Ni-NTA column purification.</LI><LI>Unfortunately, Ni-NTA column purification  failed.</LI></UL><H3> </H3><H3>Week beginning 10 March</H3><P>Redid process from last week to try to diagnose what went  wrong.
  Double digestion of DH5α (Magainin I) colony miniprep  products with EcoRI and PstII.
  Made and ran DNA gel with double digest at 120V, 100mA, 35  minutes then 135V, 100mA, 15 minutes then 120V 3 minutes.</P>
  Results were visible for lanes 1 – 4, but digests were not  visible.<H3> </H3><H3>Week beginning 17 March</H3><P><STRONG><EM>(Recruitment of team members)</EM></STRONG></P><H3>Week beginning 24 March</H3><P><STRONG><EM>(Recruitment of team members)</EM></STRONG></P><H3>Week beginning 31 March</H3><P><STRONG><EM>(Recruitment of team members)</EM></STRONG></P><H3>Week beginning 7 April</H3><P><STRONG><EM>(Recruitment of team members)</EM></STRONG></P><H3>Week beginning 14 April</H3><P><STRONG><EM>(Recruitment of team members)</EM></STRONG></P><H3>Week beginning 21 April</H3><P><STRONG><EM>(Lab introductory training)</EM></STRONG></P><H3>Week beginning 28 April</H3><P><STRONG><EM>(Lab setup work)</EM></STRONG></P><H3> </H3><H3>Week beginning 5 May</H3><P>Despite negative results on gel of double digest (thought to  be due to faulty gel electrophoresis process), we went ahead with protein  expression.
  We also transformed  BL21 with pMK-RGD vector (see Protocols) and ran a protein gel with pGEX, pET,  pMK RGD and pMK Mag 1 (samples taken from the training week during the break).
Conducted  miniprep of DH5α pMK-RGD culture and ran DNA gel on the RGD plasmid extracted. </P><H3> </H3><H3>Week beginning 12 May</H3><P>Started protein expression on transformed BL21 pMK-RGD.</P><H3> </H3><H3>Week beginning 19 May</H3><P>Completed protein expression, but cultures failed to grow.
Ran PCR for pMK RGD (reaction volume = 50 uL) with the following new recipe:</P><UL><LI>10X Thermopod Reaction Buffer – 5 uL</LI><LI>10 mM dNTP mixture – 1 uL</LI><LI>10 uM forward primer – 1 uL</LI><LI>10 uM reverse primer – 1 uL</LI><LI>Template DNA – 1 uL</LI><LI>Taq DNA polymerase – 0.5 uL</LI><LI>Nuclease-free h3O – 40.5 uL</LI></UL><P>Conditions for the PCR cycle:</P><UL><LI>95°C – 2 minutes</LI><LI>30 cycles:</LI><UL><LI>95°C – 20 seconds</LI><LI>60°C – 20 seconds</LI><LI>72°C – 20 seconds</LI></UL><LI>72°C – 5 minutes</LI><LI>4°C – infinity (hold) </LI></UL><P>Performed  protein induction on PMK-MAG1 .</P><P>Created a  glycerol stock of BL21-RGD and BL21-MAG1 and stored it at -86°C.</P><H3> </H3><H3>Week beginning 26 May</H3><P><STRONG><EM>(Exam period – no lab preparation work)</EM></STRONG></P><H3> </H3><H3>Week beginning 2 June</H3><P><STRONG><EM>(Exam period – no lab preparation work)</EM></STRONG></P><H3> </H3><H3>Week beginning 9 June</H3><P><STRONG><EM>(Exam period – no lab preparation work)</EM></STRONG></P><H3> </H3><H3>Week beginning 16 June</H3><P><STRONG><EM>(Exam period – no lab preparation work)</EM></STRONG></P><P> </P><P> </P><H2>Semester 2 </H2><P><EM>Main project  work begins.</EM><EM>With the lab set up, the team fully  formed, and preliminary work completed, we began work on the project. ­Our  focus throughout Semester 2 was on the design of the BioBricks, and ensuring  that each of our proteins could be properly expressed in transformed bacteria  after cloning and subcloning.</EM></P><H3> </H3><H3>Week 1 – 23 June</H3><P><STRONG>(24-06-14)</STRONG></P><UL><LI>Plated  BL21(DE3) + Magainin 1 glycerol stock on one plate using ‘spread’ technique at  12:45pm, placed in 37 degree incubator at 3pm (should be taken out at 11am next  day [+20 hours]).</LI></UL><P>However,  the plates had way too much  growth, almost resembling a lawn, and couldn’t be used due to a need for  ‘isolated’ colonies. After checking lab-book again, realised the glycerol stock  was made using culture from already plated colonies and instead of just  transformed cells. 
  New glycerol stock was used to plate again, this time using the  ‘streak’ technique.</P><UL><LI>SDS-PAGE gel  with MagI pellet was run. 2 samples were prepared using approximately 100μL of  SDS-PAGE Sample Buffer and a small amount of pellet. The samples were prepared  using Ash’s method of quickly boiling the samples (placing tubes in heat block  set at 100(99.9) degrees) to lyse the cells. Unfortunately, the samples  thickened and SDS-PAGE could not be run. This may be a result of specific faults  in the buffer recipe, so possibly use a RIPA buffer next time.</LI></UL><H3> </H3><H3>Week 2 – 30 June</H3><P><STRONG>Project design/planning</STRONG>
  The team agreed to have a multi-track research program. What  we are essentially doing at the moment is trying to attack this problem of  synthesizing disulfide bonded structures from multiple fronts.
Tobias  suggested a new concept for ligating peptides, carbohydrates, lipids and other  biomolecules to the star peptide structure through native chemical ligation.</P><H3> </H3><H3>Week 3– 7 July</H3><P><STRONG>Project  design/planning</STRONG>
  Further work on the idea of native chemical ligation (NCL).
Summary of progress on shuttling the MAG 1 insert with the  periplasmic export tag to the standard iGEM plasmid – </P><UL><LI>Ran PCR on pMK-RGD using primers to make RGD  compatible with pGEX vector</LI><LI>Ran PCR product on gel</LI><LI>Cut out PCR product from gel and use gel  purification kit</LI><LI>Ran restriction digestion on PCR insert product  of 3a using BamHI and EcoRI</LI><LI>Ran PCR purification using Thermo kit on product  of 4a</LI><LI>Transformed pGEX vector donated by a neighboring  lab into DH5a, plate and incubated overnight</LI><LI>Miniprepped plasmid</LI><LI>Ran restriction digestion on plasmid product of  2b using BamHI and EcoRI</LI></UL><P>Summary of previous progress on inserting RGD coding region  into pGEX vector, and expression in BL21(DE3) cells – </P><UL><LI>Ran PCR on pMK-RGD using primers to make RGD  compatible with pGEX vector</LI><LI>Ran PCR product on gel</LI><LI>Cut out PCR product from gel and use gel  purification kit</LI><LI>Ran restriction digestion on PCR insert product  of 3a using BamHI and EcoRI</LI><LI>Ran PCR purification using Thermo kit on product  of 4a</LI><LI>Transformed pGEX vector donated by a neighboring  lab into DH5a, plate and incubated overnight</LI><LI>Miniprepped plasmid</LI><LI>Ran restriction digestion on plasmid product of  2b using BamHI and EcoRI</LI></UL><P>Decided to divide work into streams as previously planned –  indicated in lab book as:</P><UL><LI>pSB1C3-MAG1 subcloning</LI><LI>pGEX-RGD cloning</LI><LI>Cytoplasmic Star Magainin 1 cloning</LI></UL><H3> </H3><H3>Week 4 – 14 July </H3><P><STRONG>pSB1C3-MAG1  subcloning – </STRONG></P><UL><LI>Ran PCR on pMK-MAG1 using primers to remove  TorTSS and add sticky ends for pSB (BBa_K525998). When run on a gel, the PCR  product was a very clear band, indicating success.</LI><LI>Ran restriction digestion on pMK-MAG1 PCR insert  product using SpeI and PstI. </LI><LI>Ran PCR on MAG1 using primers to remove TorTSS  and add sticky ends for pSB (BBa_K525998) for a second time.</LI></UL><P><STRONG>Cytoplasmic Star  Magainin 1 cloning –</STRONG> Plated DH5a cells.</P><H3> </H3><H3>Week 5 – 21 July</H3><P><STRONG>pSB1C3-MAG1  subcloning – </STRONG>
  We transformed DH5a with the pSB1C3 (RFP) plasmid provided  by iGEM. We then miniprepped the cells. However, the nanodrop suggested that  the concentration of the cells was extremely low. Gayle then ligated pSB1C3  with a MAG 1 insert. Because the concentrations were too low to see on a DNA  gel, she re-transformed the plasmid to DH5a. The idea is that we should now be  able to re-miniprep this DH5a and reclaim the ligated plasmid.<STRONG>Cytoplasmic Star  Magainin 1 cloning – </STRONG>
  Did restriction digestion using PstI and SpeI on miniprepped  T7 promoter containing pSB1C3. Ran the product on a gel, but it came up blank.  We had a thorough discussion with Ash and David about why this could be. Ash  suggested trying digesting with one enzyme at a time, having several lanes,  each with different enzyme combinations. 
  A Google search suggested that there could be a problem with  the miniprep procedure. Specifically, there could be some enzymes being left  over from the miniprep which, when the samples were heated during the  restriction digestion, ate up the DNA.
Therefore, Stan designed an experiment to test this. We had  5 conditions: </P><UL><LI>the PSB1C3 plasmid directly after the miniprep,  with no incubation or restriction digestion; </LI><LI>the plasmid heated to 37° with restriction  enzymes, </LI><LI>the plasmid heated to 37° WITHOUT restriction  enzymes, </LI><LI>a PCR insert meant to go into the plasmid heated  to 37° with restriction enzymes (the same restriction enzymes which would allow  us to like it that insert into the pSB1C3 plasmid),</LI><LI>the PCR insert heated to 37° without restriction  enzymes.</LI></UL><P>So, we're testing the effect of heat and the effect of the  restriction enzymes.
  In the resulting gel, every lane was visible, except for the  plasmid which had been double digested. This suggests that there was not  anything wrong with the restriction enzymes per se, neither with the buffer nor  the heating process. It argues against the hypothesis that there was some DNA  degrading enzyme present after the miniprep could have been activated by heat.</P><H3> </H3><H3>Week 6 – 28 July</H3><P><STRONG>Cytoplasmic Star  Magainin 1 cloning – </STRONG>
  Thawed out the plasmid samples in sample buffer from  yesterday and loaded them onto a 16 well agarose gel. Every lane was visible  except for the Axygen miniprepped plasmid treated with SpeI. However, the  Axygen miniprepped plasmid treated with a double digestion of SpeI and PstI DID  show upon the gel. After speaking with David, he suspects that we perhaps  neglected to put DNA into that well. 
  Note that these results are in direct contrast to the gel  described earlier, where we had an identical plasmid (pSB1C3 containing the P7  promoter), picked from the same plate, using the same miniprep kit, same  restriction enzymes (PstI and SpeI), for the same incubation period (2.5-3  hrs). That is, we did exactly the same thing, and this time, it showed up on  the gel, whereas before it disappeared.
  Given that the double digest with SpeI and PstI and EcoR1  and PstI worked, we went ahead and began culturing in 10 mL colonies from the  same pSB1C3 plate used in the experiment above. We made 6, 10 mL cultures. To  provide a backup, we also picked three colonies which had previously been  plated out by Lumi containing GFP, RFP, or YFP, creating 3, 10 mL cultures  which may also be miniprepped.
Further progress:</P><UL><LI>Miniprepped 10 mL cultures of pSB containing the  p7 promoter. Ran various restriction digests on the miniprep product in  preparation for ligation of pSB with our MAG1 PCR insert.</LI><LI>Ran the restriction digests from yesterday.  Results are very mysterious. Thus, re-ran  restriction digests. Results favourably suggest one of the minipreps was  successful.</LI><LI>Gel extracted pSB1C3 digested with both SpeI and  PstI.</LI></UL><H3> </H3><H3>Week 7– 4 August</H3><P><STRONG>pSB1C3-MAG1  subcloning – </STRONG>Transformation of pSB1C3 to DH5α (see Protocols).<STRONG>pGEX-RGD cloning – </STRONG>Transformation  of pGEX to DH5α (see Protocols).<STRONG>Cytoplasmic Star  Magainin 1 cloning – </STRONG>
  We were originally planning to do the ligation reaction with  the SpeI and PstI restriction enzymes AND the PCR insert. However, during that  restriction digest of the PCR insert, DNA loading dye was erroneously added. We  attempted to gel purify the PCR insert to remove the loading dye. However, for  some reason the &quot;PCR insert&quot; ran at a very high molecular weight  (approximately 2 kDa). Moreover, there is a strange double band.
Because the PCR insert appears to be screwed up, we will  have to start from the PCR step again. Planning on ordering new primers which  will allow the Magainin 1 insert to be biobrick compatible.</P><H3> </H3><H3>Week 8 – 11 August</H3><P><STRONG>pSB1C3-MAG1  subcloning –</STRONG></P><UL><LI>PLASMID: </LI><UL><LI>Stanislav performed a double restriction digest on the  linearized plasmid backbone pSB1C3. </LI><LI>Performed a miniprep on the 10 mL cultures.  Confirmed the success of the miniprep/presence of the plasmid using a DNA gel.</LI></UL><LI>INSERT: </LI><UL><LI>Performed a restriction digestion on previous pMK-MAG1  plasmid stock in order to liberate the MAG 1 insert in preparation for ligation  with pSB1C3 and ran a gel on the products. In  this gel, a distinct band was observed corresponding to the approximately 2 kDa  vector (pMK). However, the 6 kDa insert was not visible. </LI><LI>In order to get more of the insert, we propagated  more pMK-MAG1 plasmid. </LI></UL></UL><P><STRONG>pGEX-RGD cloning – </STRONG></P><UL><LI>Transformed DH5a competent cells with pGEX-6P3 and  performed a miniprep to increase the amount of plasmid stock available  for ligation. A DNA gel performed on the products suggested the miniprep was  successful. </LI><LI>A gel performed on PCR products of the RGD insert  amplified with restrictions sites added for subcloning into pGEX gave odd  results. </LI></UL><P>Both the PCR products created yesterday and the  products done over the summer break showed over 10 bands in each lane. This suggests  that there was nonspecific binding, leading to a variety of different molecular  weight products. Spoke to advisor Ash who prescribed a new PCR  temperature/timing cycle to minimise this problem.</P><P>Ran the PCR reaction (using Ash’s  suggested solution) product on a gel. Increasing the annealing temperature in  the PCR cycle did decrease the presence of multiple (~10)  bands. However, it also led to a general  reduction in the intensity of the PCR product's signal on the gel. It's not  clear that the changes to the PCR protocol actually reduced nonspecific binding  so much as just decreased the reaction yield.</P><P>As suggested by our supervisor,  Cheng, we reran the PCR reaction. This time, in order to eliminate the possibility  of nonspecific binding, we cut out the RGD insert from the PMK vector using  restriction digestion, gel extract the insert, and then used this insert as the  PCR template.</P><P><STRONG>Cytoplasmic Star  Magainin 1 cloning – </STRONG>
  We had previously ordered the primers which would both  remove the periplasmic export tag and allow for ligation into Biobrick  BBa_K525998 of our Magainin 1 Star Peptide insert. The problem with these  primers is, however, that after the ligation, there would be the Spe1  restriction site at the 5' end of the protein coding region. Therefore,  new primers were designed which would eliminate this problem. The protein  coding region will be inserted after the T7 promoter in BBa_K525998. At the 5'  end of the insert, XbaI will be used to join the insert to the plasmid's SpeI  site. At the 3' end of the insert, PstI will be used to join the insert to the  plasmids PstI site. </P><H3> </H3><H3>Week 9 – 18 August</H3><P><STRONG>pSB1C3-MAG1  subcloning –</STRONG></P><UL><LI>INSERT:</LI><UL><LI>Did a double digestion on the pMK-MAG1 plasmid.</LI><LI>Ran the double digestion product on a DNA gel  and gel-extracted.</LI><LI>Ran an analytical amount of the digested  linearized PSB1C3 backbone on a DNA gel. The plasmid was clearly visible on the  gel (Gel C35).</LI></UL><LI>PLASMID: Did a double digestion on a significant  amount of the linearized plasmid backbone and ran the double digestion product  of pMK-MAG1 on a DNA gel. However, the results on the DNA gel (Gel C35) were  inexplicable. We observed a smear in the lanes containing pMK-MAG1, not the  clear bands we expected. Moreover, the insert was not visible at all in the  lower regions of the gel.</LI></UL><P>Due to the anomalous results, we  designed an experiment performing a series of restriction digestions on the pMK  plasmid containing the Magainin 1 insert. Several controls, including single  digestions and digestions using newer tubes of restriction enzymes EcoRI and  PstI were performed.<STRONG>pGEX-RGD cloning –</STRONG>
  It was discovered that there was a mistake in the PCR  primers we have been using up to this date. Specifically, the PCR primers  amplified a segment of the DNA including the periplasmic export tag preceding  the RGD protein coding region. Therefore, new primers had to be designed for  PCR on the RGD insert.
  Nonetheless, we did a double digestion on the pMK-RGD  plasmid, making sure that the volume of the reaction is quite large so that the  insert band was be clearly visible on the DNA gel. This measure is a backup in  the event that the PCR primers are not specific in their binding to the  template. If the primers are non-specific, then we will use the isolated insert  as the template.</P><UL><LI>PLASMID: Ran an analytical amount of the  gel-extracted digested pGEX vector on a gel (Gel C35) and confirmed successfμL  gel-extraction.</LI><LI>INSERT: Ran the restriction digestion product  (PMK-RGD) on a DNA gel. However, strangely we did not get the results we  expected. Instead of seeing two bands, one corresponding to the vector in one  to the insert, we saw approximately 5 bands. This suggests that the restriction  digestion failed, and we might be seeing a multiple bands associated with the  undigested plasmid.</LI></UL><P><STRONG>Cytoplasmic Star  Magainin 1 cloning – </STRONG>
  Completed the PCR reaction to amplify the plasmid in  preparation for ligation and ran on a gel. We have previously digested the  destination plasmid using the relevant restriction enzymes. We did one final  check to ensure that the gel extraction worked by running a sample of the  product on another gel.
  A double digestion was completed on a portion of the gel  extracted PCR insert using XbaI and PstI, two restrictions sites which will  allow us to ligate this insert with the pSB1C3 plasmid (containing the T7  promoter) at the biobrick suffix. The PCR insert was then PCR purified.
  A ligation procedure was performed between pSB1C3 in the PCR  insert. The ligation product was transformed to DH5a.</P><H3> </H3><H3>Week 10 – 25 August</H3><P><STRONG>pSB1C3-MAG1  subcloning –</STRONG>
  A gel run on the restriction digestion experiment last week revealed  that there was something wrong with our old EcoRI/PstI when used in  combination. 
  We designed an experiment using gel electrophoresis of a  combination of different enzymes, used on the same plasmid, repeated with our  old and new enzymes. We found that with the NEW enzymes, the double digestion  works, and with the old enzymes it doesn't work.
  Strangely enough, it looks like the single digests with the  old enzymes still work. It's as if there's some interaction effect when the old  Eco and Post are used together where things go bonkers.
  This experiment has also revealed a limitation of our  cloning strategy. It turns out that there is actually a PstI cutting site in  pMK. Hence, you will see that there actually more than two bands in the double  digestions (and if you look closely, two bands in the single digestions with  PstI). All is not lost, however, as we can still double digest pMK-MAG1 using  different restriction enzymes, bypassing PstI. We can use EcoRI and SpeI, and  this should work (there are no EcoRI or SpeI sites in pMK).
  This sub cloning experiment was started from the restriction  digestion point again. This time, however, we digested the destination vector  (pSB1C3) and insert (contained in pMK-MAG1) with the NEW tubes of EcoRI and  PstI. A gel was run on these products. Both the destination vector, pSB1C3, and  the gel isolated insert were clearly visible on the gel.
  A ligation reaction between the Mag1 insert and the pSB1C3  linearized vector was carried out. The ligation product was transformed to DH5α  and plated to chloramphenicol-containing plates.<STRONG>pGEX-RGD cloning – </STRONG>A  gel was run on the PCR product (previously carried out). However, it appeared  that the PCR reaction was inefficient; the product had a very feint band on the  gel. After gel extraction of the product, nanodrop measurements suggested that  the concentration was in the range 5-10 ng/uL. Because the ultimate goal of  this PCR reaction was to create a lot of insert for a ligation later on, small  amounts of insert DNA were not acceptable. 
  A second PCR reaction was run in order to hopefully increase  the yield of the insert. The number of cycles was increased to 35 from 30. In a  second condition, the gel extracted PCR product from the last reaction was used  as the template.
  The products of the PCR reaction of the RGD insert were run  on a gel and were clearly visible at the correct molecular weight.
  A restriction digestion of both pGEX-6P-3 and the  PCR-amplified RGD insert was carried out. EcoRI and BamHI were used on both  vector and insert to create complementary sticky ends. The reaction was  completed and the products placed in the -20 freezer.<STRONG>Cytoplasmic Star  Magainin 1 cloning – </STRONG>Minipreps were performed on colonies containing the  ligation and the products were run on a DNA gel (Gel C43). However, no DNA was  visible on the gel, suggesting that the minipreps had failed. This is likely  because the culture conditions of the miniprep were suboptimal. However,  another gel showed that the PCR purified insert ran clearly on the gel. Thus,  we can combine that insert with the vector and likely produce a clone again.
  The ligation of the destination vector in the PCR amplified  insert was repeated today. The ligation product was then transformed to DH5α. The  ligation reaction to get the Magainin 1 insert into pSB1C3 was found to be successful.  The MAG 1 insert was mixed with the destination vector pSB1C3 in a ligation  reaction. The ligated product was transformed to DH5α and plated overnight. Colonies  were picked from agar plate and then placed in a PCR reaction (PCR-based colony  screening). The primers used in the PCR were specific to the Magainin 1 insert. 
Performing a PCR on the colonies led to a propagation of the  insert. This suggests that the colonies contained the Magainin 1 insert. And,  because the colonies were chloramphenicol resistant, they should also contain  the pSB1C3 vector. Thus, it appears that we have the vector with the insert! </P><H3> </H3><H3>Week 11 – 1 September</H3><P><STRONG>pSB1C3-MAG1  subcloning – </STRONG>From the plate of DH5α transformed with the ligation product,  PCR colony screening was performed. Only one colony showed successful  transformation on a DNA gel. This culture later showed no growth, and  transformation was suspected to have failed. Because of this, a second  transformation was carried out using the ligation product used for the previous  transformation/plating, and appeared to be successful. Glycerol stocks were  made of the cultures from the second transformation.
  Later, more colonies were miniprepped, double digested, and  ran on a gel. Strangely, as in previous gels, the only thing visible on the gel  was a very faint band with molecular weight greater than 3000. It is unclear  what that band corresponds to. Because the colony screening failed in this  manner across many colonies and two different transformations, it suggests that  there something went wrong during the ligation reaction or earlier.<STRONG>pGEX-RGD cloning –</STRONG>
  The insert for the pGEX RGD stream was PCR purified and then  run on a gel. For unknown reasons, the PCR purification product was not visible  on the gel. The nanodrop suggested that the concentration was 15 ng/µL. 10 µL  of the product were run on the gel. It therefore should have been visible.
  Note, however, that we observed a strange phenomenon where  the DNA/loading dye was floating out of the well once loaded. After trying to  load two wells, we eventually increased the loading dye amount to 5 µL. It  didn't seem as if the DNA was floating out of the well this time, but it may  still have happened.
  The final, restriction digested, PCR purified insert was again  run on a gel. This time, the gel was run with 25 µL of the insert.  Inexplicably, on the gel we observed a very high molecular weight band (greater  than 3000 kDa). This could possibly be contamination from the pGEX vector. In  any case, the insert is not pure or visible on the gel, which suggests that we  need to go back several steps and regenerate it.<STRONG>Cytoplasmic Star  Magainin 1 cloning – </STRONG>Made glycerol stocks of several cultures containing  colonies which contain the correct clone (as verified by the colony screening).  Miniprepped the cultures, double digested the resulting plasmid with EcoRI and  PstI, and ran the product on a gel (A90). Fortunately, most of our plasmids  were shown to have been successfully digested such that the vector and the  insert were both visible as two distinct bands at the correct molecular weight  on the gel. The plasmids were transformed to BL21(DE3) cells and SHuffle T7  cells.
Bacterial colonies were visible on chloramphenicol  containing agar plates with the transformation product. Thus, it appears that  this plasmid is now ready for protein expression.</P><H3> </H3><H3>Week 12 – 8 September</H3><P>This week, agar swabs of bacteria containing three versions  of the SUMO protease arrived from TU Delft. These bacteria were immediately  plated out onto agar plates with no antibiotics. They were also plated onto  plates with the antibiotics corresponding to the resistance of each plasmid.
  Work on Delft’s SUMO protease is indicated by the stream  label ‘Delft’.<STRONG>pGEX-RGD cloning – </STRONG>Performed  a double digest on the remaining sample of the PCR purified PCR product.  However, when the entirety of this restriction digest was run on a gel, it  mysteriously disappeared. This may be a problem with our apparatus, which has  been acting up lately.
  The PCR reaction to amplify the insert was carried out.  Because we've experienced very low yields from previous PCR reactions (of the  scale of 40 ng/uL), this time we increased the cycles from 35 to 45. Also, we  used the PCR product from last time as the template. The reaction appeared to  complete successfully and we ran the products on a DNA gel,  followed by a double digest with EcoRI and  PstI. However, the gel was anomalous. Although the raw PCR product and the  first PCR purification product were visible, the product of the restriction  digest disappeared. This could possibly be due to Star activity, and  appropriate follow-up experiments need to be run.<STRONG>Cytoplasmic Star Magainin  1 cloning – </STRONG>
  Protein expression turned out to be complicated when we  discovered that a chloramphenicol-containing petri dish with a sham  transformation showed colony growth. These colonies should not have appeared on  the plate because the plate had this antibiotic resistance and BL21(DE3) does  not inherently have resistance to chloramphenicol.
  In order to troubleshoot, an experiment was set up with  several controls, including conditions where a new chloramphenicol stock  solution was made up and used, as well as conditions where different cell  lines, separate from BL21(DE3), were plated out.
  We discovered that the particular substrain of BL21(DE3)  that we're using, BL21(DE3)_pLysS, is chloramphenicol resistant. The resistance  comes from the pLysS plasmid. The pLysS is supposed to inhibit constitutive  expression of the T7 polymerase. This is better for expression of toxic  proteins, because it stops leaky expression of the toxic gene (which is why we  chose this strain).
  We confirmed this with a massive experiment with 12  controls. DH5a cells did not grow on the chloramphenicol plates, but did grow  on agar plates with no antibiotic
  Our BL21(DE3) cells grew happily on the chloramphenicol  plates, even though there was no plasmid in them. The cells did not grow on  ampicillin plates.
  Fortunately, our competent SHuffle T7 cells are not  resistant to chloramphenicol. Therefore, we proceeded with protein expression  (see Protocols).
  We then discovered there was an error in the transformation,  in that an incorrect plasmid had been transformed. 
  Also, sequencing data for two of the colonies came back.  Unfortunately, the sequencing reaction failed to produce a sufficiently long  sequence. The primer/plasmid concentrations will be optimised for the next run.<STRONG>TU Delft – </STRONG>
  Today, all of the plates without antibiotics showed  proliferative growth. Only one out of three plates with antibiotics showed  growth. This was the gene for pBAD-UlpI-HIS6-TT. Several colonies from this  plate were picked and cultured in 6 mL of LB overnight in preparation for  plasmid miniprep.
  For the remaining plates, colonies were picked and streaked  onto antibiotic-containing plates to hopefully get some growth. Colonies which  were plated to antibiotic-containing plates showed growth on these new plates.
Last weekend, the FG gene/MAP peptide gene had been  transformed to SHuffle cells and plated onto chloramphenicol-containing plates.  This week, protein expression was carried out. </P><H3> </H3><H3>Week 13 – 15 September</H3><P>Continuing  to attempt to subclone the RGD gene to both pGEX-6P3 and pSB1C3. Discovered  that the reason our small ~300 bp PCR fragments were not visible was because we  were using a 1% gel, 75 V run. We found that we should have been using a 2% gel  at 150 V.
  We  optimized our PCR reaction for the RGDàpGEX cloning task. We used 45 cycles  and the product of the previous reaction as the template.
  Over the  weekend, we expressed all our BioBricks. The FG repeats gene/MAP protein and  the Star Magainin 1 were expressed in BL21(DE3) and Shuffle cells.  We had difficulty running a western blot on  the crude whole cell lysate from the weekend, with nothing showing up during  the ECL image development.
We will  continue to troubleshoot this issue.</P><H3> </H3><H3>Week 14– 22 September</H3><P>The Western Blot issue was somewhat resolved by using all of our parent  lab’s reagents. It appears that our technique is correct, and there must have  been an issue with our reagents or buffers. We suspect our PVDF membrane either  has expired or is otherwise off. However, we will run duplicates of each gel  from now on in order to determine the cause of our previous issues.
We continued the RGD cloning. We were having technical difficulties with  the RGD to pSB1C3 operation. While the PCR product is visible on a gel, the gel  extracted product is not. It is likely because the yield of the PCR is too low,  which means that because of product loss during gel extraction, we don’t have  enough DNA left to see on the subseqeuent gel.</P><H3> </H3><H3>Week 15 – 29 September</H3><P>Successfully optimized the RGD to pSB1C3 PCR reaction by stepping  through different annealing temperatures. By lowering the annealing temperature  by 4 C, we were able to improve yield of the reaction. 
  We thought we might be able to perform the ligation, but when double  digesting the linear magainin 1 gene in pSB1C3 with AgeI and PstI, three bands  showed up on the blot. There should only be two. It is not clear why this is,  and it may be due to contamination of the linear mag 1 plasmid. Although we  could put more resources into this, we decided given the limited time, we would  focus our efforts on other tasks, including protein expression and  characterization and the remaining cloning task.
  Having prepared all the vectors and inserts, we performed a ligation  between RGD and pGEX-6P3 and RGD and pSB1C3.
  The Western Blots from last week were inconclusive. There appeared to be  his-tagged proteins in some of the cultures, but the staining was splotchy, not  a clear band. Also, it was not clear which lanes had the staining.
On our advisors’ advice (H.C. Cheng), we performed a mini-scale his-tag  purification on all cultures using Ni-NTA beads in Eppendorf tubes. We will run  western blots on these purified samples next week.</P><H3> </H3><H3>Week 16 – 6 October</H3><P>The results of the mini-purification were positive, showing  his-tag staining in a western blot in the cultures. We decided to culture more  bacteria as the mini purification did not give us enough sample last time for  downstream analysis, as we had run out of cell pellet.
After the re-expression, we did a second mini-purification,  this time using more (300 uL) of Ni-NTA beads.</P><H3> </H3><H3>Week 17 – 13 October</H3><P>The second mini-purification purification appeared to work  well on a his-tag Western Blot. We therefore set out to characterise our  protein with mass spec, yielding the in-gel digestion results on the results  page.</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Melbourne/Notebook">http://2014.igem.org/Team:Melbourne/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Melbourne/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Melbourne/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Melbourne/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Melbourne/Notebook&amp;oldid=386366" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>