<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NRP-UEA-Norwich_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NRP-UEA-Norwich/Results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>House of Carbs</TITLE></P><DIV class="container"><DIV class="row"><NAV class="navbar navbar-default " role="navigation" data-offset-top="600"><DIV class="navbar-inner"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse  " id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav  " id="nav"><LI><A href="/Team:NRP-UEA-Norwich">Home</A></LI><LI><A href="/Team:NRP-UEA-Norwich/Team">Team</A></LI><LI class="dropdown " id="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project</A></LI><LI><A href="/Team:NRP-UEA-Norwich/Description">Overview</A></LI><LI><A href="/Team:NRP-UEA-Norwich/Project">Background</A></LI><LI><A href="/Team:NRP-UEA-Norwich/Approach">Approach</A></LI><LI><A href="/Team:NRP-UEA-Norwich/Results">Results</A></LI><LI><A href="/Team:NRP-UEA-Norwich/Parts">Parts</A></LI><LI><A href="/Team:NRP-UEA-Norwich/Protocols">Protocols</A></LI></UL><LI class="dropdown " id="dropdown"><A href="/Team:NRP-UEA-Norwich/ModelingAndSoftware" class="dropdown-toggle">Modelling</A></LI><LI><A href="/Team:NRP-UEA-Norwich/Software">Glyco2D</A></LI><LI><A href="/Team:NRP-UEA-Norwich/Modeling/3D">3D Models</A></LI><LI><A href="/Team:NRP-UEA-Norwich/Modeling">Kinetic Model</A></LI><LI><A href="/Team:NRP-UEA-Norwich/Attributions">Attributions</A></LI><LI><A href="/Team:NRP-UEA-Norwich/Practices">Practices</A></LI><LI><A href="/Team:NRP-UEA-Norwich/Collaborations">Collaborations</A></LI><LI><A href="/Team:NRP-UEA-Norwich/Achievements">Achievements</A></LI></DIV></DIV></NAV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="container"><DIV class="row"><DIV class="col-md-12 left"><H2 class="title1 f900" id="top2">RESULTS</H2></DIV></DIV><DIV class="row"><DIV class="col-md-3 col-sm-4 col-xs-12"><DIV class="timeline"><UL class="nav nav-tabs" role="tablist" style="height:1200px;"><LI role="presentation" class="darkblue scroll-link active" data-id="top2"><A href="#start" aria-controls="start" role="tab" data-toggle="tab"><SPAN>- Results summary</SPAN></A></LI><LI role="presentation" class="purple scroll-link" data-id="top2"><A href="#home1" aria-controls="home1" role="tab" data-toggle="tab"><SPAN>- Glycogen branching constructs</SPAN></A></LI><LI role="presentation" class="purple scroll-link  space50" data-id="top2"><A href="#home2" aria-controls="home1" role="tab" data-toggle="tab"><SPAN>- Glycogen acyltransferase constructs </SPAN></A></LI><LI role="presentation" class="purple scroll-link space50" data-id="top2"><A href="#profile" aria-controls="profile" role="tab" data-toggle="tab"><SPAN>- Glycogen content (1)</SPAN></A></LI><LI role="presentation" class="purple scroll-link" data-id="top2"><A href="#messages" aria-controls="messages" role="tab" data-toggle="tab"><SPAN>- Glycogen content (2)</SPAN></A></LI><LI role="presentation" class="purple scroll-link" data-id="top2"><A href="#settings" aria-controls="settings" role="tab" data-toggle="tab"><SPAN>- Glycogen structure</SPAN></A></LI><LI role="presentation" class="purple scroll-link" data-id="top2"><A href="#setting2" aria-controls="setting2" role="tab" data-toggle="tab"><SPAN>- Acyl-glycogen content</SPAN></A></LI><LI role="presentation" class="purple scroll-link" data-id="top2"><A href="#setting3" aria-controls="setting3" role="tab" data-toggle="tab"><SPAN>- Acylation of glycogen</SPAN></A></LI><LI role="presentation" class="purple scroll-link" data-id="top2"><A href="#setting4" aria-controls="setting4" role="tab" data-toggle="tab"><SPAN>- Cloning of plant constructs</SPAN></A></LI><LI role="presentation" class="purple scroll-link space50" data-id="top2"><A href="#setting5" aria-controls="setting5" role="tab" data-toggle="tab"><SPAN>- Chloroplast localisation</SPAN></A></LI><LI role="presentation" class="purple scroll-link" data-id="top2"><A href="#setting6" aria-controls="content4" role="tab" data-toggle="tab"><SPAN>- Starch content</SPAN></A></LI><LI role="presentation" class="aqua scroll-link" data-id="top2"><A href="#pre1" aria-controls="pre1" role="tab" data-toggle="tab"><SPAN>- Butyrate Pathway I</SPAN></A></LI><LI role="presentation" class="aqua scroll-link" data-id="top2"><A href="#pre2" aria-controls="pre2" role="tab" data-toggle="tab"><SPAN>- Butyrate Pathway II</SPAN></A></LI><LI role="presentation" class="aqua scroll-link" data-id="top2"><A href="#pre3" aria-controls="pre3" role="tab" data-toggle="tab"><SPAN>- Production of butyryl coA in <I>E. coli</I></SPAN></A></LI><LI role="presentation" class="aqua scroll-link space50" data-id="top2"><A href="#pre4" aria-controls="pre4" role="tab" data-toggle="tab"><SPAN>- Tests in simulated gut conditions</SPAN></A></LI></UL></DIV></DIV><DIV class="col-md-9 col-sm-8 col-xs-12"><DIV class="tab-content"><DIV role="tabpanel" class="tab-pane active" id="start"><DIV class="row"><DIV class="col-md-12 space50"><H2 class="title1">Results Summary</H2><H4 class="title2">As of 18th September 2015, we have achieved the following. The data can be found in the time line of key experiments below.</H4><P class="space20">1) We assembled BioBricks that alter bacterial glycogen. We have detected changes in branching but have been unable to detect the addition of acyl groups by putative acyl transferases by NMR. This might be because the amount of acylated glycogen is below the detection limit this method. We are currently considering different methods for measuring acylation and mining for more putative transferases. Expression did not interfere with glycogen production. This might be because the enzymes made few or no modifications or because, as predicted by our model, enzymes that can modify any carbon positions do not compete with the glycogen synthases and alter the structure and yield of glycogen. Interestingly, the mass of glycogen increased with one putative transferase. This will need to be repeated and investigated further as this enzyme may have an unexpected activity.</P><P class="space20">2) In plants, we have been able to direct proteins for modification of glucan chains to the chloroplast, the site of starch synthesis. We have also demonstrated that the quantity of starch produced is not affected by expression of these enzymes. We now need to produce plants stably transformed with our constructs in order to be able to obtain enough starch to analyse.</P><P class="space20">3) We have also cloned the coding sequences of the butyrate synthesis pathway, which requires eight coding sequences in two separate BioBricks and into a single BioBrick. We were unable to detect the desired production of butyryl-CoA  in lab strains of <I>E. coli</I> by LC/MS. We are now planning to troubleshoot a method to monitor the production of the butyryl-CoA and also gearing up to test the individual CDSs for function. We also conducted an experiment in a probiotic strain known as <I>E. coli</I> “Nissle” (aka Mutaflor ®). These cells were incubated with a cocktail of gut microbiota in anaerobic simulated gut conditions. We observed an LC/MS peak that is close to but does not correspond exactly to that of the desired butyryl-CoA. Further investigations are now needed to determine if is a derivative or if this is experimental noise. The gut conditions are more likely to be successful as the cells will primarily be metabolizing acetate, the input to our pathway.</P></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="home1"><DIV class="row"><DIV class="col-md-12 space50"><H2 class="title1">Making constructs to express inducible branching and debranching enzymes in <I>E. coli</I></H2><H4 class="title2">AIM:</H4><P class="space30"> We aimed to ligate the <I>E. coli</I> glycogen metabolic genes <A href="http://parts.igem.org/Part:BBa_K1618025" style="color:#002bb8;">GlgX</A> and <A href="http://parts.igem.org/Part:BBa_K1618000" style="color:#002bb8;">GlgB</A> into the standard pSB1C3 plasmid vector with and without an <A href="http://parts.igem.org/Part:BBa_J04500" style="color:#002bb8;">IPTG-inducible promoter</A>. GlgX is a glycogen debranching enzyme which cleaves α-1,6 glycosidic linkages in glycogen to remove branches. GlgB is a glycogen branching enzyme which cleaves α-1,4 glycosidic linkages and re-anneals the cleaved linkage back onto the main chain through an α-1,6 linkage, therefore creating a branch. Creating parts with the promoter allows us to investigate the effect of the expression of these enzymes on <I>E. coli</I> glycogen in an inducible system. Basic parts with just the enzyme coding sequences were also generated so that they can be used to build other composite parts in the future.</P><H4 class="title2">METHOD:</H4><P class="space20">To achieve the cloning, we digested the various DNA components (<A href="http://parts.igem.org/Part:BBa_K1618025" style="color:#002bb8;">GlgX</A> or <A href="http://parts.igem.org/Part:BBa_K1618000" style="color:#002bb8;">GlgB</A> sequence, pSB1C3 and <A href="http://parts.igem.org/Part:BBa_J04500" style="color:#002bb8;">IPTG-inducible promoter</A>) according to the restriction digest protocol with the relevant restriction enzymes to create compatible ends. The digestions were then run on an agarose gel and gel extracted to yield the digested DNA components. These were ligated and transformed into competent <I>E. coli</I> DH5α cells. A colony PCR was undertaken and liquid cultures of colonies with the correctly-sized inserts were prepared. The plasmids were purified by plasmid prep and sent for sequencing. After sequencing, a mutation in the terminator codon of the GlgX sequence was discovered and this was subsequently corrected with mutagenesis PCR to give the correct sequence. See protocols page for full listings.</P><H4 class="title2">RESULTS:</H4><P class="space20">The resulting plasmids were <A href="http://parts.igem.org/Part:BBa_K1618026" style="color:#002bb8;">GlgX with IPTG-inducible promoter</A>, <A href="http://parts.igem.org/Part:BBa_K1618025" style="color:#002bb8;">GlgX without IPTG-inducible promoter</A>, <A href="http://parts.igem.org/Part:BBa_K1618022" style="color:#002bb8;">GlgB with IPTG-inducible promoter</A>, and <A href="http://parts.igem.org/Part:BBa_K1618000" style="color:#002bb8;">GlgB without IPTG-inducible promoter</A>.</P><P class="space20">Figure 1 shows gel electrophoresis of PCR products of GlgB inserted into the backbone without the LacI promoter and RBS.  Lane 1-Ladder, 2 – 11 putative clones of GlgB without promoter, Lane 12-16- putative clones of GlgB with promoter, Lane 17 – no template control, Lane 18- RFP positive control, Lane 19- Ladder. The GlgB CDS is 2231 base pairs and the relevant PCR bands are just above the 2000 bp marker. Sanger sequencing of putative positive clones confirmed the correct insertion in wells 5, 8, 9, 10, 11, 14, and 16.</P><P style="text-align: center;"><B>Figure 1</B></P><P class="space20">Figure 2 shows gel electrophoresis of PCR products of GlgX inserted into the backbone with the LacI promoter and RBS.  Lane 1-Ladder, 2 – no template control, Lane 3- RFP positive control, Lane 4-13 – putative clones of GlgX with promoter, Lane 14- Ladder. The GlgX CDS is 2019 base pairs and the PCR bands are just above the 2000 bp marker. Sanger sequencing of putative positive clones confirmed the correct insertion in wells 5, 9 and 10.</P><P style="text-align: center;"><B>Figure 2</B></P><P class="space20">Figure 3 shows gel electrophoresis of PCR products of GlgX inserted into the backbone without the LacI promoter and RBS.  Lane 1-Ladder, 2– no template control, Lane 3- RFP positive control, Lane 4-6 – bacterial cell controls, Lane 7-14 - putative clones of GlgX without promoter, Lane 15- Ladder. The GlgX CDS is 2019 base pairs and the PCR bands are just above the 2000 bp marker. Sanger sequencing of putative positive clones confirmed the correct insertion in wells 7 through 14.</P><P style="text-align: center;"><B>Figure 3</B></P></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade in" id="home2"><DIV class="row"><DIV class="col-md-12 space50"><H2 class="title1">Making constructs to express putative α-glucan acyltransferases in E. coli</H2><H4 class="title2">AIM:</H4><P class="space30">To ligate four putative α-glucan acyltransferase genetic sequences: <A href="http://parts.igem.org/Part:BBa_K1618001" style="color:#002bb8;">MAO</A>, <A href="http://parts.igem.org/Part:BBa_K1618002" style="color:#002bb8;">Rv3030</A>, <A href="http://parts.igem.org/Part:BBa_K1618003" style="color:#002bb8;">Rv3034c</A>, and <A href="http://parts.igem.org/Part:BBa_K1618004" style="color:#002bb8;">Rv3037c</A> into the standard pSB1C3 plasmid vector with and without an a LacI promoter. </P><H4 class="title2">METHOD:</H4><P class="space20">We digested DNA components (putative acyltransferase sequences, pSB1C3 with <I>LacI</I> promoter and RBS) according to the restriction digest protocol with the relevant restriction enzymes to create compatible ends. The digestions were then run on an agarose gel and gel extracted to yield the digested DNA components. These were ligated and transformed into competent <I>E. coli</I> DH5α cells. Putative clones were screened by colony PCR and liquid cultures of colonies with the correctly-sized inserts were prepared. The plasmids were purified by plasmid prep and sent for sequencing.
</P><H4 class="title2">RESULTS:</H4><P class="space20">The resulting plasmids were <A href="http://parts.igem.org/Part:BBa_K1618005" style="color:#002bb8;">MAO with LacI promoter and RBS</A>, <A href="http://parts.igem.org/Part:BBa_K1618001" style="color:#002bb8;">MAO</A>, <A href="http://parts.igem.org/Part:BBa_K1618007" style="color:#002bb8;">Rv3030 with IPTG-inducible promoter</A>, <A href="http://parts.igem.org/Part:BBa_K1618002" style="color:#002bb8;">Rv3030</A>, <A href="http://parts.igem.org/Part:BBa_K1618008" style="color:#002bb8;">Rv3034c with LacI promoter</A> and RBS, <A href="http://parts.igem.org/Part:BBa_K1618003" style="color:#002bb8;">Rv3034c</A>, <A href="http://parts.igem.org/Part:BBa_K1618009" style="color:#002bb8;">Rv3037c with LacI promoter</A> and RBS and <A href="http://parts.igem.org/Part:BBa_K1618004" style="color:#002bb8;">Rv3037c</A>.</P><P class="space20">Figure 1 shows gel electrophoresis of PCR products of Rv3030 inserted into the backbone without the LacI promoter and RBS.  Lane 1- Ladder, Lane 2- no template control, Lane 3- positive RFP control, Lanes 4-6- putative MAO clones which were contaminated with RFP, Lanes 7-9- putative Rv3030 clones. Sanger sequencing of a putative Rv3030 (with size 850 bp) positive clone confirmed the correct insertion in wells 1 and 3.</P><P style="text-align: center;"><B>Figure 1</B></P><P class="space20">Figure 2 shows gel electrophoresis of PCR products of MAO, Rv3034c and Rv3037c inserted into the backbone without the LacI promoter and RBS.  Lane 1- Ladder, Lane 2- positive RFP control, Lane 3- no template control, Lanes 4-6- putative MAO clones, Lanes 7-9- putative Rv3034c clones, Lanes 10-14- putative Rv3037c clones, Lane 15- Ladder. Sanger sequencing of a putative MAO (with size 600 bp), Rv3034c (with size 950 bp), and Rv3037c (with size 1110) positive clones confirmed the correct insertion in wells 4 through 14.</P><P style="text-align: center;"><B>Figure 2</B></P><P class="space20">Figure 3 shows gel electrophoresis of PCR products of MAO, Rv3030 and Rv3034c inserted into the backbone with the LacI promoter and RBS.  Lane 1- Ladder, Lane 2- positive RFP control, Lane 3- no template control, Lanes 4-6- putative MAO clones, Lanes 7-9- putative Rv3030 clones, Lanes 10-12- putative Rv3034c clones, Lane 13-15- unsuccessful Rv3037c clones, Lane 16- Ladder. Sanger sequencing of a putative MAO (with size 600 bp), Rv3030 (with size 850 bp), and Rv3034c (with size 950 bp) positive clones confirmed the correct insertion in wells 4 through 12.</P><P style="text-align: center;"><B>Figure 3</B></P><P class="space20">Figure 4 shows gel electrophoresis of PCR products of Rv3037c inserted into the backbone with the LacI promoter and RBS.  Lane 1-5 putative Rv3037c clones, Lane 6- Ladder. Sanger sequencing of a putative Rv3037c (with size 1110 bp) positive clones confirmed the correct insertion in wells 1 through 5.</P><P style="text-align: center;"><B>Figure 4</B></P></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="profile"><DIV class="row"><DIV class="col-md-12 space50"><H2 class="title1">Determining if expression of glycogen branching and debranching enzymes affect <I>E. coli</I> glycogen content</H2><H4 class="title2">AIM:</H4><P class="space30">To analyse the effect of GlgX (glycogen debranching enzyme) and GlgB (glycogen branching enzyme) expression on glycogen content in <I>E. coli</I>. We initially set out to investigate this by looking at the staining of whole-cell extracts with Lugol’s solution. The iodine in the Lugol’s solution intercalates in the glucan chains to give a coloured complex. Comparison between the colours observed can give an indication of the amount and branching structure of the glycogen within the sample<SUB><A data-id="ref" style="color: #002bb8;">1</A></SUB>. </P><H4 class="title2">METHOD:</H4><P class="space20">Competent <I>E. coli</I> were transformed with Biobricks expressing GlgX or GlgB under the control of the <I>LacI</I> promoter and firstly grown in 10 mL LB media overnight at 37 °C with shaking. Each culture was used to inoculate 2 x 10 mL of fresh media, grown to an OD of approximately 0.6 and then IPTG was added to one of each duplicate culture and the cultures continued to grow, with samples taken after 1 hour, 3 hours and overnight. At each timepoint a 1 mL extract was taken, spun down to pellet the cells and this pellet was re-suspended in Lugol solution, similar to a method previously used by the <A href="https://2008.igem.org/Team:Edinburgh/Results/Glycogen2" style="color: #002bb8;">Edinburgh 2008 iGEM team</A>.</P><P class="space20">The expected colour upon Lugol addition to glycogen is a dark brown colour. However, as shown in Figures 1-5, no sample has turned this colour, with the majority of the colour seen identical to the orange/yellow of Lugol itself. We therefore repeated this method with nitrogen-limited, carbon rich minimal media M9 minimal media, which had previously been shown to lead to the accumulation of bacterial glycogen<SUB><A data-id="ref" style="color: #002bb8;">2</A></SUB>.</P><P class="space20">Cells grew very slowly under these conditions and only reached an OD of approximately 0.6 after 24 hours.</P><H4 class="title2">RESULTS:</H4><P class="space20">Figures 1 -5 show images of pelleted bacterial cells from LB media, resuspended in Lugol’s solution, and in every case it was not different to Lugol solution alone, even 18 hours after induction. The results from high carbon:nitrogen media are shown in Figure 6, in which every sample has a slightly darker, browner colour compared to the previous samples, indicating a higher glycogen presence compared to LB media. Unfortunately solid particulates remaining in the samples precluded quantifying the results by measuring the absorbance with a spectrophotometer. Whilst there was clearly more glycogen in these samples, it was not possible to distinguish colour differences between the branching and debranching enzymes or between the IPTG-induced and non-induced samples. Therefore it was decided to extract the glycogen from the bacterial cells to analyse it directly.</P><P><B>Figure 1</B>: Lugol’s control for colour comparison. This sample only contains Lugol’s solution.</P><P><B>Figure 2</B>: The pelleted bacterial cells from 1 mL overnight LB cultures with no addition of IPTG re-suspended in 100 uL Lugol’s solution.</P><P><B>Figure 3</B>: The pelleted bacterial cells from 1 mL of LB culture an hour after IPTG-addition re-suspended in 100 uL Lugol’s solution</P><P><B>Figure 4</B>: The pelleted bacterial cells from 1 mL of LB culture 3 hours after IPTG-addition re-suspended in 100 uL Lugol’s solution</P><P><B>Figure 5</B>: The pelleted bacterial cells from 1 mL of LB culture 18 hours after IPTG-addition, re-suspended in 100 uL Lugol’s solution. The lighter colours observed are due to solid particulates in the media.</P><P><B>Figure 6</B>: Pelleted bacterial cells of the branching enzymes grown overnight in low nitrogen, high carbon media with or without IPTG, and resuspended in Lugol’s solution.</P><H2 class="title1" id="ref">References</H2><P>1. Dreiling C, Brown D, Casale L, Kelly L. Muscle glycogen: Comparison of iodine binding and enzyme digestion assays and application to meat samples. Meat Science. 1987;20(3):167-177. </P><P>2. ANTOINE A, TEPPER B. Environmental Control of Glycogen and Lipid Content of Mycobacterium phlei. Journal of General Microbiology. 1969;55(2):217-226.</P></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="messages"><DIV class="row"><DIV class="col-md-12 space50"><H2 class="title1">Determining if expression of glycogen branching and debranching enzymes affect <I>E. coli</I> glycogen content (2)</H2><H4 class="title2">AIM:</H4><P class="space30">To analyse the effect of GlgX (glycogen debranching enzyme) and GlgB (glycogen branching enzyme) expression on glycogen content in <I>E. coli</I>. As we were not able see any differences in whole-cell extracts we decided to extract the glycogen from the bacterial cells and analyse it directly.</P><H4 class="title2">METHOD:</H4><P class="space20"><I>E. coli</I> transformed with inducible GlgB and GlgX were grown up in media that promoted the accumulation of glycogen as described in Glycogen Content 1 to give 1 L of induced and non-induced cultures for each construct. Glycogen was extracted from the cell pellet according to a glycogen extraction protocol, that we initially optimised for <I>E. coli</I> transformed with a control plasmid. This yielded an amorphous white solid that was confirmed to be glycogen by transmission electron microscopy (TEM). The weight of the solid was recorded and induced and non-induced samples were compared for each construct.</P><H4 class="title2">RESULTS:</H4><P class="space20">    •  GlgX Non-induced:glycogen extract mass = 31.8 mg    •  GlgX induced with IPTG: glycogen extract mass = 14.6 mg    •  GlgB Non-induced: glycogen extract mass = 77.0 mg    •  GlgB with IPTG: glycogen extract mass = 77.0 mg</P><P class="space20">The mass of the glycogen extracted when GlgX was induced was nearly half that of glycogen extracted from the same, non-induced culture. This is expected because the GlgX enzyme will de-branch glycogen resulting in linear glucan strands which are lost from the glycogen molecule. Therefore, less glycogen will be extracted from the culture in which GlgX was induced.</P><P class="space20">Both glycogen extracts from GlgB induced and non-induced bacteria weighed exactly the same. It was expected that there would not be a discernible difference in these glycogen extracts. This is because the branching enzyme GlgB, cleaves and re-anneals glucan strands, which would create a more branched structure, but no glucan strands are lost and so the amount of glycogen extracted will be similar in induced and non-induced samples.
</P><P class="space20">Representative TEM images are shown in figure 2, confirming that the white solid obtained in the extraction protocol is glycogen. </P><P style="text-align: center;"><B>Figure 1</B>: Glycogen extracts of GlgX (left) and GlgB (right). The IPTG-induced extract is on the right and the non-induced extract is on the left, in both pictures.</P><P style="text-align: center;"><B>Figure 2</B>: TEM image of GlgB without IPTG induction (left) and GlgB with IPTG induction (right). Little difference in glycogen structure can be observed in these images. The samples were prepared by adding a drop of sample (approximately 10 microlitres) onto a 3 mm 400 mesh copper/palladium (CuPd) grid. This was left to soak on the grid for about 10-15 seconds, then the excess removed by filter paper. Then a drop of uranyl acetate was placed on the grid, left for 10-15 seconds and then removed again and left to dry. The grids were imaged on and FEI Tecnai 200 kV TEM.
</P></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="settings"><DIV class="row"><DIV class="col-md-12 space50"><H2 class="title1">Determining if expression of glycogen branching and debranching enzymes affect <I>E. coli</I> glycogen structure</H2><H4 class="title2">AIM:</H4><P class="space30">We aimed to determine whether the glycogen extracted from bacteria expressing GlgB and GlgX had different structures. We wanted to see if there was a difference in the branch length distributions when GlgB or GlgX were overexpressed.
</P><H4 class="title2">METHOD:</H4><P class="space20">Approximately 1 mg glycogen that had been extracted from induced and non-induced bacterial cultures containing GlgB or GlgX constructs was dissolved in 100 uL 100 mM sodium acetate buffer, pH 4 and then treated with <I>Pseudomonas</I> Isoamylase (1 Unit) at 37 °C for 3 hours. Isoamylase removes α-1,6 linkages and therefore this treatment completely debranched the glycogen samples. A control sample of commercial glycogen was also debranched by this method. An aliquot of the debranched glycogen was then diluted 100 fold in matrix solution (1mg/mL Dihydroxybenzoic acid in 30% aq. Acetonitrile) and analysed by MALDI mass spectrometry. The remaining debranched glycogen was analysed by addition of Lugol’s solution as previously (Figure 1) </P><H4 class="title2">RESULTS:</H4><P class="space20">Figure 1 shows that the cultures clearly stained darkly, which is expected as non-branched chains stain a darker colour with Lugol’s solution. However the colour was not as dark as the control sample of debranched commercial glycogen. This might be explained by our samples being less pure or having a different branch length distribution.
</P><P class="space20">Figure 2 shows the chain length distribution in glycogen from induced GlgB and GlgX cultures. When GlgX is overexpressed the glycogen is less branched, hence there is a shift toward longer chain lengths. Conversely when GlgB is overexpressed there is a higher proportion of shorter chain lengths and a lower maximum chain length can be observed.</P><P class="space20">Therefore, in conclusion, this suggests that the branching and debranching enzymes have had the desired effect upon glycogen structure. This is because debranching enzyme shows longer, less branched, more linear glycogen chains whilst branching enzyme shows more shorter chain lengths and a lower amount of maximum chain lengths indicating branching.</P><P style="text-align: center;"><B>Figure 1</B>: Debranched glycogen samples, stained with Lugol’s solution.</P><P style="text-align: center;"><B>Figure 2</B>: MALDI mass spectra, showing the branch length distribution in glycogen induced GlgB and GlgX cultures, with peaks labelled with number of glucose residues in the chain.</P></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="setting2"><DIV class="row"><DIV class="col-md-12 space50"><H2 class="title1">Determining if expression of putative glycogen acyltransferase enzymes affect <I>E. coli</I> glycogen content</H2><H4 class="title2">AIM:</H4><P class="space30">To analyse the effect of expression of the putative acyltransferase enzymes (MAO, Rv3030, Rv3034c, Rv3037c) on <I>E. coli</I> glycogen content. This was to confirm if expression of the enzymes inhibit the synthesis of glycogen, for example by competing with glycogen synthases for the Carbon 4.</P><H4 class="title2">METHOD:</H4><P class="space20">The <I>E. coli</I> which had been transformed with the acyltransferases under control of an inducible promoter were firstly grown in LB media and then divided into two, with half being induced with IPTG, and then allowed to grow overnight. Cells were harvested from 1 mL of each culture and the cell pellet re-suspended in Lugol’s solution. This process repeated in low nitrogen, high carbon minimal media, which promotes glycogen accumulation. The cells were harvested from 1 L of media and the glycogen was extracted using the glycogen extraction protocol. </P><H4 class="title2">RESULTS:</H4><P class="space20">There was no significant colour change with Lugol staining, although the MAO cultures were noticebly lighter in colour.</P><P><B>Figure 1</B>: Overnight pelleted bacterial cells after IPTG-addition to half of the LB liquid cultures of the branching and acyltransferase enzymes resuspended in Lugol’s solution.</P><P class="space20">The masses of the glycogen extracted are as follows;</P><P>    •  MAO non-induced: glycogen extract mass = 30 mg    •  MAO induced with IPTG glycogen extract mass = 40 mg    •  Rv3030 induced with IPTG: glycogen extract mass = 60 mg    •  Rv3030 non-induced: glycogen extract mass = 30 mg    •  Rv3034c induced with IPTG: glycogen extract mass = 50 mg    •  Rv3034c non-induced: glycogen extract mass = 60 mg    •  Rv3037c induced with IPTG: glycogen extract mass = 40 mg
    •  R3037c non-induced: glycogen extract mass = 40 mg</P><P class="space20">The glycogen from all extracts was an amorphous white powder, confirmed to be glycogen by TEM imaging, as in previous experiments (not shown here). The masses observed were not significantly less when enzymes were induced, indicating that their expression did not interfere with glycogen content. Interestingly, the mass of glycogen was actually doubled upon induction of Rv3030. <B>The expression of this enzyme may increase glycogen biosynthesis by an alternative mechanism.  This will need to be repeated and investigated further as this enzyme may have an unexpected activity.</B></P></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="setting3"><DIV class="row"><DIV class="col-md-12 space50"><H2 class="title1">Determining if expression of putative glycogen acyltransferase enzymes affects the structure of glycogen in <I>E. coli</I></H2><H4 class="title2">AIM:</H4><P class="space30">To determine whether we could detect any acylation or other modifications to glycogen extracted from bacteria expressing the putative acyl transferases.</P><H4 class="title2">METHOD:</H4><P class="space20">Approximately 1 mg glycogen that had been extracted from induced and non-induced bacterial cultures containing the acyltransferase constructs was dissolved in 100 uL 100 mM sodium acetate buffer, pH 4 and then treated with <I>Pseudomonas</I> Isoamylase (1 Unit) at 37 °C for 3 hours. Isoamylase removes α-1,6 linkakes and therefore this treatment completely debranched the glycogen samples. The debranched glycogen was analysed by addition of Lugol’s solution as described in previous results sections. To help look for evidence of acetylation commercial glycogen was chemically acetylated according to a published method<SUB><A data-id="ref" style="color: #002bb8;">1</A></SUB></P><P> Samples of acetylated and non-acetylated commercial glycogen, along with samples of the extracted glycogen were then analysed by <SUB style="bottom: -0.55em; font-size:10px;">1</SUB>H NMR. Approximately 10mg of each sample was dissolved in in 600 uL D<SUB style="bottom: -1.3em;">2</SUB>O and <SUB style="bottom: -0.55em; font-size:10px;">1</SUB>H NMR spectra recorded at 400 MHz. The spectra of extracted glycogen was then compared with the controls. </P><H4 class="title2">RESULTS:</H4><P class="space20">Interestingly the glycogen debranched with isoamylase from cells expressing the putative acylation enzymes stained significantly darker than the controls (Figure 1). This experiment needs to be repeated to confirm this result. </P><P class="space20">Glycogen from cultures expressing Rv3030 and Rv3034 stainened darker with induction of the promoter, whereas for MAO and Rv3037 there was no real difference between induced and non-induced samples (Figure 1). In the case of the darker cultures, this could reflect longer branch lengths or some other modification of the glycogen structure such as acylation.</P><P><B>Figure 1</B>: De-branched glycogen samples, stained with Lugol’s solution.</P><P class="space20">Analysis of glycogen samples by NMR did not indicate the presence of acyl groups  (Figure 2). We now plan to use more sensitive methods such as a chemical assays and also to further investigate the possible structural changes by measuring branch lengths.</P><P><B>Figure 2</B>: <SUB style="bottom: -0.55em; font-size:10px;">1</SUB>H NMR spectra of acetylated and non-acetylated glycogen. The acetylated glycogen shows peaks at about 2 ppm to corresponding to acyl groups. These peaks are not apparent on the non-acetylated spectra. </P><H2 class="title1" id="ref">References</H2><P>1. Haworth W, Percival E. 324. Polysaccharides. Part XI. Molecular structure of glycogen. Journal of the Chemical Society (Resumed). 1932;:2277. </P></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="setting4"><DIV class="row"><DIV class="col-md-12 space50"><H2 class="title1">Making constructs to express putative acyltransferases in plant chloroplasts</H2><H4 class="title2">AIM:</H4><P class="space10">The aim of our prebiotic is to produce acylated/butrylated starch in plants. Methods to chemically acylate starch purified from plants already exist, but as they use strong chemicals and require heating they are not environmentally friendly. Using various acyltransferases, we hope to acylate starch in plants. We use a model plant, <I>Nicotinia benthamiana</I>, for initial tests because we can obtain results within a few days. Later on we would aim to make transgenic plants in a species that makes a lot of starch, such as maize (corn), potatoes or wheat.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H4 class="title2">METHOD:</H4><P class="space10">We used Golden Gate Cloning  and the Plant Standard Syntax<SUB><A data-id="ref" style="color: #002bb8;">1</A></SUB> to make our constructs. We used 35s promoter from Cauliflower Mosiac Virus (<A href="http://parts.igem.org/Part:BBa_K1467101" style="color:#002bb8;">BBa_K1467101</A>), to drive constitutive expression.  We made a chloroplast transit peptide (New part - <A href="http://parts.igem.org/Part:BBa_K1618028" style="color:#002bb8;">BBa_K1618028</A>) that we then used in an N-terminal translational fusion with the acyl transferases for the protein to reach the chloroplast, where starch is produced.</P><P class="space20">In order to confirm that the transit peptide was functional we made a second set of constructs that had a fluorescent reporter in a C-terminal translational fusion.</P><P class="space20">To deliver the constructs to plants we needed to use <I>Agrobacterium tumefaciens</I> as a shuttle chassis. This meant parts needed to be assembled into a binary vector with origins of replication for both <I>E. coli</I> and <I>A. tumefaciens</I>. In order to submit the parts to the registry we also assembled parts into the MoCloFlipper pSB1C3 that accepts Golden Gate Parts into the pSB1C3 backbone.</P><P>All cloning was done according to the Golden Gate one-step Digestion-Ligation Protocol provided on our protocols page <A href="https://2015.igem.org/Team:NRP-UEA-Norwich/Protocols#Ones-Step Golden Gate Digestion Ligation Protocol" style="color: #002bb8;">here</A>.</P><H4 class="title2">Results:</H4><P class="space20"><B>Successful assembly into the binary vector.</B></P><P class="space10">Putative clones were screened by colony PCR (see Protocols for method - <A href="https://2015.igem.org/Team:NRP-UEA-Norwich/Protocols#Colony PCR Protocol" style="color: #002bb8;">here</A>) using primers that flank the insertion sites (Figure 1).
</P><P><B>Figure 1</B>: The gel lanes are as follows –  1 =<I> Ladder</I>, 2&amp;3 = <I><A href="http://parts.igem.org/Part:BBa_K1618033" style="color:#002bb8;">BBa_K1618033</A></I>, 4&amp;5 = <I><A href="http://parts.igem.org/Part:BBa_K1618029" style="color:#002bb8;">BBa_K1618029</A></I>, 6&amp;7 = <I><A href="http://parts.igem.org/Part:BBa_K1618035" style="color:#002bb8;">BBa_K1618035</A></I>, 8&amp;9 = <I><A href="http://parts.igem.org/Part:BBa_K1618031" style="color:#002bb8;">BBa_K1618031</A></I>, 10&amp;11 = <I><A href="http://parts.igem.org/Part:BBa_K1618036" style="color:#002bb8;">BBa_K1618036</A></I>, 12&amp;13 = <I><A href="http://parts.igem.org/Part:BBa_K1618032" style="color:#002bb8;">BBa_K1618032</A></I>, 14&amp;15 = <I><A href="http://parts.igem.org/Part:BBa_K1618034" style="color:#002bb8;">BBa_K1618034</A></I>, 16&amp;17 = <I><A href="http://parts.igem.org/Part:BBa_K1618030" style="color:#002bb8;">BBa_K1618030</A></I>, 18 = ladder. The gel image above indicated that the complete transcriptional units were successfully cloned. </P><P class="space20">A single colony for each construct was mini-prepped and sequenced before progression to transformation into <I>A. tumefaciens</I>.</P><P class="space20"><B>Successful assembly into pSB1C3.</B></P><P class="space30">Putative clones were miniprepped (see Protocols for method - <A href="https://2015.igem.org/Team:NRP-UEA-Norwich/Protocols#QIAGEN Plasmid Prep Protocol" style="color: #002bb8;">here</A>) and screened by digestion with <I>Not</I>I (Figure 2) and also with 
<I>EcoR</I>I/<I>Pstl</I> (Figure 3) to confirm that cloning was successful and that our constructs had no internal BioBrick restriction sites. </P><P><B>Figure 2</B>: The gel lanes are as follows –  1 =<I> Ladder</I>, 2&amp;3 = <I><A href="http://parts.igem.org/Part:BBa_K1618033" style="color:#002bb8;">BBa_K1618033</A></I>, 4&amp;5 = <I><A href="http://parts.igem.org/Part:BBa_K1618029" style="color:#002bb8;">BBa_K1618029</A></I>, 6&amp;7 = <I><A href="http://parts.igem.org/Part:BBa_K1618035" style="color:#002bb8;">BBa_K1618035</A></I>, 8&amp;9 = <I><A href="http://parts.igem.org/Part:BBa_K1618031" style="color:#002bb8;">BBa_K1618031</A></I>, 10&amp;11 = <I><A href="http://parts.igem.org/Part:BBa_K1618036" style="color:#002bb8;">BBa_K1618036</A></I>, 12&amp;13 = <I><A href="http://parts.igem.org/Part:BBa_K1618032" style="color:#002bb8;">BBa_K1618032</A></I>, 14&amp;15 = <I><A href="http://parts.igem.org/Part:BBa_K1618034" style="color:#002bb8;">BBa_K1618034</A></I>, 16&amp;17 = <I><A href="http://parts.igem.org/Part:BBa_K1618030" style="color:#002bb8;">BBa_K1618030</A></I>, 18 = ladder. These indicate that there are no unwanted internal <I>Not</I>1 restriction sites within our constructs.</P><P><B>Figure 3</B>: The gel lanes are as follows – 1 = Ladder, 2&amp;3 = <I><A href="http://parts.igem.org/Part:BBa_K1618029" style="color:#002bb8;">BBa_K1618029</A></I>, 4&amp;5 = <I><A href="http://parts.igem.org/Part:BBa_K1618031" style="color:#002bb8;">BBa_K1618031</A></I>, 6&amp;7 = <I><A href="http://parts.igem.org/Part:BBa_K1618032" style="color:#002bb8;">BBa_K1618032</A></I>, 8&amp;9 = , 10&amp;11 =<I><A href="http://parts.igem.org/Part:BBa_K1618029" style="color:#002bb8;">BBa_K1618029</A></I>, 12&amp;13 = <I><A href="http://parts.igem.org/Part:BBa_K1618031" style="color:#002bb8;">BBa_K1618031</A></I>, 14&amp;15 = <I><A href="http://parts.igem.org/Part:BBa_K1618032" style="color:#002bb8;">BBa_K1618032</A></I>, 16&amp;17 = <I><A href="http://parts.igem.org/Part:BBa_K1618030" style="color:#002bb8;">BBa_K1618030</A></I>, 18 = Ladder. Lanes 2-9 were digested with Not1, while lanes 10-17 were digested with EcoR1/Pst1. The results indicate that the new tags allow our constructs to be BioBrick compatible. </P><P class="space20">A single colony for each construct was mini-prepped and sequenced before shipping to the registry.</P><H2 class="title1">References</H2><P>1. Patron <I>et al</I>. (2015) Standards for Plant Synthetic Biology: A Common Syntax for Exchange of DNA Parts doi: 10.1111/nph.13532/full</P></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="setting5"><DIV class="row"><DIV class="col-md-12 space50"><H2 class="title1">Confirming the sub-cellular localisation of acyltransferases in plant chloroplasts</H2><H4 class="title2">AIM:</H4><P class="space30">To determine if our synthetic chloroplast transit peptide would direct the putative acyl trasnferases to the plant chloroplast. The aim of this experiment is to confirm that our constructs localise the acyl transferase to the plant chloroplast.  In order for the enzymes that we have chosen to acylate starch, the enzymes need to reach the chloroplast of a cell as this is where starch is produced. </P><H4 class="title2">METHOD:</H4><P class="space20">We transformed our binary vector constructs (see results “Making constructs to express putative acyl-transferases in plant chloroplasts” for the details of these constructs) containing the acyltransfeases with an N-terminal transit peptides and  C-terminal fluorescent reporters into <I>Agrobacterium tumefaciens</I> by electroporation (see Protocols - <A href="https://2015.igem.org/Team:NRP-UEA-Norwich/Protocols#Electroporation Transformation Protocol" style="color: #002bb8;">here</A>). Colonies were checked by PCR before liquid cultures were grown from individual colonies for infiltration of plant leaves (see Protocols - <A href="https://2015.igem.org/Team:NRP-UEA-Norwich/Protocols#Agrobacterium tumefaciens" style="color: #002bb8;">here</A>). Finally, infiltrated leaves were examined by confocal microscopy to determine the subcellular localisation of the recombinant protein.
</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><H4 class="title2">RESULTS:</H4><P class="space30">Colonies of <I>A. tumefaciens</I> were screened by PCR to confirm the presence of the binary construct (Figure 1).
</P><P><B>Figure 1</B>: The gel lanes are as follows – 1 =<I><A href="http://parts.igem.org/Part:BBa_K1618029" style="color:#002bb8;">BBa_K1618029</A> control</I>, 2&amp;3 = <I><A href="http://parts.igem.org/Part:BBa_K1618029" style="color:#002bb8;">BBa_K1618029</A> Agrobacterium</I>, 4 = <I><A href="http://parts.igem.org/Part:BBa_K1618031" style="color:#002bb8;">BBa_K1618031</A> control</I>, 5&amp;6 = <I><A href="http://parts.igem.org/Part:BBa_K1618031" style="color:#002bb8;">BBa_K1618031</A> Agrobacterium</I>, 7 = Ladder, 8 = <I><A href="http://parts.igem.org/Part:BBa_K1618032" style="color:#002bb8;">BBa_K1618032</A> control</I>, 9&amp;10 = <I><A href="http://parts.igem.org/Part:BBa_K1618032" style="color:#002bb8;">BBa_K1618032</A> Agrobacterium</I>, 11 = <I><A href="http://parts.igem.org/Part:BBa_K1618030" style="color:#002bb8;">BBa_K1618030</A> control</I>, 12&amp;13 = <I><A href="http://parts.igem.org/Part:BBa_K1618030" style="color:#002bb8;">BBa_K1618030</A> Agrobacterium</I>, 14 = Ladder. The comparison between the agro-transformations and the sequence confirmed control suggests that the cloning was successful. </P><P class="space30">Leaves that had been infiltrated with <I>A. tumefaciens</I> strains containing our assembled binary constructs were images using an SP5 Leica confocal microscope (Figure 2). We used two channels: the first excites chlorophyll, which is shown in red and the second excites the fluorescent fusion protein (shown in yellow).</P><P><B>Figure 2</B>: Constructs BBa_K1618029-032 contain a yellow fluorescent protein, as well as a chloroplast transit peptide. These are confocal microscopy images of the constructs infiltrated into Nicotiana benthamiana, in which the red structures are the chlorophyll within the chloroplast, and the yellow is the fluorescent fusion protein expressed from constructs (a) <A href="http://parts.igem.org/Part:BBa_K1618029" style="color:#002bb8;">BBa_K1618029</A> (MAO enzyme), (b) <A href="http://parts.igem.org/Part:BBa_K1618031" style="color:#002bb8;">BBa_K1618031</A> (RV3034c enzyme), (c) <A href="http://parts.igem.org/Part:BBa_K1618032" style="color:#002bb8;">BBa_K1618032</A> (RV3037c enzyme), and (d) <A href="http://parts.igem.org/Part:BBa_K1618030" style="color:#002bb8;">BBa_K1618030</A> (RV3030 enzyme).
</P><P class="space30">Our results confirm that the putative acyltransferases  are successfully localised to the chloroplasts of the plants. </P></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="setting6"><DIV class="row"><DIV class="col-md-12 space50"><H2 class="title1">Determining if the expression of acyl-transferases interferes with starch accumulation in plants</H2><H4 class="title2">AIM:</H4><P class="space30">To compare the starch content of leaves infiltrated with <I>A.tumefaciens</I> strains carrying constructs expressing our putative acyltransferases.  A potential problem of using putative acyl transferases is that they will use the same C4 position of the glucose molecules that the starch synthases add new ADP-glucose units to, interfering with growth of the starch molecule. A simple test was to confirm that normal quantities of starch were being produced. </P><H4 class="title2">METHOD:</H4><P class="space20">We transformed our binary vector constructs (see results “Making constructs to express putative acyl-transferases in plant chloroplasts” for the details of these constructs) containing the acyltransfeases with an N-terminal transit peptides into <I>Agrobacterium tumefaciens</I> by electroporation (see Protocols - <A href="https://2015.igem.org/Team:NRP-UEA-Norwich/Protocols#Electroporation Transformation Protocol" style="color: #002bb8;">here</A>). Colonies were checked by PCR before liquid cultures were grown from individual colonies for infiltration of plant leaves (see Protocols - <A href="https://2015.igem.org/Team:NRP-UEA-Norwich/Protocols#Agrobacterium tumefaciens" style="color: #002bb8;">here</A>).</P><P class="space20">Three plants were infiltrated with each experiment and control:- <I>A. tumefaciens</I> with no construct (control)
     - Strains of <I>A. tumefaciens</I> that contained each of the four binary vectors expressing putative acyltransferases.
     - The plants were of the same age, same species, and were grown in the same conditions..
</P><P class="space20">After infiltration, the plants were left in a normal light/dark cycle at room temperature for 24 hours before being put in the dark for 24 hours. In the dark the plants would break down the stores of starch in their leaves.</P><P class="space20">After the dark treatment the samples from the infiltrated plants as well as control plants that had not been infiltrated were decolorized to remove the green chlorophyll and stained with iodine, which stains starch granules. A duplicate set of experiments was allowed to grow in the light for the rest of the day to build up new stores of starch. These were then assayed in the same manner to determine if expression of the putative acyltransferases interfered with starch accumulation.</P><H4 class="title2">RESULTS:</H4><P class="space20">Colonies of <I>A. tumefaciens</I> were screened by PCR to confirm the presence of the binary construct (Figure 1).</P><P><B>Figure 1</B>: The gel lanes are as follows – 1 = Ladder, 2 = <I><A href="http://parts.igem.org/Part:BBa_K1618033" style="color:#002bb8;">BBa_K1618033</A> control</I>, 3&amp;4 = <I><A href="http://parts.igem.org/Part:BBa_K1618033" style="color:#002bb8;">BBa_K1618033</A> Agrobacterium</I>, 5 = <I><A href="http://parts.igem.org/Part:BBa_K1618035" style="color:#002bb8;">BBa_K1618035</A> control</I>, 6&amp;7 = <I><A href="http://parts.igem.org/Part:BBa_K1618035" style="color:#002bb8;">BBa_K1618035</A> Agrobacterium</I>, 8 = Ladder, 9 = <I><A href="http://parts.igem.org/Part:BBa_K1618036" style="color:#002bb8;">BBa_K1618036</A> control</I>, 10&amp;11 = <I><A href="http://parts.igem.org/Part:BBa_K1618036" style="color:#002bb8;">BBa_K1618036</A> Agrobacterium</I>, 12 = <I><A href="http://parts.igem.org/Part:BBa_K1618034" style="color:#002bb8;">BBa_K1618034</A> control</I>, 13&amp;14 = <I><A href="http://parts.igem.org/Part:BBa_K1618034" style="color:#002bb8;">BBa_K1618034</A> Agrobacterium</I>. The comparison between the agro-transformations and the sequence confirmed control suggests that the cloning was successful. </P><P class="space20">After 24 hours in the dark the leaves from all samples were free of starch (<B>Figure 2A</B>). This allowed us to determine the amount of starch accumulated in the following 24 hours. Leaves sampled after 24 hours in the light all stained dark with iodine. There is no difference in the amount of staining observed in samples from leaves infiltrated with the <I>A. tumefaciens</I> strains that contained the binary vectors expressing putative acyl transferases as compared to the control samples (<B>Figure 2B</B>).</P><P><B>Figure 2</B>: Iodine staining of leaves after (A) 24 hours dark treatment, and (B) dark treatment and 8 hours light treatment. The yellow samples, indicate that no starch is present  - While there are black spots observed in some samples it is likely that these are due to damage from infiltration rather than starch. Black stain indicates the presence of starch. The level of staining doesn’t appear to differ between the different leaves and their conditions, suggesting that neither the infiltration process, or the expression of acyltransferases, affects starch production in leaves.</P><P class="space20">These results indicate that neither the infiltration process or the expression of putative acyl-transferases has an impact on the production of starch.</P></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="pre1"><DIV class="row"><DIV class="col-md-12 space50"><H2 class="title1">Cloning the Butyrate Biosynthetic Pathway - PART I</H2><P class="space30">Our overall aim was to clone the coding sequences (CDSs) of the butyrate biosynthetic pathway from <I>Coprococcus</I> sp. L2-50 (DSM 16842) for expression in <I>E. coli</I>. In total the pathway contains eight coding sequences. We managed to clone the pathway that would convert butyrate into butyryl-CoA (Pathway I) and, separately, the pathway that would convert acetyl-CoA into butyryl-CoA (Pathway II). </P><H3 class="title2">Pathway I:</H3><H4 class="title2">AIM:</H4><P class="space30">To clone the CDSs from the <A href="http://parts.igem.org/Part:BBa_K1618012" style="color:#002bb8;">BUK</A> (butyrate kinase) and <A href="http://parts.igem.org/Part:BBa_K1618013" style="color:#002bb8;">PTB</A> (<I>phosphotransbutyrylase</I>) genes from <I>Coprococcus</I> sp. with an <A href="http://parts.igem.org/Part:BBa_J04500" style="color:#002bb8;">IPTG-inducible promoter</A>. This BioBrick is designed catalyze the conversion of butyrate from/to butyryl-CoA. The PTB gene converts butyryl-CoA from/to butyryl-phosphate and the BUK gene converts butyrate-phosphate from/to butyrate.</P><H4 class="title2">METHOD:</H4><P class="space20">The CDS of the PTB and BUK coding sequences were synthesised. We digested them from their shipping plasmids with <I>XbaI</I> and <I>PstI</I>.  The pSB1C3 vector was digested with <I>EcoRI</I> and <I>PstI</I> and, the BioBrick with the LacI promoter and RBS with <I>EcoRI</I> and SpeI (See digestion protocol). The digests were run on an agarose gel and bands were gel extracted (See gel extraction protocol). The BUK CDS was ligated into the pSB1C vector together with the
IPTG-inducible promoter and RBS and the PTB CDS was added to this BioBrick to make the final pathway (<B>Figure 1</B>). 
Transformation and colony screening was carried out as per our standard protocols). Putative clones were confirmed by sequencing.</P><P><B>Figure 1</B>: Biobrick structure of the BUK/PTB pathway with the IPTG-inducible promoter (lacI)</P><H4 class="title2">RESULTS:</H4><P class="space20">Successful digestion and assembly of the CDSs of the PTB and BUK genes into the PSB1C3 vector with the LacI promoter and RBS.</P><P class="space20">In order to confirm that the gene inserts were successfully ligated, Colonies were screened by PCR (<B>Figure 2</B>). Putative clones were confirmed by Sanger sequencing. </P><P><B>Figure 2</B>: Colony PCR products of BUK/PTB construct (lane 2-7) Primers used were BUK forward and VR. Expected size to be 1486 bp adding up to 1500 bp if considering promoter sequence.</P></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="pre2"><DIV class="row"><DIV class="col-md-12 space50"><H2 class="title1">Cloning the Butyrate Biosynthetic Pathway - PART II</H2><P class="space30">Our overall aim was to clone the coding sequences (CDSs) of the butyrate biosynthetic pathway from <I>Coprococcus</I> sp. L2-50 (DSM 16842) for expression in <I>E. coli</I>. In total the pathway contains eight coding sequences. We managed to clone the pathway that would convert butyrate into butyryl-CoA (Pathway I) and, separately, the pathway that would convert acetyl-CoA into butyryl-CoA (Pathway II). </P><H3 class="title2">Pathway II:</H3><H4 class="title2">AIM:</H4><P class="space30">To clone the CDSs from the <I><A href="http://parts.igem.org/Part:BBa_K1618014" style="color:#002bb8;">thiolase</A> (THL)</I>, <A href="http://parts.igem.org/Part:BBa_K1618016" style="color:#002bb8;"><I>beta-hydroxybutyryl coenzyme</I> A <I>dehydrogenase </I></A><I> (BHBD), <A href="http://parts.igem.org/Part:BBa_K1618015" style="color:#002bb8;">Crotonase</A> (CRO), <A href="http://parts.igem.org/Part:BBa_K1618017" style="color:#002bb8;">Butyryl-CoA dehydrogenase</A> (BCD), <A href="http://parts.igem.org/Part:BBa_K1618019" style="color:#002bb8;">Electron Flavoprotein subunit</A></I><A href="http://parts.igem.org/Part:BBa_K1618019" style="color:#002bb8;"> a</A><I>(EFa)</I>, and <A href="http://parts.igem.org/Part:BBa_K1618018" style="color:#002bb8;"><I>Electron Flavoprotein subunit</I> b</A><I>(EFb)</I> genes from <I>Coprococcus</I> sp. with an IPTG-inducible promoter.  This BioBrick should catalyze the conversion of acetyl-CoA to butyryl-CoA.</P><P><B>Figure 1</B>: Schematic of the compound part with the coding sequences for THL/BHBD/CRO/BCD/EFb/EFa (<A href="http://parts.igem.org/Part:BBa_K1618021" style="color:#002bb8;">BBa_K1618021)</A></P><H4 class="title2">METHOD:</H4><P class="space20">The pSB1C3 vector was digested with <I>EcoRI</I> and <I>PstI</I> to add the <I>LacI</I> promoter and RBS, which was digested with <I>EcoRI</I> and <I>SpeI</I> (See digestion protocol). The digested plasmid was run on an agarose gel and the band was purified (See gel extraction protocol). The <I>LacI</I>-RBS part was ligated into the plasmid (See Ligation protocol) and then transformed into competent cells (See transformation protocol). The CDSs for each part of the pathway were inserted into this plasmid by digestion with <I>SpeI</I> and <I>PstI</I> and ligating in the CDSs that were released from their shipping plasmids with <I>EcoRI</I> and <I>SpeI</I>.</P><P>Colonies were screened by PCR (see colony PCR protocol) and putative clones were miniprepped and confirmed by sequencing.</P><H4 class="title2">RESULTS:</H4><P class="space20"><B>Successful digestion and assembly of 6 CDSs from the butyrate biosynthesis pathway into the PSB1C3 vector.</B></P><P class="space20">The six genes (THL, BHBD, CRO, BCD, EFb,  EFa ) and the pSB1C3 vector with the <I>Lacl</I> promoter and RBS were digested. The expected sizes are: THL=1129 bp, BHBD=900 bp, CRO=836 bp, BCD=1211 bp, EFa=1100 bp, Efb=830 bp Figure 2 shows the successful release of parts form their backbones.</P><P><B>Figure 2</B>: Agarose gel electrophoresis of restriction digestion of for the six genes of the butyrate pathway II released from their shipping plasmids as well as the backbones Lane 1=1KB Ladder, Lane 2 = pSB1C3 , Lane 3=LacI and RBS, Lane=4 PTB, Lane 5=THL, Lane 6=BCD, Lane 7=CRO, Lane 8=BHBD,Lane 9=EFa, Lane 10=EFb</P><P class="space20">Figure 3 shows colony PCR of the THL ligated to the LacI promoter and RBS. Primers used for colony PCR and sequencing were located in the plasmid vector backbone. Expected size of THL/PRO was 1500 bp.</P><P><B>Figure 3</B>: Agarose gel electrophoresis of PCR products from a colony PCR of for addition of the THL CDS to the LacI promoter and RBS. Lane 1=1KB ladder. Lane 2-8 THL/Promoter samples. Only sample 4,6 and 8 were successful.</P><P class="space20">Figure 4 (A) shows colony of BHBD ligated to THL, the LacI promoter and RBS .Expected size was 1100 bp. Figure 4(B) shows the digest of a successful clone. It can be observed that the band size of the digest increases each time from the previous suggesting the addition of the new insert.</P><P><B>Figure 4</B>: (A) Agarose gel electrophoresis of PCR products from a colony PCR of for addition of the BHBD CDS to the THL CDS, LacI promoter and RBS. Lane 1=1KB ladder. Lane 2-6 putative clones. (B) Agarose gel electrophoresis of digestion products of a positive clone. Lane 1=1 KB Ladder, Lane 2 Clone digest</P><P class="space20">Figure 5(A) shows colony of CRO ligated to BHBD, THL, the LacI promoter and RBS. Expected size was 1400 bp. Figure 5(B) shows the digest of a successful clone. It can be observed that the band size of the digest increases each time from the previous suggesting the addition of the new insert.</P><P><B>Figure 5</B>: (A) Agarose gel electrophoresis of PCR products from a colony PCR of for addition of the CRO CDS to the BHBD,THL, LacI promoter and RBS. Lane 1=1KB ladder. Lane 2-9 putative clones. Only lanes 2,3, 4, 6 and 8 were successful. (B) Agarose gel electrophoresis of digestion products of two positive clones. Lane 1=1 KB Ladder, Lanes 2-3 Digested clones </P><P class="space20">Figure 6(A) shows colony of BCD ligated to CRO, BHBD, THL, the LacI promoter and RBS .Expected size was 1900 bp. Figure 6(B) shows the digest of a successful clone. It can be observed that the band size of the digest increases each time from the previous suggesting the addition of the new insert.</P><P><B>Figure 6</B>: (A) Agarose gel electrophoresis of PCR products from a colony PCR of for addition of the BCD CDS to the CRO, BHBD,THL, LacI promoter and RBS. Lane 1=1KB ladder. Lane 2-3 putative clones. (B) Agarose gel electrophoresis of digestion products of two positive clones. Lane 1=1 KB Ladder, Lanes 2-3 digested clones </P><P class="space20">Figure 7(A) shows colony of EFb ligated to BCD, CRO, BHBD, THL, the LacI promoter and RBS. Expected size was 1500 bp. Figure 7 (B) shows the digest of a successful clone. It can be observed that the band size of the digest increases each time from the previous suggesting the addition of the new insert.</P><P><B>Figure 7</B>: (A) Agarose gel electrophoresis of PCR products from a colony PCR of for addition of the EFb CDS to the CRO, BCD, BHBD,THL, LacI promoter and RBS. Lane 1=1KB ladder. Lane 2-9 putative clones. Only lanes 2,3, 5, 6, 7, 8 and 9 were successful. (B) Agarose gel electrophoresis of digestion products of a positive clone. Lane 1=1 KB Ladder, Lanes 2 Digested clone.</P><P class="space20">Figure 8(A) shows colony of EFa ligated to EFb, BCD, CRO, BHBD, THL, the LacI promoter and RBS. Expected size was 1550 bp.</P><P><B>Figure 8</B>: (A) Agarose gel electrophoresis of PCR products from a colony PCR of for addition of the EFa CDS to the EFb, CRO, BCD, BHBD,THL, LacI promoter and RBS. Lane 1=1KB ladder. Lane 2-9 putative clones. </P></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="pre3"><DIV class="row"><DIV class="col-md-12 space50"><H2 class="title1">Measuring the production of butyryl CoA in cultures of <I>E. coli</I></H2><H4 class="title2">AIM:</H4><P class="space30">To determine if BioBricks containing the two pathways (i) BUK/PTB pathway (ii) THL/BHBD/CRO/BCD/EFb/EFa pathway 
could produce butyryl coA in <I>E. coli</I>  fed with (i) acetate or (ii) butyric acid</P><H4 class="title2">METHOD:</H4><H5 class="title2">High Pressure Liquid Chromatography (HPLC)</H5><P class="space20">In order to analyse butyryl-CoA we used HPLC. Initial development was performed using reverse phase liquid chromatography with a C18 column 100mm x 4.6mm using UV detection at 250nm and various combinations of 0.1% formic acid and acetonitrile to obtain sufficient retention.  This method was first tested on CoA and butyryl -CoA standards but no peak for CoA or butyl-CoA could be seen using this mobile phase combination. The 0.1% formic acid was replaced with water to provide a higher pH to see if this would improve the chromatography. Again no peak was seen. We suspected that the compounds may have been sensitive to active sites within the HPLC system so the water was replaced with 0.1M ammonium acetate. This resulted in a sharp peak of reasonable retention for both CoA and butyl-CoA. The final parameters used are listed below</P><P class="space20">Column: Luna C18 3µm 100mm x 4.6mmDetection: UV @ 250nm</P><P>Mobile phase:     Mobile phase A: 0.1M ammonium acetate     Mobile phase B: Acetonitrile</P><TABLE style="width:50%"><TBODY><TR><TH><B>Time</B></TH><TH><B>%A</B></TH><TH><B>%B</B></TH><TH><B>Flow (mL/min)</B></TH></TR><TR><TD>0</TD><TD>95</TD><TD>5</TD><TD>1</TD></TR><TR><TD>5</TD><TD>60</TD><TD>40</TD><TD>1</TD></TR><TR><TD>6</TD><TD>30</TD><TD>70</TD><TD>1</TD></TR><TR><TD>7</TD><TD>30</TD><TD>70</TD><TD>1</TD></TR><TR><TD>7.5</TD><TD>95</TD><TD>5</TD><TD>1</TD></TR><TR><TD>10</TD><TD>95</TD><TD>5</TD><TD>1</TD></TR></TBODY></TABLE><P class="space30">Extracts from cultures of <I>E. coli</I> containing the BioBricks for either pathway were analysed under these conditions but, due to the presence of many other peaks, it was not possible to determine if butyl-CoA was present. For this reason we decided use <B>Mass Spectrometry</B> (MS) to aid detection. Analysis was transferred to a Shimadzu single quadrupole LC/MS (Liquid Chromatograph/<B>Mass Spectrometer</B>) instrument. The column used had the same stationary phase but was 2.0 mm in diameter to allow a lower flow rate of 0.4mL/min to be used, being more suitable for operation with the MS. The concentration of the ammonium acetate was reduced to 0.05M. The MS was operated in selected ion mode to increase sensitivity, monitoring masses 838 and 836 for +ve and –ve electrospray ionisation of the butyryl -CoA</P><H5 class="title2">Sample preparation</H5><P>Plasmids were transformed into <I>E. coli</I> BL21 overexpression strain and expression was induced with IPTG overnight. Cells containing BUK/PTB were fed butyric acid to a final concentration of 50 mM and glycerol to a final concentration of 15%<SUB>1</SUB>. The six gene pathway was fed with sodium acetate to a final concentration of 33mM and glycerol to a final concentration of 15%<SUB>2</SUB>. To analyse only the intracellular metabolites cells were cooled on ice, centrifuged at 5000 rpm at 4 °C. The supernatant was discarded and pellets were re-suspended in 1mL of 6% perchloric acid. After re-suspension, 0.3 ml of 3M K<SUB style="bottom: -1.3em;">2</SUB>C0<SUB style="bottom: -1.3em;">3</SUB> was added and the cells were vortexed. Cells were centrifuged at 1300 rpm for one minute and the supernatant was filter-sterilised<SUB>3</SUB>. Filtered supernatant was analysed through HPLC following the protocol described above.</P><H4 class="title2">RESULTS:</H4><P class="space20">In order to analyse the butyryl-CoA we first had to run the standards to observe at what time butyryl-CoA was retained. The results for the retention time of butyryl-CoA are shown in Figure 1.
The first sample that was analysed was the BUK/PTB pathway, the result are shown in Figure 2.
Figure 2 No peak is formed at 2.90 minutes when the BUK/PTB sample is analysed. This suggests that no butyryl-CoA is produced or that it below the detection limit. </P><P><B>Figure 1</B>: shows the retention time of the butyryl-CoA standard that corresponds to 2.9 minutes.</P><P><B>Figure 2</B>: Showing that at the time of 2.90 minutes when the butyryl-CoA standard is retained, (top) nothing is shown for the BUK/PTB pathway (bottom) </P><P class="space20">Compared to the control (Figure 3), there are no obvious differences.</P><P><B>Figure 3</B>: Showing that at the time of 2.90 minutes when the butyryl-CoA standard is retained, (top) nothing is shown for the BUK/PTB pathway control (bottom)</P><P class="space20">Figure 4 shows the results for the six-gene pathway compared to the butyryl-CoA standard.</P><P><B>Figure 4</B>: showing the THL/BHBD/CRO/BHBD/BCD/EFb/EFa HPLC retention time for butyryl-CoA at 2.90 minutes (botton) when the butyryl-CoA standard is retained (top). </P><P class="space20">From Figure 4 (bottom) it can be seen that a small peak forms just after 2.90 minutes but this is not significant. </P><P class="space20">From this preliminary experiment we conclude that the next step is to find the optimum conditions for growth and induction based on both the amount and concentration of butyric acid/sodium acetate used and to use mass spectrometry to correlate the exact mass of butyryl-CoA from the BUK/PTB pathway to the butyryl-CoA standard. We would also need to check each of the individual genes in the pathway to check that they are functional and also determine if the pathway is being blocked at any intermediate step.</P><H2 class="title1" id="ref">References</H2><P>1.Mattam AJ, Yazdani SS. Engineering E. coli strain for conversion of short chain fatty acids to bioalcohols. Biotechnology for Biofuels 2013, 6:128. 2013; 6(128): p. 1-11. </P><P>2. Duncan S, Barcenilla A, Stewart C, Pryde S, Flint H. Acetate Utilization and Butyryl Coenzyme A (CoA):Acetate-CoA Transferase in Butyrate-Producing Bacteria from the Human Large Intestine. Applied and Environmental Microbiology. 2002;68(10):5186-5190.</P><P>3. Vadali R. Cofactor engineering of intracellular CoA/acetyl-CoA and its effect on metabolic flux redistribution in Escherichia coli. Metabolic Engineering. 2004;6(2):133-139.</P></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="pre4"><DIV class="row"><DIV class="col-md-12 space50"><H2 class="title1">Production of butyryl-CoA in <I>E. coli</I> &quot;Nissle&quot; (Mutaflor®) in gut conditions</H2><H4 class="title2">AIM:</H4><P class="space30">To determine if the THL/BHBD/CRO/BCD/EFb/EFa (Pathway II) BioBrick (<A href="http://parts.igem.org/Part:BBa_K1618021" style="color:#002bb8;">BBa_K1618021</A>) would produce butyryl CoA in a known probiotic strain when growing in simulated gut conditions.</P><H4 class="title2">METHOD:</H4><P class="space20"><A href="http://parts.igem.org/Part:BBa_K1618001" style="color:#002bb8;">BBa_K1618021</A> containing the coding sequences from THL/BHBD/CRO/BCD/EFb/EFa (Pathway II) under control of the LacI promoter was transformed into <I>E. coli Nissle</I> (1917) kindly provided to us by Ardeynapharm. We made the cells competent using a protocol from the Penn iGEM 2012 team. 
Once transformed with our construct, a single colony was used to grow a liquid culture. Expression was induced with 1M IPTG and glycerol was added to a final concentration of 15%. The cells were then grown overnight at 37 °C. 
The following day the cells were incubated for 8h in a faecal bacterial “cocktail” kindly provided by Dr. Arjan Narbad at the UK Institute of Food Research. The bacterial cocktail contained the following strains: <I>Enterococcus faecalis</I> GB 122, Bacteroides fragilis NCTC 9343, <I>Bacteroides ovatus</I> V975, <I>Bifidobacterium bifidum</I> NCIB 8807, <I>E. coli</I> K12 MG1655, <I>Clostridium perfringens</I> DSM 11780 and <I>Lactobacillus gasseri</I> DSM 20243. Growth media for this experiment (adjusted to pH 7.0 using 1M HCl and boiled under oxygen-free nitrogen for 15 minutes to provide anoxic conditions) were also provided to us by Dr. Arjan Narbad. 
Following an 8 hour incubation the cells were centrifuged for 4 minutes and the supernatant was filter-sterilized. The supernatant was then analyzed using the Liquid Chromatograph/Mass Spectrometer protocol.</P><H4 class="title2">RESULTS:</H4><P><B>Figure 1</B>: In non-induced (no IPTG added) control, no peak corresponding to butyryl coA can be seen.</P><P><B>Figure 2</B>: In the induced sample there is a peak that does not correspond to the standard. It is unlikely that this peak is butyryl CoA. Further investigations are needed to determine if this might be a derivative or if this is experimental noise.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><SECTION style="padding:50px 0px; background: #f4f4f4;" class="arrow_box-test"><DIV class="container"><DIV class="row"><DIV class="col-md-12 text-center"><H2 class="gray">MANY THANKS TO OUR SPONSORS</H2></DIV></DIV></DIV></SECTION><FOOTER><DIV class="container"><DIV class="row"><DIV class="col-md-4"><H3>Useful Links</H3><UL class="f-menu space40"><LI><A href="/Team:NRP-UEA-Norwich">Home</A></LI><LI><A href="https://2015.igem.org/Main_Page">iGEM 2015</A></LI></UL></DIV><DIV class="col-md-4"><H3>Contact Us</H3><UL class="f-menu space40"><LI>nrpuea.igem2015@gmail.com</LI><LI>Norwich Research Park,</LI><LI>Colney,</LI><LI>Norwich, NR4 7UH, UK.</LI></UL></DIV><DIV class="col-md-4"><DIV class="f-text"><P class="fsize18 space10">We are the NRP-UEA-Norwich 2015 iGEM Team. </P></DIV></DIV></DIV><DIV class="row space100"><DIV class="col-md-12 text-center"><H4 class="dark-gray">Designed and developed by the NRP-UEA-Norwich iGEM Team</H4></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>