<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Missouri_Rolla_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Missouri Rolla/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content_wrapper"><DIV id="menu"><A href="https://2016.igem.org/Team:Missouri_Rolla">Home</A><A href="https://2016.igem.org/Team:Missouri_Rolla/Project">Project</A><A href="https://2016.igem.org/Team:Missouri_Rolla/Engagement">Human Practices</A><A href="https://2016.igem.org/Team:Missouri_Rolla/Notebook">Notebook</A><A href="https://2016.igem.org/Team:Missouri_Rolla/Team">Team</A><A href="https://2016.igem.org/Team:Missouri_Rolla/Safety">Safety</A><A href="https://2016.igem.org/Team:Missouri_Rolla/Attributions">Attributions</A></DIV><DIV class="mstigem"><H1>Joe Microbe’s Self-Service Notebook</H1><P>Many of our standard procedures can be found in our <A href="https://2016.igem.org/File:T--Missouri_Rolla--LTM_2_5.pdf">Lab Training Manual</A>. Find more information about our oligos in the <A href="https://static.igem.org/mediawiki/2016/5/5c/T--Missouri_Rolla--Oligo_Inventory.xlsx">inventory</A>.</P><DIV class="mstigem"><DIV id="nb-accordion"><DIV class="within-accordion"><H3>18 October 2016</H3><DIV><H4 class="h4left"><B>Kira Buckowing, Mikayla Tessmer, Bryan Tracy, and Matt Napoli</B></H4><H4 class="h4right"><B>Start: 5:30pm</B></H4><H4><B>Restriction Digests and Gel</B></H4><B>Purpose:</B> To check the results from minipreps.<H4><B>Protocol:</B></H4> Restriction digests were run using the thermocycler to heat kill after cutting at E and P. <H5>10/18 D1-D8</H5><TABLE><TBODY><TR><TD>2.5 μL</TD><TD>10X Tango buffer (2X total)</TD></TR><TR><TD>20.5 μL</TD><TD>10/16 MP1-MP8</TD></TR><TR><TD>1 μL</TD><TD>EcorRI</TD></TR><TR><TD>1 μL</TD><TD>PstI</TD></TR></TBODY></TABLE>LTM Ed. 2 Gel
A 1% agarose gel was run at 130 V for ~45 minutes:
<H5>10/18 Gel</H5><TABLE><TBODY><TR><TD>1</TD><TD>10 μL 2-log purple DNA ladder</TD></TR><TR><TD>2</TD><TD>10uL 10/18 D1</TD></TR><TR><TD>3</TD><TD>30 μL 10/18 D2</TD></TR><TR><TD>4</TD><TD>30 μL 10/18 D3</TD></TR><TR><TD>5</TD><TD>30 μL 10/18 D4</TD></TR><TR><TD>6</TD><TD>30 μL 10/18 D5 </TD></TR><TR><TD>7</TD><TD>30 μL 10/18 D6 </TD></TR><TR><TD>8</TD><TD>30 μL 10/18 D7 </TD></TR><TR><TD>9</TD><TD>30 μL 10/18 D8 </TD></TR><TR><TD>10</TD><TD>10 μL 2-log purple DNA ladder</TD></TR></TBODY></TABLE><H4><B>Stop: 10:00pm</B></H4><H4><B>Results:</B></H4><TABLE><TBODY><TR><TD>Sample</TD><TD>ng/μL</TD><TD>260/280</TD></TR><TR><TD>10/17 MP1</TD><TD>60.5</TD><TD>1.83</TD></TR><TR><TD>10/17 MP2</TD><TD>27.9</TD><TD>1.73</TD></TR><TR><TD>10/17 MP3</TD><TD>53.9</TD><TD>1.93</TD></TR><TR><TD>10/17 MP4</TD><TD>46.6</TD><TD>1.81</TD></TR><TR><TD>10/17 MP5</TD><TD>63.5</TD><TD>1.83</TD></TR><TR><TD>10/17 MP6</TD><TD>35.2</TD><TD>1.85</TD></TR><TR><TD>10/17 MP7</TD><TD>16.2</TD><TD>1.97</TD></TR><TR><TD>10/17 MP8</TD><TD>77.2</TD><TD>1.72</TD></TR></TBODY></TABLE><H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>10/18 D1</TD><TD>10/17 MP1</TD><TD>Nothing</TD></TR><TR><TD>10/18 D2</TD><TD>10/17 MP2</TD><TD>Nothing</TD></TR><TR><TD>10/18 D3</TD><TD>10/17 MP3</TD><TD>Nothing</TD></TR><TR><TD>10/18 D4</TD><TD>10/17 MP4</TD><TD>Nothing</TD></TR><TR><TD>10/18 D5</TD><TD>10/17 MP5</TD><TD>Nothing</TD></TR><TR><TD>10/18 D6</TD><TD>10/17 MP6</TD><TD>Nothing</TD></TR><TR><TD>10/18 D7</TD><TD>10/17 MP7</TD><TD>Nothing</TD></TR><TR><TD>10/18 D8</TD><TD>10/17 MP8</TD><TD>Nothing</TD></TR></TBODY></TABLE><H4><B>Next: </B></H4>Nothing</DIV></DIV><DIV class="within-accordion"><H3>17 October 2016</H3><DIV><H4 class="h4left"><B>Kira Buckowing, Paige Deirkes, Kent Gorday, Mikayla Tessmer, Bryan Tracy, and Matt Napoli</B></H4><H4 class="h4right"><B>Start: 4:45pm</B></H4><H4><B>Minipreps</B></H4><B>Purpose:</B> To check the results from overnight broth culture growth.<H4><B>Protocol:</B></H4> Minipreps were performed using the kitless Merlin protocol, final elutions done with 100uL TE buffer.
<H4><B>Stop: 8:30pm</B></H4><H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>10/17 MP1</TD><TD>10/16 BC5</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR><TR><TD>10/17 MP2</TD><TD>10/16 BC6</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR><TR><TD>10/17 MP3</TD><TD>10/16 BC7</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR><TR><TD>10/17 MP4</TD><TD>10/16 BC8</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR><TR><TD>10/17 MP5</TD><TD>10/16 BC9</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR><TR><TD>10/17 MP6</TD><TD>10/16 BC10</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR><TR><TD>10/17 MP7</TD><TD>10/16 BC12</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR><TR><TD>10/17 MP8</TD><TD>10/16 BC13</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR></TBODY></TABLE><H4><B>Next: Check concentration of miniprep products, then digest and run a gel to check the DNA. </B></H4></DIV></DIV><DIV class="within-accordion"><H3>16 October 2016</H3><DIV><H4 class="h4left"><B>Kira Buckowing</B></H4><H4 class="h4right"><B>Start: 12:00pm</B></H4><H4><B>Chemical Transformations</B></H4><B>Purpose:</B> To check on the most recent work and get the wanted ocimene synthase in a chlor resistance backbone.
<H4><B>Protocol:</B></H4>
 HiFi assemblies were set up as per the NEB protocol and incubated in ther thermocylcler for 15 minutes at 50 C.
<H5>10/16 HiFi 1</H5><TABLE><TBODY><TR><TD>9 uL</TD><TD>10/14 D1</TD></TR><TR><TD>1 uL</TD><TD>10/14 D4</TD></TR><TR><TD>10 uL</TD><TD>HiFi Master Mix</TD></TR></TBODY></TABLE><H5>10/16 HiFi 2</H5><TABLE><TBODY><TR><TD>9 uL</TD><TD>10/14 D2</TD></TR><TR><TD>1 uL</TD><TD>10/14 D4</TD></TR><TR><TD>10 uL</TD><TD>HiFi Master Mix</TD></TR></TBODY></TABLE>

14 broth Cultures were set up from colony growth on the 10/15 chemical transformation plates.

Chemical transformations were performed using the chemically competent cells from NEB and adding either 2 uL or 6 uL of the appropriate HiFi assembly product, noted in the products section below.
<H4><B>Stop: 3:30pm</B></H4><H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>10/16 CT1 100 uL</TD><TD>10/16 HiFi 1</TD><TD>Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone, 2 uL added</TD></TR><TR><TD>10/16 CT1 250 uL</TD><TD>10/16 HiFi 1</TD><TD>Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone, 2 uL added</TD></TR><TR><TD>10/16 CT2 100 uL</TD><TD>10/16 HiFi 2</TD><TD>Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone, 2 uL added</TD></TR><TR><TD>10/16 CT2 250 uL</TD><TD>10/16 HiFi 2</TD><TD>Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone, 2 uL added</TD></TR><TR><TD>10/16 CT3 100 uL</TD><TD>10/16 HiFi 1</TD><TD>Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone, 6 uL added</TD></TR><TR><TD>10/16 CT3 250 uL</TD><TD>10/16 HiFi 1</TD><TD>Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone, 6 uL added</TD></TR><TR><TD>10/16 CT4 100 uL</TD><TD>10/16 HiFi 2</TD><TD>Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone, 6 uL added</TD></TR><TR><TD>10/16 CT4 250 uL</TD><TD>10/16 HiFi 2</TD><TD>Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone, 6 uL added</TD></TR><TR><TD>10/16 HiFi 1</TD><TD>10/14 D1 and D4</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR><TR><TD>10/16 HiFi 2</TD><TD>10/14 D2 and D4</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR><TR><TD>10/16 BC1</TD><TD>10/15 CT2 100 uL</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR><TR><TD>10/16 BC2</TD><TD>10/15 CT2 250 uL</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR><TR><TD>10/16 BC3</TD><TD>10/15 CT2 250 uL</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR><TR><TD>10/16 BC4</TD><TD>10/15 CT2 250 uL</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR><TR><TD>10/16 BC5</TD><TD>10/15 CT2 250 uL</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR><TR><TD>10/16 BC6</TD><TD>10/15 CT4 250 uL</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR><TR><TD>10/16 BC7</TD><TD>10/15 CT4 250 uL</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR><TR><TD>10/16 BC8</TD><TD>10/15 CT4 250 uL</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR><TR><TD>10/16 BC9</TD><TD>10/15 CT4 250 uL</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR><TR><TD>10/16 BC10</TD><TD>10/15 CT4 250 uL</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR><TR><TD>10/16 BC11</TD><TD>10/15 CT4 250 uL</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR><TR><TD>10/16 BC12</TD><TD>10/15 CT4 250 uL</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR><TR><TD>10/16 BC13</TD><TD>10/15 CT4 250 uL</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR><TR><TD>10/16 BC14</TD><TD>10/15 CT4 250 uL</TD><TD>Hopefully the assembled product of ocimene in the iGEM shipping backbone</TD></TR></TBODY></TABLE><H4><B>Next: Minipreps of any growth in the broth cultures and checking the plates to see if anything grows. </B></H4></DIV></DIV><DIV class="within-accordion"><H3>15 October 2016</H3><DIV><H4 class="h4left"><B>Kira Buckowing</B></H4><H4 class="h4right"><B>Start: 8:00pm</B></H4><H4><B>Chemical Transformations</B></H4><B>Purpose:</B> To check on the most recent work and get the wanted ocimene synthase in a chlor resistance backbone.
<H4><B>Protocol:</B></H4>

Chemical transformations were performed using the chemically competent cells from NEB and adding 5 uL of the appropriate ligation product.
<H4><B>Stop: 10:20pm</B></H4><H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>10/15 CT1 100 uL</TD><TD>10/14 L1</TD><TD>Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone</TD></TR><TR><TD>10/15 CT1 250 uL</TD><TD>10/14 L1</TD><TD>Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone</TD></TR><TR><TD>10/15 CT2 100 uL</TD><TD>10/14 L2</TD><TD>Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone</TD></TR><TR><TD>10/15 CT2 250 uL</TD><TD>10/14 L2</TD><TD>Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone</TD></TR><TR><TD>10/15 CT3 100 uL</TD><TD>10/14 L3</TD><TD>Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone</TD></TR><TR><TD>10/15 CT3 250 uL</TD><TD>10/14 L3</TD><TD>Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone</TD></TR><TR><TD>10/15 CT4 100 uL</TD><TD>10/14 L4</TD><TD>Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone</TD></TR><TR><TD>10/15 CT4 250 uL</TD><TD>10/14 L4</TD><TD>Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone</TD></TR></TBODY></TABLE><H4><B>Next: HiFi assembly to cover all bases and give other options for the wanted assembled product and checking the plates to see if anything grows. </B></H4></DIV></DIV><DIV class="within-accordion"><H3>14 October 2016</H3><DIV><H4 class="h4left"><B>Kira Buckowing</B></H4><H4 class="h4right"><B>Start: 5:00pm</B></H4><H4><B>Gel to verify 10/13 results and Ligations of ocimene and the pSB13C shipping backbone.</B></H4><B>Purpose:</B> To check on the most recent work and get the wanted ocimene synthase in a chlor resistance backbone.
<H4><B>Protocol:</B></H4>
LTM Ed. 2 Gel
A 1% agarose gel was run at 130 V for ~45 minutes:
<H5>10/14 Gel</H5><TABLE><TBODY><TR><TD>1</TD><TD>10 μL 2-log purple DNA ladder</TD></TR><TR><TD>2</TD><TD>10/13 1</TD></TR><TR><TD>3</TD><TD>10/13 2</TD></TR><TR><TD>4</TD><TD>10/13 A2</TD></TR><TR><TD>5</TD><TD>10/13 A8</TD></TR><TR><TD>6</TD><TD>10/13 A8</TD></TR><TR><TD>5</TD><TD>10/14 A4</TD></TR><TR><TD>7</TD><TD>10/13 A5</TD></TR><TR><TD>8</TD><TD>10/13 A6</TD></TR><TR><TD>9</TD><TD>10/13 A7</TD></TR><TR><TD>10</TD><TD>10 μL 2-log purple DNA ladder</TD></TR></TBODY></TABLE>
Digests from 10/11 were NanoDropped to determine concentrations. 

Restriction digests were prepared, then incubated and heat-killed in the thermocycler:
<H5>10/14 D1</H5><TABLE><TBODY><TR><TD>2.5 μL</TD><TD>MilliQ water</TD></TR><TR><TD>2.5 μL</TD><TD>10X Tango buffer (2X total)</TD></TR><TR><TD>18 μL</TD><TD>10/11 G4</TD></TR><TR><TD>1 μL</TD><TD>EcoRI</TD></TR><TR><TD>1 μL</TD><TD>PstI</TD></TR></TBODY></TABLE><H5>10/14 D2</H5><TABLE><TBODY><TR><TD>2.5 μL</TD><TD>MilliQ water</TD></TR><TR><TD>2.5 μL</TD><TD>10X Tango buffer (2X total)</TD></TR><TR><TD>18 μL</TD><TD>10/11 G4</TD></TR><TR><TD>1 μL</TD><TD>EcoRI</TD></TR><TR><TD>1 μL</TD><TD>PstI</TD></TR></TBODY></TABLE><H5>10/14 D3</H5><TABLE><TBODY><TR><TD>2.5 μL</TD><TD>10X Tango buffer (2X total)</TD></TR><TR><TD>20.5 μL</TD><TD>10/11 G1</TD></TR><TR><TD>1 μL</TD><TD>EcoRI</TD></TR><TR><TD>1 μL</TD><TD>PstI</TD></TR></TBODY></TABLE><H5>10/14 D4</H5><TABLE><TBODY><TR><TD>12.5 μL</TD><TD>MilliQ water</TD></TR><TR><TD>2.5 μL</TD><TD>10X Tango buffer (2X total)</TD></TR><TR><TD>8 μL</TD><TD>pSB13C iGEM Kit DNA</TD></TR><TR><TD>1 μL</TD><TD>EcoRI</TD></TR><TR><TD>1 μL</TD><TD>PstI</TD></TR></TBODY></TABLE> 
 Ligations were set up for combinations of each digested backbone and the two versions of the ocimene synthase gene.
<H5>10/14 L1</H5><TABLE><TBODY><TR><TD>9 uL</TD><TD>10/14 D1</TD></TR><TR><TD>2 uL</TD><TD>10/14 D3</TD></TR><TR><TD>1 uL</TD><TD>Ligase Buffer</TD></TR><TR><TD>1 uL</TD><TD>Ligase </TD></TR></TBODY></TABLE><H5>10/14 L2</H5><TABLE><TBODY><TR><TD>9 uL</TD><TD>10/14 D2</TD></TR><TR><TD>2 uL</TD><TD>10/14 D3</TD></TR><TR><TD>1 uL</TD><TD>Ligase Buffer</TD></TR><TR><TD>1 uL</TD><TD>Ligase </TD></TR></TBODY></TABLE><H5>10/14 L3</H5><TABLE><TBODY><TR><TD>9 uL</TD><TD>10/14 D1</TD></TR><TR><TD>2 uL</TD><TD>10/14 D4</TD></TR><TR><TD>1 uL</TD><TD>Ligase Buffer</TD></TR><TR><TD>1 uL</TD><TD>Ligase </TD></TR></TBODY></TABLE><H5>10/14 L4</H5><TABLE><TBODY><TR><TD>9 uL</TD><TD>10/14 D2</TD></TR><TR><TD>2 uL</TD><TD>10/14 D4</TD></TR><TR><TD>1 uL</TD><TD>Ligase Buffer</TD></TR><TR><TD>1 uL</TD><TD>Ligase </TD></TR></TBODY></TABLE><H4><B>Stop: 9:20pm</B></H4><H4><B>Results:</B></H4><TABLE><TBODY><TR><TD>Sample</TD><TD>ng/μL</TD><TD>260/280</TD></TR><TR><TD>10/11 GE1</TD><TD>16</TD><TD>1.79</TD></TR><TR><TD>10/11 GE3</TD><TD>14.05</TD><TD>1.75</TD></TR><TR><TD>10/11 GE4</TD><TD>12.5</TD><TD>1.74</TD></TR></TBODY></TABLE><H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>10/14 D1</TD><TD>10/11 G4</TD><TD>Ocimene gel extraction digested with E and P</TD></TR><TR><TD>10/14 D2</TD><TD>Ocimene gBlock from IDT</TD><TD>ocimene synthase digested with E and P</TD></TR><TR><TD>10/14 D3</TD><TD>10/11 G1</TD><TD>pSB13C backbone digested with E and P</TD></TR><TR><TD>10/11 G4</TD><TD>pBSIC3 from iGEM Kit DNA</TD><TD>pSB1C3 shipping backbone digested with E and P</TD></TR><TR><TD>10/11 L1</TD><TD>10/11 D1 and D3</TD><TD>Hopefully ocimene synthase in the iGEM shipping backbone.</TD></TR><TR><TD>10/11 L2</TD><TD>10/11 D2 and D3</TD><TD>Hopefully ocimene synthase in the iGEM shipping backbone.</TD></TR><TR><TD>10/11 L3</TD><TD>10/11 D1 and D4</TD><TD>Hopefully ocimene synthase in the iGEM shipping backbone.</TD></TR><TR><TD>10/11 L4</TD><TD>10/11 D2 and D4</TD><TD>Hopefully ocimene synthase in the iGEM shipping backbone.</TD></TR></TBODY></TABLE><H4><B>Next: Chemical transformations of the ligations to see if anything grows. </B></H4></DIV></DIV><DIV class="within-accordion"><H3>13 October 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday, Mikayla Tessmer, Bryan Tracy, and Matt Napoli</B></H4><H4 class="h4right"><B>Start: 4:50 pm</B></H4><H4><B>Final amplification of <I>LeupA</I> and <I>LeupB</I></B></H4><B>Purpose:</B> To clone <I>LeupA</I> and <I>LeupB</I> into pSB1C3<H4><B>Protocol:</B></H4>Two PCRs were set up and left to react in thermocycler with the same program:

<H5>10/13 A1 (9/27 A2, <I>LeupA</I>)</H5><TABLE><TBODY><TR><TD>7 μL</TD><TD>MilliQ water</TD></TR><TR><TD>10 μL</TD><TD>2x Q5 Master Mix</TD></TR><TR><TD>1 μL</TD><TD>30 ng 2/25 3</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG42</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG43</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE><H5>10/13 A2  (<I>LeupA</I> - control)</H5><TABLE><TBODY><TR><TD>8 μL</TD><TD>MilliQ water</TD></TR><TR><TD>10 μL</TD><TD>2x Q5 Master Mix</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG42</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG43</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>8</TD></TR><TR><TD>57c</TD><TD>22</TD></TR><TR><TD>72c</TD><TD>163</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X30</P>

Two more PCRs were set up and left to react in thermocycler with the same program:

<H5>10/13 A3 (9/27 A4, <I>LeupB</I>)</H5><TABLE><TBODY><TR><TD>7 μL</TD><TD>MilliQ water</TD></TR><TR><TD>10 μL</TD><TD>2x Q5 Master Mix</TD></TR><TR><TD>1 μL</TD><TD>30 ng 2/25 2</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG44</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG45</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE><H5>10/13 A4 (<I>LeupB</I> - control)</H5><TABLE><TBODY><TR><TD>8 μL</TD><TD>MilliQ water</TD></TR><TR><TD>10 μL</TD><TD>2x Q5 Master Mix</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG44</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG45</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>8</TD></TR><TR><TD>57c</TD><TD>22</TD></TR><TR><TD>72c</TD><TD>45</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X30</P>

Two PCR clean-ups were performed using Macherey-Nagel NucleoSpin kit, eluting with 20 μL buffer.

Two more PCRs were set up and left to react in thermocycler with the same program:
<H5>10/13 A5 (<I>LeupA</I> + RS)</H5><TABLE><TBODY><TR><TD>10 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>8 μL</TD><TD>10/13 1</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG4</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG5</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE><H5>10/13 A6 (- control)</H5><TABLE><TBODY><TR><TD>8 μL</TD><TD>MilliQ water</TD></TR><TR><TD>10 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG4</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG5</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>8</TD></TR><TR><TD>62c</TD><TD>22</TD></TR><TR><TD>72c</TD><TD>160</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X30</P>

Two more PCRs were set up and left to react in thermocycler with the same program:
<H5>10/13 A7 (<I>LeupB</I> + RS)</H5><TABLE><TBODY><TR><TD>10 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>8 μL</TD><TD>10/13 2</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG8</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG9</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE><H5>10/13 A8 (- control)</H5><TABLE><TBODY><TR><TD>8 μL</TD><TD>MilliQ water</TD></TR><TR><TD>10 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG8</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG9</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>8</TD></TR><TR><TD>62c</TD><TD>22</TD></TR><TR><TD>72c</TD><TD>42</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X30</P><H4><B>Stop: 8:00 pm</B></H4><H4><B>Products:</B></H4><TABLE><TBODY><TR><TD><B>Lable</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>10/13 A1</TD><TD>2/25 3</TD><TD><I>LeupA</I> initially amplified from <I>S. roseus</I> B-3062</TD></TR><TR><TD>10/13 A2</TD><TD><I>LeupA</I>initial amplification negative control</TD></TR><TR><TD>10/13 A3</TD><TD>2/25 2</TD><TD><I>LeupB</I> initially amplified from <I>S. roseus</I> B-1320</TD></TR><TR><TD>10/13 A4</TD><TD><I>LeupB</I> initial amplification negative control</TD></TR><TR><TD>10/13 A5</TD><TD>10/13 1</TD><TD><I>LeupA</I> amplified with RFC[10] prefix and suffix</TD></TR><TR><TD>10/13 A6</TD><TD><I>LeupA</I> RFC[10] prefix and suffix primer negative control</TD></TR><TR><TD>10/13 A7</TD><TD>10/13 2</TD><TD><I>LeupB</I> amplified with RFC[10] prefix and suffix</TD></TR><TR><TD>10/13 A8</TD><TD><I>LeupB</I> RFC[10] prefix and suffix primer negative control</TD></TR><TR><TD>10/13 1</TD><TD>10/13 A1</TD><TD><I>LeupA</I> initially amplified, PCR Clean-up</TD></TR><TR><TD>10/13 2</TD><TD>10/13 A3</TD><TD><I>LeupB</I> initially amplified, PCR Clean-up</TD></TR></TBODY></TABLE><H4><B>Next:</B></H4>Run a gel with 5 μL each: 10/13 1, 10/13 2, 10/13 A2, 10/13 A4, 10/13 A5, 10/13 A6, 10/13 A7, and 10/13 A8</DIV></DIV><DIV class="within-accordion"><H3>11 October 2016</H3><DIV><H4 class="h4left"><B>Kira Buckowing, Kent Gorday, Mikayla Tessmer, Bryan Tracy, and Matt Napoli</B></H4><H4 class="h4right"><B>Start: 5:00pm</B></H4><H4><B>Gel to verify 10/10 results and ocimene gBlock</B></H4><B>Purpose:</B> To check on the most recent pSB1C3 backbone digests and the ocimene gBlock from IDT.
<H4><B>Protocol:</B></H4>
LTM Ed. 2 Gel and Gel Extraction
A 1% agarose gel was run at 130 V for ~45 minutes:
<H5>10/11 Gel</H5><TABLE><TBODY><TR><TD>1</TD><TD>10/10 D2</TD></TR><TR><TD>2</TD><TD>10/10 D1</TD></TR><TR><TD>3</TD><TD>10 μL 2-log purple DNA ladder</TD></TR><TR><TD>4</TD><TD>10/10 D3</TD></TR><TR><TD>5</TD><TD>Ocimene gBlock</TD></TR></TBODY></TABLE>

The bands that showed up in the gel were cut out and gel extractions were performed using the Macherey-Nagel NucleoSpin kit, eluting with 30 μL buffer.
<H4><B>Stop: 8:20pm</B></H4><H4><B>Results:</B></H4><H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>10/11 G1</TD><TD>10/10 D1</TD><TD>pSB13C backbone</TD></TR><TR><TD>10/11 G2</TD><TD>10/9 D2</TD><TD>pSB13C backbone </TD></TR><TR><TD>10/11 G3</TD><TD>10/9 D3</TD><TD>pSB13C backbone</TD></TR><TR><TD>10/11 G4</TD><TD>Ocimene gBlock</TD><TD>ocimene synthase</TD></TR></TBODY></TABLE><H4><B>Notes:</B></H4>
G2 did not dissolve fully in the first step and is probably not a viable product.
<H4><B>Next:</B> NanoDrop for concentrations, then set up appropriate ligations with the ocimene from the gBlock itself and this gel extraction product. </H4></DIV></DIV><DIV class="within-accordion"><H3>10 October 2016</H3><DIV><H4 class="h4left"><B>Kira Buckowing, Bryan Tracy, and Matt Napoli</B></H4><H4 class="h4right"><B>Start: 6:00pm</B></H4><H4><B>Second digest on the 10/9 D1</B></H4><B>Purpose:</B> To get the second cut site at S without interference from the first cut site at E done previously.
<H4><B>Protocol:</B></H4>
A restriction digest was prepared, then incubated and heat-killed in the thermocycler:
<H5>10/10 D1</H5><TABLE><TBODY><TR><TD>20 μL</TD><TD>MilliQ water</TD></TR><TR><TD>2.5 μL</TD><TD>10X Tango buffer (2X total)</TD></TR><TR><TD>1.42 μL</TD><TD>10/9 D1</TD></TR><TR><TD>1 μL</TD><TD>SpeI</TD></TR></TBODY></TABLE><H4><B>Stop: 8:00pm</B></H4><H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>10/10 D1</TD><TD>10/9 D1</TD><TD>pSB13C backbone digested with E and S</TD></TR></TBODY></TABLE><H4><B>Next: Gel to verify products and extractions of the correct digested products. </B></H4></DIV></DIV><DIV class="within-accordion"><H3>9 October 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday, Mikayla Tessmer, and Bryan Tracy</B></H4><H4 class="h4right"><B>Start: 11:00 am</B></H4> <H4><B>Protocol:</B></H4>
A 0.7% agarose gel was run at 130 V:
<H5>10/9 Gel #1</H5><TABLE><TBODY><TR><TD>2</TD><TD>5 μL 2-log purple DNA ladder</TD></TR><TR><TD>3</TD><TD>5 μL 10/8 A1 + 5 μL 10/8 A2</TD></TR><TR><TD>4</TD><TD>5 μL 10/8 A5</TD></TR><TR><TD>5</TD><TD>5 μL 10/8 A6</TD></TR><TR><TD>6</TD><TD>5 μL 10/8 A7</TD></TR></TBODY></TABLE>
A 0.7% agarose gel was run at 78 V:
<H5>10/9 Gel #2</H5><TABLE><TBODY><TR><TD>2</TD><TD>5 μL 2-log purple DNA ladder</TD></TR><TR><TD>4</TD><TD>max volume 10/8 A3</TD></TR><TR><TD>6</TD><TD>max volume 10/8 A4</TD></TR></TBODY></TABLE>
Three PCRs were prepared and reacted with the same thermocycler program:
<H5>10/8 A1</H5><TABLE><TBODY><TR><TD>1 μL</TD><TD>MilliQ water</TD></TR><TR><TD>5 μL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>12.5 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>4 μL</TD><TD>10/8 GE3</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM oKG4</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM oKG5</TD></TR><TR><TD>25 μL</TD></TR></TBODY></TABLE><H5>10/8 A2</H5><TABLE><TBODY><TR><TD>1 μL</TD><TD>MilliQ water</TD></TR><TR><TD>5 μL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>12.5 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>4 μL</TD><TD>10/8 GE7</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM oKG4</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM oKG5</TD></TR><TR><TD>25 μL</TD></TR></TBODY></TABLE><H5>10/8 A4</H5><TABLE><TBODY><TR><TD>1 μL</TD><TD>MilliQ water</TD></TR><TR><TD>5 μL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>12.5 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>4 μL</TD><TD>10/8 GE4</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM oKG4</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM oKG5</TD></TR><TR><TD>25 μL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>8</TD></TR><TR><TD>66c</TD><TD>22</TD></TR><TR><TD>72c</TD><TD>160</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X30</P>
Three additional PCRs were prepared and reacted with the same thermocycler program:
<H5>10/8 A3</H5><TABLE><TBODY><TR><TD>1 μL</TD><TD>MilliQ water</TD></TR><TR><TD>5 μL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>12.5 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>4 μL</TD><TD>10/8 GE4</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM oKG8</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM oKG9</TD></TR><TR><TD>25 μL</TD></TR></TBODY></TABLE><H5>10/8 A5</H5><TABLE><TBODY><TR><TD>1 μL</TD><TD>MilliQ water</TD></TR><TR><TD>5 μL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>12.5 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>4 μL</TD><TD>10/8 GE3</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM oKG8</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM oKG9</TD></TR><TR><TD>25 μL</TD></TR></TBODY></TABLE><H5>10/8 A6</H5><TABLE><TBODY><TR><TD>1 μL</TD><TD>MilliQ water</TD></TR><TR><TD>5 μL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>12.5 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>4 μL</TD><TD>10/8 GE7</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM oKG8</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM oKG9</TD></TR><TR><TD>25 μL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>8</TD></TR><TR><TD>66c</TD><TD>22</TD></TR><TR><TD>72c</TD><TD>42</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X30</P>
Three restriction digests were prepared and reacted, then heat killed, in the thermocycler:
<H5>10/8 D1</H5><TABLE><TBODY><TR><TD>20 μL</TD><TD>MilliQ water</TD></TR><TR><TD>2.5 μL</TD><TD>10X Tango buffer</TD></TR><TR><TD>1.42 μL</TD><TD>3/24 MP1</TD></TR><TR><TD>1 μL</TD><TD>SpeI</TD></TR><TR><TD>24.92 μL</TD></TR></TBODY></TABLE><H5>10/8 D2-3</H5><TABLE><TBODY><TR><TD>16.5 μL</TD><TD>MilliQ water</TD></TR><TR><TD>5 μL</TD><TD>10X Tango buffer</TD></TR><TR><TD>1.42 μL</TD><TD>3/24 MP1</TD></TR><TR><TD>1 μL</TD><TD>EcoRI</TD></TR><TR><TD>1 μL</TD><TD>PstI</TD></TR><TR><TD>24.92 μL</TD></TR></TBODY></TABLE> <H4><B>Notes:</B></H4>
No bands were observed for 10/8 A3-4.
<H4><B>Stop: 6:00 pm</B></H4> <H4><B>Results:</B></H4><H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>10/9 A1</TD><TD>10/8 GE3</TD><TD><I>LeupA</I> with RFC[10] prefix and suffix added</TD></TR><TR><TD>10/9 A2</TD><TD>10/8 GE7</TD><TD><I>LeupA</I> with RFC[10] prefix and suffix added</TD></TR><TR><TD>10/9 A3</TD><TD>10/8 GE4</TD><TD><I>LeupB</I> with RFC[10] prefix and suffix added</TD></TR><TR><TD>10/9 A4</TD><TD>10/8 GE4</TD><TD><I>LeupA</I> with RFC[10] prefix and suffix added</TD></TR><TR><TD>10/9 A5</TD><TD>10/8 GE3</TD><TD><I>LeupB</I> with RFC[10] prefix and suffix added</TD></TR><TR><TD>10/9 A6</TD><TD>10/8 GE7</TD><TD><I>LeupB</I> with RFC[10] prefix and suffix added</TD></TR><TR><TD>10/9 D1</TD><TD>3/24 MP1</TD><TD>RFP in pSB1C3 digested SpeI</TD></TR><TR><TD>10/9 D2-3</TD><TD>3/24 MP1</TD><TD>RFP in pSB1C3 digested EcoRI &amp; PstI</TD></TR></TBODY></TABLE> </DIV></DIV><DIV class="within-accordion"><H3>8 October 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday, Mikayla Tessmer, Bryan Tracy, Matt Napoli, and Paige Dierkes</B></H4><H4 class="h4right"><B>Start: 2:00 pm</B></H4> <H4><B>Protocol:</B></H4>
Three 0.7% agarose gels were run at 78 V:
<H5>10/8 Gel #1 (loaded incorrectly)</H5><TABLE><TBODY><TR><TD>1</TD><TD>5 μL 2-log purple ladder</TD></TR><TR><TD>3</TD><TD>max volume 10/7 A1</TD></TR><TR><TD>5</TD><TD>max volume 10/7 A2</TD></TR></TBODY></TABLE><H5>10/8 Gel #2</H5><TABLE><TBODY><TR><TD>1</TD><TD>5 μL 2-log purple DNA ladder</TD></TR><TR><TD>3</TD><TD>max volume 10/7 A1</TD></TR><TR><TD>5</TD><TD>max volume 9/27 A4</TD></TR></TBODY></TABLE><H5>10/8 Gel #3</H5><TABLE><TBODY><TR><TD>2</TD><TD>5 μL 2-log purple DNA ladder</TD></TR><TR><TD>4</TD><TD>10 μL 3/3 A2</TD></TR></TBODY></TABLE>
5 gel extractions were performed using the NucleoSpin Gel and PCR Clean-up kit from gels 1, 2, and 3. <I>LeupA</I> appeared as the top band in its lane, while <I>LeupB</I> appeared as the brightest in the middle of its lane. Products were Nanodropped to determine concentration. A gel extraction was performed from gel 3 using an Amicon Ultrafree-DA filter.

A 1% agarose gel was run at 130 V:
<H5>10/8 Gel #4</H5><TABLE><TBODY><TR><TD>2</TD><TD>5 μL 2-log purple DNA ladder</TD></TR><TR><TD>4</TD><TD>max volume 10/8 GE5</TD></TR><TR><TD>6</TD><TD>max volume 10/8 GE6</TD></TR></TBODY></TABLE>
A 0.7% agarose gel was run at 130 V:
<H5>10/8 Gel #5</H5><TABLE><TBODY><TR><TD>2</TD><TD>5 μL 2-log purple DNA ladder</TD></TR><TR><TD>4</TD><TD>25 μL 10/8 GE5</TD></TR><TR><TD>6</TD><TD>10 μL 10/8 GE6</TD></TR></TBODY></TABLE>
A gel extraction

2 PCRs were prepared and run in a thermocycler with the same program:
<H5>10/8 A1-2</H5><TABLE><TBODY><TR><TD>6 μL</TD><TD>MilliQ water</TD></TR><TR><TD>12.5 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>4 μL</TD><TD>10 ng/μL <I>ociS</I> gblock</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM VF2</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM ociR</TD></TR><TR><TD>25 μL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>8</TD></TR><TR><TD>56c</TD><TD>22</TD></TR><TR><TD>72c</TD><TD>45</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X11</P>
2 PCRs were prepared and run in a thermocycler with the same program:
<H5>10/8 A3</H5><TABLE><TBODY><TR><TD>14 μL</TD><TD>MilliQ water</TD></TR><TR><TD>20 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>2 μL</TD><TD>30 ng/μL 2/25 1</TD></TR><TR><TD>2 μL</TD><TD>10 μM oKG44</TD></TR><TR><TD>2 μL</TD><TD>10 μM oKG45</TD></TR><TR><TD>40 μL</TD></TR></TBODY></TABLE><H5>10/8 A4</H5><TABLE><TBODY><TR><TD>14 μL</TD><TD>MilliQ water</TD></TR><TR><TD>20 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>2 μL</TD><TD>30 ng/μL 2/25 3</TD></TR><TR><TD>2 μL</TD><TD>10 μM oKG44</TD></TR><TR><TD>2 μL</TD><TD>10 μM oKG45</TD></TR><TR><TD>40 μL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>8</TD></TR><TR><TD>57c</TD><TD>22</TD></TR><TR><TD>72c</TD><TD>45</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X30</P>
2 PCRs were prepared and run in a thermocycler with the same program:
<H5>10/8 A5 (<I>LeupA</I> + RS)</H5><TABLE><TBODY><TR><TD>6 μL</TD><TD>MilliQ water</TD></TR><TR><TD>12.5 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>4 μL</TD><TD>10/8 GE3</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM oKG4</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM oKG5</TD></TR><TR><TD>25 μL</TD></TR></TBODY></TABLE><H5>10/8 A6 (<I>LeupA</I> + RS)</H5><TABLE><TBODY><TR><TD>6 μL</TD><TD>MilliQ water</TD></TR><TR><TD>12.5 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>4 μL</TD><TD>10/8 GE7</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM oKG4</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM oKG5</TD></TR><TR><TD>25 μL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>8</TD></TR><TR><TD>56c</TD><TD>22</TD></TR><TR><TD>72c</TD><TD>160</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X26</P>
Another PCR was prepared and run in a thermocycler:
<H5>10/8 A7 (<I>LeupB</I> + RS)</H5><TABLE><TBODY><TR><TD>6 μL</TD><TD>MilliQ water</TD></TR><TR><TD>12.5 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>4 μL</TD><TD>10/8 GE4</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM oKG8</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM oKG9</TD></TR><TR><TD>25 μL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>8</TD></TR><TR><TD>56c</TD><TD>22</TD></TR><TR><TD>72c</TD><TD>42</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE>  <H4><B>Results:</B></H4> <TABLE><TBODY><TR><TD>Sample</TD><TD>ng/μL</TD><TD>260/280</TD><TD>260/230</TD></TR><TR><TD>10/8 GE3</TD><TD>3.58</TD><TD>1.79</TD><TD>0.02</TD></TR><TR><TD>10/8 GE4</TD><TD>6.67</TD><TD>1.77</TD><TD>0.42</TD></TR><TR><TD>10/8 GE5</TD><TD>14.32</TD><TD>1.60</TD><TD>0.32</TD></TR><TR><TD>10/8 GE6</TD><TD>17.97</TD><TD>1.25</TD><TD>0.17</TD></TR></TBODY></TABLE><H4><B>Products:</B></H4>
Reset Table
<TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>10/8 GE3</TD><TD>10/7 A1 top band</TD><TD><I>LeupA</I> from <I>S. roseus</I> B-1320</TD></TR><TR><TD>10/8 GE4</TD><TD>9/27 A4 middle bright band</TD><TD><I>LeupB</I> from <I>S. roseus</I> B-1320</TD></TR><TR><TD>10/8 GE5</TD><TD>3/3 A2</TD><TD>RFP positive control for Macherey-Nagel kit</TD></TR><TR><TD>10/8 GE6</TD><TD>3/3 A2</TD><TD>RFP positive control for Amicon filters</TD></TR><TR><TD>10/8 GE7</TD><TD>10/7 A3 top band</TD><TD><I>LeupA</I> from <I>S. roseus</I> ISP-5076</TD></TR><TR><TD>10/8 A1</TD><TD><I>ociS</I> gblock</TD><TD><I>ociS</I> gblock amplified</TD></TR><TR><TD>10/8 A2</TD><TD><I>ociS</I> gblock</TD><TD><I>ociS</I> gblock amplified</TD></TR><TR><TD>10/8 A3</TD><TD>2/25 1</TD><TD><I>LeupB</I> from <I>S. roseus</I> ISP-5076</TD></TR><TR><TD>10/8 A4</TD><TD>2/25 3</TD><TD><I>LeupB</I> from <I>S. roseus</I> B-3062</TD></TR><TR><TD>10/8 A5</TD><TD>10/8 GE3</TD><TD><I>LeupA</I> with RFC[10] prefix and suffix added</TD></TR><TR><TD>10/8 A6</TD><TD>10/8 GE7</TD><TD><I>LeupA</I> with RFC[10] prefix and suffix added</TD></TR><TR><TD>10/8 A7</TD><TD>10/8 GE4</TD><TD><I>LeupB</I> with RFC[10] prefix and suffix added</TD></TR></TBODY></TABLE> </DIV></DIV><DIV class="within-accordion"><H3>7 October 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday, Mikayla Tessmer, Matt Napoli, Bryan Tracy, and Paige Dierkes</B></H4><H4 class="h4right"><B>Start: 2:30pm</B></H4> <H4><B>Protocol:</B></H4>
1.2% agarose gel was run:
<TABLE><TBODY><TR><TD>2</TD><TD>5 μL 2-log purple DNA ladder</TD></TR><TR><TD>4</TD><TD>5 μL 10/6 A1</TD></TR><TR><TD>6</TD><TD>5 μL 10/6 A2</TD></TR><TR><TD>8</TD><TD>5 μL 2-log purple DNA ladder</TD></TR></TBODY></TABLE>
3 PCRs were set up:
<H5>10/7 A1-3</H5><TABLE><TBODY><TR><TD>14 μL</TD><TD>MilliQ water</TD></TR><TR><TD>20 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>2 μL</TD><TD>30 ng/μL 2/25 1 (A1), 2 (A2), or 3 (A3)</TD></TR><TR><TD>2 μL</TD><TD>10 μM oKG42</TD></TR><TR><TD>2 μL</TD><TD>10 μM oKG43</TD></TR><TR><TD>40 μL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>8</TD></TR><TR><TD>57c</TD><TD>22</TD></TR><TR><TD>72c</TD><TD>163</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X30</P> <H4><B>Stop: 5:00 pm</B></H4> <H4><B>Results:</B></H4><H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>10/7 A1</TD><TD>2/25 1</TD><TD><I>LeupA</I> from <I>S. roseus</I> B-1320</TD></TR><TR><TD>10/7 A2</TD><TD>2/25 2</TD><TD><I>LeupA</I> from <I>S. roseus</I> B-3062</TD></TR><TR><TD>10/7 A3</TD><TD>2/25 3</TD><TD><I>LeupA</I> from <I>S. roseus</I> ISP-5076</TD></TR></TBODY></TABLE> </DIV></DIV><DIV class="within-accordion"><H3>6 October 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday, Bryan Tracy, and Mikayla Tessmer</B></H4><H4 class="h4right"><B>Start: 9:00 am</B></H4> <H4><B>Protocol:</B></H4>
A 0.8% agarose gel was run at 90 V:
<TABLE><TBODY><TR><TD>1</TD><TD>5 μL 2-log purple DNA ladder</TD></TR><TR><TD>2</TD><TD>5 μL 10/5 A1</TD></TR><TR><TD>3</TD><TD>5 μL 10/5 A2</TD></TR><TR><TD>4</TD><TD>5 μL 10/5 A3</TD></TR><TR><TD>5</TD><TD>5 μL 10/5 A4</TD></TR><TR><TD>6</TD><TD>5 μL 2-log purple DNA ladder</TD></TR><TR><TD>7</TD><TD>5 μL 10/5 A9</TD></TR><TR><TD>8</TD><TD>5 μL 10/5 A10</TD></TR><TR><TD>9</TD><TD>5 μL 10/5 A11</TD></TR><TR><TD>10</TD><TD>5 μL 10/5 A12</TD></TR></TBODY></TABLE>
Two PCRs were reacted in the thermocycler for the amplification of <I>ociS</I>:
<H5>10/6 A1-2</H5><TABLE><TBODY><TR><TD>9 μL</TD><TD>MilliQ water</TD></TR><TR><TD>12.5 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1 μL</TD><TD>10 ng/μL <I>ociS</I> gblock</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM VF2</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM ociR</TD></TR><TR><TD>25 μL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>TIme (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>8</TD></TR><TR><TD>60c</TD><TD>22</TD></TR><TR><TD>72c</TD><TD>44</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X11</P> <H4><B>Notes:</B></H4>
10/5 A1-16 were done with mixed <I>LeupA</I> and <I>LeupB</I> primer pairs rather than the proper oKG42/43 (<I>LeupA</I>) and oKG44/45 (<I>LeupB</I>) pairs.
<H4><B>Stop: 11:00 am</B></H4> <H4><B>Results:</B></H4><H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>10/6 A1</TD><TD><I>ociS</I> gblock</TD><TD><I>ociS</I> gblock amplified</TD></TR><TR><TD>10/6 A2</TD><TD><I>ociS</I> gblock</TD><TD><I>ociS</I> gblock amplified</TD></TR></TBODY></TABLE> </DIV></DIV><DIV class="within-accordion"><H3>5 October 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday, Kurt Studer, Bryan Tracy, and Sam Greaney</B></H4><H4 class="h4right"><B>Start: 5:30 pm</B></H4> <H4><B>Gradient PCRs of Initial <I>LeupA</I> and <I>LeupB</I></B></H4><B>Purpose:</B> To determine the best PCR conditions for amplifying <I>LeupA</I> and <I>LeupB</I> from the <I>Streptomyces roseus</I> genome
<H4><B>Protocol:</B></H4>
Sixteen PCRs were prepared and reacted with two different thermocycler programs, each containing a gradient for the annealing step:
<H5>10/5 A1-A8</H5><TABLE><TBODY><TR><TD>8.8 μL</TD><TD>MilliQ water</TD></TR><TR><TD>12.5 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1.25 μL</TD><TD>37.5 ng 2/25 2</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM oKG42</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM oKG44</TD></TR><TR><TD>25 μL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>8</TD></TR><TR><TD>gradient (A1/A5 63.0, A2/A6 60.6, A3/A7 53.3, A4/A8 51.0)c</TD><TD>22</TD></TR><TR><TD>72c</TD><TD>163</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X30</P><H5>10/5 A9-A16</H5><TABLE><TBODY><TR><TD>8.8 μL</TD><TD>MilliQ water</TD></TR><TR><TD>12.5 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1.25 μL</TD><TD>37.5 ng 2/25 2</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM oKG43</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM oKG45</TD></TR><TR><TD>25 μL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>8</TD></TR><TR><TD>gradient (A9/A13 63.0, A10/A14 60.6, A11/A15 53.3, A12/A16 51.0)c</TD><TD>22</TD></TR><TR><TD>72c</TD><TD>45</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X31</P> <H4><B>Notes:</B></H4>
10/5 A1-A8 were restarted in the thermocycler during the first cycle due to a programming error. 10/5 A9-A16 were accidentally cycled 31 instead of 30 times. The original intention was to run each set of conditions with and without GC Enhancer, but none was added.
<H4><B>Stop: 7:00 pm</B></H4> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>10/5 A1-A8</TD><TD>2/25 2</TD><TD>Gradient PCRs of <I>LeupA</I> from <I>S. roseus</I> B-1320 -GCE</TD></TR><TR><TD>10/5 A9-A16</TD><TD>2/25 2</TD><TD>Gradient PCRs of <I>LeupB</I> from <I>S. roseus</I> B-1320 -GCE</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Run a gel of 10/5 PCRs, and gel purify <I>LeupA</I> and <I>LeupB</I></DIV></DIV><DIV class="within-accordion"><H3>27 September 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: 6:00 pm</B></H4> <H4><B>Gel Electrophoresis of New<I>LeupA</I> and <I>LeupB</I> PCRs</B></H4><B>Purpose:</B> To verify success of 9/27 PCRs
<H4><B>Protocol:</B></H4>
A 1% agarose gel was run with each 9/27 PCR:
<TABLE><TBODY><TR><TD>3</TD><TD>7 μL 2-log purple DNA ladder</TD></TR><TR><TD>4</TD><TD>5 μL 9/27 A1</TD></TR><TR><TD>5</TD><TD>5 μL 9/27 A2</TD></TR><TR><TD>6</TD><TD>5 μL 9/27 A3</TD></TR><TR><TD>7</TD><TD>5 μL 9/27 A4</TD></TR></TBODY></TABLE> <H4><B>Stop: 7:00 pm</B></H4> <H4><B>Results:</B></H4><H4><B>Next:</B></H4>
PCR with 16S primers from 2/25 1 using Q5 to check Q5 Master Mix and the remaining genomic DNA prep, with a negative control for both Q5 and Taq.

</DIV></DIV><DIV class="within-accordion"><H3>27 September 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: 12:15 pm</B></H4> <H4><B>New Initial PCRs of <I>LeupA</I> and <I>LeupB</I></B></H4><B>Purpose:</B> To amplify <I>LeupA</I> and <I>LeupB</I> from the <I>Streptomyces roseus</I> genome
<H4><B>Protocol:</B></H4> <H5>9/27 A1</H5><TABLE><TBODY><TR><TD>3 μL</TD><TD>MilliQ water</TD></TR><TR><TD>4 μL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>10 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1 μL</TD><TD>30 ng 2/25 3</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG42</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG43</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE><H5>9/27 A2</H5><TABLE><TBODY><TR><TD>7 μL</TD><TD>MilliQ water</TD></TR><TR><TD>10 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1 μL</TD><TD>30 ng 2/25 3</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG42</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG43</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>8</TD></TR><TR><TD>57c</TD><TD>22</TD></TR><TR><TD>72c</TD><TD>163</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X30</P><H5>9/27 A3</H5><TABLE><TBODY><TR><TD>3 μL</TD><TD>MilliQ water</TD></TR><TR><TD>4 μL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>10 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1 μL</TD><TD>30 ng 2/25 2</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG44</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG45</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE><H5>9/27 A4</H5><TABLE><TBODY><TR><TD>7 μL</TD><TD>MilliQ water</TD></TR><TR><TD>10 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1 μL</TD><TD>30 ng 2/25 2</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG44</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG45</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>8</TD></TR><TR><TD>57c</TD><TD>22</TD></TR><TR><TD>72c</TD><TD>45</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X30</P> <H4><B>Stop: 12:50 pm</B></H4> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>9/27 A1</TD><TD>2/25 1</TD><TD><I>LeupA</I> and flanking region amplified from <I>S. roseus</I> ISP-5076 +GCE</TD></TR><TR><TD>9/27 A2</TD><TD>2/25 1</TD><TD><I>LeupA</I> and flanking region amplified from <I>S. roseus</I> ISP-5076 -GCE</TD></TR><TR><TD>9/27 A3</TD><TD>2/25 2</TD><TD><I>LeupB</I> and flanking region amplified from <I>S. roseus</I> B-1320 +GCE</TD></TR><TR><TD>9/27 A4</TD><TD>2/25 2</TD><TD><I>LeupB</I> and flanking region amplified from <I>S. roseus</I> B-1320 -GCE</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Run a gel of 9/27 PCRs, also PCR with 16S primers from 2/25 1 using Q5 to check Q5 Master Mix and the remaining genomic DNA prep, with a negative control for both Q5 and Taq.

</DIV></DIV><DIV class="within-accordion"><H3>24 September 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: 3:00 pm</B></H4> <H4><B>Gel of Initial Leupeptin and 16S PCRs</B></H4><B>Purpose:</B> To verify success of 9/23 PCRs
<H4><B>Protocol:</B></H4>
A 0.8% agarose gel was run with 9/23 PCR products:
<H5>9/24 Gel</H5><TABLE><TBODY><TR><TD>2</TD><TD>5 μL 2-log purple DNA ladder</TD></TR><TR><TD>3</TD><TD>5 μL 9/23 A1</TD></TR><TR><TD>4</TD><TD>5 μL 9/23 A2</TD></TR><TR><TD>5</TD><TD>5 μL 9/23 A3</TD></TR><TR><TD>6</TD><TD>5 μL 9/23 A4</TD></TR><TR><TD>7</TD><TD>5 μL 9/23 A5</TD></TR><TR><TD>8</TD><TD>5 μL 9/23 A6</TD></TR></TBODY></TABLE> <H4><B>Stop: 4:30 pm</B></H4> <H4><B>Results:</B></H4><H4><B>Next:</B></H4>
PCR with 16S primers from 2/25 1 using Q5 to check Q5 Master Mix and the remaining genomic DNA prep, with a negative control for both Q5 and Taq. Try LeupA/B initial PCRs again with a lower annealing temperature (59 °C), with and without GC Enhancer.

</DIV></DIV><DIV class="within-accordion"><H3>23 September 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday, Ryan Baumann, Lucas Harper, and Kurt Studer</B></H4><H4 class="h4right"><B>Start: 2:40 pm</B></H4> <H4><B>Leupeptin and 16S PCR Attempts</B></H4><B>Purpose:</B> To amplify <I>LeupA</I> and <I>LeupB</I> from the <I>Streptomyces roseus</I> genome, and verify the presence of genomic DNA in preps from three strains while testing 16S primers
<H4><B>Protocol:</B></H4>
Two PCRs were incubated in a thermocycler after new primers were diluted to 10 μM and templates to 30 ng/μL:
<H5>9/23 A1</H5><TABLE><TBODY><TR><TD>3 μL</TD><TD>MilliQ water</TD></TR><TR><TD>4 μL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>10 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1 μL</TD><TD>30 ng 2/25 1</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG42</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG43</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE><H5>9/23 A2</H5><TABLE><TBODY><TR><TD>7 μL</TD><TD>MilliQ water</TD></TR><TR><TD>10 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1 μL</TD><TD>30 ng 2/25 1</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG42</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG43</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>8</TD></TR><TR><TD>67c</TD><TD>22</TD></TR><TR><TD>72c</TD><TD>185</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X30</P>
Two more PCRs were also reacted in a thermocycler:
<H5>9/23 A3</H5><TABLE><TBODY><TR><TD>3 μL</TD><TD>MilliQ water</TD></TR><TR><TD>4 μL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>10 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1 μL</TD><TD>30 ng 2/25 2</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG44</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG45</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE><H5>9/23 A4</H5><TABLE><TBODY><TR><TD>7 μL</TD><TD>MilliQ water</TD></TR><TR><TD>10 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1 μL</TD><TD>30 ng 2/25 2</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG44</TD></TR><TR><TD>1 μL</TD><TD>10 μM oKG45</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>8</TD></TR><TR><TD>67c</TD><TD>22</TD></TR><TR><TD>72c</TD><TD>45</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X30</P>
Two more PCRs were also reacted in a thermocycler:
<H5>9/23 A4</H5><TABLE><TBODY><TR><TD>8.2 μL</TD><TD>MilliQ water</TD></TR><TR><TD>10 μL</TD><TD>2X Taq Master Mix</TD></TR><TR><TD>1 μL</TD><TD>30 ng 2/25 2</TD></TR><TR><TD>1 μL</TD><TD>10 μM 27F</TD></TR><TR><TD>1 μL</TD><TD>10 μM 1492R</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE><H5>9/23 A5</H5><TABLE><TBODY><TR><TD>8.2 μL</TD><TD>MilliQ water</TD></TR><TR><TD>10 μL</TD><TD>2X Taq Master Mix</TD></TR><TR><TD>1 μL</TD><TD>30 ng 2/25 3</TD></TR><TR><TD>1 μL</TD><TD>10 μM 27F</TD></TR><TR><TD>1 μL</TD><TD>10 μM 1492R</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>95</TD><TD>30</TD></TR><TR><TD>95c</TD><TD>22</TD></TR><TR><TD>49c</TD><TD>40</TD></TR><TR><TD>68c</TD><TD>95</TD></TR><TR><TD>68</TD><TD>300</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X30</P>
Two broth cultures in ISP medium 1 were inoculated from frozen stock using a sterile pipette tip.
<H4><B>Notes:</B></H4>
No negative control was performed for 16S amplification, so genomic DNA preps cannot conclusively be verified. No ISP-4 or ISP-1 media is left.

Some basic information about oKG42-45 is included below:
<TABLE><TBODY><TR><TD><B>Oligo</B></TD><TD><B>Sequence</B></TD><TD><B>Correct Location</B></TD><TD><B>Mispriming E-Values</B></TD></TR><TR><TD>oKG42</TD><TD>cttcacccgaatcgatgctg</TD><TD>_1163: 93441 +</TD><TD>0.84, 0.84 +- (both far)</TD></TR><TR><TD>oKG43</TD><TD>gtggtgtgttccgtttcctg</TD><TD>_1163: 101101 -</TD></TR><TR><TD>oKG44</TD><TD>caggaaacggaacacaccac</TD><TD>_1163: 101082 +</TD></TR><TR><TD>oKG45</TD><TD>gggaaaggtgaccaggaagt</TD><TD>_1163: 102794 -</TD><TD>0.069, 0.24, 0.24, etc. ++-- (all far)</TD></TR></TBODY></TABLE> <H4><B>Stop: 5:00 pm</B></H4> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>9/25 A1</TD><TD>2/25 1</TD><TD><I>LeupA</I> and flanking region amplified from <I>S. roseus</I> ISP-5076 +GCE</TD></TR><TR><TD>9/25 A2</TD><TD>2/25 1</TD><TD><I>LeupA</I> and flanking region amplified from <I>S. roseus</I> ISP-5076 -GCE</TD></TR><TR><TD>9/25 A3</TD><TD>2/25 2</TD><TD><I>LeupB</I> and flanking region amplified from <I>S. roseus</I> B-1320 +GCE</TD></TR><TR><TD>9/25 A4</TD><TD>2/25 2</TD><TD><I>LeupB</I> and flanking region amplified from <I>S. roseus</I> B-1320 -GCE</TD></TR><TR><TD>9/25 A5</TD><TD>2/25 2</TD><TD>Portion of 16S rRNA DNA amplified from <I>S. roseus</I> B-1320</TD></TR><TR><TD>9/25 A6</TD><TD>2/25 3</TD><TD>Portion of 16S rRNA DNA amplified from <I>S. roseus</I> B-3062</TD></TR><TR><TD>9/25 BC1</TD><TD>9/6 S1</TD><TD><I>S. roseus</I> ISP-5076 from frozen stock</TD></TR><TR><TD>9/25 BC2</TD><TD>9/6 S2</TD><TD><I>S. roseus</I> B-1320 from frozen stock</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Run a gel of PCR products to decide if <I>LeupA</I> and <I>LeupB</I> PCRs should be performed from all strains

</DIV></DIV><DIV class="within-accordion"><H3>6 September 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4></DIV></DIV><DIV class="within-accordion"><H3>2 September 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4> <H4><B>Culturing of and colony PCR from <I>Streptomyces roseus</I> strains</B></H4><B>Purpose:</B> To revive <I>S. roseus</I> strains and amplify <I>LeupB</I><H4><B>Protocol:</B></H4>
Twelve colony PCRs were reacted in the thermocycler with the same program:
<H5>9/2 A1-A6</H5><TABLE><TBODY><TR><TD>70 μL</TD><TD>10 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>21 μL</TD><TD>3 μL</TD><TD>MilliQ water</TD></TR><TR><TD>28 μL</TD><TD>4 μL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>7 μL</TD><TD>1 μL</TD><TD>10 μM LeupB_F</TD></TR><TR><TD>7 μL</TD><TD>1 μL</TD><TD>10 μM LeupB_R</TD></TR><TR><TD>swirl or add 1 μL</TD><TD>colony or liquid culture template</TD></TR><TR><TD>133 μL</TD><TD>20 μL</TD></TR></TBODY></TABLE><H5>9/2 A7-A12</H5><TABLE><TBODY><TR><TD>70 μL</TD><TD>10 μL</TD><TD>2X Taq Master Mix</TD></TR><TR><TD>57.4 μL</TD><TD>8.2 μL</TD><TD>MilliQ water</TD></TR><TR><TD>2.8 μL</TD><TD>0.4 μL</TD><TD>10 μM LeupB_F</TD></TR><TR><TD>2.8 μL</TD><TD>0.4 μL</TD><TD>10 μM LeupB_R</TD></TR><TR><TD>swirl or add 1 μL</TD><TD>colony or liquid culture template</TD></TR><TR><TD>133 μL</TD><TD>20 μL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>95</TD><TD>300</TD></TR><TR><TD>95c</TD><TD>17</TD></TR><TR><TD>61c</TD><TD>25</TD></TR><TR><TD>70c</TD><TD>120</TD></TR><TR><TD>70</TD><TD>300</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X30</P>
Three streaks were performed onto ISP medium 4 using sterile pipette tips, then three ISP medium 1 tube cultures were similarly inoculated.
<H4><B>Notes:</B></H4>
Plates used for streaking were left warming in incubator and had severely dried out.
 <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>9/2 BC1</TD><TD>2/21 I1</TD><TD><I>S. roseus</I> ISP-5076 in ISP m. 1</TD></TR><TR><TD>9/2 BC2</TD><TD>2/21 I2</TD><TD><I>S. roseus</I> B-1320 in ISP m. 1</TD></TR><TR><TD>9/2 BC3</TD><TD>2/21 I3</TD><TD><I>S. roseus</I> B-3062 in ISP m. 1</TD></TR><TR><TD>9/2 I1</TD><TD>2/21 I1</TD><TD><I>S. roseus</I> ISP-5076 on ISP m. 4</TD></TR><TR><TD>9/2 I2</TD><TD>2/21 I2</TD><TD><I>S. roseus</I> B-1320 on ISP m. 4</TD></TR><TR><TD>9/2 I3</TD><TD>2/21 I3</TD><TD><I>S. roseus</I> B-3062 on ISP m. 4</TD></TR><TR><TD>9/2 A1</TD><TD>2/21 I1</TD><TD><I>LeupB</I> amplified by colony PCR from <I>S. roseus</I> ISP-5076</TD></TR><TR><TD>9/2 A2</TD><TD>2/21 I2</TD><TD><I>LeupB</I> amplified by colony PCR from <I>S. roseus</I> B-1320</TD></TR><TR><TD>9/2 A3</TD><TD>2/21 I3</TD><TD><I>LeupB</I> amplified by colony PCR from <I>S. roseus</I> B-3062</TD></TR><TR><TD>9/2 A4</TD><TD>8/24 BC1</TD><TD><I>LeupB</I> amplified by colony PCR from <I>S. roseus</I> ISP-5076</TD></TR><TR><TD>9/2 A5</TD><TD>8/24 BC3</TD><TD><I>LeupB</I> amplified by colony PCR from <I>S. roseus</I> B-3062</TD></TR><TR><TD>9/2 A6</TD><TD>LeupA_F/R negative control PCR</TD></TR><TR><TD>9/2 A7</TD><TD>2/21 I1</TD><TD><I>LeupB</I> amplified by colony PCR from <I>S. roseus</I> ISP-5076(Taq)</TD></TR><TR><TD>9/2 A8</TD><TD>2/21 I2</TD><TD><I>LeupB</I> amplified by colony PCR from <I>S. roseus</I> B-1320(Taq)</TD></TR><TR><TD>9/2 A9</TD><TD>2/21 I3</TD><TD><I>LeupB</I> amplified by colony PCR from <I>S. roseus</I> B-3062(Taq)</TD></TR><TR><TD>9/2 A10</TD><TD>8/24 BC1</TD><TD><I>LeupB</I> amplified by colony PCR from <I>S. roseus</I> ISP-5076(Taq)</TD></TR><TR><TD>9/2 A11</TD><TD>8/24 BC3</TD><TD><I>LeupB</I> amplified by colony PCR from <I>S. roseus</I> B-3062(Taq)</TD></TR><TR><TD>9/2 A12</TD><TD>LeupA_F/R negative control PCR(Taq)</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4> </DIV></DIV><DIV class="within-accordion"><H3>24 August 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: 2:30 pm</B></H4> <H4><B>Reviving <I>Streptomyces roseus</I></B></H4><B>Purpose:</B> To amplify LeupA and LeupB directly from <I>S. roseus</I> cultures with lysis in thermocycler and ensure strains of <I>S. roseus</I> remain viable.
<H4><B>Protocol:</B></H4>
Three broth cultures were inoculated from single colonies into ISP media 1 and left to incubate at roughly 26°C, 200 rpm.
<H4><B>Stop: 2:45 pm</B></H4> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>8/24 BC1</TD><TD>2/23 I1</TD><TD><I>S. roseus</I> ISP-5076 in ISP m. 1</TD></TR><TR><TD>8/24 BC2</TD><TD>2/23 I2</TD><TD><I>S. roseus</I> B-1320 in ISP m. 1</TD></TR><TR><TD>8/24 BC3</TD><TD>2/23 I3</TD><TD><I>S. roseus</I> B-3062 in ISP m. 1</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Prepare 25% glycerol stocks from tube cultures, then use a small volume of cultures as template for PCR of LeupA and LeupB with a lysis step. Also streak strains onto ISP-4 plates.

</DIV></DIV><DIV class="within-accordion"><H3>6 July 2016</H3><DIV><H4 class="h4left"><B>Kira Buckowing &amp; Kaelyn Yarbrough</B></H4><H4 class="h4right"><B>Start: 2:00pm</B></H4> <H4><B>Gradient PCR for LeupB Attempt 3 and Gel to verify results,and old ocimene attempts</B></H4><B>Purpose:</B> To try and amplify the LeupB strain around the idea that the previous 5/25 A2 was the closest candidate to what was wanted and to check on the most recent ocimene attempts available.
<H4><B>Protocol:</B></H4>
Five identical PCRs were performed with different annealing temperatures using a block gradient:
<H5>7/6 A1-A5</H5><TABLE><TBODY><TR><TD>10 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>3 μL</TD><TD>MilliQ water</TD></TR><TR><TD>4 μL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>1 μL</TD><TD>1/6.3X 2/25 1</TD></TR><TR><TD>1 μL</TD><TD>10 μM LeupB_F</TD></TR><TR><TD>1 μL</TD><TD>10 μM LeupB_R</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>105</TD></TR><TR><TD>98c</TD><TD>10</TD></TR><TR><TD>gradient (A1 67.6, A2 67.9, A3 68.2, A4 68.5, A5 68.7)c</TD><TD>25</TD></TR><TR><TD>72c</TD><TD>47</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X32</P>

LTM Ed. 2 Gel
A 1% agarose gel was run at 130 V for ~45 minutes:
<H5>7/6 Gel</H5><TABLE><TBODY><TR><TD>1</TD><TD>9/11 D2 LP</TD></TR><TR><TD>2</TD><TD>10 μL 2-log purple DNA ladder</TD></TR><TR><TD>3</TD><TD>13 μL 7/6 A1</TD></TR><TR><TD>4</TD><TD>13 μL 7/6 A2</TD></TR><TR><TD>5</TD><TD>13 μL 7/6 A3</TD></TR><TR><TD>6</TD><TD>13 μL 7/6 A4</TD></TR><TR><TD>7</TD><TD>13 μL 7/6 A5</TD></TR><TR><TD>8</TD><TD>10 μL 2-log purple DNA ladder</TD></TR><TR><TD>9</TD><TD>9/11 L1 LP</TD></TR></TBODY></TABLE><H4><B>Stop: 5:30pm</B></H4> <H4><B>Results:</B></H4><H4><B>Next:</B></H4>
Try the PCR again with 5/25 A2 as the basis for LeupB, checking everything again apparently. Ocimene things need to be completely reworked, as suspected.

</DIV></DIV><DIV class="within-accordion"><H3>30 June 2016</H3><DIV><H4 class="h4left"><B>Kira Buckowing &amp; Kaelyn Yarbrough</B></H4><H4 class="h4right"><B>Start: 11:00am</B></H4> <H4><B>Gradient PCR for LeupB Attempt 2 and Gel to verify results</B></H4><B>Purpose:</B> To try and amplify the LeupB strain around the idea that the previous 5/25 A2 was the closest candidate to what was wanted.
<H4><B>Protocol:</B></H4>
Five identical PCRs were performed with different annealing temperatures using a block gradient:
<H5>6/30 A1-A5</H5><TABLE><TBODY><TR><TD>10 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>3 μL</TD><TD>MilliQ water</TD></TR><TR><TD>4 μL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>1 μL</TD><TD>1/6.3X 2/25 1</TD></TR><TR><TD>1 μL</TD><TD>10 μM LeupB_F</TD></TR><TR><TD>1 μL</TD><TD>10 μM LeupB_R</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>105</TD></TR><TR><TD>98c</TD><TD>10</TD></TR><TR><TD>gradient (A1 67, A2 67.4, A3 67.6, A4 67.8, A5 68)c</TD><TD>25</TD></TR><TR><TD>72c</TD><TD>47</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X32</P>

LTM Ed. 2 Gel
A 1% agarose gel was run at 130 V for ~45 minutes:
<H5>6/30 Gel</H5><TABLE><TBODY><TR><TD>1</TD><TD>10 μL 2-log purple DNA ladder</TD></TR><TR><TD>2</TD><TD>13 μL 6/30 A1</TD></TR><TR><TD>3</TD><TD>13 μL 6/30 A2</TD></TR><TR><TD>4</TD><TD>13 μL 6/30 A3</TD></TR><TR><TD>5</TD><TD>13 μL 6/30 A4</TD></TR><TR><TD>6</TD><TD>13 μL 6/30 A5</TD></TR><TR><TD>7</TD><TD>10 μL 2-log purple DNA ladder</TD></TR></TBODY></TABLE><H4><B>Stop: 2:30pm</B></H4> <H4><B>Results:</B></H4><H4><B>Next:</B></H4>
Try the PCR again with 5/25 A2 as the basis for LeupB, using a longer range this time, but noting that the Temps closer to 68 seemed to yield slightly better results.

</DIV></DIV><DIV class="within-accordion"><H3>22 June 2016</H3><DIV><H4 class="h4left"><B>Kira Buckowing &amp; Kaelyn Yarbrough</B></H4><H4 class="h4right"><B>Start: 4:05pm</B></H4> <H4><B>Gel Electrophoresis of 5/16 and 5/25 Gradient PCRs</B></H4><B>Purpose:</B> To check the validity of the hopefully amplified LeupA and LeupB strains
<H4><B>Protocol:</B></H4>
LTM Ed. 2 Gel
A 1% agarose gel was run at 130 V for ~45 minutes:
<H5>6/22 Gel</H5><TABLE><TBODY><TR><TD>1</TD><TD>10 μL 2-log purple DNA ladder</TD></TR><TR><TD>2</TD><TD>10 μL 5/25 A1</TD></TR><TR><TD>3</TD><TD>10 μL 5/25 A2</TD></TR><TR><TD>4</TD><TD>10 μL 5/25 A3</TD></TR><TR><TD>5</TD><TD>10 μL 5/25 A4</TD></TR><TR><TD>6</TD><TD>10 μL 2-log purple DNA ladder</TD></TR><TR><TD>7</TD><TD>10 μL 5/16 A1</TD></TR><TR><TD>8</TD><TD>10 μL 5/16 A2</TD></TR><TR><TD>9</TD><TD>10 μL 5/16 A3</TD></TR><TR><TD>10</TD><TD>10 μL 5/16 A4</TD></TR></TBODY></TABLE><H4><B>Stop: 6:15</B></H4> <H4><B>Results:</B></H4><H4><B>Next:</B></H4>
Try the PCR again with 5/25 A2 as the basis for LeupB, figure out why LeupA doesn't seem to be working.

</DIV></DIV><DIV class="within-accordion"><H3>25 May 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: </B></H4> <H4><B>Gradient PCR of LeupB and Cultures of GST-tagged hmp</B></H4><B>Purpose:</B> To isolate LeupB from <I>Streptomyces roseus</I> and GST-tag hmp for purification
<H4><B>Protocol:</B></H4>
Four identical PCRs were performed with different annealing temperatures using a block gradient:
<H5>5/25 A1-A4</H5><TABLE><TBODY><TR><TD>10 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>3 μL</TD><TD>MilliQ water</TD></TR><TR><TD>4 μL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>1 μL</TD><TD>1/6.3X 2/25 1</TD></TR><TR><TD>1 μL</TD><TD>10 μM LeupB_F</TD></TR><TR><TD>1 μL</TD><TD>10 μM LeupB_R</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>105</TD></TR><TR><TD>98c</TD><TD>10</TD></TR><TR><TD>gradient (A1 69, A2 67.7, A3 65.9, A4 64.2)c</TD><TD>25</TD></TR><TR><TD>72c</TD><TD>47</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X32</P>
Six broth cultures were inoculated after the addition of 1X ampicillin.
<H4><B>Notes:</B></H4>
5/19 CT1 and CT2 had been left in the incubator for many days and may have been growing contamination. 5/19 CT3 plates showed one possible colony, suggesting poor transformation efficiency.
<H4><B>Stop: </B></H4> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>5/25 A1</TD><TD>2/25 1</TD><TD>LeupB amplified from <I>S. roseus</I> ISP-5076</TD></TR><TR><TD>5/25 A2</TD><TD>2/25 1</TD><TD>LeupB amplified from <I>S. roseus</I> ISP-5076</TD></TR><TR><TD>5/25 A3</TD><TD>2/25 1</TD><TD>LeupB amplified from <I>S. roseus</I> ISP-5076</TD></TR><TR><TD>5/25 A4</TD><TD>2/25 1</TD><TD>LeupB amplified from <I>S. roseus</I> ISP-5076</TD></TR><TR><TD>5/25 BC1</TD><TD>5/19 CT1 streak</TD><TD>hmp in pGEX4T-1 (GST-tagged) in ampicillin</TD></TR><TR><TD>5/25 BC2</TD><TD>5/19 CT1 streak</TD><TD>hmp in pGEX4T-1 (GST-tagged) in ampicillin</TD></TR><TR><TD>5/25 BC3</TD><TD>5/19 CT1 colony</TD><TD>hmp in pGEX4T-1 (GST-tagged) in ampicillin</TD></TR><TR><TD>5/25 BC4</TD><TD>5/19 CT2 streak</TD><TD>hmp in pGEX4T-1 (GST-tagged) in ampicillin</TD></TR><TR><TD>5/25 BC5</TD><TD>5/19 CT2 streak</TD><TD>hmp in pGEX4T-1 (GST-tagged) in ampicillin</TD></TR><TR><TD>5/25 BC6</TD><TD>5/19 CT2 colony</TD><TD>hmp in pGEX4T-1 (GST-tagged) in ampicillin</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Miniprep and digest broth cultures if they grow, then run a gel to evaluate miniprep products and 5/25 A1-A4. Check BL21 transformation procedure

</DIV></DIV><DIV class="within-accordion"><H3>19 May 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: </B></H4> <H4><B>Assembly of hmp into pGEX4T-1</B></H4><B>Purpose:</B> To GST-tag hmp for purification
<H4><B>Protocol:</B></H4>
Two chemical transformations were performed into Zymo Mix &amp; Go 5α. 1.5 µL of each plasmid was mixed with 50 µL aliquots, then immediately plated onto pre-warmed plates.

A restriction digest was prepared, then incubated and heat-killed in the thermocycler:
<H5>5/19 D1</H5><TABLE><TBODY><TR><TD>1.6 μL</TD><TD>MilliQ water</TD></TR><TR><TD>3 μL</TD><TD>10X Tango buffer (2X total)</TD></TR><TR><TD>8.6 μL</TD><TD>4/28 MP6</TD></TR><TR><TD>0.9 μL</TD><TD>EcoRI</TD></TR><TR><TD>0.9 μL</TD><TD>XhoI</TD></TR><TR><TD>15 μL</TD><TD>(2 μg)</TD></TR></TBODY></TABLE>
A 1% agarose gel was run at 100 V, then the single band in lane 5 was extracted using the Gel/PCR DNA Fragment Extraction kit:
<H5>5/19 Gel #1</H5><TABLE><TBODY><TR><TD>2</TD><TD>5 μL 2-log purple DNA ladder</TD></TR><TR><TD>5</TD><TD>15 μL 5/19 D1</TD></TR></TBODY></TABLE>
A chemical transformation was performed with 1.5 μL of plasmid in 98.5 μL KCM solution, mixed with 100 μL of competent BL21. Mixture was incubated on ice for 15 minutes, then at room temperature for 25 minutes before plating.

Gel-extracted fragment was NanoDropped to determine concentration, but showed no characteristic peak at 260 nm.
<H4><B>Notes:</B></H4>
Zymo Mix &amp; Go 5α now in a third-floor freezer
<H4><B>Stop: </B></H4> <H4><B>Results:</B></H4> <TABLE><TBODY><TR><TD><B>Sample</B></TD><TD><B>ng/μL</B></TD><TD><B>260/280</B></TD><TD><B>260/230</B></TD></TR><TR><TD>5/19 GE1</TD><TD>0.8</TD><TD>-0.93</TD><TD>0.17</TD></TR><TR><TD>5/19 GE1</TD><TD>1.5</TD><TD>1.36</TD><TD>0.30</TD></TR></TBODY></TABLE> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>5/19 CT1</TD><TD>5/16 L1</TD><TD>hmp in pGEX4T-1 (GST-tagged) on ampicillin</TD></TR><TR><TD>5/19 CT2</TD><TD>5/16 L2</TD><TD>hmp in pGEX4T-1 (GST-tagged) on ampicillin</TD></TR><TR><TD>5/19 D1</TD><TD>4/28 MP6</TD><TD>pGEX4T-1 digested EcoRI &amp; XhoI</TD></TR><TR><TD>5/19 GE1</TD><TD>5/19 D1</TD><TD>pGEX4T-1 digested EcoRI &amp; XhoI, backbone purified</TD></TR><TR><TD>5/19 CT3</TD><TD>10 pg/μL pUC19</TD><TD>transformation efficiency test using pUC19 on ampicillin</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4> </DIV></DIV><DIV class="within-accordion"><H3>16 May 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday and Erin Nischwitz</B></H4><H4 class="h4right"><B>Start: 11:40 am</B></H4> <H4><B>Assembly of hmp in pGEX4T-1 and LeupA Gradient PCRs</B></H4><B>Purpose:</B> To GST-tag hmp for purification and characterization, and to amplify LeupA from <I>Streptomyces roseus</I><H4><B>Protocol:</B></H4>
Two restriction digests were prepared, then incubated and heat-killed in the thermocycler:
<H5>5/16 D1</H5><TABLE><TBODY><TR><TD>8.2 μL</TD><TD>MilliQ water</TD></TR><TR><TD>3 μL</TD><TD>10X Tango buffer</TD></TR><TR><TD>2.6 μL</TD><TD>4/28 MP6</TD></TR><TR><TD>0.6 μL</TD><TD>EcoRI</TD></TR><TR><TD>0.6 μL</TD><TD>XhoI</TD></TR><TR><TD>15 μL</TD><TD>(1 μg)</TD></TR></TBODY></TABLE><H5>5/16 D2</H5><TABLE><TBODY><TR><TD>10.2 μL</TD><TD>MilliQ water</TD></TR><TR><TD>3 μL</TD><TD>10X Tango buffer</TD></TR><TR><TD>0.6 μL</TD><TD>2/20 A2</TD></TR><TR><TD>0.6 μL</TD><TD>EcoRI</TD></TR><TR><TD>0.6 μL</TD><TD>XhoI</TD></TR><TR><TD>15 μL</TD></TR></TBODY></TABLE>
A 1% agarose gel was run at 100 V, then the single band in lane 5 was extracted using the Gel/PCR DNA Fragment Extraction kit:
<H5>5/16 Gel #1</H5><TABLE><TBODY><TR><TD>2</TD><TD>5 μL 2-log purple DNA ladder</TD></TR><TR><TD>3</TD><TD>5 μL 5/16 D2</TD></TR><TR><TD>5</TD><TD>15 μL 5/16 D1</TD></TR></TBODY></TABLE>
Four identical PCRs were performed with different annealing temperatures using a block gradient:
<H5>5/16 A1-A4</H5><TABLE><TBODY><TR><TD>10 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>3 μL</TD><TD>MilliQ water</TD></TR><TR><TD>4 μL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>1 μL</TD><TD>1/6.3X 2/25 1</TD></TR><TR><TD>1 μL</TD><TD>10 μM LeupA_F_V</TD></TR><TR><TD>1 μL</TD><TD>10 μM LeupA_R</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>105</TD></TR><TR><TD>98c</TD><TD>10</TD></TR><TR><TD>gradient (A1 72, A2 70.5, A3 68.9, A4 67.6)c</TD><TD>25</TD></TR><TR><TD>72c</TD><TD>306</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X32</P>
The gel-purified fragment was NanoDropped to determine concentration, then two ligations were incubated at 16 °C:
<H5>5/16 L1</H5><TABLE><TBODY><TR><TD>1 μL</TD><TD>10X T4 ligase buffer</TD></TR><TR><TD>3.7 μL</TD><TD>5/16 GE1</TD></TR><TR><TD>4.3 μL</TD><TD>5/16 D2</TD></TR><TR><TD>1 μL</TD><TD>T4 DNA ligase</TD></TR><TR><TD>10 μL</TD><TD>(2:1)</TD></TR></TBODY></TABLE><H5>5/16 L2</H5><TABLE><TBODY><TR><TD>1 μL</TD><TD>10X T4 ligase buffer</TD></TR><TR><TD>1.5 μL</TD><TD>5/16 GE1</TD></TR><TR><TD>6.5 μL</TD><TD>5/16 D2</TD></TR><TR><TD>1 μL</TD><TD>T4 DNA ligase</TD></TR><TR><TD>10 μL</TD><TD>(8:1)</TD></TR></TBODY></TABLE>
A 1% agarose gel was run at 130 V for 40 minutes:
<H5>5/16 Gel #2</H5><TABLE><TBODY><TR><TD>3</TD><TD>5 μL 2-log purple DNA ladder</TD></TR><TR><TD>4</TD><TD>7.5 μL 5/16 A1</TD></TR><TR><TD>5</TD><TD>7.5 μL 5/16 A2</TD></TR><TR><TD>6</TD><TD>7.5 μL 5/16 A3</TD></TR><TR><TD>7</TD><TD>7.5 μL 5/16 A4</TD></TR></TBODY></TABLE>
Two chemical transformations were performed with 1.5 μL of plasmid in 98.5 μL KCM solution, mixed with 100 μL of competent BL21. Mixture was incubated on ice for 15 minutes, then at room temperature for 25 minutes before plating.
<H4><B>Notes:</B></H4>
Zymo Mix &amp; Go DH5α no longer in Dr. Simone's freezer
<H4><B>Stop: 9:00 pm</B></H4> <H4><B>Results:</B></H4>
5/16 GE1 was reported as 3.9 μL but showed no characteristic peak.
<H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>5/16 D1</TD><TD>4/28 MP6</TD><TD>pGEX4T-1 digested EcoRI &amp; XhoI</TD></TR><TR><TD>5/16 D2</TD><TD>2/20 A2</TD><TD>hmp with GST-tagging sites digested EcoRI &amp; XhoI</TD></TR><TR><TD>5/16 GE1</TD><TD>5/16 D1</TD><TD>pGEX4T-1 digested EcoRI &amp; XhoI, purified backbone</TD></TR><TR><TD>5/16 A1</TD><TD>2/25 1</TD><TD>LeupA amplified from <I>S. roseus</I> ISP-5076</TD></TR><TR><TD>5/16 A2</TD><TD>2/25 1</TD><TD>LeupA amplified from <I>S. roseus</I> ISP-5076</TD></TR><TR><TD>5/16 A3</TD><TD>2/25 1</TD><TD>LeupA amplified from <I>S. roseus</I> ISP-5076</TD></TR><TR><TD>5/16 A4</TD><TD>2/25 1</TD><TD>LeupA amplified from <I>S. roseus</I> ISP-5076</TD></TR><TR><TD>5/16 L1</TD><TD>5/16 D1, 5/16 D2</TD><TD>hmp in pGEX4T-1 (GST-tagged)</TD></TR><TR><TD>5/16 L2</TD><TD>5/16 D1, 5/16 D2</TD><TD>hmp in pGEX4T-1 (GST-tagged)</TD></TR><TR><TD>5/16 CT1</TD><TD>5/16 L1</TD><TD>hmp in pGEX4T-1 (GST-tagged) on ampicillin</TD></TR><TR><TD>5/16 CT2</TD><TD>5/16 L2</TD><TD>hmp in pGEX4T-1 (GST-tagged) on ampicillin</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Locate competent DH5α to retry transformations and perform transformation efficiency test of BL21 if no transformants, otherwise inoculate broth cultures of transformed BL21 and miniprep&gt;digest&gt;gel to screen for the correct plasmid. Retry LeupA amplification without GC Enhancer, under the same program at two differrent annealing temperatures.

</DIV></DIV><DIV class="within-accordion"><H3>29 April 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: </B></H4> <H4><B>Verification of GST-tagged hmp and pGEX4T-1 Minipreps</B></H4><B>Purpose:</B> To determine if minipreps of GST-tagged hmp and pGEX4T-1 are the correct products
<H4><B>Protocol:</B></H4>
Seven restriction digests were prepared, then incubated and heat-killed in the thermocycler:
<H5>4/29 D1</H5><TABLE><TBODY><TR><TD>3 μL</TD><TD>10X Tango buffer (2X total)</TD></TR><TR><TD>5.7 μL</TD><TD>MilliQ water</TD></TR><TR><TD>0.6 μL</TD><TD>EcoRI</TD></TR><TR><TD>0.6 μL</TD><TD>XhoI</TD></TR><TR><TD>5.1 μL</TD><TD>4/28 MP1</TD></TR><TR><TD>15 μL</TD><TD>(1 μg)</TD></TR></TBODY></TABLE><H5>4/29 D2</H5><TABLE><TBODY><TR><TD>3 μL</TD><TD>10X Tango buffer (2X total)</TD></TR><TR><TD>5.8 μL</TD><TD>MilliQ water</TD></TR><TR><TD>0.6 μL</TD><TD>EcoRI</TD></TR><TR><TD>0.6 μL</TD><TD>XhoI</TD></TR><TR><TD>5 μL</TD><TD>4/28 MP2</TD></TR><TR><TD>15 μL</TD><TD>(1 μg)</TD></TR></TBODY></TABLE><H5>4/29 D3</H5><TABLE><TBODY><TR><TD>3 μL</TD><TD>10X Tango buffer (2X total)</TD></TR><TR><TD>7.3 μL</TD><TD>MilliQ water</TD></TR><TR><TD>0.6 μL</TD><TD>EcoRI</TD></TR><TR><TD>0.6 μL</TD><TD>XhoI</TD></TR><TR><TD>3.5 μL</TD><TD>4/28 MP3</TD></TR><TR><TD>15 μL</TD><TD>(1 μg)</TD></TR></TBODY></TABLE><H5>4/29 D4</H5><TABLE><TBODY><TR><TD>3 μL</TD><TD>10X Tango buffer (2X total)</TD></TR><TR><TD>6.8 μL</TD><TD>MilliQ water</TD></TR><TR><TD>0.6 μL</TD><TD>EcoRI</TD></TR><TR><TD>0.6 μL</TD><TD>XhoI</TD></TR><TR><TD>4 μL</TD><TD>4/28 MP4</TD></TR><TR><TD>15 μL</TD><TD>(1 μg)</TD></TR></TBODY></TABLE><H5>4/29 D5</H5><TABLE><TBODY><TR><TD>3 μL</TD><TD>10X Tango buffer (2X total)</TD></TR><TR><TD>6.3 μL</TD><TD>MilliQ water</TD></TR><TR><TD>0.6 μL</TD><TD>EcoRI</TD></TR><TR><TD>0.6 μL</TD><TD>XhoI</TD></TR><TR><TD>4.5 μL</TD><TD>4/28 MP5</TD></TR><TR><TD>15 μL</TD><TD>(1 μg)</TD></TR></TBODY></TABLE><H5>4/29 D6</H5><TABLE><TBODY><TR><TD>3 μL</TD><TD>10X Tango buffer (2X total)</TD></TR><TR><TD>6.5 μL</TD><TD>MilliQ water</TD></TR><TR><TD>0.6 μL</TD><TD>EcoRI</TD></TR><TR><TD>0.6 μL</TD><TD>XhoI</TD></TR><TR><TD>4.3 μL</TD><TD>4/28 MP6</TD></TR><TR><TD>15 μL</TD><TD>(1 μg)</TD></TR></TBODY></TABLE><H5>4/29 D7</H5><TABLE><TBODY><TR><TD>3 μL</TD><TD>10X Tango buffer (2X total)</TD></TR><TR><TD>8.2 μL</TD><TD>MilliQ water</TD></TR><TR><TD>0.6 μL</TD><TD>EcoRI</TD></TR><TR><TD>0.6 μL</TD><TD>XhoI</TD></TR><TR><TD>2.6 μL</TD><TD>4/28 MP7</TD></TR><TR><TD>15 μL</TD><TD>(1 μg)</TD></TR></TBODY></TABLE>
A 1% agarose gel was run at 130 V:
<H5>4/29 Gel #1</H5><TABLE><TBODY><TR><TD>1</TD><TD>15 μL 4/29 D1</TD></TR><TR><TD>2</TD><TD>15 μL 4/29 D2</TD></TR><TR><TD>3</TD><TD>15 μL 4/29 D3</TD></TR><TR><TD>4</TD><TD>15 μL 4/29 D4</TD></TR><TR><TD>5</TD><TD>7.5 μL 2-log purple DNA ladder</TD></TR><TR><TD>6</TD><TD>15 μL 4/29 D5</TD></TR><TR><TD>7</TD><TD>15 μL 4/29 D6</TD></TR><TR><TD>8</TD><TD>15 μL 4/29 D7</TD></TR></TBODY></TABLE> <H4><B>Stop: </B></H4> <H4><B>Results:</B></H4><H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>4/29 D1</TD><TD>4/28 MP1</TD><TD>religated pGEX4T-1 digested EcoRI &amp; XhoI</TD></TR><TR><TD>4/29 D2</TD><TD>4/28 MP2</TD><TD>religated pGEX4T-1 digested EcoRI &amp; XhoI</TD></TR><TR><TD>4/29 D3</TD><TD>4/28 MP3</TD><TD>hmp in pGEX4T-1 digested EcoRI &amp; XhoI</TD></TR><TR><TD>4/29 D4</TD><TD>4/28 MP4</TD><TD>hmp in pGEX4T-1 digested EcoRI &amp; XhoI</TD></TR><TR><TD>4/29 D5</TD><TD>4/28 MP5</TD><TD>pGEX4T-1 digested EcoRI &amp; XhoI</TD></TR><TR><TD>4/29 D6</TD><TD>4/28 MP6</TD><TD>pGEX4T-1 digested EcoRI &amp; XhoI</TD></TR><TR><TD>4/29 D7</TD><TD>4/28 MP7</TD><TD>pGEX4T-1 digested EcoRI &amp; XhoI</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4> </DIV></DIV><DIV class="within-accordion"><H3>28 April 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday and Erin Nischwitz</B></H4><H4 class="h4right"><B>Start: </B></H4> <H4><B>Minipreps of GST-tagged hmp and pGEX4T-1</B></H4><B>Purpose:</B> To obtain GST-tagged hmp for purification
<H4><B>Protocol:</B></H4>
Seven minipreps were performed using the kitless miniprep procedure, then products were NanoDropped to determine concentration.
<H4><B>Stop: </B></H4> <H4><B>Results:</B></H4> <TABLE><TBODY><TR><TD><B>Sample</B></TD><TD><B>ng/μL</B></TD><TD><B>260/280</B></TD><TD><B>260/230</B></TD></TR><TR><TD>4/28 MP1</TD><TD>195.53</TD><TD>1.90</TD><TD>2.65</TD></TR><TR><TD>4/28 MP2</TD><TD>199.29</TD><TD>1.90</TD><TD>2.68</TD></TR><TR><TD>4/28 MP3</TD><TD>285.32</TD><TD>1.91</TD><TD>2.67</TD></TR><TR><TD>4/28 MP4</TD><TD>251.04</TD><TD>1.90</TD><TD>2.55</TD></TR><TR><TD>4/28 MP5</TD><TD>221.64</TD><TD>1.85</TD><TD>1.99</TD></TR><TR><TD>4/28 MP6</TD><TD>233.73</TD><TD>1.87</TD><TD>1.76</TD></TR><TR><TD>4/28 MP7</TD><TD>384.50</TD><TD>1.87</TD><TD>2.07</TD></TR></TBODY></TABLE> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>4/28 MP1</TD><TD>4/27 BC2</TD><TD>religated pGEX4T-1</TD></TR><TR><TD>4/28 MP2</TD><TD>4/27 BC3</TD><TD>religated pGEX4T-1</TD></TR><TR><TD>4/28 MP3</TD><TD>4/27 BC5</TD><TD>hmp in pGEX4T-1</TD></TR><TR><TD>4/28 MP4</TD><TD>4/27 BC6</TD><TD>hmp in pGEX4T-1</TD></TR><TR><TD>4/28 MP5</TD><TD>4/25 BC1</TD><TD>pGEX4T-1</TD></TR><TR><TD>4/28 MP6</TD><TD>4/25 BC2</TD><TD>pGEX4T-1</TD></TR><TR><TD>4/28 MP7</TD><TD>4/25 BC3</TD><TD>pGEX4T-1</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4> </DIV></DIV><DIV class="within-accordion"><H3>27 April 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: </B></H4> <H4><B>Cultures of GST-tagged hmp</B></H4><B>Purpose:</B> To obtain GST-tagged hmp for purification
<H4><B>Protocol:</B></H4>
Six broth cultures were inoculated after the addition of 1X ampicillin.
<H4><B>Stop: </B></H4> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>4/27 BC1</TD><TD>4/26 CT1</TD><TD>religated pGEX4T-1 in ampicillin</TD></TR><TR><TD>4/27 BC2</TD><TD>4/26 CT1</TD><TD>religated pGEX4T-1 in ampicillin</TD></TR><TR><TD>4/27 BC3</TD><TD>4/26 CT1</TD><TD>religated pGEX4T-1 in ampicillin</TD></TR><TR><TD>4/27 BC4</TD><TD>4/26 CT2</TD><TD>hmp in pGEX4T-1 in ampicillin</TD></TR><TR><TD>4/27 BC5</TD><TD>4/26 CT2</TD><TD>hmp in pGEX4T-1 in ampicillin</TD></TR><TR><TD>4/27 BC6</TD><TD>4/26 CT2</TD><TD>hmp in pGEX4T-1 in ampicillin</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4> </DIV></DIV><DIV class="within-accordion"><H3>26 April 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: </B></H4> <H4><B>Religation of pGEX4T-1 and Transformation of pGEX4T-1 and hmp in pGEX4T-1</B></H4><B>Purpose:</B> To GST-tag hmp for purification and obtain more pGEX4T-1 for future use
<H4><B>Protocol:</B></H4>
A ligation was mixed and incubated at 16°C:
<H5>4/26 L1</H5><TABLE><TBODY><TR><TD>0.75 μL</TD><TD>10X T4 ligase buffer</TD></TR><TR><TD>6 μL</TD><TD>10/11 D6</TD></TR><TR><TD>0.75 μL</TD><TD>T4 DNA ligase</TD></TR><TR><TD>7.5 μL</TD></TR></TBODY></TABLE>
Two chemical transformations were performed into 50 μL Zymo Mix &amp; Go 5α, which were plated immediately onto pre-warmed plates.
<H4><B>Stop: </B></H4> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>4/26 L1</TD><TD>10/11 D6</TD><TD>religated pGEX4T-1</TD></TR><TR><TD>4/26 CT1</TD><TD>4/26 L1</TD><TD>religated pGEX4T-1 on ampicillin</TD></TR><TR><TD>4/26 CT2</TD><TD>4/19 L2</TD><TD>hmp in pGEX4T-1 on ampicillin</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4> </DIV></DIV><DIV class="within-accordion"><H3>25 April 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: </B></H4> <H4><B>Broth Cultures of pGEX4T-1</B></H4><B>Purpose:</B> To obtain additional pGEX4T-1 plasmid for future use
<H4><B>Protocol:</B></H4>
Six broth cultures were inoculated after the addition of 1X ampicillin.
<H4><B>Stop: </B></H4> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>4/25 BC1</TD><TD>8/2/15 pGEX I18</TD><TD>pGEX4T-1 in ampicillin</TD></TR><TR><TD>4/25 BC2</TD><TD>8/2/15 pGEX I18</TD><TD>pGEX4T-1 in ampicillin</TD></TR><TR><TD>4/25 BC3</TD><TD>8/2/15 pGEX I18</TD><TD>pGEX4T-1 in ampicillin</TD></TR><TR><TD>4/25 BC4</TD><TD>8/2/15 pGEX I19</TD><TD>pGEX4T-1 in ampicillin</TD></TR><TR><TD>4/25 BC5</TD><TD>8/2/15 pGEX I19</TD><TD>pGEX4T-1 in ampicillin</TD></TR><TR><TD>4/25 BC6</TD><TD>8/2/15 pGEX I19</TD><TD>pGEX4T-1 in ampicillin</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4> </DIV></DIV><DIV class="within-accordion"><H3>24 April 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4></DIV></DIV><DIV class="within-accordion"><H3>22 April 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4></DIV></DIV><DIV class="within-accordion"><H3>21 April 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: 3:00 pm</B></H4> <B>Purpose:</B> To obtain and verify GST-tagged hmp and diagnose LeupA PCRs
<H4><B>Protocol:</B></H4>
Three minipreps were performed using the kitless miniprep procedure and resuspended in 38 μL 1X TE. Products were NanoDropped to determine concentration.

Four restriction digests were mixed, then reacted and heat-killed in the thermocycler:
<H5>4/21 D1</H5><TABLE><TBODY><TR><TD>7.7 μL</TD><TD>MilliQ water</TD></TR><TR><TD>1 μL</TD><TD>10X Tango buffer</TD></TR><TR><TD>0.5 μL</TD><TD>4/19 A1</TD></TR><TR><TD>0.4 μL</TD><TD>EcoRI</TD></TR><TR><TD>0.4 μL</TD><TD>SpeI</TD></TR><TR><TD>10 μL</TD></TR></TBODY></TABLE><H5>4/21 D2</H5><TABLE><TBODY><TR><TD>3.2 μL</TD><TD>MilliQ water</TD></TR><TR><TD>1 μL</TD><TD>10X Tango buffer</TD></TR><TR><TD>5 μL</TD><TD>4/19 A2</TD></TR><TR><TD>0.4 μL</TD><TD>EcoRI</TD></TR><TR><TD>0.4 μL</TD><TD>SpeI</TD></TR><TR><TD>10 μL</TD></TR></TBODY></TABLE><H5>4/21 D3</H5><TABLE><TBODY><TR><TD>6.9 μL</TD><TD>MilliQ water</TD></TR><TR><TD>1 μL</TD><TD>10X Tango buffer</TD></TR><TR><TD>1.3 μL</TD><TD>4/21 MP2</TD></TR><TR><TD>0.4 μL</TD><TD>EcoRI</TD></TR><TR><TD>0.4 μL</TD><TD>XhoI</TD></TR><TR><TD>10 μL</TD><TD>(1 μg)</TD></TR></TBODY></TABLE><H5>4/21 D4</H5><TABLE><TBODY><TR><TD>6.1 μL</TD><TD>MilliQ water</TD></TR><TR><TD>1 μL</TD><TD>10X Tango buffer</TD></TR><TR><TD>2.1 μL</TD><TD>4/21 MP3</TD></TR><TR><TD>0.4 μL</TD><TD>EcoRI</TD></TR><TR><TD>0.4 μL</TD><TD>XhoI</TD></TR><TR><TD>10 μL</TD><TD>(1 μg)</TD></TR></TBODY></TABLE> <H4><B>Notes:</B></H4>
4/20 BC1, 2, and 4 had no growth. 4/21 MP1 was discarded after it was dropped.
<H4><B>Stop: 6:30 pm</B></H4> <H4><B>Results:</B></H4> <TABLE><TBODY><TR><TD><B>Sample</B></TD><TD><B>ng/μL</B></TD><TD><B>260/280</B></TD><TD><B>260/230</B></TD></TR><TR><TD>4/21 MP2</TD><TD>752.93</TD><TD>1.87</TD><TD>1.82</TD></TR><TR><TD>4/21 MP3</TD><TD>467.38</TD><TD>1.88</TD><TD>2.25</TD></TR></TBODY></TABLE> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>4/21 MP1</TD><TD>4/18 BC1</TD><TD>pGEX4T-1 (discarded)</TD></TR><TR><TD>4/21 MP2</TD><TD>4/20 BC3</TD><TD>hmp in pGEX4T-1</TD></TR><TR><TD>4/21 MP3</TD><TD>4/20 BC5</TD><TD>hmp in pGEX4T-1</TD></TR><TR><TD>4/21 D1</TD><TD>4/19 A1</TD><TD>LeupA amplification digested EcoRI &amp; SpeI</TD></TR><TR><TD>4/21 D2</TD><TD>4/19 A2</TD><TD>LeupA amplification digested EcoRI &amp; SpeI</TD></TR><TR><TD>4/21 D3</TD><TD>4/21 MP2</TD><TD>hmp in pGEX4T-1 digested EcoRI &amp; XhoI</TD></TR><TR><TD>4/21 D4</TD><TD>4/21 MP3</TD><TD>hmp in pGEX4T-1 digested EcoRI &amp; XhoI</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Run a gel of digests

</DIV></DIV><DIV class="within-accordion"><H3>20 April 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: 11:50 pm</B></H4> <H4><B>Broth Cultures of pGEX4T-1 and GST-tagged hmp</B></H4><B>Purpose:</B> To obtain pGEX4T-1 and GST-tagged hmp
<H4><B>Protocol:</B></H4>
Five broth cultures were inoculated from plates after the addition of 5 μL 1000X ampicillin.

4/18 BC1 was moved into the refrigerator.
<H4><B>Notes:</B></H4>
There was no growth in 4/18 BC2.
<H4><B>Stop: 1:00 am</B></H4> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>4/20 BC1</TD><TD>4/19 CT1</TD><TD>pGEX4T-1 in ampicillin</TD></TR><TR><TD>4/20 BC2</TD><TD>4/19 CT1</TD><TD>pGEX4T-1 in ampicillin</TD></TR><TR><TD>4/20 BC3</TD><TD>4/19 CT2</TD><TD>hmp in pGEX4T-1 (GST-tagged) in ampicillin</TD></TR><TR><TD>4/20 BC4</TD><TD>4/19 CT2</TD><TD>hmp in pGEX4T-1 (GST-tagged) in ampicillin</TD></TR><TR><TD>4/20 BC5</TD><TD>4/19 CT2</TD><TD>hmp in pGEX4T-1 (GST-tagged) in ampicillin</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Miniprep 4/18 BC1 and 4/20 BC1-5

</DIV></DIV><DIV class="within-accordion"><H3>19 April 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday and Erin Nischwitz</B></H4><H4 class="h4right"><B>Start: 5:00 pm</B></H4> <H4><B>New PCRs of LeupA and Ligations of hmp in pGEX4T-1</B></H4><B>Purpose:</B> To isolate LeupA from <I>Streptomyces roseus</I> strains for cloning, and GST-tag hmp.
<H4><B>Protocol:</B></H4>
Three PCR reactions were mixed without polymerase, denatured for 3 minutes on the thermocycler block at 98 °C, then incubated in the thermocycler after the addition of polymerase:
<H5>4/19 A1</H5><TABLE><TBODY><TR><TD>3 μL</TD><TD>MilliQ water</TD></TR><TR><TD>4 μL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>1 μL</TD><TD>10 μM LeupA_F_V</TD></TR><TR><TD>1 μL</TD><TD>10 μM LeupA_R</TD></TR><TR><TD>1 μL</TD><TD>1/6.3X 2/25 1</TD></TR><TR><TD>10 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE><H5>4/19 A2</H5><TABLE><TBODY><TR><TD>3 μL</TD><TD>MilliQ water</TD></TR><TR><TD>4 μL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>1 μL</TD><TD>10 μM LeupA_F_V</TD></TR><TR><TD>1 μL</TD><TD>10 μM LeupA_R</TD></TR><TR><TD>1 μL</TD><TD>1/3.7X 2/25 2</TD></TR><TR><TD>10 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE><H5>4/19 A3</H5><TABLE><TBODY><TR><TD>3 μL</TD><TD>MilliQ water</TD></TR><TR><TD>4 μL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>1 μL</TD><TD>10 μM LeupA_F_V</TD></TR><TR><TD>1 μL</TD><TD>10 μM LeupA_R</TD></TR><TR><TD>1 μL</TD><TD>1/8.5X 2/25 3</TD></TR><TR><TD>10 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>10</TD></TR><TR><TD>72c</TD><TD>400</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X32</P>
Two ligations were mixed and left to react at room temperature for four hours:
<H5>4/19 L1</H5><TABLE><TBODY><TR><TD>1 µL</TD><TD>10X T4 ligase buffer</TD></TR><TR><TD>7 µL</TD><TD>MilliQ water</TD></TR><TR><TD>1 µL</TD><TD>10/11 D6</TD></TR><TR><TD>1 µL</TD><TD>T4 DNA ligase</TD></TR><TR><TD>10 µL</TD></TR></TBODY></TABLE><H5>4/19 L2</H5><TABLE><TBODY><TR><TD>0.5 µL</TD><TD>10X T4 ligase buffer</TD></TR><TR><TD>3.5 µL</TD><TD>2/20 D2</TD></TR><TR><TD>0.5 µL</TD><TD>1/2X 10/11 D6</TD></TR><TR><TD>0.5 µL</TD><TD>T4 DNA ligase</TD></TR><TR><TD>5 µL</TD></TR></TBODY></TABLE>
A 1% agarose gel was run at 100 V until the loading dye reached 2/3 of the gel length:
<H5>4/19 Gel #1</H5><TABLE><TBODY><TR><TD>2</TD><TD>2 µL 10/11 D6 + 5.5 µL 1X TE buffer</TD></TR><TR><TD>3</TD><TD>5 µL 2-log purple DNA ladder</TD></TR><TR><TD>5</TD><TD>7.5 µL 4/19 A1</TD></TR><TR><TD>7</TD><TD>7.5 µL 4/19 A2</TD></TR><TR><TD>9</TD><TD>7.5 µL 4/19 A3</TD></TR></TBODY></TABLE>
Two chemical transformations were performed into Zymo Mix &amp; Go 5α. 1.5 µL of each plasmid was mixed with 50 µL aliquots, then incubated for 7 minutes on ice. 200 µL of SOC was added for 45 minutes of outgrowth at 37 °C, then 80 µL of each was plated.
<H4><B>Stop: 9:30 pm</B></H4> <H4><B>Results:</B></H4><H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>4/19 A1</TD><TD>2/25 1</TD><TD>LeupA amplified from <I>S. roseus</I> ISP-5076</TD></TR><TR><TD>4/19 A2</TD><TD>2/25 2</TD><TD>LeupA amplified from <I>S. roseus</I> B-1320</TD></TR><TR><TD>4/19 A3</TD><TD>2/25 3</TD><TD>LeupA amplified from <I>S. roseus</I> B-3062</TD></TR><TR><TD>4/19 L1</TD><TD>10/11 D6</TD><TD>Religated pGEX4T-1</TD></TR><TR><TD>4/19 L2</TD><TD>10/11 D6, 2/20 D2</TD><TD>hmp in pGEX4T-1 (GST-tagged)</TD></TR><TR><TD>4/19 CT1</TD><TD>4/19 L1</TD><TD>Religated pGEX4T-1 on ampicillin</TD></TR><TR><TD>4/19 CT2</TD><TD>4/19 L2</TD><TD>hmp in pGEX4T-1 (GST-tagged) on ampicillin</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Test pH of autoclaved MilliQ water with High GC Enhancer, inoculate broth cultures and miniprep from transformations, miniprep 4/18 broth cultures, then digest and gel to verify products.

</DIV></DIV><DIV class="within-accordion"><H3>18 April 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: 3:00 pm</B></H4> <H4><B>Cultures of pGEX4T-1</B></H4><B>Purpose:</B> to obtain new pGEX4T-1 for GST-tagging hmp
<H4><B>Protocol:</B></H4>
Two broth cultures were inoculated from plates after the addition of 5 μL 1000X ampicillin.
<H4><B>Stop: 4:00 pm</B></H4> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>4/18 BC1</TD><TD>8/2/15 I18</TD><TD>pGEX4T-1 in ampicillin</TD></TR><TR><TD>4/18 BC2</TD><TD>8/2/15 I19</TD><TD>pGEX4T-1 in ampicillin</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Retry LeupA PCRs with longer denaturing and extension times. Miniprep broth cultures, and retry hmp ligations with 10/11 D6.

</DIV></DIV><DIV class="within-accordion"><H3>15 April 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: 1:20 pm</B></H4> <H4><B>Protocol:</B></H4>
Eight restriction digests were prepared before incubation and heat-kill in the thermocycler:
<H5>4/15 D1</H5><TABLE><TBODY><TR><TD>7 μL</TD><TD>MilliQ water</TD></TR><TR><TD>1 μL</TD><TD>10X Tango buffer</TD></TR><TR><TD>1.2 μL</TD><TD>4/14 MP1</TD></TR><TR><TD>0.4 μL</TD><TD>EcoRI</TD></TR><TR><TD>0.4 μL</TD><TD>SpeI</TD></TR><TR><TD>10 μL</TD><TD>(1 μg)</TD></TR></TBODY></TABLE><H5>4/15 D2</H5><TABLE><TBODY><TR><TD>6.1 μL</TD><TD>MilliQ water</TD></TR><TR><TD>1 μL</TD><TD>10X Tango buffer</TD></TR><TR><TD>2.1 μL</TD><TD>4/14 MP2</TD></TR><TR><TD>0.4 μL</TD><TD>EcoRI</TD></TR><TR><TD>0.4 μL</TD><TD>SpeI</TD></TR><TR><TD>10 μL</TD><TD>(1 μg)</TD></TR></TBODY></TABLE><H5>4/15 D3</H5><TABLE><TBODY><TR><TD>5.5 μL</TD><TD>MilliQ water</TD></TR><TR><TD>1 μL</TD><TD>10X Tango buffer</TD></TR><TR><TD>2.7 μL</TD><TD>4/14 MP3</TD></TR><TR><TD>0.4 μL</TD><TD>EcoRI</TD></TR><TR><TD>0.4 μL</TD><TD>XhoI</TD></TR><TR><TD>10 μL</TD><TD>(1 μg)</TD></TR></TBODY></TABLE><H5>4/15 D4</H5><TABLE><TBODY><TR><TD>6.6 μL</TD><TD>MilliQ water</TD></TR><TR><TD>1 μL</TD><TD>10X Tango buffer</TD></TR><TR><TD>1.6 μL</TD><TD>4/14 MP4</TD></TR><TR><TD>0.4 μL</TD><TD>EcoRI</TD></TR><TR><TD>0.4 μL</TD><TD>XhoI</TD></TR><TR><TD>10 μL</TD><TD>(1 μg)</TD></TR></TBODY></TABLE><H5>4/15 D5</H5><TABLE><TBODY><TR><TD>7 μL</TD><TD>MilliQ water</TD></TR><TR><TD>1 μL</TD><TD>10X Tango buffer</TD></TR><TR><TD>1.2 μL</TD><TD>4/14 MP5</TD></TR><TR><TD>0.4 μL</TD><TD>EcoRI</TD></TR><TR><TD>0.4 μL</TD><TD>XhoI</TD></TR><TR><TD>10 μL</TD><TD>(1 μg)</TD></TR></TBODY></TABLE><H5>4/15 D6</H5><TABLE><TBODY><TR><TD>6.3 μL</TD><TD>MilliQ water</TD></TR><TR><TD>1 μL</TD><TD>10X Tango buffer</TD></TR><TR><TD>1.9 μL</TD><TD>4/14 MP6</TD></TR><TR><TD>0.4 μL</TD><TD>EcoRI</TD></TR><TR><TD>0.4 μL</TD><TD>XhoI</TD></TR><TR><TD>10 μL</TD><TD>(1 μg)</TD></TR></TBODY></TABLE><H5>4/15 D7</H5><TABLE><TBODY><TR><TD>3.8 μL</TD><TD>MilliQ water</TD></TR><TR><TD>1 μL</TD><TD>10X Tango buffer</TD></TR><TR><TD>4.4 μL</TD><TD>4/14 MP7</TD></TR><TR><TD>0.4 μL</TD><TD>EcoRI</TD></TR><TR><TD>0.4 μL</TD><TD>XhoI</TD></TR><TR><TD>10 μL</TD><TD>(1 μg)</TD></TR></TBODY></TABLE><H5>4/15 D8</H5><TABLE><TBODY><TR><TD>6.9 μL</TD><TD>MilliQ water</TD></TR><TR><TD>1 μL</TD><TD>10X Tango buffer</TD></TR><TR><TD>1.3 μL</TD><TD>4/14 MP8</TD></TR><TR><TD>0.4 μL</TD><TD>EcoRI</TD></TR><TR><TD>0.4 μL</TD><TD>XhoI</TD></TR><TR><TD>10 μL</TD><TD>(1 μg)</TD></TR></TBODY></TABLE>
A 1% agarose gel was run at 90 V until the dye reached 2/3 of the gel length:
<TABLE><TBODY><TR><TD>1</TD><TD>10 μL 4/15 D1</TD></TR><TR><TD>2</TD><TD>10 μL 4/15 D2</TD></TR><TR><TD>3</TD><TD>5 μL 2-log purple DNA ladder</TD></TR><TR><TD>4</TD><TD>10 μL 4/15 D3</TD></TR><TR><TD>5</TD><TD>10 μL 4/15 D4</TD></TR><TR><TD>6</TD><TD>10 μL 4/15 D5</TD></TR><TR><TD>7</TD><TD>10 μL 4/15 D6</TD></TR><TR><TD>8</TD><TD>5 μL 2-log purple DNA ladder</TD></TR><TR><TD>9</TD><TD>10 μL 4/15 D7</TD></TR><TR><TD>10</TD><TD>10 μL 4/15 D8</TD></TR></TBODY></TABLE> <H4><B>Stop: 5:30 pm</B></H4> <H4><B>Results:</B></H4><H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>4/15 D1</TD><TD>4/14 MP1</TD><TD>LeupA from 3/17 GE2 in pSB1C3 cut EcoRI &amp; SpeI</TD></TR><TR><TD>4/15 D2</TD><TD>4/14 MP2</TD><TD>LeupA from 3/17 GE2 in pSB1C3 cut EcoRI &amp; SpeI</TD></TR><TR><TD>4/15 D3</TD><TD>4/14 MP3</TD><TD>hmp in pGEX4T-1 cut EcoRI &amp; XhoI</TD></TR><TR><TD>4/15 D4</TD><TD>4/14 MP4</TD><TD>hmp in pGEX4T-1 cut EcoRI &amp; XhoI</TD></TR><TR><TD>4/15 D5</TD><TD>4/14 MP5</TD><TD>hmp in pGEX4T-1 cut EcoRI &amp; XhoI</TD></TR><TR><TD>4/15 D6</TD><TD>4/14 MP6</TD><TD>hmp in pGEX4T-1 cut EcoRI &amp; XhoI</TD></TR><TR><TD>4/15 D7</TD><TD>4/14 MP7</TD><TD>hmp in pGEX4T-1 cut EcoRI &amp; XhoI</TD></TR><TR><TD>4/15 D8</TD><TD>4/14 MP8</TD><TD>hmp in pGEX4T-1 cut EcoRI &amp; XhoI</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Re-try PCR amplification of LeupB and get a clean amplification of LeupA for digestion, then gel purification and ligation into pSB1C3. Obtain more pGEX4T-1 plasmid, digest then gel extract the backbone before ligating with hmp. Perform new transformations.

</DIV></DIV><DIV class="within-accordion"><H3>14 April 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: 6:45 pm</B></H4> <H4><B>Minipreps of LeupA in pSB1C3 and hmp in pGEX4T-1</B></H4><B>Purpose:</B> To miniprep transformants to obtain plasmids and verify their assembly
<H4><B>Protocol:</B></H4>
Eight minipreps were performed from the entire volume of broth culture using the kitless miniprep protocol. Spectrophotometry was used to estimate DNA concentration in the products.
<H4><B>Notes:</B></H4>
4/13 BC3 had no growth and was not miniprepped
<H4><B>Stop: 9:30 pm</B></H4> <H4><B>Results:</B></H4> <TABLE><TBODY><TR><TD><B>Sample</B></TD><TD><B>ng/μL</B></TD><TD><B>260/280</B></TD><TD><B>260/230</B></TD></TR><TR><TD>4/14 MP1</TD><TD>840.38</TD><TD>1.90</TD><TD>2.31</TD></TR><TR><TD>4/14 MP2</TD><TD>490.54</TD><TD>1.85</TD><TD>2.25</TD></TR><TR><TD>4/14 MP3</TD><TD>379.14</TD><TD>1.83</TD><TD>2.03</TD></TR><TR><TD>4/14 MP4</TD><TD>654.19</TD><TD>1.89</TD><TD>2.31</TD></TR><TR><TD>4/14 MP5</TD><TD>883.61</TD><TD>1.90</TD><TD>2.35</TD></TR><TR><TD>4/14 MP6</TD><TD>541.91</TD><TD>1.86</TD><TD>2.17</TD></TR><TR><TD>4/14 MP7</TD><TD>229.59</TD><TD>1.90</TD><TD>2.74</TD></TR><TR><TD>4/14 MP8</TD><TD>769.14</TD><TD>1.82</TD><TD>1.56</TD></TR></TBODY></TABLE> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>4/14 MP1</TD><TD>4/13 BC1</TD><TD>LeupA from 3/17 GE2 in pSB1C3</TD></TR><TR><TD>4/14 MP2</TD><TD>4/13 BC2</TD><TD>LeupA from 3/17 GE2 in pSB1C3</TD></TR><TR><TD>4/14 MP3</TD><TD>4/13 BC4</TD><TD>hmp in pGEX4T-1</TD></TR><TR><TD>4/14 MP4</TD><TD>4/13 BC5</TD><TD>hmp in pGEX4T-1</TD></TR><TR><TD>4/14 MP5</TD><TD>4/13 BC6</TD><TD>hmp in pGEX4T-1</TD></TR><TR><TD>4/14 MP6</TD><TD>4/13 BC7</TD><TD>hmp in pGEX4T-1</TD></TR><TR><TD>4/14 MP7</TD><TD>4/13 BC8</TD><TD>hmp in pGEX4T-1</TD></TR><TR><TD>4/14 MP8</TD><TD>4/13 BC9</TD><TD>hmp in pGEX4T-1</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Digest miniprep products and run a gel to verify identity

</DIV></DIV><DIV class="within-accordion"><H3>13 April 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday, Ryan Baumann, and Erin Nischwitz</B></H4><H4 class="h4right"><B>Start: 5:00 pm</B></H4> <H4><B>Digests for Verification of LeupA and hmp Transformants</B></H4><B>Purpose:</B> To verify the identity of plasmids miniprepped from LeupA and hmp transformants, and amplify LeupB
<H4><B>Protocol:</B></H4>
Two PCR reactions were incubated in the thermocycler:
<H5>4/13 A1</H5><TABLE><TBODY><TR><TD>3 μL</TD><TD>MilliQ water</TD></TR><TR><TD>10 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1 μL</TD><TD>10 μM LeupB_F</TD></TR><TR><TD>1 μL</TD><TD>10 μM LeupB_R</TD></TR><TR><TD>4 μL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>1 μL</TD><TD>1/6.3X 2/25 1</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE><H5>4/13 A2</H5> <TABLE><TBODY><TR><TD>3 μL</TD><TD>MilliQ water</TD></TR><TR><TD>10 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1 μL</TD><TD>10 μM LeupB_F</TD></TR><TR><TD>1 μL</TD><TD>10 μM LeupB_R</TD></TR><TR><TD>4 μL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>1 μL</TD><TD>1/3.7X 2/25 2</TD></TR><TR><TD>20 μL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c1</TD><TD>7</TD></TR><TR><TD>70-0.5c1</TD><TD>26</TD></TR><TR><TD>72c1</TD><TD>44</TD></TR><TR><TD>98c2</TD><TD>7</TD></TR><TR><TD>62c2</TD><TD>26</TD></TR><TR><TD>72c2</TD><TD>44</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle1 X16, cycle2 X14
4/13 A2 cracked in the thermocycler and was thrown away</P>
Four restriction digests were mixed then incubated for 1 hour at 37 °C and heat-killed in the thermocycler:
<H5>4/13 D1</H5><TABLE><TBODY><TR><TD>8 µL</TD><TD>MilliQ water</TD></TR><TR><TD>2.5 µL</TD><TD>10X Tango buffer</TD></TR><TR><TD>1 µL</TD><TD>EcoRI</TD></TR><TR><TD>1 µL</TD><TD>SpeI</TD></TR><TR><TD>12.5 µL</TD><TD>3/14 A1</TD></TR><TR><TD>25 µL</TD></TR></TBODY></TABLE><H5>4/13 D2</H5><TABLE><TBODY><TR><TD>8 µL</TD><TD>MilliQ water</TD></TR><TR><TD>2.5 µL</TD><TD>10X Tango buffer</TD></TR><TR><TD>1 µL</TD><TD>EcoRI</TD></TR><TR><TD>1 µL</TD><TD>SpeI</TD></TR><TR><TD>12.5 µL</TD><TD>3/12 A2</TD></TR><TR><TD>25 µL</TD></TR></TBODY></TABLE><H5>4/13 D3</H5><TABLE><TBODY><TR><TD>8.75 µL</TD><TD>MilliQ water</TD></TR><TR><TD>1.25 µL</TD><TD>10X Tango buffer</TD></TR><TR><TD>0.5 µL</TD><TD>EcoRI</TD></TR><TR><TD>2 µL</TD><TD>4/11 MP4</TD></TR><TR><TD>12.5 µL</TD><TD>(500 ng)</TD></TR></TBODY></TABLE><H5>4/13 D4</H5><TABLE><TBODY><TR><TD>8.75 μL</TD><TD>MilliQ water</TD></TR><TR><TD>1.25 μL</TD><TD>10X Tango buffer</TD></TR><TR><TD>0.5 μL</TD><TD>XhoI</TD></TR><TR><TD>2 μL</TD><TD>4/11 MP4</TD></TR><TR><TD>12.5 μL</TD><TD>(500 ng)</TD></TR></TBODY></TABLE>
A 1% agarose gel was run at 78 V until the bands reached 2/3 of the gel length:
<H5>4/13 Gel #1</H5><TABLE><TBODY><TR><TD>2</TD><TD>7.5 μL 4/13 A2</TD></TR><TR><TD>3</TD><TD>5 μL 2-log purple DNA ladder</TD></TR><TR><TD>4</TD><TD>25 μL 4/13 D1</TD></TR><TR><TD>5</TD><TD>25 μL 4/13 D2</TD></TR><TR><TD>6</TD><TD>12.5 μL 4/13 D3</TD></TR><TR><TD>7</TD><TD>12.5 μL 4/13 D4</TD></TR><TR><TD>8</TD><TD>5 μL 2-log purple DNA ladder</TD></TR></TBODY></TABLE>
5 µL of 1000X chloramphenicol or ampicillin was added to each of nine broth cultures, which were then inoculated from plates by wire loop.
<H4><B>Stop: 1:00 am</B></H4> <H4><B>Results:</B></H4><H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>4/13 A1</TD><TD>2/25 1</TD><TD>LeupB amplified from <I>S. roseus</I> ISP-5076</TD></TR><TR><TD>4/13 A2</TD><TD>2/25 2</TD><TD>LeupB amplified from <I>S. roseus</I> B-1320 (discarded)</TD></TR><TR><TD>4/13 D1</TD><TD>3/14 A1</TD><TD>LeupA amplified from <I>S. roseus</I> ISP-5076 digested EcoRI &amp; SpeI</TD></TR><TR><TD>4/13 D2</TD><TD>3/12 A2</TD><TD>LeupA amplified from <I>S. roseus</I> B-1320 digested EcoRI &amp; SpeI</TD></TR><TR><TD>4/13 D3</TD><TD>4/11 MP4</TD><TD>hmp in pGEX4T-1 digested EcoRI</TD></TR><TR><TD>4/13 D4</TD><TD>4/11 MP4</TD><TD>hmp in pGEX4T-1 digested XhoI</TD></TR><TR><TD>4/13 BC1</TD><TD>4/9 CT1 145 µL (white colony)</TD><TD>LeupA from 3/17 GE2 in pSB1C3 in chloramphenicol</TD></TR><TR><TD>4/13 BC2</TD><TD>4/9 CT1 145 µL (white colony)</TD><TD>LeupA from 3/17 GE2 in pSB1C3 in chloramphenicol</TD></TR><TR><TD>4/13 BC3</TD><TD>4/9 CT1 145 µL (white colony)</TD><TD>LeupA from 3/17 GE2 in pSB1C3 in chloramphenicol</TD></TR><TR><TD>4/13 BC4</TD><TD>4/9 CT2 145 µL (colony with satellites)</TD><TD>hmp in pGEX4T-1 in ampicillin</TD></TR><TR><TD>4/13 BC5</TD><TD>4/9 CT2 145 µL (colony with satellites)</TD><TD>hmp in pGEX4T-1 in ampicillin</TD></TR><TR><TD>4/13 BC6</TD><TD>4/9 CT2 145 µL (colony with satellites)</TD><TD>hmp in pGEX4T-1 in ampicillin</TD></TR><TR><TD>4/13 BC7</TD><TD>4/9 CT3 145 µL (colony with satellites)</TD><TD>hmp in pGEX4T-1 in ampicillin</TD></TR><TR><TD>4/13 BC8</TD><TD>4/9 CT3 145 µL (colony with satellites)</TD><TD>hmp in pGEX4T-1 in ampicillin</TD></TR><TR><TD>4/13 BC9</TD><TD>4/9 CT3 145 µL (colony with satellites)</TD><TD>hmp in pGEX4T-1 in ampicillin</TD></TR></TBODY></TABLE> </DIV></DIV><DIV class="within-accordion"><H3>11 April 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday and Erin Nischwitz</B></H4><H4 class="h4right"><B>Start: 5:00 pm</B></H4> <H4><B>Miniprep, digestion, and gel verification of LeupA in pSB1C3 and GST-tagged hmp</B></H4><B>Purpose:</B> To verify assembly of LeupA into pSB1C3 and hmp into pGEX4T-1
<H4><B>Protocol:</B></H4>
Four minipreps were performed using 3 mL of broth culture each under the kitless miniprep protocol. Product concentration was measured with the NanoDrop.

Four digestions were mixed and incubated in the thermocycler at 37 °C, then heat-killed:
<H5>4/11 D1</H5><TABLE><TBODY><TR><TD>19.3 µL</TD><TD>MilliQ water</TD></TR><TR><TD>2.5 µL</TD><TD>10X Tango buffer</TD></TR><TR><TD>1 µL</TD><TD>EcoRI</TD></TR><TR><TD>1 µL</TD><TD>SpeI</TD></TR><TR><TD>1.2 µL</TD><TD>4/11 MP1</TD></TR><TR><TD>25 µL</TD><TD>(1.5 µg)</TD></TR></TBODY></TABLE><H5>4/11 D2</H5><TABLE><TBODY><TR><TD>17.8 µL</TD><TD>MilliQ water</TD></TR><TR><TD>2.5 µL</TD><TD>10X Tango buffer</TD></TR><TR><TD>1 µL</TD><TD>EcoRI</TD></TR><TR><TD>1 µL</TD><TD>SpeI</TD></TR><TR><TD>2.7 µL</TD><TD>4/11 MP2</TD></TR><TR><TD>25 µL</TD><TD>(1.5 µg)</TD></TR></TBODY></TABLE><H5>4/11 D3</H5><TABLE><TBODY><TR><TD>19.1 µL</TD><TD>MilliQ water</TD></TR><TR><TD>2.5 µL</TD><TD>10X Tango buffer</TD></TR><TR><TD>1 µL</TD><TD>EcoRI</TD></TR><TR><TD>1 µL</TD><TD>SpeI</TD></TR><TR><TD>1.4 µL</TD><TD>4/11 MP3</TD></TR><TR><TD>25 µL</TD><TD>(1.5 µg)</TD></TR></TBODY></TABLE><H5>4/11 D4</H5><TABLE><TBODY><TR><TD>14.6 µL</TD><TD>MilliQ water</TD></TR><TR><TD>2.5 µL</TD><TD>10X Tango buffer</TD></TR><TR><TD>1 µL</TD><TD>EcoRI</TD></TR><TR><TD>1 µL</TD><TD>XhoI</TD></TR><TR><TD>5.9 µL</TD><TD>4/11 MP4</TD></TR><TR><TD>25 µL</TD><TD>(1.5 µg)</TD></TR></TBODY></TABLE> 

A 1% agarose gel was prepared and run at 120 V for 45 minutes:
<H5>4/11 Gel #1</H5><TABLE><TBODY><TR><TD>2</TD><TD>5 µL 2-log purple DNA ladder</TD></TR><TR><TD>3</TD><TD>7.5 µL 4/11 D1</TD></TR><TR><TD>4</TD><TD>7.5 µL 4/11 D2</TD></TR><TR><TD>5</TD><TD>7.5 µL 4/11 D3</TD></TR><TR><TD>6</TD><TD>7.5 µL 4/11 D4</TD></TR></TBODY></TABLE> 

Two ligations were mixed and incubated at room temperature overnight:
<H5>4/11 L1</H5><TABLE><TBODY><TR><TD>1 µL</TD><TD>10X T4 ligase buffer</TD></TR><TR><TD>1 µL</TD><TD>1/19X 3/26 D2</TD></TR><TR><TD>7 µL</TD><TD>4/10 D1</TD></TR><TR><TD>1 µL</TD><TD>T4 DNA ligase</TD></TR><TR><TD>10 µL</TD></TR></TBODY></TABLE><H5>4/11 L2</H5><TABLE><TBODY><TR><TD>1 µL</TD><TD>10X T4 ligase buffer</TD></TR><TR><TD>1 µL</TD><TD>1/50X 3/26 D2</TD></TR><TR><TD>7 µL</TD><TD>4/10 D2</TD></TR><TR><TD>1 µL</TD><TD>T4 DNA ligase</TD></TR><TR><TD>10 µL</TD></TR></TBODY></TABLE> <H4><B>Stop: 11:30 pm</B></H4> <H4><B>Results:</B></H4> <TABLE><TBODY><TR><TD><B>Sample</B></TD><TD><B>ng/μL</B></TD><TD><B>260/280</B></TD><TD><B>260/230</B></TD></TR><TR><TD>4/11 MP1</TD><TD>1333.07</TD><TD>1.91</TD><TD>2.37</TD></TR><TR><TD>4/11 MP2</TD><TD>551.06</TD><TD>1.87</TD><TD>2.32</TD></TR><TR><TD>4/11 MP3</TD><TD>1058.70</TD><TD>1.91</TD><TD>2.32</TD></TR><TR><TD>4/11 MP4</TD><TD>254.90</TD><TD>1.89</TD><TD>2.54</TD></TR></TBODY></TABLE><H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>4/11 MP1</TD><TD>4/10 BC1</TD><TD>LeupA from 3/17 GE2 in pSB1C3</TD></TR><TR><TD>4/11 MP2</TD><TD>4/10 BC2</TD><TD>LeupA from 3/17 GE2 in pSB1C3</TD></TR><TR><TD>4/11 MP3</TD><TD>4/10 BC3</TD><TD>LeupA from 3/17 GE2 in pSB1C3</TD></TR><TR><TD>4/11 MP4</TD><TD>4/10 BC6</TD><TD>hmp in pGEX4T-1</TD></TR><TR><TD>4/11 D1</TD><TD>4/11 MP1</TD><TD>LeupA from 3/17 GE2 in pSB1C3 digested EcoRI &amp; SpeI</TD></TR><TR><TD>4/11 D2</TD><TD>4/11 MP2</TD><TD>LeupA from 3/17 GE2 in pSB1C3 digested EcoRI &amp; SpeI</TD></TR><TR><TD>4/11 D3</TD><TD>4/11 MP3</TD><TD>LeupA from 3/17 GE2 in pSB1C3 digested EcoRI &amp; SpeI</TD></TR><TR><TD>4/11 D4</TD><TD>4/11 MP4</TD><TD>hmp in pGEX4T-1 digested EcoRI &amp; XhoI</TD></TR><TR><TD>4/11 L1</TD><TD>4/10 D1, 3/26 D2</TD><TD>LeupA from 3/17 GE1 in pSB1C3</TD></TR><TR><TD>4/11 L2</TD><TD>4/10 D2, 3/26 D2</TD><TD>LeupA from 3/17 GE3 in pSB1C3</TD></TR></TBODY></TABLE> </DIV></DIV><DIV class="within-accordion"><H3>10 April 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday and Erin Nischwitz</B></H4><H4 class="h4right"><B>Start: 10:30 am</B></H4> <H4><B>New LeupB PCR</B></H4><B>Purpose:</B> to amplify LeupB from <I>Streptomyces roseus</I><H4><B>Protocol:</B></H4>
New dilutions of primers were performed, then one PCR was incubated in the thermocycler:
<H5>4/10 A1</H5><TABLE><TBODY><TR><TD>3 µL</TD><TD>MilliQ water</TD></TR><TR><TD>10 µL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1 µL</TD><TD>10 µM LeupB_F</TD></TR><TR><TD>1 µL</TD><TD>10 µM LeupB_R</TD></TR><TR><TD>4 µL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>1 µL</TD><TD>1/3.7X 2/25 2</TD></TR><TR><TD>20 µL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>7</TD></TR><TR><TD>68c</TD><TD>26</TD></TR><TR><TD>72c</TD><TD>44</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X30</P>
A 1% agarose gel was prepared and run at 130 V for 45 minutes:
<H5>Gel 4/10 #1</H5><TABLE><TBODY><TR><TD>3</TD><TD>5 µL 2-log purple DNA ladder</TD></TR><TR><TD>5</TD><TD>7.5 µL 4/10 A1</TD></TR></TBODY></TABLE>
Six broth cultures were inoculated from plates. Two restriction digests were mixed, incubated, then heat killed in the thermocycler:
<H5>4/10 D1</H5><TABLE><TBODY><TR><TD>2.5 µL</TD><TD>10X Tango buffer</TD></TR><TR><TD>1 µL</TD><TD>EcoRI</TD></TR><TR><TD>1 µL</TD><TD>SpeI</TD></TR><TR><TD>20.5 µL</TD><TD>3/17 GE1</TD></TR><TR><TD>25 µL</TD><TD>(65.8 ng)</TD></TR></TBODY></TABLE><H5>4/10 D2</H5><TABLE><TBODY><TR><TD>2.5 µL</TD><TD>10X Tango buffer</TD></TR><TR><TD>1 µL</TD><TD>EcoRI</TD></TR><TR><TD>1 µL</TD><TD>SpeI</TD></TR><TR><TD>20.5 µL</TD><TD>3/17 GE3</TD></TR><TR><TD>25 µL</TD><TD>(31.2 ng)</TD></TR></TBODY></TABLE> <H4><B>Stop: 10:00 pm</B></H4> <H4><B>Results:</B></H4><H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>4/10 A1</TD><TD>2/25 2</TD><TD>LeupB amplified from <I>S. roseus</I> B-1320</TD></TR><TR><TD>4/10 D1</TD><TD>3/17 GE1</TD><TD>LeupA amplified and gel extracted, digested EcoRI &amp; SpeI</TD></TR><TR><TD>4/10 D2</TD><TD>3/17 GE3</TD><TD>LeupA amplified and gel extracted, digested EcoRI &amp; SpeI</TD></TR><TR><TD>4/10 BC1</TD><TD>4/9 CT1 145 µL (white colony)</TD><TD>LeupA from 3/17 GE2 in pSB1C3 in chloramphenicol</TD></TR><TR><TD>4/10 BC2</TD><TD>4/9 CT1 145 µL (white colony)</TD><TD>LeupA from 3/17 GE2 in pSB1C3 in chloramphenicol</TD></TR><TR><TD>4/10 BC3</TD><TD>4/9 CT1 145 µL (white colony)</TD><TD>LeupA from 3/17 GE2 in pSB1C3 in chloramphenicol</TD></TR><TR><TD>4/10 BC4</TD><TD>4/9 CT2 145 µL (independent colony)</TD><TD>hmp in pGEX4T-1 in ampicillin</TD></TR><TR><TD>4/10 BC5</TD><TD>4/9 CT2 145 µL (independent colony)</TD><TD>hmp in pGEX4T-1 in ampicillin</TD></TR><TR><TD>4/10 BC6</TD><TD>4/9 CT2 145 µL (colony with satellites)</TD><TD>hmp in pGEX4T-1 in ampicillin</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Re-try LeupB amplification. Miniprep broth cultures, then digest and run a gel to verify product identity.

</DIV></DIV><DIV class="within-accordion"><H3>9 April 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: 9:45 am</B></H4> <H4><B>PCR of LeupB, Ligation and Transformation of LeupA and hmp</B></H4><B>Purpose:</B> To isolate LeupB from <I>Streptomyces roseus</I> and clone LeupA and GST-tagged hmp.
<H4><B>Protocol:</B></H4>
Three PCRs were prepared and reacted in the thermocycler:
<H5>4/9 A1</H5><TABLE><TBODY><TR><TD>3 µL</TD><TD>MilliQ water</TD></TR><TR><TD>4 µL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>10 µL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1 µL</TD><TD>10 µM LeupB_F</TD></TR><TR><TD>1 µL</TD><TD>10 µM LeupB_R</TD></TR><TR><TD>1 µL</TD><TD>1/6.3X 2/25 1</TD></TR><TR><TD>20 µL</TD></TR></TBODY></TABLE><H5>4/9 A2</H5><TABLE><TBODY><TR><TD>3 µL</TD><TD>MilliQ water</TD></TR><TR><TD>4 µL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>10 µL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1 µL</TD><TD>10 µM LeupB_F</TD></TR><TR><TD>1 µL</TD><TD>10 µM LeupB_R</TD></TR><TR><TD>1 µL</TD><TD>1/3.7X 2/25 2</TD></TR><TR><TD>20 µL</TD></TR></TBODY></TABLE><H5>4/9 A3</H5><TABLE><TBODY><TR><TD>3 µL</TD><TD>MilliQ water</TD></TR><TR><TD>4 µL</TD><TD>5X Q5 High GC Enhancer</TD></TR><TR><TD>10 µL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1 µL</TD><TD>10 µM LeupB_F</TD></TR><TR><TD>1 µL</TD><TD>10 µM LeupB_R</TD></TR><TR><TD>1 µL</TD><TD>1/8.5X 2/25 3</TD></TR><TR><TD>20 µL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>7</TD></TR><TR><TD>69.6c</TD><TD>24</TD></TR><TR><TD>72c</TD><TD>40</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X30</P>
A 1% agarose gel was prepared and run at 130 V for 45 minutes:
<H5>Gel 4/9 #1</H5><TABLE><TBODY><TR><TD>1</TD><TD>5 µL 3/26 D1</TD></TR><TR><TD>2</TD><TD>5 µL 3/26 D2</TD></TR><TR><TD>3</TD><TD>7.5 µL 2-log purple DNA ladder</TD></TR><TR><TD>5</TD><TD>7.5 µL 4/9 A1</TD></TR><TR><TD>7</TD><TD>7.5 µL 4/9 A2</TD></TR><TR><TD>9</TD><TD>7.5 µL 4/9 A3</TD></TR></TBODY></TABLE>
note: pSB1C3 in lanes 1 and 2 is correct but appears to contain some undigested or single-digested product. Since the two digestions only have EcoRI in common, this could be the culprit, but it should not matter if red colonies (religated) are selected against. No amplifications of LeupB were successful.

Two ligations were mixed and reacted at room temperature for 3 hours:
<H5>4/9 L1</H5><TABLE><TBODY><TR><TD>1 µL</TD><TD>10X T4 ligase buffer</TD></TR><TR><TD>1 µL</TD><TD>1/20X 3/26 D2 (1.1 fmol pSB1C3)</TD></TR><TR><TD>7 µL</TD><TD>3/26 D3 (4.5 fmol LeupA)</TD></TR><TR><TD>1 µL</TD><TD>T4 DNA ligase</TD></TR><TR><TD>10 µL</TD></TR></TBODY></TABLE><H5>4/9 L2</H5><TABLE><TBODY><TR><TD>1 µL</TD><TD>10X T4 ligase buffer</TD></TR><TR><TD>1 µL</TD><TD>1/2X 10/11 D5</TD></TR><TR><TD>7 µL</TD><TD>2/20 D2</TD></TR><TR><TD>1 µL</TD><TD>T4 DNA ligase</TD></TR><TR><TD>10 µL</TD></TR></TBODY></TABLE>
Three chemical transformations were performed into Zymo Mix &amp; Go 5α. 1.4 µL of each plasmid was mixed with 33 µL aliquots, then incubated for 7 minutes on ice. 140 µL of SOC was added for 1 hour of outgrowth at 37 °C, then 145 µL and 14.5 µL were plated.
<H4><B>Stop: 9:30 pm</B></H4> <H4><B>Results:</B></H4><H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>4/9 A1</TD><TD>2/25 1</TD><TD>LeupB amplified from <I>S. roseus</I> ISP-5076</TD></TR><TR><TD>4/9 A2</TD><TD>2/25 2</TD><TD>LeupB amplified from <I>S. roseus</I> B-1320</TD></TR><TR><TD>4/9 A3</TD><TD>2/25 3</TD><TD>LeupB amplified from <I>S. roseus</I> B-3062</TD></TR><TR><TD>4/9 L1</TD><TD>3/26 D2, 3/26 D3</TD><TD>LeupA from 3/17 GE2 in pSB1C3</TD></TR><TR><TD>4/9 L2</TD><TD>10/11 D5, 2/20 D2</TD><TD>hmp in pGEX4T-1</TD></TR><TR><TD>4/9 CT1</TD><TD>4/9 L1</TD><TD>LeupA in pSB1C3 on chloramphenicol</TD></TR><TR><TD>4/9 CT2</TD><TD>4/9 L2</TD><TD>hmp in pGEX4T-1 on ampicillin</TD></TR><TR><TD>4/9 CT3</TD><TD>3/1 L1</TD><TD>hmp in pGEX4T-1 on ampicillin</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Culture and miniprep non-red colonies of 4/9 CT1, CT2, and CT3. Digest and run a gel to verify product identity.

</DIV></DIV><DIV class="within-accordion"><H3>26 March 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4></DIV></DIV><DIV class="within-accordion"><H3>24 March 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4></DIV></DIV><DIV class="within-accordion"><H3>23 March 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4></DIV></DIV><DIV class="within-accordion"><H3>21 March 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4></DIV></DIV><DIV class="within-accordion"><H3>18 March 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4></DIV></DIV><DIV class="within-accordion"><H3>17 March 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4></DIV></DIV><DIV class="within-accordion"><H3>14 March 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4></DIV></DIV><DIV class="within-accordion"><H3>13 March 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4></DIV></DIV><DIV class="within-accordion"><H3>12 March 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4></DIV></DIV><DIV class="within-accordion"><H3>11 March 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: 1:00 pm</B></H4> <H4><B>PCR Amplification of LeupA</B></H4><B>Purpose:</B> To isolate LeupA from the genome of 3 <I>Streptomyces roseus</I> strains.
<H4><B>Protocol:</B></H4>
Three PCRs were prepared and reacted in the thermocycler with the same program:
<H5>3/11 A1</H5><TABLE><TBODY><TR><TD>9 µL</TD><TD>MilliQ water</TD></TR><TR><TD>12.5 µL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1.25 µL</TD><TD>10 µM LeupA_F_Val</TD></TR><TR><TD>1.25 µL</TD><TD>10 µM LeupA_R</TD></TR><TR><TD>1 µL</TD><TD>1/6.3X 2/25 1</TD></TR><TR><TD>25 µL</TD></TR></TBODY></TABLE><H5>3/11 A2</H5><TABLE><TBODY><TR><TD>9 µL</TD><TD>MilliQ water</TD></TR><TR><TD>12.5 µL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1.25 µL</TD><TD>10 µM LeupA_F_Val</TD></TR><TR><TD>1.25 µL</TD><TD>10 µM LeupA_R</TD></TR><TR><TD>1 µL</TD><TD>1/3.7X 2/25 2</TD></TR><TR><TD>25 µL</TD></TR></TBODY></TABLE><H5>3/11 A3</H5><TABLE><TBODY><TR><TD>9 µL</TD><TD>MilliQ water</TD></TR><TR><TD>12.5 µL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1.25 µL</TD><TD>10 µM LeupA_F_Val</TD></TR><TR><TD>1.25 µL</TD><TD>10 µM LeupA_R</TD></TR><TR><TD>1 µL</TD><TD>1/8.5X 2/25 3</TD></TR><TR><TD>25 µL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>7</TD></TR><TR><TD>72c</TD><TD>204</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X30</P>
A 1% agarose gel was run at 130 V for 45 minutes:
<H5>Gel 3/11 #1</H5><TABLE><TBODY><TR><TD>3</TD><TD>5 µL 2-log purple DNA ladder</TD></TR><TR><TD>4</TD><TD>5 µL 3/11 A1</TD></TR><TR><TD>5</TD><TD>5 µL 3/11 A2</TD></TR><TR><TD>6</TD><TD>5 µL 3/11 A3</TD></TR></TBODY></TABLE>
note: no bands at the expected 7.3 kbp, strong undesired band in 3/11 A2
<H4><B>Stop: 5:00 pm</B></H4> <H4><B>Results:</B></H4><H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>3/11 A1</TD><TD>2/25 1</TD><TD>LeupA amplified from <I>S. roseus</I> ISP-5076</TD></TR><TR><TD>3/11 A2</TD><TD>2/25 2</TD><TD>LeupA amplified from <I>S. roseus</I> B-1320</TD></TR><TR><TD>3/11 A3</TD><TD>2/25 3</TD><TD>LeupA amplified from <I>S. roseus</I> B-3062</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Re-try amplification with Q5 High GC Enhancer

</DIV></DIV><DIV class="within-accordion"><H3>4 March 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: 4:00 pm</B></H4> <H4><B>Gel of Part-1 and Positive Control PCRs</B></H4><B>Purpose:</B> to verify amplification of Part-1 and Q5 positive control
<H4><B>Protocol:</B></H4>
A 1% agarose gel was run at 130 V for 40 minutes:
<H5>3/4 Gel #1</H5><TABLE><TBODY><TR><TD>5</TD><TD>10μL 3/3 A1</TD></TR><TR><TD>6</TD><TD>10μL 3/3 A2</TD></TR><TR><TD>7</TD><TD>10μL 2-log purple DNA ladder</TD></TR></TBODY></TABLE> <H4><B>Stop: 5:50 pm</B></H4> <H4><B>Results:</B></H4><H4><B>Next:</B></H4> </DIV></DIV><DIV class="within-accordion"><H3>3 March 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: 10:00 am</B></H4> <H4><B>Part-1 and Positive Control PCRs</B></H4><B>Purpose:</B> to amplify Part-1 for verification of its assembly, and test 2X Q5 Master Mix
<H4><B>Protocol:</B></H4>
Two PCR reactions were performed in the thermocycler under the same program:
<H5>3/3 A1</H5><TABLE><TBODY><TR><TD>7 µL</TD><TD>MilliQ water</TD></TR><TR><TD>10 µL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1 µL</TD><TD>10 µM VF2</TD></TR><TR><TD>1 µL</TD><TD>10 µM VR</TD></TR><TR><TD>1 µL</TD><TD>1/10X 3/1 L2</TD></TR><TR><TD>20 µL</TD></TR></TBODY></TABLE><H5>3/3 A2</H5><TABLE><TBODY><TR><TD>7 µL</TD><TD>MilliQ water</TD></TR><TR><TD>10 µL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1 µL</TD><TD>10 µM VF2</TD></TR><TR><TD>1 µL</TD><TD>10 µM VR</TD></TR><TR><TD>1 µL</TD><TD>5 pg/µL RFP construct</TD></TR><TR><TD>20 µL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>7</TD></TR><TR><TD>65c</TD><TD>20</TD></TR><TR><TD>72c</TD><TD>100</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X32</P> <H4><B>Stop: 11:00 am</B></H4> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>3/3 A1</TD><TD>3/1 L2</TD><TD>Part-1 assembled, ligated into pBSIC3 and amplified (Q5)</TD></TR><TR><TD>3/3 A2</TD><TD>RFP in pSBB1C3</TD><TD>Positive control of 2X Q5 Master Mix</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Run a gel of PCRs, also see Mar. 1.

</DIV></DIV><DIV class="within-accordion"><H3>1 March 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: 8:47 pm</B></H4> <H4><B>Ligation of GST-tagged hmp</B></H4><B>Purpose:</B> to purify the hmp protein and verify its mass, also demonstrating GST-tagging for characterization of future proteins.
<H4><B>Protocol:</B></H4>
Two ligations were mixed and left to react at room temperature overnight:
<H5>3/1 L1</H5><TABLE><TBODY><TR><TD>1 µL</TD><TD>10X T4 buffer</TD></TR><TR><TD>2 µL</TD><TD>10/11 D5</TD></TR><TR><TD>6 µL</TD><TD>2/20 D2</TD></TR><TR><TD>1 µL</TD><TD>T4 DNA ligase</TD></TR><TR><TD>10 µL</TD></TR></TBODY></TABLE><H5>3/1 L2</H5><TABLE><TBODY><TR><TD>1 µL</TD><TD>10X T4 buffer</TD></TR><TR><TD>4.6 µL</TD><TD>8/18 D1</TD></TR><TR><TD>3.4 µL</TD><TD>2/20 D3</TD></TR><TR><TD>1 µL</TD><TD>T4 DNA ligase</TD></TR><TR><TD>10 µL</TD></TR></TBODY></TABLE><P class="centerp">(repeat of 2/20 L1 with longer incubation and more homogeneous T4 buffer)</P> <H4><B>Stop: 9:30 pm</B></H4> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>3/1 L1</TD><TD>10/11 D5, 2/20 D2</TD><TD>hmp in pGEX4T-1</TD></TR><TR><TD>3/1 L2</TD><TD>8/18 D1, 2/20 D3</TD><TD>part 1 assembled in pSBIC3</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Transform 3/1 L1 (onto amp), purify hmp using GST columns, run a SDS-PAGE gel of GST-tagged hmp, cleaved, uncleaved, or both. Amplify LeupA and LeupB from all three 2/25 extractions. Attempt to amplify 3/1 L2 with VR &amp; VF2, also attempting a Q5 control with these primers/T<SUB>A</SUB></DIV></DIV><DIV class="within-accordion"><H3>27 February 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: 8:45 am</B></H4> <H4><B>Gibson assembly troubleshooting gel</B></H4><B>Purpose:</B> to diagnose Gibson assembly/HiFi master mix failures
<H4><B>Protocol:</B></H4>
A 1% agarose gel was run for 45 min at 130 V:
<H5>2/27 Gel #1</H5><TABLE><TBODY><TR><TD>4</TD><TD>5 µL 2-log purple DNA ladder</TD></TR><TR><TD>5</TD><TD>10 µL 2/20 HF1</TD></TR></TBODY></TABLE> <H4><B>Notes:</B></H4>
only the ratio between gblocks is significant because of troubles while loading gel
<H4><B>Stop: 10:00 am</B></H4> <H4><B>Results:</B></H4>
Image J and the ladder were used to estimate band mass and thus moles present:
<TABLE><TBODY><TR><TD><B>Band (bp)</B></TD><TD><B>Mass (ng)</B></TD><TD><B>Amount (fmol)</B></TD></TR><TR><TD>850</TD><TD>4.008</TD><TD>7.630</TD></TR><TR><TD>1000</TD><TD>4.928</TD><TD>7.974</TD></TR><TR><TD>1150</TD><TD>4.436</TD><TD>4.835</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
See Feb. 25 and Feb.20 hmp work.

</DIV></DIV><DIV class="within-accordion"><H3>25 February 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: 5:30 pm</B></H4> <H4><B><I>S. roseus</I> DNA extractions</B></H4><B>Purpose:</B> To obtain chromosomal DNA for amplification from <I>S. roseus</I> and verify amplification of ocimene synthase.
<H4><B>Protocol:</B></H4>
Three glycerol stocks were prepared and left in rm. 206 freezer.

2 mL of each broth culture was used with Machery-Nagel NucleoSpin Microbial DNA kit to produce 100 μL of each chromosomal DNA solution. No proteinase K was used, and products were nanodropped to determine concentration.

A 1% agarose gel was run for 40 min at 130 V:
<H5>2/25 Gel #1</H5><TABLE><TBODY><TR><TD>2</TD><TD>5μL 2-log purple DNA ladder</TD></TR><TR><TD>3</TD><TD>5μL 2/23 A1</TD></TR></TBODY></TABLE> <H4><B>Stop: 9:10 pm</B></H4> <H4><B>Results:</B></H4> <TABLE><TBODY><TR><TD><B>Sample</B></TD><TD><B>ng/μL</B></TD><TD><B>260/280</B></TD><TD><B>260/230</B></TD></TR><TR><TD>2/25 1</TD><TD>56.06</TD><TD>1.95</TD><TD>1.73</TD></TR><TR><TD>2/25 2</TD><TD>33.21</TD><TD>1.86</TD><TD>1.04</TD></TR><TR><TD>2/25 3</TD><TD>75.80</TD><TD>1.92</TD><TD>1.74</TD></TR></TBODY></TABLE><H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>2/25 S1</TD><TD>2/21 BC1</TD><TD><I>S. roseus</I> ISP-5076 glycerol stock</TD></TR><TR><TD>2/25 S2</TD><TD>2/21 BC2</TD><TD><I>S. roseus</I> B-1320 glycerol stock</TD></TR><TR><TD>2/25 S3</TD><TD>2/21 BC3</TD><TD><I>S. roseus</I> B-3062 glycerol stock</TD></TR><TR><TD>2/25 1</TD><TD>2/21 BC1</TD><TD><I>S. roseus</I> ISP-5076 genomic DNA</TD></TR><TR><TD>2/25 2</TD><TD>2/21 BC2</TD><TD><I>S. roseus</I> B-1320 genomic DNA</TD></TR><TR><TD>2/25 3</TD><TD>2/21 BC3</TD><TD><I>S. roseus</I> B-3062 genomic DNA</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Amplify LeupA and LeupB from genomic DNA, see Feb. 20 hmp work.

</DIV></DIV><DIV class="within-accordion"><H3>23 February 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: 7:10 pm</B></H4> <H4><B>Ocimene synthase amplification</B></H4><B>Purpose:</B> amplify part 3 gblock (ocimene synthase) for cloning into pSB1C3
<H4><B>Protocol:</B></H4>
One PCR was reacted in the thermocycler:
<H5>2/23 A1</H5><TABLE><TBODY><TR><TD>9 μL</TD><TD>MilliQ water</TD></TR><TR><TD>12.5 μL</TD><TD>2X Q5 Master Mix</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM VF2</TD></TR><TR><TD>1.25 μL</TD><TD>10 μM OciR</TD></TR><TR><TD>1 μL</TD><TD>Part 3 gblock (small drop diluted)</TD></TR><TR><TD>25 μL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>7</TD></TR><TR><TD>65c</TD><TD>20</TD></TR><TR><TD>72c</TD><TD>53</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X32</P> <H4><B>Stop: 7:40 pm</B></H4> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>2/23 A1</TD><TD>Part 3 gblock</TD><TD>ocimene synthase amplified by Q5 using VF2 &amp; OciR</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
See Feb. 20 and Feb. 21, also run a gel of 2/23 A1 (recommended 1% 130 V for 40 min, load 5μL)

</DIV></DIV><DIV class="within-accordion"><H3>23 February 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: 10:00 am</B></H4> <H4><B>Further isolation of <I>Streptomyces roseus</I></B></H4><B>Purpose:</B> To obtain chromosomal DNA from <I>S. roseus</I> strains to clone leupeptin genes from.
<H4><B>Protocol:</B></H4>
Three broth cultures and three isolation plates were inoculated from previous plates.
<H4><B>Stop: 10:45 am</B></H4> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>2/23 BC1</TD><TD>2/21 I1</TD><TD><I>S. roseus</I> ISP-5076 on ISP4</TD></TR><TR><TD>2/23 BC2</TD><TD>2/21 I2</TD><TD><I>S. roseus</I> B-1320 on ISP4</TD></TR><TR><TD>2/23 BC3</TD><TD>2/21 I3</TD><TD><I>S. roseus</I> B-3062 on ISP4</TD></TR><TR><TD>2/23 I1</TD><TD>2/21 I1</TD><TD><I>S. roseus</I> ISP-5076 on ISP4</TD></TR><TR><TD>2/23 I2</TD><TD>2/21 I2</TD><TD><I>S. roseus</I> B-1320 on ISP4</TD></TR><TR><TD>2/23 I3</TD><TD>2/21 I3</TD><TD><I>S. roseus</I> B-3062 on ISP4</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
See Feb. 20 and Feb. 21.

</DIV></DIV><DIV class="within-accordion"><H3>21 February 2016</H3><DIV><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: 11:00 am</B></H4> <H4><B>Cultures of <I>Streptomyces roseus</I></B></H4><B>Purpose:</B> To obtain chromosomal DNA from <I>S. roseus</I> strains to clone leupeptin genes from.
<H4><B>Protocol:</B></H4>
Three broth cultures and three isolation plates were inoculated from NRRL ampuoles.
<H4><B>Notes:</B></H4>
BCs incubated in benchtop incubator set to 28°C, actually 25°C. Plates in 30°C stationary.
<H4><B>Stop: 11:40 am</B></H4> <H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Description</B></TD></TR><TR><TD>2/21 BC1</TD><TD><I>S. roseus</I> ISP-5076 in ISP1 (green cap)</TD></TR><TR><TD>2/21 BC2</TD><TD><I>S. roseus</I> B-1320 in ISP1 (yellow)</TD></TR><TR><TD>2/21 BC3</TD><TD><I>S. roseus</I> B-3062 in ISP1 (red)</TD></TR><TR><TD>2/21 I1</TD><TD><I>S. roseus</I> ISP-5076 on ISP4</TD></TR><TR><TD>2/21 I2</TD><TD><I>S. roseus</I> B-1320 on ISP4</TD></TR><TR><TD>2/21 I3</TD><TD><I>S. roseus</I> B-3062 on ISP4</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
See Feb. 20, also ensure pure cultures, create a stock of each strain, use Dr. Westenberg's chromosomal DNA extraction kit.

</DIV></DIV><DIV class="within-accordion"><H3>20 February 2016</H3><DIV id="entry1" class="notebook"><H4 class="h4left"><B>Kent Gorday</B></H4><H4 class="h4right"><B>Start: 8:00 am</B></H4> <H4><B>Assembly of Part 1, Ocimene Synthase, and GST-tagged hmp</B></H4><B>Purpose:</B> To assemble part 1 and ocimene synthase into pSB1C3 for registry submission, transfer hmp into pGEX4T-1 for GST-tag purification, and culture <I>Streptomyces roseus</I> to obtain its chromosomal DNA.
<H4><B>Protocol:</B></H4>
Three PCRs were mixed as below and incubated in the thermocycler with given programs:
<H5>2/20 A1</H5><TABLE><TBODY><TR><TD>3 µL</TD><TD>5X Q5 Reaction Buffer</TD></TR><TR><TD>6.8 µL</TD><TD>MilliQ water</TD></TR><TR><TD>0.75 µL</TD><TD>10 µM VF2</TD></TR><TR><TD>0.75 µL</TD><TD>1/10X 100 µM OciS R Primer</TD></TR><TR><TD>1.2 µL</TD><TD>2.5 mM dNTPs</TD></TR><TR><TD>1 µL</TD><TD>Part 3 gblock (tiny drop diluted)</TD></TR><TR><TD>1.5 µL</TD><TD>1/10X Q5 Polymerase</TD></TR><TR><TD>15 µL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>98</TD><TD>30</TD></TR><TR><TD>98c</TD><TD>7</TD></TR><TR><TD>66c</TD><TD>20</TD></TR><TR><TD>72c</TD><TD>45</TD></TR><TR><TD>72</TD><TD>120</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X32</P> <H5>2/20 A2</H5><TABLE><TBODY><TR><TD>12.5 µL</TD><TD>2X Taq Master Mix</TD></TR><TR><TD>10.5 µL</TD><TD>MilliQ water</TD></TR><TR><TD>0.5 µL</TD><TD>10 µM hmp F Primer 7/2</TD></TR><TR><TD>0.5 µL</TD><TD>10 µM hmp R Primer 7/2</TD></TR><TR><TD>1 µL</TD><TD>1/800X 2/11 KG</TD></TR><TR><TD>25 µL</TD></TR></TBODY></TABLE><P class="centerp">note: 2/11 KG initially 338.5 ng/µL</P><TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>95</TD><TD>30</TD></TR><TR><TD>95c</TD><TD>23</TD></TR><TR><TD>60c</TD><TD>43</TD></TR><TR><TD>68c</TD><TD>95</TD></TR><TR><TD>68</TD><TD>300</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X32</P> <H5>2/20 A3</H5><TABLE><TBODY><TR><TD>12.5 µL</TD><TD>2X Taq Master Mix</TD></TR><TR><TD>10.5 µL</TD><TD>MilliQ water</TD></TR><TR><TD>0.5 µL</TD><TD>10 µM VF2</TD></TR><TR><TD>0.5 µL</TD><TD>1/10X 100 µM OciS R Primer</TD></TR><TR><TD>1 µL</TD><TD>Part 3 gblock (tiny drop diluted)</TD></TR><TR><TD>25 µL</TD></TR></TBODY></TABLE> <TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>95</TD><TD>30</TD></TR><TR><TD>95c</TD><TD>23</TD></TR><TR><TD>51c</TD><TD>43</TD></TR><TR><TD>68c</TD><TD>120</TD></TR><TR><TD>68</TD><TD>300</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE><P class="centerp">cycle X32</P>
One HiFi assembly reaction was performed with only the four insert fragments of Part 1:
<H5>2/20 HF1</H5><TABLE><TBODY><TR><TD>10 µL</TD><TD>2X HiFi Master Mix</TD></TR><TR><TD>2.1 µL</TD><TD>Part 1-1 gblock</TD></TR><TR><TD>2.5 µL</TD><TD>Part 1-2 gblock</TD></TR><TR><TD>2.5 µL</TD><TD>Part 1-3 gblock</TD></TR><TR><TD>2.9 µL</TD><TD>Part 1-4 gblock</TD></TR><TR><TD>20 µL</TD></TR></TBODY></TABLE>
note: roughly 0.4045 pmol of each
50 °C for 60 minutes in thermocycler

A 1% agarose gel was run at 130 V for 45 minutes:
<H5>Gel 2/20 #1</H5><TABLE><TBODY><TR><TD>2</TD><TD>5 µL 2-log purple DNA ladder</TD></TR><TR><TD>4</TD><TD>5 µL 2/20 A1</TD></TR><TR><TD>6</TD><TD>5 µL 2/20 A2</TD></TR></TBODY></TABLE>
note: 2/20 A2 yielded a strong band near ~1.3kbp expected product, 2/20 A1 gave no coherent product

Three restriction digests were mixed and incubated at 37 °C for 60 minutes before a 20 minute heat kill at 80 °C in thermocycler:
<H5>2/20 D1</H5><TABLE><TBODY><TR><TD>2.5 µL</TD><TD>10X Tango Buffer</TD></TR><TR><TD>20 µL</TD><TD>MilliQ water</TD></TR><TR><TD>1 µL</TD><TD>EcoRI</TD></TR><TR><TD>1 µL</TD><TD>PstI</TD></TR><TR><TD>0.5 µL</TD><TD>2/20 A3</TD></TR><TR><TD>25 µL</TD></TR></TBODY></TABLE><H5>2/20 D2</H5><TABLE><TBODY><TR><TD>2.5 µL</TD><TD>10X Tango Buffer</TD></TR><TR><TD>20 µL</TD><TD>MilliQ water</TD></TR><TR><TD>1 µL</TD><TD>EcoRI</TD></TR><TR><TD>1 µL</TD><TD>XhoI</TD></TR><TR><TD>0.5 µL</TD><TD>2/20 A2</TD></TR><TR><TD>25 µL</TD></TR></TBODY></TABLE><H5>2/20 D3</H5><TABLE><TBODY><TR><TD>2.5 µL</TD><TD>10X Tango Buffer</TD></TR><TR><TD>10.5 µL</TD><TD>MilliQ water</TD></TR><TR><TD>1 µL</TD><TD>EcoRI</TD></TR><TR><TD>1 µL</TD><TD>PstI</TD></TR><TR><TD>10 µL</TD><TD>2/20 HF1</TD></TR><TR><TD>25 µL</TD></TR></TBODY></TABLE>
note: 2/20 D3 should be roughly 20 ng/µL, 10/11 D5 is ~117.3 ng/µL, 8/18 D1 is 20.5 ng/µL

Two ligations were mixed and incubated at room temperature for 40 minutes:
<H5>2/20 L1</H5><TABLE><TBODY><TR><TD>1 µL</TD><TD>10X T4 Ligase Buffer</TD></TR><TR><TD>4.6 µL</TD><TD>8/18 D1</TD></TR><TR><TD>3.4 µL</TD><TD>2/20 D3</TD></TR><TR><TD>1 µL</TD><TD>T4 DNA Ligase</TD></TR><TR><TD>10 µL</TD></TR></TBODY></TABLE><H5>2/20 L2</H5><TABLE><TBODY><TR><TD>1 µL</TD><TD>10X T4 Ligase Buffer</TD></TR><TR><TD>5 µL</TD><TD>8/18 D1</TD></TR><TR><TD>3 µL</TD><TD>2/20 D1</TD></TR><TR><TD>1 µL</TD><TD>T4 DNA Ligase</TD></TR><TR><TD>10 µL</TD></TR></TBODY></TABLE>
note: 2/20 L1 should be roughly 2 mol pSB1C3 : 1 mol Part 1 and 2.12 ng/µL

Two additional PCRs were performed with the same program below:
<TABLE><TBODY><TR><TD><B>Temperature (°C)</B></TD><TD><B>Time (s)</B></TD></TR><TR><TD>95</TD><TD>30</TD></TR><TR><TD>95c</TD><TD>23</TD></TR><TR><TD>51c</TD><TD>43</TD></TR><TR><TD>68c</TD><TD>250</TD></TR><TR><TD>68</TD><TD>300</TD></TR><TR><TD>4</TD><TD>hold</TD></TR></TBODY></TABLE>
cycle X32

 <H5>2/20 A4</H5><TABLE><TBODY><TR><TD>12.5 µL</TD><TD>2X Taq Master Mix</TD></TR><TR><TD>10.5 µL</TD><TD>MilliQ water</TD></TR><TR><TD>0.5 µL</TD><TD>10 µM VF2</TD></TR><TR><TD>0.5 µL</TD><TD>10 µM VR</TD></TR><TR><TD>1 µL</TD><TD>1/30X 2/20 L1</TD></TR><TR><TD>25 µL</TD></TR></TBODY></TABLE><H5>2/20 A5</H5><TABLE><TBODY><TR><TD>12.5 µL</TD><TD>2X Taq Master Mix</TD></TR><TR><TD>10.5 µL</TD><TD>MilliQ water</TD></TR><TR><TD>0.5 µL</TD><TD>10 µM VF2</TD></TR><TR><TD>0.5 µL</TD><TD>10 µM VR</TD></TR><TR><TD>1 µL</TD><TD>1/30X 2/20 L2</TD></TR><TR><TD>25 µL</TD></TR></TBODY></TABLE>
Another 1% agarose gel was run at 130 V for 45 minutes:
<H5>Gel 2/20 #2</H5><TABLE><TBODY><TR><TD>2</TD><TD>5 µL 2-log purple DNA ladder</TD></TR><TR><TD>3</TD><TD>5 µL 10/11 D5</TD></TR><TR><TD>4</TD><TD>5 µL 2/20 A3</TD></TR><TR><TD>5</TD><TD>10 µL 2/20 D1</TD></TR><TR><TD>6</TD><TD>10 µL 2/20 D2</TD></TR><TR><TD>7</TD><TD>5 µL 2/20 A4</TD></TR><TR><TD>8</TD><TD>5 µL 2/20 A5</TD></TR></TBODY></TABLE>
note: good pGEX4T-1 band in 3, nothing notable in 4-5 good hmp band in 6, smears in 7-8

Additionally, two media were prepared and autoclaved for culturing <I>Streptomyces roseus</I>:
<H5>Trace Salts Solution</H5><TABLE><TBODY><TR><TD>0.1 g/100 mL</TD><TD>FeSO<SUB>4</SUB> ⋅ 7H<SUB>2</SUB>O</TD></TR><TR><TD>0.1 g/100 mL</TD><TD>MnCl<SUB>2</SUB> ⋅ 4H<SUB>2</SUB>O</TD></TR><TR><TD>0.1 g/100 mL</TD><TD>ZnSO<SUB>4</SUB> ⋅ 7H<SUB>2</SUB>O</TD></TR></TBODY></TABLE><H5>Tryptone - Yeast Extract (ISP1)</H5><TABLE><TBODY><TR><TD>5 g/L</TD><TD>Tryptone</TD></TR><TR><TD>3 g/L</TD><TD>Yeast Extract</TD></TR></TBODY></TABLE>
pH 7.0 to 7.2
<H5>Synthetic Salts - Starch Medium (ISP4/NRRL9)</H5><TABLE><TBODY><TR><TD>10 g/L</TD><TD>Soluble Starch (corn starch)</TD></TR><TR><TD>1 g/L</TD><TD>K<SUB>2</SUB>HPO<SUB>4</SUB></TD></TR><TR><TD>1 g/L</TD><TD>MgSO<SUB>4</SUB> ⋅ 7H<SUB>2</SUB>O</TD></TR><TR><TD>1 g/L</TD><TD>NaCl</TD></TR><TR><TD>2 g/L</TD><TD>(NH<SUB>4</SUB>)<SUB>2</SUB>SO<SUB>4</SUB></TD></TR><TR><TD>2 g/L</TD><TD>CaCO<SUB>3</SUB></TD></TR><TR><TD>1 mL/L</TD><TD>Trace Salts Solution</TD></TR><TR><TD>15 g/L</TD><TD>Agar</TD></TR></TBODY></TABLE><H4><B>Stop: 3:30 am</B></H4> <H4><B>Results:</B></H4> <P class="centerp">#1</P><P class="centerp">#2</P><H4><B>Products:</B></H4> <TABLE><TBODY><TR><TD><B>Label</B></TD><TD><B>Source</B></TD><TD><B>Description</B></TD></TR><TR><TD>2/20 A1</TD><TD>Part 3 gblock</TD><TD>ocimene synthase amplified by Q5 with VF2&amp;ociR</TD></TR><TR><TD>2/20 A2</TD><TD>2/11 KG</TD><TD>sequenced hmp with EcoRI &amp; XhoI added</TD></TR><TR><TD>2/20 A3</TD><TD>Part 3 gblock</TD><TD>ocimene synthase amplified by Taq</TD></TR><TR><TD>2/20 A4</TD><TD>2/20 L1</TD><TD>verification of Part 1 in pSB1C3 with VF2&amp;VR</TD></TR><TR><TD>2/20 A5</TD><TD>2/20 L2</TD><TD>verification of OciS in pSB1C3 with VF2&amp;VR</TD></TR><TR><TD>2/20 HF1</TD><TD>Part 1-1, 1-2, 1-3, 1-4 gblocks</TD><TD>Part 1 insert assembled</TD></TR><TR><TD>2/20 D1</TD><TD>2/20 A3</TD><TD>OciS insert digested E&amp;P</TD></TR><TR><TD>2/20 D2</TD><TD>2/20 A2</TD><TD>hmp with sites added digested EcoRI &amp; XhoI</TD></TR><TR><TD>2/20 D3</TD><TD>2/20 HF1</TD><TD>Part 1 insert digested E&amp;P</TD></TR><TR><TD>2/20 L1</TD><TD>8/18 D1, 2/20 D3</TD><TD>Part 1 in pSB1C3</TD></TR><TR><TD>2/20 L2</TD><TD>8/18 D1, 2/20 D1</TD><TD>OciS in pSB1C3</TD></TR></TBODY></TABLE> <H4><B>Next:</B></H4>
Ligate 10/11 D5 and 2/20 D2 together, transform and harvest GST-hmp fusion protein for column purification, claving, and SDS-PAGE. Retry amplification of OciS using 2X Q5 Master Mix, and consider ordering primers for the insert of Part 1 to diagnose HiFis. Inoculate cultures of <I>Streptomyces roseus</I> strains and obtain chromosomal DNA.
</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>