Team:Missouri Rolla/Notebook
Home Project Human Practices Notebook Team Safety Attributions
Joe Microbe’s Self-Service Notebook
Many of our standard procedures can be found in our Lab Training Manual . Find more information about our oligos in the inventory .
18 October 2016
Kira Buckowing, Mikayla Tessmer, Bryan Tracy, and Matt Napoli
Start: 5:30pm
Restriction Digests and Gel
Purpose: To check the results from minipreps.
Protocol:
Restriction digests were run using the thermocycler to heat kill after cutting at E and P.
10/18 D1-D8
2.5 μL
10X Tango buffer (2X total)
20.5 μL
10/16 MP1-MP8
1 μL
EcorRI
1 μL
PstI
LTM Ed. 2 Gel
A 1% agarose gel was run at 130 V for ~45 minutes:
10/18 Gel
1
10 μL 2-log purple DNA ladder
2
10uL 10/18 D1
3
30 μL 10/18 D2
4
30 μL 10/18 D3
5
30 μL 10/18 D4
6
30 μL 10/18 D5
7
30 μL 10/18 D6
8
30 μL 10/18 D7
9
30 μL 10/18 D8
10
10 μL 2-log purple DNA ladder
Stop: 10:00pm
Results:
Sample
ng/μL
260/280
10/17 MP1
60.5
1.83
10/17 MP2
27.9
1.73
10/17 MP3
53.9
1.93
10/17 MP4
46.6
1.81
10/17 MP5
63.5
1.83
10/17 MP6
35.2
1.85
10/17 MP7
16.2
1.97
10/17 MP8
77.2
1.72
Products:
 
Label
Source
Description
10/18 D1
10/17 MP1
Nothing
10/18 D2
10/17 MP2
Nothing
10/18 D3
10/17 MP3
Nothing
10/18 D4
10/17 MP4
Nothing
10/18 D5
10/17 MP5
Nothing
10/18 D6
10/17 MP6
Nothing
10/18 D7
10/17 MP7
Nothing
10/18 D8
10/17 MP8
Nothing
Next:
Nothing
17 October 2016
Kira Buckowing, Paige Deirkes, Kent Gorday, Mikayla Tessmer, Bryan Tracy, and Matt Napoli
Start: 4:45pm
Minipreps
Purpose: To check the results from overnight broth culture growth.
Protocol:
Minipreps were performed using the kitless Merlin protocol, final elutions done with 100uL TE buffer.
Stop: 8:30pm
Products:
 
Label
Source
Description
10/17 MP1
10/16 BC5
Hopefully the assembled product of ocimene in the iGEM shipping backbone
10/17 MP2
10/16 BC6
Hopefully the assembled product of ocimene in the iGEM shipping backbone
10/17 MP3
10/16 BC7
Hopefully the assembled product of ocimene in the iGEM shipping backbone
10/17 MP4
10/16 BC8
Hopefully the assembled product of ocimene in the iGEM shipping backbone
10/17 MP5
10/16 BC9
Hopefully the assembled product of ocimene in the iGEM shipping backbone
10/17 MP6
10/16 BC10
Hopefully the assembled product of ocimene in the iGEM shipping backbone
10/17 MP7
10/16 BC12
Hopefully the assembled product of ocimene in the iGEM shipping backbone
10/17 MP8
10/16 BC13
Hopefully the assembled product of ocimene in the iGEM shipping backbone
Next: Check concentration of miniprep products, then digest and run a gel to check the DNA.
16 October 2016
Kira Buckowing
Start: 12:00pm
Chemical Transformations
Purpose: To check on the most recent work and get the wanted ocimene synthase in a chlor resistance backbone.
Protocol:
HiFi assemblies were set up as per the NEB protocol and incubated in ther thermocylcler for 15 minutes at 50 C.
10/16 HiFi 1
9 uL
10/14 D1
1 uL
10/14 D4
10 uL
HiFi Master Mix
10/16 HiFi 2
9 uL
10/14 D2
1 uL
10/14 D4
10 uL
HiFi Master Mix
14 broth Cultures were set up from colony growth on the 10/15 chemical transformation plates.  Chemical transformations were performed using the chemically competent cells from NEB and adding either 2 uL or 6 uL of the appropriate HiFi assembly product, noted in the products section below.
Stop: 3:30pm
Products:
 
Label
Source
Description
10/16 CT1 100 uL
10/16 HiFi 1
Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone, 2 uL added
10/16 CT1 250 uL
10/16 HiFi 1
Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone, 2 uL added
10/16 CT2 100 uL
10/16 HiFi 2
Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone, 2 uL added
10/16 CT2 250 uL
10/16 HiFi 2
Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone, 2 uL added
10/16 CT3 100 uL
10/16 HiFi 1
Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone, 6 uL added
10/16 CT3 250 uL
10/16 HiFi 1
Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone, 6 uL added
10/16 CT4 100 uL
10/16 HiFi 2
Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone, 6 uL added
10/16 CT4 250 uL
10/16 HiFi 2
Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone, 6 uL added
10/16 HiFi 1
10/14 D1 and D4
Hopefully the assembled product of ocimene in the iGEM shipping backbone
10/16 HiFi 2
10/14 D2 and D4
Hopefully the assembled product of ocimene in the iGEM shipping backbone
10/16 BC1
10/15 CT2 100 uL
Hopefully the assembled product of ocimene in the iGEM shipping backbone
10/16 BC2
10/15 CT2 250 uL
Hopefully the assembled product of ocimene in the iGEM shipping backbone
10/16 BC3
10/15 CT2 250 uL
Hopefully the assembled product of ocimene in the iGEM shipping backbone
10/16 BC4
10/15 CT2 250 uL
Hopefully the assembled product of ocimene in the iGEM shipping backbone
10/16 BC5
10/15 CT2 250 uL
Hopefully the assembled product of ocimene in the iGEM shipping backbone
10/16 BC6
10/15 CT4 250 uL
Hopefully the assembled product of ocimene in the iGEM shipping backbone
10/16 BC7
10/15 CT4 250 uL
Hopefully the assembled product of ocimene in the iGEM shipping backbone
10/16 BC8
10/15 CT4 250 uL
Hopefully the assembled product of ocimene in the iGEM shipping backbone
10/16 BC9
10/15 CT4 250 uL
Hopefully the assembled product of ocimene in the iGEM shipping backbone
10/16 BC10
10/15 CT4 250 uL
Hopefully the assembled product of ocimene in the iGEM shipping backbone
10/16 BC11
10/15 CT4 250 uL
Hopefully the assembled product of ocimene in the iGEM shipping backbone
10/16 BC12
10/15 CT4 250 uL
Hopefully the assembled product of ocimene in the iGEM shipping backbone
10/16 BC13
10/15 CT4 250 uL
Hopefully the assembled product of ocimene in the iGEM shipping backbone
10/16 BC14
10/15 CT4 250 uL
Hopefully the assembled product of ocimene in the iGEM shipping backbone
Next: Minipreps of any growth in the broth cultures and checking the plates to see if anything grows.
15 October 2016
Kira Buckowing
Start: 8:00pm
Chemical Transformations
Purpose: To check on the most recent work and get the wanted ocimene synthase in a chlor resistance backbone.
Protocol:
Chemical transformations were performed using the chemically competent cells from NEB and adding 5 uL of the appropriate ligation product.
Stop: 10:20pm
Products:
 
Label
Source
Description
10/15 CT1 100 uL
10/14 L1
Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone
10/15 CT1 250 uL
10/14 L1
Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone
10/15 CT2 100 uL
10/14 L2
Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone
10/15 CT2 250 uL
10/14 L2
Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone
10/15 CT3 100 uL
10/14 L3
Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone
10/15 CT3 250 uL
10/14 L3
Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone
10/15 CT4 100 uL
10/14 L4
Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone
10/15 CT4 250 uL
10/14 L4
Chemically transformed plasmid hopefully containing ocimene synthase in the pSB1C3 iGEM shipping backbone
Next: HiFi assembly to cover all bases and give other options for the wanted assembled product and checking the plates to see if anything grows.
14 October 2016
Kira Buckowing
Start: 5:00pm
Gel to verify 10/13 results and Ligations of ocimene and the pSB13C shipping backbone.
Purpose: To check on the most recent work and get the wanted ocimene synthase in a chlor resistance backbone.
Protocol:
LTM Ed. 2 Gel A 1% agarose gel was run at 130 V for ~45 minutes:
10/14 Gel
1
10 μL 2-log purple DNA ladder
2
10/13 1
3
10/13 2
4
10/13 A2
5
10/13 A8
6
10/13 A8
5
10/14 A4
7
10/13 A5
8
10/13 A6
9
10/13 A7
10
10 μL 2-log purple DNA ladder
Digests from 10/11 were NanoDropped to determine concentrations.   Restriction digests were prepared, then incubated and heat-killed in the thermocycler:
10/14 D1
2.5 μL
MilliQ water
2.5 μL
10X Tango buffer (2X total)
18 μL
10/11 G4
1 μL
EcoRI
1 μL
PstI
10/14 D2
2.5 μL
MilliQ water
2.5 μL
10X Tango buffer (2X total)
18 μL
10/11 G4
1 μL
EcoRI
1 μL
PstI
10/14 D3
2.5 μL
10X Tango buffer (2X total)
20.5 μL
10/11 G1
1 μL
EcoRI
1 μL
PstI
10/14 D4
12.5 μL
MilliQ water
2.5 μL
10X Tango buffer (2X total)
8 μL
pSB13C iGEM Kit DNA
1 μL
EcoRI
1 μL
PstI
Ligations were set up for combinations of each digested backbone and the two versions of the ocimene synthase gene.
10/14 L1
9 uL
10/14 D1
2 uL
10/14 D3
1 uL
Ligase Buffer
1 uL
Ligase
10/14 L2
9 uL
10/14 D2
2 uL
10/14 D3
1 uL
Ligase Buffer
1 uL
Ligase
10/14 L3
9 uL
10/14 D1
2 uL
10/14 D4
1 uL
Ligase Buffer
1 uL
Ligase
10/14 L4
9 uL
10/14 D2
2 uL
10/14 D4
1 uL
Ligase Buffer
1 uL
Ligase
Stop: 9:20pm
Results:
Sample
ng/μL
260/280
10/11 GE1
16
1.79
10/11 GE3
14.05
1.75
10/11 GE4
12.5
1.74
Products:
 
Label
Source
Description
10/14 D1
10/11 G4
Ocimene gel extraction digested with E and P
10/14 D2
Ocimene gBlock from IDT
ocimene synthase digested with E and P
10/14 D3
10/11 G1
pSB13C backbone digested with E and P
10/11 G4
pBSIC3 from iGEM Kit DNA
pSB1C3 shipping backbone digested with E and P
10/11 L1
10/11 D1 and D3
Hopefully ocimene synthase in the iGEM shipping backbone.
10/11 L2
10/11 D2 and D3
Hopefully ocimene synthase in the iGEM shipping backbone.
10/11 L3
10/11 D1 and D4
Hopefully ocimene synthase in the iGEM shipping backbone.
10/11 L4
10/11 D2 and D4
Hopefully ocimene synthase in the iGEM shipping backbone.
Next: Chemical transformations of the ligations to see if anything grows.
13 October 2016
Kent Gorday, Mikayla Tessmer, Bryan Tracy, and Matt Napoli
Start: 4:50 pm
Final amplification of LeupA and LeupB
Purpose: To clone LeupA and LeupB into pSB1C3
Protocol:
Two PCRs were set up and left to react in thermocycler with the same program:
10/13 A1 (9/27 A2, LeupA)
7 μL
MilliQ water
10 μL
2x Q5 Master Mix
1 μL
30 ng 2/25 3
1 μL
10 μM oKG42
1 μL
10 μM oKG43
20 μL
10/13 A2  (LeupA - control)
8 μL
MilliQ water
10 μL
2x Q5 Master Mix
1 μL
10 μM oKG42
1 μL
10 μM oKG43
20 μL
Temperature (°C)
Time (s)
98
30
98c
8
57c
22
72c
163
72
120
4
hold
cycle X30
Two more PCRs were set up and left to react in thermocycler with the same program:
10/13 A3 (9/27 A4, LeupB)
7 μL
MilliQ water
10 μL
2x Q5 Master Mix
1 μL
30 ng 2/25 2
1 μL
10 μM oKG44
1 μL
10 μM oKG45
20 μL
10/13 A4 (LeupB - control)
8 μL
MilliQ water
10 μL
2x Q5 Master Mix
1 μL
10 μM oKG44
1 μL
10 μM oKG45
20 μL
Temperature (°C)
Time (s)
98
30
98c
8
57c
22
72c
45
72
120
4
hold
cycle X30
Two PCR clean-ups were performed using Macherey-Nagel NucleoSpin kit, eluting with 20 μL buffer.  Two more PCRs were set up and left to react in thermocycler with the same program:
10/13 A5 (LeupA + RS)
10 μL
2X Q5 Master Mix
8 μL
10/13 1
1 μL
10 μM oKG4
1 μL
10 μM oKG5
20 μL
10/13 A6 (- control)
8 μL
MilliQ water
10 μL
2X Q5 Master Mix
1 μL
10 μM oKG4
1 μL
10 μM oKG5
20 μL
Temperature (°C)
Time (s)
98
30
98c
8
62c
22
72c
160
72
120
4
hold
cycle X30
Two more PCRs were set up and left to react in thermocycler with the same program:
10/13 A7 (LeupB + RS)
10 μL
2X Q5 Master Mix
8 μL
10/13 2
1 μL
10 μM oKG8
1 μL
10 μM oKG9
20 μL
10/13 A8 (- control)
8 μL
MilliQ water
10 μL
2X Q5 Master Mix
1 μL
10 μM oKG8
1 μL
10 μM oKG9
20 μL
Temperature (°C)
Time (s)
98
30
98c
8
62c
22
72c
42
72
120
4
hold
cycle X30
Stop: 8:00 pm
Products:
Lable
Source
Description
10/13 A1
2/25 3
LeupA initially amplified from S. roseus B-3062
10/13 A2
LeupA initial amplification negative control
10/13 A3
2/25 2
LeupB initially amplified from S. roseus B-1320
10/13 A4
LeupB initial amplification negative control
10/13 A5
10/13 1
LeupA amplified with RFC[10] prefix and suffix
10/13 A6
LeupA RFC[10] prefix and suffix primer negative control
10/13 A7
10/13 2
LeupB amplified with RFC[10] prefix and suffix
10/13 A8
LeupB RFC[10] prefix and suffix primer negative control
10/13 1
10/13 A1
LeupA initially amplified, PCR Clean-up
10/13 2
10/13 A3
LeupB initially amplified, PCR Clean-up
Next:
Run a gel with 5 μL each: 10/13 1, 10/13 2, 10/13 A2, 10/13 A4, 10/13 A5, 10/13 A6, 10/13 A7, and 10/13 A8
11 October 2016
Kira Buckowing, Kent Gorday, Mikayla Tessmer, Bryan Tracy, and Matt Napoli
Start: 5:00pm
Gel to verify 10/10 results and ocimene gBlock
Purpose: To check on the most recent pSB1C3 backbone digests and the ocimene gBlock from IDT.
Protocol:
LTM Ed. 2 Gel and Gel Extraction A 1% agarose gel was run at 130 V for ~45 minutes:
10/11 Gel
1
10/10 D2
2
10/10 D1
3
10 μL 2-log purple DNA ladder
4
10/10 D3
5
Ocimene gBlock
The bands that showed up in the gel were cut out and gel extractions were performed using the Macherey-Nagel NucleoSpin kit, eluting with 30 μL buffer.
Stop: 8:20pm
Results:
Products:
 
Label
Source
Description
10/11 G1
10/10 D1
pSB13C backbone
10/11 G2
10/9 D2
pSB13C backbone
10/11 G3
10/9 D3
pSB13C backbone
10/11 G4
Ocimene gBlock
ocimene synthase
Notes:
G2 did not dissolve fully in the first step and is probably not a viable product.
Next: NanoDrop for concentrations, then set up appropriate ligations with the ocimene from the gBlock itself and this gel extraction product.
10 October 2016
Kira Buckowing, Bryan Tracy, and Matt Napoli
Start: 6:00pm
Second digest on the 10/9 D1
Purpose: To get the second cut site at S without interference from the first cut site at E done previously.
Protocol:
A restriction digest was prepared, then incubated and heat-killed in the thermocycler:
10/10 D1
20 μL
MilliQ water
2.5 μL
10X Tango buffer (2X total)
1.42 μL
10/9 D1
1 μL
SpeI
Stop: 8:00pm
Products:
 
Label
Source
Description
10/10 D1
10/9 D1
pSB13C backbone digested with E and S
Next: Gel to verify products and extractions of the correct digested products.
9 October 2016
Kent Gorday, Mikayla Tessmer, and Bryan Tracy
Start: 11:00 am
 
Protocol:
A 0.7% agarose gel was run at 130 V:
10/9 Gel #1
2
5 μL 2-log purple DNA ladder
3
5 μL 10/8 A1 + 5 μL 10/8 A2
4
5 μL 10/8 A5
5
5 μL 10/8 A6
6
5 μL 10/8 A7
A 0.7% agarose gel was run at 78 V:
10/9 Gel #2
2
5 μL 2-log purple DNA ladder
4
max volume 10/8 A3
6
max volume 10/8 A4
Three PCRs were prepared and reacted with the same thermocycler program:
10/8 A1
1 μL
MilliQ water
5 μL
5X Q5 High GC Enhancer
12.5 μL
2X Q5 Master Mix
4 μL
10/8 GE3
1.25 μL
10 μM oKG4
1.25 μL
10 μM oKG5
25 μL
10/8 A2
1 μL
MilliQ water
5 μL
5X Q5 High GC Enhancer
12.5 μL
2X Q5 Master Mix
4 μL
10/8 GE7
1.25 μL
10 μM oKG4
1.25 μL
10 μM oKG5
25 μL
10/8 A4
1 μL
MilliQ water
5 μL
5X Q5 High GC Enhancer
12.5 μL
2X Q5 Master Mix
4 μL
10/8 GE4
1.25 μL
10 μM oKG4
1.25 μL
10 μM oKG5
25 μL
 
Temperature (°C)
Time (s)
98
30
98c
8
66c
22
72c
160
72
120
4
hold
cycle X30
Three additional PCRs were prepared and reacted with the same thermocycler program:
10/8 A3
1 μL
MilliQ water
5 μL
5X Q5 High GC Enhancer
12.5 μL
2X Q5 Master Mix
4 μL
10/8 GE4
1.25 μL
10 μM oKG8
1.25 μL
10 μM oKG9
25 μL
10/8 A5
1 μL
MilliQ water
5 μL
5X Q5 High GC Enhancer
12.5 μL
2X Q5 Master Mix
4 μL
10/8 GE3
1.25 μL
10 μM oKG8
1.25 μL
10 μM oKG9
25 μL
10/8 A6
1 μL
MilliQ water
5 μL
5X Q5 High GC Enhancer
12.5 μL
2X Q5 Master Mix
4 μL
10/8 GE7
1.25 μL
10 μM oKG8
1.25 μL
10 μM oKG9
25 μL
 
Temperature (°C)
Time (s)
98
30
98c
8
66c
22
72c
42
72
120
4
hold
cycle X30
Three restriction digests were prepared and reacted, then heat killed, in the thermocycler:
10/8 D1
20 μL
MilliQ water
2.5 μL
10X Tango buffer
1.42 μL
3/24 MP1
1 μL
SpeI
24.92 μL
10/8 D2-3
16.5 μL
MilliQ water
5 μL
10X Tango buffer
1.42 μL
3/24 MP1
1 μL
EcoRI
1 μL
PstI
24.92 μL
 
Notes:
No bands were observed for 10/8 A3-4.
Stop: 6:00 pm
 
Results:
Products:
 
Label
Source
Description
10/9 A1
10/8 GE3
LeupA with RFC[10] prefix and suffix added
10/9 A2
10/8 GE7
LeupA with RFC[10] prefix and suffix added
10/9 A3
10/8 GE4
LeupB with RFC[10] prefix and suffix added
10/9 A4
10/8 GE4
LeupA with RFC[10] prefix and suffix added
10/9 A5
10/8 GE3
LeupB with RFC[10] prefix and suffix added
10/9 A6
10/8 GE7
LeupB with RFC[10] prefix and suffix added
10/9 D1
3/24 MP1
RFP in pSB1C3 digested SpeI
10/9 D2-3
3/24 MP1
RFP in pSB1C3 digested EcoRI & PstI
 
8 October 2016
Kent Gorday, Mikayla Tessmer, Bryan Tracy, Matt Napoli, and Paige Dierkes
Start: 2:00 pm
 
Protocol:
Three 0.7% agarose gels were run at 78 V:
10/8 Gel #1 (loaded incorrectly)
1
5 μL 2-log purple ladder
3
max volume 10/7 A1
5
max volume 10/7 A2
10/8 Gel #2
1
5 μL 2-log purple DNA ladder
3
max volume 10/7 A1
5
max volume 9/27 A4
10/8 Gel #3
2
5 μL 2-log purple DNA ladder
4
10 μL 3/3 A2
5 gel extractions were performed using the NucleoSpin Gel and PCR Clean-up kit from gels 1, 2, and 3. LeupA appeared as the top band in its lane, while LeupB appeared as the brightest in the middle of its lane. Products were Nanodropped to determine concentration. A gel extraction was performed from gel 3 using an Amicon Ultrafree-DA filter.  A 1% agarose gel was run at 130 V:
10/8 Gel #4
2
5 μL 2-log purple DNA ladder
4
max volume 10/8 GE5
6
max volume 10/8 GE6
A 0.7% agarose gel was run at 130 V:
10/8 Gel #5
2
5 μL 2-log purple DNA ladder
4
25 μL 10/8 GE5
6
10 μL 10/8 GE6
A gel extraction  2 PCRs were prepared and run in a thermocycler with the same program:
10/8 A1-2
6 μL
MilliQ water
12.5 μL
2X Q5 Master Mix
4 μL
10 ng/μL ociS gblock
1.25 μL
10 μM VF2
1.25 μL
10 μM ociR
25 μL
 
Temperature (°C)
Time (s)
98
30
98c
8
56c
22
72c
45
72
120
4
hold
cycle X11
2 PCRs were prepared and run in a thermocycler with the same program:
10/8 A3
14 μL
MilliQ water
20 μL
2X Q5 Master Mix
2 μL
30 ng/μL 2/25 1
2 μL
10 μM oKG44
2 μL
10 μM oKG45
40 μL
10/8 A4
14 μL
MilliQ water
20 μL
2X Q5 Master Mix
2 μL
30 ng/μL 2/25 3
2 μL
10 μM oKG44
2 μL
10 μM oKG45
40 μL
 
Temperature (°C)
Time (s)
98
30
98c
8
57c
22
72c
45
72
120
4
hold
cycle X30
2 PCRs were prepared and run in a thermocycler with the same program:
10/8 A5 (LeupA + RS)
6 μL
MilliQ water
12.5 μL
2X Q5 Master Mix
4 μL
10/8 GE3
1.25 μL
10 μM oKG4
1.25 μL
10 μM oKG5
25 μL
10/8 A6 (LeupA + RS)
6 μL
MilliQ water
12.5 μL
2X Q5 Master Mix
4 μL
10/8 GE7
1.25 μL
10 μM oKG4
1.25 μL
10 μM oKG5
25 μL
 
Temperature (°C)
Time (s)
98
30
98c
8
56c
22
72c
160
72
120
4
hold
cycle X26
Another PCR was prepared and run in a thermocycler:
10/8 A7 (LeupB + RS)
6 μL
MilliQ water
12.5 μL
2X Q5 Master Mix
4 μL
10/8 GE4
1.25 μL
10 μM oKG8
1.25 μL
10 μM oKG9
25 μL
 
Temperature (°C)
Time (s)
98
30
98c
8
56c
22
72c
42
72
120
4
hold
   
Results:
 
Sample
ng/μL
260/280
260/230
10/8 GE3
3.58
1.79
0.02
10/8 GE4
6.67
1.77
0.42
10/8 GE5
14.32
1.60
0.32
10/8 GE6
17.97
1.25
0.17
Products:
Reset Table
Label
Source
Description
10/8 GE3
10/7 A1 top band
LeupA from S. roseus B-1320
10/8 GE4
9/27 A4 middle bright band
LeupB from S. roseus B-1320
10/8 GE5
3/3 A2
RFP positive control for Macherey-Nagel kit
10/8 GE6
3/3 A2
RFP positive control for Amicon filters
10/8 GE7
10/7 A3 top band
LeupA from S. roseus ISP-5076
10/8 A1
ociS gblock
ociS gblock amplified
10/8 A2
ociS gblock
ociS gblock amplified
10/8 A3
2/25 1
LeupB from S. roseus ISP-5076
10/8 A4
2/25 3
LeupB from S. roseus B-3062
10/8 A5
10/8 GE3
LeupA with RFC[10] prefix and suffix added
10/8 A6
10/8 GE7
LeupA with RFC[10] prefix and suffix added
10/8 A7
10/8 GE4
LeupB with RFC[10] prefix and suffix added
 
7 October 2016
Kent Gorday, Mikayla Tessmer, Matt Napoli, Bryan Tracy, and Paige Dierkes
Start: 2:30pm
 
Protocol:
1.2% agarose gel was run:
2
5 μL 2-log purple DNA ladder
4
5 μL 10/6 A1
6
5 μL 10/6 A2
8
5 μL 2-log purple DNA ladder
3 PCRs were set up:
10/7 A1-3
14 μL
MilliQ water
20 μL
2X Q5 Master Mix
2 μL
30 ng/μL 2/25 1 (A1), 2 (A2), or 3 (A3)
2 μL
10 μM oKG42
2 μL
10 μM oKG43
40 μL
 
Temperature (°C)
Time (s)
98
30
98c
8
57c
22
72c
163
72
120
4
hold
cycle X30
 
Stop: 5:00 pm
 
Results:
Products:
 
Label
Source
Description
10/7 A1
2/25 1
LeupA from S. roseus B-1320
10/7 A2
2/25 2
LeupA from S. roseus B-3062
10/7 A3
2/25 3
LeupA from S. roseus ISP-5076
 
6 October 2016
Kent Gorday, Bryan Tracy, and Mikayla Tessmer
Start: 9:00 am
 
Protocol:
A 0.8% agarose gel was run at 90 V:
1
5 μL 2-log purple DNA ladder
2
5 μL 10/5 A1
3
5 μL 10/5 A2
4
5 μL 10/5 A3
5
5 μL 10/5 A4
6
5 μL 2-log purple DNA ladder
7
5 μL 10/5 A9
8
5 μL 10/5 A10
9
5 μL 10/5 A11
10
5 μL 10/5 A12
Two PCRs were reacted in the thermocycler for the amplification of ociS:
10/6 A1-2
9 μL
MilliQ water
12.5 μL
2X Q5 Master Mix
1 μL
10 ng/μL ociS gblock
1.25 μL
10 μM VF2
1.25 μL
10 μM ociR
25 μL
 
Temperature (°C)
TIme (s)
98
30
98c
8
60c
22
72c
44
72
120
4
hold
cycle X11
 
Notes:
10/5 A1-16 were done with mixed LeupA and LeupB primer pairs rather than the proper oKG42/43 (LeupA) and oKG44/45 (LeupB) pairs.
Stop: 11:00 am
 
Results:
Products:
 
Label
Source
Description
10/6 A1
ociS gblock
ociS gblock amplified
10/6 A2
ociS gblock
ociS gblock amplified
 
5 October 2016
Kent Gorday, Kurt Studer, Bryan Tracy, and Sam Greaney
Start: 5:30 pm
 
Gradient PCRs of Initial LeupA and LeupB
Purpose: To determine the best PCR conditions for amplifying LeupA and LeupB from the Streptomyces roseus genome
Protocol:
Sixteen PCRs were prepared and reacted with two different thermocycler programs, each containing a gradient for the annealing step:
10/5 A1-A8
8.8 μL
MilliQ water
12.5 μL
2X Q5 Master Mix
1.25 μL
37.5 ng 2/25 2
1.25 μL
10 μM oKG42
1.25 μL
10 μM oKG44
25 μL
 
Temperature (°C)
Time (s)
98
30
98c
8
gradient (A1/A5 63.0, A2/A6 60.6, A3/A7 53.3, A4/A8 51.0)c
22
72c
163
72
120
4
hold
cycle X30
10/5 A9-A16
8.8 μL
MilliQ water
12.5 μL
2X Q5 Master Mix
1.25 μL
37.5 ng 2/25 2
1.25 μL
10 μM oKG43
1.25 μL
10 μM oKG45
25 μL
 
Temperature (°C)
Time (s)
98
30
98c
8
gradient (A9/A13 63.0, A10/A14 60.6, A11/A15 53.3, A12/A16 51.0)c
22
72c
45
72
120
4
hold
cycle X31
 
Notes:
10/5 A1-A8 were restarted in the thermocycler during the first cycle due to a programming error. 10/5 A9-A16 were accidentally cycled 31 instead of 30 times. The original intention was to run each set of conditions with and without GC Enhancer, but none was added.
Stop: 7:00 pm
 
Products:
 
Label
Source
Description
10/5 A1-A8
2/25 2
Gradient PCRs of LeupA from S. roseus B-1320 -GCE
10/5 A9-A16
2/25 2
Gradient PCRs of LeupB from S. roseus B-1320 -GCE
 
Next:
Run a gel of 10/5 PCRs, and gel purify LeupA and LeupB
27 September 2016
Kent Gorday
Start: 6:00 pm
 
Gel Electrophoresis of New LeupA and LeupB PCRs
Purpose: To verify success of 9/27 PCRs
Protocol:
A 1% agarose gel was run with each 9/27 PCR:
3
7 μL 2-log purple DNA ladder
4
5 μL 9/27 A1
5
5 μL 9/27 A2
6
5 μL 9/27 A3
7
5 μL 9/27 A4
 
Stop: 7:00 pm
 
Results:
Next:
PCR with 16S primers from 2/25 1 using Q5 to check Q5 Master Mix and the remaining genomic DNA prep, with a negative control for both Q5 and Taq.
27 September 2016
Kent Gorday
Start: 12:15 pm
 
New Initial PCRs of LeupA and LeupB
Purpose: To amplify LeupA and LeupB from the Streptomyces roseus genome
Protocol:
 
9/27 A1
3 μL
MilliQ water
4 μL
5X Q5 High GC Enhancer
10 μL
2X Q5 Master Mix
1 μL
30 ng 2/25 3
1 μL
10 μM oKG42
1 μL
10 μM oKG43
20 μL
9/27 A2
7 μL
MilliQ water
10 μL
2X Q5 Master Mix
1 μL
30 ng 2/25 3
1 μL
10 μM oKG42
1 μL
10 μM oKG43
20 μL
 
Temperature (°C)
Time (s)
98
30
98c
8
57c
22
72c
163
72
120
4
hold
cycle X30
9/27 A3
3 μL
MilliQ water
4 μL
5X Q5 High GC Enhancer
10 μL
2X Q5 Master Mix
1 μL
30 ng 2/25 2
1 μL
10 μM oKG44
1 μL
10 μM oKG45
20 μL
9/27 A4
7 μL
MilliQ water
10 μL
2X Q5 Master Mix
1 μL
30 ng 2/25 2
1 μL
10 μM oKG44
1 μL
10 μM oKG45
20 μL
 
Temperature (°C)
Time (s)
98
30
98c
8
57c
22
72c
45
72
120
4
hold
cycle X30
 
Stop: 12:50 pm
 
Products:
 
Label
Source
Description
9/27 A1
2/25 1
LeupA and flanking region amplified from S. roseus ISP-5076 +GCE
9/27 A2
2/25 1
LeupA and flanking region amplified from S. roseus ISP-5076 -GCE
9/27 A3
2/25 2
LeupB and flanking region amplified from S. roseus B-1320 +GCE
9/27 A4
2/25 2
LeupB and flanking region amplified from S. roseus B-1320 -GCE
 
Next:
Run a gel of 9/27 PCRs, also PCR with 16S primers from 2/25 1 using Q5 to check Q5 Master Mix and the remaining genomic DNA prep, with a negative control for both Q5 and Taq.
24 September 2016
Kent Gorday
Start: 3:00 pm
 
Gel of Initial Leupeptin and 16S PCRs
Purpose: To verify success of 9/23 PCRs
Protocol:
A 0.8% agarose gel was run with 9/23 PCR products:
9/24 Gel
2
5 μL 2-log purple DNA ladder
3
5 μL 9/23 A1
4
5 μL 9/23 A2
5
5 μL 9/23 A3
6
5 μL 9/23 A4
7
5 μL 9/23 A5
8
5 μL 9/23 A6
 
Stop: 4:30 pm
 
Results:
Next:
PCR with 16S primers from 2/25 1 using Q5 to check Q5 Master Mix and the remaining genomic DNA prep, with a negative control for both Q5 and Taq. Try LeupA/B initial PCRs again with a lower annealing temperature (59 °C), with and without GC Enhancer.
23 September 2016
Kent Gorday, Ryan Baumann, Lucas Harper, and Kurt Studer
Start: 2:40 pm
 
Leupeptin and 16S PCR Attempts
Purpose: To amplify LeupA and LeupB from the Streptomyces roseus genome, and verify the presence of genomic DNA in preps from three strains while testing 16S primers
Protocol:
Two PCRs were incubated in a thermocycler after new primers were diluted to 10 μM and templates to 30 ng/μL:
9/23 A1
3 μL
MilliQ water
4 μL
5X Q5 High GC Enhancer
10 μL
2X Q5 Master Mix
1 μL
30 ng 2/25 1
1 μL
10 μM oKG42
1 μL
10 μM oKG43
20 μL
9/23 A2
7 μL
MilliQ water
10 μL
2X Q5 Master Mix
1 μL
30 ng 2/25 1
1 μL
10 μM oKG42
1 μL
10 μM oKG43
20 μL
 
Temperature (°C)
Time (s)
98
30
98c
8
67c
22
72c
185
72
120
4
hold
cycle X30
Two more PCRs were also reacted in a thermocycler:
9/23 A3
3 μL
MilliQ water
4 μL
5X Q5 High GC Enhancer
10 μL
2X Q5 Master Mix
1 μL
30 ng 2/25 2
1 μL
10 μM oKG44
1 μL
10 μM oKG45
20 μL
9/23 A4
7 μL
MilliQ water
10 μL
2X Q5 Master Mix
1 μL
30 ng 2/25 2
1 μL
10 μM oKG44
1 μL
10 μM oKG45
20 μL
 
Temperature (°C)
Time (s)
98
30
98c
8
67c
22
72c
45
72
120
4
hold
cycle X30
Two more PCRs were also reacted in a thermocycler:
9/23 A4
8.2 μL
MilliQ water
10 μL
2X Taq Master Mix
1 μL
30 ng 2/25 2
1 μL
10 μM 27F
1 μL
10 μM 1492R
20 μL
9/23 A5
8.2 μL
MilliQ water
10 μL
2X Taq Master Mix
1 μL
30 ng 2/25 3
1 μL
10 μM 27F
1 μL
10 μM 1492R
20 μL
 
Temperature (°C)
Time (s)
95
30
95c
22
49c
40
68c
95
68
300
4
hold
cycle X30
Two broth cultures in ISP medium 1 were inoculated from frozen stock using a sterile pipette tip.
Notes:
No negative control was performed for 16S amplification, so genomic DNA preps cannot conclusively be verified. No ISP-4 or ISP-1 media is left.  Some basic information about oKG42-45 is included below:
Oligo
Sequence
Correct Location
Mispriming E-Values
oKG42
cttcacccgaatcgatgctg
_1163: 93441 +
0.84, 0.84 +- (both far)
oKG43
gtggtgtgttccgtttcctg
_1163: 101101 -
oKG44
caggaaacggaacacaccac
_1163: 101082 +
oKG45
gggaaaggtgaccaggaagt
_1163: 102794 -
0.069, 0.24, 0.24, etc. ++-- (all far)
 
Stop: 5:00 pm
 
Products:
 
Label
Source
Description
9/25 A1
2/25 1
LeupA and flanking region amplified from S. roseus ISP-5076 +GCE
9/25 A2
2/25 1
LeupA and flanking region amplified from S. roseus ISP-5076 -GCE
9/25 A3
2/25 2
LeupB and flanking region amplified from S. roseus B-1320 +GCE
9/25 A4
2/25 2
LeupB and flanking region amplified from S. roseus B-1320 -GCE
9/25 A5
2/25 2
Portion of 16S rRNA DNA amplified from S. roseus B-1320
9/25 A6
2/25 3
Portion of 16S rRNA DNA amplified from S. roseus B-3062
9/25 BC1
9/6 S1
S. roseus ISP-5076 from frozen stock
9/25 BC2
9/6 S2
S. roseus B-1320 from frozen stock
 
Next:
Run a gel of PCR products to decide if LeupA and LeupB PCRs should be performed from all strains
6 September 2016
Kent Gorday
2 September 2016
Kent Gorday
 
Culturing of and colony PCR from Streptomyces roseus strains
Purpose: To revive S. roseus strains and amplify LeupB
Protocol:
Twelve colony PCRs were reacted in the thermocycler with the same program:
9/2 A1-A6
70 μL
10 μL
2X Q5 Master Mix
21 μL
3 μL
MilliQ water
28 μL
4 μL
5X Q5 High GC Enhancer
7 μL
1 μL
10 μM LeupB_F
7 μL
1 μL
10 μM LeupB_R
swirl or add 1 μL
colony or liquid culture template
133 μL
20 μL
9/2 A7-A12
70 μL
10 μL
2X Taq Master Mix
57.4 μL
8.2 μL
MilliQ water
2.8 μL
0.4 μL
10 μM LeupB_F
2.8 μL
0.4 μL
10 μM LeupB_R
swirl or add 1 μL
colony or liquid culture template
133 μL
20 μL
 
Temperature (°C)
Time (s)
95
300
95c
17
61c
25
70c
120
70
300
4
hold
cycle X30
Three streaks were performed onto ISP medium 4 using sterile pipette tips, then three ISP medium 1 tube cultures were similarly inoculated.
Notes:
Plates used for streaking were left warming in incubator and had severely dried out.  
Products:
 
Label
Source
Description
9/2 BC1
2/21 I1
S. roseus ISP-5076 in ISP m. 1
9/2 BC2
2/21 I2
S. roseus B-1320 in ISP m. 1
9/2 BC3
2/21 I3
S. roseus B-3062 in ISP m. 1
9/2 I1
2/21 I1
S. roseus ISP-5076 on ISP m. 4
9/2 I2
2/21 I2
S. roseus B-1320 on ISP m. 4
9/2 I3
2/21 I3
S. roseus B-3062 on ISP m. 4
9/2 A1
2/21 I1
LeupB amplified by colony PCR from S. roseus ISP-5076
9/2 A2
2/21 I2
LeupB amplified by colony PCR from S. roseus B-1320
9/2 A3
2/21 I3
LeupB amplified by colony PCR from S. roseus B-3062
9/2 A4
8/24 BC1
LeupB amplified by colony PCR from S. roseus ISP-5076
9/2 A5
8/24 BC3
LeupB amplified by colony PCR from S. roseus B-3062
9/2 A6
LeupA_F/R negative control PCR
9/2 A7
2/21 I1
LeupB amplified by colony PCR from S. roseus ISP-5076(Taq)
9/2 A8
2/21 I2
LeupB amplified by colony PCR from S. roseus B-1320(Taq)
9/2 A9
2/21 I3
LeupB amplified by colony PCR from S. roseus B-3062(Taq)
9/2 A10
8/24 BC1
LeupB amplified by colony PCR from S. roseus ISP-5076(Taq)
9/2 A11
8/24 BC3
LeupB amplified by colony PCR from S. roseus B-3062(Taq)
9/2 A12
LeupA_F/R negative control PCR(Taq)
 
Next:
 
24 August 2016
Kent Gorday
Start: 2:30 pm
 
Reviving Streptomyces roseus
Purpose: To amplify LeupA and LeupB directly from S. roseus cultures with lysis in thermocycler and ensure strains of S. roseus remain viable.
Protocol:
Three broth cultures were inoculated from single colonies into ISP media 1 and left to incubate at roughly 26°C, 200 rpm.
Stop: 2:45 pm
 
Products:
 
Label
Source
Description
8/24 BC1
2/23 I1
S. roseus ISP-5076 in ISP m. 1
8/24 BC2
2/23 I2
S. roseus B-1320 in ISP m. 1
8/24 BC3
2/23 I3
S. roseus B-3062 in ISP m. 1
 
Next:
Prepare 25% glycerol stocks from tube cultures, then use a small volume of cultures as template for PCR of LeupA and LeupB with a lysis step. Also streak strains onto ISP-4 plates.
6 July 2016
Kira Buckowing & Kaelyn Yarbrough
Start: 2:00pm
 
Gradient PCR for LeupB Attempt 3 and Gel to verify results,and old ocimene attempts
Purpose: To try and amplify the LeupB strain around the idea that the previous 5/25 A2 was the closest candidate to what was wanted and to check on the most recent ocimene attempts available.
Protocol:
Five identical PCRs were performed with different annealing temperatures using a block gradient:
7/6 A1-A5
10 μL
2X Q5 Master Mix
3 μL
MilliQ water
4 μL
5X Q5 High GC Enhancer
1 μL
1/6.3X 2/25 1
1 μL
10 μM LeupB_F
1 μL
10 μM LeupB_R
20 μL
 
Temperature (°C)
Time (s)
98
105
98c
10
gradient (A1 67.6, A2 67.9, A3 68.2, A4 68.5, A5 68.7)c
25
72c
47
72
120
4
hold
cycle X32
LTM Ed. 2 Gel A 1% agarose gel was run at 130 V for ~45 minutes:
7/6 Gel
1
9/11 D2 LP
2
10 μL 2-log purple DNA ladder
3
13 μL 7/6 A1
4
13 μL 7/6 A2
5
13 μL 7/6 A3
6
13 μL 7/6 A4
7
13 μL 7/6 A5
8
10 μL 2-log purple DNA ladder
9
9/11 L1 LP
Stop: 5:30pm
 
Results:
Next:
Try the PCR again with 5/25 A2 as the basis for LeupB, checking everything again apparently. Ocimene things need to be completely reworked, as suspected.
30 June 2016
Kira Buckowing & Kaelyn Yarbrough
Start: 11:00am
 
Gradient PCR for LeupB Attempt 2 and Gel to verify results
Purpose: To try and amplify the LeupB strain around the idea that the previous 5/25 A2 was the closest candidate to what was wanted.
Protocol:
Five identical PCRs were performed with different annealing temperatures using a block gradient:
6/30 A1-A5
10 μL
2X Q5 Master Mix
3 μL
MilliQ water
4 μL
5X Q5 High GC Enhancer
1 μL
1/6.3X 2/25 1
1 μL
10 μM LeupB_F
1 μL
10 μM LeupB_R
20 μL
 
Temperature (°C)
Time (s)
98
105
98c
10
gradient (A1 67, A2 67.4, A3 67.6, A4 67.8, A5 68)c
25
72c
47
72
120
4
hold
cycle X32
LTM Ed. 2 Gel A 1% agarose gel was run at 130 V for ~45 minutes:
6/30 Gel
1
10 μL 2-log purple DNA ladder
2
13 μL 6/30 A1
3
13 μL 6/30 A2
4
13 μL 6/30 A3
5
13 μL 6/30 A4
6
13 μL 6/30 A5
7
10 μL 2-log purple DNA ladder
Stop: 2:30pm
 
Results:
Next:
Try the PCR again with 5/25 A2 as the basis for LeupB, using a longer range this time, but noting that the Temps closer to 68 seemed to yield slightly better results.
22 June 2016
Kira Buckowing & Kaelyn Yarbrough
Start: 4:05pm
 
Gel Electrophoresis of 5/16 and 5/25 Gradient PCRs
Purpose: To check the validity of the hopefully amplified LeupA and LeupB strains
Protocol:
LTM Ed. 2 Gel A 1% agarose gel was run at 130 V for ~45 minutes:
6/22 Gel
1
10 μL 2-log purple DNA ladder
2
10 μL 5/25 A1
3
10 μL 5/25 A2
4
10 μL 5/25 A3
5
10 μL 5/25 A4
6
10 μL 2-log purple DNA ladder
7
10 μL 5/16 A1
8
10 μL 5/16 A2
9
10 μL 5/16 A3
10
10 μL 5/16 A4
Stop: 6:15
 
Results:
Next:
Try the PCR again with 5/25 A2 as the basis for LeupB, figure out why LeupA doesn't seem to be working.
25 May 2016
Kent Gorday
Start:
 
Gradient PCR of LeupB and Cultures of GST-tagged hmp
Purpose: To isolate LeupB from Streptomyces roseus and GST-tag hmp for purification
Protocol:
Four identical PCRs were performed with different annealing temperatures using a block gradient:
5/25 A1-A4
10 μL
2X Q5 Master Mix
3 μL
MilliQ water
4 μL
5X Q5 High GC Enhancer
1 μL
1/6.3X 2/25 1
1 μL
10 μM LeupB_F
1 μL
10 μM LeupB_R
20 μL
 
Temperature (°C)
Time (s)
98
105
98c
10
gradient (A1 69, A2 67.7, A3 65.9, A4 64.2)c
25
72c
47
72
120
4
hold
cycle X32
Six broth cultures were inoculated after the addition of 1X ampicillin.
Notes:
5/19 CT1 and CT2 had been left in the incubator for many days and may have been growing contamination. 5/19 CT3 plates showed one possible colony, suggesting poor transformation efficiency.
Stop:
 
Products:
 
Label
Source
Description
5/25 A1
2/25 1
LeupB amplified from S. roseus ISP-5076
5/25 A2
2/25 1
LeupB amplified from S. roseus ISP-5076
5/25 A3
2/25 1
LeupB amplified from S. roseus ISP-5076
5/25 A4
2/25 1
LeupB amplified from S. roseus ISP-5076
5/25 BC1
5/19 CT1 streak
hmp in pGEX4T-1 (GST-tagged) in ampicillin
5/25 BC2
5/19 CT1 streak
hmp in pGEX4T-1 (GST-tagged) in ampicillin
5/25 BC3
5/19 CT1 colony
hmp in pGEX4T-1 (GST-tagged) in ampicillin
5/25 BC4
5/19 CT2 streak
hmp in pGEX4T-1 (GST-tagged) in ampicillin
5/25 BC5
5/19 CT2 streak
hmp in pGEX4T-1 (GST-tagged) in ampicillin
5/25 BC6
5/19 CT2 colony
hmp in pGEX4T-1 (GST-tagged) in ampicillin
 
Next:
Miniprep and digest broth cultures if they grow, then run a gel to evaluate miniprep products and 5/25 A1-A4. Check BL21 transformation procedure
19 May 2016
Kent Gorday
Start:
 
Assembly of hmp into pGEX4T-1
Purpose: To GST-tag hmp for purification
Protocol:
Two chemical transformations were performed into Zymo Mix & Go 5α. 1.5 µL of each plasmid was mixed with 50 µL aliquots, then immediately plated onto pre-warmed plates.  A restriction digest was prepared, then incubated and heat-killed in the thermocycler:
5/19 D1
1.6 μL
MilliQ water
3 μL
10X Tango buffer (2X total)
8.6 μL
4/28 MP6
0.9 μL
EcoRI
0.9 μL
XhoI
15 μL
(2 μg)
A 1% agarose gel was run at 100 V, then the single band in lane 5 was extracted using the Gel/PCR DNA Fragment Extraction kit:
5/19 Gel #1
2
5 μL 2-log purple DNA ladder
5
15 μL 5/19 D1
A chemical transformation was performed with 1.5 μL of plasmid in 98.5 μL KCM solution, mixed with 100 μL of competent BL21. Mixture was incubated on ice for 15 minutes, then at room temperature for 25 minutes before plating.  Gel-extracted fragment was NanoDropped to determine concentration, but showed no characteristic peak at 260 nm.
Notes:
Zymo Mix & Go 5α now in a third-floor freezer
Stop:
 
Results:
 
Sample
ng/μL
260/280
260/230
5/19 GE1
0.8
-0.93
0.17
5/19 GE1
1.5
1.36
0.30
 
Products:
 
Label
Source
Description
5/19 CT1
5/16 L1
hmp in pGEX4T-1 (GST-tagged) on ampicillin
5/19 CT2
5/16 L2
hmp in pGEX4T-1 (GST-tagged) on ampicillin
5/19 D1
4/28 MP6
pGEX4T-1 digested EcoRI & XhoI
5/19 GE1
5/19 D1
pGEX4T-1 digested EcoRI & XhoI, backbone purified
5/19 CT3
10 pg/μL pUC19
transformation efficiency test using pUC19 on ampicillin
 
Next:
 
16 May 2016
Kent Gorday and Erin Nischwitz
Start: 11:40 am
 
Assembly of hmp in pGEX4T-1 and LeupA Gradient PCRs
Purpose: To GST-tag hmp for purification and characterization, and to amplify LeupA from Streptomyces roseus
Protocol:
Two restriction digests were prepared, then incubated and heat-killed in the thermocycler:
5/16 D1
8.2 μL
MilliQ water
3 μL
10X Tango buffer
2.6 μL
4/28 MP6
0.6 μL
EcoRI
0.6 μL
XhoI
15 μL
(1 μg)
5/16 D2
10.2 μL
MilliQ water
3 μL
10X Tango buffer
0.6 μL
2/20 A2
0.6 μL
EcoRI
0.6 μL
XhoI
15 μL
A 1% agarose gel was run at 100 V, then the single band in lane 5 was extracted using the Gel/PCR DNA Fragment Extraction kit:
5/16 Gel #1
2
5 μL 2-log purple DNA ladder
3
5 μL 5/16 D2
5
15 μL 5/16 D1
Four identical PCRs were performed with different annealing temperatures using a block gradient:
5/16 A1-A4
10 μL
2X Q5 Master Mix
3 μL
MilliQ water
4 μL
5X Q5 High GC Enhancer
1 μL
1/6.3X 2/25 1
1 μL
10 μM LeupA_F_V
1 μL
10 μM LeupA_R
20 μL
 
Temperature (°C)
Time (s)
98
105
98c
10
gradient (A1 72, A2 70.5, A3 68.9, A4 67.6)c
25
72c
306
72
120
4
hold
cycle X32
The gel-purified fragment was NanoDropped to determine concentration, then two ligations were incubated at 16 °C:
5/16 L1
1 μL
10X T4 ligase buffer
3.7 μL
5/16 GE1
4.3 μL
5/16 D2
1 μL
T4 DNA ligase
10 μL
(2:1)
5/16 L2
1 μL
10X T4 ligase buffer
1.5 μL
5/16 GE1
6.5 μL
5/16 D2
1 μL
T4 DNA ligase
10 μL
(8:1)
A 1% agarose gel was run at 130 V for 40 minutes:
5/16 Gel #2
3
5 μL 2-log purple DNA ladder
4
7.5 μL 5/16 A1
5
7.5 μL 5/16 A2
6
7.5 μL 5/16 A3
7
7.5 μL 5/16 A4
Two chemical transformations were performed with 1.5 μL of plasmid in 98.5 μL KCM solution, mixed with 100 μL of competent BL21. Mixture was incubated on ice for 15 minutes, then at room temperature for 25 minutes before plating.
Notes:
Zymo Mix & Go DH5α no longer in Dr. Simone's freezer
Stop: 9:00 pm
 
Results:
5/16 GE1 was reported as 3.9 μL but showed no characteristic peak.
Products:
 
Label
Source
Description
5/16 D1
4/28 MP6
pGEX4T-1 digested EcoRI & XhoI
5/16 D2
2/20 A2
hmp with GST-tagging sites digested EcoRI & XhoI
5/16 GE1
5/16 D1
pGEX4T-1 digested EcoRI & XhoI, purified backbone
5/16 A1
2/25 1
LeupA amplified from S. roseus ISP-5076
5/16 A2
2/25 1
LeupA amplified from S. roseus ISP-5076
5/16 A3
2/25 1
LeupA amplified from S. roseus ISP-5076
5/16 A4
2/25 1
LeupA amplified from S. roseus ISP-5076
5/16 L1
5/16 D1, 5/16 D2
hmp in pGEX4T-1 (GST-tagged)
5/16 L2
5/16 D1, 5/16 D2
hmp in pGEX4T-1 (GST-tagged)
5/16 CT1
5/16 L1
hmp in pGEX4T-1 (GST-tagged) on ampicillin
5/16 CT2
5/16 L2
hmp in pGEX4T-1 (GST-tagged) on ampicillin
 
Next:
Locate competent DH5α to retry transformations and perform transformation efficiency test of BL21 if no transformants, otherwise inoculate broth cultures of transformed BL21 and miniprep>digest>gel to screen for the correct plasmid. Retry LeupA amplification without GC Enhancer, under the same program at two differrent annealing temperatures.
29 April 2016
Kent Gorday
Start:
 
Verification of GST-tagged hmp and pGEX4T-1 Minipreps
Purpose: To determine if minipreps of GST-tagged hmp and pGEX4T-1 are the correct products
Protocol:
Seven restriction digests were prepared, then incubated and heat-killed in the thermocycler:
4/29 D1
3 μL
10X Tango buffer (2X total)
5.7 μL
MilliQ water
0.6 μL
EcoRI
0.6 μL
XhoI
5.1 μL
4/28 MP1
15 μL
(1 μg)
4/29 D2
3 μL
10X Tango buffer (2X total)
5.8 μL
MilliQ water
0.6 μL
EcoRI
0.6 μL
XhoI
5 μL
4/28 MP2
15 μL
(1 μg)
4/29 D3
3 μL
10X Tango buffer (2X total)
7.3 μL
MilliQ water
0.6 μL
EcoRI
0.6 μL
XhoI
3.5 μL
4/28 MP3
15 μL
(1 μg)
4/29 D4
3 μL
10X Tango buffer (2X total)
6.8 μL
MilliQ water
0.6 μL
EcoRI
0.6 μL
XhoI
4 μL
4/28 MP4
15 μL
(1 μg)
4/29 D5
3 μL
10X Tango buffer (2X total)
6.3 μL
MilliQ water
0.6 μL
EcoRI
0.6 μL
XhoI
4.5 μL
4/28 MP5
15 μL
(1 μg)
4/29 D6
3 μL
10X Tango buffer (2X total)
6.5 μL
MilliQ water
0.6 μL
EcoRI
0.6 μL
XhoI
4.3 μL
4/28 MP6
15 μL
(1 μg)
4/29 D7
3 μL
10X Tango buffer (2X total)
8.2 μL
MilliQ water
0.6 μL
EcoRI
0.6 μL
XhoI
2.6 μL
4/28 MP7
15 μL
(1 μg)
A 1% agarose gel was run at 130 V:
4/29 Gel #1
1
15 μL 4/29 D1
2
15 μL 4/29 D2
3
15 μL 4/29 D3
4
15 μL 4/29 D4
5
7.5 μL 2-log purple DNA ladder
6
15 μL 4/29 D5
7
15 μL 4/29 D6
8
15 μL 4/29 D7
 
Stop:
 
Results:
Products:
 
Label
Source
Description
4/29 D1
4/28 MP1
religated pGEX4T-1 digested EcoRI & XhoI
4/29 D2
4/28 MP2
religated pGEX4T-1 digested EcoRI & XhoI
4/29 D3
4/28 MP3
hmp in pGEX4T-1 digested EcoRI & XhoI
4/29 D4
4/28 MP4
hmp in pGEX4T-1 digested EcoRI & XhoI
4/29 D5
4/28 MP5
pGEX4T-1 digested EcoRI & XhoI
4/29 D6
4/28 MP6
pGEX4T-1 digested EcoRI & XhoI
4/29 D7
4/28 MP7
pGEX4T-1 digested EcoRI & XhoI
 
Next:
 
28 April 2016
Kent Gorday and Erin Nischwitz
Start:
 
Minipreps of GST-tagged hmp and pGEX4T-1
Purpose: To obtain GST-tagged hmp for purification
Protocol:
Seven minipreps were performed using the kitless miniprep procedure, then products were NanoDropped to determine concentration.
Stop:
 
Results:
 
Sample
ng/μL
260/280
260/230
4/28 MP1
195.53
1.90
2.65
4/28 MP2
199.29
1.90
2.68
4/28 MP3
285.32
1.91
2.67
4/28 MP4
251.04
1.90
2.55
4/28 MP5
221.64
1.85
1.99
4/28 MP6
233.73
1.87
1.76
4/28 MP7
384.50
1.87
2.07
 
Products:
 
Label
Source
Description
4/28 MP1
4/27 BC2
religated pGEX4T-1
4/28 MP2
4/27 BC3
religated pGEX4T-1
4/28 MP3
4/27 BC5
hmp in pGEX4T-1
4/28 MP4
4/27 BC6
hmp in pGEX4T-1
4/28 MP5
4/25 BC1
pGEX4T-1
4/28 MP6
4/25 BC2
pGEX4T-1
4/28 MP7
4/25 BC3
pGEX4T-1
 
Next:
 
27 April 2016
Kent Gorday
Start:
 
Cultures of GST-tagged hmp
Purpose: To obtain GST-tagged hmp for purification
Protocol:
Six broth cultures were inoculated after the addition of 1X ampicillin.
Stop:
 
Products:
 
Label
Source
Description
4/27 BC1
4/26 CT1
religated pGEX4T-1 in ampicillin
4/27 BC2
4/26 CT1
religated pGEX4T-1 in ampicillin
4/27 BC3
4/26 CT1
religated pGEX4T-1 in ampicillin
4/27 BC4
4/26 CT2
hmp in pGEX4T-1 in ampicillin
4/27 BC5
4/26 CT2
hmp in pGEX4T-1 in ampicillin
4/27 BC6
4/26 CT2
hmp in pGEX4T-1 in ampicillin
 
Next:
 
26 April 2016
Kent Gorday
Start:
 
Religation of pGEX4T-1 and Transformation of pGEX4T-1 and hmp in pGEX4T-1
Purpose: To GST-tag hmp for purification and obtain more pGEX4T-1 for future use
Protocol:
A ligation was mixed and incubated at 16°C:
4/26 L1
0.75 μL
10X T4 ligase buffer
6 μL
10/11 D6
0.75 μL
T4 DNA ligase
7.5 μL
Two chemical transformations were performed into 50 μL Zymo Mix & Go 5α, which were plated immediately onto pre-warmed plates.
Stop:
 
Products:
 
Label
Source
Description
4/26 L1
10/11 D6
religated pGEX4T-1
4/26 CT1
4/26 L1
religated pGEX4T-1 on ampicillin
4/26 CT2
4/19 L2
hmp in pGEX4T-1 on ampicillin
 
Next:
 
25 April 2016
Kent Gorday
Start:
 
Broth Cultures of pGEX4T-1
Purpose: To obtain additional pGEX4T-1 plasmid for future use
Protocol:
Six broth cultures were inoculated after the addition of 1X ampicillin.
Stop:
 
Products:
 
Label
Source
Description
4/25 BC1
8/2/15 pGEX I18
pGEX4T-1 in ampicillin
4/25 BC2
8/2/15 pGEX I18
pGEX4T-1 in ampicillin
4/25 BC3
8/2/15 pGEX I18
pGEX4T-1 in ampicillin
4/25 BC4
8/2/15 pGEX I19
pGEX4T-1 in ampicillin
4/25 BC5
8/2/15 pGEX I19
pGEX4T-1 in ampicillin
4/25 BC6
8/2/15 pGEX I19
pGEX4T-1 in ampicillin
 
Next:
 
24 April 2016
Kent Gorday
22 April 2016
Kent Gorday
21 April 2016
Kent Gorday
Start: 3:00 pm
 
Purpose: To obtain and verify GST-tagged hmp and diagnose LeupA PCRs
Protocol:
Three minipreps were performed using the kitless miniprep procedure and resuspended in 38 μL 1X TE. Products were NanoDropped to determine concentration.  Four restriction digests were mixed, then reacted and heat-killed in the thermocycler:
4/21 D1
7.7 μL
MilliQ water
1 μL
10X Tango buffer
0.5 μL
4/19 A1
0.4 μL
EcoRI
0.4 μL
SpeI
10 μL
4/21 D2
3.2 μL
MilliQ water
1 μL
10X Tango buffer
5 μL
4/19 A2
0.4 μL
EcoRI
0.4 μL
SpeI
10 μL
4/21 D3
6.9 μL
MilliQ water
1 μL
10X Tango buffer
1.3 μL
4/21 MP2
0.4 μL
EcoRI
0.4 μL
XhoI
10 μL
(1 μg)
4/21 D4
6.1 μL
MilliQ water
1 μL
10X Tango buffer
2.1 μL
4/21 MP3
0.4 μL
EcoRI
0.4 μL
XhoI
10 μL
(1 μg)
 
Notes:
4/20 BC1, 2, and 4 had no growth. 4/21 MP1 was discarded after it was dropped.
Stop: 6:30 pm
 
Results:
 
Sample
ng/μL
260/280
260/230
4/21 MP2
752.93
1.87
1.82
4/21 MP3
467.38
1.88
2.25
 
Products:
 
Label
Source
Description
4/21 MP1
4/18 BC1
pGEX4T-1 (discarded)
4/21 MP2
4/20 BC3
hmp in pGEX4T-1
4/21 MP3
4/20 BC5
hmp in pGEX4T-1
4/21 D1
4/19 A1
LeupA amplification digested EcoRI & SpeI
4/21 D2
4/19 A2
LeupA amplification digested EcoRI & SpeI
4/21 D3
4/21 MP2
hmp in pGEX4T-1 digested EcoRI & XhoI
4/21 D4
4/21 MP3
hmp in pGEX4T-1 digested EcoRI & XhoI
 
Next:
Run a gel of digests
20 April 2016
Kent Gorday
Start: 11:50 pm
 
Broth Cultures of pGEX4T-1 and GST-tagged hmp
Purpose: To obtain pGEX4T-1 and GST-tagged hmp
Protocol:
Five broth cultures were inoculated from plates after the addition of 5 μL 1000X ampicillin.  4/18 BC1 was moved into the refrigerator.
Notes:
There was no growth in 4/18 BC2.
Stop: 1:00 am
 
Products:
 
Label
Source
Description
4/20 BC1
4/19 CT1
pGEX4T-1 in ampicillin
4/20 BC2
4/19 CT1
pGEX4T-1 in ampicillin
4/20 BC3
4/19 CT2
hmp in pGEX4T-1 (GST-tagged) in ampicillin
4/20 BC4
4/19 CT2
hmp in pGEX4T-1 (GST-tagged) in ampicillin
4/20 BC5
4/19 CT2
hmp in pGEX4T-1 (GST-tagged) in ampicillin
 
Next:
Miniprep 4/18 BC1 and 4/20 BC1-5
19 April 2016
Kent Gorday and Erin Nischwitz
Start: 5:00 pm
 
New PCRs of LeupA and Ligations of hmp in pGEX4T-1
Purpose: To isolate LeupA from Streptomyces roseus strains for cloning, and GST-tag hmp.
Protocol:
Three PCR reactions were mixed without polymerase, denatured for 3 minutes on the thermocycler block at 98 °C, then incubated in the thermocycler after the addition of polymerase:
4/19 A1
3 μL
MilliQ water
4 μL
5X Q5 High GC Enhancer
1 μL
10 μM LeupA_F_V
1 μL
10 μM LeupA_R
1 μL
1/6.3X 2/25 1
10 μL
2X Q5 Master Mix
20 μL
4/19 A2
3 μL
MilliQ water
4 μL
5X Q5 High GC Enhancer
1 μL
10 μM LeupA_F_V
1 μL
10 μM LeupA_R
1 μL
1/3.7X 2/25 2
10 μL
2X Q5 Master Mix
20 μL
4/19 A3
3 μL
MilliQ water
4 μL
5X Q5 High GC Enhancer
1 μL
10 μM LeupA_F_V
1 μL
10 μM LeupA_R
1 μL
1/8.5X 2/25 3
10 μL
2X Q5 Master Mix
20 μL
 
Temperature (°C)
Time (s)
98
30
98c
10
72c
400
72
120
4
hold
cycle X32
Two ligations were mixed and left to react at room temperature for four hours:
4/19 L1
1 µL
10X T4 ligase buffer
7 µL
MilliQ water
1 µL
10/11 D6
1 µL
T4 DNA ligase
10 µL
4/19 L2
0.5 µL
10X T4 ligase buffer
3.5 µL
2/20 D2
0.5 µL
1/2X 10/11 D6
0.5 µL
T4 DNA ligase
5 µL
A 1% agarose gel was run at 100 V until the loading dye reached 2/3 of the gel length:
4/19 Gel #1
2
2 µL 10/11 D6 + 5.5 µL 1X TE buffer
3
5 µL 2-log purple DNA ladder
5
7.5 µL 4/19 A1
7
7.5 µL 4/19 A2
9
7.5 µL 4/19 A3
Two chemical transformations were performed into Zymo Mix & Go 5α. 1.5 µL of each plasmid was mixed with 50 µL aliquots, then incubated for 7 minutes on ice. 200 µL of SOC was added for 45 minutes of outgrowth at 37 °C, then 80 µL of each was plated.
Stop: 9:30 pm
 
Results:
Products:
 
Label
Source
Description
4/19 A1
2/25 1
LeupA amplified from S. roseus ISP-5076
4/19 A2
2/25 2
LeupA amplified from S. roseus B-1320
4/19 A3
2/25 3
LeupA amplified from S. roseus B-3062
4/19 L1
10/11 D6
Religated pGEX4T-1
4/19 L2
10/11 D6, 2/20 D2
hmp in pGEX4T-1 (GST-tagged)
4/19 CT1
4/19 L1
Religated pGEX4T-1 on ampicillin
4/19 CT2
4/19 L2
hmp in pGEX4T-1 (GST-tagged) on ampicillin
 
Next:
Test pH of autoclaved MilliQ water with High GC Enhancer, inoculate broth cultures and miniprep from transformations, miniprep 4/18 broth cultures, then digest and gel to verify products.
18 April 2016
Kent Gorday
Start: 3:00 pm
 
Cultures of pGEX4T-1
Purpose: to obtain new pGEX4T-1 for GST-tagging hmp
Protocol:
Two broth cultures were inoculated from plates after the addition of 5 μL 1000X ampicillin.
Stop: 4:00 pm
 
Products:
 
Label
Source
Description
4/18 BC1
8/2/15 I18
pGEX4T-1 in ampicillin
4/18 BC2
8/2/15 I19
pGEX4T-1 in ampicillin
 
Next:
Retry LeupA PCRs with longer denaturing and extension times. Miniprep broth cultures, and retry hmp ligations with 10/11 D6.
15 April 2016
Kent Gorday
Start: 1:20 pm
 
Protocol:
Eight restriction digests were prepared before incubation and heat-kill in the thermocycler:
4/15 D1
7 μL
MilliQ water
1 μL
10X Tango buffer
1.2 μL
4/14 MP1
0.4 μL
EcoRI
0.4 μL
SpeI
10 μL
(1 μg)
4/15 D2
6.1 μL
MilliQ water
1 μL
10X Tango buffer
2.1 μL
4/14 MP2
0.4 μL
EcoRI
0.4 μL
SpeI
10 μL
(1 μg)
4/15 D3
5.5 μL
MilliQ water
1 μL
10X Tango buffer
2.7 μL
4/14 MP3
0.4 μL
EcoRI
0.4 μL
XhoI
10 μL
(1 μg)
4/15 D4
6.6 μL
MilliQ water
1 μL
10X Tango buffer
1.6 μL
4/14 MP4
0.4 μL
EcoRI
0.4 μL
XhoI
10 μL
(1 μg)
4/15 D5
7 μL
MilliQ water
1 μL
10X Tango buffer
1.2 μL
4/14 MP5
0.4 μL
EcoRI
0.4 μL
XhoI
10 μL
(1 μg)
4/15 D6
6.3 μL
MilliQ water
1 μL
10X Tango buffer
1.9 μL
4/14 MP6
0.4 μL
EcoRI
0.4 μL
XhoI
10 μL
(1 μg)
4/15 D7
3.8 μL
MilliQ water
1 μL
10X Tango buffer
4.4 μL
4/14 MP7
0.4 μL
EcoRI
0.4 μL
XhoI
10 μL
(1 μg)
4/15 D8
6.9 μL
MilliQ water
1 μL
10X Tango buffer
1.3 μL
4/14 MP8
0.4 μL
EcoRI
0.4 μL
XhoI
10 μL
(1 μg)
A 1% agarose gel was run at 90 V until the dye reached 2/3 of the gel length:
1
10 μL 4/15 D1
2
10 μL 4/15 D2
3
5 μL 2-log purple DNA ladder
4
10 μL 4/15 D3
5
10 μL 4/15 D4
6
10 μL 4/15 D5
7
10 μL 4/15 D6
8
5 μL 2-log purple DNA ladder
9
10 μL 4/15 D7
10
10 μL 4/15 D8
 
Stop: 5:30 pm
 
Results:
Products:
 
Label
Source
Description
4/15 D1
4/14 MP1
LeupA from 3/17 GE2 in pSB1C3 cut EcoRI & SpeI
4/15 D2
4/14 MP2
LeupA from 3/17 GE2 in pSB1C3 cut EcoRI & SpeI
4/15 D3
4/14 MP3
hmp in pGEX4T-1 cut EcoRI & XhoI
4/15 D4
4/14 MP4
hmp in pGEX4T-1 cut EcoRI & XhoI
4/15 D5
4/14 MP5
hmp in pGEX4T-1 cut EcoRI & XhoI
4/15 D6
4/14 MP6
hmp in pGEX4T-1 cut EcoRI & XhoI
4/15 D7
4/14 MP7
hmp in pGEX4T-1 cut EcoRI & XhoI
4/15 D8
4/14 MP8
hmp in pGEX4T-1 cut EcoRI & XhoI
 
Next:
Re-try PCR amplification of LeupB and get a clean amplification of LeupA for digestion, then gel purification and ligation into pSB1C3. Obtain more pGEX4T-1 plasmid, digest then gel extract the backbone before ligating with hmp. Perform new transformations.
14 April 2016
Kent Gorday
Start: 6:45 pm
 
Minipreps of LeupA in pSB1C3 and hmp in pGEX4T-1
Purpose: To miniprep transformants to obtain plasmids and verify their assembly
Protocol:
Eight minipreps were performed from the entire volume of broth culture using the kitless miniprep protocol. Spectrophotometry was used to estimate DNA concentration in the products.
Notes:
4/13 BC3 had no growth and was not miniprepped
Stop: 9:30 pm
 
Results:
 
Sample
ng/μL
260/280
260/230
4/14 MP1
840.38
1.90
2.31
4/14 MP2
490.54
1.85
2.25
4/14 MP3
379.14
1.83
2.03
4/14 MP4
654.19
1.89
2.31
4/14 MP5
883.61
1.90
2.35
4/14 MP6
541.91
1.86
2.17
4/14 MP7
229.59
1.90
2.74
4/14 MP8
769.14
1.82
1.56
 
Products:
 
Label
Source
Description
4/14 MP1
4/13 BC1
LeupA from 3/17 GE2 in pSB1C3
4/14 MP2
4/13 BC2
LeupA from 3/17 GE2 in pSB1C3
4/14 MP3
4/13 BC4
hmp in pGEX4T-1
4/14 MP4
4/13 BC5
hmp in pGEX4T-1
4/14 MP5
4/13 BC6
hmp in pGEX4T-1
4/14 MP6
4/13 BC7
hmp in pGEX4T-1
4/14 MP7
4/13 BC8
hmp in pGEX4T-1
4/14 MP8
4/13 BC9
hmp in pGEX4T-1
 
Next:
Digest miniprep products and run a gel to verify identity
13 April 2016
Kent Gorday, Ryan Baumann, and Erin Nischwitz
Start: 5:00 pm
 
Digests for Verification of LeupA and hmp Transformants
Purpose: To verify the identity of plasmids miniprepped from LeupA and hmp transformants, and amplify LeupB
Protocol:
Two PCR reactions were incubated in the thermocycler:
4/13 A1
3 μL
MilliQ water
10 μL
2X Q5 Master Mix
1 μL
10 μM LeupB_F
1 μL
10 μM LeupB_R
4 μL
5X Q5 High GC Enhancer
1 μL
1/6.3X 2/25 1
20 μL
4/13 A2
 
3 μL
MilliQ water
10 μL
2X Q5 Master Mix
1 μL
10 μM LeupB_F
1 μL
10 μM LeupB_R
4 μL
5X Q5 High GC Enhancer
1 μL
1/3.7X 2/25 2
20 μL
 
Temperature (°C)
Time (s)
98
30
98c1
7
70-0.5c1
26
72c1
44
98c2
7
62c2
26
72c2
44
72
120
4
hold
cycle1 X16, cycle2 X14 4/13 A2 cracked in the thermocycler and was thrown away
Four restriction digests were mixed then incubated for 1 hour at 37 °C and heat-killed in the thermocycler:
4/13 D1
8 µL
MilliQ water
2.5 µL
10X Tango buffer
1 µL
EcoRI
1 µL
SpeI
12.5 µL
3/14 A1
25 µL
4/13 D2
8 µL
MilliQ water
2.5 µL
10X Tango buffer
1 µL
EcoRI
1 µL
SpeI
12.5 µL
3/12 A2
25 µL
4/13 D3
8.75 µL
MilliQ water
1.25 µL
10X Tango buffer
0.5 µL
EcoRI
2 µL
4/11 MP4
12.5 µL
(500 ng)
4/13 D4
8.75 μL
MilliQ water
1.25 μL
10X Tango buffer
0.5 μL
XhoI
2 μL
4/11 MP4
12.5 μL
(500 ng)
A 1% agarose gel was run at 78 V until the bands reached 2/3 of the gel length:
4/13 Gel #1
2
7.5 μL 4/13 A2
3
5 μL 2-log purple DNA ladder
4
25 μL 4/13 D1
5
25 μL 4/13 D2
6
12.5 μL 4/13 D3
7
12.5 μL 4/13 D4
8
5 μL 2-log purple DNA ladder
5 µL of 1000X chloramphenicol or ampicillin was added to each of nine broth cultures, which were then inoculated from plates by wire loop.
Stop: 1:00 am
 
Results:
Products:
 
Label
Source
Description
4/13 A1
2/25 1
LeupB amplified from S. roseus ISP-5076
4/13 A2
2/25 2
LeupB amplified from S. roseus B-1320 (discarded)
4/13 D1
3/14 A1
LeupA amplified from S. roseus ISP-5076 digested EcoRI & SpeI
4/13 D2
3/12 A2
LeupA amplified from S. roseus B-1320 digested EcoRI & SpeI
4/13 D3
4/11 MP4
hmp in pGEX4T-1 digested EcoRI
4/13 D4
4/11 MP4
hmp in pGEX4T-1 digested XhoI
4/13 BC1
4/9 CT1 145 µL (white colony)
LeupA from 3/17 GE2 in pSB1C3 in chloramphenicol
4/13 BC2
4/9 CT1 145 µL (white colony)
LeupA from 3/17 GE2 in pSB1C3 in chloramphenicol
4/13 BC3
4/9 CT1 145 µL (white colony)
LeupA from 3/17 GE2 in pSB1C3 in chloramphenicol
4/13 BC4
4/9 CT2 145 µL (colony with satellites)
hmp in pGEX4T-1 in ampicillin
4/13 BC5
4/9 CT2 145 µL (colony with satellites)
hmp in pGEX4T-1 in ampicillin
4/13 BC6
4/9 CT2 145 µL (colony with satellites)
hmp in pGEX4T-1 in ampicillin
4/13 BC7
4/9 CT3 145 µL (colony with satellites)
hmp in pGEX4T-1 in ampicillin
4/13 BC8
4/9 CT3 145 µL (colony with satellites)
hmp in pGEX4T-1 in ampicillin
4/13 BC9
4/9 CT3 145 µL (colony with satellites)
hmp in pGEX4T-1 in ampicillin
 
11 April 2016
Kent Gorday and Erin Nischwitz
Start: 5:00 pm
 
Miniprep, digestion, and gel verification of LeupA in pSB1C3 and GST-tagged hmp
Purpose: To verify assembly of LeupA into pSB1C3 and hmp into pGEX4T-1
Protocol:
Four minipreps were performed using 3 mL of broth culture each under the kitless miniprep protocol. Product concentration was measured with the NanoDrop.  Four digestions were mixed and incubated in the thermocycler at 37 °C, then heat-killed:
4/11 D1
19.3 µL
MilliQ water
2.5 µL
10X Tango buffer
1 µL
EcoRI
1 µL
SpeI
1.2 µL
4/11 MP1
25 µL
(1.5 µg)
4/11 D2
17.8 µL
MilliQ water
2.5 µL
10X Tango buffer
1 µL
EcoRI
1 µL
SpeI
2.7 µL
4/11 MP2
25 µL
(1.5 µg)
4/11 D3
19.1 µL
MilliQ water
2.5 µL
10X Tango buffer
1 µL
EcoRI
1 µL
SpeI
1.4 µL
4/11 MP3
25 µL
(1.5 µg)
4/11 D4
14.6 µL
MilliQ water
2.5 µL
10X Tango buffer
1 µL
EcoRI
1 µL
XhoI
5.9 µL
4/11 MP4
25 µL
(1.5 µg)
  A 1% agarose gel was prepared and run at 120 V for 45 minutes:
4/11 Gel #1
2
5 µL 2-log purple DNA ladder
3
7.5 µL 4/11 D1
4
7.5 µL 4/11 D2
5
7.5 µL 4/11 D3
6
7.5 µL 4/11 D4
  Two ligations were mixed and incubated at room temperature overnight:
4/11 L1
1 µL
10X T4 ligase buffer
1 µL
1/19X 3/26 D2
7 µL
4/10 D1
1 µL
T4 DNA ligase
10 µL
4/11 L2
1 µL
10X T4 ligase buffer
1 µL
1/50X 3/26 D2
7 µL
4/10 D2
1 µL
T4 DNA ligase
10 µL
 
Stop: 11:30 pm
 
Results:
 
Sample
ng/μL
260/280
260/230
4/11 MP1
1333.07
1.91
2.37
4/11 MP2
551.06
1.87
2.32
4/11 MP3
1058.70
1.91
2.32
4/11 MP4
254.90
1.89
2.54
Products:
 
Label
Source
Description
4/11 MP1
4/10 BC1
LeupA from 3/17 GE2 in pSB1C3
4/11 MP2
4/10 BC2
LeupA from 3/17 GE2 in pSB1C3
4/11 MP3
4/10 BC3
LeupA from 3/17 GE2 in pSB1C3
4/11 MP4
4/10 BC6
hmp in pGEX4T-1
4/11 D1
4/11 MP1
LeupA from 3/17 GE2 in pSB1C3 digested EcoRI & SpeI
4/11 D2
4/11 MP2
LeupA from 3/17 GE2 in pSB1C3 digested EcoRI & SpeI
4/11 D3
4/11 MP3
LeupA from 3/17 GE2 in pSB1C3 digested EcoRI & SpeI
4/11 D4
4/11 MP4
hmp in pGEX4T-1 digested EcoRI & XhoI
4/11 L1
4/10 D1, 3/26 D2
LeupA from 3/17 GE1 in pSB1C3
4/11 L2
4/10 D2, 3/26 D2
LeupA from 3/17 GE3 in pSB1C3
 
10 April 2016
Kent Gorday and Erin Nischwitz
Start: 10:30 am
 
New LeupB PCR
Purpose: to amplify LeupB from Streptomyces roseus
Protocol:
New dilutions of primers were performed, then one PCR was incubated in the thermocycler:
4/10 A1
3 µL
MilliQ water
10 µL
2X Q5 Master Mix
1 µL
10 µM LeupB_F
1 µL
10 µM LeupB_R
4 µL
5X Q5 High GC Enhancer
1 µL
1/3.7X 2/25 2
20 µL
 
Temperature (°C)
Time (s)
98
30
98c
7
68c
26
72c
44
72
120
4
hold
cycle X30
A 1% agarose gel was prepared and run at 130 V for 45 minutes:
Gel 4/10 #1
3
5 µL 2-log purple DNA ladder
5
7.5 µL 4/10 A1
Six broth cultures were inoculated from plates. Two restriction digests were mixed, incubated, then heat killed in the thermocycler:
4/10 D1
2.5 µL
10X Tango buffer
1 µL
EcoRI
1 µL
SpeI
20.5 µL
3/17 GE1
25 µL
(65.8 ng)
4/10 D2
2.5 µL
10X Tango buffer
1 µL
EcoRI
1 µL
SpeI
20.5 µL
3/17 GE3
25 µL
(31.2 ng)
 
Stop: 10:00 pm
 
Results:
Products:
 
Label
Source
Description
4/10 A1
2/25 2
LeupB amplified from S. roseus B-1320
4/10 D1
3/17 GE1
LeupA amplified and gel extracted, digested EcoRI & SpeI
4/10 D2
3/17 GE3
LeupA amplified and gel extracted, digested EcoRI & SpeI
4/10 BC1
4/9 CT1 145 µL (white colony)
LeupA from 3/17 GE2 in pSB1C3 in chloramphenicol
4/10 BC2
4/9 CT1 145 µL (white colony)
LeupA from 3/17 GE2 in pSB1C3 in chloramphenicol
4/10 BC3
4/9 CT1 145 µL (white colony)
LeupA from 3/17 GE2 in pSB1C3 in chloramphenicol
4/10 BC4
4/9 CT2 145 µL (independent colony)
hmp in pGEX4T-1 in ampicillin
4/10 BC5
4/9 CT2 145 µL (independent colony)
hmp in pGEX4T-1 in ampicillin
4/10 BC6
4/9 CT2 145 µL (colony with satellites)
hmp in pGEX4T-1 in ampicillin
 
Next:
Re-try LeupB amplification. Miniprep broth cultures, then digest and run a gel to verify product identity.
9 April 2016
Kent Gorday
Start: 9:45 am
 
PCR of LeupB, Ligation and Transformation of LeupA and hmp
Purpose: To isolate LeupB from Streptomyces roseus and clone LeupA and GST-tagged hmp.
Protocol:
Three PCRs were prepared and reacted in the thermocycler:
4/9 A1
3 µL
MilliQ water
4 µL
5X Q5 High GC Enhancer
10 µL
2X Q5 Master Mix
1 µL
10 µM LeupB_F
1 µL
10 µM LeupB_R
1 µL
1/6.3X 2/25 1
20 µL
4/9 A2
3 µL
MilliQ water
4 µL
5X Q5 High GC Enhancer
10 µL
2X Q5 Master Mix
1 µL
10 µM LeupB_F
1 µL
10 µM LeupB_R
1 µL
1/3.7X 2/25 2
20 µL
4/9 A3
3 µL
MilliQ water
4 µL
5X Q5 High GC Enhancer
10 µL
2X Q5 Master Mix
1 µL
10 µM LeupB_F
1 µL
10 µM LeupB_R
1 µL
1/8.5X 2/25 3
20 µL
 
Temperature (°C)
Time (s)
98
30
98c
7
69.6c
24
72c
40
72
120
4
hold
cycle X30
A 1% agarose gel was prepared and run at 130 V for 45 minutes:
Gel 4/9 #1
1
5 µL 3/26 D1
2
5 µL 3/26 D2
3
7.5 µL 2-log purple DNA ladder
5
7.5 µL 4/9 A1
7
7.5 µL 4/9 A2
9
7.5 µL 4/9 A3
note: pSB1C3 in lanes 1 and 2 is correct but appears to contain some undigested or single-digested product. Since the two digestions only have EcoRI in common, this could be the culprit, but it should not matter if red colonies (religated) are selected against. No amplifications of LeupB were successful.  Two ligations were mixed and reacted at room temperature for 3 hours:
4/9 L1
1 µL
10X T4 ligase buffer
1 µL
1/20X 3/26 D2 (1.1 fmol pSB1C3)
7 µL
3/26 D3 (4.5 fmol LeupA)
1 µL
T4 DNA ligase
10 µL
4/9 L2
1 µL
10X T4 ligase buffer
1 µL
1/2X 10/11 D5
7 µL
2/20 D2
1 µL
T4 DNA ligase
10 µL
Three chemical transformations were performed into Zymo Mix & Go 5α. 1.4 µL of each plasmid was mixed with 33 µL aliquots, then incubated for 7 minutes on ice. 140 µL of SOC was added for 1 hour of outgrowth at 37 °C, then 145 µL and 14.5 µL were plated.
Stop: 9:30 pm
 
Results:
Products:
 
Label
Source
Description
4/9 A1
2/25 1
LeupB amplified from S. roseus ISP-5076
4/9 A2
2/25 2
LeupB amplified from S. roseus B-1320
4/9 A3
2/25 3
LeupB amplified from S. roseus B-3062
4/9 L1
3/26 D2, 3/26 D3
LeupA from 3/17 GE2 in pSB1C3
4/9 L2
10/11 D5, 2/20 D2
hmp in pGEX4T-1
4/9 CT1
4/9 L1
LeupA in pSB1C3 on chloramphenicol
4/9 CT2
4/9 L2
hmp in pGEX4T-1 on ampicillin
4/9 CT3
3/1 L1
hmp in pGEX4T-1 on ampicillin
 
Next:
Culture and miniprep non-red colonies of 4/9 CT1, CT2, and CT3. Digest and run a gel to verify product identity.
26 March 2016
Kent Gorday
24 March 2016
Kent Gorday
23 March 2016
Kent Gorday
21 March 2016
Kent Gorday
18 March 2016
Kent Gorday
17 March 2016
Kent Gorday
14 March 2016
Kent Gorday
13 March 2016
Kent Gorday
12 March 2016
Kent Gorday
11 March 2016
Kent Gorday
Start: 1:00 pm
 
PCR Amplification of LeupA
Purpose: To isolate LeupA from the genome of 3 Streptomyces roseus strains.
Protocol:
Three PCRs were prepared and reacted in the thermocycler with the same program:
3/11 A1
9 µL
MilliQ water
12.5 µL
2X Q5 Master Mix
1.25 µL
10 µM LeupA_F_Val
1.25 µL
10 µM LeupA_R
1 µL
1/6.3X 2/25 1
25 µL
3/11 A2
9 µL
MilliQ water
12.5 µL
2X Q5 Master Mix
1.25 µL
10 µM LeupA_F_Val
1.25 µL
10 µM LeupA_R
1 µL
1/3.7X 2/25 2
25 µL
3/11 A3
9 µL
MilliQ water
12.5 µL
2X Q5 Master Mix
1.25 µL
10 µM LeupA_F_Val
1.25 µL
10 µM LeupA_R
1 µL
1/8.5X 2/25 3
25 µL
 
Temperature (°C)
Time (s)
98
30
98c
7
72c
204
72
120
4
hold
cycle X30
A 1% agarose gel was run at 130 V for 45 minutes:
Gel 3/11 #1
3
5 µL 2-log purple DNA ladder
4
5 µL 3/11 A1
5
5 µL 3/11 A2
6
5 µL 3/11 A3
note: no bands at the expected 7.3 kbp, strong undesired band in 3/11 A2
Stop: 5:00 pm
 
Results:
Products:
 
Label
Source
Description
3/11 A1
2/25 1
LeupA amplified from S. roseus ISP-5076
3/11 A2
2/25 2
LeupA amplified from S. roseus B-1320
3/11 A3
2/25 3
LeupA amplified from S. roseus B-3062
 
Next:
Re-try amplification with Q5 High GC Enhancer
4 March 2016
Kent Gorday
Start: 4:00 pm
 
Gel of Part-1 and Positive Control PCRs
Purpose: to verify amplification of Part-1 and Q5 positive control
Protocol:
A 1% agarose gel was run at 130 V for 40 minutes:
3/4 Gel #1
5
10μL 3/3 A1
6
10μL 3/3 A2
7
10μL 2-log purple DNA ladder
 
Stop: 5:50 pm
 
Results:
Next:
 
3 March 2016
Kent Gorday
Start: 10:00 am
 
Part-1 and Positive Control PCRs
Purpose: to amplify Part-1 for verification of its assembly, and test 2X Q5 Master Mix
Protocol:
Two PCR reactions were performed in the thermocycler under the same program:
3/3 A1
7 µL
MilliQ water
10 µL
2X Q5 Master Mix
1 µL
10 µM VF2
1 µL
10 µM VR
1 µL
1/10X 3/1 L2
20 µL
3/3 A2
7 µL
MilliQ water
10 µL
2X Q5 Master Mix
1 µL
10 µM VF2
1 µL
10 µM VR
1 µL
5 pg/µL RFP construct
20 µL
 
Temperature (°C)
Time (s)
98
30
98c
7
65c
20
72c
100
72
120
4
hold
cycle X32
 
Stop: 11:00 am
 
Products:
 
Label
Source
Description
3/3 A1
3/1 L2
Part-1 assembled, ligated into pBSIC3 and amplified (Q5)
3/3 A2
RFP in pSBB1C3
Positive control of 2X Q5 Master Mix
 
Next:
Run a gel of PCRs, also see Mar. 1.
1 March 2016
Kent Gorday
Start: 8:47 pm
 
Ligation of GST-tagged hmp
Purpose: to purify the hmp protein and verify its mass, also demonstrating GST-tagging for characterization of future proteins.
Protocol:
Two ligations were mixed and left to react at room temperature overnight:
3/1 L1
1 µL
10X T4 buffer
2 µL
10/11 D5
6 µL
2/20 D2
1 µL
T4 DNA ligase
10 µL
3/1 L2
1 µL
10X T4 buffer
4.6 µL
8/18 D1
3.4 µL
2/20 D3
1 µL
T4 DNA ligase
10 µL
(repeat of 2/20 L1 with longer incubation and more homogeneous T4 buffer)
 
Stop: 9:30 pm
 
Products:
 
Label
Source
Description
3/1 L1
10/11 D5, 2/20 D2
hmp in pGEX4T-1
3/1 L2
8/18 D1, 2/20 D3
part 1 assembled in pSBIC3
 
Next:
Transform 3/1 L1 (onto amp), purify hmp using GST columns, run a SDS-PAGE gel of GST-tagged hmp, cleaved, uncleaved, or both. Amplify LeupA and LeupB from all three 2/25 extractions. Attempt to amplify 3/1 L2 with VR & VF2, also attempting a Q5 control with these primers/TA
27 February 2016
Kent Gorday
Start: 8:45 am
 
Gibson assembly troubleshooting gel
Purpose: to diagnose Gibson assembly/HiFi master mix failures
Protocol:
A 1% agarose gel was run for 45 min at 130 V:
2/27 Gel #1
4
5 µL 2-log purple DNA ladder
5
10 µL 2/20 HF1
 
Notes:
only the ratio between gblocks is significant because of troubles while loading gel
Stop: 10:00 am
 
Results:
Image J and the ladder were used to estimate band mass and thus moles present:
Band (bp)
Mass (ng)
Amount (fmol)
850
4.008
7.630
1000
4.928
7.974
1150
4.436
4.835
 
Next:
See Feb. 25 and Feb.20 hmp work.
25 February 2016
Kent Gorday
Start: 5:30 pm
 
S. roseus DNA extractions
Purpose: To obtain chromosomal DNA for amplification from S. roseus and verify amplification of ocimene synthase.
Protocol:
Three glycerol stocks were prepared and left in rm. 206 freezer.  2 mL of each broth culture was used with Machery-Nagel NucleoSpin Microbial DNA kit to produce 100 μL of each chromosomal DNA solution. No proteinase K was used, and products were nanodropped to determine concentration.  A 1% agarose gel was run for 40 min at 130 V:
2/25 Gel #1
2
5μL 2-log purple DNA ladder
3
5μL 2/23 A1
 
Stop: 9:10 pm
 
Results:
 
Sample
ng/μL
260/280
260/230
2/25 1
56.06
1.95
1.73
2/25 2
33.21
1.86
1.04
2/25 3
75.80
1.92
1.74
Products:
 
Label
Source
Description
2/25 S1
2/21 BC1
S. roseus ISP-5076 glycerol stock
2/25 S2
2/21 BC2
S. roseus B-1320 glycerol stock
2/25 S3
2/21 BC3
S. roseus B-3062 glycerol stock
2/25 1
2/21 BC1
S. roseus ISP-5076 genomic DNA
2/25 2
2/21 BC2
S. roseus B-1320 genomic DNA
2/25 3
2/21 BC3
S. roseus B-3062 genomic DNA
 
Next:
Amplify LeupA and LeupB from genomic DNA, see Feb. 20 hmp work.
23 February 2016
Kent Gorday
Start: 7:10 pm
 
Ocimene synthase amplification
Purpose: amplify part 3 gblock (ocimene synthase) for cloning into pSB1C3
Protocol:
One PCR was reacted in the thermocycler:
2/23 A1
9 μL
MilliQ water
12.5 μL
2X Q5 Master Mix
1.25 μL
10 μM VF2
1.25 μL
10 μM OciR
1 μL
Part 3 gblock (small drop diluted)
25 μL
 
Temperature (°C)
Time (s)
98
30
98c
7
65c
20
72c
53
72
120
4
hold
cycle X32
 
Stop: 7:40 pm
 
Products:
 
Label
Source
Description
2/23 A1
Part 3 gblock
ocimene synthase amplified by Q5 using VF2 & OciR
 
Next:
See Feb. 20 and Feb. 21, also run a gel of 2/23 A1 (recommended 1% 130 V for 40 min, load 5μL)
23 February 2016
Kent Gorday
Start: 10:00 am
 
Further isolation of Streptomyces roseus
Purpose: To obtain chromosomal DNA from S. roseus strains to clone leupeptin genes from.
Protocol:
Three broth cultures and three isolation plates were inoculated from previous plates.
Stop: 10:45 am
 
Products:
 
Label
Source
Description
2/23 BC1
2/21 I1
S. roseus ISP-5076 on ISP4
2/23 BC2
2/21 I2
S. roseus B-1320 on ISP4
2/23 BC3
2/21 I3
S. roseus B-3062 on ISP4
2/23 I1
2/21 I1
S. roseus ISP-5076 on ISP4
2/23 I2
2/21 I2
S. roseus B-1320 on ISP4
2/23 I3
2/21 I3
S. roseus B-3062 on ISP4
 
Next:
See Feb. 20 and Feb. 21.
21 February 2016
Kent Gorday
Start: 11:00 am
 
Cultures of Streptomyces roseus
Purpose: To obtain chromosomal DNA from S. roseus strains to clone leupeptin genes from.
Protocol:
Three broth cultures and three isolation plates were inoculated from NRRL ampuoles.
Notes:
BCs incubated in benchtop incubator set to 28°C, actually 25°C. Plates in 30°C stationary.
Stop: 11:40 am
 
Products:
 
Label
Description
2/21 BC1
S. roseus ISP-5076 in ISP1 (green cap)
2/21 BC2
S. roseus B-1320 in ISP1 (yellow)
2/21 BC3
S. roseus B-3062 in ISP1 (red)
2/21 I1
S. roseus ISP-5076 on ISP4
2/21 I2
S. roseus B-1320 on ISP4
2/21 I3
S. roseus B-3062 on ISP4
 
Next:
See Feb. 20, also ensure pure cultures, create a stock of each strain, use Dr. Westenberg's chromosomal DNA extraction kit.
20 February 2016
Kent Gorday
Start: 8:00 am
 
Assembly of Part 1, Ocimene Synthase, and GST-tagged hmp
Purpose: To assemble part 1 and ocimene synthase into pSB1C3 for registry submission, transfer hmp into pGEX4T-1 for GST-tag purification, and culture Streptomyces roseus to obtain its chromosomal DNA.
Protocol:
Three PCRs were mixed as below and incubated in the thermocycler with given programs:
2/20 A1
3 µL
5X Q5 Reaction Buffer
6.8 µL
MilliQ water
0.75 µL
10 µM VF2
0.75 µL
1/10X 100 µM OciS R Primer
1.2 µL
2.5 mM dNTPs
1 µL
Part 3 gblock (tiny drop diluted)
1.5 µL
1/10X Q5 Polymerase
15 µL
 
Temperature (°C)
Time (s)
98
30
98c
7
66c
20
72c
45
72
120
4
hold
cycle X32
 
2/20 A2
12.5 µL
2X Taq Master Mix
10.5 µL
MilliQ water
0.5 µL
10 µM hmp F Primer 7/2
0.5 µL
10 µM hmp R Primer 7/2
1 µL
1/800X 2/11 KG
25 µL
note: 2/11 KG initially 338.5 ng/µL
Temperature (°C)
Time (s)
95
30
95c
23
60c
43
68c
95
68
300
4
hold
cycle X32
 
2/20 A3
12.5 µL
2X Taq Master Mix
10.5 µL
MilliQ water
0.5 µL
10 µM VF2
0.5 µL
1/10X 100 µM OciS R Primer
1 µL
Part 3 gblock (tiny drop diluted)
25 µL
 
Temperature (°C)
Time (s)
95
30
95c
23
51c
43
68c
120
68
300
4
hold
cycle X32
One HiFi assembly reaction was performed with only the four insert fragments of Part 1:
2/20 HF1
10 µL
2X HiFi Master Mix
2.1 µL
Part 1-1 gblock
2.5 µL
Part 1-2 gblock
2.5 µL
Part 1-3 gblock
2.9 µL
Part 1-4 gblock
20 µL
note: roughly 0.4045 pmol of each 50 °C for 60 minutes in thermocycler  A 1% agarose gel was run at 130 V for 45 minutes:
Gel 2/20 #1
2
5 µL 2-log purple DNA ladder
4
5 µL 2/20 A1
6
5 µL 2/20 A2
note: 2/20 A2 yielded a strong band near ~1.3kbp expected product, 2/20 A1 gave no coherent product  Three restriction digests were mixed and incubated at 37 °C for 60 minutes before a 20 minute heat kill at 80 °C in thermocycler:
2/20 D1
2.5 µL
10X Tango Buffer
20 µL
MilliQ water
1 µL
EcoRI
1 µL
PstI
0.5 µL
2/20 A3
25 µL
2/20 D2
2.5 µL
10X Tango Buffer
20 µL
MilliQ water
1 µL
EcoRI
1 µL
XhoI
0.5 µL
2/20 A2
25 µL
2/20 D3
2.5 µL
10X Tango Buffer
10.5 µL
MilliQ water
1 µL
EcoRI
1 µL
PstI
10 µL
2/20 HF1
25 µL
note: 2/20 D3 should be roughly 20 ng/µL, 10/11 D5 is ~117.3 ng/µL, 8/18 D1 is 20.5 ng/µL  Two ligations were mixed and incubated at room temperature for 40 minutes:
2/20 L1
1 µL
10X T4 Ligase Buffer
4.6 µL
8/18 D1
3.4 µL
2/20 D3
1 µL
T4 DNA Ligase
10 µL
2/20 L2
1 µL
10X T4 Ligase Buffer
5 µL
8/18 D1
3 µL
2/20 D1
1 µL
T4 DNA Ligase
10 µL
note: 2/20 L1 should be roughly 2 mol pSB1C3 : 1 mol Part 1 and 2.12 ng/µL  Two additional PCRs were performed with the same program below:
Temperature (°C)
Time (s)
95
30
95c
23
51c
43
68c
250
68
300
4
hold
cycle X32  
2/20 A4
12.5 µL
2X Taq Master Mix
10.5 µL
MilliQ water
0.5 µL
10 µM VF2
0.5 µL
10 µM VR
1 µL
1/30X 2/20 L1
25 µL
2/20 A5
12.5 µL
2X Taq Master Mix
10.5 µL
MilliQ water
0.5 µL
10 µM VF2
0.5 µL
10 µM VR
1 µL
1/30X 2/20 L2
25 µL
Another 1% agarose gel was run at 130 V for 45 minutes:
Gel 2/20 #2
2
5 µL 2-log purple DNA ladder
3
5 µL 10/11 D5
4
5 µL 2/20 A3
5
10 µL 2/20 D1
6
10 µL 2/20 D2
7
5 µL 2/20 A4
8
5 µL 2/20 A5
note: good pGEX4T-1 band in 3, nothing notable in 4-5 good hmp band in 6, smears in 7-8  Additionally, two media were prepared and autoclaved for culturing Streptomyces roseus:
Trace Salts Solution
0.1 g/100 mL
FeSO4 ⋅ 7H2O
0.1 g/100 mL
MnCl2 ⋅ 4H2O
0.1 g/100 mL
ZnSO4 ⋅ 7H2O
Tryptone - Yeast Extract (ISP1)
5 g/L
Tryptone
3 g/L
Yeast Extract
pH 7.0 to 7.2
Synthetic Salts - Starch Medium (ISP4/NRRL9)
10 g/L
Soluble Starch (corn starch)
1 g/L
K2HPO4
1 g/L
MgSO4 ⋅ 7H2O
1 g/L
NaCl
2 g/L
(NH4)2SO4
2 g/L
CaCO3
1 mL/L
Trace Salts Solution
15 g/L
Agar
Stop: 3:30 am
 
Results:
 
#1
#2
Products:
 
Label
Source
Description
2/20 A1
Part 3 gblock
ocimene synthase amplified by Q5 with VF2&ociR
2/20 A2
2/11 KG
sequenced hmp with EcoRI & XhoI added
2/20 A3
Part 3 gblock
ocimene synthase amplified by Taq
2/20 A4
2/20 L1
verification of Part 1 in pSB1C3 with VF2&VR
2/20 A5
2/20 L2
verification of OciS in pSB1C3 with VF2&VR
2/20 HF1
Part 1-1, 1-2, 1-3, 1-4 gblocks
Part 1 insert assembled
2/20 D1
2/20 A3
OciS insert digested E&P
2/20 D2
2/20 A2
hmp with sites added digested EcoRI & XhoI
2/20 D3
2/20 HF1
Part 1 insert digested E&P
2/20 L1
8/18 D1, 2/20 D3
Part 1 in pSB1C3
2/20 L2
8/18 D1, 2/20 D1
OciS in pSB1C3
 
Next:
Ligate 10/11 D5 and 2/20 D2 together, transform and harvest GST-hmp fusion protein for column purification, claving, and SDS-PAGE. Retry amplification of OciS using 2X Q5 Master Mix, and consider ordering primers for the insert of Part 1 to diagnose HiFis. Inoculate cultures of Streptomyces roseus strains and obtain chromosomal DNA.
