<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Missouri_Rolla_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Missouri Rolla/Project</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content_wrapper"><DIV id="menu"><A href="https://2016.igem.org/Team:Missouri_Rolla">Home</A><A href="https://2016.igem.org/Team:Missouri_Rolla/Project">Project</A><A href="https://2016.igem.org/Team:Missouri_Rolla/Engagement">Human Practices</A><A href="https://2016.igem.org/Team:Missouri_Rolla/Notebook">Notebook</A><A href="https://2016.igem.org/Team:Missouri_Rolla/Team">Team</A><A href="https://2016.igem.org/Team:Missouri_Rolla/Safety">Safety</A><A href="https://2016.igem.org/Team:Missouri_Rolla/Attributions">Attributions</A></DIV><DIV class="mstigem"><H1>Project</H1><P>Since its first appearance in New York in 2007, White Nose Syndrome (WNS) has quickly spread across the United States and Canada. By 2012, the US Fish and Wildlife Service estimated at least 6 million bats had already succumbed to the disease, while state and federal agencies have scrambled to close off caves and protect unaffected bat populations. North American bats hibernate over the winter for upwards of four months. During this torpor, they awaken every few weeks, but infrequently enough to conserve energy in the form of fat. Current understanding of WNS suggests the psychrophilic fungus, <I>Pseudogymnoascus destructans</I>, kills bats in two ways as their body temperature drops during hibernation. Infected bats arouse from torpor more frequently and for longer periods, using more energy without food available to replenish it. Additionally, damage to the thin wing membranes hinders flight while inducing a variety of physiological changes and inflammation.</P></DIV><DIV class="mstigem"><P>For an interactive map of the spread of White-Nose Syndrome, <A href="http://www.sciencebase.gov/gisviewer/wns/">visit the USGS tool</A>.</P><P>There is hope for infected bats, despite mortality rates in excess of 90% for some cave populations. Bats with the fungus that survive until arousal in the spring have demonstrated an ability to recover from the disease. It is likely that bats with a mild-enough infection to subsist until the end of hibernation can mount a stronger immune response as energy use is no longer suppressed.</P><P>Traditionally, efforts to kill the fungus using antifungal agents would be the intuitive response to a goal of reducing the severity of disease. A developing understanding of microbes’ roles in ecosystems and the lack of knowledge about caves, however, mean that methods which indiscriminately kill fungi could severely impact cave ecology. Antifungals could kill fungi which compete with <I>P. destructans</I> on the bats’ skin, or disrupt cave function. Handling and treating bats as they are affected during hibernation is also counterproductive, as disturbing bats wastes their valuable energy reserves. We instead began our plan for defending bats from White Nose Syndrome with the volatile compound ocimene. Potentially effective without contact with the bats, ocimene has demonstrated an ability to delay the growth of fungi, thus giving bats a better chance at fighting the infection when they arouse.</P></DIV><DIV class="mstigem"><P>We intended to produce (E)-beta-ocimene, a monoterpene, using two plasmids encoding potential mevalonate pathways. These pathways were constructed from the codon-optimized genes of <I>Enterococcus faecalis</I>, <I>Methanosarcina mazei</I>, <I>Saccharomyces cerevisiae</I>, and <I>Escherichia coli</I>, with one designed for optimum flux and the other providing extensibility for synthesis of other terpenes. Finally, the codon-optimized ocimene synthase from <I>Arabidopsis thaliana</I> would be assembled in an additional plasmid to catalyze the final step of ocimene synthesis. The mevalonate pathways would be compared, while in-vitro testing of ocimene synthase could be conducted after purifying with GST-tagging. Both ocimene itself and <I>E. coli</I> with our pathway would be tested by the Crow Lab at Georgia State University for inhibition of <I>P. destructans</I>.</P><P>Unfortunately, we were unable to assemble these plasmids for submission to the parts registry. The mevalonate pathways were designed for Gibson assembly and ordered as several Integrated DNA Technologies gblocks, but we were unable to obtain the assembled product in a backbone. We suspect this is a result of either overlaps too long for the exonuclease to process, secondary structure in overlap regions, or an issue with our method of linearizing the backbone. Our issue was exacerbated when no PCR primers were ordered for the amplification of individual gblocks, and stocks eventually ran out. Additionally, the single ocimene synthase gblock was designed for direct RFC[10] digestion and ligation, with flanking VF2 and VR sites for amplification using existing primers. After discovering the reverse complement of the VR site had instead been designed and ordering the corresponding primer, the gblock still failed to amplify under a variety of conditions.</P></DIV><DIV class="mstigem"><P>We also investigated potential methods to prevent the fungus from damaging the bats' skin without direct elimination. Analysis of the <I>P. destructans</I> secretome found a serine endopeptidase, destructin-1, responsible for much of the fungus' ability to degrade collagen in bats’ skin. Its inhibition resulted in a 77% reduction in collagen degradation at 54h/20°C, and it is believed to create ends for exopeptidases to act on. Unfortunately, the biosynthesis of most protease inhibitors remains a mystery, but a recent preprint purports to have found a gene cluster in <I>Streptomyces roseus</I> responsible for the synthesis of leupeptin, commonly used in protease inhibitor cocktails (and expensive). Named <I>leupA-D</I>, we believe that <I>leupA</I> and <I>leupB</I> are sufficient for leupeptin production from acetylleucine, leucine, and arginine. Mutation of <I>leupA</I> demonstrated that it is necessary for leupeptin synthesis, and a vector containing the region around <I>A</I> and <I>B</I> resulted in production of leupeptin in E. coli. The domains of <I>leupA</I> and <I>B</I> have homology to proteins with condensation actions similar to those proposed for the synthesis of leupeptin, including arginine and leucine. We ruled out most candidates for leucine acetyltransferase that could have been included in the 70 kbp vector with BLAST results to known genes except for a 18.84 kDa hypothetical protein, which differs significantly from the 27 kDa observed previously and occurs upstream rather than downstream as Barona-Gómez suggested two additional genes may be. From this, the presence of the acetyltransferase in leupeptin non-producing mutants of <I>S. roseus</I>, and the demonstrated ability of <I>E. coli</I> to convert acetylleucine to leucine, it is likely that some amount of acetylleucine is present naturally in <I>E. Coli</I> and sufficient for leupeptin production. <I>LeupA</I> has a predicted product similar in weight to the predicted 260 kDa for leupeptin synthase. Leupeptin acid reductase was noted to always elute with leupeptin acid synthetase, and had an estimated weight of 320 kDa, almost identical to the combined weights of LeupA and B. Thus we believe <I>leupA</I> and <I>B</I> together catalyze every step in leupeptin synthesis excluding acetylleucine formation, which might occur naturally in <I>E. coli</I>. Finally, natural leupeptin producers degrade leupeptin in the stationary phase, meaning we could see much higher yields with heterologous expression.</P><P>We obtained three <I>S. roseus</I> strains and attempted to amplify <I>leupA</I> and <I>leupB</I> from the genome using primers to add the entire RFC[10] prefix and suffix, designed using the unassembled genome sequence of strain ATCC 31245. PCR did not appear to amplify the desired products under a variety of conditions, with the long additions making the primers too long for use with the template, which has a GC content of over 70%. New primers with more favorable annealing temperatures were designed to amplify the region around each gene, and seemed to be successful. Attempts to amplify from these PCR products with the original primers to add restriction sites failed, leaving no time to order new primers and perform site-directed mutagenesis to remove a potential <I>PstI</I> site from <I>leupA</I>.</P><P>Design files for <A href="https://2016.igem.org/File:T--Missouri_Rolla--LeupA.gb">amplification of <I>leupA</I></A>, <A href="https://2016.igem.org/File:T--Missouri_Rolla--LeupB.gb">amplification of <I>leupB</I></A>, <A href="https://2016.igem.org/File:T--Missouri_Rolla--Part1.gb">ocimene part 1</A>, <A href="https://2016.igem.org/File:T--Missouri_Rolla--Part2_1.gb">ocimene part 2-1</A>, <A href="https://2016.igem.org/File:T--Missouri_Rolla--Part2_2.gb">ocimene part 2-2</A>, and <A href="https://2016.igem.org/File:T--Missouri_Rolla--Part3.gb">ocimene part 3</A>.</P><P>Find more information about the references we used in the design of our project on our <A href="https://2016.igem.org/Team:Missouri_Rolla/Attributions">attributions page</A>.</P><P id="credit">WNS map credit: U.S. Fish &amp; Wildlife ServiceBat photo credit: Wayne National ForestLab photo credit: Bob Phelan, SDELC</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>