Team:UNITN-Trento/Measurement
UNITN-TRENTO iGEM 2015
Home
Team
Project
Overview
Experiments / Protocols
Results
Proteorhodopsin (PR)
pncB
Retinal Production (BLH)
Microbial Fuel Cell (MFC)
Notebook
Modeling
Prototype
Interlab
Introduction
Experimental Design
Experiments and Protocols
Conclusions
Parts
Judging
More...
Human Practices
Attributions
Safety
Collaborations
Gallery
InterLab Measurement Study
What happens when GFP meets different promoters? Are they all the same? Here is what we found out!
Introduction & Achievements
Experimental Design
Experiments& Protocols
Final Discussion & Results
The Interlab Measurement Study
The characterization of standard parts has always been one of the main concerns in Synthetic Biology. For this very same reason, iGEM teams from all around the World were suggested to take part in the biggest measurement study ever conducted and the 2015 UNITN-Trento iGEM team answered the call. The goal of this Second International Measurement Interlab Study is to assemble three different devices, each one containing a promoter with a screening plasmid intermediate and collect as many fluorescence data as possible. iGEM teams are free to use any technique to measure their devices as long the obtained data are solid and reproducible.
Clear Data and Protocols
We listed in the ”InterLab Study” page:
All devices measured for this study
All protocols developed and adopted
Sequencing data for all measurements devices
3 BioBrick devices
We used the three BioBrick devices listed in the ”Required Devices” section.
5 techniques
We used in-vivo and in-vitro techniques for measuring RNA and protein levels:
Plate Reader
Spectrofluorometer
Flow Cytometer
RT-qPCR
Cell-Free Extract
Statistical Reliability
We have three biological replicates for each measurement, with positive and negative controls
Worksheet and Protocol
We completed the InterLab Worksheet and the InterLab Protocol
Extra Credit Assignment
We did technical replicates for each sample
Experimental Design
We used the three mandatory devices for the measurement study:
Device 1: BBa_J23101 + BBa_I13504 in pSB1C3
Device 2: BBa_J23106 + BBa_I13504 in pSB1C3
Device 3: BBa_J23117 + BBa_I13504 in pSB1C3
Negative control: BBa_R0040 in pSB1C3
Positive control: BBa_I20270 in pSB1C3.
The measurement devices were prepared by amplifying the reporter ( BBa_I20270 ) by PCR. The amplified insert was then cut with XbaI and PstI and ligated into the plasmid containing the promoter previously cut with SpEI and PstI. All the devices were confirmed by restriction digestion as well as DNA sequencing.
In-vivo Measurements
The confirmed devices were then transformed in different bacterial strains of E. coli: NEB10β, NEB Express, and JM109. Each measurement was taken at the same optical density to allow a more precise comparison of the data. For each device we have 3 biological and 3 technical measurements for each used technique. We measured in vivo fluorescence emission in different ways using Tecan Infinite 200 PRO plate reader, Varian Cary Eclipse spectrofluorometer, and BD FACSCanto flow cytometer.
In-vitro Measurements
We also focused on transcription since the characterization is about promoters. To do so we performed RT-qPCR using a BioRad CFX96 Touch™ Real-Time PCR Detection System. Additionally, we performed an in vitro characterization study, by measuring the fluorescence intensities of each device with a Cell-Free E. coli S30 Extract System with a Qiagen Rotor-Gene Q as the spectrofluorometer.
Experiments & Protocols
Extraction from the Registry
All the parts needed for the InterLab Study were extracted from the 2015 iGEM Registry Distribution Kit.
Part ID
Description
Plasmid Backbone
2015 Registry Location
BBa_K823005
Anderson promoter J23101
pSB1C3
Plate 1; 20K
BBa_K823008
Anderson promoter J23106
pSB1C3
Plate 1; 22A
BBa_K823013
Anderson promoter J23117
pSB1C3
Plate 1; 22K
BBa_I12504
RBS + GFP + 2 terminators
pSB1A2
Plate 4; 21J
BBa_R0040
TetR sequence
pSB1C3
Plate 2; 6F
BBa_I20270
Promoter MeasKit
pSB1C3
Plate 3; 8P
Polymerase Chain Reaction (PCR)
Electrophoresis gel of the reporter gene The bright band corresponds to the part BBa_I13504 . The part was amplified by Phusion PCR and digested with XbaI and PstI. We can see the expected band at around 875 bp.
The reporter gene (i.e. GFPmut3b from part BBa_I13504 ) was amplified using the Phusion polymerase from New England Biolabs and primers matching the prefix and suffix.
We used 50 ng of template with an annealing temperature of 59 °C and an extension time of 90 seconds. The PCR was confirmed by electrophoresis and subsequently purified with NucleoSpin Gel and PCR Clean-Up Kit from Macherey-Nigel. 125 ng of purified PCR were digested with 1 μl of XbaI and 1 μl of PstI overnight at 37°C. The following morning the sample was treated with 1 μl of DpnI for 2 hour at 37°C and the enzymes were deactivated for 20 min at 80 °C.
Restriction Digestion of plasmids containing the promoter
Each promoter containing plasmid was digested with 1 μl of SpeI and 1 μl of PstI at 37°C overnight. The day after 1 μl of phosphatase (CIP from New England Biolabs) was added for 2 hours at 37°C. The enzymes were then heat deactivated.
The digestion reactions were assembled in this way:
PCR Products
Plasmids
Template
3000 ng
2000 ng
Enzyme 1
2 μl
1.5 μl
Enzyme 2
2 μl
1.5 μl
Buffer (Stock 10X)
5 μl
5 μl
BSA (Stock 10X)
5 μl
5 μl
Water
Up to 50 μl
Up to 50 μl
Ligation
Plasmid:
Insert = 1:3
Control
Vector
100 ng
100 ng
Insert
125 ng
-
10X Buffer
2 μl
2 μl
Buffer (Stock 10X)
5 μl
5 μl
T4 - DNA Ligase
2 μl
2 μl
Water
Up to 20 μl
Up to 20 μl
The ligation reactions were assembled and incubated at room temperature for 1 hour according to the table beside. Subsequently 10 μl of the ligation mixture were transformed and plated into LB-agar plates with the proper antibiotic resistance. The confirmed devices were transfected in NEB10β, JM109, NEB Express.
Cells expressing the devices These three plates contains E. coli NEB Express transformed with J23101 (top left), J23106 (top right), and J23117 (bottom). Interestingly, promoters' strength are detectable by eye, depending on the colonies' color.
Cells expressing the devices. These three plates contains E. coli NEB Express transformed with J23101 (top left), J23106 (top right), and J23117 (bottom).
Parts Confirmation
Correct clones were screened by restriction digestion and confirmed by sequencing:
Cell pellets at the UV-trans illuminator Pellets of overnight NEB10β cells cultures expressing the assembled devices. Pellet's colors confirmed the promoter strength,  (from left to right) the strongest promoter J23101, the medium strength promoter J23106, the weak promoter J23117, the positive control I20270, the negative control R0040.
Electrophoresis gel of all devices We digested three colonies for each device with XbaI and PstI. The correct assembly is confirmed by the band at 910 pb.
BBa_J23101 + BBa_I13504
[Sequencing Data]
BBa_J23106 + BBa_I13504
[Sequencing Data]
BBa_J23117 + BBa_I13504
[Sequencing Data]
Glycerol stocks preparation and Sample Growth
A single colony was inoculated with a sterile pipette tip in a test tube with 10 ml of LB and antibiotic (1000:1 LB to antibiotic ratio) and placed in the thermoshaker (190 rpm, 37°C). When the culture got cloudy, 40 ml of LB+antibiotic were added to reach a final volume of 50 ml. The cells were grown until an OD600 of 0.5 and then centrifuged at 4100 rpm for 10 minutes at 4 °C. The supernatant was discarded and the cells were resuspend in 5 ml of LB + antibiotic + glycerol (20% v/v). The cells were kept on ice and were promptly aliquoted into 200 μl tubes and frozen at -80°C immediately. From this protocol we obtained a 10X concentrated glycerol stock for each sample.
The glycerol stock was thaw and added into 10 ml of LB with antibiotic, giving a  starting culture with an OD600 of 0.1. The sample were grown in a 50 mL conical  plastic tube in the termoshaker at 37°C and were grown until an OD600 0.7. At this  point 3 ml of the culture were transferred in a new tube, centrifuged it, and stored at -20°C, unless otherwise indicated.
Fluorescence readings: Tecan INFINITE 200 PRO Plate Reader
The cells were thawed and resuspended in 3 ml of PBS. An aliquot of 150 μl of each sample was placed into a White, Flat-bottomed, 96-well Costar Plate (code: 3917) and fluorescence intensities were taken with a Tecan Infinite 200 Pro Plate Reader (made in Switzerland). Excitation wavelength and emission wavelength were 395 nm and 509 nm, respectively. The gain was optimized at 70 V and kept constant for each sample. PBS was used as blank. To obtain technical replicates, fluorescence intensities were acquired for three aliquots of the same biological sample, keeping the same instrumental conditions. The raw data were adjusted for the blank value and the means across the replicates with their relative standard deviation were plotted.
Fluorescence readings: Cary Eclipse Fluorescence Spectrophotometer
The cells were thaw and resuspended in 3 ml of PBS. Each measurement was taken in a clear acrylic cuvette (REF 67.755) with a Cary Eclipse Fluorescence Spectrophotometer (made in USA). The blank was done with PBS. The excitation wavelenght used was 395 nm and the spectra were acquired between 420 nm to 650 nm. The gain was optimized at 775 V and kept the same throughout the measurement.
The fluorescence intensity of the maximum emission wavelenght (514 nm) was adjusted for the blank and the calculated means with relative standard deviations were plotted.
Fluorescence readings: BioRad CFX96 Touch Real-Time PCR Detection System
The cells were grown from glycerol stock until an OD600 of 0.7 was reached. Total  RNA was purified by using the Thermo Scientific GeneJET RNA Purification Kit, following the manufacturer`s instructions and subsequently genomic DNA was  removed from the total RNA by using the Thermo Scientific RapidOut DNA Removal Kit, following the manufacturer`s instructions. RNA levels were quantified using NanoDrop 1000 and reverse transcription of cDNA from the RNA template was  performed with Thermo Scientific RevertAid First Strand cDNA Synthesis Kit,  following the manufacturer`s instructions. qPCR reactions were performed with BioRad CFX96 Touch Real-Time PCR Detection System (made in USA) and  assembled as follows:
cDNA
5 ng
Primer Fw
180 ng
Primer Rv
180 ng
BioRad iQ SYBR® Green Supermix #1708880
Up to 10 μl
GFP Primer Fw
ATGCTTTGCGAGATACCCAG
GFP Primer Rv
TGTCTTGTAGTTCCCGTCATC
idnT Primer Fw
CTGCCGTTGCGCTGTTTATT
idnT Primer Rv
GATTTGCTCGATGGTGCGTC
Primers were designed for the reporter gene GFPmut3b and for the housekeeping gene idnT (D-gluconate transporter) as indicated above.
We then analyzed the raw data, calculating the relative fold expression of each GFP device compared to the housekeeping (ΔCt) and the related standard deviation:
Device
Relative Fold Expression (A.U.)
Standard Deviation
J23101
67,421.53
16,016.27
J23106
18,506.44
3,146.38
J23117
658.08
79.09
R0040
0.04
0.02
I20270
17,800.56
1,030.40
Fluorescence readings: BD FACSCanto Flow Cytometer
The cells were grown from glycerol stocks as described above. Differently from before when they reached the OD of 0.7 were not frozen, but were used immediately to measure fluorescence intensity. An aliquot of 5 μl of cells was diluted in 900 μl of PBS. The instrument used was a BD FACSCanto (made in USA) set with the following parameters:
FSC gain: 525 V
SSC gain: 403 V
FITC gain: 510 V
Flow rate: LOW
Total number of events in P2: 10000
Those parameters allowed the instrument to process 900/1500 events per second. We analyzed the raw data and plot the means of three replicates with relative standard deviations.
Fluorescence spectra from flow cytometer analysis of the NEB10β strain The merge shown above represents the devices J23101 (white), J23106 (yellow), J23117 (blue), and the negative control R0040 (orange). Data were analyzed using Cyflogic 1.2.
Fluorescence readings: E. coli S30 Extract System for DNA Circular
Miniprepped DNA was purified and extracted through phenol/chloroform extraction followed by ethanol precipitation. Reactions were set following Promega E. coli S30 Extract System Technical Bulletin and were performed using Qiagen Rotor-Gene Q (made in USA). Parameters for this specific experiment were set as follows:
Green channel gain: 0.67
Green channel Excitation: 365±20 nm
Green channel Emission: 510±5 nm
We analyzed the raw data and plot the means of three replicates over time:
Final Discussion
Our characterization confirmed the relative strength of the promoters
J23101 is the strongest promoter among the three, showing high expression of GFP in all three strains, regardless of the technique used. J23106 is the medium promoter and J23117 the weakest.
Ratios across promoters are kept the same
J23101/J23106 fluorescence ratios ranged from 2.0 to 4.5, depending on the strain and the technique. Differently from the other two promoters, J23117 showed a very low GFP production, as it was not detectable by eye or using the trans-illuminator and showed little fluorescence with the three techniques used.
Different techniques lead to the same results, with different sensitivities
The best way to perform a characterization is to use various techniques. Throughout our experiments we saw that each instrument has a specific sensitivity, which alters the output data. The FACS happened to be the most accurate among all, due to its extremely high intrinsic sensitivity. The plate reader also showed a good accuracy while the fluorometer was not able to detect the weakest promoter from the background noise. The use of the qPCR machine as a spectrofluorometer also gave positive results.
In vitro conditions mimic the in vivo reality
Comparing the results obtained from the cell-free extract to the others, we discovered that the promoters behave the same when working in vitro or in living bacteria. Since testing constructs in vitro is much faster than in vivo, our results suggest that it may be wise to first screen parts and/or genetic circuitry in vitro. Then the activity of a smaller subset could be confirmed with in vivo measurements.
Looking at promoters from a different angle
In vitro characterization by qPCR allows for the quantification of promoter strength by measuring RNA transcript levels, rather than by looking at the concentrations of something one step removed, i.e. protein. This approach gives a a more direct measure of promoter strength.
Promoter strength across strains and techniques Values presented are the average of three  technical replicates for each of the three biological samples (total of 9 replicates). Data normalized on the maximum expression value of each technique. Heatmap calculated with R Project.
Bacterial strain does matter
The three promoters behaved differently in the different bacterial strains used. The  bacterial strain which gave the highest fluorescence was NEB10β, which showed significantly increased expression of protein in all cases when compared with JM109 and NEB Express. Clearly this discordance among strains must to be due to their different genotypes. A different bacterial proteome (e.g. presence/lack of specific  proteases and/or chaperonins) may alter protein production, processing and folding, and thus fluorescence emission.
Strain influence on promoters efficiency Values presented are the average of three  technical replicates for each of the three biological samples (total of 9 replicates). Data normalized on the the maximum expression value of each technique. Heatmap calculated with R Project.
Institutions
University of Trento
CIBIO - Centre for Integrative Biology
iGEM HeadQuarter
Sponsors
Follow us
Facebook
Twitter
Design by Moreno Zolfo. Powered by MediaWiki and HTM5UP. This website is under the CC Attribution licence, Version 4
