<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UNITN-Trento_Results_pncB skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UNITN-Trento/Results/pncB</SPAN></H1><DIV id="bodyContent"><DIV id="super_wrap"><DIV class="no-sidebar"><DIV id="page-wrapper"><HEADER id="header"><H1 id="logo">UNITN-TRENTO <SPAN id="igem2015-spanned"><SPAN style="text-transform:lowercase">i</SPAN>GEM 2015</SPAN></H1><NAV id="nav"><UL><LI id="menulin_home"><A href="https://2015.igem.org/Team:UNITN-Trento">Home</A></LI><LI id="menulin_team"><A href="https://2015.igem.org/Team:UNITN-Trento/Team">Team</A></LI><LI id="menulin_project" class="submenu"><A href="#">Project</A></LI><LI id="menulin_project_description"><A href="https://2015.igem.org/Team:UNITN-Trento/Description">Overview</A></LI><LI id="menulin_project_experiments"><A href="https://2015.igem.org/Team:UNITN-Trento/Experiments">Experiments / Protocols</A></LI><LI id="menulin_project_results" class="submenu"><A href="https://2015.igem.org/Team:UNITN-Trento/Results">Results</A></LI><LI id="menulin__project_results_pr"><A href="https://2015.igem.org/Team:UNITN-Trento/Results/Proteorhodopsin">Proteorhodopsin (PR)</A></LI><LI id="menulin__project_results_pncB"><A href="https://2015.igem.org/Team:UNITN-Trento/Results/pncB">pncB</A></LI><LI id="menulin__project_results_blh"><A href="https://2015.igem.org/Team:UNITN-Trento/Results/BLH">Retinal Production (BLH)</A></LI><LI id="menulin__project_results_mfc"><A href="https://2015.igem.org/Team:UNITN-Trento/Results/MFC">Microbial Fuel Cell (MFC)</A></LI></UL><LI id="menulin_project_notebook"><A href="https://2015.igem.org/Team:UNITN-Trento/Notebook">Notebook</A></LI><LI id="menulin_project_modeling"><A href="https://2015.igem.org/Team:UNITN-Trento/Modeling">Modeling</A></LI><LI id="menulin_project_prototype"><A href="https://2015.igem.org/Team:UNITN-Trento/Design">Prototype</A></LI><LI id="menulin_interlab" class="submenu"><A href="https://2015.igem.org/Team:UNITN-Trento/Measurement">Interlab</A></LI><LI><A href="https://2015.igem.org/Team:UNITN-Trento/Measurement#interlab_intro" onclick="javascript:checkAndScrollTo('interlab_intro');">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:UNITN-Trento/Measurement#interlab_design" onclick="javascript:checkAndScrollTo('interlab_design');">Experimental Design</A></LI><LI><A href="https://2015.igem.org/Team:UNITN-Trento/Measurement#interlab_protocols" onclick="javascript:checkAndScrollTo('interlab_protocols');">Experiments and Protocols</A></LI><LI><A href="https://2015.igem.org/Team:UNITN-Trento/Measurement#interlab_final" onclick="javascript:checkAndScrollTo('interlab_final');">Conclusions</A></LI><LI id="menulin_parts"><A href="https://2015.igem.org/Team:UNITN-Trento/Parts">Parts</A></LI><LI id="menulin_judging"><A href="https://2015.igem.org/Team:UNITN-Trento/Judging">Judging</A></LI><LI class="submenu" id="menulin_more"><A href="#">More...</A></LI><LI id="menulin_more_human"><A href="https://2015.igem.org/Team:UNITN-Trento/Practices">Human Practices</A></LI><LI id="menulin_more_attributions"><A href="https://2015.igem.org/Team:UNITN-Trento/Attributions">Attributions</A></LI><LI id="menulin_more_safety"><A href="https://2015.igem.org/Team:UNITN-Trento/Safety">Safety</A></LI><LI id="menulin_more_collaborations"><A href="https://2015.igem.org/Team:UNITN-Trento/Collaborations">Collaborations</A></LI><LI id="menulin_more_gallery"><A href="https://2015.igem.org/Team:UNITN-Trento/Gallery">Gallery</A></LI></NAV></HEADER><SECTION id="cta" class="cta-pncb"><HEADER><H2><STRONG>PncB:</STRONG><SPAN style="font-size:0.8em; text-transform:lowercase; font-weight:300">a  Nicotinic acid -&gt; NAD producer</SPAN></H2></HEADER><UL class="buttons"><LI><P class="button small bstyle3" style="line-height:2em;" onclick="javascript:scrollToID('pncb_intro')"><SPAN class="primary">Introduction</SPAN></P></LI><LI><P class="button small bstyle4" style="line-height:2em;" onclick="javascript:scrollToID('pncb_char')"><SPAN class="primary">Characterization</SPAN></P></LI><LI><P class="button small bstyle5" style="line-height:2em;" onclick="javascript:scrollToID('pncb_conclusion')"><SPAN class="primary">Conclusions</SPAN></P></LI></UL></SECTION><ARTICLE id="main"><SECTION class="wrapper style4 container" style="margin-top:1em;"><DIV class="content"><HEADER><H3 class="wow fadeInDown">Pnc<SPAN class="bigletter">B</SPAN>: nicotinic acid phosphorbosyl-transferase</H3></HEADER><DIV class="row"><DIV class="12u 12u(narrower)  wow animated bounceInLeft"><H4 class="header4 lateral-icon wow animated fadeInDown      delay05"><SPAN>Increasing the levels of NADH</SPAN></H4><P style="clear:both;">Our goal was to boost electron production by increasing the concentration of electron carriers (i.e. NADH). To achieve this goal we decided to engineer <SPAN class="bacterium">E. coli</SPAN> with an enzyme that would provide more intracellular NAD<SUP>+</SUP>, and thus NADH<SUP><A class="sourced" onclick="javascript:scrollAndHighlight('refs_1')" href="#refs_1">[1]</A></SUP>.</P><P>PncB catalyzes one of the rate-limiting step in the NAD<SUP>+</SUP> synthesis pathway. This gene is naturally found in <SPAN class="bacterium">E. coli</SPAN> and encodes for the enzyme <SPAN class="i_enph">NAPRTase</SPAN> (nicotinic acid phosphorbosyl- transferase) that catalyzes the formation of nicotinate mono-nucleotide, a direct precursor of NAD<SUP>+</SUP> , from NA (nicotinic acid)
									<SUP><A class="sourced" onclick="javascript:scrollAndHighlight('refs_2')" href="#refs_2">[2]</A></SUP><SUP><A class="sourced" onclick="javascript:scrollAndHighlight('refs_3')" href="#refs_3">[3]</A></SUP><SUP><A class="sourced" onclick="javascript:scrollAndHighlight('refs_4')" href="#refs_4">[4]</A></SUP>.</P></DIV></DIV><DIV class="row"><DIV class="12u 12u(narrower)"><DIV class="captionbox" style="width:85%; max-width:950px;"><P class="image_caption"><SPAN>Biochemical pathway of NAD<SUP>+</SUP> synthesis</SPAN> PncB gene encodes the transcription of the NARPTase which catalyzes the formation of nicotinate mono-nucleotide from nicotinic acid.</P></DIV></DIV></DIV><DIV class="row"><DIV class="6u 12u(narrower)"><P class="image_caption"><SPAN>Unitn-Trento iGEM part 2015 (pncB)</SPAN> Upper: pncB without illegal site in pSB1C3, <A href="http://parts.igem.org/Part:BBa_K1604030" class="i_linker registry" target="_blank">BBa_K1604030</A>. Lower: pncB under the control of inducible promoter araC-pBAD, <A href="http://parts.igem.org/Part:BBa_K1604031" class="i_linker registry" target="_blank">BBa_K1604031</A></P></DIV><DIV class="6u 12u(narrower)"><P>Our device is controlled by an inducible arabinose promoter built by the <A class="i_linker registry" href="http://parts.igem.org/Part:BBa_K731201" target="_blank">Unitn iGEM team in 2012</A>. PncB was extracted by <SPAN class="i_enph italic">E. coli</SPAN> genome, the illegal site PstI was removed, and it was placed in pSB1C3 (<A href="http://parts.igem.org/Part:BBa_K1604030" class="i_linker registry" target="_blank">BBa_K1604030</A>). Subsequently it was placed under the araC-pBAD promoter (<A href="http://parts.igem.org/Part:BBa_K1604031" class="i_linker registry" target="_blank">BBa_K1604031</A>).</P></DIV></DIV></DIV></SECTION><SECTION class="wrapper style4 container"><DIV class="content"><HEADER><H3 class="wow fadeInDown">Characterization</H3></HEADER><DIV class="row"><DIV class="12u 12u(narrower)"><H4 class="header4 lateral-icon wow animated fadeInDown   delay05"><SPAN>PncB enhances NAD<SUP>+</SUP> production by 13 fold in anaerobic conditions</SPAN></H4></DIV></DIV><P>Our goal was to demonstrate that pncB <STRONG>increased intracellular levels of NAD<SUP>+</SUP>and thus NADH</STRONG>. We measured NAD<SUP>+</SUP> levels under different growth conditions modifying the culture media conditions and the presence or lack of oxygen, which is the condition of the anodic chamber of a MFC.</P><P>Colonies expressing pncB (<A href="http://parts.igem.org/Part:BBa_K1604031" class="i_linker registry" target="_blank">BBa_K1604031</A>) and the negative control (<A href="http://parts.igem.org/Part:BBa_K731201" class="i_linker registry" target="_blank">BBa_K731201</A>) were grown in LB broth. When an Optical Density (OD) of 0.6 (600nm) was reached, the cultures were centrifuged and resuspended in fresh medium and induced with 5mM of arabinose. The cultures after 5 hours of induction were transferred in sealed glass bottles with a rubber septum under an anaerobic work station. After additional 20 hours of induction we quantified the levels of NAD<SUP>+</SUP> by a colorimetric test that measures the levels of NAD<SUP>+</SUP> indirectly by quantifying the concentration of NAD<SUP>+</SUP> total (NAD<SUP>+</SUP> + NADH) and NADH only.</P><DIV class="row"><DIV class="6u 12u(narrower)"><P class="image_caption" style="clear:both;"><SPAN>NAD/NADH ratio: pncB and negative control in anaerobic condition</SPAN> NAD/NADH ratio of <A href="http://parts.igem.org/Part:BBa_K731201" class="i_linker registry" target="_blank">BBa_K731201</A> (araC-pBAD, blu) and cell expressing <A href="http://parts.igem.org/Part:BBa_K1604031" class="i_linker registry" target="_blank">BBa_K1604031</A> (pncB, green), in LB medium after 20 hours post-induction in anaerobic conditions. NAD<SUP>+</SUP> and NADH levels were quantified with Sigma NAD /NADH quantification kit (MAK037). The concentration of NAD<SUP>+</SUP> and NADH were calculated based on a standard curve.</P></DIV></DIV><DIV class="row"><DIV class="12u 12u(narrower)"><P>In anaerobic conditions we obtained our best result, however we were able to repeat this  measurement only one time due to technical difficulties with the anaerobic chamber. We also tested the performance of pncB in different conditions in LB, in a rich media  (Terrific Broth),  with the addition of Nicotinic Acid, the starting molecule used by pncB, in the presence of oxygen.  For all these samples we were able to use 3 biological replicates and with 3 technical  replicates.  Also under the conditions described above pncB enhanced NAD<SUP>+</SUP> intracellular levels.</P><DIV class="captionbox" style="max-width:860px; width:80%; margin:auto;"><P class="image_caption"><SPAN>NAD/NADH ratio between negative control BBa_K731201 and cells expressing   BBa_K1604031.</SPAN>The culture were grown in LB until an OD of 0.5 and induced with 5 mM arabinose   in the presence of oxygen. Before inducing the cells, the media were changed with: fresh LB, LB   supplemented with nicotinic acid or terrific broth. Panel A: <A href="http://parts.igem.org/Part:BBa_K731201" class="i_linker registry" target="_blank">BBa_K731201</A> in LB (blu),   <A href="http://parts.igem.org/Part:BBa_K1604031" class="i_linker registry" target="_blank">BBa_K1604031</A> in LB (green) and in LB with 10 uM nicotinic acid (yellow). Panel B: <A href="http://parts.igem.org/Part:BBa_K731201" class="i_linker registry" target="_blank">BBa_K731201</A>   (orange) and BBa_K1604031 (red) both in terrific broth.
									</P></DIV></DIV><P> We did not observe significant difference in LB versus terrific broth (2.5 and 2.7 fold, respectively), while nicotinic acid boost the NAD<SUP>+</SUP> levels by 3.7 fold respect to araC-pBAD and 1.5 compared to pncB in LB only.</P></DIV><DIV class="row"><DIV class="12u 12u(narrower)"><P style="clear:both" class="image_caption"><SPAN>Colorimetric Assay for NAD/NADH ratio in aerobic conditions </SPAN><STRONG>Left:</STRONG><STRONG>Lane B</STRONG>: samples 2-7 calibration curve (0, 20, 40, 60, 80, 100 pmole/well of NADH). <STRONG>Lane C</STRONG>: samples 2-9 NAD<SUP>+</SUP> total levels; <STRONG>Lane D</STRONG>: samples 2-9 NAD<SUP>+</SUP> total repeated with a 2 fold concentrated    sample; <STRONG>Lane E</STRONG>: NADH only; Lane F NADH only, repeated with a 2 fold concentrated sample. In lanes C-F the order of the samples is: 2 technical replicates of the negative control, and 2 technical    replicates of each of the 3 biological samples of  <A href="http://parts.igem.org/Part:BBa_K1604031" class="i_linker registry" target="_blank">BBa_K1604031</A>. The plate was read with a Tecan    Infinite M-200 pro instrument at 450 nm. The measurements were taken after 0.5, 1, 2, 3, 4 hours    to allow color development. The data shown are representative of the best measurement at 2 hours. <STRONG>Right:</STRONG> Standard curve with 0, 20, 40, 60, 80, 100 pmole/well of NADH</P></DIV><DIV class="row"><DIV class="12u 12u(narrower) wow animated bounceInRight"><H4 class="header4 lateral-icon wow animated fadeInDown delay05"><SPAN>PncB is not toxic if overexpressed in <SPAN style="font-style:italic">E.coli</SPAN></SPAN></H4><P style="clear:both;">
										NEB10β transformed with <A href="http://parts.igem.org/Part:BBa_K1604031" class="i_linker registry" target="_blank">BBa_K1604030</A> (araC-pBAD-pncB) or <A href="http://parts.igem.org/Part:BBa_K731201" class="i_linker registry" target="_blank">BBa_K731201</A>  (i.e. araC-pBAD) were grown up to an OD of 0.6, splitted in two tubes of 23 mL each and induced  with 5 mM of arabinose. Negative controls were not induced. The OD (600 nm) was measured  every 45 minutes for 5 hours.  All measurements were done for 3 different biological samples and 3 technical measures.
									</P><P>Although the growth rate is slightly decreased, due to the cell stress when expressing pncB, the data indicate that this enzyme does not have toxicity effect on the cells.</P><DIV class="captionbox" style="width:80%; max-width:700px;"><P class="image_caption"><SPAN>Growth rate of <A href="http://parts.igem.org/Part:BBa_K1604031" class="i_linker registry" target="_blank"> BBa_K1604031</A> (araC-pBAD - pncb) and <A href="http://parts.igem.org/Part:" class="i_linker registry" target="_blank">BBa_K731201</A> (i.e. araC-pBAD).</SPAN> Cells were grown up to an OD of 0.6 and splitted before induction with arabinose. <A href="http://parts.igem.org/Part:BBa_K1604031" class="i_linker registry" target="_blank"> BBa_K1604031</A> (Orange line) and <A href="http://parts.igem.org/Part:" class="i_linker registry" target="_blank">BBa_K731201</A> (green line) induced with 5 mM arabinose. <A href="http://parts.igem.org/Part:BBa_K1604031" class="i_linker registry" target="_blank"> BBa_K1604031</A> (yellow line) and <A href="http://parts.igem.org/Part:" class="i_linker registry" target="_blank">BBa_K731201</A> (blue line) not induced.</P></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION class="wrapper style4 container"><DIV class="content"><HEADER><H3 class="wow fadeInDown">To sum up...</H3></HEADER><DIV class="row"><DIV class="12u 12u(narrower)"><P>Overexpression of the gene pncB enhances significantly the intracellular level of NAD+. The best    data were obtained in anaerobic conditions where the increasing of NAD<SUP>+</SUP> was of 13 fold which are    the conditions to be used in a Microbial Fuel Cell (Anode chamber is in anaerobic condition).    When nicotinic acid was added in the medium there was a significant enhancement of NAD<SUP>+</SUP> levels (1.5 fold versus pncB and 3.7 fold versus negative control).</P><P>Also in Terrific broth there was an increase in the NAD<SUP>+</SUP> level (2.7 fold versus negative control in the same condition)     In the future it will be interesting to measures NAD<SUP>+</SUP> in anaerobic conditions with the presence of  high levels of Nicotinic Acid. We plan in the future to add a NAD<SUP>+</SUP> reducing enzyme and to give a  medium able to enhance the cell metabolism to further increase NADH intracellular levels.</P></DIV><DIV class="row"><DIV class="4u 12u(narrower) rotate-box-superhover  wow animated fadeIn"><H4 class="header4 wow animated flipInX" style="text-align:center;">New Part!</H4><P>We successfully developed and submitted a new functional part: <A href="http://parts.igem.org/Part:BBa_K1604031" class="i_linker registry" target="_blank">BBa_K1604031</A></P></DIV><DIV class="4u 12u(narrower) rotate-box-superhover  wow animated fadeIn"><H4 class="header4 wow animated flipInX" style="text-align:center;">More NAD<SUP>+</SUP> produced</H4><P>NAD<SUP>+</SUP> production was enhanced in our cells by overexpressing the pncB gene, thanks to our novel part.</P></DIV><DIV class="4u 12u(narrower) rotate-box-superhover  wow animated fadeIn"><H4 class="header4 wow animated flipInX" style="text-align:center;">Towards the pMFC</H4><P>
											 Bacteria equipped with pncB produce more electrons and are thus ideal for our Microbial Fuel Cell environment.Check out our <A href="https://2015.igem.org/Team:UNITN-Trento/Results/MFC" class="i_linker">Solar pMFC</A> results to know more!
										</P></DIV></DIV></DIV></DIV></SECTION><SECTION class="wrapper style4 container"><HEADER><H3 class="wow fadeInDown">References</H3></HEADER><DIV class=""><OL type="1" class="sourcebox"><LI>Park, D. H., and J. G. Zeikus<A href="http://www.ncbi.nlm.nih.gov/pubmed/10742202" target="_blank" class="sourcebox-link">&quot;Electricity Generation in Microbial Fuel Cells Using Neutral Red as an Electronophore.&quot;</A>
									Applied and Environmental Microbiology 66.4 (2000): 1292-297</LI><LI>Berríos-Rivera, S.<A href="http://www.ncbi.nlm.nih.gov/pubmed/12616693" target="_blank" class="sourcebox-link">&quot;The Effect of NAPRTase Overexpression on the Total Levels of NAD<SUP>+</SUP>, The NADH/NAD Ratio, and the Distribution of Metabolites in <I>Escherichia Coli.</I>&quot;</A>
									Metabolic Engineering 4.3 (2002): 238-47</LI><LI>Wubbolts MG, Terpstra P, van Beilen JB, Kingma J, Meesters HA, Witholt B.<A href="http://www.ncbi.nlm.nih.gov/pubmed/2211655" target="_blank" class="sourcebox-link">&quot;Variation of cofactor levels in Escherichia coli. Sequence analysis and expression of the pncB gene encoding nicotinic acid phosphoribosyltransferase&quot;</A>
									Journal of Biological Chemistry 1990 Oct 15;265(29):17665-72.</LI><LI>San, Ka-Yiu, George N. Bennett, Susana J. Berrı́os-Rivera, Ravi V. Vadali, Yea-Tyng Yang, Emily Horton, Fred B. Rudolph, Berna Sariyar, and Kimathi Blackwood.<A href="http://www.ncbi.nlm.nih.gov/pubmed/12009797" target="_blank" class="sourcebox-link">&quot;Metabolic Engineering through Cofactor Manipulation and Its Effects on Metabolic Flux Redistribution in Escherichia Coli.&quot;</A>
									Metabolic Engineering 4.2 (2002): 182-9</LI></OL></DIV></SECTION></ARTICLE><FOOTER id="footer"><DIV class="container"><DIV class="row"><DIV class="4u 12u(narrower) footer1"><H1>Institutions</H1><UL><LI><A href="http://www.unitn.it/en" target="_blank" class="defaultLink unitn_icon">University of Trento</A></LI><LI><A href="http://web.unitn.it/en/cibio" target="_blank" class="defaultLink cibio_icon">CIBIO - Centre for Integrative Biology</A></LI><LI><A href="https://2015.igem.org/Main_Page" target="_blank" class="defaultLink igem_icon">iGEM HeadQuarter</A></LI></UL></DIV><DIV class="5u 12u(narrower) footer1" id="sponsorBox"><H1>Sponsors</H1></DIV><DIV class="3u 12u(narrower) footer1"><H1>Follow us</H1><UL><LI><A href="https://www.facebook.com/iGEM2015Trento?fref=ts" target="_blank" class="defaultLink facebook_icon">Facebook</A></LI><LI><A href="https://twitter.com/igemtrento" target="_blank" class="defaultLink twitter_icon">Twitter</A></LI></UL></DIV></DIV></DIV><DIV style="text-align:center; font-size:0.8em;"> Design by Moreno Zolfo. Powered by MediaWiki and HTM5UP. This website is under the <A target="_blank" href="http://creativecommons.org/licenses/by/4.0/" style="color:#333;">CC Attribution licence, Version 4</A></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>