<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UNITN-Trento_Measurement skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UNITN-Trento/Measurement</SPAN></H1><DIV id="bodyContent"><DIV id="super_wrap"><DIV class="no-sidebar"><DIV id="page-wrapper"><HEADER id="header"><H1 id="logo">UNITN-TRENTO <SPAN id="igem2015-spanned"><SPAN style="text-transform:lowercase">i</SPAN>GEM 2015</SPAN></H1><NAV id="nav"><UL><LI id="menulin_home"><A href="https://2015.igem.org/Team:UNITN-Trento">Home</A></LI><LI id="menulin_team"><A href="https://2015.igem.org/Team:UNITN-Trento/Team">Team</A></LI><LI id="menulin_project" class="submenu"><A href="#">Project</A></LI><LI id="menulin_project_description"><A href="https://2015.igem.org/Team:UNITN-Trento/Description">Overview</A></LI><LI id="menulin_project_experiments"><A href="https://2015.igem.org/Team:UNITN-Trento/Experiments">Experiments / Protocols</A></LI><LI id="menulin_project_results" class="submenu"><A href="https://2015.igem.org/Team:UNITN-Trento/Results">Results</A></LI><LI id="menulin__project_results_pr"><A href="https://2015.igem.org/Team:UNITN-Trento/Results/Proteorhodopsin">Proteorhodopsin (PR)</A></LI><LI id="menulin__project_results_pncB"><A href="https://2015.igem.org/Team:UNITN-Trento/Results/pncB">pncB</A></LI><LI id="menulin__project_results_blh"><A href="https://2015.igem.org/Team:UNITN-Trento/Results/BLH">Retinal Production (BLH)</A></LI><LI id="menulin__project_results_mfc"><A href="https://2015.igem.org/Team:UNITN-Trento/Results/MFC">Microbial Fuel Cell (MFC)</A></LI></UL><LI id="menulin_project_notebook"><A href="https://2015.igem.org/Team:UNITN-Trento/Notebook">Notebook</A></LI><LI id="menulin_project_modeling"><A href="https://2015.igem.org/Team:UNITN-Trento/Modeling">Modeling</A></LI><LI id="menulin_project_prototype"><A href="https://2015.igem.org/Team:UNITN-Trento/Design">Prototype</A></LI><LI id="menulin_interlab" class="submenu"><A href="https://2015.igem.org/Team:UNITN-Trento/Measurement">Interlab</A></LI><LI><A href="https://2015.igem.org/Team:UNITN-Trento/Measurement#interlab_intro" onclick="javascript:checkAndScrollTo('interlab_intro');">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:UNITN-Trento/Measurement#interlab_design" onclick="javascript:checkAndScrollTo('interlab_design');">Experimental Design</A></LI><LI><A href="https://2015.igem.org/Team:UNITN-Trento/Measurement#interlab_protocols" onclick="javascript:checkAndScrollTo('interlab_protocols');">Experiments and Protocols</A></LI><LI><A href="https://2015.igem.org/Team:UNITN-Trento/Measurement#interlab_final" onclick="javascript:checkAndScrollTo('interlab_final');">Conclusions</A></LI><LI id="menulin_parts"><A href="https://2015.igem.org/Team:UNITN-Trento/Parts">Parts</A></LI><LI id="menulin_judging"><A href="https://2015.igem.org/Team:UNITN-Trento/Judging">Judging</A></LI><LI class="submenu" id="menulin_more"><A href="#">More...</A></LI><LI id="menulin_more_human"><A href="https://2015.igem.org/Team:UNITN-Trento/Practices">Human Practices</A></LI><LI id="menulin_more_attributions"><A href="https://2015.igem.org/Team:UNITN-Trento/Attributions">Attributions</A></LI><LI id="menulin_more_safety"><A href="https://2015.igem.org/Team:UNITN-Trento/Safety">Safety</A></LI><LI id="menulin_more_collaborations"><A href="https://2015.igem.org/Team:UNITN-Trento/Collaborations">Collaborations</A></LI><LI id="menulin_more_gallery"><A href="https://2015.igem.org/Team:UNITN-Trento/Gallery">Gallery</A></LI></NAV></HEADER><SECTION id="cta" class="cta-interlab"><HEADER><H2><STRONG>InterLab Measurement Study</STRONG></H2><P>What happens when GFP meets different promoters? Are they all the same? Here is what we found out!</P></HEADER><FOOTER><UL class="buttons"><LI><P id="interlab_intro_btn" class="button small bstyle1" style="line-height:2em;" onclick="javascript:scrollToID('interlab_intro')"><SPAN class="primary">Introduction</SPAN><SPAN class="primary">&amp; Achievements</SPAN></P></LI><LI><P id="interlab_design_btn" class="button small bstyle3" onclick="javascript:scrollToID('interlab_design')"><SPAN class="primary">Experimental</SPAN><SPAN class="primary">Design</SPAN></P></LI><LI><P id="interlab_protocols_btn" class="button small bstyle5" onclick="javascript:scrollToID('interlab_protocols')"><SPAN class="primary">Experiments</SPAN><SPAN class="primary">&amp; Protocols</SPAN></P></LI><LI><P id="interlab_final_btn" class="button small bstyle7" style="line-height:2em;" onclick="javascript:scrollToID('interlab_final')"><SPAN class="primary">Final Discussion</SPAN><SPAN class="primary">&amp; Results</SPAN></P></LI></UL></FOOTER></SECTION><ARTICLE id="main"><SECTION class="wrapper style4 container" style="margin-top:1em;"><DIV class="content"><SECTION><HEADER><H3 class="wow fadeInDown">The Interlab Measurement Study</H3></HEADER><P class="wow fadeInDown">The characterization of standard parts has always been one of the main concerns in Synthetic Biology. For this very same reason, iGEM teams from all around the World were suggested to take part in the biggest measurement study ever conducted and the 2015 UNITN-Trento iGEM team answered the call. The goal of this <SPAN class="i_enph italic">Second International Measurement Interlab Study</SPAN> is to assemble <SPAN class="i_enph">three different devices</SPAN>, each one containing a <SPAN class="i_enph">promoter</SPAN> with a screening plasmid intermediate and collect as many fluorescence data as possible. iGEM teams are free to use any technique to measure their devices as long <SPAN class="i_enph">the obtained data are solid and reproducible</SPAN>.</P><DIV class="row"><DIV class="4u 12u(narrower)"><DIV href="#" class="wow zoomIn rotate-box square-icon"><DIV class="rotate-box-info"><H4>Clear Data and Protocols</H4><P>We listed in the ”InterLab Study” page:
												<UL class="arrowed"><LI>All devices measured for this study</LI><LI>All protocols developed and adopted</LI><LI>Sequencing data for all measurements devices</LI></UL></P></DIV></DIV></DIV><DIV class="4u 12u(narrower)"><DIV href="#" class="wow zoomIn delay02 rotate-box square-icon"><DIV class="rotate-box-info"><H4><SPAN>3</SPAN> BioBrick devices</H4><P>We used the three BioBrick devices listed in the ”Required Devices” section.</P></DIV></DIV></DIV><DIV class="4u 12u(narrower)"><DIV href="#" class="wow zoomIn delay05 rotate-box square-icon"><DIV class="rotate-box-info"><H4><SPAN> 5</SPAN> techniques </H4><P>We used <SPAN class="i_enph italic">in-vivo</SPAN> and <SPAN class="i_enph italic">in-vitro</SPAN> techniques for measuring RNA and protein levels:
												<UL class="arrowed"><LI>Plate Reader</LI><LI>Spectrofluorometer</LI><LI>Flow Cytometer</LI><LI>RT-qPCR</LI><LI>Cell-Free Extract</LI></UL></P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="4u 12u(narrower)"><DIV href="#" class="wow zoomIn rotate-box square-icon"><DIV class="rotate-box-info"><H4> Statistical Reliability </H4><P>We have three biological replicates for each measurement, with positive and negative controls</P></DIV></DIV></DIV><DIV class="4u 12u(narrower)"><DIV href="#" class="wow zoomIn delay02 rotate-box square-icon"><DIV class="rotate-box-info"><H4>Worksheet and Protocol</H4><P>We completed the InterLab Worksheet and the InterLab Protocol</P></DIV></DIV></DIV><DIV class="4u 12u(narrower)"><DIV href="#" class="wow zoomIn delay05 rotate-box square-icon"><DIV class="rotate-box-info"><H4>Extra Credit Assignment</H4><P>We did technical replicates for each sample</P></DIV></DIV></DIV></DIV></SECTION></DIV></SECTION><SECTION class="wrapper style4 container" style="margin-top:1em;"><DIV class="content"><SECTION><HEADER><H3 class="wow fadeInDown">Experimental Design</H3></HEADER><P class="wow fadeInDown">We used the <SPAN class="i_enph">three mandatory devices</SPAN> for the measurement study:
								<UL class="customlist arrowed"><LI>Device 1: <A class="i_linker registry" target="_blank" target="_blank" href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A> + <A class="i_linker registry" href="http://parts.igem.org/Part:BBa_I13504">BBa_I13504</A> in pSB1C3</LI><LI>Device 2: <A class="i_linker registry" target="_blank" href="http://parts.igem.org/Part:BBa_J23106">BBa_J23106</A> + <A class="i_linker registry" registry="">BBa_I13504</A> in pSB1C3</LI><LI>Device 3: <A class="i_linker registry" target="_blank" href="http://parts.igem.org/Part:BBa_J23117">BBa_J23117</A> + <A class="i_linker registry" registry="">BBa_I13504</A> in pSB1C3</LI><LI>Negative control: <A class="i_linker registry" target="_blank" href="http://parts.igem.org/Part:BBa_R0040">BBa_R0040</A> in pSB1C3</LI><LI>Positive control: <A class="i_linker registry" target="_blank" href="http://parts.igem.org/Part:BBa_I20270">BBa_I20270</A> in pSB1C3.</LI></UL></P><DIV class="row"><DIV class="7u 12u(narrower)"><P>The measurement devices were prepared by amplifying the reporter (<A class="i_linker registry" href="http://parts.igem.org/Part:BBa_I20270" target="_blank">BBa_I20270</A>) by PCR. The amplified insert was then cut with <SPAN class="i_enph">XbaI</SPAN> and <SPAN class="i_enph">PstI</SPAN> and ligated into the plasmid containing the promoter previously cut with <SPAN class="i_enph">SpEI</SPAN> and <SPAN class="i_enph">PstI</SPAN>. All the devices were confirmed by <B>restriction digestion</B> as well as <B>DNA sequencing</B>.</P></DIV></DIV><DIV class="row"><DIV class="6u 12u(narrower) rotate-box-superhover"><H4 class="header4 wow flipInX" style="text-align:center;">In-vivo Measurements </H4><P>The confirmed devices were then transformed in different bacterial strains of <I>E. coli</I>: <SPAN class="i_enph">NEB10β</SPAN>, <SPAN class="i_enph">NEB Express</SPAN>, and <SPAN class="i_enph">JM109</SPAN>. Each measurement was taken at the same optical density to allow a more precise comparison of the data. For each device we have <SPAN class="i_enph">3 biological</SPAN> and <SPAN class="i_enph">3 technical measurements</SPAN> for each used technique. We measured in vivo fluorescence emission in different ways using <B>Tecan Infinite 200 PRO plate reader</B>, <B>Varian Cary Eclipse spectrofluorometer</B>, and <B>BD FACSCanto flow cytometer</B>.</P></DIV><DIV class="6u 12u(narrower) rotate-box-superhover"><H4 class="header4 wow flipInX" style="text-align:center;">In-vitro Measurements </H4><P>We also focused on <SPAN class="i_enph">transcription</SPAN> since the characterization is about promoters. To do so we performed RT-qPCR using a <B>BioRad CFX96 Touch™ Real-Time PCR Detection System</B>. Additionally, we performed an <SPAN class="i_enph">in vitro characterization</SPAN> study, by measuring the fluorescence intensities of each device with a <B>Cell-Free <I>E. coli</I> S30 Extract System</B> with a <B>Qiagen Rotor-Gene Q as the spectrofluorometer</B>.</P></DIV></DIV></SECTION></DIV></SECTION><SECTION class="wrapper style4 container" style="margin-top:1em;"><DIV class="content"><SECTION><HEADER><H3 class="wow fadeInDown">Experiments &amp; Protocols</H3></HEADER><H4 class="header4 displayControl">Extraction from the Registry</H4><DIV style="display:none;"><P>All the parts needed for the InterLab Study were extracted from the <SPAN class="i_enph">2015 iGEM Registry Distribution Kit</SPAN>.</P><TABLE class="standard_table"><THEAD><TR><TD>Part ID</TD><TD>Description</TD><TD>Plasmid Backbone</TD><TD>2015 Registry Location</TD></TR></THEAD><TBODY><TR><TD>BBa_K823005</TD><TD>Anderson promoter J23101</TD><TD>pSB1C3</TD><TD>Plate 1; 20K</TD></TR><TR><TD>BBa_K823008</TD><TD>Anderson promoter J23106</TD><TD>pSB1C3</TD><TD>Plate 1; 22A</TD></TR><TR><TD>BBa_K823013</TD><TD>Anderson promoter J23117</TD><TD>pSB1C3</TD><TD>Plate 1; 22K</TD></TR><TR><TD>BBa_I12504</TD><TD>RBS + GFP + 2 terminators</TD><TD>pSB1A2</TD><TD>Plate 4; 21J</TD></TR><TR><TD>BBa_R0040</TD><TD>TetR sequence</TD><TD>pSB1C3</TD><TD>Plate 2; 6F</TD></TR><TR><TD>BBa_I20270</TD><TD>Promoter MeasKit</TD><TD>pSB1C3</TD><TD>Plate 3; 8P</TD></TR></TBODY></TABLE></DIV><H4 class="header4 displayControl">Polymerase Chain Reaction (PCR)</H4><DIV class="row" style="display:none;"><DIV class="5u 12u(narrower) centered"><P class="image_caption"><SPAN> Electrophoresis gel of the reporter gene</SPAN> The bright band corresponds to the part <A class="i_linker registry" target="_blank" href="http://parts.igem.org/Part:BBa_I13504">BBa_I13504</A>. The part was amplified by Phusion PCR and digested with XbaI and PstI. We can see the expected band at around 875 bp.</P></DIV><DIV class="7u 12u(narrower)"><P>The reporter gene (i.e. <SPAN class="i_enph italic">GFPmut3b</SPAN> from part <A class="i_linker registry" href="http://parts.igem.org/Part:BBa_I13504" target="_blank">BBa_I13504</A>) was amplified using the Phusion polymerase from New England Biolabs and primers matching the prefix and suffix.</P><P>We used <SPAN class="i_enph quantity">50 ng</SPAN> of template with an annealing temperature of <SPAN class="i_enph quantity">59 °C</SPAN> and an extension time of <SPAN class="i_enph quantity">90 seconds</SPAN>. The PCR was confirmed by electrophoresis and subsequently purified with NucleoSpin Gel and PCR Clean-Up Kit from Macherey-Nigel. <SPAN class="i_enph quantity">125 ng</SPAN> of purified PCR were digested with <SPAN class="i_enph quantity">1 μl</SPAN> of <SPAN class="i_enph">XbaI</SPAN> and <SPAN class="i_enph quantity">1 μl</SPAN> of <SPAN class="i_enph">PstI</SPAN> overnight at <SPAN class="i_enph quantity">37°C</SPAN>. The following morning the sample was treated with <SPAN class="i_enph quantity">1 μl</SPAN> of <SPAN class="i_enph">DpnI</SPAN> for 2 hour at <SPAN class="i_enph quantity">37°C</SPAN> and the enzymes were deactivated for <SPAN class="i_enph quantity">20 min</SPAN>  at <SPAN class="i_enph quantity">80 °C</SPAN>.</P></DIV></DIV><H4 class="header4 displayControl">Restriction Digestion of plasmids containing the promoter</H4><DIV style="display:none;"><P>Each promoter containing plasmid was digested with <SPAN class="i_enph quantity">1 μl</SPAN> of <SPAN class="i_enph">SpeI</SPAN> and <SPAN class="i_enph quantity">1 μl</SPAN> of <SPAN class="i_enph">PstI</SPAN> at <SPAN class="i_enph quantity">37°C</SPAN> overnight. The day after <SPAN class="i_enph quantity">1 μl</SPAN> of phosphatase (CIP from New England Biolabs) was added for <SPAN class="i_enph quantity">2 hours</SPAN> at <SPAN class="i_enph quantity">37°C</SPAN>. The enzymes were then heat deactivated.</P><P>The digestion reactions were assembled in this way:</P><TABLE style="width:60%;" class="standard_table"><THEAD><TR><TD>PCR Products</TD><TD>Plasmids</TD></TR></THEAD><TBODY><TR><TD>Template</TD><TD>3000 ng</TD><TD>2000 ng</TD></TR><TR><TD>Enzyme 1</TD><TD>2 μl</TD><TD>1.5 μl</TD></TR><TR><TD>Enzyme 2</TD><TD>2 μl</TD><TD>1.5 μl</TD></TR><TR><TD>Buffer (Stock 10X)</TD><TD>5 μl</TD><TD>5 μl</TD></TR><TR><TD>BSA (Stock 10X)</TD><TD>5 μl</TD><TD>5 μl</TD></TR><TR><TD>Water</TD><TD>Up to 50 μl</TD><TD>Up to 50 μl</TD></TR></TBODY></TABLE></DIV><H4 class="header4 displayControl">Ligation</H4><DIV style="display:none;"><DIV class="row"><DIV class="7u 12u(narrower)"><TABLE class="standard_table" style="margin-top:0;"><THEAD><TR><TD>Plasmid:Insert = 1:3</TD><TD>Control</TD></TR></THEAD><TBODY><TR><TD>Vector</TD><TD>100 ng</TD><TD>100 ng</TD></TR><TR><TD>Insert</TD><TD>125 ng</TD><TD>-</TD></TR><TR><TD>10X Buffer</TD><TD>2 μl</TD><TD>2 μl</TD></TR><TR><TD>Buffer (Stock 10X)</TD><TD>5 μl</TD><TD>5 μl</TD></TR><TR><TD>T4 - DNA Ligase</TD><TD>2 μl</TD><TD>2 μl</TD></TR><TR><TD>Water</TD><TD>Up to 20 μl</TD><TD>Up to 20 μl</TD></TR></TBODY></TABLE></DIV><DIV class="5u 12u(narrower)"><P>The ligation reactions were assembled and incubated at room temperature for <SPAN class="i_enph quantity">1 hour</SPAN> according to the table beside. Subsequently <SPAN class="i_enph quantity">10 μl</SPAN> of the ligation mixture were transformed and plated into LB-agar plates with the proper antibiotic resistance. The confirmed devices were transfected in NEB10β, JM109, NEB Express.</P></DIV></DIV><DIV class="row"><DIV class="12u 12u(narrower)"><DIV class="captionbox" style="max-width:600px;"><P class="image_caption"><SPAN>Cells expressing the devices</SPAN> These three plates contains <SPAN class="bacterium">E. coli</SPAN> NEB Express transformed with J23101 (<I>top left</I>), J23106 (<I>top right</I>), and J23117 (<I>bottom</I>). Interestingly, promoters' strength are detectable by eye, depending on the colonies' color.</P></DIV> 
											
											Cells expressing the devices. These three plates contains <SPAN class="bacterium">E. coli</SPAN> NEB Express transformed with J23101 (<I>top left</I>), J23106 (<I>top right</I>), and J23117 (<I>bottom</I>). 
										</DIV></DIV></DIV><H4 class="header4 displayControl">Parts Confirmation</H4><DIV style="display:none;"><P style="margin-bottom:0.7em; margin-top:1em;">Correct clones were screened by <SPAN class="i_enph">restriction digestion</SPAN> and <SPAN class="i_enph">confirmed by sequencing</SPAN>:</P><DIV class="row"><DIV class="4u 12u(narrower) centered"><P class="image_caption"><SPAN>Cell pellets at the UV-trans illuminator</SPAN> Pellets of overnight NEB10β cells cultures expressing the assembled devices. Pellet's colors confirmed the promoter strength,  (<I>from left to right</I>) the strongest promoter J23101, the medium strength promoter J23106, the weak promoter J23117, the positive control I20270, the negative control R0040.</P></DIV><DIV class="4u 12u(narrower) centered"><P class="image_caption"><SPAN>Electrophoresis gel of all devices</SPAN> We digested three colonies for each device with XbaI and PstI. The correct assembly is confirmed by the band at 910 pb.</P></DIV><DIV class="4u 12u(narrower)"><UL class="customlist arrowed" style="margin-bottom:1em; margin-left:0 !important;"><LI><A class="i_linker registry" href="http://parts.igem.org/Part:BBa_J23101" target="_blank">BBa_J23101</A>+<A class="i_linker registry" href="http://parts.igem.org/Part:BBa_I13504" target="_blank">BBa_I13504</A><P style="margin:0;"><A class="i_linker" target="_blank" style="font-size:0.8em;" href="https://static.igem.org/mediawiki/2015/b/b2/Unitn_fasta_interlab_GFPK05.fasta.txt">[Sequencing Data]</A></P></LI><LI><A class="i_linker registry" href="http://parts.igem.org/Part:BBa_J23106" target="_blank">BBa_J23106</A>+<A class="i_linker registry" href="http://parts.igem.org/Part:BBa_I13504" target="_blank">BBa_I13504</A><P style="margin:0;"><A class="i_linker" target="_blank" style="font-size:0.8em;" href="https://static.igem.org/mediawiki/2015/6/62/Unitn_fasta_interlab_GFPK08.fasta.txt">[Sequencing Data]</A></P></LI><LI><A class="i_linker registry" href="http://parts.igem.org/Part:BBa_J23117" target="_blank">BBa_J23117</A> +<A class="i_linker registry" href="http://parts.igem.org/Part:BBa_I13504" target="_blank">BBa_I13504</A><P style="margin:0;"><A class="i_linker" target="_blank" style="font-size:0.8em;" href="https://static.igem.org/mediawiki/2015/8/83/Unitn_fasta_interlab_GFPK13.fasta.txt">[Sequencing Data]</A></P></LI></UL></DIV></DIV></DIV><H4 class="header4 displayControl">Glycerol stocks preparation and Sample Growth   </H4><DIV style="display:none;"><P>A single colony was inoculated with a sterile pipette tip in a test tube with <SPAN class="i_enph quantity">10 ml</SPAN> of LB and antibiotic (1000:1 LB to antibiotic ratio) and placed in the thermoshaker (<SPAN class="i_enph quantity">190 rpm</SPAN>, <SPAN class="i_enph quantity">37°C</SPAN>). When the culture got cloudy, <SPAN class="i_enph quantity">40 ml</SPAN> of LB+antibiotic were added to reach a final volume of <SPAN class="i_enph quantity">50 ml</SPAN>. The cells were grown until an OD<SUB>600</SUB> of <SPAN class="i_enph quantity">0.5</SPAN> and then centrifuged at <SPAN class="i_enph quantity">4100 rpm</SPAN> for <SPAN class="i_enph quantity">10 minutes</SPAN> at <SPAN class="i_enph quantity">4 °C</SPAN>. The supernatant was discarded and the cells were resuspend in <SPAN class="i_enph quantity">5 ml</SPAN> of LB + antibiotic + glycerol (<SPAN class="i_enph quantity">20% v/v</SPAN>). The cells were kept on ice and were promptly aliquoted into <SPAN class="i_enph quantity">200 μl</SPAN> tubes and frozen at <SPAN class="i_enph quantity">-80°C</SPAN> immediately. From this protocol we obtained a <SPAN class="i_enph quantity">10X</SPAN> concentrated glycerol stock for each sample.</P><P>The glycerol stock was thaw and added into <SPAN class="i_enph quantity">10 ml</SPAN> of LB with antibiotic, giving a  starting culture with an OD<SUB>600</SUB> of <SPAN class="i_enph quantity">0.1</SPAN>. The sample were grown in a <SPAN class="i_enph quantity">50 mL</SPAN> conical  plastic tube in the termoshaker at <SPAN class="i_enph quantity">37°C</SPAN> and were grown until an OD<SUB>600</SUB><SPAN class="i_enph quantity">0.7</SPAN>. At this  point <SPAN class="i_enph quantity">3 ml</SPAN> of the culture were transferred in a new tube, centrifuged it, and stored at <SPAN class="i_enph quantity">-20°C</SPAN>, unless otherwise indicated.</P></DIV><H4 class="header4 displayControl">Fluorescence readings: Tecan INFINITE 200 PRO Plate Reader   </H4><DIV style="display:none;" class="row"><DIV class="12u 12u(narrower)"><P>The cells were thawed and resuspended in <SPAN class="i_enph quantity">3 ml</SPAN> of PBS. An aliquot of <SPAN class="i_enph quantity">150 μl</SPAN> of each sample was placed into a <B>White, Flat-bottomed, 96-well Costar Plate</B> (code: 3917) and fluorescence intensities were taken with a <B>Tecan Infinite 200 Pro Plate Reader</B> (made in Switzerland). Excitation wavelength and emission wavelength were <SPAN class="i_enph quantity">395 nm</SPAN> and <SPAN class="i_enph quantity">509 nm</SPAN>, respectively. The gain was optimized at <SPAN class="i_enph quantity">70 V</SPAN> and kept constant for each sample. PBS was used as blank. To obtain technical replicates, fluorescence intensities were acquired for three aliquots of the same biological sample, keeping the same instrumental conditions. The raw data were adjusted for the blank value and the means across the replicates with their relative standard deviation were plotted.</P></DIV></DIV><H4 class="header4 displayControl">Fluorescence readings: Cary Eclipse Fluorescence Spectrophotometer   </H4><DIV style="display:none;" class="row"><DIV class="12u 12u(narrower)"><P>The cells were thaw and resuspended in <SPAN class="i_enph quantity">3 ml</SPAN> of PBS. Each measurement was taken in a clear acrylic cuvette (REF 67.755) with a <B>Cary Eclipse Fluorescence Spectrophotometer</B> (made in USA). The blank was done with PBS. The excitation wavelenght used was <SPAN class="i_enph quantity">395 nm</SPAN> and the spectra were acquired between <SPAN class="i_enph quantity">420 nm</SPAN> to <SPAN class="i_enph quantity">650 nm</SPAN>. The gain was optimized at <SPAN class="i_enph quantity">775 V</SPAN> and kept the same throughout the measurement.</P><P>The fluorescence intensity of the maximum emission wavelenght (<SPAN class="i_enph quantity">514 nm</SPAN>) was adjusted for the blank and the calculated means with relative standard deviations were plotted.</P></DIV></DIV><H4 class="header4 displayControl">Fluorescence readings: BioRad CFX96 Touch Real-Time PCR Detection System</H4><DIV style="display:none;"><P>The cells were grown from glycerol stock until an OD<SUB>600</SUB> of <SPAN class="i_enph quantity">0.7</SPAN> was reached. Total  RNA was purified by using the <B>Thermo Scientific GeneJET RNA Purification Kit</B>, following the manufacturer`s instructions and subsequently genomic DNA was  removed from the total RNA by using the <B>Thermo Scientific RapidOut DNA Removal Kit</B>, following the manufacturer`s instructions. RNA levels were quantified using <B>NanoDrop 1000</B> and reverse transcription of cDNA from the RNA template was  performed with <B>Thermo Scientific RevertAid First Strand cDNA Synthesis Kit</B>,  following the manufacturer`s instructions. qPCR reactions were performed with <B>BioRad CFX96 Touch Real-Time PCR Detection System</B> (made in USA) and  assembled as follows:</P><DIV class="row"><DIV class="6u 12u(narrower)"><TABLE class="standard_table"><TBODY><TR><TD class="heading">cDNA</TD><TD>5 ng</TD></TR><TR><TD class="heading">Primer Fw</TD><TD>180 ng</TD></TR><TR><TD class="heading">Primer Rv</TD><TD>180 ng</TD></TR><TR><TD class="heading">BioRad iQ SYBR® Green Supermix #1708880</TD><TD>Up to 10 μl</TD></TR></TBODY></TABLE></DIV><DIV class="6u 12u(narrower)"><TABLE class="standard_table"><TBODY><TR><TD class="heading">GFP Primer Fw</TD><TD>ATGCTTTGCGAGATACCCAG</TD></TR><TR><TD class="heading">GFP Primer Rv</TD><TD>TGTCTTGTAGTTCCCGTCATC</TD></TR><TR><TD class="heading">idnT Primer Fw</TD><TD>CTGCCGTTGCGCTGTTTATT</TD></TR><TR><TD class="heading">idnT Primer Rv</TD><TD>GATTTGCTCGATGGTGCGTC</TD></TR></TBODY></TABLE></DIV></DIV><P>Primers were designed for the reporter gene <SPAN class="i_enph italic">GFPmut3b</SPAN> and for the housekeeping gene <SPAN class="i_enph italic">idnT</SPAN><SPAN class="lesser">(D-gluconate transporter)</SPAN> as indicated above.</P><P> We then analyzed the raw data, calculating the relative fold expression of each GFP device compared to the housekeeping (ΔCt) and the related standard deviation:</P><TABLE class="standard_table"><THEAD><TR><TD>Device</TD><TD>Relative Fold Expression (A.U.)</TD><TD>Standard Deviation</TD></TR></THEAD><TBODY><TR><TD>J23101</TD><TD class="numeric">67,421.53</TD><TD class="numeric">16,016.27</TD></TR><TR><TD>J23106</TD><TD class="numeric">18,506.44</TD><TD class="numeric">3,146.38</TD></TR><TR><TD>J23117</TD><TD class="numeric">658.08</TD><TD class="numeric">79.09</TD></TR><TR><TD>R0040</TD><TD class="numeric">0.04</TD><TD class="numeric">0.02</TD></TR><TR><TD>I20270</TD><TD class="numeric">17,800.56</TD><TD class="numeric">1,030.40</TD></TR></TBODY></TABLE></DIV><H4 class="header4 displayControl">Fluorescence readings: BD FACSCanto Flow Cytometer</H4><DIV style="display:none;"><P>The cells were grown from glycerol stocks as described above. Differently from before when they reached the OD of <SPAN class="i_enph quantity">0.7</SPAN> were not frozen, but were used immediately to measure fluorescence intensity. An aliquot of <SPAN class="i_enph quantity">5 μl</SPAN> of cells was diluted in <SPAN class="i_enph quantity">900 μl</SPAN> of PBS. The instrument used was a <B>BD FACSCanto</B> (made in USA) set with the following parameters:</P><UL class="customlist arrowed" style="margin-bottom:1em;"><LI>FSC gain: 525 V</LI><LI>SSC gain: 403 V</LI><LI>FITC gain: 510 V</LI><LI>Flow rate: LOW</LI><LI>Total number of events in P2: 10000</LI></UL><P>Those parameters allowed the instrument to process 900/1500 events per second. We analyzed the raw data and plot the means of three replicates with relative standard deviations.</P><DIV class="row"><DIV class="12u 12u(narrower) centered"><DIV class="captionbox" style="max-width:600px;"><P class="image_caption"><SPAN> Fluorescence spectra from flow cytometer analysis of the NEB10β strain</SPAN> The merge shown above represents the devices J23101 (white), J23106 (yellow), J23117 (blue), and the negative control R0040 (orange). Data were analyzed using Cyflogic 1.2. </P></DIV></DIV></DIV></DIV><H4 class="header4 displayControl">Fluorescence readings: E. coli S30 Extract System for DNA Circular   </H4><DIV style="display:none;"><P>Miniprepped DNA was purified and extracted through phenol/chloroform extraction followed by ethanol precipitation. Reactions were set following <B>Promega <I>E. coli</I> S30 Extract System Technical Bulletin</B> and were performed using <B>Qiagen Rotor-Gene Q</B> (made in USA). Parameters for this specific experiment were set as follows:</P><UL class="customlist arrowed"><LI>Green channel gain: <SPAN class="i_enph quantity">0.67</SPAN></LI><LI>Green channel Excitation: <SPAN class="i_enph quantity">365±20 nm</SPAN></LI><LI>Green channel Emission: <SPAN class="i_enph quantity">510±5 nm</SPAN></LI></UL><P>We analyzed the raw data and plot the means of three replicates over time:</P></DIV></SECTION></DIV></SECTION><SECTION class="wrapper style4 container" style="margin-top:1em;"><DIV class="content"><SECTION><HEADER><H3 class="wow fadeInDown">Final Discussion</H3></HEADER><DIV class="row"><DIV class="12u 12u(narrower)"><H4 class="header4-resume wow fadeInDown">Our characterization confirmed the relative strength of the promoters</H4><P><SPAN class="i_enph">J23101 is the strongest promoter</SPAN> among the three, showing high expression of GFP in all three strains, regardless of the technique used. <SPAN class="i_enph">J23106 is the medium promoter</SPAN> and <SPAN class="i_enph">J23117 the weakest</SPAN>.</P><H4 class="header4-resume wow fadeInDown">Ratios across promoters are kept the same</H4><P><SPAN class="i_enph">J23101/J23106</SPAN> fluorescence ratios ranged from <SPAN class="i_enph quantity">2.0</SPAN> to <SPAN class="i_enph quantity">4.5</SPAN>, depending on the strain and the technique. Differently from the other two promoters, <SPAN class="i_enph">J23117</SPAN> showed a very low GFP production, as it was <SPAN class="i_enph">not detectable by eye</SPAN> or using the trans-illuminator and showed little fluorescence with the three techniques used.</P><H4 class="header4-resume wow fadeInDown">Different techniques lead to the same results, with different sensitivities</H4><P>The best way to perform a characterization is to <SPAN class="i_enph">use various techniques</SPAN>. Throughout our experiments we saw that each instrument has a specific sensitivity, which alters the output data. The <SPAN class="i_enph">FACS happened to be the most accurate among all</SPAN>, due to its extremely high intrinsic sensitivity. The plate reader also showed a good accuracy while the <SPAN class="i_enph">fluorometer was not able to detect the weakest promoter</SPAN> from the background noise. The use of the qPCR machine as a spectrofluorometer also gave positive results.</P><H4 class="header4-resume wow fadeInDown"><I>In vitro</I> conditions mimic the <I>in vivo</I> reality</H4><P>Comparing the results obtained from the cell-free extract to the others, we discovered that the <SPAN class="i_enph">promoters behave the same when working <I>in vitro</I> or in living bacteria</SPAN>. Since testing constructs in vitro is much faster than in vivo, our results suggest that it may be wise to first screen parts and/or genetic circuitry in vitro. Then the activity of a smaller subset could be confirmed with in vivo measurements.</P><H4 class="header4-resume wow fadeInDown">Looking at promoters from a different angle</H4><P>In vitro characterization by qPCR allows for the <SPAN class="i_enph">quantification of promoter strength by measuring RNA transcript levels</SPAN>, rather than by looking at the concentrations of something one step removed, i.e. protein. This approach gives a <SPAN class="i_enph">a more direct measure of promoter strength</SPAN>.</P><DIV class="captionbox" style="max-width:850px; width:80%;"><P class="image_caption"><SPAN> Promoter strength across strains and techniques </SPAN> Values presented are the average of three  technical replicates for each of the three biological samples (total of 9 replicates). Data normalized on the maximum expression value of each technique. Heatmap calculated with R Project.</P></DIV><H4 class="header4-resume wow fadeInDown">Bacterial strain does matter</H4><P>The three promoters behaved differently in the different bacterial strains used. The  bacterial strain which gave the highest fluorescence was <SPAN class="i_enph">NEB10β</SPAN>, which showed significantly increased expression of protein in all cases  when compared with JM109 and NEB Express. Clearly this discordance among strains must to be due to their <SPAN class="i_enph">different genotypes</SPAN>. A different bacterial proteome (e.g. presence/lack of specific  proteases and/or chaperonins) may alter protein production, processing and folding, and thus fluorescence emission.</P><DIV class="captionbox" style="max-width:850px; width:80%;"><P class="image_caption"><SPAN> Strain influence on promoters efficiency </SPAN> Values presented are the average of three  technical replicates for each of the three biological samples (total of 9 replicates). Data normalized on the the maximum expression value of each technique. Heatmap calculated with R Project.</P></DIV></DIV></DIV></SECTION></DIV></SECTION></ARTICLE><FOOTER id="footer"><DIV class="container"><DIV class="row"><DIV class="4u 12u(narrower) footer1"><H1>Institutions</H1><UL><LI><A href="http://www.unitn.it/en" target="_blank" class="defaultLink unitn_icon">University of Trento</A></LI><LI><A href="http://web.unitn.it/en/cibio" target="_blank" class="defaultLink cibio_icon">CIBIO - Centre for Integrative Biology</A></LI><LI><A href="https://2015.igem.org/Main_Page" target="_blank" class="defaultLink igem_icon">iGEM HeadQuarter</A></LI></UL></DIV><DIV class="5u 12u(narrower) footer1" id="sponsorBox"><H1>Sponsors</H1></DIV><DIV class="3u 12u(narrower) footer1"><H1>Follow us</H1><UL><LI><A href="https://www.facebook.com/iGEM2015Trento?fref=ts" target="_blank" class="defaultLink facebook_icon">Facebook</A></LI><LI><A href="https://twitter.com/igemtrento" target="_blank" class="defaultLink twitter_icon">Twitter</A></LI></UL></DIV></DIV></DIV><DIV style="text-align:center; font-size:0.8em;"> Design by Moreno Zolfo. Powered by MediaWiki and HTM5UP. This website is under the <A target="_blank" href="http://creativecommons.org/licenses/by/4.0/" style="color:#333;">CC Attribution licence, Version 4</A></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>