<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UAM_Poznan_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UAM Poznan/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="header"><A href="#" class="nav-toggle">Toggle Navigation</A><NAV class="cmn-tile-nav"><UL class="clearfix"><LI class="colour-1"><A href="#February">February</A></LI><LI class="colour-2"><A href="#March">March</A></LI><LI class="colour-3"><A href="#April">April</A></LI><LI class="colour-4"><A href="#May">May</A></LI><LI class="colour-5"><A href="#June">June</A></LI><LI class="colour-6"><A href="#July">July</A></LI><LI class="colour-7"><A href="#August">August</A></LI></UL></NAV></DIV><DIV class="largecontainer"><A name="February"><H2>February</H2></A><A name="Feb1st"><H4>1<SUP>st</SUP> week</H4></A><P>
  Our goal is to prepare a series of plasmids expressing his-tagged superfolder GFP under the control of four nontoxic inducers: arabinose, melibiose, rhamnose and xylose. We also want to check their activity under different conditions and compare them with each other. 
  First we amplified pSB1C3 plasmid with specially designed primers , in order to combine it later with our promoters' fragments. We performed the amplification with PCR reaction using Phusion polymerase; the volume of the reaction was 50μl. (See Methods). All amplifications with PCR were performed using  primers with overlaps designed with NEBuilder (See Methods).
  We amplified rhamnose promoter without <I>EcoR</I>I (from proRhaWT) and super-folder GFP (with his-tag at its C-end). We add specially designed overlaps in order to combine those elements with plasmid later on. 
  We also amplified arabinose promoter (Arashort1)- the shorter version of AraWT with operator's elements.
  We visualized PCR products under UV light after electrophoresis on 1% agarose gel (See Methods). We cut out bands that represented the desired products and purified  them on columns, according to the attached protocol. Then we checked the concentrations on NanoDrop spectrophotometer- luckily all concentrations were more than 100ng/μl.
</P><A name="Feb2nd"><H4>2<SUP>nd</SUP> week</H4></A><P>
  We performed CPEC reaction with our amplified fragments from previous week- CPEC is a circular polymerase extension cloning; the combining of different elements is possible thanks to overlaps (see Methods).
The rule is, that one should take ten times less ng of template than it has base pairs- e.g. 200bp → 20ng.</P><UL><LI>pSB1C3 + proRha1 (without <I>EcoR</I>I) + sfGFP,</LI><LI>pSB1C3 + Arashort1 (in phusion with GFP).</LI></UL><A name="Feb3rd"><H4>3<SUP>rd</SUP> week</H4></A><P>After CPEC we precipitated our constructs with chloroform/phenol extraction, 96% ethanol and sodium acetate (3M), using glicoblue for pellet visualization (see Methods).
We performed <I>E.coli</I> transformation using electroporation (See Methods) with: </P><UL><LI>AraGFP WT (as a control),</LI><LI>Arashort1,</LI><LI>proRha1 without <I>EcoR</I>I</LI></UL><P>We used DH5α and about 5μl of our constructs. Unfortunately, besides control, nothing grew on plates... 
 The possible reason of that is the falsity of NanoDrop - maybe we didn't use enough amount of different elements.</P><A name="Feb4th"><H4>4<SUP>th</SUP> week</H4></A><P>We tried to perform the assembly of our first constructs one more time using CPEC, but this time we used two times more nanograms of elements then last time.
We also prepared fragments for our next construct:  proD-sfGFP, which is an isolated, constitutive promoter; we amplified proD and sfGFP-D- with overlaps for each other and with overlaps for plasmid.</P><A name="March"><H2>March</H2></A><A name="March1st"><H4>1<SUP>st</SUP> week</H4></A><P>We performed the elution of proD and sfGFPD. Then we diluted them and use as a template in a reamplification with primers that add overlaps:</P><P>We performed elution of the desired agarose bands- everything went well and now we have fragments for the assembly with plasmid.
We also precipitated constructs from our last CPEC (Arashort1 and proRha1 without <I>EcoR</I>I).</P><A name="March2nd"><H4>2<SUP>nd</SUP> week</H4></A><P>We performed CPEC for proD-sfGFPD construct. After CPEC we precipitated this construct and purified it. 
We performed electroporation on competent cells (DH5α, about 100μl) with:</P><UL><LI>proD-sfGFP,</LI><LI>Arashort1,</LI><LI>proRha1 without <I>EcoR</I>I.</LI></UL><P>We have our first construct- proD-sfGFP! Unfortunately, that's all that grew. </P><P>We started to wonder, whether our fragments were amplified with proper primers, so we performed amplification of all fragments once again, just to be sure.
We also amplified proC (which is an isolated, constitutive promoter, just like proD; they differ from each other only in one nucleotide) and proC with a thermometer (a hairpin at its 5'UTR) and sfGFP for them. We checked it all on an agarose gel, cut out desired bands and performed an elution.</P><A name="March3rd"><H4>3<SUP>rd</SUP> week</H4></A><P>Another good construct!
 After transformation we have proRha1 without <I>EcoR</I>I. We picked single colonies and checked them on plates with M9 medium with 0,4% of rhamnose (which is an inducer) and with 0,4% of glucose (as a negative control)- bacteria on M9 with rhamnose should demonstrate fluorescency under UV light, because of sfGFP expression- proRha1  is induced by rhamnose. (See Methods)</P><P>So as we can see in the picture, our proRha1 is acting in a good way- it is induced by rhamnose (left) and the colonies don’t produce fluorescent sfGFP on different carbon source- glucose.</P><P>We picked the best colonies from proRha1 and proD and inoculated  the overnight liquid LB with them in order to multiply bacteria that took up plasmid. Then we centrifuged the medium with bacteria to get pellet. After that, we performed plasmid isolation (see Methods). We tried to check, whether it is better to isolate them with isopropanol or ethanol.
We found out that it is better to use isopropanol- on the gel, samples that were treated with isopropanol show no contamination of small RNAs (samples in the boxes- picture), whilst samples after ethanol have those low bands that represent those RNAs (=contamination).</P><P>After sequencing our plasmids, we found out that constructs were assembled in a proper way, which is a very good news.</P><A name="March4th"><H4>4<SUP>th</SUP> week</H4></A><P>We performed CPEC for these constructs:</P><UL><LI>proC,</LI><LI>proC with thermometer,</LI><LI>Arashort1 (with fragemnts that were once again amplified).</LI></UL><P>Of course we mix everything with pSB1C3 plasmid.
After CPEC we purified constructs and completed electroporation with DH5α. We also used AraGFP-WT as a control.
</P><P>Aaand... everything went well!:) We have colonies on all the plates.
We inoculated some of the colonies on the M9 minimal medium with agarose/glucose to investigate how those promoters work.</P><P>That's how proC promoters look on M9 with 0,4% of glucose:</P><P>This is Arashort1 on arabinose (left) and on glucose (right; negative control). And, as you can see, the induction occurs only in a presence of arabinose:</P><P>That's AraGFP-WT (control) on arabinose (left) and glucose (right):</P><A name="April"><H2>April</H2></A><A name="April1st"><H4>1<SUP>st</SUP> week</H4></A><P>We had our proC_thermometer construct on two Petri dishes with M9 medium with glucose- we put one in 37°C and second in 23°C. We were hoping that low temperature will stop the induction in this construct because of hairpin, that we created in the 5'UTR. 
In 37°C <I>E.coli</I> showed fluorescence  (because of sfGFP expression), which is a normal thing for constitutive promoters. 
Bacteria with proC_thermometer on plate that was in 23°C also started to show fluorescence , but after 2 days, so the induction  is slower, but it does occur. That means, that in the future we should make this hairpin stronger, if we really want to stop the induction in lower temperature (e.g. add more GC pairs).
</P><A name="April2nd3rd"><H4>2<SUP>nd</SUP> and 3<SUP>rd</SUP> week</H4></A><P>We picked the best colonies from Arashort1 and proC and inoculated the overnight liquid LB with them in order to multiply bacteria that took up plasmid. Then we centrifuged medium with bacteria to get pellet. After that we performed plasmid isolation (see Methods).
We prepared all of our plasmids for sequencing.  
After sequencing: proC looks good, but Arashort1 doesn't- in a mysterious way we assembled  it with Glo protein, not with sfGFP. So we have to repeat it, starting from the beginning.</P><A name="May"><H2>May</H2></A><A name="May1st2nd"><H4>1<SUP>st</SUP> and 2<SUP>nd</SUP> week</H4></A><P>We started amplifying fragments (with specially designed primers of course) for the next attempt for the Arashort1 assembly (let's hope that this time it will have sfGFP protein!). 
We also started amplification of fragments for Arashort2 (shorter arabinose promoter, without elements responsible for catabolic repression). Our template was AraGFP-WT. After PCR, we performed electrophoresis and elution of the desired bands:</P><P>(all bands represent Arashort2 promoter fragments, without plasmid)</P><A name="May3rd"><H4>3<SUP>rd</SUP> week</H4></A><P>We performed CPEC reaction for the assembly of  Arashort2 (Arashort2  + pSB1C3) and Arashort1 (Arashort1 + pSB1C3). We purified it after the reaction and put it in the freezer (-20°C).</P><A name="May4th"><H4>4<SUP>th</SUP> week</H4></A><P>We transformed DH5α using electroporation with Arashort1 and Arashort2 (see Methods).  After electroporation and incubation in 37°C we inoculated bacteria on Petri dishes.
Unfortunately, there was nothing on those plates the next day- bacteria didn't grow.</P><A name="June"><H2>June</H2></A><A name="June1st2nd"><H4>1<SUP>st</SUP> and 2<SUP>nd</SUP> week</H4></A><P>We performed CPEC for Arashort1 and Arashort2 one more time- we used more nanograms of different fragments (we started to think that NanoDrop is “lying” and it overstates the results- it's telling us that the sample has more DNA, than it has in the reality). </P><A name="June3rd"><H4>3<SUP>rd</SUP> week</H4></A><P>We purified constructs after CPEC (Arashort1 and Arashort2) and performed bacteria transformation- we also used AraGFP-WT construct as a control to check, whether our competent bacteria are not defective.
The transformation was successful- bacteria grew on all the plates. We inoculated some colonies on M9 minimal medium with arabinose and with glucose (as a negative control).
They started to show fluorescence on plates with arabinose, because it is required for the induction of expression in arabinose promoters.</P><P>Arashort1 (plate with glucose on the left, plate with arabinose on the right)</P><P>Arashort2 (plate with glucose on the left, plate with arabinose on the right)</P><P>We inoculated some colonies from plates with arabinose into the overnight liquid LB medium. The next day we isolated plasmids, that, we hope, carried Arashort1 and Arashort2 promoters. We checked it by sequencing- they were assembled in a good way.
</P><P>We started amplifying  fragments for the new construct- proRha2- which is a promoter induced by ramnose, but shorter- we will try to cut out the site responsible for catabolic repression. Using specially designed primers and Rha-sfGFP-WT promoter as a template, we were able to amplify shorter version- proRha2-sfGFP.</P><P>Those two bands represent shorter proRha2 with sfGFP. We cut it out  and performed the elution.</P><P>Next we reamplified it with primers adding overlaps for plasmid:</P><A name="June4th"><H4>4<SUP>th</SUP> week</H4></A><P>We performed CPEC reaction for the assembly of proRha2.
After CPEC reaction and purifying the product, we performed bacteria transformation. Luckily, everything went well and we have colonies on our Petri dishes. We inoculated some of the colonies into M9 medium with glucose as a control (left) and with rhamnose for the induction (right):</P><P>As it is seen on the picture, proRha2 is not as strong as RhaWT or proRha1 without <I>EcoR</I>I. 
We picked some colonies, inoculated them into liquid LB medium and performed plasmid isolation. Then we checked the assembly of the promoter during sequencing – the construct was assembled in a proper way.</P><P>We also prepared M9 minimum media with 0,1%, 0,25%, 0,01%, 1% and 0,5% of different sugars: arabinose, rhamnose, xylose, melibiose. 
And M9 minimum media with 0,1% of:</P><OL><LI>Glucose.</LI><LI>Fructose.</LI><LI>Lactose.</LI><LI>Sacharose.</LI><LI>Maltose.</LI><LI>Galactose.</LI><LI>Xylose.</LI><LI>Melibiose.</LI><LI>Arabinose.</LI><LI>Rhamnose.</LI><LI>Glycerol.</LI></OL><P>We then checked how bacteria with proC promoter grow on M9 with those sugars (0,1%) after 43h. We centrifuged the culture,  resuspend the pellet in Mili Q water and put probes into a freezer.
The mass of bacteria with proC that grew on different carbon sources:</P><OL><LI> 0,03</LI><LI>0,019</LI><LI>0,006</LI><LI>0,003</LI><LI>0,005</LI><LI>0,007</LI><LI>0,008</LI><LI>0,003</LI><LI>0,003</LI><LI>0,003</LI><LI>0,004</LI></OL><P>We think that those weights are so little, that there is no sense in measuring the weight in future experiments carried in 2/5ml tubes, cause it is really easy to make a mistake, and obtain false data.</P><A name="July"><H2>July</H2></A><A name="July1st"><H4>1<SUP>st</SUP> week</H4></A><P>We performed the assembly of XylWT-sfGFP promoter, which is a promoter induced by xylose (we added sfGFP to it) and is present in <I>E.coli</I>. Next we shortened this promoter and obtain XylA1. We amplified fragments for the assembly of the next constructs- XylS and XylF- those are all shorter versions. <B>XylWT: XylF-XylAwt-sfGFP</B> - contains both promoters (xylA and F) oriented in the direction in which sfGFP expression is driven by the original xylA promoter copied from Escherichia coli genome.<B>XylA1: XylF-XylA1-sfGFP</B> containing both promoters oriented in the direction in which sfGFP expression is driven by xylA promoter , the 5'UTR has been modified to make it similar to RBS from a proD. <B>XylS-sfGFP</B> - a shortened form of the XylA1 promoter, since we have expected that catabolic repression should be weakened to enable the efficient induction of this promoter on standard media like LB.<B>XylF-sfGFP</B> - a DNA fragment containing only the XylF promoter with the original UTR of XylF gene upstream the sfGFP ORF.</P><P>After reamplification of fragments with overlaps, we performed CPEC, purification of the product, and electroporation.</P><P>XylS after transformation with DH5α:</P><P>XylS on minimum medium with glucose (left) and xylose (right):</P><P>After transformation with XylF we obtain only 3 colonies. That's how they looked on minimum media- XylF is pretty weak- again glucose on the left and xylose on the right:</P><A name="July2nd"><H4>2<SUP>nd</SUP> week</H4></A><P>We inoculated XylS and XylF on M9 minimum medium with 0,01%, 0,1%, 0,25%, 0,5% and 1% of xylose and incubated the culture for 44,5h in 37°C. These are the results under UV light:</P><P>XylS is much stronger than XylF, and 0,25% of xylose in M9 medium is enough for a good expression in longer (44,5h in this example) culture.</P><P>We also checked Arashort1 and Arashort2 on M9 medium with 0,01%, 0,1%, 0,25%, 0,5% and 1% of arabinose (44,5h, 37°C): </P><P>And RhaWT and Rha without <I>EcoR</I>I - 0,01%, 0,1%, 0,25%, 0,5%, 1% of arabinose (44,5h, 37°C): </P><P>We can see that proRha1 without <I>EcoR</I>I seems to be stronger than RhaWT.</P><P>The fluorescence of all the previous samples was measured using Denovix  (see Methods).</P><A name="July3rd"><H4>3<SUP>rd</SUP> week</H4></A><P>We prepared bacterial glycerol stocks with RhaWT, proRha1 without <I>EcoR</I>I, proRha2, AraWT, Arashort1, Arashort2, XylS, XylF and proD. We started with transformation of <I>E.coli</I> with desired constructs. After that we picked the best colonies from the plates and inoculated them into overnight LB liquid medium. We centrifuged the cultures, resuspened the pellet in LB liquid medium, added glycerol (final concentration: 7%) and the antibiotic (chloramphenicol). We put stocks prepared in that way into the freezer (-20°C). (See Methods)</P><A name="July4th"><H4>4<SUP>th</SUP> week</H4></A><P>We prepared M9 minimum media with:

<UL><LI>0,25% of arabinose</LI><LI>0,5% of arabinose</LI><LI>0,5 % of arabinose+glucose</LI></UL>

and used them to examine the activity of AraWT, Arashort1 and Arashort2;
<UL><LI>0,25% of rhamnose</LI><LI>0,5% of rhamnose</LI><LI>0,5% of rhamnose+glucose</LI></UL>
and used them to examine the activity of RhaWT, Rha without <I>EcoR</I>I and proRha2.</P><P>We  prepared cultures with bacteria that carry desired constructs in liquid LB medium, so that we can use them to inoculate M9 media the next day. After inoculation we collected samples from all the cultures every 1hour, 8 times (so all the cultures lasted 8 hours) + one sample from 0h (to measure fluorescence at the beginning of the culture). </P><A name="August"><H2>August</H2></A><A name="August1st"><H4>1<SUP>st</SUP> week</H4></A><P>Once again we put bacteria with proC promoter into M9 medium with different sugars (0,5%), we also checked proC with thermometer and proD in that way. In order to start the culture we inoculated M9 media with the proper amount of bacteria, so that the final OD<SUB>600</SUB> was about 0,1. </P><P>(OD600 measurement was performed using Denovix spectrophotometer) </P><P>
After 24h we centrifuged the samples, resuspend the pellet and measured the fluorescence. 
The fluorescence of all the samples was measured using Denovix (see Methods).</P><P>We performed the assembly of melibiose promoters: </P><UL><LI>Mel-sfGFP-WT- without PstI site (biobrick standards)</LI><LI>Mel2-sfGFP- which is a shortened version; without PstI site (biobrick standards) and with changed 5'UTR- we exchanged it on 5'UTR that comes from the rhamnose promoter, it should position RBS element better. We also eliminated potential secondary structures, that could mask RBS.</LI></UL><P>Steps of the assembly: amplification with specially designed primers that add overlaps, the assembly with CPEC reaction, purification and bacteria transformation.
We picked the best colonies and inoculated them on M9 minimal medium with 0,4% of glucose (control) and melibiose (inducer).</P><P>Melibiose promoters are working- they are induced by melibiose, and there is no expression on glucose.</P><A name="August2nd"><H4>2<SUP>nd</SUP> week</H4></A><P>Once again (third time) we inoculated proC, proC with thermo and proD into 250μl M9 with 0,5% of different sugars in 5ml Eppendorf tubes. We should incubate them in 37°C for 24h, but they grew so fast, that we had to centrifuge them after 15h. The fluorescence of all the samples was measured using Denovix (see Methods).</P><P>We inoculated:</P><UL><LI>AraWT, Arashort1 and Arashort2 on M9 with: 0,5% arabinose, 0,25%arabinose + 0,25% glucose (to check how much glucose stops the expression of sfGFP under arabinose promoters); for the culture in M9 with sugar combinations we prepared 3 samples;</LI><LI> proRha1 (without <I>EcoR</I>I) and Rha2 on M9 with: 0,5% rhamnose, 0,25% rhamnose + 0,25% glucose (to check how much glucose stops the expression of sfGFP under rhamnose promoters); for the culture in M9 with sugar combinations we prepared 3 samples.</LI></UL><A name="August3rd"><H4>3<SUP>rd</SUP> week</H4></A><P>We inoculated XylWT and XylA1 into overnight liquid LB medium, so that we can make stocks the next day. We also made M9 minimum media with: 0,5% of xylose, 0,25% of xylose, 0,5% of arabinose, 0,5% of glucose, 0,5% of rhamnose.  
We inoculated all of our promoters into M9 with all sugars (0,5%), in order to check, if they are tight (so the expression is only on their inducers, not on other carbon sources), or if they tend to leak. That's a lot of tubes to label!</P><P>The samples were incubating in 37°C for 24h, next we centrifuged them, resuspend the pellet in 100μl of water and measured them on Tecan fluorometer (using special 96-well plate; see Methods). 
On Tecan we also measured 8h-cultures of arabinose and rhamnose promoters from last week.</P><A name="August4th"><H4>4<SUP>th</SUP> week</H4></A><P>In order to compare our promoters with pET system (that uses T7 polymerase) we inoculated all of them into M9 with 0,5% of glucose. We also inoculated T7 on M9 with 0,5% of all sugars. We found out, that T7 (that should be induced by lactose/IPTG) shows sfGFP expression on a high level on all carbon sources, whilst our promoters are tight. Conclusion? T7 leaks, a lot. </P><P>The fluorescence of those samples was measured on Tecan fluorometer.</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>