<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UAM_Poznan_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UAM Poznan/Experiments</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team UAM Poznan Wiki - Methods page</TITLE><A href="#" class="nav-toggle">Toggle Navigation</A><NAV class="cmn-tile-nav"><UL class="clearfix"><LI class="colour-1"><A href="DNA quantitation">DNA quantitation</A></LI><LI class="colour-2"><A href="#DNA amplification">DNA amplification</A></LI><LI class="colour-3"><A href="#Gel electrophoresis">Gelelectrophoresis</A></LI><LI class="colour-4"><A href="#Elution of DNA fragments from agarose gel">DNA elution</A></LI><LI class="colour-5"><A href="#Chromosomal DNA isolation">DNA isolation</A></LI><LI class="colour-6"><A href="#Construct and part design">Construct design</A></LI><LI class="colour-7"><A href="#Construct assembly by CPEC">CPEC assembly</A></LI><LI class="colour-8"><A href="#Media and bacterial cultures">Cultures and media</A></LI><LI class="colour-9"><A href="#Preparation of electro-competent cells">Electro-competent <I>E. coli</I></A></LI><LI class="colour-10"><A href="#Electroporation">Electroporation of <I>E.coli</I></A></LI><LI class="colour-11"><A href="#Plasmid isolation and purification">Plasmid isolation</A></LI><LI class="colour-12"><A href="#Construct verification">Construct verification</A></LI></UL></NAV></P><H2>DNA quantitation</H2><P>To quantitate plasmid DNA and PCR products, absorbance spectrum and A<SUB>260</SUB> was measured on NanoDrop spectrophotometer . Quantitation was verified by gel electrophoresis in 1-1,5% agarose gels in TAE buffer, pre-stained with ethidium bromide (100ng/ml). By gel image analysis we were also able to check the purity of plasmid DNA and quality of PCR products. </P><A name="DNA amplification"><H2>DNA amplification</H2></A><P>Primers for PCR were designed using <B>Primer 3</B> [<A href="http://nar.oxfordjournals.org/content/40/15/e115.full" target="_blank">1</A>,<A href="http://bioinformatics.oxfordjournals.org/content/23/10/1289.long" target="_blank">2</A>] to have the same calculated melting temperature of 60°C. This way fragments of similar sizes can be amplified at the same time on one thermal cycler. </P><TABLE><CAPTION>Preparation of a standard PCR mix</CAPTION><TBODY><TR><TD>5xHF buffer (NEB)</TD><TD>10μl</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>33μl</TD></TR><TR><TD>dNTP mix (10mM each)</TD><TD>1μl</TD></TR><TR><TD>Primer forward (100μM)</TD><TD>0,25μl</TD></TR><TR><TD>Primer reverse (100μM)</TD><TD>0,25μl</TD></TR><TR><TD>Plasmid template (0,1-5ng)</TD><TD>5μl</TD></TR><TR><TD>Phusion DNA polymerase (NEB)</TD><TD>0,5μl</TD></TR></TBODY></TABLE><TABLE><CAPTION>Standarized PCR conditions</CAPTION><TBODY><TR><TH>Temperature</TH><TH>Duration</TH><TH>Number of cycles</TH></TR><TR><TD>98°C</TD><TD>2min.</TD><TD>1</TD></TR><TR><TD>98°C</TD><TD>20s</TD><TD rowspan="3">22-26</TD></TR><TR><TD>60°C</TD><TD>20s</TD></TR><TR><TD>72°C</TD><TD>30s/0,5kb</TD></TR><TR><TD>72°C</TD><TD>7min.</TD><TD>1</TD></TR><TR><TD>8°C</TD><TD>∞</TD><TD>-</TD></TR></TBODY></TABLE><A name="Gel electrophoresis of DNA fragments"><H2>Gel electrophoresis of DNA fragments</H2></A><OL><LI>Weigh 0,6g of agarose and pour into a 250 ml screw cap Simax bottle.</LI><LI>Add 50ml of H<SUB>2</SUB>O<SUB>DEPC</SUB> and 1 ml of 50xTAE buffer, mix. </LI><LI>Heat up for 2 minutes in microwave oven until the agarose sol is homogeneous. Cool the bottle shortly under tap water. Add 0,5 μl ethidium bromide (10mg/ml). Mix well.</LI><LI>Pour the agarose sol into a gel casting tray. Insert the appropriate comb. If necessary, clear the bubbles with a pipette tip. Wait until the gel is fully solidified.</LI><LI>Pour TAE buffer onto the gel, remove the comb and casting gates. Pour TAE buffer to buffer tanks to cover the gel with 1mm of the buffer.</LI><LI>Mix the samples with loading dye in a 5:1 ratio. Put the samples into the wells. We used two DNA molecular weight markers: 100 bp DNA Ladder and 1 kb DNA Ladder (NEB).</LI></OL><A name="Elution of DNA fragments from agarose gel"><H2>Elution of DNA fragments from agarose gel</H2></A><P>After electrophoresis, agarose gels were photographed on UV transiluminator (SynGene) and
DNA fragments of interest were excised with a clean razor blade and placed in weighed Eppendorf tubes . DNA fragments were purified on Zippy silica columns, according to the protocol provided by <A href="http://www.zymoresearch.com/" target="_blank">ZYMO RESEARCH</A>.</P><A name="Chromosomal DNA isolation"><H2>Chromosomal DNA isolation from <I>E.coli</I></H2></A><OL><LI>Spin down 0,5 ml of E. coli culture OD=0,6 for 3 min. at 5000g and remove the supernatant.</LI><LI>Suspend the pellet in 200μl of buffer A(150 mM NaCl,100 mM EDTA pH=8,0).</LI><LI>Add 10 μl of fresh lysozyme suspension (10 mg/ml), incubate at . 37°C for 15 min.</LI><LI>Add 20 μl 10% SDS and 1μl proteinase K, mix well, incubate at 56°C for 15 min.</LI><LI>Cool down the mixture to RT, add 100 μl chloroform and 100 μl phenol pH=8,0, vortex for 1 min. Centrifuge for 5 min. at 12K.</LI><LI>Carefully transfer the upper water phase containing nucleic acids to a new tube, add equal volume of ethanol, mix well and centrifuge as above.</LI><LI>Remove the supernatant and wash the pellet twice with 70% ethanol, spin down for 1 min. at 12K, remove the supernatant carefully to leave the pellet only.</LI><LI>Add 50 μl H<SUB>2</SUB>O, incubate at 56°C for 2 min., mix well by pipetting.</LI><LI>Check the quality on 0,8% agarose gel. The main DNA band should represent molecules longer than 20 kb.</LI><LI>Use 0,1- 0,5 μg of chromosomal DNA to amplify a gene of interest. </LI></OL><P>All promoter sequences we use derive from the chromosomal DNA of E. coli DH5α.</P><A name="Construct and part design"><H2>Construct and part design</H2></A><P>Constructs were designed by using the NEBuilder with a minimal length of overlaps of 18 bp.
Parts of constructs were designed to be easily amplified with primers of the same melting temperature of 60°C. These PCR products of expected size were excised from agarose gels and reamplified with a pair of primers containing sequences of overlaps, and again purified from agarose gels. The initial primers we use to verify the structure of the final construct, as well as for sequencing. Verification of construct structure by Sanger sequencing was performed in our Faculty sequencing lab.</P><A name="Construct assembly by CPEC"><H2>Construct assembly by CPEC (circular polymerase extension cloning)</H2></A><P>The purified PCR products containing overlaps were quantitated on nanodrop spectrophotometer. Sizes of these DNA fragments to be mixed were checked by electrophoresis in agarose gels. Then fragments were mixed to obtain the molar ratio, like: 
2(promoter) : 2(CDS) : 1(vector). 
Optimal amounts of DNA fragments were as in example:
Promoter (300bp) 60ng : CDS (1000bp) 200ng: PSB1C3 vector (2070bp) 200ng.</P><TABLE><CAPTION>Preparation of a standard CPEC mix</CAPTION><TBODY><TR><TD>5xHF buffer(NEB)</TD><TD>10μl</TD></TR><TR><TD>Mixture of fragments with overlaps</TD><TD>X μl</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>X μl</TD></TR><TR bgcolor="#BFD4FF"><TD>Total</TD><TD>48,5μl</TD></TR><TR><TD>dNTP mix (25mM)</TD><TD>1μl</TD></TR><TR><TD>Phusion DNA polymerase (NEB)</TD><TD>0,5μl</TD></TR></TBODY></TABLE><TABLE><CAPTION>Standarized CPEC conditions</CAPTION><TBODY><TR><TH>Temperature</TH><TH>Duration</TH><TH>Number of cycles</TH></TR><TR><TD>98°C</TD><TD>30s</TD><TD>1</TD></TR><TR><TD>98°C</TD><TD>15s</TD><TD rowspan="3">26 cycles</TD></TR><TR><TD>45°C</TD><TD>3min. R20%</TD></TR><TR><TD>72°C</TD><TD>30s R5%</TD></TR><TR><TD>72°C</TD><TD>7min.</TD><TD>1</TD></TR><TR><TD>4°C</TD><TD>∞</TD><TD>-</TD></TR></TBODY></TABLE><P>Purification of the CPEC products before electroporation:</P><OL><LI>Add 150μl H<SUB>2</SUB>O to the sample after CPEC, 100μl of chloroform and 100μl of phenol (pH=8,0)/ vortex and centrifuge 3 min x 12000 rpm.</LI><LI>Collect the upper phase and transfer it to new eppendorf tube, add 100μl of chloroform, vortex and centrifuge as in the previous step.</LI><LI>Collect the upper phase and transfer it to new DNA LoBindTube.</LI><LI>Add 2μl of glycoblue, mix, add 1/10 V of 3M sodium acetate (pH=4,8), mix well, and 2,5-3 vol. of 99% ethanol and mix again.</LI><LI>Cool at -20°C for at least 2 h. Centrifuge 40 min x 12000 rpm at 4°C. Discard the supernatant and suspend the blue pellet in 20 μl of sterile H<SUB>2</SUB>O. Use 7 μl for electroporation.</LI></OL><A name="Media and bacterial cultures"><H2>Media and bacterial cultures</H2></A><TABLE><CAPTION>LB medium (1l liquid)</CAPTION><TBODY><TR><TD>Tryptone</TD><TD>10g</TD></TR><TR><TD>NaCl</TD><TD>10g</TD></TR><TR><TD>Yeast extract</TD><TD>5g</TD></TR></TBODY></TABLE><P>To make solid LB medium add 1,5g of agar for each 100ml of liquid. Sterilize by autoclaving.</P><TABLE><CAPTION>5xM9 salts (500ml)</CAPTION><TBODY><TR><TD>Na<SUB>2</SUB>HPO<SUB>4</SUB> x 2H<SUB>2</SUB>O</TD><TD>21,26g</TD></TR><TR><TD>KH<SUB>2</SUB>PO<SUB>4</SUB></TD><TD>7,5g</TD></TR><TR><TD>NaCl</TD><TD>1,25g</TD></TR><TR><TD>NH<SUB>4</SUB>Cl</TD><TD>2,5g</TD></TR></TBODY></TABLE><P>Sterilize by autoclaving.</P><P>Plates with M9 minimal medium (200ml):</P><OL><LI>Melt 100ml of 3% agar in the microwave and transfer to a water bath 50°C. </LI><LI>Add 53ml of sterile water preheated to 50°C.</LI><LI>Add 40ml of 5xM9 salts, kept at the same temp.</LI><LI>Add 20μl of warm 1M CaCl2 and mix quickly.</LI><LI>Add 0,4ml of warm 1M MgSO4 and mix well.</LI><LI>Add 4ml of 20% filter sterilized solution of glucose or another sugar ( final conc. 0,4%).</LI><LI>Add 2ml of 1% thiamine (filter sterilized), mix well. </LI><LI>Add proper antibiotic, mix and pour on plates (2,5-3mm). </LI></OL><A name="Preparation of electro-competent cells"><H2>Preparation of electro-competent <I>E.coli</I> cells</H2></A><OL><LI>Grow a single colony of E. coli DH5α into a 250ml flask with 100ml of LB medium overnight, at 120 rpm, at 30-37°C.</LI><LI>Transfer 0,5ml of the culture to the new 250ml flask with 100ml of LB medium and grow the culture at 37°C to get OD<SUB>600</SUB>=0,5 or 0,6.</LI><LI>After 10-15 minutes of incubation on ice, centrifuge the culture at 4000 rpm for 20 min. at 4°C.</LI><LI>Suspend bacterial pellet in 100ml of deionized water and centrifuge as above. Repeat this step one more time.</LI><LI>Suspend the cells in 40ml of ionized water with 1% of glycerol in 4°C and centrifuge as before.</LI><LI>Suspend the pellet in 1-2ml of 10% glycerol in deionized water, split the suspension into 100μl portions and freeze in liquid nitrogen. </LI><LI>Store at -80°C, for up to 1 year.</LI></OL><A name="Electroporation"><H2>Electroporation of <I>E.coli</I></H2></A><OL><LI>Incubate a portion of electro-competent cells on ice for 5-10 min and let it to thaw slowly.</LI><LI>On ice mix 50-100μl of electro-competent cells with 2-4μl of plasmid DNA (10-50ng) or 7 μl of purified DNA from CPEC to obtain 100-500 colonies. </LI><LI>Transfer the mixture to the cold electroporation cuvette 0,2mm.</LI><LI>Perform the electroporation at 2,5 kV. Pulse time should be close to 5ms.</LI><LI>Immediately add 800μl of LB medium, slowly mix and transfer to a new tube.</LI><LI>Incubate at 37°C for 30-45 min. with mixing of about 350 rpm.</LI><LI>Spread 50-100μl of the bacterial culture onto the 90mm plate with solid LB medium with the proper antibiotic.</LI><LI>Incubate at 37°C 16-30h.</LI></OL><A name="Plasmid isolation and purification"><H2>Plasmid isolation and purification</H2></A><OL><LI>Transfer 50 ml of bacterial culture grown overnight in LB medium at 37°C to a conical tube and centrifuge at 4000 - 9000 rpm at 4°C. Carefully drain off the medium. The pellet can be stored at 20°C.</LI><LI>Suspend bacterial pellet in 6 ml of TEG buffer.</LI><LI>Add 6 ml of Buffer II and mix inverting the tube. Leave the mixture for no longer than 2 min. to obtain a clear lysate.</LI><LI>Add 6 ml of Buffer III KAc, vortex quickly and centrifuge at 9000 - 12000 rpm at 4°C.</LI><LI>Transfer the supernatant to a new tube.</LI><LI>Add 0,6 – 0,7 V of isopropanol, mix and put into the fridge for 30 min.</LI><LI>Centrifuge at 9000 - 12000 rpm for 30 min. at 4°C.</LI><LI>Discard the supernatant, suspend the pellet in 800μl of miliQ water, add 5μl of RNase.</LI><LI>Incubate at 37°C for 15 min. or keep in the fridge for overnight.</LI><LI>Add equal vol. of phenol, vortex, centrifuge 3 min x 12000 rpm RT. Collect the upper phase and repeat this step.</LI><LI>Add one vol. of chloroform, vortex, centrifuge and collect the upper phase to a new Eppendorf tube.</LI><LI>Add 1/10 vol. of 3M sodium acetate pH=4,8, mix, and 0,6-0,7 V of isopropanol, mix well. </LI><LI>Centrifuge 3 min x 12000 rpm RT. Discard the supernatant.</LI><LI>Douche 150μl of 75% ethanol , centrifuge and discard the supernatant.</LI><LI>Dry the pellet for 2 min.</LI><LI>Suspend the pellet in 50-100μl of water.</LI></OL><P>Optionally, starting from step 9:</P><OL start="9"><LI>Just proceed to <A href="https://www.zymoresearch.com/dna/plasmid-dna-purification/zyppy-plasmid-miniprep-kit" target="_blank">Zippy Plasmid Miniprep Kit</A> procedure starting from step 1. One silica column will bind up to 50μg of plasmid DNA at one binding session  so one can purify 200 μg of high quality plasmid DNA on one minicolumn.</LI><LI>After checking the quality by agarose gel electrophoresis and A<SUB>260</SUB> measurement, the plasmid DNA is ready for sequencing and RE digestions, cloning, and transfections.</LI></OL><TABLE><CAPTION>Buffer TEG</CAPTION><TBODY><TR><TD>Glucose (1M), filter-sterilized</TD><TD>1,5ml</TD></TR><TR><TD>Na<SUB>2</SUB>EDTA pH=8,0 (250mM)</TD><TD>6ml</TD></TR><TR><TD>Tris-HCl pH=8,0 (1M)</TD><TD>0,75ml</TD></TR><TR><TD>RNase A (10mg/ml)</TD><TD>0,3ml</TD></TR><TR><TD>H<SUB>2</SUB>O Mili-Q</TD><TD>41,45ml</TD></TR><TR bgcolor="#BFD4FF"><TD>Total volume</TD><TD>50ml</TD></TR></TBODY></TABLE><P>Store at 20°C or (without RNase) at room temperature.</P><TABLE><CAPTION>Buffer NaOH/SDS</CAPTION><TBODY><TR><TD>NaOH (10M)</TD><TD>1ml</TD></TR><TR><TD>SDS (10%)</TD><TD>5ml</TD></TR><TR><TD>H<SUB>2</SUB>O Mili-Q</TD><TD>44ml</TD></TR><TR bgcolor="#BFD4FF"><TD>Total volume</TD><TD>50ml</TD></TR></TBODY></TABLE><P>Store tightly closed at room temperature up to 2 days.</P><TABLE><CAPTION>Buffer KAc (neutralizing)</CAPTION><TBODY><TR><TD>Glacial acetic acid</TD><TD>14ml</TD></TR><TR><TD>Potassium acetate (5M)</TD><TD>72ml</TD></TR><TR><TD>H<SUB>2</SUB>O Mili-Q</TD><TD>14ml</TD></TR><TR bgcolor="#BFD4FF"><TD>Total volume</TD><TD>100ml</TD></TR></TBODY></TABLE><P>Store at 4°C or at room temperature.</P><A name="Construct verification"><H2>Construct verification by colony PCR and Sanger sequencing</H2></A><OL><LI>Pick up several colonies of interest with a sterile toothpick and streak them on one or two new selective plates with numbered fields. Wipe the remaining part of the colony on the bottom of a 1,5 ml Eppendorf tube. Use the same numbering for tubes as for fields on plates. </LI><LI>Add 50-100μl of H<SUB>2</SUB>O to each tube, incubate at 96°C for 10 min. Cool down by centrifugation for 2 min. at 12K. Transfer 2μl to a new PCR tube.</LI><LI>Use Taq DNA polymerase for colony PCR. Prepare a standard PCR mix leaving 2 μl of the total volume for the template. Consider an additional volume for a positive control, like just an empty plasmid.</LI><LI>Add 48μl of PCR mix to every tube. Prepare a positive control. Run a standard PCR. </LI><LI>Check the sizes of amplicons by gel electrophoresis in agarose gel. </LI><LI>Sanger sequencing of constructs was performed in our Faculty facility on ABI Prism capillary system. We provided plasmids which were purified as described above (11) with VF and VR primers and primers used for amplification of parts of our constructs – these shorter ones, without overlaps. To obtain better results, i.e. longer sequences we purified some primers by electrophoresis in 15% polyacrylamide gel with 7M urea. </LI></OL><H3>References:</H3><OL><LI><A href="http://nar.oxfordjournals.org/content/40/15/e115.full" target="_blank">Koressaar T1, Remm M. (2007) Enhancements and modifications of primer design program Primer3. Bioinformatics 23(10):1289-91.</A></LI><LI><A href="http://bioinformatics.oxfordjournals.org/content/23/10/1289.long" target="_blank">Untergasser A1, Cutcutache I, Koressaar T, Ye J, Faircloth BC, Remm M, Rozen SG. (2012) Primer3—new capabilities and interfaces. Nucleic Acids Res. 40(15):e115.</A></LI></OL></DIV></DIV></DIV></DIV></BODY></HTML>