<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BioIQS-Barcelona_Standardization skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BioIQS-Barcelona/Standardization</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>BIO IQS</TITLE><BUTTON id="myBtn" class="js-scroll-trigger" title="Go to top">Top</BUTTON></P><DIV class="container h-100"><DIV class="row h-100"><DIV class="col-lg-7 my-auto"><DIV class="header-content"><H1 class="mb-5">Dry Lab | PCR Standardization</H1><A href="https://2018.igem.org/Team:BioIQS-Barcelona/Standardization#first-steps" class="btn btn-outline btn-xl js-scroll-trigger">Have a look!</A></DIV></DIV></DIV></DIV><BUTTON onclick="topFunction()" id="myBtn" class="js-scroll-trigger" title="Go to top">Top</BUTTON><SECTION class="first-steps text-center" id="first-steps"><DIV class="container"><DIV class="row"><DIV class="col-md-9 mx-auto"><H2 class="section-heading orange">First steps</H2><DIV class="col-md-auto"><H4 class="book orange block-sept comas" id="eliminateOverflowcommas"><I>The first step to develop our sensor is to amplify, express and effectively obtain the HLA-DQ heterodimer (chains α and β) from the patient's DNA, regardless of the CD-haplotype of the patient (either DQ2 or DQ8).</I></H4></DIV></DIV></DIV></DIV></SECTION><SECTION class="st-description block-sept" id="st-description"><DIV class="container"><DIV class="row"><DIV class="col-md-12 mx-auto"><DIV class="row"><DIV class="col-md-6 right center"><DIV class="row block-sept"><DIV class="col-md-12"><P class="book orange">Based on former studies, only the exons 2 and 3 from each chain codify for the extracellular domain of the HLA-DQ that interacts with the reactive gluten epitopes. With this in mind, we designed a robust model for the extraction of the α and β chains of the HLA-DQ from the genomic DNA of a celiac patient. A set of primers were designed to conduct 3 different PCR steps (in a total of 10 reactions) to obtain the α- and β-chains flanked with restriction sites for further cloning.</P></DIV></DIV></DIV><DIV class="col-md-12 mx-auto"><DIV class="row justify-content-center block-sept"><DIV class="col-md-8"><P class="book orange">Therefore, we designed 8 specific primers (forward and reverse) to amplify the exons 2 and 3 of each chain. The nomenclature is as follows: P1 to P4 for the α-chain and P1’ to P4’ for the β-chain.</P></DIV></DIV><DIV class="taulacarta"><TABLE class="table book text-b"><THEAD class="orange-intense"><TR><TH scope="col">Primer</TH><TH scope="col">Chain</TH><TH scope="col">Exon</TH><TH scope="col">Genomic Location*</TH><TH scope="col">Length (bp)**</TH></TR></THEAD><TBODY class="orange-medium"><TR><TD data-label="Primer">P1</TD><TD data-label="Chain">A</TD><TD data-label="Exon">2 (5’-&gt;3’)</TD><TD data-label="Genomic Location*">4643-4664 bp</TD><TD data-label="Length (bp)**">22</TD></TR><TR><TD data-label="Primer">P2</TD><TD data-label="Chain">A</TD><TD data-label="Exon">2 (3’-&gt;5’)</TD><TD data-label="Genomic Location*">4867-4889 bp</TD><TD data-label="Length (bp)**">23</TD></TR><TR><TD data-label="Primer">P3</TD><TD data-label="Chain">A</TD><TD data-label="Exon">3 (5’-&gt;3’)</TD><TD data-label="Genomic Location*">5302-5326 bp</TD><TD data-label="Length (bp)**">25</TD></TR><TR><TD data-label="Primer">P4</TD><TD data-label="Chain">A</TD><TD data-label="Exon">3 (3’-&gt;5’)</TD><TD data-label="Genomic Location*">5561-5583 bp</TD><TD data-label="Length (bp)**">23</TD></TR><TR><TD data-label="Primer">P1'</TD><TD data-label="Chain">B</TD><TD data-label="Exon">2 (5’-&gt;3’)</TD><TD data-label="Genomic Location*">2072-2097 bp</TD><TD data-label="Length (bp)**">26</TD></TR><TR><TD data-label="Primer">P2'</TD><TD data-label="Chain">B</TD><TD data-label="Exon">2 (3’-&gt;5’)</TD><TD data-label="Genomic Location*">2436-2456 bp</TD><TD data-label="Length (bp)**">21</TD></TR><TR><TD data-label="Primer">P3'</TD><TD data-label="Chain">B</TD><TD data-label="Exon">3 (5’-&gt;3’)</TD><TD data-label="Genomic Location*">5199-5226 bp</TD><TD data-label="Length (bp)**">28</TD></TR><TR><TD data-label="Primer">P4'</TD><TD data-label="Chain">B</TD><TD data-label="Exon">3 (3’-&gt;5’)</TD><TD data-label="Genomic Location*">5504-5527 bp</TD><TD data-label="Length (bp)**">24</TD></TR></TBODY></TABLE></DIV><P class="book orange-medium n-p"><SMALL>* All genomic locations are referred to the HLA DQA1*05:01:01:01 and DQB1*02:01:01, chain A and B respectively.</SMALL></P><P class="book orange-medium n-p"><SMALL>** Complementary primer overhangs are not taken into account for the analysis.</SMALL></P></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION id="default"><DIV class="container"><DIV class="row"><DIV class="col-md-12 mx-auto"><DIV class="row justify-content-center text-center"><DIV class="col-md-7"><H4 class="orange">Since we started the project focusing on expressing HLA-DQ from Roger’s DNA sample, primers were designed accordingly to his HLA-DQ haplotype, which is DQ2 (HLA DQA1*05:01:01:01 and DQB1*02:01:01). Remember that, for celiac disease, the haplotype nomenclature is as follows:</H4></DIV></DIV><DIV class="row block-desc"><DIV class="col-md-4"><H4 class="orange-intense">DQ2-positive</H4><P class="orange book">(HLA-DQA1*05:01 or *05:05 <B>and</B> HLA-DQB1*02:01 or *02:02)</P></DIV><DIV class="col-md-4"><H4 class="orange-intense">Half DQ2-positive</H4><P class="orange book">(HLA-DQA1*05:01 <B>or</B> 05:05 or HLA-DQB1*02:01 or 02:02)</P></DIV><DIV class="col-md-4"><H4 class="orange-intense">DQ8-positive</H4><P class="orange book">(HLA-DQA1*03 <B>and</B> HLA-DQB1*03:02)</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 mx-auto"><P class="book orange">In order to elucidate if the designed primers are capable of amplifying every CD-associated genotype, a multiple sequence alignment (MSA) was performed to evaluate genetic variability at primer regions.</P><P class="book orange">Genomic HLA-DQ sequences <A class="link" href="https://github.com/Bioiqs-iGem/HLA-DQ" target="_blank">(DQA1_gen.fasta and DQB1_gen.fasta)</A> were downloaded from IPD-IMGT/HLA (Directory hosted at the European Bioinformatics Institute) version 3.33.0. Then, the genomic sequences were curated for DQ2 and DQ8 haplotypes.</P></DIV></DIV><DIV class="taulacarta"><TABLE class="table book text-b"><TBODY class="orange-medium"><TR><TH class="orange-intense cutthis">Haplotype</TH><TH class="orange-intense cutthis">Downloaded sequences</TH><TH class="orange-intense cutthis">DQ2</TH><TH class="orange-intense cutthis">DQ8</TH></TR><TR><TD data-label="Haplotype" class="orange-intense">DQA1</TD><TD data-label="Downloaded Sequences">73</TD><TD data-label="DQ2">14</TD><TD data-label="DQ8">7</TD></TR><TR><TD data-label="Haplotype" class="orange-intense">DQB1</TD><TD data-label="Downloaded Sequences">198</TD><TD data-label="DQ2">6</TD><TD data-label="DQ8">13</TD></TR></TBODY></TABLE></DIV><DIV class="row block-desc"><DIV class="col-md-12 mx-auto"><P class="book orange">Genomic MSA surprisingly resulted on a perfect alignment for sequences sharing CD-DQ haplotypes. This means that, for primer regions, all DQ2-associated sequences match perfectly and the same happens on DQ8. Moreover, for P2, P3, P1’ and P4’, primer region alignment is perfect and so, there is no need to edit the standard primers.</P><P class="book orange-medium n-p smll">* Hidden Markov Model of P1 genomic primer region curated for DQ2 and DQ8 genotypes. In this case, there are only 3 differing bases among primer region. Since 14 DQ2 and 6 DQ8 sequences were aligned, dominant residues are those corresponding to DQ2 genotypes. DQ2 sequences presents C-C-T and DQ8 presents T-T-C.</P></DIV></DIV><DIV class="taulacarta"><TABLE class="table book text-b"><THEAD class="orange-intense"><TR><TH scope="col">Primer</TH><TH scope="col">Exon</TH><TH scope="col">Total seq.</TH><TH scope="col">DQ2</TH><TH scope="col">DQ8</TH></TR></THEAD><TBODY class="orange-medium"><TR><TD data-label="Primer">P1</TD><TD data-label="Exon">2 (5’-&gt;3’)</TD><TD data-label="Total seq.">21</TD><TD data-label="DQ2">GCTGACCACGTCGCCTCTTATG</TD><TD data-label="DQ8">GCTGACCATGTTGCCTCTTACG</TD></TR><TR><TD data-label="Primer">P2</TD><TD data-label="Exon">2 (3’-&gt;5’)</TD><TD data-label="Total seq.">21</TD><TD data-label="DQ2">CATTGGTAGCAGCGGTAGAGTTG</TD><TD data-label="DQ8">CATTGGTAGCAGCGGTAGAGTTG</TD></TR><TR><TD data-label="Primer">P3</TD><TD data-label="Exon">3 (5’-&gt;3’)</TD><TD data-label="Total seq.">21</TD><TD data-label="DQ2">AGGTTCCTGAGGTCACAGTGTTTTC</TD><TD data-label="DQ8">AGGTTCCTGAGGTCACAGTGTTTTC</TD></TR><TR><TD data-label="Primer">P4</TD><TD data-label="Exon">3 (3’-&gt;5’)</TD><TD data-label="Total seq.">21</TD><TD data-label="DQ2">CCCAGTGTTTCAGAAGAGGCTTG</TD><TD data-label="DQ8">CCCAGTGTTTCAGAAGAGGCTCA</TD></TR><TR><TD data-label="Primer">P1'</TD><TD data-label="Exon">2 (5’-&gt;3’)</TD><TD data-label="Total seq.">19</TD><TD data-label="DQ2">GAGGATTTCGTGTACCAGTTTAAGGG</TD><TD data-label="DQ8">GAGGATTTCGTGTACCAGTTTAAGGG</TD></TR><TR><TD data-label="Primer">P2'</TD><TD data-label="Exon">2 (3’-&gt;5’)</TD><TD data-label="Total seq.">19</TD><TD data-label="DQ2">TCCTCTGGGGTGGAACAAACG</TD><TD data-label="DQ8">TCAGCCGGGGTGGAACGAACA</TD></TR><TR><TD data-label="Primer">P3'</TD><TD data-label="Exon">3 (5’-&gt;3’)</TD><TD data-label="Total seq.">19</TD><TD data-label="DQ2">CCTATATCTTTCCCTGTCTGTTACTGCC</TD><TD data-label="DQ8">CC--TATCTTTCCCTGTCTGTTACTGCC</TD></TR><TR><TD data-label="Primer">P4'</TD><TD data-label="Exon">3 (3’-&gt;5’)</TD><TD data-label="Total seq.">19</TD><TD data-label="DQ2">CAATATCCCCTTACGCCACTCCAC</TD><TD data-label="DQ8">CAATATCCCCTTACGCCACTCCAC</TD></TR></TBODY></TABLE></DIV><P class="orange book">For P3’ primer, a two-base deletion is reported as <B>refSNP Cluster Report: rs756895762</B>. Therfore, this is the only primer that needs special attention, as it could compromise the capability to amplify the HLA-DQ genes by PCR of all possible celiac disease involved HLA-DQ genotypes.</P><P class="orange book">This are the Hidden Markov Models that represent the sequence conservation for the region complementary to each primer. For the α chain, the dominant bases correspond to <B>DQ2</B>, while for the β chain, the dominant bases correspond to DQ8. </P></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>