Team:BioIQS-Barcelona/Public Engagement
BIO IQS
Top
Outside the Lab | Education & Engagement
Have                         a look!
Publication in the News AIQS Magazine
We had the opportunity to present our project to the IQS                         Alumni Association in an interview that was published on the News AIQS magazine in September                         2018. Being present in the newspapers brings us the opportunity to reach more people.
You can read the entire interview here
Divulgation Activities in Barcelona
“The closer we get to people, the more they sympathize with the                             cause.”
Spain is the country that has more bars per inhabitant in the world.                             Therefore, if you ever want to catch people attention, you better go to any bar and present                             your project to the audience.
Since our project aims to ease celiac people life, we arranged                             divulgation activities over the summer in bars and coffee shops that compromise with                             scientific divulgation. It was a must that all bars had gluten-free meals in their menu.
“Learning by playing”
We offered games based on educational technology using Kahoot                             (game-based learning platform). We asked a set of questions to the audience:
What is gluten, a protein or a sugar? What is cross-contamination? Does                             gluten apports nutritional value to the food?
By answering this questions, we were expecting to clear up                             misunderstandings related to celiac disease. Over the project, we realized there are yet                             several myths associated with gluten-free food.
There is much work yet to do!
We also brought the audience the opportunity to know                         more about the iGEM competition and synthetic biology.
Beer                         Talks, Bio IQS Barcelona
Introducing gluten free food at the University
“Time to cook! The BioIQS Barcelona team changed the laboratory                                 for the kitchen for a while.”
We baked around fifty cakes, most of them gluten-free, for demonstrating                             that life without gluten is pretty tasty.
During two days we sold cakes, cookies and candies at the entry hall of                             our university. We also made a challenge that consisted of eating two pieces of cakes, one                             with gluten and the other one gluten-free, and students had to guess which one was suitable                             for celiac people.
Collaborations
We visited the iGEM UPF CRG Barcelona Team. Being able to share what we                             were investigating, how we managed to deal with deadlines and fundraising was such a good                             experience.
It took part in front of the PRBB (The Barcelona Biomedical Research                             Park), where the IGEM UPF CRG Team is working. Since this Research Park is placed in front                             of the beach, we took advantage of it and played volleyball towards the Mediterranean sea.
This meeting set the beginning of a new collaboration .                         Over the summer, our team designed a novel strategy to extract the alpha and beta chains of the                         HLA-DQ from a celiac patient's DNA using PCR. We invited them to test our protocol using a                         different DNA sample to determine whether our protocol was robust or not.
Sharing the IGEM experience
We had the opportunity to share our iGEM experience with undergraduate                             students of Biotechnology, Chemistry, and Bioengineering from IQS. We explained the satisfactions and struggles found when                             starting one project from scratch, how important it is to work together as a team, all the                             achievements, the new skills we have acquired and everything that we have learn from this                             project.
DNA information storage workshop
During the divulgation sessions in which we explained our personalized                             sensor project, we noticed that many people were surprised by the fact that the sensor would                             be built upon their DNA sequence.
As we decided to keep the focus of our education and public engagement                             activities on the necessities of the people who participated in it, we realized that we                             could do some kind of workshop to let them know the information storage capability of DNA.                             This is how the DNA information storage workshop was born. You can check the script created                             to encode a written message into DNA .
Natural information on DNA:
We prepared a postcard to explain how many information                                         is stored in the genome of a single cell or in a single human body. To give                                         sense to that large amount of data we calculated the quantity of DIN-A4 papers                                         that would be needed in order to write all the DNA sequence of the genome of a                                         single cell in a standard font (Courier New, 12) and standard margins. This                                         way, everyone without biology or computation* knowledge could also imagine the                                         quantity of information.
*(sometimes the quantity of DNA information is expressed in bytes, which is not suitable for the general population, especially                                         older people). It turns out that a normal somatic human female cell has 2                                         dotations of 23 chromosomes (including 2 X sex chromosomes) and mitochondrial                                         DNA. This is a total of two times 3,088,286,401 base pairs, according to the                                         latest human genome sequence release (GRCh38.p12). A single DINA-4 page full of                                         characters in Courier New, 12 and standard margins could contain 2773 base                                         pairs written on it. Therefore, to write all the information stored in the base                                         pairs of the genome of a single human cell it would be needed to print                                         2,186,117 pages on one face. If these pages are then piled up to form a tower                                         (considering the height of a single paper is 0.1mm), it would be 218.61 meters                                         tall (717.2 ft)! Just between the Agbar Tower (the taller building in our city,                                         Barcelona; 142m) and the Eiffel Tower (324m). Scientists have estimated that the human body contains                                         nearly 37.2 trillions of human cells (note that this does not include the                                         microbiota). So...to write all the information condensed in the DNA of one                                         human body it would take a large amount of DINA-4 papers. This amount is so                                         huge that if the papers were arranged in a straight line, the light would spend 2553 years traveling from one end to the other. This is 2553 light-years long!                                         Keep in mind that the distance from the sun to the earth is just 8.3 light-minutes, so                                         imagine the amount of data stored in such a low volume like your body.
Using these examples, the general population can easily                                         understand the information storage capability of DNA. Nearly everyone                                         (including ourselves) was pretty amazed to realize how much information human                                         bodies contain stored in DNA. After this examples, we explained that our sensor                                         was based on a piece of information stored in two genes of the DNA. So this                                         comparisons helped the general population to understand that DNA contains much                                         useful information, which is different between us and that we can extract this                                         information to create a personalized product. This helped us to explain the DNA                                         extraction and PCR process of our project to the general population. This way, they                                         are much better informed about the project and can have an opinion about it.                                         Then, we integrated that informed opinion from the general population in the                                         design of our project. Dedicating effort in explaining the projects in a simple                                         but rigorous way can help teams to receive feedback from the target population.                                         This feedback can be incorporated into the design of the project and hence make                                         it more suitable for the real world.
Synthetic DNA as an information storage system:
Synthetic biologists engineer nature to create                                         solutions to real-world problems. We decided to incorporate some synthetic                                         biology into our DNA information storage workshop to expand the general                                         population knowledge about that field. If DNA is able to compact such a large                                         amount of data in a very small volume, why not use it as an information storage                                         system? - would a synthetic biology scientist think. Furthermore, digital                                         information is being generated at an increasing rate and the storage of this                                         information requires a huge amount of servers which take up a lot of space. So                                         why not using DNA to store digital data?
A single DNA base contains more information than one                                         bit. DNA is made from a combination of four different nucleotide residues                                         (A,T,C,G), while one bit can only take up to two values (0 or 1 in binary                                         code). So one DNA base pair could contain two binary code bits of information                                         (i.e. A=00, T=01, C=10, G=11). Therefore, theoretically, information is much                                         more condensed in DNA than in bits.
Actually, there are still several factors making DNA                                         information storage not as practical as current systems. However, the increase                                         in efficiency and the drop in the cost of DNA synthesis and sequencing has                                         dramatically increased the likelihood of using DNA as an information storage                                         system. For this reason, Church et al, 2012 encoded multiple copies of the book                                         he wrote, Regenesis, in DNA and then used next-generation sequencing to read                                         it. Instead of using the maximum compaction system that DNA offers (2 bits per                                         base), the authors used one bit per base (A or C for 0, G or T for 1). This has                                         multiple advantages, one of it being avoiding extreme GC content.
To integrate synthetic biology in our DNA storage                                         workshop we reproduced the work done by Church et al, 2012 to write our own DNA                                         messages . While the authors considered secondary structure, designed a block                                         system to order their data from DNA and encoded words, images, and a javascript                                         file, our system only considered GC content to write messages of a few words or                                         mp3 files in binary code. However, our system was perfectly usable for our                                         purpose. We created a bash script which takes a text input message and converts                                         it to binary by the ASCII code using the xxd command. Then, using a random                                         function, the program assigns a DNA nucleotide to each binary value (A or C to                                         0, G or T to 1). Finally, it checks if the GC content is between the specified                                         boundaries (by default, 40-60%). Thanks to the random function, the GC content                                         is usually between 50%. Only when encoding single ASCII characters the program                                         needs to perform more than one iteration to get the appropriate GC content. The                                         generated output consists of the written message in the input format, in binary                                         and in DNA. It also incorporates a title recreating the FASTA format used to                                         identify the DNA sequences (and in this case, the message). The program was also adapted to convert songs in mp3 format to DNA sequences, to convert                                         multiple text messages in the same file and to send the generated DNA messages                                         by mail. In addition, the program displayed in a very visual way how the                                         message was being encoded in binary and then in DNA.
We used the created program to convert several short text                                         messages about synthetic biology into DNA sequences. We then printed only this                                         DNA sequences with their title and including a guide on how this information is                                         encoded on the other face of the paper. We printed nearly 300 copies of the                                         messages and gave it to students in our university. The message contained                                         information about where to locate the DNA information workshop and an                                         invitation to come in order to decode their DNA message.
This way, we attracted the students to the workshop,                                         where we told them the information about natural information on DNA. In                                         addition, we organized some other small activities related to science and                                         celiac disease (i.e. a guess what contest, in which the participants tried to                                         identify the content of a science picture between two possible options). All                                         the students who came had their message decoded by us. We could quickly                                         identify the message by its title and write down the content of the message in                                         text format in their DNA message paper.
Finally, if the participants correctly identified more                                         than 2 of 3 scientific pictures, we gave them the opportunity to write their                                         own message and have it coded in DNA and sent to their mail by the program we                                         created. Most of the messages consisted of the name of the participants and                                         some student jokes. However, the final objective was to have fun while learning                                         about DNA information storage, so mission accomplished!
Furthermore, it was very satisfactory for us to see                                         the enthusiastic attitude of the students towards that workshop. It seems that                                         most of them find special the fact of having a personal message stored in a                                         DNA sequence which could be then synthesized, introduced to and replicated by a                                         living organism. The program we developed based on the coding system of Church                                         et al, 2012, helped the students to integrate into the topic by engaging on it                                         hands on. Like the examples provided before, this section of the workshop                                         helped translate a difficult concept in a simple but rigorous way to general                                         population. In addition, it served us as a tool to attract people to the                                         workshop and explain them a little bit more about iGEM, synthetic biology and                                         our project.
You can try to convert your own written messages into DNA using our program, available in our GitHub repository . To run it, you just need to download the repository as a zip file and extract its contents. Then, you should use a Unix ternimal to access the extracted folder and execute the script you need using the command "bash script_name.sh". If you are using MacOS or Linux you can do so using the default system terminal. If you are using Windows, you will need to activate Windows Subsystem for Linux or download a Unix terminal, like the Git Bash terminal .
Demonstration video
