<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BioIQS-Barcelona_DryLab skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BioIQS-Barcelona/DryLab</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>BIO IQS</TITLE><BUTTON id="myBtn" class="js-scroll-trigger" title="Go to top">Top</BUTTON></P><DIV class="container h-100"><DIV class="row h-100"><DIV class="col-lg-7 my-auto"><DIV class="header-content"><H1 class="mb-5">Dry Lab | Overview</H1><A href="https://2018.igem.org/Team:BioIQS-Barcelona/DryLab#modelling" class="btn btn-outline btn-xl js-scroll-trigger">Have a look!</A></DIV></DIV></DIV></DIV><SECTION class="modelling text-center" id="modelling"><DIV class="container"><DIV class="row"><DIV class="col-md-9 mx-auto"><H2 class="section-heading orange">Inner beauty</H2><DIV class="col-md-auto"><H4 class="book orange block-sept comas" id="eliminateOverflowcommas"><I>Despite what some people may think, DryLab
                                occasionally may be equal or more important for a project to develop properly than the
                                WetLab. Applying this to our project, we decided to study some important
                                aspects and evaluate their viability. More specifically, our work in the DryLab
                                consisted on:</I></H4></DIV></DIV></DIV></DIV></SECTION><SECTION class="st-description block-desc" id="cl-description"><DIV class="container"><DIV class="row"><DIV class="col-md-12 mx-auto"><DIV class="row"><H3 class="orange-intense">1st: Modeling</H3></DIV><DIV class="row block-desc-b"><DIV class="col-md-6 right"><P class="book orange">We had several ideas about how the sensor could work, and
                                eventually, one rose up strongly. The first step is having an idea. It may seem simple, but it can take long, as it goes through
                                think about the idea, understand it and rethink it as many times as it is required. Maybe this reflection
                                seems intuitive, but constitutes the fundamental part of all
                                developments. Using mathematical models and computer simulations we have been able to
                                understand the dynamics underlying our sensor. Furthermore, using this model we can tune a huge variety of parameters,
                                test many eventualities and predict the tendency of the sensor.</P></DIV><DIV class="col-md-6 left center"><DIV class="col-md-12 left"><P class="book orange">Generally speaking, the bottleneck or critical working point of our idea,
                                    was being able to build up a complex network of gold
                                    nano-particles. This would enable our sensor to have an easy optical detection system. Understanding
                                    the implications associated in that procedure makes the difference while facing the real design.
                                    How can we detect the interactions between the HLA-DQ protein and gluten-derived peptides? How can this detection be easy and visual? And finally, which are the physical dynamics pervading in the system?
                                    Go to the <A href="https://2018.igem.org/Team:BioIQS-Barcelona/Model">modeling section</A> if you wish to know the answer to all this questions. </P><DIV class="row block-desc-b"><DIV class="col-md-12 left"><DIV class="row block-sept center"><DIV class="col-md-7 button"><A class="text-transform" href="https://2018.igem.org/Team:BioIQS-Barcelona/Model" target="_blank"> Modeling</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><H3 class="orange-intense">2nd: PCRs standardization</H3></DIV><DIV class="row block-desc-b"><DIV class="col-md-6 right"><P class="book orange">One of the strengths of our approach to detect patient-specific
                                reactive gluten peptides in food is that we can produce the sensor using only a DNA
                                sample from the saliva of the patient, which is not invasive. Patient’s HLA-DQ (the protein responsible for
                                recognizing and triggering the immune response) is then expressed using the <A href="https://2018.igem.org/Team:BioIQS-Barcelona/Basic_Part">PCR protocol</A>
                                designed in the WetLab. For this purpose, we use specific primers to amplify the patient’s
                                target genes and thus, it is necessary to evaluate if they will be useful for all of the
                                different celiac-associated genotypes.</P></DIV><DIV class="col-md-6 left center"><DIV class="col-md-12"><P class="book orange">A multiple sequence alignment (MSA) on all annotated
                                    celiac-associated genomic sequence was performed to evaluate the conservation between genotypes of the specific
                                    primer-binding regions.</P><P class="book orange">The results demonstrate that this regions are perfectly
                                    conserved or only slightly different for both DQ2 and DQ8 celiac haplotypes.
                                    This means that either the same primer can be used for amplifying all genotypes or
                                    that a specific DQ2- or DQ8-primer has to be used. The possibility to use the same primers for multiple patients reduces the costs associated in the personalization factor.
                                    Thus, we've created a standard method to achieve the required personalization.</P><DIV class="row block-desc-b"><DIV class="col-md-12 left"><DIV class="row block-sept center"><DIV class="col-md-8 button"><A class="text-transform" href="https://2018.igem.org/Team:BioIQS-Barcelona/Standardization" target="_blank">PCRs
                                                    Standardization</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><H3 class="orange-intense">3rd: Molecular recognition</H3></DIV><DIV class="row block-desc-b"><DIV class="col-md-6 right"><P class="book orange">Another aspect that we wanted to evaluate is how the HLA-DQ protein
                                is able to effectively recognize and bind gluten-derived peptides. Specifically, which
                                are the residues that are mediating this recognition and if they are conserved among
                                celiac haplotypes DQ2 and DQ8.</P><P class="book orange">To do so, we focused the conservation study to the residues that
                                have been previously suggested to be responsible for the peptide recognition. A multiple alignment
                                of protein amino acid sequences was performed using all the annotated HLA-DQ proteins so far.</P></DIV><DIV class="col-md-6 left center"><DIV class="col-md-12"><P class="book orange">The results demonstrate that three different residues, two in the α chain and one in
                                    the β chain, are conserved among celiac patients and
                                    absent in healthy people. This mutated residues strictly need to be present in the HLA-DQ protein in order to molecularly
                                    recognize a gluten peptide and develop the disease. Therefore, we elucidated <A href="https://2018.igem.org/Team:BioIQS-Barcelona/Celiac_disease">which are the residues</A> of the HLA-DQ protein responsible for its interaction with the epitopes involved in celiac disease.</P><DIV class="row block-desc-b"><DIV class="col-md-12 left"><DIV class="row block-sept center"><DIV class="col-md-7 button"><A class="text-transform" href="https://2018.igem.org/Team:BioIQS-Barcelona/Celiac_disease" target="_blank">Molecular
                                                    recognition</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>