<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BioIQS-Barcelona_Composite_Part skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BioIQS-Barcelona/Composite Part</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>BIO IQS</TITLE><BUTTON id="myBtn" class="js-scroll-trigger" title="Go to top">Top</BUTTON></P><DIV class="container h-100"><DIV class="row h-100"><DIV class="col-lg-7 my-auto"><DIV class="header-content"><H1 class="mb-5">Wet Lab | Expression</H1><A href="https://2018.igem.org/Team:BioIQS-Barcelona/Composite_Part#first-steps" class="btn btn-outline btn-xl js-scroll-trigger">Have
                        a look!</A></DIV></DIV></DIV></DIV><BUTTON onclick="topFunction()" id="myBtn" class="js-scroll-trigger" title="Go to top">Top</BUTTON><SECTION class="first-steps text-center" id="first-steps"><DIV class="container"><DIV class="row"><DIV class="col-md-9 mx-auto"><H2 class="section-heading orange">Before starting</H2><DIV class="col-md-auto"><H4 class="book orange block-sept comas" id="eliminateOverflowcommas"><I>The HLA-DQ is a
                            transmembrane protein that is involved in celiac disease. It is formed by two subunits,
                            α and β, each one located in a different loci from the human genome. The
                            extracellular domain of each chain form a binding cleft that allows the interaction of
                            different peptides derived from antigens (in this case, gluten derived peptides). These
                            peptides usually have between 10 and 30 residues and this complex is presented to the for
                            recognition by CD4 T-cells. </I></H4></DIV></DIV></DIV></DIV></SECTION><SECTION class="st-description block-desc b-orange-intense" id="int-human"><DIV class="container"><DIV class="row"><DIV class="col-md-12 mx-auto"><DIV class="row"><DIV class="col-md-8 left"><DIV class="row block-sept vertical-lines"><DIV class="col-md-12"><H4 class="bold white"><U>Objectives</U></H4><P class="bold white">Our goal was to obtain the extracellular domains of each subunit
                                    through a genome fishing assay or by a de novo design of a synthetic construct.
                                    These constructs would then be cloned in a bacterial host (<I>Escherichia coli</I>)
                                    or a yeast host (<I>Pichia pastoris</I>) and protein expression would be studied.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 left"><DIV class="row block-sept vertical-lines"><DIV class="col-md-12"><H4 class="bold white"><U>Important notes for protein expression</U></H4><P class="bold white">1- In line with this strategy, there were some aspects that
                                    needed to be addressed when expressing the proteins. Firstly, the elimination of
                                    the transmembrane domain could increase the expression yield and the solubility
                                    since the resulting protein is smaller. However, these changes could also affect
                                    negatively to the interaccion affinity between the two chains. Additionally, high
                                    expression vectors, such as those containing the T7 promoter, could lead to the
                                    formation of inclusion bodies and therefore, refolding protocols would be necessary
                                    to purify the interested protein.</P><P class="bold white">2- On the other hand, an analysis of posttranslational
                                    modifications (PTM) was performed to see whether the expression in <I>E.coli</I>
                                    would be sufficient or a higher evolved organism would be needed. We found two
                                    N-glycosylations (Glc-NAc) in the α-chain and one in the ß-chain, all of them
                                    placed in asparagine residues (αN103, αN143 adn ßN51). However, these PTM were
                                    found far away from the interaction pocket and therefore, we concluded that they
                                    would not affect the interaction between the gluten peptide and the HLA-DQ protein.</P><P class="bold white">Based on the PTM analysis, simple systems such as <I>E.coli</I>
                                    could be used to express this protein. However, we also found the presence of
                                    disulfide bonds in both chains that could potentially affect the interaction
                                    pocket. Since <I>E.coli</I> is unable to make disulphide bonds, we decided to
                                    include synthetic constructs to express in the yeast host <I>Pichia pastoris</I>.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION class="st-description block-desc" id="cl-description"><DIV class="container"><DIV class="row"><DIV class="col-md-12 mx-auto"><DIV class="row justify-content-center block-sept"><DIV class="col-md-8 block-sept"><H4 class="book orange-intense text-center">Construct design</H4><P class="book orange text-center">1- Construct #2a:
                            Policistronic_EColi_CodonOptimisedK12_NdeISacI. This is a gBlock fragment that contains
                            both alpha and beta chains in the same construct separated by an intergenic sequence with
                            an additional RBS. This gene fragment is flanked by two restriction enzymes NdeI and SalI
                            to enable cloning by restriction-ligation to a vector. After cloning to the vector, it can
                            purified by a Strep-tactin purification column.</P></DIV></DIV><DIV class="row justify-content-center block-sept"><DIV class="col-md-8 block-sept"><P class="book orange text-center">2- Construct #2b:
                            Policistronic_2tags_EColi_CodonOptimisedK12_NdeISacI. This construct was ordered as a
                            synthetic gene. It is basically the same as #2a but it incorporates an extra Strep-tag at
                            the end of the alpha chain.
                        </P></DIV></DIV><DIV class="row justify-content-center block-sept"><DIV class="col-md-8 block-sept"><P class="book orange text-center">3- Construct #3a: LZipper_EColi_CodonOptimisedK12_NdeISacI.
                            This is a gBlock fragment derived from #2a that incorporates the Fos and Jun region at the
                            end of each chain through a flexible linker (VDGGGGG). These regions are known to form the
                            Leucine Zippers (LZ), a protein motif that contains a periodic repetition of a leucine
                            residue at every seventh position. This heptad repeat forms a stable α-helical conformation
                            that facilitates dimerisation and increases the stability of the oligomerized proteins.
                        </P></DIV></DIV><DIV class="row justify-content-center block-sept"><DIV class="col-md-8 block-sept"><P class="book orange text-center">4- Construct #3b:
                            LZipper_2tags_EColi_CodonOptimisedK12_NdeISacI. Very similar to #3a, this gBlock fragment
                            has, in addition to the LZ, a Strep-Tag at the end of each chain to facilitate the
                            purification.
                        </P></DIV></DIV><DIV class="row justify-content-center block-sept"><DIV class="col-md-8 block-sept"><P class="book orange text-center">5- Construct #4b. Yeast_2A_2Tags_EcoRIXbaI_PichiaOptimised:
                            This construct was ordered as a synthetic gene. It contains both alpha and beta chain
                            separated by a sequence that encodes for the 2A cleavage site which is a short peptide
                            sequence that has an auto-cleavage activity. This construct would result in the expression
                            of both chains separately. The restriction sites of EcoRI and XbaI were added at the end of
                            each chain to enable cloning into yeast vectors. We also added a His-tag to enable protein
                            purification.
                        </P></DIV></DIV><DIV class="row justify-content-center block-sept"><DIV class="col-md-8 block-sept"><P class="book orange text-center">6- Construct #6d.
                            Yeast_LZ_2A_2Tags_EcoRIXbaI_PichiaOptimised: This construct is a combination of construct
                            #4b and #3a. It has been designed for yeast expression with addition of the Fos and Jun
                            motifs to increase the protein stability.
                        </P></DIV></DIV><DIV class="row justify-content-center block-sept"><DIV class="col-md-8 block-sept"><P class="book orange text-center">7- Construct #7.
                            Linker_EcoRINdeI_XbaISalI_Pichia_Ecoli_Optimized: This construct contains the extracellular
                            domain of the alpha and the beta chain connected with a flexible linker. This construct was
                            codon optimized so that it could be used in both <I>E.coli</I> and <I>Pichia pastoris</I>.
                        </P></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION class="st-description block-sept" id="st-description"><DIV class="container"><DIV class="row"><P class="orange-medium book">The intended vectors to be used for cloning purposes are described below:</P></DIV><DIV class="col-md-12"><DIV class="row justify-content-center"><DIV class="col-md-8"><UL><LI class="orange"> pET22b(+)—BsPdaC. This is a common pET22b(+) vector that has a 835bp gene
                            between the restriction sites NdeI and SacI/SalI. Behind the SalI site there is a sequence
                            that encodes for a Strep-Tag. This vector has been used to clone the constructs for <I>E.coli</I>
                            expression.
                        </LI><LI class="orange">pPICZα: This is a 3.6 kb vector used to express recombinant proteins fused
                            to N-terminal peptide encoding the Saccharomyces cerevisiae factor secretion signal. This
                            vector allow a high-level, methanol inducible expression of the gene of interest in <I>Pichia</I>
                            strain X-33. This vector contains Zeocin resistance gene for selection in both <I>E.coli</I>
                            and <I>Pichia</I> and a C-terminal peptide containing the c-myc epitope and a polyhistidine
                            (6xHis) tag for detection and purification of a recombinant fusion protein.</LI><LI class="orange">pGAPZα: This is a 3.1 kb vector designed for a high-level, constitutive
                            expression in <I>Pichia pastoris</I>. This vector contains glyceraldehyde-3-phosphate
                            dehydrogenase (GAP) promoter instead of methanol-regulated AOX1 promoter. It also contains
                            Zeocin resistance gene, a C-terminal peptide containing the c-myc epitope and a
                            polyhistidine (6xHis) tag and a Saccharomyces cerevisiae -factor secretion signal.</LI></UL></DIV></DIV></DIV></DIV></SECTION><SECTION class="st-description block-desc" id="cl-description"><DIV class="container"><DIV class="row"><DIV class="col-md-12 mx-auto"><DIV class="row justify-content-center block-sept"><DIV class="col-md-8 block-sept"><H4 class="book orange-intense text-center"><I>E.coli</I> constructs</H4><P class="book orange text-center"><U>Week 1 (Aug 20th-24th)</U>
                            Transformed all gene constructs (2b, 4b, 6d and 7) successfully and prepared a miniprep
                            from them as well as the vector pET22b-BsPdaC. All gBlocks (2a, 3a, 3b) and construct 2b
                            were digested with NdeI and SalI. A ligation was performed for each gene into the vector
                            pet22b and transformed in <I>E.coli</I>. Results of the experiment showed high background,
                            and colony PCR of 13 samples and gel analysis resulted in undesired band corresponding to
                            the undigested vector carrying the BsPdaC gene (Figure 1).</P><P class="book orange text-center"><U>Week 2 (Aug 27th-31st)</U>
                            Repeated ligation and transformation of the same constructs (2a, 2b, 3a, 3b) but no
                            colonies arose.</P><P class="book orange text-center"><U>Week 3 (Sep 3rd-7th)</U>
                            Performed restriction-ligation-transformation of construct 2b and 7 with a different
                            restriction buffer and quantifying DNA before ligation. 21 colonies were observed for
                            pet22b-2b and 7 colonies were observed for pet22b-7. 11 colonies were screened by PCR but
                            results were inconclusive due to technical error.</P><P class="book orange text-center"><U>Week 4 (Sep 12th-15th)</U>
                            Unexpected event occurred. Several petri dishes kept in the fridge froze (including
                            transformations from ligations) and lost most of the potential positive colonies. Repeated
                            colony PCR for 4 viable samples. Gel analysis showed a band around 1300bp in sample nº8
                            that would correspond to the construct 2b (Figure 2a). Made a miniprep and analytical
                            restriction of the positive colony, which showed promising results (Figure 2b). Transformed
                            the plasmid into BL21 strain <I>E.coli</I>.</P><P class="book orange text-center"><U>Week 5 (Sep 17th- 21th)</U>
                            The positive colony was sequenced and resulted to be correctly cloned into the vector. A
                            first test of expression was performed by boiling the cell pellet in a SDS-PAGE buffer and
                            then run them in a acrylamide gel, but results were not conclusive. Besides that, a Gibson
                            assembly reaction was performed using the constructs and #3b and the digested pET22b
                            vector. Electroporated cells resulted in the growth of several colonies but colony PCR and
                            analytical digestion showed a band with a lower molecular weight than expected (Figure 3).</P><P class="book orange text-center"><U>Week 6-7 (Sep 24th- Oct 5th)</U>
                            A second and third attempt of HLA-DQ expression were performed. Briefly, cultures were
                            started at an initial OD600 of 0.1 and induced with IPTG when reached 0.3. Cultures were
                            then incubated at 37ºC 220rpm for 6h and then harvested. Purification with a Strep-tactin
                            column was performed after sonication but a very small peak was observed. Pellets were
                            solubilized with 8M urea and samples were analysed with an acrylamide gel. Results show the
                            presence of two intense bands at the solubilized fraction but the molecular weight were
                            higher than expected (Figure 4).</P></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION class="st-description block-desc" id="cl-description"><DIV class="container"><DIV class="row"><DIV class="col-md-12 mx-auto"><DIV class="row justify-content-center block-sept"><DIV class="col-md-8 block-sept"><H4 class="book orange-intense text-center"><I>P. pastoris</I> constructs</H4><P class="book orange text-center">All <I>Pichia</I> derived constructs and were obtained and
                            digested correctly. Different ligation assays were performed but only one resulted to be
                            successful. We obtained potential positive colonies in the zeocin resistant petri plates.
                            However, analytical digestions of those colonies indicated that the presence of the band
                            was outside the detection limit in agarose gel 1%.</P></DIV></DIV></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>