Team:BioIQS-Barcelona/Protocols
BIO IQS
Top
Wet Lab | Protocols
Have a look!
Protocols
The protocols that we have been using over the summer are available below. They all meet high-quality standards, as they were designed together with the Supervisors of our Team. We share them in the interest of reproducibility.
Protocol 1. GENOMIC DNA EXTRACTION FROM SALIVA
Reagents
Sterile saline solution (0.17 M).
Sterile PBS: 0.01M phosphate buffer, 0.15M NaCl at pH 7.4.
Stock Proteinasa K at 1mg/mL in PBS.
Depressor.
Conical tubes (15 mL).
Sterile Eppendorfs (1.5 mL).
Steps
Perform a wash out of the mouth with sterile saline solution and discard the liquid.
Scratch lightly the mucous membrane with a depressor.
Perform a vigorous wash out of the mouth with 10 mL of sterile saline solution and transfer the volume to a sterile 15 mL conical tube. Repeat to obtain 2 samples of 10 mL.
Centrifuge the samples 10 min at 3000 rpm. Discard the supernatant carefully by decantation. Avoid pellet detachment.
Resuspend the pellet with 1 mL of DNAsa-free PBS. Transfer the volume to a clean Eppendorf.
Centrifuge 10 min at 6000 rpm.
Discard the supernatant and resuspend with 1 mL of DNAsa-free PBS.
Centrifuge again 10 min at 6000 rpm.
Resuspend the last pellet with 100-400 μl of PBS (assess the amount of pellet).
Transfer 90 µl of the last solution to a new clean Eppendorf and add 10 µl of proteinase K solution of 1 mg/mL to a final concentration of 0.1 mg/mL.
Incubate the sample 1 h to 55 °C in a bath.
Then move it to another bath at 100 °C and incubate 10 min.
Cool down the sample with ice and then store it at – 20 °C till use.
Protocol 2. PCRS FOR GENE EXTRACTION
View & download PDF
Protocol 3. BAND EXTRACTION (QIAGEN)
This protocol is designed to extract and purify DNA of 70 bp to 10 kb from standard or low-melt agarose gels in TAE or TBE buffer.
Excise the DNA fragment from the agarose gel with a clean, sharp scalpel.
Weight the gel slice in a colorless tube. Add 3 volumes of Buffer QG to 1 volume of gel (100mg - 100µl).
Incubate at 50ºC for 10 min. TO help dissolve gel, mix by vortexing the tube every 2-3 min during the incubation.
After the gel slice has dissolved completely, check that the color of the mixture is yellow.
Add 1 gel volume of isopropanol to the sample and mix.
Place a QIA spin column in a provided 2 mL collection tube.
To bind DNA, apply the sample to the QIA spin column, centrifuge for 1 min.
Discard flow-through and place QIA Spin column back in the same collection tube.
Recommended: Add 0.5 mL of Buffer QG to QIAquick column and centrifuge for 1 min.
To wash, add 0.75 mL of Buffer PE to QIAspin column and centrifuge for 1 min.
Discard the flow-through and centrifuge the QIA spin column for an additional 1 min at 13.000 rpm.
To elute DNA, add 40 µl of Buffer EB.
Protocol 4. ELECTROCOMPETENT PREPARATION
Grow overnight, shaking at 37ºC.
Inoculate the overnight culture volume need to achieve OD600 of 0.1-0.2 into 100ml LB.
Grow at 37ºC, shaking at 250-300rpm, until density reaches OD600 of 0.5-0.6.
Trasnfer cultures into pre-chulled 50mL centrifuge tubes
Spin cells for 10 minutes at 5000rpm at 4ºC.
Pour off supernatant and resuspend the pellet in 40 mL sterile ice-cold 10% glycerol
Spin as in step 7.
Repeat wash twice more with 20 ml and 10 ml of ice-cold 10% glycerol (steps 8 and 9).
Pour off supernatant and resuspend the last pellet in 400-600 µl of ice-cold 10% glycerol.
Aliquot 120 µL cells per pre-chilled microcentrifuge tubes on ice.
Store aliquots at –80ºC.
Test cells for transformation efficiency and contamination immediately.
Protocol 5. DIGESTION of pET22b
For a reaction volume of 10 µL:
Grow overnight, shaking at 37C.
Add 1 µl of NEBuffer3.1.
Add 1 µl of Ndel.
Add 1 µl of SalI
Add 50-200 ng of plasmid DNA.
Add DNAse-free water up to 10 µl.
Incubate 1.5h at 37°C.
Incubate the sample 20 min at 65°C to inactivate restriction enzymes.
For storage, keep the samples at -20°C.
Protocol 6. LIGATION PROTOCOL WITH T4 DNA LIGASE
This protocol uses a 1:3 (vector:insert) ratio. Depending on the vector this ratio shall be optimized. The total amount of DNA can be also adapted. For a reaction volume of 20 µL:
Add 2 µl of T4 DNA Ligase Reaction Buffer.
Add 0.020 pmol of vector DNA.
Add 0.060 pmol of insert DNA.
Add DNAsa-free water up to 19 µl.
Add 1 µl of T4 DNA Ligase.
Incubate overnight at 16 °C or 15-20 minutes at room temperature.
Heat inactivate at 65°C for 15 minutes.
For storage, freeze the samples at -20°C.
Protocol 7. ELECTROPORATION, COMPETENT CELLS TRANSFORMATION
Thaw electrocompetent cells on ice.
Transfer 50 µl of electrocompetent cells to a pre-chilled electroporation cuvette with 1 mm gap.
Add 1-2 µl (10 pg - 100 ng) of the DNA to electrocompetent cells.
Mix carefully and slowly by pipetting up and down.
Once DNA is added to the cells, incubate 1 minute on ice.
Perform electroporation at 1700-1800V.
Add 950 µl of room-temperature LB media to the cuvette immediately after electroporation.
Mix carefully and slowly by pipetting up and down and transfer the volume to a clean sterile tub.
Place the tube at 37°C for 60 minutes. Shake vigorously (250 rpm) or rotate.
Warm selection plates to 37°C.
Spread 100 µl of the cells onto the plates. If desired, centrifuge the remaining volume (900 µl), and resuspend the pellet in 100 µl of LB. Plate this last volume in a selection plate.
Incubate overnight at 37°C.
Protocol 8. COLONY PCR
All colony PCRs were performed as follows.
Adjust 20 µl of ddH2O.
Add 2µl of Buffer Taq 10X .
Add 1 µl dNTPS 0.2mM.
Add 1 µl Reverse primer 0.2 mM.
Add 1 µl Forward primer 0.2 mM.
Colony PCR setup. Pick a colony and inoculate to the PCR mixture.
Denaturation 94ºC 10 min.
Denaturation 94ºC 30 seconds.
Annealing temperature for each case, 30 seconds
Extension at 72ºC for X min, considering (1kb/min)
Final Extension at 72ºC for 10 min.
Hold at 4ºC.
Protocol 9. SDS-PAGE BUFFERS PREPARATION
Reagents
Electrophoresis Buffer 10x: 30 g Tris-base (Merck) 144 g Glycine (Merck) 10 g SDS (Scharlau) Fill to 1 L with H2O Milli-Q.
Sample Buffer 5x: 3 ml H2O Milli-Q 1 ml Solution C 2 ml glycerol (Merck) 1.6 ml Solution D 0.4 ml β- mercaptoethanol (Bio-Rad) 0.5 mg bromophenol blue (Bio- Rad) (small spatula).
Solution A: 30% acrylamide, 2.7 % in bis- acrylamide: Solution acrylamide/bis- acrylamide 30% Mix ratio 29:1.
Solution B: 1.5 M Tris-HCl 9.08 g Tris-base (Merck) Add 40 ml H2O Milli-Q Adjust to pH 8.8 with HCl 5N Fill to 50 ml with H2O Milli-Q.
Solution C: 0.5 M Tris-HCl 3.0 g Tris-base (Merck) Add 30 ml of H2O Milli-Q Adjust to pH 6.8 with HCl 5N Fill to 50 ml with H2O Milli-Q.
Solution D: 10%(w/V) SDS 10 g SDS (Scharlau) Fill to 100 ml with H2O Milli-Q.
Solution G: Ammonium Persulfate 20% (w/V), 500 mg (NH4)2S2O7 (Bio-Rad) 2.5 ml H2O Mill-Q.
Protocol 10. GEL PREPARATION
12% Acrylamide gel preparation. Running gel
3.4 mL H2O milli-Q.
4 mL Sol A
2.5 mL Sol B
100 µl Sol D
5 µl TEMED
Persulfate ammonia. Spatula tip
Mix thoroughly and dispense using a Pasteur pipette (leaving space for the stacking gel). Hint: leaving the Pasteur pipette filled with the running gel mixture will simulate the conditions inside the gel plates. Therefore when the mixture, inside the Pasteur pipette, is no longer liquid the gel will also be polymerized.
Cover running gel with isobutanol or H2O Milli-Q by use of Pasteur pipette.
After approximately 30 minutes, absorb the isobutanol / H2O Milli-Q using filter paper.
Stacking gel
Mix: 6.1 ml H2O Milli-Q, 2.5 ml Solution C, 1.3 ml Solution A, 100 μl Solution D, 10 µl TEMED.
Add: 50 µl Solution G.
Fill using Pasteur pipette.
Add gel comb.
Hint: leaving the Pasteur pipette filled with the stacking gel mixture will simulate the conditions inside the gel plates. Therefore when the mixture, inside the Pasteur pipette, is no longer liquid, the gel will also be polymerized.
Electrophoresis conditions.
Place gels in electrophoresis support with the short plates facing towards the interior.
Place gels in electrophoresis support with the short plates facing towards the interior.
Prepare the electrophoresis buffer: 100 ml 10X electrophoresis buffer + 900 ml Milli-Q H2O.
Mix buffer without making foam. Fill inner electrophoresis cavity with buffer. Add remaining buffer to outer electrophoresis area.
Introduce samples into a 95 ºC water bath for 10 minutes. After 10 minutes remove samples and let cool to room temperature. Centrifuge samples for 10 seconds to condense.
Protocol 11. AKTA PURIFICATION (Strep-Trap)
Prepare and filter the following solutions using 0.45 µm filters.
1 L 0.5 M NaOH solution (cleaning and conditioning).
2 L Binding buffer (PBS 50 mM NaH2PO4,300 mM NaCl, pH = 8.5.
25 ml 1 mM HABA solution in Binding buffer (column cleaning and conditioning).
Ethanol 20% (column storage).
Lysis buffer: 50mM TrisHCl, 100mM NaCl + 1mM PMSF, pH=7.5.
Strep-tactin column
Start by conditioning the StrepTrap HP column by sequentially loading 5 bed volumes of each one of the following: MilliQ water, 0.5M NaOH, MilliQ water, and Binding buffer. Then add 1 mM HABA solution until the column turns completely red. After that, Binding buffer is added until the column turns white again.
By working at a 1 ml/min flow rate, apply filtered cell lysate to the column (20ml – 20 min), and then wash the column with Binding buffer until the absorbance A280 reaches basal level.
Subsequently, load Elution buffer into the column until absorbance reaches basal level again. Track the protein fractions and collect the eluted protein fraction in a separate tube.
Proceed to wash and regenerate the column by loading 5 bed volumes of Binding buffer, followed by as many volumes of 1mM HABA solution as needed (until the column turns red) to remove remaining desthiobiotin in the column. Finally, add Binding buffer again until the column turns white, at this point the column is ready to use by another sample.
Proceed to wash and regenerate the column by loading 5 bed volumes of Binding buffer, followed by as many volumes of 1mM HABA solution as needed (until the column turns red) to remove remaining desthiobiotin in the column. Finally, add Binding buffer again until the column turns white, at this point the column is ready to use by another sample.
For final column storage, column must be sequentially cleaned with HABA, MiliQ water, 0.5 M NaOH (check for correct pH using pH strips), MiliQ water, and then stored in 20% ethanol and 4C.
Protocol 12. CELL LYSIS
Rinse cell disrupter with milliQ water and PBS. Then lyse cells by sonication, being the sonicator settings: 50% amplitude and 2’’ON-5’’OFF pulses for a total time of 8 min
Ensure that the cell solution is kept at 4ºC (ice) during the whole procedure.
The resulting cell lysate solution is centrifuged at 12000 rpm and 4ºC for 30-60 min.
After discarding the pellet, filter the supernatant through 0.45 μm filter and keep it at 4ºC until loading it into the column.
Protocol 13. INCLUSION BODIES SOLUBILISATION
Important note
Cell pellet after sonication was resuspended with 8M Urea, 25mM Tris, pH=8 and left in shaker O/N.
Protocol 14. COMASSIE STAINING
Important notes before starting:
Coomassie staining solution (1 L) is prepared by with: Coomassie staining solution consists of 0.25% Coomassie Brillant Blue R (Sigma) in a mixture of Acetic Acid (glacial), Methanol, and H2O (2:9:9).
Destaining solution (1 L) is prepared by one group: H2O, Methanol, Acetic acid (glacial) 5:4:1 mixture.
Steps:
Separate the gel plates by use of a gel wedge in order to remove the gel being careful not to tear the gel in the process. Remove the stacking gel by use of the gel wedge. Place the running gel into the staining solution. Hint: the running gel can be transferred to the staining solution easily by using a stream of H2O from a water bottle.
Cover the gel with the Coomassie staining solution.
Place recipient on an orbital shaker for 40 minutes with slight shaking.
After time has elapsed decant staining solution. Save staining solution. Note: staining solutioncan be recycled many times.
Add contrast solution to gel.
Contrast (rapid destaining)
Cover gel with the destaining solution. Place on an orbital shaker with slight shaking. When the contrast solution becomes blue with Coomassie, decant the contrast solution and add more contrast solution until protein bands become visible.
Once gel becomes destained, transfer the gel to H2O in order to reverse gel shrinkage. Place on an orbital shaker with slight shaking for 5 to 10 minutes.
The gel can be saved by introducing the gel in between plastic wrap and heat sealing.
Once the gel is conserved, scan the gel into the computer for a permanent record.
