"
Page
Discussion
View source
History
teams
Log in
 
Team:Oxford/biosensor construction
From 2014.igem.org
Construction
Show all:
Modelling
Wetlab
Introduction: how we constructed our biosensor
In order to be able to use our model and to determine whether DcmR acts as a repressor or activator in the presence of DCM, we designed and constructed the following two-plasmid system. We primarily used Gibson assembly methods and sourced most of the necessary DNA from gblocks (synthesised oligonucleotides) we had designed based in the sequenced genome of Methylobacterium DM4. This system will also form the DCM biosensor and will be integrated with an electronic circuit to complement this genetic one:
pOXON-2-dcmR-mCherry
pOXON-2-dcmR-mCherry
Production of the DCM-binding protein DcmR
Oxford iGEM 2014
pSRK-Gm-pdcmAsfGFP and pJ404-pdcmA/R-sfGFP
pSRK-Gm-pdcmAsfGFP and pJ404-pdcmA/R-sfGFP
The binding site for DcmR with expression-reporting GFP
Unfortunately, we were unable to assemble the pSRKGm-pdcmA-sfGFP construct even after multiple attempts. Since we plan to prove that this system can work in E. coli, we re-designed this construct to use a different vector with an origin of replication that is compatible with our other construct pOXON-2 (containing dcmR).
We chose to use plasmid backbone pJ404 since it contains a pBR322 origin of replication which is compatible with p15A origin of replication present in pOXON-2.
Since DcmR is predicted to regulate expression of DcmA as well as auto-regulating its own expression, we decided to insert this promoter-containing intergenic region in both orientations upstream of sfGFP. This means we have two constructs:
-  One with sfGFP in a position corresponding to the equivalent position of dcmA (labelled as ‘forward’ or PdcmA)which can express sfGFP under the PdcmA promoter.
- A second construct with sfGFP in the equivalent position of dcmR (labelled as ‘reverse' or PdcmR)that can express sfGFP under the promoter PdcmR.
These are shown below:
PJ404-pdcmA-sfGFP
PJ404-pdcmR-sfGFP
Oxford iGEM 2014
Why these plasmid backbones?
Why these two plasmid backbones?
The two plasmids are partitioned during cell division by different systems, thus an equal proportion of each plasmid is maintained in each new daughter cell.
Different antibiotic resistances will allow us to select for cells that have taken up both plasmids by application of both antibiotics.
The replication origins compatible with E.coli and pseudomonas strains.
We have used two plasmids so that we can test each part in isolation before transforming them both into the same cell.
How were the constructs made?
How were the constructs made?
Building pOXON-1
The first task in the construction of the pOXON-2-dcmR-mcherry construct was the creation of pOXON-1; pME6010 with tetracycline resistance replaced by kanamycin resistance. (N.B. The KanR gene was amplified with an optimised RBS.)
pOXON-1 was produced using the Gibson assembly method.
Building pOXON-2 and pOXON-2-dcmR
pOXON-1 was then used as  the vector for the insertion of the three gblock fragment constituting the inducible expression system of dcmR via Gibson assembly.
Upon sequencing of the product, it was determined that the version of the gblock containing the dcmR gene in the construct was actually truncated. This construct with the truncated dcmR is pOXON-2. A second Gibson assembly reaction was used to replace the truncated version with the full length gene also derived from the gblock. The resulting construct was named pOXON-2-dcmR.
Adding in mCherry
We then used pOXON-2-dcmR as the vector for the insertion of mCherry downstream of dcmR as a translational fusion by Gibson assembly.
We therefore have a system of expressing dcmR with (pOXON-2-dcmR-mCherry) and without (pOXON-2-dcmR) the mCherry fusion in order to test whether the addition of mCherry affects the action of DcmR. Both will be submitted as BioBricks in the standard pSB1C3 backbone.
All constructs were confirmed by sequencing.
Building pSRK Gm construct
We have attempted to make our second construct by inserting the pdcmAsfGFP gblock into the pSRK Gm vector by Gibson assembly. As this is proving difficult, the next approach will be to insert the two components separately and to source the DNA from sources other than the gblock. Firstly, pdcmA will be amplified from Methylobacterium extorquens DM4 genomic DNA and inserted into the pSRKGm vector. sfGFP will then be amplified from a plasmid already containing it, and added to the pSRKGm-pdcmA construct.
Oxford iGEM 2014
Retrieved from " http://2014.igem.org/Team:Oxford/biosensor_construction "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
