<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Oxford_biosensor_characterisation"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Oxford/biosensor_characterisation">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Oxford/biosensor_characterisation&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Oxford/biosensor_characterisation&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Oxford/biosensor_characterisation&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Oxford/biosensor_characterisation" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Oxford/biosensor characterisation</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="outer" style="overflow-y: scroll; overflow-x: hidden;"><DIV id="stuff" style="float:left;position:absolute;margin-left:200px;margin-right:100px; margin-top:50px;min-width:645px;"><DIV id="showwetlab"><DIV id="showmodelling"><DIV style="background-color:#D9D9D9; opacity:0.7; z-index:5; Height:75px; width:100%;font-size:65px;font-family:Helvetica;padding-top:5px; font-weight: 450;margin-top:10px;"><DIV style="background-color:white; opacity:0.9; Height:75px; width:100%;margin-top:5px:margin-bottom:5px;font-size:65px;font-family:Helvetica;padding-top:5px; color:#00000; font-weight: 450;"><CENTER><FONT style="opacity:0.7">Characterisation</FONT></CENTER></DIV></DIV><DIV style="background-color:white; border-bottom-left-radius:10px;border-radius:10px; padding-left:10px;padding-right:10px;min-width:300px;margin-top:-50px;"><DIV style="position:absolute;background-color:rgba(255,255,255,0.6);border-radius:15px; z-index:5;margin-top:-18.2%; Height:70px; width:20%;font-size:65px;font-family:Helvetica; font-weight: 450;padding-left:10px;padding-right:10px;padding-top:3px;min-width:170px;margin-bottom:3px;"><DIV style="width:100%;"><FONT style="font-size:15px;font-weight:500;">Show all:</FONT></DIV><A href="#showmodelling"><DIV class="orange_news_block1 showmodelling" style="background: #F9A7B0;border-radius:15px;color:black;float:left;height:40%;width:40%;margin-left:6%;padding-top:10px;"><CENTER><H1WHITE><FONT style="font-size:15px;font-weight:500;">Modelling</FONT></H1WHITE></CENTER></DIV></A><A href="#showwetlab"><DIV class="orange_news_block1 showwetlab" style="background: #ADD8E6;border-radius:15px;color:black;float:left;height:40%;width:40%;margin-left:3%;padding-top:10px;"><CENTER><H1WHITE><FONT style="font-size:15px;font-weight:500;">Wetlab</FONT></H1WHITE></CENTER></DIV></A></DIV><H1>Introduction: what are we characterising?</H1><FONT style="font-style: italic;">Methylobacterium Extorquens</FONT> DM4 in the presence of DCM expresses DcmA, a dichloromethane dehalogenase.
Within 1.5kb upstream of <FONT style="font-style: italic;">dcmA</FONT> and in the opposite orientation is a second gene encoding DcmR, a regulatory protein that controls expression of <FONT style="font-style: italic;">dcmA</FONT>.






In order to design and create a stable and sensitive system that responds to DCM we first need to characterise the regulatory nature of DcmR. Characterisation of this regulatory network has never been done before although it has been suggested to be a repressor [1]; we will be the first to fully characterise the mode of action of <FONT style="font-style: italic;">dcmR</FONT>. To do this we are testing the following hypotheses for DCM activating the transcription of <FONT style="font-style: italic;">dcmR</FONT>: either double repression or double activation. In other words, either DcmR represses <FONT style="font-style: italic;">dcmA</FONT> expression and DcmR is in negatively modulated by the presence of DCM; or expression of <FONT style="font-style: italic;">dcmA</FONT> requires DcmR as an activator, with DcmR in turn only activated in the presence of DCM.</DIV><DIV class="row"><A href="#show1" class="show wetlab-row" id="show1"><DIV class="wetlab"><H1WHITE>What is currently known about DcmR?</H1WHITE></DIV></A><A href="#hide1" class="hide" id="hide1"><DIV class="wetlab"><H1WHITE>What is currently known about DcmR?</H1WHITE></DIV></A><DIV class="list"><DIV class="white_news_block2"><H1>DcmR and regulation of <FONT style="font-style: italic;">dcmA</FONT> expression</H1>

Mutants with <FONT style="font-style: italic;">dcmA</FONT> and the intergenic region but without complete DcmR express <FONT style="font-style: italic;">dcmA</FONT> constitutively. Re-integration of <FONT style="font-style: italic;">dcmR</FONT> restores regulation of <FONT style="font-style: italic;">dcmA</FONT> expression at the transcriptional level [1]. In addition, it has been shown that the region including <FONT style="font-style: italic;">dcmR</FONT>, the intergenic region and <FONT style="font-style: italic;">dcmA</FONT> is sufficient to confer a DCM dependent response in genetically engineered <FONT style="font-style: italic;">Methylobacterium extorquens DM4</FONT> [2]. <H1>DcmR and DNA-binding</H1>

DcmR is thought to be a DNA binding protein as structure predicting software indicates that there is a helix-turn-helix domain at the N-terminal of the protein. Since the region between the two promoters for <FONT style="font-style: italic;">dcmR</FONT> and <FONT style="font-style: italic;">dcmA</FONT> can be deleted without any effect on regulation it has been suggested that DcmR does not bind to a secondary regulatory site in between the genes but most likely acts directly on the <FONT style="font-style: italic;">dcmA</FONT> promoter itself [1]. In addition, regulated expression of <FONT style="font-style: italic;">dcmA</FONT> is not affected when the <FONT style="font-style: italic;">dcmR</FONT> and <FONT style="font-style: italic;">dcmA</FONT> transcriptional units are placed on separate replicons thereby suggesting that their topology is independent of the regulatory network. It is therefore suggested that DcmR binds the DNA in the intergenic region with the simplest model of its mode of action being as a trans-acting DNA-binding repressor; however this remains to be fully validated [1].

We have therefore proceeded on the assumption that DcmR is directly influenced by the presence or absence of DCM and furthermore that we can use <FONT style="font-style: italic;">dcmR</FONT>, the intergenic region and <FONT style="font-style: italic;">dcmA</FONT> alone to characterise the regulatory network. 

[1] La Roche, S. D., and T. Leisinger. &quot;Identification of <FONT style="font-style: italic;">dcmR</FONT>, the regulatory gene governing expression of dichloromethane dehalogenase in Methylobacterium sp. strain DM4.&quot; Journal of bacteriology 173.21 (1991): 6714-6721. 
[2] Lopes, N., et al “Detection of dichloromethane with a bioluminescent (lux) bacterial bioreporter” J Ind Microbiol Biotechnol (2012) 39:45–53

<H1>Characterising the DcmR - DCM - P_dcmA interaction</H1>
To find out whether the DcmR acts as a repressor or an activator on the promoter of the <FONT style="font-style: italic;">dcmA</FONT> gene, we attempted to build the genetic circuit shown above on the right. Having <FONT style="font-style: italic;">dcmR</FONT> under inducible TetR expression should allow us to have very good control of the amount of DcmR present. Additionally a translational fusion with DcmR and a mCherry fluorescence tag will act as another confirmation to the amount of DcmR present.

We then extensively modelled the circuit to discover how the response of the system would differ if it was either of the two circuit systems. Click the modelling bubbles (pink) to find out exactly how we achieved this.
</DIV></DIV></DIV><DIV class="row"><A href="#show2" class="show modelling-row" id="show2"><DIV class="modelling"><H1WHITE>Modelling the first half of the genetic circuit</H1WHITE></DIV></A><A href="#hide2" class="hide" id="hide2"><DIV class="modelling"><H1WHITE>Modelling the first half of the genetic circuit</H1WHITE></DIV></A><DIV class="list"><DIV class="white_news_block2"><H1>Predicting the mCherry fluorescence</H1>
We simplified the first double repression by modelling it as an activation of <FONT style="font-style: italic;">dcmR</FONT> by anhydrous-tetracycline (ATC), albeit parameterised by different constants. This assumption is justified by the fact that we are able to precisely control the addition of ATC and measure the fluorescence of the mCherry.
 
We modelled this first step using both deterministic and stochastic models.
</DIV><DIV class="white_news_block2"><H1>Biochemical equations</H1>
The biochemical equations that describe the behaviour of the top half of the genetic circuit are:


Oxford iGEM 2014
</DIV><DIV class="white_news_block2"><H1>Deterministic</H1>

Deterministic models are very powerful tools for synthetic biology. They describe the behaviour of the bacteria at the population level and use Ordinary Differential Equations (ODEs) to relate each activation and repression. By constructing a cascade of differential equations one can build a realistic model of the average behaviour of the system.

 
The differential equation that describes this first step of the system is:




Solving this ODE in Matlab (with a zero basal transcription rate) predicts the following the response of the system:

This model works assuming that sufficient TetR is always present.

Oxford iGEM 2014

</DIV><DIV class="white_news_block2">
While the analysis of this circuit is not critical to the successful outcome of this part of the project, it will provide us with very good practice of both obtaining fluorescence time series data and accurately fitting the data to the model. It will also help us develop our methods of predicting future system behaviour. This is because this system is already well documented in the literature and so we should be able to test our methods and responses against well documented results from labs across the world.

As you can clearly see from the graph, the model predicts a large fluorescence increase as the input is added. This is the what we expect from the actual system and is the best approximation that is obtainable before we get experimental data.

In the graph above, the model is set to have a basal transcription rate of zero. This is why there is a zero fluorescence response before the input has been added - this corresponds to the Ptet promoter not being leaky. This basal rate will be calibrated alongside all of the other parameters in the model.
</DIV></DIV></DIV><DIV class="row"><A href="#show3" class="show modelling-row" id="show3"><DIV class="modelling"><H1WHITE>How can we tell the systems apart?</H1WHITE></DIV></A><A href="#hide3" class="hide" id="hide3"><DIV class="modelling"><H1WHITE>How can we tell the systems apart?</H1WHITE></DIV></A><DIV class="list"><DIV class="white_news_block2"><H1>Predicting the sfGFP fluorescence</H1><H1>Introduction</H1>
To allow us to characterize the second half of the genetic circuit (DcmR regulating sfGFP), we needed to be able to predict the difference in response. To do this, we constructed models by cascading the differential equations according to the respective circuit structures thereby producing two different potential system responses.

We then set up the differential equations necessary to solve this problem in Matlab. The method and results are as detailed below:
</DIV><DIV class="white_news_block2">
Oxford iGEM 2014
</DIV><DIV class="white_news_block2"><H1>Conclusion</H1>
 
The bottom graphs illustrate the predicted response of each system to a simultaneous step input of both DCM and ATC. As you can see, there is little difference in the predicted steady-state value of the fluorescence, however, providing the basal transcription rate of GFP is relatively low, there should be a clear difference in the level of fluorescence before either of these inputs are added. This very easily identifiable difference between the two systems will enable us to characterize the genetic circuit present in our particular system.
<H1>Calculating the parameters</H1>
Calculating the many parameters for this system will be undoubtedly challenging. 
<A href="https://2014.igem.org/Team:Oxford/calculating_parameters" target="_blank">How are we calculating the parameters?</A><A href="https://2014.igem.org/Team:Oxford/biosensor_characterisation#show7">Go to the data section where we calculated parameters for this part of the circuit.
</A></DIV></DIV></DIV><DIV class="row"><A href="#show10" class="show modelling-row" id="show10"><DIV class="modelling"><H1WHITE>Stochastic modelling</H1WHITE></DIV></A><A href="#hide10" class="hide" id="hide10"><DIV class="modelling"><H1WHITE>Stochastic modelling</H1WHITE></DIV></A><DIV class="list"><DIV class="white_news_block2"><DIV class="white_news_block2"><H1>Stochastic Modelling</H1>


Stochastic modelling uses probability theory to predict the behaviour of a system. For our project, we used it to model the expression of the mCherry protein from bacteria. 


We started with the Gillespie Algorithm, which considers the expression of mCherry to be binary; a molecule of mCherry is either expressed or degraded. Before we determined which event happened, we had to work out when the event happened. Using the random number r1 (taken from a uniform distribution between 0 and 1), we produced another random number τ, which determined the time until the next reaction.

Where α0 represents the probability that any reaction will happen, given by the following equation:


We modelled the probability of a molecule of mCherry being created using the Michaelis-Menten model (α1), incorporating a basal transcription rate (b1). For the degradation, we assumed a simple proportional relationship: the more mCherry you have, the more likely it is that a molecule degrades (δ1). The constant of proportionality will be a function of the intrinsic life time of the protein in the cell. We considered there to be no ATC originally, then a large step in ATC at time=50.

To decide if mCherry was expressed, we looked at the percentage of events which were expressions. Then we compared this to a second random number r2 (again taken from a uniform distribution from 0 to 1). If the random number was lower, then mCherry was expressed. If it was higher, then a mCherry was degraded. In this way we make a weighted random choice about whether mCherry was expressed or degraded. We only stored the time and amount of mCherry when there was a event, to save on computation time.


Stochastic modelling is useful because it can show us the stochastic effects which are often observed in individual bacteria. By calculating the variation of the mean of multiple mCherry producing bacteria, we can also work out the standard deviation. Then if we assume that the system varies with respect to the normal distribution, we can produce error bounds for the production of mCherry, such that we can say that 90% of the time we can expect the production of mCherry from a single bacterium to be within these two curves as seen in Figure 2. This could be useful for seeing if results are unexpected, or, if there are multiple outliers, that our model is incorrect. If we average an increasing number of bacteria, then the mean stochastic curve tends towards the deterministic response as seen in Figure 1. This is to be expected, as we are now looking at the system as a whole and fluctuations in the production from individual bacteria are averaged out. In terms of their use, when looking at small amounts of bacterium the stochastic model would be better, because real random fluctuations can be seen. For larger bacterial populations, the deterministic response models the growth very well. The stochastic model can also model large groups but requires large number of realisations which causes simulations to take a lot longer to run.


When we originally ran the models, we picked arbitrary constants to view the general response. Later we created a code which was able to produce a line of best fit for fluorescence data from the lab; from this line the constants (a1,b1,d1,k1) could be determined. The exact calculations can be found on the &quot;Calculating Parameters subsection&quot;.


However, because the of the size of expression term, the time between events are very small resulting in an almost deterministic response even with only 1 realisation as can be seen on Figure 3.

α1 = expression rate constant of dcmR = 16.5min-1

k1 = Michaelis - Menten constant of dcmR = 0.015 ml/ng

d1 = degradation constant of dcmR = d1=0.00385min-1

β1 = Basal transcription rate of dcmR = b1=1.81min-1
</DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><A href="#show4" class="show wetlab-row" id="show4"><DIV class="wetlab"><H1WHITE>ATC induction of mCherry expression</H1WHITE></DIV></A><A href="#hide4" class="hide" id="hide4"><DIV class="wetlab"><H1WHITE>ATC induction of mCherry expression</H1WHITE></DIV></A><DIV class="list"><DIV class="white_news_block2"><H1>Wetlab data showing response in level of mCherry expressed with different concs of ATC</H1>

By making a translational fusion of mCherry at the C terminus of the dcmR gene under the tet promoter and tet operator system (see our <A href="https://2014.igem.org/Team:Oxford/biosensor_construction">Construction page</A> for details) we could measure mCherry fluorescence to gain information about <FONT style="font-style: italic;">dcmR</FONT> induction by ATC. Expression was induced with various amounts of ATC and the following fluorescence data acquired. Exposure time was 0.2 seconds. As no calibration data was obtained using purified mCherry, the results have been left in fluorescence arbitrary units. Images were analysed using imageJ software.
mCherry fluorescence increases with amount of ATC used confirming that the dcmR gene was expressed under the control of the tet promoter and operator system.Figure 1. Epi-fluorescence microscopy images (green channel) of cells expressing dcmR-mCherry fusion induced by 0ng ATC, 50ng ATC,and 200ng (left to right)



This data was then used to refine and test our models (see below).

</DIV></DIV></DIV><DIV class="row"><A href="#show5" class="show modelling-row" id="show5"><DIV class="modelling"><H1WHITE>Adapting the model for the above data</H1WHITE></DIV></A><A href="#hide5" class="hide" id="hide5"><DIV class="modelling"><H1WHITE>Adapting the model for the above data</H1WHITE></DIV></A><DIV class="list"><DIV class="white_news_block2"><H1>Introduction</H1>
Our team were able to obtain good data for both the mCherry response of the system and the overall sfGFP response. This bubble shows how we adapted our model to make the most of the mCherry fluorescence data.

The original data is shown on the right with error bars showing the standard error of the measurements.

Standard error is calculated as the average standard deviation divided by the square root of the total number of readings.
<H1>How we used the model</H1>
However, to demonstrate the power of the computer models that we’ve built, we made our model simulate the same graph (mean fluorescence against ATC concentration added). To build this, we started from the graph shown in the 
<A href="https://2014.igem.org/Team:Oxford/biosensor_characterisation#show2">first modelling bubble</A>
 on this page, shown here with a small basal rate (see 
<A href="https://static.igem.org/mediawiki/2014/b/be/Oxford_Equations_explained.png" target="_blank">where did these equations come from?</A>
). This graph shows how the predicted fluorescence of the cells changes with time in response to an addition of ATC halfway along the time scale. At this stage, all input values, model parameters and therefore results are arbitrary.

We then ran the model for the correct amount of time (2 hours 20mins incubation with ATC) and ran it for lots of different concentrations of ATC over the range that the wet-lab team tested. The parameters are still arbitrary at this point (the same as above) and the results of the graphs are therefore arbitrary are as well, but the input values are now correct. The graphic below shows how we used the existing model to obtain the same graphs as the wet-lab team had obtained.

The numerical inputs that were used to model this data set were therefore:
 
Oxford iGEM 2014



</DIV></DIV></DIV><DIV class="row"><A href="#show7" class="show modelling-row" id="show7"><DIV class="modelling"><H1WHITE>Finding parameters</H1WHITE></DIV></A><A href="#hide7" class="hide" id="hide7"><DIV class="modelling"><H1WHITE>Finding parameters</H1WHITE></DIV></A><DIV class="list"><DIV class="white_news_block2"><H1>Parameters</H1>
As all results are arbitrary up to this point, it is now time to calculate the parameters that will make the model’s response match up with the wet-lab data. The purpose of doing this is that the model will be able to give relatively accurate predictions of the response of the bacteria to further testing, therefore making the development of the biosensor much more efficient. The amount of data here will not allow us to calculate the parameters to a high level of accuracy, but it should be able to give us some very good approximations of what we can expect.

The parameters that we need to calculate are the constants in the differential equation that governs the behaviour of the first half of the genetic circuit. This half of the system is shown again here to remind the reader which part we are considering.

These parameters are:
<LI>α1 = expression rate constant of <FONT style="font-style: italic;">dcmR</FONT></LI><LI>k1 = Michaelis - Menten constant of <FONT style="font-style: italic;">dcmR</FONT></LI><LI>d1 = degradation constant of <FONT style="font-style: italic;">dcmR</FONT></LI><LI>β1 = Basal transcription rate of <FONT style="font-style: italic;">dcmR</FONT></LI>
Remember that because the mCherry gene is tagged (translational fusion) onto the end of the <FONT style="font-style: italic;">dcmR</FONT> gene, <FONT style="font-style: oblique;"> the mCherry fluorescence will be the same as the amount of DcmR protein present</FONT>. However, there is not very comprehensive data in the literature about the values that we can expect from the behaviour of the <FONT style="font-style: italic;">dcmR</FONT> gene and its stability in vivo.

<H1>Degradation constant</H1>
The initial steady state of the system (before ATC has been added) is determined by two constants in the model. These constants are the degradation constant of DcmR and the basal transcription rate of the system. Due to the lack of numerical information in the literature on the behaviour of the <FONT style="font-style: italic;">dcmR</FONT> gene, the way of calculating these two parameters is by using the single basal rate data point from the wet-lab data (fluorescence value when 0ng of ATC has been added).

 If we assume that the half-life of the dcmR protein is 3 hours [1] (180 minutes), we can calculate the degradation constant for our model. The exponential protein decay is therefore described by:

<H1>Basal transcription rate</H1>


At the basal steady state, the rate of change of DcmR (and therefore fluorescence) is zero. As no ATC has been added to the system yet, the value of [ATC] is also zero.

This simplifies the equation to:

As we want our model to accurately predict the fluorescence, we will substitute the fluorescence value in place of the [DcmR] and rearrange:

Substituting in the value for δ1 that we found above and the basal steady state fluorescence level from the data (471 to 3 s.f.) gives the basal transcription rate as:
<H1>Expression rate constant and Michaelis - Menten constant</H1>

After the ATC has been added to the system, the value of [ATC] becomes non-zero. This means that the expression constant and the Michaelis – Menten constant start to affect the system.

To find the parameters that make the model’s output match the data values, we turned to the code that we had developed for parameter calibration. 
<A href="https://2014.igem.org/Team:Oxford/calculating_parameters" target="_blank">How are we calculating the parameters?</A>
This code gave the parameters as:
<H1>α1 = expression rate constant of <FONT style="font-style: italic;">dcmR</FONT> = 16.5 (fluorescence/min)</H1><H1>k1 = Michaelis - Menten constant of <FONT style="font-style: italic;">dcmR</FONT> = 0.015 (ml/ng)</H1></DIV></DIV></DIV><DIV class="row"><A href="#show8" class="show modelling-row" id="show8"><DIV class="modelling"><H1WHITE>Using the parameters and model analysis</H1WHITE></DIV></A><A href="#hide8" class="hide" id="hide8"><DIV class="modelling"><H1WHITE>Using the parameters and model analysis</H1WHITE></DIV></A><DIV class="list"><DIV class="white_news_block2"><H1>Entering the correct parameters</H1>

When the parameters that had been calculated above were entered into the model: 


alongside the correct inputs:


The graph below shows the model's predictions plotted in the same figure as the data points that the wet-lab team obtained for the system:

Plotting the model's output as a by interpolating between the calculated values makes the graph clearer:
<H1>Sensitivity</H1>
An important part of building mathematical models is sensitivity analysis of the results. This can be basically explained as wiggling all of the input values and parameters to see how much variations in each of these values affects the system output. This is especially important for finding parameters to describe the system as it is important to know what level of accuracy the values need to be found to provide a reasonable degree of prediction accuracy.
 
On top of this, it is possible to find what range of values the system is especially sensitive to. An example of this analysis is shown with a simple example that is relevant to our system below:

<H1>Stability</H1>
In our Engineering studies we have learnt detailed control theory. Control theory is an interdisciplinary branch of engineering and mathematics that deals with the behaviour of dynamical systems with inputs, and how their behavior is modified by feedback. The usual objective of control theory is to control a system so that its output follows a desired control signal, called the reference, which may be a fixed or changing value. This important because many dynamic systems can go unstable if they are given an unsafe set of input values and/or operating conditions.
 
However, as there are no feedback loops in this synthetic circuit, control theory analysis of this system isn't necessary.
<H1>Future experiment ideas from an Engineering design perspective</H1>

To see details about how the wet lab team then used this model to guide their work see our <A href="https://2014.igem.org/Team:Oxford/biosensor_optimisation">optimisation</A> page.
<H1>Reference</H1>
[1] Dr George Wadhams by personal communication (14/10/2014)


</DIV></DIV></DIV><DIV class="row"><A href="#show6" class="show wetlab-row" id="show6"><DIV class="wetlab"><H1WHITE>Is DcmR a repressor or an activator of the PdcmA/PdcmR promoters?</H1WHITE></DIV></A><A href="#hide6" class="hide" id="hide6"><DIV class="wetlab"><H1WHITE>Is DcmR a repressor or an activator of the PdcmA/PdcmR promoters?</H1WHITE></DIV></A><DIV class="list"><DIV class="white_news_block2">

In order to evaluate the action of DcmR we measured sfGFP expression driven by the PdcmA promoter both in the presence of DcmR and not (whether the cell was transformed with our POXON-2 plasmid (BBa_K1446003) or not). The cells were grown in EZ rich media for 4hrs followed by fluorometry readings. The fluorescence results were then normalised to the OD readings of each sample to account for differences in cell density. 



The results clearly show a decrease in sfGFP fluorescence upon addition and expression of  dcmR. Therefore we can confidently conclude that DcmR acts as a repressor in both directions for the PdcmA and PdcmR promoter. 
<H1>Conclusion: DcmR is a repressor of both the PdcmA and PdcmR promoter.</H1></DIV></DIV></DIV><DIV class="row"><A href="#show11" class="show wetlab-row" id="show11"><DIV class="wetlab"><H1WHITE>DcmR at the bidirectional promoter? - A new tool for molecular biologists</H1WHITE></DIV></A><A href="#hide11" class="hide" id="hide11"><DIV class="wetlab"><H1WHITE>DcmR at the bidirectional promoter? - A new tool for molecular biologists</H1WHITE></DIV></A><DIV class="list"><DIV class="white_news_block2">

The expression profile of PdcmA and PdcmR was analysed. This was achieved using sfGFP in place of dcmA (for PdcmA studies) or dcmR (for PdcmR studies). To investigate the relative stoichiometries of expression between the two sides of this bidirectional promoter we ran a fluorescence time course experiment for 16 hours. The OD plot is seen below. We determined the ratio of GFP fluorescence for expression in the PdcmA direction and the PdcmR direction. The results are shown below:
<STRONG>Figure 2</STRONG>shows the the gene region under investigation and manipulation in the design of our biosensor. PdcmA and PdcmR describe the two directions of a bidirectional promoter that is bound and modulated by DcmR


We can see from these results that at both growth stages the relative expression is weighted towards PdcmR. An interesting result is that in the presence of DcmR this ratio is reduced such that expression through PdcmR is relatively increased while expression through PdcmA is relatively decreased. This result is observed as a decrease in the PdcmA/PdcmR expression ratio. From our results this appears more statistically relevant in stationary phase. The cut off for exponential/stationary phase was set at 600 minutes as seen in the above graph. 
<H1>Our new bidirectional promoter offers a new tool to molecular biologists. Our new intergenic region is able to modulate different stoichiometries between genes inserted either side of it in response to the presence or absence of DcmR (available in our BioBrick BBa_K1446003). The relative stoichiometry (PdcmA/PdcmR activation) of ~0.9 being reduced to ~0.3 upon DcmR addition. This remarkable property will be crucial to any molecular biologists or future iGEM teams wishing to investigate the effect of different stoichiometries on the dynamics of various systems. </H1></DIV></DIV></DIV><DIV class="row"><A href="#show12" class="show wetlab-row" id="show12"><DIV class="wetlab"><H1WHITE>What influence does DCM have on the action of DcmR?</H1WHITE></DIV></A><A href="#hide12" class="hide" id="hide12"><DIV class="wetlab"><H1WHITE>What influence does DCM have on the action of DcmR?</H1WHITE></DIV></A><DIV class="list"><DIV class="white_news_block2">
In order to determine the influence of DCM on our DcmR repression system we grew MG1655 cells transformed with either PdcmA or PdcmR with DcmR and either with or without DCM. The cells were grown for 4 hours with no DCM added to grow to mid-log phase. After this time half the cells were treated with 30mM DCM and grown for an additional 2 hours before fluorescence was measured. The results can be seen below with fluorescence normalised for OD(600): 

For the PdcmA promoter both with and without DcmR:


For the PdcmR promoter both with and without DcmR: 


From these results we can see that in the direction of PdcmA DCM has a modulating effect of relieving DcmR repression thus DCM acts through depression. Interestingly, DCM appears to decrease sfGFP fluorescence in the absence of DcmR. 
	In the direction of PdcmR DCM appears to have no significant effect on the expression of sfGFP. Therefore in the native system we suggest that DCM has a modulating effect through depression of dcmA expression. Additionally DCM shows no effect on the auto regulation of dcmR expression. 

	In the context of our biosensor this means that DCM is capable of modulating expression of sfGFP through interaction through the PdcmA repressor DcmR.

	Ultimately we can conclude that DCM acts to encourage expression of sfGFP through derepression: consistent with our double repression model.


These images taken on an epi-fluorescence microscope (exposure time 0.4seconds) physically show the fluorescence from the same cells as used in the time course study as above.

<H1>Discussion: 

The results displayed here demonstrate the repression nature of DcmR on PdcmA and PdcmR bidirectional promoter. The modulation by DCM on the PdcmA promoter we have observed offers a constructive route towards a Dichloromethane biosensor. In this way we are able to use a reporter gene downstream of PdcmA to respond to the concentration of DCM. 

Additionally the nature of the bidirectional promoter offers a uniquely useful tool to molecular biologists. The relative stoichiometry of two genes inserted either side of the bidirectional promoter can be can be modulated through the addition of DcmR (BBa_K1446003 - Oxford iGEM). This can be utilised easily for investigations into the effect of protein stoichiometries between two interacting proteins. </H1></DIV></DIV></DIV><DIV class="row"><A href="#show9" class="show modelling-row" id="show9"><DIV class="modelling"><H1WHITE>Engineering analysis of the sfGFP data</H1WHITE></DIV></A><A href="#hide9" class="hide" id="hide9"><DIV class="modelling"><H1WHITE>Engineering analysis of the sfGFP data</H1WHITE></DIV></A><DIV class="list"><DIV class="white_news_block2">
The data that the sfGFP fluorescence data that the wet-lab team obtained presented the engineering team with the exciting possibility of fully parameterising our model of the synthetic circuit and therefore having the ability to make very good approximations of the overall behaviour of our biosensor.
<H1>Calculating the parameters</H1>
To calculate the parameters for behaviour of the second half of the genetic circuit (DcmR regulating sfGFP), we used a similar approach to the method we used to find the parameters for the top half of the circuit. This involves taking key bits of the data and analysing the corresponding equation.
<H1>Degradation rate constant (δ2)</H1>
The degradation half-life of the GFP protein is well documented in the literature and has a value of approximately 24 hours (1440 mins) [1] . This is very high and is one of the reasons that we chose GFP to be our fluorescence indicator. From this value of the half life, it is possible to calculate the degradation rate constant using mathematics:
<H1>Basal transcription rate constant (β3)</H1>
To find the basal transcription rate constant of the second half of the system (DcmR regulating sfGFP), we analysed the data of the fluorescence of just the Pdcma and the sfGFP gene. 

The data shown on the graphs below shows clearly that the fluorescence stops increasing when the bacteria stop growing in the log phase. This means that we can’t reliably use the data from the stationary phase to provide parameters. We have taken this point to be 500 minutes into the data measurement.



In terms of the modelling equations, this system is equivalent to setting [DcmR] to zero and ensuring that for this scenario, we set β1 = 0 (basal transcription rate of DcmR) . The mathematics below shows how we used the normalised data to find the basal transcription rate of sfGFP. Note how we’re using equation with parameters α3, β3 and k3 because the wet-lab team were able to prove that DcmR acts as a repressor on PdcmA.




Using the analytical solution outlined above, we were then able to characterize the single unknown parameter in the system (β3 - the basal rate of transcription of sfGFP). By again utilizing our data fitting algorithm alongside our hypothetical value for the basal rate of transcription of sfGFP,  β3 was calculated to be 4.6.


Now that we know two of the four parameters for the second half of our synthetic circuit (DcmR regulating sfGFP), we needed to calculate the final two parameters to complete our model. To do this, we analysed wet-lab data that showed the system in the presence of DcmR. This now means that the non linear term in our ODE is non zero and the analysis becomes
<H1>Expression rate constant (α3) and Michaelis - Menten constant (k3)</H1>
This data is shown here. Note the difference in the right hand plot reaffirming that DcmR acts as a repressor on PdcmA. 



Similar parameter calculation from this data gives:
<H1>α3 = 0.95 min^-1</H1><H1>k3 = 0.065 ml/ng</H1><H1>Using the completed model</H1>
Entering these parameters into the model and comparing the response to data allowed us to analyse how effective our modelling had been. An example of how the model fits with some of the data that we used to derive the parameters is shown below:



More on the further use of this model is available in the <A href="https://2014.igem.org/Team:Oxford/biosensor_optimisation">optimisation</A> section.

<H1>Reference</H1>
[1] Andersen JB1, Sternberg C, Poulsen LK, Bjorn SP, Givskov M, Molin S. New unstable variants of green fluorescent protein for studies of transient gene expression in bacteria.(1998) Appl Environ Microbiol. 64(6):2240-6
</DIV></DIV></DIV><DIV class="white_news_block2">
Oxford iGEM 2014



</DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Oxford/biosensor_characterisation">http://2014.igem.org/Team:Oxford/biosensor_characterisation</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Oxford/biosensor_characterisation" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Oxford/biosensor_characterisation" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Oxford/biosensor_characterisation&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Oxford/biosensor_characterisation&amp;oldid=399623" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>