<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_LZU-China_project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:LZU-China/project</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:LZU 2015</TITLE></P><DIV class="nav-wrapper container"><UL class="right"><LI><A href="https://2015.igem.org/Team:LZU-China/chz_index">Main</A></LI><LI><A href="#">Project</A></LI><LI><A href="https://2015.igem.org/Team:LZU-China/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:LZU-China/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:LZU-China/Attributions">Attributions</A></LI><LI><A href=" https://2015.igem.org/Team:LZU-China/safety">Safety</A></LI><LI><A href=" https://2015.igem.org/Team:LZU-China/notes">Notes</A></LI><LI><A href="https://2015.igem.org/Team:LZU-China/Collaborations">Collaboration</A></LI></UL><UL id="nav-mobile" class="side-nav"><LI><A href="#">Navbar Link</A></LI></UL></DIV><DIV class="content inorder"><DIV class="row"><DIV class="col s12 bgbanner"><DIV class="col s12"><DIV class="col s12 "><H5 class="center-align"> </H5><H2 class="left-align">This is our Project</H2><H3 class="center-align"> </H3></DIV></DIV></DIV></DIV></DIV><DIV class="content contain-box"><DIV class="row"><DIV><DIV class="col s12 m9 l9"><DIV class="row col s12"><DIV id="background" class="section scrollspy row"><DIV class="col s12"><H4> </H4></DIV><DIV id="popout" class="col s12"><H4 class="left-align">Background</H4><DIV id="Water pollution" class="section scrollspy row"><DIV id="popout" class="col s12"><H5> Water pollution </H5></DIV></DIV><P>  Water, such a vital material for all kinds of creatures, plays a rather important role in human's life. In China, Laozi, a well-known philosopher in the Spring and Autumn Period, thought highly of water and said, </P><P>The best of men is like water;</P><P>Water benefits all things.</P><P>And does not compete with them.</P><P>It dwells in (the lowly) places that all disdain</P><P>Wherein it comes near to the Tao.<SUP>[1]</SUP>
However, in our daily life, although water is noble, it is not so strong and usually gets sick. With the development of industry and modern society, water is badly polluted. So what is water pollution? Water pollution happens when toxic substances enter water bodies and degrade the quality of water. Although over 70% of the earth is covered by water, fresh water, which can be used by human, occupied only 2.53% of the global water resources. Over 80% of rivers in China have been polluted in different degree<SUP>[2]</SUP></P><DIV class="col s12"><DIV class="col s12 m6 l 6"><DIV class="card"><DIV class="card-content"><P>Some pictures about water pollution reference from the Internet.</P></DIV></DIV></DIV></DIV><P>Water pollution can be classified into organic, inorganic and so on. What we care most is heavy metal pollution because water heavy metal pollution is the current outstanding problem in the field of water pollution and becomes the focus of research in the controlling field.
          </P><P>Heavy metals from industrial processes can accumulate in nearby water body, which are toxic to marine life such as fish and shellfish, and subsequently to the humans who eat animals accumulating heavy metals. Heavy metals can slow development, result in birth defects and some are carcinogenic. The most commonly encountered toxic heavy metals in wastewater include Arsenic, Lead, Mercury, Cadmium, and the less common include Chromium, Copper, Nickel, Zinc<SUP>[3]</SUP>. Three kinds of heavy metals are of concern, including toxic metals (such as Hg, Cr, Pb, Zn, Cu, Ni, Cd, Co, Sn, etc.), precious metals (such as Pd, Pt, Ag, Au, Ru etc.) and radionuclides (such as U, Th, Ra, Am, etc.)<SUP>[4]</SUP>. Heavy metals bring about serious environmental pollution, threatening human health and ecosystem.<SUP>[5]</SUP></P><P>There are millions people are struggling with water pollution. In China, for example, about 75 percent of the population (or 1.1 billion people) are without access to unpolluted drinking water, according to standards from China<SUP>[6]</SUP>. And our university, Lanzhou University, located on northwestern China, where lacking water especially. What's worse, water pollution is also serious including heavy metal contaminants.
          </P><DIV id="Heavy metal detecting" class="section scrollspy row"><DIV id="popout" class="col s12"><H5> Heavy metal detecting </H5></DIV></DIV><P>1. Atomic Absorption Spectrometry, AAS</P><P>  Once other particles or photons with appropriate energy are running into an atom, the electron of the atom will perform the absorption transition, and the photons will be taken up. Different atoms have their specific corresponding photons, which mean they all have their own absorption spectrum. According to this phenomenon, people have created AAS to detect specific atom.</P><P>  To perform AAS to detect heavy metal ions quantitatively, an atomic absorption spectrometer is required, which is usually 50,000 to 90,000 dollars.</P><DIV class="col s12"><DIV class="col s12 m6 l 6"><DIV class="card"><DIV class="card-content"><P> Atomic Absorption Spectrometry, AAS </P></DIV></DIV></DIV></DIV><P>2. Atomic Fluorescence Spectrometry, AFS</P><P>  After the atom absorbed the photons, if the energy together with the photons is being released out, then we can detect its own AFS. The fluorescence intensity varies directly as the number of the atom.</P><P>  Similar to AAS, we need an AFSer to analyze AFS to detect heavy metal ions quantitatively. AFS is more accurate and sensitive than ASS, which means it’s more expensive than ASS. An AFSer may be up to 300,000 dollars.</P><DIV class="col s12"><DIV class="col s12 m6 l 6"><DIV class="card"><DIV class="card-content"><P>Atomic Fluorescence Spectrometry, AFS</P></DIV></DIV></DIV></DIV><P>3. Oscilloscopic polarography</P><P>  Also called as single sweep polarography. It is a method that can obtain the polarogram quickly and accurately. In order to accomplish it, a serrated pulse voltage should be added at the electrodes promptly during the late of the growth of the mercury drop on electrode. Thus, we are able to obtain a polarogram in several seconds.</P><P>  An oscilloscopic polarographer can reach 10,000 dollar.</P><DIV class="col s12"><DIV class="col s12 m6 l 6"><DIV class="card"><DIV class="card-content"><P>Oscilloscopic polarography</P></DIV></DIV></DIV></DIV><DIV class="col s12"><DIV class="col s12 m8 l8"><TABLE class="responsive-table striped"><TBODY><TR><TD>AASer</TD><TD>Max up to 300,000 dollars</TD></TR><TR><TD>AFSer</TD><TD>Max up to 66,000 dollars</TD></TR><TR><TD>Oscilloscopic polarography</TD><TD>Max up to 10,000 dollars</TD></TR><TR><TD>XRD</TD><TD>Max up to 333,000 dollars</TD></TR><TR><TD>ICP-MS</TD><TD>Max up to 500,000 dollars</TD></TR></TBODY></TABLE></DIV></DIV></DIV><P>         </P><DIV id="MFC" class="section scrollspy row"><DIV id="popout" class="col s12"><H5>MFC</H5></DIV></DIV><P>
              Instead of exploiting more water resources, which has more and more negative influence<SUP>[1]</SUP>, it's better to make less pollution and treat water pollution.<SUP>[2]</SUP> And we must know what kinds of contaminations in the water and how much are them. So we chose MFC (Microbial Fuel Cells) to detect the heavy metal in water.
          </P><P>So, what is MFC?
          </P><P>Microbial fuel cells (MFCs) provide new opportunities for the sustainable production of energy from biodegradable, reduced compounds. MFCs function on different carbohydrates but also on complex substrates present in wastewaters which is an ideal approach to solve both pollution problem and energy crisis. Compared with traditional chemical methods of sewage treatment, MFC is environmental friendly and widely adept by various condition<SUP>[3]</SUP>.
          </P><P>A MFC converts biomass energy directly into electricity. This can be achieved when bacteria switch from the natural electron acceptor, such as oxygen or nitrate, to an insoluble acceptor, such as the MFC anode. A typical microbial fuel cell consists of two compartments: anode and cathode compartments separated by a proton exchange membrane (PEM). In the anode compartment, microorganisms oxidize the fuel and generating CO<SUB>2</SUB>, electrons and protons. Electrons are transferred through an external electric circuit to the cathode compartment, while protons are transferred to the cathode compartment through the membrane. Electrons and protons are consumed in the cathode compartment, combining with oxygen to form water.
          </P><DIV class="col s12"><DIV class="card col s12 m6 l 6"><DIV class="card-content"><P>Figure 1. The working principle of a microbial fuel cell. </P></DIV></DIV></DIV><P>Substrate is metabolized by bacteria, which transfer the gained electrons to the anode. This can occur either directly through the membrane or via mobile redox shuttles. MED, redox mediator; Red oval, terminal electron shuttle in or on the bacterium.<SUP>[4]</SUP></P><P>
              After electron generated, the electron need to be transferred out of the microbe, The mechanism includes two methods:
          1.Direct transfer mechanism: Electrons are directly transferred from the microbe’s cell membrane to the anode surface (A), nanowires transfer the electron through conductive appendages, termed “nanowires”, grown by the bacteria.(B) 2.Indirect transfer mechanism: Microbes employ a secondary biomolecule ----- electron transfer mediators(ETMs) to shuttle the electron to the anode(C).




          </P><DIV class="col s12"><DIV class="card col s12 m6 l 6"><DIV class="card-content"><P>Figure2: Schematic diagram of extracellular electron transfer mechanisms for electrochemically active microorganisms in anode@</P></DIV></DIV></DIV><P>Although MFC is high efficiency of power output and environment friendly, most of the power generation MFCs are still in laboratory and not come into massive scale of industry. 
</P><P>One major drawback is the output power density is far less to adapt the standard in industry. electron transfer rate is determined by the potential difference, the reorganization energy and electron donor and receptor distance, main factors to determine the output power density of the microbial fuel cell is related to the electron transfer process, that is to say, the biological system of slow electron transfer rate is the bottleneck of the development of microbial fuel cell. 
</P><UL><LI>1.  Hou-gui, Z., Environmental issues and countermeasures in exploiting water resources of rivers. Journal of Chongqing University 2006. 5(2): p. 111-114.</LI><LI>2.  Rabaey, K. and W. Verstraete, Microbial fuel cells: novel biotechnology for energy generation. Trends in Biotechnology, 2005. 23(6): p. 291-298.</LI><LI>3.  Liu, L., et al., Electron Transfer Mediators in Microbial Electrochemical Systems. PROGRESS IN CHEMISTRY, 2014(11): p. 1859-1866.</LI><LI>4.  Chouler, J. and M. Di Lorenzo, Water Quality Monitoring in Developing Countries; Can Microbial Fuel Cells be the Answer? Biosensors, 2015. 5(3): p. 450.</LI></UL><DIV id="Riboflavin" class="section scrollspy row"><DIV id="popout" class="col s12"><H5>Riboflavin</H5></DIV></DIV><P>Riboflavin, a yellow-orange solid substance with poor solubility in water, was originally recognized as a growth factor in 1879 and named vitamin B, according to the British nomenclature system. 
          </P><P>
          Its IUPAC name is 7,8-Dimethyl-10-[(2S,3S,4R)-2,3,4,5-tetrahydroxypentyl]benzo[g]pteridine-2,4-dione<SUP>[8]</SUP>. Riboflavin has two active coenzyme forms, riboflavin 5'-phosphate (R5P; flavin mononucleotide [FMN]) and flavin adenine dinucleotide (FAD)<SUP>[9]</SUP>, which can both transfer electron. 
</P><H6>Fig 1 Riboflavin: C17H20N4O6. Reference: https://en.wikipedia.org/wiki/Riboflavin</H6><P>
          As riboflavin's structure shows, there is a transfer of two electrons from hydrogen and hydrid ions, so riboflavin can be regarded as a electron shuttle and even a kind of redox mediator. Though riboflavin is very important in mediator-driven bioelectricity generation, the detailed mechanism on electron transfer has not been precisely investigated yet<SUP>[10]</SUP>.
</P><P>
          Researchers find that increase of riboflavin biosynthesis can profounfly enhance extracellular electron transfer in bacteria and then improve the efficience of MFC. For example, the increase of riboflavin biosynthesis by Shewanella at the alkaline condition underlies the improvement of the electricity output in MFCs(Fig 2)<SUP>[11]</SUP>.
</P><DIV class="col s12"><DIV class="card col s12 m6 l 6"><DIV class="card-content"><P>Fig 2  Increased riboflavin synthesis underlies enhanced electron transfer in Shewanella.</P></DIV></DIV></DIV><P>We draw the conclusion that increased riboflavin synthesis can improve the MFC's production.</P><P>
        So riboflavin became our star. Since it can be expressed in bacteria, we can combine riboflavin and MFC to improve the electricity output of MFC. And it is easy and mature to measure the concentration of rirboflavin in aqueous solution based on the technique of OD (optical density). The peaks of absorbance occur at 223, 266, 373 and 445 nm(Fig 3)<SUP>[12]</SUP>. P. Drossler etc. investigated a wide range of pH values from pH -1.1 to pH  13.4 and made the absorption spectrum curve over different pH value(Fig 4)<SUP>[13]</SUP>.
</P><DIV class="col s12"><DIV class="card col s12 m6 l 6"><DIV class="card-content"><P> Fig 3 Absorption spectrum of riboflavin in aqweous solution. And the peaks of absorbance occur around at 223, 266, 373 and 445 nm</P></DIV></DIV></DIV><DIV class="col s12"><DIV class="card col s12 m6 l 6"><DIV class="card-content"><P>Fig 4 Absorption cross-section spectra of riboflavin in aqueous solutions at various values of pH. The riboflavin concentration used in the measurements was around 10-4 mol dm<SUP>3</SUP></P></DIV></DIV></DIV><P> In our laboratory, we used the multi-spectral scanner to analyse the optimal absorption wavelength and finally found absorption peak at 444nm is the optimum in pH 1.17. And our optimal absorption wavelength corresponds with previous researches. In order to make pH 1.17, we mixed 2ml 0.1mol/L HCl with 1ml riboflavin aqueous solutions extract from engineering bacteria. Then it is able to detect the concentration of rirboflavin by OD value.</P><P>There are many methods to get riboflavin by microorganism. In E.coli, a gene cluster, consisting of <EM>ribA, ribB, ribDG, ribH ribE</EM> and separating to different transcription units, regulates the synthesis of riboflavin. The details of riboflavin biosynthesis have been figured out(Fig 5)<SUP>[14]</SUP>.
        </P><DIV class="col s12"><DIV class="card col s12 m6 l 6"><DIV class="card-content"><DIV class="col s12 m9 l 9"><P>Fig 5 Biosynthesis of riboflavin and flavocoenzymes from Fischer, 2006 [10]. </P></DIV></DIV></DIV></DIV><P>Step A, GTP cyclohydrolase III; step B, GTP cyclohydrolase II; step C, 2-amino-5- formylamino-6-ribosylamino-4(3H)-pyrimidinone 5'-phosphate hydrolase; step D, 2,5- diamino-6-ribosylamino-4(3H)-pyrimidinone 5'-phosphate deaminase; step E, 5- amino-6-ribosylamino-2,4(1H,3H)-pyrimidinedione 5'-phosphate reductase; step F, 2,5-diamino-6-ribosylamino-4(3H)-pyrimidinone 5'-phosphate reductase; step G, 2,5- diamino-6-ribitylamino-4(3H)-pyrimidinedione 5'-phosphate deaminase; step H, hypothetical phosphatase; step I, 3,4-dihydroxy-2-butanone 4-phosphate synthase; step J, 6,7-dimethyl-8-ribityllumazine synthase; step K, riboflavin synthase; step L, flavokinase; step M, FAD synthetase; 1, GTP; 2, 2,5-diamino-6-ribosylamino-4(3H)- pyrimidinone 5'-phosphate; 3, 2-amino-5-formylamino-6-ribosylamino-4(3H)- pyrimidinone 5¡⬰hosphate; 4, 5-amino-6-ribosylamino-2,4(1H,3H)-pyrimidinedione 5'-phosphate; 5, 2,5-diamino-6-ribitylamino-4(3H)-pyrimidinone 5'-phosphate; 6, 5- amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione 5'-phosphate, 7; 5-amino-6- ribitylamino-2,4(1H,3H)-pyrimidinedione; 8, ribulose 5-phosphate; 9, 3,4-dihydroxy-2- butanone 4-phosphate; 10, 6,7-dimethyl-8-ribityllumazine; 11, riboflavin; 12, FMN; 13, FAD.</P><P>
        Green arrows mark the plant pathway; red, fate of the four-carbon precursor 9 derived from ribulose 5-phosphate.
</P><BLOCKQUOTE><UL><LI>1.     Laozi, Tao Te Ching. 516 BC. 8.</LI><LI>2.     Qi, F., C. Guodong, and M. Masao, Water Resources in China: Problems and Countermeasures. Ambio, 1999. 28(2): p. 202-203.</LI><LI>3.     Heavy metals in wastewater. 2013.</LI><LI>4.     Wang, J. and C. Chen, Biosorption of heavy metals by Saccharomyces cerevisiae: A review. Biotechnology Advances, 2006. 24(5): p. 427-451.</LI><LI>5.     Wang, J. and C. Chen, Biosorbents for heavy metals removal and their future. Biotechnology Advances, 2009. 27(2): p. 195-226.</LI><LI>6.     Hanson Ii, J.R., The World Economy: A Millennial Perspective (Book). Journal of Economic Literature, 2002. 40(4): p. 1256-1257.</LI><LI>7.     Hou-gui, Z., Environmental issues and countermeasures in exploiting water resources of rivers. Journal of Chongqing University, 2006. 5(2): p. 111-114.</LI><LI>8.     PubChem 493570.</LI><LI>9.     Riboflavin. Alternative Medicine Review, 2008. 13(4): p. 334-340.</LI><LI>10.    Jung, S.-H., et al., Impedance and Thermodynamic Analysis of Bioanode, Abiotic Anode, and Riboflavin-Amended Anode in Microbial Fuel Cells. Bulletin of the Korean Chemical Society, 2012. 33(10): p. 3349-3354.</LI><LI>11.    Yong, Y.C., et al., Increase of riboflavin biosynthesis underlies enhancement of extracellular electron transfer of Shewanella in alkaline microbial fuel cells. Bioresour Technol, 2013. 130: p. 763-8.</LI><LI>12.    Lu, C., et al., Photophysical and photochemical processes of riboflavin (vitamin B2) by means of the transient absorption spectra in aqueous solution. Science in China Series B: Chemistry, 2001. 44(1): p. 39-48.</LI><LI>13.    Dr?ssler, P., et al., pH dependence of the absorption and emission behaviour of riboflavin in aqueous solution. Chemical Physics, 2002. 282(3): p. 429-439.</LI><LI>14.    Kim, R., Biosynthesis of Vitamin B2 (Riboflavin) Studies on the Reaction Mechanism of Riboflavin Synthase. 2012.</LI></UL></BLOCKQUOTE><DIV class="col s12"><H4> </H4></DIV></DIV><DIV class="section scrollspy row"><DIV id="wetlab" class="col s12"><H4 class="left-align">Wet Lab</H4><DIV id="Introduction" class="section scrollspy row"><DIV id="popout" class="col s12"><H5>Introduction</H5></DIV></DIV><P>
          As it is known to all, Sherlock Holmes is a famous detective. Based on synthesis biology, our team creates a smart system called Micro Holmes which can monitor the pollutant inside the water in real time. Once the system was founded, through users' laptops, smartphones or some other media, we would acquire the statistics of pollutant which has been analyzed by the system. Reforming the E.Coli, we endowed them with the ability of sensing the pollutant inside the water and delivering the message of it. Therefore, raising them inside the Microbial Fuel Cell (MFC), they can rapidly augment the voltage as soon as the polluted occurred. Then, by the automatically collecting and analyzing equipment, we can know the concentration of pollutant converted by the instant electric signal. Escherichia coli, a tiny but full of power creature, perfectly plays an important part as Holmes in our project. So, are you interest in how he was created and how he works?
          </P><P>First of all Mr. Holmes must has a piercing eye to find the trail of pollutant; And after detecting pollutant, Mr. Holmes must has the ability to convey the message to us in time. Therefore, in order to constitute a series of inductivity parts which can be efficiently expressed in E.Coli BL 21 bacterial strains we link the efficient receptor of heavy metal ions with <EM>ribB</EM>--- the key synthesis gene of riboflavin. They can translate the information from the concentration of heavy metal ions to the riboflavin content, which is the product of reporter gene <EM>ribB</EM>. Riboflavin is a kind of redox mediator who works as a highly-efficient electronic transfer mediator. When the concentration of Riboflavin increased, it can improve the efficiency of electronic translocation between E.Coli and the anode and eventually augment the voltage. Combined with MFC (microbial fuel cell) system, we can measure the concentration of heavy metal ions in water through the voltage presented by MFC system and realize the conversion of heavy metal ions concentration to electrical signal. This method has lots of advantages, such as compatibility, high efficiency, instantaneity, reliability and so on.</P><P>The main idea of designing this experiment can be seen in the following flow chart.</P><DIV class="col s12"><DIV class="card col s12 m6 l 6"><DIV class="card-content"><P>Fig 1 Whole Experiment Flow Chart </P></DIV></DIV></DIV><P>In this project, parts were divided into two kinds. One is constitutive part, combined by various type of promoter, RBS, <EM>ribB</EM> and terminator. We constructed the constitutive parts to select a highly-expressed part of <EM>ribB</EM>. The other is inducible part with RBS, <EM>ribB</EM> terminator and promoter which can sense heavy metal ions. This kind of parts can highly express <EM>ribB</EM> so that it is practical to associate the concentration of heavy metal ions with the output of riboflavin. Therefore, putting the E.Coli BL 21 into the MFC, we collected the statistics of MFC on the condition that concentration of metal ion solution was already known. So we built the correlation model, then induced the E.Coli BL21 with the solution to be measured. It is easy to figure out the concentration by measuring the voltage of MFC. We can monitor the specific metal ions real-timely, quantitatively and swiftly through this way and more importantly, it has compatibility in the measurement of metal ion by changing the parts to detect different metal ions.</P><H4> </H4><DIV id="An efficient riboflavin synthesis gene:&lt;em&gt;ribB&lt;/em&gt;" class="section scrollspy row"><DIV id="popout" class="col s12"><H5>An efficient riboflavin synthesis gene:<EM>ribB</EM></H5></DIV></DIV><H5 class="left-align">An efficient riboflavin synthesis gene: <EM>ribB</EM></H5><H5 class="left-align">1 An Introduction of <EM><EM>ribB</EM></EM></H5><H5 class="left-align">1.1 The biosynthesis of riboflavin inside <EM>E.coli</EM></H5><P>
          Riboflavin,also called vitamin B2, can be synthesized by various kinds of plants and animals. The basic synthesis principal is similar, that one molecular of GTP and two molecular of ribulose 5-phosphate undergo a series of enzymatic reaction<SUP>[1]</SUP>. However, the operons vary from bacteria to bacteria. At present, we have made the principal clear in most kind of bacterium and in this project, we chose E.coli in which the riboflavin synthetic genes emerge as nonlinear gene cluster and was distributed into nonhomologous chromosome.
        </P><P>
          Fig 2-A shows the process of riboflavin synthesis and its relevant genes including RibA (coding GTP cyclohydrolase II), <EM>ribB</EM> (coding 3,4-dihydroxy-2-butanone-4-phosphate synthase), RibDG (coding a kind of bifunctional riboflavin specificdeaminase/reductase), RibH (coding lumazine synthase) and RibE (coding riboflavin synthase). The distribution of these gene can be seen in Fig 2-B<SUP>[2]</SUP>.
        </P><DIV class="row"><DIV class="col s12 m6 l6"><DIV class="card"><DIV class="card-content"><P>Fig 2-A The riboflavin biosynthesis inside Escherichia coli<SUP>[2]</SUP>.</P></DIV></DIV></DIV><DIV class="col s12 m5 l5"><DIV class="card"><DIV class="card-content"><P>Fig 2-B RibA, <EM>ribB</EM>, RibDG, RibH and RibE inside the E.coli are not on the same transcription unit but separate to different transcription units which have different promoters.</P></DIV></DIV></DIV></DIV><P>Genes in yellow are related to the synthesis of riboflavin while the grey are not known to be involved in flavin biosynthesis<SUP><SUP>[2]</SUP>.</SUP></P><H5 class="left-align">1.2 <EM><EM>ribB</EM></EM> and its function</H5><P>The synthesis of riboflavin is complicated because there are so many genes involved in this process. Besides, those genes are not focus on a same operon. According to some research, <EM>ribB</EM> is of vital importance to the synthesis of riboflavin which in charge of coding DHBP synthetase and can catalyze the synthesis of 3,4-dihydroxy-2-butanone-4-phosphate<SUP>[3]</SUP>. Researches conducted by Danielle Pedrolli and other researchers show that the FMN riboswitch, which is in the untranslated region of 5' terminal of <EM>ribB</EM>, can regulate the expression of <EM>ribB</EM> so that it can inhibit the biosynthesis of riboflavin. Thus, the enzyme translated by <EM>ribB</EM> becomes a limiter of riboflavin synthesis. Therefore, we can evidently improve the output of riboflavin by knocking out the FMN riboswitch or overexpressing <EM>ribB</EM>. Related experiment statistics can be referred from Fig 3. Compared with the whole riboflavin synthesis genes cluster, <EM>ribB</EM> has the advantages of convenience, high efficiency and short sequence as it is only 654bp. Consequently, our team choosed <EM>ribB</EM> as a report gene to construct parts and then builded the connection between heavy metal ions and <EM>ribB</EM> and finally used the product of <EM>ribB</EM> riboflavin to impact the voltage of MFC so that we realized the monitoring of heavy metal ions automatically, quantitatively and efficiently.
         </P><DIV class="row"><DIV class="col s12 m5 l5"><DIV class="card"><DIV class="card-content"><P>Fig 3-A After knocking out the FMN riboswitch, the expression of riboflavin increased. Abbreviation &quot;del&quot;, the test tube on the right side, is the E.coli knocked out the FMN riboswitch and the pNCO113 is control.</P></DIV></DIV></DIV><DIV class="col s12 m5 l5"><DIV class="card"><DIV class="card-content"><P>Fig 3-B After overexpressing <EM>ribB</EM>, the expression of riboflavin increased. &quot;<EM>ribB</EM>&quot;, the test tube on the right side, is the E.coli overexpressed <EM>ribB</EM> and the pNCO113 is control.</P></DIV></DIV></DIV></DIV><P>Fig 3 The expression of riboflavin increased<SUP>[2]</SUP>. </P><P>         </P><H5 class="left-align">2 The measurement data of several constitutive parts</H5><P>Totally, we constructed 5 kinds of constitutive parts and quantitatively measured their ability of producing of riboflavin. The character of these 5 parts can be seen in Table 1.
Table 1 Part of the message of measured part
</P><DIV class="row"><DIV class="col s12 m8 l8"><TABLE class="responsive-table striped"><H6><THEAD><TR><TH data-field="id">NO.</TH><TH data-field="name">Construction</TH></TR></THEAD><TBODY><TR><TD>K1755005</TD><TD>K608003(strong promoter +medium RBS)+ <EM>ribB</EM> + B0014(terminator)</TD></TR><TR><TD>K1755006</TD><TD>K608006(medium promoter + medium RBS) + <EM>ribB</EM> + B0014(terminator)</TD></TR><TR><TD>K1755007</TD><TD>K608002(strong promoter +strong RBS) + <EM>ribB</EM> + B0014(terminator)</TD></TR><TR><TD>K1755008</TD><TD>K608007(medium promoter +weak RBS) + <EM>ribB</EM> + B0014(terminator)</TD></TR><TR><TD>K1755009</TD><TD>K608004(strong promoter +weak RBS) + <EM>ribB</EM> + B0014(terminator)</TD></TR></TBODY></H6></TABLE></DIV></DIV><P>
        We cultured the transgened E.coli BL21 in M9 colorless culture media while at the same time set up a control with the original BL21. Compared with the control, it is seemingly that the BL21 with <EM>ribB</EM> gene part can product more riboflavin (riboflavin solution looks yellow), seeing Fig 4.
      </P><DIV class="col s12"><DIV class="card col s12 m6 l 6"><DIV class="card-content"><P>Fig 4 From left to right, respectively, the solution is BL21 control, BL21 with K1755005, BL21 with K1755006, BL21 with K1755009. 
Use spectrophotometer to measure the concentration of bacteria and riboflavin at different time. The consequence of the experiment can be seen in Fig 5.</P></DIV></DIV></DIV><DIV class="col s12"><DIV class="col s12 m6 l6"><H6 class="center-align">Figure5-A</H6></DIV><DIV class="col s0 m1 l1"> </DIV><DIV class="col s12 m5 l5"><P>Fig 5-A Bacteria concentration of  K1755005, K1755006, K1755009 bacterial strains vary with time.
</P></DIV></DIV><P>
        Concentration of the microbial strains of  K1755005, K1755006, K1755009 bacterial strains basically don't have difference at the same point in time, and the corresponding statistical analysis also showed that there are no significant differences between the three parts(the results are shown in Table 2). It shows that when bacterial strains are linked with K1755005, K1755006 or K1755009, the influence to the viability of the bacteria is very limited and can even ignore it. That's why the concentration of the microbial strains is basically the same with the different processing at the same time. In the following research we can directly compare the concentration of riboflavin to analyze the ability of riboflavin synthesis in different bacterial strains.

      </P><DIV class="col s12"><DIV class="col s12 m6 l6"><H6 class="center-align">Figure5-B</H6></DIV><DIV class="col s0 m1 l1"> </DIV><DIV class="col s12 m5 l5"><P>Fig 5-B Riboflavin concentration of  K1755005, K1755006, K1755009 bacterial strains vary with time.
          </P></DIV></DIV><P>
        The bacterial strain linked with <EM>ribB</EM> has a better performance of producing riboflavin and the promoting ability from strong to weak can be easily seen as following: K1755005, K1755009 and K1755006. Moreover, the distinction between K1755009 and K1755006 isn't obvious. The construction of these three parts are familiar and the the statistical analysis also showed that when the bacterial strain linked different parts, the ability of producing riboflavin is totally different with the original BL21 (The analysis result is shown in table 2), which proved that K1755005, K1755009 and K1755006 have all accomplished as we anticipated.
      </P><DIV class="col s12"><DIV class="col s12 m6 l6"><H6 class="center-align">Figure5-C</H6></DIV><DIV class="col s0 m1 l1"> </DIV><DIV class="col s12 m5 l5"><P>Fig 5-C Bacteria concentration of K1755007 and K1755008 bacterial strains vary with time.
          </P></DIV></DIV><P>Although there are some difference in bacteria density between strain K1755007 and strain K1755008 at the same time, the difference isn't obvious. Statistical analysis also shows that the difference of bacteria density between K1755007 and control group isn't obvious, but the difference of bacteria density between K1755008 and control group is remarkable.
          </P><DIV class="col s12"><DIV class="col s12 m6 l6"><H6 class="center-align">Figure5-D</H6></DIV><DIV class="col s0 m1 l1"> </DIV><DIV class="col s12 m5 l5"><P>Fig 5-D Riboflavin concentration of  K1755007 and K1755008 bacterial strains vary with time.
</P></DIV></DIV><P>
        After linking different parts, riboflavin expression level is distinctly different because K1755007 is constructed with strong promoter and strong RBS so that it has the strongest ability to produce riboflavin followed by K1755008, which is weaker than K1755007. The experimental phenomena coincide with theoretical analysis, showing that K1755007 and K1755008 both can work. Some problematic data have been removed.
      </P><DIV class="col s12"><DIV class="col s12 m6 l6"><H6 class="center-align">Figure5-E</H6></DIV><DIV class="col s0 m1 l1"> </DIV><DIV class="col s12 m5 l5"><P>Fig 5-E The ratio of riboflavin and bacteria concentration of different kinds of BL21 vary with time.
          </P></DIV></DIV><P>
        Through the ratio curve of riboflavin and OD value from the same serial number and the same time point, we can intuitively know that K1755007 and K1755008 can observably improve riboflavin production in E.Coli and K1755007 behaves batter than K1755008.

      </P><P>
        Using SPSS (Statistical Product and Service Solutions) paired sample t test for significance test, we obtain the following results on the condition that the confidence level is 95%.
      </P><P>Table 2 Significance test between different part and BL21</P><DIV class="row"><DIV class="col s12 m9 l9"><TABLE class="responsive-table bordered"><THEAD><TR><TH>NO.</TH><TH class="center-align">Bacteria concentration </TH><TH class="center-align">Riboflavin concentration </TH></TR></THEAD><TBODY><TR><TD>BL21</TD></TR><TR><TD>Sig.( bilateral)</TD><TD>Significant differences</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>Sig.( bilateral)</TD><TD>Significant differences</TD></TR></TBODY></TABLE>K1755005<TABLE><TBODY><TR><TD>0.142</TD><TD>No</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>0.031</TD><TD>Yes</TD></TR></TBODY></TABLE>K1755006<TABLE><TBODY><TR><TD>0.197</TD><TD>No</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>0.037</TD><TD>Yes</TD></TR></TBODY></TABLE>K1755007<TABLE><TBODY><TR><TD>0.278</TD><TD>No</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>0.000</TD><TD>Yes</TD></TR></TBODY></TABLE>K1755008<TABLE><TBODY><TR><TD>0.000</TD><TD>No</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>0.003</TD><TD>Yes</TD></TR></TBODY></TABLE>K1755009<TABLE><TBODY><TR><TD>0.126</TD><TD>No</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD>0.035</TD><TD>Yes</TD></TR></TBODY></TABLE></DIV></DIV><H5 class="left-align">3 The relationship between riboflavin and produce electricity</H5><P>Studies have shown that riboflavin can observably increase the voltage of MFC. By doing the experiment, we also confirmed that riboflavin can indeed improve the production capacity. And according to relevant data, we drew graph 5 (the data in this graph is measured in last year, the latest data is still in the measurement).
</P><DIV class="col s12"><DIV class="col s12 m6 l 6"><DIV class="card"><DIV class="card-content"><P>Fig 6 The effects of riboflavin on voltage.</P></DIV></DIV></DIV></DIV><P>
  Through the significant changes in the voltage of MFC before and after adding riboflavin, we can know that riboflavin can improve the voltage of MFC obviously.
</P><UL><LI>1.  Fischer, M. and A. Bacher, Biosynthesis of vitamin B2: Structure and mechanism of riboflavin synthase. Arch Biochem Biophys, 2008. 474(2): p. 252-65.
</LI><LI>
  2.  Pedrolli, D., et al., The <EM>ribB</EM> FMN riboswitch from Escherichia coli operates at the transcriptional and translational level and regulates riboflavin biosynthesis. FEBS J, 2015. 282(16): p. 3230-42.
</LI><LI>
  3.  Lin, J.W., Y.F. Chao, and S.F. Weng, Riboflavin synthesis genes ribE, <EM>ribB</EM>, ribH, ribA reside in the lux operon of Photobacterium leiognathi. Biochem Biophys Res Commun, 2001. 284(3): p. 587-95.
</LI></UL><DIV id="Pollutant sensor" class="section scrollspy row"><DIV id="popout" class="col s12"><H5>Pollutant sensor</H5></DIV></DIV><P>1  Cu<SUP>2+</SUP> sensor(&amp; Cr<SUP>6+</SUP>sensor)</P><P>1.1  BBa_K1755301</P><P>1.1.1  The composition of BBa_K1755301</P><P>We connected BBa_K1755401'MarO', as a promoter, with BBa_1755003'RBS+<EM>ribB</EM> CDS+B0014'to get BBa_K1755301.
</P><P>1.1.2  How does it work</P><P>MarO is a promoter, which belongs to the mar operon in E.coli. Genetic organization of the mar regulon in E.coli shows as Fig 2a. There are two promoter sites on the marO operator, and both of them can be bound by a kind of regulatory protein called MarR, which is coded by MarR and can repress the expression of marO(Fig 2b and Fig 2c)<SUP>[2]</SUP>. MarR is a very special and vital transcription factor, which can bound to either of marO's two promoter sites rs1 and rs2(rs means repressor binding site). And MarR repressor is homodimer. And it can form tetramer once there is copper for its special structure(Fig 3). Copper(II) oxidizes a cysteine residue (Cys80) on MarR to generate disulfide bonds between two MarR dimers, thereby inducing tetramer formation and the dissociation of MarR from its cognate promoter DNA, two sites of marO(Fig 4)<SUP>[1]</SUP>. Some experiments firmed the explanation(Fig 5)<SUP>[1]</SUP>.</P><P>
So we chose promoter marO as the copper sensor. Cu<SUP>2+</SUP> can oxidize Cys80 and make MarR apart from marO to derepress the MarR dimer. Than the promoter marO will start the following genes' transcription. In our part BBa_K1755301, the report gene is riboflavin. So we associated the copper concentration with the output of riboflavin. 
          </P><P>During our experiments, it is a big surprise for us to find that the marO is able to respond to Cr6+ as well though it is not so strong as Cu<SUP>2+</SUP>.    </P><DIV class="col s12"><DIV class="col s12"><DIV class="card col s12 m6 l6"><DIV class="card-content"><P>Fig 2a Genetic organization of the mar operon in E.coli.</P></DIV></DIV></DIV></DIV><DIV class="col s12"><DIV class="col s8"><DIV class="card"><DIV class="card-content"><P>Fig 2b The products of marR can repress the function of marO in E.coli</P></DIV></DIV></DIV></DIV><DIV class="col s12"><DIV class="col s12"><DIV class="card col s12 m6 l 6"><DIV class="card-content"><P>Fig 2c The 2 sites of promoter marO in E.coli </P></DIV></DIV></DIV></DIV><DIV class="col s12"><DIV class="col s12"><DIV class="card col s12 m6 l 6"><DIV class="card-content"><P>Fig 3 Crystal structure of copper(II)-oxidized MarR5CS(80C).</P></DIV></DIV></DIV><P>(a) Copper(II)-oxidized MarR5CS(80C) tetramer with two monomers from each dimer (chain A¨Cchain B and chain A'¨Cchain B') colored blue and magenta, respectively. Atoms in the side chains of Cys80 from all four of the monomers are shown as yellow spheres. (b) Close-up of the two disulfide bonds between two MarR dimers. The disulfide bonds shown in yellow sticks. Only residues Asp67(Asp67'), Cys80(Cys80') and Gly82(Gly82') from each monomer are shown for clarity. (c) Superposition of the structures of copper-oxidized MarR5CS(80C) (blue) and the previously reported SAL-complexed WT MarR(orange).</P></DIV><DIV class="col s12"><DIV class="col s12"><DIV class="card col s12 m6 l 6"><DIV class="card-content"><P>Fig 4 The mechanism of Cu<SUP>2+</SUP> derepress of MarO</P></DIV></DIV></DIV></DIV><DIV class="col s12"><DIV class="col s12"><DIV class="card col s12 m6 l 6"><DIV class="card-content"><P>Fig 5 Some experiments results verified the explanation of the mechanism of MarO<SUP>[1]</SUP></P></DIV></DIV></DIV></DIV><P> 1.1.3 Results </P><P> We linked BBa_K1755301 into pSB1C3 and transformed the new plasmid into E.coli BL-21. Then we detected the concentration of riboflavin through measuring the absorbance under 444nm and found that riboflavin production basically response to the change of metal ion concentration as it present a linear relationship with Cu2+  concentration, which indicated that BBa_K1755301 can work as we anticipated. </P><P>1.2  BBa_K1755302</P><P>1.2.1  The composition of BBa_K1755302</P><P>We connected BBa_K190024&quot;CueO+RBS&quot;, as a, with BBa_1755002&quot;<EM>ribB</EM> CDS+B0014&quot;to get BBa_K1755302.  The former part is a sensor and the latter is a reporter.</P><P>1.2.2  How does it work</P><P>CueO can express periplasmic copper-binding proteins, CueO , a multi-copper oxidase for detoxification<SUP>[1]</SUP>, which belongs to CueR Regulon gene and can be regulated by copper-responsive transcription factor CueR, too. CueO can oxidate Cu+ to Cu2+ and decrease the toxicity of copper<SUP>[3, 4]</SUP>. On the CueO promoter, CueR protectes -41 to -17 on top strand and -42 to -19 on bottom strand to repress the expression of CueO<SUP>[1]</SUP>. And it can be derepressed by Cu<SUP>2+</SUP>, too. So the CueO promoter can sense the concentration of copper and linkage it with the production of riboflavin.
</P><P>1.2.3  Results
</P><P>
          We linked BBa_K1755302 into pSB1C3 and transformed the new plasmid into E.coli BL-21. Then we detected the concentration of riboflavin through measuring the absorbance under 444nm and found that riboflavin production basically response to the change of metal ion concentration as it present a linear relationship with Cu<SUP>2+</SUP>  concentration, which indicated that BBa_K1755302 can work as we anticipated. 
</P><P>2  Hg<SUP>2+</SUP> sensor</P><P>2.1  BBa_K1755305</P><P>2.1.1  The composition of BBa_K1755305</P><P>We connected BBa_K346002'PmerT', as a promoter, with BBa_1755003'RBS+<EM>ribB</EM> CDS+B0014'to get BBa_K1755305.
</P><P>2.1.2  How does it work</P><P> PmerT is a kind of promoter from Tn21 mercury resistance (mer) operon. When there is no Hg2+, it cannot start the transcription. Upon binding the cognate metal ions, the metallated MerR homodimer causes a realignment of the promoter so that RNA polymerase contacts the -35 and -10 sequences leading to open complex formation and transcription.
(see more details http://  parts.igem.org /Part:BBa_K346002).</P><P> So PmerT promoter can be induced by Hg2+ and start the coding of <EM>ribB</EM> downstream to increase the output of riboflavin. And there is some quantity relationship about the concentration of Hg2+ and the production of riboflavin. We can calculate the the concentration of Hg2+ by our ‘Micro Holmes’ system.</P><P>2.1.3 Results</P><P>We transformed BBa_K1755305 into E.coli BL-21. Then we detected the concentration of riboflavin through measuring the absorbance under 444nm. Unfortunately, we didn't find that riboflavin production respond to the change of Hg2+ concentration, which indicated that BBa_K1755305 can not work as we anticipated. 
</P><DIV class="row"><DIV class="card col s12 m7 l7"><DIV class="card-content"><P>Fig 8: Testing K1755305 under different Hg<SUP>2+</SUP> environment</P></DIV></DIV></DIV><P>
          3 F- sensor
</P><P>3.1  BBa_K1755024
</P><P>3.1.1  The composition
</P><P>We used BBa_K911003 as a promoter and connected it with BBa_1755003'RBS+<EM>ribB</EM> CDS+B0014'to get 3.1  BBa_K1755024.
</P><P>3.1.2  How does it work
</P><P>BBa_K911003 is a fluoride sensitive riboswitch that is highly sensitive to the F-. It is a positive regulator so F- can induce the promoter to transcript. Since the report gene is at the downstream of fluoride promoter, the riboflavin production can be influenced by the change of F- concentration. So we can use the concentration of riboflavin to determine the concentration of F- and even apply into our ‘Micro Holmes’ system.
</P><P>3.1.3  Results
</P><P>We transformed BBa_K1755024 into E.coli BL-21. Then we detected the concentration of riboflavin through measuring the absorbance under 444nm. However, we didn't find regular change relationship between riboflavin production and Hg2+ concentration. The result showed that BBa_K1755024 can not work as we anticipated. 
</P><UL><LI>1.  Hao, Z., et al., The multiple antibiotic resistance regulator MarR is a copper sensor in Escherichia coli. Nat Chem Biol, 2014. 10(1): p. 21-8.</LI><LI>2.  R., C.X.H.Z.C.P., Protein photocrosslinking reveals dimer of dimers formation on MarR protein in Escherichia coli.China Science： Chemistry, 2012. 42(2): p. 223-225.</LI><LI>3.  Yamamoto, K. and A. Ishihama, Transcriptional response of Escherichia coli to external copper. Mol Microbiol, 2005. 56(1): p. 215-27.</LI><LI>4.  Stoyanov, J.V., J.L. Hobman, and N.L. Brown, CueR (YbbI) of Escherichia coli is a MerR family regulator controlling expression of the copper exporter CopA. Molecular Microbiology, 2001. 39(2): p. 502-512.</LI></UL><DIV id="Our MFC" class="section scrollspy row"><DIV id="popout" class="col s12"><H5>Our MFC</H5></DIV></DIV><P>1.Microbial Fuel Cells (MFC) are a promising technology for electricity production from a variety of materials, such as natural organic matter, complex organic waste or renewable biomass, and can be advantageously combined with applications in waste water treatment<SUP>[1]</SUP>. However, MFC have not achieved commercialization yet on account of low power output, which is mainly due to the low efficiency of extracellular electron transfer without the help of redox mediators added manually into the reactor, such as neutral red, thionin<SUP>[2]</SUP>.</P><P>The E. coli K12 was cultivated and then inoculated, and the E.coli thus obtained is referred to as ‘original bacteria’ which was subjected to a fuel cell discharge process until the cell voltage was below 0.05 V. The bacteria suspension that underwent fuel cell discharge was inoculated into a fresh medium and cultivated. The resulting E. coli is referred to as ‘electrochemical bacteria of generation I’, which was again subjected to fuel cell discharge until the cell voltage was below 0.05 V. Repeating the inoculation, cultivation and discharge cycles produced electrochemical bacteria of generation II, III, and so on<SUP>[3]</SUP>. </P><DIV class="row"><DIV class="col s12 m5 l5"><DIV class="card"><DIV class="card-content"><P>Fig 3-A After knocking out the FMN riboswitch, the expression of riboflavin increased. Abbreviation &quot;del&quot;, the test tube on the right side, is the E.coli knocked out the FMN riboswitch and the pNCO113 is control.</P></DIV></DIV></DIV><DIV class="col s12 m5 l5"><DIV class="card"><DIV class="card-content"><P>Fig 3-B After overexpressing <EM>ribB</EM>, the expression of riboflavin increased. &quot;<EM>ribB</EM>&quot;, the test tube on the right side, is the E.coli overexpressed <EM>ribB</EM> and the pNCO113 is control.</P></DIV></DIV></DIV></DIV><P>The variation of the anode and cathode potentials during the discharge process of the electrochemical bacteria of generation II.<SUP>[3]</SUP>.</P><P>In this study, we constructed the genetically modified E.coli with metal ion bio-sensor coupling <EM>ribB</EM>, which expression product is a redox mediator, to monitor Cu2+ concentration in aquatic environment based on dependency of peak voltage in MFC on Cu2+ content. </P><P>2.</P><TABLE class="responsive-table striped"><TBODY><TR><TD>Part ID</TD><TD>Characterization</TD><TD> </TD></TR><TR><TD>K1755301</TD><TD>MarO(Cu sensor) + B0032(RBS)+ <EM>ribB</EM> + B0014(terminator)</TD><TD>LZU-201501</TD></TR><TR><TD>K1755302</TD><TD>K190024(Copper Promoter) + <EM>ribB</EM> + B0014(terminator)</TD><TD>LZU-201502</TD></TR><TR><TD>K1755303</TD><TD>K540001£ºcobalt-sensitive promoter + B0032(RBS) + <EM>ribB</EM> + B0014(terminator)</TD><TD>LZU-201503</TD></TR><TR><TD>K1755305</TD><TD>K911003(fluoride sensitive riboswitch)+ B0032(RBS) + <EM>ribB</EM> + B0014(terminator)</TD><TD>LZU-201504</TD></TR><TR><TD>K1755024</TD><TD>K911003(Fluoride sensitive riboswitch)+ B0032(RBS) + <EM>ribB</EM> + B0014(terminator)</TD><TD>LZU-201505</TD></TR></TBODY></TABLE><P>Riboflavin, also known as Vitamin B2, is a efficient redox mediator as well as a stimulator of MFC. In our study, we constructed recombinant plasmids with several different kind of pollutant substrate bio-sensors coupling <EM>ribB</EM>. Bio-sensors includes copper sensor, cobalt sensor and fluoride sensor. Then, we transformed recombinant plasmids into E.coli BL 21. As time is limited, </P><P>we have only made a deep research on genetic engineered bacteria with copper bio-senso coupling <EM>ribB</EM> which can produce riboflavin when added copper in MFC anode medium. As for parts with other ion sensor, we just made them to be the formation of Biobrick. </P><P>3.</P><P>In our study, we've designed a novel MFC system. For anode strain, we've cloned a copper sensor sequence coupling with <EM>ribB</EM> into E.coli BL 21. The genetically modified bacteria is able to detect Cu 2+ and produce riboflavin to boost electrical generation. As a result, MFC peak voltage can reflect Cu2+ concentration. Phosphate Buffer'PBS'with 100 mmol/L potassium ferricyanide was packed into cathode chamber. </P><P>We aimed to build a quantitative monitor system of Cu2+ via measuring peak voltage, and finally found the interralationship between peak voltage and Cu2+ concentration. We concluded an empirical formula for our devices:</P><P>Peak voltage in MFC as genetically modified E.coli were treated with different amount of Cu2+. </P><P>We can deduce the concentration of Cu2+ in water by measuring peak voltage in MFC.</P><P>In a range of copper's low-concentration, the peak voltage rose with the increase of concentration of Cu<SUP>2</SUP>+. However, because of toxic effects of heavy metal to bacterium, when the copper content in water is over a certain concentration (about 2 mg/L) the peak voltage decreased as the concentration rises.  </P><P>MFC Assembly and Operation</P><P>1. Medium Preparation </P><P>  1.1 M9 medium (per liter contains) PH=7.2</P><P>     NH4Cl                1g</P><P>     Casein hydrolysate      1g</P><P>     KH2PO4               1g             </P><P>     Na2HPO4¡¤12H2O  15.116g  </P><P>     10ml sterilized 20% glucose and 1ml 1 mol/L MgSO4¡¤7H20 were put after 121¡䟳terilization.</P><P>1.2 LB medium (per liter contains] PH=7.0</P><P> NaCl                  10g</P><P>Yeast Extract            5g</P><P>Tryptone               10g</P><P>Before 121℃ terilization, LB medium was Aliquoted into tube (each for 5mL) and conical flask (each for 150ml), respectively used for the activation of the strain and the amplification culture.</P><P>2. Preparation of phosphate buffer (per liter contains)</P><P>K2HPO4?3H2O                       6.618g</P><P>KH2PO4                            2.858g</P><P>Potassium ferricyanide           32.925g</P><P>3. Bacteria activation and amplification culture</P><P>3.1 Bacteria activation</P><P>Materials: tips, pipettes, sterilized tubes, test-tubes rack, LB medium (tube)</P><P>Bench was cleaned by ultraviolet light vertical irradiate for 20 minutes. Then 200ul 1755301 bacterial glycerol stock  and 5ul 25mg/ml chloramphenicol were added into a tube,and was then shaked for 12 hours in a shaker at 37 degrees Celsius.</P><P>3.2 Amplification culture</P><P>Materials: tips, pipettes, sterilized tubes, test-tube rack, LB medium (conical flask)</P><P>Bench was cleaned by ultraviolet light vertical irradiate for 20 minutes.1.5ml bacteria solution and 150ul 25mg/ml chloramphenicol were added into a conical flask of LB medium, and was then shaked for 12 hours in a shaker at 37 degrees Celsius.</P><P>4. MFC Assembly</P><P>4.1 Treatment of carbon fiber felt</P><P>Preparation:</P><P>1mol/L NaOH     1L water + 40g NaOH</P><P>1mol/L HCl       1L water + 90.9mL concentrated hydrochloric acid</P><P>The carbon fiber felt was first cleaned by deionized water and was then boiled with 1mol/L NaOH and 1mol/L HCl for 30 minutes. After each boiling,it was cleaned by deionized water until its pH value is 7. Finaly it was dried at 105¡£C for 20 minutes.</P><P>4.2 Treatment of proton exchange membrane</P><P>Preparation:</P><P>0.5mol/LH2SO4    1Lwater + 27.17mL concentrated hydrochloric acid</P><P>The proton exchange membrane was boiled in turn in 30% H2O2, deionized water, 0.5mol/LH2SO4 and deionized water for 30 minutes.</P><P>4.3Sterilization</P><P>2 250ml graduated cylinders,2 250ml conical flasks,12 50ml centrifuge tubes,cathode chamber and anode chamber(with magnon),sterile water,gaskets and carbon fiber felt(fixed on titanium wire)</P><P>4.4Combination of cathode chamber and anode chamber</P><P>A proton exchange membrane was put between the anode and the cathode,fixed with gaskets and parafilm.Check for leaks with sterile water.</P><P>4.5Treatment of bacteria solution</P><P>Bacteria was cleaned with M9 medium, and the absorbance (OD values) was measured in the 600nm wavelength and were adjusted to the 1.3--1.4.</P><P>4.6Add liquids to cathode chamber and anode chamber</P><P>4.6.1anode chamber:</P><P>24ml bacteria solution after treatment,216ml M9 medium,appropriate amount 20mg/ml CuSO4 mother solution (make sure the final concentration of Cu<SUP>2</SUP>+ are 0,1,2,3,4mg/L)</P><P>4.6.2cathode chamber:</P><P>240ml PBS (with 100mmol/L K3[Fe(CN)6)])</P><P>4.7Fixed carbon fiber felt</P><P>Carbon fiber felt was put in front of the carbon fiber felt,and the titanium wire tied to it was fixed on the bottle cap of cathode chamber and anode chamber.</P><P>5.MFC Operation</P><P>The MFC was put on the magnetic disk, the temperature and the speed of magnon were set, the temperature sensors were put into liquids voltage was recorded every other minute.Find the relationship between peak voltage and the concentration of Cu<SUP>2+</SUP>.</P><P>6.Determination of bacteria concentration and expression of riboflavin</P><P>3ml bacteria solution was taken out from anode chamber every 12 hours(0h,12h,24h,36h,48h, their absorbance (OD values ) were measured in the 600nm wavelength. 1ml supernatant  was added into 2mL 0.1mol/L HCl after centrifugation(12000g,2min) their absorbance (OD values)  were measured in the 444nm wavelength,which suggects the concentration of riboflavin.</P><P>1.Oliveira, V. B., Simoes, M., Melo, L. F., and Pinto, A. M. F. R. (2013) Biochemical Engineering Journal 73, 53-64</P><P>2.Rabaey, K., and Verstraete, W. Trends in Biotechnology 23, 291-298</P><P>3.Zhang, T., Cui, C., Chen, S., Ai, X., Yang, H., Shen, P., and Peng, Z. (2006) Chemical Communications </P></DIV></DIV></DIV><DIV class="row"><DIV id="drylab" class=" section scrollspy col s12"><H4 class="left-align">Dry Lab</H4><DIV id="Overview" class="section scrollspy row"><DIV id="popout" class="col s12"><H5>Overview</H5></DIV></DIV><P>‘Micro Holmes concentration measurement system’ is a system containing electronic components, mechanical components, biology components (several kinds of genetic engineered bacteria) and software working with them. Using this system, you can monitor the contaminants concentrations whenever and wherever in a cheap and easy way. </P><P>A typical ‘Micro Holmes concentration measurement system’ at least include:</P><UL><H6><LI>A Microbial Fuel Cell -- cathode chamber and anode chamber -- carbon fiber felt  -- proton exchange membrane</LI><LI>A MCU -- such as pcDuino3 board</LI><LI> A voltage magnification circuit board</LI><LI>An Analogue-to-Digital Conversion chip</LI><LI>A set of input and output devices--C such as touch screen</LI><LI>One or several kinds of bacteria--C such as genetic engineered E.coli</LI></H6></UL><P>Till now, we have made two versions of Micro Holmes concentration measurement system, the professional version and the portable version. Their main principles are totally same. The only difference is that the professional version has a set of complicated MFCs (6 cathode and anode chambers with thermostats and magnetic stirrers) and can automatically change sample. On the other hand, the portable version has its own incomparable merits. Portable ones are cheap, handiness and fit for complex working environment.</P><P>We will bring a portable version of our ‘Micro Holmes concentration measurement system’ to the Giant Jamboree. So this Manual will concentrate on the portable version.</P><DIV id="Quick Guide" class="section scrollspy row"><DIV id="popout" class="col s12"><H5>Quick Guide</H5></DIV></DIV><P>1.  Power On: Press the power button more than 3 seconds till the power light sparkles.</P><P>2.  The touch screen will show the ‘Welcome screen’, please wait until the ‘MODE’ screen appears.</P><P>3.  Select the ‘Standard Curve (Initialization)’ mode to work out a formula showing the correlation between the concentration of contaminants and the output voltage of MFC (or load an existing one)</P><P>4.  If you want to load an existing formula, just click the ‘OPEN’ icon, and then select a save file (Then please go to step 7). If you want to work out a new formula please click ‘ADD’, and then input the current concentration of contaminants ended with the button ‘OK’.</P><P>5. The system will begin to real-time monitor the output voltage of MFC within a period of time until the system thinks it has gotten the peak voltage value. And then the concentration of the contaminants and the peak output voltage of the MFC will appear in the ‘History’ list.</P><P>6. Change the standard sample manually (portable version user) or automatically (professional version user), click ‘ADD’, and then input the new concentration of current contaminants ended with the button ‘OK’. Repeat step 5 and step 6 until all the standard samples have been measured. Then click ‘Finish’ to calculate the fitting formula.</P><P>7. Change the sample (to be tested) manually (portable version user) or automatically (professional version user). And then select the &quot;Real-time Monitoring&quot; mode.</P><P>8. In this mode, you can watch the real-time output voltage and the concentration of the sample calculated using the formula gotten in the ‘Initialization’ mode. In the right zone of the screen, you can also get the historical information and observe the variation trend of the sample concentration.
</P><P>1. We offer ‘micro Holmes cloud services for all the system users. So when your device is in the &quot;Real-time Monitoring&quot; mode, you can acquire the real-time data through the Internet any place any time.
</P><P>1. You can also use a mobile phone with Bluetooth to control your ‘Micro Holmes concentration measurement system’ in a convenient way.
</P><DIV id="Technical Details (Instruction of mv Analysis)" class="section scrollspy row"><DIV id="popout" class="col s12"><H5>Technical Details (Instruction of mv Analysis)</H5></DIV></DIV><P>About our device, I want to state from two part.</P><H6>Hardware overview
</H6><P>Our device's hardware is component of three device, the first is  MCU pcDuino3, pcDuino3 is a high performance, cost effective single board computer. It runs operation systems such as Ubuntu Linux and Android. pcDuino3 has HDMI interface to output its graphic desktop screen. It could support multi-format 1080p 60fps video decoder and 1080p 30fps H.264 and MPEG4 video encoder with its built-in hardware video processing engine. It targets specially the fast growing demands from the open source community. pcDuino3 provides easy-to-use tool chains and is compatible with the popular Arduino ecosystem such as Arduino Shields.</P><P>Features &amp; Highlights:
</P><UL><LI>1. 100% compatible with original Arduino Shields</LI><LI>2. 100% compatible with Linux and Android</LI><LI>3. Further support for:</LI><LI>4. C, C ++ with GNU tool</LI><LI>5. Java with standard Android SDK Python</LI><LI>6. Arduino pin header, Aduino UNO Slots: 14x GPIO, 2x PW M, 6x ADC, 1x UART, 1xSPI, 1x I2C</LI><LI>7. Ethernet 10M/100Mbps, WiFi, SATA</LI></UL><P>This board has very sufficient GPIO pins, it can make everything you think out, so we take it.</P><P>
And the next is the another important device, named voltage magnification, since the voltage from microorganism reaction is very low, so if we want to read the value precise, before reading this value, we must amplify it, and then connect it to my MCU's simulation pins, and by the helping of A-D chip, we can make this simulate value to digital value, then, the MCU can process this value, what out's doing is display this value to screen this screen is a touchable screen, so the input to the MCU, we can make a software to access.
</P><P>the connection of screen and pcduino
</P></DIV></DIV><DIV id="parts" class="section scrollspy row"><DIV id="popout" class="col s12"><H4 class="left-align">Advantages</H4></DIV></DIV><P>Our device has the function of detecting water pollution. Compared with the traditional detecting methods, our device has the advantages as convenient and economical.</P><P>Traditional way as AASer, AFSer, Oscilloscopic polarography, XRD and ICP-MS are very cumbersome so that in general, it needs people to collect the water sample and bring it back to their laboratory for testing. And at the same time, those devices are not only far more expensive for the public and family using but unaffordable for some developed countries as well.</P><P>Furthermore, traditional monitoring device is complex in using. People without particularly training cannot fluently use it. These problems above limited its application severely, especially when people need to monitor the water quality in wild, in long term real time or for normal families.</P><P>
            Our device, however, is light small economical and easy to use. It is quite easy for us to take it everywhere with its volume only large as two Oxford Advanced Learner's Dictionary. It has a graphical touch screen which will help the learners easy to get starting. By the way, our device can be put nearby the polluted water so that we can monitor the water at real time. Some families can utilize our device to monitor their tap water. We also provide professional version that can not only solve the problem of the hard control of traditional MFC but own the potential ability to meet the accurate and real time needs of some factories and laboratories.
          </P><P>
            Let's show you the manufacturing cost of our device. We provide three kinds of the version including the starter version, portable version and professional version.


          </P><P>Starter version</P><DIV class="row"><DIV class="col s12 m9 l9"><TABLE class="responsive-table striped"><TBODY><TR><TD>Items</TD><TD>USD</TD></TR><TR><TD>pcDuino board</TD><TD>48</TD></TR><TR><TD>LCD touch screen</TD><TD>48</TD></TR><TR><TD>voltage magnification circuit board</TD><TD>10</TD></TR><TR><TD>Analogue-to-Digital Conversion chip</TD><TD>8</TD></TR><TR><TD>Microbial Fuel Cell</TD><TD>19</TD></TR><TR><TD>proton exchange membrane (6 cm^2)</TD><TD>2</TD></TR><TR><TD>Carbon fiber felt</TD><TD>8</TD></TR><TR><TD>genetic engineered E.coli</TD><TD>1</TD></TR><TR><TD>Case and other cost</TD><TD>7</TD></TR><TR><TD>Total</TD><TD>151</TD></TR></TBODY></TABLE></DIV></DIV><P>Portable version</P><DIV class="row"><DIV class="col s12 m9 l9"><TABLE class="responsive-table striped"><TBODY><TR><TD>Items</TD><TD>USD</TD></TR><TR><TD>Raspberry Pi board</TD><TD>32</TD></TR><TR><TD>LCD screen and keyboard</TD><TD>12</TD></TR><TR><TD>voltage magnification circuit board</TD><TD>9</TD></TR><TR><TD>Analogue-to-Digital Conversion chip</TD><TD>7</TD></TR><TR><TD>Microbial Fuel Cell</TD><TD>18</TD></TR><TR><TD>Carbon fiber felt</TD><TD>8</TD></TR><TR><TD>proton exchange membrane (6 cm^2)</TD><TD>2</TD></TR><TR><TD>genetic engineered E.coli</TD><TD>1</TD></TR><TR><TD>Case and other cost</TD><TD>7</TD></TR><TR><TD>Total</TD><TD>96</TD></TR></TBODY></TABLE></DIV></DIV><P>Professional version</P><DIV class="row"><DIV class="col s12 m9 l9"><TABLE class="responsive-table striped"><TBODY><TR><TD>Items</TD><TD>USD</TD></TR><TR><TD>pcDuino board</TD><TD>48</TD></TR><TR><TD>LCD touch screen</TD><TD>48</TD></TR><TR><TD>voltage magnification circuit board</TD><TD>10</TD></TR><TR><TD>Analogue-to-Digital Conversion chip</TD><TD>8</TD></TR><TR><TD>a set of complicated MFCs (6 cathode and anode chambers)</TD><TD>66</TD></TR><TR><TD>thermostats and magnetic stirrers</TD><TD>99</TD></TR><TR><TD>Carbon fiber felt</TD><TD>8</TD></TR><TR><TD>proton exchange membrane (6 cm^2)</TD><TD>2</TD></TR><TR><TD>genetic engineered E.coli</TD><TD>1</TD></TR><TR><TD>Case and other cost</TD><TD>7</TD></TR><TR><TD>Total</TD><TD>297</TD></TR></TBODY></TABLE></DIV></DIV><DIV id="Accessibility and portability" class="section scrollspy row"><DIV id="popout" class="col s12"><H4 class="left-align">Accessibility and portability</H4><P>The accessibility and portability of our &quot;Micro Holmes Measurement System&quot; have been demonstrated by both TJU (Tianjin University, China) and SZU (Shenzhen University, China).</P><P>As previously described in the Day lab section, our &quot;Micro Holmes Measurement System&quot; contains both biology components and other non-biology components. We ask for TJU to test</P><P>We invited TJU, which is studying MFC - the same with us, to verify the accessibility and portability of the biology components in our system. Their experiments showed that our biology components are stable and repeatable enough to be used in different labs.</P><P>Here, we present their results as follows: </P><BLOCKQUOTE><P>we also helped them characterize the Cu<SUP>2+</SUP> sensing part in our MFC device.</P><P>As for BBa_K1755301, the voltage in MFC was documented. The MFC‘s anode was added with Cu<SUP>2+</SUP> from the very begging (0 min), and their voltage (with BL21-K1755301) was also documented.  </P><DIV class="kuang" style="width:470px"><DIV id="Enlarge"><P><B>Figure 5.</B><SPAN style="font-size: 14px"> The voltage of MFC with K1755301 (Room temp 25 ℃)</SPAN></P></DIV></DIV><P>The results above show a significant difference of the voltage output at around 120 min. After 240 min, the voltage output became closer. So, the Cu<SUP>2+</SUP> sensing part proves to be functional.</P><P>We can see that the <SPAN style="font-style: italic">E.coli</SPAN>, modified by the part they constructed can sense the concentration of Cu ions in different labs normally.  The data shows typically consistent.This also indicates the biological component of LZU-China’s “Micro saHolmes” measurement system has nice accessibility and portability.(Table-1)</P><DIV class="kuang" style="width:670px"><DIV id="Enlarge"><P><B>Table-1.</B><SPAN style="font-size: 14px"> The voltage of MFC when the concentration of Cu ions was 2mg/L. The data are all the peak values.</SPAN></P></DIV></DIV></BLOCKQUOTE><P>You can also get these data from <A href="https://2015.igem.org/Team:TJU/Collaborations#LZU-China"><SPAN class="blue-text text-darken-2"> their wiki pages.</SPAN></A></P><P>Moreover, we invited Shenzhen University to evaluate the usability of non- biology components in our system. After received our components, following our protocol, they assembled the device independently and operated the &quot;Micro Holmes System&quot; successfully.</P><P>Here, we present their results as follows:</P><BLOCKQUOTE><P>LZU-China also sent us the major components of their “Micro Holmes” device (including the pcDuino board, the voltage magnification circuit board, the analogue-to-digital Conversion chip, and the touch screen). So we can help them verify the accessibility and the portability of their device. According to their specification, we successfully connected these electronic components, start the device and tested the software pre-installed in their pcDuino board. Although most functions of their device can work normally, there were also some small bugs. For example, in the “initialization” UI, they put the “concentration” title in front of the “Output Voltage”. We reported these bugs to them, and they correct them. </P></BLOCKQUOTE><P>You can also get these data from <A href="https://2015.igem.org/Team:SZU_China/LZU-China"><SPAN class="blue-text text-darken-2">their wiki pages.</SPAN></A></P></DIV></DIV><DIV id="future" class="section scrollspy row"><DIV id="popout" class="col s12"><H4 class="left-align">Future</H4><H6><P>
          In this year’s competition, we constructed a series of sensors to measure the pollutant in water. We have added the E.coli-based Cu ions sensor to the MFC and it was proved that the device can sense the Cu ions concentration. Although we have completed the primary goal, we still have much work to do.
        </P><P>
          Our device can only detect the concentration of the pollutant while it cannot detect which pollution it is. We will couple the device with a system to monitor which pollutant it is before measuring the concentration. Different chromoproteins can be used in the system to distinguish different kinds of pollution.
        </P><P>Besides, we will try to develop the <EM>ribB</EM> report gene. We hope that the riboflavin-MFC system can be a new measurement standard system for quantitive measuring.</P></H6></DIV></DIV><DIV id="interlab" class="section scrollspy row"><DIV id="popout" class="col s12"><H4 class="left-align">Interlab</H4><H5 class="left-align">Overview</H5><P>As our team is participating in the Measurement track, we are asked to work in the Interlab Study.
          <A href=" https://2015.igem.org/Tracks/Measurement/Interlab_study"> Interlab Study</A>
          In this year, we are asked to build 3 devices. The devices are J23101 + I13504, J23106 + I13504 and J23117 + I13504.
          </P><P>In our study, we constructed the 3 devices in E.coli DH5 alpha and the devices were all constructed in the pSB1C3 plasmid. We used the flow cytometer (LSRFortessa, BD) as the equipment and the data was processed by TASEB tools.
          <A href=" https://synbiotools.bbn.com/?q=node/5"> TASEB tools</A> 
          In the Collections Page of Anderson Promoters, 
          <A href="http://parts.igem.org/Promoters/Catalog/Anderson "> Collections Page of Anderson Promoters</A>
          we can get the strength of the three promoters(J23101, J23106 and J23117). We had known that J23101 is the strongest and J23117 is the weakest. Our results proved this information.
          </P><H5 class="left-align">Experimental Design</H5><P>We used the BBa_I20270 as the positive control. The negative control is empty plasmid pET-28a. We constructed the three required devices (J23101 + I13504, J23106 + I13504 and J23117 + I13504) by BioBricks Assembly. We measured the devices using the flow cytometer. The data was processed by FlowJo 10</P><H5 class="left-align">Methods</H5><P>Device Construct</P><P>The parts were presented kindly by the team TJU. All the plasmids were transformed into E.coli DH5 alpha competent cells (made by the Ca ions method). We extracted the plasmid by the OMEGA Plasmid Mini Prep Kit (D6943-02). BBa_I13504 was cut by XbaI and PstI. J23101, J23106 and J23117 were cut by SpeI and PstI. The I13504 was inserted into the 3 vectors above by T4 Ligase. Finally the devices we constructed were transformed into E.coli DH5 alpha. All enzymes we used in the study is from TAKARATM. The restriction maps is below.</P><P>Cultivation</P><P>The cultivation of our bacteria was performed in test tubes (16cm long, diameter is 3cm) filled with 5ml LB medium. The incubator was kept at 37°C and 260 rpm shaking frequency. Chloramphenicol was added to each media. Chloramphenicol was added from a 1000X stock stored at -20°C for a final concentration of 25 µg/ml.</P><P>Measurement</P><P>Cells were washed by PBS twice in order to clean the medium. Measurement was processed by LSRFortessa, the channel was 488nm/510nm. The data was exported by FCSDiva as .fcs files. If one cell's FITC value is higher than 103, we believe that the cell is colored. We calculated the ratio of colored cells in total cells. The ratio*100 was set as the unit of the device's strength.</P><DIV class="col s12"><DIV class="row"><TABLE class="responsive-table striped col s12 m 8 l8"><TBODY><TR><TD>Cells</TD><TD>DH5 alpha</TD></TR><TR><TD>Incubator</TD><TD>Crystal<SUP>TM</SUP> IncuShaker</TD></TR><TR><TD>Temp</TD><TD>37℃</TD></TR><TR><TD>Flow cytometer</TD><TD>BD<SUP>TM </SUP>LSRFortessa</TD></TR><TR><TD>Excited wave</TD><TD>488nm</TD></TR><TR><TD>Filtered wave</TD><TD>510nm</TD></TR><TR><TD>Software for analyzing</TD><TD>FlowJo V10.0.6</TD></TR></TBODY></TABLE></DIV></DIV><H5 class="left-align">Results</H5><P>The restriction maps of the 3 devices we constructed is below.</P><P>As we can see by Figure.1, the strength of three devices is device1 &gt; device2 &gt; device3. The data we got clarifies the data from Collection Anderson Promoters.</P><DIV class="col s12"><DIV class="row"><TABLE class="responsive-table striped col s12 m 8 l8"><TBODY><TR><TD>J23101</TD><TD>0.70</TD></TR><TR><TD>J23106</TD><TD>0.47</TD></TR><TR><TD>J23117</TD><TD>0.06</TD></TR></TBODY></TABLE></DIV></DIV><P>         Figure.1 The devices’ strength measured by this Interlab Study Discussion
          </P><P>We can see that the values are not the same as Anderson’s Promoters. While we got the correct tendency. The value of the device-3 is lower than the NC’s, we think this may be caused by the fluorescence pollution.</P></DIV></DIV><DIV id="future" class="section scrollspy row"><DIV id="popout" class="col s12"><H4 class="left-align">Future</H4><H6><P>
          In this year’s competition, we constructed a series of sensors to measure the pollutant in water. We have added the E.coli-based Cu ions sensor to the MFC and it was proved that the device can sense the Cu ions concentration. Although we have completed the primary goal, we still have much work to do.
        </P><P>
          Our device can only detect the concentration of the pollutant while it cannot detect which pollution it is. We will couple the device with a system to monitor which pollutant it is before measuring the concentration. Different chromoproteins can be used in the system to distinguish different kinds of pollution.
        </P><P>Besides, we will try to develop the <EM>ribB</EM> report gene. We hope that the riboflavin-MFC system can be a new measurement standard system for quantitive measuring.</P></H6></DIV></DIV><DIV id="test2" class="col s12"><DIV class="col s12"><DIV class="col s1"><P> </P></DIV></DIV></DIV></DIV></DIV><DIV class="col hide-on-small-only m3 l3"><DIV class="pinnedx inorder"><UL class="sidelist section table-of-contents"><LI class="sidelist"><H5><A href="#background">Background</A></H5></LI><LI class="sidelist"><A href="#Water pollution">Water pollution</A></LI><LI class="sidelist"><A href="#Heavy metal detecting">Heavy metal detecting</A></LI><LI class="sidelist"><A href="#MFC"> MFC</A></LI><LI class="sidelist"><A href="#Riboflavin"> Riboflavin</A></LI></UL><LI class="sidelist"><H5><A href="#wetlab">Wet Lab</A></H5></LI><LI class="sidelist"><A href="#Introduction">Introduction</A></LI><LI class="sidelist"><A href="#An efficient riboflavin synthesis gene:&lt;em&gt;ribB&lt;/em&gt;">An efficient riboflavin synthesis gene:<EM>ribB</EM></A></LI><LI class="sidelist"><A href="#Pollutant sensor">Pollutant sensor</A></LI><LI class="sidelist"><A href="#Our MFC">Our MFC</A></LI><LI class="sidelist"><H5><A href="#drylab">Dry Lab</A></H5></LI><LI class="sidelist"><A href="#Overview">Overview</A></LI><LI class="sidelist"><A href="#Quick Guide">Quick Guide</A></LI><LI class="sidelist"><A href="#Technical Details (Instruction of mv Analysis)">Technical Details (Instruction of mv Analysis)</A></LI><UL class="sidelist"><LI class="sidelist"><H5><A href="#parts">Advantages </A></H5></LI></UL><UL class="sidelist"><LI class="sidelist"><H5><A href="#Accessibility and portability">Accessibility and portability </A></H5></LI></UL><UL class="sidelist"><LI class="sidelist"><H5><A href="#interlab">Interlab</A></H5></LI><LI class="sidelist"><H5><A href="#future">Future</A></H5></LI></UL></DIV></DIV></DIV></DIV></DIV><FOOTER class="page-footer teal"><DIV class="container"><DIV class="row"><DIV class="col l6 s12"><H5 class="white-text">LZU-China 2015</H5><P class="grey-text text-lighten-4">NO.222, TianShui South Road, Lanzhou, China, 730000</P><H6 class="white-text">Copyright 2015 Lanzhou University</H6></DIV><DIV class="col l3 s12"><H5 class="white-text">Acknowledgments</H5><UL><LI><A class="white-text" href="http://en.lzu.edu.cn/">Educational Administration Office, LZU</A></LI><LI><A class="white-text" href="http://chc.lzu.edu.cn/en/">Cuiying Honors College, LZU</A></LI><LI><A class="white-text" href="http://lifesc.lzu.edu.cn/">School of Life Sciences, LZU</A></LI></UL></DIV></DIV></DIV><DIV class="footer-copyright"><DIV class="container">
      Code by <A class="brown-text text-lighten-3" href="http://materializecss.com">Tigeraus</A></DIV></DIV></FOOTER></DIV></DIV></DIV></BODY></HTML>