<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SSTi-SZGD_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SSTi-SZGD/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><HEADER class="container-fluid page_header fadeIn animated"><DIV class="container transition"><NAV class="page_nav col-xs-12 collapse navbar-collapse" id="example-navbar-collapse"><UL class="nav navbar-nav"><LI class="active"><A href="https://2018.igem.org/Team:SSTi-SZGD">Home</A></LI><LI><A href="https://2018.igem.org/Team:SSTi-SZGD/Description">Project</A></LI><LI><A href="https://2018.igem.org/Team:SSTi-SZGD/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:SSTi-SZGD/Applied_Design">Application</A></LI><LI><A href="https://2018.igem.org/Team:SSTi-SZGD/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:SSTi-SZGD/Team">Acknowledgements</A></LI></UL></NAV><NAV class="sub_nav col-xs-12"><UL class="container"><LI class="col-xs-12 Project"><A href="https://2018.igem.org/Team:SSTi-SZGD/Description">Description</A><A href="https://2018.igem.org/Team:SSTi-SZGD/Design">Design</A><A href="https://2018.igem.org/Team:SSTi-SZGD/Results">Results</A><A href="https://2018.igem.org/Team:SSTi-SZGD/InterLab">InterLab</A><A href="https://2018.igem.org/Team:SSTi-SZGD/Protocols">Protocols</A><A href="https://2018.igem.org/Team:SSTi-SZGD/Safety">Safety</A><A href="https://2018.igem.org/Team:SSTi-SZGD/Notebook">Notebook</A></LI><LI class="col-xs-12 Parts"><A href="https://2018.igem.org/Team:SSTi-SZGD/Parts#All_parts">All Parts</A><A href="https://2018.igem.org/Team:SSTi-SZGD/Parts#Basic_part">Basic Part</A><A href="https://2018.igem.org/Team:SSTi-SZGD/Parts#Composite_part">Composite Part</A><A href="https://2018.igem.org/Team:SSTi-SZGD/Parts#Improved_part">Improved Biobrick</A></LI><LI class="col-xs-12 Application"><A href="https://2018.igem.org/Team:SSTi-SZGD/Applied_Design">Applied Design</A><A href="https://2018.igem.org/Team:SSTi-SZGD/Entrepreneurship">Entrepreneurship</A><A href="https://2018.igem.org/Team:SSTi-SZGD/Demonstrate">Demonstrate</A></LI><LI class="col-xs-12 Human_Practices"><A href="https://2018.igem.org/Team:SSTi-SZGD/Human_Practices">Integrated Human Practices</A><A href="https://2018.igem.org/Team:SSTi-SZGD/Public_Engagement">Public Engagement &amp; Out Reach</A><A href="https://2018.igem.org/Team:SSTi-SZGD/Awards">Awards</A></LI><LI class="col-xs-12 Acknowledgements"><A href="https://2018.igem.org/Team:SSTi-SZGD/Team">Team</A><A href="https://2018.igem.org/Team:SSTi-SZGD/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:SSTi-SZGD/Attributions">Attributions</A></LI></UL></NAV></DIV></HEADER><DIV class="page_content"><DIV class="container Content"><SECTION class="col-xs-12 col-sm-10"><P class="Ctitle"><SPAN>Overview ( Background Introduction )</SPAN></P><DIV class="Ccontent"><P>
							The aim of the project this year is to construct recombinant <I>B.subtilis</I>
							strains that are able to secrete large amount of hyaluronic acid (HA) of both high and low molecular weights. While HA of different molecular mass are already of great commercial values<SUP>[1]</SUP>
							, we also proposed to use the low molecular weight HA in the development of a novel form of cosmetic product – HA micro-needles. In designing our project we adhered to three over-arching engineering principles: 1) HA biosynthesis is achieved via a safer, cost effective, and highly productive approach compare with the traditional methods. To achieve this goal, we constructed a recombinant<I>B.subtilits 168</I>
							 strain that heterogeneously expresses HA synthase for the biosynthesis of HA, given the fact that two HA precursors, GlcUA and GlcNAc, are synthesized indigenously by B. subtilis. In addition, a number of native genes, gtaB, tuaD, glmU, glmS, and glmM, whose gene products work together to regulate the biosynthesis of GlcUA and GlcNAc. Overexpression of these native genes may help improving the accumulation of HA; 2) Low molecular weight HA is biosynthesized by hydrolyzing high-molecular-weight HA in cell cultural environment. A leech hyaluronidase gene (LHayal) encodes LHAase is heterogeneously expressed in the recombinant<I>B.subtilits 168</I>
							 strain constructed in step 1. LHAase belongs to the hyaluronate 3-glycanohydrolase subgroup and has high substrate specificity and hydrolysis efficiency; 3) To ensure that our HA product has better absorbing efficiency and can be easily applied, we designed a novel HA product—HA micro-needles patch--by using colvantly cross-linked HA hydrogel and HA molecule mixture. Altogether, we divided our project into three modules to achieve these goals collectively.</P></DIV></SECTION><SECTION class="col-xs-12 col-sm-10"><P class="title">Module 1 - HA Production</P><DIV class="content"><DIV class="Csubtitle col-xs-12 col-sm-6"><SPAN class="active">1. HA Introduction and Plasmid construction</SPAN></DIV><DIV class="Csubtitle col-xs-12 col-sm-6"><SPAN>2. HA Concentration Elevation</SPAN></DIV><DIV class="Csubtitle col-xs-12 col-sm-6"><SPAN>3. Transformation</SPAN></DIV><DIV class="Csubtitle col-xs-12 col-sm-6"><SPAN>4. Fed-batch Fermentation and Purification of HA </SPAN></DIV><DIV class="Csubtitle col-xs-12 col-sm-6"><SPAN>5. HA Characterization</SPAN></DIV><DIV class="info col-xs-12"><DIV class="One"><P class="subtitle"> HA Introduction</P>
									Hyaluronic acid (HA) is a mucopolysaccharide composed of disaccharides unit of N-acetyl glucosamine and glucuronic acid polymerization<SUP>[1]</SUP>
									. One of the mainstream industrial HA productions is through <I>streptococcus.zooepidemicus</I>
									  fermentation, which currently facing two challenges, 1) the high risk of pathogenicity and the fewer DNA manipulation techniques available restricted the use of <I>streptococcus.zooepidemicus</I>
									  species, 2) viscoelastic property of HA significantly reduces dissolved oxygen in fermentation, which is the primary obstacle in maintaining normal cell metabolism and improving HA biosynthesis.<P>
									Therefore, in order to solve the first issue, our project devoted to construct a food-grade safe (GRAS) strain,<I>B.subtilis</I>
									, for HA biosynthesis, and to increase production level of HA by regulating some of the precursor genes in the upstream synthetic pathways; for the second issue, we tried to directly produce HA at lower molecular weight with reduced viscosity.</P><P class="subtitle">(1) HA bio-synthesis pathway</P><P>
									Studies of HA biosynthesis in prokaryotes have shown that an indigenous pathway for biosynthesis of HA precursors exists in<I>B.subtilis</I>
									 genome, starting from sucrose as a carbon source, two parallel metabolic branches form which eventually, through multiple sugar intermediates, culminate in the synthesis of the two nucleotide sugar substrates, UDP-GlcUA and UDP-GlcNAc<SUP>[2]</SUP></P><P>
									In the first set of reactions, α-phosphoglucomutase (pgcA) converts glucose-6-phosphate to glucose-1-phosphate before a phosphate group from UTP is transferred to glucose-1-phosphate by UDP-glucose pyrophosphorylase (gtaB) to produce UDP-glucose. Later, UDP-glucose is oxidised by UDP-glucose dehydrogenase (tuaD) to yield the first HA precursor, UDP-glucuronic acid (UDP-GlcUA). In the second set, glucose-6-phosphate is converted to fructose-6-phosphate by phosphoglucoisomerase (pgi). Once converted, fructose-6-phosphate is tagged with an amino group transferred from a glutamine residue via amidotransferase (glmS) to produce glucosamine-6-phosphate and modified by mutase (glmM) to yield glucosamine-1-phosphate. This intermediate is then sequentially acetylated and phosphorylated by acetyltransferase and pyrophosphorylase(glmU), respectively to yield the second HA precursor, UDP-N-acetylglucosamine (UDP-GlcNAc). Once the two precursors are synthesized, hyaluronic acid synthase (hasA) polymerises the two components in an alternate manner to produce HA polymer<SUP>[2]</SUP>
									. It is important to note that HA biosynthesis is an energy-consuming process for the bacteria as several intermediates are also used in cell wall biosynthesis, biomass formation and lactate formation via glycolysis<SUP>[3]</SUP>
									. <I>Streptococcus.zooepidemicus</I>
									 has all the require pathways and genes for HA synthesis,however,<I>B.subtilis</I>
									, however, lacks the crucial hasA gene that is responsible for the final polymerization step in HA synthesis<SUP>[4]</SUP>
									.</P><DIV class="Cimg col-xs-12 col-sm-6"><P>
										Fig 1: Hyaluronic acid bio-synthesis pathway in <I>S.zooepidemicus</I>
										.</P></DIV><P class="subtitle">(2)Construction of pAX01-HasA plasmid</P><P>
									To construct a HA biosynthetic pathway in <I>B. subtilis 168</I>
									, the missing hyaluronic acid synthase encoding gene hasA was amplified from <I>S. zooepidemicus</I>
									. Heterogenous expression of HasA gene was achieved by an integration vector pAX01 to avoid plasmid loss. The constructed HasA expression cassette, under the control of an inducible promoter PxylA, was integrated at the lacA locus of the <I>B. subtilis 168</I>
									 genome.</P><P class="subtitle3">① pAX01 backbone</P><P>
									pAX01 backbone contains two antibiotic resistant genes for different chassis selections (Erm and AmpR), homologous arm lacA which is used for integration of HasA construct into <I>B. subtilis 168</I>
									 genome, a xylose-induced promoter PxylA originated from <I>B.megaterium</I>
									 , and a repressor xylR regulatory cassette. Promoter PxylA is located within xylose operon, originally to drive the expression of xylA (xylose isomerase coding gene) and xylB (xylulose kinase). xylR with its promoter located at upstream of xylose operon. It encodes xyl repressor which binds to xyl operator in the absence of xylose, repressing transcription activation.</P><P class="subtitle3">② HasA construct</P><P>
									HasA construct contains a hasA coding sequence in combination with a strong ribosome binding site (RBS). HasA coding sequence is isolated from <I>S.zooepidemicus</I>
									 encodes hyaluronic acid (HA) synthase, which is 419 amino acid long and forms part of the HA synthesis operon in <I>S.zooepidemicus</I>
									 . HasA gene was commercially synthesized and sub-cloned into pAX01 integration vector at the SacII and BamHI restriction sites.</P><DIV class="img col-xs-12"><P>
										Fig 2：Illustration of HasA construct in pAX01 plasmid</P></DIV></DIV><DIV class="Two"><P class="subtitle">Design of pP43NMK-gtaB-tuaD and pP43NMK-glmU plasmids</P><P>
									TuaD-gtaB construct was commercially synthesized by GBI Genome Service, Shenzhen, and was sub-cloned into shuttle vector pP43NMK at the KpnI and HindIII restriction sites. The expression of operon tuaD-gtaB was under the control of a constitutive promoter P43. </P><P class="subtitle3">① pP43NMK backbone</P><P>
									p43NMK backbone contains 1) P43 constitutive promoter and an B. subtilis RBS, 2) two antibiotic resistant genes for different chassis selections (Km and AmpR), 3) an E. coli replication origin (ori), 4) M13 fwd and rev universal sequences for sequencing purpose. </P><P class="subtitle3">② TuaD-gtaB construct</P><P>
									TuaD-gtaB construct contains two gene coding sequences in an operon. TuaD is one of the native genes of B. subtilis, encoding UDP-glucose 6-dehydrogenase and is 461 amino acid in length (also known as: UDP-GlcDH in megaterium); gtaB encodes UTP--glucose-1-phosphate uridylyltransferase in<I>Bacillus megaterium</I>
									.</P><DIV class="img col-xs-12"><P>
										Fig 3：Illustration of tuaD-gtaB construct in pP43NMK plasmid</P></DIV><P class="subtitle3">③ glmU construct</P><P>
									glmU construct contains a coding sequence of glmU gene, which is isolated from Mycobacterium encodes UDP-N-acetylglucosamine pyrophosphorylase. GlmU gene was commercially synthesized and sub-cloned into pP43NMK vector at the BamHI and HindIII restriction sites. The constructed glmU expression cassette is under the control of the constitutive promoter P43.</P><DIV class="img col-xs-12"><P>
										Fig 4：Illustration of glmU construct in pP43NMK plasmid</P></DIV></DIV><DIV class="Three"><P class="subtitle">Transformation</P>
									Recombinant vector pAX01-HasA was transformed into the wild-type strain <I>B.subtilis 168</I>
									 (LB+ 0.5μg/ml erythromycin) to obtain a recombinant <I>B. subtilis strain 168E</I>
									  using chemical transformation method. pP43NMK-tuaD-gtaB and pP43NMK-glmU vectors were then transformed into the recombinant strain <I>B.subtilis 168E</I>
									   (LB+50μg/ml kanamycin), respectively, to study whether overexpression of these precursor genes could elevate HA production. Cultures were grown overnight at 37℃., at shaking. Colony PCR and restriction enzyme digestion of the miniprepped plasmids confirmed that these genes were successfully transformed into <I>B.subtilis</I>
									. <P>
									The primers of colony PCR which is used for verification:<EM>HasA-F:</EM> GGTCCATAGGGCTACAAAAG
									<EM>HasA-R:</EM> ACCCTGATGCTTTAGAGGAG
									<EM>GlmU-F:</EM> GGCTGGACAAGGAACGAGAA
									<EM>GlmU-R:</EM> CGTCTCTGCTGTCAAAAGCG
									<EM>GtaB-TuaD-F:</EM> CACAGTAGCGGGTACTGGTTA
									<EM>GtaB-TuaD-R:</EM> TTGGATGCTACCTCAACTTGT
								</P></DIV><DIV class="four"><P class="subtitle">Fed-batch Fermentation of HA , HA Separation and Purification</P>
									After transformation was confirmed, single colony was inoculated into 50ml LB medium and grew overnight as a seeded culture. The next day re-inoculated into 50ml MM media (1%v/v) and grew at 37℃  at shaking 160rpm. Xylose (20g/L) was added 2h after inoculation to induce the expression of the hasA gene. Sucrose was exponentially fed at rates of 7.5, 7.5, 15.0, and 10.0g/h/L from 8 h to 12 h. A constant feed rate of 5g h/L was then maintained until the end of fermentation for 48h<SUP>[5]</SUP>
									. <P>
									The HA sample was released by the addition of 0.1% (w/v) sodium dodecyl sulfonate (SDS), culture supernatant was collected by centrifugation at 10,000g for 10min and filtered through a 0.45um micro-filter membrane<SUP>[6]</SUP>
									. HA was precipitated with two volumes of ethanol and incubated at 4°C for 1h. The precipitate was collected by centrifugation at 5000g for 10min. Then re-dissolved in an equal volume of distilled water, the steps were performed three times. Purified HA was then freeze-dried to become powder before subject to characterization. </P><DIV class="img col-xs-12"><P>
										Fig 5：Demonstration of a freeze-drying machine that freeze-dried HA molecules</P></DIV></DIV><DIV class="five"><P class="subtitle">HA Characterization</P><P class="subtitle">(1) CTAB(cetyltrimethylammonium bromide) method</P><P>
									CTAB is known as a common method for measuring the concentration of the HA. It is a cationic surface active agent which precipitates nucleic acids and acidic polysaccharides under the low ionic strength. At the same time, CTAB forms complexes with proteins and polysaccharides, acidic polysaccharides under the high ionic strength<SUP>[7]</SUP>
									. 
								</P><P>
									The nitrogen atoms of CTAB can be paired with the oxygen atoms of carboxyl groups in HA to form insoluble HA-CTAB complexes. Meanwhile, CTAB does not interact with the UDP-gluconic acid (UDP-Gluconate) and UDP-N-acetyl-D-glucosamine (UDP-GlcNAc) which is the monomers of the HA. This method is therefore specific for measuring HA concentration.
									 2.5g/L CTAB solution was prepared (containing 0.2mol/L NaOH and appropriate amount CTAB powder). 1ml re-dissolved HA powder or standard sample and 2ml CTAB solution was thoroughly mixed in colorimetric dish. The timing was started once the mixing was, and the absorbance was measured at 400nm wavelength at the 10min point. 1ml deionized water with CTAB solution was set as a blank. The standard curve was plot using a series dilution of the standards.</P><P class="subtitle">(2) Ubblelohde viscometer method</P><P>
									Ubblelohde viscometer method was used to get a rough idea of the molecular weight of HA, usually HMW-HA. Ubbelohde viscometer is an instrument which uses a capillary based method for measuring viscosity<SUP>[8]</SUP>
									. It has a reservoir on one side and a measuring bulb with a capillary on the other. Once liquid is introduced into the reservoir then sucked through the capillary and measuring bulb. Liquid then travels back through the measuring bulb and the time takes for the liquid to pass through two calibrated marks is recorded to calculate the viscosity according to the Hagen-Poiseuille law:</P><DIV class="img col-xs-12"><P>
										Fig 6: Illustration of an Ubblelohde viscometer and the formula used to viscosity calculation, in which η, ηsp, C, k, β, lnηr are solution viscosity, increased specific viscosity, concentration, huggins constant, kramer constant, and logarithmic viscosity of the capillary respectively. Volume flow measurement through the capillary at a given differential pressure is the fundamental measurement criteria for capillary viscometers. Expressed differently, the viscosity is determined by measuring the time required for a defined liquid volume to flow through a capillary tube determined by the hydrostatic pressure of the liquid. Two marks before and after a ball shaped extension enables measurement the time.freeze-dried HA molecules</P></DIV><P>
									The standard HA solution was prepared by dissolving 0.1g of HA in 100ml 0.2M NaCl solution, then diluted into different concentrations: 0.02g/100ml, 0.03g/100ml, 0.04g/100ml, and 0.05g/ml. After cleaning of the viscometer, solutions were injected to flow through the tubing and the device was inserted into a constant temperature bath. The solution was measured according to Ubblelohde viscometer instruction manual. This measurement was performed in duplicates and repeated three times under 25℃.</P></DIV></DIV></DIV></SECTION><SECTION class="col-xs-12 col-sm-10"><P class="title">Module 2 - LHAase Construction</P><DIV class="content"><DIV class="Csubtitle col-xs-12 col-sm-6"><SPAN class="active">1. Hyaluronidase Introduction</SPAN></DIV><DIV class="Csubtitle col-xs-12 col-sm-6"><SPAN>2. Plasmid Construction</SPAN></DIV><DIV class="Csubtitle col-xs-12 col-sm-6"><SPAN>3. Transformation and Expression</SPAN></DIV><DIV class="Csubtitle col-xs-12 col-sm-6"><SPAN>4. LHAase Characterization</SPAN></DIV><DIV class="Csubtitle col-xs-12 col-sm-6"><SPAN>5. LMW-HA Synthesis</SPAN></DIV><DIV class="info col-xs-12"><DIV class="One"><P class="subtitle">(1) LHAase introduction and Hydrolytic pathway of HA using LHAase</P><P>
									Hyaluronidase（HAase）is denoted to a large class of enzymes that predominantly degrade HA<SUP>[9]</SUP>
									. HAases widely exist in eukaryotes and prokaryotes, and are important physiological active substances participating in many physiological activities<SUP>[10]</SUP>
									. Based on substrate specificity and hydrolysis products, HAases are commonly grouped into three families: the first is hyaluronate lyases (EC 4.2.2.1, <I>streptococcus.zooepidemicus</I>
									  hyaluronate lyase), which is a common source of commercial HAase production but may contain endotoxins. It hydrolyzes HA on the β-1, 4 glycoside bond to generate 2-acetamido-2-deoxy-3-O-(β-D-gluco-4-enepyranosyluronic acid)-D -glucose.The second group is hyaluronate 4-glycanohydrolases (EC 3.2.1.35, Bovine testicular hyaluronidase, BTH). Commercial BTH has been widely used in clinical medicine, and its hydrolysis mechanism has been studied extensively. Besides being expensive in material source (bovine testes), BTH hydrolyzes HA by cutting the β-1, 4 glycoside bonds to produce mainly four sugar. Also it could hydrolyze chondroitin and has transglycosidation. The Third group is hyaluronate 3-glycanohydrolases (EC 3.2.1.36, Leech HAase).</P><P>
									Compared with BTH and <I>streptococcus.zooepidemicus</I>
									  HAases, leech HAase has high substrate specificity and can produce a narrow-spectrum of products. It degrades HA by crackingβ- 1, 3 - glycosidic bond, results in the reduction of side with glucuronic acid sugar fragments. The end products are mainly four and six sugars. In addition, the use of recombinant leech HAase does not pose any risk of animal cross-infection and it has no transglycosidation action<SUP>[11]</SUP>
									. Therefore, high-level production of recombinant leech HAase would be of great significance for both clinical medical treatments (such as surgery, ophthalmology and internal medicine), and producing narrow-spectrum HA oligosaccharides at the industrial scale.</P><DIV class="Cimg col-xs-12 col-sm-6"><P>
										Fig 7: Illustration of Hydrolytic pathway of HA by LHAase</P></DIV><P class="subtitle">(2) LHAase introduction</P><P>
									We cloned the first leech HAase-encoding gene, LHyal, into the recombinant HA-producing <I>B.subtilis strain 168E</I>
									 in order to achieve a cell factory synthesizing low molecular weight HA. LHyal gene (Genebank NoKJ026763) encodes LHAase (Mw=58kD), belongs to the hyaluronate 3-glycanohydrolases family. HA was hydrolyzed to oligosaccharide by hydrolyzingβ-1.3 glucosidic bonds by LHAase in a non-processive endolytic mode<SUP>[12]</SUP>
									. LHyal has superior substrate specificity and no transglycosidase activity compare with other two group of hyaluronidase. In particular, leech HAase is unable to degrade chondroitin or chondroitin sulfate. Although mammalian HAase has been widely used as a drug diffusion agent, such HAase activity is susceptible to heparin inhibition. In comparison, leech HAase activity is not affected by heparin, therefore it possesses more medical value in clinic and other medical aspects<SUP>[13]</SUP>
									.</P></DIV><DIV class="Two"><P class="subtitle">LHAase Plasmid Construction</P><P>
									In order to identify the best gene expression strategy for LHAase, two expression vectors were selected: integration vector pDG1730 and shuttle vector pMA0911. LHyal expression construct was commercially synthesized by BGI Genome Services and sub-cloned into vectors pDG1730 or pMA0911 respectively.</P><P class="subtitle3">① Design of LHAase expression vector</P><P>
									p43NMK backbone contains 1) P43 constitutive promoter and an <I>B. subtilis</I>
									 RBS, 2) two antibiotic resistant genes for different chassis selections (Km and AmpR), 3) an <I>E. coli</I>
									 replication origin (ori), 4) M13 fwd and rev universal sequences for sequencing purpose. </P><P>
									The LHyal expression
									construct contains 1) a <I>B.subtilis</I>
									 constitutive promoter PlepA, 2) 6xHis-tag, 3), a strong ribosome binding site (RBS) which is a shine-Dalgarno sequence from gsiB gene, 4) a signal peptide of AmyX that is derived from a-Amylase gene from 
									<I>B.amyloliquefaciens</I>
								 , and 5) LHyal coding sequence Leech.</P><DIV class="img col-xs-12"><P>
										Fig 8: Illustration of LHyal construct in pDG1730 plasmid</P></DIV><P class="subtitle">Promoter： </P>
									PlepA is a strong constitutive promoter which is derived from the bicistronic operon. One of the expressed proteins in the operon is protein LepA. This protein plays an important role during the translation as it can move the mRNA-tRNA complex one step back in the ribosome which is expected to improve the fidelity of translation.<P class="subtitle">6 His-tag： </P>
									Currently His-tags have been extensively applied for recombinant protein expression. Based on our literature search, 6His-tag seems to provide enough spacing for protein folding to prevent crystallization or misfolding of peptide chain, and it is a commonly used in protein purification by chromatography.<P class="subtitle">RBS： </P>
									The strong ribosome binding site (RBS) is a shine-Dalgarno sequence from gsiB gene. It can lead to a pronounced stimulation of expression when placed upstream of a variety of genes, and significant increase in the translation of the genes is observed.<P class="subtitle">Signal peptide: </P><I>Bacillus subtilis</I>
									 is a well-known chassis with highly active protein secretion system, There are mainly two protein export pathways availibale in B.subtilis: Sec-dependent translocation pathway and twin-arginine translocation (Tat) pathway<SUP>[14]</SUP>
									. Compare with Sec pathway, tat pathway has an intrinsic advantage of being able to transport folded proteins across the cytoplasmic membrane and without the requirement ATP hydrolysis, this helps retaining protein function extracellularly<SUP>[15]</SUP>
									. Tat pathway has a signature twin-arginine (RR) motif located at the border of the N-terminal domain and the hydrophobic region. AmyX signal peptide belongings to tat pathway. In our project AmyX was fused in N-terminal of the LHyal gene.<P class="subtitle">pDG1730 plasmid backbone： </P>
									contains Bsu-amyE homologous arm sequences that help LHyal construct to be integrated into the amyE locus of <I>B.subtilis</I>
									 genome by double-crossover integration. There is also an ori sequence for replication in <I>E. Coli</I>
									 , three antibiotic resistant genes for <I>E.coli</I>
									  and <I>B. subtilis</I>
									  selections (Amp, Erm and Spc), and a T1 terminator sequence. the whole construct was sub-cloned into the backbone at the BamHI and HindIII restriction sites.<P class="subtitle">pMA0911 plasmid backbone： </P>
									consists of two antibiotic resistant genes which are used for different chassis selections (Kane, Amp) and an <I>E. coli</I>
									 ori sequence. The construct was sub-cloned into the backbone at the NdeI and BamHI restriction sites.</DIV><DIV class="Three"><P class="subtitle">Transformation and Expression</P><P class="subtitle3">① Transformation</P>
									pDG1730-LHyal construct vector was transformed into 
									B.subtilits 168
									 by chemical transformation method and grew on LB agar+antibiotic at 37℃overnight. Positive selection of integration was performed with spectinomycin at 100μg/ml and negative selection of single crossover integration events with erythromycin at 0.5μg/ml. Colony PCR for verifying part integration were realized using Taq polymerase PCR system (Takara).<P>
									The primers for colony PCR polymerization: <EM>LHAase-F:</EM> ATGAAAGAGATCGCGGTGA
									<EM>LHAase-R:</EM> TTATTTTTTGCAGGCTTC
								</P><P class="subtitle3">② Fed-batch fermentation</P>
									Once colonies contain recombinant bacteria were identified, single colony was streaked out into a starter 1ml LB culture (containing 50ug/ml erythromycin) and grew at 30℃, 220rpm, overnight culture was inoculated into 50ml LB medium+ 50ug/ml erythromycin (1% v/v) and grew for 12 h at shaking before fresh 50ml LB medium+ 50ug/ml erythromycin was added. Every 12 h, fresh 50ml LB with antibiotic was fed until 48 h later a 200ml culture was achieved. Cell density was measured every 12 h. After the final feed, when OD600 reached ~2.5, the bacterial cells were removed by centrifugation and filtered through a 0.45um micro-membrane. Culture supernatant was freeze-dried to become powder before further purification.<P class="subtitle3">③ LHAase purification</P>
									LHAase was isolated and purified from freeze-dried supernatant powder by Ni-NTA spin columns (Biorad Cat.no. 31314). Briefly, dried powder was dissolved into deionized water and added 5ul RNase then the 630ul enzyme liquid was added to Ni-NTA column [QIAGEN Co Ltd.Cat NO31314] for 5min, the non-target proteins were eluted by using 600ul NPI-20 (50mM NaH2PO4, 300mMNaCl, 20mM imidazole) and was centrifugated 2 times, supernatant was collected by 890g centrifugation. Target protein was then eluted by using 300ul NPI-500 (50mM NaH2PO4, 300mMNaCl, 500mM imidazole) and supernatant collected by centrifugation. Purified enzyme was dialyzed (Regenerated cellulose dialysis membrane MWCO 8000-14000) against 50mM phosphate buffer (pH:6-7.4) overnight before HA was freeze-dried. Purified HA powder was analyzed by SDS-Polyacrylamide gel electrophoresis (PAGE) and enzymatic analyses.<P class="subtitle3">④ SDS-PAGE</P>
									Appropriate freeze-dried LHAase powder was mixed with 6xloading buffer (30mM EDTA,36%(v/v) glycerol, 0.05%(w/v), xylene cyanol FF, and 0.05% (w/v) Bromophenol blue). Sample was denatured by boiling for 10 min, and then loaded into the gel. Electrophoresis was performed at 150V, 80mA for 1 h. Gel was stained and destained before photography.</DIV><DIV class="four"><P class="subtitle">Enzymatic Characterization of LHAase</P><P class="subtitle3">① Agarose Plate Assay by using HA as a substrate</P>
									LHAase activity was confirmed using the simple plate assay<SUP>[16]</SUP>
									. The assay plate was prepared with 1mg/ml HA, 0.75g agarose, 50mM sodium citrate (pH5.5) buffer. 150ul sample, control or standard were injected into the cylindrical holes (3mm in diameter) on the agarose plates, incubated at 37℃ for 10h and covered with 10%(w/v) cetylpyridinium chloride. The formation of a distinct clear halo around the hole indicates HAase activity. Standard enzyme solution (hyaluronidase extracted from bovine testis) was set as positive control, heat-inactivated enzyme solution was set as negative control. <P class="subtitle3">② Enzyme Linked Immunosorbent Assay (ELISA)</P>
									ELISA is a very sensitive immunochemical technique used to assess the presence of specific protein (antigen or antibody) in the given sample and its quantity. An enzyme conjugated with an antibody reacts with a colorless substrate to generate a colored reaction product. LHAase activity was studied using a microorganism HA ELISA kit (Tongwei Ltd, Shanghai, Cat.no tw045410). Firstly, 40μl standard and 10ul sample were mixed and loaded into a 96 well microplate. No sample is added to the blank well. 100μl enzyme-conjugate was added to standard wells and sample wells except the blank well, then cover with an adhesive strip and incubate for 60min at 37℃. The microplate was washed 4 times using 20x eluent. Substrate A (50μl) and Substrate B (50μl) were added to each well. Gently mix and incubate for 15m at 37℃, stop solution was added to each well to induce color changes. The Optical Density (O.D.) is read at 450 nm using a microtiter plate reader within 15min. This assay was performed in duplicates and repeated three times.<P class="subtitle3">③ DNS reducing sugar method</P>
									LHAase activity analysis was further examined by measuring the amount of reducing sugar liberated from HA with the 3,5-dinitrosalicylic acid (DNS) colorimetric spectrophotometric<SUP>[17]</SUP>
									. One unit of enzymatic activity was tentatively defined as equal to the reducing power of glucuronic acid (glucose equivalents in micrograms) liberated per hour from HA at 38℃ and pH5.5. Specific activity was defined as units of enzyme per mg of protein.<P>
									In our experiment, the enzymatic reaction contained 1.6 mg/ml of HA as a substrate and an appropriate amount of LHAase in 50 mM pH 5.5 citrate buffer in a total volume of 1 ml, and was incubated at 38℃ for 10 min. The reaction was stopped by immersion in boiling water for 5 min and before adding DNS solution and further boiled for 15min to induce color changes. Standard enzyme solution (hyaluronidase extracted from bovine testis) was set as positive control, heat-inactivated enzyme solution was set as negative control.</P></DIV><DIV class="five"><P class="subtitle">(1)LMW-HA Synthesis</P>
									Currently, low molecular weight HA has attracted considerable attention because of its potential applications attributed to its unique biological properties, including the stimulation of fibroblast proliferation, collagen synthesis, and potential effects in eliminating certain cancer cells<SUP>[18]</SUP>
									. Compare with HMW-HAs, LMW-HAs can be readily absorbed by human body and contribute to the biosynthesis of HMW-HA<SUP>[19]</SUP>
									. With its important values in medical and cosmetics, it is beneficial to directly obtain low-molecular weight HA from microbial fermentation via LHAase hydrolysis.<P class="subtitle3">① Transformation</P>
									pDG1730-LHAyal and pMA0911-LHAyal vectors were transformed into the recombinant strain <I>B.subtilis 168E</I>
									, respectively, and selected by antibiotic kanamycin to study the direct biosynthesis of LMW-HA in <I>B.subtilis 168E-LHAase</I>
									. Colony PCR and restriction enzyme digestion confirmed that these vectors was successfully transformed into <I>B.subtilis 168E-LHAase</I>
									.
								<P>The primers of colony PCR which is used for verification: </P><EM>HasA-F:</EM> GGTCCATAGGGCTACAAAAG
									<EM>HasA-R:</EM> ACCCTGATGCTTTAGAGGAG
									<EM>LHAase-F:</EM> ATGAAAGAGATCGCGGTGA
									<EM>LHAase-F:</EM> ATGAAAGAGATCGCGGTGA
								<P class="subtitle3">② Fermentation and HA purification</P>
									Refer to HA Fed-batch fermentation of HA and HA separation and purification<P class="subtitle">(2)HA characterization</P><P class="subtitle3">① CTAB</P>
									Refer to CTAB (cetyltrimethylammonium bromide) method<P class="subtitle3">② HA molecular weight easurement--GPC-RI-MALS</P>
									Gel chromatography (GPC-RI-MALS) is an effective determination of molecular weight of polysaccharides, especially for low-molecular-weight HA that cannot be directly measured using a viscometer<SUP>[20]</SUP>
									. The principle is: when samples of different molecular weights pass through gel column, the distance and the time of the process will be changed according to different molecular weight of HA, therefore different substances can be separated. Then the molecular weight and abundance of elements can be detected by differential detector and multiple angle laser light scattering.<P>
									GPC-RI-MALS service was provided by Sanshu Biotechnology Co, Shanghai. Purified HA samples were sent to them. To test our HA samples, the mobile phase of the measurement is NaNO3, 10mg of HA sample was dissolved respectively by adding 1.5ml NaNO3, then the mixtures were centrifuged (12000rpm/10min) and filtered through a 0.22um filter membrane. Ultimately 100ul sample was analyzed by gel chromatography.</P></DIV></DIV></DIV></SECTION><SECTION class="col-xs-12 col-sm-10"><P class="title">Module 3- HA Micro-needle Design</P><DIV class="content"><DIV class="Csubtitle col-xs-12 col-sm-6"><SPAN class="active">1. Micro-needle Concept</SPAN></DIV><DIV class="Csubtitle col-xs-12 col-sm-6"><SPAN>2. Fabrication of cHA Hydrogel Particles</SPAN></DIV><DIV class="Csubtitle col-xs-12 col-sm-6"><SPAN>3. Preparation of the Micro-needle Mold</SPAN></DIV><DIV class="Csubtitle col-xs-12 col-sm-6"><SPAN>4. Fabrication of HA-cHA-Micro-needles</SPAN></DIV><DIV class="Csubtitle col-xs-12 col-sm-6"><SPAN>5. Water Solubility Test</SPAN></DIV><DIV class="info col-xs-12"><DIV class="One"><P class="subtitle">Micro-needle Design</P>
									At present, the preparation of micro-needles with natural and degradable polymer bio-materials has become a hot topic. In this project we designed a new type of micro-needles by using HA products from previous section.
									Cross-link is widely used in dermal filler preparation by linking one polymer chain to another. These links may take the form of covalent bonds or ionic bonds and the polymers can be either synthetic polymers or natural polymers<SUP>[21]</SUP>
									. In polymer chemistry, &quot;cross-linking&quot; usually refers to the use of cross-linking agents to promote a change in the polymers' physical properties.。
								<P>
									At present, the mainstream chemical cross-linking agents for cosmetic HA implant preparations are diethylsulfone (DVS) and 1, 4-butanediol diglycidyl ether (BDDE)<SUP>[22]</SUP>
									. DVS, although widely used in R&amp;D, has high cytotoxicity. Not only this toxic substance is likely to be accumulated in the body after implantation, but also it may affect normal tissue growth due to calcification of the implant. In comparison, BDDE is the most commonly used cross-linking agent, it is biodegradable, much less toxic and more reactive than DVS, thereby safer for biomedical applications. Cross-linked with HA using BDDE making it more stable, less susceptible to enzymatic degradation, and with increased mechanical strength<SUP>[23]</SUP>
									. Previous studies<SUP>[24]</SUP>
									 showed that under alkaline condition, the opening of BDDE rings during reaction reacts with the OH group of HA to produce crosslinked network (hydrogel).</P><P>
									Micro-needles made with cross-linked HA has better swelling ability and slow bio-degradation that could result to a prolonged effectiveness of the dermal filler, which has the potential to replace surgical approach or injection of HA for anti-wrinkle treatment. In addition, we performed water solubility test in water to understand whether cross-linking, solidifying and curing process could affect HA solubility.</P><DIV class="Cimg col-xs-12 col-sm-6"><P>
										Fig9: Hyaluronic acid cross-linking</P></DIV></DIV><DIV class="Two"><P class="subtitle">Fabrication of cHA Hydro-gel Particles</P>
									Cross-linked HA (cHA) reagent solution was prepared by mixing 200μL of 1,4-butanediol diglycidyl ether (BDDE) into 9.8mL of 0.25mol/L NaOH (pH13). Approximately 1.0g of HA powder was added to the cHA reagent solution and stirred well, and thoroughly mixed at 40°C for 5 hours. The prepared hydrogel was grounded, squeezed out and screened with a 170 mesh sieve to get particles having a diameter of less than 90μm<SUP>[25]</SUP>
									.
								</DIV><DIV class="Three"><P class="subtitle">Preparation of The Micro-needle Mold</P>
									The micro-molds made of resin were prepared by 3D-printed technology provided by the Engineering Department of our school. The needle cavities are cone shape, with 6500um in depth and 2500um in diameter, and on the micro-molds are patterned into 3 × 3 (round shape) and 10 × 10 (rectangular shape).<DIV class="img colxs12"><P>
										Fig10：Illustration of micro-needle molds</P></DIV></DIV><DIV class="four"><P class="subtitle">Fabrication of HA-cHA-micro-needles</P>
									The HA-cHA-micro-needles was prepared by mixing different proportions of uncross-linked HA powder and cross-linked HA (cHA) powder to become mixtures, the ratios tested were 1:0, 1:1, and 5:1 (HA-cHA)<SUP>[26]</SUP>
									. The mixtures were dissolved in ultrapure water and a 20% (w/v) viscous polymer solution was stirred at room temperature under a magnetic stirrer for the production of homogenous suspension. The homogenous suspension was poured into the mold, and centrifuged at 10,000 rpm for 10 min to ensure the full loading to the needle cavities. Excess liquid was removed by using a tape. This step was repeated twice until the needle cavity was full, and then the viscous polymer solution was poured onto the surface of the mold, any residual solution remaining at the edge of the mold was removed by a blade. After curing overnight at room temperature, micro-needle was successfully prepared by separating from the mold using a blade.<DIV class="Cimg col-xs-12"><P>
										Fig11: Flow chart of preparation of microneedle</P></DIV></DIV><DIV class="five"><P class="subtitle">Water Solubility Measurement</P>
									Cured micro-needles into water at room temperature for 10 min to study water solubility property. Dissolving status was recorded.</DIV></DIV></DIV></SECTION><SECTION class="Reference col-xs-12 col-sm-10"><P><SPAN class="col-xs-4 col-lg-2">Reference</SPAN></P><P class="col-xs-12 col-sm-6"><SPAN>[1] </SPAN>O'Regan, M., I. Martini. 1994. Molecular mechanisms and genetics of hyaluronan biosynthesis. [J]. Biol. Macromol. 16:283-286. HA synthesis pathway.
					</P><P class="col-xs-12 col-sm-6"><SPAN>[2] </SPAN>Sloma, A., R. Behr. 2003. Methods for producing hyaluronan in a recombinant host cell. World patent application WO03/054163.
					</P><P class="col-xs-12 col-sm-6"><SPAN>[3] </SPAN>Crater, D. L., and I. van de Rijn. 1995. Hyaluronic acid synthesis operon (has) expression in group A streptococci. [J]. Biol. Chem. 270:18452-18458.
					</P><P class="col-xs-12 col-sm-6"><SPAN>[4] </SPAN>Widner, B., M. Thomas.2000. Development of marker-free strains of Bacillus subtilis capable of secreting high levels of industrial enzymes. [J]. Microbiol. Biotechnol.25:204-212
					</P><P class="col-xs-12 col-sm-6"><SPAN>[5] </SPAN>Meng, G.,Fütterer, K.,2003.Structural frame work of fructosyl transfer in Bacillus subtilis levan sucrase.Nat.Struct.Mol.Biol.10,935–941.
					</P><P class="col-xs-12 col-sm-6"><SPAN>[6] </SPAN>Peng Jin, Guocheng Du. High-yield novel leech hyaluronidase to expedite the preparation of specific hyaluronan oligomers [J].Scientific Reports, 2014: 1-2
					</P><P class="col-xs-12 col-sm-6"><SPAN>[7] </SPAN>Yong-Hao C, Qiang W, Establishment of CTAB Turbidimetric method to determine hyaluronic acid content in fermentation broth [J] Carbohydrate Polymers 78 (2009) 178–181 
					</P><P class="col-xs-12 col-sm-6"><SPAN>[8] </SPAN>Peng Jin, Guocheng Du, Zhen Kang. High-yield novel leech hyaluronidase to expedite the preparation of specific hyaluronan oligomers[J].Scientific Reports, 2014: 1-2
					</P><P class="col-xs-12 col-sm-6"><SPAN>[9] </SPAN>Ferguson, E. L., Roberts, J.Griffiths, P. C., &amp; Thomas, D. W. (2011). Eval-uation of the physical and biological properties of hyaluronan and hyaluronan fragments. International Journal of Pharmaceutics, 420, 84–92.
					</P><P class="col-xs-12 col-sm-6"><SPAN>[10] </SPAN>Kogan, G., Soltes, L., Stern, R., &amp; Gemeiner, P. (2007). Hyaluronic acid: A natural biopolymer with a broad range of biomedical and industrial applications.Biotechnology Letters, 29, 17–25.
					</P><P class="col-xs-12 col-sm-6"><SPAN>[11] </SPAN>Jin, P., Kang, Z., Zhang . (2014). High-yield novel leechhyaluronidase to expedite the preparation of specific hyaluronan oligomers.Scientific Reports, 4, 1–8.
					</P><P class="col-xs-12 col-sm-6"><SPAN>[12] </SPAN>Linker, A., Hoffman, P. (1957). The hyaluronidase of the leech: Anendoglucuronidase. Nature, 180, 810–811.Linker, A., Meyer, K., &amp; Hoffman, P. (1960).
					</P><P class="col-xs-12 col-sm-6"><SPAN>[13] </SPAN>Nermeen S.El-Safory The production of hyaluronate oligosac-charides by leech hyaluronidase and alkali. Journal of Biological Chemistry, 235,924–927.
					</P><P class="col-xs-12 col-sm-6"><SPAN>[14] </SPAN>Sargent.F (2007) The twin-arginine transport system:moving folded proteins across membranes. Biochem SocTrans 35, 835–847.
					</P><P class="col-xs-12 col-sm-6"><SPAN>[15] </SPAN>Jongbloed JD, Grieger U,(2004) Twominimal Tat translocases in Bacillus. Mol Microbiol 54,1319–1325.
					</P><P class="col-xs-12 col-sm-6"><SPAN>[16] </SPAN>Richman, P. G. &amp; Baer, H. A convenient plate assay for the quantitation of hyaluronidase in Hymenoptera venoms. Anal. Biochem. 109, 376–381 (1980).
					</P><P class="col-xs-12 col-sm-6"><SPAN>[17] </SPAN>Jin,P.,Kang,Z.,Zhang .2014.High-yield novel leech hyaluronidase to expedite the preparation of specific hyaluronan oligomers.[J].4,4471.
					</P><P class="col-xs-12 col-sm-6"><SPAN>[18] </SPAN>Ghatak,S.,Misra,S.,Toole,B.P.,2002.Hyaluronan oligosaccharides inhibit anchorage independent growth of tumor cells by suppressing the phosphoinositide 3-kinase[J].277,38013–38020.
					</P><P class="col-xs-12 col-sm-6"><SPAN>[19] </SPAN>Boltje,T.J.,Buskas,T.,Boons,G.J.,2009.Opportunities and challenges in synthetic oligosaccharide and glycoconjugate research.Nat.Chem.1,611–622.
					</P><P class="col-xs-12 col-sm-6"><SPAN>[20] </SPAN>Tranchepain, F. Deschrevel, B.et al. (2006). A complete set of hyaluronan fragments obtainedfrom hydrolysis catalyzed by hyaluronidase: Application to studies of hyaluro-nan mass distribution by simple HPLC devices. Analytical Biochemistry, [J]348,232–242.
					</P><P class="col-xs-12 col-sm-6"><SPAN>[21] </SPAN>Ghosh K, Shu XZ, Mou R, Lombardi J, Prestwich GD, Rafailovich MH, et al., Rheological characterization of in situ cross-linkable hyaluronan hydrogels, Biomacromolecules,
2005;6(5):2857-65.
					</P><P class="col-xs-12 col-sm-6"><SPAN>[22] </SPAN>K. De Boulle, R. Glogau, A Review of the Metabolism of 1,4-Butanediol Diglycidyl Ether-Crosslinked Hyaluronic Acid Dermal Fillers[J] 39 (2013) 1758.
					</P><P class="col-xs-12 col-sm-6"><SPAN>[23] </SPAN>M. Al-Sibani, A. Al-Harrasi.Study of the effect of mixing approach on cross-linking efficiency of hyaluronic acid-based hydrogel cross-linked with 1,4-butanediol diglycidyl ether.[J] 91 (2016) 131.
					</P><P class="col-xs-12 col-sm-6"><SPAN>[24] </SPAN>B. Yang, X. Guo, H.Zang,Determination of modification degree in BDDE-modified hyaluronic acid hydrogel by SEC/MS. [J]131 (2015) 233.
					</P><P class="col-xs-12 col-sm-6"><SPAN>[25] </SPAN>Jeong YI, Kim ST, Jin SG,et al., Cisplatin incorporated hyaluronic acid nanoparticles based on ion complex formation, Journal of pharmaceutical sciences, 2008;97(3):1268-76.
					</P><P class="col-xs-12 col-sm-6"><SPAN>[26] </SPAN>Vauthier C, Bouchemal K. Methods for the preparation and manufacture of polymeric
						
						nanoparticles, Pharmaceutical research, 2009;26(5):1025-58.
					</P></SECTION></DIV></DIV><SECTION class="btn_nav animated"><UL class="container"><LI class="col-xs-12 Navigation"><DL><DT>Navigation</DT></DL></LI><LI class="col-xs-12 Home"><DL><DT><A href="https://2018.igem.org/Team:SSTi-SZGD">Home</A></DT></DL></LI><LI class="col-xs-12 Project"><DL><DT>Project</DT><DD><A href="https://2018.igem.org/Team:SSTi-SZGD/Description">Description</A></DD><DD><A href="https://2018.igem.org/Team:SSTi-SZGD/Design">Design</A></DD><DD><A href="https://2018.igem.org/Team:SSTi-SZGD/Results">Results</A></DD><DD><A href="https://2018.igem.org/Team:SSTi-SZGD/InterLab">InterLab</A></DD><DD><A href="https://2018.igem.org/Team:SSTi-SZGD/Protocols">Protocols</A></DD><DD><A href="https://2018.igem.org/Team:SSTi-SZGD/Safety">Safety</A></DD><DD><A href="https://2018.igem.org/Team:SSTi-SZGD/Notebook">Notebook</A></DD></DL></LI><LI class="col-xs-12 Parts"><DL><DT>Parts</DT><DD><A href="https://2018.igem.org/Team:SSTi-SZGD/Parts#All_parts">All Parts</A></DD><DD><A href="https://2018.igem.org/Team:SSTi-SZGD/Parts#Basic_part">Basic Part</A></DD><DD><A href="https://2018.igem.org/Team:SSTi-SZGD/Parts#Composite_part">Composite Part</A></DD><DD><A href="https://2018.igem.org/Team:SSTi-SZGD/Parts#Improved_part">Improved Biobrick</A></DD></DL></LI><LI class="col-xs-12 Application"><DL><DT>Application</DT><DD><A href="https://2018.igem.org/Team:SSTi-SZGD/Applied_Design">Applied Design</A></DD><DD><A href="https://2018.igem.org/Team:SSTi-SZGD/Entrepreneurship">Entrepreneurship</A></DD><DD><A href="https://2018.igem.org/Team:SSTi-SZGD/Demonstrate">Demonstrate</A></DD></DL></LI><LI class="col-xs-12 Human_Practices"><DL><DT>Human Practices</DT><DD><A href="https://2018.igem.org/Team:SSTi-SZGD/Human_Practices">Integrated Human Practices</A></DD><DD><A href="https://2018.igem.org/Team:SSTi-SZGD/Public_Engagement">Public Engagement &amp; Out Reach</A></DD><DD><A href="https://2018.igem.org/Team:SSTi-SZGD/Awards">Awards</A></DD></DL></LI><LI class="col-xs-12 Acknowledgements"><DL><DT>Acknowledgements</DT><DD><A href="https://2018.igem.org/Team:SSTi-SZGD/Team">Team</A></DD><DD><A href="https://2018.igem.org/Team:SSTi-SZGD/Collaborations">Collaborations</A></DD><DD><A href="https://2018.igem.org/Team:SSTi-SZGD/Attributions">Attributions</A></DD></DL></LI></UL><DIV class="container-fluid btn_copyright">Copyright © 2018 SSTi-SZGD All Rights Reserved.</DIV></SECTION><FOOTER class="container-fluid page_footer"><SECTION class="container-fluid page_copyright">
				
				Copyright © 2018 SSTi-SZGD All Rights Reserved.
				
			</SECTION></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>