<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Results_Submitted"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Results/Submitted">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Results/Submitted&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/Submitted&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/Submitted&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Results/Submitted" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Results/Submitted</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="sliderWrap"><DIV id="slider"><DIV id="openCloseWrap" style="height:110px;"><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany">Start</A></LI></DIV><DIV class="navColumn" style="width:13%;"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team">Team</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Description">Description</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Supervisors">Supervisors</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Bielefeld">University</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/BielefeldCity">Bielefeld</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Motivation">Motivation</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Acknowledgements">Acknowledgements</A></LI></UL></DIV><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project">Project</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Application">Application</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Approach">Approach</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Theory">Theory</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Protocols">Protocols</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Model">Model</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Outlook">Outlook</A></LI></UL></DIV><DIV class="navColumn" style="width:12%;"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results">Results</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Characterization">Characterization</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Used">Used BBs</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Submitted">Submitted BBs</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Sequencing">Sequencing</A></LI></UL></DIV><DIV class="navColumn" style="width:12%;"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Safety#Biosecurity">Biosecurity</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Safety#Biosafety">Biosafety</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Safety#Risk_Assesement_.28english.29">Risk Assesement</A></LI></UL></DIV><DIV class="navColumn" style="width:13%;"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Public_Relations">Public Relations</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Public_Relations/Fotos">Fotos</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Public_Relations/Press">Press</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Public_Relations/Public_Discussion">Public Discussion</A></LI></UL></DIV><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Judging">Judging</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Judging/BioBricks">BioBricks</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Judging/Helping">Helping other teams</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Judging/Comments">Comments on iGEM</A></LI></UL></DIV><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Sponsors">Sponsors</A></LI></DIV><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Contact">Contact</A></LI></DIV></DIV></DIV><DIV id="sliderController" class="sliderController">MENU</DIV></DIV><DIV id="checkParameter" style="margin-right:250px;">http://igem-bielefeld.de/img/banner_lab.png</DIV><DIV id="subnavigation"><LI><A href="/Team:Bielefeld-Germany/Results">Results</A></LI><LI><A href="/Team:Bielefeld-Germany/Results/Characterization">Characterization</A></LI><LI><A href="/Team:Bielefeld-Germany/Results/Used">Used</A></LI><LI><A href="/Team:Bielefeld-Germany/Results/Submitted">Submitted</A></LI><LI><A href="/Team:Bielefeld-Germany/Results/Sequencing">Sequencing</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Submitted_BioBricks"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Submitted BioBricks</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#VirA_receptor"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">VirA receptor</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Mutated_virG"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Mutated virG</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#virB-promoter"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">virB-promoter</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Firefly_luciferase"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Firefly luciferase</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Neomycin_.2F_kanamycin_resistance"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Neomycin / kanamycin resistance</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#BioBrick_for_virA-screenings"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">BioBrick for virA-screenings</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Reporter_constructs"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Reporter constructs</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Characterization_constructs"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">Characterization constructs</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#VirA.2FG_signaling_system"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">VirA/G signaling system</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-11"><A href="#References"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Submitted_BioBricks"> Submitted BioBricks </SPAN></H1><P>We have submitted the following <A href="/Team:Bielefeld-Germany/Results/Characterization" title="Team:Bielefeld-Germany/Results/Characterization"> working</A> BioBricks. For a more detailed theoretical background to these BioBricks check our <A href="/Team:Bielefeld-Germany/Project/Theory" title="Team:Bielefeld-Germany/Project/Theory"> theory pages</A>, too.  
</P><H3><SPAN class="mw-headline" id="VirA_receptor">VirA receptor</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Fig. 1: The VirA receptor of <I>Agrobacterium tumefaciens</I>.</B></DIV></DIV></DIV><P>The VirA receptor is used by <I>A. tumefaciens</I> to detect acetosyringone and other phenolic substances which are secreted by plants after injury. In presence of these substances VirA phosphorylates itself and afterwards VirG, a response regulator which activates <I>vir</I> box containing promoters. These promoters control genes which are used for infecting the injured plant. 
Actually we tried to use the VirA receptor already existing in the partsregistry (&lt;partinfo&gt;K238008&lt;/partinfo&gt;). But due to sequence problems (compare results in <A href="/Team:Bielefeld-Germany/Results/Characterization" title="Team:Bielefeld-Germany/Results/Characterization"> characterization</A>) we had to isolate the <I>virA</I> gene from the Ti-plasmid of <I>A. tumefaciens</I> C58 ourselves and convert it into a BioBrick conform part. We removed an illegal <I>PstI</I> restriction site in the <I>virA</I> gene by site-directed mutagenesis. 
</P><P>You can find this BioBrick here: &lt;partinfo&gt;K389001&lt;/partinfo&gt;</P><P>You can find the K389001 BioBrick under the control of a constitutive promoter (&lt;partinfo&gt;J23110&lt;/partinfo&gt;) here: &lt;partinfo&gt;K389010&lt;/partinfo&gt;.
</P><H3><SPAN class="mw-headline" id="Mutated_virG">Mutated <I>virG</I></SPAN></H3><P>Phosphorylated VirG binds to <I>vir</I> box containing promoters and activates them. VirG is activated by the acetosyringone receptor VirA. 
This version of VirG activates <I>virB</I> promoters in <I>Escherichia coli</I> without the <I>rpoA</I>-gene from <I>Agrobacterium tumefaciens</I><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Theory#Subcloning_into_E._coli_and_receptor_function_in_new_host" class="external text" rel="nofollow">(compare to Theory)</A>. For this reason the point mutations G56V and I77V are brought into the molecule <A href="http://www.springerlink.com/content/wmq06kua5qkma1au/" class="external text" rel="nofollow">YC Jung <I>et al.</I>, (2004)</A>. Because this BioBrick is synthesized, the codon usage is optimized for <I>E. coli</I> and illegal restriction sites were removed. When you use this <I>virG</I> gene in a VirA/G signaling system you do not need &lt;partinfo&gt;K238010&lt;/partinfo&gt; anymore to get the system working in <I>E. coli</I>. 
</P><P>You can find this BioBrick here: &lt;partinfo&gt;K389002&lt;/partinfo&gt;</P><H3><SPAN class="mw-headline" id="virB-promoter"><I>virB</I>-promoter</SPAN></H3><P><I>Vir</I>-promoters from <I>A. tumefaciens</I> are induced by phosphorylated VirG response regulators and control genes for infecting plants in their natural host. They are part of the VirA/G signal transduction system. 
We wanted to use the <I>vir</I>-promoter from the partsregistry (&lt;partinfo&gt;K238011&lt;/partinfo&gt;) but the same problems occurred as with the use of the VirA receptor BioBrick from the partsregistry. So we also had to create a new <I>vir</I>-promoter BioBrick (again from Ti-plasmid of <I>A. tumefaciens</I> C58).
</P><P>You can find this BioBrick here: &lt;partinfo&gt;K389003&lt;/partinfo&gt;</P><H3><SPAN class="mw-headline" id="Firefly_luciferase">Firefly luciferase</SPAN></H3><P>Bringing the firefly luciferase gene from Promega's pGL4.10[luc2] vector into a BioBrick compatible form as a sensitive reporter gene. To amplify the signal of the luciferase three different sensitivity tuners are assembled before the luciferase gene. The sensitivity tuners were created in 2007 by the iGEM team from Cambridge and amplify the readout signal. We also assembled the firefly luciferase behind three different strong constitutive promoters for gathering additional information about this reporter gene <A href="https://2007.igem.org/Cambridge/Amplifier_project#Results" class="external text" rel="nofollow">Cambridge, (2007)</A>. 
</P><P>You can find this BioBrick here: &lt;partinfo&gt;K389004&lt;/partinfo&gt;</P><P>You can find this BioBrick with sensitivity tuner 1 here: &lt;partinfo&gt;K389401&lt;/partinfo&gt;</P><P>You can find this BioBrick with sensitivity tuner 2 here: &lt;partinfo&gt;K389402&lt;/partinfo&gt;</P><P>You can find this BioBrick with sensitivity tuner 3 here: &lt;partinfo&gt;K389403&lt;/partinfo&gt;</P><P>You can find this BioBrick under the control of a weak constitutive promoter here: &lt;partinfo&gt;K389302&lt;/partinfo&gt;</P><P>You can find this BioBrick under the control of a medium strong constitutive promoter here: &lt;partinfo&gt;K389307&lt;/partinfo&gt;</P><P>You can find this BioBrick under the control of a strong constitutive promoter here: &lt;partinfo&gt;K389318&lt;/partinfo&gt;</P><H3><SPAN class="mw-headline" id="Neomycin_.2F_kanamycin_resistance">Neomycin / kanamycin resistance</SPAN></H3><P>A neomycin / kanamycin resistance gene without promoter is isolated and brought into a BioBrick compatible form. We will use the BioBrick &lt;partinfo&gt;P1003&lt;/partinfo&gt; as source for the kanamycin resistance gene. 
</P><P>You can find this BioBrick here: &lt;partinfo&gt;K389005&lt;/partinfo&gt;</P><H3><SPAN class="mw-headline" id="BioBrick_for_virA-screenings">BioBrick for <I>virA</I>-screenings</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Fig. 2: The BioBrick &lt;partinfo&gt;K389011&lt;/partinfo&gt; for <I>virA</I> screenings.</B></DIV></DIV></DIV><P>This part contains our mutated <I>virG</I> BioBrick under the control of a constitutive promoter (&lt;partinfo&gt;J23110&lt;/partinfo&gt;) and an antibiotic resistance (&lt;partinfo&gt;K389005&lt;/partinfo&gt;) under the control of the <I>virB</I> promoter (&lt;partinfo&gt;K389003&lt;/partinfo&gt;). The better the VirA receptor recognizes a substance, the stronger the antibiotic resistance will be expressed. 
</P><P>This device is used to screen <I>virA</I> receptor mutants. The <I>virA</I> receptor gene is mutated in the &lt;partinfo&gt;BBa_K389010&lt;/partinfo&gt; BioBrick. The better the receptor detects a substance, the more VirG is phosphorylated and the stronger the kanamycin resistance from this BioBrick is expressed. The screening system for a mutated <I>virA</I> gene contains this BioBrick in a plasmid with R6K ori and the BioBrick &lt;partinfo&gt;BBa_K389010&lt;/partinfo&gt; in a plasmid with ColE1 ori. Both plasmids are transformed to and screened in <I>E. coli</I> EC100D because plasmids with R6K ori are only replicated in pir+ or pir116 <I>E. coli</I> strains. Once a receptor with high sensitivity for a screened substance is found, the plasmids are isolated and transformed to e.g. <I>E. coli</I> TOP10. Because the R6K ori does not work in this strain because it is not a pir+ / pir116 strain, it is easy to separate the mutated <I>virA</I> BioBrick from the screening plasmid.
</P><P>You can find this BioBrick here: &lt;partinfo&gt;K389011&lt;/partinfo&gt;</P><H3><SPAN class="mw-headline" id="Reporter_constructs">Reporter constructs</SPAN></H3><P>The reporter constructs are similar to the <I>virA</I> screening construct (compare fig. 2) but instead of the antibiotic resistance they carry a reporter gene. The amount of produced reporter shows the activity of the VirA receptor and the <I>vir</I> promoter, respectively. If the original <I>vir</I> promoter is too weak, we will use Cambridge's sensitivity tuners to increase the output signal of our biosensor (<A href="https://2007.igem.org/Cambridge/Amplifier_project#Results" class="external text" rel="nofollow">Cambridge, 2007</A>). 
</P><P>You can find this BioBrick with luciferase readout here: &lt;partinfo&gt;K389012&lt;/partinfo&gt;</P><P>You can find this BioBrick with mRFP readout here: &lt;partinfo&gt;K389013&lt;/partinfo&gt;</P><P>You can find this BioBrick with amplified luciferase readout No. 1 here: &lt;partinfo&gt;K389411&lt;/partinfo&gt;</P><P>You can find this BioBrick with amplified luciferase readout No. 2 here: &lt;partinfo&gt;K389412&lt;/partinfo&gt;</P><P>You can find this BioBrick with amplified luciferase readout No. 3 here: &lt;partinfo&gt;K389413&lt;/partinfo&gt;</P><H3><SPAN class="mw-headline" id="Characterization_constructs">Characterization constructs</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Fig. 3: Animation of the function of a complete characterization construct with sensitivity tuner (click to enlarge). In the system without sensitivity tuner the reporter gene is directly under the control of the <I>vir</I> promoter.</B></DIV></DIV></DIV><P>The characterization constructs are like the reporter constructs but a <I>virA</I> gene under the control of a constitutive promoter is assembled before them. These constructs are used to characterize the natural, unmutated VirA/G signaling system and the <I>vir</I> promoter, respectively. The natural system is induced with acetosyringone and the readout is measured to determine the behaviour of the VirA/G signaling system. 
</P><P>You can find this BioBrick with luciferase readout here: &lt;partinfo&gt;K389015&lt;/partinfo&gt;</P><P>You can find this BioBrick with mRFP readout here: &lt;partinfo&gt;K389016&lt;/partinfo&gt;</P><P>You can find this BioBrick with amplified luciferase readout No. 1 here: &lt;partinfo&gt;K389421&lt;/partinfo&gt;</P><P>You can find this BioBrick with amplified luciferase readout No. 2 here: &lt;partinfo&gt;K389422&lt;/partinfo&gt;</P><P>You can find this BioBrick with amplified luciferase readout No. 3 here: &lt;partinfo&gt;K389423&lt;/partinfo&gt;</P><H3><SPAN class="mw-headline" id="VirA.2FG_signaling_system">VirA/G signaling system</SPAN></H3><P>A complete VirA/G signaling system without readout was submitted. The system contains a <I>virA</I> and our mutated <I>virG</I> gene under the control of a constitutive promoter and a <I>vir</I> promoter without reporter gene. 
</P><P>You can find this BioBrick here: &lt;partinfo&gt;K389017&lt;/partinfo&gt;</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><UL><LI><A href="https://2007.igem.org/Cambridge/Amplifier_project" class="external text" rel="nofollow">Cambridge iGEM Team wiki 2007, amplifier project</A></LI></UL><UL><LI>YC Jung <I>et al.</I>, (2004), <I>Mutants of </I>Agrobacterium tumefaciens virG<I> Gene That Activate Transcription of </I>vir<I> Promoter in </I>Escherichia coli<I>, Current Microbiol 49:334-340.</I></LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Bielefeld-Germany/Results/Submitted">http://2010.igem.org/Team:Bielefeld-Germany/Results/Submitted</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Results/Submitted" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Results/Submitted" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/Submitted&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/Submitted&amp;oldid=204820" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>