"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Bielefeld-Germany/Results
From 2010.igem.org
Start
Team
Description
Supervisors
University
Bielefeld
Motivation
Acknowledgements
Project
Application
Approach
Theory
Protocols
Model
Outlook
Results
Characterization
Used BBs
Submitted BBs
Sequencing
Safety
Biosecurity
Biosafety
Risk Assesement
Public Relations
Fotos
Press
Public Discussion
Judging
BioBricks
Helping other teams
Comments on iGEM
Sponsors
Contact
MENU
http://igem-bielefeld.de/img/banner_lab.png
Results
Characterization
Used
Submitted
Sequencing
Results
Figure 1: Visualization of induced (from left to right) <partinfo>K389421</partinfo>, <partinfo>K389422</partinfo> and <partinfo>K389423</partinfo> sensitivity tuner amplified vir-system.
On the next pages we are going to present you our BioBrick work.
One approach of our project was to visualize a readout by a light emitting system. So a luciferase BioBrick was built. This BioBrick is a highly sensitive reporter gene which was successfully applied in different devices with a PoPS output. The characterization of this BioBrick can be found here .
The second step was to establish a complete VirA/G signaling system from Agrobacterium tumefaciens in Escherichia coli. For this purpose three new BioBricks were created - virA, virG and a vir promoter. The VirA receptor recognizes the phenolic substance acetosyringone and transmits this information to the VirG response regulator - a bacterial transcription factor which activates vir promoters. This natural VirA/G signaling system works in E. coli and was measured and characterized by using the reporter genes mRFP and luciferase .
In the third step we tried to modify the VirA receptor by directed mutagenesis using error-prone PCR, in order to detect substances other than acetosyringone (e.g. capsaicin). For that purpose we designed a high-throughput screening system with a selective system based on the expression of a kanamycin resistance. We tried to modify the receptor by different modification strategies like error prone (random mutagenesis) or specific amino acid changes ( mut1 / mut2 ). The constructs mut1 and mut2 were done by an in silico approach. Unfortunately they did not work as expected. In a short period of time we found many positive clones, but further analysis indicated that the VirA receptor was changed to constitutive activity. Thereby, due to the short time available, we did not manage to construct a receptor that was specific for other phenolic substance, e.g. capsaicin.
We tried to create a light signal that is visible to the human eye as another part of our project. So genetic amplifiers ( Cambridge, iGEM 2007, amplifier project ) were added to our luciferase BioBrick. This amplified luciferase readout was coupled with the VirA/G signaling system in order to visualize the induction behaviour. The amplifiers worked so far and gave a visible luciferase readout but the basal expression of the vir promoter was also amplified so the induced status of the VirA/G system was not discriminable properly from the uninduced status.
We entered the following BioBricks to the partsregistry:
<groupparts>iGEM010 Bielefeld-Germany</groupparts>
Retrieved from " http://2010.igem.org/Team:Bielefeld-Germany/Results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
