"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Bielefeld-Germany/Results/Used
From 2010.igem.org
(Redirected from Team:Bielefeld-Germany/BioBricks/Used )
Start
Team
Description
Supervisors
University
Bielefeld
Motivation
Acknowledgements
Project
Application
Approach
Theory
Protocols
Model
Outlook
Results
Characterization
Used BBs
Submitted BBs
Sequencing
Safety
Biosecurity
Biosafety
Risk Assesement
Public Relations
Fotos
Press
Public Discussion
Judging
BioBricks
Helping other teams
Comments on iGEM
Sponsors
Contact
MENU
http://igem-bielefeld.de/img/banner_lab.png
Results
Characterization
Used
Submitted
Sequencing
Used BioBricks
This is a list of all BioBricks we used from the registry or created by ourselves and a short explanation, why and / or how we used them. The list is sorted alphabetically.
BioBrick (name and partnumber)
Status
Comment
double terminator, <partinfo>B0017</partinfo>: <partinfo>B0017 SpecifiedComponents</partinfo>
from registry
This is a double terminator which contains two <partinfo>B0010</partinfo> terminators. We used this double terminator instead of the commonly used <partinfo>B0015</partinfo> double terminator (which was assembled from the <partinfo>B0010</partinfo> and the <partinfo>B0012</partinfo> terminator) because of problems mentioned in the partsregistry with the <partinfo>B0012</partinfo> terminator.
RBS, <partinfo>B0034</partinfo>: <partinfo>B0034 SpecifiedComponents</partinfo>
from registry
The strong ribosomal binding site (RBS) AGGAG was always assembled.
lacI gene, <partinfo>C0012</partinfo>: <partinfo>C0012 SpecifiedComponents</partinfo>
from registry
This is a gene for the lac repressor lacI with a LVA degradation tag. We used this BioBrick to build a tightly regulated lac-operon <partinfo>K389050</partinfo> with low basal transcription. It seems that this lacI is either degraded too quickly or it does not work properly because it is not possible to repress the lac operon with this part.
mRFP, <partinfo>E1010</partinfo>: <partinfo>E1010 SpecifiedComponents</partinfo>
from registry
This BioBrick was used as a reporter gene, e.g. in the construct <partinfo>K389013</partinfo>. mRFP as a reporter gene has the advantage that it is easy to measure and comparatively stable .
lacIq promoter, <partinfo>I14032</partinfo>: <partinfo>I14032 SpecifiedComponents</partinfo>
from registry
This promoter is a strong constitutive promoter which is natively located upstream of the lacI gene in E. coli lacIq strains. It was used to build a tightly regulated lac-operon <partinfo>K389050</partinfo> in which it was assembled upstream of the lacI gene from the partsregistry <partinfo>C0012</partinfo>.
sensitivity tuner, <partinfo>I746370</partinfo>: <partinfo>I746370 SpecifiedComponents</partinfo>
from registry
We used this BioBrick to amplify our read-out signal luciferase in the part <partinfo>K389401</partinfo>. This BioBrick is supposed to amplify the effect of a PoPS device 15 - 20 fold compared to a system without sensitivity tuner ( Cambridge, iGEM 2007, amplifier project ).
sensitivity tuner, <partinfo>I746380</partinfo>: <partinfo>I746380 SpecifiedComponents</partinfo>
from registry
We used this BioBrick to amplify our read-out signal luciferase in the part <partinfo>K389402</partinfo>. This BioBrick is supposed to amplify the effect of a PoPS device 10 fold compared to a system without sensitivity tuner ( Cambridge, iGEM 2007, amplifier project ).
sensitivity tuner, <partinfo>I746390</partinfo>: <partinfo>I746390 SpecifiedComponents</partinfo>
from registry
We used this BioBrick to amplify our read-out signal luciferase in the part <partinfo>K389403</partinfo>. This BioBrick is supposed to amplify the effect of a PoPS device 30 - 35 fold compared to a system without sensitivity tuner ( Cambridge, iGEM 2007, amplifier project ).
mRFP generator, <partinfo>J04450</partinfo>: <partinfo>J04450 SpecifiedComponents</partinfo>
from registry
This BioBrick was used as a visible selection marker for cloning other BioBricks into the <partinfo>pSB1C3</partinfo> plasmid to submit them to the registry. The promoter in this mRFP generator is pretty strong so it is easy to separate colonies which carry this BioBrick from colonies which do not carry this BioBrick anymore.
strong constitutive promoter, <partinfo>J23102</partinfo>: <partinfo>J23102 SpecifiedComponents</partinfo>
from registry
A strong constitutive promoter which was used to create BioBrick <partinfo>K389318</partinfo> to test the firefly luciferase BioBrick <partinfo>K389004</partinfo> and the luciferase measurement system.
weak constitutive promoter, <partinfo>J23103</partinfo>: <partinfo>J23103 SpecifiedComponents</partinfo>
from registry
A weak constitutive promoter which was used to create BioBrick <partinfo>K389302</partinfo> to test the firefly luciferase BioBrick <partinfo>K389004</partinfo> and the luciferase measurement system.
medium strong constitutive promoter, <partinfo>J23110</partinfo>: <partinfo>J23110 SpecifiedComponents</partinfo>
from registry
This medium strong constitutive promoter from the registry was assembled before our BioBricks, so they are neither expressed too strong, so the cell suffers from metabolomic stress, nor too weak, so there is no visible effect of our BioBricks. This BioBrick was used e.g. in <partinfo>K389010</partinfo> or <partinfo>K389011</partinfo>.
medium strong constitutive promoter, <partinfo>J23115</partinfo>: <partinfo>J23115 SpecifiedComponents</partinfo>
from registry
A medium strong constitutive promoter which was used to create BioBrick <partinfo>K389307</partinfo> to test the firefly luciferase BioBrick <partinfo>K389004</partinfo> and the luciferase measurement system.
R6K origin, <partinfo>J61001</partinfo>: <partinfo>J61001 SpecifiedComponents</partinfo>
from registry
This BioBrick was used to create a plasmid without ColE1 ori. The R6K ori has a different compatibility class than the ColE1 ori. So it is used for a two plasmid screening system. The R6K ori is a medium copy plasmid in pir+ / pir116 E. coli strains like EC100D but it does not replicate in pir- strains like E. coli TOP10. So it is easy to separate a ColE1 ori plasmid from another with R6K ori by simply transforming both to e.g. E. coli TOP10.  The R6K ori was cloned into the pSB1C3 plasmid. Subsequently, the plasmid containing the R6K ori was digested with Hin6I. The largest fragment was extracted from an agarose gel and ligated with T4 DNA Polymerase. Thereby the ColE1 ori was removed, leaving a small fragment of about 30 bp.
virA, <partinfo>K238008</partinfo>: <partinfo>K238008 SpecifiedComponents</partinfo>
from registry, fixed
We wanted to use this virA BioBrick from the registry but problems occurred ( characterization ), so we created our own virA from Agrobacterium tumefaciens C58 TI-plasmid. Our own (working) virA has the partnumber <partinfo>K389001</partinfo>.
vir-promoter, <partinfo>K238011</partinfo>: <partinfo>K238011 SpecifiedComponents</partinfo>
from registry, fixed
The same problems occurred as with the virA BioBrick - this one from the registry does not work properly , so we made a vir-promoter by ourselves (<partinfo>K389003</partinfo>). The part that was sent to us is actually a tetR gene (<partinfo>C0040</partinfo>) under the control of a constitutive promoter (<partinfo>J23105</partinfo>) before a double terminator (<partinfo>B0015</partinfo>, compare our sequencing results ).
virA, <partinfo>K389001</partinfo>: <partinfo>K389001 SpecifiedComponents</partinfo>
new
The VirA receptor is used by A. tumefaciens to detect acetosyringone and other phenolic substances which are secreted by plants after injury. In presence of these substances VirA phosphorylates itself and afterwards VirG, a response regulator which activates vir promoters. These promoters control genes which are used for infecting the injured plant.  This virA gene was isolated from the TI-plasmid of A. tumefaciens C58. An illegal PstI restriction site was removed by site-directed mutagenesis. The functionality of this BioBrick was proved with <partinfo>K389015</partinfo> and <partinfo>K389016</partinfo>.
mutated virG, <partinfo>K389002</partinfo>: <partinfo>K389002 SpecifiedComponents</partinfo>
synthesized, new
This is the VirG response regulator from the VirA/G receptor system. The VirA/G receptor system is used by A. tumefaciens to detect phenolic substances secreted by injured plants, e.g. acetosyringone. In presence of these substances, VirG is activated by the VirA receptor and induces the transcription of genes under the control of a vir promoter. Normally, VirG needs the rpoA gene (RNA polymerase subunit) from A. tumefaciens to work in Escherichia coli but this virG BioBrick is mutated, so it works with the rpoA subunit from E. coli. For this reason the point mutations G56V and I77V were brought into the gene ( YC Jung et al., 2004 ). Because this BioBrick is synthesized (Mr. Gene GmbH), codon usage is optimized for E. coli and illegal restriction sites were removed. When you use this virG gene in a VirA/G signaling system you do not need <partinfo>BBa_K238010</partinfo> anymore to get the system working in E. coli.
vir-promoter, <partinfo>K389003</partinfo>: <partinfo>K389003 SpecifiedComponents</partinfo>
new
Vir-promoters from A. tumefaciens are induced by phosphorylated VirG response regulators and control genes for infecting plants in their natural host. They are part of the VirA/G signal transduction system. This is a vir promoter from A. tumefaciens C58. It is located on the TI-plasmid upstream the virB genes.
firefly luciferase, <partinfo>K389004</partinfo>: <partinfo>K389004 SpecifiedComponents</partinfo>
new
This is a BioBrick for a firefly (Photinus pyralis) luciferase. It is a sensitive reporter gene . It was isolated from Promega's pGL4.10[luc2] plasmid.
Kanamycin resistance gene, <partinfo>K389005</partinfo>: <partinfo>K389005 SpecifiedComponents</partinfo>
new / modified
This is a kanamycin / neomycin resistance gene without promoter or RBS. It is used for directed evolution of virA and mutated virA screenings, respectively. It was isolated from the BioBrick <partinfo>P1003</partinfo> (a kanamycin resistance gene with promoter and RBS).
virA generator, <partinfo>K389010</partinfo>: <partinfo>K389010 SpecifiedComponents</partinfo>
composite
This part is for expressing a VirA receptor. A medium strong constitutive promoter is used, so there is enough VirA receptor expressed but not too much so the cell suffers from the expression (VirA is a membrane protein). This version exists in a <partinfo>pSB1C3</partinfo> and a <partinfo>pSB1AT3</partinfo> vector. We used the BioBrick in the <partinfo>pSB1AT3</partinfo> plasmid for error-prone PCR and later in our two plasmid screening system together with the <partinfo>K389011</partinfo> VirA-screening device.
virA screening device, <partinfo>K389011</partinfo>: <partinfo>K389011 SpecifiedComponents</partinfo>
composite
This BioBrick contains the new, mutated virG which works in E. coli without the rpoA gene from A. tumefaciens which is expressed constitutively and a kanamycin resistance gene which is under the control of a vir promoter. It is used to screen mutants of virA which are on a different plasmid in a <partinfo>K389010</partinfo>-like part and were created by error-prone PCR. The better the VirA receptor recognizes a substance the more resistant the cell is against kanamycin. This device has to be on a plasmid with different ori than the virA which should be screened (different compatibility classes). We cloned this BioBrick into a plasmid with R6K ori which only replicates in pir+ or pir116 strains (e.g.E. coli EC100D), so we can easily seperate the two plasmids by transforming them into strains without Pir protein (e.g. E. coli TOP10).
virA reporter device with luc, <partinfo>K389012</partinfo>: <partinfo>K389012 SpecifiedComponents</partinfo>
composite
This BioBrick is similar to <partinfo>K389011</partinfo> but with the reporter luciferase instead of an antibiotic resistance gene. With this BioBrick it is possible to measure the activity of the vir promoter and the VirA receptor, respectively.
virA reporter device with mRFP, <partinfo>K389013</partinfo>: <partinfo>K389013 SpecifiedComponents</partinfo>
composite
This BioBrick is similar to <partinfo>K389011</partinfo> but with the reporter mRFP instead of an antibiotic resistance gene. With this BioBrick it is possible to measure the activity of the vir promoter and the VirA receptor, respectively.
virA screening approach test device, <partinfo>K389014</partinfo>: <partinfo>K389014 SpecifiedComponents</partinfo>
composite
This BioBrick is an assembly of <partinfo>K389010</partinfo> and <partinfo>K389011</partinfo>. It is used for testing the general screening concept of expressing an antibiotic resistance with a vir promoter after induction with acetosyringone (the natural VirA inductor).
vir promoter characterisation part with luc, <partinfo>K389015</partinfo>: <partinfo>K389015 SpecifiedComponents</partinfo>
composite
This BioBrick is an assembly of <partinfo>K389010</partinfo> and <partinfo>K389012</partinfo>. It is used for characterizing the natural VirA/G system and the vir promoter, respectively. It is also possible to demonstrate the functionality of four new created BioBricks in one device.
vir promoter characterisation part with mRFP, <partinfo>K389016</partinfo>: <partinfo>K389016 SpecifiedComponents</partinfo>
composite
This BioBrick is an assembly of <partinfo>K389010</partinfo> and <partinfo>K389013</partinfo>. It is used for characterizing the natural VirA/G system and the vir promoter, respectively.
vir promoter part, <partinfo>K389017</partinfo>: <partinfo>K389017 SpecifiedComponents</partinfo>
composite
This is a complete VirA/G signaling system without a reporter gene, so any gene of interest can be assembled to a vir promoter. We assembled some sensitivity tuners and a luciferase gene behind this BioBrick, so we get an enhanced luciferase signal (e.g. in part <partinfo>K389421</partinfo>).
tightly controlled lac operator, <partinfo>K389050</partinfo>: <partinfo>K389050 SpecifiedComponents</partinfo>
composite
A lac operator is combined with a lacIq so there is always enough lacI to repress the lac operator. The system is still inducible with IPTG but it has a low basal transcription (in theory). This part should be used to test the sensitivity of the luciferase BioBrick <partinfo>K389004</partinfo> and compare it to the sensitivity of the mRFP BioBrick <partinfo>E1010</partinfo>. But the part did not work as expected - it had a very high basal expression. The problem is probably the lacI gene <partinfo>C0012</partinfo> from the registry which has a degradation tag. Either this tag leads to that quick degradation, that the lacI repressor cannot work as a repressor anymore or the repressor itself does not work properly.
tightly controlled lac operator + luc, <partinfo>K389051</partinfo>: <partinfo>K389051 SpecifiedComponents</partinfo>
composite
The part <partinfo>K389050</partinfo> is combined with a luciferase gene as reporter. But <partinfo>K389050</partinfo> does not work as expected so this part could not be used for its original purpose (demonstrating the sensitivity of the firefly luciferase).
tightly controlled lac operator + mRFP, <partinfo>K389052</partinfo>: <partinfo>K389052 SpecifiedComponents</partinfo>
composite
The part <partinfo>K389050</partinfo> is combined with a mRFP gene as reporter. But <partinfo>K389050</partinfo> does not work as expected so this part could not be used for its original purpose (comparing the sensitivity of mRFP with the sensitivity of the firefly luciferase).
weak constitutive promoter + luc, <partinfo>K389302</partinfo>: <partinfo>K389302 SpecifiedComponents</partinfo>
composite
Luciferase under the control of a weak constitutive promoter.
medium strong constitutive promoter + luc, <partinfo>K389307</partinfo>: <partinfo>K389307 SpecifiedComponents</partinfo>
composite
Luciferase under the control of a medium strong constitutive promoter.
strong constitutive promoter + luc, <partinfo>K389318</partinfo>: <partinfo>K389318 SpecifiedComponents</partinfo>
composite
Luciferase under the control of a strong constitutive promoter.
sensitivity tuner no. 1 without promoter + luc readout, <partinfo>K389401</partinfo>: <partinfo>K389401 SpecifiedComponents</partinfo>
new / modified
The <partinfo>I746370</partinfo> sensitivity tuner was modified: the arabinose promoter and the GFP readout were erased from the original part and a firefly luciferase BioBrick was added as a new readout. This part can be used behind a random promoter and gives an enhanced luciferase readout signal.
sensitivity tuner no. 2 without promoter + luc readout, <partinfo>K389402</partinfo>: <partinfo>K389402 SpecifiedComponents</partinfo>
new / modified
The <partinfo>I746380</partinfo> sensitivity tuner was modified: the arabinose promoter and the GFP readout were erased from the original part and a firefly luciferase BioBrick was added as a new readout. This part can be used behind a random promoter and gives an enhanced luciferase readout signal.
sensitivity tuner no. 3 without promoter + luc readout, <partinfo>K389403</partinfo>: <partinfo>K389403 SpecifiedComponents</partinfo>
new / modified
The <partinfo>I746390</partinfo> sensitivity tuner was modified: the arabinose promoter and the GFP readout were erased from the original part and a firefly luciferase BioBrick was added as a new readout. This part can be used behind a random promoter and gives an enhanced luciferase readout signal.
virA reporter device with sensitivity tuner no. 1 + luc readout, <partinfo>K389411</partinfo>: <partinfo>K389411 SpecifiedComponents</partinfo>
composite
This is a modified <partinfo>I746370</partinfo> sensitivity tuner with luciferase readout behind a vir promoter. This part also contains a virG gene under the control of a constitutive promoter. The plan is to assemble a mutated virA from a <partinfo>K389010</partinfo>-like BioBrick in front of this part to get a strong readout signal in order to reach our goal - make spiciness visible. The mutated virA is generated with error-prone PCR and screened together with the <partinfo>K389011</partinfo> Biobrick.
virA reporter device with sensitivity tuner no. 2 + luc readout, <partinfo>K389412</partinfo>:<partinfo>K389412 SpecifiedComponents</partinfo>
composite
This is a modified <partinfo>I746380</partinfo> sensitivity tuner with luciferase readout behind a vir promoter. This part also contains a virG gene under the control of a constitutive promoter. The plan is to assemble a mutated virA from a <partinfo>K389010</partinfo>-like BioBrick in front of this part to get a strong readout signal in order to reach our goal - make spiciness visible. The mutated virA is generated with error-prone PCR and screened together with the <partinfo>K389011</partinfo> Biobrick.
virA reporter device with sensitivity tuner no. 3 + luc readout, <partinfo>K389413</partinfo>: <partinfo>K389413 SpecifiedComponents</partinfo>
composite
This is a modified <partinfo>I746390</partinfo> sensitivity tuner with luciferase readout behind a vir promoter. This part also contains a virG gene under the control of a constitutive promoter. The plan is to assemble a mutated virA from a <partinfo>K389010</partinfo>-like BioBrick in front of this part to get a strong readout signal in order to reach our goal - make spiciness visible. The mutated virA is generated with error-prone PCR and screened together with the <partinfo>K389011</partinfo> Biobrick.
vir promoter characterisation part with sensitivity tuner no. 1 + luc readout, <partinfo>K389421</partinfo>: <partinfo>K389421 SpecifiedComponents</partinfo>
composite
This is a modified <partinfo>I746370</partinfo> sensitivity tuner with luciferase readout behind a vir promoter characterisation BioBrick, so it also contains a virG and an unmutated virA gene under the control of a constitutive promoter. This BioBrick is used to prove the function of the modified sensitivity tuner and to check whether the sensitivity tuner enhances the luciferase signal enough to make it visible.
vir promoter characterisation part with sensitivity tuner no. 2 + luc readout, <partinfo>K389422</partinfo>: <partinfo>K389422 SpecifiedComponents</partinfo>
composite
This is a modified <partinfo>I746380</partinfo> sensitivity tuner with luciferase readout behind a vir promoter characterisation BioBrick, so it also contains a virG and an unmutated virA gene under the control of a constitutive promoter. This BioBrick is used to prove the function of the modified sensitivity tuner and to check whether the sensitivity tuner enhances the luciferase signal enough to make it visible.
vir promoter characterisation part with sensitivity tuner no. 3 + luc readout, <partinfo>K389423</partinfo>: <partinfo>K389423 SpecifiedComponents</partinfo>
composite
This is a modified <partinfo>I746370</partinfo> sensitivity tuner with luciferase readout behind a vir promoter characterisation BioBrick, so it also contains a virG and an unmutated virA gene under the control of a constitutive promoter. This BioBrick is used to prove the function of the modified sensitivity tuner and to check whether the sensitivity tuner enhances the luciferase signal enough to make it visible.
Kanamycin-resistance cassette, <partinfo>P1003</partinfo>: <partinfo>P1003 SpecifiedComponents</partinfo>
from registry
This kanamycin resistance cassette BioBrick was used as a template to create <partinfo>K389005</partinfo>.
ccdB-gene, <partinfo>P1010</partinfo>: <partinfo>P1010 SpecifiedComponents</partinfo>
from registry
The ccdB gene targets the gyrase of Escherichia coli and is lethal for all E. coli strains without the gyrase mutation gyrA462 ( openwetware ). The ccdB BioBrick is used for the 3A-assembly as a positive selection marker.
high-copy plasmid with Amp and Tet resistance, <partinfo>pSB1AT3</partinfo>: <partinfo>pSB1AT3 SpecifiedComponents</partinfo>
from registry
This plasmid was used in 3A-assemblies and as the backbone of the mutated and unmutated <partinfo>K389010</partinfo> part.
high-copy plasmid with Cm resistance, <partinfo>pSB1C3</partinfo>: <partinfo>pSB1C3 SpecifiedComponents</partinfo>
from registry
This is the plasmid used to submit the BioBricks.
lac operator, <partinfo>R0010</partinfo>: <partinfo>R0010 SpecifiedComponents</partinfo>
from registry
This is a lac operator which is inducible with IPTG and repressed by lacI. It was used to build a tightly regulated lac operator (<partinfo>K389050</partinfo>) by adding a lacI repressor gene under the control of the lacIq promoter.
References
Cambridge, iGEM 2007, amplifier project .
http://openwetware.org/wiki/CcdB , CcdB (seen on 10.10.10).
YC Jung et al. (2004) Mutants of Agrobacterium tumefaciens virG Gene That Activate Transcription of vir Promoter in Escherichia coli' , Current Microbiol 49:334-340.
Retrieved from " http://2010.igem.org/Team:Bielefeld-Germany/Results/Used "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
