<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Results_Characterization"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Results/Characterization">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Results/Characterization&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/Characterization&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/Characterization&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Results/Characterization" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Results/Characterization</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="contentSub">(Redirected from <A href="/wiki/index.php?title=Team:Bielefeld-Germany/BioBricks/Tests&amp;redirect=no" title="Team:Bielefeld-Germany/BioBricks/Tests">Team:Bielefeld-Germany/BioBricks/Tests</A>)</DIV><DIV id="sliderWrap"><DIV id="slider"><DIV id="openCloseWrap" style="height:110px;"><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany">Start</A></LI></DIV><DIV class="navColumn" style="width:13%;"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team">Team</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Description">Description</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Supervisors">Supervisors</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Bielefeld">University</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/BielefeldCity">Bielefeld</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Motivation">Motivation</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Acknowledgements">Acknowledgements</A></LI></UL></DIV><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project">Project</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Application">Application</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Approach">Approach</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Theory">Theory</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Protocols">Protocols</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Model">Model</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Outlook">Outlook</A></LI></UL></DIV><DIV class="navColumn" style="width:12%;"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results">Results</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Characterization">Characterization</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Used">Used BBs</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Submitted">Submitted BBs</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Sequencing">Sequencing</A></LI></UL></DIV><DIV class="navColumn" style="width:12%;"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Safety#Biosecurity">Biosecurity</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Safety#Biosafety">Biosafety</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Safety#Risk_Assesement_.28english.29">Risk Assesement</A></LI></UL></DIV><DIV class="navColumn" style="width:13%;"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Public_Relations">Public Relations</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Public_Relations/Fotos">Fotos</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Public_Relations/Press">Press</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Public_Relations/Public_Discussion">Public Discussion</A></LI></UL></DIV><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Judging">Judging</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Judging/BioBricks">BioBricks</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Judging/Helping">Helping other teams</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Judging/Comments">Comments on iGEM</A></LI></UL></DIV><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Sponsors">Sponsors</A></LI></DIV><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Contact">Contact</A></LI></DIV></DIV></DIV><DIV id="sliderController" class="sliderController">MENU</DIV></DIV><DIV id="checkParameter" style="margin-right:250px;">http://igem-bielefeld.de/img/banner_lab.png</DIV><DIV id="subnavigation"><LI><A href="/Team:Bielefeld-Germany/Results">Results</A></LI><LI><A href="/Team:Bielefeld-Germany/Results/Characterization">Characterization</A></LI><LI><A href="/Team:Bielefeld-Germany/Results/Used">Used</A></LI><LI><A href="/Team:Bielefeld-Germany/Results/Submitted">Submitted</A></LI><LI><A href="/Team:Bielefeld-Germany/Results/Sequencing">Sequencing</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#.3Cpartinfo.3EK238008.3C.2Fpartinfo.3E:_virA"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">&lt;partinfo&gt;K238008&lt;/partinfo&gt;: virA</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#.3Cpartinfo.3EBBa_K238011.3C.2Fpartinfo.3E:_vir-promoter"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">&lt;partinfo&gt;BBa_K238011&lt;/partinfo&gt;: vir-promoter</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#.3Cpartinfo.3EP1010.3C.2Fpartinfo.3E:_ccdB-gene"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">&lt;partinfo&gt;P1010&lt;/partinfo&gt;: ccdB-gene</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#.3Cpartinfo.3EK389004.3C.2Fpartinfo.3E:_Luciferase_from_pGL4.10.5Bluc2.5D"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">&lt;partinfo&gt;K389004&lt;/partinfo&gt;: Luciferase from pGL4.10[luc2]</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#.3Cpartinfo.3EK389011.3C.2Fpartinfo.3E:_VirA_screening_device"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">&lt;partinfo&gt;K389011&lt;/partinfo&gt;: VirA screening device</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#.3Cpartinfo.3EK389015.3C.2Fpartinfo.3E:_VirA.2FG_reporter_device_with_luciferase"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">&lt;partinfo&gt;K389015&lt;/partinfo&gt;: VirA/G reporter device with luciferase</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#.3Cpartinfo.3EK389016.3C.2Fpartinfo.3E:_VirA.2FG_reporter_device_with_mRFP"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">&lt;partinfo&gt;K389016&lt;/partinfo&gt;: VirA/G reporter device with mRFP</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#.3Cpartinfo.3EK389052.3C.2Fpartinfo.3E:_Tightly_regulated_lac_operon_with_mRFP_readout"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">&lt;partinfo&gt;K389052&lt;/partinfo&gt;: Tightly regulated lac operon with mRFP readout</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#.3Cpartinfo.3EK389421.3C.2Fpartinfo.3E.2C_.3Cpartinfo.3EK389422.3C.2Fpartinfo.3E.2C_.3Cpartinfo.3EK389423.3C.2Fpartinfo.3E:_Sensitivity_Tuner_amplified_Vir-test_system"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">&lt;partinfo&gt;K389421&lt;/partinfo&gt;, &lt;partinfo&gt;K389422&lt;/partinfo&gt;, &lt;partinfo&gt;K389423&lt;/partinfo&gt;: Sensitivity Tuner amplified Vir-test system</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Mutated_.3Cpartinfo.3EK389001.3C.2Fpartinfo.3E_.28exemplary.29:_Results_of_the_directed_mutagenesis"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Mutated &lt;partinfo&gt;K389001&lt;/partinfo&gt; (exemplary): Results of the directed mutagenesis</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#References"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id=".3Cpartinfo.3EK238008.3C.2Fpartinfo.3E:_virA">&lt;partinfo&gt;K238008&lt;/partinfo&gt;: <I>virA</I></SPAN></H1><P>We wanted to use this part in our project, but could only obtain unexpected/faulty restriction patterns. Finally we chose to sequence the part, hoping to find the cause for the maintained restriction patterns. Unfortunately we could not approve the sequence of &lt;partinfo&gt;BBa_K238008&lt;/partinfo&gt; deposited in the parts registry so that we chose to design our own VirA BioBrick. I strongly recommend using our VirA since it has been approved by multiple means, e.g. restriction patterns and sequencing (&lt;partinfo&gt;K389001&lt;/partinfo&gt;).
</P><H1><SPAN class="mw-headline" id=".3Cpartinfo.3EBBa_K238011.3C.2Fpartinfo.3E:_vir-promoter">&lt;partinfo&gt;BBa_K238011&lt;/partinfo&gt;: <I>vir</I>-promoter</SPAN></H1><P>We wanted to use this part in our project, but could only obtain unexpected/faulty restriction patterns. Finally we chose to sequence the part, hoping to find the cause for the maintained restriction patterns. Unfortunately we could not approve the sequence of &lt;partinfo&gt;BBa_K238011&lt;/partinfo&gt; deposited in the parts registry so that we chose to design our own VirA BioBrick. I strongly recommend using our VirA since it has been approved by multiple means, e.g. restriction patterns and sequencing (&lt;partinfo&gt;K389003&lt;/partinfo&gt;).
</P><H1><SPAN class="mw-headline" id=".3Cpartinfo.3EP1010.3C.2Fpartinfo.3E:_ccdB-gene">&lt;partinfo&gt;P1010&lt;/partinfo&gt;: <I>ccdB</I>-gene</SPAN></H1><P>The <I>ccdB</I> gene targets the gyrase of <I>Escherichia coli</I> and is lethal for all <I>E. coli</I> strains without the gyrase mutation gyrA462 (<A href="http://openwetware.org/wiki/CcdB" class="external text" rel="nofollow">Openwetware</A>). The <I>ccdB</I> BioBrick is used for the 3A-assembly as a positive selection marker. 
We transformed this BioBrick into <I>E. coli</I> JM109, DH5Î±, TOP10, XL1-Blue, EC100D and DB3.1. <I>E. coli</I> JM109, XL1-Blue and DH5Î± seem to be <I>ccdB</I> resistant because there were as much colonies after P1010 transformation as observed with DB3.1. The P1010 works as expected in <I>E. coli</I> TOP10, EC100D (no colonies after transformation) and DB3.1 (many colonies after transformation).
</P><CENTER>Table 1: Results of the transformation of the cell-death gene <I>ccdB</I>, BioBrick &lt;partinfo&gt;P1010&lt;/partinfo&gt;, into different strains of <I>E. coli</I>. 
<TABLE cellpadding="10" style="border-collapse: collapse; border-width: 1px; border-style: solid; border-color: #000"><TBODY><TR><TH style="border-style: solid; border-width: 1px"><I>E. coli</I> strain
</TH><TH style="border-style: solid; border-width: 1px"> Resistant to <I>ccdB</I>?
</TH><TH style="border-style: solid; border-width: 1px"> Expected result?
</TH><TH style="border-style: solid; border-width: 1px"> Gyrase genotype  (<A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6T39-47PNXC3-F3&amp;_user=2459438&amp;_coverDate=01%2F28%2F1994&amp;_rdoc=1&amp;_fmt=high&amp;_orig=search&amp;_origin=search&amp;_sort=d&amp;_docanchor=&amp;view=c&amp;_acct=C000057302&amp;_version=1&amp;_urlVersion=0&amp;_userid=2459438&amp;md5=dfcdeab4c210c1f4ec70de318d013c15&amp;searchtype=a" class="external text" rel="nofollow">Metcalf <I>et al.</I>, 1994</A>; <A href="http://openwetware.org/wiki/E._coli_genotypes" class="external text" rel="nofollow">Openwetware</A>)
</TH></TR><TR><TD style="border-style: solid; border-width: 1px"> DB3.1
</TD><TD style="border-style: solid; border-width: 1px"> yes
</TD><TD style="border-style: solid; border-width: 1px"> yes
</TD><TD style="border-style: solid; border-width: 1px"> gyrA462
</TD></TR><TR><TD style="border-style: solid; border-width: 1px"> DH5Î±
</TD><TD style="border-style: solid; border-width: 1px"> yes
</TD><TD style="border-style: solid; border-width: 1px"> no
</TD><TD style="border-style: solid; border-width: 1px"> gyrA96
</TD></TR><TR><TD style="border-style: solid; border-width: 1px"> EC100D
</TD><TD style="border-style: solid; border-width: 1px"> no
</TD><TD style="border-style: solid; border-width: 1px"> yes
</TD><TD style="border-style: solid; border-width: 1px"> WT
</TD></TR><TR><TD style="border-style: solid; border-width: 1px"> JM109
</TD><TD style="border-style: solid; border-width: 1px"> yes
</TD><TD style="border-style: solid; border-width: 1px"> no
</TD><TD style="border-style: solid; border-width: 1px"> gyrA96
</TD></TR><TR><TD style="border-style: solid; border-width: 1px"> TOP10
</TD><TD style="border-style: solid; border-width: 1px"> no
</TD><TD style="border-style: solid; border-width: 1px"> yes
</TD><TD style="border-style: solid; border-width: 1px"> WT
</TD></TR><TR><TD style="border-style: solid; border-width: 1px"> XL1-Blue
</TD><TD style="border-style: solid; border-width: 1px"> yes
</TD><TD style="border-style: solid; border-width: 1px"> no
</TD><TD style="border-style: solid; border-width: 1px"> gyrA96
</TD></TR></TBODY></TABLE></CENTER><P>
It seems that not only the gyrase mutation gyrA462 is causing a <I>ccdB</I> resistance. Also the gyrase mutation gyrA96 gives <I>E. coli</I> a <I>ccdB</I> resistance. This should be kept in mind when assembling BioBricks with the 3A assembly.
</P><H1><SPAN class="mw-headline" id=".3Cpartinfo.3EK389004.3C.2Fpartinfo.3E:_Luciferase_from_pGL4.10.5Bluc2.5D">&lt;partinfo&gt;K389004&lt;/partinfo&gt;: Luciferase from pGL4.10[luc2]</SPAN></H1><P><A href="/Team:Bielefeld-Germany/Results/Characterization/K389004#mRFP_vs._luciferase_as_reporter_gene" title="Team:Bielefeld-Germany/Results/Characterization/K389004"> For a comparison between mRFP and luciferase as reporter genes click here. </A></P><P>Some important parameters determined by the characterization experiments are shown in tab. 2. For more information concerning these experiments click on the corresponding link in tab. 2 or click here: 
</P><DIV style="font-size:20px; text-align:center; font-weight:bold;"><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Characterization/K389004">Detailed information...</A></DIV><CENTER>Table 2: Parameters for &lt;partinfo&gt;K389004&lt;/partinfo&gt;. 
<TABLE cellpadding="10" style="border-collapse: collapse; border-width: 1px; border-style: solid; border-color: #000"><TBODY><TR><TH style="border-style: solid; border-width: 1px"> Experiment
</TH><TH style="border-style: solid; border-width: 1px"> Result
</TH></TR><TR><TD style="border-style: solid; border-width: 1px"><A href="/Team:Bielefeld-Germany/Results/Characterization/K389004#Accumulation_of_luciferase" title="Team:Bielefeld-Germany/Results/Characterization/K389004"> Behaviour during cultivation</A></TD><TD style="border-style: solid; border-width: 1px"><UL><LI> production is growth dependent
</LI><LI> degradation in stationary growth phase
</LI></UL></TD></TR><TR><TD style="border-style: solid; border-width: 1px"><A href="/Team:Bielefeld-Germany/Results/Characterization/K389004#Kinetic_of_luciferin_conversion" title="Team:Bielefeld-Germany/Results/Characterization/K389004"> Kinetic of luciferin conversion</A></TD><TD style="border-style: solid; border-width: 1px"> max. output between 20 - 40 s
</TD></TR><TR><TD style="border-style: solid; border-width: 1px"><A href="/Team:Bielefeld-Germany/Results/Characterization/K389004#Sensitivity" title="Team:Bielefeld-Germany/Results/Characterization/K389004"> Limit of detection (LOD)</A></TD><TD style="border-style: solid; border-width: 1px"> 162 RLU ~ 0.3Â % of &lt;partinfo&gt;J23103&lt;/partinfo&gt; output
</TD></TR><TR><TD style="border-style: solid; border-width: 1px"><A href="/Team:Bielefeld-Germany/Results/Characterization/K389004#Sensitivity" title="Team:Bielefeld-Germany/Results/Characterization/K389004"> Limit of quantification (LOQ)</A></TD><TD style="border-style: solid; border-width: 1px"> 306 RLU ~ 0.7Â % of &lt;partinfo&gt;J23103&lt;/partinfo&gt; output
</TD></TR></TBODY></TABLE></CENTER><H1><SPAN class="mw-headline" id=".3Cpartinfo.3EK389011.3C.2Fpartinfo.3E:_VirA_screening_device">&lt;partinfo&gt;K389011&lt;/partinfo&gt;: VirA screening device</SPAN></H1><DIV style="font-size:20px; text-align:center; font-weight:bold;"><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Characterization/K389011">Detailed information...</A></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Figure 1: Ratio of surviving colonies of <I>E. coli</I> EC100D carrying unmutated &lt;partinfo&gt;K389010&lt;/partinfo&gt; and &lt;partinfo&gt;K389011&lt;/partinfo&gt; plated on PA agar plates with chloramphenicol, ampicillin and different concentrations of kanamycin. Comparison between cells that were induced with acetosyringone with cells that were not induced.</DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id=".3Cpartinfo.3EK389015.3C.2Fpartinfo.3E:_VirA.2FG_reporter_device_with_luciferase">&lt;partinfo&gt;K389015&lt;/partinfo&gt;: VirA/G reporter device with luciferase</SPAN></H1><P>Some important parameters determined by the characterization experiments are shown in tab. 3. For more information concerning these experiments click on the corresponding link in tab. 3 or click here: 
</P><DIV style="font-size:20px; text-align:center; font-weight:bold;"><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Characterization/K389015">Detailed information...</A></DIV><CENTER>Table 3: Parameters for &lt;partinfo&gt;K389015&lt;/partinfo&gt;. 
<TABLE border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TH>Experiment
</TH><TH>Characteristic
</TH><TH>Value
</TH></TR><TR><TD rowspan="4"><A href="/Team:Bielefeld-Germany/Results/Characterization/K389015#Transfer_function" title="Team:Bielefeld-Germany/Results/Characterization/K389015"> Transfer Function</A></TD><TD>Maximum induction level
</TD><TD>2.2 fold
</TD></TR><TR><TD>Maximum induction level reached
</TD><TD>200 ÂµM acetosyringone
</TD></TR><TR><TD>Hill coefficient
</TD><TD>1.09
</TD></TR><TR><TD>Switch Point
</TD><TD>31.6 ÂµM acetosyringone
</TD></TR><TR><TD rowspan="3"><A href="/Team:Bielefeld-Germany/Results/Characterization/K389015#Growth_functions_and_Luciferase_expression_for_BBa_K389015" title="Team:Bielefeld-Germany/Results/Characterization/K389015"> Doubling time / h</A></TD><TD>without plasmid
</TD><TD>1.98
</TD></TR><TR><TD>carrying K389015
</TD><TD>2.24
</TD></TR><TR><TD>carrying K389015 with 400 ÂµM acetosyringone
</TD><TD>2.67
</TD></TR><TR><TD rowspan="2">Response time
</TD><TD>Induction: <A href="/Team:Bielefeld-Germany/Results/Characterization/K389015#Response_time" title="Team:Bielefeld-Germany/Results/Characterization/K389015"> exponential phase</A></TD><TD>&gt;1 h
</TD></TR><TR><TD>Induction: <A href="/Team:Bielefeld-Germany/Results/Characterization/K389015#Data_Analysis" title="Team:Bielefeld-Germany/Results/Characterization/K389015"> begin of cultivation</A></TD><TD>max. induction at OD<SUB>600</SUB> = 1 +/- 0.5
</TD></TR><TR><TD rowspan="3"><A href="/Team:Bielefeld-Germany/Results/Characterization/K389015#Plasmid_conformation_analysis" title="Team:Bielefeld-Germany/Results/Characterization/K389015"> Conformation analysis</A></TD><TD>ratio ccc monomer /Â %
</TD><TD>91
</TD></TR><TR><TD>ratio ccc dimer /Â %
</TD><TD>3.7
</TD></TR><TR><TD>ratio oc forms /Â %
</TD><TD>5.3
</TD></TR></TBODY></TABLE></CENTER><H1><SPAN class="mw-headline" id=".3Cpartinfo.3EK389016.3C.2Fpartinfo.3E:_VirA.2FG_reporter_device_with_mRFP">&lt;partinfo&gt;K389016&lt;/partinfo&gt;: VirA/G reporter device with mRFP</SPAN></H1><P>Protocols for <A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Protocols#Cultivation_for_measuring_mRFP_and_Luciferase_expression" class="external text" rel="nofollow">Cultivation</A> and <A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Protocols#Measuring_of_mRFP" class="external text" rel="nofollow">Measurement</A></P><P>Some important parameters determined by the characterization experiments are shown in tab. 4. For more information concerning these experiments click on the corresponding link in tab. 4 or click here: 
</P><DIV style="font-size:20px; text-align:center; font-weight:bold;"><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Characterization/K389016">Detailed information...</A></DIV><CENTER>Table 4: Parameters for &lt;partinfo&gt;K389016&lt;/partinfo&gt;. 
<TABLE border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TH>Experiment
</TH><TH>Characteristic
</TH><TH>Value
</TH></TR><TR><TD rowspan="4"><A href="/Team:Bielefeld-Germany/Results/Characterization/K389016#Transfer_function_of_BBa_K389016" title="Team:Bielefeld-Germany/Results/Characterization/K389016"> Transfer Function</A></TD><TD>Maximum induction level
</TD><TD>2.6 fold
</TD></TR><TR><TD>Maximum induction level reached
</TD><TD>150 ÂµM acetosyringone
</TD></TR><TR><TD>Hill coefficient
</TD><TD>1.67
</TD></TR><TR><TD>Switch Point
</TD><TD>26.5 ÂµM acetosyringone
</TD></TR><TR><TD rowspan="4"><A href="/Team:Bielefeld-Germany/Results/Characterization/K389016#Growth_functions_and_mRFP_expression_for_BBa_K389016" title="Team:Bielefeld-Germany/Results/Characterization/K389016"> Doubling time / h</A></TD><TD>without plasmid
</TD><TD>1.98
</TD></TR><TR><TD>carrying K389016
</TD><TD>2.57
</TD></TR><TR><TD>carrying K389016 with 150 ÂµM acetosyringone
</TD><TD>2.77
</TD></TR><TR><TD>carrying K389016 with 1000 ÂµM acetosyringone
</TD><TD>3.01
</TD></TR><TR><TD rowspan="3"><A href="/Team:Bielefeld-Germany/Results/Characterization/K389016#Plasmid_conformation_analysis" title="Team:Bielefeld-Germany/Results/Characterization/K389016"> Conformation analysis</A></TD><TD>ratio ccc monomer /Â %
</TD><TD>91.2
</TD></TR><TR><TD>ratio ccc dimer /Â %
</TD><TD>3.2
</TD></TR><TR><TD>ratio oc forms /Â %
</TD><TD>5.6
</TD></TR><TR><TD rowspan="5"><A href="/Team:Bielefeld-Germany/Results/Characterization/K389016#Different_possible_inducers" title="Team:Bielefeld-Germany/Results/Characterization/K389016"> Inducers</A></TD><TD>Induction by
</TD><TD>Acetosyringone
</TD></TR><TR><TD rowspan="4">No Induction by
</TD><TD>Capsaicin
</TD></TR><TR><TD>Dopamine
</TD></TR><TR><TD>Homovanillic acid
</TD></TR><TR><TD>3-Methoxytyramine
</TD></TR></TBODY></TABLE></CENTER><H1><SPAN class="mw-headline" id=".3Cpartinfo.3EK389052.3C.2Fpartinfo.3E:_Tightly_regulated_lac_operon_with_mRFP_readout">&lt;partinfo&gt;K389052&lt;/partinfo&gt;: Tightly regulated <I>lac</I> operon with mRFP readout</SPAN></H1><P>This construct was plated for plasmid isolation in a <I>lacI<SUP>q</SUP></I> negative <I>E. coli</I> strain after assembly - and we have never seen such red plates when working with constructs with mRFP downstream of a promoter. This <I>lac</I> operon definitely shows a very high basal transcription, so it is not tightly repressed. It seems that the <I>lacI</I> repressor &lt;partinfo&gt;BBa_C0012&lt;/partinfo&gt; is not suitable for this purpose due to its LVA degradation tag or it does not work properly. Another indicator for this assumption is the experience page of &lt;partinfo&gt;C0012&lt;/partinfo&gt;.
</P><H1><SPAN class="mw-headline" id=".3Cpartinfo.3EK389421.3C.2Fpartinfo.3E.2C_.3Cpartinfo.3EK389422.3C.2Fpartinfo.3E.2C_.3Cpartinfo.3EK389423.3C.2Fpartinfo.3E:_Sensitivity_Tuner_amplified_Vir-test_system">&lt;partinfo&gt;K389421&lt;/partinfo&gt;, &lt;partinfo&gt;K389422&lt;/partinfo&gt;, &lt;partinfo&gt;K389423&lt;/partinfo&gt;: Sensitivity Tuner amplified Vir-test system</SPAN></H1><DIV style="font-size:20px; text-align:center; font-weight:bold;"><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Characterization/Sen_Tuner">Detailed information...</A></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 2: Amplification factor of induced, 50 ÂµM Acetosyringone (red) and not induced (green) modified Sensitivity Tuner &lt;partinfo&gt;K389421&lt;/partinfo&gt;, &lt;partinfo&gt;K389422&lt;/partinfo&gt; and &lt;partinfo&gt;K389423&lt;/partinfo&gt;, Standard deviation shown.</B></DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 3: Vizualitation of induced (from left to right) &lt;partinfo&gt;K389421&lt;/partinfo&gt;, &lt;partinfo&gt;K389422&lt;/partinfo&gt; and &lt;partinfo&gt;K389423&lt;/partinfo&gt; sensitivity tuner amplified vir-system.</B></DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Mutated_.3Cpartinfo.3EK389001.3C.2Fpartinfo.3E_.28exemplary.29:_Results_of_the_directed_mutagenesis">Mutated &lt;partinfo&gt;K389001&lt;/partinfo&gt; (exemplary): Results of the directed mutagenesis</SPAN></H1><DIV style="font-size:20px; text-align:center; font-weight:bold;"><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Characterization/Exemplary_results_DM">Detailed information...</A></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 4: Luciferase production rates of the exemplary clone â21â are shown under conditions of induction with acetosyringone (21-A), capsaicin (21-C) and in uninduced state (21-U). The right bar indicates the production rate of the native system without acetosyringone (212-U), all the values in the figure had been normalized to.</B></DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><UL><LI>Behrens B, Eppendorf AG, Laborpraxis, Nr.20, Reinste Plasmid-DNA in nur 9 Minuten.
</LI></UL><UL><LI><A href="http://openwetware.org/wiki/CcdB" class="external free" rel="nofollow">http://openwetware.org/wiki/CcdB</A>, CcdB (seen on 10.10.10).
</LI></UL><UL><LI><A href="http://openwetware.org/wiki/E._coli_genotypes" class="external free" rel="nofollow">http://openwetware.org/wiki/E._coli_genotypes</A>, E. coli genotypes (seen on 10.10.10).
</LI></UL><UL><LI>Metcalf, WW <I>et al.</I> (1994) <A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6T39-47PNXC3-F3&amp;_user=2459438&amp;_coverDate=01%2F28%2F1994&amp;_rdoc=1&amp;_fmt=high&amp;_orig=search&amp;_origin=search&amp;_sort=d&amp;_docanchor=&amp;view=c&amp;_acct=C000057302&amp;_version=1&amp;_urlVersion=0&amp;_userid=2459438&amp;md5=dfcdeab4c210c1f4ec70de318d013c15&amp;searchtype=a" class="external text" rel="nofollow"><I>Use of the rep technique for allele replacement to construct new Escherichia coli hosts for maintenance of R6KÎ» origin plasmids at different copy numbers</I></A>, Gene 138(1):1-7.
</LI></UL><UL><LI>Stadler J, Lemmens R, Nyhammar T 2004, <I>Plasmid DNA purification</I>, The J. of Gene Medicine,Vol.6, pp.54âS66
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Bielefeld-Germany/Results/Characterization">http://2010.igem.org/Team:Bielefeld-Germany/Results/Characterization</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Results/Characterization" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Results/Characterization" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/Characterization&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/Characterization&amp;oldid=203455" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>