<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Project_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Project/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Project/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project/Protocols&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Project/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Project/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="sliderWrap"><DIV id="slider"><DIV id="openCloseWrap" style="height:110px;"><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany">Start</A></LI></DIV><DIV class="navColumn" style="width:13%;"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team">Team</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Description">Description</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Supervisors">Supervisors</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Bielefeld">University</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/BielefeldCity">Bielefeld</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Motivation">Motivation</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Acknowledgements">Acknowledgements</A></LI></UL></DIV><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project">Project</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Application">Application</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Approach">Approach</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Theory">Theory</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Protocols">Protocols</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Model">Model</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Outlook">Outlook</A></LI></UL></DIV><DIV class="navColumn" style="width:12%;"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results">Results</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Characterization">Characterization</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Used">Used BBs</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Submitted">Submitted BBs</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Sequencing">Sequencing</A></LI></UL></DIV><DIV class="navColumn" style="width:12%;"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Safety#Biosecurity">Biosecurity</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Safety#Biosafety">Biosafety</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Safety#Risk_Assesement_.28english.29">Risk Assesement</A></LI></UL></DIV><DIV class="navColumn" style="width:13%;"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Public_Relations">Public Relations</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Public_Relations/Fotos">Fotos</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Public_Relations/Press">Press</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Public_Relations/Public_Discussion">Public Discussion</A></LI></UL></DIV><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Judging">Judging</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Judging/BioBricks">BioBricks</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Judging/Helping">Helping other teams</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Judging/Comments">Comments on iGEM</A></LI></UL></DIV><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Sponsors">Sponsors</A></LI></DIV><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Contact">Contact</A></LI></DIV></DIV></DIV><DIV id="sliderController" class="sliderController">MENU</DIV></DIV><DIV id="checkParameter" style="margin-right:250px;">http://igem-bielefeld.de/img/banner_chili.jpg</DIV><DIV id="subnavigation"><LI><A href="/Team:Bielefeld-Germany/Project">Project</A></LI><LI><A href="/Team:Bielefeld-Germany/Project/Application">Application</A></LI><LI><A href="/Team:Bielefeld-Germany/Project/Approach">Approach</A></LI><LI><A href="/Team:Bielefeld-Germany/Project/Theory">Theory</A></LI><LI><A href="/Team:Bielefeld-Germany/Project/Protocols">Protocols</A></LI><LI><A href="/Team:Bielefeld-Germany/Project/Model">Model</A></LI><LI><A href="/Team:Bielefeld-Germany/Project/Outlook">Outlook</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Lab_protocols"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Lab protocols</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Preparation_of_electrocompetent_E._coli_cells"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Preparation of electrocompetent E. coli cells</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Preparation_of_heat_shock_competent_E.coli_cells"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Preparation of heat shock competent E.coli cells</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Transformation_via_electroporation"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Transformation via electroporation</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Heat_shock_transformation"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Heat shock transformation</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Silver_BioBrick_Assembly"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Silver BioBrick Assembly</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Suffix_Insertion"><SPAN class="tocnumber">1.5.1</SPAN><SPAN class="toctext">Suffix Insertion</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Prefix_Insertion"><SPAN class="tocnumber">1.5.2</SPAN><SPAN class="toctext">Prefix Insertion</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Variations"><SPAN class="tocnumber">1.5.3</SPAN><SPAN class="toctext">Variations</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-10"><A href="#3A_assembly"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">3A assembly</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Digestion"><SPAN class="tocnumber">1.6.1</SPAN><SPAN class="toctext">Digestion</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Ligation"><SPAN class="tocnumber">1.6.2</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Phusion_PCR"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Phusion PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Colony_PCR"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Construction_of_a_plasmid_with_R6K_origin_of_replication_.28ori.29"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">Construction of a plasmid with R6K origin of replication (ori)</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Determination_of_minimal_inhibitory_concentration_.28MIC.29_of_kanamycin"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">Determination of minimal inhibitory concentration (MIC) of kanamycin</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Directed_mutagenesis_.E2.80.93_Error-prone_PCR_and_screening_system"><SPAN class="tocnumber">1.11</SPAN><SPAN class="toctext">Directed mutagenesis – Error-prone PCR and screening system</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#1._Error-prone_PCR_Step"><SPAN class="tocnumber">1.11.1</SPAN><SPAN class="toctext">1. Error-prone PCR Step</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#2._Cloning_of_virA_variants"><SPAN class="tocnumber">1.11.2</SPAN><SPAN class="toctext">2. Cloning of virA variants</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#3._Primary_selection_of_novel_virA_variants"><SPAN class="tocnumber">1.11.3</SPAN><SPAN class="toctext">3. Primary selection of novel virA variants</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#4._Quantitative_analysis_of_virA_variants_after_induction_with_novel_substances"><SPAN class="tocnumber">1.11.4</SPAN><SPAN class="toctext">4. Quantitative analysis of virA variants after induction with novel substances</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#Restriction_analysis"><SPAN class="tocnumber">1.12</SPAN><SPAN class="toctext">Restriction analysis</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Cultivation_for_measuring_mRFP_and_Luciferase_expression"><SPAN class="tocnumber">1.13</SPAN><SPAN class="toctext">Cultivation for measuring mRFP and Luciferase expression</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Measuring_of_mRFP"><SPAN class="tocnumber">1.14</SPAN><SPAN class="toctext">Measuring of mRFP</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Measuring_of_Luciferase"><SPAN class="tocnumber">1.15</SPAN><SPAN class="toctext">Measuring of Luciferase</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#Chemicals.2C_material_etc."><SPAN class="tocnumber">1.16</SPAN><SPAN class="toctext">Chemicals, material etc.</SPAN></A></LI><LI class="toclevel-3 tocsection-27"><A href="#Enzymes"><SPAN class="tocnumber">1.16.1</SPAN><SPAN class="toctext">Enzymes</SPAN></A></LI><LI class="toclevel-3 tocsection-28"><A href="#Kits"><SPAN class="tocnumber">1.16.2</SPAN><SPAN class="toctext">Kits</SPAN></A></LI><LI class="toclevel-3 tocsection-29"><A href="#Inducer_Stock_Solutions"><SPAN class="tocnumber">1.16.3</SPAN><SPAN class="toctext">Inducer Stock Solutions</SPAN></A></LI><LI class="toclevel-3 tocsection-30"><A href="#TAE_buffer"><SPAN class="tocnumber">1.16.4</SPAN><SPAN class="toctext">TAE buffer</SPAN></A></LI><LI class="toclevel-3 tocsection-31"><A href="#DNA_loading_buffer"><SPAN class="tocnumber">1.16.5</SPAN><SPAN class="toctext">DNA loading buffer</SPAN></A></LI><LI class="toclevel-3 tocsection-32"><A href="#LB_medium"><SPAN class="tocnumber">1.16.6</SPAN><SPAN class="toctext">LB medium</SPAN></A></LI><LI class="toclevel-3 tocsection-33"><A href="#Cell_Culture_Lysis_Reagent"><SPAN class="tocnumber">1.16.7</SPAN><SPAN class="toctext">Cell Culture Lysis Reagent</SPAN></A></LI><LI class="toclevel-1 tocsection-34"><A href="#References"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Lab_protocols"> Lab protocols </SPAN></H1><H2><SPAN class="mw-headline" id="Preparation_of_electrocompetent_E._coli_cells"> Preparation of electrocompetent <I>E. coli</I> cells </SPAN></H2><P>modified from <A href="http://openwetware.org/wiki/Quint_Lab:electrocompetent_cells_e.coli" class="external text" rel="nofollow">Quint Lab</A></P><P>
Materials: 
</P><UL><LI>	10 mL LB-Medium (1 % Bacto Trypton; 0.5 % Yeast Extract; 0.5 % NaCl)  
</LI><LI>	2 L cooled bidest H<SUB>2</SUB>O 
</LI><LI>	200 mL cooled, sterile-filtered 10 % glycerol 
</LI><LI>	Box with ice-water for 2-litre-flask 
</LI><LI>	4 pre-cooled 250 mL (or 2x500 mL) bins for centrifugation 
</LI><LI>	2 pre-cooled 50 mL Falcons 
</LI><LI>	Centrifuge, pre-cooled to 2 °C (max. 4 °C) 
</LI></UL><P>
Protocol
</P><UL><LI>	Inoculate 2x10 mL LB with bacterial stock; incubate over night at 37 °C and 200 rpm 
</LI><LI>	Inoculate 2x250 mL LB in 1-litre-flask with OD<SUB>600</SUB>=0.1 at 37 °C or at 19°C over night
</LI><LI>	Incubate until OD<SUB>600</SUB> 0.4-0.6 (~5 h)
</LI><LI>	The following steps are done at 2-4 °C (best in a cold room) 
</LI></UL><UL><LI>	Cool 1L-culture 10-15 minutes in ice water (shake sometimes) 
</LI><LI>	Divide culture into 2x5 cooled 50 mL Falcon-tubes for centrifugation 
</LI><LI>	Centrifuge 10 min at 4°C with 4000 rcf 
</LI><LI>	Discard supernatant 
</LI><LI>	Resuspend pellet in 5 mL bidest. H<SUB>2</SUB>O 
</LI><LI>	Add bidest. H<SUB>2</SUB>O up to 50 mL 
</LI><LI>	Centrifuge 10 min at 4°C with 4000 rcf 
</LI><LI>	Discard supernatant immediately 
</LI><LI>	Resuspend pellet in residual supernatant 
</LI><LI>	Add bidest H<SUB>2</SUB>O up to 50 mL 
</LI><LI>	Centrifuge 10 min at 4°C with 4000 rcf 
</LI><LI>	Discard supernatant immediately 
</LI><LI>	Resuspend pellet in residual supernatant 
</LI><LI>	Transfer suspension in 2x50 mL Falcons 
</LI><LI>	Add 10% glycerol up to 50 mL 
</LI><LI>	Centrifuge 5 min at 4°C with 4000 rcf 
</LI><LI>	Discard supernatant 
</LI><LI>	Estimate volume of the pellet; fill up with equal volume of 10% glycerol 
</LI><LI>	Resuspend pellet on ice; ‘’’don´t vortex!!’’’ (just shake cautiously) 
</LI><LI>	Divide cells into 150 µL aliquots (use 1.5 mL Eppis) 
</LI><LI>	Freeze in liquid N<SUB>2</SUB> or dry-ice 
</LI><LI>	Store at -80°C
</LI></UL><H2><SPAN class="mw-headline" id="Preparation_of_heat_shock_competent_E.coli_cells"> Preparation of heat shock competent <I>E.coli</I> cells </SPAN></H2><UL><LI>	Inoculate 2x5 mL LB with bacterial stock; incubate over night at 37°C and 200 rpm 
</LI><LI>	Inoculate 2x250 mL SOB in 1-litre-flasks with OD<SUB>600</SUB>=0.1
</LI><LI>	Incubate at 19°C until OD<SUB>600</SUB>=0.4-0.6 (20h)
</LI><LI>	The following steps are done at 2-4°C (best in a cold room) 
</LI></UL><UL><LI>	Cool 1 L of culture 10-15 minutes in ice water (shake sometimes) 
</LI><LI>	Divide culture into 2x5 cooled 50 mL Falcon-tubes for centrifugation 
</LI><LI>	Centrifuge 10 min at 4°C with 4000 rcf 
</LI><LI>	Discard supernatant
</LI><LI>       Resuspend cells in 80 mL TB buffer
</LI><LI>       Cool cells for 10 min on ice
</LI><LI>	Centrifuge 10 min at 4°C with 2500 rcf
</LI><LI>	Discard supernatant
</LI><LI>       Resuspend cells in 20 mL TB buffer
</LI><LI>       Add 1.5 mL Dimethyl sulfoxide (DMSO) for each 20 mL cells (DMSO concentration = 7%) 
</LI><LI>       Cool cells for 10 min on ice
</LI><LI>	Divide cells into 150 µL Alicuots (use 1.5 mL Eppis)
</LI><LI>	Freeze in liquid N<SUB>2</SUB> or dry-ice 
</LI><LI>	Store at -80°C
</LI></UL><H2><SPAN class="mw-headline" id="Transformation_via_electroporation"> Transformation via electroporation </SPAN></H2><UL><LI> Thaw 50 µL competent <I>E.coli</I> cells on ice, dilute with icecold 50 µL glycerol (10 %) if necessary
</LI><LI> Add 0.5-5 µL plasmid to 50 µl electrocompetent cells
</LI><LI> Store cells on ice for 1 minute 
</LI><LI> Electroporate at U = 2.5 kV, C = 25 µF, R = 200 Ώ 
</LI><LI> Transfer transformation reaction to 450 µL SOC-Medium and shake 1 h at 37°C 
</LI><LI> Centrifuge 2 min at 800 rpm and plate on selective LB-Medium
</LI></UL><H2><SPAN class="mw-headline" id="Heat_shock_transformation"> Heat shock transformation </SPAN></H2><UL><LI> Thaw 150 µL competent <I>E. coli</I> cells on ice
</LI></UL><UL><LI> Add max. 10 µL DNA (the less the better your transformation works but at least about 50 ng vector)
</LI></UL><UL><LI> Incubate 30 min on ice
</LI></UL><UL><LI> Heatshock: 42 °C, 45 s (water bath because of quick heat transfer)
</LI></UL><UL><LI> 1 min on ice
</LI></UL><UL><LI> Add 1 mL prewarmed SOC-medium
</LI></UL><UL><LI> Incubate: 45 - 60 min, 37 °C
</LI></UL><UL><LI> Plate 100 µL
</LI></UL><UL><LI> Spin down the remaining cells (2 min, 5000 g), discard most of the supernatant, resuspend the cells in the remaining medium and plate them
</LI></UL><H2><SPAN class="mw-headline" id="Silver_BioBrick_Assembly"> Silver BioBrick Assembly </SPAN></H2><P>modified from <A href="http://openwetware.org/wiki/Silver:_BB_Strategy" class="external text" rel="nofollow">Silver lab</A>:
</P><P>This assembly method can be used for BioBricks which are bigger than 150 bp. The BioBrick should be at least 500 bp bigger or smaller than the backbone. The BioBrick, which complies with these conditions, is used as the insert and is assembled into the prefix or suffix of the other used BioBrick, called vector. So you have to differentiate between a prefix and a suffix insertion.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Silver Suffix Insertion</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Silver Prefix Insertion</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Suffix_Insertion"> Suffix Insertion </SPAN></H3><UL><LI> Digestion of insert: at least 700 ng DNA / 10 µL volume, 1 µL 10x Tango buffer, 0.5 µL XbaI, 1 µL PstI. Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert. 
</LI></UL><UL><LI> Digestion of vector about 700 ng DNA / 10 µL volume, 1 µL 10x orange buffer, 0.5 µL SpeI, 0.5 µL PstI. Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µL SAP (shrimp alcaline phosphatase) and 1.2 µL 10 x SAP buffer, incubate for 1 h at 37 °C. Clean up the vector with a PCR clean-up kit. 
</LI></UL><UL><LI> Ligation: after digestion and clean-up: 50 - 200 ng of vector, 3 - 10 fold molar access of insert, 20 µL ligation volume, 2 µL T4-Ligase-Buffer, 1 µL T4-Ligase. Incubate for 1 h at 37 °C, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.
</LI></UL><H3><SPAN class="mw-headline" id="Prefix_Insertion"> Prefix Insertion </SPAN></H3><UL><LI> Digestion of insert: at least 700 ng DNA / 10 µL volume, 1 µL 10x BamHI buffer, 0.5 µL EcoRI, 0.5 µL SpeI. Digest for 2 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert. 
</LI></UL><UL><LI> Digestion of vector about 700 ng DNA / 10 µL volume, 1 µL 10 x Tango buffer, 0.5 µL EcoRI, 0.5 µL XbaI. Digest for 2h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µL SAP (shrimp alcaline phosphatase) and 1.2 µL 10 x SAP buffer, incubate for 1 h at 37 °C. Clean up the vector with a PCR clean-up kit. 
</LI></UL><UL><LI> Ligation: after digestion and clean-up: 50 - 200 ng of vector, 3 - 10 fold molar access of insert, 20 µL ligation volume, 2 µL T4-Ligase-Buffer, 1 µL T4-Ligase. Incubate for 1 h at 37 °C, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.
</LI></UL><H3><SPAN class="mw-headline" id="Variations"> Variations </SPAN></H3><UL><LI> A digestion over night is possible. If you digest over night use only 0.1 µL restriction enzyme. 
</LI></UL><UL><LI> It is also possible to use PCR product as insert. Digest after PCR with corresponding restriction enzymes and clean up with PCR clean-up kit. This could lead to higher yields of insert DNA because a lot of DNA gets lost during the gel electrophoresis clean up. 
</LI></UL><UL><LI> Sometimes some BioBricks are hard to assemble. Then you have to clean up the vector by gel electrophoresis as well.
</LI></UL><H2><SPAN class="mw-headline" id="3A_assembly">3A assembly</SPAN></H2><P>Modified from <A href="http://ginkgobioworks.com/support/BioBrick_Assembly_Manual.pdf" class="external text" rel="nofollow">BioBrick Assembly Manual by Ginkgo BioWorks</A></P><H3><SPAN class="mw-headline" id="Digestion">Digestion</SPAN></H3><UL><LI> Thaw DNA from upstream and downstream part and the destination plasmid on ice. 
</LI></UL><UL><LI>  Destination plasmid has to carry the <I>ccdB</I> gene &lt;partinfo&gt;P1010&lt;/partinfo&gt; as insert and has to have a different antibiotic resistance than the plasmids carrying the upstream and downstream parts
</LI></UL><UL><LI>  DNA has to be cleaned (by MiniPrep or after a PCR)
</LI></UL><UL><LI> 500 ng DNA / digestion mix for upstream and downstream part, 150 ng DNA / digestion mix for destination plasmid (total volume of mix 10 µL, dilute with ddH<SUB>2</SUB>0 if necessary)
</LI></UL><UL><LI> Add 1 µL of buffer and restriction enzymes as shown in the following table: 
</LI></UL><CENTER><TABLE cellpadding="10" style="border-collapse: collapse; border-width: 1px; border-style: solid; border-color: #000"><TBODY><TR><TH style="border-style: solid; border-width: 1px"> Upstream part
</TH><TH style="border-style: solid; border-width: 1px"> Downstream part
</TH><TH style="border-style: solid; border-width: 1px"> Destination plasmid
</TH></TR><TR><TD style="border-style: solid; border-width: 1px"> enzyme 1
</TD><TD style="border-style: solid; border-width: 1px"> 1 µL EcoRI
</TD><TD style="border-style: solid; border-width: 1px"> 0.5 µL XbaI
</TD><TD style="border-style: solid; border-width: 1px"> 0.5 µL EcoRI
</TD></TR><TR><TD style="border-style: solid; border-width: 1px"> enzyme 2
</TD><TD style="border-style: solid; border-width: 1px"> 1 µL SpeI
</TD><TD style="border-style: solid; border-width: 1px"> 1 µL PstI
</TD><TD style="border-style: solid; border-width: 1px"> 0.5 µL PstI
</TD></TR><TR><TD style="border-style: solid; border-width: 1px"> buffer
</TD><TD style="border-style: solid; border-width: 1px"> BamHI
</TD><TD style="border-style: solid; border-width: 1px"> Tango
</TD><TD style="border-style: solid; border-width: 1px"> Orange
</TD></TR></TBODY></TABLE></CENTER><UL><LI> Incubation of the digestion mixes for 2 h at 37 °C, afterwards heat inactivation for 20 min at 80 °C
</LI></UL><UL><LI> Continue with ligation or freeze the mixes
</LI></UL><H3><SPAN class="mw-headline" id="Ligation">Ligation</SPAN></H3><UL><LI> Ligation mix: 
</LI></UL><UL><LI> 2 µL ddH<SUB>2</SUB>O
</LI></UL><UL><LI> 5 µL of every digestion mix (so 15 µL in total)
</LI></UL><UL><LI> 2 µL T4-DNA-ligase buffer (thaw on ice!)
</LI></UL><UL><LI> 1 µL T4-DNA-ligase
</LI></UL><UL><LI> Incubate 1 h at 37 °C, afterwards heat inactivation for 5 min at 70 °C
</LI></UL><UL><LI> Freeze ligation mix or continue with transformation (heatshock or electroporation)
</LI></UL><H2><SPAN class="mw-headline" id="Phusion_PCR"> Phusion PCR </SPAN></H2><UL><LI> Perform a Phusion PCR for amplifying BioBricks or gaining new BioBricks
</LI></UL><UL><LI> For one PCR-Reaction mix:
</LI><LI> 10 µL 5x HF-Buffer
</LI><LI> 4 µL dNTPs (2.5 mM each)
</LI><LI> 1.5 µL DMSO
</LI><LI> 31.5 µL H<SUB>2</SUB>O
</LI><LI> 1 µL Primer Mix
</LI><LI> 1 µL Template
</LI><LI> 1 µL Phusion DNA-Polymerase
</LI></UL><UL><LI> PCR program: 
</LI><LI> Start: 30 sec, 98 °C
</LI><LI> 35 cycles of: 
</LI><LI> 10 sec, 98 °C
</LI><LI> 30 sec, 58-66 °C (depending on primers)
</LI><LI> 15-30 sec / 1 kb template, 72 °C
</LI></UL><LI> 10 min, 72 °C
</LI><LI> 4 °C until further processing
</LI><UL><LI> Perform a gel electrophoresis:
</LI><LI> Check the fragment size
</LI><LI> Depending whether there are fragments with wrong size perform a clean up via gel electrophoresis or just with a PCR clean-up kit
</LI></UL><H2><SPAN class="mw-headline" id="Colony_PCR"> Colony PCR </SPAN></H2><UL><LI> Pick one colony with a sterile tip and elute it in 100 µL ddH<SUB>2</SUB>0 or medium
</LI></UL><UL><LI> Store the colony in 4°C while colony PCR is running
</LI></UL><UL><LI> One reaction mix contains: 
</LI><LI> 2.5 µL 10x buffer
</LI><LI> 0.75 µL MgCl<SUB>2</SUB></LI><LI> 1 µL dNTPs
</LI><LI> 0.5 µL primer mix (prefix/suffix primers or sequencing primers)
</LI><LI> 19.25 µL ddH<SUB>2</SUB>O
</LI><LI> 0.5 µL <I>taq</I>-polymerase
</LI><LI> 0.5 µL template
</LI></UL><UL><LI> PCR program: 
</LI><LI> Start: 8 min, 98 °C
</LI><LI> 30 cycles of: 
</LI><LI> 30 s, 98 °C
</LI><LI> 30 s, 60 °C
</LI><LI> 30 s / 1 kb template, 72 °C
</LI></UL><LI> Finish: 5 min, 72 °C
</LI><UL><LI> Gel electrophoresis: check the fragment size
</LI></UL><UL><LI> Plate the correct colony
</LI></UL><H2><SPAN class="mw-headline" id="Construction_of_a_plasmid_with_R6K_origin_of_replication_.28ori.29">Construction of a plasmid with R6K origin of replication (ori)</SPAN></H2><P><B>Cloning of R6K origin into plasmid pSB1C3</B></P><UL><LI>	Perform a restriction digest of pSB1C3 and pSB1A2::&lt;partinfo&gt;J61001&lt;/partinfo&gt; with EcoRI and PstI 
</LI><LI>	Dephosphorylate restricted pSB1C3
</LI><LI>	Separate BBa_J61001 and pSB1A2 backbone by agarose gel electrophoresis and extract the ~400 bp fragment (BBa_J61001)
</LI><LI>	Purify restricted fragments with gel extraction kit </LI><LI>	Ligate restricted and purified pSB1C3 and BBa_J61001
</LI><LI>	Transform ligated product into heatshock competent <I>E. coli</I> TOP 10 cells 
</LI><LI>	Streak out on selective LB agar plates containing chloramphenicol (10 µg µL<SUP>-1</SUP>)
</LI><LI>	Incubate overnight at 37°C
</LI></UL><P><B>Removal of pMB1/ColE1 origin of replication</B></P><UL><LI>	Streak out single colonies on selective LB agar plates containing chloramphenicol (10 µg µL<SUP>-1</SUP>) 
</LI><LI>	Incubate overnight at 37°C
</LI><LI>	Isolate plasmid DNA
</LI><LI>	Perform a restriction digest of pSB1C3::BBa_J61001 with Hin6I
</LI><LI>	Separate fragments by agarose gel electrophoresis
</LI><LI>	Extract and purify largest fragment (1758 bp) with gel extraction kit
</LI><LI>	Religate extracted fragment
</LI><LI>	Transform ligation product into heatshock competent <I>E. coli</I> EC100D cells
</LI></UL><P>Comment: The resulting plasmid contains 32 remaining basepairs of the pMB1/ColE1 origin. This fragment does NOT enable replication in <I>E. coli</I> pir<SUP>-</SUP> strains.
</P><H2><SPAN class="mw-headline" id="Determination_of_minimal_inhibitory_concentration_.28MIC.29_of_kanamycin">Determination of minimal inhibitory concentration (MIC) of kanamycin</SPAN></H2><UL><LI> Transform the plasmid K389212 (native <I>virA</I>) in bacteria which are already containing K389101 (kanamycin resistance read out) using <A href="#Transformation_via_electroporation">electroporation</A>.
</LI><LI> Plate 100 µL of the transformed bacteria in different steps of dilution (10<SUP>-1</SUP> to 10<SUP>-6</SUP>) on LB-Agar with ampicillin (100 µg mL<SUP>-1</SUP>) and chloramphenicol (10 µg mL<SUP>-1</SUP>).
</LI><LI> Incubate the plates upside down at 37 °C for 16 – 20 h.
</LI><LI> Compare the plates of different dilution steps and select those, showing a high but countable number of colonies (aprox. 100 – 500).
</LI></UL><P>
In the following steps the colonies are transferred to LB-Agar with rising concentrations of kanamycin either in the presence of acetosyringone or without any inductor.
</P><UL><LI> Transfer the colonies of the selected plates to LB-Agars described below using replica plating. Take care to operate from lowest up to highest concentration of kanamycin. You must also use a fresh and sterile stamp every time you start a new sequence of plating.
</LI></UL><UL><LI> Composition of the LB-Agars (plating sequence from top to bottom)
</LI></UL><UL><LI> A) Uninduced status (no acetosyringone added)
</LI></UL><CENTER><TABLE cellpadding="10" style="border-collapse: collapse; border-width: 0.5px; border-style: solid; border-color: #000"><TBODY><TR><TD style="border-style: solid; border-width: 0.5px"> Positive control
</TD><TD style="border-style: solid; border-width: 0.5px"> Ampicillin (100 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Chloramphenicol (10 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Acetosyringone (<B>0 µM</B>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Kanamycin (<B>0 µg mL<SUP>-1</SUP></B>)
</TD></TR><TR><TD style="border-style: solid; border-width: 0.5px"> Ampicillin (100 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Chloramphenicol (10 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Acetosyringone (<B>0 µM</B>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Kanamycin (<B>10 µg mL<SUP>-1</SUP></B>)
</TD></TR><TR><TD style="border-style: solid; border-width: 0.5px"> Ampicillin (100 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Chloramphenicol (10 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Acetosyringone (<B>0 µM</B>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Kanamycin (<B>25 µg mL<SUP>-1</SUP></B>)
</TD></TR><TR><TD style="border-style: solid; border-width: 0.5px"> Ampicillin (100 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Chloramphenicol (10 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Acetosyringone (<B>0 µM</B>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Kanamycin (<B>50 µg mL<SUP>-1</SUP></B>)
</TD></TR><TR><TD style="border-style: solid; border-width: 0.5px"> Ampicillin (100 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Chloramphenicol (10 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Acetosyringone (<B>0 µM</B>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Kanamycin (<B>100 µg mL<SUP>-1</SUP></B>)
</TD></TR><TR><TD style="border-style: solid; border-width: 0.5px"> Ampicillin (100 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Chloramphenicol (10 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Acetosyringone (<B>0 µM</B>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Kanamycin (<B>200 µg mL<SUP>-1</SUP></B>)
</TD></TR></TBODY></TABLE></CENTER><UL><LI> B) Induced status (200 µM acetosyringone)
</LI></UL><CENTER><TABLE cellpadding="10" style="border-collapse: collapse; border-width: 0.5px; border-style: solid; border-color: #000"><TBODY><TR><TD style="border-style: solid; border-width: 0.5px"> Positive Control
</TD><TD style="border-style: solid; border-width: 0.5px"> Ampicillin (100 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Chloramphenicol (10 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Acetosyringone (<B>200 µM</B>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Kanamycin (<B>0 µg mL<SUP>-1</SUP></B>)
</TD></TR><TR><TD style="border-style: solid; border-width: 0.5px"> Ampicillin (100 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Chloramphenicol (10 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Acetosyringone (<B>200 µM</B>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Kanamycin (<B>10 µg mL<SUP>-1</SUP></B>)
</TD></TR><TR><TD style="border-style: solid; border-width: 0.5px"> Ampicillin (100 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Chloramphenicol (10 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Acetosyringone (<B>200 µM</B>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Kanamycin (<B>25 µg mL<SUP>-1</SUP></B>)
</TD></TR><TR><TD style="border-style: solid; border-width: 0.5px"> Ampicillin (100 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Chloramphenicol (10 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Acetosyringone (<B>200 µM</B>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Kanamycin (<B>50 µg mL<SUP>-1</SUP></B>)
</TD></TR><TR><TD style="border-style: solid; border-width: 0.5px"> Ampicillin (100 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Chloramphenicol (10 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Acetosyringone (<B>200 µM</B>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Kanamycin (<B>100 µg mL<SUP>-1</SUP></B>)
</TD></TR><TR><TD style="border-style: solid; border-width: 0.5px"> Ampicillin (100 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Chloramphenicol (10 µg mL<SUP>-1</SUP>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Acetosyringone (<B>200 µM</B>)
</TD><TD style="border-style: solid; border-width: 0.5px"> Kanamycin (<B>200 µg mL<SUP>-1</SUP></B>)
</TD></TR></TBODY></TABLE></CENTER><UL><LI> Incubate the plates upside down at 37 °C for 16 – 20 h.
</LI><LI> Count the number of colonies on each plate.
</LI><LI> Calculate the ration of colonies grown on each plate, compared to the number of colonies on the first plate of each sequence (positive control).
</LI></UL><H2><SPAN class="mw-headline" id="Directed_mutagenesis_.E2.80.93_Error-prone_PCR_and_screening_system">Directed mutagenesis – Error-prone PCR and screening system</SPAN></H2><H3><SPAN class="mw-headline" id="1._Error-prone_PCR_Step">1. Error-prone PCR Step</SPAN></H3><P>Mix:
</P><UL><LI> 10 µL 100 mM Tris-HCL, pH 8.3
</LI><LI> 2.5 µL 2M KCL
</LI><LI> 3.5 µL 200 mM MgCL<SUB>2</SUB></LI><LI> 2 µL 25 mM MnCL<SUB>2</SUB> (ddd immediately before initiation of the PCR)
</LI><LI> 4 µL 25 mM dCTP
</LI><LI> 4 µL 25 mM dTTP
</LI><LI> 4 µL 5 mM dATP
</LI><LI> 4 µL 5 mM dGTP
</LI><LI> 0.5 µL 100 µM 5´primer (equals 50 pmol)
</LI><LI> 0.5 µL 100 µM 3´primer (equals 50 pmol)
</LI><LI> x µL Template K389212 – total amount of 1.5 pmol
</LI><LI> 5 µL 1 U/µL <I>taq</I> DNA polymerase
</LI><LI> Add MiliQ water to 100 µL total volume
</LI></UL><UL><LI> PCR program:
</LI><LI> Start: 8 min, 98 °C (melt)
</LI><LI> 24 cycles of:
</LI><LI> 30 s, 98 °C (melt)
</LI><LI> 30 s, 60 °C (anneal)
</LI><LI> 30 s / 1 kb template, 72 °C (extension)
</LI></UL><LI> Final extension: 5 min, 72 °C 
</LI><LI> End: Keep at 4 °C
</LI><H3><SPAN class="mw-headline" id="2._Cloning_of_virA_variants">2. Cloning of <I>virA</I> variants</SPAN></H3><P>Clone the randomly mutated <I>virA</I> sequences (2.5 kB) under the control of the promoter &lt;partinfo&gt;J23110&lt;/partinfo&gt; and the RBS &lt;partinfo&gt;B0034&lt;/partinfo&gt; in the &lt;partinfo&gt;pSB1AT3&lt;/partinfo&gt; backbone.
</P><P>(See above for <A href="#Silver_BioBrick_Assembly">protocol of Prefix- / Suffix-Insertion</A>)
</P><H3><SPAN class="mw-headline" id="3._Primary_selection_of_novel_virA_variants">3. Primary selection of novel <I>virA</I> variants</SPAN></H3><P>Transform the libraries of <I>virA</I> variants into electro-competent <I>E. coli</I> cells, in which &lt;partinfo&gt;K389011&lt;/partinfo&gt; in a plasmid with <A href="#Construction_of_a_plasmid_with_R6K_origin_of_replication_.28ori.29"> R6K ori</A> was already transformed and thereby the read out system via kanamycin resistance is already present.
</P><P>(See above for <A href="#Transformation_via_electroporation">protocol of transformation</A>)
</P><P>Plate the transformed bacteria on selective LB-Agar. When producing the plates add the following mixture of substances to the hand-warm LB-Agar:
</P><UL><LI> 1 mL ampicillin stock solution (final concentration 100 µg mL<SUP>-1</SUP>)
</LI><LI> 286 µL chloramphenicol stock solution (final concentration 10 µg mL<SUP>-1</SUP>)
</LI><LI> These antibiotica are used to ensure stability of both plasmids
</LI></UL><UL><LI> 50 mL substance-mix – including capsaicin, dopamine, homovanillic acid and 3-O methyldopamine (all final concentration of 200 µM)
</LI><LI> Any bacteria with a <I>virA</I> variation that is activated by at least one of these novel inductors will express the kanamycin resistance
</LI></UL><UL><LI> 2 mL Kanamycin stock solution (final: 100 µg mL<SUP>-1</SUP>)
</LI><LI> Compareable high kanamycin concentration to allow only these bacteria to grow which show a strong expression o the kanamycin resistance. (This optimal concentration was previously determined in MIC analysis.)
</LI></UL><P>Incubate plates upside down at 37 °C for 16 – 20 h.
The grown colonies will include a <I>virA</I> variant that is either constitutively active or had been induced by at least one of the tested substances.
</P><H3><SPAN class="mw-headline" id="4._Quantitative_analysis_of_virA_variants_after_induction_with_novel_substances">4. Quantitative analysis of virA variants after induction with novel substances</SPAN></H3><P>To quantify the induction profile of the selected <I>virA</I> variants it is appropriate to change the read out system from kanamycin resistence to luciferase expression. This complex and time consuming task can easily be achieved without any cloning step, when using the advantage of our two plasmid system:
</P><UL><LI> Pick a colony of interest and plate it on a fresh LB-Agar, containing Amp (100 µg mL<SUP>-1</SUP>) and Cm (10 µg mL<SUP>-1</SUP>)
</LI><LI> Incubate upside down at 37 °C for 16 – 20 h.
</LI><LI> Isolate plasmids from the bacterial lawn with a miniprep.
</LI><LI> In this step two plasmids are isolated – One includes the <I>virA</I> variants in a normal &lt;partinfo&gt;pSB1AT3&lt;/partinfo&gt; backbone. The second type with the kanamycin read out has the special R6K ori, which can only amplify in certain <I>E. coli</I> strains.
</LI></UL><UL><LI> Transform the isolated plasmids to a strain which is not capable to amplify the R6K ori (<I>e.g.</I> TOP10)
</LI><LI> Plate the transformants on LB-Agar with ampicillin (100 µg mL<SUP>-1</SUP>) and incubate upside down at 37 °C for 16 – 20 h.
</LI><LI> Isolate plasmids from the bacterial lawn with a miniprep.
</LI><LI> Here you are only harvesting the plasmid including the <I>virA</I> variant in &lt;partinfo&gt;pSB1AT3&lt;/partinfo&gt;, since the other plasmid was not amplified in the grown <I>E. coli</I>.
</LI></UL><UL><LI> Transform the plasmid with the <I>virA</I> variant into competent cells, already including a second plasmid with a read out system luciferase or mRFP.
</LI></UL><UL><LI> Grow the transformants in shake flaks in LB-Media containing ampicillin (100 µg mL<SUP>-1</SUP>), chloramphenicol (10 µg mL<SUP>-1</SUP>) and 200 µM of one testing substance (<I>e.g.</I> capsaicin). As a control you should also grow the same bacteria without any inductor and with the native inductor acetosyringone.
</LI><LI> Take samples in certain intervals (<I>e.g.</I> 1 h) and <A href="#Measuring_of_mRFP"> measure the activity of the luciferase or mRFP</A>.
</LI></UL><H2><SPAN class="mw-headline" id="Restriction_analysis"> Restriction analysis </SPAN></H2><UL><LI> Digest BioBrick of interest: about 400 ng DNA / 10 µL volume, 1 µL 10x orange buffer, 0.5 µL NotI or PstI. Digest for 2 h at 37 °C. NotI is used to determine the length of the BioBrick and the plasmid backbone, PstI ist used to determine the length of the BioBrick in the plasmid backbone. 
</LI></UL><UL><LI> Gel electrophoresis: add 2 µL loading buffer to every digestion mix, apply about 100 - 200 ng DNA / pocket in gel. Don't forget to apply the uncut BioBrick as well. A good agarose concentration for BioBricks between 0.2 and 3 kb is 1.5 %. The smaller your BioBrick of interest is the higher the agarose concentration should be and vice versa. The gel electrophoresis is made with TAE-buffer. Be sure that you melt your agarose gel in the same buffer you use for the electrophoresis later. 
</LI></UL><H2><SPAN class="mw-headline" id="Cultivation_for_measuring_mRFP_and_Luciferase_expression"> Cultivation for measuring mRFP and Luciferase expression </SPAN></H2><UL><LI> Inoculate 10 mL LB containing desired Antibiotic with glyccerol stock
</LI><LI> Cultivate over night at 37 °C and 175 rpm
</LI><LI> Measure the OD<SUB>600</SUB></LI><LI> Prepare shake flasks with LB, antibiotic and inducer
</LI><LI> For Luciferase Measurement at least 10 mL starting volume
</LI><LI> For mRFP Measurement at least 20 mL starting volume
</LI></UL><LI> Inoculate the main culture with a starting OD<SUB>600</SUB> of 0.1
</LI><LI> Cultivate at 37 °C and 175 rpm
</LI><LI> Take a sample at least every hour and measure the OD<SUB>600</SUB></LI><LI> The sample handling depends on the readout strategy and is described in the corresponding measuring protocol (<A href="#Measuring_of_mRFP"> mRFP</A> or <A href="#Measuring_of_Luciferase"> Luciferase</A>)
</LI><H2><SPAN class="mw-headline" id="Measuring_of_mRFP"> Measuring of mRFP </SPAN></H2><UL><LI> Take at least 500 µL sample for each measurement (200 µL is needed for one measurement) so you can perform a repeat determination
</LI><LI> Freeze samples at -80 °C for storage
</LI><LI> To measure the samples thaw at room temperature and fill 200 µL of each sample in one well of a black, flat bottom 96 well microtiter plate (perform at least a repeat determination)
</LI><LI> Measure the fluorescence in a platereader (we used a Tecan Infinite® m200 platereader) with following settings:
</LI><LI> 20 sec orbital shaking (1 mm amplitude with a frequency of 87.6 rpm)
</LI><LI> Measurement mode: Top
</LI><LI> Excitation: 584 nm
</LI><LI> Emission: 620 nm
</LI><LI> Number of reads: 25
</LI><LI> Manual gain: 150
</LI><LI> Integration time: 20 µs
</LI></UL><H2><SPAN class="mw-headline" id="Measuring_of_Luciferase"> Measuring of Luciferase </SPAN></H2><P>For the luciferase detection we used a <A href="http://www.promega.com/tbs/tb281/tb281.pdf" class="external text" rel="nofollow">Promega Luciferase Assay System</A>, containing a Cell Culture Lysis Reagent, Luciferase Assay Substrate and Luciferase Assay Buffer
</P><P>
Protocol:
</P><UL><LI> Prepare reaction tubes with 10 µL of high salt buffer (1M K<SUB>2</SUB>HPO<SUB>4</SUB>, 20mM EDTA, pH 7.8)
</LI><LI> Add 90 µL sample, mix and freeze at -80 °C
</LI><LI> For the measurement thaw by placing the tubes in room temperature water
</LI><LI> Add 300 µL of freshly prepared lysis mix (<A href="#Cell_Culture_Lysis_Reagent"> 1X Cell Culture Lysis Reagent</A>, 1.25 mg mL<SUP>-1</SUP> lysozyme, 2.5 mg mL<SUP>-1</SUP> BSA, add water for desired volume)
</LI><LI> Mix and incubate the cells for 10 minutes at room temperature
</LI><LI> Prepare the Luciferase Assay Reagent, by adding 10 mL of Luciferase Assay Buffer to the vial containing the Luciferase Assay Substrate
</LI><LI> Fill each well of a white, flat bottom 96 well microtiter plate with 20 µL of cell lysate
</LI><LI> For the detection of luciferase use a plate reading luminometer with injector for the Luciferase Assay Reagent and following settings (we used a Promega GloMax®-Multi Detection System with dual injector):
</LI><LI> Injection volume of Luciferase Assay Reagent: 100 µL
</LI><LI> Delay: 20 secs
</LI><LI> Integration: 3 secs
</LI></UL><H2><SPAN class="mw-headline" id="Chemicals.2C_material_etc."> Chemicals, material etc. </SPAN></H2><H3><SPAN class="mw-headline" id="Enzymes">Enzymes</SPAN></H3><TABLE class="wikitable" style="text-align:left"><TBODY><TR><TH> Enzyme </TH><TH style="padding-left:5px;">Producer
</TH></TR><TR><TD> Pfu DNA-polymerase </TD><TD style="padding-left:5px;">Promega
</TD></TR><TR><TD> Phusion DNA-polymerase </TD><TD style="padding-left:5px;">Finnzymes
</TD></TR><TR><TD> Restriction enzymes    </TD><TD style="padding-left:5px;">Fermentas
</TD></TR><TR><TD> Shrimp alcaline phosphatase </TD><TD style="padding-left:5px;">Fermentas
</TD></TR><TR><TD> T4-DNA-Ligase </TD><TD style="padding-left:5px;">Fermentas
</TD></TR><TR><TD><I>taq</I> DNA-polymerase </TD><TD style="padding-left:5px;">Bioline
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Kits">Kits</SPAN></H3><TABLE class="wikitable" style="text-align:left"><TBODY><TR><TH> Function </TH><TH style="padding-left:5px;"> Name
</TH></TR><TR><TD> Plasmid purification </TD><TD style="padding-left:5px;"> Fermentas GeneJET™ Plasmid Miniprep Kit
</TD></TR><TR><TD> PCR Cleanup </TD><TD style="padding-left:5px;"> Macherey Nagel NucleoSpin® Extract II
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Inducer_Stock_Solutions"> Inducer Stock Solutions </SPAN></H3><UL><LI> Acetosyringone: 20 mM solved in 10 % (v/v) DMSO and 90 % (v/v) H<SUB>2</SUB>O
</LI><LI> Capsaicin: 20 mM solved in Methanol
</LI><LI> Dopamine: 20 mM solved in H<SUB>2</SUB>O
</LI><LI> Homovanillic acid: 20 mM solved in H<SUB>2</SUB>O
</LI><LI> 3-Methoxytyramine: 20 mM solved in H<SUB>2</SUB>O
</LI></UL><H3><SPAN class="mw-headline" id="TAE_buffer">TAE buffer</SPAN></H3><P>For 1 L of 50 x TAE buffer you need:
</P><UL><LI> 242.48 g Tris
</LI></UL><UL><LI> 41.02 g Sodiumacetate
</LI></UL><UL><LI> 18.612 g EDTA
</LI></UL><UL><LI> Adjust pH to 7.8
</LI></UL><UL><LI> Solve in dH<SUB>2</SUB>O
</LI></UL><P>20 mL of the stock is diluted in 1 L dH<SUB>2</SUB>O for the gel electrophoresis.
</P><H3><SPAN class="mw-headline" id="DNA_loading_buffer">DNA loading buffer</SPAN></H3><UL><LI> 50 % (v/v) glycerol
</LI></UL><UL><LI> 1 mM EDTA
</LI></UL><UL><LI> 0.1 % (w/v) bromphenol blue
</LI></UL><UL><LI> Solve in ddH20
</LI></UL><H3><SPAN class="mw-headline" id="LB_medium">LB medium</SPAN></H3><P>For 1 L of LB medium you need: 
</P><UL><LI> 10 g Trypton
</LI></UL><UL><LI> 5 g yeast extract
</LI></UL><UL><LI> 10 g NaCl
</LI></UL><UL><LI> 12 g Agar-Agar (for plates)
</LI></UL><UL><LI> Adjust pH to 7.0
</LI></UL><H3><SPAN class="mw-headline" id="Cell_Culture_Lysis_Reagent">Cell Culture Lysis Reagent</SPAN></H3><UL><LI> 25 mM Tris-phosphate (pH 7.8)
</LI><LI> 2 mM DTT
</LI><LI> 2 mM 1,2-diaminocyclohexane-N,N,N´,N´-tetraacetic acid
</LI><LI> 10 % glycerol
</LI><LI> 1 % Triton® X-100
</LI></UL><H1><SPAN class="mw-headline" id="References"> References </SPAN></H1><UL><LI>Openwetware: Quint Lab: electrocompetent cells e.coli, <A href="http://openwetware.org/wiki/Quint_Lab:electrocompetent_cells_e.coli" class="external free" rel="nofollow">http://openwetware.org/wiki/Quint_Lab:electrocompetent_cells_e.coli</A>, October 24th 2010.
</LI></UL><UL><LI>Openwetware: Silver: BB Strategy, <A href="http://openwetware.org/wiki/Silver:_BB_Strategy" class="external free" rel="nofollow">http://openwetware.org/wiki/Silver:_BB_Strategy</A>, October 24th 2010.
</LI></UL><UL><LI>BioBrick Assembly Manual, Ginkgo BioWorks, <A href="http://ginkgobioworks.com/support/BioBrick_Assembly_Manual.pdf" class="external free" rel="nofollow">http://ginkgobioworks.com/support/BioBrick_Assembly_Manual.pdf</A>, October 24th 2010.
</LI></UL><UL><LI><I>Luciferase Assay System Technical Bulletin</I> #TB281, Promega Cooperation, <A href="http://www.promega.com/tbs/tb281/tb281.pdf" class="external free" rel="nofollow">http://www.promega.com/tbs/tb281/tb281.pdf</A>, October 24th 2010.
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Bielefeld-Germany/Project/Protocols">http://2010.igem.org/Team:Bielefeld-Germany/Project/Protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Project/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Project/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project/Protocols&amp;oldid=205296" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>