<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Results_Used"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Results/Used">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Results/Used&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/Used&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/Used&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Results/Used" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Results/Used</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="contentSub">(Redirected from <A href="/wiki/index.php?title=Team:Bielefeld-Germany/BioBricks/Used&amp;redirect=no" title="Team:Bielefeld-Germany/BioBricks/Used">Team:Bielefeld-Germany/BioBricks/Used</A>)</DIV><DIV id="sliderWrap"><DIV id="slider"><DIV id="openCloseWrap" style="height:110px;"><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany">Start</A></LI></DIV><DIV class="navColumn" style="width:13%;"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team">Team</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Description">Description</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Supervisors">Supervisors</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Bielefeld">University</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/BielefeldCity">Bielefeld</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Motivation">Motivation</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Acknowledgements">Acknowledgements</A></LI></UL></DIV><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project">Project</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Application">Application</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Approach">Approach</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Theory">Theory</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Protocols">Protocols</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Model">Model</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Outlook">Outlook</A></LI></UL></DIV><DIV class="navColumn" style="width:12%;"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results">Results</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Characterization">Characterization</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Used">Used BBs</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Submitted">Submitted BBs</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Sequencing">Sequencing</A></LI></UL></DIV><DIV class="navColumn" style="width:12%;"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Safety#Biosecurity">Biosecurity</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Safety#Biosafety">Biosafety</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Safety#Risk_Assesement_.28english.29">Risk Assesement</A></LI></UL></DIV><DIV class="navColumn" style="width:13%;"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Public_Relations">Public Relations</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Public_Relations/Fotos">Fotos</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Public_Relations/Press">Press</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Public_Relations/Public_Discussion">Public Discussion</A></LI></UL></DIV><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Judging">Judging</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Judging/BioBricks">BioBricks</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Judging/Helping">Helping other teams</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Judging/Comments">Comments on iGEM</A></LI></UL></DIV><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Sponsors">Sponsors</A></LI></DIV><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Contact">Contact</A></LI></DIV></DIV></DIV><DIV id="sliderController" class="sliderController">MENU</DIV></DIV><DIV id="checkParameter" style="margin-right:250px;">http://igem-bielefeld.de/img/banner_lab.png</DIV><DIV id="subnavigation"><LI><A href="/Team:Bielefeld-Germany/Results">Results</A></LI><LI><A href="/Team:Bielefeld-Germany/Results/Characterization">Characterization</A></LI><LI><A href="/Team:Bielefeld-Germany/Results/Used">Used</A></LI><LI><A href="/Team:Bielefeld-Germany/Results/Submitted">Submitted</A></LI><LI><A href="/Team:Bielefeld-Germany/Results/Sequencing">Sequencing</A></LI></DIV><H1><SPAN class="mw-headline" id="Used_BioBricks"> Used BioBricks </SPAN></H1><P>This is a list of all BioBricks we used from the registry or created by ourselves and a short explanation, why and / or how we used them. The list is sorted alphabetically. 
</P><TABLE cellpadding="10" style="border-collapse: collapse; border-width: 1px; border-style: solid; border-color: #000"><TBODY><TR><TH style="border-style: solid; border-width: 1px"> BioBrick (name and partnumber)
</TH><TH style="border-style: solid; border-width: 1px"> Status
</TH><TH style="border-style: solid; border-width: 1px"> Comment
</TH></TR><TR><TD style="border-style: solid; border-width: 1px"> double terminator, &lt;partinfo&gt;B0017&lt;/partinfo&gt;: &lt;partinfo&gt;B0017 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> from registry
</TD><TD style="border-style: solid; border-width: 1px"><P>This is a double terminator which contains two &lt;partinfo&gt;B0010&lt;/partinfo&gt; terminators. We used this double terminator instead of the commonly used &lt;partinfo&gt;B0015&lt;/partinfo&gt; double terminator (which was assembled from the &lt;partinfo&gt;B0010&lt;/partinfo&gt; and the &lt;partinfo&gt;B0012&lt;/partinfo&gt; terminator) because of problems mentioned in the partsregistry with the &lt;partinfo&gt;B0012&lt;/partinfo&gt; terminator. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> RBS, &lt;partinfo&gt;B0034&lt;/partinfo&gt;: &lt;partinfo&gt;B0034 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> from registry
</TD><TD style="border-style: solid; border-width: 1px"><P>The strong ribosomal binding site (RBS) AGGAG was always assembled. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"><I>lacI</I> gene, &lt;partinfo&gt;C0012&lt;/partinfo&gt;: &lt;partinfo&gt;C0012 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> from registry
</TD><TD style="border-style: solid; border-width: 1px"><P>This is a gene for the lac repressor <I>lacI</I> with a LVA degradation tag. We used this BioBrick to build a tightly regulated <I>lac</I>-operon &lt;partinfo&gt;K389050&lt;/partinfo&gt; with low basal transcription. It seems that this <I>lacI</I> is either degraded too quickly or it does not work properly because it is not possible to repress the <I>lac</I> operon with this part. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> mRFP, &lt;partinfo&gt;E1010&lt;/partinfo&gt;: &lt;partinfo&gt;E1010 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> from registry
</TD><TD style="border-style: solid; border-width: 1px"><P>This BioBrick was used as a reporter gene, <I>e.g.</I> in the construct &lt;partinfo&gt;K389013&lt;/partinfo&gt;. mRFP as a reporter gene has the advantage that it is <A href="/Team:Bielefeld-Germany/Results/Characterization/K389004#mRFP_vs._luciferase_as_reporter_gene" title="Team:Bielefeld-Germany/Results/Characterization/K389004"> easy to measure and comparatively stable</A>. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"><I>lacI<SUP>q</SUP></I> promoter, &lt;partinfo&gt;I14032&lt;/partinfo&gt;: &lt;partinfo&gt;I14032 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> from registry
</TD><TD style="border-style: solid; border-width: 1px"><P>This promoter is a strong constitutive promoter which is natively located upstream of the <I>lacI</I> gene in <I>E. coli</I> lacI<SUP>q</SUP> strains. It was used to build a tightly regulated <I>lac</I>-operon &lt;partinfo&gt;K389050&lt;/partinfo&gt; in which it was assembled upstream of the <I>lacI</I> gene from the partsregistry &lt;partinfo&gt;C0012&lt;/partinfo&gt;. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> sensitivity tuner, &lt;partinfo&gt;I746370&lt;/partinfo&gt;: &lt;partinfo&gt;I746370 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> from registry
</TD><TD style="border-style: solid; border-width: 1px"><P>We used this BioBrick to amplify our read-out signal luciferase in the part &lt;partinfo&gt;K389401&lt;/partinfo&gt;. This BioBrick is supposed to amplify the effect of a PoPS device 15 - 20 fold compared to a system without sensitivity tuner (<A href="https://2007.igem.org/Cambridge/Amplifier_project#Results" class="external text" rel="nofollow">Cambridge, iGEM 2007, amplifier project</A>). 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> sensitivity tuner, &lt;partinfo&gt;I746380&lt;/partinfo&gt;: &lt;partinfo&gt;I746380 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> from registry
</TD><TD style="border-style: solid; border-width: 1px"><P>We used this BioBrick to amplify our read-out signal luciferase in the part &lt;partinfo&gt;K389402&lt;/partinfo&gt;. This BioBrick is supposed to amplify the effect of a PoPS device 10 fold compared to a system without sensitivity tuner (<A href="https://2007.igem.org/Cambridge/Amplifier_project#Results" class="external text" rel="nofollow">Cambridge, iGEM 2007, amplifier project</A>). 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> sensitivity tuner, &lt;partinfo&gt;I746390&lt;/partinfo&gt;: &lt;partinfo&gt;I746390 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> from registry
</TD><TD style="border-style: solid; border-width: 1px"><P>We used this BioBrick to amplify our read-out signal luciferase in the part &lt;partinfo&gt;K389403&lt;/partinfo&gt;. This BioBrick is supposed to amplify the effect of a PoPS device 30 - 35 fold compared to a system without sensitivity tuner (<A href="https://2007.igem.org/Cambridge/Amplifier_project#Results" class="external text" rel="nofollow">Cambridge, iGEM 2007, amplifier project</A>). 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> mRFP generator, &lt;partinfo&gt;J04450&lt;/partinfo&gt;: &lt;partinfo&gt;J04450 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> from registry
</TD><TD style="border-style: solid; border-width: 1px"><P>This BioBrick was used as a visible selection marker for cloning other BioBricks into the &lt;partinfo&gt;pSB1C3&lt;/partinfo&gt; plasmid to submit them to the registry. The promoter in this mRFP generator is pretty strong so it is easy to separate colonies which carry this BioBrick from colonies which do not carry this BioBrick anymore. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> strong constitutive promoter, &lt;partinfo&gt;J23102&lt;/partinfo&gt;: &lt;partinfo&gt;J23102 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> from registry
</TD><TD style="border-style: solid; border-width: 1px"><P>A strong constitutive promoter which was used to create BioBrick &lt;partinfo&gt;K389318&lt;/partinfo&gt; to test the firefly luciferase BioBrick &lt;partinfo&gt;K389004&lt;/partinfo&gt; and the luciferase measurement system. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> weak constitutive promoter, &lt;partinfo&gt;J23103&lt;/partinfo&gt;: &lt;partinfo&gt;J23103 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> from registry
</TD><TD style="border-style: solid; border-width: 1px"><P>A weak constitutive promoter which was used to create BioBrick &lt;partinfo&gt;K389302&lt;/partinfo&gt; to test the firefly luciferase BioBrick &lt;partinfo&gt;K389004&lt;/partinfo&gt; and the luciferase measurement system. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> medium strong constitutive promoter, &lt;partinfo&gt;J23110&lt;/partinfo&gt;: &lt;partinfo&gt;J23110 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> from registry
</TD><TD style="border-style: solid; border-width: 1px"><P>This medium strong constitutive promoter from the registry was assembled before our BioBricks, so they are neither expressed too strong, so the cell suffers from metabolomic stress, nor too weak, so there is no visible effect of our BioBricks. This BioBrick was used <I>e.g.</I> in &lt;partinfo&gt;K389010&lt;/partinfo&gt; or &lt;partinfo&gt;K389011&lt;/partinfo&gt;. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> medium strong constitutive promoter, &lt;partinfo&gt;J23115&lt;/partinfo&gt;: &lt;partinfo&gt;J23115 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> from registry
</TD><TD style="border-style: solid; border-width: 1px"><P>A medium strong constitutive promoter which was used to create BioBrick &lt;partinfo&gt;K389307&lt;/partinfo&gt; to test the firefly luciferase BioBrick &lt;partinfo&gt;K389004&lt;/partinfo&gt; and the luciferase measurement system. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> R6K origin, &lt;partinfo&gt;J61001&lt;/partinfo&gt;: &lt;partinfo&gt;J61001 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> from registry
</TD><TD style="border-style: solid; border-width: 1px"><P>This BioBrick was used to create a plasmid without ColE1 ori. The R6K ori has a different compatibility class than the ColE1 ori. So it is used for a two plasmid screening system. The R6K ori is a medium copy plasmid in pir+ / pir116 <I>E. coli</I> strains like EC100D but it does not replicate in pir- strains like <I>E. coli</I> TOP10. So it is easy to separate a ColE1 ori plasmid from another with R6K ori by simply transforming both to <I>e.g. E. coli</I> TOP10. 
The R6K ori was cloned into the pSB1C3 plasmid. Subsequently, the plasmid containing the R6K ori was digested with Hin6I. The largest fragment was extracted from an agarose gel and ligated with T4 DNA Polymerase. Thereby the ColE1 ori was removed, leaving a small fragment of about 30 bp.
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"><I>virA</I>, &lt;partinfo&gt;K238008&lt;/partinfo&gt;: &lt;partinfo&gt;K238008 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> from registry, fixed
</TD><TD style="border-style: solid; border-width: 1px"><P>We wanted to use this <I>virA</I> BioBrick from the registry but problems occurred (<A href="/Team:Bielefeld-Germany/Results/Characterization" title="Team:Bielefeld-Germany/Results/Characterization"> characterization</A>), so we created our own <I>virA</I> from <I>Agrobacterium tumefaciens</I> C58 TI-plasmid. Our own (working) <I>virA</I> has the partnumber &lt;partinfo&gt;K389001&lt;/partinfo&gt;. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"><I>vir</I>-promoter, &lt;partinfo&gt;K238011&lt;/partinfo&gt;: &lt;partinfo&gt;K238011 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> from registry, fixed
</TD><TD style="border-style: solid; border-width: 1px"><P>The same problems occurred as with the <I>virA</I> BioBrick - this one from the registry <A href="/Team:Bielefeld-Germany/Results/Characterization" title="Team:Bielefeld-Germany/Results/Characterization"> does not work properly</A>, so we made a <I>vir</I>-promoter by ourselves (&lt;partinfo&gt;K389003&lt;/partinfo&gt;). The part that was sent to us is actually a <I>tetR</I> gene (&lt;partinfo&gt;C0040&lt;/partinfo&gt;) under the control of a constitutive promoter (&lt;partinfo&gt;J23105&lt;/partinfo&gt;) before a double terminator (&lt;partinfo&gt;B0015&lt;/partinfo&gt;, compare our <A href="/Team:Bielefeld-Germany/Results/Sequencing" title="Team:Bielefeld-Germany/Results/Sequencing"> sequencing results</A>). 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"><I>virA</I>, &lt;partinfo&gt;K389001&lt;/partinfo&gt;: &lt;partinfo&gt;K389001 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> new
</TD><TD style="border-style: solid; border-width: 1px"><P>The VirA receptor is used by <I>A. tumefaciens</I> to detect acetosyringone and other phenolic substances which are secreted by plants after injury. In presence of these substances VirA phosphorylates itself and afterwards VirG, a response regulator which activates <I>vir</I> promoters. These promoters control genes which are used for infecting the injured plant. 
This <I>virA</I> gene was isolated from the TI-plasmid of <I>A. tumefaciens</I> C58. An illegal PstI restriction site was removed by site-directed mutagenesis. The functionality of this BioBrick was proved with &lt;partinfo&gt;K389015&lt;/partinfo&gt; and &lt;partinfo&gt;K389016&lt;/partinfo&gt;. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> mutated <I>virG</I>, &lt;partinfo&gt;K389002&lt;/partinfo&gt;: &lt;partinfo&gt;K389002 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> synthesized, new
</TD><TD style="border-style: solid; border-width: 1px"><P>This is the VirG response regulator from the VirA/G receptor system. The VirA/G receptor system is used by <I>A. tumefaciens</I> to detect phenolic substances secreted by injured plants, <I>e.g.</I> acetosyringone. In presence of these substances, VirG is activated by the VirA receptor and induces the transcription of genes under the control of a <I>vir</I> promoter. Normally, VirG needs the <I>rpoA</I> gene (RNA polymerase subunit) from <I>A. tumefaciens</I> to work in <I>Escherichia coli</I> but this <I>virG</I> BioBrick is mutated, so it works with the <I>rpoA</I> subunit from <I>E. coli</I>. For this reason the point mutations G56V and I77V were brought into the gene (<A href="http://www.springerlink.com/content/wmq06kua5qkma1au/" class="external text" rel="nofollow">YC Jung <I>et al.</I>, 2004</A>). Because this BioBrick is synthesized (Mr. Gene GmbH), codon usage is optimized for <I>E. coli</I> and illegal restriction sites were removed. When you use this <I>virG</I> gene in a VirA/G signaling system you do not need &lt;partinfo&gt;BBa_K238010&lt;/partinfo&gt; anymore to get the system working in <I>E. coli</I>.
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"><I>vir</I>-promoter, &lt;partinfo&gt;K389003&lt;/partinfo&gt;: &lt;partinfo&gt;K389003 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> new
</TD><TD style="border-style: solid; border-width: 1px"><P><I>Vir</I>-promoters from <I>A. tumefaciens</I> are induced by phosphorylated VirG response regulators and control genes for infecting plants in their natural host. They are part of the VirA/G signal transduction system. This is a <I>vir</I> promoter from <I>A. tumefaciens</I> C58. It is located on the TI-plasmid upstream the <I>virB</I> genes.
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> firefly luciferase, &lt;partinfo&gt;K389004&lt;/partinfo&gt;: &lt;partinfo&gt;K389004 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> new
</TD><TD style="border-style: solid; border-width: 1px"><P>This is a BioBrick for a firefly (<I>Photinus pyralis</I>) luciferase. It is a <A href="/Team:Bielefeld-Germany/Results/Characterization/K389004#Sensitivity" title="Team:Bielefeld-Germany/Results/Characterization/K389004"> sensitive reporter gene</A>. It was isolated from Promega's pGL4.10[luc2] plasmid. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> Kanamycin resistance gene, &lt;partinfo&gt;K389005&lt;/partinfo&gt;: &lt;partinfo&gt;K389005 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> new / modified
</TD><TD style="border-style: solid; border-width: 1px"><P>This is a kanamycin / neomycin resistance gene without promoter or RBS. It is used for directed evolution of <I>virA</I> and mutated <I>virA</I> screenings, respectively. It was isolated from the BioBrick &lt;partinfo&gt;P1003&lt;/partinfo&gt; (a kanamycin resistance gene with promoter and RBS). 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"><I>virA</I> generator, &lt;partinfo&gt;K389010&lt;/partinfo&gt;: &lt;partinfo&gt;K389010 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> composite
</TD><TD style="border-style: solid; border-width: 1px"><P>This part is for expressing a VirA receptor. A medium strong constitutive promoter is used, so there is enough VirA receptor expressed but not too much so the cell suffers from the expression (VirA is a membrane protein). This version exists in a &lt;partinfo&gt;pSB1C3&lt;/partinfo&gt; and a &lt;partinfo&gt;pSB1AT3&lt;/partinfo&gt; vector. We used the BioBrick in the &lt;partinfo&gt;pSB1AT3&lt;/partinfo&gt; plasmid for error-prone PCR and later in our two plasmid screening system together with the &lt;partinfo&gt;K389011&lt;/partinfo&gt; VirA-screening device. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"><I>virA</I> screening device, &lt;partinfo&gt;K389011&lt;/partinfo&gt;: &lt;partinfo&gt;K389011 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> composite
</TD><TD style="border-style: solid; border-width: 1px"><P>This BioBrick contains the new, mutated <I>virG</I> which works in <I>E. coli</I> without the <I>rpoA</I> gene from <I>A. tumefaciens</I> which is expressed constitutively and a kanamycin resistance gene which is under the control of a <I>vir</I> promoter. It is used to screen mutants of <I>virA</I> which are on a different plasmid in a &lt;partinfo&gt;K389010&lt;/partinfo&gt;-like part and were created by error-prone PCR. The better the VirA receptor recognizes a substance the more resistant the cell is against kanamycin. This device has to be on a plasmid with different ori than the <I>virA</I> which should be screened (different compatibility classes). We cloned this BioBrick into a plasmid with R6K ori which only replicates in pir+ or pir116 strains (<I>e.g.</I><I>E. coli</I> EC100D), so we can easily seperate the two plasmids by transforming them into strains without Pir protein (<I>e.g. E. coli</I> TOP10). 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"><I>virA</I> reporter device with luc, &lt;partinfo&gt;K389012&lt;/partinfo&gt;: &lt;partinfo&gt;K389012 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> composite
</TD><TD style="border-style: solid; border-width: 1px"><P>This BioBrick is similar to &lt;partinfo&gt;K389011&lt;/partinfo&gt; but with the reporter luciferase instead of an antibiotic resistance gene. With this BioBrick it is possible to measure the activity of the <I>vir</I> promoter and the VirA receptor, respectively. </P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"><I>virA</I> reporter device with mRFP, &lt;partinfo&gt;K389013&lt;/partinfo&gt;: &lt;partinfo&gt;K389013 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> composite
</TD><TD style="border-style: solid; border-width: 1px"><P>This BioBrick is similar to &lt;partinfo&gt;K389011&lt;/partinfo&gt; but with the reporter mRFP instead of an antibiotic resistance gene. With this BioBrick it is possible to measure the activity of the <I>vir</I> promoter and the VirA receptor, respectively. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"><I>virA</I> screening approach test device, &lt;partinfo&gt;K389014&lt;/partinfo&gt;: &lt;partinfo&gt;K389014 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> composite
</TD><TD style="border-style: solid; border-width: 1px"><P>This BioBrick is an assembly of &lt;partinfo&gt;K389010&lt;/partinfo&gt; and &lt;partinfo&gt;K389011&lt;/partinfo&gt;. It is used for testing the general screening concept of expressing an antibiotic resistance with a <I>vir</I> promoter after induction with acetosyringone (the natural VirA inductor). 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"><I>vir</I> promoter characterisation part with luc, &lt;partinfo&gt;K389015&lt;/partinfo&gt;: &lt;partinfo&gt;K389015 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> composite
</TD><TD style="border-style: solid; border-width: 1px"><P>This BioBrick is an assembly of &lt;partinfo&gt;K389010&lt;/partinfo&gt; and &lt;partinfo&gt;K389012&lt;/partinfo&gt;. It is used for <A href="/Team:Bielefeld-Germany/Results/Characterization/K389015" title="Team:Bielefeld-Germany/Results/Characterization/K389015"> characterizing</A> the natural VirA/G system and the <I>vir</I> promoter, respectively. It is also possible to demonstrate the functionality of four new created BioBricks in one device. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"><I>vir</I> promoter characterisation part with mRFP, &lt;partinfo&gt;K389016&lt;/partinfo&gt;: &lt;partinfo&gt;K389016 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> composite
</TD><TD style="border-style: solid; border-width: 1px"><P>This BioBrick is an assembly of &lt;partinfo&gt;K389010&lt;/partinfo&gt; and &lt;partinfo&gt;K389013&lt;/partinfo&gt;. It is used for <A href="/Team:Bielefeld-Germany/Results/Characterization/K389016" title="Team:Bielefeld-Germany/Results/Characterization/K389016"> characterizing</A> the natural VirA/G system and the <I>vir</I> promoter, respectively. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"><I>vir</I> promoter part, &lt;partinfo&gt;K389017&lt;/partinfo&gt;: &lt;partinfo&gt;K389017 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> composite
</TD><TD style="border-style: solid; border-width: 1px"><P>This is a complete VirA/G signaling system without a reporter gene, so any gene of interest can be assembled to a <I>vir</I> promoter. We assembled some sensitivity tuners and a luciferase gene behind this BioBrick, so we get an enhanced luciferase signal (<I>e.g.</I> in part &lt;partinfo&gt;K389421&lt;/partinfo&gt;). 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> tightly controlled <I>lac</I> operator, &lt;partinfo&gt;K389050&lt;/partinfo&gt;: &lt;partinfo&gt;K389050 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> composite
</TD><TD style="border-style: solid; border-width: 1px"><P>A lac operator is combined with a lacI<SUP>q</SUP> so there is always enough <I>lacI</I> to repress the lac operator. The system is still inducible with IPTG but it has a low basal transcription (in theory). This part should be used to test the sensitivity of the luciferase BioBrick &lt;partinfo&gt;K389004&lt;/partinfo&gt; and compare it to the sensitivity of the mRFP BioBrick &lt;partinfo&gt;E1010&lt;/partinfo&gt;. But the part did not work as expected - it had a very high basal expression. The problem is probably the <I>lacI</I> gene &lt;partinfo&gt;C0012&lt;/partinfo&gt; from the registry which has a degradation tag. Either this tag leads to that quick degradation, that the <I>lacI</I> repressor cannot work as a repressor anymore or the repressor itself does not work properly. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> tightly controlled <I>lac</I> operator + luc, &lt;partinfo&gt;K389051&lt;/partinfo&gt;: &lt;partinfo&gt;K389051 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> composite
</TD><TD style="border-style: solid; border-width: 1px"><P>The part &lt;partinfo&gt;K389050&lt;/partinfo&gt; is combined with a luciferase gene as reporter. But &lt;partinfo&gt;K389050&lt;/partinfo&gt; does not work as expected so this part could not be used for its original purpose (demonstrating the sensitivity of the firefly luciferase). 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> tightly controlled <I>lac</I> operator + mRFP, &lt;partinfo&gt;K389052&lt;/partinfo&gt;: &lt;partinfo&gt;K389052 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> composite
</TD><TD style="border-style: solid; border-width: 1px"><P>The part &lt;partinfo&gt;K389050&lt;/partinfo&gt; is combined with a mRFP gene as reporter. But &lt;partinfo&gt;K389050&lt;/partinfo&gt; does not work as expected so this part could not be used for its original purpose (comparing the sensitivity of mRFP with the sensitivity of the firefly luciferase). 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> weak constitutive promoter + luc, &lt;partinfo&gt;K389302&lt;/partinfo&gt;: &lt;partinfo&gt;K389302 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> composite
</TD><TD style="border-style: solid; border-width: 1px"><P>Luciferase under the control of a weak constitutive promoter. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> medium strong constitutive promoter + luc, &lt;partinfo&gt;K389307&lt;/partinfo&gt;: &lt;partinfo&gt;K389307 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> composite
</TD><TD style="border-style: solid; border-width: 1px"><P>Luciferase under the control of a medium strong constitutive promoter. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> strong constitutive promoter + luc, &lt;partinfo&gt;K389318&lt;/partinfo&gt;: &lt;partinfo&gt;K389318 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> composite
</TD><TD style="border-style: solid; border-width: 1px"><P>Luciferase under the control of a strong constitutive promoter. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> sensitivity tuner no. 1 without promoter + luc readout, &lt;partinfo&gt;K389401&lt;/partinfo&gt;: &lt;partinfo&gt;K389401 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> new / modified
</TD><TD style="border-style: solid; border-width: 1px"><P>The &lt;partinfo&gt;I746370&lt;/partinfo&gt; sensitivity tuner was modified: the arabinose promoter and the GFP readout were erased from the original part and a firefly luciferase BioBrick was added as a new readout. This part can be used behind a random promoter and gives an enhanced luciferase readout signal. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> sensitivity tuner no. 2 without promoter + luc readout, &lt;partinfo&gt;K389402&lt;/partinfo&gt;: &lt;partinfo&gt;K389402 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> new / modified
</TD><TD style="border-style: solid; border-width: 1px"><P>The &lt;partinfo&gt;I746380&lt;/partinfo&gt; sensitivity tuner was modified: the arabinose promoter and the GFP readout were erased from the original part and a firefly luciferase BioBrick was added as a new readout. This part can be used behind a random promoter and gives an enhanced luciferase readout signal. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> sensitivity tuner no. 3 without promoter + luc readout, &lt;partinfo&gt;K389403&lt;/partinfo&gt;: &lt;partinfo&gt;K389403 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> new / modified
</TD><TD style="border-style: solid; border-width: 1px"><P>The &lt;partinfo&gt;I746390&lt;/partinfo&gt; sensitivity tuner was modified: the arabinose promoter and the GFP readout were erased from the original part and a firefly luciferase BioBrick was added as a new readout. This part can be used behind a random promoter and gives an enhanced luciferase readout signal. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"><I>virA</I> reporter device with sensitivity tuner no. 1 + luc readout, &lt;partinfo&gt;K389411&lt;/partinfo&gt;: &lt;partinfo&gt;K389411 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> composite
</TD><TD style="border-style: solid; border-width: 1px"><P>This is a modified &lt;partinfo&gt;I746370&lt;/partinfo&gt; sensitivity tuner with luciferase readout behind a <I>vir</I> promoter. This part also contains a <I>virG</I> gene under the control of a constitutive promoter. The plan is to assemble a mutated <I>virA</I> from a &lt;partinfo&gt;K389010&lt;/partinfo&gt;-like BioBrick in front of this part to get a strong readout signal in order to reach our goal - make spiciness visible. The mutated <I>virA</I> is generated with error-prone PCR and screened together with the &lt;partinfo&gt;K389011&lt;/partinfo&gt; Biobrick. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"><I>virA</I> reporter device with sensitivity tuner no. 2 + luc readout, &lt;partinfo&gt;K389412&lt;/partinfo&gt;:&lt;partinfo&gt;K389412 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> composite
</TD><TD style="border-style: solid; border-width: 1px"><P>This is a modified &lt;partinfo&gt;I746380&lt;/partinfo&gt; sensitivity tuner with luciferase readout behind a <I>vir</I> promoter. This part also contains a <I>virG</I> gene under the control of a constitutive promoter. The plan is to assemble a mutated <I>virA</I> from a &lt;partinfo&gt;K389010&lt;/partinfo&gt;-like BioBrick in front of this part to get a strong readout signal in order to reach our goal - make spiciness visible. The mutated <I>virA</I> is generated with error-prone PCR and screened together with the &lt;partinfo&gt;K389011&lt;/partinfo&gt; Biobrick. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"><I>virA</I> reporter device with sensitivity tuner no. 3 + luc readout, &lt;partinfo&gt;K389413&lt;/partinfo&gt;: &lt;partinfo&gt;K389413 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> composite
</TD><TD style="border-style: solid; border-width: 1px"><P>This is a modified &lt;partinfo&gt;I746390&lt;/partinfo&gt; sensitivity tuner with luciferase readout behind a <I>vir</I> promoter. This part also contains a <I>virG</I> gene under the control of a constitutive promoter. The plan is to assemble a mutated <I>virA</I> from a &lt;partinfo&gt;K389010&lt;/partinfo&gt;-like BioBrick in front of this part to get a strong readout signal in order to reach our goal - make spiciness visible. The mutated <I>virA</I> is generated with error-prone PCR and screened together with the &lt;partinfo&gt;K389011&lt;/partinfo&gt; Biobrick. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"><I>vir</I> promoter characterisation part with sensitivity tuner no. 1 + luc readout, &lt;partinfo&gt;K389421&lt;/partinfo&gt;: &lt;partinfo&gt;K389421 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> composite
</TD><TD style="border-style: solid; border-width: 1px"><P>This is a modified &lt;partinfo&gt;I746370&lt;/partinfo&gt; sensitivity tuner with luciferase readout behind a <I>vir</I> promoter characterisation BioBrick, so it also contains a <I>virG</I> and an unmutated <I>virA</I> gene under the control of a constitutive promoter. This BioBrick is used to prove the function of the modified sensitivity tuner and to check whether the sensitivity tuner enhances the luciferase signal enough to make it visible. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"><I>vir</I> promoter characterisation part with sensitivity tuner no. 2 + luc readout, &lt;partinfo&gt;K389422&lt;/partinfo&gt;: &lt;partinfo&gt;K389422 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> composite
</TD><TD style="border-style: solid; border-width: 1px"><P>This is a modified &lt;partinfo&gt;I746380&lt;/partinfo&gt; sensitivity tuner with luciferase readout behind a <I>vir</I> promoter characterisation BioBrick, so it also contains a <I>virG</I> and an unmutated <I>virA</I> gene under the control of a constitutive promoter. This BioBrick is used to prove the function of the modified sensitivity tuner and to check whether the sensitivity tuner enhances the luciferase signal enough to make it visible. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"><I>vir</I> promoter characterisation part with sensitivity tuner no. 3 + luc readout, &lt;partinfo&gt;K389423&lt;/partinfo&gt;: &lt;partinfo&gt;K389423 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> composite
</TD><TD style="border-style: solid; border-width: 1px"><P>This is a modified &lt;partinfo&gt;I746370&lt;/partinfo&gt; sensitivity tuner with luciferase readout behind a <I>vir</I> promoter characterisation BioBrick, so it also contains a <I>virG</I> and an unmutated <I>virA</I> gene under the control of a constitutive promoter. This BioBrick is used to prove the function of the modified sensitivity tuner and to check whether the sensitivity tuner enhances the luciferase signal enough to make it visible. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> Kanamycin-resistance cassette, &lt;partinfo&gt;P1003&lt;/partinfo&gt;: &lt;partinfo&gt;P1003 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> from registry
</TD><TD style="border-style: solid; border-width: 1px"><P>This kanamycin resistance cassette BioBrick was used as a template to create &lt;partinfo&gt;K389005&lt;/partinfo&gt;. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"><I>ccdB</I>-gene, &lt;partinfo&gt;P1010&lt;/partinfo&gt;: &lt;partinfo&gt;P1010 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> from registry
</TD><TD style="border-style: solid; border-width: 1px"><P>The <I>ccdB</I> gene targets the gyrase of <I>Escherichia coli</I> and is lethal for all <I>E. coli</I> strains without the gyrase mutation gyrA462 (<A href="http://openwetware.org/wiki/CcdB" class="external text" rel="nofollow">openwetware</A>). The <I>ccdB</I> BioBrick is used for the 3A-assembly as a positive selection marker. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> high-copy plasmid with Amp and Tet resistance, &lt;partinfo&gt;pSB1AT3&lt;/partinfo&gt;: &lt;partinfo&gt;pSB1AT3 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> from registry
</TD><TD style="border-style: solid; border-width: 1px"><P>This plasmid was used in 3A-assemblies and as the backbone of the mutated and unmutated &lt;partinfo&gt;K389010&lt;/partinfo&gt; part. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"> high-copy plasmid with Cm resistance, &lt;partinfo&gt;pSB1C3&lt;/partinfo&gt;: &lt;partinfo&gt;pSB1C3 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> from registry
</TD><TD style="border-style: solid; border-width: 1px"><P>This is the plasmid used to submit the BioBricks. 
</P></TD></TR><TR><TD style="border-style: solid; border-width: 1px"><I>lac</I> operator, &lt;partinfo&gt;R0010&lt;/partinfo&gt;: &lt;partinfo&gt;R0010 SpecifiedComponents&lt;/partinfo&gt;</TD><TD style="border-style: solid; border-width: 1px"> from registry
</TD><TD style="border-style: solid; border-width: 1px"><P>This is a <I>lac</I> operator which is inducible with IPTG and repressed by <I>lacI</I>. It was used to build a tightly regulated <I>lac</I> operator (&lt;partinfo&gt;K389050&lt;/partinfo&gt;) by adding a <I>lacI</I> repressor gene under the control of the lacI<SUP>q</SUP> promoter. 
</P></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><UL><LI><A href="https://2007.igem.org/Cambridge/Amplifier_project#Results" class="external text" rel="nofollow">Cambridge, iGEM 2007, amplifier project</A>.
</LI></UL><UL><LI><A href="http://openwetware.org/wiki/CcdB" class="external free" rel="nofollow">http://openwetware.org/wiki/CcdB</A>, CcdB (seen on 10.10.10).
</LI></UL><UL><LI>YC Jung <I>et al.</I> (2004) <A href="http://www.springerlink.com/content/wmq06kua5qkma1au/" class="external text" rel="nofollow"><I>Mutants of </I>Agrobacterium tumefaciens virG<I> Gene That Activate Transcription of </I>vir<I> Promoter in </I>Escherichia coli'</A><B>, Current Microbiol 49:334-340.</B></LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Bielefeld-Germany/Results/Used">http://2010.igem.org/Team:Bielefeld-Germany/Results/Used</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Results/Used" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Results/Used" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/Used&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Results/Used&amp;oldid=197537" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>