<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Project_Theory"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Project/Theory">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Project/Theory&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project/Theory&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project/Theory&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Project/Theory" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Project/Theory</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="sliderWrap"><DIV id="slider"><DIV id="openCloseWrap" style="height:110px;"><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany">Start</A></LI></DIV><DIV class="navColumn" style="width:13%;"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team">Team</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Description">Description</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Supervisors">Supervisors</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Bielefeld">University</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/BielefeldCity">Bielefeld</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Motivation">Motivation</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Team/Acknowledgements">Acknowledgements</A></LI></UL></DIV><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project">Project</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Application">Application</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Approach">Approach</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Theory">Theory</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Protocols">Protocols</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Model">Model</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Project/Outlook">Outlook</A></LI></UL></DIV><DIV class="navColumn" style="width:12%;"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results">Results</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Characterization">Characterization</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Used">Used BBs</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Submitted">Submitted BBs</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Sequencing">Sequencing</A></LI></UL></DIV><DIV class="navColumn" style="width:12%;"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Safety">Safety</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Safety#Biosecurity">Biosecurity</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Safety#Biosafety">Biosafety</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Safety#Risk_Assesement_.28english.29">Risk Assesement</A></LI></UL></DIV><DIV class="navColumn" style="width:13%;"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Public_Relations">Public Relations</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Public_Relations/Fotos">Fotos</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Public_Relations/Press">Press</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Public_Relations/Public_Discussion">Public Discussion</A></LI></UL></DIV><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Judging">Judging</A></LI><UL><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Judging/BioBricks">BioBricks</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Judging/Helping">Helping other teams</A></LI><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Judging/Comments">Comments on iGEM</A></LI></UL></DIV><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Sponsors">Sponsors</A></LI></DIV><DIV class="navColumn"><LI><A href="https://2010.igem.org/Team:Bielefeld-Germany/Contact">Contact</A></LI></DIV></DIV></DIV><DIV id="sliderController" class="sliderController">MENU</DIV></DIV><DIV id="checkParameter" style="margin-right:250px;">http://igem-bielefeld.de/img/banner_chili.jpg</DIV><DIV id="subnavigation"><LI><A href="/Team:Bielefeld-Germany/Project">Project</A></LI><LI><A href="/Team:Bielefeld-Germany/Project/Application">Application</A></LI><LI><A href="/Team:Bielefeld-Germany/Project/Approach">Approach</A></LI><LI><A href="/Team:Bielefeld-Germany/Project/Theory">Theory</A></LI><LI><A href="/Team:Bielefeld-Germany/Project/Protocols">Protocols</A></LI><LI><A href="/Team:Bielefeld-Germany/Project/Model">Model</A></LI><LI><A href="/Team:Bielefeld-Germany/Project/Outlook">Outlook</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Agrobacterium_tumefaciens"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Agrobacterium tumefaciens</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Native_receptor"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Native receptor</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#VirA_receptor_structure"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">VirA receptor structure</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Phenolic_Compounds"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Phenolic Compounds</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Inducing_enhancers"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Inducing enhancers</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Subcloning_into_E._coli_and_receptor_function_in_new_host"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Subcloning into E. coli and receptor function in new host</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-8"><A href="#Read_out_system"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Read out system</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Firefly_Luciferase"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Firefly Luciferase</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Output-signal_amplification_by_Sensitivity_Tuner_implementation"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Output-signal amplification by Sensitivity Tuner implementation</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Receptor_modification_strategy"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Receptor modification strategy</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Random_mutagenesis_by_error-prone_PCR_.28EP-PCR.29"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Random mutagenesis by error-prone PCR (EP-PCR)</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Directed_mutagenesis"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Directed mutagenesis</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#Screening_system"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Screening system</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Development_of_a_high-troughput_screening"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Development of a high-troughput screening</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Determination_of_minimal_inhibitory_concentration_.28MIC.29_of_kanamycin"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Determination of minimal inhibitory concentration (MIC) of kanamycin</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Primary_selection_of_virA_variants_with_novel_binding_properties"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">Primary selection of virA variants with novel binding properties</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Quantitative_analysis_of_virA_variants_after_induction_with_novel_substances"><SPAN class="tocnumber">6.4</SPAN><SPAN class="toctext">Quantitative analysis of virA variants after induction with novel substances</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#References"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Introduction"> Introduction </SPAN></H1><P>In our iGEM project we tried to create an <I>Escherichia coli</I> cell which is capable to sense capsaicin in a complex sample and report the concentration with a luciferase light signal. We combined a native receptor system of <I>Agrobacterium tumefaciens</I> with the readout system of the firefly luciferase. The native receptor senses the phenolic compound acetosyringone. For this reason we used directed evolution to modify the binding region of the native receptor to generate a new capsaicin receptor because of the chemical similarities of acetosyringone and capsaicin. We established a screening system based on antibiotic concentration gradients to screen the newly generated receptors. In our <I>E. coli</I> acetosyringone sensing system several parts derived from different organisms were assembled. The readout system taken from firefly, luciferase, the receptor sensing system derived from <I>A. tumefaciens</I> and sensitivity tuners consist of phage DNA.
</P><H1><SPAN class="mw-headline" id="Agrobacterium_tumefaciens"><I>Agrobacterium tumefaciens</I></SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 1: Image by Martha Hawe</B></DIV></DIV></DIV><P>
The model organism <I>A. tumefaciens</I> is a soil bacterium and can be found at nearly every place in the world. <I>Agrobacteria</I> became known as a phyto-pathogen leading to the crown gall disease in dicotyledonous species (<A href="http://www.springerlink.com/content/hm17520m287ht766/" class="external text" rel="nofollow">DeCleene M and DeLay J, 1976</A>). The infection is caused by a gene transfer system located on an extrachromosomal element, the Ti-plasmid. Furthermore, the infection can be divided into several steps: The first step is the localisation of the hurt plant by the bacteria. Predominantely <I>A. tumefaciens</I> senses phenolic compunds from hurt plants, but also aldose monosaccharides, low pH and low phosphate (<A href="http://www.annualreviews.org/doi/abs/10.1146/annurev.phyto.41.052002.095701?journalCode=phyto" class="external text" rel="nofollow">Palmer AG. <I>et al.</I>2004</A>; <A href="http://mmbr.asm.org/cgi/content/abstract/69/1/155" class="external text" rel="nofollow">Brencic A and Winans SC, 2005</A>). When <I>A. tumefaciens</I> recognizes phenols with the VirA receptor, a signal transduction cascade is initiated leading to the expression of virulence genes. The next step is a physical interaction with the host plant. A type three secretion system is responsible for the DNA transfer of the Ti-plasmid from the bacterium into the host. The DNA is translocated to the nucleus, leading to the gene expression and the production of opin. <I>A. tumefaciens</I> uses the reprogrammed plant cells for metabolite production and therefore as a nutrient supplier.
</P><P>For biotechnological purposes the Ti-plasmid was disharmed. Instead of the native transfer region (T-region) and a gene of interest could be easily introduced into the Ti-plasmid. <I>Agrobacterium</I>-mediated DNA transfer is one of the most commonly used techniques of plant transformation (<A href="http://people.uleth.ca/~alicja.ziemienowicz/extra/Ziemienowicz_ABP2001.pdf" class="external text" rel="nofollow">Ziemienowicz A, 2001</A>).
</P><H1><SPAN class="mw-headline" id="Native_receptor"> Native receptor </SPAN></H1><P><I>A. tumefaciens</I> needs a precise recognition system for potential hosts to gain an evolutionary advance. The native sensing system is a two-component phospho-relay system in which VirA is a transmembrane-bound sensor while VirG is the intracellular response regulator (<A href="http://www.biomedcentral.com/content/pdf/gb-2002-3-10-reviews3013.pdf" class="external text" rel="nofollow">Wolanin PM <I>et al.</I>, 2002</A>). The two genes for the sensing system are <I>virA</I> and <I>virG</I> (<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1166964/" class="external text" rel="nofollow">Stachel SE and Nester EW, 1986</A>) which are constitutively expressed at a basal level. VirA is a histidine kinase. An autophosphorylation occurs at the His-474 residue, after sensing the phenol 3,5-dimethoxyacetophenone,  acetosyringone (<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC208549/?pageindex=1&amp;tool=pmcentrez" class="external text" rel="nofollow">Huang Y <I>et al.</I>, 1990</A>Â ; <A href="http://jb.asm.org/cgi/content/abstract/172/2/531" class="external text" rel="nofollow">Jin SG <I>et al.</I>, 1990a</A>). In the next step in the signal transduction cascade, the phosphorylated VirA leads to the transfer of the phosphate to Asp-52 residue of VirG (<A href="http://jb.asm.org/cgi/content/abstract/172/2/531" class="external text" rel="nofollow">Jin SG <I>et al.</I>, 1990a</A>Â ; <A href="http://jb.asm.org/cgi/content/abstract/172/9/4945" class="external text" rel="nofollow">Jin SG <I>et al.</I>, 1990b</A>Â ; <A href="http://nar.oxfordjournals.org/content/18/23/6909.short" class="external text" rel="nofollow">Pazour GJ and Das A, 1990</A>). VirG is the response regulator of the two-component system <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1082791/" class="external text" rel="nofollow">(Brencic A, Winans SC, 2005)</A> and acts as a transcription factor. Hence it binds to the virulence box (vir Box) containing promoters, for example the <I>virB</I> promoter (<A href="http://jb.asm.org/cgi/content/abstract/172/2/531" class="external text" rel="nofollow">Jin SG <I>et al.</I>, 1990a</A>Â ; <A href="http://jb.asm.org/cgi/content/abstract/172/3/1241" class="external text" rel="nofollow">Pazour GJ and Das A, 1990</A>).
</P><H2><SPAN class="mw-headline" id="VirA_receptor_structure"> VirA receptor structure </SPAN></H2><P>The VirA receptor consists of 829 amino acids and is a transmembrane protein in the inner menbrane of <I>A. tumefaciens</I> (<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC401051/" class="external text" rel="nofollow">Melchers LS, 1989</A>). VirA spans the inner membrane, with two transmembrane domains, a large periplasmic region, and a large C-terminal cytoplasmic domain (<A href="http://jb.asm.org/cgi/content/abstract/176/11/3242" class="external text" rel="nofollow">Banta LM, 1994</A>). VirA directly senses the phenolic compounds for vir activation (<A href="http://linkinghub.elsevier.com/retrieve/pii/S0378111996003289" class="external text" rel="nofollow">Lee YW <I>et al.</I>, 1996</A>). Therefore the linker domain is essential for induction by phenolic compounds (<A href="http://jb.asm.org/cgi/content/abstract/174/21/7033" class="external text" rel="nofollow">Chang CH and Winans SC., 1992</A>). The linker region is located in the cytosolic site at position 280 to 414 (<A href="http://linkinghub.elsevier.com/retrieve/pii/S0378111996003289" class="external text" rel="nofollow">Lee YW <I>et al.</I>, 1996</A>). This region between the amino acids 283 and 304 was highly conserved in four different strains of <I>Agrobacterium</I>, and therefore likely to serve as the receptor region for the phenolic inducers which are common to all four strains (<A href="http://www.springerlink.com/index/G548R7745685341P.pdf" class="external text" rel="nofollow">Turk SC <I>et al.</I>, 1994</A>).
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 2: The structur of virA (Melchers et al., 1989)</B></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 3: The funtional parts of the virA receptor (Lee et al., 1996)</B></DIV></DIV></DIV></DIV><P>
Chang and Winnans (1992) revealed in their studies the parts of the VirA receptor which are essential for the signal transduction (<A href="http://jb.asm.org/cgi/content/abstract/174/21/7033" class="external text" rel="nofollow">Chang CH and Winans SC., 1992</A>). A structured model for different inducing conditions are shown in figure 4.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 4Â : The different binding conditions (Chang and  Winans, 1992)</B></DIV></DIV></DIV></DIV><DIV style="font-size:16px; font-weight:bold;"><A href="/Team:Bielefeld-Germany/Project/Approach#The_Approach">For information about modulation strategy click here</A></DIV><H2><SPAN class="mw-headline" id="Phenolic_Compounds"> Phenolic Compounds </SPAN></H2><P>The ligand receptor interaction between acetosyringone and VirA is based on the interaction of several chemical groups. First of all the hydroxylated aromat is essential. Methoxy groups in the <I>ortho</I> position of the phenol play a crucial role in the signaling as well. It should be mentioned that dimethoxy compounds have a higher activity than monomethoxy compounds. The acetyl and alkyl groups in <I>para</I> position enhance the binding affinity. VirA activating compounds must have two methoxy groups in <I>ortho</I> position and an additional carbonyl group on the R3 chain. The potential capacity of the group <I>para</I> to the phenolic hydroxyl group is associated with higher activities. Moreover the chirality at this carbon center is critical for the inducing activity (<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC372852/?pageindex=1&amp;tool=pmcentrez" class="external text" rel="nofollow">Winans SC, 1992</A>).
Regarding to the proton transfer model of Hess <I>et al.</I> (1996) the VirA activator transfers a proton to the basic area receptor binding site. The allosteric change leads to the phosphotransfer and the signaltransduction (<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC52402/?pageindex=1&amp;tool=pmcentrez" class="external text" rel="nofollow">Hess KM <I>et al.</I>, 1991</A>).
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:192px;"><DIV class="thumbcaption"><B>Figure 5: Chemical structure of phenol </B></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 6: The proton transfer model according to Hess et al., 1991</B></DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 7: Compounds with a structural similarity, which induce virA Winans, 1992</B></DIV></DIV></DIV></DIV><DIV style="font-size:16px; font-weight:bold;"><A href="/Team:Bielefeld-Germany/Project/Approach#The_Approach">See more possible compounds by clicking here</A></DIV><H2><SPAN class="mw-headline" id="Inducing_enhancers"> Inducing enhancers </SPAN></H2><P>The sensitivity of this system is highly enhanced when additional aldose monosacchardic suggars occur in the environment of <I>Agrobacterium</I>. The sugar binding protein ChvE interacts with the VirA receptor, leading to a much stronger <I>vir</I> gene expression (<A href="http://jb.asm.org/cgi/content/abstract/176/11/3242" class="external text" rel="nofollow">Banta LM, <I>et al.</I>, 1994</A>).
</P><H2><SPAN class="mw-headline" id="Subcloning_into_E._coli_and_receptor_function_in_new_host"> Subcloning into <I>E. coli</I> and receptor function in new host </SPAN></H2><P>In our project we decided to work with <I>E. coli</I> instead of <I>A. tumefaciens</I>. The transcription procedure in <I>E. coli</I> is very similar to <I>A. tumefaciens</I> but not complete homolog. In <I>E. coli</I> the  <I>rpoA</I> gene - encoding the Î±-subunit of RNA polymerase in <I>A. tumefaciens</I> - is not present but essential for the transcription of a <I>virB</I> promoter-driven genes (<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC103583/?report=abstract&amp;tool=pmcentrez" class="external text" rel="nofollow">Lohrke SM <I>et al.</I>, 1990</A>) For this reason it was necessary to subclone a modified <I>virG</I> gene that is capable to be trancribed by the <I>E. coli</I> expression system.
</P><P>Yong-Chul <I>et al.</I> (2004) described VirG mutants that are capable of expressing the <I>virB</I> promoter-driven genes in <I>E. coli</I> without the requirement for the RpoA from <I>A. tumefaciens</I>, suggesting that the<I>virG</I> mutants are able to interact with the transcription system of <I>E. coli</I> (<A href="http://www.springerlink.com/content/wmq06kua5qkma1au/fulltext.pdf" class="external text" rel="nofollow">Yong-Chul J <I>et al.</I>, 2004</A>).
In VirG the amino acid at position 56 is likely to play a key role in the interaction with the RpoA of <I>E. coli</I>. 
Regarding to Yong-Chul J <I>et al.</I> (2004) we used <I>virG</I> mutants, with amino acid substitutions of G56V and I77V that are capable of activating <I>vir</I> genes in <I>E. coli</I> in response to inducer acetosyringone in a VirA-dependent manner.
</P><H1><SPAN class="mw-headline" id="Read_out_system"> Read out system </SPAN></H1><H2><SPAN class="mw-headline" id="Firefly_Luciferase"> Firefly Luciferase</SPAN></H2><P>We are using luciferase as the read out system. The luciferase was originally extracted from the firefly <I>Photinus pyralis</I>. It is one of the best studied and characterized read out systems. The luciferase enzyme catalyses the chemical reaction from its substrate luceferin to oxyluciferin and light (<A href="http://mcb.asm.org/cgi/reprint/7/2/725" class="external text" rel="nofollow">De Wet <I>et al.</I> 1986</A>). Figure 8 shows the reaction in detail.
</P><P>We used luciferase as the read out system because it causes only slight negligible noise, hence the signal to noise ration is excellent.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 8: Firefly luciferase reaction</B> (De Wet <I>et al.</I>1986)</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Output-signal_amplification_by_Sensitivity_Tuner_implementation">Output-signal amplification by Sensitivity Tuner implementation</SPAN></H2><P>Using a standard, inducible promoter with reporter system often results in weak reporter expression. So difficulties in quantification can occur. The quantification can be enhanced by an amplification of the transcription rate of the desired reporter genes. Such an amplification can be realized by using a so called <I>sensitivity tuner</I> device. This takes place as promoter induction upregulates a phage activator, which binds to a phage promoter upstream of a reporter gene. As result, a PoPs input (Inducer) generates a PoPs output at a higher signal. PoPs is equivalent to the flow of RNA polymerase molecules along DNA (<A href="http://jb.asm.org/cgi/content/abstract/178/19/5668" class="external text" rel="nofollow">Julien and Calendar, 1996</A>Â ; <A href="https://2007.igem.org/Cambridge/Amplifier_project" class="external text" rel="nofollow">iGEM Team Cambridge, 2007</A>).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 9: Gene sequence of final test construct including Sensitivity Tuner elements.</B></DIV></DIV></DIV></DIV><P><B>Purpose of Sensitivity Tuner application</B></P><P>We presumed weak expression rates of our reporter luciferase indicated by pretesting the native system &lt;partinfo&gt;K389015&lt;/partinfo&gt;. For having a broader range of quantification for our prototype test system, an amplification device was implemented. 
For amplifying the output signal of luciferase induced by acetosyringone, three sensitivity tuners, distinguished by the amplification factor, were combined with our detection system. To modify the sensitivity tuner for our purpose we took BioBricks with amplification factors from 10 (&lt;partinfo&gt;I746380&lt;/partinfo&gt;) to 35 (&lt;partinfo&gt;I746390&lt;/partinfo&gt;), removed the pBAD/araC promoter (&lt;partinfo&gt;I0500&lt;/partinfo&gt;) and GFP read out (&lt;partinfo&gt;E0040&lt;/partinfo&gt;) and replaced it by the reporter gene luciferase &lt;partinfo&gt;K389004&lt;/partinfo&gt;(Figure 9). This enhanced luciferase BioBrick was assembled to the VirA/G signaling system BioBrick. The benefits of luciferase as reporter gene instead of GFP are a broader range of measurement, higher sensitivity and low half-live making cinetic tests possible (<A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6W9V-4F031H9-30&amp;_user=10&amp;_coverDate=01%2F31%2F1989&amp;_rdoc=1&amp;_fmt=high&amp;_orig=search&amp;_origin=search&amp;_sort=d&amp;_docanchor=&amp;view=c&amp;_searchStrId=1514624813&amp;_rerunOrigin=google&amp;_acct=C000050221&amp;_version=1&amp;_urlVersion=0&amp;_userid=10&amp;md5=ee400628b119490fcdc44ccdd856c4e8&amp;searchtype=a" class="external text" rel="nofollow">Williams <I>et al.</I>1989</A>).
</P><P>For test results click <A href="https://2010.igem.org/Team:Bielefeld-Germany/Results/Tests#BBa_K389421.2C_BBa_K389422.2C_BBa_K389423:_Sensitivity_Tuner_amplified_Vir-test_system" class="external text" rel="nofollow">Sensitivity Tuner amplified Vir-test system</A></P><H1><SPAN class="mw-headline" id="Receptor_modification_strategy"> Receptor modification strategy </SPAN></H1><P>The smartest way of receptor-modification is initiated by a silico approach based on a 3D structure of the native receptor. Followed by primer mutagenesis of the computantional gained results a precise adapted peptide emerges. This concept has been proven by <A href="http://www.nature.com/nature/journal/v423/n6936/abs/nature01556.html" class="external text" rel="nofollow">Looger <I>et al.</I>, 2003</A>.
</P><P>Because of time limitations within the iGEM competition and a lack of biological data in literature - no x-ray cristalography structure data for VirA linker region available- this strategy was not applicable. 
Therefore we developed two different stragies in our MARSS project:
</P><P>The first strategy was an <U> error prone PCR </U> approach by building a mutant data base for selection. The second strategy was a <U> primer mutagenesis based </U> approach.
</P><H2><SPAN class="mw-headline" id="Random_mutagenesis_by_error-prone_PCR_.28EP-PCR.29"> Random mutagenesis by error-prone PCR (EP-PCR)</SPAN></H2><P>In order to detect novel substances (<I>e.g.</I> capsaicin) with the VirA receptor, the first step was to create a mutagenised library of <I>virA</I> variants, which could subsequently be screened for new binding characteristics. Plenty of different strategies for mutagenesis of DNA are known, including the use of nucleotide analogues, bacteria containing mutator genes, the mutagenesis with UV light or chemicals and inaccurate PCR (<A href="http://genome.cshlp.org/content/2/1/28.short" class="external text" rel="nofollow">Cadwell RC and Joyce GF, 1992</A>).
</P><P>When designing our strategy we rejected the use of bacteria strains with high mutation rates, since the changes in base sequence would occur all over the transformed plasmids. Thereby some mutations would also take place in the backbone of the plasmid, or might even been found in the standardized BioBrick prefix and suffix. We also excluded the possibility of using UV light or mutagenic chemicals, due to reasons of safety and minimizing the exposure of toxic substances.
</P><P>In our experiment we wanted to alter only the part of the plasmid coding for the VirA receptor, while using a not harmful and thereby safe technique. Thus, our method of choice was inaccurate PCR that allows the exclusive variations of a distinct region of a plasmid, which is defined by the location of the upstream and downstream primers. This mutagenic method of PCR, called error-prone PCR (EP-PCR) has been described and improved a lot in scientific community (<A href="http://www.springerlink.com/content/r62q360t82764508/#section=746282&amp;page=1" class="external text" rel="nofollow">McCullum EO <I>et al.</I>, 2010</A>).
</P><P>The basic principle of this technique uses the natural high infidelity of the <I>taq</I> DNA polymerase, which can even be increased by special changes in buffer conditions compared to standard PCR. These alterations may include the unequal distribution of dNTPs (5 mM purines, 25 mM pyrimidines) as well as an increased amount of MgCl<SUB>2</SUB> and the addition of MnCl<SUB>2</SUB>. The total rate of base exchange can be adjusted by the number of PCR cycles, since mutations will accumulate during the exponential amplification of the sequence (<A href="http://onlinelibrary.wiley.com/doi/10.1002/0471142727.mb0803s51/full" class="external text" rel="nofollow">Wilson DS and Keefe AD, 2000</A>). The experimental conditions of the performed error-prone PCR are described in the section âprotocolsâ.
</P><H2><SPAN class="mw-headline" id="Directed_mutagenesis"> Directed mutagenesis </SPAN></H2><P>After the identification of the 100 amino acids linker region responsible for VirA ligand binding (<A href="/Team:Bielefeld-Germany/Project/Theory#VirA_receptor_structure" title="Team:Bielefeld-Germany/Project/Theory">Compare VirA-Receptor</A>) we compared this region with well characterized capsaicin receptors derived from animal model organisms (TRPV1). The conserved receptor region is shown in the figure beneath.
</P><P>The species-specific sensitivity of TRPV1 can be ascribed to about eight amino acids in the vicinity of TM3 (<A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6WSN-4C5H6M5-F&amp;_user=2459438&amp;_coverDate=02%2F08%2F2002&amp;_rdoc=1&amp;_fmt=high&amp;_orig=search&amp;_origin=search&amp;_sort=d&amp;_docanchor=&amp;view=c&amp;_acct=C000057302&amp;_version=1&amp;_urlVersion=0&amp;_userid=2459438&amp;md5=93a9e63f48d785a0b400131a058ee269&amp;searchtype=a" class="external text" rel="nofollow">Jordt an Julius 2002</A>).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:557px;"><DIV class="thumbcaption"><B> Figure 10: Molecular determinants of species-specific vanilloid sensitivity. Sequence alignment of rat (top), human (middle) and chicken (bottom) VR1 within the TM3-4 region is shown. Conserved residues are indicated by black background. The chimera V3/C contains a minimal segment of rat VR1 that is sufficient to confer vanilloid sensitivity (<A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6WSN-4C5H6M5-F&amp;_user=2459438&amp;_coverDate=02%2F08%2F2002&amp;_rdoc=1&amp;_fmt=high&amp;_orig=search&amp;_origin=search&amp;_sort=d&amp;_docanchor=&amp;view=c&amp;_acct=C000057302&amp;_version=1&amp;_urlVersion=0&amp;_userid=2459438&amp;md5=93a9e63f48d785a0b400131a058ee269&amp;searchtype=a" class="external text" rel="nofollow">Jordt and Julius, 2002</A>)</B>.</DIV></DIV></DIV></DIV><P>We further aligned the TM3 region of the TRPV1 receptor to the ligand binding region of the native VirA receptor from <I>Agrobacterium</I> by the use of the tool <B>MU</B>ltiple <B>S</B>equence <B>C</B>omparison by Log-Expectation (<A href="http://www.ebi.ac.uk/Tools/muscle/index.html" class="external text" rel="nofollow">Muscle</A>). 
</P><P><A href="/Team:Bielefeld-Germany/Project/Approach#Directed_mutagenesis" title="Team:Bielefeld-Germany/Project/Approach"><B>For detailed approach of directed mutagenesis for VirA_mut1 and VirA_mut2 click here</B></A></P><H1><SPAN class="mw-headline" id="Screening_system"> Screening system </SPAN></H1><H2><SPAN class="mw-headline" id="Development_of_a_high-troughput_screening">Development of a high-troughput screening</SPAN></H2><P>The screening of randomly mutagenised genes for a desired function or application is always a very time-consuming procedure (<A href="http://www.ncbi.nlm.nih.gov/pubmed/1496376" class="external text" rel="nofollow">Beaudry and Joyce, 1992</A>). It requires a huge amount of material and might takes several months or even years to result in a promising new version of a gene (<A href="http://mbe.oxfordjournals.org/content/17/7/1050.long" class="external text" rel="nofollow">Hanczyc and Dorit, 2000</A>). As we faced the challenge to modify the VirA receptor in only a few weeks we designed a strategy for a fast high-throughput screening by using subsequent steps of different read out systems and a strategy with two different plasmids.
</P><P>In the first step after mutagenesis of <I>virA</I> it is necessary to separate thousands of transformants with minor or unwanted changes in the <I>virA</I> gene, from few bacteria that included interesting <I>virA</I> variants. Thus, we wanted to construct our system to lead in the expression of a kanamycin resistance after the induction of the VirA receptor, enabling the quick exclusion of all unwanted <I>virA</I> variants.
</P><P>For that purpose a kanamycin resistance cassette should be set under control of the <I>virB</I> promoter, leading in the expression of aminoglycoside phosphotransferase (APH) that can inactivate kanamycin. The mode of inactivation is the transfer of the y-phosphate from ATP to the hydroxyl group at C3 of the antibiotic (<A href="http://www.bioscience.org/1999/v4/d/wright/wright.pdf" class="external text" rel="nofollow">Wright and Thompson, 1999</A>). This phosphorylation results in the loss of binding capacity of the aminoglycoside to the 30S subunit of bacterial ribosomes, which would lead to inhibition of protein synthesis without the presence of the APH (<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1365070/pdf/brjclinpharm00008-0014.pdf" class="external text" rel="nofollow">Begg EJ and Barclay ML, 1995</A>).
</P><P>As indicated by the first results, the reporter genes under control of the <I>virB</I> promoter showed a slight but measureable expression without any induction of VirA with acetosyringone. This basal transcription resulted in the growth of bacteria without induction of VirA at normal working concentrations of kanamycin of 25 to 50 Âµg mL<SUP>-1</SUP> (<A href="http://books.google.de/books?hl=de&amp;lr=&amp;id=9mO2Fx0CuEYC&amp;oi=fnd&amp;pg=PA4&amp;dq=Molecular+Cloning:+A+Laboratory+Manual,+Vol+1.+&amp;ots=Ctw-SlcSLm&amp;sig=FvweaKb_qiUxClNNiRfdwpmbqdo#v=onepage&amp;q&amp;f=false" class="external text" rel="nofollow">Sambrook J and Russell DW, 2001</A>).
</P><P>Concludingly, prior to the screening experiments it was necessary to adjust the concentration of kanamycin, which inhibits the growth of uninduced bacteria, while allowing bacteria to grow when an appropriate inductor was present and able to activate VirA. This analysis was performed using the method of determination of minimal inhibitory concentrations (MIC) as described below.
</P><H2><SPAN class="mw-headline" id="Determination_of_minimal_inhibitory_concentration_.28MIC.29_of_kanamycin">Determination of minimal inhibitory concentration (MIC) of kanamycin</SPAN></H2><P>There are several ways to investigate the susceptibility of bacteria to inhibiting drugs like antibiotics. Nevertheless, the result of all these tests is the amount of an assayed substance that inhibits visible growth of the bacteria, called the minimal inhibitory concentration (MIC). The most common way of determination is to grow bacteria in liquids with several concentrations of the inhibiting drug. This procedure is chosen mostly, since many different conditions can be measured at the same time by using microtiterplates (<A href="http://www.nature.com/nprot/journal/v3/n2/abs/nprot.2007.521.html" class="external text" rel="nofollow">Wiegand I <I>et al.</I>, 2008</A>).
</P><P>As we planned not to use the kanamycin in liquid culture but in LB-Agar, we chose to determine the MIC at the same conditions as the desired experiment. Therefore, we planned to construct <I>E. coli</I> inhabiting the native VirA/G signaling system and a kanamycin resistance read out and plated a small volume in different dilutions on LB-Agar without kanamycin. The grown colonies could then be transferred to agar plates with rising concentrations of kanamycin using replica plating. By counting the colonies it should be possible to calculate the percentage of colonies that could withstand each kanamycin concentration. This experiment should be carried out with and without acetosyringone to determine a kanamycin concentration induced <I>E. coli</I> could withstand, while the same population of bacteria dies without the presence of acetosyringone.
</P><H2><SPAN class="mw-headline" id="Primary_selection_of_virA_variants_with_novel_binding_properties">Primary selection of <I>virA</I> variants with novel binding properties</SPAN></H2><P>After the determination of the MIC the screening for <I>virA</I> variants with new binding properties could be started. The aim of this screening is to find versions of <I>virA</I> that can be induced by one of the tested substances capsaicin, homovanillic acid, dopamine and 3-O methyldopamine.
</P><P>For that purpose one should transform the mutagenised variants of <I>virA</I> to <I>E. coli</I> and plate the bacteria on LB-agar with the determined kanamycin concentration. At the same time a mixture of all mentioned substances should be present in the agar. All bacteria including a <I>virA</I> variant that is activated by at least one of the substances will grow on the selective agar, since it expresses the kanamycin resistance. With this step it is thereby possible to select thousands of bacteria with unwanted versions of <I>virA</I> from few individuals with wanted binding properties.
</P><P>At this point it must be mentioned that some grown colonies might still be non-induced and false positive results. As the <I>virA</I> gene has been randomly changed before, it is possible that some variants occur where the receptor is always active. This would lead to a constitutive expression of the reporter gene and thereby a high level of kanamycin resistance.
To exclude those false positive clones, the bacteria should be tested whether they are only resistance to the MIC of kanamycin when one of the tested substances is present. Every colony that can withstand the high kanamycin concentration without any inductor includes a constitutive version of virA and should be discarded in further analysis.
</P><H2><SPAN class="mw-headline" id="Quantitative_analysis_of_virA_variants_after_induction_with_novel_substances">Quantitative analysis of <I>virA</I> variants after induction with novel substances</SPAN></H2><P>Should we find some bacteria that respond to the presence of the tested substances (capsaicin, homovanillic acid, dopamine and 3-O methyldopamine) by growing on the MIC of kanamycin, it is desirable to quantify the induction. For that purpose it is appropriate to change the read out system from kanamycin resistence to luciferase expression. This complex and time consuming task can easily be achieved without any cloning step, when using the advantage of our two plasmid system.
</P><P>After primary selection each bacteria includes two plasmids with different origins of replication (oris). The plasmid with the <I>virA</I> is in a common &lt;partinfo&gt;pSB1AT3&lt;/partinfo&gt; backbone with a ColE1 ori. Contrary to that the read out plasmid with KanR has a special ori, named R6K, which can only amplify in <I>E. coli</I> strains that express the gene <I>pir</I> to produce the so called Pir protein. Most of the strains used in laboratory are pir<SUP>-</SUP> but few (<I>e.g</I> EC100D) are pir<SUP>+</SUP> (<A href="http://www.ncbi.nlm.nih.gov/pubmed/17383678" class="external text" rel="nofollow">Bowers et al., 2007</A>).
</P><P>The setup of the different oris was chosen to separate both plamids at this experimental stage. To change the read out system from KanR to luciferase, one just needs to perform two transformations and isolations of plasmids. In the first step plasmids are isolated from colonies with positive binding properties to one of the tested substances. This mixture of plasmids with ColE1 and R6K oris is then transformed to a pir<SUP>-</SUP> strain (<I>e.g.</I> TOP10). In the following only the plasmid with ColE1 ori will be amplified during the growth of the transformants, leading to pure plasmids with <I>virA</I> when plasmids are isolated for a second time. In the last step this isolated DNA can be transformed to bacteria including another read out plasmid (<I>e.g.</I> with luciferase).
</P><H1><SPAN class="mw-headline" id="References"> References </SPAN></H1><UL><LI>Banta LM, Joerger RD, Howitz VR, Campbell AM, Binns AN., (1994) <I>Glu-255 outside the predicted ChvE binding site in VirA is crucial for sugar enhancement of acetosyringone perception by Agrobacterium tumefaciens</I>, J Bacteriol. 176(11):,3242-9. 
</LI></UL><UL><LI>Beaudry AA, Joyce GF (1992) <I>Directed evolution of an RNA enzyme</I>, Science, 257(5070):, 635-41.
</LI></UL><UL><LI>Begg EJ, Barclay ML (1995) <I>Aminoglycosides â 50 years on</I>, Br J clin Pharmac, 39:, 597-603.
</LI></UL><UL><LI>Bowers LM, KrÃ¼ger R, Filutowicz M (2007), <I>Mechanism of origin activation by monomers of R6K-encoded pi protein</I>, J Mol Biol, 368(4):, 928-938.
</LI></UL><UL><LI>Brencic A, Winans SC (2005), <I>Detection of and response to signals involved in host-microbe interactions by plant-associated bacteria</I>, Microbiol Mol Biol, Rev 69:, 155â194. 
</LI></UL><UL><LI>Cadwell RC and Joyce GF (1992), <I>Randomization of genes by PCR mutagenesis</I>, Genome Res., 28-33.
</LI></UL><UL><LI>Chang CH, Winans SC., (1992) <I>Functional roles assigned to the periplasmic, linker, and receiver domains of the Agrobacterium tumefaciens VirA protein</I>, J Bacteriol., 174(21):, 7033-9. 
</LI></UL><UL><LI>DeCleene M, DeLay J (1976), <I>The host range of crown gall</I>, Bot Rev 42, 389â466 
</LI></UL><UL><LI>De Wet JR, Wood KV, De Luca M, Helinski DR and Subramani S (1987), <I>Firefly Luciferase Gene: Structure and Expression in Mammalian Cells</I>, Molecular and Cell Biolog, Feb 1987, 725-737
</LI></UL><UL><LI>Hanczyz MM, Dortit RL (2000), <I>Replicability and Recurrence in the Experimental Evolution of a Group I Ribozyme</I>, Mol Biol Evol, 17 (7):, 1050-1060
</LI></UL><UL><LI>Hess KM, Dudley MW, Lynn DG, Joerger RD, Binns AN., (1991) <I>Mechanism of phenolic activation of Agrobacterium virulence genes: development of a specific inhibitor of bacterial sensor/response systems</I>, Proc. Natl. Acad. Sci., USA 88:, 7854â58
</LI></UL><UL><LI>Huang Y, Morel P, Powell B, Cado CI (1990), <I>VirA, a corregulator of Ti-specific virulence genes, is phosphorylated in vitro</I>, J Bacteriol 172:, 1142â1144
</LI></UL><UL><LI>Jin S, Roitsch T, Ankenbauer RG, Gordon MP, Nester EW (1990), <I>The VirA protein of Agrobacterium tumefaciens is autophosphorylated and is essential for vir gene regulation</I>, J Bacteriol 172:, 525â530 
</LI></UL><UL><LI>Jin SG, Prusti RK, Roitsch T, Ankenbauer RG, Nester EW (1990a), <I>Phosphorylation of the VirG protein of Agrobacterium tumefaciens by the autophosphorylated VirA protein: Essential role in biological activity of VirG</I>, J Bacteriol, 172:, 4945â4950 
</LI></UL><UL><LI>Jin SG, Roitsch T, Christie PJ, Nester EW (1990b), <I>The regulatory VirG protein specifically binds to a cis-acting regulatory sequence involved in transcriptional activation of Agrobacterium tumefaciens virulence genes</I>, J Bacteriol, 172:, 531â537 
</LI></UL><UL><LI>Jordt SE, Julius D (2002), <I>Molecular Basis for Species-Specific Sensitivity to âHotâ Chili Peppers</I>, Cell, 108:, 421-430
</LI></UL><UL><LI>Kyunghee Lee (1996), <I>A structure-based activation model of phenol-receptor protein interactions. </I></LI></UL><UL><LI>Lee YW, Jin S, Sim WS, Nester EW (1996), <I>The sensing of plant signal molecules by Agrobacterium: genetic evidence for direct recognition of phenolic inducers by the VirA protein</I>, Gene, 179(1):, 83-8. 
</LI></UL><UL><LI>Lin, Y, Gao R, Binns A, Lynn D (2008), <I>Bacterial signal transduction: networks and drug targets</I> Adv Exp Med Biol., 161â177.
</LI></UL><UL><LI>Lohrke SM, Nechaev S, Yang H, Severinov K, Jin SJ (1999), <I>Transcriptional activation of Agrobacterium tumefaciens virulence gene promoters in Escherichia coli requires the A. tumefaciens RpoA gene, encoding the alpha subunit of RNA polymerase</I>, J Bacteriol, 181:, 4533â4539 
</LI></UL><UL><LI>Looger L, Dwyer MA, Smith JJ and Hellinga HW  (2003), <I>Computational design of receptor and sensor proteins with novel functions</I>, Nature, 423:, 185-190
</LI></UL><UL><LI>McCullen CA, Binns AN  (2006), <I>Agrobacterium tumefaciens and Plant Cell Interactions and Activities Required for Interkingdom Macromolecular Transfer</I>, Annu. Rev. Cell Dev. Biol., 22:, 101-127 
</LI></UL><UL><LI>McCullum EO, Willliams BAR, Zhang J, Chaput JC (2010) <I>Random Mutagenesis by Error-Prone PCR, In vitro mutagenesis protocols</I>, Methods in Molecular Biology, Vol 634:, 103-109.
</LI></UL><UL><LI>Melchers LS, (1989), <I>Membrane topology and functional analysis of the sensory protein VirA of Agrobacterium tumefaciens</I>, The EMBO Journal, 8(7):, 1919-1925 
</LI></UL><UL><LI>MUltiple Sequence Comparison by Log-Expectation. MUSCLE <A href="http://www.ebi.ac.uk/Tools/muscle/index.html" class="external free" rel="nofollow">http://www.ebi.ac.uk/Tools/muscle/index.html</A></LI></UL><UL><LI>Palmer AG, Gao R, Maresh J, Erbil WK, Lynn DG (2004), <I>Chemical biology of multi-host/pathogen interactions: chemical perception and metabolic complementation</I>, Annu Rev Phytopathol, 42:, 439â464. 
</LI></UL><UL><LI>Pazour GJ, Das A (1990a), <I>Characterization of the VirG binding site of Agrobacterium tumefaciens</I>, Nucleic Acids Res, 18:, 6909â6913 
</LI></UL><UL><LI>Pazour GJ, Das A (1990b), <I>virG, an Agrobacterium tumefaciens transcriptional activator, initiates translation at a UUG codon and is a sequence-specific DNA-binding protein</I>, J Bacteriol, 172:, 1241â 1249. 
</LI></UL><UL><LI>Sambrook J, Russel DW (2001), <I>Molecular Cloning â A Laboratory Manual</I>, Cold Spring Harbor Laboratory Press, 1:
</LI></UL><UL><LI>Scott M, Lohrke SM (2001), <I>Reconstitution of Acetosyringone-Mediated Agrobacterium tumefaciens Virulence Gene Expression in the Heterologous Host Escherichia coli</I>, J. of Bacteriology, 183(12):,3704â3711  
</LI></UL><UL><LI>Shimoda N, Toyoda-Yamamoto A, Shinsuke S, Machica Y (1993), <I>Genetic evidence for an interaction between the VirA sensor protein and the ChvE sugar-binding protein of Agrobacterium</I> J. Biol. Chem., 268:,26552â58 
</LI></UL><UL><LI>Stachel SE, Nester EW, (1986) <I>The genetic and transcriptional organization of the vir region of the A6 Ti plasmid of Agrobacterium tumefaciens</I>, EMBO J, 5:, 1445â1454. 
</LI></UL><UL><LI>Turk SC, van Lange RP, Regensburg-TuÃ¯nk TJ, Hooykaas PJ., 1994, <I>Localization of the VirA domain involved in acetosyringone-mediated vir gene induction in Agrobacterium tumefaciens</I>, Plant Mol Biol. 25(5):,899-907 
</LI></UL><UL><LI>Wiegand I, Hilpert K, Hancok REW (2008) <I>Agar and broth dilution methods to determine the minimal inhibitory concentration (MIC) of antimicrobial substances</I>, Nature Protocols, 3:, 163-175.
</LI></UL><UL><LI>Williams TM, Burlein JE, Ogden S, Kricka LJ and Kant JA, 1989 <I>Advantages of firefly luciferase as a reporter gene: Application to the interleukin-2 gene promoter</I>, Analytical Biochemistry, 176:, 28-32 
</LI></UL><UL><LI>Winans SC., 1992, Two-way chemical signaling in Agrobacterium-plant interactions., Microbiol Rev. 56(1):,12-31.
</LI></UL><UL><LI>Wilson DS, Keefe AD (2000) <I>Random Mutagenesis by PCR</I>, Current Protocols in Molecular Biology, 8.3.1-8.3.9. 
</LI></UL><UL><LI>Wolanin PM, Thomason PA, Stock J.B. (2002), <I>Histidine protein kinases: key signal transducers outside the animal kingdom</I>, Genome Biol 3: REVIEWS3013. 
</LI></UL><UL><LI>Wright GD, Thompson PR (1999) <I>Aminoglycoside phosphotransferases: Proteins, Structure and Mechanism</I>, Frontiers in Bioscience 4, d9-21.
</LI></UL><UL><LI>Yi Han Linn, Goa R, Bins AN, Lynn DG (2008), <I>Capturing the VirA/VirG TCS of Agrobacterium tumefaciens</I>, Adv Exp Med Biol. 631:, 161-77 
</LI></UL><UL><LI>Yi Han Linn, Goa R, Bins AN, Lynn DG (2007), <I>The initial steps in Agrobacterium tumefaciens pathogenesis: chemical biology of host recognition, Agrobacterium: From Biology to Biotechnology</I>, 2008, 221-241
</LI></UL><UL><LI>Yong-Chul J, Yunrong G, Donghai W, Shouguang J (2004), <I>Mutants of Agrobacterium tumefaciens virG Gene That Activate Transcription of vir Promoter in Escherichia coli</I>, Current Microbiology, 49, 334â340 
</LI></UL><UL><LI>Ziemienowicz A (2001), Odyssey of agrobacterium T-DNA,. Acta Biochim Pol. 2001, 48(3):, 623-35.
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Bielefeld-Germany/Project/Theory">http://2010.igem.org/Team:Bielefeld-Germany/Project/Theory</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Project/Theory" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Project/Theory" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project/Theory&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project/Theory&amp;oldid=199109" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>