"
Page
Discussion
View source
History
teams
Log in
 
Team:Bielefeld-Germany/Results/Characterization
From 2010.igem.org
(Redirected from Team:Bielefeld-Germany/BioBricks/Tests )
Start
Team
Description
Supervisors
University
Bielefeld
Motivation
Acknowledgements
Project
Application
Approach
Theory
Protocols
Model
Outlook
Results
Characterization
Used BBs
Submitted BBs
Sequencing
Safety
Biosecurity
Biosafety
Risk Assesement
Public Relations
Fotos
Press
Public Discussion
Judging
BioBricks
Helping other teams
Comments on iGEM
Sponsors
Contact
MENU
http://igem-bielefeld.de/img/banner_lab.png
Results
Characterization
Used
Submitted
Sequencing
Contents
1<partinfo>K238008</partinfo>: virA
2<partinfo>BBa_K238011</partinfo>: vir-promoter
3<partinfo>P1010</partinfo>: ccdB-gene
4<partinfo>K389004</partinfo>: Luciferase from pGL4.10[luc2]
5<partinfo>K389011</partinfo>: VirA screening device
6<partinfo>K389015</partinfo>: VirA/G reporter device with luciferase
7<partinfo>K389016</partinfo>: VirA/G reporter device with mRFP
8<partinfo>K389052</partinfo>: Tightly regulated lac operon with mRFP readout
9<partinfo>K389421</partinfo>, <partinfo>K389422</partinfo>, <partinfo>K389423</partinfo>: Sensitivity Tuner amplified Vir-test system
10Mutated <partinfo>K389001</partinfo> (exemplary): Results of the directed mutagenesis
11References
<partinfo>K238008</partinfo>: virA
We wanted to use this part in our project, but could only obtain unexpected/faulty restriction patterns. Finally we chose to sequence the part, hoping to find the cause for the maintained restriction patterns. Unfortunately we could not approve the sequence of <partinfo>BBa_K238008</partinfo> deposited in the parts registry so that we chose to design our own VirA BioBrick. I strongly recommend using our VirA since it has been approved by multiple means, e.g. restriction patterns and sequencing (<partinfo>K389001</partinfo>).
<partinfo>BBa_K238011</partinfo>: vir-promoter
We wanted to use this part in our project, but could only obtain unexpected/faulty restriction patterns. Finally we chose to sequence the part, hoping to find the cause for the maintained restriction patterns. Unfortunately we could not approve the sequence of <partinfo>BBa_K238011</partinfo> deposited in the parts registry so that we chose to design our own VirA BioBrick. I strongly recommend using our VirA since it has been approved by multiple means, e.g. restriction patterns and sequencing (<partinfo>K389003</partinfo>).
<partinfo>P1010</partinfo>: ccdB-gene
The ccdB gene targets the gyrase of Escherichia coli and is lethal for all E. coli strains without the gyrase mutation gyrA462 ( Openwetware ). The ccdB BioBrick is used for the 3A-assembly as a positive selection marker.  We transformed this BioBrick into E. coli JM109, DH5α, TOP10, XL1-Blue, EC100D and DB3.1. E. coli JM109, XL1-Blue and DH5α seem to be ccdB resistant because there were as much colonies after P1010 transformation as observed with DB3.1. The P1010 works as expected in E. coli TOP10, EC100D (no colonies after transformation) and DB3.1 (many colonies after transformation).
Table 1: Results of the transformation of the cell-death gene ccdB, BioBrick <partinfo>P1010</partinfo>, into different strains of E. coli.
E. coli strain
Resistant to ccdB?
Expected result?
Gyrase genotype
( Metcalf et al., 1994 ; Openwetware )
DB3.1
yes
yes
gyrA462
DH5α
yes
no
gyrA96
EC100D
no
yes
WT
JM109
yes
no
gyrA96
TOP10
no
yes
WT
XL1-Blue
yes
no
gyrA96
It seems that not only the gyrase mutation gyrA462 is causing a ccdB resistance. Also the gyrase mutation gyrA96 gives E. coli a ccdB resistance. This should be kept in mind when assembling BioBricks with the 3A assembly.
<partinfo>K389004</partinfo>: Luciferase from pGL4.10[luc2]
For a comparison between mRFP and luciferase as reporter genes click here.
Some important parameters determined by the characterization experiments are shown in tab. 2. For more information concerning these experiments click on the corresponding link in tab. 2 or click here:
Detailed information...
Table 2: Parameters for <partinfo>K389004</partinfo>.
Experiment
Result
Behaviour during cultivation
production is growth dependent
degradation in stationary growth phase
Kinetic of luciferin conversion
max. output between 20 - 40 s
Limit of detection (LOD)
162 RLU ~ 0.3 % of <partinfo>J23103</partinfo> output
Limit of quantification (LOQ)
306 RLU ~ 0.7 % of <partinfo>J23103</partinfo> output
<partinfo>K389011</partinfo>: VirA screening device
Detailed information...
Figure 1: Ratio of surviving colonies of E. coli EC100D carrying unmutated <partinfo>K389010</partinfo> and <partinfo>K389011</partinfo> plated on PA agar plates with chloramphenicol, ampicillin and different concentrations of kanamycin. Comparison between cells that were induced with acetosyringone with cells that were not induced.
<partinfo>K389015</partinfo>: VirA/G reporter device with luciferase
Some important parameters determined by the characterization experiments are shown in tab. 3. For more information concerning these experiments click on the corresponding link in tab. 3 or click here:
Detailed information...
Table 3: Parameters for <partinfo>K389015</partinfo>.
Experiment
Characteristic
Value
Transfer Function
Maximum induction level
2.2 fold
Maximum induction level reached
200 µM acetosyringone
Hill coefficient
1.09
Switch Point
31.6 µM acetosyringone
Doubling time / h
without plasmid
1.98
carrying K389015
2.24
carrying K389015 with 400 µM acetosyringone
2.67
Response time
Induction: exponential phase
>1 h
Induction: begin of cultivation
max. induction at OD600 = 1 +/- 0.5
Conformation analysis
ratio ccc monomer / %
91
ratio ccc dimer / %
3.7
ratio oc forms / %
5.3
<partinfo>K389016</partinfo>: VirA/G reporter device with mRFP
Protocols for Cultivation and Measurement
Some important parameters determined by the characterization experiments are shown in tab. 4. For more information concerning these experiments click on the corresponding link in tab. 4 or click here:
Detailed information...
Table 4: Parameters for <partinfo>K389016</partinfo>.
Experiment
Characteristic
Value
Transfer Function
Maximum induction level
2.6 fold
Maximum induction level reached
150 µM acetosyringone
Hill coefficient
1.67
Switch Point
26.5 µM acetosyringone
Doubling time / h
without plasmid
1.98
carrying K389016
2.57
carrying K389016 with 150 µM acetosyringone
2.77
carrying K389016 with 1000 µM acetosyringone
3.01
Conformation analysis
ratio ccc monomer / %
91.2
ratio ccc dimer / %
3.2
ratio oc forms / %
5.6
Inducers
Induction by
Acetosyringone
No Induction by
Capsaicin
Dopamine
Homovanillic acid
3-Methoxytyramine
<partinfo>K389052</partinfo>: Tightly regulated lac operon with mRFP readout
This construct was plated for plasmid isolation in a lacIq negative E. coli strain after assembly - and we have never seen such red plates when working with constructs with mRFP downstream of a promoter. This lac operon definitely shows a very high basal transcription, so it is not tightly repressed. It seems that the lacI repressor <partinfo>BBa_C0012</partinfo> is not suitable for this purpose due to its LVA degradation tag or it does not work properly. Another indicator for this assumption is the experience page of <partinfo>C0012</partinfo>.
<partinfo>K389421</partinfo>, <partinfo>K389422</partinfo>, <partinfo>K389423</partinfo>: Sensitivity Tuner amplified Vir-test system
Detailed information...
Figure 2: Amplification factor of induced, 50 µM Acetosyringone (red) and not induced (green) modified Sensitivity Tuner <partinfo>K389421</partinfo>, <partinfo>K389422</partinfo> and <partinfo>K389423</partinfo>, Standard deviation shown.
Figure 3: Vizualitation of induced (from left to right) <partinfo>K389421</partinfo>, <partinfo>K389422</partinfo> and <partinfo>K389423</partinfo> sensitivity tuner amplified vir-system.
Mutated <partinfo>K389001</partinfo> (exemplary): Results of the directed mutagenesis
Detailed information...
Figure 4: Luciferase production rates of the exemplary clone “21” are shown under conditions of induction with acetosyringone (21-A), capsaicin (21-C) and in uninduced state (21-U). The right bar indicates the production rate of the native system without acetosyringone (212-U), all the values in the figure had been normalized to.
References
Behrens B, Eppendorf AG, Laborpraxis, Nr.20, Reinste Plasmid-DNA in nur 9 Minuten.
http://openwetware.org/wiki/CcdB , CcdB (seen on 10.10.10).
http://openwetware.org/wiki/E._coli_genotypes , E. coli genotypes (seen on 10.10.10).
Metcalf, WW et al. (1994) Use of the rep technique for allele replacement to construct new Escherichia coli hosts for maintenance of R6Kλ origin plasmids at different copy numbers , Gene 138(1):1-7.
Stadler J, Lemmens R, Nyhammar T 2004, Plasmid DNA purification, The J. of Gene Medicine,Vol.6, pp.54–S66
Retrieved from " http://2010.igem.org/Team:Bielefeld-Germany/Results/Characterization "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
