<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Groningen_results_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Groningen/results.html">Page               </A></LI><LI><A href="/Talk:Team:Groningen/results.html">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Groningen/results.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Groningen/results.html&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Groningen/results.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Groningen/results.html</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV id="header"><DIV id="menu"><UL class="L1"><LI><A href="https://2008.igem.org/Team:Groningen">Home</A></LI><LI><A href="https://2008.igem.org/Team:Groningen/Introduction">Introduction</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/design.html">Design</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/design.html#interval_switch">Interval Switch</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/design.html#genetic_circuit">Genetic Circuit</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/design.html#physical_system">Physical System</A></LI></UL><LI class="sub"><A href=""> Modeling </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/modeling_SingleCell.html"> Single-cell </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/modeling_Spatial.html"> Spatial model </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/modeling_files.html">Model Files</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/results.html">Results</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/results.html#results"> Results </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/results.html#conclusion"> Conclusion </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/iGEM_Criteria">iGEM Criteria</A></LI><LI class="sub"><A href="">About...</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/team.html"> The Team </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/rug.html"> Groningen University </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/references.html"> References </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/acknowledgements.html"> Acknowledgements </A></LI></DIV></DIV><DIV id="results" class="location"><DIV id="view"><DIV id="textblock"><H1>Results</H1><DIV class="headcontainer"><H2 id="results">Bba_K077557 is inducible with HSL</H2></DIV><P>This is the first part of the genetic circuit to implement Conway’s game of life. We tested the part for correctness
by sequencing. This part should respond to HSL by producing GFP and LuxI. We expected that this
part may not be very inducible because if it is only a bit leaky it will produce more HSL via LuxI. This positive
forward loop could result in the E. coli cells being green even if these cells were not stimulated with HSL.
We tested this part and compared it to T9002 (see figure 7.3). From figure 7.2 we can see that we can induce
K077557 with HSL but not in the same way as our positive control T9002 (figure 7.1). K077557 reaches
a fluorescence which is 100 times lower than T9002 (figure 7.1 and figure 7.2) next to this K077557 takes
much longer to get induced compared to T9002 (data not shown). Also we observed that we can only induce
K077557 if it is grown directly from EZ plate in liquid EZ. After ON growth in liquid culture all cells were induced,
even diluting up to 50 times did not solve this ‘auto induction’ (figure 7.1).
            <TABLE class="image-right"><CAPTION align="bottom">
            Figure 7.1. Flow cytometry results
of ‘auto induced’ K077557 (red
&amp; blue) and the positive control
T9002 (purple &amp; light blue). A total
of 20.000 events were measured.
Y-axis indicates number of events.
X-axis indicates fluorescence on a
logarithmic scale. The results of
K077557 in this graph show the
auto inducing behavior of K077557
after overnight growth (see text).
              </CAPTION></TABLE></P><P>In addition we performed
a functional test with K077557 on EZ solid medium plates. In figure 7.4 we compared T9002 with K077557.
In the middle we plated a sender colony and on opposite sides we spotted colonies of T9002 and K077557 at
constant intervals. We expected that if this part of the system would be fully functional it could expand the
signal from the sender colony and thereby the sender could induce more colonies of K077557 compared to
T9002. From figure 7.4 it can be observed that only the GFP output from T9002 is visible on the plate. This
coincides with the flow results which show that T9002 exhibits a far greater response to HSL induction. The
question which arises here is: why is the output of K077557 so much lower than the output of T9002 and
why is the response of T9002 quicker? The differences in these constructs on a genetic level are: different
plasmid, different genetic content. T9002 is harbored on pSB1A2 whereas K077557 is harbored on pSB1AC3.
These plasmids are both high copy pUC19 derivatives, they differ in size around 1000 base pairs which is the
chloramphenicol resistance which is only present on pSB1AC3. This difference in plasmids does not seem to
be the cause of the difference we observe in GFP expression although it adds to the size difference between
the two constructs (plasmid + part).        
          <TABLE class="image-left"><CAPTION align="bottom">
            Figure 7.2. Flow cytometry results of K077557. A total of 20.000 events were measured. Y-axis indicates number of events. X-axis indicates fluorescence on a logarithmic scale. Z-axis indicates time after induction.
          </CAPTION></TABLE></P><TABLE class="image-right"><CAPTION align="bottom">
                Figure 7.3. Genetic composition of test construct T9002 and K077557 the first part of the system.
              </CAPTION></TABLE><P>K077557 has a total size of 5749bp versus 4024 bp for T9002. Apart from the plasmids the size difference is
due to the presence of the luxI gene (800bp). The luxI gene is transcribed together with GFP in K077557. This
could be the cause of the time delay in fluorescence compared to T9002. Furthermore the promoters in front
of the GFPs are different which could also have an effect on the speed of GFP build up, this, however, does
not seem to be the case (see paragraph 7.3). Another important difference is the presence of a so called LVA
tag on the GFP of K077557. The LVA tag has no or little influence on the fluorescence of the GFP molecule, it
only results in a 40 times higher breakdown speed [26]. This could therefore have an effect on the amount of
fluorescence which is build up after induction. From these data we can conclude that the first part of the system
is functional and can induce itself. The fluorescence of K077557 is probably so much lower than T9002
because of the LVA tag.
        </P><TABLE class="image-center"><CAPTION align="bottom">
                Figure 7.4. Image of E. coli colonies reacting to HSL produced by sender device. Cells were spotted at approximately the same distance. The picture was taken after 2 days of growth at
                37 0C with a standard digital SLR camera using a ultra violet lamp to excite the GFP. The sender device in the middle produces HSL (obtained from the Weiss group). Left from the sender device K077557 shows no visible GFP. On the right side the first 4 colonies of T9002 turn green.
              </CAPTION></TABLE><DIV class="headcontainer"><H2>Bba_K077667 is constructed instead of Bba_K077668</H2></DIV><P>Originally we wanted to construct K077668 which contains the adjusted R0062 promoter (K077200). Since
we had trouble obtaining K077200 and time was running out we decided to use the alternative part K077667
with the normal R0062 promoter. K077667 is the alternative second part of the CGOL system which we created
from two synthesized parts and one part from the registry. The construct has been checked by sequencing.
We could not check it on a functional basis because the output of GFP in the first part (K077557) of the
system was lower than expected. Due to lack of time (synthesis was delayed) we were not able to test its
functionality on the basis of its own outputs (AiiA, CIl).</P><DIV class="headcontainer"><H2>R0062 and R0065 have similar response to HSL</H2></DIV><P>
            Our design is dependent on the promoters R0062 and R0065 therefore we wanted to investigate the responsiveness
to HSL. We decided to test this in two ways using two different outputs, GFP and LacZ. Unfortunately
we did not manage to make the GFP construct. The LacZ constructs were K077037 (R0065) and K077126
(R0062). K077126 is very similar in design to previously constructed T9003 (it also contains R0062) with the
only difference being the order of the subparts (figure 7.5). The part T9003 was used in the pSB3K3 backbone
which has a low copy number all the other parts were harbored in pSB1A2 which has a high copy number.
The parts were checked according to our quality control procedure and finally approved by sequencing. In
figure 7.6 we can see that both R0062 constructs K077126 and T9003 show similar expression of LacZ upon
induction although the background level of K077126 seems to be slightly higher. This higher background
could be due to the higher copy number of pSB1A2. The R0065 (K077037) construct is also inducible; it shows
a slightly higher output (approximately 20%) and also a higher background than T9003. From these data we
can conclude that both promoters can be induced with HSL, resulting in transcription rates which are in the
same order of magnitude.
            <TABLE class="image-right"><CAPTION align="bottom">
                Figure 7.5. Genetic content of testconstructs used to compare the HSL dependent promoters R0062 and R0065.
              </CAPTION></TABLE></P><TABLE class="image-center"><CAPTION align="bottom">
                Figure 7.6. Beta galactosidase assay with K077037 (red), K077126 (blue) and T9003 (green). Samples were induced with 100nM HSL for 30 minutes. Values are relative, T9003 uninduced was set to 1.
              </CAPTION></TABLE><H2> T9003 is a functional part </H2><P>We tested the functionality of part T9003 (figure 7.7) through an beta galactosidase assay. This part works as
expected, the expression of LacZ was dependent on the HSL concentration. We suggest that the status of this
part in the registry should be changed to working.</P><TABLE class="image-center"><CAPTION align="bottom">
                Figure 7.7. Beta galactosidase assay
with T9003. T9003 was induced with
100nM HSL for 30 minutes. Values
are relative, T9003 uninduced was set
to 1. Bars on top indicate standard
deviation.
              </CAPTION></TABLE><DIV class="headcontainer"><H2>Bba_F1610 is faulty</H2></DIV><P>We wanted to use this sender device to test physical systems. This part from the database is not functional;
we could not induce receiver devices with it. We did a PCR and restriction analysis which showed wrong
bands. Finally we sequenced the part which showed that the sequence does not match the sequence from
the database</P><DIV class="headcontainer"><H2 id="constructed">Constructed Parts</H2></DIV><P>In table 7.1 the constructed parts are listed. All of these have been successfully assembled by the standard
assembly method on partsregestry, except for K077555 and K077666, these two were synthesized. All the
parts were checked by PCR and restriction and finally sequenced to confirm that they were correct.</P><TABLE id="constructed_parts"><TBODY><TR><TH>Part</TH><TH>Subparts</TH><TH>Available</TH></TR><TR><TD>K077001</TD><TD>R1051 + P0412</TD><TD>yes</TD></TR><TR><TD>K077002</TD><TD>R0062 + P0151</TD><TD>yes</TD></TR><TR><TD>K077004</TD><TD>R0065 + P0151</TD><TD>yes</TD></TR><TR><TD>K077006</TD><TD>R1051 + P0412</TD><TD>yes</TD></TR><TR><TD>K077007</TD><TD>R0010 + S0105</TD><TD>yes</TD></TR><TR><TD>K077008</TD><TD>R1051 + I0460</TD><TD>yes</TD></TR><TR><TD>K077011</TD><TD>S0100 + I0460</TD><TD>yes</TD></TR><TR><TD>K077018</TD><TD>R0065 + P0451</TD><TD>yes</TD></TR><TR><TD>K077019</TD><TD>R0062 + P0451</TD><TD>yes</TD></TR><TR><TD>K077037</TD><TD>R0065 + K077124</TD><TD>yes</TD></TR><TR><TD>K077124</TD><TD>E0435 + S03119</TD><TD>yes</TD></TR><TR><TD>K077126</TD><TD>R0062 + E0435 + C0062</TD><TD>yes</TD></TR><TR><TD>K077555</TD><TD>R0065 + B0034 + C0061 + B0034 + J04031 + B0015</TD><TD>yes</TD></TR><TR><TD>K077557</TD><TD>K077555 + S03119</TD><TD>yes</TD></TR><TR><TD>K077666</TD><TD>B0034 + C0051 + B0034 + C0060 + B0015</TD><TD>yes</TD></TR><TR><TD>K077667</TD><TD>R0062 + K077666</TD><TD>yes</TD></TR><TR><TD>K077024</TD><TD>E0840 + S03119</TD><TD>planned</TD></TR><TR><TD>K077200</TD><TD>adjusted R0062</TD><TD>planned</TD></TR></TBODY></TABLE><DIV class="headcontainer"><H2 id="conclusion">Conclusion</H2></DIV><H3>Modeling</H3><P>We have modeled the switching ability of the cells both on a single-cell level and by a spatial simulation. On
the single-cell level the desired interval switch behavior was recovered, although the range of the interaction
has to be tuned still by feeding more realistic parameters to the model. The transient response at high input
concentrations obscures the measurement slightly; while this requires further investigation we expect it to
be avoidable. A novel approach, using one-to-one conversion from BioBricks to ‘modelbricks’, was used. The
spatial simulations show that when doing experiments with sender cells, only the receiver cells that are at a
certain distance give GFP expression. Furthermore in our modeling effort we succeeded in tuning the system
so that a single cell can distinguish between having one, two or three active sender cells, which demonstrates
it is suitable for use in cellular automata. However, it remains uncertain whether or not the used parameters
are realistic and this still has to be confirmed through real biological experiments.</P><H3>Wet work</H3><P>We created a set of parts which could result in Conway’s Game of Life-like behavior. The GFP output of this
system was too low to investigate its behavior fully, although we demonstrated the predicted behavior partly.
The biggest challenge for the design seems to be output power versus fast dynamics. Escherichia coli cells
can produce GFP quickly and in large quantities. The degradation proces on the other hand is very slow (half
life of approximatley 24 hours) elongating the time between generations in Conway’s Game of Life. However
to generate interesting patterns with Conway’s Game of Life we need at least 3-5 generations. Therefore we
used LVA tagged GFP which is degraded quicker but therefore also gives a lower output. The drawback of this
is that the amount of GFP is smaller which means that in some setups fluorescence is hard to measure. We
compared two HSL dependent promoters R0065 and R0062 and concluded that the responsiveness to HSL is
approximately the same.</P><P>To test these HSL dependent promoters we created an HSL dependent promoter test part - K077124, which
can be used to measure the output (LacZ) of different promoters. We also tested the functionality of T9003
and concluded it is a functional part. The part F1610 does not seem to be correct.</P><H3>Future Work</H3><P>Testing the system on other outputs than the GFP, this could tell more about the functionality of the system.
Also setting up experiments which show GFP on a smaller scale (not on a colony level) could improve our
knowledge about the system. The CIl repressor should be tested to see if this is really blocking the transcription
of luxI and gfp. The protein AiiA needs to be tested for its HSL degrading abilities and dynamics.</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Groningen/results.html">http://2008.igem.org/Team:Groningen/results.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Groningen/results.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Groningen/results.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Groningen/results.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Groningen/results.html&amp;oldid=100711" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>