<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Groningen_modeling_Spatial_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Groningen/modeling_Spatial.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Groningen/modeling_Spatial.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Groningen/modeling_Spatial.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Groningen/modeling_Spatial.html&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Groningen/modeling_Spatial.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Groningen/modeling Spatial.html</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV id="header"><DIV id="menu"><UL class="L1"><LI><A href="https://2008.igem.org/Team:Groningen">Home</A></LI><LI><A href="https://2008.igem.org/Team:Groningen/Introduction">Introduction</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/design.html">Design</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/design.html#interval_switch">Interval Switch</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/design.html#genetic_circuit">Genetic Circuit</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/design.html#physical_system">Physical System</A></LI></UL><LI class="sub"><A href=""> Modeling </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/modeling_SingleCell.html"> Single-cell </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/modeling_Spatial.html"> Spatial model </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/modeling_files.html">Model Files</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/results.html">Results</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/results.html#results"> Results </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/results.html#conclusion"> Conclusion </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/iGEM_Criteria">iGEM Criteria</A></LI><LI class="sub"><A href="">About...</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/team.html"> The Team </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/rug.html"> Groningen University </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/references.html"> References </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/acknowledgements.html"> Acknowledgements </A></LI></DIV></DIV><DIV id="modeling_Spatial" class="location"><DIV id="view"><DIV id="textblock"><H1>Modeling: Spatial Approach</H1><DIV class="headcontainer"><H2>Introduction</H2></DIV><P>In this chapter we will present the model we used to do spatial simulations of our interval switch network.
This network is intended as part of a cellular automaton that serve as a detector that can differentiate between
having ’too few’ and ’too many’ neighbors, in this chapter we will try to demonstrate this concept for
the case where no neighbors is ’too few’ and two neighbors is ’too many’.</P><DIV class="headcontainer"><H2>Network Overview</H2></DIV><P>The network we simulate is shown in Figure 6.7. It consists of one hybrid promoter, inhibited by CI and activated
by HSL.LuxR, which facilitates GFP expression. The second promoter is a modified lux promoter which
is activated by HSL.LuxR and facilitates CI expression.
Because the lux promoter is modified in such a way that the binding site is less easily bound to, we expect
that more HSL.LuxR is needed to express enough CI to block the hybrid promoter. This means that when HSL.
LuxR is too low, the hybrid promoter (and GFP expression) will not be turned on, and when HSL.LuxR exceeds
a certain treshold, the lux promoter will be activated and its CI expression will block the hybrid promoter,
stopping GFP expression. The result is a network that will only show a GFP response if the HSL.LuxR concentration
is inbetween two limits.</P><TABLE class="image-center"><CAPTION align="bottom">
                Figure 6.7. Device diagram.
              </CAPTION></TABLE><DIV class="headcontainer"><H2>Spatial Model</H2></DIV><P>In this section we will give a specification of our model on a ’high level’, meaning that each sensing and protein
producing part is modelled as a single Ordinary Differential Equation (ODE) using Hill equations 
<SPAN class="refcontainer">[23]</SPAN></P><DIV class="reference">U. Alon. An Introduction to Systems Biology: Design Principles of Biological Circuits. Chapman &amp; Hall/ Crc Mathematical and Computational Biology Series, Chapman &amp; Hall/CRC, July 2006</DIV>.
For example a Lux-promoter that is connected to a Ribosomal Binding Site (RBS) and is followed by a protein
coding region will be modelled using a single Hill equation that returns the protein production as a function
of the activators/inhibitors of the promoter.
In this way, each of the producing parts that depend on promoter activation or inhibition can be modeled
using a single Hill Equation of the form equation 1 for promoters that are activated by S<SPAN class="sub">a</SPAN> and equation 2 for
promoters that are inhibited by S<SPAN class="sub">i</SPAN>. Where Vmax represents the maximal expression (without inhibition and/
or maximal activation) of the protein and the parameter K<SPAN class="sub">m</SPAN> represents the concentration of the inhibitor/
activator at which the production rate is half-maximal.<P>Of course we also have to model ’standard’ linear reactions such as decay (reaction 48). We do this using
Mass Action (MA) kinetics (resulting in equation 49). Note that species depicted in brackets [], represent the
concentration.</P><P>In the following subsections we will present the model we used to simulate the behavior of our gene
network. The Matlab source code itself can be downloaded from <A href="https://2008.igem.org/Team:Groningen/modeling_files.html">'Modeling Files'</A></P><H3>Species</H3><P>This section lists the different species that are present in the model.</P><H3>Parameters</H3><P>
The parameters needed for our model have been taken mostly from a single paper 
<SPAN class="refcontainer">[24]</SPAN></P><DIV class="reference">S. Basu, Y. Gerchman, C.H. Collins, F.H. Arnold, and R. Weiss. A synthetic multicellular system for programmed pattern formation. Nature, 434(7037):1130–1134</DIV>
. and estimated from the values found there. Other sources for parameters are 
<SPAN class="refcontainer">[9]<DIV class="reference">S. Basu, R. Mehreja, S. Thiberge, M. T. Chen, and R. Weiss. Spatiotemporal control of gene expression with pulse-generating networks. PNAS US<SPAN class="sub">a</SPAN>, 101(17):6355–6360, April 2004.</DIV></SPAN> 
and 
<SPAN class="refcontainer">[25]<DIV class="reference">D. Braun, S. Basu, and R. Weiss. Parameter estimation for two synthetic gene networks: a case study. Acoustics, Speech, and Signal Processing, Proceedings (ICASSP ’05), 5. IEEE International Conference, March 2005</DIV></SPAN>
.
<DIV class="headcontainer"><H2>Ordinary Differential Equations</H2></DIV><P>In this section we list the Differential Equations that make up our model. Species depicted in brackets [], depict
the concentration of a substance. Each differential equation represents the derivative of the concentration
of a substance, in other words the production of a species at a single time-step.</P><P>These ODEs take into account the diffusion of HSL to and from other cells (the current cell is (x,y)). For this
model we consider a 2-dimensional grid and it is assumed that each cell has four neighboring cells.</P><H3>LuxR</H3><P>
Because LuxR is constitutively expressed, we simply keep the concentration of LuxR constant at 0.5μM. This
S<SPAN class="sub">a</SPAN>me approach is used in 
<SPAN class="refcontainer">[24]</SPAN></P><DIV class="reference">S. Basu, Y. Gerchman, C.H. Collins, F.H. Arnold, and R. Weiss. A synthetic multicellular system for programmed pattern formation. Nature, 434(7037):1130–1134</DIV> 
for the wild-type LuxR.
<P>
HSL and LuxR have to form a complex first before they can form the HSL.LuxR complex, this is modeled in de
ODE by quadratizing both terms 
<SPAN class="refcontainer">[24]</SPAN></P><DIV class="reference">S. Basu, Y. Gerchman, C.H. Collins, F.H. Arnold, and R. Weiss. A synthetic multicellular system for programmed pattern formation. Nature, 434(7037):1130–1134</DIV> .
<P>GFP response depends on HSL.LuxR and CI concentrations, where the first acts as an activator and the second
serves as an inhibitor.</P><P>The amount of CI expressed depends on the modified lux promoter, meaning that expression is activated by
the amount of HSL.LuxR. Because this promoter is weaker then the promoter that activates GFP expression,
i.e. it needs more HSL.LuxR, we can create a band of HSL.LuxR concentrations where GFP is expressed but CI
is not yet (sufficiently) expressed.</P><DIV class="headcontainer"><H2>Simulations</H2></DIV><P>
To solve this collection of ODEs we used MATLAB’s 
<SPAN class="refcontainer">[11]</SPAN></P><DIV class="reference">Matlab. The Language of Technical Computing, The Mathworks Inc., version 7.6.0.324 (R2008a)</DIV> 
 ode15s function. The .m file we used for the simulations
can be found on the website and contains all the listed ODE functions.
To study the response of our model we included sender populations, referred to below as sender cells, in our
spatial grid. These populations constantly produce 1μM of HSL and are depicted in the images as red cells.
Because we are interested in the steady states, the images shown are snapshots taken after running the
model for approximately 1000 time-steps (minutes).
In the following sections we will explore several parameters and try to approach our desired result, i.e. an interval
switch that is tweaked for use in a cellular automaton. The grid-size used in the following experiments
is 17 x 17 (’uninteresting regions’ (i.e. without GFP expression) are cropped from the images).
<DIV class="headcontainer"><H2>Single Sender Cell Simulations</H2></DIV><H3>Standard Parameters</H3><P>When we simply run our model with the parameters described in the paper and include a single sender
cell, we get the result as shown in Figure 6.8. We can see that, as expected, a band of GFP forms around the
sender cells, indicating that the cells that are near a sender get too much HSL and the cells that are far away
from the sender does not get enough HSL for proper activation.</P><TABLE class="image-left"><CAPTION align="bottom">         Figure 6.8. GFP response with a single sender cell.
              </CAPTION></TABLE><H3>Decreasing lux Promoter Sensitivity</H3><P>After setting the concentration of HSL.LuxR that is needed for half-maximal expression of CI twice as high
(to 0.16μM), we get the result shown in Figure 6.9. There we can see that more HSL is required to produce
enough CI to inhibit GFP output, slightly increasing the width of the band.</P><TABLE class="image-right"><CAPTION align="bottom">
                Figure 6.9. GFP response with a single sender cell, with K<SPAN class="sub">l(a)</SPAN>increased
to 0.16μM.
              </CAPTION></TABLE><DIV class="headcontainer"><H2>Multiple Sender Cell Simulations</H2></DIV><H3>Standard Parameters</H3><P>When we run the model with the normal parameters and two sender cells, we get the result as shown in
Figure 6.10. This indicates that cells that receive HSL from both senders now approach the high threshold
and show a lesser GFP response.</P><TABLE class="image-left"><CAPTION align="bottom">
                Figure 6.10. GFP response with two sender cells (default parameters)
              </CAPTION></TABLE><H3>Tweaked Parameters</H3><P>We now try to adjust the parameters so that we get a response that is suitable for use in a cellular automaton.
In this case we try to tweak the model so that when two neighboring cells are ON, a cell does
not show a GFP response, and when only a single neighboring cell is ON, a cell does show a GFP response.
After playing around with the parameters we found that one way of achieving this behavior is by drastically
decreasing HSL diffusion to 0.00001 mm<SPAN class="super">2</SPAN> min<SPAN class="super">-1</SPAN> and increasing HSL decay to 0.03 min<SPAN class="super">-1</SPAN>, the resulting snapshot
of the grid is shown in Figure 6.11. </P><TABLE class="image-right"><CAPTION align="bottom">
                Figure 6.11. GFP response with two sender cells. HSL diffusion set to
0.00001mm<SPAN class="super">2</SPAN> min<SPAN class="super">-1</SPAN> and HSL decay set to 0.03 min<SPAN class="super">-1</SPAN>.
              </CAPTION></TABLE><P>A nice thing to note is that such parameter-tweaks can also be done
in ’real biology’ like by modifying the pH of the solution to increase HSL decay or increasing the density of the
medium or distance between cells to reduce HSL diffusion speed.</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Groningen/modeling_Spatial.html">http://2008.igem.org/Team:Groningen/modeling_Spatial.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Groningen/modeling_Spatial.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Groningen/modeling_Spatial.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Groningen/modeling_Spatial.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Groningen/modeling_Spatial.html&amp;oldid=102637" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>