<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Groningen_Introduction"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Groningen/Introduction">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Groningen/Introduction&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Groningen/Introduction&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Groningen/Introduction&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Groningen/Introduction" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Groningen/Introduction</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV id="header"><DIV id="menu"><UL class="L1"><LI><A href="https://2008.igem.org/Team:Groningen">Home</A></LI><LI><A href="https://2008.igem.org/Team:Groningen/Introduction">Introduction</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/design.html">Design</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/design.html#interval_switch">Interval Switch</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/design.html#genetic_circuit">Genetic Circuit</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/design.html#physical_system">Physical System</A></LI></UL><LI class="sub"><A href=""> Modeling </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/modeling_SingleCell.html"> Single-cell </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/modeling_Spatial.html"> Spatial model </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/modeling_files.html">Model Files</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/results.html">Results</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/results.html#results"> Results </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/results.html#conclusion"> Conclusion </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/iGEM_Criteria">iGEM Criteria</A></LI><LI class="sub"><A href="">About...</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/team.html"> The Team </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/rug.html"> Groningen University </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/references.html"> References </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/acknowledgements.html"> Acknowledgements </A></LI></DIV></DIV><DIV id="introduction" class="location"><DIV id="view"><DIV id="textblock"><H1>Introduction</H1><DIV class="headcontainer"><H2>Cellular Automata</H2></DIV><P>Cellular automata consist of a regular array of cells, termed a grid, of arbitrary dimension. 
      Any cell can be in two or more distinguishable states. 
      Given an initial configuration of these states the grid evolves without any further intervention,
      hence the term automaton, with every new configuration based solely on the previous configuration 
      and a set of fixed rules. Often the evolution of cellular automata progresses in a discrete way, 
      in this case every new configuration is called a ‘generation’, nevertheless continuous cellular automata do 
      indeed exist (fig 1.1). Since the 1940’s when the concept of a cellular automaton first arose as a purely 
      mathematical problem 
      <SPAN class="refcontainer">[1]</SPAN></P><DIV class="reference">J. von Neumann, and A.W. Burks. Theory of Self-Reproducing Automata. Univ. of Illinois Press, Urbana IL, 1966.</DIV>
      , these systems have found new implementations in a broad range of disciplines. 
      In physics, chemistry and biology cellular automata are widely used in simulations involving emerging 
      complex behavior and self-organization, while from the viewpoint of computer science the fact that some 
      cellular automata display what is called turing completeness is of considerable interest <SPAN class="refcontainer">[2]<DIV class="reference">S. Wolfram. Universality and complexity in cellular automata. Physica D 10:1-35, 1984.</DIV></SPAN>. 
      Also in cryptography cellular automata are considered to be useful because of their uni-directional 
      evolutionary character 
      <SPAN class="refcontainer">[3]<DIV class="reference">S. Sen, C. Shaw, D.R. Chowdhuri, N. Ganguly, and P.P. Chaudhuri. Cellular Automata Based Cryptosystem (CAC), ICICS ‘02. Proceedings of the 4th International Conference on Information and Communications Security, 303--314. Springer-Verlag, London, UK 2002. </DIV></SPAN>.
        

        
      <P>Additionally these systems are thought to be present in nature as being, for example, the mechanism leading to texture formation on animal skin. Computer models of cellular automata in the continuum approximation are able to reproduce patterns similar to these textures, which in nature emerge from the continuous interaction between pigment cells 
      <SPAN class="refcontainer">[4]</SPAN></P><DIV class="reference">G.B. Ermentrout, and L. Edelstein-Keshet. Cellular Automata Approaches to Biological Modeling. Journal of Theoretical Biology,160(1):97-133, January 1993 </DIV>. From a synthetic biology point of view, this connection between a formal, algorithmical mathematical model and similar expression observed in living cells poses the interesting question of whether this behavior can be syntheticallly implemented within living cells, thereby making them generate controllable patterns.<P>The approach that synthetic biology propagates, designing modular genetic components and combining these to comprise more complex genetic systems, is particularly fit to tackle a problem of this kind since in the workings of cellular automata we can roughly distinguish the following abillities that cells should posess: to switch between a discrete number of states, to communicate information about their state to their neighbors and to interpret and act upon information received from their neighbors. Within the synthetic biology framework these functionalities can be independently designed and subsequently combined to display total system behavior.</P><DIV class="headcontainer"><H2>Conway's Game of Life</H2></DIV><P>The particular cellular automaton which we aim to design is ‘Conway’s Game of Life’, 
      one of the best known and most extensively studied cellular automata. It was proposed by mathematician 
      John Conway <SPAN class="refcontainer">[6]</SPAN></P><DIV class="reference">Martin Gardner. Mathematical Games: The fantastic combinations of John Conway’s new solitaire game “Life”. Scientific American 223:120-123, October 1970.</DIV> in 1970 as a simplification of previous research by von Neumann [1]. In Conway’s Game of Life
       cells exist and interact on a two-dimensional grid. Two states are distinguished, in the game’s language 
       these states are termed ‘alive’ or ‘dead’, and cells can switch between these two states depending on 
       their number of live neighbors: if either more than three or less than three of a cells neighbors are 
       alive, that cell switches to the dead state. If exactly three of a cells neighbors are alive, that cell 
       switches to the live state. We use a slight, but not uncommon, simplification of the original ruleset, where having two alive neighbors would imply that a cell remains in its current state. The terminology stems from the idea that, defined as above, the game can be used to simulate ‘real life’ processes where living entities tend to die in overcrowded regions whereas also they are not able to survive on their own.
        <TABLE class="image-left"><CAPTION align="bottom">Figure 1.1: the Gosper’s Glider Gun, an example of a discrete automaton.</CAPTION></TABLE><DIV class="headcontainer"><H2>Quorum Sensing</H2></DIV><P>To design living cells that display this behavior we will use the natural ability found in cells to communicate their presence to cells in their proximity and likewise sense the presence of other cells by means of quorum sensing mechanisms. Quorum sensing is used by bacteria to coordinate behaviors based on the density of their population. Bacteria that are capable of quorum sensing constantly produce certain signal molecules called autoinducers. Based on the concentration of these signal molecules bacteria respond, and regulate a number of different processes. Quorum sensing can occur within a single bacterial species, as well as between different species. An example of quorum sensing regulated bacterial behavior is the formation of biofilm <SPAN class="refcontainer">[7]</SPAN></P><DIV class="reference">B.L. Bassler. Small Talk: cell-to-cell communiction in bacteria. Cell, 109, 2002.</DIV>. Another example is the production of signal molecule by Vibrio fischeri to adjust the production of luciferase, a light emitting protein, by cells present in close proximity. There are different types of quorum sensing systems which have their own signal molecules. One class of signal molecules is that of the acyl-homoserine lactone autoinducers. This class of signal molecules is used by different species of bacteria like V. fischeri, and P. aeruginosa. Each species uses a slightly different acyl-HSL, and there is a strong homology in the proteins involved in the quorum sensing systems of acyl-homoserine lactone producing bacteria. For instance TraR in A. tumefaciens, LasR in P. aeruginosa an LuxR in V. fischeri.<P>The acyl-HCL system was first described in V. fischeri <SPAN class="refcontainer">[8]</SPAN></P><DIV class="reference">C.H. Collins, F.H. Arnold, and J.R. Leadbetter. Directed evolution of Vibrio fischeri LuxR for increased sensitivity to a broad spectrum of acyl-homoserine lactones. Molecular Microbiology, 55, 2005.</DIV>. The acyl-HSL of V. fischeri is 3-hexanoyl homoserine lactone (3OC6-HSL)2. 3OC6-HSL is synthesized by the LuxI protein. 3OC6-HSL binds to the transcription regulator LuxR and dimeric LuxR with bound 3OC6-HSL then binds to DNA and activates transcription of the Lux operon, which result in the production of luciferase. In this project the quorum sensing genes luxI and luxR will be used to make so called ‘sender’ and ‘receiver’ cells. This was done before by Basu et al <SPAN class="refcontainer">[9]<DIV class="reference">S. Basu, R. Mehreja, S. Thiberge, M. T. Chen, and R. Weiss. Spatiotemporal control of gene expression with pulse-generating networks. PNAS USA, 101(17):6355–6360, April 2004.</DIV></SPAN>. Here, luxI was put behind a constitutive promoter on a plasmid in one strain, and luxR in another. In the latter the LuxR-HSL complex activates certain genes. This is one of the applications where these quorum sensing genes can be used.<P>Using this signalling mechanism the cells will be defined to have an ON-state, in which they continuously send a signal to their neighbors, and an OFF state in which cells do not send at all. In order for a cell to count the number of neighboring cells in the ON-state we will develop a novel genetic detection mechanism which will be able to distinguish between three types of signals: ‘low’, ‘medium’ and ‘high’. In parallel with Conway’s Game of Life these will be assigned the meaning of respectively ‘less than three neighbors in the ON-state’, ‘three neighbors in the ON-state’ and ‘more than three neighbors in the ON-state’. In the lab we aim to engineer these genetic functionalities in order to work in E.coli cells and additionally determine a physical setup such that the E.coli cultures can live and interact in a grid-like, two dimensional environment as they do in Conway’s Game of Life.</P><DIV class="headcontainer"><H2>To be continued...</H2></DIV><P>As yet we have not succeeded in developing a fully operational system, nevertheless in this report we describe the steps we took towards being able to reproduce Conway’s Game of Life in vivo. The <A href="https://2008.igem.org/Team:Groningen/design.html#interval_switch">the Design/Interval Switch menu</A> describes the design and workings of the detection mechanism, the interval switch. The complete genetic circuit design and the choice of genetic parts are explained in &quot;<A href="https://2008.igem.org/Team:Groningen/design.html#genetic_circuit">Genetic Circuit</A>&quot;. In &quot;<A href="https://2008.igem.org/Team:Groningen/design.html#physical_system">Physical Setup</A>&quot; we provide suggestions for a suitable physical setup in which to incorporate the cells. In &quot;<A href="https://2008.igem.org/Team:Groningen/modeling_SingleCell.html">Single-Cell model</A>&quot; and &quot;<A href="https://2008.igem.org/Team:Groningen/modeling_Spatial.html">Spatial Model</A>&quot; computer modeling work is presented and expectations of the system’s behavior are discussed both on the single cell level as well as involving cellular interactions on a grid (spatial model). And finally, in &quot;<A href="https://2008.igem.org/Team:Groningen/results.html">Results</A>&quot; we provide the results we obtained up till now, discuss these results and provide recommendations for further work on this topic. We also see how we measure up to get some <A href="https://2008.igem.org/Team:Groningen/iGEM_Criteria">medals</A>! Also check out our <A href="https://2008.igem.org/Team:Groningen/team.html">Team</A> and the other options under &quot;About...&quot; </P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Groningen/Introduction">http://2008.igem.org/Team:Groningen/Introduction</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Groningen/Introduction" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Groningen/Introduction" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Groningen/Introduction&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Groningen/Introduction&amp;oldid=103050" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>