<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Groningen_modeling_SingleCell_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Groningen/modeling_SingleCell.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Groningen/modeling_SingleCell.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Groningen/modeling_SingleCell.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Groningen/modeling_SingleCell.html&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Groningen/modeling_SingleCell.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Groningen/modeling SingleCell.html</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV id="header"><DIV id="menu"><UL class="L1"><LI><A href="https://2008.igem.org/Team:Groningen">Home</A></LI><LI><A href="https://2008.igem.org/Team:Groningen/Introduction">Introduction</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/design.html">Design</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/design.html#interval_switch">Interval Switch</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/design.html#genetic_circuit">Genetic Circuit</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/design.html#physical_system">Physical System</A></LI></UL><LI class="sub"><A href=""> Modeling </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/modeling_SingleCell.html"> Single-cell </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/modeling_Spatial.html"> Spatial model </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/modeling_files.html">Model Files</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/results.html">Results</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/results.html#results"> Results </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/results.html#conclusion"> Conclusion </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/iGEM_Criteria">iGEM Criteria</A></LI><LI class="sub"><A href="">About...</A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/team.html"> The Team </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/rug.html"> Groningen University </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/references.html"> References </A></LI><LI class="sub"><A href="https://2008.igem.org/Team:Groningen/acknowledgements.html"> Acknowledgements </A></LI></DIV></DIV><DIV id="modeling_SingleCell" class="location"><DIV id="view"><DIV id="textblock"><H1>Modeling: Single-Cell Approach</H1><DIV class="headcontainer"><H2>Interval Switch, overview</H2></DIV><P>The genetic interval switch is designed in order for a cell to be able to detect three distinct signalling levels,
termed ’low’, ’medium’ and ’high’. This functionality is mainly determined by the use of an inducible/repressible
hybrid promoter combined with a mutated inducible promoter with weakened binding affinity, which
react to the same activating protein. A schematic representation of the parts which, as we will see, constitute
the desired behavior is displayed in figure 6.1.</P><TABLE class="image-center"><CAPTION align="bottom">
                Figure 6.1. Schematic overview of the genetic interval switch.
              </CAPTION></TABLE><P>In this setup the signalling molecule is 3OC6HSL, which can bind to LuxR to form a complex able to induce
transcription downstream of both the hybrid promoter as well as the mutated promoter. Formation of this
complex is directly proportional to the concentration of 3OC6HSL since LuxR is present in abundance. This
due to the gene producing it being downstream of the Tet-promoter, which is constitutively on. The GFP
reporter gene is located downstream of the hybrid promoter and transcription will start when the 3OC6HSLconcentration
reaches a certain threshold. In comparison to this the threshold at which CI production will
start is slightly higher since the promoter’s binding site is weakened by mutation. If the 3OC6HSL concentration
exceeds the latter threshold CI production will strongly quench GFP synthesis since it binds to the hybrid
promoter to prevent further transcription. The behavior of this genetic design, within a single cell, has been
simulated using the Mathworks MATLAB SimBiology R2008b package <SPAN class="refcontainer">[10]</SPAN></P><DIV class="reference">Simbiology. Model, design, and simulate biochemical pathways. The Mathworks Inc. version 2.3 (R2008a)</DIV><P>
For the modeling we used a novel approach, described throughout this text, where the simulation results
have been obtained by one-to-one conversion of Registry parts to ’modelbricks’. A script has been developed
to automatically merge these submodels, obeying PoPS/RiPS standards 
<SPAN class="refcontainer">[10]</SPAN></P><DIV class="reference">D. Endy. Foundations for engineering biology. Nature, 438:449–453, 2005</DIV>
, to be able to study larger systems.
Figure 6.2 shows that this way of modelling naturally fits in with the synthetic biology approach, since
it allows for the modular approach to be used throughout the whole design process. The ’modelbricks’ thus
obtained are exchangeable and reusable in other projects. The hard part of this approach is that modeling
should be done in considerable detail, and therefore many parameters have to be accurately measured. On
the other hand, when a part is well characterized and modelled it can be put in a library of ’modelbricks’,
quite like the Registry, and used in many different projects. Both the script for merging parts and the library
of ’modelbricks’ used for this project are available on our iGEM Wiki <A href="https://2008.igem.org/Team:Groningen/modeling_files.html">(Model Files)</A>.
<TABLE class="image-center"><CAPTION align="bottom">
Figure 6.2: If a computer model is not comprised of parts, bottom left corner, it may be hard to change the design
whithout having to resort to a fully new model. On the other hand, if the nesecarry parts are modeled separately (top
right), this allows for quickly simulating and evaluating different designs since parts can be interchanged in a matter of
seconds. In the design process of synthetic biological systems the path via the top right figure (blue arrows) allows for
better interplay between the different design phases. Whereas along the path via the lower left figure the rigidness of
the modeling phase may sometimes interfere with the flexibility of the total design process.
              </CAPTION></TABLE><DIV class="headcontainer"><H2>Simulation Results</H2></DIV><P>Figures 6.3 6.4 and 6.5 show some of the key results we obtained with this model. The switching mechanism
is apparent from figure 6.3 where we keep track of the states of the hybrid promoters, while slowly increasing
3OC6HSL concentration. These promoters have two binding sites, so-called ’two-operator promoters’.
States are depicted as (’binding site 1’ ; ’binding site 2’) where LuxR can bind exclusively at site 1, inducing
transcription, while CI can bind exclusively at site 2, preventing transcription (regardless of the state of site
1). Partcularly in figure 6.5 the sought for response curve of the interval switch is clearly present. Although,
as is apparent in figure 6.4, the system is expected to display undesirable transient effects. Additionally,
with these parameters, we expect the peak to be quite narrow and centered around an extremely high HSL
concentration. These features can be changed, for instance by altering promoter binding affinities. But the
parameters as they are presented below are based either on literature or educated guesses. And the fact
that they do lead to, at least qualitatively right, behavior was an indication to progress with the wetwork. The
modular approach used led to results which are qualitatively identical to modelling the system using a more
classical approach (the small difference in results seem due to using different parameter sets). The method
used for merging the submodels will of course not work for all types of genetic parts but we tend to believe
that it can work for all genetic systems consisting of (inducible/repressible) promoters, RBS’s, coding regions
(genes) and terminators (and it seems that these four types of parts cover most of the iGEM projects).</P><TABLE class="image-center"><CAPTION align="bottom">
Figure 6.3. State information of the hybrid promoters, obtained by slowly increasing HSL concentration within the cell.
At low input concentrations no inducers are bound to the promoter binding sites so no transcription is taking place; at
medium concentrations a small part of the promoters will be bound by luxR.HSL complex, causing transcription of the
GFP gene; at higher concentrations GFP expression is quenched due to the prescence of CI causing transitions from the
(LuxR.HSL;unbound)-state to the (LuxR.HSL;cI)-state.
              </CAPTION></TABLE><TABLE class="image-center"><CAPTION align="bottom">
Figure 6.4. GFP output concentration versus time, t=0 defined to be the moment of HSL-induction. Initial transient
response of the system prevents immediate measurement of the cell’s state. This effect can possibly be overcome by
introducing a feed-forward loop, but allowing the system to relax by measuring one hour after induction suffices to yield
the desired response peak.
              </CAPTION></TABLE><TABLE class="image-center"><CAPTION align="bottom">
                Figure 6.5. Resulting response curve obtained by taking ’snapshots’ of the system, for a range of input concentrations,
at one hour after induction. With the parameters used, partly taken from literature and partly guessed, the band is
clearly observable although it is narrow. The width of the band is tunable by altering the (mutated) LuxPr binding affinity.
The leakiness above the upper threshold stems from ongoing competition between LuxR.HSL and CI for the operator
binding sites, the value being one third of the maximum response is acceptable. The peak is centred around a HSL
concentration of 1e5nM, which is unrealistically high when compared to values used in comparable experiments.
              </CAPTION></TABLE><DIV class="headcontainer"><H2 id="modelbrick">Detailed Description of the Model</H2></DIV><P>The modeling technique used here is based on the work ofMarchisio and Stelling 
<SPAN class="refcontainer">[21]</SPAN></P><DIV class="reference">M.A. Marchisio, and J. Stelling. Computational design of synthetic gene circuits with composable parts. Bioinformatics, 24(17):1903–1910, 2008</DIV>
    .Within this modelling
scheme every registry part has been modeled in detail to act as an individual unit, with its own inner structure,
which can be connected to other parts via its input and output ’channels’. We want to show that these
connections can be established according to a set of fixed rules. The rules, although they will certainly not be
universal, for obvious reasons, are thought to be applicable to at least a certain class of synthetic biological
devices. In addition to the registry part units, the intracellular environment will be represented by similar
connectable units termed ’pools’. Finally all these units should be embedded in a chassis, representing the
physical cell.<TABLE class="image-center"><CAPTION align="bottom">
Figure 6.6. An example of connecting the parts and pools in a genetic circuit. Red arrows indicate the flow of Polymerases
per second (PoPS), while blue arrows indicate Ribosomes per second (RiPS) flow.
              </CAPTION></TABLE><P>The role of the pools is displayed in figure 6.6. Two transcription regions are shown, A and B. A can be induced
by an external signal, while B is induced by the product of A. All freely moving objects in this picture either
reside in their pool or are taken up by the system when needed for the transcription process. The diagram
shows that each part stands alone except for a few channels connecting it to some other parts. If we observe
the connections we note: promoters couple to RNAp-pool, RBS, and perhaps an inducer molecule pool; RBS’s
connect always to promoter, ribosome-pool and gene coding regions; Coding regions always connect to RBS,
their product molecule pool, ribosome-pool and terminator; And terminators always connect to coding region
and RNAp-pool. For this specific circuit the connections can be made using an algorithm, provided there
is knowledge about which molecule induces/represses which promoter and which molecule is produced by
which coding region. But since this is a fixed property of promoters and coding regions the information can
be incorporated in their ’modelbricks’.</P><P>In the description of the ’modelbricks’ -most parameter choices are based on 
<SPAN class="refcontainer">[20]</SPAN></P><DIV class="reference">S. Hooshangi, S. Thiberge, and R. Weiss. Ultrasensitivity and noise propagation in a synthetic transcriptional cascade. PNAS USA, 102(10):3581–3586, 2005.</DIV>
           and 
<SPAN class="refcontainer">[21]<DIV class="reference">M.A. Marchisio, and J. Stelling. Computational design of synthetic gene circuits with composable parts. Bioinformatics, 24(17):1903–1910, 2008</DIV></SPAN>
           , only
the parameters governing the promoter with the weakened operator had to be guessed, since at the time of
writing this part had not been fully developed and tested.<DIV class="headcontainer"><H2>E.coli Chassis</H2></DIV><P>
This is the cellular environment of the device. Next to the features which are characteristic for the E.coli
strain used, the chassis contains so called transcription factor and signalling pools. The pool symbolizes the
freedom of the species to move around within the cell and therefore its ability to interact with any of the
parts inside the cell, at any time. Transcription factors (TF’s) are species which influence an operator by either
activating or repressing transcription. Signalling molecules are any species that interact with the extracellular
world. The chassis should contain one pool for every TF and signalling species. A species resides in its
pool whenever it is not interacting with any of the parts, decay and particular transformation processes (e.g.
protein maturation, polymerization) take place exclusively inside the pool. In this modeling scheme it is considered
sufficient to characterize a particular strain by its polymerase and ribosome contents and its volume.
The polymerases and ribosomes each have their own pool, which are assumed to be present in the chassis
by default.
</P><DIV class="headcontainer"><H2>Pools</H2></DIV><P>
Next to the standard ribosome and polymerase pools the chassis contains two transcription factor pools, for
LuxR and CI, and two signalling pools, for 3OC6HSL and GFP. Below is a list of all of the species and parameters
used in these pools, this property of the species is emphasized by means of the adjective ’floating’.
</P><P> The detailed discription of all the modelbricks and their pools can be found <A href="https://static.igem.org/mediawiki/2008/f/f7/Groningen2008_Modelling_details.pdf"> in this appendix.</A></P><P>The diagram above (click for full version) shows the structure of the complete, assembled interval switch. It can also be found in the .sbproj-file in the <A href="https://2008.igem.org/Team:Groningen/modeling_files.html">'model files' section</A>. It is recommended when working with the model to set up the Simbiology diagram in a similarly structured way.</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Groningen/modeling_SingleCell.html">http://2008.igem.org/Team:Groningen/modeling_SingleCell.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Groningen/modeling_SingleCell.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Groningen/modeling_SingleCell.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Groningen/modeling_SingleCell.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Groningen/modeling_SingleCell.html&amp;oldid=100030" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>