<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Genspace_Measurement skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Genspace/Measurement</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Genspace">HOME </A></LI><LI class="menu_item"> TEAM
			</LI><LI><A href=" https://2016.igem.org/Team:Genspace/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Collaborations">★  Collaborations </A></LI></UL><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:Genspace/Description"> ★  Description </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Proof"> ★ Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Demonstrate"> ★ Demonstrate </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Notebook"> Notebook </A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:Genspace/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Composite_Part"> ★ Composite Parts </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Genspace/Safety"> SAFETY </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Genspace/Attributions">★  ATTRIBUTIONS </A></LI><LI class="menu_item"> HUMAN PRACTICES 
			</LI><LI><A href="https://2016.igem.org/Team:Genspace/Human_Practices"> Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/HP/Silver">★ Silver </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/HP/Gold">★ Gold </A></LI><LI class="menu_item"> AWARDS 
			</LI><LI><A href="https://2016.igem.org/Team:Genspace/Measurement">★  Measurement </A></LI></DIV><DIV class="content_wrapper"><DIV class="column full_size"><H1>Measurement Award</H1><DIV class="content"><H2 class="c0"><SPAN class="c8">Background</SPAN></H2><P class="c0"><SPAN>The plasmid pSB1C3 is a favored vector for iGEM projects, its high copy number making it a workhorse for gene expression.  However, different sources give different estimates of its copy number and no iGEM teams have independently verified these claims.  Genspace developed a qPCR assay to verify plasmid copy number (PCN) in </SPAN><SPAN class="c5">E. coli</SPAN><SPAN> and made the first measurements of pSB1C3.</SPAN></P><P class="c0"><SPAN>The pSB1XX plasmid series uses the pMB1 origin or replication.  A point mutation in the replication primer, RNA II, allows the PCN to exceed 100 copies per cell, unless replication is repressed by the </SPAN><SPAN class="c5">rop</SPAN><SPAN> protein (Lin-Chao et al. 1992).  Plasmids encoding the </SPAN><SPAN class="c5">rop</SPAN><SPAN> gene, such as pBR322, have a much lower copy number, typically closer to 18 copies per cell (</SPAN><SPAN class="c9"><A class="c10" href="https://www.google.com/url?q=https://drive.google.com/file/d/0B7L2OR2SM-b0QVNjRDBPdEFBclk/view?usp%3Dsharing&amp;sa=D&amp;ust=1476920272252000&amp;usg=AFQjCNFtU0kiUZdEskSgBWK9OarEFXUiYQ">Lee et al 2005</A></SPAN><SPAN>).  Additionally, incubation temperature has been shown to alter the copy number of plasmids using this origin of replication.</SPAN></P><P class="c0"><SPAN>To minimize variability caused by loss of nucleic acid during purification, a lysis protocol was researched that would allow cell lysate to be used directly as the template for qPCR (</SPAN><SPAN class="c9"><A class="c10" href="https://www.google.com/url?q=https://drive.google.com/file/d/0B7L2OR2SM-b0dFRMYnlmVW9XM1E/view?usp%3Dsharing&amp;sa=D&amp;ust=1476920272254000&amp;usg=AFQjCNGubsJiD7ltmNgXLmUaD_U_F9xYWA">Shatzkes et al 2014</A></SPAN><SPAN>).  A TaqMan® hydrolysis probe was designed to target the LacZ gene and LacZ (BBa_K909006) was cloned into pSB1C3.  Because a copy of LacZ exists on the </SPAN><SPAN class="c5">E. coli</SPAN><SPAN> chromosome, lysate generated from cells without the plasmid was used as a standard, while lysate from cells with the plasmid was expected to have PCN+1 copies of the target sequence.</SPAN></P><P class="c0"><SPAN>In addition to measuring plasmid copy number, it was decided that steady-state mRNA count would also be a valuable measurement to make.  While we did not have enough time to make any mRNA measurements, the assay was designed with the intention of being usable on a DNA or RNA template.  To support this flexibility, the Verso One-Step qRT-PCR kit was chosen for our assay.  The Verso kit comes with an RT enhancer that degrades dsDNA during the reverse transcription step, allowing users to skip DNase treatment of samples.  Used as directed, the kit works for qRT-PCR, while withholding the RT enhancer and skipping the RT stage allows the kit to function as a qPCR assay.  Because samples do not require DNase treatment, both qPCR and qRT-PCR can be run using the same sample.</SPAN></P><P class="c0"><SPAN>Video: How TaqMan Works</SPAN></P><P class="c0"><SPAN class="c9"><A class="c10" href="https://www.google.com/url?q=https://www.youtube.com/watch?v%3DfkUDu042xic&amp;sa=D&amp;ust=1476920272256000&amp;usg=AFQjCNFrASby-X8IPgbQQIR3I04LSZ6JFw">https://www.youtube.com/watch?v=fkUDu042xic</A></SPAN></P></DIV><DIV class="content"><H2 class="c0"><SPAN class="c8">Sample Preparation Procedure</SPAN></H2><P class="c0"><SPAN>Bacterial cultures for testing were grown overnight to saturation, and the OD</SPAN><SPAN class="c12">600</SPAN><SPAN> was measured using the following formula:</SPAN></P><P class="c0"><SPAN>OD</SPAN><SPAN class="c12">600</SPAN><SPAN> of 1.0 = 8x10</SPAN><SPAN class="c1">8</SPAN><SPAN> cells/mL</SPAN></P><P class="c0"><SPAN>When necessary, 10-fold dilutions were prepared to ensure optical density remained within the linear range of our spectrophotometer.</SPAN></P><P class="c0"><SPAN>200,000,000 cells were taken and pelleted, then resuspended in 1mL of CL Buffer for a final concentration of 1 million cells per 5 microliters.  Cells were lysed at room temperature for at least 5 minutes, then frozen at -20°C.  10-fold dilutions of lysate were made using CL buffer to reduce DNA concentration to a manageable amount for qPCR.  For the standard, 20,000,000,000 cells were pelleted and lysed, and 10-fold serial dilutions in CL buffer were prepared.</SPAN></P></DIV><DIV class="content"><H2 class="c0"><SPAN class="c8">qPCR Procedure</SPAN></H2><P class="c0"><SPAN>Reactions were set up using the chemistry described in the kit’s manual.  A master mix was created and 20uL were distributed to each reaction tube.  5uL of cell lysate (containing 10</SPAN><SPAN class="c1">5</SPAN><SPAN> cells for the test articles) were added to each reaction.  A 3-point standard was used consisting of 10</SPAN><SPAN class="c1">5</SPAN><SPAN>, 10</SPAN><SPAN class="c1">6</SPAN><SPAN>, and 10</SPAN><SPAN class="c1">7</SPAN><SPAN> lysed </SPAN><SPAN class="c5">E. coli</SPAN><SPAN> cells without the plasmid.  All samples were run in triplicate.  The reverse transcription step was skipped and reverse transcriptase enzyme was inactivated during the 95°C hot start and qPCR continued normally using DNA as a template.</SPAN></P><P class="c0"><SPAN class="c3">Each reaction has the following chemistry:</SPAN></P><P class="c0 c4"><SPAN class="c3">12.5uL Verso qRT-PCR Mix</SPAN></P><P class="c0 c4"><SPAN class="c3">5.45uL Molecular Biology Grade Water</SPAN></P><P class="c0 c4"><SPAN class="c3">5.0uL Cell lysate</SPAN></P><P class="c0 c4"><SPAN class="c3">1.8uL TaqMan Primer/Probe Mix</SPAN></P><P class="c0 c4"><SPAN class="c3">0.25uL Verso Enzyme Mix</SPAN></P><OL class="c11 lst-kix_9c2vkm96i5w4-0 start" start="1"><LI class="c0 c7"><SPAN class="c3">To minimize reaction variability, make a master mix first by multiplying the above volumes (withholding lysate) by the number of reactions you plan to run, with an excess of 2-3 reactions' worth.</SPAN></LI><LI class="c0 c7"><SPAN class="c3">Distribute 20uL of master mix into each reaction tube.  Because the edge effect impairs data quality in the outer tubes, avoid using them for anything other than a positive or negative control</SPAN></LI><LI class="c0 c7"><SPAN class="c3">Distribute 5uL of the appropriate lysate (standard or test) to each reaction tube.  For the negative control, use 5uL of CL Buffer.</SPAN></LI></OL><P class="c0"><SPAN class="c3 c8">qPCR Cycles</SPAN></P><P class="c0"><SPAN class="c3">Run qPCR with the following cycles:</SPAN></P><P class="c0 c4"><SPAN class="c3">95C for 15 minutes</SPAN></P><P class="c0 c4"><SPAN class="c3">40 cycles of:</SPAN></P><P class="c0 c6"><SPAN class="c3">95C for 15 seconds</SPAN></P><P class="c0 c6"><SPAN class="c3">60C for 1 minute</SPAN></P></DIV><DIV class="content"><H2>Part 3. Plasmid Copy Measurement</H2><P>A preliminary experiment was run using lysate from 1 million cells per reaction.  Eight replicates were run for each sample type (with or without the reporter).  After excluding outliers caused by edge effects, average CT values for six replicates for each sample were compared:</P><P>Test:                17.02</P><P>Control:        21.48</P><DIV class="sub-content"><P class="c0"><SPAN><B>pSB1C3 absolute quantification run #1</B></SPAN></P><P>Lysate from 1 million stationary phase cells harboring K909006-pSB1C3 was run against a 3-point standard of 106, 107, and 108 copies.  Linear regression indicates approximately 18.2 copies of the target sequence for every cell in the reaction, or around 17 plasmid copies per cell.</P><P class="c0"><SPAN><B>pSB1C3 absolute quantification run #2</B></SPAN></P><P>Lysate from 100,000 mid-log phase cells harboring K909006-pSB1C3 was compared against a 3-point standard of 105, 106, and 107 copies.  Due to the reduced amplification efficiency of the 108-copy standard in run 1, cell numbers were reduced 10-fold in all subsequent experiments.  Linear regression indicates approximately 13.4 copies of the target sequence for every cell in the reaction, or around 12-13 plasmid copies per cell.</P><P>Note: The K909006-pSB1C3 harboring cells used for this run were lysed in mid-log phase, which may account for the reduced PCN.</P><P class="c0"><SPAN><B>pSB1C3 absolute quantification run #3</B>Lysate from 100,000 stationary phase cells harboring K909006-pSB1C3 was compared against a 3-point standard of 105, 106, and 107 copies.  Linear regression indicates approximately 30.9 copies of the target sequence for every cell in the reaction, or around 30 plasmid copies per cell.</SPAN></P><P class="c0"><SPAN><B>pSB1C3 absolute quantification run #4</B></SPAN></P><P>Lysate from 100,000 stationary phase cells harboring K909006-pSB1C3 was compared against a 2-point standard of 105 and 106 1.1x106 copies.  The 1.1x106-copy standard was created using lysate from 105 cells as well as 106 copies of purified plasmid.  This point was created to test for variance in amplification efficiency of plasmid vs. genomic template.  Linear regression indicates approximately 25.5 copies of the target sequence for every cell in the reaction, or around 24-25 plasmid copies per cell.

		Three qPCR runs using stationary phase cells and one run using mid-log phase cells indicate a PCN of around 12-13 copies during log growth, increasing to around 24 copies per cell during stationary phase.
		</P>
					
					Please refer to the <A href="https://github.com/genspace/iGEM-qPCR-Copy-Number-Analysis/blob/master/qpcr_pSB1C3_Absolute_Quantification.ipynb">Jupyter Notebook</A> for qPCR copy number analysis.

				</DIV><DIV class="sub-content"><H3>Gel Electrophoresis of Cell Lysate</H3>
					Because qPCR consistently gave results significantly lower than the expected copy number, an alternate means of direct copy number analysis was attempted.

					1 billion cells were pelleted and resuspended in 100uL of CL Buffer.  At a size of 5339bp, 20 billion copies of K909006-pSB1C3 should weigh 115ng at a concentration of 1.15ng/uL.  10uL of lysate from E. coli  was run on a 0.6% agarose gel with purified K909006-pSB1C3 at known concentrations for comparison of band brightness.

					

					Lanes were loaded in the following order (left to right)
					E. coli Top 10 with no plasmids
					10ng purified K909006-pSB1C3
					E. coli Top 10 harboring K909006-pSB1C3
					50ng purified K909006-pSB1C3

					The band for the plasmid in E. coli lysate was much closer in brightness to the 10ng band than the 50ng one, indicating that close to 20 copies were harbored in each lysed cell.
				</DIV><H5 class="c0"><SPAN class="c8">Conclusions</SPAN></H5><P class="c0"><SPAN>Due to time constraints, only a few qPCR runs could be completed prior to the Wiki freeze, so this data is still preliminary.  Additional sets should be run to determine the reproducibility of these results.  Furthermore, the range of copy numbers detected for stationary phase cultures suggests that the copy number may take some time to stabilize after cell division halts.</SPAN></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>