Team:Genspace/Description
MENU ▤
HOME
TEAM
Team
★  Collaborations
PROJECT
★  Description
Experiments
★ Proof of Concept
★ Demonstrate
Results
Notebook
PARTS
Parts
★ Basic Parts
★ Composite Parts
SAFETY
★  ATTRIBUTIONS
HUMAN PRACTICES
Human Practices
★ Silver
★ Gold
AWARDS
★  Measurement
Project Description
Summary
Tardigrades or “water bears” are one of nature’s best examples of durability. These creatures have the ability to withstand temperatures ranging from -272 to 150 degrees Celsius, pressures approximately six times that found at the deepest parts of the ocean, and solar radiation equivalent to 250 times the amount that would prove fatal to humans. One of the key mechanisms involved in the tardigrade’s ability to endure such harsh conditions is called cryptobiosis. Cryptobiosis is the ability of certain organisms to survive in an inanimate state while nearly halting their metabolic processes. During cryptobiosis in tardigrades, these creatures assume a contracted position, lose greater than 95% of their body’s water, and synthesize protective proteins and sugars.
Our project has three main parts.
Desiccation Tolerance: Transportation and storage of medical and industrial proteins can be an issue due to instability. Improvement of the storage stability of samples of valuable microbes in areas outside the cold chain is also a concern. To help address these needs, we cloned several tardigrade proteins thought to be responsible for their amazing resilience and transferred them to other organisms to see if they provided protection from desiccation.
Tardigrade Model: Since tardigrades are somewhere between Caenorhabditis elegans and Drosophila melanogaster on the tree of life, they are attractive models for studying development. We investigated how this might be achieved by using CRISPR to knock out tardigrade genes implicated in development.
Plasmid Copy Measurement: Gold Medal Requirement Number 2 We developed a novel qPCR-based method for measuring plasmid copy number and in doing so improved the characterization of the part pSB1C3, the standard iGEM plasmid backbone.
Part 1. Can Tardigrade Proteins Protect Other Organisms?
One class of proteins found to be expressed during cryptobiosis are the intrinsically disordered proteins (IDPs). These proteins have no rigid tertiary structure while in an aqueous environment but quickly adapt a fixed conformation upon dehydration. It is these fixed conformations that are thought to prevent the destructive gathering or aggregation of cellular components upon desiccation. After identifying several IDPs, we sought to express their genes in Escherichia coli to measure the increased desiccation resistance conferred by these proteins as well as provide a possible means of large-scale production of such proteins.
Most expenses involved in delivering vaccines to developing nations are in the refrigeration of these vaccines. Circumventing these expenses may lead to wider and cheaper vaccine distribution in areas with poor infrastructure. If tardigrade IDPs protect cellular proteins from aggregation during desiccation, mixing these proteins in solution with vaccine antigens or attenuated microbial species prior to vaccine desiccation may stabilize the vaccine and prolong its room-temperature shelf life.
IDPs: The Magic Proteins
IDPs are ubiquitously found in a many stress-resistant organisms. Given that IDPs are a major factor in tardigrade resistance to desiccation and other stresses, our team began with research into IDPs. Our goal was to identify tardigrade IDP-coding genes that would confer protection from desiccation when expressed in E. coli. Because tardigrade IDP genes are not well understood, we also thought it would be valuable to explore better-characterized IDP genes from related organisms.
IDP proteins in the late embryogenesis abundant LEA protein family were first shown to confer resistance to osmotic stress in cotton seeds by Dure et.al. in their 1981 paper entitled "Developmental biochemistry of cottonseed embryogenesis and germination: changing messenger ribonucleic acid populations as shown by in vitro and in vivo protein synthesis”. The name late embryogenesis abundant (LEA) is due to the build up of these proteins in the late embryogenesis stage of development. In 2011, Chakrabortee reported that nematode and plant LEA proteins protect cells during desiccation stress by preventing protein aggregation.
The 2011 Chakrabortee paper investigated the IDPs AavLEA1 from the nematode Aphelenchus avenae and Em from Triticum aestivum (common wheat). Because of the close relatedness between nematodes and tardigrades, we focused on AavLEA1.
Other candidate genes for E. coli expression were RVLEAM and MAHS, which are protectants found in the tardigrade species Ramazzottius varieornatus. We focused on these because they have been shown in “Novel Mitochondria-Targeted Heat-Soluble Proteins” (Tanaka et. al. 2015) to improve osmotic tolerance of human cells.
In addition to these previously described proteins, we also chose three genes coding for LEA proteins from Hypsibius dujardini that are not represented in the current literature, which we designated as HDLEA1, HDLEA2 and HDLEA3. These H. dujardini unique genes were chosen because each represents a different cluster of IDPs that are similar across many tardigrade species. By desiccating and rehydrating E. coli expressing these genes, we were able to test for tolerance of osmotic stress conferred by these proteins.
Part 2. Using Tardigrades as a Developmental Model
Although organisms such as D. melanogaster and C. elegans are the conventional model organisms used to study development, the unique abilities of tardigrades compared to these two organisms enable us to potentially traverse uncharted terrain in the differential expression of genes upon development.
To explore tardigrades as a possible developmental organism, our team set out to be the first to use the CRISPR-Cas9 system in the most well-studied species of tardigrades, H. dujardini. It is our hope that the efforts of our team will help unravel the wealth of benefits available to us upon the capitalization of biomimetics as well as encourage future discoveries in the creature that carries the deceptively unpropitious epithet, “the water bear”.
Using CRISPR - Cas9 for Germline Editing
CRISPR-Cas9 is recently discovered system used to knock out genes, thereby allowing us to study their function. Based on the success of transferring D. melanogaster protocols for siRNA as shown in Tenlen 2013, we hypothesized protocols for CRISPR-mediated genome manipulation could be similarly applied. If the genes of H. dujardini are amenable to CRISPR editing, then this system could be used to discover what genes are responsible for the tolerance to extreme conditions that this organism possesses.
Two components of the CRISPR-Cas9 that are vital to the system working are the Cas9 nuclease along with guide RNAs (gRNAs) targeting the gene of interest.  In our strategy, we microinjected the Cas9 protein with gRNAs. These gRNAs and Cas9 suspensions were injected into the gonads of tardigrades.
Part 3. Measurement of Copy Number of pSB1C3
Gold Medal Requirement Number 2
The backbone pSB1C3 is one of the workhorses of most iGEM projects as well as the backbone of choice for constructs stored in the registry.  However, no iGEM team has ever reported any attempt to verify its presumed copy number (100-300 copies per cell).  Our team is the first to have developed a qPCR assay to measure the copy number of this backbone in the E. coli strain Top 10, a variant of the DH10B strain provided by Life Technologies.
The plasmid pSB1C3 is a favored vector for iGEM projects, its high copy number making it a workhorse for gene expression.  However, different sources give different estimates of its copy number and no iGEM teams have independently verified these claims.  Genspace developed a qPCR assay to verify plasmid copy number (PCN) in E. coli and made the first measurements of pSB1C3.
The pSB1XX plasmid series uses the pMB1 origin or replication.  A point mutation in the replication primer, RNA II, allows the PCN to exceed 100 copies per cell, unless replication is repressed by the rop protein (Lin-Chao et al. 1992).  Plasmids encoding the rop gene, such as pBR322, have a much lower copy number, typically closer to 18 copies per cell ( Lee et al 2005 ).  Additionally, incubation temperature has been shown to alter the copy number of plasmids using this origin of replication.
To minimize variability caused by loss of nucleic acid during purification, a lysis protocol was researched that would allow cell lysate to be used directly as the template for qPCR ( Shatzkes et al 2014 ).  A TaqMan® hydrolysis probe was designed to target the LacZ gene and LacZ (BBa_K909006) was cloned into pSB1C3.  Because a copy of LacZ exists on the E. coli chromosome, lysate generated from cells without the plasmid was used as a standard, while lysate from cells with the plasmid was expected to have PCN+1 copies of the target sequence.
In addition to measuring plasmid copy number, it was decided that steady-state mRNA count would also be a valuable measurement to make.  While we did not have enough time to make any mRNA measurements, the assay was designed with the intention of being usable on a DNA or RNA template.  To support this flexibility, the Verso One-Step qRT-PCR kit was chosen for our assay.  The Verso kit comes with an RT enhancer that degrades dsDNA during the reverse transcription step, allowing users to skip DNase treatment of samples.  Used as directed, the kit works for qRT-PCR, while withholding the RT enhancer and skipping the RT stage allows the kit to function as a qPCR assay.  Because samples do not require DNase treatment, both qPCR and qRT-PCR can be run using the same sample.
References
Altiero T, and Rebecchi L. "Rearing Tardigrades: Results and Problems." Zoologischer Anzeiger 240 (2001): 217-21.
Boothby TC, Tenlen JR, Smith FW, Wang JR, Patanella KA, Nishimura EO, Tintori SC, Li Q, Jones CD, Yandell M, Messina DN, Glasscock J, and Goldstein R. “Evidence for Extensive Horizontal Gene Transfer from the Draft Genome of a Tardigrade.” PNAS 112 (2015): 15976-15981.
Chang AC, and Cohen SN. "Construction and Characterization of Amplifiable Multicopy DNA Cloning Vehicles Derived from the P15A Cryptic Miniplasmid." Journal of Bacteriology 134.3 (1978): 1141-156.
Chakrabortee S, Tripathi R, Watson M, Kaminski Schierle GSK, Kurniawan DP, Kaminski CF, Wise MJ, and Tunnacliffe A. “Intrinsically disordered proteins as molecular shields.” Molecular BioSystems 8 (2012):210-219.
Farboud B, and Meyer B. "Dramatic Enhancement of Genome Editing by CRISPR/Cas9 Through Improved Guide RNA Design." Genetics 199 (2015): 959-71.
Förster F, Bessier D, Grohme MA, Liang C, Mali B, Siegl AM, Engelmann JC, Shkumatov AV, Schokriae E, Müller T, Schnölzer M, Schill RO, Frohme M, and Dandekar T. "Transcriptome Analysis in Tardigrade Species Reveals Specific Molecular Pathways for Stress Adaptations." Bioinformatics and Biology Insights 6 (2012): 69-96.
Gabriel WN, McNuff R, Patel SK, Gregory TR, Jeck WR, Jones CD, and Goldstein R. "The Tardigrade Hypsibius Dujardini, a New Model for Studying the Evolution of Development." Developmental Biology 312 (2007): 545-59.
Guidetti R, Altiero T, and Rebecchi L. "On Dormancy Strategies in Tardigrades." Journal of Insect Physiology 57 (2011): 567-76. National Center for Biotechnology Information. U.S. National Library of Medicine, 12 Mar. 2011. Web. 15 July 2016.
Hand SC, Menze MA, Toner M, Boswell L, and Moore, D, "LEA Proteins during Water Stress: Not Just for Plants Anymore" (2011). Faculty Research & Creative Activity. Paper 53.
Koutsovoulos G, Kumar S, Laetsch DR, Stevens L, Daub J, Conlon C, Maroon H, Thomas F, Aboobaker A, and Blaxter M. "No Evidence for Extensive Horizontal Gene Transfer in the Genome of the Tardigrade Hypsibius Dujardini." PNAS 113.9 (2016): 5053-058.
Lin-Chao S, Chen WT, and Wong TT. "High Copy Number of the PUC Piasmid Results from a Rom/Rop-suppressible Point Mutation In RNA II." Molecular Microbiology 6.22 (1992): 3385-393. Web.
Mali et al.: Transcriptome survey of the anhydrobiotic tardigrade Milnesium tardigradum in comparison with Hypsibius dujardini and Richtersius coronifer. BMC Genomics 2010 11:168.
Paix A, Folkmann A, Rasoloson D, and Seydoux G. "High Efficiency, Homology-Directed Genome Editing in Caenorhabditis Elegans Using CRISPR-Cas9 Ribonucleoprotein Complexes." Genetics 201 (2015): 47-54.
Schill RO, Mali B, Dandekar T, Schnölzer M, Reuter D, and Frohme M. "Molecular Mechanisms of Tolerance in Tardigrades: New Perspectives for Preservation and Stabilization of Biological Material." Biotechnology Advances 27 (2007): 348-52.
Shatzkes K, Teferedegne B, and Murata H. "A Simple, Inexpensive Method for Preparing Cell Lysates Suitable for Downstream Reverse Transcription Quantitative PCR." Scientific Reports 4.4659 (2014): n. pag.
Sternberg SH, Redding S, Jinek M, Greene EC, and Doudna JA. "DNA Interrogation by the CRISPR RNA-guided Endonuclease Cas9." Nature 507.7490 (2014): 62-67.
Tanaka S, Tanaka J, Miwa Y, Horikawa DD, Katayama T, Arakawa K, et al. (2015) Novel Mitochondria-Targeted Heat-Soluble Proteins Identified in the Anhydrobiotic Tardigrade Improve Osmotic Tolerance of Human Cells. PLoS ONE 10(2):e0118272. doi:10.1371/journal.pone.0118272
Tenlen JR, McCaskill S, and Goldstein R. "RNA Interference Can Be Used to Disrupt Gene Function in Tardigrades." Dev Genes Evol. 223.3 (2013): 171-81.
Welnicz W, Grohme MA, Kaczmarek L, Schill RO, and Frohme M. "Anhydrobiosis in Tardigrades-The Last Decade." ResearchGate. Journal of Insect Physiology, 1 Apr. 2011. Web. 25 Aug. 2016.
Xu Hua Fu B, Hansen LL, Artiles KL, Nonet ML, and Fire AZ. "Landscape of Target:guide Homology Effects on Cas9-mediated Cleavage." Nucleic Acids Research 42.22 (2014): 13778-3787.
Zhang J, Pritchard E, Hu X, Valentin T, Panilatis B, Omenetto FG, Kaplan, LD “Stabilization of vaccines and antibiotics in silk and eliminating the cold chain”. PNAS, April 12,  2012
Yamaguchi A, Tanaka S, Yamaguchi S, Kuwahara H, Takamura C, et al. (2012) Two Novel Heat-Soluble Protein Families Abundantly Expressed in an Anhydrobiotic Tardigrade. PLoS ONE 7(8): e44209. doi:10.1371/journal.pone.0044209
