Team:Genspace/Notebook
MENU ▤
HOME
TEAM
Team
★  Collaborations
PROJECT
★  Description
Experiments
★ Proof of Concept
★ Demonstrate
Results
Notebook
PARTS
Parts
★ Basic Parts
★ Composite Parts
SAFETY
★  ATTRIBUTIONS
HUMAN PRACTICES
Human Practices
★ Silver
★ Gold
AWARDS
★  Measurement
Genspace iGEM notebook page
Week 1: June 26 - July 2
6/27/16:
Objective: Introduction to bacterial transformation and 3A Assembly (Lab safety training)
Transformed E.coli with RFP insert
Short lesson on 3A Assembly plus first time hands on work
Given three parts: TT10 (upstream), 11NB0015 (downstream), pSB1K3 (backbone)
Results: New iGem members were able to learn how to conduct transformation according to protocol given and also learned something about 3A Assembly.
6/28/16:
Objective: To learn and go over different useful lab techniques and protocols
Todo list: Inspect plates from yesterday's transformation, colony selection + colony PCR, gel electrophoresis, and start overnight from colony PCR
Results: High school students that are new to the team learned about the different protocols and techniques that will be helpful throughout the iGem project.
6/29/16:
Objective: To learn the mechanism and construct of 3A Assembly and digests of the three parts.
Sat down and had a lesson about 3A Assembly
High school students learned the mechanism and the construct behind it
Performed digests on the three given parts: RFP (upstream), GFP ( downstream), pSB1A3 (backbone)
Since the upstream is RFP and downstream GFP the colonies after the 3A Assembly should be orange looking
Results: New members now has a better idea of what 3A Assembly is about and starting to understand the basics of the iGem project.
6/30/16:
Objective: To finish and continue the 3A Assembly from yesterday ( ligation + transformation)
Introduced to ligation and the mechanism behind it
Transformed ligated samples into competent Top10 cells
Executed the transformation protocol
Plated the samples
Learned how to properly label plates and samples
Come back tomorrow to check on the plates
Results: Team members successfully learned how to do a 3A Assembly as well as bacterial transformation.
7/1/16:
Objective: Another lesson on 3A Assembly and learned about the Qubit Fluorometer
Short lesson on 3A Assembly
Learned how to use Qubit to read DNA concentrations
Results: Refreshed members mind about 3A Assembly and learned about a new technique on reading DNA concentrations.
Week 2: July 3 - July 9
7/5/16:
Objective: Desiccation control of E. Coli
Introduced to dehydration protocol, given by mentor Mike Flanagan
Introduced to serial dilution protocol
Verified genes/sequences for synthesis and tardigrade-specific promoters (night meeting)
Results: Conducted our very first desiccation experiment of E.coli.
7/7/16:
Objective: To go over the iGEM project/proposal
Listed and discussed the components of our project as a refresher
Results: Everyone now has an idea of what the project is about and different parts to the project can now be planned out and carry on.
Week 3: July 10 - July 16
7/11/16:
Objective: Learning to and inspecting plates
Discussed completed tasks from the previous week
Introduced to ampicillin aliquot protocol
Made more LB
Herded tardigrades
Started overnight, will perform serial dilutions tomorrow
Results: Learned to herd tardigrades for the first time, something that will be crucial for our later tardigrade tests, and everyone understands the protocols for the more common laboratory activities.
7/12/16:
Objective: Working on serial dilutions placed in overnight
Decided on LEA proteins, how to design gRNAs, and began looking for desiccation viability tests for E. coli (night meeting)
Girls and guys given individual tasks to elucidate our project
Resuspended bacterial colonies to determine survivorship rates after set dehydration time
Discussed possible contamination issues
Results: Getting a head start and went over the desiccation experiment.
7/14/16
Objective: Counting colonies and come up with a survivability curve for the data
Looked over colonies and created semi-log plot to determine survivorship rates
Tested tardigrade negative phototaxis
Made competent cells
Introduced to YPD Agar, produced YPD Agar
Wrangled tardigrades
Results: The desiccation experiment went well and data was collected from each plate. The data then transformed into a graph. Throughout the day more tasks were accomplished and this helps saving time.
7/15/16
Objective: Tardigrade work
Made LB
Introduced to yeast nutrient and yeast medium protocols, executed protocols
Made Chlorella medium - tardigrade food
Poured plates
Results: We learned how to culture tardigrades and prepare Chlorella medium.
Week 4: July 17 - July 23
7/18/16:
Objective: Transforming E. coli with fluorescent proteins (RFP + GFP) and given weekly schedule
Transformed E. coli with RFP and GFP
Executed two different wait time on ice for transformation
Some people left cells on ice for 30 minutes and others 5 minutes
Given schedule of the week from mentor
Monday: transformation of RFP + GFP
Tuesday: check colonies and PCR out parts
Wednesday: PCR clean ups, digest, and ligations
Thursday: transformation from ligation
Friday: view plates and colonies
Results: When using ampicillin backbone, the wait time on ice can be shortened. It does not impact the results for transformation in any significant ways. Thus it saves a lot of time and is still efficient. The given week schedule allow team members to plan ahead and also review what to do everyday.
7/19/16:
Objective: Colony PCR of transformation and second desiccation experiment
View plates from yesterday
Colony PCR of selected colonies from the transformation
Started a second E.coli desiccation experiment
Results: Plates from yesterday came out well and colony PCR was done. Started our second desiccation experiment hoping the results to be almost the same as the first experiment. Aiming for consistency.
7/20/16:
Objective: Qubit reading of plasmids from last week, tardigrades wrangling, and colony PCR for the CRISPR workshop
Helped colony PCR samples for the CRISPR workshop
Qubit reading of last week’s plasmids from the mini preps
Made and poured more LB plates
Tardigrade wrangling
Results: Have more experience on how to use the Qubit and also learned how to isolate tardigrades.
7/21/16:
Objective: Colony counting from desiccation experiment and 3A Assembly of RFP and GFP
Ligated parts from digests on 7/18/16
Counted colonies on plates from the desiccation experiment conducted on 7/19/16
Transformation was done after the ligation
Results: Ligated parts of the digests and continued with the 3A Assembly. The transformation steps went well and the plates will be inspected tomorrow. The colonies from the desiccation experiment was pretty consistent with the first.
Week 5: July 24 - July 30
7/25/16:
Objective: Troubleshooting issues with GFP and RFP transformation
The plates did not come out the way we wanted it to be. So therefore an investigation of what we did for the past week was listed
Day 1: Transforming E.coli - positive control worked therefore the transformation should work
Day 2: Colony selection and Colony PCR - might have selected wrong colonies to work with
Day 3: PCR clean ups - found foreign DNA inside samples after the cleanup was done
Day4: Transformation using legations - possibly wrong backbone/ not enough backbone or possible error with ligation
Used old plates and re selected colonies for Colony PCR
Results: The transformation from yesterday did not turn out the way it's suppose to be. Therefore we came up possible errors that may occurred during different days. As a result we re conducted the experiment from Colony selection hoping that it would work this time.
7/26/16:
Objective: Continuing to troubleshoot
PCR cleanup the RFP and GFP
Made 0.7% agarose gel for viewing the base pairs of the plasmid mini preps
Gel results weren’t great, most likely due to using incorrect pipette size
Plasmid mini prep of the GFP and RFP
Qubit readings of the RFP and GFP after the plasmid mini prep
Digested upstream (RFP), downstream (GFP), and backbone (pSB1A3)
Results: Everyone on the team ended up redoing all the steps from past week. We are hoping that we can identify the problem through this rerun.
7/27/16:
Objective: Finishing troubleshooting
Worked with remaining components of 3A assembly (ligation + transformation)
Produced 0.7% agarose gel to see results of digest
Replicated linearized plasmid backbones using PCR
Began collaborating with Stony Brook University
Results: Last day of troubleshooting our mistakes. Hope everything will turn out the way we wanted tomorrow. The plates will be viewed tomorrow as well.
7/28/16:
Objective: Checking transformed cells, completing linearized backbone, and making yeast media
NEB 2.1 digests of the backbones (1A3, 1C3, and 1K3)
PCR cleanup
Reviewing transformed cells
Positive worked, negative worked, other colonies exist in other plates but it's taking awhile for those colonies to take in the fluorescence
Made yeast media
Discussed Stony Brook collaboration
Gel was ran with the digests of the backbones
Results: The plates came out to be ok. It took the colonies a while to turn red. On the good side the positive and negative controls worked. The gel for chloramphenicol and kanamycin came out good. But the ampicillin did not have the correct base pairs.
7/30/16:
Objective: Receiving and working with materials from Stony Brook
Transformed plasmid into competent cells (top10)
Gave Stony Brook materials for our experiment for replication of our experiment
Confirming they have same results
Began liquid culture of Stony Brook yeast (BY4741)
Results: First collaboration with a different iGem team. This fulfills one of the gold medal requirements.
Week 6: July 31 - August 6
7/31/16:
Objective: Continuing to work with Stony Brook materials
Created 4 aliquots
Took 1 colony that had been transformed with plasmid and turned into liquid culture yesterday
Result is significant amount of colonies, possibly not right amount
Results: First collaboration with a different iGem team. This fulfills one of the gold medal requirements.
8/1/16:
Objective: Cloning Biobricks, Re PCR-ing linearized backbones (ampicillin only), performing colony PCR on plates from 7/27/16, and making yeast competent cells
Transforming Stony Brook yeast into competent cells
Received Zymo Prep Kit competent cell and transformation protocol
Performed colony PCR on plates from 7/27
Result lacked orange color, thus performing another colony PCR
PCR’d linearized pSB1A3 plasmid backbone
Introduced to g-blocks and cloned RFP, AAVLEA3, and pSB1C3 backbone
Results: Received g blocks from IDT and we can now start the iGem project by re suspending those g blocks. The ampicillin backbone from the other day did not turn out great so therefore we re linearized the backbone.
8/2/16:
Objective: Make more competent cells and also run a gel
Competent cells were made and stored in -80C
Planned ahead for human practices event ( outreach program at the NY Hall of Science)
Replicated 1A3 plasmids and ran a gel
Result shows contamination because of presence of genomic DNA
Results: The 1A3 backbone did not work again but luckily we found out the reason why it wasn't working. The freshly made competent cells can now allow us to conduct more bacterial transformation in the next month or so.
8/3/16:
Objective: Restriction digest and ligation, make/pour plates, and creating pSB1A3
Performed restriction digests and ligations
Poured more plates for future experiments
Results: Digested and ligated the g blocks we received the other day and transformation of the ligation will take place tomorrow.
8/4/16:
Objective: Transforming g-blocks and amplifying backbones
Transformed and plated ligations from yesterday
Resuspended and performed restriction digest of latest DNA synthesis
PCR linearization of pSB3K3 and pSB4A3
Running gel with PCR linearization of pSB1A3
Results: The gel came out well for pSB1A3 and we started the linearization for two other backbones. The transformation of the ligation from yesterday was conducted and the plates will be checked on tomorrow morning.
Week 7: August 7 - August 13
8/7/16:
Objective: PCR of ligations and also running a gel
Selected 5 colonies from each plate (plates consisted of either: LEA and backbone; RFP and backbone; or LEA and backbone)
A gel was ran for the selected colonies
Started overnights for the colonies that were good
Results: The gel came out nice for all the samples. Roughly all of the colonies has the correct number of base pairs.
8/8/16:
Objective: Mini Prep of the overnights and Qubit, continuing 3A assembly (ligation + transformation), and tardigrade wrangling
Waking up promoters and ribosome binding sites from iGEM registry
Woke up J0500, B0034, and J04500
Ligation (GFP) and digest (HDLEA3)
Transformation of GFP ligation
Tardigrade wrangling
Mini prepped LEA proteins and RFP cultures
Created glycerol stocks of LEA proteins and RFP
Performed qubit
Prepared mini prepped plasmid DNA for sequencing
Results: The finished 3A Assembly samples are now transformated and mini prepped. We can now prepare those samples and have them ready for sequencing.
8/9/16:
Objective: Continuing 3A assembly, performing colony PCR, making plates, and tardigrade wrangling
Resuspended and digested new DNA (HDLEA2, MAHS)
Ligated and transformed yesterday’s digest (HDLEA3)
Miniprepped overnights from yesterday (RFP, AAVLEA1, RvLEA)
Performed colony PCR on yesterday’s transformations
Made Chlor/LB and SC Leu- plates
Made LB
Wrangled tardigrades
Results: We receive new g blocks today and resuspended them. We finished the ligation from yesterday's transformation and conducted Colony PCR of those samples. During remaining of the time fresh plates were made for the transformation.
8/10/16:
Objective: Performing colony PCR, performing ligation, and analyze gels
Performed colony PCR of GFP 1-5 and HDLEA3
Ligation of HDLEA2 and MAHS
Miniprepped I0500 and qubit analysis
Analyzed plasmid products on gels (AAVLEA3, RFP-071816, RvLEA)
Results: Since the g blocks of different samples came in different times, the steps we’re up to for each sample is different. We continued and picked up from where we left off yesterday.
8/11/16:
Objective: PCRing plasmid products and transforming g-blocks
PCR’d out insert of all plasmid products
AAVLEA3, RFP-071816, RvLEA
Transformed ligated HDLEA2 and MAHS and plated
Ran 1.0% agarose gel to analyze PCR product of GFP and HDLEA3
Made overnight of colony resuspension from yesterday 8/10
Results: Were able to analyze the result of the PCR of GFP and HDLEA3
Week 8: August 14 - August 20
8/15/16:
Objective: Split into teams and worked on different parts (Making new construct, performing colony PCR, and waking up parts)
Resuspended new primer (I0500-B0034-Spe1 RVS)
PCR’d the construct
Colony PCR’d yesterday’s MAHS and HDLEA2 transformations
Started overnight and prepared for sequencing
Ran AAVLEA3, RFP-07186, and RvLEA3 in gel
Prepared AAVLEA3 and RvLEAM for sequencing
Miniprepped and qubit’d HDLEA3-1 and HDLEA3-3
Woke up parts identified on Thursday and made glycerol stock of J04450
Results: By splitting into teams we were able to get different tasks done at a faster rate. Different experiments and protocols took place at once. Unfortunately, the colonies from colony PCR did not have any hits. So therefore the colony PCR of HDLEA2 and MAHS will be repeated tomorrow.
8/16/16:
Objective: To redo colony PCR of HDLEA2 and MAHS. Analyzing primer and linearized plasmid backbones
Ran 1.0% agarose gel for analyzing I0500 primer and 3K3 and 4A3 linearized backbone
Colony PCR’d 5 colonies from each plate of MAHS and HDLEA2
Performed colony selection for overnight of promoters
Performed Dpn1 digest and Qubit of linearized backbones
Results: Unlike yesterday, we were able to get hit on for the MAHS and also HDLEA2. An overnight of both samples were prepared and by tomorrow the overnight will be made into glycerol stock and mini prepped.
8/17/16:
Objective: Miniprepping and ligating digests
Miniprepped L1, L2, and L3 cultures from glycerol stocks (and of pSB1C3 cultures)
Qubit’d miniprepped plasmid DNA
Ran 0.7% gel for HDLEA2, MAHS, RFP, pSB1C3, and L1-L3 miniprep products
Discussed LEA proteins to be amplified and sequenced
Results: Valuable information was gained regarding the miniprep products of the LEA proteins during the gel; crucial information regarding the amplification and sequencing of the LEA proteins was also discussed
8/18/16:
Objective: Split into different teams and worked on different parts.
Amplified LEA proteins AAvLEA1, RvLEAM, and HDLEA3
Transformed HDLEA1 ligation
Suspended primers VF2 and VR
Re Qubit and sequence sample prep
Performed PCR cleanup and qubit of I0500 insert into B0034 plasmid (1A3)
Results: Transformation of the HDLEA1 ligation would be crucial for our later experiments
8/19/16:
Objective: Running gel with PCR products
Ran gel with HDLEA2, MAHS, and RFP products
Results: Valuable information regarding the LEA proteins was obtained through the gel
Week 9: August 21 - August 27
8/22/16:
Objective: Performing restriction digests and reviewing sequencing results (was once again split into different teams)
Performed restriction digest of I0500 and pSB1A3
Performed colony selection, PCR, and made overnight of B0034 and HDLEA1
Created overnight of the colony PCR of HDLEA2 and MAHS
Reviewed sequencing results from Genewiz for pre-3A assembly
Reviewed I0500 and RBS protocol
Digest of LEA proteins with chloramphenicol linearized backbone
Results: Team members were once again split into different teams. Each team had different jobs and worked on different parts. At the same time we were introduced to I0500 and the ribosome binding site protocol. The team is trying to come up with different but new parts to be submitted to the iGem registry.
8/23/16:
Objective: Noticed mistakes and redid necessary steps.
Resuspended G blocks and also ligated + transformed
Ran a gel and performed colony PCR
Ran gel electrophoresis of B0034 and HDLEA1
B0034 was okay, HDLEA1 not good
Re-performed colony PCR of HDLEA1 samples
Started overnight for B0034 samples
Results: Resuspension, ligation, and transformation of the g-blocks would be important for later DNA submission; determining the efficacy of the HDLEA1 meant we knew what we had to work on
8/24/16:
Objective: Performing colony PCR, running a gel, miniprepping and qubitting
Performed colony PCR of I0500, AAvLEA1, RvLEAM, HDLEA3, HDLEA1, and MAHS
Ran gel for digest
Created glycerol stock of B0034
Miniprepped and qubit’d B0034
Results: The colonies we selected for each sample came out to be satisfying. Each of the proteins had at least 1 hit and an overnight was started for each.
8/25/16:
Objective: Performing colony PCR and miniprepping and qubitting LEA proteins
Performed colony PCR of AAvLEA1
Miniprepped HDLEA3 and HDLEA1
Qubit’d HDLEA1, HDLEA3, AAvLEA1, MAHS, and I0500 + B0034
PCR linearization and amplification of HDLEA1, HDLEA3, I0500 + B0034, MAHS, and AAvLEA1
Also created overnights of above constructs for glycerol stocks
Results: Due to the delay in AAvLEA1, every other proteins is a step ahead. Running in parallel colony PCR and gel analysis of AAvLEA1 came out to be satisfying. Overnights were started and it is sure catching up. The other proteins were qubitted and had low concentrations. PCR amplifications were ran trying to increase the concentration. The amplification came ended up well.
Week 10: August 28 - September 3
8/28/16:
Objective: Team brunch and tardigrade hunting
Ran gel for colony PCR of RvLEAM, PCR products of amplified LEA proteins, and I0500
Made glycerol stocks of HDLEA1, HDLEA3, and AAvLEA1
Results: The team brunch brought the team members together to discuss the progression of the project and also different team members brought in different moss and algae samples hoping to find some tardigrades in them. After brunch, a gel was tran for the LEA proteins and the gel came out to be fairly well.
8/29/16:
Objective: Performing Dpn1 digest, PCR cleanups, Qubit analysis, and 3A digest of LEA proteins
Performed Dpn1 digest and PCR cleanup of amplified products
Qubit’d products
Performed 3A digest of HDLEA1, HDLEA3, AAvLEA1, and J04560 (with K880006 as backbone)
Results: The digest of each LEA protein is finished. Tomorrow we will be ligating the parts together hoping the final ligation will give us positive and happy results.
8/30/16:
Objective: PCR clean up of MAHS and RVLEAM, qubit reading of only RVLEAM, ligation of HDLEA1, HDLEA3, AAVLEA1, AND JO4560
Miniprepped RVLEAM (1) and (4)
Qubit readings came out low therefore a digest was done and also ran a gel
Forgot to make glycerol stock for RVLEAM, therefore a new overnight was made
6x ratio ligation of the LEAs
Results: The ligation was successfully finished and we are now ready to transform the LEAs.
8/31/16:
Objective: Made glycerol stock for MAHS and RVLEAM, plasmid miniprep, Qubit reading, transformation of the ligations of the LEAs from 8/30/16
Made glycerol stock of MAHS since it went missing. Made glycerol stock for RVLEAM from new overnight made on 8/30/16
Plasmid miniprepped the two LEAs
Qubit readings came out low, a digest and gel was ran
Results: Transformation of the LEAs is finished and the plates will be ready to view tomorrow. Hope there will be great news!
9/1/16:
Objective: Review plates from the LEAs ligations, colony PCR and gel analyzation of PCR products
Transformation of the LEAs came out great but the colonies were green
Colony PCR was done and a gel was ran
The gel came out fine but there were bands at 3000bp resulting in inconclusive results
Results: The results of the transformed plates fluorescing green indicates the misusage of the promoter, RBS, and backbone part. The goal was to use the K880005 part, but the green colonies show that this was not the part that was ligated. Colony PCR was conducted to analyze the confusion. This concluded that the wrong vector was used for the insert. Thus initiating a rebuilding of parts.
Week 11: September 8 - September 14
9/8/16:
Objective: Rebuilding whole construct: Restriction Digest and Ligation of PCR products with gel analyzation
Restriction digest of MAHS, HDLEA1, HDLEA3, and K880005 from plasmid DNA
Gel analyzation of PCR products
Ligation of inserts to vector (K880005)
Results: Digest of plasmid DNA were analyzed through a gel. The parts were the correct size, thus used for ligation into correct vector part.
9/12/16
Objective: Bacterial Transformation of ligated parts from 9/8/16. Designed new primers to fix problem on HDLEA2 coding sequence.
Bacterial transformation of ligated parts from previous lab day into E. coli.
HDLEA2 had an illegal PstI restriction site in middle of its CDS. Primers were designed using Goldengate assembly to target illegal site and change one base pair on codon.
Results: Problem with HDLEA2 was analyzed and solution was designed. Continuation of fixing the problem with plasmid product from 9/8/16 through bacterial transformation.
9/13/16
Objective: Colony PCR of plates from 9/12/16.
Plates were analyzed and colony PCR was started to see if part was properly built and correct size.
Results: Inspection of transformed colonies on plates. Colony PCR executed to conclude on correct size and part used for ligation.
9/14/16
Objective: Gel analyzation of Colony PCR products and Overnights made from colony suspensions
PCR products from previous day were run on a 0.7% agarose gel to compare base pair size results to those designed.
Overnights were made from picking one sample from each LEA protein (HDLEA1 #1, HDLEA3 #2, AAvLEA1 #8, and MAHS #8)
Results: One colony suspension sample was picked from each protein from colony PCR. Insert size was compared using gel electrophoresis. These samples will be miniprep for sequencing.
Week 12: September 15 - September 21
9/15/16
Objective: Plasmid miniprep of overnights + preparation of sequencing plasmids.
Glycerol stock made of each overnight. Flash freezed in liquid nitrogen.
Plasmid miniprep of left over overnight after glycerol stock.
QuBit analysis of DNA for concentration to prepare for sequencing.
Samples each had forward and reverse primer sent with.
Results: Plasmids that were isolated had high enough concentrations to be sent for sanger sequencing to conclude on the correct built part: Promoter + RBS + 1C3 Backbone + Insert from each protein (PR1_’Insert’)
9/19/16
Objective: Discussion on sequencing results. Focus on one protein to do quality and method control experiment: Restriction digest and ligation of MAHS with bacterial transformation.
Sequencing results came back inconclusive where there were multiple problems among the constructs. We chose to choose one protein to focus on to isolate problems.
Restriction digest of MAHS. Used linearized plasmid from 29, August. 2016.
Ligation of MAHS insert with PR1_ vector.
Bacterial transformation of construct into cells.
Results: Results from sequencing all constructs were inconclusive. Many major problems came up. Chose to focus on one protein to go through the whole process for sequencing to provide an insight to the problems.
*Problem with bacterial transformation: LB was not added to cells prior to 2-hour incubation period. Transformation was redone on 9/20/16
9/21/16
Objective: Colony PCR of plates with gel analyzation.
Colonies were picked from plates for colony PCR.
PCR products were placed on a 0.7% gel to conclude on correct size.
Overnights made of colonies with correct base pairs.
Results: Colony PCR led to two colonies with the correct size. Overnights were made from the colony suspension to be mini prepped for plasmid DNA.
Week 13: September 22 - September 28
9/22/16
Objective: Plasmid miniprep and PCR Amplification from overnights.
Plasmid miniprep of the two overnights after glycerol stock.
QuBit analysis of plasmid DNA. Concentrations were very low.
PCR amplification to follow to up DNA concentration before sending to sequencing.
Results: After plasmid miniprep, the analysis of DNA concentration was low, thus we amplified the plasmid products.
9/24/16
Objective: Start of Microinjection of tardigrades with Cas-9 system.
Travelled to Stony Brook University to use microinjection machine for other part of project regarding the insertion of fluorescent protein into tardigrades.
Tardigrades needed to be isolated and placed on microinjection slide to be injected.
The lab was based on microinjection C. elegans, which were microscopically larger than tardigrades, thus there were problems on injecting them without them sliding around.
Results: New slides were made where a coverslip was placed on top and adhesed to a slide to prevent tardigrades from slipping. These slides would be used for next injection trip.
9/26/16
Objective: To add downstream of parts to PRI__X__1C3 and to start overnight of PRI_MAH(S)_1C3
Linearization and restriction digest of PRI_1C3
Linearization, DpnI, PCR clean up of LEA proteins (HDLEA1, HDLEA3, and AAvLEA1)
Ran a gel for the digests
Results: The gel for each of the samples came out nice and all samples loaded have the correct number of base pairs. Only one out of the four samples is chosen to start an overnight.
9/27/16
Objective: Ligation of PRI__1C3 with PRI__X, bacterial transformation for pSB1C3 with PRI__ and mini prepped J04450 and PRI_MAHS_1C3
Made glycerol stock for PRI_MAHS_1C3 and J04450+pSB1C3
Qubit reading after the mini prep
Transformation of PRI_MAHS with plasmid K880005 (“5”)
Results: The backbone is successfully ligated and the transformation of MAHS has already taken place and plates will be checked on tomorrow for colony check and Colony PCR.
9/28/16
Objective: Ligation of PR1_HDLEA1, 3, AAvLEA1 with 1C3 backbone, Colony PCR of the transformed plates as well as starting overnights for those Colony re-suspensions.
Ligation of the remaining LEA proteins and also bacterial transformation
Started overnights for those samples
Results: Performed ligations of the remaining LEA proteins and also started overnights for them. By tomorrow we can Colony PCR the colonies to check for the correct base pairs.
Week 14: September 29 - October 6
9/29/16
Objective: Colony PCR of plates from previous day. Preparation of primers to fix PstI site on HDLEA2.
Colony PCR conducted to conclude on the correct part and size for each supposed sample.
Gel analyzation of PCR products
Overnights of colony suspensions.
Viewed tardigrades under the microscope
Results: Colony suspensions were inoculated into LB and were placed into rotating incubator to be grown for miniprep.
*HDLEA2 will not be submitted as a part to registry due to the time it will consume to fix the PstI site.
10/6/16
Objective: Bacterial transformation of ligations.
Overnights of colony suspensions did not grow.
Ligations from 9/28/16 were used to redo bacterial transformation.
Note: 5 microliters of ligation PCR product was used for transformation.
Results: Due to the colony suspensions not surviving living in fridge for more than a few days, they did not grow in overnight for miniprep. Bacterial transformation to insert ligated parts into cells.
Week 15: October 7- October 13
10/7/16
Objective: Colony PCR of colonies from 10/6/16. Counting plates from desiccation experiment with PR1_MAHS_1C3.
Picked colonies from plates and made colony suspensions in molecular biology grade water.
Colony PCR products were analyzed on a gel to conclude on correct part and size.
Plates from desiccation experiment on PR1_MAHS_1C3 were counted and data was collected.
Results: Process to sequence LEA proteins is almost at end. Colony PCR of colonies from transformed plates were picked for overnights. Plates from desiccation experiment were counted.
10/8/16
Objective: Miniprep of colony overnights.
Glycerol stocks were made of the overnights. Flash freezed in liquid nitrogen.
Mini prepped the 6 overnights.
QuBit analysis of plasmid DNA concentrations.
Results: Mini prep of overnights were measured for DNA concentration. One sample was picked from each protein and will be amplified for higher concentration before sequencing.
10/10/16
Objective: Prepare samples for sequencing.
Each plasmid DNA sample was calculated for a volume to be added to tubes to be sent out for sequencing.
Three samples were prepared along with one forward primer and one reverse primer.
Results: Samples from plasmid miniprep were prepared for sequencing.
10/11/16
Objective: Desiccation experiment for PR1_MAHS_IC3 and PR1_HDLEA1_1C3, PR1_HDLEA_3, and PR1_AAvLEA1_1C3.
Start of desiccation experiment for LEA proteins.
There are four different plates of each protein with a reference. Each are plated in triplets where there are three plates for each hour: 0, 4, 8, and 24 hours.
Results: Start of desiccation experiment for other LEA proteins.
10/13/16
Objective: Counting plates from desiccation experiment and overnight of other LEA proteins for next desiccation experiment.
Plates on the agar side were marked with a 1 square centimeter square.
Overnight of PR1_HDLEA1_1C3 and PR1_HDLEA3_1C3, with PR1_IC3 as control.
Results: After counting the plates, it showed that there may have been a contamination during plating since there were inconsistent colonies and there was a very widespread lawn of colonies.
Week 16: October 18
10/18/16
Objective: Formatting parts into BioBricks to be submitted to iGEM registry. Digestion of GFP gBlock.
A list was made to direct each well of the plates for each part to be submitted.
A total of 9 parts will be physically submitted.
8 of the parts are plasmid DNA that have been placed into wells to be evaporated of liquid form into a dry form.
Digest of GFP for H. dujardini.
Ligation of backbone with insert.
Bacterial transformation of ligated part.
Results: Plates for BioBrick standard were opened to dry out the plasmid DNA of each part to be submitted. Format for each part was filled in. Digest of GFP for ligation into pSB1C3 backbone to be plasmid DNA to be submitted.
