<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Genspace_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Genspace/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Genspace">HOME </A></LI><LI class="menu_item"> TEAM
			</LI><LI><A href=" https://2016.igem.org/Team:Genspace/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Collaborations">★  Collaborations </A></LI></UL><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:Genspace/Description"> ★  Description </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Proof"> ★ Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Demonstrate"> ★ Demonstrate </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Notebook"> Notebook </A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:Genspace/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Composite_Part"> ★ Composite Parts </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Genspace/Safety"> SAFETY </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Genspace/Attributions">★  ATTRIBUTIONS </A></LI><LI class="menu_item"> HUMAN PRACTICES 
			</LI><LI><A href="https://2016.igem.org/Team:Genspace/Human_Practices"> Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/HP/Silver">★ Silver </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/HP/Gold">★ Gold </A></LI><LI class="menu_item"> AWARDS 
			</LI><LI><A href="https://2016.igem.org/Team:Genspace/Measurement">★  Measurement </A></LI></DIV><DIV class="content_wrapper"><DIV class="column full_size"><DIV id="header">Experiments Page</DIV><DIV id="contents"><P class="c1"><SPAN class="c4 c7 c0 c10">Part 1. Desiccation Tolerance</SPAN></P><P class="c1"><SPAN>Our team set forth to determine whether certain genes expressed in tardigrades during cryptobiosis could be applied to preservation of other organisms or bioproducts.  It has been hypothesized that these proteins either function as chaperones activated by hyperosmotic conditions, or they play a role in DNA repair following rehydration.  Rather than test each hypothesis individually, we elected for a simple experimental design - expression of these genes in other organisms to test whether the phenotype was transferable.  </SPAN><SPAN class="c8">E. coli</SPAN><SPAN> was chosen as our chassis due to the ease and rapidity of cloning in that organism.</SPAN></P><P class="c1"><SPAN>To evaluate desiccation tolerance, we incubated aliquots of saturated </SPAN><SPAN class="c8">E. coli</SPAN><SPAN> cultures at 37°C on parafilm inside an empty Petri dish to prevent contamination.  Once dried, residues were held at 37°C until rehydrated, diluted, and plated.  Plates were incubated overnight and colony counts were made the following day.  As a control, we tested </SPAN><SPAN class="c8">E. coli</SPAN><SPAN> harboring the same plasmid, minus the LEA protein CDS.  This ensured that any difference in desiccation tolerance was granted by the protein itself and not anything else on the plasmid.</SPAN></P><P class="c1"><SPAN class="c4 c7">Cloning Strategy</SPAN></P><P class="c1"><SPAN class="c9">Our focus was on the </SPAN><SPAN class="c4">H. dujardini</SPAN><SPAN class="c9"> tardigrade species, because its genome has been fully sequenced. The initial publication of this genome by Boothby in 2015 led to widespread reporting of the organism’s ability to utilize horizontal gene transfer, with 17.5% of its genome coding for non-tardigrade genes. This extraordinary finding was debunked by Koutsovoulos et. al. in 2016, who concluded the non-tardigrade genes were likely contamination artefacts. The Koutsovoulos paper included a link to the newly sequenced and annotated </SPAN><SPAN class="c4">H. dujardini</SPAN><SPAN class="c9"> genome (available here: </SPAN><SPAN class="c3 c0"><A class="c6" href="https://www.google.com/url?q=http://badger.bio.ed.ac.uk/H_dujardini/&amp;sa=D&amp;ust=1476938065435000&amp;usg=AFQjCNGg4A3bqJCyawjUdkrcGhOMLrDTPA">http://badger.bio.ed.ac.uk/H_dujardini/</A></SPAN><SPAN class="c9">) </SPAN></P><P class="c1"><SPAN class="c9">We used the above website to identify </SPAN><SPAN class="c4">H. dujardini</SPAN><SPAN class="c9"> candidate LEA protein gene homologs. Only one gene annotated as an LEA gene was identified in a BLAST of the AavLEA1 against the </SPAN><SPAN class="c4">H. dujardini</SPAN><SPAN class="c9"> genome: HDLEA1. </SPAN></P><P class="c1"><SPAN class="c9">Further investigation into </SPAN><SPAN class="c4">H. dujardini</SPAN><SPAN class="c9"> LEA proteins led us to a the Forster 2012 paper, in which the scientists had constructed a phylogenetic tree comparing tardigrade LEA proteins to those of other organisms. The </SPAN><SPAN class="c4">H. dujardini</SPAN><SPAN class="c9"> LEA proteins clustered into three groups. One of these was represented by a single gene, that for the gene we designated HDLEA1. We chose representatives of </SPAN><SPAN class="c4">H. dujardini</SPAN><SPAN class="c9"> LEA proteins from the additional two groups at random and designated these HDLEA2 and HDLEA3.</SPAN></P><P class="c1"><SPAN class="c3 c0"><A class="c6" href="https://www.google.com/url?q=http://parts.igem.org/File:Bbi-6-2012-069f4.jpg&amp;sa=D&amp;ust=1476938065440000&amp;usg=AFQjCNGhSEy7iO-Fcuhfy8w4ediNhWgaBg">http://parts.igem.org/File:Bbi-6-2012-069f4.jpg</A></SPAN><SPAN class="c9"> &lt;&lt;------- tree</SPAN></P><P class="c1"><SPAN class="c9">The six IDP genes that we had synthesized by IDT were:</SPAN></P><P class="c1"><SPAN class="c9">HDLEA1 </SPAN></P><P class="c1"><SPAN class="c9">HDLEA2 </SPAN></P><P class="c1"><SPAN class="c9">HDLEA3</SPAN></P><P class="c1"><SPAN class="c9">RvLEAM</SPAN></P><P class="c1"><SPAN class="c9">MAHS</SPAN></P><P class="c1"><SPAN class="c9">AAVLEA1</SPAN></P><P class="c1"><SPAN class="c9">A strong Anderson promoter and strong RBS were used to drive constitutive expression of the IDPs in the vector pSB1C3</SPAN></P><P class="c1"><SPAN class="c4 c7">Expression of Tardigrade IDPs in E. Coli </SPAN></P><P class="c1"><SPAN class="c9">Given the fact that </SPAN><SPAN class="c4">E. Coli</SPAN><SPAN class="c9"> in one of the most widely used organisms in genetic engineering, we chose it to measure the effect of tardigrade IDPs on survival after periods of desiccation.</SPAN></P><P class="c1"><SPAN class="c4 c7">Protocols</SPAN></P><P class="c1"><SPAN class="c0">BioBrick Assembly</SPAN></P><P class="c1"><SPAN>Restriction Digests</SPAN></P><P class="c1"><SPAN>Restriction digests were performed in 10μL reactions with 100ng DNA for a concentration of 100ng/μL in 1X NEBuffer 2.1.  0.5μL of each restriction enzyme was used.  Enzymes used were dependent upon the placement of the part in the final construct:</SPAN></P><P class="c1"><SPAN>Plasmid backbone: EcoRI/PstI</SPAN></P><P class="c1"><SPAN>gBlock insertion: EcoRI/PstI</SPAN></P><P class="c1"><SPAN>Upstream Part: EcoRI/SpeI</SPAN></P><P class="c1"><SPAN>Downstream Part: XbaI/PstI</SPAN></P><P class="c1"><SPAN class="c0">Ligations</SPAN></P><P class="c1"><SPAN>Equimass ligations were carried out in 10μL of 1X T4 Ligase buffer using 0.5μL of T4 DNA Ligase.  Roughly 75-80ng of combined DNA was used in each ligation.</SPAN></P><P class="c1"><SPAN class="c0">Colony PCR</SPAN></P><P class="c1"><SPAN>Colonies were picked using sterile toothpicks and resuspended in 5μL of sterile LB.</SPAN></P><P class="c1"><SPAN>Colony PCR was carried out using OneTaq 2X Master Mix (M0482S, New England Biolabs) according to the manufacturer’s instructions, using 1μL of colony suspension as template.</SPAN></P><P class="c1"><SPAN class="c4 c7">E. coli Desiccation Assay Development</SPAN></P><P class="c1"><SPAN class="c9">The protocol we used was developed by Mike Flanagan [@Chesterornot](</SPAN><SPAN class="c3 c0"><A class="c6" href="https://www.google.com/url?q=https://twitter.com/chesterornot&amp;sa=D&amp;ust=1476938065452000&amp;usg=AFQjCNExewZ0M1eyg7He00suJ-9KjnnKiw">https://twitter.com/chesterornot</A></SPAN><SPAN class="c9">):</SPAN></P><P class="c1"><SPAN class="c9"> 20 microliters of overnight </SPAN><SPAN class="c4">E. coli</SPAN><SPAN class="c9"> cultures were pipetted onto parafilm, and simultaneously, 20 microliters were plated for CFUs. These spots were desiccated at 37C. After a designated incubation time, the spots were resuspended and plated for colony forming units (CFUs). CFUs recovered after incubation were compared to the CFUs from the initial CFUs and plotted as a percent of the original population. Strains containing LEA gene construct plasmids were compared against strains containing empty vector plasmid. </SPAN></P><P class="c1"><SPAN class="c4 c10 c7 c0">Part 2. Tardigrade Model</SPAN></P><P class="c1"><SPAN>The development of a CRISPR protocol for use in tardigrades was a challenging task, as no team has yet reported the successful use of CRISPR in this phylum.  For inspiration, we turned to other ecdysozoans, </SPAN><SPAN class="c8">D. melanogaster</SPAN><SPAN> and </SPAN><SPAN class="c8">C. elegans</SPAN><SPAN>, the two closest model relatives to phylum Tardigrada.  In these organisms, high efficiency edits have been made, not by expressing Cas9 in the animal, but by injecting the Cas9:sgRNA ribonucleoprotein (RNP) complex directly into the animals (Paix et al. 2015).</SPAN></P><P class="c1"><SPAN>Another important consideration was the propagation of Cas9 DNA.  Given the reported hardiness of tardigrades, our lab took great precautions to eliminate any chance of creating a construct that might escape into the wild.  In addition to careful handling of all samples, we sought to integrate this concern into the design of our experiments.  Again, the microinjection of RNPs instead of expression of Cas9 appealed to us, as the exclusion of Cas9-encoding DNA from the experiment prevented any possibility of inadvertently propagating the Cas9 gene.</SPAN></P><P class="c1"><SPAN>Finally, we needed to prove that microinjection was a procedure that tardigrades could be subjected to.  Fortunately, we found a recent publication describing the successful use of RNA interference in tardigrades by injecting dsRNA into adult animals near the gonad (Tenlen et al. 2013).  This success encouraged us to follow the authors’ injection protocol, using Cas9:sgRNA RNPS assembled in vitro in lieu of dsRNA.  Finally, prior to injecting any Cas9 into tardigrades, our team injected them with phenol red to prove that our injection system would not harm the animals.</SPAN></P><P class="c1"><SPAN class="c4 c7">Deciding Which Genes to Target</SPAN></P><P class="c1"><SPAN class="c9">In the paper “RNAi interference can be used to disrupt gene function in Tardigrades” by Tenlen et. al., six developmental genes were targeted including </SPAN><SPAN class="c4">hd-mag-1</SPAN><SPAN class="c9"> which encodes mago nashi. Since the resultant phenotype of the disruption of these genes were presented in the paper, we knew using those genes gave us an outcome with which to base the effects of the knockouts of our experiments.</SPAN></P><P class="c1"><SPAN class="c9">The genome of the tardigrade species Hypsibius dujardini has been sequenced and can be found at: </SPAN><SPAN class="c3 c0"><A class="c6" href="https://www.google.com/url?q=http://badger.bio.ed.ac.uk/H_dujardini/&amp;sa=D&amp;ust=1476938065461000&amp;usg=AFQjCNGZO_aKTK73VYDNWqpuoacvceaqAg">http://badger.bio.ed.ac.uk/H_dujardini/</A></SPAN><SPAN class="c0 c9">.  </SPAN><SPAN class="c9">We used the accession numbers to get the partial mRNA transcripts which were subsequently blasted against the H. Dujardini genome to locate  the complete gene sequence.  </SPAN></P><P class="c1"><SPAN class="c4 c7">Blast Results</SPAN></P><P class="c1"><SPAN class="c4">Actin-1</SPAN><SPAN class="c9"> Gene (Hd-act-1) – Found 745 bp of this gene within </SPAN><SPAN class="c3 c0"><A class="c6" href="https://www.google.com/url?q=https://raw.githubusercontent.com/genspace/iGEM-Tardigrade-CRISPR-Analysis/master/nHd.2.3.scaf00209&amp;sa=D&amp;ust=1476938065463000&amp;usg=AFQjCNHZsqCi9nbwQX56e9Bsbp5WfF6JDw">scaffold 209</A></SPAN><SPAN class="c9"> of the </SPAN><SPAN class="c3 c0"><A class="c6" href="https://www.google.com/url?q=http://badger.bio.ed.ac.uk/H_dujardini/fileDownload/zip_download?fileName%3DnHd.2.3.abv500.fna&amp;sa=D&amp;ust=1476938065464000&amp;usg=AFQjCNGAb3QGdV5xPtwxczCJM1LQ84uG5g">tardigrade genome</A></SPAN><SPAN class="c9">.</SPAN></P><P class="c1"><SPAN class="c4">Mago Nashi</SPAN><SPAN class="c9"> Gene (Hd-mag-1) –  Found 586 bp of this gene within </SPAN><SPAN class="c0 c3"><A class="c6" href="https://www.google.com/url?q=https://raw.githubusercontent.com/genspace/iGEM-Tardigrade-CRISPR-Analysis/master/nHd.2.3.scaf00020&amp;sa=D&amp;ust=1476938065465000&amp;usg=AFQjCNG1VWpLAcyTNibxreRVPWPj1In1-A">scaffold 20</A></SPAN><SPAN class="c9"> of the </SPAN><SPAN class="c3 c0"><A class="c6" href="https://www.google.com/url?q=http://badger.bio.ed.ac.uk/H_dujardini/fileDownload/zip_download?fileName%3DnHd.2.3.abv500.fna&amp;sa=D&amp;ust=1476938065466000&amp;usg=AFQjCNHr-k9Rd1-HEtyN_WsNP32wRnn0fQ">tardigrade  genome</A></SPAN><SPAN class="c9">. </SPAN></P><P class="c1"><SPAN class="c9">The H. Dujardini blast results were used to make the guide RNAs targeting the </SPAN><SPAN class="c4">mago nashi</SPAN><SPAN class="c9"> &amp;</SPAN><SPAN class="c4">actin genes</SPAN><SPAN class="c9">.  </SPAN></P><P class="c1"><SPAN class="c4 c7">Guide RNA Design</SPAN></P><P class="c1"><SPAN class="c9">The `ngg2` </SPAN><SPAN class="c3 c0"><A class="c6" href="https://www.google.com/url?q=https://github.com/RobersonLab/ngg2&amp;sa=D&amp;ust=1476938065469000&amp;usg=AFQjCNFl23ZnBOOtT6fCpTcobbx-xIsMcQ">python script</A></SPAN><SPAN class="c9"> produced by UW's </SPAN><SPAN class="c3 c0"><A class="c6" href="https://www.google.com/url?q=http://www.robersonlab.org/&amp;sa=D&amp;ust=1476938065469000&amp;usg=AFQjCNGvGmEymm4zGHrhEngxzx30zYiHzg">Roberson Lab</A></SPAN><SPAN class="c9"> was used for identifying all unique PAM sites within the Hypsibius Dujardini Tardigrade genome. Please see </SPAN><SPAN class="c3 c0"><A class="c6" href="https://www.google.com/url?q=https://github.com/genspace/iGEM-Tardigrade-CRISPR-Analysis/blob/master/iGEM_Tardigrade_CRISPR_Analysis.ipynb&amp;sa=D&amp;ust=1476938065470000&amp;usg=AFQjCNHH4D3d30zKFH806U6CLLGDJo1Xbw">iGEM Tardigrade CRISPR Analysis notebook</A></SPAN><SPAN class="c9"> for gRNA identification.</SPAN></P><P class="c1"><SPAN class="c4 c7">Protocols</SPAN></P><P class="c1"><SPAN class="c0">Microinjection Protocol</SPAN></P><OL class="c20 lst-kix_9086x5ofkqcb-0 start" start="1"><LI class="c1 c17 c16"><SPAN>The tardigrades were transferred from a 96 - well plate containing algae into wells containing only spring water (no food)  and left to sit for a day in order to ensure that they had excreted any remaining food that was in their stomachs.</SPAN></LI><LI class="c1 c17 c16"><SPAN>In order to anesthetize the tardigrades, 50mM stock solution of tetramisole was diluted to 5mM in spring water.</SPAN></LI><LI class="c1 c17 c16"><SPAN>100 microliters of the diluted anesthetic were transferred to a depression slide after which the tardigrades were transferred into the diluted anesthetic on the depression slide.</SPAN></LI><LI class="c1 c16 c17"><SPAN>The tardigrades were left on this slide until there was little to no movement seen in them.</SPAN></LI></OL><P class="c1"><SPAN>            (Note: Do not keep the depression slide over the light of the microscope as it increases the  evaporation of the anesthetic solution).</SPAN></P><P class="c1"><SPAN>       5.  Prepare an injection slide by annealing a coverslip onto a microscope slide by applying nail polish to the ends. (The purpose of the coverslip is to have something to brace the tardigrades against while injecting).</SPAN></P><P class="c1"><SPAN>       6. Applying a drop of microinjection oil onto one of the edges of the coverslip.</SPAN></P><P class="c1"><SPAN>       7. Transfer one of the anesthetized tardigrades into this oil.</SPAN></P><P class="c1"><SPAN>Instructions for the preparation of the RNase free injection buffer can be found here:</SPAN></P><P class="c1"><SPAN class="c0 c12"><A class="c6" href="https://www.google.com/url?q=http://www.uwtransgenics.org/protocols/preparation-of-crisprcas9-mrna-for-microinjection/&amp;sa=D&amp;ust=1476938065473000&amp;usg=AFQjCNET3Pa3-eQJlqmGjFcPZ4B8nFL4hw">http://www.uwtransgenics.org/protocols/preparation-of-crisprcas9-mrna-for-microinjection/</A></SPAN></P><P class="c1"><SPAN class="c0"> Preparation of Cas9 and sgRNA for injection</SPAN></P><P class="c1"><SPAN>      8. You should have one 1.5 ml tube containing 50 microliters of injection buffer without guide RNAs and also other 1.5 ml tubes containing 50 microliters of injection buffer with each separately containing different guide RNAs of the genes you are targeting.</SPAN></P><P class="c1"><SPAN>     9. If your Cas9 was purchased from NEB (product M0641T), dilute by taking 1 ul of the Cas9 and adding it to 9 ul of the injection buffer containing no guide RNAs in a separate tube. </SPAN></P><P class="c1"><SPAN>    10. Afterwards take 1.6 ul of this diluted Cas9 and add it to each of your injection tubes including the tube containing no guide RNAs.</SPAN></P><P class="c1"><SPAN>    11. Incubate all of your tubes prepared with Cas9 and guide RNAs in a 37C incubator for 15 minutes in order to complex the Cas9 and guide RNAs.</SPAN></P><P class="c1"><SPAN>    12. After incubation, centrifuge these tubes for 15 mins at max 13,500 rpm.</SPAN></P><P class="c1"><SPAN>    13. After centrifugation, add 10 microliters of the solution you are going to inject first into tube along with 1 microliter of phenol red.</SPAN></P><P class="c1"><SPAN>    14. Load 0.7 microliters of the resulting mixture into a microinjection needle and affix the needle into the needle holder.</SPAN></P><P class="c1"><SPAN class="c0">Microinjection System Usage</SPAN><SPAN>  </SPAN></P><P class="c1"><SPAN>   15. Put the microscope slide containing the coverslip and anesthetized tardigrade in oil onto the microinjection stage.</SPAN></P><P class="c1"><SPAN>   16. Lower the needle until it is just above the surface of the oil and above the microscope light</SPAN></P><P class="c1"><SPAN>   17. View the tardigrade under 40x until it is in focus.</SPAN></P><P class="c1"><SPAN>   18. Next, slowly lower the needle until it also is in focus ( Be very gentle as if you lower too fast the needle tip will break)</SPAN></P><P class="c1"><SPAN>       19. Once both the tardigrade and injection needle are in focus, move the needle pass the tardigrade until it (the needle)  is against the coverslip. Gently rub the tip of the needle against the coverslip while repeated pressing the ‘inject’ button to see if any liquid comes out. You are attempting to break the tip of the needle to allow a small amount of liquid out.</SPAN></P><P class="c1"><SPAN>      20. Once liquid in the form of circular orbs are visibly seen to be released from the needle tip upon pressing of the ‘inject’ button, adjust the pressure so that the size of the orbs you are seeing are roughly 1/10th the size of the tardigrade.</SPAN></P><P class="c1"><SPAN>     21. Use the tip of the needle to pin the tardigrade against the coverslip and maneuver the needle back and forth until the tardigrade is in a sprawled out position. </SPAN></P><P class="c1"><SPAN>    22. Inject the circular looking parts on the back of the tardigrades.</SPAN></P><P class="c1"><SPAN class="c0">Recovery</SPAN></P><P class="c1"><SPAN>23. Transfer the injected tardigrade back into a well filled with spring water and some algae.</SPAN></P><P class="c1"><SPAN>24. Repeat the above procedures for all of the tardigrades you want to inject.</SPAN></P><P class="c1"><SPAN class="c4 c10 c7 c0">Part 3. Plasmid Copy Measurement</SPAN></P><P class="c1"><SPAN>To determine the copy number of pSB1C3, we had to take several things into consideration.  qPCR data is meaningless if amplification efficiency isn’t identical across samples.  Furthermore, any loss of DNA during purification would add variability to our results.  To avoid these issues, we chose to target a sequence that would be found once in the </SPAN><SPAN class="c8">E. coli</SPAN><SPAN> genome and once on our plasmid, and use raw cell lysate as our template.  Ideally, a multiplex assay would have been run using different chromosomal vs. plasmid targets, however our single-channel system allowed the use of only a single probe per reaction.  A standard was generated by lysing a known number of </SPAN><SPAN class="c8">E. coli</SPAN><SPAN> cells, generating an equal number of copies of the target sequence.  Test articles were then produced by lysing a known number of cells harboring K909006-pSB1C3.  Every time a qPCR run was performed, a new standard and test article was produced to ensure identical handling of all samples tested.  Using lysate for the input into all reactions ensured that any inefficiencies caused by the lysate would impact the samples and standard equally.  Finally, we researched a lysis solution that would minimally impact qPCR efficiency (Shatzkes et al. 2014).</SPAN></P><P class="c1"><SPAN class="c7 c8">Sample Preparation</SPAN></P><P class="c1"><SPAN>Bacterial cultures for testing were grown overnight to saturation, and the OD</SPAN><SPAN class="c14">600</SPAN><SPAN> was measured using the following formula:</SPAN></P><P class="c1"><SPAN>OD</SPAN><SPAN class="c14">600</SPAN><SPAN> of 1.0 = 8x10</SPAN><SPAN class="c13">8</SPAN><SPAN> cells/mL</SPAN></P><P class="c1"><SPAN>When necessary, 10-fold dilutions were prepared to ensure optical density remained within the linear range of our spectrophotometer.</SPAN></P><P class="c1"><SPAN>200,000,000 cells were taken and pelleted, then resuspended in 1mL of CL Buffer for a final concentration of 1 million cells per 5 microliters.  Cells were lysed at room temperature for at least 5 minutes, then frozen at -20°C.  10-fold dilutions of lysate were made using CL buffer to reduce DNA concentration to a manageable amount for qPCR.  For the standard, 20,000,000,000 cells were pelleted and lysed, and 10-fold serial dilutions in CL buffer were prepared.</SPAN></P><P class="c1"><SPAN class="c0">Bleach Gels for RNA Electrophoresis</SPAN></P><P class="c1"><SPAN>RNA gels were prepared using 2% agarose and 5% Chlorox bleach in 1X TAE. </SPAN></P><P class="c1"><SPAN> For 50mL gels, 2.5uL Ethidium Bromide was added after heating.  </SPAN></P><P class="c1"><SPAN>Samples were mixed with 2X RNA Loading Buffer (B0363S, New England Biolabs)  </SPAN></P><P class="c1"><SPAN>Gels were run at 80v.</SPAN></P><P class="c1"><SPAN>Oligos were designed targeting the LacZ gene with the following sequences:</SPAN></P><P class="c1 c16"><SPAN>LacZ TaqMan FWD:        5’-CACCGATATTATTTGCCCGATGTAC-3’</SPAN></P><P class="c1 c16"><SPAN>LacZ TaqMan PRB:        5’-AAGACCAGCCCTTCCC-3’</SPAN></P><P class="c1 c16"><SPAN>LacZ TaqMan RVS:        5’-GGACCATTTCGGCACAGC-3’</SPAN></P><P class="c1"><SPAN>Reactions were set up using the chemistry described in the kit’s manual.  A master mix was created and 20uL were distributed to each reaction tube.  5uL of cell lysate (containing 10</SPAN><SPAN class="c13">5</SPAN><SPAN> cells for the test articles) were added to each reaction.  A 3-point standard was used consisting of 10</SPAN><SPAN class="c13">5</SPAN><SPAN>, 10</SPAN><SPAN class="c13">6</SPAN><SPAN>, and 10</SPAN><SPAN class="c13">7</SPAN><SPAN> lysed </SPAN><SPAN class="c8">E. coli</SPAN><SPAN> cells without the plasmid.  All samples were run in triplicate.  The reverse transcription step was skipped and reverse transcriptase enzyme was inactivated during the 95°C hot start and qPCR continued normally using DNA as a template.</SPAN></P><P class="c1"><SPAN class="c5">Each reaction has the following chemistry:</SPAN></P><P class="c1 c15"><SPAN class="c5">12.5uL Verso qRT-PCR Mix</SPAN></P><P class="c1 c15"><SPAN class="c5">5.45uL Molecular Biology Grade Water</SPAN></P><P class="c1 c15"><SPAN class="c5">5.0uL Cell lysate</SPAN></P><P class="c1 c15"><SPAN class="c5">1.8uL TaqMan Primer/Probe Mix</SPAN></P><P class="c1 c15"><SPAN class="c5">0.25uL Verso Enzyme Mix</SPAN></P><OL class="c20 lst-kix_rqgvrkmkbjfy-0 start" start="1"><LI class="c11 c1"><SPAN class="c5">To minimize reaction variability, make a master mix first by multiplying the above volumes (withholding lysate) by the number of reactions you plan to run, with an excess of 2-3 reactions' worth.</SPAN></LI><LI class="c1 c11"><SPAN class="c5">Distribute 20uL of master mix into each reaction tube.  Because the edge effect impairs data quality in the outer tubes, avoid using them for anything other than a positive or negative control</SPAN></LI><LI class="c11 c1"><SPAN class="c5">Distribute 5uL of the appropriate lysate (standard or test) to each reaction tube.  For the negative control, use 5uL of CL Buffer.</SPAN></LI></OL><H3 class="c1 c22" id="h.uyqaziv6qljg"><SPAN>qPCR Cycles</SPAN></H3><P class="c1"><SPAN class="c5">Run qPCR with the following cycles:</SPAN></P><P class="c1 c15"><SPAN class="c5">95C for 15 minutes</SPAN></P><P class="c1 c15"><SPAN class="c5">40 cycles of:</SPAN></P><P class="c1 c21"><SPAN class="c5">95C for 15 seconds</SPAN></P><P class="c1 c21"><SPAN class="c5">60C for 1 minute</SPAN></P><P class="c1"><SPAN>Reaction Cycles:</SPAN></P><P class="c1"><SPAN>95°C for 15 minutes</SPAN></P><P class="c1"><SPAN>40 cycles of:</SPAN></P><P class="c1"><SPAN>95°C for 15 seconds</SPAN></P><P class="c1"><SPAN>60°C for 1 minute</SPAN></P><P class="c1"><SPAN class="c0">End-Point PCR for </SPAN><SPAN class="c0 c8">rop</SPAN><SPAN class="c0"> Detection</SPAN></P><P class="c1"><SPAN>Rop detection primer sequences:</SPAN></P><P class="c1 c16"><SPAN>rop FWD:        5’-TTAACATGGCCCGCTTTATC-3’</SPAN></P><P class="c1 c16"><SPAN>rop RVS:        5’-TCAGAGGTTTTCACCGTCATC-3’</SPAN></P><P class="c1"><SPAN>PCR was carried out using OneTaq 2X Master Mix (M0482S, New England Biolabs) according to the manufacturer’s instructions.</SPAN></P><P class="c1"><SPAN>Gel electrophoresis showed no amplification in either </SPAN><SPAN class="c8">E. coli</SPAN><SPAN> Top 10 or K909006-pSB1C3.  Given this result, the unexpected presence of the </SPAN><SPAN class="c8">rop</SPAN><SPAN> protein is not likely to be responsible for the low copy number detected for this plasmid.</SPAN></P><P class="c1"><SPAN class="c0">Plasmid Copy Number measurement via Gel Electrophoresis</SPAN></P><P class="c1"><SPAN>1 billion cells were pelleted and resuspended in 100uL CL Buffer.  Lysate was run on a 0.6% agarose gel at 80v alongside purified samples of K909006-pSB1C3 at various concentrations for brightness comparison.</SPAN></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>