<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Genspace_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Genspace/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Genspace">HOME </A></LI><LI class="menu_item"> TEAM
			</LI><LI><A href=" https://2016.igem.org/Team:Genspace/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Collaborations">★  Collaborations </A></LI></UL><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:Genspace/Description"> ★  Description </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Proof"> ★ Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Demonstrate"> ★ Demonstrate </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Notebook"> Notebook </A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:Genspace/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/Composite_Part"> ★ Composite Parts </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Genspace/Safety"> SAFETY </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Genspace/Attributions">★  ATTRIBUTIONS </A></LI><LI class="menu_item"> HUMAN PRACTICES 
			</LI><LI><A href="https://2016.igem.org/Team:Genspace/Human_Practices"> Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/HP/Silver">★ Silver </A></LI><LI><A href="https://2016.igem.org/Team:Genspace/HP/Gold">★ Gold </A></LI><LI class="menu_item"> AWARDS 
			</LI><LI><A href="https://2016.igem.org/Team:Genspace/Measurement">★  Measurement </A></LI></DIV><DIV class="content_wrapper"><DIV class="column full_size"><H2>Results</H2><DIV class="content"><H2>Part 1. Desiccation Tolerance</H2><H3>Desiccation Test Results for HDLEA1 (K2128204) and MAHS (K2128200)</H3><P>The desiccation protection part of our iGEM project involved determining how proteins in extremophile species such as tardigrades could assist bacteria to survive in desiccation circumstances.  To this end it was necessary to identify which proteins to pursue, to successfully synthesize and assemble those proteins into functional genetic circuits, to identify a protocol for quantitative evaluation of desiccation survival, and to execute that protocol.  These efforts would have benefits in the routine sharing of strains of bacteria between members of the research community.  They would also potentially open up new avenues of investigation and applications involving bacteria under limited-hydration environments.</P><P>As a result of literature surveys, several proteins were identified for investigation, especially those belonging to a class of Late Embryonic Abundant (LEA) proteins.  Sequences for these candidates were successfully codon-optimized for use in E. coli, synthesized, placed into plasmid backbones and confirmed through sequencing.  These parts were further assembled into functional genetic circuits to both support expression of the proteins of interest and also to identify reference controls against which we could compare performance.  These composite parts were successfully sequenced in order to ensure evaluation of known entities.  A protocol was determined and tested to quantify results in a manner that balanced time, effort and cost. Finally, this protocol was used to evaluate a test LEA protein (HDLEA1).</P><P>Part K2128204 (and the underlying Coding Sequence in part K2128004) was validated using a desiccation protocol that demonstrated the part worked as expected (i.e., improved survivability when the HDLEA1 coding sequence was expressed compared to when it was not expressed).  The K2128204 part consists of the Biobrick part K880005 (i.e., a strong constitutive promoter J23100 and strong ribosome binding site B0034) followed by the HDLEA1 coding sequence from K2128004. When placed on the high-copy-number plasmid backbone pSB1C3, maximal expression of HDLEA1 is expected.  The comparative reference was K880005, also on pSB1C3 backbone (i.e., the plasmid under test minus the HDLEA1 coding sequence).</P><P>The desiccation protocol was initially run to identify serial dilution values for each hour of desiccation that would result in a number of colony forming units per plate (CFU/plate) on the order of 30-300.   The OD600 of the initial overnight culture was taken (using a 1:10 dilution proxy) in order to obtain the expected number of CFUs in 20uL using the formula (0.02mL)*(8x108 CFU/mL/OD600)*OD600.  Excellent agreement (within ±25%) with the observed average CFU/plate was found for both the reference and test plasmid systems after correcting for the dilution factor for H=0 hours of desiccation (i.e., no desiccation).</P><P>For each hour (H=0, 4, 7.5 and 22.5) and each plasmid system under test (i.e., the K880005 reference and the HDLEA1-based K2128204), data were collected in triplicate and used to obtain an average and standard deviation for CFU/plate.  Plates with evidence of contamination or pipetting error were removed (a total of two plates out of 24).  The dilution factor was corrected for in order to obtain the CFU in the desiccated 20uL aliquot and normalized by the measured CFU in a non-desiccated 20uL aliquot to obtain a survival rate versus time (i.e., fraction of surviving colonies).  The numerical results are contained in Tables 1 and 2 for K880005 (REF) and K2128204 (TEST_HDLEA1), respectively.  </P><P>
			Table 1: Reference (REF) data (raw, adjusted for dilution, and normalized)
		</P><P>
			Table 2: HDLEA1-related (TEST_HDLEA1) data (raw, adjusted for dilution, and normalized)
		</P><P>
			The combined graphical results for the normalized data are shown in Figure 1 below.
			
			Figure 1: Survival Rate vs. Desiccation (HDLEA1)
		</P><P>As expected, the fraction of surviving cells decays monotonically with increasing desiccation time for both the reference (“REF”) and HDLEA1-based systems (“TEST_HDLEA1”).  Beyond four hours, the plasmid expressing HDLEA1 shows an improved ability to survive desiccation in a manner that is statistically significant based on measured standard deviations.  In particular, the amount improvement is estimated to be 3.2-fold (±1.5) for H=7 hours of desiccation and 2.5-fold (±1.3) for H=22.5 hours of desiccation.  This approximately three-fold improvement in survivability validates that the HDLEA1-related part (K2128204) works as expected.</P><P>
			In contrast, the preliminary testing for the corresponding MAHS LEA protein (expressed using the part K2128200) does not show an improvement in desiccation survivability as shown in Table 3 and Figure 2.
			
			Table 3: MAHS-related (TEST_MAHS) data (raw, adjusted for dilution, and normalized)
		</P><P>
			Data beyond H=7 hours of desiccation had failed positive controls that invalidated those results.  (Positive controls for hours H=0, 4 and 7 were successful.)  While those control issues could not be resolved in time for iGEM deadlines, it was desired to supply this preliminary information as a guide to other teams that may wish consider use of this part.
			
			Figure 2: Survival Rate vs. Desiccation (MAHS)			
		</P><P>Based on this data, there is no evidence that the MAHS-related system (K2128200) provides any survivability benefit over the reference system (K880005).  </P></DIV><DIV class="content"><H2>Part 2. Using Tardigrades as a Developmental Model</H2><P>While we were able to successfully microinject the Cas9 protein along with the guide RNAs targeting different developmental genes. The microinjections proved fatal to the injected tardigrades and thus the only information we were able to gain is that future microinjectors of the Cas9 protein would need to place a premium on preventing the lethality of such injections. Things to consider would be:</P><OL><LI>Microinjection technique and pressure used to inject</LI><LI>Concentration of the Cas9 Protein</LI><LI>Whether injections to the tardigrades’ body parts other than the gonads prove lethal</LI></OL><P>Unfortunately due to time we were not able to alter these different aspects of our protocol.</P></DIV><DIV class="content"><H2>Part 3. Plasmid Copy Measurement</H2><P>A preliminary experiment was run using lysate from 1 million cells per reaction.  Eight replicates were run for each sample type (with or without the reporter).  After excluding outliers caused by edge effects, average CT values for six replicates for each sample were compared:</P><P>Test:                17.02</P><P>Control:        21.48</P><DIV class="sub-content"><P class="c0"><SPAN><B>pSB1C3 absolute quantification run #1</B></SPAN></P><P>Lysate from 1 million stationary phase cells harboring K909006-pSB1C3 was run against a 3-point standard of 106, 107, and 108 copies.  Linear regression indicates approximately 18.2 copies of the target sequence for every cell in the reaction, or around 17 plasmid copies per cell.</P><P class="c0"><SPAN><B>pSB1C3 absolute quantification run #2</B></SPAN></P><P>Lysate from 100,000 mid-log phase cells harboring K909006-pSB1C3 was compared against a 3-point standard of 105, 106, and 107 copies.  Due to the reduced amplification efficiency of the 108-copy standard in run 1, cell numbers were reduced 10-fold in all subsequent experiments.  Linear regression indicates approximately 13.4 copies of the target sequence for every cell in the reaction, or around 12-13 plasmid copies per cell.</P><P>Note: The K909006-pSB1C3 harboring cells used for this run were lysed in mid-log phase, which may account for the reduced PCN.</P><P class="c0"><SPAN><B>pSB1C3 absolute quantification run #3</B>Lysate from 100,000 stationary phase cells harboring K909006-pSB1C3 was compared against a 3-point standard of 105, 106, and 107 copies.  Linear regression indicates approximately 30.9 copies of the target sequence for every cell in the reaction, or around 30 plasmid copies per cell.</SPAN></P><P class="c0"><SPAN><B>pSB1C3 absolute quantification run #4</B></SPAN></P><P>Lysate from 100,000 stationary phase cells harboring K909006-pSB1C3 was compared against a 2-point standard of 105 and 106 1.1x106 copies.  The 1.1x106-copy standard was created using lysate from 105 cells as well as 106 copies of purified plasmid.  This point was created to test for variance in amplification efficiency of plasmid vs. genomic template.  Linear regression indicates approximately 25.5 copies of the target sequence for every cell in the reaction, or around 24-25 plasmid copies per cell.

Three qPCR runs using stationary phase cells and one run using mid-log phase cells indicate a PCN of around 12-13 copies during log growth, increasing to around 24 copies per cell during stationary phase.
</P>
			
			Please refer to the <A href="https://github.com/genspace/iGEM-qPCR-Copy-Number-Analysis/blob/master/qpcr_pSB1C3_Absolute_Quantification.ipynb">Jupyter Notebook</A> for qPCR copy number analysis.

		</DIV><DIV class="sub-content"><H3>Gel Electrophoresis of Cell Lysate</H3>
			Because qPCR consistently gave results significantly lower than the expected copy number, an alternate means of direct copy number analysis was attempted.

			1 billion cells were pelleted and resuspended in 100uL of CL Buffer.  At a size of 5339bp, 20 billion copies of K909006-pSB1C3 should weigh 115ng at a concentration of 1.15ng/uL.  10uL of lysate from E. coli  was run on a 0.6% agarose gel with purified K909006-pSB1C3 at known concentrations for comparison of band brightness.

			

			Lanes were loaded in the following order (left to right)
			E. coli Top 10 with no plasmids
			10ng purified K909006-pSB1C3
			E. coli Top 10 harboring K909006-pSB1C3
			50ng purified K909006-pSB1C3

			The band for the plasmid in E. coli lysate was much closer in brightness to the 10ng band than the 50ng one, indicating that close to 20 copies were harbored in each lysed cell.
		</DIV><H5 class="c0"><SPAN class="c8">Conclusions</SPAN></H5><P class="c0"><SPAN>Due to time constraints, only a few qPCR runs could be completed prior to the Wiki freeze, so this data is still preliminary.  Additional sets should be run to determine the reproducibility of these results.  Furthermore, the range of copy numbers detected for stationary phase cultures suggests that the copy number may take some time to stabilize after cell division halts.</SPAN></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>