Team:Genspace/Collaborations
MENU ▤
HOME
TEAM
Team
★  Collaborations
PROJECT
★  Description
Experiments
★ Proof of Concept
★ Demonstrate
Results
Notebook
PARTS
Parts
★ Basic Parts
★ Composite Parts
SAFETY
★  ATTRIBUTIONS
HUMAN PRACTICES
Human Practices
★ Silver
★ Gold
AWARDS
★  Measurement
We helped 3 other iGEM teams:
1. Stony Brook iGEM Team
https://2016.igem.org/Team:Stony_Brook
The Stony Brook iGEM team undertook the task of investigating the survivability of yeast under desiccation, an experiment equivalent to the one our team has conducted on E. Coli. Given the many potential uses that desiccated yeast could offer to iGEM as well as the scientific community as a whole, we’ve agreed to replicate the desiccation experiments performed by Stony Brook in an effort to corroborate their data. Their protocol consisted of exposing yeast to 2, 4, 6, and 8 hours of dehydration in an incubator. 20 microliters of yeast were placed on separate segments of parafilm which were left in an incubator for the aforementioned amounts of time. Afterwards the yeast was rehydrated and plated. Approximately 2 days later the colonies on the plates were counted. The results of our study are shown below:
Note: Given that the amount of yeast that would grow after little or no desiccation would surpass the capacity of the plates or prove difficult to count, we performed a 1 to 10,000 dilution of the 0 hrs of desiccation and multiplied the colonies by 10,000 to account for the dilution. A 1 to 100 dilution was performed on the 2 hrs of desiccation and the colonies multiplied by 100. There were no other dilutions performed on the 4-8 hrs of desiccation.
2. UGA iGEM Team
https://2016.igem.org/Team:UGA-Georgia
We helped the UGA-Georgia iGEM Team by characterizing their mCherry-based reporter system
For the second year in a row, the Genspace iGEM team helped the University of Georgia iGEM Team by participating in the Archaeal InterLab Study to further characterize the reproducibility of their mCherry reporter system.  This year, samples were formaldehyde-fixed after oxygen exposure and prior to sending, allowing fluorescence readings to be taken directly without overnight growth.  Triplicate reads were taken with excitation at 590nm and emission at 645nm.
Sample ID
Read 1
Read 2
Read 3
MS
0
5
3
WT
16
10
0
A
12
11
11
B
17
28
13
C
148
127
123
Important: As soon as you receive the samples, keep them covered in foil. Exposure to light affects the fluorescence measurement.
You have been given 5 samples: 3 different mutants (A, B, C), a wild type (WT) and a sample of suspension buffer (MS).  Each tube contains 1mL of the sample; triplicate readings of each sample should be taken. Therefore, 1mL should be enough to run each sample 3 times.
*Make sure to vortex tubes before beginning experiment
Plate Reader Instructions:
Using sample “C” determine the linear range for your measurement device.
We recorded the highest fluorescence from sample “C,” so if this is found to be in the linear range, then all the other samples will be as well.
Our device required a 200µL sample, and directly dispensing 200µL from the 1mL sample was within the linear range. No dilution was necessary.
Our measurement device is a BioTek Synergy HT plate reader (96 well) and we use the Gen5 software.
Excitation wavelength (nm): 590/20
Emission wavelength (nm): 645/40
Scaled* to High Wells
Set point Temperature: 25 OC
*Your measurement device and protocol may vary from ours and other teams, so please update our form with your specific parameters. Email Ghazal Motakef at gm76967@uga.edu and Anu Romesh at ar74782@uga.edu upon receiving this shipment for the link to the registration form and data form or for any additional questions.
3. BioBricks Foundation iGEM Team
https://2016.igem.org/Team:BioBricks
This year, the BioBricks team worked on a functional standard of gene expression and developed an assay to determine whether a gene of interest would have compatibility issues with any module being inserted into the system.  Genspace ran this assay on samples provided by the BioBricks team to test its reproducibility in different labs.
Materials Provided:
7 stab cultures of DH5αZ1 w/ constructs…
“White”
“Red”
“Cyan”
“CFP-RFP 15%”
“CFP-RFP 40%”
“CFP-RFP 80%”
“RFP-CFP”
Materials/Equipment Needed:
LB + Chloramphenicol
LB + Chloramphenicol Agar Plates
Plate Reader
OD700 (not OD600 to avoid overlapping with the 610nm emission from mCherry)
excitation/emissions - 434/474nm (mTurquoise2) and 587/610nm (mCherry)
96 well plate w/ AeraSeal
Shaking incubator that works on 96 well plate
Protocol:
Plate Samples from stab cultures on LB + Chloramphenicol plates
Pick a colony and start an overnight culture with LB + Chloramphenicol for each construct
Back-dilute each @ 1:50 for a final volume of at least 3mL
With each culture, Seed an entire row with 200µL/well (12 replicates/sample)
With LB + CAM as a blank in the first row
Measure OD700, Fluorescence 434/474 and 587/610
Cover with AeraSeal and place in shaking incubator.
Take measurements hourly until all OD700’s have stabilized (~10 hours)
