"
Page
Discussion
View source
History
teams
Log in
 
Team:USyd-Australia/pUS203
From 2014.igem.org
pUS201
pUS203
pUS204
Primers
gBlocks
pUS203 Controllable Integrase production
To see this part on the iGEM Parts catalogue please click here .
Sequences for this part are also available in SnapGene (.dna file) , FASTA (.fa file) , GenBank Standard (.gb file) and Plain Text (.txt file) file format.
Aim
Our aim was to create a controllable Integrase producing plasmid as a submission part. From the submission plasmid the controllable Integrase Part could then be ligated into a convenient low-copy plasmid to help reduce the potential toxicity of Integrase to the cell.
Approach
The 2010 Paris Bettencourt team had developed a controllable Integrase part for use a population counter in their project. We contacted them to see if we could source the part from them but it was unfortunately not possible. We set about creating the BioBrick using the araC-pBAD from the well characterized part K731201 and ordering the IntI1 as a gBlock. The araC-pBAD system in pSB1C3 had previously been prepared by Honours student Sam Ross (called the SamR construct). Our next steps were the design of the gBlock and integration of the IntI1 gBlock into the SamR construct.
Materials
DNA
SamR Construct: araC-pBAD in pSC1B3
IntI1 gBlock : to be inserted into SamR
pUS44: Verified plasmid containing an AttI site, used in Validation
aeBlue: Cassette PCR'ed from K1033929 for us in validation
Primers
iGEM1413
iGEM1414
iGEM1415
iGEM1416
iGEM16
iGEM25
Inti1 gBlock Design
The code for the sequence for the IntI1 gene was sourced from pUS2056 but is highly conserved so this choice was out of convenience as the sequence was locally available. It was modified by Tom Geddes, Rokiah Alford and our supervisor Dr Nicholas Coleman to:
Include a Sac1 restriction enzyme site at the beginning before the Inti1 gene so it could be excised independently of the promoter if needed
Include a different ribosome binding site
Include Gibson ends that overlapped with the backbone for simple Gibson Assembly later with the SamR construct.
Remove an illegal Spe1 site near the end to make it compliant with iGEM requirements
Remove a promoter pC that was coded in the opposite direction to the promoter we wanted to use and would have clashed with the controllable promotion of this part
Our first attempt to Gibson Assemble the SamR  construct and IntI1 gBlock failed because there was extra code after the code for pBAD which we had not anticipated. This meant that the Gibson Assembly could not properly integrate the gBlock. We discussed numerous options to try and either get rid of the problem sequence on the SamR construct or modify the ends of the gBlock to accommodate the problem sequence. In the end it was decided that a re-design of the gBlock would be the most efficient way to solve the problem. Back to top
Inti1 gBlock ReDesign
Three major changes were decided:
Changed the code for the ribosome cut site as a base pair was incorrect
To put an Nhe1 restriction enzyme cut site between the IntI1 gene and the promoters on the SamR construct. This was done to help in the ligation of this BioBrick into the low-copy backbone pUS202.
Changing of the end sequences to accommodate the extra code present on the SamR construct
Back to top
Construction
The IntI1 gBlock was inserted into the SamR construct using Gibson Asssembly . The ends of the gBlock in the redesign were compatible with the ends of the SamR construct and after running the Gibson Assembly reaction we used a variety of methods to prove that our gBlock had inserted correctly.
These changed were performed by our supervisor Nick Coleman in consultation with Rokiah Alford. The redesigned gBlock was able to be successfully Gibsoned into the SamR backbone. To verify the presence of the insert we used junction primers which would amplify over both ends of the insert and could only amplify if the gBlock had been integrated into the SamR construct.
Junction primer PCR
Junction primers for the junction of promoter and gblock were iGEM1413 and iGEM1414 and were expected to give a fragment size of 377 base pairs.
9 out of the 12 colonies screened gave a positive result and these were further screened with a second set of junction primers on the other side of the gBlock. The junction primers for this were iGEM1416 and iGEM25 and were expected to yielad a fragment size of 432bp.
Again 9 of the 12 colonies were positive here. The PCR product was sequenced and corresponded with the expected sequence for the plasmid. 3 of the 8 colonies were selected to be grown up and Plasmid Prepped to remove the plasmid and allow restriction digest and sequencing of the plasmid for further verification.
Restriction Digest
The pUS203 plasmid was further confirmed with restriction digestion. Spe1, Pst1 and EcoR1 are in the prefix and suffix and should cut.  An Nhe1 restriction site was added into the gBlock code and so a cut with this enzyme would indicate the gBlock had been inserted. Sac1 should not cut and should be identical to the uncut control lane.
The digest results were as expected using SnapGene’s Virtual Digest tool. We were satisfied with these results despite the Sac1 error because the other fragments were as expected and most importantly because Nhe1 cut, which could only have occurred if the gblock had been incorporated into the plasmid. The Sac1 sample was heat-treated to inactive the Sac1 enzyme and used was used as a template to PCR the BioBrick. The results were still as expected which showed that the unexpected action of the Sac1 seen in the gel for the digest of this plasmid luckily didn’t reflect an error in the biobrick code.
Biobrick PCR and sequencing
The final check for this plasmid was to PCR the entire biobrick fragment using iGEM16 and iGEM25 and sequencing the product.
Back to top
Validation
To validate our araC-pBAD/IntI1 BioBrick it would need to be present in a low copy plasmid in order to reduce the amount produced as it is potentially toxic to the cell at high concentrations. Our intial plan was to insert the BioBrick into pUS202 but it turned out not to be low-copy after validation. The back up option was to ligate the BioBrick into pSB6A1. The initial attempt was not successful and due to time constraints we decided to try and test the integron construct in the pSB1C3 it was currently in even with the potential toxicity.
To test for the proper expression of our integron gene in the BioBrick we wanted to have an AttI site and integrase protein present in E. coli. This was so that upon transformation with a cassette (which has an AttC site) a co-integrate would quickly form with the AttI site and before the cassettes  could be lost out of the cell. Upon integration of the cassette the Gentomycin resistance (GmR) present on the cassette is expressed allowing us to select for the correct cells on Gentomycin plates. The negative control had no arabinose induction and so the cassette should not integrate and the cells should not be Gm resistant.
The initial experiments did not yield positive results and due to time constraints we have not been able to troubleshoot the experiment.
Back to top
Retrieved from " http://2014.igem.org/Team:USyd-Australia/pUS203 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
