<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_USyd-Australia_ProtocolHTML"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:USyd-Australia/ProtocolHTML">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:USyd-Australia/ProtocolHTML&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:USyd-Australia/ProtocolHTML&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:USyd-Australia/ProtocolHTML&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:USyd-Australia/ProtocolHTML" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:USyd-Australia/ProtocolHTML</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><H2>Agarose Gel Electrophoresis (<A href="https://2013.igem.org/Team:SydneyUni_Australia/Safety#Electrophoresis">Safety</A>)</H2><UL style="font-weight: normal;">
Gel electrophoresis is used to separate DNA based on their size and mobility through agarose. It relies on the fact that DNA is negatively charged, and that a uniform electric field can be applied across the semi-permeable agarose gel. 
For a 1% agarose gel:
<OL><LI>
Add 1 g of agarose to 100 mL TBE.
</LI><LI>
Gently mix and microwave heat until dissolved.
</LI><LI>
Prepare gel tray. Depending on the tank, use masking tape or end-formers to seal the two ends of the gel tray. If using end formers, seal the ends by pipetting a small volume of gel solution into the contact between the end formers and tank and allow to set.


</LI><LI>
Allow gel solution to cool before adding 0.5 µL of Gel Red and pouring into gel tray. Alternatively, exclude the Gel Red and stain with Ethidium Bromide after running the gel.
</LI><LI>
Insert the well comb into the appropriate position in the gel.
</LI><LI>
Once set remove the comb, tape/end formers and place into electrophoresis chamber and submerge in TBE.
</LI><LI>
Mix samples with loading buffer (use 1:5 ratio of loading buffer to sample) and pipette into wells. 
</LI><LI>
Apply 120-180V, depending on time constraints and desired resolution.
</LI></OL></UL><A href="#protstart">Back to top</A><H2>Enzymatic Ligation Assisted by Nucleases (ELAN)</H2><UL style="font-weight: normal;"><A href="http://www.nature.com/nprot/journal/v2/n9/abs/nprot.2007.325.html">ELAN</A> is a reaction in which ligation and enzymatic digestion of compatible but non-identical ends, like XbaI and SpeI, occurs simultaneously. We used this to produce stable circular cassettes.  Our final protocol, based on work in the PhD Thesis of Dr. Alicia Gestal:
<UL><LI>150U XbaI
</LI><LI>150U SpeI-HF
</LI><LI>2000U T4 DNA Ligase
</LI><LI>1X NEB Buffer 4
</LI><LI>1mM ATP
</LI><LI>2ug Clean PCR product
</LI></UL>
Make up the reaction volume to 1mL with sterile Milli-Q water.
Incubate overnight at room temperature, after which ligation can optionally be continued at 4degC.<A href="#protstart">Back to top</A><H2>Gibson Assembly</H2><UL style="font-weight: normal;">
Gibson Assembly is a method used to join multiple, overlapping fragments of DNA in a single reaction. It requires that adjacent fragments contain overlapping regions of about 20-80 bp. There are three enzymes involved in the reaction; an exonuclease that exposes a single-stranded 3’ overhang, a ligase that joins fragments with complementary overhangs and a polymerase that fills in the gaps left by the exonuclease.  
The full protocol details can be found at the <A href="https://www.neb.com/products/e2611-gibson-assembly-master-mix">NEB website</A><OL><LI> Add 10 µL 2x Gibson Assembly Reaction Buffer to each tube.
</LI><LI> Add the DNA fragments to be assembled; for a 2-3 fragment assembly use 0.02-0.5 pmol, and for a 4-6 fragment assembly use 0.2-1 pmol.*
</LI><LI> Add 10 µL commercially provided positive control DNA to the positive control tube.
</LI><LI> Make all tubes up to 20 µL*** with ROW.
</LI><LI>Incubate samples in a thermocycler at 50°C for 60 minutes. Following incubation, store samples on ice or at -20°C for subsequent transformation.
</LI></OL>
 * Optimized cloning efficiency is 50–100 ng of vectors with 2–3 fold of excess inserts. Use 5 times more of inserts if size is less than 200 bp. Optimized cloning efficiency is 50-100 ng of vectors with 2-3 fold of excess inserts. Use 5 times more of inserts if size is less than 200 bps. 
** Control reagents are provided for two experiments.
*** If greater numbers of fragments are assembled, additional Gibson Assembly Master Mix may be required

Our actual amounts we used when we finally got a working Gibson Assembly:

<CENTER><TABLE class="wikitable"><TBODY><TR><TH> aldA </TH><TH> p450 </TH><TH> tetR
</TH></TR><TR><TD>Water </TD><TD> 1.75 µL </TD><TD> 1 µL </TD><TD> 2 µL
</TD></TR><TR><TD>~ 2000bp pSB1C3 (at 25ng/µL), 0.15pmol/µL </TD><TD> 3 µL </TD><TD> 3 µL </TD><TD> 4 µL
</TD></TR><TR><TD>~ 600bp gBlocks (at 20ng/µL), 0.5pmol/µL </TD><TD> 2 x 2 µL </TD><TD> 1 x 2 µL </TD><TD> -
</TD></TR><TR><TD>~ 600bp gBlocks (at 40ng/µL), 1.0pmol/µL </TD><TD> - </TD><TD> 4 x 1  µL </TD><TD> -
</TD></TR><TR><TD>~ 400bp gBlocks (at 20ng/µL), 0.75pmol/µL </TD><TD> 1.3 µL </TD><TD> - </TD><TD> 2 x 2 µL
</TD></TR><TR><TD>NEB Gibson Assembly Master Mix</TD><TD> 10  µL </TD><TD> 10 µL </TD><TD> 10 µL
</TD></TR><TR><TD><B>20 µL</B></TD><TD><B>20 µL</B></TD><TD><B>20 µL</B></TD></TR></TBODY></TABLE></CENTER></UL><A href="#protstart">Back to top</A><H2>Heat-shock Transformation (<A href="https://2013.igem.org/Team:SydneyUni_Australia/Safety#Transformation">Safety</A>)</H2><UL style="font-weight: normal;">
Transformation is a technique by which DNA may be inserted into competent cells. Transformation occurs naturally when cells uptake and express exogenous DNA from their environment. The process can be reproduced in the lab. 
Heat shock transformation uses a rapid change in temperature to cause plasmids to enter cells via pores in the membrane. The introduced DNA will often contain a marker gene (often antibiotic resistance) so that successfully transformed cells can be grown on selective media (often an antibiotic) which corresponds to the marker gene. <OL><LI>
Set-up a heat-block or water-batch at 42°C.
</LI><LI>
Retrieve aliquots of cells from -80°C freezer. Thaw on ice.
</LI><LI>
Add 1 pg-100 ng (1 µL - 5 µL) of plasmid DNA into cell suspension.
</LI><LI>
Incubate tubes 42°C for 30-45 seconds.
</LI><LI>
Return to ice and quickly add 1 mL of LB-broth. Incubate tubes for 1 hour on the 37°C shakers.
</LI><LI>
Spread-plate 100 µL of cells on LB containing the appropriate antibiotic.
</LI><LI>
Alternatively, try diluting or concentrating cells before plating. For instance, centrifuge the tubes, pour off supernatant, resuspend cells in the final drop remaining in the tube and spread-plate the last 100 µL of cells.
</LI><LI>
Incubate overnight at 37°C.
</LI></OL></UL><A href="#protstart">Back to top</A><H2>Ligation</H2><UL style="font-weight: normal;">
Ligase facilitates binding between complementary sequences of DNA. Ligation allows fragments of DNA with sticky ends to be joined together, but the enzymes also has other uses like in Gibson Assembly. Ligase is not thermostable, so there is an efficiency trade-off between increasing the rate of ligation and the rate of ligase degradation at higher temperatures. The following protocol is for the ligation we performed most often; that of cloning an PCR product into a plasmid prep. The following volumes thus hold only with the concentration of our plasmid prep. The ligation reaction is quite flexible, and the following protocol can be applied to general ligations, as long as care is taken to maintain a ~3:1 molar ratio of insert:vector, and that the total DNA concentration does not exceed 10 ng/µl. <OL><LI>
Digest plasmid e.g. Use 250 ng in digest volume of 100 µL although there is a wide acceptable range for this. (See restriction digest protocol)
</LI><LI>
At the same time as step 1, digest PCR product e.g. Use 1 µg in 100 µL digest. (See restriction digest protocol)
</LI><LI>
Combine on ice 2 µL of T4 ligase buffer (10x), 8 µL purified insert, 8 µL of purified vector, and 2 µL of T4 DNA ligase. Make sure ligase enzyme remains in ice at all times. 
</LI><LI>
Split solution between two reaction tubes. Incubate one tube for an hour at room temperature. Incubate the other overnight at 4°C. 
</LI><LI>
Throw out ligase buffer. Do not return to -80°C freezer. 
</LI></OL></UL><A href="#protstart">Back to top</A><H2>PCR</H2><UL style="font-weight: normal;">
The Polymerase Chain Reaction is used to amplify segments of DNA to high concentrations as a screen for the presence of the target or for further manipulation. The following protocol is readily tweaked depending on the specifics of the template DNA, primers and polymerase of choice.<OL><LI>
PCR of multiple samples is sped up significantly by making a master mix; the volumes below are for one 50 µL reaction, but the volumes would be multiplied by the number of reactions required and mixed together in a single tube. 

- 5 µL 10x Pfu buffer/NEB Thermopol Buffer

- 1 µL dNTPs at 10 uM (final 200mM)

- 1 µL primer (F) (final 0.5-1uM)

- 1 µL primer (R) (final 0.5-1uM)

- 40.5 µL sterile MQ water

- 0.5 µL Pfu/Taq polymerase
</LI><LI>
Aliquot master mix into PCR tubes (49 µL) then add 1 µL of the template DNA. Alternatively, if performing a colony PCR, aliquot 50 µL of master mix into a tube and resuspend cells directly from the plate into the tube. (Only dip the toothpick 3-5 times in the master mix).
</LI><LI>
Start the thermocycler at the setting desired. Cycles must have a peak high enough to denature template DNA,  a trough low enough to allow annealing of primer pair, followed by an intermediate stage for the optimal polymerase activity.
</LI></OL>
NB. Keep enzyme on ice and return to the freezer ASAP. 

</UL><A href="#protstart">Back to top</A><H2>Plasmid Mini Prep - 100 mL (<A href="https://2013.igem.org/Team:SydneyUni_Australia/Safety#PlasmidMiniPrep">Safety</A>)</H2><UL style="font-weight: normal;">
This is the process used to extract plasmids from bacterial cells. Plasmids can then be used for screening or further manipulation.

<OL><LI> Pellet 100 mL culture in 2 x 50 mL Falcon tubes. Resuspend cells in 4 mL TE buffer, combine resuspended pellets in one tube. 
</LI><LI>
Add 8 mL lysis solution (SDS-OH), mix well by inversion and shaking(~10 sec). Should go viscous. Leave at room temp for 15 min.  
</LI><LI>
Add 6 mL ice-cold precipitation solution (K.Ac). Shake briefly – essential that K.Ac is thoroughly mixed in. Viscosity should disappear, and white precipitate appears. Keep on ice 15 min. 
</LI><LI>
Spin at top speed (4000 rpm) in cold Centaur centrifuge for 15 min. Recover tube immediately and handle gently (pellet is soft and easily resuspended). Pour supernatant into new tube. Try to avoid the white junk, but don’t worry if little bits of it get transferred.
</LI><LI>
Add an equal volume isopropanol (~15 mL), mix well ice 15 min. 
</LI><LI>
Spin at top speed (4000 rpm) in Centaur centrifuge (doesn’t need to be cold) for 15 min, pour off supernatant, keep pellet. 
</LI><LI>
Add 10 mL 70% ethanol to pellet, resuspend by brief vortexing, leave for 5 min at room temp. Spin 15 min in Centaur centrifuge (doesn’t need to be cold). Pour off supernatant again. 
</LI><LI>
Drain off excess supernatant by gentle tapping on paper towel, then heat at 50°C for approx 10 min to remove ethanol and isopropanol. 
</LI><LI>
Redissolve pellet in 2 mL TE with mixing (tapping tube ~ 1 min , don’t vortex too much from this point onward). Can heat if necessary (eg. 50°C, 10-30 min). Note that plasmid DNA is much more soluble than chromosomal DNA, and dissolves preferentially. Solution should look slightly viscous (traces of chromosomal DNA still present).  
</LI><LI>
Split prep into 2 x 1 mL in Eppi tubes. Extract each tube with phenol: chloroform: isoamyl (PCI), as follows. Suck up 500 µL PCI from under the aqueous layer in the reagent bottle, transfer to Eppi tube. Vortex for ~5-10 sec until a uniform milky white emulsion is obtained. Centrifuge 5 min. Transfer top phase (aqueous) to a new Eppi tube. Discard bottom phase (PCI) into phenol waste. Avoid the white junk at the interface between phases.   
</LI><LI>
Repeat solvent extractions using 500 µl chloroform:isoamyl (CI, µl) per tube, as described for PCI above. After mixing &amp; centrifugation, keep top aqueous phase, discard bottom CI phase into waste.   
</LI><LI>
Split DNA prep into 4 equally sized aliquots (~400 µl each) Precipitate DNA by adding 1/10-volume 3M Na-acetate (~40 µl) to each tube, and then add 2 volumes cold ethanol (~1 ml). 
Incubate &gt;2 hr at -20°C (overnight is fine).
</LI><LI>
Spin for 10 min, drain off supernatant, rinse pellet with 70% ethanol (as above, but using 500 µl 70% EtOH), drain excess EtOH off, then dry 10 min at 50°C. 
</LI><LI>
Redissolve plasmid in 100 µL EB. Expected yields range from approx 10 µg plasmid per ml culture (eg pUC/pGEM) down to 0.2 µg plasmid per mL culture (eg. RSF1010). Final expected DNA conc. may range from approx 10-500 ng/µL. 
</LI></OL><B>Solutions: (see Sambrook Appendix 1)</B>

•  TE (solution I): 10 mM Tris, 10 mM EDTA, pH 8. Autoclaved.

•  Lysis sol’n (solution II): 0.2 M NaOH, 1% SDS. Prepare fresh from separate   
     stocks (NaOH – 2 M, Autoclaved ; SDS – 10%)   
        
•  Precipitation sol’n (solution III): 3 M potassium, 5 M acetate, pH 4.8. 							 Autoclaved.

•  Na-acetate: 3M, pH 4.8 (adjust with conc. acetic acid), autoclaved.

•  EB (Elution buffer): 5 mM Tris, pH 8. Autoclaved. 

NOTE: RNAse can be added to TE buffer at the start, or to the EB/TE at the end. RNA doesn’t interfere with most things, but can make gels look messy and obscure small DNA bands. Add RNase from conc., boiled stock (10 mg/ml) to final conc. of ~100 µg/ml. </UL><A href="#protstart">Back to top</A><H2>Preparation of Common Solutions</H2><UL style="font-weight: normal;"><B>LB media</B>
LB media is a common growth media used for the propagation of bacterial cells. 
<OL><LI>Mix 10 g Tryptone, 5 g Yeast extract, 5 g NaCl and 1 L of water, or use the same ratio in the required volume.</LI><LI>Autoclave the solution to sterilise.</LI><LI>Antibiotics can be added to make the media selective for antibiotic resistant cells. This must be added after autoclaving.</LI></OL><B>Potassium-phosphate Buffer</B>
This buffer is Cl<SUP>-</SUP> free, which was very important during our chloride assays. Cells needed to be harvested and washed repeatedly in potassium-phosphate buffer to minimise the noise caused by salt from growth in LB. 
<OL><LI>Add 2.27 g/L K<SUB>2</SUB>HPO<SUB>4</SUB> to 0.95 g/l KH<SUB>2</SUB>PO<SUB>4</SUB>.</LI><LI>Make up to 1 L with water.
</LI></OL></UL><A href="#protstart">Back to top</A><H2>Preparation of RbCl Competent Bacterial Cells (<A href="https://2013.igem.org/Team:SydneyUni_Australia/Safety#rbcl">Safety</A>)</H2><UL style="font-weight: normal;">
Competent bacterial cells are used for transformation of plasmids to allow propagation of plasmids and for expression of the genes of interest. <OL><LI>
Streak E.coli strain onto LB agar and incubate overnight at 37°C.
</LI><LI>
Inoculate 5 mL LB broth with a single colony from LB plate. Incubate at 37°C with shaking.
</LI><LI>
Measure OD<SUB>600</SUB> of overnight culture. Aseptically inoculate 100 mL LB in 500 mL Schott Bottle with enough culture to obtain OD<SUB>600</SUB> of ~ 0.05.
</LI><LI>
Grow cells at 37°C, shaking until culture reaches OD<SUB>600</SUB> of ~ 0.5.
</LI><LI>
Aseptically transfer to centrifuge bottles and keep on ice.
</LI><LI>
Harvest cells at 4000 rpm, 4°C for 10 min in cold centrifuge.
</LI><LI>
Working in cold room with cells in ice, pour off supernatant and remove residual liquid by pipetting.
</LI><LI>
Resuspend pellet gently in 33 mL RF1 solution. Mix by pipetting up and down.
</LI><LI>
Incubate on wet ice for 1 hour.
</LI><LI>
Pellet bacteria at 4000 rpm 4°C for 10 min.
</LI><LI>
Again remove supernatant (work in cold room).
</LI><LI>
Resuspend pellet in 8 mL RF2 solution and mix.
</LI><LI>
Incubate on wet ice for 15 min.
</LI><LI>
Working quickly, aliquot 110 µL of cell suspension into pre-chilled microcentrifuge tubes. Once dispensed, collect all tubes and store in -80°C freezer.
</LI></OL></UL><A href="#protstart">Back to top</A><H2>QiaQuick DNA Purification Kit</H2><UL style="font-weight: normal;">
Working with DNA often involves adding enzymes (polymerases, restriction enzymes, ligases), which may subsequently need to be removed before they prevent or contaminate the next stage of work. DNA purification is pretty quick and easy using a store-bought Kit. 

Also consult the instruction booklet that comes with the Qiagen kit – the protocol below only gives the bare essentials required. This protocol below is good for restriction fragments, plasmids, and PCR products. It is NOT good for genomic or chromosomal DNA, which is too big to stick to the column effectively. Use the FastPrep reagents or CTAB-phenol type prep instead for genomic DNA. 


<OL><LI> Mix your DNA sample with the appropriate buffer, in the appropriate ratio: 
     - DNA &lt; 4 kb: Mix 1 vol sample with 3 vol of QG buffer.
     - DNA &gt; 4 kb: Mix 1 vol sample with 3 vol of QG buffer + 1 vol isopropanol. 
     - PCR products: Mix 1 vol sample with 5 vol PB buffer. 
</LI><LI> Load the mixture onto a Qiaquick spin column (purple) and spin 30 sec.   
    Discard the flow-through, and replace spin column in the catch tube. 
       The spin column will hold a max. of 800 µL sample and has a max. binding  
   capacity of approx 10 µg DNA. You can wash through multiple 800 µL aliquots
   of DNA+QG if you have a lot of sample, so long as the total amount of DNA
   added doesn’t exceed approx 10 µg per column. 
</LI><LI> Add 750 µl of buffer PE to the column, allow to sit for ~2 min, then spin 30 sec,
   discard flow-through, replace spin column in catch tube.   
</LI><LI> Spin again for 30 sec to remove all traces of PE from the column. Discard both
     the flow-through and catch tube, and transfer spin column onto a clean
     Kimwipe. Leave the column lid open. Transfer Kimwipe to 50°C incubator box,
     and allow to dry for 10 min. 
</LI><LI> Transfer spin column to a sterile 1.5 mL Eppi tube, and add 50 µL* of EB buffer
     (5 mM Tris, pH 8) to the centre of the spin column – ie on the membrane, not
     the walls of tube. Allow to sit for 5 min. Spin 30 sec, retain Eppi tube with DNA
     solution in EB, discard spin column.  
</LI>

* Can reduce this to as little as 20 µL EB to give a more concentrated DNA solution, but keep in mind you will lose approx 3-5 µL EB during the procedure.  


</OL></UL><A href="#protstart">Back to top</A><H2>Restriction Digest </H2><UL style="font-weight: normal;">
Restriction digestion employs restriction enzymes which recognise specific sites in a DNA sequence and will cut the strands at those sites. This may be used for diagnostic purposes by separating DNA into fragments of predictable lengths or for assembly by generating short overlaps at the restriction sites. 

<OL><LI> Add 5 µL of appropriate 10x buffer to a microcentrifuge tube. The buffer choice depends on the restriction enzyme used and can be checked from readily available tables.
</LI><LI> Add DNA sample solution (~1 µg DNA).
</LI><LI> Make tube up to 49 µL with reverse osmosis water.
</LI><LI> Add restriction enzyme; 10 U activity is sufficient, which is normally the activity of 1 µL restriction enzyme solution. The restriction enzyme volume should not be more than 1/10 of the total reaction volume.
</LI><LI> Incubate for an hour at the optimal temperature for restriction enzyme activity; again, this should be checked from relevant data tables.
</LI></OL>

Restriction digests with two enzymes simultaneously are also possible and were performed over the course of the project. This involves the same protocol as above except that the reaction mixture is made up to 48 µL in step 3. This is only possible if the enzymes have compatible reaction buffers and optimal temperatures; if they do not, then you must perform two sequential digests with a purification step in between (see QiaQuick DNA Purification Kit protocol)

</UL><A href="#protstart">Back to top</A><H2>SDS-PAGE (<A href="https://2013.igem.org/Team:SydneyUni_Australia/Safety#sdspage">Safety</A>)</H2><UL style="font-weight: normal;">
SDS-PAGE is a technique used to separate proteins based on molecular mass. SDS differs from other gel electrophoresis methods as it uses sodium dodecyl sulphate (SDS), an anion detergent, to denature protein and produce a negative charge based on their molecular mass.
<B>Sample Preparation</B><OL><LI> Inoculate 10 mL of LB broth with each culture. Leave on shaker at 37°C overnight.
</LI><LI> Spin liquid cultures at top speed (4000 rpm) in cold Centaur centrifuge for 5 min at 4°C.
</LI><LI> Remove supernatant and resuspend cells in 1 mL elution buffer.
</LI><LI> Lyse cells using glass beads in a beadbeater (5.5 m/s, 30 s).
</LI><LI> Spin for 5 min to pellet cell debris.
</LI><LI> Transfer supernatant to clean microfuge tubes and use in SDS-PAGE or store at -20°C.
</LI><LI> For use in SDS-PAGE, add 2x loading buffer (0.06 M Tris, 2% (w/v) SDS, 5% (v/v) ß-mercaptoethanol, 10% (v/v) glycerol 0.1% (w/v) bromophenol blue, pH 6.8) to equal volume of sample.
</LI><LI> Heat each sample to 99°C for 5 min.
</LI></OL><B>Gel preparation:</B> the gel follows the general method of Laemmli; a stacking gel is used to force the samples into a sharp band, and a resolving gel to separate by mass.
<OL><LI> Prepare the resolving gel first by adding reagents as below. 
               - ROW – 3.4 mL 
               - Buffer – 2.5 mL 
               - 30% acrylamide/bisacrylamide – 4 mL 
               - 10% SDS – 0.1 mL 
               - 10% ammonium persulfate solution – 50 µL 
               - TEMED – 5 µL 
</LI><LI> After adding the TEMED, immediately pipette the liquid mixture into the gel apparatus until it reaches approximately three-quarters of the way to the top.
</LI><LI> Pipette ~2 mm ROW onto the top of the gel and leave to set.
</LI><LI> Once the resolving gel has set, prepare the stacking gel as below. 
               - ROW – 3.05 mL 
               - Buffer – 1.25 mL 
               - 30% acrylamide/bisacrylamide – 0.65 mL 
               - 10% SDS – 0.05 mL 
               - 10% ammonium persulfate solution – 25 µL 
               - TEMED – 10 µL 
</LI><LI> After adding the TEMED, pour off the water on top of the resolving gel and pipette in the stacking gel mixture until it reaches ~5 mm from the top. Add-well formers and allow to set.
</LI><LI> Once the gel has set, add running buffer (0.025 M Tris, 0.192 M glycine, 0.1% w/v SDS). There are two buffer levels; in the inner chamber buffer should be added so that it just covers the wells, and in the outer chamber it should be added to ~1-2 cm above the bottom of the gel. Remove the well-former.
</LI><LI> Load protein standard ladder and different volumes of treated cell lysates to each well as appropriate.
</LI><LI> Run gel for ~40 min at 200 V. Monitor progression of the experiment by migration of the tracking dye.
</LI><LI> Once the gel has finished running, switch off power and remove the gel from the apparatus. 
</LI><LI> Stain with Coomassie blue (0.005% w/v in acetic acid (7% v/v) and methanol (10% v/v)) rocking gently for 15 min.
</LI><LI> Pour off the staining solution and add destaining solution (acetic acid (10% v/v) and methanol (10% v/v)). Rock gently with the destaining solution for 30 min.
</LI><LI> Pour off destaining solution and add water. Image the gel.
</LI></OL></UL><A href="#protstart">Back to top</A><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:USyd-Australia/ProtocolHTML">http://2014.igem.org/Team:USyd-Australia/ProtocolHTML</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:USyd-Australia/ProtocolHTML" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:USyd-Australia/ProtocolHTML" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:USyd-Australia/ProtocolHTML&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:USyd-Australia/ProtocolHTML&amp;oldid=398695" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></UL></DIV></DIV></DIV></BODY></HTML>