Team:IIT-Madras/Improve
Team IIT-Madras
Home
Team
Team Collaborations
Project
Description Design Notebook Contribution Demonstrate Improve Attribution
Human Practices
Silver HP Integrated and Gold Public Engagement
Improve
As part of their iGEM 2014 project, Team University of Pennsylvania characterized a fascinating and underused organism: Magnetospirillum magneticum (AMB-1), a bacterium that aligns with magnetic fields. By developing, testing, and optimizing protocols for its growth and transformation, and then making them easily accessible in a convenient Strain Spec Sheet, they tried to establish AMB-1 as an easily engineered organism.
The Penn University 2014 strain spec sheet
As part of their human practices, they contacted other iGEM teams participating that year to compile characterization and protocols data if they were working with unconventional organisms too. They designed a “biological chassis specification sheet” that contained necessary information and protocols for the successful growth and engineering of the organisms.
With very similar ideas and intentions, and much bigger ambition, we created an open source database of characterization and protocols data for unconventional chassis organisms. We improved on University of Pennsylvania Team’s idea of creating “biological chassis specification sheet” and turned it into a much bigger version.
The blueprint of the data collected for each chassis entry is as given below.
Scientific name (genus, species, and strain) and taxonomy
Biosafety level
A description of what makes the host special for synthetic biology applications.
Genotype
Growth characteristics
Culture sources for obtaining the organism
Maintenance protocols
Transformation protocols
Other protocols
Biobrick parts for this organism
Vectors for the organism
Metabolic models
Genome sequence
We’ve tried to keep it compact yet comprehensive, taking feedback from the stakeholders, people who’ve worked with new chassis and filled data on the database.
In an effort to make our database a one-stop solution to working with chassis, we added tools that can help in working with the biochemical and genomics of the chassis. We plan to develop an assortment of tools to make it easier to work with a new chassis.
Currently, our biggest tool achievement has been the development of an interactive codon optimizer which can be used to manually edit the optimized sequence to remove restriction sites or reduce GC content, using a drop-down menu that shows codons that can be used to replace the current codon without changing the amino acid.
In the future, we plan to develop more chassis tools such as those for genome analysis and metabolic simulations.
Contact Us
igemiitmadras@gmail.com
Department of Biotechnology
Indian Institute of Technology Madras
Sardar Patel Road
Chennai, India - 600036
