<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_HKUST_Project_Module_1"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:HKUST/Project/Module_1">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:HKUST/Project/Module_1&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:HKUST/Project/Module_1&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:HKUST/Project/Module_1&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:HKUST/Project/Module_1" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:HKUST/Project/Module 1</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><P><TITLE>Team: HKUST</TITLE></P><DIV id="body_content_project"><DIV id="sidebar"><UL><LI><H3 class="cat">Team</H3></LI><LI><A href="https://2010.igem.org/Team:HKUST">Here we are!</A></LI><LI><H3 class="cat">Project</H3></LI><LI><A href="https://2010.igem.org/Team:HKUST/Project">Abstract</A></LI><LI><A href="https://2010.igem.org/Team:HKUST/Project/Module_1">Module 1</A></LI><LI><A href="https://2010.igem.org/Team:HKUST/Project/Module_2">Module 2</A></LI><LI><H3 class="cat">Lab Notebook</H3></LI><LI><A href="https://2010.igem.org/Team:HKUST/Notebook">Calender</A></LI><LI><H3 class="cat">Human Pratice</H3></LI><LI><A href="https://2010.igem.org/Team:HKUST/Human_Practice#workshop">Workshop</A></LI><LI><A href="https://2010.igem.org/Team:HKUST/Human_Practice#comments">Comments</A></LI><LI><A href="https://2010.igem.org/Team:HKUST/Human_Practice#memories">Memories</A></LI><LI><H3 class="cat">Acknowledgement</H3></LI><LI><A href="https://2010.igem.org/Team:HKUST/Acknowledgement#igem">HKUST iGEM team</A></LI><LI><A href="https://2010.igem.org/Team:HKUST/Acknowledgement#workshop">Workshop</A></LI></UL></DIV><DIV id="module_1"><P class="h1"><B> Autoinducing Peptide (AIP) Sensor on Lactobacillus Cell Membrane</B></P><P class="h2"><B>Background:</B></P><P class="h2"><B>1.	Two-Component Signaling System (TCS)</B></P><P class="content">In many bacterial species, Two-Component  Signaling System (TCS) plays a prominent role in quorum sensing. A classical TCS  consists of a transmembrane receptor, which is a histidine protein kinase (HPK),  and a cytoplasmic response regulator (RR)<STRONG> [1]</STRONG>.When  bound by autoinducing peptides (AIP), the transmembrane receptor HPK passes a  phosphate group to the downstream cytoplasmic response regulator RR. The  cytoplasmic response regulator will then up-regulates or down-regulates various  virulence and bacteriocin productions under specific situations <STRONG>[2]</STRONG>. Figure  1 below illustrates the components and transduction pathway of a typical two-component  signaling system:
</P><P class="h2"><B>2.AgrBDCA System in <EM>Staphylococcus aureus</EM></B></P><P class="content">The <EM>agr</EM> locus in <EM>Staphylococcus aureus</EM> consists  of two transcription units RNA II and RNA III; they are responsible for self-regulation  and effect response respectively <STRONG>[3][4]</STRONG>. RNA II, driven by P2 promoter, contains 4  genes agrB, agrD, agrC and agrA which are responsible for <EM>agr</EM> quorum sensing in <EM>S. aureus</EM>.  Products of agrB and agrD work cooperatively to produce an AIP, which is later secreted  out of the cell and functions as a signaling molecule. Gene products AgrC and  AgrA constitute a typical two-component signaling system, in which AgrC, the  AIP sensor, consists of two domains, a transmembrane receptor domain and a  cytoplasmic histidine protein kinase (HPK) domain. Once the transmembrane  receptor domain of AgrC is bound by AIP, its cytoplasmic HPK domain will pass a  phosphate group downwards to AgrA, the response regulator. AgrA functions as a  transcription factor and activates the two promoters P2 and P3 <STRONG>[5][6]</STRONG>.  RNA III, driven by P3 promoter, is related to exotoxin production in <EM>S. aureus</EM><STRONG> [7]</STRONG>.Figure 2 below shows the pathways involved  in <EM>S. aureus</EM> AgrBDCA system:
</P><P class="content">It has been  reported that there are altogether 4 types of <EM>agr</EM> loci; each type of <EM>agr</EM> locus encodes AgrC and corresponding AIP slightly different from others. AgrC  is only activated by AIP belonging to the same group and is usually inhibited  by AIP from other groups <STRONG>[8]</STRONG>.Different groups of AgrC and their corresponding  AIPs are highly involved in various human diseases or infections. For example,  food poisoning caused by staphylococcal enterotoxins is usually associated with <EM>agr</EM> group I and <EM>agr</EM> group II <EM>S. aureus</EM>; toxic  shock syndrome toxin 1 (TSST-1), which results in high fever, low blood  pressure and malaise of human, is produced by <EM>agr</EM> group III <EM>S. aureus</EM><STRONG>[9]</STRONG>.
 </P><P class="content">The characteristics  of <EM>S. aureus</EM> AgrBDCA system  aforementioned make it possible to build an engineered AIP sensor on the membrane  of a non-pathogenic bacterial species. The engineered AIP sensor, localized on the  non-virulent bacteria, could therefore detect the presence of AIP molecules  released by <EM>S. aureus</EM>. Upon receiving  this signal, the non-virulent bacteria could be automatically induced to  synthesize AIP’s competitor, RNAIII inhibiting peptide (RIP), and hence, repress  the activation of AgrC and toxin production in <EM>S. aureus</EM>.
</P><P class="h2">
3.	PlnABCD System in <EM>Lactobacillus plantarum</EM> WCFS1
</P><P class="content">Several <EM>agr</EM>-like  quorum sensing systems in <EM>Lactobacillus plantarum  WCFS1</EM> have been identified, among which the PlnABCD system is the best  studied <STRONG>[1]</STRONG>. The PlnABCD system, similar  to the AgrBDCA system, is a self-regulating system in <EM>L. plantarum WCFS1</EM>. The transmembrane receptor HPK PlnB, together  with two response regulators PlnC and PlnD, constitutes a two-component signaling  system (TCS) in <EM>L. plantarum WCFS1</EM>. PlnA,  the inducing peptide (IP) of <EM>L. plantarum  WCFS1</EM>, activates PlnB and thereby phosphoylates PlnC or PlnD. PlnC and PlnD  are upregulator and downreuglater of plnA promoter respectively, and therefore,  would activate/suppress the transcription initiation of plnABCD. The transcription  of plnABCD is usually associated with <EM>Lactobacillus  plantarum WCFS1</EM> bacteriocin production <STRONG>[10]</STRONG>.  Figure 3 below shows the pathways involved in <EM>Lactobacillus plantarum WCFS1</EM> PlnABCD system:</P><P class="content">Transmembrane  signal sensors of both PlnABCD and AgrBDCA systems, i.e. PlnB (of <EM>L. plantarum WCFS1</EM>) and AgrC (of<EM> S.aureus</EM>), belong to a same subgroup of  histidine protein kinases – the HPK10 subfamily. PlnB and AgrC share  a highly homologous cytoplasmic HPK domain regarding amino acid sequence,  tertiary structure and biochemical function <STRONG>[6][10]</STRONG>. However, the sequence of PlnB and AgrC’s transmembrane  domain are distinct from each other, probably due to the ligand binding  specificity required by their corresponding signaling molecules.</P><P class="content">Based on the  homology between AgrBDCA and PlnABCD at the cytoplasmic HPK domain, we decided  to construct a chimeric AIP sensor in<EM> Lactobacillus</EM>.  Such a fusion receptor would link the transmembrane signal sensing domain of AgrCand the cytoplasmic HPK domain of PlnB.  By localizing such a fusion protein on <EM>Lactobacillus </EM>plasma membrane, it is hoped that <EM>Lactobacillus </EM>can successfully detect the presence of autoinducing peptides (AIPs)  produced by <EM>S. aureus</EM>. Upon the  detection of AIP molecules, the chimeric protein kinase will transduce the  signal to <EM>L. plantarum WCFS1 </EM>intrinsic  downstream pathways.</P><P class="h2"><STRONG>Experiment Design:</STRONG></P><P class="h2"><STRONG>Construction of Chimeric AIP Receptor </STRONG></P><P class="content">Johnsborg reported  in her study that the transmembrane domain, other than the cytoplasmic domain,  of a pheromone receptor would be responsible for the inducing specificity of  corresponding signal molecules <STRONG>[10]</STRONG>.A fusion AIP sensor, AgrC  (transmembrane domain) - PlnB (cytoplasmic catalytic domain), is designed to  sense AIP molecules released by <EM>S.aureus</EM> and thereby activate <EM>Lactobacillus </EM>endogenous  response regulators.</P><P class="content">We fused the  transmembrane domain of AgrC (amino acid 1-188, AIP receptor in <EM>S.aureus</EM>) with the cytoplasmic HPK  domain of PlnB (amino acid 212-441, pheromone receptor in <EM>L.plantarum WCFS1</EM>). The fusion point is Leucine, a conserved amino  acid in both AgrC and PlnB (186 in AgrC/212 in PlnB), linking the transmembrane  domain and cytoplasmic HPK domain. The AgrC-PlnB chimeric receptor was cloned  into the multiple cloning site of <EM>e.coli  - Lactobacillus</EM>shuttle vector pMG36e, which contains a constitutive  promoter P32. </P><P class="h2"><STRONG>Localization Test of Chimeric Receptor </STRONG></P><P class="content">To first examine  the proper localization of AgrC-PlnB on the membrane of <EM>L. plantarum WCFS1</EM>, coding sequence of fluorescence protein mCherry  was added after the chimeric receptor AgrC-PlnB. Since the N terminal of AgrC  and PlnB is putatively associated with membrane localization, mCherry is fused  at C terminal of the fusion construct, as denoted below:</P><P class="content">I.AgrC-PlnB fused with mCherry fluorescence protein. P32  constitutive promoter in shuttle vector pMG36e will drive the transcription  process. 
II.Complete AgrC fused with mCherry, driven by P32.</P><P class="content">The localization  test will be performed by observing pMG36e-transformed <EM>L. plantarum WCFS1</EM> under fluorescence microscope.</P><P class="h2"><STRONG>Tests of the Functionality of the Chimeric Receptor</STRONG></P><P class="h2"><STRONG>Experiment 1: Test in <EM>L.plantarum WCFS1</EM></STRONG></P><P class="content">The GusA reporter  assay will be used in testing the functionality of the chimeric receptor AgrC-PlnB  in <EM>L. plantarum WCFS1</EM>. A gusA  reporter unit consisting of an inducible plnA promoter and a gusA reporter gene  was cloned into pMG36e at upstream of P32 promoter; the agrC/agrC-plnB was  cloned into the multiple cloning site of pMG36e which is at the downstream of  P32 promoter. As for the control groups, an empty pMG36e and a pMG36e which 1has  only the gusA reporter unit were prepared. The construct-inserted shuttle  vectors will then be transformed into <EM>L. plantarum  WCFS1</EM>.</P><P class="content">
I.	P32 promoter alone;
II.	gusA reporter unit driven by inducible promoter plnA; P32 promoter alone downstream;
III.	gusA reporter driven by inducible promoter plnA; P32 promoter driving agrC-plnB AIP receptor
IV.	gusA reporter driven by inducible promoter plnA; P32 promoter driving agrC AIP receptor</P><P class="content">Each of the four  groups aforementioned will be treated by 1) AIP (inducing peptide for <EM>S. aureus</EM>) induction 2) IP (inducing  peptide for <EM>Lactobacillus</EM>) induction  and 3) control without pheromone induction. </P><P class="content">Due to the  existence of endogenous <EM>pln</EM> locus in <EM>L. plantarum WCFS1</EM>, response regulator  PlnC would possibly bind to the plnA promoter in shuttle vector pMG36e as well  as the plnA promoter in <EM>L. plantarum  WCFS1</EM> genomic DNA. PlnA in the genomic DNA would then synthesize inducing  peptides IP, which could bind to the original transmembrane sensor PlnB.  Consequently, phosphorylated PlnC would result in background noise for testing  our designed constructs. In another word, the natural pathway of <EM>pln</EM> quorum sensing and the introduced  pathway of AIP signal transduction would interfere with each other; the gusA  expression level could not directly indicate the response from chimeric AIP  sensor. </P><P class="content">Once  synthesized IP, synthesized AIP or placebo was applied, GusA expression level would  be examined every 1 hour for consecutive 10 hours. Group 3) is expected to give  the highest level of gusA expression among all AIP-induced groups.</P><P class="h2"><B>Experiment 2: Test in <EM>L.sakei</EM> Lb790</B></P><P class="content">To reduce the  noise brought by the cross talk between the natural pathway of <EM>pln</EM> quorum sensing and the introduced  pathway of AIP signal transduction, modified constructs will be built and  transformed into <EM>L. sakei Lb790</EM>, a <EM>Lactobacillus</EM> strain which does not have  the <EM>pln</EM> locus. A new part of plnC following  RBS is added to the construct to enable the proper signal transduction. </P><P class="content">
I.	P32 promoter alone, serving as a control group 
II.	gusA reporter driven by inducible promoter plnA with P32 promoter alone serving as a control group 
III.	gusA reporter driven by inducible promoter plnA with P32 promoter driving agrC-plnB AIP receptor and plnC  
IV.	gusA reporter driven by inducible promoter plnA with P32 promoter driving agrC AIP receptor and plnC as a control group to Group III</P><P class="content">GusA  expression level will be measured 3 hours after pheromone induction. Compared  with the control groups, the chimeric receptor AgrC-PlnB would be proved to be  functional if the GusA expression was comparatively high.</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:HKUST/Project/Module_1">http://2010.igem.org/Team:HKUST/Project/Module_1</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:HKUST/Project/Module_1" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:HKUST/Project/Module_1" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:HKUST/Project/Module_1&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:HKUST/Project/Module_1&amp;oldid=87282" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>