"
Page
Discussion
View source
History
teams
Log in
 
Team:Edinburgh/Project/Non-antibiotic-Markers/Gene-replacement-strategy
From 2012.igem.org
Gmail
Google+
Twitter
Facebook
Blogger
Home
Welcome!
Team
Meet the team!
Profile
Project
Bio-electric Interface
Alternative selectable and counter-selectable markers
Citrobacter freundii
Parts
Modelling
Kappa
Biosensors
Human
Practices
Crowdfunding
DIY Bio
Perceptions of the Public
Blog
Laws and Legislations
Questioning Friendliness
Failure
Blue-Sky Research
Safety
Question 1
Question 2
Question 3
Question 4
Notebook
Attributions
Bio-electric Interface
Background
BioBrick Cloning and Characterisation
Microbial Half Fuel Cells
Discussion
Alternative Selectable and Counter-selectable Markers
Introduction
Nitroreductase
Sucrose Hydrolase
Haloalkane Dehalogenase
Levansucrase
Gene Replacement
Strategy
Citrobacter freundii
Introduction
Replicon compatibility
Growth in sea salts
Sugar use
Lac promoter
Genome sequencing
Valencia Biocampus
BioBrick Characterisation
Alternative selectable and counter-selectable markers:
Two-step protocol for gene replacement using a selection-counterselection cassette
This protocol is useful if you want to replace a gene X with another gene Y. The example used here will detail the replacement of the Citrobacter freundii cephalosporinase gene with the limonene synthase gene, what makes our cells less antibiotic resistant and more citrus-scented (Figure 1). The idea was pioneered by Edinburgh's 2010 iGEM team - you can read more about their BRIDGE (BioBrick Recombination In Direct Genomic Editing) project here
Figure 1: Two-step gene replacement strategy, simplified schematic.
The first step in the process is creating homology arms by PCR for the selection-counterselection cassette that are homologous to the regions flanking the cephalosporinase gene. The arm upstream of the cassette should have an EcoRI site at its 5’ end and the arm downstream of the cassette should have a PstI site at its 3’ end.
The upstream arm can then be digested with EcoRI and the downstream arm with PstI and these can then be ligated to the selection-counterselection cassette that was digested with both enzymes.
A PCR using the outer primer pair should next be done to generate lots of a single linear product containing all three components (the upstream arm + cassette + downstream arm).
Cells should first be transformed with a lambda red plasmid such as pSC101-gbaA (commercially available from GeneBridges http://www.genebridges.com/). This plasmid allows the cell to take up linear pieces of DNA without degrading them.
They can then be transformed with this linear piece of DNA and hopefully some of them will undergo homologous recombination, cutting out the cephalosporinase gene and replacing it with the selection-counterselection cassette. After this step, the cells should be plated onto a kanamycin-containing medium to select for the cells that have taken up the cassette.
The second step involves the replacement of this cassette with the limonene synthase gene – the cells that have grown on the kanamycin plate should be transformed with single stranded DNA containing the appropriate upstream arm + limonene synthase (or any other BioBrick) + downstream arm.  Again, in some of the cells homologous recombination will cause the excision of the cassette and insertion of the limonene synthase gene into the vacated region.
This time, in order to select for the cells that have lost the cassette ( counter-select for the cells that have the cassette), they should be plated out onto media that contain sucrose – cells that can grow on this medium (and are not red) have lost the levansucrase gene, and so the cassette, and are therefore good candidates for having had the cephalosporinase gene replaced by the limonene synthase gene.
Finally, grow the cells at a non-permissive temperature to remove the lambda red plasmid and enjoy your lemon-scented cells!
<<Prev __6/6__Next>>
Methods (expand)
Inserting gene into a biobrick vecor: Cloning a PCR product into a biobrick vector protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:bbcloning) however NEB buffers were used.
DNA gel preparation: Analysing DNA by gel electrophoresis protocol on OpanWetWare (http://openwetware.org/wiki/Cfrench:AGE) however 0.5*TAE rather than 1*TAE was used.
Colony PCR screen: Screening colonies by PCR protocol on OpenWetWare http://openwetware.org/wiki/Cfrench:PCRScreening
Transformations: Preparing and using compenent E.coli cells protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:compcellprep1)
PCR reactions : Cloning parts by PCR with Kod polymerase protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:KodPCR)
Minipreps : Plasmid DNA minipreps from Escerichia coli JM109 and similar strains protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:minipreps1)
Digests to linearise the DNA frangment/determine size of insert: Analytical restriction digests protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:restriction1)
DNA purification: Purifying a PCR product from solution protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:DNAPurification1) however 165 ul NaI, 5 ul glass beads,180 ul wash buffer and 10 ul EB were used.
DNA preparation for sequencing: 2.5 ul miniprepped DNA, 2 ul water and 1 ul forward primer ( specific for biobrick prefix) or reverse primer (specific for biobrick suffix)
Nitroreductase activity assay: Overnight liquid cultures of nitroreductase strains were centrifuged at 10000 rpm for 5 mins to pellet the cells. The cells were then resuspended in 250 ul PBS and 1 ul DTT to ensure that cellular proteins are not oxidized. The solution was sonicated 6* (10 s sonication+20 s rest). The supernatant was separated from the pellet by centrifugation and used for the NADH-dependent nitroreductase activity assay.
To assess background activity NADH (5 ul) and bacterial supernatant (5 ul) were added to 0.8 ml PBS and mixed. OD340 was measured for 1 minute. DNBA(5 ul) was added to the same cuvette to start the reaction and change in OD340 was monitored for 1 minute. DMSO(5 ul) was used a control (DNBA is dissolved in DMSO)
The protein concentration of each of the supernants was estimated by by Bradford protein assay using the Pierce reagent protocol on OpenWetWare(http://openwetware.org/wiki/Cfrench:ProteinAssay)
Close methods.
Works Cited (expand)
French, C., & Kowal, M. (2010, 09 24). B. subtilis levansucrase. Lethal to E.coli in presence of sucrose. Retrieved 2012, from Registry of standard biological parts: http://partsregistry.org/Part:BBa_K322921
Gay, P., Coq, D. l., Strinmetz, M., Ferrari, E., & Hoch, J. A. (1983). Cloning Structural Gene SacB, which Codes for Exoenzyme Levansucrase of Bacillus subtilis: Expression of the Gene in Esherichia coli. Journal of Bacteriology , 1424-1431.
Jahreis, K., Bentler, L., Bockmann, J., Hans, S., Meyer, A., Siepelmeyer, J., et al. (2002). Adaptation of sucrose metabolism in the Escherichia coli Wild-Type Strain EC31132. Journal of Bacteriology, 5307-5316.
Keuning, S., Janssen, D. B., & Witholt, B. (1985). Purification and Characterisation of Hyrdrolytic Haloalkane Dehalogenase from Xanthobacter autotrophicus GJ10. Journal of Bacteriology, 635-639.
Naested, H., Fennema, M., Hao, L., Andersen, M., Janssen, D. B., & Mundy, J. (1999). A bacterial haloalkane dehalogenase gene as a negative selectable marker in Arabidopsis. The Plant Journal, 571-576.
Nicklin, C. E., & Bruce, N. C. (1998). Aerobic degradation of 2,4,6-Trinitrotoluene by Enterobacter cloaceae PB2 and by Pentaerythritol tetranitrate reductase. Applied and environmental microbiology , 2864-2868.
Nillius, D., Muller, J., & Muller, N. (2011). Nitroreductase (GlNR1) increases susceptibility of Giardia lamblia and Escherichia coli to nitro drugs. Journal of antimicrobial chemotherapy, 1029-1035.
Kang et al. (2009). "Levan: Applications and Perspectives". Microbial Production of Biopolymers and Polymer Precursors. Caister Academic Press
Dahech, I, Belghith, K. S., Hamden, K., Feki, A., Belghith, H. and Mejdoub, H. (2011) Antidiabetic activity of levan polysaccharide in alloxan-induced diabetic rats. International Journal of Biological Macromolecules 49(4):742-746
Close cited works.
Retrieved from " http://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Gene-replacement-strategy "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
