<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Project_Bioelectric-Interface_Bio-electric-Interface-BioBricks-Cloning"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/Project/Bioelectric-Interface/Bio-electric-Interface-BioBricks-Cloning">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Project/Bioelectric-Interface/Bio-electric-Interface-BioBricks-Cloning&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Project/Bioelectric-Interface/Bio-electric-Interface-BioBricks-Cloning&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Project/Bioelectric-Interface/Bio-electric-Interface-BioBricks-Cloning&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Project/Bioelectric-Interface/Bio-electric-Interface-BioBricks-Cloning" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Project/Bioelectric-Interface/Bio-electric-Interface-BioBricks-Cloning</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="header"><DIV id="banner"><DIV id="follow"><UL><LI><SPAN class="plainlinks"><A class="follow-links" id="gmail" href="mailto:EdiGEM12@gmail.com"><P class="link-details">Gmail</P></A></SPAN></LI><LI><SPAN class="plainlinks"><A class="follow-links" id="gplus" href="https://plus.google.com/104634974040525950216/posts"><P class="link-details">Google+</P></A></SPAN></LI><LI><SPAN class="plainlinks"><A class="follow-links" id="twitter" href="https://twitter.com/#!/EdiGEM12"><P class="link-details">Twitter</P></A></SPAN></LI><LI><SPAN class="plainlinks"><A class="follow-links" id="facebook" href="http://www.facebook.com/Edigem2012"><P class="link-details">Facebook</P></A></SPAN></LI><LI><SPAN class="plainlinks"><A class="follow-links" id="blogger" href="http://synthbiopunk.blogspot.com/"><P class="link-details">Blogger</P></A></SPAN></LI></UL></DIV></DIV><DIV id="main-menu"><UL><LI id="home-page"><UL><A href="https://2012.igem.org/Team:Edinburgh" onmouseover="blind_down('home-page');" onmouseout="blind_up('home-page');"><P>Home</P></A></UL></LI><LI class="tab-description" id="home-page-tab-description" style="display:none;"><P>Welcome!</P></LI></UL><LI id="team-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Team" onmouseover="blind_down('team-page');" onmouseout="blind_up('team-page');"><P>Team</P></A></UL></LI><LI class="tab-description" id="team-page-tab-description" style="display:none;"><P>Meet the team!</P></LI><LI id="profile-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Profile" id="edinburgh-profile" onmouseover="blind_down('profile-page');" onmouseout="blind_up('profile-page');"><P>Profile</P></A></UL></LI><LI id="project-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Project" onmouseover="blind_down('project-page');" onmouseout="blind_up('project-page');"><P>Project</P></A></UL></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface">
							Bio-electric Interface </A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers">
							Alternative selectable and counter-selectable markers</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/0-Introduction"><I>Citrobacter freundii</I></A></LI><LI id="parts-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Parts" onmouseover="blind_down('parts-page');" onmouseout="blind_up('parts-page');"><P>Parts</P></A></UL></LI><LI id="modelling-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Modeling" onmouseover="blind_down('modeling-page');" onmouseout="blind_up('modeling-page');"><P>Modelling</P></A></UL></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Modelling/Kappa/Introduction">
							Kappa</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Modelling/Biosensors/Introduction">
							Biosensors</A></LI><LI id="human-practices-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices" onmouseover="blind_down('human-pr-page');" onmouseout="blind_up('human-pr-page');"><P>Human Practices</P></A></UL></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Crowdfunding">Crowdfunding</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/DIYbio">DIY Bio</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Perceptions-of-the-Public">Perceptions of the Public</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Blog">Blog</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Laws-and-Legislations">Laws and Legislations</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Friendly">Questioning Friendliness</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Failure">Failure</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Blue-Sky_Research">Blue-Sky Research</A></LI><LI id="safety-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Safety" onmouseover="blind_down('safety-page');" onmouseout="blind_up('safety-page');"><P>Safety</P></A></UL></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Safety#question1">
							Question 1</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Safety#question2">
							Question 2</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Safety#question3">
							Question 3</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Safety#question4">
							Question 4</A></LI><LI id="notebook-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Notebook" onmouseover="blind_down('notebook-page');" onmouseout="blind_up('notebook-page');"><P>Notebook</P></A></UL></LI><LI id="attributions-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Attributions" onmouseover="blind_down('attributions-page');" onmouseout="blind_up('attributions-page');"><P>Attributions</P></A></UL></LI></DIV></DIV><DIV id="page-middle"><DIV id="page-navigation"><UL><LI id="nav-bio-el-interface"><P>Bio-electric Interface</P></LI><LI id="nav-bio-el-background"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface">
					Background</A></LI><LI id="nav-bio-el-cloning"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface/Bio-electric-Interface-BioBricks-Cloning">
					BioBrick Cloning and Characterisation</A></LI><LI id="nav-bio-el-fuel-cells"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface/Microbial-Half-Fuel-Cells">
					Microbial Half Fuel Cells</A></LI><LI id="nav-bio-el-discussion"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface/Discussion">
					Discussion</A></LI></UL><LI id="nav-markers"><P>Alternative Selectable and Counter-selectable Markers</P></LI><LI id="nav-markers-introduction"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers">
					Introduction</A></LI><LI id="nav-markers-nitroreductase"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Nitroreductase">
					Nitroreductase</A></LI><LI id="nav-markers-sucrose-hydrolase"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Sucrose-Hydrolase">
					Sucrose Hydrolase</A></LI><LI id="nav-markers-dhla"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/DhlA">
					Haloalkane Dehalogenase</A></LI><LI id="nav-markers-Plac-RFP-SacB"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Plac-RFP-SacB">
					Levansucrase</A></LI><LI id="nav-markers-gene-rep"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Gene-replacement-strategy ">
					Gene ReplacementStrategy </A></LI><LI id="nav-lemon"><P><I>Citrobacter freundii</I></P></LI><LI id="nav-lemon-0"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/0-Introduction">
					Introduction</A></LI><LI id="nav-lemon-1"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/1-Replicon-compatibility">
					Replicon compatibility</A></LI><LI id="nav-lemon-2"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/2-Growth-in-sea-salts">
					Growth in sea salts</A></LI><LI id="nav-lemon-3"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/3-Sugar-use">
					Sugar use</A></LI><LI id="nav-lemon-4"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/4-Lac-promoter">
					Lac promoter</A></LI><LI id="nav-lemon-5"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/5-Genome-sequencing">
					Genome sequencing</A></LI><LI id="nav-lemon-6"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/6-Valencia">
					Valencia BiocampusBioBrick Characterisation</A></LI></DIV><DIV id="page-content"><DIV class="text"><P class="h1">
Bio-electric Interface:

BioBrick Cloning and Characterisation(Methods and Results)
			</P><P class="h2">
Methods
			</P><P class="normal-text">
- Bacteria used: <I>Escherichia coli</I> JM109 and <I>Shewanella oneidensis</I> MR-1. Both organisms were obtained from cultures in Chris French’s lab at the University of Edinburgh. <I>S. oneidensis</I> cultures were grown on LB agar at room temperature not exceeding 30°C. Plates were subcultured each week. <I>E. coli</I> cultures were grown on LB agar in a 37 degree incubator overnight and then left to grow at room temperature and subcultured by lab staff when needed.

- PCR: most PCR reactions were performed using the following OpenWetWare protocol CFrench: <A href="http://openwetware.org/wiki/Cfrench:KodPCR">KodPCR</A>. Optimal annealing temperature for <I>S. oneidensis</I> genes was found to be around 50-52°C while <I>E. coli</I> genes showed good results with annealing temperatures in the range of 50-55°C. <I>S. oneidensis</I> cell suspension in sterile water was used as template for <I>mtrA, MtrCAB, S. oneidensis ccmA-E</I> and <I>ccmF-H</I> genes. <I>E. coli</I> cell suspension in sterile water was used as template for <I>napC </I> and <I>E. coli</I><I>ccmA-H</I> genes.

- PolyA tailing: for several genes polyA tailing was performed using Taq polymerase and the following protocol: 20 minutes denaturation at 95°C, followed by addition of Taq polymerase, followed by 15 minutes extension at 72°C.

- Gel electrophoresis: gel analysis was used following OpenWetWare CFrench: <A href="http://openwetware.org/wiki/Cfrench:AGE">AGE protocol</A> except 0,5 TAE buffer was used rather that 1x TAE. Gel staining was done using the SYBR-Safe staining solution.

- Gel purification and DNA purification: for <I>ccm</I> and several ligation attempts for other genes the PCR samples were run on the gel then the appropriate bands were cut out and purified using standardised QIAquick Gel Extraction Kits. For pure PCR products OpenWetWare protocol CFrench: <A href="http://openwetware.org/wiki/Cfrench:DNAPurification1">DNAPurification1</A> was used.

- Vectors used: For most reaction the standard BioBrick vector pSB1C3 (provided by the Registry) was used, except for samples that were subjected to polyA tailing which were then ligated into the pGEM vector (Promega)

- Restriction digestions: Restriction digests were performed for PCR products along with vector digestion following the OpenWetWare <A href="http://openwetware.org/wiki/Cfrench:restriction1">CFrench:restriction1 protocol</A>. For enhanced efficiency, varying ratios of insert to vector were used with the optimum ratio reached at about 3:1 to 5:1 ratio of insert digest to vector digest. Analytical restriction digests were also performed for miniprep samples using the original protocol.

- Ligations: Digested samples were mixed with 1 ul T4 ligase buffer and 1 ul T4 ligase and mixed with water to reach the final volume of 20 ul if necessary. Alternatively, polyA tailed PCR sampels were mixed with pGEM vector and used directly for ligation.

- Fusion PCR: following the ligation the samples were used as template for fusion PCR, following KodPCR protocol using forward primer for the gene and reverse primer for the vector. Extension time was adjusted to suit the length of the vector with insert.

- Transformation: Ligation and fusion PCR products were used to transform <I>E coli</I> JM109 competent cells using the OpenWetWare protocol <A href="http://openwetware.org/wiki/Cfrench:compcellprep1">Cfrench:compcellprep1</A> for preparation of competent cells and cell transformation).

- Transformed cell selection: Transformed cells were spread on LB agar with chloramphenicol (when using pSB1C3 vector) or LB agar with carbenicillin, Xgal and IPTG. Following overnight incubation at 37°C white colonies were chosen (rather than red colonies from pSB1C3-RFP vector or blue colonies from pGEM vector) and subcultured on plates containing the same medium.

- Miniprepping: Subcultures were used to set up overnight liquid cultures in 2,5 ml of LB. Miniprepping was performed using either the OpenWetWare protocol <A href="http://openwetware.org/wiki/Cfrench:minipreps1">Cfrench:minipreps1</A> or standarised QIAprep Spin MiniPrep Kit. Minipreps were then restriction digested and run on a gel.

- Sequencing: size confirmed minipreps were then sent for sequencing at the University of Edinburgh GenePool.
			</P><P class="h2">
Results
			</P><P class="normal-text"><B>NapC</B>
- Throughout the summer we have managed to clone the <I>napC</I> gene from <I>E coli</I> (<A href="http://partsregistry.org/Part:BBa_K917003">BBa_K917003</A>). We have inserted the gene into the standard BioBrick vector pSB1C3 and submitted it to the parts Registry. We have then linked the <I>napC</I> gene to the lac promoter (<A href="http://partsregistry.org/Part:BBa_K917012">BBa_K917012</A>) to characterise its functionality.<B>Figure 1:</B><I>napC</I> PCR
<B>Figure 2:</B><I>napC</I> miniprep
<B>Figure 3:</B> pLac-napC construct analytical digestion with XbaI and PstI 
Lanes 3, 4 = pSB1C3-Plac-lacZ'-napC, clones 1 and 2, digested with XbaI-PstI. Clone 2 looks as expected, clone 1 has an unexpected band around 0.6 kb. 
<B>MtrA</B>
- We also managed to obtain the <I>mtrA</I> gene (<A href="http://partsregistry.org/Part:BBa_K917008">BBa_K917008</A>) of <I>S. oneidensis</I> and cloned it into the pSB1C3 plasmid.<B>Figure 4:</B> MtrA PCR
<B>Figure 5:</B><I>mtrA</I> transformation
<B>Figure 6:</B><I>mtrA</I> miniprep

The obtained <I>mtrA</I> gene contains an internal PstI site which needs to be mutated out prior to submission and use.
<B>CymA</B>
We have managed to clone the <I>cymA</I> gene (<A href="http://partsregistry.org/Part:BBa_K917009">BBa_K917009</A>) from <I>S. oneidensis</I>. We have tested the gene for internal restriction sites and linked the <I>cymA</I> gene to the lac promoter (<A href="http://partsregistry.org/Part:BBa_K917014">BBa_K917014</A>) to characterise its functionality.<B>Figure 7:</B> lanes 3, 4 = pSB1C3-cymA clones 1 and 2, analytically digested with EcoRI. Band has size correct for linearised plasmid with the gene(3kb) 
lanes 5, 6 = pSB1C3-cymA clones 1 and 2, double digested with EcoRI/SpeI. <B>Figure 8:</B> pSB1C3-cymA clones 1 and 2, testing internal restriction sites.  
Lanes 1, 2 = clones 1 and 2, NdeI.
Lanes 3, 4 = clone 1, XbaI and XbaI/HindIII.  
Lanes 5, 6 =  clone 2, XbaI and XbaI/HindIII. 
Gel results appear as expected <B>Figure 9:</B> Lanes 4 to 6, pSB1C3-Plac-lacZ'-cymA clones 1-3, analytically digested with XbaI-PStI. 
Clones 1 and 2 show bands of appropriate sizes. <B><I>ccm</I> cytochrome maturation cluster of <I>E. coli</I></B>
We have cloned the <I>E. coli</I><I>ccm</I> gene cluster (<A href="http://partsregistry.org/Part:BBa_K917006">BBa_K917006</A>), analysed its internal restriction sites and linked it with the lac promoter (<A href="http://partsregistry.org/Part:BBa_K917013">BBa_K917013</A>).<B>Figure 10:</B><I>E coli</I><I>ccm</I> genes PCR (right lanes)<B>Figure 11:</B> pSB1C3-ccm clones 1-6, digested with EcoRI. 
Clones 1 and 5 show bands of expected size, the other clones are too small <B>Figure 12:</B> pSB1C3-ccm clones 1 and 5, testing internal restriction sites. 
Lanes 1, 2 = EcoRI/SpeI digestion. 
Lanes 3, 4 = BamHI digestion. 
Lanes 5, 6 = ClaI digestion. 
Results appear as expected assuming 2 of the 3 ClaI sites are uncuttable due to overlapping dam methylation <B>Figure 13:</B> Lanes 1, 2 = pSB1C3-Plac-lacZ'-ccm, clones 1 and 2, digested with XbaI-PstI.  
Clone 2 looks as expected, clone 1 has an unexpected band around 0.6 kb. <B>MtrCAB and <I>S. oneidensis ccm</I></B>
We have also obtained good quality pure PCR products of <I>mtrCAB</I> and <I>ccm</I> genes from <I>S. oneidensis</I><B>Figure 14:</B> MtrCAB PCR
<B>Figure 15:</B><I>ccm</I> genes from <I>S. oneidensis</I> and <I>E. coli</I></P><P class="h2">
BioBrick characterisation </P><P class="normal-text"><I>ccm, napC and cymA</I>
We have tested the expression of NapC and CymA proteins in <I>ccm</I> transformed cells (<I>ccm</I> gene cluster BioBrick was transferred into pSB4K5 vector to allow for double epxression of <I>ccm</I> and <I>cymA/napC</I> genes). Cell pellets were scanned prior to sonication and <I>cymA</I> and <I>napC</I> transformed cells show slightly more intense colour, indicating higher concentration of haem in cells. 
Following the sonication protein samples were run on a gel and were stained for haem using the following protocol: 
The NuPAGE® gel was first soaked in 100 ml solution-I 
(30 ml methanol, 70 ml 250 mM NaAc, pH=5.2) for 10 min, and then was moved into 50 ml 
solution-II (15 mg 3,3',5,5'-Tetramethylbenzidine (TMBZ), 15 ml methanol and 35 ml 250 mM NaAc, pH=5.2; TMBZ was prepared 
by dissolving the powder in methanol and placed in the dark) in an opaque box to prevent 
the light. The box was incubated for 25 min at room temperature. 150 ul 30% (w/w) H2O2 
solution was then added into the box which was further incubated in the dark for 3~5 min 
after gentle mixing. The stained bands were fixed by washing the gel with dH2O.

Haem staining resulted in clear bands forming in all samples, probably representing one of the Ccm proteins, most likely CcmE (haem chaperone protein). In <I>napC</I> and <I>cymA</I> samples additional bands were present, indicated by the arrows.
Haem staining: lane 1: pure cytochrome c <B>control</B>, 
lane 2: protein extract from <I><B>ccm</B></I> transformed cells, membrane fraction; 
lane 3: protein extract from <I><B>ccm</B></I><B> and <I>cymA</I></B> transformed cells, membrane fraction; 
lane 4: protein extract from <I><B>ccm</B></I><B> and <I>napC</I></B> transformed cells, membrane fraction; 
lane 5: protein extract from <I><B>ccm</B></I> transformed cells, cytosol fraction; 
lane 6: protein extract from <I><B>ccm</B></I><B> and <I>cymA</I></B> transformed cells, cytosol fraction; 
lane 7: protein extract from <I><B>ccm</B></I><B> and <I>napC</I></B> transformed cells, cytosol fraction
 
These results show that expression of CymA and NapC is successful. Interestingly NapC predominates in the soluble fraction and CymA appears in the membrane fraction. This is an unexpected result as both proteins are intermembrane proteins. We hope to inspect this fact closer to determine its significance in our system. 
			</P><P class="normal-text" style="text-align:center"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface"><SPAN class="intense-emphasis">&lt;&lt;Prev</SPAN></A><SPAN style="color:white;">__</SPAN>2/4<SPAN style="color:white;">__</SPAN><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface/Microbial-Half-Fuel-Cells"><SPAN class="intense-emphasis">Next&gt;&gt;</SPAN></A></P><P class="h2"><A name="bibliography">Bibliography</A><A class="cursor-pointer" onclick="expand('works-cited')">(expand)</A></P><P id="works-cited" class="normal-text"><B>1. Jensen, H. M., Albers, A. E., Malley, K. R., Londer, Y. Y. , Cohen, B. E., Helms, B. A., Weigele, P., Groves, J. T. &amp; Ajo-Franklin, C. M.</B> (2010). <I>Engineering of a synthetic electron conduit in living cells.</I> PNAS  107, 19213-19218
<B>2. Stewart, V., Lu, Y. &amp; Darwin, A. J.</B> (2002). <I>Periplasmic Nitrate Reductase (NapABC Enzyme) supports Anaerobic Respiration by Escherichia coli K-12.</I> Journal of Bacteriology 184, 1314-1323
<B>3. Marritt, S. J., Lowe, T. G., Bye, J., McMillan, D.G.G., Shi, L., Frederickson, J., Zachara, J., Richardson, D. J., Cheesman, M. R., Jeuken L.J.C. &amp; Butt, J. N.</B> (2012). <I>A functional description of CymA, an electron-transfer hub supporting anaerobic respiratory flexibility in Shewanella.</I> Biochemical Journal 444, 465-474
<B>4. Richter, K., Schicklberger, M., Gescher, J.</B> (2011). <I>Dissimilatory reduction of extracellular electron acceptors in anaerobic respiration.</I> Applied and Environmental Microbiology 78, 913-921
<A class="cursor-pointer" onclick="collapse('works-cited')">Close bibliography.</A></P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface/Bio-electric-Interface-BioBricks-Cloning">http://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface/Bio-electric-Interface-BioBricks-Cloning</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Project/Bioelectric-Interface/Bio-electric-Interface-BioBricks-Cloning" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Project/Bioelectric-Interface/Bio-electric-Interface-BioBricks-Cloning" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Project/Bioelectric-Interface/Bio-electric-Interface-BioBricks-Cloning&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Project/Bioelectric-Interface/Bio-electric-Interface-BioBricks-Cloning&amp;oldid=288765" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>