<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Project_Non-antibiotic-Markers_Nitroreductase"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/Project/Non-antibiotic-Markers/Nitroreductase">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Project/Non-antibiotic-Markers/Nitroreductase&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Project/Non-antibiotic-Markers/Nitroreductase&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Project/Non-antibiotic-Markers/Nitroreductase&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Project/Non-antibiotic-Markers/Nitroreductase" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Project/Non-antibiotic-Markers/Nitroreductase</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="header"><DIV id="banner"><DIV id="follow"><UL><LI><SPAN class="plainlinks"><A class="follow-links" id="gmail" href="mailto:EdiGEM12@gmail.com"><P class="link-details">Gmail</P></A></SPAN></LI><LI><SPAN class="plainlinks"><A class="follow-links" id="gplus" href="https://plus.google.com/104634974040525950216/posts"><P class="link-details">Google+</P></A></SPAN></LI><LI><SPAN class="plainlinks"><A class="follow-links" id="twitter" href="https://twitter.com/#!/EdiGEM12"><P class="link-details">Twitter</P></A></SPAN></LI><LI><SPAN class="plainlinks"><A class="follow-links" id="facebook" href="http://www.facebook.com/Edigem2012"><P class="link-details">Facebook</P></A></SPAN></LI><LI><SPAN class="plainlinks"><A class="follow-links" id="blogger" href="http://synthbiopunk.blogspot.com/"><P class="link-details">Blogger</P></A></SPAN></LI></UL></DIV></DIV><DIV id="main-menu"><UL><LI id="home-page"><UL><A href="https://2012.igem.org/Team:Edinburgh" onmouseover="blind_down('home-page');" onmouseout="blind_up('home-page');"><P>Home</P></A></UL></LI><LI class="tab-description" id="home-page-tab-description" style="display:none;"><P>Welcome!</P></LI></UL><LI id="team-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Team" onmouseover="blind_down('team-page');" onmouseout="blind_up('team-page');"><P>Team</P></A></UL></LI><LI class="tab-description" id="team-page-tab-description" style="display:none;"><P>Meet the team!</P></LI><LI id="profile-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Profile" id="edinburgh-profile" onmouseover="blind_down('profile-page');" onmouseout="blind_up('profile-page');"><P>Profile</P></A></UL></LI><LI id="project-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Project" onmouseover="blind_down('project-page');" onmouseout="blind_up('project-page');"><P>Project</P></A></UL></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface">
							Bio-electric Interface </A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers">
							Alternative selectable and counter-selectable markers</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/0-Introduction"><I>Citrobacter freundii</I></A></LI><LI id="parts-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Parts" onmouseover="blind_down('parts-page');" onmouseout="blind_up('parts-page');"><P>Parts</P></A></UL></LI><LI id="modelling-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Modeling" onmouseover="blind_down('modeling-page');" onmouseout="blind_up('modeling-page');"><P>Modelling</P></A></UL></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Modelling/Kappa/Introduction">
							Kappa</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Modelling/Biosensors/Introduction">
							Biosensors</A></LI><LI id="human-practices-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices" onmouseover="blind_down('human-pr-page');" onmouseout="blind_up('human-pr-page');"><P>Human Practices</P></A></UL></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Crowdfunding">Crowdfunding</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/DIYbio">DIY Bio</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Perceptions-of-the-Public">Perceptions of the Public</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Blog">Blog</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Laws-and-Legislations">Laws and Legislations</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Friendly">Questioning Friendliness</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Failure">Failure</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Blue-Sky_Research">Blue-Sky Research</A></LI><LI id="safety-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Safety" onmouseover="blind_down('safety-page');" onmouseout="blind_up('safety-page');"><P>Safety</P></A></UL></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Safety#question1">
							Question 1</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Safety#question2">
							Question 2</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Safety#question3">
							Question 3</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Safety#question4">
							Question 4</A></LI><LI id="notebook-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Notebook" onmouseover="blind_down('notebook-page');" onmouseout="blind_up('notebook-page');"><P>Notebook</P></A></UL></LI><LI id="attributions-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Attributions" onmouseover="blind_down('attributions-page');" onmouseout="blind_up('attributions-page');"><P>Attributions</P></A></UL></LI></DIV></DIV><DIV id="page-middle"><DIV id="page-navigation"><UL><LI id="nav-bio-el-interface"><P>Bio-electric Interface</P></LI><LI id="nav-bio-el-background"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface">
					Background</A></LI><LI id="nav-bio-el-cloning"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface/Bio-electric-Interface-BioBricks-Cloning">
					BioBrick Cloning and Characterisation</A></LI><LI id="nav-bio-el-fuel-cells"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface/Microbial-Half-Fuel-Cells">
					Microbial Half Fuel Cells</A></LI><LI id="nav-bio-el-discussion"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface/Discussion">
					Discussion</A></LI></UL><LI id="nav-markers"><P>Alternative Selectable and Counter-selectable Markers</P></LI><LI id="nav-markers-introduction"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers">
					Introduction</A></LI><LI id="nav-markers-nitroreductase"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Nitroreductase">
					Nitroreductase</A></LI><LI id="nav-markers-sucrose-hydrolase"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Sucrose-Hydrolase">
					Sucrose Hydrolase</A></LI><LI id="nav-markers-dhla"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/DhlA">
					Haloalkane Dehalogenase</A></LI><LI id="nav-markers-Plac-RFP-SacB"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Plac-RFP-SacB">
					Levansucrase</A></LI><LI id="nav-markers-gene-rep"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Gene-replacement-strategy ">
					Gene ReplacementStrategy </A></LI><LI id="nav-lemon"><P><I>Citrobacter freundii</I></P></LI><LI id="nav-lemon-0"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/0-Introduction">
					Introduction</A></LI><LI id="nav-lemon-1"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/1-Replicon-compatibility">
					Replicon compatibility</A></LI><LI id="nav-lemon-2"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/2-Growth-in-sea-salts">
					Growth in sea salts</A></LI><LI id="nav-lemon-3"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/3-Sugar-use">
					Sugar use</A></LI><LI id="nav-lemon-4"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/4-Lac-promoter">
					Lac promoter</A></LI><LI id="nav-lemon-5"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/5-Genome-sequencing">
					Genome sequencing</A></LI><LI id="nav-lemon-6"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/6-Valencia">
					Valencia BiocampusBioBrick Characterisation</A></LI></DIV><DIV id="page-content"><DIV class="text"><P class="h1">
Alternative selectable and counter-selectable markers:

Nitroreductase (<I>nfsI</I>)
			</P><P class="h2">
Background
			</P><P class="normal-text">
Nitroreductase is an <I>Enterobacter cloacae</I> enzyme which reduces nitrogen containing compounds  <A href="#bibliography" onclick="expand('works-cited');">(Nicklin &amp; Bruce, 1998)</A>. Other nitroreductases were found to convert nitro drugs such as metronidazole into their active forms, which is an essential part of their toxicity <A href="#bibliography" onclick="expand('works-cited');">(Nillius, Muller, &amp; Muller, 2011)</A>. Bearing this in mind, we decided to look into nitroreductase's potential as a counter-selectable marker.
			</P><P class="h2">
Cloning
			</P><P class="h3">
pSB1C3-nitroreductase (<A href="http://partsregistry.org/Part:BBa_K917004">BBa_K917004</A>)
			</P><P class="normal-text">
The <I>nfsI</I> gene was cloned using these <A class="cursor-pointer" onclick="expand('pSB1C3-primers');">primers</A> and inserted into the standard BioBrick vector pSB1C3. This construct was confirmed through <A class="cursor-pointer" onclick="expand('pSB1C3-seq');">sequencing</A>.
<A class="cursor-pointer" onclick="expand('pSB1C3-method')">Method</A>.
			</P><P class="normal-text" id="pSB1C3-primers"><I>Forward primer: GCTA gaattcgcggccgcttctagag caccagg agttgtt atg gat
Reverse primer: CATG ctgcag cggccgc t actagt a tta tt AGCACTCGG TCACAATCGT</I><A class="cursor-pointer" onclick="collapse('pSB1C3-primers');">Close the primers.</A></P><P class="normal-text" id="pSB1C3-seq"><I>Sequencing results:
aacttataaatattcttaggcttatctctagggaggatttctggaattcgcggccgcttctagagcaccaggagttgttctggatatgatttctgtcgccctgaaacggcactccaccaaggcgttcgaccc
cgctaaaaaactgaccgcatacgatccggaaaagatcaaacccctgctgcaataccgtccgtccaacaccctgtcccagccgtggcactttattgtccttgcaccgaggaaggtaaaccttgcgtggtttcc
tctgccgaaagcacttacgtcttctacgatcgcaaaacgctggacgcttctctcgtggtggtgttctgcgcgaaaaccgcttcggatgatgccttcatggaacgcttggtggatcatgaagaacccgatggc
cggt</I><A class="cursor-pointer" onclick="collapse('pSB1C3-seq');">Close the sequencing results.</A></P><P class="normal-text" id="pSB1C3-method"><I>Method: The nitroreductase PCR product was purified and digested with EcoRI HF and SpeI together with pSB1C3.  These were ligated, E.coli cells transformed with the ligation and the white colonies (RFP  disruption) were miniprepped. Detailed methods can be found in methods section.</I><B>Figure 1:</B> DNA gel of PCR product of BS-nitred with primers specific for nitroreductase. The product is around 0.6-0.7 kb which corresponds to the size of nitroreductase gene, around 0.6 kb.<B>Figure 2:</B> DNA gel of pSB1C3-nitroreductase ligation. The band is around 2.5-2.6 kb which corresponds to the vector pSB1C3 (around 2 kb) together with the nitroreductase (0.6 kb). Sample 2 was confirmed with sequencing.<A class="cursor-pointer" onclick="collapse('pSB1C3-method')">Close the method.</A></P><P class="h3">Plac-lacZ'-nitroreductase (<A href="http://partsregistry.org/Part:BBa_K917005">BBa_K917005</A>)
			</P><P class="normal-text">
A promoter and a reporter gene were then added in front of the nitroreductase gene (Plac-lacZ').
<A class="cursor-pointer" onclick="expand('Plac-lacZ-method')">Method</A>.
			</P><P class="normal-text" id="Plac-lacZ-method"><I>Method: The sequence confirmed  pSB1C3- nitroreductase was digested with EcoRI HF and XbaI while Edinbrick1 was digested with EcoRI HF and SpeI.  These were ligated together. The ligations were transformed into cells and the transformants plated on LB+chloramphenicol+IPTG+Xgal plate. The blue colonies (contain lacZ) were used for the following experiments.  Colony PCR screen of pooled blue Plac-lacZ'-nitroreductase transformants with lacZ forward primer and reverse nitroreductase primer showed a band corresponding to lacZ-nitroreductase.</I><I><B>Figure 3:</B> DNA gel with Colony PCR products of pooled blue Plac-lacZ'-nitroreductase transformants with lacZ forward primer and reverse nitroreductase primer resulted in in bands around 1.2-1.3 kb which correspond  to nitroreductase (0.6 kb) plus lacZ (0.6 kb).

To confirm the presence of Plac-lacZ'-nitroreductase in pSB1C3, the samples in the smallest pool were minipreped, digested with EcoRI and SpeI to check the size of the insert.
</I><I><B>Figure 4:</B> DNA gel with Plac-lacZ'-nitroreductase which was digested with EcoRI HF and SpeI. The biggest band is likely to correspond to pSB1C3 around 2.2-2.3 kb, the middle band is likely to correspond to  Plac-lacZ'-nitroreductase around 1.5 kb and the smallest fragment is unknown.</I><I><B>Figure 5:</B> DNA gel with Plac-lacZ'-nitroreductase which was digested with EcoRI HF to linearise the DNA. There are two distinctive bands, one around 3.0 kb and one around 3.6 kb likely to correspond to pSB1C3 with Plac-lacZ'-nitroreductase and Plac-lacZ'.</I>
This DNA was further purified to give a single plasmid corresponding to Plac-lacZ'-nitroreductase. <B>Figure 6:</B> DNA gel of Plac-lacZ'-nitroreductase digested with XbaI and PstI. The band around 1.2 kb corresponds to the Plac-lacZ'-nitroreductase fragment while the band at 2 kb corresponds to the vector. The band just above 3 kb is likely to be the undigested plasmid.

<A class="cursor-pointer" onclick="collapse('Plac-lacZ-method')">Close the method.</A></P><P class="h3">PstI restriction site <A class="cursor-pointer" onclick="expand('PstI');">(expand)</A></P><P class="normal-text" id="PstI"><I>The original <A href="http://www.ncbi.nlm.nih.gov/nuccore/M63808.1">sequence</A> used for primer design has a PstI restriction site, but our sequencing results suggests that there is no such site. The sequence confirmed pSB1C3-nitroreductase was digested with PstI and run alongside an undigested sample.</I><I><B>Figure 7:</B> DNA gel with pSB1C3-nitroreductase undigested and digested with PstI. Only one band at around 3 kb is visible corresponding to the linearized plasmid confirming that there is no PstI restriction site.</I><A class="cursor-pointer" onclick="collapse('PstI')">Close the method.</A></P><P class="h2">
Characterisation
			</P><P class="h3">
Specific activity- BS-nitred
			</P><P class="normal-text">
Before cloning the nitroreductase gene into the BioBrick vector, 3 different plasmids containing 3 nitroreductase genes with different promoters (and a control containing no nitroreductase gene) were used to test nitroreductase specific activity. The method is detailed in the methods section.

The change of NADH concentration was estimated by the change of OD340 absorbance per minute, background is subtracted and specific activity calculated. The results are presented in the diagram below. The experiment was done in triplicate. The control only had DMSO instead of DNBA substrate(which was used dissolved in DMSO) showed no change in absorbance (data not shown). <B>Figure 8:</B> Comparison of the specific activity of 3 nitroreductase genes in different vectors with different promoters and control. Error bars show the standard error of the mean. 

BS-nitred was used further for characterisation experiments as it showed the highest specific activity.

			</P><P class="h3">
Specific activity- Plac-lacZ'-nitroreductase in pSB1C3
			</P><P class="normal-text">
Specific activity was assessed in the BioBricked nitroreductase using the same method.The results are shown in the diagram below. The experiment was done in triplicate. <B>Figure 9:</B> Comparison of the specific activity of the BioBricked nitroreductase gene and control under induced and uninduced conditions (+ and - IPTG respectively). Error bars show the standard error of the mean. 

This graph shows that only the Plac-lacZ'-nitroreductase with IPTG induction shows nitroreductase activity. In addition, this activity is similar to the nitroreductase in the BlueScript vector(diagram above). </P><P class="h3"><A name="plates">Plates</A></P><P class="normal-text">
The following characterization results are produced from the pre-BioBrick form of nitroreductase (nitroreductase in BlueScript vector with lac promoter). Due to the similarity of the vectror, the identical regulation and very similar specific activity (previous section), we believe that the BioBricked Plac-lacZ'-nitroreductase will behave very similarly.
To determine the relative toxicity of different compounds, 5 ul of DMSO, MTZ and DNBA were added at three distinct spots on a freshly spread plate and the amount of clearing was measured (in centimeters).
DMSO was determined to be non-toxic, DNBA showed small difference between the different strains while MTZ distinctively more toxic to BS-nitred and BS-contol.

Numerous plate experiments with MTZ concentration ranging from 0 ug/ml to 300 ug/ml and various concentrations of DNBA and NFT were made to determine concentrations at which BS-control was growing but where BS-nitredâs growth is inhibited. Similar growth patterns were observed in DNBA and NFT plates. All metronidazole experiments showed inhibited growth of BS-nitred in comparison to BS-control however the inhibition was never 100 %, which is required for nitroreductase to be used as a counterselectable marker.<B>Figure 10:</B> Overnight plates with 100 ug/ml MTZ concentration with and without IPTG with different nitroreductase strains and control. BS-nitredâs growth was inhibited in comparison with BS-control however there are still some BS-nitred colonies growing.<B>Figure 11:</B> Comparison of growth of BS-contol and BS-nitred at 90 ug/ml metronidazole. BS-nitredâs growth is clearly inhibited in comparison to BS-control however growth inhibition is not absolute. 
We could not find a concentration of metronidazole at which nitroreductase containing cellsâ growth was inhibited while control cells were growing. We determined that this gene is not suitable as a counter-selectable marker on plates.
			</P><P class="h3">Liquid cultures
			</P><P class="normal-text">
The growth of nitroreductase-containing and control strains was assessed in liquid medium as well. The cells were grown in aerobic or anaerobic conditions with and without MTZ, in triplicate. <B>Figure 12:</B> Comparison of growth patterns of BS-nitred and BS-control in 150 ug/ml metronidazole in aerobic cultures. Initial OD600 was extracted and error bars are standard error of the mean.<B>Figure 13:</B> Comparison of growth patterns of BS-nitred and BS-control in 150 ug/ml metronidazole in anaerobic cultures. Initial OD600 was extracted and error bars are standard error of the mean.

The results in aerobic cultures are promising since nitroreductase-containing cells have not grown while the control cells are growing.
Once we obtained the BioBricked version of <I>nfsI</I>, we proceeded to testing this construct to show that it has similar activity to the gene in the BlueScript vector. Transformants and controls were incubated overnight in LB bottles containing either no, or 150ug/ml, metronidazole and OD readings were taken the following day. These results can be seen in Figure 14 below.
<B>Figure 14:</B> Comparison of growth between cells transformed with pSB1C3 plasmids containing either <I>nfsI</I> or control. 150 ug/ml metronidazole was added to bottles that contain it. 
 			</P><P class="h2"><I> Citrobacter freundii </I> characterisation
			</P><P class="normal-text">
We tried to see whether this construct would work in <I>Citrobacter freundii </I>, but we did not find a metronidazole concentration that inhibited its growth (we tried adding metronidazole up the concentration of 350ug/ml but this still gave an OD reading of 0.766). 

We have measured nitroreductase activity in the <I>nfsI</I> and control transformants and have found that the specific activity of nitroreductase was 14U/mg in the <I>nfsI</I> transformant, which is much higher than the specific activity we found in <I> E. coli </I> this is consistent with the finding that the Lac promoter is much stronger in <I> Citrobacter freundii </I>, as it is unregulated.

Since the nitroreductase enzyme itself seems to be working fine, it could mean that <I> Citrobacter freundii </I> is inherently resistant to activated metronidazole, so this is not a good counter-selectable marker option for <I> Citrobacter freundii </I>.

This is not necessarily a problem, as we have shown that our other counterselectable marker, <I>sacB</I> works very well in <I> Citrobacter freundii </I>, so that gene could instead be used for counterselection in this organism.
			</P><P class="h2">
Conclusions
			</P><P class="normal-text"><UL style="list-style-type:circle;padding:10px;"><LI>We successfully cloned the nitroreductase gene and inserted it into the BioBrick vector.</LI><LI>We extensively characterized the nitroreductase gene on plates and in liquid cultures.</LI><LI>We troubleshooted the Plac-lacZ'-nitroreductase clone and managed to purify it. </LI><LI>We are developing novel (to the best of our knowledge) counter-selection system which may have advantages over currently used systems.</LI><LI>We determined that nitroreductase is most suitable as a counter-selectable marker for <I>E. coli</I>in liquid aerobic cultures at 150 ug/ml metronidazole.</LI></UL></P><P class="normal-text" style="text-align:center"><A href="https://2012.igem.org/wiki/index.php?title=Team:Edinburgh/Project/Non-antibiotic-Markers"><SPAN class="intense-emphasis">&lt;&lt;Prev</SPAN></A><SPAN style="color:white;">__</SPAN>2/6<SPAN style="color:white;">__</SPAN><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Sucrose-Hydrolase"><SPAN class="intense-emphasis">Next&gt;&gt;</SPAN></A></P><P id="methods-heading" class="h2">
Methods <A class="cursor-pointer" onclick="expand('all-methods')">(expand)</A></P><P id="all-methods" class="normal-text">
Inserting gene into a biobrick vecor: Cloning a PCR product into a biobrick vector protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:bbcloning) however NEB buffers were used.

DNA gel preparation: Analysing DNA by gel electrophoresis protocol on OpanWetWare (http://openwetware.org/wiki/Cfrench:AGE) however 0.5*TAE rather than 1*TAE was used.

Colony PCR screen: Screening colonies by PCR protocol on OpenWetWare http://openwetware.org/wiki/Cfrench:PCRScreening

Transformations: Preparing and using compenent E.coli cells protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:compcellprep1)

PCR reactions : Cloning parts by PCR with Kod polymerase protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:KodPCR)

Minipreps : Plasmid DNA minipreps from Escerichia coli JM109 and similar strains protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:minipreps1)

Digests to linearise the DNA frangment/determine size of insert: Analytical restriction digests protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:restriction1)

DNA purification: Purifying a PCR product from solution protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:DNAPurification1) however 165 ul NaI, 5 ul glass beads,180 ul wash buffer and 10 ul EB were used.

DNA preparation for sequencing: 2.5 ul miniprepped DNA, 2 ul water and 1 ul forward primer ( specific for biobrick prefix) or reverse primer (specific for biobrick suffix)

Nitroreductase activity assay: Overnight liquid cultures of nitroreductase strains were centrifuged at 10000 rpm for 5 mins to pellet the cells. The cells were then resuspended in 250 ul PBS and 1 ul DTT to ensure that cellular proteins are not oxidized. The solution was sonicated 6* (10 s sonication+20 s rest). The supernatant was separated from the pellet by centrifugation and used for the NADH-dependent nitroreductase activity assay.

To assess background activity NADH (5 ul) and bacterial supernatant (5 ul) were added to 0.8 ml PBS and mixed. OD340 was measured for 1 minute. DNBA(5 ul) was added to the same cuvette to start the reaction and change in OD340 was monitored for 1 minute. DMSO(5 ul) was used a control (DNBA is dissolved in DMSO)

The protein concentration of each of the supernants was estimated by by Bradford protein assay using the Pierce reagent protocol on OpenWetWare(http://openwetware.org/wiki/Cfrench:ProteinAssay)
<A class="cursor-pointer" onclick="collapse('all-methods')">Close methods.</A></P><P class="h2"><A name="bibliography">Works Cited</A><A class="cursor-pointer" onclick="expand('works-cited')">(expand)</A></P><P id="works-cited" class="normal-text"><B>French, C., &amp; Kowal, M.</B> (2010, 09 24). <I>B. subtilis levansucrase. Lethal to E.coli in presence of sucrose.</I> Retrieved 2012, from Registry of standard biological parts: http://partsregistry.org/Part:BBa_K322921
<B>Gay, P., Coq, D. l., Strinmetz, M., Ferrari, E., &amp; Hoch, J. A.</B> (1983). <I>Cloning Structural Gene SacB, which Codes for Exoenzyme Levansucrase of Bacillus subtilis: Expression of the Gene in Esherichia coli.</I> Journal of Bacteriology , 1424-1431.
<B>Jahreis, K., Bentler, L., Bockmann, J., Hans, S., Meyer, A., Siepelmeyer, J., et al.</B> (2002). <I>Adaptation of sucrose metabolism in the Escherichia coli Wild-Type Strain EC31132.</I> Journal of Bacteriology, 5307-5316.
<B>Keuning, S., Janssen, D. B., &amp; Witholt, B.</B> (1985). <I>Purification and Characterisation of Hyrdrolytic Haloalkane Dehalogenase from Xanthobacter autotrophicus GJ10.</I> Journal of Bacteriology, 635-639.
<B>Naested, H., Fennema, M., Hao, L., Andersen, M., Janssen, D. B., &amp; Mundy, J.</B> (1999). <I>A bacterial haloalkane dehalogenase gene as a negative selectable marker in Arabidopsis.</I> The Plant Journal, 571-576.
<B>Nicklin, C. E., &amp; Bruce, N. C.</B> (1998). <I>Aerobic degradation of 2,4,6-Trinitrotoluene by Enterobacter cloaceae PB2 and by Pentaerythritol tetranitrate reductase.</I> Applied and environmental microbiology , 2864-2868.
<B>Nillius, D., Muller, J., &amp; Muller, N.</B> (2011). <I>Nitroreductase (GlNR1) increases susceptibility of Giardia lamblia and Escherichia coli to nitro drugs.</I> Journal of antimicrobial chemotherapy, 1029-1035.
<B>Kang et al.</B> (2009). <I>&quot;Levan: Applications and Perspectives&quot;.</I> Microbial Production of Biopolymers and Polymer Precursors. Caister Academic Press
<B>Dahech, I, Belghith, K. S., Hamden, K., Feki, A., Belghith, H. and Mejdoub, H.</B> (2011) <I>Antidiabetic activity of levan polysaccharide in alloxan-induced diabetic rats.</I> International Journal of Biological Macromolecules <B>49</B>(4):742-746
<A class="cursor-pointer" onclick="collapse('works-cited')">Close cited works.</A></P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Nitroreductase">http://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Nitroreductase</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Project/Non-antibiotic-Markers/Nitroreductase" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Project/Non-antibiotic-Markers/Nitroreductase" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Project/Non-antibiotic-Markers/Nitroreductase&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Project/Non-antibiotic-Markers/Nitroreductase&amp;oldid=288808" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>