<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Project_Non-antibiotic-Markers_Plac-RFP-SacB"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/Project/Non-antibiotic-Markers/Plac-RFP-SacB">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Project/Non-antibiotic-Markers/Plac-RFP-SacB&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Project/Non-antibiotic-Markers/Plac-RFP-SacB&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Project/Non-antibiotic-Markers/Plac-RFP-SacB&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Project/Non-antibiotic-Markers/Plac-RFP-SacB" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Project/Non-antibiotic-Markers/Plac-RFP-SacB</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="header"><DIV id="banner"><DIV id="follow"><UL><LI><SPAN class="plainlinks"><A class="follow-links" id="gmail" href="mailto:EdiGEM12@gmail.com"><P class="link-details">Gmail</P></A></SPAN></LI><LI><SPAN class="plainlinks"><A class="follow-links" id="gplus" href="https://plus.google.com/104634974040525950216/posts"><P class="link-details">Google+</P></A></SPAN></LI><LI><SPAN class="plainlinks"><A class="follow-links" id="twitter" href="https://twitter.com/#!/EdiGEM12"><P class="link-details">Twitter</P></A></SPAN></LI><LI><SPAN class="plainlinks"><A class="follow-links" id="facebook" href="http://www.facebook.com/Edigem2012"><P class="link-details">Facebook</P></A></SPAN></LI><LI><SPAN class="plainlinks"><A class="follow-links" id="blogger" href="http://synthbiopunk.blogspot.com/"><P class="link-details">Blogger</P></A></SPAN></LI></UL></DIV></DIV><DIV id="main-menu"><UL><LI id="home-page"><UL><A href="https://2012.igem.org/Team:Edinburgh" onmouseover="blind_down('home-page');" onmouseout="blind_up('home-page');"><P>Home</P></A></UL></LI><LI class="tab-description" id="home-page-tab-description" style="display:none;"><P>Welcome!</P></LI></UL><LI id="team-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Team" onmouseover="blind_down('team-page');" onmouseout="blind_up('team-page');"><P>Team</P></A></UL></LI><LI class="tab-description" id="team-page-tab-description" style="display:none;"><P>Meet the team!</P></LI><LI id="profile-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Profile" id="edinburgh-profile" onmouseover="blind_down('profile-page');" onmouseout="blind_up('profile-page');"><P>Profile</P></A></UL></LI><LI id="project-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Project" onmouseover="blind_down('project-page');" onmouseout="blind_up('project-page');"><P>Project</P></A></UL></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface">
							Bio-electric Interface </A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers">
							Alternative selectable and counter-selectable markers</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/0-Introduction"><I>Citrobacter freundii</I></A></LI><LI id="parts-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Parts" onmouseover="blind_down('parts-page');" onmouseout="blind_up('parts-page');"><P>Parts</P></A></UL></LI><LI id="modelling-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Modeling" onmouseover="blind_down('modeling-page');" onmouseout="blind_up('modeling-page');"><P>Modelling</P></A></UL></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Modelling/Kappa/Introduction">
							Kappa</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Modelling/Biosensors/Introduction">
							Biosensors</A></LI><LI id="human-practices-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices" onmouseover="blind_down('human-pr-page');" onmouseout="blind_up('human-pr-page');"><P>Human Practices</P></A></UL></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Crowdfunding">Crowdfunding</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/DIYbio">DIY Bio</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Perceptions-of-the-Public">Perceptions of the Public</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Blog">Blog</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Laws-and-Legislations">Laws and Legislations</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Friendly">Questioning Friendliness</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Failure">Failure</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Blue-Sky_Research">Blue-Sky Research</A></LI><LI id="safety-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Safety" onmouseover="blind_down('safety-page');" onmouseout="blind_up('safety-page');"><P>Safety</P></A></UL></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Safety#question1">
							Question 1</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Safety#question2">
							Question 2</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Safety#question3">
							Question 3</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Safety#question4">
							Question 4</A></LI><LI id="notebook-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Notebook" onmouseover="blind_down('notebook-page');" onmouseout="blind_up('notebook-page');"><P>Notebook</P></A></UL></LI><LI id="attributions-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Attributions" onmouseover="blind_down('attributions-page');" onmouseout="blind_up('attributions-page');"><P>Attributions</P></A></UL></LI></DIV></DIV><DIV id="page-middle"><DIV id="page-navigation"><UL><LI id="nav-bio-el-interface"><P>Bio-electric Interface</P></LI><LI id="nav-bio-el-background"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface">
					Background</A></LI><LI id="nav-bio-el-cloning"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface/Bio-electric-Interface-BioBricks-Cloning">
					BioBrick Cloning and Characterisation</A></LI><LI id="nav-bio-el-fuel-cells"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface/Microbial-Half-Fuel-Cells">
					Microbial Half Fuel Cells</A></LI><LI id="nav-bio-el-discussion"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface/Discussion">
					Discussion</A></LI></UL><LI id="nav-markers"><P>Alternative Selectable and Counter-selectable Markers</P></LI><LI id="nav-markers-introduction"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers">
					Introduction</A></LI><LI id="nav-markers-nitroreductase"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Nitroreductase">
					Nitroreductase</A></LI><LI id="nav-markers-sucrose-hydrolase"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Sucrose-Hydrolase">
					Sucrose Hydrolase</A></LI><LI id="nav-markers-dhla"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/DhlA">
					Haloalkane Dehalogenase</A></LI><LI id="nav-markers-Plac-RFP-SacB"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Plac-RFP-SacB">
					Levansucrase</A></LI><LI id="nav-markers-gene-rep"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Gene-replacement-strategy ">
					Gene ReplacementStrategy </A></LI><LI id="nav-lemon"><P><I>Citrobacter freundii</I></P></LI><LI id="nav-lemon-0"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/0-Introduction">
					Introduction</A></LI><LI id="nav-lemon-1"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/1-Replicon-compatibility">
					Replicon compatibility</A></LI><LI id="nav-lemon-2"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/2-Growth-in-sea-salts">
					Growth in sea salts</A></LI><LI id="nav-lemon-3"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/3-Sugar-use">
					Sugar use</A></LI><LI id="nav-lemon-4"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/4-Lac-promoter">
					Lac promoter</A></LI><LI id="nav-lemon-5"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/5-Genome-sequencing">
					Genome sequencing</A></LI><LI id="nav-lemon-6"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/6-Valencia">
					Valencia BiocampusBioBrick Characterisation</A></LI></DIV><DIV id="page-content"><DIV class="text"><P class="h1">
Alternative selectable and counter-selectable markers:

Levansucrase <I>(sacB) </I></P><P class="h2">
Background
			</P><P class="normal-text">
SacB is the levansucrase enzyme from <I>Bacillus subtilis</I><A href="#bibliography" onclick="expand('works-cited');">(Gay, Coq, Strinmetz, Ferrari, &amp; Hoch, 1983)</A> which converts sucrose into fructose polymers which are lethal to <I>Esherichia coli</I><A href="#bibliography" onclick="expand('works-cited');">(French &amp; Kowal, 2010)</A>. This part was deposited into the Registry by Team Edinburgh 2010 and can be used as a counter selectable marker <A href="#bibliography" onclick="expand('works-cited');">(French &amp; Kowal, 2010)</A>. Our aim is to improve the part by assessing its counter selection efficiency.
			</P><P class="h2">
Cloning
			</P><P class="normal-text">
The plac-RFP fragment was obtained from the standard BioBrick <A class="cursor-pointer" onclick="expand('plasmid');">plasmid</A> and inserted in front of the sacB BioBrick.  <A class="cursor-pointer" onclick="expand('method');">Method</A>. The construct was confirmed with <A class="cursor-pointer" onclick="expand('sequencing');">sequencing</A>.
			</P><P class="normal-text" id="plasmid"><I>PCR of pSB1K3 plasmid (Kanamycin resistance) with psBNX3 insF2 forward primer (specific for BioBrick prefix) and dsred r2 reverse primer (specific for RFP) was prepared in order to obtain the plac-RFP fragment.</I><I><B>Figure 1</B>: DNA gel of the PCR product from pSB1K3 amplification with primers specific to the BioBrick prefix and RFP. The band is abound 1 kb which corresponds to the expected size of plac-RFP.</I><A class="cursor-pointer" onclick="collapse('plasmid');">Close the plasmid.</A></P><P class="normal-text" id="method"><I>The plac-RFP PCR product was purified and digested with EcoRI HF and SpeI. The <I>sacB</I> BioBrick deposited in 2010 <A href="#bibliography" onclick="expand('works-cited');">(French &amp; Kowal, 2010)</A> was digested with EcoRI and XbaI. These were ligated together after purification. <I>E.coli</I> cells were transformed with the ligation. The red transformants were minipreped, digested with EcoRI HF in order to linearise them and with EcoRI HF and PstI in order to check the size of the insert.</I><I><B>Figure 2:</B> DNA gel of miniprepped red clones of linearized plac-RFP-SacB ligation transformants. The band is around 4.5 kb which corresponds to pSB1C3 (2kb)+ +</I><I><B>Figure 3:</B> The same clones were digested with EcoRI HF and PstI to check the size of the insert. The band is around 2.5 kb which corresponds to SacB (1.5 kb)+ plac-rfp (1kb).</I><A class="cursor-pointer" onclick="collapse('method');">Close the method.</A></P><P class="normal-text" id="sequencing"><I>Sequencing resultsForward primer:
Ctttaaaaaaaatcccttagctttcgctaaggtgatttctggaattcgcggccgcttctagagcaatac
gcaaaccgtttcaccccgcgcgttggccgattcattaatgcagctggcacgacaggtttcccgactgga
aagcgggcagtgagcgcaacgcaattaatgtgagttagctcactcattaggcaccccaggctttacact
ttatgcttccggctcgtatgttgtgtggaattgtgagcggataacaatttcacacatactagataaaga
ggagaaatactagatggcttcctccgaagacgttatcaaagagttcatgcgtttcaaagttcttatgga
aggttccgttaactgtcactagttcgaaatcgaaggtgaatgtgaaggtcgtccgtactaaggtaccca
gactgctaaactgaaagttactaaag

Reverse primer:
aggggccttaaacataaacttttcggttttagaaaagggcagggtggtgacaccttgcccttttttgcc
ggactgcagctactagtaatttatttgttaactgttaattgtccttgttcaaggatgctgtctttgaca
acagatgttttcttgcctttgatgttcagcaggaagcttggcgcaaacgttgattgtttgtctgcgtaa
aatcctctgtttgtcatatagcttgtaatcacgacattgtttcctttcgcttgaggtacagcgaagtgt
gagtaattaaaggttacatcgttaggatcaagatccatttttaacacatggcctgttttgttcagcggc
ttgtatgggccatttaaagaattagaaactttaccaagcatgttaatatcgttagacttatttccgtca
atccttatttttgatccgcgggagtcatttaacaggtaccatttgccgttcattttattttcgttcgcg
cgtctatttctttttgttactttgttttatgcaatcacgttttcattccttttttaattttgtatcatcgt</I><A class="cursor-pointer" onclick="collapse('sequencing');">Close the sequencing results.</A></P><P class="normal-text">The idea of placing <I>sacB</I> under the control of the lac promoter is to create kanamycin-independent control. Moreover, by changing the IPTG levels, the level of selection can be controlled.

One of the weaknesses of <I>sacB</I> selection often mentioned in the literature is that sacB loss-of-function mutants also survive, and this may occur at higher frequency than loss of the cassette. RFP is added to ensure that cells which have lost the cassette (should be white because of loss of RFP) can be distinguished from cells which have lost SacB function (should be red as RFP is still present). This would allow us to assess the counter-selection efficiency of SacB.
We also prepared a Kan-plac-RFP-sacB selection-counterselection cassette. 
			</P><P class="h2">
Characterisation
			</P><P class="normal-text">
The growth of SacB (bottom) and a control (top) were tested by adding 1.3 g solid sucrose plus 0.5 ml sterile water into a well in the middle of the plate, as shown in Figure 4 below.. <B>Figure 4</B>: A quick test of SacB transformants' growth in the presence of sucrose. The growth of SacB (bottom) is inhibited near the well in comparison to the control (top). 
			</P><P class="h3"><I>Citrobacter freundii</I></P><P class="normal-text"><I>SacB</I> and control (pSB1C3 only) transformants were streaked onto LB plates (with either cml40 or cml15) and solid sucrose was added to the regions indicated by the crosses or circles in Figure 5. The plates were incubated at 37Â°C overnight.
<B>Figure  5</B>: A quick test of <I>sacB</I> and controlâs ability to grow in the presence of sucrose. The zone of clearance indicates that the cells containing <I>sacB</I> are inhibited by sucrose while the controls are unaffected. The control on the plate on the right is purple due to both expressing the RFP gene and turning blue because of the X-gal on the plate.

The bottom streak on plate on the left in Figure 5 contains cells that have the selection-counterselection cassette so this test shows that the counterselection component of the cassette is working.  

We also tested the selection component of this cassette by streaking the cells onto plates containing both chloramphenicol and kanamycin and incubated overnight. These results can be seen in Figure 6.
<B>Figure 6:</B> Plate showing that the kanamycin resistance component of the selection-counterselection cassette works. The control did not grow at all, even after 4 days, as it was inhibited by the kanamycin, while the cells containing the cassette show growth. Growth is weaker because we have found that using the concentration of chloramphenicol we normally use (cml40) in combination with the sacB in <I>Citrobacter freundii</I> causes slower growth, presumably because levansucrase expression is unregulated in these bacteria which puts a strain on the cell.
			</P><P class="h3">
Sugar-dependent selection-counterselection cassette concept
			</P><P class="normal-text">
The reason we did not use our sucrose hydrolase gene for the selection component of the selection-counterselection cassette was because both it and levansucrase depend on the same substrate â sucrose â for their function, so this would have not yielded good results.

We have found that <I> Citrobacter freundii </I> can use sucrose as a sole carbon source without the need for the sucrose hydrolase gene â this means that if we want to develop a non-antibiotic resistance-dependent selection-counterselection cassette for this organism, we need to use a different sugar. One sugar that, according to our findings, <I>Citrobacter freundii </I> cannot use is xylitol â see our âXylitol dehydrogenaseâ section for more details on this. We could therefore combine this gene with the levansucrase gene to obtain an antibiotic resistance-independent selection-counterselection cassette that dependent entirely on sugars.
			</P><P class="h3">
Levans
			</P><P class="normal-text">
We have found that under sublethal doses of sucrose, the <I>E. coli </I> cells start to secrete a lot of gloopy substance, which we believe to be levans (Figure 7), the fructose polymers formed because of the levansucrase activity. 
<B>Figure 7:</B> Secretion of levans outside of the cell â it is only present on the lower two colonies but is absent in the two controls above.

We find this important to mention, as levans have got several applications in various fields <A href="#bibliography" onclick="expand('works-cited');">(Kang <I>et al.</I>, 2009)</A>.
<UL id="list"><LI>Food
		</LI><LI>has prebiotic effects</LI><LI>provides dietary fibres</LI><LI>reduces serum cholesterol levels </LI><LI>it can be used as a food additive in the following ways: a stabilizer, an emulsifier, a formulation aid, surface-finishing agent, an encapsulating agent, and a carrier of flavours and fragrances </LI></UL><LI>Pharmaceutical industry 
		</LI><LI>can be used as a coating material for drugs </LI><LI>has anti-tumour properties <I>in vitro</I></LI><LI>can be used as a blood plasma volume expander</LI><LI>has anti-diabetic effects <A href="#bibliography" onclick="expand('works-cited');">(Dahech <I>et al.</I>, 2011)</A></LI><LI>levan derivatives are shown to be anti-AIDS agents</LI><LI>Cosmetics
		</LI><LI>good as a cell-proliferating agent</LI><LI>skin moisturising agent</LI><LI>reduces skin irritation</LI><LI>Industry
		</LI><LI>it can be used in a two-phase liquid good for the separation of biological samples</LI><LI>it can be used an environmentally friendly adhesive</LI><LI>it can be used as a temporary adhesive in water-soluble form</LI><LI>it can be used as a water-resistant adhesive for a long period of time in cross-linked form</LI><LI>it can form a water-resistant film</LI><LI>it acts as a cryoprotectant for the preservation of animal cells and fish</LI></P><P class="h2">
Conclusions:
			</P><P class="normal-text"><UL style="padding-left:10px; list-style-type:circle;"><LI>We prepared a plac-RFP-SacB construct which can be used for assessing counter-selection efficiency. (<A href="http://partsregistry.org/Part:BBa_K917002">BBa_K917002</A>)</LI><LI>We placed <I>sacB</I> under the lac promoter which allows IPTG dependent control rather than kanamycin dependent control and IPTG concentration-dependent control of the levels of selection.</LI><LI>We added RFP to allow distinguishing loss of the counter-selection cassette form loss of SacB fuction.</LI><LI>We prepared and characterized a Kan-plac-RFP-sacB selection-counterselection cassette in both <I>E. coli</I> and <I> Citrobacter freundii </I> (<A href="http://partsregistry.org/Part:BBa_K917010">BBa_K917010</A>)</LI></UL></P><P class="normal-text" style="text-align:center"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/DhlA"><SPAN class="intense-emphasis">&lt;&lt;Prev</SPAN></A><SPAN style="color:white;">__</SPAN>5/6<SPAN style="color:white;">__</SPAN><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Gene-replacement-strategy"><SPAN class="intense-emphasis">Next&gt;&gt;</SPAN></A></P><P id="methods-heading" class="h2">
Methods <A class="cursor-pointer" onclick="expand('all-methods')">(expand)</A></P><P id="all-methods" class="normal-text">
Inserting gene into a biobrick vecor: Cloning a PCR product into a biobrick vector protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:bbcloning) however NEB buffers were used.

DNA gel preparation: Analysing DNA by gel electrophoresis protocol on OpanWetWare (http://openwetware.org/wiki/Cfrench:AGE) however 0.5*TAE rather than 1*TAE was used.

Colony PCR screen: Screening colonies by PCR protocol on OpenWetWare http://openwetware.org/wiki/Cfrench:PCRScreening

Transformations: Preparing and using compenent E.coli cells protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:compcellprep1)

PCR reactions : Cloning parts by PCR with Kod polymerase protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:KodPCR)

Minipreps : Plasmid DNA minipreps from Escerichia coli JM109 and similar strains protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:minipreps1)

Digests to linearise the DNA frangment/determine size of insert: Analytical restriction digests protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:restriction1)

DNA purification: Purifying a PCR product from solution protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:DNAPurification1) however 165 ul NaI, 5 ul glass beads,180 ul wash buffer and 10 ul EB were used.

DNA preparation for sequencing: 2.5 ul miniprepped DNA, 2 ul water and 1 ul forward primer ( specific for biobrick prefix) or reverse primer (specific for biobrick suffix)

Nitroreductase activity assay: Overnight liquid cultures of nitroreductase strains were centrifuged at 10000 rpm for 5 mins to pellet the cells. The cells were then resuspended in 250 ul PBS and 1 ul DTT to ensure that cellular proteins are not oxidized. The solution was sonicated 6* (10 s sonication+20 s rest). The supernatant was separated from the pellet by centrifugation and used for the NADH-dependent nitroreductase activity assay.

To assess background activity NADH (5 ul) and bacterial supernatant (5 ul) were added to 0.8 ml PBS and mixed. OD340 was measured for 1 minute. DNBA(5 ul) was added to the same cuvette to start the reaction and change in OD340 was monitored for 1 minute. DMSO(5 ul) was used a control (DNBA is dissolved in DMSO)

The protein concentration of each of the supernants was estimated by by Bradford protein assay using the Pierce reagent protocol on OpenWetWare(http://openwetware.org/wiki/Cfrench:ProteinAssay)
<A class="cursor-pointer" onclick="collapse('all-methods')">Close methods.</A></P><P class="h2"><A name="bibliography">Works Cited</A><A class="cursor-pointer" onclick="expand('works-cited')">(expand)</A></P><P id="works-cited" class="normal-text"><B>French, C., &amp; Kowal, M.</B> (2010, 09 24). <I>B. subtilis levansucrase. Lethal to E.coli in presence of sucrose.</I> Retrieved 2012, from Registry of standard biological parts: http://partsregistry.org/Part:BBa_K322921
<B>Gay, P., Coq, D. l., Strinmetz, M., Ferrari, E., &amp; Hoch, J. A.</B> (1983). <I>Cloning Structural Gene SacB, which Codes for Exoenzyme Levansucrase of Bacillus subtilis: Expression of the Gene in Esherichia coli.</I> Journal of Bacteriology , 1424-1431.
<B>Jahreis, K., Bentler, L., Bockmann, J., Hans, S., Meyer, A., Siepelmeyer, J., et al.</B> (2002). <I>Adaptation of sucrose metabolism in the Escherichia coli Wild-Type Strain EC31132.</I> Journal of Bacteriology, 5307-5316.
<B>Keuning, S., Janssen, D. B., &amp; Witholt, B.</B> (1985). <I>Purification and Characterisation of Hyrdrolytic Haloalkane Dehalogenase from Xanthobacter autotrophicus GJ10.</I> Journal of Bacteriology, 635-639.
<B>Naested, H., Fennema, M., Hao, L., Andersen, M., Janssen, D. B., &amp; Mundy, J.</B> (1999). <I>A bacterial haloalkane dehalogenase gene as a negative selectable marker in Arabidopsis.</I> The Plant Journal, 571-576.
<B>Nicklin, C. E., &amp; Bruce, N. C.</B> (1998). <I>Aerobic degradation of 2,4,6-Trinitrotoluene by Enterobacter cloaceae PB2 and by Pentaerythritol tetranitrate reductase.</I> Applied and environmental microbiology , 2864-2868.
<B>Nillius, D., Muller, J., &amp; Muller, N.</B> (2011). <I>Nitroreductase (GlNR1) increases susceptibility of Giardia lamblia and Escherichia coli to nitro drugs.</I> Journal of antimicrobial chemotherapy, 1029-1035.
<B>Kang et al.</B> (2009). <I>&quot;Levan: Applications and Perspectives&quot;.</I> Microbial Production of Biopolymers and Polymer Precursors. Caister Academic Press
<B>Dahech, I, Belghith, K. S., Hamden, K., Feki, A., Belghith, H. and Mejdoub, H.</B> (2011) <I>Antidiabetic activity of levan polysaccharide in alloxan-induced diabetic rats.</I> International Journal of Biological Macromolecules <B>49</B>(4):742-746
<A class="cursor-pointer" onclick="collapse('works-cited')">Close cited works.</A></P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Plac-RFP-SacB">http://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Plac-RFP-SacB</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Project/Non-antibiotic-Markers/Plac-RFP-SacB" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Project/Non-antibiotic-Markers/Plac-RFP-SacB" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Project/Non-antibiotic-Markers/Plac-RFP-SacB&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Project/Non-antibiotic-Markers/Plac-RFP-SacB&amp;oldid=288083" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>