<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Project_Citrobacter-Freundii_5-Genome-sequencing"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/Project/Citrobacter-Freundii/5-Genome-sequencing">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Project/Citrobacter-Freundii/5-Genome-sequencing&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Project/Citrobacter-Freundii/5-Genome-sequencing&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Project/Citrobacter-Freundii/5-Genome-sequencing&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Project/Citrobacter-Freundii/5-Genome-sequencing" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Project/Citrobacter-Freundii/5-Genome-sequencing</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="header"><DIV id="banner"><DIV id="follow"><UL><LI><SPAN class="plainlinks"><A class="follow-links" id="gmail" href="mailto:EdiGEM12@gmail.com"><P class="link-details">Gmail</P></A></SPAN></LI><LI><SPAN class="plainlinks"><A class="follow-links" id="gplus" href="https://plus.google.com/104634974040525950216/posts"><P class="link-details">Google+</P></A></SPAN></LI><LI><SPAN class="plainlinks"><A class="follow-links" id="twitter" href="https://twitter.com/#!/EdiGEM12"><P class="link-details">Twitter</P></A></SPAN></LI><LI><SPAN class="plainlinks"><A class="follow-links" id="facebook" href="http://www.facebook.com/Edigem2012"><P class="link-details">Facebook</P></A></SPAN></LI><LI><SPAN class="plainlinks"><A class="follow-links" id="blogger" href="http://synthbiopunk.blogspot.com/"><P class="link-details">Blogger</P></A></SPAN></LI></UL></DIV></DIV><DIV id="main-menu"><UL><LI id="home-page"><UL><A href="https://2012.igem.org/Team:Edinburgh" onmouseover="blind_down('home-page');" onmouseout="blind_up('home-page');"><P>Home</P></A></UL></LI><LI class="tab-description" id="home-page-tab-description" style="display:none;"><P>Welcome!</P></LI></UL><LI id="team-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Team" onmouseover="blind_down('team-page');" onmouseout="blind_up('team-page');"><P>Team</P></A></UL></LI><LI class="tab-description" id="team-page-tab-description" style="display:none;"><P>Meet the team!</P></LI><LI id="profile-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Profile" id="edinburgh-profile" onmouseover="blind_down('profile-page');" onmouseout="blind_up('profile-page');"><P>Profile</P></A></UL></LI><LI id="project-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Project" onmouseover="blind_down('project-page');" onmouseout="blind_up('project-page');"><P>Project</P></A></UL></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface">
							Bio-electric Interface </A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers">
							Alternative selectable and counter-selectable markers</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/0-Introduction"><I>Citrobacter freundii</I></A></LI><LI id="parts-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Parts" onmouseover="blind_down('parts-page');" onmouseout="blind_up('parts-page');"><P>Parts</P></A></UL></LI><LI id="modelling-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Modeling" onmouseover="blind_down('modeling-page');" onmouseout="blind_up('modeling-page');"><P>Modelling</P></A></UL></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Modelling/Kappa/Introduction">
							Kappa</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Modelling/Biosensors/Introduction">
							Biosensors</A></LI><LI id="human-practices-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices" onmouseover="blind_down('human-pr-page');" onmouseout="blind_up('human-pr-page');"><P>Human Practices</P></A></UL></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Crowdfunding">Crowdfunding</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/DIYbio">DIY Bio</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Perceptions-of-the-Public">Perceptions of the Public</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Blog">Blog</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Laws-and-Legislations">Laws and Legislations</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Friendly">Questioning Friendliness</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Failure">Failure</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Blue-Sky_Research">Blue-Sky Research</A></LI><LI id="safety-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Safety" onmouseover="blind_down('safety-page');" onmouseout="blind_up('safety-page');"><P>Safety</P></A></UL></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Safety#question1">
							Question 1</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Safety#question2">
							Question 2</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Safety#question3">
							Question 3</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Safety#question4">
							Question 4</A></LI><LI id="notebook-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Notebook" onmouseover="blind_down('notebook-page');" onmouseout="blind_up('notebook-page');"><P>Notebook</P></A></UL></LI><LI id="attributions-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Attributions" onmouseover="blind_down('attributions-page');" onmouseout="blind_up('attributions-page');"><P>Attributions</P></A></UL></LI></DIV></DIV><DIV id="page-middle"><DIV id="page-navigation"><UL><LI id="nav-bio-el-interface"><P>Bio-electric Interface</P></LI><LI id="nav-bio-el-background"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface">
					Background</A></LI><LI id="nav-bio-el-cloning"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface/Bio-electric-Interface-BioBricks-Cloning">
					BioBrick Cloning and Characterisation</A></LI><LI id="nav-bio-el-fuel-cells"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface/Microbial-Half-Fuel-Cells">
					Microbial Half Fuel Cells</A></LI><LI id="nav-bio-el-discussion"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface/Discussion">
					Discussion</A></LI></UL><LI id="nav-markers"><P>Alternative Selectable and Counter-selectable Markers</P></LI><LI id="nav-markers-introduction"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers">
					Introduction</A></LI><LI id="nav-markers-nitroreductase"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Nitroreductase">
					Nitroreductase</A></LI><LI id="nav-markers-sucrose-hydrolase"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Sucrose-Hydrolase">
					Sucrose Hydrolase</A></LI><LI id="nav-markers-dhla"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/DhlA">
					Haloalkane Dehalogenase</A></LI><LI id="nav-markers-Plac-RFP-SacB"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Plac-RFP-SacB">
					Levansucrase</A></LI><LI id="nav-markers-gene-rep"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Gene-replacement-strategy ">
					Gene ReplacementStrategy </A></LI><LI id="nav-lemon"><P><I>Citrobacter freundii</I></P></LI><LI id="nav-lemon-0"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/0-Introduction">
					Introduction</A></LI><LI id="nav-lemon-1"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/1-Replicon-compatibility">
					Replicon compatibility</A></LI><LI id="nav-lemon-2"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/2-Growth-in-sea-salts">
					Growth in sea salts</A></LI><LI id="nav-lemon-3"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/3-Sugar-use">
					Sugar use</A></LI><LI id="nav-lemon-4"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/4-Lac-promoter">
					Lac promoter</A></LI><LI id="nav-lemon-5"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/5-Genome-sequencing">
					Genome sequencing</A></LI><LI id="nav-lemon-6"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/6-Valencia">
					Valencia BiocampusBioBrick Characterisation</A></LI></DIV><DIV id="page-content"><DIV class="text"><P class="h1"><I>Citrobacter freundii</I> Characterisation:

Genome sequencing
			</P><P class="normal-text">
One major advantage of <I>E. coli</I> over <I>C. freundii</I> is the fact that there is a lot of sequence information available on the former, while no whole genome sequences exist for the latter. In order to get the ball rolling, the genomes of two <I>C. freundii</I> strains (the type strain, ATCC 8090 and another strain our lab had, called SBS 197) were sequenced in Newcastle by Drs Wendy Smith and Anil Wipat with IonTorrent Sequencing.
<SPAN class="intense-emphasis">To the best of our knowledge, no other iGEM projects have yielded genome sequencing data before, so this is a first both for our team, Europe and iGEM in general.</SPAN>
While a complete assembly of the sequence reads could not be done, these sequences constitute the first step towards unraveling the genome of our proposed chassis organism and even in this form they can provide valuable information.

Since our project was concerned with making synthetic biology safer, we wanted to see whether <I>Citrobacter freundii</I> contains any antibiotic resistance genes in its genome. With the help of the sequencing data (Figure 1), we have found that both strains have got several beta-lactamases encoded within their genomes, which normally confer these bacteria resistance to antibiotics with beta-lactam rings, such as ampicillin.
<B>Figure 1:</B> A snapshot of the sequencing reads that covered one of the beta-lactamase regions in <I>Citrobacter freundii</I>, showing good sequence coverage.

This is a puzzling find, as neither strain shows any resistance to carbenicillin, a synthetic ampicillin analogue. Nonetheless, a lot of gram negative bacteria carry beta-lactamases within their genomes, so even if <I>Citrobacter freundii</I> were released into the environment, it should not lead to an increase in the spread of antibiotic resistance. 

The raw sequencing files (zip files) and the contigs that have been assembled de novo (.fa format) can be accessed at the public <SPAN class="plainlinks"><A target="_blank" href="https://www.dropbox.com/sh/wg3a7jnvsr02hxn/7clt9E1xaS">Dropbox folder</A></SPAN> along with pdf files of the assembly reports. In addition, the automated annotation spread sheets (done by RAST) for both strains can also be accessed from this location.</P><P class="h2">
Lac operator sequence analysis
			</P><P class="normal-text">
One possible reason the Lac promoter coupled to our BioBricks is not regulated is because its LacI binding sequence might be different from that of the native <I>Citrobacter freundii</I> operator sequence. To test this, we have done a sequence alignment of the region where we think the <I>Citrobacter freundii</I> operator region might be with the consensus operator sequence in <I>E. coli</I> (5'-T GGAATTGTGAGCGGATAACAATT-3').  The sequence alignment can be seen in Figure 2 below.
<B>Figure 2:</B> Sequence alignment between our <I>Citrobacter freundii</I> sequence and the consensus <I>E. coli</I> operator region sequence

As it can be seen from this sequence alignment, the <I>Citrobacter freundii</I> sequence, while showing some similarities, is not completely identical to the <I>E. coli </I> consensus sequence, which might be the reason.
We then looked at the sequence of the LacR repressor protein in <I>Citrobacter freundii</I>, as it is known that the N-terminal sequence of this protein is what binds to DNA. We wanted to see whether there are any differences between this proteinâs N-terminal sequence and that of <I>E. coli</I> MG1655, the strain that is most often used in iGEM and in labs in general. The protein BLAST results can be seen in Figure 3.
<B>Figure 2:</B> protein BLAST results showing homology between the <I>Citrobacter freundii</I> (Query) and <I>E. coli </I>
K-12 MG1655 (Subject) Lac repressor protein sequence.
This result shows that the N-terminus of this protein is well conserved between these two organisms, but there are a few differences in amino acids that may account for this protein not being able to regulate a foreign Lac promoter.
			</P><P class="normal-text" style="text-align:center"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/4-Lac-promoter"><SPAN class="intense-emphasis">&lt;&lt;Prev</SPAN></A><SPAN style="color:white;">__</SPAN>6/7<SPAN style="color:white;">__</SPAN><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/6-Valencia"><SPAN class="intense-emphasis">Next&gt;&gt;</SPAN></A></P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/5-Genome-sequencing">http://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/5-Genome-sequencing</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Project/Citrobacter-Freundii/5-Genome-sequencing" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Project/Citrobacter-Freundii/5-Genome-sequencing" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Project/Citrobacter-Freundii/5-Genome-sequencing&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Project/Citrobacter-Freundii/5-Genome-sequencing&amp;oldid=289021" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>