"
Page
Discussion
View source
History
teams
Log in
 
Team:Edinburgh/Project/Non-antibiotic-Markers/Sucrose-Hydrolase
From 2012.igem.org
Gmail
Google+
Twitter
Facebook
Blogger
Home
Welcome!
Team
Meet the team!
Profile
Project
Bio-electric Interface
Alternative selectable and counter-selectable markers
Citrobacter freundii
Parts
Modelling
Kappa
Biosensors
Human
Practices
Crowdfunding
DIY Bio
Perceptions of the Public
Blog
Laws and Legislations
Questioning Friendliness
Failure
Blue-Sky Research
Safety
Question 1
Question 2
Question 3
Question 4
Notebook
Attributions
Bio-electric Interface
Background
BioBrick Cloning and Characterisation
Microbial Half Fuel Cells
Discussion
Alternative Selectable and Counter-selectable Markers
Introduction
Nitroreductase
Sucrose Hydrolase
Haloalkane Dehalogenase
Levansucrase
Gene Replacement
Strategy
Citrobacter freundii
Introduction
Replicon compatibility
Growth in sea salts
Sugar use
Lac promoter
Genome sequencing
Valencia Biocampus
BioBrick Characterisation
Alternative selectable and counter-selectable markers:
Sucrose Hydrolase (cscA)
Background
Sucrose hydrolase is an enzyme from Escherichia coli O157:H7 strain Sakai which is involved in sucrose utilisation (Jahreis, et al., 2002) .  Transforming Escherichia coli K12 strains with sucrose hydrolase allows the cells to grow with sucrose as a sole carbon source, something the untransformed K12 strain cannot do. This allows this gene to be used as a selectable marker.
Cloning
cscA cloning
The cscA gene was cloned PCR with cscA specific primers. Figure 1.
Figure 1: DNA gel of PCR amplification with primers specific for cscA . The product is around 1.4-1.5 kb which corresponds to the size of cscA gene, around 1.5 kb.
Close figure 1.
This fragment was inserted into the standard BioBrick vector pSB1C3. Figure 2.
Figure 2: DNA gel of pSBIC3-cscA ligation digested with EcoRI in order to linearise the plasmid. The band is around 3.5 kb which corresponds to the vector pSBIC3 (around 2 kb) together with the cscA gene (arounf 1.5 kb)
Close figure 2.
A promoter and a reporter gene were added in front of the cscA gene (Plac-lacZ'). Figure 3.
Figure 3: DNA gel of pSB1C3-Plac-lacZ'-cscA digested with XbaI and PstI. The clear band just above 2kb corresponds both to the size of the vector and the Plac-lacZ'-cscA fragment. The band around 4 kb is likely to correspond to the undigested plasmid.
Close figure 3.
cscA selection plasmid
In order to create a cscA selection plasmid, we wanted to replace the chloramphenicol resistance in pSB1C3 with cscA. The cscA and pSB1C3 gene were cloned using these primers . Method . However, this resulted in no successful pSB1C3-cscA ligation transformants.
Forward primer: GCTA gaattcgcggccgcttctagag caccagg agttgtt atg gat
Reverse primer: CATG ctgcag cggccgc t actagt a tta tt AGCACTCGG TCACAATCGT
Figure 4: DNA gel of PCR products of pSB1C3 without chloramphenicol and cscA. One product is around 1.4 kb which corresponds to the size of cscA gene, the other is around 2.2 kb which corresponds to the pSB1C3 vector without cml resistance.
Close the primers.
Method: The purified cscA and psB1C3 PCR products were digested with NdeI and ClaI. Both products were ligated and E.coli cells transformed with the ligation.
Close the method.
Characterisation
Plates
Plate characterisation showed that cscA is a suitable selectable marker- only cells which had the gene grew on sucrose as a sole carbon sourse (Figure 5). The drawback of this antibiotic-free selectable marker is that more time is required for the growth of the cscA cells on sucrose plates (we incubated them overnight at 37°C+4 days at room temperature, but they might have grown faster had we left them in the incubator).
Figure 5: Cells transformed with cscA ( BBa_K917000 ) (bottom row) as well as control cells (top row) were spread on LB plate, minimal plate with sucrose, minimal plate with glucose and minimal plate with no sugars, straight after transformation (without preselection on chloramphenicol).  Neither the cscA nor the control cells grow on minimal media with no sugars and grew well on LB and minimal plate with glucose. However, cscA cells are growing on minimal media with sucrose while the control cells are not.
Liquid Cultures
In order to better quantify our results, we have decided to grow our transformants in liquid media and measure OD after overnight incubation. We set up bottles with the same media as we have used for the plates (LB, M9 minimal medium with no sugars, M9 with 1% glucose and M9 with 1% sucrose), inoculated them with cscA or control transformants grown overnight and incubated them overnight before measuring OD. The results can be seen in Figure 6 below.
Figure 6: Comparison of growth between cells containing the sucrose hydrolase (cscA) selectable marker and control. LB and M9 glucose were used as positive controls, M9 with no sugars was used as a negative control.
Citrobacter xylitol selection marker strategy
In addition to E. coli, we were also working with the organism Citrobacter freundii over summer.
Unfortunately, we could not test our sucrose hydrolase selection system in this organism, as it can already degrade sucrose naturally. We have therefore devised the concept for an alternative sugar selection system that could be used in Citrobacter freundii. This sugar selection system is based on the sugar alcohol xylitol – our Citrobacter freundii sugar use experiments show that it cannot grow on this sugar as a sole carbon source, so it seems to be an ideal candidate for selectable marker design.
As some organisms can use and degrade xylitol, we have found an enzyme, called
xylitol dehydrogenase , which oxidizes xylitol to xylulose. This gene can be found in (for example) the gram negative rod Gluconobacter oxydans which is also friendly to humans, as it is not known to be pathogenic and in addition it is also used in various fields of biotechnology for example in the construction of bionsensors or for vinegar, vitamin C or sorbitol production.
The cloning strategy could be the same as was used to make the sucrose hydrolase BioBrick and assessing its effectivity could also be done following the same protocols, but of course, replacing sucrose with xylitol where needed.
This non-antibiotic selectable marker could be coupled up with our levansucrase (sacB counterselectable marker to form a selection-counterselection cassette that depends entirely on the presence of sugars, rather than antibiotics.
Conclusions:
We successfully cloned the sucrose hydrolase gene and inserted it into the BioBrick vector. ( BBa_K917000 )
We extensively characterised the sucrose hydrolase gene on plates and in liquid cultures.
We determined its suitability as a selectable marker.
We have developed a conceptual sugar-based selection system for Citrobacter freundii
<<Prev __3/6__ Next>>
Methods (expand)
Inserting gene into a biobrick vecor: Cloning a PCR product into a biobrick vector protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:bbcloning) however NEB buffers were used.
DNA gel preparation: Analysing DNA by gel electrophoresis protocol on OpanWetWare (http://openwetware.org/wiki/Cfrench:AGE) however 0.5*TAE rather than 1*TAE was used.
Colony PCR screen: Screening colonies by PCR protocol on OpenWetWare http://openwetware.org/wiki/Cfrench:PCRScreening
Transformations: Preparing and using compenent E.coli cells protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:compcellprep1)
PCR reactions : Cloning parts by PCR with Kod polymerase protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:KodPCR)
Minipreps : Plasmid DNA minipreps from Escerichia coli JM109 and similar strains protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:minipreps1)
Digests to linearise the DNA frangment/determine size of insert: Analytical restriction digests protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:restriction1)
DNA purification: Purifying a PCR product from solution protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:DNAPurification1) however 165 ul NaI, 5 ul glass beads,180 ul wash buffer and 10 ul EB were used.
DNA preparation for sequencing: 2.5 ul miniprepped DNA, 2 ul water and 1 ul forward primer ( specific for biobrick prefix) or reverse primer (specific for biobrick suffix)
Nitroreductase activity assay: Overnight liquid cultures of nitroreductase strains were centrifuged at 10000 rpm for 5 mins to pellet the cells. The cells were then resuspended in 250 ul PBS and 1 ul DTT to ensure that cellular proteins are not oxidized. The solution was sonicated 6* (10 s sonication+20 s rest). The supernatant was separated from the pellet by centrifugation and used for the NADH-dependent nitroreductase activity assay.
To assess background activity NADH (5 ul) and bacterial supernatant (5 ul) were added to 0.8 ml PBS and mixed. OD340 was measured for 1 minute. DNBA(5 ul) was added to the same cuvette to start the reaction and change in OD340 was monitored for 1 minute. DMSO(5 ul) was used a control (DNBA is dissolved in DMSO)
The protein concentration of each of the supernants was estimated by by Bradford protein assay using the Pierce reagent protocol on OpenWetWare(http://openwetware.org/wiki/Cfrench:ProteinAssay)
Close methods.
Works Cited (expand)
French, C., & Kowal, M. (2010, 09 24). B. subtilis levansucrase. Lethal to E.coli in presence of sucrose. Retrieved 2012, from Registry of standard biological parts: http://partsregistry.org/Part:BBa_K322921
Gay, P., Coq, D. l., Strinmetz, M., Ferrari, E., & Hoch, J. A. (1983). Cloning Structural Gene SacB, which Codes for Exoenzyme Levansucrase of Bacillus subtilis: Expression of the Gene in Esherichia coli. Journal of Bacteriology , 1424-1431.
Jahreis, K., Bentler, L., Bockmann, J., Hans, S., Meyer, A., Siepelmeyer, J., et al. (2002). Adaptation of sucrose metabolism in the Escherichia coli Wild-Type Strain EC31132. Journal of Bacteriology, 5307-5316.
Keuning, S., Janssen, D. B., & Witholt, B. (1985). Purification and Characterisation of Hyrdrolytic Haloalkane Dehalogenase from Xanthobacter autotrophicus GJ10. Journal of Bacteriology, 635-639.
Naested, H., Fennema, M., Hao, L., Andersen, M., Janssen, D. B., & Mundy, J. (1999). A bacterial haloalkane dehalogenase gene as a negative selectable marker in Arabidopsis. The Plant Journal, 571-576.
Nicklin, C. E., & Bruce, N. C. (1998). Aerobic degradation of 2,4,6-Trinitrotoluene by Enterobacter cloaceae PB2 and by Pentaerythritol tetranitrate reductase. Applied and environmental microbiology , 2864-2868.
Nillius, D., Muller, J., & Muller, N. (2011). Nitroreductase (GlNR1) increases susceptibility of Giardia lamblia and Escherichia coli to nitro drugs. Journal of antimicrobial chemotherapy, 1029-1035.
Kang et al. (2009). "Levan: Applications and Perspectives". Microbial Production of Biopolymers and Polymer Precursors. Caister Academic Press
Dahech, I, Belghith, K. S., Hamden, K., Feki, A., Belghith, H. and Mejdoub, H. (2011) Antidiabetic activity of levan polysaccharide in alloxan-induced diabetic rats. International Journal of Biological Macromolecules 49(4):742-746
Close cited works.
Retrieved from " http://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Sucrose-Hydrolase "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
