<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Project_Non-antibiotic-Markers_Sucrose-Hydrolase"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/Project/Non-antibiotic-Markers/Sucrose-Hydrolase">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Project/Non-antibiotic-Markers/Sucrose-Hydrolase&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Project/Non-antibiotic-Markers/Sucrose-Hydrolase&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Project/Non-antibiotic-Markers/Sucrose-Hydrolase&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Project/Non-antibiotic-Markers/Sucrose-Hydrolase" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Project/Non-antibiotic-Markers/Sucrose-Hydrolase</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="header"><DIV id="banner"><DIV id="follow"><UL><LI><SPAN class="plainlinks"><A class="follow-links" id="gmail" href="mailto:EdiGEM12@gmail.com"><P class="link-details">Gmail</P></A></SPAN></LI><LI><SPAN class="plainlinks"><A class="follow-links" id="gplus" href="https://plus.google.com/104634974040525950216/posts"><P class="link-details">Google+</P></A></SPAN></LI><LI><SPAN class="plainlinks"><A class="follow-links" id="twitter" href="https://twitter.com/#!/EdiGEM12"><P class="link-details">Twitter</P></A></SPAN></LI><LI><SPAN class="plainlinks"><A class="follow-links" id="facebook" href="http://www.facebook.com/Edigem2012"><P class="link-details">Facebook</P></A></SPAN></LI><LI><SPAN class="plainlinks"><A class="follow-links" id="blogger" href="http://synthbiopunk.blogspot.com/"><P class="link-details">Blogger</P></A></SPAN></LI></UL></DIV></DIV><DIV id="main-menu"><UL><LI id="home-page"><UL><A href="https://2012.igem.org/Team:Edinburgh" onmouseover="blind_down('home-page');" onmouseout="blind_up('home-page');"><P>Home</P></A></UL></LI><LI class="tab-description" id="home-page-tab-description" style="display:none;"><P>Welcome!</P></LI></UL><LI id="team-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Team" onmouseover="blind_down('team-page');" onmouseout="blind_up('team-page');"><P>Team</P></A></UL></LI><LI class="tab-description" id="team-page-tab-description" style="display:none;"><P>Meet the team!</P></LI><LI id="profile-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Profile" id="edinburgh-profile" onmouseover="blind_down('profile-page');" onmouseout="blind_up('profile-page');"><P>Profile</P></A></UL></LI><LI id="project-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Project" onmouseover="blind_down('project-page');" onmouseout="blind_up('project-page');"><P>Project</P></A></UL></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface">
							Bio-electric Interface </A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers">
							Alternative selectable and counter-selectable markers</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/0-Introduction"><I>Citrobacter freundii</I></A></LI><LI id="parts-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Parts" onmouseover="blind_down('parts-page');" onmouseout="blind_up('parts-page');"><P>Parts</P></A></UL></LI><LI id="modelling-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Modeling" onmouseover="blind_down('modeling-page');" onmouseout="blind_up('modeling-page');"><P>Modelling</P></A></UL></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Modelling/Kappa/Introduction">
							Kappa</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Modelling/Biosensors/Introduction">
							Biosensors</A></LI><LI id="human-practices-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices" onmouseover="blind_down('human-pr-page');" onmouseout="blind_up('human-pr-page');"><P>Human Practices</P></A></UL></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Crowdfunding">Crowdfunding</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/DIYbio">DIY Bio</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Perceptions-of-the-Public">Perceptions of the Public</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Blog">Blog</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Laws-and-Legislations">Laws and Legislations</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Friendly">Questioning Friendliness</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Failure">Failure</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Human_Practices/Blue-Sky_Research">Blue-Sky Research</A></LI><LI id="safety-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Safety" onmouseover="blind_down('safety-page');" onmouseout="blind_up('safety-page');"><P>Safety</P></A></UL></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Safety#question1">
							Question 1</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Safety#question2">
							Question 2</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Safety#question3">
							Question 3</A></LI><LI class="tab-sub-menu"><A href="https://2012.igem.org/Team:Edinburgh/Safety#question4">
							Question 4</A></LI><LI id="notebook-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Notebook" onmouseover="blind_down('notebook-page');" onmouseout="blind_up('notebook-page');"><P>Notebook</P></A></UL></LI><LI id="attributions-page"><UL><A href="https://2012.igem.org/Team:Edinburgh/Attributions" onmouseover="blind_down('attributions-page');" onmouseout="blind_up('attributions-page');"><P>Attributions</P></A></UL></LI></DIV></DIV><DIV id="page-middle"><DIV id="page-navigation"><UL><LI id="nav-bio-el-interface"><P>Bio-electric Interface</P></LI><LI id="nav-bio-el-background"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface">
					Background</A></LI><LI id="nav-bio-el-cloning"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface/Bio-electric-Interface-BioBricks-Cloning">
					BioBrick Cloning and Characterisation</A></LI><LI id="nav-bio-el-fuel-cells"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface/Microbial-Half-Fuel-Cells">
					Microbial Half Fuel Cells</A></LI><LI id="nav-bio-el-discussion"><A href="https://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface/Discussion">
					Discussion</A></LI></UL><LI id="nav-markers"><P>Alternative Selectable and Counter-selectable Markers</P></LI><LI id="nav-markers-introduction"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers">
					Introduction</A></LI><LI id="nav-markers-nitroreductase"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Nitroreductase">
					Nitroreductase</A></LI><LI id="nav-markers-sucrose-hydrolase"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Sucrose-Hydrolase">
					Sucrose Hydrolase</A></LI><LI id="nav-markers-dhla"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/DhlA">
					Haloalkane Dehalogenase</A></LI><LI id="nav-markers-Plac-RFP-SacB"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Plac-RFP-SacB">
					Levansucrase</A></LI><LI id="nav-markers-gene-rep"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Gene-replacement-strategy ">
					Gene ReplacementStrategy </A></LI><LI id="nav-lemon"><P><I>Citrobacter freundii</I></P></LI><LI id="nav-lemon-0"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/0-Introduction">
					Introduction</A></LI><LI id="nav-lemon-1"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/1-Replicon-compatibility">
					Replicon compatibility</A></LI><LI id="nav-lemon-2"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/2-Growth-in-sea-salts">
					Growth in sea salts</A></LI><LI id="nav-lemon-3"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/3-Sugar-use">
					Sugar use</A></LI><LI id="nav-lemon-4"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/4-Lac-promoter">
					Lac promoter</A></LI><LI id="nav-lemon-5"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/5-Genome-sequencing">
					Genome sequencing</A></LI><LI id="nav-lemon-6"><A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/6-Valencia">
					Valencia BiocampusBioBrick Characterisation</A></LI></DIV><DIV id="page-content"><DIV class="text"><P class="h1">
Alternative selectable and counter-selectable markers:

Sucrose Hydrolase <I> (cscA)</I></P><P class="h2">
Background
			</P><P class="normal-text">
Sucrose hydrolase is an enzyme from <I>Escherichia coli</I> O157:H7 strain Sakai which is involved in sucrose utilisation <A href="#bibliography" onclick="expand('works-cited');">(Jahreis, et al., 2002)</A>.  Transforming <I>Escherichia coli</I> K12 strains with sucrose hydrolase allows the cells to grow with sucrose as a sole carbon source, something the untransformed K12 strain cannot do. This allows this gene to be used as a selectable marker.
			</P><P class="h2">
Cloning
			</P><P class="h3"><I>cscA</I> cloning 
			</P><P class="normal-text">
The <I>cscA</I> gene was cloned PCR with <I>cscA</I> specific primers. <A onclick="expand('figure1');">Figure 1.</A></P><P class="normal-text" id="figure1"><B>Figure 1:</B> DNA gel of PCR amplification with primers specific for <I> cscA </I>. The product is around 1.4-1.5 kb which corresponds to the size of <I> cscA </I> gene, around 1.5 kb.<A onclick="collapse('figure1');">Close figure 1.</A></P><P class="normal-text">This fragment was inserted into the standard BioBrick vector pSB1C3. <A onclick="expand('figure2');">Figure 2.</A></P><P class="normal-text" id="figure2"><B>Figure 2:</B> DNA gel of pSBIC3-<I>cscA</I> ligation digested with EcoRI in order to linearise the plasmid. The band is around 3.5 kb which corresponds to the vector pSBIC3 (around 2 kb) together with the <I>cscA</I> gene (arounf 1.5 kb)<A onclick="collapse('figure2');">Close figure 2.</A></P><P class="normal-text">A promoter and a reporter gene were added in front of the <I>cscA</I> gene (Plac-lacZ'). <A onclick="expand('figure3');">Figure 3.</A></P><P class="normal-text" id="figure3"><B>Figure 3:</B> DNA gel of pSB1C3-Plac-lacZ'-cscA digested with XbaI and PstI. The clear band just above 2kb corresponds both to the size of the vector and the Plac-lacZ'-cscA fragment. The band around 4 kb is likely to correspond to the undigested plasmid. <A onclick="collapse('figure3');">Close figure 3.</A></P><P class="h3">
cscA selection plasmid
			</P><P class="normal-text">
In order to create a <I>cscA</I> selection plasmid, we wanted to replace the chloramphenicol resistance in pSB1C3 with <I>cscA</I>. The <I>cscA</I> and pSB1C3 gene were cloned using these <A class="cursor-pointer" onclick="expand('CscA-primers');">primers</A>. <A class="cursor-pointer" onclick="expand('CscA-method')">Method</A>. However, this resulted in no successful pSB1C3-cscA ligation transformants.
			</P><P class="normal-text" id="CscA-primers"><I>Forward primer: GCTA gaattcgcggccgcttctagag caccagg agttgtt atg gat
Reverse primer: CATG ctgcag cggccgc t actagt a tta tt AGCACTCGG TCACAATCGT</I><I><B>Figure 4:</B> DNA gel of PCR products of pSB1C3 without chloramphenicol and <I>cscA</I>. One product is around 1.4 kb which corresponds to the size of <I>cscA</I> gene, the other is around 2.2 kb which corresponds to the pSB1C3 vector without cml resistance.</I><A class="cursor-pointer" onclick="collapse('CscA-primers');">Close the primers.</A></P><P class="normal-text" id="CscA-method"><I>Method: The purified <I>cscA</I> and psB1C3 PCR products were digested with NdeI and ClaI. Both products were ligated and E.coli cells transformed with the ligation.</I><A class="cursor-pointer" onclick="collapse('CscA-method');">Close the method.</A></P><P class="h2">
Characterisation
			</P><P class="h3">
Plates
			</P><P class="normal-text">
Plate characterisation showed that <I>cscA</I> is a suitable selectable marker- only cells which had the gene grew on sucrose as a sole carbon sourse (Figure 5). The drawback of this antibiotic-free selectable marker is that more time is required for the growth of the <I>cscA</I> cells on sucrose plates (we incubated them overnight at 37°C+4 days at room temperature, but they might have grown faster had we left them in the incubator).
<B>Figure 5:</B> Cells transformed with <I>cscA</I> (<A href="http://partsregistry.org/Part:BBa_K917000">BBa_K917000</A>) (bottom row) as well as control cells (top row) were spread on LB plate, minimal plate with sucrose, minimal plate with glucose and minimal plate with no sugars, straight after transformation (without preselection on chloramphenicol).  Neither the <I>cscA</I> nor the control cells grow on minimal media with no sugars and grew well on LB and minimal plate with glucose. However, <I>cscA</I> cells are growing on minimal media with sucrose while the control cells are not.
			</P><P class="h3">
Liquid Cultures
			</P><P class="normal-text">
In order to better quantify our results, we have decided to grow our transformants in liquid media and measure OD after overnight incubation. We set up bottles with the same media as we have used for the plates (LB, M9 minimal medium with no sugars, M9 with 1% glucose and M9 with 1% sucrose), inoculated them with <I>cscA</I> or control transformants grown overnight and incubated them overnight before measuring OD. The results can be seen in Figure 6 below.
<B>Figure 6:</B> Comparison of growth between cells containing the sucrose hydrolase (<I>cscA</I>) selectable marker and control. LB and M9 glucose were used as positive controls, M9 with no sugars was used as a negative control.
			</P><P class="h2"><A name="xylitol">Citrobacter xylitol selection marker strategy</A></P><P class="normal-text">
In addition to <I> E. coli</I>, we were also working with the organism <I>Citrobacter freundii</I> over summer. 
Unfortunately, we could not test our sucrose hydrolase selection system in this organism, as it can already degrade sucrose naturally. We have therefore devised the concept for an alternative sugar selection system that could be used in <I>Citrobacter freundii</I>. This sugar selection system is based on the sugar alcohol xylitol – our <A href="https://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/3-Sugar-use#liquid-media"><I>Citrobacter freundii</I> sugar use experiments</A> show that it cannot grow on this sugar as a sole carbon source, so it seems to be an ideal candidate for selectable marker design.

As some organisms <I>can</I> use and degrade xylitol, we have found an enzyme, called <VB>xylitol dehydrogenase , which oxidizes xylitol to xylulose. This gene can be found in (for example) the gram negative rod <I> Gluconobacter oxydans </I> which is also friendly to humans, as it is not known to be pathogenic and in addition it is also used in various fields of biotechnology for example in the construction of bionsensors or for vinegar, vitamin C or sorbitol production.

The cloning strategy could be the same as was used to make the sucrose hydrolase BioBrick and assessing its effectivity could also be done following the same protocols, but of course, replacing sucrose with xylitol where needed. 

This non-antibiotic selectable marker could be coupled up with our levansucrase (<I>sacB</I> counterselectable marker to form a selection-counterselection cassette that depends entirely on the presence of sugars, rather than antibiotics.
			</VB></P><P class="h2">
Conclusions:
			</P><P class="normal-text"><UL style="padding-left:10px; list-style-type:circle;"><LI>We successfully cloned the sucrose hydrolase gene and inserted it into the BioBrick vector. (<A href="http://partsregistry.org/Part:BBa_K917000">BBa_K917000</A>)</LI><LI>We extensively characterised the sucrose hydrolase gene on plates and in liquid cultures.</LI><LI>We determined its suitability as a selectable marker.</LI><LI>We have developed a conceptual sugar-based selection system for <I>Citrobacter freundii </I></LI></UL></P><P class="normal-text" style="text-align:center"><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Nitroreductase"><SPAN class="intense-emphasis">&lt;&lt;Prev</SPAN></A><SPAN style="color:white;">__</SPAN>3/6<SPAN style="color:white;">__</SPAN><A href="https://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/DhlA"><SPAN class="intense-emphasis">Next&gt;&gt;</SPAN></A></P><P id="methods-heading" class="h2">
Methods <A class="cursor-pointer" onclick="expand('all-methods')">(expand)</A></P><P id="all-methods" class="normal-text">
Inserting gene into a biobrick vecor: Cloning a PCR product into a biobrick vector protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:bbcloning) however NEB buffers were used.

DNA gel preparation: Analysing DNA by gel electrophoresis protocol on OpanWetWare (http://openwetware.org/wiki/Cfrench:AGE) however 0.5*TAE rather than 1*TAE was used.

Colony PCR screen: Screening colonies by PCR protocol on OpenWetWare http://openwetware.org/wiki/Cfrench:PCRScreening

Transformations: Preparing and using compenent E.coli cells protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:compcellprep1)

PCR reactions : Cloning parts by PCR with Kod polymerase protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:KodPCR)

Minipreps : Plasmid DNA minipreps from Escerichia coli JM109 and similar strains protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:minipreps1)

Digests to linearise the DNA frangment/determine size of insert: Analytical restriction digests protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:restriction1)

DNA purification: Purifying a PCR product from solution protocol on OpenWetWare (http://openwetware.org/wiki/Cfrench:DNAPurification1) however 165 ul NaI, 5 ul glass beads,180 ul wash buffer and 10 ul EB were used.

DNA preparation for sequencing: 2.5 ul miniprepped DNA, 2 ul water and 1 ul forward primer ( specific for biobrick prefix) or reverse primer (specific for biobrick suffix)

Nitroreductase activity assay: Overnight liquid cultures of nitroreductase strains were centrifuged at 10000 rpm for 5 mins to pellet the cells. The cells were then resuspended in 250 ul PBS and 1 ul DTT to ensure that cellular proteins are not oxidized. The solution was sonicated 6* (10 s sonication+20 s rest). The supernatant was separated from the pellet by centrifugation and used for the NADH-dependent nitroreductase activity assay.

To assess background activity NADH (5 ul) and bacterial supernatant (5 ul) were added to 0.8 ml PBS and mixed. OD340 was measured for 1 minute. DNBA(5 ul) was added to the same cuvette to start the reaction and change in OD340 was monitored for 1 minute. DMSO(5 ul) was used a control (DNBA is dissolved in DMSO)

The protein concentration of each of the supernants was estimated by by Bradford protein assay using the Pierce reagent protocol on OpenWetWare(http://openwetware.org/wiki/Cfrench:ProteinAssay)
<A class="cursor-pointer" onclick="collapse('all-methods')">Close methods.</A></P><P class="h2"><A name="bibliography">Works Cited</A><A class="cursor-pointer" onclick="expand('works-cited')">(expand)</A></P><P id="works-cited" class="normal-text"><B>French, C., &amp; Kowal, M.</B> (2010, 09 24). <I>B. subtilis levansucrase. Lethal to E.coli in presence of sucrose.</I> Retrieved 2012, from Registry of standard biological parts: http://partsregistry.org/Part:BBa_K322921
<B>Gay, P., Coq, D. l., Strinmetz, M., Ferrari, E., &amp; Hoch, J. A.</B> (1983). <I>Cloning Structural Gene SacB, which Codes for Exoenzyme Levansucrase of Bacillus subtilis: Expression of the Gene in Esherichia coli.</I> Journal of Bacteriology , 1424-1431.
<B>Jahreis, K., Bentler, L., Bockmann, J., Hans, S., Meyer, A., Siepelmeyer, J., et al.</B> (2002). <I>Adaptation of sucrose metabolism in the Escherichia coli Wild-Type Strain EC31132.</I> Journal of Bacteriology, 5307-5316.
<B>Keuning, S., Janssen, D. B., &amp; Witholt, B.</B> (1985). <I>Purification and Characterisation of Hyrdrolytic Haloalkane Dehalogenase from Xanthobacter autotrophicus GJ10.</I> Journal of Bacteriology, 635-639.
<B>Naested, H., Fennema, M., Hao, L., Andersen, M., Janssen, D. B., &amp; Mundy, J.</B> (1999). <I>A bacterial haloalkane dehalogenase gene as a negative selectable marker in Arabidopsis.</I> The Plant Journal, 571-576.
<B>Nicklin, C. E., &amp; Bruce, N. C.</B> (1998). <I>Aerobic degradation of 2,4,6-Trinitrotoluene by Enterobacter cloaceae PB2 and by Pentaerythritol tetranitrate reductase.</I> Applied and environmental microbiology , 2864-2868.
<B>Nillius, D., Muller, J., &amp; Muller, N.</B> (2011). <I>Nitroreductase (GlNR1) increases susceptibility of Giardia lamblia and Escherichia coli to nitro drugs.</I> Journal of antimicrobial chemotherapy, 1029-1035.
<B>Kang et al.</B> (2009). <I>&quot;Levan: Applications and Perspectives&quot;.</I> Microbial Production of Biopolymers and Polymer Precursors. Caister Academic Press
<B>Dahech, I, Belghith, K. S., Hamden, K., Feki, A., Belghith, H. and Mejdoub, H.</B> (2011) <I>Antidiabetic activity of levan polysaccharide in alloxan-induced diabetic rats.</I> International Journal of Biological Macromolecules <B>49</B>(4):742-746
<A class="cursor-pointer" onclick="collapse('works-cited')">Close cited works.</A></P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Sucrose-Hydrolase">http://2012.igem.org/Team:Edinburgh/Project/Non-antibiotic-Markers/Sucrose-Hydrolase</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Project/Non-antibiotic-Markers/Sucrose-Hydrolase" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Project/Non-antibiotic-Markers/Sucrose-Hydrolase" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Project/Non-antibiotic-Markers/Sucrose-Hydrolase&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Project/Non-antibiotic-Markers/Sucrose-Hydrolase&amp;oldid=288843" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>