"
Page
Discussion
View source
History
teams
Log in
 
Team:Edinburgh/Project/Citrobacter-Freundii/5-Genome-sequencing
From 2012.igem.org
Gmail
Google+
Twitter
Facebook
Blogger
Home
Welcome!
Team
Meet the team!
Profile
Project
Bio-electric Interface
Alternative selectable and counter-selectable markers
Citrobacter freundii
Parts
Modelling
Kappa
Biosensors
Human
Practices
Crowdfunding
DIY Bio
Perceptions of the Public
Blog
Laws and Legislations
Questioning Friendliness
Failure
Blue-Sky Research
Safety
Question 1
Question 2
Question 3
Question 4
Notebook
Attributions
Bio-electric Interface
Background
BioBrick Cloning and Characterisation
Microbial Half Fuel Cells
Discussion
Alternative Selectable and Counter-selectable Markers
Introduction
Nitroreductase
Sucrose Hydrolase
Haloalkane Dehalogenase
Levansucrase
Gene Replacement
Strategy
Citrobacter freundii
Introduction
Replicon compatibility
Growth in sea salts
Sugar use
Lac promoter
Genome sequencing
Valencia Biocampus
BioBrick Characterisation
Citrobacter freundii Characterisation:
Genome sequencing
One major advantage of E. coli over C. freundii is the fact that there is a lot of sequence information available on the former, while no whole genome sequences exist for the latter. In order to get the ball rolling, the genomes of two C. freundii strains (the type strain, ATCC 8090 and another strain our lab had, called SBS 197) were sequenced in Newcastle by Drs Wendy Smith and Anil Wipat with IonTorrent Sequencing.
To the best of our knowledge, no other iGEM projects have yielded genome sequencing data before, so this is a first both for our team, Europe and iGEM in general.
While a complete assembly of the sequence reads could not be done, these sequences constitute the first step towards unraveling the genome of our proposed chassis organism and even in this form they can provide valuable information.
Since our project was concerned with making synthetic biology safer, we wanted to see whether Citrobacter freundii contains any antibiotic resistance genes in its genome. With the help of the sequencing data (Figure 1), we have found that both strains have got several beta-lactamases encoded within their genomes, which normally confer these bacteria resistance to antibiotics with beta-lactam rings, such as ampicillin.
Figure 1: A snapshot of the sequencing reads that covered one of the beta-lactamase regions in Citrobacter freundii, showing good sequence coverage.
This is a puzzling find, as neither strain shows any resistance to carbenicillin, a synthetic ampicillin analogue. Nonetheless, a lot of gram negative bacteria carry beta-lactamases within their genomes, so even if Citrobacter freundii were released into the environment, it should not lead to an increase in the spread of antibiotic resistance.
The raw sequencing files (zip files) and the contigs that have been assembled de novo (.fa format) can be accessed at the public Dropbox folder along with pdf files of the assembly reports. In addition, the automated annotation spread sheets (done by RAST) for both strains can also be accessed from this location.
Lac operator sequence analysis
One possible reason the Lac promoter coupled to our BioBricks is not regulated is because its LacI binding sequence might be different from that of the native Citrobacter freundii operator sequence. To test this, we have done a sequence alignment of the region where we think the Citrobacter freundii operator region might be with the consensus operator sequence in E. coli (5'-T GGAATTGTGAGCGGATAACAATT-3').  The sequence alignment can be seen in Figure 2 below.
Figure 2: Sequence alignment between our Citrobacter freundii sequence and the consensus E. coli operator region sequence
As it can be seen from this sequence alignment, the Citrobacter freundii sequence, while showing some similarities, is not completely identical to the E. coli consensus sequence, which might be the reason. We then looked at the sequence of the LacR repressor protein in Citrobacter freundii, as it is known that the N-terminal sequence of this protein is what binds to DNA. We wanted to see whether there are any differences between this protein’s N-terminal sequence and that of E. coli MG1655, the strain that is most often used in iGEM and in labs in general. The protein BLAST results can be seen in Figure 3.
Figure 2: protein BLAST results showing homology between the Citrobacter freundii (Query) and E. coli
K-12 MG1655 (Subject) Lac repressor protein sequence. This result shows that the N-terminus of this protein is well conserved between these two organisms, but there are a few differences in amino acids that may account for this protein not being able to regulate a foreign Lac promoter.
<<Prev __6/7__ Next>>
Retrieved from " http://2012.igem.org/Team:Edinburgh/Project/Citrobacter-Freundii/5-Genome-sequencing "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
