"
Page
Discussion
View source
History
teams
Log in
 
Team:Edinburgh/Project/Bioelectric-Interface/Bio-electric-Interface-BioBricks-Cloning
From 2012.igem.org
Gmail
Google+
Twitter
Facebook
Blogger
Home
Welcome!
Team
Meet the team!
Profile
Project
Bio-electric Interface
Alternative selectable and counter-selectable markers
Citrobacter freundii
Parts
Modelling
Kappa
Biosensors
Human
Practices
Crowdfunding
DIY Bio
Perceptions of the Public
Blog
Laws and Legislations
Questioning Friendliness
Failure
Blue-Sky Research
Safety
Question 1
Question 2
Question 3
Question 4
Notebook
Attributions
Bio-electric Interface
Background
BioBrick Cloning and Characterisation
Microbial Half Fuel Cells
Discussion
Alternative Selectable and Counter-selectable Markers
Introduction
Nitroreductase
Sucrose Hydrolase
Haloalkane Dehalogenase
Levansucrase
Gene Replacement
Strategy
Citrobacter freundii
Introduction
Replicon compatibility
Growth in sea salts
Sugar use
Lac promoter
Genome sequencing
Valencia Biocampus
BioBrick Characterisation
Bio-electric Interface:
BioBrick Cloning and Characterisation(Methods and Results)
Methods
- Bacteria used: Escherichia coli JM109 and Shewanella oneidensis MR-1. Both organisms were obtained from cultures in Chris French’s lab at the University of Edinburgh. S. oneidensis cultures were grown on LB agar at room temperature not exceeding 30°C. Plates were subcultured each week. E. coli cultures were grown on LB agar in a 37 degree incubator overnight and then left to grow at room temperature and subcultured by lab staff when needed.
- PCR: most PCR reactions were performed using the following OpenWetWare protocol CFrench: KodPCR . Optimal annealing temperature for S. oneidensis genes was found to be around 50-52°C while E. coli genes showed good results with annealing temperatures in the range of 50-55°C. S. oneidensis cell suspension in sterile water was used as template for mtrA, MtrCAB, S. oneidensis ccmA-E and ccmF-H genes. E. coli cell suspension in sterile water was used as template for napC and E. coli ccmA-H genes.
- PolyA tailing: for several genes polyA tailing was performed using Taq polymerase and the following protocol: 20 minutes denaturation at 95°C, followed by addition of Taq polymerase, followed by 15 minutes extension at 72°C.
- Gel electrophoresis: gel analysis was used following OpenWetWare CFrench: AGE protocol except 0,5 TAE buffer was used rather that 1x TAE. Gel staining was done using the SYBR-Safe staining solution.
- Gel purification and DNA purification: for ccm and several ligation attempts for other genes the PCR samples were run on the gel then the appropriate bands were cut out and purified using standardised QIAquick Gel Extraction Kits. For pure PCR products OpenWetWare protocol CFrench: DNAPurification1 was used.
- Vectors used: For most reaction the standard BioBrick vector pSB1C3 (provided by the Registry) was used, except for samples that were subjected to polyA tailing which were then ligated into the pGEM vector (Promega)
- Restriction digestions: Restriction digests were performed for PCR products along with vector digestion following the OpenWetWare CFrench:restriction1 protocol . For enhanced efficiency, varying ratios of insert to vector were used with the optimum ratio reached at about 3:1 to 5:1 ratio of insert digest to vector digest. Analytical restriction digests were also performed for miniprep samples using the original protocol.
- Ligations: Digested samples were mixed with 1 ul T4 ligase buffer and 1 ul T4 ligase and mixed with water to reach the final volume of 20 ul if necessary. Alternatively, polyA tailed PCR sampels were mixed with pGEM vector and used directly for ligation.
- Fusion PCR: following the ligation the samples were used as template for fusion PCR, following KodPCR protocol using forward primer for the gene and reverse primer for the vector. Extension time was adjusted to suit the length of the vector with insert.
- Transformation: Ligation and fusion PCR products were used to transform E coli JM109 competent cells using the OpenWetWare protocol Cfrench:compcellprep1 for preparation of competent cells and cell transformation).
- Transformed cell selection: Transformed cells were spread on LB agar with chloramphenicol (when using pSB1C3 vector) or LB agar with carbenicillin, Xgal and IPTG. Following overnight incubation at 37°C white colonies were chosen (rather than red colonies from pSB1C3-RFP vector or blue colonies from pGEM vector) and subcultured on plates containing the same medium.
- Miniprepping: Subcultures were used to set up overnight liquid cultures in 2,5 ml of LB. Miniprepping was performed using either the OpenWetWare protocol Cfrench:minipreps1 or standarised QIAprep Spin MiniPrep Kit. Minipreps were then restriction digested and run on a gel.
- Sequencing: size confirmed minipreps were then sent for sequencing at the University of Edinburgh GenePool.
Results
NapC
- Throughout the summer we have managed to clone the napC gene from E coli ( BBa_K917003 ). We have inserted the gene into the standard BioBrick vector pSB1C3 and submitted it to the parts Registry. We have then linked the napC gene to the lac promoter ( BBa_K917012 ) to characterise its functionality.
Figure 1: napC PCR
Figure 2: napC miniprep
Figure 3: pLac-napC construct analytical digestion with XbaI and PstI
Lanes 3, 4 = pSB1C3-Plac-lacZ'-napC, clones 1 and 2, digested with XbaI-PstI. Clone 2 looks as expected, clone 1 has an unexpected band around 0.6 kb.
MtrA
- We also managed to obtain the mtrA gene ( BBa_K917008 ) of S. oneidensis and cloned it into the pSB1C3 plasmid.
Figure 4: MtrA PCR
Figure 5: mtrA transformation
Figure 6: mtrA miniprep
The obtained mtrA gene contains an internal PstI site which needs to be mutated out prior to submission and use.
CymA
We have managed to clone the cymA gene ( BBa_K917009 ) from S. oneidensis. We have tested the gene for internal restriction sites and linked the cymA gene to the lac promoter ( BBa_K917014 ) to characterise its functionality.
Figure 7: lanes 3, 4 = pSB1C3-cymA clones 1 and 2, analytically digested with EcoRI. Band has size correct for linearised plasmid with the gene(3kb)
lanes 5, 6 = pSB1C3-cymA clones 1 and 2, double digested with EcoRI/SpeI.
Figure 8: pSB1C3-cymA clones 1 and 2, testing internal restriction sites.
Lanes 1, 2 = clones 1 and 2, NdeI.
Lanes 3, 4 = clone 1, XbaI and XbaI/HindIII.
Lanes 5, 6 =  clone 2, XbaI and XbaI/HindIII.
Gel results appear as expected
Figure 9: Lanes 4 to 6, pSB1C3-Plac-lacZ'-cymA clones 1-3, analytically digested with XbaI-PStI.
Clones 1 and 2 show bands of appropriate sizes.
ccm cytochrome maturation cluster of E. coli
We have cloned the E. coli ccm gene cluster ( BBa_K917006 ), analysed its internal restriction sites and linked it with the lac promoter ( BBa_K917013 ).
Figure 10: E coli ccm genes PCR (right lanes)
Figure 11: pSB1C3-ccm clones 1-6, digested with EcoRI.
Clones 1 and 5 show bands of expected size, the other clones are too small
Figure 12: pSB1C3-ccm clones 1 and 5, testing internal restriction sites.
Lanes 1, 2 = EcoRI/SpeI digestion.
Lanes 3, 4 = BamHI digestion.
Lanes 5, 6 = ClaI digestion.
Results appear as expected assuming 2 of the 3 ClaI sites are uncuttable due to overlapping dam methylation
Figure 13: Lanes 1, 2 = pSB1C3-Plac-lacZ'-ccm, clones 1 and 2, digested with XbaI-PstI.
Clone 2 looks as expected, clone 1 has an unexpected band around 0.6 kb.
MtrCAB and S. oneidensis ccm
We have also obtained good quality pure PCR products of mtrCAB and ccm genes from S. oneidensis
Figure 14: MtrCAB PCR
Figure 15: ccm genes from S. oneidensis and E. coli
BioBrick characterisation
ccm, napC and cymA
We have tested the expression of NapC and CymA proteins in ccm transformed cells (ccm gene cluster BioBrick was transferred into pSB4K5 vector to allow for double epxression of ccm and cymA/napC genes). Cell pellets were scanned prior to sonication and cymA and napC transformed cells show slightly more intense colour, indicating higher concentration of haem in cells.
Following the sonication protein samples were run on a gel and were stained for haem using the following protocol:
The NuPAGE® gel was first soaked in 100 ml solution-I  (30 ml methanol, 70 ml 250 mM NaAc, pH=5.2) for 10 min, and then was moved into 50 ml  solution-II (15 mg 3,3',5,5'-Tetramethylbenzidine (TMBZ), 15 ml methanol and 35 ml 250 mM NaAc, pH=5.2; TMBZ was prepared  by dissolving the powder in methanol and placed in the dark) in an opaque box to prevent  the light. The box was incubated for 25 min at room temperature. 150 ul 30% (w/w) H2O2  solution was then added into the box which was further incubated in the dark for 3~5 min  after gentle mixing. The stained bands were fixed by washing the gel with dH2O.
Haem staining resulted in clear bands forming in all samples, probably representing one of the Ccm proteins, most likely CcmE (haem chaperone protein). In napC and cymA samples additional bands were present, indicated by the arrows.
Haem staining: lane 1: pure cytochrome c control,
lane 2: protein extract from ccm transformed cells, membrane fraction;
lane 3: protein extract from ccm and cymA transformed cells, membrane fraction;
lane 4: protein extract from ccm and napC transformed cells, membrane fraction;
lane 5: protein extract from ccm transformed cells, cytosol fraction;
lane 6: protein extract from ccm and cymA transformed cells, cytosol fraction;
lane 7: protein extract from ccm and napC transformed cells, cytosol fraction
These results show that expression of CymA and NapC is successful. Interestingly NapC predominates in the soluble fraction and CymA appears in the membrane fraction. This is an unexpected result as both proteins are intermembrane proteins. We hope to inspect this fact closer to determine its significance in our system.
<<Prev __2/4__ Next>>
Bibliography (expand)
1. Jensen, H. M., Albers, A. E., Malley, K. R., Londer, Y. Y. , Cohen, B. E., Helms, B. A., Weigele, P., Groves, J. T. & Ajo-Franklin, C. M. (2010). Engineering of a synthetic electron conduit in living cells. PNAS  107, 19213-19218
2. Stewart, V., Lu, Y. & Darwin, A. J. (2002). Periplasmic Nitrate Reductase (NapABC Enzyme) supports Anaerobic Respiration by Escherichia coli K-12. Journal of Bacteriology 184, 1314-1323
3. Marritt, S. J., Lowe, T. G., Bye, J., McMillan, D.G.G., Shi, L., Frederickson, J., Zachara, J., Richardson, D. J., Cheesman, M. R., Jeuken L.J.C. & Butt, J. N. (2012). A functional description of CymA, an electron-transfer hub supporting anaerobic respiratory flexibility in Shewanella. Biochemical Journal 444, 465-474
4. Richter, K., Schicklberger, M., Gescher, J. (2011). Dissimilatory reduction of extracellular electron acceptors in anaerobic respiration. Applied and Environmental Microbiology 78, 913-921
Close bibliography.
Retrieved from " http://2012.igem.org/Team:Edinburgh/Project/Bioelectric-Interface/Bio-electric-Interface-BioBricks-Cloning "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
