<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UFSCar-Brasil_results_html skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UFSCar-Brasil/results.html</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="pusher"><DIV class="ui seven item team fluid fixed large stackable inverted secondary pointing network menu" style="padding-top: 14px;border-left-width: 0px;border-right-width: 0px;"><DIV class="ui raised vertical inverted center aligned vertical segment fluid container"><A href="https://2015.igem.org/Team:UFSCar-Brasil" class="item">Home</A><DIV class="ui dropdown item">
          Project
          <DIV class="menu"><A href=" https://2015.igem.org/Team:UFSCar-Brasil/Description" class="item">Overview</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/problem.html" class="item">Issue</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/plasmolise.html" class="item">Plasmolysis and Limonene synthase</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/proteinsolublizationkit.html" class="item">Protein Solubilization Toolkit</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/limoneno.html" class="item">Kill Switch</A></DIV></DIV><DIV href="team.html" class="ui dropdown item">
          Team
        <DIV class="menu"><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/team.html" class="item">Our Team</A><A href="https://2015.igem.org/Team:UFSCar-Brasil/Attributions" class="item">Attributions</A></DIV></DIV><DIV class="ui dropdown item">
          Achievements
          <DIV class="menu"><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/achievements.html" class="item">Overview</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/parts.html" class="item">Parts</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/results.html" class="item">Results</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/judging.html" class="item">Judging Criteria</A></DIV></DIV><DIV class="ui dropdown item">
          Policy &amp; Practices
          <DIV class="menu"><A href=" https://2015.igem.org/Team:UFSCar-Brasil/Practices" class="item">Overview</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/political.html" class="item">Political</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/social.html" class="item">Social</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/educacional.html" class="item">Educational</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/environmental.html" class="item">Safety/Security</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/meeting.html" class="item">Meetings</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/fundraising.html" class="item">Fundraising</A></DIV></DIV><DIV class="ui dropdown item">
          Modeling
          <DIV class="menu"><A href=" https://2015.igem.org/Team:UFSCar-Brasil/Modeling" class="item">Overview</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/part1.html" class="item">Plasmolysis</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/part2.html" class="item">Protein Solubilization Toolkit</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/part3.html" class="item">Kill Switch</A></DIV></DIV><DIV class="ui dropdown item">
          Notebook
          <DIV class="menu"><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/protocols.html" class="item">Protocols</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/biobricks.html" class="item">Problems</A><A href=" https://2015.igem.org/Team:UFSCar-Brasil/Collaborations" class="item">Collaborations</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/labs.html" class="item">Safety</A><A href=" https://2015.igem.org/Team:UFSCar-Brasil/Acknowledgements" class="item">Acknowledgements</A></DIV></DIV><DIV class="item"><A href="https://www.facebook.com/igemufscar" class="ui facebook button">Like us!</A></DIV></DIV></DIV><DIV class="ui vertical center aligned inverted segment"><DIV class="ui fluid container"><DIV class="ui text container"><H1 class="ui teal header">
        Results
      </H1><H2 class="ui teal header">What did we observe?</H2></DIV></DIV></DIV><DIV class="ui vertical stripe segment container"><H3 class="ui header" id="overview">Plasmolysis</H3><P> Main objective of this section is to verify the viability of <I>Escherichia coli</I> cells under treatments with polyethylene glycol (PEG 6000) for different time lapses, and the retaken of metabolism after plasmolysis. To this, we submited this organism to different concentrations of PEG 6000 and have assessed its growth hability. Assays of plasmolysis induction were performed using <I>E. coli</I> K-12 strain Dh5α cells obtained from a culture in lysogeny broth until an optical density at 600 nm of ~ 1.5-2.0, finally cells were harvested by centrifugation and washed with sterile 0.9% saline. Cells were resuspended in plasmolysis media (2% (vo./vol.) glycerol, 50 mM sodium acetate, 10 mM zinc chloride) containing various different concentrations of PEG 6000 (5, 10, 15, 20, 25 and 30% wt/vol). Proportion of cells in the final solution was 0.1% wt/vol. All media and their corresponding sterile controls were kept in triplicate at room temperature.</P><P> Primary spectrophotometry analysis, by checking optical density at 600 nm of all triplicates with different concentrations of PEG 6000, as well as the control medium without PEG showed cell growth was inhibited with increasing PEG 6000 concentrations used for stress condition induction (Figure 1).</P><H6 class="ui center aligned header"><B>Figure 1</B>: Experimental plot of absorbance versus time in different PEG 6000 concentration.</H6><P> Figure 1 shows no significant variation of weekly results with increasing PEG 6000 concentration. Optical density of bacterial suspension decreased with rising of PEG 6000 concentration. The growth could be verified since environment concerned an available carbon source for the microorganism, as glycerol. Cells number in the suspension influences the absorbance readings directly, however, possible cells arrangement and conformational changes could help formation of mobile cell complexes changes which affect considerably obtained data. It is due to light reflection direct linking with conformational and structural arrangement of these cells. To verify this, microscopy experiments were carried out through the Gram method (Figure 2).</P><H6 class="ui center aligned header"><B>Figure 2</B>: Cells conformational changes in optical microscopy reveals cell arangement to reduce entropy in medium through hydrophobic interactions. In (A) control cells without PEG 6000;
(B) 5% PEG 6000; (C) 10% PEG 6000; (D) 15% PEG 6000. All pictures were made using the same degree of magnification.</H6><P> Above images show increasing PEG concentration in solution inducing formation of cells complexes. This can demonstrate a thermodynamically favorable response to the presence of an extremely hydrophilic molecule in the medium. It is expected that those cells suffer a soft osmotic stress and come into plasmolysis more easily. All this process, could avoid the loss of viability even in solutions containing high contents of osmolyte.
Cell viability after plasmolysis, as well as the viability of the control medium without polyethylene glycol were verified weekly by direct plating of 100ul of each triplicate in LB solid medium, also concurrently with the plating serial dilutions in solid medium. The counting of colony forming units demonstrated that viability decreased to zero in control samples, without polyethylene glycol, for about a month and one week. But, in samples containing PEG 6000, this viability was kept (Figure 3).</P><H6 class="ui center aligned header"><B>Figure 3</B>: Viability after 7 Weeks.</H6><P> During tests carried out, some optical density readings of triplicates containing 25% of PEG 6000 returned unexpected values ​​and were further investigated as possible contamination signs. Some triplicate platings were done on solid medium and optical microscopy assessment was done. It was found to Staphylococcus sp. contamination in those samples, and after they were excluded.
Preliminary results indicate a promising maintenance of bacterial viability at room temperature using plasmolysis. Our results still indicate concentrations near 15% PEG 6000 representing the best option for this purpose. It is possible to note that cells viability suffers a decline in concentrations higher or lower than 15%. Possibly, lower osmolyte concentrations are not enough to promote plasmolysis efficiently, whereas higher concentrations may generate a irreversible osmotic stress frame, not supported by the microorganism (Paga &amp; MacKey, 2000).  In this sense, concentrations over 15% osmolyte concentrations can severely inhibit the molecular activity of cells, as shown by other authors, which further investigated this effect in gene induction (Rojas et al., 2014) and proteins synthesis (Clark and Parker, 1984).</P><DIV class="ui vertical stripe segment container"><H3 class="ui header" id="overview">Promoter UspA</H3><P> The universal stress protein A is a response of <I>E. coli</I> cells to growth arrest, and its lacking generates cells with defective growth. For more information about this important protein, please feel free to visit the respective Wikigenes page (https://www.wikigenes.org/e/gene/e/948007.html). 
The universal stress protein A promoter (also as known as PuspA) is a well characterized promoter element, inducible under several stress conditions (Prytz et al. 2003; Dyk et al. 1995; Nyström and Neidhardt 1992; 1994). This genetic element is dependent to sigma-70 factor (Nyström and Neidhardt 1992; 1994). Its regulation is done through the concentration of a specific stationary phase allormone, guanosine-5'-diphosphate-3'-diphosphate (ppGp), as described elsewhere (Farewell et al. 1998b). The ppGp activate the transcription of downstream elements through a positive regulation of the β-subunit of RNA polymerase. In this sense, the PuspA element is considered a stationary phase promoter. However, in the work of Prytz et al. (2003) a constitutive transcription promotion was observed. 
Diverse conditions make the <I>E. coli</I> cell enter to stress state, like heat shocks, starvation, osmotic stress, ultraviolet light and other conditions. In these situations, the RNA polimerase sigma factors (ropS) trigger the expression of chaperones and other stress protector molecules, in order to help the cell survive. Previous works have showed the power of the element PuspA, like shown in the Table 1.</P><H6 class="ui center aligned header"><B>Table 1.</B>Stress situations capable to induce the element PuspA. Response is given as a ratio of increase in signal of tested cells when compared with control cells.</H6><P>The force of this genetic element is important for construction of several devices, since environmental monitoring of toxic compounds to devices of triggered expression like this specific situation. We have drawn the sequence for an improved PuspA promoter from an analysis of 400bp upstream the Universal Protein A (UspA) gene in <I>E. coli</I> K12. First we have used the Scope tool (http://genie.dartmouth.edu/scope/), and we found the domain WWRBAM:</P><H6 class="ui center aligned header"><B>Figure 4.</B> Domain WWRBAM obtained from analysis of 400pb upstream starting of UspA gene, using Scope.</H6><P>This analysis indicated to us the sequence 5'-AAGCAT-3' as vital and potential component of promoter region. Restarting the analysis with Neural Network Promoter Predictor (http://www.fruitfly.org/seq_tools/promoter.html), the following sequence was obtained:</P><H6 class="ui center aligned header">5' – TGAGTTTTCAATCACCTTTCCATCCACCTTATATTAAGCATGGAGG - 3'</H6><P>This sequence has a transcription starting at bolded T (-10) and italicized the sigma factor binding sites. The confidence of the promotion activity of this sequence was estimated as 100%. Using this part attached to the iGEM prefix and suffix through polymerase chain reaction (PCR), the construction of pSB1C3 derived vector was carried out with current assembly methods, as shown below (the orientation is 5' &gt; 3'):</P><H6 class="ui center aligned header"><B>Figure 5.</B> Promoter final construct.</H6><P>Interestingly, the lineages carrying these plasmids have a slow growth behavior, in comparison to cells carrying the pSB1C3 plasmid. In order to report its activity, we have fusioned the biobrick part BBa_E0840 (RBS/B0030+GFP/E0040+Terminator/B0015) downstream to our promoter making other new biobrick. This construct was used to evaluate the puspA element activity. Since the start, we observed a green color in colonies of <I>E. coli</I> DH5α after their growth at 37°C and posterior incubation for 16h at 4°C in lysogenic agar supplemented with chloramphenicol 10 μg/mL (LB agar plus Chloram.). These colonies were selected and passed through a confirmatory polymerase chain reaction. The best producing clone was selected and used for further experimentation and plasmid production.</P><DIV class="ui vertical stripe segment container"><H4 class="ui header" id="overview"><B>Cold shock performance</B></H4><P>The <I>E. coli</I> DH5α clone carrying the construct puspA::gfp was grown in 100 mL of lysogenic broth containing chloramphenicol as previously referred (LB plus chloram) in a 500 mL Erlenmeyer flask under 200 rpm at 37°C. After culture reach optical density of 0.5, a sample of 5 mL was collected for protein analysis. The remaining broth was transferred to a 4°C chamber without agitation for 16 h. The cells were harvested by centrifugation. The protein analysis was carried out through a SDS-PAGE, using normalized samples, and a GFP fluorescence reading in black Elisa plates using Viktor (Perkin-Elmer).</P><H6 class="ui center aligned header"><B>Figure 6.</B> GFP expression using two constructs (<A href="http://parts.igem.org/Part:BBa_K1620005">BBa_K1620005</A> and <A href="http://parts.igem.org/Part:BBa_K1620006">BBa_K1620006</A>) under cold shock. The S and I fractions refers to soluble and insoluble fractions, obtained after ultrasonic lysis followed by high speed centrifugation and pellet washes. The black arrow indicates GFP expected height in SDS-PAGE using BenchMark protein ladder (Invitrogen).</H6><P>The SDS-PAGE revealed a similar effect of GFP production using the both tested promoters puspA and a control constitutive promoter pJ23101 (Bba_J23101). In this sense, the constitutive promotion of puspA was reinforced. But, when the fluorescence method was assessed, the fluorescence of samples obtained from puspA promotion was clearly brighter than the control promoter system. Furthermore, the protein quantities despite equivalent are not of the same qualities since the GFP bright is proportional to its folding. In this sense, we can conclude that puspA is not only constitutive in certain conditions; it is more suitable for complex folding proteins.</P><H6 class="ui center aligned header"><B>Figure 7.</B> Lysate fluorescence of overnight expressed GFP under cold shock of two constructs (<A href="http://parts.igem.org/Part:BBa_K1620005">BBa_K1620005</A> and <A href="http://parts.igem.org/Part:BBa_K1620006">BBa_K1620006</A>). The significant difference observed was evidenced through t-test at 0.05% of significance. Statistical inferences were made using GraphPad Prism v.5.0.</H6><DIV class="ui vertical stripe segment container"><H4 class="ui header" id="overview"><B>Osmotic shock induced by PEG 6000</B></H4><P>As previously reported, we choose to test the promoter element puspA::gfp under different osmotic pressures. The compound poly(ethylene glycol) 6000 kDa, also known as PEG 6000, is a hydrophilic compound which could adsorbs high amounts of water molecules, without change the ionic charge in medium, in this sense, only the reduction of the osmotic pressure would be tested.</P><P>The strain <I>E. coli</I>DH5α_puspA::gfp was grown under soft conditions (Lysogenic Broth Chloramphenicol supplemented, 37°C – 200 rpm) overnight, and cells were harvested from 2 mL cultures by centrifugation (4,000 rpm – 4°C, 6 min). These cells were resuspended in 1 mL of Lysogenic broth containing the right antibiotic and different concentrations of PEG 6000 ranging from 0 to 30% (in 2% steps until 20%). The 96 deep well plates were incubated at 37°C for 4 h, under 200 rpm. Finally, the cells were harvested by centrifugation as described before and the lysis was carried out after resuspension of cells in 180 μL of lysis buffer (10 mM Tris-HCl, 100 mM NaCl, 50 mM NaH2PO4, pH 7.5) with 20 μL volume of neutral detergent based lysis solution 10X (10mM EDTA, 10 mg/mL Lisozyme, 10% Triton X-100, 100 mM Tris-HCl – pH 8) at room temperature for 15 min with careful agitation at each 5 min. After lysis, the cell debris was precipitated by long time centrifugation (4,000 rpm – 4°C, 45 min) and the supernatant was collected to a 96 well Elisa black plate. The GFP was measured in Viktor (Perkin Elmer) plate fluorometer. </P><P>A 6xHis-Tagged version of our GFP was previously cloned into pET28a vector. The expression was carried out using a cell culture of <I>E. coli</I> Rosetta(DE3) at optical density of 0.5 in lysogenic broth supplement with chloramphenicol (20 μg/mL), kanamycin (25 μg/mL) and 0.4mM IPTG. The induction time and temperature were 4h at 37°C, respectively, in a shaker (200 rpm). Cells were harvested by centrifugation and lysed by ultra-sonication. The lysate was further centrifuged and 0.22 μm filtered. The GFP present in supernatant was purified using an imidazole gradient in a Ni^{2+}-NTA (Qiagen) column. The expression and purification profiles follow attached. A standard curve of purified GFP and fluorescence was previously established allowing the comparison of molecule number and promoter elements.</P><H6 class="ui center aligned header"><B>Figure 8.</B> GFP purification and expression under control of T7 promoter profile in SDS-PAGE 12%. Pure protein obtained in 250 mM (A and B) were used after dialysis to construction of standard curve. A BCA protein quantitation (Promega) was carried out. Legend: M – Marker BenchMark Protein Ladder (Invitrogen), NI – non induced, IN – insoluble portion of induced sample,  S – soluble portion of induced sample, FT – flow through, W – wash, Imidazol gradient (10 mM – 250 mM, with different fractions A-C).</H6><P>High amounts of GFP have shown a quantic quenching phenomenon, where the equipment is not able to detect proportional changes when high protein quantities are changed. In figure 9A, the plateau is evident and starts at 48 nM, to avoid this, we have adopted a standard curve (Figure 9B) using the linear region of the graphic (between 0 nM and 20 nM), transforming the GFP concentration (nM) to molecules. The collected data follow below:</P><H6 class="ui center aligned header"><B>Figure 9.</B> GFP standard curve (A) in nM per fluorescence and (B) in molecules per fluorescence, from the linear ranging obtained in (A). Each reading was taken three times and using three different samples. Each value was superposed. The regression curve was used to produce the following results. Statistical inferences were made using GraphPad Prism v.5.0.</H6><H6 class="ui center aligned header"><B>Figure 10.</B> Lysate fluorescence of 4h expressed GFP under osmotic shock of two constructs (<A href="http://parts.igem.org/Part:BBa_K1620005">BBa_K1620005</A> and <A href="http://parts.igem.org/Part:BBa_K1620006">BBa_K1620006</A>). A significant difference was observed between every pair of values evidenced through t-test at 0.05% of significance, except that higher than 18% PEG 6000. Statistical inferences were made using GraphPad Prism v.5.0.</H6><P>First of all, the puspA promoter has a higher activity than pJ23101 in all tested PEG concentrations. The data revealed a trend to reduction of GFP production along the PEG 6000 concentration increases from 0% to 30% (wt. / vol.). Higher concentrations of PEG 6000 turns the water molecules less available to cell, in this sense, we observe a phenomenon called plasmolysis. During plasmolysis the bacterial metabolism is suspended or reduced to very low rates, allied to a cell volume reduction implying in a retraction of the cytoplasmic volume. Despite the occurrence of all these events, no one of them is observable, even under microscope. This process could be useful for us, in order to long term conservation of the bacterial cultures, to distribution of our biosynthetic limonene based repellent.</P><P>The preliminary results still indicated a need for osmotic pressure changes triggered by ionic compounds to start or reinforces puspA activity. Previous studies used NaCl as an osmotic agent. We believe that these ions trigger a complete stimulus, involving efflux pumps. Our results allied to literature indicated to us, the need of an ionic unbalance allied to higher osmotic pressures and lower water activities to trigger puspA.</P><H6 class="ui center aligned header"><B>Figure 11.</B> GFP expression under control of different promoters along osmotic shock.  SDS-PAGE 12%. The black arrow indicates GFP expected height. M – Marker BenchMark Protein Ladder (Invitrogen).</H6><P>In this sense, this stress caused by non-ionic compounds, like PEG 6000, seems to be not so effective to production of interesting proteins coupled to this promoter element. Other informations seem to corroborate with our statements, since the carbon sources should be depleted to a full puspA activity, as previously described (Dyk et al. 1995, Gawand and Griffiths 2005). In a strict sense, the carbon sources in this test medium just are not depleted, and are reinforced by PEG 6000, a glycol polymer and potential useful energetic source for <I>E. coli</I> (EcoCyc, Pathway: ethylene glycol degradation) as seen at Figure 12.</P><H6 class="ui center aligned header"><B>Figure 12.</B><I>E. coli</I> K-12 substr. MG1655 pathway of ethylene glycol degradation. Extracted from the following <A href="http://biocyc.org/ECOLI/NEW-IMAGE?type=PATHWAY&amp;object=PWY0-1280&amp;detail-level=3">Link</A>.</H6><P>In summary, the plasmolysis technique was extremely useful to stop the metabolic activity of bacterial cell. A reversal of this framework could trigger the production of proteins coupled to the element puspA. This is the base of our final repellent product, but in order to element characterization, we still observe that increasing concentrations of this osmolyte are not effective to rise up the puspA activity due to its nutritional base for cells.</P><DIV class="ui vertical stripe segment container"><H4 class="ui header" id="overview"><B>Carbon starvation</B></H4><P>As previously referred the promoter puspA is sensitive to growth arrest and starvation. In order to study its behavior the cells carrying puspA::gfp and pJ23101::gfp were grown overnight in Lysogenic broth supplemented with chloramphenicol at 37°C under 200 rpm. Then, the cells were harvested by centrifugation and washed with sterile Tris-Saline (10mM Tris-HCl pH 8.0, 150 mM NaCl). Cells were resuspended in two different media: Lysogenic broth and Poor Medium (2 g/L Ammonium Sulfate, 13.6 g/L Monobasic Potassium Phosphate, 25 mg Lysogenic Broth (HiMedia), 10 μg/mL Cloramphenicol, filter sterilized), at equal optical densities. The cultures were incubated at 37°C under 200 rpm for 4h. Samples were collected at each hour and in the initial times. Finally, the cells were lysed as to the test of osmotic shock. A SDS-PAGE was used to compare the crude protein extracts, and the fluorometer was used to quantify the total produced GFP (as previously referred).</P><H6 class="ui center aligned header"><B>Figure 13.</B> Carbon starving effect in promotion by elements (A) pJ23101 and (B) puspA. Black arrows show the expected height for GFP band. M – Bench Marker Protein Ladder (Invitrogen), 0h – 4h refers to times of induction, S and I – Soluble and Insoluble portions.</H6><P>A complete profile of expression was obtained (Figure 13). The pJ23101 promotion occurs at high levels in the first three hours, like element puspA, in rich medium. Indeed, seems that the crude amounts of GFP are higher in the puspA, reinforcing the results obtained before. However, the quantities of insoluble GFP are also higher in puspA than pJ23101. The results still reveals that genes promoted by puspA, during starvation, have a kind of protection of proteolysis, when compared with the pJ23101 in poor medium.</P><P>These results (Figure 13 and 14) are in accordance to previously reported by Gawand and Griffiths (2005), where the cells response to starvation using element puspA was higher at 3h from the starting of the experiment and is more accentuated than other stress promoters, and also to the constitutive promoters. In this sense, the use of this promoter in poor nutritive solutions is recommended, and the maximum answer occurs at 3h.</P><H6 class="ui center aligned header"><B>Figure 14.</B> Fluorescence analysis of produced GFP under different promotion systems in rich (LB) and poor (MM) media. A significant difference was observed between pJ23101 and puspA in rich media, evidenced through t-test at 0.05% of significance. Statistical inferences were made using GraphPad Prism v.5.0.</H6><DIV class="ui vertical stripe segment container"><H3 class="ui header" id="overview">Chaperones</H3><P> Regarding the second module of our project, we built the biobricks <A href="http://parts.igem.org/Part:BBa_K1620002">BBa_K1620002</A> and <A href="http://parts.igem.org/Part:BBa_K1620003">BBa_K1620003</A>, corresponding to Hsps (small heat shock proteins) IbpA and IbpB. Aimming to assist in the correct folding of limonene synthase and to provide a larger quantity of functional enzyme and consequently D-limonene.</P><H6 class="ui center aligned header"><B>Figure 15</B>: Band pattern obtained from polymerase chain reaction using specific primers. Here, it is possible to observe right band patterns for UspA (<A href="http://parts.igem.org/Part:BBa_K1620000">BBa_K1620000</A>), Zasp (<A href="http://parts.igem.org/Part:BBa_K1620001">BBa_K1620001</A>), SmtA (<A href="http://parts.igem.org/Part:BBa_K1620007">BBa_K1620007</A>), Zur (<A href="http://parts.igem.org/Part:BBa_K1620004">BBa_K1620004</A>), IbpA (<A href="http://parts.igem.org/Part:BBa_K1620002">BBa_K1620002</A>).</H6><H6 class="ui center aligned header"><B>Figure 16</B>: Band pattern obtained from samples digestion with EcoRI and PstI enzymes. Here, it is possible to observe right band patterns for UspA (<A href="http://parts.igem.org/Part:BBa_K1620000">BBa_K1620000</A>), Zasp (<A href="http://parts.igem.org/Part:BBa_K1620001">BBa_K1620001</A>), SmtA (<A href="http://parts.igem.org/Part:BBa_K1620007">BBa_K1620007</A>), Zur (<A href="http://parts.igem.org/Part:BBa_K1620004">BBa_K1620004</A>), IbpB (<A href="http://parts.igem.org/Part:BBa_K1620003">BBa_K1620003</A>).</H6><DIV class="ui vertical stripe segment container"><H3 class="ui header" id="overview">Kill Switch</H3><P>Regarding the biosafety of our product, the module 3 of our project corresponds to a killswitch. It is composed by the assassin genes Killer Red and Barnase under the control of promoter Zasp (isolated from <I>znuABC</I> operon), the proteic factor Zur associated to a constitutive promoter and the protein SmtA associated to promoter UspA (universal stress global response regulator). The transcription of the assassin genes is regulated in response to the intracellular concentration of \(Zn^{2+}\), in this way the factor Zur associated to \(Zn^{2+}\) acts as a repressor bounded to Zasp blocking the transcription of the assassin genes. The quantity of intracellular available \(Zn^{2+}\) will lower as SmtA increases, because it associates to free \(Zn^{2+}\) in medium. As the available \(Zn^{2+}\) lowers the proteic factor Zur switches off from the promoter region allowing the transcription of assassin genes that will generate reactive oxygen species and RNases causing cell death. By this mechanism we guarantee a relatively short lifespan (around 4 hours according to our modelling) of our bacteria. Enough time for our product to fulfill its repellent action, and not enough to cause risks to the users’ health or environment.
Regarding the killswitch in our project, we built the biobrics Zinc uptake regulation protein (<A href="http://parts.igem.org/Part:BBa_K1620004">BBa_K1620004</A>) and <I>Synechococcus</I> metallothionenin A (<A href="http://parts.igem.org/Part:BBa_K1620007">BBa_K1620007</A>).</P><DIV class="ui vertical stripe segment container"><H3 class="ui header" id="overview">Limonene synthase</H3><P>We tried, repeatedly, to realize the amplification of the sequence referent to Limonene Synthase + Terminator B0015, but we did not have success. The results observed in agarose gel showed an unspecific band pattern.
Considering those attempts, we suppose that the sequence, corresponding to limonene synthase, was not synthesized correctly (neither from IDT, nor <A href="http://parts.igem.org/Part:BBa_I742111">BBa_I742111</A>), which interfered in our project execution.</P><DIV class="ui vertical stripe segment container"><H3 class="ui header" id="overview">References</H3><P>Paga R., MacKey B. 2000. Relationship between Membrane Damage and Cell Death in Pressure-Treated <I>Escherichia coli</I> Cells: Differences between Exponential – and Stationary – Phase Cells and Variation among Strains. Applied And Environmental Microbiology.</P><P>Clark D., Parker J. 1984. Proteins induced by high osmotic pressure in <I>Escherichia coli</I>. FEMS Microbiology Letters.</P><P>Rojas E., Theriot J.A., Huanga K.C. 2014. Response of <I>Escherichia coli</I> growth rate to osmotic shock. Department of Bioengineering. Stanford University.</P><P>Dyk T.K.V., Smulski D.R., Reed T.R., Belkin S., Vollmer A.C. and LaRossa R. 1995. Responses to Toxicants of an <I>Escherichia coli</I> Strain Carrying a uspA9::lux Genetic Fusion and an <I>E. coli</I> Strain Carrying a grpE9::lux Fusion Are Similar. Applied and Environmental Microbiology 61 (11): 4124–4127. DOI: 0099-2240/95/$04.0010.</P><P>Farewell A., Kvint K. and Nyström T. 1998. uspB, a New sigmaS-Regulated Gene in <I>Escherichia coli</I> Which Is Required for Stationary-Phase Resistance to Ethanol. Journal of Bacteriology 180 (23): 6140–6147. DOI: 0021-9193/98/$04.0010.</P><P>Gawande P.V. and Griffiths M.W. 2005. Effects of environmental stresses on the activities of the uspA, grpE and rpoS promoters of <I>Escherichia coli</I> O157:H7. International Journal of Food Microbiology 99: 91–98.</P><P>Nyström T. and Neidhardt F.C. 1992. Cloning, mapping and nucleotide sequencing of a gene encoding a universal stress protein in <I>Escherichia coli</I>. Mol. Microbiol. 6: 3187–3198.</P><P>Nyström T. and Neidhardt F.C. 1994. Expression and role of the universal stress protein, UspA, of <I>Escherichia coli</I> during growth arrest. Mol. Microbiol. 11: 537–544.</P><P>Prytz I., Sandén A.M., Nyström T., Farewell A., Wahlström A., Förberg C., Pragai Z., Barer M., Harwood C. and Larsson G. 2003. Fed-Batch Production of Recombinant Beta-Galactosidase Using the Universal Stress Promoters uspA and uspB in High Cell Density Cultivations. Biotechnology and Bioengineering 83 (5): 595-603. DOI: 10.1002/bit.10716.</P></DIV><H3 class="ui center aligned header">Our amazing sponsors!</H3><DIV class="ui inverted vertical footer segment" style="padding-top:0;padding-bottom:0;"><DIV class="ui fluid container"><DIV class="ui stackable inverted divided equal width stackable padded grid"><DIV class="column"><H4 class="ui inverted header">Links</H4><DIV class="ui inverted link list"><A class="item" href="https://igem.org/Main_Page">iGEM</A><A class="item" href="http://www2.ufscar.br/home/index.php">UFSCar</A></DIV></DIV><DIV class="column"><H4 class="ui inverted header">About</H4><DIV class="ui inverted link list"><A class="item" href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/team.html">Team</A></DIV></DIV><DIV class="column"><H4 class="ui inverted header">Thanks for reading our wiki!</H4><P>Come Again :)</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>