<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UFSCar-Brasil_part2_html skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UFSCar-Brasil/part2.html</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="pusher"><DIV class="ui seven item team fluid fixed large stackable inverted secondary pointing network menu" style="padding-top: 14px;border-left-width: 0px;border-right-width: 0px;"><DIV class="ui raised vertical inverted center aligned vertical segment fluid container"><A href="https://2015.igem.org/Team:UFSCar-Brasil" class="item">Home</A><DIV class="ui dropdown item">
          Project
          <DIV class="menu"><A href=" https://2015.igem.org/Team:UFSCar-Brasil/Description" class="item">Overview</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/problem.html" class="item">Issue</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/plasmolise.html" class="item">Plasmolysis and Limonene synthase</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/proteinsolublizationkit.html" class="item">Protein Solubilization Toolkit</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/limoneno.html" class="item">Kill Switch</A></DIV></DIV><DIV href="team.html" class="ui dropdown item">
          Team
        <DIV class="menu"><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/team.html" class="item">Our Team</A><A href="https://2015.igem.org/Team:UFSCar-Brasil/Attributions" class="item">Attributions</A></DIV></DIV><DIV class="ui dropdown item">
          Achievements
          <DIV class="menu"><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/achievements.html" class="item">Overview</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/parts.html" class="item">Parts</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/results.html" class="item">Results</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/judging.html" class="item">Judging Criteria</A></DIV></DIV><DIV class="ui dropdown item">
          Policy &amp; Practices
          <DIV class="menu"><A href=" https://2015.igem.org/Team:UFSCar-Brasil/Practices" class="item">Overview</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/political.html" class="item">Political</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/social.html" class="item">Social</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/educacional.html" class="item">Educational</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/environmental.html" class="item">Safety/Security</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/meeting.html" class="item">Meetings</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/fundraising.html" class="item">Fundraising</A></DIV></DIV><DIV class="ui dropdown item">
          Modeling
          <DIV class="menu"><A href=" https://2015.igem.org/Team:UFSCar-Brasil/Modeling" class="item">Overview</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/part1.html" class="item">Plasmolysis</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/part2.html" class="item">Protein Solubilization Toolkit</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/part3.html" class="item">Kill Switch</A></DIV></DIV><DIV class="ui dropdown item">     Notebook
          <DIV class="menu"><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/protocols.html" class="item">Protocols</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/biobricks.html" class="item">Problems</A><A href=" https://2015.igem.org/Team:UFSCar-Brasil/Collaborations" class="item">Collaborations</A><A href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/labs.html" class="item">Safety</A><A href=" https://2015.igem.org/Team:UFSCar-Brasil/Acknowledgements" class="item">Acknowledgements</A></DIV></DIV><DIV class="item"><A href="https://www.facebook.com/igemufscar" class="ui facebook button">Like us!</A></DIV></DIV></DIV><DIV class="ui vertical center aligned inverted segment"><DIV class="ui fluid container"><DIV class="ui text container"><H1 class="ui teal header">
        Protein Solubilization toolkit
      </H1><H2 class="ui teal header">Where are the singular effects?</H2></DIV></DIV></DIV><DIV class="ui vertical stripe segment stackable container"><H3 class="ui header" id="Overview">Simulation and analysis</H3><P> In this modeling step, we will develop a study about different chaperones arrangement efficiency in the proteins folding, using as example our protein of interest, Limonene Synthase. As it was not possible to obtain the complete system of this toolkit,
    there are no available experimental data to reinforce our findings. Nevertheless, the modeling of this section is extremely significant and could be useful to another teams with some similar problems. In this sense, we decided to model our system
    starting from a simulated dataset where we use fake data, and in the future, real data could simply replace them.</P><P> First of all, the experiment to measure the efficiency of each chaperone and arrangement of them was conceived in the following form:</P><P> All chaperones would be tested alone (IbpA+B, ClpB and DnaK), in pairs and all together. The proteins would be encoded in plasmid pSB1C3 under control of a constitutive promoter, allied to our gene of interest under the same promoter. Protein of interest
    would be produced during a few hours, and the cells would be harvested and lysed by ultrasound. Finally, the cell debris would be precipitated by centrifugation and the soluble and insoluble portions would be recovered. Then, in a SDS-PAGE the band of interest
    would be quantified by optical densitometry. Chaperone efficiency coefficient would be given as the ratio between soluble to insoluble bands.</P><P> Simulated data following this procedures were provided in the table below. A control group without any chaperones was included to comparison, and all measurements were made in triplicates. This table shows portion of well folded proteins (soluble)
    and misfolded ones (insoluble).</P><P><B>Table 1:</B> Simulated data using different chaperone sets and the solubility after the treatment.</P><TABLE class="ui table"><THEAD><TR><TH>Chaperone arrangements</TH><TH>Optical density of Insoluble band</TH><TH>Optical density of Soluble band</TH></TR></THEAD><TBODY><TR><TD>control group without chaperones</TD><TD>800</TD><TD>0</TD></TR><TR><TD>control group without chaperones</TD><TD>865</TD><TD>0</TD></TR><TR><TD>control group without chaperones</TD><TD>855</TD><TD>0</TD></TR><TR><TD>Ibp</TD><TD>600</TD><TD>124</TD></TR><TR><TD>Ibp</TD><TD>658</TD><TD>165</TD></TR><TR><TD>Ibp</TD><TD>659</TD><TD>188</TD></TR><TR><TD>ClpB</TD><TD>649</TD><TD>133</TD></TR><TR><TD>ClpB</TD><TD>655</TD><TD>132</TD></TR><TR><TD>ClpB</TD><TD>689</TD><TD>160</TD></TR><TR><TD>dnaK</TD><TD>699</TD><TD>144</TD></TR><TR><TD>dnaK</TD><TD>699</TD><TD>150</TD></TR><TR><TD>dnaK</TD><TD>689</TD><TD>161</TD></TR><TR><TD>Ibp_ClpB</TD><TD>400</TD><TD>250</TD></TR><TR><TD>Ibp_ClpB</TD><TD>458</TD><TD>266</TD></TR><TR><TD>Ibp_ClpB</TD><TD>423</TD><TD>267</TD></TR><TR><TD>Ibp_dnaK</TD><TD>465</TD><TD>200</TD></TR><TR><TD>Ibp_dnaK</TD><TD>465</TD><TD>220</TD></TR><TR><TD>Ibp_dnaK</TD><TD>462</TD><TD>280</TD></TR><TR><TD>dnaK_clpB</TD><TD>765</TD><TD>180</TD></TR><TR><TD>dnaK_clpB</TD><TD>733</TD><TD>199</TD></TR><TR><TD>dnaK_clpB</TD><TD>756</TD><TD>187</TD></TR><TR><TD>Ibp_Clpb_dnak</TD><TD>300</TD><TD>300</TD></TR><TR><TD>Ibp_Clpb_dnak</TD><TD>356</TD><TD>310</TD></TR><TR><TD>Ibp_Clpb_dnak</TD><TD>346</TD><TD>330</TD></TR></TBODY></TABLE><P>Chaperones efficiency coefficient still means the proportion of correct and incorrectly folded proteins. Considering the case of 100% of solubility, this coefficient tends to infinity, since the insoluble portion tends to zero. In this sense, a case
    of this type would be impossible to work, to prevent this, we decided work with protein yield. Protein yield coefficient would be given as percent (0-100) and calculated as follows:</P><H5 class="ui center aligned header"> $$Y = 100*[Soluble/(Soluble+Insoluble)] \tag{1}$$</H5><P><B>Table 2:</B> Chaperone arrangements related to chaperones efficiency and protein yield coefficients.</P><TABLE class="ui table"><THEAD><TR><TH>Chaperone arrangement</TH><TH>Soluble/Insoluble</TH><TH>Yield %</TH></TR></THEAD><TBODY><TR><TD>control group without chaperones</TD><TD>0</TD><TD>0</TD></TR><TR><TD>control group without chaperones</TD><TD>0</TD><TD>0</TD></TR><TR><TD>control group without chaperones</TD><TD>0</TD><TD>0</TD></TR><TR><TD>Ibp</TD><TD>0,20666667</TD><TD>17,12707</TD></TR><TR><TD>Ibp</TD><TD>0,25075988</TD><TD>20,0486</TD></TR><TR><TD>Ibp</TD><TD>0,28528073</TD><TD>22,19599</TD></TR><TR><TD>ClpB</TD><TD>0,20493066</TD><TD>17,00767</TD></TR><TR><TD>ClpB</TD><TD>0,20152672</TD><TD>16,77255</TD></TR><TR><TD>ClpB</TD><TD>0,23222061</TD><TD>18,8457</TD></TR><TR><TD>dnaK</TD><TD>0,20600858</TD><TD>17,08185</TD></TR><TR><TD>dnaK</TD><TD>0,21459227</TD><TD>17,66784</TD></TR><TR><TD>dnaK</TD><TD>0,23367199</TD><TD>18,94118</TD></TR><TR><TD>Ibp_ClpB</TD><TD>0,625</TD><TD>38,46154</TD></TR><TR><TD>Ibp_ClpB</TD><TD>0,58078603</TD><TD>36,74033</TD></TR><TR><TD>Ibp_ClpB</TD><TD>0,63120567</TD><TD>38,69565</TD></TR><TR><TD>Ibp_dnaK</TD><TD>0,43010753</TD><TD>30,07519</TD></TR><TR><TD>Ibp_dnaK</TD><TD>0,47311828</TD><TD>32,11679</TD></TR><TR><TD>Ibp_dnaK</TD><TD>0,60606061</TD><TD>37,73585</TD></TR><TR><TD>dnaK_clpB</TD><TD>0,23529412</TD><TD>19,04762</TD></TR><TR><TD>dnaK_clpB</TD><TD>0,27148704</TD><TD>21,35193</TD></TR><TR><TD>dnaK_clpB</TD><TD>0,2473545</TD><TD>19,83033</TD></TR><TR><TD>Ibp_Clpb_dnak</TD><TD>1</TD><TD>50</TD></TR><TR><TD>Ibp_Clpb_dnak</TD><TD>0,87078652</TD><TD>46,54655</TD></TR><TR><TD>Ibp_Clpb_dnak</TD><TD>0,95375723</TD><TD>48,81657</TD></TR></TBODY></TABLE><P>In order to observe this large set of combinations relate to each other in terms of yield, we used a hierarchical analysis model of clustering available on a free webserver, <A href="http://genomes.urv.cat/UPGMA/">DENDROUPGMA</A>. The calculations were done used in the creation of dendogram shown in Figure 1. This analysis basically shows together those data that are mathematically closer and separates those who are further away. As we can see, the values obtained with all chaperones acting together,
    here named &quot;all&quot;, are more isolated from other combinations for being the most efficient treatment of the others and so it is statistically more 'distant' to the other groups. Similarly we see the double &quot;Ibp+clpB&quot; and &quot;Ibp+DnaK&quot; grouped in the same
    node, so we conclude that their values are closer enough to not differentiate them and therefore cannot be separated.</P><H6 class="ui center aligned header"><B>Figure 1</B>: Dendogram of distances using simulated dataset generating a clustering complex behaviour.</H6><P>Cophenetic correlation coefficient, which brings how feaseable the dendogram is, was estimated as 0.84134 and values higher than 0.7 are considered as a good-fit. It can be seen that the combination of &quot;all&quot; chaperones together is farther away from
    the others, which was expected since its yield reached the highest value. Arrangements between &quot;Ibp+DnaK&quot; and &quot;Ibp+clpB&quot; are together on a branch, indicating that their yield values are closer than others. In addition, they are nearby branch of combination
    of all chaperones, indicating its yield not as high as the performance of all chaperones together, although close enough. This may indicate combinations &quot;Ibp+DnaK&quot; and &quot;Ibp+clpB&quot; as good substitutes for all three chaperones, when it would not be possible
    the complete arrangement.</P><P>As possible to observe, the protein yield using chaperones arrangement are not trivial, since there will be cross interactions hard to predict its behaviour jointly taking in account the single values. Predicting the interference between chaperones
    system, we equate all possible combinations of yield corresponding to each combination and multiply them by a mathematical factor \(x_i \). The goal is to interpret the values of these constants, to make it possible to compare the influence that each
    chaperone component isolated influences on the set of combinations.</P><P>Each index \(x_i\) represents an arrangement of all studied chaperones, in this way: \(x_1 \rightarrow Ibp\); \(x_2\rightarrow Clbp\); \(x_3\rightarrow DnaK\); \(x_4\rightarrow Ibp+Clbp\);\(x_5\rightarrow Ibp+DnaK\);\(x_6 \rightarrow Clbp+DnaK\); \(x_7
    \rightarrow Ibp+DnaK+Clpb\).</P><P>Results obtained for this system were:</P><H5 class="ui center aligned header"> $$x_1=(471/124)x_7-(251/248)x_6 \tag{2}$$</H5><H5 class="ui center aligned header"> $$x_2= -(314/71)x_7 +(502/213)x_6 \tag{3}$$</H5><H5 class="ui center aligned header"> $$x_3=(471/109)x_7 - (251/218)x_6 \tag{4}$$</H5><H5 class="ui center aligned header"> $$x_4=(251/612)x_6 \tag{5}$$</H5><H5 class="ui center aligned header"> $$x_5=(1884/503)x_7-(502/503)x_6 \tag{6}$$</H5><H5 class="ui center aligned header"> $$x_6=x_6 \tag{7}$$</H5><H5 class="ui center aligned header"> $$x_7=x_7 \tag{8}$$</H5><P>In this way, it will be possible to describe all parameters taking in account just two values \(x_6\) and \(x_7\). There is a relationship between all chaperones and the two combinations (all of them and “DnaK+Clpb”. Fixing one of them, it will be possible
    to study the behaviour of others using graphical tools (Figure 2 and 3).</P><H6 class="ui center aligned header"><B>Figure 2</B>: Solutions to values of \(x_i\) in function of values of \(x_6\) with \(x_7\) fixed equals to 1.</H6><H6 class="ui center aligned header"><B>Figure 3</B>: Solutions to values of \(x_i\) in function of values of \(x_7\) with \(x_6\) fixed equals to 1.</H6><P>It was possible to see that there is no single solution to our problem, and probably even with actual values there is no single solution. Although, the graphics can nonetheless take some dependency relationships between the coefficients, such
    as if \(x_7 \) is fixed \(x_2 \) component increases with slope greater than \(x_1 \), which decreases with the increasing of \(x_6 \) value. This indicates that ClpB influence in the chaperone combinations are higher than Ibp in these arrangements.
    In this sense, this result is reversed to from the moment when increase the amounts related to \(x_7 \). As it was not possible to obtain experimental data to validate our discussions and predict outcomes, we are pleased to leave this modeling as
    a theoretical model that can be deployed and tested in future projects of our or another iGEM team that wants to employ it.</P><H3 class="ui header" id="overview">References</H3><P>S. Garcia-Vallve, J. Palau and A. Romeu (1999) Horizontal gene transfer in glycosyl hydrolases inferred from codon usage in <I>Escherichia coli</I> and <I>Bacillus subtilis</I>. Molecular Biology and Evololution 16(9):1125-1134.</P></DIV><H3 class="ui center aligned header">Our amazing sponsors!</H3><DIV class="ui inverted vertical footer segment" style="padding-top:0;padding-bottom:0;"><DIV class="ui fluid container"><DIV class="ui stackable inverted divided equal width stackable padded grid"><DIV class="column"><H4 class="ui inverted header">Links</H4><DIV class="ui inverted link list"><A class="item" href="https://igem.org/Main_Page">iGEM</A><A class="item" href="http://www2.ufscar.br/home/index.php">UFSCar</A></DIV></DIV><DIV class="column"><H4 class="ui inverted header">About</H4><DIV class="ui inverted link list"><A class="item" href="https://2015.igem.org/wiki/index.php?title=Team:UFSCar-Brasil/team.html">Team</A></DIV></DIV><DIV class="column"><H4 class="ui inverted header">Thanks for reading our wiki!</H4><P>Come Again :)</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>