Team:UChicago/Notebook
MENU
HOME
TEAM
Team
Collaborations
PROJECT
Description
Design
Experiments
Notebook
Results
Improve
Attributions
PARTS
SAFETY
PUBLIC ENGAGEMENT
June-----
6/12/2017
·Made a list of things to order for the future
·Printed most of the protocols; they are located on the lab bench when needed
·Made LB, LB amp, and YED plates
       ·Note: LB amp plates did not turn out as well as the others; will reattempt on 6/13/2017
·LABELLING ON PLATES
        ·One red stripe → LB plate
        ·One red stripe with one blue stripe → LB Amp plate
        ·One black stripe → YED plates
·The plates have been left on the bench overnight; they will be put in the fridge tomorrow morning. Jason had cleared out a spot for us in the Glick fridge for our use!
6/13/2017
·Redid the LB-amp plates
·Made the Drop-Out plates (-Arg)
·Talked to Shannon about cleaning our glassware
·Labelled our glassware
·The petri dishes are in the fridge (all made within the last two days)
6/14/2017
·Learned how to use the Nanodrop
·Made two stocks of 500 mL LB media (1000 mL total)
·Testing optimal DNA concentrations for bacterial transformation:
       ·Spread six different DNA concentrations of the 126 ng/μl plasmid sample (AJ43) Jason gave us onto LB + amp plates
       ·Left these plates overnight in a 37℃ incubator
              ·Plate 1 = negative control (no DNA, contains 50 μl EB and 50 μl competent cells)
              ·Plate 2 = 453.6 ng DNA (3.6 μL DNA, 46.4 μL EB, 50 μL CC)
              ·Plate 3 = 315 ng DNA (2.5 μL DNA, 47.5 μL EB, 50 μL CC)
              ·Plate 4 = 239.4 ng DNA (1.9 μL DNA, 48.1 μL EB, 50 μL CC)
              ·Plate 5 = 201.6 ng DNA (1.6 μL DNA, 48.4 μL EB, 50 μL CC)
              ·Plate 6 = 163.8 ng DNA (1.3 μL DNA, 48.7 μL EB, 50 μL CC)
·Took a small sample of pOW1 (in frozen bacteria) from stock in the -80℃ fridge
·Put the pOW1 into 5 mL of LB + amp liquid media (1 μL ampicillin / 1 mL LB)
·Left the sample in the 37℃ shaker overnight to incubate
6/15/2017
·Retrieved the AJ43 plates
       ·Negative control = no colonies
       ·1 = too many colonies to count
       ·2 = too many colonies to count
       ·pOW1 plating concentrations for next week:
              ·1 μL pOW1 + 49 μL EB + 50 μL CC
              ·2.5 μL pOW1 + 47.5 μL EB + 50 μL CC
              ·5 μL pOW1 + 45 μL EB + 50 μL CC
6/19/2017
·Transformed the pOW1 DNA into competent cells
·Incubated the transformations for 2 hours in a 37℃ shaker
·Spread the samples on LB + amp plates
·Left the plates overnight in a 37℃incubator
6/20/2017
·Observed the plates from the previous transformation
·Retrieved pOW1 plates
       ·Negative Control
              ·No colonies as expected
       ·5 ul of plasmid + 45 ul of buffer
              
              ·3 colonies
              ·Picked the colonies and put them in LB media
                     ·Left in shaker to grow the bacteria (will perform a mini-prep once incubation is complete)
                     ·Made a glycerol stock of pOW1 bacteria
6/21/2017
·Mini-prepped the pOW1 bacteria that we incubated overnight in the 37℃ shaker
       ·Sample 1 = 16.5 ng/μL pOW1
       ·Sample 2 = 17.2 ng/μL pOW1
·pOW1 is in A44 (-80℃ fridge)
       ·Incubated (in the 37℃ shaker) a small sample of the pOW1 we retrieved from the fridge
6/23/2017
·Added MilliQ water to the qcF, qcR, ArgArsF, ArgArsR primers (ng primer x 10 = # μL of water)
·Mutagenesis PCR’d the pSB1C3 with qcF and qcR primers
6/26/2017
·PCR’d the pOW1 with ArgArsF and ArgArsR
·Restriction digest the PCR products with DpNI
·Restriction digest the pSB1C3 with HpaI ·Dephosphorylated the DNA
       ·When we ran a gel of the digest, we saw that the bands of our gel looked distorted
       ·We did not put in enough TAE buffer for both gels
6/27/2017
·Ran the gel again (PCR pSB1C3 products, DpNI cut pSB1C3mut, HpaI cut pSB1C3mut)
       ·We can see the PCR products, but it’s blurry
       ·Can’t see anything else
·Redid the PCR for psB1C3 using qcF and qcR
·Transformed the DpNI-digested psB1C3mut into competent cells
·DpNI-digested remaining psB1C3 samples
       ·Transformed these samples as well
6/28/2017
·No colonies grew from our transformations yesterday
       ·Realized that this is because we plated them on ampicillin plates, whereas psB1C3mut is actually chloroamphenicol resistant
·Made LB + chloroamphenicol plates, plated transformations of DpNI-cut psB1C3mut into competent cells
·Performed a restriction digest on the PCR product (psB1C3mut) from yesterday using DpNI
·Transformed more DpNI-cut psB1C3mut into competent cells and plated them on LB + chloroamphenicol
6/29/2017
·Nothing grew on the plates! :( :(
·Performed another restriction digest on the psB1C3mut w/ DpNI using protocol from the GeneHackers manual and w/ the NEB restriction buffer (instead of Promega)
·Transformed the psB1C3mut cut w/ DpNI into MachI competent cells using Jason’s protocol
·DpNI-cut and transformed the psB1C3mut
·Plated the transformation on LB + CAM plates
·Nothing grew
·Realized that the CAM we used may have just been a stock
·Made three separate 1:1000 dilutions of the CAM tubes (we think stock) in the refrigerator
·Made LB + CAM plates with these dilutions we made today
·Conducted PCR on the pOW1 w/ ArgArsR and ArgArsF
·Ran a gel w/ the pOW1 PCR product
       ·Did not observe any DNA in the gel
·Conducted two-step amplification (Kasey’s PCR method) for psB1C3 just in case
·Ran a gel of just our pOW1 DNA
       ·Observed that we do have DNA, at least
6/30/2017
·pSB1C3mut grew on the LB + CAM plates
·For some reason negative control grew a lot too (possible that we accidentally plated + control on - plate)
·Made LB + CAM liquid media
·Did the first day step of miniprep using samples 1 and 2 from the 500 microliter CAM plates
July-----
7/2/2017
·Did the second day step of miniprep
       ·Left some bacteria to incubate
·DNA concentrations were rather low, will do it again on Monday
7/3/2017
·Did the second day step of miniprep again
·Cut the DNA collected yesterday (low concentrations) with HpAI and EcoRI
·Made a glycerol stock of the pSB1C3mut bacteria
       ·Put the tube with the bacteria in the -80℃ fridge, in the corner of the green iGEM box
·Ran a gel with the miniprep samples from today cut with HpAI and EcoRI
       ·Nothing showed up
·Transformed the psB1C3mut (cut with DpNI) into competent cells
       ·Spread the bacteria on 500 microliter CAM per 500 mL LB plates
·PCR’d the pOW1 with ArgArsR and ArgArsF primers
       ·Ran a gel with just the DNA from the miniprep (Aman + Guillermo’s and Janice’s) and the pOW1 amplified sequence (the PCR product)
       ·There is smearing in the gel. Next time we will run the gel at a lower voltage to see if that helps the problem
7/4/2017
·The negative control grew which should not have happened. We hypothesized that something’s wrong with our chloramphenicol, and we need to buy new chloramphenicol. The chloramphenicol broth that we made before was also compromised by something, as there are strings of white substance. The broth is also cloudy.
7/5/2017
·Ran two-step and one-step PCR on pOW1 using Pfu Turbo polymerase
·Ran a gel of the pOW1 PCR products (amplified region should be around 2.55 kb)
       
·We think that the first sample (second lane) may contain the ideal pOW1 amplified region
·From nanodrop, this sample (TZ 1) is 491.5 ng/μL
·Jason confirmed that our gel looks good
7/6/2017
·Ran PCR of 13 out of 15 of the centromeric regions
       ·Used the full genomic DNA
·Ran gels, did not receive any results
·Received the chloramphenicol powder
·Made chloramphenicol stock and dilutions
7/7/2017
·Ran another gel of a centromeric region PCR
       ·No results
·Cut psB1C3mut with DpNI
       ·Transformed this DNA w/ MachI competent cells
       ·Spread the transformations (along w/ + and - controls) onto LB + CAM plates ·Cultured pOW1-RFP from glycerol stock in -80℃ fridge
·Made LB + CAM liquid media and LB + CAM plates
·Ran PCR of the INV_1A, INV_2A, INV_3A regions in the Pichia pastors chromosomes (along with a positive control-- the Pichia genomic DNA + primers given to us by Jason which we know already work)
7/8/2017
·Ran a gel of the control given to us by Jason, INV_1A, INV_2A, INV_3A
·A band for INV_1A showed up:
       
·Mini-prepped the RFP gene from pOW1-RFP
·PCR’d the pOW1-RFP
7/10/2017
·Ran a gel of the pOW1-RFP PCR product
·There was a result for the first sample:
       
·Gel purified the band containing RFP
·Redid the PCR for the control, INV_2A, INV_3A (with 55℃ melting temperature instead)
       ·No results, will redo PCR (with 56℃ Tm and 4 minute elongation time) tomorrow
·Digested the pSB1C3mut PCR products with DpNI
·Transformed the pSB1C3mut into competent cells
       ·Plated on LB + CAM
7/11/2017
·There was a lawn on every single plate
·Must be something wrong with the plates still!
·We ordered LB + CAM plates on the Buysite, so hopefully they come today
·PCR’d the control, INV_2A, INV_3A (with 56℃ melting temperature and 4:15 elongation time)
·Gel purified the pOW1 PARS2 and SCARG4 region
       ·Concentration was 4.2 ng/μL
·PCR’d psB1C3 again
7/12/2017
·PCR’d chromosome 1: region 2 and chromosome 1: region 3
·Ran a gel of the control, INV_2A, INV_3A
       ·Only INV_2A showed up
              
·Gel purified INV_2A
·Cut the psB1C3mut with DpNI
·Transformed the DpNI-cut psB1C3mut into competent cells
·Plated the transformants onto the new LB + CAM plates
7/13/2017
·Nothing grew on the plates
·Redid the transformations on the LB + CAM plates we bought
·PCR’d 1:2 and 1:3
7/14/2017
·Nothing grew on the plates again
       ·Will troubleshoot at the meeting today
·Ran a gel of 1:2 and 1:3 PCR products and the pOW1-RFP PCR products again
·Realized that the psB1C3 comes in linearized form
7/15/2017
·Nothing grew on the plates
·Redid the transformation with the second ligation product
7/16/2017
·Nothing grew on the plates :(
·Went to Zumba :)
7/17/2017
·Nothing grew on the plates
·PCR’d the longest centromeric region
7/18/2017
·Nothing worked
·PCR’d another centromeric region, but that didn’t work either
·Transformed psB1C3-RFP into competent cells
       ·Plated on LB + CAM
7/19/2017
·psB1C3-RFP grew on the LB + CAM
       
·Made new LB + CAM liquid media
·Cultured 6 different colonies of psB1C3-RFP in the liquid media
·Purified RFP again from the pOW1-RFP PCR product
       ·DNA concentration = 62.8 ng/uL
·PCR’d the fifth region of chromosome 2
7/20/2017
·Mini-prepped the psB1C3-RFP DNA from the E. coli
·Performed mutagenesis
       ·DpNI digestion
·Transformed and plated the psB1C3-RFPmut DpNI digested DNA
7/21/2017
·Nothing grew on the plates
·Ran a gel of the mini-prepped DNA and the PCR products
·There was a result for the PCR products
·Cut the psB1C3-RFP with NotI
       ·Ran a gel, it didn’t work
7/22/2017
·Cut the psB1C3-RFP with NotI
·Ran a gel, gel extracted the larger fragment
·Ligated
       ·Ran a gel, nothing came up
·Cut and gel extracted again
       ·Transformed directly after ligation, plated on 1:1000 LB + CAM
7/23/2017
·Everything grew in lawns
·Put a small amount of the bacteria in liquid LB + CAM media, hoping that selects for our desired plasmid
7/24/2017
·Everything grew in lawns on the plates
       ·Replated the transformations on 1:500 LB + CAM
·Ran a PCR of region 1 for chromosome 2
       ·No results
·Reran the PCR, left it overnight
7/25/2017
·Everything grew in lawns on the plates
·Mini-prepped the bacteria we put in liquid LB + CAM media
·Did a diagnostic digest using ScaI and PstI
       ·It didn’t work
·Ran a gel of the mini-prepped bacteria
       ·Above 10 kb, we must not have the plasmid of interest
·Transformed the ligated psB1C3 fragment (cut by NotI) into more competent cells
·Plated the transformations on newly delivered (store-made) LB + CAM plates
·Ran a gel of the PCR we conducted last night
       ·It didn’t work
7/26/2017
·Nothing grew on the plates
       ·Redid the transformation procedure
·Did PCR for chromosomal sequence again, no results
7/28/2017
·Received the linearized psB1C3 backbone from iGEM!
       ·Cut with XbaI and SpeI
       ·Ligated back together
       ·Transformed into competent cells
       ·Plated on LB + CAM plates
7/29/2017
·Colonies grew on the plates
·Inoculated colonies
7/30/2017
·Mini-prepped, got 13 ng/uL and 23 ng/uL samples of circularized psB1C3
7/31/2017
·Mutagenized the psB1C3
·Cut with DpNI
·Transformed into competent cells and plated on LB + CAM
·PCR’d region 1 of chromosome 1
August-----
8/1/2017
·Colonies grew on the plates (containing mutagenized psB1C3)!
       
       
·Cultured two single colonies
·Ran a gel for the PCR from last night
       ·It didn’t work
·PCR’d region 1 of chromosome 1 (again)
·Started a culture of Pichia pastoris (in order to practice transformation tomorrow)
8/2/2017
·Mini-prepped the psB1C3mut
·Ran a diagnostic digest using HpaI and EcoRI
·Ran a gel of the PCR product (chromosome 1, region 1)
       ·No results
·PCR’d region 3 of chromosome 1
·Finished the Pichia pastoris transformation, plated the yeast
8/3/2017
·Ran a gel of the PCR product (chromosome 1, region 3)
       ·No results
·PCR’d region 1 of chromosome 1
·Did 4 more site-directed mutagenesis reactions on the re-ligated psB1C3 (in case the ones Steve is midi-prepping didn’t work in terms of being mutated)
       ·Cut with DpNI
·Cut the 4 reactions with HpaI and EcoRI as part of a diagnostic digest
       ·Ran a gel of these diagnostic digests
              
·Transformed the PCR products into bacteria, plated on LB + CAM
8/4/2017
·Colonies grew on the plates
       
       
       
·Cultured single colonies from the plates
·Ran a gel of last night’s PCR
       ·No results
·Cut both the 7/31/17 and 8/3/17 PCR mutagenesis products with HpaI and PvuII
       ·There were results for all 4
              
·This means that we can go ahead with the miniprep tomorrow, and that Steve can go ahead with his midiprep
·Tried some PCRs for the in-vivo assembly
       ·None of them worked
8/5/2017
·Mini-prepped the psB1C3mut backbone
       ·Got concentrations of 7 ng/uL and 17 ng/uL
8/7/2017
·Cut the psB1C3mut backbone with HpaI
·Did a spin-cleanup of this DNA
·Dephosphorylated the DNA using CIP treatment
·Gel purified the CIP-treated DNA
·Ran a Gibson assembly of this cut psB1C3mut backbone (vector) with the ArgArs fragment
·Transformed the Gibson assembly product into E. coli
·Started an overnight culture of Pichia pastoris (for transformation tomorrow)
8/8/2017
·Nothing grew on the plates
·Ran a gel of yesterday’s Gibson assembly product and of the CIP-treated psB1C3mut backbone
September-----
9/2/2017
·Digest of RFP (5ul -> 0.1645ug) and pSD000 (10ul -> 0.225ug) following promega protocol for 20ul total reaction volume
·Incubated digest at 37C for 1hr 15min
·Used QIAquick Nucleotide removal kit to spin purify. Eluted in 40 ul of buffer EB.
·Products stored in DNA box in bench freezer
       ·RFP: 39.8 ng/ul; pSD000: 41.9 ng/ul
·Made LB and YPAUD media, but couldn’t autoclave. Stored on top shelf in cold room (steve autoclaved them later)
9/3/2017
·Used NEB Ligation calculator to determine amount of vector and insert to use ( ) for a 3:1 insert:vector ratio (vector = 4.6kb, insert = 0.77 kb) to give 100 ng (2.4 ul) vector and 50.1 ng (1.3 ul) insert
·Incubated 20 minutes at 4C and 3 hrs at room temp
·Heat inactivation at 70C for 10 minutes
9/30/2017
·Transformed the psD000 + RFP and the psD000 samples located in the fridge into E. coli
·Ligated psD000 with RFP
October-----
10/1/2017
·Nothing grew for the psD000 + RFP
·However, picked psD000 colonies to miniprep tomorrow (because we are running out of psD000
10/2/2017
·Transformed the psD000 + RFP (psD001) we ligated on the 30th into E. coli competent cells
·Miniprepped psD000 (70 ng/uL concentration)
10/3/2017
·Colonies appeared for psD001 but they were not red
·Still inoculated three cultures for miniprep tomorrow
10/4/2017
·Miniprepped the psD001 (107 ng/uL concentration)
10/7/2017
·Prepared a yeast culture for Pichia pastoris transformation tomorrow with psD000
10/8/2017
·Transformed psD000 (but I don’t think it worked, it was sloppy) into Pichia
·Digested psD000 with XbaI, NRK treated and desphosphorylated using CIP treatment
10/10/2017
·The yeast transformation didn’t work
10/18/2017
·Prepared a yeast culture for Pichia pastoris transformation tomorrow with psD000 and psD001
·Received the plasmids from the WUSTL team for our collaboration!
10/19/2017
·Transformed Pichia pastoris with psD000
·Transformed Pichia pastoris with psD001
GeneHackers 2017
