<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Exeter_RNA_Riboswitches2 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Exeter/RNA Riboswitches2</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="wrapper"><NAV class="navbar navbar-default"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON id="collapse-button" type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="span12 offset1 nav navbar-nav"><LI class="dropdown btn-group"><BUTTON type="button" class="btn nav-btn"><A href="https://2015.igem.org/Team:Exeter/Description">PROJECT</A></BUTTON><BUTTON type="button" class="btn dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><SPAN class="sr-only">Toggle Dropdown</SPAN></BUTTON></LI><LI><A href="https://2015.igem.org/Team:Exeter/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Exeter/BovineTB">Bovine TB</A></LI><LI><A href="https://2015.igem.org/Team:Exeter/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Exeter/RNA_Riboswitches">Toehold Background</A></LI><LI><A href="https://2015.igem.org/Team:Exeter/diary">General Diary</A></LI><LI><A href="https://2015.igem.org/Team:Exeter/Future">Future</A></LI></UL><LI class="dropdown btn-group"><BUTTON type="button" class="btn nav-btn"><A href="https://2015.igem.org/Team:Exeter/Toehold_Design">LAB</A></BUTTON><BUTTON type="button" class="btn dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><SPAN class="sr-only">Toggle Dropdown</SPAN></BUTTON></LI><LI><A href="https://2015.igem.org/Team:Exeter/Toehold_Design">Toehold Design</A></LI><LI><A href="https://2015.igem.org/Team:Exeter/Experiments">Experiments</A></LI><LI><A href="https://2015.igem.org/Team:Exeter/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Exeter/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Exeter/Interlab">Interlab</A></LI><LI class="dropdown btn-group"><BUTTON type="button" class="btn nav-btn"><A href="https://2015.igem.org/Team:Exeter/Modeling">MODELLING</A></BUTTON><BUTTON type="button" class="btn dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><SPAN class="sr-only">Toggle Dropdown</SPAN></BUTTON></LI><LI><A href="https://2015.igem.org/Team:Exeter/Modeling#simulation">Simulation</A></LI><LI><A href="https://2015.igem.org/Team:Exeter/Modeling#parameter">Parameter Scanning</A></LI><LI><A href="https://2015.igem.org/Team:Exeter/Modeling#scripts">Scripts</A></LI><LI><A href="https://2015.igem.org/Team:Exeter/Modelling_Diary">Modelling Diary</A></LI><LI class="dropdown btn-group"><BUTTON type="button" class="btn nav-btn"><A href="https://2015.igem.org/Team:Exeter/Practices">BEYOND THE BENCH</A></BUTTON><BUTTON type="button" class="btn dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><SPAN class="sr-only">Toggle Dropdown</SPAN></BUTTON></LI><LI><A href="https://2015.igem.org/Team:Exeter/Practices#introduction">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Exeter/Practices#timeline">What we did</A></LI><LI><A href="https://2015.igem.org/Team:Exeter/Practices#integration">Integration and design</A></LI><LI class="dropdown btn-group"><BUTTON type="button" class="btn nav-btn"><A href="https://2015.igem.org/Team:Exeter/Team">PARTICIPANTS</A></BUTTON><BUTTON type="button" class="btn dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><SPAN class="sr-only">Toggle Dropdown</SPAN></BUTTON></LI><LI><A href="https://2015.igem.org/Team:Exeter/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Exeter/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Exeter/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Exeter/Sponsors">Sponsors</A></LI><LI class="dropdown btn-group"><BUTTON type="button" class="btn nav-btn"><A href="https://2015.igem.org/Team:Exeter/Parts">PARTS</A></BUTTON><BUTTON type="button" class="btn dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><SPAN class="sr-only">Toggle Dropdown</SPAN></BUTTON></LI><LI><A href="https://2015.igem.org/Team:Exeter/Parts#toeholds">Toeholds</A></LI><LI><A href="https://2015.igem.org/Team:Exeter/Parts#reporters">Reporters</A></LI><LI class="dropdown btn-group"><BUTTON type="button" class="btn nav-btn"><A href="https://2015.igem.org/Team:Exeter/Medal_Requirements">MEDALS</A></BUTTON><BUTTON type="button" class="btn dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><SPAN class="sr-only">Toggle Dropdown</SPAN></BUTTON></LI><LI><A href="https://2015.igem.org/Team:Exeter/Medal_Requirements#bronze">Bronze</A></LI><LI><A href="https://2015.igem.org/Team:Exeter/Medal_Requirements#silver">Silver</A></LI><LI><A href="https://2015.igem.org/Team:Exeter/Medal_Requirements#gold">Gold</A></LI></DIV></DIV></NAV><H1>RNA and Riboswitches</H1><P style="font-size: 20px; color: #608341"><B><EM>&quot;In the beginning, RNA was a simple molecule, but over time it has gained many functions. From self-replication, to storing and utilising information, to regulating cellular pathways, it is an example to all molecules...&quot;</EM></B></P><CENTER><TABLE class="buttons"><TBODY><TR><TD><A onclick="$('.RNA, .Regulation, .Ribo, .Toehold').hide('slow');$('.Basic').show('slow'); slider400.reloadSlider();"><DIV id="Basic">
    The Basic Principles
  </DIV></A></TD><TD>  </TD><TD><A onclick="$('.Basic, .Regulation, .Ribo, .Toehold').hide('slow');$('.RNA').show('slow')"><DIV id="RNA">
    The RNA Molecule
  </DIV></A></TD><TD>  </TD><TD><A onclick="$('.Basic, .RNA, .Ribo, .Toehold').hide('slow');$('.Regulation').show('slow')"><DIV id="Regulation">
    RNA in Regulation
  </DIV></A></TD><TD>  </TD><TD><A onclick="$('.Basic, .RNA, .Regulation, .Toehold').hide('slow');$('.Ribo').show('slow')"><DIV id="Ribo">
    General Riboswitches
  </DIV></A></TD><TD>  </TD><TD><A onclick="$('.Basic, .RNA, .Regulation, .Ribo').hide('slow');$('.Toehold').show('slow')"><DIV id="Toehold">
    Toehold Switches
  </DIV></A></TD></TR></TBODY></TABLE>
Click the buttons above to see information on the corresponding topic.
</CENTER><DIV class="Basic"><H2>The Basic Principles</H2><H4>The Basics of DNA:</H4><DIV style="float:left; width:170px; padding: 10px;"><DIV style="width:150px; border: solid 1px black;">
Figure 1: DNA in a linear, double stranded helix structure.
</DIV></DIV><DIV style="float:right; width:520px; padding: 10px;"><DIV style="width:500px; border: solid 1px black;">
Figure 2: Base pairing within a double helix DNA molecule.
</DIV></DIV><P>
DNA (deoxyribonucleic acid) is a biological molecule found in all forms of life, excepting some types of viruses. DNA is known as a polymer molecule, which means that it is made up of many subunits. In DNA, these subunits are known as nucleotides/bases, of which there are four types; adenine (A), thymine (T), guanine (G), and cytosine (C). Each nucleotide in DNA has three main sections; a phosphate group, a deoxyribose sugar, and the nucleotide (either A, T, C or G). These nucleotides are joined together via phosphodiester bonds between the phosphate group of one nucleotide's phosphate group and another nucleotide's deoxyribose sugar to form a phosphate backbone, which makes up the backbone of DNA. The DNA molecule also has direction (i.e. it has a beginning and an end). The beginning is known as the 5' (five prime) end, and the end is known as the 3' (three prime). New DNA bases (nucleotides) join on to the 3' end of the existing DNA molecule. (figure 1).
As well as nucleotides being able to join to adjacent nucleotides via phosphodiester bonds, each type of nucleotide is able to bond to another specific nucleotide perpendicular (at right angles to) the phosphate backbone via H-bonds in a process known as base pairing. In DNA, adenine (A) is able to base pair to thymine (T), and guanine (G) to cytosine (C). Nucleotides which base pair are called complementary, therefore A &amp; T are complementary, as are G &amp; C. In nature, DNA is rarely found as a single strand, instead it is found as a complex of two DNA strands, one wrapped around the other to give the familiar double stranded helix structure associated with DNA. Each DNA strand is anti-parallel and complementary to the other (i.e. their directions are opposite and where one strand has, for example, an A, the other will have a T) (figure 2).
DNA's primary role in cells is to store genetic information. The information stored on DNA molecules refers to the characteristics and functions of a cell, and therefore the entire organism. In multicellular organisms (e.g. animals), each cell contains identical genetic information, however the information which is used depends on the type of cell. For example, cells which make up the eyes will use information corresponding to sight and eye colour, while muscle cells will use information which corresponds to contraction and relaxation of the cells during use.
The genetic information on DNA is stored in discrete units called genes. Each gene contains information which corresponds to at least one characteristic/function of the cell (And therefore the organism), and is encoded in the language of nucleotides. The sequence of nucleotides within a gene (e.g. ATTCTGCTA) is used to produce a specific molecule (normally a protein). This process is described in more detail in the next section; The Central Dogma. (Figure 3).
</P><H4>The Basics of RNA:</H4><DIV style="float:left; width:320px; padding: 10px;"><DIV style="width:300px; border: solid 1px black;">
Figure 4: Adenine (A) and uracil (U) base pairing.
</DIV></DIV>

RNA (RiboNucleic Acid) is similar to DNA in that it is a polymer molecule and made up of nucleotides/bases, but it differs in a few crucial ways. The first is that usually RNA is made up of only a single strand, as opposed to DNA's double stranded structure. The second is that in RNA, the Tyrosine (T) base is not used, instead it is replaced with Uracil (U), so that A pairs with U (figure 4).
This molecule has many, many functions in biology and is absolutely fundamental to life as we know it. It is also the molecule around which the science of our project is based. In the next section (The RNA Molecule), RNA is discussed in further detail.
<H4>The Basics of Proteins:</H4><P>
Proteins have many functions and properties. Proteins are also polymer molecules made up of subunits, but unlike with DNA and RNA these subunits are not bases/nucleotides, they are amino acids. Amino acids are relatively simple molecules which all share a generic structure, but have different functional (R) groups (figure 5). The interactions of the functional groups, both with other functional groups of the same/different proteins, and with other molecules/etc. in its environment, gives the protein its overall function. These functions can range from catalytic speed up the rate of a reaction) to structural (shape/strength of a cell), to virulence (causing disease in a host).</P><H4>The Central Dogma:</H4><P>
In molecular biology, the central dogma explains the flow of genetic information; DNA to RNA to proteins. Essentially, this means that information is stored in the form of DNA (as explained above), converted into RNA, and then used to synthesise proteins. <U>Transcription:</U>
The process of converting DNA to RNA is termed transcription (figure 6). Usually, a type of RNA called messenger RNA (mRNA) is synthesised during transcription using DNA as a template, meaning that the sequence of the RNA molecule is determined by the sequence of the DNA from which it is copied. There are many reasons why an intermediate is required instead of simply using DNA, including:
</P><UL><LI>Protection of DNA: damage to DNA can cause unfavourable mutations so it is safer to use a 'copy' rather than the original,</LI><LI>Regulatory reasons: the presence or absence of RNA can correspond to the presence/absence of the protein which it encodes for, meaning that it can be used to control cellular pathways</LI><LI>Inability of DNA to reach protein machinery: in eukaryotic cells (animals, plants, fungi, etc.), the DNA is separated from the rest of the cell by a nuclear envelope, DNA is unable to pass through this envelope but RNA is able to</LI></UL><DIV style="float:right; width:220px; padding: 10px;"><DIV style="float:right; width:200px; border: solid 1px black;">
Figure 8: tRNA (transfer RNA) secondary structure.
</DIV></DIV><U>Translation:</U><P>
The second part of the central dogma is the synthesis of proteins using mRNA. mRNA is able to encode for amino acids through the use of 'triplets', also known as 'codons'. These are simply three bases on mRNA which corresponds to a single amino acid, of which there are 21 (natural) types (figure 7). For example, the codon AUG codes for the amino acid methionine (M).</P><DIV style="float:left; width:320px; padding: 10px;"><DIV style="width:300px; border: solid 1px black;">
Figure 7: Amino acid codon table.
</DIV></DIV><P>
While codons allow mRNA to encode the amino acid sequence of a protein, they do not explain how this information is used practically. In order to do this, we must look at another type of RNA; tRNA (transfer RNA). As can be seen in figure 8, tRNA has an interesting secondary structure, and two important regions. The first of these regions is the attachment site at the top of the tRNA, which is where a specific amino acid to the tRNA is able to attach. The second region is the anti-codon at the bottom of the molecule. The anti-codon is complementary to the codon for the amino acid which is attached to that tRNA, allowing the tRNA to bind to the mRNA, and hence ensure that the amino acid is added to the sequence in the correct place.</P><DIV style="float:right; width:220px; padding: 10px;"><DIV style="float:right; width:200px; border: solid 1px black;">
Figure 9: A ribosome bound to a strand of mRNA.
</DIV></DIV><P>
There is still one more main part of the translation mechanism which is missing, and that is how the amide bonds between amino acids are formed in order to synthesis the protein. Once again, RNA comes to the rescue, this time in the form of rRNA (ribosomal RNA). The are different types of rRNA, and they come together (along with some proteins) to form a specific complex called a ribosome (figure 9, also pictured in our logo). The ribosome's job is to bind to the mRNA and 'read' along it, ensuring that the correct tRNAs are added at the right time (figure 10)
</P></DIV><DIV class="RNA"><H2>The RNA Molecule</H2><P>
As has been mentioned briefly, RNA is a single stranded, helically structured polymer molecule made up of nucleotides/bases. While it may seem that this structure is simpler than DNA, because it doesn't have all of its bases already paired to its complementary strand means that the RNA's nucleotides are free to base pair in many different ways. For example, the RNA molecule could base pair with itself (figure 1a) or other molecules to form a complex (figure 1b).

The ways in which the RNA bases interact defines the (secondary) structure of the molecule, so therefore the sequence of the RNA molecule defines the structure of the molecule. This means that if a specific RNA structure is required, then it should be able to be achieved by giving the RNA a specific sequence. This is shown in figure 2. The RNA molecule has two sections which are complementary to each other, which can therefore base pair to create a stem region. The bases which are not complementary remain un-paired and create a loop at the top of the stem section. The fact that RNA is able to fold into many types of secondary structures means that it can have a variety of functions, including those of tRNA and rRNA, which were discussed in the previous section.
RNA can also play an important role in regulating cell processes, this will be discussed in detail in the next section.

</P></DIV><DIV class="Regulation"><H2>RNA in Regulation</H2><P>
There are a few ways in which RNA can be involved in regulation. The first of these is also the simplest; the amount of mRNA present. Simply, if there is more mRNA, then more protein will be made, and the pathway in which the protein is involved increases in activity. If there is less mRNA present, then the reverse occurs. While this idea may be simple conceptually, there are many ways in which the amount of mRNA can be controlled. The first is also a simple idea; produce less mRNA from the DNA in the first place. This can be achieved by 'down regulating' the expression of the gene which encodes for the mRNA through inactivation of transcription factors or activation of inhibitors.
Another way in which the amount of mRNA can be controlled is through the degradation of existing mRNA. This can be achieved through the use of a complex called RISC (RNA-induced silencing complex) and dsRNA (double stranded RNA) or shRNA (small hairpin RNA). The mechanism for this is shown in figure 2. Essentially, the dsRNA/shRNA is cleaved in several places to form small dsRNA fragments, now termed miRNA (micro RNA). The miRNA can then bind to the RISC and one strand is digested, causing it to become ssRNA (single stranded). The miRNA can now act as a guide strand and bind to an mRNA which has a complementary section. Once bound, the RISC can cleave the mRNA, inactivating it. This mechanism is found in eukaryotic cells (animals, plants, fungi, etc.).
Another way in which RNA can be involved in regulation is through the formation of riboswitches. These are discussed in detail in the next section.

</P></DIV><DIV class="Ribo"><H2>Riboswitches</H2><P>
Riboswitches can be thought of as a part of an mRNA molecule which is capable of regulating itself. There are many different types, each with slightly different mechanisms, however all types of riboswitches share in common that they have an 'on' and 'off' state, and that this state can be determined by the binding of a (small) molecule, i.e a ligand. Below are some types of riboswitches and the mechanisms by which they 'turn on/off' mRNA.
</P><H4>Transcriptional control:</H4><P>
These types of riboswitches are able to control whether/how much of the mRNA is transcribed (made from the DNA). The way in which is these switches work is  by the formation of either a terminator or anti-terminator secondary structure, depending on whether a ligand is bound or not. Figure 1 shows a general mechanism for this. Briefly, during transcription a ligand can bind and change the conformation of the RNA which has already been synthesised. This conformational change can cause the formation of either a terminator or anti-terminator. If a terminator is formed, then transcription is halted and the full mRNA is not produced, however if an anti-terminator is formed then transcription is able to progress and the full mRNA is able to be made.
</P><H4>Translational control - RBS sequestration:</H4><P>
Riboswitches which control translation allow the full mRNA to be produced no matter what, however the state of the switch decides whether the mRNA is translated into a protein. mRNA contains a ribosome binding site (RBS) to which a ribosome can bind. The ribosome then reads along the RNA until it reaches an AUG codon - a start codon, at which point the protein begins to be synthesised. The riboswitch is able to stop this from happening by taking on a conformation which can sequester the RBS away from the ribosome, and hence inhibit translation/protein synthesis (figure 2).
</P><H4>Translational control - self-cleavage:</H4><P>
These types of riboswitches are similar to the type described above in that they both exhibit control at the translational level, and they both inhibit translation by sequestering the RBS, however this type of riboswitch has a slightly different mechanism. The action of the ligand binding/un-binding changes the conformation of the switch such that a cleavage site is either exposed or hidden. When the cleavage site is exposed, the switch can be cut in that place and release the mRNA, along with its RBS, and allow it to be translated. While the switch is whole, however, the RBS remains hidden (figure 3).
</P><H4>Other types of Riboswitches:</H4><P>
There are many more types of riboswitches than those listed above, and each different riboswitch will have a slightly different mechanism, however from those described above the idea of a riboswitch should be clear.
</P></DIV><DIV class="Toehold"><H2>Toehold Switches</H2><P>
As has been discussed in the previous sections, RNA is an important molecule which is involved in many functions, including cellular regulation. Discussed in some detail in the previous section were riboswitches and the different types and mechanisms of action. For our project we have designed and improved upon a specific type of riboswitch; a toehold switch.
Toehold switches are riboswitches which regulate at the transcriptional level via RBS sequestration, and they are so named for the toehold structure which is an integral part of the riboswitch (figure 1). The basic mechanism of action is that an RNA molecule with a complementary sequence to that of the switch region of the toehold switch binds to the switch and causes the structure to open up, removing the toehold structure and revealing the RBS to allow the ribosome to bind. This mechanism is discussed in further detail below.</P><H4>Green et al. 2014:</H4>

Riboswitches are found in abundance naturally in bacteria and work well as regulators, however synthetic biology is in the business of taking things found in nature and adapting them to our own needs. This is exactly what a research group led by Alexander Green (<A href="http://collinslab.mit.edu/files/cell_green.pdf">Green <EM>et al.</EM> 2014</A>) has done.
Riboswitches have the potential to be amazingly useful tools in a range of areas (discussed in more detail <A href="">To Be Added</A>), however natural riboswitches have a few issues which means that they are not as easy to use as tools as they might be. One of the limitations of these riboswitches is that they tend to have a low dynamic range. The dynamic range can be thought of as the ratio between the high and low levels of a signal - in the case of riboswitches this would be the ratio between the levels of protein controlled by the riboswitch when switch is on vs. off. Typically, natural riboswitches have dynamic ranges of about 55 fold for riboswitches which enhance protein production, and about 10 fold for those which repress production. Another limitation is that natural riboswitches tend to have significant cross-talk (i.e. their activity is able to be altered by more than one input), which can make their specificity relatively low.
Synthetic biology is based on being able to easily engineer biological 'tools' to our own needs, however the structure of some natural riboswitches can make this difficult. Figure 2 shows a generic structure of many natural RNA-binding riboswitches. The region labelled as the 'switch region' is where the trigger RNA binds to activate the riboswitch. As can be seen, there are two regions which give constraints to the switch region sequence. The first of these are that a section near the middle of the switch region must show complementation to the RBS, the second is that the end of the region must follow the YUNR motif. The YUNR motif is a pattern of nucleotides which allows the binding of trigger RNA as either a loop-linear interaction, or a loop-loop interaction (figure 2). These constraints increase the difficulty of engineering these riboswitches as trigger RNAs have the same constraints as the switch region. This can cause cross-talk between riboswitches within the same system as the triggers will have regions which show homology (the same/very similar) due to the same constraints being imposed upon them. If this issue could be worked around, then not only would it make riboswitches easier to engineer, but also reduce the issue of cross talk. In fact, this is exactly what Green <EM>et al.</EM> did.
The research group led by Green designed a toehold switch of the general structure shown in figure 3. As can be seen, the constraints placed upon the switch region in natural riboswitches are no longer present.<H4>Toehold switch mechanism:</H4>
The toehold switch mechanism is similar to any other RNA-binding riboswitch which regulates at the translational level. The trigger RNA binds to the switch region of the toehold in a linear-linear way, causing the toehold structure to open up. This then releases the RBS from the loop, allowing a ribosome to bind it in a linear-linear way. The ribosome can then read along the coding region of the toehold switch, hence giving off a signal. The Green <EM>et al.</EM> 2014 paper shows that toehold switches of this design are able to have dynamic ranges of over 400 (comparable to below 60 for natural riboswitches), and a crosstalk level of below 12%. These changes mean that toehold switches are more suitable for use in synthetic systems.<H4>Our Development:</H4>
These toehold switches show amazing potential in many areas, but we think that the area of diagnostics could greatly benefit from the development of toehold switches. This is why we decided to develop the toehold switches made by Green <EM>et al.</EM>. We plan to develop a standard toehold switch which can be changed in a relatively simple way in order to detect any given RNA, and hence diagnose many different diseases. We also wish to make a set of toehold switches with different indicators, mainly fluorescence, colour change, and luminescence. Another of our goals is to characterise the use of our toehold switches in a cell free system, meaning that unlike in the original paper, bacterial cells will not be required to express the toehold switches therefore removing biosafety issues in a diagnostic application of our switches.</DIV><FOOTER><DIV class="container-fluid"><DIV class="row"><UL><LI style="list-style:none"><SPAN style="text-align:center; padding-top:10px;"> Contact us:  exeterigem@gmail.com </SPAN></LI></UL></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>