<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_OUC-China_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:OUC-China/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_content_wrapper"><TITLE>Team OUC-China: Main</TITLE><BUTTON onclick="topFunction()" id="myBtn" title="回顶部">TOP</BUTTON><DIV id="page-wrapper"><NAV id="nav"><UL><LI><A href="https://2018.igem.org/Team:OUC-China">Main</A></LI><LI><A href="#">Project</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Description">Overview</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Demonstrate">Demonstrate</A></LI></UL><LI><A href="#">Lab WORK</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/InterLab">InterLab</A></LI><LI><A href="#">Model</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Model">Overview</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/miniToe">miniToe</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/miniToe Family">miniToe Family</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/polycistron">polycistron</A></LI><LI><A href="#">Parts</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Parts">List</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Improve">Improve</A></LI><LI><A href="#">HP</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Public_Engagement">Public Engagement</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Team">Team Member</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Notebook">Notebook</A></LI></NAV></DIV><DIV class="sticky"><DIV class="topleft2"><UL class="side-nav" style="width:150px"><LI><A href="#tips1" id="thu">1. BACKGROUND</A></LI><LI><A href="#tips2" id="thu">2. THE FIRST SYSTEM-MINITOE</A></LI><LI><A href="#tips3" id="thu">3. THE SECOND SYSTEM-MINITOE FAMILY</A></LI><LI><A href="#tips4" id="thu">4. THE THIRD SYSTEM-MINITOE POLYCISTRON</A></LI><LI><A href="#tips5" id="thu">5. MINITOE BASED MOTILITY DETECTION SYSTEM</A></LI><LI><A href="#tips6" id="thu">6. IN THE FUTURE</A></LI></UL></DIV></DIV><DIV class="nb"><DIV class="sk"><SECTION class="box features"><H2 class="major"><FONT size="7"><SPAN>Design</SPAN></FONT></H2><H3><FONT size="6">1. Background</FONT></H3><H4><FONT size="5">1.1 Background of miniToe Family</FONT>The endoribonuclease Csy4 from CRISPR family is the main role of miniToe system. Csy4 (Cas6f) is a 21.4 kDa protein which recognizes and cleaves a specific 22nt RNA hairpin. In type I and type III CRISPR systems, the specific Cas6 endoribonuclease splits the pre-crRNAs in a sequence-specific way to generate 60-nucleotide (nt) crRNA products in which segments of the repeat sequence flank the spacer (to target &quot;foreign&quot; nucleic acid sequence) [1]. Inactivation of the Cas proteins leads to a total loss of the immune mechanism function.
The Csy4 protein consists of an N-terminal ferredoxin-like domain and a C-terminal domain. This C-terminal domain is responsible for pre-crRNA recognition and binding. The pre-crRNA target site adopts a stem-loop structure (the specific 22nt RNA hairpin) with five base pairs in A-form helical stem capped by GUAUA loop containing a sheared G11-A15 base pair and a bulged nucleotide U14. In the binding structure of Csy4-RNA complex, the RNA stem-loop straddles the β-hairpin formed by strands β6-7 of Csy4[2]. And once the Csy4/RNA complex formed, the structure will stay stable and hard to separate.		
<DIV align="center"><P>Fig.1-1 The structure of Csy4 
Fig.1-2 The structure 22nt hairpin can be recognized by Csy4 
Fig.1-3 The Csy4/RNA complex.</P></DIV></H4><H3><FONT size="6">2. THE FIRST SYSTEM: MINITOE</FONT></H3><H4><FONT size="5">2.1 NEW METHOD: MINITOE</FONT></H4>								
Based on function of Csy4, we design a new cis-regulatory RNA element named miniToe which can be recognized by Csy4 [3]. The whole system works as a translational activator including three modular parts: 
1. A cis-repressive RNA (crRNA) served as a translation suppressor by pairing with RBS as the critical part of miniToe structure. 
2. A Csy4 site as a linker between cis-repressive RNA and RBS, which can be specifically cleaved upon Csy4 function. 
3. A CRISPR endoribonuclease Csy4.
<DIV align="center"><P>Fig.2-1 The composition of miniToe system.</P></DIV>
In the project, the superfolder green fluorescent protein (sfGFP) is the reporter gene to reflect output of our system under miniToe regulation, the expression of this gene is driven by a constitutive promoter and the RBS near cis-repressive RNA coding region with miniToe design.
Firstly, the stability of miniToe structure is crucial. Hence, before wet experiment, we predicted the structure of full-length transcript of this circuit as well as miniToe structure [4][5].
<DIV align="center"><P>Fig.2-2 The structural prediction of the circuit and miniToe.</P></DIV>
To verify the feasibility and function of miniToe, the following circuits were designed for testing the function of miniToe. An inducible promoter P <I>tac </I> controls the expression level of Csy4. The cis-repressive RNA coding sequence is inserted at the upstream of reporter (sfGFP) gene，which is controlled by a constitutive promoter from Anderson family named J23119.<DIV align="center"><P>Fig.2-3 The two plasmids of miniToe test system.</P></DIV> 
Without Csy4,the crRNA pairs with RBS very well, so the switch just turns off which means that no protein will be produced. Otherwise, with the presence of Csy4, the translation turns on. In this way, the expression of downsteam gene can be regulated.

<DIV align="center"><P>Fig.2-4 The mechanism of miniToe system.</P></DIV> 		

By experiments, the system can work well! <A href="https://2018.igem.org/Team:OUC-China/Results">Click here for more details!</A><DIV align="center"><P>Fig.2-5 The result of first system.</P></DIV><H4><FONT size="5">2.2 The characteristics of miniToe</FONT></H4>
1. The Csy4 protein and miniToe structure have high binding affinity [1]. It shows that the miniToe system may control the state of expression in a digital-like way (ON/OFF). When the switch is at OFF state, the downstream gene expression is tightly blocked, the reaction grows very slowly in the beginning but accelerating rapidly once the Csy4 protein truncates the cis-repressive RNA element.  2. Compared to the small RNA based on riboswitch [6], the insertion of hairpin provides Csy4 with a recognition and cleavage site so that the Csy4 may enhance the steric hindrance effect between cis-repressive RNA and RBS when we need to release the cis-repressive RNA for opening the downstream gene expression, which could promote translation activation.
  3. Compared to the toehold switch [7], miniToe does not need to redesign cis-repressive RNA case by case because the cis-repressive RNA is not paired with protein coding region.				
							<H3><FONT size="6">3. The second system: miniToe family</FONT></H3><H4><FONT size="5">3.1 miniToe family: The models help us go further </FONT></H4>After testing first system, miniToe system, the dry lab members explore deeply!
After building the ODE model, we used it to simulate the dynamics of sfGFP. Compared with the experimental data, it fits perfectly, which indicates that the model is reliable about first system. By analyzing the sensitivity of the sfGFP level in the system to cleavage rate by model, it is not difficult to predict that the cleavage rate has an influence on the expression of sfGFP. It means we may change the expression level of sfGFP if we employ different mutants of Csy4 proteins. At the same time, the wet lab members present the mutation of miniToe structure may also cause changes.

<DIV align="center"><DIV align="center"><P>Fig.3-1 The ODE model for the first system.</P></DIV><DIV align="center"><P>Fig.3-2 The model about sensitivity analysis of the sfGFP.</P></DIV></DIV><H4><FONT size="5">3.2 The principles of designing mutants </FONT></H4>For Csy4 mutants
1. Some key sites in the Csy4 are really crucial for keeping the stability of structure and maintaining the functions of recognition and cleavage. Mutations on those sites may have a serious influence on our system. By point mutation, we hoped to get a library of mutants, which could provide several Csy4 mutant candidates with recognition and cleavage rates shows as a &quot;ladder&quot;. For hairpin mutants
2. We need to avoid breaking the recognition site and keep the function of cleavage relatively stable. If we break the key site G20, it may lead to the damage of cleavage function. 
3. The stability of secondary structure is vital, so we need to focus on each hairpin's Gibbs free energy during design.  
4. The aim is to obtain different hairpins which have various activity for Csy4 recognition and RNA cleavage.	<H4><FONT size="5">3.3 Models help us deeply!</FONT></H4>
 Here a model helps to design the mutants of Csy4 and hairpin. According to the principles of design we mentioned above, four key problems is important in the miniToe model design: 
 
1.Does the Csy4 dock correctly with the miniToe structure?

2.How about the binding capability between the Csy4 and miniToe structure? 

3.How about the capability of cleavage between the Csy4 and miniToe structure? 

4.Does cis-repressive release from the RBS? 

For the Csy4 mutants, the molecular dynamics method was used as our tools. Based on Jiří Šponer’s work [8], we chose four significant symbols of mutants: the interaction matrix; the binding free energy, the distance of Ser151(OG)-G20(N2') and the RMSD of the cleaved-product complex. By comparing the differences between various Csy4 mutants with wild-type Csy4 by above four significant symbols, we finally designed four Csy4 mutants: Csy4-Q104A, Csy4-Y176F, Csy4-F155A, and Csy4-H29A based on model prediction.           
For the hairpin mutants, the bioinformatics and machine learning become our tools. By exploring the commonality of hairpin in the following step: 
 
1. Firstly, we found the hairpin which is also Repeat Area [9] in CRISPR system and scores them.  

2. Secondly, with the help of the SVM algorithm[10], training a model to score the hairpin.  

3. Finally, by using the model, five hairpins are selected: miniToe-1, miniToe-2, miniToe-3, miniToe-4, miniToe-5.                 
<A href="https://2018.igem.org/Team:OUC-China/miniToe_Family">Click here for more details about model! </A><DIV align="center"><P>Fig.3-3 We have selected five Csy4s and six hairpins.</P></DIV>With the help of our model, 4 Csy4 mutants and 5 hairpin mutants are selected. We tested each mutant and got positive data supporting model prediction. Then we set up a function verification experiment with 5*6 combinations of Csy4 and hairpin including wild types. By testing all of them, 10 members work successfully as expectation. So, our second system, miniToe family, has 10 combinations which was designed and selected.	
<DIV align="center"><P>Fig.3-4 The chart shows 30 combinations in experiments</P></DIV><H3><FONT size="6">4. The third system: miniToe polycistron</FONT></H3><H4><FONT size="5">4.1 Why we do </FONT></H4>Many applications of synthetic biology need to regulate the expression profile of multiple genes. Microorganisms with programmable and engineered metabolic pathways are employed as a reaction vessel to natural or unnatural products. It involves the introduction of several genes encoding the enzymes of a metabolic pathway [11][12]. Indeed, pathway optimization requires to adjust the expression of multiple genes at appropriately pattern, for example, the synthetic of poly-3-hydroxybutyrate and Mevalonate [13]. 
		As has been done in the prokaryotes, grouping a cluster of genes into a single polycistron is a convenient method for regulating genes simultaneously. Thus, for the sake of tuning the expression pattern of genes within polycistron, we hope to develop a powerful regulation tool by the miniToe system. We name this system miniToe polycistron which contains several genes in one circuit with different miniToe design. Our aim for this part is achieving different expression profile of the genes by miniToe in polycistrons compared with normal polycistrons. 
	<H4><FONT size="5">4.2 How we do</FONT></H4><DIV align="center"><P>Fig.4-1 The mechanisms of miniToe polycistron
	 </P></DIV> 
By inserting miniToe hairpins between intergenetic regions, it will tune the translation level of corresponding proteins. 1. First, sfGFP and mCherry is used as a test system in bi-cistron circuit.
2. Then we selected some miniToe parts and inserted them between, before and behind sfGFP and mCherry. For example, for bi-cistron, then three miniToe parts will be inserted. for three genes in polycistron, then four miniToe parts will be inserted, and so on. The reasons why we design like this are below: 


		1) Without Csy4, the existence of miniToe structure in polycistron will inhibit the gene expression when we don’t want to open the switch. The cis-repressive RNA in miniToe has complementary sequence of adjacent RNA region (RBS) to prevent the binding of ribosomes.
		2) Also, the RNA secondary structure of miniToe with Csy4 binding keeping at 3’ UTR after Csy4 cleavage is a protection mechanism to prevent RNA degradation. And this function is based on the high stability and affinity between Csy4 and target RNA structure [14].  
		3)The function of each miniToe has specific recognition and cleavage rates, which will make it possible to regulate the gene flexibly. 
		All in all, miniToe polycistron system has two components, Csy4 and the circuit of polycistron.
With Csy4 protein, the polycistron will be cut into several mRNA chains with RNA/Csy4 complex at the 3’ UTR. The capability of RNA degradation protection will be much stronger, because of the high stability and affinity of Csy4 binding, which increase energy threshold for RNA degradation from 3’ UTR. So, the RNA degradation rate will be much lower. For the 5’ end degradation, the Csy4 cut will leave a OH- at 5’ end. the cleavage capability of RNase E will be much lower because there is no pyrophosphate in the 5’ end. Qi’s work [14] has proved that OH-mRNAs exhibit higher gene expression than 5’ PPP-mRNAs. 					
		It is worth mentioning that, our work this year is an improvement based on 2016 OUC-China. The comparison is in demonstrate page, the chapter 5. <A href="https://2018.igem.org/Team:OUC-China/Demonstrate"> Click here to see more! </A><H3><FONT size="6">5. miniToe based Motility detection system</FONT></H3><H4><FONT size="5">5.1 THE APPLICATION OF MINITOE: REGULATION on MOTA</FONT></H4>

As a translation regulation tool, miniToe can also be used in application scenario of molecular mechanism research. Sometimes scientists may puzzle with the functions of certain gene or protein without in-depth study. One general method to study them is knock-out or knock-in methods. In this way, organisms show some phenotypic change related to particular gene. However, if we want to know better about the functions of the gene, we may need more tools to change gene expression at different levels. 
	This year, we used miniToe system to control the motions of <I>Escherichia coli</I>. We transformed the miniToe system into <I>E. coli</I> whose motility is regulated by the motor protein, MotA. MotA provides a channel for the proton gradient required for generation of torque [15]. Δ<I>MotA</I> strains (the <I>motA</I>-deletion strain) can synthesize flagella without function, because they are unable to generate the torque required for flagellar rotation. Expression of <I>motA</I> from a plasmid has been shown to restore motility in Δ<I>motA</I> strains. Our miniToe system is expected to control MotA protein expression with different levels. We got a strain without <I>motA</I> by gene knock-out and then transform <I>motA</I> gene under the control of miniToe family. The <I>E. coli</I> may restore motility under our control and exhibit the application value and potential of our miniToe tool.
	<H4><FONT size="5">5.2 How we do</FONT></H4><I>E. coli</I> RP6666 (knocked out <I>motA</I>)lacks capacity of motion. To use miniToe system for translation control on <I>motA</I>, we constructed a circuit and put <I>motA</I> gene at the downstream of miniToe part, then transformed the plasmid into <I>E. coli</I> RP6666. By inducing the expression of Csy4, <I>motA</I> could be controlled indirectly, thus making <I>E. coli</I> RP6666 strain regain the capacity of motion.
			<DIV align="center"><P>Fig.5-1 The process of motility detection system
	 </P></DIV><H3><FONT size="6">6. In the future</FONT></H3>					
		In conclusion, we have demonstrated the design of modular translational activators with CRISPR endoribonuclease Csy4 named miniToe and design four systems which are improved step by step.	
	  In the future, there are some works to perfect project. First, we would like to enlarge project by finding more and more mutants. By finding and designing more mutants a larger library may be created which can enlarge the function of toolkit. Second, we have tested the ratio of regulation in miniToe polycistron. In the future, man-made setting can be used to make calibration curve which will help us to know project deeply.	
<H3>REFERENCE</H3> 
    
[1].Przybilski R, Richter C, Gristwood T, et al. <I>Csy4 is responsible for CRISPR RNA processing in Pectobacterium atrosepticum.</I>[J]. Rna Biology, 2011, 8(3):517-528.

[2].Haurwitz R E, Jinek M, Wiedenheft B, et al. <I>Sequence- and structure-specific RNA processing by a CRISPR endonuclease</I>[J]. Science, 2010, 329(5997):1355-1358.

[3].Du P, Miao C, Lou Q, et al. <I>Engineering Translational Activators with CRISPR-Cas System</I>[J]. Acs Synthetic Biology, 2016, 5(1):74.

[4].Hofacker I L. <I>Vienna RNA secondary structure server</I>[J]. Nucleic Acids Research, 2003, 31(13):3429-3431.

[5].M. Zuker. <I>Mfold web server for nucleic acid folding and hybridization prediction. </I>Nucleic Acids Res. 31 (13), 3406-3415, 2003.

[6].Mandal M, Breaker R R. <I>Gene regulation by riboswitches.</I>[J]. Nature Reviews Molecular Cell Biology, 2004, 5(6):451-63.

[7].Green A, Silver P, Collins J, et al. <I>Toehold switches: de-novo-designed regulators of gene expression.</I>[J]. Cell, 2014, 159(4):925-939.

[8].Estarellas C, Otyepka M, Koča J, et al. <I>Molecular dynamic simulations of protein/RNA complexes: CRISPR/Csy4 endoribonuclease</I>.[J]. Biochimica Et Biophysica Acta, 2015, 1850(5):1072-1090.

[9].Edgar R C. <I>PILER-CR: Fast and accurate identification of CRISPR repeats</I>[J]. Bmc Bioinformatics, 2007, 8(1):1-6.

[10].Schölkopf B, Tsuda K, Vert J P. <I>Support Vector Machine Applications in Computational Biology</I>[C]// MIT Press, 2004:71-92.

[11].Pfleger, B.F., et al., <I>Combinatorial engineering of intergenic regions in operons tunes expression of multiple genes. </I>Nat Biotechnol, 2006. 24(8): p. 1027-32.

[12].Xu, C., et al., <I>Cellulosome stoichiometry in Clostridium cellulolyticum is regulated by selective RNA processing and stabilization.</I> Nat Commun, 2015. 6: p. 6900.

[13].Smolke, C.D. and J.D. Keasling, <I>Effect of gene location, mRNA secondary structures, and RNase sites on expression of two genes in an engineered operon. </I>Biotechnol Bioeng, 2002. 80(7): p. 762-76.

[14].Qi L, Haurwitz R E, Shao W, et al. <I>RNA processing enables predictable programming of gene expression</I>[J]. Nature Biotechnology, 2012, 30(10):1002.

[15].Ravichandar J D, Bower A G, Julius A A, et al. <I>Transcriptional control of motility enables directional movement ofEscherichia coliin a signal gradient</I>[J]. Scientific Reports, 2017, 7(1).
</SECTION></DIV></DIV><DIV class="copyright1">Contact Us : oucigem@163.com  | ©2018 OUC IGEM.All Rights Reserved. </DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>