Team:OUC-China/polycistron
Team OUC-China: Main
TOP
Main
Project
Overview
Design
Results
Demonstrate
Lab WORK
Experiments
Safety
InterLab
Model
Overview
miniToe
miniToe Family
polycistron
Parts
List
Improve
HP
Human Practices
Public Engagement
Team
Team Member
Collaborations
Attributions
Notebook
polycistron
In our miniToe polycistron system, we build a coupled transcription-translation model considering several events in prokaryotes to get a deep understanding of polycistron. Then we simplify this model into a more flexible model to predict how the miniToe structure changes the relative expression level in polycistron.
1.Current model for polycistron expression system
Before we modeling our system, we firstly give a short review on the model of the polycistron expression system. For the common model, they believe that the mRNA of different cistrons in different positions has the same abundance and if they have the same translation rate, the protein which produced by different cistrons is equal if we do not consider the decay rate of mRNA. But in the truth, the natural polycistron has many strategies to regulate the protein abundance such as the overleap or hairpin in 3’. And for the synthetic polycistron, many things just like the transcription polarity and translation coupling play important roles. Many of them control the protein by control the mRNA abundance. So a more precise model for polycistron is needed[1].
2.The coupled transcription-translation model for polycistron
In this part we will present a coupled transcription-translation model for the polycistron in prokaryotes. The model is based on the Andre S Riberio’s work[2], he presents a coupled transcription-translation model for monocistron. We have done some works to extend the model to use in the polycistron.
2.1 The origin model for monocistron
The origin model build by Andre S Riberio is a stochastic delayed differential equation model in sequence-level, and it can be divided into two mian part: the transcriptional part and the translational part.   The transcriptional part can be described by the following events:
(1)Initiation and promoter complex formation:
Pr
o
+
R
N
A
p
→
R
N
A
p
·
Pr
o
(
τ
O
C
)
(2)Promoter clearance:
R
N
A
p
·
Pr
o
+
U
[
1
,
△
R
N
A
p
+
1
]
→
O
1
+
Pr
o
(3)Elongation:
A
n
+
U
n
+
△
R
N
A
p
+
1
→
O
n
+
1
+
U
n
−
△
R
N
A
p
+
U
R
n
−
△
R
N
A
p
(4)Activation:
O
n
→
A
n
(5)Pausing:
O
n
→
k
p
O
n
p
O
n
←
1
/
τ
p
O
n
p
(6)Pause release due to collision:
O
n
p
+
A
n
−
2
△
R
N
A
p
−
1
→
0.8
k
m
O
n
+
A
n
−
2
△
R
N
A
p
−
1
i
(7) Pause release by collision
O
n
p
+
A
n
−
2
△
R
N
A
p
−
1
→
0.2
k
m
O
n
+
A
n
−
2
△
R
N
A
p
−
1
p
(8)Arrest:
O
n
→
k
a
r
O
n
a
r
O
n
←
1
/
τ
a
r
O
n
a
r
(9)Editing:
O
n
→
k
e
c
O
n
c
o
r
r
e
c
t
i
n
g
O
n
←
1
/
τ
c
O
n
c
o
r
r
e
c
t
i
n
g
(10)Premature termination:
O
n
→
k
p
r
e
R
N
A
p
+
U
[
n
−
△
R
N
A
p
,
n
+
△
R
N
A
p
]
(11)Pyrophosporolysis:
O
n
+
U
n
−
△
R
N
A
p
−
1
+
U
R
n
−
△
R
N
A
p
−
1
→
k
p
y
r
o
O
n
−
1
+
U
n
+
△
R
N
A
p
−
1
(12)Completion:
A
n
l
a
s
t
→
k
f
R
N
A
p
+
U
[
n
l
a
s
t
,
n
l
a
s
t
−
△
R
N
A
p
]
(13) mRNA degradation:
A
n
l
a
s
t
→
k
d
r
ϕ
In the 13 reaction equations above, the Pro stands for the promoter region, the RNAp is RNA polymerase while the Pro-RNAp stands for the promoter which is occupied by the RNA polymerase. An, On and Un are standing for the n th nucleotides in the stage of activated, occupied and unoccupied. U[strat,end] stands for the nucleotides in the range from start number to end number in index. Onp, Onar and Oncorrecting represents the a paused, arrested and error correcting at position n. And due to the temporal steric, the RNAp will occupied about (2ΔRNAp+1) nucleotides.  URn denotes transcribed ribonucleotides which are free.
The translation part can be described by the following events:
(1)Initiation:
R
i
b
+
U
R
[
1
,
n
+
△
R
i
b
+
1
]
→
k
t
r
a
n
s
_
i
n
i
t
O
1
R
+
R
i
b
R
(2)Stepwise translocation:
A
n
−
3
R
+
U
R
[
n
+
△
R
i
b
−
3
,
n
+
△
R
i
b
−
1
]
→
k
t
m
O
n
−
2
R
O
n
−
2
R
→
k
t
m
O
n
−
1
R
O
n
−
1
R
→
k
t
m
O
n
R
+
U
R
[
n
+
△
R
i
b
−
2
,
n
+
△
R
i
b
]
(3)Activation:
O
n
R
→
k
t
r
a
n
s
[
A
,
B
,
C
]
A
n
R
(4)Back-translocation:
O
n
−
1
R
+
U
R
[
n
+
△
R
i
b
−
2
,
n
−
△
R
i
b
]
→
k
b
t
A
n
−
3
R
+
U
R
[
n
+
△
R
i
b
−
3
,
n
+
△
R
i
b
−
1
]
(5)Drop-off:
O
n
R
→
k
d
r
o
p
Rib+
U
R
[
n
−
△
R
i
b
,
n
+
△
R
i
b
]
(6)Trans-translation:
R
→
k
u
[
R
i
b
R
]
R
i
b
(7)Elongation completion:
A
R
n
l
a
s
t
→
k
u
[
R
i
b
R
]
R
i
b
+
U
[
n
l
a
s
t
,
n
l
a
s
t
−
△
R
i
b
]
R
+
P
prem
(8)Folding and activation:
P
prem
→
k
f
o
l
d
P
(9)Protein degradation:
P
→
k
d
e
c
ϕ
In the 8 reaction equations above, the Rib stands for the free ribosome while the RibR represents to the ribosome which is binding to the RNA chain.
△
R
N
A
p
represents to the footprint of ribosome. Every ribosome will occupied about (
2
△
R
ib+1
) nucleotides.  URn , ORn and  ARn are the ribonucleic equivalent fo Un, On and An in transcriptional part, which has similarity meaning.
2.2 The model we improve for the polycistron
Fig.2-1 the organization of operon
Now we have known the coupled transcription-translation model for monocistron. In order to extend it to use in the bi-cistron like Fig.2-1, we simply add another translational part into the old model. So our new model has one translational part and two translational parts for two CDSs in the bi-cistron. Now the most important things are to build the relationship between the two translational part.
The first thing we need to reconsider that is to recalculate the initiation translation rate for the second CDS because this parameter is influenced by translation coupling.
For the translate rates of the second CDS, k2 , can be calculated by the following formula in statistical thermodynamics[3]:
k
2
∝
r
r
e
i
n
i
t
i
a
t
i
o
n
(
2
)
+
e
−
β
⋅
Δ
G
t
o
t
a
l
(
2
)
The formula is divided into two parts to describe the transcript coupling. The first part, rreinitiation(2) , showing that the ribosome terminates the translation of upstream CDS then dissociate and re-initiate the translation of downstream CDS, is called the ribosome re-initiation. The second part, e-βΔGtotal , showing that the elongate along the upstream CDS and unfolding the mRNA structure which increases the expression of the upstream CDS is called de novo ribosome initiation. The two kinds of initiation can be seen in the Fig.2-2.
Fig.2-2 two kinds of initiation
The first part in formula can be calculated by the following formula:
r
r
e
i
n
i
t
i
a
t
i
o
n
(
2
)
=
k
p
⋅
k
r
e
i
n
i
t
i
a
t
i
o
n
(
d
1
,
2
)
⋅
e
−
β
⋅
Δ
G
t
o
t
a
l
(
1
)
Where the kreinitiation(d1,2)                                         refers to the intergenic distance dependence and the kp refers to the proportionality constant between the ribosome assemble rate and the translation initiation rate.
For the  kreinitiation(d1,2) is proved that can be calculate by the formula following:
k
r
e
i
n
i
t
i
a
t
i
o
n
(
d
)
=
{
0
.0072
±
0
.0048
0
≤
d
≤
25
0
.0220
d
=
−
4
0
.0072+0
.0004
⋅
(
d
+
10
)
−
10
≤
d
≤
−
25
Where the
d
=
x
s
t
a
r
t
−
x
s
t
o
p
−
3
,  xstart refers to the first nucleotides in
j
th CDS’s start codon while the   xstop refers to first nucleotides in i th CDS ‘s stop codon.  And it also points that the                                         kp=10  .
The
Δ
G
t
o
t
a
l
(
1
)
refers to the total binding free energy between the ribosome and 5’UTR, according to the equation:
Δ
G
t
o
t
a
l
(
1
)
=
Δ
G
m
R
N
A
−
r
R
N
A
+
Δ
G
s
p
a
c
i
n
g
+
Δ
G
s
t
a
r
t
+
Δ
G
s
tan
d
b
y
−
Δ
G
m
R
N
A
The
Δ
G
m
R
N
A
−
r
R
N
A
refers to the free energy of folding for mRNA-rRNA complex, which is negative.
The
Δ
G
s
p
a
c
i
n
g
refers to the free energy for the non-optimal physical distance between SD sequence and the start codon, which is positive.
The
Δ
G
s
t
a
r
t
refers to the free energy for
t
R
N
A
f
M
E
T
-start codon complex, which is negative.
The
Δ
G
s
tan
d
b
y
refers to the free energy for the , which is negative.
The
Δ
G
m
R
N
A
refers to the free energy of folding for 5’UTR, which is negative.
All these energies can be calculated by the NUPACK suit of energy with mFold 3.0 RNA energy parameter. And five energy can be seen vividly in the Fig.2-3.
Fig.2-3 the five part of the total binding free energy
The second part in formula can be calculated by the following formula:
Δ
G
t
o
t
a
l
(
2
)
=
Δ
G
m
R
N
A
−
r
R
N
A
+
Δ
G
s
p
a
c
i
n
g
+
Δ
G
s
t
a
r
t
+
Δ
G
s
tan
d
b
y
−
Δ
G
n
o
n
c
o
u
p
l
i
n
g
−
Δ
G
c
o
u
p
l
i
n
g
F
c
o
u
p
l
i
n
g
Where the
Δ
G
c
o
u
p
l
i
n
g
refers to the free energy of folding for all inhibitory RNA structure that block the standby site, overlap with SD sequence, spacer region or the downstream footprint region of ribosome.
The
Δ
G
n
o
n
c
o
u
p
l
i
n
g
refers to the free energy of all the other RNA structure except the inhibitory RNA structure. And the
F
c
o
u
p
l
i
n
g
can be calculated by the following formula:
F
c
o
u
p
l
i
n
g
=
1
1
+
C
⋅
e
β
⋅
Δ
G
t
o
t
a
l
(
1
)
Which the
C
is the ribosome-assisted unfolding coefficient.
C
=
0.81
in our study.
The second thing we need to reconsider that is to recalculate the premature termination rate for the second CDS because this parameter is influenced by transcription polarity. Even there is only one transcriptional part, the premature termination rate of the second CDS is higher than the first CDS because the Rho factor will bind to RNA in the intergenic regions to cause rho-dependent termination. And almost 80% of premature termination is caused by Rho-dependent termination. We are going to use the Queuing Theory to build a model to describe it but the model hasn’t finished yet.
Here I will give the most essential idea of the unfinished model. We know the original model of queuing theory has assumed that the two cars will not have an accident and no car try to jump the queue. But we will regard the Rho-factor as the car which jumps the queue and the interaction between Rho-factor and RNA polymerase will be described as the accident. Once the waiting time for the first ribosome is increasing, it will be regarded as the happened of Rho-dependent termination.  So by breaking the two basic assumptions of queuing theory, we will have the description of transcriptional polarity. The Rho-dependent termination rate will be calculated as the expectation of the accident times per minutes while the new premature termination rate will be described as the sum of the original premature termination rate and the Rho-dependent termination rate.
2.3 Explore the mRNA abundance using the model
We constructed a polycistron which has two LacZ gene in the model. And then correcting the premature termination and premature termination rate of the second CDS. The other parameters can be found in Andre S Riberio’s work. And carrying out the simulation in StochPy and SGNSim. We get the distribution of mRNA at t=100s and t=600s in Fig2-4 and Fig 2-5.
Fig.2-4 the distribution of mRNA at t=100s
Fig.2-5 the distribution of mRNA at t=600s
The mRNA distribution in 100s stands for the origin time of mRNA distribution while the mRNA distribution in 600s stans for the final state of mRNA distribution. Fig.2-4 shows that the mRNA of first CDSs is produced while the mRNA of the second CDSs haven't been produced now. Which shows that the mRNA of different cistrons in different positions has different translational time. Fig.2-5 shows that the there is a decrease between the mRNA abundance between two CDSs which suggest that the mRNA of different cistrons in different positions has different abundance.
3.A flexible model for polycistron
We have got some points from the coupled transcription-translation model: the mRNA of different cistrons in different positions has different abundance. This phenomenon may be caused by premature termination or something others, and this will result in the different protein level. And different protein level also caused by the different translation time.
The coupled transcription-translation model is too complex and hard to operate. Here we propose a framework to explain the more realistic situation happened in the polycistron but still hope it also can keep the simple forms.
Considering a polycistron like the Fig.2-1 shows, it contains a promoter, a 5’UTR and two CDSs which is separated by the intergenic regions, and following a terminator in the end.
The reaction can be described following four main steps:
(1)The transcription of two CDSs region:
→
k
1
m
R
N
A
1
→
k
2
m
R
N
A
2
Here we divided the polycistron into two part with different transcription paraments
k
1
,
k
2
to deal with the problem of different mRNA abundance due to the premature termination. The two paraments
k
1
,
k
2
is totally a sequence-dependent as we discuss before.
(2)The degradation of mRNA
m
R
N
A
1
→
k
d
1
ϕ
m
R
N
A
2
→
k
d
2
ϕ
The degradation of RNA also divided into two parts with different transcription paraments
k
d
1
,
k
d
2
to deal with the problem of different translational time for two mRNA. Each
k
d
i
can be divided into two part:
k
d
i
=
k
d
−
k
recoup
The
k
recoup
denotes the recoup item for the translational time difference and the
k
d
denotes the common degradation rate of mRNA.
(3)The translation of protein.
m
R
N
A
1
→
k
p
1
m
R
N
A
1
 + Protein
1
m
R
N
A
2
→
k
p
2
m
R
N
A
2
 + Protein
2
In the translation of two proteins, the two paraments used to describe the translation also should be different considering the translation coupling. And we build a thermodynamic model to calculated it before in the 2.2, now we will give a conclusion for it.
For the translate rates of the first CDS, k1 , can be calculated by the following formula in statistical thermodynamics :
k
1
∝
e
−
β
⋅
Δ
G
t
o
t
a
l
(
1
)
For the translate rates of the second CDS, k2 , can be calculated by the following formula in statistical thermodynamics:
k
2
∝
r
r
e
i
n
i
t
i
a
t
i
o
n
(
2
)
+
e
−
β
⋅
Δ
G
t
o
t
a
l
(
2
)
The calculation method is discussed in 2.2.
(4)The degradation of proteins
Protein
1
→
k
d
p
1
ϕ
Protein
2
→
k
d
p
2
ϕ
We don’t have too much discussion in the proteins degradation here. But we still have something to remind. As the protein we used for our study, sfGFP and RFP, are quite stable exactly. The degradation rate almost equals to the dilution due to the growth diffusion of E. coli colonies.
This model is the simple forms of the coupled transcription-translation model, it keeps the easy form but also reflect the common phenomenon which will happen in the transcript and translation of polycistron including transcript polarity and translation coupling.
4.How the miniToe structure affect the ratio of two CDSs
Fig.4-1 The working processing of miniToe
miniToe polycistron system has two components, Csy4 and the circuit of polycistron. With Csy4 protein, the polycistron will be cut into several mRNA chains with RNA/Csy4 complex at the 3’ UTR as the Fig.4-1 showing. The capability of RNA degradation protection will be much stronger, because of the high stability and affinity of Csy4 binding, which increases the energy threshold for RNA degradation from 3’ UTR. So, the RNA degradation rate will be much lower. For the 5’ end degradation, the Csy4 cut will leave an OH- at 5’ end. the cleavage capability of RNase E will be much lower because there is no pyrophosphate in the 5’ end. Qi’s work has proved that OH-mRNAs exhibit higher gene expression than 5’ PPP-mRNAs.
So by inserting different hairpin which has different binding ability into the miniToe site, we can control the half-life time of mRNA for two CDS in bi-cistron.
5.The main role of cleavage rate for miniToe polycistron
In our miniToe system, the cleavage rate plays an important role in different regulation level. But in the miniToe polycistron system, the cleavage will not influence our goals that change ratio of two CDSs. Now we will give a simple model which based on the OUC-China 2016 [4] to explain the relationship between the cleavage rate and the ratio of two proteins in stable level.
We know that the cleavage of Csy4 can open the switch and the cleavage can affect the product curve which we have been prove in the model work for first and second system. Now we will discuss how does the cleavage rate effect the ratio of two CDSs.
The polycistron system model that OUC-China 2016 constructed is describe by the following reaction:
(1)Translation of polycistron mRNA :
→
k
r
m
R
N
A
(2)The translation of polycistron mRNA:
m
R
N
A
→
K
p
1
m
R
N
A
+
Pr
o
t
e
i
n
1
+
Pr
o
t
e
i
n
2
(3)Cleavage by Enzyme:
m
R
N
A
→
K
d
1
m
R
N
A
1
+
m
R
N
A
2
(4)The translation of two cleaved mRNA
m
R
N
A
1
→
K
p
11
m
R
N
A
1
+
Pr
o
t
e
i
n
1
m
R
N
A
2
→
K
p
12
m
R
N
A
2
+
Pr
o
t
e
i
n
2
(5)The degradation of all species
m
R
N
A
→
K
d
0
ϕ
m
R
N
A
1
→
K
d
11
ϕ
m
R
N
A
2
→
K
d
12
ϕ
Pr
o
t
e
i
n
1
→
K
d
p
1
ϕ
Pr
o
t
e
i
n
2
→
K
d
p
2
ϕ
We use the parameters in following table estimated by the OUC-China 2016:
Symbol
Definition
Value
Kr
The constant of Transcription
100
Kp1
The translation rate of polycistron
67.51
Kd1
The cleavage rate of Enzyme
0.185
Kp11
The translation rate of mRNA1
4.005
Kp12
The translation rate of mRNA2
4.005
Kd0
The degradation rate of polycistron mRNA
17.625
Kd11
The degradation rate of mRNA1
18.864
Kd12
The degradation rate of mRNA2
21.816
Kdp1
The degradation rate of protein1
0.0625
Kdp2
The degradation rate of protein2
0.01
By the reaction equation and parameters above, we explore the relationship between the cleavage rate and the ratio of two proteins in the stable level. Fig.5-1 is the result.
Fig.5-1 The relationship between the cleavage rate and the ratio of two proteins in stable level.
From the Fig.5-1, we can see that the ratio of two proteins in stable level change little with the changing cleavage rate in a large range, which prove the word we mentioned before.
In conclusion, the cleavage rate in our miniToe polycistron plays an important role in changing the shape of the product curve while having little effect in the ratio of two proteins in a stable level.
6. The provement of miniToe System
According to the all the model we discussed before, we can have a conclusion that our miniToe polycistron can be divided into two parts of the miniToe system we discuss in the model work for our first system. The reasons are as following
1) Firstly, we have proved that the cleavage rate in our miniToe polycistron plays a role in changing the shape of the product curve while having little effect on the ratio of two proteins in the stable level. Which give a base to used the polycistron model we discussed in Chapter 3.
2) Secondly, in the flexible model which is simplified from coupled transcription-translation model, we treat the two cistrons as two parts in order to a precious description of the dynamics of polycistron.
According to the analyse below, our miniToe system can be easily modeled as the first system. So we can also prove our miniToe structure’s function by giving the sensitivity analysis of our first system in Fig.6-1.
Fig.6-1 the sensitivity analysis of our first system
As we can see in the Figure.6-1, the parameter kd3 also sensitive to GFP level which means that you can change it to influence the product. By choosing the hairpin which can increase the kd3, we can design the ratio of expression level for two cistrons in polycistron.
7. Future Work
1)To explore the relationship between free binding energy of Csy4 and mRNA degradation rate.
2)To finish the Queuing Theory for the calculation of Premature termination rate.
PS. I really want to finish this two parts before the wiki freeze permanently but this is quite hard for one person. I hope I can finish it as an after-iGEM work for the model part. And anyone who interests in this model can contact me with e-mail: chonpcaacpnohc@gmail.com. This e-mail address also for the correction of my model mistake.
8.Reference
[1]. Quax T E, Wolf Y I, Koehorst J J, et al. Differential translation tunes uneven production of operon-encoded proteins[J]. Cell Reports, 2013, 4(5):938-944.
[2].Mäkelä J, Lloyd-Price J, Yli-Harja O, et al. Stochastic sequence-level model of coupled transcription and translation in prokaryotes[J]. Bmc Bioinformatics, 2011, 12(1):121.
[3]. Tian T, Salis H M. A predictive biophysical model of translational coupling to coordinate and control protein expression in bacterial operons.[J]. Nucleic Acids Research, 2015, 43(14):7137-7151.
[4]. https://2016.igem.org/Team:OUC-China/Model
Contact Us : oucigem@163.com  | ©2018 OUC IGEM.All Rights Reserved.
