<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_OUC-China_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:OUC-China/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_content_wrapper"><TITLE>Team OUC-China: Main</TITLE><BUTTON onclick="topFunction()" id="myBtn" title="回顶部">TOP</BUTTON><DIV id="page-wrapper"><NAV id="nav"><UL><LI><A href="https://2018.igem.org/Team:OUC-China">Main</A></LI><LI><A href="#">Project</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Description">Overview</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Demonstrate">Demonstrate</A></LI></UL><LI><A href="#">Lab WORK</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/InterLab">InterLab</A></LI><LI><A href="#">Model</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Model">Overview</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/miniToe">miniToe</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/miniToe Family">miniToe Family</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/polycistron">polycistron</A></LI><LI><A href="#">Parts</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Parts">List</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Improve">Improve</A></LI><LI><A href="#">HP</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Public_Engagement">Public Engagement</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Team">Team Member</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:OUC-China/Notebook">Notebook</A></LI></NAV></DIV><DIV class="nb"><DIV class="sk"><SECTION class="box features"><H2 class="major"><SPAN>Overview</SPAN></H2><H3><FONT size="6" color="#008B45">Achievements</FONT></H3><FONT size="4" color="black">
★1. Models played an important role in our project. For the first system, we built an ODE model. By analyzing  the sensitivity of sfGFP in first system, we found that the cleavage rate has an influence on the expression level of sfGFP.So we have the ideas to design some Csy4 and miniToe mutants. Then the model guides the design the Csy4 mutants and hairpin mutants. In the last system we build a coupled transcription-translation model considering several events in prokaryotes to get a deep understanding of polycistron.
★2. The models in the first and second systems were designed by ourselves independently.The third model about  miniToe polycistron is an improvement based on Andre S Ribeiro's work.
★3. We documented our model's contributions to our project on our <AHREF> Model page, including our assumptions, relevant data, model results, and a clear explanation  of our model.The most important part is that we explained the colse relationship between the wet lab work and  dry lab work in our <AHREF> Designpage.</AHREF></AHREF></FONT>





The aim of our project is to develop a better post-transcriptional regulation strategy and use it in monocistron and polycistron. Here we built models to design and predict our work.


                                        
<H3>miniToe —— a better transcriptional regulate strategy </H3>
    To achieve a better post-transcriptional regulation strategy, we designed a system which is composed of an RNA endoribonuclease (Csy4) and an RNA module named miniToe. We modeled to describe the dynamics of the miniToe system and found a way to achieve different regulation level. The ODEs and molecular dynamics were two main tools to explore it. We used the ODEs to describe the reaction curve and the molecular dynamics in order to give some explanations for experimental data.
     
    Below you can follow the several questions we point out to have a better understanding of model work and the miniToe system. We will discuss some structures of Csy4 in different stages (<B>Q1</B>), some structures of miniToe system in different stages (<B>Q2</B>), the reaction order and some key points of miniToe system (<B>Q3</B>), the simulation of ODEs model (<B>Q4</B>), some significant symbols in molecular dynamics (<B>Q5</B>) and the ways to different regulation levels (<B>Q6</B>).
<H4> Q1: What is the structure of Csy4? </H4><SECTION class="block"><DIV class="case-block"><DIV>click to see more</DIV><DIV>click to hide</DIV></DIV><DIV class="detail">
      The endoribonuclease Csy4 from CRISPR family is the main role of the miniToe system. Csy4 (Cas6f) is a 21.4 kDa protein which can recognize and cleave a specific 22nt RNA hairpin which consists of an N-terminal ferredoxin-like domain and a C-terminal domain. The later domain constitutes most of the recognition interactions with the RNA. The RNA adopts a stem-loop structure (the specific 22nt RNA hairpin) with five base pairs in A-form helical stem capped by GUAUA loop containing a sheared G11-A15 base pair and a bulged nucleotide U14. In the binding structure of Csy4-RNA complex, the RNA stem-loop straddles the β-hairpin formed by strands β6-7 of Csy4. The Fig.1 and Fig.2 shows two structure of Csy4: with and without the hairpin bound.<DIV align="center"><P>Fig.1 The structure of Csy4 without the hairpin bound. (PDB ID: 4AL5, resolution 2.0 A)</P></DIV><DIV align="center"><P>Fig.2 The structure of Csy4 with the hairpin bound. (PDB ID: 4AL5, resolution 2.0 A)</P></DIV></DIV></SECTION><H4> Q2: What is the structure of miniToe? </H4><SECTION class="block"><DIV class="case-block"><DIV>click to see more</DIV><DIV>click to hide</DIV></DIV><DIV class="detail">
        Based on the function of Csy4, we designed a new cis-regulatory RNA element named miniToe, which can be recognized by Csy4. The whole system works as a translational activator including three modular parts: 

1. A cis-repressive RNA (crRNA) serves as a translation suppressor by pairing with RBS and therefore constitutes the critical part of the miniToe structure. 
2. A Csy4 site serves as a linker between cis-repressive RNA and RBS, which can be specifically cleaved by Csy4 enzyme. 
3. Csy4 enzyme --- A CRISPR endoribonuclease.
<DIV align="center"><P>Fig.3 The structure of miniToe.</P></DIV>      
        
Fig.4 and Fig.5 blow show the two complex of miniToe structures: with and without specific site of hairpin cleaved, which is called the precursor complex and product complex respectively.
        <DIV align="center"><P>Fig.4 The precursor complex of wild-type Csy4.</P></DIV><DIV align="center"><P>Fig.5 The product complex of wild-type Csy4.</P></DIV></DIV></SECTION><H4> Q3: What is the reaction process and key points of miniToe system? </H4><SECTION class="block"><DIV class="case-block"><DIV>click to see more</DIV><DIV>click to hide</DIV></DIV><DIV class="detail"><DIV align="center"><P>Fig.6 The working process of miniToe system.</P></DIV> 						
  All the reaction happened in our miniToe system can be described chronologically by following five main steps in Fig.6:
(1) The miniToe is produced and accumulated.
 (2) Csy4 is produced after induced by IPTG.
 (3) Csy4 binds to the miniToe structure and forms the Csy4-miniToe complex.
 (4) Csy4 cleaves the special site and divides the miniToe structure into two parts: the Csy4-crRNA complex and the mRNA of sfGFP.
 (5) sfGFP is produced.
   From the description above, we can get four key problems in our system to make sure whether our system can work successfully:
 (1) Can Csy4 dock correctly with the miniToe structure (hairpin)?
 (2) How about the ability of binding between the Csy4 and miniToe structure (hairpin)?
 (3) How about the ability of cleavage between the Csy4 and miniToe structure (hairpin)?
 (4) Can cis-repressive RNA be released from the RBS successfully?</DIV></SECTION><H4> Q4: How about the simulation results of the ODEs model? </H4><SECTION class="block"><DIV class="case-block"><DIV>click to see more</DIV><DIV>click to hide</DIV></DIV><DIV class="detail">According to the work process, we built an ODEs model and simulated our miniToe system for 30h, the result can be seen in the Fig.7.
	  
	  <DIV align="center"><P>Fig.7 The dynamics of sfGFP by model prediction</P></DIV> 	
  We compared the experimental data to the simulation result, find it fit perfectly as Fig.8 shows.
        <DIV align="center"><P>Fig.8 The comparison between experimental data and simulation data.</P></DIV><A href="https://2018.igem.org/Team:OUC-China/miniToe" style="font-size: 25px; color: blue; text-decoration:none;">  To see more details </A></DIV></SECTION><H4> Q5: How about simulation result of the molecular dynamics? </H4><SECTION class="block"><DIV class="case-block"><DIV>click to see more</DIV><DIV>click to hide</DIV></DIV><DIV class="detail">
      The main work our molecular dynamics is to give an explanation for the experimental data at atom level and the four key points we have discussed before. The experiment has proved that miniToe is working well, which means the four key points work well too. We will present you the result of molecular dynamics following the four key points.
  For the first key point, we have the interaction matrix to describe the molecular docking, and the heatmap of the matrix can be seen in Fig.9.
<DIV align="center"><P>Fig.9 The heatmap of interaction matrix for wild-type Csy4.</P></DIV>
 For the second key point, we calculated the binding free Energy of Csy4/RNA complex. The result of binding free energy for wild-type Csy4 is     <MATH><MROW><MO>△</MO><MSUB><MI>G</MI><MROW><MI>b</MI><MI>i</MI><MI>n</MI><MI>d</MI><MI>i</MI><MI>n</MI><MI>g</MI></MROW></MSUB><MO>=</MO><MO>−</MO><MN>59154.9251</MN><MI>k</MI><MI>j</MI><MO>/</MO><MTEXT>mol</MTEXT></MROW></MATH>  h
.

 For the third key point, we checked the distance of Thr151(OG)-G20(N2’), which is a key interaction in the active site of Csy4 to describe the ability of cleavage. The distance curve of Thr151(OG)-G20(N2’) for wild-type Csy4 can be seem in Fig.10. 
<DIV align="center"><P>Fig.10. The distance of Thr151(OG)-G20(N2’) in wild-type Csy4.</P></DIV>        

For the last key point, we used the RMSD of product to describe the release of crRNA. The result can be seen in the Fig.11.
        
<DIV align="center"><P>Fig.11. the RMSD of product complex for wild-type Csy4</P></DIV> 
        
The RMSD is unstable and give an explain to experiment that crRNA is release from RBS.
<A href="https://2018.igem.org/Team:OUC-China/miniToe_Family#JC1" style="font-size: 25px; color: blue; text-decoration:none;">  To see more details </A></DIV></SECTION><H4> Q6: How to achieve the goal of different regulate level? </H4><SECTION class="block"><DIV class="case-block"><DIV>click to see more</DIV><DIV>click to hide</DIV></DIV><DIV class="detail">
       When we made the sensitivity analysis for our ODEs model, we found that the cleavage rate of Csy4 can influence the expression in GFP, which indicated that once we changed the wild-type Csy4 into some mutation, we can achieve the different expression of GFP. The Fig.12 shows what will happen in the GFP expression curve if we change the cleavage rate of Csy4. It can achieve the goal of different level of regulation.<DIV align="center"><P>Fig.12 The curve of sfGFP with the changing cleavage rate.</P></DIV><A href="https://2018.igem.org/Team:OUC-China/miniToe#MM" style="font-size: 25px; color: blue; text-decoration:none;">  To see more details </A></DIV></SECTION><H3> miniToe Family —— The way to fifferent regulate level 
      In the miniToe family, the protein and hairpin were mutated to meet the goal of the different regulation level. In this part, the model can help us design mutants. Importantly, we used different strategies to design the feature of Cys4 and the hairpin. For example, molecular dynamics played an important role in designing protein mutants, and the bioinformatics and machine learning supported us to find the hairpin mutants of our interest.

  
     So here comes the another four key questions: how to design our Csy4 mutants (<B>Q7</B>), how does the method work in design and the result (<B>Q8</B>), the difference between Csy4 designing and hairpin mutants designing and how to solve it (<B>Q9</B>) and the results of the mutants designing (<B>10</B>).
      
</H3><H4> Q7: How to design the Csy4 mutants? </H4><SECTION class="block"><DIV class="case-block"><DIV>click to see more</DIV><DIV>click to hide</DIV></DIV><DIV class="detail">  We found four important sites in wet lab, Gln104, Tyr176, Phe155 and His29, which play important roles in binding and cleavage in Csy4 structure.(Fig.13) Considering the existing 20 amino acids in nature, there were 80 mutants to be explored if we had only one site to be mutated.
<DIV align="center"><P>Fig.13 The four important sites in Csy4. </P></DIV>  In <B>Q3</B>, we have discussed four key points which can directly influence our miniToe system. In addition, according to the molecular dynamics results in <B>Q5</B>, we can describe the four key points through four significant symbols. 
        
Now we are going to construct a logic line to show how to use the three main information above to design Csy4 mutants:
        
        
       
What we have proved through the experiment is that the wild-type Csy4 can work well with the miniToe system, which means that all the key points we have discussed before didn't affect the wild-type Csy4. The wild-type Csy4 can dock correctly with the miniToe structure and had a good ability to bind and cleave the miniToe structure. Finally, the crRNA can be released from the RBS. So we choose the wild-type Csy4 as a standard, and all Csy4 mutants can check the four points by comparing with wild-type Csy4.
  

Now for the four points in <B>Q3</B> we have discussed the mathematical forms in <B>Q5</B>. And the most important thing is how to make a comparison between the mutant and wild-type Csy4 enzymes, which will be discussed in <B>Q8</B>.
 
</DIV></SECTION><H4> Q8: How does the design methods work?</H4><SECTION class="block"><DIV class="case-block"><DIV>click to see more</DIV><DIV>click to hide</DIV></DIV><DIV class="detail">
    In <B>Q7</B>, we have discussed the full logic lines about how to design the Csy4 mutants. Here we will give the comparison method for the four key points in miniToe system. And we did this comparison between the mutant and wild-type Csy4 enzymes.
        
    
        
        Now we have four mathematical forms including two curves, a numerical value, and a matrix. Four things can be divided into two kinds of data: the matrix and the numerical value. The interaction matrix and the curve can be regarded as a matrix because the curve is discrete, and the binding free energy is a numerical value. 
        
    
        
       We used Euclidean distance to describe the difference between the two matric: <DIV align="center"><MATH><MROW><MI>D</MI><MO stretchy="false">(</MO><MI>p</MI><MO>,</MO><MSUB><MI>q</MI><MROW><MI>W</MI><MI>T</MI></MROW></MSUB><MO stretchy="false">)</MO><MO>=</MO><MSQRT><MROW><MSTYLE displaystyle="true"><MUNDEROVER><MO>∑</MO><MI>i</MI><MI>m</MI></MUNDEROVER><MROW><MSTYLE displaystyle="true"><MUNDEROVER><MO>∑</MO><MI>j</MI><MI>n</MI></MUNDEROVER><MROW><MSUP><MROW><MO stretchy="false">(</MO><MSUB><MI>p</MI><MROW><MI>i</MI><MO>,</MO><MI>j</MI></MROW></MSUB><MO>−</MO><MSUP><MI>q</MI><MROW><MI>W</MI><MI>T</MI></MROW></MSUP><MSUB><MROW><MI>i</MI><MO>,</MO><MI>j</MI></MROW></MSUB><MO stretchy="false">)</MO></MROW><MN>2</MN></MSUP></MROW></MSTYLE></MROW></MSTYLE></MROW></MSQRT></MROW></MATH></DIV> We used the formula below to calculate the difference of binding free energy between the wild type and mutants: <DIV align="center"><MATH><MROW><MI>ln</MI><MO stretchy="false">(</MO><MSUB><MI>K</MI><MROW><MI>d</MI><MI>r</MI><MI>e</MI><MI>l</MI></MROW></MSUB><MO stretchy="false">)</MO><MO>=</MO><MI>ln</MI><MO stretchy="false">(</MO><MFRAC><MROW><MSUB><MI>K</MI><MROW><MI>d</MI><MI>W</MI><MI>T</MI></MROW></MSUB></MROW><MROW><MSUB><MI>K</MI><MROW><MI>d</MI><MI>M</MI><MI>U</MI><MI>T</MI></MROW></MSUB></MROW></MFRAC><MO stretchy="false">)</MO><MO>=</MO><MO>△</MO><MO>△</MO><MSUB><MI>G</MI><MROW><MI>b</MI><MI>i</MI><MI>n</MI><MI>d</MI><MI>i</MI><MI>n</MI><MI>g</MI></MROW></MSUB></MROW></MATH></DIV>   According to description above, we defined four value to compare the four key points between the mutant and wild-type Cys4 enzymes:
					<MATH><MROW><MSUB><MI>D</MI><MN>1</MN></MSUB><MO stretchy="false">(</MO><MI>int</MI><MI>t</MI><MI>e</MI><MI>r</MI><MI>a</MI><MI>c</MI><MI>t</MI><MI>i</MI><MI>o</MI><MI>n</MI><MI>m</MI><MI>a</MI><MI>t</MI><MI>r</MI><MI>i</MI><MI>x</MI><MO stretchy="false">)</MO></MROW></MATH>

										
,										
										
										
		<MATH><MROW><MI>ln</MI><MO stretchy="false">(</MO><MSUB><MI>K</MI><MROW><MI>d</MI><MI>r</MI><MI>e</MI><MI>l</MI></MROW></MSUB><MO stretchy="false">)</MO></MROW></MATH>
,
<MATH><MROW><MSUB><MI>D</MI><MN>3</MN></MSUB><MO stretchy="false">(</MO><MI>T</MI><MI>h</MI><MI>r</MI><MN>151</MN><MO>−</MO><MI>G</MI><MN>20</MN><MI>c</MI><MI>u</MI><MI>r</MI><MI>v</MI><MI>e</MI><MO stretchy="false">)</MO></MROW></MATH>
,
<MATH><MROW><MSUB><MI>D</MI><MN>4</MN></MSUB><MO stretchy="false">(</MO><MI>R</MI><MI>M</MI><MI>S</MI><MI>D</MI><MO stretchy="false">)</MO></MROW></MATH>
.
				 	 By using the four values, five Csy4 mutants were designed and shown in table below.

            <TABLE width="200" border="1"><TBODY><TR><TH scope="col">Csy4</TH><TH scope="col"><MATH><MROW><MSUB><MI>D</MI><MN>1</MN></MSUB></MROW></MATH></TH><TH scope="col"><MATH><MROW><MI>ln</MI><MO stretchy="false">(</MO><MSUB><MI>K</MI><MROW><MI>d</MI><MI>r</MI><MI>e</MI><MI>l</MI></MROW></MSUB><MO stretchy="false">)</MO></MROW></MATH></TH><TH scope="col"><MATH><MROW><MSUB><MI>D</MI><MN>3</MN></MSUB></MROW></MATH></TH><TH scope="col"><MATH><MROW><MSUB><MI>D</MI><MN>4</MN></MSUB></MROW></MATH></TH></TR><TR><TH scope="row">WT</TH><TD align="center">0</TD><TD align="center">0</TD><TD align="center">0</TD><TD align="center">0</TD></TR><TR><TH scope="row">Q104A</TH><TD align="center">0.483</TD><TD align="center">2483</TD><TD align="center">9.48</TD><TD align="center">30.82</TD></TR><TR><TH scope="row">Y176F</TH><TD align="center">0.592</TD><TD align="center">-382</TD><TD align="center">11.61</TD><TD align="center">40.62</TD></TR><TR><TH scope="row">F155A</TH><TD align="center">0.233</TD><TD align="center">-1627</TD><TD align="center">13.41</TD><TD align="center">35.71</TD></TR><TR><TH scope="row">H29A</TH><TD align="center">0.173</TD><TD align="center">833</TD><TD align="center">15.29</TD><TD align="center">316.22</TD></TR></TBODY></TABLE><A href="https://2018.igem.org/Team:OUC-China/miniToe_Family#wer" style="font-size: 25px; color: blue; text-decoration:none;">  To see more details </A></DIV></SECTION><H4> Q9: How to design the hairpin mutant? </H4><SECTION class="block"><DIV class="case-block"><DIV>click to see more</DIV><DIV>click to hide</DIV></DIV><DIV class="detail">
        The design of hairpin mutants is quite different from the Csy4 mutants due to the large library. In theory, except for the two cleaved sites, G20 and C21, we can generate 4<SUP>20</SUP> mutants totally. 
        
        Combining the bioinformatics and machine learning, we presented an algorithm to pre-processing our big mutation library. The flow chart of the pre-processing algorithm is shown below.
   <DIV align="center"><P>Fig.14 The flow chart of the pre-processing algorithm </P></DIV>
    
        The SVM model was training well and the results were shown below.
         
 <DIV align="center"><P>Fig.15 The training result </P></DIV>
         After training the SVM model, we used it to evaluate the hairpin mutants. We selected the hairpin mutants with high ranks to check the four key points. Finally, we determined the five hairpin mutants below. The following chart shows the DR-Score which is the evaluated results from the SVM model. 
        <TABLE width="200" border="1"><TBODY><TR><TH scope="col">Hairpin-Mutant</TH><TH scope="col"><MATH><MROW><MI>D</MI><MI>R</MI><MO>−</MO><MI>S</MI><MTEXT>core</MTEXT></MROW></MATH></TH></TR><TR><TD align="center">miniToe1</TD><TD align="center">76.6306</TD></TR><TR><TD align="center">miniToe2</TD><TD align="center">65.6278</TD></TR><TR><TD align="center">miniToe3</TD><TD align="center">66.7160</TD></TR><TR><TD align="center">miniToe4</TD><TD align="center">62.5537</TD></TR><TR><TD align="center">miniToe5</TD><TD align="center">52.9794</TD></TR></TBODY></TABLE><A href="https://2018.igem.org/Team:OUC-China/miniToe_Family#werr" style="font-size: 25px; color: blue; text-decoration:none;">  To see more details </A></DIV></SECTION><H4> Q10: How about the mutants designing results? </H4><SECTION class="block"><DIV class="case-block"><DIV>click to see more</DIV><DIV>click to hide</DIV></DIV><DIV class="detail">After designing the protein mutants and hairpin mutants, we tested them in the wet lab. 		
	<DIV align="center"><P>Fig.16 The experimental results of mutants.</P></DIV>  And we try to gave a comparison between the special value we used before for evaluating the mutants and experimental results to check our model.
  For the protein mutants, we gave a comparison between D<SUB>3</SUB> and experimental results.				<DIV align="center"><P>Fig.17 The comparison between model and experimental results for different protein mutants.</P></DIV>  As we can see in the Fig.17, we can find the inner relationship between D<SUB>3</SUB> and experimental results: the D<SUB>3</SUB> value describes the difference in the ability of cleavage between the wild-type and mutants. A higher D<SUB>3</SUB> value means a weaker cleavage ability.
  For the hairpin mutants, we gave a comparison between DR-Score and experimental results.(Fig.18)
		<DIV align="center"><P>Fig.18 The comparison between model and experimental results for hairpin mutants.</P></DIV>  We can find the inner relationship between DR-Score and experimental results except miniToe 1. It is reasonable because the machine learning is quite sensitive to the data amounts and the R<SUP>2</SUP> is not 1 in our training result of SVM model.  Importantly, we tested 30 combinations of Csy4 enzymes and hairpins in wet lab.(Fig.19)
			<DIV align="center"><P>Fig.19 The heatmap results of our 30 combinations</P></DIV></DIV></SECTION><H3>miniToe polycistron —— Not just the monocistron</H3>
    
 In our miniToe polycistron system, we build a coupled transcription-translation model considering several events in prokaryotes to get a deep understanding of polycistron. Then we simplify this model into a more flexible model to predict how the miniToe structure changes the relative expression level in polycistron.
    
    Here I want to discuss the five problems to understand the model work: What does the coupled transcription-translation model tell us? (<B>Q11</B>) How about the flexible model? (<B>Q12</B>) How does the miniToe structure affect the ratio of expression level of cistrons in polycistron? (<B>Q13</B>) What is the main role of cleavage rate in miniToe polycistron system? (<B>Q14</B>) The proof of  miniToe structure in polycistron. (<B>Q15</B>)
<H4> Q11: What does the coupled transcription-translation model tell us? </H4><SECTION class="block"><DIV class="case-block"><DIV>click to see more</DIV><DIV>click to hide</DIV></DIV><DIV class="detail">
    We have consturcted an coupled transcription-translation model to simulaiton the mRNA distribution in the origin time and final time of polycistron system. The Fig.20 and Fig.21 is the mRNA distribution in t=100s and t = 600s.
    
        <DIV align="center"><P>Fig.20 The mRNA distribution in t=100s</P></DIV><DIV align="center"><P>Fig.21 The mRNA distribution in t=600s</P></DIV>
    
    
    The difference of two picture give us two information:
    
    1.The different cistrons in diffrernt positions have different mRNA abundance.
    
    2. The different cistrons in diffrernt positions have different translational time.
    
    <A href="https://2018.igem.org/Team:OUC-China/polycistron#werr" style="font-size: 25px; color: blue; text-decoration:none;">  To see more details </A></DIV></SECTION><H4>Q12: How about the flexibel model we simplify? </H4><SECTION class="block"><DIV class="case-block"><DIV>click to see more</DIV><DIV>click to hide</DIV></DIV><DIV class="detail">
    We have simplify the coupled transcription-translation model into a more flexible model to describe the dynamics of polycistron. This model is including four mian part, take an bicistron as an example:
        (1)The transcription of two CDSs region:
<DIV align="center"><MATH><MTABLE columnalign="left"><MTR><MTD><MOVER><MO>→</MO><MROW><MSUB><MI>k</MI><MN>1</MN></MSUB></MROW></MOVER><MI>m</MI><MI>R</MI><MI>N</MI><MSUB><MI>A</MI><MN>1</MN></MSUB></MTD></MTR><MTR><MTD><MOVER><MO>→</MO><MROW><MSUB><MI>k</MI><MN>2</MN></MSUB></MROW></MOVER><MI>m</MI><MI>R</MI><MI>N</MI><MSUB><MI>A</MI><MN>2</MN></MSUB></MTD></MTR></MTABLE></MATH></DIV> Here we divided the polycistron into two part with different transcription paraments <MATH><MROW><MSUB><MI>k</MI><MN>1</MN></MSUB></MROW></MATH>
  , <MATH><MROW><MSUB><MI>k</MI><MN>2</MN></MSUB></MROW></MATH>
  to deal with the problem of different mRNA abundance due to the premature termination. The two paraments <MATH><MROW><MSUB><MI>k</MI><MN>1</MN></MSUB></MROW></MATH>
 ,<MATH><MROW><MSUB><MI>k</MI><MN>2</MN></MSUB></MROW></MATH>
  is totally  sequence-dependent.										
			(2)The degradation of mRNA		<DIV align="center"><MATH><MTABLE columnalign="left"><MTR><MTD><MI>m</MI><MI>R</MI><MI>N</MI><MSUB><MI>A</MI><MN>1</MN></MSUB><MOVER><MO>→</MO><MROW><MI>k</MI><MSUB><MI>d</MI><MN>1</MN></MSUB></MROW></MOVER><MI>ϕ</MI></MTD></MTR><MTR><MTD><MI>m</MI><MI>R</MI><MI>N</MI><MSUB><MI>A</MI><MN>2</MN></MSUB><MOVER><MO>→</MO><MROW><MI>k</MI><MSUB><MI>d</MI><MN>2</MN></MSUB></MROW></MOVER><MI>ϕ</MI></MTD></MTR></MTABLE></MATH></DIV>	The degradation of RNA also divided into two parts with different transcription paraments  <MATH><MROW><MSUB><MI>k</MI><MROW><MI>d</MI><MN>1</MN></MROW></MSUB></MROW></MATH>
 , <MATH><MROW><MSUB><MI>k</MI><MROW><MI>d</MI><MN>2</MN></MROW></MSUB></MROW></MATH>
 to deal with the problem of different translational time for two mRNA. Each <MATH><MROW><MSUB><MI>k</MI><MROW><MI>d</MI><MTEXT>i</MTEXT></MROW></MSUB></MROW></MATH>
 can be divided into two part:
<DIV align="center"><MATH><MROW><MSUB><MI>k</MI><MROW><MI>d</MI><MTEXT>i</MTEXT></MROW></MSUB><MTEXT>=</MTEXT><MSUB><MI>k</MI><MI>d</MI></MSUB><MO>−</MO><MSUB><MI>k</MI><MROW><MTEXT>recoup</MTEXT></MROW></MSUB></MROW></MATH></DIV>The <MATH><MROW><MSUB><MI>k</MI><MROW><MTEXT>recoup</MTEXT></MROW></MSUB></MROW></MATH>
 denotes the recoup item for the translational time difference and the<MATH><MROW><MSUB><MI>k</MI><MI>d</MI></MSUB></MROW></MATH>
  denotes the common degradation rate of mRNA.
(3)The translation of protein.
<DIV align="center"><MATH><MTABLE columnalign="left"><MTR><MTD><MI>m</MI><MI>R</MI><MI>N</MI><MSUB><MI>A</MI><MN>1</MN></MSUB><MOVER><MO>→</MO><MROW><MI>k</MI><MSUB><MI>p</MI><MN>1</MN></MSUB></MROW></MOVER><MI>m</MI><MI>R</MI><MI>N</MI><MSUB><MI>A</MI><MN>1</MN></MSUB><MSUB><MTEXT> + Protein</MTEXT><MN>1</MN></MSUB></MTD></MTR><MTR><MTD><MI>m</MI><MI>R</MI><MI>N</MI><MSUB><MI>A</MI><MN>2</MN></MSUB><MOVER><MO>→</MO><MROW><MI>k</MI><MSUB><MI>p</MI><MN>2</MN></MSUB></MROW></MOVER><MI>m</MI><MI>R</MI><MI>N</MI><MSUB><MI>A</MI><MN>2</MN></MSUB><MSUB><MTEXT> + Protein</MTEXT><MN>2</MN></MSUB></MTD></MTR></MTABLE></MATH></DIV>In the translation of two proteins, the two paraments used to describe the translation also should be different considering the translation coupling. 							

 										
		(4)The degradation of proteins
<DIV align="center"><MATH><MTABLE columnalign="left"><MTR><MTD><MSUB><MTEXT>Protein</MTEXT><MN>1</MN></MSUB><MOVER><MO>→</MO><MROW><MI>k</MI><MI>d</MI><MSUB><MI>p</MI><MN>1</MN></MSUB></MROW></MOVER><MI>ϕ</MI></MTD></MTR><MTR><MTD><MSUB><MTEXT>Protein</MTEXT><MN>2</MN></MSUB><MOVER><MO>→</MO><MROW><MI>k</MI><MI>d</MI><MSUB><MI>p</MI><MN>2</MN></MSUB></MROW></MOVER><MI>ϕ</MI></MTD></MTR></MTABLE></MATH></DIV>	We don’t have too much discussion in the proteins degradation here.

This model is the simple forms of the coupled transcription-translation model, it keeps the easy form but also reflect the common phenomenon which will happen in the transcript and translation of polycistron including transcript polarity and translation coupling.  									
										
<A href="https://2018.igem.org/Team:OUC-China/polycistron#wer" style="font-size: 25px; color: blue; text-decoration:none;">  To see more details </A></DIV></SECTION><H4>Q13: How does the miniToe structure affect the ratio of expression level for cistrons in polycistron? </H4><SECTION class="block"><DIV class="case-block"><DIV>click to see more</DIV><DIV>click to hide</DIV></DIV><DIV class="detail">
    Our miniToe structure can be used to protect the 5' end and 3' end of mRNA, which can be used to control the half-life of mRNA to control the ratio of expression level for cistrons in polycistron. As the Fig.22 show, the Csy4/RNA complex can protect the 5' end of mRNA. By choosing the different hairpin which has the different binding capability, the mRNA will have different half-life.
        
        <DIV align="center"><P>Fig.22 The mechanisms of miniToe polycistron</P></DIV></DIV></SECTION><H4>Q14: What is the main role of cleavage rate in miniToe polycistron system? </H4><SECTION class="block"><DIV class="case-block"><DIV>click to see more</DIV><DIV>click to hide</DIV></DIV><DIV class="detail">
    Using the model constructed by OUC-China 2016, we explore the  relationship between the cleavage rate and the ratio of two proteins in stable level. Fig.23 is the result of it.
        
        <DIV align="center"><P>Fig.23 the  relationship between the cleavage rate and the ratio of two proteins in stable level</P></DIV>
        From the Fig.23 we can find that the cleavage rate in our miniToe polycistron plays a role in changing the shape of product curve which have been proved in the first system, while having little effect in the ratio of two proteins in the stable level.<A href="https://2018.igem.org/Team:OUC-China/polycistron#werrr" style="font-size: 25px; color: blue; text-decoration:none;">  To see more details </A></DIV></SECTION><H4>Q15: How to prove the effection of our miniToe polycistron in the polycistron ? </H4><SECTION class="block"><DIV class="case-block"><DIV>click to see more</DIV><DIV>click to hide</DIV></DIV><DIV class="detail">
    Using all the problem we disscuss before, we transfrom the complex model into the simple model which we have discussed before. By using the sensitivity analysis in Fig.24, we sucessful prove the function of miiniToe structure in polycistron.
        
        <DIV align="center"><P>Fig.24 the sensitivity analysis of fistly model</P></DIV><A href="https://2018.igem.org/Team:OUC-China/polycistron#werrrr" style="font-size: 25px; color: blue; text-decoration:none;">  To see more details </A></DIV></SECTION><H3>Reference</H3><A href="https://2018.igem.org/Team:OUC-China/miniToe_Family#aa">Reference for miniToe Family</A><A href="https://2018.igem.org/Team:OUC-China/polycistron#aaa">Reference for miniToe Polycistron</A></SECTION></DIV></DIV><DIV class="copyright1">Contact Us : oucigem@163.com  | ©2018 OUC IGEM.All Rights Reserved. </DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>