<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Navarra_BG_project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Navarra BG/project</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="oz-body-wrap"><HEADER class="default-header generic-header"><DIV class="container-fluid"><DIV class="header-wrap"><DIV class="header-top d-flex justify-content-between align-items-center"><DIV class="main-menubar d-flex align-items-center"><NAV class="hide"><A href="https://2018.igem.org/Team:Navarra_BG">Home</A><A href="https://2018.igem.org/Team:Navarra_BG/team">Team</A><A href="https://2018.igem.org/Team:Navarra_BG/project">Project</A><A href="https://2018.igem.org/Team:Navarra_BG/results">Results</A><A href="http://parts.igem.org/cgi/dna_transfer/batch_list.cgi?group_id=3226" target="_blank">Part Submissions</A><A href="https://2018.igem.org/Team:Navarra_BG/safety">Safety</A><A href="https://igem.org/2018_Judging_Form?id=2687" target="_blank">Judging Form</A><A href="https://2018.igem.org/Team:Navarra_BG/collaborations">Collaborations</A></NAV></DIV></DIV></DIV></DIV></HEADER><SECTION class="about-area pt-100 pb-30"><DIV class="container"><DIV class="row align-items-center"><DIV class="col-lg-12"><DIV class="story-content"><H2 id="scroll-to">The <SPAN>Project</SPAN></H2><P>What would you take to space? Imagine you were about to participate in an expedition to a planet far away from Earth and think in the things that you would take with you. Oxygen, water and food would be important things to begin with. Maybe you are also thinking of your favourite books, music, some electronic devices, something to write, to draw... You should also take medicines. Maybe you or another member of the crew needs them on a daily basis, or just in case anyone becomes ill during the expedition. Now, think: what if your trip was of 5, 10 or 15 years long? It may be possible to carry supplies for some months in the spaceship, but in those years everything will spoil. Their conservation would be hard and costly, and they would take too much space.</P><P>To avoid this, scientists from different parts of the world are working in making plants grow and survive in other habitats. Astronauts in the International Space Station have already eaten plants they had grown themselves in space. So, it looks like people participating in future long-term space missions would be able to grow and eat their own plants. However, how about nutrients that aren’t found naturally in plants? Or how about those medicines that we’ve recently talked about? How could astronauts produce specific protein compounds? Well, that is what our project is about.</P><P>Broadly speaking, large-scale production of recombinant proteins involves two steps: (i) synthesis of proteins using genetically modified organisms, and (ii) purification of the proteins. The first step has been mainly limited to <EM>Escherichia coli</EM> and <EM>Saccharomyces cerevisiae</EM>, although increasingly popular plant-based systems offer the potential for safe, economical and high-capacity production for many proteins of pharmaceutical and nutritional interest. Protein purification processes involve multiple steps. Various systems based on the  production of genetically engineered fusion proteins (i.e. an affinity tag covalently linked to a target protein) have been developed to simplify these costly processes. Among them, the plant-based technology involving targeting of oleosin-fused proteins to organelles known as oilbodies has been shown to be cost- effective and enable high levels of production and purification of recombinant proteins.</P><P>Starch is the main storage carbohydrate in vascular plants, its abundance as a naturally occurring compound of living terrestrial biomass being surpassed only by cellulose. Synthesized by different isoforms of starch synthases (SS) this polyglucan accumulates as dense and insoluble granules in the plastids. One SS isoform (the GBSS) is bound to the starch granule. In this project we propose to develop a simple and cost- effective plant-based method for production and purification of recombinant proteins. The system is based on the production of plants transiently expressing a target protein (the green fluorescence protein, GFP) fused to GBSS. Transformed plant tissues will be milled in a suitable aqueous buffer and the starch granules will be purified from plant tissue-derived impurities through a series of simple centrifugation and wash/elution steps as in this aqueous environment the starch granule can be made to precipitate. The GBSS::GFP will be engineered to contain a unique cleavage site recognized by a specific protease, enabling the GFP to be separated from the GBSS into the aqueous buffer, while the GBSS remains embedded the starch granule. Once treated with the protease, the starch granules will be removed by centrifugation while the highly purified cleaved GFP can be further purified using conventional downstream processing.</P></DIV></DIV></DIV></DIV></SECTION><SECTION class="about-generic-area project-weeks"><DIV class="container border-top-generic"><DIV class="row"><DIV class="col-md-3"><H3 class="about-title mb-70">First <SPAN>week</SPAN></H3></DIV><DIV class="col-md-9"><DIV class="row"><DIV class="col-md-12"><DIV class="float-left mr-50 mb-20"><SPAN class="pie"><STRONG>Fig. 1</STRONG> Working in a biological safety cabinet.</SPAN></DIV><P>The objective of our first experiment was to genetically modify plants of the species <EM>Nicotiana benthamiana</EM> to make them produce our target proteins, which will get bonded to the plant’s starch, something that will make them much easier to purify. To achieve this, we first needed to modify bacteria from the species <EM>Agrobacterium tumefaciens</EM> by introducing into them a plasmid that contained our genes. This way, we will be able to use the modified bacteria as a vector to insert the genes in the plants.</P><P>The first step was building the plasmid that was later going to get introduced in the bacteria and that contained the DNA fragment that we wanted to bring inside the plant’s genome. This DNA fragment had the information to produce two different proteins: the GBSS (Granule-Bound  Starch Synthase), that increases the production of starch and bonds to the starch; and the GFP (Green Fluorescent Protein), that gets attached to the GBSS and served us as a model protein. We chose it because under UV light it is fluorescent, so it would be easy for us to observe if the transformation had worked as we wanted. However, any other protein could have been used in its place.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><DIV class="float-right ml-50 mb-20"><SPAN class="pie"><STRONG>Fig. 2</STRONG> E.Z.N.A. Plasmid DNA Mini Kit I</SPAN></DIV><P>In order to build our plasmid, we mixed all the parts of the insert with an empty vector and with some enzymes, and we left the mix react in a thermomixer.</P><P>The next step was to introduce the plasmid inside bacteria from the species <EM>Escherichia coli</EM> through an electroporation.</P><P>After some hours, we cultured the bacteria using the method of the seeding balls and in a medium that contained, apart from other substances, an antibiotic, kanamycin, so we could select only the bacteria that had our plasmid.</P><P>After they had grown, we observed that there were blue colonies, which indicated that they had the empty plasmid, and white colonies, which indicated that they had the correct plasmid, with the insert.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><DIV class="float-left mr-50 mb-20"><SPAN class="pie"><STRONG>Fig. 3</STRONG> Nicotiana benthamiana plants</SPAN></DIV><P>We selected the 6 best colonies, we put each of them in a separated recipient with LB, and we left them in an agitator until the next day.</P><P>When we got the recipients out of the agitator we centrifuged the liquid to separate the medium from the bacteria, and afterwards we made the bacteria undergo a miniprep (Fig. 2). After purifying them, we kept the plasmids in a freezer.</P><P>We also seeded our<EM>Nicotiana benthamiana</EM> plants.</P></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION class="about-generic-area project-weeks"><DIV class="container border-top-generic"><DIV class="row"><DIV class="col-md-3"><H3 class="about-title mb-70">Second <SPAN>week</SPAN></H3></DIV><DIV class="col-md-9"><DIV class="row"><DIV class="col-md-12"><P>After isolating and purifying our plasmids, we had to check that they had been correctly formed. To do this, we performed a digestion with restriction enzymes and an electrophoresis in an agarose gel. Only one of the samples we tried was positive, so we continued working with it.</P><DIV class="text-center mb-20 mb-20"><SPAN class="pie"><STRONG>Fig. 4</STRONG> Agarose gel electrophoresis technique. Positive colony: 4</SPAN></DIV><P>The next step was to build the definitive plasmid that we will put later in the plant. The plasmid we had already purified contained the genetic sequence we wanted the plant to have in the future, but because of its structure, it was not suitable to be introduced into the plant. So we did another reaction to take the transcriptional unit from our plasmid and a kanamycin resistance gene (something we would use in the future as a selection marker if we wanted to transform the plant stably), and introduce it all into a larger plasmid, suitable for the next steps. This is something that the cloning system we used, GoldenBraid, allowed us to make.</P><P>After that, we repeated what we had done with the other plasmid. We electroporated it in <EM>E. coli</EM> bacteria to clone it.</P><P>We also sterilized seeds of the plant Arabidopsis thaliana, sowed them in plates and left them growing in the growing chamber until next week, for a second experiment.</P></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION class="about-generic-area project-weeks"><DIV class="container border-top-generic"><DIV class="row"><DIV class="col-md-3"><H3 class="about-title mb-70">Third <SPAN>week</SPAN></H3></DIV><DIV class="col-md-9"><DIV class="row"><DIV class="col-md-12"><P>Once we had our final plasmid, the next step was to clone it once again. After introducing it in <EM>E. coli</EM> bacteria and then growing them in agar plates with spectinomycin, we picked the best white colonies (confirming the plasmid had been correctly introduced) and we grew them in liquid LB at 37ºC.</P><P>The following step was to perform a miniprep to extract the cloned plasmids from our grown <EM>E. coli</EM>. As we needed to confirm if the plasmid had been correctly built we carried out an electrophoresis, with a faultless result:</P><DIV class="text-center mb-20 mb-20"><SPAN class="pie"><STRONG>Fig. 5</STRONG> Agarose gel electrophoresis technique. Positive colonies.</SPAN></DIV><P>The last step was to insert our correctly built plasmid in <EM>Agrobacterium tumefaciens</EM> (the bacteria that will finally be able to genetically modify our plants) conducting a simple electroforesis, and leave them growing in plates with rifampycin (to avoid the growth of other species of bacteria) and spectinomycin, to finally be able to select the right colonies.</P><P>Subsequently, we transplanted the previously grown <EM>Arabidopsis thaliana</EM> plants to new plates, in groups of 6.</P></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION class="about-generic-area project-weeks"><DIV class="container border-top-generic"><DIV class="row"><DIV class="col-md-3"><H3 class="about-title mb-70">Fourth <SPAN>week</SPAN></H3></DIV><DIV class="col-md-9"><DIV class="row"><DIV class="col-md-12"><DIV class="float-right ml-50 mb-20"><SPAN class="pie"><STRONG>Fig. 6</STRONG> Infiltration in the leaves of Nicotiana benthamiana.</SPAN></DIV><P>By this moment, we were then supposed to have our <EM>Agrobacterium tumefaciens</EM> bacteria with our well built plasmid, but we did a PCR and a gel electrophoresis to be sure.</P><P>Once we were sure everything was correct, we let our bacteria grow in LB with <EM>rifampicin</EM> and spectinomycin. At the same time, we also grew <EM>Agrobacterium tumefaciens</EM> with the silencing vector P19. For the infiltration, we mixed both liquids with the bacteria.</P><P>Finally, it was time to transform our plants (Fig. 6) The modification we did was transitory, because we just wanted to confirm that our method worked, but in the future a stable modification will be possible.</P><P>We infiltrated our bacteria inside the plants leaves. We used a syringe and making a light pressure in the back part of the leaves the liquid got into them through the stomas. Inside the pant, the bacteria introduced our genes inside the plant’s cells.</P><P>We also started the second part of our experiment, which consisted on comparing how plants grew by their own and how they grew next to some microorganisms, like the fungus <EM>Alternaria alternata</EM>. It is known that some microorganisms like our fungus produce volatile compounds, that cause metabolism changes in plants, which lead to an increase in the starch production and the growing rate (Sanchez López et al. 2016). We used this to increase the starch production in our plants, and as a result, also increase in the amount of our protein we could produce.</P><P>Once we had <EM>Arabidopsis thaliana</EM> wild type plants and the <EM>Alternaria alternata</EM> fungus grown, we put some of the plants with the fungus and others without it and we let both of them grow.</P><DIV class="text-center mb-20 mb-20"><SPAN class="pie"><STRONG>Fig. 7</STRONG> Arabidopsis thaliana in contact with thefungus Alternaria alternata.</SPAN></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION class="about-generic-area project-weeks"><DIV class="container border-top-generic"><DIV class="row"><DIV class="col-md-3"><H3 class="about-title mb-70">Fifth <SPAN>week</SPAN></H3></DIV><DIV class="col-md-9"><DIV class="row"><DIV class="col-md-12"><P>Our <EM>Nicotiana benthamiana</EM> plants had already been infiltrated with <EM>Agrobacterium tumefaciens</EM> bacteria. Therefore, our plants were supposed to be expressing our target protein (GFP, Green Fluorescent Protein). Nevertheless, before continuing, we had to verify that our plants had really been modified. To do so, we observed some of the leaves with a confocal microscope, which allowed us to see the fluorescence of the protein.</P><DIV class="row"><DIV class="col-md-6 text-center mb-20 mb-20"><SPAN class="pie"><STRONG>Fig. 8</STRONG> Confocal microscope. Green color demonstratesthe presence of GFP in the chloroplast.</SPAN></DIV><DIV class="col-md-6 text-center mb-20 mb-20"><SPAN class="pie"><STRONG>Fig. 9</STRONG> Confocal microscope.</SPAN></DIV></DIV><P>Once we knew for sure that the plants were producing GFP, the following step was to purify the protein. As we have already commented, an important part of our project is to be able to purify proteins (in our case, the GFP) in a simpler and cheap way with regard to the conventional methods. With our method, we think that we will make viable for astronauts the production of proteins of therapeutic and nutritional interest in the space.</P><DIV class="float-left mr-50 mb-20"><SPAN class="pie"><STRONG>Fig. 10</STRONG> Microscope image where we canobserve the starch granules<A href="http://www.sciencesource.com" target="_blank">www.sciencesource.com</A></SPAN></DIV><P>Our idea to simplify and cheapen the process of purification is to take advantage of the physical properties of a substance that plants produce naturally: starch. Starch is a compound that plants use to store the energy which they produce across the process of photosynthesis. In the process of its production many different enzymes operate. And one of them is the GBSS, Granule-Bound Starch Synthase, which will be very useful in our project because of its property of remaining &quot;stuck&quot; to the granules of starch. In addition, starch is a very dense substance and it is insoluble in water, which will be esential for the process of purification.</P><P>Our protein is bound to the GBSS protein, which is bound to starch granules. We think that, taking advantage of the density and insolubility in water of starch, we can separate these joined substances (starch + GBSS + protein) through a simple centrifugation. Then, we will use a protease to break the binding between the GBSS and the target protein. The result we hope to achieve is a liquid with the highly purified protein, obtained in a fast, easy and cheap way.</P><P>Theoretically, all of the above should work, but experimental confirmation is needed to prove it. So we went on to purify our protein. The first thing we did was to collect some leaves of our transformed plants and crush them. Then, we beated them with a house blender them and filtered the liquid (Fig. 11) we obtained to separate it from the solid remains of the leaves.</P><DIV class="text-center mb-20 mb-20"><SPAN class="pie"><STRONG>Fig. 11</STRONG> Filtering the liquid for purification of the protein.</SPAN></DIV><P>Next, we centrifuged the liquid to separate the starch + GBSS + GFP from the rest of the substances. When centrifuged, the starch precipitates and stays in the bottom part of the tube forming a pellet, while the rest of the liquid substances remain in the upper part forming the supernatant. After throwing away the supernatant, we had our starch + GBSS + purified GFP. We took this photograph of our starch under the microscope:</P><DIV class="text-center mb-20 mb-20"><SPAN class="pie"><STRONG>Fig. 12</STRONG> Confocal microscope. Green color demonstrates the presence of GFP in the starch.</SPAN></DIV><P>Next, we subjected the starch + GBSS + GFP to a lyophilization process, which serves to dehydrate it, so we can accurately measure the amount of starch we have been able to obtain from the leaves. Then we had to add the protease (in our case, we used Xa factor) that will cut the junction between the GBSS and our protein of interest (GFP). We also prepared a control with the sample without the protease, something that will later be useful to show that it is really the protease what causes the GFP to be released.</P><P>Finally, after the reaction, we only had to make one last centrifugation to separate the starch + GBSS, which remain in the bottom of the tube (pellet), from the GFP, which stays up (supernatant). With a pipette, we separated the supernatant from the pellet and passed it to another tube. This way, we obtained our purified protein.</P><P>Regarding the volatile compounds experiment, by this week the difference in the amount of starch, but at this time, the difference in the growth of plants with fungus and plants without fungus is visible to the naked eye:</P><DIV class="text-center mb-20 mb-20"><SPAN class="pie"><STRONG>Fig. 13</STRONG> Arabidopsis thaliana without (A) and in (B)contact with the fungus Alternaria alternata.</SPAN></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION class="about-generic-area project-weeks"><DIV class="container border-top-generic"><DIV class="row"><DIV class="col-md-3"><H3 class="about-title mb-70">Sixth <SPAN>week</SPAN></H3></DIV><DIV class="col-md-9"><DIV class="row"><DIV class="col-md-12"><P>The last step of our first experiment was to verify and demonstrate if our method of purification had worked. That is to say, if we really had obtained GFP from our plants.</P><DIV class="float-right ml-50 mb-20"><SPAN class="pie"><STRONG>Fig. 14</STRONG> Cuvette used in the western blot technique.</SPAN></DIV><P>To do this, we used the Western Blot technique. The first step of this technique was to perform an electroforesis with the sample where we are supposed to have pure GFP. (Fig. 14). After that, the following step was to transfer the proteins from the gel to a membrane. Later, we added a specific antibody for GFP, that is to say, a protein that adheres only to GFP and to any other protein.</P><DIV class="float-right ml-50 mb-20"><SPAN class="pie"><STRONG>Fig. 15</STRONG> Developed in the western blot technique.</SPAN></DIV><P>Later, we added a second antibody that got adhered to the first antibody. This way, in our membrane we had all the GFP molecules sticked to a protein which was sticked to a second protein. This second antibody had an enzyme linked to it that, in presence of specific substances, became visible. For that reason, to detect the presence of GFP in the membrane, we only had to put it in contact with these substances and see if any stains appeared. Where a stain appears, we know that there is GFP.</P></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION class="about-generic-area project-weeks"><DIV class="container border-top-generic"><DIV class="row"><DIV class="col-md-3"><H3 class="about-title mb-70">Seventh <SPAN>week</SPAN></H3></DIV><DIV class="col-md-9"><DIV class="row"><DIV class="col-md-12"><P>Now that we had obtained our target protein, we focused in the second part of our experiment. With this experiment, we aimed to prove that we could increase the efficiency of our system to produce proteins by exposing our plants to the volatile compounds produced by some microorganisms.</P><P>We cultivated two groups of plants from the species <EM>Arabidopsis thaliana</EM>: one of them in the presence of the fungus <EM>Alternaria alternata</EM> and the other one by its own, to compare both groups. Some days later, the difference in growth between the plants was very noticeable. What really mattered to us, however, was the amount of starch, so this week we had to measure it.</P><DIV class="float-right ml-50 mb-20"><SPAN class="pie"><STRONG>Fig. 16</STRONG> Spectrophotometer for Multiskan microplates.</SPAN></DIV><P>In order to do this, the first thing we did was to take some leaves from our plants at the end of the light period to extract as much starch as possible from them. After that, we crushed them with the aid of liquid nitrogen. Later, we separated the starch from other substances, such as glucose or fructose, by using ethanol, heat and centrifugations.</P><P>The next step was to “break down” the starch in glucose, using the amyloglucosidase enzyme. After this, we measured the quantity of glucose using a spectrophotometer (Fig. 16). As expected, the results confirmed that the amounts of starch were much greater in the plants that had been exposed to volatile compounds than in those that had grown alone: from 10 to 12 times more.</P></DIV></DIV></DIV></DIV></DIV></SECTION><FOOTER class="section-gap"><DIV class="container"><DIV class="footer-bottom"><DIV class="row"><DIV class="col-md-8"><P class="footer-text m-0"><A href="mailto:igem@planetastem.com">igem@planetastem.com</A></P></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>