<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SVCE_CHENNAI_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SVCE CHENNAI/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>ReguloGEM - SVCE_CHENNAI</TITLE><TITLE>ReguloGEM - SVCE_CHENNAI</TITLE></P><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-animations"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-animations" data-hover="dropdown" data-animations="fadeInUp"><UL class="nav navbar-nav"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Project </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI#overview">Overview </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Adaptor">Adaptor </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/RNApHmeter">RNA pHmeter </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Applications">Applications </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Protocols">Protocols </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Notebook">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Safety">Safety </A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Results </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Parts">Parts </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Demonstrate">Wetlab </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/INSILICO">Insilico Analysis</A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/InterLab">Interlab </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Software">Software</A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Model">Modelling </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/HP/Silver">Public Outreach </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/HP/Gold_Integrated">Integrated HP</A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Collaborations">Collaboration </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Team </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Members">Members </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Attributions">Attributions </A></LI></DIV></DIV><DIV class="container"><SECTION class="sectionSpacing" id="Modelling"><DIV class="row"><DIV class="col-sm-12 col-md-12 col-lg-12 text-center"><H1 id="heading"> MODELLING </H1></DIV></DIV><DIV class="row"><DIV class="col-sm-12 col-md-12 col-lg-12 text-center"><A class="miniNav" href="#OverviewL">  Overview </A><A class="miniNav" href="#ReactionsL">  Reactions </A><A class="miniNav" href="#ResultsL">  Results </A><A class="miniNav" href="#ParametersL">  Parameters </A></DIV></DIV></SECTION><SECTION class="sectionSpacing" id="OverView"><DIV id="OverviewL" class="row blueNote"><DIV class="col-sm-12 col-md-12 col-lg-12 text-center"><H2 class="subheading"> Overview </H2></DIV></DIV><DIV class="row"><DIV class="col-sm-12 col-md-12 col-lg-12"><P class="paragraphs">
                        The highlight of SVCE_CHENNAI’s ReguloGEM project is the adaptor that converts translational regulation to transcriptional regulation. Our model of the adaptor here is used to determine the amount of protein produced with respect to if the riboswitch present upstream is in its active or inactive form.
                        For this, we have used the SimBiology desktop, which is an applet of MatLab, to model both the active as well as inactive states of our adaptor with the constitutive RBS. For comparison purposes, we have also modeled normal GFP production, to compare and contrast the two cases of GFP production, with and without the adaptor. 
                        The reason that we’ve decided to model our adaptor is that since it is a fairly new concept, we wanted to run simulations that would help the team understand our project more, while simultaneously helping us to predict the behavior of the adaptor which would help us in our actual experiments.
                    </P></DIV></DIV></SECTION><SECTION class="sectionSpacing" id="Reactions"><DIV id="ReactionsL" class="row blueNote"><DIV class="col-sm-12 col-md-12 col-lg-12 text-center"><H2 class="subheading"> Reactions </H2></DIV></DIV><DIV class="row"><DIV class="col-sm-12 col-md-12 col-lg-12"><P class="paragraphs">
                       We would like to acknowledge Team Lund hereas they had been kind enough to help us form the following reactions and differential equations.
                       The following is a diagram of the adaptor circuit:
                   </P></DIV></DIV><DIV class="row"><H3 class="mainPoint text-center" style="color:#22B2E7; font-weight: bold; font-size: x-large;">Adaptor</H3></DIV><DIV class="row"><H3 class="mainPoint text-center" style="color:#22B2E7; font-weight: bold; font-size: x-large;">Adaptor with constitutive RBS</H3></DIV><DIV class="row"><DIV class="col-sm-12 col-md-12 col-lg-12"><P class="paragraphs">
                      We assume two conformations of the constitutive riboswitch corresponding to its active and inactive form, ON and OFF and their mRNA transcripts are given as M <SUB>a </SUB>and M <SUB> b </SUB>, respectively.
                  </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12 col-md-12 col-lg-12"><H3 class="paragraphs mainPoint" style="color:#22B2E7; font-weight: bold; font-size: x-large;"> 1. Transcription of the mRNA transcripts M <SUB>a </SUB> and M <SUB>b </SUB>: </H3></DIV></DIV><DIV class="row"><DIV class="col-sm-12 col-md-12 col-lg-12"><P class="paragraphs">
                 The above differential equation shows the rate of change of promoter concentration with time, which is zero, since the promoter concentration is constant.
             </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12 col-md-12 col-lg-12"><P class="paragraphs">
            The next two DEs show the rate of change of the mRNA transcripts MA and MB of the ON and OFF states, whose degradation is represented by the degradation terms given in the equation.
        </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12 col-md-12 col-lg-12"><H3 class="paragraphs mainPoint" style="color:#22B2E7; font-weight: bold; font-size: x-large;"> 2. Formation of TNAc peptide: </H3></DIV></DIV><DIV class="row"><DIV class="col-sm-12 col-md-12 col-lg-12"><P class="paragraphs">

                        The above DE corresponds to the rate of change of TNAc concentration over time. A degradation term has been added to depict the degradation of TNAc peptide. The translated polypeptide T will be considered as a transcription factor using the following logical basis. mRNA Elongation will occur if TnaC acts upon the ribosome by stalling it over the terminator site, thus preventing rho-dependent termination. Conceptually, this corresponds to the positive action of a molecule on the initiation of transcription. This resembles the action of an activator (positive transcription factor) on an enhancer sequence. Ergo, the TnaC is considered a transcription factor and modeled as such.

                    </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12 col-md-12 col-lg-12"><P class="paragraphs">
                        The above two equations show the rate of change of the RNA-polymerase with time, both in its bound and unbound forms.
                    </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12 col-md-12 col-lg-12"><H3 class="paragraphs mainPoint" style="color:#22B2E7; font-weight: bold; font-size: x-large;"> 3. Formation of the protein GFP: </H3></DIV></DIV><DIV class="row"><DIV class="col-sm-12 col-md-12 col-lg-12"><P class="paragraphs">

                The above two DEs show the rate of change of concentration of the mRNA and the peptide forms of the protein GFP.
                The below reactions correspond to the degradation of the above species:
            </P></DIV></DIV></SECTION><SECTION class="sectionSpacing" id="Results"><DIV id="ResultsL" class="row blueNote"><DIV class="col-sm-12 col-md-12 col-lg-12 text-center"><H2 class="subheading"> Results </H2></DIV></DIV><DIV class="row"><DIV class="col-sm-12 col-md-12 col-lg-12"><P class="paragraphs">
                            The below is the model of the adaptor containing the constitutive RBS, designed in the SimBiology applet in MATLAB,
                        </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12 col-md-12 col-lg-12"><H3 class="paragraphs mainPoint" style="color:#22B2E7; font-weight: bold; font-size: x-large;"> ACTIVE STATE </H3></DIV></DIV><DIV class="row"><DIV class="col-sm-12 col-md-12 col-lg-12"><P class="paragraphs">
                            The Stop time was set to 118,800 seconds (33 HOURS) and the following plot was generated:
                        </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12 col-md-12 col-lg-12"><H3 class="paragraphs mainPoint" style="color:#22B2E7; font-weight: bold; font-size: x-large;"> INACTIVE STATE </H3></DIV></DIV><DIV class="row"><DIV class="col-sm-12 col-md-12 col-lg-12"><P class="paragraphs">
                            Since the constitutive RBS does not have an inactive conformation, GFP is produced constitutively irrespective of the conditions. However, considering any other riboswitch in its stead, the plot may look like this:
                        </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12 col-md-12 col-lg-12"><P class="paragraphs">
                           Clearly there is no GFP production when the adaptor is placed under the inactive conformation of the riboswitch.
                       </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12 col-md-12 col-lg-12"><H3 class="paragraphs mainPoint" style="color:#22B2E7; font-weight: bold; font-size: x-large;"> GFP PRODUCTION WITHOUT AN ADAPTOR:</H3></DIV></DIV><DIV class="row"><DIV class="col-sm-12 col-md-12 col-lg-12"><P class="paragraphs">
                      Here we have designed a standard genetic model, consisting of a promoter, an RBS and the GFP gene. On simulating this model, the plot obtained looked like this:
                   </P></DIV></DIV><DIV class="row"><DIV class="col-sm-12 col-md-12 col-lg-12"><P class="paragraphs">
                    This was essential to our further understanding of the adaptor. The comparison between the two cases clearly showed a difference in their production; in the model without adaptor, GFP production started almost instantaneously, i.e., from time t=0. But in the model containing the adaptor, GFP production starts after a while. From this, we were able to infer that the adaptor has more control and does not permit much basal transcription.
                    Furthermore, it was also inferred that by replacing the constitutive RBS present upstream of the adaptor with a translational riboswitch, we can gain even more transcriptional control.
                    From the above model, the following conclusions can be made:

                    1. The team got to understand the workings of the adaptor with the help of the model
                    
                    2. The adaptor reduces the level of basal expression

                    3.  By replacing the constitutive RBS of the adaptor with basically any other riboswitch, we can hold transcriptional control.

  </P></DIV></DIV></SECTION><SECTION class="sectionSpacing" id="Parameters"><DIV id="ParametersL" class="row blueNote"><DIV class="col-sm-12 col-md-12 col-lg-12 text-center"><H2 class="subheading"> Parameters </H2></DIV></DIV><DIV class="row"><DIV class="col-sm-12 col-md-12 col-lg-12"><TABLE class="table table-bordered"><THEAD><TR><TH class="tablePoints"> RATE CONSTANT</TH><TH class="tablePoints"> UNITS</TH><TH class="tablePoints"> VALUES</TH><TH class="tablePoints"> DESCRIPTION</TH></TR></THEAD><TBODY><TR><TD class="tablePoints"> k <SUB>tA </SUB></TD><TD class="tablePoints"> 1/s</TD><TD class="tablePoints"> 5*10<SUP> -12 </SUP></TD><TD class="tablePoints"> Transcriptional rate constant of M <SUB>A </SUB></TD></TR><TR><TD class="tablePoints"> k <SUB> tB </SUB></TD><TD class="tablePoints"> 1/s</TD><TD class="tablePoints"> 5*10<SUP> -12 </SUP></TD><TD class="tablePoints"> Transcriptional rate constant of M <SUB>B </SUB></TD></TR><TR><TD class="tablePoints"> k <SUB> cA </SUB></TD><TD class="tablePoints"> 1/s</TD><TD class="tablePoints"> 10 <SUP> -1 </SUP></TD><TD class="tablePoints"> Conformational change rate constant M <SUB> A </SUB></TD></TR><TR><TD class="tablePoints"> k <SUB> cB </SUB></TD><TD class="tablePoints"> 1/s</TD><TD class="tablePoints"> 10 <SUP> 1 </SUP></TD><TD class="tablePoints"> Conformational change rate constant M <SUB> B </SUB></TD></TR><TR><TD class="tablePoints"> k <SUB> tsA </SUB></TD><TD class="tablePoints"> 1/s</TD><TD class="tablePoints"> 10 <SUP> -3 </SUP></TD><TD class="tablePoints"> Translational rate constant M <SUB> A </SUB></TD></TR><TR><TD class="tablePoints"> k <SUB> tsB</SUB></TD><TD class="tablePoints"> 1/s</TD><TD class="tablePoints"> 10 <SUP> -2 </SUP></TD><TD class="tablePoints"> Translational rate constant M <SUB> B </SUB></TD></TR><TR><TD class="tablePoints"> k <SUB> a </SUB></TD><TD class="tablePoints"> nM/s</TD><TD class="tablePoints"> 0.027</TD><TD class="tablePoints"> Assosciation rate constant of TNAc-RNAp</TD></TR><TR><TD class="tablePoints">  k <SUB> a </SUB></TD><TD class="tablePoints"> 1/s</TD><TD class="tablePoints"> 0.023</TD><TD class="tablePoints"> Dissociation rate constant of TNAc-RNAp</TD></TR><TR><TD class="tablePoints"> k <SUB> tT </SUB></TD><TD class="tablePoints"> 1/s</TD><TD class="tablePoints"> 0.0166</TD><TD class="tablePoints"> Transcriptional rate constant GFP</TD></TR><TR><TD class="tablePoints"> k <SUB> tsM </SUB></TD><TD class="tablePoints"> 1/s</TD><TD class="tablePoints"> 0.004</TD><TD class="tablePoints"> Translational rate constant GFP</TD></TR><TR><TD class="tablePoints"> k <SUB> dA </SUB></TD><TD class="tablePoints"> 1/s</TD><TD class="tablePoints"> 10 <SUP> -3 </SUP></TD><TD class="tablePoints"> Degradation rate constant M <SUB> A </SUB></TD></TR><TR><TD class="tablePoints"> k <SUB> dB </SUB></TD><TD class="tablePoints"> 1/s</TD><TD class="tablePoints"> 10 <SUP> -3 </SUP></TD><TD class="tablePoints"> Degradation rate constant M <SUB> B </SUB></TD></TR><TR><TD class="tablePoints"> k <SUB> dM </SUB></TD><TD class="tablePoints"> 1/s</TD><TD class="tablePoints"> 0.00685</TD><TD class="tablePoints"> Degradation rate constant GFP mRNA</TD></TR><TR><TD class="tablePoints"> k <SUB> dT </SUB></TD><TD class="tablePoints"> 1/s</TD><TD class="tablePoints"> 0.011</TD><TD class="tablePoints"> Degradation rate constant TNAc</TD></TR><TR><TD class="tablePoints"> k <SUB> dP </SUB></TD><TD class="tablePoints"> 1/s</TD><TD class="tablePoints"> 5.38*10 <SUP> -6 </SUP></TD><TD class="tablePoints">  Degradation rate constant GFP</TD></TR></TBODY></TABLE></DIV></DIV></SECTION><DIV class="col-sm-12 col-md-12 col-lg-12"><H3 class="paragraphs mainPoint" style="color:#22B2E7; font-weight: bold; font-size: x-large;"> REFERENCES </H3><OL style="font-size: 18px;"><LI>Chang, C. L.,Lei Qi, Lucks J.B., Segall-Shapiro T.H., Wang, D.,Mutalik V.K.&amp;Arkin, A.P.An adaptor from translational to transcriptional control enables predictable assembly of complex regulation. Nat.Methods9,1088–1094(2012). </LI><LI><A href="https://2009.igem.org/Team:UAB-Barcelona/Modeling2"> Team:UAB-Barcelona (2009) </A></LI><LI>Beisel CL, Smolke CD. Design Principles for Riboswitch Function. Arkin AP, ed. PLoS Computational Biology. 2009;5(4):e1000363. doi:10.1371/journal.pcbi.1000363. </LI><LI>Sanchez A, Garcia HG, Jones D, Phillips R, Kondev J. Effect of Promoter Architecture on the Cell-to-Cell Variability in Gene Expression. Wasserman WW, ed. PLoS Computational Biology. 2011;7(3):e1001100. doi:10.1371/journal.pcbi.1001100. </LI><LI><A href="https://2007.igem.org/Imperial/Wet_Lab/Results/Res1.4"> Team:Imperial(2007) </A></LI></OL></DIV></DIV></DIV><TITLE>ReguloGEM - SVCE_CHENNAI</TITLE><TITLE>ReguloGEM - SVCE_CHENNAI</TITLE><NAV class="navbar navbar-inverse navbar-fixed-top" role="navigation"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-animations"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-animations" data-hover="dropdown" data-animations="fadeInUp"><UL class="nav navbar-nav"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Project </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI#overview">Overview </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Adaptor">Adaptor </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/RNApHmeter">RNA pHmeter </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Applications">Applications </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Protocols">Protocols </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Notebook">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Safety">Safety </A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Results </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Parts">Parts </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Demonstrate">Wetlab </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/INSILICO">Insilico Analysis</A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/InterLab">Interlab </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Software">Software </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Model">Modelling </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/HP/Silver">Public Outreach </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/HP/Gold_Integrated">Integrated HP</A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Collaborations">Collaboration </A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Team </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Members">Members </A></LI><LI><A href="https://2017.igem.org/Team:SVCE_CHENNAI/Attributions">Attributions </A></LI></DIV></DIV></NAV><SECTION id="sponsors"><DIV class="row"><DIV class="col-sm-12 col-md-2 col-lg-2 col-md-offset-2 col-lg-offset-2 text-center"><P class="emailId"> igemsvce@gmail.com </P></DIV><DIV class="col-sm-12 col-md-2 col-lg-2 text-center"><A href="https://twitter.com/igemsvce" target="_blank">
                    twitter
                </A></DIV><DIV class="col-sm-12 col-md-2 col-lg-2 text-center"><A href="https://github.com/SVCECHENNAI2017" target="_blank">
                    github
                </A></DIV><DIV class="col-sm-12 col-md-2 col-lg-2 text-center"><A href="https://www.facebook.com/teamsvce/" target="_blank">
                    facebook
                </A></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></BODY></HTML>