Team:SVCE CHENNAI/Model
ReguloGEM - SVCE_CHENNAI
ReguloGEM - SVCE_CHENNAI
Toggle navigation
Project
Overview
Adaptor
RNA pHmeter
Applications
Protocols
Notebook
Safety
Results
Parts
Wetlab
Insilico Analysis
Interlab
Software
Modelling
Human Practices
Public Outreach
Integrated HP
Collaboration
Team
Members
Attributions
MODELLING
  Overview   Reactions   Results   Parameters
Overview
The highlight of SVCE_CHENNAI’s ReguloGEM project is the adaptor that converts translational regulation to transcriptional regulation. Our model of the adaptor here is used to determine the amount of protein produced with respect to if the riboswitch present upstream is in its active or inactive form.
For this, we have used the SimBiology desktop, which is an applet of MatLab, to model both the active as well as inactive states of our adaptor with the constitutive RBS. For comparison purposes, we have also modeled normal GFP production, to compare and contrast the two cases of GFP production, with and without the adaptor.
The reason that we’ve decided to model our adaptor is that since it is a fairly new concept, we wanted to run simulations that would help the team understand our project more, while simultaneously helping us to predict the behavior of the adaptor which would help us in our actual experiments.
Reactions
We would like to acknowledge Team Lund hereas they had been kind enough to help us form the following reactions and differential equations.                        The following is a diagram of the adaptor circuit:
Adaptor
Adaptor with constitutive RBS
We assume two conformations of the constitutive riboswitch corresponding to its active and inactive form, ON and OFF and their mRNA transcripts are given as M a and M b , respectively.
1. Transcription of the mRNA transcripts M a and M b :
The above differential equation shows the rate of change of promoter concentration with time, which is zero, since the promoter concentration is constant.
The next two DEs show the rate of change of the mRNA transcripts MA and MB of the ON and OFF states, whose degradation is represented by the degradation terms given in the equation.
2. Formation of TNAc peptide:
The above DE corresponds to the rate of change of TNAc concentration over time. A degradation term has been added to depict the degradation of TNAc peptide. The translated polypeptide T will be considered as a transcription factor using the following logical basis. mRNA Elongation will occur if TnaC acts upon the ribosome by stalling it over the terminator site, thus preventing rho-dependent termination. Conceptually, this corresponds to the positive action of a molecule on the initiation of transcription. This resembles the action of an activator (positive transcription factor) on an enhancer sequence. Ergo, the TnaC is considered a transcription factor and modeled as such.
The above two equations show the rate of change of the RNA-polymerase with time, both in its bound and unbound forms.
3. Formation of the protein GFP:
The above two DEs show the rate of change of concentration of the mRNA and the peptide forms of the protein GFP.                 The below reactions correspond to the degradation of the above species:
Results
The below is the model of the adaptor containing the constitutive RBS, designed in the SimBiology applet in MATLAB,
ACTIVE STATE
The Stop time was set to 118,800 seconds (33 HOURS) and the following plot was generated:
INACTIVE STATE
Since the constitutive RBS does not have an inactive conformation, GFP is produced constitutively irrespective of the conditions. However, considering any other riboswitch in its stead, the plot may look like this:
Clearly there is no GFP production when the adaptor is placed under the inactive conformation of the riboswitch.
GFP PRODUCTION WITHOUT AN ADAPTOR:
Here we have designed a standard genetic model, consisting of a promoter, an RBS and the GFP gene. On simulating this model, the plot obtained looked like this:
This was essential to our further understanding of the adaptor. The comparison between the two cases clearly showed a difference in their production; in the model without adaptor, GFP production started almost instantaneously, i.e., from time t=0. But in the model containing the adaptor, GFP production starts after a while. From this, we were able to infer that the adaptor has more control and does not permit much basal transcription.                     Furthermore, it was also inferred that by replacing the constitutive RBS present upstream of the adaptor with a translational riboswitch, we can gain even more transcriptional control.                     From the above model, the following conclusions can be made:                      1. The team got to understand the workings of the adaptor with the help of the model                                          2. The adaptor reduces the level of basal expression                      3.  By replacing the constitutive RBS of the adaptor with basically any other riboswitch, we can hold transcriptional control.
Parameters
RATE CONSTANT
UNITS
VALUES
DESCRIPTION
k tA
1/s
5*10 -12
Transcriptional rate constant of M A
k tB
1/s
5*10 -12
Transcriptional rate constant of M B
k cA
1/s
10 -1
Conformational change rate constant M A
k cB
1/s
10 1
Conformational change rate constant M B
k tsA
1/s
10 -3
Translational rate constant M A
k tsB
1/s
10 -2
Translational rate constant M B
k a
nM/s
0.027
Assosciation rate constant of TNAc-RNAp
k a
1/s
0.023
Dissociation rate constant of TNAc-RNAp
k tT
1/s
0.0166
Transcriptional rate constant GFP
k tsM
1/s
0.004
Translational rate constant GFP
k dA
1/s
10 -3
Degradation rate constant M A
k dB
1/s
10 -3
Degradation rate constant M B
k dM
1/s
0.00685
Degradation rate constant GFP mRNA
k dT
1/s
0.011
Degradation rate constant TNAc
k dP
1/s
5.38*10 -6
Degradation rate constant GFP
REFERENCES
Chang, C. L.,Lei Qi, Lucks J.B., Segall-Shapiro T.H., Wang, D.,Mutalik V.K.&Arkin, A.P.An adaptor from translational to transcriptional control enables predictable assembly of complex regulation. Nat.Methods9,1088–1094(2012).
Team:UAB-Barcelona (2009)
Beisel CL, Smolke CD. Design Principles for Riboswitch Function. Arkin AP, ed. PLoS Computational Biology. 2009;5(4):e1000363. doi:10.1371/journal.pcbi.1000363.
Sanchez A, Garcia HG, Jones D, Phillips R, Kondev J. Effect of Promoter Architecture on the Cell-to-Cell Variability in Gene Expression. Wasserman WW, ed. PLoS Computational Biology. 2011;7(3):e1001100. doi:10.1371/journal.pcbi.1001100.
Team:Imperial(2007)
ReguloGEM - SVCE_CHENNAI
ReguloGEM - SVCE_CHENNAI
Toggle navigation
Project
Overview
Adaptor
RNA pHmeter
Applications
Protocols
Notebook
Safety
Results
Parts
Wetlab
Insilico Analysis
Interlab
Software
Modelling
Human Practices
Public Outreach
Integrated HP
Collaboration
Team
Members
Attributions
igemsvce@gmail.com
twitter
github
facebook
