<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SEU_O_China_Model"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SEU_O_China/Model">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SEU_O_China/Model&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SEU_O_China/Model&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SEU_O_China/Model&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SEU_O_China/Model" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SEU O China/Model</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><H1>Modeling</H1><DIV id="col_nav"><DIV class="col_list"><CENTER><H2>Simulation</H2><UL><LI id="in">Introduction</LI><LI id="lis">Light Induce Micro</LI><LI id="lib">Light Induce Macro</LI><LI id="au">Auto-differentiation</LI></UL><H2>Data Process</H2><UL><LI id="pa">Parameter Estimation</LI><LI id="ex">Experiment Data</LI></UL><H2>Application</H2><UL><LI id="co">Bio-Computer</LI></UL></CENTER></DIV></DIV><DIV id="col_left"><DIV class="content" id="incontent"><H3>System Simulation</H3><P><B>Introduction</B></P><P>In order to verify the availability and probable effect of our design scheme, simulation mathematical models based on Cellular Automata technique have been conducted.
</P><P>In catering to our past experiment scheme, we have constructed two models as follows:
</P><UL><LI>Light Sensing Model: use light to trigger the asymmetry process of the colony;
</LI><LI>Movement Model: use more complicated pathways to induce the break of symmetry.
</LI></UL><P>A cellular automaton is in nature a finite-state machine in discrete time as well as space studied in computability theory,mathematics, physics, complexity science, theoretical biology and microstructure modeling.
</P><P>This model provides a significant reference to the appraise of realistic experiments by simulating the whole pattern changing process, which consists of division, movement, death and some relevant ones.
</P><UL><LI>The code of our program can be downloaded <A href="https://static.igem.org/mediawiki/2012/1/15/SeuoModelCodeAll.zip" class="external text" rel="nofollow">here</A>.
</LI></UL></DIV><DIV class="content" id="liscontent"><H3>Light Induced Micro Model</H3><P>
To display the simulation results directly, we have constructed a micro model based on cellular automata. Each minimum spatial unit is able to hold up at most one cell, which can be characterized by a cellular automaton; Meanwhile, another property that wonders in every minimum spatial unit is the density of AHL, which relies on the density diffusion equation. The detailed cellular automata rules would be as follows.
</P><P><B>Basic Principles</B></P><P>
a) Cell types
</P><P>Each minimum spatial unit in this cellular automata model is able to hold up at most one cell. Cells can be divided into two types, normal cells(marked by green) and special cells(marked by red). Red cells are transferred from green cells by the trigger of light. The type of cells is denoted by ‘C’ as follows:
</P><DIV class="center"><DIV class="floatnone"><A href="/File:Seuofun1.gif" title="File:Seuofun1.gif">File:Seuofun1.gif</A></DIV></DIV><UL><LI>Equation 1.
</LI></UL><P>‘n’ represents the n th period, ‘(i, j)’ represents the location of the cell. Same below
Red cells emit AHL at the rate of ‘q’ in every period.
</P><P>
b) Cell Division and Cellular States
</P><P>Both kinds of cells are likely to divide. A mature cell possesses a division probability of Pdiv  every time unit and gets two green cells after division. The newly generated cell would grow closely to the original cell and would emerge at the eight neighboring positions with equal probability. If the eight neighboring positions had been occupied, then the original cell would not be able to divide.
</P><P>Consider the realistic condition where continuous cell division is impossible, so we use Φ to denote cellular states. At the very beginning Φ=1; Then with the cells growing, Φ=Φ+1 with every period; When Φ=τ, the cell would be mature enough to divide with a division probability of Pdiv  during every period; After every division, the Φ(n;i,j) of the original and new cells would both change into:
</P><DIV class="center"><DIV class="floatnone"><A href="/File:Seuofun2.GIF" title="File:Seuofun2.GIF">File:Seuofun2.GIF</A></DIV></DIV><UL><LI>Equation 2.
</LI></UL><P>However, in our system, as a signal molecule, AHL can trigger the antisense Ftsz part which can be functional as a division repressor. So Pdiv is actually a constant but a parameter proportional to the density of AHL. With several steps of calculations and experiments, the relationship between Pdiv and the density of AHL can be induced as follows:
</P><DIV class="center"><DIV class="floatnone"><A href="/File:Seuofun3.GIF" title="File:Seuofun3.GIF">File:Seuofun3.GIF</A></DIV></DIV><UL><LI>Equation 3.
</LI></UL><P>Among it both ‘a’ and ‘b’ are unknown parameters.
</P><P>
c) Cell Movement
</P><P>In the light sensing model, due to the big influence of light from the outer environment, cell movement has almost nothing to do with final results both in simulation or realistic experiments. As a result, we simplify the movement of cells. However, the movement of cells would no longer be negligible in the afterwards models. Further details on its rules and results to come afterwards.
</P><P>
d) Cell Death
</P><P>In all models mentioned in this text, cell death has merely minor influence on our final results so it is neglected in all.
</P><P><B>AHL Density Equation</B></P><P>AHL density can be expressed as u(r,t). Take the discrete characteristic of cellular automata into consideration, just denote it as   . As a solute, AHL has the phenomenon of diffusion and decomposition. At mentioned above, red cells would spread out AHL. In conclusion, AHL complies to diffusion equation:
</P><DIV class="center"><DIV class="floatnone"><A href="/File:Seuofun4.GIF" title="File:Seuofun4.GIF">File:Seuofun4.GIF</A></DIV></DIV><UL><LI>Equation 4.
</LI></UL><P>u: AHL density; D: diffusion constant;  q: AHL releasing rate of a single triggered cell; γ describes the decomposition process of AHL; m: total number of cells.
</P><P><B>Simulation Procedure</B></P><P>At the very beginning, AHL density u(r, 0)=0 on the infinite plane. In order to eliminate the asymmetry in the results that might be caused by the asymmetry of the original state, there is only one green cell at first. Choose several separate cells manually to shed light on. With the sliding improvement of the colony from merely one cell, green cells would eventually reach the light area where they would transform into red cells. Red cells begin to release AHL. When the quantity of red cells accumulates to a certain threshold, high density of AHL in certain areas would repress the division of cells and begin to trigger the differentiation of cell and therefore break the symmetry of the colony.
</P><P><B>Results</B></P><TABLE border="0" cellspacing="0" cellpadding="5" align="center"><TBODY><TR><TD align="center">Control group result
</TD><TD align="center">Experiment group result
</TD></TR></TBODY></TABLE><P><CENTER>Control Group:

Experiment Group:
</CENTER></P><P><B>Modeling Parameters</B></P><UL><LI>Recovery time: tr=2
</LI><LI>Releasing Constant: q=600;
</LI><LI>Decomposition Constant: y=1;
</LI><LI>Diffusion Constant: D=0.15;
</LI></UL></DIV><DIV class="content" id="libcontent"><H3>Light Induced Macro Model</H3><P>
The micro model is somehow quite easy to understand. We can relatively directly observe and analyze the location and state of each cell, so that this model is suitable for small-scale colony pattern vicissitude analysis. However the application of several random factors like division probability or others eventually complicates the anticipation process. Moreover, variations exist every time of simulation. We have to observe the shape of colony directly(with eyes) on a larger scale, which would obey the statistical rules. For the sake of seeking the most probable situation, we have a macro model to simulate. Macro model differs from the micro model mainly in that the number of cells in a spatial unit no longer limits to one but pertains to another characteristic: cellular density. The macro model would on one hand reduce the quantity of calculation and on the other hand reflects the possible results more realisticly. 
</P><P><B>Basic Principles</B></P><P>Compared to micro model, the AHL density u(r, t) for every spatial unit possesses the same meaning and remain compliant to equation(4). Whereas, the characteristic of cells is no longer weighed by whether existed or not, nor does  the type or state of a single cell be considered, but using the density of green normal cells   and that   of red special cells
</P><P>
a) The Cellular Density Logistic Model
</P><P>In the micro model, we can discuss the division process and growing period of single cells, but not in the macro model any more. For the concept of single cells is eliminated and instead we just take into consideration the quantity of cells in certain areas which would obey some kinds of rules. Here, we applied the classic Logistic Model to lineate the quantity visisitude of colony. With the original number of cells ranging from 1 and 200 and a environmental limit of 100, the changing situation of the number of cells is shown as below:
</P><DIV class="center"><DIV class="floatnone"><A href="/File:Seuologist.GIF" title="File:Seuologist.GIF">570px</A></DIV></DIV><UL><LI>Logistic model with initial cell number of 1 and 200
</LI></UL><P>The equation is shown as follows:
</P><DIV class="center"><DIV class="floatnone"><A href="/File:Seuofun5.GIF" title="File:Seuofun5.GIF">File:Seuofun5.GIF</A></DIV></DIV><UL><LI>Equation 5.
</LI></UL><P>Among which, k is a proportion constant, Nm denotes the environmental cell volume at the location r and time t and is a function of the AHL density u(r, t); N(r, t) denotes the number of cells at the location r and time t.
</P><P>Logistic Model is a somehow realistic model which takes not only the growth of number but also the environmental tension after the growing of the number into consideration and therefore makes it more practical than the miro model.</P><P>As high AHL density would repress cell division, we can regard the environmental volume as a function of AHL density. AHL density gets higher and environmental volume Nm gets lower. The relationship might be as follows:
</P><DIV class="center"><DIV class="floatnone"><A href="/File:Seuofun6.GIF" title="File:Seuofun6.GIF">File:Seuofun6.GIF</A></DIV></DIV><UL><LI>Equation 6.
</LI></UL><P>Among which both 'd' and 'e' are unknown parameters,and d&gt;0.
</P><P>After the change of cellular numbers, due to the background that red special cells can only be transformed by light, all newly-built cells in this Logistic Model would be green cells.
</P><P>
b) Cell Movement
</P><P>As the movement of green cells is random movement, therefore the movement of green normal cells relies on the formula below:
</P><P>Among which 'V' denotes the diffusion constant or a reflection of cellular rate.
</P><P>The movement of red cells is the same.
</P><P>Other parts and simulation principles of macro model are just the same with the micro model.
</P><P><B>Results</B></P><TABLE border="0" cellspacing="0" cellpadding="5" align="center"><TBODY><TR><TD align="center">Control group result
</TD><TD align="center">Experiment group result
</TD></TR></TBODY></TABLE><P><CENTER>
Control Group:
Experiment Group:
</CENTER></P><P><B>Modeling Parameters</B></P><UL><LI>Releasing Constant: q=150;
</LI><LI>Decomposition Constant: y=1;
</LI><LI>Diffusion Constant: D=0.20;
</LI><LI>Cellular Movement Constant: V=1;
</LI><LI>Parameter between AHL density and Nm: a=20, b=0.5, c=0.009, d=0.01
</LI></UL></DIV><DIV class="content" id="aucontent"><H3>Auto-differentiation</H3><P>As the light sensing model requires outer signal to trigger the break of symmetry, this is not a rather ideal condition. Therefore we manage to realize the break of symmetry with more complicated pathways.
</P><P><B>Basic Principles</B></P><P>The basis of auto-differentiation model is cellular automata, which includes a number of same rules as the micro model. But the most obvious difference lies in that the red special cells are not triggered by light stimulus any more but rely on a constant probability of during every division period. Under this circumstance, the locations of red cells are discrete and unstable and as a result, the movement of red cells requires consideration.
</P><P>
a) Cell Movement
</P><P>Red special cells tend to move to high AHL density areas. The tendency would be as follows: Red cells originally move in random directions and at a rate of vr; Red cells have two movement tendencies, turning to a direction or moving along. Every step of movement generates a density gap of Δu and this value would be determinant to the choice of turning around or moving along. Lower density gap would lead to higher probability of turning around. And therefore:
</P><DIV class="center"><DIV class="floatnone"><A href="/File:Seuofun7.GIF" title="File:Seuofun7.GIF">File:Seuofun7.GIF</A></DIV></DIV><UL><LI>Equation 7.
</LI></UL><P>Among which, 'f' and 'g' are unknown parameters.
</P><P>Green cells still comply to the rule of random movement.
</P><P>b) Cell Division
So long as in the auto-differentiation model, red special cells are generated by cell division, the newly-built cell would own a probability of Pred to become red and a probability of 1-Pred to become green. The original cell would stay in its previous state. The ration of red/green would remain almost constant by this method.
</P><P>The other rules, such as the diffusion of AHL molecules, are the same as the light induced mode.
</P><P><B>Simulation Process</B></P><P>The original state would be the same as the micro model where the AHL density u(r, 0)=0 on the infinite plane. Several red special and green cells are randomly distributed in a rather tiny area.
</P><P>After the simulation has begun, the size of colony slowly increases with cell division and movement. Red cells would release AHL and at the same time tend to move to high density areas and lead to the appearance of several piles of red cells, around which AHL density is high enough to repress cell division. With the different division rate of cells (mainly green cells), the symmetry would be broken and the colony would turn into different patterns.
</P><P>The final shape of colony is mainly controlled by the clustering phenomenon of red cells. And as this clustering phenomenon can be influenced by a number of random factors, the eventual colony is unlikely to be symmetry. Meanwhile, there has not existed an appropriate method to analyze the clustering behavior, which means the final shape in the movement model cannot be controlled accurately yet.
</P><P><B>Results</B></P><TABLE border="0" cellspacing="0" cellpadding="5" align="center"><TBODY><TR><TD align="center">Control group result
</TD><TD align="center">Experiment group result
</TD></TR></TBODY></TABLE><P><CENTER>Control Group:

Experiment Group:
</CENTER></P><P><B>Modeling Parameters</B></P><UL><LI>Recovery time: tr=2
</LI><LI>Releasing Constant: q=600;
</LI><LI>Decomposition Constant: y=1;
</LI><LI>Diffusion Constant: D=0.15;
</LI><LI>Cellular Movement Constant: V=2;
</LI><LI>Ratio of Red Cells: 0.01
</LI></UL></DIV><DIV class="content" id="pacontent"><H3>Parameter Estimation</H3><P><B>Introduction</B></P><P>To build a biobrick building, it is important to make sure each part of the building can work as expected. Sometimes, what we concern is not only a part is right or wrong, but precisely how it works. What is the generating rate of a molecule? How can we describe the efficiency of a asRNA silencing? Quantifying the experiment data, parameter estimation is a vital step when we try to combine project design, mathematical modeling and biological experiment. 
</P><P>Here, we are going to introduce an approximate Bayesian computation method for parameter inference in gene expression model, and perform a little test on our experiment data.
</P><P><B>Approximate Bayesian computation (ABC)</B></P><P>ABC methods have been conceived with the aim of inferring posterior distribution where likelihood functions are computationally intractable or too costly to evaluate. They exploit the computational efficiency of model simulation techniques by replacing the calculation of the likelihood with a comparison between the observed and simulated data.
</P><P>Let θ be a parameter vector to be estimated. Given the prior distribution &amp;pi(θ), the goal is to approximate the posterior distribution, 
</P><DIV class="center"><DIV class="floatnone"><A href="/File:Seuofun8.GIF" title="File:Seuofun8.GIF">File:Seuofun8.GIF</A></DIV></DIV><P>where f(x|θ) is the likelihood ofθgiven the data x. The output of an ABC algorithm is a sample of parameters from a distribution
</P><DIV class="center"><DIV class="floatnone"><A href="/File:Seuofun9.GIF" title="File:Seuofun9.GIF">File:Seuofun9.GIF</A></DIV></DIV><P>(D,D' is observed data and stimulated data respectively, ρ(D,D') is a kind of distance, εis a tolerance). If ε is sufficiently small, then the distribution
</P><DIV class="center"><DIV class="floatnone"><A href="/File:Seuofun10.GIF" title="File:Seuofun10.GIF">File:Seuofun10.GIF</A></DIV></DIV><P>will be a good approximation for the posterior distribution 
</P><DIV class="center"><DIV class="floatnone"><A href="/File:Seuofun11.GIF" title="File:Seuofun11.GIF">File:Seuofun11.GIF</A></DIV></DIV><P>The simplest ABC algorithm is the ABC rejection sampler (Pritchard et al.1999), which is as follow
</P><OL><LI>Generate θ* from &amp;pi(θ)
</LI><LI>Simulate D' from model M with parameterθ
</LI><LI>Calculate a measure of distance ρ(D,D') between D’ and D
</LI><LI>Accept θ if ρ&lt;ε, and return 1.
</LI></OL><P>The disadvantage of the ABC rejection sampler is that the acceptance rate is low when the prior distribution is very different from the posterior distribution. To avoid this problem, an ABC method based on Markov chain Monte Carlo was introduced (Marjoram et al.2003). The ABC MCMC algorithm proceeds as follows:
</P><OL><LI>If at θ propose a move to θ’ according to a transition kernel q(θ-&gt;θ’)
</LI><LI>Generate D’ using model M with parameters θ’
</LI><LI>If ρ(D,D’)&lt;=ε, go to 4, and otherwise stay at θ and return 1
</LI><LI>Calculate:  <A href="/File:Seuofun12.GIF" title="File:Seuofun12.GIF">File:Seuofun12.GIF</A></LI><LI>Move to θ’ with probability h, else stay at θ.
</LI></OL><P><B>Gene expression model</B></P><P>This part concerns the posterior distribution of parameter in gene expression model via approximate Bayesian computation.
</P><DIV class="center"><DIV class="floatnone"><A href="/File:Seuogenemode.GIF" title="File:Seuogenemode.GIF">570px</A></DIV></DIV><P>In this model, the relationship of the concentration of gene, mRNA, protein can be represented the following differential equations:
</P><DIV class="center"><DIV class="floatnone"><A href="/File:Seuofun13.GIF" title="File:Seuofun13.GIF">File:Seuofun13.GIF</A></DIV></DIV><P>r: concentration of mRNA.
p: concentration of protein.
V: degradation rate of mRNA.
U: degradation rate of protein.
C: gene transcription coefficient.
L: mRNA translation coefficient.
</P><P>Here, we take the vector (U, V) as (0.05, 0.5), and obtain the posterior distribution of C and L respectively.
</P><P>The data {rd[i],pd[i]} are noisy observations of the stimulated gene expression model with parameter values set at (C, L) = (1, 1). We sample nine data points (for mRNA and protein) from the solution of the system for parameter (U, V) = (0.05, 0.5) and add Gaussian noise N(0,0.03^2) (figure 1). Let the distance function d((rd,pd),(r,p)), between the data {rd[i],pd[i]}, i=1,2,…,10 , and a simulated solution for proposed parameters, {r [i],p [i]}, be the sum of squared errors.
</P><DIV class="center"><DIV class="floatnone"><A href="/File:Seuofun14.GIF" title="File:Seuofun14.GIF">570px</A></DIV></DIV><P>The distance between our noisy data and the deterministic solution for (C, L) = (1, 1) from which the data were generated is 1.5. The prior distributions for C and L are taken to be uniform, C, L~U (-5, 5). The inferred posterior distributions are shown in figure 2.
</P><DIV class="center"><DIV class="floatnone"><A href="/File:Seuoabcmod1.GIF" title="File:Seuoabcmod1.GIF">File:Seuoabcmod1.GIF</A></DIV></DIV><UL><LI>Figure 1: Trajectory of mRNA (solid curve) and protein (dashed curve) concentrations of gene expression model and the data points (circles, mRNA data; asterisks, protein) data).
</LI></UL><DIV class="center"><DIV class="floatnone"><A href="/File:Seuoabcmod2.GIF" title="File:Seuoabcmod2.GIF">File:Seuoabcmod2.GIF</A></DIV></DIV><UL><LI>Figure 2 Parameters inferred by the ABC MCMC
</LI></UL><P>Actually, there is a time delay in gene transcription and translation, so we can improve gene expression model as follows:
</P><DIV class="center"><DIV class="floatnone"><A href="/File:Seuofun15.GIF" title="File:Seuofun15.GIF">File:Seuofun15.GIF</A></DIV></DIV><P>α,β: the time delay in gene transcription and translation. With the (α,β)=(1,1), a new result can be acquired following the previous solution.
</P><P>We also sample nine data points from the solution of the system for parameter (U, V) = (0.05, 0.5) and add Gaussian noise N(0,0.3^2). (figure 3)
</P><DIV class="center"><DIV class="floatnone"><A href="/File:Seuoabcmod3.GIF" title="File:Seuoabcmod3.GIF">File:Seuoabcmod3.GIF</A></DIV></DIV><UL><LI>Figure 3: Trajectory of mRNA (solid curve) and protein (dashed curve) concentrations of gene expression model and the data points (circles, mRNA data; asterisks, protein) data) considering time delay in gene transcription and translation.
</LI></UL><P>The new inferred posterior distributions are shown in figure 4.
</P><DIV class="center"><DIV class="floatnone"><A href="/File:Seuoabcmod4.GIF" title="File:Seuoabcmod4.GIF">File:Seuoabcmod4.GIF</A></DIV></DIV><UL><LI>Figure 4 Parameters inferred by the ABC MCMC considering time delay in gene transcription and translation.
</LI></UL><P>References
</P><P>Pritchard,J.,Seielstad,M.T.,Perez-Lezaun,A.&amp;Feldman,M.W.1999 Population growth of human Y chromosomes: a study of Y chromosome microsatellites. Mol. Biol.Evo.16, 1791-1798.
</P><P>Marjoram, P.,Molitor,J., Plagnol, V.&amp;Tavare,S.2003 Markov chain Monte Carlo without likelihoods. Proc.Natl Acad. Sci. USA 100,15 324-15 328.
</P></DIV><DIV class="content" id="excontent"><H3>Experiment Data</H3><P>To show the how the ABC method work, here we take the TBY experiment data as an example.
</P><P><B>Image Process</B></P><P>In order to transform the experiment photograph into quantified data, we write a simple image processing program.
</P><P>The program firstly determine the region of culture dish, and then extract the points of high brightness, which is the bacteria colony. By calculating the rate of total colony area and culture dish area, we get the colony covering rate.
</P><UL><LI>The original photograph
</LI></UL><UL><LI>The colony pattern extracted by our program. The colony covering rate is 0.2774. For more data, see the <A href="https://2012.igem.org/Team:SEU_O_China/Result#incontent" class="external text" rel="nofollow">result page</A>.
</LI></UL><P><B>Regression</B></P><P>With the all the colony covering rate data, we can calculate out the relationship between colony covering rate, which represent the expression rate of FtsZ, and the density of IPTG, which directly related to the expression of antisense FtsZ mRNA. To match then experiment data and real biology process, we choose the function R=a*exp(b*IPTG(mM))
</P><DIV class="center"><DIV class="floatnone"><A href="/File:Seuoexp1.GIF" title="File:Seuoexp1.GIF">File:Seuoexp1.GIF</A></DIV></DIV><UL><LI>Regression map of colony covering rate and IPTG density. The function is R1=74.09*exp(-0.88*IPTG(mM))
</LI></UL><P><B>Distribution</B></P><P>Using the regression parameter as input, we can get the posterior distributions of unknown parameters through the ABC model.
</P><DIV class="center"><DIV class="floatnone"><A href="/File:Seuoexp2.GIF" title="File:Seuoexp2.GIF">400px</A></DIV></DIV><UL><LI>Posterior distribution of parameter b.
</LI></UL><P><B>Disscussion</B></P><P>It seems that the ABC method is unnecessary in this situation, since with the regressed function and a F-test, we can easily accept a certain parameter. So, why should we use ABC?
</P><P>First, in most biological process, the distribution of a parameter is unknown. There is a high risk when we simply suggest that the parameter has a F-distribution. But with the ABC method, we can actually calculate the distribution out, which enable us to look deep into the mechanism behind the single parameter.
</P><P>In addition, while building a large system with several related parts, the ABC method would become more important. There is always an error when we use a single parameter to determine the system efficiency. With the system growing up, a part linking to another, the error will be larger and larger during the iteration of part connection, but we do not even know about it sometimes. For example, a parameter of part A is 1.0 with a 0.95 certainty, and another parameter of part B is 2.0 with a 0.9 certainty. When we connect the two parts together, most times we simply combine the two functions together, and get a parameter. However, though the parameter may still be right, the confidence of it is much lower. 
</P><P>This problem may never happen in the ABC method, because instead of a single parameter, we can always get a posterior distribution. Thus, we can see the span of the distribution while combining the systems together, and try to control it through the process.
</P></DIV><DIV class="content" id="cocontent"><H3>Bio-Computer</H3><P><B>Our Idea</B></P><P>Founding on our existed system, we appoint the light signal provided in specific positions as an input and density of cells in certain points as an output.
</P><P><B>Feasibility Analysis</B></P><P>For such a bio-computing system, every cell stores and transfers its state under the control of neighboring signal. This would allow for a construction of a cellular automata system on a biological level. Though only a simple logical gate presented here, an infinite memory space could be assumed and such an equivalent Turing Machine would make almost any further complicated calculation at least theoretically possible.
</P><P><B>Advantage</B></P><P>The density of cells in certain points can be measured by light absorbtion, which would use light both as input and output, making it easy to execute. Meanwhile, combinations of several levels of logical gates would be available in rather tiny areas (even single colony).     
</P><P><B>Realization</B></P><P>See the colony a n-in, 1-out system. Use Boolean algebra parameter 1/0 to denote the existence or nonexistence of light in N input points of r1, r2, … to rn . The output is determined by whether the cellular density is larger than a threshold value of h. If it is larger than 1 then the output equals 1, otherwise the output equals 0. The output results can be judged by eyes or more professional instruments such as spectrophotometers.
</P><P><B>Results</B></P><P><B>One dimension model for a 2-Input NAND Gate</B></P><UL><LI>Light Input Location: 35, 45.
</LI><LI>Cellular Density Output Location: 40.
</LI><LI><A href="/File:Seuomodequ1.GIF" title="File:Seuomodequ1.GIF">File:Seuomodequ1.GIF</A></LI><LI>Truth Table:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TD> A </TD><TD> B </TD><TD> L </TD><TD align="center">Result
</TD></TR><TR><TD> 0 </TD><TD> 0 </TD><TD> 1 </TD><TD><A href="/File:Seuomodfig1.GIF" title="File:Seuomodfig1.GIF">450px</A></TD></TR><TR><TD> 1 </TD><TD> 0 </TD><TD> 1 </TD><TD><A href="/File:Seuomodfig2.GIF" title="File:Seuomodfig2.GIF">450px</A></TD></TR><TR><TD> 1 </TD><TD> 1 </TD><TD> 0 </TD><TD><A href="/File:Seuomodfig3.GIF" title="File:Seuomodfig3.GIF">450px</A></TD></TR></TBODY></TABLE><P><B>One dimension model for a 2-Input NOR Gate</B></P><UL><LI>Light Input Location: 38, 42.
</LI><LI>Cellular Density Output Location: 40.
</LI><LI><A href="/File:Seuomodequ2.GIF" title="File:Seuomodequ2.GIF">File:Seuomodequ2.GIF</A></LI><LI>Truth Table:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TD> A </TD><TD> B </TD><TD> L </TD><TD align="center">Result
</TD></TR><TR><TD> 0 </TD><TD> 0 </TD><TD> 1 </TD><TD><A href="/File:Seuomodfig4.GIF" title="File:Seuomodfig4.GIF">450px</A></TD></TR><TR><TD> 0 </TD><TD> 1 </TD><TD> 0 </TD><TD><A href="/File:Seuomodfig5.GIF" title="File:Seuomodfig5.GIF">450px</A></TD></TR><TR><TD> 1 </TD><TD> 1 </TD><TD> 0 </TD><TD><A href="/File:Seuomodfig6.GIF" title="File:Seuomodfig6.GIF">450px</A></TD></TR></TBODY></TABLE><P><B>Two dimension model for a 4-Input NAND Gate</B></P><UL><LI>Light Input Location: A(31,40), B(49,40), C(40,31), D(40,49).
</LI><LI>Cellular Density Output Location: (40,40).(Shown below)
</LI></UL><DIV class="center"><DIV class="floatnone"><A href="/File:Seuomodfig7.GIF" title="File:Seuomodfig7.GIF">500px</A></DIV></DIV><UL><LI><A href="/File:Seuomodequ3.GIF" title="File:Seuomodequ3.GIF">File:Seuomodequ3.GIF</A></LI><LI>Truth Table:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TD> A </TD><TD> B </TD><TD> C </TD><TD> D </TD><TD> L </TD><TD align="center">Result </TD><TD> Density
</TD></TR><TR><TD> 0 </TD><TD> 0 </TD><TD> 0 </TD><TD> 0 </TD><TD> 1 </TD><TD><A href="/File:Seuomodfig8.GIF" title="File:Seuomodfig8.GIF">400px</A></TD><TD> 54.9100
</TD></TR><TR><TD> 0 </TD><TD> 0 </TD><TD> 1 </TD><TD> 0 </TD><TD> 1 </TD><TD><A href="/File:Seuomodfig9.GIF" title="File:Seuomodfig9.GIF">399px</A></TD><TD> 41.2043
</TD></TR><TR><TD> 0 </TD><TD> 0 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD><A href="/File:Seuomodfig10.GIF" title="File:Seuomodfig10.GIF">400px</A></TD><TD> 29.6017
</TD></TR><TR><TD> 1 </TD><TD> 0 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD><A href="/File:Seuomodfig11.GIF" title="File:Seuomodfig11.GIF">400px</A></TD><TD> 21.2919
</TD></TR><TR><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 0 </TD><TD><A href="/File:Seuomodfig12.GIF" title="File:Seuomodfig12.GIF">400px</A></TD><TD> 16.2095
</TD></TR></TBODY></TABLE><P><B>Two dimension model for a 4-Input NOR Gate</B></P><UL><LI>Light Input Location: A(37,40), B(43,40), C(40,37), D(40,43).
</LI><LI>Cellular Density Output Location: (40,40).(Shown below)
</LI></UL><DIV class="center"><DIV class="floatnone"><A href="/File:Seuomodfig13.GIF" title="File:Seuomodfig13.GIF">500px</A></DIV></DIV><UL><LI><A href="/File:Seuomodequ4.GIF" title="File:Seuomodequ4.GIF">File:Seuomodequ4.GIF</A></LI><LI>Truth Table:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TD> A </TD><TD> B </TD><TD> C </TD><TD> D </TD><TD> L </TD><TD align="center">Result </TD><TD> Density
</TD></TR><TR><TD> 0 </TD><TD> 0 </TD><TD> 0 </TD><TD> 0 </TD><TD> 1 </TD><TD><A href="/File:Seuomodfig14.GIF" title="File:Seuomodfig14.GIF">400px</A></TD><TD> 54.9100
</TD></TR><TR><TD> 0 </TD><TD> 0 </TD><TD> 1 </TD><TD> 0 </TD><TD> 0 </TD><TD><A href="/File:Seuomodfig15.GIF" title="File:Seuomodfig15.GIF">400px</A></TD><TD> 18.2540
</TD></TR><TR><TD> 0 </TD><TD> 0 </TD><TD> 1 </TD><TD> 1 </TD><TD> 0 </TD><TD><A href="/File:Seuomodfig16.GIF" title="File:Seuomodfig16.GIF">400px</A></TD><TD> 8.5590
</TD></TR><TR><TD> 1 </TD><TD> 0 </TD><TD> 1 </TD><TD> 1 </TD><TD> 0 </TD><TD><A href="/File:Seuomodfig17.GIF" title="File:Seuomodfig17.GIF">400px</A></TD><TD> 6.1014
</TD></TR><TR><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 0 </TD><TD><A href="/File:Seuomodfig18.GIF" title="File:Seuomodfig18.GIF">400px</A></TD><TD> 4.4611
</TD></TR></TBODY></TABLE><P><B>Summary</B></P><P>Dating back to even 1990s, the potential of bio-computing has already been agreed on by scientists. For this 6th generation of neuron computer, bio-logical gates are highly likely to be the fundamental unit of it. Though somehow naïve compared to electronic logical gates, we can see from our model the possibility of constructing more reliable bio-logical gates. The density of cells in certain points can be measured by light digesting, which would use light both as input and output, making it easy to execute. Meanwhile, combinations of several levels of logical gates would be available in rather tiny areas (even single colony). It has shown satisfying results under the cascaded condition. 
</P><P>These days, the development of normal electronic chips has met barriers from quantum level. Moreover, the requirement for smaller and more energy-saving chips has been increasing, which makes the research on bio-logical gates more meaningful than ever. However, the problem of volume and time-delay in our design should be eliminated by applying more sophisticated algorisms for more compatible and systematic design. Anyway, the bio-logical gate design under our system is still promising and worth improving.
</P><P><B>Modeling Parameters</B></P><UL><LI>Cellular Movement Constant: V=1;
</LI><LI>Threshold Value for Output 1/0: h=20;
</LI><LI>Releasing Constant: q=150;
</LI><LI>Decomposition Constant: y=1;
</LI><LI>Diffusion Constant: D=0.20;
</LI></UL></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:SEU_O_China/Model">http://2012.igem.org/Team:SEU_O_China/Model</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SEU_O_China/Model" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SEU_O_China/Model" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SEU_O_China/Model&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SEU_O_China/Model&amp;oldid=222300" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>