"
Page
Discussion
View source
History
teams
Log in
 
Team:SEU O China/Result
From 2012.igem.org
Results
Biobricks
Judging form
Attribution
Experiment Results
Division Inhibition
Light Sensor
Modeling Results
Light Induced System
Auto-differentiation
Bio-Computer
Biobricks
How our system works:
Our Favorite New Parts:
Name:
TBY
ID:
BBa_K897720
Type:
RNA
Length:
382bp
Description:
This part encodes a fragment of antisense FtsZ RNA protected by paired termini structure under T7 promoter. Inhibiting the expression of FtsZ, it can strongly repress the cell division in E.coli.
*TBY means "Te Bie Yuan", or "very round" in English...
Other Parts We like:
Name:
FtsZ_B0015
ID:
BBa_K897624
Type:
RNA
Length:
283bp
Description:
This part encodes a fragment of antisense FtsZ RNA combined with a terminator. It can repress the cell division in E.coli when used with a promoter.
.
Name:
Paired Termini
ID:
BBa_K897318
Type:
Other
Length:
146bp
Description:
This part encodes a paired termini with a MCS in the middle of it. By insert target antisense RNA into the MCS, it can protect the antisense RNA from degradation and enhance the efficiency or antisense RNA silencing.
.
Name:
S03419_B0015
ID:
BBa_S05053
Type:
Intermediate
Length:
2418bp
Description:
This part is the combination of S03419 and B0015, a part of light sensor system. Once equipped with a promoter, BBa_S05053 can produce the Cph8,the key menbrane-bound protein of red light sensor.
.
Name:
K081024_I13507
ID:
BBa_S05054
Type:
Intermediate
Length:
1828bp
Description:
BBa_S05054 is an intermediate constructed by the conjunction of K081024 and I13507. As a part of light sensor, it contains the PcyA coding sequence(I15009), the OmpR-controlled promoter(ROO82) and the mRFP reporter（I13507. With a combination of J13002 and I15008,this part can produce the enzymes, whicn can catalyzed the two procedures required by conversion of haem into PCB,and offer the OmpC promoter.
.
Name:
R0010_P0451
ID:
BBa_S05055
Type:
Intermediate
Length:
1138bp
Description:
This part is the pre-half of the collins toggle switch, which include a lac promoter (ROO1O) and cI coding part(P0451). It can be used as a toggle switch when combined with R0051_I732820.
.
Data For Pre-existing Parts:
BBa_M30109 Experience:
BBa_M30109 seems not as available as mentioned on the website. We tried PCR and extraction of plasmids successively but both of them failed. It might be the size that resulted in the instability of those test procedures since BBa_M30109 has an astonishing size of 4333bp: with such a giant fragment it may lead to multiple mistakes in replication.
BBa_R0051 Experience:
BBa_R0051 does not work well due to its rather small size. We transformed it for three times, and none of them succeed. We designed primers to PCR it out from K091230 successfully, but failed to perform the digestion since it is too small to extract from gel. It may work better if adding some nonsense sequence before the promoter.
BBa_J5526 Experience:
We tried to transform BBa_J5526 several times. It grows well, but never turns red.
BBa_S03419 Experience:
BBa_S03419 is available and the sequence is proved to be correct.
BBa_B0015 Experience:
BBa_B0015 is available and the sequence is proved to be correct.
BBa_K081024 Experience:
BBa_K081024 is available and the sequence is proved to be correct.
BBa_I13507 Experience:
BBa_I13507 is available and the sequence is proved to be correct.
BBa_I15008 Experience:
Unfortunately, the inversion of BBa_I15008 succeeded but we lost the correct seed due to an accident.
BBa_J13002 Experience:
Both PCR and Extraction of plasmids suggest a correct size of J13002, but we failed to get the available and reliable sequencing report. So we cannot confirm that J13002 is sequencing-correct or not.
BBa_R0010 Experience:
Sadly, problems came to our experiments when we tried to recover the BBa_R0010’s seed conserved by ourselves. The seed we conserved on that day was somehow polluted so that we failed to extract the correct and available plasmids. But the sequencing report was once proved to be correct.
BBa_R0082 Experience: (together with BBa_R0083 and BBa_R0084)
Both PCR and Extraction of plasmids suggest a correct size of those 3 promoters once, but we failed to get the available and reliable sequencing report since the seeds seem to be unstable and the size of them is too small.
Division Inhibition
The antisense FtsZ, protected by hair-pin structure and termed TBY or BBa_897720, is regulated by T7 promoter and regarded as the core part of our project. The antisense FtsZ sequence mainly aims at silenting the FtsZ gene and control the self-renewal and differentiation of E.coli. Since the antisense FtsZ gene in BBa_897720 is regulated by the T7 promoter that is specifically combined with T7 RNA polymerase, a endogenous protein in BL21(DE3) strain under the control of the lac UV5 promoter inducible by IPTG, the number of colonies is expected to decrease with the an increase in IPTG. The confirmatory experiment below verifys the validity of TBY and our hypothesis.
After integrating the plasmid with an insert of BBa_897720 into BL21(DE3) strain, we cultivated those bacteria on the surface of solid LB medium in 4 dfferent IPTG levels. The conditions was strictly controlled at the temperature of 37 ℃ and it last for 10 hours.  The controls aims to rule out the possibility of damaged to the bacteria from IPTG. Details are illustrated below:
IPTG(0.0mM)
IPTG(0.5mM)
IPTG(1.0mM)
IPTG(2.0mM)
TBY  *0.01
TBY  *0.1
TBY-  *0.1
TBY-  *0.01
TBY represents the BL21(DE3) bacteria carring plasmids with an insert of BBa_K897720. TBY- means that the BL21(DE3) bacteria carried the plasmids as same as the plasmids of TBY but without the insert of BBa_K8977720.
TBY(-) are diluted tenfold and hundredfold separately, marked as “*0.1” and “*0.01”.
The number in the parenthesis following the “IPTG” means the concentration of IPTG.
For original full-size picture, see here .
In order to analyse the repressible effect of FtsZ, we calculate the colony coverage of every sample. Details are illustrated in the Model Part, and the results of analysis are shown below.
IPTG(0.0mM)
IPTG(0.5mM)
IPTG(1.0mM)
IPTG(2.0mM)
TBY  *0.01
TBY  *0.1
TBY-  *0.1
TBY- *0.01
Colony coverage Map
IPTG(0.0mM)
IPTG(0.5mM)
IPTG(1.0mM)
IPTG(2.0mM)
TBY*0.01
93.23
9.21
0.65
0.29
TBY*0.1
76.30
44.38
27.74
19.37
TBY-*0.1
94.03
95.85
92.39
93.55
TBY-*0.01
80.01
82.51
86.74
88.97
Percentage of colony coverage
FtsZ repressible effect
Both of the red line and the blue line in the illustration show that the colony coverage decreases while the level of IPTG increases, which suggests that the growth of colony is somehow repressed. Moreover, the yellow line reveals that the IPTG has little side effect on colony. As a result, FtsZ truly represses the growth of bacteria and also provide experimental evidence for our project scheme.
For the efficiency of paired termini structure, we have done some additional experiments. Detailed results can be seen here.
https://2012.igem.org/Team:SEU_O_China/Result/AddExperiment
Light Sensor
RED LIGHT
Scheme A
The experiments of light sensor part starts from BBa_M30109, the devised composite that can produce Cph8 and ho1 & pcyA,the enzymes required by convertion of Haem into PCB.The OmpC promoter is also involved in this composite. However, BBa_M30109 is not as available as mentioned on the website. The PCR failed to amplificate the fragment of BBa_M30109. It might be the giant size of this part results in the failure of PCR.Besides the plasmids extracted from the E.coli bacteria can hardly be told from the electrophoretogram (Fig.1). As a result, we decided to choose "scheme B".
Scheme B
We deside to constructe two composites that can produce Cph8, termed c-Cph8 and enzymes we need, named c-PCB, seperately.
C-Cph8 basically contains BBa_R0010, BBa_S03419 and BBa_B0015. The combination of BBa_S03419 and BBa_B0015 first succeeded at the beginning of August. The sequence report suggest the correct sequencing information of both BBa_S03419 and BBa_B0015. Sadly, problems came to our experiments when we tried to recover the BBa_R0010’s seed conserved by ourselves. The seed we conserved on that day was somehow polluted so that we failed to extract the correct and available plasmids. So we just finished the construction of BBa_S03419 and BBa_B0015, termed BBa_S05053, which contained an RBS, BBa_I15008 and the terminator BBa_B0015.
C-PCB is expected to produce the two vital enzymes, ho1 and pcyA that catalyze the procedure of conversion of haem into PCB. We planned to combine BBa_J13002, BBa_I15008, BBa_K081024 and a reporter protein(mRFP) together. When we finished the combination of BBa_K081024 and BBa_I13507(mRFP), we were astonished to find out mRFP was expressed without Cph8. Meanwhile, the combination of BBa_I15008 and BBa_J13002 failed. The good news is that the sequencing report confirmed the sequences of BBa_13002, BBa_K081024 and BBa_I13507 were seperately right. Unfortunately, the inversion of BBa_I15008 succeeded but we lost the correct seed due to an accident. As a result, we finished the combination of BBa_K081024 and BBa_I13507, termed BBa_S05054 automatically by partsregistry.org.
Discussion
The expression of mRFP in Scheme B suggest that the OmpC in BBa_K081024 was somehow activated. Further work was conducted to figure out why the OmpC seemed to present a function of a continuous promoter. After blasting the sequence of Cph1 at NIBC ( http://www.ncbi.nlm.nih.gov ), we discovered that the BL21(DE3) strain has endogenous EnvZ, which is the key part of Cph1that can phosphorylate the OmpR and cause the activation of OmpC followed by the expression of downstream genes, BBa_I13507. Hence, once the strain has endogenous EnvZ, BBa_K081024 & BBa_I13507, also known as BBa_S05054 expresses mRFP without Cph8 not to mention the regulation of red light.
Now it is confirmed that as long as equipped with a promoter,RBS and BBa_I15008, the OmpC promoter in BBa_S05054 can work normally in a endogenous-EnvZ-free system. Besides, BBa_S05054 can also be utilized to test the exisitence of endogenous EnvZ.
BLUE LIGHT
Blue light system is much simple compared with red light system, since it involves just one kit, BBa_K238013. It eventually failed in the procedures of combination with other part of our project.
Toggle switch
The toggle switch part of experiment is not very successful.
We have constructed the lac-cI part of the toggle switch, but the pcI-lacI part is failed. The most problem happened around the cI promoter, R0051, which is only 49bp in length. After the failure of several transformation and depletion of the distributed kit, we try to PCR this part out from K091230, a cI promoted mRFP reporter.
Though the PCR works well and we actually get the target fragment, we failed to perform follow experiment because we cannot extract DNA from the enzyme digestion product since the 49bp size is beyond the ability of our Gel Extraction Kit.
After that, we tried to add some nonsense sequence before K091230, and PCR the sequence and R0051 out, but finally did not make it before the deadline.
Judging form
iGEM Medals for Non-Software Teams
We believe our team deserves the Gold Medal.
Because we met the following criteria
Requirements for a Bronze Medal:
Y Register the team, have a great summer, and plan to have fun at the Regional Jamboree.
Y Successfully complete and submit this iGEM 2012 Judging form.
Y Create and share a Description of the team's project using the iGEM wiki and the team's parts using the Registry of Standard Biological Parts.
Y Plan to present a Poster and Talk at the iGEM Jamboree.
Y Enter information detailing at least one new standard BioBrick Part or Device in the Registry of Standard Biological Parts. Including:
Y Primary nucleaic acid sequence
Y Description of function
Y Authorship
Y Safety notes, if relevant.
Y Acknowedgment of sources and references
Y Submit DNA for at least one new BioBrick Part or Device to the Registry.
Additional Requirements for a Silver Medal:
Y Demonstrate that at least one new BioBrick Part or Device of your own design and construction works as expected; characterize the operation of your new part/device.
Y Enter this information and other documentation on the part's 'Main Page' section of the Registry
Part Number(s): BBa_K897318 , BBa_K897624 , BBa_K897720
Additional Requirements for a Gold Medal:
Y Help another iGEM team by, for example, characterizing a part, debugging a construct, or modeling or simulating their system.
Link to this information on your wiki. Page name: https://2012.igem.org/Team:SEU_O_China/Result/Helping
Y Outline and detail a new approach to an issue of Human Practice in synthetic biology as it relates to your project, such as safety, security, ethics, or ownership, sharing, and innovation.
Link to this information on your wiki.Page name: https://2012.igem.org/Team:SEU_O_China/HumanPractice#sarcontent
iGEM Prizes
All teams are eligible for special prizes at the Jamborees. To help the judges, please indicate if you feel you should be evaluated for any of the following special prizes:
Y Best Human Practice Advance
Y Best Experimental Measurement
Y Best Model
Please explain briefly why you should receive any of these special prizes:
We did an interesting and useful human practice work, use an Approximate Bayesian Computation method for experiment data measurement, construct a cellular automata model to simulate our project.
Team_Parts
To help the judges evaluate your parts, please identify 3 of your parts that you feel are best documented and are of the highest quality.
Best new BioBrick part (natural): None
Best new BioBrick part (engineered): BBa_K897720
Best improved part(s): None
List any other parts you would like the judges to examine: BBa_K897624 , BBa_K897318 , BBa_K897720
Please explain briefly why the judges should examine these other parts:
Those parts are constructed by ourselves, have detailed experiment results, and can be proved workable.
iGEM Safety
For iGEM 2012 teams are asked to detail how they approached any issues of biological safety associated with their projects.
The iGEM judges expect that you have answered the four safety questions (Safety page) on your iGEM 2012 wiki.
Please provide the link to that page: Page name: https://2012.igem.org/Team:SEU_O_China/Safety
Attribution and Contributions For iGEM 2012 the description of each project must clearly attribute work done by the team and distinguish it from work done by others, including the host labs, advisors, and instructors. Please provide the link to that page, or comments in the box below:
See Attribution on this page.
Attribution
Our project comes into being with our heart and soul under the great support and help from teachers, organizations and people around us.
We would like to convey our sincere thanks to those people for their great helps and unswerving supports. Those were just what we needed most but never expected!
FINANCIAL SUPPORTS
Ms. Xia Fang, Section Chief of Research section of Eductaional Administration, provides us with a great financial supports.
Prof. Zuhong Lu, PhD Advisor,professor of State Key Laboratory of biological electronics and Biomedical Engineering School, offers us an unexpected finacial support. Prof. Lu ‘s not only help us solve financial problems but also encourage us and torch us when we get a halt in our project and promise us the facility&lab permission when needed.
MANAGERIAL ADVICES
Mr. Feng Wang, Vice President of Biomecial Engineering School, provides us a great help on project manangement.
Ms. Lei Huang, Senior Engineer of Biomedical Engineering School, helps us manage the project and give advices when we were in a tight corner.
FACILITY & LAB PERMISSION
Prof.Jingke Wang, PhD Advisor,professor of State Key Laboratory of biological electronics and Biomedical Engineering School, promises us the facility&lab permission and gives us experimental advices.
Prof. Zuhong Lu (mentioned above)
ACADEMIC GUIDANCE
Prof. Zuhong Lu (mentioned above)
Mr.Jingke Wang (mentioned above)
Chunpeng He, a doctor student of Biomedical Engineering School, equips us with practical skills required in wet experiment and involved in experimental design.
UNSPECIFIC BUT IMPRESIVE ASSISTANCE
Team SEU_A provides us support and help on project and experiment and shares an amazing memory in lab.
Nanjing_China & Nanjing_China_Bio offer us kits and advices we need.
Undergraduate student Mengyi Zhang, who firstly introduce synthesis biology to us, and organize the iGEM competition in our school, will always be kept in mind.
Contribution
Passions and efforts of persons in our team are imbues every part of the project.
Project design
Muyuan Cheng, Xuelong Fan, Xiaodan Xu and Xingyu Zhou mainly construct the blueprint of the project.
Wet experiments
Shen Gao takes in charge of the management of wet experiment.
Zhengyang Jia, Muyuan Cheng, Xuelong Fan, Xiaodan Xu and Xingyu Zhou are also involved in this part and devote plenty of time and energy during the past 3 months.
Mathematical Modeling
Xintong Lin, Kai Gui, and Muyuan Chen have done great job of modeling.
Document
All the team members.
Wiki pages construction
Muyuan Chen and Xiaodan Xu complete most of the work.
Visual Design
Peng Cheng and Le Zhou design the visual style of our logo, wiki page, and presentation ppt.
Unspecific but supportive assistance
Greatly thank Xuelong Fan for doing all kind of administrative work, including contacting professors, raising funds, dealing with formal procedures and so on.
We also helped another iGEM team, SEU_A . For more details, see the helping page .
Light Induced System
An overview of the system can be seen here.
Micro-scale Model:
Detailed approach can be seen here.
Control group result
Experiment group result
Macro-scale Model:
Detailed approach can be seen here.
Control group result
Experiment group result
Auto-differentiation System
An overview of the system can be seen here.
Simulation Results
Detailed approach can be seen here.
Control group result
Experiment group result
Bio-Computer
System overview and detailed approach can be seen here.
One dimension model for a 2-Input NOR Gate
Light Input Location: 38, 42.
Cellular Density Output Location: 40.
File:Seuomodequ2.GIF
Truth Table:
A
B
L
Result
0
0
1
450px
0
1
0
450px
1
1
0
450px
Two dimension model for a 4-Input NAND Gate
Light Input Location: A(31,40), B(49,40), C(40,31), D(40,49).
Cellular Density Output Location: (40,40).(Shown below)
500px
File:Seuomodequ3.GIF
Truth Table:
A
B
C
D
L
Result
Density
0
0
0
0
1
400px
54.9100
0
0
1
0
1
399px
41.2043
0
0
1
1
1
400px
29.6017
1
0
1
1
1
400px
21.2919
1
1
1
1
0
400px
16.2095
Retrieved from " http://2012.igem.org/Team:SEU_O_China/Result "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
