<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tsinghua-E_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Tsinghua-E/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tsinghua-E/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tsinghua-E/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tsinghua-E/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tsinghua-E/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tsinghua-E/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="nav-wrap"><UL id="nav"><LI><A href="https://2013.igem.org/Team:Tsinghua-E" id="homelink">Home</A></LI><LI><A class="drop" href="https://2013.igem.org/Team:Tsinghua-E/Team" id="teamlink">Team</A></LI><LI><A href="https://2013.igem.org/Team:Tsinghua-E/Team">Members</A></LI><LI><A class="dropheader" href="https://2013.igem.org/Team:Tsinghua-E/Team/Outtakes">Outtakes</A></LI><LI><A class="dropheader" href="https://2013.igem.org/Team:Tsinghua-E/ContactUs">University</A></LI><LI><A href="http://tsinghua.hdskjz.com/default.aspx" target="_blank">Laboratory</A></LI></UL><LI><A class="drop" href="https://2013.igem.org/Team:Tsinghua-E/Project" id="projectlink">Project</A></LI><LI><A href="https://2013.igem.org/Team:Tsinghua-E/Project">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Tsinghua-E/Result">Results</A></LI><LI><A href="https://2013.igem.org/Team:Tsinghua-E/Film">Film</A></LI><LI><A href="/Team:Tsinghua-E/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Tsinghua-E/Parts" id="partslink">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Tsinghua-E/Part1" id="partslink">Part1</A></LI><LI><A href="https://2013.igem.org/Team:Tsinghua-E/Part2" id="partslink">Part2</A></LI><LI><A href="https://2013.igem.org/Team:Tsinghua-E/Part3" id="partslink">Part3</A></LI><LI><A href="https://2013.igem.org/Team:Tsinghua-E/Part4" id="partslink">Part4</A></LI><LI><A class="drop" href="https://2013.igem.org/Team:Tsinghua-E/Notebook" id="notebooklink">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Tsinghua-E/Notebook">Journal</A></LI><LI><A href="https://2013.igem.org/Team:Tsinghua-E/Protocol">Protocols</A></LI><LI><A class="drop" href="https://2013.igem.org/Team:Tsinghua-E/Humanpractice" id="outreachlink">Humanpractice</A></LI><LI><A href="https://2013.igem.org/Team:Tsinghua-E/Humanpractice">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Tsinghua-E/Biofarm">Synthetic biofarm</A></LI><LI><A href="https://2013.igem.org/Team:Tsinghua-E/Bacteriapainting">Bacteria painting</A></LI><LI><A href="https://2013.igem.org/Team:Tsinghua-E/Lecture">Lecture</A></LI><LI><A href="https://2013.igem.org/Team:Tsinghua-E/Attributions" id="sponsorslink">Attributions</A></LI></DIV><DIV style="position:absolute;top:176px;left:56px;width:105px;height:186px;z-index:99;"><UL class="leftdh"><LI><A href="https://2013.igem.org/Team:Tsinghua-E/Notebook">Journal</A></LI><LI><A href="https://2013.igem.org/Team:Tsinghua-E/Protocol">Protocols</A></LI></UL></DIV><DIV class="neirong" style="height:10625px;"><P>Notice:Except for specially mentioned, all the following vectors were constructed by In-Fusion HD Cloning Kit (produced by Clontech). To be brief, every fragment of desired DNA was linearized by primers designed following the protocol provided by Takara infusion cloning guide to ensure accurately 15 base pair (bp) overlap homology with each adjacent fragments. The gel-purified fragments were mixed with 2uL infusion cloning kit Premix solution and ddH2O was used to adjust the mixture to 10uL. The amount of the added fragments was based on the Takara infusion cloning guide.</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week1.286.23-6.30.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week1(6.23-6.30)</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Week2.287.1-7.7.29"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Week2(7.1-7.7)</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Week3.287.8-7.14.29"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Week3(7.8-7.14)</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Week4.287.15-7.22.29"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Week4(7.15-7.22)</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Week5.EF.BC.887-23-7.29.EF.BC.89"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Week5（7-23-7.29）</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Week6.EF.BC.887.30-8.5.EF.BC.89"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Week6（7.30-8.5）</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Week7.288.6-8.13.29"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Week7(8.6-8.13)</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Week8.EF.BC.888.14-8.21.EF.BC.89"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Week8（8.14-8.21）</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Week9.288.22-8.29.29"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Week9(8.22-8.29)</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Week10.288.30-9.5.29"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Week10(8.30-9.5)</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Week11.289.6-9.12.29"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Week11(9.6-9.12)</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Week12.289.13-9.19.29"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Week12(9.13-9.19)</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Week13.289.20-9.27.29"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">Week13(9.20-9.27)</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Week1.286.23-6.30.29"><B>Week1(6.23-6.30)</B></SPAN></H2><P>Literature was reviewed carefully and the corresponding gene candidates were confirmed. Some of them were submitted to be synthesized. Some of students were trained for basic molecule biology experiment operation. 
</P><H2><SPAN class="mw-headline" id="Week2.287.1-7.7.29"><B>Week2(7.1-7.7)</B></SPAN></H2><P>The primers needed in our project is designed and synthesized. Some chemicals were also purchased. Some of students were trained for basic molecule biology experiment operation and further some characterization experiment operation.
</P><H2><SPAN class="mw-headline" id="Week3.287.8-7.14.29"><B>Week3(7.8-7.14)</B></SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure.1 <I>dnaQ</I> PCR amplification from <I>E.coli</I> BL21(DE3)</DIV></DIV></DIV><P>We cloned <I>dnaQ</I> gene from <I>E.coli</I> BL21(DE3) by PCR and utilized overlap extension PCR to introduce mutagenesis in <I>dnaQ</I> by the following protocol.
</P><P>Firstly, we used <I>E.coli</I> BL21 (DE3) strain as template to amplify <I>dnaQ</I> gene by PCR with mut-F and mut-R primers, respectively. After purification of PCR product (the 〖E.Z.N.A.〗^TM Gel Extraction Kit, produced by Omega Bio-tek), it was used as template to introduce point mutation by two parallel PCRs with the following two sets of primers: mut-L73W-F and mut-L73W-R (generating L73W); mut-A164V-F and mut-A164V-R (generating A164V). The three PCR products were gel-purified and combined by 5ng,respectively and assembled with mut-F and mut-R primers. The resulting overall product was T-cloned into Takara pMD-19 T-vector (produced by Takara BioTechnology (DALIAN)CO.,LTD.) and sequenced to confirm the sequence. The L73W and A164V mutant BL21 (DE3) <I>dnaQ</I> was named after <I>mutD</I>.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure. 2 tryptophan sensor construction with <I>lacZ</I> as output signal</DIV></DIV></DIV><P>Maltose hydrolase gene <I>malQ</I> was cloned from <I>E.coli</I> JW3995.
</P><P>In parallel, we also constructed tryptophan sensor circuit. According to the report about the translating ribosome by nascent peptide based tryptophan degradation gene cluster regulation mechanism, we previously synthesized the corresponding <I>tnaC</I> coupling <I>Rho</I> sequence. This gene cluster was assembled with <I>lacZ</I> gene by overlap PCR and the gel-purified fragment was cloned between NcoI and HindIII sites of pTrc99A vector by restriction enzyme digest and T4 ligase induced ligation. All the information about primers was shown in “primer information”. This plasmid was named after pTrc99A_trp sensor_lacZ.
</P><H2><SPAN class="mw-headline" id="Week4.287.15-7.22.29"><B>Week4(7.15-7.22)</B></SPAN></H2><P>In this week, we tried to clone <I>mutD</I> and <I>sfGFP</I> cluster downstream of <I>ara</I>BAD promoter (pBAD). We amplified pSC101-Cm-pBAD and <I>sfGFP</I> fragment from plasmid AraC_pBAD_CI_OR222-sfGFP, which was kindly provided by Professor Ouyang Qi in Peking University. <I>mutD</I> was introduced by three different ribozyme binding site (RBS) sequence upstream and also overlap homology with primers. All the primers information was listed in the appendix, primer information. The three fragments were assembled by infusion cloning as described above. The partial sequence of RBS-mutD op-sfGFP was sequenced to confirm. The three constructed plasmids were named after pBAD_B0030-mutD-sfGFP, pBAD_B0032-mutD-sfGFP and pBAD_SDA_RBS-mutD-sfGFP, respectively.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure. 3 <I>in vivo</I> mutation machine construction with <I>mutD</I> and <I>sfGFP</I> series expression</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:422px;"><DIV class="thumbcaption">Figure. 4 measurement of novel translating ribosome based tryptophan sensor by galactosidase activity</DIV></DIV></DIV><P>In this week, we also tested our synthetic tryptophan sensor performance by galactosidase activity. Some protocol seemed not to work for some culture condition and measurement conditions. The final successful protocol was shown as below. Firstly, we picked up <I>E.coli</I> JM109 carrying pTrc99A_trp sensor_lacZ single clone and cultured it in Luria-Bertani (LB) medium (Tryptone 10g/L, Yeast Extract 5g/L, NaCl  10g/L) with concentration of ampicillin at 50mg/L for overnight. On the next morning, the 600nm optical density (OD600) of the culture was measured to be 3.090. This seed was transferred into 2mL LB medium with the same concentration of ampicillin by a ratio of 5% in a 5mL 48 well culture plate (Huayang Chengxun Cor. Beijing China). After 1.6h, OD600 of every well reached about 0.6. Each well was also supplemented with 720uL fresh LB medium. Isopropylβ-D-1-Thiogalactopyranoside (IPTG) and tryptophan stock solution was also added into each well to the final concentration of 0.4mM and a gradient of 0.4, 0.5, 0.6, 0.8, 1, 2, 3mM, respectively. After 21h, the galactosidase activity was measured as described below.
</P><P>Firstly, OD600 of the final culture was measured. Then mix 400uL culture with 2.8mL Z buffer (Na2HPO4.7H2O 16.1g/L, NaH2PO4.H2O 5.5g/L, KCl 0.75g/L, MgSO4.7H2O 0.25g/L, 2-thiol ethanol 2.7ml/L, adjusted pH to 7.0). Nextly, mix 2mL mixture with 250uL 1mg/mL MUG DMSO solution to react for 15min. Finally, 300uL 1M NaOH solution was added to quench the reaction and the fluorescence was measured with excitation and emission wavelength at 360 and 445 nm, respectively. The fluorescence intensity was normalized by culture OD600 and reaction time 15min as the activity of galactosidase (MUG units=F360/445/OD600/t). The results of the test were shown in Figure. 4.
</P><H2><SPAN class="mw-headline" id="Week5.EF.BC.887-23-7.29.EF.BC.89"><B>Week5（7-23-7.29）</B></SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:422px;"><DIV class="thumbcaption">Figure. 5 sweet pressure part construction with tac as promoter</DIV></DIV></DIV><P>	We constructed a plasmid expressing <I>malQ</I> responding to the concentration of tryptophan. Tryptophan sensor part and <I>malQ</I> were PCR amplified and gel-purified while pTrc99A was linearized between NcoI and PstI. Then the fragments were assembled by Takara infusion cloning kit. The plasmid was named pTrc99A_trp sensor_malQ. We firstly transformed the infusion product into DH5a component cell and 20 single colonies were cultured and confirmed by PCR. And then we extracted the plasmid from the confirmed transformants with the 〖E.Z.N.A.〗^TM  plasmid mini kit I (produced by Omega Bio-tek) and transformed the plasmid into <I>E.coli</I> JW3379. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:422px;"><DIV class="thumbcaption">Figure. 6 growth test of JW3379 transformed with pTrc99A_trp sensor_malQ</DIV></DIV></DIV><P>After completing the construction, we tested whether the <I>E.coli</I> JW3379 transformed with pTrc99A_trp sensor_malQ have the growth dependence on tryptophan. We cultured the transformed <I>E.coli</I> JW3379 in M9 media (100 mL Solution I(NaCl 0.5g/L,NH_4 Cl 1g/L ,Na_2 HPO_4∙〖12H〗_2 O 17.1g/L, KH_2 PO_4  3g/L),100 µL 1M MgSO_4,500µL 0.1M CaCl_2  ,4mL 20%(mass fraction) maltose) containing maltose as the single carbon source and added tryptophan to a concentration gratitude of 0 mM、1 mM、2 mM、3 mM、10 mM、20mM while adding the IPTG with OD600 reaching  0.6. The growth curve was shown in Figure.6.
</P><P>Opposing to our expectation, the growth has a negative dependence on tryptophan. It may be caused by the little demand to <I>malQ</I> of the cell and thus the high expression level of <I>malQ</I> can become a burden to the growth of the cell. This problem might be more disturbing for tac promoter in original plasmid pTrc99A was not strict so the leakage effect was serious.
</P><H2><SPAN class="mw-headline" id="Week6.EF.BC.887.30-8.5.EF.BC.89"><B>Week6（7.30-8.5）</B></SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:422px;"><DIV class="thumbcaption">Figure.7  malQ enzyme activity <I>in vivo</I> against the concentration of tryptophan 12 hours after IPTG induction</DIV></DIV></DIV><P>To confirm our hypothesis about the tryptophan dependent malQ expression system, we tested the enzyme activity of malQ present in the cell 12 hours after adding IPTG. The enzyme activity is defined as the moles of maltose to glucose convertion per minute per OD600 at 37℃. And the glucose assay was conducted using the EnzyChromTM Glucose Assay Kit (produced by BioAssay Systems). We plotted the enzyme activity against the concentration of tryptophan and the result was shown as figure.7.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure.8 <I>tetA</I> cloned from pCM110</DIV></DIV></DIV><P>It was shown that when the concentration of tryptophan was below 10mM, the maltase activity increased with the level of tryptophan, which was the same as our expectation. However, when the concentration of tryptophan continued to increase, the enzyme activity started to decrease, showing the upper limit of tryptophan to be more or less 10mM.
</P><P>In parallel, we constructed tryptophan sensor coupling <I>tetA</I> circuits which can provide selection pressure to tryptophan provider microorganisms by antibiotics resistance. Similar to <I>mutD</I>, three different RBS sequences (B0030, B0032, RBS derived from the upstream of wild type <I>tnaA</I>) were introduced upstream of <I>tetA</I> (cloned from pCM110) by PCR amplification. Plasmid pTrc99A_trp sensor_lacZ was also linearized by PCR to remove <I>lacZ</I> and at the same time introduced with tetA overlap homology. All the primers information was listed in the appendix, primer information. The partial functional part of this plasmid was sequenced. The three constructed plasmids were named after pTrc99A_trp sensor_B0030_tetA, pTrc99A_trp sensor_B0032_tetA and pTrc99A_trp sensor_B0030_tetA, respectively. In this iGEM project, they were named after bitter pressure part 30, 32 and ori, respectively.
</P><H2><SPAN class="mw-headline" id="Week7.288.6-8.13.29"><B>Week7(8.6-8.13)</B></SPAN></H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure. 9 sweet pressure part construction with pBAD as promoter</DIV></DIV></DIV><P>	Simultaneously, because of the failure of our sweet part to have desired growth dependence on tryptophan measured in the last week, we redesigned our circuit and replaced tac promoter in pTrc99A with more strict <I>ara</I>BAD promoter (pBAD). At the same time, we designed random RBS library to control the expression level of malQ. In this way, a proper maltase expression level could be selected and expected to avoid the overexpression problem. The pBAD sequence was derived from (pRedET (amp),produced by Red^R/ ET^RRecombination). Hence, pBAD and tryptophan sensor were PCR amplified, gel-purified and assembled by overlap PCR.
</P><P>A 5’ end RBS random primer (malQ_RBS_library-F) for <I>malQ</I> was designed to produce the RBS library. pTrc99A plasmid was PCR amplified to remove tac promoter sequence and simultaneously linearized. These three gel-purified fragments were assembled by Takara infusion cloning kit. The plasmid with a random RBS library for <I>malQ</I> was named after pTrc99A_pBAD_trp sensor_random RBS_malQ. For the poor efficiency of the <I>E.coli</I> JW3379 component cell, we firstly transformed the infusion product into <I>E.coli</I> DH5a component cells and 120 single colonies were cultured and confirmed by PCR. After this step, we cocultured confirmed transformants in a 10mL LB medium and extracted the plasmid with the 〖E.Z.N.A.〗^TM  plasmid mini kit I (produced by Omega Bio-tek). The plasmid was transformed into JW3379 and 48 single clones were picked. JW3379 chemically component cells were prepared by the following protocol. This library was named after sweet pressure part RBS library in this iGEM project.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:422px;"><DIV class="thumbcaption">Figure. 10 mutation rate measurement induced by our engineered <I>mutD</I> mutator</DIV></DIV></DIV><P>In this week, we also measured the mutation rate increase induced by mutD mutator. The basic protocol was provided as attachment “mutation rate measurement”. To be brief, strains <I>E.coli</I> JM109 carrying vectors pBAD_B0030-mutD-sfGFP, pBAD_B0032-mutD-sfGFP and pBAD_SDA_RBS-mutD- sfGFP were cultured in LB medium and induced by 1g/L arabinose and the specific fluorescence of sfGFP (excitation 488nm, emission 512nm) (fluorescence to OD600) was measured. Thus, we made sure that pBAD_B0030-mutD-sfGFP, and pBAD_SDA_RBS-mutD- sfGFP have relatively higher mutD expression. Thus, only the mutation rate of pBAD_B0030-mutD-sfGFP, and pBAD_SDA_RBS-mutD- sfGFP were measured in the condition of arabinose induction by 1g/L and strains carrying pBAD_B0030-mutD-sfGFP without arabinose induction and wild type JM109 were set as negative control. We followed the protocol provided and found that pBAD_SDA_RBS-mutD- sfGFP could increase the genome mutation rate up to 10 times compared with negative control. The results were shown as the figure below.
</P><H2><SPAN class="mw-headline" id="Week8.EF.BC.888.14-8.21.EF.BC.89"><B>Week8（8.14-8.21）</B></SPAN></H2><P>In this week, we tested the performance of our constructed <I>tetA</I> based selection pressure circuit. We were trying very hard to finely tune the culture condition to make our system have desired tryptophan dependent property. However, we only obtained very weak such property. First of all, we cultured <I>E.coli</I> JM109 carrying pTrc99A_trp sensor_RBS_tetA series of plasmid in M9 medium (100 mL Solution I(NaCl 0.5g/L,NH_4 Cl 1g/L ,Na_2 HPO_4∙〖12H〗_2 O 17.1g/L, KH_2 PO_4  3g/L),100 µL 1M MgSO_4,500µL 0.1M CaCl_2  ,4mL 20%(mass fraction) glucose). However, their growth in such medium was quite weak, which made us change to test this circuit in LB medium. The general protocol was firstly to culture seed bacterium in LB medium for overnight. The seed medium was added to fresh LB medium in the next morning to adjust the initial OD600=0.05. When OD600 of this new medium reached about 0.6, IPTG, tryptophan and tetracycline were added to adjust the final concentration to be 24mg/L, 2mM, and designed concentration, respectively. OD600 was measured and growth curve was obtained for further analysis. All the condition test experiment was conducted in 24-well plate with 1.5mL LB medium in each unit and further confirmation was conducted in a 100mL flask containing 20mL LB medium. In the first round, we tested some low concentration of tetracycline (0, 12, 18, 24ug/mL) which was added when induced by IPTG and tryptophan but found no significant difference in growth. Therefore, we increased the concentration of tetracycline and added it at the initiation of the growth. Some interesting results were obtained and described as the Figure below. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Figure. 11 bitter pressure part function: growth rate against tetracycline addition  (extracellular addition of tryptophan)</DIV></DIV></DIV></DIV><P>It was found that growth of microorganisms was significantly influenced by tetracycline, which was shown that as the increase of the concentration of tetracycline, the growth rate decreased. We also found that for pTrc99A_trp sensor_RBS-ori_tetA, its growth was severely inhibited by the induction of IPTG. The growth of bacterium in the condition with zero IPTG and tryptophan addition proved that the leakage expression of tetA existed so that our desired condition should be in a high tetracycline concentration. When we further observed the growth independence on tryptophan addition, we found our desired positive relationship between tryptophan and growth with tetracycline concentration to be around 84 and 120 ug/mL and in the host carrying plasmid pTrc99A_trp sensor_RBS-ori_tetA. The results were shown in Figure. 5. For our bitter pressure part with RBS B0030 and B0032, desired positive relation was very poor. For these system with in vitro tryptophan addition, we could clearly found that in the control group (tetracycline=0mg/L) tryptophan addition showed significant inhibit to cell growth, which significantly reduced the positive function to growth caused by tryptophan addition and following tetA expression. What was more, in vitro tryptophan addition might face kinetics problem. Amino acid was transported into the cell by positive transport mechanism and the concentration of amino acid was quite different in and out of the cell . Our test system might fail due to the poor and similar intracellular tryptophan concentration although quite different extracellular concentration. Although such unexpected things happened, we still obtained weak but clear desired growth property in one certain system with certain <I>tetA</I> expression level. To overcome the problems mentioned before and proved bitter part’s possibility in in vivo evolution, we designed following in vivo system for further characteristics.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:337px;"><DIV class="thumbcaption">Figure. 12 bitter pressure part function: growth rate against concentration of tryptophan (extracellular addtion)</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption">Figure. 13 sweet pressure part performance test in M9 medium with 4 and 6g/L maltose as single carbon source, respectively</DIV></DIV></DIV><P>We also selected our engineered sweet pressure part RBS library by the following protocol. 48 strains form our library were selected and cultured in m9 medium with 8g/L glucose for overnight. On the next day, seed medium was transferred to fresh M9 medium in 24 well plate with 8g/L glucose or 8g/L maltose as single carbon source, respectively. Initial OD600 was set as 0.16. The growth curve of each strain was recorded and four strains with biggest final OD600 ration between in glucose and maltose M9 medium were selected. The best one was named after sweet pressure part strain 3-4. 
After that, we measure the performance of our sweet pressure part strain 3-4 in a tryptophan addition fashion. The colony from LB agar plate was picked up and cultured in M9 medium with 8g/L glucose for overnight. On the next day, seed medium was transferred to fresh M9 medium in 24 well plate with 2, 4, 6 and 8g/L maltose as single carbon source, respectively. Initial OD600 was set as 0.16. when OD600 of the medium reached 0.6, 0, 0.5 and 1g/L L-arabinose, 0 and 1mM was added as induction. The growth curve was measured and shown in the figure below.
</P><P>It could be found that for conditions of 4 or 6g/L maltose in M9 medium, when inducer arabinose was added as 1g/L, strains induced by 1mM tryptophan obtained much better growth after 12h than the corresponding negative control without tryptophan addition. In other conditions, this trend was not so significant, which inferred that the condition for our sweet pressure part to work must be finely tuned. What is more, we have proved that in the in vitro tryptophan induction measurement for tetA pathway that tryptophan addition greatly inhibited growth of cells. Therefore, the trend of better growth induced by tryptophan addition was very strong proof to the successful construction of our sweet pressure part. It worked in this particular condition.
</P><H2><SPAN class="mw-headline" id="Week9.288.22-8.29.29"><B>Week9(8.22-8.29)</B></SPAN></H2><P>To overcome the inhibition and kinetics problem during the test for our pressure system, we turned to <I>in vivo</I> system in this week. Fortunately, our lab has constructed before some engineered <I>E.coli</I> BL21(DE3) with different tryptophan productivity in fermentation (unpublished data). They were achieved by the combination deletion of several important regulation genes or branch pathway, also the overexpression of engineered tryptophan pathway (also induced by IPTG). After 48h fermentation in 24 well plate with each fermentation volumn as 1.5mL, three strains with the overall productivity as 0, 0.07, 0.15g/L were named after trp000, trp007, trp015, respectively. Thus, we transformed our bitter pressure part 30, 32 and ori into them by chemical transformation, respectively. Hence, engineered strains trp000-30, trp000-32, trp000-ori, trp007-30, trp007-32, trp007-ori, trp015-30, trp015-32, trp015-ori were constructed.
</P><P>First of all, their tryptophan productivity was measured. For we wanted to know their productivity in our evolution conditions, we didn’t applied the previous fermentation condition (30℃ after IPTG induction for 48h). Oppositely, we applied the conditions that would be applied in our evolution protocol as 37℃ after IPTG induction for 24h. The IPTG application concentration was set as 24mg/L. Results was shown as below. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:422px;"><DIV class="thumbcaption">Figure.14 bitter pressure part function: growth rate against concentration of tryptophan (intracellular tryptophan production)</DIV></DIV></DIV><P>It was clear that transformation of our engineered bitter pressure parts didn’t significantly influence original strains’ tryptophan relative productivity. Hence, we further tested these three parts’ performance by measuring the growth rate of these nine strains in different tetracycline concentration. For protocol, please refer to our note part. The final results were shown as growth dynamics below.
</P><P>As shown here, strains carrying our bitter pressure part with RBS B0030 and B0032 showed good tryptophan dependent growth property within the first 15h after culture. Further, as the increase of tetracycline, the selection pressure increased and the growth rate of strains decreased. However, with increased selection pressure, especially for B0032, it could be observed that with time window of about 15h, strains with higher tryptophan overproduction showed much higher growth rate with the growth of tryptophan non producer trp000 nearly inhibited by high concentration of tetracycline. However, it was confused to find that strains trp000 carrying bitter pressure part vector tended to grow beyond this time window. That was hypothesized to be due to the inactivity of tetracycline. To test this hypothesis, we conducted an experiment that. Another obvious phenomenon was such tryptophan dependent growth difference was more significant when IPTG induction was conducted. Thus, according to the result of tryptophan dependent growth, we designed the evolution protocol for tryptophan overproduction strains.
</P><H2><SPAN class="mw-headline" id="Week10.288.30-9.5.29"><B>Week10(8.30-9.5)</B></SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:422px;"><DIV class="thumbcaption">Figure 15. Tryptophon production of rbs30 mutation stains</DIV></DIV></DIV><P>In this week, we began to evolve our E.coli Bl21. Mutation part and Bitter defender part (rbs30) were transferred into E.coli Bl21. Colonies were picked up from agar plate to culture in LB. Seed culture was added to 1.5mL M9YE media(initial OD was 0.02) in 24-well plate with 50mg/L ampicillin, 24mg/L chloromycetin, 24mg/L IPTG and 80mg/L tetracycline as the first round mutation. After 270 minutes, the first round products were as seed culture added to another new 1.5mL M9YE media like above, but with higher tetracycline concentration as the second round mutation. After the third round mutation, the culture were coated on LB agar plate, picked up 24 single colonies to culture in 24-well plate for 48h and detected tryptophon production by HPLC. Result was shown in figure 15. With the help of mutation part, we got mutation mixture with different tryptophon yield range from 0-1.508mg/L.
</P><H2><SPAN class="mw-headline" id="Week11.289.6-9.12.29"><B>Week11(9.6-9.12)</B></SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:422px;"><DIV class="thumbcaption">Figure 16. Tryptophon production of rbs30 mutation stains</DIV></DIV></DIV><P>In this week, we continue to evolve the original Bl21 with mutation part and bitter defender part(rbs30) in order to test the universality of our mutation strategy. Colonies were picked up from agar plate to culture in LB. Seed culture was added to 1.5mL M9YE media(initial OD was 0.02) in 24-well plate with 50mg/L ampicillin, 24mg/L chloromycetin, 24mg/L IPTG and 80mg/L tetracycline as the first round mutation. After 610 minutes, the first round products were as seed culture added to another new 1.5mL M9YE media like above, but with higher tetracycline concentration as the second round mutation. After the third round mutation, the culture were coated on LB agar plate, picked up 24 single colonies to culture in 24-well plate for 48h and detected trptophon production by HPLC. Result was shown in figure 16. It also shown diversity of our mutation strategy. However, the mixture exhibited lower productivity. The mixture was a survival from high tetracycline concentration for its resistance tolerance, another evolution type of our mutation.
</P><H2><SPAN class="mw-headline" id="Week12.289.13-9.19.29"><B>Week12(9.13-9.19)</B></SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:422px;"><DIV class="thumbcaption">Figure 17. Tryptophon production of rbs30 mutation stains</DIV></DIV></DIV><P>In this week, we started to evolve Bl21 with mutation part and another bitter defender part(rbs32). Mutation part and Bitter defender part (rbs30) were transferred into E.coli Bl21. Colonies were picked up from agar plate to culture in LB. Seed culture was added to 1.5mL M9YE media(initial OD was 0.02) in 24-well plate with 50mg/L ampicillin, 24mg/L chloromycetin, 24mg/L IPTG and 80mg/L tetracycline as the first round mutation. After 270 minutes, the first round products were as seed culture added to another new 1.5mL M9YE media like above, but with higher tetracycline concentration as the second round mutation. After the third round mutation, the culture were coated on LB agar plate, picked up 24 single colonies to culture in 24-well plate for 48h and detected tryptophon production by HPLC. Result was shown in figure 17. With the help of mutation part, we also got mutation mixture with different tryptophon yield range from 0-2.028mg/L. The yield was obviously improved.
</P><H2><SPAN class="mw-headline" id="Week13.289.20-9.27.29"><B>Week13(9.20-9.27)</B></SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:422px;"><DIV class="thumbcaption">Figure 18. Tryptophon production of control and mutation stains</DIV></DIV></DIV><P>We got high productivity stains during three rounds mutation, and we picked up the highest yield of rbs30 and rbs32 to culture in LB. Seed culture was added to 1.5mL M9YE media(initial OD was 0.02) in 24-well plate with 50mg/L ampicillin and cultured for 48h to detect tryptophon concentration. The result was shown in figure 18. The control Bl21 almost did not contain trptophon and the other two controls rbs30 and rbs32 with shown low productivity. Obviously, the two mutation strains had higher productivity. The yield of rbs30-16 was 2.4 times higher than control rbs30 and rbs32-3 was 2.7 times higher than control rbs32. That proved our mutation strategy successfully accelerates evolution to produce more tryptophon.
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Tsinghua-E/Notebook">http://2013.igem.org/Team:Tsinghua-E/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tsinghua-E/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tsinghua-E/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tsinghua-E/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tsinghua-E/Notebook&amp;oldid=214361" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>