<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_KU_Leuven_Research_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:KU Leuven/Research/Results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="main-navm"><DIV class="nav-itemm"><DIV id="main"><A href="https://2015.igem.org/Team:KU_Leuven"><SPAN>main</SPAN></A></DIV></DIV><DIV class="nav-itemm"><DIV id="research"><A href="https://2015.igem.org/Team:KU_Leuven/Research"><SPAN>Research</SPAN></A></DIV></DIV><DIV class="nav-itemm"><DIV id="interlab"><A href="https://2015.igem.org/Team:KU_Leuven/InterLabStudy"><SPAN>Interlab</SPAN></A></DIV></DIV><DIV class="nav-itemm"><DIV id="modeling"><A href="https://2015.igem.org/Team:KU_Leuven/Modeling"><SPAN>Modeling</SPAN></A></DIV></DIV><DIV class="nav-itemm"><DIV id="outreach"><A href="https://2015.igem.org/Team:KU_Leuven/Practices"><SPAN>Outreach</SPAN></A></DIV></DIV><DIV class="nav-itemm"><DIV id="project"><A href="https://2015.igem.org/Team:KU_Leuven/Future"><SPAN>Future</SPAN></A></DIV></DIV><DIV class="nav-itemm"><DIV id="team"><A href="https://2015.igem.org/Team:KU_Leuven/Team"><SPAN>Team</SPAN></A></DIV></DIV><DIV class="nav-itemm"><DIV id="notebook"><A href="https://2015.igem.org/Team:KU_Leuven/Notebook"><SPAN>Notebook</SPAN></A></DIV></DIV></DIV><DIV class="main-nav"><DIV id="centernav"><DIV class="nav-item"><DIV id="main"><A href="https://2015.igem.org/Team:KU_Leuven"><SPAN>main</SPAN></A></DIV></DIV><DIV class="nav-item"><DIV id="research"><A href="https://2015.igem.org/Team:KU_Leuven/Research"><SPAN>Research</SPAN></A></DIV></DIV><DIV class="nav-item"><DIV id="interlab"><A href="https://2015.igem.org/Team:KU_Leuven/InterLabStudy"><SPAN>Interlab</SPAN></A></DIV></DIV><DIV class="nav-item"><DIV id="modeling"><A href="https://2015.igem.org/Team:KU_Leuven/Modeling"><SPAN>Modeling</SPAN></A></DIV></DIV><DIV class="nav-item"><DIV id="outreach"><A href="https://2015.igem.org/Team:KU_Leuven/Practices"><SPAN>Outreach</SPAN></A></DIV></DIV><DIV class="nav-item"><DIV id="project"><A href="https://2015.igem.org/Team:KU_Leuven/Future"><SPAN>Future</SPAN></A></DIV></DIV><DIV class="nav-item"><DIV id="team"><A href="https://2015.igem.org/Team:KU_Leuven/Team"><SPAN>Team</SPAN></A></DIV></DIV><DIV class="nav-item"><DIV id="notebook"><A href="https://2015.igem.org/Team:KU_Leuven/Notebook"><SPAN>Notebook</SPAN></A></DIV></DIV></DIV></DIV><DIV class="summaryheader"><DIV class="summaryimg"><DIV class="head"><H2>Results</H2></DIV></DIV></DIV><DIV class="summarytext1"><DIV class="center"><DIV class="togglebar"><DIV class="toggleone"><H2>Double Knockouts</H2></DIV><DIV id="toggleone"><P>
The Centre of Microbial and Plant Genetics (KU Leuven) provided us with three <I>E. coli</I> K-12 strains with each one representing the knock-out for the genes <I>tar</I>, <I>tsr</I> or <I>cheZ</I>. The kanamycin cassette of the <I>tar</I> knock-out strain was removed by the enzyme flippase on pCP20. This excision was checked by PCR. The original knock-out strain of <I>tar</I> was used as a positive control giving a band at 1232 bp on gel. If the cassette is removed, a band at 438 bp is visible. Ten colonies were tested and all have lost their cassette (Figure 1).</P><DIV class="center"><DIV id="image1"><H4><DIV id="figure1">Figure 1</DIV> Excision of the kanamycin cassette of the knock-out of tar. 
As a positive control, the original knock-out of tar was used which should show a band at 1232 basepairs. If the kanamycin cassette is removed, a band at 438 bp will be visible. Besides, a 1 kb Plus DNA ladder of GeneRuler was used.</H4></DIV></DIV><P>The PCP20 plasmid contains a temperature sensitive origin of replication. To remove this plasmid, the colonies were grown overnight at 42°C. PCP20 is resistant to ampicillin - this characteristic is useful to verify the removal of the plasmid. Single colonies were streaked on one LB plate with and one without ampicillin. Figure 2 proves that the PCP20 plasmid is removed in all mutant cells.</P><DIV class="center"><DIV id="image2"><H4><DIV id="figure2">Figure 2</DIV>Test to prove that the ampicillin resistant PCP20 plasmid is removed. The left plate contains ampicillin, the right plate contains no antibiotic.</H4></DIV></DIV><P>We received the lysate from Oscar Torres. The donor strains (ΔcheZ and Δtsr) were infected with this lysate. In figure 3, the plaques, as a result of the infection, are visible. Some of the plaques will contain DNA of ΔcheZ and Δtsr due the sloppy packaging mechanism of the phage P1.</P><DIV class="center"><DIV id="image3"><H4><DIV id="figure3">Figure 3</DIV>Plaques after infection of our donor strains (Δ<I>cheZ</I> and <I>Δtsr</I>).</H4></DIV></DIV><P>The lysate was plated out on LB plates as control. No colonies are visible in figure 4, this means that the lysate is not contaminated by cells.</P><DIV class="center"><DIV id="image4"><H4><DIV id="figure4">Figure 4</DIV>Control to check if the lysate of ΔcheZ and Δtsr is not contaminated with cells.</H4></DIV></DIV><P>The plaques of the acceptor strains were extracted and different amounts of lysate were used to infect our donor strain (Δtar). The resulted cells were plated out on kanamycin plates to select the right colonies (Figure 4).</P><DIV class="center"><DIV id="image5"><H4><DIV id="figure5">Figure 5</DIV>
Infection of the acceptor strain (Δtar) by lysate originating from donor strains (ΔcheZ and Δtsr)</H4></DIV></DIV><P>The Tar knock-out cells without the kanamycin cassette were also plated out on kanamycin as a control. In figure 5 is visible that there is no growth.
</P><DIV class="center"><DIV id="image6"><H4><DIV id="figure6">Figure 6</DIV>
Tar knock-out cells without the kanamycin cassette were plated out on kanamycin plates as control.</H4></DIV></DIV><P>Different colonies were screened to confirm the knock-out in cheZ. If the cassette is not there, a band should show at 863 bp. If the cassette is still there, there should be a band at 581 bp. As a positive control we used a knockout Tar strain which does not contain the kanamycin cassette anymore.</P><DIV class="center"><DIV id="image7"><H4><DIV id="figure7">Figure 7</DIV>
PCR to check that cheZ is knocked out in Δtar. The positive control is a knock-out in tar which lost the plasmid. </H4></DIV></DIV><P>To confirm that the knock-out in tsr was successful, a PCR was performed. A knock-out in tsr should give a band at 1304 bp while the original gene should give a band at 1964 bp. As positive control, a knock-out in tar which lost the kanamycin cassette was used. </P><DIV class="center"><DIV id="image8"><H4><DIV id="figure8">Figure 8</DIV>
PCR to check that tsr is knocked out in Δtar. The positive control is a knock-out in tar which lost the plasmid. </H4></DIV></DIV><P>Especially in the new constructed ΔtarΔcheZ, the chance exists that the knock-out in tar is undone due the sloppy packaging mechanism of the phage P1. Therefore, the knock-out in tar is checked again by PCR (see figure 9). The first positive control (c1) is a tar knock-out who has lost the kanamycin cassette. As a second control (c2), a cheZ knock-out strain was used. The gel of figure 8 shows that all our ΔtarΔtsr strains are ok, while only two of the ΔtarΔcheZ are still ok. </P><DIV class="center"><DIV id="image9"><H4><DIV id="figure9">Figure 9</DIV>
PCR to check the knock-out in tar after the P1 transduction</H4></DIV></DIV><P>Finally, the all the genes of the operon containing cheZ were checked by PCR (see figure 10). As positive control the tar knock-out which does not contain the plasmid anymore was used, but each time with the  primers corresponding to the checked gene. The negative control contains the mastermix but without template.</P><DIV class="center"><DIV id="image10"><H4><DIV id="figure10">Figure 10</DIV>
PCR to check the operon of ΔtarΔcheZ</H4></DIV></DIV></DIV></DIV><DIV class="togglebar"><DIV class="toggletwo"><H2>Motility Test</H2></DIV><DIV id="toggletwo"><P>
The cheZ knock-outs are not able to swim anymore. Therefor we performed a phenotypical motility test. The result is visible in figure 11. </P><DIV class="center"><DIV id="image11"><H4><DIV id="figure11">Figure 11</DIV>
Motility test of our cheZ knock-out</H4></DIV></DIV></DIV></DIV><DIV class="togglebar"><DIV class="togglethree"><H2>Gibson Assembly Method</H2></DIV><DIV id="togglethree"><P><B>Method 1</B>
The first Gibson assembly was performed using only gBlocks without a vector. Since the T5 exonuclease cleaves nucleotides from 5’ to 3’, an extra fill-in step was added after the 1 hour incubation at 50°C. Appropriate primers were added together with Phusion DNA polymerase and T4 DNA ligase to recreate the blunt insert fragment. The ligated fragments were checked by PCR and both reactions showed a band at the expected height (Figure 12). The 1-2-3 reaction however showed a lot of aspecific bands. The 5-6 fragment was stored untill further use and the 1-2-3 fragment was then digested to be directionally cloned in a pSB1C3 backbone and transformed in electrocompetent <I>E. cloni</I> cells.  
</P><DIV class="center"><DIV id="image12"><H4><DIV id="figure12">Figure 12</DIV>PCR check of assembled gBlocks.</H4></DIV></DIV><P>Positive colonies of 1-2-3 were further checked by colony PCR. Of this check, only one colony seemed to have the right insert (Figure 13). Restriction digestion was however negative (Figure not shown). </P><DIV class="center"><DIV id="image13"><H4><DIV id="figure13">Figure 13</DIV>Colony PCR of 1-2-3 assembled pSB1C3 plasmids</H4></DIV></DIV><P><B>Method 2</B>
Because our method was aberrant compared to the IDT protocol, a vector with correct overhangs was PCR amplified. This vector was used in a one-reaction Gibson assembly. After transformation, liquid cultures were grown from which the plasmid DNA was isolated. Restriction mapping was performed (Figure 14) and showed presence of only pUC19. </P><DIV class="center"><DIV id="image14"><H4><DIV id="figure14">Figure 14</DIV>Restriction digestion map using NotI and BglII for 1-2-3 and NcoI for 5-6</H4></DIV></DIV><P><B>Method 3</B>
To get rid of the template pUC, the PCR amplified backbone was digested with DpnI. This restriction enzyme only cuts methylated DNA and is thus inactive on PCR amplified pUC and synthesised gBlocks.
Even after digestion with DpnI, pUC19 kept appearing (Figure 15). The correct plasmid was gel purified and again transformed in <I>E. cloni</I>. After miniprep, gel analysis was negative for the assembled plasmid and positive for pUC19 only. </P><DIV class="center"><DIV id="image15"><H4><DIV id="figure15">Figure 15</DIV>Miniprepped positive pSB1C3-1-2-3 (lane a-b), pUC-1-2-3 (lane c-k) and pUC-5-6 (lane l-r) clones</H4></DIV></DIV><P><B>Method 4</B>
To optimize the vector PCR, pUC19 was first linearized using the unique restriction enzyme XbaI. Again, the PCR product was digested with DpnI. This plasmid was also transformed as a negative control and did not show any positive colonies. The plates that should contain the insert did show positive colonies. Gel analysis of the positive colonies again showed only a positive band for pUC19. Even though it appeared that the assembly worked (shown by PCR and gel separation) and that the template was completely degraded, the pUC19 vector reappears almost after every purification. In the future, plasmid insert load should be reduced to make the plasmid more stable over time. </P></DIV></DIV><DIV class="togglebar"><DIV class="togglefour"><H2>OHHL detection</H2></DIV><DIV id="togglefour"><P>
In the first step, the <I>Chromobacterium violacein</I> CV026 was grown together with different concentrations of OHHL. The <I>C. violacein</I> CV026 was added to the mixtures at an OD of 1.11. The cells were grown for 24 hours in air-lid culture tubes at 30 °C in a shaking incubator (200 rpm). In Figure 16, it is clearly visible that a violet pigment is produced. </P><DIV class="center"><DIV id="imageAHL1"><H4><DIV id="figure16">Figure 16</DIV>
Culture tubes of inoculated <I>Chromobacterium violacein</I> CV026 with different amounts of OHHL. click to enlarge</H4></DIV></DIV><P> First the OD of our cultures was measured in a cuvette (1 cm). Then the violacein was isolated from the cells by centrifugation, resuspension in DMSO and a second centrifugation step (Figure 17).
</P><DIV class="center"><DIV id="imageAHL2"><H4><DIV id="figure17">Figure 17</DIV>
Violacein was removed from the samples by centrifugation and resuspension in DMSO. Click to enlarge</H4></DIV></DIV><P>After isolating violacein from our samples, 200 µL was pipetted into a 96-well falcon microtiter plate and the absorbance was measured at 585 nm. In total, three technical replicates were measured to estimate the pipetting and measuring error (Figure 18).
</P><DIV class="center"><DIV id="imageAHL3"><H4><DIV id="figure18">Figure 18</DIV>
96-well falcon microtiter plate containing the three technical replicates of the dilution series. Click to enlarge</H4></DIV></DIV><P>First a broad concentration range was used to estimate the linear part. This range was made by a two-fold dilution series. When measuring the absorbance, LB medium was used as blank. Later the absorbance values of the blank were subtracted from the absorbance values of the standards. Then these values were divided by the absorbance values at 600 nm measured in the microtiterplate which gives an indication of the cell number.  Eventually the values were corrected by setting the point with a concentration of 0 mM OHHL in the origin. These values were plotted in figure 19. The concentrations  2.56 mM and 5.12 mM were left out because these values were not distinguishable from the blank. This can be explained because the OHHL is dissolved in DMSO which lowers the growth of <I>C. violaceum</I> CV026. Between the concentrations 0.64 mM and 1.28 mM, the curve is stagnating. This is probably due to saturation of the medium or the inhibitory effect of DMSO.  In a next step, a more narrow range was investigated.
</P><DIV class="center"><DIV id="imageAHL4"><H4><DIV id="figure19">Figure 19</DIV>
First estimation of the OHHL standard curve. click to enlarge</H4></DIV></DIV><P>In the second experiment, a range between 0.01 and 0.10 mM was investigated. The difference with the previous experiment is that the OD was first measured in 1 cm cuvettes and that the violacein was afterwards isolated. In this way, the plate reader does not contain cells. This optimisation is done, because we noticed in the previous experiment that violacein was produced more quickly in a culture tube than in a microtiterplate, probably due to the amount of supplied oxygen. Another reason for this working method is because estimating the amount of cells is more standardised by using cuvettes of 1 cm than using a microtiterplate.
</P><P> Table 1 contains the processed values of the absorbance measurements. The processing is similar to the previous experiment. First the absorbance of DMSO was subtracted from the absorbance of the standards. Thereafter, these values were divided by the OD measured in 1 cm cuvettes. And finally, these values were corrected by putting the standard with a concentration of 0 mM OHHL in the origin.
</P><H4>Table 1: Absorbance at 585 nm divided by the OD </H4><DIV class="center"><DIV class="datatable"><TABLE><TBODY><TR><TH colspan="12" class="emerged">Concentration (mM) </TH></TR><TR><TH>0.10</TH><TH>0.09</TH><TH>0.08</TH><TH>0.07</TH><TH>0.06</TH><TH>0.05</TH><TH>0.04</TH><TH>0.03</TH><TH>0.02</TH><TH>0.01</TH><TH>0.00</TH></TR><TR class="lightrow"><TH>T1  (corrected)</TH><TD>0.126</TD><TD>0.072</TD><TD>0.110</TD><TD>0.093</TD><TD>0.057</TD><TD>0.083</TD><TD>0.052</TD><TD>0.049</TD><TD>0.076</TD><TD>0.011</TD><TD>0.000</TD></TR><TR><TH>T2  (corrected)</TH><TD>0.130</TD><TD>0.073</TD><TD>0.110</TD><TD>0.076</TD><TD>0.057</TD><TD>0.085</TD><TD>0.048</TD><TD>0.043</TD><TD>0.076</TD><TD>0.008</TD><TD>0.000</TD></TR><TR class="lightrow"><TH>T3  (corrected)</TH><TD>0.121</TD><TD>0.071</TD><TD>0.078</TD><TD>0.091</TD><TD>0.054</TD><TD>0.083</TD><TD>0.053</TD><TD>0.049</TD><TD>0.051</TD><TD>0.008</TD><TD>0.000</TD></TR><TR><TH>Average</TH><TD>0.125</TD><TD>0.072</TD><TD>0.099</TD><TD>0.087</TD><TD>0.056</TD><TD>0.084</TD><TD>0.051</TD><TD>0.047</TD><TD>0.068</TD><TD>0.009</TD><TD>0.000</TD></TR><TR class="lightrow"><TH>Standard deviation</TH><TD>0.004</TD><TD>0.001</TD><TD>0.019</TD><TD>0.009</TD><TD>0.002</TD><TD>0.001</TD><TD>0.003</TD><TD>0.003</TD><TD>0.015</TD><TD>0.002</TD><TD>0.000</TD></TR></TBODY></TABLE></DIV></DIV><P>In figure 20, our standard curve is plotted. A linear correlation between the absorbance and the concentration OHHL can be found. The variance of the technical replicates, visualised by the error bars, and the variance of the regression curve, shown by the R<SUP>2</SUP> value, can be explained by pipetting and measuring errors. Also, working with biological cells generates a background noise.  This standard curve could give an estimation of bacterial AHL production. But it is important to keep in mind that there is background noise. Please note that we only had two attemps to perform this experiment, the first time the broad range was investigated, the second time the more narrow range was investigated. Optimisation of this curve can be done by making more biological and technical replicas.
</P><DIV class="center"><DIV id="imageAHL5"><H4><DIV id="figure20">Figure 20</DIV>
Standard curve ranging from 0 to 0.1 mM. The error bars represent the standard deviation between the technical replicates.</H4></DIV></DIV></DIV></DIV><DIV class="togglebar"><DIV class="togglefive"><H2>Leucine detection</H2></DIV><DIV id="togglefive"><P>In comparison to HPLC, the chosen method would be less time consuming without the need of specialized equipment. Due to a lack of time, we were not able to complete the plasmid assembly and therefore we could not quantify the amount of leucine produced by the designed bacteria. But we did an attempt to test the quantification method by making the standard curve. </P><P>
The standard curve from 0 to 100 µM did not give clear signals, so the working method needs optimisation. Reasons for this result could be the use of different enzyms than mentioned in the article. Because the enzymes originate from other organisms than mentioned in Kugimiya and Fukada (2015), it is possible that the enzymes have another efficiency and as a consequence need another ratio substrate over enzyme. Additionally, we did not have the same equipment as described in the article. We had to manually pipet the luminol solution while in equipment described in the acticle this happens autimatically. Probably there was too much time between adding the luminol solution and measuring.
</P><P>Please note that we were only able to do one attempt on this experiment.
</P></DIV></DIV><DIV class="togglebar"><DIV class="togglesix"><H2>BioBricks construction</H2></DIV><DIV id="togglesix"><P>
To make characterization easier, DNA for fusion proteins was designed and ordered in gBlock format. A His-tag was fused to LuxI and GFP to CheZ. These two new constructs are the basic BioBricks submitted.  Using PCR, the iGEM prefix and suffix were added to this basic parts as well as to the parts containing a RBS. Cutting the PCR fragments with EcoRI and PstI was not favorable due to the non-existing extra nucleotides necessary for the enzymes to cut. Therefore, EagI, a restriction enzyme also cutting in the NotI site was used to clone the fragment in an empty pSB1C3 vector.
BBa_J23101 was transformed in <I>E. cloni</I> to multiply the amount of vector DNA.
After miniprepping, the BioBrick was cut with EagI together with a phosphatase to overcome self-ligation.
T4 DNA ligase was used and a 1:2 vector-insert ratio was added. Since digestion by EagI does not allow directional cloning, multiple colonies were tested by colony PCR to check insert directionality (Figure 21).
</P><DIV class="center"><DIV id="image21"><H4><DIV id="figure21">Figure 21</DIV>Gel to check directionality. A band around 1400bp using the insert-reverse primer pair (up) show the gene in correct orientation (forward),  a band around 1600bp using the insert-forward primer (down) show the gene in reversed orientation. </H4></DIV></DIV><P>The correct colonies were selected, miniprepped and sent for sequencing. 
To characterize the CheZ-GFP BioBrick, the fragment containing a RBS was cloned directly after a strong promotor (BBa_J23101). Figure 22 shows the gel right before ligation.
The colonies were checked by restriction mapping using BcuI and PstI (results not shown). The DNA sequence was also confirmed by DNA sequencing. Results can be provided by email. The presence of colonies expressing GFP proves that the plasmid was designed and cloned correctly (Figure 23). </P><DIV class="center"><DIV id="image22"><H4><DIV id="figure22">Figure 22</DIV>Gel after purification. Lanes 2-5: insert (1400bp). Lane 6: linearized vector. Lanes 7-10 : insert. Lane 11: linearized vector</H4></DIV></DIV><DIV class="center"><DIV id="image23"><H4><DIV id="figure23">Figure 23</DIV>GFP is expressed in the cells. This confirms the correct construction of the BioBrick</H4></DIV></DIV><P>Further characterization could be done by transforming the <I>cheZ</I> knockout Keio strain with this plasmid. These cells should then regain their possibility to swim.</P></DIV></DIV></DIV></DIV><DIV class="subsections"><DIV class="subsectionwrapper"><DIV class="subtextrow"><DIV class="subtext"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Idea"><H2>Idea</H2><P> A detailed description about the interaction between our two cells and the genetic circuit can be found here.
</P></A></DIV><DIV class="subtext"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Methods"><H2>Methods</H2><P> Here you can find the performed steps to create two different cell types and detailed quantification methods to determine interesting parameters.
</P></A></DIV></DIV></DIV><DIV class="subsectionwrapper"><DIV class="subtextrow"><DIV class="subtext"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Basic_Part"><H2>Basic Parts</H2><P>Our new, self-designed basic parts necessary to control cell-cell interactions and <I>E. Coli</I> motility.
</P></A></DIV><DIV class="subtext"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Composite_Part"><H2>Composite Parts</H2><P> Our basic parts were combined with each other and with existing iGEM promotors, RBS and terminators.
</P></A></DIV><DIV class="subtext"><A href="https://2015.igem.org/Team:KU_Leuven/Research"><H2>Back</H2><P> Go back to the Research page.
</P></A></DIV></DIV><DIV class="subimgrowm"><DIV class="subimgm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Idea"><B>Idea</B></A></DIV><DIV class="subtextm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Idea"><P>
A detailed description about the interaction between our two cells and the genetic circuit can be found here.

        </P></A></DIV></DIV><DIV class="subimgrowm"><DIV class="subimgm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Methods"><B>Methods</B></A></DIV><DIV class="subtextm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Methods"><P>
Here you can find the performed steps to create two different cell types and detailed quantification methods to determine interesting parameters.
        </P></A></DIV></DIV><DIV class="subimgrowm"><DIV class="subimgm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Basic_Part"><B>Basic Parts</B></A></DIV><DIV class="subtextm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Basic_Part"><P>
          Our new, self-designed basic parts necessary to control cell-cell interactions and <I>E. Coli</I> motility.
        </P></A></DIV></DIV><DIV class="subimgrowm"><DIV class="subimgm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Composite_Part"><B>Composite Parts</B></A></DIV><DIV class="subtextm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Composite_Part"><P>
          Our basic parts were combined with each other and with existing iGEM promotors, RBS and terminators.
        </P></A></DIV></DIV><DIV class="subimgrowm"><DIV class="subimgm"><A href="https://2015.igem.org/Team:KU_Leuven/Research"><B>Back</B></A></DIV><DIV class="subtextm"><A href="https://2015.igem.org/Team:KU_Leuven/Research"><P>
         Go back to the Research page.
        </P></A></DIV></DIV></DIV></DIV><DIV id="summarywrapper"><DIV class="summary"><H3> Contact </H3><P style="font-size:1.3em; text-align: center">
Address: Celestijnenlaan 200G room 00.08 - 3001 Heverlee
Telephone: +32(0)16 32 73 19
Email: igem@chem.kuleuven.be</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>