<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_KU_Leuven_Research_Methods skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:KU Leuven/Research/Methods</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="main-navm"><DIV class="nav-itemm"><DIV id="main"><A href="https://2015.igem.org/Team:KU_Leuven"><SPAN>main</SPAN></A></DIV></DIV><DIV class="nav-itemm"><DIV id="research"><A href="https://2015.igem.org/Team:KU_Leuven/Research"><SPAN>Research</SPAN></A></DIV></DIV><DIV class="nav-itemm"><DIV id="interlab"><A href="https://2015.igem.org/Team:KU_Leuven/InterLabStudy"><SPAN>Interlab</SPAN></A></DIV></DIV><DIV class="nav-itemm"><DIV id="modeling"><A href="https://2015.igem.org/Team:KU_Leuven/Modeling"><SPAN>Modeling</SPAN></A></DIV></DIV><DIV class="nav-itemm"><DIV id="outreach"><A href="https://2015.igem.org/Team:KU_Leuven/Practices"><SPAN>Outreach</SPAN></A></DIV></DIV><DIV class="nav-itemm"><DIV id="project"><A href="https://2015.igem.org/Team:KU_Leuven/Future"><SPAN>Future</SPAN></A></DIV></DIV><DIV class="nav-itemm"><DIV id="team"><A href="https://2015.igem.org/Team:KU_Leuven/Team"><SPAN>Team</SPAN></A></DIV></DIV><DIV class="nav-itemm"><DIV id="notebook"><A href="https://2015.igem.org/Team:KU_Leuven/Notebook"><SPAN>Notebook</SPAN></A></DIV></DIV></DIV><DIV class="main-nav"><DIV id="centernav"><DIV class="nav-item"><DIV id="main"><A href="https://2015.igem.org/Team:KU_Leuven"><SPAN>main</SPAN></A></DIV></DIV><DIV class="nav-item"><DIV id="research"><A href="https://2015.igem.org/Team:KU_Leuven/Research"><SPAN>Research</SPAN></A></DIV></DIV><DIV class="nav-item"><DIV id="interlab"><A href="https://2015.igem.org/Team:KU_Leuven/InterLabStudy"><SPAN>Interlab</SPAN></A></DIV></DIV><DIV class="nav-item"><DIV id="modeling"><A href="https://2015.igem.org/Team:KU_Leuven/Modeling"><SPAN>Modeling</SPAN></A></DIV></DIV><DIV class="nav-item"><DIV id="outreach"><A href="https://2015.igem.org/Team:KU_Leuven/Practices"><SPAN>Outreach</SPAN></A></DIV></DIV><DIV class="nav-item"><DIV id="project"><A href="https://2015.igem.org/Team:KU_Leuven/Future"><SPAN>Future</SPAN></A></DIV></DIV><DIV class="nav-item"><DIV id="team"><A href="https://2015.igem.org/Team:KU_Leuven/Team"><SPAN>Team</SPAN></A></DIV></DIV><DIV class="nav-item"><DIV id="notebook"><A href="https://2015.igem.org/Team:KU_Leuven/Notebook"><SPAN>Notebook</SPAN></A></DIV></DIV></DIV></DIV><DIV class="summaryheader"><DIV class="summaryimg"><DIV class="head"><H2> Methods </H2></DIV></DIV></DIV><DIV class="summarytext1"><DIV class="part"><P>On this page, you can find all the methods and protocols used in the lab to obtain our results. For some techniques we included some basic theory since it is a prerequisite to get acquainted with the details behind these techniques before using them. To learn more about them click the titles below.</P></DIV></DIV><DIV class="center"><DIV class="togglebar"><DIV class="toggleone"><H2>P1 transduction</H2></DIV><DIV id="toggleone"><P><B>Theory</B>
              To be able to create patterns, two different cell types called A and B will have to interact with each other. In order to achieve the desired behavior, the cells used in the experiments were derived from K12 <I>Escherichia coli</I> strains with introduction of specific knockouts. Cell type A has a deletion of <I>tar</I> and <I>tsr</I> genes, whereas in cell type B both <I>tar</I> and <I>cheZ</I> genes are knocked out. The Keio collection is composed of a set of precisely defined single-gene deletion mutants of all non-essential genes in <I>E. coli</I> K-12. The targeted genes were replaced by a kanamycin resistance cassette. The kanamycin cassette is enclosed between two FRT sites making excision possible using FLP recombinase (reference 1). FLP recombinase triggers an intramolecular recombination between FRT repeats in the chromosome. Since both the antibiotic resistance gene and the plasmid replication region are surrounded by two FRT sites, both are to be eliminated (Figure 1, step 1). 
             A genetic procedure for moving selectable mutations of interest, called the P1 transduction, was used. Since the packaging of the bacteriophage P1 is rather inaccurate, it will on occasion package the DNA of its bacterial host instead of its own phage chromosome. This implies that the lysate will contain either packaged phage or bacterial DNA. After infection of a second host with this lysate, a transfer of parts of the chromosome from the donor strain into the receiver strain will take place. Those DNA pieces can then recombine using the FRT sites and hereby be incorporated permanently into the chromosome of the new strain. Here, the recombination was triggered by selection on kanamycin. (reference 2) 
              In general, we used three steps to obtain the double knock-outs (Figure 1). In the first step, the kanamycin cassette of the <I>tar</I> knock-out strain was removed by flippase, coded on plasmid PCP20. Afterwards, the temperature sensitive plasmid was removed by growing the cells overnight at 42°C. In a third step, the <I>tar</I> knock-out strain was infected by lysate originating from the <I>tsr</I> and <I>cheZ</I> knock-out strains. After selection on kanamycin plates, we obtained the double knock-outs. These knock-outs were confirmed by PCR. For more information, please check our result page. </P><DIV class="center"><DIV id="image1"><H4><DIV id="figure1">Figure 1</DIV>
Scheme of P1 transduction</H4></DIV></DIV><P><B> Protocol </B></P><DL><DT> 1. Preparation of lysate starting from stock plate of phage</DT><DD>1. Make an overnight culture of <I>E. coli</I> MG1655. </DD><DD>2. Take 500 µL overnight culture and add the phage P1. Incubate overnight at 37°C. </DD><DD>3. Take single plaques of the P1 stock plate and bring this in a sterile Eppendorf tube together with 200 µL of mQ.</DD><DD>4. Overnight extraction while shaking at 37°C.</DD><DD>5. Add 0.01, 0.1, 10 and 100 µL of extraction to 500 µL of a stationary phase culture of <I>E. coli</I> MG1655. Vortex and plate out.</DD><DD>6. Add LB soft agar containing 10 mM MgSO<SUB>4</SUB> and 5 mM CaCl<SUB>2</SUB> and incubate at 37°C.</DD><DD>7. Choose the plate with the best lysates.</DD><DD>8. Sterilize your spoon using a Bunsen burner, cool it down with water and wash it with 100% ethanol.</DD><DD>9. Cut out a plaque from the soft agar and put this in a 10 mL syringe.</DD><DD>10. Press the content of the syringe in an Eppendorf tube and centrifuge for 10 minutes at 14000 rpm.</DD><DD>11. Take 650 µl and bring this in a new Eppendorf tube.</DD><DD>12. Extraction with 30 µL of CHCl<SUB>3</SUB>. </DD><DD>13. Vortex vigorously.</DD><DD>14. Store lysate at 4°C.</DD><DT> 2. Preparation of the lysate of donor strain</DT><DD>1. Firstly, centrifuge the lysate to ensure the chloroform is at the bottom of the Eppendorf tube. Then add 0.1, 1, 10 and 100 µl of lysate to 500 µL stationary phase overnight culture of the donor strain.</DD><DD>2. Add LB soft agar containing 10 mM MgSO<SUB>4</SUB> and 5 mM CaCl<SUB>2</SUB>. Incubate this at 37°C.</DD><DD>3. Sterilize your spoon in a Bunsen flame, cool it down with water and wash with 100% ethanol.</DD><DD>4. Centrifuge the Eppendorf tubes 10 minutes at 14000 rpm.</DD><DD>5. Transfer 650 µL into a new Eppendorf tube. </DD><DD>6. Extract with 30 µL of CHCl<SUB>3</SUB>. </DD><DD>7. Vortex vigorously. </DD><DD>8. Store the lysate at 4°C.</DD><DT> 3. Transduction to acceptor strain</DT><DD>1. Concentrate 500 µL of stationary phase overnight acceptor strain culture five times in LB with 10 mM MgSO<SUB>4</SUB> and 5 mM CaCl<SUB>2</SUB>. </DD><DD>2. Add 0.1, 1, 10 and 100 µL of donor strain lysate to 100 µL acceptor strain. </DD><DD>3. Incubate 30 minutes at 37°C.</DD><DD>4. Plate out on selective medium and incubate overnight. </DD><DD>5. Plate out lysate-only to check for contamination as well.</DD></DL></DIV></DIV><DIV class="togglebar"><DIV class="toggletwo"><H2>Gibson assembly</H2></DIV><DIV id="toggletwo"><P><B>Theory</B>
The Gibson assembly, as described by Gibson et al., is a rapid DNA assembly method that assures directional cloning of fragments in one single reaction. To perform the Gibson assembly, three essential enzymes are needed: a mesophylic nuclease, a thermophylic ligase and a high fidelity polymerase. Therefore, the NEBuilder<SUP>Ⓡ</SUP> HiFi DNA Assembly Master Mix (New England Biolabs) was used. In the first step of this reaction, the exonuclease rapidly cleaves off the 5’ DNA ends. The exonuclease is unstable at 50°C and gets degraded early in the process. In the second step, the designed sequence overlaps anneal and the polymerase starts filling in the gaps. The ligase then covalently joins both ends finalizing the plasmid assembly for transformation. This text was based on <A href="https://www.idtdna.com/pages/docs/default-source/user-guides-and-protocols/gibson-assembly.pdf?sfvrsn=16">the IDT description as seen on 13/09/2015</A>. In the following paragraph, our optimized protocol is given.</P><DIV class="center"><DIV id="image1"><H4><DIV id="figure1">Figure 1</DIV>
Gibson assembly reaction and its essential components <I>E. coli</I></H4></DIV></DIV><P><B>Protocol</B><DL><DT>Create pUC 19 plasmid with complementary sequence overhangs</DT><DD>1. Linearize plasmid using the unique restriction enzyme XbaI</DD><DD> 2. Create complementary sequence overhangs using a overhang-PCR </DD><DD> 3. Remove the uncut original pUC19 template by a digestion with DpnI</DD><DT> Gibson Assembly </DT><DD> 1. Mix linear DNA fragments (gBlocks) and the linearized vector in the appriopriate molar ratio ( between 1:1 and 1:2) together with the NEBuilder<SUP>Ⓡ</SUP> reaction mix</DD><DD> 2. Incubate for 1h at 50°C </DD></DL></P></DIV></DIV><DIV class="togglebar"><DIV class="togglethree"><H2>Motility Test Assay</H2></DIV><DIV id="togglethree"><P><B>Protocol</B></P><DL><DD>1. Prepare selective media (LB with 0.25% agar (2,5 g/L) in Petri dishes (85 mm dia).</DD><DD>2. Apply 1.5 µL of the diluted cell suspensions from mid-log-phase cultures (~2×105 cells/µL (OD=0.5)) to the center of the plates, and let dry in air for 15 min. </DD><DD>3. Incubate at 37 °C for 10 h. </DD></DL></DIV></DIV><DIV class="togglebar"><DIV class="togglefourhalf"><H2>OHHL quantification</H2></DIV><DIV id="togglefourhalf"><P><B>Theory</B>
N-acyl homoserine lactones (AHL) are small diffusible molecules used in cell-to-cell signaling by Gram-negative bacteria. <I>Chromobacterium violaceum</I> is a Gram-negative bacterium that produces the violet pigment violacein as a result of sensing AHL. AHL is produced by the autoinducer synthase CviI and released in the environment. When a quorum has been reached, AHL diffuses back into the bacteria and binds the transcriptional regulator CviR. This activates the expression of specific genes which leads to the production of violacein. </P><P>In our project, the mutant <I>C. violaceum</I> CV026 was used to quantify the amount of N-(3-Oxohexanoyl)-L-homoserine lactone (OHHL), a specific type of AHL, produced by the LuxI enzyme in our <I>E. coli</I> strains. The CV026 strain is <I>cviI</I> gene deficient and therefore requires exogenous addition of AHL to produce violacein. The idea is to plot a standard curve in which CV026 is induced with different concentrations of AHL. In the standard curve, results were normalized to the OD.</P><P>To quantify the amount of AHL produced by <I>E. coli</I>, these were grown and pelleted after which CV026 was added. After incubating for several hours, CV026 and violacein were spun down and the supernatant was removed. The pellet was resuspended in dimethyl sulfoxide and again centrifuged to separate the violacein dissolved in dimethyl sulfoxide from the cells. Absorbance of the supernatans was measured at 585 nm.</P><P><B>Protocol</B></P><DIV class="center"><DL><DT> 1. Make a standard curve</DT><P>The goal is to plot the absorbance of violacein against the concentration of OHHL around 0.04 mM </P><DL><DD>1. Make a OHHL stock solution of 10 mM.</DD><DD>2. Make a dilution series of OHHL in LB medium. Take the further dilution with CV026 into account. </DD><DD>3. Add <I>C. violaceum</I> CV026 in a 1:10 ratio to the end volume. </DD><DD>4. Incubate for 24 hours at 30°C in a shaking incubator (200 rpm).</DD><DD>5. Measure the OD (600 nm) value in 1 cm cuvettes.</DD><DD>6. Take 1 mL of the mixture and centrifuge for 10 minutes at 13 000 rpm.</DD><DD>7. Discard the supernatant and dissolve the pellet in 500 µl dimethyl sulfoxide. Vortex the solution vigorously for 30 seconds to completely solubilize violacein. </DD><DD>8. Add 200 µl of the violacein-containing supernatant to a 96-well falcon microplate.</DD><DD>9. Read the absorbance with a microplate reader at a wavelength of 585 nm.</DD></DL><DIV class="center"><DIV class="datatable"><TABLE><TBODY><TR><TH>End concentration (in mM)</TH><TH>Volume of 10 mM OHHL stock (in mL)</TH><TH>Volume LB medium  (in mL)</TH><TH>Volume of <I>C. violacein</I> CV026 (in mL)</TH></TR><TR class="lightrow"><TD>0.100</TD><TD>0.020</TD><TD>1.780</TD><TD>0.200 </TD></TR><TR><TD>0.090</TD><TD>0.018</TD><TD>1.782</TD><TD>0.200 </TD></TR><TR class="lightrow"><TD>0.080</TD><TD>0.016</TD><TD>1.784</TD><TD>0.200</TD></TR><TR><TD>0.070</TD><TD>0.014</TD><TD>1.786</TD><TD>0.200</TD></TR><TR class="lightrow"><TD>0.060</TD><TD>0.012</TD><TD>1.788</TD><TD>0.200</TD></TR><TR><TD>0.050</TD><TD>0.010</TD><TD>1.790</TD><TD>0.200</TD></TR><TR class="lightrow"><TD>0.040</TD><TD>0.008</TD><TD>1.792</TD><TD>0.200</TD></TR><TR><TD>0.030</TD><TD>0.006</TD><TD>1.794</TD><TD>0.200</TD></TR><TR class="lightrow"><TD>0.020</TD><TD>0.004</TD><TD>1.796</TD><TD>0.200</TD></TR><TR><TD>0.010</TD><TD>0.002</TD><TD>1.798</TD><TD>0.200</TD></TR><TR class="lightrow"><TD>0.000</TD><TD>0.000</TD><TD>1.800</TD><TD>0.200</TD></TR><TR><TD>Negative control</TD><TD>0.000</TD><TD>2.000</TD><TD>0.000</TD></TR><TR class="lightrow"><TD><I>C. violaceum</I>CV026</TD><TD>0.000</TD><TD>0.000</TD><TD>2.000</TD></TR></TBODY></TABLE></DIV></DIV><DL><DT> 2. Preparation of media </DT><DD>1. Incubate <I>Chromobacterium violaceum</I> CV026 for 16-18h.</DD><DD>2. Take 1 mL of the sample that you would like to quantify. Measure the OD(600 nm) and centrifuge (max speed 15000 rpm, 10 min). If you want to include the amount of AHL inside the cell, cells should be lysed in advance.</DD><DD>3. Inoculate and incubate the <I>C. violaceum</I> CV026 to OD(600 nm) = 0.1 in air-lid culture tubes containing LB and LB supplemented with 1 mL supernatants of your sample at 27°C (150 rev/min agitation) for 24 h in a shaking incubator. </DD></DL><DIV class="datatable"><TABLE><TBODY><TR><TH rowspan="2">Material </TH><TH colspan="2" class="emerged">Flask (ml) </TH></TR><TR><TH>Negative Control</TH><TH> Sample</TH></TR><TR class="lightrow"><TD>LB broth</TD><TD>18</TD><TD>17</TD></TR><TR><TD><I>Chromobacterium violaceum</I> CV026</TD><TD>1</TD><TD>1</TD></TR><TR class="lightrow"><TD> Cell sample</TD><TD>0</TD><TD>1</TD></TR><TR><TD>Dimethyl sulfoxide</TD><TD>1</TD><TD>1</TD></TR><TR class="lightrow"><TD>Total</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE></DIV><DT> 3. Quantification</DT><DL><DD>1. Centrifuge 1 mL culture (10 min at 13000 rev/min) to precipitate the insoluble violacein.</DD><DD>2. Discard supernatant (culture)  and add 1 ml of dimethyl sulfoxide to the pellet.</DD><DD>3. Vortex the solution vigorously for 30 s to completely solubilize violacein and centrifuge at 13000 rev/min for 10 min to remove the cells. </DD><DD>4. Add 200 µL of the violacein-containing supernatants to 96-well flat-bottomed microplates.
</DD><DD>5. Read the absorbance with a microplate reader at a wavelength of 585 nm.</DD></DL><P><B>References</B></P><DL><DD>1. iGEM ETH Zurich (2013). Modeling part overview <A href="https://2013.igem.org/Team:ETH_Zurich/Modeling/Overview">  [online] </A></DD><DD>2. Choo, J.H., Rukayadi, Y. and Hwang, J.K. (2005). Inhibition of bacterial quorum sensing by vanilla extract. Letters in Applied Microbiology. <A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1472-765X.2006.01928.x/epdf" target="_blank"> [online] </A></DD></DL><DIV class="togglebar"><DIV class="togglethreehalf"><H2>Leucine Detection</H2></DIV><DIV id="togglethreehalf"><P><B>Theory</B>
The chemiluminescence detection assay of Kugimiya and Fukada (2015) was used as a reference to quantify the amount of leucine produced by IlvE gene containing cells. In this technique, leucine-tRNA synthetase in combination with a luminol chemiluminescence reaction was used to detect leucine. In the article, a selective quantification from 1 to 20 µM leucine was mentioned and was further optimized in our protocol.
Below, the reaction equations can be seen. In the first step, leucine-tRNA synthetase (LeuRS) is activated by ATP to form leucyl-AMP with formation of a side product - pyrophosphate. The formation of pyrophosphate is further used to detect the amount of leucine. After the addition of inorganic pyrophosphatase, this enzyme hydrolyses pyrophosphate to phosphate. When pyruvate oxidase and pyruvate is added to phosphate, it results in the formation of acetyl phosphate and hydrogen peroxide. In the fourth reaction, the hydrogen peroxide in combination with luminol and horseradish peroxidase leads to the emition of light. This luminescence is detected with a luminometer.</P><DIV class="center"><DIV id="image1"><H4><DIV id="figure1">Figure 2</DIV>
                Reaction series for leucine detection - Kugimiya and Fukada (2015). Click to enlarge
            </H4></DIV></DIV><P>
The idea is to grow the bacteria on minimal medium without leucine. These bacteria are spun down and the supernatant is further investigated in the presence of leucine. The luminescence originating from the bacterial samples is compared with the standard curve. Firstly, a big range is used for the standard curve (0 to 100 μM) to then further zoom in to the biologically relevant concentrations.</P><P><B>Protocol</B><DL><DD>A mixture (40 µL) of leucine (0 - 100 µM), 1 mM ATP, 10 mM KCl, 5 mM MgCl<SUB>2</SUB> and human leucyl-tRNA synthetase (6.25 μg/mL) (Abcam) is made in 15 mM HEPES-NaOH (pH 8.0). This mixture is then heated untill 80°C for 45 min on a heating block (shaking at 300 rpm).</DD><DD>After cooling cown on ice, the second reaction mixture (10 µL) is added. It contains 2.5 mM sodium pyruvate, 5.0 mM MgCl<SUB>2</SUB>, 300 μM thiamine pyrophosphate, 0.08 μM FAD, 0.5 unit/mL inorganic pyrophosphatase (from yeast, Thermo Scientific) and 20 units/mL pyruvate oxidase (from <I>Aerococcus sp.</I>, Sigma) in 50 mM HEPES-NaOH (pH 6.8).</DD><DD>The next step is to spin the mixture down for 30 s at 8000 rpm. The samples are then heated at 40°C for 30 min on a heating block (600 rpm). After spinning down at 8000 rpm for 30 seconds, the solution is added to a white 96-well plate.</DD><DD>A solution (100 µL) containing 60 μM luminol and 5.0 unit/mL horseradish peroxidase (Feinbiochemica Heidelberg) in 800 mM carbonate (NaHCO<SUB>3</SUB>-NaOH) buffer (pH 9.0) is added. Finally, the microplate reader (BioTek SynergyMx) measured the luminescence for 3 seconds at 427-429 nm.</DD></DL></P><P><B>References</B></P><DL><DD>1. Kugimiya, A. and Fukada, R. (2015). Chemiluminescence Detection of Serine, Proline, Glycine, Asparagine, Leucine, and Histidine by Using Corresponding Aminoacyl-tRNA Synthetases as Recognition Elements. Appl Biochem Biotechnol. <A href="http://link.springer.com/article/10.1007/s12010-015-1639-6" target="_blank"> [online] </A></DD></DL></DIV></DIV><DIV class="togglebar"><DIV class="togglefour"><H2>Electrocompetent cells</H2></DIV><DIV id="togglefour"><P><B>Materials</B><DL><DD>- 1L sterile LB without NaCl (10g tryptone, 5g yeast extract per 1L)</DD><DD>- 500 mL of 10% v/v glycerol</DD><DD>- Cold falcon tubes of 50 mL</DD><DD>- Cold eppies and pipette tips</DD></DL></P><P><B>Protocol</B></P><DL><DT> Day 1:</DT><DD>1. Strike the cells on a plate and grow overnight at 37°C.</DD><DT>Day 2</DT><DD>1. Pick a single colony from your plate and grow it in 1-3 mL salt free LB overnight at 37°C.</DD><DT>Day 3: </DT><DD>1.	Grow 300-400 mL cells (without salt) in 37°C until the OD reaches 0.6 (use a starting culture).</DD><DD>2.	Cool down on ice and from now on perform all the steps at 4°C.</DD><DD>3.	Spin the cells down in falcon tubes (3500 g, 20 min, 4°C). Using falcon tubes ensures no detergents present.</DD><DD>4.	Resuspend the pellet in 30 mL icecold 10% glycerol (filtered to a disposable bottle to ensure the absence of detergents). Spin down the cells (5000 g, 10 min, 4°C). Repeat this step 3 times. </DD><DD>5.	Resuspend the cells in 10% glycerol to obtain a dense pulp (usually not more than 1.5 mL). </DD><DD>6.	Take 50 µL sample and do the electroporation test (without DNA). You should have a pulse of 4-6 msec. If shorter, wash the cells once again with 30 mL glycerol.</DD><DD>7.	Aliquot the cells (50 µL), quick-freeze in liquid nitrogen and store at -80°C.</DD><DIV class="togglebar"><DIV class="togglefive"><H2>Electroporation</H2></DIV><DIV id="togglefive"><P><B>Materials:</B></P><DL><DD>- DNA</DD><DD>- Electrocompetent cells</DD><DD>- SOC medium</DD><DD>- Ice-cold cuvettes</DD></DL><P><B>Protocols:</B></P><DL><DD>1. Add 1 µL DNA to 50 µL of electrocompetent cells in an ice-cold cuvette (1 mm).</DD><DD>2. Electroporate (Eppendorf, 1700 V, 4 msec).</DD><DD>3. Add 950 µL of SOC solution.</DD><DD>4. Incubate for one hour at 37°C.</DD><DD>5. Plate 50 µL out on pre-warmed plates (37°C).</DD><DL><DIV class="togglebar"><DIV class="togglesix"><H2>Miniprep</H2></DIV><DIV id="togglesix"><P><B>Materials:</B></P><P><DL><DD>Overnight liquid cultures</DD><DD>Thermo Scientific geneJET Plasmid Miniprep Kit</DD></DL></P><P><B>Protocol :</B></P><DL><DD>1. Transfer the liquid cultures to an Eppendorf tube and spin down at 13400 rpm for 1 to 2 minutes</DD><DD>2. Resuspend the pelleted cells in 250 µL of the Resuspension solution. The cells should be resuspended completely by vortexing or pipetting up and down until no cell clumps remain.</DD><DD>3. Add 250 µL of the Lysis Solution and mix thoroughly by inverting the tube 4-6 times until the solution becomes viscous and slightly clear. <B>Note.</B>  Do not vortex to avoid shearing of chromosomal DNA. 
Do not incubate for more than 5 min to avoid denaturation of supercoiled plasmid DNA. </DD><DD>4. Add 350 µL of the Neutralization Solution and mix immediately and thoroughly by inverting the tube 4-6 times. <B>Note.</B> It is important to mix thoroughly and gently after the addition of the Neutralization Solution to avoid localized precipitation of  bacterial cell debris. The neutralized bacterial lysate shoud become cloudy.</DD><DD>5. Centrifuge for 5 min to pellet cell debris and chromosomal DNA. </DD><DD>6. Transfer the supernatant to a supplied GeneJET spin column by decanting or pipetting. Avoid disturbing or transferring the white precipitate. <B>Note.</B> Close the bag with GeneJET Spin Columns tightly after each use! </DD><DD>7. Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube. </DD><DD>8. Add 500 µL of the Wash Solution (diluted with ethanol prior to first use) to the GeneJET spin column. Centrifuge for 30-60 seconds and discard the flow-through. Place the column back into the same collection tube. </DD><DD>9. Repeat the washing procedure (step 8) using 500 µL of the Wash Solution. </DD><DD>10. Discard the flow-through and centrifuge for an additional minute to remove residual Wash Solution. This step is essential to avoid residual ethanol in plasmid preps.</DD><DD>11. Transfer the GeneJET spin column to a fresh 1.5 mL microcentrifuge tube. Add 50 µL of the Elution Buffer to the center of the GeneJET spin column membrane to elute the plasmid DNA. Take care not to touch the membrane with the pipette tip. Incubate for 2 min at room temperature and centrifuge for 2 min. <B>Note.</B> An additional elution step (optional) with Elution Buffer or water will recover residual DNA from the membrane and increase the overall yield by 10-20%. For elution of plasmids or cosmids &gt;20 kb, prewarm Elution Buffer to 70°C before applying to silica membrane. </DD><DD>12. Discard the column and store the purified plasmid DNA at -20°C. </DD></DL></DIV></DIV><DIV class="togglebar"><DIV class="toggleseven"><H2>PCR Purification</H2></DIV><DIV id="toggleseven"><P><B>Materials:</B></P><DL><DD>PCR mixture</DD><DD>Thermo Scientific GeneJET PCR Purification Kit</DD></DL><P><B>Protocol:</B></P><DL><DD>1. Add a 1:1 volume of Binding Buffer to completed PCR mixture and mix thoroughly. Check the color of the solution. A yellow color indicates an optimal pH for DNA binding. If the color of the solution is orange or violet, add 10 µL of 3 M sodium acetate, pH 5.2 solution and mix. The color of the mix will become yellow.</DD><DD>2. Transfer up to 800 µL of the solution from step 1 to the GeneJET purification column. Centrifuge for 30-60 s. Discard the flow-through. <B>Note</B>. If the total volume exceeds 800 µL, the solution can be added to the column in stages. After the addition of 800 µL of solution, centrifuge the column for 30-60 s and discard flow-through. Repeat until the entire solution has been added to the column membrane. Close the bag with GeneJET Purification Columns tightly after each use! </DD><DD>3. Add 700 µL of Wash Buffer to the GeneJET purification column. Centrifuge for 30-60 s. Discard the flow-through and place the purification column back into the collection tube.</DD><DD>4. Transfer the GeneJET purification column to a clean 1.5 mL microcentrifuge tube, add 50 µL of Elution Buffer to the center of the GeneJET purification column membrane and centrifuge for 1 min. <B>Note</B>. For low DNA amounts, the elution volume can be reduced to increase DNA concentration. An elution volume between 20-50 µL does not significantly reduce the DNA yield. However, elution volumes less than 10 µL are not recommended. If DNA fragment is &gt;10 kb, prewarm Elution Buffer to 65°C before applying to column. If the elution volume is 10 µL and DNA amount is ≥5 µg, incubate column for 1 min at room temperature before centrifugation.</DD><DD>5. Discard the GeneJET purification column and store the purified DNA at -20°C. </DD><DL><DIV class="togglebar"><DIV class="toggleeight"><H2>Gel Purification</H2></DIV><DIV id="toggleeight"><P><B>Materials:</B><DL><DD>- Gel fragment </DD><DD>- QIAquick Gel Extraction Kit </DD></DL></P><P><B>Protocol : </B></P><DL><DD>1. Excise the DNA fragment from the agarose gel with a clean, sharp scalpel. Minimize the size of the gel slice by removing extra agarose. </DD><DD>2. Weigh the gel slice in a colorless tube. Add 3 volumes of Buffer QG to 1 volume of gel (100 mg ~ 100 µl). <B>Note</B>. For &gt;2% agarose gels, add 6 volumes of Buffer QG. The maximum amount of gel slice per QIAquick column is 400 mg; for gel slices &gt;400 mg use more than one QIAquick column. </DD><DD>3. Incubate at 50°C for 10 min (or until the gel slice has completely dissolved). To help dissolve gel, mix by vortexing the tube every 2–3 min during the incubation. IMPORTANT: Solubilize agarose completely. For &gt;2% gels, increase incubation time. 
After the gel slice has dissolved completely, check that the color of the mixture is yellow (similar to Buffer QG without dissolved agarose). If the color of the mixture is orange or violet, add 10 µl of 3 M sodium acetate, pH 5.0, and mix. The color of the mixture will turn to yellow. The adsorption of DNA to the QIAquick membrane is efficient only at pH ≤7.5. Buffer QG contains a pH indicator which is yellow at pH ≤7.5 and orange or violet at higher pH, allowing easy determination of the optimal pH for DNA binding. </DD><DD>4. Place a QIAquick spin column in a provided 2 ml collection tube. 
To bind DNA, apply the sample to the QIAquick column, and centrifuge for 1 min. The maximum volume of the column reservoir is 800 µL. For sample volumes of more than 800 µL, simply load and spin again. </DD><DD>5. Discard flow-through and place QIAquick column back in the same collection tube. Collection tubes are re-used to reduce plastic waste. </DD><DD>6.  To wash, add 0.75 mL of Buffer PE to QIAquick column and centrifuge for 1 min. Note: If the DNA will be used for salt sensitive applications, such as blunt-end ligation and direct sequencing, let the column stand 2–5 min after addition of Buffer PE, before centrifuging. </DD><DD>7. Discard the flow-through and centrifuge the QIAquick column for an additional minute at ≥10,000 x g (~13,000 rpm). IMPORTANT: Residual ethanol from Buffer PE will not be completely removed unless the flow-through is discarded before this additional centrifugation.</DD><DD>8. Place QIAquick column into a clean 1.5 mL microcentrifuge tube. </DD><DD>9. To elute DNA, add 50 µL of Buffer EB (10 mM Tris·Cl, pH 8.5) to the center of the QIAquick membrane and centrifuge the column for 1 min at maximum speed. Alternatively, for increased DNA concentration, add 30 µl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge for 1 min. IMPORTANT: Ensure that the elution buffer is dispensed directly onto the QIAquick membrane for complete elution of bound DNA. The average eluate volume is 48 µl from 50 µl elution buffer volume, and 28 µl from 30 µl. Elution efficiency is dependent on pH. The maximum elution efficiency is achieved between pH 7.0 and 8.5. When using water, make sure that the pH value is within this range, and store DNA at –20°C as DNA may degrade in the absence of a buffering agent. The purified DNA can also be eluted in TE (10 mM Tris·Cl, 1 mM EDTA, pH 8.0), but the EDTA may inhibit subsequent enzymatic reactions.
</DD><DIV class="togglebar"><DIV class="togglenine"><H2>Nanodrop measurement</H2></DIV><DIV id="togglenine"><P><B>Materials</B></P><DL><DD> Thermo Scientific Nanodrop 2000 </DD></DL><P><B>Protocol:</B></P><P><DL><DD>1. Measure Blank using the nanometer with 2 µL the buffer you used to elute your sample.</DD><DD>2. Put 2 µL of the sample on the nanometer, and average the result.</DD></DL></P></DIV></DIV><DIV class="subsections"><DIV class="subsectionwrapper"><DIV class="subtextrow"><DIV class="subtext"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Idea"><H2>Idea</H2><P>A detailed description about the interaction between our two cells and the genetic circuit can be found here.

</P></A></DIV><DIV class="subtext"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Basic_Part"><H2>Basic Parts</H2><P> Our new, self-designed basic parts necessary to control cell-cell interactions and <I>E. Coli</I> motility.</P></A></DIV></DIV></DIV><DIV class="subsectionwrapper"><DIV class="subtextrow"><DIV class="subtext"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Composite_Part"><H2>Composite Parts</H2><P> Our basic parts were combined with each other and with existing iGEM promotors, RBS and terminators.
</P></A></DIV><DIV class="subtext"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Results"><H2>Results</H2><P> The results of our experiments will appear here.
</P></A></DIV><DIV class="subtext"><A href="https://2015.igem.org/Team:KU_Leuven/Research"><H2>Back</H2><P> Go back to the Research page.
</P></A></DIV></DIV><DIV class="subimgrowm"><DIV class="subimgm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Idea"><B>Idea</B></A></DIV><DIV class="subtextm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Idea"><P>
A detailed description about the interaction between our two cells and the genetic circuit can be found here.

        </P></A></DIV></DIV><DIV class="subimgrowm"><DIV class="subimgm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Basic_Part"><B>Basic Parts</B></A></DIV><DIV class="subtextm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Basic_Part"><P>
          Our new, self-designed basic parts necessary to control cell-cell interactions and <I>E. Coli</I> motility.
        </P></A></DIV></DIV><DIV class="subimgrowm"><DIV class="subimgm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Composite_Part"><B>Composite Parts</B></A></DIV><DIV class="subtextm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Composite_Part"><P>
          Our basic parts were combined with each other and with existing iGEM promotors, RBS and terminators.
        </P></A></DIV></DIV><DIV class="subimgrowm"><DIV class="subimgm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Results"><B>Results</B></A></DIV><DIV class="subtextm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Results"><P>
        Click here to discover our results
        </P></A></DIV></DIV><DIV class="subimgrowm"><DIV class="subimgm"><A href="https://2015.igem.org/Team:KU_Leuven/Research"><B>Back</B></A></DIV><DIV class="subtextm"><A href="https://2015.igem.org/Team:KU_Leuven/Research"><P>
         Go back to the Research page.
        </P></A></DIV></DIV></DIV></DIV><DIV id="summarywrapper"><DIV class="summary"><H3> Contact </H3><P style="font-size:1.3em; text-align: center">
Address: Celestijnenlaan 200G room 00.08 - 3001 Heverlee
Telephone: +32(0)16 32 73 19
Email: igem@chem.kuleuven.be</P></DIV></DIV></DL></DIV></DIV></DL></DL></DIV></DIV></DL></DL></DIV></DIV></DL></DIV></DIV></DL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>