<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_KU_Leuven_Research_Idea skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:KU Leuven/Research/Idea</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="main-navm"><DIV class="nav-itemm"><DIV id="main"><A href="https://2015.igem.org/Team:KU_Leuven"><SPAN>main</SPAN></A></DIV></DIV><DIV class="nav-itemm"><DIV id="research"><A href="https://2015.igem.org/Team:KU_Leuven/Research"><SPAN>Research</SPAN></A></DIV></DIV><DIV class="nav-itemm"><DIV id="interlab"><A href="https://2015.igem.org/Team:KU_Leuven/InterLabStudy"><SPAN>Interlab</SPAN></A></DIV></DIV><DIV class="nav-itemm"><DIV id="modeling"><A href="https://2015.igem.org/Team:KU_Leuven/Modeling"><SPAN>Modeling</SPAN></A></DIV></DIV><DIV class="nav-itemm"><DIV id="outreach"><A href="https://2015.igem.org/Team:KU_Leuven/Practices"><SPAN>Outreach</SPAN></A></DIV></DIV><DIV class="nav-itemm"><DIV id="project"><A href="https://2015.igem.org/Team:KU_Leuven/Future"><SPAN>Future</SPAN></A></DIV></DIV><DIV class="nav-itemm"><DIV id="team"><A href="https://2015.igem.org/Team:KU_Leuven/Team"><SPAN>Team</SPAN></A></DIV></DIV><DIV class="nav-itemm"><DIV id="notebook"><A href="https://2015.igem.org/Team:KU_Leuven/Notebook"><SPAN>Notebook</SPAN></A></DIV></DIV></DIV><DIV class="main-nav"><DIV id="centernav"><DIV class="nav-item"><DIV id="main"><A href="https://2015.igem.org/Team:KU_Leuven"><SPAN>main</SPAN></A></DIV></DIV><DIV class="nav-item"><DIV id="research"><A href="https://2015.igem.org/Team:KU_Leuven/Research"><SPAN>Research</SPAN></A></DIV></DIV><DIV class="nav-item"><DIV id="interlab"><A href="https://2015.igem.org/Team:KU_Leuven/InterLabStudy"><SPAN>Interlab</SPAN></A></DIV></DIV><DIV class="nav-item"><DIV id="modeling"><A href="https://2015.igem.org/Team:KU_Leuven/Modeling"><SPAN>Modeling</SPAN></A></DIV></DIV><DIV class="nav-item"><DIV id="outreach"><A href="https://2015.igem.org/Team:KU_Leuven/Practices"><SPAN>Outreach</SPAN></A></DIV></DIV><DIV class="nav-item"><DIV id="project"><A href="https://2015.igem.org/Team:KU_Leuven/Future"><SPAN>Future</SPAN></A></DIV></DIV><DIV class="nav-item"><DIV id="team"><A href="https://2015.igem.org/Team:KU_Leuven/Team"><SPAN>Team</SPAN></A></DIV></DIV><DIV class="nav-item"><DIV id="notebook"><A href="https://2015.igem.org/Team:KU_Leuven/Notebook"><SPAN>Notebook</SPAN></A></DIV></DIV></DIV></DIV><DIV class="summaryheader"><DIV class="summaryimg"><DIV class="head"><H2> Idea </H2></DIV></DIV></DIV><DIV class="summarytext1"><DIV class="scientific"><DIV class="part"><H2> Introduction </H2><P>
Intrigued by the patterns occuring in nature, our research started by designing possible interaction schemes, genetic circuits and browsing specialised literature to find interesting papers concerning the formation of patterns using
 <I>Escherichia coli</I>
 (<I>E. coli</I>). For example, Basu <I>et al.</I> created ring-like patterns based on the chemical gradients of an acyl-homoserine lactone (OHHL) signal that is synthesized by ‘sender’ cells. In ‘receiver’ cells, designed genetic networks respond to differences in OHHL concentrations
<SUP><A href="#Basu2005">[1]</A></SUP>. 
Liu <I> et al.</I> created periodic stripes of high and low cell densities of 
<I>E. coli</I>, by inhibiting cell motility when cell density was high
<SUP><A href="#Liu2011">[2]</A></SUP>. 
Combining our innovative and altered chemotaxis intercellular relationship with basic principles from both papers, allowed us to design our own circuit which is elucidated in the following paragraphs.
</P><DIV class="center"><DIV id="image1"><H4><DIV id="figure1">Figure 1</DIV>
Signaling cascade for motility in <I> E. coli </I></H4></DIV></DIV></DIV><DIV class="part"><H2> Our design </H2><P>
Two different cell types, called A and B, interact and create patterns. In order to achieve the desired behavior, the cells used in our experiments are derived from 
<I>Escherichia coli </I> 
K12 and have specific knockouts in the genome. Cell type A has a deletion of genes
<I>tar</I> and 
<I>tsr</I>, whereas in cell type B 
<I>tar</I> and 
<I>cheZ</I> genes are knocked out. Tar and Tsr are two major chemotactic receptors, that guide the cells towards nutrients. When a certain small molecule binds one of these receptors, a signaling cascade is initiated. In our specific case, cell’s A deletion of both 
<I>tsr</I> and 
<I>tar</I> causes them to be insensitive to leucine (as a repellent and attractant). Cell B on the other hand, which lost only Tar, is repelled by leucine since chemoreceptors are still present here.
<SUP><A href="#Khan2004">[3]</A></SUP>
Upon recognition of the small molecule (in this case leucine), the receptor transduces the signal through a set of Che proteins. This group of methylesterases and phosphatases regulates the rotation of the flagella (Figure 1). In <I> E. coli</I>, the motors turn counterclockwise (CCW) in their default state, allowing several filaments on the cell to join together in a bundle and propel the cell smoothly forward. The interaction of phospho-CheY and a motorprotein causes the motors to switch to clockwise (CW) rotation, inducing dissociation of the filament bundle and tumbling of the cell. (Figure 1)
<SUP><A href="#Sarkar2010">[4]</A></SUP>
To regulate CheY activity, a protein called CheZ removes a phosphate group and subsequently inhibits its activity. Cells lacking this protein  are not able to swim and will tumble excessively and incessantly
<SUP><A href="#Kuo1987">[5]</A></SUP>. 
</P><DIV class="center"><DIV id="image2"><H4><DIV id="figure2">Figure 2</DIV> Left: plasmid A, Right: plasmid B. Click to enlarge </H4></DIV></DIV><P>
Both cells are transfected with only one plasmid each (Figure 2). Both plasmids contain a temperature sensitive cI repressor protein, which is constitutively expressed. This repressor is only able to bind to the lambda promoter at temperatures below a certain threshold (between 30°C and 42°C). At elevated temperatures, cI is unable to bind to the promoter region enabling RNA polymerase to start transcription.
<SUP><A href="#Villaverde1993">[6]</A></SUP> 
From the lambda promoter in cell A, two essential protagonists for our system are expressed: the OHHL-producing enzyme LuxI and the transcription activator LuxR
<SUP><A href="#Fuqua2001">[7]</A></SUP>. 
On the contrary, cell B only contains the lambda promoter coupled to the <I>luxR</I> gene. In cell A, we coupled the expression of an autoaggregation factor Ag43-YFP and a leucine-producing enzyme Transaminase B to the OHHL-LuxR regulated promoter. Ag43 causes cells A to stick together and form clumps
<SUP><A href="#Ullet2006">[8]</A></SUP>. 
At the same time, Transaminase B catalyses the production of the repellent leucine
<SUP><A href="#Cox1989">[9]</A></SUP>. 
The repellent has no impact on cell A, since it does not contain the necessary receptors  anymore. Cell B on the other hand is repelled by leucine, and contains the Lux promoter coupled to CheZ-GFP and the transcription repressor PenI. This repressor binds to the Pen-promoter 
<SUP><A href="#Wittman">[10]</A></SUP>, which regulates the transcription of a red fluorescent protein (RFP). The use of fluorescent fusion proteins specific for a certain state (aggregated, swimming, tumbling) facilitates the read-out of our patterns. In order to measure the concentration of the key proteins LuxI and LuxR, they were fused with a His-tag and an E-tag respectively. To ensure rapid degradation of proteins whose expression is dependent on OHHL concentration, an LVA tag was fused to CheZ, GFP and RFP. (Figure 3)
</P><DIV class="center"><DIV id="image3"><H4><DIV id="figure3"> Figure 3 </DIV>
The designed circuit </H4></DIV></DIV></DIV><DIV class="part"><H2>Hypotheses</H2><P>
We expect that by raising the temperature, cells of type A will start to secrete OHHL. This OHHL will activate cells A to produce Ag43 and leucine and cells B to express CheZ-GFP and PenI. Cells A will start to form distinct spots on the agar plate and will emit yellow light (λ = 528 nm). At the same time, cells B will be repelled from cells A by leucine, the expression of PenI will repress the expression of RFP and thus the red color, whereas the expression of CheZ will enable the bacteria to swim and render them green. When they have swum too far away from cells A, the concentration of OHHL decreases to levels which are too low for LuxR to bind to the promotor. When LuxR cannot bind anymore, CheZ and PenI will no longer be expressed causing the bacteria to tumble incessantly and become red again. Figure 4 shows the patterns we expect in the end  
</P><DIV class="center"><DIV id="image4"><H4><DIV id="figure4">Figure 4</DIV>
Expected patterns</H4></DIV></DIV></DIV></DIV><DIV class="easy"><DIV class="part"><H2> Introduction </H2><P>
Scientists have been interested in pattern formation for a long time. By using different techniques they were already able to generate fascinating and sometimes complex patterns on an agar plate (see Figure 1). </P><DIV class="center"><DIV id="image1"><H4> 
Figure 1

Example of pattern formation on a petri dish (picture from Chenli Liu <I> et al. </I> ) </H4></DIV></DIV><P>
The basic principles behind pattern formation relies on two fundamental properties of bacteria: the communication between different cells and their ability to swim. Our project combines both properties, and adds another dimension to them. Besides, we included a mechanism that causes the first type of bacteria to clump together and at the same time chases away the second type of cells. According to our first raw models, this should allow to generate patterns.</P></DIV><DIV class="part"><H2> Our circuit </H2><P>
Our circuit consists of two different cells, custom-made for this project. The first one, called cell type A, was created in such a way that it will never be repelled by itself or by the other cells. The second one, called cell type B, is unable to swim under normal circumstances and is sensitive to certain repelling molecules.  Both cells produce different proteins. These cellular workforces execute a variety of functions: </P><P>

Cell A contains a self-designed plasmid. Here, among others, two genes (LuxI and LuxR) will be expressed.  LuxI produces a small molecule called OHHL.  Binding of OHHL together with LuxR to a specific DNA sequence, called a promoter, is needed to start production of a second group of proteins (Transaminase B and Ag43). Ag43 is some kind of molecular velcro, causing cell A to aggregate.  Transaminase B on the other hand produces the molecules that cause cells B to be repelled. </P><P>

Cell B on the other hand only contains LuxR instead of both LuxI and LuxR as was the case for cells A. Because of the lack of the OHHL producing enzyme LuxI, the proteins with a LuxR and OHHL dependent production (in this case CheZ and PenI) are only expressed when cells B are close to cells A. In this case, expression of CheZ restores the ability to swim, whereas PenI stops the synthesis of the red fluorescent protein. This red color indicates the standard and non-induced situation for cell B (unable to swim), and is used as an easy read-out to distinguish between the different cells. </P><DIV class="center"><DIV id="image3"><H4><DIV id="figure2"> Figure 2 </DIV>
The designed circuit </H4></DIV></DIV><P>
In order to visualize induced patterns in an efficient way, we indicated some proteins (Ag43 and CheZ) with a color by fusing them with two fluorescent proteins. The entire system is temperature sensitive, since the expression of LuxI and LuxR will only start when the temperature is above 42°C. To achieve this we included a protein (cI) that binds to the DNA at a specific site and blocks production of LuxI and/or LuxR. However, at higher temperatures, this protein will start to loose its three dimensional structure and the blockade will be lifted. This will in turn initiate the whole pattern formation process. Figure 2 gives a summary of our circuit</P></DIV><DIV class="part"><H2> Expectations </H2><P>
So, in theory, when the temperature rises, patterns will be formed. Cells A will start to form aggregates and will thereby obtain a yellow color. At the same time, they will produce the repellents for cells B. In cells B, the red color will disappear, and the swimming motility will be restored. A green color will appear simultaneously. However, when cell B is too far from cell A, the second group of proteins is no longer expressed, leading to a subsequent loss of motility and green color. The red fluorescent color will return afterwards. </P><P>
Normally, this will produce distinct yellow spots, with a green circle of cells B that are able to swim around the cells A. At the edges, the immotile red cells B will start to appear again. Figure 4 shows the patterns we expect in the end </P><DIV class="center"><DIV id="image4"><H4> 
 Figure 4
Expected patterns</H4></DIV></DIV></DIV></DIV><DIV class="scientific"><DIV class="summaryheader"><DIV class="summaryimg"><DIV class="head"><H2> References </H2></DIV></DIV></DIV><DIV class="reference"><DIV class="part"><TABLE><TBODY><TR valign="top"><TD class="bibtexnumber">
[1]
    </TD><TD class="bibtexitem">
Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold &amp; Ron Weiss “A synthetic multicellular system for programmed pattern formation” Nature 434 (2005): 1130-1134.
    </TD></TR><TR valign="top"><TD class="bibtexnumber">
[2]
    </TD><TD class="bibtexitem">
Chenli Liu, Xiongfei Fu, Lizhong Liu, Xiaojing Ren, Carlos K.L. Chau, Sihong Li, Lu Xiang, Hualing Zeng, Guanhua Chen, Lei-Han Tang, Peter Lenz, Xiaodong Cui, Wei Huang, Terence Hwa, Jian-Dong Huang” Sequential Establishment of Stripe Patterns in an Expanding Cell Population” Science 334 no. 6053 (2011) : 238-241.
    </TD></TR><TR valign="top"><TD class="bibtexnumber">
[3]
    </TD><TD class="bibtexitem">
Khan, S., and D. R. Trentham. “Biphasic Excitation by Leucine in Escherichia Coli Chemotaxis.” Journal of Bacteriology 186, no. 2 (2004): 588-592.
    </TD></TR><TR valign="top"><TD class="bibtexnumber">
[4]
    </TD><TD class="bibtexitem">
Sarkar MK, Paul K, Blair D , “Chemotaxis signaling protein CheY binds to the rotor protein FliN to control the direction of flagellar rotation in Escherichia coli.” PNAS 107, no. 20 (2010): 9370-9375.
    </TD></TR><TR valign="top"><TD class="bibtexnumber">
[5]
    </TD><TD class="bibtexitem">
Kuo, Scot C., and D. E. Koshland. “Roles of cheY and cheZ Gene Products in Controlling Flagellar Rotation in Bacterial Chemotaxis of Escherichia Coli.” Journal of Bacteriology 169, no. 3 (1987): 1307–14.
    </TD></TR><TR valign="top"><TD class="bibtexnumber">
[6]
    </TD><TD class="bibtexitem">
Villaverde, A., A. Benito, E. Viaplana, and R. Cubarsi. “Fine Regulation of cI857-Controlled Gene Expression in Continuous Culture of Recombinant Escherichia Coli by Temperature.” Applied and Environmental Microbiology 59, no. 10 (1993): 3485–87.
    </TD></TR><TR valign="top"><TD class="bibtexnumber">
[7]
    </TD><TD class="bibtexitem">
Clay Fuqua , Matthew R. Parsek , and E. Peter Greenberg “Regulation of Gene Expression by Cell-to-Cell Communication: Acyl-Homoserine Lactone Quorum Sensing” , Annu. Rev. Genet. (2001) 35:439–68
    </TD></TR><TR valign="top"><TD class="bibtexnumber">
[8]
    </TD><TD class="bibtexitem">
Ulett, G. C. “Antigen-43-Mediated Autoaggregation Impairs Motility in Escherichia Coli.” Microbiology 152, no. 7 (2006): 2101–2110.
    </TD></TR><TR valign="top"><TD class="bibtexnumber">
[9]
    </TD><TD class="bibtexitem">
James L. Cox, Betty J. Cox , Vincenzo Fidanza , David H. Calhoun “The complete nucleotide sequence of the iZvGMEDA cluster of Escherichia coli K-12” Gene 56, Issues 2–3 (1987): 185–198.
    </TD></TR><TR valign="top"><TD class="bibtexnumber">
[10]
    </TD><TD class="bibtexitem">
Wittman, V., H. C. Lin, and H. C. Wong. “Functional Domains of the Penicillinase  Repressor of Bacillus Licheniformis.” Journal of Bacteriology 175, no. 22 (1993):  7383–7390. <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC206883/" target="_blank">[10]  </A></TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV class="subsections"><DIV class="subsectionwrapper"><DIV class="subtextrow"><DIV class="subtext"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Methods"><H2>Methods</H2><P> Here you can find the performed steps to create two different cell types and detailed quantification methods to determine interesting parameters.
</P></A></DIV><DIV class="subtext"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Basic_Part"><H2>Basic Parts</H2><P> Our new, self-designed basic parts necessary to control cell-cell interactions and <I>E. Coli</I> motility.</P></A></DIV></DIV></DIV><DIV class="subsectionwrapper"><DIV class="subtextrow"><DIV class="subtext"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Composite_Part"><H2>Composite Parts</H2><P> Our basic parts were combined with each other and with existing iGEM promotors, RBS and terminators.
</P></A></DIV><DIV class="subtext"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Results"><H2>Results</H2><P> The results of our experiments will appear here.
</P></A></DIV><DIV class="subtext"><A href="https://2015.igem.org/Team:KU_Leuven/Research"><H2>Back</H2><P> Go back to the Research page.
</P></A></DIV></DIV><DIV class="subimgrowm"><DIV class="subimgm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Methods"><B>Methods</B></A></DIV><DIV class="subtextm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Methods"><P>
Here you can find the performed steps to create two different cell types and detailed quantification methods to determine interesting parameters.
        </P></A></DIV></DIV><DIV class="subimgrowm"><DIV class="subimgm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Basic_Part"><B>Basic Parts</B></A></DIV><DIV class="subtextm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Basic_Part"><P>
          Our new, self-designed basic parts necessary to control cell-cell interactions and <I>E. coli</I> motility.
        </P></A></DIV></DIV><DIV class="subimgrowm"><DIV class="subimgm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Composite_Part"><B>Composite Parts</B></A></DIV><DIV class="subtextm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Composite_Part"><P>
          Our basic parts were combined with each other and with existing iGEM promotors, RBS and terminators.
        </P></A></DIV></DIV><DIV class="subimgrowm"><DIV class="subimgm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Results"><B>Results</B></A></DIV><DIV class="subtextm"><A href="https://2015.igem.org/Team:KU_Leuven/Research/Results"><P>
        Click here to discover our results
        </P></A></DIV></DIV><DIV class="subimgrowm"><DIV class="subimgm"><A href="https://2015.igem.org/Team:KU_Leuven/Research"><B>Back</B></A></DIV><DIV class="subtextm"><A href="https://2015.igem.org/Team:KU_Leuven/Research"><P>
         Go back to the Research page.
        </P></A></DIV></DIV></DIV></DIV><DIV id="summarywrapper"><DIV class="summary"><H3> Contact </H3><P style="font-size:1.3em; text-align: center">
Address: Celestijnenlaan 200G room 00.08 - 3001 Heverlee
Telephone: +32(0)16 32 73 19
Email: igem@chem.kuleuven.be</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>