Team:KU Leuven/Journal/Newsfeed
main
Research
Interlab
Modeling
Outreach
Future
Team
Notebook
main
Research
Interlab
Modeling
Outreach
Future
Team
Notebook
Show all
Newsfeed
Week 6 (03-07/08/2015)
hide
-We received material kindly given by KOLO Instruments -Paulussen Freddy
-Searching hotels in Bordeaux during the symposium.
-Folders and brochures are ordered.
hide
-Write protocol for AHL detection
-Write abstract
hide
-Perform gel electrophoresis to confirm double knock-outs: 2 colonies of ΔTar ΔCheZ and 4 colonies of ΔTar ΔTsr still had ΔTar. This means we have our double mutants! We still need to check ΔTar ΔCheZ to be sure the rest of the operon is ok.
-We performed a motility test to verify the phenotypical change of knocking out CheZ
-Assembly of gBlocks by using the Gibson Assembly Method
-We decided to participate at the iGEM 2015 Measurement Interlab Study. Therefore we transformed E. cloni with the biobricks J23101, I12504, J23106 and J23117.
hide
-Write report about Simbiology
-Extending Hybrid Model to 2D
hide
- Update Facebook & Twitter
hide
-Contact potential keynote speakers for the symposium
hide
-The History-page is online!
hide
-Making images for wiki icons for subsections of the Newsfeed
-Making tattoo-images to use as gadget
Week 5 (27-31/07/2015)
hide
-Plane tickets to Boston are booked
-Hotels Boston are booked
-We have two new sponsors: Eppendorf & LRD!
hide
-Make protocol for plasmid assembly
-Make protocol for leucine detection
-Make protocol for AHL detection
-Design & order primers for the Gibson Assembly Method
hide
-Making double knock-out strains by P1 transduction
-Performing PCR and gel electrophoresis to confirm that we have double knock-outs
hide
-The 2D models on 100 by 100 grid are working
hide
- Update Facebook & Twitter
-Mail library for putting the display online
hide
-Contacting possible key speakers
-Making a survey about public perception of synthetic biology
-Mail Bordeaux for meet up francophone
hide
-Modeling page online
-Adjust the description of the project
hide
-Making images for wiki team presentation
-Making images for wiki icons for subsections of the Newsfeed
-Making animation that represents our pattern forming bacteria
Week 4 (20-24/07/2015)
hide
-Meeting Toulouse team at 07/21/15 in Brussels
hide
-Search parameters necessary in mathematical model
-Design and order the designed plasmid gBlocks
hide
-Performing a PCR and gel electrophoresis to confirm that the kanamycin cassette has successfully been removed from ΔTar and make a stock of this new strain.
-Preparations for phage P1 lysate for making the double knock-out strains
hide
-Simulating cell A and cell B in SymBiology
-Looking for usable constants
-Adapting the 2D continuous model
hide
- Update Facebook & Twitter
-Making understandable mail about our project for family and friends in English and Dutch
-Mail for collaboration with Indian team
-Send reminders to contact responsible people for adding a text of our iGEM project to the websites of our faculties
hide
-Looking for possible key speakers
-Design flyer
hide
-Team page is added
hide
-Making images for wiki team presentation
-Making images of animals with new patterns for the wiki
Week 3 (13-17/07/2015)
hide
-Construct plasmid
-Design & order primers for controlling the knock-out processes
-Research to an alternative knock-out technique for double knockouts
hide
-Calculation of transformation efficiency of competent cells (E. cloni)
-Order 3 knock-out strains (ΔTar, ΔTsr and ΔCheZ) & prepare a stock
-Order Chromobacterium violaceum CV026 transposon mutant for use in AHL detection & prepare a stock
-Removing the kanamycin resistance gene of the ΔTar by transforming a plasmid with recombinase gene
hide
-Making and adapting the 1D hybrid and continuous model to the conditions of the wet lab
-Making a simple 2D continuous model
-Thinking about true parameters
-Making a 1D model with pdepe in Matlab -Explore symbiology
-Making a 2D model with PDE toolbox in Matlab (not ready yet)
hide
- Update Facebook & Twitter
- Contact responsible people for adding a text of our iGEM project to the websites of our faculties
- Contacting Toulouse team 2015 for meeting them at 07/21/15
- Interview with Jo De Wachter of the Science department
hide
-Preparing e-mail for symposium and contact possible key speaker
hide
-The description of our project is online
hide
-Make images for wiki
Week 2 (06-10/07/2015)
hide
-Discussion with modeling team: which parameters do they need?
-Search experiments for quantification of specific proteins, small molecules and amino acids
-Decide on promoters of the plasmid
-Construct the plasmids
-Make a working scheme (strategy)
hide
-Prepare LB agar medium
-Prepare competent cells (E. cloni) and test competency
hide
-Literature research into hybrid models
-Further work on single cell agent-based model
-Implemented simple one-dimensional hybrid model
-Explore PDE Toolbox
-Work on an implicit continuous model
-Try if it’s possible to simulate pattern formation of bacteria in COMSOL
hide
-Update Facebook & Twitter: picture and text of every individual team member
-Contact responsible people for adding a text of our iGEM project to the websites of our faculties
hide
-Thinking of school projects for primary school and secondary school
-Thinking of symposium
hide
-Helping with images and layout of wiki
-Helping with images and brochure for sponsors
Week 1 (01-03/07/2015)
hide
-Lab safety training
-Discussing tasks and practical arrangements (tickets Boston)
-Taking photos to use in the brochure, on the wiki and for social media
-Take a tour through our high tech bio laboratory
-Meeting with potential sponsor
hide
-Searching for strains and biobricks for our circuit
hide
-Set up GitHub
-Constructed simple single cell agent-based model -Worked on an explicit discretization of a continuous model
hide
-Link Facebook with Twitter
hide
-’Coming soon’ page is online
hide
-Helping with images and layout of wiki
-Helping with images and brochure for sponsors
Back
