<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NYMU-Taipei_ymiw2_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NYMU-Taipei/ymiw2.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NYMU-Taipei/ymiw2.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/ymiw2.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/ymiw2.html&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NYMU-Taipei/ymiw2.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NYMU-Taipei/ymiw2.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>NYMU iGEM</TITLE></P><DIV id="ymi_topcontainer"><DIV id="ymi_top"><A href="https://2012.igem.org/Team:NYMU-Taipei/ymiproject.html" target="_parent">Project Venusian</A> ·
    <A href="https://2012.igem.org/Team:NYMU-Taipei/ymim1" target="_parent">Modeling</A> · 
    <A href="https://2012.igem.org/Team:NYMU-Taipei/ymihpa.html">Human Practice</A> · 
    <A href="https://2012.igem.org/Team:NYMU-Taipei/ymijf.html">Extras</A> · 
    <A href="https://2012.igem.org/Team:NYMU-Taipei/ymit1">Team</A> · 
    <A href="https://2012.igem.org/Main_Page">iGEM</A></DIV></DIV><DIV id="ymi_container"><DIV id="ymi_left_column"><DIV class="text_area" align="justify"><DIV class="title">Week 5 ~ Week 8 (7/29~8/25)</DIV><DIV align="left"><P>Week 5 (7/29~8/4)</P><P><STRONG>Sunday</STRONG>SRB</P><P>          * Other alternatives decision: find out specific enzyme for  sulfur reducing from KEGG, paper survey<STRONG>
    Monday</STRONG>SRB</P><P>      * Paper survey (waiting for primer also)
    * Design primer for NosZ and NorCB<STRONG>Tuesday</STRONG>SRB(Cloning ofCycI)</P><P>      * PCR construction for new pSB1C3 (we named it MF-pSB1C3):  MfeI-Xba I-pSB1C3 backbone- Sbf I-Spe I ( for tater gene which has endogenous  EcoRI, PstI cutting site)
    * PCR of CyC I
    Template DNA
    Forward primer: CyC I -FP (MX_ SmtA _F)
    Reverse primer: CyC I -RP (SbfIS_ SmtA _R)
    conclusion: positive
    *PCR clean up
    *enzyme cut
    Rescrition enzyme: pSB1C3: MfeI.SbfI
    SmtA: MfeI.SbfI</P><UL><LI>NosZ+NorCB</LI></UL><P>      * PCR of NosZ
    Template DNA: pseudomonas aeruginosa
    Forward primer: NosZ -FP (EX_ NosZ _F)
    Reverse primer: NosZ -RP (SbfIS_ NosZ _R)
    conclusion: positive
    *PCR clean up
    *enzyme cut
    Rescrition enzyme: MF-pSB1C3: EcoRI. SbfI
    NosZ: EcoRI. SbfI
    * PCR of NorCB
    Template DNA
    Forward primer: NorCB -FP (EX _ NorCB _F)
    Reverse primer: NorCB -RP (SbfIS_ NorCB _R)
    conclusion: positive
    *PCR clean up
    *enzyme cut
    Rescrition enzyme: MF-pSB1C3: EcoRI. SbfI
    NorCB: EcoRI. SbfI
    To combine NosZ and NorCB, we  need to do another enzyme cut
    *enzyme cut
    Rescrition enzyme: MF-pSB1C3: EcoRI. SbfI
    NosZ: EcoRI. SpeI 
    NorCB: XbaI. SbfI<STRONG>Wednesday</STRONG>SRB</P><P>      * Clean up
    * Ligation: MF-pSB1C3-CycI 
    conclusion: positive
    * Transformation of competent cell
    -prepare Cm20 plate
    - Plate the bacteria onto plates and place  at 37oC overnight</P><UL><LI>NosZ+NorCB</LI></UL><P>      * Clean up
    * Ligation: MF-pSB1C3-NosZ
    MF-pSB1C3-NorCB
    MF-pSB1C3-NosZ-NorCB
    conclusion: positive
    * Transformation of competent cell
    -prepare Cm20 plate
    - Plate the bacteria onto plates and place  at 37oC overnight</P><P><STRONG>
    Thursday</STRONG> SRB</P><P>      * acquire many colony on the plate
    -pick 8 colony on each plate(Cm1~Cm8)
    * Colony PCR
    Forward  primer: CmR gene-R-SpeI
    Reverse primer: CycI-RP
    Conclusion: Cm1~ Cm 8 positive</P><UL><LI>NosZ+NorCB</LI></UL><P>      * acquire some colony( &lt;=8 ) on the plate
    - pick 8 colony on MF-pSB1C3-NosZ plate(Cm1~Cm8)
    - pick 6 colony on MF-pSB1C3-NorCB plate(Cm1~Cm6)- pick 8 colony on MF-pSB1C3-NosZ-NorCB plate(Cm1~Cm8)
    * Colony PCR
    Forward  primer: CmR gene-R-SpeI
    Reverse primer(for MF-pSB1C3-NosZ): NosZ –RP
    Conclusion: Cm1~ Cm 8 positive
    Reverse primer(for MF-pSB1C3-NorCB): NorCB –RP
    Conclusion: Cm2 &amp; Cm 4 positive
    Reverse prime(for MF-pSB1C3-NosZ-NorCB)r: NorCB -RP
    Conclusion: Cm1. Cm 5. Cm6 positive</P><P><STRONG>
    Friday</STRONG>SRB</P><P>      *  Enzyme check 
    -MF-pSB1C3-  CycI
    Enzyme:  XbaI + SpeI (2157+2938)
    Conclusion:  positive</P><UL><LI>NosZ+NorCB</LI></UL><P>      * Enzyme check 
    MF-pSB1C3-NosZ
    Enzyme: XbaI + SpeI (2157+2938)
    Conclusion: positive
    MF-pSB1C3-NorCB
    Enzyme: XbaI + SpeI (2157+2938)
    Conclusion: positive
    MF-pSB1C3-NosZ-NorCB
    Enzyme: XbaI + SpeI (2157+2938)
    Conclusion: positive<STRONG>Saturday</STRONG>SRB</P><P>      * Send for sequencing
    Conclusion: positive</P><UL><LI>   NosZ+NorCB</LI></UL><P>      * Send for sequencing
    Conclusion: positive
    Week 6 (8/5~8/11)<STRONG>Sunday</STRONG>SRB</P><P>      * Failure to activate dried Desufulvibrio Desulficans ( ask for  help from BCRC)</P><UL><LI>NirN+NirS</LI></UL><P>      * PCR of NirN
    Template DNA: pseudomonas aeruginosa
    Forward primer: NirN -FP (EX_ NirN _F)
    Reverse primer: NirN -RP (BamHI_ NirN _R)
    conclusion: positive
    *PCR clean up
    * PCR of NirS
    Template DNA: pseudomonas aeruginosa
    Forward primer: NirS -FP (BamHI_ NirS _F)
    Reverse primer: NirS -RP (PS_ NirS _R)
    conclusion: positive
    *PCR clean up
    *enzyme cut
    Rescrition enzyme: pSB1C3: EcoRI.SpeI
    NirN: EcoRI.BamHI
    NirS: BamHI.SpeI<STRONG>Monday</STRONG>SRB</P><P>      * sub clone CyC I gene from MfeI- XbaI-pSB1C3-SbfI-SpeI-CyC I  into pTrc-kan plasmid( GE lab constructed plasmid for Cyanobacteria transgene  technique, site #1)
    * Ptrc-kan cut with EcoRI, XbaI, #4 bfr, while  MfeI-XbaI-pSB1C3-SbfI-SpeI- CyC I cut with, MfeI, Sbf I, #4 (smart trick : MfeI  and EcoRI form scar, SbfI and XbaI form scar, in order to prevent endogenous  EcoRI and PstI site)</P><UL><LI>NirN+NirS</LI></UL><P>      * Ligatiom, transform with kan solid agar plate
    * Clean up
    * Ligation: pSB1C3-NirN-RBS-NirS 
    conclusion: positive
    * Transformation of competent cell
    -prepare Cm20 plate
    - Plate the bacteria onto plates and place  at 37oC overnight</P><P><STRONG>Tuesday</STRONG>SRB</P><P>      * Colony PCR to check whether pTrc-kan-CyC I is correct and  amplification the bacteria colony
    * acquire many colony on the plate
    -pick 8 colony on each plate(Kan1~Kan8)
    * Forward  primer: VF2
    Reverse primer: CycI-RP
    Conclusion: Kan1~ Kan 4 positive</P><UL><LI>NirN+NirS</LI></UL><P>      * acquire many colony on the plate
    -pick 8 colony on each plate(Cm1~Cm8)
    * Colony PCR
    Forward  primer: CmR gene-R-SpeI
    Reverse primer: NirS-RP
    Conclusion: Cm2~ Cm 6.Cm8 positive<STRONG>Wednesday</STRONG>SRB</P><P>      * Enzyme  check 
    -MF-pSB1C3- CycI
    Enzyme: MfeI + SpeI  (1307+2406)
    Conclusion: positive</P><UL><LI>NirN+NirS</LI></UL><P>      * Enzyme check 
    pSB1C3- NirN-NirS
    Enzyme: MfeI + SpeI  (1307+2406)
    Conclusion: positive <STRONG>Thursday</STRONG>SRB</P><P>      * Paper survey for hot spring bacteria and purple sulfur  bacteria, to double check whether they are better choices for our project to go  on? Or we should still use Sulfur reducing bacteria</P><UL><LI>NirN+NirS</LI></UL><P>      * Send for sequencing
    Conclusion: positive  </P><P><STRONG>Friday</STRONG>SRB</P><P>      * BCRC finish the bacteria type analysis about Desufulvibrio  Desulficans and ready to amplifies this SRB for us (thanks to God!!)<STRONG>Saturday</STRONG>SRB</P><P>      * Paper survey for hot spring bacteria and purple sulfur  bacteria, to double check whether they are better choices for our project to go  on? Or we should still use Sulfur reducing bacteria
    * Start to prepare for the Taiwan iGEMers’ fest <STRONG>(We are the host)</STRONG></P><P>
    Week 7 (8/12~8/18)<STRONG>Sunday</STRONG>
      * Prepare for the Taiwan iGEMers’ fest(exp delayed for three  days!!)<STRONG>Monday</STRONG>
      * Prepare for the Taiwan iGEMers’ fest(exp delayed for three  days!!)<STRONG>Tuesday</STRONG>
      * Prepare for the Taiwan iGEMers’ fest(exp delayed for three  days!!)<STRONG>Wednesday</STRONG>
      * Taiwan iGEMers’ fest(NYMU-Taipei.NTU-Taida. NCTU_Formosa)<STRONG>Thursday</STRONG>
      * Discuss the advantages and disadvantages of our project after  Taiwan iGEMers’ fest.
      * Get some rest<STRONG>Friday</STRONG>
      * Paper survey for hot spring bacteria and purple sulfur  bacteria, to double check whether they are better choices for our project to go  on? Or we should still use Sulfur reducing bacteria<STRONG>Saturday</STRONG><STRONG>      *</STRONG> Primer design for sulfite reductase in  Desufulvibrio Desulficans: DsrI, DsrII
      * Waiting for primers to come</P><P>Week 8 (8/19~8/25)</P><P><STRONG>Sunday</STRONG>SRB</P><P>      * PCR of DsrI and DsrII
    Template: genomic DNA from  Desufulvibrio  Desulficans
    For DsrI:
    Forward primer: DsrI -FP (MX_ DsrI _F)
    Reverse primer: DsrI -RP (BamHI _ DsrI _R)
    conclusion: positive
    For DsrII:
    Forward primer: DsrII -FP (BamHI_DsrI _F)
    Reverse primer: DsrII -RP (SbfI-SpeI_DsrII _R)
    *PCR clean up
    *enzyme cut
    Rescrition enzyme: MF-pSB1C3: MfeI.SpeI
    DsRI-BamHI-DsrII: MfeI.SpeI</P><UL><LI>NosZ-NorCB</LI></UL><P>      * sub clone NosZ+NorCB gene from pSB1C3-NosZ-NorCB into pTrc-Amp  plasmid( GE lab constructed plasmid for Cyanobacteria transgene technique, site  #3)
    * Ptrc-Amp cut with EcoRI, XbaI, #4 bfr, while pSB1C3-NosZ-NorCB  cut with, EcoRI, Spe I</P><P> </P><P><STRONG>Monday</STRONG>SRB</P><P>      * Clean up
    * Ligation: MF-pSB1C3-DsRI-BamHI-DsrII 
    conclusion: positive
    * Transformation of competent cell
    -prepare Cm20 plate
    - Plate the bacteria onto plates and place  at 37oC overnight</P><UL><LI>NosZ-NorCB</LI></UL><P>      * Ligatiom, transform with kan solid agar plate
    * Clean up
    * Ligation: pSB1C3-NosZ-NorCB
    conclusion: positive
    * Transformation of competent cell
    -prepare Amp20 plate
    - Plate the bacteria onto plates and place  at 37oC overnight</P><UL><LI>SQR</LI></UL><P>      * Synechococcus sp. PCC7002 was cultured in the medium A2, a  modification of Stevens et al. (1973) on a rotary shaker (100 rpm). The initial  concentration of cells is controlled to an OD730 of 0.1 in fresh medium.  Different concentration of sodium sulfide is added, and  3-(3,4-dichlorophenyl)-1,1-dimethylurea (DCMU) was put into experimental group.  Cells are grown in the six-well-plates and monitored under OD730 every 24 hour.<STRONG>Tuesday</STRONG>SRB</P><P>      * acquire many colony on the plate
    -pick 8 colony on each plate(Cm1~Cm8)
    * Colony PCR
    Forward primer: VF2
    Reverse primer: DsrII-RP
    Conclusion: Cm 3 &amp;Cm 5 positive</P><UL><LI>NosZ-NorCB</LI></UL><P>      * acquire many colony on the plate
    -pick 8 colony on each plate(Amp 1~ Amp 8)
    * Colony PCR
    Forward primer: VF2
    Reverse primer: NorCB-RP
    Conclusion: Amp 1~ Amp 5 positive</P><UL><LI>SQR</LI></UL><P>      * SQR part experiment<STRONG>Wednesday</STRONG>SRB</P><P>      * Enzyme  check 
    MF-pSB1C3- DsrI-BamHI-DsrII
    Enzyme: XbaI + SpeI  (1307+2406)
    Conclusion: positive</P><UL><LI>NosZ-NorCB</LI></UL><P>* Enzyme check 
    pSB1C3-NosZ-NorCB 
    Enzyme: XbaI + SpeI  (1307+2406)
    Conclusion: positive</P><UL><LI>SQR</LI></UL><P>      * SQR part experiment<STRONG>Thursday</STRONG>SRB</P><P>      * Send for sequencing
    Conclusion: positive</P><UL><LI>NosZ-NorCB</LI></UL><P>* Send for sequencing
    Conclusion: positive</P><UL><LI>SQR</LI></UL><P>      * SQR part experiment<STRONG>Friday</STRONG>SRB</P><P>      * Transfer Ptrc-PNSI-Kan- CyC I gene into cyanbacteria (<EM>sp.</EM> PCC 7942)</P><UL><LI>NosZ-NorCB</LI></UL><P>      * Transfer Ptrc -PNSIII-Amp- NosZ-NorCB gene into cyanbacteria  (<EM>sp.</EM> PCC 7942)</P><UL><LI>SQR</LI></UL><P>      * SQR part experiment<STRONG>Saturday</STRONG>SRB</P><P>      * Microscopy analysis of Desufulvibrio Desulficans</P><UL><LI>SQR</LI></UL><P>      * SQR part experiment</P></DIV></DIV></DIV><DIV id="ymi_right_column"><DIV class="drawers-wrapper"><UL class="drawers"><LI class="drawer"><H2 class="drawer-handle">Notebook</H2></LI><LI><A title="Week 1 ~ Week 4" href="https://2012.igem.org/Team:NYMU-Taipei/ymiw1.html">Week 1 ~ Week 4</A></LI><LI><A title="Week 5 ~ Week 8" href="https://2012.igem.org/Team:NYMU-Taipei/ymiw2.html">Week 5 ~ Week 8</A></LI><LI><A title="Week 9 ~ Week 11" href="https://2012.igem.org/Team:NYMU-Taipei/ymiw3.html">Week 9 ~ Week 11</A></LI></UL><H2 class="drawer-handle">Extras</H2><UL><LI><A title="Achievements" href="https://2012.igem.org/Team:NYMU-Taipei/ymijf.html">Achievements</A></LI><LI><A title="Safety" href="https://2012.igem.org/Team:NYMU-Taipei/ymisf.html">Safety</A></LI><LI><A title="Collaboration with NTU" href="https://2012.igem.org/Team:NYMU-Taipei/ymico1.html">Collaboration with NTU</A></LI><LI><A title="NYMU Bioenergy Breakthrough" href="https://2012.igem.org/Team:NYMU-Taipei/ymibt.html">NYMU Bioenergy
Breakthrough</A></LI><LI><A title="Parts" href="https://2012.igem.org/Team:NYMU-Taipei/ymiparts.html">Parts</A></LI></UL><LI class="drawer last"><H2 class="drawer-handle">Protocol</H2></LI><LI><A title="J774 macrophage cell line culturing" href="https://2012.igem.org/Team:NYMU-Taipei/ymip1.html">J774 macrophage cell 
                culturing</A></LI><LI><A title="Mesenchymal stem cells culturing" href="https://2012.igem.org/Team:NYMU-Taipei/ymip2.html">Mesenchymal stem cells
culturing</A></LI><LI><A title="Reprogramming of Somatic Cells into Stem &amp; Separation of iPS cells" href="https://2012.igem.org/Team:NYMU-Taipei/ymip3.html">Reprogramming of Somatic Cells into Stem &amp; Separation of iPS cells</A></LI></DIV></DIV></DIV></DIV><P> </P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:NYMU-Taipei/ymiw2.html">http://2012.igem.org/Team:NYMU-Taipei/ymiw2.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NYMU-Taipei/ymiw2.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NYMU-Taipei/ymiw2.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NYMU-Taipei/ymiw2.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NYMU-Taipei/ymiw2.html&amp;oldid=287159" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>