"
Page
Discussion
View source
History
teams
Log in
 
Team:NYMU-Taipei/ymiw3.html
From 2012.igem.org
NYMU iGEM
Project Venusian · Modeling · Human Practice · Extras · Team · iGEM
Week 9 ~ Week 11 (8/26~9/15)
Week 9 (7/29~8/4)
Sunday
SRB
      * Sub clone DsrI-BamHI-DsrII gene from pSB1C3 into Ptrc-kan  plasmid
* Ptrc-kan plasmid cut with EcoRI, XbaI
* MF-pSB1C3 -DsrI-BamHI-DsrII gene Cut  with MfeI, SpeI
*(smart trick: MfeI and EcoRI form scar,  SbfI and SpeI form scar, to prevent endogenous EcoRI and PstI site)
SQR
* Synechococcus sp. PCC7002 was cultured  in the medium A2, rotary shaker and initial cell concentration are the same as  the last experiment. DCMU is diluted with A2 medium into different  concentration, and 10 mM of sodium sulfide is added to the experimental group.  The measurement method and frequency of cell growth has been described  previously.
Monday
SRB
      * Ligation, transformation pTrc-kan-DsrI-BamHI-DsrII plasmid  into DH5alpha competent cell, cultured with Kan solid agar plate
SQR
      * SQR part experiment
Tuesday
SRB
      * colony PCR and enzyme check to pick up correct colonies
* acquire many colony on the plate
-pick 8 colony on each plate(Kan 1~ Kan 8)
* Forward  primer: VF2
Reverse primer: DsrII-RP
Conclusion: Kan 1 & Kan 6~8 positive
SQR
      * SQR part experiment
Wednesday
SRB
      * Send for sequencing
Conclusion: positive
SQR
      * SQR part experiment
Thursday
SRB
      * Function assay with pTrc-kan-CyC I gene transformed E.Coli
* Transfer pTrc-kan-DsrI-BamHI-DsrII gene into genomic DNA of  Cyanobacteria
SQR
      * SQR part experiment
Friday
SRB
      * Cyanobacteria CyC I gene stable clone established (wish we  can establish cyanobacteria DsrI-BamHI-DsrII stable clone too!!)
SQR
      * SQR part experiment
Saturday
SQR
      * SQR part experiment
Week   10 (9/2~9/8)
Sunday
NapABC
      * primer design for NapABC and new backbone MS-pSB1C3
SQR
      * With the result of previous  experiment, the concentration of both sodium sulfide and DCMU is adjusted.  Sodium sulfide is diluted from 40mM to 2.5mM, while DCMU 0.5 μM  is added into the experimental group.
Monday
* PCR construction for new pSB1C3 (we named it MS-pSB1C3,  different from pSB1C3 and MF-pSB1C3): MfeI-Xba I-pSB1C3 backbone- Pst I-Spe I
* PCR of NapABC
Template DNA: pseudomonas aeruginosa
Forward primer: NapABC -FP (MX_ NapABC _F)
Reverse primer: NapABC -RP (PS_ NapABC _R)
conclusion: positive
*PCR clean up
*enzyme cut
Rescrition enzyme: MS-pSB1C3: MfeI.SpeI
NapABC: MfeI.SpeI
SQR
      * SQR part experiment
Tuesday
NapABC
* Clean up
* Ligation: MS-pSB1C3-RBS-NapABC
conclusion: positive
* Transformation of competent cell
-prepare Cm20 plate
- Plate the bacteria onto plates and place  at 37oC overnight
SQR
      * SQR part experiment
Wednesday
NapABC
      * acquire many colony on the plate
-pick 8 colony on each plate(Cm1~Cm8)
* Colony PCR
Forward  primer: CmR gene-R-SpeI
Reverse primer: NapABC-RP
Conclusion: Cm1~ Cm 4. Cm6. Cm7 positive
SQR
      * SQR part experiment
Thursday
NapABC
      * Enzyme  check
MS-pSB1C3-RBS-NapABC
Enzyme: SacI-HF  (1307+2406)
Conclusion: positive
SQR
      * SQR part experiment
Friday
NapABC
      * Send for sequencing
Conclusion: positive
SQR
      * SQR part experiment
Saturday
SQR
      * SQR part experiment
Week   11 (9/9~9/15)
Sunday
SQR
      * The effect of sodium sulfide on Synechococcus SP. PCC 7942  growth rate
* In this experiment, we aimed to examine whether the SQR  Synechococcus SP. PCC 7942 expressed is functional. Synechococcus SP. PCC 7942  with SQR and wild type was cultivated in BG-11 medium (Allen M.M. 1968) on the  same rotary shaker described in the first experiment. Initial cell  concentration is fixed to an OD730 of 0.1 in the medium. 2.5mM of sodium  sulfide is added into the experimental group, and DCMU is diluted from 1μM to  0.5μM.
Monday
NirN-RBS-NirS-RBS-NapABC
* Sub clone NirN-RBS-NirS  gene from pSB1C3 and NapABC gene from MS-pSB1C3 into Ptrc-kan plasmid
* Ptrc-kan plasmid cut with EcoRI, XbaI
* MF-pSB1C3 -DsrI-BamHI-DsrII gene Cut  with MfeI, SpeI
*(smart trick: MfeI and EcoRI  form scar, SbfI and SpeI form scar, to prevent endogenous EcoRI and PstI site)
SQR
      * SQR part experiment
* DCMU concentration and cell growth
From the last experiment, we  found it necessary to perform another independent experiment to find the  optimal DCMU concentration of Synechococcus SP. PCC 7942. In addition, the  initial cell concentration was adjusted to an OD730 of 0.2 in BG-11 medium. DCMU  is diluted from 1μM to 0.1251μM. Except of cell concentration, all condition  remained the same for both sqr and wild type strain of Synechococcus SP. PCC  7942.
Tuesday
SQR
      * SQR part experiment
Wednesday
SQR
      * SQR part experiment
Thursday
SQR
      * SQR part experiment
Friday
SQR
      * SQR part experiment
Saturday
SQR
      * SQR part experiment
Notebook
Week 1 ~ Week 4
Week 5 ~ Week 8
Week 9 ~ Week 11
Extras
Achievements
Safety
Collaboration with NTU
NYMU Bioenergy
Breakthrough
Parts
Protocol
J774 macrophage cell
culturing
Mesenchymal stem cells
culturing
Reprogramming of Somatic Cells into Stem & Separation of iPS cells
 
Retrieved from " http://2012.igem.org/Team:NYMU-Taipei/ymiw3.html "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
