"
Page
Discussion
View source
History
teams
Log in
 
Team:NYMU-Taipei/ymiw2.html
From 2012.igem.org
NYMU iGEM
Project Venusian · Modeling · Human Practice · Extras · Team · iGEM
Week 5 ~ Week 8 (7/29~8/25)
Week 5 (7/29~8/4)
Sunday
SRB
      * Other alternatives decision: find out specific enzyme for  sulfur reducing from KEGG, paper survey
Monday
SRB
      * Paper survey (waiting for primer also)
* Design primer for NosZ and NorCB
Tuesday
SRB(Cloning ofCycI)
      * PCR construction for new pSB1C3 (we named it MF-pSB1C3):  MfeI-Xba I-pSB1C3 backbone- Sbf I-Spe I ( for tater gene which has endogenous  EcoRI, PstI cutting site)
* PCR of CyC I
Template DNA
Forward primer: CyC I -FP (MX_ SmtA _F)
Reverse primer: CyC I -RP (SbfIS_ SmtA _R)
conclusion: positive
*PCR clean up
*enzyme cut
Rescrition enzyme: pSB1C3: MfeI.SbfI
SmtA: MfeI.SbfI
NosZ+NorCB
      * PCR of NosZ
Template DNA: pseudomonas aeruginosa
Forward primer: NosZ -FP (EX_ NosZ _F)
Reverse primer: NosZ -RP (SbfIS_ NosZ _R)
conclusion: positive
*PCR clean up
*enzyme cut
Rescrition enzyme: MF-pSB1C3: EcoRI. SbfI
NosZ: EcoRI. SbfI
* PCR of NorCB
Template DNA
Forward primer: NorCB -FP (EX _ NorCB _F)
Reverse primer: NorCB -RP (SbfIS_ NorCB _R)
conclusion: positive
*PCR clean up
*enzyme cut
Rescrition enzyme: MF-pSB1C3: EcoRI. SbfI
NorCB: EcoRI. SbfI
To combine NosZ and NorCB, we  need to do another enzyme cut
*enzyme cut
Rescrition enzyme: MF-pSB1C3: EcoRI. SbfI
NosZ: EcoRI. SpeI
NorCB: XbaI. SbfI
Wednesday
SRB
      * Clean up
* Ligation: MF-pSB1C3-CycI
conclusion: positive
* Transformation of competent cell
-prepare Cm20 plate
- Plate the bacteria onto plates and place  at 37oC overnight
NosZ+NorCB
      * Clean up
* Ligation: MF-pSB1C3-NosZ
MF-pSB1C3-NorCB
MF-pSB1C3-NosZ-NorCB
conclusion: positive
* Transformation of competent cell
-prepare Cm20 plate
- Plate the bacteria onto plates and place  at 37oC overnight
Thursday
 SRB
      * acquire many colony on the plate
-pick 8 colony on each plate(Cm1~Cm8)
* Colony PCR
Forward  primer: CmR gene-R-SpeI
Reverse primer: CycI-RP
Conclusion: Cm1~ Cm 8 positive
NosZ+NorCB
      * acquire some colony( <=8 ) on the plate
- pick 8 colony on MF-pSB1C3-NosZ plate(Cm1~Cm8)
- pick 6 colony on MF-pSB1C3-NorCB plate(Cm1~Cm6)
- pick 8 colony on MF-pSB1C3-NosZ-NorCB plate(Cm1~Cm8)
* Colony PCR
Forward  primer: CmR gene-R-SpeI
Reverse primer(for MF-pSB1C3-NosZ): NosZ –RP
Conclusion: Cm1~ Cm 8 positive
Reverse primer(for MF-pSB1C3-NorCB): NorCB –RP
Conclusion: Cm2 & Cm 4 positive
Reverse prime(for MF-pSB1C3-NosZ-NorCB)r: NorCB -RP
Conclusion: Cm1. Cm 5. Cm6 positive
Friday
SRB
      *  Enzyme check
-MF-pSB1C3-  CycI
Enzyme:  XbaI + SpeI (2157+2938)
Conclusion:  positive
NosZ+NorCB
      * Enzyme check
MF-pSB1C3-NosZ
Enzyme: XbaI + SpeI (2157+2938)
Conclusion: positive
MF-pSB1C3-NorCB
Enzyme: XbaI + SpeI (2157+2938)
Conclusion: positive
MF-pSB1C3-NosZ-NorCB
Enzyme: XbaI + SpeI (2157+2938)
Conclusion: positive
Saturday
SRB
      * Send for sequencing
Conclusion: positive
   NosZ+NorCB
      * Send for sequencing
Conclusion: positive
Week 6 (8/5~8/11)
Sunday
SRB
      * Failure to activate dried Desufulvibrio Desulficans ( ask for  help from BCRC)
NirN+NirS
      * PCR of NirN
Template DNA: pseudomonas aeruginosa
Forward primer: NirN -FP (EX_ NirN _F)
Reverse primer: NirN -RP (BamHI_ NirN _R)
conclusion: positive
*PCR clean up
* PCR of NirS
Template DNA: pseudomonas aeruginosa
Forward primer: NirS -FP (BamHI_ NirS _F)
Reverse primer: NirS -RP (PS_ NirS _R)
conclusion: positive
*PCR clean up
*enzyme cut
Rescrition enzyme: pSB1C3: EcoRI.SpeI
NirN: EcoRI.BamHI
NirS: BamHI.SpeI
Monday
SRB
      * sub clone CyC I gene from MfeI- XbaI-pSB1C3-SbfI-SpeI-CyC I  into pTrc-kan plasmid( GE lab constructed plasmid for Cyanobacteria transgene  technique, site #1)
* Ptrc-kan cut with EcoRI, XbaI, #4 bfr, while  MfeI-XbaI-pSB1C3-SbfI-SpeI- CyC I cut with, MfeI, Sbf I, #4 (smart trick : MfeI  and EcoRI form scar, SbfI and XbaI form scar, in order to prevent endogenous  EcoRI and PstI site)
NirN+NirS
      * Ligatiom, transform with kan solid agar plate
* Clean up
* Ligation: pSB1C3-NirN-RBS-NirS
conclusion: positive
* Transformation of competent cell
-prepare Cm20 plate
- Plate the bacteria onto plates and place  at 37oC overnight
Tuesday
SRB
      * Colony PCR to check whether pTrc-kan-CyC I is correct and  amplification the bacteria colony
* acquire many colony on the plate
-pick 8 colony on each plate(Kan1~Kan8)
* Forward  primer: VF2
Reverse primer: CycI-RP
Conclusion: Kan1~ Kan 4 positive
NirN+NirS
      * acquire many colony on the plate
-pick 8 colony on each plate(Cm1~Cm8)
* Colony PCR
Forward  primer: CmR gene-R-SpeI
Reverse primer: NirS-RP
Conclusion: Cm2~ Cm 6.Cm8 positive
Wednesday
SRB
      * Enzyme  check
-MF-pSB1C3- CycI
Enzyme: MfeI + SpeI  (1307+2406)
Conclusion: positive
NirN+NirS
      * Enzyme check
pSB1C3- NirN-NirS
Enzyme: MfeI + SpeI  (1307+2406)
Conclusion: positive
Thursday
SRB
      * Paper survey for hot spring bacteria and purple sulfur  bacteria, to double check whether they are better choices for our project to go  on? Or we should still use Sulfur reducing bacteria
NirN+NirS
      * Send for sequencing
Conclusion: positive
Friday
SRB
      * BCRC finish the bacteria type analysis about Desufulvibrio  Desulficans and ready to amplifies this SRB for us (thanks to God!!)
Saturday
SRB
      * Paper survey for hot spring bacteria and purple sulfur  bacteria, to double check whether they are better choices for our project to go  on? Or we should still use Sulfur reducing bacteria
* Start to prepare for the Taiwan iGEMers’ fest (We are the host)
Week 7 (8/12~8/18)
Sunday
      * Prepare for the Taiwan iGEMers’ fest(exp delayed for three  days!!)
Monday
      * Prepare for the Taiwan iGEMers’ fest(exp delayed for three  days!!)
Tuesday
      * Prepare for the Taiwan iGEMers’ fest(exp delayed for three  days!!)
Wednesday
      * Taiwan iGEMers’ fest(NYMU-Taipei.NTU-Taida. NCTU_Formosa)
Thursday
      * Discuss the advantages and disadvantages of our project after  Taiwan iGEMers’ fest.
      * Get some rest
Friday
      * Paper survey for hot spring bacteria and purple sulfur  bacteria, to double check whether they are better choices for our project to go  on? Or we should still use Sulfur reducing bacteria
Saturday
      * Primer design for sulfite reductase in  Desufulvibrio Desulficans: DsrI, DsrII
      * Waiting for primers to come
Week 8 (8/19~8/25)
Sunday
SRB
      * PCR of DsrI and DsrII
Template: genomic DNA from  Desufulvibrio  Desulficans
For DsrI:
Forward primer: DsrI -FP (MX_ DsrI _F)
Reverse primer: DsrI -RP (BamHI _ DsrI _R)
conclusion: positive
For DsrII:
Forward primer: DsrII -FP (BamHI_DsrI _F)
Reverse primer: DsrII -RP (SbfI-SpeI_DsrII _R)
*PCR clean up
*enzyme cut
Rescrition enzyme: MF-pSB1C3: MfeI.SpeI
DsRI-BamHI-DsrII: MfeI.SpeI
NosZ-NorCB
      * sub clone NosZ+NorCB gene from pSB1C3-NosZ-NorCB into pTrc-Amp  plasmid( GE lab constructed plasmid for Cyanobacteria transgene technique, site  #3)
* Ptrc-Amp cut with EcoRI, XbaI, #4 bfr, while pSB1C3-NosZ-NorCB  cut with, EcoRI, Spe I
 
Monday
SRB
      * Clean up
* Ligation: MF-pSB1C3-DsRI-BamHI-DsrII
conclusion: positive
* Transformation of competent cell
-prepare Cm20 plate
- Plate the bacteria onto plates and place  at 37oC overnight
NosZ-NorCB
      * Ligatiom, transform with kan solid agar plate
* Clean up
* Ligation: pSB1C3-NosZ-NorCB
conclusion: positive
* Transformation of competent cell
-prepare Amp20 plate
- Plate the bacteria onto plates and place  at 37oC overnight
SQR
      * Synechococcus sp. PCC7002 was cultured in the medium A2, a  modification of Stevens et al. (1973) on a rotary shaker (100 rpm). The initial  concentration of cells is controlled to an OD730 of 0.1 in fresh medium.  Different concentration of sodium sulfide is added, and  3-(3,4-dichlorophenyl)-1,1-dimethylurea (DCMU) was put into experimental group.  Cells are grown in the six-well-plates and monitored under OD730 every 24 hour.
Tuesday
SRB
      * acquire many colony on the plate
-pick 8 colony on each plate(Cm1~Cm8)
* Colony PCR
Forward primer: VF2
Reverse primer: DsrII-RP
Conclusion: Cm 3 &Cm 5 positive
NosZ-NorCB
      * acquire many colony on the plate
-pick 8 colony on each plate(Amp 1~ Amp 8)
* Colony PCR
Forward primer: VF2
Reverse primer: NorCB-RP
Conclusion: Amp 1~ Amp 5 positive
SQR
      * SQR part experiment
Wednesday
SRB
      * Enzyme  check
MF-pSB1C3- DsrI-BamHI-DsrII
Enzyme: XbaI + SpeI  (1307+2406)
Conclusion: positive
NosZ-NorCB
* Enzyme check
pSB1C3-NosZ-NorCB
Enzyme: XbaI + SpeI  (1307+2406)
Conclusion: positive
SQR
      * SQR part experiment
Thursday
SRB
      * Send for sequencing
Conclusion: positive
NosZ-NorCB
* Send for sequencing
Conclusion: positive
SQR
      * SQR part experiment
Friday
SRB
      * Transfer Ptrc-PNSI-Kan- CyC I gene into cyanbacteria (sp. PCC 7942)
NosZ-NorCB
      * Transfer Ptrc -PNSIII-Amp- NosZ-NorCB gene into cyanbacteria  (sp. PCC 7942)
SQR
      * SQR part experiment
Saturday
SRB
      * Microscopy analysis of Desufulvibrio Desulficans
SQR
      * SQR part experiment
Notebook
Week 1 ~ Week 4
Week 5 ~ Week 8
Week 9 ~ Week 11
Extras
Achievements
Safety
Collaboration with NTU
NYMU Bioenergy
Breakthrough
Parts
Protocol
J774 macrophage cell
culturing
Mesenchymal stem cells
culturing
Reprogramming of Somatic Cells into Stem & Separation of iPS cells
 
Retrieved from " http://2012.igem.org/Team:NYMU-Taipei/ymiw2.html "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
