<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NYMU-Taipei_ymisf_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NYMU-Taipei/ymisf.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NYMU-Taipei/ymisf.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/ymisf.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/ymisf.html&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NYMU-Taipei/ymisf.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NYMU-Taipei/ymisf.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>NYMU iGEM</TITLE></P><DIV id="ymi_topcontainer"><DIV id="ymi_top"><A href="https://2012.igem.org/Team:NYMU-Taipei/ymiproject.html" target="_parent">Project Venusian</A> ·
    <A href="https://2012.igem.org/Team:NYMU-Taipei/ymim1" target="_parent">Modeling</A> · 
    <A href="https://2012.igem.org/Team:NYMU-Taipei/ymihpa.html">Human Practice</A> · 
    <A href="https://2012.igem.org/Team:NYMU-Taipei/ymijf.html">Extras</A> · 
    <A href="https://2012.igem.org/Team:NYMU-Taipei/ymit1">Team</A> · 
    <A href="https://2012.igem.org/Main_Page">iGEM</A></DIV></DIV><DIV id="ymi_container"><DIV id="ymi_left_column"><DIV class="text_area" align="justify"><DIV class="title">Safety</DIV><DIV align="left"><P><STRONG>
    1. Would any of your project ideas raise  safety issues in terms of researcher safety, public safety, or environmental safety</STRONG></P>

Our projects are mainly associated with  pollutent reduction and endosymbiosis. For discussing the safety issues  connected with our project, this assessment will start by listing the organisms  and biobricks we used and developed in compliance with the guideline of iGEM  Safety page correspondingly.

The following organisms are species  which we derived our biobricks from or practiced transformation and cloning.  This list includes <EM>Desulfovibrio desulfuricans</EM>, <EM>Pseudomonas  aeruginosa</EM> PAO1, <EM>Dictyostelium discoideum</EM>, <EM>Escherichia  coli</EM>, <EM>Synechococcus elongatus</EM> PCC7942, <EM>Synechocystis</EM> sp.  PCC6803 and <EM>Synechococcus</EM> sp. PCC7002. None of the listed  organisms is a pathogen. As for the biobricks we used, most of them are  bioparts from previous iGEM teams. These parts from iGEM kits are considered  safe.

        Since our project is related to  contaminant water, exhaust air and heavy metal in environments, we need to  thoroughly examine every process of the experiments we conduct. Under the  strict guidance of our faculty instructors, all transmissions and operations of  biological materials meet the relevant laws and regulations. As a result of the  heartily suggestion and assistance of our laboratory fellows, we developed a  series of experiments to demonstrate our project without violating any national  regulations or university guideline requirements. The details of our  experiments are available on the Experiment page.
</DIV><DIV align="left"><STRONG>2. </STRONG><STRONG>Do any of the new BioBrick parts (or devices)  that you made this year raise safety issues? If yes,</STRONG><LI>Did you document these issues in the  Registry?</LI><LI>How did you manage to handle the safety  issue?</LI><LI>How could other teams learn from your  experience?</LI>
There are not any new BioBricks we made raise safety issue. They are mainly reductases from organisms mentioned above. Nitrate, nitrite, nitrous oxides reductase and derived from Pseudomonas aeruginosa PAO1. Thanks to Prof. Hwan-You Chang from National Tsing Hua University, we operated the transferring and cloning on purified genomic DNA provided from his lab. Therefore, we did not need to cultivate P. aeruginosa PAO1. The biobricks related to nitrogen reductase are ultimately transfected onto the genome of Synechococcus sp. PCC7942, and all of the according procedures followed the strict regulations of BSL 1 (Basic Biosafety Level 1). The same operation standard (BSL 1) is also applied to our experiments of the sulfate reducing enzymes derived from P. aeruginosa PAO1 as well. In addition, the sqr (sulfide quinone reductase) from Synechococcus sp. 7002, and like other biobricks mentioned above, is expressed in Synechococcus PCC7942 following the same safety requirements too.


        The other new biobricks we made includes zinTp, the promoter whose activity is induced by cadmium ion; and mnt protein, the ion transporter that pumps cadmium ions inside bacteria. These genes have no harm to either human being or the environment.


        Since there is little safety concern according to the properties of our new biobricks, we deduced that the new biobricks we submitted are safe and harmless.<P> </P></DIV><DIV align="left"><P><STRONG> 3. Is there a  local biosafety group, committee, or review board at your institution?</STRONG></P><LI>If yes, what does your local biosafety  group think about your project?</LI><LI>If no, which specific biosafety rules or  guidelines do you have to consider in your country?</LI><P> </P>The Center of Environmental Protection  and Safety and Health on our campus is in charge of promoting and managing the  safety and health issue in National Yang-Ming University. The following links  will direct you to the website which contains regulation guidelines of  biological safety issues. <A href="http://ces-e.web.ym.edu.tw/front/bin/ptlist.phtml?Category=5">The Center of  Environmental Protection and Safety and Health - Biological Safety</A>.  All the experiments we do or conduct in our projects are following the rules  listed in there.
   <P> </P>Though most of our projects are  considerably safe, one part of our projects involves in the manipulation of  cadmium ions, which is restricted under the local regulations. With the help  and advice of the Center of Environmental Protection and Safety and Health, we  conducted our cadmium related experiments in accordance with the associated  guidelines.
    <P> </P></DIV><DIV align="left"><P><STRONG>4. Do you have any other ideas how to deal  with safety issues that could be useful for future iGEM competitions? How could  parts, devices and systems be made even safer through biosafety engineering?</STRONG></P><P> </P>One of our projects specifically deals with biosafety issues related to the environment. Our Soil Cleaner project is mainly about collecting Cadmium from the environment, containing several interesting designs to avoid gene flow and limit the leakage of artificial organisms into the environment. We took advantage of using previous biobricks to make sure the engineered E. coli can only live inside of Dictyostelium discoideum, which avoids unwanted gene exchange between our engineered E. coli and other wild type bacteria.
           <P> </P>By cloning LLO and invasin genes into the E. coli cells, the bacteria can get into the amoeba and live inside there. Invasin helps the bacterial cells to be endocytosed by the amoeba host. Then Listeriolysin O makes the bacterial cells escaped from phagosomes in the cytoplasm of the amoeba host.
         
          <P> </P>In case that the E. coli cells get rid of amoeba, we designed a mechanism by using endolysin and holin to cause the death of bacterial cells. Endolysin is an enzyme that degrades the bacterial peptidoglycan cell wall, resulting in the lysis of the bacterial cells. Holins are small membrane proteins that let endolysin get to where it works. Anti-holin acts as anti-toxin by binding to holin, inhibiting it activity. We also used the regulation of a LuxR promoter, a promoter which will only express downstream genes when the protein LuxR and signal molecule AHL produced by protein LuxI, are both exist inside the bacteria, in this design. Engineering continual expression endolysin, holin and LuxR; and LuxR promoter as the promoter for anti-holin gene in E. coli, the bacteria will not survive without AHL. We clone the LuxI gene to the plasmid in Dictyostelium discoideum, making the amoeba to produce AHL. By this way, when E. coli stay in the amoeba, the AHL will be sensed by LuxR, maintaining the transcription of anti-holin gene, preventing the E. coli from lysis. But when the E. coli cells leave the amoeba, there will be no AHL so no anti-holin will be produced and then the E. coli will be killed by endolysin.  <P> </P>
         As for constraining the Soil Cleaner in the contaminated area, a death switch is designed to restrict amoeba from migrating into nature. This kill-switch has two man-made operons. The first one includes zinTp promoter. This promoter activity is induce by cadmium ion; and its constructive gene is lacI. The other one includes BBa_R0010 promoter. This promoter whose activity is inhibited by the protein lacI; and the constructive genes are aifA and HlyA. AifA is a protein that causes the apoptosis of Dictyostelium discoideum cells. HlyA is a tag that sticks after the target protein. With this tag, E. coli will secrete the target protein. When cadmium ions exist, zinTp works and lacI is generated, so the expression of both aifA and HlyA is repressed. If there is no cadmium ions, BBa_R0010 will activate the expression of aifA and HlyA, so the protein aifA will be secreted right into the cytoplasm of amoeba, causing amoeba apoptosis.  <P> </P>
        Moreover, instead of using traditional endogenous plasmids to transform cyanobacteria, we cloned our biobricks into vectors that will stably recombine with homologous genomic DNA alternatively. Thus, horizontal gene transfer is avoided and the security and safety of our project is greatly improved.
 		  </DIV></DIV></DIV><DIV id="ymi_right_column"><DIV class="drawers-wrapper"><UL class="drawers"><LI class="drawer"><H2 class="drawer-handle">Extras</H2></LI><LI><A title="Achievements" href="https://2012.igem.org/Team:NYMU-Taipei/ymijf.html">Achievements</A></LI><LI><A title="Safety" href="https://2012.igem.org/Team:NYMU-Taipei/ymisf.html">Safety</A></LI><LI><A title="Collaboration with NTU" href="https://2012.igem.org/Team:NYMU-Taipei/ymico1.html">Collaboration with NTU</A></LI><LI><A title="NYMU Bioenergy Breakthrough" href="https://2012.igem.org/Team:NYMU-Taipei/ymibt.html">NYMU Bioenergy
Breakthrough</A></LI><LI><A title="Parts" href="https://2012.igem.org/Team:NYMU-Taipei/ymiparts.html">Parts</A></LI></UL><LI class="drawer"><H2 class="drawer-handle">Protocol</H2></LI><LI><A title="J774 macrophage cell line culturing" href="https://2012.igem.org/Team:NYMU-Taipei/ymip1.html">J774 macrophage cell line culturing</A></LI><LI><A title="Mesenchymal stem cells culturing" href="https://2012.igem.org/Team:NYMU-Taipei/ymip2.html">Mesenchymal stem cells culturing</A></LI><LI><A title="Reprogramming of Somatic Cells into Stem &amp; Separation of iPS cells" href="https://2012.igem.org/Team:NYMU-Taipei/ymip3.html">Reprogramming of Somatic Cells into Stem &amp; Separation of iPS cells</A></LI><LI class="drawer last"><H2 class="drawer-handle">Notebook</H2></LI><LI><A title="Week 1 ~ Week 4" href="https://2012.igem.org/Team:NYMU-Taipei/ymiw1.html">Week 1 ~ Week 4</A></LI><LI><A title="Week 5 ~ Week 8" href="https://2012.igem.org/Team:NYMU-Taipei/ymiw2.html">Week 5 ~ Week 8</A></LI><LI><A title="Week 9 ~ Week 11" href="https://2012.igem.org/Team:NYMU-Taipei/ymiw3.html">Week 9 ~ Week 11</A></LI></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:NYMU-Taipei/ymisf.html">http://2012.igem.org/Team:NYMU-Taipei/ymisf.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NYMU-Taipei/ymisf.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NYMU-Taipei/ymisf.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NYMU-Taipei/ymisf.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NYMU-Taipei/ymisf.html&amp;oldid=287138" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>