<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hong_Kong-CUHK_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hong Kong-CUHK/Design</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer" style="overflow: hidden"><DIV id="menu"><UL><LI class=""><A id="current" href="https://2015.igem.org/Team:Hong_Kong-CUHK">HOME|</A></LI><LI class=""><A href="#" onmouseover="MM_effectBlind('hidel1', 500, '0%', '100%', false)">ABOUT|</A></LI><LI class=""><A href="https://2015.igem.org/Team:Hong_Kong-CUHK/Attributions">Our Team</A></LI><LI class=""><A href="https://2015.igem.org/Team:Hong_Kong-CUHK/Sponsor">Attributions</A></LI></UL><LI class=""><A href="#" onmouseover="MM_effectBlind('hidel2', 500, '0%', '100%', false)">PROJECT|</A></LI><LI><A href="https://2015.igem.org/Team:Hong_Kong-CUHK/Team">Magnetosome</A></LI><LI><A href="https://2015.igem.org/Team:Hong_Kong-CUHK/insertion_kit">The insertion kit</A></LI><LI class=""><A href="https://2015.igem.org/Team:Hong_Kong-CUHK/methods">Experiments &amp; Methods</A></LI><LI class=""><A href="https://2015.igem.org/Team:Hong_Kong-CUHK/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Hong_Kong-CUHK/Description">Part Improvement</A></LI><LI><A href="https://2015.igem.org/Team:Hong_Kong-CUHK/Design">Application</A></LI><LI><A href="https://2015.igem.org/Team:Hong_Kong-CUHK/Modeling">Modeling</A></LI></DIV><LI class=""><A href="https://2015.igem.org/Team:Hong_Kong-CUHK/Parts">PARTS|</A></LI><LI class=""><A href="https://2015.igem.org/Team:Hong_Kong-CUHK/Notebook" onmouseover="MM_effectBlind('hidel4', 500, '0%', '100%', false)">NOTEBOOK|</A></LI><LI><A href="https://2015.igem.org/Team:Hong_Kong-CUHK/Notebook">Wet Lab Journal</A></LI><LI><A href="https://2015.igem.org/Team:Hong_Kong-CUHK/Experiments">Protocols</A></LI><LI class=""><A href="https://2015.igem.org/Team:Hong_Kong-CUHK/Safety">SAFETY|</A></LI><LI class=""><A href="https://2015.igem.org/Team:Hong_Kong-CUHK/Practices" onmouseover="MM_effectBlind('hidel5', 500, '0%', '100%', false)">POLICY &amp; PRACTICE|</A></LI><LI><A href="https://2015.igem.org/Team:Hong_Kong-CUHK/Questionnaire">Questionnaire</A></LI><LI><A href="https://2015.igem.org/Team:Hong_Kong-CUHK/iQuestionnaire">International Questionnaire</A></LI><LI><A href="https://2015.igem.org/Team:Hong_Kong-CUHK/Collaborations">COLLABORATIONS|</A></LI><LI><A href="https://igem.org/Main_Page">iGEM</A></LI></DIV><DIV id="contentContainer" width="80%"><CENTER><DIV style="text-align:justify; text-justify:inter-ideograph; width:800px;"><H1>Insertion Kit for Protein Expression on Magnetosome Membrane</H1><CENTER><P style="font-size:12px"><B>Figure 1:</B> The structure of the transmembrane mamC protein</P></CENTER><P>This simple construct consists of<I> mamC</I> gene, a gene coding for a transmembrane protein (1) (Figure 1) on the magnetosome membrane. Unlike usual recombinant methods to put our insert between multiple restriction sites, we put <I>mamC</I> in front of them. Now any protein we desired can be attached onto the magnetosome membrane just by fusing it with the <I>mamC</I> gene <B>by inserting it between the multiple restriction sites</B>.</P><P>(For your interest, this is done by removing the stop codon of<I> mamC</I> and the start codon of the desired protein, for example an antibody, making it a mamC-fused protein). </P><P>As we are now putting multiple restriction sites downstream of <I>mamC</I>, therefore we can insert any desire genes afterwards. Thus, we name it our insertion kit.</P><CENTER><P style="font-size:12px"><B>Figure 2:</B> The vector map of the transmembrane mamC protein</P></CENTER><CENTER><H1><B>Magnetosome + Insertion Kit = Multi-application !!!</B></H1></CENTER><H1>Applications</H1><H2><B>(1) Protein Extraction</B></H2><TABLE style="border:0px"><TBODY><TR><TD style="border: 0px"><DIV class="photoLeft" style="width:350px"><P align="left" style="font-size:12px"><B>Figure 3:</B> Protein Extraction Kit - Specific antibodies are fused to the C-terminus of mamC on magnetosome membrane </P></DIV><P>One of the first applications with our magnetosome and insertion kit is protein extraction (Figure 3). Protein extraction (or <B>protein purification</B>) is a series of processes intended to isolate one or a few proteins of interest from a protein mixture [2].  </P><P><FONT color="#0000ff">Among a myriad of extraction/purification techniques, affinity ligand techniques represent currently the most powerful tool available to downstream processing in terms of their selectivity and recovery. Though standard liquid chromatography is currently the most often used technique for isolation and purification of target proteins and peptides, one of the biggest disadvantages is that it is not capable to cope with samples containing particulate material. This make early stages of purification process, where suspended solid and fouling component are found in sample, unmanageable. Besides, magnetic separation is usually comparatively gentle to the target proteins or peptides. This avoids the problem of having larger protein complex to be broken up by traditional column chromatography techniques. [3] </FONT></P></TD></TR><TR><TD style="border: 0px"><P style="margin-bottom: 1.5em">Using magnetic separation is great, but why not chemically produced magnetic beads? And here are the reasons to synthesize them biologically: </P></TD></TR><TR><TD style="border: 0px"><DIV class="photoLeft" style="width:250px"><P align="left" padding="10" style="font-size:12px"><B>Figure 4:</B> Lower efficiency of binding due to non-specific orientation of antibodies by chemical methods</P></DIV><P>The magnetosome we produce has a smaller size (30 - 120 nm) than that of traditional magnetic beads (1 - 4.5 μm), so magnetosome with antibodies could have a higher binding efficiency due to a larger surface area-to-volume ratio [4]. Besides, using our biological construct design, we can fuse any protein of interest to the transmembrane protein of magnetosome (mamC) [5], for example antibodies, then we can isolate the antibody-antigen complex by magnetic force. Chemical methods currently can attach antibody to synthesized magnetic beads, however, the orientation attachment is non-specific. Not only causing a large decrease in the availability of binding sites, this also leads to easier detachment of the antibody from the magnetosome compared to our biological method (Figure 4). </P><DIV class="photoRight" width="500px"><P align="right" style="font-size:12px"><B>Figure 5:</B> The binding of GFP to GFP nanobody onto magnetosome membrane</P></DIV><P>To test our application, we chose green fluorescent protein (GFP) as our target protein, as it is easy to be detected with its fluorescence for characterization (Figure 5). To capture GFP, we first fuse the GFP nanobody onto the magnetosome membrane. This nanobody has a high affinity binding towards GFP (dissociation constant = 1.4 nM [6]) so we can easily extract GFP or GFP-tagged protein by magnetic force.</P></TD></TR><TR><TD style="border: 0px"><H2><B>(2) Water Treatment</B></H2><DIV class="photoLeft" style="width:420px"><P align="left" style="font-size:12px"><B>Figure 6:</B> Heavy metal binding peptide are fused to mamC proteins on the magnetosome membrane. The magnetosomes are captured by magnetic bars while water with heavy metals flows through.</P></DIV><P>Heavy metal is one of the major components in marine pollution due to industrial wastes, vehicle emissions, lead-acid batteries, aging water supply infrastructure and more. Different kinds of heavy metal ions, such as Pb, Cu and Ni, are found in the polluted water system [7]. Moreover, owing to the lead-in-water scandal happened recently in Hong Kong [8], we decided to use our engineered magnetite help tackle the problem as well. The principle of doing so is similar to that of protein extraction. However, instead of fusing an antibody with mamC [5], metal binding peptide will be fused instead. The protein sequences for specific metal binding peptides of different metals are known. By expressing different heavy metal binding peptides onto magnetic beads, heavy metal ions could then be captured and be easily removed from water by magnet. The heavy metal can then be released from the binding peptide using EDTA solution so that magnet beads can be reused again (Figure 6). To test the efficiency of our application, we decided to test the Pb-binding peptide first. From one of the research papers, a high affinity lead-binding peptide TNTLSNN was found with a maximum adsorption loading (q<SUB>max</SUB>) of lead is 526 μmol/g dry cell weight [9]. </P></TD></TR><TR><TD style="border: 0px"><H2><B>(3) Microbial Fuel Cell</B></H2><P><FONT color="#cc11ff">The design of our microbial fuel cell involves a simple construct inside the bacteria <I>Azotobacter vinelandii</I> which consists of <B>(1) hydrogenase</B> and <B>(2) OprF porin proteins</B>.</FONT></P><P>  
The expression of the <I>hoxKGZ</I> genes would produce <B>membrane-bound hydrogenase</B> which converts hydrogen to protons and electrons inside the bacteria (H<SUB>2</SUB>→ 2 H<SUP>+</SUP> + 2 e<SUP>−</SUP>) [10]. </P><P><FONT color="#cc11ff">The electron produced will then transport along the periplasmic space and be picked up by the electron acceptor, e.g. methylene blue. After electron binding, the reduced electron acceptor will be transported out of the cell through OprF porin [11] and possibly other membrane channels to the anode chamber. With OprF porins overexpressed to the bacteria, the reduced electron acceptors can be exported more effectively. And if possible, with mutated OprF porins which have a higher open probability overexpressed [12], it could further increase the efficiency of electron acceptors diffusing in and out of the cell. In other words, more electrons could reach the anode chamber at a higher rate.  </FONT></P><DIV class="photoRight" width="500px"><P align="right" style="font-size:12px"><B>Figure 7:</B> Model of our microbial fuel cell</P></DIV><P><FONT color="#cc11ff">After the reduced electron carriers escape from the bacterial cell, the electrons will then stick to the anode and this release of an electron will free the electron acceptor. The oxidized electron acceptor will be transported back to the periplasmic space of the bacteria to pick up another electron and so on. The electrons then pass through the circuit to the cathode, and reduce the oxidant (oxygen) in the cathode chamber. Through the continuous reduction and oxidation processes, an electric current will be generated for our microbial fuel cell.</FONT></P><P>Furthermore, this is also where our magnetosome comes into play. Together with our magnetosome expressed in the microbial fuel cell bacteria, <I>Azotobacter vinelandii</I>. The bacteria can be brought to the electrode with much closer physical contact rather than randomly dispersed within the culture solution. With a shorter diffusion distance, the diffusion rate for the electron to the electrode can be greatly increase. Ultimately, the application of magnetosome can contribute in the improvement of efficiency of existing microbial fuel cells.</P><H2>
References
</H2><P>
1. XU, Jun, et al. Surface expression of protein A on magnetosomes and capture of pathogenic bacteria by magnetosome/antibody complexes. <I>Frontiers in microbiology</I>, 2014, 5.
</P><P>
2. AHMED, Hafiz. Principles and reactions of protein extraction, purification, and characterization. <I>CRC Press</I>, 2004.
</P><P>
3. SAFARIK, Ivo; SAFARIKOVA, Mirka. Magnetic techniques for the isolation and purification of proteins and peptides. <I>BioMagnetic Research and Technology</I>, 2004, 2.1: 7.
</P><P>
4. THURBER, Greg M.; WITTRUP, K. Dane. A mechanistic compartmental model for total antibody uptake in tumors. <I>Journal of theoretical biology</I>, 2012, 314: 57-68.
</P><P>
5. XU, Jun, et al. Surface expression of protein A on magnetosomes and capture of pathogenic bacteria by magnetosome/antibody complexes. <I>Frontiers in microbiology</I>, 2014, 5. 
</P><P>
6. KUBALA, Marta H., et al. Structural and thermodynamic analysis of the GFP: GFP‐nanobody complex. <I>Protein science</I>, 2010, 19.12: 2389-2401.
</P><P>
7. JÄRUP, Lars. Hazards of heavy metal contamination. <I>British medical bulletin</I>, 2003, 68.1: 167-182.
</P><P>
8. Gloria Chan. Lead 80 times the safe limit found in water at Hong Kong public housing estate where scandal broke. <I>South China Morning Post</I>, 2015.
</P><P>
9. NGUYEN, Thuong TL, et al. Selective lead adsorption by recombinant <I>Escherichia coli</I> displaying a lead-binding peptide. <I>Applied biochemistry and biotechnology</I>, 2013, 169.4: 1188-1196.
</P><P>
10. SAYAVEDRA-SOTO, LUIS A.; ARP, DANIEL J. The <I>hoxZ</I> gene of the Azotobacter vinelandii hydrogenase operon is required for activation of hydrogenase. <I>Journal of bacteriology</I>, 1992, 174.16: 5295-5301.
</P><P>
11. YONG, Yang‐Chun, et al. Enhancement of extracellular electron transfer and bioelectricity output by synthetic porin. <I>Biotechnology and bioengineering</I>, 2013, 110.2: 408-416.
</P><P>
12. SUGAWARA, Etsuko; NAGANO, Keiji; NIKAIDO, Hiroshi. Factors affecting the folding of Pseudomonas aeruginosa OprF porin into the one-domain open conformer. <I>MBio</I>, 2010, 1.4: e00228-10. 
</P></TD></TR></TBODY></TABLE></DIV></CENTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>