Team:Hong Kong-CUHK/Description
HOME |
ABOUT |
Our Team
Attributions
PROJECT |
Magnetosome
The insertion kit
Experiments & Methods
Results
Part Improvement
Application
Modeling
PARTS |
NOTEBOOK |
Wet Lab Journal
Protocols
SAFETY |
POLICY & PRACTICE |
Questionnaire
International Questionnaire
COLLABORATIONS |
iGEM
Part Improvement
Background
Magnetosomes, an organelle encapsulating magnetic iron crystal, can be applied in many aspects. One of these applications is to construct a more efficient microbial fuel cell (MFC). MFC is a device which uses electrons produced by microorganism to generate electricity. If we genetically modify the bacteria Azotobacter vinelandii to have magnetosomes, magnetosomes inside them would be attracted towards the electrodes by magnetic force and in the process, bringing the whole bacteria along with it. As a result, the physical distance between the bacteria and electrodes will be decreased, thus an increase in the efficiency of the MFC as the diffusion rate for the electron to the electrode can be greatly increased.
Additionally, in the review of the previous iGEM teams, the idea of constructing an MFC has been popular. For example, the iGEM 2013 Bielefeld-Germany team also constructed an MFC. After a brief study of their project, we understood that one of their components is the oprF gene ( K1172501 ). The team has claimed that oprF, an outer membrane porin, could increase the efficiency of MFC by allowing electron shuttle-mediated extracellular electron transfer from bacteria to electrodes.
Investigation on K1172501
However, after studying carefully, we found that the translated sequence of K1172501 contains premature stop codons. After translation, the sequence of K1172501 provided by the Bielefeld-Germany team will not be able to translate into an oprF porin protein. As the DNA sequence of K1172501 is greatly different from oprF DNA sequence from Pseudomonas fluorescens, the bacteria Germany team claimed to obtain oprF gene sequence from.
OprF in Azotobacter vinelandii
We found that OprF exists on the outer membrane of A. vinelandii, the bacteria we have been working on. Therefore we chose it to provide an alternative OprF. The sequence provided by A. vinelandii can be completely translated to form OprF with no stop codon appearing in the gene except in the last residue. Here we provide the biobrick, K1648045 and we are planning to provide K1648047 for insertion with different promoters.
Figure 1: The photo of 1% agarose gel electrophoresis. L: DNA ladder. Lane 1: PCR product of oprF encoding from Azotobacter vinelandii strain DJ genome.
Figure 2: Checking of recombinant plasmid using double digestion. L: DNA ladder. Lane 1-3: Recombination Template for pSB1C3-oprF ( K1648045 ) with double digestion cut at EcoRI and PstI sites; with single digestion at PstI site; without digestion.
Mutated oprF with Higher Efficiency
Furthermore, to construct a more efficient MFC, a mutated OprF with 5-point mutations is utilized. According to a paper concerning the factors affecting the conformation of OprF, we found that mutations on all 4 Cys to Ser residues, and Lys to Gly residues at 189th position (K189G; C201S; C210S; C216S; C230S) of A. vinelandii oprF would have higher probability in open-channel conformation 5 times more than WT oprF [2]. With the introduction of this mutated OprF into the bacteria, it is expected that the electron carrier diffusion into or out of the bacteria, as well as the efficiency of MFC, would be increased by 5 fold. Knowing that E. coli is capable to form porin using plasmid DNA [1], we used it to carry out the investigation on the oprF efficiency compare to K1172501 , oprF from A. vinelandii and mutated OprF ( K1648046 ).
Characterization of Different oprF
For comparison, identical promoter, J13002 (a constitutive promoter), was added before the gene in pSB1C3 for constitutive expression of different oprF in bacteria. There are K1648048 for oprF from A. vinelandii, K1648049 for mutated oprF from A. vinelandii and K1648050 for K1172501 from Germany iGEM team.
Figure 3: Checking of recombinant plasmid using double digestion. L: DNA ladder. Lane 1-3: Recombination Template for J13002-oprF ( K1648048 ) with double digestion cut at EcoRI and PstI sites; with single digestion at PstI site; without digestion.
Figure 4: Checking of recombinant plasmid using double digestion. L: DNA ladder. Lane 1-2: Recombination Template for R0040-oprF* ( K1648049 ) with double digestion cut at EcoRI and PstI sites; with single digestion at PstI site.
Experiment Set-up and Ongoing Test
In the experiment, we will use the colour change of methylene blue as an indicator to compare the efficiency between the transformed bacteria with different oprF plasmids and wild type bacteria. The desired function of the oprF porin protein for this experiment is to allow the diffusion of (reduced) electron carriers in and out of the periplasmic membrane from the outside of the cell. As the electron carrier (e.g. NAD+) picks up an electron in the periplasmic space (i.e. being reduced to NADH) and diffuse out of the cell through the porin protein, the electron on the NADH will transfer to methylene blue (the mediator solution outside the cell). When the methylene blue is reduced to form leucomethylene blue, it turns from blue to colourless. Hence, the rate of transmission of electron carrier is calculated by the rate of reduction of methylene blue. The experiment is planned to carry out soon. The plasmids will also be transformed into A. vinelandii for the construction of our MFC.
References
1. Sugawara, Etsuko, Keiji Nagano, and Hiroshi Nikaido. "Factors affecting the folding of Pseudomonas aeruginosa OprFporin into the one-domain open conformer." MBio 1.4 (2010): e00228-10.
2. Yong, Yang‚ÄêChun, et al. "Enhancement of extracellular electron transfer and bioelectricity output by synthetic porin." Biotechnology and bioengineering110.2 (2013): 408-416.
