<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Leiden_Collaborations skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Leiden/Collaborations</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="no-js" id="loader-wrapper"><DIV id="BestOnLargeScreens">Although the mobile version has the same content, this wiki is best viewed on larger screens </DIV></DIV><DIV class="mainContent"><DIV class="container-text"><H1 class="SectionTitle">Collaborations</H1><P>During the iGEM competition we have prioritised building meaningful relationships and collaborations with other teams.
        Through them we learned more about other projects, built friendships, experienced the diversity which is inherent to the competition and were able to further improve our own project.
        Below are the highlights of our endeavors. </P></DIV><SECTION class="popout chapter"><DIV class="collapsible"> Determining HMF stress responses for iGEM Bordeaux</DIV><DIV class="collapsibleContent"><P><A href="https://2018.igem.org/Team:Bordeaux" class="colouredlink" target="_blank">iGEM team Bordeaux</A> saw an opportunity to collaborate with us when they found out through our social media coverage that our project involved the
          detection of bacterial
          cell stress.
          iGEM Bordeauxs project is about producing a bacterial strain that transforms hydroxymethylfurfural (HMF)
          to furandicarboxylic acid (FDCA).
          <SPAN class="blobs"><A href="https://2018.igem.org/Team:Bordeaux" class="popoutBlob" target="_blank">iGEM Bordeaux uses HMF in their project as an intermediate
              to create FDCA</A><SPAN>HMF
            </SPAN></SPAN> is a byproduct of the degradation of cellulose or other polysaccharides and is known to be toxic to bacteria. FCDA is a platform molecule that can be polymerized in plastics or biofuels.
          Because HMF is a known toxin, iGEM Bordeaux suggested it would be interesting for us to test this on our bacterial stress detection system, and we wholeheartedly agreed.
          Following a skype session to discuss the details of our collaboration, Bordeaux send us 2mL of solution containing 5 g/L HMF. This allowed us to test the compound at various concentrations on four of our stress detection strains: <SPAN class="blobs"><A href="http://parts.igem.org/Part:BBa_K2610019" class="popoutBlob" target="_blank">DnaK is involved in
              the process of chromosomal DNA replication<SUP>[1][2]</SUP>. Read more about pDnaK on its iGEM registry page</A><SPAN>pDnaK</SPAN></SPAN>, <SPAN class="blobs"><A href="http://parts.igem.org/Part:BBa_K2610021" class="popoutBlob" target="_blank">RpoE
              has been shown to seemingly be involved in the exocytoplasmic stress response. Read more about pRpoE on its iGEM registry page</A><SPAN>pRpoE</SPAN></SPAN>, <SPAN class="blobs"><A href="http://parts.igem.org/Part:BBa_K2610031" class="popoutBlob" target="_blank">pSoxS has been found to be involved in the superoxide pathway and responds to DNA damage. Read more about pSoxS on its iGEM registry page</A><SPAN>pSoxS</SPAN></SPAN>
          and <SPAN class="blobs"><A href="http://parts.igem.org/Part:BBa_K2610036" class="popoutBlob" target="_blank">CspA is involved in cold-shock stress pathways. Furthermore, it is known that the transcription of the CspA-promoter can be
              abolished almost
              completely by the use of rifampicin or nalidixic acid<SUP>[3]</SUP>. Read more about pCspA on its iGEM registry
              page</A><SPAN>pCspA</SPAN></SPAN>.
          Results of exposure to HMF show no stress reaction for the pCspA-GFP (Figure 1A) and pRpoE-GFP (Figure 1C) strains. CspA is involved in metabolism related stress and RpoE is involved in protein synthesis stress.
          Because both strains did not invoke a stress reaction, there is no indication that HMF delivers stress in these pathways. However, to firmly conclude these stresses are not being experienced, more extensive testing is needed, since these
          are just two promoters involved in these stress pathways.
          Both the pDnaK-GFP (Figure 1B) and pSoxS-GFP strains (Figure 1D) do show some response when exposed to 0.5 μg/mL HMF. DnaK is involved in the process of chromosomal DNA replication and SoxS is involved in DNA damage repair.
          Therefore, these results indicate that HMF delivers stress to bacteria by damaging their DNA and disturbing their replication process. However, the stress being experienced at these HMF concentrations seems to be low.</P><FIGURE class="resultfig wide"><DIV class="FigFlexbox"><FIGCAPTION><B>Figure 1. pDnaK-GFP and pSoxS-GFP show a slight stress response to HMF </B>
              Four stress detecting strains were exposed to multiple concentrations of HMF. Figure 1A shows little to no response of the pCspA-strain to all 5 concentrations of HMF. Figure 1B shows a reaction of the pDnaK-strain when exposed to 0.5
     μg/mL HMF. Figure 1C shows no reaction of the pRpoE-strain for all 5 HMF concentrations. Figure 1D shows a reaction of the pSoxS-strain when exposed to 0.5 μg/mL HMF.
            </FIGCAPTION></DIV></FIGURE><P>
          We have shared these results with iGEM Bordeaux. Hopefully helping them further with their project, providing them with insight into the effect of HMF on bacterial cell processes. We are very thankful for this meaningful collaboration and
          look forward to meeting their team again in Boston.</P><DIV class="smaller refs"><H3>References</H3><P id="Ref1">[1]: EMBL-EBI, I. (n.d.). InterPro. Chaperone DnaK. Retrieved October 10, 2018, from <A href="http://www.ebi.ac.uk/interpro/entry/IPR012725"> interpro</A></P><P id="Ref2">[2]: European Bioinformatics Institute, Protein Information Resource, Swiss Institute of Bioinformatics (SIB). (2018, June 20). Chaperone protein DnaK. Retrieved October 10, 2018 from <A href="https://www.uniprot.org/uniprot/P0A6Y8">SIB</A></P><P id="Ref3">[3]: Yamanaka K, Inouye M. (2006 Apr 6). Induction of CspA, an <I>E. coli</I> major cold-shock protein, upon nutritional upshift at 37 degrees C. <I>Genes Cells</I>, (4), 279-290.
          </P></DIV></DIV></SECTION><SECTION class="popout chapter"><DIV class="collapsible">Validating stress in mammalian and bacterial cells with iGEM UC Davis</DIV><DIV class="collapsibleContent"><P>We contacted <A href="https://2018.igem.org/Team:UC_Davis" class="colouredlink" target="_blank">iGEM team UC Davis</A> via twitter after we found out that our projects were both involving similar ideas. While our project focuses on finding
          bacterial cell
          stress, iGEM UC Davis is focusing on detecting mammalian cell stress.
          To discuss our approaches, problems within the lab, human practices and other aspects of our projects we set up several skype meetings and maintained extensive email contact. While keeping in touch during the summer, we eventually agreed
          on a collaboration:
          Leiden would test the toxicity of H<SUB>2</SUB>O<SUB>2</SUB> on bacterial cells and UC Davis would test the toxicity of nalidixic acid on mammalian cells. To ensure our exchanged results would be beneficial for each other, we discussed the
          exact protocols
          beforehand and fine tuned them to each others wishes.
          We first proved the potential of our system using a pSoxS-GFP construct, containing a promoter of the <I>E. coli</I> DNA-damage pathway, and nalidixic acid, a known DNA damaging antibiotic. Therefore, we requested UC Davis to test multiple
          concentrations of nalidixic acid on their mammalian cell lines. With these experiments it would be possible to determine whether this antibiotic only stresses the bacterial cells, or also the mammalian cells. In the future this could be a
          very important follow up test of the screening method developed by us. This way, potential antibiotic candidates can quickly be tested to ensure they do not stress mammalian cells.
          
          The results of UC Davis’ experiment are shown in figure 2. Unfortunately, no dose-dependency was observed and from this data alone we cannot conclude that nalidixic acid induces a stress response in mammalian cells <SPAN class="blobs"><A href="https://2018.igem.org/Team:UC_Davis" class="popoutBlob" target="_blank">For more information view UC Davis' wiki</A><SPAN> under these conditions. </SPAN></SPAN></P><FIGURE class="resultfig horizontal"><FIGCAPTION><B>Figure 2. Induction of mammalian stress responses as measured by UC Davis.</B> A slight increase in signal is observed, but the error bars are too high to draw meaningful conclusions from this data. For more information
            about
            the experiment and relevant protocols, visit the wiki of UC Davis. </FIGCAPTION></FIGURE><P>In return, iGEM UC Davis requested us to test the effects of hydrogen peroxide on bacterial cells to allow iGEM UC Davis to better understand their own results on testing hydrogen peroxide on mammalian cells. We tested hydrogen peroxide
          at a concentration of 100 ppm on 10 different stress reactive strains (Figure 3). The strains pSoxS, pUmuc, pBaeS and plbpA do not show a positive stress response when exposed to 100 ppm hydrogen peroxide. However, the strains <SPAN class="blobs"><A href="http://parts.igem.org/Part:BBa_K2610018" class="popoutBlob" target="_blank">The KatG gene encodes for hydroperoxidase I. It protects aerobic, phosphate-starved cells from oxidative damage. Read more about pKatG on its iGEM
              registry
              page</A><SPAN> pKatG</SPAN></SPAN>, <SPAN class="blobs"><A href="http://parts.igem.org/Part:BBa_K2610021" class="popoutBlob" target="_blank"> RpoE is involved in the attachtment of RNA polymerase ti specific initiation sites. Therefore
              this promoter is related to the stress pathway related to protein synthesis<SUP>[5]</SUP>. Read more about pRpoE on its iGEM registry page </A><SPAN> pRpoE</SPAN></SPAN>, <SPAN class="blobs"><A href="http://parts.igem.org/Part:BBa_K2610025" class="popoutBlob" target="_blank"> Spy is a chaperone protein related to the folding of proteins. This promoter is therefore related to protein synthesis stress<SUP>[6]</SUP>. Read more about pSpY on its iGEM registry page]</A><SPAN>
              pSpY
            </SPAN></SPAN>, <SPAN class="blobs"><A href="http://parts.igem.org/Part:BBa_K2610026" class="popoutBlob" target="_blank"> pInaA is involved in the regulation of homeastasis<SUP>[7]</SUP>. Read more about pInaA on its iGEM registry page
            </A><SPAN>
              pInaA </SPAN></SPAN> and <SPAN class="blobs"><A href="http://parts.igem.org/Part:BBa_K2610022" class="popoutBlob" target="_blank"> CpxR is a chaperone protein related to the folding of proteins. This promoter is therefore related to
              protein synthesis stress<SUP>[6]</SUP>. Read more about CpxR on its iGEM registry page</A><SPAN> pCpxR</SPAN></SPAN> do show a positive stress response following hydrogen peroxide exposure. pSoxS, pUmuc, pBaeS and plbpA are
          involved in
          DNA damage stress responses<SUP>[1][2]</SUP>, envelope stress <SUP>[3]</SUP> and cell division<SUP>[4]</SUP>, respectively. For these strains hydrogen peroxide does not invoke a stress reaction, indicating that these pathways are not
          upregulated and suggesting the
          bacterial cell does not experience these kinds of stress. Because these stress visualizing strains reacted to the exposure to 100ppm H2O2 these results indicate an involvement of H2O2 in the stress pathways of oxidative stress, protein
          synthesis and homeostasis stress.</P><FIGURE class="resultfig"><FIGCAPTION><B>Figure 3. H<SUB>2</SUB>O<SUB>2</SUB> induces a stress response in <I>E. coli</I>.</B> 10 stress detecting strains were expoxed to 100 ppm hydrogen peroxide (+) and a negative control without hydrogen peroxide (-). H<SUB>2</SUB>O<SUB>2</SUB>
            does not induce a response in the wildtype, which does not contain a GFP gene and is therefore used
            as negative control. The strains containing pSoxS, pUmuc, pBaeS and plbpA also do not show a stress response when exposed to 100 ppm hydrogen peroxide. The strains pKatG, pRpoE, pSpy, pInaA and pCpxR do show a positive stress response
            due to hydrogen peroxide exposure. </FIGCAPTION></FIGURE><P>We look back on this collaboration as a very positive influence on our project as we were able to test the toxicity of nalidixic acid, a known antibiotic, on mammalian cells. We think this is a very meaningful addition to our own system.
          Thank you iGEM UC Davis for your collaborative input and we look forward to meeting all of you in Boston!
        </P><DIV class="smaller refs"><H3>References</H3><P id="Ref1">[1]: iGEM Registry of Standard Biological Parts. (2013) Part:BBa_K554000. Retrieved October 10, 2018 from http://parts.igem.org/Part:BBa_K554000</P><P id="Ref2">[2]: UniProt. (2018, september 12) <I>P04152 (UMUC_ECOLI)</I>. Retrieved October 10, 2018 from https://www.uniprot.org/uniprot/P04152</P><P id="Ref3">[3]: Raffa RG, Raivio TL. (2002, september). <I>A third envelope stress signal transduction pathway in Escherichia coli</I>. Mol Microbiol. 45(6):1599-611.
          </P><P id="Ref3">[4]: Tao, J., Sang, Y., Teng, Q., Ni, J., Yang, Y., Tsui, S. K.-W., &amp; Yao, Y.-F. (2015). <I>Heat shock proteins IbpA and IbpB are required for NlpI-participated cell division in Escherichia coli.</I> Frontiers in Microbiology,
            6, 51.
          </P><P id="Ref3">[5]: UniProt. (2018, september 12) <I>P0AGB6 (RPOE_ECOLI)</I>. Retrieved October 10, 2018 from https://www.uniprot.org/uniprot/P0AGB6
          </P><P id="Ref3">[6]: Srivastava SK, Lambadi PR, Ghosh T, Pathania R, Navani NK. (2014, september 10) <I>Genetic regulation of spy gene expression in Escherichia coli in the presence of protein unfolding agent ethanol. Gene</I> 548(1):142-8.
          </P><P id="Ref3">[7]: UniProt. (2018, july 18) <I>P27294 (INAA_ECOLI)</I>. Retrieved October 10, 2018 from https://www.uniprot.org/uniprot/P27294
          </P></DIV></DIV></SECTION><SECTION class="popout chapter"><DIV class="collapsible">Mentoring the Rotterdam iGEM team</DIV><DIV class="collapsibleContent"><P>Early into the iGEM competition we were contacted by <A href="https://2018.igem.org/Team:Rotterdam_HR" class="colouredlink" target="_blank">iGEM Rotterdam</A>. At this point, they were just starting up the first Rotterdam iGEM team, which
          in fact also is the first team of a Dutch university of applied sciences to ever compete in this competition. Amongst others, they were looking for pointers on how to choose their project. Of course we gladly met up with them to assist in
          any way we could. This formed the foundation for our mentorship of iGEM Rotterdam during the iGEM 2018 competition.Over the course of the competition we had several coaching sessions with iGEM Rotterdam in order to discuss their
          progress and to pass on information that has been passed on to us from our predecessors. We also shared our own findings with them. 
          Sessions started with a general introduction concerning laboratory progress, fundraising, PR and human practices. After the introduction, the teams were divided into expert groups in order to discuss problems in more detail. This gave
          both teams the opportunity to learn from the other team’s ideas on everything in the iGEM competition. We provided them with tips and tricks on ligation, social media and crowdfunding campaigns. During the second coaching session we delved
          deeper into negotiations with the university and designing promotional material, notably the banner for the Jamboree. Moreover, the team managers remained in <SPAN title="#BFFs">close contact</SPAN> throughout the competition in order to
          address possible problems as
          quickly and as efficiently as possible.  We consider it an honour to have gotten the chance to mentor such a nice and dedicated team of individuals and are grateful for all that we were able to learn in return. We know that they
          will excel in Boston with their magnificent hardware project.
        </P></DIV></SECTION><SECTION class="popout chapter"><DIV class="collapsible">Dutch Meet-up and BBQ</DIV><DIV class="collapsibleContent"><P>After meeting the other Dutch teams for the first time at the National Institute for Public Health and Environment in April, we wanted to remain in contact. However, the next Dutch meeting - the Eindhoven mini Jamboree - was scheduled for
          the end of October. Therefore, we were inspired to organise an informal meet-up opportunity during the summer. Laboratory work would be in full swing for all teams and we thought it would be a nice moment to get together and share our
          experiences. We were are happy that all Dutch teams were represented during our Meet-up!

          The Meet-up kicked off with some snacks and drinks during which the teams could relax and get to know each other in a more personal way. After this, the BBQ was fired up.
          The teams could enjoy delicious food prepared by our own chef, <A href="https://2018.igem.org/Team:Leiden/Team">Chiel van Amstel</A>. Serving halal meat, vegan and vegetarian options, we made sure everyone was provided quality food. 
          Next, we hosted a pub quiz to stimulate conversation about all teams’ respective projects. To our pleasant surprise we saw that the teams are well versed in the other Dutch iGEM projects. The absolute winners of the Pub Quiz were iGEM
          teams Groningen and Eindhoven! 
          In short, the Dutch Meet-up and BBQ was a wonderful event where plans for collaborations were furthered and ideas for next meet-ups were exchanged. We would like to thank all teams for a lovely evening. </P><P>
          You can read more about the teams that attended this Meet-up on their collaboration pages:<A href="https://2018.igem.org/Team:Groningen/Collaborations" class="colouredlink" target="_blank">Groningen</A><A href="https://2018.igem.org/Team:TUDelft/Collaborations" class="colouredlink" target="_blank">Delft</A><A href="https://2018.igem.org/Team:TU-Eindhoven/Collaborations" class="colouredlink" target="_blank">Eindhoven</A><A href="https://2018.igem.org/Team:Rotterdam_HR/Collaborations" class="colouredlink" target="_blank">Rotterdam</A><A href="https://2018.igem.org/Team:Utrecht/Collaborations" class="colouredlink" target="_blank">Utrecht</A></P></DIV></SECTION><SECTION class="popout chapter"><DIV class="collapsible"> Other collaborations</DIV><DIV class="collapsibleContent"><H2 class="SubSectionTitle">Instagram Challenge</H2><P>This year we took the opportunity to participate in the “Instagram Challenge For July” by <A href="https://2018.igem.org/Team:Stony_Brook" class="colouredlink" target="_blank">iGEM Stony Brook</A>.
          This allowed us to enlighten our social network about our project, the people behind it and to give them a glance in the life of an iGEM’er. We love initiatives like this which enable participating teams to promote iGEM worldwide, showcase
          team diversity and dedication!</P><H2 class="SubSectionTitle">Music Video</H2><P>Near the end of our time in the lab, we were approached by <A href="https://2018.igem.org/Team:TecCEM" class="colouredlink" target="_blank">iGEM TEC CEM</A> for a music video collaboration. Their idea was to get as many iGEM teams together
          as possible to participate in a music video for the song: “Labs Like you”, which
          was rewritten from “Girls Like You” by Maroon 5. We were very excited to participate, and fortunately - and unsurprisingly - so were many other iGEM teams! There we so many interested teams that iGEM TEC CEM decided to write a second song
          to, “Moves Like iGEM”, based on “Moves Like Jagger” by Maroon 5. We took this fun opportunity and recorded a short piece for the music video, which iGEM TEC CEM edited into the music video. We really appreciate the opportunity that we
          received from them and encourage anyone to view the <A href="https://www.youtube.com/watch?v=R7vdiGLLJZA" class="colouredlink" target="_blank"> result!</A></P><H2 class="SubSectionTitle">European Meet-Up</H2><P>From the 20th to the 22nd of July 2018, <A href="https://2018.igem.org/Team:Munich" class="colouredlink" target="_blank">iGEM Munich</A> organised the first European Meet-up for 2018. As we were eager to meet the other iGEM participants, a
          delegation of our team drove to Munich. Here we enjoyed three days of interesting lectures, poster presentations, social events and sightseeing. It was a great opportunity to meet many iGEM participants and learn about all their great
          projects. We look forward to seeing everyone again in Boston and to learn how all the projects turned out.</P><H2 class="SubSectionTitle">Postcards</H2><P> Over the course of the summer we were approached by <A href="https://2018.igem.org/Team:Duesseldorf" class="colouredlink" target="_blank">iGEM HHU</A> to participate in a very cool collaboration set-up called: Postcards. The idea was to
          let as many teams as wanted to participate design a postcard related to their project. We could all send them to iGEM HHU and they would redistribute them. Sending us 50 of our own design with addresses of all the other teams and one
          from every other participating team. We have received all the postcards and were able to decorate our office with all of them. Thank you iGEM HHU for this great collaboration, we really enjoyed looking through all the great designs! </P><H2 class="SubSectionTitle"> Eindhoven Mini Jamboree</H2><P>We had the pleasure of attending the Mini Jamboree in Eindhoven on the 19th of October. <A target="_blank" href="https://2018.igem.org/Team:TU-Eindhoven" class="colouredlink">Eindhoven</A> organised a mock jamboree in order for the Dutch
          team to practice presenting their posters and their final presentation. They even arranged for professional jury members to be present. Afterwards the day was concluded with food and drinks. We immensely appreciate the effort of iGEM
          Eindhoven and thank them for inviting us.
        </P></DIV></SECTION></DIV><DIV class="UnderPageNav"><A href="https://2018.igem.org/Team:Leiden/InterLab" class="NavBlock"><DIV class="NavBlockText"><SPAN>InterLab</SPAN>Click here to read how we collaborated with laboratories around the world in the Interlab study
      </DIV></A><A href="https://2018.igem.org/Team:Leiden" class="NavBlock"><DIV class="NavBlockText"><SPAN>Homepage</SPAN>Click here to visit out Homepage and continue reading about our project
      </DIV></A><A href="https://2018.igem.org/Team:Leiden/Public_Engagement" class="NavBlock"><DIV class="NavBlockText"><SPAN>Education &amp; Public Engagement</SPAN>Click here to read about the development of our education and public engagement program
      </DIV></A></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>