<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Leiden_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Leiden/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="no-js" id="loader-wrapper"><DIV id="BestOnLargeScreens">Although the mobile version has the same content, this wiki is best viewed on larger screens </DIV><DIV id="BadonEdge">Loading a bunch of figures...</DIV></DIV><DIV class="mainContent"><DIV class="container-text"><H1 class="SectionTitle">Results</H1><P> The world is in sore need of new antibiotics. <SPAN class="blobs"><A href="https://2018.igem.org/Team:Leiden/Description" class="popoutBlob" target="_blank">
            Read more about the issue of antibiotic resistance on our Project Description page</A><SPAN> Antibiotic resistance</SPAN></SPAN> is an ever growing problem and alarmingly few novel antibiotic
        therapies have been developed in the past decades<SUP>[1]</SUP>. Based on <SPAN class="blobs"><A href="https://2018.igem.org/Team:Leiden/Human_Practices" class="popoutBlob" target="_blank"> Read more about our discussions with these experts
            on our Integrated Human Practices page</A><SPAN> conversations with experts</SPAN></SPAN> in the field of antibiotic discovery,
        we determined two key missed opportunities in antibiotic discovery research. Firstly, current screening efforts suffer from tunnel
        vision: only the lethality of compounds is considered. This overlooks a potential goldmine of compounds that are not lethal, but
        stressful. Stressful compounds can be used in synergistic combination therapies that enhance current antibacterial treatments or
        create entirely novel ones. Moreover, combination therapies could delay the onset of antibiotic resistance because multiple resistance
        pathways are required to combat these therapies<SUP>[2]</SUP>. A second missed opportunity is the early determination of information about the
        drugs mode of action, which can aid in shortening drug discovery pipelines.
        
        With these considerations in mind, we aimed to develop a screening platform capable of detecting specific bacterial cell stresses.
        For this, we created a library of reporter strains using promoters of genes involved in stress pathways. We made two types of reporter
        strains: 1) GFP reporters which can be quantitatively analysed using flow cytometry or plate reader measurements, and 2) visual reporters
        that use chromoproteins to create a readout visible to the naked eye. Visual reporters can be used in experiments where qualitative analysis
        is cumbersome or not essential, such as overlays or disk diffusions. Furthermore, visual reporters can be used in labs where expensive equipment
        necessary for GFP measurements is not available.
      </P><DIV class="smaller refs"><H3>References</H3><P id="Ref1">[1]: Silver, L. L. (2011). Challenges of Antibacterial Discovery. Clinical Microbiology Reviews, 24(1), 71–109. https://doi.org/10.1128/cmr.00030-10
        </P><P id="Ref2">
          [2]: Xu, X., Xu, L., Yuan, G., Wang, Y., Qu, Y., &amp; Zhou, M. (2018). Synergistic combination of two antimicrobial agents closing each other’s mutant selection windows to prevent antimicrobial resistance. Scientific Reports, 8(1).
          doi:10.1038/s41598-018-25714-z
        </P></DIV></DIV><SECTION class="popout chapter"><DIV class="collapsible"> Plasmid construction – Creating 26 reporter strains for the detection of cellular stress
      </DIV><DIV class="collapsibleContent"><P>
          To identify genes that are upregulated in response to various stresses in <I>E. coli</I>, <SPAN class="blobs XXL"><A href="https://2018.igem.org/Team:Leiden/Parts" class="popoutBlob" target="_blank"> To consult this literature, please
              visit the
              respective part pages of each promoter. A full list of our parts can be found on our parts page </A><SPAN>literature was consulted.</SPAN></SPAN> Next, to isolate the promoters of these genes, a 300 bp region upstream of the
          stress-related genes was amplified by PCR from <I>E. coli</I> genomic DNA. <SPAN class="blobs"><A href="https://2018.igem.org/Team:Leiden/Parts" class="popoutBlob" target="_blank"> Visit our primers page for a list of primers used</A><SPAN>
              Forward primers</SPAN></SPAN> contained EcoRI, XbaI overhangs and reverse primers contained PstI and SpeI overhangs, in order to create parts in the BioBrick format (Figure 1). Visit our primers page for a list of primers used. The
          promoter fragments were subsequently ligated into chloramphenicol backbones (<A href="http://parts.igem.org/Part:pSB1C3" class="colouredlink" target="_blank">pSB1C3</A>). See figure 2 for an overview of all promoters and their stress
          pathway of origin. To ensure future teams and other researchers will be able to use our work, we submitted a total of 26 stress-related promoters and one novel constitutive promoter (<SPAN class="colouredlink" onclick="ScrollTo('#Chapter5');
          OpenClicked('#Chapter5');">see
            the chromoprotein section</SPAN>) as
          <SPAN class="blobs"><A href="https://2018.igem.org/Team:Leiden/Parts" class="popoutBlob" target="_blank"> Visit our Parts to see all the parts we submitted to the iGEM registry</A><SPAN> basic parts</SPAN></SPAN> to the iGEM part registry.

        </P><FIGURE class="resultfig"><FIGCAPTION><B>Figure 1. Amplification of genomic stress-related promoters.</B> 300 bp regions of upstream of stress related genes in the <I>E. coli</I> genome were amplified by PCR and used for cloning of the reporter constructs. Not all
            amplifications were successful as in the case of pAcpP.
          </FIGCAPTION></FIGURE><FIGURE class="resultfig"><FIGCAPTION><B>Figure 2. Overview of promoters isolated from the <I>E. coli</I> genome and stress pathways of origin
            </B></FIGCAPTION></FIGURE><P>For the creation of reporter strains, our isolated promoters were cloned upstream of <A href="http://parts.igem.org/Part:BBa_E0040" class="colouredlink" target="_blank">GFP</A>,
          <A href="http://parts.igem.org/Part:BBa_J06504" class="colouredlink" target="_blank">mCherry</A>,
          <A href="http://parts.igem.org/Part:BBa_K592009" class="colouredlink" target="_blank">AmilCP</A> (blue chromoprotein) or
          <A href="http://parts.igem.org/Part:BBa_K1033932" class="colouredlink" target="_blank">spisPink</A> (pink chromoprotein) reporter genes. Cloning was performed using the
          <SPAN class="blobs"><A href="https://2018.igem.org/Team:Leiden/Experiments" class="popoutBlob" target="_blank"> Visit our Protocols page for more information</A><SPAN> BioBrick assembly method.</SPAN></SPAN> Correct insertion was verified
          by restriction analysis with NdeI and PvuI, see figure 3. 18 of these constructs were submitted as
          <SPAN class="blobs"><A href="https://2018.igem.org/Team:Leiden/Parts" class="popoutBlob" target="_blank"> Visit our Parts page to view all the parts we submitted to the iGEM registry</A><SPAN> composite parts.</SPAN></SPAN></P><FIGURE class="resultfig"><FIGCAPTION><B>Figure 3. Construction of reporter plasmids using the BioBrick system.</B> A) Plasmid map of a
            typical stress reporter plasmid. Shown here is the <A href="http://parts.igem.org/Part:pSB1C3" class="colouredlink" target="_blank">pSB1C3</A> plasmid containing the promoter of recN followed by the gfp gene,
            yielding a construct of 3169 bp. B). The restriction analysis with NdeI and PvuI used to confirm correct insertion of the promoters.
            Shown are the constructs with the promoters of umuC, soxS, secA, relA, recN, and recA.</FIGCAPTION></FIGURE><H2 class="SubSectionTitle">Conclusion</H2><P>
          26 promoter regions of stress-related genes and one novel constitutive promoter were successfully
          amplified from the <I>E. coli</I> genome, ligated into the chloramphenicol backbone (<A href="http://parts.igem.org/Part:pSB1C3" class="colouredlink" target="_blank">pSB1C3</A>) and submitted as BioBrick basic parts.
          These promoters were used to create reporter strains with GFP, mCherry, AmilCP and spisPink as reporter genes, 18 of which were submitted as BioBrick
          <SPAN class="blobs"><A href="https://2018.igem.org/Team:Leiden/Parts" class="popoutBlob" target="_blank"> Visit our Parts to see all the parts we submitted to the iGEM registry </A><SPAN> composite parts.</SPAN></SPAN></P></DIV></SECTION><SECTION class="popout chapter"><DIV class="collapsible"> Validation of stress-induced GFP expression.
      </DIV><DIV class="collapsibleContent"><H2 class="SubSectionTitle">Characterising our promoter repertoire</H2><P>The goal of our system is to enable detection of specific types of cellular stress using reporter strains.
          To this end, we performed characterisation experiments of our GFP reporter strains using flow cytometry. To establish which reporter strains could successfully detect different
          types of cellular stresses, we incubated cells with a range of stressors with different targets in the cell, see table 1 for an overview.

        </P><FIGURE class="resultfig"><TABLE class="ResultTable"><CAPTION><B>Table 1. Stress pathways targeted and stressors used in the initial characterisation of our GFP reporter strains.</B></CAPTION><TBODY><TR><TH>Stress pathway</TH><TH>Stressor</TH></TR><TR><TD>Oxidative stress</TD><TD>Hydrogen peroxide</TD></TR><TR><TD>Envelope stress</TD><TD>Ethanol</TD></TR><TR><TD>DNA damage</TD><TD>Nalidixic acid, zeocin</TD></TR><TR><TD>RNA synthesis</TD><TD>Rifampicin</TD></TR><TR><TD>Protein synthesis</TD><TD>Apramycin, chloramphenicol, hygromycin, kanamycin, streptomycin, tetracycline</TD></TR><TR><TD>Cell wall synthesis</TD><TD>Ampicillin, carbenicillin, vancomycin</TD></TR></TBODY></TABLE></FIGURE><P>
          After measuring using flow cytometry, results of this preliminary screening were summarised in a heatmap showing relative expression levels which were normalised to incubation without stressors (Figure 4).
          For a detailed description of the measurement process, see our <A href="https://2018.igem.org/Team:Leiden/Experiments" class="colouredlink" target="_blank">Protocols</A> and
          <A href="https://2018.igem.org/Team:Leiden/Measurement" class="colouredlink" target="_blank">Measurement</A> pages.
        </P><FIGURE class="resultfig"><FIGCAPTION><B>Figure 4. Heatmap showing relative GFP expression after exposure to stressors.</B> Values for each promoter are normalised to its respective negative control (no stressor, not shown). Appendix A shows the values with
            standard deviation and number of experiment replications.
            <I>E. coli</I> DH5α is included as a negative control to measure the autofluorescence of cells. For a detailed description of the measurements and
            experimental setups, see the <A href="https://2018.igem.org/Team:Leiden/Experiments" class="colouredlink" target="_blank">Protocols</A> and
            <A href="https://2018.igem.org/Team:Leiden/Measurement" class="colouredlink" target="_blank">Measurement</A> pages.
          </FIGCAPTION></FIGURE><P>
          Our heatmap shows large variation in the response of promoters to our library of stressors, thereby showcasing the potential of a system utilizing
          stress promoters to determine and distinguish bacterial stress types. Of the 24 promoters we tested, 8 promoters showed significant upregulation in
          response to one or a few stressors from our utilized library, shown in table 2. Promoters with no significant response will need further validation
          using additional stressors to fully exploit their use.
        </P><FIGURE class="resultfig"><TABLE class="ResultTable"><TBODY><TR><TH>Reporter construct</TH><TH>BioBrick code</TH><TH>Inducing stressor</TH></TR><TR><TD>pKatG-GFP</TD><TD><A href="http://parts.igem.org/Part:BBa_K2610018" class="colouredlink Blue" target="_blank">BBa_K2610018</A></TD><TD>H2O2</TD></TR><TR><TD>pRelA-GFP</TD><TD>N/A</TD><TD>Carbenicillin, streptomycin</TD></TR><TR><TD>pSoxS-GFP</TD><TD><A href="http://parts.igem.org/Part:BBa_K2610031" class="colouredlink Blue" target="_blank">BBa_K2610031</A></TD><TD>H2O2, nalidixic acid</TD></TR><TR><TD>pSecA-GFP</TD><TD><A href="http://parts.igem.org/Part:BBa_K2610011" class="colouredlink Blue" target="_blank">BBa_K2610011</A> (promoter only)</TD><TD>Kanamycin</TD></TR><TR><TD>pRpoE-GFP</TD><TD><A href="http://parts.igem.org/Part:BBa_K2610021" class="colouredlink Blue" target="_blank">BBa_K2610021</A> (promoter only)</TD><TD>Ethanol, chloramphenicol, nalidixic acid</TD></TR><TR><TD>pSpy-GFP</TD><TD><A href="http://parts.igem.org/Part:BBa_K2610025" class="colouredlink Blue" target="_blank">BBa_K2610025</A> (promoter only)</TD><TD>Ethanol, chloramphenicol, nalidixic acid</TD></TR><TR><TD>pInaA-GFP</TD><TD><A href="http://parts.igem.org/Part:BBa_K2610026" class="colouredlink Blue" target="_blank">BBa_K2610026</A> (promoter only)</TD><TD>H2O2, nalidixic acid</TD></TR><TR><TD>pClpB-GFP</TD><TD><A href="http://parts.igem.org/Part:BBa_K2610020" class="colouredlink Blue" target="_blank">BBa_K2610020</A></TD><TD>Ethanol</TD></TR></TBODY></TABLE></FIGURE><H2 class="SubSectionTitle">Validating our promoters using confocal microscopy</H2><P>

          The pSoxS-GFP (<A href="http://parts.igem.org/Part:BBa_K2610031" class="colouredlink" target="_blank">BBa_K2610031</A>) BioBrick was also validated using confocal microscopy. Incubation of our reporter strain with
          nalidixic acid can be seen
          to increase GFP expression compared to basal GFP expression levels (Figure 5).

        </P><FIGURE class="resultfig"><FIGCAPTION><B>Figure 5. Visualisation of GFP expression of the pSoxS-GFP (<A href="http://parts.igem.org/Part:BBa_K2610031" class="colouredlink" target="_blank">BBa_K2610031</A>) strain during nalidixic acid treatment
              using confocal microscopy.</B>
            Wild-type DH5α and the pSoxS-GFP (<A href="http://parts.igem.org/Part:BBa_K2610031" class="colouredlink" target="_blank">BBa_K2610031</A>) strain were incubated with nalidixic acid (50 ng/mL) for two hours.
            A/B)
            Wild-type DH5α.
            C/D) pSoxS-GFP without stressor. Basal GFP expression can be observed. E/F) pSoxS-GFP with 50 ng/mL nalidixic acid. An increase in expression
            compared to the negative control can be observed. Pictures were obtained by using a confocal microscope. The scale bar represents 10 µm.
          </FIGCAPTION></FIGURE><P>
          One of our BioBricks, pCspA-GFP (<A href="http://parts.igem.org/Part:BBa_K2610037" class="colouredlink" target="_blank">BBa_K2610037</A>), showed a significant downregulation as a specific response to all protein
          synthesis inhibitors in our
          library. It should be noted that this response could be caused by inhibition of GFP production due to the protein synthesis stressors.
          Similarly, in our <A href="https://2018.igem.org/Team:Leiden/Measurement" class="colouredlink" target="_blank">Measurement</A> page we show that protein synthesis inhibitors lead to lower observed GFP expression for constitutive promoters,
          the same could be occurring with pCspA which has a high basal expression level.
          
          However, we still believe this result to be relevant as other promoters show no apparent specific downregulation by protein synthesis inhibitors only. Furthermore, we validated this construct further using confocal microscopy (Figure 6).
          We induced the pCspA-GFP (<A href="http://parts.igem.org/Part:BBa_K2610037" class="colouredlink" target="_blank">BBa_K2610037</A>)
          BioBrick with kanamycin and included a pSoxS-mCherry (<A href="http://parts.igem.org/Part:BBa_K2610032" class="colouredlink" target="_blank">BBa_K2610032</A>) as a positive control for protein synthesis. Although the
          base-level signal of mCherry is much lower than that of GFP, we can see that GFP expression is selectively inhibited when the strains are treated with kanamycin while mCherry is not. 
          Whether the culprit of decreased GFP expression is protein synthesis inhibition or promoter regulation could be confirmed further in the future by using a <I>Tet-On Tet-Off</I> system<SUP>[3]</SUP>. Here, downregulation of pCspA would
          cause an increase in GFP expression. A response of pCspA due to protein synthesis inhibitors could then be confirmed by the observation that incubation with such agents would lead to an increase in GFP production.
        </P><FIGURE class="resultfig"><FIGCAPTION><B>Figure 6. Selective downregulation of GFP expression in the pCspA-GFP strain. </B>
            A coculture of pCspA-GFP (<A href="http://parts.igem.org/Part:BBa_K2610037" class="colouredlink" target="_blank">BBa_K2610037</A>) and pSoxS-mCherry (<A href="http://parts.igem.org/Part:BBa_K2610032" class="colouredlink" target="_blank">BBa_K2610032</A>)
            was incubated for two hours with ampicillin (50 µg/mL), nalidixic acid (50 ng/mL) and a combination of nalidixic acid and kanamycin (50 µg/mL). A) Treatment with ampicillin as a
            negative control. A high base level of GFP expression from the pCspA-GFP strain can be observed. B) Treatment with nalidixic acid for induction of pSoxS-mCherry showed no significant increase in mCherry expression. C) Treatment with
            nalidixic acid and kanamycin decreases expression of GFP while mCherry levels are unaffected.</FIGCAPTION></FIGURE><H2 class="SubSectionTitle">Determining promoter dose dependency</H2><P>
          We validated the GFP reporters pKatG (<A href="http://parts.igem.org/Part:BBa_K2610018" class="colouredlink" target="_blank">BBa_K2610018</A>), pSoxS (<A href="http://parts.igem.org/Part:BBa_K2610031" class="colouredlink" target="_blank">BBa_K2610031</A>),
          pSpy (<A href="http://parts.igem.org/Part:BBa_K2610025" class="colouredlink" target="_blank">BBa_K2610025</A>), pRpoE (<A href="http://parts.igem.org/Part:BBa_K2610021" class="colouredlink" target="_blank">BBa_K2610021</A>)
          and pInaA (<A href="http://parts.igem.org/Part:BBa_K2610026" class="colouredlink" target="_blank">BBa_K2610026</A>) further with a dose-dependence study using the stressors they were originally found to respond
          strongest to, the results of which are shown in figure 7. For all promoters, we see a clear dose-dependent response. Our pSoxS-GFP (<A href="http://parts.igem.org/Part:BBa_K2610031" class="colouredlink" target="_blank">BBa_K2610031</A>)
          BioBrick in particular displays the power of our system. An increase of GFP expression can be observed at a concentration 20 times lower than the observed lethal dose.
        </P><FIGURE class="resultfig wide"><FIGCAPTION><B>Figure 7. Dose-dependence study of selected reporter constructs.</B> GFP reporters of pKatG, pSoxS, pSpy, pRpoE and pInaA were treated with increasing concentrations of the stressors shown to induce these promoters in the
            preliminary screening (figure 4). A decrease in detected GFP expression at higher concentrations can be attributed to lethality of the stressor.
          </FIGCAPTION></FIGURE><H2 class="SubSectionTitle">Conclusion</H2><P>Flow cytometry measurements with promoter-GFP constructs show that reporter strains for different types of cellular stress can be selectively and dose-dependently induced by different types of stressors. The selectivity in particular
          establishes an important proof of concept for our system, as this shows that multiple reporters could theoretically be combined to detect different stresses in a single cell line. Of 26 promoters, 8 showed a significant response to our
          library of stressors. More research is needed to discover inducing compounds for the remaining reporter strains.</P><DIV class="smaller refs"><H3>References</H3><P id="Ref3">[3]: Baron, U., &amp; Bujard, H. (2000). Tet repressor-based system for regulated gene expression in eukaryotic cells: Principles and advances. Methods in Enzymology, 327, 401–421. https://doi.org/10.1016/s0076-6879(00)27292-3
          </P></DIV></DIV></SECTION><SECTION class="popout chapter"><DIV class="collapsible"> Signal amplification
      </DIV><DIV class="collapsibleContent"><P>
          We have established that our stress reporters can be used to detect cell stress at stressor concentrations much lower than the lethal dose.
          To improve these results, we attempted to increase the dose response GFP expression even further, to allow for detection of even lower
          concentrations of stressful substances. Additionally, increased signal intensity would facilitate GFP-based stress detection with less sensitive
          detection methods. Furthermore, a readout may be obtained in a shorter time period, which would increase throughput of compound screening.
          This amplification method could also be applied to our visual reporters based on chromoproteins (discussed further in the <SPAN class="colouredlink" onclick="ScrollTo('#Chapter5');
          OpenClicked('#Chapter5');">visual stress
            detection system section</SPAN>).
          Although we have shown our chromoprotein stress reporters work, the readout is not easily visible and only occurred after two to three days.
          

          We attempted to amplify the reporter signal in two ways. First of all, we tried to create a T7 RNA amplification system. For this, we coupled stress promoters to the T7 RNA Polymerase gene (<A href="http://parts.igem.org/Part:BBa_K2610007" class="colouredlink" target="_blank">BBa_K2610007</A>).
          This should result in the T7 RNA Polymerase being produced in response to stress. Next, we coupled a T7 Promoter (<A href="http://parts.igem.org/Part:BBa_K2610005" class="colouredlink" target="_blank">BBa_K2610005</A>)
          - which can only be read by the T7 RNA Polymerase -
          to visual reporters. Unfortunately, we were not able to validate a working T7 enhancement system. Despite this, we have submitted our pSoxS-T7 RNA Polymerase (<A href="http://parts.igem.org/Part:BBa_K2610028" class="colouredlink" target="_blank">BBa_K2610028</A>)
          and pT7-GFP (<A href="http://parts.igem.org/Part:BBa_K2610029" class="colouredlink" target="_blank">BBa_K2610029</A>) BioBricks to the registry in order to give others the
          opportunity to test this system.
          
          Thereafter, we tried increasing the amount of promoter regions and GFP genes on one plasmid, see table 3 for the constructs used.
        </P><FIGURE class="resultfig"><TABLE class="ResultTable"><CAPTION><B>Table 3. Constructs used for signal amplification</B></CAPTION><TBODY><TR><TH>Construct</TH><TH>BioBrick code</TH></TR><TR><TD>pSoxS-GFP</TD><TD><A href="http://parts.igem.org/Part:BBa_K2610031" class="colouredlink Blue" target="_blank">BBa_K2610031</A></TD></TR><TR><TD>pSoxS-GFP-pSoxS-GFP</TD><TD><A href="http://parts.igem.org/Part:BBa_K2610034" class="colouredlink Blue" target="_blank">BBa_K2610034</A></TD></TR><TR><TD>pSoxS-GFP-GFP-pSoxS-GFP-GFP</TD><TD><A href="http://parts.igem.org/Part:BBa_K2610035" class="colouredlink Blue" target="_blank">BBa_K2610035</A></TD></TR></TBODY></TABLE></FIGURE><P>
          Using these amplified BioBricks, a nalidixic acid dose-dependency study was performed.
          A clear increase in both absolute and relative signals compared to the negative controls can be observed in the amplified BioBricks (pSoxS-GFP-pSoxS-GFP and pSoxS-GFP-GFPpSoxS-GFP-GFP), compared to pSoxS-GFP (Figure 8). This shows that
          the GFP signal can be amplified by simple multiplication of promoter regions and GFP genes on one plasmid.
        </P><FIGURE class="resultfig wide"><FIGCAPTION><B>Figure 8. Amplification of the signal obtained from the pSoxS-GFP strain.</B> Increasing promoter regions and GFP genes of our pSoxS-GFP reporter is shown to increase both absolute and relative signals after treatment with
            nalidixic acid. These measurements were performed in the same flow cytometry session so that mean fluorescence intensity (MFI) values can be confirmed.
          </FIGCAPTION></FIGURE><P>
          A disadvantage of this multiplication is that it also increases the basal expression level. Nevertheless, we observe an increase in relative expression for the amplified BioBricks, meaning they are still beneficial. In the future, we hope
          to increase signal strength further and keep the GFP expression as low as possible when the strain is not stressed. Further research into the T7-RNA-polymerase amplification system could provide the answer to this challenge.

        </P><H2 class="SubSectionTitle">Conclusion</H2><P>

          We successfully amplified the pSoxS-GFP stress reporter strain by increasing the amount of promoters and GFP genes in the plasmid. This enables detection of even lower concentrations of stressful substances, allows signals to be detected
          by
          less sensitive detection devices and facilitates faster signal detection. When a visual reporter such as a chromoprotein is used, the time to obtain a significant readout may also be reduced.
        </P></DIV></SECTION><SECTION class="popout chapter"><DIV class="collapsible"> Validating our system for non-antibiotic compounds
      </DIV><DIV class="collapsibleContent"><P>With earlier experiments, we have shown that our system can be used to detect lethal substances in low - stressful - concentrations. In our final product, we want to be able to detect non-lethal stressful substances for use in synergistic
          combination therapies with other stressful substances. In this chapter, we show that our system can be used to detect stress caused by non-lethal compounds which are capable of enhancing the effect of antibiotics.
        </P><H2 class="SubSectionTitle">Verification of non-lethality
        </H2><P>
          To be able to test non-lethal stressful substances, we first had to ensure substances we were using were not lethal. To that end, we tested 19 natural compounds, or extracts thereof, available in the supermarket and sometimes heralded in
          popular culture for their medicinal properties. We first assessed the growth of <I>E. coli</I> following addition of the potential stressors compared to the addition of water. Three of these compounds were found to be lethal in the
          concentration
          used, as seen in figure 9. Due to time constraints, we decided to test three compounds which did not shows lethality and growth inhibitions, and we thought were interesting based on literature. These three substances are ascorbic acid<SUP>[4][5]</SUP>,
          cannabis<SUP>[6][7]</SUP> and ginger<SUP>[8][9]</SUP>.

        </P><FIGURE class="resultfig"><FIGCAPTION><B>Figure 9. Assessment of lethality of 19 products available in the supermarket.</B> Wild-type DH5α was incubated with natural products and OD was measured in a plate reader. The growth was normalised to the negative control
            (addition of milliQ).
          </FIGCAPTION></FIGURE><H2 class="SubSectionTitle">Stress induction of non-lethal compounds
        </H2><P>
          Extracts of ginger and cannabis and a vitamin C solution were incubated with GFP reporter strains containing promoters of the following genes: <I>dnaK, cutF, clpB, rseA, cspA, relA, and nusA</I>, results of which are shown in figure 10.
          These promoters were chosen for two reasons: 1) they originate from a broad spectrum of stress pathways and 2) we wanted to validate promoters that did not show a significant response in the preliminary screening (with the exception of
          clpB).
        </P><FIGURE class="resultfig"><FIGCAPTION><B>Figure 10. pClpB is induced by ascorbic acid. </B> 7 promoter-GFP strains were incubated with ascorbic acid, ginger and cannabis. Only pClpB-GFP (<A href="http://parts.igem.org/Part:BBa_K2610020" class="colouredlink" target="_blank">BBa_K2610020</A>)
            showed a significant response to ascorbic acid.
          </FIGCAPTION></FIGURE><P>
          The highest response was seen for ascorbic acid with the clpB promoter. We therefore decided to continue with ascorbic acid and performed a dose-dependence study. To exclude the possibility of stress occurring due to the acidity of
          ascorbic acid, we included pH controls. These were determined by measuring the pH of ascorbic acid solutions in LB and bringing fresh LB samples to the same pH using concentrated HCl. The results are shown in figure 11.

        </P><FIGURE class="resultfig horizontal"><FIGCAPTION><B>Figure 11. Dose-dependency of pClpB-GFP (<A href="http://parts.igem.org/Part:BBa_K2610020" class="colouredlink" target="_blank">BBa_K2610020</A>) induction by ascorbic acid.</B> Ascorbic acid
            concentrations are denoted in mg/mL. A significant stress response higher than the pH control was observed for incubation with 2.5 and 10 mg/mL ascorbic acid.
          </FIGCAPTION></FIGURE><P>
          Ascorbic acid is seen to induce significant stress not observed in the pH control for the clpB promoter at a concentration of 2.5 mg/mL. However, the GFP response decreases sharply at 5 mg/mL and rises again at 10 mg/mL. This is contrary
          to expectations, as we expected to see a dose response, similar to figure 7. Unfortunately, due to time constraints we were unable to repeat the experiment to rule out an experimental error. Still, based on the results in figures 9 and 10
          we can draw the cautious conclusion that ascorbic acid induces the clpB promoter and is stressful at these concentrations.
          
          Having confirmed that ascorbic acid induces cellular stress, we set out to confirm its synergistic use in combination with antibiotics. The MIC values of the antibiotics were determined with and without the addition of ascorbic acid
          (Figure 12). A clear drop in the MIC value was observed for ampicillin, chloramphenicol and nalidixic acid, up to 82% in the case of chloramphenicol. This is a
          remarkable result considering these three antibiotics all have different modes of action and ascorbic acid is apparently able to enhance all of them.
        </P><FIGURE class="resultfig wide"><FIGCAPTION><B>Figure 12. Addition of ascorbic acid dramatically reduces MIC values of selected antibiotics. </B> The MIC values of ampicillin, apramycin, chloramphenicol, nalidixic acid, tetracycline and streptomycin were determined with
            and without the addition of ascorbic acid (5 mg/mL).
          </FIGCAPTION></FIGURE><H2 class="SubSectionTitle">Conclusion</H2><P>
          In the future, we intend our stress detection system to be used for discovery of non-lethal compounds that induce cellular stress. These substances can synergistically enhance current antibiotics or be used in novel combination therapies.
          We have shown that ascorbic acid is an ideal candidate compound. First, we verified its non-lethality. Next, we observed that it induced the <I>clpB</I> promoter showing that it is a stressful compound. Finally, we proved that ascorbic
          acid enhances the effect of ampicillin, chloramphenicol and nalidixic acid. These experiments demonstrate that our system is capable of detecting stressful compounds, which can be used in synergistic combination therapies.

          </P><DIV class="smaller refs"><H3>References</H3><P id="Ref4">[4] Tajkarimi, M., &amp; Ibrahim, S. A. (2011). Antimicrobial activity of ascorbic acid alone or in combination with lactic acid on Escherichia coli O157:H7 in laboratory medium and carrot juice. Food Control, 22(6), 801–804.
              https://doi.org/10.1016/j.foodcont.2010.11.030

            </P><P id="Ref5">[5] Mathew, S., Verghese, R., &amp; David, A. (2017). Antimicrobial activity of Vitamin C demonstrated on uropathogenic Escherichia coli and Klebsiella pneumoniae. Journal of Current Research in Scientific Medicine, 3(2), 88.
              https://doi.org/10.4103/jcrsm.jcrsm_35_17

            </P><P id="Ref6">[6] Ali, E. M. M., Almagboul, A. Z. I., Khogali, S. M. E., &amp; Gergeir, U. M. A. (2012). Antimicrobial Activity of Cannabis sativa L. Chinese Medicine, 03(01), 61–64. https://doi.org/10.4236/cm.2012.31010

            </P><P id="Ref7">[7] Chakraborty, S., Afaq, N., Singh, N., &amp; Majumdar, S. (2018). Antimicrobial activity of Cannabis sativa, Thuja orientalis and Psidium guajava leaf extracts against methicillin-resistant Staphylococcus aureus. Journal of
              Integrative Medicine, 16(5), 350–357. https://doi.org/10.1016/j.joim.2018.07.005

            </P><P id="Ref8">[8] Patel, R. V., Thaker, V. T., &amp; Patel, V. (2011). Antimicrobial activity of ginger and honey on isolates of extracted carious teeth during orthodontic treatment. Asian Pacific Journal of Tropical Biomedicine, 1(1),
              867–871. https://doi.org/10.1016/s2221-1691(11)60124-x

            </P><P id="Ref9">[9] Gupta, S., &amp; Ravishankar, S. (2005). A Comparison of the Antimicrobial Activity of Garlic, Ginger, Carrot, and Turmeric Pastes Against Escherichia coli O157:H7 in Laboratory Buffer and Ground Beef. Foodborne Pathogens
              and Disease, 2(4), 330–340. https://doi.org/10.1089/fpd.2005.2.330

            </P></DIV></DIV></SECTION><SECTION class="popout chapter"><DIV class="collapsible"> A visual stress detection system</DIV><DIV class="collapsibleContent"><H2 class="SubSectionTitle">A new positive control - Isolation and characterisation of a novel constitutive promoter
        </H2><P>During our project, we isolated the GapA promoter (<A href="http://parts.igem.org/Part:BBa_K2610040" class="colouredlink" target="_blank">BBa_K2610040</A>) from the <I>E. coli</I> genome. Contrary to our stress
          promoters,
          this promoter has a constitutive high basal expression level, as can be seen in the spun down cultures of our pink and blue chromoprotein BioBricks:
          pGapA-AmilCP (<A href="http://parts.igem.org/Part:BBa_K2610041" class="colouredlink" target="_blank">BBa_K2610041</A>) and pGapA-spisPink (<A href="http://parts.igem.org/Part:BBa_K2610042" class="colouredlink" target="_blank">BBa_K2610042</A>),
          Figure 13. In contrast to many synthetic promoters, this novel promoter provides an indication of high levels of <I>natural</I> expression. Therefore, this promoter can be used as a future positive control of reporter production - such as
          GFP - in a natural situation.
        </P><FIGURE class="resultfig horizontal"><FIGCAPTION><B>Figure 13. Pelleted liquid cultures of pGapA-chromoprotein strains show high levels of expression. </B> The shown pellets were spun down from 50 mL of overnight culture.
          </FIGCAPTION></FIGURE><H2 class="SubSectionTitle">Chromoprotein-based stress detection for overlays and disk diffusion</H2><P><SPAN class="blobs XXL"><A style="cursor:default" class="popoutBlob" target="_blank"> Experiments in which potential antibiotic producing strains or disks containing antibiotics are placed on a plate, after which a bacterial test strain is
              seeded on top.
              Compounds
              diffuse through the agar and - if lethal - prevent bacterial growth surrounding the strain or disk, thereby creating a visible “zone of inhibition&quot;</A><SPAN> Overlays and disk diffusions</SPAN></SPAN> have been pivotal in the past for
          the discovery and characterisation of antibiotics<SUP>[10][11]</SUP>. Because these experiments are carried out on agar plates, a GFP-based readout is not suitable, since GFP production would be cumbersome to measure reproducibly on a
          plate. Furthermore, quantitative measurements are not essential in this case, as these techniques are purely used to identify interesting compounds or antibiotic producing strains. Therefore, we chose for a
          <SPAN class="blobs"><A href="https://2018.igem.org/Team:Leiden/Human_Practices" class="popoutBlob" target="_blank"> Read more about this decision on our Integrated Human Practices page </A><SPAN>readout visible to the naked
              eye</SPAN></SPAN> in the form of chromoproteins.
          Disk diffusion and overlay experiments display halos where bacteria cannot grow. At the edge of such a halo, there is a zone with a <SPAN class="blobs"><A href="https://2018.igem.org/Team:Leiden/Model" class="popoutBlob" target="_blank">
              Read more about this zone, and the determination of it on our Model page </A><SPAN> non-lethal antibiotic concentration</SPAN></SPAN>. In this zone we could potentially detect cellular stress. To test this, we created a plasmid
          containing a blue chromoprotein regulated by a stress promoter, pSoxS-AmilCP (<A href="http://parts.igem.org/Part:BBa_K2610033" class="colouredlink" target="_blank">BBa_K2610033</A>) and pGapA-spisPink (<A href="http://parts.igem.org/Part:BBa_K2610033" class="colouredlink" target="_blank">BBa_K2610033</A>). From our flow cytometry experiments, we knew that pSoxS should react to nalidixic acid. We attempted a disk diffusion with our pSoxS-AmilCP BioBricks, which is shown in figure 14. A
          discoloration around the halo can be seen, which is not seen in the negative control. This shows we can visualize bacterial cell stress using chromoproteins.

        </P><FIGURE class="resultfig wide"><FIGCAPTION><B>Figure 14. Cellular stress can be visualised on agar-plates using chromoprotein reporter strains. </B> Disk diffusions with nalidixic acid were carried out using wild-type DH5α and our pSoxS-AmilCP (<A href="http://parts.igem.org/Part:BBa_K2610033" class="colouredlink" target="_blank">BBa_K2610033</A>) strain.
            A) A faint discolouration around the halos in the right agar plate can be observed which is indicative of non-lethal cellular stress. B) Disk diffusion with pSoxS-AmilCP (<A href="http://parts.igem.org/Part:BBa_K2610033" class="colouredlink" target="_blank">BBa_K2610033</A>) and nalidixic acid. Two eppendorf tubes containing the pSoxS-AmilCP (<A href="http://parts.igem.org/Part:BBa_K2610033" class="colouredlink" target="_blank">BBa_K2610033</A>)
            strain (left) and a wild-type DH5α (right) are shown for reference. Again, a darker colour around
            the halo is seen in the disk diffusion.
          </FIGCAPTION></FIGURE><P>
          We also attempted to achieve similar results in liquid cultures by incubating bacteria expressing the pSoxS-AmilCP BioBrick with increasing concentrations of nalidixic acid, results of which are shown in figure 15. However, we found no
          discernible difference in color for the stressed vs. unstressed strains. Evidently, the stress caused by nalidixic acid at our used concentrations is not enough to create a difference in expression visible to the naked eye. Further
          optimisation with, among other things, different incubation times is necessary to achieve the desired results.

        </P><FIGURE class="resultfig"><FIGCAPTION><B>Figure 15. Attempted dose-response in chromoprotein liquid cultures.</B> pSoxS-AmilCP (<A href="http://parts.igem.org/Part:BBa_K2610033" class="colouredlink" target="_blank">BBa_K2610033</A>) strains
            were stressed overnight with four concentrations of nalidixic acid. No dose-dependent response was observed.
          </FIGCAPTION></FIGURE><H2 class="SubSectionTitle">Conclusion</H2><P>
          We isolated a novel constitutive promoter from the <I>E. coli</I> genome and showed a high natural expression level of chromoproteins. Additionally, we have shown that chromoproteins can be used as visual reporters for cell stress in
          conditions
          representative of overlay and disk diffusion experiments. This is the first time visual reporters have been used in such a way. This opens the door for many agar plate-based detection systems. In addition, visual reporters can be used in
          labs where expensive plate readers and flow cytometers that measure GFP expression are not available. Since our BioBricks are available on the registry, experiments using these strains can begin today!

        </P><DIV class="smaller refs"><H3>References</H3><P id="Ref10">[10]: Balouiri, M., Sadiki, M., &amp; Ibnsouda, S. K. (2016). Methods for in vitro evaluating antimicrobial activity: A review. Journal of Pharmaceutical Analysis, 6(2), 71–79. https://doi.org/10.1016/j.jpha.2015.11.005
          </P><P id="Ref11">[11]: Lewis, K. (2013). Platforms for antibiotic discovery. Nature Reviews Drug Discovery, 12(5), 371–387. https://doi.org/10.1038/nrd3975
          </P></DIV></DIV></SECTION><SECTION class="popout chapter"><DIV class="collapsible"> Conclusion and future perspectives</DIV><DIV class="collapsibleContent"><H2 class="SubSectionTitle">Conclusion</H2><P>
          The global challenge of antibiotic resistance requires novel approaches in the field of antibiotic discovery. Two major missed opportunities in antibiotic research are the detection of non-lethal stressful compounds and the early
          determination of a compound’s mode of action. With Fifty Shades of Stress, we tackle both issues.
          
          To achieve this, we isolated promoters of stress-related genes from the <I>E. coli</I> genome using PCR and submitted 27 of them as <SPAN class="blobs"><A href="https://2018.igem.org/Team:Leiden/Parts" class="popoutBlob" target="_blank">          Read
              about each of our parts on our Parts page</A><SPAN> basic parts</SPAN></SPAN> to the registry.
          The promoters were cloned upstream of two fluorescent reporter genes, <A href="http://parts.igem.org/Part:BBa_E0040" class="colouredlink" target="_blank">GFP</A> and <A href="http://parts.igem.org/Part:BBa_J06504" class="colouredlink" target="_blank">mCherry</A>,
          and two visual reporter genes (chromoproteins), <A href="http://parts.igem.org/Part:BBa_K592009" class="colouredlink" target="_blank">amilCP</A> (blue) and <A href="http://parts.igem.org/Part:BBa_K1033932" class="colouredlink" target="_blank">spisPink</A>
          (pink). 18 of these constructs were submitted as
          <SPAN class="blobs"><A href="https://2018.igem.org/Team:Leiden/Parts" class="popoutBlob" target="_blank"> Read about each of our parts on our Parts page</A><SPAN> composite parts.</SPAN></SPAN>
          To characterize the stress responsiveness of our promoters, we subjected 24 GFP reporter strains to a library of stressors that elicit different
          types of stress. 8 reporters yielded a clear increase in GFP expression in response to one or more compounds in our library. Several were
          validated further using confocal microscopy and/or dose-response studies. Additionally, one reporter, pCspA-GFP (<A href="http://parts.igem.org/Part:BBa_K2610037" class="colouredlink" target="_blank">BBa_K2610037</A>),
          was presumably selectively downregulated, although more research is needed to confirm this.
          
          To improve signal detection, we have amplified GFP expression by increasing the number of promoters and GFP genes in one plasmid. We performed the experiment using the pSoxS (<A href="http://parts.igem.org/Part:BBa_K2610030" class="colouredlink" target="_blank">BBa_K2610030</A>) BioBrick and the signal increases almost tenfold. These parts were also submitted as composite parts.
          
          To prove that our system could be used to detect stressful compounds, we tested a range of natural products for their lethality. Three of these compounds were tested on different stress promoters and ascorbic acid was found to be
          stress-inducing. Therefore, ascorbic acid was incubated together with several antibiotics and the MIC value was found to drop drastically, up to 82% for ampicillin. Thereby proving our system is capable of detecting stressful compounds
          suitable for antibiotic combination therapies.
          
          For experiments with antibiotics on agar plates - such as disk diffusions or overlays - we set out to create a visual reporter strain using chromoproteins, an approach which has not been attempted before. We performed a disk diffusion
          experiment using our pSoxS-AmilCP (<A href="http://parts.igem.org/Part:BBa_K2610033" class="colouredlink" target="_blank">BBa_K2610033</A>) BioBrick as a reporter and found that a discoloration occurred in the area
          where the antibiotic concentration is non-lethal. This is an important proof of concept for visual reporters to be used in agar-based experiments.

        </P><H2 class="SubSectionTitle">Towards an application</H2><P>
          We have shown that the foundation for Fifty Shades of Stress is solid. The next step is creating an easy-to-use application from our project. To that end, we have created a <SPAN class="blobs"><A href="https://2018.igem.org/Team:Leiden/Applied_Design" class="popoutBlob" target="_blank"> Read about our product design in detail on our Product Design wiki page</A><SPAN> product design</SPAN></SPAN> for an easy-to-use bacterial cell stress detection kit with the name 50S.O.S.
          
          We want to establish a proof of concept for three important aspects of the kit before full scale production can begin. First, we want GFP expression by our reporter strains to be measurable in plate readers. This would make measurement
          faster and open up the possibility of obtaining real-time data. Unfortunately, we experienced repeated complications when measuring in the plate reader available to us, therefore preventing us from examining this possibility further.
          However, quantifying GFP expression in <I>E. coli</I> strains using a plate reader is not new and should work<SUP>[12]</SUP>. We surmise that the cause of our failed plate reader experiments may be unsuitable transparent plates or diluting
          the cultures too much before stressing them.
          
          A second goal for our future application is to enable testing of multiple stress promoters in a single bacterial strain, by using fluorescent reporters with different emission wavelengths. We attempted this with the GFP and mCherry
          reporters using co-transformations and by combining two reporter constructs in one plasmid. However, neither technique yielded a replication of the results obtained with the single-promoter GFP or mCherry plasmids. This may be due to a
          lower basal expression level of mCherry or lower fluorescence intensity. Still, we submitted two combined plasmids as composite parts for other teams and researchers to use and improve: pSoxS-mCherry-pKatG-GFP (<A href="http://parts.igem.org/Part:BBa_K2610043" class="colouredlink" target="_blank">BBa_K2610043</A>) and pSoxS-mCherry-pMicF-GFP (<A href="http://parts.igem.org/Part:BBa_K2610044" class="colouredlink" target="_blank">BBa_K2610044</A>). Also, we do not foresee
          any fundamental reason why our combination plasmids should not have worked<SUP>[13]</SUP>, so we are confident further research could easily allow for the combinations of multiple stress reporters in a single bacterial strain.
          
          A third important aspect of our 50S.O.S. kit is the use of freeze dried strains, to allow for safer GMO transport. This is a common technique and should work according to literature. We have performed such freeze drying experiments with
          our strains using <SPAN class="blobs"><A href="https://2018.igem.org/Team:Leiden/Experiments" class="popoutBlob" target="_blank"> More information can be found on our Protocols page</A><SPAN> glycerol.</SPAN></SPAN>
          Although we were able to revive our strains when using high glycerol concentrations, reviving was no longer successful when using lower concentrations of glycerol. Therefore, we believe future attempts should use sucrose, which is more
          commonly used for freeze drying <I>E. coli</I><SUP>[14]</SUP>.
          
          Although we did not yet achieve success with these three extra aspects for our kit design, the techniques described above all have precedents in literature. That means it is simply a matter of time before they can all be integrated into a
          full-fledged 50S.O.S. system. For now, researchers can use each of our publicly available stress detection strains by obtaining them from the iGEM registry. This way, we are one step closer to discovering novel antibiotic therapies and
          solving the antibiotic resistance crisis.

        </P><DIV class="smaller refs"><H3>References</H3><P id="Ref12">[12] Srivastava, S. K., Lambadi, P. R., Ghosh, T., Pathania, R., &amp; Navani, N. K. (2014). Genetic regulation of spy gene expression in Escherichia coli in the presence of protein unfolding agent ethanol. Gene, 548(1), 142–148.
            https://doi.org/10.1016/j.gene.2014.07.003

          </P><P id="Ref13">[13] Bjørkøy, G., Lamark, T., Pankiv, S., Øvervatn, A., Brech, A., &amp; Johansen, T. (2009). Chapter 12 Monitoring Autophagic Degradation of p62/SQSTM1. Methods in Enzymology, , 181–197.
            https://doi.org/10.1016/s0076-6879(08)03612-4

          </P><P id="Ref14">[14] Leslie, S.B., Israeli, E., Lighthart, B., Crowe, J.H., &amp; Crowe, L.M. (1995). Trehalose and sucrose protect both membranes and proteins in intact bacteria during drying. Appl Environ Microbiol, 3592-3597.

          </P></DIV></DIV></SECTION><SECTION class="popout chapter"><DIV class="collapsible"> Appendix A - Heatmap data</DIV><DIV class="collapsibleContent"><P>
          Below the data for the heatmap with the standard error and the amount of replicates included is shown.
        </P></DIV></SECTION></DIV><DIV class="UnderPageNav"><A href="https://2018.igem.org/Team:Leiden/Measurement" class="NavBlock"><DIV class="NavBlockText"><SPAN>Measurement</SPAN>Click here to read how we ensured good measurement practices during out project
      </DIV></A><A href="https://2018.igem.org/Team:Leiden/Demonstrate" class="NavBlock"><DIV class="NavBlockText"><SPAN>Demonstrate</SPAN>Click here to read how we demonstrated our project
      </DIV></A><A href="https://2018.igem.org/Team:Leiden/Parts" class="NavBlock"><DIV class="NavBlockText"><SPAN>Parts</SPAN>Click here to see all the parts we submitted as BioBricks
      </DIV></A><A href="https://2018.igem.org/Team:Leiden/Model" class="NavBlock"><DIV class="NavBlockText"><SPAN>Model</SPAN>Click here to read how we used modeling to improve our experiments
      </DIV></A></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>